<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Lab_Labjournal_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Lab/Labjournal/September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Lab/Labjournal/September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/September&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Lab/Labjournal/September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Lab/Labjournal/September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round white_bg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#AID"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">AID</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#2012-09-03"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">2012-09-03</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#2012-09-04"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2012-09-04</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#2012-09-05"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">2012-09-05</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#2012-09-06"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">2012-09-06</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#2012-09-07"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">2012-09-07</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#2012-09-08"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">2012-09-08</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#2012-09-09"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">2012-09-09</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#2012-09-10"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">2012-09-10</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#2012-09-11"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">2012-09-11</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#2012-09-12"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">2012-09-12</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#2012-09-13"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">2012-09-13</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#2012-09-14"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">2012-09-14</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#2012-09-15"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">2012-09-15</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#2012-09-16"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">2012-09-16</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#2012-09-17"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">2012-09-17</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#2012-09-18"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">2012-09-18</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#2012-09-19"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">2012-09-19</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#2012-09-20"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">2012-09-20</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#2012-09-21"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">2012-09-21</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#2012-09-22"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">2012-09-22</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-22"><A href="#Antibody"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Antibody</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#2012-09-01"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">2012-09-01</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#2012-09-03_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">2012-09-03</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#2012-09-04_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">2012-09-04</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#2012-09-05_2"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">2012-09-05</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#2012-09-06_2"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">2012-09-06</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#2012-09-07_2"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">2012-09-07</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#2012-09-08_2"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">2012-09-08</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#2012-09-09_2"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">2012-09-09</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#2012-09-10_2"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">2012-09-10</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#2012-09-11_2"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">2012-09-11</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#2012-09-12_2"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">2012-09-12</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#2012-09-13_2"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">2012-09-13</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#2012-09-14_2"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">2012-09-14</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#2012-09-15_2"><SPAN class="tocnumber">2.14</SPAN><SPAN class="toctext">2012-09-15</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#2012-09-16_2"><SPAN class="tocnumber">2.15</SPAN><SPAN class="toctext">2012-09-16</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#2012-09-17_2"><SPAN class="tocnumber">2.16</SPAN><SPAN class="toctext">2012-09-17</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#2012-09-18_2"><SPAN class="tocnumber">2.17</SPAN><SPAN class="toctext">2012-09-18</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#2012-09-19_2"><SPAN class="tocnumber">2.18</SPAN><SPAN class="toctext">2012-09-19</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#2012-09-20_2"><SPAN class="tocnumber">2.19</SPAN><SPAN class="toctext">2012-09-20</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#2012-09-21_2"><SPAN class="tocnumber">2.20</SPAN><SPAN class="toctext">2012-09-21</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#2012-09-22_2"><SPAN class="tocnumber">2.21</SPAN><SPAN class="toctext">2012-09-22</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#2012-09-23"><SPAN class="tocnumber">2.22</SPAN><SPAN class="toctext">2012-09-23</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#2012-09-24"><SPAN class="tocnumber">2.23</SPAN><SPAN class="toctext">2012-09-24</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#2012-09-25"><SPAN class="tocnumber">2.24</SPAN><SPAN class="toctext">2012-09-25</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#2012-09-26"><SPAN class="tocnumber">2.25</SPAN><SPAN class="toctext">2012-09-26</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#2012-09-28"><SPAN class="tocnumber">2.26</SPAN><SPAN class="toctext">2012-09-28</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#Virus"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Virus</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#2012-09-02"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">2012-09-02</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#2012-09-03_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">2012-09-03</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#2012-09-04_3"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">2012-09-04</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#2012-09-05_3"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">2012-09-05</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#2012-09-06_3"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">2012-09-06</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#2012-09-09_3"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">2012-09-09</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#2012-09-10_3"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">2012-09-10</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#2012-09-11_3"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">2012-09-11</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#2012-09-12_3"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">2012-09-12</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#2012-09-13_3"><SPAN class="tocnumber">3.10</SPAN><SPAN class="toctext">2012-09-13</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#2012-09-14_3"><SPAN class="tocnumber">3.11</SPAN><SPAN class="toctext">2012-09-14</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#2012-09-15_3"><SPAN class="tocnumber">3.12</SPAN><SPAN class="toctext">2012-09-15</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#2012-09-16_3"><SPAN class="tocnumber">3.13</SPAN><SPAN class="toctext">2012-09-16</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#2012-09-17_3"><SPAN class="tocnumber">3.14</SPAN><SPAN class="toctext">2012-09-17</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#2012-09-18_3"><SPAN class="tocnumber">3.15</SPAN><SPAN class="toctext">2012-09-18</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#2012-09-19_3"><SPAN class="tocnumber">3.16</SPAN><SPAN class="toctext">2012-09-19</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#2012-09-20_3"><SPAN class="tocnumber">3.17</SPAN><SPAN class="toctext">2012-09-20</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#2012-09-21_3"><SPAN class="tocnumber">3.18</SPAN><SPAN class="toctext">2012-09-21</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#2012-09-22_3"><SPAN class="tocnumber">3.19</SPAN><SPAN class="toctext">2012-09-22</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#2012-09-25_2"><SPAN class="tocnumber">3.20</SPAN><SPAN class="toctext">2012-09-25</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#2012-09-29"><SPAN class="tocnumber">3.21</SPAN><SPAN class="toctext">2012-09-29</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#2012-10-16"><SPAN class="tocnumber">3.22</SPAN><SPAN class="toctext">2012-10-16</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#2012-10-16_2"><SPAN class="tocnumber">3.23</SPAN><SPAN class="toctext">2012-10-16</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="AID">AID</SPAN></H2><H3><SPAN class="mw-headline" id="2012-09-03"><P style="background-color: rgb(240, 20, 70);">2012-09-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Miniprep of mVENUS construct and wildtype AID from transfected CHO cells</P><P><B>Investigators:</B></P><P>Rico
</P><P><B>Aim:</B></P><UL><LI>Miniprep of YFP-construct and wildtype AID from transfected CHO cells to proof whether it is possible to transform E.coli with the purified construct for sequencing
</LI></UL><P><B>Method:</B></P><UL><LI>Miniprep of cells
</LI></UL><UL><LI>Elution with 50 ÂµL
</LI></UL><P><B>Results:</B></P><P>CHO transfected with YFP WT-AID = 2.8 ng/ÂµL</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of mVenus and wildtype AID isolated samples and pCEP4</P><P><B>Investigators:</B> Tom</P><P><B>Time:</B> 2012-09-03</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent E. coli cells (XL 1 Blue)
</LI></UL><UL><LI> Venus plasmid - sample 1
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of miniprep sample and 1 ÂµL of pCEP4 were used
</P><P>Plate incubation start: 1:30 pm
</P><P><B>Results:</B></P><P>ready for growing mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Test digestion (Pst1) &amp; gel electrophoresis of pAK100+scFV</P><B>Investigators:</B>Chris, Rico<B>Aim:</B>*testing the miniprep samples of pAK100+scFV<B><P>Results:</P></B>a 600 bp instead of 480bp<P style="background-color: rgb(238, 221, 130); font-weight: bold;">repetition of test digestion (Pst1) &amp; gel electrophoresis of pAK100+scFV</P><B>Investigators:</B><P>Chris<B>Aim:</B>*testing the miniprep samples of pAK100+scFV</P><P>Results:a 600 bp instead of 480bp</P><H3><SPAN class="mw-headline" id="2012-09-04"><P style="background-color: rgb(240, 20, 70);">2012-09-04</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">preparation of competent E.coli XL1 blue cells</P><P><B>Investigators:</B></P><P>Basia, Chris
</P><P><B>Method:</B></P><P>via standard manual
</P><P><B>Results:</B></P><P>50 Eppendorf tubes with 100 ÂµL competent XL1 blue</P><P><B>further tasks:</B></P><P>testing competence
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transfection of CHO cells with wt AID, AID without NES, with NLS+Kozak sequence, AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B></P><P>Rico
</P><P><B>Aim:</B></P><UL><LI>Transfection of CHO-cells with wt AID + small antibody construct, AID without NES, with NLS+Kozak sequence + small antibody construct, AID without NES, with NLS+Kozak sequence+eGFP + small antibody construct, small antibody construct
</LI></UL><P><B>Method:</B></P><UL><LI> 7 Âµg total DNA was used for transfection in 700 ÂµL total volume:</LI></UL><UL><LI> 3.5 Âµg wt AID + 3.5 Âµg small antibody construct</LI></UL><UL><LI> 3.5 Âµg AID without NES, with NLS+Kozak sequence + 3.5 Âµg small antibody construct</LI></UL><UL><LI> 3.5 Âµg AID without NES, with NLS+Kozak sequence+eGFP + 3.5 Âµg small antibody construct</LI></UL><UL><LI> 7 Âµg small antibody construct</LI></UL><UL><LI> 17.5 ÂµL PEI in 700 ÂµL Optimum/PEI-Mix
</LI></UL><UL><LI> vortex for 10 s 3 x
</LI></UL><UL><LI> 2 min incubation
</LI></UL><UL><LI> mix DNA solution with PEI solution (1:1)
</LI></UL><UL><LI> 15 min incubation
</LI></UL><UL><LI> add 400 ÂµL PEI/DNA-solution to the CHO cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Tom S., Chris</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-2</TD><TD> II3637 </TD><TD> pSB1C3 Forward </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-2 </TD><TD> II3638 </TD><TD> pSB1C3 Reverse </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-2 </TD><TD> II3639 </TD><TD> AID-C-Term Forward </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-2 </TD><TD> II3640 </TD><TD> eGFP-N-Term Reverse </TD></TR><TR><TD> Pak100+ScFv L1 </TD><TD> II3688 </TD><TD> GATC_std_RPC </TD></TR><TR><TD> Pak100+ScFv L1 </TD><TD> II3689 </TD><TD> GATC_std_sr_Hind3-672953 </TD></TR><TR><TD> Pak100+ScFv RH1 </TD><TD> II3690 </TD><TD> GATC_std_RPC </TD></TR><TR><TD> Pak100+ScFv RH1 </TD><TD> II3691 </TD><TD> GATC_std_sr_Hind3-672953 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of pCEP4</P><P><B>Investigators:</B> Tom S.
</P><P><B>Time:</B> 2012-09-04 6 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Amp 25 mg/ml stock solution</LI></UL><UL><LI>plate with cultures: pCEP4 (from 2012-09-03)
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>1 culture of pCEP4 plate in 5 ml LB medium + 5 ÂµL amp.
</P><P>(--&gt; 1 culture)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> repetition of digestion of pAK100&amp; scFV pKMEF425bla (SfiI&amp;AscI)</P><P><B>Investigators:</B> Chris
</P><P><B>Time:</B> 2012-09-04 9 pm</P><P><B>Method:</B></P><P>sample preparation:</P><P>5ÂµL (2000 ng) pAK100, 20 ÂµL H2O, 3 ÂµL Neb3, 2ÂµL AscI </P><P>25 ÂµL ScFV, 3 ÂµL Neb3, 2ÂµL AscI</P><P>incubation over night at 37Â°C
</P><P><B>Further tasks:</B></P><UL><LI> addition of BSA, SfiI and incubation at 50 Â°C </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of ER2738 cells</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-04 7 pm</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, glycerol stock of ER2738 cells</P><P><B>Method:</B> inoculation in 20 ml LB medium + 20Âµl tetracycline stock, shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>preparation of competent cells and helper phage culturing
</P><H3><SPAN class="mw-headline" id="2012-09-05"><P style="background-color: rgb(240, 20, 70);">2012-09-05</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of helper phage</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-05 10am</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, overnight culture of ER2738 cells, helper phage stock solution, Kanamycin, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>amplification and clean up of helper phage according to the manual</P><P><B>Further tasks:</B></P><P>continuation on the next day
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of competent cells ER2738</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-05 10am</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, overnight culture of ER2738 cells, CaCl2, Glycerol</P><P><B>Method:</B></P><P>preparation of competent cells according to the manual</P><P><B>Results:</B></P><P>competent cells ready to use</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of pBad with wtAID</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-05 6pm</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar plates with ampicillin
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent ER2738 cells
</LI></UL><UL><LI> pBad plasmid with wtAID
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 5 Âµl of plasmid was used
</P><P>Plate incubation start: 7:30 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Documentation of growing CHO cells under the microscope (24h incubation) </P><P><B>Investigators:</B> Rico, Tom S.</P><P><B>Time:</B> 2012-09-05</P><P><B>Materials:</B></P><UL><LI> six well plates with growing CHO cells (24h), which were transfected with EGFR-construct alone or in combination with WT AID or AID without NES, with NLS+Kozak sequence or AID without NES, with NLS+Kozak sequence+eGFP
</LI></UL><UL><LI> Fluorescence microscope
</LI></UL><P><B>Method:</B></P><P>take a photo of each well with a filter for GFP, YFP and brightfield
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Isolation of mutation rate samples (EGFR-construct <B>(EGFR-C)</B> alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wildtype AID from transfected CHO-Cells (<B>24h incubation</B>))</P><P><B>Investigators:</B></P><P>Rico, Tom S.
</P><P><B>Aim:</B></P><UL><LI>Isolation of mutation rate samples (EGFR-construct alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wild type AID from transfected CHO cells) to separate the plasmids for sequencing
</LI></UL><P><B>Method:</B></P><UL><LI>Miniprep of cells
</LI></UL><UL><LI>Elution with 20 ÂµL
</LI></UL><P><B>Results:</B></P><P>CHO-transfected with EGFR-C = 20,7 ng/ÂµL</P><P>CHO-transfected with EGFR-C and WT AID = 18,1 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence = 55,7 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence+eGFP = 40,0 ng/ÂµL</P><P><B>Further tasks:</B></P><P>transformation of the plasmids
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of plasmids from cells which grew for 24h (Mutation rate) </P><P><B>Investigators:</B> Tom</P><P><B>Time:</B> 2012-09-05</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent E. coli cells (XL 1 Blue)
</LI></UL><UL><LI> samples of 24h incubation
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 1:2 dilutions of prepped samples were used
</P><P><B>Results:</B></P><P>ready for growing mutants
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-09-06"><P style="background-color: rgb(240, 20, 70);">2012-09-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of helper phage - continuation</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-06 </P><P><B>Materials:</B></P><P>LB medium, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>continuation of clean up of helper phage according to the manual</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of ER2738 cells with AID in pBAD</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-06 5:30 pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, plate with transformed colonies from 5.9.2012</P><P><B>Method:</B> inoculation in 20 ml LB medium + 20Âµl ampicillin stock, shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>preparation of competent ER2738 cells with AID in pBAD
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Documentation of growing CHO-Cells under the microscope (48h incubation) </P><P><B>Investigators:</B> Rico, Tom S.</P><P><B>Time:</B> 2012-09-06</P><P><B>Materials:</B></P><UL><LI> six well plates with growing CHO cells (24h) which were transfected with EGFR-construct alone or in combination with WT AID or AID without NES, with NLS+Kozak sequence or AID without NES, with NLS+Kozak sequence+eGFP
</LI></UL><UL><LI> Fluorescence microscope
</LI></UL><P><B>Method:</B></P><P>take a photo of each well with a filter for GFP, YFP and brightfield
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Isolation of mutation rate samples (EGFR-construct <B>(EGFR-C)</B> alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wild type AID from transfected CHO cells (<B>48h incubation</B>))</P><P><B>Investigators:</B></P><P>Rico, Tom S.
</P><P><B>Aim:</B></P><UL><LI>Isolation of mutation rate samples (EGFR-construct alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wild type AID from transfected CHO cells) to separate the plasmids for sequencing
</LI></UL><P><B>Method:</B></P><UL><LI>Miniprep of cells
</LI></UL><UL><LI>Elution with 20 ÂµL
</LI></UL><P><B>Results:</B></P><P>CHO-transfected with EGFR-C = 45,9 ng/ÂµL</P><P>CHO-transfected with EGFR-C and WT AID = 41,7 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence = 24,8 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence+eGFP = 18,6 ng/ÂµL</P><P><B>Further tasks:</B></P><P>transform plasmids
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Inoculation of plasmid samples of the 24h retransformation plates</P><P><B>Investigators:</B> Tom S.
</P><P><B>Time:</B> 2012-09-06 4pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Amp 25 mg/ ml stock solution</LI></UL><UL><LI>plate with cultures: EGFR-C</LI></UL><P>EGFR-C-WT AID</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP</P><P>(all from 2012.09.06)</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>5 cultures per plate in 5 ml LB medium + 5ÂµL amp.
</P><P>(--&gt; 20 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Checking the sequencing data</P><P><B>Investigators:</B> Tom S.
</P><P><B>Aim:</B> choose the right constructs</P><P><B>Results:</B></P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH in pSB1C3 2-1-2 -&gt; is good</P><H3><SPAN class="mw-headline" id="2012-09-07"><P style="background-color: rgb(240, 20, 70);">2012-09-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of competent cells ER2738+AID in pBAD</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-07 10am</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, overnight culture of ER2738 cells with AID in pBAD, CaCl2, Glycerol</P><P><B>Method:</B></P><P>preparation of competent cells according to the manual</P><P><B>Results:</B></P><P>competent cells ready to use</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of pAK100 with scFV into ER2738 with AID in pBAD </P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-07 7pm</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar plates with ampicillin
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent ER2738 cells
</LI></UL><UL><LI> pAK100 plasmid with scFV L1
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 4 Âµl of plasmid was used
</P><P>Plate incubation start: 7:00 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of 24h cultures for mutation rates</P><P><B>Investigators:</B></P><P>Rico, Tom S. </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures of 24h cultures for mutation rates
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>EGFR-C 1</TD><TD>378,3</TD></TR><TR><TD>EGFR-C 2</TD><TD>318,0</TD></TR><TR><TD>EGFR-C 3</TD><TD>385,6</TD></TR><TR><TD>EGFR-C 4</TD><TD>303,9</TD></TR><TR><TD>EGFR-C 5</TD><TD> 368,6 </TD></TR><TR><TD>EGFR-C-WT AID 1</TD><TD>274,9</TD></TR><TR><TD>EGFR-C-WT AID 3</TD><TD>393,8</TD></TR><TR><TD>EGFR-C-WT AID 4</TD><TD>313,5</TD></TR><TR><TD>EGFR-C-WT AID 5</TD><TD>299,6</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 1</TD><TD>232,7</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 2</TD><TD>376,5</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 3</TD><TD>399,2</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 4</TD><TD>394,2</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 5</TD><TD> 243,0 </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD>408,4</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 2</TD><TD>391,5</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 3</TD><TD>440,2</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD>364,5</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> 468,3 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>send to sequencing
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing for mutation rate (24h)</P><P><B>Investigators:</B></P><P>Tom S.</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>EGFR-C 1</TD><TD> II3641 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C 2 </TD><TD> II3642 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C 3</TD><TD> II3643 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C 4 </TD><TD> II3644 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C 5</TD><TD> II3645 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-WT AID 1 </TD><TD> II3646 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-WT AID 3</TD><TD> II3647 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-WT AID 4 </TD><TD> II3648 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-WT AID 5</TD><TD> II3649 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 1 </TD><TD> II3650 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 2</TD><TD> II3651 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 3 </TD><TD> II3652 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 4</TD><TD> II3653 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 5 </TD><TD> II3654 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD> II3655 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 2 </TD><TD> II3656 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 3 </TD><TD> II3657 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 4 </TD><TD> II3658 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> II3659 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP pool </TD><TD> II3660 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence pool</TD><TD> II3661 </TD><TD> pcDNA 3.1-FP </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Documentation of growing CHO cells under the microscope (72h incubation) </P><P><B>Investigators:</B> Rico, Tom S.</P><P><B>Time:</B> 2012-09-07</P><P><B>Materials:</B></P><UL><LI> six well plates with growing CHO cells (72h) which were transfected with EGFR-construct alone or in combination with WT AID or AID without NES, with NLS+Kozak sequence or AID without NES, with NLS+Kozak sequence+eGFP
</LI></UL><UL><LI> Fluorescence microscope
</LI></UL><P><B>Method:</B></P><P>take a photo of each well with a filter for GFP, YFP and brightfield
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Isolation of mutation rate samples (EGFR-construct <B>(EGFR-C)</B> alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wild type AID from transfected CHO cells (<B>72h incubation</B>))</P><P><B>Investigators:</B></P><P>Rico
</P><P><B>Aim:</B></P><UL><LI>Isolation of mutation rate samples (EGFR-construct alone, with AID without NES, with NLS+Kozak sequence, with AID without NES, with NLS+Kozak sequence+eGFP and with wild type AID from transfected CHO cells) to separate the plasmids for sequencing
</LI></UL><P><B>Method:</B></P><UL><LI>Miniprep of cells
</LI></UL><UL><LI>Elution with 50 ÂµL
</LI></UL><P><B>Results:</B></P><P>CHO-transfected with EGFR-C = 16,3 ng/ÂµL</P><P>CHO-transfected with EGFR-C and WT AID = 15,3 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence = 23,9 ng/ÂµL</P><P>CHO-transfected with EGFR-C and AID without NES, with NLS+Kozak sequence+eGFP = 21,5 ng/ÂµL</P><P><B>Further tasks:</B></P><P>transform plasmids
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of samples from cells which grew for 48h and 72h (Mutation rate) </P><P><B>Investigators:</B>Rico, Tom S.</P><P><B>Time:</B> 2012-09-07</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent E. coli cells (XL 1 Blue)
</LI></UL><UL><LI> samples of 48h and 72h incubation
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 1:5 dilutions of miniprep samples were used
</P><P><B>Results:</B></P><P>ready for growing mutants
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-09-08"><P style="background-color: rgb(240, 20, 70);">2012-09-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Inoculation of plasmid samples of the 48h retransformation plates</P><P><B>Investigators:</B> Basia
</P><P><B>Time:</B> 2012-09-08 6pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Amp 25 mg/ ml stock solution</LI></UL><UL><LI>plate with cultures: EGFR-C</LI></UL><P>EGFR-C-WT AID</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP</P><P>(all from 2012.09.07)</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>5 cultures per plate in 5 ml LB medium + 5ÂµL amp.
</P><P>(--&gt; 20 cultures)</P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Inoculation of cells ER2738 with AID in pBAD and scFV in pAK100</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-08 11am</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, chloramphenicol stock solution, arabinose 10% stock solution, plates with ER2738 cells with AID in pBAD and scFV in pAK100, </P><P><B>Method:</B></P><P>picking clones from a plate with ER2738 cells with AID in pBAD and scFV in pAK100 into 200ml of LB medium with antibiotics (ampicillin and chloramphenicol) and 0,1% arabinose or 0,01% arabinose (2 flasks, 200ml LB in each)</P><P><B>Results:</B></P><P>Slow growth of the cells
</P><P><B>Further tasks:</B></P><P>Inoculation of overnight culture without arabinose
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Inoculation of cells ER2738 with AID in pBAD and scFV in pAK100</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-08 5pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, chloramphenicol stock solution, plates with ER2738 cells with AID in pBAD and scFV in pAK100, </P><P><B>Method:</B></P><P>picking clones from a plate with ER2738 cells with AID in pBAD and scFV in pAK100 into 5ml of LB medium with antibiotics (ampicillin and chloramphenicol) </P><P><B>Results:</B></P><P> culture grew
</P><P><B>Further tasks:</B></P><P>preparation of phages
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">PCR of AID with Thiophosphate primers</P><P><B>Investigators:</B> Rico</P><P><B>Materials:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (BBa_K103001 AID in pSB1A2 10 ng/Âµl) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing</TD><TD> 66 </TD><TD> 20 </TD><TD> 17 </TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD>annealing+elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>PLICing variants
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Digestion of wt AID</P><P><B>Investigators:</B> Rico</P><P><B>Aim:</B> get the backbone for the Potsdam standard cloning vector
</P><P><B>Materials:</B></P><UL><LI>digestion with 1 ÂµL XbaI and 1 ÂµL PstI</LI></UL><P><B>Further Tasks:</B></P><UL><LI>electrophoretic separation
</LI></UL><UL><LI>ligation of the new standard cloning vector
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-09"><P style="background-color: rgb(240, 20, 70);">2012-09-09</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of the phages for phage display</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-09 10am-11pm</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, chloramphenicol stock solution, overnight culture of ER2738 cells, helper phage, Kanamycin, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>1. 2 Erlenmeyer flasks 100ml LB in each + 100Âµl of ampicillin stock solution + 100Âµl of chloramphenicol stock solution
</P><P>2. one with no arabinose, the other one with 0,01% arabinose (when OD600 0,3-0,5)
</P><P>3. addition of 30Âµl of helper phages (cleaned up on 6.9.2012) - when OD600 0,3-0,5
</P><P>4. further amplification and clean up of phage according to the manual</P><P><B>Further tasks:</B></P><P>continuation on the next day
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of 48h cultures for mutation rates</P><P><B>Investigators:</B></P><P>Basia </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures of 48h cultures for mutation rates
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>EGFR-C 1</TD><TD>201.6</TD></TR><TR><TD>EGFR-C 2</TD><TD>110.1</TD></TR><TR><TD>EGFR-C 3</TD><TD>106.1</TD></TR><TR><TD>EGFR-C 4</TD><TD>180.0</TD></TR><TR><TD>EGFR-C 5</TD><TD> 128.1 </TD></TR><TR><TD>EGFR-C-WT AID 1</TD><TD>121,1</TD></TR><TR><TD>EGFR-C-WT AID 2</TD><TD>118,4</TD></TR><TR><TD>EGFR-C-WT AID 3</TD><TD>136,1</TD></TR><TR><TD>EGFR-C-WT AID 4</TD><TD>110.8</TD></TR><TR><TD>EGFR-C-WT AID 5</TD><TD>122,1</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 1</TD><TD>157,1</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 2</TD><TD>125,3</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 3</TD><TD>133,9</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 4</TD><TD>176,5</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 5</TD><TD> 142,3 </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD>112,2</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 2</TD><TD>125,1</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 3</TD><TD>107,8</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD>123,6</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> 114,2 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>send to sequencing
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Inoculation of plasmid samples of the 72h retransformation plates</P><P><B>Investigators:</B> Basia
</P><P><B>Time:</B> 2012-09-09 6pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Amp 25 mg/ ml stock solution</LI></UL><UL><LI>plate with cultures: EGFR-C</LI></UL><P>EGFR-C-WT AID</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP</P><P>(all from 2012.09.07)</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>5 cultures per plate in 5 ml LB medium + 5ÂµL amp.
</P><P>(--&gt; 20 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Inoculation of cells ER2738 with AID in pBAD and cells ER2738 without AID in pBAD</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-08 8:30pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, tetracyclin stock solution, competent ER2738 cells with AID in pBAD and competent ER2738 cells without AID in pBAD </P><P><B>Method:</B></P><P>100Âµl of competent ER2738 cells with or without AID in pBAD and scFV in pAK100 into 5ml of LB medium with antibiotics (ampicillin or tetracyclin) </P><P><B>Results:</B></P><P> cultures grew
</P><P><B>Further tasks:</B></P><P>infection with phages and selection for mutated clones
</P><H3><SPAN class="mw-headline" id="2012-09-10"><P style="background-color: rgb(240, 20, 70);">2012-09-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of phage - continuation</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-10 </P><P><B>Materials:</B></P><P>LB medium, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>continuation of clean up of phages for phage display</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Infection with phages of the cells ER2738 with AID in pBAD and without AID in pBAD</P><P><B>Investigators:</B> Basia /Chris</P><P><B>Time:</B> 2012-09-10 2pm</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, chloramphenicol stock solution, ampicillin stock solution, overnight culture of ER2738 cells (once with once without AID in pBAD), phages cleaned up on 10.09.2012</P><P><B>Method:</B></P><P>1. 4 Erlenmeyer flasks 100ml LB in each + 100Âµl of ampicillin stock solution (into the flasks with AID in pBAD) - x2 or + 100Âµl of tetracycline stock solution (into flasks without AID) - x2 </P><P>2. two with no arabinose (into the flasks without AID in pBAD), the other two with 0,01% arabinose (when OD600 0,3-0,5, into flasks with AID)</P><P>3. addition of 280Âµl of phages (cleaned up on 10.9.2012) - when OD600 0,3-0,5</P><P>4. addition of chloramphenicol stock solution - 1h after infection with phages</P><P>5. further amplification of infected cells in 32Â°C</P><P><B>Further tasks:</B></P><P>Plating of the colonies onto the LB plates with appropriate antibiotics
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Plating of the colonies onto the LB plates with appropriate antibiotics</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-09-10 9pm</P><P><B>Materials:</B></P><P>Plates with LB medium with tetracycline and chloramphenicol and plates with LB medium with ampicillin and chloramphenicol, cultures infected with phages</P><P><B>Method:</B></P><P>850Âµl of the cultures infected with phages (see 10.09.2012 - phage infection) were centrifuged and the supernatant was discarded. 20Âµl of the resuspended pellet was used for plating</P><P>each culture was plated on the plates with appropriate antibiotics (E. coli without AID -infected with Phages 1&amp;2 on Tet and and Chloramphenicol, E. coli with AID -infected with Phages 1&amp;2 on Chloramphenicol and amp) </P><P><B>Further tasks:</B></P><P>preparation of mutated vectors for sequences
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Purification of transformed plasmids from the transfected CHO cells</P><P><B>Aim:</B> purification of the transformed single chain construct with YFP from the CHO cells 3 days after transfection
</P><P><B>Investigators:</B> Maria, Rico
</P><P><B>Method:</B> purification kit from Thermo Scientific
</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>EGFR-C 1</TD><TD>367.3</TD></TR><TR><TD>EGFR-C 2</TD><TD>342.6</TD></TR><TR><TD>EGFR-C 3</TD><TD>276.0</TD></TR><TR><TD>EGFR-C 4</TD><TD>285.1</TD></TR><TR><TD>EGFR-C 5</TD><TD> 263.3 </TD></TR><TR><TD>EGFR-C-WT AID 1</TD><TD>288.3</TD></TR><TR><TD>EGFR-C-WT AID 2</TD><TD>301.4</TD></TR><TR><TD>EGFR-C-WT AID 3</TD><TD>292.7</TD></TR><TR><TD>EGFR-C-WT AID 4</TD><TD>317.9</TD></TR><TR><TD>EGFR-C-WT AID 5</TD><TD>293.4</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 1</TD><TD>373.1</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 2</TD><TD>356.8</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 3</TD><TD>399.4</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 4</TD><TD>357.1</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 5</TD><TD> 325.4 </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD>315.4</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 2</TD><TD>305.6</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 3</TD><TD>359.5</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD>330.2</TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> 374.2 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-09-11"><P style="background-color: rgb(240, 20, 70);">2012-09-11</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">picking clones and overnight culture of the colonies grown on the plates from 10.9.2012 and XL1 BLue with CFP and YFP</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-03 7 pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, chloramphenicol stock solution, tetracycline stock solution, plates with colonies from 10.9.2012</P><P><B>Method:</B> inoculation in 5 ml LB medium + 5Âµl ampicillin chloramphenicol &amp; tetracycline:</P><P>the only colony of ER2738 with AID infected with Phage 1 inoculated</P><P>inoculation in 5 ml LB medium + 5Âµl chloramphenicol &amp; tetracycline:</P><P>ER2738 without AID infected with Phage 1 and 2-&gt; 5 single colonies, 1 mixed culture (500 Âµl LB were spread on the plate and 10 ÂµL used for inoculation)of each </P><P>inoculation in 50 ml LB medium + 50 Âµl ampicillin</P><P>XL1-Blue with CFP and Xl1-Blue with YFP
</P><P>shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>Miniprep, inoculation of overnight cultures (CFP, YFP, Mixed cultures - P1-AID, P1+AID, P2-AID)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transfection of CHO cells in 6 well plate and ibidi dishes</P><P><B>Investigator</B>: Rico
</P><P><B>Aim</B>: transfection of CHO cells with EGFR-C/modified AID, EGFR-C/ wt AID and EGFR-C alone in 6 well dishes and modified AID alone in ibidi dishes
</P><P><B>Further Tasks:</B> documentation of fluorescence with fluorescence microscope
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">PCR of RFP with restriction sites for <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A></P><P><B>Investigator</B>: Sascha, Rico
</P><P><B>Aim</B>: amplification of RFP to add the restriction sites for <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A></P><P><B>Materials:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (BBa_K103001 AID in pSB1A2 10 ng/Âµl) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing</TD><TD> 70 </TD><TD> 20 </TD><TD> 17 </TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD>annealing+elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Furhter Tasks:</B> ligation of RFP and digested pSB1C3
</P><H3><SPAN class="mw-headline" id="2012-09-12"><P style="background-color: rgb(240, 20, 70);">2012-09-12</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures</P><P><B>Investigators:</B></P><P>Chris/Basia </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>CFP (AmpR)</TD><TD>184</TD></TR><TR><TD>Phage 1 C1</TD><TD>740</TD></TR><TR><TD>Phage 1 C2</TD><TD>843</TD></TR><TR><TD>Phage 1 C3</TD><TD>740</TD></TR><TR><TD>Phage 1 C4</TD><TD>620</TD></TR><TR><TD>Phage 1 C5</TD><TD>685</TD></TR><TR><TD>Phage 2 C1</TD><TD>842</TD></TR><TR><TD>Phage 2 C2</TD><TD>689</TD></TR><TR><TD>Phage 2 C3</TD><TD>622</TD></TR><TR><TD>Phage 2 C4</TD><TD>683</TD></TR><TR><TD>Phage 2 C5</TD><TD>777.6</TD></TR></TBODY></TABLE><P>Phage 1 was produced under expression of AID (mutated, but there was no AID while reinfection of E.coli)</P><P>Phage 2 is the control(Phage production and Infection without AID)</P><P>C1-C5 where picked colonies
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> send DNA from colonies after first round of phage display to sequencing</P><P><B>Investigators:</B></P><P>Basia/Chris</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>Phage1(mut) C1</TD><TD> II3662 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage1(mut) C2</TD><TD> II3663 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage1(mut) C3</TD><TD> II3664 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage1(mut) C4</TD><TD> II3665 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage1(mut) C5</TD><TD> II3666 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage2(not mut) C1</TD><TD> II3667 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage2(not mut) C2</TD><TD> II3668 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage2(not mut) C3</TD><TD> II3669 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage2(not mut) C4</TD><TD> II3670 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>Phage2(not mut) C5</TD><TD> II3671 </TD><TD>GATC_Std_RPC </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Inoculation of main cultures: CFP (ampR), mixed colonies of mutated and non mutated infected colonies (in E.coli without AID -&gt;Tet, CM-resistance)</P><P><B>Investigators:</B>Basia/Chris</P><P><B>Aim:</B></P><P>Purfication of CFP protein, determination of mutation rates via phage display
</P><P><B>Materials:</B> LB medium, tetracycline stock solution, chloramphenicol stock solution, ampicillin stock solution, overnight culture of CFP producing cells, overnight culture of <I>E. coli</I> infected with phages
</P><P><B>Method:</B></P><P>Preparation of CFP main culture:
</P><P>2x2ml of overnight culture was added to 2x 500ml LB medium with 500Âµl of ampicillin each
</P><P>Preparation of main cultures for phage display:
</P><P>1x 1ml of overnight culture of non-mutated cells were added to 100ml of LB medium with 100Âµl of tetracycline
</P><P>1x 1ml of overnight culture of 1 x mutated cells were added to 100ml of LB medium with 100Âµl of tetracycline and 100Âµl of ampicillin
</P><P><B>Further tasks:</B></P><P>Induction of CFP; new cycle of phage production
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Induction of CFP</P><P><B>Investigators:</B>Basia/Chris</P><P><B>Aim:</B></P><P>Purfication of CFP protein
</P><P><B>Materials:</B> Main culture XL1 Blue with CFP, IPTG stock solution
</P><P><B>Method:</B></P><P>250Âµl of 1M IPTG stock solution was added to each of 500ml of main cultures when OD600 reached 0,3-0,5
</P><P><B>Further tasks:</B></P><P>protein purification
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of the phages for phage display</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-12 </P><P><B>Materials:</B></P><P>main culture of ER2738 with pBAD, pAK100 , helper phage, Kanamycin, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>1. addition of arabinose to the culture with pBAD with AID: 0,01% arabinose (when OD600 0,3-0,5)
</P><P>3. addition of 30Âµl of helper phages (cleaned up on 6.9.2012) - when OD600 0,3-0,5
</P><P>4. further amplification and clean up of phage according to the manual</P><P><B>Further tasks:</B></P><P>continuation on the next day
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Inoculation of cells ER2738 with AID in pBAD and cells ER2738 without AID in pBAD</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-12 8:30pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, tetracyclin stock solution, competent ER2738 cells with AID in pBAD and competent ER2738 cells without AID in pBAD </P><P><B>Method:</B></P><P>100Âµl of competent ER2738 cells with or without AID in pBAD into 5ml of LB medium with antibiotics (ampicillin or tetracyclin) </P><P><B>Results:</B></P><P> cultures grew
</P><P><B>Further tasks:</B></P><P>infection with phages and selection for mutated clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Tom S.</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>mod. AID+eGFP 4</TD><TD> II3676 </TD><TD> pSB1C3 Forward </TD></TR><TR><TD> mod. AID+eGFP 4 </TD><TD> II3677 </TD><TD> pSB1C3 Reverse </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of RFP in pSB1C3 and transformation</P><P><B>Investigators:</B> Rico</P><P><B>Aim:</B> Ligation of RFP in pSB1C3
</P><P><B>Method:</B> 1 ÂµL RFP, 1 ÂµL digested pSB1C3, 1 ÂµL T4-Ligase, 1 ÂµL T4-Ligation buffer, 6 ÂµL water, transformation in Xl 1 E.coli (E.coli were spread without IPTG)
</P><P><B>Further Tasks:</B></P><P>O/N culture of clones
</P><H3><SPAN class="mw-headline" id="2012-09-13"><P style="background-color: rgb(240, 20, 70);">2012-09-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of phage - continuation</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-13 </P><P><B>Materials:</B></P><P>LB medium, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>continuation of clean up phages for phage display according to the manual</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Isolation of CFP from XL1-Blue</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Materials:</B></P><P><B>Method:</B></P><P>-resuspend pellet (in 20 mL 10mM Hepes 500mM NaCl Buffer pH 7.4) </P><P>-lysis by sonication: 3x2min, 6mm tip, 50% amplitude</P><P>-centrifugationÂ´for 30 min, 19000 rpm 4Â°C</P><P>-sonication of supernatant for 1 min</P><P>-filtering with 0.45 mm filter</P><P>-loading extract on IMAC-column(Ni-NTA) (equilibrated with 30 column volumes of 10 mM HEPES buffer-pH 7.4; 500mM NaCL)</P><P>-loading protein extract with flow rate of 0.5 mL/min</P><P>-wash column with 30mM imidazole</P><P>-elution of protein with 50 mM imidazole in 10 mM HEPES bufer pH 7.4 (this was a mistake-200mM imidazole is the correct elution buffer, but it worked anyways)</P><P>-storage of eluted protein on ice in the fridge</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">checking the SDS-Gel &amp; spectrum of CFP</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Results:</B></P><P>-spectra look like CFP spectra (max. absorbtion at 436 nm) max emission 474 &amp; nm
</P><P>-in the strong diluted elution fraction the visible lane is as large as CFP (~28 kDA)
</P><P>-this lane is also strong in the washed fraction
</P><H3><SPAN class="mw-headline" id="2012-09-14"><P style="background-color: rgb(240, 20, 70);">2012-09-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing for mutation rate (48h)</P><P><B>Investigators:</B></P><P>Tom S.</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>EGFR-C 1</TD><TD> AKM001W075 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C 2 </TD><TD> AKM001W076 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C 3</TD><TD> AKM001W077 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C 4 </TD><TD> AKM001W078 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C 5</TD><TD> AKM001W079 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-WT AID 1 </TD><TD> AKM001W080 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-WT AID 2</TD><TD> AKM001W081 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-WT AID 3</TD><TD> AKM001W082 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-WT AID 4 </TD><TD> AKM001W083 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-WT AID 5</TD><TD> AKM001W084 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 1 </TD><TD> AKM001W085 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 2</TD><TD> AKM001W086 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 3 </TD><TD> AKM001W087 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence 4</TD><TD> AKM001W088 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence 5 </TD><TD> AKM001W089 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD> AKM001W090 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 2 </TD><TD> AKM001W091 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 3 </TD><TD> AKM001W092 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD> EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 4 </TD><TD> AKM001W093 </TD><TD> pcDNA 3.1-FP </TD></TR><TR><TD>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> AKM001W094 </TD><TD> pcDNA 3.1-FP </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">checking sequences after first round (1x mut) of phage display</P><P><B>Investigator:</B> Chris</P><P><B>results:</B> not a single mutation in one of the 5 &quot;mutated sequences&quot; or one of the 5 controls (without AID)</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">labeling GFP with Biotin (as a matter of fact GFP is fused with a helix-&gt; CREBzipGFP)</P><P><B>Investigator:</B> Basia/ Chris</P><P><B>Material:</B> EZ-LinkÂ® Sulfo-NHS-LC-Biotin </P><P><B>Method:</B></P><P>-add 10 fold molar exess of biotin reagent to CREBzipGFP </P><P>-CREBzipGFP was thought to have an concentration of 1.3 mg/mL (MW=36.172 kDa), therefore 36 ÂµL of 10 mM EZ-LinkÂ® Sulfo-NHS-LC-Biotin where added </P><P>-after biotinylation the concetration of CREBzipGFP-Biotin was shown to be 4.075 mg/mL -maybe to less Biotin reagent was used
</P><P>-the concentration of remaining CREBzipGFP without Biotin was 0.3 mg/mL
</P><P><B>Further tasks:</B> testing the success of biotinylation with &quot;ELISA&quot;
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">inoculation of mixed plates ER2738-pAK100(not mutated) &amp; ER2738-pAK100 with AID(2x mutated)</P><P><B>Investigator:</B> Basia/ Chris</P><P>-addition of 1 mL LB medium on plates, mixing colonies with Drigalski spatulas</P><P>-addition of 20 ÂµL of mixed cultures to 5 mL LB with Tet,CM (for ER2738+pAK100-not mutated) and to 5 mL LB with Tet,CM, Amp for ER2738+pAK100 with AID(2x mutated)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">O/N cultures of RFP in pSB1C3 clones</P><P><B>Investigator:</B> Rico
</P><P><B>Aim:</B> O/N culture of RFP in pSB1C3 clones, clones have to grow approximately 24 h at 37 Â°C and incubate O/N at 6-7 Â°C (after that you can see the red fluorescence under LED light)
</P><P><B>Method:</B> picking clones, 1:1000 dilution of IPTG stock solution (1 M)
</P><P><B>Further Tasks:</B> purification of plasmids
</P><H3><SPAN class="mw-headline" id="2012-09-15"><P style="background-color: rgb(240, 20, 70);">2012-09-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of the phages for phage display</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-09-15 </P><P><B>Materials:</B></P><P>over night cultures of ER2738 with pBAD and pAK(double mut) &amp; ER2738 with pAK(not mut) , helper phage, Kanamycin, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>1. starting main culture of ER2738 with pBAD and pAK(double mut) &amp; ER2738 with pAK(not mut)-addition of 2 mL over night culture to 100 mL LB(Tet,Cm for ER2738 with pAK /Tet,Cm, Amp for ER2738 with pBAD and pAK)</P><P>2. addition of arabinose to the culture with pBAD with AID: 0,02% arabinose (when OD600 0,3-0,5)</P><P>3. addition of 30Âµl of helper phages (cleaned up on 6.9.2012) - when OD600 0,3-0,5</P><P>4. further amplification and clean up of phage according to the manual</P><P><B>Further tasks:</B></P><P>continuation on the next day
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">overnight cultures of ER2738 and ER2738 with AID for phage display and AID-expression test </P><P><B>Investigator:</B>Chris</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, tetracyclin stock solution, competent ER2738 cells with AID in pBAD and competent ER2738 cells without AID in pBAD </P><P><B>Method:</B></P><P>100Âµl of competent ER2738 cells with or without AID in pBAD into 5ml of LB medium with antibiotics (ampicillin &amp; tetracyclin/ tetracyclin) </P><P><B>Results:</B></P><P> cultures grew
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Elisa of biotinylated eGFP</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><P>CREBzipGFP-biotin (c=4.075 mg/mL), CREBzipGFP for control (c=0.3 mg/mL; É280=22,015 M-1 cm-1; MW= 36,172 Da), block buffer (1% BSA in PBS), SAV-HPR-solution(SAV-HPR 1:5000 &amp; 1&amp;% BSA in PBS), substrate solution (20:78:1:1 5x --substrate Buffer: H20: Substrate-&gt;OPD: 1% H2O2), stop solution (1 M H2SO4,50mM Na2SO3 in water), platereader
</P><P><B>Method:</B></P><P>CREBzipGFP-biotin (20 Âµg/mL,10 Âµg/mL,5 Âµg/mL, 1 Âµg/mL) CREBzipGFP-control (20 Âµg/mL,10 Âµg/mL,5 Âµg/mL, 1 Âµg/mL), PBS as blank: 50 ÂµL/well</P><P>wait 2:30 h </P><P>washed 2 times w/tap water </P><P>block: 100 ÂµL/well 1% BSA/PBS </P><P>washed 5 times w/tap water </P><P>50 ÂµL/well SAV-HRP-solution, 1h </P><P>washed 10 times w/tap water </P><P>50 ÂµL/well substrate solution - 5,5 min </P><P>50 ÂµL/well stop solution,
</P><P>measure OD 492 </P><P><B>Results:</B></P><P><B>Further tasks:</B> couple CREBzipGFP-biotin with magnetic beads
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Purification of RFP in pSB1C3 from O/N cultures</P><P><B>Investigator:</B>Rico</P><P><B>Materials:</B> Purification kit from Thermo Scientific
</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>concentration in ng/ÂµL</B></TD></TR><TR><TD>RFP 1</TD><TD> 136.9 </TD></TR><TR><TD> RFP 2 </TD><TD> 113.8 </TD></TR><TR><TD>RFP 3</TD><TD> 106.2 </TD></TR><TR><TD> RFP 4 </TD><TD> 99.8 </TD></TR><TR><TD>RFP 5</TD><TD> 108.8 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Digestion of purified new standard cloning vector with Apa I and Sph I</P><P><B>Investigator:</B>Rico</P><P><B>Method:</B></P><P>7 ÂµL RFP 1, 1 ÂµL Apa I, 1 ÂµL Sph I, 3 ÂµL buffer, 18 ÂµL water, incubation @ 37 Â°C for 2 h
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Transformation of purified AID plasmids in E.coli for determining the mutation rate</P><P><B>Investigator:</B>Rico</P><P><B>Materials:</B> Transformation protocol for E.coli
</P><H3><SPAN class="mw-headline" id="2012-09-16"><P style="background-color: rgb(240, 20, 70);">2012-09-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Induction calibration of AID in pBAD vector</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-16 </P><P><B>Materials:</B></P><P>LB medium, protein gel equipment, ampicillin stock solution, tetracycline stock solution, arabinose stock solution</P><P><B>Method:</B></P><P>1. Inoculation of 200Âµl of overnight culture (ER2738+AID in pBAD) into 20ml of LB medium with tet and amp.</P><P>2. addition of arabinose 0,01%, when OD600 0,3-0,5 </P><P>3. Samples taken at T0, T3 and T5 (at time zero, after 3h and after 5h)</P><P>4. The OD600 was measured before taking the samples in order to adjust the protein amount</P><P>5. Cell suspension was centrifuged for 3min 16100 xg and pellet was resuspended in 100Âµl of lyase sample buffer 4x.</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparation of phage - continuation</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-16 </P><P><B>Materials:</B></P><P>LB medium, PEG-NaCl solution, TBS buffer</P><P><B>Method:</B></P><P>continuation of clean up phages for phage display</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Infection with phages of the cells ER2738 with AID in pBAD and without AID in pBAD</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-16 2pm</P><P><B>Materials:</B></P><P>LB medium, tetracycline stock solution, chloramphenicol stock solution, ampicillin stock solution, overnight culture of ER2738 cells (once with once without AID in pBAD), phages cleaned up on 16.09.2012, arabinose stock solution</P><P><B>Method:</B></P><P>1. 2 Erlenmeyer flasks 100ml LB in each + 100Âµl of ampicillin + 100Âµl tetracycline stock solution (into the flask with AID in pBAD) or + 100Âµl of tetracycline stock solution (into flask without AID) - x2 </P><P>2. 1 with no arabinose (into the flask without AID in pBAD), the other one with 0,02% arabinose (when OD600 0,3-0,5, into flask with AID)</P><P>3. addition of 280Âµl of phages (cleaned up on 10.9.2012) - when OD600 0,3-0,5</P><P>4. addition of chloramphenicol stock solution - 1h after infection with phages</P><P>5. further amplification of infected cells in 32Â°C</P><P><B>Further tasks:</B></P><P>Plating of the colonies onto the LB plates with appropriate antibiotics
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Plating of the colonies onto the LB plates with appropriate antibiotics</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-10 7pm</P><P><B>Materials:</B></P><P>Plate with LB medium with tetracycline and chloramphenicol and plates with LB medium with ampicillin, tetracycline, 0,01% arabinose and chloramphenicol, cultures infected with phages</P><P><B>Method:</B></P><P>850Âµl of the cultures infected with phages (see 16.09.2012 - phage infection) were centrifuged and the supernatant was discarded. 20Âµl of the resuspended pellet were used for plating</P><P>each culture was plated on the plates with appropriate antibiotics (E. coli without AID -infected with non-mutated phages on Tet and Cm, E. coli with AID -infected with mutated phages on Cm, tet, ara and amp) </P><P><B>Further tasks:</B></P><P>preparation of mutated vectors for sequences
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Inoculation of plasmid samples of the 48h retransformation plates</P><P><B>Investigators:</B> Basia
</P><P><B>Time:</B> 2012-09-08 6pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Cm 35 mg/ ml stock solution</LI></UL><UL><LI>plate with cultures: </LI></UL><P>EGFR-C-WT AID</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence</P><P>EGFR-C-AID without NES, with NLS+Kozak sequence+eGFP</P><P>(all from 2012.09.15)</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>5 cultures per plate in 5 ml LB medium + 5ÂµL Cm.
</P><P>(--&gt; 20 cultures)</P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><H3><SPAN class="mw-headline" id="2012-09-17"><P style="background-color: rgb(240, 20, 70);">2012-09-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of 48h cultures for (AID) mutation rates</P><P><B>Investigators:</B></P><P>Tom, Rico </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures of 48h cultures for (AID) mutation rates
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 1</TD><TD>371.2</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 2</TD><TD>368.4</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 3</TD><TD>340.8</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 4</TD><TD>391.3</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 5</TD><TD> 373.9 </TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD>393.0</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 2</TD><TD>413.1</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 3</TD><TD>379.7</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD>376.5</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD>389.3</TD></TR><TR><TD>WT (AID) 1</TD><TD>366.4</TD></TR><TR><TD>WT (AID) 2</TD><TD>389.9</TD></TR><TR><TD>WT (AID) 3</TD><TD>359.5</TD></TR><TR><TD>WT (AID) 4</TD><TD>367.3</TD></TR><TR><TD>WT (AID) 5</TD><TD> 365.2 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>send to sequencing
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Tom S., Rico</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence 1</TD><TD> II3682 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence 2</TD><TD> II3683 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence 3</TD><TD> AI2677 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence 4</TD><TD> AI2678 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence 5</TD><TD> AI2679 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+eGFP 1</TD><TD> AI2680 </TD><TD> CMV Forward </TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD> AI2681 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+eGFP 3</TD><TD> AI2682 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+eGFP 4</TD><TD> AI2683 </TD><TD> CMV Forward </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+eGFP 5</TD><TD> AI2684 </TD><TD> CMV Forward </TD></TR><TR><TD> WT (AID) 1</TD><TD> AI2685 </TD><TD> CMV Forward </TD></TR><TR><TD> WT (AID) 2</TD><TD> AI2686 </TD><TD> CMV Forward </TD></TR><TR><TD> WT (AID) 3</TD><TD> AI2687 </TD><TD> CMV Forward </TD></TR><TR><TD> WT (AID) 4</TD><TD> AI2688 </TD><TD> CMV Forward </TD></TR><TR><TD> WT (AID) 5</TD><TD> AI2689 </TD><TD> CMV Forward </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Checking AID induction test with SDS-PAGE gel</P><P><B>Investigators:</B> Basia, Chris </P><P><B>Results:</B></P><P>There is no overexpressed lane that differs in the induced samples. No AID expression or too less arabinose???
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">picking clones and overnight culture of the colonies grown on the plates from 16.9.2012 </P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-09-17 7 pm</P><P><B>Materials:</B></P><P>LB medium, ampicillin stock solution, chloramphenicol stock solution, tetracycline stock solution, plates with colonies from 16.9.2012</P><P><B>Method:</B> inoculation of 5 clones from mutated clones in 5 ml LB medium + 5Âµl ampicillin, chloramphenicol &amp; tetracycline:</P><P>inoculation of 5 clones from the non mutated plate in 5 ml LB medium + 5Âµl chloramphenicol &amp; tetracycline:</P><P>inoculation of XL1-Blue with AID in pBAD in 5 ml LB medium + 5 Âµl ampicillin + 5Âµl tetracycline - for a new induction test</P><P>shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>Miniprep, induction calibration</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> PLIcing of Thio-AID, Ligation with digested new RFC standard cloning vector 1</P><P><B>Investigators:</B></P><P>Rico </P><P><B>Aim:</B></P><P>Ligation of AID in new cloning vector 1</P><P><B>Method:</B></P><UL><LI> 8 ÂµL Thio-AID was treated with 1 ÂµL Cleavage Buffer (0.5 Tris-HCl, pH 9.0), 0.4 ÂµL Milli-Q water and 0.6 ÂµL iodine stock solution (100 mM)
</LI></UL><UL><LI> incubate @ 70 Â°C for 5 min
</LI></UL><UL><LI> mixed with digested RFP standard cloning vector (purified, non purfied, with and without ligase) @ room temperature for 1 h
</LI></UL><UL><LI> transform into E.coli
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-18"><P style="background-color: rgb(240, 20, 70);">2012-09-18</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Induction calibration of AID in pBAD vector</P><P><B>Investigators:</B> Basia/Chris</P><P><B>Time:</B> 2012-09-18 </P><P><B>Materials:</B></P><P>LB medium, protein gel equipment, ampicillin stock solution, tetracycline stock solution, arabinose stock solution</P><P><B>Method:</B></P><P>1. Inoculation of 200Âµl of overnight culture (ER2738+AID in pBAD) into 20ml of LB medium with tet and amp.</P><P>2. addition of arabinose 0,01%, when OD600 0,3-0,5 </P><P>3. Samples taken at T0, T3 and T5 (at time zero, after 3h and after 5h)</P><P>4. The OD600 was measured before taking the samples in order to adjust the protein amount</P><P>5. Cell suspension was centrifuged for 3min 16100 xg and pellet was resuspended in 100Âµl of lyase sample buffer 4x.</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures</P><P><B>Investigators:</B></P><P>Chris/Basia </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>mut 1</TD><TD>530</TD></TR><TR><TD>mut 2</TD><TD> 595</TD></TR><TR><TD>mut 3</TD><TD> 221,7</TD></TR><TR><TD>mut 4!</TD><TD> 63.5</TD></TR><TR><TD>mut 5</TD><TD> 360</TD></TR><TR><TD>mut 6</TD><TD> 350</TD></TR><TR><TD>non mut 1</TD><TD>782,2</TD></TR><TR><TD>non mut 2</TD><TD>523,3</TD></TR><TR><TD>non mut 3</TD><TD>780</TD></TR><TR><TD>non mut 4</TD><TD>651</TD></TR><TR><TD>non mut 5</TD><TD>688</TD></TR></TBODY></TABLE><P>mut: cells that were always under arabinose and AID conditions</P><P>non mut: cells that are used as controls - never used arabinose and AID during cell culturing</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> send DNA from colonies after third round of phage display to sequencing</P><P><B>Investigators:</B></P><P>Basia/Chris</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>mut 1</TD><TD> AI2691</TD><TD>GATC_Std_RPC </TD></TR><TR><TD>mut 2</TD><TD> AI2692 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>mut 3</TD><TD> AI2693 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>mut 4!</TD><TD> AI2699 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>mut 5</TD><TD> AI2700 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>mut 6</TD><TD> AI2717 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>non mut 1</TD><TD> AI2694 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>non mut 2</TD><TD> AI2695 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>non mut 3</TD><TD> AI2696 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>non mut 4</TD><TD> AI2697 </TD><TD>GATC_Std_RPC </TD></TR><TR><TD>non mut 5</TD><TD> AI2698 </TD><TD>GATC_Std_RPC </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">O/N culture of transformed E.coli with AID in RFP new standard cloning vector</P><P><B>Investigators:</B> Rico, Sascha</P><P><B>Time:</B> 2012-09-18 </P><P><B>Method:</B></P><P>picking of clones, which show no red fluorescence indicating that the ligation was successful
</P><H3><SPAN class="mw-headline" id="2012-09-19"><P style="background-color: rgb(240, 20, 70);">2012-09-19</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Checking AID induction test with SDS-PAGE gel</P><P><B>Investigators:</B> Basia, Chris </P><P><B>Results:</B></P><P>There is no overexpressed lane that differs in the induced samples. Maybe the AID is low expressed because of the way the construct is built - 11 bp between the start codon and RBS instead of 7? The arrow shows where more less should be a band of AID.
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of O/N cultures of RFP new standard cloning vector</P><P><B>Investigators:</B> Rico, Sascha </P><P><B>Method:</B> Miniprep kit from Thermo scientific
</P><P><B>Results:</B></P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>concentration in ng/ÂµL</B></TD></TR><TR><TD>unpurified, + Ligase</TD><TD> 170.5 </TD></TR><TR><TD> unpurified, - Ligase </TD><TD> 174.8 </TD></TR><TR><TD>purified, + Ligase</TD><TD> 189.9 </TD></TR><TR><TD> purified, - Ligase </TD><TD> 179.1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Test digestion of purified AID in new standard cloning vector</P><P><B>Investigators:</B> Rico, Sascha </P><P><B>Results:</B></P><UL><LI> all variants show the expected sizes of the digested fragments</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Sending AID variants in new cloning vector and RFP cloning vector for sequencing</P><P><B>Investigators:</B> Rico </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Transfection of CHO cells with scFv-YFP, Nanobody-mCHerry and modified AID-eGFP construct for FACS</P><P><B>Investigators:</B> Rico</P><P><B>Method:</B> 2 Âµg DNA was used for transfection in 6 well plates</P><H3><SPAN class="mw-headline" id="2012-09-20"><P style="background-color: rgb(240, 20, 70);">2012-09-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Fluorescence microscopy of transfected CHO cells for FACS</P><P><B>Investigators:</B> Rico </P><P><B>Results:</B></P><P>Cells shows a much higher red fluorescence (transfected with Nanobody-mCherry construct) than exprected
</P><H3><SPAN class="mw-headline" id="2012-09-21"><P style="background-color: rgb(240, 20, 70);">2012-09-21</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Control of the sequencing results after phage display</P><P><B>Investigators:</B> Basia</P><P><B>Results:</B></P><P>not a single mutation was introduced into the sequence. We think it is due to lack of AID expression in the pBAD vector.</P><H3><SPAN class="mw-headline" id="2012-09-22"><P style="background-color: rgb(240, 20, 70);">2012-09-22</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Selection of the antibodies from the supernatant after the Cre recombinase treatment via magnetic beads binding</P><P><B>Investigators:</B> Basia</P><P><B>Materials:</B></P><P>Dyna Magnetic Beads bound with Streptavidin, biotinylated eGFP, supernatant after the CHO cell culturing, PBS buffer
</P><P><B>Method:</B> According to the Dyna beads Trial Kit Manual: 800pmol of biotinylated eGFP was used, 1ml of supernatant was added to the eGFP coupled with magnetic beads through streptavidin and biotin and incubated for 1hour with delicate mixing at room temperature. The samples were then resuspended in 100Âµl of loading buffer and loaded on the SDS protein gel.</P><P><B>Results:</B></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Checking sequencing results for new standard</P><P><B>Investigators:</B> Rico</P><P><B>Results:</B></P><P>all sequences show no mutation in the coding sequence, suffix and prefix
</P><H2><SPAN class="mw-headline" id="Antibody">Antibody</SPAN></H2><H3><SPAN class="mw-headline" id="2012-09-01"><P style="background-color: rgb(240, 20, 70);">2012-09-01</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of Geneart construct, Venus, pAK100+scFv425 R H1, R H2, R L1, R L2, L1, L2 </P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-09-01 9:30 am</P><P><B>Materials: </B>overnight cultures Geneart construct (4x), Venus (Tom), pAK100+scFv425 R H1, R H2, R L1, R L2, L1, L2 (Chris), Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><P>Venus: â¦. ng/Âµl</P><P>pAK100+scFv425 R H1: --- ng/Âµl</P><P>pAK100+scFv425 R H2: --- ng/Âµl&lt;br&lt;</P><P>pAK100+scFv425 R L1: --- ng/Âµl</P><P>pAK100+scFv425 R L2: --- ng/Âµl</P><P>pAK100+scFv425 L1: --- ng/Âµl</P><P>pAK100+scFv425 L2: --- ng/Âµl</P><P>Geneart construct 1: --- ng/Âµl</P><P>Geneart construct 2: --- ng/Âµl</P><P>Geneart construct 3: --- ng/Âµl</P><P>Geneart construct 4: --- ng/Âµl</P><P>+ </P><P>Glycerole stocks of Geneart construct and pAK100+scFv425 R H1, R H2, R L1, R L2, L1, L2
</P><P><B>Further tasks: </B></P><H3><SPAN class="mw-headline" id="2012-09-03_2"><P style="background-color: rgb(240, 20, 70);">2012-09-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-09-03 09:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> seeding of zeocin selected CHO cells in 6-well + ibidi-dish for stable transfection and modified AID-GFP
</LI></UL><UL><LI> investigation of ordering phenolredfree medium
</LI></UL><UL><LI> searching fort he right version of imageJ
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> mutagenesis-PCR to remove PstI-sites off scFv-Tmd-EYFP-construct </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Template: scFv-TEV-TMD-EYFP
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.RCF25 and 2.mut: RCF25 prefix and 1st mutation PstI in scFV (Tm=72Â°C)
</LI><LI> Primer 3.mut and 4.mut: 1st and 2nd mutation PstI in scFV (Tm= 72Â°C)
</LI><LI> Primer 5.mut and &amp;.RCF25: 2nd mutation and RCF25 suffix (Tm= 72Â°C)
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for primer 1 and 2</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HF buffer</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer 1.RCF25 td&gt;</TD><TD>2,5</TD></TR><TR><TD>Primer 2.mut: RCF25 prefix and 1st mutation PstI in scFV  </TD><TD>2,5</TD></TR><TR><TD> DNA (scFv-TEV-TMD-EYFP</TD><TD> 1</TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for Primer 1+2</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 30</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 2 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>50Âµl mix for primer 3 and 4</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HF buffer</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer 3.mut: 1st mutation PstI in scFV td&gt;</TD><TD>2,5</TD></TR><TR><TD> Primer 4.mut: 1st mutation PstI in scFV </TD><TD>2,5</TD></TR><TR><TD> DNA (scFv-TEV-TMD-EYFP)</TD><TD> 1</TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for Primer 3+4</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 30</TD></TR><TR><TD> annealing/elongation </TD><TD> 72 </TD><TD> 15 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-09-04_2"><P style="background-color: rgb(240, 20, 70);">2012-09-04</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-09-04 09:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> stable transfection of clon 4, according to stable transfection protocoll
</LI></UL><UL><LI> transfection of AID-GFP in 6-well plate
</LI></UL><UL><LI> virus infection of CHO cells in 6-well plates and ibidi dishes
</LI></UL><UL><LI> evaluation of cell culture images
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of pcdna5frt with NheI and ApaI </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest NheI
</LI><LI> Fast Digest ApaI
</LI><LI> 10x FD Green Buffer
</LI><LI> pcdna5frt (705,4ng/Âµl)
</LI><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 5Âµl pcdna5frt (705,4ng/Âµl)
</LI><LI> 2Âµl NheI
</LI><LI> 2Âµl ApaI
</LI><LI> 3Âµl 10x FD Green Buffer
</LI><LI> 18Âµl sterile water
</LI><LI> digestion for 2,h at 37Â°C</LI></UL><P><B>&gt;Results:</B><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> dephosphorylating the vector pcDNA5FRT digested with NheI and ApaI</P><P><B>Investigator:</B>Maria</P><P><B>Aim:</B> dephosphorylating the digested pcDNA5-FRT to prevent re-ligation </P><P><B>Materials:</B></P><UL><LI> Antarctic Phosphatase
</LI></UL><UL><LI> 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> digested pcDNA5-FRT
</LI></UL><P><B>Method:</B></P><UL><LI> 3,3Âµl 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> 1Âµl Antarctic Phosphatase
</LI></UL><UL><LI> incubation for 30min at 37Â°C to dephosphorylate 5Â´-ends of pcDNA5-FRT
</LI></UL><UL><LI> heat inactivation of Phosphatase for 5min at 65Â°C
</LI></UL><UL><LI> store at -20Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> ligation of pcDNA5-FRT dephosphorylated vector with geneart construct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean-Up of dephosphorylated vector </P><P><B>Investigator:</B>Maria</P><P><B>Aim:</B> cleaning of dephosphorylated vector</P><P><B>Materials:</B></P><UL><LI> PCR-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> concentration of cleaned dephosphorylated pcDNA5FRT = 60,4 ng/Âµl
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> colony-PCR of ligated pcDNA5-FRT_geneart-construct clones </P><P><B>Investigator:</B> Maria</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_geneart-constructs clones </P><P><B>Materials:</B></P><UL><LI> Taq DNA-polymerase 1kb/1min
</LI></UL><UL><LI> 10x Taq DNA-polymerase buffer</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer in C-terminus of CMV of pcDNA5-FRT-backbone: fp_qRT-CMV_rev
</LI></UL><UL><LI> reverse primer in C-terminus in mcherry: ps_mcherry_rev_BamHI * Thermocycler
</LI></UL><P><B>Methods:</B></P><UL><LI> 20Âµl colony-PCR mix for 20 clones:
</LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Standard Taq-DNA-Polymerase Buffer </TD><TD>40</TD></TR><TR><TD>dNTPs</TD><TD>8</TD></TR><TR><TD>Primer (Forward)</TD><TD>8</TD></TR><TR><TD>Primer (Reverse)</TD><TD>8</TD></TR><TR><TD> DNA </TD><TD> clone from plate </TD></TR><TR><TD> Taq Polymerase </TD><TD> 2 </TD></TR><TR><TD> water </TD><TD> 334 </TD></TR></TBODY></TABLE><P>Primer forward: fp_qRT-CMV_rev(10ÂµM)</P><P>Primer reverse: ps_mcherry_rev_BamHI (10ÂµM)</P><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 95 </TD><TD>300 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 25 </TD><TD> 30</TD></TR><TR><TD> annealing </TD><TD> 51 </TD><TD> 25 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 68 </TD><TD> 130 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 86 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><UL><LI> clone-colonies were transferred to 20 Âµl colony-PCR mix, and onto LB-Amp plate, respectively </LI></UL><UL><LI> LB-AMP-plate were incubated o.n. at 37Â°C</LI></UL><UL><LI> negative control: clon #20 from religation-control (ligation-ratio 1:0) from 2012-09-03
</LI></UL><P><B>further Taks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> picking positive clones for o.n.-culture
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of digested scFv-TEV-TMD-EYFP </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 30Âµl digested clon4 geneart-nanobody-construct (451ng/Âµl)
</LI><LI> 1% agarose gel, 100ml
</LI><LI> 105V, 75min
</LI></UL><P><B>Results:</B></P><UL><LI> digested clon4 geneart-nanobody-construct (451ng/Âµl) with DNA @ 2029bp
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel extraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of three genes generated by mutagenesis-PCR from 2012-09-03 and assembled gene(primer 3+4) with gene(primer5+6)</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI><LI> PCR-products 
</LI></UL><P><B>Method:</B>
Each PCR-product: 10Âµl mix:
</P><UL><LI> 1Âµl DNA
</LI><LI>1Âµl 10x FD Green
</LI><LI> 8Âµl steril water
</LI><LI> 1% agarose gel, 100ml
</LI><LI> 105V, 75min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> gel extraction
</LI><LI> ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> mutagenesis-PCR to remove PstI-sites off scFv-Tmd-EYFP-construct </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Template: scFv-TEV-TMD-EYFP
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC, 10x Phusion buffer GC 
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.RCF25 and 2.mut: RCF25 prefix and 1st mutation PstI in scFV (Tm=72Â°C)
</LI><LI> Primer 3.mut and 4.mut: 1st and 2nd mutation PstI in scFV (Tm= 72Â°C)
</LI><LI> Primer 5.mut and &amp;.RCF25: 2nd mutation and RCF25 suffix (Tm= 72Â°C)
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mixes for  mutagenesis-PCR of gene(primer1+2)</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD>HF-Buffer 10</TD><TD>HF-Buffer+DMSO 10</TD><TD>GC-Buffer 10</TD><TD>GC-Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 1.RCF25 prefix </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 2  1st mutation PstI in scFV 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> DNA (scFv-TEV-TMD-EYFP) 1ng/Âµl</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>50Âµl mixes for  mutagenesis-PCR of gene(primer(3+4)</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD>HF-Buffer 10</TD><TD>HF-Buffer+DMSO 10</TD><TD>GC-Buffer 10</TD><TD>GC-Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 3. 1st mutation in PstI in scFv 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 4. 2nd mutation in PstI in scFv </TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TD> DNA (scFv-TEV-TMD-EYFP) 1ng/Âµl</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>50Âµl mixes for  mutagenesis-PCR of gene(primer(5+6)</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD>HF-Buffer 10</TD><TD>HF-Buffer+DMSO 10</TD><TD>GC-Buffer 10</TD><TD>GC-Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 5. 2nd mutation in PstI in scFv </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer: RCF25 suffix </TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TD> DNA (scFv-TEV-TMD-EYFP) 1ng/Âµl</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration for gene(primer1+2) [s]</B></TD><TD><B>duration for gene(primer3+4) [s]</B></TD><TD><B>duration for gene(primer5+6) [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>30 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 10 </TD><TD> 10 </TD><TD> 5 </TD></TR><TR><TD> elongation</TD><TD> 72 </TD><TD> 5 </TD><TD> 8&lt; /td&gt;</TD><TD> 20 </TD><TD> 5 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 25</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 8 </TD><TD> 15&lt; /td&gt;</TD><TD> 30 </TD><TD> 25 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 600 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> â </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Taks:</B></P><UL><LI> analytical gelelectrophoresis
</LI><LI> PCR-Clean up 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis mutagenesis -PCR to remove PstI-sites off scFv-Tmd-EYFP-construct gene(primer1+2), gene(primer3+4), gene(primer5+6)</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 105V
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> PCR-Clean up
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean up preparative gelelectrophoresis of scFv-TEV-TMD-EYFP-constructs after mutagenesis-PCR and concentration measurement</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><PRE>* according to CleanUp-protocol
</PRE><P>Gene (primer1+2)=
Gene (primer3+4)=
Gene (primer5+6)=
</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> assembly-PCR to of gene(primer1+2) with gene(primer(3+4)</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Template: scFv-TEV-TMD-EYFP
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF 
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.RCF25 prefix (Tm=72Â°C)
</LI><LI> Primer6-.RCF25 suffix (Tm= 72Â°C)
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mixes for  assembly-PCR of gene(primer1+2) with gene(primer3+4)</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD>HF-Buffer 10</TD><TD>HF-Buffer+DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 1.RCF25 prefix </TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer6-.RCF25 suffix </TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> template gene(primer1+2) 1ng/Âµl</TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD> template gene(primer3+4) 0,25ng/Âµl</TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of assembled gene(primer1+2) with gene(primer3+4) to gene12 </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 48Âµl of assembled gene12
</LI></UL><PRE>* 1% agarose gel, 100ml
</PRE><UL><LI> 105V, 75min
</LI></UL><P><B>Results:</B></P><UL><LI> assembly was successful; DNA @ bp
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gel extraction of  assembled gene(primer1+2) with gene(primer3+4) to gene12  and concentration measurement</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> gene12= -- ng/Âµl
</LI></UL><P><B>Further tasks:</B></P><UL><LI> assembly of gene12 to gene(primer5+6) </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> assembly-PCR of gene12 and gene(primer5+6) to gene4</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates:gene12  and gene(primer5+6) 
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC, 10x Phusion buffer GC 
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer: 1.RCF25 RCF25 prefix (Tm=72Â°C)
</LI><LI> Primer: RCF25 suffix (Tm= 72Â°C)
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mixes for  assembly-PCR of gene12 with gene(primer5+6)</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD> GC -Buffer 10</TD><TD> GC -Buffer+DMSO 10</TD><TD> HF -Buffer 10</TD><TD> HF -Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 1.RCF25 prefix </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 2  1st mutation PstI in scFV 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> gene12 (2,4ng/Âµl)</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> gene(primer5+6) (1ng/Âµl)</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration for gene4 GC-buffer [s]</B></TD><TD><B>duration for gene4 GC-buffer + DMSO [s]</B></TD><TD><B>duration for gene4 HF-buffer [s]</B></TD><TD><B>duration for gene4 HF-buffer + DMSO [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>30 </TD><TD>30 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 10 </TD><TD> 10 </TD><TD> 10 </TD><TD> 5 </TD></TR><TR><TD> elongation</TD><TD> 72 </TD><TD> 20 </TD><TD> 20 </TD><TD> 20 </TD><TD> 20 </TD><TD> 5 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 25</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 35 </TD><TD> 35 </TD><TD> 35</TD><TD> 35</TD><TD> 25 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 600 </TD><TD> 600 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> â </TD><TD> â </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Taks:</B></P><UL><LI> analytical gel electrophoresis
</LI><LI> PCR-Clean up 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of assembled gene4</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl gene4, 1Âµl FD Green, 8Âµl water
</LI><LI> 105V,80min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> preparative gel electrophoresis of gene4 
</LI><LI> gel extraction of gene4 </LI></UL><H3><SPAN class="mw-headline" id="2012-09-05_2"><P style="background-color: rgb(240, 20, 70);">2012-09-05</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-09-05 10:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> change of medium of transfected (stable transfection clon 4 and AID-GFP ) cells (04.09.2012)
</LI></UL><UL><LI> passaging of CHOâs
</LI></UL><UL><LI> seeding of 2x 6-wells with each 2*10^5 cells/well
</LI></UL><UL><LI> seeding of one ibidi dish with 5*10^4 cells/well
</LI></UL><UL><LI> talking with Daniel to check the FACS options at the junior group of potsdam university
</LI></UL><UL><LI> investigation about Geniticin and possible resistance genes
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of colony PCR with 20 clones of ligated pcDNA5-FRT-geneart construct </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> analytical gelelectrophoresis to check successful ligation (geneart construct and pcDNA5-FRT vector)</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD FDGreen
</LI></UL><P><B>Method:</B></P><UL><LI> 3Âµl 10x FD Green, 15Âµl of each PCR product
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 70 min at 120V
</LI></UL><P><B>Results:</B></P><UL><LI>no precise bands </LI></UL><P><B>Further Tasks:</B></P><UL><LI> re-doing colony PCR
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of assembled gene4</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 48Âµl of assembled gene4
</LI></UL><PRE>* 1% agarose gel, 100ml
</PRE><UL><LI> 105V, 85min
</LI></UL><P><B>Results:</B></P><UL><LI> assembly was successful; DNA @ bp
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gel extraction of  assembled gene4 </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> gene4= -- ng/Âµl
</LI></UL><P><B>Further tasks:</B></P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gel extraction of  assembled gene(primer1+2) with gene(primer3+4) to gene12  and concentration measurement</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> gene12= -- ng/Âµl
</LI></UL><P><B>Further tasks:</B></P><UL><LI> assembly of gene12 to gene(primer5+6) </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR of scFv-only with RE-site for biobricks</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates:gene12  and gene(primer5+6) 
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC, 10x Phusion buffer GC 
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer scFv: Primer 7.RFC25 prefix and 8.RCF25 suffix
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mixes</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD> GC -Buffer 10</TD><TD> GC -Buffer+DMSO 10</TD><TD> HF -Buffer 10</TD><TD> HF -Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 7.Rcf25 prefix </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 8.RCF25 suffix </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> gene12 (1ng/Âµl)</TD><TD> 10 </TD><TD> 10 </TD><TD> 10 </TD><TD> 10 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 33,5 </TD><TD> 32 </TD><TD> 33,5 </TD><TD> 32 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration for scFv-only GC-buffer [s]</B></TD><TD><B>duration for scFv-only GC-buffer + DMSO [s]</B></TD><TD><B>duration for scFv-only HF-buffer [s]</B></TD><TD><B>duration for scFv-only HF-buffer + DMSO [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>30 </TD><TD>30 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 72</TD><TD> 15 </TD><TD> 15 </TD><TD> 15 </TD><TD> 15 </TD><TD> 5 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 600 </TD><TD> 600 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> â </TD><TD> â </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Taks:</B></P><UL><LI> analytical gel electrophoresis
</LI><LI> PCR-Clean up
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-06_2"><P style="background-color: rgb(240, 20, 70);">2012-09-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-09-06 10:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> stable transfected CHOâs (clone 4) were diluted from 6-well in 75cm^2 flasks
</LI></UL><UL><LI> capturing images of virus infected cells
</LI></UL><UL><LI> talking with Daniel concerning use of FACS
</LI></UL><UL><LI> planing of AK verification at the Cellmembrane ==&gt; anti flag tag
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> colony-PCR of ligated pcDNA5-FRT_geneart-construct clones from 05.09.12 </P><P><B>Investigator:</B> Maria</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_geneart-constructs clones </P><P><B>Materials:</B></P><UL><LI> 20Âµl colony-PCR mix for 32 clones:
</LI></UL><UL><LI> Taq DNA-polymerase 1kb/1min
</LI></UL><UL><LI> 10x Taq DNA-polymerase buffer</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer in C-terminus of CMV of pcDNA5-FRT-backbone: fp_qRT-CMV_rev (Tm=55Â°C)
</LI></UL><UL><LI> reverse primer in C-terminus in mcherry: ps_mcherry_rev_BamHI (Tm= 59,31Â°C)
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Standard Taq-DNA-Polymerase Buffer </TD><TD>64</TD></TR><TR><TD>dNTPs</TD><TD>12,8</TD></TR><TR><TD>Primer (Forward)</TD><TD>12,8</TD></TR><TR><TD>Primer (Reverse)</TD><TD>12,8</TD></TR><TR><TD> DNA </TD><TD> clone from plate </TD></TR><TR><TD> Taq Polymerase </TD><TD> 3,2 </TD></TR><TR><TD> water </TD><TD> 534,4 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 95 </TD><TD>300 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 25 </TD><TD> 30</TD></TR><TR><TD> annealing </TD><TD> 51 </TD><TD> 25 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 68 </TD><TD> 130 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 86 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><UL><LI> clone-colonies were transferred to 20Âµl colony-PCR mix, and onto LB-Amp plate, respectively </LI></UL><UL><LI> LB-AMP-plate were incubated o.n. at 37Â°C</LI></UL><UL><LI> negative control: clone #1 from religation-control (ligation-ratio 1:0) from 2012-09-05, clone 2 from transformation control of new competent E.coli Xl 1 blue with pcDNA5FRT
</LI></UL><P><B>further Taks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> picking positive clones for o.n. culture
</LI></UL><UL><LI> o.n. culture of positive clones
</LI></UL><UL><LI> test digestion
</LI></UL><UL><LI> o.n. culture for endotoxin free preparation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of colony PCR with 32 clones of ligated pcDNA5-FRT-geneart construct, circular vector and religated vector </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> analytical gelelectrophoresis to check successful ligation (geneart construct and pcDNA5-FRT vector)</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD FDGreen
</LI></UL><P><B>Method:</B></P><UL><LI> 3Âµl 10x FD Green, 15Âµl of each PCR product
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 120V
</LI></UL><P><B>Results:</B></P><UL><LI>no precise bands </LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking promising clones
</LI></UL><UL><LI> o.n. culture of promising clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of pcDNA5FRT into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigators:</B> Maria</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI></UL><UL><LI> pcDNA5FRT
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> 20Âµl of resuspended cell-suspension were plated on a LB-Amp-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of assembled gene4</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl ScFv of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 105V,80min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> preparative digestion of scFv-only_delta PstI, scFv-construct_delta PstI, BBA_K404316
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of scFv-only_delta PstI, scFv-construct_delta PstI, BBA_K404316 with NgoMIV and AgeI </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest NgoMIV
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> pcdna5frt (705,4ng/Âµl)
</LI><LI> sterile water</LI></UL><P><B>Method:</B></P><UL><LI> approximately 500ng of DNA (scFv-only_delta PstI, scFv-construct_delta PstI, BBA_K404316)
</LI><LI> 1Âµl NgoMIV 
</LI><LI> 1Âµl AgeI
</LI><LI> 3Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 30Âµl
</LI><LI> digestion for 1,5h at 37Â°C</LI></UL><P><B>&gt;Results:</B></P><UL><LI> suboptimal restrictions enzymes
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> new digestion with XbaI and AgeI
</LI><LI> new assembly to receive more DNA
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR to generate  biobrick-ready genes from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC,  10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.1 and 1.2: Signal peptide-Nanobody-Fc 
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 3.1 and 3.2: Nanobody 
</LI><LI> Primer 4.1 and 4.2(=1.2): Fc 
</LI><LI> Primer5.1 and 5.2: TMD-mcherry 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for every biobrick-ready gene, PCR-mix applies for all genes   </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD> GC -Buffer 10</TD><TD> GC -Buffer+DMSO 10</TD><TD> HF -Buffer 10</TD><TD> HF -Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 1.1: Signal peptide-Nanobody-Fc; Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.1:Nanobody; Primer 4.1: Fc; Primer5.1: TMD-mcherry   </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 1.2: Signal peptide-Nanobody-Fc; Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.2: Nanobody; Primer 4.2(=1.2): Fc; Primer 5.2: TMD-mcherry   </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> geneart-nanobody construct (1ng/Âµl)td&gt;</TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>Program for gene2,4,5</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 14 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 15</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Program for gene1</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Program for gene3</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 6 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 8</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of assembled gene4</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 105V,80min
</LI></UL><P><B>Results:</B></P><UL><LI> unspecific and incorrect DNA bands </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preperative digestion of genart-nanobody-construct to facilitate better annealing of primer to geneart-nanobody construct
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-07_2"><P style="background-color: rgb(240, 20, 70);">2012-09-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-09-07 10:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI></UL><UL><LI> capturing images of AID-GFP
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking promising clones of ligated pcDNA5-FRT-geneart construct and of new competent cells with pcDNA5FRT, inoculation of o.n.-culture </P><P><B>Investigators:</B> Maria</P><P><B>Materials:</B></P><UL><LI> 11x 15 ml inoculation tubes
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Ampicilin
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 10x o.n.-culture of promising pcDNA5-FRT-geneart construct clones
</LI></UL><UL><LI> 1x o.n.-culture of control for new competent cells
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Ampicillin
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of geneart-nanobody-construct with NheI and ApaI </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest NheI
</LI><LI> Fast Digest ApaI
</LI><LI> 10x FD Green Buffer
</LI><LI> geneart-nanobody-construct  (450ng/Âµl)
</LI><LI> sterile water</LI></UL><P><B>Method:</B></P><UL><LI>  3Âµl geneart-nanobody-construct  (450ng/Âµl)
</LI><LI> 1Âµl NgoMIV 
</LI><LI> 1Âµl AgeI
</LI><LI> 3Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 30Âµl
</LI><LI> digestion for 1,5h at 37Â°C</LI></UL><P><B>&gt;Results:</B></P><UL><LI> suboptimal restrictions enzymes
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> new digestion with XbaI and AgeI
</LI></UL><P>new assembly to receive more DNA
</P><H3><SPAN class="mw-headline" id="2012-09-08_2"><P style="background-color: rgb(240, 20, 70);">2012-09-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of promising pcDNA5-FRT-geneart construct clones and new competent cells with pcDNA5FRT</P><P><B>Investigator:</B> Maria</P><P><B>Materials: </B>overnight cultures pcDNA5-FRT-geneart construct clones, control of new competent cells with pcDNA5FRT, Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><P>Clon 4: 589,7 ng/Âµl</P><P>Clon 5: 628 ng/Âµl&lt;br&lt;</P><P>Clon 7: 568,7 ng/Âµl</P><P>Clon 10: 613,4 ng/Âµl</P><P>Clon 11: 573,1 ng/Âµl</P><P>Clon 12: 682,7 ng/Âµl</P><P>Clon 14: 648,1 ng/Âµl</P><P>Clon 16: 615,8 ng/Âµl</P><P>Clon 22: 653,4 ng/Âµl</P><P>Clon 27: 602,5 ng/Âµl</P><P>Clon 29: 613,9 ng/Âµl</P><P>Controle pcDNA5FRT: 205,3 ng/Âµl</P><P>Glycerole stocks of pcDNA5-FRT-geneart construct clones</P><P><B>Further tasks: </B>analytical digestion, analytical gelelectrophoresis, stable transfection of CHO cells</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">analytical digestion of ligated pcDNA5-FRT-geneart construct and pcDNA5FRT control</P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> all 10 prepared pcDNA5-FRT-geneart construct clones
</LI></UL><UL><LI> pcDNA5FRT vector control for new competent cells
</LI></UL><UL><LI> FastDigest ScaI and ApaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 10Âµl mix: 1Âµl ScaI for ligated construct, 1Âµl ApaI for vector, 1Âµl 10x FD Green Buffer, 1Âµl of each clone (approximately 200ng DNA), sterile water add to 10Âµl
</LI></UL><UL><LI> incubation at 37Â°C for 30 min
</LI></UL><P><B>Further tasks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelelectrophoresis of analytical digested pcDNA5-FRT-geneart construct and pcDNA5FRT control </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> checking plasmid-size after ligation of pcDNA5-FRT-geneart construct and pcDNA5FRT control in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI></UL><UL><LI> 120 V
</LI></UL><P><B>Results:</B></P><UL><LI> ligation successfully </LI></UL><P><B>Further Tasks:</B></P><UL><LI> o.n. (50 Âµl) of positive clones 5, 14, 16 and 29 for endotoxin free prep
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-09_2"><P style="background-color: rgb(240, 20, 70);">2012-09-09</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Endotoxin free preparation of clones 5, 14,16 and 29 </P><P><B>Investigator:</B>Maria</P><P><B>Materials: </B>endotoxin free Mediprep kit, overnight culture CMV-Venus</P><P><B>Methods: </B>according to manual </P><P><B>Results: </B>&lt;b/&gt;</P><P>clone 5: 6909,5 ng/Âµl</P><P>clone 14: 6243,6 ng/Âµl</P><P>clone 16: 3837,3 ng/Âµl</P><P>clone 29: 2454,8 ng/Âµl</P><P><B>Further tasks: </B>transient and stable transfection of CHO cells</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of scFv only and scFv construct ligated into pSB1C3 for overnight culture (vector and construct digested with AgeI and NgoMVI)</P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> 15 ml inoculation tubes
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Chloramphenicol
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 8x o.n.-culture of scFv only
</LI></UL><UL><LI> 8x o.n.-culture of scFv construct
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Cm
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> seeding CHO cells in 6-well plate for stable transfection
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of digested nanobody-construct , digested psB1C3-vector and analytical electrophoresis of wt-AID with thiophosphate overhang</P><P><B>Investigator:</B> Rico,Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> digested DNA( nanobody-construct , psB1C3-vector) and  wt-AID with thiophosphate overhang
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 38Âµl of digestd nanobody-construct(NheI, ApaI)
</LI><LI> 28Âµl of digested psB1C3
</LI><LI> 2x 2Âµl of wt-Thio-AID
</LI><LI> 1% agarose gel, 100ml
</LI><LI> 105V, 85min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gel extraction of  digested nanobody-construct , digested psB1C3-vector</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> nanobody-construct(NheI + ApaI)= -- ng/Âµl
</LI><LI> psB1C3 (XbaI + PstI) 
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR-clean up of wt-Thio-Aid
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR to generate  biobrick-ready genes 2,4,5 from digested geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC,  10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 4.1 and 4.2(=1.2): Fc 
</LI><LI> Primer5.1 and 5.2: TMD-mcherry 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>PCR mastermixes ( for 50Âµl single mix for every biobrick-ready gene, PCR-mix applies for all genes)  </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD> GC -Buffer 30</TD><TD> GC -Buffer+DMSO 30</TD><TD> HF -Buffer310</TD><TD> HF -Buffer + DMSO 30</TD></TR><TR><TD>dNTPs</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD></TR><TR><TD>Mastermix for primer: Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 4.1: Fc; Primer5.1: TMD-mcherry   </TD><TD> 10</TD><TD> 10</TD><TD> 10</TD><TD> 10</TD></TR><TR><TD> Mastermix for primer: Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ;  Primer 4.2(=1.2): Fc; Primer 5.2: TMD-mcherry   </TD><TD> 10</TD><TD> 10</TD><TD> 10</TD><TD> 10</TD></TR><TR><TD> geneart-nanobody construct (1ng/Âµl)td&gt;</TD><TD> 3</TD><TD> 3 </TD><TD> 3 </TD><TD> 3 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 4,5 </TD><TD> 0 </TD><TD> 4,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5 </TD></TR><TR><TD> water </TD><TD> 97,5 </TD><TD>93 </TD><TD> 97,5 </TD><TD> 93 </TD></TR></TBODY></TABLE><P><B>Program for gene2,4,5</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 14 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 15</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR to generate  biobrick-ready genes 1 from digested geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer GC,  10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.1 and 1.2: Signal peptide-Nanobody-Fc 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for every biobrick-ready gene, PCR-mix applies for all genes   </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion buffer</TD><TD> GC -Buffer 10</TD><TD> GC -Buffer+DMSO 10</TD><TD> HF -Buffer 10</TD><TD> HF -Buffer + DMSO 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD> Primer 1.1: Signal peptide-Nanobody-Fc </TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> Primer 1.2: Signal peptide-Nanobody-Fc</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD><TD> 2,5</TD></TR><TR><TD> geneart-nanobody construct (1ng/Âµl)td&gt;</TD><TD> 3</TD><TD> 3</TD><TD> 3 </TD><TD> 3 </TD></TR><TR><TD> DMSO</TD><TD> 0 </TD><TD> 1,5 </TD><TD> 0 </TD><TD> 1,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD><TD> 31 </TD><TD> 32,5 </TD><TD> 31 </TD></TR></TBODY></TABLE><P><B>Program for gene1</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of amplified gene2,4,5,1</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 110V,75min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> furhter amplification of genes 1, 2, 4
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-10_2"><P style="background-color: rgb(240, 20, 70);">2012-09-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Mini Prep of pcDNA5-FRT-scFv only and -scFv construct clones </P><P><B>Investigator:</B>Maria</P><P><B>Materials: </B>overnight cultures pcDNA5-FRT-scFv only /-scFv construct clones, Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><P>scFv only </P><P>Clon I 1: 54,2 ng/Âµl</P><P>Clon I 2: 70,1 ng/Âµl&lt;br&lt;</P><P>Clon I 3: 60,6 ng/Âµl</P><P>Clon I 4: 168,3 ng/Âµl</P><P>Clon II 1: 138,6 ng/Âµl</P><P>Clon II 2: 160,2 ng/Âµl</P><P>Clon II 3: 149,8 ng/Âµl</P><P>Clon II 4: 138,8 ng/Âµl</P><P>Clon III 1: 160,5 ng/Âµl</P><P>Clon III 2: 170,5 ng/Âµl</P><P>Clon III 3: 127,9 ng/Âµl</P><P>Clon III 4: 170,9 ng/Âµl</P><P>Clon VI 1: 155,4 ng/Âµl</P><P>Clon VI 2: 116 ng/Âµl</P><P>Clon VI 3: 128,1 ng/Âµl</P><P>Clon VI 4: 96,4 ng/Âµl</P><P><B>Further tasks: </B>analytical digestion, analytical gelelectrophoresis</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">analytical digestion of ligated pcDNA5-FRT-scFv only and scFv construct</P><P><B>Investigators:</B>Maria</P><P><B>Materials:</B></P><UL><LI> all 16 prepared pcDNA5-FRT-scFv only and âscFv construct clones
</LI></UL><UL><LI> FastDigest ScaI and AcuI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 10Âµl mix: 1Âµl AcuI, 1Âµl 10x FD Green Buffer, 2Âµl or 1Âµl of each clone respectively (approximately 150 ng DNA), sterile water add to 10Âµl
</LI></UL><UL><LI> incubation at 37Â°C for 30 min
</LI></UL><P><B>further tasks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelelectrophoresis of analytical digested pcDNA5-FRT-scFv only and âscFv construct </P><P><B>Investigators:</B>Maria</P><P><B>Aim:</B> checking plasmid-size after ligation of pcDNA5-FRT-scFv only and -scFv construct in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI></UL><UL><LI> 120 V
</LI></UL><P><B>Results:</B></P><UL><LI> ligation not successful </LI></UL><P><B>Further Tasks:</B></P><UL><LI> Redoing ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: taking cre recombinase (pBS185 CMV-Cre) into culture </P><P><B>Investigators:</B>Maria</P><P><B>Aim:</B> taking cre recombinase into culture</P><P><B>Materials:</B></P><UL><LI> cre recombinase (addgene: pBS185)
</LI></UL><UL><LI> Agar plate with ampecillin
</LI></UL><P><B>Method:</B></P><UL><LI> streaking cre recombinase onto agar ampicillin plate
</LI></UL><P><B>Results:</B></P><UL><LI> cre recombinase clones for o.n. culture </LI></UL><P><B>Further Tasks:</B></P><UL><LI> o.n. culture of cre recombinase
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> stable transfection of CHO cells with Nanobody construct (Clone 29) in CHO cells
</LI><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> taking HT1080 cells into culture
</LI><LI> starting to test sensitivity of CHO Flp-In cells to G418: cells were seeded in 24-well plate and incubated with different concentrations of G418 (0,1 mg/ml - 1,5mg/ml in 15 steps) for 10 days
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR to generate  biobrick-ready genes from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.1 and 1.2: Signal peptide-Nanobody-Fc 
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 3.1 and 3.2: Nanobody 
</LI><LI> Primer 4.1 and 4.2(=1.2): Fc 
</LI><LI> Primer5.1 and 5.2: TMD-mcherry 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for every biobrick-ready gene, PCR-mix applies for all genes   </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD> 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD> Primer 1.1: Signal peptide-Nanobody-Fc; Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.1:Nanobody; Primer 4.1: Fc; Primer5.1: TMD-mcherry   </TD><TD> 2,5</TD></TR><TR><TD> Primer 1.2: Signal peptide-Nanobody-Fc; Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.2: Nanobody; Primer 4.2(=1.2): Fc; Primer 5.2: TMD-mcherry   </TD><TD> 2,5</TD></TR><TR><TD> geneart-nanobody construct (2ng/Âµl)</TD><TD> 2 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for gene2,4,5</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD>5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 16</TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 16</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P>&lt;/table&gt;<B>Program for gene1</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Program for gene3</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 6 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 8</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of amplified gene1,2,3,4,5</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 110V,80min
</LI></UL><P><B>Results:</B></P><UL><LI> only genes 3 (~350bp) and 5(~800bp) show DNA at desired bp-size</LI></UL><P><B>Further Tasks:</B></P><UL><LI> further amplification of genes 1, 2, 4
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-11_2"><P style="background-color: rgb(240, 20, 70);">2012-09-11</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> assembly PCR of mutated scFv without PstI -TEV-TMD-EYFP construct </P><P><B>Investigator:</B> Maria</P><P><B>AIM:</B> assembly of scFV-TEV-TMD-EGFP without PstI restriction sites </P><P><B>Materials:</B></P><UL><LI> Template mutated scFv and TEV-TMD-EGFP
</LI></UL><UL><LI> Phusion-polymerase
</LI></UL><UL><LI> 10x Phusion buffer GC</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer: small-1-fw-prefix25 (Tm=72Â°C)
</LI></UL><UL><LI> reverse primer: small-6-rw-suffix-25 (Tm= 72Â°C)
</LI></UL><UL><LI> DSMSO
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><P><B>Mastermix for triplex batch</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion GC buffer</TD><TD>30</TD></TR><TR><TD>dNTPs</TD><TD>3</TD></TR><TR><TD>Primer (Forward)</TD><TD>7,5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>7,5</TD></TR><TR><TD> template 1 </TD><TD> 3 </TD></TR><TR><TD> template 2 </TD><TD> 3 </TD></TR><TR><TD> DMSO </TD><TD> 4,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1,5 </TD></TR><TR><TD> water </TD><TD> 87 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5 </TD></TR><TR><TD> annealing + elongation</TD><TD> 72 </TD><TD> 30 </TD><TD> 5 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 25</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 35 </TD><TD> 25 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Taks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of PCR assembly product scFv-TEV-TMD-EGFP </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> checking assembled scFv-TEV-TMD-EGFP PCR result in 1% agarose gel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI></UL><UL><LI> 120 V
</LI></UL><P><B>Results:</B></P><UL><LI> assembly PCR successfully </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparative digestion with AgeI and XbaI
</LI></UL><UL><LI> preparative gelelectrophoresis
</LI></UL><UL><LI> gel extraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> assembly PCR of mutated scFv without PstI and TEV-TMD-EYFP construct </P><P><B>Investigator:</B> Maria</P><P><B>AIM:</B> assembly of scFV-TEV-TMD-EGFP without PstI restriction sites in 4 batches</P><P><B>Materials:</B></P><UL><LI> Template mutated scFv and TEV-TMD-EGFP
</LI></UL><UL><LI> Phusion-polymerase
</LI></UL><UL><LI> 10x Phusion buffer GC and 10x Phusion HD buffer</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer: small-1-fw-prefix25 (Tm=72Â°C)
</LI></UL><UL><LI> reverse primer: small-6-rw-suffix-25 (Tm= 72Â°C)
</LI></UL><UL><LI> DSMSO
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><P><B>Mastermix 1, 2x</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion GC buffer</TD><TD>20</TD></TR><TR><TD>dNTPs</TD><TD>2</TD></TR><TR><TD>Primer (Forward)</TD><TD>5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>5</TD></TR><TR><TD> template 1 </TD><TD> 2 </TD></TR><TR><TD> template 2 </TD><TD> 2 </TD></TR><TR><TD> DMSO </TD><TD> - </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1 </TD></TR><TR><TD> water </TD><TD> 63 </TD></TR></TBODY></TABLE><P><B>Mastermix 2, 2x</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion GC buffer</TD><TD>20</TD></TR><TR><TD>dNTPs</TD><TD>2</TD></TR><TR><TD>Primer (Forward)</TD><TD>5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>5</TD></TR><TR><TD> template 1 </TD><TD> 2 </TD></TR><TR><TD> template 2 </TD><TD> 2 </TD></TR><TR><TD> DMSO </TD><TD> 3 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1 </TD></TR><TR><TD> water </TD><TD> 60 </TD></TR></TBODY></TABLE><P><B>Mastermix 3, 2x</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HC buffer</TD><TD>20</TD></TR><TR><TD>dNTPs</TD><TD>2</TD></TR><TR><TD>Primer (Forward)</TD><TD>5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>5</TD></TR><TR><TD> template 1 </TD><TD> 2 </TD></TR><TR><TD> template 2 </TD><TD> 2 </TD></TR><TR><TD> DMSO </TD><TD> - </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1 </TD></TR><TR><TD> water </TD><TD> 63 </TD></TR></TBODY></TABLE><P><B>Mastermix 4, 2x</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HC buffer</TD><TD>20</TD></TR><TR><TD>dNTPs</TD><TD>2</TD></TR><TR><TD>Primer (Forward)</TD><TD>5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>5</TD></TR><TR><TD> template 1 </TD><TD> 2 </TD></TR><TR><TD> template 2 </TD><TD> 2 </TD></TR><TR><TD> DMSO </TD><TD> 3 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1 </TD></TR><TR><TD> water </TD><TD> 60 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 30 </TD><TD> 1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5 </TD></TR><TR><TD> annealing + elongation</TD><TD> 72 </TD><TD> 30 </TD><TD> 5 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 25</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 35 </TD><TD> 25 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further Taks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> taking cre recombinase (pBS185 CMV-Cre) into culture for single clone picking </P><P><B>Investigators:</B>Maria</P><P><B>Aim:</B> taking cre recombinase into culture</P><P><B>Materials:</B></P><UL><LI> cre recombinase (addgene: pBS185)
</LI></UL><UL><LI> Agar plate with ampicillin
</LI></UL><P><B>Method:</B></P><UL><LI> streaking cre recombinase onto agar ampicillin plate, single colonies wanted
</LI></UL><P><B>Results:</B></P><UL><LI> cre recombinase clones for o.n. culture </LI></UL><P><B>Further Tasks:</B></P><UL><LI> o.n. culture of cre recombinase
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of eGFP ( BBa_404316) into XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigators:</B> Maria</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI></UL><UL><LI> BBa_404316
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> 20Âµl of re-suspended cell-suspension were plated on a LB-Amp-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> modified PCR to generate  biobrick-ready genes from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 4.1 and 4.2(=1.2): Fc 
</LI><LI> Primer5.1 and 5.2: TMD-mcherry 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for every biobrick-ready gene, PCR-mix applies for all genes   </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD> 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD> Primer5.1Â : TMD-mcherryÂ ; Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; 4.1: Fc</TD><TD> 2,5</TD></TR><TR><TD> Primer 5.2: TMD-mcherryÂ ; Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 4.2(=1.2): Fc </TD><TD> 2,5</TD></TR><TR><TD> digested geneart-nanobody construct (10ng/Âµl)</TD><TD> 2 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for gene2,4</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD>15</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 20</TD><TD> 15</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>15</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of amplified gene 2,4</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 115V,80min
</LI></UL><P><B>Results:</B></P><UL><LI> only gene5 (~800bp) show DNA at desired bp-size</LI></UL><P><B>Further Tasks:</B></P><UL><LI> furhter amplification of genes 2, 4
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> modified PCR to generate  biobrick-ready genes (2 and 4) from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 4.1 and 4.2(=1.2): Fc 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>50Âµl mix for every biobrick-ready gene, PCR-mix applies for all genes   </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD> 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD> Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; 4.1: Fc</TD><TD> 2,5</TD></TR><TR><TD> Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 4.2(=1.2): Fc </TD><TD> 2,5</TD></TR><TR><TD> digested geneart-nanobody construct (10ng/Âµl)td&gt;</TD><TD> 2 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for gene2,4</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD>15</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 20</TD><TD> 15</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>15</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-09-12_2"><P style="background-color: rgb(240, 20, 70);">2012-09-12</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of scFV construct with AgeI and XbaI </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> digestion of scFv construct with AgeI and XbaI to generate sticky ends for ligation into pSB1C3 </P><P><B>Materials:</B></P><UL><LI> Fast Digest AgeI
</LI></UL><UL><LI> Fast Digest XbaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> scFv construct
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><P>2x</P><UL><LI> 20Âµl scFv construct
</LI></UL><UL><LI> 2Âµl NheI
</LI></UL><UL><LI> 2Âµl ApaI
</LI></UL><UL><LI> 3Âµl 10x FD Green Buffer
</LI></UL><UL><LI> 5Âµl sterile water
</LI></UL><UL><LI> digestion for 2,5h at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of digested scFv-TEV-TMD-EYFP </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> preparative gelelectrophoresis to separate digested scFv-TEV-TMD-EYFP construct from cutted overhangs in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 2x 30Âµl digested scFv construct
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 100V
</LI></UL><P><B>Results:</B></P><UL><LI> digested scFv construct
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelextracton of digested scFv-TEV-TMD-EYFP construct out of 1% agarosegel </P><P><B>Investigators:</B>Maria</P><P><B>Aim:</B> gelextraction and preparation of cleaned scFv-TEV-TMD-EYFP construct </P><P><B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> 8,3 ng/Âµl
</LI></UL><P><B>Further tasks:</B></P><UL><LI> ligation into pSB1C3
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of scFv-construct with modified PCR mix </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> analytical gelelectrophoresis to check PCr result</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD FDGreen
</LI></UL><P><B>Method:</B></P><UL><LI> 3Âµl 10x FD Green, 15Âµl of each PCR product
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 120V
</LI></UL><P><B>Results:</B></P><UL><LI>no precise bands </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> assembly PCR of mutated scFv without PstI out of part 1 (primer 1 and 2) and part 2 (primer 3 and 4) and mutated scF without PstI (primer 1 and 4) </P><P><B>Investigator:</B> Maria</P><P><B>AIM:</B> assembly of scFV without PstI restriction sites out of part 1 and 2 + multiplication of mutated scFv construct without PstI sites </P><P><B>Materials:</B></P><UL><LI> Template mutated scFv part 1 and 2, mutated scFv construct
</LI></UL><UL><LI> Phusion-polymerase
</LI></UL><UL><LI> 10x Phusion buffer GC</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer: small-I-fw-prefix25 (Tm=72Â°C)
</LI></UL><UL><LI> reverse primer: small-6-rw-suffix-25 (Tm= 72Â°C)
</LI></UL><UL><LI> DSMSO
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><P><B>Mastermix for multiplication of mutated scFv</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion GC buffer</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>2,5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>2,5</TD></TR><TR><TD> template 1 </TD><TD> 1 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Mastermix for triplex batch assembly part 1 and 2</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion GC buffer</TD><TD>30</TD></TR><TR><TD>dNTPs</TD><TD>3</TD></TR><TR><TD>Primer (Forward)</TD><TD>7,5</TD></TR><TR><TD>Primer (Reverse)</TD><TD>7,5</TD></TR><TR><TD> template 1 </TD><TD> 1,75 (5,7 ng/Âµl) </TD></TR><TR><TD> template 2 </TD><TD> 1,36 (3,4 ng/Âµl) </TD></TR><TR><TD> DMSO </TD><TD> 4,5 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1,5 </TD></TR><TR><TD> water </TD><TD> 86,67 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 5</TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 10 </TD><TD> 5 </TD></TR><TR><TD> annealing + elongation</TD><TD> 72 </TD><TD> 30 </TD><TD> 5 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 25</TD></TR><TR><TD> annealing </TD><TD> 72 </TD><TD> 35 </TD><TD> 25 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of cre recombinase and BBa_404316, inoculation of o.n.-culture </P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> 15 ml inoculation tubes
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Ampicillin
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 1x o.n.-culture of cre recombinase
</LI></UL><UL><LI> 1x o.n.-culture of BBa-404316
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Ampicillin
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> seeding CHO cells in Ibidi Dish for transfection and Virus infection experiment
</LI><LI> dilution of stably transfected CHO Clone 29 into 75cmÂ² flasks (+550Âµg/ml Hygromycin)(according to protocol)
</LI><LI> co-transfection of Nanobody construct clone 29 and modified AID with eGFP (according to protocol)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of amplified genes 2 and 4 and preparative gel electrophoresis of digested geneart-nanobody construct </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 110V,80min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> amplification of gene 1
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR to generate  biobrick-ready gene 1 from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> DMSO
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 1.1 and 1.2: Signal peptide-Nanobody-Fc 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B> 4x 50Âµl PCR-mix for gene1</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD>40</TD></TR><TR><TD>dNTPs</TD><TD>4</TD></TR><TR><TD> Primer 1.1: Signal peptide-Nanobody-Fc </TD><TD> 10</TD></TR><TR><TD> Primer 1.2: Signal peptide-Nanobody-Fc </TD><TD> 10</TD></TR><TR><TD> geneart-nanobody construct (25ng/Âµl)</TD><TD> 1 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 2 </TD></TR><TR><TD> water </TD><TD> 133</TD></TR></TBODY></TABLE><UL><LI> PCR-Mastermix of gene1 was split into 4x 50Âµl mixes
</LI></UL><P><B>Program for gene1</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD> 15</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 20 </TD><TD> 15</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>15</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of amplified genes 2 and 4 and preparative gel electrophoresis of digested geneart-nanobody construct </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI><LI> digested geneart-nanobody construct (~2,2Âµg)
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 100V,95min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> PCR-clean up of genes 1,4,5
</LI><LI> digestion of genes 1,4,5 and psB1C3 with XbaI and AgeI
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-13_2"><P style="background-color: rgb(240, 20, 70);">2012-09-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of cre recombinase, BBa_404316 and YFP</P><P><B>Investigator:</B> Maria</P><P><B>Materials: </B>overnight cultures, Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><P>Cre recombinaseÂ : 192,8 ng/Âµl</P><P>BBa_404316: 242,5 ng/Âµl&lt;br&lt;</P><P>YFP: 130,6 ng/Âµl</P><P>Glycerol stocks of cre recombinase and BBa_404316</P><P><B>Further tasks: </B>preparative digestion of BBa_404316 and 50ml culture of cre recombinase for endotoxin free preparation</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of assembly PCR result scFv only (primer 7 and 8) </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> analytical gelelectrophoresis to check PCR result</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD FDGreen
</LI></UL><P><B>Method:</B></P><UL><LI> 1Âµl 10x FD Green, 1Âµl of each PCR product, add sterile water to 10 Âµl
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 120V
</LI></UL><P><B>Results:</B></P><UL><LI>precise bands </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparative gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of PCR result for scFV only (primer 7 and 8)</P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> preparative gelelectrophoresis of assembled scFv only in 1% agarose gel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 2x 30Âµl of scFv only
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 100V
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelextracton of PCR product scFv only out of 1% agarosegel </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> gelextraction and preparation of cleaned scFv only construct </P><P><B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> 1: 86,1 ng/Âµl
</LI></UL><UL><LI> 2: 108,2 ng/Âµl
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> digestion with AgeI and XbaI
</LI></UL><UL><LI> ligation into pSB1C3
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of scFV only and BBa_404316 with AgeI and XbaI </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> digestion of scFv only and BBa_404316 (pSB1C3 with RCF25 standard) with AgeI and XbaI </P><P><B>Materials:</B></P><UL><LI> Fast Digest AgeI
</LI></UL><UL><LI> Fast Digest XbaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> scFv only
</LI></UL><UL><LI> BBa_404316
</LI></UL><UL><LI> steril water
</LI></UL><P><B>Method:</B></P><P>2x</P><UL><LI> 8,25Âµl BBa_404316
</LI></UL><UL><LI> 1Âµl AgeI
</LI></UL><UL><LI> 1Âµl XbaI
</LI></UL><UL><LI> 3Âµl 10x FD Green Buffer
</LI></UL><UL><LI> 16,75 Âµl sterile water
</LI></UL><P>1x</P><UL><LI> 13,8Âµl scFv only
</LI></UL><UL><LI> 1Âµl AgeI
</LI></UL><UL><LI> 1Âµl XbaI
</LI></UL><UL><LI> 3Âµl 10x FD Green Buffer
</LI></UL><UL><LI> 11,2Âµl sterile water
</LI></UL><UL><LI> digestion for 2,5h at 37Â°C
</LI></UL><UL><LI>heat inactivation at 65Â°C for 5 min
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> PCR clean-up of digested scFV only
</LI></UL><UL><LI> dephosphorylation of pSB1C3
</LI></UL><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean-Up of digested scFv only construct </P><P><B>Investigator:</B>Maria</P><P><B>Aim:</B> purifying scFv only construct</P><P><B>Materials:</B></P><UL><LI> PCR-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> concentration of digested scFv only 66,8 ng/Âµl
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> dephosphorylating the digested pSB1C3 vector</P><P><B>Investigator:</B>Maria</P><P><B>Aim:</B>dephosphorylating the digested pSB1C3 vector to prevent re-ligation </P><P><B>Materials:</B></P><UL><LI> Antarctic Phosphatase
</LI></UL><UL><LI> 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> digested pSB1C3
</LI></UL><P><B>Method:</B></P><UL><LI> 3,3Âµl 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> 1Âµl Antarctic Phosphatase
</LI></UL><UL><LI> incubation for 30min at 37Â°C to dephosphorylate 5Â´-ends of pSB1C3
</LI></UL><UL><LI> heat inactivation of Phosphatase for 5min at 65Â°C
</LI></UL><UL><LI> store at -20Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparative gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of digested and dephosphorylated pSB1C3</P><P><B>Investigators:</B>Maria</P><P><B>Aim:</B> preparative gelelectrophoresis of digested and dephosphorylated pSB1C3 in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 2x 30Âµl digested pSB1C3
</LI></UL><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 100V
</LI></UL><P><B>Results:</B></P><UL><LI> cleaned and digested pSB1C3 and scFv only construct
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelextracton of digested and dephosphorylated pSB1C3 out of 1% agarose gel </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> gelextraction and preparation of digested and dephosphorylated pSB1C3 </P><P><B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B></P><UL><LI> I: 49,7 ng/Âµl, II: 53,3ng/Âµl
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> ligation with scFv only and scFv construct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> ligation of digested scFv-only and scFV construct and dephosporylated pSB1C3 </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> ligation of digested scFv-only and scFv construct with pSB1C3</P><P><B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI></UL><UL><LI> T4 DNA ligae
</LI></UL><UL><LI> ligase buffer
</LI></UL><UL><LI> digested scFv-only and scFv construct
</LI></UL><UL><LI> digested and dephosporylated pSB1C3
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B></P><P>I scFv only:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI></UL><UL><LI> 2,5Âµl T4 DNA-ligase buffer
</LI></UL><UL><LI> 1,24Âµl digested scFv only (22,26ng/Âµl)
</LI></UL><UL><LI> 1Âµl digested pSB1C3 (26,65ng/Âµl)
</LI></UL><UL><LI> 14,26Âµl sterile water
</LI></UL><P>II scFv construct:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI></UL><UL><LI> 2,5Âµl T4 DNA-ligase buffer
</LI></UL><UL><LI> 4,81Âµl digested scFv construct (13,1ng/Âµl)
</LI></UL><UL><LI> 1Âµl digested pSB1C3 (26,65ng/Âµl)
</LI></UL><UL><LI> 10,69Âµl sterile water
</LI></UL><UL><LI> incubation for 1h at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competent <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pSB1C3-scFv-only and pSB1C3-scFv-construct into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI></UL><UL><LI> 10 Âµl of pSB1C3-scFv-only and pSB1C3-scFv-construct
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> 20Âµl of resuspended cell-suspension were plated on a LB-Cm-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator: </B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> seeding HT1080 in Ibidi Dish for Virus infection experiment
</LI><LI> transfection of CHO cells in Ibidi Dish with Nanobody construct (according to protocol)
</LI><LI> passaging of HEK AAV 293
</LI><LI> registration at FACS core facility at Deutsches Rheumaforschungszentrum
</LI><LI> planning of further experiments
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean up genes biobrick-ready genes 1,4,5</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><UL><LI> according to CleanUp-protocol
</LI></UL><P>Gene 1 (Signal peptide-Nanobody-Fc) = 112,4ng/Âµl
Gene 4 (Fc): =  17,1ng/Âµl
Gene 5 (TMD-mcherry) = 193ng/Âµl 
<B>Further Taks:</B></P><UL><LI> preparative digestion of genes 1,4,5 with XbaI and AgeI
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of genes 1,4 and 5 with XbaI and AgeI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XbaI
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> genes 1,4,5
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
2x</P><UL><LI> 15Âµl of gene1 (112,4ng/Âµl)
</LI><LI> 10Âµl og gene 5 (193ng/Âµl)
</LI><LI> 25Âµl of gene4 (17,1ng/Âµl)
</LI><LI> 2Âµl NheI
</LI><LI> 2Âµl ApaI
</LI><LI> 4Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 40Âµl
</LI><LI> digestion for 2h at 37Â°C</LI></UL><P><B>&gt;Results:</B><B>Further Tasks:</B></P><UL><LI> PCR-clean up of digested genes 1,4,5
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gel electrophoresis followed by gel extraction of digested genes 1,4,5</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI><LI> digested genes 1,4,5
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> whole digestion-mix of each gene
</LI><LI> 100V,95min
</LI></UL><P><B>Results:</B></P><UL><LI>desired DNA bands for gene1: approximately 880bp, gene 4: 700bp, gene5: 800
</LI></UL><P><B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><UL><LI> according to gel extraction-protocol
</LI></UL><P>Gene 1 (Signal peptide-Nanobody-Fc) = 18,3ng/Âµl, R=1,75 
Gene 4 (Fc): =  10,4ng/Âµl, R= 1,72
Gene 5 (TMD-mcherry) = 68,1ng/Âµl, R= 1,82 
<B>Further Taks:</B></P><UL><LI> ligation of digested genes 1,4,5 into digested psB1C3 (X+A)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> modified PCR to generate biobrick-ready genes (2 and 3) from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Primer 3.1 and 3.2: Nanobody 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B> 4x 50Âµl PCR-mastermix for gene2 and 3, PCR-mix applies for all genes </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><I>volume [ÂµL]'</I></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD> 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD> Primer 2.1: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.1: Nanobody </TD><TD> 2,5</TD></TR><TR><TD> Primer 2.2: TEV-LoxP-TMD-mcherry-LoxPÂ ; Primer 3.2: Nanobody </TD><TD> 2,5</TD></TR><TR><TD> digested geneart-nanobody construct (10ng/Âµl)td&gt;</TD><TD> 2 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for gene2</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD>15</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 20</TD><TD> 15</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>15</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Program for gene3</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>5</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD> 5</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 6 </TD><TD> 5</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>25</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 8</TD><TD>25</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-09-14_2"><P style="background-color: rgb(240, 20, 70);">2012-09-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pSB1C3-scFv-only and pSB1C3-scFv-construct into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI></UL><UL><LI> 10 Âµl of pSB1C3-scFv-only and pSB1C3-scFv-construct
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> 20Âµl of resuspended cell-suspension were plated on a LB-CM-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> dilution of stably transfected cells (scFv construct clone 4, originally from transfected Well 3) to 1-2 cells per 150Âµl and seeding in 3x 96-Well plates
</LI><LI> passaging of HT1080
</LI><LI> infection of CHO cells in Ibidi Dish with Virus (YFP on surface and CFP as GOI)(according to protocol)
</LI><LI> continuation of G418 selection experiment
</LI><LI> planning of further experiments
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of amplified genes 2 and 3</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 110V,85min
</LI></UL><P><B>Results:</B></P><UL><LI>  gene 3 shows sufficient bp-size (~350bp)</LI></UL><P><B>Further Tasks:</B></P><UL><LI> further PCR of gene2
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Topic: ligation of digested dephosporylated pSB1C3 (X+A) and digested genes 1,4,5 (X+A) </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI><LI> T4 DNA ligae
</LI><LI> ligase buffer
</LI><LI> digested genes 1,4,5
</LI><LI> digested and dephosporylated pSB1C3
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B>
1st ligation mix-gene1:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI> 32,3 ng insert (gene1)
</LI><LI> sterile water ad 20Âµl</LI></UL><P>2nd ligation mix-gene4:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI> 32,3 ng insert (gene4)
</LI><LI> sterile water ad 20Âµl</LI></UL><P>3rd ligation mix-gene1:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI> 22,9 ng insert (gene5)
</LI><LI> sterile water ad 20Âµl</LI></UL><UL><LI> incubation for 40min at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competent <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pSB1C3-gene 1, 4, 5 into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI><LI> 15 Âµl of each ligation mix  
</LI><LI> icebox
</LI><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI><LI> 20Âµl of resuspended cell-suspension were plated on a LB-Cm-plate
</LI><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> modified PCR to generate biobrick-ready genes 2 from geneart-nanobody construct</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Templates: digested geneart-nanobody construct
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF
</LI><LI> dNTPs (10mM)
</LI><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP 
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B> 4x 50Âµl PCR-mastermix for gene2 , PCR-mix applies for all genes</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HF-buffer</TD><TD> 10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD> Primer 2.1: TEV-LoxP-TMD-mcherry-LoxP </TD><TD> 2,5</TD></TR><TR><TD> Primer 2.2: TEV-LoxP-TMD-mcherry-LoxP </TD><TD> 2,5</TD></TR><TR><TD> digested geneart-nanobody construct (10ng/Âµl)</TD><TD> 2 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 32,5 </TD></TR></TBODY></TABLE><P><B>Program for gene2</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD> annealing </TD><TD> gradient: 55Â°C, 60Â°C, 70Â°C </TD><TD> 10 </TD><TD>15</TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 20</TD><TD> 15</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD>15</TD></TR><TR><TD>annealing/elongation </TD><TD> 72 </TD><TD> 20</TD><TD>15</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-09-15_2"><P style="background-color: rgb(240, 20, 70);">2012-09-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of ligated scFv-only-pSB1C3 and scFv-construct-pSB13C </P><P><B>Investigator:</B> Kathi</P><P><B>Materials:</B></P><UL><LI> 11x 15 ml inoculation tubes
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Chloramphenicol</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 5x o.n.-culture of scFv-only-pSB1C3
</LI></UL><UL><LI> 5x o.n.-culture of scFv-construct-pSB1C3
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Chloramphenicol
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of ligated pSB1C3-genes1,4 and 5 </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> 3x5 ml inoculation tubes 
</LI><LI> LB-Medium
</LI><LI> Chloramphenicol
</LI><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 5x o.n.-culture of pSB1C3-gene1(Signal peptide-Nanobody-Fc)
</LI><LI> 5x o.n.-culture of pSB1C3-gene4(Fc)
</LI><LI> 5x o.n. culture of psB1C3-gene5(TMD-mcherry)
</LI><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Chloramphenicol
</LI><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of amplified genes 2 </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> 2Âµl of each PCR-mix, 1Âµl FD Green, 8Âµl water
</LI><LI> 110V,85min
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> PCR-Clean up 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-clean up of digested genes 2 and 3</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><UL><LI> according to CleanUp-protocol
</LI></UL><P>Gene 2 (Signal peptide-Nanobody-Fc) = --ng/Âµl, R=--
Gene 3 (Fc): =  --ng/Âµl, R= --
</P><P><B>Further Taks:</B></P><UL><LI> preparative digestion of genes 2 and 3 with XbaI and AgeI
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of genes 2 and 3 with XbaI and AgeI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XbaI
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> genes 1,4,5
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
2x</P><UL><LI> --Âµl of gene1 (--ng/Âµl)
</LI><LI> --Âµl og gene 5 (---ng/Âµl)
</LI><LI> 25Âµl of gene4 (---ng/Âµl)
</LI><LI> 2Âµl NheI
</LI><LI> 2Âµl ApaI
</LI><LI> 4Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 40Âµl
</LI><LI> digestion for 2h at 37Â°C</LI></UL><P><B>Results:</B><B>Further Tasks:</B></P><UL><LI> preparative gel electrophoresis and gel extraction of digested gene2 and 3 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gel electrophoresis followd by gel extraction of digested genes 2 and 3</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI><LI> amplified DNA
</LI><LI> digested genes 2 and 3
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> whole digestion-mix of each gene
</LI><LI> 100V,95min
</LI></UL><P><B>Results:</B></P><P><B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><UL><LI> according to gel extraction-protocol
</LI></UL><P>Gene 2 (TEV-LoxP-TMD-mcherry-LoxP ) = 18,3ng/Âµl, R=1,75 
Gene 3 (Nanobody): =  10,4ng/Âµl, R= 1,72
<B>Further Taks:</B></P><UL><LI> ligation of digested genes 2 and 3 into digested psB1C3 (X+A)
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-16_2"><P style="background-color: rgb(240, 20, 70);">2012-09-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of scFv-only-pSB1C3 and scFv-construct-pSB1C3 clones</P><P><B>Investigator:</B> Maria</P><P><B>Materials: </B>overnight cultures scFv-only-pSB1C3 and scFv-construct-pSB1C3 clones, Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><UL><LI> scFv-only-pSB1C3
</LI></UL><P>Clon 1: 288,3ng/Âµl</P><P>Clon 2: 309,8ng/Âµl&lt;br&lt;</P><P>Clon 3: 408,2ng/Âµl</P><P>Clon 4: 100,7ng/Âµl</P><P>Clon 5: 363,8ng/Âµl</P><UL><LI>scFv-construct-pSB1C3
</LI></UL><P>Clon 1: 398,8ng/Âµl</P><P>Clon 2: 439,3ng/Âµl</P><P>Clon 3: 337,4ng/Âµl</P><P>Clon 4: 80ng/Âµl</P><P>Clon 5: 125,9ng/Âµl</P><P>Glycerole stocks of all clones</P><P><B>Further Tasks: </B>analytical digestion, analytical gelelectrophoresis</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">analytical digestion of ligated scFv-only-pSB1C3 and scFv-construct-pSB1C3 with PstI and XbaI</P><P><B>Investigator:</B> Maria</P><P><B>Materials:</B></P><UL><LI> all 10 prepared scFv-only-pSB1C3 and scFv-construct-pSB1C3 clones
</LI></UL><UL><LI> pSB1C3 vector control
</LI></UL><UL><LI> FastDigest PstI and XbaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 10Âµl mix: 1Âµl PstI, 1Âµl 10x FD Green Buffer, 1Âµl of each clone (1:2, approximately 200ng DNA), 7Âµl sterile water
</LI></UL><UL><LI> 11Âµl mix: 1Âµl XbaI, 1Âµl 10x FD Green Buffer, 1Âµl of each clone (1:2, approximately 200ng DNA), 7Âµl sterile water
</LI></UL><UL><LI> incubation at 37Â°C for 45 min
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analytical gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelelectrophoresis of analytical digested scFv-only-pSB1C3 and scFv-construct-pSB1C3 clones </P><P><B>Investigator:</B> Maria</P><P><B>Aim:</B> checking plasmid-size after ligation in 1% agarose gel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarose gel, 100ml
</LI></UL><UL><LI> 120 V
</LI></UL><P><B>Results:</B></P><UL><LI> ligation partly successful </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparation for sequencing
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging HEK AAV 293
</LI><LI> passaging HT1080
</LI><LI> change of medium in 75cmÂ² flasks with stably transfected CHO clone 29 and clone 4 (+550Âµg/ml Hygromycin)
</LI><LI> capturing images of virus-infection in Ibidi Dish 8see Results)
</LI><LI> continuation of G418 selection experiment
</LI><LI> seeding CHO in Ibidi Dishes for transfection and virus infection experiment
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> ligation of digested dephosporylated pSB1C3 (X+A) and digested genes 2 and 3 (X+A) </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI><LI> T4 DNA ligase
</LI><LI> ligase buffer
</LI><LI> digested genes 2 and 3
</LI><LI> digested and dephosporylated pSB1C3
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B>
1st ligation mix-gene2:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI> 27 ng insert (gene1)
</LI><LI> sterile water ad 20Âµl</LI></UL><P>2nd ligation mix-gene3:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI> 10,5 ng insert (gene3)
</LI><LI> sterile water ad 20Âµl</LI><LI> incubation for 40min at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competent <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of psB1C3-gene1,4 and 5 constructs</P><P><B>Investigator:</B> Sascha<B>Materials: </B>overnight cultures of psB1C3-gene1,4 and 5 constructs, Mini prep kit <B>Methods: </B>according to manual <B>Results: </B></P><UL><LI> psB1C3-gene1
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl</P><UL><LI> psB1C3-gene4
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl</P><UL><LI> psB1C3-gene5
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl
Glycerole stocks of all clones<B>Further Tasks: </B></P><UL><LI> test digestion, analytical gelelectrophoresis</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> test digestion of psB1C3-gene1,4 and 5 constructs  XbaI and AgeI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XbaI
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> psB1C3-gene1,4 and 5 constructs
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
Mastermix for test digestion: for 16 reactions; subsequently specified for one reaction </P><UL><LI> ~ 100ng DNA
</LI><LI> 1Âµl NheI
</LI><LI> 1Âµl ApaI
</LI><LI> 1Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 10Âµl
</LI><LI> digestion for 30min at 37Â°C</LI></UL><P><B>Results:</B><B>Further Tasks:</B></P><UL><LI> analytical gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of psB1C3-gene1,4 and 5 constructs </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> 10Âµl of each digestion mix
</LI><LI> 110V,85min
</LI></UL><P><B>Results:</B></P><UL><LI>  all genes show satisfaying bp-size
</LI><LI> gene 1.2, gene1.5, gene4.2 and gene5.2 picked up for sequencing</LI></UL><P><B>Further Tasks:</B></P><UL><LI> sequencing</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  planing immunostaining to detect expressed scFv on surface of CHO cells</P><P><B>Investigator:</B> Kerstin, Sascha<B>Materials:</B></P><UL><LI> pubmed
</LI><LI> online data bases
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-17_2"><P style="background-color: rgb(240, 20, 70);">2012-09-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin
</LI><LI> change of medium in 75cmÂ² flasks with stably transfected CHO clone 29 (+550Âµg/ml Hygromycin)
</LI><LI> capturing images of virus-infection in Ibidi Dish (see Results)
</LI><LI> seeding cells for co-transfection with Nanobody construct and Cre Recombinase
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> choosing clones  and preparing ligated psB1C3-genes1,4,5, scFv-only and scFv-construct for ordering sequencing by GATC</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI>dilute DNA  according to GATC requirements
</LI><LI>Geneious</LI><LI>Using psB1C3 primer fw/rv</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analysis of sequencing results
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of geneart-nanobody-construct with XbaI and AgeI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XbaI
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> geneart-nanobody-construct 
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
2x digestoin mix</P><UL><LI> ~2,3Âµg geneart-nanobody-construct 
</LI><LI> 2Âµl NheI
</LI><LI> 2Âµl ApaI
</LI><LI> 4Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 40Âµl
</LI><LI> digestion for 2h at 37Â°C</LI></UL><P><B>Results:</B><B>Further Tasks:</B></P><UL><LI> preparative gel electrophoresis and gel extraction of digested geneart-nanobody-construct 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gel electrophoresis followed by gel extraction of digested geneart-nanobody-construct </P><P><B>Investigator:</B> Basia/ Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> amplified DNA
</LI><LI> geneart-nanobody-construct 
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 100ml
</LI><LI> whole digestion-mix of each gene
</LI><LI> 100V,95min
</LI></UL><P><B>Results:</B></P><P><B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </LI></UL><P><B>Methods:</B></P><UL><LI> according to gel extraction-protocol
</LI></UL><P>geneart-nanobody-construct = ng/Âµl, R=
</P><P><B>Further Tasks:</B></P><UL><LI> ligation of digested geneart-nanobody-construct  into digested psB1C3 (X+A)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> ligation of digested dephosporylated pSB1C3 (X+A) and digested geneart-nanobody-construct  (X+A) </P><P><B>Investigator:</B> Basia<B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI><LI> T4 DNA ligase
</LI><LI> ligase buffer
</LI><LI> digested geneart-nanobody-construct 
</LI><LI> digested and dephosporylated pSB1C3
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B>
1st ligation mix-gene2:
</P><UL><LI> 1Âµl T4 DNA ligase
</LI><LI> 2Âµl T4 DNA-ligase buffer
</LI><LI> 20 ng dig. psB1C3
</LI><LI>  ng insert (geneart-nanobody-construct )
</LI><LI> sterile water ad 20Âµl</LI><LI> incubation for 40min at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competent <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pSB1C3-geneart-nanobody-construct into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigator:</B> Basia<B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicillin, 37 Â°C thermo mixer, centrifuge,</LI><LI> 15 Âµl of each ligation mix  
</LI><LI> icebox
</LI><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI><LI> 20Âµl of resuspended cell-suspension were plated on a LB-Cm-plate
</LI><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of ligated pSB1C3-genes2 and 3</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> 2x 5 ml inoculation tubes 
</LI><LI> LB-Medium
</LI><LI> Chloramphenicol
</LI><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 5x o.n.-culture of pSB1C3-gene2(TEV-LoxP-TMD-mcherry-Lox)
</LI><LI> 5x o.n.-culture of pSB1C3-gene3(Nanobody)
</LI><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Chloramphenicol
</LI><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  planing immunostaining to detect expressed scFv on surface of CHO cells</P><P><B>Investigator:</B> Kerstin, Sascha<B>Materials:</B></P><UL><LI> pubmed
</LI><LI> online data bases
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-18_2"><P style="background-color: rgb(240, 20, 70);">2012-09-18</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin
</LI><LI> passaging of HEK AAV 293
</LI><LI> passaging of HT1080
</LI><LI> continuation of G418 selection experiment
</LI><LI> seeding CHO cells for transfection and FACS
</LI><LI> seeding cells for co-transfection with Nanobody construct and Cre Recombinase
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of psB1C3-gene 2,3 constructs</P><P><B>Investigator:</B> Sascha<B>Materials: </B>overnight cultures of psB1C3-gene2,3 constructs, Mini prep kit <B>Methods: </B>according to manual <B>Results: </B></P><UL><LI> psB1C3-gene2
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl</P><UL><LI> psB1C3-gene3
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl
Glycerole stocks of all clones<B>Further Tasks: </B></P><UL><LI> test digestion, analytical gelelectrophoresis</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> test digestion of psB1C3-gene2 and 3 constructs  XbaI and AgeI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XbaI
</LI><LI> Fast Digest AgeI
</LI><LI> 10x FD Green Buffer
</LI><LI> psB1C3-gene1,4 and 5 constructs
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
Mastermix for test digestion: for 11 reactions; subsequently specified for one reaction </P><UL><LI> ~ 100ng DNA
</LI><LI> 1Âµl NheI
</LI><LI> 1Âµl ApaI
</LI><LI> 1Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 10Âµl
</LI><LI> digestion for 30min at 37Â°C</LI></UL><P><B>&gt;Results:</B><B>Further Tasks:</B></P><UL><LI> analytical gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of psB1C3-gene2 and 3constructs </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 10Âµl of each digestion mix
</LI><LI> 115V,80min
</LI></UL><P><B>Results:</B></P><UL><LI>  all genes show satisfaying bp-size
</LI><LI> gene 2.5 and gene3.3  picked up for sequencing</LI></UL><P><B>Further Tasks:</B></P><UL><LI> sequencing</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> choosing clones  and preparing ligated psB1C3-genes2, 3 construct for ordering sequencing by GATC</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI>dilute DNA  according to GATC requirements
</LI><LI>Geneious</LI><LI>Using psB1C3 primer fw/rv</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analysis of sequencing result
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> picking clones of ligated pSB1C3-geneart-nanobody-construct</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> 1x 5 ml inoculation tubes 
</LI><LI> LB-Medium
</LI><LI> Chloramphenicol
</LI><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 5x o.n.-culture of pSB1C3-geneart-nanobody construct
</LI><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Chloramphenicol
</LI><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  planing immunostaining to detect expressed scFv on surface of CHO cells</P><P><B>Investigator:</B> Kerstin, Sascha<B>Materials:</B></P><UL><LI> pubmed
</LI><LI> online data bases
</LI></UL><P><B>Results:</B></P><UL><LI> Pacific Blue goat anti-mouse IgG
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-19_2"><P style="background-color: rgb(240, 20, 70);">2012-09-19</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> change of medium in 75cmÂ² flasks with stably transfected CHO clone 4 (+550Âµg/ml Hygromycin)
</LI><LI> co-transfection of CHO cells with nanobody construct (clone 29) and Cre Recombinase in 2x 25cmÂ² culture flasks 
</LI><LI> HEK 293 (20000 cells/well) and HeLa (10000 cells/well) were seeded in Ibidi Dish for transfection
</LI><LI> first attempt at live-cell immunfluorescence: incubation with primary antibody (anti-flag-tag antibody) in serum-free medium for 1h on shaker, after careful washing incubation with secondary antibody (Pacific Blue Alexa FluorÂ®) for 1h, capturing of images was done with the fluorescence microscope
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  looking for new transmembrane domains and signal peptides</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> pubmed
</LI><LI> online data bases
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stock and Mini Prep of psB1C3-nanobody construct</P><P><B>Investigator:</B> Sascha<B>Materials: </B>overnight cultures of psB1C3-nanobody construct, Mini prep kit <B>Methods: </B>according to manual <B>Results: </B></P><UL><LI> psB1C3-nanobody construct
</LI></UL><P>Clon 1: ng/Âµl
Clon 2: ng/Âµl&lt;br&lt;
Clon 3: ng/Âµl
Clon 4: ng/Âµl
Clon 5: ng/Âµl
Glycerole stocks of all clones<B>Further Tasks: </B></P><UL><LI> test digestion, analytical gelelectrophoresis</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> test digestion of psB1C3-nanobody construct  with XhoI</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> Fast Digest XhoI
</LI><LI> 10x FD Green Buffer
</LI><LI> psB1C3-nanobody
</LI><LI> sterile water
</LI></UL><P><B>Method:</B>
Mastermix for test digestion: for 6 reactions; subsequently specified for one reaction </P><UL><LI> ~ 100ng DNA
</LI><LI> 1Âµl XhoII
</LI><LI> 1Âµl 10x FD Green Buffer
</LI><LI> sterile water ad 10Âµl
</LI><LI> digestion for 30min at 37Â°C</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> analytical gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gel electrophoresis of psB1C3-nanobody construct </P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 10Âµl of each digestion mix
</LI><LI> 115V,80min
</LI></UL><P><B>Results:</B></P><UL><LI> selecting clone 1 for sequencing</LI></UL><P><B>Further Tasks:</B></P><UL><LI> sequencing</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> choosing clones  and preparing ligated psB1C3-nanobody construct for ordering sequencing by GATC</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI>dilute DNA  according to GATC requirements
</LI><LI>Geneious</LI><LI>Using psB1C3 primer fw/rv</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analysis of sequencing result
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  planing of detecting nanobody and scFv in transfected CHO cells </P><P><B>Investigator:</B> Kerstin, Sascha<B>Materials:</B></P><UL><LI> pubmed
</LI><LI> online data bases
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-20_2"><P style="background-color: rgb(240, 20, 70);">2012-09-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> capturing images of stably transfected CHO clone 4 cells and transiently transfected CHO clone 29 cells with the confocal microscope of Dr. Baumann of the University of Potsdam (see Results)
</LI><LI> passaging of CHO cells w/ zeocin
</LI><LI> passaging of HT1080
</LI><LI> evaluation of G418 selection experiment: sensitivity of CHO cells towards G418
</LI><LI> transfection of HEK293 and HeLa cells with scFv construct (clone 4) and Nanobody construct (clone 29)	
</LI><LI> change of medium in 75cmÂ² flasks with stably transfected CHO Clone 29 (+550Âµg/ml Hygromycin)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of ligated psB1C3-gene1, 4, 5 constructs </P><P><B>Investigators:</B> Maria, Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from  psB1C3-gene1, 4, 5 constructs
</LI><LI> Geneious
</LI></UL><P><B>Results:</B></P><UL><LI>  all sequencing results show complementarity with theoretical gene constructs created in geneious</LI></UL><H3><SPAN class="mw-headline" id="2012-09-21_2"><P style="background-color: rgb(240, 20, 70);">2012-09-21</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> cell sorting at the FACS facility of the Deutsches Rheuma-Forschungszentrum (DRFZ): cells were sorted for cells successfully co-transfected with Clone 4 (YFP) and modified AID (GFP) and for cells co-transfected with Clone 29 (mCherry) and modified AID (GFP)
</LI><LI> Immunfluorescence: stably transfected CHO Clone 4 cells were fixed in Ibidi Dishes with 4% PFA for 20min and permeabilized with 0,5% Triton X-100 in PBS for 10 min. The wells were blocked with 1% BSA in PBS for 30min. Incubation with primary antibody (Anti-FlagÂ® M1 Monoclonal Antibody from SIGMA) followed over night at 4Â°C. 
</LI><LI> passaging of HEK293 cells and seeding of 10cm dishes for production of virus
</LI><LI> capturing images of transiently transfected HeLa and HEK293 cells (transfected with Clone 4 and 29)(see Results)
</LI><LI> planning of experiments that should be carried out before the 26. September
</LI><LI> change of medium in 96-well plates with CHO clone 4 cells (+ 550Âµg/ml Hygromycin)
</LI><LI> collecting supernatant of CHO cells co-transfected with Clone 29 and Cre Recombinase
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of ligated psB1C3-gene 2, 3 and -nanobody constructs </P><P><B>Investigators:</B> Maria, Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from  psB1C3-gene2, 3 and - nanobodyconstructs
</LI><LI> Geneious
</LI></UL><P><B>Results:</B></P><UL><LI>  all sequencing results show complementarity with theoretical gene constructs created in geneious</LI></UL><H3><SPAN class="mw-headline" id="2012-09-22_2"><P style="background-color: rgb(240, 20, 70);">2012-09-22</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Biobrick glycerol stock, inoculation of o.n.-culture </P><P><B>Investigators:</B> Maria</P><P><B>Materials:</B></P><UL><LI> 12x 15 ml inoculation tubes
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Chloramphenicol
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 12x glycerol stock of Biobrick gene1,4,5,6 scFv only, scFv construct
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Chloramphenicol
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of Biobrick gene 2 and 3 into new XL1-blue competent <I>E. coli</I> cells</P><P><B>Investigators:</B> Maria</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Chloramphenicol, 37 Â°C thermo mixer, centrifuge,</LI></UL><UL><LI> Biobrick gene 2 and 3
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> new competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> 20Âµl of resuspended cell-suspension were plated on a LB-CM-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones for inoculation of o.n culture
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> continuation of Immunfluorescence: cells were washed three times with PBS/PBS-Triton and then incubated with secondary antibody (Pacific Blueâ¢ goat anti-mouse IgG (H+L) from <I>life technologies</I>) for two hours. After further washing steps pictures were taken with the fluorescence microscope. (see Results)
</LI><LI> planning of Western Blot for detection of scFv and Nanobody in the cells
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-23"><P style="background-color: rgb(240, 20, 70);">2012-09-23</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Mini Prep of Biobrick gene 1, 4, 5, 6, scFv only, scFv construct </P><P><B>Investigator:</B>Maria</P><P><B>Materials: </B>overnight cultures Biobrick gene 1, 4, 5, 6, scFv only, scFv construct, Mini prep kit </P><P><B>Methods: </B>according to manual </P><P><B>Results: </B></P><P>scFv only </P><P>Gene 1 a: 355,1 ng/Âµl</P><P>Gene 1 b: 351,9 ng/Âµl&lt;br&lt;</P><P>Gene 4 a: 313,9 ng/Âµl</P><P>Gene 4 b: 340,5 ng/Âµl&lt;br&lt;</P><P>Gene 5 a: 346,5 ng/Âµl</P><P>Gene 5 b: 354,8 ng/Âµl&lt;br&lt;</P><P>Gene 6 a: 426,1 ng/Âµl</P><P>Gene 6 b: 478,0 ng/Âµl&lt;br&lt;</P><P>Gene scFv only a: 360,5 ng/Âµl</P><P>Gene scFv only b: 303,4 ng/Âµl&lt;br&lt;</P><P>Gene scFv construct a: 435,3 ng/Âµl</P><P>Gene scFv construct b: 250,6 ng/Âµl&lt;br&lt;</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of HT1080
</LI><LI> dilution of stably transfected cells (Nanobody construct clone 29, originally from transfected Well 3) to 1-2 cells per 150Âµl and seeding in 4x 96-Well plates
</LI><LI> seeding of stably transfected CHO Clone 29 cells in Ibidi Dish 
</LI><LI> change of medium in 75cmÂ² flasks with stably transfected cells (Clone 4 and 29)
</LI><LI> harvest cells for Western Blot (CHO not transfected, CHO stably transfected Clone 4 and 29) 
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> planning primer for integration of new RAGE-transmembrane domain and RAGE-signalpeptide </P><P><B>Investigator:</B> Sascha </P><P><B>Materials and Methods:</B></P><UL><LI> Geneious 
</LI><LI> paper</LI></UL><H3><SPAN class="mw-headline" id="2012-09-24"><P style="background-color: rgb(240, 20, 70);">2012-09-24</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture </P><P><B>Investigator:</B>Kerstin<B>Topic: </B>cell-culture </P><UL><LI> passaging of CHO cells w/ zeocin and w/o zeocin
</LI><LI> seeding CHO cells in Ibidi Dish as control for immunfluorescence experiment with stably transfected Clone 29 cells (planned for 25.09.12)
</LI><LI> passaging HEK 293
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Primer design and ordering</P><P><B>Investigator:</B> Sascha </P><P><B>Materials and Methods:</B> Geneious </P><P><B>Results:</B></P><UL><LI> primerI-fw:rage-signal, xbaI,
</LI><LI>GCCTCTAGAGCTAGCATGGCAGCCGGAACAGCAGTTGGAGCCTGGGTGCTGGTCCTCAGTCTGTGGGGGGCAGTAGTAGGTGCTGACTAC
</LI></UL><P>AAAGACGAAGTGCAATTGCAG
</P><UL><LI> primerII-rv:c-terminus scfv-te
</LI><LI>GGGCCAGGGCGAGGGTGCCGCTTCCGCCGCCACCGCTTCCCCCTTGAAAATATAAATTCTCTCCAGATCCCCGTTTGATCT
</LI></UL><LI> primerIII-fw:n-terminus of RAGE
</LI><LI>ACCCTCGCCCTGGCCCTGGGAATCTTGGGCGGCCTGGGGACCGCTGCCCTCTTG
</LI><LI> primerIV-rv:c-terminus of RAGE
</LI><LI>CCTTCTCTGCCACAGGATCACGCCGATCAAGAGGGCAGCGGTCCC
</LI><LI> primerV-fw:nterminus of eyfp w
</LI><LI>CGTGATCCTGTGGCAGAGAAGGTCTGGAGTGAGCAAGGGCG
</LI><LI> primerVI: c-terminus of eyfp w
</LI><LI>GCGGCCGCTACTAGTAGGGCCCTTACTTGTACAGCTCGTC
</LI><H3><SPAN class="mw-headline" id="2012-09-25"><P style="background-color: rgb(240, 20, 70);">2012-09-25</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">  SDS-PAGE of sonified CHO cells transfected with pcDNA5FRT geneart-nanobody- and scFv-TMD-EYFP constructs</P><P><B>Investigator:</B>Sascha<B>Materials:</B></P><UL><LI> TEV-protease
</LI><LI> Rotiload SDS-laoding buffer
</LI><LI> heatblock
</LI><LI> SDS-PAGE equipment, TEMED, APS
</LI></UL><P><B>Method:</B></P><UL><LI>SDS-PAGE Laemli-protocol
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-26"><P style="background-color: rgb(240, 20, 70);">2012-09-26</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Western Blot (of  SDS-PAGEs) of sonified CHO cells transfected with pcDNA5FRT geneart-nanobody- and scFv-TMD-EYFP constructs</P><P><B>Investigator:</B>Sascha<B>Materials:</B></P><UL><LI> Ponceau-solution
</LI><LI> Whatman paper* PVDF membranes
</LI><LI> Blotting buffer
</LI><LI> Rotiblock
</LI><LI> TBS, TBS+T
</LI><LI> semi-dry plot equipment
</LI><LI> primary antibodies: anti-Flag-tag mouse antibody to detect scFv/scFv construct; anti-human Fc-antibody labeled with biotin
</LI><LI> secondary antibodies: antimouse antibody labeleld with HRP to detect anti-Flag-antibody;streptavidin-HRP conjugat to detect biotinylated anti-Fc-antibody  
</LI><LI> ECL(SuperSignal West Pico Trial Kit); Thermo Scientific)
</LI></UL><P><B>Method:</B></P><UL><LI> Western Blot, semi-dry
</LI></UL><P><B>Results:</B></P><UL><LI> scFv â after Ponceau staining:
</LI><LI>nanobody â after Ponceau staining:
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Primer design and ordering</P><P><B>Investigator:</B> Sascha </P><P><B>Materials and Methods:</B> Geneious </P><P><B>Results:</B></P><UL><LI> new_primerVI: c-terminus of ey
</LI><LI> GAGCTGCAGCGGCCGCTACTAGTAGGGCCCTTACTTGTACAGCTCGTC</LI></UL><H3><SPAN class="mw-headline" id="2012-09-28"><P style="background-color: rgb(240, 20, 70);"> 2012-09-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">PCR:  integration of new RAGE-transmembrane domain and RAGE-signalpeptide into scFv-Tmd-EYFP-construct </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Template: scFv-only Biobrick
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF</LI><LI> dNTPs (10mM)
</LI><LI> primerI-fw:rage-signal, xbaI, primerII-rv:c-terminus scfv-te
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>3x master mix </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HF buffer</TD><TD>30</TD></TR><TR><TD>dNTPs</TD><TD>3</TD></TR><TR><TD> primerI-fw:rage-signal td&gt;</TD><TD>7,5</TD></TR><TR><TD> primerII-rv:c-terminus scfv-te </TD><TD>7,5</TD></TR><TR><TD> template scFv-only Biobrick (5ng/Âµl)</TD><TD> 3</TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1,5 </TD></TR><TR><TD> water </TD><TD> 97,5</TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 15</TD></TR><TR><TD> annealing </TD><TD> gradient: 60;65;70 </TD><TD> 12 </TD><TD> 15 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 14 </TD><TD> 15 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 15</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 20 </TD><TD> 15 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight:bold;">PCR:  amplification of EYFP with overhang of new RAGE-TMD</P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> Template: EYFP_Bba_E0030
</LI><LI> Phusion-polymerase 
</LI><LI> 10x Phusion buffer HF</LI><LI> dNTPs (10mM)
</LI><LI> primerV-fw:nterminus of eyfp w; primerVI: c-terminus of eyfp w
</LI><LI> Thermocycler</LI></UL><P><B>Methods:</B><B>3x master mix </B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>10x Phusion HF buffer</TD><TD>30</TD></TR><TR><TD>dNTPs</TD><TD>3</TD></TR><TR><TD> primerV-fw:nterminus of eyfp w td&gt;</TD><TD>7,5</TD></TR><TR><TD> primerVI: c-terminus of eyfp w </TD><TD>7,5</TD></TR><TR><TD> template EYFP (20ng/Âµl)</TD><TD> 3</TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1,5 </TD></TR><TR><TD> water </TD><TD> 97,5</TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD>30</TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 8 </TD><TD> 30</TD></TR><TR><TD> annealing </TD><TD> gradient: 60;65;69 </TD><TD> 13 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 14 </TD><TD> 30</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: analytical gelelectrophoresis of amplified eYFP with RAGE-TMD and RAGE-signalpeptide overhangs</P><P><B>Investigator:</B> Sascha<B>Materials:</B></P><UL><LI> agarose
</LI><LI> 1xTAE-buffer
</LI><LI> 10xFD Green Buffer </LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI><LI> 105V
</LI></UL><P><B>Results:</B></P><UL><LI> only amplified eYFP showed sufficient bp size</LI></UL><P><B>Further Tasks:</B></P><UL><LI> modifying PCR
</LI><LI> new primer design for advanced PCR
</LI></UL><H2><SPAN class="mw-headline" id="Virus">Virus</SPAN></H2><H3><SPAN class="mw-headline" id="2012-09-02"><P style="background-color: rgb(240, 20, 70);"> 2012-09-02</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparing viral stocks</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> preparation of the primary virus stock</P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 29.08.2012 (with following plasmids: p01, p06, phelper, psb1c3)
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath </LI></UL><P><B>Method:</B></P><UL><LI> transfer transfected cells (incl. DMEM) to a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 15 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 <B>g</B>, RT)
</LI></UL><UL><LI> supernatant = virus stock
</LI></UL><UL><LI> sore at -80 Â°C </LI></UL><P><B>Results:</B></P><UL><LI> virus stock â with YFP on the surface and GOI = CFP </LI></UL><P><B>Further tasks</B></P><UL><LI> infection
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-03_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: picking clones of p10_Psb1c3_CMV_kozak_sortase_myc and p10_Psb1c3_CMV_kozak_sortase </P><P><B>Investigator:</B> Xenia</P><P><B>Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> ampicillin-stock solution (100 mg/mL)
</LI></UL><UL><LI> plates with <I>E. coli</I> XL1 blue with plasmids: p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase</LI></UL><P><B>Method:</B></P><UL><LI> picking clones with p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase
</LI></UL><UL><LI> over night cultur (50 mL LB medium; 37 Â°C; 300 rpm; 17 hours)</LI></UL><P><B>Further tasks:</B></P><UL><LI> endotoxin free midiprep
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: transfection </P><P><B>Investigators:</B> Kerstin and Stefan </P><P><B>Aim:</B></P><UL><LI>The plasmids were transfected in HEK-cells with the following plasmids:
</LI></UL><UL><LI> pSB1C: GOI = CFP (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> P01: YFP linked on VP1 (1875 ng)
</LI></UL><UL><LI> p06: VP1 knock out (1875 ng)</LI></UL><P><B> Materials:</B></P><UL><LI> Medium: opti Mem
</LI></UL><UL><LI> transfection reagent: PEI
</LI></UL><UL><LI> 15 Âµg DNA per each transfection
</LI></UL><P><B> Methods:</B></P><UL><LI> transfection preparation
</LI></UL><UL><LI> 1875 ng per plasmid; add 200 ÂµL opti MEM per each transfection
</LI></UL><UL><LI> 4.8 * DNA --&gt; 72 ÂµL PEI add 200 ÂµL per each transfection
</LI></UL><UL><LI> three times 10 s mixing
</LI></UL><UL><LI> 2 min incubation at RT
</LI></UL><UL><LI> centrifagation
</LI></UL><UL><LI> add the PEI approach to the DNA approach
</LI></UL><UL><LI> mixing
</LI></UL><UL><LI> 15 min incubation at RT
</LI></UL><UL><LI> add the transfection approach to the HEK AAV 293 cells
</LI></UL><P><B>Further tasks</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation, preparation of the primary virus stocks
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-04_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-04</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: exchange medium</P><P><B> Investigator: </B> Kathi </P><P><B> Materials</B>:
</P><UL><LI> DMEM
</LI></UL><UL><LI> PBS
</LI></UL><P><B> Methods: </B></P><UL><LI> asprirate the old medium
</LI></UL><UL><LI> wash with PBS
</LI></UL><UL><LI> add new DMEM on the cells
</LI></UL><P><B> Further tasks:</B></P><UL><LI> virus preparation on 06.09.2012 (three days virus maturation)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparation of the plasmids</P><P><B>Investigators:</B> Kathi </P><P><B>Aim:</B> endotoxin free preparation of p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase </P><P><B>Materials and Methods:</B></P><UL><LI> endotoxinfree preparation using the Plasmid plus medi kit (QIAGEN)
</LI></UL><UL><LI> the endotoxic free preparations was tested by disgestion and agarose gel electrophoresis
</LI></UL><UL><LI>digestion: 5 ÂµL DNA + 2 ÂµL FD green buffer + 1 ÂµL of each enzym (XbaI and PstI) and 23 ÂµL water (37 Â°C and 60 min)
</LI></UL><UL><LI> gel electrophoresis: 1% agarose gel, 100 V, 60 min
</LI></UL><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> p10_PsB1c_CMV_kozak_sortase_myc: c= 314,6 ng/ÂµL v=~100 ÂµL
</LI></UL><UL><LI> p10_PsB1c_CMV_kozak_sortase: c= 334,2 ng/ÂµL v=~100 ÂµL
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><UL><LI> the plasmid is wrong â wrong sizes of the plasmids fragments
</LI></UL><P><B> further tasks</B></P><UL><LI> new preparation of the plasmids
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Infection </P><P><B>Investigators:</B> Kathi </P><P><B>Aim:</B> infection with the virus with YFP on the surface and CFP = GOI; Infected CHO-cells </P><P><B>Materials: </B></P><UL><LI> Virus-stock (prepared on 02.09.2012)
</LI></UL><UL><LI> CHO-cells
</LI></UL><UL><LI> opti HEM
</LI></UL><P><B> Methods:</B></P><UL><LI> the virus stock was diluted with opti MEM (1:2; 1:10)
</LI></UL><UL><LI> the CHO cells was infected with the same volume of virus solution with the following dilution: 1:1; 1;2; 1;10
</LI></UL><UL><LI> negative control: without virus stock
</LI></UL><P><B> Further tasks</B></P><UL><LI> microscopy of the infected CHO-cells after 24 and 48 hours
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: picking clones of p10_Psb1c3_CMV_kozak_sortase_myc and p10_Psb1c3_CMV_kozak_sortase </P><P><B>Investigator:</B> Xenia</P><P><B>Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> ampicillin 100 mg/mL stock solution</LI></UL><P>plates with <I>E. coli</I> XL1 blue with plasmids: p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase&lt;b
</P><P><B>Method:</B></P><UL><LI> picking clones with p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase
</LI></UL><UL><LI> over night culture (50 mL LB medium; 37 Â°C; 300 rpm; 17 hours)</LI></UL><P><B>Further tasks:</B></P><UL><LI> endotoxin free midiprep
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-05_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-05</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Planing vector plasmid with neomycin cassete</P><P><B>Investigators:</B> Xenia</P><P><B>Aim: </B> planing how to digest and ligate the vectors for the vector plasmid with neomycin cassette </P><P><B>Material and Methods:</B> Geneious </P><P><B>Results: </B></P><UL><LI> pSB1C3 with lITR: cut with SpeI and PstI
</LI></UL><UL><LI> pcDNA3(+) with neomycin cassette: cut with XbaI and PstI
</LI></UL><UL><LI> pSB1C3 with lITR_CMV_neomycin_hGH: cut with SpeI and PstI
</LI></UL><UL><LI> pSB1C3 with rITR: cut with XbaI and PstI
</LI></UL><P><B>Further tasks:</B></P><UL><LI> design and ordering primer
</LI></UL><UL><LI> practical part:
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparation of the plasmids</P><P><B>Investigator:</B> Kathi </P><P><B>Aim:</B> endotoxin free preparation of p10_psb1c3_cmv_kozak_sortase_myc and p10_psb1c3_cmv_kozak_sortase </P><P><B>materials and methods:</B></P><UL><LI> endotoxinfree preparation using the medi plasmid kit (QIAGEN)
</LI></UL><UL><LI> the endotoxinfree preparations was tested by digestion and agarose gel electrophoresis
</LI></UL><UL><LI>digestion: 10 ÂµL DNA (1:10 diluted) + 2 ÂµL Green buffer + 1 ÂµL of each enzym (XbaI and PstI) and 17 ÂµL water (37 Â°C and 60 min)
</LI></UL><UL><LI> gel electrophoresis: 1% agarose gel, 100 V, 60 min
</LI></UL><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> p10_PsB1c_CMV_kozak_sortase_myc: c= 1462 ng/ÂµL v=~200 ÂµL
</LI></UL><UL><LI> p10_PsB1c_CMV_kozak_sortase: c= 1350 ng/ÂµL v=~200 ÂµL
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><UL><LI> the plasmid is wrong â wrong sizes of the plasmids fragments
</LI></UL><UL><LI> the wrong antibiotic was used in the over night culture
</LI></UL><P><B> Further tasks</B></P><UL><LI> retransformation of the right p10_psb1c_CMV kozak_sortase_myc and
</LI></UL><P>p10_psb1c_CMV kozak_sortase plasmids
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: transfection </P><P><B>Investigator:</B> Kathi </P><P><B>Aim:</B></P><UL><LI>The plasmids were transfected in HEK-cells with the following plasmids:
</LI></UL><UL><LI> pSB1C: GOI is CFP (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> P10_PsB1c3_CMV_kozak_sortase_myc (1875 ng)
</LI></UL><UL><LI> p06: VP2 knock out (1875 ng)</LI></UL><UL><LI>The plasmids were transfected in HEK-cells with the following plasmids:
</LI></UL><UL><LI> pSB1C: GOI is CFP (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> P10_PsB1c3_CMV_kozak_sortas (1875 ng)
</LI></UL><UL><LI> p06: VP2 knockut 1875 ng)</LI></UL><P><B> Materials: </B></P><UL><LI> PEI (transfection reagent)
</LI></UL><UL><LI> opti MEM (transfection medium)
</LI></UL><UL><LI> endotoxin free preparated plasmids
</LI></UL><P><B>Further tasks</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation â preparation of the virus
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-06_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Primer design and ordering</P><P><B>Investigator:</B> Xenia </P><P><B>Materials and Methods:</B> Geneious </P><P><B>Results:</B></P><UL><LI> Primer (forward) with XbaI recognition site:
</LI></UL><UL><LI>GTCTAGACACGGAATGTGTGTCAGTTAGGGTGTGG
</LI></UL><UL><LI>Primer (reverse, complement) with SpeI and PstI recognition site:
</LI></UL><UL><LI>TACTGCAGCCCCTACTAGTATCGGGCAGACATGATAAGATACATTGATGAGTTTGG
</LI></UL><P>Annealing temp.: 71 Â°C</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: exchange medium</P><P><B> Investigator: </B> Kathi </P><P><B> Materials</B>:
</P><UL><LI> DMEM
</LI></UL><UL><LI> PBS
</LI></UL><P><B> Methods: </B></P><UL><LI> asprirate the old medium
</LI></UL><UL><LI> wash with PBS
</LI></UL><UL><LI> add new DMEM to the cells
</LI></UL><P><B> Further tasks:</B></P><UL><LI> virus preparation on 09.09.2012 (four days virus maturation)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparing viral stocks</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> virus preparation; virus constuct: YFP on the surface and GOI=CFP </P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 03.09.2012 (with following plasmids: p01, p06, phelper, psb1c3)
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath </LI></UL><P><B>Method:</B></P><UL><LI> transfer transfected cells (incl. DMEM) to a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 10 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 <B>g</B>, RT)
</LI></UL><UL><LI> supernatant = virus stock
</LI></UL><UL><LI> sore at -80 Â°C </LI></UL><P><B>Results:</B></P><UL><LI> virus stock â with YFP on the surface and GOI = CFP </LI></UL><P><B>Further tasks</B></P><UL><LI> infection of the CHO-cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: microscopy</P><P><B>Investigators: </B> Stefan and Kerstin </P><P>&lt;b&gt; Aim:  virus preparation </P><P><B>Materials and methods:</B></P><UL><LI> microscopy of the infeced CHO cells (02.09.2012)</LI></UL><P><B>Results:</B></P><UL><LI> there are yellow fluorescence particle visible
</LI></UL><UL><LI> after bleaching there arnÂ´t yellow fluorescence particle vibible
</LI></UL><P><B>Further tasks</B></P><UL><LI> another infection with a positive control (infected HT1080 cells)
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-09_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-09</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparing viral stocks</P><P><B>Investigator: </B> Kathi </P><P><B>Aim: </B> virus preparation </P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 05.09.2012 (with following plasmids: p10_PsB1c_CMV_kozak_sortase_myc, VP2 knock out, phelper, PsB1c3GOI=CFP and p10_PsB1c_CMV_kozak_sortase, VP2 knock out, phelper, PsB1c3GOI=CFP)
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath </LI></UL><P><B>Method:</B></P><UL><LI> transfer transfected cells (incl. DMEM) to a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 15 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 <B>g</B>, RT)
</LI></UL><UL><LI> supernatant = virus stock
</LI></UL><UL><LI> sore at -80 Â°C </LI></UL><P><B>Results:</B></P><UL><LI> virus stock â surface: sortase + myc-tag; GOI= CFPvirus stock â surface: sortase; GOI= CFP</LI></UL><UL><LI> virus stock â surface: sortase; GOI= CFPvirus stock â surface: sortase; GOI= CFP</LI></UL><P><B>Further tasks</B></P><UL><LI> infection
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-10_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: re-transformation </P><P><B>Investigator:</B> Kathi</P><P><B>Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> competent cells (XL1-blue cells)</LI></UL><P><B>Method:</B></P><UL><LI> transformation via manual, 1 ÂµL of the sample (PsB1c3_CMV_kozak_sortase_myc and PsB1c3_CMV_kozak_sortase prepared on 17.08.2012) were used
</LI></UL><P><B>Further tasks:</B></P><UL><LI> picking clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-11_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-11</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparation of the plasmids</P><P><B>Investigator:</B> Kathi </P><P><B>Aim:</B>endotoxinfree preparation of the following plasmids: GOI=CFP, CFP linked on VP2 and GOI=mVenus </P><P><B>Materials and Methods:</B></P><UL><LI> endotoxinfree preparation using the plasmid plus medi kit (QIAGEN)
</LI></UL><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> GOI=CFP â c= 1159 ng/ÂµL; v= 200 ÂµL
</LI></UL><UL><LI> CFP linked on VP2 â c= 776 ng/ÂµL; v= 200 ÂµL
</LI></UL><UL><LI> GOI=mVenus â c= 532 ng/ÂµL; v= 200 ÂµL
</LI></UL><P><B> Further tasks</B></P><UL><LI> transfection with GOI= mVenus and CFP linked on VP2
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR </P><P><B>Investigator:</B> Xenia </P><P><B>Aim:</B> PCR-amplificate: Neomycin cassette with restriction sites XbaI as prefix and SpeI and PstI as suffix
</P><P><B> Materials: </B></P><UL><LI>PCR
</LI></UL><UL><LI>Vektor with antibiotic resistance: pcDNA3(+)
</LI></UL><UL><LI>Primer:
</LI></UL><UL><LI>r_primer2_ neomycin_cassette-spe_pst (reversed):GTCTAGACACGGAATGTGTGTCAGTTAGGGTGTGG
</LI></UL><UL><LI>f_primer_neomycin_cassette_xbaI_fertig:TACTGCAGCCCCTACTAGTATCGGGCAGACATGATAAGATACATTGATGAGTTTGG
</LI></UL><UL><LI>Phusion Polymerase, dNTPs, HF buffer
</LI></UL><UL><LI>Gel elektrophorese
</LI></UL><UL><LI>Nucleo Spin Gel and PCR Cleanup (Macherey-nagel)
</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>2</TD></TR><TR><TD>Primer (Reverse)</TD><TD>2</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0.5 </TD></TR><TR><TD> water </TD><TD> 33.5 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><A href="/File:UP12_Xenia_11.09.12.gif" title="File:UP12 Xenia 11.09.12.gif">100px</A></P><P>The concentration of the template was too high (1 Âµg/ÂµL). The digestion couldn't be done because the template contains the same restriction sites as the PCR-amplificate</P><P><B>Further tasks:</B></P><UL><LI> PCR with lower template concentration
</LI></UL><UL><LI> Gel electophorese
</LI></UL><UL><LI> Digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR </P><P><B>Investigator:</B> Xenia </P><P><B>Aim:</B> PCR-amplificate: Neomycin cassette with restriction sites XbaI as prefix and SpeI and PstI as suffix </P><P><B> Materials: </B></P><UL><LI>Vektor with antibiotic resistance: pcDNA3(+)
</LI></UL><UL><LI>Primer:
</LI></UL><UL><LI>r_primer2_ neomycin_cassette-spe_pst (reversed):GTCTAGACACGGAATGTGTGTCAGTTAGGGTGTGG
</LI></UL><UL><LI>f_primer_neomycin_cassette_xbaI_fertig:TACTGCAGCCCCTACTAGTATCGGGCAGACATGATAAGATACATTGATGAGTTTGG
</LI></UL><UL><LI>Phusion Polymerase, dNTPs, HF buffer
</LI></UL><UL><LI>Gel elektrophorese
</LI></UL><UL><LI> NucleoSpin Gel ans PCR Clean-up Kit (macherey-nagel)
</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>2</TD></TR><TR><TD>Primer (Reverse)</TD><TD>2</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0.5 </TD></TR><TR><TD> water </TD><TD> 33.5 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 71 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 8 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI>Gel electophorese
</LI></UL><UL><LI>Digestion
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-12_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-12</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Topic: transfection </P><P><B>Investigator:</B> Kerstin </P><P><B>Aim:</B></P><UL><LI>The plasmids were transfected in HEK-cells with the following plasmids:
</LI></UL><UL><LI> CFP linked on VP2 (1875 ng)
</LI></UL><UL><LI> VP2 knock out (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> PsB1c3-GOI=CFP (1875 ng)</LI></UL><P><B> Materials: </B></P><UL><LI> PEI (transfection reagent)
</LI></UL><UL><LI> opti MEM (transfection medium)
</LI></UL><UL><LI> endotoxin free preparated plasmids
</LI></UL><P><B>Further tasks</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation â preparation of the virus (14.09.2012)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Gelelectrophoresis with PCR-amplificate</P><P><B>Investigator: </B> Xenia </P><P><B> Aim: </B> check whether the PCR worked </P><P><B>Materials:</B></P><UL><LI> Geneious
</LI></UL><UL><LI> agarose
</LI></UL><UL><LI> 1x TAE-buffer
</LI></UL><UL><LI> 10x FD green Buffer
</LI></UL><UL><LI> PCR-sample
</LI></UL><P><B>Method:</B></P><P>Gelelectrophoresis of PCR Neomycin cassette (1533bp) uncut</P><P><B>Results:</B></P><P><A href="/File:UP12_Xenia_Neomycin_12.09.12.gif" title="File:UP12 Xenia Neomycin 12.09.12.gif">100px</A></P><P>No band indicating template is visible. The neomycin cassette will be cut.</P><P><B>further tasks:</B></P><UL><LI> purification with PCR-clean up
</LI></UL><UL><LI> Digestion of neomycin cassette</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Purification of PCR-amplificate with PCR clean-up Kit</P><P><B>Investigator: </B> Xenia </P><P><B> Aim: </B> Purification of PCR-amplificate </P><P><B>Materials:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up Kit (macherey-nagel)
</LI></UL><P><B>Results:</B></P><UL><LI> concentration of amplified neomycin cassette: 106.2 ng/ÂµL</LI></UL><P><B>Further tasks:</B></P><UL><LI> Digestion of amplified neomycin cassette </LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: purification of p10-pSB1C3-CMV-Kozak-Sortase</P><P><B>Investigator: </B> Xenia </P><P><B> Aim: </B> purification of p10-pSB1C3-CMV-Kozak-Sortase </P><P><B>Materials:</B> Plasmid Plus Midi Kit (QIAGEN)</P><P><B>Results:</B></P><P>concentration of plasmid:
</P><UL><LI>p10-pSB1C3-CMV-Kozak-Sortase: 2045.8 ng/ÂµL </LI></UL><P><B>Further tasks:</B></P><UL><LI>transfection
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: purification of left and right ITR plasmids with miniprep</P><P><B>Investigator: </B> Xenia </P><P><B> Aim:</B> purification of left and right ITR plasmids </P><P><B>Materials:</B></P><UL><LI> Geneious
</LI></UL><UL><LI> GeneJET Plasmid Miniprep Kit
</LI></UL><UL><LI> Gel electrophoresis </LI></UL><P><B>Results:</B></P><P>concentration of the plasmids:
</P><UL><LI>left ITR: 174.4 ng/ÂµL
</LI></UL><UL><LI>left ITR: 224.2 ng/ÂµL
</LI></UL><UL><LI>right ITR: 87.2 ng/ÂµL
</LI></UL><UL><LI>right ITR: 75.5 ng/ÂµL
</LI></UL><P>Gel electrophoresis
</P><P>The left_ITR plasmid consists of 2.217 kbp and right ITR consists of 2.216 kbp</P><P><B>Further tasks:</B></P><P>digestion
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B> Xenia and Tobias </P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD></TR><TR><TD>p10-pSB1C3-CMV-Kozak-Sortase</TD><TD> II3673 </TD><TD> p140:Viral Brick 587KO-His rev </TD></TR><TR><TD>p10-pSB1C3-CMV-Kozak-Sortase </TD><TD> II3674 </TD><TD> p37: Primer cap 2800 for </TD></TR><TR><TD>p10-pSB1C3-CMV-Kozak-Sortase </TD><TD> II3675 </TD><TD> p39: Primer cap 3500 for </TD></TR><TR><TD>p10-pSB1C3-CMV-Kozak-Sortase </TD><TD> standard Primer GATC </TD><TD> CMV-Primer </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - resuspend EGFR construct</P><P><B>Investigator: </B> Tobi </P><P><B> Aim:</B> resuspend of the EGFR construct</P><P><B>Materials and Methods:</B></P><UL><LI> resuspend the EGFR in 50 ÂµL water</LI></UL><P><B>Further tasks:</B></P><UL><LI> transformation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - transformation</P><P><B>Investigator: </B> Tobi </P><P><B> Aim:</B> transformation of the EGFR construct</P><P><B>Materials and Methods:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> competent cells (XL1-blue cells)
</LI></UL><UL><LI> transformation via manual, 10 ÂµL of the ligation were used</LI></UL><P><B>Further tasks:</B></P><UL><LI> picking clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-13_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus preparation - transfection</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> transfection of HEK-AAv 293 cells -&gt; virus contruct with kozak_sortase_myc + GOI=CFP and kozak_sortase + GOI=CFP </P><P><B>Materials:</B></P><UL><LI> plasmids:
</LI></UL><UL><LI> p10_psb1c_CMV_kozak_sortase_myc (1875 ng)
</LI></UL><UL><LI> p05 - VP2 knock out (1875 ng)
</LI></UL><UL><LI> GOI = CFP (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> PEI (transfection reagent)
</LI></UL><UL><LI> opti MEM (transfection medium)
</LI></UL><P><B>Methods:</B></P><UL><LI> transfection preparation1875 ng per plasmid
</LI></UL><UL><LI> add 200 ÂµL opti MEM-medium per each transfection
</LI></UL><UL><LI> 4.8 * DNA --&gt; 72 ÂµL PEI add 200 ÂµL per each transfection
</LI></UL><UL><LI> three times 10 s mixing
</LI></UL><UL><LI> 2 min incubation at RT
</LI></UL><UL><LI> centrifagation
</LI></UL><UL><LI> add the PEI preparation to the DNA preparation
</LI></UL><UL><LI> mixing
</LI></UL><UL><LI> 15 min incabation at RT
</LI></UL><UL><LI> add the transfection preparation to the HEK AAV 293 cells
</LI></UL><P><B>Further tasks:</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation â preparation of the virus (16.09.2012)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus preparation - exchange medium</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> change the medium of the transfected HEK cells (transfection date: 12.09.2012) </P><P><B>Materials:</B></P><UL><LI> PBS
</LI></UL><UL><LI> DNEN
</LI></UL><P><B>Methods:</B></P><UL><LI> asprirate the old medium
</LI></UL><UL><LI> wash with PBS
</LI></UL><UL><LI> add new DMEM to the cells
</LI></UL><P><B>Further tasks:</B></P><UL><LI> virus preparation on 17.09.2012 (three days virus maturation)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: p10_psb1c_CMV_kozak_sortase - +myc and - myc - Digestion</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> test digestion </P><P><B>Materials:</B></P><UL><LI> plasmids:
</LI></UL><UL><LI> p10_psb1c_CMC_kozak_sortase_myc
</LI></UL><UL><LI> p10_psb1c_CMC_kozak_sortase
</LI></UL><UL><LI> enzymes: XbaI and PstI
</LI></UL><UL><LI> 10 x FD green buffer
</LI></UL><P><B>Methods:</B></P><UL><LI> Digestion approache:
</LI></UL><UL><LI> 5 ÂµL DNA (1:100 dilute) + 1 ÂµL of each enzyme + 3 ÂµL 10 x FD green buffer + 21 ÂµL water
</LI></UL><UL><LI> 1 hour at 37 Â°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: p10_psb1c_CMV_kozak_sortase - +myc and - myc - gel-electrophoresis</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> control of the test digest </P><P><B>Materials:</B></P><UL><LI> 1% agarose gel
</LI></UL><UL><LI> 1 x TAE buffer
</LI></UL><P><B>Methods:</B></P><UL><LI> run: 60 min at 100 V
</LI></UL><P><B>Further tasks:</B></P><UL><LI> sequencing
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - picking clones</P><P><B>Investigator: </B> Tobias </P><P><B> Aim: </B> picking two clones </P><P><B>Materials and Mtehods:</B></P><UL><LI> picking two clones and incubate over night in LB medium (incl. Amp)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> DNA preparation
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-14_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Sequencing</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> send the re-tranfected p10_psb1c3_CMV_kozak_sortase_myc and p10_psb1c3_CMV_kozak_sortase for sequncing</P><P><B>Materials:</B></P><UL><LI> Primer
</LI></UL><UL><LI> concentration and volum of the DNA: c= 50 ng/ÂµL; v= 100 ÂµL
</LI></UL><P><B>Methods:</B> GATC </P><P><B>Further tasks:</B>Geneious analysis </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: exchange medium </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> change the medium of the transfected HEK cells (transfection date: 13.09.2012)</P><P><B>Materials:</B></P><UL><LI> PBS
</LI></UL><UL><LI> DMEM
</LI></UL><P><B>Methods:</B></P><UL><LI> asprirate the old medium
</LI></UL><UL><LI> wash with PBS
</LI></UL><UL><LI> give new DMEM on the cells
</LI></UL><P><B>Further tasks:</B></P>
virus preparation on 16.09.2012 (three days virus maturation) <P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: infection</P><P><B>Investigator: </B> Kerstin </P><P><B> Aim: </B> infection of the HT1080, CHO-cells and CHO-cells with nanobody</P><P><B>Materials and Methods:</B></P><UL><LI> the virus was added to the cells
</LI></UL><P><B>Further tasks:</B></P><UL><LI> aspriration of the medium (inkl. virus) and add of new virus-medium mix
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: transfection</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> transfection of HEK-AAv 293 cells -&gt; virus contruct with CFP on the surface and GOI= mVenus</P><P><B>Materials </B></P><UL><LI> plasmids:
</LI></UL><UL><LI> CFP linked on VP2 (1875 ng)
</LI></UL><UL><LI> VP2 knock out (1875 ng)
</LI></UL><UL><LI> GOI = mVenus (1875 ng)
</LI></UL><UL><LI> phelper (1875 ng)
</LI></UL><UL><LI> PEI (transfection reagent)
</LI></UL><UL><LI> opti MEM (transfection medium)
</LI></UL><P><B>Methods:</B></P><UL><LI> transfection preparation1875 ng per plasmid; add 200 ÂµL opti MEM-medium per each transfection
</LI></UL><UL><LI> 4.8 * DNA --&gt; 72 ÂµL PEI add 200 ÂµL per each transfection
</LI></UL><UL><LI> three times 10 s mixing
</LI></UL><UL><LI> 2 min incubation at RT
</LI></UL><UL><LI> centrifagation
</LI></UL><UL><LI> add the PEI preparation to the DNA preparation
</LI></UL><UL><LI> mixing
</LI></UL><UL><LI> 15 min incabation at RT Ã¤
</LI></UL><UL><LI> add the transfection preparation to the HEK AAV 293 cells
</LI></UL><P><B>Further tasks:</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation â preparation of the virus (17.09.2012)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - DNA preparation</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> preparation of the EGFR over night culture </P><P><B>Materials and Methods:</B></P><UL><LI> over night culture
</LI></UL><UL><LI> GeneJET Plasmid Miniprep Kit
</LI></UL><P><B>Results:</B></P><UL><LI> concentration
</LI></UL><UL><LI> EGFR - clon 1: c= 321 ng/ÂµL v= 50 ÂµL
</LI></UL><UL><LI> EGFR - clon 2: c= 383 ng/ÂµL v= 50 ÂµL
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic:EGFR - preparatively digestion </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> digest the EGFR construct with XbaI and AgeI for the BioBrick and with XbaI and PstI for the arabinose plasmid</P><P><B>Materials:</B></P><UL><LI> 10 x FD green buffer
</LI></UL><UL><LI> DNA (preped with JetGen-Kit)
</LI></UL><UL><LI> XbaI, AgeI and PstI
</LI></UL><P><B>Methods:</B></P><UL><LI> EGFR-construct clon 1:
</LI></UL><UL><LI> degestion approach (BioBrick): 6,2 ÂµL DNA (~ 2 Âµg DNA) + 2 ÂµL of each enzyme (XbaI and AgeI) + 3 ÂµL FD green buffer + 18,8 ÂµL water
</LI></UL><UL><LI> degestion approach (Arabinose): 6,2 ÂµL DNA (~ 2 Âµg DNA) + 2 ÂµL of each enzyme (XbaI and PstI) + 3 ÂµL FD green buffer + 18,8 ÂµL water
</LI></UL><UL><LI> EGFR-construct clon 2:
</LI></UL><UL><LI> degestion approach (BioBrick): 5,2 ÂµL DNA (~ 2 Âµg DNA) + 2 ÂµL of each enzyme (XbaI and AgeI) + 3 ÂµL FD green buffer + 19,8 ÂµL water
</LI></UL><UL><LI> degestion approach (Arabinose): 5,2 ÂµL DNA (~ 2 Âµg DNA) + 2 ÂµL of each enzyme (XbaI and PstI) + 3 ÂµL FD green buffer + 19,8 ÂµL water
</LI></UL><UL><LI> 2,5 hours; 37 Â°C
</LI></UL><P><B>Further tasks:</B></P><P>gel electrophoresis
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - gel-electrophoresis </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> seperation of the fragments </P><P><B>Materials and Methods:</B></P><UL><LI> 1% agarose
</LI></UL><UL><LI> 1 x TBE buffer
</LI></UL><UL><LI> 90 min at 95 V
</LI></UL><P><B>Results</B>:
</P><P><B>Further tasks:</B></P><P>gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - ligation </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> ligate the EGFR construct into the BioBrick plasmid (Backbone) and the arabinose plasmid</P><P><B>Materials:</B></P><UL><LI> T4 ligase
</LI></UL><UL><LI> T4 ligase buffer
</LI></UL><P><B>Methods:</B></P><UL><LI> ligase approach:
</LI></UL><UL><LI> EGFR-BioBrick:
</LI></UL><UL><LI> 2 ÂµL Ta ligase buffer + 1 ÂµL ligase + 1.2 ÂµL psb1cplasmid (backbone) + 6.9 ÂµL EGFR construckt (digest with XbaI, AgeI)
</LI></UL><UL><LI> EGFR-agabinose:
</LI></UL><UL><LI> 2 ÂµL Ta ligase buffer + 1 ÂµL ligase + 3.7 ÂµL arabinose plasmid (backbone) + 3.3 ÂµL EGFR construckt (digest with XbaI, PstI)
</LI></UL><P><B>Further tasks:</B></P><P>transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - transformation </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> transformation - EGFR construct/BioBrick and EGFR/arabinose </P><P><B>Materials:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> competent cells (XL1-blue cells)
</LI></UL><UL><LI> antibiotics
</LI></UL><UL><LI> arabinose backbone - Amp
</LI></UL><UL><LI> BioBrick backbone - CM
</LI></UL><P><B>Methods:</B></P><UL><LI> transformation via manual, 10 ÂµL of the ligation were used
</LI></UL><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: lITR-antibiotic-rITR - picking clones </P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> picking clones - over nicht clutute </P><P><B>Materials and Methods:</B></P><UL><LI> LB-medium with CM
</LI></UL><P><B>Further tasks:</B></P><P>DNA peparation
</P><H3><SPAN class="mw-headline" id="2012-09-15_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus-construct - Virus preparation</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> preparing the primary virus stock --&gt; CFP on the surface anf GOI = mVenus</P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 14.09.2012
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath
</LI></UL><P><B>Methods:</B></P><UL><LI> transfer transfected cells (incl. DMEM) into a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 15 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 g, RT)
</LI></UL><UL><LI> supernatant = virus stocksore at -80 Â°C
</LI></UL><P><B>Results:</B></P><UL><LI> primary virus stock with CFP on the surface and mVenus = GOI
</LI></UL><P><B>Further tasks:</B></P><UL><LI> infection
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: exchange medium</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> Change the medium of the transfected HEK cells (transfection date: 14.09.2012) </P><P><B>Materials:</B></P><UL><LI> PBS
</LI></UL><UL><LI> DMEM
</LI></UL><P><B>Methods:</B></P><UL><LI> asprirate the old medium
</LI></UL><UL><LI> wash with PBS
</LI></UL><UL><LI> add new DMEM to the cells
</LI></UL><P><B>Further tasks:</B></P><UL><LI> virus preparation on 17.09.2012 (three days virus maturation)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - picking clones</P><P><B>Investigator: </B> kathi</P><P><B> Aim: </B> picking clones - the EGFR contruct into the arabinose and psb1c plasmid </P><P><B>Materials and Methods:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> antibiotics
</LI></UL><UL><LI> arabinose plasmid - Amp
</LI></UL><UL><LI> psb1c plasmid - CM
</LI></UL><UL><LI> over night incubation - shaking - 37 Â°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> arabinose plasmid - mainculter - start expression
</LI></UL><UL><LI> psb1c plasmid - preparation of the DNA
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ITR: preparation</P><P><B>Investigator: </B> kathi</P><P><B> Aim: </B> preparation of the ITR-neo over night cultur</P><P><B>Materials and Methods:</B></P><UL><LI> GeneJET Plasmid Miniprep Kit
</LI></UL><P><B> Results:</B></P><UL><LI> concentration
</LI></UL><UL><LI> clone 1: c= 335 ng/ÂµL v= 50 ÂµL
</LI></UL><UL><LI> clone 2: c= 325 ng/ÂµL v= 50 ÂµL
</LI></UL><P><B>Further tasks:</B></P><P>test digestion
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ITR: Digestion</P><P><B>Investigator: </B> kathi</P><P><B> Aim: </B>Digestion of the prapared plasmids (leftITR+neo)</P><P><B>Materials:</B></P><UL><LI> 10 x FD green buffer
</LI></UL><UL><LI> DNA (preped with JetGen-Kit)
</LI></UL><UL><LI> SpeI and PstI
</LI></UL><P><B> Methods:</B></P><UL><LI> 2 Âµg DNA + 3 ÂµL FD green buffer + 2 ÂµL of each enzyme + add 30 ÂµL water
</LI></UL><P><B> Results:</B></P><P><B>Further tasks:</B></P><P>new digestion with less DNA - 1000 ng and 500 ng
</P><H3><SPAN class="mw-headline" id="2012-09-16_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus-construct - Virus preparation</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> preparing the primary virus stock --&gt; p10_psb1c3_CMV_kozak_sortase_myc and p10_psb1c3_CMV_kozak_sortase</P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 13.09.2012
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath
</LI></UL><P><B>Methods:</B></P><UL><LI> transfer transfected cells (incl. DMEM) to a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 15 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 g, RT)
</LI></UL><UL><LI> supernatant = virus stock store at -80 Â°C
</LI></UL><P><B>Results:</B></P><UL><LI> primary virus stock
</LI></UL><UL><LI> p10_psb1c3_CMV_kozak_sortase_myc
</LI></UL><UL><LI> p10_psb1c3_CMV_kozak_sortase
</LI></UL><P><B>Further tasks:</B></P><UL><LI> infection
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - preparation (BioBrick)</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> preparation of the EGFR construct </P><P><B>Materials and Methods:</B></P><UL><LI> over night culture
</LI></UL><UL><LI> GeneJET Plasmid Miniprep Kit
</LI></UL><P><B>Results:</B></P><UL><LI> concentration EGFR (construct)- clone 1: c= 285 ng/ÂµL v= 50 ÂµL
</LI></UL><UL><LI> concentration EGFR (construct)- clone 1: c= 385 ng/ÂµL v= 50 ÂµL
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - Digestion (BioBrick)</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> Test Digestion </P><P><B>Materials</B></P><UL><LI> fast digest (FD) enzymes (XbaI and AgeI)
</LI></UL><UL><LI> FD green buffer
</LI></UL><P><B> Methods:</B></P><UL><LI> Digestion: 5 ÂµL (1:10 diluted DNA) + 3 ÂµL FD green buffer + 1 ÂµL of each enzyme add 30 ÂµL water
</LI></UL><P><B>Results:</B></P><UL><LI> the bands seems to be correct
</LI></UL><P><B>Further tasks:</B></P><P>send for sequencing
</P><H3><SPAN class="mw-headline" id="2012-09-17_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus-construct - Virus preparation</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> preparing the primary virus stock --&gt; CFP on the surface and GOI= mVenus</P><P><B>Materials:</B></P><UL><LI> cell-culture --&gt; HEK AAV 293 transfected cells on 14.09.2012
</LI></UL><UL><LI> 80 Â°C freezer
</LI></UL><UL><LI> 37 Â°C water bath
</LI></UL><P><B>Methods:</B></P><UL><LI> transfer transfected cells (incl. DMEM) to a falcon tube
</LI></UL><UL><LI> 4 times: 10 min freeze the falcon tube (-80 Â°C) and 15 min unfreezing (37 Â°C water bath)
</LI></UL><UL><LI> centrifugation (10 min, 10000 g, RT)
</LI></UL><UL><LI> supernatant = virus stock store at -80 Â°C
</LI></UL><P><B>Results:</B></P><UL><LI> primary virus stock
</LI></UL><UL><LI> CFP on the surface and YFP = GOI
</LI></UL><P><B>Further tasks:</B></P><UL><LI> infection
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - expression start</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> start the expression with arabinose</P><P><B>Materials:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> arabinose [10%]
</LI></UL><UL><LI> ampicilin
</LI></UL><P><B>Methods:</B></P><UL><LI> 200 ÂµL over night culture into fresh LB medium (incl. Amp.)v= 150 ÂµL
</LI></UL><UL><LI> incubation till OD 0,4
</LI></UL><UL><LI> start of the expression + 450 ÂµL 10% arabinose solution
</LI></UL><UL><LI> 4 hours incubation 37 Â°C
</LI></UL><UL><LI> the pellet can be store at -20 Â°C
</LI></UL><P><B>Further tasks:</B></P><P>preparation of the EGFR
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: qPCR</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> test the genetic virus titer of the virus with YFP on the surface and GOI = CFP</P><P><B>Materials:</B></P><UL><LI> 2 x QuantiFast SYBR-green (QIAGEN)
</LI><LI> Rnase free water
</LI><LI> RNaseI (NEB)
</LI><LI> CMV primer
</LI><LI> CMV_forward_qPCR: 5' - GGGACTTTCCTACTTGGCA - 3'
</LI><LI> CMV_reverse_qPCR: 5' - GGCGGAGTTGTTACGACA - 3'
</LI></UL><P><B>Methods:</B></P><UL><LI> the stard qPDR protocol were used
</LI></UL><P><B> Results:</B></P><UL><LI> there are no evaluable quantification curves
</LI></UL><P><B> Further tasks: </B></P><UL><LI> repetition of the experiment
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-18_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-18</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ITR-neo Digestion</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> digestion of the ITR-neo contruct</P><P><B>Materials and Methods:</B></P><UL><LI>ITR-neo clone 1
</LI></UL><UL><LI> 3 ÂµL DNA (1000 ng)+ 1,5 ÂµL per enzyme (PstI; SpeI) + 3 ÂµL buffer 4 (NEB) 22 ÂµL water
</LI></UL><UL><LI> 1,5 ÂµL DNA (500 ng) + 1,5 ÂµL per enzyme (PstI; SpeI) + 3 ÂµL buffer 4 (NEB) 23,5 ÂµL water
</LI></UL><UL><LI>ITR-neo clone 2
</LI></UL><UL><LI> 3 ÂµL DNA (1000 ng)+ 1,5 ÂµL per enzyme (PstI; SpeI) + 3 ÂµL buffer 4 (NEB) 22 ÂµL water
</LI></UL><UL><LI> 1,5 ÂµL DNA (500 ng) + 1,5 ÂµL per enzyme (PstI; SpeI) + 3 ÂµL buffer 4 (NEB) 23,5 ÂµL water
</LI></UL><UL><LI> 37 Â°C; 60 min
</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> shorter digestion time
</LI></UL><UL><LI> desing of new primer
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - send for sequencing</P><P><B>Investigator: </B> kathi </P><P><B> Aim: </B> send the EGFR in the psb1c plasmid for sequencing</P><P><B>Materials and Methods:</B></P><UL><LI> send the EGFR clone 1 and clone 2 (c= 50 ng/ÂµL)
</LI></UL><UL><LI> Primer forward and reverse psb1c primer
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: picking clones</P><P><B>Investigator:</B> Xenia</P><P><B>Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol sock solution (25 mg/mL)
</LI></UL><UL><LI> plates with <I>E. coli</I> with plasmids: EGFR </LI></UL><P><B>Method:</B></P><UL><LI> picking clones with EGFR
</LI></UL><UL><LI> over night cultur (50 mL LB medium; 37 Â°C; 300 rpm; 17 hours)</LI></UL><P><B>Further tasks:</B></P><UL><LI> expression of EGFR
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-19_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-19</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - expression start</P><P><B>Investigator: </B> Xenia </P><P><B> Aim: </B> start the expression with arabinose</P><P><B>Materials:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> arabinose [10%]
</LI></UL><UL><LI> ampicilin
</LI></UL><P><B>Methods:</B></P><UL><LI> 25 mL overnight culture into fresh 1 L LB medium (incl. Amp.)v= 1 mL
</LI></UL><UL><LI> incubation till OD 0.4 
</LI></UL><UL><LI> start of the expression: 5 mL 10% arabinose solution (c=0.05Â %)
</LI></UL><UL><LI> 4 hours incubation of clone I by 30 Â°C and of clone II by 37 Â°C.
</LI></UL><UL><LI> the pellet can be stored at -20 Â°C
</LI></UL><P><B>Results:</B></P><UL><LI> clone I was incubated till OD 0.39 and 
</LI><LI> clone II was incubated till OD 0.42
</LI></UL><P><B>Further tasks:</B></P><P>Western blot: the band should appear by 24,8 kDa</P><UL><LI>Gel: 12.5Â %</LI></UL><UL><LI>Sample for western blot: 80 ÂµL Sample (OD ~ 2.5) + 20 ÂµL Roti-Load1 reducing</LI></UL><UL><LI>denaturation 10 min; 95 Â°C
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-20_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - SDS-PAGE</P><P><B>Investigators: </B> Xenia and Kathi </P><P><B> Aim: </B> separation of the EGFR using the SDS PAGE - control of the expresion</P><P><B>Materials:</B></P><UL><LI> 12.5% SDS-gel
</LI></UL><UL><LI> 1 x TBS buffer
</LI></UL><P><B>Methods:</B></P><UL><LI> load 20 ÂµL of the praparted samples to the SDS gel
</LI></UL><UL><LI> 100 V 1.5 hours
</LI></UL><P><B>Further tasks:</B></P><P>Western blot: the band should appear by 24,8 kDa</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - western blot</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> detection of the EGFR</P><P><B>Materials:</B></P><UL><LI> membrane
</LI></UL><UL><LI> TBS-buffer + 5% milk powder
</LI></UL><P><B>Methods:</B></P><UL><LI> semi-dry plot of the SDS-PAGE
</LI></UL><UL><LI> blocking of the memrane over night
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ITR - PCR clean up</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> PCR clean up of the 0.4 PCR</P><P><B>Materials and Methods:</B></P><UL><LI> NucleoSpin Gel and PCR Clean-up Kit (macherey-nagel)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ITR - Digestion</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> contol of the PCR product and the ligation (leftITR-neo)</P><P><B>Materials:</B></P><UL><LI> fast digest enzyms
</LI></UL><UL><LI> fast digest green buffer
</LI></UL><UL><LI> Enzyms: PstI, XbaI, SpeI
</LI></UL><P><B>Methods:</B></P><UL><LI> leftITR-neo
</LI></UL><UL><LI> Digestion: 1 ÂµL SpeI + 0,3 ÂµL PstI + 3 ÂµL FD green buffer add 30 ÂµL water (three times: 10 min, 20 min and 40 min digestion time)
</LI></UL><UL><LI> PCR
</LI></UL><UL><LI> Digestion: 2 ÂµL XbaI + 2 ÂµL PstI + 3 ÂµL FD green buffer add 30 ÂµL water
</LI></UL><UL><LI> leftITR
</LI></UL><UL><LI> Digestion: 2 ÂµL SpeI + 2 ÂµL PstI + 3 ÂµL FD green buffer add 30 ÂµL water
</LI></UL><P>one hour at 37 Â°C</P><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> new PCR with neu primer
</LI></UL><UL><LI> new ligation with the neu pcr product
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-21_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-21</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: EGFR - staining of the membrane</P><P><B>Investigators: </B> Xenia and Kathi </P><P><B> Aim: </B> staining of the membrane</P><P><B>Materials:</B></P><UL><LI> TBST buffer + milk powder
</LI></UL><UL><LI> antibody:
</LI></UL><P><B>Methods:</B></P><UL><LI> washing three times of the membrane with TBST
</LI></UL><UL><LI> 15 min incubation with TBST buffer
</LI></UL><UL><LI> three times washing of the membrane
</LI></UL><UL><LI> incubation with the antibody
</LI></UL><UL><LI> three times washing of the membrane with TBST
</LI></UL><UL><LI> staining of the membraine
</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> periplasma preparation</LI></UL><H3><SPAN class="mw-headline" id="2012-09-22_3"><P style="background-color: rgb(240, 20, 70);"> 2012-09-22</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus - transfection</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> The plasmids were transfected in HEK-cells with the following plasmids: pSB1C: GOI = CFP (1875 ng) phelper (1875 ng) P01: YFP linked on VP1 (1875 ng) p06: VP1 knock out (1875 ng)</P><P><B>Materials:</B></P><UL><LI> Medium: opti Mem
</LI></UL><UL><LI> transfection reagent: PEI 15 Âµg per each transfection
</LI></UL><P><B>Methods:</B></P><UL><LI> transfection preparation 1875 ng per plasmid; add 200 ÂµL opti MEM per each transfection 4.8 * DNA --&gt; 72 ÂµL PEI add 200 ÂµL per each transfection
</LI></UL><UL><LI> three times 10 s mixing
</LI></UL><UL><LI> 2 min incubation at RT
</LI></UL><UL><LI> centrifagation
</LI></UL><H3><SPAN class="mw-headline" id="2012-09-25_2"><P style="background-color: rgb(240, 20, 70);"> 2012-09-25</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Virus - qPCR</P><P><B>Investigator: </B> Kathi </P><P><B> Aim: </B> test the genomic virus titer of the viruses with YFP on the surface and CFP=GOI</P><P><B>Materials:</B></P><UL><LI> RNase free watreÃ¤
</LI><LI> primer
</LI><LI> CMV_forward_qPCR: 5' - GGGACTTTCCTACTTGGCA - 3'
</LI><LI> CMV_reverse_qPCR: 5' - GGCGGAGTTGTTACGACA - 3'
</LI></UL><LI> DNaseI (NEB)
</LI><LI> 2x QuantiFast SYBR-green (QIAGEN)
</LI><P><B>Methods:</B></P><UL><LI> the real time standard protocol were used
</LI></UL><P><B> Results:</B></P><UL><LI> there are 3,47 copies of the virus in each PCR
</LI></UL><P>&lt;/div&gt;</P><H3><SPAN class="mw-headline" id="2012-09-29"><P style="background-color: rgb(240, 20, 70);"> 2012-09-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR - Kanamycin cassette</P><P><B>Investigator: </B> Xenia </P><P><B>Materials:</B></P><UL><LI> Kanamycin resistance cassette
</LI></UL><UL><LI> r_primer2_ neomycin cassette-spe_pst-fertig2: CCAAACTCATCAATGTATCTTATCATGTCTGCCCGATACTAGTAGGGGCTGCAG
</LI></UL><UL><LI> RFC10prefix_f_primer_neomycin_cassette_xbaI_fertig: GAATTCGCGGCCGCTTCTAGACGGAATGTGTGTCAGTTAGGGTGTGG
</LI></UL><P><B>Methods:</B></P><UL><LI> final concentration of DNA: 200 ng
</LI></UL><UL><LI> Annealing temperature: 71 Â°C
</LI></UL><P><B>Further tasks:</B></P><P>Gelelectrophorese or PCR with final concentration of DNA: 20 ng
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR - Kanamycin cassette</P><P><B>Investigator: </B> Xenia </P><P><B>Materials:</B></P><UL><LI> Kanamycin resistance cassette
</LI></UL><UL><LI> r_primer2_ neomycin cassette-spe_pst-fertig2: CCAAACTCATCAATGTATCTTATCATGTCTGCCCGATACTAGTAGGGGCTGCAG
</LI></UL><UL><LI> RFC10prefix_f_primer_neomycin_cassette_xbaI_fertig: GAATTCGCGGCCGCTTCTAGACGGAATGTGTGTCAGTTAGGGTGTGG
</LI></UL><P><B>Methods:</B></P><UL><LI> final concentration of DNA: 20 ng
</LI></UL><UL><LI> Annealing temperature: 71 Â°C
</LI></UL><P><B>Further tasks:</B></P><P>Gelelectrophorese
</P><H3><SPAN class="mw-headline" id="2012-10-16"><P style="background-color: rgb(240, 20, 70);"> 2012-10-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Gelelectrophoresis </P><P><A href="/File:UP12_neomycin-2012-10-16.gif" title="File:UP12 neomycin-2012-10-16.gif">150px</A></P><P>the first two samples are with 200 ng and the 3rd sample is with 20 ng  final concentration of template
</P><H3><SPAN class="mw-headline" id="2012-10-16_2"><P style="background-color: rgb(240, 20, 70);"> 2012-10-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR clean up </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/September">http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/September</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Lab/Labjournal/September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Lab/Labjournal/September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/September&amp;oldid=290506" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>