"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Registry
From 2009.igem.org
University of Aberdeen iGEM 2009
Home
Our Project
Introduction, Overview and Conclusion
Parameter Database
Ethics and Safety
Wet Lab
Overview
AND Gate >
Overview
Cloning Strategy
Results
LacI-Latch >
Overview
Cloning Strategy
Results
Quorum Sensing>
Overview
Cloning Strategy
Results
Beta-Galactosidase Assay
Wet Lab Procedures
Modelling
Overview and Conclusions
System Models >
Deterministic
Stochastic
SimBiology
Parameters
Hill Input Functions
PDE's/Next Steps
Chemotaxis
System Investigation >
Dissociation Constants
Sensitivity Simulations
QS Problems
Amended Model
QS Activation Point
Combined Model
Downloads
Notebook
AND Gate
LacI-Latch
Quorum Sensing
Modelling
BioBricks
BioBricks Submitted
BioBrick Experience
Our Team
Overview
Team Members
Instructors
Supervisors
 
University of Aberdeen - Pico Plumber
iGEM 2009
BioBrick Experience
BBa_B0030
Aberdeen_Scotland 2009
The miniprep, single and double digests all worked. However we did not use this part for further cloning.
BBa_B0034
Aberdeen_Scotland 2009
Our miniprep, double digest and gel worked properly. We used this part in fusion PCR for building our biobrick . The fusion PCR, cloning and sequence were all correct.
BBa_C0040
Aberdeen_Scotland 2009
The transformation, miniprep and the gel (undigested and digested with the EcoRI and SpeI) worked as expected.
BBa_C0051
Aberdeen_Scotland 2009
Our miniprep, double digest and single digest worked as expected. We used this part for further cloning which was successful. The sequencing was correct. However we can not confirm that this part was working as we did not test it's functionality.
BBa_E0840
Aberdeen_Scotland 2009
Our miniprep, double and single digests worked as expected. We used this biobrick for building two of our parts ( and '). We experienced very satisfactory cloning with high efficiency of transformants. The sequencing was correct.  We discovered that all the bacteria containing this biobrick were releasing fluorescence even though there was no promoter upstream. The level of fluorescence observed was dependent on the plasmid copy number. We noticed that colonies displayed a visible, yellow-green colour, therefore selection was very easy.
BBa_I0462
Aberdeen_Scotland 2009
The transformation, miniprep and digests, using XbaI and PstI, worked as expected.  We did not test this part further, however, and can not confirm this parts functionality.
BBa_I13600
Aberdeen_Scotland 2009
The gel analysis and the plasmid miniprep worked quite nicely as expected. But later the construct was further tested and characterised by combining with the bio-brick BBa_K182005 which contained a tet repressor under the control of a CI operator. So, the CFP expression of BBa_I13600was repressed as expected when tested with the BBa_K182005. With the addition of Anhydrotetracycline (an analogue of tetracycline) the CFP expression was restored and this was confirmed performing the fluorescent microscopy assay.
Expression can be visualised very nicely under a flurescent microscope. Even without having ideal filters. We used a GFP filter for the picture below.
Transforming BBa_K182005 into these cells should stop the production of CFP, since it expresses TetR. If it works as anticipated, the K182005 Tet repressor should bind to the Tet Operator of I13600 which would stop the expression of CFP.
Since K182005 is on an Ampicillin / chloramphenicol vector (pSB1AC3) and I13600 is on an Ampicillin vector, we decided to first transform I13600 into a cell and select for Ampicillin. We then checked for CFP expression and made the cells competent. After this we transformed K182005 into these cells and selected for Chloramphenicol.
With both plasmids now in the cell we checked for fluorescence, in the presence and absence of anhydrotetracycline (AHT or ATc), as can be seen in the picture below.
The above Pictures demonstrate that TetR effectivly binds to the I13600 Tet Operator.
BBa_I13601
Aberdeen_Scotland 2009
The gel analysis and the plasmid miniprep worked quite nicely as expected. In addition, the construct was further tested by adding IPTG which took off the lacRepressor of the LacO and hence enabling the CFP expression.
BBa_I732100
Aberdeen_Scotland 2009
The plasmid miniprep and the gel worked as expected. However, since we did not use this part for our cloning step we cannot comment on the sequencing or the other features of this part.
BBa_I732820
BBa_J37032
Aberdeen_Scotland 2009
Experimental Design
The  BioBrick expresses GFP under control of a LuxR responsive promoter. The  part was provided by iGEM-HQ, was succesfully mini-preppred and transformed into E.coli.   was used to test our constitutive LuxR/LuxI operon (). For that purpose SCS1 E.coli was transformed either with  alone (referred to as pIR in figure below),   alone, or a combination of both plasmids. The single transformants acted as negative controls.
Results
It would be expected that the  transformant would not fluoresce, since it does not express GFP. This was confirmed (top panel, figure below, showing light microscope pictures on the left, and fluorescent images on the right).  Similarly, the single  transformant was not expected to exhibit much fluorescence, since there is no endogenous expression of LuxR or LuxI in E.coli. A combination of  and  would be expected to produce high amounts of fluorescence at high cell densities as quorum sensing triggers the Lux promoter of . However, in overnight stationary phase cultures at high density, it was found that the  construct expresses the same high amount of GFP fluorescence with or without the presence of LuxR (compare middle panel [J37032 single transformant] with bottom panel [J37032/pIR double transformant])
Conclusion
We conclude that  is either not responsive to LuxR, or that the pLux promoter is extremely leaky, driving significant GFP expression in the absence of LuxR. It therefore cannot easily be used to test quorum sensing due to this high background level of GFP expression.
BBa_K081008
Aberdeen_Scotland 2009
Transformed OK and digested OK with enzymes EcoRI and SpeI.
BBa_R0040
Aberdeen_Scotland 2009
The initial gel worked although the obvious confirmation was not possible due to the very small fragment size (54bp).But later it was confirmed following its usage by forming the part BBa_K182001 through the PCR gel analysis and the sequencing.The plasmid miniprep also worked.
BBa_R0051
Aberdeen_Scotland 2009
The initial gel worked although the obvious confirmation was not possible due to the very small fragment size (49bp).But later it was confirmed following its usage by forming the part BBa_K182002 through the PCR gel analysis and the sequencing.The plasmid miniprep also worked.
BBa_R0062
Aberdeen_Scotland 2009
Our miniprep, digest and gel gave expected results.  However we did not use this part for our cloning.
BBa_S03518
Aberdeen_Scotland 2009
The transformation, miniprep and the gel (undigested and digested with the EcoRI and SpeI) worked. Further testings were performed by the formation of the BBa_K182002, PCR analysis, sequencing and everything  worked flawlessly, exceeding our expectations.
pSB1AC3
Aberdeen_Scotland 2009
The transformation, miniprep and the gel (undigested and digested with the EcoRI and PstI) worked. Further testings were performed by the formation of the  and , PCR analysis, sequencing and everything worked as expected.
pSB1AT3
Aberdeen_Scotland 2009
Plasmid rescued contained BBa_04450).  The transformation, miniprep and the gel (undigested and digested with the EcoRI and PstI) worked. Further testings were performed through the PCR analysis and subsequent sequencing.
pSB3C5
Aberdeen_Scotland 2009
The transformation was twice unsuccessful.  Plasmid rescued contained BBa_J04450.
pSB3T5
Aberdeen_Scotland 2009
Plasmid rescued contained BBa_J04450.  Transformation, miniprep, double and single digests all worked properly. It was noticeable that colonies were a light pink colour. We used this part for further cloning. This plasmid gave us a very high efficiency of cloning.
pSB4C5
Aberdeen_Scotland 2009
Digestion with EcoRI and PstI worked well and complete digestion was observed. When pSB4C5 - I52002 is transformed into XL1-blue cells, colonies are obtained. pSB4C5-I52002 carries the ccdB gene, toxic to wild-type E.coli strains. This plasmid was propagated in our experiments using the Invitrogen strain DB3.1, carrying the gyrA462 allele that confers resistance to this ccdB allele. During the generation of recombinant clones, the ccdB gene was excised and replaced by other BioBricks. For some of these experiments, E.coli strain XL1-Blue was employed. However, other iGEM teams should be aware that XL1-Blue also carries a gyrA allele, explaining our observation that this common E.coli strain was not killed by non-recombinant pSB14C5.
pSB4T5
Aberdeen_Scotland 2009
The transformation was twice unsuccessful.  Plasmid rescued contained BBa_I52001.
  Back to Our BioBrick's
Continue to Our Team  
 
Back to Top of Page
Retrieved from " http://2009.igem.org/Team:Aberdeen_Scotland/Registry "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
