"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/parameters
From 2009.igem.org
University of Aberdeen iGEM 2009
Home
Our Project
Introduction, Overview and Conclusion
Parameter Database
Ethics and Safety
Wet Lab
Overview
AND Gate >
Overview
Cloning Strategy
Results
LacI-Latch >
Overview
Cloning Strategy
Results
Quorum Sensing>
Overview
Cloning Strategy
Results
Beta-Galactosidase Assay
Wet Lab Procedures
Modelling
Overview and Conclusions
System Models >
Deterministic
Stochastic
SimBiology
Parameters
Hill Input Functions
PDE's/Next Steps
Chemotaxis
System Investigation >
Dissociation Constants
Sensitivity Simulations
QS Problems
Amended Model
QS Activation Point
Combined Model
Downloads
Notebook
AND Gate
LacI-Latch
Quorum Sensing
Modelling
BioBricks
BioBricks Submitted
BioBrick Experience
Our Team
Overview
Team Members
Instructors
Supervisors
 
University of Aberdeen - Pico Plumber
iGEM 2009
Internal Dynamics Parameters
When modeling any process, it is essential to use parameters which reflect reality, otherwise the model will not give accurate predictions. Thus we spend a considerable time researching, analysing and estimating parameters so that our model will reflect real life behaviour as accuratly as possible.
Parameter
Description
Value
Unit
Reference
Rate of production of HSL from LuxI
0.45
1/s
[1]
HSL
Rate of diffusion of HSL in/out of the cell
0.4
1/s
[1]
IPTG
Rate of diffusion of IPTG in/out of the cell
0.014
1/s
[3]
pproduction
Number of plasmids
10
medium copy plasmid number; decided within the team
platch
Number of plasmids
10
medium copy plasmid number; decided within the team
plysis
Number of plasmids
10
medium copy plasmid number; decided within the team
panti-lysis
Number of plasmids
10
medium copy plasmid number; decided within the team
pQS
Number of plasmids
10
medium copy plasmid number; decided within the team
max_production
Maximal production rate of lux box promoter
0.44
pops
[1]
min_production
Minimal production rate of lux box promoter
0.013
pops
estimated
latch
Maximal production rate of latch promoter
0.28
pops
See Below
lysis
Maximal production rate of lysis promoter
0.0426
pops
See Below
anti-lysis
Maximal production rate of anti-lysis promoter
0.0066
pops
See Below
QS
Maximal production rate of QS promoter
0.018
pops
See Below
KLacI
Dissociation constant for LacI to LacO
700
m
See Explanation
KLacI-IPTG
Dissociation constant for IPTG to LacI
1200
m
See Explanation
KTetR
Dissociation constant for TetR to TetO
7000
m
See Explanation
KcI
Dissociation constant for cI to operon
7000
m
See Explanation
KP
Dissociation constant for P to lux box
700
m
See Explanation
nLacI
Hill coefficient
2
[10]
ncI
Hill coefficient
2
[11]
nTetR
Hill coefficient
3
[1]
nP
Hill coefficient
2
[1]
mRNA
Degradation of mRNA
0.00288
1/s
[2]
X
Degradation of X
0.00000802
1/s
[5]
Y
Degradation of Y
0.00000802
1/s
[5]
λ-CI
Degradation of lambda CI
0.002888
1/s
[5]
LacI
Degradation of LacI
0.001155
1/s
[8]
TetR
Degradation of TetR
0.00288811
1/s
tagged; half-life of 4 min
Holin
Degradation of Holin
0.0002
1/s
estmated; half-life of an hour
Endolysin
Degradation of Endolysin
0.0002
1/s
estimated; half-life of an hour
Antiholin
Degradation of Antiholin
0.0002
1/s
estimated; half-life of an hour
LuxI
Degradation of LuxI
0.002888
1/s
tagged; half-life of 4 min
LuxR
Degradation of LuxR
0.0002
1/s
[1]
HSL
Degradation of HSL
0.00016667
1
[8]; value for AHL
Protein
Translation rate of Protein
0.1
1/s
[2]
P
Rate of formation of the HSL-LuxI complex
0.00010
1/ms
[1]
-P
Rate of dissociation of the HSL-LuxI complex
0.003
1/s
[1]
Calculating promoter strengths
A brief discription of how some of our promoter strengths were calculated follows:
The paper "Measuring the activity of BioBrick promoters using an in vivo reference standard" published in the Journal of Biological Engineering makes J23101 their reference standard, giving it a strength of 0.03 POPS.
We can directly compare the strengths of all the J series constititive promoters using other promoters in the above paper and the promoters in the standard registry of parts website. From this page we can calculate that if J23101 =0.03 pops then our original LuxI and LuxR promoter J23107 has a strength of ~0.018 POPS and our ammended model LuxR promoter has a strength of ~0.01 POPS. We also find that our constitutive Antiholin promoter has a strength of ~0.0066 POPS
LuxR promoter polymerase per second (PoPS)  The following
BioBrick F2620 defines the LuxR responsive promoter. This has been assessed as having 6.6 PoPS promoter activity, i.e. 6.6 RNA polymerisation events per second.  However, this seemed unlikely;  the most active promoter in E.coli has about 2 polymerisations per second. At the iGEM meeting in Edinburgh this year Dr Stansfield attended on behalf of the team were he met Barry Canton, the actual experimenter who deduced the 6.6 PoPS.  When questioned he stated that he measured the PoPS value in PoPs per cell;  the LuxR promoter/GFP construct was on a ‘15 copies per cell’ plasmid, thus there were 15 copies of the LuxR promoter.  The actual PoPS =  6.6/15 = 0.44 PoPS   i.e. consistent with known E.coli promoters.
An estimate for the strength of different E.coli promoters, including lambda P(L) is in the following paper:
Journal of Molecular Biology Volume 292, Issue 1,  Activities of constitutive promoters in Escherichia coli1
S.-T Liang1, 2, M Bipatnath1, Y.-C Xu1, S.-L Chen2, P Dennis3
It states the PoPs values for several promoters as:
Pspc,  26 polymerase initiations per minute  P(L), 17 polymerase initiations per minute  Pbla   1.7 polymerase initiations per minute  PRNAI, 17 polymerase initiations per minute
Using the P(L) strength of 17 polymerase initiations per minute we can easily calculate the PoPs value.
  Back to SimBiology
Continue to Hill Input Functions  
References
[1] Goryachev, A.B., D.J. Toh and T. Lee. “System analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constant.” BioSystems 2006: 83, 178-187.
[2] Alon, Uri. “An Introduction to Systems Biology Design Principles of Biological Circiuts.” London: Chapman & Hall/CRC, 2007.
[3] Kepes, A., 1960, “Etudes cinetiques sur la galactoside-permease D'Escherichia coli. Biochim.” Biophys. Acta 40, 70-84.
[4] Canton, B. and Anna Labno. “Part: BBa_F2620.” BioBrick Registry.  13th August 2009. < http://partsregistry.org/Part:BBa_F2620 >
[5] Andersen JB, Sternberg C, Poulsen LK, Bjorn SP, Givskov M, Molin S. “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria.” Appl Environ Microbiol. 1998 Jun; 64(6):2240-6.
[6] Elowitz MB, Leibler S.; “A synthetic oscillatory network of transcriptional regulators.” Nature 2000 Jan; 403(6767):335-8.;
[7] BCCS-Bristol 2008. “Modelling Parameters” iGEM wiki. 13th August 2009. < https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters >
[8] Subhayu Basu; “A synthetic multicellular system for programmed pattern formation.” Nature April 2005: 434, 1130-1134
[9] KULeuven 2008. “Cell Death”iGEM wiki. 13th August 2009. < https://2008.igem.org/Team:KULeuven/Model/CellDeath >
[10] ETHZ 2007. “Engineering” iGem wiki. 13th August 2009. < https://2007.igem.org/ETHZ/Engineering >
[11] Bintu, Lacramioara, Terence Hwa. “Transcriptional regulation by the numbers: applications.” Current Opinion in Genetics & Development 2005, 15:125–135.
 
Back to Top of Page
Retrieved from " http://2009.igem.org/Team:Aberdeen_Scotland/parameters "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
