<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aberdeen_Scotland_parameters"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Aberdeen_Scotland/parameters">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aberdeen_Scotland/parameters&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/parameters&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/parameters&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aberdeen_Scotland/parameters" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aberdeen Scotland/parameters</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Aberdeen iGEM 2009</TITLE></P><DIV id="pagewrapper"><DIV id="titleheader"><DIV id="mainnavigation"><DIV class="menuwidth0"><DIV class="menuwidth1"><DIV class="menuwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2009.igem.org/Team:Aberdeen_Scotland" class="menu">Home</A></LI></UL><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/Project">Our Project</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/Project">Introduction, Overview and Conclusion</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameterdatabase">Parameter Database</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/ethics">Ethics and Safety</A></LI></UL><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab">Wet Lab</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/andgate/overview"><SPAN class="drop"><SPAN>AND Gate &gt;</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/andgate/overview">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/andgate/cloning">Cloning Strategy</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/andgate/results">Results</A></LI></UL><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/lacilatch"><SPAN class="drop"><SPAN>LacI-Latch &gt;</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/lacilatch">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/lacilatch/cloning">Cloning Strategy</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/lacilatch/results">Results</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/quorumsensing"><SPAN class="drop"><SPAN>Quorum Sensing&gt;</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/quorumsensing">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/quorumsensing/cloningstrategy">Cloning Strategy</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/quorumsensing/results">Results</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/betagalactosidaseassay">Beta-Galactosidase Assay</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/WetLab/Protocols">Wet Lab Procedures</A></LI><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/modeling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/modeling">Overview and Conclusions</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/internal/deterministic"><SPAN class="drop"><SPAN>System Models &gt;</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/internal/deterministic">Deterministic</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/internal/stochastic">Stochastic</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/internal/SimBiology">SimBiology</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">Parameters</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/hillinput">Hill Input Functions</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/modeling/pde">PDE's/Next Steps</A></LI></UL><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/chemotaxis">Chemotaxis</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/overview"><SPAN class="drop"><SPAN>System Investigation &gt;</SPAN></SPAN></A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">Dissociation Constants</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_3">Sensitivity Simulations</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_4">QS Problems</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_5">Amended Model</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_6">QS Activation Point</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/modeling/combined_model">Combined Model</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/modeling/downloads">Downloads</A></LI><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/notebook/andgate">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/notebook/andgate">AND Gate</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/notebook/lacilatch">LacI-Latch</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/notebook/quorumsensing">Quorum Sensing</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/notebook/modeling">Modelling</A></LI></UL><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/Ourparts">BioBricks</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/Ourparts">BioBricks Submitted</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/Registry">BioBrick Experience</A></LI></UL><UL><LI><A class="menu" href="https://2009.igem.org/Team:Aberdeen_Scotland/OurTeam" title="Team">Our Team</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/OurTeam">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/OurTeam/TeamMembers">Team Members</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/OurTeam/Instructors">Instructors</A></LI><LI><A href="https://2009.igem.org/Team:Aberdeen_Scotland/OurTeam/Advisors">Supervisors</A></LI></UL></DIV></DIV></DIV></DIV><H3 class="toptitle"> </H3></DIV><DIV id="mainpic"><DIV id="pagetitlewrapper"><H3 class="maintitle">University of Aberdeen - Pico Plumber</H3><DIV id="pagesubtitle">iGEM 2009</DIV></DIV></DIV></DIV><DIV class="wide"><H2><SPAN class="mw-headline" id="Internal_Dynamics_Parameters"> Internal Dynamics Parameters </SPAN></H2><P>When modeling any process, it is essential to use parameters which reflect reality, otherwise the model will not give accurate predictions. Thus we spend a considerable time researching, analysing and estimating parameters so that our model will reflect real life behaviour as accuratly as possible.
</P><P><TABLE class="parameter"><TBODY><TR><TD><B>Parameter</B></TD><TD><B>Description</B></TD><TD><B>Value</B></TD><TD><B>Unit</B></TD><TD><B>Reference</B></TD></TR><TR><TD>Rate of production of HSL from LuxI</TD><TD>0.45</TD><TD>1/s</TD><TD>[1]</TD></TR><TR><TD><SUB>HSL</SUB></TD><TD>Rate of diffusion of HSL in/out of the cell</TD><TD>0.4</TD><TD>1/s</TD><TD>[1]</TD></TR><TR><TD><SUB>IPTG</SUB></TD><TD>Rate of diffusion of IPTG in/out of the cell</TD><TD>0.014</TD><TD>1/s</TD><TD>[3]</TD></TR><TR><TD>p<SUB>production</SUB></TD><TD>Number of plasmids </TD><TD>10</TD><TD>medium copy plasmid number; decided within the team</TD></TR><TR><TD>p<SUB>latch</SUB></TD><TD>Number of plasmids </TD><TD>10</TD><TD>medium copy plasmid number; decided within the team</TD></TR><TR><TD>p<SUB>lysis</SUB></TD><TD>Number of plasmids </TD><TD>10</TD><TD>medium copy plasmid number; decided within the team</TD></TR><TR><TD>p<SUB>anti-lysis</SUB></TD><TD>Number of plasmids </TD><TD>10</TD><TD>medium copy plasmid number; decided within the team</TD></TR><TR><TD>p<SUB>QS</SUB></TD><TD>Number of plasmids </TD><TD>10</TD><TD>medium copy plasmid number; decided within the team</TD></TR><TR><TD><SUB>max_production</SUB></TD><TD>Maximal production rate of lux box promoter</TD><TD>0.44</TD><TD>pops</TD><TD>[1]</TD></TR><TR><TD><SUB>min_production</SUB></TD><TD>Minimal production rate of lux box promoter</TD><TD>0.013</TD><TD>pops</TD><TD>estimated</TD></TR><TR><TD><SUB>latch</SUB></TD><TD>Maximal production rate of latch promoter</TD><TD>0.28</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD></TR><TR><TD><SUB>lysis</SUB></TD><TD>Maximal production rate of lysis promoter</TD><TD>0.0426</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD></TR><TR><TD><SUB>anti-lysis</SUB></TD><TD>Maximal production rate of anti-lysis promoter</TD><TD>0.0066</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD></TR><TR><TD><SUB>QS</SUB></TD><TD>Maximal production rate of QS promoter</TD><TD>0.018</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD></TR><TR><TD>K<SUB>LacI</SUB></TD><TD>Dissociation constant for LacI to LacO </TD><TD>700</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD></TR><TR><TD>K<SUB>LacI-IPTG</SUB></TD><TD>Dissociation constant for IPTG to LacI </TD><TD>1200</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD></TR><TR><TD>K<SUB>TetR</SUB></TD><TD>Dissociation constant for TetR to TetO </TD><TD>7000</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD></TR><TR><TD>K<SUB>cI</SUB></TD><TD>Dissociation constant for cI to operon </TD><TD>7000</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD></TR><TR><TD>K<SUB>P</SUB></TD><TD>Dissociation constant for P to lux box </TD><TD>700</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD></TR><TR><TD>n<SUB>LacI</SUB></TD><TD>Hill coefficient </TD><TD>2</TD><TD>[10]</TD></TR><TR><TD>n<SUB>cI</SUB></TD><TD>Hill coefficient </TD><TD>2</TD><TD>[11]</TD></TR><TR><TD>n<SUB>TetR</SUB></TD><TD>Hill coefficient </TD><TD>3</TD><TD>[1]</TD></TR><TR><TD>n<SUB>P</SUB></TD><TD>Hill coefficient </TD><TD>2</TD><TD>[1]</TD></TR><TR><TD><SUB>mRNA</SUB></TD><TD>Degradation of mRNA</TD><TD>0.00288</TD><TD>1/s</TD><TD>[2]</TD></TR><TR><TD><SUB>X</SUB></TD><TD>Degradation of X</TD><TD>0.00000802</TD><TD>1/s</TD><TD>[5]</TD></TR><TR><TD><SUB>Y</SUB></TD><TD>Degradation of Y</TD><TD>0.00000802</TD><TD>1/s</TD><TD>[5]</TD></TR><TR><TD><SUB>λ-CI</SUB></TD><TD>Degradation of lambda CI</TD><TD>0.002888</TD><TD>1/s</TD><TD>[5]</TD></TR><TR><TD><SUB>LacI</SUB></TD><TD>Degradation of LacI</TD><TD>0.001155</TD><TD>1/s</TD><TD>[8]</TD></TR><TR><TD><SUB>TetR</SUB></TD><TD>Degradation of TetR</TD><TD>0.00288811</TD><TD>1/s</TD><TD>tagged; half-life of 4 min</TD></TR><TR><TD><SUB>Holin</SUB></TD><TD>Degradation of Holin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estmated; half-life of an hour</TD></TR><TR><TD><SUB>Endolysin</SUB></TD><TD>Degradation of Endolysin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estimated; half-life of an hour</TD></TR><TR><TD><SUB>Antiholin</SUB></TD><TD>Degradation of Antiholin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estimated; half-life of an hour</TD></TR><TR><TD><SUB>LuxI</SUB></TD><TD>Degradation of LuxI</TD><TD>0.002888</TD><TD>1/s</TD><TD>tagged; half-life of 4 min</TD></TR><TR><TD><SUB>LuxR</SUB></TD><TD>Degradation of LuxR</TD><TD>0.0002</TD><TD>1/s</TD><TD>[1]</TD></TR><TR><TD><SUB>HSL</SUB></TD><TD>Degradation of HSL</TD><TD>0.00016667</TD><TD>1</TD><TD>[8]; value for AHL</TD></TR><TR><TD><SUB>Protein</SUB></TD><TD>Translation rate of Protein</TD><TD>0.1</TD><TD>1/s</TD><TD>[2]</TD></TR><TR><TD><SUB>P</SUB></TD><TD>Rate of formation of the HSL-LuxI complex</TD><TD>0.00010</TD><TD>1/ms</TD><TD>[1]</TD></TR><TR><TD><SUB>-P</SUB></TD><TD>Rate of dissociation of the HSL-LuxI complex</TD><TD>0.003</TD><TD>1/s</TD><TD>[1]</TD></TR></TBODY></TABLE></P></DIV><DIV class="wide"><A name="promotor"><H1><SPAN class="mw-headline" id="Calculating_promoter_strengths"> Calculating promoter strengths </SPAN></H1><P>A brief discription of how some of our promoter strengths were calculated follows:
</P><P>The paper &quot;Measuring the activity of BioBrick promoters using an in vivo reference standard&quot; published in the Journal of Biological Engineering makes J23101 their reference standard, giving it a strength of 0.03 POPS.
</P><P>We can directly compare the strengths of all the J series constititive promoters using other promoters in the above paper and the promoters in the standard registry of parts website. From this page we can calculate that if J23101 =0.03 pops then our original LuxI and LuxR promoter J23107 has a strength of ~0.018 POPS and our ammended model LuxR promoter has a strength of ~0.01 POPS. We also find that our constitutive Antiholin promoter has a strength of ~0.0066 POPS
</P><P>LuxR promoter polymerase per second (PoPS) 
The following  </P></A><A href="http://partsregistry.org/Part:BBa_F2620">BioBrick F2620</A> defines the LuxR responsive promoter. This has been assessed as having 6.6 PoPS promoter activity, i.e. 6.6 RNA polymerisation events per second.  However, this seemed unlikely;  the most active promoter in E.coli has about 2 polymerisations per second. At the iGEM meeting in Edinburgh this year Dr Stansfield attended on behalf of the team were he met Barry Canton, the actual experimenter who deduced the 6.6 PoPS.  When questioned he stated that he measured the PoPS value in PoPs per cell;  the LuxR promoter/GFP construct was on a ‘15 copies per cell’ plasmid, thus there were 15 copies of the LuxR promoter.  The actual PoPS =  6.6/15 = 0.44 PoPS   i.e. consistent with known E.coli promoters.
<P>An estimate for the strength of different E.coli promoters, including lambda P(L) is in the following paper:
</P><P>Journal of Molecular Biology
Volume 292, Issue 1, 
Activities of constitutive promoters in Escherichia coli1
</P><P>S.-T Liang1, 2, M Bipatnath1, Y.-C Xu1, S.-L Chen2, P Dennis3
</P><P>It states the PoPs values for several promoters as:
</P><P>Pspc,  26 polymerase initiations per minute 
P(L), 17 polymerase initiations per minute 
Pbla   1.7 polymerase initiations per minute 
PRNAI, 17 polymerase initiations per minute 
</P><P>
Using the P(L) strength of 17 polymerase initiations per minute we can easily calculate the PoPs value.
</P><P><TABLE class="nav"><TBODY><TR><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/internal/SimBiology">  Back to SimBiology</A></TD><TD align="right"><A href="https://2009.igem.org/Team:Aberdeen_Scotland/hillinput">Continue to Hill Input Functions  </A></TD></TR></TBODY></TABLE></P></DIV><DIV class="wide"><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>[1] Goryachev, A.B., D.J. Toh and T. Lee. “System analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constant.” BioSystems 2006: 83, 178-187.
</P><P>[2] Alon, Uri. “An Introduction to Systems Biology Design Principles of Biological Circiuts.” London: Chapman &amp; Hall/CRC, 2007.
</P><P>[3] Kepes, A., 1960, “Etudes cinetiques sur la galactoside-permease D'Escherichia coli. Biochim.” Biophys. Acta 40, 70-84.
</P><P>[4] Canton, B. and Anna Labno. “Part: BBa_F2620.” BioBrick Registry.  13th August 2009. &lt;<A href="http://partsregistry.org/Part:BBa_F2620" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_F2620</A>&gt;</P><P>[5] Andersen JB, Sternberg C, Poulsen LK, Bjorn SP, Givskov M, Molin S. “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria.” Appl Environ Microbiol. 1998 Jun; 64(6):2240-6.
</P><P>[6] Elowitz MB, Leibler S.; “A synthetic oscillatory network of transcriptional regulators.” Nature 2000 Jan; 403(6767):335-8.;
</P><P>[7] BCCS-Bristol 2008. “Modelling Parameters” iGEM wiki. 13th August 2009. &lt;<A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters" class="external free" rel="nofollow">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A>&gt;</P><P>[8] Subhayu Basu; “A synthetic multicellular system for programmed pattern formation.” Nature April 2005: 434, 1130-1134 
</P><P>[9] KULeuven 2008. “Cell Death”iGEM wiki. 13th August 2009. &lt;<A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath" class="external free" rel="nofollow">https://2008.igem.org/Team:KULeuven/Model/CellDeath</A>&gt;</P><P>[10] ETHZ 2007. “Engineering” iGem wiki. 13th August 2009. &lt;<A href="https://2007.igem.org/ETHZ/Engineering" class="external free" rel="nofollow">https://2007.igem.org/ETHZ/Engineering</A>&gt;</P><P>[11] Bintu, Lacramioara, Terence Hwa. “Transcriptional regulation by the numbers: applications.” Current Opinion in Genetics &amp; Development 2005, 15:125–135.
</P></DIV><P> </P><DIV id="wikifooter"><P><A href="#top">Back to Top of Page</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Aberdeen_Scotland/parameters">http://2009.igem.org/Team:Aberdeen_Scotland/parameters</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aberdeen_Scotland/parameters" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aberdeen_Scotland/parameters" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/parameters&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aberdeen_Scotland/parameters&amp;oldid=157385" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>