<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_IIT_Madras_Scribbles"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:IIT_Madras/Scribbles">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:IIT_Madras/Scribbles&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/Scribbles&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/Scribbles&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:IIT_Madras/Scribbles" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:IIT Madras/Scribbles</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>bar</TITLE></P><H3><U>Ideas from Brainstorming Sessions</U></H3><H3>1. Biomolecular Sensor</H3><P>We propose to make a sensor which can detect the presence of a particular bio-molecule (say a protein).
We have discussed two possible pathways systems to achieve the same:
<UL><LI>Surface receptor/ Ab can identify the biomolecule by binding to it and sending corresponding signal to the core gene complex which will show the respective result probably by an FP.</LI><LI>Ab on the flagella will detect the bio-molecule and this will result in dysfunctional flagella. So in presence of chemotaxis the coloured cells will be scattered in the media separated from the other cells with functional flagella.</LI></UL></P><H3>2. A New Screening Technique</H3><P>If we can express the Bacterial Rhodopsin pathway in E. coli, this can be used as a
new screening technique for cloning. We can hence screen it using two inputs
Light and maybe Lactose.</P><H3>3. Perceptron Learning</H3><P>Bacteria can be used as a processor. It should be viewed as a weight matrix. Thus
we are making the bacteria/weight matrix “learn” certain inputs and analyze the
same after learning.
This can be a good foundational advance and also we can find its importance
in medicine wherein the “learned matrix” can be introduced into the body and
the output can be analyzed. In this example bacterial matrix is detecting other
bacteria, which is essentially having bacteria as input and bacteria as a processor.</P><H3>4. New Technique for Protein Expression</H3><P>Instead of expressing protein by a Double stranded DNA vector, it would be great if we could express the protein on the capsid of the virus which can easily be isolated from the bacteria and hence protein isolation can be improved and be easier.</P><H3>5. Allergen Sensor</H3><P><B>Theory:</B> Many people are allergic to many things. The common test to identify the
respective allergens is to inject very small quantities of allergens into the body
and look for allergic responses.<B>Idea:</B> What if we can express some common allergens as surface proteins on the
bacteria? This can be used to elicit some antigen-antibody response or we can
have this showing some colour code for different allergens interactions.
</P><H3>6. Using RuBisCO as a CO2 Sensor</H3><P><B>Theory</B>: RuBisCO is an enzyme with carbozylase and oxidase action. It acts on the
substrate ribulose bi phosphate. An important enzyme in &quot;fixing&quot; CO2. Now, in
the process of evolution the effectiveness of this enzyme didnt increase. Its a very
slow enzyme and the carboxylase action of this enzyme is preferred when the
local co2 conc is high. We studied the c3 and c4 pathways for better
understanding.
Many scientists view this as a solution to global warming if this enzyme has better
kinetics, then plants can grow soon and fixation of co2 can be a faster process.
A concept called directed evolution was done wherein this enzyme was
introduced into E.Coli. Random mutations were performed to get a better
enzyme. The best achieved was kinetically 5 times better compared to the wild
type.<B>Idea:</B>Rubisco can be easily introduced in E.Coli without any problems of folding due to
chaperonions called groES and groEL present in E.Coli.And a part of the C3 cycle
should be introduced too for the availability of substrate. And we should find a
promotor which is regulated by one of these intermediates.
</P><H3>7. Rust Eating Bacteria</H3><P><B>Theory:</B> Introduce enzymes required to convert Fe2O3 to Fe into bacteria. Could
be used for rust treatment.<B>Comments:</B> The whole concept was looked at very skeptically since the team had
problems with the use of eating away Fe2O3 and how the Fe would be deposited.
The source of energy for the same also was vague.
Protein called Ferritin was talked about which could potentially help us prevent rust.</P><H3>8. Organic Touch Sensor</H3><P><B>Inspiration:</B> Thigmonastic movements of mimosa pudica<B>Theory:</B> Turgorins are molecules released in mimosa pudica which opens up the
k+ gates and cl- gates. k+ ions move out of the cell along with water molecules
and cl- ions enter the cell - there is a loss in turgor pressure which is why the
elaves close - similar mechanism is seen in venus fly trap - as the fly sits on the
leaf, the turgorins reduce the turgor pressure at leaf blade- which causes the
leaves to close upon the insect and then the digestive juices are secreted.<B>Outcome:</B> A plate of cells- touch sensitive- when a finger is used to trace a letter
on the plate, those cells are activated- either we can have the turgorins being
replaced with a fluorescence protein or we can have it being expressed with
fluorescence protein- half life of the protein must be low - then the sensitized
cells create the image - a software can be used to analyse the letter and can be
displayed or even say pronounced.<B>Difficulties:</B> We were not able to find the exact pathway of these thigmonastic
movements work.
</P><H3>9. UV Sensor</H3><P><B>Theory:</B> UV is very important for DNA Repair and Vit D Synthesis. So these
proteins/promotors involved in this can be used.<B>Outcome:</B> Changes in UV is recorded by these promotors. It can be linked to a
fluorescence protein.<B>Difficulties:</B> Vitamin D synthesis is mostly in mammalian cells and DNA Repair is
in eukaryotic cells.
May be this will involve usage of yeast.</P><H3>10. Flesh Eating Bacteria</H3><P><B>Theory:</B> Necrotizing fasciitis is a disease in which human flesh is eaten up by the
bacteria. The idea was to modify the bacteria causing this or to transform the
mechanism utilized in this bacteria to make E.Coli selectively cause fasciitis
which is used to eat up tumor tissues.
<B>Outcomes:</B> The pathway involved is very complex and cannot be transcribed into
E.coli. Also the bacteria is too dangerous to risk using it as a treatment method.</P><H3>11. Deinococcus Radioduran</H3><P><B>Theory:</B> Implementing the RecFOR genome reconstitution of Dienococcus
Radiodurans pathways in eukaryotes to speed up or improve the genomic DNA
repair.<B>Outcomes:</B> Loads of possible versions of the same were thought of. The pathway
was explored in a little detail.<B>Conclusions:</B> The pathway involves high degree of specialisation in terms of the
DNA repair pathways, and the organisation of genes. It also involves the
formation of a 4 Cell colony which survives as a single cell with a single
transcriptionally active genome. Many other difficulties were instrumental in
suppressing the idea to a large extent.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:IIT_Madras/Scribbles">http://2011.igem.org/Team:IIT_Madras/Scribbles</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:IIT_Madras/Scribbles" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:IIT_Madras/Scribbles" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:IIT_Madras/Scribbles&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:IIT_Madras/Scribbles&amp;oldid=262201" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>