"
Page
Discussion
View source
History
teams
Log in
 
Team:IIT Madras/Dry lab/Modelling
From 2011.igem.org
bar
In-Silico - Comparative Growth Analysis of Wild type vs PR Transformed cells
(Metabolic Modeling using Constraint Based Reconstruction and Analysis)
Abstract
An extensive Reconstruction and Flux Balance Analysis study of metabolic pathways in E.coli at the genome scale, considering 1668 metabolites and 2383 reactions and their respective stoichiometry matrices was carried out using a Constraint Based approach. This model was validated with negative regulation of reactions by comparing with literature available for Oxidative Phosphorylation inhibitors. By including variations in substrate (glucose) concentrations under limiting conditions, we analyzed the global effects of Proteorhodopsin a.k.a. PR, (light-dependent proton pump) activity on the host system. Such a model which analyzes global effects on metabolic pathways is a novel addition to pre-existing kinetic models (at the protein level) of PR action.
Hypothesis
Increase in cell growth rate due to the proton efflux generated by Proteorhodopsin in minimal carbon media.
Model Design
Reconstruction and Mathematical Modeling of E.coli K12-MG1655 pathway with Proteorhodopsin. Literature data:
Genome scale metabolic model thermodynamic data for genome scale E.coli K-12 MG1655 was derived. This was done by alignment with genomic annotation and the metabolic content of EcoCyc, characterization and quantification of biomass components and maintenance requirements of cell required for growth of the cell and thermodynamic data for reactions [1].
Reconstruction of the pathway was carried out to suit our project, hence involving the effects due to Proteorhodpsin pumping activity. Data for pH gradient [2], the delta [H+] [3] was taken from literature and hence flux was calculated to formulate a comprehensive model.
(Click on the links below for more details on the methods and simulations)
Model Construction
A Systems Biology Markup Language (SBML) file was created for the E.Coli transformed with PR (model_PR) and Wildtype(model_WT). The flux balance studies were done by constraint based reconstruction and analysis FBA computations, which fall into the category of constraint-based reconstruction and analysis (COBRA) methods using the COBRA toolbox. The COBRA Toolbox is a freely available Matlab toolbox that can be used to perform a variety of COBRA methods, including many FBA-based methods. In Matlab, the models are structures with fields, such as 'rxns' (a list of all reaction names), 'mets' (a list of all metabolite names) and 'S' (the stoichiometric matrix). The function 'optimizeCbModel' is used to perform FBA. Also, gene deletion analysis and their effect on growth rates can also be modeled using COBRA toolbox.
Protocol for Metabolic Modeling
Genome scale models provide us with an ideal platform to study effects of addition or deletion of genes in the system. We were especially interested in quantifying variations in flux rates of various reactions due to the H+ pumping activity of proteorhodopsin. For which we composed a network for the pathway in Systems Biology Markup Language. This was further validated and analysed using COBRA toolbox in MATLAB. This helped us to compare growth rates for wild type model (Model_WT) and Mutant Model with Proteorhodopsin (Model_PR)
Click here to download SBML file for the genome scale e.coli (K-12 MG1665) model including Proteorhodopsin
Validation of Model
Simulations for Proof of Concept
References
"A genome-scale metabolic reconstruction for Escherichia coli K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information" Adam M Feist, Christopher S Henry, Jennifer L Reed, Markus Krummenacker, Andrew R Joyce, Peter D Karp,Linda J Broadbelt, Vassily Hatzimanikatis and Bernhard Ø Palsson,Molecular Systems Biology-2007
"Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host" A. Martinez, A. S. Bradley†, J. R. Waldbauer, R. E. Summons and E. F. DeLong,PNAS-2007
"Light-powering Escherichia coli with proteorhodopsin" Jessica M. Walter, Derek Greenfield, Carlos Bustamante and Jan Liphardt,PNAS-2007
Retrieved from " http://2011.igem.org/Team:IIT_Madras/Dry_lab/Modelling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
