<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CCU_Taiwan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CCU Taiwan/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>No Sidebar - Helios by HTML5 UP</TITLE></P><DIV id="page-wrapper"><DIV id="header"><DIV class="inner"><HEADER><H1><A href="https://2017.igem.org/Team:CCU_Taiwan" id="logo">Experiments</A></H1></HEADER></DIV><NAV id="nav"><UL><LI><A href="https://2017.igem.org/Team:CCU_Taiwan">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Design">Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Design#CSP">CSP detector</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Design#Lactate">Lactate detector</A></LI></UL><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Test_paper">Test paper</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Applied_Design">Applied design</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Notebook">Notebook</A></LI><LI><A href="#">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Hardware">Hardware overview</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Device_design">Device design</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Device_detection">Device detection</A></LI><LI><A href="#">Software</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/IOT system">IOT system</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/APP">APP</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Machine_learning">Machine learning</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Model">Modeling</A></LI><LI><A href="#">Human practice</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Human_Practices">Human practice overview</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/HP/Gold_Integrated">Integrate &amp; Gold</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Engagement">Public engagemant</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Safety">Safety</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Basic_Part">Basic parts</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Composite_Part">Composite parts</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Members">Team members</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:CCU_Taiwan/Medals">Medals</A></LI></NAV></DIV><DIV class="wrapper style1"><DIV class="container"><ARTICLE id="main" class="special"><HEADER><DIV class="div_nav"><NAV class="toc_nav" id="toc_show"><UL><LI><A href="#exp">Experiment</A></LI><LI><A href="#trans">Transformation</A></LI><LI><A href="#pat">Patch and small scale culture</A></LI><LI><A href="#mini">Mini extraction:</A></LI><LI><A href="#cell">Cell lysis</A></LI><LI><A href="#dig">Digestion</A></LI><LI><A href="#lig">Ligation protocol</A></LI><LI><A href="#run">Run electrophoresis gel</A></LI><LI><A href="#pcr">PCR</A></LI><LI><A href="#bs">Competent Cell</A></LI><LI><A href="#ls">Lactate sensor Function test</A></LI><LI><A href="#ft">Functional test for Bacillus subtilis</A></LI></UL></NAV><ARTICLE class="contents_nav"><SECTION><DIV id="exp"><H2>Experiment</H2></DIV><H4>General protocol</H4><DIV id="trans"><H3>Transformation ( by 1.5 ml Eppendorf tube)</H3><P><UL class="”a”"><LI>1.	2.5 μg DNA + 25 μl competent cell (ECOS 101 DH5α).</LI><LI>2.	On ice 30 min.</LI><LI>3.	42<SUP>o</SUP>C heat shock 30 sec.</LI><LI>4.	On ice 5 min.</LI><LI>5.	(In laminar flow) Add LB to 500 μl,37<SUP>o</SUP>C 170 rpm incubate 2 hour.</LI><LI>6.	Plate 200 μl ( 300 μl remained) spread over plate by spreader.</LI><LI>7.	Incubate the plate in 37<SUP>o</SUP>C overnight.</LI></UL></P></DIV><DIV id="pat"><H3>Patch and small scale culture</H3><P><UL class="”a”"><LI>1. Pick a colony by toothpick then “patch” on another agar plate as bacteria stock. Incubate in 37<SUP>o</SUP>C 14 hour~16 hour.</LI><LI>2. Dip the toothpick into 5 ml LB (with antibiotic) as small scale culture for plasmid extraction. Incubate in 37<SUP>o</SUP>C 170 rpm for 14 hour~16 hour.</LI></UL></P><P class="content-image" style="text-align:center !important;">Fig 1. Patch on agar.</P></DIV><DIV id="mini"><H3>Mini extraction</H3><P><UL class="”a”"><LI>Tthe protocol of commercial gel extraction kit (&quot;Biokit&quot;).</LI></UL></P></DIV><DIV id="cell"><H3>Cell lysis</H3><P><UL class="”a”"><LI>1. Centrifuge the 4 ml bacterial culture at 14,000 RCF, 1 min.</LI><LI>2. Resuspend the pellet with 1 ml phosphate buffer saline (PBS) to clean the residual medium.</LI><LI>3. Centrifuge the bacterial mixture at 14,000 RCF, 1 min.</LI><LI>4. Repeat the step 2 and 3 twice.</LI><LI>5. Resuspend the pellet with 0.4 ml PBS and add 2 μl 1 M PMSF.</LI><LI>6. Sonicate the bacterial mixture on ice with 3W output, pulse 30 sec and pause 30 sec, three cycle.</LI></UL></P></DIV><DIV id="dig"><H3>Digestion</H3><P><UL class="”a”"><LI>1. 0.5 μg DNA + 0.2 μl fast digestion restriction enzyme ( 10 unit/ μl, final 0.25 μg/unit) + 1 μl 10X buffer + mQ H<SUB>2</SUB>O → total volume 10 μl. </LI><LI>2. Bath in 37<SUP>o</SUP>C warm water for 15 min.</LI></UL></P></DIV><DIV id="lig"><H3>Ligation protocol</H3><P><UL class="”a”"><LI>1. Quantify the concentration of the linear DNAs that you are going to ligate.</LI><LI>2. Calculate the molar weight of DNA. Converse the unit into ng/ fmol.</LI><LI>3. Calculate how much nanogram is required for 20 fmol of backbone and 60 fmol of insert.</LI><LI>4. Then by the quantified concentration, find out that how much μl is required.</LI><LI>5. Adjust the total volume to 8.9 μl/tube.</LI><LI>6. Add 1 μl of 10x ligation buffer and 0.1 μl of T4 ligase (5 unit/μl)</LI><LI>7. Place it in 16<SUP>o</SUP>C water bath, overnight.</LI><LI>PS:</LI></UL><UL style="”list-style-image:none;list-style-type:" cirle=""><LI> The ideal volume for DNA ligation is 10 μl, but it is adding the volume to 15~20 μl is fine, since part of the liquid will vaporize during ligation.</LI><LI>It is recommended that the concentration for the backbone is 20 fmol.</LI><LI>The molar ratio for backbone : insert = 1 : 3 </LI><LI>The ligase required 0.1~0.5 unit/tube.</LI></UL></P></DIV><DIV id="run"><H3>Run electrophoresis gel</H3><P><UL class="”a”"><LI>1.	Make a suitable agarose gel (depend on your DNA size), X (g) agarose + Y (ml) 0.5X TAE , X/ Y = 1.2%~1.5% (&lt;0.5k), heating until all agarose melted.</LI><LI>2.	Cool down the agarose gel (use the back of your hand to check it, as long as it does not burn your hand, it is suitable temperature), add EtBr (final, 5 μl/ 100 ml) into agarose gel and mix well.</LI><LI>3.	Put agarose gel into mold and wait until it become solidification.</LI><LI>4.	Load samples into each well, the recipe is show below.</LI><LI>5.	Run gel in electrophoresis tank under 100 V.</LI></UL></P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Marker</TH><TH class="tg-yw4l">Sample</TH></TR><TR><TD class="tg-b7b8">Suitable ladder</TD><TD class="tg-b7b8">DNA</TD></TR><TR><TD class="tg-yw4l">2 μl</TD><TD class="tg-yw4l">10 μl</TD></TR><TR><TD class="tg-b7b8">6X dye</TD><TD class="tg-b7b8">2 μl</TD><TD class="tg-b7b8">2 μl</TD></TR><TR><TD class="tg-yw4l">mQ H2O</TD><TD class="tg-yw4l">8 μl</TD><TD class="tg-yw4l">0</TD></TR><TR><TD class="tg-b7b8">Total volume</TD><TD class="tg-b7b8">12 μl</TD><TD class="tg-b7b8">12 μl</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 1. Recipe for gel electrophoresis sample. </P><DIV id="pcr"><H3>PCR</H3><P>
Purpose: to check whether our plasmid has successfully transformed into <I>E. coli</I>, DH5α.<UL class="”a”"><LI>1.	PCR condition finding</LI></UL><UL style="”list-style:circle;padding-left:15" px=""><LI>  * P.S. Prepare the stock solution first, which consist of mQH<SUB>2</SUB>O、10x buffer、forward primer、backward primer、dNTP and DNA, then add the DNA polymerase before starting the PCR process.</LI><LI>0.5~1 tube shall be added to the ideal amount when preparing the stock solution.</LI><LI>It is recommended to dilute the amount polymerase needed to 1 unit/ μl, to decrease the error of adding small amount of volume.</LI></UL></P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">DNA (10 ng)</TH><TH class="tg-yw4l">10 ng</TH></TR><TR><TD class="tg-b7b8">10x buffer</TD><TD class="tg-b7b8">5 μl</TD></TR><TR><TD class="tg-yw4l">forward primer (2.5 μM)</TD><TD class="tg-yw4l">2.5 μl</TD></TR><TR><TD class="tg-b7b8">Reverse primer (2.5 μM)</TD><TD class="tg-b7b8">2.5 μl</TD></TR><TR><TD class="tg-yw4l">dNTP (10 μM)</TD><TD class="tg-yw4l">1 μl</TD></TR><TR><TD class="tg-b7b8">DNA polymerase (5 unit/μl)</TD><TD class="tg-b7b8">1~2 unit</TD></TR><TR><TD class="tg-yw4l">Total volume , add water to 50 μl</TD><TD class="tg-yw4l">50 μl</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 2. Recipe for PCR.</P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Final concentration</TH></TR><TR><TD class="tg-b7b8">DNA</TD><TD class="tg-b7b8">10 ng/ 50 μl</TD></TR><TR><TD class="tg-yw4l">Buffer</TD><TD class="tg-yw4l">1x</TD></TR><TR><TD class="tg-b7b8">Forward primer</TD><TD class="tg-b7b8">0.125 μM</TD></TR><TR><TD class="tg-yw4l">Backward primer</TD><TD class="tg-yw4l">0.125 μM</TD></TR><TR><TD class="tg-b7b8">dNTP</TD><TD class="tg-b7b8">0.2 μM</TD></TR><TR><TD class="tg-yw4l">DNA polymerase</TD><TD class="tg-yw4l">1~2 unit</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 3. Final concertation for PCR.</P><P><UL class="”a”"><LI>2.	Condition entering</LI></UL><UL class="”a”;padding-left:15px;"><LI>a.	94°C for 1 min.</LI><LI>b.	94°C for 30 sec</LI><LI>c.	The specific temperature for primer to bind on DNA, 30 sec (pSBBS1C 55°C, pSBBS4S 65°C)</LI><LI>d.	72°C for 1 min</LI><LI>e.	GOTO 2, 25 cycle</LI><LI>f. 	16°C for 10 min (to cool the machine)</LI><LI>g.	Run gel</LI><LI>h.	<A href="https://2017.igem.org/Team:CCU_Taiwan/Results" style="color:yellow">Results</A></LI></UL></P></DIV><DIV id="bs"><H3><I>B. subtilis</I> Competent Cell Preparation and Transformation</H3><P><UL class="”a”"><LI>1.	Streak recipient strain on LB Agar plate. Incubate overnight (18 hour) at 37°C. </LI><LI>2.	Inoculate a few colonies into 4.5 ml of Medium A in a 16×125 mm test tube that lacks visible scratches. Mix the contents of the tube thoroughly. Read its optical density at 600 nm in the spectrophotometer. Adjust the OD<SUB>600</SUB> to be 0.1-0.2, maintaining the volume at 4.5 ml. </LI><LI>3.	Incubate at 37°C with vigorous aeration. Read the OD<SUB>600</SUB> every 20 min, plotting OD<SUB>600</SUB> against time on semi-log paper. After a brief lag, the OD should increase logarithmically—that is, they should fall on a straight line. Note the point at the culture leaves log growth—the graph points fall below the straight line. In <I>B. subtilis</I> genetics, this point is known as t<SUB>0</SUB>. It should take 60~90 minutes of incubation and occur at OD<SUB>600</SUB>=0.4-0.6. </LI><LI>4.	Continue incubation for 90 minutes after the cessation of log growth (t<SUB>90</SUB>). Transfer 0.05 ml of this culture into 0.45 ml of pre-warmed Medium B in a 16×125 mm test tube. Set up one tube for each transformation you intend to perform, plus an extra for a DNA-less control (negative control). </LI><LI>5.	Incubate the diluted cultures at 37°C with vigorous aeration for 90 min. At this point, the cultures should be highly competent. </LI><LI>6.	Add 1 µg of DNA to the competent cells and incubate at 37°C with aeration for 30 minutes. </LI><LI>7.	Plate 200μl of the transformed cells onto selective agar.</LI></UL></P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Yeast extract</TH><TH class="tg-yw4l">10 g</TH></TR><TR><TD class="tg-b7b8">Casamino acids</TD><TD class="tg-b7b8">2 g</TD></TR><TR><TD class="tg-yw4l">Distilled water</TD><TD class="tg-yw4l">900 ml</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 4. Recipe of 10× Medium A base.</P><P>Autoclave, then add: 50% glucose, filter-sterilized 100 ml</P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></TH><TH class="tg-yw4l">20 g</TH></TR><TR><TD class="tg-yw4l">K<SUB>2</SUB>HPO<SUB>4</SUB>·P3H<SUB>2</SUB>O</TD><TD class="tg-yw4l">183 g</TD></TR><TR><TD class="tg-yw4l">KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD class="tg-yw4l">60 g</TD></TR><TR><TD class="tg-yw4l">Sodium citrate</TD><TD class="tg-yw4l">10 g</TD></TR><TR><TD class="tg-yw4l">MgSO<SUB>4</SUB>·P7H<SUB>2</SUB>O</TD><TD class="tg-yw4l">2 g</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 5. Recipe of 10× Bacillus salts.</P><P>Add ddwater to 1000 ml, autoclave 121°C, 1.2 atm for 60 minutes.</P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Sterile water</TH><TH class="tg-yw4l">81ml</TH></TR><TR><TD class="tg-b7b8">10× Medium A base</TD><TD class="tg-b7b8">10 ml</TD></TR><TR><TD class="tg-yw4l">10× Bacillus salts</TD><TD class="tg-yw4l">9 ml</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 6: Recipe of Medium A.</P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Medium A</TH><TH class="tg-yw4l">10 ml</TH></TR><TR><TD class="tg-b7b8">50 mM CaCl2<SUB>·2H<SUB>2</SUB>O</SUB></TD><TD class="tg-b7b8">0.1 ml</TD></TR><TR><TD class="tg-yw4l">250 nM MgCl<SUB>2</SUB>·6H<SUB>2</SUB>O</TD><TD class="tg-yw4l">0.1 ml</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 7. Recipe of Medium B.</P><P>*Need to add threonine (5 mg/ml) for strains carrying an insertion in thrC, 100μl/10 ml medium.</P><P><A href="https://2017.igem.org/Team:CCU_Taiwan/Results" style="color:yellow">Results</A></P><P>Reference<UL class="”a”"><LI>1. Bacillus Genetic Stock Center Catalog of Strains, Seventh Edition Volume 4: Integration Vectors for Gram-Positive Organisms, page 17</LI><LI>2. LMU-Munich_2012_Bacillus_subtilis_vectors</LI></UL></P></DIV><DIV id="ls"><H3>Lactate sensor Function test</H3><P>
As we mention before, we what to evaluate the risk of caries by detecting the three parameters in saliva: concentration of CSP (competence-stimulating peptide), lactate level and pH value. 
About our lactate sensor, the function what we need is not only detect lactate, but also quantize it. So we did this experiment to insμre that the GFP light intensity as different as lactate level.

</P><P><UL class="”a”"><LI>1.	Pre-culture in 15 ml test tube</LI><LI>Culture bacteria (<I>E. coli</I>-DH5α,pSB1A2-O1-J23118-O2-GFP-J23118-LldR) in 5 ml LB + 5 μl Amp<SUP>R</SUP> (100 μg/ ml, final), incubate in 37°C 14 hour.</LI><LI>Also pre-culture the bacteria that don't have fluoresces gene as negative control (<I>E. coli</I>-DH5α,pUCIDT-kan<SUP>R</SUP> –pBAD-mazEF).</LI><LI>2.	Refresh in 250 ml Erlenmeyer flask</LI><LI>20 ml LB + 400 μl pre-culture bacteria + 20 μl Amp<SUP>R</SUP> (100 μg/ml, final), 37°C, 170 rpm incubate for 90 min. Also refresh negative control.</LI><LI>3.	Transfer bacteria to 96 well plate.</LI></UL></P><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l" rowspan="2">Final concentration of lactate</TH><TH class="tg-yw4l" rowspan="2">Refreshed bacteria(per well)</TH><TH class="tg-yw4l" rowspan="2">1M lactate (per well)</TH><TH class="tg-yw4l" rowspan="2">mQ H2O (per well)</TH></TR><TR><TD class="tg-yw4l">Positive control</TD><TD class="tg-yw4l" colspan="2">0.012 μg/ml fluorescein 200 μl</TD><TD class="tg-yw4l">20 μl</TD></TR><TR><TD class="tg-b7b8">Negative control</TD><TD class="tg-b7b8" rowspan="7">200 μl</TD><TD class="tg-b7b8">0 μl</TD><TD class="tg-b7b8">20 μl</TD></TR><TR><TD class="tg-yw4l">0 mM</TD><TD class="tg-yw4l">0 μl</TD><TD class="tg-yw4l">20 μl</TD></TR><TR><TD class="tg-b7b8">2 mM</TD><TD class="tg-b7b8">4 μl</TD><TD class="tg-b7b8">16 μl</TD></TR><TR><TD class="tg-yw4l">4 mM</TD><TD class="tg-yw4l">8 μl</TD><TD class="tg-yw4l">12 μl</TD></TR><TR><TD class="tg-b7b8">6 mM</TD><TD class="tg-b7b8">12 μl</TD><TD class="tg-b7b8">8 μl</TD></TR><TR><TD class="tg-yw4l">8 mM</TD><TD class="tg-yw4l">16 μl</TD><TD class="tg-yw4l">4 μl</TD></TR><TR><TD class="tg-b7b8">10 mM</TD><TD class="tg-b7b8">20 μl</TD><TD class="tg-b7b8">0 μl</TD></TR><TR><TD class="tg-yw4l">Blank</TD><TD class="tg-yw4l">200 μl LB</TD><TD class="tg-yw4l">0 μl</TD><TD class="tg-yw4l">20 μl</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 8. Recipe of lactate sensor function test.</P><UL class="”a”"><LI>4. Add three glass beads for each well.</LI></UL><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">repeat</TH><TH class="tg-yw4l">1</TH><TH class="tg-yw4l">2</TH><TH class="tg-yw4l">3</TH><TH class="tg-yw4l">4</TH><TH class="tg-yw4l">5</TH><TH class="tg-yw4l">6</TH><TH class="tg-yw4l">7</TH><TH class="tg-yw4l">8</TH><TH class="tg-yw4l">9...</TH></TR><TR><TD class="tg-b7b8">1</TD><TD class="tg-b7b8">A</TD><TD class="tg-b7b8">positive</TD><TD class="tg-b7b8">negative</TD><TD class="tg-b7b8">0 mM</TD><TD class="tg-b7b8">2 mM</TD><TD class="tg-b7b8">4 mM</TD><TD class="tg-b7b8">6 mM</TD><TD class="tg-b7b8">8 mM</TD><TD class="tg-b7b8">10 mM</TD><TD class="tg-b7b8">blank</TD></TR><TR><TD class="tg-yw4l">2</TD><TD class="tg-yw4l">B</TD><TD class="tg-yw4l">positive</TD><TD class="tg-yw4l">negative</TD><TD class="tg-yw4l">0 mM</TD><TD class="tg-yw4l">2 mM</TD><TD class="tg-yw4l">4 mM</TD><TD class="tg-yw4l">6 mM</TD><TD class="tg-yw4l">8 mM</TD><TD class="tg-yw4l">10 mM</TD><TD class="tg-yw4l">blank</TD></TR><TR><TD class="tg-b7b8">3</TD><TD class="tg-b7b8">C</TD><TD class="tg-b7b8">positive</TD><TD class="tg-b7b8">negative</TD><TD class="tg-b7b8">0 mM</TD><TD class="tg-b7b8">2 mM</TD><TD class="tg-b7b8">4 mM</TD><TD class="tg-b7b8">6 mM</TD><TD class="tg-b7b8">8 mM</TD><TD class="tg-b7b8">10 mM</TD><TD class="tg-b7b8">blank</TD></TR></TBODY></TABLE><P class="content-image" style="text-align:center !important;">Table 9. Concentration of lactate in lactate sensor function test.</P><P><UL class="”a”"><LI>5. After finish loading plate, put them into incubator and fix well, 37°C, 170 rpm.   </LI></UL></P><P class="content-image" style="text-align:center !important;">Fig 2. 96 well plate for fluorescence.</P><P class="content-image" style="text-align:center !important;">Fig 3. 96 well plate forOD<SUB>600</SUB>.</P><P><UL class="”a”"><LI> (We didn't take a photo for our experimental plate. This plate is just for example, that is water in those well.)</LI><LI>6. Measurement:</LI><LI>Each 15 min, measure the fluorescence and OD<SUB>600</SUB> by plate reader and Elisa reader respectively.</LI></UL></P></DIV><DIV id="ft"><H3>Functional test for Bacillus subtilis</H3><P>
Supposedly, GFP should appear when competence stimulating peptide (CSP) was added. To make sure this function works properly in Bacillus subtilis, we've designed a functional test for it.<UL class="”a”"><LI>1. The transformed Bacillus subtilis which contains the target genes we need was incubated in LB medium at 25°C, overnight.</LI><LI>2. Measure the OD<SUB>600</SUB>, then dilute the bacterial culture to 0.4, 0.2, 0.1, and 0.05 in the value of OD<SUB>600</SUB>.</LI><LI>3. Dilute CSP into 4 different concentration – 1 mg/ml, 500 g/ml, 250 g/ml, 125 g/ml</LI><LI>4. Load the mixture above into a black microplate and a clear microplate as the picture shown below.</LI><LI>5. Measure the clear plate with the absorbance of wavelength 595 nm, and the black plate with a fluorescent plate reader (excitation: 485 nm, emission: 520 nm ) every 15 minutes.</LI><LI>6. <A href="https://2017.igem.org/Team:CCU_Taiwan/Results" style="color:yellow">Results</A></LI></UL></P><P class="content-image" style="text-align:center !important;">Fig 4. 96 well plate for <I>B. subtilis</I> function test.</P><P>
For G1 ~ G3: Bacterial culture OD<SUB>600</SUB> = 0.4, For G4 ~ G6: Bacterial culture OD<SUB>600</SUB> = 0.2, For G7 ~ G9: Bacterial culture OD<SUB>600</SUB> = 0.1, For G10 ~ G12: Bacterial culture OD<SUB>600</SUB> = 0.05, CSP-free for line G and H</P></DIV></DIV></SECTION></ARTICLE></DIV></HEADER></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>