Team:CCU Taiwan/Safety
No Sidebar - Helios by HTML5 UP
Safety
Home
Project
Description
Biosensor
CSP detector
Lactate detector
Test paper
Experiments
Results
Demonstrate
Applied design
InterLab
Notebook
Hardware
Hardware overview
Device design
Device detection
Software
IOT system
APP
Machine learning
Modeling
Human practice
Human practice overview
Silver HP
Integrate & Gold
Public engagemant
Entrepreneurship
Safety
Parts
Parts
Basic parts
Composite parts
Team
Team members
Attributions
Collaborations
Medals
Overview
Organism we used
B. subtilis
E. coli
References
Overview
Genetic modified organism escaping from lab is a serious problem since it will bring unpredictable impact to our ecosystem. In our part design we cared a lot about biosafety, not only we applied the very effective bacteria suicide mechanism but prudently chose the organism that slightly harm to environment. We also cautiously deal with waste we produced after experiment and participated in the safety training to ensure everyone who conducting experiment have minimal probability to cause biohazard pollution.
Organism we used
Two type of chassis we chose are E. coli DH5α & B. subtilis strain 168(ATCC ® 23857).
B. subtilis suicide mechanism
The B. subtilis isn’t harmful for human according to the Globally Harmonized System (GHS). We used two plasmids, pSBBS1C and pSBBS4S which can lead B. subtilis to have two gene defects, disability to decomposing starch and synthesizing threonine. When pSBBS4S is a backbone that would inserted into thrC locus, lest B. subtilis synthesize the necessary amino acid, threonine.
Another plasmid, pSBBS1C, would be inserted into amyE locus, the function of decomposing starch would be knocked down. According to this reason, we can select which colony we want with starch test (figures shown below). These defects cause B. subtilis has little chance to survive when it escaped from our lab or device.
Figure 1
If the plasmid (pSBBS1C) is successively transformed to B. subtilis, it lost the ability of decomposing starch, and those that we need (the left).
E. coli suicide mechanism
We introduced a MazEF suicide system into E. coli. This system is induced by arabinose. MazEF is a toxin-antitoxin system including two overlapping gene segment－ mazE & mazF, where mazF is a stable toxin and mazE is a labile antitoxin. MazF sequence will translate the sequence-specific mRNA endoribonuclease mazF which will cut certain mRNA down, inhibiting bulk protein synthesis and repress cell mitosis. Relatively, mazE is the antitoxin for mazF and neutralize the toxicity of mazF. In general, continuous generation of MazE can prevent the situation of cell death by antagonizing mazF. However, once mazE ceased to be produced, it would be degraded by the protease ClpAP. Hence, the remaining toxin mazF will cause the cell death by cleaving mRNA.
Figure 2 MazEF function test - Test 1
Blue line: Negative control; Orange line: arabinose conc.=130 μM;Gray line: arabinose conc.=13μM.
In the beginning, we add arabinose to the E. coli, and the kill switch start. During the first hour, mazE and mazF are produced. Because of the toxin-antitoxin characteristic, E. coli grows normally in the first hour. After one hour, the mazE and mazF concentrations reach saturation, and then start degrading. Compared with mazF, mazE degrades faster, so mazF cut mRNA for important protein. Finally, E. coli cannot survive. The OD600 value we observe is that it maintains a stable value.
Figure 3 MazEF function test - Test 2
The Agar plate that is result of mazEF function test-survival test at 5 hour.
Figure 4 MazEF function test - Test 2
Following five hours, we test the survival rate every hour (we only see result of Negative control and 6 mM Arabinose, because kill switch is independent on concentration of arabinose.)
By adding different concentration of arabinose, we found the most appropriate arabinose concentration to induce the suicide system. After five hours, almost all E. coli dead.
References
1. (Amitai, Yassin, & Engelberg-Kulka, 2004; Curtis, Takeuchi, Gram, & Knudsen, 2017; Muñoz-Gómez, Lemonnier, Santos-Sierra, Berzal-Herranz, & Díaz-Orejas, 2005; Ogden, Haggerty, Stoner, Kolodrubetz, & Schleif, 1980; Siegele & Hu, 1997)Amitai, S., Yassin, Y., & Engelberg-Kulka, H. (2004). MazF-mediated cell death in Escherichia coli: a point of no return. Journal of bacteriology, 186(24), 8295-8300.
2. Curtis, T. D., Takeuchi, I., Gram, L., & Knudsen, G. M. (2017). The Influence of the Toxin/Antitoxin mazEF on Growth and Survival of Listeria monocytogenes under Stress. Toxins, 9(1), 31.
3. Muñoz-Gómez, A. J., Lemonnier, M., Santos-Sierra, S., Berzal-Herranz, A., & Díaz-Orejas, R. (2005). RNase/anti-RNase activities of the bacterial parD toxin-antitoxin system. Journal of bacteriology, 187(9), 3151-3157.
4. Ogden, S., Haggerty, D., Stoner, C. M., Kolodrubetz, D., & Schleif, R. (1980). The Escherichia coli L-arabinose operon: binding sites of the regulatory proteins and a mechanism of positive and negative regulation. Proceedings of the National Academy of Sciences, 77(6), 3346-3350.
5. Siegele, D. A., & Hu, J. C. (1997). Gene expression from plasmids containing the araBAD promoter at subsaturating inducer concentrations represents mixed populations. Proceedings of the National Academy of Sciences, 94(15), 8168-8172.
6. https://www.google.com.tw/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwidybOJ453
XAhVKjZQKHQCxAiAQFgglMAA&url=http%3A%2F%2Fparts.igem.org%2FPart%3ABBa_K82
3022&usg=AOvVaw3wqpYa_OyUmZZ0XIbJFh1V
7. https://www.google.com.tw/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwibqJyT453XAhX
Fk5QKHa7dDE4QFggvMAE&url=http%3A%2F%2Fparts.igem.org%2FPart%3ABB
a_K823023&usg=AOvVaw0eK-1WvdH4R7eB9bWNmXIH
