Team:CCU Taiwan/Experiments
No Sidebar - Helios by HTML5 UP
Experiments
Home
Project
Description
Biosensor
CSP detector
Lactate detector
Test paper
Experiments
Results
Demonstrate
Applied design
InterLab
Notebook
Hardware
Hardware overview
Device design
Device detection
Software
IOT system
APP
Machine learning
Modeling
Human practice
Human practice overview
Silver HP
Integrate & Gold
Public engagemant
Entrepreneurship
Safety
Parts
Parts
Basic parts
Composite parts
Team
Team members
Attributions
Collaborations
Medals
Experiment
Transformation
Patch and small scale culture
Mini extraction:
Cell lysis
Digestion
Ligation protocol
Run electrophoresis gel
PCR
Competent Cell
Lactate sensor Function test
Functional test for Bacillus subtilis
Experiment
General protocol
Transformation ( by 1.5 ml Eppendorf tube)
1. 2.5 μg DNA + 25 μl competent cell (ECOS 101 DH5α).
2. On ice 30 min.
3. 42oC heat shock 30 sec.
4. On ice 5 min.
5. (In laminar flow) Add LB to 500 μl,37oC 170 rpm incubate 2 hour.
6. Plate 200 μl ( 300 μl remained) spread over plate by spreader.
7. Incubate the plate in 37oC overnight.
Patch and small scale culture
1. Pick a colony by toothpick then “patch” on another agar plate as bacteria stock. Incubate in 37oC 14 hour~16 hour.
2. Dip the toothpick into 5 ml LB (with antibiotic) as small scale culture for plasmid extraction. Incubate in 37oC 170 rpm for 14 hour~16 hour.
Fig 1. Patch on agar.
Mini extraction
Tthe protocol of commercial gel extraction kit ("Biokit").
Cell lysis
1. Centrifuge the 4 ml bacterial culture at 14,000 RCF, 1 min.
2. Resuspend the pellet with 1 ml phosphate buffer saline (PBS) to clean the residual medium.
3. Centrifuge the bacterial mixture at 14,000 RCF, 1 min.
4. Repeat the step 2 and 3 twice.
5. Resuspend the pellet with 0.4 ml PBS and add 2 μl 1 M PMSF.
6. Sonicate the bacterial mixture on ice with 3W output, pulse 30 sec and pause 30 sec, three cycle.
Digestion
1. 0.5 μg DNA + 0.2 μl fast digestion restriction enzyme ( 10 unit/ μl, final 0.25 μg/unit) + 1 μl 10X buffer + mQ H2O → total volume 10 μl.
2. Bath in 37oC warm water for 15 min.
Ligation protocol
1. Quantify the concentration of the linear DNAs that you are going to ligate.
2. Calculate the molar weight of DNA. Converse the unit into ng/ fmol.
3. Calculate how much nanogram is required for 20 fmol of backbone and 60 fmol of insert.
4. Then by the quantified concentration, find out that how much μl is required.
5. Adjust the total volume to 8.9 μl/tube.
6. Add 1 μl of 10x ligation buffer and 0.1 μl of T4 ligase (5 unit/μl)
7. Place it in 16oC water bath, overnight.
PS:
The ideal volume for DNA ligation is 10 μl, but it is adding the volume to 15~20 μl is fine, since part of the liquid will vaporize during ligation.
It is recommended that the concentration for the backbone is 20 fmol.
The molar ratio for backbone : insert = 1 : 3
The ligase required 0.1~0.5 unit/tube.
Run electrophoresis gel
1. Make a suitable agarose gel (depend on your DNA size), X (g) agarose + Y (ml) 0.5X TAE , X/ Y = 1.2%~1.5% (<0.5k), heating until all agarose melted.
2. Cool down the agarose gel (use the back of your hand to check it, as long as it does not burn your hand, it is suitable temperature), add EtBr (final, 5 μl/ 100 ml) into agarose gel and mix well.
3. Put agarose gel into mold and wait until it become solidification.
4. Load samples into each well, the recipe is show below.
5. Run gel in electrophoresis tank under 100 V.
Marker
Sample
Suitable ladder
DNA
2 μl
10 μl
6X dye
2 μl
2 μl
mQ H2O
8 μl
0
Total volume
12 μl
12 μl
Table 1. Recipe for gel electrophoresis sample.
PCR
Purpose: to check whether our plasmid has successfully transformed into E. coli, DH5α.
1. PCR condition finding
  * P.S. Prepare the stock solution first, which consist of mQH2O、10x buffer、forward primer、backward primer、dNTP and DNA, then add the DNA polymerase before starting the PCR process.
0.5~1 tube shall be added to the ideal amount when preparing the stock solution.
It is recommended to dilute the amount polymerase needed to 1 unit/ μl, to decrease the error of adding small amount of volume.
DNA (10 ng)
10 ng
10x buffer
5 μl
forward primer (2.5 μM)
2.5 μl
Reverse primer (2.5 μM)
2.5 μl
dNTP (10 μM)
1 μl
DNA polymerase (5 unit/μl)
1~2 unit
Total volume , add water to 50 μl
50 μl
Table 2. Recipe for PCR.
Final concentration
DNA
10 ng/ 50 μl
Buffer
1x
Forward primer
0.125 μM
Backward primer
0.125 μM
dNTP
0.2 μM
DNA polymerase
1~2 unit
Table 3. Final concertation for PCR.
2. Condition entering
a. 94°C for 1 min.
b. 94°C for 30 sec
c. The specific temperature for primer to bind on DNA, 30 sec (pSBBS1C 55°C, pSBBS4S 65°C)
d. 72°C for 1 min
e. GOTO 2, 25 cycle
f.  16°C for 10 min (to cool the machine)
g. Run gel
h. Results
B. subtilis Competent Cell Preparation and Transformation
1. Streak recipient strain on LB Agar plate. Incubate overnight (18 hour) at 37°C.
2. Inoculate a few colonies into 4.5 ml of Medium A in a 16×125 mm test tube that lacks visible scratches. Mix the contents of the tube thoroughly. Read its optical density at 600 nm in the spectrophotometer. Adjust the OD600 to be 0.1-0.2, maintaining the volume at 4.5 ml.
3. Incubate at 37°C with vigorous aeration. Read the OD600 every 20 min, plotting OD600 against time on semi-log paper. After a brief lag, the OD should increase logarithmically—that is, they should fall on a straight line. Note the point at the culture leaves log growth—the graph points fall below the straight line. In B. subtilis genetics, this point is known as t0. It should take 60~90 minutes of incubation and occur at OD600=0.4-0.6.
4. Continue incubation for 90 minutes after the cessation of log growth (t90). Transfer 0.05 ml of this culture into 0.45 ml of pre-warmed Medium B in a 16×125 mm test tube. Set up one tube for each transformation you intend to perform, plus an extra for a DNA-less control (negative control).
5. Incubate the diluted cultures at 37°C with vigorous aeration for 90 min. At this point, the cultures should be highly competent.
6. Add 1 µg of DNA to the competent cells and incubate at 37°C with aeration for 30 minutes.
7. Plate 200μl of the transformed cells onto selective agar.
Yeast extract
10 g
Casamino acids
2 g
Distilled water
900 ml
Table 4. Recipe of 10× Medium A base.
Autoclave, then add: 50% glucose, filter-sterilized 100 ml
(NH4)2SO4
20 g
K2HPO4·P3H2O
183 g
KH2PO4
60 g
Sodium citrate
10 g
MgSO4·P7H2O
2 g
Table 5. Recipe of 10× Bacillus salts.
Add ddwater to 1000 ml, autoclave 121°C, 1.2 atm for 60 minutes.
Sterile water
81
ml
10× Medium A base
10 ml
10× Bacillus salts
9 ml
Table 6: Recipe of Medium A.
Medium A
10 ml
50 mM CaCl2·2H2O
0.1 ml
250 nM MgCl2·6H2O
0.1 ml
Table 7. Recipe of Medium B.
*Need to add threonine (5 mg/ml) for strains carrying an insertion in thrC, 100μl/10 ml medium.
Results
Reference
1. Bacillus Genetic Stock Center Catalog of Strains, Seventh Edition Volume 4: Integration Vectors for Gram-Positive Organisms, page 17
2. LMU-Munich_2012_Bacillus_subtilis_vectors
Lactate sensor Function test
As we mention before, we what to evaluate the risk of caries by detecting the three parameters in saliva: concentration of CSP (competence-stimulating peptide), lactate level and pH value.
About our lactate sensor, the function what we need is not only detect lactate, but also quantize it. So we did this experiment to insμre that the GFP light intensity as different as lactate level.
1. Pre-culture in 15 ml test tube
Culture bacteria (E. coli-DH5α,pSB1A2-O1-J23118-O2-GFP-J23118-LldR) in 5 ml LB + 5 μl AmpR (100 μg/ ml, final), incubate in 37°C 14 hour.
Also pre-culture the bacteria that don't have fluoresces gene as negative control (E. coli-DH5α,pUCIDT-kanR –pBAD-mazEF).
2. Refresh in 250 ml Erlenmeyer flask
20 ml LB + 400 μl pre-culture bacteria + 20 μl AmpR (100 μg/ml, final), 37°C, 170 rpm incubate for 90 min. Also refresh negative control.
3. Transfer bacteria to 96 well plate.
Final concentration of lactate
Refreshed bacteria(per well)
1M lactate (per well)
mQ H2O (per well)
Positive control
0.012 μg/ml fluorescein 200 μl
20 μl
Negative control
200 μl
0 μl
20 μl
0 mM
0 μl
20 μl
2 mM
4 μl
16 μl
4 mM
8 μl
12 μl
6 mM
12 μl
8 μl
8 mM
16 μl
4 μl
10 mM
20 μl
0 μl
Blank
200 μl LB
0 μl
20 μl
Table 8. Recipe of lactate sensor function test.
4. Add three glass beads for each well.
repeat
1
2
3
4
5
6
7
8
9...
1
A
positive
negative
0 mM
2 mM
4 mM
6 mM
8 mM
10 mM
blank
2
B
positive
negative
0 mM
2 mM
4 mM
6 mM
8 mM
10 mM
blank
3
C
positive
negative
0 mM
2 mM
4 mM
6 mM
8 mM
10 mM
blank
Table 9. Concentration of lactate in lactate sensor function test.
5. After finish loading plate, put them into incubator and fix well, 37°C, 170 rpm.
Fig 2. 96 well plate for fluorescence.
Fig 3. 96 well plate forOD600.
(We didn't take a photo for our experimental plate. This plate is just for example, that is water in those well.)
6. Measurement:
Each 15 min, measure the fluorescence and OD600 by plate reader and Elisa reader respectively.
Functional test for Bacillus subtilis
Supposedly, GFP should appear when competence stimulating peptide (CSP) was added. To make sure this function works properly in Bacillus subtilis, we've designed a functional test for it.
1. The transformed Bacillus subtilis which contains the target genes we need was incubated in LB medium at 25°C, overnight.
2. Measure the OD600, then dilute the bacterial culture to 0.4, 0.2, 0.1, and 0.05 in the value of OD600.
3. Dilute CSP into 4 different concentration – 1 mg/ml, 500 g/ml, 250 g/ml, 125 g/ml
4. Load the mixture above into a black microplate and a clear microplate as the picture shown below.
5. Measure the clear plate with the absorbance of wavelength 595 nm, and the black plate with a fluorescent plate reader (excitation: 485 nm, emission: 520 nm ) every 15 minutes.
6. Results
Fig 4. 96 well plate for B. subtilis function test.
For G1 ~ G3: Bacterial culture OD600 = 0.4, For G4 ~ G6: Bacterial culture OD600 = 0.2, For G7 ~ G9: Bacterial culture OD600 = 0.1, For G10 ~ G12: Bacterial culture OD600 = 0.05, CSP-free for line G and H
