<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_testtest skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/testtest</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/6/61/Ox_biofilmdrawing.jpeg)"><H3>Description</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="section" id="overview"><DIV class="slim"><H2>Overview</H2><DIV class="quote quote-full"><P>
                            &quot;Years from now we will treat most infections with bacteria and not antibiotics.&quot;
                        </P><H3>Professor James Malone-LeeBarlow Professor of Geriatric Medicine</H3></DIV><P>
                        A solution is urgently needed for the increasing number of infections caused by antibiotic-resistant bacteria. The engineering of bacterial cells to fight human pathogens is a promising alternative to administering traditional antibiotics. Our project involves the use of synthetic biology to engineer living therapeutics that have the potential to treat urinary tract infections (UTIs), which are a major public health concern in developed countries. This is largely due to growing antibiotic resistance.
                    </P></DIV></DIV><DIV class="section" id="problem"><DIV class="slim"><H2>The Problem</H2><UL><LI>Antibiotics have negative side effects and their resistance is a growing problem</LI><LI>UTIs are the <STRONG>most common hospital-acquired infection</STRONG> and the bacteria that cause them are frequently <STRONG>resistant to antibiotics</STRONG></LI><LI>Antibiotic resistance in UTIs is caused by <STRONG>biofilms</STRONG></LI><LI>Biofilms are currently estimated to be responsible for over 65% of all hospital-acquired infections</LI><LI><STRONG>Current UTI treatments are ineffective</STRONG> and fail to prevent recurrent infections</LI></UL></DIV></DIV><DIV class="section" id="antibio"><DIV class="slim"><H2>Problem with Antibiotics</H2><P>
                        Antibiotic use is associated with numerous negative side effects, allergies and reactions. The most common side effects of antibiotics all impact the digestive system and occur in around one in ten people. <STRONG>Around one person in fifteen has an allergic reaction to antibiotics</STRONG>, especially penicillin and cephalosphorins [<A href="#references">10</A>]. Half of the patients we spoke to during our project wanted us to find an alternative to using antibiotics, owing to the severity of the negative side effects that they experience.
                    </P><P>
                        Despite their negative side effects, antibiotics have been used so widely and for so long that the <STRONG>targeted microbes have adapted to become resistant</STRONG>, reducing the effectiveness of the drugs. A vicious cycle ensues in which ineffective antibiotic treatments leads to overprescription and overexposure, which amplifies the problem of antibiotic resistance. Global antibiotic consumption grew by 30% between 2000 and 2010. [<A href="#references">27</A>] Described by the Chief Medical Officer of England as “a threat equal to that of terrorism”, the growing resistance to antibiotics is hindering the effective prevention and treatment of an ever increasing range of infections.
                    </P><DIV class="quote quote-full"><P>
                            antibiotic resistance poses a catastrophic threat. If we don't act now, any one of us could go into hospital in 20 years for minor surgery and die because of an ordinary infection that can't be treated by antibiotics.
                        </P><H3>Professor Dame Sally DaviesChief Medical OfficerMarch 2013</H3></DIV><P>
                        The World Health Organisation’s antibiotic resistance Global Report on Surveillance, reports increasing worldwide incidences of antimicrobial resistance, in particular antibiotic resistance. This highlights very high rates of resistance in bacteria that cause common healthcare associated and community-acquired infections, such as UTIs [<A href="#references">1</A>]. UTIs account for over 7 million doctor visits per year. <STRONG>Catheter associated UTIs (CAUTIs) are the most commonly acquired infection in hospitals, and there is a high incidence of antibiotic resistance in the bacteria that cause UTIs globally</STRONG> [<A href="#references">2</A>].
                    </P></DIV></DIV><DIV class="section" id="biofilms"><DIV class="slim"><H2>Biofilms</H2><DIV class="image image-full"><P>
                                Major structural elements of bacterial biofilms.
                            </P></DIV><P>
                        Biofilms are aggregates of surface-associated microorganisms that are encased in a matrix of highly-hydrated extracellular polymeric substances, which include extracellular polysaccharides, extracellular DNA, as well as proteins [<A href="#references">6</A>]. Van Leeuwenhoek, using his simple microscopes in 1684, first observed microorganisms on tooth surfaces and can be credited with the discovery of microbial biofilms. &quot;The number of these animalcules in the scurf of a man's teeth are so many that I believe they exceed the number of men in a kingdom.&quot; - Leeuwenhoek
                    </P><P>
                        We now know a great deal more about biofilms. Environmental changes are responsible for the transition from planktonic growth to biofilm [<A href="#references">13</A>] and cause changes in the expression of surface molecules, virulence factors, and metabolic status. This allows the bacteria to acquire properties that enable their survival in unfavourable conditions [<A href="#references">14,15</A>], such as in the presence of antibiotics.
                    </P><P>
                         The low nutrient and oxygen levels at the bottom of the biofilm give rise to metabolically-inactive bacteria, better known as persister cells. <STRONG>These persister cells are rendered unsusceptible to most traditional antibiotics, which rely on bacterial metabolism to exert cell-killing effect</STRONG> [<A href="#references">7</A>].
                    </P></DIV></DIV><DIV class="section" id="utis"><DIV class="slim"><H2>UTIs</H2><P>
                        Biofilms are currently estimated to be responsible for over 65% of nosocomial infections and 80% of all microbial infections [<A href="#references">16</A>]. Bacterial biofilms play an important role in UTIs. UTIs are caused by the pathogenic invasion of the urinary tract, which causes an inflammatory response of the urothelium.
                    </P><P>
                        It is estimated that approximately 40% of women have had a UTI at some time in their lives [<A href="#references">10</A>]. UTIs may be caused by a variety of different organisms, most commonly bacteria. The most frequent cause of UTI in adult women is <EM>Escherichia coli</EM>, accounting for approximately 85% of community-acquired UTIs and 25-50% of hospital-acquired UTIs. Nosocomial infections may involve more aggressive organisms such as <EM>Pseudomonas aeruginosa</EM> and <EM>Enterobacter</EM> species.
                    </P></DIV></DIV><DIV class="section" id="solution"><DIV class="slim"><H2>The Solution</H2><UL><LI>Break down bacterial biofilms to liberate the bacteria encased within and reduce the dose of antibiotics required</LI><LI>Directly kill the bacteria encased within the biofilms to provide an alternative to antibiotics</LI></UL></DIV><DIV id="solution-overview"><DIV class="slim"><H3>Overview</H3><P>
                            Our solution is focused on providing a treatment for UTIs because conventional antibiotics are unable to treat these and other biofilm-associated infections. Given the prevalence of such infections, there is a growing need for alternative therapeutic agents that can specifically degrade biofilms and kill the bacteria encased within. The use of synthetic biology to produce enzymes is the most effective way to achieve this specificity based on current technology. Our solution aims to investigate how bacterial biofilm disrupting proteins and antimicrobial proteins can be exported from <EM>E. coli</EM> and subsequently retain their antibiofilm/antimicrobial function. Using this secretion device we seek to create a system that offers persistent protection against biofilm formation.
                            
                        </P><P>
                            We have designed a device that can exert antibiofilm and antimicrobial activity against <EM>E. coli</EM> and <EM>P. aeruginosa</EM>, the two leading causes of CAUTIs [<A href="#references">4</A>]. A nonpathogenic laboratory strain of <EM>E. coli</EM> is used as the expression host for the production of these enzymes as a proof-of-concept. The antibiofilm enzymes that we are using are Dispersin B and Micrococcal DNase, and the antimicrobial proteins that we are using are Art-175 and Microcin S.
                        </P></DIV></DIV><DIV id="solution-degrading-biofilm"><DIV class="slim"><H3>Degrading the Biofilm</H3><P>
                            Prof. Malone-Lee stressed to us that sensitivity is a greater problem than complete antibiotic resistance. “Many more strains of bacteria are just insensitive to low doses of antibiotics, many can be overcome by high doses over long periods of time. Resistance is definitely not absolute.” Breaking down the biofilm increases the sensitivity of the bacteria embedded within it. Planktonic bacteria are metabolically active and are thus prone to antibiotics, meaning that lower doses are required.
                        </P><P>
                            DspB (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659200">BBa_K1659200</A>) is an enzyme produced by <EM>Aggregatibacter actinomycetemcomitans</EM>, a species of bacteria found in the human oral cavity that grows almost exclusively in the form of biofilms. Structural analysis of Dispersin B showed that the enzyme only works specifically against the β-1,6-glycosidic linkages found in poly-N-acetylglucosamine, which is a polysaccharide structural element found in the biofilms of <EM>E. coli</EM> but not in <EM>P. aeruginosa</EM>. An additional enzyme would need to be used to target the polysaccharide component of <EM>P. aeruginosa</EM> biofilms.
                        </P><P>
                            Micrococcal DNase (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659300">BBa_K1659300</A>) is an endo-exonuclease that non-specifically catalyzes the hydrolysis of single- and double-stranded DNA under basic conditions and in the presence of Ca<SUP>2+</SUP> ions, and is known to be able to speed up DNA hydrolysis by up to 1016 times [<A href="#references">17</A>]. We are using DNase to break down the extracellular DNA component of biofilms.
                        </P></DIV></DIV><DIV id="solution-killing-bacteria"><DIV class="slim"><H3>Killing the Bacteria</H3><P>
                            Although antibiotic resistance is not absolute, it does pose a very big threat to the effective treatment of many infections. The insensitivity of bacteria to antibiotics can also be attributed to increasing antibiotic resistance. As described above, antibiotics also have many side effects that reduce patient quality of life and decrease the likelihood of completing a course of antibiotics. With all of this in mind, our solution does not only break down the biofilm, but also kills the bacteria embedded within so as to provide an alternative to antibiotics.
                        </P><DIV id="solution-killing-bacteria-art175"><H4>Art-175</H4><P>
                                Art-175 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659000">BBa_K1659000</A>) derive their name from “artificial endolysins”. Endolysins are bacteriophage-encoded peptidoglycan hydrolases that pass through the cytoplasmic membrane, degrading the peptidoglycan layer and inducing the lysis of the infected cell.
                            </P></DIV><DIV id="solution-killing-bacteria-mccs"><H4>MccS</H4><P>
                                MccS (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659100">BBa_K1659100</A>) is a type of microcin, a subclass of antibacterial proteins known as bacteriocins. Microcins are small, enterobacteria-produced bacteriocins that exert antibacterial activity against closely-related species, and MccS is produced by <EM>E. coli</EM> present in the probiotic drug Symbioflor 2 that has been shown to successfully treat gastrointestinal disorders.
                         </P><P>
                                For more information, please visit our <A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A> page.
                            </P></DIV><P>
                            Current clinically-relevant pathogens have not been seen to exhibit resistance against our antimicrobial proteins of choice. Art-175 has been experimentally shown to be not susceptible to resistance development, likely because it targets the structural element of the bacterial cell wall that is highly conserved across species and difficult to mutate [<A href="#references">8</A>]. The mechanism by which Microcin S exerts antimicrobial activity is still currently unknown, but no bacterial strains except for the original strain of probiotic <EM>E. coli</EM> which produces Microcin S has been shown to be resistant to it thus far [<A href="#references">9</A>].
                        </P></DIV></DIV><DIV id="solution-secreting"><DIV class="slim"><H3>Secreting the Proteins</H3><P>
                            The proven secretion of folded, functional proteins across both bacterial cell membranes is a challenge for present day microbiologists. Our solution requires that we can export DspB, DNase, Art-175 and MccS out of the expression host and into the local biofilm environment. To achieve this, signal sequences are fused to the enzymes to target them for export through natural <EM>E. coli</EM> secretion pathways. Using this mechanism we can direct our anti-biofilm and antimicrobial agents at a biofilm infected surface as they are being produced.
                        </P><DIV id="solution-sectreting-dsba"><H4>DsbA</H4><P>
                                DsbA is a oxidoreductase protein found predominantly in Gram-negative bacteria, which functions as a protein-folding factor [<A href="#references">19, 20</A>]. The 2-19 peptide sequence of DsbA is a signal sequence that can direct passenger proteins for co-translational export via the signal recognition particle pathway [<A href="#references">21, 22</A>]. It has recently been shown that the DsbA signal sequence is capable of mediating passenger protein secretion under a selection of different induction temperatures [<A href="#references">23</A>].
                            </P><P>
                                Parts:  <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659002">BBa_K1659002</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659201">BBa_K1659201</A>,<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659301">BBa_K1659301</A></P></DIV><DIV id="solution-sectreting-yebf"><H4>YebF</H4><P>
                                YebF is a 13kDa protein of unknown function that is perhaps the only protein that has been conclusively documented to be secreted into the extracellular medium by a laboratory <EM>E. coli</EM> strain. At the N-terminus, YebF has a 2.2 kDa sec-leader sequence which mediates its translocation through the bacterial inner membrane via the Sec pathway, and is cleaved upon translocation into the periplasm to give the 10.8 kDa &quot;mature&quot; form [<A href="#references">24</A>]. Export from periplasm into the extracellular space takes places via the Omp pathway. YebF has been used successfully to mediate the secretion of recombinant proteins [<A href="#references">25,26</A>].
                            </P><P>
                                Part: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659003">BBa_K1659003</A></P></DIV><DIV id="solution-secreting-fla"><H4>Fla</H4><P>Flagellin are the constituent subunits of the helical filament substructure of bacterial flagella. In the flagellar-building process, flagellin are exported out of the cell sequentially by the flagellum-specific export apparatus. F. Vonderviszt et al. demonstrated through their work that the signal sequence responsible for allowing the flagellar export system to identify and export Salmonella flagellin is its 26-47 amino acid residue segment [<A href="#references">18</A>].</P><P>
                                Part: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659001">BBa_K1659001</A></P></DIV></DIV></DIV></DIV><DIV class="section" id="delivery"><DIV class="slim"><H2>Delivery</H2><P>
                        A major part of our project involves investigating what is the best way to deliver our biofilm-degrading and antimicrobial enzymes to the site of infection in the urinary tract.
                    </P><P>
                        As we have mentioned above, patients with recurrent, complicated cases of UTI often get their infections from an already-inserted catheter which may have to be there and cannot be removed for a variety of other medical reasons. In view of that, we decided to conceptualize an initial delivery method which was centered on the catheter.
                    </P><P>
                        Our AlgiBeads design involves encapsulating our therapeutic, enzyme-secreting bacteria in sodium alginate beads. These beads are immobilized in a modified section of a catheter, from which the bacteria can secrete the therapeutic enzymes into the infected urinary tract. On our <A href="https://2015.igem.org/Team:Oxford/Design">Design</A> page, thorough consideration was given to the AlgiBeads delivery method, including issues of safety and practicality.
                    </P><P>
                        However, based on some preliminary data obtained for gene expression and diffusion rates, our <A href="https://2015.igem.org/Team:Oxford/Modeling">computational models</A> predicted that the equilibrium concentration of enzymes in solution based on the AlgiBeads delivery method would be too low when compared against the known concentrations required for biofilm degradation.
                    </P><P>
                        As such, we have had to instead consider an alternative delivery method - the introduction of our enzyme-releasing therapeutic engineered bacteria into the urinary microbiome, whereby the problem of low enzyme concentration in solution will be overcome by the close proximity between the therapeutic bacteria and the pathogenic bacteria. Another benefit of having therapeutic bacteria as part of the microbiome is of course that the treatment becomes preventive in nature, with the therapeutic bacteria now part of the bacterial community in the body constantly releasing pathogen-killing enzymes.
                    </P><P>
                        Of course, altering the microbiome comes with its own set of hazards, and we hope to mitigate it at least in part by doubling up the pathogen-killing mechanism as a population control mechanism for the engineered bacteria as well:
                    </P><DIV class="image image-full" style="object-align:center"><P>
                            How our 3-part engineered microbe works:
                                1. Constant secretion of biofilm-degrading enzyme
                                2. Production and accumulation of antibacterial Art-175
                                3. Detection of pathogenic bacteria via quorum sensing
                                4. Permeabilization of inner membrane by T4 Holin
                                5. Access and lysis of host cell wall by Art-175
                                6. Release of Art-175 and lysis of target cell
                            
                        </P></DIV><P>
                        Art-175 is normally prevented from reaching the cell wall of the expression host by the inner membrane. When a large amount of pathogenic bacteria is present, the quorum sensing signals trigger the production of T4 Holin, which permeabilizes the inner membrane, allowing Art-175 to reach the cell wall and degrade it. This causes lysis of the host cell and releases the accumulated Art-175 in a single high-concentration pulse, killing the pathogenic bacteria and achieving population control of the expression host at the same time.
                    </P></DIV><DIV class="section" id="results"><DIV class="slim"><H2>Results</H2><P>
                        Through our experimental work we were able to obtain preliminary evidence suggesting the validity of these points:
                    </P><UL><LI>DsbA-DNase and DsbA-DspB can be secreted in a fully folded and functional state</LI><LI>Both DNase and DspB are able to degrade biofilms</LI><LI>Art-175 is able to exert cell lytic activity on planktonic <I>E. coli</I> and <I>P. putida</I></LI><LI>Art-175 is able to kill a portion of biofilm-encased <I>P. putida</I> cells</LI></UL><P>
                        The results and in-depth discussion of our experimental work can be found on the <A href="https://2015.igem.org/Team:Oxford/Experiments">Experiments</A> page.
                    </P></DIV></DIV><DIV class="section" id="improving-part-function"><DIV class="slim"><H2>Improving Part Function</H2><P>
                        Improving the function of another team’s part: BBa_K729004
                    </P><P>
                        Team UCL 2012 also had a part comprising Staphylococcal DNase with a DsbA tag upstream of it. We were interested in finding out:
                    </P><UL><LI>Whether the DsbA 2-19 sequence is able to facilitate the export of this part of expression host organism E. coli MG1655</LI><LI>Whether the Staphylococcal nuclease can degrade E. coli biofilms (it was shown to degrade S. aureus biofilms in <A href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005822">Mann et al, 2009</A>)
                    </LI></UL><DIV class="image image-full"><P>Figure 14: SDS-PAGE of <I>E. coli</I> MG1655 BBa_K729004 [pBAD], 0% ara supernatant (A) and E. coli MG1655 BBa_K729004 [pBAD], 0.2% ara supernatant (B)</P></DIV><P> Figure 14 shows the successful DsbA-directed secretion of DNase across both cell membranes. </P><P> A is the supernatant of uninduced E. coli MG1655 BBa_K729004 [pBAD], whilst B is the supernatant of 0.2% induced E. coli MG1655 BBa_K729004 [pBAD]. The band is approximately 21 kDa, corresponding to the size of DsbA-DNase.</P><DIV class="image image-full"><P>Figure 15: Expression host MG1655 BBa_K729004 [pBAD] biofilm growth assay </P></DIV><P> Figure 15 shows the effect of inducing the expression of BBa_K729004 [pBAD] on the ability of the host to form biofilms. The control (MG1655, pBAD/HisB, 0.2% ara) and MG1655, BBa_K729004[pBAD], 0% ara are both able to grow biofilms, as shown by the intensity of the crystal violet staining. When BBa_K729004[pBAD] is expressed, the intensity of the crystal violet staining is reduced, showing a diminished ability to grow biofilm. This data suggests that the secretion of DNase is able to inhibit biofilm formation. </P></DIV></DIV><DIV class="section" id="conclusion"><DIV class="slim"><H2>Conclusion</H2><P>
                        Through our experimental work, we have successfully created and submitted 12 sequence-confirmed BioBrick parts, 7 of which we rigorously characterized for antibacterial and/or antibiofilm function. We validated that Art-175 and Microcin S are both potent antibacterials, the former of which is shown to be even capable of killing antibiotic-resistant biofilm-encased bacteria. On the antibiofilm side of things, we not only showed that the enzymes of interest, DNase and DspB, were successfully exported across both membrane layers of <I>E. coli</I> following our modification of them with secretion tags, but also proved that the enzymes are able to refold properly post-secretion such that they retain their enzymatic function.
                    </P><P>
                        In conclusion, we achieved our aim of creating bacterial &quot;living therapeutics&quot; - strains of bacteria genetically engineered to secrete functional antibiofilm and antimicrobial proteins towards the treatment of UTIs.
                    </P></DIV></DIV><DIV class="section" id="future"><DIV class="slim"><H2>Future</H2><P>
                        To develop our project beyond a proof-of-concept design, we would adopt a more suitable chassis, such as <EM>Lactococcus lactis</EM>. <EM>L. lactis</EM> has been widely used as a expression host for the production of proteins in both the medical and food industries. Being a Gram-positive species of bacteria, it is less likely to be killed by the same mechanisms as major Gram-negative pathogens such as <I>E. coli</I> and <I>P. aeruginosa</I> (e.g. Art-175's peptidoglycan lysis ability is specific for Gram-negative bacteria). On top of that, being Gram-positive means that it will not pose the problems of endotoxicity brought about by the outer membranes of Gram-negative bacteria. Using <EM>E. coli</EM> as our host was purely a starting point, in view of its ease-of-use as well as availability of pre-existing resources.
                    </P><P>
                        In addition to secreting antibiofilm/antimicrobial proteins, a comprehensive treatment for UTIs would be a bacteria engineered to also sense and move towards biofilms. We conducted extensive literature review on this in the early stages of the project but, due to the time restraints of a summer project, could not put our ideas into practice. With further work, we would incorporate both a sensing and chemotaxis mechanism into our design.
                    </P><P>
                        Nurses, doctors and professors all raised to us the issue of targeting the multiple bacterial and fungal species that are involved in UTIs, highlighting the fact that the problem extends further than <EM>E. coli</EM> and <EM>P. aeruginosa</EM>. We have explored how we would approach this in the <A href="https://2015.igem.org/Team:Oxford/Practices">Practices</A> page.
                    </P><P>
                        Beyond the scientific issues of implementation, thinking seriously about the questions of ethics and public acceptance is also crucial for the further development of synbio-based medical therapies especially in view of the fact that it is currently illegal to even bring genetically-modified organisms outside of the laboratory environment. We have explored this theme also in the <A href="https://2015.igem.org/Team:Oxford/Practices">Practices</A> page.
                    </P></DIV></DIV><DIV id="references"><H2>References</H2><OL class="references"><LI>Global Report on Surveillance of Antimicrobial Resistance: 2014. WHO.</LI><LI>Johnson, J.R., 2004. Laboratory diagnosis of urinary tract infections in adult patients. Clinical infectious diseases : an official publication of the Infectious Diseases Society of America, 39(6), p.873; author reply 873–874.</LI><LI>Zalewska-Piatek, B. et al., 2013. Biochemical characteristic of biofilm of uropathogenic Escherichia coli Dr+ strains. Microbiological Research, 168, pp.367–378.</LI><LI>Sievert, D.M. et al., 2013. antibiotic-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2009-2010. Infection control and hospital epidemiology : the official journal of the Society of Hospital Epidemiologists of America, 34(1), pp.1–14. Available at: <A href="http://www.ncbi.nlm.nih.gov/pubmed/23221186">http://www.ncbi.nlm.nih.gov/pubmed/23221186</A>.</LI><LI>Fux, C. a. et al., 2005. Survival strategies of infectious biofilms. Trends in Microbiology, 13(1), pp.34–40.</LI><LI>Flemming, H.-C. &amp; Wingender, J., 2010. The biofilm matrix. Nature reviews. Microbiology, 8(9), pp.623–633. Available at: <A href="http://dx.doi.org/10.1038/nrmicro2415">http://dx.doi.org/10.1038/nrmicro2415</A>.</LI><LI>Høiby, N. et al., 2010. Antibiotic resistance of bacterial biofilms. International Journal of antibiotic Agents, 35(4), pp.322–332.</LI><LI>Briers, Y. et al., 2014. Art-175 is a highly efficient antibiotic against multidrug-resistant strains and persisters of Pseudomonas aeruginosa. antibiotic Agents and Chemotherapy, 58(7), pp.3774–3784.</LI><LI>Gunzer, F., 2013. Bacterially-formed microcin S, a new antibiotic peptide, effective against pathogenic microorganisms, e.g. enterohaemorrhagic Escherichia coli (EHEC), European Patent EP2557163A1.</LI><LI>Antibiotics - Side effects. Avaolable from: <A href="http://www.nhs.uk/Conditions/Antibiotics-penicillins/Pages/Side-effects.aspx">http://www.nhs.uk/Conditions/Antibiotics-penicillins/Pages/Side-effects.aspx</A> [5/06/2015]</LI><LI>C. M. Kunin, “Urinary tract infections in females,” Clinical Infectious Diseases, vol. 18, no. 1, pp. 1–12, 1994.</LI><LI>J. W. Warren, “Catheter-associated urinary tract infections,” Infectious Disease Clinics of North America, vol. 11, no. 3, pp. 609–622, 1997</LI><LI>A. P. Lenz, K. S. Williamson, B. Pitts, P. S. Stewart, and M. J. Franklin, “Localized gene expression in Pseudomonas aeruginosa biofilms,” Applied and Environmental Microbiology, vol. 74, no. 14, pp. 4463–4471, 2008.</LI><LI>L. Zhang and T. Mah, “Involvement of a novel efflux system in biofilm-specific resistance to antibiotics,” Journal of Bacteriology, vol. 190, no. 13, pp. 4447–4452, 2008.</LI><LI>J. Klebensberger, A. Birkenmaier, R. Geffers, S. Kjelleberg, and B. Philipp, “SiaA and SiaD are essential for inducing autoaggregation as a specific response to detergent stress in Pseudomonas aeruginosa,” Environmental Microbiology, vol. 11, no. 12, pp. 3073–3086, 2009</LI><LI>U. Römling and C. Balsalobre, “Biofilm infections, their resilience to therapy and innovative treatment strategies,” Journal of Internal Medicine, vol. 272, no. 6, pp. 541–561, 2012</LI><LI>Hale, S.P., Poole, L.B. &amp; Gerlt, J. a, 1993. Mechanism of the reaction catalyzed by staphylococcal nuclease: identification of the rate-determining step. Biochemistry, 32(29), pp.7479–7487</LI><LI>Vondervizst, F., Sajó, R., Dobó, J., &amp; Závodszky, P. (2012). The Use of a Flagellar Export Signal for the Secretion of Recombinant Proteins in Salmonella. In: Recombinant Gene Expression - Reviews and Protocols, Methods in Molecular Biology, 824, 131-143.</LI><LI>Guddat, L.W., Bardwell, J.C. &amp; Martin, J.L., 1998. Crystal structures of reduced and oxidized DsbA: investigation of domain motion and thiolate stabilization. Structure (London, England : 1993), 6(6), pp.757–767.</LI><LI>Heras, B. et al., 2009. DSB proteins and bacterial pathogenicity. Nature reviews. Microbiology, 7(3), pp.215–225.</LI><LI>Schierle, C.F. et al., 2003. The DsbA signal sequence directs efficient, cotranslational export of passenger proteins to the Escherichia coli periplasm via the signal recognition particle pathway. Journal of Bacteriology, 185(19), pp.5706–5713.</LI><LI>Steiner, D. et al., 2006. Signal sequences directing cotranslational translocation expand the range of proteins amenable to phage display. Nature biotechnology, 24(7), pp.823–831.</LI><LI>Božić, N. et al., 2013. The DsbA signal peptide-mediated secretion of a highly efficient raw-starch-digesting, recombinant α-amylase from Bacillus licheniformis ATCC 9945a. Process Biochemistry, 48(3), pp.438–442.</LI><LI>Zhang, G., Brokx, S. &amp; Weiner, J.H., 2006. Extracellular accumulation of recombinant proteins fused to the carrier protein YebF in Escherichia coli. Nature biotechnology, 24(1), pp.100–104.</LI><LI>Fisher, A.C. et al., 2011. Production of secretory and extracellular N-linked glycoproteins in Escherichia coli. Applied and Environmental Microbiology, 77(3), pp.871–881.</LI><LI>Hwang, I.Y. et al., 2014. Reprogramming microbes to be pathogen-Seeking killers. ACS Synthetic Biology, 3(4), pp.228–237.</LI><LI>Dramatic rise seen in antibiotic use. Available from: <A href="http://www.nature.com/news/dramatic-rise-seen-in-antibiotic-use-1.18383?WT.mc_id=TWT_NatureNews">http://www.nature.com/news/dramatic-rise-seen-in-antibiotic-use-1.18383?WT.mc_id=TWT_NatureNews</A> [17/09/2015]</LI></OL></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#overview">Overview</A></LI><LI><A href="#problem">The Problem</A></LI><LI><A href="#antibio">The Problem with Antibiotics</A></LI><LI><A href="#biofilms">Biofilms</A></LI><LI><A href="#utis">UTIs</A></LI><LI><A href="#solution">The Solution</A></LI><LI><A href="#solution-overview">Overview</A></LI><LI><A href="#solution-degrading-biofilm">Degrading the Biofilm</A></LI><LI><A href="#solution-killing-bacteria">Killing the Bacteria</A></LI><LI><A href="#solution-killing-bacteria-art175">Art-175</A></LI><LI><A href="#solution-killing-bacteria-mccs">MccS</A></LI></UL><LI><A href="#solution-secreting">Secreting the Proteins</A></LI><LI><A href="#solution-sectreting-dsba">DspA</A></LI><LI><A href="#solution-sectreting-yebf">YebF</A></LI><LI><A href="#solution-secreting-fla">Fla</A></LI><LI><A href="#delivery">Delivery</A></LI><LI><A href="#results">Results</A></LI><LI><A href="#improving-part-function">Improving Part Function</A></LI><LI><A href="#conclusion">Conclusion</A></LI><LI><A href="#future">Future</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>