<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Blog skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Blog</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/2/27/Ox_blog_background2.png)"><H3>Blog</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="section" id="blog_september"><H5>September</H5><DIV class="section" id="blog_wikifreeze"><DIV class="slim"><H2>Countdown to WikiFreeze</H2><P>
                                After 10 weeks of lab work and several months of planning the final day of the project has arrived. The Wiki freeze at 5 am (BST) is the final submission deadline; after this no further additions to our website can be made.
                            </P><P>
                                Unsurprisingly things are a little hectic here as we try to get everything finished.  The main goal is to get all our pages coded and live on the website. So far we have finished our extensive policy and practices pages and the design page for our project. More still needs to be done and we are likely to be coding for the next few hours.
                            </P><P>
                                As well as this, over the last few days, members of our team have been preparing a presentation, which will tell the story of our project. We will be presenting this at the Jamboree to teams from around the world.
                            </P><P>
                                Other members of the team have focused on getting our poster ready to print. This is the other main way we will present our project at Boston, providing a stand-alone explanation of what we’ve been doing.
                            </P></DIV></DIV><DIV class="section" id="blog_radio"><DIV class="slim"><H2>A Cheeky Visit to the Radio Station</H2><P>
                                At 10am this morning (10/09/15), Helen and Mabel left the Biochemistry Department to walk to the studios of BBC Radio Oxford, ready for an 11am broadcast. After half an hour’s walking, they were glad they’d left that much time, since the building was still on in sight. However, they trusted the map Mabel had on her phone, which said they were only two minutes away, and arrived with plenty of time to sign in, have a quick sit down and a glass of water, and to go over their plan of what they were going to say.
                            </P><P>
                                The producer then came and collected them from reception, being very welcoming and enthusiastic about our work, and shortly afterwards, they were introduced to the presenter of the show, Kat Orman. They had a chat about the outline of the show, including basic questions that would be asked, and then it was time to perform!
                            </P><P>
                                After a few introductory statistics and soundbites of quotes provided by the producer, the interview started. The main topic of conversation of antibiotic resistance. Within this, they talked about why we chose our project, what outreach we’re doing, and how we would combat the prospect of a society who demand to be prescribed antibiotics. Additionally, they spoke about how <A class="definition" title="DNA" data-content="Contains the genetic code for life, using only four types of molecules. It forms the familiar double helix structure and can self-replicate under certain conditions.">DNA</A>
 cloning works, in very simple terms, and the concept of synthetic biology. Kat, like her producer, was very enthusiastic about our project, claiming that it had the potential to save the world! We said that we hoped that, if our idea were to be developed further, it would.
                            </P><P>
                                Finally, Helen and Mabel took a few pictures with the microphones and Kat, before leaving the building, buzzing with excitement all the way home. We’d like to thank BBC Radio Oxford for inviting us to come and speak about our project.
                            </P><P>                           Until tomorrow…
                            </P></DIV></DIV><DIV class="section" id="WHATS_UP_030915"><DIV class="slim"><H2>WHAT'S UP? 03/09/15</H2><P>
                                We are now arriving at the last two weeks of work time before the Wiki freeze, on the 18th of September. While time is running short, lab work is still in full swing and a number of important developments are taking place. Collaboration is a central aspect of <A class="definition" title="iGEM" data-content="Students doing something good with synthetic biology. Stands for International Genetically Engineered Machine.">iGEM</A>, with teams from across the world helping each other to troubleshoot experiments, work on shared designs and to provide information. Here at Oxford, we have collaborated with several teams; for the Bordeaux university team, we have helped them carry out toxicity <A class="definition" title="assay" data-content="An experiment to prove your stuff works.">assay</A>s on their species including <EM>Putida</EM> and <EM>Rhodobacter</EM>. We have also used our extensive <A class="definition" title="PCR" data-content="Polymerase chain reaction. The technique of amplifying DNA by several orders of magnitude.">PCR</A>
 experience from earlier in the project to help Warwick obtain their <A class="definition" title="biobrick" data-content="A specific base sequence that does something you want. Like a lego brick.">biobrick</A>s. Meanwhile, we have carried on characterising our own bacteria and we have been continuing the crucial process of detecting and quantifying our secreted therapeutics. Additionally, another member of our team has been working on developing our <A class="definition" title="AlgiBeads" data-content="A bacterium-sized container to stop bacteria from being released to the outside world. Contains pores to let out the useful things that our bacteria produce.">AlgiBeads</A>. These represent an additional method of containment for our bacteria and, along with the <A class="definition" title="catheter" data-content="A small, flexible tube inserted into the body to remove fluid. Urinary tract infections are a common side effect of using these.">catheter</A>
 and kill switches, allow us to ensure the safety of our bacteria as a therapeutic agent.
                            </P><P>
                                The main focus of the final few weeks will be to collect and organise everything in order to present it on the wiki for the world to see. So far our wiki has several pages already live. Several more are soon to follow, including our lab notebook, detailing all the experimental work we have been doing, as well as our policy and practice pages.
                            </P><P>
                                Even though we are in the final few weeks, a few last bits of public outreach are still to follow. One member of our team has been at a specialist UTI clinic and will be returning detailed information from experts and patients opinions on our project. As well as this, a school event will be carried out at the natural history museum to highlight the importance of antibiotic resistance and our project to GCSE students. Lastly, we have a survey soliciting the public about their preferred method of containment. This can be viewed <A href="https://docs.google.com/forms/d/1fEX4Oyf3-ra_M5V55XG0VmE44NazdLlETS6fvSe40ds/viewform">here</A>.
                            </P><P>
                                Until tomorrow...
                            </P></DIV></DIV></DIV><DIV class="section" id="blog_august"><H5>August</H5><DIV class="section" id="blog_antibiotic_resistance"><DIV class="slim"><H2>ANTIBIOTIC RESISTANCE, FINDING A SOLUTION</H2><P>
                                Antibiotics represent one of the great medical achievements of the last century; allowing an effective and broad method of treating infections, they revolutionised medical treatments. However, the century to come is likely to be dominated by the need to replace them, their utility severely restricted by antibiotic resistance. Antibiotic resistance is not a surprise, but rather a natural consequence of the principles of evolution. The onus on the scientific community is to find a solution; at Oxford iGEM, we believe that a targeted solution is essential. To show this, we have developed a treatment to a specific and serious example of antibiotic resistance: the <A class="definition" title="urinary tract infection" data-content="An infection in the bladder or kidney, symptoms include inflammation, needing to urinate more frequently, and occasionally blood in the urine. Treatments currently rely on antibiotics, and typically follow a two week course. 50% of women acquire one in their lifetime.">urinary tract infection</A>
. As the world health organisation director general recently stated: &quot;No action today means no treatment tomorrow&quot;.
                            </P><P>
                                Antibiotic resistance is defined as the inability of an antimicrobial agent to treat an infection by a given bacteria that it could once effectively treat. As a result, over time, a given drug is less and less effective at treating an infection. The problem itself is not new and has been observed since the 1950s, around the time antibiotic use reached large scales. While some bacteria have been resistant to a given antibiotic prior to it being used as a medical treatment, the majority of resistance cases are the result of evolutionary selection pressure due to human use.
                            </P><P>
                                This can be understood from a simple scenario. If a patient has an infection of a given bacteria and is given an antibiotic under normal conditions, the antibiotic will kill the bacteria and cure the infection. If, however, a mutation occurs in a single bacteria that allows it to survive in the presence of the antibiotic, then it will survive to reproduce the resistant bacteria. Therefore it has an advantage. When the bacteria reproduces, its offspring will now also be resistant, hence the population of bacteria able to survive the antibiotic increases and so the infection persists.
                            </P><P>
                                This however, doesn't represent the entirety of the problem; bacteria are able to transfer genetic material between one another. This is termed horizontal <A class="definition" title="gene" data-content="A section of DNA which codes for a protein.">gene</A>
 transfer and allows bacterial species to acquifer resistance to multiple antibiotics. This results in the so called superbugs such as MRSA.
                            </P><P>
                                As such, over time, the available antibiotics to treat a given bacteria infection begin to decrease. For instance, in the case of the STD chlamydia, only azithromycin is now effective. The pressure therefore is very real and it is highly likely that no current antibiotic will be effective against such a common infection in the near future.The historical tactic to solve resistance is to develop new antibiotics that target the same bacteria in a different way. An example of this is ampicillin. Related to penicillin, it acts to prevent bacteria from producing functional cell walls. It does this by inhibiting a different <A class="definition" title="enzyme" data-content="A molecule which speeds up a chemical reaction - a biological catalyst. The reaction does not involve these molecules.">enzyme</A>
: transpeptidase, rather than DD-transpeptidase. This strategy, while in the short term effective, does not solve the inherent problem of resistance, but merely delays it. The problem is costly with an estimated billion pounds a year spent on treating resistant infections by the NHS alone; globally, resistance presents a cost of trillions.
                            </P><P>
                                Over recent years, there has been a shift from developing new antibiotic compounds to considering targeted solutions to the most subborn cases of antibiotic resistance. This new focus is made possible by new technologies in synthetic biology. This is an emerging science that utilises genetic engineering and molecular cloning. The purpose is to genetically modify bacteria in order to use them as functional agents. The applications are diverse; modified bacteria could be used to detect cancer cells inside the body or provide targeted drug delivery.
                            </P><P>
                                Our project involves using <EM><A class="definition" title="E. coli" data-content="The fastest-growing strain of bacteria that we know everything about. Certain strains can make you ill, but you won’t find those ones in the lab.">E. coli</A></EM> to target the main causes of UTIs, namely <EM>Pseudomonas aeruginosa</EM> and <A class="definition" title="pathogen" data-content="Any organism or virus that can cause disease.">pathogen</A>ic <EM>E. coli</EM>, both of which form an antibiotic resistant film inside the urinary tract. Our bacteria sense specific molecules secreted by these bacteria and, upon doing so, release therapeutic agents that disperse the <A class="definition" title="biofilm" data-content="A community of bacteria (or other microorganisms) adhering to a surface and each other, held together by secreted slime-like polymers that create a more favorable environment for the bacteria and protect them from environmental stresses and attack from a host’s immune system (in the case of pathogenic bacteria).">biofilm</A> and subsequently kill the bacteria. Our project represents a more complete method of treating the infection. A biofilm provides resistance to antibiotics due to its composition. As such, the use of biofilm dispersing agents removes the bacteria's defences ensuring the entire population is destroyed. In contrast to conventional antibiotics, this reduces the chance of a few bacteria surviving due to resistance and so removes the risk of repeat infections.
                            </P></DIV></DIV><DIV class="section" id="WHATS_UP_200815"><DIV class="slim"><H2>WHAT'S UP? 20/08/15</H2><P>
                                With just four weeks to go, our project is entering its final phase, yet there's still plenty of work to be done. The team is now concentrated on characterisation, our biobricks are ready to be sent, and we are currently analysing the interlab data. Characterisation involves a number of key areas. The first stage was to carry out toxicity assays, which allow us to determine if our secreted products are toxic to our modified cells. Overall expression has been largely non toxic to the bacteria which is a big success. The next major challenge has been to determine whether the bacteria are successfully secreting our therapeutic molecules. We have used three secretion systems and, at present, one has been confirmed, with the other two being tested currently. Our other main line of experimentation is to determine the effects of our products on biofilms. In order to ensure safety, we are using a closely related, but non-pathogenic, form of pseudomonas.
                            </P><P>  Now that our wiki is ready, we can begin the process of putting our data in its completed form, ready for the jamboree. At the moment, our wiki has our project outline and <A class="definition" title="model" data-content="A simplified or idealised description of a system or process, usually mathematical, that can be used to predict how it will behave.">model</A>ling data with more content to follow soon. Away from the bench, policy and practices are making further progress. A member of our team is spending the week visiting a UTI clinic in London. This will allow us to conduct interviews with researchers and patients and to gather more feedback on our project. We have also been publicising our project on various media, with responses from, among others, Guardian science and the Wellcome Trust. Over the next few weeks up to wiki freeze, our team will finish characterisation, and complete our wiki. We will also be preparing our poster and presentation ready for Boston.
                            </P><P>
                                Until tomorrow...
                            </P></DIV></DIV><DIV class="section" id="WHATS_UP_060815"><DIV class="slim"><H2>WHAT'S UP? 06/08/15</H2><P>
                                Time is moving fast and the jamboree is drawing closer, we are currently 7 weeks into the project and our story is beginning to take shape. In the wet lab, we have now completed the cloning and preparatory work, with several biobricks ready to be sent; some unique and others improved upon. Characterisation and data collection are now our main focus, which means use of a new range of techniques. Members of the team have been getting to grips with toxicity assays to determine whether our products are harmful to the bacteria. Others have been working on fluorescence microscopy which allows us to detect the level of expression. Our next step will be to carry out secretion experiments and test our products on a target biofilm.
                            </P><P>
                                Alongside the wet lab, significant experimental modelling has been carried out, with members of our team modelling, amongst other things, the gene expression of our secretion systems as well as the rate of diffusion of our secreted products. This represents an important companion to our wet lab work and will inform later experiments.
                            </P><P>
                                Policy and practice is the other main area of focus; following an interview with medical professionals, we have begun to consider the importance of fungi in the urinary tract. By contacting other experts, we have begun to formulate a strategy to target specific fungi, which will mean that our method is a more complete solution to UTIs. Science outreach is also a key element of our project. Currently, members of the team are taking part in a summer school in Oxford to promote synthetic biology to prospective applicants. We will also be raising awareness of antibiotic resistance and targeted solutions. Later in the summer, other members of the team will be giving a similar talk to high school pupils of a nearby school. As a team, we will also be taking part in the UK iGEM meet up taking place later in the month at UCL. This will be our first opportunity to discuss our project with other teams.
                            </P><P>
                                Our public content is also nearing completion. Our newly designed wiki is nearly complete and content will begin to appear soon. Additionally we have been producing videos of members of the team engaging the public and explaining our project. These are available on YouTube.
                            </P><P>
                                Until tomorrow...
                            </P></DIV></DIV></DIV><DIV class="section" id="blog_july"><H5>July</H5><DIV class="section" id="WHATS_UP_210715"><DIV class="slim"><H2>WHAT'S UP? 21/07/15</H2><P>
                                With 4 weeks down and 6 to go solUTIon is making fast and exciting progress. Wet lab is a major focus with all members of the team pitching in to complete our targets. Several milestones have been reached including the successful generation of our team's new and also improved biobricks, which are ready to be shipped. Also our DNA constructs have been successfully inserted into their expression vectors ready for characterisation. Some sequences over the last few weeks have proved stubborn and as a team we have had to embrace new techniques including gradient and DMSO PCR.
                            </P><P>
                                Over the next few weeks, our team will begin the process of characterisation; this represents the main phase of experimenting and will provide the data we will present in Boston. Some of our early targets include testing Art175's effectiveness against <EM>P. aeruginosa</EM>, one of the target pathogens, and also the utility of DNase. These represent the main therapeutics our modified bacteria will secrete in order to destroy the biofilm.
                 </P><P>
                                Policy and practices is a key part of our project and one we have been successfully integrating alongside the wet lab work. Several areas have been of particular focus over the last few weeks including safety and education. Several members of our team have focused on safety, considering areas of ethical concern and practical contamination prevention. This has included consideration of a new catheter design to contain our modified bacteria and allow the therapeutics to be readily secreted. As well as the design of a kill switch, this represents an important element of control over our modified bacteria. The kill switch allows for the bacteria to be selectively destroyed in the absence of the target pathogen or if the bacteria carry out unwanted colonisation. The kill switch provides an additional level of control over our bacteria and addresses important public concerns about the use of genetically modified organisms.
                            </P><P>
                                Education has been the other major focus of our P&amp;P over the last few weeks. An early goal of our team was to raise public awareness for both synthetic biology and the problem of antibiotic resistance of which UTIs present a significant example. Our team has engaged the public with surveys on public perception of synthetic biology and their thoughts on the use of modified organisms as an alternative to antibiotics. The results for both have been encouraging. To expand on this, we have further events planned, including a summer school workshop to expand science outreach to children, as well as a presentation on our project at the local natural history museum. For more information on our prgoress, access our Twitter for updates as well as YouTube where members of our team are providing more detailed explanations.
                            </P><P>
                                Until tomorrow...
                            </P></DIV></DIV><DIV class="section" id="marijuana"><DIV class="slim"><H2>SYNTHETIC BIOLOGY IS MAKING MEDICAL MARIJUANA DRUGS CHEAPER</H2><P>
                                Marijuana is best known for its use as a recreational drug. The substance comes from the leaves of the Cannabis plant, and is made up of a number of compounds; most of these are compounds known as cannabinoids. Cannabinoids are a group of compounds which act on the cannabinoid receptors in the brain, and at least 85 cannabinoids are found in marijuana collected from the Cannabis plant. These cause a variety of different effects between them; of these compounds, tetra-hydro-cannabinol (THC) and cannabidiol (CBD) are the most abundant.
                            </P><P>
                                Cannabinoids exert their effects by binding to cannabinoid receptors, of which there are 2 types. Cannabinoid receptor type 1 (CB1) is found at the synapses of the central nervous system; primarilt sensory neurones. It is the most abundant GPCR found in the brain, and is also found in other tissues such as muscle, liver and adipose (fatty) tissue. CB2 is found primarily in the cells of the immune system.
                            </P><P>
                                It is illegal in the UK and most US states to smoke marijuana, both recreationally and medically. However, the UK Home Office and the US Federal Drug Administration (FDA) have both approved the use of certain cannabinoids from the Cannabis plant to treat medial conditions. This is because determining the effects that one or two cannabinoids have on the body is much easier than testing the effects of dozens at once (i.e. direct from the cannabis leaf). Medical trials have shown that the cannabinoids THC and CBD can have positive medicinal effects, as I shall describe later.
                            </P><P>
                                As a result, a number of pharmaceutical companies are making money legally by growing Cannabis. Unfortunately, the process of getting these target cannabinoids is very time-consuming and low-yielding. First, the plants must be clutivated for months in government-inspected facilities; then, the low quantities of target cannabinoids (THC and/or CBD) must be extracted and purified to form the final product. To make matters worse, plants tend to make several variations of the target compound rather than just one; this further reduces the yield, and doesn't really match the strict manufacturing standards of other pharmaceutical products, such as aspirin or paracetamol.
                            </P><P>
                                Ultimately, the drawbacks of this so-called &quot;pharming&quot; technique mean that these products must be sold at higher prices. However, several companies are now unlocking the power of synthetic biology to make the production of cannabinoids easier. A front-runner in this is GW pharmaceuticals - by mutating genes within the Cannabis plant to alter their activity, they have been able to adjust the relative proportions of different cannabinoids that the plant produces.
                            </P><P>
                                The flow chart below shows how the biosynthesis of each product is controlled by each of four regions (known as &quot;loci&quot;) within the nuclear DNA of the plant. As you can see in the shaded region, the relationship between certain loci and the cannabinoid synthesis pathway is still not fully understood!
   </P><P>
                                The aim of GW pharmaceuticals is to alter the activity of these loci, and therefore bias the cannabinoid synthesis pathway towards the production of CBD and THC, so that the yield from their plants can be increased. Using the products from their plants, GW pharma is selling a drug called Sativex (which contains an approximate 1:1 ratio of THC and CBD in an alcoholic solution) to help combat spasticity in multiple sclerosis sufferers; this drug has been legalised in several countries including the UK, Spain, Germany, Denmark, Canada, and Italy. Sativex is also being trialled for use in the alleviation of nausea in patients undergoing chemotherapy, and has proved very successful so far.
                            </P><P>
                                So GW pharmaceuticals could be on to something, and for more information on the biochemistry of the cannabis biosynthesis pathway, check out the footnote at the end of this article. However, perhaps a more exciting prospect for cannabinoid synthesis is being studied by the research group Hyasynth Bio. They are attempting to engineer yeast that can make any one of the dozens of cannabinoids found in the natural plant. They believe this platform could be used to supply cannabinoid compounds for pharmaceutical applications faster, cheaper, and more safely than through pharming methods!
                            </P><P>
                                So, it appears that Cannabis can have several uses away from the recreational scene, and may prove extremely beneficial to sufferers of diseases such as cancer and multiple sclerosis. If you want to read more synthetic biology articles, check out the rest of our WordPress site - you can find out what the Oxford iGEM 2015 Team have been up to!
                            </P><DIV id="biosynthesis_detail"><H3>The Biosynthesis of Cannabis - a little more detail</H3><P>
                                    In Fig. 1 you will have seen the names and acronyms of a number of compunds, which probably say nothing about the actual pathway! So I'll try to decribe it more clearly here.
                                </P><P>
                                    As shown in Fig. 1, there are two compounds whose expression (the amount of them that is produced in the leaf) is controlled by a specific area of the Cannabis plant <A class="definition" title="genome" data-content="All of the genes within an organism.">genome</A> called locus A. These two compounds acts as the &quot;core&quot; upon which other things are added, later on in the pathway. Divarinic acid is identical to olivetolic acid, except for a hydrocarbon chain coming off the phenolic ring, which in the latter is slightly longer:
                                </P><P>
                                    As the carboxyl group (-COOH) and a hydroxyl group (-OH) are found on neighbouring carbons, decarboxylation can occur upon heating in the absence of enzyme. Therefore, the pathway shown in Fig. 1 assumes that decarboxylation occurs before the addition of geranyl pyrophosphate. Fig. 3 shows one of the routes through the pathway, using the skeletal structures of the compounds involved:
                                </P><P>
                                    It is likely that Hyansynth Bio aree attempting to integrate the genetic codes of the enzymes in the cannabinoid biosynthesis pathway into the yeast genome, so that they can be produced in their yeast cells!
                                </P></DIV></DIV><DIV class="section" id="WHATS_UP_070715"><DIV class="slim"><H2>WHAT'S UP? 07/07/15</H2><H3>We have biobricks!</H3><P>
                                    This morning we had the sequence data back from some of our mini-preps and the sequences showed that we have successfully cloned 4 of our inserts into the pSB-1C3 shipping vector.
                                </P><P>
                                    The sequence data was interesting to analyse; from the 8 samples we sent, 1 was the wrong insert all together (the sequence aligned with LsrGFP and not Lsr Holin, indicating some haphazard gel loading), Dispersin B with DspA secretion tag has a 2964P-&gt;T point mutation and another was completely missing the insert. For the latter two of these, we're sequencing different mini-prep samples tomorrow. For another of the samples, only the forawrd read was successful and so we've asked them to repeat the reverse read. The other 4 were successful and the biobricks are safe in the freezer.
                                </P><P>
                                    The project is progressing well. We can now find our way around the lab pretty well and are getting used to the different protocols. There are still gBlocks that are proving particularly stubborn to PCR and so we're trying a gradient PCR tomorrow. We've got some of our inserts into expression vectors, which are also going for sequencing. We're considering different functional assays to use and work for interlab is well underway. Furthermore, Leon has taken it upon himself to rate our clothes out of 10 every day. Currently, Leon is on 1,111 and Lychee has evened out at 0 (after the -5).
                                </P><P>
                                    A few of us are heading up to the John Radcliffe Hosptial this week to speak to more nurses and we're skyping the Delft iGEM team on Thursday to discuss modelling collaboration. Shout out to two of our team members, Silas and Henry, who both got firsts in their Part I exams. Until tomorrow!
                                </P></DIV></DIV></DIV><DIV class="section" id="blog_june"><H5>June</H5><DIV class="section" id="WHATS_UP_250615"><DIV class="slim"><H2>WHAT'S UP? 25/06/15</H2><P>
                                    4 days in and we have successful transformations! The first week in the lab has been a steep learning curve. For the first couple of days, the whole team was doing wet lab work, learning the basics of performing PCR and doing gel electrophoresis. We extracted successful PCR bands and <A class="definition" title="restriction digest" data-content="The process of cutting DNA into smaller segments.">restriction digest</A>ed our products before determining the concentrations of DNA using the nanodrop machine, after which we ligated our template DNA into the appropriate vector. We then prepared competent <EM>E. coli</EM> and transformed our DNA!
                                </P><P>
                                    We've had to repeat a few of the protocols. 7 (one was faint so it was also repeated) of the 14 PCR lanes gave bands on the first day, the DNA from which has been ligated into pSB-1C3 <A class="definition" title="plasmid" data-content="A small, circular, double stranded piece of DNA that is physically separated from the bacterial chromosome and can replicate independently. Plasmids can be duplicated and transferred between bacteria (even between species) or taken up from the environment, so they're an ideal way of getting new genes into bacteria in the lab.">plasmid</A>s and transformed into <EM>E. coli</EM>. We repeated PCR for 8 of our samples and two gave bands, the DNA from which is hopefully transforming our <EM>E. coli</EM> cells tonight. For the samples that are proving stubborn to PCR, we are adjusting concentrations of template DNA and the annealing temperatures in an attempt to coerce our samples into amplifying.
                                </P><P>
                                    On Tuesday, we ran a workshop for high school students attending a science summer school at Corpus Christi College, during which we went over gene structure and biobricks. We discussed the applications of synthetic biology and worked in groups to come up with potential construct designs. Tomorrow we're going up to the Churchill Hospital to visit the Urology department and talk to nurses about our project and the problem of UTIs. Also tomorrow, we are presenting our project to Biochemistry Department alumni to show them what current undergraduates are doing. Next week, we'll be splitting up into smaller lab groups and starting the Interlab study. Until tomorrow!
                                </P></DIV></DIV><DIV class="section" id="WHATS_UP_210615"><DIV class="slim"><H2>WHAT'S UP? 21/06/15</H2><P>
                                    Hi iGEM world!
                                </P><P>
                                    We haven't posted anything for a while because term and exams have been taking up all our time. But school's out now and we're preparing for a busy summer of iGEM. Our DNA arrived a few weeks ago and has been waiting for us in the fridge. The primers are in the post and we're busy filling in safety forms for induction into the lab tomorrow. And so after months of reading about secretion systems, studying biofilms and sussing out how to use SnapGene, we're ready to get started!
                                </P><P>
                                    An aim of the project is to outreach to as many different groups of people to ask for feedback on our project, and to spread the message of iGEM and synbio. In lieu of this, last week we skyped with team:ITB Indonesia to talk about both our projects. They brought up concerns about depletion of our bacterial population in the secretion system and we talked about the possibility of <A class="definition" title="antimicrobial resistance" data-content="Bacteria adapt to their surroundings, and can become resistant to powerful antibiotics if allowed to survive. The risk of developing antimicrobial resistance increases when you don’t finish your course of antibiotics.">antimicrobial resistance</A>. Their project is concerned with exrtaction of oil/dispersing oil spills.
                                </P><P>
                                    We have a preliminary design for the wiki (which hasn't been implemented yet) and meetings about the modelling approaches we can take for the project. Henry has been reading up on mass action ratios and Michaelis Menten and dry lab team is preparing to start planning the modelling tomorrow. Silas has been taking footage for the videos (&quot;biofilms&quot;) he will be making throughout the summer and James has been looking into the effects of our antibiofilm agents on the bladder biome.
                                </P><P>
                                    We're also giving our first &quot;Concepts of Synthetic Biology&quot; this Tuesday to high school students attending a summer school in Oxford as well as talking about our project at the upcoming open days. June and Mabel have been busy contacting nurses, doctors and professors and we've had some promising comments on our ideas. We're looking forward to meetings to discuss our project further.
                                </P><P>
                                    Keep track of our weekly blog posts about our progress in the lab. Until tomorrow!
                                </P></DIV></DIV><DIV class="section" id="blog_gene_loss"><DIV class="slim"><H2>GENE LOSS IN PLASMIDS: CHALLENGING SYNTHETIC BIOLOGY</H2><P>
                                    Synthetic biology allows for the manipulation of living organisms in order to generate novel functions. This new field has the potential to revolutionise medical treatments, industrial protocols and potentially how we interact with the planet. Key to synthetic biology is the ability to manipulate the genetic information of the target. In most instances, this involves the addition of genes not normally present, hence producing a novel function. To do this, scientists use a vector (which simply means a method of transferring genetic information from one organism to another), the most common of which is the plasmid.
                                </P><P>
                                    While the use of plasmids is the standard method of transformation, use of plasmids presents some unique challenges, most important of which is the loss of plasmids over successive generations. This so called gene loss effectively removes the inserted genes removing the synthetic organism's novel function. As we shall discuss, understanding of gene loss presents both challenges and opportunities.
                                </P><P>
                                    One of the core elements of synthetic biology is applying biological molecules from one environment to another in order to solve a problem. To do this, genetic information must be transferred. Fortunately, bacteria already possess several methods of transferring genetic information from one to aother. By far the most common is the use of plasmids. Plasmids are circular loops of DNA that exist separately from the main bacterial genome; they can replicate independently and be transferred between members of the same and different species. Plasmids commonly contain genes for antibiotic resistance and are the method by which species acquire resistance to multiple antibiotics.
                                </P><P>
                                    While plasmids provide a mechanism of insertion of specific genes into a given bacteria, they do not represent a single permanent change. Over time, transformed bacteria may lose their plasmid and so their novel function. The result is that over time, the bacteria become less effective. It can be observed that over successive generations of cell division the likihood that both daughter cells receive a plasmid decreases, hence the frequency of the desired genes in the population decreases.
                                </P><P>
                                    This is the so called gene loss, the inserted genes are effectively lost from the population. The reasons behind this are complex and not completely understood, and while the probability of a single transformed cell division producing plasmid-free daughter cells is very low, the likelihood of such an event is believed to be disproportionately increased by the reduced growth rate of transformed cells. One possible explanation for this is due to the additional genes transformed bacteria are able to carry out more processes. This in turn increases the metabolic burden on the bacteria and so the overall growth rate is reduced, meaning less frequent cell division. As such, over time, more transformed cells die than are replaced giving a net loss of transformed cells in a population. If this is the case, it may present an inherent shelf life on synthetic organisms.
                                </P><P>
                                    While seemingly unavoidable, plasmid loss may not be entirely negative and instead may present a new opportunity for biocontainment. One of the key challenges with synthetic biology is controlling the modified organisms when they're used outside the lab. Genetically modified organisms may have inherent advantages as their new characteristics are from a distinct environment and, as such, efforts must be made to ensure they do not escape the environment of their intended use. Biocontainment may involve physical isolation (for instance inside a container that prevents the bacteria escaping into the wild), or genetic controls. These are called kill switches, genetic circuits inserted into the bacteria that cause them to die in the absence of a specific molecule usually provided in the growth media.
                                </P><P>
                                    Both systems however have problems; physical containment can fail due to improper production, while kill switches effectively generate a selection pressure in favour of those with mutated kill switches as these bacteria will survive. Plasmid loss therefore may represent an effective way of ensuring genetically modified organisms do not cause damage to the natural ecosystem. In theory, if you could manipulate the rate of plasmid loss, you could control how long your bacteria remained modified, and also revert them to their non-transformed state, hence preventing the risk of colonisation after their function has been fulfilled. While this remains theoretical, it could represent an effective control on the use of synthetic organisms in the real world.
                                </P></DIV></DIV></DIV><DIV class="section" id="blog_may"><H5>May</H5><DIV class="section" id="human_embryo"><DIV class="slim"><H2>THE FIRST GM HUMAN EMBRYO: HAS A LINE BEEN CROSSED?</H2><P>
                                    In April 2015, Junjiu Huang and his research team in Guangzhou, China, released a report confirming that they had genetically modified the DNA within 86 human embryos. This has prompted furious ethical debate amongst scientists across the globe, and raises serious questions about the course of gene editing research in the years to come.
                                </P><H3>Human Embryos?</H3><P>        Yes, human embryos, but not the type we would normally think of. It turns out that 2-5% of embryos formed by IVF consist of an egg fertilised by two sperm, instead of the usual single sperm cell. This event makes the embryos non-viable, such that they cannot result in a live birth. Huang's research team collected 86 of these non-viable human embryos from a fertility clinic, and these were the cells which were used in the experiment.
                                </P><H3>Treating the disease: beta-thalassaemia</H3><P>
                                    Huang and his colleagues were looking for ways to cure a hereditary disease called beta-thalassaemia, which is a blood disorder that can prove fatal for some individuals. This disease is caused by the mutation in the HBB gene, which contains coding information for building the oxygen carrier haemoglobin, found in the blood. A mutation in this gene, a mistake in the genetic instruction manual for building haemoglobin, means than an individual with beta-thalassaemia had a reduced capacity to carry oxygen in their blood. As a result, sufferers of the disease described above experience tiredness, shortness of breath and other symptoms associated with a lack of oxygen supply to the organs.
                                </P><H3>Repairing the instruction manual: CRISPR/Cas9</H3><P>
                                    Huang's research group used an effective, low-cost gene-editing technique called CRISPR/Cas9, which was first put to use in 2013. In this experiment, the CRISPR/Cas9 complex, and other molecules designed to replace the faulty DNA of the HBB gene, were injected into the cell of interest, and subsequently travelled to the nucleus. Once inside the nucleus, CRISPR/Cas9 was intended to cut and replace the faulty region of the HBB gene.
                                </P><H3>Procedure and Results</H3><P>
                                    The team injected 86 embryos with CRISPR/Cas9, along with the other molecules required to replace the faulty DNA. The researchers then waited for 48 hours, by which time the embryos would have grown to roughly 8 cells each. Of the 71 embryos that survived, 53 were genetically tested. This revealed that only 4 of those embryos had seen their HBB gene corrected.
                                </P><P>
                                    &quot;If you want to do it with normal embryos, you need to be close to 100%&quot; says Huang, commenting on his results. &quot;That's why we stopped. We still think it's too immature&quot;. In addition to their low success rate, his team found that CRISPR/Cas9 had caused a surprising number of off-target mutatinos in these cells, something that would have had potentially fatal consequences for a viable embryo. Huang suggests that the abnormal environment of the doubly-fertilised egg cells could have been responsible for the large number of off-target effects caused by CRISPR/Cas9, a case not observed in the experiments using human adult cells or animal embryos.
                                </P><H3>A Community Divided: The Backlash</H3><P>
                                    Some feel that the report, published in scientific journal <EM>Protein &amp; Cell</EM>, has crossed an ethical line. &quot;No researcher has the moral warrant to flout the globally widespread policy agreement against altering the human germline,&quot; Marcy Darnovsky, executive director of the non-profit Centre for Genetics and Society in Berkeley, California, wrote in a statement. Edward Lanphier, president of Sangamo BioSciences in Richmond, California believes that the low success of this investigation should be enough to prompt a ban on such research using human embryos; &quot;I think the paper itself actually provides all of the data that we kind of pointed to,&quot; he says.
                                </P><P>
                                    However, many people have arguments to support Huang's research. George Church, a geneticist at Harvard Medical School, claims that the technology is not immature at all, and that the CRISPR/Cas9 &quot;kit&quot; used by Huang was not the most up to date version available. John Harris, a bioethicist at Manchester University believes &quot;It's no worse than what happens in IVF all the time, which is that non-viable embryos are discarded&quot;. Harris sees no justification for a ban on research in this area, and argues that the technique could eventually be used in clinics, just so long as the potential harm of the treatment is outweighed by the downsides of having the genetic disease untreated; &quot;It's not as if the alternative is safe,&quot; he says. &quot;People with genetic diseases are going to go on reproducing.&quot;
                                </P><H3>In My Opinion...</H3><P>
                                    This report marks a positive step in gene-editing research. If more of this can be done (that is, using only non-viable embryos), the improvement on our understanding of early human development could increase dramatically, and could potentially lead to effective treatment of thalassaemia and many other genetic diseases such as Huntington's and cystic fibrosis, which currently have no cures. As of May 2015, four other teams in China are believed to be conducting experiments using non-viable embryos, and Huang's team aims to develop a technique to decrease the number of off-target mutations introduced by CRISPR/Cas9, this time going back to the use of adult human cells and/or animal models.
                                </P><P>
                                    There is, however, a danger that Huang's paper could be the start of a slippery slope. His research comes as close to the ethical &quot;line&quot; as one can get; let us hope that no one stumbles over it.
                                </P></DIV><DIV class="section" id="How_To_Construct_Design"><DIV class="slim"><H2>THE HOW-TOS OF CONSTRUCT DESIGN</H2><P>
                                        One of the main aims of genetic engineering is to get bacteria to make <A class="definition" title="protein" data-content="An essential part of all living organisms. They are long and fold up into complicated structures and are made up of amino acids.">protein</A>s that they cannot normally synthesise. For bacteria to be able to do this, we must take the DNA that codes for these new proteins and transfer it into bacteria, so that they can understand it and make proteins from it. One of the ways that DNA can be moved into bacteria is by using a plasmid.
                                    </P><P>
                                        A plasmid is a circular piece of DNA. The DNA that codes for the new proteins can be added into a plasmid, which can then be put into bacteria (through a process called transfection) and, providing the plasmid contains all the necessary extra features for gene expression, the bacteria will be able to make the brand new proteins. For example, human insulin is made using bacteria that have been genetically-engineered in this way. It used to be that the only way to obtain insulin was purifying it from the pancreas of cows and pigs that had been slaughtered for food. These days, we can put the gene for human insulin into a plasmid, put this plasmid into bacteria and have the bacteria make insulin for us.
                                    </P><P>
                                        The DNA sequences that you want to add to a plasmid can come from a variety of sources. In the simplest case, the sequence you are looking for is already a biobrick, such as the protein DNase. For this component of our project, we used the iGEM parts registry to loacte the DNA sequence. Many of the parts we need to use for our project are already biobricks. This includes the <A class="definition" title="promoter" data-content="The section of DNA that the RNA polymerase enzyme binds to before it starts making the RNA strand - it is needed to start transcription, so it sits in the DNA before a gene. Regulates whether a gene is &quot;on&quot; or &quot;off&quot; and to what extent. They can be made to be sensitive to certain conditions so that if a bacterium senses a change in environment it can up or down regulate the expression of a certain gene (so there will be more or less of the protein encoded by that gene, produced in the cell).">promoter</A> regions, which are regions of DNA that initiate the expression of a particular gene (in effect an on/off switch), and the ribosome binding sites that facilitate the initiation of protein synthesis. If the sequence you need is not a biobrick, you can find it on online databases such as GeneBank.
                                    </P><P>
                                        In other cases, you may have the amino acid sequence of a protein you want to express, but not the DNA sequence that you need to insert into the plasmid. For us, this was the case with the protein Art175. The paper from which we sourced the sequence of Art175 only gave the amino acid sequence. We used the IDT codon optimisation to get the DNA sequence. This software gives the DNA sequence when you enter the amino acids. This programme also optimises the DNA sequence for expression in <EM>E. coli</EM> (or whichever bacteria species you are wanting to express your proteins of interest in). This is an important step, for which I will now describe the biochemistry.
                                    </P><P>
                                        Amino acids are the building blocks of proteins. There are 20 amino acids and each is coded for by three DNA bases, which are called codons. As there are 4 different nucleotides (A, T, C, G), there are 64 different possible codons. With only 20 amino acids, this means that some amino acids are coded for by more than one codon. For example, AGA, AGG, AGT and ACG all code for the amino acid serine. The relative frequency of codon use vaires widely depending on the organism and organelle. For example, AGA is the most commonly used codon for serine in <EM>E. coli</EM>. When taking the amino acid sequence of the protein you want to express in <EM>E. coli</EM>, the codon optimisation software gives the DNA sequence using the codons that are most commonly used by <EM>E. coli</EM>. This is important to avoid very slow or incorrect protein synthesis.
                                    </P><P>
                                        When you have the codon optimised DNA sequences, restriction enzymes are then used to cut and stick the sequences and plasmids. These enzymes cut at specific sequences of DNA called restriction sites and allow us to stick our pieces of DNA together. iGEM specifies certain restriction sites at the start and end of DNA sequences so that any sequences can be cut and stuck together using only a few restriction enzymes. If the restriction enzyme sites were present <EM>within</EM> any of the DNA sequences (and not only within the prefixes and suffixes) then the restriction enzymes would cut within and mix up all the sequences, rendering them useless. Before using restriction enzymes, it is therefore crucial to check all of the DNA sequences for any disallowed restriction enzyme sites.
                                    </P><P>
                                        For example, the DNA sequence where the EcoRI restriction enzyme cuts is GAATTC. This sequence is present in the DNA sequence for the protein Dispersin B, meaning that EcoRI would cut within the coding sequence, leading to a shortened amino acid sequence and a non-functional protein. With this in mind, it is necessary to change the codon to alter the restriction site whilst at the same time ensuring that the same amino acid is coded for (a different amino acid means an incorrect protein). GAA codes for glutamic acid but so does GAG; thus by changing the third nucleotide from A to G, the final polypeptide sequence remains unchanged (glutamic acid is still coded for) but there is no longer an EcoRI restriction wite within the DspB coding sequence.
                                    </P><P>
                                        Once you have all the DNA sequences for everything you need, it's then time to assemble your constructs in a software like SnapGene. Together, the coding DNA sequences and the control elements make the constructs that need designing. For example, a part of our project looks at different ways of secreting anti-biofilm agents out of <EM>E. coli</EM>. If we're considering <A class="definition" title="artilysin" data-content="An artifically modified version of naturally occuring proteins called lysins that attack bacteria by cleaving their cell wall.">artilysin</A> (one of the proteins we want to secrete), the construct needs to contain the DNA sequence for artilysin as well as a secretion signal (a tag that directs the protein to a secretion system), a histidine tag (enables us to purify the protein at a later stage) and start and stop sequences (enable the bacteria to know where to start reading the DNA and where to stop). We put all of these parts together in the right order so that the bacteria can understand what to make and how to make it. These constructs have to be madr for each individual component of the project. The DNA order can be made after ensuring that the constructs are suitable for DNA synthesis by a company such as IDT.
                                    </P><P>
                                        This is the stage we are at with our project. Most of the Easter vacation was spent finalising our project and the last few weeks have involved sorting our DNA sequences and playin around with SnapGene. Currently, we're making sure that our constructs comply with IDT DNA synthesis rules and double checking that everything is in the right place and right order to put into plasmids when the DNA arrives. After adding the final touches to our constructs, we should be ready to order by the end of next week.
                                    </P></DIV></DIV><DIV class="section" id="blog_PCR"><DIV class="slim"><H2>THE POLYMERASE CHAIN REACTION - IMPLICATIONS OF DNA AMPLIFICATION</H2><DIV id="blog_PCR_introduction"><H3>Introduction</H3><P>
                                            During our time in the lab, we have been getting to grips with several important biochemical techniques, one of which is the Polymerase Chain Reaction, or PCR. This is a technique which is used to amplify the DNA in a sample, copying it many times so that you end up with a quantity of DNA that can be used for other techniques and analysis. As I'm sure you can imagine, this is very helpful in the lab, but it also has many implications for the outside world.
                                        </P></DIV><DIV id="blog_PCR_what"><H3>What do you need?</H3><OL><LI><STRONG>DNA template</STRONG>: the template is essentially the thing you want to amplify. The reaction can't make new DNA by itself, but it can copy existing DNA to make more of it. You also need to know the base sequence either side of the section you want to copy because you need to design primers.</LI><LI><STRONG>Primers</STRONG>: In addition to not being able to make new DNA, the reaction needs a small stretch of DNA to latch on to to start the copying process. These small stretches are called primers. They need to be designed so that they have a complementary sequence to the template, so that they can stick, and be extended according to the sequence of the template.</LI><LI><STRONG>dNTPS</STRONG>: These are the building blocks of DNA. They are the individual nucleotide bases (A, T, C, and G) and are added sequentially onto the end of primer in order to extend it.</LI><LI><STRONG>Enzyme</STRONG>: The enzyme is the thing which performs the reaction. Without it, the reaction would be unable to proceed at any appreciable rate. During our project, we’ve used two different enzymes for PCR: Q5 and Phusion. Due to the characteristics of each enzyme, you need to use different quantities of the other reaction mixture components. For example, the Q5 mix already contains dNTPs, but Phusion doesn’t. Enzymes may also have different optimum temperatures, so you need to take this into account when performing the reaction.</LI><LI><STRONG>Water</STRONG>: This ensures that each of the components is in the correct relative concentration, because if you have, for example, too high a concentration of DNA, you will get too much non-specific binding, especially between primers.</LI></OL></DIV><DIV id="blog_PCR_how"><H3>How does the reaction work?</H3><P>
                                            The reaction works by cycling through a series of different temperatures to allow the different stages of the reaction to occur. Firstly, the reaction mix is heated to 98oC to split the double stranded DNA into single stranded DNA, giving two strands of template.
                                        </P><P>
                                            After this the temperature of the reaction is lowered to allow the primers to anneal to the template. The precise temperature needed is determined by the enzyme being used, and the primer sequences. Our annealing temperatures ranged from about 65-72 degrees Celsius.
                                        </P><P>
                                            The final temperature in the cycle is the optimum temperature for the enzyme you’re using. This allows primer extension to occur at the faster rate possible.
                          </P><P>
                                            These temperatures are then cycled, so you get 25-30 cycles of denaturing, annealing and extending. During each cycle, you double the amount of DNA that you started with, meaning that this is a very efficient technique for amplifying.
                                        </P></DIV><DIV id="blog_PCR_applications"><H3>Applications</H3><P>
                                            In the lab, DNA amplification is important for increasing your DNA concentration for future reactions. When we received our DNA stocks, they were at concentrations that were far too low for decent transformations, meaning that we wouldn’t have been able to achieve appreciable results if we used the DNA as it was. By amplifying it, we were able to make our DNA go further, saving money, and get more meaningful data from our bacteria.
                                        </P><P>
                                            Outside the lab, PCR is still important. One interesting application is in forensic science. At crime scenes, there are often very few pieces of evidence which point towards who performed the crime. Finger prints can be very useful, but sometimes there aren’t any. However, often you can find some biological trace of the criminal, such as blood, or skin cells. These contain the DNA of the criminal, which, if it can be sequenced, can point the finger towards the culprit. However, the DNA present at the crime scene will be at very low concentrations. The solution? It needs to be amplified! The scientists doing the PCR need to be very careful in this situation though, because if their DNA gets into the sample, they could be implicated in the investigation.
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="blog_april"><H5>April</H5><DIV class="section" id="weaponry"><DIV class="slim"><H2>UTI-INSPIRED, BIOFILM-TARGETED MICROBIAL WEAPONRY</H2><P>
                                        Urinary tract infection (UTI) is one of the most common infectious human diseases in the world, with some 150 million cases occurring globally per year. It is also the most common cause of hospital-acquired infections in the developed world, accounting for approximately 40% of documented cases in the United States in 2009. On average, UTI accounts for more than 1 million hospitalisations and $1.6 billion in medical expenses each year in the USA alone.
                                    </P><P>
                                        Uropathogenic <EM>E. coli</EM> is the leading cause of UTIs. It is responsible for 80-85% of community-acquired UTIs, whereas for hospital-acquired UTIs, other bacteria such as <A class="definition" title="P. aeruginosa" data-content="Like E. coli, it is a well documented, fast-growing bacteria. It is more dangerous than E. coli and is harmful to humans, so we would need higher permissions to work with it in the lab."><EM>P. aeruginosa</EM></A> also play a significant role. Among the UTIs acquired in the hospital, about 75% are associated with the use of urinary catheters, tubes inserted into the bladded through the urethra to drain urine which in 1 in 4 hospitalised patients end up receiving.
                                    </P><P>
                                        Bacteria like to live as sessile communities adhered to surfaces instead of isolated planktonic individuals. These immobilised bacterial communities protect themselves by creating biofilms (what we typically call bacterial slime), jelly-like coatings comprising a mixture of long carbohydrates, proteins, and DNA molecules. In the context of UTIs, such biofilms are responsible for causing inflammation as well as providing a safe haven in which the bacteria can replicate and mature. Biofilms have accumulated within their matrix enzymes that destroy antimicrobials; this helps protect embedded bacteria against our immune system as well as administered antibiotics. Biofilms associated with UTIs can form readily on both the surface of our living cells and that of the inserted catheters, and result in infections that are largely recurrent and chronic, even after antibiotic administration.
                                    </P><P>
                                        Given that antibiotic resistance in UTI-causing bacteria is such a significant problem that it warrants even multiple specific mentions in WHO's recent antimicrobial resistance global report, the administration of prolonged-courses of antibiotics in attempt to treat biofilm-protected UTIs would of course be ill-advised. Clinical resistance towards even last-resort antibiotics like carbapenems and tigecyclines have surfaced, and at present there has yet to be any antibiotics capable of degrading and destroying bacterial biofilms released for clinical use.
                                    </P><P>
                                        Inspired by how UTIs, both their own clinical significance and with respect to the bigger picture of modern antibiotic resistance, and with bacterial biofilms as a tangible, well-characterised target in mind, our team has decided to pursue the creation of a synthetic biological system that can detect the presence of bacterial biofilms and subsequently destroy both the biofilms and the offending bacteria. To achieve that aim, our system would have to be able to perform the following tasks autonomously:
                                    </P><OL><LI>Identify the occurrence of biofilm production and other virulent behaviour by pathogenic bacteria through the detection of signals that they use to communicate</LI><LI>Release biofilm-degrading proteins into the infected environment to break down the biofilm</LI><LI>Release antimicrobial proteins into the infected environment to specifically kill bacteria without the use of antibiotics.</LI></OL><P>
                                        Enzymatic degradation of biofilms, especially the <EM>E. coli</EM> biofilms which we are interested in is a subject that has been extensively researched on, and various iGEM teams in past years have taken on projects related to biofilm degradation (eg Groningen 2014, Indonesia 2014, Lyon-INSA 2012, UBC 2010). Most studies however involved the <A class="definition" title="lysis" data-content="Breaking open of the cell wall, which kills the cell.">lysis</A> of the enzyme-synthesizing bacterial chassis after the synthetic biological production of biofilm-degrading enzymes, either for the isolation and purification of said enzymes or for direct one-off release of enzymes into the extracellular environment for therapeutic purposes.
                                    </P><P>
                                        Our team is interested in not only a one-off pathogen-responsive antibiofilm/antimicrobial-releasing system as described above, but also a design with an enzyme secretion system that can exert its antibiofilm/antimicrobial effect without killing the chassis cell. This design would be especially useful in prophylactic applications, whereby these engineered antibiofilm bacteria can, for example, be immobilised on the surface of medical catheters or implants where the formation of biofilms are highly undesirable and autonomously secrete enzymes that destroy the pathogens' protective biofilms as well as the pathogenic bacteria themselves.
                                    </P><P>
                                        We will house our designed antibiofilm/antimicrobial system within <EM>E. coli</EM> K12, which is an extensively-studied, safe-to-use standard model organism widely adopted in both microbial genetic research as well as industrial applications. To achieve the design aims outlined above, we will need to functionalize our K12 with the following biological components:
                                    </P><OL><LI><EM>Promoters</EM> that can trigger protein synthesis in response to the quorum sensors of pathogenic bacteria. In our case, we are targeting pathogenic <EM>P. aeruginosa</EM> and <EM>E. coli</EM>.</LI><LI><EM>Antibiofilm enzymes</EM>, specifically those that can hydrolyse the biofilms of <EM>P.aeruginosa</EM> and <EM>E. coli</EM>.</LI><LI><EM>Antimicrobial enzymes</EM> that can kill <EM>P. aeruginosa</EM> and <EM>E.coli</EM>.</LI><LI><EM>Secretion tags</EM> which mark the synthesised enzymes for secretion from the bacteria.</LI><LI><EM>Lysins</EM> that destroy our K12 to allow the release of the synthesised enzymes in a one-off, high concentration pulse.</LI></OL><P>
                                        We will be discussing the technical details of these parts mentioned here as well as some important underlying biochemical concepts in our next post, so please do stay tuned!
                                    </P></DIV></DIV><DIV class="section" id="shewy"><DIV class="slim"><H2>THE RISE AND FALL OF SHEWY</H2><P>
                                        The project Shewy has its name from Shewanella, a bacteria that seemingly can carry out electrosynthesis. Based on the paper &quot;Towards Electrosynthesis in Shewanella&quot;, we decided to further investigate the scheme for electron transfer in Shewy and to look at reversing it so that the bacteria were passing electrons into their environment.
                                    </P><H3>The aim: generate reductive power</H3><P>
                                        As much as this project seemed ambitious, there were numerous applications for the reductive power of Shewy. By fixing carbon dioxide with protons from water, and sending electrons from Shewy straight into oxygen, bacteria could possibly evolve oxygen, meaning that the bacteria could be used in oxygen evolving devices in spacesuits or rockets.
                                    </P><P>
                                        Mabel and Fred came up with the idea that by using reverse electron transfer we could send electrons to some other compound that we want to reduce. For instance, if the electrode is on and we are driving the electron transport chain in the reverse direction, we would add an enzyme that breaks down the fumarate (original electron acceptor) and then direct the electrons to whatever we want to reduce.
                                    </P><P>
                                        The main application we had in mind with Shewy was the synthesis of biopolymers. The idea was to use reverse electron flow to fix CO<SUB>2</SUB> to lactate and then polymerise lactic acid to form PLA, which is one of the main plastics used in 3D printing. Printing circuits was another application of the Shewy idea. If the metals could be selectively reduced in an small, controlled space, they could be &quot;printed&quot; to generate circuits. By removing the proton gradient generating components in Shewy, and inserting bacteriorhodopsin, a focused light source would cause metal to be reduced in a controlled space.
                                    </P><P>
                                        In light of these ideas, we spent much of the first half of the Easter term reading around reversed electron flow. However, although Shewy is a biochemically challenging and interesting project, further research showed that it might need a lot of work if we were to render it into synthetic biology. It seemed that at least a hundred enzymes were involved in the reduction of oxygen, we needed to know inputs and outputs of various complexes, and should develop a way to specifically relate the project with engineering aspect crucial to synthetic biology. While these discussions were ongoing, Leon found a research group that had been working on a similar project and whose funding has been reduced to zero. Having done lots of research without making a lot of progress, we decided that it could take longer than this summer to narrow down the project, let alone do the lab work required.
                                    </P><P>
                                        Our Shewy space travel nose-dived in a meeting we had at the start of March, during which we realised we needed to take a step back and reassess our original ideas. George had previously worked in a clinic treating urinary tract infections (UTIs) and drew everyone's attention to how significant a problem they are. UTIs are caused by uropathogenic bacteria that invade the cells lining the bladder and form biofilms. It was here that our focus turned, particularly as this was in accordance with our survey results and linked in well with the microbiome topic that had been popular previously. This meeting marked a change of direction for our project, meaning we had to return straight back to the drawing board. We were then faced with the question:
                                    </P><P>
                                        How could synthetic biology be used to better solve a problem that science and medicine is currently failing to address?
                                    </P></DIV></DIV><DIV class="section" id="blog_backstory"><DIV class="slim"><H2>A BACKSTORY</H2><P>
                                        Bacteria do mean business; however, before there was any notion of what business this may be and how we could incorporate it into our project, we first had to establish a team. Last year, the Oxford iGEM team had to set up everything from scratch as the University had never entered the competition before. Their project did very well, winning them a gold medal in Boston. It was last year's team which began the process of starting iGEM 2015. Once they had sparked interest, they set up some preliminary meetings so that we could start to finalise a team, and come up with a strategy for what we would do over the Christmas vacation.
                                    </P><P>
                                        During the vacation, we surveyed the general public so that we could find out their opinions on synthetic biology, what we could do to improve its profile, and how they wanted synthetic biology to help them. The responses we got to our surveys were truly diverse and, at times, reflected the public's lack of understanding about what synthetic biology can and cannot do. Duke received a response suggesting that we use synthetic biology to stop racism. While we're confident in synthetic biology's huge potential, we weren't sure whether it extended quite this far. There were also numerous suggestions that we &quot;create&quot; more energy, showing to us that perhaps more effort is needed to educate the public on the first law of thermodynamics.
                                    </P><P>
                                        Despite these scientifically unsound suggestions, we gathered that the public would love to see synthetic biology being put to use in the areas of environment and medicine, particularly in relation to global warming and antibiotic resistance. We each presented on an idea based on our surveys and voted on the top 6 topics. These were SPACE as proudly presented by Sam, using the human microbiome (which later evolved into our actual project) by Raffy, metal extraction by Leon, desalination by Duke, biological readouts in contact lenses and plants by Will and Lychee respectively.
                                    </P><P>
                                        Given Sam's great enthusiasm for space, we started exploring it in greater detail as a team. We tried combining some of the other ideas with space exploration and discussed different ways that synthetic biology could help with the long journey to the next planet/system/galaxy. This is where we also started exploring Fred's idea of using <EM>Shewanella oneidensis</EM> (&quot;Shewy&quot;) as a long term food or energy source in space. It was interesting during these early stages of the project, for the biochemists as least to puzzle over some unfamiliar aspects of biochemistry and see how we could incorporate them into our project. However, at times we were too bio heavy, perhaps lacking a focus on the engineering aspects of what we're trying to achieve. Instead of approaching things from an engineer's perspective, targeted on making a design that works with as few original ideas as possible (to loosely quote Freeman Dyson), we fully went down the biochemist's route of 1) a wacky idea, 2) lots of wacky idea discussion and finally 3) does this actually work?
                                    </P><P>
                                        Outcome: half a term spent on Shewy...
                                    </P></DIV></DIV><DIV class="section" id="the_hunt"><DIV class="slim"><H2>THE HUNT: BACTERIA STYLE</H2><P>
                                        Bacteria mean business. The swarm growing from the left, <EM>Myxococcus xanthus</EM>, hunt their prey by spreading as what scientists call a biofilm aka &quot;bacterial slime&quot;.
                                    </P><P>
                                        Biofilms can form virtually anywhere. Not only are they responsible for dental plaque, the muck on the inside of fish tanks and the slime on recently fresh fruit, they are also involved in up to <STRONG>80% of all the microbial infections</STRONG> that we acquire.
                                    </P><P>
                                        They are responsible for causing urinary tract infections (UTIs), the most common infection acquired at hospital. <EM>Streptococcus pneumonia</EM> make biofilms and cause pneumonia. Biofilm infections lead to chronic wounds. Most permanent indwelling medical devices, such as joint prosthetics and heart valves, will have a biofilm community growing on them. Biofilms are behind a whole host of medical, industrial and environmental problems that are very costly and technically challenging to remedy.
                                    </P><P>
                                        In nature, the majority of bacteria are not free-living; instead they are found associated in these biofilms, ingenius living environments made by the microorganisms to aid their growth - and their assault on other living things.
                                    </P><P>
                                        Such an environment serves as a shelter from which the bacteria can launch their attack, whether it be on the urinary tract, the lungs of a cystic fibrosis patient or the various plants and pipelines for aquaculture. And such an environment also serves as a hide-out: there is currently no commercial antibiotic that specifically targets bacterial biofilms.
                                    </P><P>      It's time to start a war on bacterial biofilms and, with the increasing prevalence of drug-resistant strains, a completely novel approach is essential.
                                    </P><P>
                                        This is where synthetic biology and the International Genetically Engineered Machine (iGEM) Competition join the story. Synthetic biology is about harnessing the tools that evolution has already developed for us. All we need to add is a little imagination!
                                    </P></DIV></DIV><DIV class="section" id="life_rationalisation"><DIV class="slim"><H2>THE ENGINEER'S RATIONALISATION OF LIFE</H2><DIV class="quote quote-full"><P>&quot;This isn't a fantasy look at the future. We are doing the future&quot;</P><H3>Craig Venter</H3></DIV><P>
                                        Synthetic biology takes the processes designed by nature and categorises them into standardised systems, allowing bacteria to be designed, built and tested, like how you would build a car.
                                    </P><P>
                                        It's about combining the skills of an engineer and engineering principles with molecular biologists and systems biology thinking. The nature of synthetic biology is interdisciplinary, drawing on a number of scientific fields to provide engineering solutions to some of our biggest problems.
                                    </P><P>
                                        It's about appreciating the complexity of living systems and then sorting them into useable parts, parts that can be built back up into new and exciting tools - tools that will rid of pollution, enhance food production, produce energy, do the housework and, in our iGEM project, destroy biofilms, kill pathogenic bacteria and provide a novel approach to treating microbial infections.
                                    </P><P>
                                        It's about selecting a feature of an organism, identifying its DNA, cataloging it and engineering it into a biological chassis. The approach is beautifully simple: the designs of biology applied to the logic of engineering and then put back into nature.
                                    </P><P>
                                        It's about a lot more than bacteria. Synthetic biology is the engineer's rationalisation of life: where we once aimed just to explain the world, synthetic biology now gives us the capacity to improve it.
                                    </P></DIV></DIV><DIV class="section" id="hello_world"><DIV class="slim"><H2>HELLO WORLD!</H2><P>
                                        Welcome! We are the 2015 Oxford iGEM team.
                                    </P><H3>What's iGEM all about?</H3><P>
                                        iGEM is a socially-driven research competition in the field of synthetic biology. Participating teams of undergraduates from all over the world work throughout the summer with the aim of using synthetic biology to create solutions to major social or environmental problems. As such, the competition is just as much about &quot;beyond the lab&quot; work in the form of outreach and human practices as it's about the actual technology derived from our lab research. A summer of hard work culminated in the presentation we'll give at the iGEM Jamboree, which will be happening in Boston at the end of September.
                                    </P><H3>What problem are we interested in?</H3><P>
                                        Antimicrobial resistance is an increasingly serious threat to global public health and is an issue in all partts of the world, and one of the ways through which bacteria confer themselves protection against antimicrobial drugs is the growing of biofilms. Biofilms, or &quot;bacterial slime&quot;, are responsible for a whole host of medical, industrial and environmental problems that are very costly and technically challenging to remedy. The scale of the problem is huge, with up to 80% of all infections involving the formation of a biofilm. Some examples where biofilms pose especially big an issue are urinary tract infections (UTIs), catheter and implant infections, dental plaque formation as well as infections in cystic fibrosis patients. In industry and infrastructure, biofilms are also the main culprit behind the fouling of various plants and pipelines for aquaculture, water treatment, and food production.
                                    </P><P>
                                        Both antimicrobial resistance and the other problems associated with biofilm formation are big issues in their own right but are especially problematic when they're combined. The bacteria, already constantly evolving to afford themselves more innate resistance against antibiotics, produce biofilms as protective layers that shield them from the drugs even more comprehensively.
                                    </P><H3>How are we proposing to solve the problem?</H3><P>
                                        There is currently no commercial antibiotic that specifically targets bacterial biofilms, but researchers have identified a range of bacterially-derived biomolecules that degrade and destroy biofilms. As such, our project involves the engineering of bacterial strains to make them produce and secrete enzymes that can destroy the pathogenic bacteria and the biofilms they make. The beauty of the antibiofilm agents we plan to use is that they have been shown not to induce resistance in the target bacteria, meaning that having them continually produced at a low level will not be nearly as bad as with traditional antibiotics. Our system is applicable to a whole host of biofilm environments and with a simple design that can be used in multiple sectors, we hope to get a step further in providing a novel approach to treating microbial infections.
                                    </P><P>
                                        In terms of product formulation and design, we hope to ultimately arrive at a functional proof-of-concept e.g. an enzyme-secreting infection-clearing catheter or a modular system that continuously and cheaply cleans out pipelines, and to achieve that we have a 16-strong interdisciplinary team comprising 1 engineer, 2 physicists, 2 chemists, 2 biologists, and 9 biochemists working on different aspects of the project in parallel.
                                    </P><H3>How can you find out what we're up to?</H3><P>
                                        This blog will chart our progress at each step of the way, from the initial conceptualisation of the project, to the execution of our experimental work in the lab and subsequently our flying out to Boston to attend and present out project at the Jamboree. Our team's Wiki page, which will contain all the technical details of our project will also be online in due course. What's more, we'll be hosting events in Oxford related to our project and Synthetic Biology as a whole, information about which you can find on the blog.
                                    </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked affix-top sm-hidden xs-hidden" data-spy="affix"><LI><A href="#blog_september">September</A></LI><LI><A href="#blog_wikifreeze">Countdown to the WikiFreeze</A></LI><LI><A href="#blog_radio">A Cheeky Visit to the Radio Station</A></LI><LI><A href="#WHATS_UP_030915">What's Up? 03/09/15</A></LI></UL><LI><A href="#blog_august">August</A></LI><LI><A href="#blog_antibiotic_resistance">Antibiotic Resistance, Finding a Solution</A></LI><LI><A href="#WHATS_UP_200815">What's Up? 20/08/15</A></LI><LI><A href="#WHATS_UP_060815">What's Up? 06/08/15</A></LI><LI><A href="#blog_july">July</A></LI><LI><A href="#WHATS_UP_210715">What's Up? 21/07/15</A></LI><LI><A href="#marijuana">Synthetic Biology is Making Medical Marijuana Drugs Cheaper</A></LI><LI><A href="#biosynthesis_detail">The biosynthesis of Cannabis - a little more detail</A></LI><LI><A href="#WHATS_UP_070715">What's Up? 07/07/15</A></LI><LI><A href="#blog_june">June</A></LI><LI><A href="#WHATS_UP_250615">What's Up? 25/06/15</A></LI><LI><A href="#WHATS_UP_210615">What's Up? 21/06/15</A></LI><LI><A href="#blog_gene_loss">Gene Loss in Plasmids:Challenging Synthetic Biology</A></LI><LI><A href="#blog_may">May</A></LI><LI><A href="#human_embryo">The First GM Human Embryo: Has a Line Been Crossed?</A></LI><LI><A href="#How_To_Construct_Design">The How-Tos of Construct Design</A></LI><LI><A href="#blog_PCR">PCR - Implications of DNA Amplification</A></LI><LI><A href="#blog_PCR_introduction">Introduction</A></LI><LI><A href="#blog_PCR_what">What do you need?</A></LI><LI><A href="#blog_PCR_how">How does the reaction work?</A></LI><LI><A href="#blog_PCR_applications">Applications</A></LI><LI><A href="#blog_april">April</A></LI><LI><A href="#weaponry">UTI-Inspired, Biofilm-Targeted Microbial Weaponry</A></LI><LI><A href="#shewy">The Rise and Fall of Shewy</A></LI><LI><A href="#blog_backstory">A Backstory</A></LI><LI><A href="#the_hunt">The Hunt: Bacteria Style</A></LI><LI><A href="#life_rationalisation">The Engineer's Rationalisation of Life</A></LI><LI><A href="#hello_world">Hello World!</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>