<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image:url(https://static.igem.org/mediawiki/2015/a/ab/Ox_modelling-background.jpg)"><H3>Modelling</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="slim"><DIV class="section" id="introduction"><H2>Introduction</H2><P>
                            Mathematical <A class="definition" title="model" data-content="A simplified or idealised description of a system or process, usually mathematical, that can be used to predict how it will behave.">modelling</A> plays a crucial role in Synthetic Biology by acting as a link between the conception and the physical realisation of a biological circuit. Our modelling team has evaluated the effectiveness of initial designs, and has provided insight into how the system can (or must) be improved.
                        </P><P>
                            Our team experimentally validated that <EM>Escherichia coli</EM> can secrete enzymes which break down the biofilms associated with urinary infections. However, it is difficult to directly measure whether our enzymes are produced in a sufficient quantity to be a more effective treatment than antibiotics. We measured gene expression and diffusion of widely-used chemicals, and then used our model to estimate the number of <EM>E. coli</EM> cells that would make our project a more effective treatment than antibiotics. We expect to have to improve our system to make it realistic.
                        </P><P>
                            To help readers of all kinds and specialisations understand this page we have produced guides for all the modelling techniques used in this section. They are available in our Modelling Tutorial page and will be linked to when appropriate.
                        </P></DIV><DIV class="section" id="characterising-our-cells"><H2>Gene expression rates</H2><P>
                            In this section we look at our cells in isolation in order to assess their functionality and answer important questions such as “how long does it take to produce a certain concentration of product?” The end result - the final concentration of useful enzyme that is produced in the cell - is required for our diffusion model.
                        </P><DIV id="characterising-our-cells-arab"><H3>Arabinose-induced expression</H3><P>
                                We have decided to use an <A class="definition" title="arabinose" data-content="A sugar which is commonly used to induce gene expression.">arabinose</A>-<A class="definition" title="induced" data-content="When we say expression is induced, we mean that it only happens when the inducer is present.">induced</A><A class="definition" title="promoter" data-content="The section of DNA that the RNA polymerase enzyme binds to before it starts making the RNA strand - it is needed to start transcription, so it sits in the DNA before a gene. Regulates whether a gene is &quot;on&quot; or &quot;off&quot; and to what extent. They can be made to be sensitive to certain conditions so that if a bacterium senses a change in environment it can up or down regulate the expression of a certain gene (so there will be more or less of the protein encoded by that gene, produced in the cell).">promoter</A> for the <A class="definition" title="gene expression" data-content="The production of a protein within the cell.">expression</A> of a number of our <A class="definition" title="protein" data-content="An essential part of all living organisms. They are long and fold up into complicated structures and are made up of amino acids.">proteins</A>. This promoter can be modelled as the following chemical system:
                            </P>

                            \[(Arab:AraC)\overset{K}{\rightarrow}mRNA\overset{\alpha}{\rightarrow}P\]
                            \[mRNA\overset{\gamma_{1}}{\rightarrow}\phi\quad P\overset{\gamma_{2}}{\rightarrow}\phi\]
                            <P>
                                Our promoter, pBAD, binds to <A class="definition" title="AraC" data-content="A compound which stops our E. coli from producing enzymes">AraC</A> and this represses <A class="definition" title="transcription" data-content="The process of converting DNA to mRNA.">transcription</A> of <A class="definition" title="mRNA" data-content="Messenger RNA (mRNA) carries the information of the gene we wish to express to the ribosome. The protein is built at the ribosome.">mRNA</A>. Arabinose will bind to AraC and form the Arab:AraC compound, allowing transcription to occur.
                            </P><P>
                                For this system we will assume that AraC is always in large concentration and that its binding to arabinose happens on a faster time scale to transcription. Therefore, we do not need to consider the individual concentrations of arabinose and AraC, instead we just need to include the concentration of the complex (Arab:AraC). The rate \(K\) is not just a simple constant and is given as the <A class="definition" title="Hill function" data-content="The Hill function looks like a smooth step on a stairwell. Depending on the Hill coefficient n, the step can be made to be sharper or smoother.">Hill function</A> in the equations below.
                            </P><P>
                                Using <A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial#gene-expression-networks-michaelis">Michaelis-Mentin kinetics</A>, we arrive at the equations:
                            </P>

                                \[\dfrac{d[mRNA]}{dt}=K_{max}\dfrac{[Arab:AraC]^{n}}{K_{half}^{n}+[Arab:AraC]^{n}}-\gamma_{1}[mRNA]\]

                                \[\dfrac{d\left[P\right]}{dt}=\alpha\left[mRNA\right]-\gamma_{2}\left[P\right]\]
                            <P>
                                Where \([Arab]\), \([AraC]\), \([Arab:AraC]\), \([mRNA]\) and \([P]\) represent the concentrations of arabinose, AraC, Arab:AraC, mRNA and our product protein respectively. We define the remaining symbols in the table below.
                                <TABLE class="table table-striped"><THEAD><TR><TH>Symbol</TH><TH>Definition</TH><TH>Initial Value/Literature Value</TH><TH>Fitted</TH></TR><TR><TD>\(\alpha\)</TD><TD><A class="definition" title="translation" data-content="The process of converting mRNA to a protein. Occurs at the ribosome.">Translation</A>  rate</TD><TD>\(15ntd\: s^{-1}\)/length of sequence [<A href="#references">6</A>]</TD><TD>\(6.6ntd\: s^{-1}\)/length of sequence</TD></TR><TR><TD>\(\gamma_{1}\)</TD><TD><A class="definition" title="combining the rates" data-content="Reference 5 links to a cell division time of 22.5 minutes. Reference 10 links to a mRNA half life of 6.8 minutes. The combined rate is therefore given by ln(2)/22.5/60+ln(2)/6.8/60.">Combined</A><A class="definition" title="degradation rate" data-content="Proteins and mRNA are unstable and will decay into other products. The degradation rate tells us how quickly this process happens.">degradation</A> and <A class="definition" title="dilution rate" data-content="Bacterial cells divide up to every half hour. Each time this happens, the amount of mRNA or protein present in the cell halves.">dilution</A> rate of mRNA</TD><TD>\(2.2\times10^{-3}s^{-1}\) [<A href="#references">5</A>, <A href="#references">10</A>]</TD><TD>\(1.1\times10^{-2}s^{-1}\)</TD></TR><TR><TD>\(\gamma_{2}\)</TD><TD><A class="definition" title="combining the rates" data-content="Reference 5 links to a cell division time of 22.5 minutes. Reference 11 links to a GFP half life of 24 hours.">Combined</A> degradation and dilution rate of <A class="definition" title="GFP" data-content="Green fluorescent protein. It is relatively easy to measure the amount of green light that this protein produces.">GFP</A></TD><TD>\(5.2\times10^{-4}s^{-1}\) [<A href="#references">5</A>, <A href="#references">11</A>]</TD><TD>\(1.1\times10^{-2}s^{-1}\)</TD></TR><TR><TD>\(K_{max}\)</TD><TD>Maximal <A class="definition" title="transcription" data-content="The synthesis of mRNA from DNA.">transcription</A> rate</TD><TD>\(50ntd\: s^{-1}\)/length of sequence [<A href="#references">6</A>]</TD><TD>\(47ntd\: s^{-1}\)/length of sequence</TD></TR><TR><TD>\(K_{half}\)</TD><TD>Half-maximal transcription rate</TD><TD>\(160\mu M\) [<A href="#references">7</A>]</TD><TD>\(100\mu M\)</TD></TR><TR><TD>\(n\)</TD><TD><A class="definition" title="Hill coefficient" data-content="A measure of the smoothness of the graph of gene expression.">Hill coefficient</A></TD><TD>\(2.65\) [<A href="#references">8</A>]</TD><TD>\(2.73\)</TD></TR></THEAD></TABLE></P><P>
                                This table contains literature values for the parameters, found from a number of sources. We then measured <A class="definition" title="GFP" data-content="Green fluorescent protein. It is relatively easy to measure the amount of green light that this protein produces.">GFP</A> expression in <EM>E. coli</EM> to extract experimental values. Here is a plot showing the fit of this model to our experimental data.
                            </P><DIV class="image image-full"><P>
                                    Results showing GFP concentration as a function of time, matched to our deterministic model. Errors are given to one standard deviation and an arbitrary scaling factor is included as a fitted parameter.
                                </P></DIV><P>
                                We can now calculate the limiting concentrations that our products will be expressed. The dilution rate and Hill coefficient of our cells is the same for GFP and our proteins, but the transcription and translation rates are dependent on the sequence length of the protein. Here is a table showing the relevant proteins and sequence lengths:
                            </P><TABLE class="table table-striped"><THEAD><TR><TH>
                                            Product
    </TH><TH>
                                            Sequence Length (/bp)
                                        </TH></TR></THEAD><TBODY><TR><TD>
                                        pBAD HisB DNase DsbA
                                    </TD><TD>
                                        621
                                    </TD></TR><TR><TD>
                                        pBAD HisB MccS
                                    </TD><TD>
                                        414
                                    </TD></TR><TR><TD>
                                        pBAD HisB Art-175 DsbA
                                    </TD><TD>
                                        987
                                    </TD></TR><TR><TD>
                                        pBAD HisB Art-175 YebF
                                    </TD><TD>
                                        1284
                                    </TD></TR><TR><TD>
                                        pBAD HisB Art-E
                                    </TD><TD>
                                        632
                                    </TD></TR><TR><TD>
                                        pBAD HisB Art-175 Fla
                                    </TD><TD>
                                        1095
                                    </TD></TR><TR><TD>
                                        pBAD HisB Art-175
                                    </TD><TD>
                                        936
                                    </TD></TR><TR><TD>
                                        pBAD HisB DNase
                                    </TD><TD>
                                        570
                                    </TD></TR></TBODY></TABLE><P>
                                We now can run our model of the system by solving the set of equations using the MATLAB <A class="definition" title="ODE" data-content="Ordinary differential equation. Describes the small-scale changes of variables.">ordinary differential equation</A> solver ode15s. Below is a plot of the concentration of product against time for each protein expressed with this inducer-promoter pair where the expression is induced by a step function:
                            </P><DIV class="image image-full"><P>
                                    Model data for each of the enzymes we plan to release, using the parameters we found from our experimental data. We found our limiting concentrations were of order \(\mu M\).
                                </P></DIV><P>
                                The advantage of this method is that we have not had to directly measure expression data for all of our enzymes, which is a difficult process. We conclude that we <EM>should</EM> obtain enzyme expression of order \(\mu M\) within 350 minutes. However, the scaling factor we introduced in our fitting function is no substitute for a calibration curve to match GFP fluorescence with GFP concentration. For this reason, we conservatively estimate that our proteins are expressed at \(nM\) concentration.
                            </P></DIV></DIV></DIV><DIV class="slim"><DIV class="section" id="delivery"><H2>Delivery</H2><P>
                            With the information about the rates of production and concentrations of our products, we can look at how the products behave once they leave the cell. Our enzymes are first secreted from the cells, and then out of our containment beads to the biofilms they target. We can provide an estimate of the time scale that our project is working on and assess any need for optimisation of enzyme efficiency.
                        </P><DIV id="delivery-dispersin"><H3>Dispersin B</H3><P>
                                Dispersin B is one of the anti-<A class="definition" title="biofilm" data-content="A layer of polysaccharides and extracellular DNA that render antibiotics ineffective. They stick to surfaces, which is why you find them on urinary catheters.">biofilm</A> agents we are using in our project and will be the focus of this delivery section. As such we will assume that conclusions reached apply to all of our enzymes.
                            </P><P>
                                A concentration of Dispersin B of 60μg/ml is required to destroy a biofilm that has already formed on a surface [<A href="#references">1</A>]. This equates to a concentration of 1.5μM. This is higher than the steady-state gene expression concentration we can expect from our cells, meaning that our system cannot rely solely on diffusion to transport our enzymes to the biofilm. We will therefore model these diffusion systems assuming that our cells are expressing at a 2μM concentration and later we will look at optimising the gene expression to this level.
                            </P></DIV><DIV id="delivery-beads"><H3>Beads</H3><DIV id="delivery-beads-diffusion"><H4>Diffusion</H4><P>
                                    The bead delivery system consists of our cells being contained in <A class="definition" title="alginate" data-content="An inexpensive, non-toxic chemical which can be used to trap cells in place, while letting proteins move freely.">alginate</A> spheres. Water is passed through a container filled with the beads allowing our enzymes to diffuse from the alginate to where they are required. More details about the design of the system can be found <A href="https://2015.igem.org/Team:Oxford/Design#beads">here</A>. Below is a diagram of our cells secreting proteins out of the beads that they are contained in.
                                </P><DIV class="image image-full"><P>
                                        Diagram of secretion through our containment beads.
                                    </P></DIV><P>
                                    To determine the <A class="definition" title="convection mass transfer coefficient" data-content="A measure of how quickly a particle can move from one place to another.">convection mass transfer coefficient</A> of Dispersin B from our gel spheres we looked at the diffusion data obtained from <A href="https://2015.igem.org/Team:Oxford/Beads#03-09-2015">this experiment</A> involving the diffusion of <A class="definition" title="crystal violet" data-content="A staining chemical used to mark cell structures and make them more visible under light microscopy. It is also the chemical used in Gram’s staining method for classifying bacteria into those that are Gram positive (the cell walls are stained) and Gram negative (the cell walls are not stained).">crystal violet</A> from our beads. By analysing the system we can produce a theoretical formula for the concentration of crystal violet in the bulk water as a function of time:
                                </P>

                                \[c_{f}=\dfrac{c_{bo}}{1+\frac{V_{f}}{V_{b}}}\left(1-\exp\left(\dfrac{-K_{m}A_{b}\left(1+\frac{V_{f}}{V_{b}}\right)t}{V_{f}}\right)\right)\]
                                <TABLE class="table table-striped"><THEAD><TR><TH>Symbol</TH><TH>Definition</TH><TH>Value</TH><TH>Units</TH></TR><TR><TD>\(A_{b}\)</TD><TD>Total surface area of the beads</TD><TD>\(0.024\)</TD><TD>\(m^{2}\)</TD></TR><TR><TD>\(V_{b}\)</TD><TD>Total volume of beads</TD><TD>\(1.4\times10^{-5}\)</TD><TD>\(m^{3}\)</TD></TR><TR><TD>\(c_{bo}\)</TD><TD>Initial dye concentration in beads</TD><TD>\(0.025\)</TD><TD>\(M\)</TD></TR><TR><TD>\(V_{f}\)</TD><TD>Volume of fluid surrounding the beads</TD><TD>\(V_{f}=V_{fo}-\dfrac{1\times10^{-6}}{10}t\)</TD><TD>\(m^{3}\)</TD></TR><TR><TD>\(V_{fo}\)</TD><TD>Initial volume of fluid surrounding the beads</TD><TD>\(1\times10^{-4}\)</TD><TD>\(m^{3}\)</TD></TR><TR><TD>\(t\)</TD><TD>Time</TD><TD>\(-\)</TD><TD>\(min\)</TD></TR><TR><TD>\(c_{f}\)</TD><TD>Concentration of fluid surrounding beads</TD><TD>\(-\)</TD><TD>\(M\)</TD></TR><TR><TD>\(K_{m}\)</TD><TD><A class="definition" title="convection mass transfer coefficient" data-content="A measure of how quickly a particle can move from one place to another.">Convection mass transfer coefficient</A></TD><TD>To be fitted</TD><TD>\(mmin^{-1}\)</TD></TR></THEAD></TABLE><P>
                                    The volume of fluid is also a function of time in order to account for the removal of 1ml of water every 10 minutes. The area and volume of the beads is equal to that of 660 spheres with diameter 3.39mm.
                                </P><P>
                                    However, the number of beads is an estimate. Because of this, in order to fit the curve to the experimental data we must scale the experimental data by an unknown factor. Therefore we pre-multiply our equation with an arbitrary scaling factor which, along with the convection diffusion coefficient - \(K_{m}\) - is determined by our fitting function.
                                </P><P>
                                    By fitting the model to our data we returned the value of \(K_{m} = 1.7265\times 10^{-5} mmin^{-1}\).
                                </P><DIV class="image image-full"><P>
                                        We measure the concentration of crystal violet dye as it diffuses out of our containment beads. Errors are given to one standard deviation and data is fitted to a deterministic model to find the convection mass transfer co-efficient. From this we can determine we would require \(100m^{3}\) of beads to reach the required concentration of our own enzymes.
                                    </P></DIV><P><A class="definition" title="Dispersin B" data-content="One of the anti-biofilm agents we are using in our project.">Dispersin B</A> is a significantly larger molecule than crystal violet so this diffusion coefficient will not be close to that for Dispersin B. To account for this, we estimated the convective mass transfer coefficient \(K_{m}\) for Dispersin B using that obtained for crystal violet by assuming that \(K_{m}\) is proportional to the diffusion constant in water D.
                                </P>

                                \[\left(K_{m}\right)_{DispersinB} = \dfrac{D_{DispersinB}}{D_{crystal violet}}\left(K_{m}\right)_{crystal violet} \]

                                <TABLE class="table table-striped"><THEAD><TR><TH>Symbol</TH><TH>Definition</TH><TH>Value</TH><TH>Units</TH></TR><TR><TD>
                                            \(D_{crystal violet}\)
                                        </TD><TD>
                                            Mass diffusivity of crystal violet in water
                                        </TD><TD>
                                            \(2.87\times10^{9}\)[<A href="#references">2</A>]
                                        </TD><TD>
                                            \(\mu m^{2}s^{-1}\)
                                        </TD></TR><TR><TD>
                                            \(D_{Dispersin B}\)
                                        </TD><TD>
                                            Mass diffusivity of Dispersin B in water
                                        </TD><TD>
                                            \(100\) [<A href="#references">3</A>]
                                        </TD><TD>
                                            \(\mu m^{2}s^{-1}\)
                                        </TD></TR></THEAD></TABLE><P>
                                    By substituting in these values we arrive at \(\left(K_{m}\right)_{DispersinB} = 6.03\times10^{-13} mmin^{-1}\)
                                </P></DIV><DIV id="delivery-beads-mass-exchange"><H4>Mass Exchange</H4><P>
                                    This result allows us to theorise a mass exchange system. As a first estimate we will assume that the flow through the beads is sufficiently slow to use the convection diffusion coefficient found above. It is also assumed that the gene expression happens on a faster time scale than the diffusion from the beads to the water, enabling us to assume the concentration of enzyme in the beads remains constant. This is supported by our gene expression models. We can now visualize how the concentrations of the fluid will vary with distance along the mass exchanger:
                                </P><DIV class="image image-full"><P>
                                        Visualisation of the concentrations of the fluid and the beads along our mass exchanger
                                    </P></DIV><P>
                                    The overall system can now be described with the equation:
                                </P>

                                \[J = K_mA\dfrac{c_{fo}-c_{fi}}{\ln\left(\dfrac{c_{B}-c_{fi}}{c_{B}-c_{fo}}\right)}\]
                                <P>
                                    Therefore
                                </P>

                                \[A = J\dfrac{\ln\left(\dfrac{c_{B}-c_{fi}}{c_{B}-c_{fo}}\right)}{K_{m}\left(c_{fo}-c_{fi}\right)}\]
                                <P>
                                    Where \(J=Q\left(c_{fo}-c_{fi}\right)\) and \(Q\) is the volume flow rate of water. We have chosen a flow rate range of 10-100ml/min as this is accepted as a safe artificial bladder fill rate [<A href="#references">4</A>]. This range results in the following number of beads required to reach the desired concentration:
                                </P><DIV class="image image-full"><P>
                                        Relationship between the number of bacteria-containment beads required to reach a particular flow rate of our enzymes. These are the flow rates we require for practical use.
                                    </P></DIV><P>
                                    Therefore a volume of between \(20.3-203m^3\) of beads is required, assuming a <A class="definition" title="packing efficiency" data-content="Our containment-beads are spherical. Therefore, in a collection of beads, some space will be taken up by the gaps between the beads.">packing efficiency</A> of 64% [<A href="#references">9</A>].
                                </P><P>
                                    This estimation relied upon the flow of fluid around the beads being sufficiently slow such that it may be approximated as stationary, so that mass transfer occurs as natural convection. However, because of the (likely) large volume of beads compared to the cross-sectional area of the catheter, this flow of fluid may have a non-negligable velocity.
                                </P></DIV></DIV></DIV><DIV class="section" id="conclusion"><H2>Conclusion</H2><P>
                            Using gene expression and diffusion models, we estimated that we would need around \(100m^3\) of beads to deliver enough enzymes to clear a urinary-infection-associated biofilm. Our treatment will only be more effective than antibiotics if we make our enzymes many orders of magnitude more efficient. This led us to consider an alternative design.
                        </P></DIV></DIV><DIV class="section" id="references"><H2>References</H2><OL class="references"><LI>Jeffrey B. Kaplan; Dispersin B polypeptides and uses thereof. Patent PI 8580551, Nov 12, 2013</LI><LI><A href="http://physicalpharmacy2013.blogspot.co.uk/2013/05/practical-4.html">http://physicalpharmacy2013.blogspot.co.uk/2013/05/practical-4.html</A></LI><LI>&quot;Physical Biology of the Cell&quot;, Rob Phillips, Jane Kondev and Julie Theriot (2009). Page 110</LI><LI>Kim S-Y, Ko SH, Shin MJ, et al. Phasic Changes in Bladder Compliance During Filling Cystometry of the Neurogenic Bladder. Annals of Rehabilitation Medicine. 2014;38(3):342-346. doi:10.5535/arm.2014.38.3.342.</LI><LI>Liang ST, Ehrenberg M, Dennis P, Bremer H. Decay of rplN and lacZ mRNA in Escherichia coli. J Mol Biol. 1999 May 14 288(4):521-38. p.524 right column bottom paragraph</LI><LI>Proshkin S, Rahmouni AR, Mironov A, Nudler E. Cooperation between translating ribosomes and RNA polymerase in transcription elongation. Science. 2010 Apr 23 328(5977):504-8. p.505 table 1</LI><LI>Sourjik V, Berg HC. Functional interactions between receptors in bacterial chemotaxis. Nature. 2004 Mar 25 428(6981):437-41.p.439 left column top paragraph</LI><LI>Salto R, Delgado A, Michán C, Marqués S, Ramos JL. Modulation of the function of the signal receptor domain of XylR, a member of a family of prokaryotic enhancer-like positive regulators. J Bacteriol. 1998 Feb180(3):600-4. p.601 right column</LI><LI>S. Torquato, T. M. Truskett, and P. G. Debenedetti Is Random Close Packing of Spheres Well Defined? 2000 Phys. Rev. Lett. 84, 2064</LI><LI>Selinger DW, Saxena RM, Cheung KJ, Church GM, Rosenow C. Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. Genome Res. 2003 Feb13(2):216-23. p.217 left column 2nd paragraph</LI><LI>Andersen JB, Sternberg C, Poulsen LK, Bjorn SP, Givskov M, Molin S. New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria. Appl Environ Microbiol. 1998 Jun64(6):2240-6.</LI></OL></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked affix-top sm-hidden xs-hidden" data-spy="affix"><LI><A href="#introduction">Introduction</A></LI><LI><A href="#characterising-our-cells">Gene expression rates</A></LI><LI><A href="#characterising-our-cells-arab">Arabinose-induced promoter</A></LI></UL><LI><A href="#delivery">Delivery</A></LI><LI><A href="#delivery-dispersin">Dispersin B</A></LI><LI><A href="#delivery-beads">Beads</A></LI><LI><A href="#delivery-beads-diffusion">Diffusion</A></LI><LI><A href="#delivery-beads-mass-exchange">Mass Exchanger</A></LI><A href="#conclusion">Conclusion</A><LI><A href="#references">References</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>