Team:Oxford/Test/Notebook1
OXFORD
PROJECT
Description
Design
Safety
Parts
Collaborations
Achievements
PRACTICES
Human Practices
Questionnaire Results
Urinary Biome Study
Lab To Clinic
WET LAB
Overview & Results
Notebook
Protocols
AlgiBeads Notebook
Chemicals
Synbiota
Interlab Study
DRY LAB
Modelling
Tutorials
PEOPLE
Team
Attributions
Contact
BLOG
Notebook
Key
*
A - pSB-1C3 LasR Holin
B - pSB-1C3 LasR sfGFP
C - pSB-1C3 Lsr sfGFP
D - pSB-1C3 Lsr Holin
E - pSB-1C3 DNase DsbA
F - pSB-1C3 DspB YebF
G - pSB-1C3 DspB
H - pSB-1C3 MccS
I - pSB-1C3 DspB Fla
J - pSB-1C3 DspB DsbA
K - pSB-1C3 Art-175 DsbA
L - pSB-1C3 Art-175 YebF
M - pSB-1C3 Art-E
N - pSB-1C3 Art-175 Fla
#
A - pBAD 33 LasR Holin
B - pBAD 33 LasR sfGFP
C - pBAD 33 Lsr sfGFP
D - pBAD 33 Lsr Holin
E - pBAD HisB DNase DsbA
F - pBAD HisB QC DspB YebF
G - pBAD HisB QC DspB
H - pBAD HisB MccS
I - pBAD HisB QC DspB Fla
J - pBAD HisB QC DspB DsbA
K - pBAD HisB Art-175 DsbA
L - pBAD HisB Art-175 YebF
M - pBAD HisB Art-E
N - pBAD HisB Art-175 Fla
O - pBAD HisB Art-175
P - pBAD HisB DNase
Cloning
Week 1
Day 1
Preparation of Stock Solutions
1. gBlocks
The gBlocks ordered from IDT arrived in the form of vials of 200¬µg solid DNA powder.
(refer to BioBricks page for information on DNA sequences)
The gBlocks were made into 10ng/¬µl stock solutions in Milli-Q water for storage:
mass/ng
conc/ng¬µl-1
final volume¬µl
200
10
20
2. Primers
The forward and reverse primers ordered from IDT came in 32.4nmol and 34.3nmol of solid respectively.
Sequences
Forward - CTTTTTTGCCGGACTGC
Reverse - ATGATTTCTGGAATTCGC
The primers were made into 100¬µM stock solutions in Milli-Q water for storage:
amt/10-9mol
conc/10-6M
final volume/10-6L
32.4
100
324
34.3
100
343
Preparation of Reaction Solutions
1. gBlocks
2¬µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20¬µl to make 1ng/¬µl-1
2. Primers
2¬µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20¬µl to make 10¬µM reaction solutions.(The solutions are labelled as "Prefix primer" and "suffix primer" in eppendorf tubes in the fridge)
Polymerase Chain Reaction
25¬µl reactions were run according to the PCR protocol here
* The final concentrations of the primers were noted as they are needed to determine the annealing temperatures for the primers, which can be done using NEB‚Äôs online tool here.
** Add components in order of decreasing volume for maximum ease-of-pipetting.
*** When reaction mixture is being made up, all components as well as the mixture itself are to be kept on ice, as the Master Mix is a high-fidelity polymerase that will recognize the primers as being incorrectly base-paired and be able to hydrolyse the primers if kept at room temperature.
**** Use Q5 enzyme in the cold room to avoid defrosting and freezing the original stock of Q5 enzyme. This could decrease the activity of Q5 enzyme. Bring ice bucket to the cold room to bring Q5 into the bench.
***** Make sure that the primer and small amounts of DNA and primer doesn‚Äôt stick onto the side of the tube or the tip.
The reaction mixture tubes were positioned in an Eppendorf Mastercycler nexus X2 and the PCR program was run.
* DNA denaturation can be performed at 98‚ÑÉ because of the high thermal stability of the Q5 polymerase
** A PCR takes 20-30 seconds to extend a sequence by 1kb, and since our longest sequence is ~2kb the extension time was determined to be 60s per cycle.
Gel Electrophoresis of PCR-Amplified gBlocks
An agarose gel was prepared according to the agarose prepartion protocol.
DNA preparation:
The contents of the PCR tubes need to be stained with a loading dye to help visualize its migration.
To each 25¬µl of content in each PCR tube, 10¬µl of blue loading dye was added.
Day 2
Gel Electrophoresis of PCR-Amplified gBlocks (continued from 22/06)
gBlock sizes for reference
Lane 1: 10¬µl of 2-Log Ladder
Lanes 2-15: 20¬µl of DNA and loading dye mixture prepared on 22/06
A potential difference of 120V was applied across the electrodes (the higher the voltage, the faster the gel will run but the poor the separation will be; DNA separation is typically done in the 120-140V range) for 80 minutes. As long as DNA has passed ‚Öî of the column or purple dye have passed purple area of the gel, the gel is ready to get into the EtBr.
The gel was then stained and the bands were visualized; protocol
Results
PCR Results
The 7 bands corresponding to expected DNA sizes were excised using a razor blade for cleaning and extraction. The other sequences, along with J which only showed a very weak band, will be re-PCRed under modified conditions at a later date.
The excised gel chunks were transferred to centrifugation tubes.
Extraction of DNA(PCR product) from Agarose Gel
The extraction was performed using the E.Z.N.A. Gel Extraction Kit made by Omega Biotek, according to the Spin Protocol.
The excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. The heaviest chunk of gel weighed 0.16g and as such 160¬µl of buffer was added to each tube.
* As the tubes were spun with their lids open, they were placed such that lids pointed away from the direction of spinning.
Restriction Digest of Extracted PCR product
Restriction digest was performed using EcoRI-HF (5‚Äô) and SpeI (3‚Äô) restriction enzymes.
NEB‚Äôs Double Digest Finder was invoked and it was determined that CutSmart Buffer would be used for the digest.
The buffer was completely defrosted and shaken before use.
There is a recommended digestion protocol on NEBCloner. 50¬µL reaction mixtures were set up with component volumes as recommended, except for the DNA where all 30¬µL of the extraction mixture (in elution buffer) were used in the digest. There would be up to 50ng of DNA in each tube of extraction mixture (from the gel bands).
Incubation at 37‚ÑÉ was also done for 2 hours (ThermoMixer program 3) instead of the recommended 5-15 minutes, with shaking at 300rpm.
Restriction Enzyme Digest of the iGEM HQ linearised pSB-1c3
125ng of pSB-1C3 was dissolved in 5mL Milli-Q water.
The digest was performed using the modified NEBCloner protocol.
* We did not add phosphatase because it is assumed that with different sticky ends the vector cannot religate
DNA ‚ÄòCleanup‚Äô using EZNA enzymatic reaction kit
Upon completion of restriction digest incubation, the gBlocks and the plasmid backbones were purified again using the E.Z.N.A. Gel Extraction Kit. PCR products and plasmid backbones need to be purified in order to remove remaining impurities including agarose gel and restriction enzymes. At the elution step, the gBlock inserts were eluted using 30¬µL elution buffer whereas the plasmid backbone was eluted using 50¬µL of it.
DNA Quantification using NanoDrop
1¬µL water and tissue were first used to clean the stage. A blank reading was made using 1¬µL of elution buffer, and 1¬µL of each sample was measured for concentration:
Sample
Concentration/ng¬µl
C
11.3
D
3.5
E
0.8
H
0.9
J
1.6
L
7.7
N
3.6
pSB-1C3
1.6
Ligation of gBlock Sequences with pSB-1C3 Backbone
Ligation was completed; protocol
Preparation of Competent E. coli Cells
A sample of E. coli DH5…ë stored at -80‚ÑÉ was defrosted and inoculated in 5mL of LB in a 125mL conical flask (volume of LB 10% of flask volume so as to achieve sufficient aeration). The culture was left to grow overnight at 37‚ÑÉ.
Re-PCR of DNA Sequences
Two sets of sequences A,B,F,G, I, J, K, and M were PCRed using the same protocol as 22/06, with one set using a 55‚ÑÉ annealing temperature and another using 50‚ÑÉ annealing temperature.
Day 3
Re-PCR of DNA Sequences
Refer to protocol from 1.0
PCR Results
The amplified sequences A, B, F, G, I, K, J, and M were loaded with blue dye and run on agarose gel as per standard protocol
Sequences G and J showed bands corresponding to the expected sizes at both 50‚ÑÉ and 55‚ÑÉ annealing temperatures.
The bands were excised and extracted according to standard protocol (E.Z.N.A. Gel Extraction).
Plasmid recovery
The concentration of pSB1C3 plasmid recovered on 23/06 according to the NanoDrop was low (1.6ng/¬µl). To obtain more plasmids, we restriction digested a construct made by last year‚Äôs, pSB1C3 + abijk, to recover the pSB-1C3 content from the construct.
Set up the following reaction mixture:
Component
Volume/¬µl
DNA (pSB1C3 + insert)
5
EcoR1-HF
1
SpeI
1
Cutsmart buffer
2
Water
11
During the incubation, set up 1% agarose gel by putting 1g powder agarose in 100ml; 1% gel is more efficient at separating larger fragments
Results
Incubate the mix above for 2hrs at 37‚ÑÉ
Incubate for 30mins at 95‚ÑÉ to inactivate restriction enzymes (Inactivation temperature of SpeI is 80‚ÑÉ, and EcoR1-HF is 65‚ÑÉ)
Cool and spin (there will be condensation at the top of the eppendorf)
Add 1¬µl CIP (phosphatase) and incubate for 30mins at 37‚ÑÉ
Add loading dye
Load each DNA on the gel
Stain in EtBr for 30 + 20 mins (staining not very clear after 30 mins)
Extraction of ‚Äústicky‚Äù pSB1C3 (recovered from 2014 pSB1C3+abijk) from Gel
E.Z.N.A. Gel Extraction
Gel weight:
0.53g ‚áí 0.53mL Binding Buffer
30¬µL of elution buffer used
Repeat PCR with higher DNA concentration (for the sequences that previously did not yield clear bands)
Sequences A, B, F, G, I, J, K and M have gone through a second round of PCR as they previously failed to yield clear enough bands, and ethidium bromide staining shows that G and J have been successfully isolated as a result.
A, B, F, I, K, L and M still have not had successful PCR runs, as such an alternative PCR protocol with higher DNA concentration was attempted on them instead:
Reagent
Volume/¬µl
gBlock Template (1ng/¬µl)
5
Forward Primer
1.25
Reverse Primer
1.25
* A total of 5ng gBlock as opposed to 1ng at 58‚ÑÉ annealing temperature present in reaction mixture; rest of the protocol is same as prescribed
Results
Alternative protocol does not work.
Gel extraction of G and J
Standard E.Z.N.A. Gel Extraction protocol followed.
Gel weights:
J - 0.34g ‚áí 0.4mL Binding Buffer
G - 0.3g ‚áí 0.4mL Binding Buffer
30¬µL of elution buffer used.
Restriction Digest of G and J
A restriction digest was then performed on the extracted DNA.
NanoDrop Quantification
J: 2.7ng/¬µl
G: 1.7 ng/¬µl
pSB-1C3: 1.4ng/ul - there is 8-9ul of this left in the freezer in drawer 4,  labelled with Psb1c3 EcoRI, SpeI and 24/06/2015
Ligation
Ligation perform according to protocol.
Transforming E. coli cells with Plasmid DNA
Competent e.coli cells were first prepared, according to the protocol here.
Sample C,D,E,H,J,L and N to be transformed, acoording to the protocol here.
Day 4
Growth and Culture of Bacteria
For the protocol please click here.
Plates incubated overnight show that vectors corresponding to C,D,E,H,J,L and N were taken up [however, later we find that ligation had failed, and so gBlocks C,D,E,H,J,L and N were never inserted]. There are at least 5-7 colonies for each biobrick.
Choose three colonies from each plate. The colony should not be too small or too large and should be reasonably spaced from the others.
Separately incubate each colony in test tubes overnight.
This process would significantly increase the amount of plasmids containing biobricks that we want. Plasmids can be extracted later.
Transformation of J and G (24/06 PCR Product) into competent E. coli cells
Competent cells are already made in stocks and can be found in the -80‚ÑÉ freezer: we don‚Äôt need to prepare them again. J and G comes from repeat PCR done in Day 3.
Primer Design
Since A, B, F, G, I, K, M have repeatedly failed to be PCR-amplified, longer primers were designed to replace the old primers for the PCRing of these gene sequences.
Primers that would give the sequences new restriction sites to allow their insertion into pBAD33 and pBAD/HisB expression vectors were also designed.
Day 5
Plasmid Extraction
For the plasmid extraction protocol see here.
pSB1C3 shipping vector containing gBlocks from Day 1, as well as blank pSB-1C3 shipping vector, pBAD/HisB expression plasmids, and pBAD33 expression plasmids were extracted from overnight cultures of E. coli DH5 using E.Z.N.A. Plasmid DNA Mini Kit I.
(refer to pages 10-12 for Mini Kit protocol. Some special notes:
All optional steps were carried out except equilibration step.
After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.
Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.
The precipitate formed in following step 7 does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.
The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.
Plasmid Quantification
1¬µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
DNA
c/ngŒºl-1
DNA
c/ngŒºl-1
pBAD/HisB
36.4
mccS (H1)
61.5
pBAD33
34.4
mccS (H2)
129.6
pSB1C3
142.1
DspB+DsbA (J1)
38.8
lsr + GFP (C2)
122.9
DspB+DsbA (J2)
46.3
lsr + GFP (C3)
40.3
DspB+DsbA (J3)
41.9
lsr + Holin (D1)
40.2
Art175+YebF (L1)
64.6
lsr + Holin (D2)
Art175+YebF (L2)
lsr + Holin (D3)
44.1
Art175+YebF (L3)
43.2
DNase+DsbA (E1)
77.4
Art175 + Fla (N1)
103.5
DNase+DsbA (E2)
43.4
Art175 + Fla (N2)
48.8
DNase+DsbA (E3)
40.7
Art175 + Fla (N3)
134.7
Plasmid Digest
The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37‚ÑÉ for 90 minutes. For the protocol see here.
Gel layout
10¬µl of blue gel loading dye was added to each tube of digested plasmids. 20¬µl from the contents of each tube were then loaded into a 30-well agarose gel according to the following schematic:
(*the pSB-1C3 well has pSB-1C3 with abijk insert)
The ladder well has 10¬µl of 2-Log Ladder added into it.
A potential difference of 120V was applied across the gel for 40 minutes before it was stained with ethidium bromide for 30 minutes.
Analysis of Results
Analysis
The first three lanes produced expected results: pBAD33 and pBAD/HisB being ‚Äúblank‚Äù plasmid backbones, the pSB-1C3-abijk lane giving two bands corresponding to similar sizes (being pSB-1C3 and abijk insert respectively).
All the other lanes seem to be showing only products in the 2kb range, which corresponds roughly to the size of the pSB-1C3 backbone, but the sizes are not uniform. Nonetheless, we know that the plasmid extraction step was carried out correctly as it did yield products approximating what we were looking for.
If it was a matter of the ligation step carried out on 23/06 failing entirely, we should be expecting a uniform row of backbone bands. Instead, there are minor but noticeable size variations between each band which cannot be successfully explained by failed digestion/ligation. It is thus speculated that the pSB-1C3 stock we received from iGEM HQ had suffered from varying extents of DNA degradation such that the restriction enzyme cut sites on their ends were no longer.
Week 2
Day 6
PCR of samples C,D,E,H,J,L and N
Samples C,D,E,H,J,L and N were PCRed according to the protocol here.
* Use the Q5 HF Master Mix that has been kept at 4‚ÑÉ in the cold room and not the frozen sample because repeated freeze thaw cycles are not good for the Master Mix.
** Remade 50¬µl 10uM primer stock from the 100uM stock solution (5¬µl primer in 45¬µl Milli-Q water)
*** Use DNA 10ng/¬µl stock solutions made up on Day 1
Gel electrophoresis of  C,D,E,H,J,L and N
Results
Refer to gel electrophoresis protocol in section 1.1
Only J showed no clear band corresponding to the expected sequence size.
EZNA gel extraction of C,D,E,H,J,L and N
Refer to section 1.13 of the protocol guide.
Excise bands C,D,E,H,L, and N using razor blade, and the excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. For instance, the heaviest band was 0.27g, requiring 0.3ml Binding Buffer to each eppendorf tube.
Restriction Digest of Gel Extracted C,D,E,H,J,L and N
Refer to section 1.2 for the protocol.
Following the gel extraction spin protocol above, extracted PCR DNA needs to be ‚Äòcleaned up‚Äô of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in 30/06 (Day 7).
Recovery of pSB-1C3 vector from 2014 pSB-1C3 + insert
Component
Volume (10¬µl)
pSB-1C3 + insert
10
SpeI
1
EcoR1 HF
1
Cutsmart
2
Milli-Q
6
Incubate for 2hrs 37‚ÑÉ
Heat and inactivate the restriction enzymes at 95‚ÑÉ for 20mins
Cool down in the room temperature, spin (condensation on the side of the eppendorf from heating)
Add 1¬µl CIP and incubate at 37‚ÑÉ for 30mins
Add 5¬µl loading dye
* Phosphatase is added to prevent the vector religating to insert. In the case of PCR amplification of (non-plasmid) gene sequences phosphatase does not need to be added to PCR product because those two ends are unlikely to ligate onto itself.
** Must start preparing for agarose gel to run 2014 pSB1c3 + Insert. The gel will distinguish pSB1c3 vector from insert, allowing us to extract the vector from the gel and ligate with PCR product. Ligation is to be done in 30/06 (Day 7).
Results
Gel electrophoresis of pSB1C3.
EZNA gel extraction protocol on recovered pSB1c3 vector
Refer to section 1.13 for the protocol.
Start with excising the band that corresponds to the base pair length of pSB1c3 vector.
Following the gel extraction spin protocol above, extracted Vector DNA needs to be ‚Äòcleaned up‚Äô of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in Day7.
Growth and culture of E. coli transformed with 24/06 PCR Product (J and G)
Refer to section 1.6 for the protocol.
* LB has to be clear. The LB on the shelf was cloudy and therefore contaminated from last week so start with a new bottle
** Colonies were of variable size which could mean that some of the colonies are contaminated. Therefore, when picking the 6 colonies, pick 2 small, 2 medium and 2 large colonies
Day 7
From yesterday
E.Z.N.A enzymatic reaction cleanup protocol for Restriction Digest products of C, D, E, H, L, and N ( 1.21 )
At the elution step, the gBlock inserts were eluted using 30¬µL elution buffer whereas the plasmid backbone was eluted using 50¬µL of it.
E.Z.N.A gel extraction protocol for pSB-1c3 vector isolated from 2014 pSB1C3+insert ( 1.13 )
E.Z.N.A enzymatic reaction protocol for pSB-1C3 gel ( 1.21 )
Ligation of 29/06 PCR products to pSB-1C3 ( 1.3 )
Some notes:
Our component volumes were slightly different from that in day 2 due to the different amount of gBlock/vector we had. (We divided the amount of pSB-1C3 between our 6 samples)
Component
Volume/Œºl
Digested DNA (gBlock)
30
pSB-1C3
8
T4 DNA ligase buffer
5
T4 DNA ligase
1
MilliQ water
6
Mixtures were incubated on thermomixer at 16 ‚ÑÉ for 16 hours until 8.45 a.m. on 1/7/15, taking care to vortex before placing on thermomixer.
Plasmid Extraction using miniprep kit ( 1.7 )
Sequences G [pSB-1C3] and J [pSB-1C3] were extracted from overnight cultures of E. coli DH5ùõº using E.Z.N.A. Plasmid DNA Mini Kit I.
refer to pages 10-12 for Mini Kit protocol. Some special notes:
All optional steps were carried out except equilibration step.
After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.
After addition of solution I/RNAse, vortexing/vigorous shaking of the tubes should be avoided to prevent shearing of nucleus and undesirable accidental extraction of chromosomal DNA.
After addition of solution I/RNAse, resuspension of pellet can be done by dragging the tube along an eppendorf rack.
Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.
After addition of solution II, the waiting time before proceeding to the next step should not be more than 5 minutes.
The precipitate formed in following addition of solution III does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.
The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.
50¬µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
Plasmid Quantification: Nanodrop ( 1.22 Nanodrops are usually done following restriction digest and transformation)
1¬µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
DNA
Conc. /ngŒºl-1
DNA
Conc. /ngŒºl-1
J1
62.4
G1
67.4
J2
77.2
G2
81.5
J3
39.5
G3
80.2
J4
61.4
G4
88.0
J5
37.9
G5
26.3
J6
134.3
G6
95.7
Plasmid Restriction Digest for 24/06 PCR Product 1.2
* 0.5ul enzyme despite digesting a plasmid because test digest
**must refer to the master sheet each time
As many tubes were being handled, the required reagents were pre-mixed:
Reagent
Volume / ŒºL
2.1 Buffer
50
Milli-Q
350
EcoRI-HF
12.5
PstI
12.5
3ŒºL of each insert-containing plasmid was transferred into PCR tubes, and 17ŒºL of the reagent mix was added to each PCR tube.
The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37‚ÑÉ for 120 minutes.
Gel Electrophoresis of Digested Plasmids from 24/06 PCR Product 1.1
Results
5¬µl of blue gel loading dye was added to each tube of digested plasmids.
20¬µl from the contents of each tube loaded. Gel run under 80V for 40 minutes:
G5, J3, and J5 were also the samples that showed irregularly low concentrations when analysed with the NanoDrop.
As such, G5, J3, and J5 were discarded while the other samples, being potentially-viable BioBricks, were freeze-stored for verification by gene sequencing at a future date.
Preparing new primer stock solutions
Primer
amt / 10-9 mol
conc / 10-6 M
Milli-Q vol / 10-6 L
Posy Suffix Holin
28.7
100
287
SMAP 29 Forward
31.2
100
312
DspB Forward
27.7
100
277
Art-E Prefix
35.4
100
354
DNase Forward
18.2
100
182
YebF Forward
29.1
100
291
Art-E Suffix
19.4
100
194
Art 175 Forward
27.5
100
275
Fla Forward
26.5
100
265
Pre-prefix Holin
22.9
100
229
MccS Forward
29.2
100
292
DsbA Forward
27.0
100
270
Protocol:
Pulse spin primers
Add fresh Milli-Q as above table
Leave for 30 mins at 37C in the shaking block (allowing DNA to resuspend)
Dilute to 10¬µM
PCR Amplification of gBlocks Using New Primers
***Refer to the Master Table for detailing gBlock-primer combinations for both preparation for the gene sequences‚Äô eventual transformation into pSB1C3 shipping vector as well as pBAD33 or pBAD/HisB expression vector.
Primer naming and explantion can be found here.
With reference to the Master Table , standard PCR reaction mixtures (1uL 1ng/uL DNA template, 1.25uL 10uM forward primer, 1.25uL 10uM reverse primer, 9uL Milli-Q water, 12.5uL Q5 Master Mix) was set up for A*-F*, H*, I*, K*-N*, A#-E#, K#, N#, L#, H#, M#, O#, and P#.
G* and J* were not run because we already potentially have viable BioBricks for them from the plasmid extraction done earlier today (24/06 PCR batch).
DspB-containing setups J#, I#, F#, G# were not run because DspB has a BspHI restriction site in the middle of its sequence. The DspB-containing gBlocks will first be QuickChange-PCRed as an insert in the shipping vector to remove the restriction site by introducing a point mutation to codon-swap before being redigested out of the shipping vector and ligated into the expression vector.
Some compromises had to be made in terms of annealing temperature as there were only 2 PCR machines available and hence only 4 different programs could be run in parallel. The annealing temperatures were set up as below:
Annealing T / ‚ÑÉ
Label
72
A*-F*, H*, I*, K*-N*, A#-D#
67
E#, K#, N#, P#
61
L#, H#
70
M#, O#
Reaction times and other temperatures were set up according to standard PCR protocol 1.0
Day 8
Gel Electrophoresis of 30/06 PCR Products
Protocol 1.2
Results
5uL loading dye added per PCR tube. Gel run under 120V for 45 minutes:
Sizes of each fragments can be referred to the Master Table , and the following fragments that show clear bands have been excised and sent for sequencing.
Excised: C*, D*, H*, L*,M*, N*, C#, D#, E#, H#, K#, L#, M#, N#, P#
Restriction Digest of 30/06 PCR Products
Protocol 1.2
Grouping by required restriction enzymes -
EcoRI-HF + PstI-HF: C*, D*, H*, L*, M*, N*
Reagent mix (7-portion) first made up:
Component
Volume / ¬µL
CutSmart Buffer
35
Milli-Q
98
EcoRI-HF
3.5
PstI-HF
3.5
20¬µL of reagent mix was added to each tube containing 30uL DNA products.
BspHI + PstI-HF: E#, K#, N#, L#, H#
Reagent mix (6-portion) first made up:
Component
Volume / ¬µL
CutSmart Buffer
30
Milli-Q
84
EcoRI-HF
3
PstI-HF
3
20¬µL of reagent mix was added to each tube containing 30¬µL DNA products.
BamHI: C#, D#
Since there are only two tubes to be handles the reagents were directly added to each tube:
Component
Volume / ¬µL
3.1 Buffer
5
Milli-Q
14
EcoRI-HF
0.5
PstI-HF
0.5
NcoI, PstI: M#, P#
Since there are only two tubes to be handles the reagents were directly added to each tube:
Component
Volume / ¬µL
3.1 Buffer
5
Milli-Q
14
Ncol
0.5
PstI
0.5
Incubation at 37¬∞C for 2 hours, with 300 rpm shaking. Upon completion, samples stored at -20¬∞C.
Plasmid Design
Quikchange plasmids for DspB were designed and ordered.
VF2 and VR plasmids, used for sequencing pSB1C3-contained inserts were also ordered.
Transformation of 29/06 ligation products (containing C, D, E, H, L, N gBlocks)
Protocol 1.5
*only 9 tubes of competent DH5alpha E. coli left in the freezer after this round of transformation, as such more will need to be made by the end of the week
Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
Then add volume of E. coli according to protocol, using beads to spread across petri dish.
Day 9
Plasmid restriction digest - pSB1C3, pBAD33, and pBAD/HisB
Restriciton digest of SB1C3, pBAD33, and pBAD/HisB completely, using the 1.2 protocol and consulting the master table for the correct buffer and restiction enzymes.
Resultant Gel
100mL 1% agarose made up, small plate loaded and remainder left in bottle on shelf.
Total volume per tube is 35uL. Each tube‚Äôs contents split into two wells (17.5¬µL per well), and gel was run on 80V p.d. for 45 minutes.
* Gel can help tell if any digestion occurred at all - if no digestion occurred plasmid would still remain circular, and circular plasmid would encounter more resistance migrating through the gel matrix than linear plasmid of the same size and hence appear to be bigger than expected when compared against the ladder.
Bands were excised and the heaviest chunk was 0.66g. 670uL of XP2
Binding Buffer was added to each excised chunk to initiate E.Z.N.A. Gel Extraction protocol.
Enzymatic Reaction Clean-up - 30/06 PCR Products
Protocol is 1.13
Retrieve products (on labelled yellow rack) from -20¬∞C and perform clean-up.
The volume of the restriction digest reaction done on 01/07 was 50¬µL per tube hence according to protocol 50¬µL of XP2 Binding Buffer added to each tube.
Upon completion of protocol, tubes placed back into -20¬∞C until plasmids ready for ligation.
Growth and Culture of Bacteria - 29/06 PCR Products
Refer to section 1.6 of the protocol guide.
*note: new bags of inoculation loops are placed next to the 37¬∞C incubators for agar plates
LB agar plates of C, D, E, H, L, N collected.
Three colonies selected from each plate to set up three separate overnight cultures each.
Ligation of 30/06 PCR Products(
Refer to section 1.3 of the protocol guide.
Component
Volume/¬µ
Digested and cleaned PCR products
30
Digested and cleaned plasmids (pSB1C3: C*, D*, H*, L*, M*, N*; pBAD33: C#, D#;pBAD/HisB: E#, K#, N#, L#, H#, M#, P#)
8 for pSB1C3 10 for pBAD337 for pBAD/HisB
T4 DNA Ligase
1
T4 Buffer
5
Milli-Q
6 for pSB1C3 4 for pBAD33 7 for pBAD/HisB
Incubate reaction mixture at 16¬∞C overnight.
Preparation of Competent E. coli DH5alpha
Refer to section 1.4 of the protocol guide.
*Sterile technique used
Test tube filled with 5mL LB broth.
GW opened new bag of inoculation loops to steal a colony off one of Elaine‚Äôs streaked plates (marked 29/06) and inoculated it in the filled test tube.
Test tube left to incubate at 37¬∞C overnight.
Day 3
Plasmid for 29/06 PCR Products
Refer to section 1.7 of the protocol guide.
Yesterday, collected LB agar plates of C, D, E, H, L, N and Interlab are incubated overnight. Selected three colonies from each of C, D, E, H, L and N, and only one for each of the Interlab colonies (we assume that the plasmids provided by iGEM HQ for Interlab were all the same).
Therefore we shall be performing EZNA plasmid extraction on 22 samples. Follow EZNA Mini Kit I Spin Protocol (pg10-12).
50¬µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
Also, aspirate = pipetting!
Measured the concentration of the different samples using nanodrop
Sample
Concentration (ng/Œºl)
C1
46.7
C2
35.6
C3
52.7
D1
43.2
D2
95.8
D3
30.4
E1
45.1
E2
112.4
E3
46.7
H1
71.6
H2
48.2
H3
48.5
L1
48.7
L2
56.2
L3
69.2
N1
57.0
N2
47.0
N3
68.3
Digest aliquots of each of the 22 samples, and run on gel. Refer to protocol 1.2
From gel, determine the degree of success of these samples.
Preparation of Competent E. coli DH5alpha
Refer to protocol 1.4
Transformation of 30/06 PCR Products (Ligated on 02/07) (Protocol in section 1.5)
Refer to protocol in section 1.5
Antibiotics:
pSB1C3, pBAD33 - Chl
for C*, D*, H*, L*, M*, N*, C#, D#
pBAD/HisB - Amp
for E#, K#, N#, L#, H#, M#, P#
Raffy, Leon: Moving of cells plated on 3/7/2015 from the incubator to the cold room. All pSB1C3 plates had reasonable colony density. A significant proportion of pBAD/HisB plated had no colonies. Plates with no colonies will be reincubated for half a day on 6/7/2015.
Week 3
Day 11
Preparation of Overnight Cultures for 30/06 PCR Products
Plate
Colony
Plate
Colony
C#
1
C#e
1
D#
0
D#e
0
E#
2
E#e
2
H#
0
H#e
3
K#
0
K#e
0
L#
0
L#e
0
M#
0
M#e
0
N#
0
N#e
0
P#
1
P#e
0
Plate
Colony
Plate
Colony
C*
Yes
C*e
Yes
D*
3
D*e
0
H*
Yes
H*e
Yes
L*
Yes
L*e
Yes
M*
0
M*e
3
N*
Yes
N*e
Yes
Colonies grew for L* and N*, but because the MiniPrep run on 03/07 already showed the correct bands for them, their colonies were not cultured overnight.
Sequence
Colonies picked
C*
3
D*
3
H*
3
M*
3
C#
2
E#
4
H#
3
P#
1
See protocol guide to find out how to make overnight cultures.
PCR amplification of A,B,D,E,F,I,K,M for pSB1C3
1. Primer used:
* primer pair (pre prefix holin and post suffix holin)
Comments:
Attempting A, B, F and I again as they have never been successfully amplified before.Attempting A, B, F and I again as they have never been successfully amplified before.
NB: F and I are DspB-containing gBlock which we eventually hope to QuikChange to get rid of the BspHI restriction site in them and put them into pBAD/HisB expression vector
2. PCR all the # sequences with the appropriate primers
Follow 1.0 PCR protocol, and primer list is in the Master Table .
Restriction digest for pSB1C3, pBAD33 and pBAD/HisB
Gel photo
Follow restriction digest protocol.
pSB1C3 and pBAD/HisB were successfully digested and eluted in 1% gel. Band for pBAD33 could not be found.
Excised bands were stored in -20‚ÑÉ for cleanup tomorrow.
Primer Preparation
Preparing new primers (solution and dilution) - VF2, VR, QuikChange forward, QuikChange reverse.
First make 100uM out of the freeze dry solid:
Spin down solid
The amount of primer is given in y nmol for each tube. Add 10y ¬µL of water to each tube to make 100¬µM.
Shake/vortex and mix evenly.
For VF2, VR - take 3.2ul out of the 100¬µM stock and dilute with 96.8¬µL Milli-Q to make 3.2pmol/uL sequencing primer solution.
For the QuikChange primers, dilute to 10uM as per normal.
Sequencing of BioBricks
5¬µl of each plasmid along with 100uL of 3.2pmol/uL sequencing primer sent to SourceBioScience for sequencing:
Sequence
Label Assigned
C 3
pSBLsrGFP3
D 2
pSBLsrHolin2
E 2
SBDNaseDsbA2
G 6
pSBDspB6
H 1
pSBMccS1
J 2
pSBDspBDsbA2
L 3
pSBArt175YebF3
N 3
SBArt175Fla3
Gel Electrophoresis of PCR Products
Correct bands were obtained for D*, C#, G#, H#, J#, K#, L#, N#. Still awaiting reply on insert length for O# and P# to determine whether the bands are correct.
Day 12
NanoDrop Analysis of Plasmids Digested on 06/07 (carried out after gel extraction)
Plasmid
Conc / nguL-1
pSB1C3
5, 9.5
pBAD/HisB
1.8, 1.6
Since pBAD/HisB is very low in concentration, more of it will be digested.
Restriction Digest of pBAD33
Component
Volume/¬µL
Plasmids
10
BamHI
1
Milli-Q Water
7
3.1 Buffer
2
37‚ÑÉ, 2 hours (heat up another block to 95‚ÑÉ)
Heat inactivation, 95‚ÑÉ, 20 minutes
Melt, cool, and pour 1% agarose
Pulse spin
CIP dephosphorylation (1 ¬µL CIP), 37‚ÑÉ, 30 minutes
Load dye and run gel
pBAD/HisB digest - 6uL water, 1¬µL NcoI, 1¬µL PstI, 2¬µL 3.1, 10¬µL plasmid
Gel extraction of digested plasmids
Plasmids digested on 6/7:
400¬µL of Binding Buffer added to each tube containing excised band.
After first elution, concentrations turned out to be low (see NanoDrop table at the top of this document), hence a second elution was done, again using 50¬µL of elution buffer.
Plasmids digested on 7/7:
340uL of Binding Buffer; one elution with 50¬µL.
Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)
Follow EZNA DNA Mini Kit I Spin Protocol - eluted 50Œºl
freezer: expression vector ones (the ones with #) on orange rack in bottom drawer (will be dealt with tomorrow), psb vector ones (the ones with *) in white box with other psb constructs in top drawer
NanoDrop analysis:
Sequence
Conc / ng¬µL-1
Sequence
Conc / ng¬µL-1
C* 1
58.3
M* 3
49.2
C* 2
67.5
C# 1
64.5
C* 3
66.0
C# 2
25.1
D* 1
6.5
E# 1
37.8
D* 2
43.3
E# 2
43.1
D* 3
56.7
E# 3
30.5
H* 1
30.0
E# 4
67.3
H* 2
45.9
H# 1
36.5
H* 3
44.5
H# 2
28.8
M* 1
51.1
H# 3
37.3
M* 2
53.1
P# 1
179.2
Gel electrophoresis of Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)
Electrophoresis of 10uL plasmid with 5uL loading dye
Gel Photo
N.B. forgot to digest - will do this on 08/07/15 (tomorrow)
Analysis of sequencing data from 06/07/15
Of the stuff sent for sequencing (03/07 miniprep):
Sample
Results
Action Point
C*3
corresponds to pSB1C3 Lsr GFP
-
D*2
Corresponds to pSB1C3 Lsr GFP (does not correspond to pSB1C3 Lsr Holin as expected - contamination in all three D* wells were already apparent in gel (see 06/07))
Wrong insert, need to redo from scratch (start from PCR again)
E*3
DsbA DNAse missing, sequencing only gives blank pSB1C3 vector
Send another miniprep product in the triplicate (E*1 or E*2) for sequencing.
G*6
Forward sequence good enough to confirm (~70% length of DspB sequence) that it‚Äôs pSB1C3 DspB, but reverse sequence dropped off too early for double confirmation
Ask Source Bioscience to redo VR (reverse) sequence
H*1
Corresponds to pSB1C3 MccS
-
J*2
Corresponds to DsbA DspB, but base 2964 had a C->A point mutation
Send another miniprep product in the triplicate (J*1 or J*3) for sequencing
L*3
Corresponds to pSB1C3 Art-175 YebF
-
N*3
Corresponds to pSB1C3 Art-175 Fla
-
Results: 4, potentially 5 BioBricks successfully made in [pSB1C3] format. Successful BioBricks to be stored in separate box in preparation for sending to Registry, creating Database page etc.
Ligation of 06/07 PCR to Appropriate Digested Plasmids
pBAD33: C#1, C#2 (one of these were wrongly digested using CutSmart instead of 3.1), D# (previously digested using wrong buffer), D#3.1
pSB1C3: D*
pBAD/HisB: E#, G#, H#, J#, K#, L#, N#, O# (previously digested using wrong buffer), O#3.1, P# (previously digested using wrong buffer), P#3.1
Refer to section of the protocol guide.
Left overnight at 16C.
Day 13
To Do
Prepare plates for transformation
Transformation of ligated plasmids
Gradient PCR:
Group A: A*, B*, F*, I*, K*
Group B: A#, B#
Group C: I#, F#
To Note
Re-sequenced VR read and DspB confirmed as correct = another biobrick.
QuikChange PCR on DspB
Standard PCR protocol 1.0
72C annealing temp
2 min extension time
Add 1ul DpnI and incubate at 37C for two hours
Transformation
Add 1ul PCR product to 100ul competent cells. Add the remaining PCR product (24ul) to another eppendorf of 100ul competent cells
Follow standard transformation protocol
Restriction Digest
Incubate at 37oC for 60 mins - BamHI has star activity so it cannot be left for long
Component
Volume/¬µL
Component
Volume/¬µL
Component
Volume/¬µL
C#
5
C*/D*/H*/M*
5
E#/H#/P#
5
3.1 NEB 10x buffer
2
2.1 NEB 10x buffer
2
3.1 NEB 10x buffer
2
Bam HI
0.5
PstI
0.5
Bam HI
0.5
(no 2nd RE)
-
EcoRI-HF
0.5
EcoRI-HF
0.5
MilliQ
12.5
MilliQ
12
MilliQ
12
Gel electrophoresis of restriction digest
Gel Photo
Sent for sequencing:
Sequence
Concentration
Label Assigned
E* 1
45.1
SBDNaseDsbA1
J* 1
62.4
SBDspBDsbA1
C# 1
64.5
33LsrGFP1
H# 3
37.3
BMccS3
E# 4
67.3
BDNaseDspA4
P# 1
179.2
BDNase1
D* 3
56.7
SBLsrHolin3
H* 2
45.9
SBMccS2
M* 2
53.1
SBArtE2
Transformation of E. coli with ligation products from day12 (06/07 PCR and interlab sequences) using the standard protocol described by diagrams for day 3. Transformed E. coli then plated and incubated overnight
Interlab
Week 2
Day 8
Preparation of Interlab Study BioBricks
10¬µL Milli-Q added to wells 20K, 22A, and 22K in Plate 1, and well 21J in Plate 4 in the iGEM Distribution Kit to resuspend the necessary BioBricks [pSB1C3] to prepare for the Interlab study.
The plates were kept on mild shaking until the afternoon.
Transformation of Interlab Study BioBricks into DH5alpha 1.5
Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
Then add volume of E. coli according to protocol, using beads to spread across petri dish.
Day 9
Growth and Culture of Bacteria
Refer to section 1.6 of the protocol guide.
Only one overnight culture each set up for iGEM distros (20K, 22A, 22K, 21J) as it can be rightly assumed that plasmids supplied by iGEM HQ are high purity.
Day 10
Plasmid for PCR Products
Digest aliquots of each of the samples, and run on gel. (Refer to protocol in section 1.2 )
From gel, determine the degree of success of these samples.
Week 3
Day 11
Restriction Digest of InterLab Study Plasmids
20K, 22A, 22K:
Component
Volume / ¬µL
Plasmids
10
SpeI
1
PstI-HF
1
Milli-Q Water
6
CutSmart
2
21J: Same as above, but replace SpeI for XbaI.
Incubated for 2hrs at 37‚ÑÉ, then enzymes heat inactivated at 95‚ÑÉ.
Restriction Digest of InterLab BioBricks
Gel Photo
Pulse spin
Dephosphorylate 20K, 22A, 22K, BUT NOT 21J
Load dye and run gel
Gel image (transilluminator lens was quite grainy today)
Appropriate bands excised for extraction (I13504 for last lane)
Gel extraction of InterLab BioBricks
340uL of Binding Buffer;
one elution with 50uL for 20K, 22A, 22K; 30uL for 21J
Ligation of InterLab sequences
Insert: digested 21J
Plasmids: 20K, 22A, 22K
Component
Volume/¬µL
T4 Ligase
1
T4 Buffer
4
Insert
10
Plasmid
10
Milli-Q
26
NB: total reaction volume of 51¬µL is 1¬µL more than ideal
NB: total reaction volume of 51uL is 1uL more than ideal
Cloning
Week 1
Day 1
Preparation of Stock Solutions
Preparation of Reaction Solutions
Polymerase Chain Reaction
Gel Electrophoresis of PCR-Amplified gBlocks
Day 2
Gel Electrophoresis
Results
Extraction of DNA
Restriction Digest PCR
Restriction Enzyme Digest
DNA ‚ÄòCleanup‚Äô
NanoDrop
Ligation
Preparation of E. coli
Re-PCR of DNA
Day 3
Re-PCR of DNA Sequences
Plasmid Recovery
Extraction of ‚Äústicky‚Äù pSB1C3
Repeat PCR with higher DNA concentration
Gel extraction of G and J
NanoDrop
Ligation
Transformation
Day 4
Growth and culture of bacteria
Transformation of G and J
Primer Design
Day 5
Plasmid Extraction
Plasmid Quantification
Plasmid Digest
Gel Electrophoresis of Digested Plasmids
Analysis
Week 2
Day 6
PCR
Gel Electrophpresis
Gel Extraction
Restriction Digest
Recovery of pSB-1C3 vector
EZNA gel extraction protocol on recovered pSB1c3 vector
Growth and culture of E. coli transformed J and G
Day 7
Yesterday
Miniprep
Nanodrop
Plasmid Digest
Stock Solutions
PCR Amplification with New Primers
Day 8
Gel Electrophoresis
Restriction Digest
Plasmid Design
Transformation
Day 9
Plasmid Digest
Enzymatic Reaction  Clean-up
Growth and Culture
Ligation
Preparation of DH5
Day 10
Plasmid for PCR Products
Preparation of DH5
Week 3
Day 11
Overnight Cultures
PCR
Restriction Digest
Primer Preparation
Sequencing
Running the Gel
Day 12
NanoDrop
Restriction Digest
Gel Extraction
Plasmid Extraction
Gel Electrophoresis
Analysis of Sequencing Data
Ligation
Day 13
To Do
To Note
QuickChange
Transformation
Restriction Digest
Gel Electrophroesis
Sequencing
Extra
Interlab
Week 2
Day 8
Preparation of Interlab Study BioBricks
Transformation
Day 9
Growth and Culture of Bacteria
Day 10
Plasmid for PCR Product
Week 3
Day 11
Restriction Digest
Day 12
Gel Extraction
Made with love by Oxford iGEM
