<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/d/dc/Ox_Day1Leon.JPG)"><H3>Experiments</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="section" id="intro"><DIV class="slim"><H2 style="text-align:left">Introduction</H2><DIV class="image image-full" style="object-align:center"><P>
                            How our 3-part engineered microbe works:
                                1. Constant secretion of biofilm-degrading enzyme
                                2. Production and accumulation of antibacterial Art-175
                                3. Detection of pathogenic bacteria via quorum sensing
                                4. Permeabilization of inner membrane by T4 Holin
                                5. Access and lysis of host cell wall by Art-175
                                6. Release of Art-175 and lysis of target cell
                            
                        </P></DIV><P>
                        Our enzymatic approach to the treatment of urinary tract infections (UTIs) is centred on the design of a &quot;pathogen killing&quot; engineered microbe which can in principle be incorporated into the urinary microbiome of UTI patients to achieve not only a therapeutic effect, but also potentially a prophylactic effect. The engineered microbe needs to contain three key features:
                        <UL><LI>Constant secretion of <STRONG>biofilm-degrading enzymes</STRONG> - degrading the biofilms of the pathogenic bacteria reduces their resistance towards antibiotics</LI><LI>Production and intracellular accumulation of enzymes that can <STRONG>kill both the pathogenic bacteria and our engineered microbial host</STRONG> upon release into the extracellular medium</LI><LI>A <STRONG>quorum sensing mechanism</STRONG> that triggers the release of the antibacterial enzymes in the presence of pathogenic bacteria</LI></UL></P><P>
                        Due to constraints in time and resources, we focused our experimental efforts towards the development of proof-of-concepts for only the first two features.
                    </P><P>
                        Through our experimental work with secretion assays, biofilm assays, and cell-killing assays we were able to obtain preliminary in vitro data suggesting that the <A href="https://2015.igem.org/Team:Oxford/Parts">BioBrick parts</A> which we designed to allow our microbial host to produce the relevant biofilm-degrading enzymes and bacteria-killing enzymes are indeed able to function as expected, exerting antibiofilm and bactericidal activity against bacterial strains closely related by species and/or genus to the pathogens involved in catheter-associated urinary tract infections.
                    </P><P>
                        In this section we will discuss only the experimental data that contributes directly to the final delivery design of the therapeutics. The remainder of the characterization data for each part can be found at the respective <A href="https://2015.igem.org/Team:Oxford/Parts">part pages</A>.
                    </P></DIV></DIV><DIV class="section" id="strains"><DIV class="slim"><H2 style="text-align:left">Bacterial Strains and Growth Cultures</H2><P><I>E. coli</I> DH5α was used for all cloning purposes. The <I>E. coli</I> strains MG1655 and RP437 ∆FliC, as well as the multi-effector knockout BSL-1 strain of <I>Y. enterocolitica</I>, IML421asd, were used as expression hosts. Cultures for cloning were grown in antibiotic-supplemented Lysogeny Broth (LB) at 37°C. The <I>E. coli</I> expression host cultures were grown in antibiotic-supplemented Lysogeny Broth (LB) at 37°C, while cultures of <I>Y. enterocolitica</I> IML421asd were grown in Brain Heart Infusion (BHI) media supplemented with diaminopimelic acid (DAP) and the appropriate antibiotic at 30°C.
                    </P><P>
                        Bacterial cultures were grown overnight (16-20 hours) to stationary phase before being subcultured for characterization experiments.
                </P></DIV></DIV><DIV class="section" id="cloning"><DIV class="slim"><H2 style="text-align:left">Plasmid Construction &amp; Gene Cloning</H2><P>
                        Each gene sequence for our <A href="https://2015.igem.org/Team:Oxford/Parts">parts</A> was directly synthesized pre-fused with 1) a sequence coding for a hexahistidine tag downstream of it, and 2) the BioBrick prefix and suffix sequences containing the EcoRI/XbaI and SpeI/PstI restriction sites attached upstream and (further) downstream of it respectively through IDT. The sequences were amplified using PCR and inserted into the <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> BioBrick standard vector backbone before subsequently being cloned into <I>E. coli</I> DH5α for plasmid storage and submission to the <A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2015&amp;group=Oxford">Registry</A>.
                    </P><P>
                        For gene expression studies, our parts, contained in the pSB1C3 backbone, were extracted via Miniprep. The NcoI restriction site was introduced upstream of each of our gene sequences (except BBa_K1659501 and BBa_K1659601) via PCR to facilitate their insertion into the arabinose-inducible <A href="https://www.thermofisher.com/order/catalog/product/V43001">pBAD/HisB</A> commercial expression vector, and the insert-containing expression vectors were subsequently cloned separately into the standard laboratory <I>E. coli</I> K-12 strain MG1655 as well as a chemotaxis knockout strain <I>E. coli</I> RP437 ∆FliC.
                    </P><P>
                        In the later stages of experimentation, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659003">BBa_K1659003</A>[pBAD] was also cloned into <I>Y. enterocolitica</I><A href="http://elifesciences.org/content/2/e00792">IML421asd</A> for further characterization of gene expression.
                    </P></DIV></DIV><DIV class="section" id="biofilm"><DIV class="slim"><H2 style="text-align:left">Biofilm-degrading Enzymes</H2><P>
                        According to the design requirements, our genetic constructs coding for biofilm-degrading enzymes need to achieve the following:
                        <UL><LI>Trigger the production of biofilm-degrading enzymes and facilitate the secretion of said enzymes from the engineered bacteria</LI><LI>Ensure that the enzymes, after secretion into the extracellular medium, are still correctly folded such that they retain their enzymatic biofilm-degrading function</LI></UL></P><P>
                        The parts which we characterized for this section of the design are <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659211">BBa_K1659211</A> (DsbA-DspBx) and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659301">BBa_K1659301</A> (DsbA-DNase).
                    </P><DIV class="section" id="secretion"><H3>Secretion Assaying</H3><DIV class="image image-full" style="object-align:center"><P>
                                Click image for full protocol, including the recipes for the buffers used to purify each protein.
                            </P></DIV><P>
                            Stationary cultures of MG1655 DsbA-DspBx[pBAD] and MG1655 DsbA-DNase[pBAD] were subcultured into fresh media at a 1:20 ratio and enzyme secretion was induced for 4 hours using L-arabinose. Subsequent purification of protein from the cell-free supernatant and visualization using SDS-PAGE confirms that <STRONG>proteins of the expected size are present in the supernatant</STRONG> and hence most likely successfully secreted by the respective engineered bacterial strains.
                        </P><DIV class="image image-full" style="object-align:center"><P>
                                Left figure: Lane D = DsbA-DspB, ~45 kDa; Right figure: Lane C = DsbA-DNase, ~22 kDa
                            </P></DIV></DIV><DIV class="section" id="antibiofilm"><H3>Biofilm Degradation</H3><P>To study the effects of the secreted enzymes on <I>E. coli</I> biofilms, we conducted two types of experiments:</P><UL><LI><A href="https://2015.igem.org/Team:Oxford/Protocols#4-1">Biofilm viability assays</A> which investigated whether the secretion of the biofilm-degrading enzymes can prevent the therapeutic engineered bacteria, which are <I>E. coli</I> strains themselves, from forming biofilms.</LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols#4-2">Biofilm inhibition assays</A> which investigated whether the enzyme-secreting cells can inhibit control cells which do not secrete the enzymes from forming biofilms.</LI><DIV class="section" id="biofilmviability"><H4>Biofilm viability assay</H4><P>
                                Secretion of functional biofilm-degrading enzymes should <STRONG>prevent the engineered bacteria from forming their own biofilms</STRONG> or at least reduce the amount of biofilms which they form. To investigate the validity of this hypothesis, 1:200 subcultures of enzyme-secreting cells were grown for 72 hours with different concentrations of arabinose inducer and the results were compared against control cells grown with the same set of arabinose concentrations.
                            </P><DIV class="image image-full" style="object-align:center"><P>
                                    Crystal violet staining results after 72 hours; high expression of DsbA-DNase and DsbA-DspB (0.2%) result in significantly reduced biofilm formation
                                </P></DIV><P>
                                The results show significantly reduced OD<SUB>590</SUB> values, which quantify the amount of stained biofilm (crystal violet dye stains the intact cells encased within the biofilm matrix as well as the biofilm matrix material itself) remaining, in cultures where expression of DsbA-DNase (BBa_K1659301) or DsbA-DspB (BBa_K1659211) are induced to high levels (0.2% arabinose). The correlation suggests that the production and secretion of these enzymes can lead to reduced biofilm formation, and we proceed to investigate whether the enzyme-secreting cells can also inhibit control cells present in the same culture from forming biofilms as well.
                            </P></DIV><DIV class="section" id="biofilminhibit"><H4>Co-culture Biofilm Inhibition Assay</H4><P>
                                Screening of growth times versus density of biofilms grown for the culturing of <I>E. coli</I> biofilms showed that <STRONG>beyond 72 hours, biofilm density plateaus</STRONG> to more or less the same value <STRONG>independent of what the initial subculture dilution was</STRONG> (1:100, 1:200, 1:500, 1:1000 were tested). In view of that, the ability of the enzyme-secreting cells to inhibit control cells from forming biofilms can be tested using a co-culture of the two cell types:
                            </P><UL><LI>
                                    If the gene expression of BBa_K1659301 and BBa_K1659211 merely prevents the engineered bacteria from forming their own biofilms via some alternative intracellular mechanism instead of triggering the secretion of functional biofilm-degrading enzymes, the control cells in co-culture would still be able to form biofilms and over a time course of &gt;72 hours achieve plateau biofilm density.
                                </LI><LI>
                                    If the gene expression actually triggers the secretion of functional biofilm-degrading enzymes, the control cells in co-culture would form a lower biofilm density by the end of the time course.
                                </LI></UL><P>
                                To test the hypothesis, co-cultures containing 4 parts engineered bacterial cells and 1 part control cells were set up using the same growth and analysis protocol as the biofilm viability assays above, for a growth time course of 120 hours:
                            </P><DIV class="image image-full" style="object-align:center"><P>
                                    DsbA-DspB gene expression reduces biofilm density formed by control cells in co-culture with enzyme-secreting cells
                                </P></DIV><P>
                                While DsbA-DNase failed to produce consistent enough results for conclusions to be drawn, DsbA-DspB was able to bring about a ~50% reduction in biofilm density in co-culture when compared against the controls.
                            </P></DIV><DIV class="section" id="fremarks"><H3>Section Remarks</H3><P>
                            Overall, this section serves to provide some preliminary proof-of-function for the antibiofilm parts DsbA-DNase (BBa_K1659301) and DsbA-DspB (BBa_K1659211) by showing that:
                        </P><UL><LI>Both the enzymes are successfully secreted</LI><LI>Gene expression for both enzymes can inhibit biofilm formation by the <I>E. coli</I> expression host</LI><LI>Gene expression for DsbA-DspB can inhibit the formation of biofilm by control cells in co-culture</LI></UL><P>
                             However, the results above cannot rigorously establish a causal link between biofilm inhibition and enzymatic activity as biofilm inhibition experiments using purified solutions of the two enzymes were not performed.
                        </P><P>
                            On top of that, it has yet to be established as to whether the secreted enzymes are able to degrade bacterial biofilms which were already pre-formed, as well as whether at experimental gene expression levels are the secreted enzymes able to degrade biofilms to an appreciable enough rate for the engineered microbe to be able to work effectively in a real-life therapeutic setting.                            
                        </P></DIV><DIV class="section" id="antibacterials"><DIV class="slim"><H2 style="text-align:left">Antibacterial Enzymes</H2><P>
                        As outlined in the design aim, the therapeutic engineered bacteria need to be able to <STRONG>produce and accumulate antibacterial enzymes intracellularly</STRONG>, and upon detecting the presence of a high concentration of pathogenic bacteria <STRONG>release the accumulated enzymes in a high-concentration pulse</STRONG> to simultaneously achieve both pathogen-killing as well as self-killing for the purposes of bacterial population control.
                    </P><P>
                        The artilysin Art-175 (which we will subsequently call Art175), our main antibacterial enzyme of study, is capable of killing Gram-negative bacteria by penetrating through its outer membrane and degrading its cell wall. Our experimental aims with regards to Art-175 are twofold:
                    </P><UL><LI>The engineered bacteria must be able to produce and accumulate Art175 within itself without killing itself.</LI><LI>Once Art175 is exported to the extracellular medium, it must be able to kill both the expression host as well as the target pathogenic bacteria.</LI></UL><P>
                        For the final implementable design, the proposed mechanism of export for Art175 is the <STRONG>permeabilization of the engineered bacteria's inner membrane using T4 Holin</STRONG> (mimicking the holin-endolysin pathway used in nature by bacteriophages to lyse their host bacteria) upon detection of quorum-sensing signals sent out by pathogenic bacteria.
                    </P><P>
                         However, to achieve greater compartmentalization in terms of function-testing we opted to instead to investigate whether Art175 can fulfill the two requirements above through a substitute methodology - instead of using a separate export mechanism, we designed BioBrick parts separately coding for the production of <A href="https://2015.igem.org/Team:Oxford/Parts#Antibacterials">Art175 with and without a secretion tag</A> to <STRONG>mimic its behaviour in the intracellular and extracellular environments</STRONG> respectively.
                    </P><DIV class="section" id="lysis"><H3>Host Cell Lysis Assay</H3><P>
                            Of the three secretion-tagged Art175 parts (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659001">BBa_K1659001</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659002">9002</A>, and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659003">9003</A>), BBa_K1659003, the part containing the YebF tag has been shown to produce the most consistent results and hence will be the subject of comparison against the non-tagged Art175 part, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659000">BBa_K1659000</A>.
                        </P><P>
                            Induction of gene expression for BBa_K1659000 (Art175) and BBa_K1659003 (YebF-Art175) separately in 1:100 subcultures of stationary phase engineered bacteria while tracking the changes in cell density by means of OD<SUB>600</SUB> measurements in a 96-well plate allows us to determine whether Art175 behaves differently in terms of host cell-killing from the inside or outside of the cell:
                        </P><DIV class="image image-full" style="object-align:center"><P>
                                Left: Art175 does not cause cell lysis when accumulated intracellularly; Right: With the YebF secretion tag, Art175 is now able to pass through the cell inner membrane and exert lytic activity on the host cell peptidoglycan. Note that it is unclear whether YebF-Art175 lyses the peptidoglycan on its way out of the cell through the export apparatus, or is full exported first before re-entering the cell to exert its lytic activity, though is mechanistic detail is inconsequential for our purposes.
                            </P></DIV><P>
                            As seen in the results, when compared against two control types (cells that are only cloned with a blank plasmid backbone (blue line) and cells containing the YebF coding sequence but does not have the gene expression induced (orange line)), production of non-secreted Art175 (graph on the left) does not lead to host cell lysis even at high levels of gene expression, whereas gene expression for the secretion-tagged Art175 (graph on the right) leads to significant amounts of host cell lysis some time during the mid-log phase which is presumably when the enzyme starts being produced and secreted in large amounts.
                        </P></DIV><DIV class="section" id="killing"><H3>Target Cell Lysis Assay</H3><P>
                            Having obtained results showing that YebF-Art175 is capable of killing the engineered <I>E. coli</I> expression host cells, we needed an alternative expression host organism that is able to better survive the production and secretion of Art175 such that a high enough cell count within the culture can be maintained. This is to ensure that a <STRONG>high enough concentration of enzymes will be secreted into the extracellular medium, allowing us to test the antibacterial potency of its subsequent cell-free isolate</STRONG> against target pathogen cells.
                        </P><P><I>Y. enterocolitica</I> is speculated to have a cell wall that is more resistant to lysis than <I>E. coli</I> from the fact that in microbiology protocols much more lysozyme is required to effectively catalyze their cell lysis. Being a fellow member of the family Enterobacteriaceae together with <I>E. coli</I>, it should also be able to use the same codons for translation. As such, we opted to use <I>Y. enterocolitica</I> IML421asd, a <A class="definition" title="&lt;i&gt;Y. enterocolitica IML421asd&lt;/i&gt;" data-content="IML421asd is created from the BSL-2 strain O:9 E40 by Cornelis, G.R. et al by deleting 6 of the 7 Yop effectors which confer virulence to the &lt;i&gt;Y. enterocolitica&lt;/i&gt; strain as well as the chromosomal gene &lt;i&gt;asd&lt;/i&gt; essential for peptidoglycan biosynthesis, making it avirulent and auxotrophic.">Biosafety Level 1</A> of the bacteria, as our alternative expression host for BBa_K1659003.
                        </P><P>
                            We cloned BBa_K1659003[pBAD] into IML421asd and induced gene expression when the subculture was at late-exponential growth phase using 0.2% arabinose to trigger its secretion of YebF-Art175 and found that IML421asd <A href="http://parts.igem.org/Part:BBa_K1659003#Characterization">survives far better under gene expression than <I>E. coli</I> MG1655</A>. Assuming that the cloned gene causes IML421asd to produce YebF-Art175 at a comparable level to that of <I>E. coli</I> MG1655, this provides preliminary evidence that IML421asd is a suitable substitute expression host for the gene.
                        </P><P>
                            Following that, we isolated the cell-free supernatant of a subculture of BBa_K1659003[pBAD] IML421asd which was subjected to 0.2% arabinose-induced gene expression for 4 hours. We incubated subcultures of wild-type <I>P. putida</I>, which we used as a safer substitute target organism in place of pathogenic <I>P. aeruginosa</I>, in the YebF-Art175-containing supernatant and compared its cell density (OD<SUB>600</SUB>) changes against controls which do not contain the enzyme:
                        </P><DIV class="image image-full" style="object-align:center"><P>
                                The &quot;non-induced control&quot; contains a subculture of BBa_K1659003[pBAD] IML421asd grown with 0% arabinose under otherwise the same conditions as the gene expression culture.
                            </P></DIV><P>
                            We found that the supernatant where gene expression for YebF-Art175 was previously induced is able to significantly inhibit the cell growth of <I>P. putida</I>. 
                        </P><P>
                            Taking the fact that arabinose-induced gene expression does lead to some decline in the cell density of BBa_K1659003[pBAD] IML421asd as evidence that YebF-Art175 is indeed being produced and exported functionally, these results suggest that the YebF-Art175 present in the supernatant is inhibiting the growth of <I>P. putida</I> through the exertion cell wall lytic activity.
                        </P></DIV><DIV class="section" id="persister"><H3>Persister Cell Lysis Assay</H3><P>
                            The results above provide preliminary evidence suggesting that YebF-Art175 is able to kill planktonic <I>P. putida</I>. However, given that the <STRONG>main complication in UTIs is the persister cells in biofilms being highly resistant to antibiotics</STRONG>, it would be greatly beneficial to UTI treatment if Art175 was able to kill said persister cells while they are still encased within their biofilms.
                        </P><P>
                            With that in mind, we isolated the cell-free supernatant of YebF-Art175 expression cultures as per the target cell lysis assay above, but this time instead of incubating planktonic subcultures of <I>P. putida</I> in it, we filled the wells of a 96-well plate containing pre-formed <I>P. putida</I> biofilms with said supernatant to investigate its persister cell-killing ability.
                        </P><P>
                            We again used crystal violet staining to quantify the number of intact cells remaining within the biofilm after incubation by measuring OD<SUB>590</SUB>, comparing it against biofilms incubated with non-expression control cultures:
                        </P><P>
                            The results show a ~30% decrease in number of intact cells stained when the biofilms were incubated with the supernatant from the YebF-Art175 gene expression culture, suggesting that the secreted YebF-Art175 has achieved some degree of success in lysing biofilm encased cells.
                        </P></DIV><DIV class="section" id="lremarks"><H3>Section Remarks</H3><P>
                            Overall, we found that:
                        </P><UL><LI>Art175 gene expression does not lead to host cell lysis in <I>E. coli</I></LI><LI>YebF-Art175 gene expression leads to host cell lysis in <I>E. coli</I></LI><LI>YebF-Art175 gene expression provides supernatant that can inhibit growth of planktonic <I>P. putida</I></LI><LI>YebF-Art175 gene expression provides supernatant that can kill biofilm-encased <I>P. putida</I> cells to a certain extent</LI></UL><P>
                            There is still much to be investigated in terms of the cell lytic function of Art175 before conclusive statements can be made - for one, the lack of protein purification and visualization data for the enzyme prevents one from conclusively saying that YebF-Art175 is being produced, and following that the cell lysis observed from supernatant studies cannot be definitively attributed to the action of Art175.
                        </P><P>
                            The lack of granularity in our investigation of lytic activity against biofilm-encased cells also prevents us from concluding that the cells killed are definitely persister cells, as it is known that not all cells in a biofilm are metabolically-inactive and hence more informative experiments need to be conducted before one can tell exactly the nature of the cells being killed.
                        </P><P>
                            In spite of that, in view of our three findings above, Art175 remains an ideal candidate for the fulfillment of the design requirement for the antibacterial component of our microbial machine.
                        </P><P>
                            We were also at one point interested in the antibacterial peptide MccS, but because its mechanism of action could not fit into the holin-endolysin design which we needed, we decided not to pursue it further. Information about MccS can be found <A href="https://2015.igem.org/Team:Oxford/Experiments/MccS">here</A>.
                        </P></DIV></DIV></DIV><DIV class="section" id="conc"><DIV class="slim"><H2 style="text-align: left">Conclusion</H2><P>
                    Through our experimental work, we have acquired extensive preliminary evidence suggesting that the BioBrick parts making up antibiofilm and antibacterial components in the design of our UTI-treating engineered microbe should be able to function as expected.
                </P><P>
                    Other than the needed improvements in experimental design to allow the drawing of more rigorous conclusions as outlined in the respective Section Remarks above, further work on our design should be focused on:
                </P><UL><LI>
                        Testing the antibiofilm and antibacterial function of our Parts in environments more properly simulating a pathogenic context - bacterial biofilms behave very differently when formed on the smooth plastic surface of a 96-well plate as compared to when they're formed on human tissue, as such the therapy's medical potential can only be properly verified through experiments designed with such detail in mind.
                    </LI><LI>
                        Studying how the antibiofilm and antibacterial can work together to achieve greater bacterial-killing capacity.
                    </LI><LI>
                        Creating parts for and implementing the final quorum sensing component to achieve a complete, responsive, functioning engineered microbe.
                    </LI></UL></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#intro">Introduction</A></LI><LI><A href="#strains">Bacterial Strains &amp; Growth Cultures</A></LI><LI><A href="#cloning">Plasmid Construction &amp; Gene Cloning</A></LI><LI><A href="#biofilm">Biofilm-degrading Enzymes</A></LI><LI><A href="#secretion">Secretion Assaying</A></LI><LI><A href="#antibiofilm">Biofilm Degradation</A></LI><LI><A href="#biofilmviability">Biofilm Viability Assay</A></LI><LI><A href="#biofilminhibit">Co-culture Biofilm Inhibition Assay</A></LI></UL><LI><A href="#fremarks">Section Remarks</A></LI><LI><A href="#antibacterials">Antibacterial Enzymes</A></LI><LI><A href="#lysis">Host Cell Lysis Assay</A></LI><LI><A href="#killing">Target Cell Lysis Assay</A></LI><LI><A href="#persister">Persister Cell Lysis Assay</A></LI><LI><A href="#lremarks">Section Remarks</A></LI><LI><A href="#conc">Conclusion</A></LI></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>