<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Test_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Test/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/d/dc/Ox_background.JPG)"><H3>Protocols</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="slim"><DIV class="section" id="1"><H2>1.0 PCR (Polymerase Chain Reaction)</H2><P>
                        PCR is used to amplify a specific region of DNA.
                    </P><P>
                        Before starting:
                        <UL><LI>Defrost DNA templates and primers</LI><LI>Use the 150\(\mu\)l aliquots of the Q5 Master Mix that are stored in the iGEM box in the 4℃ cold room. This avoids repeat freeze-thaw of the stock solution. Bring ice bucket to the cold room to bring Q5 to the bench.</LI><LI>Label PCR tubes</LI></UL></P><P>
                        Reaction Mix:
                        <TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/\(\mu\)l</TH><TH>Final conc/nM</TH></TR><TR><TD>Q5 HF Master Mix</TD><TD>12.5</TD><TD>-</TD></TR><TR><TD>10\(\mu\)M Forward Primer</TD><TD>1.25</TD><TD>500</TD></TR><TR><TD>10\(\mu\)M Reverse Primer</TD><TD>1.25</TD><TD>500</TD></TR><TR><TD>1ng/\(\mu\)l<SUP>-1</SUP></TD><TD>1.0</TD><TD>-</TD></TR><TR><TD>Milli-Q Water</TD><TD>9.0&gt;</TD><TD>-</TD></TR></THEAD></TABLE></P><P>
                        Reaction protocol:
                    </P><P>
                        This is an example, times may vary based on the polymerase used.
                        <TABLE class="table table-striped"><THEAD><TR><TH>Stage</TH><TH>Number of Cycles</TH><TH>Temperature/℃</TH><TH>Time/min</TH></TR><TR><TD>Initial denaturation</TD><TD>1</TD><TD>98</TD><TD>2</TD></TR><TR><TD>Denaturation</TD><TD>-</TD><TD>98</TD><TD>0.5</TD></TR><TR><TD>Annealing</TD><TD>25</TD><TD>Annealing temp.</TD><TD>0.5</TD></TR><TR><TD>Extension</TD><TD>-</TD><TD>72</TD><TD>0.5 per 1kb</TD></TR><TR><TD>Final extension</TD><TD>1</TD><TD>72</TD><TD>5</TD></TR></THEAD></TABLE></P><P>
                        During and after preparing tubes:
                        <UL><LI>Make sure that the primer and small amounts of DNA do not stick onto the side of the tube or tip</LI><LI>Use the calculated annealing temperature</LI></UL></P><P>
                        You are always going to want to run the PCR products on a gel. Set up the gel once you have started the PCR.
                    </P><P>
                        When the PCR has finished:
                    </P><P>
                        Add blue loading dye to each PCR tube (be careful not to add the purple ladder accidentally) according to the volume of PCR product you are running and type of stain. For example, if you are using the 5x dye and you're running 20\(\mu\)l PCR product, add around 4\(\mu\)l dye. '5x' refers to the total DNA solution volume compared to the loading dye.
                    </P><P>
                        Reaction mix for Phusion:
                        <TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/\(\mu\)l</TH></TR><TR><TD>Phusion Buffer</TD><TD>10</TD></TR><TR><TD>Phusion enzyme</TD><TD>0.5</TD></TR><TR><TD>dNTPs</TD><TD>5</TD></TR><TR><TD>Template</TD><TD>0.5</TD></TR><TR><TD>Milli-Q</TD><TD>34</TD></TR></THEAD></TABLE></P><DIV id="1-1"><H3>1.1 Gel Electrophoresis</H3><P><TABLE class="table table-striped"><THEAD><TR><TH>Fragment Size</TH><TH>Agarose gel w/v %</TH><TH>Mass of agarose in 200ml 0.5x TBE/g</TH></TR><TR><TD>&gt;3kb</TD><TD>0.8</TD><TD>1.6</TD></TR><TR><TD>&lt;1kb</TD><TD>2</TD><TD>4</TD></TR><TR><TD>In between</TD><TD>1</TD><TD>2</TD></TR></THEAD></TABLE></P><P>
                            For a large 1% gel, prepare 200mL agarose
                            <OL><LI>Heat 2g agarose in 200ml 0.5x TBE for 2 minutes under full power in the microwave (use a 500mL Duran bottle, and place a weighing boat underneath it to prevent the causing of a mess in the event the mixture boils over; DO NOT fully tighten the Duran cap).</LI><LI>Check if the agarose has been fully dissolved. Heat it further if gel strands are visible.</LI><LI>Hold the lid with paper and <I>gently</I> swirl</LI><LI>Leave the agarose solution to cool at 50℃ for 20 minutes.</LI><LI>Pour agarose onto gel plate in a setting tray with appropriately-sized combs already fixed onto it, and leave for 20 minutes to let it set.</LI><LI>When the agarose has set, remove the combs and transfer the gel plate from the setting tray to the electrophoresis chamber.</LI><LI>Flood the gel plate with 0.5x TBE buffer up until right above the top of the wells.</LI><LI>The gel should be positioned such that the positive (red) electrode is on the far side of the gel from the wells, as the negatively-charged DNA will migrate towards the positive electrode.</LI><LI>Load 10\(\mu\)L DNA ladder in lane 1 and 20\(\mu\)L PCR product in subsequent lanes</LI><LI>120V for a big gel (200mL agarose) or 80V for a small gel(100mL agarose)</LI></OL></P><DIV id="1-1-1"><H4>1.11 Staining the Gel</H4><OL><LI>Pick up the gel keeping it flat and allow the excess buffer to run off</LI><LI>Using your hands, slide gel carefully into a vat of ethidium bromide</LI><LI>Set the vat to gentle shaking for 30/40 minutes</LI><LI>Pick up the gel <STRONG>using a spatula</STRONG> and rinse off the ethidium bromide</LI></OL></DIV><DIV id="1-1-2"><H4>1.12 Visualizing DNA using UV Transilluminator</H4><OL><LI>Place the gel on the transilluminator stage and adjust stage height appropriately.</LI><LI>Set the transilluminator using the GeneSnap program such that the light emitted is UV (instead of white light) and the software filter is configured to pick up EtBr fluorescence.</LI><LI>Adjust the contrast such that the bands can be clearly seen.</LI><LI>Adjust the focus using the focusing rings to sharpen the image.</LI><LI>Save the image in the naming format “dd_mm_yy” to Disk C: → Lab users → iGEM in .sdg file format, and additionally export it as a .tif file.</LI><LI>Print a picture off for your own records</LI><LI>Label eppendorfs according to successful bands</LI><LI>Excise bands and slide into appropriate eppendorf</LI></OL></DIV><DIV id="1-1-3"><H4>1.13 Extraction of DNA (PCR product) from Gel</H4><P>
                                Remember, when spinning tubes with their lids open, place them so that lids are pointed away from the direction of spinning.
                                <OL><LI>Zero the weighing scale to weight of eppendorf</LI><LI>Weigh each of the bands</LI><LI>Dissolve excised chunks in a minimum of 1mL of XP2 Binding Buffer per gram of gel</LI></OL></P><P>
                                Green box on our shelf - E.Z.N.A Gel Extraction Kit made by Omega Bioteck, according to the Spin Protocol.
                            </P><P>
                                Elute PCR products into 30\(\mu\)l and plasmid DNA into 50\(mu\)l.
                            </P></DIV></DIV><DIV id="1-2"><H3>1.2 Restriction Digest PCR or Plasmid DNA</H3><UL><LI>Use enzymes and buffer according to &quot;Master Table&quot;</LI><LI>Defrost and shake buffers</LI><LI>Keep enzymes in yellow freezing block and keep out of freezer for as short a time as possible</LI></UL><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/\(\mu\)l</TH></TR><TR><TD>DNA</TD><TD>30</TD></TR><TR><TD>Buffer</TD><TD>5</TD></TR><TR><TD>EcoRI-HF</TD><TD>0.5</TD></TR><TR><TD>SpeI</TD><TD>0.5</TD></TR><TR><TD>Milli-Q Water</TD><TD>14</TD></TR></THEAD></TABLE><P>
                            A point to note concerning the volume of restriction enzyme
                            <UL><LI>0.5\(\mu\)l for PCR DNA</LI><LI>1.0\(\mu\)l for Plasmid DNA </LI><LI>However, if you are doing a test digest (i.e. after a mini-prep) use 0.5\(\mu\)l enzyme, despite digesting a plasmid</LI></UL><OL><LI>Incubate at 37℃ for 2 hours (ThermoMixer program 3) with shaking at 300rpm.</LI><LI>Heat inactivated for 30 minutes at 95℃</LI><LI>Dephosphorylate the plasmid using 1\(\mu\)l CIP at 37℃ for 3 minutes</LI></OL></P><DIV id="1-2-1"><H4>1.21 DNA 'Clean Up' using EZNA Enzymatic Reaction Kit</H4><P>
                                Protocol can be found at the end of ENZA gel extraction booklet.
                            </P><P>
                                Elute PCR products into 30\(\mu\)l and plasmid DNA in 50\(\mu\)l.
                            </P></DIV><DIV id="1-2-2"><H4>1.22 Nanodrop</H4><OL><LI>Clean stage with 1\(\mu\)l water and tissue</LI><LI>Make a blank reading using 1\(\mu\)l of water and wipe off</LI><LI>Make another blank reading using 1\(\mu\)l of elution buffer and wipe off</LI><LI>Measure concentration of 1\(\mu\)l of each sample</LI></OL></DIV></DIV><DIV id="1-3"><H3>1.3 Ligation</H3><H4>Overnight protocol</H4><P>
                            Defrost T4 DNA Ligase on ice.
                        </P><P>
                            Keep in freezing block when on bench and add last to the reaction mixture.
                        </P><P>
                            Mass of vector DNA : Mass of insert DNA roughly 1:3
                        </P><P>
                            Generally, because you only get 50\(\mu\)L from the plasmid digest, split evenly between how many inserts there are.
                        </P><P>
                            The component volumes are:
                        </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume\(\mu\)l</TH></TR><TR><TD>Digested DNA (gBlock)</TD><TD>29</TD></TR><TR><TD>Digested pSB-1C3</TD><TD>7</TD></TR><TR><TD>T4 DNA Ligase Buffer</TD><TD>5</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD>Milli-Q</TD><TD>8</TD></TR></THEAD></TABLE><P>
                            Incubate at 16℃ overnight.
                        </P></DIV><DIV id="1-4"><H3>1.4 Preparation of Competent E.coli Cells</H3><H4>Overnight protocol</H4><P>
                            DH5ɑ is stored at -80℃
                        </P><OL><LI>Defrost and inoculate in 5mL of LB in a 125 mL conical flask (volume of LB 10% of flask volume so as to achieve sufficient aeration)</LI><LI>Grow culture overnight at 37℃</LI></OL><H4>Next Day</H4><P>
                            Turn on the centrifuge and cool to 4℃
                        </P><P>
                            Always keep TFBI and TFBII on ice.
                        </P><P>
                            Important to keep volumes accurate; else your cells will grow at different rates and ODs will be all over the place.
                        </P><OL><LI>Transfer 1ml of overnight culture into 100ml LB in a 500ml conical flask.</LI><LI>Incubate at 37℃ until OD<SUB>600nm</SUB> reaches 0.4-0.6. It is good to stop at OD = 0.35, as the bacteria are now replicating exponentially i.e. will only take 20 more minutes until OD = 0.7 (which is far too high).</LI><LI>While incubating, pre-chill 20 eppendorfs and 2x50mL Falcon tubes (which are found in the cupboard next to the sink) to 4℃ in ice bucket.</LI><LI>Once OD is correct, decant culture into 2 x 50ml falcon tubes. <STRONG>Once this is done, the cells must never be higher than 4℃.</STRONG></LI><LI>Centrifuge at 2000 rpm at 4℃ for 20 mins. Make sure you are at the centrifuge when this 20 minutes is up to rapidly proceed to the next step. Close the centrifuge lid to maintain 4℃.</LI><LI>Discard supernatant and resuspend pellets in 1ml TFBI.</LI><LI>Add further 10ml TFBI using 10ml electronic pipette.</LI><LI>Leave on ice for 20 mins (or more).</LI><LI>Centrifuge at 2000 rpm at 4℃ for 10 mins (shorter duration as cells are already now permeabilized).</LI><LI>Discard supernatant.</LI><LI>Gently resuspend pellet in 2ml TFBII i.e. don’t vortex, just flick pellets in the TFBII, which has glycerol for frost protection (prevent crystal formation which lyses cells).</LI><LI>Now we have total 4mL of cell-TFBII suspension. Aliquot ~200ul into ~20 pre-cooled eppendorf tubes.</LI><LI>Store at -80℃ in the freezer in the back room.</LI></OL></DIV><DIV id="1-5"><H3>1.5 Transformation</H3><H4>Overnight Protocol</H4><P>
                            To transform cells you need to prepare agar plates. This can be done when you have a spare hour during the day. Plates can be left on the bench (lid side down) until you need them. Plates can also be made during the transformation protocol.
                        </P><P>
                            200ul aliquots of competent cells are stored at -80C in the back. These take around 30 minutes to defrost and so these need to be taken out of the freezer and put on ice to defrost. 100ul of competent cell = 1 transformation.
                        </P><H4>Preparing the cells for transformation</H4><P>
                            For ligated DNA:
                        </P><OL><LI>Put the eppendorfs of ligated DNA into ice to cool to 4℃.</LI><LI>Draw 100\(\mu\)l of competent cells of each aliquot to each 50ul ligation product.</LI><LI>Incubate on ice for 30 minutes. If you are preparing plates as you go along, the LB agar should be done microwaving at this point.</LI><LI>Bring ice bucket to the water bath.</LI><LI><STRONG>Heat shock</STRONG> - transfer straight from ice bucket into 45℃ water bath for 45 seconds.</LI><LI>Transfer back to ice for 1 minute.</LI><LI> Add 800\(\mu\)l LB broth to each tube and incubate at 37℃ gently shaking for 1 hour.</LI></OL><P>
                            For plasmid DNA:
                        </P><OL><LI>Thaw the plasmid DNA for transformation or use ligation product.</LI><LI>Pre-chill number of eppendorfs to match number of transformations.</LI><LI>Thaw appropriate number of 200\(\mu\)l aliquots of competent E. coli cells to prepare 100\(\mu\)l eppendorfs to match number of transformations.</LI><LI>Draw 100\(\mu\)l out of each aliquot (competent cells) and transfer into empty pre-chilled eppendorfs.</LI><LI>Transfer the full volume (usually ~50\(\mu\)l) of the plasmid DNA into the respective E. coli aliquots (we use all of it as we expect the plasmid concentration to be relatively low). For the stock pSB1C3, add only 1\(\mu\)L of the plasmid to the 100\(\mu\)L of cell suspension as the plasmid is at much higher concentration than for e.g. ligation products.</LI><LI>Incubate on ice for 30 minutes. If you are preparing plates as you go along, the LB agar should be done microwaving at this point.</LI><LI>Bring ice bucket to the water bath.</LI><LI><STRONG>Heat shock</STRONG> - transfer straight from ice bucket into 45℃ water bath for 45 seconds.</LI><LI>Transfer back to ice for 1 minute.</LI><LI>Add 800\(\mu\)l LB broth to each tube and incubate at 37℃ gently shaking for 1 hour.</LI></OL><DIV id="1-5-1"><H4>1.51 Preparing the Plates</H4><H6>In the laminar flow hood</H6><OL><LI>LB is in agar form on the shelf</LI><LI>Label the bottle</LI><LI>Loosen the lid, place on plastic dish and microwave on simmer for 20 mins. Melt LB agar as cells are being thawed. Each 500mL bottle of agar makes ~20 plates</LI><LI>Prepare 30 mg/ml chloramphenicol in EtOH e.g. 300 mg chloramphenicol in 10ml EtOH and 100 mg/ml ampicillin solution in MilliQ e.g. 1g ampicillin in 10ml MilliQ</LI><LI>Cool in 50℃ water bath 30 mins for smaller bottle, slightly longer for larger ones. The bottle is cool enough when you can just about comfortably carry it to the laminar flow hood</LI></OL><P>
                                Add x mL media to x uL antibiotic e.g. 100ul chloramphenicol solution to 100ml agar. Pour the plates accordingly. Place close to the wall of the hood to prevent contamination. Agar takes around 20 mins to solidify.
                            </P></DIV><DIV id="1-5-2"><H3>1.52 Spreading Plates</H3><P>
                                How you do this depends a bit on how many plates you are spreading but the idea is to pipette cells onto the plate and spread using glass beads, without leaving too much time between these two steps so that the cells dry in a drop in the middle.
                            </P><H6>Sterile Technique</H6><OL><LI>Pipette 100\(\mu\)l of the cell onto the plate</LI><LI>Flame the mouth of the bottle that contains the glass beads and tip around six onto the plate, flaming the mouth again before putting the lid back on (don’t put the lid down on the bench)</LI><LI>Move the plate until streaks from glass beads fill the plate</LI><LI>Discard the glass beads using the funnel into alcohol (located at the end of the bench near the heating blocks)</LI><LI>Spin down the remaining E. coli (max speed, 1 min) and repeat steps 1-4 onto a separate plate (label so as to distinguish between plates clearly)</LI><LI>Incubate upside down at 37℃ overnight, clearly labelled with date</LI></OL></DIV></DIV><DIV id="1-6"><H3>1.6 Growth and Culture of Bacteria</H3><H4>Overnight protocol</H4><P>
                            This process significantly increases the amount of plasmid that contains biobrick that we want. Plasmids can be extracted later.
                        </P><P>
                            Antibiotics are in the freezer - ampicillin needs defrosting before you start.
                        </P><P>
                            The LB you use has to be transparent (cloudy = contaminated).
                        </P><H6>Sterile Technique</H6><P>
                            The tip of each test tube and Duran bottle must be sterilised with bunsen burner each time when you are preparing the tubes and each time you transfer a colony.
                        </P><P>
                            [Arrange your hands in a way that you have your thumbs free and lids do not get placed on the desk. This is difficult to explain so ask someone who has done it before to show you.]
                        </P><OL><LI>Choose three colonies from each plate. The colony should not be too small or too large and should be reasonably spaced from the others.</LI><LI>Label a test tube for each colony.</LI><LI>Pour 5ml LB broth that has antibiotic to be diluted 1/1000 fold to each tube i.e. if you have 10 plates and you are preparing 30 tubes, you will need 30x5 = 150ml LB with 150\(\mu\)l antibiotic already added. Therefore, each tube contains 5ml LB and 5ul antibiotic</LI><LI>Using inoculation spatulas, pick colony and transfer to appropriate tube. When taking an inoculation spatula from the packet, be careful not to reach into the packet; instead, push handle out of packet to keep sterile</LI><LI>Push down and bring the colony directly into LB without touching the sides</LI><LI>Remove the test tubes from the rack and place in 37℃ overnight</LI></OL></DIV><DIV id="1-7"><H3>1.7 Mini-prep</H3><P>
                            E.Z.N.A. Plasmid DNA Mini Kit I
                        </P><P>
                            If you haven’t done a mini-prep before ask someone who has to go through it with you
                        </P><UL><LI>Carry out all optional steps except equilibration step</LI><LI>Repeat 1st centrifugation (step 2) until all the LB broth has been spun down and all the <I>E. coli</I> have been collected - will help increase yield later</LI><LI>After centrifugation in step 2, pulse tubes before the excess supernatant was removed through pipetting</LI><LI>After addition of solution I/RNAse, vortexing/vigorous shaking of the tubes should be avoided to prevent shearing of nucleus and undesirable accidental extraction of chromosomal DNA.</LI><LI>After addition of solution I/RNAse, resuspension of pellet can be done by dragging the tube along an eppendorf rack.</LI><LI>Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA</LI><LI>After addition of solution II, the waiting time before proceeding to the next step should not be more than 5 minutes</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA</LI><LI>The precipitate formed in following step 7 does not pellet well after centrifugation in step 8, and hence the supernatant needs to be removed immediately to prevent resuspension</LI><LI><P>
                                    Elution step:
                                </P></LI><LI>place elution buffer in 55℃ water bath (50\(\mu\)l per miniprep)</LI><LI>pipette warmed elution buffer onto the HiBind column and let it sit for 3 mins</LI><LI>centrifuge at max speed for 1 minute</LI><LI>pipette up filtrate and pipette back onto HiBind column</LI><LI>centrifuge at max speed again</LI></UL><P>
                            → Nanodrop → Restriction Digest → Gel electrophoresis → Sequence
                        </P></DIV></DIV></DIV><DIV class="slim"><DIV class="section" id="2"><H2>2.0 Toxicity Assay </H2><P><OL><LI>Use 96 well flat bottom plate</LI><LI>Fill each eppendorf with 985 µL LB + antibiotic.</LI><LI>Add 5 µL of stationary culture</LI><LI>Add 10 µL of appropriate Arabanose concentration (e.g. 10ul ara 20% into 1 ml total volume to make 0.2% final conc)</LI><LI>Vortex and then transfer 200 µL to the appropriate well.</LI><LI>For co-culturing, make up control MG1655/ control RP437 with appropriate Arabanose concentration</LI></OL></P></DIV></DIV><DIV class="slim"><DIV class="section" id="3"><H2>3.0 TCA Protein Precipitation</H2><H4>Stock Solutions</H4><P>
                        100% (w/v) Trichloroacetic acid (TCA)
             </P><P>
                        recipe: dissolve 500g TCA (as shipped) into 350 ml dH2O, store at RT
                    </P><H4>Precipitation protocol</H4><OL><LI>Have bacteria grown in appropriate antibiotic-supplemented media to stationary phase the night before.</LI><LI>Dilute an aliquot of the stationary phase bacteria (1/20 dilution) in an appropriate antibiotic-supplemented media and grow to desired OD600 (typically: ~0.6 - 0.8; for our E. coli it takes about 1 - 1.5 hours in LB at 37℃).</LI><LI>Add an appropriate amount of arabinose to achieve a final concentration of 0.2% (e.g. 200\(\mu\)l to 20ml) in the culture and incubate further (1 hour gives some secretion, 4 hours should give extensive secretion).</LI><LI>Spin down 1.5mL of the cell cultures at full-speed for 5 minutes, and transfer 1.35mL of the supernatant into a separate microcentrifuge tube.</LI><LI>Add 150\(\mu\)L of 100% TCA into the supernatant, vortex mix, and centrifuge it at full speed for 15 minutes in the cold room. Put acetone in ice in the meantime.</LI><LI>(Stay in the cold room) After centrifugation is complete, discard the supernatant and add 900\(\mu\)L of ice-cold acetone to each tube. Vortex/shake briefly to wash off remaining TCA from the pellet, and spin down again for 5 to 10 minutes at full speed.</LI><LI>Discard supernatant and let pellet dry for about 30 minutes/place in 95℃ heat block to drive off acetone</LI></OL><DIV id="3-1"><H3>3.1 SDS-PAGE</H3><DIV class="image image-right"><P>Running the gel</P></DIV><OL><LI>Prepare 1x SDS loading dye containing a final NaOH concentration of 50\(\mu\)M.</LI><LI>Resuspend the pellets using 30\(\mu\)L of said dye, and boil at 99℃ for 5 minutes or until the pellet dissolves to give a blue solution.</LI><LI>If the resulting solution is yellow, adding 5\(\mu\)L of 20 mM NaOH should turn it blue. Be careful when opening lids of hot eppendorfs - if the interior of the tube is steamy, let it cool down and subsequently spin it down to pull down the liquid first.</LI><LI>Obtain PAGE gel cassette from cold room, and dilute some 20x SDS buffer (found on top of bench) to 1x (800mL typically needed to fill PAGE tank). NB: Use the tank <STRONG>without</STRONG> the yellow strips on the inside.</LI><LI>PAGE tank can be found in one of the cabinets at Tom’s bench.</LI><LI>Load cassettes and 1x buffer into tank.</LI><LI>Obtain ladder (SDS 2-colour dye) from Jia’s freezer, load 15uL into well in cassette.</LI><LI>Load 15\(\mu\)L of each sample into wells.</LI><LI>Run gel for at least 40 minutes at 100mA per gel.</LI><LI>Once the dyefront is at a desirable position, remove gel from cassette and place it in a square petri dish. Stain with Instant Blue for 1 hour but preferably overnight.</LI></OL></DIV><DIV id="3-2"><H3>3.2 Western Blot</H3><DIV id="3-2-1"><H4>Day 1</H4><P>
                                Follows on from SDS-PAGE
                            </P><UL><LI>In SDS-PAGE, must load 10 μl of benchmark ladder in lane 1 and 10 ul 2-Colour SDS Marker in the lane 2, so that the orientation of the gel can be determined and the efficiency of blotting checked later.</LI><LI>
                                    You will need approx. 1L cold blotting buffer; to make 1L.
                                    </LI><LI>3g Tris base</LI><LI>14.4 g glycine</LI><LI>200 ml methanol</LI><LI>10 ml 10 % SDS</LI><LI>then make up to 1L with water</LI></UL><P><STRONG>How to make a sandwich</STRONG></P><P>
                                Blot on to PVDF membrane using the following layout in your sandwich:
                            </P><DIV class="image image-right"><P>How to Make a Sandwich<SUP>1</SUP></P></DIV><UL><LI>The black face of the blotting cassette is the 'negative' side whilst the white face is the 'positive' side</LI><LI>Construct the sandwich layer by layer, from the black face of the casing upwards to white face</LI><LI>Sandwich must be made whilst it is submerged in a Tupperware box of cold blotting buffer</LI><LI>Use tweezers to handle the layers</LI><LI>Every layer must have air bubbles removed using the plate spreading tool</LI></UL><P><STRONG>Components</STRONG></P><UL><LI>
                                    2 sponges
                                    </LI><LI>Found in cupboard under microwave</LI><LI>Soak in blotting buffer, smooth out bubbles, then place on cassette</LI></UL><LI>
                                    6 (2 layers of 3) pieces of filter paper
                                    </LI><LI>Should be cut to the same size as the gel (8cm x 9cm)</LI><LI>It is found between the drawers under the nanodrop machine.</LI><LI>Soak 3 pieces of paper at once</LI><LI>
                                    Gel from SDS-PAGE
                                    </LI><LI>Remove the gel from the plastic casing, place in blotting buffer and then onto the cassette</LI><LI>
                                    PVDF Membrane
                                    </LI><LI>Found in the labelled drawer under the gel running area</LI><LI>The membrane is white and is protected within two sheets of blue paper</LI><LI>Try not to touch the membrane itself, even with gloves on</LI><LI>Instead handle membrane with tweezers</LI><LI>Before immersing in blotting buffer, soak in methanol (found under fume hood) for 5 minutes</LI><LI>Cut the edge of the PVDF membrane in the top left corner, ie nearest the well of lane 1 (This will allow you to keep track of which side of the membrane you have blotted on to later!)</LI><P>
                                After adding all components according to the diagram above, close the blotting cassette and place in the running tank.
                                <UL><LI>If only one gel is to be blotted then fill the other space with a second cassette containing two blotting pads.</LI></UL></P><P>
                                Insert a cooling block containing frozen Milli-Q.
                                <UL><LI>These are found in the lowest drawer of the fridge next to the iGEM fridge</LI></UL></P><P>
                                Fill tank with cold blotting buffer, place tank in an ice bucket and run at 0.5 A 1 hour.
                            </P><P>
                                Disassemble blotting apparatus and place blot in 5% milk powder in PBS with shaking overnight.
                            </P></DIV><DIV id="3-2-2"><H4>Day 2</H4><P>
                                Rinse blot in PBS.
                            </P><P>
                                Then wash in fresh PBS with shaking, whilst preparing primary antibody solution.
                            </P><P>
                                Prepare primary antibody solution:
                                <UL><LI>This is achieved by diluting the anti-His antibody by 1/1000 into 1% milk powder PBS</LI><LI>10µl aliquots of Ab at 1000x have been prepared, ready for addition to 10ml of 1% milk powder PBS</LI><LI>10 ml is required per blot</LI></UL></P><P>
                                Incubate at room temperature with shaking in a clean square petri dish for at least one hour. Seal the petri dish with parafilm to ensure that the blot does not dry out.
                            </P><P>
                                Rinse the blot with PBS
                            </P><P>
                                Then wash (timings are minimum, not exact, use tweezers when moving blot):
                                <UL><LI>10 mins PBS</LI><LI>10 mins PBS with 0.2 % Tween 20</LI><LI>10 mins PBS with 0.2 % Tween 20</LI><LI>10 mins PBS</LI></UL></P><P>
                                Prepare secondary antibody
                                <UL><LI>Dilute secondary antibody by 1/1000 in 1 % Milk Powder PBS</LI><LI>Secondary antibody is called 'Rabbit anti-mouse', and is found in fridge under the shelves where SDS, glycerol, CaCl solutions etc are stored</LI><LI>10 ml is required per blot</LI></UL></P><P>
                                Incubate membrane with secondary antibody solution at room temperature with shaking for at least 1 hour
                            </P><P>
                                Rinse the blot with PBS and then wash (timings are minimum, not exact):
                                <UL><LI>10 mins PBS</LI><LI>10 mins PBS with 0.2 % Tween 20</LI><LI>10 mins PBS with 0.2 % Tween 20</LI><LI>10 mins PBS</LI></UL></P><P><STRONG>Detection using chemiluminescence</STRONG></P><P>
                                Prepare HRP substrate
                                <UL><LI>The kit for this is found in the shelf immediately on the left as you enter the cold room. It contains two solutions, A and B, which must be mixed in a 1:1 ratio</LI><LI>You must add 0.1 ml of mixture per cm^2 of PVDF membrane you are using</LI><LI>Therefore, as our PVDF should be 8cm x 9cm = 72cm^2, you must use a total of 7.2ml, therefore 3.6ml of each solution</LI></UL></P><P>
                                Using tweezers, lift the blot from the petri dish and then touch the edge of the blot on some blue roll to remove excess liquid.
                            </P><P>
                                Place on a piece of OHP film
                            </P><P>
                                Pour the HRP substrate onto the blot and incubate at room temperature for 5 minutes.
                            </P><P>
                                Lift the blot from the substrate using tweezers and touch the edge of the blot on some blue roll to remove excess liquid.
                            </P><P>
                                Cover the blot with OHP film and remove any air bubbles.
                            </P><P>
                                Place the blot on the gel doc and focus using white light, use the uv ruler to determine that the focus is sharp if necessary
                            </P><P>
                                Make sure that the doc is set to no light and no filter.
                                <UL><LI>Set up multiple exposure runs each of 30s for at least 30 minutes (i.e. 60 exposures).</LI><LI>Make sure the software is set to sum the images (you will get 60 images, each with effectively a longer exposure time).</LI><LI>Save the best image for analysis.</LI></UL></P></DIV></DIV><DIV id="3-3"><H3>3.3 Ni<SUP>2+</SUP> Affinity Chromatography</H3><DIV id="3-3-buff"><H4>Buffers</H4><P>
                                DspB:
                                <UL><LI>Resuspension buffer: 20 mM Tris (pH 8.0), 500 mM NaCl.</LI><LI>Wash buffer: 20 mM Tris (pH 8.0), 500 mM NaCl containing 5 mM imidazole.</LI><LI>Elution buffer: 20 mM Tris (pH 8.0), 500 mM NaCl) containing 100 mM imidazole.</LI></UL></P><P>
                                Art-175:
                                <UL><LI>Resuspension buffer: 20 mM HEPES, 1,000 mM NaCl, 0 mM imidazole [pH 7.4]</LI><LI>Wash buffer: 20 mM HEPES, 1,000 mM NaCl, 20 mM imidazole [pH 7.4]</LI><LI>Elution buffer: 20 mM HEPES, 500 mM NaCl, 500 mM imidazole [pH 7.4]</LI></UL></P><P>
                                DNase:
                                <UL><LI>Resuspension buffer: 50 mM sodium phosphate, pH 8.0, 0.3 M sodium chloridine.</LI><LI>Wash buffer: 50 mM sodium phosphate, pH 8.0, 0.3 M sodium chloridine and 10mM imidazole.</LI><LI>Elution buffer: 50 mM sodium phosphate, pH 8.0, 0.3 M sodium chloridine and 250mM imidazole.</LI></UL></P></DIV><DIV id="3-3-1"><H4>Day 1</H4><P>
                                Set up overnights. Add appropriate antibiotic and grow overnight at 37C.
                            </P></DIV><DIV id="3-3-2"><H4>Day 2</H4><P>
                                First thing in the morning prepare the growth medium - the volume is dependant on how much you wish to purify - 20 mL total with 1 mL of overnight culture. Grow flasks at 37C with shaking at 225 rpm until OD<SUB>600</SUB> is 0.4 - 0.6.
                            </P><P>
                                Induce protein expression at appropriate temperature incubator with 0.2% ara for 4 hours.
                            </P><P><STRONG>Supernatant purification</STRONG><UL><LI>Spin cultures at full speed for 15 minutes</LI><LI>Transfer the supernatant into labelled falcon tubes</LI><LI>Filter supernatant</LI></UL></P><P><STRONG>Lysate purification </STRONG><UL><LI>Keep on ice throughout</LI><LI>Spin culture at full speed for 15 mins and discard the supernatant.</LI><LI>Add 15 ml of resuspension buffer</LI><LI>Vortex</LI><LI>Add 20 mg lysozyme</LI><LI>Sonicate - 4 min, 5 seconds on, 15 seconds off</LI><LI>Filter</LI></UL></P><P><STRONG>Setting Up Columns</STRONG></P><P>
                                Acquire the Ni-NTA Agarose slurry from the fridge and <I>gently</I> swirl until the Nickel beads are resuspended.
                            </P><P>
                                Pour 1 mL Ni-NTA Agarose slurry into a chromatography column. Allow to settle for at least 10 minutes.
                            </P><P>
                                Equilibrate the column with 30 mL resuspension buffer or Milli-Q.
             </P><P>
                                Apply the filtered supernatant or lysate to the equilibrated nickel column.
                            </P><P>
                                Wash the column with wash buffer.
                            </P><P>
                                Elute the protein using the elution buffer. Collect 6 1 ml fractions of the eluate. Protein usually elutes in the first three/four fractions.
                            </P><P>
                                Assay the fractions for protein using, for example, a Bradford assay.
                            </P><P>
                                Pool the fractions containing significant quantities of protein and dialyze overnight (against Water using a 10,000 MW cut-off dialysis membrane; these can be stored at -20C)
                            </P></DIV></DIV></DIV></DIV><DIV class="slim"><DIV class="section" id="4"><H3>4.0 Growing Biofilms<SUP>2</SUP></H3><P>
                        1/100 dilution of overnight stationary culture in broth
                    </P><P>
                        If cells require chemical inducer add the appropriate concentration
                    </P><P>
                        100µl into each well of the 96 well plate of round bottom plates
                    </P><P>
                        Incubate for 24hrs (minimum) at appropriate temperature:
                        <UL><LI>E. coli: 25C (on the bench)</LI><LI>Putida: 30C</LI></UL></P><P>  NB: always grow biofilms in triplet repeats or more, especially if comparing between different plates as biofilms are unreliable
                    </P><P>
                        After incubation:
                        <UL><LI>Decant the culture away to remove the planktonic cells</LI><LI>Wash the cells by submerging in a beaker of Milli-Q, replacing the water as necessary until it turns clear ensuring most of the planktonic cells are gone </LI><LI>Fill each well with 200µl 0.1% crystal violet solution using the multi-channel pipette</LI><LI>Leave to stain for 15-20 minutes</LI><LI>Decant solution into the bucket of Verkon</LI><LI>Wash using above method of submerging into Milli-Q (until the water remains clear)</LI><LI>Leave the plate to dry</LI><LI>At this stage, the plate can be left indefinitely (stained biofilms remain stained)</LI></UL></P><P>
                        To analyse the biofilm, dissolve in 200µl solubilizer and put in plate reader to read a spectra between 500-600nm
                    </P></DIV></DIV><DIV class="slim"><DIV class="section" id="5"><H2>5.0 Preparing Supplemented M9 Media Fluorescence Assay / Microscopy</H2><H4>Component concentrations</H4><P>
                        M9 minimal medium containing glycerol as the carbon source supplemented with -
                        <UL><LI>1 mM thiamine hydrochloride</LI><LI>0.2% casamino acids</LI></UL></P><H4>Protocol</H4><P>
                        For 1L of media:
                        <UL><LI>500ml 2 x M9 salts</LI><LI>10ml 40% glycerol</LI><LI>20ml 10% casamino acids</LI><LI>2ml 1M MgSO<SUB>4</SUB></LI><LI>200µl 0.5M CaCl<SUB>2</SUB></LI><LI>419.8ml Milli-Q</LI></UL></P><P>
                        M9 media stocks can be made by combining above solutions using sterile technique
                    </P><P>
                        Light-sensitive thiamine hydrochloride should be added right before the use of media. Thiamine hydrochloride stock can be prepared using the following method.
                        <UL><LI>
                                3ml 100 mg/ml thiamine hydrochloride (per 1L of M9 media)
                                </LI><LI>Dissolve 100 mg per ml H<SUB>2</SUB>O</LI><LI>Filter-sterilize using a 0.22µm filter</LI><LI>Light-sensitive: stored at -20C in foil in falcon tube (add only before use and fresh with antibiotics</LI></UL></P><P>
                        Add antibiotic(1:1000 dilution) and store in the sterile duran bottle.
                    </P><DIV id="5-1"><H3>5.1 Fluorescence Assay</H3><P>
                            Perform media swap:
                            <OL><LI>Spin down 1 mL stationary phase overnight culture</LI><LI>Discard supernatant</LI><LI>Resuspend in 1 mL unsupplemented M9</LI><LI>1/100 dilution M9 media</LI><LI>Grow to mid-log phase which gives OD value of 0.2-0.5.</LI><LI>100µL mid-log phase culture to each well </LI><LI>Run OD script how you would for toxicity; number of cycles depends on how much time you have but roughly 60-80 cycles</LI><LI>Also set to grow a fluorescence curve</LI><LI>Right click - for toxicity OD 600, put another protocol for GFP </LI></OL></P></DIV></DIV></DIV><DIV class="slim"><DIV class="section" id="6"><H2>6.0 Flow Cytometry</H2><H4>Day 1</H4><P>
                        Pick a single colony of each strain to be tested and set up overnight cultures.
                    </P><H4>Day 2</H4><P>
                        Subculture each strain in a 1 in 20 dilution in LB broth with appropriate antibiotic (chloramphenicol or ampicillin) and grow at 37 oC in the shaking incubator until OD600 = 0.6.
    </P><P>
                        Transfer 1 ml of mid log culture to eppendorfs for measuring in the flow cytometer
                    </P><P>
                        Load the preset settings of FSC 560, SSC420, BLH1 200 and flow rate 100µl/min on the flow cytometer.
                    </P><P>
                        Start the performance test for the flow cytometer with performance beads and machine solutions.
                    </P><P>
                        Vortex the samples and then measure them  by running 1mL of cell culture through the flow cytometer with excitation wavelength at 488nm and emission detected at wavelength 530nm.
                    </P><P>
                        Data was analysed using the Attune Nxt software.
                    </P></DIV></DIV><DIV class="slim"><DIV class="section" id="7"><H2>7.0 Microscopy (and subsequent analysis using MicrobeTracker)</H2><OL><LI>In the last step of sample preparation (after media swap to M9 and 1:20 dilution), grow the cells to OD<SUB>600</SUB> = 0.45.</LI><LI>Prepare 1% agarose gel (1g of agarose in 100mL of milliQ water) as platform for viewing cells on.</LI><LI>Pour agarose gel from previous step onto a glass slide between two square cover slips and push down from above using another square cover slip.</LI><LI>Once agarose gel has set, pipette 2µL of cell culture onto it.</LI><LI>Remove the two cover slips on the sides but keep the one on top of the gel.</LI><LI>Apply a drop of immersion oil (n = 1.5) onto the cover slip and mount the slide on the microscope, cover slip side down</LI><LI>View images on microscope and adjust focus accordingly.</LI><LI>Once satisfied with images, capture them using the ANDOR camera using GFP excitation at a wavelength of 476nm and an emission filter of 525nm.</LI><LI>Export images into MicrobeTracker.</LI><LI>Use MicrobeTracker to locate cells on the image and quantify their fluorescence in mean pixel intensity per cell.</LI></OL></DIV></DIV><DIV class="slim"><DIV class="section" id="8"><H2>8.0 Guide to Making Biobeads</H2><DIV id="8-1"><H3>8.1 Agaraose Bead Preparation</H3><P>
                            In a fume cupboard, break up a standard (polystyrene) petri dish into small pieces and dissolve in the minimum amount of ethyl acetate.
                        </P><P>
                            Prepare a required volume of 1.5 % Agarose solution in breaker with screw cap. Microwave on high for 2 minutes and then cool to 40 °C in water bath, when microwaving ensure the screw cap is placed on loosely.
                        </P><P>
                            Remove agarose solution from water bath and bring to laminar airflow cupboard. Pour the agarose into petri dish to a depth of 1cm. Allow to set, this should take roughly 15 minutes.
                        </P><P>
                            Still under laminar flow, use an autoclaved 1 cm diameter hole borer to core out the required number of identical agarose cylinders and place in a second petri dish and allow drying.
                        </P><P>
                            Bring uncoated ‘beads’ to fume cupboard. Using autoclaved needles, pick up individual beads and dip in ethyl acetate-polystyrene solution, stand each bead on needle upright in fume cupboard on blob of blu-tac to allow ethyl acetate evaporation and the coating to set.  When the coating is almost set the beads should be able to be handled through gloves without damaging the coat. Remove needle and mould the polymer coat over the needle hole, sealing them.
                        </P></DIV><DIV id="8-2"><H3>8.2 Sodium Alginate Bead Prepartation</H3><P>
                            Make up required volume of 1.2% alginate solution in dH<SUB>2</SUB>O. Slowly add Sodium Alginate to hot water and add a magnetic stirrer bar. Then leave to stir until all solid is dissolved.
                        </P><P>
                            Transfer this mixture in a 2mL syringe with a needle attached and drop the mixture into a solution of 0.1M calcium chloride. Vary the size of the syringe and height from which you drop and record the effect on the beads. Leave the beads in the CaCl<SUB>2</SUB> solution for 5 minutes and then filter the beads out.
                        </P><DIV id="8-21"><H4>8.21 Encapsulation of Bacteria</H4><P>
                                Take 1mL of cells in culture medium. Spin down in a centrifuge at maximum speed for 1 minute. Remove the culture medium. Re-suspend cells in 1mL of sodium alginate.
                            </P><P>
                                Make beads using the same method as above.
                            </P></DIV></DIV><DIV id="8-3"><H3>8.3 Diffusion Rates</H3><P>
                            First make up solutions of Sodium Fluorescein of the following concentrations (µM) by serial dilution of 0.02M Sodium Fluorescein:
                        </P><P>
                            1, 2, 3, 4, 5, 10, 20, 30, 40, 50.
                        </P><P>
                            Then make up the following concentrations (%) of Crystal Violet by serial dilution of 0.1% Crystal Violet solution:
                        </P><P>
                            0.05, 0.01, 0.005, 0.001, 0.0005, 0.0001, 0.00005, 0.00001, 0.000005, 0.000001.
                        </P><P>
                            Make up 100mL of 1.2% Sodium Alginate solution and leave to cool.
                        </P><P>
                            Using the UV-VIS spectrometer measure the absorption of 1mL of Sodium Fluorescein solution at each concentration. The wavelength of light should be set to 480nm, as this is roughly the wavelength at which the absorption maxima lies. Then measure the absorption of 1mL of Crystal Violet solution. The wavelength of light should be set to 590nm, as this is roughly the wavelength at which the absorption maxima lies. This data can then be used to plot a calibration curve.
                        </P><P>
                            To make the beads, mix 2mL of 50µM Sodium Fluorescein solution with 18mL of Sodium Alginate, or mix 2mL of 0.1% Crystal Violet with 18mL of Sodium Alginate. Filter the beads and wash with Milli-Q.
                        </P><P>                   Place the beads into 100mL (±5%) of Milli-Q.
                        </P><P>
                            Remove a 1mL portion of reaction solution out every 10 minutes and measure the absorption at 480nm for Sodium Fluorescein or 590nm for Crystal Violet.
                        </P></DIV></DIV></DIV><DIV class="slim"><DIV id="references"><OL class="references"><LI>http://www.kollewin.com/plus/list.php?tid=7</LI><LI>Merritt, J. H., Kadouri, D. E., and O’Toole, G. a. (2011). Growing and analyzing static biofilms. Current Protocols in Microbiology, (SUPPL. 22), 1–17. http://doi.org/10.1002/9780471729259.mc01b01s22</LI></OL></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#1">1.0 PCR (Polymerase Chain Reaction)</A></LI><LI><A href="#1-1">1.1 Gel Electrophoresis</A></LI><LI><A href="#1-1-1">1.11 Staining the Gel</A></LI><LI><A href="#1-1-2">1.12 Visualizing DNA using UV Transilluminator</A></LI><LI><A href="#1-1-3">1.13 Extraction of DNA (PCR product) from Gel</A></LI></UL><LI><A href="#1-2">1.2 Restriction Digest PCR or Plasmid DNA</A></LI><LI><A href="#1-2-1">1.21 DNA 'Clean Up' using EZNA Enzymatic Reaction Kit</A></LI><LI><A href="#1-2-2">1.22 Nanodrop</A></LI><LI><A href="#1-3">1.3 Ligation</A></LI><LI><A href="#1-4">1.4 Preparation of Competent E.coli Cells</A></LI><LI><A href="#1-5">1.5 Transformation</A></LI><LI><A href="#1-5-1">1.51 Preparing the Plates</A></LI><LI><A href="#1-5-2">1.52 Spreading Plates</A></LI><LI><A href="#1-6">1.6 Growth and Culture of Bacteria</A></LI><LI><A href="#1-7">1.7 Mini-prep</A></LI><LI><A href="#2">2.0 Toxicity Assay</A></LI><LI><A href="#3">3.0 TCA Protein Precipitation</A></LI><LI><A href="#3-1">3.1 SDS-PAGE</A></LI><LI><A href="#3-2">3.2 Western Blot</A></LI><LI><A href="#3-2-1">Day 1</A></LI><LI><A href="#3-2-2">Day 2</A></LI><LI><A href="#3-3">3.4 Ni<SUP>2+</SUP> Affinity Chromatography</A></LI><LI><A href="#3-3-buff">Buffers</A></LI><LI><A href="#3-3-1">Day 1</A></LI><LI><A href="#3-3-2">Day 2</A></LI><LI><A href="#4">4.0 Growing Biofilms</A></LI><LI><A href="#5">5.0 Preparing Supplemented M9 Media Fluorescence Assay / microscopy</A></LI><LI><A href="#5-1">5.1 Fluorescence Assay</A></LI><LI><A href="#6">6.0 Flow Cytometry</A></LI><LI><A href="#7">7.0 Microscopy (and subsequent analysis using MicrobeTracker)</A></LI><LI><A href="#8">8.0 Guide to Making Biobeads</A></LI><LI><A href="#8-1">8.1 Agaraose Bead Preparation</A></LI><LI><A href="#8-2">8.2 Sodium Alginate Bead Prepartation</A></LI><LI><A href="#8-21">8.21 Encapsulation of Bacteria</A></LI><LI><A href="#8-3">8.3 Diffusion Rates</A></LI><LI><A href="#references">References</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>