Team:Oxford/Test/Notebook
OXFORD
PROJECT
Description
Design
Safety
Parts
Collaborations
Achievements
PRACTICES
Human Practices
Questionnaire Results
Urinary Biome Study
Lab To Clinic
WET LAB
Overview & Results
Notebook
Protocols
AlgiBeads Notebook
Chemicals
Synbiota
Interlab Study
DRY LAB
Modelling
Tutorials
PEOPLE
Team
Attributions
Contact
BLOG
Notebook
Parts Guide
Parts Guide
In the cloning section of our notebook, our sequences are each assigned a letter code followed by a symbol. The letter codes denote the identity of sequences as follows:
A - pLacI-LasR; pLas-RBS-T4Holin
B - pLacI-LasR; pLas-RBS-sfGFP
C - pLsr-RBS-sfGFP ( BBa_K1659501 )
D - pLsr-RBS-T4Holin ( BBa_K1659601 )
E - DsbA-DNase ( BBa_K1659301 )
F - YebF-DspB
G - DspB ( BBa_K1659200 , BBa_K1659210 )
H - MccS ( BBa_K1659100 )
I - Fla-DspB
J - DsbA-DspB ( BBa_K1659201 , BBa_K1659211 )
K - DsbA-Art175 ( BBa_K1659002 )
L - YebF-Art175 ( BBa_K1659003 )
M - ArtE ( BBa_K1659088 )
N - Fla-Art175 ( BBa_K1659001 )
O - Art175 ( BBa_K1659000 )
P - DNase ( BBa_K1659300 )
The symbols following the letter codes denote the type of plasmid which the sequence was being cloned into:
* - pSB1C3
# - pBAD33 for sequences A, B, C, and D; pBADHisB for all other sequences
Cloning
Week 1
Day 1
Preparation of Stock Solutions
1. gBlocks
The gBlocks ordered from IDT arrived in the form of vials of 200µg solid DNA powder.
(refer to BioBricks page for information on DNA sequences)
The gBlocks were made into 10ng/µl stock solutions in Milli-Q water for storage:
mass/ng
conc/ngµl-1
final volumeµl
200
10
20
2. Primers
The forward and reverse primers ordered from IDT came in 32.4nmol and 34.3nmol of solid respectively.
Sequences
Forward - CTTTTTTGCCGGACTGC
Reverse - ATGATTTCTGGAATTCGC
The primers were made into 100µM stock solutions in Milli-Q water for storage:
amt/10-9mol
conc/10-6M
final volume/10-6L
32.4
100
324
34.3
100
343
Preparation of Reaction Solutions
1. gBlocks
2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 1ng/µl-1
2. Primers
2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 10µM reaction solutions. (The solutions are labelled as "Prefix primer" and "suffix primer" in eppendorf tubes in the fridge.)
Polymerase Chain Reaction
25µl reactions were run according to the PCR protocol here
* The final concentrations of the primers were noted as they are needed to determine the annealing temperatures for the primers, which can be done using NEB’s online tool here.
** Add components in order of decreasing volume for maximum ease-of-pipetting.
*** When reaction mixture is being made up, all components as well as the mixture itself are to be kept on ice, as the Master Mix is a high-fidelity polymerase that will recognize the primers as being incorrectly base-paired and be able to hydrolyse the primers if kept at room temperature.
**** Use Q5 enzyme in the cold room to avoid defrosting and freezing the original stock of Q5 enzyme. This could decrease the activity of Q5 enzyme. Bring ice bucket to the cold room to bring Q5 into the bench.
***** Make sure that the primer and small amounts of DNA and primer doesn’t stick onto the side of the tube or the tip.
The reaction mixture tubes were positioned in an Eppendorf Mastercycler nexus X2 and the PCR program was run.
* DNA denaturation can be performed at 98℃ because of the high thermal stability of the Q5 polymerase
** A PCR takes 20-30 seconds to extend a sequence by 1kb, and since our longest sequence is ~2kb the extension time was determined to be 60s per cycle.
Gel Electrophoresis of PCR-Amplified gBlocks
An agarose gel was prepared according to the agarose prepartion protocol.
DNA preparation:
The contents of the PCR tubes need to be stained with a loading dye to help visualize its migration.
To each 25µl of content in each PCR tube, 10µl of blue loading dye was added.
Day 2
Gel Electrophoresis of PCR-Amplified gBlocks (continued from 22/06)
gBlock sizes for reference
Lane 1: 10µl of 2-Log Ladder
Lanes 2-15: 20µl of DNA and loading dye mixture prepared on 22/06
A potential difference of 120V was applied across the electrodes (the higher the voltage, the faster the gel will run but the poor the separation will be; DNA separation is typically done in the 120-140V range) for 80 minutes. As long as DNA has passed ⅔ of the column or purple dye have passed purple area of the gel, the gel is ready to get into the EtBr.
The gel was then stained and the bands were visualized; protocol
Results
PCR Results
The 7 bands corresponding to expected DNA sizes were excised using a razor blade for cleaning and extraction. The other sequences, along with J which only showed a very weak band, will be re-PCRed under modified conditions at a later date.
The excised gel chunks were transferred to centrifugation tubes.
Extraction of DNA (PCR product) from Agarose Gel
The extraction was performed using the E.Z.N.A. Gel Extraction Kit made by Omega Biotek, according to the Spin Protocol.
The excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. The heaviest chunk of gel weighed 0.16g and as such 160µl of buffer was added to each tube.
* As the tubes were spun with their lids open, they were placed such that lids pointed away from the direction of spinning.
Restriction Digest of Extracted PCR product
Restriction digest was performed using EcoRI-HF (5’) and SpeI (3’) restriction enzymes.
NEB’s Double Digest Finder was invoked and it was determined that CutSmart Buffer would be used for the digest.
The buffer was completely defrosted and shaken before use.
There is a recommended digestion protocol on NEB Cloner. 50µL reaction mixtures were set up with component volumes as recommended, except for the DNA where all 30µL of the extraction mixture (in elution buffer) were used in the digest. There would be up to 50ng of DNA in each tube of extraction mixture (from the gel bands).
Incubation at 37℃ was also done for 2 hours (ThermoMixer program 3) instead of the recommended 5-15 minutes, with shaking at 300rpm.
Restriction Enzyme Digest of the iGEM HQ linearised pSB-1c3
125ng of pSB-1C3 was dissolved in 5mL Milli-Q water.
The digest was performed using the modified NEBCloner protocol.
* We did not add phosphatase because it is assumed that with different sticky ends the vector cannot religate
DNA ‘Cleanup’ using EZNA enzymatic reaction kit
Upon completion of restriction digest incubation, the gBlocks and the plasmid backbones were purified again using the E.Z.N.A. Gel Extraction Kit. PCR products and plasmid backbones need to be purified in order to remove remaining impurities including agarose gel and restriction enzymes. At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.
DNA Quantification using NanoDrop
1µL water and tissue were first used to clean the stage. A blank reading was made using 1µL of elution buffer, and 1µL of each sample was measured for concentration:
Sample
Concentration/ngµl
C
11.3
D
3.5
E
0.8
H
0.9
J
1.6
L
7.7
N
3.6
pSB-1C3
1.6
Ligation of gBlock Sequences with pSB-1C3 Backbone
Ligation was completed; protocol
Preparation of Competent E. coli Cells
A sample of E. coli DH5ɑ stored at -80℃ was defrosted and inoculated in 5mL of LB in a 125mL conical flask (volume of LB 10% of flask volume so as to achieve sufficient aeration). The culture was left to grow overnight at 37℃.
Re-PCR of DNA Sequences
Two sets of sequences A,B,F,G, I, J, K, and M were PCRed using the same protocol as 22/06, with one set using a 55℃ annealing temperature and another using 50℃ annealing temperature.
Day 3
Re-PCR of DNA Sequences
Refer to protocol from 1.0
PCR Results
The amplified sequences A, B, F, G, I, K, J, and M were loaded with blue dye and run on agarose gel as per standard protocol
Sequences G and J showed bands corresponding to the expected sizes at both 50℃ and 55℃ annealing temperatures.
The bands were excised and extracted according to standard protocol (E.Z.N.A. Gel Extraction).
Plasmid recovery
The concentration of pSB1C3 plasmid recovered on 23/06 according to the NanoDrop was low (1.6ng/µl). To obtain more plasmids, we restriction digested a construct made by last year’s, pSB1C3 + abijk, to recover the pSB-1C3 content from the construct.
Set up the following reaction mixture:
Component
Volume/µl
DNA (pSB1C3 + insert)
5
EcoR1-HF
1
SpeI
1
Cutsmart buffer
2
Water
11
During the incubation set up 1% agarose gel by putting 1g powder agarose in 100ml; 1% gel is more efficient at separating larger fragments
Results
Incubate the mix above for 2hrs at 37℃
Incubate for 30mins at 95℃ to inactivate restriction enzymes (Inactivation temperature of SpeI is 80℃, and EcoR1-HF is 65℃)
Cool and spin (there will be condensation at the top of the eppendorf)
Add 1µl CIP (phosphatase) and incubate for 30mins at 37℃
Add loading dye
Load each DNA on the gel
Stain in EtBr for 30 + 20 mins (staining not very clear after 30 mins)
Extraction of “sticky” pSB1C3 (recovered from 2014 pSB1C3+abijk) from Gel
E.Z.N.A. Gel Extraction
Gel weight:
0.53g ⇒ 0.53mL Binding Buffer
30µL of elution buffer used
Repeat PCR with higher DNA concentration (for the sequences that previously did not yield clear bands)
Sequences A, B, F, G, I, J, K and M have gone through a second round of PCR as they previously failed to yield clear enough bands, and ethidium bromide staining shows that G and J have been successfully isolated as a result.
A, B, F, I, K, L and M still have not had successful PCR runs, as such an alternative PCR protocol with higher DNA concentration was attempted on them instead:
Reagent
Volume/µl
gBlock Template (1ng/µl)
5
Forward Primer
1.25
Reverse Primer
1.25
* A total of 5ng gBlock as opposed to 1ng at 58℃ annealing temperature present in reaction mixture; rest of the protocol is same as prescribed
Results
Alternative protocol does not work.
Gel extraction of G and J
Standard E.Z.N.A. Gel Extraction protocol followed.
Gel weights:
J - 0.34g ⇒ 0.4mL Binding Buffer
G - 0.3g ⇒ 0.4mL Binding Buffer
30µL of elution buffer used.
Restriction Digest of G and J
A restriction digest was then performed on the extracted DNA.
NanoDrop Quantification
J: 2.7ng/µl
G: 1.7 ng/µl
pSB-1C3: 1.4ng/ul - there is 8-9ul of this left in the freezer in drawer 4,  labelled with Psb1c3 EcoRI, SpeI and 24/06/2015
Ligation
Ligation perform according to protocol.
Transforming E. coli cells with Plasmid DNA
Competent e.coli cells were first prepared, according to the protocol here.
Sample C,D,E,H,J,L and N to be transformed, according to the protocol here.
Day 4
Growth and Culture of Bacteria
For the protocol please click here.
Plates incubated overnight show that vectors corresponding to C,D,E,H,J,L and N were taken up [however, later we find that ligation had failed, and so gBlocks C,D,E,H,J,L and N were never inserted]. There are at least 5-7 colonies for each biobrick.
Choose three colonies from each plate. The colony should not be too small or too large and should be reasonably spaced from the others.
Separately incubate each colony in test tubes overnight.
This process would significantly increase the amount of plasmids containing biobricks that we want. Plasmids can be extracted later.
Transformation of J and G (24/06 PCR Product) into competent E. coli cells
Competent cells are already made in stocks and can be found in the -80℃ freezer: we don’t need to prepare them again. J and G comes from repeat PCR done in Day 3.
Primer Design
Since A, B, F, G, I, K, M have repeatedly failed to be PCR-amplified, longer primers were designed to replace the old primers for the PCRing of these gene sequences.
Primers that would give the sequences new restriction sites to allow their insertion into pBAD33 and pBAD/HisB expression vectors were also designed.
Day 5
Plasmid Extraction
For the plasmid extraction protocol see here.
pSB1C3 shipping vector containing gBlocks from Day 1, as well as blank pSB-1C3 shipping vector, pBAD/HisB expression plasmids, and pBAD33 expression plasmids were extracted from overnight cultures of E. coli DH5 using E.Z.N.A. Plasmid DNA Mini Kit I.
(refer to pages 10-12 for Mini Kit protocol. Some special notes:
All optional steps were carried out except equilibration step.
After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.
Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.
The precipitate formed in following step 7 does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.
The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.
Plasmid Quantification
1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
DNA
c/ngμl-1
DNA
c/ngμl-1
pBAD/HisB
36.4
mccS (H1)
61.5
pBAD33
34.4
mccS (H2)
129.6
pSB1C3
142.1
DspB+DsbA (J1)
38.8
lsr + GFP (C2)
122.9
DspB+DsbA (J2)
46.3
lsr + GFP (C3)
40.3
DspB+DsbA (J3)
41.9
lsr + Holin (D1)
40.2
Art175+YebF (L1)
64.6
lsr + Holin (D2)
Art175+YebF (L2)
lsr + Holin (D3)
44.1
Art175+YebF (L3)
43.2
DNase+DsbA (E1)
77.4
Art175 + Fla (N1)
103.5
DNase+DsbA (E2)
43.4
Art175 + Fla (N2)
48.8
DNase+DsbA (E3)
40.7
Art175 + Fla (N3)
134.7
Plasmid Digest
The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 90 minutes. For the protocol see here.
Gel layout
10µl of blue gel loading dye was added to each tube of digested plasmids. 20µl from the contents of each tube were then loaded into a 30-well agarose gel according to the following schematic:
(*the pSB-1C3 well has pSB-1C3 with abijk insert)
The ladder well has 10µl of 2-Log Ladder added into it.
A potential difference of 120V was applied across the gel for 40 minutes before it was stained with ethidium bromide for 30 minutes.
Analysis of Results
Analysis
The first three lanes produced expected results: pBAD33 and pBAD/HisB being “blank” plasmid backbones, the pSB-1C3-abijk lane giving two bands corresponding to similar sizes (being pSB-1C3 and abijk insert respectively).
All the other lanes seem to be showing only products in the 2kb range, which corresponds roughly to the size of the pSB-1C3 backbone, but the sizes are not uniform. Nonetheless, we know that the plasmid extraction step was carried out correctly as it did yield products approximating what we were looking for.
If it was a matter of the ligation step carried out on 23/06 failing entirely, we should be expecting a uniform row of backbone bands. Instead, there are minor but noticeable size variations between each band which cannot be successfully explained by failed digestion/ligation. It is thus speculated that the pSB-1C3 stock we received from iGEM HQ had suffered from varying extents of DNA degradation such that the restriction enzyme cut sites on their ends were no longer.
Week 2
Day 6
PCR of samples C,D,E,H,J,L and N
Samples C,D,E,H,J,L and N were PCRed according to the protocol here.
* Use the Q5 HF Master Mix that has been kept at 4℃ in the cold room and not the frozen sample because repeated freeze thaw cycles are not good for the Master Mix.
** Remade 50µl 10uM primer stock from the 100uM stock solution (5µl primer in 45µl Milli-Q water)
*** Use DNA 10ng/µl stock solutions made up on Day 1
Gel electrophoresis of  C,D,E,H,J,L and N
Results
Refer to gel electrophoresis protocol in section 1.1
Only J showed no clear band corresponding to the expected sequence size.
EZNA gel extraction of C,D,E,H,J,L and N
Refer to section 1.13 of the protocol guide.
Excise bands C,D,E,H,L, and N using razor blade, and the excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. For instance, the heaviest band was 0.27g, requiring 0.3ml Binding Buffer to each eppendorf tube.
Restriction Digest of Gel Extracted C,D,E,H,J,L and N
Refer to section 1.2 for the protocol.
Following the gel extraction spin protocol above, extracted PCR DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in 30/06 (Day 7).
Recovery of pSB-1C3 vector from 2014 pSB-1C3 + insert
Component
Volume (10µl)
pSB-1C3 + insert
10
SpeI
1
EcoR1 HF
1
Cutsmart
2
Milli-Q
6
Incubate for 2hrs 37℃
Heat and inactivate the restriction enzymes at 95℃ for 20mins
Cool down in the room temperature, spin (condensation on the side of the eppendorf from heating)
Add 1µl CIP and incubate at 37℃ for 30mins
Add 5µl loading dye
* Phosphatase is added to prevent the vector religating to insert. In the case of PCR amplification of (non-plasmid) gene sequences phosphatase does not need to be added to PCR product because those two ends are unlikely to ligate onto itself.
** Must start preparing for agarose gel to run 2014 pSB1c3 + Insert. The gel will distinguish pSB1c3 vector from insert, allowing us to extract the vector from the gel and ligate with PCR product. Ligation is to be done in 30/06 (Day 7).
Results
Gel electrophoresis of pSB1C3.
EZNA gel extraction protocol on recovered pSB1c3 vector
Refer to section 1.13 for the protocol.
Start with excising the band that corresponds to the base pair length of pSB1c3 vector.
Following the gel extraction spin protocol above, extracted Vector DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in Day7.
Growth and culture of E. coli transformed with 24/06 PCR Product (J and G)
Refer to section 1.6 for the protocol.
* LB has to be clear. The LB on the shelf was cloudy and therefore contaminated from last week so start with a new bottle
** Colonies were of variable size which could mean that some of the colonies are contaminated. Therefore, when picking the 6 colonies, pick 2 small, 2 medium and 2 large colonies
Day 7
From yesterday
E.Z.N.A enzymatic reaction cleanup protocol for Restriction Digest products of C, D, E, H, L, and N ( 1.21 )
At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.
E.Z.N.A gel extraction protocol for pSB-1c3 vector isolated from 2014 pSB1C3+insert ( 1.13 )
E.Z.N.A enzymatic reaction protocol for pSB-1C3 gel ( 1.21 )
Ligation of 29/06 PCR products to pSB-1C3 ( 1.3 )
Some notes:
Our component volumes were slightly different from that in day 2 due to the different amount of gBlock/vector we had. (We divided the amount of pSB-1C3 between our 6 samples)
Component
Volume/μl
Digested DNA (gBlock)
30
pSB-1C3
8
T4 DNA ligase buffer
5
T4 DNA ligase
1
MilliQ water
6
Mixtures were incubated on thermomixer at 16 ℃ for 16 hours until 8.45 a.m. on 1/7/15, taking care to vortex before placing on thermomixer.
Plasmid Extraction using miniprep kit ( 1.7 )
Sequences G [pSB-1C3] and J [pSB-1C3] were extracted from overnight cultures of E. coli DH5𝛼 using E.Z.N.A. Plasmid DNA Mini Kit I.
refer to pages 10-12 for Mini Kit protocol. Some special notes:
All optional steps were carried out except equilibration step.
After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.
After addition of solution I/RNAse, vortexing/vigorous shaking of the tubes should be avoided to prevent shearing of nucleus and undesirable accidental extraction of chromosomal DNA.
After addition of solution I/RNAse, resuspension of pellet can be done by dragging the tube along an eppendorf rack.
Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.
After addition of solution II, the waiting time before proceeding to the next step should not be more than 5 minutes.
The precipitate formed in following addition of solution III does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.
The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.
50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
Plasmid Quantification: Nanodrop ( 1.22 Nanodrops are usually done following restriction digest and transformation)
1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
DNA
Conc. /ngμl-1
DNA
Conc. /ngμl-1
J1
62.4
G1
67.4
J2
77.2
G2
81.5
J3
39.5
G3
80.2
J4
61.4
G4
88.0
J5
37.9
G5
26.3
J6
134.3
G6
95.7
Plasmid Restriction Digest for 24/06 PCR Product 1.2
* 0.5ul enzyme despite digesting a plasmid because test digest
**must refer to the master sheet each time
As many tubes were being handled, the required reagents were pre-mixed:
Reagent
Volume / μL
2.1 Buffer
50
Milli-Q
350
EcoRI-HF
12.5
PstI
12.5
3μL of each insert-containing plasmid was transferred into PCR tubes, and 17μL of the reagent mix was added to each PCR tube.
The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 120 minutes.
Gel Electrophoresis of Digested Plasmids from 24/06 PCR Product 1.1
Results
5µl of blue gel loading dye was added to each tube of digested plasmids.
20µl from the contents of each tube loaded. Gel run under 80V for 40 minutes:
G5, J3, and J5 were also the samples that showed irregularly low concentrations when analysed with the NanoDrop.
As such, G5, J3, and J5 were discarded while the other samples, being potentially-viable BioBricks, were freeze-stored for verification by gene sequencing at a future date.
Preparing new primer stock solutions
Primer
amt / 10-9 mol
conc / 10-6 M
Milli-Q vol / 10-6 L
Posy Suffix Holin
28.7
100
287
SMAP 29 Forward
31.2
100
312
DspB Forward
27.7
100
277
Art-E Prefix
35.4
100
354
DNase Forward
18.2
100
182
YebF Forward
29.1
100
291
Art-E Suffix
19.4
100
194
Art 175 Forward
27.5
100
275
Fla Forward
26.5
100
265
Pre-prefix Holin
22.9
100
229
MccS Forward
29.2
100
292
DsbA Forward
27.0
100
270
Protocol:
Pulse spin primers
Add fresh Milli-Q as above table
Leave for 30 mins at 37C in the shaking block (allowing DNA to resuspend)
Dilute to 10µM
PCR Amplification of gBlocks Using New Primers
***Refer to the Master Table for detailing gBlock-primer combinations for both preparation for the gene sequences’ eventual transformation into pSB1C3 shipping vector as well as pBAD33 or pBAD/HisB expression vector.
Primer naming and explantion can be found here.
With reference to the Master Table , standard PCR reaction mixtures (1uL 1ng/uL DNA template, 1.25uL 10uM forward primer, 1.25uL 10uM reverse primer, 9uL Milli-Q water, 12.5uL Q5 Master Mix) was set up for A*-F*, H*, I*, K*-N*, A#-E#, K#, N#, L#, H#, M#, O#, and P#.
G* and J* were not run because we already potentially have viable BioBricks for them from the plasmid extraction done earlier today (24/06 PCR batch).
DspB-containing setups J#, I#, F#, G# were not run because DspB has a BspHI restriction site in the middle of its sequence. The DspB-containing gBlocks will first be QuickChange-PCRed as an insert in the shipping vector to remove the restriction site by introducing a point mutation to codon-swap before being redigested out of the shipping vector and ligated into the expression vector.
Some compromises had to be made in terms of annealing temperature as there were only 2 PCR machines available and hence only 4 different programs could be run in parallel. The annealing temperatures were set up as below:
Annealing T / ℃
Label
72
A*-F*, H*, I*, K*-N*, A#-D#
67
E#, K#, N#, P#
61
L#, H#
70
M#, O#
Reaction times and other temperatures were set up according to standard PCR protocol 1.0
Day 8
Gel Electrophoresis of 30/06 PCR Products
Protocol 1.2
Results
5uL loading dye added per PCR tube. Gel run under 120V for 45 minutes:
Sizes of each fragments can be referred to the Master Table , and the following fragments that show clear bands have been excised and sent for sequencing.
Excised: C*, D*, H*, L*,M*, N*, C#, D#, E#, H#, K#, L#, M#, N#, P#
Restriction Digest of 30/06 PCR Products
Protocol 1.2
Grouping by required restriction enzymes -
EcoRI-HF + PstI-HF: C*, D*, H*, L*, M*, N*
Reagent mix (7-portion) first made up:
Component
Volume / µL
CutSmart Buffer
35
Milli-Q
98
EcoRI-HF
3.5
PstI-HF
3.5
20µL of reagent mix was added to each tube containing 30uL DNA products.
BspHI + PstI-HF: E#, K#, N#, L#, H#
Reagent mix (6-portion) first made up:
Component
Volume / µL
CutSmart Buffer
30
Milli-Q
84
EcoRI-HF
3
PstI-HF
3
20µL of reagent mix was added to each tube containing 30µL DNA products.
BamHI: C#, D#
Since there are only two tubes to be handles the reagents were directly added to each tube:
Component
Volume / µL
3.1 Buffer
5
Milli-Q
14
EcoRI-HF
0.5
PstI-HF
0.5
NcoI, PstI: M#, P#
Since there are only two tubes to be handles the reagents were directly added to each tube:
Component
Volume / µL
3.1 Buffer
5
Milli-Q
14
Ncol
0.5
PstI
0.5
Incubation at 37°C for 2 hours, with 300 rpm shaking. Upon completion, samples stored at -20°C.
Plasmid Design
Quikchange plasmids for DspB were designed and ordered.
VF2 and VR plasmids, used for sequencing pSB1C3-contained inserts were also ordered.
Transformation of 29/06 ligation products (containing C, D, E, H, L, N gBlocks)
Protocol 1.5
*only 9 tubes of competent DH5alpha E. coli left in the freezer after this round of transformation, as such more will need to be made by the end of the week
Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
Then add volume of E. coli according to protocol, using beads to spread across petri dish.
Day 9
Plasmid restriction digest - pSB1C3, pBAD33, and pBAD/HisB
Restriciton digest of SB1C3, pBAD33, and pBAD/HisB completely, using the 1.2 protocol and consulting the master table for the correct buffer and restiction enzymes.
Resultant Gel
100mL 1% agarose made up, small plate loaded and remainder left in bottle on shelf.
Total volume per tube is 35uL. Each tube’s contents split into two wells (17.5µL per well), and gel was run on 80V p.d. for 45 minutes.
* Gel can help tell if any digestion occurred at all - if no digestion occurred plasmid would still remain circular, and circular plasmid would encounter more resistance migrating through the gel matrix than linear plasmid of the same size and hence appear to be bigger than expected when compared against the ladder.
Bands were excised and the heaviest chunk was 0.66g. 670uL of XP2
Binding Buffer was added to each excised chunk to initiate E.Z.N.A. Gel Extraction protocol.
Enzymatic Reaction Clean-up - 30/06 PCR Products
Protocol is 1.13
Retrieve products (on labelled yellow rack) from -20°C and perform clean-up.
The volume of the restriction digest reaction done on 01/07 was 50µL per tube hence according to protocol 50µL of XP2 Binding Buffer added to each tube.
Upon completion of protocol, tubes placed back into -20°C until plasmids ready for ligation.
Growth and Culture of Bacteria - 29/06 PCR Products
Refer to section 1.6 of the protocol guide.
*note: new bags of inoculation loops are placed next to the 37°C incubators for agar plates
LB agar plates of C, D, E, H, L, N collected.
Three colonies selected from each plate to set up three separate overnight cultures each.
Ligation of 30/06 PCR Products(
Refer to section 1.3 of the protocol guide.
Component
Volume/µ
Digested and cleaned PCR products
30
Digested and cleaned plasmids (pSB1C3: C*, D*, H*, L*, M*, N*; pBAD33: C#, D#;pBAD/HisB: E#, K#, N#, L#, H#, M#, P#)
8 for pSB1C3 10 for pBAD337 for pBAD/HisB
T4 DNA Ligase
1
T4 Buffer
5
Milli-Q
6 for pSB1C3 4 for pBAD33 7 for pBAD/HisB
Incubate reaction mixture at 16°C overnight.
Preparation of Competent E. coli DH5alpha
Refer to section 1.4 of the protocol guide.
*Sterile technique used
Test tube filled with 5mL LB broth.
GW opened new bag of inoculation loops to steal a colony off one of Elaine’s streaked plates (marked 29/06) and inoculated it in the filled test tube.
Test tube left to incubate at 37°C overnight.
Day 10
Plasmid for 29/06 PCR Products
Refer to section 1.7 of the protocol guide.
Yesterday, collected LB agar plates of C, D, E, H, L, N and Interlab are incubated overnight. Selected three colonies from each of C, D, E, H, L and N, and only one for each of the Interlab colonies (we assume that the plasmids provided by iGEM HQ for Interlab were all the same).
Therefore we shall be performing EZNA plasmid extraction on 22 samples. Follow EZNA Mini Kit I Spin Protocol (pg10-12).
50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
Also, aspirate = pipetting!
Measured the concentration of the different samples using nanodrop
Sample
Concentration (ng/μl)
C1
46.7
C2
35.6
C3
52.7
D1
43.2
D2
95.8
D3
30.4
E1
45.1
E2
112.4
E3
46.7
H1
71.6
H2
48.2
H3
48.5
L1
48.7
L2
56.2
L3
69.2
N1
57.0
N2
47.0
N3
68.3
Digest aliquots of each of the 22 samples, and run on gel. Refer to protocol 1.2
From gel, determine the degree of success of these samples.
Preparation of Competent E. coli DH5alpha
Refer to protocol 1.4
Transformation of 30/06 PCR Products (Ligated on 02/07) (Protocol in section 1.5)
Refer to protocol in section 1.5
Antibiotics:
pSB1C3, pBAD33 - Chl
for C*, D*, H*, L*, M*, N*, C#, D#
pBAD/HisB - Amp
for E#, K#, N#, L#, H#, M#, P#
Raffy, Leon: Moving of cells plated on 3/7/2015 from the incubator to the cold room. All pSB1C3 plates had reasonable colony density. A significant proportion of pBAD/HisB plated had no colonies. Plates with no colonies will be reincubated for half a day on 6/7/2015.
Week 3
Day 11
Preparation of Overnight Cultures for 30/06 PCR Products
Plate
Colony
Plate
Colony
C#
1
C#e
1
D#
0
D#e
0
E#
2
E#e
2
H#
0
H#e
3
K#
0
K#e
0
L#
0
L#e
0
M#
0
M#e
0
N#
0
N#e
0
P#
1
P#e
0
Plate
Colony
Plate
Colony
C*
Yes
C*e
Yes
D*
3
D*e
0
H*
Yes
H*e
Yes
L*
Yes
L*e
Yes
M*
0
M*e
3
N*
Yes
N*e
Yes
Colonies grew for L* and N*, but because the MiniPrep run on 03/07 already showed the correct bands for them, their colonies were not cultured overnight.
Sequence
Colonies picked
C*
3
D*
3
H*
3
M*
3
C#
2
E#
4
H#
3
P#
1
See protocol guide to find out how to make overnight cultures.
PCR amplification of A,B,D,E,F,I,K,M for pSB1C3
1. Primer used:
* primer pair (pre prefix holin and post suffix holin)
Comments:
Attempting A, B, F and I again as they have never been successfully amplified before.Attempting A, B, F and I again as they have never been successfully amplified before.
NB: F and I are DspB-containing gBlock which we eventually hope to QuikChange to get rid of the BspHI restriction site in them and put them into pBAD/HisB expression vector
2. PCR all the # sequences with the appropriate primers
Follow 1.0 PCR protocol, and primer list is in the Master Table .
Restriction digest for pSB1C3, pBAD33 and pBAD/HisB
Gel photo
Follow restriction digest protocol.
pSB1C3 and pBAD/HisB were successfully digested and eluted in 1% gel. Band for pBAD33 could not be found.
Excised bands were stored in -20℃ for cleanup tomorrow.
Primer Preparation
Preparing new primers (solution and dilution) - VF2, VR, QuikChange forward, QuikChange reverse.
First make 100uM out of the freeze dry solid:
Spin down solid
The amount of primer is given in y nmol for each tube. Add 10y µL of water to each tube to make 100µM.
Shake/vortex and mix evenly.
For VF2, VR - take 3.2ul out of the 100µM stock and dilute with 96.8µL Milli-Q to make 3.2pmol/uL sequencing primer solution.
For the QuikChange primers, dilute to 10uM as per normal.
Sequencing of BioBricks
5µl of each plasmid along with 100uL of 3.2pmol/uL sequencing primer sent to SourceBioScience for sequencing:
Sequence
Label Assigned
C 3
pSBLsrGFP3
D 2
pSBLsrHolin2
E 2
SBDNaseDsbA2
G 6
pSBDspB6
H 1
pSBMccS1
J 2
pSBDspBDsbA2
L 3
pSBArt175YebF3
N 3
SBArt175Fla3
Gel Electrophoresis of PCR Products
Correct bands were obtained for D*, C#, G#, H#, J#, K#, L#, N#. Still awaiting reply on insert length for O# and P# to determine whether the bands are correct.
Day 12
NanoDrop Analysis of Plasmids Digested on 06/07 (carried out after gel extraction)
Plasmid
Conc / nguL-1
pSB1C3
5, 9.5
pBAD/HisB
1.8, 1.6
Since pBAD/HisB is very low in concentration, more of it will be digested.
Restriction Digest of pBAD33
Component
Volume/µL
Plasmids
10
BamHI
1
Milli-Q Water
7
3.1 Buffer
2
37℃, 2 hours (heat up another block to 95℃)
Heat inactivation, 95℃, 20 minutes
Melt, cool, and pour 1% agarose
Pulse spin
CIP dephosphorylation (1 µL CIP), 37℃, 30 minutes
Load dye and run gel
pBAD/HisB digest - 6uL water, 1µL NcoI, 1µL PstI, 2µL 3.1, 10µL plasmid
Gel extraction of digested plasmids
Plasmids digested on 6/7:
400µL of Binding Buffer added to each tube containing excised band.
After first elution, concentrations turned out to be low (see NanoDrop table at the top of this document), hence a second elution was done, again using 50µL of elution buffer.
Plasmids digested on 7/7:
340uL of Binding Buffer; one elution with 50µL.
Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)
Follow EZNA DNA Mini Kit I Spin Protocol - eluted 50μl
freezer: expression vector ones (the ones with #) on orange rack in bottom drawer (will be dealt with tomorrow), psb vector ones (the ones with *) in white box with other psb constructs in top drawer
NanoDrop analysis:
Sequence
Conc / ngµL-1
Sequence
Conc / ngµL-1
C* 1
58.3
M* 3
49.2
C* 2
67.5
C# 1
64.5
C* 3
66.0
C# 2
25.1
D* 1
6.5
E# 1
37.8
D* 2
43.3
E# 2
43.1
D* 3
56.7
E# 3
30.5
H* 1
30.0
E# 4
67.3
H* 2
45.9
H# 1
36.5
H* 3
44.5
H# 2
28.8
M* 1
51.1
H# 3
37.3
M* 2
53.1
P# 1
179.2
Gel electrophoresis of Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)
Gel Photo
Electrophoresis of 10uL plasmid with 5uL loading dye
N.B. forgot to digest - will do this on 08/07/15 (tomorrow)
Analysis of sequencing data from 06/07/15
Of the stuff sent for sequencing (03/07 miniprep):
Sample
Results
Action Point
C*3
corresponds to pSB1C3 Lsr GFP
-
D*2
Corresponds to pSB1C3 Lsr GFP (does not correspond to pSB1C3 Lsr Holin as expected - contamination in all three D* wells were already apparent in gel (see 06/07))
Wrong insert, need to redo from scratch (start from PCR again)
E*3
DsbA DNAse missing, sequencing only gives blank pSB1C3 vector
Send another miniprep product in the triplicate (E*1 or E*2) for sequencing.
G*6
Forward sequence good enough to confirm (~70% length of DspB sequence) that it’s pSB1C3 DspB, but reverse sequence dropped off too early for double confirmation
Ask Source Bioscience to redo VR (reverse) sequence
H*1
Corresponds to pSB1C3 MccS
-
J*2
Corresponds to DsbA DspB, but base 2964 had a C->A point mutation
Send another miniprep product in the triplicate (J*1 or J*3) for sequencing
L*3
Corresponds to pSB1C3 Art-175 YebF
-
N*3
Corresponds to pSB1C3 Art-175 Fla
-
Results: 4, potentially 5 BioBricks successfully made in [pSB1C3] format. Successful BioBricks to be stored in separate box in preparation for sending to Registry, creating Database page etc.
Ligation of 06/07 PCR to Appropriate Digested Plasmids
pBAD33: C#1, C#2 (one of these were wrongly digested using CutSmart instead of 3.1), D# (previously digested using wrong buffer), D#3.1
pSB1C3: D*
pBAD/HisB: E#, G#, H#, J#, K#, L#, N#, O# (previously digested using wrong buffer), O#3.1, P# (previously digested using wrong buffer), P#3.1
Refer to section of the protocol guide.
Left overnight at 16C.
Day 13
To Do
Prepare plates for transformation
Transformation of ligated plasmids
Gradient PCR:
Group A: A*, B*, F*, I*, K*
Group B: A#, B#
Group C: I#, F#
To Note
Re-sequenced VR read and DspB confirmed as correct = another biobrick.
QuikChange PCR on DspB
Standard PCR protocol 1.0
72C annealing temp
2 min extension time
Add 1ul DpnI and incubate at 37C for two hours
Transformation
Add 1ul PCR product to 100ul competent cells. Add the remaining PCR product (24ul) to another eppendorf of 100ul competent cells
Follow standard transformation protocol
Restriction Digest
Incubate at 37oC for 60 mins - BamHI has star activity so it cannot be left for long
Component
Volume/µL
Component
Volume/µL
Component
Volume/µL
C#
5
C*/D*/H*/M*
5
E#/H#/P#
5
3.1 NEB 10x buffer
2
2.1 NEB 10x buffer
2
3.1 NEB 10x buffer
2
Bam HI
0.5
PstI
0.5
Bam HI
0.5
(no 2nd RE)
-
EcoRI-HF
0.5
EcoRI-HF
0.5
MilliQ
12.5
MilliQ
12
MilliQ
12
Gel electrophoresis of restriction digest
Gel Photo
Sent for sequencing:
Sequence
Concentration
Label Assigned
E* 1
45.1
SBDNaseDsbA1
J* 1
62.4
SBDspBDsbA1
C# 1
64.5
33LsrGFP1
H# 3
37.3
BMccS3
E# 4
67.3
BDNaseDspA4
P# 1
179.2
BDNase1
D* 3
56.7
SBLsrHolin3
H* 2
45.9
SBMccS2
M* 2
53.1
SBArtE2
Transformation of E. coli with ligation products from day12 (06/07 PCR and interlab sequences) using the standard protocol described by diagrams for day 3. Transformed E. coli then plated and incubated overnight
Day 14
Group A
C*, G*, H*, L*, N*, M* are biobricks
G*, J*, D* Sent for resequencing
A*, B*, F* I* K* - gradient PCR - currently in the freezer, run gel electrophoresis in afternoon, doing staining tomorrow
E*, J*, D* - redid PCR trying Q5 mastermix from 2014 and 2015 (gel poured for electrophoresis tomorrow)
Group B
C# - redo PCR (sequencing didn't not give expected resuts)
A#, B# - gradient (already currently in freezer - need to put onto gel)
C#,D# - colonies
Group C
E#, H#, P# - successful sequencing (with exception of point mutation in plasmid)
I# and F# - gradient PCR
M#, O# - redo from PCR
J#, K#, L#, N#, G#, P#, O# - colonies
G# - wait for G* to come back and QC if successful
(if QC doesn’t work again, treated with dpn1, gel, excise, clean and add buffer and ligase, transformation)
Is DH5alpha suitable for QC?
week, grow up competent cells for 2 E coli strains:
FliC deletion strain - FliC exported through Fla export system
MG1655 - close to WT E. coli - more tolerant (characterise in MG1655)
* primers are the templates for J#, I#, F#, G#
Results of sequencing:
Sequence
Result
E* 1
Forward read aligns, ask to repeat VR. Aligns to MccS (which is actually H*). Check the size of the PCR fragment. Redo PCR
J* 1
Forward and reverse reads align but highlight a point mutation at 847 but looking at the chromatogram there is also a peak for G.Send another, preempt another won’t work so start PCR
C# 1
Both reads are of insufficient quality. Look at the gel (sizes are off - two bands entirely diff size but both >2k - wrong)
H# 3
Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine. Point mutation in plasmid stock?
E# 4
Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine.
P# 1
Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine.
D* 3
Both reads align but there is a deletion at 2205. Send another.
H* 2
Both reads align.
M* 2
Both reads align.
G* has a 70bp insert at the start of the coding region (even though it’s been quikchanged). Send another.
PCR To Redo
E*, J*, D*
To Sequence
J* (not J1, J6 or J2) - J4
D* (not D3, D1 too low) - D2
G* (G4, G3, G2) - G4
To Overnight
The following tubes were put in incubator with selected colonies for over night culture (1x unless specified otherwise):
3x 3.1 C#
G#
3x P#
3.1 O#
3x pBAD33
C#
3x 3.1 D#
22A
pSB1C3
D#
3x 22K
20K
3x K#
L#
3x N#
3x J#
Day 15
Group A
Check sequencing data for G*, J*, D* :
G* = large anomalous insert at start of coding sequence → send another (G3) but also redo PCR verdict: G*3 sequencing data correct - no need further PCR
J*4 = biobrick
D* = large deletion in sequencing data → new batch currently being PCRed
(A,B,F,I,K)* - stain gel → visualize gel → 09/07 gradient PCR turned out to have no amplified products at all
(E,J,D)* -  load dye → run gel → stain gel → visualize gel → only D* yielded bands → excise bands for (10/07) PCR products → cleanup → digest
Quikchange J*4 under gentle conditions to remove BspHI restriction site within its DspB sequence → wrong procedure was carried out (forgot to do DnpI enzyme incubation, and was PCRed despite quikchange products being meant to be directly transformed) → need to redo
pSB1C3 plasmids made from (09/07) overnight cultures → miniprep → digest with EcoRI, SpeI → gel poured and awaiting electrophoresis on Monday
E* was PCRed again using both standard and DMSO PCR protocol → but gel was frozen before excision and hence messed up → need to redo
J*4 QuikChange
Standard PCR protocol using QuikChange primers and following conditions:
Stage
Temperature
Time
Initial denaturation
98
30sec
Denaturnation
98
10sec
Annealing
65
30sec
Extension
72
3min
Final extension
72
2min
Group B
(A,B)# - stain gel → visualize gel → no bands for (09/07) gradient PCR products
D# - load dye → run gel → stain gel → visualize gel → excise bands for (10/07) PCR products → extract → digest → not cleaned up yet
(C,D)# - miniprep from (09/07) overnight culture → 10uL digested but not gelled yet
pBAD33 in (09/07) overnight culture - miniprep → digest → visualize → excise → fresh pBAD for ligation
Group C
(I, F)# - stain gel → visualize gel → no bands for (09/07) gradient PCR products
(M,O)# - load dye → run gel → stain gel → visualize gel → excise bands for (10/07) PCR products → extract → digest → not cleaned up yet
(J,K,L,N,G,P,O)# - miniprep from (09/07) overnight culture → 10uL digested but not gelled yet
More pBAD/HisB needs to be made → heat shock → plate colonies
Over the Weekend
Overnight cultures for plasmids setup
Quikchange for J*4 - parameter testing (previously we used the full 61.4ng/uL for quikchange - may be the reason why it never worked (for Q5MM’s efficacy, NEB recommends a total of 1-25ng of plasmid DNA in a 25µL reaction mixture)
1µL of 61.4ng/µL J*4 diluted into 60µL total volume to make 1.02ng/µL J*4
3 setups were made - 1, 5, 10µL of 1.02ng/µL J*4 at two annealing temperatures (65 and 70 degC) - making 6 tubes 3 minute extension time
Also E* and E*DMSO PCR
Week 4
Day 16
Leon
Minipreps for plasmids → concentrations noted on each tube
10µL of pSB1C3 digested
15.1µL of pBAD33
30µL of pBAD/HisB
Run E*, E*DMSO on the gel → bands very faint → both bands combined into one tube for gel extraction
Split into two tubes at the Binding Buffer step → at the end, one tube had 7ng/µL and the other had 9ng/uL of DNA
React J*4 QC with DpnI: all 24µL of all 6 tubes of J*4 from the QC PCR reaction placed into digest mixture (made up according to DpnI search using NEBCloner protocol desginer). Buffer = CutSmart; 2 hours incubation
Clean up digests done on Friday → cleaned up and ready for ligation
Digest plasmids and both tubes of E*
Day 1 (overnight) procedure carried out for the preparation of competent DH5alpha, MG1655, deltaFliC E. coli
Group A
Failed Gel
Helen and Drizzy: gradient PCR of A#, B#, F#, I#
A# used up the last of the 2014 Q5 MasterMix
Failed Gel
Mabel and Ria: normal PCR of A*, B*, F*, I*, K*
Mabel: prefix + post suffix holin primers, annealing temp = 58℃ (so set PCR to 56℃)
Ria: pre suffix holin + suffix primers, annealing temp = 63oC (set PCR to 61oC)
All failed - no DNA bands
Ria
Gel Photo
First gel of digests from Day 15 overran, re-do digest and gel electrophoresis etc.
Day 17
Plasmid Backbones
Dephosphorylation
1µL CIP added to heat inactivated pSB1C3, pBAD33, pBAD/HisB; incubation for 1 hour.
Enzymatic Reaction Cleanup
After cleanup, NanoDrop shows conc all are about 30ng/µL.
Ligation
All 29µL of each insert used, with 7µL of plasmid.
June
10/07 PCR Batch (D*, D#, M#, O#)
Ligation set up and left overnight.
13/07 PCR Batch (E*, E* DMSO)
Enzymatic reaction cleanup completed. Ligation set up and left overnight.
Preparation of Competent Cells
DH5alpha, MG1655, deltaFliC
Protocol
OD when incubation stopped:
DH5alpha - 0.302
MG1655 - 0.345 (achieved in one hour); 0.902 (1.5 hours - overshot because used more than 1mL, also wildtypes seem to grow faster than DH5alpha)
deltaFliC - 0.350; 0.805
The 0.902 MG1655 are labelled MG; the 0.345 MG1655 are labelled MGA and MGB.
The 0.805 deltaFliC are labelled with an extra dot • on the tubes.
Sequencing
Construct
Successful
Label assigned (1st sequence)
pBADArt175DsbA
K2
Art175DsbA2
pBADArt175Fla
N3
Art175Fla3
pBADArt175
O1
Art1751
pBADDNase
P2
DNase2
PCR of C#, D#, L#
Worked beautifully.
PCR of A*, B*, I*, F*, K*, A#, B#, F#, I#
K*: given to Chris for Phusion polymerase, will get results by the end of the week
Run gel of gBlock stock on 1% agarose with SYBR safe DNA Gel Stain in LAB buffer - no bands i.e. the working stock possibly has no DNA!
Nanodrop of gBlock stock: A = 7.9 ng/μl, C = 7.7 ng/μl (i.e. there is DNA)
N.B. A, B, I and F are our 4 largest gBlocks which we cannot amplify.
Driscoll's work: A, B, I and F attempted on a MgCl2 gradient of 0.5mM, 1mM, 2.5mM and 5mM. Ran PCR but was told that there was no DNA in the solutions so didn’t run gel, the PCR products currently sit in freezer but don’t think it’s worth it to run a gel as there is clearly nothing or not enough in the G-block stocks. The plan was to also to run a single PCR with A,B, F and I with 1% Triton X 100 and with double primer concentration but didn’t get around to doing it after hearing the results of the straight G block gel.
Helen: A# attempted with 10% DMSO. B# attempted with 10% glycerol. F# attempted with 5% formamide. I# attempted with 10% formamide. Ran a gel of these PCRs, and none showed any bands.
Day 18
Check Plates
As expected, MGA and MGB are more competent (grew more extensively on antibiotic plate) than MG(OD=0.9 when taken out of flask)
FliC0.35 also more competent than FliC0.8
Transform Ligated Products into DH5alpha
E*, E*DMSO, D*, D#, M#, O#
Ligation
Set up ligations for C#, D#, L# (PCR batch 14/07)
Mixed up the samples so will be ligating tomorrow.
Overnight Culturing
QC J*4 (3) grew colonies.
Conditions used for its Quikchange PCR was: 10.2ng plasmid DNA in the 25µL PCR reaction.
By the end of the day, a few more of the J*4s also had colonies
J*4 (3) and J*4 (1), which seemed to had the densest colony growth, had overnight cultures set up (3x each) in Chl LB broth.
PCRs
Same as previous but re-diluted primer stock and used 1µl of IDT gBlock (i.e. 10ng of DNA) for A*
Run at 2 different cycles: either 98 and 72degC for 2 mins or 98, 72 annealing, 65 for 1 min and 72 for 1 min for extension.
Gel: SYBR safe and agarose in LAB - see 14/07 .
NO BANDS, but the primer dimer band for the 65 degC extension was brighter when visualised with blue light.
Leon to email IDT about further recommendations.
Transform completed plasmids into FliC and MG1655 Competent Cells
N#, P#, H# and E#
Making up M9 modified media
We made it up using the recipe from this website for 500ml. We filter sterilised all components except milliQ water and stored it on the bench with LB.
Day 19
Mini-prep overnighted J*
6 colonies grown overnight
Alterations to protocol:
Spin down entire volume of LB broth to ensure all E. coli are used in the extraction.
Pulse after spinning down all the E. coli to remove excess LB
Heat elution buffer to 55 degC before eluting 50ul, spin down, re-pipette filtrate into HiBind column and spin down again
Nanodrop values (values are quite consistent - good idea to apply all modifications for any mini-preps in the future)
Eppendorfs in freezer drawer awaiting potential sequencing
note: (1), (2), (3) are from plate 1 (corresponding to QC PCR at 70degC annealing temp, 1ng plasmids); (4), (5), (6) are from plate 3 (corresponding to QC PCR at 70degC annealing temp, 10ng plasmids)
[DNA] (ng/µl)
J*4 (1)
175.2
J*4 (2)
172.2
J*4 (3)
153.3
J*4 (4)
194.5
J*4 (5)
181.0
J*4 (6)
173.9
Diagnostic restriction digest of J* + gel electrophoresis
Gel Photo
20µl reaction for 90 minutes in PCR tubes
See protocol
Gel: 1% agarose, 120V
Ladder
J*1
J*2
J*3
J*4
J*5
J*6
Lack of ladder but all consistent - send for sequencing tomorrow
Pick Colonies
Pick colonies of N#, P#, H# and E# (delta FliC and MG1655) and culture overnight
Pick colonies for E*, E*DMSO, D*, D#, M#, O# and culture overnight
PCR
O# and QC G*
QC PCR protocol
Ran PCR on 1ng, 5ng, and 10ng of template DNA with 3-minute extension time.
Digest PCR products in DpnI, according to the protocol here
^has been done. Gel currently running.
Transform all the samples into separate cultures of competent DH5alpha
^has not been done
Re-do C#, D#, L# PCR
Restriction Digestion and Ligation
Restriction digest done on R and L.
Set up ligations for C#, D#, L# (PCR batch 14/07), running over night. After the gel only two managed to be extracted, one of C# or D# (R#) and L#.
Day 20
Leon
3 frozen samples each were stored in the -80 degC freezer for E# (FliC), E# (MG1655), H# (FliC), H# (MG1655), N# (FliC), N# (MG1655), P# (FliC), P# (MG1655) following the overnight cultures that were set up on 16/07
They are in our usual freezer drawer, and within it the bottom most slot of the third column from the front, in a box labelled “iGEM transformed cells, MG1655 and FliC”
Raffy
Sent J*4 (4) for sequencing
Mabel and Duke
Miniprep - E*, E*DMSO, D*, D#, M#, O#
Miniprepped overnight cultures from June and Raffy above. Plasmids were eluted with 50ul of elution buffer.
Nanodrop:
Plasmid type
ng/µl
D#
72.2
D# 2015 e1
160.6
D# 2015 e2
42.0
D# 2015
65.6
D* 2015
238.4
D* 2015 e
40.4
D* 2
288.9
D# e
31.2
D* 2e
378.7
E*1
333.9
E*1 e
287.5
E*2
370.6
E*2 e
367.1
M#e
50.5
M# 2015
50.8
M#e 2015
62.0
O#
49.9
O# 2015
82.7
O# e
44.1
Restriction digest of minipreps completed - find the protocol here 20µl reactions.
Ria and Lychee
Transform R#, L# into DH5alpha. R# is Chl and L# is Amp antibiotic
Helen and Kyle
Digested plasmids for successful PCRs, plasmids not yet cleaned up.
Over the Weekend
Agar plates from 17/07 transferred to cold room
Sequencing data for J*4(4) received (order no: 4254194) - QC successfully corrected desired base, but also overzealously introduced two repeated sequences corresponding to the primer sequence 1. after the QC area of interest and 2. at the start of the DspB sequence (see SnapGene file: QC J*4 wrong). Colonies on plate 3 should all have wrongly QC-ed plasmids as a result.
Will send either samples (1), (2), or (3) for sequencing (as they are from plate 1 - PCRed under different conditions), and also overnight culture some colonies from plates 4, 5, and 6 as they were QC PCRed with a lower annealing temperature (lower possibility of wrong extension?)
Overnight cultures set up:
Number of Tubes
Plasmid of Interest
3
L#“
3
“R#”
3
F*
3
I*
4
QC J*4: (A), (B) are from plate 5e (PCRed at 5ng plasmid, 65degC annealing) (C), (D) are from plate 6e (PCRed at 10ng plasmid, 65degC annealing)
1
Interlab 22A
1
Interlab 22K
1
Interlab 20K
On 20/07 Day 21
Interlab cultures - transfer 700uL of the respective cultures into appropriately-labelled microcentrifuge tubes and add 300uL of 60% glycerol to each of them. Vortex for even mixing and flash freeze at -80degC. The rest of the cultures can be miniprepped to replenish plasmid supplies for further transformation into MG1655 and deltaFliC.
All other cultures.
Week 5
Day 21
Post-digest cleanup → ligation
For C#, D#, L# from the 16/07 PCR batch
Sequencing of QC J*4
J*4 (4) sequencing data shown to have many extra inserts, despite the desired mutagenetic site actually being correctly replaced. Two paths will be pursued:
J*4(1) (which is from plate 1 - another PCR condition (1ng plasmid)) to be sent for sequencing - obtain 2.5µL plasmids and dilute 1:1 in 2.5µL of Milli-Q. Sent for sequencing using VF2 and VR primers.
J*4(1) (which is from plate 1 - another PCR condition (1ng plasmid)) to be sent for sequencing - obtain 2.5uL plasmids and dilute 1:1 in 2.5uL of Milli-Q. Sent for sequencing using VF2 and VR primers.
If and when the sequences are correct, we will save the corresponding tube as a complete BioBrick (annotating that it has been QuikChanged), digest out the insert, cleanup, PCR the NcoI restriction site in, digest, and finally ligate into pBAD/HisB to make a functional plasmid.
Sequencing Data Received
K#1 confirmed - refer order number 4253587
Gel electrophoresis confirmation of 17/07 digested miniprep products
Plasmids -
pSB1C3: D*, E*
pBAD33: D#
pBAD/HisB: M#, O#
After which, send for sequencing depending on degree of success. Unfortunately the gel ran too far and ‘ladder’ didn’t show up, which suggests that it is actually a mislabeled tube of loading dye. Re-made digests as only 3 ul of original is used. Will run gel again tomorrow with fresh ladder.
Miniprepping of 19/07 overnight cultures
Miniprep L#, “R#”, F*, I*, QC J*4
For the rest - miniprep the full volume (see “upgraded” protocol: spin down cells → decant → resuspend in smaller volume before beginning the miniprep; pre-warm elution buffer to 55degC; run the same elution mixture through the column twice for maximum extraction)
Digest 10µL → gel run (gel is now in cold room, awaiting staining and visualization tomorrow)
Transformations
K#1 → MG1655, deltaFliC
Plate Streaking
Frozen stocks of MG1655, deltaFliC N#, P#, H#, E# streaked on agar plates.
Day 22
Sequencing Data Received
QuikChange J*4 (1) and ( C ) returned correct sequences (refer to order nos 4254450 and 4254692 respectively). Both are now stored in “Complete BioBricks” box.
PCR Amplification to make J# from QuikChanged J*4[pSB1C3]
J*4(1) was used; 3 min extension time (lol too long)
Transformations
Transform C#, D#, L#, O# into DH5alpha
Transform pBAD33, pBAD/HisB into MG1655, deltaFliC for positive control.
Overnight Cultures
From newly transformed plated cultures:
MG and FliC - InterLab, K#
DH5alpha - (+) and (-) controls for InterLab
From streaked cultures:
MG and FliC - E#, H#, N#, P#
Run Gel for miniprepped J*QC, F*, I*, R#, L#
Gel Photo
F*, I* insert band very very faint - most probably because vector:insert ratio is about hundredfold
→ redo direct digest using 2µL (20ng) stock gBlock, ligation with less plasmid (~10ng)
Kyle already doing C#, D#, L#
Top: J*4A, J*4B, J*4C, J*4D, F*1, F*2, F*3, I*1, I*2, I*3, L#1, L#2, L#3
Bottom: R#2, R#3
Sent F*1, I*3 for sequencing to find out what is going on with confusing 3kb/2kb double band. Results (order no: 4254965) shows both samples containing only blank pSB1C3.
Restriction digest and ligation of A*, B*, F*, I*, and K* directly from the G blocks
Digested and dephosphorylated psb1c3 with EcoRI HF and PstI HF, made 1ng/µl psb1c3 solution.
Digested A*, B*, F*, I*, and K* with EcoRI HF and PstI HF, made 15ng/ul insert solution.
Cleaned up both inserts and vectors.
Set up ligations with 15ng insert and 5ng vector.
Duke and Lychee
Gel Photo
Managed to successfully run the gel for the test digests failed on 20/07/15. Image of gel shows that digestion following miniprep failed, but the presence of 3kb bands suggests that the ligation was in fact successful. Could send them for sequencing to find out.
Strong 2kb and 3kb bands for D* and E
D# only showed 3kb band
M# and O# show bands slightly larger than 3kb
Gel was loaded in the following order:
1st row: -Ladder-D*IIe-D*2015-D#e-D*2015e-D#2015e2-O#e-D*II-E*1e-M#2015-O#2015-M#2015-E*2-D#-D#2015-
2nd row: -Ladder-E*2e-O#-E1*-D#2015e1-M#e
PCR of QuikChanged J*4(1) with primers for pBAD restriction sites
Gel Photp
Gel extraction EZNA carried out to combine both bands into one sample.
Day 23
Frozen Stock Preparation
K# in MG1655 and deltaFliC made into frozen stocks
Tasks Completed
Transformation of A*, B*, F*, I*, and K*
Digestion of J# and pBADHisB, then ligated.
Overnight cultures of C#, D#, L#, O#
PCR amplification of O#
Test Digest for D*IIe, E*1 compared with digest of completed BioBrick N*3
Test digest shows no problem with restriction enzymes, as completed BioBrick was successfully digested.
Recommendation - for the parts that are going to be redone, it is recommended that one person takes it through.
Day 24
Overnights
Pick colonies and overnights for A*, B*, F*, G*, I*, and K*
Set up overnights for toxicity assay again: E#, H#, K#, N#, P#
Transformations
Transformation of empty pBAD33 and pBADHisB into FliC and MG1655
Transform QCJ# into DH5alpha - Colonies grown
Sequencing Data
M#e and D#2015 sequencing data shows no insert (from 21/07 miniprep digest gel run photo)
Miniprep C#, D#, L#
Test-digest → gel run:
Ligations
Ligate O# into pBAD/HisB
Done overnight, ready for day 25 transformation.
Day 25
Miniprep
A*, B*, F*, I*, K*, G*QC
Sample
ng/µl
A*1
56.8
A*2
65.7
A*3
57.5
Be*1
56.6
Be*2
101.7
Be*3
52.0
Ge*1
30.2
Ge*2
55.2
F*1
362.5
F*2
111.1
F*3
182.5
I*1
335.6
I*2
411.1
I*3
414.1
K*1
373.9
K*13/td>
K*3
Gel Electrophoresis of minipreps
Ladder
A* 1-3
B* 1-3
F* 1-3
G* 1-3
I* 2
I* 3
I* 1
K* 1-3
Gel Photo
G* and F*2 to be sent for sequencing on Monday.
Transform
J#QC → DH5alpha - Amp
DsbADNase (from iGEMHQ) → DH5alpha - Chl
Over the Weekend
Streak plates
EHKNP# - 2 species (10 plates) Amp
(+)(-) - 3 species (6 plates) Chl
Overnight Cultures
pBAD/HisB in MG1655, deltaFliC; pBAD33 in MG1655, deltaFliC, O#
Miniprep
Miniprep of QCJ#
Week 6
Day 26
To Do
Make up more modified M9 media
Melt agar and agarose
Send G*1, F*2 for sequencing
Miniprep + digest O#
Digest J# QC, run gel of O# and J# digest
Gel Photo
No insert for O#, Sent J#3 for sequencing
Day 27
Raffy
PCR C,D,L,M,N#
Day 28
Gel run for 28/07 PCR products, Band Excision, Clean Up
C#, D#, L#, M#, O#
Digest + clean up
pBAD33, C# and D# with BamHI
pBAD HisB, L#, M# and O# with NcoI and PstI
ng/µl
ng/µl
C#
1.1
O#
1.3
D#
0.4
pBAD 33
6.1
L#
0.3
pBAD HisB
3.9
M#
1.1
Ligatation
C# and D# with pBAD 33
L#, M# and O# with pBAD HisB
See protocol here.
Volume/µL
Insert
34
T4 DNA Ligase
1
DNA Ligase Buffer
10
C#
D#
L#
M#
O#
Plasmid
2.5
1.0
0.1
0.2
0.2
Milli-Q
2.5
4.0
4.9
4.8
4.8
N.B. plasmid:insert ratio is significantly lower than 1:3 - will see if this leads to better ligation results
Success
QC J#3 sequence success (ref: 4256167)
Transform successful QC J#3 into MG1655, deltaFliC
Day 29
Sequencing Data
G*e2 QC sequence success (ref: 4256185)
→ PCR it to make #
Gel band excised and stored at -20degC
Transformations
Transform expression contructs ligated yesterday into DH5alpha
C#, D#, L#, M#, O#
(only 1µL plasmid used, may potentially not work)
Transform QCJ# into MG1655, deltaFliC.
Plate out BioBrick (T4 Holin) agar stab
Plated on Amp (pSB1A2)
Day 30
Overnight cultures
For QCJ# MG and deltaFliC; C#; T4 Holin BioBrick
Transformation into DH5α
C#, D#, L#, M#, O#
To Do
Cleanup; Digest; Ligate QCG#
Over the Weekend
Make stocks of J#QC
Innoculate C# D# L# M# O#
transform #s and interlab for characterisation
Make stocks of J#QC
Overnights of #s and interlab for characterisation (all LB media)
Miniprep C# D# L# M# O# and Bba
Week 7
Day 31
Miniprep
Gel Photo
Minipreps for C#, D#, L#, M#, O# → Test digest and run gel (done yesterday)
Send D#1, L#3, O#1 for sequencing
Day 32
Sequencing Results (4257633):
pBADHisB Art-175 (O#1) - success
pBAD33 Lsr Holin (D#1) - 1 mutation, ask them to resequence
pBADHisB YebF Art-175 (L#3) - 1 mutation, ask them to resequence
Day 33
George
Miniprep G#
Helen
PCR K # → * KHTS primers, annealing temperature 72*C, extension time 2 mins
Cleanup C#, M# digested → ligate
Mabel and James
Set up overnights - add J# to them
All in LB Chl or LB Amp (inc. interlab)
Day 34
Kyle
Sequences confirmed (4257633) for D#, L#, O#. Therefore transform into MG, FliC.
Transform C* into MG, FliC
Helen
Transform ligated C#, M# into DH5alpha.
Day 35
To Do
i)
If colonies have formed from the transformation of ligated C# and M# in DH5alpha, pick colonies and set up overnight cultures in the afternoon. Someone will need to come in on Saturday to miniprep these cultures, consult Mabel for tips and tricks on miniprepping if necessary.
James and Lychee performed minipreps on Saturday and will update on where the miniprepped DNA are in the freezer.
ii)
If colonies have formed for the MG1655 and deltaFliC with C*, D#, L#, O#, pick colonies and set up overnight cultures in the afternoon. These cultures do not need to be miniprepped. The next day, make two tubes of frozen stocks out of each culture, and also since it is now the end of the week, streak some of the liquid cultures onto agar plates supplemented with the appropriate antibiotics. If unsure about where frozen stocks of expression cultures should go, consult Mabel.
James and Lychee made frozen stocks out of these cultures on Saturday.
iii)
E* has yet to be inserted into pSB1C3: PCR from gBlock solution.
E* PCR failed. We shall consider PCR-ing directly from gBlocks again, or design long primers to PCR E* out of the completed E# (inserted in pBAD/HisB) plasmid.
iv)
We currently have D# (Lsr Holin), but not an endolysin needed to complete the Holin-Endolysin lysis system. In case our Art-E part (M#) again fails to successfully be cloned, procure BBa_K112806 [pSB1C3] from the 2015 Distribution Kit (Plate 3, Well 12K) and transform into DH5alpha. Subsequently we can potentially insert a promoter in front of it to complete the lysis system.
Colonies formed and the plate was transferred to the cold room.
v)
Since B* failed to PCR to give the Synbiota LasR and pLas parts that we needed, we will instead obtain them from the Distribution Kit as well. They are both in pSB1C3, LasR being BBa_C0179 in Plate 3 Well 6M, and pLas being BBa_R0079 in Plate 3 Well 5C respectively.
Colonies formed and the plates were transferred to the cold room.
vi)
G# miniprep yesterday showed no insert: Using QC G* as template, PCR using primers DspB forward and Post Suffix Holin to make it into G# for cloning into expression vector.
PCR succeeded, excised gel bands kept in freezer.
vii)
Continue with KHTS - ligate overnight, provided someone can put them in the freezer tomorrow morning.
Transferred into freezer.
viii)
Transform the part 2015 Kit Plate 2 Well 17M and then grow in agar plate. Silas will need to film doing the transformation and the plating process.
Completed.
Week 8
Day 36
To Do
Test digest C#, M#, if working send them for sequencing.
PCR E* (normally).
Cleanup G# from excised gel, digest (remember to digest some pBAD/HisB plasmid as well), heat inactivate, dephosphorylate the plasmid only, ligate the both of them.
Transform K* (KHTS).
Synbiota PCR
N.B. Our original gBlock solutions are 10ng/µL - use 2µL of that for the PCR
Day 37
PCR C# from C*pSB1C3, M# from M*
Why won’t E* PCR??
G# ligated into HisB (the tube was labelled G* into HisB)
K* transformed - check if colonies formed
Day 38
Out of Q5, so use Pfu - this involves adding nucleotides, different annealing temperatures - ask Chris or George
BioBricks
F and I with new restriction enzyme combination (XbaI, PstI-HF) - up to ligation overnight (Mabel)
PCR K# to K* (Kyle)
Expression vectors
E# - try one more time but then if it doesn’t work come to Leon or Raffy to design long primers (Kyle)
G# pick colonies (Kyle)
C# and M# repeat PCR (Helen)
Day 39
Design (E,D)# to (E,D)* primers
Transform SBArtE2 into DH5alpha i.e. M* hopefully
Transform the ligated F*, I*s which were directly digested from gBlock using X, P
Miniprep G#
NB: did phusion go back in the freezer from which it came? Ensure it doesn’t go in our freezer.
Day 40
Pick M*, F* and I* colonies
PCR G* to G# and E# to E*
K* and C# ligated overnight - transform
Over the Weekend
Miniprep of F*, I*, K*
Nanodrop:
ng/µl
ng/µl
ng/µl
F*1
353.5
I*1
464.1
M*1
204.2
F*2
430.0
I*2
428.4
M*2
148.3
F*3
351.3
I*3
363.8
M*3
189.1
N.B. I*1, I*2, M*1 and M*3 are not pure samples (curves on nanodrop had weird peaks) - these are the eppendorfs without smiley faces drawn next to the [DNA] on the side of the tube.
N.B.B. Mabel has a new PB for [DNA] on the QIAGEN miniprep kit ;)
Restriction digest: 20ul reaction, 37degC for 90 mins
M* also digested with XbaI and PstI-HF - assume that the insert would have the same MCS as F* and I* if they’re all meant to be for biobricks. See protocol here.
Gel: 1% agarose
Only needed ~35 mins to stain well - due to high [DNA]??
M*2 can be sent for sequencing (only ‘normal’ curve on nanodrop)
F*1
F*2
F*3
I*1
I*2
I*3
M*1
M*2
M*3
Gel Photo
Streak all frozen stocks - in two locations
Bottom most box in the 3rd column has a portion of the frozen stocks but not the new ones
New ones are in the topmost box in the second column
Overnights:
All interlabs (unnecessary duplicates of deltaFliC but nevermind) and extra 20K MG and 20K DH5
E# and J# + controls (duplicated)
P. putida
Week 9
Day 41
Last few bits of cloning
Transform C* (in complete biobricks box) into M51655 and RP437 (oops C* is already transformed and D* doesn’t exist)
Send M*, F* and I* for sequencing
→ F1, M2 and I3 sent
Day 42
PCR M# from M*
Synbiota PCR
O* (Art-175 on its own) isn’t a BioBrick - let’s make it!
Day 43
Test digest of C# -> run on gel -> send C#1dF for sequencing -> not right.
All C# miniprep values were very very low, probably due to tranformation into wrong strain. PCR C# tomorrow.
PCR G# -> run on gel -> gel extraction -> restriction digest.
Day 44
Carry on from gel extraction of G# - restriction digest -> clean up -> nanodrop (Kyle has values) -> ligation.
PCR C* to C# again.
Day 45
C# gel - excise bands, continue
G# ligation - This is in the front of our top freezer drawer. Transform this into DH5a
Week 10
Day 46
Constructs PCR
New primers arrived - but mistake made with M (thought it was # to * but actually need * to #, so using old primers ordered previously)
Construct
Fwd primer
Rev primer
Annealing temp.
P (# to *)
DNase HTS
Art E suffix
72
K (# to *)
DsbA HTS
Art E suffix
72
E (# to *)
DsbA HTS
Art E suffix
72
C (# to *)
Lsr Holin HTS F
Lsr Holin HTS R
71
M (* to #)
SMAP 29
Art E suffix
70
Also used Phusion Polymerase but primers are at 10µM instead of 25µM.
Ladder
T4 Holin
pLsr
C*
E*
K*
P*
Gel Photo
G# colonies picked, in overnight broth in 37 degree incubator. Plates are in the cold room.
Day 47
Molecular cloning of constructs
Gel extraction of C*, E*, K*, P* - eluted with 35ul of EB
Restriction digest of C*, E*, K*, P* and pSB-1C3 plasmid with XbaI and PstI-HF in a 50ul reaction, 37 degC for 1 hour at 300rpm
95degC heat inactivation for 30mins
Add 1µl CIP for pSB-1C3 only at 37degC for 30 mins
Cleanup - elute insert with 35µl EB buffer, plasmid with 50µl
Ligation set ups: 50ul reaction, use all 34ul of eluted DNA
Incubate at 16 degC overnight
N.B. There is some reaction mixture of M* to M# left over from 24/08 so will use this to run a gel with the synbiota PCRs.
Day 48
Kyle
G# miniprep -> nanodrop -> test digest
Mabel
M# mentioned in synbiota section
Transformation of C*, E*, K* and P* into DH5α
Plated transformations onto Chl plates - watch out for potential mixup of P* and K* in the low concentration bacteria plates
PCR of D# to D* using Pre prefix holin and post suffix holin primers, standard Phusion set up, annealing temperature 71degC
Day 49
PCR of O# to *, using new primers that arrived on 24/08 (thought they were for M# - * but were actually for O)
Forward primer: Art-175 HTS, Reverse primer: Art-E suffix
Diluted to 25uM, annealing temperature 71degC
Three different setups:
Phusion RDP protocol
Phusion RDP protocol with GC buffer from Chris (optimised for primers with a high GC content e.g. Rhodobacter genome)
Phusion RDP protocol with 5% DMSO
Template was not digested before PCR
RDP
RDP
GC buffer
GC buffer
DMSO
DMSO
Gel made with water…………………………………!@%*#%$^
E*, K* and P* (we actually already have C*) - 2 colonies picked from each plate, so total of 4 overnights per construct (high and low conc plates)
Day 50
Repeat PCR of O# to *, Phusion set up as on 27/08 but no DMSO or buffer change.
Miniprep of E*, K* and P*: all eluted in 50ul EB buffer
Nanodrop (adjusting volume added to digest to keep [DNA] roughly the same)
37 degC for 90 mins
Gel: 1% TBE agarose, 120V
it didn’t melt!  YAAAAAAAAAASSSSSSS >8)
E*1
E*2
E* [ ] 1
E* [ ] 2
K*1
K*2
K* [ ] 1
K* [ ] 2
P*1
P*2
P* [ ] 1
P* [ ] 2
O#-* 1
O#-* 2
O#-* 3
O#-* 4
Excised all O* bands (now in freezer)
K* [ ] 1 can be sent for sequencing (insert length should be 1030bp)
Week 11
Day 51
Molecular cloning day YAYZ
Gel extraction of D* and O* PCRs
Digest of D*, O* (50ul set ups) and pSB-1C3 (100ul set ups) with XbaI and PstI-HF
Heat inactivate inserts and plasmid at 95degC for 30 mins
Add CIP to pSB-1C3 and incubate at 37degC for 30 mins
Clean up: inserts eluted in 35µl, plasmid eluted in 50µl
Ligations: 50ul set ups
All with 5ul of 10x T4 DNA ligase buffer and 1ul T4 DNA ligase
From the nanodrop values omitted O*3 because [DNA] is too low.
Incubate at 16degC for 16 hours
PCR G* to G#
3 different conditions: RDP protocol, RDP protocol with GC buffer, RDP protocol with 5% DMSO
Gel: 1% TBE agarose, 80V
5% DMSO
5% DMSO
GC buffer
GC buffer
RDP
RDP
Something is wrong with the camera :/ - everything except the DMSO PCRs have been excised
Transformation of pSB-1C3 into DH5α - plate onto Chl plates
Accidentally transformed all the DH5α - need to make DH5α competent cells from plate in cold room (given on 23/07 so not sure how successful this is going to be…)
To Do
Made 5ml of Chl antibiotic stock (into EtOH)
Overnights: re-pick from E* and P* plates into Chl LB
3 colonies per plate, 12 overnights in total
Day 52
Mabel
Miniprep E* and P* overnights - all eluted in 50µl buffer
20µl digest with XbaI and PstI-HF
Gel: 1% TBE agarose, 120V
1kb+ ladder
P* 1-3
P*e 1-3
E* 1-3
E*e 1-3
June
Making DH5alpha Competent Cells
20 eppendorf tubes are restocked in -80C
Transforming O* and C*
Plates are : O*1, O*1e, O*2, O*4, D*1, D*2, D*1e, D*2e
x10 conc plates gave good colonies
Picking colonies and setting up overnight cultures for psb1c3 transformed in DH5alpha
Miniprepping E* and P* overnights
Day 53
Repeat digest of E* and P* cos I’m an idiot and i forgot to write down what they were
E*2
E*3
E*e 2
P*2
P*3
P*e 2
Send E*2 and E*e2 for sequencing forward read only - whichever one comes back will get a reverse read.
Miniprep of pSB-1C3 - all eluted in 50µl
Digest with just XbaI to linearise for gel
1% TBE agarose, 120V
pSB-1C3 1
pSB-1C3 2
pSB-1C3 3
pSB-1C3 4
pSB-1C3 5
Mystery of the massive bands: case closed.
The massive band is linearised pSB-1C3!!
Day 54
Miniprep of pBAD HisB, D* and O* overnights - all eluted in 50µl EB
Digest: * with XbaI and PstI-HF in cutsmart buffer, HisB with NcoI in 3.1 buffer
Moved into freezer - run gel tomorrow
Moved E*2 into biobricks box :)
Day 55
Digest of M#, G#, pBad HisB
G#: BspHI, PstI-HF in 3.1 (should be cutsmart but Mabel forgot to write the buffer for that set of enzymes)
M#: NcoI, PstI in 3.1
pBAD HisB: Nco, PstI in 3.1
Digests from Kyle
Heat inactivate at 95 degC for 30 mins
Add CIP to pBAD HisB for 30 mins at 37 degC
Clean up: HisB eluted in 50µl, inserts in 35µl
Ligation into pBAD HisB: all with 5µl buffer and 1µl DNA ligase
N.B. The G# ligations are slightly over 50ul - diluted HisB too far in digest
Leon to move ligations onto Mabel’s orange rack in the -20 freezer.
Molecular cloning
Run gel (1% TBE agarose, 120V) of O*, D*, pBAD HisB and remaining P# - * PCR from 24/08 with 1kb+ ladder
D*1
D*2
O*1
O*2
pBAD HisB
P# - *
P# - *
P# - * PCR has been excised and gel extracted - on orange rack in freezer
Send O*1 (as Art-175 1) and K* high conc.1 (as Art-175 DsbA 1) for sequencing with correct primers
Week 12
Day 56
Transformation of #s
M# and G# RDP and GC
Plated on amp.
Day 57
Sequencing: send K* and O* for reverse reads with reverse primers
Biobrick submission: biobricks box contents correlates with master table and exceed the minimum required [ ] for submission.
Transformations of G# and M# have been binned - will not be making anymore #s, focus on last round of biobricks.
PCR of D* and P* - aim to troubleshoot/test as many different variables as possible
Not enough of P# so only carry out Phusion PCR - rest is given to Duke to transform.
P# to P* (Phusion)
D# to D* (Phusion)
D# to D* (Phusion)
D# to D* (Q5)
Mabel when putting the gel in the tank: This feels slightly watery… it’ll be fine!
*gel starts running funny after 5 minutes*
Mabel: James………..
James: …???
Ria: hmm it looks weird.  Those bands are really slow.  Did you make the gel with water?
Mabel: No of course not who would make a gel with water.
James: I did! ……………...
Mabel: ……………………………………………..
WHY HA-MES WHY T^T T^T T^T T^T T^T T^T T^T T^T T^T
Gel put straight into EtBr, entire section of gel that has been run excised - sort out tomorrow
Day 58
PCR rescue mission: Gel extraction, re-run on gel, excise.
Q5 P# to P* PCRs repeated as they didn’t run far on gel yesterday.
P# to P* (Phusion)
D# to D* (Phusion)
D# to D* (Phusion)
D# to D* (Q5)
D*= 880bp, P* = 570bp
PCRs saved :) all bands excised!  (count yourself lucky Ha-mes)
Day 60
Transform pBADHisB GFP into MG and RP.
Transform O# D# hybrid. Overnights also made directly from post-heat shock incubation culture.
Overnights: rhodo, putida, LYer, EM, ER, JM, JR, HisM, HisR, OM, OR, DM, DR
1 tube - no AB LB
1 tube - Amp, BHI
1 tube - no AB SUX
8 tubes - Amp LB
2 tubes - Chl LB
K* high conc 1 and O*1 reverse reads correct - put into biobricks box :D
Gel extraction of 2nd gel of D* and P* PCRs - all eluted in 35ul buffer EB. N.B. potential mix up of D Phu 2 and D Phu 3 reaction
37degC for 90 minutes. Heat inactivate at 95degC for 30mins. Add CIP to pSB-1C3, incubate at 37 degC for mins
If [insert] < 3.0 ng/ul, did not proceed with ligation (can’t get dilution of pSB-1C3 to the correct ratio)
Plasmid dilutions: 1:2 except for pSB-1C3 (XbaI + PstI) by adding 49µl of MilliQ
All with 1ul T4 DNA ligase and 5µl T4 DNA ligase buffer
16˚C for 16 hours
Over the Weekend
Transformation of D* and P* ligations from 11/09 - plate on Chl agar.
Check transformations:
15 successful plates (thank you June!)
2 overnights made of for each plate with colonies (LB Chl) - 30 in total
Also made 2 x P# overnights (LB Amp) and 4 x interlab overnights for Ria (LB Chl)
Week 13
Day 61
Minipreps pf D*, P* and P#
Digest set ups: 37 deg for 90 mins
N.B. 2µl of template used for D Q5 1a, D Q4 4b, P#1 and P#2 due to lower [DNA]
P# digested with NcoI and PstI-HF in cutsmart buffer
Gels: 2% TBE agarose (for better separation as D* = 880bp and P* = 570bp), 120V, both used Thermo Fisher 1kb+ ladder
P#1
P Phusion batch a 1,3,4,7,8
D Phusion batch a 1,2,4,5,7,8
D Q5 1a
P#2
P Phusion batch b 1,3,4,7,8
D Phusion batch b 1,2,4,5,7,8
D Q5 1a
P#1 moved to pBAD + inserts box, P#2 discarded
D Q5 2a
D Q5 4a
D Q5 7a
D Q5 2b
D Q5 4b
D Q5 7b
*weeps* No D* or P* :(((((((
Molecular cloning: over and out ;)
Characterization
Plan
Overview of sequences in pBAD
pBAD 33 LasR Holin
A#
No PCR
pBAD 33 LasR sfGFP
B#
pBAD 33 Lsr sfGFP
C#
Overnight Culture
pBAD 33 Lsr Holin
D#
Miniprepped, awaiting gel --> sequencing
pBAD HisB DspB YebF
E#
Transformed into MG1655, deltaFliC
pBAD HisB DspB
F#
pBAD HisB MccS
G#
QC in progress
pBAD HisB DspB Fla
H#
Transformed into MG1655, deltaFliC
pBAD HisB DspB DsbA
I#
pBAD HisB Art-175 DsbA
J#
QC introduced extra sequences (order no 4254194); send another
pBAD HisB Art-175 YebF
K#
Sequence confirmed (order no 4253587), awaiting transform
pBAD HisB Art-E
L#
Overnight culture
pBAD HisB Art-175 Fla
M#
Miniprepped, awaiting gel --> sequencing
pBAD HisB Art-175
N#
Transformed into MG1655, deltaFliC
pBAD HisB Art-175
O#
Miniprepped, awaiting gel --> sequencing
pBAD HisB DNase
P#
Transformed into MG1655, deltaFliC
Currently completed pBADs (frozen stocks of plasmids in MG1655 and deltaFliC made):
With secretion tag - E# (DsbA DNase), N# (Fla Art-175)
Toxicity assay: dilute down with varying conc of inducer
Different growth conditions - flagellar need TB media 30degC
Fluorescent
Plate streaking → colony picking → liquid culture → spin down → supernatant contains proteins
Protein purification using His-tag affinity (SDS-PAGE then Ni(II)/Co(II) column/nickel-chelated horseradish peroxidase? Or use Western Blot using antibodies selective for His-tag?)
Nanodrop at 200nm and 280nm to measure protein concentration in eluent
Assaying for DNase:
Culturing on DNase test agar (if secretion tag works, no need to purify protein) → visualize DNA hydrolysis either by i) HCl flooding to precipitate DNA out ii) using a dye such as toluidine blue
If we want to make use of the purified proteins - need to have a solution-based assay that can detect polymerized DNA vs hydrolysed DNA?
Assaying for antipseudomonas activity: the Chang paper used PAO1 as target (for DNAse, MccS):
Grow PAO1 in 150uL medium in 96-well, incubate 24h with peg lids → rinse pegs with PBS → immerse pegs in plate containing purified DNase → crystal violet staining (alternatively maybe can just grow biofilms at bottom of wells - harder to wash)
Grow PAO1 similarly, then immerse in culture of DNase-secreting E. coli
Crystal violet staining: stain, rinse, de-stain, measure absorbance at 575nm
Assaying for Art-175: basically incubate PA (the original Art-175 paper used PA14) with purified Art-175 (or supernatant since our Art-175 has secretion tag) and monitor OD at 600nm over time (refer to paper)
Without secretion tag - H# (MccS), P# (DNase)
Need to lyse cells (buffer or sono?)
MccS - Chang’s original paper simply tested for antibacterial activity by adding purified MccS to a culture of PAO1
Secretion tag yes/no study for DNase since we have both
See if DNAse present extracellularly even when there’s no secretion tag
Cell Growth
Week 5
Day 23
Toxicity Assay
Arabinose induction of pBAD expression was studied at various arabinose concentrations to test for toxicity of proteins encoded by our inserts.
Each well has total volume of 200µL - culture (1/200 dilution), arabinose, media.
96-well plate loaded in triplicate according to the following plan:
DF = deltaFliC
MG = MG1655
Percentages denote final w/v concentration of arabinose in each well
Initial monitoring shows that the wells containing only Blank LB has increasing OD600 over time - likely to be contaminated LB, makes the rest of the dataset unreliable as well as the cells were inoculated using the same bottle of LB.
Forgot to turn on incubator so the entire curve was done at room temperature.
Day 24
Toxicity Assay
Forgot to turn on incubator so the entire curve was done at room temperature.
Day 25
Toxicity Assay
Wrong antibiotic in supposed positive control wells.
Temperature - 37degC.
Week 6
Day 26
Toxicity Assay
96 well plate layout
Pipetting triplicates only - 1 eppendorf split between 3 wells
Eppendorf Set Up
Volume (µL)
LB + Amp OR Tryptone + Amp (both 1:1000 dilution of Amp)
985
20% OR 2% OR 0.2% Arabinose
10
Stationary phase overnight culture
5
Plate reader: toxicity assay script
90 cycles, 900 seconds
200rpm shake when idle
LB Amp: 37degC
Tryptone Amp: room temp (forgot to set to 30
Day 27
Reapeat of Yesterday
Toxicity assay, in Tryptone Broth at 30degC
Day 29
Toxicity Assay
Replace 0.002% Arabinose with MilliQ.
85 cycles, 900 seconds, 30degC
Week 8
Day 36
Toxicity Assay
It has been previously established for E#, J#, K#, N#, P# that 0.2% arabinose incubation is non-toxic (for H# deltaFliC, toxic even under 0.002%). For all constructs, 0.3% and 0.4% are both quite toxic levels of induction.
We now want to investigate whether 0.2% arabinose induction is toxic for new constructs D#, L#, and O#.
Day 37
Set up 1%, 2% and 4% arabinose toxicity assay for (E, H, J, K, N, P)# and two more rows for controls.
Day 38
Toxicity Assay
Set up 1%, 2% and 4% arabinose toxicity assay for (E, H, J, K, N, P)# and two more rows for controls.
Toxicity data will be available mid-morning and will show if we can use a higher % ara; if higher [ara] works, incubate overnights with 4% ara (or appropriate according to toxicity assay) and SDS-PAGE.
Expression and Secretion
Week 6
Day 29
TCA Protein Precipitation
For the protocol see here.
Using old supernatants, and potentially spin down the cells from yesterday.
Couldn’t have worked because arabinose wasn’t added to cell cultures → suspension wouldn’t have had secreted proteins.
Day 30
Secretion Assay
Culture cells to mid-log in LB and TB
For LB, keep in 37degC throughout
For TB, once reaching mid-log, move to 30degC
Culture for another 1 hour
Take 1.6mL of the culture, centrifuge, keep supernatant and discard cells
TCA precipitation protocol
(can refer to Andi’s by going back out to this week’s folder)
Catch Jia when she’s around and get her to show us where the SDS-PAGE stuff are.
Week 7
Day 32
Secretion Assay
EJNK# in both strains, 1.5 hours incubation at LB at 37degC (1:10 dilution of overnight stationary culture), then 4 hours incubation with 0.2% final conc of arabinose.
Day 34
TCA-Precipitate
TCA-precipitate cell-free supernatant of EJKN# and run SDS-PAGE
E#MG, N#MG no secretion; J#MG, K#MG, J#DF, K#DF, N#DF lysed; E#DF successful secretion.
Week 8
Day 36
Supernatant Acquisition
Wash 1.5mL of overnight cultures of E#, J#, K#, L#, N#, ctrl MG1655 in antibiotic-free LB Broth twice and grow them in 1:50 dilution with 0.2% arabinose in conical flasks. Obtain 4x1.4mL aliquots of their supernatants after 4 hours incubation at 37degC.
Do the same for H# MG1655, except don’t obtain aliquots of its supernatant. Instead, obtain aliquots of culture so that we eventually can lyse the cells to obtain MccS from the lysate.
SDS-Page
Obtain supernatants of deltaFliC constructs from cold room - carry out standard SDS-PAGE protocol.
Day 38
Assaying
Run PAGE on the supernatants obtained from the M9 MG1655 cultures from yesterday. Do this first thing in the morning and hopefully as we move onto doing the interlab stuff the gel will develop enough to show some bands.
If bands formed on the PAGE we left to develop overnight, we will incubate the now cleaned biofilm plates with the same supernatants used to PAGE yesterday.
Day 38
So far, only E# has worked for SDS-PAGE. Need to optimize for the rest of the constructs - (D,E,H,J,K,L,N,O,P)#
Today, let’s just focus on the Art-175 constructs because it’s too difficult to do all of them. So, that’s (L, N, O and K)# overnights - see TCA precipitation (taking up8 flasks)
When setting up flasks for supernatants, wash with antibiotic free media.
→ Also try Art-175 supernatants against the biofilm degradation plates
Day 39
Run SDS-PAGE in the morning for the supernatants
(L, E, J and P)# prepare supernatants and run PAGE. Potentially do more if growing in falcon tubes
Day 40
(L, E, J and P)# left secreting overnight - isolate supernatants for SDS-PAGE
Week 9
Day 42
Set Up for TCA
Dilute the 4 Art-175 overnights - 16 conical flasks
Secrete at 18C overnight and TCA ppt tomorrow
Secretion
Prepare 50/100 ml volumes of Ara + cell cultures for purification from supernatant
Determine buffers for purification.
Day 43
Expression
Lysis and His tag purification of P# (DNase)
SDS-PAGE for Nickel column fractions collected Day 42
TCA ppt for overnight secretions of Art-175 constructs - store ppts
TCA ppt for the overnights (secretion for four hours in the afternoon)                                             → Dilute 1/20 in LB + appropriate antibiotic first thing in the morning so that we can start secretion asap
Purification
So far - DNase DsbA and DspB DsbA expressed 4hrs, DNase expressed overnight, Art-175 group expressed 4 hrs and overnight
Setting up secretion for DsbA tagged and DspB that I can run on a column tomorrow - 4 hrs secretion
Western Blot
Art-175 group expressed 4hrs and overnight - ready to PAGE O,K,N,L
Set up overnights for DsbA group - prepare TCA ppts from overnights tomorrow - also need to set up secretion overnight for DsbA group.
Day 44
Expression
Art-175 buffers made up
Run columns for Art-175 group (50ml + 20ml 4hrs secretion with no resuspension buffer) + DNase (resuspension buffer)
Day 45
Expression
MG constructs 1/40 dilution and purify supernatants to run on column
Lysis of cell pellets to run on column
SDS-PAGE for Art-175 group
Week 10
Day 46
Grew up 3x J# DF, and H# MG overnights to 0.6 OD, and followed protocol past NiNTA purification, ready for SDS-PAGE tomorrow.
No bands.
Day 47
Duke (aka king of banter)
Listened to reggae music (you know, Bob Marley, UB40, the classics) whilst eluting L#, M# and O# (all in DeltaFli) through nickel column, ready for SDS-PAGE tomorrow.
Day 49
Western Blot
NB: Duke/Silas when analysing the results, refer to the gels in the Coomassie blue stain - there is some inconsistency as to how lanes were loaded i.e. if a lane was left free between the ladder and the samples
Refer to the Western blotting protocol
Primary Ab stored as 10 uL aliquots in the -20CTo immunostain, dilute by 1/1000 in 10 mL 1% milk powder and PBS (the full 10 uL aliquot in 10ml milk solution). NB: Andrea said that this will probably be too bright - let’s see what she thinks when we come to analyse the data
The PBS and PBS + Tween 20 (Tween 20 is the same as Tween - it’s a typo on the protocol) solutions are on our shelf
The secondary Ab is in the back fridge, top shelf, has a mouse drawn on it. This also requires 1/1000 dilution into 10 mL 1% milk solution (remember to dilute the 5% milk solution with PBS, not water) for the second staining step
Refer to Andrea’s help for the visualisation step
Secrection
Constructs induced to express and pellets frozen - stored in the -80C
Day 50
Western Blotting
Carry on with immunostaining of the blots on the shaker
Carry on with immunostaining of the blots on the shaker
They were washing in PBS overnight - this is not normally required - in the future stick to the timings on the protocol
Immunostain the two blots that have been in milk solution overnight
Biofilms
Week 5
Day 25
Test crystal violet staining of biofilms (using 23/07 tox assay plate)
Protocol.
0.1% w/v crystal violet solution made up by dissolving 0.080g crystal violet in 80mL Milli-Q.
96-well plate was taken out of plate reader at 10am and left on the bench until 4pm.
Contents of plate decanted into biological waste bucket and each well of plate washed with tap water for 4 rinses to remove planktonic cells.
125µL of 0.1% crystal violet solution added to each well, and left to stain for 20 minutes.
Contents of plate decanted and each well rinsed thoroughly again to remove unstained crystal violet.
Plate left to dry overnight.
Comments: Growth curve assay plates are non-ideal for biofilm screening as the shaking incubation of the plates in the plate reader means that biofilms are formed more on the walls of the wells than the bottom of the wells.
Week 6
Day 27
Biofilm Cultivation Using Existing Transformed Strains
Biofilms grown in triplicate wells according to tox assay layout, except the wells with 0.002% final arabinose conc is replaced with 0% arabinose conc (i.e. arabinose replaced with Milli-Q).
Each well has 100uL total volume of 1:100 diluted overnight culture.
Round bottomed plate was used.
Left for incubation on bench for 48 hours.
Day 30
Biofilm Staining
First, measure the OD600nm of the biofilm culture.
Decant and wash the round-bottom well plate that has been incubating at room temperature for 2.5 days (see protocol)
Do CV staining and monitor 500-600nm using whichever machine is available
If biofilms are shown to grow, we will proceed with monitoring biofilm kinetics
Week 7
Day 31
Assays
Various biofilm assays
3 modified E. coli plates and co-culture with wild-type (for testing of biofilm inhibition); 3 wild-type plates for eventual testing of biofilm degradation
N.B.30degC plate (modified E. coli plate) has B6 and B7 swapped around
Day 34
Set Up
Set up conical flask cultures for E# and J# (both strains) to test with wt-E. coli biofilm plates
1:10 dilution with LB amp
1.5 hours initial incubation, 4 hours incubation with 0.2% final conc of arabinose (190µl of 20% arabinose in 19ml of solution), incubation from 12:30 to 4:30pm
OD values after initial incubation:
hisB MG : 1.12, hisB deltaFliC : 0.73, E# MG : 1.1, E# deltaFliC : 0.72, J# MG : 0.74, J# deltaFliC : 1.2
4mL of supernatant isolated from each culture, kept in cold room for addition to E. coli biofilms set up on 03/08
Day 35
Biofilm Assaying
Set up P. putida 96-well biofilm plates and incubate for 48 hours at 30 degC without shaking: P. putida overnight culture diluted 1:100 in a square petri dish in LB Broth (no antibiotics), and 100uL of the diluted culture filled into each well of one tissue culture-treated round bottom-well 96-well plate.
Set up E. coli MG1655 96-well biofilm plates and incubate for 48 hours at room temperature: MG1655 (with pBAD/HisB) diluted 1:100 in a square petri dish in LB Broth supplemented with ampicillin, and 100uL of the diluted culture filled into each well of one tissue culture-treated round bottom-well 96-well plate.
Incubate E. coli biofilm plates that were cleaned yesterday with supernatants from E# and J# that were isolated yesterday.
Wash some overnight cultures of E#, J#, K#, N#, and control deltaFliC with non-antibiotic tryptone broth (same way as Interlab step ii) twice, then grow them in non-antibiotic tryptone broth (1:50 dilution, make 20mL cultures) with a final arabinose concentration of 0.2% for 4 hours. The supernatant here will eventually be used for both secretion assay and biofilm assay/cell-killing assay with P. putida. 4x 1.4mL aliquots of supernatants were obtained from each culture.
Week 8
Day 36
Biofilm Assaying
NB: All we’ve been doing thus far is getting constructed E. coli to grow biofilms (do our constructed cells with ara make biofilms? do cells not secreting grow biofilms? Always do an OD during biofilm prep to make sure you have cells!)
Currently, we also have wild type E. coli growing biofilms to which we add constructed E. coli supernatant
Retrieve plates that have E. coli biofilms being incubated with supernatants from 37 degC incubator, wash and do CV staining to test for biofilm degradation (Leon remember to fill in the layout file for this (dated 07/08). (Raffy, the OD of these plates wasn’t taken thus ii) being done)
Retrieve MG1655-biofilm growth plate (for eventual degradation) from benchtop, take OD600 reading to make sure every well has had cells in them, then decant and wash planktonic cells away. The plate is then ready to be incubated with supernatants.
Retrieve P. putida-biofilm growth plate (for eventual degradation) from 30 degC, take OD600 reading to make sure every well has had cells in them, then decant and wash planktonic cells away. The plate is then ready to be incubated with supernatants (make sure antibiotic free supernatants - the ones in cold room made up on Friday are antibiotic free).
Set up again biofilm viability plates using cultures of our constructed E. coli. Can potentially set up two plates, both of which left on bench (~25degC incubation). (Raffy, is biofilm growth viable?)
Set up biofilm inhibitory plates by culturing P. putida in supernatants of our constructed E. coli, as well as non-constructed E. coli as controls.
Day 37
If visual inspection tells you that P. putida biofilms indeed have formed in the previous 96-well plate, set up two more of them using the test tube of P. putida in the 30 degC incubator (1:100 dilution per well, 100µL of diluted culture per well).
Day 38
Biofilm Viability Assays
Can our constructed cells form biofilms - set up after lunchtime.
PAGE tells us that supernatants aren’t great, so even if we see degradation, results aren’t conclusive.
Biofilm degradation
E. coli and putida plates are drying - test Art-175 supernatants.
Biofilm inhibition
Incubate bacteria together with the stuff that forms biofilm and see if biofilm forms.
Day 39
We have set up viability (on the bench) and is according to Raffy’s toxicity map 12/08, leave another day to grow.
Set up another viability assay same as yesterday
Also have set up degradation for putida and E. coli - write the map down; stain tomorrow afternoon
Grow some more biofilms to degrade
Biofilm inhibition assays - grow E. coli biofilms in DspB and DNase supernatant
Day 40
E. coli biofilms growing on the bench (set up at 5 on 14/08) - leave until Saturday
Read biofilm degradation - retrieve 96 well plates from incubator. Read OD600, then wash and read OD spectra between 500-600
Use the overnight supernatants to degrade the biofilms growing from 12/08
Set up another inhibition plate
Week 9
Day 42
Set Up
O# and L# O% arabinose growth curve and biofilm viability
Test 1 day old putida biofilm with supernatants
Biofilm viability for E# and J#
NB: *** Take OD reading before you stain the biofilm (just because they don’t form biofilms when you induce expression)
Didn’t overnight putida - overnight tomorrow to test to cell killing assay on - also grow another putida biofilm
Week 11
Day 54
P. putida testing
1 plate of biofilm grown for 1 day
K, L, N, O, HisB, grown at 30°C for 2 hours, then induced with 0.4% ara at 30°C for 4 hours, then left to sit at room temperature for 2 hours
100µL of supernatant from above added to 100µL of stationary culture of P. putida, and 100µL of 1:20 diluted culture which have been left sitting for 4 hours.
Run in clariostar together with toxicity assay
Preliminary observations: Toxicity in H and L as expected
N supernatant displays some extent of P. putida-killing capacity - secretion conditions to be further investigated.
Biofilm Viability Assay
Stationary cultures diluted 1:100 into 200uL Amp-supplemented LB per well.
Incubation at room temperature.
K MG 0.2% ara (1-3)
L MG 0.2% ara (4-6)
N MG 0.2% ara (7-9)
O MG 0.2% ara (10-12)
K MG 0.02% ara (1-3)
L MG 0.02% ara (4-6)
N MG 0.02% ara (7-9)
O MG 0.02% ara (10-12)
K MG 0% ara (1-3)
L MG 0% ara (4-6)
N MG 0% ara (7-9)
O MG 0% ara (10-12)
ctrl MG 0.2% ara (1-3)
ctrl MG 0.02% ara (1-3)
ctrl MG 0% ara (1-3)
Observation: L MG 0.2% and 0.02% shows significantly impeded cell growth (low cell density) - perhaps it would be better to test for biofilm growth by placing them in wells for induction using a mid-log culture (grown without arabinose beforehand).
Biofilm viability assay
1:100 dilution of stationary cultures into 100µL of antibiotic-supplemented LB in each well of round-bottom 96-well plate.
1 plate for MG, 1 plate for RP. Same layout.
Incubation at room temperature for 2 day.
0.2% ara (1-3)
0.02% ara (4-6)
0% ara (7-9)
10-12
E
HisB 0.2% ara
H
HisB 0.02% ara
J
HisB 0% ara
K
L
N
O
P
Washed and stained biofilm viability assay, but not yet analyzed
Preliminary observation seems to suggest that incubating for one day is not enough to form detectable biofilms.
Over the Weekend
Biofilm degradation assay
Retrieve 48-hour putida biofilm plate from Day 54 - attempt degradation using artilysin supernatants
(on Monday, consult Chris on how to tune Ca2+ concentration and pH of supernatant)
Analyze MG Day 54 biofilms
Cell Killing
Week 8
Day 38
Incubate putida with supernatant (add supernatant as you would media when setting up plates of putida).
NB: growing of flagella (i.e. Fla secretion) will work better, according to George, at 30˚C
Day 39
Lyse P, O and H to show that it is being expressed just not secreted - ask Chris or George about sonication.
Fluorescence
Week 6
Day 29
Growth Curve
For the protocol see here.
Plate reader hanged on first run; forgot to turn the incubator back on after resetting the program, so it’s a room temperature growth curve.
Day 30
Assay
Fill black well plate with 100uL of stat phase overnights, same layout as usual, take instantaneous read (fluor and OD) using FluorStar.
Week 8
Day 36
Quorum-Sensing Fluorescence Assay
Wash C* MG1655 and deltaFliC in modified M9 media (+ thiamine supplement) once. Dilute 1:100 and monitor growth in microplate wells.
Week 10
Day 50
Quorum-Sensing Fluorescence
1/20 dilution of C* and pBAD33 in LB + Chl
Incubate until OD ~ 0.6
Notes:
C* is pLsr - GFP
“We are using this part to qualitatively test that pLsr functions as expected in response to AI-2, as well as quantitatively measure the kinetics of gene expression when pLsr is induced by a given amount of AI-2.”
We want to show that GFP expression can be induced by AI-2. To do this, it is necessary to use the supernatant of C* in exponential phase and culture stationary phase bacteria (overnight culture) in the supernatant
Producedure:
Incubate cultures to correct OD
Transfer to a falcon tube and spin down at full spin for 15 mins
Transfer the supernatant to separate falcon tubes
Set up 70-80 cycles (~25-30 hours) OD + GFP
Flow cytometry
Interlab
Week 2
Day 8
Preparation of Interlab Study BioBricks
10µL Milli-Q added to wells 20K, 22A, and 22K in Plate 1, and well 21J in Plate 4 in the iGEM Distribution Kit to resuspend the necessary BioBricks [pSB1C3] to prepare for the Interlab study.
The plates were kept on mild shaking until the afternoon.
Transformation of Interlab Study BioBricks into DH5alpha 1.5
Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
Then add volume of E. coli according to protocol, using beads to spread across petri dish.
Day 9
Growth and Culture of Bacteria
Refer to section 1.6 of the protocol guide.
Only one overnight culture each set up for iGEM distros (20K, 22A, 22K, 21J) as it can be rightly assumed that plasmids supplied by iGEM HQ are high purity.
Day 10
Plasmid for PCR Products
Digest aliquots of each of the samples, and run on gel. (Refer to protocol in section 1.2 )
From gel, determine the degree of success of these samples.
Week 3
Day 11
Restriction Digest of InterLab Study Plasmids
20K, 22A, 22K:
Component
Volume / µL
Plasmids
10
SpeI
1
PstI-HF
1
Milli-Q Water
6
CutSmart
2
21J: Same as above, but replace SpeI for XbaI.
Incubated for 2hrs at 37℃, then enzymes heat inactivated at 95℃.
Day 12
Restriction Digest of InterLab BioBricks
Gel Photo
Pulse spin
Dephosphorylate 20K, 22A, 22K, BUT NOT 21J
Load dye and run gel
Gel image (transilluminator lens was quite grainy today)
Appropriate bands excised for extraction (I13504 for last lane)
Gel extraction of InterLab BioBricks
340uL of Binding Buffer;
one elution with 50uL for 20K, 22A, 22K; 30uL for 21J
Ligation of InterLab sequences
Insert: digested 21J
Plasmids: 20K, 22A, 22K
Component
Volume/µL
T4 Ligase
1
T4 Buffer
4
Insert
10
Plasmid
10
Milli-Q
26
NB: total reaction volume of 51µL is 1µL more than ideal
NB: total reaction volume of 51uL is 1uL more than ideal
Day 15
Group D
Miniprep plasmid extraction for 20K and 22A tubes.
Nanodrop concentration measurement. Tom noted that most of our concentrations are too low for sequencing (≤100ng/μl) and suggested that we elute using 35μl elution buffer that had been prewarmed in 55℃ water bath and then pass the filtrate through column and spin down again for better yields.
In the afternoon we performed the restriction digests of the plasmid and prepared the gel to be run on Monday for the digest products.
The gel has been left in a tank with buffer and a lid.
Week 4
Day 16
Lychee
Run gel for digested plasmids from 10/7. Check if bands are of correct size, if so, send sample for sequencing
Gel run no.1: Bands overrun into the bottom half of the gel - redo. But 21K[22A] and 21J[22K] seemed to show correct-sized bands → send for sequencing
Did gel extraction for 21J bands from gel but did not obtain a satisfactory concentration of DNA -> restriction digest to obtain 21J tomorrow
21J[22A] and 21J[22K(2)] sent for sequencing
Day 17
Group D
Restriction digest for 20A to get 21J. Religate 21J into plasmid.
Digest the remainder of 10/07 miniprep products and run on gel
Rest of bands are still not giving the right inserts, but 20K seems to give the right size → sent for sequencing on 15/07, and stored in freezer
Day 20
Ria and Lychee
Interlab plasmids 20K, 22A, 22K are ready for transformation.
All 20K, 22A, 22K plasmids used up in transformation into DH5alpha because there didn’t seem to be enough plasmids - need to miniprep a portion of the overnight cultures of these transformed cells to replenish plasmid stock.
Week 5
Day 21
Miniprepping of 19/07 overnight cultures
For interlabs - make frozen stock with 700µL of each culture (add 300µL glycerol to it, vortex, and put in -80degC); miniprep the rest of the cultures to replenish plasmid stocks.
Transformations
Interlab (22A, 22K, 20K) (after miniprep is done)→ MG1655, deltaFliC
Interlab positive and negative controls:
(+) - BBa_I20270 (Kit Plate 3, Well 8P)
(-) - BBa_R0040 (Kit Plate 2, Well 6F)
Resuspend in 10uL Milli-Q, then transform into DH5alpha
Plate Streaking
Frozen stocks of DH5alpha 20K, 22K, 22A also streaked
Day 23
Miniprep of DH5α (+) and (-)
Stocks made - 700µl LB + 300µl 60% glycerol
Nanodop:
[DNA] (ng/µl)
DH5α (+) 1
39.6
DH5α (+) 2
27.1
DH5α (-) 1
28.5
DH5α (-) 2
33.7
Transformation into FliC and MG1655 - left in incubator overnight
Frozen Stock Preparation
Interlab 20K, 22K, 22A in MG1655 and deltaFliC made into frozen stocks
Day 24
Overnights
Pick colonies and overnights for DH5α (+) and (-) in FliC and MG1655.
Day 25
Frozen Stocks
For (+), (-) in MG1655, deltaFliC
Overnight cultures in M9 Modified Media
(+), (-) DH5alpha, MG1655, deltaFliC from frozen stocks.
20K, 22K, 22A DH5alpha from plate
20K, 22K, 22A MG, FliC from frozen stock
All Chl.
Over the Weekend
Overnight cultures taken out onto the bench after being incubated at 37degC, 225 rpm orbital shaking for 20 hours.
100µL culture loaded into each well for GFP fluorescence and OD600 measurement.
Sodium fluorescein used as standard (absolute units) for fluorescence data to be compared against. 1.66g fluorescein (free acid) dissolved in 5mL pH8.0 Tris HCl to obtain 1M fluorescein solution, neutralized with 5mL 2M NaOH to obtain 10mL of 0.5M sodium fluorescein (NaFluo).
Serial dilution of NaFluo performed for construction of calibration curve.
Week 6
Day 27
Growth curves for InterLab
*note: 20K DH5alpha cultures failed to grow (overnight test tubes very clear)
*see protocol information in the relevant Excel files
Fluorescence Microscopy Trial Run
Single 50mL flask filled with 10mL M9 modified media, inoculated with 1mL overnight culture of 20K MG1655; grown for 2 hours up to OD600 = 0.2.
(note: growth in M9 modified media is very slow compared to that in LB broth, use higher concentrations in the future)
Day 28
Instantaneous Read
GFP fluorescence and OD600
Culturing Strains to Mid-Log
2mL of each overnight culture added to 10mL modified M9 media in 50mL conical flasks, incubated at 37degC, 225 rpm for ~3 hours.
Day 29
Cultivate cells for microscopy
Microscopy done for MG1655 22A, 20K, (22K was already in stat phase despite being OD600 = 0.2 -- cells were short, non-fluorescent); DH5alpha 22A, 20K, 22K
Week 7
Day 31
Microscopy
Dilutions of overnight cultures in 1:5 and preparing slides for microscopy.
Overnights for Interlab cultures were done in LB Chl, and upon 1:5 dilution in M9 stationary phase was achieved within 1.5 hours (validation via OD 0.7-0.9 and microscopy).
Day 32
June
1:5 dilution of interlabs in M9 media, and 45 min of incubation at 37C gives OD value of 0.4 to 0.5. DH5alpha strains showed reasonable fluorescence.
Day 33
June
Set up interlab for microscopy
Day 34
Set Up
Set up conical flask cultures of interlabs for microscopy and flow cytometry.
1:10 dilution in M9 media for 45 minutes gave OD values ranging from 0.27 to 0.35. Only MG1655 cultures were grown and analyzed due to limited number of flasks.
Day 35
Tasks
Tasks (no specific order; grab Leon for 4. and 5. at an appropriate time):
Make new bottle of M9 Modified Media without thiamine inside.
Wash (i.e. spin down 1mL of culture for 3 minutes at full speed, then decant, and resuspend in unmodified M9 salts solution) overnight interlab cultures that were grown in LB, and take instantaneous fluorescence reads and OD600. Repeat as many reads as you like (can reuse the same plate because the wells will be filled up with the same type of cells anyway, just make sure to fully empty them each time) until the 1mL of culture you washed is depleted (just so that we don’t waste cells and the availability of the machine).
Set up growth assay - aliquot M9 Modified Media into smaller bottle and add thiamine and antibiotics appropriately. Wash LB interlab cultures using this supplemented M9 media, and grow in plate reader for appropriate duration. We only have access to the FluorStar overnight today.
Grow up cells of your choice in conical flasks in LB (under 1 hour if 1:5 dilution; if longer growth time desired dilute in 1:20, 1:50, or even 1:100 as necessary). Stop incubation of these cells at OD600 ~ 0.4 - 0.7, then wash with unsupplemented M9 salts to halt growth. These cells can then be used for microscopy and flow cytometry.
Microscopy: remember to book the microscope under the “confocal” sheet outside the door. Also, after completing session, consult Chris for demonstration on how to clean up microscope objective lens.
Flow cytometry: Consult Chris to arrange a time when George is also available, and work out how to use the flow cytometer properly.
Microscopy image analysis: If no one else has a laptop with MATLAB installed, grab Leon to learn microscopy image processing from Chris. Before that, get Leon to copy all the image files we’ve had thus far out of the microscopy PC.
Week 5
Day 36
Tasks
Basically repeat as much of the Day 35 workflow as possible, briefly:
Wash overnight cultures in M9 media (unsupplemented) for single-read fluorescence and OD600
Wash overnight cultures in modified M9 media (+ thiamine supplement) for growth curve
Set up flasks of washed culture (appropriate dilution) in modified M9 media (+ thiamine supplement) for microscopy and flow cytometry.
Day 37
M9 salts haven't arrived, should start interlab from making M9 media. Same as usual, except please do catch GDubz for cytometry.
Wash overnight cultures in M9 media (unsupplemented) for single-read fluorescence and OD600
Wash overnight cultures in modified M9 media (+ thiamine supplement) for growth curve
Set up flasks of washed culture (appropriate dilution) in modified M9 media (+ thiamine supplement) for microscopy and flow cytometry.
Day 38
Focus on MGs
Flow cytometry - 1/25 cultures in at 9.20
Set up for growth curves (only have 2 sets)
Trouble shooting for microscopy
Ask Chris to go through using the microscope
Try a 1/10 dilution
Day 39
Grow falcon tubes - DH5alpha this time → for cytometry and microscopy (1:25 dilution for flask shown to be good for MG)
All cultures overnighted - can do full growth curve
Day 40
Grow falcon tubes - 15x interlab overnights 1:20 dilution into Chl LB, for cytometry and microscopy.
All cultures overnighted - can do full growth curve
Day 41
22 overnights to dilute 1/20 in LB + Chl and culture until mid-log OD ~0.4-0.7. Follow flow cyt protocol
Microscopy with mid-log phase cultures
Look at the InterLab page - Imperial 2014. Analyse the microscopy - Chris to show June or Lychee. Not bothering with the growth curves. Compile the flow cytometry data. Microplates - Leon
Day 42
Finish microscopy
Set up calibration curve with following concentrations of sodium fluorescein:
0.01µM
0.1µM
0.2µM
1µM
2µM
5µM
10µM
20µM
This should give a calibration curve that tallies perfectly with the range of interest for spectrophotometer settings.
Day 43
Analysis of microscopy images for 20K, 22K and 22A in different bacterial strains (DH5ɑ, MG and ΔfliC) using MicrobeTracker.
Prepared overnights for (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC to visualise under microscope tomorrow (20/8)
Day 44
(+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC were visualised on the microscope.
Overnights set up for (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC for observing tomorrow
Day 45
Visualised (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC
Used MicrobeTracker to identify cells in images and ran GFP mesh analysis on MicrobeTracker
Week 5
Day 47
Microscopy
Visualised 22AΔfliC on the microscope.
Histograms plotted for mean pixel intensity of cells across different strains and the different genes using MicrobeTracker.
Day 48
Mean pixel intensity per cell for each biological replicate calculated using MicrobeTracker. Mean and standard deviations calculated for the different biological replicates.
Flow cytometry data recorded its mean and standard deviation calculated.
Synbiota
Week 7
Day 33
PCRs
pLas RDP F and R - use B gBlocks (157bp) - 72*C
T4 Holin F and R - use D gBlock (672bp) - 72*C
ArtE F and R - use M gBlock (594bp) - 72*C
pLsr F and R - use D gBlock (90bp) - 72*C
LasR F and R - use B gBlock (717bp) - 72*C
PCR conditions: 10-40s per kb recommended. None of the sections are 1kb, and for ease of PCR, I’ve used a 20s extension time for all. If this doesn’t work, we can always try again, but I think shorter than 20s might be too quick… I’ve also used 30 cycles.
PCR products in freezer, ready for gel tomorrow morning. Gel has been poured and will be left in buffer overnight.
Day 34
Gel Photo
Run gel for K# → K* and Synbiota RDP PCR done yesterday
For LasR and pLas, the RDP parts can be PCR-ed from parts already in the iGEM distribution kit instead.
LasR - BBa_C0179: 2015 distro Kit Plate 3, Well 6M (pSB1C3)
pLas - BBa_R0079: 2015 distro Kit Plate 3, Well 5C (pSB1C3)
Transform these tomorrow.
Week 8
Day 37
Synbiota PCR - Art-E from M or T4 holin from D; synbiota pack is in the cold room - they supply their own ladder as well as buffer to dilute primers in. Phusion can be obtained from Chris, and we have our own Phusion HF buffer in the -20deg
Miniprep the overnight cultures for T4Endo, LasR, pLas
Day 38
Continue with Synbiota clean-up protocol (Art-E PCRed yesterday according to protocol and is in the freezer) (Mabel - now given to Leon to run gel)
Day 39
Gel Photo
Carry on with the Art-E synbiota gel in the cold room
No bands - George to repeat PCR
Clean up synbiota digests in freezer (these are plasmids that have been linearised for PCR). PCR according to synbiota protocol with other Synbiota PCRs - in synbiota box in freezer.
Day 40
Carry from on Synbiota PCR - PCR tubes in working shelf of the freezer
Week 9
Day 41
Email Synbiota with gel picture
Re-PCR pLas and pLsr
PCR Art-E from M* if sequence is good
Gel from Friday:
Lane 1 - not sure
Lane 2 - is pLsr - run again to make sure
Lane 3 - LasR (ignore the larger band)
Lane 4 - T4 holin (primers designed to only amplify holin)
Day 43
PCR all synbiota parts again. plsr and t4 holin worked, other two failed. Will keep trying PCR with plas and LasR.
Day 44
PCR plas and LasR again. gel -> still nothing. Ran another PCR using a different protocol, but still got nothing.
Week 10
Day 47
Gel order: Ladder, T4 Holin, pLsr, M*- #, M*- #
Deformed gel - agarose may have been mixed with water instead of TBE
Repeat PCR from very beginning… :( :( :(
Linearise template in 10ul digests
Heat inactivate for 30mins at 95degC before setting up PCR according to RDP protocol
Run on 2% agarose gel - 1g in 50ml, 120V for 30 mins (this is too short - can afford to run longer for better separation)
Gel order: Ladder, pLsr, T4 Holin, Art-E, LasR, pLas, M* - #, M* - #
Excised all bright bands except for Art-E and pLas - repeat PCR tomorrow
Day 48
Gel extraction/cleanup of synbiota and M* - # bands
Synbiota: eluted with 55degC 2 x 50ul TE buffer
M#: eluted with 55 degC 50ul EB buffer
Emailed Synbiota with gel picture and nanodrop values
Repeat PCR of Art-E and pLas, but digest template with 1ul of template regardless of [DNA] in 10ul digest
Template: T4 Endo* and pLas pSB (i.e. the other eppendorf of template to the ones used on 25/08 PCRs)
Also add DMSO to PCR set up, at 5% and 10%.
Day 49
Run gel of Art-E and pLas PCR from Day 48
N.B. 5% or 10% = % of DMSO in PCR set up
D#- *
D#- *
Art-E 5%
Art-E 5%
Art-E 10%
Art-E 10%
pLas 5%
pLas 5%
pLas 10%
pLas 10%
Left ladder: 50bp (from Chris, in synbiota box)
Right ladder: 1kb+ ladder
Farmost right well has loading dye (trying to identify an unnamed eppendorf on bench)
Art-E’s brightest bands are too small - it is ~220bp long when it should be 629bp long.  Potentially due to primers being ordered wrong and getting promoter?
Excised both D* bands (880bp, now in freezer next to M#)
Awaiting reply from Patrick for Synbiota
Week 11
Day 52
Patrick replied - reattempt with:
Double the reaction volume for those that worked (200ul rxn)
GC buffer for those that didn’t work
Art-E and pLas: use template from digests done on 26/08
pLsr, T4 Holin and LasR: linearise template in 10ul reactions as on 25/08, with the exception of LasR which needs [template] correction.
Art-E and pLas: GC buffer
pLsr, T4 Holin, LasR: 200ul reaction (will need to divide into 4 wells)
Gel: 2% agarose in TBE, 120V
LasR
LasR
LasR
LasR
T4 Holin
T4 Holin
T4 Holin
T4 Holin
pLsr
pLsr
pLsr
pLsr
pLas (GC)
pLas (GC)
Art-E (GC)
Art-E (GC)
Excised all T4 Holin and pLsr bands - in freezer
Day 53
Cleanup of Synbiota pLsr and T4Holin
Had to re-PCR(used x4 elution buffer and had nanodrop results of pLsr: 16 and T4Holin: 9)…… Sorry Mabel!!!
Waiting for the protocol of next step from Synbiota
Day 54
Proceed with Synbiota RDP Parts Creation Protocol - BsaI digestion Protocol.
BsaI Digestion (2.5hrs)
DNA cleanup
Measuring and adjusting the part concentration
Day 55
Repeat pLas, LasR and Art-E at 55 and 59 degC annealing temperatures
Digested templates from 25/08 (LasR) or 26/08 (Art-E and pLas)
Only do 50ul reaction - if it works then scale up
2% TBE agarose, 120V
LasR 55
pLas 55
ArtE 55
LasR 59
pLas 59
Art-E
LasR works!    YAAAAAAAAAAASSSSSSSSSSSSSSSSSSSSS
Scale up after the weekend at 59 degC
Week 12
Day 60
Synbiota RDP assembly of D*
Cloning
Week 1
Day 1
Preparation of Stock Solutions
Preparation of Reaction Solutions
Polymerase Chain Reaction
Gel Electrophoresis of PCR-Amplified gBlocks
Day 2
Gel Electrophoresis
Results
Extraction of DNA
Restriction Digest PCR
Restriction Enzyme Digest
DNA ‘Cleanup’
NanoDrop
Ligation
Preparation of E. coli
Re-PCR of DNA
Day 3
Re-PCR of DNA Sequences
Plasmid Recovery
Extraction of “sticky” pSB1C3
Repeat PCR with higher DNA concentration
Gel extraction of G and J
NanoDrop
Ligation
Transformation
Day 4
Growth and culture of bacteria
Transformation of G and J
Primer Design
Day 5
Plasmid Extraction
Plasmid Quantification
Plasmid Digest
Gel Electrophoresis of Digested Plasmids
Analysis
Week 2
Day 6
PCR
Gel Electrophpresis
Gel Extraction
Restriction Digest
Recovery of pSB-1C3 vector
EZNA gel extraction protocol on recovered pSB1c3 vector
Growth and culture of E. coli transformed J and G
Day 7
Yesterday
Miniprep
Nanodrop
Plasmid Digest
Stock Solutions
PCR Amplification with New Primers
Day 8
Gel Electrophoresis
Restriction Digest
Plasmid Design
Transformation
Day 9
Plasmid Digest
Enzymatic Reaction  Clean-up
Growth and Culture
Ligation
Preparation of DH5
Day 10
Plasmid for PCR Products
Preparation of DH5
Week 3
Day 11
Overnight Cultures
PCR
Restriction Digest
Primer Preparation
Sequencing
Running the Gel
Day 12
NanoDrop
Restriction Digest
Gel Extraction
Plasmid Extraction
Gel Electrophoresis
Analysis of Sequencing Data
Ligation
Day 13
To Do
To Note
QuickChange
Transformation
Restriction Digest
Gel Electrophroesis
Sequencing
Extra
Day 14
Group A
Group B
Group C
Sequencing Results
PCR To Redo
To Sequence
To Overnight
Day 15
Group A
J*4 QuikChange
Group B
Group C
Over the Weekend
Week 4
Day 16
Leon
Group A
Ria
Day 17
Plasmid Backbones
Dephosphorylation
Enzymatic Reaction Cleanup
Ligation
June
10/07 PCR Batch (D*, D#, M#, O#)
13/07 PCR Batch (E*, E* DMSO)
Preparation of Competent Cells
Sequencing
PCR of C#, D#, L#
PCR of A*, B*, I*, F*, K*, A#, B#, F#, I#
Day 18
Check Plates
Transform Ligated Products into DH5alpha
Ligation
Overnight Cultures
PCRs
Transform completed plasmids into FliC and MG1655 Competent Cells
Making up M9 modified media
Day 19
Mini-prep overnighted J*
Diagnostic restriction digest of J* + gel electrophoresis
Pick Colonies
PCR
Restriction Digestion and Ligation
Day 20
Leon
Raffy
Mabel and Duke
Ria and Lychee
Helen and Kyle
Over the Weekend
Week 5
Day 21
Post-digest cleanup → ligation
Sequencing of QC J*4
Sequencing Data Received
Gel electrophoresis confirmation of 17/07 digested miniprep products
Miniprepping of 19/07 overnight cultures
Transformations
Plate Streaking
Day 22
Sequencing Data Received
PCR Amplification to make J# from QuikChanged J*4[pSB1C3]
Transformations
Run Gel for miniprepped J*QC, F*, I*, R#, L#
Restriction digest and ligation of A*, B*, F*, I*, and K* directly from the G blocks
Duke and Lychee
PCR of QuikChanged J*4(1) with primers for pBAD restriction sites
Day 23
Frozen Stock Preparation
Tasks completed
Test Digest for D*IIe, E*1 compared with digest of completed BioBrick N*3
Day 24
Overnights
Transformation
Sequencing Data
Miniprep C#, D#, L#
Ligations
Day 25
Miniprep
Gel Electrophoresis of minipreps
Transform
Over the Weekend
Streaking Plates
Overnights
Miniprep
Week 6
Day 26
To Do
Digest J# QC, run gel of O# and J# digest
Day 27
Raffy
Day 28
Gel run for 28/07 PCR products, band excision, clean up
Digest + clean up
Ligation
Success
Day 29
Sequencing Data
Transformations
Plate out BioBrick (T4 Holin) agar stab
Day 30
Overnight Cultures
Transformations
To Do
Over the Weekend
Week 7
Day 31
Miniprep
Day 32
Sequencing Results
Day 33
George
Helen
Mabel and James
Day 34
Kyle
Helen
Day 35
To Do
i)
ii)
iii)
iv)
v)
vi)
vii)
viii)
Week 8
Day 36
To Do
Day 37
Day 38
Day 39
Day 40
Over the Weekend
Week 9
Day 41
Last Few Bits of Cloning
Day 42
Day 43
Day 44
Day 45
Week 10
Day 46
Constructs PCR
Day 47
Molecular Cloning of Constructs
Day 48
Kyle
Mabel
Day 49
Day 50
Week 11
Day 51
Molecular Cloning Day
To Do
Day 52
Mabel
June
Day 53
Day 54
Day 55
Digest of M#, G#, pBad HisB
Digests from Kyle
Molecular Cloning
Week 12
Day 56
Transformation of #s
Day 57
Day 58
Day 60
Over the weekend
Week 13
Day 61
Characterization
Plan
Overview of sequences in pBAD
Currently completed pBADs
Cell Growth
Week 5
Day 23
Toxicity Assay
Day 24
Toxicity Assay
Day 25
Toxicity Assay
Week 6
Day 26
Toxicity Assay
Day 27
Repeat of Yesterday
Day 29
Toxicity Assay
Week 8
Day 36
Toxicity Assay
Day 37
Day 38
Expression and Secretion
Week 6
Day 29
TCA Protein Precipitation
Day 30
Secretion Assay
Week 7
Day 32
Secretion Assay
Day 34
TCA-Precipitate
Week 8
Supernatant Acuisition
SDS-Page
Day 37
Assaying
Day 38
Day 39
Day 40
Week 9
Day 42
Set Up for TCA
Secretion
Day 43
Expression
Purification
Western Blot
Day 44
Expression
Day 45
Expression
Week 10
Day 46
Day 47
Duke (aka king of banter)
Day 49
Western Blot
Secretion
Day 50
Western Blot
Biofilms
Week 5
Day 25
Test Crystal Violet Staining of Biofilms
Week 6
Day 27
Biofilm Cultivation Using Existing Transformed Strains
Day 30
Biofilm Staining
Week 7
Day 31
Assays
Day 34
Set Up
Day 35
Biofilm Assaying
Week 8
Day 36
Biofilm Assaying
Day 37
Day 38
Biofilm viability assays
Biofilm degradation
Biofilm inhibition
Day 39
Day 40
Week 9
Day 42
Set Up
Week 11
Day 54
P. putida testing
Biofilm Viability Assay
Over the Weekend
Cell Killing
Week 8
Day 38
Day 39
Fluorescence
Week 6
Day 29
Growth Curve
Day 30
Assay
Week 8
Day 36
Quorum-Sensing Fluorescence
Week 10
Day 50
Quorum-Sensing Fluorescence
Interlab
Week 2
Day 8
Preparation of Interlab Study BioBricks
Transformation
Day 9
Growth and Culture of Bacteria
Day 10
Plasmid for PCR Product
Week 3
Day 11
Restriction Digest
Day 12
Gel Extraction
Day 15
Group D
Week 4
Day 16
Lychee
Day 17
Group D
Digest the remainder of 10/07 miniprep products and run on gel
Day 18
Sequence K1 and 20K
Check Sequencing Data
Day 20
Ria and Lychee
Week 5
Day 21
Miniprep of 19/07 overnight cultures
Transformation
Plate Streaking
Day 23
Miniprep of DH5α (+) and (-)
Frozen Stock Preparation
Day 24
Overnights
Day 25
Frozen Stocks
Overnight cultures in M9 Modified Media
Over the Weekend
Week 6
Day 27
Growth Curves
Fluorescence Microscopy Trial Run
Day 28
Instantaneous Read
Culturing Strains to Mid-Log
Day 29
Cultivate Cells for Microscopy
Week 7
Day 31
Microscopy
Day 32
June
Day 33
June
Day 34
Set Up
Day 35
Tasks
Week 8
Day 36
Tasks
Day 37
Day 38
Trouble Shooting for Microscopy
Day 39
Day 40
Week 9
Day 41
Day 42
Day 43
Day 44
Day 45
Week 10
Day 47
Microscopy
Day 48
Synbiota
Week 7
Day 33
Day 34
Week 8
Day 37
Day 38
Day 39
Day 40
Week 9
Day 41
Day 43
Day 44
Week 10
Day 47
Day 48
Day 49
Week 11
Day 51
Day 52
Day 53
Day 54
Week 12
Day 60
Made with love by Oxford iGEM
