<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_UTB skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/UTB</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/1/1a/Ox_urinarybiomepic_480.jpg)"><H3>Microbiome Study</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="section" id="Urinary_Tract_Biome"><DIV class="slim"><H2>Urinary Tract Biome</H2><P>
                        Until just five years ago it was believed that urine was sterile until it reached the urethra and lacking in any associated microbiota – the bladder was unhelpfully left out of the Human Microbiome Project in 2008. It was not until the first application of 16S r<A class="definition" title="RNA" data-content="RNA contains all the information of DNA code, but holds it in a different form so that amino acids can be transcribed from it.">RNA</A> sequencing to urine samples in 2010 that evidence of a “bacterial flora” was found in healthy, culture negative patients.
                    </P><P>
                        Bacteria constitute 90% of all cells in the human body (and 1-3% of total body mass) and this symbiotic relationship is crucial for maintaining health and for proper development of the host. In sites other than the urinary tract, it is now known that certain bacterial organisms perform functions that are useful for the human host. Even when a bacterial flora performs no actively helpful functions, it protects the host by occupying a niche in the body that could otherwise be colonized by harmful <A class="definition" title="pathogen" data-content="Any organism or virus that can cause disease.">pathogen</A>ic bacteria. For example: patients that have lost their gastrointestinal tract microbiome due to intensive chemotherapy experience debilitating side effects, but can be cured via a faecal transplant from a healthy patient to “re-seed” their microbiome.
                    </P><P>
                      Unfortunately much of the urinary tract microbiome is still a mystery. Certain dominant species - such as <EM>Lactobacillus</EM> and <A class="definition" title="E. coli" data-content="The fastest-growing strain of bacteria that we know everything about. Certain strains can make you ill, but you won’t find those ones in the lab."><EM>Escherichia coli</EM></A> - have been identified that correlate with positive and negative health outcomes respectively for patients, but how this happens and what role they play in the microbiome is still unknown. Further complicating this is the fact that the composition of the microbiome differs between males and females and changes with age; several age-specific genera have already been identified. Nevertheless, what effects our secreted products would have on this bacterial community is an important concern in our project that needs to be addressed.
                    </P><DIV class="quote quote-full"><P>“Perhaps the urinary microbiome, even if ephemeral, is itself protective and thus antibiotic treatments sometimes might cause more harm than good”</P><H3>Linda BrubakerAmerican Urological Society</H3></DIV><P>
                      Another potential concern for this application of our project is whether any of the <A class="definition" title="protein" data-content="An essential part of all living organisms. They are long and fold up into complicated structures and are made up of amino acids.">protein</A>s secreted by our bacteria could act as antigens and produce an unwanted immune response from the patient – i.e. are “immunogenic”. An immune response would lead to the production of antibodies that could inactivate the therapeutic effects of the treatment, or in rare cases produce adverse effects such as inflammation.
                    </P></DIV></DIV><DIV class="section" id="UTB_DNase"><DIV class="slim"><H2>DNase</H2><P>
                        The evidence for our DNase (<EM>Staphylococcus aureus</EM> Nuclease B) being immunogenic is clear-cut - it is even used by the National Cancer Institute in Maryland as an antigen to help <A class="definition" title="model" data-content="A simplified or idealised description of a system or process, usually mathematical, that can be used to predict how it will behave.">model</A> the genetic control of the immune response to it, which results in the inactivation of the <A class="definition" title="P. aeruginosa" data-content="Like E. coli, it is a well documented, fast-growing bacteria. It is more dangerous than E. coli and is harmful to humans, so we would need higher permissions to work with it in the lab."><EM>Pseudomonas aeruginosa</EM></A> with antibodies. There is no easy solution to this going forward - all DNases are immunogenic. There has been long running research into the use of DNases as therapeutics to treat cystic fibrosis and systemic lupus erythematosus (SLE), but the immune response of the patient has always caused severe side effects. Trials during the 60’s to treat SLE with bovine DNase showed that chronic treatment was completely impossible for this reason. More recently recombinant human DNase (rhDNase I, marketed as Pulmozyme/Dornase Alfa) has been developed to treat the symptoms of cystic fibrosis, but allergic reactions to the drug are still common. While rhDNase activity is impaired the least by the immune system compared to other DNases, it carries a risk of causing an autoimmune response to the endogenous form of the protein. The risk of this happening and the severity of the allergic response decreases the more distantly related the native species of the DNase used is, so <EM>Staphylococcus aureus</EM> Nuclease B may be one of the safer options, albeit less effective than other alternatives due to inactivation by antibodies.
                    </P><P>
                        The effects of our secreted DNase on the bacterial flora of the urinary tract will largely depend on the specificity of our other secreted products. The destruction of extracellular <A class="definition" title="DNA" data-content="Contains the genetic code for life, using only four types of molecules. It forms the familiar double helix structure and can self-replicate under certain conditions.">DNA</A> of <EM>Pseudomonas aeruginosa</EM> and <EM>Streptococcus pneumoniae</EM> can change the properties of <A class="definition" title="biofilm" data-content="A community of bacteria (or other microorganisms) adhering to a surface and each other, held together by secreted slime-like polymers that create a more favorable environment for the bacteria and protect them from environmental stresses and attack from a host’s immune system (in the case of pathogenic bacteria).">biofilm</A>s formed by these bacteria, but it is not known whether the same would be true for other bacterial species. These changes to the biofilm would permit the increased penetration of our secreted antimicrobials, so the consequences for any given species in the microbiome will depend on its sensitivity to our other secreted products (assuming that the properties of their biofilms are indeed altered). On a wider micro-ecological scale it is possible that the continual use of DNase on the biome over an extended period of time would cause a gradual decline in genetic diversity of the commensal (friendly) bacteria (and thus a decline in the health of the microbiome) due to the chronic destruction of extracellular DNA – but this is purely conjectural, and should not matter on the timescale of our treatment plan.
                    </P></DIV></DIV><DIV class="section" id="UTB_DisB"><DIV class="slim"><H2>Dispersin B</H2><P>
                      Dispersin is highly immunogenic in vivo. The name “dispersin” comes from its native function in the pathogen <EM>Aggregatibacter actinomycetemcomitans</EM>, where it is used to break up sections of its own biofilm so that small pieces can disperse and colonize new sites in the gastrointestinal tract. As a <A class="definition" title="virulence factor" data-content="A chemical or molecule secreted by pathogens, which contributes to their pathogenicity; examples include secreted toxins, and chemicals that allow the bacteria to colonize a particular site in a host via attachment.">virulence factor</A>, it is no surprise that the human immune system will react to it in some way – potentially by trying to inactivate it with antibodies.
                    </P><DIV class="quote quote-full"><P>“…the use of dispersin B or other <A class="definition" title="enzyme" data-content="A molecule which speeds up a chemical reaction - a biological catalyst. The reaction does not involve these molecules.">enzyme</A>s that degrade extracellular polymeric substances (EPS) may be limited to industrial applications owing to the immunogenic properties of such biomolecules.”</P><H3>Should we stay or should we go: mechanisms and ecological consequences for biofilm dispersal: McDougald et al, Nature Reviews Microbiology 10, 39-50 (January 2012)</H3></DIV><P>
                        Assuming that our dispersin B is active in the urinary tract however, it is impossible to tell what exactly the consequences will be for the urinary tract microbiome. Dispersin B works by hydrolysing linear polymers of N-acetyl-D-glucosamines found in the biofilm matrices, which works for the specific pathogenic biofilms we are interesting in treating.
                    </P><P>
                      All bacterial species in the urinary tract live in biofilms (otherwise they would be washed out), but whether their biofilms are broken up would depend on whether the linkage between the N-acetyl-D-glucosamine monomers was a linear 1,-6 linkage or not. Since a variety of other nonlinear linkage arrangements exist we can probably assume that only a fraction of the urinary tract microbiome will be affected.
                    </P><P>
                        However this should be taken with a pinch of salt, since as of yet there is no species/genus specific data on what type of linkage is present, thus making it impossible to tell whether any given species (such as <EM>Lactobacillus</EM>) might be affected.
                    </P></DIV></DIV><DIV class="section" id="UTB_Art"><DIV class="slim"><H2>Artilysin</H2><P>
                        While there have been concerns over the potential immunogenicity of lysins, so far this has not been found to be a major problem by researchers looking to use them as novel antimicrobials for use in treatment. While they can provoke an antibody response, the activity of the lysins is unaffected by the binding of lysin-specific antibodies.
                    </P><DIV class="quote quote-full"><P>“...several unique characteristics of lysins makes them attractive enzybiotics over small molecule antibiotics. These include (i) their specificity for the pathogens without disturbing the normal microflora, (ii) the low chance of developing bacterial resistance, and (iii) their ability to kill colonizing pathogens on mucosal surfaces.”</P><H3>Engineered bacteriophage lysins as novel anti-infectives, Yang et al. Front Microbiol. 2014, 5: 542</H3></DIV><P>
                        Lysins should not pose any threat to the urinary microbiome. In nature lysins are highly specific, often restricted in their activity to a particular bacterial genus. They also have a modular structure that makes them particularly amenable to bioengineering, possessing a cell-binding domain (which controls the specificity) and a catalytic domain (which controls the activity). These can be independently substituted and altered to achieve desired characteristics in an artificial lysin.
                    </P><P>
                        Even though our particular <A class="definition" title="artilysin" data-content="An artifically modified version of naturally occuring proteins called lysins that attack bacteria by cleaving their cell wall">artilysin</A>, Art-175, is far less specific than we initially thought and targets a large range of gram-negative bacteria, engineering a functional replacement is definitely feasible with current bioengineering technology. Such a replacement would be specific to the pathogenic genus <EM>Pseudomonas</EM> but have no effect whatsoever on the remainder of the microbiome.
                    </P><P>
                        If we kept with Art-175, then its specificity would only extend to gram-negative species – which leaves more than half of the microbiome unaffected, including <EM>Lactobacillus</EM>, which appears to be the single most important known strain in the female urinary microbiome.
                    </P></DIV></DIV><DIV class="section" id="UTB_Holin"><DIV class="slim"><H2>T4 Holin</H2><P>
                      Very little information exists on T4 holin, but we can speculate that it most likely is not immunogenic - or at least not immunogenic at the low levels it might naturally occur in the urinary biome as a result of the activity of T4 phages (viruses which solely infect bacteria). Our T4 holin is not being secreted. Since it will only ever be released when our cell lyse, it is likely that no more T4 holin will be produced than the normal and minute background level of any bacterial biome. In addition, since T4 holin acts from the inside of cells and not from the outside, any liberated T4 holin will have precisely zero effect on the bacteria of the urinary microbiome.
                    </P></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#Urinary_Tract_Biome">Urinary Tract Biome</A></LI><LI><A href="#UTB_DNase">DNase</A></LI><LI><A href="#UTB_DisB">Dispersin B</A></LI><LI><A href="#UTB_Art">Artilysin</A></LI><LI><A href="#UTB_Holin">T4 Holin</A></LI></UL></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>