<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Test_Modeling2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Test/Modeling2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading"><H3>Modeling</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV class="section" id="introduction"><DIV class="slim"><H2>Introduction</H2><P>
                            Modeling allows us to observe what happens to a system when you change the parameters that describe it. By quantifying the systemâ€™s response to these variables, we can aid the wet lab team in their design of their biological system. By repeating the modeling, data-fitting and refining schedule we approach a sufficient description of our required design.
                        </P><P>
                            Our modeling is made up of three pieces: gene expression, diffusion and biofilm modeling. By modeling gene expression, we can inform the lab team which parameters they can choose freely - and which ones they must pick carefully - to maximise the output of our product. The gene copy number is one example of these parameters. By modeling diffusion, we can incorporate into our previous model the effect of our protein being separated from the biofilm it is trying to interact with. If this is a large effect, the lab team must change the design of their system. Finally, by modeling the formation of a biofilm and its interaction with our product, we can decide upon the required concentrations of product we need to clear the urinary tract infection. In all cases we use both deterministic and stochastic models to investigate our system.
                        </P></DIV></DIV><DIV class="section" id="gene-expression"><DIV class="slim"><H2>Gene Expression</H2><DIV id="gene-expression-deterministic"><H3>Deterministic</H3><P>
                                A deterministic model evolves according to ordinary or partial differential equations. We model the following system:
                            </P>
                                \[\overset{\alpha_{1}}{\rightarrow}Arab\quad\overset{\alpha_{5}}{\rightarrow}AraC\]

                                \[Arab+AraC\mathrel{\mathop{\rightleftharpoons}^{\mathrm{\alpha_{2}}}_{\mathrm{\alpha_{3}}}}Arab:AraC\]

                                \[\overset{K}{\rightarrow}mRNA\overset{\alpha_{4}}{\rightarrow}P\]

                                \[Arab\overset{\gamma_{1}}{\rightarrow}\phi\quad AraC\overset{\gamma_{2}}{\rightarrow}\phi\quad mRNA\overset{\gamma_{3}}{\rightarrow}\phi\quad P\overset{\gamma_{4}}{\rightarrow}\phi\]
                            <P>
                                Using the following deterministic equations [<A href="#Ref4">4</A>]:
                            </P>
                                \[\dfrac{d\left[Arab\right]}{dt}=\alpha_{1}+\alpha_{2}\left[Arab\right]\left[AraC\right]-\alpha_{3}\left[Arab:AraC\right]-\gamma_{1}\left[Arab\right]\]

                                \[\dfrac{d\left[Arab:AraC\right]}{dt}=\alpha_{3}\left[Arab:AraC\right]-\alpha_{2}\left[Arab\right]\left[AraC\right]\]

                                \[\dfrac{d\left[AraC\right]}{dt}=\alpha_{5}+\alpha_{2}\left[Arab\right]\left[AraC\right]-\alpha_{3}\left[Arab:AraC\right]-\gamma_{2}\left[AraC\right]\]

                                \[\dfrac{d[mRNA]}{dt}=K_{max}\dfrac{[Arab:AraC]^{n}}{K_{half}^{n}+[Arab:AraC]^{n}}-\gamma_{3}[mRNA]\]

                                \[\dfrac{d\left[P\right]}{dt}=\alpha_{4}\left[mRNA\right]-\gamma_{4}\left[P\right]\]
                            <P>
                                Where we define the symbols
                                <TABLE border="1"><TBODY><TR><TH>Symbol</TH><TH>Definition</TH><TH>Initial Value/Literature Value</TH><TH>Fitted</TH></TR><TR><TD>\([Arab]\)</TD><TD>The concentration of Arabinose</TD><TD>\(1\times10^{-4}M\)</TD><TD>-</TD></TR><TR><TD>\([AraC]\)</TD><TD>The concentration of AraC</TD><TD>\(1\times10^{-5}M\)</TD><TD>-</TD></TR><TR><TD>\([Arab:AraC]\)</TD><TD>The concentration of associated Arabinose and AraC</TD><TD>\(0\)</TD><TD>-</TD></TR><TR><TD>\([mRNA]\)</TD><TD>The concentration of mRNA</TD><TD>\(0\)</TD><TD>-</TD></TR><TR><TD>\([P]\)</TD><TD>The concentration of our product</TD><TD>\(0\)</TD><TD>-</TD></TR><TR><TD>\(\alpha_{1}\)</TD><TD>Basal production of Arabinose</TD><TD>???</TD><TD>?</TD></TR><TR><TD>\(\alpha_{2}\)</TD><TD>Association constant</TD><TD>\(2.8\times10^{7}s^{-1}\) [<A href="#Ref7">7</A>]</TD><TD>?</TD></TR><TR><TD>\(\alpha_{3}\)</TD><TD>Dissociation constant</TD><TD>\(0.022s^{-1}\) [<A href="#Ref7">7</A>]</TD><TD>?</TD></TR><TR><TD>\(\alpha_{4}\)</TD><TD>Translation rate</TD><TD>\(15ntd\: s^{-1}\)/length of sequence [<A href="#Ref6">6</A>]</TD><TD>?</TD></TR><TR><TD>\(\alpha_{5}\)</TD><TD>Basal production of AraC</TD><TD>???</TD><TD>?</TD></TR><TR><TD>\(\gamma_{1}\)</TD><TD> Degradation rate of Arabinose </TD><TD>\(5.13\times10^{-4}s^{-1}\) [<A href="#Ref5">5</A>]</TD><TD>?</TD></TR><TR><TD>\(\gamma_{2}\)</TD><TD>Degradation rate of AraC</TD><TD>\(5.13\times10^{-4}s^{-1}\) [<A href="#Ref5">5</A>]</TD><TD>?</TD></TR><TR><TD>\(\gamma_{3}\)</TD><TD>Degradation rate of mRNA</TD><TD>\(5.13\times10^{-4}s^{-1}\) [<A href="#Ref5">5</A>]</TD><TD>?</TD></TR><TR><TD>\(\gamma_{4}\)</TD><TD>Degradation rate of product</TD><TD>\(5.13\times10^{-4}s^{-1}\) [<A href="#Ref5">5</A>]</TD><TD>?</TD></TR><TR><TD>\(K_{max}\)</TD><TD>Maximal transcription rate</TD><TD>\(50ntd\: s^{-1}\)/length of sequence [<A href="#Ref6">6</A>]</TD><TD>?</TD></TR><TR><TD>\(K_{half}\)</TD><TD>Half-maximal transcription rate</TD><TD>\(160\mu M\) [<A href="#Ref8">8</A>]</TD><TD>?</TD></TR><TR><TD>\(n\)</TD><TD>Hill coefficient</TD><TD>\(2.65\) [<A href="#Ref3">3</A>]</TD><TD>?</TD></TR></TBODY></TABLE></P><P>
                                To evaluate these parameters, we will fit the data created by the wet lab team. We allow a tolerance of a factor of ten either side of our initial guess for each parameter. The fitted parameters will be given in the fourth column and will be found using the MATLAB function fmincon. To solve the equations numerically we use the MATLAB equation solver ode15s. We plot three solutions for various parameters below.
                            </P></DIV><DIV id="gene-expression-stochastic"><H3>Stochastic</H3><P>
                                A stochastic model is anything which includes some random element, and is useful for modeling small systems or those with some uncertainty. We produced a video tutorial on how to model gene expression networks stochastically using Gillespie's algorithm.[<A href="#Ref2">2</A>, <A href="#Ref4">4</A>].
                            </P><DIV class="algorithm"><H3>Gillespie's Algorithm</H3><OL><LI>Find random numbers \(r_{1}\) and \(r_{2}\) which are uniformly distributed between 0 and 1.</LI><LI>Find propensities \(\alpha_{i}\) of \(N\) possible reactions. Find \(\alpha_{0}=\sum\limits_{i=1}^N \alpha_{i}\).</LI><LI>Compute time \(\tau=\dfrac{1}{\alpha_{0}}\ln\left(\dfrac{1}{r_{1}}\right)\).</LI><LI>Find the reaction \(j\) such that \(\dfrac{1}{\alpha_{0}}\sum\limits_{i=1}^{j-1} \alpha_{i}\leq r_{2}&lt;\dfrac{1}{\alpha_{0}}\sum\limits_{i=1}^{j} \alpha_{i}\).</LI><LI>Implement reaction \(j\) and increment time \(t\) to \(t+\tau\). </LI><LI>Repeat steps 1-5 until a sufficient time or computation budget has been exhausted.</LI></OL></DIV><P>
                                Turning the reaction rates in our model into propensities is simple - for example, the propensity for the association reaction
                                \(Arab+AraC\rightarrow Arab:AraC\) is \(\alpha_{3}\left[Arab\right]\left[AraC\right]\).
                            </P></DIV></DIV></DIV><DIV class="section" id="diffusion"><DIV class="slim"><H2>Diffusion</H2></DIV><DIV id="diffusion-deterministic"><DIV class="slim"><H3>Deterministic</H3><P>
                                The standard from for the diffusion equation,
                            </P>
                                \[\dfrac{d[P]}{dt}=D\dfrac{d^{2}[P]}{dx^{2}}\]
                            <P>
                                where \([P]\) is the concentration of our product, \(t\) is time and \(x\) is distance and \(D\) is the diffusion constant, can be solved analytically in any number of dimensions:
                            </P>
                                \[[P]\left(x,t\right)=\dfrac{1}{\left(4\pi Dt\right)^{\frac{N}{2}}}\exp\left(\dfrac{-\sum\limits_{i=1}^N x_{i}^{2}}{4Dt}\right)\]
                            <P>
                                where \(x_{i}\) is a generalised position co-ordinate and \(N\) is the dimensionality of the system. This solution only applies to a source of [P] at the position \(x=0\). We can also solve for a grid of 1 or 2D points using finite difference models given in the <A href="#modeling-tutorial">tutorial section</A>.
                            </P></DIV></DIV><DIV id="diffusion-stochastic"><DIV class="slim"><H3>Stochastic</H3><P>
                                We have the option of using three stochastic models for diffusion - random walk, random velocity and compartmental Gillespie.
                            </P><H4>Random Walk</H4><P>
                                We initially take a collection of particles and, for each one, compute the number
                            </P>
                                \[dr=\sqrt{2Ddt}\xi_{r}\]
                            <P>
                                where \(dt\) is the size of our timestep, \(dr\) is the absolute distance moved in the timestep and \(\xi_{r}\) is a normally distributed random number with mean 0 and variance 1. To calculate the distance that particles have actually moved depends on what dimensionality we are operating in. We work in polar or spherical poar co-ordinates in 2D and 3D respectively, and so sample angles \(\phi\) and \(\theta\) and uniformly distributed in the ranges [0,\(2\pi\)] and [0,\(\pi\)].
                            </P><H4>Random Velocity</H4><P>
                                We initially take a collection of particles at position \(\mathbf{r_{0}}\). We give them random velocities uniformly distributed in the range \(\left[-2\mathbf{v},2\mathbf{v}\right]\) (where \(v\) is determined by thermal fluctuations) and decide that the direction of these velocities should change direction (or not) with some probability after every timestep. We summarise this process in the algorithm below.
                                <OL class="algorithm"><LI>For each particle, generate a random speed \(s\) in a random direction.</LI><LI>Compute the position of the particle after a timestep \(\Delta t\).</LI><LI>Generate a random number \(r\) uniformly distributed between 0 and 1.</LI><LI>If \(r&lt;\dfrac{s^{2}}{2D}\Delta t\), calculate a new random direction of velocity for the particle.</LI><LI>Repeat steps 2-5 for each particle until a certain time or computational budget is exhausted.</LI></OL></P><H4>Compartmental diffusion; Gillespie</H4><P>
                                We can divide our region into a system of compartments
                            </P>
                                \[A_{1} \mathrel{\mathop{\rightleftharpoons}^{\mathrm{k}}_{\mathrm{k}}} A_{2} \mathrel{\mathop{\rightleftharpoons}^{\mathrm{k}}_{\mathrm{k}}} ... \mathrel{\mathop{\rightleftharpoons}^{\mathrm{k}}_{\mathrm{k}}} A_{N-1} \mathrel{\mathop{\rightleftharpoons}^{\mathrm{k}}_{\mathrm{k}}} A_{N}\]
                            <P>
                                such that the only difference in their propensities is related to the number of particles contained in each compartment. This will produce a correct model of diffusion if the proportionality constant \(k\) is chosen as
                            </P>
                                \[k=\dfrac{D}{(dx)^{2}}\]
                            <P>
                                where \(dx\) is the physical size of each compartment [<A href="#Ref2">2</A>]. We now run the Gillespie algorithm but with \(2(N-1)\) reactions to consider.
                            </P><H4>Comparison of methods</H4><P>
                                All of the methods provide similar levels of accuracy (depending on the size of the compartments used in the Gillespie model) and so we choose based on the simplicity of implementation. The random walk method is generally the fastest and easiest to code, so we choose this one.
                            </P><H4>The challenge</H4><P>
                                Since we are modelling diffusion in the catheter, the topology of the system is a ring, neglecting edge effects of the cylinder our bacteria will be placed in. To do this we use a random walk model in two dimensions. We confine our particles to a ring of a fixed radius and initially coat the ring with particles.
                            </P></DIV></DIV></DIV><DIV class="section" id="modeling-tutorial"><DIV class="slim"><H2>Modeling Tutorial</H2></DIV><DIV id="modeling-tutorial-deterministic"><DIV class="slim"><H3>Deterministic</H3><P>
                                Our goal is to model a system of reactions such as the reversible reaction
                            </P>
                                \[A \mathrel{\mathop{\rightleftharpoons}^{\mathrm{k_{+}}}_{\mathrm{k_{-}}}} B\]
                            <P>
                                Where \(A\) and \(B\) are our two species and \(k_{+}\) and \(k_{-}\) will be determined shortly. To do this, we make the intuitive assumption that the rate at which something reacts is proportional to how much of it we've got. That is, the rate of change of \(B\) with time is
                            </P>
                                \[\dfrac{dB}{dt}=k_{+}A\]
                            <P>
                                where \(k_{+}\) is simply a proportionality constant. This assumption is called the law of mass action. This of course is not the complete picture, as we forgot to include the fact that \(B\) can react back to form \(A\). So the complete system of equations to solve is
                            </P>
                            \[\dfrac{dA}{dt}=k_{-}B-k_{+}A\]
                            \[\dfrac{dB}{dt}=k_{+}A-k_{-}B\]
                            <P>
                                At this point, you can type these into your favourite computing program and solve them.
                            </P></DIV></DIV><DIV id="modeling-tutorial-stochastic"><DIV class="slim"><H3>Stochastic</H3><P>
                                The simplest way to add an element of 'randomness' into an equation is to add a random term into the differential equation. This turns it into a stochastic differential equation, such as:
                            </P>
                                \[\dfrac{dx}{dt}=c+r\]
                            <P>
                                which determines the evolution of a particle travelling at a speed \(c\), with a random number \(r\) added into the mix just for fun. We choose \(r\) to be normally distributed with a mean value of 0 and a variance of, say, 1. We evaluate the position of the particle at multiple timesteps (with some small time \(dt\) separating them) and plot our solution in the Figure below.
                            </P><P>
                                We can do better than this when dealing with systems such as our example in the previous section. We use the Gillespie algorithm to find:
                            </P><OL><LI>The <STRONG>time</STRONG> taken between reactions</LI><LI>The <STRONG>reaction</STRONG> that took place in that time.</LI></OL><P>
                                as neither of these could be determined by our initial method [<A href="#Ref2">2</A>, <A href="#Ref4">4</A>].
                            </P><P><STRONG>1)</STRONG> To find the time taken, we make an assumption about the form of the probability distribution that the time between reactions holds. It makes sense to think of it as a falling exponential, such that reactions more often happen quickly rather than slowly, and that the more probable a reaction is, the shorter the time between subsequent reactions. This naturally leads to the form
                            </P>
                                \[P(t)=e^{-\alpha_{0}t}\]
                            <P>
                                where \(P(t)\) is the probability of any reaction occuring in time \(t\), and \(\alpha_{0}\) tells us something about the likelihood of the reaction. We plot this in the Figure. In fact we call \(\alpha_{0}\) the sum of the propensities \(\alpha_{i}\) (where the dummy variable \(i\) runs from 1 to the number of possible reactions, \(N\), that could occur). In the case of our example system, the propensity for the reaction \(A\) to \(B\) would be \(k_{+}A\) and the propensity for the reaction \(B\) to \(A\) is \(k_{-}B\) so in this case \(\alpha_{0}=k_{+}A+k_{-}B\).
                            </P><P>
                                To decide the time it took for a reaction to occur, we sample a random number \(r_{1}\) in the range [0,1] and stick that number on the y axis of the graph. We read off from our graph a corresponding time and we're done - we now know when the next reaction took place.
                            </P><P><STRONG>2)</STRONG> To decide which reaction took place, we need the probabilities of each reaction occuring being proportional to the ratio of their propensities. We split up the domain of 0 to 1 into the normalised ratios of the propensities of our problem, recalling the propensities we calculated. We could have, in this case, selected the backwards reaction B to A. We can generalise this to the form given in the Gillespie Algorithm above.
                            </P><DIV class="image image-full"><P>This is a caption. Let's see what it looks like.</P></DIV></DIV></DIV><DIV id="modeling-tutorial-finite-difference"><DIV class="slim"><H3>Finite Difference Models</H3><P>
                                For finite difference models, we set up a grid of positions that we want to solve our equations at, separated by some fixed difference \(dx\) and at time steps separated by time \(dt\). Subscripts of our concentration P represent the spacial co-ordinates of the grid-point, while superscript represents the location in time. We define a constant \(r=\dfrac{dt}{(dx)^{2}}\) which must be less than 0.5 for the scheme to be convergent.
                            </P><P>
                                The 1D formula to iterate over for the finite difference model of diffusion is
                            </P>
                                \[P_{j}^{n+1}=(1-2r)P_{j}^{n}+r(P_{j-1}^{n}+P_{j+1}^{n})\]
                            <P>
                                The 2D formula to iterate over is
                            </P>
                                \[P_{i,j}^{n+1}=(1-4r)P_{i,j}^{n}+r(P_{i,j-1}^{n}+P_{i,j+1}^{n}+P_{i-1,j}^{n}+P_{i+1,j}^{n})\]
                            <P>
                                Similarly, for 3D:
                            </P>
                                \[P_{i,j,k}^{n+1}=(1-6r)P_{i,j,k}^{n}+r(P_{i,j-1,k}^{n}+P_{i,j+1,k}^{n}+P_{i-1,j,k}^{n}+P_{i+1,j,k}^{n}+P_{i,j,k-1}^{n}+P_{i,j,k+1}^{n})\]
                            <P>
                                we (arbitrarily) choose our boundary conditions to be continuous at the boundary.
                            </P></DIV></DIV></DIV><DIV class="section" id="references"><H2>References</H2><OL class="references"><LI>Synthetic Biology - A Primer, Paul S. Freemont, Richard I Kenne et al., 2012 Imperial College Press</LI><LI>R. Erban, S. J. Chapman, and P. Maini. A practical guide to stochastic simulations of reaction- diffusion processes, 2007. Available <A href="http://arxiv.org/abs/0704.1908">here</A></LI><LI>Salto R, Delgado A, MichÃ¡n C, MarquÃ©s S, Ramos JL. Modulation of the function of the signal receptor domain of XylR, a member of a family of prokaryotic enhancer-like positive regulators. J Bacteriol. 1998 Feb180(3):600-4. p.601 right column</LI><LI>Brian Ingalls,Mathematical Modeling in Systems Biology: An Introduction, 2012 University of Waterloo</LI><LI>Liang ST, Ehrenberg M, Dennis P, Bremer H. Decay of rplN and lacZ mRNA in Escherichia coli. J Mol Biol. 1999 May 14 288(4):521-38. p.524 right column bottom paragraph</LI><LI>Proshkin S, Rahmouni AR, Mironov A, Nudler E. Cooperation between translating ribosomes and RNA polymerase in transcription elongation. Science. 2010 Apr 23 328(5977):504-8. p.505 table 1</LI><LI>Nelson HC, Sauer RT. Lambda repressor mutations that increase the affinity and specificity of operator binding.Cell. 1985 Sep42(2):549-58. p.552 table 3</LI><LI>Sourjik V, Berg HC. Functional interactions between receptors in bacterial chemotaxis. Nature. 2004 Mar 25 428(6981):437-41.p.439 left column top paragraph</LI></OL></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked affix-top sm-hidden xs-hidden" data-spy="affix"><LI><A href="#introduction">Introduction</A></LI><LI><A href="#gene-expression">Gene Expression</A></LI><LI><A href="#gene-expression-deterministic">Deterministic</A></LI><LI><A href="#gene-expression-stochastic">Stochastic</A></LI></UL><LI><A href="#diffusion">Diffusion</A></LI><LI><A href="#diffusion-deterministic">Deterministic</A></LI><LI><A href="#diffusion-stochastic">Stochastic</A></LI><LI><A href="#modeling-tutorial">Modeling Tutorial</A></LI><LI><A href="#modeling-tutorial-deterministic">Deterministic</A></LI><LI><A href="#modeling-tutorial-stochastic">Stochastic</A></LI><LI><A href="#modeling-tutorial-finite-difference">Finite Difference Models</A></LI><LI><A href="#references">References</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>