<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Test_Notebook1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Test/Notebook1</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading"><H3>Notebook</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV id="notebook-key-button" data-spy="affix"><H3>Key</H3></DIV><DIV id="notebook-key"><DIV class="col-md-6"><H2>*</H2><P><STRONG>A</STRONG>  - pSB-1C3 LasR Holin</P><P><STRONG>B</STRONG>  - pSB-1C3 LasR sfGFP</P><P><STRONG>C</STRONG>  - pSB-1C3 Lsr sfGFP</P><P><STRONG>D</STRONG>  - pSB-1C3 Lsr Holin</P><P><STRONG>E</STRONG>  - pSB-1C3 DNase DsbA</P><P><STRONG>F</STRONG>  - pSB-1C3 DspB YebF</P><P><STRONG>G</STRONG>  - pSB-1C3 DspB</P><P><STRONG>H</STRONG>  - pSB-1C3 MccS</P><P><STRONG>I</STRONG>  - pSB-1C3 DspB Fla</P><P><STRONG>J</STRONG>  - pSB-1C3 DspB DsbA</P><P><STRONG>K</STRONG>  - pSB-1C3 Art-175 DsbA</P><P><STRONG>L</STRONG>  - pSB-1C3 Art-175 YebF</P><P><STRONG>M</STRONG>  - pSB-1C3 Art-E</P><P><STRONG>N</STRONG>  - pSB-1C3 Art-175 Fla</P></DIV><DIV class="col-md-6"><H2>#</H2><P><STRONG>A</STRONG>  - pBAD 33 LasR Holin </P><P><STRONG>B</STRONG>  - pBAD 33 LasR sfGFP</P><P><STRONG>C</STRONG>  - pBAD 33 Lsr sfGFP</P><P><STRONG>D</STRONG>  - pBAD 33 Lsr Holin</P><P><STRONG>E</STRONG>  - pBAD HisB DNase DsbA</P><P><STRONG>F</STRONG>  - pBAD HisB QC DspB YebF</P><P><STRONG>G</STRONG>  - pBAD HisB QC DspB</P><P><STRONG>H</STRONG>  - pBAD HisB MccS</P><P><STRONG>I</STRONG>  - pBAD HisB QC DspB Fla</P><P><STRONG>J</STRONG>  - pBAD HisB QC DspB DsbA</P><P><STRONG>K</STRONG>  - pBAD HisB Art-175 DsbA</P><P><STRONG>L</STRONG>  - pBAD HisB Art-175 YebF</P><P><STRONG>M</STRONG>  - pBAD HisB Art-E</P><P><STRONG>N</STRONG>  - pBAD HisB Art-175 Fla</P><P><STRONG>O</STRONG>  - pBAD HisB Art-175</P><P><STRONG>P</STRONG>  - pBAD HisB DNase</P></DIV></DIV><DIV class="slim"><DIV class="section" id="cloning"><H2>Cloning</H2><DIV class="section" id="1"><H5>Week 1</H5><DIV id="1-1"><H3>Day 1</H3><DIV id="1-1-prep-stock"><H4>Preparation of Stock Solutions</H4><P><STONG>1. gBlocks</STONG></P><P>
                                            The gBlocks ordered from IDT arrived in the form of vials of 200µg solid DNA powder.
                                        </P><P><A href="#">(refer to BioBricks page for information on DNA sequences)</A></P><P>
                                                The gBlocks were made into 10ng/µl stock solutions in Milli-Q water for storage:
                                        </P><TABLE border="1"><TBODY><TR><TH>mass/ng</TH><TH>conc/ngµl<SUP>-1</SUP></TH><TH>final volumeµl</TH></TR><TR><TD>200</TD><TD>10</TD><TD>20</TD></TR></TBODY></TABLE><P><STONG>2. Primers</STONG></P><P>
                                            The forward and reverse primers ordered from IDT came in 32.4nmol and 34.3nmol of solid respectively.
                                        </P><UL><H6>Sequences</H6><LI>Forward - CTTTTTTGCCGGACTGC</LI><LI>Reverse - ATGATTTCTGGAATTCGC</LI></UL><P>
                                        The primers were made into 100µM stock solutions in Milli-Q water for storage:
                                    </P><TABLE border="1"><TBODY><TR><TH>amt/10<SUP>-9</SUP>mol</TH><TH>conc/10<SUP>-6</SUP>M</TH><TH>final volume/10<SUP>-6</SUP>L</TH></TR><TR><TD>32.4</TD><TD>100</TD><TD>324</TD></TR><TR><TD>34.3</TD><TD>100</TD><TD>343</TD></TR></TBODY></TABLE></DIV><DIV id="1-1-prep-reaction"><H4>Preparation of Reaction Solutions</H4><P><STONG>1. gBlocks</STONG></P><P>
                                        2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 1ng/µl<SUP>-1</SUP></P><P><STONG>2. Primers</STONG></P><P>
                                        2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 10µM reaction solutions.(The solutions are labelled as &quot;Prefix primer&quot; and &quot;suffix primer&quot; in eppendorf tubes in the fridge)
                                    </P></DIV><DIV id="1-1-pcr"><H4>Polymerase Chain Reaction</H4><P>
                                        25µl reactions were run according to the PCR protocol <A href="#">here</A></P><P>
                                        * The final concentrations of the primers were noted as they are needed to determine the annealing temperatures for the primers, which can be done using NEB’s online tool <A href="http://tmcalculator.neb.com/#!/">here.</A></P><P>
                                        ** Add components in order of decreasing volume for maximum ease-of-pipetting.
                                    </P><P>
                                        *** When reaction mixture is being made up, all components as well as the mixture itself are to be kept on ice, as the Master Mix is a high-fidelity polymerase that will recognize the primers as being incorrectly base-paired and be able to hydrolyse the primers if kept at room temperature.
                                    </P><P>
                                        **** Use Q5 enzyme in the cold room to avoid defrosting and freezing the original stock of Q5 enzyme. This could decrease the activity of Q5 enzyme. Bring ice bucket to the cold room to bring Q5 into the bench.
                                    </P><P>
                                        ***** Make sure that the primer and small amounts of DNA and primer doesn’t stick onto the side of the tube or the tip.
                                    </P><P>
                                        The reaction mixture tubes were positioned in an Eppendorf Mastercycler nexus X2 and the PCR program was run.
                                    </P><P>
                                        * DNA denaturation can be performed at 98℃ because of the high thermal stability of the Q5 polymerase
                                    </P><P>
                                        ** A PCR takes 20-30 seconds to extend a sequence by 1kb, and since our longest sequence is ~2kb the extension time was determined to be 60s per cycle.
                                    </P></DIV><DIV id="1-1-gel"><H4>Gel Electrophoresis of PCR-Amplified gBlocks</H4><P>
                                        An agarose gel was prepared according to the <A href="#">agarose prepartion protocol.</A></P><P>
                                        DNA preparation:
                                    </P><P>
                                        The contents of the PCR tubes need to be stained with a loading dye to help visualize its migration.
                                    </P><P>
                                        To each 25µl of content in each PCR tube, 10µl of blue loading dye was added.
                                    </P></DIV></DIV><DIV id="1-2"><H3>Day 2</H3><DIV id="1-2-gel-continued"><H4>Gel Electrophoresis of PCR-Amplified gBlocks (continued from 22/06)</H4><P><A href="#">gBlock sizes for reference</A></P><P>
                                        Lane 1: 10µl of <A href="#https://www.neb.com/products/n0550-quick-load-purple-2-log-dna-ladder-0-1-10-0-kb">2-Log Ladder</A></P><P>        Lanes 2-15: 20µl of DNA and loading dye mixture prepared on <A href="#week-1-day-1-gel-electrophoresis-of-pcr-amplified-gblocks">22/06</A></P><P>
                                        A potential difference of 120V was applied across the electrodes (the higher the voltage, the faster the gel will run but the poor the separation will be; DNA separation is typically done in the 120-140V range) for 80 minutes. As long as DNA has passed ⅔ of the column or purple dye have passed purple area of the gel, the gel is ready to get into the EtBr.
                                    </P><P>
                                        The gel was then stained and the bands were visualized; <A href="#">protocol</A></P></DIV><DIV id="1-2-results"><H4>Results</H4><DIV class="image image-left"><P>PCR Results</P></DIV><P>
                                        The 7 bands corresponding to expected DNA sizes were excised using a razor blade for cleaning and extraction. The other sequences, along with J which only showed a very weak band, will be re-PCRed under modified conditions at a later date.
                                    </P><P>
                                        The excised gel chunks were transferred to centrifugation tubes.
                                    </P></DIV><DIV id="1-2-extraction"><H4>Extraction of DNA(PCR product) from Agarose Gel</H4><P>
                                        The extraction was performed using the E.Z.N.A. Gel Extraction Kit made by Omega Biotek, according to the <A href="#http://omegabiotek.com/store/wp-content/uploads/2014/01/D2500.D2501-Gel-Extraction-Kit-Combo-Online.pdf">Spin Protocol. </A></P><P>
                                        The excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. The heaviest chunk of gel weighed 0.16g and as such 160µl of buffer was added to each tube.
                                    </P><P>
                                        * As the tubes were spun with their lids open, they were placed such that lids pointed away from the direction of spinning.
                                    </P></DIV><DIV id="1-2-restriction-pcr"><H4>Restriction Digest of Extracted PCR product</H4><P>
                                        Restriction digest was performed using EcoRI-HF (5’) and SpeI (3’) restriction enzymes.
                                    </P><P>
                                        NEB’s Double Digest <A href="#https://www.neb.com/tools-and-resources/interactive-tools/double-digest-finder">Finder</A>was invoked and it was determined that CutSmart Buffer would be used for the digest.
                                    </P><P>
                                        The buffer was completely defrosted and shaken before use.
                                    </P><P>
                                        There is a recommended digestion protocol on <A href="#http://nebcloner.neb.com/#!/protocol/re/double/EcoRI-HF,SpeI">NEBCloner.</A> 50µL reaction mixtures were set up with component volumes as recommended, except for the DNA where all 30µL of the extraction mixture (in elution buffer) were used in the digest. There would be up to 50ng of DNA in each tube of extraction mixture (from the gel bands).
                                    </P><P>
                                        Incubation at 37℃ was also done for 2 hours (ThermoMixer program 3) instead of the recommended 5-15 minutes, with shaking at 300rpm.
                                    </P></DIV><DIV id="1-2-restriction-enzyme"><H4>Restriction Enzyme Digest of the iGEM HQ linearised pSB-1c3</H4><P>
                                        125ng of pSB-1C3 was dissolved in 5mL Milli-Q water.
                                    </P><P>
                                        The digest was performed using the modified NEBCloner protocol.</P><P>
                                        * We did not add phosphatase because it is assumed that with different sticky ends the vector cannot religate
                                    </P></DIV><DIV id="1-2-clean-up"><H4>DNA ‘Cleanup’ using EZNA enzymatic reaction kit</H4><P>
                                        Upon completion of restriction digest incubation, the gBlocks and the plasmid backbones were purified again using the E.Z.N.A. Gel Extraction Kit. PCR products and plasmid backbones need to be purified in order to remove remaining impurities including agarose gel and restriction enzymes. At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.
                                    </P></DIV><DIV id="1-2-nanodrop"><H3>DNA Quantification using NanoDrop</H3><P>
                                        1µL water and tissue were first used to clean the stage. A blank reading was made using 1µL of elution buffer, and 1µL of each sample was measured for concentration:
                                    </P><TABLE border="1"><TBODY><TR><TH>Sample</TH><TH>Concentration/ngµl</TH></TR><TR><TD>C</TD><TD>11.3</TD></TR><TR><TD>D</TD><TD>3.5</TD></TR><TR><TD>E</TD><TD>0.8</TD></TR><TR><TD>H</TD><TD>0.9</TD></TR><TR><TD>J</TD><TD>1.6</TD></TR><TR><TD>L</TD><TD>7.7</TD></TR><TR><TD>N</TD><TD>3.6</TD></TR><TR><TD>pSB-1C3</TD><TD>1.6</TD></TR></TBODY></TABLE></DIV><DIV id="1-2-ligation"><H3>Ligation of gBlock Sequences with pSB-1C3 Backbone</H3><P>
                                        Ligation was completed; <A href="#">protocol</A></P></DIV><DIV id="1-2-prep-e.coli-"><H3>Preparation of Competent E. coli Cells</H3><P>
                                        A sample of E. coli DH5ɑ stored at -80℃ was defrosted and inoculated in 5mL of LB in a 125mL conical flask (volume of LB 10% of flask volume so as to achieve sufficient aeration). The culture was left to grow overnight at 37℃.
                                    </P></DIV><DIV id="1-2-re-pcr"><H3>Re-PCR of DNA Sequences</H3><P>
                                        Two sets of sequences A,B,F,G, I, J, K, and M were PCRed using the same protocol as 22/06, with one set using a 55℃ annealing temperature and another using 50℃ annealing temperature.
                                    </P></DIV></DIV><DIV class="section" id="1-3"><H3>Day 3</H3><DIV id="1-3-re-pcr"><H4>Re-PCR of DNA Sequences</H4><P>
                                        Refer to protocol from <A href="#">1.0</A></P><DIV class="image image-right"><P>PCR Results</P></DIV><P>
                                        The amplified sequences A, B, F, G, I, K, J, and M were loaded with blue dye and run on agarose gel as per standard protocol
                                    </P><P>
                                        Sequences G and J showed bands corresponding to the expected sizes at both 50℃ and 55℃ annealing temperatures.
                                    </P><P>
                                        The bands were excised and extracted according to standard <A href="#">protocol</A> (E.Z.N.A. Gel Extraction).
                                    </P></DIV><DIV id="1-3-plasmid-recovery"><H4>Plasmid recovery</H4><P>
                                        The concentration of pSB1C3 plasmid recovered on 23/06 according to the NanoDrop was low (1.6ng/µl). To obtain more plasmids, we restriction digested a construct made by last year’s, pSB1C3 + abijk, to recover the pSB-1C3 content from the construct.
                                    </P><P>
                                        Set up the following reaction mixture:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/µl</TH></TR><TR><TD>DNA (pSB1C3 + insert)</TD><TD>5</TD></TR><TR><TD>EcoR1-HF</TD><TD>1</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>Cutsmart buffer</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>11</TD></TR></TBODY></TABLE><P>
                                        During the incubation, set up 1% agarose gel by putting 1g powder agarose in 100ml; 1% gel is more efficient at separating larger fragments
                                    </P><DIV class="image image-left"><P>Results</P></DIV><OL><LI>Incubate the mix above for 2hrs at 37℃</LI><LI>Incubate for 30mins at 95℃ to inactivate restriction enzymes (Inactivation temperature of SpeI is 80℃, and EcoR1-HF is 65℃)</LI><LI>Cool and spin (there will be condensation at the top of the eppendorf)</LI><LI>Add 1µl CIP (phosphatase) and incubate for 30mins at 37℃</LI><LI>Add loading dye</LI><LI>Load each DNA on the gel</LI><LI>Stain in EtBr for 30 + 20 mins (staining not very clear after 30 mins)</LI></OL></DIV><DIV id="1-3-extraction"><H4>Extraction of “sticky” pSB1C3 (recovered from 2014 pSB1C3+abijk) from Gel</H4><P>
                                        E.Z.N.A. Gel Extraction
                                    </P><H6>Gel weight:</H6><P>
                                        0.53g ⇒ 0.53mL Binding Buffer
                                    </P><P>
                                        30µL of elution buffer used
                                    </P></DIV><DIV id="1-3-repeat-pcr"><H4>Repeat PCR with higher DNA concentration (for the sequences that previously did not yield clear bands)</H4><P>
                                        Sequences A, B, F, G, I, J, K and M have gone through a second round of PCR as they previously failed to yield clear enough bands, and ethidium bromide staining shows that G and J have been successfully isolated as a result.
                                    </P><P>
                                        A, B, F, I, K, L and M still have not had successful PCR runs, as such an alternative PCR protocol with higher DNA concentration was attempted on them instead:
                                    </P><TABLE border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume/µl</TH></TR><TR><TD>gBlock Template (1ng/µl)</TD><TD>5</TD></TR><TR><TD>Forward Primer</TD><TD>1.25</TD></TR><TR><TD>Reverse Primer</TD><TD>1.25</TD></TR></TBODY></TABLE><P>
                                        * A total of 5ng gBlock as opposed to 1ng at 58℃ annealing temperature present in reaction mixture; rest of the protocol is same as prescribed
                                    </P><DIV class="image image-left"><P>Results</P></DIV><P>
                                        Alternative protocol does not work.
                                    </P></DIV><DIV id="1-3-g-j"><H4>Gel extraction of G and J</H4><P>
                                        Standard E.Z.N.A. Gel Extraction protocol followed.
                                    </P><H6>Gel weights:</H6><P>
                                        J - 0.34g ⇒ 0.4mL Binding Buffer
                                    </P><P>
                                        G - 0.3g ⇒ 0.4mL Binding Buffer
                                    </P><P>
                                        30µL of elution buffer used.
                                    </P></DIV><DIV id="1-3-g-j-2"><H4>Restriction Digest of G and J</H4><P>
                                        A <A href="#">restriction digest</A> was then performed on the extracted DNA.
                                    </P></DIV><DIV id="1-3-nanodrop"><H4>NanoDrop Quantification</H4><UL><LI>J: 2.7ng/µl</LI><LI>G: 1.7 ng/µl</LI><LI>pSB-1C3: 1.4ng/ul - there is 8-9ul of this left in the freezer in drawer 4,  labelled with Psb1c3 EcoRI, SpeI and 24/06/2015</LI></UL></DIV><DIV id="1-3-ligation"><H4>Ligation</H4><P>
                                        Ligation perform according to
                                        <A href="#">protocol.</A></P></DIV><DIV id="1-3-transform"><H4>Transforming E. coli cells with Plasmid DNA </H4><P>
                                        Competent e.coli cells were first prepared, according to the protocol <A href="#">here.</A></P><P>
                                        Sample C,D,E,H,J,L and N to be transformed, acoording to the protocol <A href="#">here.</A></P></DIV><DIV class="section" id="1-4"><H3>Day 4</H3><DIV id="1-4-growth"><H4>Growth and Culture of Bacteria </H4><P>
                                            For the protocol please click <A href="#">here.</A></P><P>
                                            Plates incubated overnight show that vectors corresponding to C,D,E,H,J,L and N were taken up [however, later we find that ligation had failed, and so gBlocks C,D,E,H,J,L and N were never inserted]. There are at least 5-7 colonies for each biobrick.
                                        </P><P>
                                            Choose three colonies from each plate. The colony should not be too small or too large and should be reasonably spaced from the others.
                                        </P><P>
                                            Separately incubate each colony in test tubes overnight.
                                        </P><P>
                                            This process would significantly increase the amount of plasmids containing biobricks that we want. Plasmids can be extracted later.
                                        </P></DIV><DIV id="1-4-transform"><H4>Transformation of J and G (24/06 PCR Product) into competent E. coli cells</H4><P>
                                            Competent cells are already made in stocks and can be found in the -80℃ freezer: we don’t need to prepare them again. J and G comes from repeat PCR done in <A href="1-3-g-j">Day 3.</A></P></DIV><DIV id="1-4-primer"><H4>Primer Design</H4><P>
                                            Since A, B, F, G, I, K, M have repeatedly failed to be PCR-amplified, longer primers were designed to replace the old primers for the PCRing of these gene sequences.
                                        </P><P>
                                            Primers that would give the sequences new restriction sites to allow their insertion into pBAD33 and pBAD/HisB expression vectors were also designed.
                                        </P></DIV></DIV><DIV class="section" id="1-5"><H3>Day 5</H3><DIV id="1-5-extraction"><H4>Plasmid Extraction</H4><P>
                                            For the plasmid extraction protocol see <A href="#">here.</A></P><P>
                                            pSB1C3 shipping vector containing gBlocks from Day 1, as well as blank pSB-1C3 shipping vector, pBAD/HisB expression plasmids, and pBAD33 expression plasmids were extracted from overnight cultures of E. coli DH5 using E.Z.N.A. Plasmid DNA Mini Kit I.
                                        </P><P>
                                            (refer to <A href="#http://goo.gl/WhmuUb">pages 10-12</A>for Mini Kit protocol. Some special notes:
                                        </P><UL><LI>All optional steps were carried out except equilibration step.</LI><LI>After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.</LI><LI>Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.</LI><LI>The precipitate formed in following step 7 does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.</LI></UL></DIV><DIV id="1-5-quantification"><H3>Plasmid Quantification</H3><P>
                                            1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
                                        </P><TABLE border="1"><TBODY><TR><TH>DNA</TH><TH>c/ngμl<SUP>-1</SUP></TH><TH>DNA</TH><TH>c/ngμl<SUP>-1</SUP></TH></TR><TR><TD>pBAD/HisB</TD><TD>36.4</TD><TD>mccS (H1)</TD><TD>61.5</TD></TR><TR><TD>pBAD33</TD><TD>34.4</TD><TD>mccS (H2)</TD><TD>129.6</TD></TR><TR><TD>pSB1C3</TD><TD>142.1</TD><TD>DspB+DsbA (J1)</TD><TD>38.8</TD></TR><TR><TD>lsr + GFP (C2)</TD><TD>122.9</TD><TD>DspB+DsbA (J2)</TD><TD>46.3</TD></TR><TR><TD>lsr + GFP (C3)</TD><TD>40.3</TD><TD>DspB+DsbA (J3)</TD><TD>41.9</TD></TR><TR><TD>lsr + Holin (D1)</TD><TD>40.2</TD><TD>Art175+YebF (L1)</TD><TD>64.6</TD></TR><TR><TD>lsr + Holin (D2)</TD><TD>Art175+YebF (L2)</TD></TR><TR><TD>lsr + Holin (D3)</TD><TD>44.1</TD><TD>Art175+YebF (L3)</TD><TD>43.2</TD></TR><TR><TD>DNase+DsbA (E1)</TD><TD>77.4</TD><TD>Art175 + Fla (N1)</TD><TD>103.5</TD></TR><TR><TD>DNase+DsbA (E2)</TD><TD>43.4</TD><TD>Art175 + Fla (N2)</TD><TD>48.8</TD></TR><TR><TD>DNase+DsbA (E3)</TD><TD>40.7</TD><TD>Art175 + Fla (N3)</TD><TD>134.7</TD></TR></TBODY></TABLE></DIV><DIV id="1-5-digest"><H4>Plasmid Digest</H4><P>
                                            The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 90 minutes. For the protocol see <A href="#">here.</A></P></DIV><DIV id="1-5-gel"><DIV class="image image-right"><P>Gel layout</P></DIV><P>
                                            10µl of blue gel loading dye was added to each tube of digested plasmids. 20µl from the contents of each tube were then loaded into a 30-well agarose gel according to the following schematic:
                                        </P></DIV><P>
                                        (*the pSB-1C3 well has pSB-1C3 with abijk insert)
                                    </P><P>                  The ladder well has 10µl of 2-Log Ladder added into it.
                                    </P><P>
                                        A potential difference of 120V was applied across the gel for 40 minutes before it was stained with ethidium bromide for 30 minutes.
                                    </P></DIV><DIV id="1-5-analysis"><H4>Analysis of Results</H4><DIV class="image image-left"><P>Analysis</P></DIV><P>
                                        The first three lanes produced expected results: pBAD33 and pBAD/HisB being “blank” plasmid backbones, the pSB-1C3-abijk lane giving two bands corresponding to similar sizes (being pSB-1C3 and abijk insert respectively).
                                    </P><P>
                                        All the other lanes seem to be showing only products in the 2kb range, which corresponds roughly to the size of the pSB-1C3 backbone, but the sizes are not uniform. Nonetheless, we know that the plasmid extraction step was carried out correctly as it did yield products approximating what we were looking for.
                                    </P><P>
                                        If it was a matter of the ligation step carried out on 23/06 failing entirely, we should be expecting a uniform row of backbone bands. Instead, there are minor but noticeable size variations between each band which cannot be successfully explained by failed digestion/ligation. It is thus speculated that the pSB-1C3 stock we received from iGEM HQ had suffered from varying extents of DNA degradation such that the restriction enzyme cut sites on their ends were no longer.
                                    </P></DIV></DIV></DIV><DIV class="section" id="2"><H5>Week 2</H5><DIV class="secction" id="2-6"><H3>Day 6</H3><DIV id="2-6-pcr"><H4>PCR of samples C,D,E,H,J,L and N</H4><P>
                                        Samples C,D,E,H,J,L and N were PCRed according to the protocol <A href="#">here.</A></P><P>
                                        * Use the Q5 HF Master Mix that has been kept at 4℃ in the cold room and not the frozen sample because repeated freeze thaw cycles are not good for the Master Mix.
                                    </P><P>
                                        ** Remade 50µl 10uM primer stock from the 100uM stock solution (5µl primer in 45µl Milli-Q water)
                                    </P><P>
                                        *** Use DNA 10ng/µl stock solutions made up on Day 1
                                    </P></DIV><DIV id="2-6-gel"><H4>Gel electrophoresis of  C,D,E,H,J,L and N</H4><DIV class="image image-right"><P>Results</P></DIV><P>                                   Refer to gel electrophoresis protocol in <A href="#">section 1.1</A></P><P>
                                        Only J showed no clear band corresponding to the expected sequence size.
                                    </P></DIV><DIV id="2-6-extraction"><H4>EZNA gel extraction of C,D,E,H,J,L and N </H4><P>
                                        Refer to section <A href="#">1.13</A> of the protocol guide.
                                    </P><P>
                                        Excise bands C,D,E,H,L, and N using razor blade, and the excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. For instance, the heaviest band was 0.27g, requiring 0.3ml Binding Buffer to each eppendorf tube.
                                    </P></DIV><DIV id="2-6-digest"><H4>Restriction Digest of Gel Extracted C,D,E,H,J,L and N</H4><P>
                                        Refer to section <A href="#">1.2</A> for the protocol.
                                    </P><P>
                                        Following the gel extraction spin protocol above, extracted PCR DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in 30/06 (Day 7).
                                    </P></DIV><DIV id="2-6-recover"><H4>Recovery of pSB-1C3 vector from 2014 pSB-1C3 + insert</H4><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume (10µl)</TH></TR><TR><TD>pSB-1C3 + insert</TD><TD>10</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>EcoR1 HF</TD><TD>1</TD></TR><TR><TD>Cutsmart</TD><TD>2</TD></TR><TR><TD>Milli-Q</TD><TD>6</TD></TR></TBODY></TABLE><OL><LI>Incubate for 2hrs 37℃</LI><LI>Heat and inactivate the restriction enzymes at 95℃ for 20mins</LI><LI>Cool down in the room temperature, spin (condensation on the side of the eppendorf from heating)</LI><LI>Add 1µl CIP and incubate at 37℃ for 30mins</LI><LI>Add 5µl loading dye</LI></OL><P>
                                        * Phosphatase is added to prevent the vector religating to insert. In the case of PCR amplification of (non-plasmid) gene sequences phosphatase does not need to be added to PCR product because those two ends are unlikely to ligate onto itself.
                                    </P><P>
                                        ** Must start preparing for agarose gel to run 2014 pSB1c3 + Insert. The gel will distinguish pSB1c3 vector from insert, allowing us to extract the vector from the gel and ligate with PCR product. Ligation is to be done in 30/06 (Day 7).
                                    </P><DIV class="image image-right"><P>Results</P></DIV><P>
                                        Gel electrophoresis of pSB1C3.
                                    </P></DIV><DIV id="2-6-ezna"><H4>EZNA gel extraction protocol on recovered pSB1c3 vector</H4><P>
                                        Refer to section <A href="#">1.13</A> for the protocol.
                                    </P><P>
                                        Start with excising the band that corresponds to the base pair length of pSB1c3 vector.
                                    </P><P>
                                        Following the gel extraction spin protocol above, extracted Vector DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in Day7.
                                    </P></DIV><DIV id="2-6-growth"><H4>Growth and culture of E. coli transformed with 24/06 PCR Product (J and G)</H4><P>
                                        Refer to section <A href="#">1.6</A> for the protocol.
                                    </P><P>
                                        * LB has to be clear. The LB on the shelf was cloudy and therefore contaminated from last week so start with a new bottle
                                    </P><P>
                                        ** Colonies were of variable size which could mean that some of the colonies are contaminated. Therefore, when picking the 6 colonies, pick 2 small, 2 medium and 2 large colonies
                                    </P></DIV></DIV><DIV class="section" id="2-7"><H3>Day 7</H3><DIV id="2-7-yesterday"><H4>From yesterday</H4><P>
                                        E.Z.N.A enzymatic reaction cleanup protocol for Restriction Digest products of C, D, E, H, L, and N (<A href="#">1.21</A>)
                                        <UL><LI>At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.</LI></UL></P><P>
                                        E.Z.N.A gel extraction protocol for pSB-1c3 vector isolated from 2014 pSB1C3+insert (<A href="#">1.13</A>)
                                    </P><P>
                                        E.Z.N.A enzymatic reaction protocol for pSB-1C3 gel (<A href="#">1.21</A>)
                                    </P><H6>Ligation of 29/06 PCR products to pSB-1C3 (<A href="#">1.3</A>)</H6><P>
                                        Some notes:
                                    </P><P>
                                        Our component volumes were slightly different from that in day 2 due to the different amount of gBlock/vector we had. (We divided the amount of pSB-1C3 between our 6 samples)
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/μl</TH></TR><TR><TD>Digested DNA (gBlock)</TD><TD>30</TD></TR><TR><TD>pSB-1C3</TD><TD>8</TD></TR><TR><TD>T4 DNA ligase buffer</TD><TD>5</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>6</TD></TR></TBODY></TABLE><P>
                                        Mixtures were incubated on thermomixer at 16 ℃ for 16 hours until 8.45 a.m. on 1/7/15, taking care to vortex before placing on thermomixer.
                                    </P></DIV><DIV id="2-7-extract"><H4>Plasmid Extraction using miniprep kit (<A href="#">1.7</A>)</H4><P>
                                        Sequences G [pSB-1C3] and J [pSB-1C3] were extracted from overnight cultures of E. coli DH5𝛼 using E.Z.N.A. Plasmid DNA Mini Kit I.
                                    </P><P>
                                        refer to pages <A href="http://goo.gl/WhmuUb">10-12</A> for Mini Kit protocol. Some special notes:
                                    </P><UL><LI>All optional steps were carried out except equilibration step.</LI><LI>After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.</LI><LI>After addition of solution I/RNAse, vortexing/vigorous shaking of the tubes should be avoided to prevent shearing of nucleus and undesirable accidental extraction of chromosomal DNA.</LI><LI>After addition of solution I/RNAse, resuspension of pellet can be done by dragging the tube along an eppendorf rack.</LI><LI>Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.</LI><LI>After addition of solution II, the waiting time before proceeding to the next step should not be more than 5 minutes.</LI><LI>The precipitate formed in following addition of solution III does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.</LI></UL><P>
                                        50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
                                    </P></DIV><DIV id="2-7-nanodrop"><H4>Plasmid Quantification: Nanodrop (<A href="#">1.22</A> Nanodrops are usually done following restriction digest and transformation)</H4><P>
                                        1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
                                    </P><TABLE border="1"><TBODY><TR><TH>DNA</TH><TH>Conc. /ngμl<SUP>-1</SUP></TH><TH>DNA</TH><TH>Conc. /ngμl<SUP>-1</SUP></TH></TR><TR><TD>J1</TD><TD>62.4</TD><TD>G1</TD><TD>67.4</TD></TR><TR><TD>J2</TD><TD>77.2</TD><TD>G2</TD><TD>81.5</TD></TR><TR><TD>J3</TD><TD>39.5</TD><TD>G3</TD><TD>80.2</TD></TR><TR><TD>J4</TD><TD>61.4</TD><TD>G4</TD><TD>88.0</TD></TR><TR><TD>J5</TD><TD>37.9</TD><TD>G5</TD><TD>26.3</TD></TR><TR><TD>J6</TD><TD>134.3</TD><TD>G6</TD><TD>95.7</TD></TR></TBODY></TABLE></DIV><DIV id="2-7-plasmid-digest"><H4>Plasmid Restriction Digest for 24/06 PCR Product <A href="#">1.2</A></H4><P>
                                        * 0.5ul enzyme despite digesting a plasmid because test digest
                                    </P><P>
                                        **must refer to the master sheet each time
                                    </P><P>
                                        As many tubes were being handled, the required reagents were pre-mixed:
                                    </P><TABLE border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume / μL</TH></TR><TR><TD>2.1 Buffer</TD><TD>50</TD></TR><TR><TD>Milli-Q</TD><TD>350</TD></TR><TR><TD>EcoRI-HF</TD><TD>12.5</TD></TR><TR><TD>PstI</TD><TD>12.5</TD></TR></TBODY></TABLE><P>
                                        3μL of each insert-containing plasmid was transferred into PCR tubes, and 17μL of the reagent mix was added to each PCR tube.
                                    </P><P>
                                        The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 120 minutes.
                                    </P></DIV><DIV id="2-7-gel"><H4>Gel Electrophoresis of Digested Plasmids from 24/06 PCR Product <A href="#">1.1</A></H4><DIV class="image image-right"><P>Results</P></DIV><P>
                                        5µl of blue gel loading dye was added to each tube of digested plasmids.
                                    </P><P>
                                        20µl from the contents of each tube loaded. Gel run under 80V for 40 minutes:
                                    </P><P>
                                        G5, J3, and J5 were also the samples that showed irregularly low concentrations when analysed with the NanoDrop.
                                    </P><P>
                                        As such, G5, J3, and J5 were discarded while the other samples, being potentially-viable BioBricks, were freeze-stored for verification by gene sequencing at a future date.
                                    </P></DIV><DIV id="2-7-stock"><H4>Preparing new primer stock solutions</H4><TABLE border="1"><TBODY><TR><TH>Primer</TH><TH>amt / 10<SUP>-9</SUP> mol</TH><TH>conc / 10<SUP>-6</SUP> M</TH><TH>Milli-Q vol / 10<SUP>-6</SUP> L</TH></TR><TR><TD>Posy Suffix Holin</TD><TD>28.7</TD><TD>100</TD><TD>287</TD></TR><TR><TD>SMAP 29 Forward</TD><TD>31.2</TD><TD>100</TD><TD>312</TD></TR><TR><TD>DspB Forward</TD><TD>27.7</TD><TD>100</TD><TD>277</TD></TR><TR><TD>Art-E Prefix</TD><TD>35.4</TD><TD>100</TD><TD>354</TD></TR><TR><TD>DNase Forward</TD><TD>18.2</TD><TD>100</TD><TD>182</TD></TR><TR><TD>YebF Forward</TD><TD>29.1</TD><TD>100</TD><TD>291</TD></TR><TR><TD>Art-E Suffix</TD><TD>19.4</TD><TD>100</TD><TD>194</TD></TR><TR><TD>Art 175 Forward</TD><TD>27.5</TD><TD>100</TD><TD>275</TD></TR><TR><TD>Fla Forward</TD><TD>26.5</TD><TD>100</TD><TD>265</TD></TR><TR><TD>Pre-prefix Holin</TD><TD>22.9</TD><TD>100</TD><TD>229</TD></TR><TR><TD>MccS Forward</TD><TD>29.2</TD><TD>100</TD><TD>292</TD></TR><TR><TD>DsbA Forward</TD><TD>27.0</TD><TD>100</TD><TD>270</TD></TR></TBODY></TABLE><P>
                                        Protocol:
                           </P><OL><LI>Pulse spin primers</LI><LI>Add fresh Milli-Q as above table</LI><LI>Leave for 30 mins at 37C in the shaking block (allowing DNA to resuspend)</LI><LI>Dilute to 10µM</LI></OL></DIV><DIV id="2-7-pcr-new"><H4>PCR Amplification of gBlocks Using New Primers</H4><P>
                                        ***Refer to the <A href="#">Master Table</A> for detailing gBlock-primer combinations for both preparation for the gene sequences’ eventual transformation into pSB1C3 shipping vector as well as pBAD33 or pBAD/HisB expression vector.
                                    </P><P>
                                        Primer naming and explantion can be found <A href="#">here.</A></P><P>
                                        With reference to the <A href="#">Master Table</A>, standard PCR reaction mixtures (1uL 1ng/uL DNA template, 1.25uL 10uM forward primer, 1.25uL 10uM reverse primer, 9uL Milli-Q water, 12.5uL Q5 Master Mix) was set up for A*-F*, H*, I*, K*-N*, A#-E#, K#, N#, L#, H#, M#, O#, and P#.
                                    </P><P>
                                        G* and J* were not run because we already potentially have viable BioBricks for them from the plasmid extraction done earlier today <A href="#1-3">(24/06 PCR batch).</A></P><P>
                                        DspB-containing setups J#, I#, F#, G# were not run because DspB has a BspHI restriction site in the middle of its sequence. The DspB-containing gBlocks will first be QuickChange-PCRed as an insert in the shipping vector to remove the restriction site by introducing a point mutation to codon-swap before being redigested out of the shipping vector and ligated into the expression vector.
                                    </P><P>
                                        Some compromises had to be made in terms of annealing temperature as there were only 2 PCR machines available and hence only 4 different programs could be run in parallel. The annealing temperatures were set up as below:
                                    </P><TABLE border="1"><TBODY><TR><TH>Annealing T / ℃</TH><TH>Label</TH></TR><TR><TD>72</TD><TD>A*-F*, H*, I*, K*-N*, A#-D#</TD></TR><TR><TD>67</TD><TD>E#, K#, N#, P#</TD></TR><TR><TD>61</TD><TD>L#, H#</TD></TR><TR><TD>70</TD><TD>M#, O#</TD></TR></TBODY></TABLE><P>
                                        Reaction times and other temperatures were set up according to standard PCR protocol <A href="#">1.0</A></P></DIV></DIV><DIV class="section" id="2-8"><H3>Day 8</H3><DIV id="2-8-gel"><H4>Gel Electrophoresis of 30/06<A href="2-7"> PCR Products</A></H4><P>
                                        Protocol <A href="#">1.2</A></P><DIV class="image image-right"><P>Results</P></DIV><P>
                                        5uL loading dye added per PCR tube. Gel run under 120V for 45 minutes:
                                    </P><P>
                                        Sizes of each fragments can be referred to the <A href="#">Master Table</A>, and the following fragments that show clear bands have been excised and sent for sequencing.
                                    </P><P>
                                        Excised: C*, D*, H*, L*,M*, N*, C#, D#, E#, H#, K#, L#, M#, N#, P#
                                    </P></DIV><DIV id="2-8-digest"><H4>Restriction Digest of 30/06 PCR Products</H4><P>
                                        Protocol <A href="#">1.2</A></P><P>
                                        Grouping by required restriction enzymes -
                                    </P><H6>EcoRI-HF + PstI-HF: C*, D*, H*, L*, M*, N*</H6><P>
                                        Reagent mix (7-portion) first made up:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>CutSmart Buffer</TD><TD>35</TD></TR><TR><TD>Milli-Q</TD><TD>98</TD></TR><TR><TD>EcoRI-HF</TD><TD>3.5</TD></TR><TR><TD>PstI-HF</TD><TD>3.5</TD></TR></TBODY></TABLE><P>
                                        20µL of reagent mix was added to each tube containing 30uL DNA products.
                                    </P><H6>BspHI + PstI-HF: E#, K#, N#, L#, H#</H6><P>
                                        Reagent mix (6-portion) first made up:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>CutSmart Buffer</TD><TD>30</TD></TR><TR><TD>Milli-Q</TD><TD>84</TD></TR><TR><TD>EcoRI-HF</TD><TD>3</TD></TR><TR><TD>PstI-HF</TD><TD>3</TD></TR></TBODY></TABLE><P>
                                        20µL of reagent mix was added to each tube containing 30µL DNA products.
                                    </P><H6>BamHI: C#, D#</H6><P>
                                        Since there are only two tubes to be handles the reagents were directly added to each tube:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>3.1 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>14</TD></TR><TR><TD>EcoRI-HF</TD><TD>0.5</TD></TR><TR><TD>PstI-HF</TD><TD>0.5</TD></TR></TBODY></TABLE><H6>NcoI, PstI: M#, P#</H6><P>
                                        Since there are only two tubes to be handles the reagents were directly added to each tube:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>3.1 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>14</TD></TR><TR><TD>Ncol</TD><TD>0.5</TD></TR><TR><TD>PstI</TD><TD>0.5</TD></TR></TBODY></TABLE><P>
                                        Incubation at 37°C for 2 hours, with 300 rpm shaking. Upon completion, samples stored at -20°C.
                                    </P></DIV><DIV id="2-8-plasmid"><H4>Plasmid Design</H4><P>
                                        Quikchange plasmids for DspB were designed and ordered.
                                    </P><P>
                                        VF2 and VR plasmids, used for sequencing pSB1C3-contained inserts were also ordered.
                                    </P></DIV><DIV id="2-8-transform"><H4>Transformation of <A href="#2-6">29/06</A> ligation products (containing C, D, E, H, L, N gBlocks)</H4><P>
                                        Protocol <A href="#">1.5</A></P><P>
                                        *only 9 tubes of competent DH5alpha E. coli left in the freezer after this round of transformation, as such more will need to be made by the end of the week
                                    </P><P>
                                        Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
                                    </P><P>
                                        Then add volume of E. coli according to protocol, using beads to spread across petri dish.
                                    </P></DIV></DIV><DIV class="section" id="2-9"><H3>Day 9</H3><DIV id="2-9-plasmid-digest"><H4>Plasmid restriction digest - pSB1C3, pBAD33, and pBAD/HisB</H4><P>
                                        Restriciton digest of SB1C3, pBAD33, and pBAD/HisB completely, using the <A href="#">1.2</A> protocol and consulting the <A href="#">master table</A> for the correct buffer and restiction enzymes.
                                    </P><DIV class="image image-left"><P>Resultant Gel</P></DIV><P>
                                        100mL 1% agarose made up, small plate loaded and remainder left in bottle on shelf.
                                    </P><P>
                                        Total volume per tube is 35uL. Each tube’s contents split into two wells (17.5µL per well), and gel was run on 80V p.d. for 45 minutes.
                                    </P><P>
                                        * Gel can help tell if any digestion occurred at all - if no digestion occurred plasmid would still remain circular, and circular plasmid would encounter more resistance migrating through the gel matrix than linear plasmid of the same size and hence appear to be bigger than expected when compared against the ladder.
                                    </P><P>
                                        Bands were excised and the heaviest chunk was 0.66g. 670uL of XP2
                                    </P><P>                           Binding Buffer was added to each excised chunk to initiate E.Z.N.A. Gel Extraction protocol.
                                    </P></DIV><DIV id="2-9-clean-up"><H4>Enzymatic Reaction Clean-up - <A href="2-7">30/06</A> PCR Products</H4><P>
                                        Protocol is <A href="#">1.13</A></P><P>
                                        Retrieve products (on labelled yellow rack) from -20°C and perform clean-up.
                                    </P><P>
                                        The volume of the restriction digest reaction done on 01/07 was 50µL per tube hence according to protocol 50µL of XP2 Binding Buffer added to each tube.
                                    </P><P>
                                        Upon completion of protocol, tubes placed back into -20°C until plasmids ready for ligation.
                                    </P></DIV><DIV id="2-9-growth"><H4>Growth and Culture of Bacteria - 29/06 PCR Products</H4><P>
                                        Refer to section <A href="#">1.6</A> of the protocol guide.
                                    </P><P>
                                        *note: new bags of inoculation loops are placed next to the 37°C incubators for agar plates
                                    </P><P>
                                        LB agar plates of C, D, E, H, L, N collected.
                                    </P><P>
                                        Three colonies selected from each plate to set up three separate overnight cultures each.
                                    </P></DIV><DIV id="2-9-ligation"><H4>Ligation of <A href="#2-7">30/06</A> PCR Products(</H4><P>
                                        Refer to section <A href="#">1.3</A> of the protocol guide.
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/µ</TH></TR><TR><TD>Digested and cleaned PCR products</TD><TD>30</TD></TR><TR><TD>Digested and cleaned plasmids (pSB1C3: C*, D*, H*, L*, M*, N*; pBAD33: C#, D#;pBAD/HisB: E#, K#, N#, L#, H#, M#, P#)</TD><TD>8 for pSB1C3 10 for pBAD337 for pBAD/HisB</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD>T4 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>6 for pSB1C3 4 for pBAD33 7 for pBAD/HisB</TD></TR></TBODY></TABLE><P>
                                        Incubate reaction mixture at 16°C overnight.
                                    </P></DIV><DIV id="2-9-prep"><H4>Preparation of Competent E. coli DH5alpha</H4><P>
                                        Refer to section <A href="#">1.4</A> of the protocol guide.
                                    </P><P>
                                        *Sterile technique used
                                    </P><P>
                                        Test tube filled with 5mL LB broth.
                                    </P><P>
                                        GW opened new bag of inoculation loops to steal a colony off one of Elaine’s streaked plates (marked 29/06) and inoculated it in the filled test tube.
                                    </P><P>
                                        Test tube left to incubate at 37°C overnight.
                                    </P></DIV></DIV><DIV class="section" id="2-10"><H3>Day 3</H3><DIV id="2-10-plasmid"><H4>Plasmid for 29/06 PCR Products</H4><P>
                                        Refer to section <A href="#">1.7</A> of the protocol guide.
                                    </P><P>
                                        Yesterday, collected LB agar plates of C, D, E, H, L, N and Interlab are incubated overnight. Selected three colonies from each of C, D, E, H, L and N, and only one for each of the Interlab colonies (we assume that the plasmids provided by iGEM HQ for Interlab were all the same).
                                    </P><P>
                                        Therefore we shall be performing EZNA plasmid extraction on 22 samples. Follow EZNA Mini Kit I Spin Protocol (pg10-12).
                                    </P><P>
                                        50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
                                    </P><P>
                                        Also, aspirate = pipetting!
                                    </P><P>
                                        Measured the concentration of the different samples using nanodrop
                                    </P><TABLE border="1"><TBODY><TR><TH>Sample</TH><TH>Concentration (ng/μl)</TH></TR><TR><TD>C1</TD><TD>46.7</TD></TR><TR><TD>C2</TD><TD>35.6</TD></TR><TR><TD>C3</TD><TD>52.7</TD></TR><TR><TD>D1</TD><TD>43.2</TD></TR><TR><TD>D2</TD><TD>95.8</TD></TR><TR><TD>D3</TD><TD>30.4</TD></TR><TR><TD>E1</TD><TD>45.1</TD></TR><TR><TD>E2</TD><TD>112.4</TD></TR><TR><TD>E3</TD><TD>46.7</TD></TR><TR><TD>H1</TD><TD>71.6</TD></TR><TR><TD>H2</TD><TD>48.2</TD></TR><TR><TD>H3</TD><TD>48.5</TD></TR><TR><TD>L1</TD><TD>48.7</TD></TR><TR><TD>L2</TD><TD>56.2</TD></TR><TR><TD>L3</TD><TD>69.2</TD></TR><TR><TD>N1</TD><TD>57.0</TD></TR><TR><TD>N2</TD><TD>47.0</TD></TR><TR><TD>N3</TD><TD>68.3</TD></TR></TBODY></TABLE><P>
                                        Digest aliquots of each of the 22 samples, and run on gel. Refer to protocol <A href="#">1.2</A></P><P>
                                        From gel, determine the degree of success of these samples.
                                    </P></DIV><DIV id="2-10-prep"><H4>Preparation of Competent E. coli DH5alpha</H4><P>
                                        Refer to protocol <A href="#">1.4</A></P></DIV><DIV id="2-10-transform"><H4>Transformation of 30/06 PCR Products (Ligated on 02/07) (Protocol in section 1.5)</H4><P>
                                        Refer to protocol in section 1.5
                                    </P><P>
                                        Antibiotics:
                                    </P><UL><LI>
                                            pSB1C3, pBAD33 - Chl
                                            <P>
                                                for C*, D*, H*, L*, M*, N*, C#, D#

                                            </P></LI><LI>
                                            pBAD/HisB - Amp
                                            <P>
                                                for E#, K#, N#, L#, H#, M#, P#
                                            </P></LI></UL><P>
                                        Raffy, Leon: Moving of cells plated on 3/7/2015 from the incubator to the cold room. All pSB1C3 plates had reasonable colony density. A significant proportion of pBAD/HisB plated had no colonies. Plates with no colonies will be reincubated for half a day on 6/7/2015.
                                    </P></DIV></DIV></DIV><DIV class="section" id="3"><H5>Week 3</H5><DIV class="section" id="3-11"><H3>Day 11</H3><DIV id="3-11-prep"><H4>Preparation of Overnight Cultures for 30/06 PCR Products</H4><TABLE border="1"><TBODY><TR><TH>Plate</TH><TH>Colony</TH><TH>Plate</TH><TH>Colony</TH></TR><TR><TD>C#</TD><TD>1</TD><TD>C#e</TD><TD>1</TD></TR><TR><TD>D#</TD><TD>0</TD><TD>D#e</TD><TD>0</TD></TR><TR><TD>E#</TD><TD>2</TD><TD>E#e</TD><TD>2</TD></TR><TR><TD>H#</TD><TD>0</TD><TD>H#e</TD><TD>3</TD></TR><TR><TD>K#</TD><TD>0</TD><TD>K#e</TD><TD>0</TD></TR><TR><TD>L#</TD><TD>0</TD><TD>L#e</TD><TD>0</TD></TR><TR><TD>M#</TD><TD>0</TD><TD>M#e</TD><TD>0</TD></TR><TR><TD>N#</TD><TD>0</TD><TD>N#e</TD><TD>0</TD></TR><TR><TD>P#</TD><TD>1</TD><TD>P#e</TD><TD>0</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>Plate</TH><TH>Colony</TH><TH>Plate</TH><TH>Colony</TH></TR><TR><TD>C*</TD><TD>Yes</TD><TD>C*e</TD><TD>Yes</TD></TR><TR><TD>D*</TD><TD>3</TD><TD>D*e</TD><TD>0</TD></TR><TR><TD>H*</TD><TD>Yes</TD><TD>H*e</TD><TD>Yes</TD></TR><TR><TD>L*</TD><TD>Yes</TD><TD>L*e</TD><TD>Yes</TD></TR><TR><TD>M*</TD><TD>0</TD><TD>M*e</TD><TD>3</TD></TR><TR><TD>N*</TD><TD>Yes</TD><TD>N*e</TD><TD>Yes</TD></TR></TBODY></TABLE><P>
                                        Colonies grew for L* and N*, but because the MiniPrep run on 03/07 already showed the correct bands for them, their colonies were not cultured overnight.
                                    </P><TABLE border="1"><TBODY><TR><TH>Sequence</TH><TH>Colonies picked</TH></TR><TR><TD>C*</TD><TD>3</TD></TR><TR><TD>D*</TD><TD>3</TD></TR><TR><TD>H*</TD><TD>3</TD></TR><TR><TD>M*</TD><TD>3</TD></TR><TR><TD>C#</TD><TD>2</TD></TR><TR><TD>E#</TD><TD>4</TD></TR><TR><TD>H#</TD><TD>3</TD></TR><TR><TD>P#</TD><TD>1</TD></TR></TBODY></TABLE><P>
                                        See <A href="#">protocol guide</A> to find out how to make overnight cultures.
                                    </P></DIV><DIV id="3-11-pcr"><H4>PCR amplification of A,B,D,E,F,I,K,M for pSB1C3</H4><P><STRONG>1. Primer used:</STRONG></P><P>* primer pair (pre prefix holin and post suffix holin)</P><P>Comments:</P><P>
                                        Attempting A, B, F and I again as they have never been successfully amplified before.Attempting A, B, F and I again as they have never been successfully amplified before.
                                    </P><P>
                                        NB: F and I are DspB-containing gBlock which we eventually hope to QuikChange to get rid of the BspHI restriction site in them and put them into pBAD/HisB expression vector
                                    </P><P><STRONG>2. PCR all the # sequences with the appropriate primers</STRONG></P><P>
                                        Follow <A href="#">1.0</A> PCR protocol, and primer list is in the <A href="#">Master Table</A>.
                                    </P></DIV><DIV id="3-11-digest"><H4>Restriction digest for pSB1C3, pBAD33 and pBAD/HisB</H4><DIV class="image image-right"><P>Gel photo</P></DIV><P>
                                        Follow <A href="#">restriction digest</A> protocol.
                                    </P><P>
                                        pSB1C3 and pBAD/HisB were successfully digested and eluted in 1% gel. Band for pBAD33 could not be found.
                                    </P><P>
                                        Excised bands were stored in -20℃ for cleanup tomorrow.
                                    </P></DIV><DIV id="3-11-primer-prep"><H4>Primer Preparation</H4><P>
                                        Preparing new primers (solution and dilution) - VF2, VR, QuikChange forward, QuikChange reverse.
                                    </P><P>
                                        First make 100uM out of the freeze dry solid:
                                    </P><OL><LI>Spin down solid</LI><LI>The amount of primer is given in y nmol for each tube. Add 10y µL of water to each tube to make 100µM.</LI><LI>Shake/vortex and mix evenly.</LI></OL><P>
                                        For VF2, VR - take 3.2ul out of the 100µM stock and dilute with 96.8µL Milli-Q to make 3.2pmol/uL sequencing primer solution.
                                    </P><P>
                                        For the QuikChange primers, dilute to 10uM as per normal.
                                    </P></DIV><DIV id="3-11-sequencing"><H4>Sequencing of BioBricks</H4><P>
                                        5µl of each plasmid along with 100uL of 3.2pmol/uL sequencing primer sent to SourceBioScience for sequencing:
                                    </P><TABLE border="1"><TBODY><TR><TH>Sequence</TH><TH>Label Assigned</TH></TR><TR><TD>C 3</TD><TD>pSBLsrGFP3</TD></TR><TR><TD>D 2</TD><TD>pSBLsrHolin2</TD></TR><TR><TD>E 2</TD><TD>SBDNaseDsbA2</TD></TR><TR><TD>G 6</TD><TD>pSBDspB6</TD></TR><TR><TD>H 1</TD><TD>pSBMccS1</TD></TR><TR><TD>J 2</TD><TD>pSBDspBDsbA2</TD></TR><TR><TD>L 3</TD><TD>pSBArt175YebF3</TD></TR><TR><TD>N 3</TD><TD>SBArt175Fla3</TD></TR></TBODY></TABLE></DIV><DIV id="3-11-gel"><H4>Gel Electrophoresis of PCR Products</H4><P>
                                        Correct bands were obtained for D*, C#, G#, H#, J#, K#, L#, N#. Still awaiting reply on insert length for O# and P# to determine whether the bands are correct.
                                    </P></DIV></DIV><DIV class="section" id="3-12"><H3>Day 12</H3><DIV id="3-12-nanodrop"><H4>NanoDrop Analysis of Plasmids Digested on 06/07 (carried out after gel extraction)</H4><TABLE border="1"><TBODY><TR><TH>Plasmid</TH><TH>Conc / nguL<SUP>-1</SUP></TH></TR><TR><TD>pSB1C3</TD><TD>5, 9.5</TD></TR><TR><TD>pBAD/HisB</TD><TD>1.8, 1.6</TD></TR></TBODY></TABLE><P>
                                        Since pBAD/HisB is very low in concentration, more of it will be digested.
                                    </P></DIV><DIV id="3-12-digest"><H4>Restriction Digest of pBAD33</H4><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>Plasmids</TD><TD>10</TD></TR><TR><TD>BamHI</TD><TD>1</TD></TR><TR><TD>Milli-Q Water</TD><TD>7</TD></TR><TR><TD>3.1 Buffer</TD><TD>2</TD></TR></TBODY></TABLE><OL><LI>37℃, 2 hours (heat up another block to 95℃)</LI><LI>Heat inactivation, 95℃, 20 minutes</LI><LI>Melt, cool, and pour 1% agarose</LI><LI>Pulse spin</LI><LI>CIP dephosphorylation (1 µL CIP), 37℃, 30 minutes</LI><LI>Load dye and run gel</LI></OL><P>
                                        pBAD/HisB digest - 6uL water, 1µL NcoI, 1µL PstI, 2µL 3.1, 10µL plasmid
                                    </P></DIV><DIV id="3-12-extraction"><H4>Gel extraction of digested plasmids</H4><P><STRONG>Plasmids digested on 6/7:</STRONG></P><P>
                                        400µL of Binding Buffer added to each tube containing excised band.
                                    </P><P>
                                        After first elution, concentrations turned out to be low (see NanoDrop table at the top of this document), hence a second elution was done, again using 50µL of elution buffer.
                                    </P><P><STRONG>Plasmids digested on 7/7:</STRONG></P><P>
                                        340uL of Binding Buffer; one elution with 50µL.
                                    </P></DIV><DIV id="3-12-plasmid-extract"><H4>Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)</H4><P>
                                        Follow EZNA DNA Mini Kit I Spin Protocol - eluted 50μl
                                    </P><P>
                                        freezer: expression vector ones (the ones with #) on orange rack in bottom drawer (will be dealt with tomorrow), psb vector ones (the ones with *) in white box with other psb constructs in top drawer
                                    </P><P>
                                        NanoDrop analysis:
                                    </P><TABLE border="1"><TBODY><TR><TH>Sequence</TH><TH>Conc / ngµL<SUP>-1</SUP></TH><TH>Sequence</TH><TH>Conc / ngµL<SUP>-1</SUP></TH></TR><TR><TD>C* 1</TD><TD>58.3</TD><TD>M* 3</TD><TD>49.2</TD></TR><TR><TD>C* 2</TD><TD>67.5</TD><TD>C# 1</TD><TD>64.5</TD></TR><TR><TD>C* 3</TD><TD>66.0</TD><TD>C# 2</TD><TD>25.1</TD></TR><TR><TD>D* 1</TD><TD>6.5</TD><TD>E# 1</TD><TD>37.8</TD></TR><TR><TD>D* 2</TD><TD>43.3</TD><TD>E# 2</TD><TD>43.1</TD></TR><TR><TD>D* 3</TD><TD>56.7</TD><TD>E# 3</TD><TD>30.5</TD></TR><TR><TD>H* 1</TD><TD>30.0</TD><TD>E# 4</TD><TD>67.3</TD></TR><TR><TD>H* 2</TD><TD>45.9</TD><TD>H# 1</TD><TD>36.5</TD></TR><TR><TD>H* 3</TD><TD>44.5</TD><TD>H# 2</TD><TD>28.8</TD></TR><TR><TD>M* 1</TD><TD>51.1</TD><TD>H# 3</TD><TD>37.3</TD></TR><TR><TD>M* 2</TD><TD>53.1</TD><TD>P# 1</TD><TD>179.2</TD></TR></TBODY></TABLE></DIV><DIV id="3-12-gel-2"><H4>Gel electrophoresis of Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)</H4><P>
                                        Electrophoresis of 10uL plasmid with 5uL loading dye
                                    </P><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        N.B. forgot to digest - will do this on 08/07/15 (tomorrow)
                                    </P></DIV><DIV id="3-12-sequence"><H4>Analysis of sequencing data from 06/07/15</H4><P>
                                        Of the stuff sent for sequencing (03/07 miniprep):
                                    </P><TABLE border="1"><TBODY><TR><TH>Sample</TH><TH>Results</TH><TH>Action Point</TH></TR><TR><TD>C*3</TD><TD>corresponds to pSB1C3 Lsr GFP</TD><TD>-</TD></TR><TR><TD>D*2</TD><TD>Corresponds to pSB1C3 Lsr GFP (does not correspond to pSB1C3 Lsr Holin as expected - contamination in all three D* wells were already apparent in gel (see 06/07))</TD><TD>Wrong insert, need to redo from scratch (start from PCR again)</TD></TR><TR><TD>E*3</TD><TD>DsbA DNAse missing, sequencing only gives blank pSB1C3 vector</TD><TD>Send another miniprep product in the triplicate (E*1 or E*2) for sequencing.</TD></TR><TR><TD>G*6</TD><TD>Forward sequence good enough to confirm (~70% length of DspB sequence) that it’s pSB1C3 DspB, but reverse sequence dropped off too early for double confirmation</TD><TD>Ask Source Bioscience to redo VR (reverse) sequence</TD></TR><TR><TD>H*1</TD><TD>Corresponds to pSB1C3 MccS</TD><TD>-</TD></TR><TR><TD>J*2</TD><TD>Corresponds to DsbA DspB, but base 2964 had a C-&gt;A point mutation</TD><TD>Send another miniprep product in the triplicate (J*1 or J*3) for sequencing</TD></TR><TR><TD>L*3</TD><TD>Corresponds to pSB1C3 Art-175 YebF</TD><TD>-</TD></TR><TR><TD>N*3</TD><TD>Corresponds to pSB1C3 Art-175 Fla</TD><TD>-</TD></TR></TBODY></TABLE><P>
                                        Results: 4, potentially 5 BioBricks successfully made in [pSB1C3] format. Successful BioBricks to be stored in separate box in preparation for sending to Registry, creating Database page etc.
                                    </P></DIV><DIV id="3-12-ligation"><H4>Ligation of 06/07 PCR to Appropriate Digested Plasmids</H4><P>
                                        pBAD33: C#1, C#2 (one of these were wrongly digested using CutSmart instead of 3.1), D# (previously digested using wrong buffer), D#3.1
                                    </P><P>
                                        pSB1C3: D*
                                    </P><P>
                                        pBAD/HisB: E#, G#, H#, J#, K#, L#, N#, O# (previously digested using wrong buffer), O#3.1, P# (previously digested using wrong buffer), P#3.1
                                    </P><P>
                                        Refer to section  of the protocol guide.
                                    </P><P>
                                        Left overnight at 16C.
                                    </P></DIV></DIV><DIV class="section" id="3-13"><H3>Day 13</H3><DIV id="3-13-to-do"><H4>To Do</H4><P>
                                        Prepare plates for transformation
                                    </P><P>
                                        Transformation of ligated plasmids
                                    </P><P>
                                        Gradient PCR:
                                    </P><UL><LI>Group A: A*, B*, F*, I*, K*</LI><LI>Group B: A#, B#</LI><LI>Group C: I#, F# </LI></UL></DIV><DIV id="3-13-to-note"><H4>To Note</H4><P>
                                        Re-sequenced VR read and DspB confirmed as correct = another biobrick.
                                    </P></DIV><DIV id="3-13-quick-change"><H4>QuikChange PCR on DspB</H4><P>
                                        Standard PCR protocol <A href="#">1.0</A></P><UL><LI>72C annealing temp</LI><LI>2 min extension time</LI></UL><P>
                                        Add 1ul DpnI and incubate at 37C for two hours
                                    </P></DIV><DIV id="3-13-transform"><H4>Transformation</H4><P>
                                        Add 1ul PCR product to 100ul competent cells. Add the remaining PCR product (24ul) to another eppendorf of 100ul competent cells
                                    </P><P>
                                        Follow standard transformation protocol </P></DIV><DIV id="3-13-digest"><H4>Restriction Digest</H4><P>
                                        Incubate at 37oC for 60 mins - BamHI has star activity so it cannot be left for long
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/µL</TH><TH>Component</TH><TH>Volume/µL</TH><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>C#</TD><TD>5</TD><TD>C*/D*/H*/M*</TD><TD>5</TD><TD>E#/H#/P#</TD><TD>5</TD></TR><TR><TD>3.1 NEB 10x buffer</TD><TD>2</TD><TD>2.1 NEB 10x buffer</TD><TD>2</TD><TD>3.1 NEB 10x buffer</TD><TD>2</TD></TR><TR><TD>Bam HI</TD><TD>0.5</TD><TD>PstI</TD><TD>0.5</TD><TD>Bam HI</TD><TD>0.5</TD></TR><TR><TD>(no 2nd RE)</TD><TD>-</TD><TD>EcoRI-HF</TD><TD>0.5</TD><TD>EcoRI-HF</TD><TD>0.5</TD></TR><TR><TD>MilliQ</TD><TD>12.5</TD><TD>MilliQ</TD><TD>12</TD><TD>MilliQ</TD><TD>12</TD></TR></TBODY></TABLE></DIV><DIV id="3-13-gel"><H4>Gel electrophoresis of restriction digest</H4><DIV class="image image-left"><P>Gel Photo</P></DIV></DIV><DIV id="3-13-sequencing"><H4>Sent for sequencing:</H4><TABLE border="1"><TBODY><TR><TH>Sequence</TH><TH>Concentration</TH><TH>Label Assigned</TH></TR><TR><TD>E* 1</TD><TD>45.1</TD><TD>SBDNaseDsbA1</TD></TR><TR><TD>J* 1</TD><TD>62.4</TD><TD>SBDspBDsbA1</TD></TR><TR><TD>C# 1</TD><TD>64.5</TD><TD>33LsrGFP1</TD></TR><TR><TD>H# 3</TD><TD>37.3</TD><TD>BMccS3</TD></TR><TR><TD>E# 4</TD><TD>67.3</TD><TD>BDNaseDspA4</TD></TR><TR><TD>P# 1</TD><TD>179.2</TD><TD>BDNase1</TD></TR><TR><TD>D* 3</TD><TD>56.7</TD><TD>SBLsrHolin3</TD></TR><TR><TD>H* 2</TD><TD>45.9</TD><TD>SBMccS2</TD></TR><TR><TD>M* 2</TD><TD>53.1</TD><TD>SBArtE2</TD></TR></TBODY></TABLE></DIV><DIV id="3-13-extra"><P>
                                        Transformation of E. coli with ligation products from day12 (06/07 PCR and interlab sequences) using the standard protocol described by diagrams for day 3. Transformed E. coli then plated and incubated overnight
                                    </P></DIV></DIV></DIV></DIV><DIV class="section" id="interlab"><H2>Interlab</H2><DIV class="section" id="2-i"><H5>Week 2</H5><DIV class="section" id="2-8-i"><H3>Day 8</H3><DIV id="2-8-i-prep"><H4>Preparation of Interlab Study BioBricks</H4><P>
                                        10µL Milli-Q added to wells 20K, 22A, and 22K in Plate 1, and well 21J in Plate 4 in the iGEM Distribution Kit to resuspend the necessary BioBricks [pSB1C3] to prepare for the Interlab study.
                                    </P><P>
                                        The plates were kept on mild shaking until the afternoon.
                                         </P></DIV><DIV id="2-8-i-transform"><H4>Transformation of Interlab Study BioBricks into DH5alpha <A href="#">1.5</A></H4><P>
                                        Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
                                    </P><P>
                                        Then add volume of E. coli according to protocol, using beads to spread across petri dish.
                                    </P></DIV></DIV><DIV class="section" id="2-9-i"><H3>Day 9</H3><DIV id="2-9-i-growth"><H4>Growth and Culture of Bacteria</H4><P>
                                        Refer to section <A href="#">1.6</A> of the protocol guide.
                                    </P><P>
                                        Only one overnight culture each set up for iGEM distros (20K, 22A, 22K, 21J) as it can be rightly assumed that plasmids supplied by iGEM HQ are high purity.
                                    </P></DIV></DIV><DIV class="section" id="2-10-i"><H3>Day 10</H3><DIV id="2-10-i-gel"><H4>Plasmid for PCR Products</H4><P>
                                        Digest aliquots of each of the samples, and run on gel. (Refer to protocol in section <A href="#">1.2</A>)
                                    </P><P>
                                        From gel, determine the degree of success of these samples.
                                    </P></DIV></DIV></DIV><DIV class="section" id="3-i"><H5>Week 3</H5><DIV class="section" id="3-11-i"><H3>Day 11</H3><DIV id="3-11-i-digest"><H4>Restriction Digest of InterLab Study Plasmids</H4><P>
                                        20K, 22A, 22K:
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>Plasmids</TD><TD>10</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>PstI-HF</TD><TD>1</TD></TR><TR><TD>Milli-Q Water</TD><TD>6</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR></TBODY></TABLE><P>
                                        21J: Same as above, but replace SpeI for XbaI.
                                    </P><P>
                                        Incubated for 2hrs at 37℃, then enzymes heat inactivated at 95℃.
                                    </P></DIV><DIV id="3-12-i-digest"><H4>Restriction Digest of InterLab BioBricks</H4><DIV class="image image-right"><P>Gel Photo</P></DIV><OL><LI>Pulse spin</LI><LI>Dephosphorylate 20K, 22A, 22K, <STRONG>BUT NOT 21J</STRONG></LI><LI>Load dye and run gel</LI></OL><P>
                                        Gel image (transilluminator lens was quite grainy today)
                                    </P><P>
                                        Appropriate bands excised for extraction (I13504 for last lane)
                                    </P></DIV><DIV id="3-12-i-extraction"><H4>Gel extraction of InterLab BioBricks</H4><P>
                                        340uL of Binding Buffer;
                                    </P><P>
                                        one elution with 50uL for 20K, 22A, 22K; 30uL for 21J
                                    </P></DIV><DIV id="3-12-i-ligation"><H4>Ligation of InterLab sequences</H4><P>
                                        Insert: digested 21J
                                    </P><P>
                                        Plasmids: 20K, 22A, 22K
                                    </P><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>T4 Buffer</TD><TD>4</TD></TR><TR><TD>Insert</TD><TD>10</TD></TR><TR><TD>Plasmid</TD><TD>10</TD></TR><TR><TD>Milli-Q</TD><TD>26</TD></TR></TBODY></TABLE><P>
                                        NB: total reaction volume of 51µL is 1µL more than ideal
                                    </P><P>
                                        NB: total reaction volume of 51uL is 1uL more than ideal
                                    </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#cloning">Cloning</A></LI><LI><A href="#1">Week 1</A></LI><LI><A href="#1-1">Day 1</A></LI><LI><A href="#1-1-prep-stock">Preparation of Stock Solutions</A></LI><LI><A href="#1-1-prep-reaction">Preparation of Reaction Solutions</A></LI><LI><A href="#1-1-pcr">Polymerase Chain Reaction</A></LI><LI><A href="#1-1-gel">Gel Electrophoresis of PCR-Amplified gBlocks</A></LI></UL><LI><A href="#1-2">Day 2</A></LI><LI><A href="#1-2-gel-continued">Gel Electrophoresis</A></LI><LI><A href="#1-2-results">Results</A></LI><LI><A href="#1-2-extraction">Extraction of DNA</A></LI><LI><A href="#1-2-restriction-pcr">Restriction Digest PCR</A></LI><LI><A href="#1-2-restriction-enzyme">Restriction Enzyme Digest</A></LI><LI><A href="#1-2-clean-up">DNA ‘Cleanup’</A></LI><LI><A href="#1-2-nanodrop">NanoDrop</A></LI><LI><A href="#1-2-ligation">Ligation</A></LI><LI><A href="#1-2-prep-e.coil">Preparation of E. coli</A></LI><LI><A href="#1-2-re-pcr">Re-PCR of DNA</A></LI><LI><A href="#1-3">Day 3</A></LI><LI><A href="#1-3-re-pcr">Re-PCR of DNA Sequences</A></LI><LI><A href="#1-3-plasmid-recovery">Plasmid Recovery</A></LI><LI><A href="#1-3-extraction">Extraction of “sticky” pSB1C3</A></LI><LI><A href="#1-3-repeat-pcr">Repeat PCR with higher DNA concentration</A></LI><LI><A href="#1-3-g-j">Gel extraction of G and J</A></LI><LI><A href="#1-3-nanodrop">NanoDrop</A></LI><LI><A href="#1-3-ligation">Ligation</A></LI><LI><A href="#1-3-transform">Transformation</A></LI><LI><A href="#1-4">Day 4</A></LI><LI><A href="#1-4-growth">Growth and culture of bacteria</A></LI><LI><A href="#1-4-transform">Transformation of G and J</A></LI><LI><A href="#1-4-primer">Primer Design</A></LI><LI><A href="#1-5">Day 5</A></LI><LI><A href="#1-5-extraction">Plasmid Extraction</A></LI><LI><A href="#1-5-quantification">Plasmid Quantification</A></LI><LI><A href="#1-5-digest">Plasmid Digest</A></LI><LI><A href="#1-5-gel">Gel Electrophoresis of Digested Plasmids</A></LI><LI><A href="#1-5-analysis">Analysis</A></LI><LI><A href="#2">Week 2</A></LI><LI><A href="#2-6">Day 6</A></LI><LI><A href="#2-6-pcr">PCR</A></LI><LI><A href="#2-6-gel">Gel Electrophpresis</A></LI><LI><A href="#2-6-extraction">Gel Extraction</A></LI><LI><A href="#2-6-digest">Restriction Digest</A></LI><LI><A href="#2-6-recover">Recovery of pSB-1C3 vector</A></LI><LI><A href="#2-6-ezna">EZNA gel extraction protocol on recovered pSB1c3 vector</A></LI><LI><A href="#2-6-growth">Growth and culture of E. coli transformed J and G</A></LI><LI><A href="#2-7">Day 7</A></LI><LI><A href="#2-7-yesterday">Yesterday</A></LI><LI><A href="#2-7-extract">Miniprep</A></LI><LI><A href="#2-7-nanodrop">Nanodrop</A></LI><LI><A href="#2-7-plasmid-digest">Plasmid Digest</A></LI><LI><A href="#2-7-stock">Stock Solutions</A></LI><LI><A href="#2-7-pcr-new">PCR Amplification with New Primers</A></LI><LI><A href="#2-8">Day 8</A></LI><LI><A href="#2-8-gel">Gel Electrophoresis</A></LI><LI><A href="#2-8-digest">Restriction Digest</A></LI><LI><A href="#2-8-plasmid">Plasmid Design</A></LI><LI><A href="#2-8-transform">Transformation</A></LI><LI><A href="#2-9">Day 9</A></LI><LI><A href="#2-9-plasmid-digest">Plasmid Digest</A></LI><LI><A href="#2-9-clean-up">Enzymatic Reaction  Clean-up</A></LI><LI><A href="#2-9-growth">Growth and Culture</A></LI><LI><A href="#2-9-ligation">Ligation</A></LI><LI><A href="#2-9-prep">Preparation of DH5</A></LI><LI><A href="#2-10">Day 10</A></LI><LI><A href="#2-10-plasmid">Plasmid for PCR Products</A></LI><LI><A href="#2-10-prep">Preparation of DH5</A></LI><LI><A href="#3">Week 3</A></LI><LI><A href="#3-11">Day 11</A></LI><LI><A href="#3-11-prep">Overnight Cultures</A></LI><LI><A href="#3-11-pcr">PCR</A></LI><LI><A href="#3-11-digest">Restriction Digest</A></LI><LI><A href="#3-11-primer-prep">Primer Preparation</A></LI><LI><A href="#3-11-sequencing">Sequencing</A></LI><LI><A href="#3-11-gel">Running the Gel</A></LI><LI><A href="#3-12">Day 12</A></LI><LI><A href="#3-12-nanodrop">NanoDrop</A></LI><LI><A href="#3-12-digest">Restriction Digest</A></LI><LI><A href="#3-12-extraction">Gel Extraction</A></LI><LI><A href="#3-12-plasmid-extract">Plasmid Extraction</A></LI><LI><A href="#3-12-gel-2">Gel Electrophoresis</A></LI><LI><A href="#3-12-sequence">Analysis of Sequencing Data</A></LI><LI><A href="#3-12-ligation">Ligation</A></LI><LI><A href="#3-13">Day 13</A></LI><LI><A href="#3-13-to-do">To Do</A></LI><LI><A href="#3-13-to-note">To Note</A></LI><LI><A href="#3-13-quickchange">QuickChange</A></LI><LI><A href="#3-13-transform">Transformation</A></LI><LI><A href="#3-13-digest">Restriction Digest</A></LI><LI><A href="#3-13-gel">Gel Electrophroesis</A></LI><LI><A href="#3-13-sequencing">Sequencing</A></LI><LI><A href="#3-13-extra">Extra</A></LI><LI><A href="#interlab">Interlab</A></LI><LI><A href="#2-i">Week 2</A></LI><LI><A href="#2-8-i">Day 8</A></LI><LI><A href="#2-8-i-prep">Preparation of Interlab Study BioBricks</A></LI><LI><A href="#2-8-i-transform">Transformation</A></LI><LI><A href="#2-9-i">Day 9</A></LI><LI><A href="#2-9-i">Growth and Culture of Bacteria</A></LI><LI><A href="#2-10-i">Day 10</A></LI><LI><A href="#2-10-i-gel">Plasmid for PCR Product</A></LI><LI><A href="#3-1">Week 3</A></LI><LI><A href="#3-11-i">Day 11</A></LI><LI><A href="#3-11-i-digest">Restriction Digest</A></LI><LI><A href="#3-12-i">Day 12</A></LI><LI><A href="#3-12-i-extraction">Gel Extraction</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>