<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Test_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Test/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Oxford">OXFORD</A></DIV><NAV class="navbar-collapse collapse" id="ox-navbar"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Questionnaires">Questionnaire Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/UTB">Urinary Biome Study</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Next">Lab To Clinic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Experiments">Overview &amp; Results</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Beads">AlgiBeads Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Synbiota">Synbiota</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Modeling/Tutorial">Tutorials</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Contact">Contact</A></LI><LI><A href="https://2015.igem.org/Team:Oxford/Blog">BLOG</A></LI></NAV></DIV><DIV id="main-container" class="container-fluid"><DIV class="container-fluid page-heading" style="background-image: url(https://static.igem.org/mediawiki/2015/7/79/Backgroundnotebook.png)"><H3>Notebook</H3></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-9"><DIV id="notebook-key-button" data-spy="affix"><H3>Parts Guide</H3></DIV><DIV id="notebook-key"><H2>Parts Guide</H2><DIV class="notebook-key-content"><P>
                            In the cloning section of our notebook, our sequences are each assigned a letter code followed by a symbol. The letter codes denote the identity of sequences as follows:
                        </P><P class="list-of-parts"><STRONG>A</STRONG> - pLacI-LasR; pLas-RBS-T4Holin <STRONG>B</STRONG> - pLacI-LasR; pLas-RBS-sfGFP <STRONG>C</STRONG> - pLsr-RBS-sfGFP (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659501">BBa_K1659501</A>) <STRONG>D</STRONG> - pLsr-RBS-T4Holin (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659601">BBa_K1659601</A>) <STRONG>E</STRONG> - DsbA-DNase (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659301">BBa_K1659301</A>) <STRONG>F</STRONG> - YebF-DspB <STRONG>G</STRONG> - DspB (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659200">BBa_K1659200</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659210">BBa_K1659210</A>) <STRONG>H</STRONG> - MccS (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659100">BBa_K1659100</A>) <STRONG>I</STRONG> - Fla-DspB <STRONG>J</STRONG> - DsbA-DspB (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659201">BBa_K1659201</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659211">BBa_K1659211</A> ) <STRONG>K</STRONG> - DsbA-Art175 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659002">BBa_K1659002</A>) <STRONG>L</STRONG> - YebF-Art175 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659003">BBa_K1659003</A>) <STRONG>M</STRONG> - ArtE (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659088">BBa_K1659088</A>) <STRONG>N</STRONG> - Fla-Art175 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659001">BBa_K1659001</A>) <STRONG>O</STRONG> - Art175 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659000">BBa_K1659000</A>) <STRONG>P</STRONG> - DNase (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1659300">BBa_K1659300</A>) </P><P>
                            The symbols following the letter codes denote the type of plasmid which the sequence was being cloned into:
                            <STRONG>*</STRONG> - pSB1C3 <STRONG>#</STRONG> - pBAD33 for sequences A, B, C, and D; pBADHisB for all other sequences
              </P></DIV></DIV><DIV class="slim"><DIV class="section" id="cloning"><H2>Cloning</H2><DIV class="section" id="1"><H5>Week 1</H5><DIV id="1-1"><H3>Day 1</H3><DIV id="1-1-prep-stock"><H4>Preparation of Stock Solutions</H4><P><STONG>1. gBlocks</STONG></P><P>
                                            The gBlocks ordered from IDT arrived in the form of vials of 200µg solid DNA powder.
                                        </P><P><A href="#">(refer to BioBricks page for information on DNA sequences)</A></P><P>
                                                The gBlocks were made into 10ng/µl stock solutions in Milli-Q water for storage:
                                        </P><TABLE class="table table-striped"><THEAD><TR><TH>mass/ng</TH><TH>conc/ngµl<SUP>-1</SUP></TH><TH>final volumeµl</TH></TR><TR><TD>200</TD><TD>10</TD><TD>20</TD></TR></THEAD></TABLE><P><STONG>2. Primers</STONG></P><P>
                                            The forward and reverse primers ordered from IDT came in 32.4nmol and 34.3nmol of solid respectively.
                                        </P><UL><H6>Sequences</H6><LI>Forward - CTTTTTTGCCGGACTGC</LI><LI>Reverse - ATGATTTCTGGAATTCGC</LI></UL><P>
                                        The primers were made into 100µM stock solutions in Milli-Q water for storage:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>amt/10<SUP>-9</SUP>mol</TH><TH>conc/10<SUP>-6</SUP>M</TH><TH>final volume/10<SUP>-6</SUP>L</TH></TR><TR><TD>32.4</TD><TD>100</TD><TD>324</TD></TR><TR><TD>34.3</TD><TD>100</TD><TD>343</TD></TR></THEAD></TABLE></DIV><DIV id="1-1-prep-reaction"><H4>Preparation of Reaction Solutions</H4><P><STONG>1. gBlocks</STONG></P><P>
                                        2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 1ng/µl<SUP>-1</SUP></P><P><STONG>2. Primers</STONG></P><P>
                                        2µl of each stock solution were diluted in Milli-Q water to achieve final solution volumes of 20µl to make 10µM reaction solutions. (The solutions are labelled as &quot;Prefix primer&quot; and &quot;suffix primer&quot; in eppendorf tubes in the fridge.)
                                    </P></DIV><DIV id="1-1-pcr"><H4>Polymerase Chain Reaction</H4><P>
                                        25µl reactions were run according to the PCR protocol <A href="#">here</A></P><P>
                                        * The final concentrations of the primers were noted as they are needed to determine the annealing temperatures for the primers, which can be done using NEB’s online tool <A href="http://tmcalculator.neb.com/#!/">here.</A></P><P>
                                        ** Add components in order of decreasing volume for maximum ease-of-pipetting.
                                    </P><P>
                                        *** When reaction mixture is being made up, all components as well as the mixture itself are to be kept on ice, as the Master Mix is a high-fidelity polymerase that will recognize the primers as being incorrectly base-paired and be able to hydrolyse the primers if kept at room temperature.
                                    </P><P>
                                        **** Use Q5 enzyme in the cold room to avoid defrosting and freezing the original stock of Q5 enzyme. This could decrease the activity of Q5 enzyme. Bring ice bucket to the cold room to bring Q5 into the bench.
                                    </P><P>
                                        ***** Make sure that the primer and small amounts of DNA and primer doesn’t stick onto the side of the tube or the tip.
                                    </P><P>
                                        The reaction mixture tubes were positioned in an Eppendorf Mastercycler nexus X2 and the PCR program was run.
                                    </P><P>
                                        * DNA denaturation can be performed at 98℃ because of the high thermal stability of the Q5 polymerase
                                    </P><P>
                                        ** A PCR takes 20-30 seconds to extend a sequence by 1kb, and since our longest sequence is ~2kb the extension time was determined to be 60s per cycle.
                                    </P></DIV><DIV id="1-1-gel"><H4>Gel Electrophoresis of PCR-Amplified gBlocks</H4><P>
                                        An agarose gel was prepared according to the <A href="#">agarose prepartion protocol.</A></P><P>
                                        DNA preparation:
                                    </P><P>
                                        The contents of the PCR tubes need to be stained with a loading dye to help visualize its migration.
                                    </P><P>
                                        To each 25µl of content in each PCR tube, 10µl of blue loading dye was added.
                                    </P></DIV></DIV><DIV id="1-2"><H3>Day 2</H3><DIV id="1-2-gel-continued"><H4>Gel Electrophoresis of PCR-Amplified gBlocks (continued from 22/06)</H4><P><A href="#">gBlock sizes for reference</A></P><P>
                                        Lane 1: 10µl of <A href="#https://www.neb.com/products/n0550-quick-load-purple-2-log-dna-ladder-0-1-10-0-kb">2-Log Ladder</A></P><P>
                                        Lanes 2-15: 20µl of DNA and loading dye mixture prepared on <A href="#week-1-day-1-gel-electrophoresis-of-pcr-amplified-gblocks">22/06</A></P><P>
                                        A potential difference of 120V was applied across the electrodes (the higher the voltage, the faster the gel will run but the poor the separation will be; DNA separation is typically done in the 120-140V range) for 80 minutes. As long as DNA has passed ⅔ of the column or purple dye have passed purple area of the gel, the gel is ready to get into the EtBr.
                                    </P><P>
                                        The gel was then stained and the bands were visualized; <A href="#">protocol</A></P></DIV><DIV id="1-2-results"><H4>Results</H4><DIV class="image image-left"><P>PCR Results</P></DIV><P>
                                        The 7 bands corresponding to expected DNA sizes were excised using a razor blade for cleaning and extraction. The other sequences, along with J which only showed a very weak band, will be re-PCRed under modified conditions at a later date.
                                    </P><P>
                                        The excised gel chunks were transferred to centrifugation tubes.
                                    </P></DIV><DIV id="1-2-extraction"><H4>Extraction of DNA (PCR product) from Agarose Gel</H4><P>
                                        The extraction was performed using the E.Z.N.A. Gel Extraction Kit made by Omega Biotek, according to the <A href="#http://omegabiotek.com/store/wp-content/uploads/2014/01/D2500.D2501-Gel-Extraction-Kit-Combo-Online.pdf">Spin Protocol. </A></P><P>
                                        The excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. The heaviest chunk of gel weighed 0.16g and as such 160µl of buffer was added to each tube.
                                    </P><P>
                                        * As the tubes were spun with their lids open, they were placed such that lids pointed away from the direction of spinning.
                                    </P></DIV><DIV id="1-2-restriction-pcr"><H4>Restriction Digest of Extracted PCR product</H4><P>
                                        Restriction digest was performed using EcoRI-HF (5’) and SpeI (3’) restriction enzymes.
                                    </P><P>
                                        NEB’s Double Digest <A href="#https://www.neb.com/tools-and-resources/interactive-tools/double-digest-finder">Finder</A> was invoked and it was determined that CutSmart Buffer would be used for the digest.
                                    </P><P>
                                        The buffer was completely defrosted and shaken before use.
                                    </P><P>
                                        There is a recommended digestion protocol on <A href="#http://nebcloner.neb.com/#!/protocol/re/double/EcoRI-HF,SpeI">NEB Cloner.</A> 50µL reaction mixtures were set up with component volumes as recommended, except for the DNA where all 30µL of the extraction mixture (in elution buffer) were used in the digest. There would be up to 50ng of DNA in each tube of extraction mixture (from the gel bands).
                                    </P><P>
                                        Incubation at 37℃ was also done for 2 hours (ThermoMixer program 3) instead of the recommended 5-15 minutes, with shaking at 300rpm.
                                    </P></DIV><DIV id="1-2-restriction-enzyme"><H4>Restriction Enzyme Digest of the iGEM HQ linearised pSB-1c3</H4><P>
                                        125ng of pSB-1C3 was dissolved in 5mL Milli-Q water.
                                    </P><P>
                                        The digest was performed using the modified NEBCloner protocol.</P><P>
                                        * We did not add phosphatase because it is assumed that with different sticky ends the vector cannot religate
                                    </P></DIV><DIV id="1-2-clean-up"><H4>DNA ‘Cleanup’ using EZNA enzymatic reaction kit</H4><P>
                                        Upon completion of restriction digest incubation, the gBlocks and the plasmid backbones were purified again using the E.Z.N.A. Gel Extraction Kit. PCR products and plasmid backbones need to be purified in order to remove remaining impurities including agarose gel and restriction enzymes. At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.
                                    </P></DIV><DIV id="1-2-nanodrop"><H4>DNA Quantification using NanoDrop</H4><P>
                                        1µL water and tissue were first used to clean the stage. A blank reading was made using 1µL of elution buffer, and 1µL of each sample was measured for concentration:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sample</TH><TH>Concentration/ngµl</TH></TR><TR><TD>C</TD><TD>11.3</TD></TR><TR><TD>D</TD><TD>3.5</TD></TR><TR><TD>E</TD><TD>0.8</TD></TR><TR><TD>H</TD><TD>0.9</TD></TR><TR><TD>J</TD><TD>1.6</TD></TR><TR><TD>L</TD><TD>7.7</TD></TR><TR><TD>N</TD><TD>3.6</TD></TR><TR><TD>pSB-1C3</TD><TD>1.6</TD></TR></THEAD></TABLE></DIV><DIV id="1-2-ligation"><H4>Ligation of gBlock Sequences with pSB-1C3 Backbone</H4><P>
                                        Ligation was completed; <A href="#">protocol</A></P></DIV><DIV id="1-2-prep-e.coli-"><H4>Preparation of Competent E. coli Cells</H4><P>
                                        A sample of E. coli DH5ɑ stored at -80℃ was defrosted and inoculated in 5mL of LB in a 125mL conical flask (volume of LB 10% of flask volume so as to achieve sufficient aeration). The culture was left to grow overnight at 37℃.
                                    </P></DIV><DIV id="1-2-re-pcr"><H4>Re-PCR of DNA Sequences</H4><P>
                                        Two sets of sequences A,B,F,G, I, J, K, and M were PCRed using the same protocol as 22/06, with one set using a 55℃ annealing temperature and another using 50℃ annealing temperature.
                                    </P></DIV></DIV><DIV class="section" id="1-3"><H3>Day 3</H3><DIV id="1-3-re-pcr"><H4>Re-PCR of DNA Sequences</H4><P>
                                        Refer to protocol from <A href="#">1.0</A></P><DIV class="image image-right"><P>PCR Results</P></DIV><P>
                                        The amplified sequences A, B, F, G, I, K, J, and M were loaded with blue dye and run on agarose gel as per standard protocol
                                    </P><P>
                                        Sequences G and J showed bands corresponding to the expected sizes at both 50℃ and 55℃ annealing temperatures.
                                </P><P>
                                        The bands were excised and extracted according to standard <A href="#">protocol</A> (E.Z.N.A. Gel Extraction).
                                    </P></DIV><DIV id="1-3-plasmid-recovery"><H4>Plasmid recovery</H4><P>
                                        The concentration of pSB1C3 plasmid recovered on 23/06 according to the NanoDrop was low (1.6ng/µl). To obtain more plasmids, we restriction digested a construct made by last year’s, pSB1C3 + abijk, to recover the pSB-1C3 content from the construct.
                                    </P><P>
                                        Set up the following reaction mixture:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/µl</TH></TR><TR><TD>DNA (pSB1C3 + insert)</TD><TD>5</TD></TR><TR><TD>EcoR1-HF</TD><TD>1</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>Cutsmart buffer</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>11</TD></TR></THEAD></TABLE><P>
                                        During the incubation set up 1% agarose gel by putting 1g powder agarose in 100ml; 1% gel is more efficient at separating larger fragments
                                    </P><DIV class="image image-left"><P>Results</P></DIV><OL><LI>Incubate the mix above for 2hrs at 37℃</LI><LI>Incubate for 30mins at 95℃ to inactivate restriction enzymes (Inactivation temperature of SpeI is 80℃, and EcoR1-HF is 65℃)</LI><LI>Cool and spin (there will be condensation at the top of the eppendorf)</LI><LI>Add 1µl CIP (phosphatase) and incubate for 30mins at 37℃</LI><LI>Add loading dye</LI><LI>Load each DNA on the gel</LI><LI>Stain in EtBr for 30 + 20 mins (staining not very clear after 30 mins)</LI></OL></DIV><DIV id="1-3-extraction"><H4>Extraction of “sticky” pSB1C3 (recovered from 2014 pSB1C3+abijk) from Gel</H4><P>
                                        E.Z.N.A. Gel Extraction
                                    </P><H6>Gel weight:</H6><P>
                                        0.53g ⇒ 0.53mL Binding Buffer
                                    </P><P>
                                        30µL of elution buffer used
                                    </P></DIV><DIV id="1-3-repeat-pcr"><H4>Repeat PCR with higher DNA concentration (for the sequences that previously did not yield clear bands)</H4><P>
                                        Sequences A, B, F, G, I, J, K and M have gone through a second round of PCR as they previously failed to yield clear enough bands, and ethidium bromide staining shows that G and J have been successfully isolated as a result.
                                    </P><P>
                                        A, B, F, I, K, L and M still have not had successful PCR runs, as such an alternative PCR protocol with higher DNA concentration was attempted on them instead:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Reagent</TH><TH>Volume/µl</TH></TR><TR><TD>gBlock Template (1ng/µl)</TD><TD>5</TD></TR><TR><TD>Forward Primer</TD><TD>1.25</TD></TR><TR><TD>Reverse Primer</TD><TD>1.25</TD></TR></THEAD></TABLE><P>
                                        * A total of 5ng gBlock as opposed to 1ng at 58℃ annealing temperature present in reaction mixture; rest of the protocol is same as prescribed
                                    </P><DIV class="image image-left"><P>Results</P></DIV><P>
                                        Alternative protocol does not work.
                                    </P></DIV><DIV id="1-3-g-j"><H4>Gel extraction of G and J</H4><P>
                                        Standard E.Z.N.A. Gel Extraction protocol followed.
                                    </P><H6>Gel weights:</H6><P>
                                        J - 0.34g ⇒ 0.4mL Binding Buffer
                                    </P><P>
                                        G - 0.3g ⇒ 0.4mL Binding Buffer
                                    </P><P>
                                        30µL of elution buffer used.
                                    </P></DIV><DIV id="1-3-g-j-2"><H4>Restriction Digest of G and J</H4><P>
                                        A <A href="#">restriction digest</A> was then performed on the extracted DNA.
                                 </P></DIV><DIV id="1-3-nanodrop"><H4>NanoDrop Quantification</H4><UL><LI>J: 2.7ng/µl</LI><LI>G: 1.7 ng/µl</LI><LI>pSB-1C3: 1.4ng/ul - there is 8-9ul of this left in the freezer in drawer 4,  labelled with Psb1c3 EcoRI, SpeI and 24/06/2015</LI></UL></DIV><DIV id="1-3-ligation"><H4>Ligation</H4><P>
                                        Ligation perform according to
                                        <A href="#">protocol.</A></P></DIV><DIV id="1-3-transform"><H4>Transforming E. coli cells with Plasmid DNA </H4><P>
                                        Competent e.coli cells were first prepared, according to the protocol <A href="#">here.</A></P><P>
                                        Sample C,D,E,H,J,L and N to be transformed, according to the protocol <A href="#">here.</A></P></DIV></DIV><DIV class="section" id="1-4"><H3>Day 4</H3><DIV id="1-4-growth"><H4>Growth and Culture of Bacteria </H4><P>
                                        For the protocol please click <A href="#">here.</A></P><P>
                                        Plates incubated overnight show that vectors corresponding to C,D,E,H,J,L and N were taken up [however, later we find that ligation had failed, and so gBlocks C,D,E,H,J,L and N were never inserted]. There are at least 5-7 colonies for each biobrick.
                                    </P><P>
                                        Choose three colonies from each plate. The colony should not be too small or too large and should be reasonably spaced from the others.
                                    </P><P>
                                        Separately incubate each colony in test tubes overnight.
                                    </P><P>
                                        This process would significantly increase the amount of plasmids containing biobricks that we want. Plasmids can be extracted later.
                                    </P></DIV><DIV id="1-4-transform"><H4>Transformation of J and G (24/06 PCR Product) into competent E. coli cells</H4><P>
                                        Competent cells are already made in stocks and can be found in the -80℃ freezer: we don’t need to prepare them again. J and G comes from repeat PCR done in <A href="1-3-g-j">Day 3.</A></P></DIV><DIV id="1-4-primer"><H4>Primer Design</H4><P>
                                        Since A, B, F, G, I, K, M have repeatedly failed to be PCR-amplified, longer primers were designed to replace the old primers for the PCRing of these gene sequences.
                                    </P><P>
                                        Primers that would give the sequences new restriction sites to allow their insertion into pBAD33 and pBAD/HisB expression vectors were also designed.
                                    </P></DIV></DIV><DIV class="section" id="1-5"><H3>Day 5</H3><DIV id="1-5-extraction"><H4>Plasmid Extraction</H4><P>
                                        For the plasmid extraction protocol see <A href="#">here.</A></P><P>
                                        pSB1C3 shipping vector containing gBlocks from Day 1, as well as blank pSB-1C3 shipping vector, pBAD/HisB expression plasmids, and pBAD33 expression plasmids were extracted from overnight cultures of E. coli DH5 using E.Z.N.A. Plasmid DNA Mini Kit I.
                                    </P><P>
                                        (refer to <A href="#http://goo.gl/WhmuUb">pages 10-12</A>for Mini Kit protocol. Some special notes:
                                    </P><UL><LI>All optional steps were carried out except equilibration step.</LI><LI>After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.</LI><LI>Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.</LI><LI>The precipitate formed in following step 7 does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.</LI></UL></DIV><DIV id="1-5-quantification"><H4>Plasmid Quantification</H4><P>
                                        1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>DNA</TH><TH>c/ngμl<SUP>-1</SUP></TH><TH>DNA</TH><TH>c/ngμl<SUP>-1</SUP></TH></TR><TR><TD>pBAD/HisB</TD><TD>36.4</TD><TD>mccS (H1)</TD><TD>61.5</TD></TR><TR><TD>pBAD33</TD><TD>34.4</TD><TD>mccS (H2)</TD><TD>129.6</TD></TR><TR><TD>pSB1C3</TD><TD>142.1</TD><TD>DspB+DsbA (J1)</TD><TD>38.8</TD></TR><TR><TD>lsr + GFP (C2)</TD><TD>122.9</TD><TD>DspB+DsbA (J2)</TD><TD>46.3</TD></TR><TR><TD>lsr + GFP (C3)</TD><TD>40.3</TD><TD>DspB+DsbA (J3)</TD><TD>41.9</TD></TR><TR><TD>lsr + Holin (D1)</TD><TD>40.2</TD><TD>Art175+YebF (L1)</TD><TD>64.6</TD></TR><TR><TD>lsr + Holin (D2)</TD><TD>Art175+YebF (L2)</TD></TR><TR><TD>lsr + Holin (D3)</TD><TD>44.1</TD><TD>Art175+YebF (L3)</TD><TD>43.2</TD></TR><TR><TD>DNase+DsbA (E1)</TD><TD>77.4</TD><TD>Art175 + Fla (N1)</TD><TD>103.5</TD></TR><TR><TD>DNase+DsbA (E2)</TD><TD>43.4</TD><TD>Art175 + Fla (N2)</TD><TD>48.8</TD></TR><TR><TD>DNase+DsbA (E3)</TD><TD>40.7</TD><TD>Art175 + Fla (N3)</TD><TD>134.7</TD></TR></THEAD></TABLE></DIV><DIV id="1-5-digest"><H4>Plasmid Digest</H4><P>
                                        The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 90 minutes. For the protocol see <A href="#">here.</A></P></DIV><DIV id="1-5-gel"><DIV class="image image-right"><P>Gel layout</P></DIV><P>
                                        10µl of blue gel loading dye was added to each tube of digested plasmids. 20µl from the contents of each tube were then loaded into a 30-well agarose gel according to the following schematic:
                                    </P><P>
                                        (*the pSB-1C3 well has pSB-1C3 with abijk insert)
                                    </P><P>
                                        The ladder well has 10µl of 2-Log Ladder added into it.
                                    </P><P>
                                        A potential difference of 120V was applied across the gel for 40 minutes before it was stained with ethidium bromide for 30 minutes.
                                    </P></DIV><DIV id="1-5-analysis"><H4>Analysis of Results</H4><DIV class="image image-left"><P>Analysis</P></DIV><P>
                                        The first three lanes produced expected results: pBAD33 and pBAD/HisB being “blank” plasmid backbones, the pSB-1C3-abijk lane giving two bands corresponding to similar sizes (being pSB-1C3 and abijk insert respectively).
                                    </P><P>
                                        All the other lanes seem to be showing only products in the 2kb range, which corresponds roughly to the size of the pSB-1C3 backbone, but the sizes are not uniform. Nonetheless, we know that the plasmid extraction step was carried out correctly as it did yield products approximating what we were looking for.
                                    </P><P>
                                        If it was a matter of the ligation step carried out on 23/06 failing entirely, we should be expecting a uniform row of backbone bands. Instead, there are minor but noticeable size variations between each band which cannot be successfully explained by failed digestion/ligation. It is thus speculated that the pSB-1C3 stock we received from iGEM HQ had suffered from varying extents of DNA degradation such that the restriction enzyme cut sites on their ends were no longer.
                                    </P></DIV></DIV></DIV><DIV class="section" id="2"><H5>Week 2</H5><DIV class="secction" id="2-6"><H3>Day 6</H3><DIV id="2-6-pcr"><H4>PCR of samples C,D,E,H,J,L and N</H4><P>
                                        Samples C,D,E,H,J,L and N were PCRed according to the protocol <A href="#">here.</A></P><P>
                                        * Use the Q5 HF Master Mix that has been kept at 4℃ in the cold room and not the frozen sample because repeated freeze thaw cycles are not good for the Master Mix.
                                    </P><P>
                                        ** Remade 50µl 10uM primer stock from the 100uM stock solution (5µl primer in 45µl Milli-Q water)
                                    </P><P>
                                        *** Use DNA 10ng/µl stock solutions made up on Day 1
            </P></DIV><DIV id="2-6-gel"><H4>Gel electrophoresis of  C,D,E,H,J,L and N</H4><DIV class="image image-right"><P>Results</P></DIV><P>
                                        Refer to gel electrophoresis protocol in <A href="#">section 1.1</A></P><P>
                                        Only J showed no clear band corresponding to the expected sequence size.
                                    </P></DIV><DIV id="2-6-extraction"><H4>EZNA gel extraction of C,D,E,H,J,L and N </H4><P>
                                        Refer to section <A href="#">1.13</A> of the protocol guide.
                                    </P><P>
                                        Excise bands C,D,E,H,L, and N using razor blade, and the excised agarose chunks needed to be dissolved in a minimum of 1mL of XP2 Binding Buffer per gram of gel. For instance, the heaviest band was 0.27g, requiring 0.3ml Binding Buffer to each eppendorf tube.
                                    </P></DIV><DIV id="2-6-digest"><H4>Restriction Digest of Gel Extracted C,D,E,H,J,L and N</H4><P>
                                        Refer to section <A href="#">1.2</A> for the protocol.
                                    </P><P>                    Following the gel extraction spin protocol above, extracted PCR DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in 30/06 (Day 7).
                                    </P></DIV><DIV id="2-6-recover"><H4>Recovery of pSB-1C3 vector from 2014 pSB-1C3 + insert</H4><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume (10µl)</TH></TR><TR><TD>pSB-1C3 + insert</TD><TD>10</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>EcoR1 HF</TD><TD>1</TD></TR><TR><TD>Cutsmart</TD><TD>2</TD></TR><TR><TD>Milli-Q</TD><TD>6</TD></TR></THEAD></TABLE><OL><LI>Incubate for 2hrs 37℃</LI><LI>Heat and inactivate the restriction enzymes at 95℃ for 20mins</LI><LI>Cool down in the room temperature, spin (condensation on the side of the eppendorf from heating)</LI><LI>Add 1µl CIP and incubate at 37℃ for 30mins</LI><LI>Add 5µl loading dye</LI></OL><P>
                                        * Phosphatase is added to prevent the vector religating to insert. In the case of PCR amplification of (non-plasmid) gene sequences phosphatase does not need to be added to PCR product because those two ends are unlikely to ligate onto itself.
                                    </P><P>
                                        ** Must start preparing for agarose gel to run 2014 pSB1c3 + Insert. The gel will distinguish pSB1c3 vector from insert, allowing us to extract the vector from the gel and ligate with PCR product. Ligation is to be done in 30/06 (Day 7).
                                    </P><DIV class="image image-right"><P>Results</P></DIV><P>
                                        Gel electrophoresis of pSB1C3.
                                    </P></DIV><DIV id="2-6-ezna"><H4>EZNA gel extraction protocol on recovered pSB1c3 vector</H4><P>
                                        Refer to section <A href="#">1.13</A> for the protocol.
                                    </P><P>
                                        Start with excising the band that corresponds to the base pair length of pSB1c3 vector.
                                    </P><P>
                                        Following the gel extraction spin protocol above, extracted Vector DNA needs to be ‘cleaned up’ of restriction enzyme and agarose. The protocol for this can be found from the last enzymatic protocol in EZNA gel extraction kit. This process is to be done in Day7.
                                    </P></DIV><DIV id="2-6-growth"><H4>Growth and culture of E. coli transformed with 24/06 PCR Product (J and G)</H4><P>
                                        Refer to section <A href="#">1.6</A> for the protocol.
                                    </P><P>
                                        * LB has to be clear. The LB on the shelf was cloudy and therefore contaminated from last week so start with a new bottle
                                    </P><P>
                                        ** Colonies were of variable size which could mean that some of the colonies are contaminated. Therefore, when picking the 6 colonies, pick 2 small, 2 medium and 2 large colonies
                                    </P></DIV></DIV><DIV class="section" id="2-7"><H3>Day 7</H3><DIV id="2-7-yesterday"><H4>From yesterday</H4><P>
                                        E.Z.N.A enzymatic reaction cleanup protocol for Restriction Digest products of C, D, E, H, L, and N (<A h="" hvref="#">1.21</A>)
               <UL><LI>At the elution step, the gBlock inserts were eluted using 30µL elution buffer whereas the plasmid backbone was eluted using 50µL of it.</LI></UL></P><P>
                                        E.Z.N.A gel extraction protocol for pSB-1c3 vector isolated from 2014 pSB1C3+insert (<A href="#">1.13</A>)
                                    </P><P>
                                        E.Z.N.A enzymatic reaction protocol for pSB-1C3 gel (<A href="#">1.21</A>)
                                    </P><H6>Ligation of 29/06 PCR products to pSB-1C3 (<A href="#">1.3</A>)</H6><P>
                                        Some notes:
                                    </P><P>
                                        Our component volumes were slightly different from that in day 2 due to the different amount of gBlock/vector we had. (We divided the amount of pSB-1C3 between our 6 samples)
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/μl</TH></TR><TR><TD>Digested DNA (gBlock)</TD><TD>30</TD></TR><TR><TD>pSB-1C3</TD><TD>8</TD></TR><TR><TD>T4 DNA ligase buffer</TD><TD>5</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>6</TD></TR></THEAD></TABLE><P>
                                        Mixtures were incubated on thermomixer at 16 ℃ for 16 hours until 8.45 a.m. on 1/7/15, taking care to vortex before placing on thermomixer.
                                    </P></DIV><DIV id="2-7-extract"><H4>Plasmid Extraction using miniprep kit (<A href="#">1.7</A>)</H4><P>
                                        Sequences G [pSB-1C3] and J [pSB-1C3] were extracted from overnight cultures of E. coli DH5𝛼 using E.Z.N.A. Plasmid DNA Mini Kit I.
                                    </P><P>
                                        refer to pages <A href="http://goo.gl/WhmuUb">10-12</A> for Mini Kit protocol. Some special notes:
                                    </P><UL><LI>All optional steps were carried out except equilibration step.</LI><LI>After centrifugation in step 2, the tubes were pulsed before the excess supernatant was removed through pipetting.</LI><LI>After addition of solution I/RNAse, vortexing/vigorous shaking of the tubes should be avoided to prevent shearing of nucleus and undesirable accidental extraction of chromosomal DNA.</LI><LI>After addition of solution I/RNAse, resuspension of pellet can be done by dragging the tube along an eppendorf rack.</LI><LI>Steps 6 and 7 (involving solutions II and III need to be carried out in quick succession (adhering to the short incubation time) to ensure good results. It is advisable to do these two steps in pairs as in step 6 the tubes need to be tightly capped once solution II is added.</LI><LI>After addition of solution II, the waiting time before proceeding to the next step should not be more than 5 minutes.</LI><LI>The precipitate formed in following addition of solution III does not pellet well after centrifugation in step 8, and hence the suspension needs to be removed immediately to prevent resuspension.</LI><LI>The inversion in step 6 needs to be done gently so that genomic DNA of the bacteria are not extracted along with the desired plasmid DNA.</LI></UL><P>
                                        50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
                                    </P></DIV><DIV id="2-7-nanodrop"><H4>Plasmid Quantification: Nanodrop (<A href="#">1.22</A> Nanodrops are usually done following restriction digest and transformation)</H4><P>
                                        1µl of the each of the extracted plasmids were dropped onto the NanoDrop machine for concentration quantification. The results are as below:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>DNA</TH><TH>Conc. /ngμl<SUP>-1</SUP></TH><TH>DNA</TH><TH>Conc. /ngμl<SUP>-1</SUP></TH></TR><TR><TD>J1</TD><TD>62.4</TD><TD>G1</TD><TD>67.4</TD></TR><TR><TD>J2</TD><TD>77.2</TD><TD>G2</TD><TD>81.5</TD></TR><TR><TD>J3</TD><TD>39.5</TD><TD>G3</TD><TD>80.2</TD></TR><TR><TD>J4</TD><TD>61.4</TD><TD>G4</TD><TD>88.0</TD></TR><TR><TD>J5</TD><TD>37.9</TD><TD>G5</TD><TD>26.3</TD></TR><TR><TD>J6</TD><TD>134.3</TD><TD>G6</TD><TD>95.7</TD></TR></THEAD></TABLE></DIV><DIV id="2-7-plasmid-digest"><H4>Plasmid Restriction Digest for 24/06 PCR Product <A href="#">1.2</A></H4><P>
                                        * 0.5ul enzyme despite digesting a plasmid because test digest
                                    </P><P>
                                        **must refer to the master sheet each time
                                    </P><P>
                                        As many tubes were being handled, the required reagents were pre-mixed:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Reagent</TH><TH>Volume / μL</TH></TR><TR><TD>2.1 Buffer</TD><TD>50</TD></TR><TR><TD>Milli-Q</TD><TD>350</TD></TR><TR><TD>EcoRI-HF</TD><TD>12.5</TD></TR><TR><TD>PstI</TD><TD>12.5</TD></TR></THEAD></TABLE><P>
                                        3μL of each insert-containing plasmid was transferred into PCR tubes, and 17μL of the reagent mix was added to each PCR tube.
                                    </P><P>
                                        The plasmids were digested using restriction enzymes EcoRI-HF and PstI (NEB) at 37℃ for 120 minutes.
                                    </P></DIV><DIV id="2-7-gel"><H4>Gel Electrophoresis of Digested Plasmids from 24/06 PCR Product <A href="#">1.1</A></H4><DIV class="image image-right"><P>Results</P></DIV><P>
                                        5µl of blue gel loading dye was added to each tube of digested plasmids.
                                    </P><P>
                                        20µl from the contents of each tube loaded. Gel run under 80V for 40 minutes:
                                    </P><P>
                                        G5, J3, and J5 were also the samples that showed irregularly low concentrations when analysed with the NanoDrop.
                                    </P><P>
                                        As such, G5, J3, and J5 were discarded while the other samples, being potentially-viable BioBricks, were freeze-stored for verification by gene sequencing at a future date.
                                    </P></DIV><DIV id="2-7-stock"><H4>Preparing new primer stock solutions</H4><TABLE class="table table-striped"><THEAD><TR><TH>Primer</TH><TH>amt / 10<SUP>-9</SUP> mol</TH><TH>conc / 10<SUP>-6</SUP> M</TH><TH>Milli-Q vol / 10<SUP>-6</SUP> L</TH></TR><TR><TD>Posy Suffix Holin</TD><TD>28.7</TD><TD>100</TD><TD>287</TD></TR><TR><TD>SMAP 29 Forward</TD><TD>31.2</TD><TD>100</TD><TD>312</TD></TR><TR><TD>DspB Forward</TD><TD>27.7</TD><TD>100</TD><TD>277</TD></TR><TR><TD>Art-E Prefix</TD><TD>35.4</TD><TD>100</TD><TD>354</TD></TR><TR><TD>DNase Forward</TD><TD>18.2</TD><TD>100</TD><TD>182</TD></TR><TR><TD>YebF Forward</TD><TD>29.1</TD><TD>100</TD><TD>291</TD></TR><TR><TD>Art-E Suffix</TD><TD>19.4</TD><TD>100</TD><TD>194</TD></TR><TR><TD>Art 175 Forward</TD><TD>27.5</TD><TD>100</TD><TD>275</TD></TR><TR><TD>Fla Forward</TD><TD>26.5</TD><TD>100</TD><TD>265</TD></TR><TR><TD>Pre-prefix Holin</TD><TD>22.9</TD><TD>100</TD><TD>229</TD></TR><TR><TD>MccS Forward</TD><TD>29.2</TD><TD>100</TD><TD>292</TD></TR><TR><TD>DsbA Forward</TD><TD>27.0</TD><TD>100</TD><TD>270</TD></TR></THEAD></TABLE><P>
                                        Protocol:
                                    </P><OL><LI>Pulse spin primers</LI><LI>Add fresh Milli-Q as above table</LI><LI>Leave for 30 mins at 37C in the shaking block (allowing DNA to resuspend)</LI><LI>Dilute to 10µM</LI></OL></DIV><DIV id="2-7-pcr-new"><H4>PCR Amplification of gBlocks Using New Primers</H4><P>
                                        ***Refer to the <A href="#">Master Table</A> for detailing gBlock-primer combinations for both preparation for the gene sequences’ eventual transformation into pSB1C3 shipping vector as well as pBAD33 or pBAD/HisB expression vector.
                                    </P><P>
                                        Primer naming and explantion can be found <A href="#">here.</A></P><P>
                                        With reference to the <A href="#">Master Table</A>, standard PCR reaction mixtures (1uL 1ng/uL DNA template, 1.25uL 10uM forward primer, 1.25uL 10uM reverse primer, 9uL Milli-Q water, 12.5uL Q5 Master Mix) was set up for A*-F*, H*, I*, K*-N*, A#-E#, K#, N#, L#, H#, M#, O#, and P#.
                                    </P><P>
                                        G* and J* were not run because we already potentially have viable BioBricks for them from the plasmid extraction done earlier today <A href="#1-3">(24/06 PCR batch).</A></P><P>
                                        DspB-containing setups J#, I#, F#, G# were not run because DspB has a BspHI restriction site in the middle of its sequence. The DspB-containing gBlocks will first be QuickChange-PCRed as an insert in the shipping vector to remove the restriction site by introducing a point mutation to codon-swap before being redigested out of the shipping vector and ligated into the expression vector.
                                    </P><P>
                                        Some compromises had to be made in terms of annealing temperature as there were only 2 PCR machines available and hence only 4 different programs could be run in parallel. The annealing temperatures were set up as below:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Annealing T / ℃</TH><TH>Label</TH></TR><TR><TD>72</TD><TD>A*-F*, H*, I*, K*-N*, A#-D#</TD></TR><TR><TD>67</TD><TD>E#, K#, N#, P#</TD></TR><TR><TD>61</TD><TD>L#, H#</TD></TR><TR><TD>70</TD><TD>M#, O#</TD></TR></THEAD></TABLE><P>
                                        Reaction times and other temperatures were set up according to standard PCR protocol <A href="#">1.0</A></P></DIV></DIV><DIV class="section" id="2-8"><H3>Day 8</H3><DIV id="2-8-gel"><H4>Gel Electrophoresis of 30/06<A href="2-7"> PCR Products</A></H4><P>
                                        Protocol <A href="#">1.2</A></P><DIV class="image image-right"><P>Results</P></DIV><P>
                                        5uL loading dye added per PCR tube. Gel run under 120V for 45 minutes:
                                    </P><P>
                                        Sizes of each fragments can be referred to the <A href="#">Master Table</A>, and the following fragments that show clear bands have been excised and sent for sequencing.
                                    </P><P>
                                        Excised: C*, D*, H*, L*,M*, N*, C#, D#, E#, H#, K#, L#, M#, N#, P#
                                    </P></DIV><DIV id="2-8-digest"><H4>Restriction Digest of 30/06 PCR Products</H4><P>
                                        Protocol <A href="#">1.2</A></P><P>
                                        Grouping by required restriction enzymes -
                                    </P><H6>EcoRI-HF + PstI-HF: C*, D*, H*, L*, M*, N*</H6><P>
                                        Reagent mix (7-portion) first made up:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>CutSmart Buffer</TD><TD>35</TD></TR><TR><TD>Milli-Q</TD><TD>98</TD></TR><TR><TD>EcoRI-HF</TD><TD>3.5</TD></TR><TR><TD>PstI-HF</TD><TD>3.5</TD></TR></THEAD></TABLE><P>
                                        20µL of reagent mix was added to each tube containing 30uL DNA products.
                                    </P><H6>BspHI + PstI-HF: E#, K#, N#, L#, H#</H6><P>
                                        Reagent mix (6-portion) first made up:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>CutSmart Buffer</TD><TD>30</TD></TR><TR><TD>Milli-Q</TD><TD>84</TD></TR><TR><TD>EcoRI-HF</TD><TD>3</TD></TR><TR><TD>PstI-HF</TD><TD>3</TD></TR></THEAD></TABLE><P>
                                        20µL of reagent mix was added to each tube containing 30µL DNA products.
                                    </P><H6>BamHI: C#, D#</H6><P>
                                        Since there are only two tubes to be handles the reagents were directly added to each tube:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>3.1 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>14</TD></TR><TR><TD>EcoRI-HF</TD><TD>0.5</TD></TR><TR><TD>PstI-HF</TD><TD>0.5</TD></TR></THEAD></TABLE><H6>NcoI, PstI: M#, P#</H6><P>
                                        Since there are only two tubes to be handles the reagents were directly added to each tube:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>3.1 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>14</TD></TR><TR><TD>Ncol</TD><TD>0.5</TD></TR><TR><TD>PstI</TD><TD>0.5</TD></TR></THEAD></TABLE><P>
                                        Incubation at 37°C for 2 hours, with 300 rpm shaking. Upon completion, samples stored at -20°C.
                                    </P></DIV><DIV id="2-8-plasmid"><H4>Plasmid Design</H4><P>
                                        Quikchange plasmids for DspB were designed and ordered.
                                    </P><P>
                                        VF2 and VR plasmids, used for sequencing pSB1C3-contained inserts were also ordered.
                                    </P></DIV><DIV id="2-8-transform"><H4>Transformation of <A href="#2-6">29/06</A> ligation products (containing C, D, E, H, L, N gBlocks)</H4><P>
                                        Protocol <A href="#">1.5</A></P><P>
                                        *only 9 tubes of competent DH5alpha E. coli left in the freezer after this round of transformation, as such more will need to be made by the end of the week
          </P><P>
                                        Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
                                    </P><P>
                                        Then add volume of E. coli according to protocol, using beads to spread across petri dish.
                                    </P></DIV></DIV><DIV class="section" id="2-9"><H3>Day 9</H3><DIV id="2-9-plasmid-digest"><H4>Plasmid restriction digest - pSB1C3, pBAD33, and pBAD/HisB</H4><P>
                                        Restriciton digest of SB1C3, pBAD33, and pBAD/HisB completely, using the <A href="#">1.2</A> protocol and consulting the <A href="#">master table</A> for the correct buffer and restiction enzymes.
                                    </P><DIV class="image image-left"><P>Resultant Gel</P></DIV><P>
                                        100mL 1% agarose made up, small plate loaded and remainder left in bottle on shelf.
                                    </P><P>
                                        Total volume per tube is 35uL. Each tube’s contents split into two wells (17.5µL per well), and gel was run on 80V p.d. for 45 minutes.
                                    </P><P>
                                        * Gel can help tell if any digestion occurred at all - if no digestion occurred plasmid would still remain circular, and circular plasmid would encounter more resistance migrating through the gel matrix than linear plasmid of the same size and hence appear to be bigger than expected when compared against the ladder.
                                    </P><P>
                                        Bands were excised and the heaviest chunk was 0.66g. 670uL of XP2
                                    </P><P>
                                        Binding Buffer was added to each excised chunk to initiate E.Z.N.A. Gel Extraction protocol.
                                    </P></DIV><DIV id="2-9-clean-up"><H4>Enzymatic Reaction Clean-up - <A href="2-7">30/06</A> PCR Products</H4><P>
                                        Protocol is <A href="#">1.13</A></P><P>
                                        Retrieve products (on labelled yellow rack) from -20°C and perform clean-up.
                                    </P><P>
                                        The volume of the restriction digest reaction done on 01/07 was 50µL per tube hence according to protocol 50µL of XP2 Binding Buffer added to each tube.
                                    </P><P>
                                        Upon completion of protocol, tubes placed back into -20°C until plasmids ready for ligation.
                                    </P></DIV><DIV id="2-9-growth"><H4>Growth and Culture of Bacteria - 29/06 PCR Products</H4><P>
                                        Refer to section <A href="#">1.6</A> of the protocol guide.
                                    </P><P>
                                        *note: new bags of inoculation loops are placed next to the 37°C incubators for agar plates
                                    </P><P>
                                        LB agar plates of C, D, E, H, L, N collected.
                                    </P><P>
                                        Three colonies selected from each plate to set up three separate overnight cultures each.
                                    </P></DIV><DIV id="2-9-ligation"><H4>Ligation of <A href="#2-7">30/06</A> PCR Products(</H4><P>
                                        Refer to section <A href="#">1.3</A> of the protocol guide.
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/µ</TH></TR><TR><TD>Digested and cleaned PCR products</TD><TD>30</TD></TR><TR><TD>Digested and cleaned plasmids (pSB1C3: C*, D*, H*, L*, M*, N*; pBAD33: C#, D#;pBAD/HisB: E#, K#, N#, L#, H#, M#, P#)</TD><TD>8 for pSB1C3 10 for pBAD337 for pBAD/HisB</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD>T4 Buffer</TD><TD>5</TD></TR><TR><TD>Milli-Q</TD><TD>6 for pSB1C3 4 for pBAD33 7 for pBAD/HisB</TD></TR></THEAD></TABLE><P>
                                        Incubate reaction mixture at 16°C overnight.
                                    </P></DIV><DIV id="2-9-prep"><H4>Preparation of Competent E. coli DH5alpha</H4><P>
                                        Refer to section <A href="#">1.4</A> of the protocol guide.
                                    </P><P>
                                        *Sterile technique used
                                    </P><P>
                                        Test tube filled with 5mL LB broth.
                                    </P><P>
                                        GW opened new bag of inoculation loops to steal a colony off one of Elaine’s streaked plates (marked 29/06) and inoculated it in the filled test tube.
                                    </P><P>
                                        Test tube left to incubate at 37°C overnight.
          </P></DIV></DIV><DIV class="section" id="2-10"><H3>Day 10</H3><DIV id="2-10-plasmid"><H4>Plasmid for 29/06 PCR Products</H4><P>
                                        Refer to section <A href="#">1.7</A> of the protocol guide.
                                    </P><P>
                                        Yesterday, collected LB agar plates of C, D, E, H, L, N and Interlab are incubated overnight. Selected three colonies from each of C, D, E, H, L and N, and only one for each of the Interlab colonies (we assume that the plasmids provided by iGEM HQ for Interlab were all the same).
                                    </P><P>
                                        Therefore we shall be performing EZNA plasmid extraction on 22 samples. Follow EZNA Mini Kit I Spin Protocol (pg10-12).
                                    </P><P>
                                        50µL of elution buffer per tube was used at the end because plasmid DNA is being eluted.
                                    </P><P>
                                        Also, aspirate = pipetting!
                                    </P><P>
                                        Measured the concentration of the different samples using nanodrop
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sample</TH><TH>Concentration (ng/μl)</TH></TR><TR><TD>C1</TD><TD>46.7</TD></TR><TR><TD>C2</TD><TD>35.6</TD></TR><TR><TD>C3</TD><TD>52.7</TD></TR><TR><TD>D1</TD><TD>43.2</TD></TR><TR><TD>D2</TD><TD>95.8</TD></TR><TR><TD>D3</TD><TD>30.4</TD></TR><TR><TD>E1</TD><TD>45.1</TD></TR><TR><TD>E2</TD><TD>112.4</TD></TR><TR><TD>E3</TD><TD>46.7</TD></TR><TR><TD>H1</TD><TD>71.6</TD></TR><TR><TD>H2</TD><TD>48.2</TD></TR><TR><TD>H3</TD><TD>48.5</TD></TR><TR><TD>L1</TD><TD>48.7</TD></TR><TR><TD>L2</TD><TD>56.2</TD></TR><TR><TD>L3</TD><TD>69.2</TD></TR><TR><TD>N1</TD><TD>57.0</TD></TR><TR><TD>N2</TD><TD>47.0</TD></TR><TR><TD>N3</TD><TD>68.3</TD></TR></THEAD></TABLE><P>               Digest aliquots of each of the 22 samples, and run on gel. Refer to protocol <A href="#">1.2</A></P><P>
                                        From gel, determine the degree of success of these samples.
                                    </P></DIV><DIV id="2-10-prep"><H4>Preparation of Competent E. coli DH5alpha</H4><P>
                                        Refer to protocol <A href="#">1.4</A></P></DIV><DIV id="2-10-transform"><H4>Transformation of 30/06 PCR Products (Ligated on 02/07) (Protocol in section 1.5)</H4><P>
                                        Refer to protocol in section 1.5
                                    </P><P>
                                        Antibiotics:
                                    </P><UL><LI>
                                            pSB1C3, pBAD33 - Chl
                                            <P>
                                                for C*, D*, H*, L*, M*, N*, C#, D#

                                            </P></LI><LI>
                                            pBAD/HisB - Amp
                                            <P>
                                                for E#, K#, N#, L#, H#, M#, P#
                                            </P></LI></UL><P>
                                        Raffy, Leon: Moving of cells plated on 3/7/2015 from the incubator to the cold room. All pSB1C3 plates had reasonable colony density. A significant proportion of pBAD/HisB plated had no colonies. Plates with no colonies will be reincubated for half a day on 6/7/2015.
                                    </P></DIV></DIV></DIV><DIV class="section" id="3"><H5>Week 3</H5><DIV class="section" id="3-11"><H3>Day 11</H3><DIV id="3-11-prep"><H4>Preparation of Overnight Cultures for 30/06 PCR Products</H4><TABLE class="table table-striped"><THEAD><TR><TH>Plate</TH><TH>Colony</TH><TH>Plate</TH><TH>Colony</TH></TR><TR><TD>C#</TD><TD>1</TD><TD>C#e</TD><TD>1</TD></TR><TR><TD>D#</TD><TD>0</TD><TD>D#e</TD><TD>0</TD></TR><TR><TD>E#</TD><TD>2</TD><TD>E#e</TD><TD>2</TD></TR><TR><TD>H#</TD><TD>0</TD><TD>H#e</TD><TD>3</TD></TR><TR><TD>K#</TD><TD>0</TD><TD>K#e</TD><TD>0</TD></TR><TR><TD>L#</TD><TD>0</TD><TD>L#e</TD><TD>0</TD></TR><TR><TD>M#</TD><TD>0</TD><TD>M#e</TD><TD>0</TD></TR><TR><TD>N#</TD><TD>0</TD><TD>N#e</TD><TD>0</TD></TR><TR><TD>P#</TD><TD>1</TD><TD>P#e</TD><TD>0</TD></TR></THEAD></TABLE><TABLE class="table table-striped"><THEAD><TR><TH>Plate</TH><TH>Colony</TH><TH>Plate</TH><TH>Colony</TH></TR><TR><TD>C*</TD><TD>Yes</TD><TD>C*e</TD><TD>Yes</TD></TR><TR><TD>D*</TD><TD>3</TD><TD>D*e</TD><TD>0</TD></TR><TR><TD>H*</TD><TD>Yes</TD><TD>H*e</TD><TD>Yes</TD></TR><TR><TD>L*</TD><TD>Yes</TD><TD>L*e</TD><TD>Yes</TD></TR><TR><TD>M*</TD><TD>0</TD><TD>M*e</TD><TD>3</TD></TR><TR><TD>N*</TD><TD>Yes</TD><TD>N*e</TD><TD>Yes</TD></TR></THEAD></TABLE><P>
                                        Colonies grew for L* and N*, but because the MiniPrep run on 03/07 already showed the correct bands for them, their colonies were not cultured overnight.
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sequence</TH><TH>Colonies picked</TH></TR><TR><TD>C*</TD><TD>3</TD></TR><TR><TD>D*</TD><TD>3</TD></TR><TR><TD>H*</TD><TD>3</TD></TR><TR><TD>M*</TD><TD>3</TD></TR><TR><TD>C#</TD><TD>2</TD></TR><TR><TD>E#</TD><TD>4</TD></TR><TR><TD>H#</TD><TD>3</TD></TR><TR><TD>P#</TD><TD>1</TD></TR></THEAD></TABLE><P>
                                        See <A href="#">protocol guide</A> to find out how to make overnight cultures.
                                    </P></DIV><DIV id="3-11-pcr"><H4>PCR amplification of A,B,D,E,F,I,K,M for pSB1C3</H4><P><STRONG>1. Primer used:</STRONG></P><P>* primer pair (pre prefix holin and post suffix holin)</P><P>Comments:</P><P>
                                        Attempting A, B, F and I again as they have never been successfully amplified before.Attempting A, B, F and I again as they have never been successfully amplified before.
                                    </P><P>
                                        NB: F and I are DspB-containing gBlock which we eventually hope to QuikChange to get rid of the BspHI restriction site in them and put them into pBAD/HisB expression vector
                                    </P><P><STRONG>2. PCR all the # sequences with the appropriate primers</STRONG></P><P>
                                        Follow <A href="#">1.0</A> PCR protocol, and primer list is in the <A href="#">Master Table</A>.
                                    </P></DIV><DIV id="3-11-digest"><H4>Restriction digest for pSB1C3, pBAD33 and pBAD/HisB</H4><DIV class="image image-right"><P>Gel photo</P></DIV><P>
                                        Follow <A href="#">restriction digest</A> protocol.
                                    </P><P>
                                        pSB1C3 and pBAD/HisB were successfully digested and eluted in 1% gel. Band for pBAD33 could not be found.
                                    </P><P>
                                        Excised bands were stored in -20℃ for cleanup tomorrow.
                                    </P></DIV><DIV id="3-11-primer-prep"><H4>Primer Preparation</H4><P>
                                        Preparing new primers (solution and dilution) - VF2, VR, QuikChange forward, QuikChange reverse.
                                    </P><P>
                                        First make 100uM out of the freeze dry solid:
                                    </P><OL><LI>Spin down solid</LI><LI>The amount of primer is given in y nmol for each tube. Add 10y µL of water to each tube to make 100µM.</LI><LI>Shake/vortex and mix evenly.</LI></OL><P>
                                        For VF2, VR - take 3.2ul out of the 100µM stock and dilute with 96.8µL Milli-Q to make 3.2pmol/uL sequencing primer solution.
                                    </P><P>
                                        For the QuikChange primers, dilute to 10uM as per normal.
                                    </P></DIV><DIV id="3-11-sequencing"><H4>Sequencing of BioBricks</H4><P>
                                        5µl of each plasmid along with 100uL of 3.2pmol/uL sequencing primer sent to SourceBioScience for sequencing:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sequence</TH><TH>Label Assigned</TH></TR><TR><TD>C 3</TD><TD>pSBLsrGFP3</TD></TR><TR><TD>D 2</TD><TD>pSBLsrHolin2</TD></TR><TR><TD>E 2</TD><TD>SBDNaseDsbA2</TD></TR><TR><TD>G 6</TD><TD>pSBDspB6</TD></TR><TR><TD>H 1</TD><TD>pSBMccS1</TD></TR><TR><TD>J 2</TD><TD>pSBDspBDsbA2</TD></TR><TR><TD>L 3</TD><TD>pSBArt175YebF3</TD></TR><TR><TD>N 3</TD><TD>SBArt175Fla3</TD></TR></THEAD></TABLE></DIV><DIV id="3-11-gel"><H4>Gel Electrophoresis of PCR Products</H4><P>
                                        Correct bands were obtained for D*, C#, G#, H#, J#, K#, L#, N#. Still awaiting reply on insert length for O# and P# to determine whether the bands are correct.
                                    </P></DIV></DIV><DIV class="section" id="3-12"><H3>Day 12</H3><DIV id="3-12-nanodrop"><H4>NanoDrop Analysis of Plasmids Digested on 06/07 (carried out after gel extraction)</H4><TABLE class="table table-striped"><THEAD><TR><TH>Plasmid</TH><TH>Conc / nguL<SUP>-1</SUP></TH></TR><TR><TD>pSB1C3</TD><TD>5, 9.5</TD></TR><TR><TD>pBAD/HisB</TD><TD>1.8, 1.6</TD></TR></THEAD></TABLE><P>
                                        Since pBAD/HisB is very low in concentration, more of it will be digested.
                                    </P></DIV><DIV id="3-12-digest"><H4>Restriction Digest of pBAD33</H4><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>Plasmids</TD><TD>10</TD></TR><TR><TD>BamHI</TD><TD>1</TD></TR><TR><TD>Milli-Q Water</TD><TD>7</TD></TR><TR><TD>3.1 Buffer</TD><TD>2</TD></TR></THEAD></TABLE><OL><LI>37℃, 2 hours (heat up another block to 95℃)</LI><LI>Heat inactivation, 95℃, 20 minutes</LI><LI>Melt, cool, and pour 1% agarose</LI><LI>Pulse spin</LI><LI>CIP dephosphorylation (1 µL CIP), 37℃, 30 minutes</LI><LI>Load dye and run gel</LI></OL><P>
                                        pBAD/HisB digest - 6uL water, 1µL NcoI, 1µL PstI, 2µL 3.1, 10µL plasmid
                                    </P></DIV><DIV id="3-12-extraction"><H4>Gel extraction of digested plasmids</H4><P><STRONG>Plasmids digested on 6/7:</STRONG></P><P>
                                        400µL of Binding Buffer added to each tube containing excised band.
                                    </P><P>
                                        After first elution, concentrations turned out to be low (see NanoDrop table at the top of this document), hence a second elution was done, again using 50µL of elution buffer.
                                    </P><P><STRONG>Plasmids digested on 7/7:</STRONG></P><P>
                                        340uL of Binding Buffer; one elution with 50µL.
                                    </P></DIV><DIV id="3-12-plasmid-extract"><H4>Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)</H4><P>
                                        Follow EZNA DNA Mini Kit I Spin Protocol - eluted 50μl
                                    </P><P>
                                        freezer: expression vector ones (the ones with #) on orange rack in bottom drawer (will be dealt with tomorrow), psb vector ones (the ones with *) in white box with other psb constructs in top drawer
                                    </P><P>
                                        NanoDrop analysis:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sequence</TH><TH>Conc / ngµL<SUP>-1</SUP></TH><TH>Sequence</TH><TH>Conc / ngµL<SUP>-1</SUP></TH></TR><TR><TD>C* 1</TD><TD>58.3</TD><TD>M* 3</TD><TD>49.2</TD></TR><TR><TD>C* 2</TD><TD>67.5</TD><TD>C# 1</TD><TD>64.5</TD></TR><TR><TD>C* 3</TD><TD>66.0</TD><TD>C# 2</TD><TD>25.1</TD></TR><TR><TD>D* 1</TD><TD>6.5</TD><TD>E# 1</TD><TD>37.8</TD></TR><TR><TD>D* 2</TD><TD>43.3</TD><TD>E# 2</TD><TD>43.1</TD></TR><TR><TD>D* 3</TD><TD>56.7</TD><TD>E# 3</TD><TD>30.5</TD></TR><TR><TD>H* 1</TD><TD>30.0</TD><TD>E# 4</TD><TD>67.3</TD></TR><TR><TD>H* 2</TD><TD>45.9</TD><TD>H# 1</TD><TD>36.5</TD></TR><TR><TD>H* 3</TD><TD>44.5</TD><TD>H# 2</TD><TD>28.8</TD></TR><TR><TD>M* 1</TD><TD>51.1</TD><TD>H# 3</TD><TD>37.3</TD></TR><TR><TD>M* 2</TD><TD>53.1</TD><TD>P# 1</TD><TD>179.2</TD></TR></THEAD></TABLE></DIV><DIV id="3-12-gel-2"><H4>Gel electrophoresis of Plasmid Extraction from 06/07 Overnight Cultures (30/06 PCR)</H4><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                        Electrophoresis of 10uL plasmid with 5uL loading dye
                                    </P><P>
                                        N.B. forgot to digest - will do this on 08/07/15 (tomorrow)
                                    </P></DIV><DIV id="3-12-sequence"><H4>Analysis of sequencing data from 06/07/15</H4><P>
                                        Of the stuff sent for sequencing (03/07 miniprep):
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sample</TH><TH>Results</TH><TH>Action Point</TH></TR><TR><TD>C*3</TD><TD>corresponds to pSB1C3 Lsr GFP</TD><TD>-</TD></TR><TR><TD>D*2</TD><TD>Corresponds to pSB1C3 Lsr GFP (does not correspond to pSB1C3 Lsr Holin as expected - contamination in all three D* wells were already apparent in gel (see 06/07))</TD><TD>Wrong insert, need to redo from scratch (start from PCR again)</TD></TR><TR><TD>E*3</TD><TD>DsbA DNAse missing, sequencing only gives blank pSB1C3 vector</TD><TD>Send another miniprep product in the triplicate (E*1 or E*2) for sequencing.</TD></TR><TR><TD>G*6</TD><TD>Forward sequence good enough to confirm (~70% length of DspB sequence) that it’s pSB1C3 DspB, but reverse sequence dropped off too early for double confirmation</TD><TD>Ask Source Bioscience to redo VR (reverse) sequence</TD></TR><TR><TD>H*1</TD><TD>Corresponds to pSB1C3 MccS</TD><TD>-</TD></TR><TR><TD>J*2</TD><TD>Corresponds to DsbA DspB, but base 2964 had a C-&gt;A point mutation</TD><TD>Send another miniprep product in the triplicate (J*1 or J*3) for sequencing</TD></TR><TR><TD>L*3</TD><TD>Corresponds to pSB1C3 Art-175 YebF</TD><TD>-</TD></TR><TR><TD>N*3</TD><TD>Corresponds to pSB1C3 Art-175 Fla</TD><TD>-</TD></TR></THEAD></TABLE><P>
                                        Results: 4, potentially 5 BioBricks successfully made in [pSB1C3] format. Successful BioBricks to be stored in separate box in preparation for sending to Registry, creating Database page etc.
                                    </P></DIV><DIV id="3-12-ligation"><H4>Ligation of 06/07 PCR to Appropriate Digested Plasmids</H4><P>
                                        pBAD33: C#1, C#2 (one of these were wrongly digested using CutSmart instead of 3.1), D# (previously digested using wrong buffer), D#3.1
                                    </P><P>
                                        pSB1C3: D*
                                    </P><P>
                                        pBAD/HisB: E#, G#, H#, J#, K#, L#, N#, O# (previously digested using wrong buffer), O#3.1, P# (previously digested using wrong buffer), P#3.1
                                    </P><P>
                                        Refer to section  of the protocol guide.
                                    </P><P>
                                        Left overnight at 16C.
                                    </P></DIV></DIV><DIV class="section" id="3-13"><H3>Day 13</H3><DIV id="3-13-to-do"><H4>To Do</H4><P>
                                        Prepare plates for transformation
                                    </P><P>
                                        Transformation of ligated plasmids
                                    </P><P>
                                        Gradient PCR:
                                    </P><UL><LI>Group A: A*, B*, F*, I*, K*</LI><LI>Group B: A#, B#</LI><LI>Group C: I#, F# </LI></UL></DIV><DIV id="3-13-to-note"><H4>To Note</H4><P>              Re-sequenced VR read and DspB confirmed as correct = another biobrick.
                                    </P></DIV><DIV id="3-13-quick-change"><H4>QuikChange PCR on DspB</H4><P>
                                        Standard PCR protocol <A href="#">1.0</A></P><UL><LI>72C annealing temp</LI><LI>2 min extension time</LI></UL><P>
                                        Add 1ul DpnI and incubate at 37C for two hours
                                    </P></DIV><DIV id="3-13-transform"><H4>Transformation</H4><P>
                                        Add 1ul PCR product to 100ul competent cells. Add the remaining PCR product (24ul) to another eppendorf of 100ul competent cells
                                    </P><P>
                                        Follow standard transformation protocol </P></DIV><DIV id="3-13-digest"><H4>Restriction Digest</H4><P>
                                        Incubate at 37oC for 60 mins - BamHI has star activity so it cannot be left for long
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/µL</TH><TH>Component</TH><TH>Volume/µL</TH><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>C#</TD><TD>5</TD><TD>C*/D*/H*/M*</TD><TD>5</TD><TD>E#/H#/P#</TD><TD>5</TD></TR><TR><TD>3.1 NEB 10x buffer</TD><TD>2</TD><TD>2.1 NEB 10x buffer</TD><TD>2</TD><TD>3.1 NEB 10x buffer</TD><TD>2</TD></TR><TR><TD>Bam HI</TD><TD>0.5</TD><TD>PstI</TD><TD>0.5</TD><TD>Bam HI</TD><TD>0.5</TD></TR><TR><TD>(no 2nd RE)</TD><TD>-</TD><TD>EcoRI-HF</TD><TD>0.5</TD><TD>EcoRI-HF</TD><TD>0.5</TD></TR><TR><TD>MilliQ</TD><TD>12.5</TD><TD>MilliQ</TD><TD>12</TD><TD>MilliQ</TD><TD>12</TD></TR></THEAD></TABLE></DIV><DIV id="3-13-gel"><H4>Gel electrophoresis of restriction digest</H4><DIV class="image image-left"><P>Gel Photo</P></DIV></DIV><DIV id="3-13-sequencing"><H4>Sent for sequencing:</H4><TABLE class="table table-striped"><THEAD><TR><TH>Sequence</TH><TH>Concentration</TH><TH>Label Assigned</TH></TR><TR><TD>E* 1</TD><TD>45.1</TD><TD>SBDNaseDsbA1</TD></TR><TR><TD>J* 1</TD><TD>62.4</TD><TD>SBDspBDsbA1</TD></TR><TR><TD>C# 1</TD><TD>64.5</TD><TD>33LsrGFP1</TD></TR><TR><TD>H# 3</TD><TD>37.3</TD><TD>BMccS3</TD></TR><TR><TD>E# 4</TD><TD>67.3</TD><TD>BDNaseDspA4</TD></TR><TR><TD>P# 1</TD><TD>179.2</TD><TD>BDNase1</TD></TR><TR><TD>D* 3</TD><TD>56.7</TD><TD>SBLsrHolin3</TD></TR><TR><TD>H* 2</TD><TD>45.9</TD><TD>SBMccS2</TD></TR><TR><TD>M* 2</TD><TD>53.1</TD><TD>SBArtE2</TD></TR></THEAD></TABLE></DIV><DIV id="3-13-extra"><P>
                                        Transformation of E. coli with ligation products from day12 (06/07 PCR and interlab sequences) using the standard protocol described by diagrams for day 3. Transformed E. coli then plated and incubated overnight
                                    </P></DIV></DIV><DIV class="section" id="3-14"><H3>Day 14</H3><DIV id="3-14-group-a"><H4>Group A</H4><P>
                                        C*, G*, H*, L*, N*, M* are biobricks
                                    </P><P>
                                        G*, J*, D* Sent for resequencing
                                    </P><P>
                                        A*, B*, F* I* K* - gradient PCR - currently in the freezer, run gel electrophoresis in afternoon, doing staining tomorrow
                                    </P><P>
                                        E*, J*, D* - redid PCR trying Q5 mastermix from 2014 and 2015 (gel poured for electrophoresis tomorrow)
                                    </P></DIV><DIV id="3-14-group-b"><H4>Group B</H4><P>C# - redo PCR (sequencing didn't not give expected resuts)</P><P>A#, B# - gradient (already currently in freezer - need to put onto gel)</P><P>C#,D# - colonies</P></DIV><DIV id="3-14-group-c"><H4>Group C</H4><P>E#, H#, P# - successful sequencing (with exception of point mutation in plasmid)</P><P>I# and F# - gradient PCR </P><P>M#, O# - redo from PCR</P><P>J#, K#, L#, N#, G#, P#, O# - colonies</P><P>G# - wait for G* to come back and QC if successful</P><P>(if QC doesn’t work again, treated with dpn1, gel, excise, clean and add buffer and ligase, transformation)</P><P>Is DH5alpha suitable for QC?</P><P> week, grow up competent cells for 2 E coli strains:</P><UL><LI>FliC deletion strain - FliC exported through Fla export system </LI><LI>MG1655 - close to WT E. coli - more tolerant (characterise in MG1655)</LI></UL><P>* primers are the templates for J#, I#, F#, G#</P></DIV><DIV id="3-14-sequence"><H4>Results of sequencing:</H4><TABLE class="table table-striped"><THEAD><TR><TH>Sequence</TH><TH>Result</TH></TR><TR><TD>E* 1</TD><TD>Forward read aligns, ask to repeat VR. Aligns to MccS (which is actually H*). Check the size of the PCR fragment. Redo PCR</TD></TR><TR><TD>J* 1</TD><TD>Forward and reverse reads align but highlight a point mutation at 847 but looking at the chromatogram there is also a peak for G.Send another, preempt another won’t work so start PCR</TD></TR><TR><TD>C# 1</TD><TD>Both reads are of insufficient quality. Look at the gel (sizes are off - two bands entirely diff size but both &gt;2k - wrong)</TD></TR><TR><TD>H# 3</TD><TD>Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine. Point mutation in plasmid stock? </TD></TR><TR><TD>E# 4</TD><TD>Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine.</TD></TR><TR><TD>P# 1</TD><TD>Forward read aligns but reverse read of insufficient quality. Potential deletion at T297 between promoter and RBS but otherwise should be fine.</TD></TR><TR><TD>D* 3</TD><TD>Both reads align but there is a deletion at 2205. Send another.</TD></TR><TR><TD>H* 2</TD><TD>Both reads align.</TD></TR><TR><TD>M* 2</TD><TD>Both reads align.</TD></TR></THEAD></TABLE><P>
                                        G* has a 70bp insert at the start of the coding region (even though it’s been quikchanged). Send another.
                                    </P></DIV><DIV id="3-14-pcr"><H4>PCR To Redo</H4><P>E*, J*, D*</P></DIV><DIV id="3-14-to-sequence"><H4>To Sequence</H4><P>J* (not J1, J6 or J2) - J4</P><P>D* (not D3, D1 too low) - D2 </P><P>G* (G4, G3, G2) - G4</P></DIV><DIV id="3-14-overnight"><H4>To Overnight</H4><P>The following tubes were put in incubator with selected colonies for over night culture (1x unless specified otherwise):</P><TABLE class="table table-striped"><TBODY><TR><TD>3x 3.1 C#</TD><TD>G#</TD><TD>3x P#</TD><TD>3.1 O#</TD></TR><TR><TD>3x pBAD33</TD><TD>C#</TD><TD>3x 3.1 D#</TD><TD>22A</TD></TR><TR><TD>pSB1C3</TD><TD>D#</TD><TD>3x 22K</TD><TD>20K</TD></TR><TR><TD>3x K#</TD><TD>L#</TD><TD>3x N#</TD><TD>3x J#</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="section" id="3-15"><H3>Day 15</H3><DIV id="3-15-group-a"><H4>Group A</H4><P>
                                        Check sequencing data for G*, J*, D* :
                                    </P><UL><LI>G* = large anomalous insert at start of coding sequence → send another (G3) but also redo PCR verdict: G*3 sequencing data correct - no need further PCR</LI><LI>J*4 = biobrick</LI><LI>D* = large deletion in sequencing data → new batch currently being PCRed</LI></UL><P>
                                        (A,B,F,I,K)* - stain gel → visualize gel → 09/07 gradient PCR turned out to have no amplified products at all
                                    </P><P>
                                        (E,J,D)* -  load dye → run gel → stain gel → visualize gel → only D* yielded bands → excise bands for (10/07) PCR products → cleanup → digest
                                    </P><P>
                                        Quikchange J*4 under gentle conditions to remove BspHI restriction site within its DspB sequence → wrong procedure was carried out (forgot to do DnpI enzyme incubation, and was PCRed despite quikchange products being meant to be directly transformed) → need to redo
                                    </P><P>
                                        pSB1C3 plasmids made from (09/07) overnight cultures → miniprep → digest with EcoRI, SpeI → gel poured and awaiting electrophoresis on Monday
                                    </P><P>
                                        E* was PCRed again using both standard and DMSO PCR protocol → but gel was frozen before excision and hence messed up → need to redo
                                    </P></DIV><DIV id="3-15-quick"><H4>J*4 QuikChange</H4><P>
                                        Standard PCR protocol using QuikChange primers and following conditions:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Stage</TH><TH>Temperature</TH><TH>Time </TH></TR><TR><TD>Initial denaturation </TD><TD>98</TD><TD>30sec</TD></TR><TR><TD>Denaturnation</TD><TD>98</TD><TD>10sec</TD></TR><TR><TD>Annealing</TD><TD>65</TD><TD>30sec</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>3min</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>2min</TD></TR></THEAD></TABLE></DIV><DIV id="3-15-group-b"><H4>Group B</H4><P>
                                        (A,B)# - stain gel → visualize gel → no bands for (09/07) gradient PCR products
                                    </P><P>
                                        D# - load dye → run gel → stain gel → visualize gel → excise bands for (10/07) PCR products → extract → digest → not cleaned up yet
                                    </P><P>
                                        (C,D)# - miniprep from (09/07) overnight culture → 10uL digested but not gelled yet
                                    </P><P>
                                        pBAD33 in (09/07) overnight culture - miniprep → digest → visualize → excise → fresh pBAD for ligation
                                    </P></DIV><DIV id="3-15-group-c"><H4>Group C</H4><P>
                                        (I, F)# - stain gel → visualize gel → no bands for (09/07) gradient PCR products
                                    </P><P>
                                        (M,O)# - load dye → run gel → stain gel → visualize gel → excise bands for (10/07) PCR products → extract → digest → not cleaned up yet
                                    </P><P>
                                        (J,K,L,N,G,P,O)# - miniprep from (09/07) overnight culture → 10uL digested but not gelled yet
                                    </P><P>
                                        More pBAD/HisB needs to be made → heat shock → plate colonies
                                    </P></DIV><DIV id="3-15-weekend"><H4>Over the Weekend</H4><P>
                                        Overnight cultures for plasmids setup
                                    </P><P>
                                        Quikchange for J*4 - parameter testing (previously we used the full 61.4ng/uL for quikchange - may be the reason why it never worked (for Q5MM’s efficacy, NEB recommends a total of 1-25ng of plasmid DNA in a 25µL reaction mixture)
                                    </P><P>
                                        1µL of 61.4ng/µL J*4 diluted into 60µL total volume to make 1.02ng/µL J*4
                                    </P><P>
                                        3 setups were made - 1, 5, 10µL of 1.02ng/µL J*4 at two annealing temperatures (65 and 70 degC) - making 6 tubes 3 minute extension time
                                    </P><P>
                                        Also E* and E*DMSO PCR
                                    </P></DIV></DIV></DIV><DIV class="section" id="4"><H5>Week 4</H5><DIV class="section" id="4-16"><H3>Day 16</H3><DIV id="4-16-leon"><H4>Leon</H4><P>
                                        Minipreps for plasmids → concentrations noted on each tube
                                    </P><UL><LI>10µL of pSB1C3 digested</LI><LI>15.1µL of pBAD33</LI><LI>30µL of pBAD/HisB</LI></UL><P>
                                        Run E*, E*DMSO on the gel → bands very faint → both bands combined into one tube for gel extraction
                                    </P><P>
                                        Split into two tubes at the Binding Buffer step → at the end, one tube had 7ng/µL and the other had 9ng/uL of DNA
                                    </P><P>
                                        React J*4 QC with DpnI: all 24µL of all 6 tubes of J*4 from the QC PCR reaction placed into digest mixture (made up according to DpnI search using NEBCloner protocol desginer). Buffer = CutSmart; 2 hours incubation
                                    </P><P>
                                        Clean up digests done on Friday → cleaned up and ready for ligation
                                    </P><P>
                                        Digest plasmids and both tubes of E*
                                    </P><P>
                                        Day 1 (overnight) procedure carried out for the preparation of competent DH5alpha, MG1655, deltaFliC E. coli
                                    </P></DIV><DIV id="4-16-group-a"><H4>Group A</H4><DIV class="image image-left"><P>Failed Gel</P></DIV><P>
                                        Helen and Drizzy: gradient PCR of A#, B#, F#, I#
                                    </P><P>
                                        A# used up the last of the 2014 Q5 MasterMix
                                    </P><DIV class="image image-right"><P>Failed Gel</P></DIV><P>
                                        Mabel and Ria: normal PCR of A*, B*, F*, I*, K*
                                    </P><P>
                                        Mabel: prefix + post suffix holin primers, annealing temp = 58℃ (so set PCR to 56℃)
                                    </P><P>
                                        Ria: pre suffix holin + suffix primers, annealing temp = 63oC (set PCR to 61oC)
                                    </P><P>
                                        All failed - no DNA bands
                                    </P></DIV><DIV id="4-16-ria"><H4>Ria</H4><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        First gel of digests from Day 15 overran, re-do digest and gel electrophoresis etc.
                                    </P></DIV></DIV><DIV class="section" id="4-17"><H3>Day 17</H3><DIV class="section" id="4-17-plasmid"><H4>Plasmid Backbones</H4><DIV id="4-17-plasmid-de"><H6>Dephosphorylation</H6><P>
                                            1µL CIP added to heat inactivated pSB1C3, pBAD33, pBAD/HisB; incubation for 1 hour.
                                        </P></DIV><DIV id="4-17-plasmid-clean-up"><H6>Enzymatic Reaction Cleanup</H6><P>
                                            After cleanup, NanoDrop shows conc all are about 30ng/µL.
                                        </P></DIV><DIV id="4-17-plasmid-ligation"><H6>Ligation</H6><P>
                                            All 29µL of each insert used, with 7µL of plasmid.
                                        </P></DIV></DIV><DIV class="section" id="4-17-june"><H4>June</H4><DIV id="4-17-june-pcr-1"><H6>10/07 PCR Batch (D*, D#, M#, O#)</H6><P>
                                            Ligation set up and left overnight.
                                        </P></DIV><DIV id="4-17-june-pcr-2"><H6>13/07 PCR Batch (E*, E* DMSO)</H6><P>
                                            Enzymatic reaction cleanup completed. Ligation set up and left overnight.
                                        </P></DIV></DIV><DIV class="section" id="4-17-prep"><H4>Preparation of Competent Cells</H4><P>
                                        DH5alpha, MG1655, deltaFliC
                                    </P><P><A href="#">Protocol</A></P><P>
                                        OD when incubation stopped:
                                    </P><UL><LI>DH5alpha - 0.302</LI><LI>MG1655 - 0.345 (achieved in one hour); 0.902 (1.5 hours - overshot because used more than 1mL, also wildtypes seem to grow faster than DH5alpha)</LI><LI>deltaFliC - 0.350; 0.805</LI></UL><P>
                                        The 0.902 MG1655 are labelled MG; the 0.345 MG1655 are labelled MGA and MGB.
                                    </P><P>
                                        The 0.805 deltaFliC are labelled with an extra dot • on the tubes.
                                    </P></DIV><DIV class="section" id="4-17-sequence"><H4>Sequencing</H4><TABLE class="table table-striped"><THEAD><TR><TD>Construct</TD><TD>Successful</TD><TD>Label assigned (1st sequence)</TD></TR></THEAD><TBODY><TR><TD>pBADArt175DsbA</TD><TD>K2</TD><TD>Art175DsbA2</TD></TR><TR><TD>pBADArt175Fla</TD><TD>N3</TD><TD>Art175Fla3</TD></TR><TR><TD>pBADArt175</TD><TD>O1</TD><TD>Art1751</TD></TR><TR><TD>pBADDNase</TD><TD>P2</TD><TD>DNase2</TD></TR></TBODY></TABLE></DIV><DIV class="section" id="4-17-pcr-1"><H4>PCR of C#, D#, L#</H4><P>
                                        Worked beautifully.
                                    </P></DIV><DIV class="section" id="4-17-pcr-2"><H4>PCR of A*, B*, I*, F*, K*, A#, B#, F#, I#</H4><P>
                                        K*: given to Chris for Phusion polymerase, will get results by the end of the week
                                    </P><P>
                                        Run gel of gBlock stock on 1% agarose with SYBR safe DNA Gel Stain in LAB buffer - no bands i.e. the working stock possibly has no DNA!
                                    </P><P>
                                        Nanodrop of gBlock stock: A = 7.9 ng/μl, C = 7.7 ng/μl (i.e. there is DNA)
                                    </P><P>
                                        N.B. A, B, I and F are our 4 largest gBlocks which we cannot amplify.
                                    </P><P>
                                        Driscoll's work: A, B, I and F attempted on a MgCl2 gradient of 0.5mM, 1mM, 2.5mM and 5mM. Ran PCR but was told that there was no DNA in the solutions so didn’t run gel, the PCR products currently sit in freezer but don’t think it’s worth it to run a gel as there is clearly nothing or not enough in the G-block stocks. The plan was to also to run a single PCR with A,B, F and I with 1% Triton X 100 and with double primer concentration but didn’t get around to doing it after hearing the results of the straight G block gel.
                                    </P><P>
                                        Helen: A# attempted with 10% DMSO. B# attempted with 10% glycerol. F# attempted with 5% formamide. I# attempted with 10% formamide. Ran a gel of these PCRs, and none showed any bands.
                                    </P></DIV></DIV><DIV class="section" id="4-18"><H3>Day 18</H3><DIV id="4-18-plates"><H4>Check Plates</H4><P>
                                        As expected, MGA and MGB are more competent (grew more extensively on antibiotic plate) than MG(OD=0.9 when taken out of flask)
                                    </P><P>
                                        FliC0.35 also more competent than FliC0.8
                                    </P></DIV><DIV id="4-18-transform"><H4>Transform Ligated Products into DH5alpha</H4><P>
                                        E*, E*DMSO, D*, D#, M#, O#
                                    </P></DIV><DIV id="4-18-ligation"><H4>Ligation</H4><P>
                                        Set up ligations for C#, D#, L# (PCR batch 14/07)
                                    </P><P>
                                        Mixed up the samples so will be ligating tomorrow.
                                    </P></DIV><DIV id="4-18-overnight"><H4>Overnight Culturing</H4><P>
                                        QC J*4 (3) grew colonies.
                                    </P><P>
                                        Conditions used for its Quikchange PCR was: 10.2ng plasmid DNA in the 25µL PCR reaction.
                                    </P><P>
                                        By the end of the day, a few more of the J*4s also had colonies
                                    </P><P>
                                        J*4 (3) and J*4 (1), which seemed to had the densest colony growth, had overnight cultures set up (3x each) in Chl LB broth.
                                    </P></DIV><DIV id="4-18-pcr"><H4>PCRs</H4><P>
                                        Same as previous but re-diluted primer stock and used 1µl of IDT gBlock (i.e. 10ng of DNA) for A*
                                    </P><P>
                                        Run at 2 different cycles: either 98 and 72degC for 2 mins or 98, 72 annealing, 65 for 1 min and 72 for 1 min for extension.
                                    </P><P>              Gel: SYBR safe and agarose in LAB - see <A href="#">14/07</A>.
                                    </P><P>
                                        NO BANDS, but the primer dimer band for the 65 degC extension was brighter when visualised with blue light.
                                    </P><P>
                                        Leon to email IDT about further recommendations.
                                    </P></DIV><DIV id="4-18-transform-2"><H4>Transform completed plasmids into FliC and MG1655 Competent Cells</H4><P>
                                        N#, P#, H# and E#
                                    </P></DIV><DIV id="4-18-m9"><H4>Making up M9 modified media</H4><P>
                                        We made it up using the recipe from this <A href="#http://openwetware.org/wiki/Knight:M9_supplemented_media">website</A> for 500ml. We filter sterilised all components except milliQ water and stored it on the bench with LB.
                                    </P></DIV></DIV><DIV class="section" id="4-19"><H3>Day 19</H3><DIV id="4-19-miniprep"><H4>Mini-prep overnighted J*</H4><P>
                                        6 colonies grown overnight
                                    </P><P>
                                        Alterations to protocol:
                                    </P><OL><LI>Spin down entire volume of LB broth to ensure all E. coli are used in the extraction.</LI><LI>Pulse after spinning down all the E. coli to remove excess LB</LI><LI>Heat elution buffer to 55 degC before eluting 50ul, spin down, re-pipette filtrate into HiBind column and spin down again</LI></OL><P>
                                        Nanodrop values (values are quite consistent - good idea to apply all modifications for any mini-preps in the future)
                                    </P><P>
                                        Eppendorfs in freezer drawer awaiting potential sequencing
                                    </P><P>
                                        note: (1), (2), (3) are from plate 1 (corresponding to QC PCR at 70degC annealing temp, 1ng plasmids); (4), (5), (6) are from plate 3 (corresponding to QC PCR at 70degC annealing temp, 10ng plasmids)
                                    </P><TABLE class="table table-striped"><THEAD><TR><TD>[DNA] (ng/µl)</TD></TR></THEAD><TBODY><TR><TD>J*4 (1)</TD><TD>175.2</TD></TR><TR><TD>J*4 (2)</TD><TD>172.2</TD></TR><TR><TD>J*4 (3)</TD><TD>153.3</TD></TR><TR><TD>J*4 (4)</TD><TD>194.5</TD></TR><TR><TD>J*4 (5)</TD><TD>181.0</TD></TR><TR><TD>J*4 (6)</TD><TD>173.9</TD></TR></TBODY></TABLE></DIV><DIV id="4-19-digest"><H4>Diagnostic restriction digest of J* + gel electrophoresis</H4><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        20µl reaction for 90 minutes in PCR tubes
                                    </P><P>
                                        See <A href="#">protocol</A></P><P>
                                        Gel: 1% agarose, 120V
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>Ladder</TD><TD>J*1</TD><TD>J*2</TD><TD>J*3</TD><TD>J*4</TD><TD>J*5</TD><TD>J*6</TD></TR></TBODY></TABLE><P>
                                        Lack of ladder but all consistent - send for sequencing tomorrow
                                    </P></DIV><DIV id="4-19-pick"><H4>Pick Colonies</H4><P>
                                        Pick colonies of N#, P#, H# and E# (delta FliC and MG1655) and culture overnight
                                    </P><P>
                                        Pick colonies for E*, E*DMSO, D*, D#, M#, O# and culture overnight
                                    </P></DIV><DIV id="4-19-pcr"><H4>PCR</H4><P>
                                        O# and QC G*
                                    </P><P><A href="#https://www.neb.com/protocols/2013/01/26/q5-site-directed-mutagenesis-kit-protocol-e0554">QC PCR protocol</A></P><P>
                                        Ran PCR on 1ng, 5ng, and 10ng of template DNA with 3-minute extension time.
                                    </P><P>
                                        Digest PCR products in DpnI, according to the protocol <A href="#http://nebcloner.neb.com/#!/protocol/re/single/DpnI">here</A></P><P>
                                        ^has been done. Gel currently running.
                                    </P><P>
                                        Transform all the samples into separate cultures of competent DH5alpha
                                    </P><P>
                                        ^has not been done
                                    </P><P>
                                        Re-do C#, D#, L# PCR
                                    </P></DIV><DIV id="4-19-digest-2"><H4>Restriction Digestion and Ligation</H4><P>
                                        Restriction digest done on R and L.
                                    </P><P>
                                        Set up ligations for C#, D#, L# (PCR batch 14/07), running over night. After the gel only two managed to be extracted, one of C# or D# (R#) and L#.
                                    </P></DIV></DIV><DIV class="section" id="4-20"><H3>Day 20</H3><DIV id="4-20-leon"><H4>Leon</H4><P>
                                        3 frozen samples each were stored in the -80 degC freezer for E# (FliC), E# (MG1655), H# (FliC), H# (MG1655), N# (FliC), N# (MG1655), P# (FliC), P# (MG1655) following the overnight cultures that were set up on 16/07
                                    </P><P>
                                        They are in our usual freezer drawer, and within it the bottom most slot of the third column from the front, in a box labelled “iGEM transformed cells, MG1655 and FliC”
                                    </P></DIV><DIV id="4-20-raffy"><H4>Raffy</H4><P>
                                        Sent J*4 (4) for sequencing
                                    </P></DIV><DIV id="4-20-mabel"><H4>Mabel and Duke</H4><P>
                                        Miniprep - E*, E*DMSO, D*, D#, M#, O#
                                    </P><P>
                                        Miniprepped overnight cultures from June and Raffy above. Plasmids were eluted with 50ul of elution buffer.
                                    </P><P>
                                        Nanodrop:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TD>Plasmid type</TD><TD>ng/µl</TD></TR></THEAD><TBODY><TR><TD>D# </TD><TD>72.2</TD></TR><TR><TD>D# 2015 e1</TD><TD>160.6</TD></TR><TR><TD>D# 2015 e2</TD><TD>42.0</TD></TR><TR><TD>D# 2015</TD><TD>65.6</TD></TR><TR><TD>D* 2015</TD><TD>238.4</TD></TR><TR><TD>D* 2015 e</TD><TD>40.4</TD></TR><TR><TD>D* 2</TD><TD>288.9</TD></TR><TR><TD>D# e</TD><TD>31.2</TD></TR><TR><TD>D* 2e</TD><TD>378.7</TD></TR><TR><TD>E*1</TD><TD>333.9</TD></TR><TR><TD>E*1 e</TD><TD>287.5</TD></TR><TR><TD>E*2</TD><TD>370.6</TD></TR><TR><TD>E*2 e</TD><TD>367.1</TD></TR><TR><TD>M#e</TD><TD>50.5</TD></TR><TR><TD>M# 2015</TD><TD>50.8</TD></TR><TR><TD>M#e 2015</TD><TD>62.0</TD></TR><TR><TD>O#</TD><TD>49.9</TD></TR><TR><TD>O# 2015</TD><TD>82.7</TD></TR><TR><TD>O# e</TD><TD>44.1</TD></TR></TBODY></TABLE><P>
                                        Restriction digest of minipreps completed - find the protocol <A href="#">here</A> 20µl reactions.
                                    </P></DIV><DIV id="4-20-ria"><H4>Ria and Lychee</H4><P>
                                        Transform R#, L# into DH5alpha. R# is Chl and L# is Amp antibiotic
                                    </P></DIV><DIV id="4-20-helen"><H4>Helen and Kyle</H4><P>
                                        Digested plasmids for successful PCRs, plasmids not yet cleaned up.
                                    </P></DIV><DIV id="4-20-weekend"><H4>Over the Weekend</H4><P>
                                        Agar plates from 17/07 transferred to cold room
                                    </P><P>
                                        Sequencing data for J*4(4) received (order no: 4254194) - QC successfully corrected desired base, but also overzealously introduced two repeated sequences corresponding to the primer sequence 1. after the QC area of interest and 2. at the start of the DspB sequence (see SnapGene file: QC J*4 wrong). Colonies on plate 3 should all have wrongly QC-ed plasmids as a result.
                                    </P><P>
                                        Will send either samples (1), (2), or (3) for sequencing (as they are from plate 1 - PCRed under different conditions), and also overnight culture some colonies from plates 4, 5, and 6 as they were QC PCRed with a lower annealing temperature (lower possibility of wrong extension?)
                                    </P><P>
                                        Overnight cultures set up:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TD>Number of Tubes</TD><TD>Plasmid of Interest</TD></TR></THEAD><TBODY><TR><TD>3</TD><TD>L#“</TD></TR><TR><TD>3</TD><TD>“R#”</TD></TR><TR><TD>3</TD><TD>F*</TD></TR><TR><TD>3</TD><TD>I*</TD></TR><TR><TD>4</TD><TD>QC J*4: (A), (B) are from plate 5e (PCRed at 5ng plasmid, 65degC annealing) (C), (D) are from plate 6e (PCRed at 10ng plasmid, 65degC annealing)</TD></TR><TR><TD>1</TD><TD>Interlab 22A</TD></TR><TR><TD>1</TD><TD>Interlab 22K</TD></TR><TR><TD>1</TD><TD>Interlab 20K</TD></TR></TBODY></TABLE><P>
                                        On 20/07 <A href="#5-21">Day 21</A></P><P>
                                        Interlab cultures - transfer 700uL of the respective cultures into appropriately-labelled microcentrifuge tubes and add 300uL of 60% glycerol to each of them. Vortex for even mixing and flash freeze at -80degC. The rest of the cultures can be miniprepped to replenish plasmid supplies for further transformation into MG1655 and deltaFliC.
                                    </P><P>
                                        All other cultures.
                                    </P></DIV></DIV></DIV><DIV class="section" id="5"><H5>Week 5</H5><DIV class="section" id="5-21"><H3>Day 21</H3><DIV id="5-21-post"><H4>Post-digest cleanup → ligation</H4><P>
                                        For C#, D#, L# from the 16/07 PCR batch
                                    </P></DIV><DIV id="5-21-sequence"><H4>Sequencing of QC J*4</H4><P>
                                        J*4 (4) sequencing data shown to have many extra inserts, despite the desired mutagenetic site actually being correctly replaced. Two paths will be pursued:
                                    </P><OL><LI>J*4(1) (which is from plate 1 - another PCR condition (1ng plasmid)) to be sent for sequencing - obtain 2.5µL plasmids and dilute 1:1 in 2.5µL of Milli-Q. Sent for sequencing using VF2 and VR primers.</LI><LI>J*4(1) (which is from plate 1 - another PCR condition (1ng plasmid)) to be sent for sequencing - obtain 2.5uL plasmids and dilute 1:1 in 2.5uL of Milli-Q. Sent for sequencing using VF2 and VR primers.</LI></OL><P>
                                        If and when the sequences are correct, we will save the corresponding tube as a complete BioBrick (annotating that it has been QuikChanged), digest out the insert, cleanup, PCR the NcoI restriction site in, digest, and finally ligate into pBAD/HisB to make a functional plasmid.
                                    </P></DIV><DIV id="5-21-data"><H4>Sequencing Data Received</H4><P>
                                        K#1 confirmed - refer order number 4253587
                                    </P></DIV><DIV id="5-21-gel"><H4>Gel electrophoresis confirmation of 17/07 digested miniprep products</H4><P>
                                        Plasmids -
                                    </P><UL><LI>pSB1C3: D*, E*</LI><LI>pBAD33: D#</LI><LI>pBAD/HisB: M#, O#</LI></UL><P>
                                        After which, send for sequencing depending on degree of success. Unfortunately the gel ran too far and ‘ladder’ didn’t show up, which suggests that it is actually a mislabeled tube of loading dye. Re-made digests as only 3 ul of original is used. Will run gel again tomorrow with fresh ladder.
                                    </P></DIV><DIV id="5-21-miniprep"><H4>Miniprepping of 19/07 overnight cultures</H4><P>
                                        Miniprep L#, “R#”, F*, I*, QC J*4
                                    </P><P>
                                        For the rest - miniprep the full volume (see “upgraded” protocol: spin down cells → decant → resuspend in smaller volume before beginning the miniprep; pre-warm elution buffer to 55degC; run the same elution mixture through the column twice for maximum extraction)
                                    </P><P>
                                        Digest 10µL → gel run (gel is now in cold room, awaiting staining and visualization tomorrow)
                                    </P></DIV><DIV id="5-21-trasnform"><H4>Transformations</H4><P>
                                        K#1 → MG1655, deltaFliC
                                    </P></DIV><DIV id="5-21-plate"><H4>Plate Streaking</H4><P>
                                        Frozen stocks of MG1655, deltaFliC N#, P#, H#, E# streaked on agar plates.
                                    </P></DIV></DIV><DIV class="section" id="5-22"><H3>Day 22</H3><DIV id="5-22-data"><H4>Sequencing Data Received</H4><P>
                                        QuikChange J*4 (1) and ( C ) returned correct sequences (refer to order nos 4254450 and 4254692 respectively). Both are now stored in “Complete BioBricks” box.
                                    </P></DIV><DIV id="5-22-pcr"><H4>PCR Amplification to make J# from QuikChanged J*4[pSB1C3]</H4><P>
                                        J*4(1) was used; 3 min extension time (lol too long)
                                    </P></DIV><DIV id="5-22-transform"><H4>Transformations</H4><P>
                                        Transform C#, D#, L#, O# into DH5alpha
                                    </P><P>
                                        Transform pBAD33, pBAD/HisB into MG1655, deltaFliC for positive control.
                                    </P></DIV><DIV id="5-22-overnight"><H4>Overnight Cultures</H4><P>
                                        From newly transformed plated cultures:
                                    </P><UL><LI>MG and FliC - InterLab, K#</LI><LI>DH5alpha - (+) and (-) controls for InterLab</LI></UL><P>
                                        From streaked cultures:
                                    </P><UL><LI>MG and FliC - E#, H#, N#, P#</LI></UL></DIV><DIV id="5-22-gel"><H4>Run Gel for miniprepped J*QC, F*, I*, R#, L#</H4><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        F*, I* insert band very very faint - most probably because vector:insert ratio is about hundredfold
                                    </P><P>
                                        → redo direct digest using 2µL (20ng) stock gBlock, ligation with less plasmid (~10ng)
                                    </P><P>
                                        Kyle already doing C#, D#, L#
                                    </P><P>
                                        Top: J*4A, J*4B, J*4C, J*4D, F*1, F*2, F*3, I*1, I*2, I*3, L#1, L#2, L#3
                                    </P><P>
                                        Bottom: R#2, R#3
                                    </P><P>
                                        Sent F*1, I*3 for sequencing to find out what is going on with confusing 3kb/2kb double band. Results (order no: 4254965) shows both samples containing only blank pSB1C3.
                                    </P></DIV><DIV id="5-22-digest"><H4>Restriction digest and ligation of A*, B*, F*, I*, and K* directly from the G blocks</H4><P>
                                        Digested and dephosphorylated psb1c3 with EcoRI HF and PstI HF, made 1ng/µl psb1c3 solution.
                                    </P><P>
                                        Digested A*, B*, F*, I*, and K* with EcoRI HF and PstI HF, made 15ng/ul insert solution.
                                    </P><P>                                      Cleaned up both inserts and vectors.
                                    </P><P>
                                        Set up ligations with 15ng insert and 5ng vector.
                                    </P></DIV><DIV id="5-22-duke"><H4>Duke and Lychee</H4><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                        Managed to successfully run the gel for the test digests failed on 20/07/15. Image of gel shows that digestion following miniprep failed, but the presence of 3kb bands suggests that the ligation was in fact successful. Could send them for sequencing to find out.
                                    </P><P>
                                        Strong 2kb and 3kb bands for D* and E
                                    </P><P>
                                        D# only showed 3kb band
                                    </P><P>
                                        M# and O# show bands slightly larger than 3kb
                                    </P><P>
                                        Gel was loaded in the following order:
                                    </P><UL><LI>1st row: -Ladder-D*IIe-D*2015-D#e-D*2015e-D#2015e2-O#e-D*II-E*1e-M#2015-O#2015-M#2015-E*2-D#-D#2015-</LI><LI>2nd row: -Ladder-E*2e-O#-E1*-D#2015e1-M#e</LI></UL></DIV><DIV id="5-22-pcr-2"><H4>PCR of QuikChanged J*4(1) with primers for pBAD restriction sites</H4><DIV class="image image-left"><P>Gel Photp</P></DIV><P>
                                        Gel extraction EZNA carried out to combine both bands into one sample.
                                    </P></DIV></DIV><DIV class="section" id="5-23"><H3>Day 23</H3><DIV id="5-23-frozen"><H4>Frozen Stock Preparation</H4><P>
                                        K# in MG1655 and deltaFliC made into frozen stocks
                                    </P></DIV><DIV id="5-23-tasks"><H4>Tasks Completed</H4><P>
                                        Transformation of A*, B*, F*, I*, and K*
                                    </P><P>
                                        Digestion of J# and pBADHisB, then ligated.
                                    </P><P>
                                        Overnight cultures of C#, D#, L#, O#
                                    </P><P>
                                        PCR amplification of O#
                                    </P></DIV><DIV id="5-23-digest"><H4>Test Digest for D*IIe, E*1 compared with digest of completed BioBrick N*3</H4><P>
                                        Test digest shows no problem with restriction enzymes, as completed BioBrick was successfully digested.
                                    </P><P>
                                        Recommendation - for the parts that are going to be redone, it is recommended that one person takes it through.
                                    </P></DIV></DIV><DIV class="section" id="5-24"><H3>Day 24</H3><DIV id="5-24-overnight"><H4>Overnights</H4><P>
                                        Pick colonies and overnights for A*, B*, F*, G*, I*, and K*
                                    </P><P>
                                        Set up overnights for toxicity assay again: E#, H#, K#, N#, P#
                                    </P></DIV><DIV id="5-24-transform"><H4>Transformations</H4><P>
                                        Transformation of empty pBAD33 and pBADHisB into FliC and MG1655
                                    </P><P>
                                        Transform QCJ# into DH5alpha - Colonies grown
                                    </P></DIV><DIV id="5-24-data"><H4>Sequencing Data</H4><P>
                                        M#e and D#2015 sequencing data shows no insert (from 21/07 miniprep digest gel run photo)
                                    </P></DIV><DIV id="5-24-miniprep"><H4>Miniprep C#, D#, L#</H4><P>
                                        Test-digest → gel run:
                                    </P></DIV><DIV id="5-24-ligate"><H4>Ligations</H4><P>
                                        Ligate O# into pBAD/HisB
                                    </P><P>
                                        Done overnight, ready for day 25 transformation.
                                    </P></DIV></DIV><DIV class="section" id="5-25"><H3>Day 25</H3><DIV id="5-25-miniprep"><H4>Miniprep</H4><P>
                                        A*, B*, F*, I*, K*, G*QC
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Sample</TH><TH>ng/µl</TH></TR></THEAD><TBODY><TR><TD>A*1</TD><TD>56.8</TD></TR><TR><TD>A*2</TD><TD>65.7</TD></TR><TR><TD>A*3</TD><TD>57.5</TD></TR><TR><TD>Be*1</TD><TD>56.6</TD></TR><TR><TD>Be*2</TD><TD>101.7</TD></TR><TR><TD>Be*3</TD><TD>52.0</TD></TR><TR><TD>Ge*1</TD><TD>30.2</TD></TR><TR><TD>Ge*2</TD><TD>55.2</TD></TR><TR><TD>F*1</TD><TD>362.5</TD></TR><TR><TD>F*2</TD><TD>111.1</TD></TR><TR><TD>F*3</TD><TD>182.5</TD></TR><TR><TD>I*1</TD><TD>335.6</TD></TR><TR><TD>I*2</TD><TD>411.1</TD></TR><TR><TD>I*3</TD><TD>414.1</TD></TR><TR><TD>K*1</TD><TD>373.9</TD></TR><TR><TD>K*13/td&gt;
                                            </TD></TR><TR><TD>K*3</TD></TR></TBODY></TABLE></DIV><DIV id="5-25-gel"><H4>Gel Electrophoresis of minipreps</H4><TABLE class="table table-striped"><TBODY><TR><TD>Ladder</TD><TD>A* 1-3</TD><TD>B* 1-3</TD><TD>F* 1-3</TD><TD>G* 1-3</TD><TD>I* 2</TD><TD>I* 3</TD><TD>I* 1</TD><TD>K* 1-3</TD></TR></TBODY></TABLE><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                        G* and F*2 to be sent for sequencing on Monday.
                                    </P></DIV><DIV id="5-25-transform"><H4>Transform</H4><P>
                                        J#QC → DH5alpha - Amp
                                    </P><P>
                                        DsbADNase (from iGEMHQ) → DH5alpha - Chl
                                    </P></DIV><DIV class="section" id="5-25-weekend"><H4>Over the Weekend</H4><DIV id="5-25-weekend-plates"><H6>Streak plates</H6><P>
                                            EHKNP# - 2 species (10 plates) Amp
                                        </P><P>
                                            (+)(-) - 3 species (6 plates) Chl
                                        </P></DIV><DIV id="5-25-weekend-overnight"><H6>Overnight Cultures</H6><P>
                                            pBAD/HisB in MG1655, deltaFliC; pBAD33 in MG1655, deltaFliC, O#
                                        </P></DIV><DIV id="5-25-weekend-miniprep"><H6>Miniprep</H6><P>
                                            Miniprep of QCJ#
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="6"><H5>Week 6</H5><DIV class="section" id="6-26"><H3>Day 26</H3><DIV id="6-26-to-do"><H4>To Do</H4><UL><LI>Make up more modified M9 media</LI><LI>Melt agar and agarose</LI><LI>Send G*1, F*2 for sequencing</LI><LI>Miniprep + digest O#</LI></UL></DIV><DIV id="6-26-digest"><H4>Digest J# QC, run gel of O# and J# digest</H4><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        No insert for O#, Sent J#3 for sequencing
                                    </P></DIV></DIV><DIV class="section" id="6-27"><H3>Day 27</H3><DIV id="6-27-raffy"><H4>Raffy</H4><P>
                                        PCR C,D,L,M,N#
                                    </P></DIV></DIV><DIV class="section" id="6-28"><H3>Day 28</H3><DIV id="6-28-gel"><H4>Gel run for 28/07 PCR products, Band Excision, Clean Up</H4><P>
                                        C#, D#, L#, M#, O#
                                    </P></DIV><DIV id="6-28-digest"><H4>Digest + clean up</H4><P>
                                        pBAD33, C# and D# with BamHI
                                    </P><P>
                                        pBAD HisB, L#, M# and O# with NcoI and PstI
                                    </P><TABLE class="table table-striped"><TBODY><TR><TH>ng/µl</TH><TH>ng/µl</TH></TR><TR><TD>C#</TD><TD>1.1</TD><TD>O#</TD><TD>1.3</TD></TR><TR><TD>D#</TD><TD>0.4</TD><TD>pBAD 33</TD><TD>6.1</TD></TR><TR><TD>L#</TD><TD>0.3</TD><TD>pBAD HisB</TD><TD>3.9</TD></TR><TR><TD>M#</TD><TD>1.1</TD></TR></TBODY></TABLE></DIV><DIV id="6-28-ligate"><H4>Ligatation</H4><P>
                                        C# and D# with pBAD 33
                                    </P><P>
                                        L#, M# and O# with pBAD HisB
                                    </P><P>
                                        See protocol <A href="#">here.</A></P><TABLE class="table table-striped"><TBODY><TR><TH>Volume/µL</TH></TR><TR><TD>Insert</TD><TD>34</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD>DNA Ligase Buffer</TD><TD>10</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>C#</TH><TH>D#</TH><TH>L#</TH><TH>M#</TH><TH>O#</TH></TR><TR><TH>Plasmid</TH><TD>2.5</TD><TD>1.0</TD><TD>0.1</TD><TD>0.2</TD><TD>0.2</TD></TR><TR><TH>Milli-Q</TH><TD>2.5</TD><TD>4.0</TD><TD>4.9</TD><TD>4.8</TD><TD>4.8</TD></TR></TBODY></TABLE><P>
                                        N.B. plasmid:insert ratio is significantly lower than 1:3 - will see if this leads to better ligation results
                                    </P></DIV><DIV id="6-28-sucsess"><H4>Success</H4><P>
                                        QC J#3 sequence success (ref: 4256167)
                                    </P><P>
                                        Transform successful QC J#3 into MG1655, deltaFliC
                                    </P></DIV></DIV><DIV class="section" id="6-29"><H3>Day 29</H3><DIV id="6-29-data"><H4>Sequencing Data</H4><P>
                                        G*e2 QC sequence success (ref: 4256185)
                                    </P><P>
                                        → PCR it to make #
                                    </P><P>
                                        Gel band excised and stored at -20degC
                                    </P></DIV><DIV id="6-29-transform"><H4>Transformations</H4><P>
                                        Transform expression contructs ligated yesterday into DH5alpha
                                    </P><P>
                                        C#, D#, L#, M#, O#
                                    </P><P>
                                        (only 1µL plasmid used, may potentially not work)
                                    </P><P>
                                        Transform QCJ# into MG1655, deltaFliC.
                                    </P></DIV><DIV id="6-29-plate"><H4>Plate out BioBrick (T4 Holin) agar stab</H4><P>
                                        Plated on Amp (pSB1A2)
                                    </P></DIV></DIV><DIV class="section" id="6-30"><H3>Day 30</H3><DIV id="6-30-overnight"><H4>Overnight cultures</H4><P>
                                        For QCJ# MG and deltaFliC; C#; T4 Holin BioBrick
                                    </P></DIV><DIV id="6-30-transform"><H4>Transformation into DH5α</H4><P>
                                        C#, D#, L#, M#, O#
                                    </P></DIV><DIV id="6-30-to-do"><H4>To Do</H4><P>
                                        Cleanup; Digest; Ligate QCG#
                                    </P></DIV><DIV id="6-30-weekend"><H4>Over the Weekend</H4><P>
                                        Make stocks of J#QC
                                    </P><P>
                                        Innoculate C# D# L# M# O#
                                    </P><P>
                                        transform #s and interlab for characterisation
                                    </P><P>
                                        Make stocks of J#QC
                                    </P><P>         Overnights of #s and interlab for characterisation (all LB media)
                                    </P><P>
                                        Miniprep C# D# L# M# O# and Bba
                                    </P></DIV></DIV></DIV><DIV class="section" id="7"><H5>Week 7</H5><DIV class="section" id="7-31"><H3>Day 31</H3><DIV id="7-31-miniprep"><H4>Miniprep</H4><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                        Minipreps for C#, D#, L#, M#, O# → Test digest and run gel (done yesterday)
                                    </P><P>
                                        Send D#1, L#3, O#1 for sequencing
                                    </P></DIV></DIV><DIV class="section" id="7-32"><H3>Day 32</H3><DIV id="7-32-results"><H4>Sequencing Results (4257633):</H4><UL><LI>pBADHisB Art-175 (O#1) - success</LI><LI>pBAD33 Lsr Holin (D#1) - 1 mutation, ask them to resequence</LI><LI>pBADHisB YebF Art-175 (L#3) - 1 mutation, ask them to resequence</LI></UL></DIV></DIV><DIV class="section" id="7-33"><H3>Day 33</H3><DIV id="7-33-george"><H4>George</H4><P>
                                        Miniprep G#
                                    </P></DIV><DIV id="7-33-helen"><H4>Helen</H4><P>
                                        PCR K # → * KHTS primers, annealing temperature 72*C, extension time 2 mins
                                    </P><P>
                                        Cleanup C#, M# digested → ligate
                                    </P></DIV><DIV id="7-33-mabel"><H4>Mabel and James</H4><P>
                                        Set up overnights - add J# to them
                                    </P><P>
                                        All in LB Chl or LB Amp (inc. interlab)
                                    </P></DIV></DIV><DIV class="section" id="7-34"><H3>Day 34</H3><DIV id="7-34-kyle"><H4>Kyle</H4><P>
                                        Sequences confirmed (4257633) for D#, L#, O#. Therefore transform into MG, FliC.
                                    </P><P>
                                        Transform C* into MG, FliC
     </P></DIV><DIV id="7-34-helen"><H4>Helen</H4><P>
                                        Transform ligated C#, M# into DH5alpha.
                                    </P></DIV></DIV><DIV class="section" id="7-35"><H3>Day 35</H3><DIV class="section" id="7-35-to-do"><H4>To Do</H4><DIV id="7-35-to-do-i"><H6>i)</H6><P>
                                            If colonies have formed from the transformation of ligated C# and M# in DH5alpha, pick colonies and set up overnight cultures in the afternoon. Someone will need to come in on Saturday to miniprep these cultures, consult Mabel for tips and tricks on miniprepping if necessary.
                                        </P><P>
                                            James and Lychee performed minipreps on Saturday and will update on where the miniprepped DNA are in the freezer.
                                        </P></DIV><DIV id="7-35-to-do-ii"><H6>ii)</H6><P>
                                            If colonies have formed for the MG1655 and deltaFliC with C*, D#, L#, O#, pick colonies and set up overnight cultures in the afternoon. These cultures do not need to be miniprepped. The next day, make two tubes of frozen stocks out of each culture, and also since it is now the end of the week, streak some of the liquid cultures onto agar plates supplemented with the appropriate antibiotics. If unsure about where frozen stocks of expression cultures should go, consult Mabel.
                                        </P><P>
                                            James and Lychee made frozen stocks out of these cultures on Saturday.
                                        </P></DIV><DIV id="7-35-to-do-iii"><H6>iii)</H6><P>
                                            E* has yet to be inserted into pSB1C3: PCR from gBlock solution.
                                        </P><P>
                                            E* PCR failed. We shall consider PCR-ing directly from gBlocks again, or design long primers to PCR E* out of the completed E# (inserted in pBAD/HisB) plasmid.
                                        </P></DIV><DIV id="7-35-to-do-iv"><H6>iv)</H6><P>
                                            We currently have D# (Lsr Holin), but not an endolysin needed to complete the Holin-Endolysin lysis system. In case our Art-E part (M#) again fails to successfully be cloned, procure BBa_K112806 [pSB1C3] from the 2015 Distribution Kit (Plate 3, Well 12K) and transform into DH5alpha. Subsequently we can potentially insert a promoter in front of it to complete the lysis system.
                                        </P><P>
                                            Colonies formed and the plate was transferred to the cold room.
                                        </P></DIV><DIV id="7-35-to-do-v"><H6>v)</H6><P>
                                            Since B* failed to PCR to give the Synbiota LasR and pLas parts that we needed, we will instead obtain them from the Distribution Kit as well. They are both in pSB1C3, LasR being BBa_C0179 in Plate 3 Well 6M, and pLas being BBa_R0079 in Plate 3 Well 5C respectively.
                                        </P><P>
                                            Colonies formed and the plates were transferred to the cold room.
                                        </P></DIV><DIV id="7-35-to-do-vi"><H6>vi)</H6><P>
                                            G# miniprep yesterday showed no insert: Using QC G* as template, PCR using primers DspB forward and Post Suffix Holin to make it into G# for cloning into expression vector.
                                        </P><P>
                                            PCR succeeded, excised gel bands kept in freezer.
                                        </P></DIV><DIV id="7-35-to-do-vii"><H6>vii)</H6><P>
                                            Continue with KHTS - ligate overnight, provided someone can put them in the freezer tomorrow morning.
                                        </P><P>
                                            Transferred into freezer.
                                        </P></DIV><DIV id="7-35-to-do-viii"><H6>viii)</H6><P>
                                            Transform the part <A href="#http://parts.igem.org/Part:BBa_K741002">2015 Kit Plate 2 Well 17M</A> and then grow in agar plate. Silas will need to film doing the transformation and the plating process.
                                        </P><P>
                                            Completed.
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="8"><H5>Week 8</H5><DIV class="section" id="8-36"><H3>Day 36</H3><DIV class="section" id="8-36-to-do"><H4>To Do</H4><OL><LI>Test digest C#, M#, if working send them for sequencing.</LI><LI>PCR E* (normally).</LI><LI>Cleanup G# from excised gel, digest (remember to digest some pBAD/HisB plasmid as well), heat inactivate, dephosphorylate the plasmid only, ligate the both of them.</LI><LI>Transform K* (KHTS).</LI><LI>Synbiota <A href="#https://drive.google.com/open?id=0B7GmyDM3E1rKYll3bmY1dktjVHM)">PCR</A></LI></OL><P>
                                        N.B. Our original gBlock solutions are 10ng/µL - use 2µL of that for the PCR
                                    </P></DIV></DIV><DIV class="section" id="8-37"><H3>Day 37</H3><OL><LI>PCR C# from C*pSB1C3, M# from M*</LI><LI>Why won’t E* PCR??</LI><LI>G# ligated into HisB (the tube was labelled G* into HisB)</LI><LI>K* transformed - check if colonies formed</LI></OL></DIV><DIV class="section" id="8-38"><H3>Day 38</H3><P>
                                    Out of Q5, so use Pfu - this involves adding nucleotides, different annealing temperatures - ask Chris or George
                                </P><P>
                                    BioBricks
                                </P><OL><LI> F and I with new restriction enzyme combination (XbaI, PstI-HF) - up to ligation overnight (Mabel)</LI><LI>PCR K# to K* (Kyle)</LI></OL><P>
                                    Expression vectors
                                </P><OL><LI>E# - try one more time but then if it doesn’t work come to Leon or Raffy to design long primers (Kyle)</LI><LI>G# pick colonies (Kyle)</LI><LI>C# and M# repeat PCR (Helen)</LI></OL></DIV><DIV class="section" id="8-39"><H3>Day 39</H3><OL><LI>Design (E,D)# to (E,D)* primers</LI><LI>Transform SBArtE2 into DH5alpha i.e. M* hopefully</LI><LI>Transform the ligated F*, I*s which were directly digested from gBlock using X, P</LI><LI>Miniprep G#</LI></OL><P>
                                    NB: did phusion go back in the freezer from which it came? Ensure it doesn’t go in our freezer.
                                </P></DIV><DIV class="section" id="8-40"><H3>Day 40</H3><OL><LI>Pick M*, F* and I* colonies</LI><LI>PCR G* to G# and E# to E* </LI><LI>K* and C# ligated overnight - transform </LI></OL><DIV id="8-40-weekend"><H4>Over the Weekend</H4><P>
                                        Miniprep of F*, I*, K*
                                    </P><P>
                                        Nanodrop:
                                    </P><TABLE class="table table-striped"><TBODY><TR><TH>ng/µl</TH><TH>ng/µl</TH><TH>ng/µl</TH></TR><TR><TD>F*1</TD><TD>353.5</TD><TD>I*1</TD><TD>464.1</TD><TD>M*1</TD><TD>204.2</TD></TR><TR><TD>F*2</TD><TD>430.0</TD><TD>I*2</TD><TD>428.4</TD><TD>M*2</TD><TD>148.3</TD></TR><TR><TD>F*3</TD><TD>351.3</TD><TD>I*3</TD><TD>363.8</TD><TD>M*3</TD><TD>189.1</TD></TR></TBODY></TABLE><P>
                                        N.B. I*1, I*2, M*1 and M*3 are not pure samples (curves on nanodrop had weird peaks) - these are the eppendorfs without smiley faces drawn next to the [DNA] on the side of the tube.
                                    </P><P>
                                        N.B.B. Mabel has a new PB for [DNA] on the QIAGEN miniprep kit ;)
                                    </P><P>
                                        Restriction digest: 20ul reaction, 37degC for 90 mins
                                    </P><P>
                                        M* also digested with XbaI and PstI-HF - assume that the insert would have the same MCS as F* and I* if they’re all meant to be for biobricks. See protocol <A href="#">here.</A></P><P>
                                        Gel: 1% agarose
     </P><P>
                                        Only needed ~35 mins to stain well - due to high [DNA]??
                                    </P><P>
                                        M*2 can be sent for sequencing (only ‘normal’ curve on nanodrop)
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>F*1</TD><TD>F*2</TD><TD>F*3</TD><TD>I*1</TD><TD>I*2</TD><TD>I*3</TD><TD>M*1</TD><TD>M*2</TD><TD>M*3</TD></TR></TBODY></TABLE></DIV><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                    Streak all frozen stocks - in two locations
                                </P><P>
                                    Bottom most box in the 3rd column has a portion of the frozen stocks but not the new ones
                                </P><P>
                                    New ones are in the topmost box in the second column
                                </P><P>
                                    Overnights:
                        </P><UL><LI>All interlabs (unnecessary duplicates of deltaFliC but nevermind) and extra 20K MG and 20K DH5</LI><LI>E# and J# + controls (duplicated)</LI><LI>P. putida</LI></UL></DIV></DIV><DIV class="section" id="9"><H5>Week 9</H5><DIV class="section" id="9-41"><H3>Day 41</H3><DIV id="9-41-last"><H4>Last few bits of cloning</H4><P>
                                        Transform C* (in complete biobricks box) into M51655 and RP437 (oops C* is already transformed and D* doesn’t exist)
                                    </P><P>
                                        Send M*, F* and I* for sequencing
                                    </P><P>
                                        → F1, M2 and I3 sent
                                    </P></DIV></DIV><DIV class="section" id="9-42"><H3>Day 42</H3><OL><LI>PCR M# from M*</LI><LI>Synbiota PCR</LI><LI>O* (Art-175 on its own) isn’t a BioBrick - let’s make it!</LI></OL></DIV><DIV class="section" id="9-43"><H3>Day 43</H3><P>                    Test digest of C# -&gt; run on gel -&gt; send C#1dF for sequencing -&gt; not right.
                                </P><P>
                                    All C# miniprep values were very very low, probably due to tranformation into wrong strain. PCR C# tomorrow.
                                </P><P>
                                    PCR G# -&gt; run on gel -&gt; gel extraction -&gt; restriction digest.
                                </P></DIV><DIV class="section" id="9-44"><H3>Day 44</H3><P>
                                    Carry on from gel extraction of G# - restriction digest -&gt; clean up -&gt; nanodrop (Kyle has values) -&gt; ligation.
                                </P><P>
                                    PCR C* to C# again.
                                </P></DIV><DIV class="section" id="9-45"><H3>Day 45</H3><P>
                                    C# gel - excise bands, continue
                                </P><P>
                                    G# ligation - This is in the front of our top freezer drawer. Transform this into DH5a
                                </P></DIV></DIV><DIV class="section" id="10"><H5>Week 10</H5><DIV class="section" id="10-46"><H3>Day 46</H3><DIV id="10-46-mabel"><H4>Constructs PCR</H4><P>
                                        New primers arrived - but mistake made with M (thought it was # to * but actually need * to #, so using old primers ordered previously)
                                    </P><TABLE class="table table-striped"><TBODY><TR><TH>Construct</TH><TH>Fwd primer</TH><TH>Rev primer</TH><TH>Annealing temp.</TH></TR><TR><TD>P (# to *)</TD><TD>DNase HTS</TD><TD>Art E suffix</TD><TD>72</TD></TR><TR><TD>K (# to *)</TD><TD>DsbA HTS</TD><TD>Art E suffix</TD><TD>72</TD></TR><TR><TD>E (# to *)</TD><TD>DsbA HTS</TD><TD>Art E suffix</TD><TD>72</TD></TR><TR><TD>C (# to *)</TD><TD>Lsr Holin HTS F</TD><TD>Lsr Holin HTS R</TD><TD>71</TD></TR><TR><TD>M (* to #)</TD><TD>SMAP 29</TD><TD>Art E suffix</TD><TD>70</TD></TR></TBODY></TABLE><P>
                                        Also used Phusion Polymerase but primers are at 10µM instead of 25µM.
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>Ladder</TD><TD>T4 Holin</TD><TD>pLsr</TD><TD>C*</TD><TD>E*</TD><TD>K*</TD><TD>P*</TD></TR></TBODY></TABLE><DIV class="image image-left"><P>Gel Photo</P></DIV><P>
                                        G# colonies picked, in overnight broth in 37 degree incubator. Plates are in the cold room.
                                    </P></DIV></DIV><DIV class="section" id="10-47"><H3>Day 47</H3><DIV id="10-47-clone"><H4>Molecular cloning of constructs</H4><P>
                                        Gel extraction of C*, E*, K*, P* - eluted with 35ul of EB
                                    </P><P>
                                        Restriction digest of C*, E*, K*, P* and pSB-1C3 plasmid with XbaI and PstI-HF in a 50ul reaction, 37 degC for 1 hour at 300rpm
                                    </P><P>
                                        95degC heat inactivation for 30mins
                                    </P><P>
                                        Add 1µl CIP for pSB-1C3 only at 37degC for 30 mins
                                    </P><P>
                                        Cleanup - elute insert with 35µl EB buffer, plasmid with 50µl
                                    </P><P>
                                        Ligation set ups: 50ul reaction, use all 34ul of eluted DNA
                                    </P><P>
                                        Incubate at 16 degC overnight
                                    </P><P>
                                        N.B. There is some reaction mixture of M* to M# left over from 24/08 so will use this to run a gel with the synbiota PCRs.
                                    </P></DIV></DIV><DIV class="section" id="10-48"><H3>Day 48</H3><DIV id="10-48-kyle"><H4>Kyle</H4><P>
                                        G# miniprep -&gt; nanodrop -&gt; test digest
                                    </P></DIV><DIV id="10-48-mabel"><H4>Mabel</H4><P>      M# mentioned in synbiota section
                                    </P><P>
                                        Transformation of C*, E*, K* and P* into DH5α
                                    </P><P>
                                        Plated transformations onto Chl plates - watch out for potential mixup of P* and K* in the low concentration bacteria plates
                                    </P><P>
                                        PCR of D# to D* using Pre prefix holin and post suffix holin primers, standard Phusion set up, annealing temperature 71degC
                                    </P></DIV></DIV><DIV class="section" id="10-49"><H3>Day 49</H3><P>
                                    PCR of O# to *, using new primers that arrived on 24/08 (thought they were for M# - * but were actually for O)
                                </P><P>
                                    Forward primer: Art-175 HTS, Reverse primer: Art-E suffix
                                </P><P>
                                    Diluted to 25uM, annealing temperature 71degC
                                </P><P>
                                    Three different setups:
                                    <OL><LI>Phusion RDP protocol</LI><LI>Phusion RDP protocol with GC buffer from Chris (optimised for primers with a high GC content e.g. Rhodobacter genome)</LI><LI>Phusion RDP protocol with 5% DMSO</LI></OL></P><P>
                                    Template was not digested before PCR
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>RDP</TD><TD>RDP</TD><TD>GC buffer</TD><TD>GC buffer</TD><TD>DMSO</TD><TD>DMSO</TD></TR></TBODY></TABLE><P>
                                    Gel made with water…………………………………!@%*#%$^
                                </P><P>
                                    E*, K* and P* (we actually already have C*) - 2 colonies picked from each plate, so total of 4 overnights per construct (high and low conc plates)
                                </P></DIV><DIV class="section" id="10-50"><H3>Day 50</H3><P>
                                    Repeat PCR of O# to *, Phusion set up as on <A href="#10-49">27/08</A> but no DMSO or buffer change.
                                </P><P>
                                    Miniprep of E*, K* and P*: all eluted in 50ul EB buffer
                                </P><P>
                                    Nanodrop (adjusting volume added to digest to keep [DNA] roughly the same)
                                </P><P>
                                    37 degC for 90 mins
                </P><P>
                                    Gel: 1% TBE agarose, 120V
                                </P><P>
                                    it didn’t melt!  YAAAAAAAAAASSSSSSS &gt;8)
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>E*1</TD><TD>E*2</TD><TD>E* [ ] 1</TD><TD>E* [ ] 2</TD><TD>K*1</TD><TD>K*2</TD><TD>K* [ ] 1</TD><TD>K* [ ] 2</TD></TR><TR><TD>P*1</TD><TD>P*2</TD><TD>P* [ ] 1</TD><TD>P* [ ] 2</TD><TD>O#-* 1</TD><TD>O#-* 2</TD><TD>O#-* 3</TD><TD>O#-* 4</TD></TR></TBODY></TABLE><P>
                                    Excised all O* bands (now in freezer)
                                </P><P>
                                    K* [ ] 1 can be sent for sequencing (insert length should be 1030bp)
                                </P></DIV></DIV><DIV class="section" id="11"><H5>Week 11</H5><DIV class="section" id="11-51"><H3>Day 51</H3><DIV id="11-51-mabel"><H4>Molecular cloning day YAYZ</H4><P>
                                        Gel extraction of D* and O* PCRs
                                    </P><P>
                                        Digest of D*, O* (50ul set ups) and pSB-1C3 (100ul set ups) with XbaI and PstI-HF
                                    </P><P>
                                        Heat inactivate inserts and plasmid at 95degC for 30 mins
                                    </P><P>
                                        Add CIP to pSB-1C3 and incubate at 37degC for 30 mins
                                    </P><P>
                                        Clean up: inserts eluted in 35µl, plasmid eluted in 50µl
                                    </P><P>
                                        Ligations: 50ul set ups
                                    </P><P>
                                        All with 5ul of 10x T4 DNA ligase buffer and 1ul T4 DNA ligase
                                    </P><P>
                                        From the nanodrop values omitted O*3 because [DNA] is too low.
                                    </P><P>
                                        Incubate at 16degC for 16 hours
                                    </P><P>                               PCR G* to G#
                                    </P><P>
                                        3 different conditions: RDP protocol, RDP protocol with GC buffer, RDP protocol with 5% DMSO
                                    </P><P>
                                        Gel: 1% TBE agarose, 80V
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>5% DMSO</TD><TD>5% DMSO</TD><TD>GC buffer</TD><TD>GC buffer</TD><TD>RDP</TD><TD>RDP</TD></TR></TBODY></TABLE><P>
                                        Something is wrong with the camera :/ - everything except the DMSO PCRs have been excised
                                    </P><P>
                                        Transformation of pSB-1C3 into DH5α - plate onto Chl plates
                                    </P><P>
                                        Accidentally transformed all the DH5α - need to make DH5α competent cells from plate in cold room (given on 23/07 so not sure how successful this is going to be…)
                                    </P></DIV><DIV id="11-51-to"><H4>To Do</H4><P>
                                        Made 5ml of Chl antibiotic stock (into EtOH)
                                    </P><P>
                                        Overnights: re-pick from E* and P* plates into Chl LB
                                        <UL><LI>3 colonies per plate, 12 overnights in total</LI></UL></P></DIV></DIV><DIV class="section" id="11-52"><H3>Day 52</H3><DIV id="11-52-mabel"><H4>Mabel</H4><P>
                                        Miniprep E* and P* overnights - all eluted in 50µl buffer
                                    </P><P>
                                        20µl digest with XbaI and PstI-HF
                                    </P><P>
                                        Gel: 1% TBE agarose, 120V
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>1kb+ ladder</TD><TD>P* 1-3</TD><TD>P*e 1-3</TD><TD>E* 1-3</TD><TD>E*e 1-3</TD></TR></TBODY></TABLE></DIV><DIV id="11-52-june"><H4>June</H4><P>
                                        Making DH5alpha Competent Cells
                                        <UL><LI>20 eppendorf tubes are restocked in -80C</LI></UL></P><P>
                                        Transforming O* and C*
                                        <UL><LI>Plates are : O*1, O*1e, O*2, O*4, D*1, D*2, D*1e, D*2e</LI><LI>x10 conc plates gave good colonies</LI></UL></P><P>
                                        Picking colonies and setting up overnight cultures for psb1c3 transformed in DH5alpha
                                    </P><P>
                                        Miniprepping E* and P* overnights
                                    </P></DIV></DIV><DIV class="section" id="11-53"><H3>Day 53</H3><P>
                                    Repeat digest of E* and P* cos I’m an idiot and i forgot to write down what they were
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>E*2</TD><TD>E*3</TD><TD>E*e 2</TD><TD>P*2</TD><TD>P*3</TD><TD>P*e 2</TD></TR></TBODY></TABLE><P>
                                    Send E*2 and E*e2 for sequencing forward read only - whichever one comes back will get a reverse read.
                                </P><P>
                                    Miniprep of pSB-1C3 - all eluted in 50µl
                                </P><P>
                                    Digest with just XbaI to linearise for gel
                                </P><P>
                                    1% TBE agarose, 120V
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>pSB-1C3 1</TD><TD>pSB-1C3 2</TD><TD>pSB-1C3 3</TD><TD>pSB-1C3 4</TD><TD>pSB-1C3 5</TD></TR></TBODY></TABLE><P>
                                    Mystery of the massive bands: case closed.
                                </P><P>
                                    The massive band is linearised pSB-1C3!!
                                </P></DIV><DIV class="section" id="11-54"><H3>Day 54</H3><P>
                                    Miniprep of pBAD HisB, D* and O* overnights - all eluted in 50µl EB
                                </P><P>
                                    Digest: * with XbaI and PstI-HF in cutsmart buffer, HisB with NcoI in 3.1 buffer
                                </P><P>
                                    Moved into freezer - run gel tomorrow
                                </P><P>
                                    Moved E*2 into biobricks box :)
                                </P></DIV><DIV class="section" id="11-55"><H3>Day 55</H3><DIV id="11-55-digest"><H4>Digest of M#, G#, pBad HisB</H4><P>
                                        G#: BspHI, PstI-HF in 3.1 (should be cutsmart but Mabel forgot to write the buffer for that set of enzymes)
                                    </P><P>
                                        M#: NcoI, PstI in 3.1
                                    </P><P>
                                        pBAD HisB: Nco, PstI in 3.1
                                    </P></DIV><DIV id="11-55-digest-2"><H4>Digests from Kyle</H4><P>
                                        Heat inactivate at 95 degC for 30 mins
                                    </P><P>
                                        Add CIP to pBAD HisB for 30 mins at 37 degC
                                    </P><P>
                                        Clean up: HisB eluted in 50µl, inserts in 35µl
                                    </P><P>
                                        Ligation into pBAD HisB: all with 5µl buffer and 1µl DNA ligase
                                    </P><P>
                                        N.B. The G# ligations are slightly over 50ul - diluted HisB too far in digest
                                    </P><P>
                                        Leon to move ligations onto Mabel’s orange rack in the -20 freezer.
                                    </P></DIV><DIV id="11-55-clone"><H4>Molecular cloning</H4><P>
                                        Run gel (1% TBE agarose, 120V) of O*, D*, pBAD HisB and remaining P# - * PCR from 24/08 with 1kb+ ladder
                                    </P><TABLE class="table table-striped"><TBODY><TR><TD>D*1</TD><TD>D*2</TD><TD>O*1</TD><TD>O*2</TD><TD>pBAD HisB</TD><TD>P# - *</TD><TD>P# - *</TD></TR></TBODY></TABLE><P>
                                        P# - * PCR has been excised and gel extracted - on orange rack in freezer
                                    </P><P>
                                        Send O*1 (as Art-175 1) and K* high conc.1 (as Art-175 DsbA 1) for sequencing with correct primers
                                    </P></DIV></DIV></DIV><DIV class="section" id="12"><H5>Week 12</H5><DIV class="section" id="12-56"><H3>Day 56</H3><DIV id="12-56-transform"><H4>Transformation of #s</H4><P>
                                        M# and G# RDP and GC
                                    </P><P>
                                        Plated on amp.
                                    </P></DIV></DIV><DIV class="section" id="12-57"><H3>Day 57</H3><P>
                                    Sequencing: send K* and O* for reverse reads with reverse primers
                                </P><P>
                                    Biobrick submission: biobricks box contents correlates with master table and exceed the minimum required [ ] for submission.
                                </P><P>
                                    Transformations of G# and M# have been binned - will not be making anymore #s, focus on last round of biobricks.
                                </P><P>
                                    PCR of D* and P* - aim to troubleshoot/test as many different variables as possible
                                </P><P>
                                    Not enough of P# so only carry out Phusion PCR - rest is given to Duke to transform.
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>P# to P* (Phusion)</TD><TD>D# to D* (Phusion)</TD></TR><TR><TD>D# to D* (Phusion)</TD><TD>D# to D* (Q5)</TD></TR></TBODY></TABLE><P>
                                    Mabel when putting the gel in the tank: This feels slightly watery… it’ll be fine!
                                </P><P>
                                    *gel starts running funny after 5 minutes*
                                </P><P>
                                    Mabel: James………..
                                </P><P>
                                    James: …???
                                </P><P>
                                    Ria: hmm it looks weird.  Those bands are really slow.  Did you make the gel with water?
                                </P><P>                                 Mabel: No of course not who would make a gel with water.
                                </P><P>
                                    James: I did! ……………...
                                </P><P>
                                    Mabel: ……………………………………………..
                                </P><P>
                                    WHY HA-MES WHY T^T T^T T^T T^T T^T T^T T^T T^T T^T
                                </P><P>
                                    Gel put straight into EtBr, entire section of gel that has been run excised - sort out tomorrow
                                </P></DIV><DIV class="section" id="12-58"><H3>Day 58</H3><P>
                                    PCR rescue mission: Gel extraction, re-run on gel, excise.
                                </P><P>
                                    Q5 P# to P* PCRs repeated as they didn’t run far on gel yesterday.
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>P# to P* (Phusion)</TD><TD>D# to D* (Phusion)</TD></TR><TR><TD>D# to D* (Phusion)</TD><TD>D# to D* (Q5)</TD></TR></TBODY></TABLE><P>                          D*= 880bp, P* = 570bp
                                </P><P>
                                    PCRs saved :) all bands excised!  (count yourself lucky Ha-mes)
                                </P></DIV><DIV class="section" id="12-60"><H3>Day 60</H3><P>
                                    Transform pBADHisB GFP into MG and RP.
                                </P><P>
                                    Transform O# D# hybrid. Overnights also made directly from post-heat shock incubation culture.
                                </P><P>
                                    Overnights: rhodo, putida, LYer, EM, ER, JM, JR, HisM, HisR, OM, OR, DM, DR
                                    <UL><LI>1 tube - no AB LB</LI><LI>1 tube - Amp, BHI</LI><LI>1 tube - no AB SUX</LI><LI>8 tubes - Amp LB</LI><LI>2 tubes - Chl LB</LI></UL></P><P>
                                    K* high conc 1 and O*1 reverse reads correct - put into biobricks box :D
                                </P><P>
                                    Gel extraction of 2nd gel of D* and P* PCRs - all eluted in 35ul buffer EB. N.B. potential mix up of D Phu 2 and D Phu 3 reaction
                                </P><P>
                                    37degC for 90 minutes. Heat inactivate at 95degC for 30mins. Add CIP to pSB-1C3, incubate at 37 degC for mins
                                </P><P>                    If [insert] &lt; 3.0 ng/ul, did not proceed with ligation (can’t get dilution of pSB-1C3 to the correct ratio)
                                </P><P>
                                    Plasmid dilutions: 1:2 except for pSB-1C3 (XbaI + PstI) by adding 49µl of MilliQ
                                </P><P>
                                    All with 1ul T4 DNA ligase and 5µl T4 DNA ligase buffer
                                </P><P>
                                    16˚C for 16 hours
                                </P><DIV class="section" id="12-60-weekend"><H4>Over the Weekend</H4><P>
                                        Transformation of D* and P* ligations from 11/09 - plate on Chl agar.
                                    </P><P>
                                        Check transformations:
                                    </P><P>
                                        15 successful plates (thank you June!)
                                    </P><P>
                                        2 overnights made of for each plate with colonies (LB Chl) - 30 in total
                                    </P><P>
                                        Also made 2 x P# overnights (LB Amp) and 4 x interlab overnights for Ria (LB Chl)
                                    </P></DIV></DIV></DIV><DIV class="section" id="13"><H5>Week 13</H5><DIV class="section" id="13-61"><H3>Day 61</H3><P>
                                    Minipreps pf D*, P* and P#
                                </P><P>
                                    Digest set ups: 37 deg for 90 mins
                                </P><P>
                                    N.B. 2µl of template used for D Q5 1a, D Q4 4b, P#1 and P#2 due to lower [DNA]
                                </P><P>
                                    P# digested with NcoI and PstI-HF in cutsmart buffer
                                </P><P>
                                    Gels: 2% TBE agarose (for better separation as D* = 880bp and P* = 570bp), 120V, both used Thermo Fisher 1kb+ ladder
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>P#1</TD><TD>P Phusion batch a 1,3,4,7,8</TD><TD>D Phusion batch a 1,2,4,5,7,8</TD><TD>D Q5 1a</TD></TR><TR><TD>P#2</TD><TD>P Phusion batch b 1,3,4,7,8</TD><TD>D Phusion batch b 1,2,4,5,7,8</TD><TD>D Q5 1a</TD></TR></TBODY></TABLE><P>
                                    P#1 moved to pBAD + inserts box, P#2 discarded
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>D Q5 2a</TD><TD>D Q5 4a</TD><TD>D Q5 7a</TD><TD>D Q5 2b</TD><TD>D Q5 4b</TD><TD>D Q5 7b</TD></TR></TBODY></TABLE><P>
                                    *weeps* No D* or P* :(((((((
                                </P><P>
                                    Molecular cloning: over and out ;)
                                </P></DIV></DIV></DIV><DIV class="section" id="characterization"><H2>Characterization</H2><DIV class="section" id="plan-c"><H5>Plan</H5><DIV class="section" id="plan-c-overview"><H4>Overview of sequences in pBAD</H4><TABLE class="table table-striped"><TBODY><TR><TD>pBAD 33 LasR Holin</TD><TD>A#</TD><TD>No PCR</TD></TR><TR><TD>pBAD 33 LasR sfGFP</TD><TD>B#</TD></TR><TR><TD>pBAD 33 Lsr sfGFP</TD><TD>C#</TD><TD>Overnight Culture</TD></TR><TR><TD>pBAD 33 Lsr Holin</TD><TD>D#</TD><TD>Miniprepped, awaiting gel --&gt; sequencing</TD></TR><TR><TD>pBAD HisB DspB YebF</TD><TD>E#</TD><TD>Transformed into MG1655, deltaFliC</TD></TR><TR><TD>pBAD HisB DspB</TD><TD>F#</TD></TR><TR><TD>pBAD HisB MccS</TD><TD>G#</TD><TD>QC in progress</TD></TR><TR><TD>pBAD HisB DspB Fla</TD><TD>H#</TD><TD>Transformed into MG1655, deltaFliC</TD></TR><TR><TD>pBAD HisB DspB DsbA</TD><TD>I#</TD></TR><TR><TD>pBAD HisB Art-175 DsbA</TD><TD>J#</TD><TD>QC introduced extra sequences (order no 4254194); send another</TD></TR><TR><TD>pBAD HisB Art-175 YebF</TD><TD>K#</TD><TD>Sequence confirmed (order no 4253587), awaiting transform</TD></TR><TR><TD>pBAD HisB Art-E</TD><TD>L#</TD><TD>Overnight culture</TD></TR><TR><TD>pBAD HisB Art-175 Fla</TD><TD>M#</TD><TD>Miniprepped, awaiting gel --&gt; sequencing</TD></TR><TR><TD>pBAD HisB Art-175</TD><TD>N#</TD><TD>Transformed into MG1655, deltaFliC</TD></TR><TR><TD>pBAD HisB Art-175</TD><TD>O#</TD><TD>Miniprepped, awaiting gel --&gt; sequencing</TD></TR><TR><TD>pBAD HisB DNase</TD><TD>P#</TD><TD>Transformed into MG1655, deltaFliC</TD></TR></TBODY></TABLE></DIV><DIV id="plan-c-complete"><H4>Currently completed pBADs (frozen stocks of plasmids in MG1655 and deltaFliC made):</H4><P><STROMG>With secretion tag - E# (DsbA DNase), N# (Fla Art-175)</STROMG></P><P>
                                    Toxicity assay: dilute down with varying conc of inducer
                                </P><P>
                                    Different growth conditions - flagellar need TB media 30degC
                                </P><P>
                                    Fluorescent
                                </P><P>
                                    Plate streaking → colony picking → liquid culture → spin down → supernatant contains proteins
                                </P><P>
                                    Protein purification using His-tag affinity (SDS-PAGE then Ni(II)/Co(II) column/nickel-chelated horseradish peroxidase? Or use Western Blot using antibodies selective for His-tag?)
                                </P><P>
                                    Nanodrop at 200nm and 280nm to measure protein concentration in eluent
                                </P><P>
                                    Assaying for DNase:
                                </P><OL><LI>Culturing on DNase test agar (if secretion tag works, no need to purify protein) → visualize DNA hydrolysis either by i) HCl flooding to precipitate DNA out ii) using a dye such as toluidine blue</LI><LI>If we want to make use of the purified proteins - need to have a solution-based assay that can detect polymerized DNA vs hydrolysed DNA?</LI></OL><P>
                                    Assaying for antipseudomonas activity: the Chang paper used PAO1 as target (for DNAse, MccS):
                                </P><OL><LI>Grow PAO1 in 150uL medium in 96-well, incubate 24h with peg lids → rinse pegs with PBS → immerse pegs in plate containing purified DNase → crystal violet staining (alternatively maybe can just grow biofilms at bottom of wells - harder to wash)</LI><LI>Grow PAO1 similarly, then immerse in culture of DNase-secreting E. coli</LI></OL><P>
                                    Crystal violet staining: stain, rinse, de-stain, measure absorbance at 575nm
                                </P><P>
                                    Assaying for Art-175: basically incubate PA (the original Art-175 paper used PA14) with purified Art-175 (or supernatant since our Art-175 has secretion tag) and monitor OD at 600nm over time (refer to paper)
                                </P><P><STRONG>Without secretion tag - H# (MccS), P# (DNase)</STRONG></P><P>
                                    Need to lyse cells (buffer or sono?)
                                </P><P>
                                    MccS - Chang’s original paper simply tested for antibacterial activity by adding purified MccS to a culture of PAO1
                                </P><UL><LI>Secretion tag yes/no study for DNase since we have both</LI><LI>See if DNAse present extracellularly even when there’s no secretion tag</LI></UL></DIV></DIV><DIV class="section" id="cell"><H1>Cell Growth</H1><DIV class="section" id="cell-5"><H5>Week 5</H5><DIV class="section" id="cell-5-23"><H3>Day 23</H3><DIV id="cell-5-23-tox"><H4>Toxicity Assay</H4><P>
                                            Arabinose induction of pBAD expression was studied at various arabinose concentrations to test for toxicity of proteins encoded by our inserts.
                                        </P><P>
                                            Each well has total volume of 200µL - culture (1/200 dilution), arabinose, media.
                                        </P><P>
                                            96-well plate loaded in triplicate according to the following plan:
                                        </P><P>
                                            DF = deltaFliC
                                        </P><P>
                                            MG = MG1655
                                        </P><P>
                                            Percentages denote final w/v concentration of arabinose in each well
        </P><P>
                                            Initial monitoring shows that the wells containing only Blank LB has increasing OD600 over time - likely to be contaminated LB, makes the rest of the dataset unreliable as well as the cells were inoculated using the same bottle of LB.
                                        </P><P>
                                            Forgot to turn on incubator so the entire curve was done at room temperature.
                                        </P></DIV></DIV><DIV class="section" id="cell-5-24"><H3>Day 24</H3><DIV id="cell-5-24-tox"><H4>Toxicity Assay</H4><P>
                                            Forgot to turn on incubator so the entire curve was done at room temperature.
                                        </P></DIV></DIV><DIV class="section" id="cell-5-25"><H3>Day 25</H3><DIV id="cell-5-25-tox"><H4>Toxicity Assay</H4><P>
                                            Wrong antibiotic in supposed positive control wells.
                                        </P><P>
                                            Temperature - 37degC.
                                        </P></DIV></DIV></DIV><DIV class="section" id="cell-6"><H5>Week 6</H5><DIV class="section" id="cell-6-26"><H3>Day 26</H3><DIV id="cell-6-26-tox"><H4>Toxicity Assay</H4><P>
                                            96 well plate <A href="#">layout</A></P><P>
                                            Pipetting triplicates only - 1 eppendorf split between 3 wells
                                        </P><TABLE class="table table-striped"><TBODY><TR><TH>Eppendorf Set Up</TH><TH>Volume (µL)</TH></TR><TR><TD>LB + Amp OR Tryptone + Amp (both 1:1000 dilution of Amp)</TD><TD>985</TD></TR><TR><TD>20% OR 2% OR 0.2% Arabinose</TD><TD>10</TD></TR><TR><TD>Stationary phase overnight culture</TD><TD>5</TD></TR></TBODY></TABLE><P>
                                            Plate reader: toxicity assay script
                                        </P><UL><LI>90 cycles, 900 seconds</LI><LI>200rpm shake when idle</LI><LI>LB Amp: 37degC</LI><LI>Tryptone Amp: room temp (forgot to set to 30</LI></UL></DIV></DIV><DIV class="section" id="cell-6-27"><H3>Day 27</H3><DIV id="cell-6-27-tox"><H4>Reapeat of Yesterday</H4><P>
                                            Toxicity assay, in Tryptone Broth at 30degC
                                        </P></DIV></DIV><DIV class="section" id="cell-6-29"><H3>Day 29</H3><DIV id="cell-6-29-tox"><H4>Toxicity Assay</H4><P>
                                            Replace 0.002% Arabinose with MilliQ.
                                        </P><P>
                                            85 cycles, 900 seconds, 30degC
                                        </P></DIV></DIV></DIV><DIV class="section" id="cell-8"><H5>Week 8</H5><DIV class="section" id="cell-8-36"><H3>Day 36</H3><DIV id="cell-8-36-tox"><H4>Toxicity Assay</H4><P>
                                            It has been previously established for E#, J#, K#, N#, P# that 0.2% arabinose incubation is non-toxic (for H# deltaFliC, toxic even under 0.002%). For all constructs, 0.3% and 0.4% are both quite toxic levels of induction.
                                        </P><P>
                                            We now want to investigate whether 0.2% arabinose induction is toxic for new constructs D#, L#, and O#.
                                        </P></DIV></DIV><DIV class="section" id="cell-8-37"><H3>Day 37</H3><P>
                                        Set up 1%, 2% and 4% arabinose toxicity assay for (E, H, J, K, N, P)# and two more rows for controls.
                                    </P></DIV><DIV class="section" id="cell-8-38"><H3>Day 38</H3><DIV class="section" id="cell-8-38-tox"><H4>Toxicity Assay</H4><P>
                                            Set up 1%, 2% and 4% arabinose toxicity assay for (E, H, J, K, N, P)# and two more rows for controls.
                                        </P><P>
                                            Toxicity data will be available mid-morning and will show if we can use a higher % ara; if higher [ara] works, incubate overnights with 4% ara (or appropriate according to toxicity assay) and SDS-PAGE.
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="protein"><H1>Expression and Secretion</H1><DIV class="section" id="protein-6"><H5>Week 6</H5><DIV class="section" id="protein-6-29"><H3>Day 29</H3><DIV id="protein-6-29-tca"><H4>TCA Protein Precipitation</H4><P>
                                            For the protocol see <A href="#">here.</A></P><P>
                                            Using old supernatants, and potentially spin down the cells from yesterday.
                                        </P><P>
                                            Couldn’t have worked because arabinose wasn’t added to cell cultures → suspension wouldn’t have had secreted proteins.
                                        </P></DIV></DIV><DIV class="section" id="protein-6-30"><H3>Day 30</H3><DIV id="protein-6-30-secrection"><H4>Secretion Assay</H4><P>
                                            Culture cells to mid-log in LB and TB
                                        </P><P>
                                            For LB, keep in 37degC throughout
                                        </P><P>
                                            For TB, once reaching mid-log, move to 30degC
                                        </P><P>
                                            Culture for another 1 hour
                                        </P><P>
                                            Take 1.6mL of the culture, centrifuge, keep supernatant and discard cells
                                        </P><P>
                                            TCA precipitation protocol
                                        </P><P>
                                            (can refer to Andi’s by going back out to this week’s folder)
                                        </P><P>
                                            Catch Jia when she’s around and get her to show us where the SDS-PAGE stuff are.
                                        </P></DIV></DIV></DIV><DIV class="section" id="protein-7"><H5>Week 7</H5><DIV class="section" id="protein-7-32"><H3>Day 32</H3><DIV id="protein-7-32-secrection"><H4>Secretion Assay</H4><P>
                                            EJNK# in both strains, 1.5 hours incubation at LB at 37degC (1:10 dilution of overnight stationary culture), then 4 hours incubation with 0.2% final conc of arabinose.
                                        </P></DIV></DIV><DIV class="section" id="protein-7-34"><H3>Day 34</H3><DIV id="protein-7-34-tca"><H4>TCA-Precipitate</H4><P>
                                            TCA-precipitate cell-free supernatant of EJKN# and run SDS-PAGE
                   </P><P>
                                            E#MG, N#MG no secretion; J#MG, K#MG, J#DF, K#DF, N#DF lysed; E#DF successful secretion.
                                        </P></DIV></DIV></DIV><DIV class="section" id="protein-8"><H5>Week 8</H5><DIV class="section" id="protein-8-36"><H3>Day 36</H3><DIV id="protein-8-36-super"><H4>Supernatant Acquisition</H4><P>
                                            Wash 1.5mL of overnight cultures of E#, J#, K#, L#, N#, ctrl MG1655 in antibiotic-free LB Broth twice and grow them in 1:50 dilution with 0.2% arabinose in conical flasks. Obtain 4x1.4mL aliquots of their supernatants after 4 hours incubation at 37degC.
                                        </P><P>
                                            Do the same for H# MG1655, except don’t obtain aliquots of its supernatant. Instead, obtain aliquots of culture so that we eventually can lyse the cells to obtain MccS from the lysate.
                                        </P></DIV><DIV id="proetin-8-36-sds"><H4>SDS-Page</H4><P>
                                            Obtain supernatants of deltaFliC constructs from cold room - carry out standard SDS-PAGE protocol.
                                        </P></DIV></DIV><DIV class="section" id="protein-8-37"><H3>Day 38</H3><DIV id="protein-8-37-assay"><H4>Assaying</H4><P>
                                             Run PAGE on the supernatants obtained from the M9 MG1655 cultures from yesterday. Do this first thing in the morning and hopefully as we move onto doing the interlab stuff the gel will develop enough to show some bands.
                                        </P><P>
                                            If bands formed on the PAGE we left to develop overnight, we will incubate the now cleaned biofilm plates with the same supernatants used to PAGE yesterday.
                                        </P></DIV></DIV><DIV class="section" id="protein-8-38"><H3>Day 38</H3><P>
                                        So far, only E# has worked for SDS-PAGE. Need to optimize for the rest of the constructs - (D,E,H,J,K,L,N,O,P)#
                                    </P><P>
                                        Today, let’s just focus on the Art-175 constructs because it’s too difficult to do all of them. So, that’s (L, N, O and K)# overnights - see TCA precipitation (taking up8 flasks)
                                    </P><P>
                                        When setting up flasks for supernatants, wash with antibiotic free media.
                                    </P><P>
                                        → Also try Art-175 supernatants against the biofilm degradation plates
                                    </P></DIV><DIV class="section" id="protein-8-39"><H3>Day 39</H3><P>
                                        Run SDS-PAGE in the morning for the supernatants
                                    </P><P>
                                        (L, E, J and P)# prepare supernatants and run PAGE. Potentially do more if growing in falcon tubes
                                    </P></DIV><DIV class="section" id="protein-8-40"><H3>Day 40</H3><P>
                                        (L, E, J and P)# left secreting overnight - isolate supernatants for SDS-PAGE
                                    </P></DIV></DIV><DIV class="section" id="protein-9"><H5>Week 9</H5><DIV class="section" id="protein-9-42"><H3>Day 42</H3><DIV id="protein-9-42-tca"><H4>Set Up for TCA</H4><P>
                                            Dilute the 4 Art-175 overnights - 16 conical flasks
                                        </P><P>
                                            Secrete at 18C overnight and TCA ppt tomorrow
                                        </P></DIV><DIV id="protein-9-42-secretion"><H4>Secretion</H4><P>
                                            Prepare 50/100 ml volumes of Ara + cell cultures for purification from supernatant
                                        </P><P>
                                            Determine buffers for purification.
                                        </P></DIV></DIV><DIV class="section" id="protein-9-43"><H3>Day 43</H3><DIV id="protein-9-43-express"><H4>Expression</H4><P>
                                            Lysis and His tag purification of P# (DNase)
                                        </P><P>
                                            SDS-PAGE for Nickel column fractions collected Day 42
                                        </P><P>
                                            TCA ppt for overnight secretions of Art-175 constructs - store ppts
                                        </P><P>
                                            TCA ppt for the overnights (secretion for four hours in the afternoon)
                                            → Dilute 1/20 in LB + appropriate antibiotic first thing in the morning so that we can start secretion asap
                                        </P></DIV><DIV id="protein-9-43-pur"><H4>Purification</H4><P>
                                            So far - DNase DsbA and DspB DsbA expressed 4hrs, DNase expressed overnight, Art-175 group expressed 4 hrs and overnight
                                        </P><P>
                                            Setting up secretion for DsbA tagged and DspB that I can run on a column tomorrow - 4 hrs secretion
                                        </P></DIV><DIV id="protein-9-43-wester"><H4>Western Blot</H4><P>
                                            Art-175 group expressed 4hrs and overnight - ready to PAGE O,K,N,L
                                        </P><P>
                                            Set up overnights for DsbA group - prepare TCA ppts from overnights tomorrow - also need to set up secretion overnight for DsbA group.
                                        </P></DIV></DIV><DIV class="section" id="protein-9-44"><H3>Day 44</H3><DIV id="protein-9-44-express"><H4>Expression</H4><P>
                                            Art-175 buffers made up
                                        </P><P>
                                            Run columns for Art-175 group (50ml + 20ml 4hrs secretion with no resuspension buffer) + DNase (resuspension buffer)
                                        </P></DIV></DIV><DIV class="section" id="protein-9-45"><H3>Day 45</H3><DIV id="protein-9-45-express"><H4>Expression</H4><P>
                                            MG constructs 1/40 dilution and purify supernatants to run on column
                                        </P><P>
                                            Lysis of cell pellets to run on column
                                        </P><P>
                                            SDS-PAGE for Art-175 group
                                        </P></DIV></DIV></DIV><DIV class="section" id="protein-10"><H5>Week 10</H5><DIV class="section" id="protein-10-46"><H3>Day 46</H3><P>
                                        Grew up 3x J# DF, and H# MG overnights to 0.6 OD, and followed protocol past NiNTA purification, ready for SDS-PAGE tomorrow.
                                    </P><P>
                                        No bands.
                                    </P></DIV><DIV class="section" id="protein-10-47"><H3>Day 47</H3><DIV id="protein-10-47-duke"><H4>Duke (aka king of banter)</H4><P>
                                            Listened to reggae music (you know, Bob Marley, UB40, the classics) whilst eluting L#, M# and O# (all in DeltaFli) through nickel column, ready for SDS-PAGE tomorrow.
                                        </P></DIV></DIV><DIV class="section" id="protein-10-49"><H3>Day 49</H3><DIV id="protein-10-49-western"><H4>Western Blot</H4><P>
                                            NB: Duke/Silas when analysing the results, refer to the gels in the Coomassie blue stain - there is some inconsistency as to how lanes were loaded i.e. if a lane was left free between the ladder and the samples
                                        </P><P>
                                            Refer to the Western blotting protocol
                                        </P><OL><LI>Primary Ab stored as 10 uL aliquots in the -20CTo immunostain, dilute by 1/1000 in 10 mL 1% milk powder and PBS (the full 10 uL aliquot in 10ml milk solution). NB: Andrea said that this will probably be too bright - let’s see what she thinks when we come to analyse the data</LI><LI> The PBS and PBS + Tween 20 (Tween 20 is the same as Tween - it’s a typo on the protocol) solutions are on our shelf</LI><LI>The secondary Ab is in the back fridge, top shelf, has a mouse drawn on it. This also requires 1/1000 dilution into 10 mL 1% milk solution (remember to dilute the 5% milk solution with PBS, not water) for the second staining step</LI><LI>Refer to Andrea’s help for the visualisation step</LI></OL></DIV><DIV id="protein-10-49-secretion"><H4>Secrection</H4><P>
                                            Constructs induced to express and pellets frozen - stored in the -80C
                                        </P></DIV></DIV><DIV class="section" id="protein-10-50"><H3>Day 50</H3><DIV id="protein-10-50-western"><H4>Western Blotting</H4><P>                   Carry on with immunostaining of the blots on the shaker
                                            <UL><LI>Carry on with immunostaining of the blots on the shaker</LI><LI>They were washing in PBS overnight - this is not normally required - in the future stick to the timings on the protocol</LI></UL></P><P>
                                            Immunostain the two blots that have been in milk solution overnight
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="bio"><H1>Biofilms</H1><DIV class="section" id="bio-5"><H5>Week 5</H5><DIV class="section" id="bio-5-25"><H3>Day 25</H3><DIV id="bio-5-25-test"><H4>Test crystal violet staining of biofilms (using 23/07 tox assay plate)</H4><P><A href="#http://www.ncbi.nlm.nih.gov/pubmed/18770545">Protocol.</A></P><P>
                                            0.1% w/v crystal violet solution made up by dissolving 0.080g crystal violet in 80mL Milli-Q.
                                        </P><P>
                                            96-well plate was taken out of plate reader at 10am and left on the bench until 4pm.
                                        </P><P>
                                            Contents of plate decanted into biological waste bucket and each well of plate washed with tap water for 4 rinses to remove planktonic cells.
                                        </P><P>
                                            125µL of 0.1% crystal violet solution added to each well, and left to stain for 20 minutes.
                                        </P><P>
                                            Contents of plate decanted and each well rinsed thoroughly again to remove unstained crystal violet.
                                        </P><P>
                                            Plate left to dry overnight.
                                        </P><P>
                                            Comments: Growth curve assay plates are non-ideal for biofilm screening as the shaking incubation of the plates in the plate reader means that biofilms are formed more on the walls of the wells than the bottom of the wells.
                                        </P></DIV></DIV></DIV><DIV class="section" id="bio-6"><H5>Week 6</H5><DIV class="section" id="bio-6-27"><H3>Day 27</H3><DIV id="bio-6-27-cultivation"><H4>Biofilm Cultivation Using Existing Transformed Strains</H4><P>
                                            Biofilms grown in triplicate wells according to tox assay layout, except the wells with 0.002% final arabinose conc is replaced with 0% arabinose conc (i.e. arabinose replaced with Milli-Q).
                              </P><P>
                                            Each well has 100uL total volume of 1:100 diluted overnight culture.
                                        </P><P>
                                            Round bottomed plate was used.
                                        </P><P>
                                            Left for incubation on bench for 48 hours.
                                        </P></DIV></DIV><DIV class="section" id="bio-6-30"><H3>Day 30</H3><DIV id="bio-6-30-stain"><H4>Biofilm Staining</H4><P>
                                            First, measure the OD600nm of the biofilm culture.
                                        </P><P>
                                            Decant and wash the round-bottom well plate that has been incubating at room temperature for 2.5 days (see protocol)
                                        </P><P>
                                            Do CV staining and monitor 500-600nm using whichever machine is available
                                        </P><P>
                                            If biofilms are shown to grow, we will proceed with monitoring biofilm kinetics
                                        </P></DIV></DIV></DIV><DIV class="section" id="bio-7"><H5>Week 7</H5><DIV class="section" id="bio-7-31"><H3>Day 31</H3><DIV id="bio-7-31-assay"><H4>Assays</H4><P>
                                            Various biofilm assays
                                        </P><P>
                                            3 modified E. coli plates and co-culture with wild-type (for testing of biofilm inhibition); 3 wild-type plates for eventual testing of biofilm degradation
                                        </P><P>
                                            N.B.30degC plate (modified E. coli plate) has B6 and B7 swapped around
                                        </P></DIV></DIV><DIV class="section" id="bio-7-34"><H3>Day 34</H3><DIV id="bio-7-34-setup"><H4>Set Up</H4><P>
                                            Set up conical flask cultures for E# and J# (both strains) to test with wt-E. coli biofilm plates
                                        </P><P>
                                            1:10 dilution with LB amp
                                        </P><P>
                                            1.5 hours initial incubation, 4 hours incubation with 0.2% final conc of arabinose (190µl of 20% arabinose in 19ml of solution), incubation from 12:30 to 4:30pm
                                        </P><P>
                                            OD values after initial incubation:
                                        </P><P>                   hisB MG : 1.12, hisB deltaFliC : 0.73, E# MG : 1.1, E# deltaFliC : 0.72, J# MG : 0.74, J# deltaFliC : 1.2
                                        </P><P>
                                            4mL of supernatant isolated from each culture, kept in cold room for addition to E. coli biofilms set up on 03/08
                                        </P></DIV></DIV><DIV class="section" id="bio-7-35"><H3>Day 35</H3><DIV id="bio-7-35-assay"><H4>Biofilm Assaying</H4><P>
                                            Set up P. putida 96-well biofilm plates and incubate for 48 hours at 30 degC without shaking: P. putida overnight culture diluted 1:100 in a square petri dish in LB Broth (no antibiotics), and 100uL of the diluted culture filled into each well of one tissue culture-treated round bottom-well 96-well plate.
                                        </P><P>
                                            Set up E. coli MG1655 96-well biofilm plates and incubate for 48 hours at room temperature: MG1655 (with pBAD/HisB) diluted 1:100 in a square petri dish in LB Broth supplemented with ampicillin, and 100uL of the diluted culture filled into each well of one tissue culture-treated round bottom-well 96-well plate.
                                        </P><P>
                                            Incubate E. coli biofilm plates that were cleaned yesterday with supernatants from E# and J# that were isolated yesterday.
                                        </P><P>
                                            Wash some overnight cultures of E#, J#, K#, N#, and control deltaFliC with non-antibiotic tryptone broth (same way as Interlab step ii) twice, then grow them in non-antibiotic tryptone broth (1:50 dilution, make 20mL cultures) with a final arabinose concentration of 0.2% for 4 hours. The supernatant here will eventually be used for both secretion assay and biofilm assay/cell-killing assay with P. putida. 4x 1.4mL aliquots of supernatants were obtained from each culture.
                                        </P></DIV></DIV></DIV><DIV class="section" id="bio-8"><H5>Week 8</H5><DIV class="section" id="bio-8-36"><H3>Day 36</H3><DIV id="bio-8-36-assay"><H4>Biofilm Assaying</H4><P>
                                            NB: All we’ve been doing thus far is getting constructed E. coli to grow biofilms (do our constructed cells with ara make biofilms? do cells not secreting grow biofilms? Always do an OD during biofilm prep to make sure you have cells!)
                                        </P><P>
                                            Currently, we also have wild type E. coli growing biofilms to which we add constructed E. coli supernatant
                                        </P><P>
                                            Retrieve plates that have E. coli biofilms being incubated with supernatants from 37 degC incubator, wash and do CV staining to test for biofilm degradation (Leon remember to fill in the layout file for this (dated 07/08). (Raffy, the OD of these plates wasn’t taken thus ii) being done)
                                        </P><P>
                                            Retrieve MG1655-biofilm growth plate (for eventual degradation) from benchtop, take OD600 reading to make sure every well has had cells in them, then decant and wash planktonic cells away. The plate is then ready to be incubated with supernatants.
                                        </P><P>
                                            Retrieve P. putida-biofilm growth plate (for eventual degradation) from 30 degC, take OD600 reading to make sure every well has had cells in them, then decant and wash planktonic cells away. The plate is then ready to be incubated with supernatants (make sure antibiotic free supernatants - the ones in cold room made up on Friday are antibiotic free).
                                        </P><P>
                                            Set up again biofilm viability plates using cultures of our constructed E. coli. Can potentially set up two plates, both of which left on bench (~25degC incubation). (Raffy, is biofilm growth viable?)
                                        </P><P>
                                            Set up biofilm inhibitory plates by culturing P. putida in supernatants of our constructed E. coli, as well as non-constructed E. coli as controls.
                                        </P></DIV></DIV><DIV class="section" id="bio-8-37"><H3>Day 37</H3><P>
                                        If visual inspection tells you that P. putida biofilms indeed have formed in the previous 96-well plate, set up two more of them using the test tube of P. putida in the 30 degC incubator (1:100 dilution per well, 100µL of diluted culture per well).
                                    </P></DIV><DIV class="section" id="bio-8-38"><H3>Day 38</H3><DIV id="bio-8-39-assay"><H4>Biofilm Viability Assays</H4><P>
                                            Can our constructed cells form biofilms - set up after lunchtime.
                                        </P><P>
                                            PAGE tells us that supernatants aren’t great, so even if we see degradation, results aren’t conclusive.
                                        </P></DIV><DIV id="bio-8-38-de"><H4>Biofilm degradation </H4><P>
                                            E. coli and putida plates are drying - test Art-175 supernatants.
                                        </P></DIV><DIV id="bio-8-38-in"><H4>Biofilm inhibition</H4><P>
                                            Incubate bacteria together with the stuff that forms biofilm and see if biofilm forms.
                                        </P></DIV></DIV><DIV class="section" id="bio-8-39"><H3>Day 39</H3><P>
                                        We have set up viability (on the bench) and is according to Raffy’s toxicity map 12/08, leave another day to grow.
                                        <UL><LI>Set up another viability assay same as yesterday</LI></UL></P><P>
                                        Also have set up degradation for putida and E. coli - write the map down; stain tomorrow afternoon
                                    </P><P>
                                        Grow some more biofilms to degrade
                                    </P><P>
                                        Biofilm inhibition assays - grow E. coli biofilms in DspB and DNase supernatant
                                    </P></DIV><DIV class="section" id="bio-8-40"><H3>Day 40</H3><P>
                                        E. coli biofilms growing on the bench (set up at 5 on 14/08) - leave until Saturday
                                    </P><P>
                                        Read biofilm degradation - retrieve 96 well plates from incubator. Read OD600, then wash and read OD spectra between 500-600
                                    </P><P>
                                        Use the overnight supernatants to degrade the biofilms growing from 12/08
                                    </P><P>
                                        Set up another inhibition plate
                                    </P></DIV></DIV><DIV class="section" id="bio-9"><H5>Week 9</H5><DIV class="section" id="bio-9-42"><H3>Day 42</H3><DIV id="bio-9-42-setup"><H4>Set Up</H4><UL><LI>O# and L# O% arabinose growth curve and biofilm viability</LI><LI>Test 1 day old putida biofilm with supernatants</LI><LI>Biofilm viability for E# and J#</LI></UL><P>
                                            NB: *** Take OD reading before you stain the biofilm (just because they don’t form biofilms when you induce expression)
                                        </P><P>
                                            Didn’t overnight putida - overnight tomorrow to test to cell killing assay on - also grow another putida biofilm
                                        </P></DIV></DIV></DIV><DIV class="section" id="bio-11"><H5>Week 11</H5><DIV class="section" id="bio-11-54"><H3>Day 54</H3><DIV id="bio-11-54-test"><H4>P. putida testing</H4><P>
                                            1 plate of biofilm grown for 1 day
                                        </P><P>
                                            K, L, N, O, HisB, grown at 30°C for 2 hours, then induced with 0.4% ara at 30°C for 4 hours, then left to sit at room temperature for 2 hours
                                        </P><P>
                                            100µL of supernatant from above added to 100µL of stationary culture of P. putida, and 100µL of 1:20 diluted culture which have been left sitting for 4 hours.
                 </P><P>
                                            Run in clariostar together with toxicity assay
                                        </P><P>
                                            Preliminary observations: Toxicity in H and L as expected
                                        </P><P>
                                            N supernatant displays some extent of P. putida-killing capacity - secretion conditions to be further investigated.
                                        </P></DIV><DIV id="bio-11-54-bio"><H4>Biofilm Viability Assay</H4><P>
                                            Stationary cultures diluted 1:100 into 200uL Amp-supplemented LB per well.
                                        </P><P>
                                            Incubation at room temperature.
                                        </P><TABLE class="table table-striped"><TBODY><TR><TD>K MG 0.2% ara (1-3)</TD><TD>L MG 0.2% ara (4-6)</TD><TD>N MG 0.2% ara (7-9)</TD><TD>O MG 0.2% ara (10-12)</TD></TR><TR><TD>K MG 0.02% ara (1-3)</TD><TD>L MG 0.02% ara (4-6)</TD><TD>N MG 0.02% ara (7-9)</TD><TD>O MG 0.02% ara (10-12)</TD></TR><TR><TD>K MG 0% ara (1-3)</TD><TD>L MG 0% ara (4-6)</TD><TD>N MG 0% ara (7-9)</TD><TD>O MG 0% ara (10-12)</TD></TR><TR><TD>ctrl MG 0.2% ara (1-3)</TD></TR><TR><TD>ctrl MG 0.02% ara (1-3)</TD></TR><TR><TD>ctrl MG 0% ara (1-3)</TD></TR></TBODY></TABLE><P>
                                            Observation: L MG 0.2% and 0.02% shows significantly impeded cell growth (low cell density) - perhaps it would be better to test for biofilm growth by placing them in wells for induction using a mid-log culture (grown without arabinose beforehand).
                                        </P><H4>Biofilm viability assay</H4><P>
                                            1:100 dilution of stationary cultures into 100µL of antibiotic-supplemented LB in each well of round-bottom 96-well plate.
                                        </P><P>
                                            1 plate for MG, 1 plate for RP. Same layout.
                                        </P><P>
                                            Incubation at room temperature for 2 day.
                                        </P><TABLE class="table table-striped"><TBODY><TR><TD>0.2% ara (1-3)</TD><TD>0.02% ara (4-6)</TD><TD>0% ara (7-9)</TD><TD>10-12</TD></TR><TR><TD>E</TD><TD>HisB 0.2% ara</TD></TR><TR><TD>H</TD><TD>HisB 0.02% ara</TD></TR><TR><TD>J</TD><TD>HisB 0% ara</TD></TR><TR><TD>K</TD></TR><TR><TD>L</TD></TR><TR><TD>N</TD></TR><TR><TD>O</TD></TR><TR><TD>P</TD></TR></TBODY></TABLE><P>
                                            Washed and stained biofilm viability assay, but not yet analyzed
                                        </P><P>
                                            Preliminary observation seems to suggest that incubating for one day is not enough to form detectable biofilms.
                                        </P></DIV><DIV id="bio-11-54-weekend"><H4>Over the Weekend</H4><P>
                                            Biofilm degradation assay
                                            <UL><LI>Retrieve 48-hour putida biofilm plate from Day 54 - attempt degradation using artilysin supernatants</LI><LI>(on Monday, consult Chris on how to tune Ca2+ concentration and pH of supernatant)</LI></UL></P><P>
                                            Analyze MG Day 54 biofilms
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="kill"><H1>Cell Killing</H1><DIV class="section" id="kill-8"><H5>Week 8</H5><DIV class="section" id="kill-8-38"><H3>Day 38</H3><P>
                                        Incubate putida with supernatant (add supernatant as you would media when setting up plates of putida).
                                    </P><P>
                                        NB: growing of flagella (i.e. Fla secretion) will work better, according to George, at 30˚C
                                    </P></DIV><DIV class="section" id="kill-8-39"><H3>Day 39</H3><P>
                                        Lyse P, O and H to show that it is being expressed just not secreted - ask Chris or George about sonication.
                                    </P></DIV></DIV></DIV><DIV class="section" id="fluor"><H1>Fluorescence</H1><DIV class="section" id="fluor-6"><H5>Week 6</H5><DIV class="section" id="fluor-6-29"><H3>Day 29</H3><DIV id="fluor-6-29-growth"><H4>Growth Curve</H4><P>
                                            For the protocol see <A href="#">here.</A></P><P>
                                            Plate reader hanged on first run; forgot to turn the incubator back on after resetting the program, so it’s a room temperature growth curve.
                                        </P></DIV></DIV><DIV class="section" id="fluor-6-30"><H3>Day 30</H3><DIV id="fluor-6-30-assay"><H4>Assay</H4><P>
                                            Fill black well plate with 100uL of stat phase overnights, same layout as usual, take instantaneous read (fluor and OD) using FluorStar.
                                        </P></DIV></DIV></DIV><DIV class="section" id="fluor-8"><H5>Week 8</H5><DIV class="section" id="fluor-8-56"><H3>Day 36</H3><DIV id="fluor-8-56-quorum"><H4>Quorum-Sensing Fluorescence Assay</H4><P>
                                            Wash C* MG1655 and deltaFliC in modified M9 media (+ thiamine supplement) once. Dilute 1:100 and monitor growth in microplate wells.
                                        </P></DIV></DIV></DIV><DIV class="section" id="fluor-10"><H5>Week 10</H5><DIV class="section" id="fluor-10-50"><H3>Day 50</H3><DIV id="fluor-10-50-quor"><H4>Quorum-Sensing Fluorescence</H4><P>
                                            1/20 dilution of C* and pBAD33 in LB + Chl
                                        </P><P>
                                            Incubate until OD ~ 0.6
                                        </P><P>
                                            Notes:
                                            <UL><LI>C* is pLsr - GFP</LI><LI>“We are using this part to qualitatively test that pLsr functions as expected in response to AI-2, as well as quantitatively measure the kinetics of gene expression when pLsr is induced by a given amount of AI-2.”</LI><LI>We want to show that GFP expression can be induced by AI-2. To do this, it is necessary to use the supernatant of C* in exponential phase and culture stationary phase bacteria (overnight culture) in the supernatant</LI></UL></P><P>
                                            Producedure:
                                            <UL><LI>Incubate cultures to correct OD</LI><LI>Transfer to a falcon tube and spin down at full spin for 15 mins</LI><LI>Transfer the supernatant to separate falcon tubes</LI></UL></P><P>
                                            Set up 70-80 cycles (~25-30 hours) OD + GFP
                                        </P></DIV></DIV></DIV></DIV><DIV class="section" id="flow"><H1>Flow cytometry</H1></DIV></DIV><DIV class="section" id="interlab"><H2>Interlab</H2><DIV class="section" id="2-i"><H5>Week 2</H5><DIV class="section" id="2-8-i"><H3>Day 8</H3><DIV id="2-8-i-prep"><H4>Preparation of Interlab Study BioBricks</H4><P>
                                        10µL Milli-Q added to wells 20K, 22A, and 22K in Plate 1, and well 21J in Plate 4 in the iGEM Distribution Kit to resuspend the necessary BioBricks [pSB1C3] to prepare for the Interlab study.
                                    </P><P>
                                        The plates were kept on mild shaking until the afternoon.
                                         </P></DIV><DIV id="2-8-i-transform"><H4>Transformation of Interlab Study BioBricks into DH5alpha <A href="#">1.5</A></H4><P>
                                        Conduct plating under the filter hood. Add antibiotic to molten agar whilst in bottle, such that the antibiotic (Chl or Amp) is diluted by 1000 times. Then gently mix. Pour just enough agar to cover the surface of the petri dishes. Plates should take ~30mins to set.
                                    </P><P>
                                        Then add volume of E. coli according to protocol, using beads to spread across petri dish.
                                    </P></DIV></DIV><DIV class="section" id="2-9-i"><H3>Day 9</H3><DIV id="2-9-i-growth"><H4>Growth and Culture of Bacteria</H4><P>
                                        Refer to section <A href="#">1.6</A> of the protocol guide.
                                    </P><P>
                                        Only one overnight culture each set up for iGEM distros (20K, 22A, 22K, 21J) as it can be rightly assumed that plasmids supplied by iGEM HQ are high purity.
                                    </P></DIV></DIV><DIV class="section" id="2-10-i"><H3>Day 10</H3><DIV id="2-10-i-gel"><H4>Plasmid for PCR Products</H4><P>
                                        Digest aliquots of each of the samples, and run on gel. (Refer to protocol in section <A href="#">1.2</A>)
                                    </P><P>
                                        From gel, determine the degree of success of these samples.
                                    </P></DIV></DIV></DIV><DIV class="section" id="3-i"><H5>Week 3</H5><DIV class="section" id="3-11-i"><H3>Day 11</H3><DIV id="3-11-i-digest"><H4>Restriction Digest of InterLab Study Plasmids</H4><P>
                                        20K, 22A, 22K:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume / µL</TH></TR><TR><TD>Plasmids</TD><TD>10</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>PstI-HF</TD><TD>1</TD></TR><TR><TD>Milli-Q Water</TD><TD>6</TD></TR><TR><TD>CutSmart</TD><TD>2</TD></TR></THEAD></TABLE><P>
                                        21J: Same as above, but replace SpeI for XbaI.
                                    </P><P>
                                        Incubated for 2hrs at 37℃, then enzymes heat inactivated at 95℃.
                                    </P></DIV><DIV id="3-12-i"><H3>Day 12</H3><DIV id="3-12-i-digest"><H4>Restriction Digest of InterLab BioBricks</H4><DIV class="image image-right"><P>Gel Photo</P></DIV><OL><LI>Pulse spin</LI><LI>Dephosphorylate 20K, 22A, 22K, <STRONG>BUT NOT 21J</STRONG></LI><LI>Load dye and run gel</LI></OL><P>
                                            Gel image (transilluminator lens was quite grainy today)
                                        </P><P>
                                            Appropriate bands excised for extraction (I13504 for last lane)
                                        </P></DIV><DIV id="3-12-i-extraction"><H4>Gel extraction of InterLab BioBricks</H4><P>
                                            340uL of Binding Buffer;
                                        </P><P>
                                            one elution with 50uL for 20K, 22A, 22K; 30uL for 21J
                                        </P></DIV><DIV id="3-12-i-ligation"><H4>Ligation of InterLab sequences</H4><P>
                                            Insert: digested 21J
                                        </P><P>
                                            Plasmids: 20K, 22A, 22K
                                        </P><TABLE class="table table-striped"><THEAD><TR><TH>Component</TH><TH>Volume/µL</TH></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>T4 Buffer</TD><TD>4</TD></TR><TR><TD>Insert</TD><TD>10</TD></TR><TR><TD>Plasmid</TD><TD>10</TD></TR><TR><TD>Milli-Q</TD><TD>26</TD></TR></THEAD></TABLE><P>                       NB: total reaction volume of 51µL is 1µL more than ideal
                                        </P><P>
                                            NB: total reaction volume of 51uL is 1uL more than ideal
                                        </P></DIV></DIV></DIV><DIV class="section" id="3-15-i"><H3>Day 15</H3><DIV id="3-15-i-group-d"><H4>Group D</H4><P>
                                        Miniprep plasmid extraction for 20K and 22A tubes.
                                    </P><P>
                                        Nanodrop concentration measurement. Tom noted that most of our concentrations are too low for sequencing (≤100ng/μl) and suggested that we elute using 35μl elution buffer that had been prewarmed in 55℃ water bath and then pass the filtrate through column and spin down again for better yields.
                                    </P><P>
                                        In the afternoon we performed the restriction digests of the plasmid and prepared the gel to be run on Monday for the digest products.
                                    </P><P>
                                        The gel has been left in a tank with buffer and a lid.
                                    </P></DIV></DIV></DIV><DIV class="section" id="4-i"><H5>Week 4</H5><DIV class="section" id="4-16-i"><H3>Day 16</H3><DIV id="4-16-i-lychee"><H4>Lychee</H4><P>
                                        Run gel for digested plasmids from 10/7. Check if bands are of correct size, if so, send sample for sequencing
                                    </P><P>
                                        Gel run no.1: Bands overrun into the bottom half of the gel - redo. But 21K[22A] and 21J[22K] seemed to show correct-sized bands → send for sequencing
                                    </P><P>
                                        Did gel extraction for 21J bands from gel but did not obtain a satisfactory concentration of DNA -&gt; restriction digest to obtain 21J tomorrow
                                    </P><P>
                                        21J[22A] and 21J[22K(2)] sent for sequencing
                                    </P></DIV></DIV><DIV class="section" id="4-17-i"><H3>Day 17</H3><DIV id="4-17-i-group-d"><H4>Group D</H4><P>
                                        Restriction digest for 20A to get 21J. Religate 21J into plasmid.
                                    </P></DIV><DIV id="4-17-i-digest"><H4>Digest the remainder of 10/07 miniprep products and run on gel</H4><P>
                                        Rest of bands are still not giving the right inserts, but 20K seems to give the right size → sent for sequencing on 15/07, and stored in freezer
                                    </P></DIV></DIV><DIV class="section" id="4-20-i"><H3>Day 20</H3><DIV id="4-20-i-ria"><H4>Ria and Lychee</H4><P>
                                        Interlab plasmids 20K, 22A, 22K are ready for transformation.
                                    </P><P>
                                        All 20K, 22A, 22K plasmids used up in transformation into DH5alpha because there didn’t seem to be enough plasmids - need to miniprep a portion of the overnight cultures of these transformed cells to replenish plasmid stock.
                                    </P></DIV></DIV></DIV><DIV class="section" id="5-i"><H5>Week 5</H5><DIV class="section" id="5-21-i"><H3>Day 21</H3><DIV id="5-21-i-miniprep"><H4>Miniprepping of 19/07 overnight cultures</H4><P>
                                        For interlabs - make frozen stock with 700µL of each culture (add 300µL glycerol to it, vortex, and put in -80degC); miniprep the rest of the cultures to replenish plasmid stocks.
                                    </P></DIV><DIV id="5-21-i-transform"><H4>Transformations</H4><P>
                                        Interlab (22A, 22K, 20K) (after miniprep is done)→ MG1655, deltaFliC
        </P><P>
                                        Interlab positive and negative controls:
                                    </P><UL><LI>(+) - BBa_I20270 (Kit Plate 3, Well 8P)</LI><LI>(-) - BBa_R0040 (Kit Plate 2, Well 6F)</LI></UL><P>
                                        Resuspend in 10uL Milli-Q, then transform into DH5alpha
                                    </P></DIV><DIV id="5-21-i-plate"><H4>Plate Streaking</H4><P>
                                        Frozen stocks of DH5alpha 20K, 22K, 22A also streaked
                                    </P></DIV></DIV><DIV class="section" id="5-23-i"><H3>Day 23</H3><DIV id="5-23-i-mini-prep"><H4>Miniprep of DH5α (+) and (-)</H4><P>
                                        Stocks made - 700µl LB + 300µl 60% glycerol
                                    </P><P>
                                        Nanodop:
                                    </P><TABLE class="table table-striped"><THEAD><TR><TH>[DNA] (ng/µl)</TH></TR></THEAD><TBODY><TR><TH>DH5α (+) 1</TH><TD>39.6</TD></TR><TR><TH>DH5α (+) 2</TH><TD>27.1</TD></TR><TR><TH>DH5α (-) 1</TH><TD>28.5</TD></TR><TR><TH>DH5α (-) 2</TH><TD>33.7</TD></TR></TBODY></TABLE><P>
                                        Transformation into FliC and MG1655 - left in incubator overnight
                                    </P></DIV><DIV id="5-23-i-frozen"><H4>Frozen Stock Preparation</H4><P>
                                        Interlab 20K, 22K, 22A in MG1655 and deltaFliC made into frozen stocks
                                    </P></DIV></DIV><DIV class="section" id="5-24-i"><H3>Day 24</H3><DIV id="5-24-i-overnight"><H4>Overnights</H4><P>
                                        Pick colonies and overnights for DH5α (+) and (-) in FliC and MG1655.
                                    </P></DIV></DIV><DIV class="section" id="5-25-i"><H3>Day 25</H3><DIV id="5-25-i-frozen"><H4>Frozen Stocks</H4><P>
                                        For (+), (-) in MG1655, deltaFliC
                                    </P></DIV><DIV id="5-25-i-overnight"><H4>Overnight cultures in M9 Modified Media</H4><P>
                                        (+), (-) DH5alpha, MG1655, deltaFliC from frozen stocks.
                                    </P><P>
                                        20K, 22K, 22A DH5alpha from plate
                                    </P><P>
                                        20K, 22K, 22A MG, FliC from frozen stock
                                    </P><P>
                                        All Chl.
                                    </P></DIV><DIV id="5-25-i-weekend"><H4>Over the Weekend</H4><P>
                                        Overnight cultures taken out onto the bench after being incubated at 37degC, 225 rpm orbital shaking for 20 hours.
                                    </P><P>
                                        100µL culture loaded into each well for GFP fluorescence and OD600 measurement.
                                    </P><P>
                                        Sodium fluorescein used as standard (absolute units) for fluorescence data to be compared against. 1.66g fluorescein (free acid) dissolved in 5mL pH8.0 Tris HCl to obtain 1M fluorescein solution, neutralized with 5mL 2M NaOH to obtain 10mL of 0.5M sodium fluorescein (NaFluo).
                     </P><P>
                                        Serial dilution of NaFluo performed for construction of calibration curve.
                                    </P></DIV></DIV></DIV><DIV class="section" id="6-i"><H5>Week 6</H5><DIV class="section" id="6-27-i"><H3>Day 27</H3><DIV id="6-27-i-growth"><H4>Growth curves for InterLab</H4><P>
                                        *note: 20K DH5alpha cultures failed to grow (overnight test tubes very clear)
                                    </P><P>
                                        *see protocol information in the relevant Excel files
                                    </P></DIV><DIV id="6-27-i-trial"><H4>Fluorescence Microscopy Trial Run</H4><P>
                                        Single 50mL flask filled with 10mL M9 modified media, inoculated with 1mL overnight culture of 20K MG1655; grown for 2 hours up to OD600 = 0.2.
                                    </P><P>
                                        (note: growth in M9 modified media is very slow compared to that in LB broth, use higher concentrations in the future)
                                    </P></DIV></DIV><DIV class="section" id="6-28-i"><H3>Day 28</H3><DIV id="6-28-i-read"><H4>Instantaneous Read</H4><P>
                                        GFP fluorescence and OD600
                                    </P></DIV><DIV id="6-28-i-mid-log"><H4>Culturing Strains to Mid-Log</H4><P>
                                        2mL of each overnight culture added to 10mL modified M9 media in 50mL conical flasks, incubated at 37degC, 225 rpm for ~3 hours.
                                    </P></DIV></DIV><DIV class="section" id="6-29-i"><H3>Day 29</H3><DIV id="6-29-i-micro"><H4>Cultivate cells for microscopy</H4><P>
                                        Microscopy done for MG1655 22A, 20K, (22K was already in stat phase despite being OD600 = 0.2 -- cells were short, non-fluorescent); DH5alpha 22A, 20K, 22K
                                    </P></DIV></DIV></DIV><DIV class="section" id="7-i"><H5>Week 7</H5><DIV class="section" id="7-31-i"><H3>Day 31</H3><DIV id="7-31-i-micro"><H4>Microscopy</H4><P>
                                        Dilutions of overnight cultures in 1:5 and preparing slides for microscopy.
                                    </P><P>
                                        Overnights for Interlab cultures were done in LB Chl, and upon 1:5 dilution in M9 stationary phase was achieved within 1.5 hours (validation via OD 0.7-0.9 and microscopy).
                                    </P></DIV></DIV><DIV class="section" id="7-32-i"><H3>Day 32</H3><DIV id="7-32-i-june"><H4>June</H4><P>
                                        1:5 dilution of interlabs in M9 media, and 45 min of incubation at 37C gives OD value of 0.4 to 0.5. DH5alpha strains showed reasonable fluorescence.
                                    </P></DIV></DIV><DIV class="section" id="7-33-i"><H3>Day 33</H3><DIV id="7-33-i-june"><H4>June</H4><P>
                                        Set up interlab for microscopy
                                    </P></DIV></DIV><DIV class="section" id="7-34-i"><H3>Day 34</H3><DIV id="7-24-i-set-up"><H4>Set Up</H4><P>
                                        Set up conical flask cultures of interlabs for microscopy and flow cytometry.
                                    </P><P>
                                        1:10 dilution in M9 media for 45 minutes gave OD values ranging from 0.27 to 0.35. Only MG1655 cultures were grown and analyzed due to limited number of flasks.
                                    </P></DIV></DIV><DIV class="section" id="7-35-i"><H3>Day 35</H3><DIV id="7-35-i-tasks"><H4>Tasks</H4><P>
                                        Tasks (no specific order; grab Leon for 4. and 5. at an appropriate time):
                                    </P><OL><LI>Make new bottle of M9 Modified Media without thiamine inside.</LI><LI>Wash (i.e. spin down 1mL of culture for 3 minutes at full speed, then decant, and resuspend in unmodified M9 salts solution) overnight interlab cultures that were grown in LB, and take instantaneous fluorescence reads and OD600. Repeat as many reads as you like (can reuse the same plate because the wells will be filled up with the same type of cells anyway, just make sure to fully empty them each time) until the 1mL of culture you washed is depleted (just so that we don’t waste cells and the availability of the machine).</LI><LI>Set up growth assay - aliquot M9 Modified Media into smaller bottle and add thiamine and antibiotics appropriately. Wash LB interlab cultures using this supplemented M9 media, and grow in plate reader for appropriate duration. We only have access to the FluorStar overnight today.</LI><LI>Grow up cells of your choice in conical flasks in LB (under 1 hour if 1:5 dilution; if longer growth time desired dilute in 1:20, 1:50, or even 1:100 as necessary). Stop incubation of these cells at OD600 ~ 0.4 - 0.7, then wash with unsupplemented M9 salts to halt growth. These cells can then be used for microscopy and flow cytometry.</LI><LI>Microscopy: remember to book the microscope under the “confocal” sheet outside the door. Also, after completing session, consult Chris for demonstration on how to clean up microscope objective lens.</LI><LI>Flow cytometry: Consult Chris to arrange a time when George is also available, and work out how to use the flow cytometer properly.</LI><LI>Microscopy image analysis: If no one else has a laptop with MATLAB installed, grab Leon to learn microscopy image processing from Chris. Before that, get Leon to copy all the image files we’ve had thus far out of the microscopy PC.</LI></OL></DIV></DIV></DIV><DIV class="section" id="8-i"><H5>Week 5</H5><DIV class="section" id="8-36-i"><H3>Day 36</H3><DIV id="8-36-i-tasks"><H4>Tasks</H4><P>
                                        Basically repeat as much of the <A href="#7-35">Day 35</A> workflow as possible, briefly:
                                    </P><OL><LI>Wash overnight cultures in M9 media (unsupplemented) for single-read fluorescence and OD600</LI><LI>Wash overnight cultures in modified M9 media (+ thiamine supplement) for growth curve</LI><LI>Set up flasks of washed culture (appropriate dilution) in modified M9 media (+ thiamine supplement) for microscopy and flow cytometry.</LI></OL></DIV></DIV><DIV class="section" id="8-37-i"><H3>Day 37</H3><P>
                                    M9 salts haven't arrived, should start interlab from making M9 media. Same as usual, except please do catch GDubz for cytometry.
                                </P><OL><LI>Wash overnight cultures in M9 media (unsupplemented) for single-read fluorescence and OD600</LI><LI>Wash overnight cultures in modified M9 media (+ thiamine supplement) for growth curve</LI><LI>Set up flasks of washed culture (appropriate dilution) in modified M9 media (+ thiamine supplement) for microscopy and flow cytometry.</LI></OL></DIV><DIV class="section" id="8-38-i"><H3>Day 38</H3><P>
                                    Focus on MGs
                                </P><P>
                                    Flow cytometry - 1/25 cultures in at 9.20
                                </P><P>
                                    Set up for growth curves (only have 2 sets)
                                </P><DIV id="8-38-i-trouble"><H4>Trouble shooting for microscopy</H4><P>
                                        Ask Chris to go through using the microscope
                                    </P><P>
                                        Try a 1/10 dilution
                                    </P></DIV></DIV><DIV class="section" id="8-39-i"><H3>Day 39</H3><OL><LI>Grow falcon tubes - DH5alpha this time → for cytometry and microscopy (1:25 dilution for flask shown to be good for MG)</LI><LI>All cultures overnighted - can do full growth curve</LI></OL></DIV><DIV class="section" id="8-40-i"><H3>Day 40</H3><P>
                                    Grow falcon tubes - 15x interlab overnights 1:20 dilution into Chl LB, for cytometry and microscopy.
                                </P><P>
                                    All cultures overnighted - can do full growth curve
                                </P></DIV></DIV><DIV class="section" id="9-i"><DIV class="section" id="9-41-i"><H3>Day 41</H3><OL><LI>22 overnights to dilute 1/20 in LB + Chl and culture until mid-log OD ~0.4-0.7. Follow flow cyt protocol</LI><LI>Microscopy with mid-log phase cultures</LI></OL><P>
                                    Look at the InterLab page - Imperial 2014. Analyse the microscopy - Chris to show June or Lychee. Not bothering with the growth curves. Compile the flow cytometry data. Microplates - Leon
                                </P></DIV><DIV class="section" id="9-42-i"><H3>Day 42</H3><P>
                                    Finish microscopy
                                </P><P>
                                    Set up calibration curve with following concentrations of sodium fluorescein:
                   </P><TABLE class="table table-striped"><TBODY><TR><TD>0.01µM</TD><TD>0.1µM</TD><TD>0.2µM</TD><TD>1µM</TD><TD>2µM</TD><TD>5µM</TD><TD>10µM</TD><TD>20µM</TD></TR></TBODY></TABLE><P>
                                    This should give a calibration curve that tallies perfectly with the range of interest for spectrophotometer settings.
                                </P></DIV><DIV class="section" id="9-43-i"><H3>Day 43</H3><P>
                                    Analysis of microscopy images for 20K, 22K and 22A in different bacterial strains (DH5ɑ, MG and ΔfliC) using MicrobeTracker.
                                </P><P>
                                    Prepared overnights for (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC to visualise under microscope tomorrow (20/8)
                                </P></DIV><DIV class="section" id="9-44-i"><H3>Day 44</H3><P>
                                    (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC were visualised on the microscope.
                                </P><P>
                                    Overnights set up for (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC for observing tomorrow
                                </P></DIV><DIV class="section" id="9-45-i"><H3>Day 45</H3><P>
                                    Visualised (+)MG, (+)DH5ɑ, (-)DH5ɑ, (+)ΔfliC, (-)ΔfliC
                                </P><P>
                                    Used MicrobeTracker to identify cells in images and ran GFP mesh analysis on MicrobeTracker
                                </P></DIV></DIV><DIV class="section" id="10-i"><H5>Week 5</H5><DIV class="section" id="10-i-47"><H3>Day 47</H3><DIV id="10-i-47-micro"><H4>Microscopy</H4><P>
                                        Visualised 22AΔfliC on the microscope.
                                    </P><P>
                                        Histograms plotted for mean pixel intensity of cells across different strains and the different genes using MicrobeTracker.
                                    </P></DIV></DIV><DIV class="section" id="10-i-48"><H3>Day 48</H3><P>
                                    Mean pixel intensity per cell for each biological replicate calculated using MicrobeTracker. Mean and standard deviations calculated for the different biological replicates.
                                </P><P>
                                    Flow cytometry data recorded its mean and standard deviation calculated.
                                </P></DIV></DIV></DIV><DIV class="section" id="syn"><H2>Synbiota</H2><DIV class="section" id="syn-7"><H5>Week 7</H5><DIV class="section" id="syn-7-33"><H3>Day 33</H3><P>
                                    PCRs
                                </P><UL><LI>pLas RDP F and R - use B gBlocks (157bp) - 72*C</LI><LI>T4 Holin F and R - use D gBlock (672bp) - 72*C</LI><LI>ArtE F and R - use M gBlock (594bp) - 72*C</LI><LI>pLsr F and R - use D gBlock (90bp) - 72*C</LI><LI>LasR F and R - use B gBlock (717bp) - 72*C</LI></UL><P>
                                    PCR conditions: 10-40s per kb recommended. None of the sections are 1kb, and for ease of PCR, I’ve used a 20s extension time for all. If this doesn’t work, we can always try again, but I think shorter than 20s might be too quick… I’ve also used 30 cycles.
                                </P><P>
                                    PCR products in freezer, ready for gel tomorrow morning. Gel has been poured and will be left in buffer overnight.
                                </P></DIV><DIV class="section" id="syn-7-34"><H3>Day 34</H3><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                    Run gel for K# → K* and Synbiota RDP PCR done yesterday
                                </P><P>
                                    For LasR and pLas, the RDP parts can be PCR-ed from parts already in the iGEM distribution kit instead.
                                </P><UL><LI>LasR - BBa_C0179: 2015 distro Kit Plate 3, Well 6M (pSB1C3)</LI><LI>pLas - BBa_R0079: 2015 distro Kit Plate 3, Well 5C (pSB1C3)</LI></UL><P>
                                    Transform these tomorrow.
                                </P></DIV></DIV><DIV class="section" id="syn-8"><H5>Week 8</H5><DIV class="section" id="syn-8-37"><H3>Day 37</H3><P>
                                    Synbiota PCR - Art-E from M or T4 holin from D; synbiota pack is in the cold room - they supply their own ladder as well as buffer to dilute primers in. Phusion can be obtained from Chris, and we have our own Phusion HF buffer in the -20deg
                                </P><P>
                                    Miniprep the overnight cultures for T4Endo, LasR, pLas
                                </P></DIV><DIV class="section" id="syn-8-38"><H3>Day 38</H3><P>
                                    Continue with Synbiota clean-up protocol (Art-E PCRed yesterday according to protocol and is in the freezer) (Mabel - now given to Leon to run gel)
                                </P></DIV><DIV class="section" id="syn-8-39"><H3>Day 39</H3><DIV class="image image-right"><P>Gel Photo</P></DIV><P>
                                    Carry on with the Art-E synbiota gel in the cold room
                                </P><P>
                                    No bands - George to repeat PCR
                                </P><P>
                                    Clean up synbiota digests in freezer (these are plasmids that have been linearised for PCR). PCR according to synbiota protocol with other Synbiota PCRs - in synbiota box in freezer.
                                </P></DIV><DIV class="section" id="syn-8-40"><H3>Day 40</H3><P>
                                    Carry from on Synbiota PCR - PCR tubes in working shelf of the freezer
                                </P></DIV></DIV><DIV class="section" id="syn-9"><H5>Week 9</H5><DIV class="section" id="syn-9-41"><H3>Day 41</H3><P>
                                    Email Synbiota with gel picture
                                </P><P>
                                    Re-PCR pLas and pLsr
                                </P><P>
                                    PCR Art-E from M* if sequence is good
                                </P><P>              Gel from Friday:
                                    <UL><LI>Lane 1 - not sure</LI><LI>Lane 2 - is pLsr - run again to make sure</LI><LI>Lane 3 - LasR (ignore the larger band)</LI><LI>Lane 4 - T4 holin (primers designed to only amplify holin)</LI></UL></P></DIV><DIV class="section" id="syn-9-43"><H3>Day 43</H3><P>
                                    PCR all synbiota parts again. plsr and t4 holin worked, other two failed. Will keep trying PCR with plas and LasR.
                                </P></DIV><DIV class="section" id="syn-9-44"><H3>Day 44</H3><P>
                                    PCR plas and LasR again. gel -&gt; still nothing. Ran another PCR using a different protocol, but still got nothing.
                                </P></DIV></DIV><DIV class="section" id="syn-10"><H5>Week 10</H5><DIV class="section" id="syn-10-47"><H3>Day 47</H3><P>
                                    Gel order: Ladder, T4 Holin, pLsr, M*- #, M*- #
                                </P><P>
                                    Deformed gel - agarose may have been mixed with water instead of TBE
      </P><P>
                                    Repeat PCR from very beginning… :( :( :(
                                </P><P>
                                    Linearise template in 10ul digests
                                </P><P>
                                    Heat inactivate for 30mins at 95degC before setting up PCR according to RDP protocol
                                </P><P>
                                    Run on 2% agarose gel - 1g in 50ml, 120V for 30 mins (this is too short - can afford to run longer for better separation)
                                </P><P>
                                    Gel order: Ladder, pLsr, T4 Holin, Art-E, LasR, pLas, M* - #, M* - #
                                </P><P>
                                    Excised all bright bands except for Art-E and pLas - repeat PCR tomorrow
                                </P></DIV><DIV class="section" id="syn-10-48"><H3>Day 48</H3><P>
                                    Gel extraction/cleanup of synbiota and M* - # bands
                                </P><P>
                                    Synbiota: eluted with 55degC 2 x 50ul TE buffer
                                </P><P>
                                    M#: eluted with 55 degC 50ul EB buffer
                                </P><P>
                                    Emailed Synbiota with gel picture and nanodrop values
                                </P><P>
                                    Repeat PCR of Art-E and pLas, but digest template with 1ul of template regardless of [DNA] in 10ul digest
                                </P><P>
                                    Template: T4 Endo* and pLas pSB (i.e. the other eppendorf of template to the ones used on 25/08 PCRs)
                                </P><P>
                                    Also add DMSO to PCR set up, at 5% and 10%.
                                </P></DIV><DIV class="section" id="syn-10-49"><H3>Day 49</H3><P>
                                    Run gel of Art-E and pLas PCR from Day 48
                                </P><P>
                                    N.B. 5% or 10% = % of DMSO in PCR set up
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>D#- *</TD><TD>D#- *</TD><TD>Art-E 5%</TD><TD>Art-E 5%</TD><TD>Art-E 10%</TD><TD>Art-E 10%</TD><TD>pLas 5%</TD><TD>pLas 5%</TD><TD>pLas 10%</TD><TD>pLas 10%</TD></TR></TBODY></TABLE><P>
                                    Left ladder: 50bp (from Chris, in synbiota box)
                                </P><P>                               Right ladder: 1kb+ ladder
                                </P><P>
                                    Farmost right well has loading dye (trying to identify an unnamed eppendorf on bench)
                                </P><P>
                                    Art-E’s brightest bands are too small - it is ~220bp long when it should be 629bp long.  Potentially due to primers being ordered wrong and getting promoter?
                                </P><P>
                                    Excised both D* bands (880bp, now in freezer next to M#)
                                </P><P>
                                    Awaiting reply from Patrick for Synbiota
                                </P></DIV></DIV><DIV class="section" id="syn-11"><H5>Week 11</H5><DIV class="section" id="syn-11-52"><H3>Day 52</H3><P>
                                    Patrick replied - reattempt with:
                                    <UL><LI>Double the reaction volume for those that worked (200ul rxn)</LI><LI>GC buffer for those that didn’t work</LI></UL></P><P>
                                    Art-E and pLas: use template from digests done on 26/08
                                </P><P>
                                    pLsr, T4 Holin and LasR: linearise template in 10ul reactions as on 25/08, with the exception of LasR which needs [template] correction.
                                </P><P>
                                    Art-E and pLas: GC buffer
                                </P><P>
                                    pLsr, T4 Holin, LasR: 200ul reaction (will need to divide into 4 wells)
                                </P><P>
                                    Gel: 2% agarose in TBE, 120V
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>LasR</TD><TD>LasR</TD><TD>LasR</TD><TD>LasR</TD><TD>T4 Holin</TD><TD>T4 Holin</TD><TD>T4 Holin</TD><TD>T4 Holin</TD></TR><TR><TD>pLsr</TD><TD>pLsr</TD><TD>pLsr</TD><TD>pLsr</TD><TD>pLas (GC)</TD><TD>pLas (GC)</TD><TD>Art-E (GC)</TD><TD>Art-E (GC)</TD></TR></TBODY></TABLE><P>
                                    Excised all T4 Holin and pLsr bands - in freezer
                                </P></DIV><DIV class="section" id="syn-11-53"><H3>Day 53</H3><H4>Cleanup of Synbiota pLsr and T4Holin</H4><P>
                                    Had to re-PCR(used x4 elution buffer and had nanodrop results of pLsr: 16 and T4Holin: 9)…… Sorry Mabel!!!
                                </P><P>
                                    Waiting for the protocol of next step from Synbiota
                                </P></DIV><DIV class="section" id="syn-11-54"><H3>Day 54</H3><P>
                                    Proceed with Synbiota RDP Parts Creation Protocol - BsaI digestion Protocol.
                                </P><P>
                                    BsaI Digestion (2.5hrs)
                                </P><P>
                                    DNA cleanup
                                </P><P>
                                    Measuring and adjusting the part concentration
                                </P></DIV><DIV class="section" id="syn-11-55"><H3>Day 55</H3><P>
                                    Repeat pLas, LasR and Art-E at 55 and 59 degC annealing temperatures
                                </P><P>
                                    Digested templates from 25/08 (LasR) or 26/08 (Art-E and pLas)
                                </P><P>
                                    Only do 50ul reaction - if it works then scale up
                                </P><P>
                                    2% TBE agarose, 120V
                                </P><TABLE class="table table-striped"><TBODY><TR><TD>LasR 55</TD><TD>pLas 55</TD><TD>ArtE 55</TD><TD>LasR 59</TD><TD>pLas 59</TD><TD>Art-E</TD></TR></TBODY></TABLE><P>
                                    LasR works!    YAAAAAAAAAAASSSSSSSSSSSSSSSSSSSSS
                                </P><P>
                                    Scale up after the weekend at 59 degC
                                </P></DIV></DIV><DIV class="section" id="syn-12"><H5>Week 12</H5><DIV class="section" id="syn-12-60"><H3>Day 60</H3><P>
                                    Synbiota RDP assembly of D*
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked" data-spy="affix"><LI><A href="#cloning">Cloning</A></LI><LI><A href="#1">Week 1</A></LI><LI><A href="#1-1">Day 1</A></LI><LI><A href="#1-1-prep-stock">Preparation of Stock Solutions</A></LI><LI><A href="#1-1-prep-reaction">Preparation of Reaction Solutions</A></LI><LI><A href="#1-1-pcr">Polymerase Chain Reaction</A></LI><LI><A href="#1-1-gel">Gel Electrophoresis of PCR-Amplified gBlocks</A></LI></UL><LI><A href="#1-2">Day 2</A></LI><LI><A href="#1-2-gel-continued">Gel Electrophoresis</A></LI><LI><A href="#1-2-results">Results</A></LI><LI><A href="#1-2-extraction">Extraction of DNA</A></LI><LI><A href="#1-2-restriction-pcr">Restriction Digest PCR</A></LI><LI><A href="#1-2-restriction-enzyme">Restriction Enzyme Digest</A></LI><LI><A href="#1-2-clean-up">DNA ‘Cleanup’</A></LI><LI><A href="#1-2-nanodrop">NanoDrop</A></LI><LI><A href="#1-2-ligation">Ligation</A></LI><LI><A href="#1-2-prep-e.coil">Preparation of E. coli</A></LI><LI><A href="#1-2-re-pcr">Re-PCR of DNA</A></LI><LI><A href="#1-3">Day 3</A></LI><LI><A href="#1-3-re-pcr">Re-PCR of DNA Sequences</A></LI><LI><A href="#1-3-plasmid-recovery">Plasmid Recovery</A></LI><LI><A href="#1-3-extraction">Extraction of “sticky” pSB1C3</A></LI><LI><A href="#1-3-repeat-pcr">Repeat PCR with higher DNA concentration</A></LI><LI><A href="#1-3-g-j">Gel extraction of G and J</A></LI><LI><A href="#1-3-nanodrop">NanoDrop</A></LI><LI><A href="#1-3-ligation">Ligation</A></LI><LI><A href="#1-3-transform">Transformation</A></LI><LI><A href="#1-4">Day 4</A></LI><LI><A href="#1-4-growth">Growth and culture of bacteria</A></LI><LI><A href="#1-4-transform">Transformation of G and J</A></LI><LI><A href="#1-4-primer">Primer Design</A></LI><LI><A href="#1-5">Day 5</A></LI><LI><A href="#1-5-extraction">Plasmid Extraction</A></LI><LI><A href="#1-5-quantification">Plasmid Quantification</A></LI><LI><A href="#1-5-digest">Plasmid Digest</A></LI><LI><A href="#1-5-gel">Gel Electrophoresis of Digested Plasmids</A></LI><LI><A href="#1-5-analysis">Analysis</A></LI><LI><A href="#2">Week 2</A></LI><LI><A href="#2-6">Day 6</A></LI><LI><A href="#2-6-pcr">PCR</A></LI><LI><A href="#2-6-gel">Gel Electrophpresis</A></LI><LI><A href="#2-6-extraction">Gel Extraction</A></LI><LI><A href="#2-6-digest">Restriction Digest</A></LI><LI><A href="#2-6-recover">Recovery of pSB-1C3 vector</A></LI><LI><A href="#2-6-ezna">EZNA gel extraction protocol on recovered pSB1c3 vector</A></LI><LI><A href="#2-6-growth">Growth and culture of E. coli transformed J and G</A></LI><LI><A href="#2-7">Day 7</A></LI><LI><A href="#2-7-yesterday">Yesterday</A></LI><LI><A href="#2-7-extract">Miniprep</A></LI><LI><A href="#2-7-nanodrop">Nanodrop</A></LI><LI><A href="#2-7-plasmid-digest">Plasmid Digest</A></LI><LI><A href="#2-7-stock">Stock Solutions</A></LI><LI><A href="#2-7-pcr-new">PCR Amplification with New Primers</A></LI><LI><A href="#2-8">Day 8</A></LI><LI><A href="#2-8-gel">Gel Electrophoresis</A></LI><LI><A href="#2-8-digest">Restriction Digest</A></LI><LI><A href="#2-8-plasmid">Plasmid Design</A></LI><LI><A href="#2-8-transform">Transformation</A></LI><LI><A href="#2-9">Day 9</A></LI><LI><A href="#2-9-plasmid-digest">Plasmid Digest</A></LI><LI><A href="#2-9-clean-up">Enzymatic Reaction  Clean-up</A></LI><LI><A href="#2-9-growth">Growth and Culture</A></LI><LI><A href="#2-9-ligation">Ligation</A></LI><LI><A href="#2-9-prep">Preparation of DH5</A></LI><LI><A href="#2-10">Day 10</A></LI><LI><A href="#2-10-plasmid">Plasmid for PCR Products</A></LI><LI><A href="#2-10-prep">Preparation of DH5</A></LI><LI><A href="#3">Week 3</A></LI><LI><A href="#3-11">Day 11</A></LI><LI><A href="#3-11-prep">Overnight Cultures</A></LI><LI><A href="#3-11-pcr">PCR</A></LI><LI><A href="#3-11-digest">Restriction Digest</A></LI><LI><A href="#3-11-primer-prep">Primer Preparation</A></LI><LI><A href="#3-11-sequencing">Sequencing</A></LI><LI><A href="#3-11-gel">Running the Gel</A></LI><LI><A href="#3-12">Day 12</A></LI><LI><A href="#3-12-nanodrop">NanoDrop</A></LI><LI><A href="#3-12-digest">Restriction Digest</A></LI><LI><A href="#3-12-extraction">Gel Extraction</A></LI><LI><A href="#3-12-plasmid-extract">Plasmid Extraction</A></LI><LI><A href="#3-12-gel-2">Gel Electrophoresis</A></LI><LI><A href="#3-12-sequence">Analysis of Sequencing Data</A></LI><LI><A href="#3-12-ligation">Ligation</A></LI><LI><A href="#3-13">Day 13</A></LI><LI><A href="#3-13-to-do">To Do</A></LI><LI><A href="#3-13-to-note">To Note</A></LI><LI><A href="#3-13-quickchange">QuickChange</A></LI><LI><A href="#3-13-transform">Transformation</A></LI><LI><A href="#3-13-digest">Restriction Digest</A></LI><LI><A href="#3-13-gel">Gel Electrophroesis</A></LI><LI><A href="#3-13-sequencing">Sequencing</A></LI><LI><A href="#3-13-extra">Extra</A></LI><LI><A href="#3-14">Day 14</A></LI><LI><A href="#3-14-group-a">Group A</A></LI><LI><A href="#3-14-group-b">Group B</A></LI><LI><A href="#3-14-group-c">Group C</A></LI><LI><A href="#3-14-sequence">Sequencing Results</A></LI><LI><A href="#3-14-pcr">PCR To Redo</A></LI><LI><A href="#3-14-to-sequence">To Sequence</A></LI><LI><A href="#3-14-overnight">To Overnight</A></LI><LI><A href="#3-15">Day 15</A></LI><LI><A href="#3-15-group-a">Group A</A></LI><LI><A href="#3-15-quick">J*4 QuikChange</A></LI><LI><A href="#3-15-group-b">Group B</A></LI><LI><A href="#3-15-group-c">Group C</A></LI><LI><A href="#3-15-weekend">Over the Weekend</A></LI><LI><A href="#4">Week 4</A></LI><LI><A href="#4-16">Day 16</A></LI><LI><A href="#4-16-leon">Leon</A></LI><LI><A href="#4-16-group-a">Group A</A></LI><LI><A href="#4-16-ria">Ria</A></LI><LI><A href="#4-17">Day 17</A></LI><LI><A href="#4-17-plasmid">Plasmid Backbones</A></LI><LI><A href="#4-17-plasmid-de">Dephosphorylation</A></LI><LI><A href="#4-17-plasmid-cleanup">Enzymatic Reaction Cleanup</A></LI><LI><A href="#4-17-ligation">Ligation</A></LI><LI><A href="#4-17-june">June</A></LI><LI><A href="#4-17-june-pcr-1">10/07 PCR Batch (D*, D#, M#, O#)</A></LI><LI><A href="#4-17-june-pcr-2">13/07 PCR Batch (E*, E* DMSO)</A></LI><LI><A href="#4-17-prep">Preparation of Competent Cells</A></LI><LI><A href="#4-17-sequence">Sequencing</A></LI><LI><A href="#4-17-pcr-1">PCR of C#, D#, L#</A></LI><LI><A href="#4-17-pcr-2">PCR of A*, B*, I*, F*, K*, A#, B#, F#, I#</A></LI><LI><A href="#4-18">Day 18</A></LI><LI><A href="#4-18-plates">Check Plates</A></LI><LI><A href="#4-18-transform">Transform Ligated Products into DH5alpha</A></LI><LI><A href="#4-18-ligation">Ligation</A></LI><LI><A href="#4-18-overnight">Overnight Cultures</A></LI><LI><A href="#4-18-pcr">PCRs</A></LI><LI><A href="#4-18-transform-2">Transform completed plasmids into FliC and MG1655 Competent Cells</A></LI><LI><A href="#4-18-m9">Making up M9 modified media</A></LI><LI><A href="#4-19">Day 19</A></LI><LI><A href="#4-19-miniprep">Mini-prep overnighted J*</A></LI><LI><A href="#4-19-digest">Diagnostic restriction digest of J* + gel electrophoresis</A></LI><LI><A href="#4-19-pick">Pick Colonies</A></LI><LI><A href="#4-19-pcr">PCR</A></LI><LI><A href="#4-19-digest-2">Restriction Digestion and Ligation</A></LI><LI><A href="#4-20">Day 20</A></LI><LI><A href="#4-20-leon">Leon</A></LI><LI><A href="#4-20-raffy">Raffy</A></LI><LI><A href="#4-20-mabel">Mabel and Duke</A></LI><LI><A href="#4-20-ria">Ria and Lychee</A></LI><LI><A href="#4-20-helen">Helen and Kyle</A></LI><LI><A href="#4-20-weekend">Over the Weekend</A></LI><LI><A href="#5">Week 5</A></LI><LI><A href="#5-21">Day 21</A></LI><LI><A href="#5-21-post">Post-digest cleanup → ligation</A></LI><LI><A href="#5-21-sequence">Sequencing of QC J*4</A></LI><LI><A href="#5-21-data">Sequencing Data Received</A></LI><LI><A href="#5-21-gel">Gel electrophoresis confirmation of 17/07 digested miniprep products</A></LI><LI><A href="#5-21-miniprep">Miniprepping of 19/07 overnight cultures</A></LI><LI><A href="#5-21-transform">Transformations</A></LI><LI><A href="#5-21-plate">Plate Streaking</A></LI><LI><A href="#5-22">Day 22</A></LI><LI><A href="#5-22-data">Sequencing Data Received</A></LI><LI><A href="#5-22-pcr">PCR Amplification to make J# from QuikChanged J*4[pSB1C3]</A></LI><LI><A href="#5-22-transform">Transformations</A></LI><LI><A href="#5-22-gel">Run Gel for miniprepped J*QC, F*, I*, R#, L#</A></LI><LI><A href="#5-22-digest">Restriction digest and ligation of A*, B*, F*, I*, and K* directly from the G blocks</A></LI><LI><A href="#5-22-duke">Duke and Lychee</A></LI><LI><A href="#5-22-pcr-2">PCR of QuikChanged J*4(1) with primers for pBAD restriction sites</A></LI><LI><A href="#5-23">Day 23</A></LI><LI><A href="#5-23-frozen">Frozen Stock Preparation</A></LI><LI><A href="#5-23-tasks">Tasks completed</A></LI><LI><A href="#5-23-digest">Test Digest for D*IIe, E*1 compared with digest of completed BioBrick N*3</A></LI><LI><A href="#5-24">Day 24</A></LI><LI><A href="#5-24-overnight">Overnights</A></LI><LI><A href="#5-24-transform">Transformation</A></LI><LI><A href="#5-24-data">Sequencing Data</A></LI><LI><A href="#5-24-miniprep">Miniprep C#, D#, L#</A></LI><LI><A href="#5-24-ligate">Ligations</A></LI><LI><A href="#5-25">Day 25</A></LI><LI><A href="#5-25-miniprep">Miniprep</A></LI><LI><A href="#5-25-gel">Gel Electrophoresis of minipreps</A></LI><LI><A href="#5-25-transform">Transform</A></LI><LI><A href="#5-25-weekend">Over the Weekend</A></LI><LI><A href="#5-25-weekend-plates">Streaking Plates</A></LI><LI><A href="#5-25-weekend-overnight">Overnights</A></LI><LI><A href="#5-25-weekend-miniprep">Miniprep</A></LI><LI><A href="#6">Week 6</A></LI><LI><A href="#6-26">Day 26</A></LI><LI><A href="#6-26-to-do">To Do</A></LI><LI><A href="#6-26-digest">Digest J# QC, run gel of O# and J# digest</A></LI><LI><A href="#6-27">Day 27</A></LI><LI><A href="#6-27-raffy">Raffy</A></LI><LI><A href="#6-28">Day 28</A></LI><LI><A href="#6-28-gel">Gel run for 28/07 PCR products, band excision, clean up</A></LI><LI><A href="#6-28-digest">Digest + clean up</A></LI><LI><A href="#6-28-ligate">Ligation</A></LI><LI><A href="#6-28-success">Success</A></LI><LI><A href="#6-29">Day 29</A></LI><LI><A href="#6-29-data">Sequencing Data</A></LI><LI><A href="#6-29-transform">Transformations</A></LI><LI><A href="#6-29-plate">Plate out BioBrick (T4 Holin) agar stab</A></LI><LI><A href="#6-30">Day 30</A></LI><LI><A href="#6-30-overnight">Overnight Cultures</A></LI><LI><A href="#6-30-transform">Transformations</A></LI><LI><A href="#6-30-to-do">To Do</A></LI><LI><A href="#6-30-weekend">Over the Weekend</A></LI><LI><A href="#7">Week 7</A></LI><LI><A href="#7-31">Day 31</A></LI><LI><A href="#7-31-miniprep">Miniprep</A></LI><LI><A href="#7-32">Day 32</A></LI><LI><A href="#7-32-results">Sequencing Results</A></LI><LI><A href="#7-33">Day 33</A></LI><LI><A href="#7-33-george">George</A></LI><LI><A href="#7-33-helen">Helen</A></LI><LI><A href="#7-33-mabel">Mabel and James</A></LI><LI><A href="#7-34">Day 34</A></LI><LI><A href="#7-34-kyle">Kyle</A></LI><LI><A href="#7-34-helen">Helen</A></LI><LI><A href="#7-35">Day 35</A></LI><LI><A href="#7-35-to-do">To Do</A></LI><LI><A href="#7-35-to-do-i">i)</A></LI><LI><A href="#7-35-to-do-ii">ii)</A></LI><LI><A href="#7-35-to-do-iii">iii)</A></LI><LI><A href="#7-35-to-do-iv">iv)</A></LI><LI><A href="#7-35-to-do-v">v)</A></LI><LI><A href="#7-35-to-do-vi">vi)</A></LI><LI><A href="#7-35-to-do-vii">vii)</A></LI><LI><A href="#7-35-to-do-viii">viii)</A></LI><LI><A href="#8">Week 8</A></LI><LI><A href="#8-36">Day 36</A></LI><LI><A href="#8-36-to-do">To Do</A></LI><LI><A href="#8-37">Day 37</A></LI><LI><A href="#8-38">Day 38</A></LI><LI><A href="#8-39">Day 39</A></LI><LI><A href="#8-40">Day 40</A></LI><LI><A href="#8-40-weekend">Over the Weekend</A></LI><LI><A href="#9">Week 9</A></LI><LI><A href="#9-41">Day 41</A></LI><LI><A href="#9-41-last">Last Few Bits of Cloning</A></LI><LI><A href="#9-42">Day 42</A></LI><LI><A href="#9-43">Day 43</A></LI><LI><A href="#9-44">Day 44</A></LI><LI><A href="#9-45">Day 45</A></LI><LI><A href="#10">Week 10</A></LI><LI><A href="#10-46">Day 46</A></LI><LI><A href="#10-46-mabel">Constructs PCR</A></LI><LI><A href="#10-47">Day 47</A></LI><LI><A href="#10-47-clone">Molecular Cloning of Constructs</A></LI><LI><A href="#10-48">Day 48</A></LI><LI><A href="#10-48-kyle">Kyle</A></LI><LI><A href="#10-48-mabel">Mabel</A></LI><LI><A href="#10-49">Day 49</A></LI><LI><A href="#10-50">Day 50</A></LI><LI><A href="#11">Week 11</A></LI><LI><A href="#11-51">Day 51</A></LI><LI><A href="#11-51-mabel">Molecular Cloning Day</A></LI><LI><A href="#11-51-to">To Do</A></LI><LI><A href="#11-52">Day 52</A></LI><LI><A href="#11-52-mabel">Mabel</A></LI><LI><A href="#11-52-june">June</A></LI><LI><A href="#11-53">Day 53</A></LI><LI><A href="#11-54">Day 54</A></LI><LI><A href="#11-55">Day 55</A></LI><LI><A href="#11-55-digest">Digest of M#, G#, pBad HisB</A></LI><LI><A href="#11-55-digest-2">Digests from Kyle</A></LI><LI><A href="#11-55-clone">Molecular Cloning</A></LI><LI><A href="#12">Week 12</A></LI><LI><A href="#12-56">Day 56</A></LI><LI><A href="#12-56-transform">Transformation of #s</A></LI><LI><A href="#12-57">Day 57</A></LI><LI><A href="#12-58">Day 58</A></LI><LI><A href="#12-60">Day 60</A></LI><LI><A href="#12-60-weekend">Over the weekend</A></LI><LI><A href="#13">Week 13</A></LI><LI><A href="#13-61">Day 61</A></LI><LI><A href="#characterization">Characterization</A></LI><LI><A href="#plan-c">Plan</A></LI><LI><A href="#plan-c-overview">Overview of sequences in pBAD</A></LI><LI><A href="#plan-c-complete">Currently completed pBADs</A></LI><LI><A href="#cell">Cell Growth</A></LI><LI><A href="#cell-5">Week 5</A></LI><LI><A href="#cell-5-23">Day 23</A></LI><LI><A href="#cell-5-23-tox">Toxicity Assay</A></LI><LI><A href="#cell-5-24">Day 24</A></LI><LI><A href="#cell-5-24-tox">Toxicity Assay</A></LI><LI><A href="#cell-5-25">Day 25</A></LI><LI><A href="#cell-5-25-tox">Toxicity Assay</A></LI><LI><A href="#cell-6">Week 6</A></LI><LI><A href="#cell-6-26">Day 26</A></LI><LI><A href="#cell-26-tox">Toxicity Assay</A></LI><LI><A href="#cell-6-27">Day 27</A></LI><LI><A href="#cell-6-27-tox">Repeat of Yesterday</A></LI><LI><A href="#cell-6-29">Day 29</A></LI><LI><A href="#cell-6-29-tox">Toxicity Assay</A></LI><LI><A href="#cell-8">Week 8</A></LI><LI><A href="#cell-8-36">Day 36</A></LI><LI><A href="#cell-8-36">Toxicity Assay</A></LI><LI><A href="#cell-8-37">Day 37</A></LI><LI><A href="#cell-8-38">Day 38</A></LI><LI><A href="#protein">Expression and Secretion</A></LI><LI><A href="#protein-6">Week 6</A></LI><LI><A href="#protein-6-29">Day 29</A></LI><LI><A href="#protein-6-29-tca">TCA Protein Precipitation</A></LI><LI><A href="#protein-6-30">Day 30</A></LI><LI><A href="#protein-6-30-secrection">Secretion Assay</A></LI><LI><A href="#protein-7">Week 7</A></LI><LI><A href="#protein-7-32">Day 32</A></LI><LI><A href="#protein-7-32-secretion">Secretion Assay</A></LI><LI><A href="#protein-7-34">Day 34</A></LI><LI><A href="#protein-7-34-tca">TCA-Precipitate</A></LI><LI><A href="#protein-8">Week 8</A></LI><LI><A href="#protein-8-36-super">Supernatant Acuisition</A></LI><LI><A href="#protein-8-36-sds">SDS-Page</A></LI><LI><A href="#protein-8-37">Day 37</A></LI><LI><A href="#protein-8-37-assay">Assaying</A></LI><LI><A href="#protein-8-38">Day 38</A></LI><LI><A href="#protein-8-39">Day 39</A></LI><LI><A href="#protein-8-40">Day 40</A></LI><LI><A href="#protein-9">Week 9</A></LI><LI><A href="#protein-9-42">Day 42</A></LI><LI><A href="#protein-9-42-tca">Set Up for TCA</A></LI><LI><A href="#protein-9-42-secretion">Secretion</A></LI><LI><A href="#protein-9-43">Day 43</A></LI><LI><A href="#protein-9-43-express">Expression</A></LI><LI><A href="#protein-9-43-pur">Purification</A></LI><LI><A href="#protein-9-43-western">Western Blot</A></LI><LI><A href="#protein-9-44">Day 44</A></LI><LI><A href="#proetin-9-44-express">Expression</A></LI><LI><A href="#protein-9-45">Day 45</A></LI><LI><A href="#proetin-9-45-express">Expression</A></LI><LI><A href="#protein-10">Week 10</A></LI><LI><A href="#protein-10-46">Day 46</A></LI><LI><A href="#protein-10-47">Day 47</A></LI><LI><A href="#protein-10-47-duke">Duke (aka king of banter)</A></LI><LI><A href="#protein-10-49">Day 49</A></LI><LI><A href="#protein-10-49-western">Western Blot</A></LI><LI><A href="#protein-10-49-secretion">Secretion</A></LI><LI><A href="#protein-10-50">Day 50</A></LI><LI><A href="#protein-10-50-western">Western Blot</A></LI><LI><A href="#bio">Biofilms</A></LI><LI><A href="#bio-5">Week 5</A></LI><LI><A href="#bio-5-25">Day 25</A></LI><LI><A href="#bio-5-25-test">Test Crystal Violet Staining of Biofilms</A></LI><LI><A href="#bio-6">Week 6</A></LI><LI><A href="#bio-6-27">Day 27</A></LI><LI><A href="#bio-6-27-cultivation">Biofilm Cultivation Using Existing Transformed Strains</A></LI><LI><A href="#bio-6-30">Day 30</A></LI><LI><A href="#bio-6-30-stain">Biofilm Staining</A></LI><LI><A href="#bio-7">Week 7</A></LI><LI><A href="#bio-7-31">Day 31</A></LI><LI><A href="#bio-7-31-assay">Assays</A></LI><LI><A href="#bio-7-34">Day 34</A></LI><LI><A href="#bio-7-34-setup">Set Up</A></LI><LI><A href="#bio-7-35">Day 35</A></LI><LI><A href="#bio-7-35-assay">Biofilm Assaying</A></LI><LI><A href="#bio-8">Week 8</A></LI><LI><A href="#bio-8-36">Day 36</A></LI><LI><A href="#bio-8-36-assay">Biofilm Assaying</A></LI><LI><A href="#bio-8-37">Day 37</A></LI><LI><A href="#bio-8-38">Day 38</A></LI><LI><A href="#bio-8-38-assay">Biofilm viability assays</A></LI><LI><A href="#bio-8-38-de">Biofilm degradation</A></LI><LI><A href="#bio-8-38-in">Biofilm inhibition</A></LI><LI><A href="#bio-8-39">Day 39</A></LI><LI><A href="#bio-8-40">Day 40</A></LI><LI><A href="#bio-9">Week 9</A></LI><LI><A href="#bio-9-42">Day 42</A></LI><LI><A href="#bio-9-42-setup">Set Up</A></LI><LI><A href="#bio-11">Week 11</A></LI><LI><A href="#bio-11-54">Day 54</A></LI><LI><A href="#bio-11-54-test">P. putida testing</A></LI><LI><A href="#bio-11-54-bio">Biofilm Viability Assay</A></LI><LI><A href="#bio-11-54-weekend">Over the Weekend</A></LI><LI><A href="#kill">Cell Killing</A></LI><LI><A href="#kill-8">Week 8</A></LI><LI><A href="#kill-8-38">Day 38</A></LI><LI><A href="#kill-8-39">Day 39</A></LI><LI><A href="#fluor">Fluorescence</A></LI><LI><A href="#fluor-6">Week 6</A></LI><LI><A href="#fluor-6-29">Day 29</A></LI><LI><A href="#fluor-6-29-growth">Growth Curve</A></LI><LI><A href="#fluor-6-30">Day 30</A></LI><LI><A href="#fluor-6-30-assay">Assay</A></LI><LI><A href="#fluor-8">Week 8</A></LI><LI><A href="#fluor-8-56">Day 36</A></LI><LI><A href="#fluor-8-56-quorum">Quorum-Sensing Fluorescence</A></LI><LI><A href="#fluor-10">Week 10</A></LI><LI><A href="#fluor-10-50">Day 50</A></LI><LI><A href="#fluor-8-50-quorum">Quorum-Sensing Fluorescence</A></LI><LI><A href="#interlab">Interlab</A></LI><LI><A href="#2-i">Week 2</A></LI><LI><A href="#2-8-i">Day 8</A></LI><LI><A href="#2-8-i-prep">Preparation of Interlab Study BioBricks</A></LI><LI><A href="#2-8-i-transform">Transformation</A></LI><LI><A href="#2-9-i">Day 9</A></LI><LI><A href="#2-9-i">Growth and Culture of Bacteria</A></LI><LI><A href="#2-10-i">Day 10</A></LI><LI><A href="#2-10-i-gel">Plasmid for PCR Product</A></LI><LI><A href="#3-i">Week 3</A></LI><LI><A href="#3-11-i">Day 11</A></LI><LI><A href="#3-11-i-digest">Restriction Digest</A></LI><LI><A href="#3-12-i">Day 12</A></LI><LI><A href="#3-12-i-extraction">Gel Extraction</A></LI><LI><A href="#3-15-i">Day 15</A></LI><LI><A href="#3-15-i-group-d">Group D</A></LI><LI><A href="#4-i">Week 4</A></LI><LI><A href="#4-16-i">Day 16</A></LI><LI><A href="#4-16-i-lychee">Lychee</A></LI><LI><A href="#4-17-i">Day 17</A></LI><LI><A href="#4-17-i-group-d">Group D</A></LI><LI><A href="#4-17-i-june">Digest the remainder of 10/07 miniprep products and run on gel</A></LI><LI><A href="#4-18-i">Day 18</A></LI><LI><A href="#4-18-i-sequence-1">Sequence K1 and 20K</A></LI><LI><A href="#4-18-i-sequence-2">Check Sequencing Data</A></LI><LI><A href="#4-20-i">Day 20</A></LI><LI><A href="#ria">Ria and Lychee</A></LI><LI><A href="#5-i">Week 5</A></LI><LI><A href="#5-21-i">Day 21</A></LI><LI><A href="#5-21-i-miniprep">Miniprep of 19/07 overnight cultures</A></LI><LI><A href="#5-21-i-transform">Transformation</A></LI><LI><A href="#5-21-i-plate">Plate Streaking</A></LI><LI><A href="#5-23-i">Day 23</A></LI><LI><A href="#5-23-i-miniprep">Miniprep of DH5α (+) and (-)</A></LI><LI><A href="#5-23-i-frozen">Frozen Stock Preparation</A></LI><LI><A href="#5-24-i">Day 24</A></LI><LI><A href="#5-24-overnight">Overnights</A></LI><LI><A href="#5-25-i">Day 25</A></LI><LI><A href="#5-25-i-frozen">Frozen Stocks</A></LI><LI><A href="#5-25-i-overnight">Overnight cultures in M9 Modified Media</A></LI><LI><A href="#5-25-i-weekend">Over the Weekend</A></LI><LI><A href="#6-i">Week 6</A></LI><LI><A href="#6-27-i">Day 27</A></LI><LI><A href="#6-27-i-growth">Growth Curves</A></LI><LI><A href="#6-27-i-trial">Fluorescence Microscopy Trial Run</A></LI><LI><A href="#6-28-i">Day 28</A></LI><LI><A href="#6-28-i-read">Instantaneous Read</A></LI><LI><A href="#6-28-i-mid-log">Culturing Strains to Mid-Log</A></LI><LI><A href="#6-29-i">Day 29</A></LI><LI><A href="#6-29-i-micro">Cultivate Cells for Microscopy</A></LI><LI><A href="#7-i">Week 7</A></LI><LI><A href="#7-31-i">Day 31</A></LI><LI><A href="#7-31-i-micro">Microscopy</A></LI><LI><A href="#7-32-i">Day 32</A></LI><LI><A href="#7-32-i-june">June</A></LI><LI><A href="#7-33-i">Day 33</A></LI><LI><A href="#7-33-i-june">June</A></LI><LI><A href="#7-34-i">Day 34</A></LI><LI><A href="#7-34-i-set-up">Set Up</A></LI><LI><A href="#7-35-i">Day 35</A></LI><LI><A href="#7-35-i-tasks">Tasks</A></LI><LI><A href="#8-i">Week 8</A></LI><LI><A href="#8-36-i">Day 36</A></LI><LI><A href="#8-36-i-tasks">Tasks</A></LI><LI><A href="#8-37-i">Day 37</A></LI><LI><A href="#8-38-i">Day 38</A></LI><LI><A href="#8-38-i-trouble">Trouble Shooting for Microscopy</A></LI><LI><A href="#8-39-i">Day 39</A></LI><LI><A href="#8-40-i">Day 40</A></LI><LI><A href="#9-i">Week 9</A></LI><LI><A href="#9-41-i">Day 41</A></LI><LI><A href="#9-42-i">Day 42</A></LI><LI><A href="#9-43-i">Day 43</A></LI><LI><A href="#9-44-i">Day 44</A></LI><LI><A href="#9-45-i">Day 45</A></LI><LI><A href="#10-i">Week 10</A></LI><LI><A href="#10-i-47">Day 47</A></LI><LI><A href="#10-i-47-micro">Microscopy</A></LI><LI><A href="#10-i-48">Day 48</A></LI><LI><A href="#syn">Synbiota</A></LI><LI><A href="#syn-7">Week 7</A></LI><LI><A href="#syn-7-33">Day 33</A></LI><LI><A href="#syn-7-34">Day 34</A></LI><LI><A href="#syn-8">Week 8</A></LI><LI><A href="#syn-8-37">Day 37</A></LI><LI><A href="#syn-8-38">Day 38</A></LI><LI><A href="#syn-8-39">Day 39</A></LI><LI><A href="#syn-8-40">Day 40</A></LI><LI><A href="#syn-9">Week 9</A></LI><LI><A href="#syn-9-41">Day 41</A></LI><LI><A href="#syn-9-43">Day 43</A></LI><LI><A href="#syn-9-44">Day 44</A></LI><LI><A href="#syn-10">Week 10</A></LI><LI><A href="#syn-10-47">Day 47</A></LI><LI><A href="#syn-10-48">Day 48</A></LI><LI><A href="#syn-10-49">Day 49</A></LI><LI><A href="#syn-11">Week 11</A></LI><LI><A href="#syn-11-51">Day 51</A></LI><LI><A href="#syn-11-52">Day 52</A></LI><LI><A href="#syn-11-53">Day 53</A></LI><LI><A href="#syn-11-54">Day 54</A></LI><LI><A href="#syn-12">Week 12</A></LI><LI><A href="#syn-12-60">Day 60</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" id="foot"><DIV class="col-md-12"><P>
                        Made with love  by Oxford iGEM
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>