Team:Westminster UK/Parts
Project
Description Natural Degradation Chemical Preparation Modelling Safety Awards
Lab
Notebook Protocols InterLab
Parts
Part collection Bioinformatics
Social
Human practices Education & Engagement Collaborations
Team
Team members Attributions Sponsors
Bioinformatics
Pathway
The toluene (tod) degradation pathway was the main inspiration to this study. Since toluene and styrene both have similar chemical structures (figure 1), it was worth attempting to use the enzymes from the tod pathway to degrade the toxic styrene.
Figure 1  showing the degradation of 3-methylcatechol in the toluene pathway at the bottom, the top shows predicted degradation of 3-vinylcatechol using todE.
As previously mentioned, the tod pathway has multiple enzymes working in the degradation of toluene (figure 2), the first one being the TDO enzyme. TodX, known as the transporter protein is not shown in figure 2, but it was one of the enzymes investigated by our team, as we wanted to check whether it would also be able transport styrene into the bacteria as it does with toluene (reference).
Figure 2 showing the predicted degradation of styrene, using the enzymes from the toluene pathway.
Function of TodE enzyme
TodE enzyme, a catechol 2,3-dioxygenase (C23O), from the Tod pathway, is native to the break down of 3-methylcatechol into 6-methyl 2-hydroxy 6-oxo-methylhexa-2,4-dienoate (6-methyl HODA). Previous research suggested that it's possible for styrene to get degraded by the tod pathway, however they presented that everything goes smoothly up until the formation of 3-vinylcatechol, from that point, TodE gets inactivated by 3-vinylcatechol itself, and the degradation can not carry on. The styrene then won't end up getting converted into pyruvate by the rest of the degradation pathway, and won’t to be used as a energy source by the bacteria. The accumulation of 3-vinylcatechol also leads to toxicity within the cell, leading to cell death (George et al., 2010). When 3-vinylcatechol gets degraded by TodE, it turns into 2-hydroxy-6-vinylhexa-2,4-dienoate (6-vinyl HODA), instead of 6-methyl HODA.
Motif investigations
Before starting the analysis, motifs for the todE gene needed to be discovered. A BLAST search was carried out with the Pseudomonas. putida F1 todE sequence. A total of 15 todE sequences from different organisms and strains (including  P. putida F1) were aligned. Accession number of the sequences included: WP_087013802.1, OFW86286.1, WP_011598997.1, BAA06872.1, PVG83184.1, WP_105416154.1, WP_109329734.1, BAM76241.1, BAH80175.1, WP_034815878.1, WP_083868340.1, WP_084398232.1, WP_029044277.1, EWS65147.1 and P. putida F1 todE. Both EMBL-EBI clustal omega and MEGA7 muscle was used to align the sequences. Main residues of the active site (pocket) was investigated and were discovered to be F187, H241 and Y250.
Figure 3 showing the phylogenetic tree of the 15 aligned todE sequence
Attempt to modify of TodE (failed miserably)
SwissDock was used to dock 3-vinylcatechol with unmodified P. putida F1 TodE enzyme. The docking was visualised using UCSF Chimera. A total of 39 docking positions and rotations of the compound were discovered inside the pocket, figure 4 shows the unmodified TodE enzyme. The main residues of the enzyme were first visualized when zoomed in, with main residues highlighted red (figure 5). In all the images of the dockings, the amino acids that were not close to the pocket were hidden for the purpose of simplicity and easier visualisation.
Figure 4 showing the whole TodE enzyme. Model predicted by SwissModel, docking prediction by SwissDock (Grosdidier, Zoete and Michielin, 2011) and visualisation by UCSF Chimera.
Figure 5 showing one putative binding positions out of the 39 possible positions.  Model-predicted by SwissModel, docking prediction by SwissDock (Grosdidier, Zoete and Michielin, 2011) and visualisation by UCSF Chimera.
Figure 4 showing the whole TodE enzyme, in all the images of the dockings, the amino acids that were not close to the pocket were hidden for the sake of simplicity and easier visualisation.
After this step, the amino acids surrounding the pocket were the focus of this analysis. Close amino acids were coloured orange, and the ones a bit further were coloured yellow (this was done for the sake of visualisation), figure 6 shows this.
Figure 6 showing the close up of the TodE enzyme active site with the 3-vinylcatchol in one of the 39 possible positions. The colours approximate how close the interaction may be with 3-vinylcatechol, red is the residues (the most important in enzyme function), orange coloured amino acids slightly further away and yellow coloured amino acids being even further and may have less interaction with 3-vinylcatechol directly. Model predicted by SwissModel, docking prediction by SwissDock (Grosdidier, Zoete and Michielin, 2011) and visualisation by UCSF Chimera.
Video 1
Video showing the different compound putative positions in the pocket of TodE, predicted by SwissDock (Grosdidier, Zoete and Michielin, 2011), visualised in UCSF Chimera (Pettersen et al., 2004). Some of the positions seem almost identical, while others are visibly different. The total positions predicted was more than 39 at the beginning, all the predictions that were outside of the pocket were emitted and final 39 positions that were within the pocket is visualised as seen in the video. The number of predictions could’ve been reduced further, however at this point of the research and after consulting with the team and instructors, we understood how much time it would require to achieve accurate predict the correct docking with in silico alone.. At this point we realised we won’t be able to carry out any specific mutations as we intended at the beginning of the study. We wanted to mutate the enzyme to open the pocket of TodE slightly to allow 3-vinylcatechol to bind in more easily. Allowing more conversion of 3-vinylcatechol. With further research, we also saw papers suggesting that the size of the TodE pocket may not be the only reason why this pathway don’t carry on as well from this point. Paper showed that there is inactivation of the TodE enzyme by 3-vinylcatechol itself.
In order to be able to really accurately understand the true docking of 3-vinylcatechol in the TodE enzyme, the position of 3-methylcatechol in catechol 2,3-dioxygenase (C23O) (TodE) was observed. The crystal structure of catechol 2,3-dioxygenase was observed in the protein data bank (PDB) (PDB ID: 2WL9). Ligand explorer function was used, and the expression system of this assembly was E. coli BL21, the structure is known to be a homo-4 mer (Cho et al., 2010).
Figure 7 shows the crystal structure of catechol 2,3-dioxygenase (TodE), with the iron 3+ cation visualised as the purple ball, the green ball is the visualisation of magnesium ion.
Figure 8 showing 3-methylcatechol (MBD), the ligand in C23O, indicated by the orange lines.
Figure 9 showing the interaction between the iron 3+ cation (FE) and 3-methylcatechol, shown as dotted white lines. The interactions are labeled with the distances. This picture also shows relevant amino acids in the metal interaction including H149, HIS212, TYR253 and GLU263.
Figure 10 shows the hydrogen bond (pink dotted lines) of the 3-methylcatechol with TYR253. The oxygen on 3-methylcatechol also have interaction with the oxygen and nitrogen on the ASN246 of C23O.
Figure 11 showing the hydrophobic interactions between the 3-methylcatechol and the surrounding amino acids of C23O. The interactions are also labeled with the distances. This image shows that TYR175, PHE189, HIS244 and TYR253 may all have an importance in the enzyme-ligand interaction.
Figure 12 showing another picture of the interactions between 3-methylcatechol and C23O. Hydrogen bonds are shown in blue, hydrophobic interactions are shown as grey, metal interactions are shown as purple, Pi interactions are shown as orange and green and turquoise are halogen bonds. Model displayed in PDB.
We used structural bioinformatics as a tool to really understand the TodE enzyme and it’s interaction with 3-vinylcatechol as closely as possible. Although we didn’t manage to predict the precise position of 3-vinylcatechol in the pocket, we did gain a lot of understanding regarding what to do next in understanding the TodE enzyme better.  One of the things that can be done in the future, is carrying out X-ray crystallisation of the TodE enzyme with 3-vinylcatechol to observe the 3D structure. Observing the 3D structure would greatly help to find the putative ligand binding position. This was something we unfortunately didn’t have the time and resources for to carry-out. From there we could visualise the interaction of the ligand with the amino acids of the TodE enzyme as well as the ionic interactions, hydrogen bonds and the interaction of the compound with the iron and magnesium ions. From here, it’s possible to understand what amino acids are essential to the binding, and which ones are not. The ones that are then not as essential, they can be modified to allow a better fitting of the compound in the pocket. From there it goes back to laboratory work where the enzyme needs to be synthesised and purified. Perhaps multiple mutated TodE versions need to be produced, where many of these have a high likelihood to not folding or forming the correct enzyme 3D structure. The functional enzymes would be characterised and the efficiency is compared with the function of 3-vinylcatechol degradation with the unmodified TodE, that was taken straight from the toluene degradation pathway. This would give us the understanding whether we can modify the TodE enzyme to efficiently degrade down 3-vinylcatechol to 6-vinyl HODA.
References:
George, K., Kagle, J., Junker, L., Risen, A. and Hay, A. (2010). Growth of Pseudomonas putida F1 on styrene requires increased catechol-2,3-dioxygenase activity, not a new hydrolase. Microbiology, 157(1), pp.89-98.
Swiss Model citations:
Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R., Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., Lepore, R., Schwede, T. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 46(W1), W296-W303 (2018).
Bienert, S., Waterhouse, A., de Beer, T.A.P., Tauriello, G., Studer, G., Bordoli, L., Schwede, T. The SWISS-MODEL Repository - new features and functionality. Nucleic Acids Res. 45, D313-D319 (2017).
Guex, N., Peitsch, M.C., Schwede, T. Automated comparative protein structure modelling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. Electrophoresis 30, S162-S173 (2009).
Benkert, P., Biasini, M., Schwede, T. Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics 27, 343-350 (2011).
Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L., Schwede, T. Modeling protein quaternary structure of homo- and hetero-oligomers beyond binary interactions by homology. Scientific Reports 7 (2017).
SwissDock citations:
Grosdidier, A., Zoete, V. and Michielin, O. (2011). Fast docking using the CHARMM force field with EADock DSS. Journal of Computational Chemistry, 32(10), pp.2149-2159.
Grosdidier, A., Zoete, V. and Michielin, O. (2011). SwissDock, a protein-small molecule docking web service based on EADock DSS. Nucleic Acids Research, 39(suppl), pp.W270-W277.
UCSF Chimera citation: Pettersen, E., Goddard, T., Huang, C., Couch, G., Greenblatt, D., Meng, E. and Ferrin, T. (2004). UCSF Chimera?A visualization system for exploratory research and analysis. Journal of Computational Chemistry, 25(13), pp.1605-1612.
Westminster_UK
igem.westminster.2018@gmail.com
