<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-Singapore_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-Singapore/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>NTU-Singapore</TITLE></P><DIV id="wrapper"><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><UL class="nav navbar-right top-nav" style="margin: auto; width: 1063px;"><LI class="dropdown" style="margin-left:0px;"><A href="https://2016.igem.org/Team:NTU-Singapore" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> NTU-SG</SPAN></A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NTU-Singapore/Collaborations" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> Collaboration</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> Projects</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Interlab"><SPAN style="top: -34px; float: left; left: 50px;" class="navbar-name"> Interlab Measurement </SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Description" style="top: -20px; position: relative;"><SPAN style="left: 9px;" class="navbar-name"> Description</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Proof" style="height: 49px;"><SPAN style="width: 96px;
								left: 10px;
								float: left;
								margin-left: 6px;
								top: -6px;" class="navbar-name"> Proof of concept </SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Demonstrate"><SPAN style="left: 9px;" class="navbar-name"> Demonstrate</SPAN></A></LI></UL></NAV></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> Parts</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Basic_Part"><SPAN style="left: 8px;" class="navbar-name"> Basic Parts</SPAN></A></LI><LI><A style="height: 49px" href="https://2016.igem.org/Team:NTU-Singapore/Composite_Part"><SPAN style="    float: left;
								width: 96px;
								top: -6px;
								left: 13px;" class="navbar-name"> CRISPR<SPAN style="text-transform: lowercase;">y</SPAN> Collection</SPAN></A></LI></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> Team</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Team"><SPAN style="left: 13px;" class="navbar-name">Adventurers</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/Attributions"><SPAN style="left: 13px;" class="navbar-name">Attribution</SPAN></A></LI><LI><A style="height: 49px" href="https://2016.igem.org/Team:NTU-Singapore/Notebook"><SPAN style=" left: 13px;" class="navbar-name">Notebook</SPAN></A></LI></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"><SPAN class="navbar-name"> Human Practice</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/HP/Silver"><SPAN class="navbar-name"> Silver</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:NTU-Singapore/HP/Gold"><SPAN class="navbar-name"> Gold</SPAN></A></LI></DIV><DIV class="jumbotron content-page" style="font-family: 'Just Me Again Down Here', cursive; padding-top:150px; height: 340px;"><H1 style="font-size: 70px;"><SPAN style="color: #0089A7; ">CRISPRy</SPAN> adventurers helping out!</H1></DIV><DIV class="content-overview" style="padding: 0px 66px; width: 872px;"><DIV class="circle-overview circle3"><A href="#nus" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">team<SPAN style="font-weight: bold">NUS Singapore</SPAN></SPAN></A></DIV><DIV class="circle-overview circle1"><A href="#macquarie" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Team<SPAN style="font-weight: bold">Macquarie Australia</SPAN></SPAN></A></DIV></DIV><DIV class="page-header"><H2>Team Macquarie Australia</H2></DIV><DIV class="well"><P>The NTU Singapore iGEM team was pleasantly partnered with Macquarie team for collaboration. The project involved for both our team were quite different, in which Macquaire team dealed with photosynthetic system while our team dealed with CRISPR-based gene editing technology. Thus, we had made several Skype meetings with Macquaire team and more frequently emailed each other to share different concepts of the projects. Finally, both teams had reached consensus in using the CRISPR-based gene editing technique for both of the teams’ works.</P></DIV><DIV style="margin: auto; width: 800px; "><P style="font-size:18px; text-align: center;">Our Skype session with Team Macquarie :)</P></DIV><DIV class="page-header"><H1>Compare the gene expression level of Mg2+ Chelatase Operon genes by RT-PCR</H1></DIV><DIV class="well"><P>With the intention of producing functional Mg-chelatase that eventually give rise to  Mg-protoporphyrin product, Macquaire team had constructed a Mg-Chelatase plasmid (<A href="http://parts.igem.org/Part:BBa_K1998000" style="color: cadetblue;    font-weight: bold;">BBa_K1998000</A>) that consist of ChII1, ChID, Gun4, ChII2, CTH1 and ChIH genes, in which their expression are co-regulated under pLac promoter, with ChIH expressed under a second pLac promoter.</P><P>Team Macquarie had overexpressed the genes and ran SDS-page gel to check the expression of the genes. However, without antibodies for these proteins, a western blot could not be done to verify the identity and quantity of the protein. Thus, our team had helped them to quantitate each of the gene expression level by RT-PCR.
                                </P><P>The results of RT-PCR is shown as below:</P><DIV style="margin: auto; width: 800px; "><P style="font-size:18px; text-align: center;">Relative Expression of Mg2+ Chelatase gene after IPTG induction</P></DIV><P>Formula: 2^-(ΔΔCt) is used to calculate the fold change of gene expression after induction, in which
ΔCt(control)1= Ct(uninduced operon genes)- Ct(Ampicillin gene)
ΔCt(test)2 = Ct(Induced operon genes)-Ct(Induced Ampicillin gene)
Finally, use ΔCt(test)2- ΔCt(control)1 to get ΔΔCt
                                </P><P>
As shown from the above bar charts, with 0.5Mm,1.0mM and 1.5mM of IPTG induction, there is only slight increase of gene expression level by only 2-3 fold. However, with 2.0Mm of IPTG induction, there is much greater fold change of more than 2-3 folds. It is worth to notice that the RT-PCR efficiency might be different for different genes due to their size of PCR products and the formula used above is assuming the PCR efficiency to be 100% that eventually resulted in 2 copies of products.</P></DIV><DIV class="page-header"><H1>Advise and information sharing with Team Macquarie on HemH gene knock-out in bacterial genome</H1></DIV><DIV class="well"><P>Macquaire iGEM team members wish to knock-out HemH gene in DH5α using CRIPSR-based system. HemH gene is responsible to encode ferrochelatase enzyme that insert ferrous ion into protoporphyrin ring IX. By knocking out this gene, protoporphyrin ring IX can be redirected for the production of chlorophyll a instead of heme.
</P><P>As the CRISPR-Cas based bacterial genome editing application is relatively unexplored and underpresented, our team is helpful in advising Macquiare iGem team that it is not preferable to use <A href="http://blog.addgene.org/crispr-methods-for-bacterial-genome-engineering" style="    color: cadetblue;    font-weight: bold;">CRISPR-Cas based system for bacterial genome editing</A>. This is because later we have realized that lethal effect has been associated with CRISPR-induced double-stranded break on targeted gene. This phenomenon can be explained by low occurrence of non-homologous end joining (NHEJ), one of the DNA break repairing mechanism in bacteria.  Hence, this lead to <A href="http://www.cell.com/trends/microbiology/pdf/S0966-842X(15)00019-0.pdf" style="
    color: cadetblue;
    font-weight: bold;
">irreversible DNA breakage</A>. Thus, in this situation, CRISPR-Cas system is used as a selection force rather as primary editing tool.By this selection system, only successfully recombined or modified cells are able to survive.  </P><P>As a proof of concept, this lethal effect can be shown from CRISPR-based HemH gene knock-out results that accomplished by Macquarie team. When they try to introduce purified Cas9 protein with a series of concentrations for HemH gene targeting,cells with higher amount of Cas9 protein shows significant slower cell growth. For Cas9 concentration such as 17uM, cell growth are almost negligible.This again highlight the lethal effect of high concentration of Cas9.  </P></DIV><DIV class="page-header"><H1>Generate HemH knock-out using lambda-red mutagenesis method</H1></DIV><DIV class="well"><P>Following this, our team is keen in helping Macquarie team to generate HemH knock-out bacterial strain by using another commonly used bacterial genome editing approach, known as lamba-RED recombination system. The HemH gene is knocked out by replacing it with a flp-flanked Kanamycin selection cassette. Due to the accumulation of protoporphyrin ring IX substrates, HemH knock-out resulted in light-sensitive bacteria that grow only in the dark. </P><DIV style="margin: auto; width: 800px; "><P style="font-size:18px; text-align: center;">Red: 50 nt Homology arms used for the PCR of KanR cassette. Purple: Primers used for colony PCR screening</P></DIV><P>Mutant screens were then conducted after 2 days of bacteria growing. The below gel image shows one of our screening results.</P><DIV style="margin: auto; width: 480px; "><P style="font-size:18px; text-align: center;">a) Colony PCR with primers on hemH. b) Colony PCR with primer Kanamycin gene.</P></DIV><P>
After HemH knock out, there should be no bands appears for HemH gene in the gel image while a band size of 1000bp reflecting Kanamycin gene is expected to be shown. It is unfortunate that we failed to knock out HemH gene. We had screen for hundreds colonies but none turned out to be positive.To explain the survival of bacteria against Kanamycin resistance even though Kanamycin cassette is not inserted into their genome,might be that pKD4 that originally serves as template plasmids for flp-flanked Kanamycin cassette are not fully digested with DpnI.  The second possibility is due to random insertion of Kanamycin cassette. 
</P></DIV><DIV class="page-header"><H1>Team Macquarie helped us too!! :))</H1></DIV><DIV class="well"><P>Last but not least, we would like to acknowledge and thank Team Macquarie’s effort in helping us with tasks stated as follow:</P><P>1. With RT-PCR primer designed and sent by Team Macquarie, our team was able to finish characterising the gene expression for the Mg-chelatase plasmid on time.</P><P>2.	Team Macquarie also helped us to characterise two SpCas9 mutants, 462(<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2130001" style="    color: cadetblue;    font-weight: bold;">BBa_K2130001</A>) and 459(<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2130002" style="    color: cadetblue;    font-weight: bold;">BBa_K2130002</A>)  . p.s. They also footed the shipping bill too. :P</P><P>To do so, our team had changed the FLAG purification tag into 6xHistidine tag that associated with plasmid carrying mutated SpCas9 as Team Macquarie only has kits for His-tag purification. Following purification, an in-vitro assay assay was done by Macquarie team to compare the efficiency of the mutants evolved by our team.
</P><P>The 6x Histidine tag was added upstream of FLAG purification tag. Immediately after Histidine tag, 2x stop codon were added to stop translation. The primer design for inserting 6x Histidine purification tag was shown as below:
</P><DIV style="margin: auto; width: 600px; "><P style="font-size:18px; text-align: center;">Gibson primer design for 6X His tag cloning.</P></DIV></DIV><DIV id="exhibition" class="page-header"><H2>Team NUS Singapore</H2></DIV><DIV class="well"><P>We recieved an invitation from NUS-Singapore to collaborate on our iGEM competition and were excited to pay them a visit.</P><P>During our visit, we learned about their project regarding the use of E. coli as a drug delivery system to target cancer cells. The surface marker used for the attachment of E. coli to cancer cells is CD44v6, an isoform of CD44 with exon v6, a cell surface glycoprotein expressed specifically in cancer cells. We thought that it would be meaningful if Team NUS could have a negative control where the cancer cell line already has the CD44v6 knock-ed out. With this cell line, they could test for the targeting specificity of the E. Coli. </P><P>With an online search we found that the CD44 gene is heavily spliced to produce different kinds of mutants. CD44v6 is the isoform that is expressing exon v6(tenth exon), which is only present in malignant cancer cell lines. To generate a negative control for Team NUS, we targeted three regions of the CD44 gene. We targeted the first two exons present in all isoforms of CD44 so that they can all be inactivated at once. For the third target site, we made a segmental deletion targeted to exon v6. The target sites were made as far as possible for the splice junctions to prevent disruption of the junction.</P><DIV style="margin: auto; width: 1200px; "><P style="font-size:18px; text-align: center;">gRNA design to target exon v6 for segmental in-frame deletion.</P></DIV><P>After cloning of the gRNA into the SpCas9 vector that we used for project Evaluation, we then transfected the constructs into HeLa and MDA-231 cells. However, before transfection, we made a mycoplasma test and found out that the cells we obtained from Lab A were mycoplasma contaminated. </P><DIV style="margin: auto; width: 180px; "><P style="font-size:18px; text-align: center;">Mycoplasma test</P></DIV><P>Hence, we sourced for clean cells from other labs to start our transfection. With our current transfection reagent, TurboFect, the transfection efficiency is very low for MDA231 cells but HeLa cells can be transfected.</P><DIV style="margin: auto; width: 800px; "><P style="font-size:18px; text-align: center;">24 hours after transfection of HeLa cells</P></DIV><P>After 24 hours, we sorted for transfected cells and made serial dilutions to single cells per well, which are still expanding. However, the cells takes quite some time to expand and is not able to be sequence confirmed for Team NUS's this year's competition.</P></DIV></DIV></DIV></BODY></HTML>