Team:NTU-Singapore/Description
NTU-Singapore
Toggle navigation
NTU-SG
Collaboration
Projects
Interlab Measurement
Description
Proof of concept
Demonstrate
Parts
Basic Parts
CRISPRy Collection
Team
Adventurers
Attribution
Notebook
Human Practice
Silver
Gold
Playing with Cas9 and Co.
Our team focuses on two major aspects that should be improved for the maturation of the CRISPR/Cas9 technology. Size and Efficiency. Project Truncation addresses size matters while Project Evaluation and Evolution address problems related to efficiency.
Project Truncation
Project Evaluation
Project Evolution
Project Truncation
The catalytically inactive version of Cas9, dCas9, has also been exploited for its ability as a DNA homing device.  Depending on the proteins bound to dCas9, regions targeted by the dCas9 can be epigenetically altered, thereby, providing means of epigenetic regulation to the desired locus.
One of the major impediments to CRISPR therapeutics is the huge size of the Cas9 proteins which places it’s gene length close to the limits of the AAV virus vector which is ~5kb. Current Cas9 homologs are ~3-4kb in size. In the case of the dCas9, which is heavily dependent upon its fusion partner, carries an additional load of it fusion protein. Hence, it is important to shrink dCas9 in order to have more flexibility on the size of epigenetic regulators fused.
In year 2013, Team Freiburg attempted to truncate the dCas9 protein from Streptococcus pyogenes, SP-dCas9. However, no clear methodology wes used by the team to determine truncation regions as the crystal structures of Cas9 was unavailable at that time, only to be published on 2014. This year, with available crystal structures and domain architecture of the dCas9 clearly outlined, we decided to improve on the truncation rationally.
Truncations made by Team Freiburg 2013 on SpCas91
As the function of dCas9 is only in binding, we truncated several domains of Cas9 that are not essential for binding or not having interactions with the gRNA or DNA. Studies on SP-Cas9 truncations have been done on the REC domain, with only the ∆ REC2 mutant retained the ability for DNA cleavage. We started with deleting the REC2 domain to see if we could have similar results with our GFP reporter assay.
Then, we started on the nuclease domains, RuvCI, RuvCII, RuvCIII and HNH. Motivated by our results, we continued on the REC1 domain. Lastly, we went on to explore if truncations can be made on the PAM interacting domains of SP-dCas9.
Check out our Proof of Concept page on the truncation adventure!
Project Evaluation
Currently, there have been several Cas9/Cpf1 enzymes discovered and each has their own PAM specificity and sizes. For the Cas9 protein, the PAM sequence is on the 3' end of the target sequence. On the other hand, Cpf1 has a unique PAM sequence which is on the 5' end of target sequence. Another difference is that Cpf1 produces a sticky end after double strand break while Cas9 makes a blunt end break.
For our iGEM project, we made a comparison of efficiency among 5 Cas9/Cpf1 endonucleases: Streptococcus pyogenes Cas9 (SpCas9), Staphylococcus aureus Cas9 (SaCas9), Neiserria meningitides Cas9 (NmCas9), Acidaminococcus sp. BV3L6 (AsCpf1) and Lachnospiraceae bacterium ND2006 (LbCpf1). We targeted these Cas9/Cpf1 to 5 genes, ALK, EGFR, NF1, KDM6A and STAG2 with increasing expression levels and epigenetic modifcation of H3K27Ac.
a. ChIP-seq for H3K27Ac for five genes. b. RT-qPCR to measure the expression of 5 genes. Data taken from Dr. Tan Meng How's lab
From our CRISPR survey, we have found that most of the public are concerned about the specificity of Cas9 on genome editing. Studies have shown that when shorter gRNA lengths were used, specificity of Cas9 can be increased . Hence, for each gene, we compared what is the optimal length of gRNA required for editing, starting from 17bp to 24bp. . We then computed and compared the editing efficiency via T7E assay which detects the amounts of indels on the target. In order to have fair comparisons, all 5 Cas9/Cpf1 are targeted to the same target sequence. We further tested the HDR efficiency of the Cas9 and Cpf1 with ssODN (single-stranded donor oligonucleotides).
Project Evolution
62% of the public we surveyed during our exhibition on Aug 5 and 6 expressed interest in using CRISPR/Cas9 to cure diseases such as cancer in the future, but they are also concerned about the potential risks/side effects associated with this technology due to the relatively low efficiency of the current CRISPR/Cas system. We were then highly motivated by the popular demand from the general public and brainstormed the possible methods to improve the efficiency of CRISPR/Cas technology in human cells.
In order to improve the efficiency of the system, one of the methods that our team proposed is to use directed evolution to screen for more efficient SpCas9 variants in cleaving DNA. By comparing the cleaving efficiency of the SpCas9 variants and wild type SpCas9, we can prove that the cleaving efficiency of the selected mutant(s) is better than the wild type, and hence can be used to improve the CRISPR/Cas9 system.
In order to achieve this, firstly, random mutagenesis was conducted on the RuvCII-HNH-RuvCIII nuclease domain (Addgene plasmid BPK746 #65767. BPK764 was a gift from Keith Joung)2 to generate libraries of mutants. We used Genemorph II EZClone domain mutagenesis kit (Agilent Scientific)3 to generate SpCas9 variants . A few colonies were randomly selected to assess the mutation rate of the mutagenesis kit and thus determine whether the mutants generated can be used. All colonies were then pooled together and the mutated plasmids were extracted (named EZ). A few libraries of EZ plasmids (each library contains at least 1000 colonies) were generated and subjected to the screening process.
Following this, competition assay was conducted to isolate more efficient SpCas9 variants. Both the EZ plasmids and wild type::M13F (SpCas9WT::M13F) plasmid were introduced into BL21(DE3) that carries the selection plasmid, BBa_K2130004. The M13F sequence was introduced into the plasmid backbone of SpCas9WT, which allows us to differentiate the EZ plasmids from the WT plasmid via colony PCR. As we perform iterative rounds of competition assays, a few colonies were sequenced to determine their Cas9 mutations. The recurring variants were selected as the potential candidates for more efficient SpCas9. These selected variants of SpCas9 were recreated using Gibson Assembly in order to examine their cleaving efficiency in human cells. An alternative method employed to recreate the mutants was QuikChange site-directed mutagenesis (Agilent Scientific).
After the variants being recreated, we cloned in different gRNAs (eg. WAS-2, TAT) into the plasmids and transfected HEK293T cells with these plasmids. T7 Endonuclease assay was conducted to determine the cleaving efficiency of various variants compared to the wild type SpCas9.
Work flow of evolution project
Reference
1.Truncation of the dCas9 protein. (2013), Team Freiburg. https://2013.igem.org/Team:Freiburg/Project/truncation
2.Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Kleinstiver BP, Prew MS, Tsai SQ, Topkar VV, Nguyen NT, Zheng Z, Gonzales AP, Li Z, Peterson RT, Yeh JJ, Aryee MJ, Joung JK. Nature. 2015 Jun 22. doi: 10.1038/nature14592. 10.1038/nature14592 PubMed 26098369
3.Daugherty, P. S., Chen, G., Iverson, B. L. and Georgiou, G. (2000) Proc Natl Acad Sci U S A 97(5):2029-34
