Team:Utrecht/Notebook
Top
Receptor Assay
BRET Assay
Methylation Assay
Receptor Assay
❯
August
❮
❯
September
❮
October
BRET Assay
❯
July
❮
❯
August
❮
❯
September
❮
October
Methylation Assay
❯
July
❮
❯
August
❮
❯
September
❮
October
Mo
Tu
We
Th
Fr
Sa
Su
Morning
Performed Calibration 1, 2, and 3 according to protocol. All measurements were performed using the plate reader of Seino.
Table 1. content 96 wells plate cal.1 and cal.2
1
2
3
4
5
6
7
8
9
10
11
12
A
L
dd
B
L
dd
C
L
dd
D
L
dd
E
MS
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
F
MS
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
G
MS
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
H
MS
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
dd
Calibration 1
L= 100 uL Ludox CL-X (stored at 4C)
dd= 100 uL ddH20
Measurement: Abs600, turn off pathlength correction
Calibration 2
MS= 200 ul Microsphere Stock Solution
dd= 100 uL ddH20
green= serial dilution was performed with a micropipet from E1,F1,G1,H1 - E11,F11,G11,H11 by a volume of 100uL.             Before every transfer solution was pipetted up and down 3x, after every transfer tips were discharged.
Measurement: Abs600, re-mix befor putting in plate reader and prevent bubbles, path length correction off
Calibration 3
1xFC= 200 mL 1xFC (100uL 10x fluorescein + 900ul 1x PBS pH 7.4, tube was covered with foil
P= 100 uL 1x PBS pH 7.4
green= serial dilution was performed with a micropipet from A1,B1,C1,D1 - A11,B11,C11,D11 by a volume of 100uL. Before every transfer solution was pipetted up and down 3x, after every transfer tips were discharged.
Measurement: FL, 530nm/30nm bandpass, 25-30nm with recommened excitation of 485nm, emission 520-530nm of the filter. Path length correction was turned off
Table 2. content 96 wells plate cal.3
1
2
3
4
5
6
7
8
9
10
11
12
A
1xFC
P
P
P
P
P
P
P
P
P
P
P
B
1xFC
P
P
P
P
P
P
P
P
P
P
P
C
1xFC
P
P
P
P
P
P
P
P
P
P
P
D
1xFC
P
P
P
P
P
P
P
P
P
P
P
E
F
G
H
Afternoon
LBC plates were made according to the protocol used on the wall
250ml LB 2x added to melted 250 ml WA 2x using a microwave
0.5ml was added to final solution
plates were dried in 37C incubator
Transformation device 3 + negative control interlab study
Device 3 (number 5) showed a low GFP expression, so it was tried to re-preform the tranformation. Negative control of the interlab (number 1) was not performed last time due to lack of LBC plates so was also performed.
Protocol Transformation
lskdfjslkjgdlkdfjgldkjlakmflsvmclkxmfklmgdmlkcvmblcmkb
21/08/18
Lorenzo
1/9M (RFP) was transformed from the iGEM kit according to the protocol . The coding sequence of the custom receptor, and copper promoter flanked by biobrick sites were ordered from IDT and used directly.
22/08/18
Lorenzo
Transformation was successful and a colony was picked and inoculated in chloramphenicol.
23/08/18
Felix, Lorenzo
Miniprep was performed according to the protocol. Cu-promotor         → 250 ng/μl Custom receptor    → 360 ng/μl RFP            → 91.7 ng/μl
Cu-promoter and RFP were placed in pSB1K3, custom receptor was placed in pSB1K3 as well. Cu-promoter DNA was digested with EcoRI and SpeI, RFP DNA was digested with XbaI and PstI. The custom receptor and the target-vector were digested with EcoRI and PstI.
For each of the three samples a mastermix was made. For each digestion the DNA concentration was reduced to 25 ng/ul. 4 ul of DNA sample and target vector sample were transferred to a PCR tube with 4 ul of the according restriction mix. The samples were incubated for 1 hour at 37 C. After 1 hour the samples were taken out of the incubator and enzymes were heat deactivated at 80°C for 20 min. 1ul of target vector (pSB1K3), 2 ul of Cu-promotor restricted DNA, and 2 ul restricted RFP DNA were added to one PCR tube. Furthermore 1ul DNA T4 ligase buffer and 0.5 ul of T4 ligase were added. The total volume was made up to 10 ul with 3.5 ul dH2O. The samples were incubated for one hour (RT). Enzymes were deactivated with heat kill 80°C 20 min.
DNA was transformed in DH5α according to the iGEM protocol .
24/08/18
Lorenzo
The transformation was successful. 8 colonies were picked to perform colony PCR according to the protocol . (Cu-promotor::RFP 100%, Cu-promotor::RFP 500%, Custom receptor 100%, Custom Receptor 500%).
Note: The date on the plates is date of preparation not transformation
27/08/18
Felix
Samples from colony PCR  ( 24/08/18 ) were analysed with a 0.8% Agar gel electrophoresis. Custom receptor = 1926 bp Cu-promoter::RFP = 1238 bp
1-8
Custom receptor 100%
9-16
Custom receptor 500%
17-24
Cu::RFP 100%
25-32
Cu::RFP 500%
28/08/18
Lorenzo, Felix
The results obtained in the gel electrophoresis DNA analysis ( 27/8/18 ) was not satisfactory. Therefore four new colonies were picked for colony PCR in accordance with the protocol .
1-4
Cu::RFP 100%
5-8
Cu::RFP 500%
9-12
Custom receptor 100%
13-16
Custom receptor 500%
The PCR samples were analysed with gel electrophoresis using 0.8% agarose gel. Custom receptor = 1926 bp Cu-promoter::RFP = 1238 bp
Samples 4, 9, 12, 13, 14 were selected and inoculated in 5 ml LB containing kanamycin overnight. A safe of these samples was made as well.
As the Cu::RFP sample did not yield the desired results, new cloning procedure was initiated using the iGEM protocol and buffer 2.1 instead of enzyme specific buffers. In order to ensure that we would have enough genetic material, we cloned the gBlock `copper-promoter blunt into the vector pBSK.
29/08/18
Felix
The DNA of the ( 28/08/18 ) samples was purified according to the miniprep protocol .
Sample
Concentration ng/µl
Description
1 - 8
69.9
Cu-Promotor+ RFP
9
77.5
Custom Receptor
12
91.8
Custom Receptor
13
49.4
Custom Receptor
14
85.4
Custom Receptor
The prior prepared clonation ( 28/08/18 ) was transformed in DH5α according to the iGEM protocol.
Safe made from samples Samples 4, 9, 12, 13, 14 ( 08/28/18 )
30/08/18
Felix
No colonies of the transformation( 29/8/18 ) were observed. The purified DNA of samples 4,12,14 ( 29/8/18 ) were send for sequencing with forward and reverse primer.
05/09/18
Lorenzo
Samples from sequencing were analysed. Custom receptor (12) was placed in the pSB1K3 backbone correctly. The sequence of Cu-promoter::RFP was difficult to interpret. Therefore Cu-promoter::RFP (4) was incubated again in 5 ml LB + Kanamycin from the safe overnight.
06/09/18
Jolijn
The Cu::RFP DNA was purified according to the miniprep protocol with 5 ml bacteria instead of 1.5 ml and eluted in 20 µl elution buffer. DNA was sent for sequencing.
Cu + RFP (4) Code:
Primer
1BA9ZAB414
VF
1BA9ZAB413
VR
10/09/18
Lorenzo
The sequence of Cu::RFP did not contain the copper promotor, only RFP.
11/09/18
Lorenzo
The RFP DNA was cloned into the Copper-promoter vector by cutting the copper-promoter vector with SpeI and PstI, and the RFP vector with XbaI and PstI overnight at 37 °C.
12/09/18
Felix
The prior cut samples were ligated. 2.4 µl RFP DNA was used as insert with 1 µl of Cu promotor backbone. After heat inactivation the sample was transformed according to the iGEM protocol in DH5α and plated on Ampicillin plates.
13/09/18
Lorenzo
Transformation was successful. However, close examination of the samples under the fluorescence binoculars revealed that none of the colonies expressed RFP.
2 colonies were picked and inoculated in 5 ml LB + ampicillin.
22/09/18
Lorenzo
Since our previous attempts didn’t yield any results we decided to place the copper promoter gBlock with the restriction enzymes EcoRI and SpeI. Simultaneously, the RFP vector (PSB1C3) was cut with EcoRI and XbaI. Both reactions were done overnight  in NEB Buffer 2.1.
23/09/18
Lorenzo
The digestion product from 09/22/18 was ligated according to the iGEM ligation protocol . 2 µl of ligation product was transformed to E. coli DH5α.
24/09/18
Lorenzo
The reaction mix was inactivated by heating it to 80°C for 20 minutes. A ligation was set up with the fragments, and 9 µl of the ligation product was transformed to 100 µl of competent DH5α.
25/09/18
Felix
The transformation was successful. 6 colonies could be spotted. These were inoculated overnight in 5 ml LB with chloramphenicol.
26/09/18
Lorenzo, Felix
The DNA of the inoculated cultures was minipreped according to the miniprep protocol . After the first spin down we noticed that the bacterial pellet of one of the colonie number 6 was slightly red. Because it is known that RFP expressing bacteria turn red even without excitation, we concluded that it was most likely that this colony expressed RFP, and thus that the transformation worked. After the isolation of the DNA, a colony PCR was done with the VF and VR primer. The total size of the band that we expected was 824 + 156 -6 = 974 bp for the fragment + 271 bp for the part outside the BB_sites generated by the VF and VR primer. The total size of the fragment should thus be 1245 bp. The fragment in lane 6 lies between the 1000 bp band and the 1500 bp band and starts slightly lower than the middle, indicating that it is most likely a band of correct size.
The DNA of this colony was send for sequencing.
29/09/18
Lorenzo
The sequencing results showed that the construct was correct. Since both the promoter::RFP fusion, and the Custom receptor were now available to us on plasmid backbones with different resistance genes, both constructs were transformed to E. coli DH5α.
30/09/18
Lorenzo, Felix
The transformation was successful. A stock solution of 50 mM CuSO4 was made for experiments with the newly transformed copper sensitive bacteria. 2 colonies were inoculated overnight for further experiments.
01/10/18
Lorenzo
100 µl of the overnight cultures from 30/09/18 was transferred to two tubes containing 5 ml of clean LB. One tube contained 500 µM CuSO4. Both tubes were inoculated for 45 minutes at 37°C.   400 µl of this culture was washed in 1 ml of PBS and RFP expression was measured with an emission scan between 570 and 640 nm in a Carry Eclipse Fluorescence Photospectrometer using 400 µl of the copper bacterium containing PBS solution. The excitation wavelength used was 555 nm.  No fluorescence could be measured, which was probably due to the low amount of bacteria.
02/10/18
Jolijn
A new overnight inoculation of the double transformed bacteria was made.
03/10/18
Felix, Lorenzo
The experiment of 10/01/18 was repeated, but this time CuSO4 was added to the inoculation at a final concentration of 500 µM, 250 µM, 50 µM, and 5 µM, or 0 µM. The Bacteria were inoculated for 2 hrs at 37°C.
12/10/18
Felix
The following bacteria (DH5α) were inoculated in 5 ml minimal medium, and  5 ml LB with matching antibiotic.
Cu-promotor::RFP // Custom receptor     → Ampicillin, Chloramphenicol Cu-promotor::RFP                    → Chloramphenicol
13/10/18
Jolijn, Felix, Lorenzo
Bacteria in minimal medium did not grow.  Cu-promoter::RFP // Custom receptor samples were red coloured while bacteria not containing the custom receptor were yellowish. Fluorescence images were taken to confirm RFP expression. The bacteria containing the Cu-promoter::RFP // Custom receptor were washed 1x in M9 and incubated again overnight in M9 containing 0.4% glucose.
In order to confirm aspartic acid induced RFP expression the experiment was repeated. The following bacteria (DH5α) were inoculated in 5 ml M9 + 0.4% glucose, and in 5 ml LB with matching antibiotic.
Cu-promotor::RFP // Custom receptor     → Ampicillin, Chloramphenicol Cu-promotor::RFP                    → Chloramphenicol
Cu-promoter :: RFP Normarski
Cu-promoter :: RFP Nomarski Fluorescence
Cu-promoter :: RFP // Custom receptor
Cu-promoter::RFP // Custom receptor fluorescence (falsely coloured)
14/10/18
Felix, Jolijn
Bacteria in M9 did not grow. Bacteria in LB did grow but did not express GFP observed with the naked eye. 4 ml bacteria of each of the two samples was washed with 1x with M9 and incubated with M9 + 0.4% glucose.
After 4 hours of  incubation in M9 + 0.4% glucose the samples were split up in the following fractions. After 4 hours 500 μl was used for fluorescence measurements using the prior described fluorescence photospectrometer (ex. 585 nm).  After 7 hours the remaining 500 μl was used for measurements.
Sample
Asp (10 μ 50 mM)
Cu (2 μl 25 mM)
Custom Receptor
1
+
-
Custom Receptor
2
-
-
Cu-promoter :: RFP
3
+
-
Cu-Promoter :: RFP
4
-
+
Cu-Promoter :: RFP
5
-
-
Sample 1 is not representative at 8 hours since the remaining volume of bacteria was less than the volume in the cuvette. Therefore this sample was diluted which resulted in a increased intensity as the cuvette was now fully filled, however, the measurement did not reach its full potential.
The Cu-promoter::RFP // Custom receptor sample washed ( 13/10/18 ) and incubated overnight was split up in two 4 ml fractions. To one fraction 40 μl  50 mM Aspartic acid was added and incubated for 4 hours.  After 4 hours 500 μl was used for fluorescence measurements, this was repeated at 8 and 11 hours
Although these data are not conclusive it indicates that the custom receptor has a reaction on the addition of aspartate. Therefore new samples were inoculated in 5 ml LB.
Custom // Cu-promoter :: RFP
3x
Custom // Cu-promoter :: RFP Asp+ 500  μM
1x
Cu-promoter :: RFP
3x
Cu-promoter :: RFP  Asp+ 500 μM
1x
Cu-promoter :: RFP 50 μM CuSO4
1x
15/10/18
Felix, Jolijn
The three custom receptor samples and one co-promoter::RFP sample did not fully grow after 9 hours of inoculation, these samples were put back into the incubator for 15 hours and can be seen in the table underneath. The other samples were incubated for 4 hours in M9.
Sample
Volume
OD600
1. Co-promotor + RFP
4 mL
1.54
2. Co-promotor + RFP
4 mL
1.35
3. Co-promotor + RFP
3 mL
2.70
4. Promotor + RFP + Aspartate
3 mL
0.47
Copper was added to the samples incubated in M9 in concentrations of 25 and 50 µM. RFP expression was measured at t = 0, t = 2. However, results from the scan were inconclusive.
The three custom receptor samples and one cu-promoter::RFP sample incubated in total for 24 hours. These samples were used for new inoculations as described below in increased volumes. The remaining volume of the inoculations were pelleted and washed once with Milli Q and resuspended in M9, in total, all samples were resuspended in 5 ml M9 (Orange in the table). During the pelleting the pellets of samples containing custom receptor turned red.
Sample
Medium
Volume
Custom receptor // Cu-promoter::RFP
LB + ampicillin and chloramphenicol
100 ml
Cu-promoter::RFP
LB + chloramphenicol
100 ml
Custom receptor // Cu-promoter::RFP
SV + 0.4% glucose + ampicillin and chloramphenicol
100 ml
Cu-promoter::RFP
SV + 0.4% glucose + chloramphenicol
100 ml
Cu-promoter::RFP
LB + chloramphenicol
5 ml
Cu-promoter::RFP
LB + 25 μM CuSO4 + chloramphenicol
5 ml
Cu-promoter::RFP
LB + 50 μM CuSO4 + chloramphenicol
5 ml
Cu-promoter::RFP
LB + 100 μM CuSO4 + chloramphenicol
5 ml
3x Custom receptor // Cu-promoter::RFP
M9 + 0.4% glucose
5 ml
1x Cu-promoter::RFP
M9 + 0.4% glucose
5 ml
16/10/18
Felix, Jolijn
The samples that were incubated with copper the day before, were measured in PBS with 0.4% glucose.    Copper was added to the samples with custom receptor//Cu-promotor::RFP and Cu-promotor::RFP in concentrations of 25, 50, 100 and 250 µM. Next, RFP fluorescence was measured at t = 0h, t = 1.5h and t = 2.5h.
5 mL of the 25 mL overnight LB cultures of Custom receptor // Cu-promoter::RFP (OD600 of 0.675) and Cu-promoter::RFP (OD600 of 0.690) were used to attain the following solutions:
Sample with nothing added
Sample with 125 µM aspartate
Sample with 250 µM aspartate
Sample with 500 µM aspartate
Sample with 50 µM copper
RFP fluorescence was measured at t = 0, t = 4, and t = 6. There was no fluorescence. Upon measuring the fluorescence of the strain from which the samples originated (incubated in LB overnight), only slight fluorescence was visible.
30/07/18
Lorenzo, Floor
Isolated biobricks (listed in table below) and transformed them to E. coli Dh5ɑ according to the transformation protocol .
Biobrick
Kit location*
Part name
Part size (bp)
BBa_K608003
1/5A
Promoter +RBS
56
BBa_K569017
1/8K
CheY
390
BBa_K629003
1/18G
CheZ
644
BBa_I759001
3/3J
Rluc
936
BBa_B0017
3/6C
Terminator
128 **
BBa_I15017
4/21N
eYFP
717
* location name is as follows: plate number/well number (e.g. 1/5A means plate 1 well 5A) ** this part contains two copies of BBa_B0010 (64 bp each) but the exact size of the fragment is unspecified on the wiki page.
31/07/18
Lorenzo, Floor
Transformation was successful, colonies were picked and inoculated in 5 ml LB containing the correct antibiotic.
01/08/18
Lorenzo, Floor
Isolated the plasmids containing the biobricks from the inoculated colonies from 31/07/18 according to the plasmid purification protocol . (for concentrations see table below)
Concentration (ug/ml)
Biobrick
Kit location*
Part name
Replicate 1
Replicate 2
BBa_K608003
1/5A
Promoter +RBS
76,8
77,8
BBa_K569017
1/8K
CheY
51
50,5
BBa_K629003
1/18G
CheZ
83,1
111,9
BBa_I759001
3/3J
Rluc
81,2
89,7
BBa_B0017
3/6C
Terminator
137,8
75,4
BBa_I15017
4/21N
eYFP
70
70,9
Performed a PCR, as described in the PCR protocol , to amplify the parts CheY, CheZ, Rluc, eYFP so that they can be used for Gibson assembly later in the project. The primers used are listed in the table below.
Part
Primer name
Primer sequence 5’--> 3’
CheZ
CheZ_F1
gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag
CheZ_R1
CCTTGGAAGCCATTCCACCTCCACCTCCAAATCCAAGACTATCCAACAAATCG
Rluc
Rluc_F1
AGTCTTGGATTTGGAGGTGGAGGTGGAATGGCTTCCAAGGTGTACG
Rluc_R1
tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag
eYFP
YFP_F1
gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag
YFP_R1
CTTATCAGCCATTCCACCTCCACCTCCCTTGTACAGCTCGTCCATGCCG
CheY
CheY_F1
CGAGCTGTACAAGGGAGGTGGAGGTGGAATGGCTGATAAGGAATTGAAG
CheY_R1
tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag
02/08/18
Lorenzo
Performed a gel electrophoresis on the PCR product from 01/08/18 on a 1.5% agarose gel  according to the gel electrophoresis protocol . The resulting bands were isolated according to the gel purification protocol .
PCR product
Concentration (ug/ml)
CheY
/
Rluc
45,6
CheZ
59,9
eYFP
26,6
A second PCR was performed to retrieve CheY. Also eYFP, LuxA, and LuxB were amplified with  overhang extension PCR to add a gibson overhang to these parts. Used primers are listed in the table below.
Part
Primer name
Sequence 5’ -> 3’
LuxA
M13 fwd
tgtaaaacgacggccagt
LuxAB_Gibson_R1
gatttGGAGGTGGAGGTGGAATGAAGTTTGGAAACTTCCTG
LuxB
LuxAB_Gibson_F1
gatttGGAGGTGGAGGTGGAATGAAGTTTGGAAACTTCCTG
T3
tccctttagtgagggttaat
eYFP
YFP_F1
gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag
YFP_R1
CTTATCAGCCATTCCACCTCCACCTCCCTTGTACAGCTCGTCCATGCCG
CheY
CheY_F1
CGAGCTGTACAAGGGAGGTGGAGGTGGAATGGCTGATAAGGAATTGAAG
CheY_R1
tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag
The PCR product was ran on a 1.5% agarose gel (figure 2)
03/08/18
Lorenzo
LUXA and LuxB were isolated using PCR directly from the gBlock, since we expected the product of 02/08/18 to contain an error. The PCR product was ran on a 0.8% agarose gel and the correct bands were isolated according to the gel extraction protocol.
06/08/18
Floor
The CheZ and RLuc, and LuxA and LuxB fragments were fused together using Gibson assembly as described in the Gibson protocol . 9 µl of each Gibson product was transformed to 100 µl of DH5ɑ, since this increases the amount of successful transformations.
07/08/18
Floor
The transformations were successful. 2 colonies from each plate were picked and analysed for bands of a correct size through colony PCR as described in the colony PCR protocol . These bacteria were inoculated overnight at 37 °C in a rotary shaker.
08/08/18
Lorenzo
DNA of inoculations ( 07/08/18 ) was purified in accordance with the miniprep protocol .  The acquired DNA was sent for sequencing.
09/08/18
Lorenzo
PCR was performed according to the protocol to create the correct overhangs for gibson assembly of CheY.
10/08/18
Jolijn, Felix
PCR product from 09/08/18 was ran on 0.8% gel and isolated the DNA using the Gel purification protocol
15/08/18
Floor, Lorenzo
eYFP::CheY was assembled according to the Gibson cloning protocol . The cloning product was transformed into DH5α according to the iGEM transformation protocol .
16/08/18
Lorenzo
Made a plate containing the positive colonies and inoculated them overnight.
Colony PCR of (( 18/08/18 eYFP::CheY)( 08/07/18 CheZ::Rluc)) was performed according to the protocol . Positive samples were selected and inoculated overnight.
17/08/18
Felix eYFP::CheY DNA 16/08/18
Inoculation from colony
Concentration (ng/µl)
1
224.5
2
277.5
3
334.4
4
280.4
20/08/18
Felix
The purified DNA samples ( 17/08/18 ) were sent for sequencing with m13 forward and reverse primer.
27/08/18
Lorenzo, Floor
Performed a PCR of CheY, Lux AB, CheZ and eYFP with the following primers:
DNA
Forward Primer
Reverse Primer
1. CheY (kit)
Long Forward
CheY reverse
2. CheY (kit)
Short forward
CheY reverse
3. CheY (Gblock)
GblockCheY forward
Reverse 2
4. LuxAB
LuxAB forward, CheZ overhang
M13 reverse
5. CheZ (kit)
CheZ forward 1
CheZ reverse, luxAB overhang
6. eYFP
eYFP forward
eYFP reverse
The samples were ran on an 0.8% gel and bands were isolated
1
2
3
4
5
6
10
10.1
5.7
3.7
0.9
0
28/08/18
Floor
Repeat of 27/08/18 , difference: PCR settings as KOD polymerase with 35 cycles.  Ran the product on 0.8% (w/v) agarose gel and isolated it using the gel purification protocol .
The fragments were used to perform a Gibson assembly of eYFP::CheY, and 9 µl of the product was transformed to 100 µl DH5α.
29/08/18
Floor
We checked the plates from 28/08/18 but none of the transformations were successful.
03/09/18
Lorenzo
We performed the gibson reaction of eYFP::CheY again. The gibson product was transformed directly to DH5α. Furthermore, a Gibson overhang for the CheZ and LuxAB fragments was created by PCR with primers that were extended. The PCR products were ran on a 0.8% agarose gel. The gel revealed faint bands, but no ladder. Hence we didn’t proceed with these samples.
11/09/18
Lorenzo
Since our constructs used during our project needed a promoter in order to be expressed promoter BBa_K608003 was isolated from the DNA distribution kit (plate 1 well 5A). This DNA was transformed to DH5α.
12/09/18
Felix
Colonies from 11/09/18 were picked and inoculated overnight.
13/09/18
Lorenzo
DNA from inoculations was isolated according to the miniprep protocol . The promoter part was was cloned in front of Both eYFP::CheY from 28/08/18 , and CheZ::RLuc using the iGEM 3A assembly protocol . The samples were transformed to DH5α.
14/09/18
Lorenzo
eYFP::CheY plates were observed under the fluorescence binoculars. 2 YFP colonies could be observed. These colonies were inoculated overnight at 37°C.
15/09/18
Felix
The inoculated bacteria were used for a miniprep and the DNA was send for sequencing.
06/10/18
Felix Oligo annealing
Oligos containing the promoter CheZ::RLuc part sequence were annealed according to the annealing protocol
CheZ Rluc Digestion
Two CheZ::Rluc samples (CheZ::Rluc 440 ng/μl, CheZ::Rluc 430 ng/μl) were digested with Xbal and EcoR1 according to the iGEM restriction digest protocol .
Oligo CheZ::Rluc ligation
Protomoter Oligo was ligated into the Xbal Rluc digested CheZ::Rluc backbone according to the iGEM T4 ligase protocol . 1 μl  of CheZ::Rluc backbone with 1 μl of oligo was used. Both 440 ng/μl and 430 ng/μl sample were ligated with the oligo.
Transformation
The new vector product (promoter::CheZ::Rluc) was transformed into DH5α according to the iGEM transformation protocol .
CheZ Rluc A
CheZ Rlux B
Anneal A = Oligo insert::CheZ::Rluc        (originating from sample CheZ::Rluc 430 ng/μl ) Anneal B = Oligo insert::Chez::Rluc         (originating from sample CheZ::Rluc 440 ng/μl )
07/10/18
Felix
Transformation was succesful
Three colonies were picked from transformation A and two colonies were picked from transformation B, both were inoculated in 5 ml LB containing ampicillin.
08/10/18
Lorenzo
DNA was purified from the following samples using the miniprep protocol and sent for sequencing.
Sample
Concentration ng/μl
CheZ Rluc A4
584.1
CheZ Rluc B4
549.6
CheZ Rluc A2
522.3
CheZ Rluc A1
463.8
CheZ Rluc B3
539.9
The Gibson cloning was transformed in DH5α according to the protocol .
09/10/18
Lorenzo
Transformation was successful
Two colonies were picked and inoculated in 5 ml LB containing ampicillin.
10/10/18
Lorenzo, Felix
DNA from the two inoculation of 1/5A::CheZ::Rluc after Gibson cloning was retrieved with miniprep according to the protocol in the following concentrations.
Sample 1:  139.9 ng/μl Sample 2:  75.5 ng/μl
11/10/18
Felix, Jolijn, Mike
In order to implement the complete BRET-pair and the mutated TAR-receptor, all three constructs needed to be in different vectors. Therefore, the CheZ Rluc and 1/5A::eYFP::Chey DNA was put in the iGEM pSB1K3 vector. Both samples including the vector were cut with EcoRI and PstI. Assembly of products was performed according to the iGEM 3A assembly protocol resulting in the following samples:
1/5A eYFP CheY in pSB1K3 1/5A CheZ Rluc in pSB1K3 (sample 1) 1/5A CheZ Rluc in pSB1K3 (sample 2)
These samples were transformed in combination with the mutated TAR receptor DNA. The following samples were transformed.
Sample
Content
Cell line
1
SDM Q491A
DH5α
2
SDM E309A
DH5α
3
[⅕A::eYFP::CheY] Kan
U1250
[SDM Q491A] Chlo
[CheZ::Rluc] Amp
4
[CheZ::Rluc (sample 1)] Kan
U1250
[SDM Q491A] Chlo
[⅕A::eYFP::CheY] Amp
5
[CheZ::Rluc (sample 2)] Kan
U1250
[SDM Q491A] Chlo
[⅕::eYFP::CheY] Amp
6
[[⅕A::eYFP::CheY] Kan
U1250
[SDM E309A] Chlo
[CheZ::Rluc] Amp
7
[CheZ::Rluc (sample 1)] Kan
U1250
[SDM E309A] Chlo
[⅕A+eYFP+CheY] Amp
8
[CheZ::Rluc (sample 2)] Kan
U1250
[SDM E309A] Chlo
[⅕A::eYFP::CheY] Amp
9
[⅕A::eYFP::CheY] Kan
DH5α
10
[CheZ::Rluc (sample 1)] Kan
DH5α
11
[CheZ::Rluc (sample 2)] Kan
DH5α
12/10/18
Felix, Lorenzo
SDM transformations (Lab Journal lab journal Methylation ) and triple transformations were successful. When the plates were inspected under the fluorescence binoculars, several eYFP positive colonies could be identified that weren’t RFP positive. This indicated that these bacteria contained all 3 plasmids, and weren’t antibiotic resistant due to self ligation events of the linearized vector backbone used on 11/10/18 .
2 colonies were picked and inoculated in 5 ml LB with ampicillin, kanamycin, chloramphenicol. The selection of colonies was based on increased YFP expression and low RFP expression (background plasmid signal).
The following samples were transformed
Sample
Cell line
6/15B (TAR WT),
U1250
FRET pair
U1250
BRET pair (1/5A::CheZ::Rluc)(1/5A::eYFP::CheY)
U1250
6/15B (TAR WT), FRET pair
U1250
6/15B (TAR WT), BRET pair (1/5A::CheZ::Rluc)(1/5A::eYFP::CheY)
U1250
FRET pair, SDM E309A
U1250
FRET pair, Q491A
U1250
13/10/18
Felix, Jolijn
Inoculations were spun down, washed once with 1x PBS and resuspended in 1x PBS. 500 μg coerentazine was solubilized in ethanol to reach a final concentration of 7.5 mM. An aliquot was taken from this stock to make a 30x solution with 1x PBS. 480 μl bacteria were put in a 400 μl quartz cuvette and 20 μl  30x coerentazine solution was added. The solution was oxygenated using a pipette. No luminescence could be observed, indicating that the coelenterazine is not oxidized by luciferase. Fluorescence measurements of a wavelength that excites YFP,  proved YFP to be present.
Jolijn
17/10/18
Samples that were transformed on 12/10/18 , consisting of both the BRET pair and the tar receptors with altered amino acid sequence (to mimic methylation) were inoculated in LB.
Jolijn
17/10/18
The inoculated samples were transferred to a citrate phosphate buffer (0.15 M, pH 5). Coelenterazine was added to a final concentration of 7.5 µM. Measurements were performed, there was no measurable luminescence.
30/07/18
Jolijn
Transformation of Tar part 1
The Tar, Tar GFP and Tar GFP His plasmids received on a filter paper from team Technion Israël 2016 were dissolved in 80 µL elution buffer. ~40 µL of plasmid dissolved in elution buffer was extracted. Competent DH5α cells were then transformed with the dissolved plasmids according to the iGEM protocol . Since the type of antibiotic resistance was unknown, Kanamycin, Ampicillin and Chloramphenicol were used as antibiotics.
31/07/18
Jolijn
Transformation of Tar part 2
The transformation of Tar GFP His on chloramphenicol was the only successful transformation. Two colonies were picked and inoculated in 5 ml LB containing chloramphenicol.
01/08/18
Jolijn
Transformation of Tar GFP His part 3
Tar GFP His DNA was isolated from the inoculation ( 31/07/18 ) according to the MiniPrep protocol . An aliquot of the left bacteria was plated in fresh agar plates.
Transformation of Tar part 1
The DNA concentration of the Tar, Tar GFP and Tar GFP His samples were measured, concentrations are listed in the table below.
Sample
Concentration
Tar
10.4 µg/mL
Tar GFP
9.1 µg/mL
Tar GFP His
11.1 µg/mL
The desired concentration of DNA for transformation is 100 ng in a 50 µL solution. Due to the low concentration of our samples, 5 µL of DNA sample was added to the 50 µL DH5α competent cells.  Additionally, the Tar receptor (6/4j) and Tar receptor with promotor (6/15B) plasmids from the distribution kit were transformed as described in the iGEM transformation protocol .
02/08/18
Felix, Jolijn
Transformation of Tar part 2
All transformations were successful, with a high colony density. One colony per sample was inoculated for 6 hours. After 6 hours the colonies were put in 5 ml LB + chloramphenicol and incubated overnight 37°C 200 rpm. Samples were as described below.
Tar A
Ta
Tar B
Tb
Tar Kit A
Tka
Tar Kit B
Tkb
Tar GFP A
Tga
Tar GFP B
Tgb
Tar GFP His A (Israel)
Tgha
Tar GFP His B (Israel)
Tghb
Promotor Tar Kit A
Ptka
Promotor Tar Kit B
Ptkb
03/08/18
Felix, Jolijn
Transformation of Tar part 2
DNA was retrieved from the following samples using miniprep according to protocol .
Receptor
ug/ml
T B
65.8
T A
66.5
Tk A
60.7
Tk B
75.4
Tg A
73.3
Tg B
69.1
Tgh B
61.6
TghA
77.7
Ptk A
68.6
Ptk B
78.9
Site Directed Mutagenesis (SDM) was performed to mimic constant methylation of the Tar receptors. The experiment was carried out according to the SDM protocol . Only B samples from the duplo were used for SDM. Following primers were used to specifically mutate Q491A and E309A.
E491A_F
GCATCGCTGGTGCAGGCGTCAGCTGCCGCCGCC
E491A_R
GGCGGCGGCAGCTGACGCCTGCACCAGCGATGC
Q309A_F
CTGCCGCCAGCATGGAGGCGCTCACCGCGACAGTGAAG
Q309A_R
CTTCACTGTCGCGGTGAGCGCCTCCATGCTGGCGGCAG
06/08/18
Jolijn
The prior made ( 03/08/18 ) SDM plasmids were transformed in duplo into DH5α cells according to the iGEM protocol .
07/08/18
Felix
Transformation was successful. The colony density was very high, which made it difficult to pick specific colonies for further incubation. Yet, colonies were picked and activated in 1 ml LB + chloramphenicol for 5 hours. After 5 hours no bacterial growth was observed.  Colonies were picked again and inoculated overnight in 5 ml LB + chloramphenicol. To ease colony picking, a colony  of each sample was picked roughly and spread and diluted on a new agar plate + chloramphenicol. This plate was incubated overnight at 37°C.
08/08/18
Jolijn, Felix
The colonies picked on 07/08/18 were diluted successfully. The plasmid DNA of the inoculations  ( 07/08/18 ) was purified according the miniprep protocol .
Following concentrations were observed.
Receptor
Concentration ug/ml
Tb2
43.6
Tb1
49.1
Tgb2
52.7
Tgb1
28.1
Tkb2
46.33
Tkb1
46.3
Tgh2
15.3
Tgh1
50.8
Ptkb2
3
Ptkb1
32.6
Tb2 and Tgb2 were sent for sequencing, although DNA concentration was not optimal.   DNA concentrations were not satisfactory, new colonies were picked and inoculated in 5 ml LB with chloramphenicol. A control was included to compare retrieved DNA concentrations.
09/08/18
Jolijn, Felix
DNA from inoculations was retrieved using miniprep according to the protocol. One modification was applied, the elution buffer was heated (60°C) before use and only 30 µl was applied on the spin column.
Receptor
Concentration (µl/ml)
T1
102.9
T2
106.2
Tg1
85
Tg2
110.2
Tk1
33.9
Tk2
68.8
Ptk1
13.3
Ptk2
71
Tgh1
48.2
Tgh2
49.1
Control (Vector = PbsK)
428.4
13/08/18
Felix
TG, TG2, T1,T2 were sent for sequencing.
14/08/18
Jolijn
Sequencing data revealed presence of GFP in all the sent sequences.
27/08/18
Jolijn
Colony picking of E. coli
E. coli used for the interlab study were picked and inoculated for an hour.
Genomic PCR
The obtained samples were used for colony PCR. Afterwards, the samples were loaded on a 0.8% agarose gel at 120 V for 20 minutes. Bands were isolated and purified, using the gel purification protocol. The concentrations were too low to continue (1.3µL and 1.9µL).
29/08/18
Jolijn
Receptor assay inoculations (custom receptor//Cu-Promoter::RFP sample 13 and 14 were used for genomic PCR. 1 mL of these cell cultures were used and spun down on 12000 x g for 1 minute. The supernatant was removed and resuspended in 100 µL milli-Q, after which custom 13 was heated at 85 °C’. The samples were diluted 0, 10 and 100 times and the resulting dilutions were used for 1-Taq PCR using the KOD polymerase protocol .
Next, 50 µL loading buffer was added and all samples from custom 13 and 14 were loaded in lane 2 and lane 3 of a 0.8% agarose gel respectively. The bands were isolated and put together. The DNA concentration was 146.1 µg/mL.
30/08/18
Jolijn
T4 PNK Phosphorylation and T4 ligation
The DNA fragments were phosphorylated according to the T4 PNK Phosphorylation protocol Using the following pipet scheme:
Compound
Volume
DNA
15 µL
10x buffer
2 µL
ATP 10 mM
2 µL
T4 PNK
1 µL
Next, ligation was done in order to ligate the isolated Tar receptor DNA into the pBSK vector. This was done according to the following pipetting scheme:
Compound
Volume
T4 ligase
0.9 µL
Buffer
1 µL
Vector
1 µL
DNA
7.60 µL
After four hours of incubation, competent DH5α cells were transformed with the sample in duplo according to the iGEM transformation protocol .
11/09/18
Jolijn, Lorenzo
Transformation was successful, colonies were picked.  Cells were incubated in 4 mL LB with 4 µL ampicillin.
30/09/18
Lorenzo
Tar receptor received from Groningen and transformed into DH5a according to the protocol .
03/10/18
Lorenzo
Tar receptor DNA was isolated using miniprep, DNA concentration was 356.3 ng/ul.  First site directed mutagenesis was performed (E491A) using phusion polymerase and listed HPLC purified primers.
E491A_F
GCATCGCTGGTGCAGGCGTCAGCTGCCGCCGCC
E491A_R
GGCGGCGGCAGCTGACGCCTGCACCAGCGATGC
Q309A_F
CTGCCGCCAGCATGGAGGCGCTCACCGCGACAGTGAAG
Q309A_R
CTTCACTGTCGCGGTGAGCGCCTCCATGCTGGCGGCAG
04/10/18
SDM1 (E491A) was digested with DpnI for 1.5 hours and heat inactivated for 30 min.  SDM2 (Q309A) was performed with SDM1 (E491A) as template. As the DNA concentration could not be retrieved, a titer of three different DNA concentrations was in three different DNA volumes:
0.3 ul
0.7 ul
1.1 ul
SDM1 (E491A) and SDM 2 (Q309A) were transformed in DH5a. During transformation, heat shock was performed inaccurately, therefore, the transformation was performed again.
1.1 (E491A)          →         SDM 1 2.1 (Q309A)          →        SDM A 0,3 ul 2,1 (Q309A)        →         SDM B 0,7 ul 2,1 (Q309A)         →         SDM C 1,1 ul
Second Transformation:
1.1 (E491A)        →         SDM 1 second time 2.1  (Q309A)        →         SDM 2.1 A second time 0.3 ul 2.1  (Q309A)        →         SDM 2.1 B second time 0,7 ul 2.1  (Q309A)        →         SDM 2.1 C second time 1.1 ul
05/10/18
Felix Several transformations were successful. Two colonies from the following transformations were inoculated 1.1 (E491A) first time 1.1 (E491A) second time 2.1 (Q309A) B 0,7 ul
05/10/18
Felix, Lorenzo
DNA from the following samples was purified according to the miniprep protocol.
DNA 6/15B (Groningen)         (DNA 6/15B     → SDM 1.3 (E491A)) SDM (E491A) 1.2 A            (SDM 1.2 A     → SDM 2.2 (Q309A) A) SDM (E491A) 1.2 B              (SDM 1.2 B     → SDM 2.2 (Q309A) B)
Following samples were transformed.
Chloramphenicol + Ampicillin, U1250
pVS88 + Tar SDM 1.2  (E491A) A
pVS88 + Tar SDM 2.1  (Q309A) A
pVS88 + Tar SDM 2.2 (Q309A) A
pVS88 + Tar SDM 2.2 (Q309A) B
Chloramphenicol DH5α
Tar SDM 2.2 A (Q309A) (Not digested with DpnI overnight)
Tar SDM 2.2 B (Q309A) (Not digested with DpnI overnight)
Tar SDM 2.1 A (Q309A)
Tar SDM 2.1 B (Q309A)
Tar SDM 2.1 C (Q309A)
Tar SDM 1.3 (E491A)
07/10/18
Jolijn
Following sequences were transformed according to the iGEM protocol to DH5α
SDM 2.1 (Q309A) C
SDM 2.2 (Q309A) A after overnight DpnI
SDM 2.2 (Q309A) B after overnight DpnI
Transformations were succesfull
Following sequences were inoculated
CheZ Rluc A (1)
CheZ Rluc B (1)
CheZ Rluc A (2)
CheZ Rluc A (3)
CheZ Rluc B (2)
SDM 2.1 (Q309A) B
SDM 1.3 (E491A) (1)
SDM 1.3 (E491A) (2)
08/10/18
Felix
DNA was acquired from samples according to the miniprep protocol with the following DNA concentrations.
Sample
Concentration (ng/μl)
SDM 2.1 (Q309A) B
171.1
SDM 1.3 (E491A) (1)
212.2
SDM 1.3 (E491A) (2)
110.3
Inoculation
Transformations: SDM 2.2 (Q309A) A after overnight DpnI, SDM 2.2 (Q309A) B after overnight DpnI, SDM 2.1 (Q309A) C  (1.1 μl), were successful.
Two colonies were picked from the following samples:
SDM 2.2 (Q309A) A after overnight DpnI
SDM 2.2 (Q309A) B after overnight DpnI
One colony was picked from
SDM 2.1 (Q309A) C  (1.1 μl)
09/10/18
Felix
DNA from the following inoculations was purified
2x SDM 2.2 (Q309A) A after overnight DpnI
2x SDM 2.2 (Q309A) B after overnight DpnI
1x SDM 2.1 (Q309A) C (1.1 μl)
The following concentrations were measured and sent for sequencing.
Sample
Concentration (ng/μ)
Sent for Seq.
SDM 2.1 (Q309A)  C
253.2
VF: 1BA9ZAB682
VR: 1BA9ZAB681
SDM 2.2 (Q309A) A
188
SDM 2.2 (Q309A) A2
309
VF: 1BA9ZAB678
VR: 1BA9ZAB677
SDM 2.2 (Q309A) B
219.7
VF: 1BA9ZAB680
VR: 1BA9ZAB679
SDM 2.2 (Q309A) B2
173.7
10/10/18
Felix
Analysis of the sequences revealed that none of the samples contained the desired DNA sequence with the specific mutation.
11/10/18
Felix, Lorenzo
Again SDM was performed (SDM Q491A as well as SDM E309A) according to the protocol using phusion polymerase with lower primer concentrations of vector DNA. SDM Q491A as well as SDM E309A were transformed in DH5α according to the protocol. The two SDM samples were also transformed in U1250 with the BRET pair (lab journal BRET ).
11/10/18
Felix
SDM transformations were successful. 2 colonies from each SDM sample were picked and inoculated in 5 ml LB with chloramphenicol. SDM transformation in combination with the BRET pair were successful as well (lab journal BRET ).
E309A
Q491A
Project Design
Results
Protocols
