<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Utrecht_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Utrecht/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><BUTTON onclick="topFunction()" id="BackToTop" title="Go to top">Top</BUTTON><DIV class="sidenav" id="side-menu"><UL><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Aim1">Receptor</A></LI><LI><A href="#Aim2">BRET</A></LI><LI><A href="#Aim3">Methylation</A></LI></UL></DIV><DIV class="customelementM5A" id="Introduction"><P> DeTaXion is a bacterial-based sensor for the detection of pharmaceutical contaminants in surface water. In its final form, DeTaXion will consist of a series of bacteria that can detect a wide variety of compounds. This will be achieved by engineering the chemotaxis pathway of <I>Escherichia coli</I> to emit a visible light signal in response to different pharmaceutical compounds. The project consists of three main aims:

Aim 1: Explore the possibility of using the ligand binding domains from different transmembrane receptors.

Aim 2: Development of a Bioluminescence Resonance Energy Transfer (BRET) readout for the activity of the chemotaxis pathway.

Aim 3: Adjusting the dynamic range of the chemotaxis system to enable detection of a broad range of contaminant concentrations.

Below we describe the results in each of these aims, including a description of the successful use of our submitted parts BBa_K2736101 and BBa_K2736105.
</P></DIV><DIV class="customelementM5B" id="Aim1"><H3>Explore the possibility of using the ligand binding domains from different transmembrane receptors.
</H3><H4>Approach</H4><P>In its final form DeTaXion will use customized receptors that can detect different compounds. These receptors will be designed from a variety of existing transmembrane receptors (from the chemotaxis and other pathways), and further engineered with the help of prediction software such as Haddock, to provide the best possible binding affinity between receptor and ligand.</P><P>To provide a proof of concept within the scope of this project, we tested the possibility of swapping ligand binding domains.</P><FIGURE style="width : 40%;float: right;"><FIGCAPTION>Figure 1: Schematic representation of the Tar receptor, CusS receptor, and our Hybrid receptor. Both Tar and CusR receptors contain a HAMP domain, involved in signal transduction to a kinase control domain. Based on the homology between the HAMP domains, we designed a hybrid receptor based on the extracellular domain of the Tar receptor and intracellular domains of the CusS receptor. </FIGCAPTION></FIGURE><H4>Results</H4><P>We created a hybrid receptor consisting of the intracellular domains of the bacterial copper sensing protein CusS, fused to the ligand binding domain (LBD) of the Tar receptor (Figure 1). The Tar receptor is one of the main receptors of the chemotaxis pathway, and senses Aspartate (Bi et al., 2013). CusS is part of a two-component signaling pathway and senses copper and silver ions. In the presence of copper, CusS phosphorylates the response regulator CusR, which in turn interacts with a copper responsive promoter element, activating gene expression (Munson, Lam, Outten, &amp; O’Halloran, 2000). We chose this configuration because it would allow us to test the possibility of generating hybrid receptors using a simple transcriptional readout, without having to be dependent on Aim 2 (generation of a functional BRET pair).</P><P>To test the functionality of our receptor, we generated a new BioBrick (BBa_K2736105) consisting of a copper sensitive promoter followed by the coding sequence of the red fluorescent protein (RFP, from BBa_K516032) (Figure 2). By measuring the intensity of RFP emission in a photospectrometer or by fluorescence microscopy, we can determine both whether the receptor functions, as well as to what degree it transduces a signal.</P><P>We transformed <I>E. coli</I> strain DH5α with plasmids encoding the RFP reporter, as well as our hybrid receptor. As a control, we transformed <I>E. coli</I> with the reporter alone. We grew both strains overnight in LB, which contains Aspartate and hence should result in RFP expression when our hybrid receptor is present. After overnight culture, we assessed the presence or absence of RFP by fluorescence microscopy. We observed bright RFP expression in bacteria transformed with both receptor and reporter encoding plasmids, but only very low levels of RFP in bacteria transformed with the reporter construct alone (Figure 3).</P><FIGURE style="width : 40%;float: right;margin-bottom: 3%;"><FIGCAPTION>Figure 2:  Reporter construct consisting of RFP under control of the CopA copper-sensitive promoter (BioBrick BBa_K2736105).</FIGCAPTION></FIGURE><P>To further test the responsiveness of the Hybrid receptor to Aspartate, we attempted to grow our <I>E. coli</I> strain in a minimal medium lacking Aspartate (M9 + 0.4% glucose), which should not induce expression of RFP. However, we were unsuccessful at culturing bacteria in this medium. As an alternative, we grew bacteria overnight in LB (causing RFP expression), and then transferred them to M9 + 0.4% glucose for 8 hours to allow part of the RFP to degrade. We then split the bacteria over two tubes, and added 250 uM Aspartate to one of these samples. After 4 hours, we quantified the levels of RFP expression using a spectrophotometer. Even after 8 hours in M9, levels of RFP were still very high. Nevertheless, we observed slightly elevated levels of RFP in the bacterial culture incubated with Aspartate, compared to untreated control (Figure 4). While this indicates that our receptor is responsive to Aspartate, this result will need to be strengthened in future experiments.</P><FIGURE style="width : 40%;float: right;"><FIGCAPTION>Figure 3: Hybrid Tar/CusS receptor induces expression of RFP. (A) Expression of  RFP in bacteria expressing Hybrid receptor and carrying reporter construct. (B) Expression of RFP in bacteria carrying the RFP reporter alone. Scale bars are 10 um.</FIGCAPTION><FIGCAPTION>Figure 4: Emission spectrum of bacteria expressing Hybrid receptor (BBa_K2736109) and RFP reporter (BBa_K2736105). Red line corresponds to bacteria treated with Aspartate, and blue line corresponds to untreated bacteria. Samples were illuminated at 585 nm. RFP peak emission is at 607 nm.</FIGCAPTION></FIGURE><H4> Conclusion </H4><P>The expression of RFP in bacteria transformed with hybrid receptor and RFP reporter, but not in bacteria transformed with reporter alone, demonstrates the functionality of our copper-sensitive RFP reporter (submitted part BBa_K2736105). Is also shows that its expression is dependent on the presence of the hybrid Tar/CusS receptor. Presumably the expression is due to the presence of Aspartate in LB medium. The slight increase in RFP expression upon addition of Aspartate after incubation in M9 medium (lacking Aspartate) for 8 hours further indicates that our receptor is indeed responsive to Aspartate. Together, these results demonstrate the feasibility of using alternative ligand binding domains to broaden the range of targets detectable by DeTaXion. Key future experiments include further investigation of the Aspartate responsiveness of the hybrid receptor, and the generation of receptors carrying ligand binding domains engineered to bind pharmaceutical compounds.</P></DIV><DIV class="customelementM5A" id="Aim2"><H3>Aim 2: Development of a Bioluminescence Resonance Energy Transfer (BRET) readout for the activity of the chemotaxis pathway.</H3><H4>Approach</H4><P>To detect the activity of the chemotaxis pathway, we designed a Bioluminescence Resonance Energy Transfer (BRET)-based sensor, inspired by a previously made chemotaxis BRET-pair (Cui et al. 2014). We fused Renilla luciferase (RLuc) C-terminally to CheZ, and eYFP to CheY N-terminally. The final form of DeTaXion will give a quantifiable BRET signal, which will be measured as the change in the ratio of RLuc over eYFP luminescence (R/Y). In our experiments we defined a positive BRET signal as a change in R/Y of at least 0.5% because this has been shown to represent a detectable BRET signal (Cui et al., 2014; Shimizu, Delalez, Pichler, &amp; Berg, 2006). </P><H4>Results</H4><FIGURE style="width : 40%;float: right;"><FIGCAPTION>Figure 5: Expression of eYFP::CheY. (A) <I>E. coli</I> DH5 transformed with eYFP::CheY and  (BBa_K2736101) and CheZ::RLuc (BBa_K2736103). (B) <I>E. coli</I> UU1250 transformed with eYFP::CheY and  (BBa_K2736101) and CheZ::RLuc (BBa_K2736103). Scale bars are 10 um.</FIGCAPTION></FIGURE><P>The coding sequences of the BRET pair proteins were created by Gibson assembly of the parts BBa_K569017 and BBa_I15017, and BBa_K629003 and BBa_I159001 to create eYFP::CheY (part BBa_K2736100), and CheZ::RLuc (part BBa_K2736102) respectively. Subsequently, 3A assembly was used to add the promoter part BBa_K608003 to these coding sequences (parts BBa_K2736101 and BBa_K2736103 respectively). The resulting vectors were transformed to either <I>E. coli</I> DH5α, which expresses wild type Tar, or <I>E. coli</I> strain UU1250, which lacks the Tar gene.</P><P>To confirm expression of eYFP::CheY we imaged bacteria by fluorescence microscopy. Expression of YFP was clearly detectable, confirming the presence of eYFP::CheY (Figure 5).</P><P>We next incubated the bacterial cells in PBS containing 7.5 µM coelenterazine (CAS# 55779-48-1 Nanolight #3031), the substrate for Renilla luciferase. However, we did not observe any emission of light, neither by immunofluorescence microscopy nor using a photospectrometer. We hypothesized that this result was due to the low permeability of coelenterazine through the cell wall. However, due to time constraints we were unable to further investigate this.  </P><H4>Discussion</H4><P>Our results demonstrate the expression of eYFP::CheY (BBa_K2736101). However, we were not able to measure BRET activity due to lack of bioluminescence from Renilla luciferase. We suspect that this is due to an inability of the luciferase substrate coelenterazine to penetrate into <I>E. coli</I>, as coelenterazine is known to have low penetration of bacterial cells. Solving this issue is the key next experiment to be performed, followed by detection of BRET signal in the presence and absence of the ligand Aspartate.</P></DIV><DIV class="customelementM5B" id="Aim3"><H3>Aim 3: Adjusting the dynamic range of the chemotaxis system to enable detection of a broad range of contaminant concentrations.</H3><H4>Approach</H4><P>To facilitate the measurement of multiple ranges of concentrations, we adjusted the sensitivity of the receptor. This approach is based on the methylation state of the Tar receptor, which influences its sensitivity to ligands. In the unmethylated state, the receptor efficiently transduces the ligand binding signal to the downstream pathway (<A href="https://2018.igem.org/Team:Utrecht/Design">see project design</A>). Upon receptor methylation, the conformational change required to transduce the signal, is energetically less favorable. Therefore, when the same amount of ligand is present, the level of signal transduction is lower than in the unmethylated receptor state. The Tar receptor has four acidic residues that can be methylated (Q295, E302, Q309, E491). The methylation of glutamine (Q) or glutamic acid (E) residues can be mimicked by mutating these residues to alanine (A) (Krembel et al. 2015). </P><H4>Results and discussion</H4><P>Our <A href="https://2018.igem.org/Team:Utrecht/Model">computational modeling</A> indicated that the mutation of the methylation sites should indeed result in altered sensitivity of the receptor. We therefore chose to generate three different methylation state Tar variants:</P><OL><LI>The wild type state, the unmethylated Tar receptor (QEQE, most sensitive);</LI><LI>A Tar receptor with one mimicked methylation site (QEAE, average sensitivity);</LI><LI>A Tar receptor with two mimicked methylation sites (QEAA, least sensitive).</LI></OL><P>We received the wild-type Tar receptor from a <A href="https://2018.igem.org/Team:Utrecht/Collaborations#Tar">collaboration with iGEM team Groningen</A>. We next used site-directed mutagenesis to introduce the Q309A and E491A mutations, resulting in BioBrick parts BBa_K2736108 and BBa_K2736106 respectively. Sequencing confirmed the successful generation of these components. However, because the BRET assay was not yet functional, we were unable to test the relative activity of these Tar receptors in the time frame of the competition.</P></DIV><DIV class="customelementM5A"><DIV class="references"><H4> References </H4>
Bi, S., Yu, D., Si, G., Luo, C., Li, T., Ouyang, Q., … Lai, L. (2013). Discovery of novel chemoeffectors and rational design of <I>Escherichia coli</I> chemoreceptor specificity. Proceedings of the National Academy of Sciences. <A href="https://doi.org/10.1073/pnas.1306811110">https://doi.org/10.1073/pnas.1306811110</A>

Cui, B., Wang, Y., Song, Y., Wang, T., Li, C., Wei, Y., … Shen, X. (2014). Bioluminescence resonance energy transfer system for measuring dynamic protein-protein interactions in bacteria. mBio. <A href="https://doi.org/10.1128/mBio.01050-14">https://doi.org/10.1128/mBio.01050-14</A>

Krembel A, Colin R, Sourjik V (2015) Importance of Multiple Methylation Sites in <I>Escherichia coli</I> Chemotaxis. PLoS ONE 10(12): e0145582.

Munson, G. P., Lam, D. L., Outten, F. W., &amp; O’Halloran, T. V. (2000). Identification of a copper-responsive two-component system on the chromosome of <I>Escherichia coli</I> K-12. Journal of Bacteriology. <A href="https://doi.org/10.1128/JB.182.20.5864-5871.2000">https://doi.org/10.1128/JB.182.20.5864-5871.2000</A>
 
-Shimizu, T. S., Delalez, N., Pichler, K., &amp; Berg, H. C. (2006). Monitoring bacterial chemotaxis by using bioluminescence resonance energy transfer: Absence of feedback from the flagellar motors. Proceedings of the National Academy of Sciences. <A href="https://doi.org/10.1073/pnas.0510958103">https://doi.org/10.1073/pnas.0510958103</A></DIV></DIV><DIV class="navigateArrows"><DIV class="arrowLeft"><A href="https://2018.igem.org/Team:Utrecht/Parts"> Biobricks</A></DIV><DIV class="arrowLeft"><A href="https://2018.igem.org/Team:Utrecht/Design"> Project Design</A></DIV><DIV class="arrowRight"><A href="https://2018.igem.org/Team:Utrecht/Attributions">Attributions </A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>