<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Utrecht_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Utrecht/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><BUTTON onclick="topFunction()" id="BackToTop" title="Go to top">Top</BUTTON><DIV class="anotherWrapper"><DIV class="tab"><BUTTON id="tabR" onclick="changeContent(2)">Receptor Assay</BUTTON><BUTTON id="tabB" onclick="changeContent(3)">BRET Assay</BUTTON><BUTTON id="tabM" onclick="changeContent(4)">Methylation Assay</BUTTON></DIV></DIV><DIV class="calenderContainer"><DIV id="monthHeaderR" class="monthHeader"><H1> Receptor Assay </H1><DIV class="month" id="monthRA"><DIV onclick="createMonth(30, 5, inactiveSeptember, 'S');" class="next">❯</DIV><DIV>August</DIV></DIV><DIV class="month" id="monthRS"><DIV onclick="createMonth(31, 2, inactiveAugust,'A');" class="prev">❮</DIV><DIV onclick="createMonth(31, 0, inactiveOctober, 'O');" class="next">❯</DIV><DIV>September</DIV></DIV><DIV class="month" id="monthRO"><DIV onclick="createMonth(30, 5, inactiveSeptember,'S');" class="prev">❮</DIV><DIV>October</DIV></DIV></DIV><DIV id="monthHeaderB" class="monthHeader"><H1> BRET Assay </H1><DIV class="month" id="monthBJ"><DIV onclick="createMonth(31, 2, inactiveAugust,'A');" class="next">❯</DIV><DIV>July</DIV></DIV><DIV class="month" id="monthBA"><DIV onclick="createMonth(31, 6, inactiveJuly, 'J');" class="prev">❮</DIV><DIV onclick="createMonth(30, 5, inactiveSeptember, 'S');" class="next">❯</DIV><DIV>August</DIV></DIV><DIV class="month" id="monthBS"><DIV onclick="createMonth(31, 2, inactiveAugust, 'A');" class="prev">❮</DIV><DIV onclick="createMonth(31, 0, inactiveOctober, 'O');" class="next">❯</DIV><DIV>September</DIV></DIV><DIV class="month" id="monthBO"><DIV onclick="createMonth(30, 5, inactiveSeptember,'S');" class="prev">❮</DIV><DIV>October</DIV></DIV></DIV><DIV id="monthHeaderM" class="monthHeader"><H1> Methylation Assay </H1><DIV class="month" id="monthMJ"><DIV onclick="createMonth(31, 2, inactiveAugust, 'A');" class="next">❯</DIV><DIV>July</DIV></DIV><DIV class="month" id="monthMA"><DIV onclick="createMonth(31, 6, inactiveJuly, 'J');" class="prev">❮</DIV><DIV onclick="createMonth(30, 5, inactiveSeptember, 'S');" class="next">❯</DIV><DIV>August</DIV></DIV><DIV class="month" id="monthMS"><DIV onclick="createMonth(31, 2, inactiveAugust, 'A');" class="prev">❮</DIV><DIV onclick="createMonth(31, 0, inactiveOctober, 'O');" class="next">❯</DIV><DIV>September</DIV></DIV><DIV class="month" id="monthMO"><DIV onclick="createMonth(30, 5, inactiveSeptember,'S');" class="prev">❮</DIV><DIV>October</DIV></DIV></DIV><DIV class="weekdays"><DIV class="weekday">Mo</DIV><DIV class="weekday">Tu</DIV><DIV class="weekday">We</DIV><DIV class="weekday">Th</DIV><DIV class="weekday">Fr</DIV><DIV class="weekday">Sa</DIV><DIV class="weekday">Su</DIV></DIV></DIV></DIV><DIV class="notebookContainer"><DIV class="customelementM5B" id="entrydayIA8"><H3> Morning </H3><P>Performed Calibration 1, 2, and 3 according to protocol. All measurements were performed using the plate reader of Seino. </P><TABLE class="InterlabTable" style="width: 20%"><CAPTION style="font-size: 1.5em;background-color: #A4C2F4">Table 1. content 96 wells plate cal.1 and cal.2
</CAPTION><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TD>A</TD><TD>L</TD><TD>dd</TD></TR><TR><TD>B</TD><TD>L</TD><TD>dd</TD></TR><TR><TD>C</TD><TD>L</TD><TD>dd</TD></TR><TR><TD>D</TD><TD>L</TD><TD>dd</TD></TR><TR><TD>E</TD><TD style="color: green">	MS</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">	dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD>dd</TD></TR><TR><TD>F</TD><TD style="color: green">MS</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">	dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD>dd</TD></TR><TR><TD>G</TD><TD style="color: green">	MS</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD>dd</TD></TR><TR><TD>H</TD><TD style="color: green">	MS</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD style="color: green">dd</TD><TD>dd</TD></TR></TBODY></TABLE><H5> Calibration 1 </H5><UL><LI>L= 100 uL Ludox CL-X (stored at 4C)</LI><LI>dd= 100 uL ddH20</LI><LI>Measurement: Abs600, turn off pathlength correction</LI></UL><H5>Calibration 2 </H5><UL><LI>MS= 200 ul Microsphere Stock Solution</LI><LI>dd= 100 uL ddH20</LI><LI>green= serial dilution was performed with a micropipet from E1,F1,G1,H1 - E11,F11,G11,H11 by a volume of 100uL.             Before every transfer solution was pipetted up and down 3x, after every transfer tips were discharged.</LI><LI>Measurement: Abs600, re-mix befor putting in plate reader and prevent bubbles, path length correction off</LI></UL><H5>Calibration 3</H5><UL><LI>1xFC= 200 mL 1xFC (100uL 10x fluorescein + 900ul 1x PBS pH 7.4, tube was covered with foil</LI><LI>P= 100 uL 1x PBS pH 7.4</LI><LI>green= serial dilution was performed with a micropipet from A1,B1,C1,D1 - A11,B11,C11,D11 by a volume of 100uL. Before every transfer solution was pipetted up and down 3x, after every transfer tips were discharged. </LI><LI>Measurement: FL, 530nm/30nm bandpass, 25-30nm with recommened excitation of 485nm, emission 520-530nm of the filter. Path length correction was turned off</LI></UL><TABLE class="InterlabTable" style="width: 20%"><CAPTION style="font-size: 1.5em;background-color: #A4C2F4">Table 2. content 96 wells plate cal.3</CAPTION><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR><TR><TD>A</TD><TD>1xFC</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD></TR><TR><TD>B</TD><TD>1xFC</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD></TR><TR><TD>C</TD><TD>1xFC</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD></TR><TR><TD>D</TD><TD>1xFC</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD><TD>P</TD></TR><TR><TD>E</TD></TR><TR><TD>F</TD></TR><TR><TD>G</TD></TR><TR><TD>H</TD></TR></TBODY></TABLE><H3> Afternoon </H3><H5>LBC plates were made according to the protocol used on the wall</H5><UL><LI>250ml LB 2x added to melted 250 ml WA 2x using a microwave </LI><LI>0.5ml was added to final solution</LI><LI>plates were dried in 37C incubator</LI></UL><H5>Transformation device 3 + negative control interlab study</H5><UL><LI>Device 3 (number 5) showed a low GFP expression, so it was tried to re-preform the tranformation. Negative control of the interlab (number 1) was not performed last time due to lack of LBC plates so was also performed. </LI><LI><A href="https://2018.igem.org/Team:Utrecht/Protocol#Transformation">Protocol Transformation</A></LI></UL></DIV><DIV class="customelementM5B" id="entrydayIJ2"> 
lskdfjslkjgdlkdfjgldkjlakmflsvmclkxmfklmgdmlkcvmblcmkb
</DIV></DIV><DIV class="customelementM5B" id="entrydayRA21"><H2>21/08/18</H2><B>Lorenzo</B><P>1/9M (RFP) was transformed from the iGEM kit according to the protocol. The coding sequence of the custom receptor, and copper promoter flanked by biobrick sites were ordered from IDT and used directly. </P></DIV><DIV class="customelementM5B" id="entrydayRA22"><H2>22/08/18</H2><B>Lorenzo</B><P>Transformation was successful and a colony was picked and inoculated in chloramphenicol. </P></DIV><DIV class="customelementM5B" id="entrydayRA23"><H2>23/08/18</H2><B>Felix, Lorenzo</B><P>Miniprep was performed according to the <A href="https://2018.igem.org/Team:Utrecht/Protocol#PlasmidPurification">protocol.</A>
Cu-promotor         → 250 ng/μl
Custom receptor    → 360 ng/μl
RFP            → 91.7 ng/μl</P><P>Cu-promoter and RFP were placed in pSB1K3, custom receptor was placed in pSB1K3 as well. Cu-promoter DNA was digested with EcoRI and SpeI, RFP DNA was digested with XbaI and PstI. The custom receptor and the target-vector were digested with EcoRI and PstI. 
</P><P>For each of the three samples a mastermix was made. For each digestion the DNA concentration was reduced to 25 ng/ul. 4 ul of DNA sample and target vector sample were transferred to a PCR tube with 4 ul of the according restriction mix. The samples were incubated for 1 hour at 37 C. After 1 hour the samples were taken out of the incubator and enzymes were heat deactivated at 80°C for 20 min. 1ul of target vector (pSB1K3), 2 ul of Cu-promotor restricted DNA, and 2 ul restricted RFP DNA were added to one PCR tube. Furthermore 1ul DNA T4 ligase buffer and 0.5 ul of T4 ligase were added. The total volume was made up to 10 ul with 3.5 ul dH2O. The samples were incubated for one hour (RT). Enzymes were deactivated with heat kill 80°C 20 min. </P><P>DNA was transformed in DH5α according to the iGEM protocol.</P></DIV><DIV class="customelementM5B" id="entrydayRA24"><H2>24/08/18</H2><B>Lorenzo</B><P>The transformation was successful. 8 colonies were picked to perform colony PCR according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#ColonyPCR">protocol</A>. (Cu-promotor::RFP 100%, Cu-promotor::RFP 500%, Custom receptor 100%, Custom Receptor 500%). </P><P>Note: The date on the plates is date of preparation not transformation</P></DIV><DIV class="customelementM5B" id="entrydayRA27"><H2>27/08/18</H2><B>Felix</B><P>Samples from colony PCR  (<A onclick="calenderlink('RA24')">24/08/18</A>) were analysed with a 0.8% Agar gel electrophoresis.
Custom receptor = 1926 bp 
Cu-promoter::RFP = 1238 bp</P><TABLE style="float: left"><TBODY><TR><TD>	1-8 			</TD><TD>	Custom receptor 100%			</TD></TR><TR><TD>	9-16			</TD><TD>	Custom receptor 500%			</TD></TR><TR><TD>	17-24			</TD><TD>	Cu::RFP 100%			</TD></TR><TR><TD>	25-32			</TD><TD>	Cu::RFP 500% 			</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayRA28"><H2>28/08/18</H2><B>Lorenzo, Felix</B><P>The results obtained in the gel electrophoresis DNA analysis (<A onclick="calenderlink('RA27')">27/8/18</A>) was not satisfactory. Therefore four new colonies were picked for colony PCR in accordance with the <A href="https://2018.igem.org/Team:Utrecht/Protocols#ColonyPCR">protocol</A>.</P><TABLE><TBODY><TR><TD> 1-4 </TD><TD>	Cu::RFP 100% </TD></TR><TR><TD> 5-8 </TD><TD>	Cu::RFP 500% </TD></TR><TR><TD> 9-12 </TD><TD>	Custom receptor 100%  </TD></TR><TR><TD> 13-16 </TD><TD> Custom receptor 500% </TD></TR></TBODY></TABLE><P>The PCR samples were analysed with gel electrophoresis using 0.8% agarose gel. 
Custom receptor = 1926 bp 
Cu-promoter::RFP = 1238 bp </P><P>Samples 4, 9, 12, 13, 14 were selected and inoculated in 5 ml LB containing kanamycin overnight. A safe of these samples was made as well. </P><P>As the Cu::RFP sample did not yield the desired results, new cloning procedure was initiated using the iGEM protocol and buffer 2.1 instead of enzyme specific buffers. In order to ensure that we would have enough genetic material, we cloned the gBlock `copper-promoter blunt into the vector pBSK.</P></DIV><DIV class="customelementM5B" id="entrydayRA29"><H2>29/08/18</H2><B>Felix</B><P>The DNA of the (<A onclick="calenderlink('RA28')">28/08/18</A>) samples was purified according to <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification"> the miniprep protocol</A>. </P><TABLE><TBODY><TR><TH>	Sample			</TH><TH>	Concentration ng/µl	</TH><TH>	Description	</TH></TR><TR><TD>	1 - 8			</TD><TD>	69.9	</TD><TD>	Cu-Promotor+ RFP	</TD></TR><TR><TD>	9			</TD><TD>	77.5	</TD><TD>	Custom Receptor	</TD></TR><TR><TD>	12			</TD><TD>	91.8	</TD><TD>	Custom Receptor	</TD></TR><TR><TD>	13			</TD><TD>	49.4	</TD><TD>	Custom Receptor	</TD></TR><TR><TD>	14			</TD><TD>	85.4	</TD><TD>	Custom Receptor	</TD></TR></TBODY></TABLE><P>The prior prepared clonation (<A onclick="calenderlink('RA28')">28/08/18</A>) was transformed in DH5α according to the iGEM protocol.</P><P>Safe made from samples Samples 4, 9, 12, 13, 14 (<A onclick="calenderlink('RA18')">08/28/18</A>)</P></DIV><DIV class="customelementM5B" id="entrydayRA30"><H2>30/08/18</H2><B>Felix</B><P>No colonies of the transformation(<A onclick="calenderlink('RA29')">29/8/18</A>) were observed.

The purified DNA of samples 4,12,14 (<A onclick="calenderlink('RA29')">29/8/18</A>) were send for sequencing with forward and reverse primer.
 </P></DIV><DIV class="customelementM5B" id="entrydayRS5"><H2>05/09/18</H2><B>Lorenzo</B><P>Samples from sequencing were analysed. Custom receptor (12) was placed in the pSB1K3 backbone correctly. The sequence of Cu-promoter::RFP was difficult to interpret. Therefore Cu-promoter::RFP (4) was incubated again in 5 ml LB + Kanamycin from the safe overnight.</P></DIV><DIV class="customelementM5B" id="entrydayRS6"><H2>06/09/18</H2><B>Jolijn</B><P>The Cu::RFP DNA was purified according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">miniprep protocol</A> with 5 ml bacteria instead of 1.5 ml and eluted in 20 µl elution buffer. DNA was sent for sequencing. </P><TABLE><TBODY><TR><TH>	Cu + RFP (4) Code:			</TH><TH>	Primer			</TH></TR><TR><TD>	1BA9ZAB414			</TD><TD>	VF			</TD></TR><TR><TD>	1BA9ZAB413			</TD><TD>	VR			</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayRS10"><H2>10/09/18</H2><B>Lorenzo</B><P>The sequence of Cu::RFP did not contain the copper promotor, only RFP. </P></DIV><DIV class="customelementM5B" id="entrydayRS11"><H2>11/09/18</H2><B>Lorenzo</B><P>The RFP DNA was cloned into the Copper-promoter vector by cutting the copper-promoter vector with SpeI and PstI, and the RFP vector with XbaI and PstI overnight at 37 °C.</P></DIV><DIV class="customelementM5B" id="entrydayRS12"><H2>12/09/18</H2><B>Felix</B><P>The prior cut samples were ligated. 2.4 µl RFP DNA was used as insert with 1 µl of Cu promotor backbone. After heat inactivation the sample was transformed according to the iGEM protocol in DH5α and plated on Ampicillin plates.      </P></DIV><DIV class="customelementM5B" id="entrydayRS13"><H2>13/09/18</H2><B>Lorenzo</B><P>Transformation was successful. However, close examination of the samples under the fluorescence binoculars revealed that none of the colonies expressed RFP.</P><P>2 colonies were picked and inoculated in 5 ml LB + ampicillin. </P></DIV><DIV class="customelementM5B" id="entrydayRS22"><H2>22/09/18</H2><B>Lorenzo</B><P>Since our previous attempts didn’t yield any results we decided to place the copper promoter gBlock with the restriction enzymes EcoRI and SpeI. Simultaneously, the RFP vector (PSB1C3) was cut with EcoRI and XbaI. Both reactions were done overnight  in NEB Buffer 2.1. </P></DIV><DIV class="customelementM5B" id="entrydayRS23"><H2>23/09/18</H2><B>Lorenzo</B><P>The digestion product from <A onclick="calenderlink('RS22')">09/22/18</A> was ligated according to the iGEM ligation protocol. 2 µl of ligation product was transformed to <I>E. coli</I> DH5α.</P></DIV><DIV class="customelementM5B" id="entrydayRS24"><H2>24/09/18</H2><B>Lorenzo</B><P>The reaction mix was inactivated by heating it to 80°C for 20 minutes. A ligation was set up with the fragments, and 9 µl of the ligation product was transformed to 100 µl of competent DH5α.</P></DIV><DIV class="customelementM5B" id="entrydayRS25"><H2>25/09/18</H2><B>Felix</B><P>The transformation was successful. 6 colonies could be spotted. These were inoculated overnight in 5 ml LB with chloramphenicol.</P></DIV><DIV class="customelementM5B" id="entrydayRS26"><H2>26/09/18</H2><B>Lorenzo, Felix</B><P>The DNA of the inoculated cultures was minipreped according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">miniprep protocol</A>. After the first spin down we noticed that the bacterial pellet of one of the colonie number 6 was slightly red. Because it is known that RFP expressing bacteria turn red even without excitation, we concluded that it was most likely that this colony expressed RFP, and thus that the transformation worked. After the isolation of the DNA, a colony PCR was done with the VF and VR primer. The total size of the band that we expected was 824 + 156 -6 = 974 bp for the fragment + 271 bp for the part outside the BB_sites generated by the VF and VR primer. The total size of the fragment should thus be 1245 bp. The fragment in lane 6 lies between the 1000 bp band and the 1500 bp band and starts slightly lower than the middle, indicating that it is most likely a band of correct size.</P><P>The DNA of this colony was send for sequencing.</P></DIV><DIV class="customelementM5B" id="entrydayRS29"><H2>29/09/18</H2><B>Lorenzo</B><P>The sequencing results showed that the construct was correct. Since both the promoter::RFP fusion, and the Custom receptor were now available to us on plasmid backbones with different resistance genes, both constructs were transformed to <I> E. coli </I> DH5α.</P></DIV><DIV class="customelementM5B" id="entrydayRS30"><H2>30/09/18</H2><B>Lorenzo, Felix</B><P>The transformation was successful.
A stock solution of 50 mM CuSO<SUB>4</SUB> was made for experiments with the newly transformed copper sensitive bacteria. 2 colonies were inoculated overnight for further experiments.</P></DIV><DIV class="customelementM5B" id="entrydayRO1"><H2>01/10/18</H2><B>Lorenzo</B><P>100 µl of the overnight cultures from <A onclick="calenderlink('RS30')">30/09/18</A> was transferred to two tubes containing 5 ml of clean LB. One tube contained 500 µM CuSO<SUB>4</SUB>. Both tubes were inoculated for 45 minutes at 37°C.  
400 µl of this culture was washed in 1 ml of PBS and RFP expression was measured with an emission scan between 570 and 640 nm in a Carry Eclipse Fluorescence Photospectrometer using 400 µl of the copper bacterium containing PBS solution. The excitation wavelength used was 555 nm.  No fluorescence could be measured, which was probably due to the low amount of bacteria.</P></DIV><DIV class="customelementM5B" id="entrydayRO2"><H2>02/10/18</H2><B>Jolijn</B><P>A new overnight inoculation of the double transformed bacteria was made.</P></DIV><DIV class="customelementM5B" id="entrydayRO3"><H2>03/10/18</H2><B>Felix, Lorenzo</B><P>The experiment of <A onclick="calenderlink('RO1')">10/01/18</A> was repeated, but this time CuSO<SUB>4</SUB> was added to the inoculation at a final concentration of 500 µM, 250 µM, 50 µM, and 5 µM, or 0 µM. The Bacteria were inoculated for 2 hrs at 37°C. </P></DIV><DIV class="customelementM5B" id="entrydayRO12"><H2>12/10/18</H2><B>Felix</B><P>The following bacteria (DH5α) were inoculated in 5 ml minimal medium, and  5 ml LB with matching antibiotic.</P><P>Cu-promotor::RFP // Custom receptor     → Ampicillin, Chloramphenicol
Cu-promotor::RFP                    → Chloramphenicol  </P></DIV><DIV class="customelementM5B" id="entrydayRO13"><H2>13/10/18</H2><B>Jolijn, Felix, Lorenzo</B><P>Bacteria in minimal medium did not grow.  Cu-promoter::RFP // Custom receptor samples were red coloured while bacteria not containing the custom receptor were yellowish. Fluorescence images were taken to confirm RFP expression. The bacteria containing the Cu-promoter::RFP // Custom receptor were washed 1x in M9 and incubated again overnight in M9 containing 0.4% glucose. </P><P>In order to confirm aspartic acid induced RFP expression the experiment was repeated. The following bacteria (DH5α) were inoculated in 5 ml M9 + 0.4% glucose, and in 5 ml LB with matching antibiotic. </P><P>Cu-promotor::RFP // Custom receptor     → Ampicillin, Chloramphenicol
Cu-promotor::RFP                    → Chloramphenicol  </P><P>Cu-promoter :: RFP Normarski</P><P>Cu-promoter :: RFP Nomarski Fluorescence</P><P>Cu-promoter :: RFP // Custom receptor </P><P>Cu-promoter::RFP // Custom receptor fluorescence (falsely coloured) </P></DIV><DIV class="customelementM5B" id="entrydayRO14"><H2>14/10/18</H2><B>Felix, Jolijn</B><P>Bacteria in M9 did not grow. Bacteria in LB did grow but did not express GFP observed with the naked eye. 4 ml bacteria of each of the two samples was washed with 1x with M9 and incubated with M9 + 0.4% glucose. </P><P>After 4 hours of  incubation in M9 + 0.4% glucose the samples were split up in the following fractions. After 4 hours 500 μl was used for fluorescence measurements using the prior described fluorescence photospectrometer (ex. 585 nm).  After 7 hours the remaining 500 μl was used for measurements. </P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>Asp (10 μ 50 mM)</TH><TH>	Cu (2 μl 25 mM)	</TH></TR><TR><TD>	Custom Receptor	</TD><TD>	1	</TD><TD>	+	</TD><TD>	-			</TD></TR><TR><TD>	Custom Receptor	</TD><TD>	2	</TD><TD>	-	</TD><TD>	-			</TD></TR><TR><TD>	Cu-promoter :: RFP	</TD><TD>	3	</TD><TD>	+	</TD><TD>	-			</TD></TR><TR><TD>	Cu-Promoter :: RFP	</TD><TD>	4	</TD><TD>	-	</TD><TD>	+			</TD></TR><TR><TD>	Cu-Promoter :: RFP	</TD><TD>	5	</TD><TD>	-	</TD><TD>	-			</TD></TR></TBODY></TABLE><P>Sample 1 is not representative at 8 hours since the remaining volume of bacteria was less than the volume in the cuvette. Therefore this sample was diluted which resulted in a increased intensity as the cuvette was now fully filled, however, the measurement did not reach its full potential. </P><P>The Cu-promoter::RFP // Custom receptor sample washed (<A onclick="calenderlink('RO13')">13/10/18</A>) and incubated overnight was split up in two 4 ml fractions. To one fraction 40 μl  50 mM Aspartic acid was added and incubated for 4 hours.  After 4 hours 500 μl was used for fluorescence measurements, this was repeated at 8 and 11 hours</P><P>Although these data are not conclusive it indicates that the custom receptor has a reaction on the addition of aspartate. Therefore new samples were inoculated in 5 ml LB.</P><TABLE><TBODY><TR><TD>	Custom // Cu-promoter :: RFP	</TD><TD>	3x 							</TD></TR><TR><TD>	Custom // Cu-promoter :: RFP Asp+ 500  μM	</TD><TD>	1x							</TD></TR><TR><TD>	Cu-promoter :: RFP 	</TD><TD>	3x							</TD></TR><TR><TD>	Cu-promoter :: RFP  Asp+ 500 μM	</TD><TD>	1x							</TD></TR><TR><TD>	Cu-promoter :: RFP 50 μM CuSO<SUB>4</SUB></TD><TD>	1x							</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayRO15"><H2>15/10/18</H2><B>Felix, Jolijn</B><P>The three custom receptor samples and one co-promoter::RFP sample did not fully grow after 9 hours of inoculation, these samples were put back into the incubator for 15 hours and can be seen in the table underneath. The other samples were incubated for 4 hours in M9. </P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Volume	</TH><TH>	OD600	</TH></TR><TR><TD>	1. Co-promotor + RFP 	</TD><TD>	4 mL	</TD><TD>	1.54	</TD></TR><TR><TD>	2. Co-promotor + RFP 	</TD><TD>	4 mL	</TD><TD>	1.35	</TD></TR><TR><TD>	3. Co-promotor + RFP	</TD><TD>	3 mL	</TD><TD>	2.70	</TD></TR><TR><TD>	4. Promotor + RFP + Aspartate	</TD><TD>	3 mL	</TD><TD>	0.47	</TD></TR></TBODY></TABLE><P>Copper was added to the samples incubated in M9 in concentrations of 25 and 50 µM. RFP expression was measured at t = 0, t = 2. However, results from the scan were inconclusive.</P><P>The three custom receptor samples and one cu-promoter::RFP sample incubated in total for 24 hours. These samples were used for new inoculations as described below in increased volumes. The remaining volume of the inoculations were pelleted and washed once with Milli Q and resuspended in M9, in total, all samples were resuspended in 5 ml M9 (Orange in the table). During the pelleting the pellets of samples containing custom receptor turned red. 
</P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Medium	</TH><TH>	Volume	</TH></TR><TR><TD>	Custom receptor // Cu-promoter::RFP	</TD><TD>	LB + ampicillin and chloramphenicol 	</TD><TD>	100 ml	</TD></TR><TR><TD>	Cu-promoter::RFP 	</TD><TD>	LB + chloramphenicol	</TD><TD>	100 ml	</TD></TR><TR><TD>	Custom receptor // Cu-promoter::RFP	</TD><TD>	SV + 0.4% glucose + ampicillin and chloramphenicol	</TD><TD>	100 ml	</TD></TR><TR><TD>	Cu-promoter::RFP	</TD><TD>	SV + 0.4% glucose + chloramphenicol	</TD><TD>	100 ml	</TD></TR><TR><TD>	Cu-promoter::RFP	</TD><TD>	LB + chloramphenicol	</TD><TD>	5 ml	</TD></TR><TR><TD>	Cu-promoter::RFP	</TD><TD>	LB + 25 μM CuSO4 + chloramphenicol	</TD><TD>	5 ml	</TD></TR><TR><TD>	Cu-promoter::RFP	</TD><TD>	LB + 50 μM CuSO4 + chloramphenicol	</TD><TD>	5 ml	</TD></TR><TR><TD>	Cu-promoter::RFP	</TD><TD>	LB + 100 μM CuSO4 + chloramphenicol	</TD><TD>	5 ml	</TD></TR><TR><TD>	3x Custom receptor // Cu-promoter::RFP	</TD><TD>	M9 + 0.4% glucose	</TD><TD>	5 ml	</TD></TR><TR><TD>	1x Cu-promoter::RFP	</TD><TD>	M9 + 0.4% glucose	</TD><TD>	5 ml	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayRO16"><H2>16/10/18</H2><B>Felix, Jolijn</B><P>The samples that were incubated with copper the day before, were measured in PBS with 0.4% glucose.



Copper was added to the samples with custom receptor//Cu-promotor::RFP and Cu-promotor::RFP in concentrations of 25, 50, 100 and 250 µM. Next, RFP fluorescence was measured at t = 0h, t = 1.5h and t = 2.5h. 
</P><P>5 mL of the 25 mL overnight LB cultures of Custom receptor // Cu-promoter::RFP (OD600 of 0.675) and Cu-promoter::RFP (OD600 of 0.690) were used to attain the following solutions:</P><OL><LI>Sample with nothing added</LI><LI>Sample with 125 µM aspartate </LI><LI>Sample with 250 µM aspartate</LI><LI>Sample with 500 µM aspartate</LI><LI>Sample with 50 µM copper</LI></OL><P>RFP fluorescence was measured at t = 0, t = 4, and t = 6. There was no fluorescence. Upon measuring the fluorescence of the strain from which the samples originated (incubated in LB overnight), only slight fluorescence was visible.</P></DIV><DIV class="customelementM5B" id="entrydayBJ30"><H2>30/07/18</H2><B>Lorenzo, Floor</B><P>Isolated biobricks (listed in table below) and transformed them to <I>E. coli</I> Dh5ɑ according to the transformation protocol.</P><TABLE><TBODY><TR><TH>	Biobrick	</TH><TH>	Kit location*	</TH><TH>	Part name	</TH><TH>	Part size (bp)	</TH></TR><TR><TD>	BBa_K608003</TD><TD>	1/5A	</TD><TD>	Promoter +RBS	</TD><TD>	56	</TD></TR><TR><TD>	BBa_K569017</TD><TD>	1/8K	</TD><TD>	CheY	</TD><TD>390	</TD></TR><TR><TD>	BBa_K629003</TD><TD>	1/18G	</TD><TD>	CheZ	</TD><TD>	644	</TD></TR><TR><TD>	BBa_I759001</TD><TD>	3/3J	</TD><TD>	Rluc	</TD><TD>	936	</TD></TR><TR><TD>	BBa_B0017</TD><TD>	3/6C	</TD><TD>	Terminator	</TD><TD>	128 **	</TD></TR><TR><TD>	BBa_I15017</TD><TD>	4/21N	</TD><TD>	eYFP	</TD><TD>	717	</TD></TR></TBODY></TABLE><P>* location name is as follows: plate number/well number (e.g. 1/5A means plate 1 well 5A)
** this part contains two copies of BBa_B0010 (64 bp each) but the exact size of the fragment is unspecified on the wiki page.</P></DIV><DIV class="customelementM5B" id="entrydayBJ31"><H2>31/07/18</H2><B>Lorenzo, Floor</B><P>Transformation was successful, colonies were picked and inoculated in 5 ml LB containing the correct antibiotic.</P></DIV><DIV class="customelementM5B" id="entrydayBA1"><H2>01/08/18</H2><B>Lorenzo, Floor</B><P>Isolated the plasmids containing the biobricks from the inoculated colonies from <A onclick="calenderlink('BA1')">31/07/18</A> according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">plasmid purification protocol</A>. (for concentrations see table below)</P><TABLE><TBODY><TR><TH>	Concentration (ug/ml)		</TH></TR><TR><TH>	Biobrick	</TH><TH>	Kit location*	</TH><TH>	Part name	</TH><TH>	Replicate 1	</TH><TH>	Replicate 2	</TH></TR><TR><TD>	BBa_K608003	</TD><TD>	1/5A	</TD><TD>	Promoter +RBS	</TD><TD>	76,8	</TD><TD>	77,8	</TD></TR><TR><TD>	BBa_K569017	</TD><TD>	1/8K	</TD><TD>	CheY	</TD><TD>	51	</TD><TD>	50,5	</TD></TR><TR><TD>	BBa_K629003	</TD><TD>	1/18G	</TD><TD>	CheZ	</TD><TD>	83,1	</TD><TD>	111,9	</TD></TR><TR><TD>	BBa_I759001	</TD><TD>	3/3J	</TD><TD>	Rluc	</TD><TD>	81,2	</TD><TD>	89,7	</TD></TR><TR><TD>	BBa_B0017	</TD><TD>	3/6C	</TD><TD>	Terminator	</TD><TD>	137,8	</TD><TD>	75,4	</TD></TR><TR><TD>	BBa_I15017	</TD><TD>	4/21N	</TD><TD>	eYFP	</TD><TD>	70	</TD><TD>	70,9	</TD></TR></TBODY></TABLE><P>Performed a PCR, as described in the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PCR">PCR protocol</A>, to amplify the parts CheY, CheZ, Rluc, eYFP so that they can be used for Gibson assembly later in the project. The primers used are listed in the table below.</P><TABLE><TBODY><TR><TH>	Part	</TH><TH>	Primer name			</TH><TH>	Primer sequence 5’--&gt; 3’			</TH></TR><TR><TD>	CheZ	</TD><TD>	CheZ_F1			</TD><TD>	gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag			</TD></TR><TR><TD>	CheZ_R1			</TD><TD>	CCTTGGAAGCCATTCCACCTCCACCTCCAAATCCAAGACTATCCAACAAATCG 			</TD></TR><TR><TD>	Rluc	</TD><TD>	Rluc_F1			</TD><TD>	AGTCTTGGATTTGGAGGTGGAGGTGGAATGGCTTCCAAGGTGTACG      			</TD></TR><TR><TD>	Rluc_R1			</TD><TD>	tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag			</TD></TR><TR><TD>	eYFP	</TD><TD>	YFP_F1     			</TD><TD>	gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag			</TD></TR><TR><TD>	YFP_R1     			</TD><TD>	CTTATCAGCCATTCCACCTCCACCTCCCTTGTACAGCTCGTCCATGCCG     			</TD></TR><TR><TD>	CheY	</TD><TD>	CheY_F1     			</TD><TD>	CGAGCTGTACAAGGGAGGTGGAGGTGGAATGGCTGATAAGGAATTGAAG          			</TD></TR><TR><TD>	CheY_R1    			</TD><TD>	tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag			</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayBA2"><H2>02/08/18</H2><B>Lorenzo</B><P>Performed a gel electrophoresis on the PCR product from <A onclick="calenderlink('BA1')">01/08/18</A> on a 1.5% agarose gel  according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#GelElectrophoresis">gel electrophoresis protocol</A>. The resulting bands were isolated according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#GelExtraction">gel purification protocol</A>.</P><TABLE><TBODY><TR><TH>	PCR product	</TH><TH>	Concentration (ug/ml)							</TH></TR><TR><TD>	CheY	</TD><TD>	/							</TD></TR><TR><TD>	Rluc	</TD><TD>	45,6							</TD></TR><TR><TD>	CheZ	</TD><TD>	59,9							</TD></TR><TR><TD>	eYFP	</TD><TD>	26,6							</TD></TR></TBODY></TABLE><P>
A second PCR was performed to retrieve CheY. Also eYFP, LuxA, and LuxB were amplified with  overhang extension PCR to add a gibson overhang to these parts. Used primers are listed in the table below. 
</P><TABLE><TBODY><TR><TH>	Part	</TH><TH>	Primer name	</TH><TH>	Sequence 5’ -&gt; 3’ </TH></TR><TR><TD>	LuxA	</TD><TD>	M13 fwd	</TD><TD> tgtaaaacgacggccagt	</TD></TR><TR><TD>	LuxAB_Gibson_R1	</TD><TD>	gatttGGAGGTGGAGGTGGAATGAAGTTTGGAAACTTCCTG					</TD></TR><TR><TD>	LuxB	</TD><TD>	LuxAB_Gibson_F1	</TD><TD> gatttGGAGGTGGAGGTGGAATGAAGTTTGGAAACTTCCTG			</TD></TR><TR><TD>	T3	</TD><TD>	tccctttagtgagggttaat					</TD></TR><TR><TD>	eYFP	</TD><TD>	YFP_F1	</TD><TD>	gccagtgaattgtaatacgactcactatagggcgaattgggaattcgcggccgcttctag					</TD></TR><TR><TD>	YFP_R1	</TD><TD>	CTTATCAGCCATTCCACCTCCACCTCCCTTGTACAGCTCGTCCATGCCG					</TD></TR><TR><TD>	CheY	</TD><TD>	CheY_F1	</TD><TD>	CGAGCTGTACAAGGGAGGTGGAGGTGGAATGGCTGATAAGGAATTGAAG					</TD></TR><TR><TD>	CheY_R1	</TD><TD>	tcactaaagggaacaaaagctggagctccaccgcggtggcctgcagcggccgctactag					</TD></TR></TBODY></TABLE><P>The PCR product was ran on a 1.5% agarose gel (figure 2)</P></DIV><DIV class="customelementM5B" id="entrydayBA3"><H2>03/08/18</H2><B>Lorenzo</B><P>LUXA and LuxB were isolated using PCR directly from the gBlock, since we expected the product of <A onclick="calenderlink('BA2')">02/08/18</A> to contain an error. The PCR product was ran on a 0.8% agarose gel and the correct bands were isolated according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#GelExtraction">gel extraction protocol.</A></P></DIV><DIV class="customelementM5B" id="entrydayBA6"><H2>06/08/18</H2><B>Floor</B><P>The CheZ and RLuc, and LuxA and LuxB fragments were fused together using Gibson assembly as described in the <A href="https://2018.igem.org/Team:Utrecht/Protocols#Gibson">Gibson protocol</A>. 9 µl of each Gibson product was transformed to 100 µl of DH5ɑ, since this increases the amount of successful transformations.</P></DIV><DIV class="customelementM5B" id="entrydayBA7"><H2>07/08/18</H2><B>Floor</B><P>The transformations were successful. 2 colonies from each plate were picked and analysed for bands of a correct size through colony PCR as described in the <A href="https://2018.igem.org/Team:Utrecht/Protocols#ColonyPCR">colony PCR protocol</A>. These bacteria were inoculated overnight at 37 °C in a rotary shaker.</P></DIV><DIV class="customelementM5B" id="entrydayBA8"><H2>08/08/18</H2><B>Lorenzo</B><P>DNA of inoculations (<A onclick="calenderlink('BA7')">07/08/18</A>) was purified in accordance with the<A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification"> miniprep protocol</A>.  The acquired DNA was sent for sequencing.</P></DIV><DIV class="customelementM5B" id="entrydayBA9"><H2>09/08/18</H2><B>Lorenzo</B><P>PCR was performed according to the protocol to create the correct overhangs for gibson assembly of CheY. 
</P></DIV><DIV class="customelementM5B" id="entrydayBA10"><H2>10/08/18</H2><B>Jolijn, Felix</B><P>PCR product from <A onclick="calenderlink('BA9')">09/08/18</A> was ran on 0.8% gel and isolated the DNA using the <A href="https://2018.igem.org/Team:Utrecht/Protocol#GelExtraction">Gel purification protocol</A></P></DIV><DIV class="customelementM5B" id="entrydayBA15"><H2>15/08/18</H2><B>Floor, Lorenzo</B><P>eYFP::CheY was assembled according to the <A href="https://2018.igem.org/Team:Utrecht/Protocol#Gibson">Gibson cloning protocol</A>. The cloning product was transformed into DH5α according to the iGEM transformation protocol.</P></DIV><DIV class="customelementM5B" id="entrydayBA16"><H2>16/08/18</H2><B>Lorenzo</B><P>Made a plate containing the positive colonies and inoculated them overnight.</P><P>Colony PCR of ((<A onclick="calenderlink('BA18')">18/08/18</A> eYFP::CheY)(<A onclick="calenderlink('BA19')">08/07/18</A> CheZ::Rluc)) was performed according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#ColonyPCR">protocol</A>. Positive samples were selected and inoculated overnight. </P></DIV><DIV class="customelementM5B" id="entrydayBA17"><H2>17/08/18</H2><B>Felix</B>
eYFP::CheY DNA <A onclick="calenderlink('BA16')">16/08/18</A><TABLE><TBODY><TR><TH>	Inoculation from colony	</TH><TH>	Concentration (ng/µl)							</TH></TR><TR><TD>	1	</TD><TD>	224.5							</TD></TR><TR><TD>	2	</TD><TD>	277.5							</TD></TR><TR><TD>	3	</TD><TD>	334.4							</TD></TR><TR><TD>	4	</TD><TD>	280.4							</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayBA20"><H2>20/08/18</H2><B>Felix</B><P>The purified DNA samples (<A onclick="calenderlink('BA17')">17/08/18</A>) were sent for sequencing with m13 forward and reverse primer.</P></DIV><DIV class="customelementM5B" id="entrydayBA27"><H2>27/08/18</H2><B> Lorenzo, Floor</B><P> Performed a PCR of CheY, Lux AB, CheZ and eYFP with the following primers:</P><TABLE><TBODY><TR><TH>	DNA	</TH><TH>	Forward Primer	</TH><TH>	Reverse Primer	</TH></TR><TR><TD>	1. CheY (kit)	</TD><TD>	Long Forward	</TD><TD>	CheY reverse	</TD></TR><TR><TD>	2. CheY (kit)	</TD><TD>	Short forward	</TD><TD>	CheY reverse	</TD></TR><TR><TD>	3. CheY (Gblock)	</TD><TD>	GblockCheY forward	</TD><TD>	Reverse 2	</TD></TR><TR><TD>	4. LuxAB	</TD><TD>	LuxAB forward, CheZ overhang	</TD><TD>	M13 reverse	</TD></TR><TR><TD>	5. CheZ (kit)	</TD><TD>	CheZ forward 1	</TD><TD>	CheZ reverse, luxAB overhang	</TD></TR><TR><TD>	6. eYFP	</TD><TD>	eYFP forward	</TD><TD>	eYFP reverse	</TD></TR></TBODY></TABLE><P>The samples were ran on an 0.8% gel and bands were isolated </P><TABLE><TBODY><TR><TD>	1	</TD><TD>	2	</TD><TD>	3	</TD><TD>	4	</TD><TD>		5	</TD><TD>	6	</TD></TR><TR><TD>	10	</TD><TD>	10.1	</TD><TD>	5.7	</TD><TD>	3.7	</TD><TD>		0.9	</TD><TD>	0	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayBA28"><H2>28/08/18</H2><B>Floor</B><P>Repeat of <A onclick="calenderlink('BA27')">27/08/18</A>, difference: PCR settings as KOD polymerase with 35 cycles. 
Ran the product on 0.8% (w/v) agarose gel and isolated it using the <A href="https://2018.igem.org/Team:Utrecht/Protocols#GelExtraction">gel purification protocol</A>. </P><P>The fragments were used to perform a Gibson assembly of eYFP::CheY, and 9 µl of the product was transformed to 100 µl DH5α.
</P></DIV><DIV class="customelementM5B" id="entrydayBA29"><H2>29/08/18</H2><B>Floor</B><P>We checked the plates from <A onclick="calenderlink('BA28')">28/08/18</A> but none of the transformations were successful. </P></DIV><DIV class="customelementM5B" id="entrydayBS3"><H2>03/09/18</H2><B>Lorenzo</B><P>We performed the gibson reaction of eYFP::CheY again. The gibson product was transformed directly to DH5α. Furthermore, a Gibson overhang for the CheZ and LuxAB fragments was created by PCR with primers that were extended. The PCR products were ran on a 0.8% agarose gel. The gel revealed faint bands, but no ladder. Hence we didn’t proceed with these samples.</P></DIV><DIV class="customelementM5B" id="entrydayBS11"><H2>11/09/18</H2><B>Lorenzo</B><P>Since our constructs used during our project needed a promoter in order to be expressed promoter BBa_K608003 was isolated from the DNA distribution kit (plate 1 well 5A). This DNA was transformed to DH5α.</P></DIV><DIV class="customelementM5B" id="entrydayBS12"><H2>12/09/18</H2><B>Felix</B><P>Colonies from <A onclick="calenderlink('BS11')">11/09/18</A> were picked and inoculated overnight.</P></DIV><DIV class="customelementM5B" id="entrydayBS13"><H2>13/09/18</H2><B>Lorenzo</B><P>DNA from inoculations was isolated according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">miniprep protocol</A>. The promoter part was was cloned in front of Both eYFP::CheY from <A onclick="calenderlink('BA28')">28/08/18</A>, and CheZ::RLuc using the iGEM 3A assembly protocol. The samples were transformed to DH5α.</P></DIV><DIV class="customelementM5B" id="entrydayBS14"><H2>14/09/18</H2><B>Lorenzo</B><P>eYFP::CheY plates were observed under the fluorescence binoculars. 2 YFP colonies could be observed. These colonies were inoculated overnight at 37°C. </P></DIV><DIV class="customelementM5B" id="entrydayBS15"><H2>15/09/18</H2><B>Felix</B><P>The inoculated bacteria were used for a miniprep and the DNA was send for sequencing.</P></DIV><DIV class="customelementM5B" id="entrydayBO6"><H2>06/10/18</H2><B>Felix</B><B>Oligo annealing</B><P>Oligos containing the promoter CheZ::RLuc part sequence were annealed according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#Annealing">annealing protocol</A></P><B>CheZ Rluc Digestion</B><P>Two CheZ::Rluc samples (CheZ::Rluc 440 ng/μl, CheZ::Rluc 430 ng/μl) were digested with Xbal and EcoR1 according to the iGEM restriction digest protocol. </P><B>Oligo CheZ::Rluc ligation</B><P>Protomoter Oligo was ligated into the Xbal Rluc digested CheZ::Rluc backbone according to the iGEM T4 ligase protocol . 1 μl  of CheZ::Rluc backbone with 1 μl of oligo was used. Both 440 ng/μl and 430 ng/μl sample were ligated with the oligo.</P><B>Transformation</B><P>The new vector product (promoter::CheZ::Rluc) was transformed into DH5α according to the iGEM transformation protocol. </P><UL><LI>    CheZ Rluc A</LI><LI>     CheZ Rlux B</LI></UL><P>Anneal A = Oligo insert::CheZ::Rluc        (originating from sample CheZ::Rluc 430 ng/μl )
Anneal B = Oligo insert::Chez::Rluc         (originating from sample CheZ::Rluc 440 ng/μl )</P></DIV><DIV class="customelementM5B" id="entrydayBO7"><H2>07/10/18</H2><B>Felix</B><P> Transformation was succesful </P><P> Three colonies were picked from transformation A and two colonies were picked from transformation B, both were inoculated in 5 ml LB containing ampicillin. </P></DIV><DIV class="customelementM5B" id="entrydayBO8"><H2>08/10/18</H2><B>Lorenzo</B><P>DNA was purified from the following samples using the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">miniprep protocol</A> and sent for sequencing.</P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Concentration ng/μl		</TH></TR><TR><TD>	CheZ Rluc A4	</TD><TD>	584.1							</TD></TR><TR><TD>	CheZ Rluc B4	</TD><TD>	549.6							</TD></TR><TR><TD>	CheZ Rluc A2	</TD><TD>	522.3							</TD></TR><TR><TD>	CheZ Rluc A1	</TD><TD>	463.8							</TD></TR><TR><TD>	CheZ Rluc B3	</TD><TD>	539.9							</TD></TR></TBODY></TABLE><P> The Gibson cloning was transformed in DH5α according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#Gibson">protocol</A>.  </P></DIV><DIV class="customelementM5B" id="entrydayBO9"><H2>09/10/18</H2><B>Lorenzo</B><P>Transformation was successful</P><P>Two colonies were picked and inoculated in 5 ml LB containing ampicillin. </P></DIV><DIV class="customelementM5B" id="entrydayBO10"><H2>10/10/18</H2><B>Lorenzo, Felix</B><P>DNA from the two inoculation of 1/5A::CheZ::Rluc after Gibson cloning was retrieved with miniprep according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">protocol</A> in the following concentrations. </P>
Sample 1:  139.9 ng/μl
Sample 2:  75.5 ng/μl
</DIV><DIV class="customelementM5B" id="entrydayBO11"><H2>11/10/18</H2><B>Felix, Jolijn, Mike</B><P>In order to implement the complete BRET-pair and the mutated TAR-receptor, all three constructs needed to be in different vectors. Therefore, the CheZ Rluc and 1/5A::eYFP::Chey DNA was put in the iGEM pSB1K3 vector. Both samples including the vector were cut with EcoRI and PstI. Assembly of products was performed according to the iGEM 3A assembly protocol resulting in the following samples:</P>
1/5A eYFP CheY in pSB1K3 
1/5A CheZ Rluc in pSB1K3 (sample 1) 
1/5A CheZ Rluc in pSB1K3 (sample 2) <P>These samples were transformed in combination with the mutated TAR receptor DNA. The following samples were transformed. </P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Content	</TH><TH>	Cell line	</TH></TR><TR><TD>	1	</TD><TD>	SDM Q491A	</TD><TD>	DH5α	</TD></TR><TR><TD>	2	</TD><TD>	SDM E309A	</TD><TD>	DH5α	</TD></TR><TR><TD>	3	</TD><TD>	[⅕A::eYFP::CheY] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM Q491A] Chlo	</TD></TR><TR><TD>	[CheZ::Rluc] Amp	</TD></TR><TR><TD>	4	</TD><TD>	[CheZ::Rluc (sample 1)] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM Q491A] Chlo	</TD></TR><TR><TD>	[⅕A::eYFP::CheY] Amp	</TD></TR><TR><TD>	5	</TD><TD>	[CheZ::Rluc (sample 2)] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM Q491A] Chlo	</TD></TR><TR><TD>	[⅕::eYFP::CheY] Amp	</TD></TR><TR><TD>	6	</TD><TD>	[[⅕A::eYFP::CheY] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM E309A] Chlo	</TD></TR><TR><TD>	[CheZ::Rluc] Amp	</TD></TR><TR><TD>	7	</TD><TD>	[CheZ::Rluc (sample 1)] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM E309A] Chlo	</TD></TR><TR><TD>	[⅕A+eYFP+CheY] Amp	</TD></TR><TR><TD>	8	</TD><TD>	[CheZ::Rluc (sample 2)] Kan	</TD><TD>	U1250	</TD></TR><TR><TD>	[SDM E309A] Chlo	</TD></TR><TR><TD>	[⅕A::eYFP::CheY] Amp	</TD></TR><TR><TD>	9	</TD><TD>	[⅕A::eYFP::CheY] Kan	</TD><TD>	DH5α	</TD></TR><TR><TD>	10	</TD><TD>	[CheZ::Rluc (sample 1)] Kan	</TD><TD>	DH5α	</TD></TR><TR><TD>	11	</TD><TD>	[CheZ::Rluc (sample 2)] Kan	</TD><TD>	DH5α	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayBO12"><H2>12/10/18</H2><B>Felix, Lorenzo</B><P>SDM transformations (Lab Journal lab journal <A onclick="tablink('M')"> Methylation </A>) and triple transformations were successful. When the plates were inspected under the fluorescence binoculars, several eYFP positive colonies could be identified that weren’t RFP positive. This indicated that these bacteria contained all 3 plasmids, and weren’t antibiotic resistant due to self ligation events of the linearized vector backbone used on <A onclick="calenderlink('BO11')">11/10/18</A>.</P><P>2 colonies were picked and inoculated in 5 ml LB with ampicillin, kanamycin, chloramphenicol. The selection of colonies was based on increased YFP expression and low RFP expression (background plasmid signal). </P><P>The following samples were transformed</P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>			Cell line	</TH></TR><TR><TD>	6/15B (TAR WT),	</TD><TD>			U1250	</TD></TR><TR><TD>	FRET pair	</TD><TD>			U1250	</TD></TR><TR><TD>	BRET pair (1/5A::CheZ::Rluc)(1/5A::eYFP::CheY)	</TD><TD>			U1250	</TD></TR><TR><TD>	6/15B (TAR WT), FRET pair	</TD><TD>			U1250	</TD></TR><TR><TD>	6/15B (TAR WT), BRET pair (1/5A::CheZ::Rluc)(1/5A::eYFP::CheY)	</TD></TR>			U1250	<TR><TD>	FRET pair, SDM E309A	</TD><TD>			U1250	</TD></TR><TR><TD>	FRET pair, Q491A	</TD><TD>			U1250	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayBO13"><H2>13/10/18</H2><B>Felix, Jolijn</B><P>Inoculations were spun down, washed once with 1x PBS and resuspended in 1x PBS. 500 μg coerentazine was solubilized in ethanol to reach a final concentration of 7.5 mM. An aliquot was taken from this stock to make a 30x solution with 1x PBS. 480 μl bacteria were put in a 400 μl quartz cuvette and 20 μl  30x coerentazine solution was added. The solution was oxygenated using a pipette. No luminescence could be observed, indicating that the coelenterazine is not oxidized by luciferase. Fluorescence measurements of a wavelength that excites YFP,  proved YFP to be present. </P></DIV><DIV class="customelementM5B" id="entrydayBO16"><B>Jolijn</B><H2>17/10/18</H2><P>Samples that were transformed on <A onclick="calenderlink('BO12')">12/10/18</A>, consisting of both the BRET pair and the tar receptors with altered amino acid sequence (to mimic methylation) were inoculated in LB.</P></DIV><DIV class="customelementM5B" id="entrydayBO16"><B>Jolijn</B><H2>17/10/18</H2><P>The inoculated samples were transferred to a citrate phosphate buffer (0.15 M, pH 5). Coelenterazine was added to a final concentration of 7.5 µM. Measurements were performed, there was no measurable luminescence.  </P></DIV><DIV class="customelementM5B" id="entrydayMJ30"><H2>30/07/18</H2><B>Jolijn</B><H3>Transformation of Tar part 1</H3><P>The Tar, Tar GFP and Tar GFP His plasmids received on a filter paper from team Technion Israël 2016 were dissolved in 80 µL elution buffer. ~40 µL of plasmid dissolved in elution buffer was extracted. Competent DH5α cells were then transformed with the dissolved plasmids according to the iGEM protocol. Since the type of antibiotic resistance was unknown, Kanamycin, Ampicillin and Chloramphenicol were used as antibiotics.</P></DIV><DIV class="customelementM5B" id="entrydayMJ31"><H2>31/07/18</H2><B>Jolijn</B><H3>Transformation of Tar part 2</H3><P>The transformation of Tar GFP His on chloramphenicol was the only successful transformation. Two colonies were picked and inoculated in 5 ml LB containing chloramphenicol. </P></DIV><DIV class="customelementM5B" id="entrydayMA1"><H2>01/08/18</H2><B>Jolijn</B><H3>Transformation of Tar GFP His part 3</H3><P>Tar GFP His DNA was isolated from the inoculation (<A onclick="calenderlink('MJ31')">31/07/18</A>) according to the MiniPrep <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">protocol</A>. An aliquot of the left bacteria was plated in fresh agar plates. </P><H3>Transformation of Tar part 1</H3><P>The DNA concentration of the Tar, Tar GFP and Tar GFP His samples were measured, concentrations are listed in the table below.  </P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>			Concentration	</TH></TR><TR><TD>	Tar 	</TD><TD>			10.4 µg/mL	</TD></TR><TR><TD>	Tar GFP	</TD><TD>			9.1 µg/mL	</TD></TR><TR><TD>	Tar GFP His	</TD><TD>			11.1 µg/mL	</TD></TR></TBODY></TABLE><P>
The desired concentration of DNA for transformation is 100 ng in a 50 µL solution. Due to the low concentration of our samples, 5 µL of DNA sample was added to the 50 µL DH5α competent cells.  Additionally, the Tar receptor (6/4j) and Tar receptor with promotor (6/15B) plasmids from the distribution kit were transformed as described in the iGEM transformation protocol.

</P></DIV><DIV class="customelementM5B" id="entrydayMA2"><H2>02/08/18</H2><B>Felix, Jolijn</B><H3>Transformation of Tar part 2</H3><P>All transformations were successful, with a high colony density. One colony per sample was inoculated for 6 hours. After 6 hours the colonies were put in 5 ml LB + chloramphenicol and incubated overnight 37°C 200 rpm. Samples were as described below. </P><TABLE><TBODY><TR><TD>	Tar A      	</TD><TD>			          Ta	</TD></TR><TR><TD>	Tar B      	</TD><TD>			          Tb    	</TD></TR><TR><TD>	Tar Kit A   	</TD><TD>			         Tka	</TD></TR><TR><TD>	Tar Kit B       	</TD><TD>			     Tkb	</TD></TR><TR><TD>	Tar GFP A    	</TD><TD>			        Tga	</TD></TR><TR><TD>	Tar GFP B   	</TD><TD>			         Tgb	</TD></TR><TR><TD>	Tar GFP His A (Israel)       	</TD><TD>			 Tgha	</TD></TR><TR><TD>	Tar GFP His B (Israel)    	</TD><TD>			    Tghb	</TD></TR><TR><TD>	Promotor Tar Kit A     	</TD><TD>			   Ptka	</TD></TR><TR><TD>	Promotor Tar Kit B        	</TD><TD>			 Ptkb    	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayMA3"><H2>03/08/18</H2><B>Felix, Jolijn</B><H3>Transformation of Tar part 2</H3><P>DNA was retrieved from the following samples using miniprep according to <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">protocol</A>. </P><TABLE><TBODY><TR><TH>	Receptor	</TH><TH>			ug/ml	</TH></TR><TR><TD>	T B	</TD><TD>			65.8	</TD></TR><TR><TD>	T A	</TD><TD>			66.5	</TD></TR><TR><TD>	Tk A	</TD><TD>			60.7	</TD></TR><TR><TD>	Tk B	</TD><TD>			75.4	</TD></TR><TR><TD>	Tg A	</TD><TD>			73.3	</TD></TR><TR><TD>	Tg B	</TD><TD>			69.1	</TD></TR><TR><TD>	Tgh B	</TD><TD>			61.6	</TD></TR><TR><TD>	TghA	</TD><TD>			77.7	</TD></TR><TR><TD>	Ptk A	</TD><TD>			68.6	</TD></TR><TR><TD>	Ptk B	</TD><TD>			78.9	</TD></TR></TBODY></TABLE><P>Site Directed Mutagenesis (SDM) was performed to mimic constant methylation of the Tar receptors. The experiment was carried out according to the <A href="https://2018.igem.org/Team:Utrecht/Protocols#SDM">SDM protocol</A>. Only B samples from the duplo were used for SDM. Following primers were used to specifically mutate Q491A and E309A. </P><TABLE><TBODY><TR><TD>	E491A_F	</TD><TD>			GCATCGCTGGTGCAGGCGTCAGCTGCCGCCGCC	</TD></TR><TR><TD>	E491A_R	</TD><TD>			GGCGGCGGCAGCTGACGCCTGCACCAGCGATGC	</TD></TR><TR><TD>	Q309A_F	</TD><TD>			CTGCCGCCAGCATGGAGGCGCTCACCGCGACAGTGAAG	</TD></TR><TR><TD>	Q309A_R	</TD><TD>			CTTCACTGTCGCGGTGAGCGCCTCCATGCTGGCGGCAG	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayMA6"><H2>06/08/18</H2><B>Jolijn</B><P>The prior made (<A onclick="calenderlink('MA3')">03/08/18</A>) SDM plasmids were transformed <I>in duplo</I> into DH5α cells according to the iGEM protocol.</P></DIV><DIV class="customelementM5B" id="entrydayMA7"><H2>07/08/18</H2><B>Felix</B><P>Transformation was successful. The colony density was very high, which made it difficult to pick specific colonies for further incubation. Yet, colonies were picked and activated in 1 ml LB + chloramphenicol for 5 hours. After 5 hours no bacterial growth was observed. 
Colonies were picked again and inoculated overnight in 5 ml LB + chloramphenicol. To ease colony picking, a colony  of each sample was picked roughly and spread and diluted on a new agar plate + chloramphenicol. This plate was incubated overnight at 37°C.  </P></DIV><DIV class="customelementM5B" id="entrydayMA8"><H2>08/08/18</H2><B>Jolijn, Felix</B><P>The colonies picked on <A onclick="calenderlink('MA7')">07/08/18</A> were diluted successfully. The plasmid DNA of the inoculations  (<A onclick="calenderlink('MA7')">07/08/18</A>) was purified according the <A href="https://2018.igem.org/Team:Utrecht/Protocols#PlasmidPurification">miniprep protocol</A>.</P><P>Following concentrations were observed. </P><TABLE><TBODY><TR><TH>	Receptor	</TH><TH>			Concentration ug/ml 	</TH></TR><TR><TD>	Tb2 	</TD><TD>			43.6	</TD></TR><TR><TD>	Tb1	</TD><TD>			49.1	</TD></TR><TR><TD>	Tgb2	</TD><TD>			52.7	</TD></TR><TR><TD>	Tgb1	</TD><TD>			28.1	</TD></TR><TR><TD>	Tkb2	</TD><TD>			46.33	</TD></TR><TR><TD>	Tkb1	</TD><TD>			46.3	</TD></TR><TR><TD>	Tgh2	</TD><TD>			15.3	</TD></TR><TR><TD>	Tgh1	</TD><TD>			50.8	</TD></TR><TR><TD>	Ptkb2	</TD><TD>			3	</TD></TR><TR><TD>	Ptkb1	</TD><TD>			32.6	</TD></TR></TBODY></TABLE><P>Tb2 and Tgb2 were sent for sequencing, although DNA concentration was not optimal. 

DNA concentrations were not satisfactory, new colonies were picked and inoculated in 5 ml LB with chloramphenicol. A control was included to compare retrieved DNA concentrations.  </P></DIV><DIV class="customelementM5B" id="entrydayMA9"><H2>09/08/18</H2><B>Jolijn, Felix</B><P>DNA from inoculations was retrieved using miniprep according to the protocol. One modification was applied, the elution buffer was heated (60°C) before use and only 30 µl was applied on the spin column. </P><TABLE><TBODY><TR><TH>	Receptor	</TH><TH>			Concentration (µl/ml)	</TH></TR><TR><TD>	T1	</TD><TD>			102.9	</TD></TR><TR><TD>	T2	</TD><TD>			106.2	</TD></TR><TR><TD>	Tg1	</TD><TD>			85	</TD></TR><TR><TD>	Tg2	</TD><TD>			110.2	</TD></TR><TR><TD>	Tk1	</TD><TD>			33.9	</TD></TR><TR><TD>	Tk2	</TD><TD>			68.8	</TD></TR><TR><TD>	Ptk1	</TD><TD>			13.3	</TD></TR><TR><TD>	Ptk2	</TD><TD>			71	</TD></TR><TR><TD>	Tgh1	</TD><TD>			48.2	</TD></TR><TR><TD>	Tgh2	</TD><TD>			49.1	</TD></TR><TR><TD>	Control (Vector = PbsK)	</TD><TD>			428.4	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayMA13"><H2>13/08/18</H2><B>Felix</B><P>TG, TG2, T1,T2 were sent for sequencing.</P></DIV><DIV class="customelementM5B" id="entrydayMA14"><H2>14/08/18</H2><B>Jolijn</B><P>Sequencing data revealed presence of GFP in all the sent sequences. </P></DIV><DIV class="customelementM5B" id="entrydayMA27"><H2>27/08/18</H2><B>Jolijn</B><H3>Colony picking of E. coli</H3><P><I>E. coli</I> used for the interlab study were picked and inoculated for an hour.</P><H3>Genomic PCR </H3><P>The obtained samples were used for colony PCR. Afterwards, the samples were loaded on a 0.8% agarose gel at 120 V for 20 minutes. Bands were isolated and purified, using the gel purification protocol. The concentrations were too low to continue (1.3µL and 1.9µL).</P></DIV><DIV class="customelementM5B" id="entrydayMA29"><H2>29/08/18</H2><B>Jolijn</B><P>Receptor assay inoculations (custom receptor//Cu-Promoter::RFP sample 13 and 14 were used for genomic PCR. 1 mL of these cell cultures were used and spun down on 12000 x g for 1 minute. The supernatant was removed and resuspended in 100 µL milli-Q, after which custom 13 was heated at 85 °C’. The samples were diluted 0, 10 and 100 times and the resulting dilutions were used for 1-Taq PCR using the KOD polymerase <A href="https://2018.igem.org/Team:Utrecht/Protocols#PCR">protocol</A>.</P><P>Next, 50 µL loading buffer was added and all samples from custom 13 and 14 were loaded in lane 2 and lane 3 of a 0.8% agarose gel respectively. The bands were isolated and put together. The DNA concentration was 146.1 µg/mL.</P></DIV><DIV class="customelementM5B" id="entrydayMA30"><H2>30/08/18</H2><B>Jolijn</B><H3>T4 PNK Phosphorylation and T4 ligation</H3><P>The DNA fragments were phosphorylated according to the T4 PNK Phosphorylation protocol Using the following pipet scheme:</P><TABLE><TBODY><TR><TH>	Compound	</TH><TH>			Volume	</TH></TR><TR><TD>	DNA 	</TD><TD>			15 µL	</TD></TR><TR><TD>	10x buffer	</TD><TD>			2 µL	</TD></TR><TR><TD>	ATP 10 mM	</TD><TD>			2 µL	</TD></TR><TR><TD>	T4 PNK	</TD><TD>			1 µL	</TD></TR></TBODY></TABLE><P>Next, ligation was done in order to ligate the isolated Tar receptor DNA into the pBSK vector. This was done according to the following pipetting scheme:</P><TABLE><TBODY><TR><TH>	Compound	</TH><TH>			Volume	</TH></TR><TR><TD>	T4 ligase	</TD><TD>			0.9 µL	</TD></TR><TR><TD>	Buffer	</TD><TD>			1 µL	</TD></TR><TR><TD>	Vector	</TD><TD>			1 µL	</TD></TR><TR><TD>	DNA	</TD><TD>			7.60 µL	</TD></TR></TBODY></TABLE><P>After four hours of incubation, competent DH5α cells were transformed with the sample in duplo according to the iGEM transformation protocol.</P></DIV><DIV class="customelementM5B" id="entrydayMS11"><H2>11/09/18</H2><B>Jolijn, Lorenzo</B><P>Transformation was successful, colonies were picked.  Cells were incubated in 4 mL LB with 4 µL ampicillin. </P></DIV><DIV class="customelementM5B" id="entrydayMS30"><H2>30/09/18</H2><B>Lorenzo</B><P>Tar receptor received from Groningen and transformed into DH5a according to the protocol.</P></DIV><DIV class="customelementM5B" id="entrydayMO3"><H2>03/10/18</H2><B>Lorenzo</B><P>Tar receptor DNA was isolated using miniprep, DNA concentration was 356.3 ng/ul. 
First site directed mutagenesis was performed (E491A) using phusion polymerase and listed HPLC purified primers. 
</P><TABLE><TBODY><TR><TD>	E491A_F	</TD><TD>			GCATCGCTGGTGCAGGCGTCAGCTGCCGCCGCC	</TD></TR><TR><TD>	E491A_R	</TD><TD>			GGCGGCGGCAGCTGACGCCTGCACCAGCGATGC	</TD></TR><TR><TD>	Q309A_F	</TD><TD>			CTGCCGCCAGCATGGAGGCGCTCACCGCGACAGTGAAG	</TD></TR><TR><TD>	Q309A_R	</TD><TD>			CTTCACTGTCGCGGTGAGCGCCTCCATGCTGGCGGCAG	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayMO4"><H2>04/10/18</H2><P>SDM1 (E491A) was digested with DpnI for 1.5 hours and heat inactivated for 30 min. 
SDM2 (Q309A) was performed with SDM1 (E491A) as template. As the DNA concentration could not be retrieved, a titer of three different DNA concentrations was in three different DNA volumes: </P><UL><LI>0.3 ul</LI><LI>0.7 ul</LI><LI>1.1 ul</LI></UL><P>SDM1 (E491A) and SDM 2 (Q309A) were transformed in DH5a. During transformation, heat shock was performed inaccurately, therefore, the transformation was performed again.</P>

1.1 (E491A)          →         SDM 1 
2.1 (Q309A)          →        SDM A 0,3 ul
2,1 (Q309A)        →         SDM B 0,7 ul
2,1 (Q309A)         →         SDM C 1,1 ul 

<P>Second Transformation: </P>
1.1 (E491A)        →         SDM 1 second time 
2.1  (Q309A)        →         SDM 2.1 A second time 0.3 ul 
2.1  (Q309A)        →         SDM 2.1 B second time 0,7 ul 
2.1  (Q309A)        →         SDM 2.1 C second time 1.1 ul </DIV><DIV class="customelementM5B" id="entrydayMO5"><H2>05/10/18</H2><B>Felix</B>
Several transformations were successful. 
Two colonies from the following transformations were inoculated
1.1 (E491A) first time
1.1 (E491A) second time
2.1 (Q309A) B 0,7 ul
</DIV><DIV class="customelementM5B" id="entrydayMO6"><H2>05/10/18</H2><B>Felix, Lorenzo</B><P>DNA from the following samples was purified according to the miniprep protocol. </P>

DNA 6/15B (Groningen)         (DNA 6/15B     → SDM 1.3 (E491A))  
SDM (E491A) 1.2 A            (SDM 1.2 A     → SDM 2.2 (Q309A) A)  
SDM (E491A) 1.2 B              (SDM 1.2 B     → SDM 2.2 (Q309A) B) <P>Following samples were transformed.</P><P>Chloramphenicol + Ampicillin, U1250</P><UL><LI>  pVS88 + Tar SDM 1.2  (E491A) A</LI><LI>pVS88 + Tar SDM 2.1  (Q309A) A</LI><LI>    pVS88 + Tar SDM 2.2 (Q309A) A</LI><LI>    pVS88 + Tar SDM 2.2 (Q309A) B</LI></UL><P>Chloramphenicol DH5α</P><UL><LI>Tar SDM 2.2 A (Q309A) (Not digested with DpnI overnight)</LI><LI>    Tar SDM 2.2 B (Q309A) (Not digested with DpnI overnight)</LI><LI>     Tar SDM 2.1 A (Q309A)</LI><LI>     Tar SDM 2.1 B (Q309A)</LI><LI>     Tar SDM 2.1 C (Q309A)</LI><LI>    Tar SDM 1.3 (E491A)</LI></UL></DIV><DIV class="customelementM5B" id="entrydayMO7"><H2>07/10/18</H2><B>Jolijn</B><P>Following sequences were transformed according to the iGEM protocol to DH5α</P><UL><LI>    SDM 2.1 (Q309A) C </LI><LI>    SDM 2.2 (Q309A) A after overnight DpnI </LI><LI>    SDM 2.2 (Q309A) B after overnight DpnI </LI></UL><P> Transformations were succesfull </P><B> Following sequences were inoculated </B><UL><LI>CheZ Rluc A (1)</LI><LI>CheZ Rluc B (1)</LI><LI>CheZ Rluc A (2)</LI><LI>CheZ Rluc A (3)</LI><LI>CheZ Rluc B (2)</LI><LI>SDM 2.1 (Q309A) B</LI><LI>SDM 1.3 (E491A) (1)</LI><LI>SDM 1.3 (E491A) (2)</LI></UL></DIV><DIV class="customelementM5B" id="entrydayMO8"><H2>08/10/18</H2><B>Felix</B><P>DNA was acquired from samples according to the miniprep protocol with the following DNA concentrations. </P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Concentration (ng/μl)			</TH></TR><TR><TD>	SDM 2.1 (Q309A) B	</TD><TD>	171.1			</TD></TR><TR><TD>	SDM 1.3 (E491A) (1)	</TD><TD>	212.2			</TD></TR><TR><TD>	SDM 1.3 (E491A) (2)	</TD><TD>	110.3			</TD></TR></TBODY></TABLE><B>Inoculation</B><P>Transformations: SDM 2.2 (Q309A) A after overnight DpnI, SDM 2.2 (Q309A) B after overnight DpnI, SDM 2.1 (Q309A) C  (1.1 μl), were successful.</P><P>Two colonies were picked from the following samples:</P><LI>SDM 2.2 (Q309A) A after overnight DpnI</LI><LI>SDM 2.2 (Q309A) B after overnight DpnI </LI><P>One colony was picked from </P><UL><LI>SDM 2.1 (Q309A) C  (1.1 μl)</LI></UL></DIV><DIV class="customelementM5B" id="entrydayMO9"><H2>09/10/18</H2><B>Felix</B><P>DNA from the following inoculations was purified</P><UL><LI>2x SDM 2.2 (Q309A) A after overnight DpnI</LI><LI>2x SDM 2.2 (Q309A) B after overnight DpnI</LI><LI>1x SDM 2.1 (Q309A) C (1.1 μl)</LI></UL><P>The following concentrations were measured and sent for sequencing.</P><TABLE><TBODY><TR><TH>	Sample	</TH><TH>	Concentration (ng/μ)	</TH><TH>	Sent for Seq. 	</TH></TR><TR><TD>	SDM 2.1 (Q309A)  C	</TD><TD>	253.2	</TD><TD>	VF: 1BA9ZAB682	</TD></TR><TR><TD>	VR: 1BA9ZAB681	</TD></TR><TR><TD>	SDM 2.2 (Q309A) A	</TD><TD>	188	</TD></TR><TR><TD>	SDM 2.2 (Q309A) A2	</TD><TD>	309	</TD><TD>	VF: 1BA9ZAB678 	</TD></TR><TR><TD>	VR: 1BA9ZAB677	</TD></TR><TR><TD>	SDM 2.2 (Q309A) B	</TD><TD>	219.7	</TD><TD>	VF: 1BA9ZAB680	</TD></TR><TR><TD>	VR: 1BA9ZAB679	</TD></TR><TR><TD>	SDM 2.2 (Q309A) B2	</TD><TD>	173.7	</TD></TR></TBODY></TABLE></DIV><DIV class="customelementM5B" id="entrydayMO10"><H2>10/10/18</H2><B>Felix</B><P>Analysis of the sequences revealed that none of the samples contained the desired DNA sequence with the specific mutation. </P></DIV><DIV class="customelementM5B" id="entrydayMO11"><H2>11/10/18</H2><B>Felix, Lorenzo</B><P>Again SDM was performed (SDM Q491A as well as SDM E309A) according to the protocol using phusion polymerase with lower primer concentrations of vector DNA. SDM Q491A as well as SDM E309A were transformed in DH5α according to the protocol. The two SDM samples were also transformed in U1250 with the BRET pair (lab journal <A onclick="tablink('B')"> BRET </A>).</P></DIV><DIV class="customelementM5B" id="entrydayMO12"><H2>11/10/18</H2><B>Felix</B><P>SDM transformations were successful. 2 colonies from each SDM sample were picked and inoculated in 5 ml LB with chloramphenicol. SDM transformation in combination with the BRET pair were successful as well (lab journal <A onclick="tablink('B')">BRET</A>).</P>
E309A

Q491A
</DIV><DIV class="navigateArrows"><DIV class="arrowLeft"><A href="https://2018.igem.org/Team:Utrecht/Design"> Project Design</A></DIV><DIV class="arrowRight"><A href="https://2018.igem.org/Team:Utrecht/Results">Results </A></DIV><DIV class="arrowRight"><A href="https://2018.igem.org/Team:Utrecht/Protocols">Protocols </A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>