<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse-INSA-UPS_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse-INSA-UPS/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="MENU" class="shadow-lg sticky-top"><NAV class="navbar navbar-expand-lg navbar-dark bg-dark shadow"><A class="navbar-brand" href="https://2018.igem.org/Team:Toulouse-INSA-UPS"> 
					Toulouse INSA-UPS
				</A><DIV class="collapse navbar-collapse flex-row-reverse navbarNavAll"><UL class="navbar-nav"><LI class="nav-item active dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Awards" id="navbarSafety">üèÜ</A></LI><LI class="nav-item active dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Toulouse-INSA-UPS" id="navbarSafety">HOME</A></LI><LI class="nav-item active dropdown"><A class="nav-link" href="#" id="navbarTeam" data-toggle="dropdown">TEAM</A><DIV class="dropdown-menu dropdown-menu-right"><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Team">Team Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Sponsors">Sponsors</A></DIV></LI><LI class="nav-item active dropdown"><A class="nav-link" href="#" id="navbarProject" data-toggle="dropdown">PROJECT</A><DIV class="dropdown-menu dropdown-menu-right"><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Description"><STRONG>Description</STRONG></A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Design"><STRONG>Design</STRONG></A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Results"><STRONG>Results</STRONG></A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Demonstrate"><STRONG>Demonstrate</STRONG></A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Applied_Design"><STRONG>Product Design</STRONG></A></DIV></LI><LI class="nav-item active dropdown"><A class="nav-link" id="navbarParts" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Model">MODEL</A></LI><LI class="nav-item active dropdown"><A class="nav-link" href="#" id="navbarParts" data-toggle="dropdown">PARTS</A><DIV class="dropdown-menu dropdown-menu-right"><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Parts">Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Basic_Part">Basic Parts</A></DIV></LI><LI class="nav-item active dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Safety" id="navbarSafety">SAFETY</A></LI><LI class="nav-item active dropdown"><A class="nav-link" href="#" id="navbarHP" data-toggle="dropdown">HUMAN PRACTICES</A><DIV class="dropdown-menu dropdown-menu-right"><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Public_Engagement">Education &amp; Engagement</A></DIV></LI><LI class="nav-item active dropdown"><A class="nav-link" href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Achievements" id="navbarAwards">ACHIEVEMENTS</A></LI></UL></DIV></NAV></DIV><MAIN class="container-fluid"><DIV class="row"><DIV class="col-10 col-lg-8 text-justify" id="CONTENT"><DIV class="column full_size"><H1 class="heavy">DESIGN</H1><DIV id="Description_tab"><UL class="nav nav-pills d-flex flex-row flex-wrap flex-lg-nowrap justify-content-around justify-content-lg-start" role="tablist"><LI class="nav-item"><A class="nav-link btn btn-secondary rounded-top corner-bottom mr-1 active" data-toggle="pill" href="#Design_Part1" role="tab" style="color:white !important;">Our System</A></LI><LI class="nav-item"><A class="nav-link btn btn-secondary rounded-top corner-bottom mr-1" data-toggle="pill" href="#Design_Part2" role="tab" style="color:white !important;">Validating Our System</A></LI><LI class="nav-item"><A class="nav-link btn btn-secondary rounded-top corner-bottom mr-1" data-toggle="pill" href="#Design_Part3" role="tab" style="color:white !important;">The Applications</A></LI><LI class="nav-item"><A class="nav-link btn btn-secondary rounded-top corner-bottom mr-1" data-toggle="pill" href="#Design_Part4" role="tab" style="color:white !important;">Versatility of the System</A></LI><LI class="nav-item"><A class="nav-link btn btn-secondary rounded-top corner-bottom mr-1" data-toggle="pill" href="#Design_Part5" role="tab" style="color:white !important;"><I>In vivo</I> functionalisation</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade show active" id="Design_Part1" role="tabpanel"><H2 class="heavy">Our System</H2><P>To exploit the potential of cellulose as a material capable of being functionalised, we designed a three-headed linker platform named Cerberus. The protein will allow us to bind a huge range of molecules to cellulose with a broad range of properties, thanks to its three fixating protein structures representing the three heads of the system.</P><P>We ran our tests on two different types of cellulose: a commercial Avicel was used to make colloidal particles named Regenerated Amorphous Cellulose (RAC), and bacterial cellulose produced by the bacteria <I>Gluconacetobacter hansenii</I>. Our three proteins (Sirius, Orthos and Cerberus) were designed to be produced in two chassis:<I> Escherichia coli</I> and <I>Pichia pastoris</I>. We chose to work with both a bacterium and a yeast because they both have their advantages.<I> E. coli </I>is a well-studied microorganism with a fast growth rate while <I>P. pastoris </I>is antibiotic-resistant, an ability that comes in handy when wanting to produce and antibacterial peptide, and it has a strong secretion capability. In order to validate our protein design we built a workflow, integrating <I>in silico</I>, <I>in vivo</I> and <I>in situ</I> validation steps as showed on Figure 1.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 1:</STRONG> Overview of our experimental procedure</I></FIGCAPTION></FIGURE><H3 class="heavy">Choosing the First Head</H3><P>The platform binds to every type of cellulose through its protein domain of the type 3 Carbohydrate Binding Module (CBM3a) family. The Imperial iGEM team from 2014 proved that of all the CBMs, it has one of the highest specificities for crystalline cellulose. Also, as it comes from the bacterium <I>Clostridium thermocellum</I>, it is thermostable. </P><H3 class="heavy">Choosing the Second Head</H3><P>The first linker consists in a streptavidin head, which enables us to bind biotin and biotinylated compounds by using the strong affinity these two molecules have for each other. This interaction is in fact one of the strongest non-covalent bonds found in Nature with a dissociation constant of 10<SUP>-13</SUP> M, which is 10<SUP>3</SUP>-10<SUP>6</SUP> greater than any interaction between a ligand and its specific antibodies.</P><P>Of all the interacting pairs known and used today, we chose the biotin-streptavidin because of its appealing characteristics suitable for a whole range of applications<SUP>1</SUP>. The high affinity ensures that once formed the complex will not be changed by pH variations or washes, and that the binding is highly specific and will only specifically target certain molecules. Due to its small size, biotin does not alter their biological activity or other properties when attached to macromolecules. Finally, what makes streptavidin a great choice for our fusion protein is that it is very stable and its ability to bind to biotin is not easily destroyed even under harsh conditions.</P><P> Also, the fact that biotinylated compounds such as fluorophores, enzymes and proteins are commercially available encouraged us to use this system in our project. All these aspects also explain why the use of the streptavidin-biotin interaction was developed in the field of biotechnology. It allows a broad range of applications, especially in immunological and nucleic acid hybridization assays.</P><P>During our experiments, we decided to use two types of streptavidin: the wild-type which is tetrameric because we have data on its interaction with biotin as several iGEM teams already described the system, and mSA2 which is monomeric and the result of a fusion of streptavidin and rhizavidin because we feared that the tetrameric streptavidin would cause aggregation of Cerberus<SUP>2</SUP>. Both have their pros and cons, for this reason we chose to work with both. Due to its capacity to bind four molecules of biotin per tetramer, the wild-type has an unmatched affinity for biotin. However, the monomeric version mSA2 has a shorter nucleic sequence and it is less prone to have non-specific interactions. Of all the existing non-tetrameric streptavidin, mSA2 has the highest biotin affinity (K<SUB>D</SUB> = 2.8 nM)<SUP>3</SUP>, which increases its ability to bind biotinylated compounds. Additionally, mSA2 is easier to produce as our experiments have shown, so we did not conduct all our proofs of concept with the tetrameric type.</P><H3 class="heavy">Choosing the Third Head </H3><P>The second linker of our three heads system is composed of 4-azido-L-phenylalanine (AzF), an unnatural amino acid, used to covalently bind alkyne-decorated molecules by click chemistry.</P><P>At the beginning of the 21<SUP>st</SUP> century, the emergence of the reaction called ‚Äúclick chemistry‚Äù revolutionized bioconjugate chemistry. Thanks to this, coupling molecular fragments to synthesize conjugates has become easier. Following this, biorthogonal reactions were developed. The most widely used is the copper-catalyzed azide-alkyne cycloaddition reaction, shortened to CuAAC<SUP>4</SUP>. Catalysed by Cu(I) ions, this reaction involves the azide group of our UnAA and an alkyne group. The main problem of this method is that the cytotoxicity of copper on cells limits its use in assays that take place in the cellular environment. To circumvent this problem, the strain-promoted azide‚àíalkyne cycloaddition reaction, shortened to SPAAC, was introduced in 2004 and is used more and more nowadays. This reaction presents the advantage of not needing a catalyst, so it not toxic for the cells, and it can be implemented in physiologic pH conditions (around 7.5).</P><H3 class="heavy">References</H3><I><OL><LI>Diamandis EP, Christopoulos TK: The biotin-(strept)avidin system: principles and applications in biotechnology. Clin Chem 1991, 37:625-636.</LI><LI>Lim KH, Huang H, Pralle A, Park S: Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 2013, 110:57-67. DOI: 10.1002/bit.24605.</LI><LI>Qingqing Li QL, Xiaojian Ma, Yaping Wang, Mengjie Dong, Zhen Zhang and Lixin Ma: High-level expression of biotin ligase BirA from Escherichia coli K12 in Pichia pastoris KM71.</LI><LI>Pickens CJ, Johnson SN, Pressnall MM, Leon MA, Berkland CJ: Practical Considerations, Challenges, and Limitations of Bioconjugation via Azide-Alkyne Cycloaddition. Bioconjug Chem 2018, 29:686-701. DOI: 10.1021/acs.bioconjchem.7b00633.</LI></OL></I></DIV><DIV class="tab-pane fade" id="Design_Part2" role="tabpanel"><H2 class="heavy">Validating Our System</H2><P>Validating each head of our system was the first step we had to go through, even before thinking about adding a whole range of properties. In order to do so, we designed three proteins: Sirius, Orthos and Cerberus. The aim of these proteins is to validate our system one head after the other.</P><H3 class="heavy">Experimental Plan: Sirius</H3><P>The first step of our experimental plan was to prove the affinity of our CBM3a to cellulose. To do that, we designed our first protein, Sirius. It was constructed by fusing CBM3a to mRFP1, which is an engineered version of DsRED but with a monomeric characteristic (Figure 1).</P><DIV class="center"><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 1:</STRONG> Representation of our Sirius protein with its CBM3a fused with mRFP1</I></FIGCAPTION></FIGURE></DIV><P>This part was cloned into pET28 containing a pT7Lac promoter and a <I>C</I>-terminus His-tag. Once the cloning was validated by sequencing, we chose to transform it into <I>E. coli</I> BL21 DE3 because of its high protein production yield. Once produced, Sirius was purified by Immobilized Metal Affinity Chromatography (IMAC) on cobalt resin column, and the different fractions we collected were analysed on SDS-PAGE to make sure our protein was produced at the expected size. </P><P>Then, the binding affinity to cellulose was proved by using regenerated Avicel in a method called pull-down assay (Figure 2). The pull-down assay consists in separating soluble and insoluble compounds by using centrifugation. Two controls were used during the experiment: one with the Tris-HCl buffer and the Avicel, and the other with mRFP1 alone and the Avicel. These controls allow us to prove that the fluorescent protein binds to cellulose through our CBM3a and not directly.  We then measured the fluorescence intensity in our tubes after several washes.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 2:</STRONG> Experimental plan to validate the CBM3a head with our Sirius protein by using pull-down assay</I></FIGCAPTION></FIGURE><H3 class="heavy">Experimental Plan: Orthos</H3><P>After validating the affinity of our CBM3a to cellulose, we moved on the validation of the activity of the streptavidin head with a second protein called Orthos (Figure 3). Orthos is the result of the fusion of binding the molecular domain, CBM3a, and the streptavidin linker, mSA2. This part ends with an amber stop codon, preventing the His-tag from the pET28 environment from being translated.</P><DIV class="center"><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 3:</STRONG> Representation of our Orthos protein with its CBM3a and streptavidin heads. A TEV site (orange) is present just before the streptavidin</I></FIGCAPTION></FIGURE></DIV><P>Between the <I>C</I>-terminus of the streptavidin amino acid sequence and the <I>N</I>-terminus of the linker of the CBM3a, we introduced a TEV-protease site. This insertion may provide some strategy of releasing of the streptavidin and the biotinylated protein interacting with it if required, simply by incubating our complex with TEV protease.</P><P>After validating Orthos by sequencing, we produced it in <I>E. coli</I> BL21 DE3 and purified it by using the cellulose affinity proven with Sirius.</P><P>To study its affinity for biotinylated proteins, we needed to <I>in vivo</I> biotinylate proteins. Thus, we fused mTagBFP with an Avi-tag, and cloned it in pETDuet1 which already had the sequence coding for BirA. BirA is a small protein able to recognise the 15-residues Avi-tag and to biotinylate the molecule attached to it. Once the system was transformed into <I>E. coli</I> BL21 DE3 and induced with IPTG, we used the cellular lysate to prove the affinity of Orthos for biotinylated compounds by measuring the fluorescence after a pull-down assay (Figure 4). The controls used for Orthos were the same ones than for Sirius: one with the Tris-HCl buffer, and one with the biotinylated fluorescent protein.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 4:</STRONG> Experimental plan to validate the streptavidin head with our Orthos protein by using the CBM3a affinity for cellulose and the pull-down assay</I></FIGCAPTION></FIGURE><H3 class="heavy">Experimental Plan: Cerberus</H3><P>Now that two heads were validated, we still had to prove the activity of unnatural amino acid. For that, we used a third protein called Cerberus (Figure 5). Cerberus is a protein of 380 to 386 amino acids (depending on the type of streptavidin chosen) and 41 kDa. The affinity of the CBM3a to crystalline cellulose is around 2.9x10<SUP>7</SUP> M<SUP>-1</SUP>.<SUP>1</SUP></P><DIV class="center"><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 5:</STRONG> Representation of our Cerberus protein with its three binding domains, CBM3a, streptavidin and 4-azido-L-phenylalanine</I></FIGCAPTION></FIGURE></DIV><P>The construction is similar to Orthos‚Äô but to incorporate the UnAA, we had to hack into the genetic code. To do so, we brought an orthogonal aminoacyl tRNA synthase (aaRS)/tRNA pair to <I>E. coli</I>. The amber stop codon present in our construction is now recognised by the tRNA and it becomes an incorporation site for the 4-azido-L-phenylalanine (AzF).</P><P>This strategy was conceived for specific purification because the His-tag will only be attached to AzF-containing proteins. Once validated, the part was co-transformed into <I>E. coli</I> BL21 DE3 with pEVOL-AzF, a plasmid containing tRNA and aaRS from <I>Methylococcus ianaschii</I>. We purified the production by using IMAC purification on cobalt column (Figure 6).</P><P> To validate the activity of the UnAA, we used fluorescein grafted with DBCO to perform SPAAC. This compound clicked with Cerberus‚Äôs head and after several washes we measured the difference of fluorescence between cellulose samples containing or not our Cerberus protein with the fluorophore.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 6:</STRONG> Experimental plan to validate the AzF head with our Cerberus protein by using pull-down assay</I></FIGCAPTION></FIGURE><H3 class="heavy">References </H3><OL><LI><I>Tomme P, Boraston A, McLean B, Kormos J, Creagh AL, Sturch K, Gilkes NR, Haynes CA, Warren RA, Kilburn DG: Characterization and affinity applications of cellulose-binding domains. J Chromatogr B Biomed Sci Appl 1998, 715:283-296.</I></LI></OL></DIV><DIV class="tab-pane fade" id="Design_Part3" role="tabpanel"><H2 class="heavy">The Applications of Our System</H2><H3 class="heavy">Paramagnetic Beads</H3><P>We first aimed to bind paramagnetic beads on the UnAA with SPAAC reaction, imagining that we could this way create magnetic cellulose that could be used as a new way to extract compounds in cells, for example, by biotinylating and binding the molecule of interest on the streptavidin head. We bought beads already attached to a DBCO function, and we put them in presence of our Cerberus under agitation overnight to allow the click reaction to occur. The next day, a pull down-assay is made, and the magnetism is observed after several washes by putting the tubes next to a magnet and by observing the functionalised cellulose moving towards it (Figure 1). The three controls were the Tris-HCl buffer, Cerberus alone, and the paramagnetic beads alone.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 1:</STRONG> Experimental plan for testing the paramagnetic beads on the AzF head</I></FIGCAPTION></FIGURE><H3 class="heavy">Antibacterial Peptide, scygonadin</H3><P>The thought of having an antibacterial bandage is very appealing and it becomes more interesting with the TEV site we added. This small site enables us to release the antibacterial molecule directly into the wound just by adding a protease. After a week of culturing <I>Pichia pastoris</I> engineered to produce the scygonadin, the supernatant is collected after centrifugation and put with our Cerberus overnight under agitation. A pull-down assay is realised to purify the complex Cerberus-scygonadin.  Then, the sample is put on a paper disk on a Petri dish coated with bacteria. The inhibition halos are observed the next day after incubation (Figure 2). The three controls are Cerberus, scygonadin and an antibiotic.</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 2:</STRONG> Experimental plan for testing scygonadin on a petri dish coated with bacteria (1: scygonadin; 2: scygonadin bound to Cerberus; 3: Cerberus; 4: antibiotic)</I></FIGCAPTION></FIGURE><H3 class="heavy">Testing the TEV site</H3><P>This experiment is the same as the validation of our Orthos protein, except that one of the washes of the pull-down assay is replaced with a solution containing the TEV protease (see our Experiments page for further details) and that we use our Cerberus protein instead. The protein is incubated with the protease for 2 hours at 30¬∞C. The fluorescence is then observed after centrifugation, showing that it is now present in the supernatant and not in the pellet as it used to be, showing and the biotinylated fluorescent protein was cut off from the rest of our protein. In the elution fraction, no fluorescence should be observed and measured as it was in the supernatant, and hence replaced with Tris-HCl buffer. The negative control is the biotinylated protein with cellulose. This test was performed at the end of our time in the lab, which explains why we did not have time to optimize the protocol after the first try.</P><H3 class="heavy">Binding Two Different Molecules at Once</H3><P>Cerberus is designed to bind a wide variety of molecules, but it is also possible to bind two different molecules on its streptavidine and AzF heads. Interestingly, it is also possible to invert the function of its heads with small adapter molecules like biotin with a DBCO moeity. To illustrate both possibilities, we decided to bind the paramagnetic beads on the AzF head, then a DBCO-Biotin on the streptavidin head and finally a fluorescein-azide molecule on the newly added DBCO moeity (Figure 3).</P><FIGURE class="figure" style="text-align:center;"><FIGCAPTION class="figure-caption"><I><STRONG>Figure 3:</STRONG> Experimental plan for binding FAM-azide on the streptavidin head and DBCO-Paramagnetic beads on the AzF head</I></FIGCAPTION></FIGURE><P>The click chemistry between the UnAA and the paramagnetic beads was made overnight, then a pull-down assay is realised to purify it. The molecules DBCO-biotin and FAM-azide are then added, and a pull-down assay is made. The magnetism is once again observed with a magnet, but this time on an UV bench so we can observe the fluorescent cellulose.</P><H3 class="heavy">Graphene and Carbon Nanotubes</H3><P>Binding graphene and carbon nanotubes to cellulose could bring interesting physical properties to it like rigidity. To bind these two compounds to our UnAA, they first needed to be activated by a reaction of diazotisation. The activation is validated by a Thermogravimetric Analysis (TGA) where the mass loss is measured to prove that the function was added in the first place. The reaction between the alkyne group and the azide function of the UnAA is still a click-chemistry but it is now copper-catalyzed (CuAAC). We only had time for the activation of the graphene and carbon nanotubes, hence we did not conduct the CuAAC reaction to functionalise our cellulose.</P><H3 class="heavy">FRET</H3><P>To validate the distance between the streptavidin and the AzF heads calculated by 3D molecular modelling, we thought about using F√∂rster resonance energy transfer (FRET). We aimed to bind fluorescein on the AzF head and biotinylated BFP on the streptavidin head. Fluorescein would have been the donor chromophore and BFP the acceptor chromophore. The distance between the two heads is around 6nm (see our Modelling page), so the FRET would have been observed as its range is usually between 1 and 10nm. Due to a lack of time, we have not been able to perform this test.</P></DIV><DIV class="tab-pane fade" id="Design_Part4" role="tabpanel"><H2 class="heavy">Cerberus: a Versatile System</H2><H3 class="heavy">Primary Usage: Adding New Functions to Cellulose</H3><P>Cerberus was first thought of for binding a variety of compounds thanks to its two very dissimilar linkers. The streptavidin linker can associate biotin and biotinylated compounds while the UnAA-containing linker will click molecules that possess an alkyne function. Using one of these heads and the CBM3a head allows to functionalise cellulose and opens the way to easily create new innovative materials.</P><H3 class="heavy">Secondary Usage: Combination of the Three Heads</H3><P>Besides adding a function, the structure of Cerberus also permits to add two functions at the same time on cellulose (for example, magnetism and fluorescence). Such combinations expand even more the possibilities of the platform. To illustrate, imagine a fabric with Cerberus with both color and antibiotic properties: you can track if the antibiotic is still present just by assessing the color of the fabric.</P><H3 class="heavy">Third Usage: Forget About Cellulose!</H3><P>The possibilities of Cerberus can go over the cellulose application field. Indeed, one could imagine applications based solely on the AzF and streptavidin heads. For example, if a bacterium has been modified to express biotin at its surface, we should be able to remove it from a culture with paramagnetic beads fixed on Cerberus' AzF head. For such applications, the CBM3a is not interesting for its cellulose binding property but for the stability it offers to the whole structure.</P></DIV><DIV class="tab-pane fade" id="Design_Part5" role="tabpanel"><H2 class="heavy">Functionalising Bacterial Cellulose in a Bioreactor</H2><P>A final goal of our project was to demonstrate Cerberus' ability to functionalise cellulose <I>in vivo</I>. After validating each head separately and the ability of our platform to functionalise commercial cellulose, we had to make sure that the bacterial cellulose produced by <I>Gluconacetobacter hansenii</I> could be functionalised. To do so, we agitated a thin piece of bacterial drown in a purified fraction of Sirius and washed it several times to observe that the cellulose remains pink, demonstrating that our protein remains bound to it. The negative control used is the bacterial cellulose drown in the same fluorescent protein but without fusing it with our CBM3a. Normally, after washing this cellulose, the colour pink should not remain as the protein does not naturally bind to cellulose.</P><P> Once that the test is validated, we can imagine putting our chassis <I>Pichia pastoris</I> and <I>Gluconacetobacter hansenii</I> together in a bioreactor, so they can respectively produce our linker with the desired function and the bacterial cellulose. Tests were run to see if the two microorganisms could grow together but due to the acidification of the medium by <I>G. hansenii</I>, the results were not conclusive. This part remains to be optimised and other experimental conditions needs to be tested. For example, as <I>P. pastoris</I> growth rate is slower than <I>G. hansenii</I>, we could test putting one after the other, and we could have a system permitting to keep the pH stable.</P></DIV></DIV></DIV></DIV></DIV></DIV></MAIN></DIV><DIV class="container-fluid bg-secondary shadow-lg text-light" id="FOOTER"><DIV class="row"><DIV class="col-10 text-justify" id="FOOTER_CONTENT"><DIV id="DISCLAIMER"><I><P><I>No dogs were harmed over the course of this iGEM project.</I></P><P class="sponsor">The whole <B>Toulouse INSA-UPS team</B> wants to thank our sponsors, especially:</P><P class="sponsor">And many more. For futher information about our sponsors, please consult <B><U><A href="https://2018.igem.org/Team:Toulouse-INSA-UPS/Sponsors" class="footer">our Sponsors page</A></U></B>.</P><P>The content provided on this website is the fruit of the work of the <B>Toulouse INSA-UPS</B> iGEM Team. As a deliverable for the iGEM Competition, it falls under the Creative Commons Attribution 4.0. Thus, all content on this wiki is available under the Creative Commons Attribution 4.0 license (or any later version). For futher information, please consult the  official website of Creative Commons.</P><P>This website was designed with Bootstrap (4.1.3). Bootstrap is a front-end library of component for html, css and javascript. It relies on both Popper and jQuery. For further information, please consult the official website of Bootstrap.</P></I></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>