<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU_China_Project_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU China/Project Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
            Visual selection system
        </TITLE></P><DIV class="main-indents"><DIV class="my_main"><SECTION id="content"><DIV style="width=960px;padding: 0px; margin: 0; background-color: rgb(255,255,255); color: #000;"><HEADER><NAV style=""><UL class="sf-menu"><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                    Home
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                            Abstract
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Background">
                                            Background
                                        </A></LI></UL><LI class="current"><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                    Project &amp; Results

                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                            Visual selection
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Project2">
                                            Optimization
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Parts">
                                            Parts
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Notebook">
                                            Notebook
                                      </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Model">
                                    Modeling
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices">
                                    Human Practice
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Coloring" style="cursor:pointer;">
                                            Coloring book
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Construction" onclick="$.scrollTo('#Construction',500)" style="cursor:pointer;">
                                            Construction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Counseling" onclick="$.scrollTo('#Counseling',500)" style="cursor:pointer;">
                                            Counseling
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Collaborations" onclick="$.scrollTo('#Collaborations',500)" style="cursor:pointer;">
                                           Collaborations
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Conferences" onclick="$.scrollTo('#Conferences',500)" style="cursor:pointer;">
                                           Conferences
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Safety">
                                    Biosafety
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                    Team
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                            Introduction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Team/Members">
                                            Team Members
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Attributions">
                                            Attributions
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                    Judging
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                            Medal Criteria
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging/Engagement">
                                            Education &amp; Public Engagement award
                                        </A></LI></NAV></HEADER><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;margin-bottom:20px;
                        filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                        -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                        -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                        box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Background" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90"><B>Establishment of a markerless visual selection system for knocking devices into the genome of <I>L. lactis</I> (Visual selection system)</B></P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Background</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
                        Currently, most of the devices are introduced into <I>L. lactis</I> by plasmids. Various vectors containing constitutive or inductive promoters have been developed, such as pMG36e, pNZ8148. However, the devices introduced by plasmids had two disadvantages. First, most plasmids have selection markers (genes) for antibiotic resistance, which is forbidden to be used in food or digestive tract of human and animals directly. Second, the plasmids were unstable in cytoplasm in the absence of selection pressure. 
                    </P><P style="font-family:verdana;font-size:17px;">
                        The alternative choice is knocking the devices into the genomes of <I>L. lactis</I>. Double cross over recombination using non-replicative or conditional replicative plasmids is the widely used strategy. For instance, using temperature sensitive plasmid, at restrictive temperature, single-crossover integration of devices can be selected by certain antibiotics. Under lower temperature and in the absence of antibiotics, the second cross-over is occurred at a very low rate and the desired device knocked-in strains can be obtained. However, to get desired device knocked-in strains, antibiotic resistance of clones was checked, and only a few clones were antibiotic susceptible. The antibiotic susceptible clones were then verified by PCR and less than half of them were the desired device knocked-in strains, the others were the parental strains. (Figure 1.1)
                    </P><DIV style="text-align:center;margin-bottom:20px"><P id="Design" style="text-align:center;">Figure 1.1 Schematic of device integration. Abbreviations: LA, left arm for integration; RA, right arm for integration; ARG, antibiotic resistant gene.</P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Design</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
                        To get device knocked-in recombinant <I>L. lactis</I> without antibiotic resistant genes and make this process easy and highly efficient, we try to establish a visual selection system based on the insoluble blue compounds generated by the <I>lacZ</I> gene encoded β-galactosidase catalyzed X-gal (also abbreviated BCIG for 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside) hydrolysis. To build this system, we chose the His locus as the targeting locus. The His locus is required for the synthesis of histidine, an essential amino acid for <I>L. lactis</I> growth. The recombinant <I>L. lactis</I> strains can only be survived with the supplementation of histidine in the culture but cannot survive without histidine. The His fragment was introduced into a temperature-sensitive conditional replicative shuttle plasmid and we named this plasmid as pHis. Then we placed the <I>lacZ</I> gene driven by the <I>PnisZ</I> promoter into the His locus and we name this plasmid as pBlue. The next step is to integrate the <I>lacZ</I> gene into the genome of NZ9000 to yield the NZ-Blue recombinant strain. The <I>lacZ</I> gene encoded β-galactosidase can hydrolyze the X-gal from colorless substance to insoluble blue compounds, thus NZ-blue strain forms blue colonies on X-gal plate supplemented with nisin. Finally, the device carrying plasmid based on pHis (pDevice) will be introduced to NZ-Blue to replace the <I>lacZ</I> gene and the desired device knocked-in strains will be white when cultured with X-gal and supplemented with nisin. (Figure 1.2)
                    </P><DIV style="text-align:center;margin-bottom:20px"><P id="Result" style="text-align:center;">Figure 1.2 The markerless integration visual selection system for <I>L. lactis</I>. A). Map of the integration plasmid pHis; B). Map of the integration plasmid pBlue; C). Schematic of the construction of NZ-Blue; D). Schematic of high-efficiency device integration selection. Abbreviations: His, integration fragment from <I>L. lactis</I>; HisLA, left arm of His; HisRA, right arm of His; Ts Ori, temperature-sensitive replicon from pCrePA2; <I>Em<SUP>r</SUP></I>, erythromycin resistant gene; <I>Amp<SUP>r</SUP></I>, ampicillin resistant gene; <I>P</I>, <I>PnisZ</I>, the promoter of <I>lacZ</I>; <I>lacZ</I>, <I>β</I>-galactosidase from <I>Lactobacillus acidophilus</I>.
                            </P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Result</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
                        The temperature-sensitive conditional replicative plasmid pHis was used to introduce the ‘<I>PnisZ:: lacZ::Terminator</I>’ fragments into the NZ9000 chromosome of <I>L. lactis</I>. The constructed plasmid pBlue was transformed into NZ9000 by electroporation. To obtain NZ-Blue via homologous sequences (HisLA or HisRA), the NZ9000-pBlue strains were grown in M17GS (M17 broth with 0.55 % (w/v) sucrose and 0.5 % (w/v) glucose) containing erythromycin and incubated at 38.5 °C overnight to integrate the pBlue DNA sequence into the genome of NZ9000 through single cross over. Then, the cultures were diluted in M17GS medium without erythromycin and incubated at 28 °C to promote the double cross over and excision of the antibiotic resistance genes. The subcultures were diluted and plated on M17GS with X-Gal and nisin. Blue colonies obtained by serial plating were screened for erythromycin susceptibility. Colonies in which <I>lacZ</I> gene integration had occurred will appear as blue on X-gal plate supplemented with nisin and erythromycin susceptible. The erythromycin susceptible and blue colonies selected from thousands of colonies were further subject to PCR verification using the extracted genomic DNA as PCR template. DNA sequencing was also used to confirm that no genetic mutations were introduced during the experiment process. After this difficult and tedious task, we successfully got the <I>lacZ</I> integration strain and we name it NZ-Blue. (Figure 1.3)
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">  Figure 1.3 Confirmation analysis of the constructed NZ-Blue strain. A). Schematic of genomes of NZ9000 and NZ-Blue. Abbreviations: LO, RO, LF, LR, primers used in Multiple-PCR analysis; HisLA, left arm of His; HisRA, right arm of His; <I>PnisZ</I>, promoter induced by nisin; <I>lacZ</I>, β-galactosidase from <I>Lactobacillus acidophilus</I>; <I>Ter</I>, terminator. B). Multiple-PCR analysis of the NZ-Blue and NZ9000. The primer combinations used in PCR are presented upon the lanes. Genomic DNA used in PCR are presented upon the lines. C). Colony of NZ-Blue. The 1 kb DNA ladder marker is shown to the left (M).
							</P></DIV><P style="font-family:verdana;font-size:17px;">
                        To test the efficiency and capacity of our markerless visual selection system for knocking devices into the genome of <I>L. lactis</I>, 5 devices with different lengths (ranging from 1 kb to 14 kb) were PCR amplified and inserted into the pHis vector. The 5 constructed pDevice plasmids were transformed into NZ-blue strain through electroporation, respectively. To obtain integrant resulting from a single-crossover event via homologous sequences (HisLA or HisRA), the strains containing the plasmids were grown in M17GS containing erythromycin and incubated at 38.5 °C overnight. Then, the cultures were diluted in M17GS medium without antibiotic and incubated at 28 °C to obtain double crossover and excision of the antibiotic resistant genes. The subcultures were diluted and plated on M17GS with X-Gal and nisin. According to our design, the white colonies will be the ones that the devices have been knocked into the genome of NZ-Blue. At this time, we found that among the pool of blue colonies, usually none or one to several white colonies can be easily identified in the plates, indicating the double crossover is really occurred at a very low rate. Multiple-PCR analysis and DNA sequencing confirm the white colonies were the right ones. As expected, 100% of the white colonies are the devices successfully knocked-in strains.
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure 1.4 Characterization of 5 different length devices recombinant <I>L. lactis</I>. A). Schematic representation of the 5 different length devices, theoretical size (kb) of each of the PCR products generated with the primer combinations was shown. B). Representative White colonies from the pool of blue colonies. C). Confirmation analysis of the 5 different length devices recombinant <I>L. lactis</I> by multiple-PCR analysis. The primer combinations used in PCR are presented upon the lanes. The 1 kb DNA ladder marker is shown to the left (M1) and λ/HindIII marker is shown to the right (M2).
					        </P></DIV><P style="font-family:verdana;font-size:17px;">
                        We further summarized the double cross over occurrence rate and the accuracy rate of obtaining these 5 different devices knocked-in strains. As shown in Table 1, though the positive rate is really low in the 10<SUP>-5</SUP> to 10<SUP>-4</SUP> range, the accurate rate is 100%. In conclusion, our makerless visual selection system can effectively integrate devices ranging from 1 kb to 14 kb. Furthermore, the white one is the right one and the accuracy rate is 100%. How easy and wonderful!
                    </P><P style="font-family:verdana;font-size:17px;">
					    Now, the visual selection system including NZ-Blue strain and pHis plasmid were constructed.
					</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;font-family:verdana;font-size:17px;">Table 1 The positive rate and accuracy rate of 5 different devices knocked-in <I>L. lactis</I> strains
			                </P></DIV><P style="font-family:verdana;font-size:17px;">
					    To confirm whether these devices integrated in genome of NZ-Blue are functional, we verified the shortest device, the mCherry device. The mCherry device can give red fluorescence after stimulation, we found that all the 58 white colonies appeared on the X-gal plate gave red fluorescence, demonstrated that all the white colonies are the right ones. (Figure 1.5)
					</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
							    Figure 1.5 Functional verification of the mCherry device. A) Schematic representation of the mCherry device. B) The 58 mCherry device knocked in strains on X-gal plate supplemented with nisin appeared as white colonies. C) The 58 mCherry device knocked in strains on plate supplemented with erythromycin cannot grow. D) After stimulation, the 58 mCherry device knocked in strains gave red fluorescence.
					        </P></DIV><P style="font-family:verdana;font-size:17px;">
					    We also verified the longest device, the polysaccharide <I>viaB</I> device, which can alter the surface property of the cell. We found that the NZ-Vi strain exhibited as clumps. Under microscope, a transparent zone around the bacteria can be easily observed, indicating the polysaccharide capsule is successfully expressed. We conclude that the devices knocked in are functional. (Figure 1.6)
					</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
							    Figure 1.6 Confirmation analysis of the constructed NZ-Vi strain. A). Schematic the NZ-Vi. abbreviations: LO, RO, PzRec, TerRec, primers used in Multiple-PCR analysis; HisLA, left arm of His; HisRA, right arm of His; <I>PnisZ</I>, promoter induced by nisin; <I>nsr,</I> nisin resistant gene; <I>Ter</I>, terminator; <I>tviA, tviB, tviC, tviD, tviE, vexA, vexB, vexC, vexD, vexE</I>, gene of the <I>viaB</I> operon. B). View of the NZ-Blue and NZ-Vi culture. C) Microscopic view of NZ-Blue and NZ-Vi after staining.
					        </P></DIV></DIV><P id="Reference" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90"><B>References</B></P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P> [1] Delorme, C., Ehrlich, S.D., and Renault, P. (1992). Histidine biosynthesis genes in Lactococcus lactis subsp. lactis. J Bacteriol 174, 6571-6579.
[2] Pinto, J.P., Zeyniyev, A., Karsens, H., Trip, H., Lolkema, J.S., Kuipers, O.P., et al. (2011). pSEUDO, a genetic integration standard for Lactococcus lactis. Appl Environ Microbiol 77, 6687-6690.
[3] Simoes-Barbosa, A., Abreu, H., Silva Neto, A., Gruss, A., and Langella, P. (2004). A food-grade delivery system for Lactococcus lactis and evaluation of inducible gene expression. Appl Microbiol Biotechnol 65, 61-67.
[4] Solem, C., Defoor, E., Jensen, P.R., and Martinussen, J. (2008). Plasmid pCS1966, a new selection/counterselection tool for lactic acid bacterium strain construction based on the oroP gene, encoding an orotate transporter from Lactococcus lactis. Appl Environ Microbiol 74, 4772-4775.
[5] Wetter, M., Goulding, D., Pickard, D., Kowarik, M., Waechter, C.J., Dougan, G., et al. (2012). Molecular characterization of the viaB locus encoding the biosynthetic machinery for Vi capsule formation in Salmonella Typhi. PLoS One 7, e45609.
</P></DIV></DIV></DIV></SECTION><FOOTER><DIV style="padding:20px 0 0 0;margin:0;float:right;width:300px;color:#fff;"><UL style="font-size:14px;"><LI><SPAN style="text-decoration:underline;color:#fff;left:20px;">Contact us</SPAN></LI><LI><SPAN>Email:</SPAN><A href="Mailto:igem@tmmu.edu.cn" style="color:#fff">igem@tmmu.edu.cn</A></LI><LI><SPAN>Address:</SPAN> Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV><DIV class="card-holder hide"><DIV class="card-wrapper"><A onclick="$.scrollTo('#Background',500)"><DIV class="card"><SPAN class="card-content">
                            Background
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Design',500)"><DIV class="card"><SPAN class="card-content">
                            Design
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Result',500)"><DIV class="card"><SPAN class="card-content">
                            Result
                        </SPAN></DIV></A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>