Team:TMMU China/Project Results/Project2
Optimized NICE
Home
Abstract
Background
Project & Results
Visual selection Optimization Parts Notebook
Modeling
Human Practice
Coloring book Construction Counseling Collaborations Conferences
Biosafety
Team
Introduction Team Members Attributions
Judging
Medal Criteria
Education & Public Engagement award
Optimization of the nisin resistance of device integrated NZ9000
Background
Nisin immunity genes and nisin resistant genes
The NICE system inducer, nisin, is a 34-residue food grade antibacterial peptide produced by some strains of L. lactis which is active against wide range of gram-positive bacteria and is  widely used in the food industry as a preservative. Nisin inhibits growth of Gram-positive cells in two ways (Figure 2.1). First, nisin inhibits cell wall synthesis by binding to the lipid II molecule in the membrane. Second, nisin molecules can span across the lipid bilayer to make a pore in the cell membrane which causes the leakage of cytoplasm. The L. lactis NZ9000 strain has no resistant mechanism and cannot tolerate high levels of nisin concentration in the medium.
Figure 2.1 Antibacterial mechanisms of nisin.
In nisin producing strains, the nisin operon contains four nisin immunity genes, nisI, nisF, nisE and nisG. Among them, nisI encodes a lipoprotein that serves as the first defense for nisin immunity. (Figure 2.2A) In non-nisin-producing L. lactis strains, nisin resistance could be conferred by a specific nisin resistance (nsr) gene, which encodes a 35-kDa nisin resistance protein (NSR, Figure 2.2B). NSR could proteolytically inactivate nisin by removing six amino acids from the carboxyl “tail” of nisin. The truncated nisin (nisin 1–28) displayed a markedly reduced affinity for the cell membrane and showed significantly diminished pore-forming potency in the membrane. A 100-fold reduction of bactericidal activity was detected for nisin 1–28 in comparison to that for the intact nisin. What’s more, the truncated nisin can still induce gene expression through the NICE system. Thus, the incorporation of the nisI or nsr gene into the NZ9000 will lead to tolerance of NZ9000 at high levels of nisin concentration.
Figure 2.2 The structures of nisin immunity protein NisI (A) and the nisin resistance protein NSR (B).
Design
The NZ9000 or NZ-Blue strain (nisKR integrated) is sensitive to nisin. We inferred that the introduced of the nisI or nsr gene might render NZ9000 to tolerate high levels of nisin concentration while maintain the induced gene expression capacity of the NZ9000. To achieve the goal, we constructed two plasmids (Figure 2.3) to test whether the incorporation of nisI or nsr can really increase the nisin tolerance of NZ9000 and NZ-Blue. The expression of nisI or nsr gene is driven by the P32 promoter, a constitutive promoter.
Figure 2.3 Map of plasmid pMG36e-nsr (A) and pMG36e-nisI (B). Abbreviations: P32, promoter; nsr, nisin resistant gene; nisI, nisin immunity gene; Ter, terminator; Emr, erythromycin resistant gene; Ori, replicon.
Result
The two plasmids were successfully constructed, and were transformed into NZ9000. After that we measure the growth curve of NZ9000, pMG36e-nisI/NZ9000 and pMG36e-nsr/NZ9000 under different concentrations of nisin. As shown in Figure 2.3, without nisin, the growth curve of these 3 strains exhibits no difference. However, under 200 IU/mL nisin concentration, the growth of NZ9000 is severely inhibited, the pMG36e-nisI/NZ9000strain can still grows to a moderate level, while the pMG36e-nisI/NZ9000 strain can still grow to a level similar to that without nisin, which means the NSR protein can protect L. lactis at high nisin concentrations, while the protect efficiency of NisI is not as pronounced as NSR. Finally, under 500 IU/mL nisin concentration, only the pMG36e-nsr/NZ9000 strain can still grow, the NZ9000 and pMG36e-nisI/NZ9000 strain cannot grow. Based on these results and to make it versatile, we incorporated the nsr gene into the pDevice vector. This time, the device of interest integrated in genome of NZ9000 can achieve high expression level in the presence of high nisin concertation.
Figure 2.4 Growth Curve of NZ9000 (■), pMG36e-nisI/NZ9000 (△) and pMG36e-nsr/NZ9000 (◇) in different nisin concentrations. Abbreviations: IU, nisin international unit.
References
[1] Hacker, C., Christ, N.A., Duchardt-Ferner, E., Korn, S., Gobl, C., Berninger, L., et al. (2015). The Solution Structure of the Lantibiotic Immunity Protein NisI and Its Interactions with Nisin. J Biol Chem 290, 28869-28886. [2] Khosa, S., Lagedroste, M., and Smits, S.H. (2016). Protein Defense Systems against the Lantibiotic Nisin: Function of the Immunity Protein NisI and the Resistance Protein NSR. Front Microbiol 7, 504. [3] Sun, Z., Zhong, J., Liang, X., Liu, J., Chen, X., and Huan, L. (2009). Novel mechanism for nisin resistance via proteolytic degradation of nisin by the nisin resistance protein NSR. Antimicrob Agents Chemother 53, 1964-1973.
Optimization of the protein localization system
Background
The localization of protein will affect its function. To make protein function at its right place, the protein localization system should be optimized. To secrete the protein out of the cell, a signal peptide of Usp45 protein was ligated to the target protein. The secretion is very efficiently.  Fusion of the signal peptide of Usp45 to a couple of proteins resulted in an efficient way of secreting the coded proteins into the media. The fusion protein will meet a chaperone protein, SecB, which bring the fusion protein to a protein secretion apparatus. This apparatus cleaves the signal peptide and frees the proteins from the cell.
To make protein of interest to be cell surface displayed, anchoring motif should be considered at the first. The most exploited anchoring regions are those with the LPXTG motifs that bind the proteins in a covalent way to the cell wall. However, to achieve versatile application, it is better to use anchoring domains that interact with the cell wall in a non-covalent way. The major autolysin of L. lactis, the cell wall hydrolase AcmA contains 3 tandem arranged LysM motifs and separated by stretches of 21 to 31 amino acids, this region is collectively termed as the cA anchoring domain. The cA domain can be fused to the N- and C- terminus of functional proteins, and can bind proteins to the cell walls of a broad range of gram-positive bacteria. The cA domain could also be used to immobilize soluble enzymes in an active form on the cell surface of L. lactis. Like AcmA, these fusion enzymes could bind to L. lactis cells when added from the outside. Thus, cA fusion proteins produced using other expression systems can also be attached to the cell wall of L. lactis.
Figure 3.1 Schematic representation of the protein localization system.
Design
To verify the function of SPusp45 and the cA anchoring domain, here we designed three devices, the NZ-Cytosol, NZ-Secretion and NZ-Surface devices as shown in Fig 3.2. We took the β-galactosidase protein as an target protein. The β-galactosidase protein is fused to the SPusp45 signal peptide or fused to the cA domain with the SPusp45 signal peptide, driven by the PnisZ promoter and followed by the nisin resistant gene nsr.
Figure 3.2 Design of the protein localization devices. A) NZ-Cytosol, B) NZ-Secretion, C) NZ-Surface. Abbreviations: PnisZ, promoter; SPusp45, signal peptide of usp45 from L. lactis; cA, anchor motif of acmA from L. lactis; nsr, nisin resistant gene; Ter, terminator.
Result
To demonstrate whether the SPusp45 signal peptide and the cA anchoring domain could indeed secret and anchor the protein of interest at the surface of L. lactis, the placZ-Cytosol and placZ-Surface plasmids were introduced into NZ9000 respectively, which were named NZ-Cytosol and NZ-Surface. As stated above, the cA domain come from the AcmA protein, which is an autolysin of L. lactis that cleaves the peptidoglycan to release the duplicated bacteria. Since the location sites of AcmA autolysin, substrate peptidoglycan, is now occupied by the β-galactosidase-cA fusion proteins secreted by SPusp45, the autolysin activity is hindered, thus cell separation will be interfered. Indeed, we found that under microscopic, the NZ-Surface cells were poorly separated compared to the NZ-Cytosol strain. Further more, using a polyclonal antibody against β-galactosidase, we found that the β-galactosidase protein is present at the cell wall and cytoplasm, which means the signal peptide, SPusp45, and anchor motif, cA domain, worked as we designed. In contrast, all the β-galactosidase proteins were present in the cytoplasm in the NZ-cytosol strain. (Figure 3.3)
Figure 3.3 The functional verification of the NZ-Cytosol and NZ-Surface strains. A). Microscopic picture of the NZ-Cytosol and NZ-surface strains. B). Western blot of different fractions from NZ9000, NZ-Cytosol and NZ-Surface. The protein fused with cA motif was located on the bacterial cell wall and the cytosol expressed protein was located in the bacterial cytoplasm. This mean the protein fused with cA motif was secreted by SPusp45 and was anchored in the cell wall by cA motif. The lacZ was used as an example. The strain used in western blot are presented upon the lanes.
To combined the markerless visual selection system, the improved the nisin resistant of device integrated NZ9000 and the optimized protein localization system together, four plasmids were constructed (Figure 3.4). First, the promoter (double promoter, P32 (constitutive promoter) and PnisZ (inductive promoter)), MCS (multi-clone site), nsr with RBS and terminator were inserted in the AscI site of pHis. The constructed plasmid was named pCyt. Second, the signal peptide SPusp45 was inserted in the site between PnisZ and MCS by seamless cloning. The constructed plasmid was named pSec. Last, the cA domain was inserted the sites in front of MCS and after MCS. The constructed plasmid was named pSD.1 and pSD.2 to fuse the cA domain at the N-terminus or C-terminus of target protein. All plasmid can be linearized by SmaI and XbaI site of MCS, and all devices inserted in MCS of the 4 plasmids were recommended by using seamless cloning.
References
[1] Buist, G., Steen, A., Kok, J., and Kuipers, O.P. (2008). LysM, a widely distributed protein motif for binding to (peptido)glycans. Mol Microbiol 68, 838-847. [2] Kalyanasundram, J., Chia, S.L., Song, A.A., Raha, A.R., Young, H.A., and Yusoff, K. (2015). Surface display of glycosylated Tyrosinase related protein-2 (TRP-2) tumour antigen on Lactococcus lactis. BMC Biotechnol 15, 113. [3] Le Loir, Y., Nouaille, S., Commissaire, J., Bretigny, L., Gruss, A., and Langella, P. (2001). Signal peptide and propeptide optimization for heterologous protein secretion in Lactococcus lactis. Appl Environ Microbiol 67, 4119-4127. [4] Lee, S.Y., Choi, J.H., and Xu, Z. (2003). Microbial cell-surface display. Trends Biotechnol 21, 45-52. [5] Raha, A.R., Varma, N.R., Yusoff, K., Ross, E., and Foo, H.L. (2005). Cell surface display system for Lactococcus lactis: a novel development for oral vaccine. Appl Microbiol Biotechnol 68, 75-81. [6] Saleem, M., Brim, H., Hussain, S., Arshad, M., Leigh, M.B., and Zia ul, H. (2008). Perspectives on microbial cell surface display in bioremediation. Biotechnol Adv 26, 151-161.
Summary
With the NZ-Blue strain we created and the 4 plasmids (pCyt, pSec, pSD.1 and pSD.2) we provided, we can get devices knocked into L. lactis without antibiotics resistance genes. The recombinant strains can express protein by constitutive promoter (P32) and inductive promoter (PnisZ). If we use nisin to promote the PnisZ promotor, the resistant gene nsr will make the devices integrated strains to tolerate high levels of nisin concentration and protein of interest will functionally expressed at its right place. All these work made L. lactis the Mr. Right chassis in synthetic biology.
Figure 3.4 The improvements of the protein localization system. A). Integration plasmid for cytosol expression, pCyt (Left), and the device-inserted plasmid pCyt-Device (Right). B). Integration plasmid for secretion expression, pSec (Left), and the device-inserted plasmid pSec-Device (Right). C). Integration plasmid for surface display, pSD.1/pSD.2 (Left), and the device-inserted plasmid pSD.1-Device/pSD.2-Device (Right). Abbreviation: HisLA, left arm of His; HisRA, right arm of His; P, double promoter, P32 (constitutive promoter) and PnisZ (inductive promoter); MCS, multi-clone site; SPusp45, signal peptide of usp45 from L. lactis; cA; anchor motif of acmA from L. lactis; nsr, nisin resistant gene; Ter, terminator; Ts Ori, temperature-sensitive replicon; Emr, erythromycin resistant gene. All devices were cloned into the SmaI and XbaI site of MCS of the 4 plasmids by seamless cloning.
Contact us
Email: igem@tmmu.edu.cn
Address: Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
1-Background
1-Design
1-Result
2-Background
2-Design
2-Result
Summary
