<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU_China_Project_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU China/Project&amp;Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
            Project
        </TITLE></P><DIV class="main-indents"><DIV class="my_main"><SECTION id="content"><DIV style="width=960px;padding: 0px; margin: 0; background-color: rgb(255,255,255); color: #000;"><HEADER><NAV style=""><UL class="sf-menu"><LI><A href="Home.html">
                                    Home
                                </A></LI><LI><A href="Home.html">
                                            Abstract
                                        </A></LI><LI><A href="Background.html">
                                            Background
                                        </A></LI></UL><LI class="current"><A href="Project0.html">
                                    Project &amp; Results

                                </A></LI><LI><A href="Project0.html">
                                            project0
                                        </A></LI><LI><A href="Project1.html">
                                            Visual selection
                                        </A></LI><LI><A href="Project2.html">
                                            Optimized NICE
                                        </A></LI><LI><A href="Project3.html">
                                            Surface display
                               </A></LI><LI><A href="Project4.html">
                                            Capsule production
                                        </A></LI><LI><A href="Modeling.html">
                                    Modeling
                                </A></LI><LI><A href="Notebook.html">
                                    Notebook
                                </A></LI><LI><A href="Human Practice.html">
                                    Human Practice
                                </A></LI><LI><A href="Biosafety.html">
                                    Biosafety
                                </A></LI><LI><A href="Team_introduction.html">
                                    Team
                                </A></LI><LI><A href="Team_introduction.html">
                                            Introductions
                                        </A><A href="Team_Members.html">
                                            Team Members
                                        </A><A href="Team_Attributions.html">
                                            Attributions
                                        </A></LI></NAV></HEADER><P id="Visual selection system" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">
                    Establishment of a markerless visual selection system for knocking devices into the genome of <I>L. lactis</I></P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P id="Background" style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000">
                    Background
                </P><P>
                        Currently, most of the devices are introduced into <I>L. lactis</I> by plasmids. Various vectors containing constitutive or inductive promoters have been developed, such as pMG36e, pNZ8148. However, the devices introduced by plasmids had two disadvantages. First, most plasmids have selection markers (genes) for antibiotic resistance, which is forbidden to be used in food or digestive tract of human and animals directly. Second, the plasmids were unstable in cytoplasm and might lost when selection pressure is absent.
                    </P><P>
                        The alternative choice is knocking the devices into the genomes of <I>L. lactis</I>. Double cross over recombination using non-replicative or conditional replicative plasmids is the widely applied strategy. For instance, using temperature sensitive plasmid, at restrictive temperature, single-crossover integration of devices can be selected by certain antibiotics. Under lower temperature and in the absence of antibiotics, the second cross-over is occurred at a very low rate and the desired device knocked-in strains can be obtained. However, to get desired device knocked-in strains, antibiotic resistance of clones were checked, and only a few clones were antibiotic susceptible. The antibiotic susceptible clones were then verified by PCR and only half of them were the desired device knocked-in strains, the other half were the parental strains. 
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure3: The nisin controlled expression (NICE) system</P></DIV><P id="Design" style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000">
                        Design
                    </P><P>
                        To get device knocked-in recombinant <I>L. lactis</I> without antibiotic resistant genes and make this process easy and highly efficient, we try to establish a visual selection system based on the insoluble blue compounds generated by the lacZ gene encoded β-galactosidase catalyzed X-gal (also abbreviated BCIG for 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside) hydrolysis. To build this system, we first chose the His locus as the targeting locus, for the His locus is required for the synthesis of histidine, an essential amino acid for <I>L. lactis</I> growth, thus the recombinant <I>L. lactis</I> strains can only be survived with the supplementation of histidine in the culture but can not survive without histine. The His fragment was introduced into a temperature-sensitive conditional replicative shuttle plasmid and we named this plasmid as pHis. Then we placed the lacZ gene driven by the pNisZ promoter into the His locus and we name this plasmid as pBlue. The next step is to integrate the lacZ gene into the genome of NZ9000 to yield the NZ-Blue recombinant strain. The lacZ gene encoded β-galactosidase can hydrolyze the X-gal from colorless substance to insoluble blue compounds, thus NZ-blue strain forms blue colonies on X-gal plate supplemented with nisin. Finally, the device carrying plasmid based on pHis (pDevice) will be introduced to NZ-Blue to replace the lacZ gene and the desired device knocked-in strains will be white when cultured with X-gal and supplemented with nisin.
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure1.2: The markerless integration visual selection system for <I>L. lactis</I></P></DIV><P id="Result" style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000">
                        Result
                    </P><P>
                        The temperature-sensitive conditional replicative plasmid pHis was used to introduce the ‘<I>PnisZ</I>::<I>lacZ::Terminator’ </I>fragments into the NZ9000 chromosome of <I>L. lactis</I>. The constructed plasmids pBlue was transformed into NZ9000 by electroporation. To obtain NZ-Blue via homologous sequences (HisF or HisR), the NZ9000-pBlue strains were grown in M17GS (M17 broth with 0.55 % sucrose and 0.5 % glucose) containing erythromycin and incubated at 38.5 °C overnight to integrate the pBlue DNA sequence into the genome of NZ9000 through single cross over. Then, the cultures were diluted in M17GS medium without erythromycin and incubated at 28 °C to promote the double cross over and excision of the antibiotic resistance genes. The subcultures were diluted and plated on M17GS with X-Gal and nisin. Blue colonies obtained by serial plating were screened for erythromycin susceptibility. Colonies in which lacZ gene integration had occurred will appear as blue on X-gal plate supplemented with nisin and erythromycin susceptible. The erythromycin susceptible and blue colonies selected from thousands of colonies were further subject to PCR verification using the extracted genomic DNA as PCR template. DNA sequencing was also used to confirm that no genetic mutations were introduced during the experiment process. After this difficult and tedious task, we successfully got the lacZ integration strain and we name it NZ-Blue.
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure1.3: Chracterization of the NZ-Blue strain. A). Schematic representation of genomes of NZ9000 and NZBlue. The primers used for PCR is depicted. Theoretical size (bp) of each of the PCR products generated with the primer combinations was also shown. B). On X-gal plate, the recombinant NZBlue strain were appeared as blue. C). Multiple-PCR analysis of the NZ9000 and NZBlue recombinant strain. The 1 kb DNA ladder marker is shown to the left (M).</P></DIV><P>
                        To test the efficiency and capacity of our markerless visual selection system for knocking devices into the genome of <I>L. lactis</I>, 5 devices with different lengths (ranging from 1 kb to 14 kb) were PCR amplified and inserted into the pHis vector. The 5 constructed pDevice plasmids were transformed into NZ-blue strain through electroporation, respectively. To obtain integrant resulting from a single-crossover event via homologous sequences (HisF or HisR), the strains containing the plasmids were grown in M17GS containing erythromycin and incubated at 38.5 °C overnight. Then, the cultures were diluted in M17GS medium without antibiotic and incubated at 28 °C to obtain double crossover and excision of the antibiotic resistant genes. The subcultures were diluted and plated on M17GS with X-Gal and nisin. According to our design, the white colonies will be the ones that the devices have been knocked into the genome of NZ-Blue. At this time, we found that among the pool of blue colonies, usually none or one to several white colonies can be easily identified in the plates, indicating the double crossover is really occurred at a very low rate. Multiple-PCR analysis and DNA sequencing with primer pairs specific to genomic regions lying out the homologous left and right arms were carried out to confirm whether the white colonies were the right ones. As expected, 100% of the white colonies are the devices successfully knocked-in strains.
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure1.4: Characterization of 5 different length devices recombinant <I>L. lactis</I> A). Schematic representation of the 5 different length devices, theoretical size (bp) of each of the PCR products generated with the primer combinations was shown. B). Representative White colonies from the pool of blue colonies. C) PCR analysis of the 5 different length devices recombinant <I>L. lactis</I> strains.</P></DIV><P>
                        We further summarized the double cross over occurrence rate and the accuracy rate of obtaining these 5 different devices knocked-in strains. As shown in Table 1, though the positive rate is really low in the 10^-5 to 10^-4 range, the accurate rate is 100%. The mCherry device can give red bioluminescence after stimulation, we found that all the 58 white colonies appeared on the X-gal plate gave red bioluminescence, demonstrated that all the white colonies are the right ones. In conclusion, our makerless visual selection system can effectively integrate devices ranging from 1 kb to 14 kb. Furthermore, the white one is the right one and the accuracy rate is 100%. How easy and wonderful!
                    </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Table1:The positive rate and accuracy rate of 5 different devices knocked-in <I>L. lactis</I> strains</P></DIV></DIV></DIV></SECTION><FOOTER><DIV class="copyright">
                        在这里我们可以改成蓝色的背景，白色的图标
                    </DIV></FOOTER></DIV></DIV></DIV></DIV><DIV class="card-holder hide"><DIV class="card-wrapper"><A onclick="$.scrollTo('#Background',500)"><DIV class="card"><SPAN class="card-content">
                            Background
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Design',500)"><DIV class="card"><SPAN class="card-content">
                            Design
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Result',500)"><DIV class="card"><SPAN class="card-content">
                            Result
                        </SPAN></DIV></A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>