<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU_China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
           Modeling
        </TITLE></P><DIV class="main-indents"><DIV class="my_main"><SECTION id="content"><DIV style="width=960px;padding: 0px; margin: 0; background-color: rgb(255,255,255); color: #000;"><HEADER><NAV style=""><UL class="sf-menu"><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                    Home
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                            Abstract
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Background">
                                            Background
                                        </A></LI></UL><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                    Project &amp; Results

                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                            Visual selection
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Project2">
                                            Optimization
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Parts">
                                            Parts
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Notebook">
                                            Notebook
                                        </A></LI><LI class="current"><A href="https://2016.igem.org/Team:TMMU_China/Model">
                                    Modeling
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices">
                                    Human Practice
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Coloring" style="cursor:pointer;">
                                            Coloring book
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Construction" onclick="$.scrollTo('#Construction',500)" style="cursor:pointer;">
                                            Construction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Counseling" onclick="$.scrollTo('#Counseling',500)" style="cursor:pointer;">
                                            Counseling
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Collaborations" onclick="$.scrollTo('#Collaborations',500)" style="cursor:pointer;">
                                           Collaborations
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Conferences" onclick="$.scrollTo('#Conferences',500)" style="cursor:pointer;">
                                           Conferences
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Safety">
                                    Biosafety
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                    Team
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                            Introduction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Team/Members">
                                            Team Members
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Attributions">
                                            Attributions
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                    Judging
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                            Medal Criteria
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging/Engagement">
                                            Education &amp; Public Engagement award
                                        </A></LI></NAV></HEADER><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Overview" style="margin-left:400px;margin-top:20px;font-size:30px;font-family:verdana;color:#FF0000">Overview</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="font-family:verdana;font-size:17px;">
							Mathematical modeling and experimental research are complementary to each other. We established a novel visual selection for device knocked into the genome of <I>L. lactis</I>. We also optimized the nisin controlled expression (NICE) system, the protein surface display system. In the NICE system, nisin was used as the food grade gene expression inducer. However, nisin is also an antibacterial peptide. A low concentration of nisin will lead to a low expression of the target protein. However, a high concentration of nisin will inhibit the growth of bacteria, which will also decrease the total expression level. To resolve the dilemma we incorporated the nisin immunity or resistant genes <I>nisI</I> or <I>nsr</I> into the NICE system. However, it is still difficult to directly determine whether our products are produced sufficiently, and to evaluate the total expression amount at the community level.
						</P><P style="font-family:verdana;font-size:17px;">
						   To resolve these problems, we build 4 models to predict the gene expression rate, the growth rate, the protein production and secretion rate, and the total protein expression amount at the community level under different concentrations of nisin. To help readers understand the specific theory behind our model, we also wrote notes for all the modeling techniques.At last, we expect to improve mathematical modeling to make it more convenient to use. So we collaborated with UESTC who helped us to post a visual interface of our model results on the web page.<A href="http://bio101.uestc.edu.cn/tmmu/"><FONT style="font-size:20px;color:red">Click here to know more about the imformations.</FONT></A></P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Model1" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Model 1:The Gene Expression Rate</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P id="Introduction" style="font-weight:bold;font-size:20px;">
							Introduction
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
							In order to predict the protein expression rate and amount under different nisin concentrations, we treated the bacteria in isolation, which means at the single cell level. This prediction will be incorporated into the equation to calculate the total protein amount at the community level.
						</P><P id="Modeling" style="font-weight:bold;font-size:20px;">
							Modeling
						</P><DIV><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						   In the NICE system, the PnisZ or PnisA promoter controlled gene expression rate depends on the concentration of nisin. We try to predict the gene expression rate under different nisin concentrations. In one research, the quantitative relationship between the extracellular concentrations of nisin and the efficiency of PnisZ were measured (Guo et al., 2013). We retrieved the data (Table 1) to get a fitting equation, which will be used in the protein production model.</P><P style="text-align:center;display:block;margin:10xp auto;">Table 1 Data used in the fitting</P><P style="font-family:verdana;font-size:17px;">The log-sigmoid function was used to fit, because the nisin receptors on the cell membrane is limited, and this function value will not continue to increase as x reach to a certain value. This is suit for the characteristics of our promoter.</P><P style="font-family:verdana;font-size:17px;">In the NICE system, the protein of interest production process is shown as below：</P><P style="font-family:verdana;font-size:17px;">We tried to use Michaelis-Mentin kinetics to predict the protein production rate. We made several modifications on the equation to accommodate the transcriptional regulatory mode of the PnisZ promoter.</P><P style="font-family:verdana;font-size:17px;">The Michaelis-Mentin kinetics equations:</P><P style="font-family:verdana;font-size:17px;">Attention：We replaced [PnisZ] in the Michaelis-Mentin kinetics by F(X) that we obtain by fitting, and we try to transform this typical math model to describe the direct proportion relationship between [PnisZ] and the concentrate of extracellular nisin.</P><P style="font-family:verdana;font-size:17px;">[PnisZ], [mRNA] and [P] represent the concentrations of PnisZ, mRNA and product protein respectively. Coefficient k was got by the experimental result.</P><P style="font-family:verdana;font-size:17px;">The remaining symbols are defined in Table 2.</P><P style="text-align:center">Table 2 Definition of symbols used in the Michaelis-Mentin kinetics</P></DIV><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							Analysis and Results
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">Using the model we established, we predict the protein production rate under different nisin concentrations as shown in Figureure 1.1. The higher concentrations of nisin will lead to higher expression levels of protein of interest.</P><P style="text-align:center;">Figure 1.1 Simulated protein production levels under different nisin conditions</P><P style="font-family:verdana;font-size:17px;">To test whether our model can be applied to the real situation, we compared the expression levels of mCherry proteins and the simulated results. As shown in Figureure 1.2, the two curves are well matched.</P><CENTER><P>Figure 1.2 The fluorescence intensity levels produced by mCherry proteins along with the simulated protein levels under different concentrations of nisin.</P></CENTER><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							References
						</P><P style="margin-top:-20px;">[1]https://2015.igem.org/Team:Oxford/Modeling
						[2]Guo, T., Hu, S., and Kong, J. (2013). Functional analysis and randomization of the nisin-inducible promoter for tuning gene expression in Lactococcus lactis. Curr Microbiol 66, 548-554.
						[3]Iost, I., and Dreyfus, M. (1995). The stability of Escherichia coli lacZ mRNA depends upon the simultaneity of its synthesis and translation. EMBO J 14, 3252-3261.[4]
Proshkin, S., Rahmouni, A.R., Mironov, A., and Nudler, E. (2010). Cooperation between translating ribosomes and RNA polymerase in transcription elongation. Science 328, 504-508.[5]
Selinger, D.W., Saxena, R.M., Cheung, K.J., Church, G.M., and Rosenow, C. (2003). Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. Genome Res 13, 216-223.[6]
Sourjik, V., and Berg, H.C. (2004). Functional interactions between receptors in bacterial chemotaxis. Nature 428, 437-441.</P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Model2" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Model 2: The protein secretion model</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P id="Introduction" style="font-weight:bold;font-size:20px;">
							Introduction
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">To be surface displayed, proteins should be secreted after translation and then anchored to the cell wall, which is really a long journey. In our project, we used the signal peptide SPusp45 to direct protein secretion and the cA domain to make proteins surface displayed. In this section, we try to simulate the secretion process in bacteria, and predict the amount of protein can be surface displayed at a given time.</P><P id="Modeling" style="font-weight:bold;font-size:20px;">
							Modeling
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">A recursion equation was used to calculate the amount of protein that on the cell wall. The value of  was determined by reference (Lin et al., 2016), the definition of symbols are listed in Table 3.</P><P style="text-align:center;">Table 3 Definition of symbols used in the recursion equation</P><P id="Analysis and Results" style="margin-top:20px;font-weight:bold;font-size:20px;">
							Analysis and Results
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">Using the secretion model, we found that 89.46% of the expressed protein will be secreted and anchored to the cell, when the expression of protein is stable as shown in Figure 2. The result is similar to several reported protein secretion ratio with the signal peptide SPusp45 (87%(Lin et al., 2016), 89%(Borrero et al., 2011) ).</P><P style="text-align:center">Figure 2 The amount of secreted proteins (yellow) and the total expressed proteins (blue)</P><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							References
						</P><P style="margin-top:-20px;">[1]Szymanska, Z., Parisot, M., and Lachowicz, M. (2014). Mathematical modeling of the intracellular protein dynamics: the importance of active transport along microtubules. J Theor Biol 363, 118-128.[2]Borrero, J., Jimenez, J.J., Gutiez, L., Herranz, C., Cintas, L.M., and Hernandez, P.E. (2011). Use of the usp45 lactococcal secretion signal sequence to drive the secretion and functional expression of enterococcal bacteriocins in Lactococcus lactis. Appl Microbiol Biotechnol 89, 131-143.[3]Lin, J., Zou, Y., Ma, C., Liang, Y., Ge, X., Chen, Z., and She, Q. (2016). Construction and characterization of three protein-targeting expression system in Lactobacillus casei. FEMS Microbiol Lett 363.</P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Model3" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Model 3: The Growth rate model</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P id="Introduction" style="font-weight:bold;font-size:20px;">
							Introduction
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">The mathematical models that can predict the growth rate of a microorganism population under a set of environmental conditions is useful for many research. Many growth models have been proposed, such as the widely used Logistics model. However, when primary models are applied to bacterial growth curves, the biological variability is reduced to a single curve defined by some kinetic parameters (lag time and growth rate), and sometimes the models give poor fits in some regions of the curve. Moreover, the number of experiment is usually limited for most iGEM teams (relative to the threshold to get credible growth curve). To solve this problem and get the credible growth curve, we built a random effect multiplicative heteroscedastic model. The model combines nonparametric theory and bootstrap techniques that can get a prediction band with a probability of 0.95(95%) to predict the optimal growth curve.</P><P id="Modeling" style="font-weight:bold;font-size:20px;">
							Modeling
						</P><P style="font-family:verdana;font-size:17px;">The process of this non-parametric model is shown in Figure 3. The theory basis of this model (including 4 steps) was supplemented in the note.</P><P style="text-align:center;">Figure 3.1 The process of the non-parametric model</P><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							Analysis and Results
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">As shown in Figure 3.2, we found that the non-parametric model can predict the process of bacteria growth rate very well, and the nisin antibacterial activity is enhanced as the concentration increased. The results were obtained by only 3 groups of experimental data, indicating that this model can get the credible results by a smaller amount of data.</P><P style="text-align:center;">Figure 3.2 Simulated bacteria growth rate of L. lactis under different nisin concentrations.</P><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							Reference
						</P><P style="margin-top:-20px;">[1]Szymanska, Z., Parisot, M., and Lachowicz, M. (2014). Mathematical modeling of the intracellular protein dynamics: the importance of active transport along microtubules. J Theor Biol 363, 118-128.</P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Model4" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Model 4: The total protein level at the community level</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P id="Introduction" style="font-weight:bold;font-size:20px;">
							Introduction
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">In model 1, we treated bacteria at the single cell level. In model 2, we predict the secretion and surface display efficiency of proteins base on the SPusp45 secretion signal peptide and cA anchoring domain. In model 3, we used non-parametric method to predict the growth rate of bacteria at the community level. Here, we try to combine the 3 models together to get a compound model to calculate the total protein level that can be surface displayed at the community level.</P><P style="text-align:center;">Figure 4.1 Schematic representation of the total protein level at the community level.</P><P id="Modeling" style="font-weight:bold;font-size:20px;">
							Modeling
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">We combined the best growth curve and the gene expression rate, using matlab to realize our idea by programming.</P><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							Analysis and Results
						</P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">We found that when the concentration of nisin is 1 IU, the efficiency of protein expression level that can reach a peak is better than other nisin concentrations. This combined model can be used in the nisin controlled expression system, and reduce the workload of researchers.</P><P style="text-align:center;">Figure 4.2 The total protein expression level under different nisin concentrations.</P><P id="Analysis and Results" style="font-weight:bold;font-size:20px;">
							References
						</P><P style="margin-top:-20px;">[1]Fisher, Irving (1897). A brief introduction to the infinitesimal calculus. 
						   [2]André Weil: Number theory. An approach through history. From Hammurapi to Legendre. Birkhauser Boston, Inc., Boston, MA, 1984, ISBN 0-8176-4565-9, p. 28.
                        </P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;margin-bottom: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Visualization" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Model Visualization</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						   We expect to improve mathematical modeling to make it more convenient to use. And we collaborated with UESTC who helped us to post a visual interface of our model results on the web page.<A href="http://bio101.uestc.edu.cn/tmmu/"><FONT style="font-size:20px;color:red">Click here to know more about the imformations.</FONT></A></P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;margin-bottom: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Note" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">NOTE</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						   The process of this non-parametric model is shown in Fig 3. The theory basis of this model (including 4 steps) was supplemented in the note. Please see more details.
						   </P></DIV></DIV></DIV></SECTION><FOOTER><DIV style="padding:20px 0 0 0;margin:0;float:right;width:300px;color:#fff;"><UL style="font-size:14px;"><LI><SPAN style="text-decoration:underline;color:#fff;left:20px;">Contact us</SPAN></LI><LI><SPAN>Email:</SPAN><A href="Mailto:igem@tmmu.edu.cn" style="color:#fff">igem@tmmu.edu.cn</A></LI><LI><SPAN>Address:</SPAN> Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038</LI></UL></DIV></FOOTER></DIV></DIV><DIV class="card-holder hide"><DIV class="card-wrapper"><A onclick="$.scrollTo('#Model1',500)"><DIV class="card"><SPAN class="card-content">
                            Gene Expression
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Model2',500)"><DIV class="card"><SPAN class="card-content">
                            Protein Secretion
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Model3',500)"><DIV class="card"><SPAN class="card-content">
                            Growth rate
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Model4',500)"><DIV class="card"><SPAN class="card-content">
                            Total Protein
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Visualization',500)"><DIV class="card"><SPAN class="card-content">
                            Visualization
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Note',500)"><DIV class="card"><SPAN class="card-content">
                            Note
                        </SPAN></DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>