Team:TMMU China/Notebook
Notebook
Home
Abstract
Background
Project & Results
Visual selection Optimization Parts Notebook
Modeling
Human Practice
Coloring book Construction Counseling Collaborations Conferences
Biosafety
Team
Introduction Team Members Attributions
Judging
Medal Criteria
Education & Public Engagement award
Journal
February: We got to know iGEM and synthetic biology. Associate professor Hu Qiwen organized a lecture on iGEM and synthetic biology, and began to recruit team members to establish the first team in our school and in Chongqing.
March: After a month of learning and training, 21 team members were gathered to establish the TMMU-China team. Team registration was completed.
April: Team members consulted literature and brainstormed to discuss problems in the project and team constrcution, and set the project as "promote Lactococcus lactis to be a better chassis in synthetic biology".
May: Instructed by our mentors, we designed the markerless integration visual selection system for Lactococcus Lactis, Then we ordered primers, and started experiments.
June: The NZ-blue strain was constructed and the utility of it was tested.
July: We completed the optimizations of the system, and constructed the plasmid containing the promoter PnisZ and nisin resistance gene nsr. Some of the members set out to design and finish the mathematic modeling.
August: We completed construction of the protein surface display system, by displaying LacZ via connecting Usp45-cA on the surface of L. lactis
September: We completed the construction of all the 6 Parts.
October: We at last perfected our preparation for presentation, poster and wiki.
Collaboration
After building up our own team and during the process of completing our project, we found our weaknesses in many aspects. Therefore, we sought advice from other teams and meanwhile helped them in some ways.
February 22nd UESTC-China: They introduced iGEM to us and invited us to participate in it.
March 12th  NUDT-China: They guided us in terms of team construction, and introduced the rules of the iGEM, which inspired us.
June 11th Nanjing-China: While designing the project, we encountered some problems. After communication with Nanjing-China, they gave us useful advice and helped us to re-group our thoughts in our project design.
August 5th SCU-China: In the Southwest Union, we knew that the team of Sichuan University had difficulties in verifying the efficiency of an antibacterial peptide. So we helped them in the verification. At the same time, we invited their members in charge of modeling to provide technical guidance for us.
September 27th UESTC-software: In the Southwest Union, UESTC-software brought up with some problems in designing DNA sequencing primers, so we helped them to design the primers they needed. In turn, they helped us to achieve visualization of the modeling by using software.
Protocols
1. Bacteria and culture media
Bacteria  strains used in this study are NZ9000 (L.  lactis) and DH5α (E. coli). DH5α was  used as the cloning host. E. coli were  cultured in Luria-Bertani (LB) medium (agar or broth) at 37 °C. L. lactis was cultured in M17GS (M17 broth  supplemented with 0.5 % (wt/vol) glucose, 0.55 % (wt/vol) sucrose) medium at 30  °C (agar or broth). Ampicillin was used at the concentration of 100 μg/mL for E. coli. Erythromycin was used at the concentration  of 250 μg/mL for E. coli and 20 μg/mL  for L. lactis, respectively. X-Gal was  used at the concentration of 40 μg/mL and nisin was used at the concentration 0.1  μg/mL if needed.
2. Polymerase Chain Reaction
All  PCR reactions were done with Fast Taq Mix DNA polymerase from Novoprotein Scientific Inc (Shanghai, China) and Primerstar Max DNA polymerase from Takara Bio Inc. (Dalian, China).
(1). Fast Taq Mix DNA polymerase
For colony PCR, Fast Taq polymerase is used. As the template, the colony                         of E.coli is picked from a plate and culture the E.coli when the medium                         become turbid. The component and cycling conditions are listed in the table below.
2×Fast Taq Mix (SinoBio)
Cycling conditions
Component
volume(μL)
Tempreture(℃)
Time
Cycle
Premix Taq
5
94
5 min
1
Bacteria solution
1
94
30 sec
35
Primer F (20 μM)
0.2
55
30 sec
Primer R (20 μM)
0.2
72
1kb/min
dd H 2 O
3.6
72
5 min
1
Total
10
16
1 min
1
(2). Primerstar Max DNA polymerase
Primerstar Max DNA polymerase is used when high fidelity is needed. Primerstar                         Max has been used for amplifying DNA fragments and generating specific                         mutations in genes with the use of primers. The cycling program is listed                         in the table below.
2×PrimerStar Max (Takara)
Cycling conditions
Component
Volume(μL)
Tempreture (℃)
Time
Cycle
PrimerStar Max
25
98
5 min
1
DNA Template
1
98
10 sec
35
Primer F (20 μM)
1
52
15 sec
Primer R (20 μM)
1
72
10 sec/kb
dd H 2 O
22
72
5 min
1
Total
50
16
1 min
1
3. Restriction enzyme digestion and T4 DNA ligation
(1).Restriction enzymes used in this work were FastDigest enzymes (Thermo Scientific). Combine the following reaction components list at room temperature and the mixtures were incubated with recommended temperatures and time.
Componet
Volume(μL)
FastDigest Enzyme 1
2.5  μL
FastDigest Enzyme 2
2.5  μL
FastDigest Green Buffer
5  μL
Plasmid or DNA fragment
up to 5 μg
dd H 2 O
add to 50μL
Total
50 μL
The digestion productions were purified by the Wizard SV Gel and PCR Clean-Up Kit (Promega Corporation).
(2).DNA ligation used in this work was DNA Ligation Kit Ver.2.1 from Takara Bio Inc. (Dalian, China). Mix linearized plasmid vector DNA and a DNA fragment in a total volume of 2.5 μL. The amounts of vector: fragment is 0.03 pmol : 0.03 - 0.3 pmol. Add an equal volume of ligation mixture (2.5 μL) as the DNA solution and mix thoroughly. Incubate at 16 °C for 30 minutes.
4. Seamless Cloning
For cloning our target fragments into vectors, NovoRec seamless cloning KIT from Novoprotein Scientific Inc. was used. Seamless cloning can simultaneously combine one or more PCR products with a linearized vector when the DNA to be joined shares 15 – 25 bp of homology at each end. The required homology can be easily generated by adding complementary sequence to the ends of the PCR primers. No additional treatment of the PCR fragment is required, such as restriction digestion, ligation, phosphorylation, or blunt-end polishing. The efficiency of NovoRec PCR Seamless Cloning is over 95 %.
(1). The acquisition of the linearized vector by enzyme digestion (> 15 ng/μL).
(2). A small sequence (15-25 bps) overlapped with the end of the cloning site will be added onto the insert through a PCR step.
(3). Recombination reaction.
ddH2O
up  to 20 μL
5 ×  Buffer
4 μL
Linearized  cloning vector
0.03  pmol
PCR  products of insertions
0.09  ~ 0.3 pmol
Enzyme
2 μL
The recommended amount of vector for recombination reaction is 0.03 pmol. The recommended amount of insertion for recombination reaction is 0.09 ~ 0.3 pmol.
(4). Incubate the mixture at 37 °C for 20 min and transform it into E. coli.
5. Transformation of E.coli
Example Protocol: Standard heat-shock transformation of chemically competent bacteria.
Take competent cells out of -80°C and thaw on ice (approximately 20-30min). Take agar plates (containing the appropriate antibiotic) out of 4°C to warm up to room temperature or place in 37°C incubator. Mix 1 to 5μl of DNA (usually 10pg to 100ng) into 20-50μL of competent cells in a microcentrifuge or falcon tube. GENTLY mix by flicking the bottom of the tube with your finger a few times. Note: Transformation efficiencies will be approximately 10-fold lower for ligation of inserts to vectors than for an intact control plasmid. Place the competent cell/DNA mixture on ice for 20-30min. Heat shock each transformation tube by placing the bottom 1/2 to 2/3 of the tube into a 42°C water bath for 30-60 seconds (45sec is usually ideal, but this varies depending on the competent cells you are using). Put the tubes back on ice for 2 min. Add 250-500μl LB or SOC media (without antibiotic) and grow in 37°C shaking incubator for 45min. Note: This outgrowth step allows the bacteria time to generate the antibiotic resistance proteins encoded on the plasmid backbone so that they will be able to grow once plated on the antibiotic containing agar plate. This step is not critical for Ampicillin resistance but is much more important for other antibiotic resistances. Plate some or all of the transformation onto a 10cm LB agar plate containing the appropriate antibiotic. Note: We recommend that you plate 50μL on one plate and the rest on a second plate. This gives the best chance of getting single colonies, while allowing you to recover all transformants. Incubate plates at 37°C overnight.
6. Transformation of Lactococcus lactis
(1) Preparation of competent L. lactis cells
To obtain competent cells, the cultures were grown to an optical density at 600 nm of 0.5 to 0.8 and then diluted 100-fold in SGM17 (M17GS containing 0.5 M sucrose) supplemented with glycine. After growth at 30 °C to an optical density at 600 nm of 0.2 to 0.4, the cells were harvested by centrifugation at 4 °C at 5,000x g. Following two washes in ice-cold 0.5 M sucrose containing 10% glycerol, the cells were suspended in 1/100 culture volume of washing solution and then stored in aliquots at -85 °C until use.
(2) Transformation by electroporation
The cell suspensions were thawed on ice. Portions (40 μL) were mixed with 1 μL of DNA and then transferred to an ice-cooled electroporation cuvette (2-mm electrode gap) and exposed to a single electrical pulse. The pulse was delivered by a Gene-Pulser (Bio-Rad, Calif.) set at 25 μF and normally at 2.0 kV. The cuvette was connected in parallel to a 200-Ω resistor (pulse controller; Bio-Rad), resulting in time constants of 4.5 to 5ms.
(3) Incubation and plate
Immediately following the discharge, the suspensions were mixed with 0.96 ml of ice-cold SGM17MC (SGM17 containing 20 mM MgCl2 and 2 mM CaCl2) and left on ice for about 5 min. Appropriate dilutions were then made in SGM17MC, and the cells were incubated at 30°C for 2h. The plates M17GS contained 20 μg/mL of erythromycin for the selection of erythromycin-resistant transformants. Transformants were enumerated after 2 days of incubation at 30°C.
Contact us
Email: igem@tmmu.edu.cn
Address: Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
Journal
Collaborations
Protocols
