Team:TMMU China
Home
Home
Abstract
Background
Project & Results
Visual selection Optimization Parts Notebook
Modeling
Human Practice
Coloring book Construction Counseling Collaborations Conferences
Biosafety
Team
Introduction Team Members Attributions
Judging
Medal Criteria
Education & Public Engagement award
Abstract
Establishment of bacterial or eukaryotic chassis for plugging-in and plugging-out genetic circuits and new-to-nature functionalities is the foundational work for synthetic biology. Lactic acid bacteria (LAB) are a group of microbes that are generally recognized as safe (GRAS) and can be optimized to be the next generation of synthetic biology chassis, especially in the food industry and therapeutics. Lactococcus lactis (L. lactis) is the most promising LAB chassis, but little effort has been made to improve it to be a better chassis. Two methods are generally adopted to introduce the foreign device into L. lactis. One is using plasmids for the introduction. This method is simple to manipulate, but the plasmids will become unstable in the absence of selection pressure, and the most frequently used selection markers, the antibiotic resistance genes, are forbidden from being used in food and therapeutics. The other is integrating the foreign device into the L. lactis genome. This method is more stable, but the current genome integration systems are time-consuming and labor-intensive.
To overcome the disadvantages of the genome-integrating method, we established a visual selection system by inserting a visual marker, PnisZ promoter controlled lacZ gene, into the His locus of L. lactis strain NZ9000 chromosome. The constructed strain formed blue colonies in the agar plate containing X-gal and nisin inducer. To clone the device of interest, lacZ gene was replaced by the device of interest. The recombinant strains then formed white colonies. Based on this strategy, we can easily pick up the white colonies from the pool of blue colonies. The result show the strategy is working. The white colonies we picked are all the desired recombinant strains, while the blue colonies are not. Thereby, we can easily find the right colonies by only picking up the white colonies. “Mr. White is Mr. Right!”
We further tested the efficiency and capacity of the markerless visual selection system for knocking devices into the genome of L. lactis. A total of 5 devices with different lengths (ranging from 1 kb to 14 kb) were inserted into the genome of L. lactis. We verified that the shortest mCherry device and the longest polysaccharide Vi device (14 kb) were successfully and functionally expressed. To increase the expression efficiency in L. lactis, we optimized the gene expression system to tolerate a higher nisin concentration through introducing nisin immunity gene or nisin resistance gene into L. lactis. To enable proteins to function at the right place, we also introduced signal peptide for secretion and cell wall anchoring domain for surface display into versatile plasmids. By this way, L. lactis became the right chassis with versatile equipment that can be widely applied in synthetic biology.
Background
Contact us
Email: igem@tmmu.edu.cn
Address: Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038
