<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU_China_Project_Results_Project2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU China/Project Results/Project2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
            Optimized NICE
        </TITLE></P><DIV class="main-indents"><DIV class="my_main"><SECTION id="content"><DIV style="width=960px;padding: 0px; margin: 0; background-color: rgb(255,255,255); color: #000;"><HEADER><NAV style=""><UL class="sf-menu"><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                    Home
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                            Abstract
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Background">
                                            Background
                                        </A></LI></UL><LI class="current"><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                    Project &amp; Results

                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                            Visual selection
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Project2">
                                            Optimization
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Parts">
                                            Parts
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Notebook">
                                            Notebook
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Model">
                                    Modeling
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices">
                                    Human Practice
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Coloring" style="cursor:pointer;">
                                            Coloring book
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Construction" onclick="$.scrollTo('#Construction',500)" style="cursor:pointer;">
                                            Construction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Counseling" onclick="$.scrollTo('#Counseling',500)" style="cursor:pointer;">
                                            Counseling
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Collaborations" onclick="$.scrollTo('#Collaborations',500)" style="cursor:pointer;">
                                           Collaborations
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Conferences" onclick="$.scrollTo('#Conferences',500)" style="cursor:pointer;">
                                           Conferences
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Safety">
                                    Biosafety
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                    Team
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                            Introduction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Team/Members">
                                            Team Members
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Attributions">
                                            Attributions
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                    Judging
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                            Medal Criteria
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging/Engagement">
                                            Education &amp; Public Engagement award
                                        </A></LI></NAV></HEADER><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Background1" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Optimization of the nisin resistance of device integrated NZ9000</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Background</B></P><SPAN style="font-weight:bold;font-family:verdana;font-size:17px;">Nisin immunity genes and nisin resistant genes</SPAN><P style="margin-top:0px;font-family:verdana;font-size:17px;">
						    The NICE system inducer, nisin, is a 34-residue food grade antibacterial peptide produced by some strains of <I>L. lactis</I> which is active against wide range of gram-positive bacteria and is  widely used in the food industry as a preservative. Nisin inhibits growth of Gram-positive cells in two ways (Figure 2.1). First, nisin inhibits cell wall synthesis by binding to the lipid II molecule in the membrane. Second, nisin molecules can span across the lipid bilayer to make a pore in the cell membrane which causes the leakage of cytoplasm. The <I>L. lactis</I> NZ9000 strain has no resistant mechanism and cannot tolerate high levels of nisin concentration in the medium.
                        </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
							    Figure 2.1 Antibacterial mechanisms of nisin.
							</P></DIV><P style="font-family:verdana;font-size:17px;">
						   In nisin producing strains, the nisin operon contains four nisin immunity genes, <I>nisI, nisF, nisE</I> and <I>nisG</I>. Among them, <I>nisI</I> encodes a lipoprotein that serves as the first defense for nisin immunity. (Figure 2.2A) In non-nisin-producing <I>L. lactis</I> strains, nisin resistance could be conferred by a specific nisin resistance (<I>nsr</I>) gene, which encodes a 35-kDa nisin resistance protein (NSR, Figure 2.2B). NSR could proteolytically inactivate nisin by removing six amino acids from the carboxyl “tail” of nisin. The truncated nisin (nisin 1–28) displayed a markedly reduced affinity for the cell membrane and showed significantly diminished pore-forming potency in the membrane. A 100-fold reduction of bactericidal activity was detected for nisin 1–28 in comparison to that for the intact nisin. What’s more, the truncated nisin can still induce gene expression through the NICE system. Thus, the incorporation of the <I>nisI</I> or <I>nsr</I> gene into the NZ9000 will lead to tolerance of NZ9000 at high levels of nisin concentration.  
    					</P><DIV style="text-align:center;margin-bottom:20px"><P id="Design1" style="text-align:center;">
							    Figure 2.2 The structures of nisin immunity protein NisI (A) and the nisin resistance protein NSR (B).
							</P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Design</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						   The NZ9000 or NZ-Blue strain (<I>nisKR</I> integrated) is sensitive to nisin. We inferred that the introduced of the <I>nisI</I> or <I>nsr</I> gene might render NZ9000 to tolerate high levels of nisin concentration while maintain the induced gene expression capacity of the NZ9000. To achieve the goal, we constructed two plasmids (Figure 2.3) to test whether the incorporation of <I>nisI</I> or <I>nsr</I> can really increase the nisin tolerance of NZ9000 and NZ-Blue. The expression of <I>nisI</I> or <I>nsr</I> gene is driven by the <I>P32</I> promoter, a constitutive promoter.
						</P><DIV style="text-align:center;margin-bottom:20px"><P id="Result1" style="text-align:center;">
							   Figure 2.3 Map of plasmid pMG36e-<I>nsr</I> (A) and pMG36e-<I>nisI</I> (B). Abbreviations: <I>P32</I>, promoter; <I>nsr</I>, nisin resistant gene; <I>nisI</I>, nisin immunity gene; <I>Ter</I>, terminator; <I>Em<SUP>r</SUP></I>, erythromycin resistant gene; Ori, replicon.
							</P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Result</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						    The two plasmids were successfully constructed, and were transformed into NZ9000. After that we measure the growth curve of NZ9000, pMG36e-<I>nisI</I>/NZ9000 and pMG36e-<I>nsr</I>/NZ9000 under different concentrations of nisin. As shown in Figure 2.3, without nisin, the growth curve of these 3 strains exhibits no difference. However, under 200 IU/mL nisin concentration, the growth of NZ9000 is severely inhibited, the pMG36e-nisI/NZ9000strain can still grows to a moderate level, while the pMG36e-<I>nisI</I>/NZ9000 strain can still grow to a level similar to that without nisin, which means the NSR protein can protect <I>L. lactis</I> at high nisin concentrations, while the protect efficiency of NisI is not as pronounced as NSR. Finally, under 500 IU/mL nisin concentration, only the pMG36e-<I>nsr</I>/NZ9000 strain can still grow, the NZ9000 and pMG36e-<I>nisI</I>/NZ9000 strain cannot grow. Based on these results and to make it versatile, we incorporated the nsr gene into the pDevice vector. This time, the device of interest integrated in genome of NZ9000 can achieve high expression level in the presence of high nisin concertation.
						</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">Figure 2.4 Growth Curve of NZ9000 (■), pMG36e-<I>nisI</I>/NZ9000 (△) and pMG36e-<I>nsr</I>/NZ9000 (◇) in different nisin concentrations. Abbreviations: IU, nisin international unit.</P></DIV><P id="Reference" style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>References</B></P><DIV style="margin-left:20px;margin-right:20px;margin-top:-20px;font-size:18px;"><P> [1] Hacker, C., Christ, N.A., Duchardt-Ferner, E., Korn, S., Gobl, C., Berninger, L., et al. (2015). The Solution Structure of the Lantibiotic Immunity Protein NisI and Its Interactions with Nisin. J Biol Chem 290, 28869-28886.
                               [2] Khosa, S., Lagedroste, M., and Smits, S.H. (2016). Protein Defense Systems against the Lantibiotic Nisin: Function of the Immunity Protein NisI and the Resistance Protein NSR. Front Microbiol 7, 504.
       [3] Sun, Z., Zhong, J., Liang, X., Liu, J., Chen, X., and Huan, L. (2009). Novel mechanism for nisin resistance via proteolytic degradation of nisin by the nisin resistance protein NSR. Antimicrob Agents Chemother 53, 1964-1973.
                               </P></DIV></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Background2" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Optimization of the protein localization system</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Background</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						    The localization of protein will affect its function. To make protein function at its right place, the protein localization system should be optimized. To secrete the protein out of the cell, a signal peptide of Usp45 protein was ligated to the target protein. The secretion is very efficiently.  Fusion of the signal peptide of Usp45 to a couple of proteins resulted in an efficient way of secreting the coded proteins into the media. The fusion protein will meet a chaperone protein, SecB, which bring the fusion protein to a protein secretion apparatus. This apparatus cleaves the signal peptide and frees the proteins from the cell.
                        </P><P style="font-family:verdana;font-size:17px;">					   To make protein of interest to be cell surface displayed, anchoring motif should be considered at the first. The most exploited anchoring regions are those with the LPXTG motifs that bind the proteins in a covalent way to the cell wall. However, to achieve versatile application, it is better to use anchoring domains that interact with the cell wall in a non-covalent way. The major autolysin of <I>L. lactis</I>, the cell wall hydrolase AcmA contains 3 tandem arranged LysM motifs and separated by stretches of 21 to 31 amino acids, this region is collectively termed as the cA anchoring domain. The cA domain can be fused to the N- and C- terminus of functional proteins, and can bind proteins to the cell walls of a broad range of gram-positive bacteria. The cA domain could also be used to immobilize soluble enzymes in an active form on the cell surface of <I>L. lactis</I>. Like AcmA, these fusion enzymes could bind to <I>L. lactis</I> cells when added from the outside. Thus, cA fusion proteins produced using other expression systems can also be attached to the cell wall of <I>L. lactis</I>.   
    					</P><DIV style="text-align:center;margin-bottom:20px"><P id="Design2" style="text-align:center;">
							    Figure 3.1 Schematic representation of the protein localization system.
							</P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Design</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						    To verify the function of SPusp45 and the cA anchoring domain, here we designed three devices, the NZ-Cytosol, NZ-Secretion and NZ-Surface devices as shown in Fig 3.2. We took the β-galactosidase protein as an target protein. The β-galactosidase protein is fused to the SPusp45 signal peptide or fused to the cA domain with the SPusp45 signal peptide, driven by the <I>PnisZ</I> promoter and followed by the nisin resistant gene <I>nsr</I>.
						</P><DIV style="text-align:center;margin-bottom:20px"><P id="Result2" style="text-align:center;">
							    Figure 3.2 Design of the protein localization devices. A) NZ-Cytosol, B) NZ-Secretion, C) NZ-Surface. Abbreviations: <I>PnisZ</I>, promoter; <I>SPusp45</I>, signal peptide of <I>usp45</I> from <I>L. lactis</I>; <I>cA</I>, anchor motif of <I>acmA</I> from <I>L. lactis</I>; <I>nsr</I>, nisin resistant gene; <I>Ter</I>, terminator.
							</P></DIV><P style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>Result</B></P><P style="margin-top:-20px;font-family:verdana;font-size:17px;">
						    To demonstrate whether the SPusp45 signal peptide and the cA anchoring domain could indeed secret and anchor the protein of interest at the surface of <I>L. lactis</I>, the p<I>lacZ</I>-Cytosol and p<I>lacZ</I>-Surface plasmids were introduced into NZ9000 respectively, which were named NZ-Cytosol and NZ-Surface. As stated above, the cA domain come from the AcmA protein, which is an autolysin of <I>L. lactis</I> that cleaves the peptidoglycan to release the duplicated bacteria. Since the location sites of AcmA autolysin, substrate peptidoglycan, is now occupied by the β-galactosidase-cA fusion proteins secreted by SPusp45, the autolysin activity is hindered, thus cell separation will be interfered. Indeed, we found that under microscopic, the NZ-Surface cells were poorly separated compared to the NZ-Cytosol strain. Further more, using a polyclonal antibody against β-galactosidase, we found that the β-galactosidase protein is present at the cell wall and cytoplasm, which means the signal peptide, SPusp45, and anchor motif, cA domain, worked as we designed. In contrast, all the β-galactosidase proteins were present in the cytoplasm in the NZ-cytosol strain. (Figure 3.3)
						</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
							  Figure 3.3 The functional verification of the NZ-Cytosol and NZ-Surface strains. A). Microscopic picture of the NZ-Cytosol and NZ-surface strains. B). Western blot of different fractions from NZ9000, NZ-Cytosol and NZ-Surface. The protein fused with cA motif was located on the bacterial cell wall and the cytosol expressed protein was located in the bacterial cytoplasm. This mean the protein fused with cA motif was secreted by SPusp45 and was anchored in the cell wall by cA motif. The <I>lacZ</I> was used as an example. The strain used in western blot are presented upon the lanes.
							</P><P style="font-family:verdana;font-size:17px;">
						   To combined the markerless visual selection system, the improved the nisin resistant of device integrated NZ9000 and the optimized protein localization system together, four plasmids were constructed (Figure 3.4). First, the promoter (double promoter, <I>P32</I> (constitutive promoter) and <I>PnisZ</I> (inductive promoter)), MCS (multi-clone site), <I>nsr</I> with RBS and terminator were inserted in the AscI site of pHis. The constructed plasmid was named pCyt. Second, the signal peptide <I>SPusp45</I> was inserted in the site between <I>PnisZ</I> and MCS by seamless cloning. The constructed plasmid was named pSec. Last, the cA domain was inserted the sites in front of MCS and after MCS. The constructed plasmid was named pSD.1 and pSD.2 to fuse the cA domain at the N-terminus or C-terminus of target protein. All plasmid can be linearized by SmaI and XbaI site of MCS, and all devices inserted in MCS of the 4 plasmids were recommended by using seamless cloning.
						</P><P id="Reference" style="margin-left:0px;margin-top:20px;font-size:20px;font-family:verdana;color:#000000"><B>References</B></P><DIV style="margin-left:20px;margin-right:20px;margin-top:-20px;font-size:18px;"><P> [1] Buist, G., Steen, A., Kok, J., and Kuipers, O.P. (2008). LysM, a widely distributed protein motif for binding to (peptido)glycans. <I>Mol Microbiol</I> 68, 838-847.
                               [2] Kalyanasundram, J., Chia, S.L., Song, A.A., Raha, A.R., Young, H.A., and Yusoff, K. (2015). Surface display of glycosylated Tyrosinase related protein-2 (TRP-2) tumour antigen on Lactococcus lactis. <I>BMC Biotechnol</I> 15, 113.
                               [3] Le Loir, Y., Nouaille, S., Commissaire, J., Bretigny, L., Gruss, A., and Langella, P. (2001). Signal peptide and propeptide optimization for heterologous protein secretion in Lactococcus lactis. <I>Appl Environ Microbiol</I> 67, 4119-4127.
							   [4] Lee, S.Y., Choi, J.H., and Xu, Z. (2003). Microbial cell-surface display. <I>Trends Biotechnol</I> 21, 45-52.
							   [5] Raha, A.R., Varma, N.R., Yusoff, K., Ross, E., and Foo, H.L. (2005). Cell surface display system for Lactococcus lactis: a novel development for oral vaccine. <I>Appl Microbiol Biotechnol</I> 68, 75-81.
							   [6] Saleem, M., Brim, H., Hussain, S., Arshad, M., Leigh, M.B., and Zia ul, H. (2008). Perspectives on microbial cell surface display in bioremediation. <I>Biotechnol Adv</I> 26, 151-161.
                               </P></DIV></DIV></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                            filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                            -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                            -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                            box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Summary" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">Summary</P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="font-family:verdana;font-size:17px;">
						   With the NZ-Blue strain we created and the 4 plasmids (pCyt, pSec, pSD.1 and pSD.2) we provided, we can get devices knocked into <I>L. lactis</I> without antibiotics resistance genes. The recombinant strains can express protein by constitutive promoter (<I>P32</I>) and inductive promoter (<I>PnisZ</I>). If we use nisin to promote the <I>PnisZ</I> promotor, the resistant gene <I>nsr</I> will make the devices integrated strains to tolerate high levels of nisin concentration and protein of interest will functionally expressed at its right place. All these work made <I>L. lactis</I> the Mr. Right chassis in synthetic biology.
						</P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
							Figure 3.4 The improvements of the protein localization system. A). Integration plasmid for cytosol expression, pCyt (Left), and the device-inserted plasmid pCyt-Device (Right). B). Integration plasmid for secretion expression, pSec (Left), and the device-inserted plasmid pSec-Device (Right). C). Integration plasmid for surface display, pSD.1/pSD.2 (Left), and the device-inserted plasmid pSD.1-Device/pSD.2-Device (Right). Abbreviation: HisLA, left arm of His; HisRA, right arm of His; P, double promoter, <I>P32</I> (constitutive promoter) and <I>PnisZ</I> (inductive promoter); MCS, multi-clone site; <I>SPusp45</I>, signal peptide of <I>usp45</I> from <I>L. lactis</I>; cA; anchor motif of <I>acmA</I> from <I>L. lactis</I>; <I>nsr</I>, nisin resistant gene; <I>Ter</I>, terminator; Ts Ori, temperature-sensitive replicon; <I>Em<SUP>r</SUP></I>, erythromycin resistant gene. All devices were cloned into the SmaI and XbaI site of MCS of the 4 plasmids by seamless cloning.
							</P></DIV></DIV></DIV></DIV></SECTION><FOOTER><DIV style="padding:20px 0 0 0;margin:0;float:right;width:300px;color:#fff;"><UL style="font-size:14px;"><LI><SPAN style="text-decoration:underline;color:#fff;left:20px;">Contact us</SPAN></LI><LI><SPAN>Email:</SPAN><A href="Mailto:igem@tmmu.edu.cn" style="color:#fff">igem@tmmu.edu.cn</A></LI><LI><SPAN>Address:</SPAN> Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038</LI></UL></DIV></FOOTER></DIV></DIV><DIV class="card-holder hide"><DIV class="card-wrapper"><A onclick="$.scrollTo('#Background1',500)"><DIV class="card"><SPAN class="card-content">
                            1-Background
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Design1',500)"><DIV class="card"><SPAN class="card-content">
                            1-Design
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Result1',500)"><DIV class="card"><SPAN class="card-content">
                            1-Result
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Background2',500)"><DIV class="card"><SPAN class="card-content">
                            2-Background
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Design2',500)"><DIV class="card"><SPAN class="card-content">
                            2-Design
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Result2',500)"><DIV class="card"><SPAN class="card-content">
                            2-Result
                        </SPAN></DIV></A></DIV><DIV class="card-wrapper"><A onclick="$.scrollTo('#Summary',500)"><DIV class="card"><SPAN class="card-content">
                            Summary
                        </SPAN></DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>