<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TMMU_China_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TMMU China/Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
            Background
        </TITLE></P><DIV class="main-indents"><DIV class="my_main"><SECTION id="content"><DIV style="width=960px;padding: 0px; margin: 0; background-color: rgb(255,255,255); color: #000;"><HEADER><NAV style=""><UL class="sf-menu"><LI class="current"><A href="https://2016.igem.org/Team:TMMU_China">
                                    Home
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China">
                                            Abstract
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Background">
                                            Background
                                        </A></LI></UL><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                    Project &amp; Results

                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Project_Results">
                                            Visual selection
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Project2">
                                            Optimization
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Project_Results/Parts">
                                            Parts
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Notebook">
                                            Notebook
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Model">
                                    Modeling
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices">
                                    Human Practice
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Coloring" style="cursor:pointer;">
                                            Coloring book
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Construction" onclick="$.scrollTo('#Construction',500)" style="cursor:pointer;">
                                            Construction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Counseling" onclick="$.scrollTo('#Counseling',500)" style="cursor:pointer;">
                                            Counseling
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Collaborations" onclick="$.scrollTo('#Collaborations',500)" style="cursor:pointer;">
                                           Collaborations
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Human_Practices#Conferences" onclick="$.scrollTo('#Conferences',500)" style="cursor:pointer;">
                                           Conferences
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Safety">
                                    Biosafety
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                    Team
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Team">
                                            Introduction
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Team/Members">
                                            Team Members
                                        </A><A href="https://2016.igem.org/Team:TMMU_China/Attributions">
                                            Attributions
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                    Judging
                                </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging">
                                            Medal Criteria
                                        </A></LI><LI><A href="https://2016.igem.org/Team:TMMU_China/Judging/Engagement">
                                            Education &amp; Public Engagement award
                                        </A></LI></NAV></HEADER><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                        filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                        -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                        -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                        box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="chassis" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">
                                Synthetic biology chassis
                            </P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="font-family:verdana;font-size:17px;">
                                    In synthetic biology and the iGEM competition, scientists and iGEMers are familiar with the concept of chassis, the conceptual extension of a living cell, including <I>Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae</I>, and so on. Many contemporary synthetic biology researches rely on the use of bacterial or eukaryotic chassis for plugging-in and plugging-out genetic circuits and new-to-nature functionalities.
                                </P><P style="font-family:verdana;font-size:17px;">
                                    The molecular biology workhorse, <I>E. coli</I>, is an ideal chassis in synthetic biology owing to our deep understanding of its biologic characteristics, its convenience of genetic manipulation and the relatively a large number of well-studied gene regulatory systems. It is therefore not surprising that <I>E. coli</I> has dominated in the iGEM competition. However, the opportunistic pathogen nature of <I>E. coli</I>, the lipopolysaccarides (LPS) contamination, and the intolerance to physicochemical stress and harsh operating conditions  hinder it to be an ideal choice for industrial application. Thus, the host of many productions  has been switched from <I>E. coli</I> to <I>S. cerevisiae</I>, such as the anti-malaria drug, semi-synthetic artemisinin. (Figure 1)
                                </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
                                        Figure 1 Classic chassis used in synthetic biology
                                    </P></DIV></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;margin-top: 20px;
                        filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                        -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                        -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                        box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Lactic" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">
                                Lactic acid bacteria
                            </P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="font-family:verdana;font-size:17px;">
                                    Lactic acid bacteria (LAB) are a group of Gram positive, non-sporulating bacteria that include microorganisms from different genuses such as <I>Leuconostoc, Lactococcus, Lactobacillus, Pedicoccus</I>, and <I>Streptococcus</I>. LAB do not contain LPS attached to the cell membrane. The absence of such endotoxins avoids the possibility of anaphylactic shock when LAB are administered in humans. LAB has a long history of safe use in food production and preservation. It is used to ferment a range of raw materials, such as milk, which is used to produce cheese (species of <I>Lactococcus</I>) and yoghurt (species of <I>Streptococcus</I> and <I>Lactobacillus</I>). Some strains have also been used as probiotic bacteria producers of metabolites and macromolecules that are able to maintain and promote human health. LAB have been classified as food grade microorganisms, which are generally recognized as safe (GRAS) organisms by the US Food and Drug Administration (FDA) and fulfill criteria of the qualified presumption of safety (QPS) according to the European Food Safety Authority (EFSA).
                                </P><P style="font-family:verdana;font-size:17px;">
                                    Over the past decade, there has been an increasing interest in the use of LAB as a synthetic biology chassis. For instance, in therapeutics, LAB are used as live delivery vehicles for DNA, peptides, single-chain antibody fragments (ScFvs), antigens, cytokines, enzymes and allergens to treat or prevent infectious diseases, cancer, gastrointestinal tract (GIT) diseases, diabetes, and so on. (Figure 2) In iGEM, several teams have tried to ameliorate LAB to be new chassis in synthetic biology. In 2013, Uppsala-Sweden team designed and tested many promoters, reporter genes, vectors, toxin-antitoxin systems, and signal peptides to make <I>Lactobacillus</I>  new probiotic chassis in synthetic biology. They submitted about 70 parts in different series which are subsequently widely used by many iGEM teams. Moreover, many teams have applied LAB in different tracks for different tasks. <I>Lactobacillus bulgaricus</I> was used to build a sustainable and savory drug delivery system (Massachusetts Institute of Technology-USA, 2008). The lycopene and astaxanthin pathway of <I>Lactobacillus casei.</I> was engineered by Uppsala University-Sweden in 2013). <I>Lactococcus lactis</I> was used to create a smart bandage that can heal burn wound and fight against bacterial infections (Groningen University- Netherlands, 2014).
                                </P><DIV style="text-align:center;margin-bottom:20px"><P style="text-align:center;">
                                            Figure 2 Current applications of lactic acid bacteria
                                        </P></DIV></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;
                        filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                        -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                        -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                        box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="NZ9000" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90"><I>Lactococcus lactis</I> and <I>L. lactis</I> NZ9000
                            </P><DIV style="margin-left:20px;margin-right:20px;font-size:18px;"><P style="font-family:verdana;font-size:17px;"><I>L. lactis</I> is one of the best characterized LAB with detailed knowledge on genetics, metabolism, and biodiversity. <I>L. lactis</I> is of crucial importance for manufacturing dairy products, such as buttermilk and cheese. <I>L. lactis</I> has once been named the official state microbe of Wisconsin, the number one cheese-producing state in the United States. In addition to its traditional use in food fermentations, <I>L. lactis</I> has increasingly been used in modern biotechnological applications, including cancer, diabetes, vaccination, infection protection and animal diseases. <I>L. lactis</I> has also become famous as the first genetically modified organism to be used alive for the treatment of human disease (Recombinant <I>L. lactis</I> secreting IL-10 for the treatment of Crohn disease, ClinicalTrials.gov Identifier: NCT00729872). In the iGEM competition, the 2014 Groningen University- Netherlands team engineered <I>L. lactis</I> to detect burn wound bacteria pathogens <I>S. aureus</I> and <I>Pseudomonas aeruginosa (P. aeruginosa)</I> and to secret antimicrobial peptides and biofilm-dissolving proteins to fight against <I>S. aureus</I> and <I>P. aeruginosa</I> infections at the site of burn wounds. They also created a smart bandage to contain the engineered <I>L. lactis</I>. The 2015 Utah State University-USA team created phage resistant <I>L. lactis</I> by integrating reporter system and kill switch that can be applied in cheese industry. The 2015 Concordia University-Canada team tried to develop a protein scaffold that can be customized and used by synthetic biologists to perform various tasks. 
                                </P><P style="font-family:verdana;font-size:17px;"><I>L. lactis</I> is the first LAB whose genome was fully sequenced. It is easy to be manipulated with many cloning and expression tools, including the nisin controlled expression (NICE) system, the Cre-lox system, the single stranded DNA recombineering method and so on. Among them, the NICE system is the most widely used protein expression system for <I>L. lactis</I>. (Figure 3) It is based on the control of strong nisin inducible promoters (<I>PnisA</I> or <I>PnisZ</I>) and the cognate signal transduction NisKR two-component system. The integration of the nisKR genes downstream of the pepN gene of the international prototype MG1363 yielded the <I>L. lactis</I> NZ9000 strain. Since then, <I>L. lactis</I> NZ9000 has been distributed to laboratories worldwide and has become the “laboratory workhorse” for protein expression and has been used in many physiological studies.
                                </P><DIV style="text-align:center;margin-bottom:20px;align=center"><P style="text-align:center;">
                                        Figure 3 The nisin controlled expression (NICE) system
                               </P></DIV><P style="font-family:verdana;font-size:17px;">
                                    However, few efforts have been made to improve <I>L. lactis</I> NZ9000 to be a better synthetic biology chassis. Firstly, easy selection and efficient device knocked-in system  have not been established in NZ9000 currently. Secondly, because the NICE system inducer nisin is also a food grade antibacterial peptide, the intolerance of NZ9000 to high levels of nisin concentration compromises the gene expression efficiency. Thirdly, to make proteins of interest function at different places, secretion or surface display systems should also be optimized. Based on the situations above, we aimed to build and optimize  the systems above to make <I>L. lactis</I> NZ9000 a better chassis in synthetic biology.
                                </P></DIV></DIV><DIV style="width:938px;height:auto;border:#86C166 2px solid;background:#fff;margin-left: auto;margin-right: auto;margin-top: 20px;margin-bottom:20px;
                        filter:progid:DXImageTransform.Microsoft.Shadow(color=#0C4842,direction=120,strength=3);/*ie*/
                        -moz-box-shadow: 2px 2px 10px #0C4842;/*firefox*/
                        -webkit-box-shadow: 2px 2px 10px #0C4842;/*safari或chrome*/
                        box-shadow:2px 2px 10px ##0C4842;/*opera或ie9*/"><P id="Reference" style="margin-left:20px;margin-top:20px;font-size:30px;font-family:verdana;color:#00AA90">
                                References
                            </P><DIV style="margin-left:20px;margin-right:20px;font-family:verdana;font-size:16px;"><P> [1] Cano-Garrido, O., Seras-Franzoso, J., and Garcia-Fruitos, E. (2015). Lactic acid bacteria: reviewing the potential of a promising delivery live vector for biomedical purposes. Microb Cell Fact 14, 137.
[2] Danchin, A. (2012). Scaling up synthetic biology: Do not forget the chassis. FEBS Lett 586, 2129-2137.
[3] Gaspar, P., Carvalho, A.L., Vinga, S., Santos, H., and Neves, A.R. (2013). From physiology to systems metabolic engineering for the production of biochemicals by lactic acid bacteria. Biotechnol Adv 31, 764-788.
[4] Linares, D.M., Kok, J., and Poolman, B. (2010). Genome sequences of Lactococcus lactis MG1363 (revised) and NZ9000 and comparative physiological studies. J Bacteriol 192, 5806-5812.
[5] Wells, J.M., and Mercenier, A. (2008). Mucosal delivery of therapeutic and prophylactic molecules using lactic acid bacteria. Nat Rev Microbiol 6, 349-362.
</P></DIV></DIV></DIV></SECTION><FOOTER><DIV style="padding:20px 0 0 0;margin:0;float:right;width:300px;color:#fff;"><UL style="font-size:14px;"><LI><SPAN style="text-decoration:underline;color:#fff;left:20px;">Contact us</SPAN></LI><LI><SPAN>Email:</SPAN><A href="Mailto:igem@tmmu.edu.cn" style="color:#fff">igem@tmmu.edu.cn</A></LI><LI><SPAN>Address:</SPAN> Third Military Medical University, No.30 Gaotanyan Street Shapingba District, Chongqing, P.R.China 400038</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>