Team:TU Darmstadt/Project/Bio/Safety
Home
SLA 3D-Printer
LabSurfing
Bricks
RnD-Nerf
Sitemap
Team
Contact
PROJECT
Biotechnology
Monomers
Ethylene Glycol
Itaconic Acid
Xylitol
Results
In Vitro Degradation
In Vitro Bioreactor
Xylan Degradation
Polyester Degradation
Safety
Killswitch Design
Results
Modeling
Riboswitch Design
Structure Prediction
Membrane Modeling
Chemistry
Engineering
Software
Hardware
Hardening and Printing
Policy & Practices
LabSurfing
Application
Techno-moral scenarios
Expert talks & Conference
PARTS
Part Collection
NOTEBOOK
Labjournal
Methods & Materials
COLLABORATIONS
Testing
"Escape the lab"-Tour
Open Science
ATTRIBUTIONS
TEAM
Allocations
Project Management in Science
Acknowledgements
GALLERY
SLA 3D Printer
Lab Impressions
Workshops
Experts
LabSurfing
SITEMAP
Safety
Introduction
Biosafety and biocontainment are important aspects to consider when using transgenic oganisms for production processes. In order to reduce the risk of uncontrolled proliferation of our bacteria we followed an active containment strategy for our iGEM project. Utilizing the toxic polypeptide hokD we designed a killswitch that induces cell death of our production bacteria in the case of an unintentional release into the environment. The hokD gene originates from the relB-Operon of E. coli and was introduced to the BioBrick registry by the iGEM team of TU Darmstadt in 2014. The polypeptide shows 40% homology to the polypeptide of the host killing (hok) gene and leads to the same cell death by loss of membrane potential when overexpressed[1].
One of the most important aspects to consider for the use of a toxic gene product to induce cell death is a tight regulation of toxin production in order to avoid unwanted cell death under production or laboratory conditions. In our last yearâ€™s project we were able to control the expression of hokD by cloning it downstream of the araC-regulated pBAD-Promoter ( BBa_K808000 ) which is tight in the presence of glucose and can be induced by the addition of arabinose.  In our project this year we wanted to go a step further by using a riboregulator for posttranscriptional regulation of the hokD-expression.
We chose a trans-activating riboregulator which is based in the interaction of two short RNA-sequences: the cis-repressing RNA (crRNA) and the trans-activating RNA (taRNA). The crRNA is fused upstream of the gene of interest (GOI) and masks the ribosome binding site (RBS) by forming a hairpin-secondary structure which prevents the translation of the GOI-mRNA. If the taRNA is present it forms a RNA-RNA-complex with the crRNA. This leads to a helix shift and the release of the RBS and therefore to GOI expression[2].
It has been shown that riboregulators allow a tight control of peptide production with low basal expression levels which makes them best suited for designing killswitches[3].
Figure 1: Interaction of taRNA and crRNA leads to expression of GOI.
References
[1] Knudsen, S., et al., Development and testing of improved suicide functions for biological containment of bacteria. Appl Environ Microbiol, 1995. 61(3): p. 985-91.
[2] Suess, B., et al., A theophylline responsive riboswitch based on helix slipping controls gene expression in vivo. Nucleic Acids Research, 2004. 32(4): p. 1610-1614.
[3] Callura, J.M., et al., Tracking, tuning, and terminating microbial physiology using synthetic riboregulators. Proceedings of the National Academy of Sciences, 2010. 107(36): p. 15898-15903.
