Team:TU Darmstadt/Project/Bio/InVitroDegradation/sec2
Home
SLA 3D-Printer
LabSurfing
Bricks
RnD-Nerf
Sitemap
Team
Contact
PROJECT
Biotechnology
Monomers
Ethylene Glycol
Itaconic Acid
Xylitol
Results
In Vitro Degradation
In Vitro Bioreactor
Xylan Degradation
Polyester Degradation
Safety
Killswitch Design
Results
Modeling
Riboswitch Design
Structure Prediction
Membrane Modeling
Chemistry
Engineering
Software
Hardware
Hardening and Printing
Policy & Practices
LabSurfing
Application
Techno-moral scenarios
Expert talks & Conference
PARTS
Part Collection
NOTEBOOK
Labjournal
Methods & Materials
COLLABORATIONS
Testing
"Escape the lab"-Tour
Open Science
ATTRIBUTIONS
TEAM
Allocations
Project Management in Science
Acknowledgements
GALLERY
SLA 3D Printer
Lab Impressions
Workshops
Experts
LabSurfing
SITEMAP
Contents
1 Xylan Degradation
1.1 Introduction
1.2 Experimental approach
1.3 Results
1.4 References
Xylan Degradation
Introduction
Xylan defines a group of hemicelluloses, polysaccharides based on xylose, mainly occurring in plant cell walls. In hardwoods up to 35 % of the whole hemicellulose content are Xylanes (1). Also the Xylose Monomer is suitable for generation of photopolymers and therefore of use in our project. For xylan degradation three enzymes were chosen, which cleave either the xylan main chain (xynA (2)) or side chains (aes(3), Ruxyn1). The side chains of xylan vary by the type of utilized xylan. Xylan of hardwood mainly has acetyl groups, while softwood xylan lacks acetyl groups but has arabinose residues instead. Both xylan types are targeted by aes (degradation of acetyl residues) and Ruxyn1 (degradation of arabinose residues).Luckily all three degradation enzymes were already listed in the iGEM registry and ready for usage in our model system. The parts we used were BBa_K805012 , BBa_K1216002 , BBa_K1175005 .
Figure 1 Xylan structure.(source: https://en.wikipedia.org/wiki/Xylan#/media/File:Xylan.svg)
Experimental approach
In this model degradataion pathway the protein scaffold by iGEM TU Darmstadt 2014 (based on Dueber et al., 2009 (7)) was fused with a 15 amino acid long peptide (silica-tag) (6), binding to silica surfaces as shown by iGEM Leeds 2013. Green Fluorescent Protein (GFP) was fused with the silica-tag for proving silica-binding abilities of the peptide. For easy purification of both fusion proteins N-terminal polyhistidine tags were added. For increasing the degradation efficiency of xylan, protein scaffolding was prepared by fusing the cognate binding peptides of the three protein domains GBD, SH3 and PDZ to the degradation enzymes Ruxyn1, aes and xynA, respectively. All coding sequences were assembled in an operon under the control of a T7 promoter system, which was co-transformed with the scaffolding plasmid into E. coli Top10. In the next step three different characterization elements for later on optimization are important:
1) To check the binding to a silica-surface and stability of boundary we fused the Si4-tag by iGEM Leeds 2013 (2) with GFP ( BBa_E0040 ), YFP ( BBa_E0030 ) and CFP ( BBa_E0020 ) using Overlap PCR / Biobrick Assembly. After Expression and Purification we want to visualize the boundary to a silica-coated surface in different buffer conditions and time dependency off the boundary.
2) By linking the zinc finger ligand sequence to GFP (BBa_E0040), YFP (BBa_E0030) and CFP (BBa_E0020) and combining purified protein scaffold domains with the fluorescent proteins we want to control the boundary of ligand scaffold domain in different buffer conditions.
3) To verify the advantage of our protein scaffolding we want to compare the yield of degrading xylan only based on the three degradation enzymes in one composite to the combination of the composite and the protein-scaffold.
Results
After we verified the correctness of the scaffold construct sequence it was transformed into E. coli BL21 from which we isolated proteins after induction for 3 hours at 37°C with 1mM IPTG. Afterwards cells were harvested and lysed. By SDS-PAGE we were able to validate the overexpression of the protein scaffold in E. coli. As mentioned before we added a Si4-silica tag to the scaffold sequence to enable enzyme immobilization. In this way proteins responsible for xylan degradation are brought into near proximity to facilitate reaction flow. Shown is a SDS-PAGE containing samples taken before and after induction of Scaffold-Si4 protein expression.
Figure 1 Scan of the SDS PAGE containing the marker (M), PageRuler Prestained ProteinLadder (Thermoscientific) and the harvested cells after (1) and before induction (2) with IPTG
Exemplarily the overexpression of degradation enzyme Acetyl Esterase fused to a linker binding the SH3 domain of the protein scaffold is shown.
Figure 2 Scan of the SDS PAGE containing the marker (M), PageRuler Prestained ProteinLadder (Thermoscientific) and the harvested cells after (1) and before induction (2) with IPTG
References
Sixta, Herbert, ed. (2006). Handbook of pulp 1. Weinheim, Germany: Wiley-VCH Verlag GmbH & Co. KGaA. pp. 28–30
M.G. Paice, R. Bourbonnais, M. Desrochers, L. Jurasek, M. Yaguchi, A xylanase gene from Bacillus subtilis: nucleotide sequence and comparison with B. pumilus gene. Arch. Microbiol., 144, 201-206(1986)
R1 Peist, A Koch, P Bolek, S Sewitz, T Kolbus, W Boos, Characterization of the aes gene of Escherichia coli encoding an enzyme with esterase activity. J Bacteriol. 1997, 179:7679-7686.
J. Zhou, L. Bao, L. Chang, Y. Zhou and H. Lu, Biochemical and kinetic characterization of GH43 beta-D-xylosidase/alpha-L-arabinofuranosidase and GH30 alpha-L-arabinofuranosidase/beta-D -xylosidase from rumen metagenome. J. Ind. Microbiol. Biotechnol. 39, 143-152 (2012)
R. R. Naik, L. L. Brott, S. J. Clarson, M. O. Stone, Silica-precipitating peptides isolated from a combinatorial phage display peptide library. J Nanosci Nanotechnol 2, 95-100 (2002).
R. J. Conrado et al., DNA-guided assembly of biosynthetic pathways promotes improved catalytic efficiency. Nucleic Acids Res 40, 1879-1889 (2012).
J. E. Dueber et al., Synthetic protein scaffolds provide modular control over metabolic flux. Nat Biotechnol 27, 753-759 (2009).
