Team:TU Darmstadt/Project/Bio/Monomeres/EthyleneGlycol
Home
SLA 3D-Printer
LabSurfing
Bricks
RnD-Nerf
Sitemap
Team
Contact
PROJECT
Biotechnology
Monomers
Ethylene Glycol
Itaconic Acid
Xylitol
Results
In Vitro Degradation
In Vitro Bioreactor
Xylan Degradation
Polyester Degradation
Safety
Killswitch Design
Results
Modeling
Riboswitch Design
Structure Prediction
Membrane Modeling
Chemistry
Engineering
Software
Hardware
Hardening and Printing
Policy & Practices
LabSurfing
Application
Techno-moral scenarios
Expert talks & Conference
PARTS
Part Collection
NOTEBOOK
Labjournal
Methods & Materials
COLLABORATIONS
Testing
"Escape the lab"-Tour
Open Science
ATTRIBUTIONS
TEAM
Allocations
Project Management in Science
Acknowledgements
GALLERY
SLA 3D Printer
Lab Impressions
Workshops
Experts
LabSurfing
SITEMAP
Biotechnological production of ethylene glycol in Escherichia coli
Figure 1: ethylene glycol pathway
For the generating of a monomer toolbox using polyesterification reactions a dialcohol is essential, as it can build up chains with acid groups in a condensation reaction. Ethyleneglycol (EG) is the smallest stable dialcohol and is already used as a basic chemical for industrial polymer production. For example, the widely used polyethylene terephthalate (PET) includes ethylene as a monomer.
 
Ethyleneglycol is currently produced in an amount of approximately 21 million tons per year by a reaction of ethyleneoxide and water. Although first biotechnological factories started production in Taiwan in 2011, their output is not comparable to the chemically synthesized amount. Moreover, the used process bases on sugarcrane which is problematic concerning the pressure this might put on prices for food products.(3)
 
A future production which bases on renewable non-food resources could be the fermentation of xylose with engineered E. coli expressing D-xylose dehydrogenases.(1)
 
Our pathway bases on the previously published paper by Liu et al. (1) which showed a possible EG production in     E. coli by introducing the D-xylose dehydrogenase xylB (BBa_K1602009) . In order to provide the pathway in different model organisms we added the remaining enzymes which are normally encoded in the E. coli genome. Furthermore a D-xylonolactone lactonase was added.(2)
 
The first reaction is performed by xylB. It is a NAD+ dependant dehydrogenase and catalyzes the reaction of D-xylose to D-xylonolactone. This lactone can either be hydrolyzed to D-xylonic acid spontaneously or actively by a D-xylonolactonase. In order to prevent a possible bottleneck at this step the lactonase xylC (BBa_K1602010) was added to the pathway. Both enzymes original host is Caulobacter crescentus and for the use in E. coli we used synthesized and codon optimized versions of the genes.
 
In the next step D-xylonic acid does react to 2-dehydro-3-deoxy-D-pentonate in a reaction catalyzed by the dehydratase yjhG (BBa_K1602012) . From there yagE (BBa_K1602011) performs a cleavage at the C3-C4 bond and forms glycoaldehyde and pyruvate. While pyruvate can be metabolized in the citric acid cycle, glycoaldehyde is reduced to ethyleneglycol by the NADP+ dependent aledhyde reductase YqhD (BBa_K1602013) .
Liu H, Ramos KR, Valdehuesa KN, Nisola GM, Lee WK, Chung WJ. Biosynthesis of ethylene glycol in Escherichia coli. Appl Microbiol Biotechnol. 2013;97(8):3409-17.
Toivari MH, Nygard Y, Penttila M, Ruohonen L, Wiebe MG. Microbial D-xylonate production. Appl Microbiol Biotechnol. 2012;96(1):1-8.
http://www.chemicals-technology.com/projects/greencolbiomegplant/
