Team:TU Darmstadt/Notebook/sec1/K1602002
Home
SLA 3D-Printer
LabSurfing
Bricks
RnD-Nerf
Sitemap
Team
Contact
PROJECT
Biotechnology
Monomers
Ethylene Glycol
Itaconic Acid
Xylitol
Results
In Vitro Degradation
In Vitro Bioreactor
Xylan Degradation
Polyester Degradation
Safety
Killswitch Design
Results
Modeling
Riboswitch Design
Structure Prediction
Membrane Modeling
Chemistry
Engineering
Software
Hardware
Hardening and Printing
Policy & Practices
LabSurfing
Application
Techno-moral scenarios
Expert talks & Conference
PARTS
Part Collection
NOTEBOOK
Labjournal
Methods & Materials
COLLABORATIONS
Testing
"Escape the lab"-Tour
Open Science
ATTRIBUTIONS
TEAM
Allocations
Project Management in Science
Acknowledgements
GALLERY
SLA 3D Printer
Lab Impressions
Workshops
Experts
LabSurfing
SITEMAP
K1602002 - B0034-
D
-Xylonic Acid Dehydratases (B0034-yjhG)
Figure 1 Gel of B0034-yjhG. The size of the amplified product was around 2,2 kbp. DNA marker: 2-Log DNA Ladder (NEB).
The BBa_B0034 ribosomal binding site was added upstream of the yjhG -gene by cloning the gene in the pSB1A2‑vector containing the RBS.
The pSB1A2‑vector was cut using SpeI and PstI and later dephosphorylated. The gene was digested using XbaI and SpeI and ligated in the vector with T4‑ligase. After transformation in E. coli Top 10 the resulting colonies were screened for positive clones by performing colony PCRs with VF2 and VR oligonucleotides. The plasmids of positive clones were purified and the inserts verified by sanger-sequencing. Afterwards the pSB1A2‑vector was replaced by the pSB1C3 vector using EcoRI and SpeI. The dephosphorylation and ligation were performed.
