<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="height: 0px;"><UL class="nav navbar-nav"> a
{
    color: #e35c5c;
}
.col-md-4 a{
    color: #e35c5c;
}
.nav &gt; li &gt; a:focus, .nav &gt; li &gt; a:hover, .nav-pills &gt; li.active &gt; a
{
    background-color: #e35c5c;
}


#HQ_page th
{
 background-color: #e35c5c;
 padding: 10px;
 color: #292000;
 border: none;
 font-size: 2rem;
}
#HQ_page td
{
 color: #292000;
 border: none;
 font-family: 'Didact Gothic';
 font-size: 2rem;
}

#HQ_page table
{
    border: none;
}

#hq_page p
{
    font-size: 2rem;
    
}
div p
{
    color: #292000;
}

.col-md-8 a:visited{
    color: #e35c5c;
}

h2, h3, h4{
    color: #e35c5c;
}
ol li{
    color: #000;
}
.column li a:hover{
    color: #FFF;
}
<DIV class="column"><DIV class="row"><DIV class="col-md-8"><H2>Parts</H2><P>During the course of our project we uploaded a number of parts to the <A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2016&amp;group=Newcastle"> registry </A>. These parts can be seen in the table below, and in more detail further down this page. </P><P><FIGCAPTION>Table 1: Complete list of parts added to the registry</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Type</TH><TH>Description</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895000"> BBa_K1895000</A></TD><TD>Project</TD><TD>Heat induced promoter system v1</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895001"> BBa_K1895001 </A></TD><TD><EM>RBS</EM> RBS</TD><TD><I>E. coli's</I> natural ribosome binding site - rpoH</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895002"> BBa_K1895002</A></TD><TD>Regulatory</TD><TD><EM>dnaK</EM> promoter</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895003"> BBa_K1895003</A></TD><TD>Regulatory</TD><TD><EM>htpG</EM> promoter</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895004"> BBa_K1895004</A></TD><TD>Composite</TD><TD>Porin Overexpression for Microbial Fuel Cell Use</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895005">BBa_K1895005</A></TD><TD>Composite</TD><TD>Controlled OrpF Expression Device for Microbial Fuel Cell</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895006">BBa_K1895006</A></TD><TD>Composite</TD><TD>Alternative electrically induced expression system using the dnaK promoter</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895007">BBa_K1895007</A></TD><TD>Coding</TD><TD>Sequence for σ<SUP>32</SUP> transcription factor</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895995">BBa_K1895995</A></TD><TD>Composite</TD><TD>Time Delay Service</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895996">BBa_K1895996</A></TD><TD>Composite</TD><TD>Red light sensitive 'LDR'</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895997">BBa_K1895997</A></TD><TD>Composite</TD><TD>Blue sensitive 'LDR'</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895998">BBa_K1895998</A></TD><TD>Coding</TD><TD>SmtA metallothionein and LVA </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895999">BBa_K1895999</A></TD><TD>Composite</TD><TD>Arabinose controlled 'resistor'</TD></TR></TBODY></TABLE><DIV id="bio-bulb"><H3>P<EM><SUB>htpG</SUB></EM> Heat Induced 'Light Bulb' - <A href="http://parts.igem.org/Part:BBa_K1895000"> BBa_K1895000</A></H3><P>We aimed to engineer <SPAN class="species-name">Escherichia coli</SPAN> (<SPAN class="species-name">E. coli</SPAN>) such that it fluoresces when an electrical current is passed through its growth medium, via the use of an inducible promoter responsive to one of the effects of electricity passing through a liquid, i.e. heat.</P><P>HtpG is a stress protein in <SPAN class="species-name">E. coli</SPAN>, produced during conditions of cellular stress including heat. Previous work has shown that <SPAN class="gene-name">htpG</SPAN> expression increases under high temperature conditions, for example, 45 °C. The promoter P<SUB><SPAN class="gene-name">htpG</SPAN></SUB> lies upstream of <SPAN class="gene-name">htpG</SPAN> and regulates the gene’s expression in response to heat.</P><P>The <SPAN class="species-name">E. coli</SPAN> sigma factor 32 is required to bind to RNA polymerase holoenzyme in order to drive transcription of genes downstream of P<SUB><SPAN class="gene-name">htpG</SPAN></SUB>. Sigma<SUP>32</SUP> (σ<SUP>32</SUP>) is a product of the <SPAN class="gene-name">rpoH</SPAN> gene.  Our design includes additional restriction sites  up and downstream of <SPAN class="gene-name">rpoH</SPAN> (BamHI and BglII respectively) to allow us to either remove <SPAN class="gene-name">rpoH</SPAN>, or add further copies of this gene to the construct. The rationale behind this is to tune the sensitivity of the P<SUB><SPAN class="gene-name">htpG</SPAN></SUB> heat shock response via the cellular abundance of σ<SUP>32</SUP> and RNA polymerase activity. For example, having extra copies of <SPAN class="gene-name">rpoH</SPAN> expressed as a result of a small variation in heat would act as a positive feedback loop to amplify P<SUB><SPAN class="gene-name">htpG</SPAN></SUB> activity and accordingly, the overall expression of the downstream genes in our operon construct.</P><P>To monitor the response of cells to the heat shock response, and to produce a ‘lightbulb’, we will use Green Fluorescent Protein (GFP). GFP is a jellyfish derived protein often used as a reporter of promoter activity.  We are using a variation of GFP, the superfolder GFP (sfGFP) as this generates a greater fluorescent signal when excited with light at 485 nm. In terms of this experiment, sfGFP will be used to ensure the construct works and that our cells are indeed responsive to an electric current via heat shock stimulation of P<SUB><SPAN class="gene-name">htpG</SPAN></SUB>. Higher heat-shock temperatures would hopefully result in greater <SPAN class="gene-name">sfgfp</SPAN> expression due to higher P<SUB><SPAN class="gene-name">htpG</SPAN></SUB> activity, measured through plate reader assays.  As with the <SPAN class="gene-name">rpoH</SPAN> gene, we have incorporated two restriction sites (BsmBI for Golden Gate cloning) up and downstream of the gene. This will allow us to change sfGFP for other reporters, such as luciferase. Luciferase would allow the emission of light like an actual lightbulb unlike sfGFP which requires an excitation wavelength of light to generate fluorescence.</P><P>Our design utilizes the natural ribosome binding site (RBS) present upstream of <SPAN class="gene-name">rpoH</SPAN> in <SPAN class="species-name">E. coli</SPAN>, which we will be adding to the registry (<A href="http://parts.igem.org/Part:BBa_K1895001">BBa_K1895001</A>). This is to ensure that the ribosome does bind to the mRNA and translate the protein correctly without potential adverse effects that a non-native RBS may have. We will however make variants of the RBS with two different medium bicistronic RBS in order to modulate the transcription of the transcribed mRNA encoding σ<SUP>32</SUP>. The medium strength bicistronic RBS will avoid the problem of placing too high a translational burden on the cell. We will then test all three variants to determine the best RBS to use in the final design.</P><P>Upstream of <SPAN class="gene-name">sfGFP</SPAN>, we will use the <A href="http://parts.igem.org/Part:BBa_0034">BBa_0034</A> high strength RBS for maximum translation of <SPAN class="gene-name">sfGFP</SPAN>.</P><H4>Our Construct</H4><P><FIGURE><FIGCAPTION>Figure 1a: SBOL Visual diagram of our lightbulb construct as synthesized, without terminator.</FIGCAPTION></FIGURE></P><P><FIGCAPTION>Figure 1b: SBOL Visual diagram of our lightbulb construct as designed with a terminator part. BioBrick prefix and suffix restriction sites also shown.</FIGCAPTION></P><P>Caution: When getting our lightbulb construct synthesised, we encountered issues with the synthesis of the terminator (part <A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A>) used in this design. As this was delaying our progress, we decided to request the construct without a terminator, and then to clone this in ourselves at a later point. Unfortunately due to running out of time we were unable to do this and so our final construct as used and characterised does not include a terminator downstream of <SPAN class="gene-name">sfGFP</SPAN>. We have detailed this on the parts page of the registry. If you use our construct, consider adding <A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A> as we planned or using an alternative such as <A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A> which we use in later designs.</P></DIV><P><FIGCAPTION>Table 2: component parts required for the construction of part BBa_K1895000</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J45504">BBa_J45504</A></TD><TD><EM>E. coli</EM> PhtpG promoter</TD><TD>This is the natural P<EM><SUB>htpG</SUB></EM> promoter found in <EM>E. coli</EM></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895001">BBa_K1895001</A></TD><TD><EM>rpoH</EM> RBS</TD><TD>The natural RBS site upstream of <EM>rpoH</EM> gene in <EM>E. coli</EM></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1895007">BBa_K1895007</A></TD><TD><EM>rpoH</EM></TD><TD>The <EM>E. coli</EM><EM>rpoH</EM> gene encoding σ<SUP>32</SUP>. We refactored the natural sequence to remove an illegal BioBrick PstI restriction site</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A></TD><TD>RBS</TD><TD>Elowitz high strength RBS</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A></TD><TD>sfGFP</TD><TD>Superfolder Green Fluorescent Protein coding sequence</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A></TD><TD>Terminator</TD><TD>Artificial large terminator. See above for details on our use of this part</TD></TR><TR><TD><A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A></TD><TD>Backbone</TD><TD>We are using the standard BioBrick backbone part pSB1C3 as this will make it easier to submit the part to the registry at a later date.</TD></TR></TBODY></TABLE><DIV id="alt-bulb"><H3>Alternative 'Light Bulb' construct using P<EM><SUB>dnaK</SUB></EM>  inducible promoter - <A href="http://parts.igem.org/Part:BBa_K1895006"> BBa_K1895006</A></H3><P>One of the concerns we had with our lightbulb construct is that, due to the inclusion of the <SPAN class="gene-name">rpoH</SPAN> coding sequence we would end up causing the expression of <SPAN class="gene-name">sfgfp</SPAN> without inducing the circuit through heat. We thought that this could occur because it is impossible for any promoter to be fully 'off', that is, there is always some constituitive expression. As a result of this constituitive expression <SPAN class="protein-name">σ<SUP>32</SUP></SPAN> would be produced which would in turn induce the htpG promoter causing run-away amplification of the circuit and high levels of <SPAN class="gene-name">sfgfp</SPAN> expression.</P><P>To preempt these difficulties we also designed a construct without amplification using an alternative heat-shock promoter, the <A href="http://parts.igem.org/Part:BBa_K1895002">dnaK promoter</A>. The role of dnaK is that of a protein chaperone, it binds to proteins that have misfolded due to the stress. These proteins are then refolded in an HSP-mediated process or degraded <A href="https://www.researchgate.net/publication/12525083_The_heat_shock_response_of_Escherichia_coli">(Arsène et al., 2000)</A>. Like other members of the Hsp70 family, <I>dnaK</I> in <I>E. coli</I> is upregulated by heat stress <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC248460/">(Ritossa, F., 1996)</A>. Consequently, its promoter must be stronger during heat shock to allow for overexpression. As heat stress can be triggered by the joule effect heating, that is heat produced by passing an electric current through a medium, systems with dnaK as their promoter we hypothesise be able to be induced electrically. We designed this construct to test that hypothesis.</P><H4>Our Construct</H4><P><FIGURE><FIGCAPTION>Figure 2a: SBOL Visual diagram of our alternative lightbulb construct using the dnaK promoter.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 2b: SBOL Visual diagram of our alternative lightbulb construct with parts registry numbers shown.</FIGCAPTION></FIGURE></P><P><FIGCAPTION>Table 3: Genetic parts required for the assembly of the P<EM><SUB>dnaK</SUB></EM> alternative lightbulb construct</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0080">BBa_R0080</A></TD><TD>dnaK Promoter</TD><TD>This is the dnaK promoter, it is upregulated in <SPAN class="species-name">E. coli</SPAN> by heat shock causing increased downstream expression.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B0034">BBa_R0034</A></TD><TD>Elowitz RBS</TD><TD>This is a standard RBS based that used in the construction of the repressilator (Elowitz, 1999). We chose it as it has a high efficiency RBS and is widely used in iGEM projects.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A></TD><TD>Superfolder GFP</TD><TD>This is the coding sequence for super folder GFP. We have chosen to use
            this as our reporter because it can easily be quantified using a plate by
            taking the OD600 measurement. This is harder to quantify with more visible
            reporters like amilCP.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0012">BBa_B0012</A></TD><TD>Terminator</TD><TD>This is a standard, widely used terminator in iGEM projects. We switched to using this terminator when we had synthesis problems with the double loop terminator used in our other constructs (<A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A>).</TD></TR></TBODY></TABLE><DIV id="bio-varistor"><H3>Arabinose Controlled 'Variable Resistor'- <A href="http://parts.igem.org/Part:BBa_K1895999">BBa_K1895999</A></H3><P>One of the key components in an electrical circuit is the resistor, used to reduce current flow and adjust output signal levels. We plan to produce a device within <SPAN class="species-name">Escherichia coli</SPAN> to generate a response analagous to a variable resistor. Our device will aim to vary the amount of free ions in an electrolyte. Ion uptake will be controlled by the expression of <SPAN class="gene-name">smtA</SPAN>. SmtA is a metallothionein that can bind to heavy metal ions like Cadmium (II), Zinc (II) and Copper (II).</P><P>SmtA has been used in a number of iGEM projects and is in the registry (<A href="http://parts.igem.org/Part:BBa_K519010">BBa_K519010</A>). It has previously been used in experiments for Cadmium (II) uptake, see <A href="https://2011.igem.org/Team:Tokyo-NoKoGen/metallothionein">Tokyo-NokoGen 2011</A>. We will be examining firstly, the impact of SmtA of Zinc (II) concentration rather than Cadmium (II) and then the impact that this has on the resistivity of the Zinc (II) containing media. In this instance we will be using Zinc sulfate (ZnSO<SUB>4</SUB>) in solution where it disassociates into Zn<SUP>2+</SUP> and SO<SUB>4</SUB><SUP>2-</SUP> ions. Various concentrations of Zinc sulfate have <A href="http://sites.chem.colostate.edu/diverdi/all_courses/CRC%20reference%20data/electrical%20conductivity%20of%20aqueous%20solutions.pdf">known electrical conductivity</A>. When <SPAN class="gene-name">smtA</SPAN> is expressed it will render the Zn<SUP>2+</SUP> unavailable and thereby reduce the conductivity of the solution.</P><P>We will be placing <SPAN class="gene-name">smtA</SPAN> under the control of an AraC regulated promoter (P<SUB>Bad)</SUB> allowing the expression of <SPAN class="gene-name">smtA</SPAN> to be controlled by the addition or removal of arabinose.</P><H4>Our Construct</H4><P>We will be synthesising two variants of our construct, one with and one without an ssRA degradation tag. The presence of a degradation tag causes the protein to be degraded by one of two proteases ClpXP or ClpAP. Their normal function is to prevent the ribosome from becoming ‘stuck’ on truncated mRNA. The addition of a degradation tag will allow us to see if we gain finer control over the resistance by increasing the rate of protein degradation. Both variants of the construct will include a polyhistidine-tag to allow for protein purification from cultures.</P><FIGURE><FIGCAPTION>Figure 3a: The design of our arabinose controlled 'varistor'.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3b: Our arabinose controlled 'varistor' with parts registry numbers included.</FIGCAPTION></FIGURE><P><FIGCAPTION>Table 4: Genetic parts required for the assembly of the L-arabinose controlled 'variable resistor'</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0080">BBa_R0080</A></TD><TD>AraC regulated promoter</TD><TD>This is the AraC regulated promoter. Transcription from this promoter takes place in the presence arabinose. Without arabinose present there should be no transcription as AraC will block transcription occurring.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A></TD><TD>Standard RBS</TD><TD>This is a standard RBS based that used in the construction of the repressilator (Elowitz, 1999). This is a high efficiency RBS and is widely used in iGEM projects. It is also present in the 2016 distribution kit.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K519010">BBa_K519010</A></TD><TD>SmtA Coding Sequence</TD><TD>This is the coding sequence for SmtA originally from <SPAN class="species-name">Synechococcus spp</SPAN> , a cyanobacterial strain.</TD></TR><TR><TD><A href="http://parts.igem.org/Protein_domains/Degradation">LVA-TAG</A></TD><TD>Degradation Tag</TD><TD>This is an ssRA protein degradation tag. Tagged proteins are degraded by the proteases ClpXP or ClpAP. There are a number of tag sequences, variants of AANDENYALAA, with the last three amino acids varying. The last three amino acids determine the half-life of the protein. LVA is a fast protein degradation tag. We use this to ensure that the resistance is reduced quickly after removal of arabinose.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A></TD><TD>Terminator</TD><TD>This is an artificial terminator part and was chosen because it has a high forward efficiency of 0.99.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A></TD><TD>Backbone</TD><TD>We are using the standard BioBrick backbone part pSB1C3 as this will make it easier to submit the part to the registry at a later date. We have also included restriction sites around protein and tag so that it can be replaced with a part without the tag to see if this has any effect on Zinc uptake.</TD></TR></TBODY></TABLE><DIV id="bio-ldr-red"><H3>OmpR Controlled Red 'Light Dependent Resistor' - Virtual Part <A href="http://parts.igem.org/Part:BBa_K1895996">BBa_K1895996</A></H3><P>We plan to engineer <EM>E. coli</EM> to behave like a light dependent
        resistor (LDR). A light dependent resistor (or photoresistor) varies its resistance with light intensity. Ordinarily, the resistance of an LDR decreases with increasing incident light and vice versa. We aim to do this by using <EM>E. coli</EM> to vary the amount of free
        ions in an electrolyte in response to light. Ion uptake will be controlled by the
        expression of <SPAN class="gene-name">smtA</SPAN>. As described above, SmtA is a metallothionein that can bind to heavy metal ions
        like Cadmium (II), Zinc (II) and Copper (II).</P><P>SmtA has been used in a number of iGEM projects and is in the registry
        (<A href="http://parts.igem.org/Part:BBa_K519010">BBa_K519010</A>). It has
        previously been used in experiments for Cadmium (II) uptake, see <A href="https://2011.igem.org/Team:Tokyo-NoKoGen/metallothionein">Tokyo-NokoGen 2011</A>
        and for <A href="http://parts.igem.org/Part:BBa_K190021">accumulating Zinc (II)
        intracellularly</A>. We will be examining firstly, the impact of SmtA on Zinc
        (II) concentration rather than Cadmium (II) and then the impact that this has on
        the resistivity of the Zinc (II) containing media. In this instance we will be
        using Zinc sulfate (ZnSO<SUB>4</SUB>) in solution where it disassociates into
        Zn<SUP>2+</SUP> and SO<SUB>4</SUB><SUP>2-</SUP> ions. Various concentrations of
        Zinc sulfate have <A href="http://sites.chem.colostate.edu/diverdi/all_courses/CRC%20reference%20data/electrical%20conductivity%20of%20aqueous%20solutions.pdf">
        known Electrical conductivity</A> . When <SPAN class="gene-name">smtA</SPAN> is expressed it will render the
        Zn<SUP>2+</SUP> unavailable and thereby reduce the conductivity of the
        solution.</P><P>For this LDR we will be using the red light detection system from the <A href="http://parts.igem.org/Coliroid">Colliroid project</A> ( <A href="http://www.nature.com/nature/journal/v438/n7067/full/nature04405.html#f1">Levskaya et al. 2005</A>).
        In this scheme the production of SmtA which affects the resistivity is placed
        under the control of the OmpF upstream promoter (BBa_R0084). We propose to
        engineer a system where this promoter is repressed in the dark and has increased
        transcription in (red) light. This allows the device to mimic the behaviour of a
        traditional electronic LDR whereby resistance is decreased in the light and
        increased in the dark.</P><P>The OmpF promoter is repressed by phosphorylated OmpR, OmpR-P. In normal
        conditions the <EM>E. coli</EM> cell contains free OmpR which can be phosphorylated by
        expression of a protein with an EnvZ domain. One such protein is the fusion
        protein, Cph8 (BBa_I15010). In the dark this protein phosphorylates OmpR and so prevents SmtA production, increasing the ‘resistance’ on the OmpF promoter. In the light, the light
        responsive domain Cph1 inhibits the activity of the EnvZ is prevented from
        phosphorylating OmpR and therefore allows SmtA production and decreased
        resistance.</P><P>Note that this will only work in <EM>E. coli</EM> which are naturally deficient in
        EnvZ.</P><P>In order for the light responsive domain of the fusion protein Cph8 to sense
        red light, the formation of a chromophore is required. This is done by the
        production of two proteins, HO1 and PcyA together with the Cph8. In our system
        these will be constitutively expressed to create the red light sensor.</P><H2>Our Construct</H2><P>There are two parts to our construct, the red light sensing component and the
        SmtA production component. It is presented below as a one plasmid system with the
        following parts.</P><P><FIGURE><FIGCAPTION>Figure 4a: SBOL Visual diagram of our designed red light sensitive 'LDR'.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 4b: Our red light sensitive 'LDR' with parts annotated with registry numbers.</FIGCAPTION></FIGURE></P><P><FIGCAPTION>Table 6: Genetic parts required for the assembly of the OmpR controlled Red 'Light Dependent Resistor' construct</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><EM>E. coli</EM> constitutive promoter</TD><TD>We will use a  constitutive promoter regulated by the <EM>E. coli</EM> housekeeping σ<SUP>70</SUP> sigma factor. Consequently, there should be active RNA polymerase present to transcribe from this promoter at all stages during the bacterial growth cycle, but in particular during exponential growth. We have chosen BBa_J23100, an artificial promoter due to its widespread use, documentation and comparatively short sequence (35bp).</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_I15008">BBa_I15008</A></TD><TD>HO1</TD><TD>This is one of the proteins required for chromophore formation.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_Il5009">BBa_I15009</A></TD><TD>PcyA</TD><TD>This is another of the proteins required for chromophore formation.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_I15010">BBa_I15010</A></TD><TD>Cph8</TD><TD>This is a fusion protein consisting of a light receptor domain and EnvZ domain. In the dark the EnvZ domain of this protein phosphorylates free OmpR in the cell which represses the OmpF promoter and induces the OmpC promoter.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0084">BBa_R0084</A></TD><TD>OmpR-P Promoter</TD><TD>This part is the promoter usually found upstream of OmpF. It is repressed by phosphorylated OmpR.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K519">BBa_K519010</A></TD><TD>SmtA</TD><TD>This is the coding sequence for SmtA originally from <EM>Synechococcus spp</EM>, a cyanobacterial strain.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A></TD><TD>Terminator</TD><TD>This is an artificial terminator part and was chosen because it has a high forward efficiency of 0.99.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A></TD><TD>Backbone</TD><TD>We are using the standard BioBrick backbone part pSB1C3 as this will make it easier to submit the part to the registry at a later date.</TD></TR></TBODY></TABLE><H3>LDR construction</H3><P>All parts listed above aside from Cph8 are provided in the 2016 iGEM distribution. Cph8 (BBa_I15010) is however available to order from the registry. The remaining parts
        are in the distribution so BioBrick assembly can be performed. However because of
        the large number of parts, and there are no intermediaries in the registry or
        distribution kit, this construct would be unwieldy to assemble in this
        manner.</P><DIV id="bio-ldr-blue"><H3>YF1-FixJ Controlled Blue 'Light Dependent Resistor' - Virtual part <A href="http://parts.igem.org/Part:BBa_K1895997">BBa_K1895997</A></H3><P>We can also produce a variation of our LDR to have a construct regulated by blue light.

        By placing smtA under the control of a FixJ-P (phosphorylated FixJ)
        promoter, this allows it to be regulated by blue light through a series of
        reactions with its response regulator protein YF1 (below).</P><FIGURE><FIGCAPTION>Figure 5: Diagram showing the interaction between YFP and its response regulator FixJ.</FIGCAPTION></FIGURE><P>In the absence of light, YF1 undergoes autophosphorylation to produce YF1-P
        which can then phosphorylate FixJ. This in turn activates the transcription of
        the downstream protein, which in this case, is SmtA. Thus, in the presence of light,
        SmtA is not produced and so conductivity does not change, whilst in the absence
        of light SmtA is produced resulting in a decrease in resistance.</P><P>Clearly, this behaviour is the inverse of an electrical LDR where resistance increases with light intensity. To mimic this behaviour
        using biological circuits we would place an inverter before the FixK2 promoter
        (which is activated by FixJ-P). The inverter is constructed by placing the
        desired output, here SmtA, under the control of a lambda cl regulated promoter
        (BBa_R0051). As lambda cl represses the promoter having this produced under
        control of FixK2 promoter inverts the system so that SmtA is produced in the
        presence of light rather than the absence thereof. <A href="http://parts.igem.org/Part:BBa_K592020">BBa_K592020</A> is an example of a part
        that uses this technique.</P><H4>Our Construct</H4><P>The non-inverted construct is shown in Figure 6a and 6b and the inverted construct in
        Figure 6c and 6d. Currently our device is shown as a single plasmid system but there is no
        reason that the two separate sub-components could not be split into a dual plasmid 
        system (with compatible origins of replication and differing resistance markers) for easier assembly.</P><FIGURE><FIGCAPTION>Figure 6a: SBOL Visual diagram showing the design of our uninverted blue light sensitive 'LDR'.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6b: SBOL Visual diagram showing the design of our uninverted blue light sensitive 'LDR' with parts registry numbers.</FIGCAPTION></FIGURE><P><FIGURE><FIGCAPTION>Figure 6c: SBOL Visual diagram showing the design of our inverted blue light sensitive 'LDR'.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 6d: SBOL Visual diagram showing the design of our inverted blue light sensitive 'LDR' with parts registry numbers.</FIGCAPTION></FIGURE></P><P><FIGCAPTION>Table 7a and 7b: Genetic parts required for the assembly of the YF1-FixJ controlled blue 'Light Dependent Resistor' construct </FIGCAPTION></P><P>For the non-inverted construct the parts are as follows:</P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A></TD><TD><EM> constitutive promoter</EM></TD><TD>We will use <A href="http://parts.igem.org/Promoters/Catalog/Constitutive">a σ<SUP>70</SUP> constitutive promoter</A> as this is the main <SPAN class="species-name">E. coli</SPAN> sigma factor. Consequently, there should be RNA polymerase present to transcribe from this promoter at all stages during the bacterial growth cycle. Specifically, we have chosen <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>, an artificial promoter due to its widespread use, documentation and comparatively short sequence (35bp).</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K5902016">BBa_K592016</A></TD><TD>FixJ &amp; YF1 with RBSs</TD><TD>The YF1 and FixJ coding sequences are provided as a composite part together with standard RBSs in part BBa_K592016 which we have chosen for ease of assembly, in the event that we or future teams wish to use the BioBrick standard assembly to produce our part.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</A></TD><TD>FixK2 promoter</TD><TD>This is the wild-type promoter to which phosphorylated FixJ binds. It is reported that this promoter has very little leaky activity in the absence of FixJ.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K519010">BBa_K519010</A></TD><TD>SmtA</TD><TD>This is the coding sequence for SmtA originally from <EM>Synechococcus spp</EM>, a cyanobacterial strain.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A></TD><TD>Terminator</TD><TD>This is an artificial terminator part and was chosen because it has a high forward efficiency of 0.99.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A></TD><TD>Backbone</TD><TD>We are using the standard BioBrick backbone part pSB1C3 as this will make it easier to submit the part to the registry at a later date.</TD></TR></TBODY></TABLE><P>For the inverted part there are additional parts as follows:</P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_C0051">BBa_C0051</A></TD><TD>Lambda CI</TD><TD>This is the repressor protein from Lambda phage it represses the promoter BBa_R0051.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>, <A href="http://parts.igem.org/Part:BBa_B0012&gt;BBa_B0012&lt;/a&gt;&lt;/td&gt;
		&lt;td&gt;Double stop terminators&lt;/td&gt;
		&lt;td&gt;These are the terminators used in the composite part BBa_S04617 which is replicated in our construct.&lt;/td&gt;
	&lt;/tr&gt;
	&lt;tr&gt;
		&lt;td&gt;&lt;a href=" http:="">BBa_R0051</A></TD><TD>Lambda CI controlled promoter</TD><TD>This is a promoter from Lambda phage that is repressed by lambda Cl (BBa_C0051).</TD></TR></TBODY></TABLE><H3>Construction and BioBrick assembly</H3><P>We can construct this device using synthesised gBlocks from our free allowance provided by IDT. </P><P>There exist a number of intermediate assembly components in the parts
        distribution that can be used to assemble our part faster if we use BioBrick
        assembly. Notably, <A href="http://parts.igem.org/Part:BBa_S04617">BBa_S04617</A>
        contains the inverter, <A href="http://parts.igem.org/Part:BBa_K592016">BBa_K592016</A> contains the FixJ and
        YF1. The two devices can be constructed separately as follows:</P><H4><EM><U>Constitutive Production Device</U></EM></H4><OL><LI><P>Digest the terminator BBa_B1006 with EcoRI &amp; XbaI and purify.</P></LI><LI><P>Digest BBa_K592016 with EcoRI &amp; SpeI and purify.</P></LI><LI><P>Mix both parts &amp; Ligate to form intermediate YF1FixJ+Terminator construct.</P></LI><LI><P>Digest the intermediate YF1FixJ+Terminator construct with EcoRI &amp; XbaI and the constitutive promoter
            BBa_J23100 with EcoRI &amp; SpeI.</P></LI><LI><P>Mix and ligate these to form the constitutive production device.</P></LI></OL><H4><EM><U>SmtA Expression Device (inverted):</U></EM></H4><OL><LI><P>Digest BBa_K519010 with EcoRI &amp; XbaI and purify.</P></LI><LI><P>Digest BBa_S04617 with EcoRI &amp; SpeI and purify.</P></LI><LI><P>Mix and ligate to produce intermediate part: inverted SmtA production.</P></LI><LI><P>Digest terminator with EcoRI &amp; XbaI.</P></LI><LI><P>Digest inverted SmtA production intermediate with EcoRI &amp; SpeI.</P></LI><LI><P>Mix and ligate purified parts together to produce SmtA expression device.</P></LI></OL><P>To produce the non-inverted device replace BBa_S04617 with the SmtA coding
        sequence and use an additional step to join this to an RBS, we suggest the
        standard RBS BBa_B0034 as this has good efficiency.</P><DIV id="bio-capacitor"><H3>Time Delay System (A Biological 'Capacitor') - Virtual Part <A href="http://parts.igem.org/Part:BBa_K1895995">BBa_K1895995</A></H3><P>We plan to engineer <EM>E. coli</EM> to mimic one of the properties of a
        capacitor; the ability to accumulate and hold charge for some time before
        discharging. This is shown in the idealised graph below.</P><P><FIGURE><FIGCAPTION>Figure 7: Idealised capacitor charge-discharge cycle.</FIGCAPTION></FIGURE></P><P>An electrical capacitor accumulates charge whilst a voltage is applied and
        then discharges when the voltage stops being applied. We make an analogy between
        the outputs of voltage signal and protein concentration. Whereas an electrical capacitor
        accumulates charge, a biological ‘capacitor’ would accumulate
        proteins. As an electrical capacitor has a maximum charge it can
        accumulate, within a bacterial cell there is a maximum protein concentration that can accumulate as determined by its production and degradation rate. Once proteins stop being
        accumulated in the cell it ‘discharges’ by having these drive the
        production of an output signal.</P><P>In this construct we use L-arabinose to mimic a voltage signal. This is entirely for experimental purposes, there is no reason that this device cannot be modified to respond to a stress signal, for instance through the P<EM><SUB>htpG</SUB></EM> heat shock response promoter explored elsewhere in our work. Although in this case we show that
        proteins can be accumulated there is no reason why actual charge, in the
        form of a potential difference could not be generated across the cell membrane.
        There are already examples of <A href="https://en.wikipedia.org/wiki/Membrane_potential">membrane potentials</A> in
        biology, the most obvious being found in neurons. This is something that has been
        explored by iGEM teams in the past e.g., <A href="https://2008.igem.org/IGEM:Cambridge/2008/Notebook/Voltage/K%2B_Concentrations">Cambridge
        (2008)</A>. Importantly we show through modelling that the charge-discharge cycle
        can be mimicked in biological cells through the use of repressor/inducer
        competition. This could be merged with work on membrane potentials in the
        future.</P><P>Because we use a constitutively on promoter, the TetR repressible promoter
        (<A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>) the default state
        of the system is ‘charging’. In this state lambda repressor (<A href="http://parts.igem.org/Part:BBa_C1051">BBa_C1051</A>) accumulates in the cell
        together with 434 repressor (<A href="http://parts.igem.org/Part:BBa_C0052">BBa_C0052</A>). The amount of 434
        repressor grows faster than that of lambda repressor because there are two coding
        sequences for the protein in the circuit. This is to ensure that it outcompetes
        (on average) the lambda repressor so that there is a low output signal whilst in
        the charging state. This occurs because 434 repressor represses the output
        promoter whilst lambda repressor induces it. In out device the output signal is
        sfGFP (<A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A>).</P><P>We can switch the state of the system to the discharging state by causing the
        expression of TetR. To facilitate this we have used an L-arabinose promoter
        coupled with the TetR coding sequence to give us a chemical ‘off
        switch’. Once TetR is produced the system enters the discharging state,
        as no further protein synthesis in our construct is induced, and so the amount of 434
        repressor and lambda repressor start to decay. The 434 repressor is tagged with a
        very fast <A href="http://parts.igem.org/Protein_domains/Degradation">ssRA
        degradation tag</A>, the LVA degradation tag so that it will be broken down
        faster than lambda repressor.</P><P>As this happens, there will reach a point where the 434 repressor stops
        out-competing the lambda repressor and the output will start to be produced as it
        is induced by the lambda repressor. The level of 434 and
        lambda repressor will continue to fall until the output stops being produced and
        the system has completely ‘discharged’ and is in a resting state. At
        this point the removal of L-arabinose and the addition of tetracycline or an
        analogue thereof (which binds to TetR and prevents it from repressing the
        promoter) would switch the system back into the charging state and the process
        can begin again.</P><H4>Construct</H4><P>Our construct, as shown here, is a composite device which is
        built entirely of BioBrick parts already in the registry. </P><FIGURE><FIGCAPTION>Figure 8a: SBOL Visual diagram showing the design of our biological 'capacitor'.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 8b: SBOL Visual diagram showing the design of our biological 'capacitor' with part numbers.</FIGCAPTION></FIGURE><P>The parts used are as
        follows:</P><P><FIGCAPTION>Table 8: Genetic parts required for the assembly of the 'Biological capacitor' construct</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0080">BBa_R0080</A></TD><TD>L-Arabinose inducible promoter</TD><TD>This part is an L-Arabinose inducible promoter with very low level
            expression in the absence of L-Arabinose. Be aware, if the <EM>E. coli</EM>
            strain used constitutively expresses AraC then this promoter will
            ‘leak’. Check the <A href="http://openwetware.org/wiki/E._coli_genotypes">strain list</A> for
            information.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A></TD><TD>TetR</TD><TD>This is the coding sequence for TetR which represses BBa_R0040.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A></TD><TD>TetR Repressible Promoter</TD><TD>This promoter can be repressed by TetR. Be
            aware, if the strain used expresses TetR constitutively then this promoter
            will be repressed. Check the <A href="http://openwetware.org/wiki/E._coli_genotypes">strain list</A> for
            information.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_C0052&gt;BBa_C0052&lt;/a&gt;&lt;/td&gt;

            &lt;td&gt;434 Repressor&lt;/td&gt;

            &lt;td&gt;Represses the output promoter.&lt;/td&gt;
          &lt;/tr&gt;

          &lt;tr&gt;
            &lt;td&gt;&lt;a href=" http:="">BBa_C0051</A></TD><TD>Lambda Repressor</TD><TD>Induces the output promoter.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_I12006&gt;BBa_I12006&lt;a&gt;&lt;/td&gt;

            &lt;td&gt;Modified Promoter Part&lt;/td&gt;

            &lt;td&gt;This is a modified promoter part, originally the lambda Prm promoter. The
            modification allows it to be activated by lambda repressor and repressed by
            434 repressor.&lt;/td&gt;
          &lt;/tr&gt;

          &lt;tr&gt;
            &lt;td&gt;&lt;a href=" http:="">BBa_I746916</A></TD><TD>sfGFP</TD><TD>This is the coding sequence for super folder GFP. We have chosen to use
            this as our reporter because it can easily be quantified using a plate reader or microscopy.</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_J23100&gt;BBa_B1006&lt;/a&gt;&lt;/td&gt;

            &lt;td&gt;Standard Terminator&lt;/td&gt;

            &lt;td&gt;We chose to use this promoter from the registry as it has a high forward
            efficiency.&lt;/td&gt;
          &lt;/tr&gt;

          &lt;tr&gt;
            &lt;td&gt;&lt;a href=" http:="">BBa_B0034</A></TD><TD>RBS</TD><TD>We chose to use this RBS from the registry as it is efficient and widely
            used in iGEM projects.</TD></TR></TBODY></TABLE><DIV id="mfc-porin-overexpression"><H3>Porin Overexpression for Microbial Fuel Cells - <A href="http://parts.igem.org/Part:BBa_K1895004"> BBa_K1895004/</A><A href="http://parts.igem.org/Part:BBa_K1895005">BBa_K1895005</A></H3><P>As part of our iGEM project we worked on trying to improve the current output from a microbial fuel cell. Microbial fuel cells which use <SPAN class="species-name">E. coli</SPAN> depend on a mediator, usually methylene blue to transfer electrons between the electrodes of the fuel cell and the organism's electron transport chain. The mediator enters the cell through porins in the cell membrane and 'steals' electrons, therefore becoming oxidised. These electrons are then carried to the positively charged cathode, where they are lost, allowing electrons to flow around the circuit.</P><P>By increasing the amount of mediator that can enter the cell we hope to be able to increase the electron flow rate, or current, from our microbial fuel cell. This occurs because the electrons can be moved more freely and their movement via the mediator is less limited by the rate at which the cell can transport the mediator across its membrane.</P><P>One of the ways of increasing the amount of mediator entering and exiting the cell is to increase the number of porins. For this purpose we have designed a device to control the overexpression of one of the porin <A href="http://www.rcsb.org/pdb/explore.do?structureId=2OMF">OmpF</A>.</P><P>We chose OmpF because it occurs naturally in <SPAN class="species-name">E. coli</SPAN> and is suitable for the mediator we use in our fuel cell (which is a variant of the <A href="http://www.ncbe.reading.ac.uk/ncbe/materials/METABOLISM/fuelcell.html">University of Reading's NCBE kit</A>). OmpF is permeable for molecules smaller than 600 Da. Since <A href="http://biocyc.org/compound?orgid=META&amp;id=CPD0-1625">methylene blue</A> is 284 Da, it will pass easily through the OmpF pore.</P><P>By adding L-arabinose to the growth medium, the number of OmpF porins in the <SPAN class="species-name">E. coli</SPAN> membrane should increase, allowing increased extracellular transport of methylene blue and faster electron shuttling between the cells and the electrodes. This should correspond to an increase in current. This function is shown below.</P><FIGURE><FIGCAPTION>Figure 9: Diagram showing the role of additional OmpF porins in increased mediator transport in a microbial fuel cell.</FIGCAPTION></FIGURE><P>Our OmpF overexpression device is simple in construction, consisting of a P<SUB>Bad</SUB> promoter, standard Elowitz RBS, the OmpF coding sequence and a standard terminator. We have added this as part number <A href="http://parts.igem.org/Part:BBa_K1895004">BBa_K1895004</A> in the registry.</P><FIGURE><FIGCAPTION>Figure 10a: SBOL Visual Diagram of our OmpF expression construct.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 10b: SBOL Visual Diagram of our OmpF expression construct with parts registry numbers.</FIGCAPTION></FIGURE><P><FIGCAPTION>Table 9: Genetic parts required for the assembly of the OmpF 'Microbial Fuel Cell' construct</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K206000">BBa_K206000</A></TD><TD>P<SUB>Bad</SUB> Strong</TD><TD>Promotes the downstream expression of OmpF when L-arabinose is present in the growth medium.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034">BBa_B0034</A></TD><TD>RBS (Elowitz)</TD><TD>A standard, efficient RBS, as used in our other constructs.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K864204">BBa_K864204</A></TD><TD>OmpF</TD><TD>This is the coding sequence for OmpF, a natural porin protein found in <EM>E. coli</EM>. and involved in the transport of small molecules across the cell membrane.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0012">BBa_B0012</A></TD><TD>Terminator</TD><TD>This is a standard, widely used terminator in iGEM projects. We switched to using this terminator when we had synthesis problems with the double loop terminator used in our other constructs (<A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A>).</TD></TR></TBODY></TABLE><P>We are also investigating an alternative way of increasing the amount of mediator transport through our work which is the expression of a larger membrane porin. One of the limiting factors of a microbial fuel cell is the low permeability of the cell membrane, which limits mediator transport. Team Bielefeld in 2013 expressed the OprF porin protein from <EM>Pseudomonas fluorescens</EM> in <SPAN class="species-name">E. coli</SPAN> which increased cell membrane permeability (<A href="http://parts.igem.org/Part:BBa_K1172503">BBa_K1172503</A>). This is because the porin size is much larger than <SPAN class="species-name">E. coli</SPAN> natural porins (such as OmpF, above). This allows more mediator to pass through the membrane.</P><P><FIGURE><FIGCAPTION>Figure 11: Diagram showing the role of OprF porin in increased mediator transport in a microbial fuel cell.</FIGCAPTION></FIGURE></P><P>
One of the problems the Bielefeld team reported was the reduced growth of the <SPAN class="species-name">E. coli</SPAN> due to metabolic stress. This is particularly noticeable when using the T7 promoter, as in their part BBa_K1172502. Population size is important for fuel cell output. The more cells there are, the more electron donors there are and so theoretically the greater the current. To overcome this issue we have changed their promoter to a P<SUB>Bad</SUB> promoter which is regulated by L-arabinose monosaccharide. Thus replication stress is reduced during the first part of the growth curve. This should allow the bacterial population to grow rapidly in the fuel cell, ensuring we get a large population before L-arabinose is added (this increased level of growth should be seen by a higher OD600 value closer to 4.0 (growth of wild type) after 10 hours of growth). At this point the current should then further increase due to the larger porin expression. 
</P><P>This part is similar in construction to our OmpF expression part, with the protein coding sequence swapped to OprF rather than OmpR. This is part number <A href="http://parts.igem.org/Part:BBa_K1895005">BBa_K1895005</A> in the registry.</P><P><FIGURE><FIGCAPTION>Figure 12a: SBOL Visual Diagram showing our OprF expression construct.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 12b: SBOL Visual Diagram showing our OprF expression construct with parts registry numbers.</FIGCAPTION></FIGURE></P><P><FIGCAPTION>Table 10: Genetic parts required for the assembly of the OprF containing 'Microbial Fuel Cell' construct</FIGCAPTION></P><TABLE><TABLE class="wikitable"><TBODY><TR><TH>Part No.</TH><TH>Name</TH><TH>Purpose</TH></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K206000">BBa_K206000</A></TD><TD>P<SUB>Bad</SUB> Strong</TD><TD>Promotes the downstream expression of OmpF when L-arabinose is present in the growth medium.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034">BBa_B0034</A></TD><TD>RBS (Elowitz)</TD><TD>A standard, efficient RBS, as used in our other constructs.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1172501">BBa_K1172501</A></TD><TD>OprF</TD><TD>This is the coding sequence for OprF, a large porin protein found in <EM>Pseudomonas fluorescens</EM>.</TD></TR><TR><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0012">BBa_B0012</A></TD><TD>Terminator</TD><TD>This is a standard, widely used terminator in iGEM projects. We switched to using this terminator when we had synthesis problems with the double loop terminator used in our other constructs (<A href="http://parts.igem.org/Part:BBa_B1006">BBa_B1006</A>).</TD></TR></TBODY></TABLE><DIV class="col-md-4" data-spy="affix-right"><UL class="nav nav-pills nav-stacked" id="right-menu"><LI><H3><A href="#bio-bulb">Constructed part: Light Bulb</A></H3></LI><LI><H3><A href="#bio-varistor">Constructed part: Arabinose Varistor</A></H3></LI><LI><H3><A href="#bio-ldr-red">Virtual part: LDR</A></H3></LI><LI><H3><A href="#bio-capacitor">Virtual part: Biological Capacitor</A></H3></LI><LI><H3><A href="#mfc-porin-overexpression">Constructed part: Microbial Fuel Cell with Porin Overexpression</A></H3></LI></UL></DIV></TABLE></TABLE></DIV></TABLE></DIV></TABLE></TABLE></DIV></TABLE></DIV></TABLE></DIV></TABLE></DIV></TABLE></TABLE></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>