"
Page
Discussion
View source
History
teams
Log in
 
Team:HUST-China/Protocol
From 2014.igem.org
oo
Overview
Project
Background
Design
Toolkit
Experiment
Protocol
Result
Modeling
Human Pratice
Team
Protocol
Part 1: The Construction of the Worker System
Step 1: Gene Cloning
To find the optimal temperature for oprF and cynRTS composite amplification, we set a gradient of annealing temperature. The result shows that 58℃ is suitable to this PCR. The PCR product was stored in -20℃. The PCR reaction system was listed in table 1-1.
Table 1-1 Gradient PCR System
Components(50μl)
Volume(ml)
PrimerStar Buffer
10
dNTPs(2.5mM)
5
Primer-F(10μM)
1.5
Primer-R(10μM)
1.5
Template
1.5
PrimerStar
0.5
ddH2O
30
Step 2: Optimization of oprF
We choose the 188aa or 196aa of OprF as the CBP fusion points. Primers were designed (Table1-2) to clone the target fragment, and then the GS linker and/or the gene fragment coding CBP were added at the C-terminal of the target fragment.
The protein with GS linker, using the 188aa as CBP fusion point, was named as OprF-1. While the protein without GS linker, using the 196aa as CBP fusion point, was named as OprF-2.
Table 1-2  Primers of Optimization oprF
Primer
Sequence(5'→3')
oprF-CBP-F
CCGGAATTCAACTGAAGAACACCTT
oprF-CBP-R1
CCAGCCGCCATGATGCGGGGAAACCGGTTCCGGAGCCGGAGCGGC
oprF-CBP-R2
ATAGTTTAGCGGCCGCCGGCCAGCCGCCATGATGCGGGGA
oprF-GS-F
CCGGAATTCAACTGAAGAACACCTT
oprF-GS-R1
TGAACCTCCGCCACCTTTCGAACCACCGAAGTTGAAG
oprF-GS-R2
CCAGCCGCCATGATGCGGGGATGAACCTCCGCCACC
oprF-GS-R3
ATAGTTTAGCGGCCGCCGGCCAGCCGCCATGATGCGGGGA
Step 3: Construction of Vectors
pET28a vector and the PCR product were digested with EcoRI and NotI from Takara, and then retrieved and purified with DNA retrieve kits from Omega.
Then, pET28a(+) vector, gene oprF-1/oprF-2/cynRTS composite were linked together with T4 ligase to form a new vector: pET28a(+)-oprF-1/oprF-2/RTS. The reaction system for digestion and conjunction were listed in the table 1-3, 1-4 and the digestion results were displayed in fig1-1.
Table 1-3  Reaction System for Digestion
Components(50μl)
Volume(μl)
10×H Buffer
5
BSA
5
EcoRI
2.5
NotI
2.5
Fragment
25
ddH2O
10
Conditions
37℃ 1h
Table 1-4  for Gene Conjunction
Components
System components(10μl)
Buffer(with T4 ligase)
5
Insert
4.5
Vector
0.5
Conditions
16℃ 1h
Fig 1-1 Digested gene and plasmid
Step 4: Transforming the Plasmids into E. coli BL21 Strain
Step 4: Transforming the Plasmids into E. coli BL21 Strain 
Plasmids pET28a(+)-oprF-1 or pET28a(+)-oprF-2 or pET28a(+)-flA was transformed into E. coli BL21(DE3) and protein expression was analyzed.The strains were grown in LB medium containing 100ug/ml kanamycin at 37℃, 250rpm until OD600 reached 0.4–0.6. We then added IPTG to 0.5mM and continued the incubation at 28℃ overnight to induce the overexpression of OprF-CBP or OprF-GS-CBP or flA. The cells were collected, re-suspended with 10mM imidazole containing 0.1mM protease inhibitor PMSF and then disrupted using Selecta Sonopuls. After centrifugation, the sediment was treated with 1xSDS gel loading buffer and kept in boiling water for 5 minutes and applied to SDS-PAGE. The result was displayed in Fig 1-4.
Table 1-5  Reaction System of Colonial PCR validation
Components(10μl)
Volume(ml)
2×Ex Taq Mix 
5
Primer-F(10μmol/L)
0.3
Primer-R(10μmol/L) 
0.3
Template
0.3
H2O
4.1
Table 1-6  Reaction System of Double Digestion (37 ℃ 1h)
Components(10μl)
Volume(μl)
10×H Buffer 
1
BSA
1
EcoRI 
0.5
NotI
0.5
ddH2O
7
Fig 1-2 Gel images of novel plasmids with OprF-CBP/GS-CBP testified results
Fig 1-3 Gel images of novel plasmids with cynRTS composite testified results
The E. coli BL21(DE3) strain transformed with pET28a(+)-flA was kindly donated by Prof. David O'Hagan and Prof. James H. Naismith from University of St Andrews, Saint Andrews, Scotland, United Kingdom.
Step 5: Expression of Protein
Plasmid pET28a(+)-oprF-1/oprF-2/cynRTS composite/flA was transformed into E. coli BL21(DE3) and we get for protein expression analysis. The strains were grown in Luria broth containing 100ug/ml kanamycin at 37℃, 250rpm until an absorbance of 0.4–0.6 at 600 nm was reached. We then added IPTG to 0.5mM and continued the incubation at 28℃ overnight to induce the overexpression of OprF-CBP/OprF-GS-CBP. The cells were collected, suspended with 10mM imidazole containing 0.1mM protease inhibitor PMSF and then disrupted using Selecta Sonopuls. After centrifugation, the sediment was treated with 1*SDS gel loading buffer and kept in boiling water for 5 minutes and applied to SDS-PAGE.
Step 6: Surface Displaying Copper Ions
To identify that whether our OprF has anchored on the cell membrane of E. coli, we performed immunofluorescence assay. HA tag was added to the N-terminal of OprF-CBP so that the recombinant protein OprF-CBP-HA can be specifically recognized by anti-HA antibody. When FITC labeled anti-IgG antibody was used as the secondary antibody and interacted with the primary antibody, green fluorescence could be observed in the cell membrane of E. coli under the fluorescent microscope.
Part 2: The Construction of the Instructor System
All the parts except PpcoA we used to construct E. instructor are from kits in Distribution 2013.They are listed in table2-1.
Table 2-1 Parts from Kits in Distribution 2013
Name
Parts
Well
Short Description
CII
BBa_P0153
9A(plate3,2013)
Protein
CI
BBa_C0051
3A(plate3,2013)
Protein
RFP
BBa_E1010
18F(plate5,2013)
engineered mutant of red fluorescent protein
GFP
BBa_E0044
14G(plate5,2013)
mGFP mut3b+AAV
PR
BBa_R0051
6K(plate5,2013)
CI regulated promoter
PRE
BBa_R0053
6M(plate5,2013)
CII regulated promoter
Assemble all the elements via standard restrict sites:
1. PCR: PpcoA (add EcoRI, XbaI, SpeI and PstI restriction sites)
2. Double digestion: PpcoA PCR products and rbs+CII(LVA)+ter+ter/pSB1C3(BBa_P0153):
3. Colony PCR Validation (primer PpcoA F and PpcoA R):
The plasmid pPcoA-rbs+CII+LVA/pSB1C3 was extracted and sequenced.
4. Double digestion: PpcoA-rbs+CII+LVA/pSB1C3 and rbs+CI+tag-dT:
5. Colony PCR Validation(primer PpcoA F and cI R):
The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT/pSB1C3 was extracted and sequenced.
6. Double digestion: PCR product of rbs+mRFP+tag and dT/pSB1C3:
7. Colony PCR Validation (primer mRFP F1 and mRFP R1):
The plasmid rbs+mRFP+tag-dT/pSB1C3 were extracted and sequenced.
8. Double digestion: PRE/pMD-19T and rbs+mRFP+tag-dT/pSB1C3:
9. Colony PCR Validation (primer PRE F and mRFP R1):
The plasmid PRE-rbs+mRFP+tag-dT/pMD19-T was extracted and sequenced.
step 2: Transformation of bacteria and colony PCR(primer VF2 and VR)
1. Promoter lambda (cI regulated) with GFP (+LVA) reporter in pSB1A3:
The plasmid PR-rbs+GFP+LVA/pSB1C3 was extracted and sequenced.
2. Double digestion:（3A assembly）
PRE- rbs+mRFP+tag-dT/pMD19-T
3. PpcoA-rbs+CII+LVA-rbs+CI+tag-dT/pSB1C3
4. Colony PCR validation (primer cI F and mRFP R1):
The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT/pSB1C3 was extracted and sequenced using primer VF2 and VR.
5. Double digestion PR-rbs+GFP+LVA/pSB1C3
6. PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT/pSB1C3
7. Colony PCR Validation (primer PRE F and GFP R):
The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT-PR-rbs+GFP+LVA/pSB1C3 was extracted and sequenced using primer VF2 and VR.
Part 3: The Construction of the Kill Switch
Plasmid construction:
PCR: CI repressor from E. coli phage lambda (+LVA) in pSB1C3（BBa_C0051）was amplified to add RBS using primer:
cI F: 5'–TATGAATTCTCTAGATAAGGAGATATAATGAGCACAAAAAAG-3' and cI R: 5'–TAATCTGCAGACTAGTGCGATCTACACTAGCACTATC-3'
Double digestion:
PCR product of cI+rbs obtained using gel extraction kit and double terminator (dT) in pSB1C3 were applied to double digestion. The reaction systems were listed in table 3-1, 3-2.
Table3-1 Digestion of Fragment (cI+rbs):
Reaction system
50 μl
DNA solution
41 μl
EcoRI
2μl
SpeI
2 μl
10×H
5 μl
Table 3-2 Digestion of pSB1C3-dT:
Reaction system
50 μl
plasmid
41 μl
EcoRI
2μl
SpeI
2 μl
10×M
5 μl
The mixture was incubated in 37℃ for 2h , then gel electrophoresis and gel extraction were performed:
Then, we linked them together at room temperature for 1h. The reaction system listed in table 3-3.
Table 3-3 Reaction System of  Ligation
Ligation system
10μl
cI+rbs
4μl
Vector
1μl
T4 Ligase(contain 10x T4 Ligase Buffer)
5μl
Colony PCR validation:
Select monoclonal and applied to PCR:
The positive colone was cultured in LB overnight for plasmid extraction and sequencing.
Assemble all elements one by one using the same method:
Double digestion:
PL lacI (BBa_R0011) in pSB1A3 and pSB1C3-cI+rbs-dT
Colony PCR validation(primer VF2 and cI R):
The plasmid PL lacI-rbs+cI-dT/pSB1C3 was extracted and sequenced.
Double digestion:
PL/pMD-19T and rbs+mRFP+tag PCR products:
Colony PCR validation(primer VF2 and mRFP R1):
Colony PCR validation(primer VF2 and mRFP R1):
The plasmid PL -rbs+mRFP+tag/pSB1C3 was extracted and sequenced.
Double digestion:
PL -rbs+mRFP+tag/pMD19-T and PL lacI-rbs+cI-dT/pSB1C3:
Colony PCR validation (primer VF2 and cI R):
The plasmid PL lacI-rbs+cI-dT- PL -rbs+mRFP+tag/pMD-19T was extracted and sequenced.
Part 4: Measuring of Growth Curve
Step 1:
200μL overnight-cultured bacterial was added to vials containing newly prepared culture medium (containing 100μg/ml Kanamycin). Set one vial with culture medium as blank. All the samples were cultured in the shaking incubator with rotational speed at 200 rpm and temperature at 37℃.
Step 2:
OD600 values was measured using the nucleic acid analyzer. When the OD600 values of culture reached 0.6, 200μL culture was added to a new vial containing 5mL LB, 100μg/ml Kanamycin, 0.1M IPTG and CN-/F-/Cu2+ with different concentration. Set one vial without bacteria as blank. All these samples were cultured in the shaking incubator with rotational speed at 200 rpm and temperature at 37℃.
Step 3:
OD600 of samples from the vials were measured every 30 min.
Part 5: The Standardization of Parts
Materials (used in this part)
The materials we used in this part are listed in table 5-1 and table 5-2
Table 5-1: Bacterial strains and plasmids
Strains and vectors
Relevant genotype and characteristics
Originate
E. coli DH5α
Strains
conserved in the lab
pMD18T
Vectors
TaKaRa Biotechnology (DaLian)Co.,Ltd.
pSB1C3
Vectors
iGEM package
Table 5-2: Primers' Name and Sequence
Primer
Sequence(5’→3’)
oprF-F
G GAATTC GCGGCCGC T TCTAGAG ATGAAACTGAAGAACACC
oprF-R
TTT CTGCAG CGGCCGC T ACTAGT ACCAGCCGCCATGAT
ompC-F
G GAATTC GCGGCCGC T TCTAGA G ATGCGTCTTGGCTT
ompC-R
TGCA CTGCAG CGGCCGC T ACTAGT ATTAGAACTGGTAAACC
RTS-F
G GAATTC GCGGCCGC T TCTAGA G TCAGAACGGTTTGG
RTS-R
TGCA CTGCAG CGGCCGC T ACTAGT ACTACCGTGATTCATTTC
flA-F
G GAATTC GCGGCCGC T TCTAGA G ATGGCTGCCAACAGCAC
flA-R
TGCA CTGCAG CGGCCGC T ACTAGT ATCAGCGGGCCTCGACCC
ompC-m-A
CCGCTTCTAGAGATGCGTCTTGGCTT
ompC-m-B
GGTGTCACCACCGAACTCTGGCAGT
ompC-m-C
ACTGCCAGAGTTCGGTGGTGACACC
ompC-m-D
GTTACGCCACTAGTAAAGCCTTCAC
ompC-m-E
ATTGCAGGTAAGCCAGGGACGGACG
ompC-m-F
GTGAAGGCTTTGCTAGTGGCGTAAC
ompC-m-G
CGTCCGTCCCTGGCTTACCTGCAAT
ompC-m-H
CCGCTACTAGTATTAGAACTGGTAAACC
Step 1: Gene Standardized
P4R amplification with the primers set as the template (pET28a-OprF-CBP/OprF-GS-CBP/cynRTS composite/FLA). The conditions of the reaction were listed in table 5-3.
Table 5-3: The Reaction System to harvest target genes
Components(50μl)
Volume(μl)
Template
1.25
dNTPs
2.5
10×LA PCR Buffer
5
LA Tag
0.5
Primer-F(10μmol/L)
2.5
Primer-R(10μmol/L)
2.5
ddH2O
35.75
Step 2: Obliteration of the Illegal Restriction Sites in the Gene
It is necessary to add standard restriction enzyme sites-EcoRI/XbaI/SpeI/PstI to both terminals of each gene. However, the sequence of ompC contains EcoRI, SpeI and PstI restriction site. So we obliterated the restriction sites by site-directed mutagenesis based on overlap extension PCR. Briefly, target mutation (GAATTC→GAGTTC, ACTAGT→GCTAGT , CTGCAG→CTCCAA, ) was introduced into primers (ompC-m-A/B/C/D/E/F/G/H in table 5-2), and the four previous PCR products(ompC-AB, ompC-CD, ompC-EF, ompC-GH) were used as template for the second PCR, and the two previous PCR products(ompC-AD, ompC-EH) were used as template for the third PCR. The final PCR segment with target mutation sites was then cloned into pMD18-T vector for sequencing. The three PCR systems were listed in table 5-4, 5-5, 5-6.
Table 5-4: Reaction System of the 1st PCR
Components(50μl)
Volume(μl)
2×Ex Taq Mix
25
ompC-m-A/C/E/G(10μM)
1.5
ompC-m-B/D/F/H(10μM)
1.5
Template
1.5
H2O
20.5
Table 5-5: Reaction System of the 2nd PCR
Components(50μl)
Volume(μl)
2×Ex Taq Mix
25
ompC-A/E(10μM)
2.5
ompC-D/H(10μM)
2.5
ompC-AB/EF
5
ompC-CD/GH
5
H2O
10
Table 5-6: Reaction System of 3rd PCR
Components(50μl)
Volume(μl)
2×Ex Taq Mix
25
ompC-F(10μM)
2.5
ompC-R(10μM)
2.5
ompC-AD
5
ompC-EH
5
H2O
10
Step 3: Insertion of the Standard Parts into plasmid backbone-pSB1C3
pSB1C3 backbone and the standardized parts was digested with EcoRI and PstI and then retrieved and purified with kits from Omega. pSB1C3 and the standardized parts were then linked together to from new Biobricks.
Step 4 : Submission of New Standardized Parts
After completing all the validation, the four parts were submitted to iGEM official organization in the early October and arrived in New York in 8th Oct. The information was listed in the table 5-7.
Standard biobricks
description
pSB1C3-oprF-1(EcoRI PstI）
A major outer membrane protein of Pseudomonas aeruginosa functions as a nonspecific porin to allow the passage of small hydrophilic molecules. We cut the protein at 188aa and add a CBP tag to C-terminal with a GS linker between them.
pSB1C3-oprF-2(EcoRI PstI）
A major outer membrane protein of Pseudomonas aeruginosa functions as a nonspecific porin to allow the passage of small hydrophilic molecules. We cut the protein at 196aa and add a CBP tag to C-terminal.
pSB1C3-ompC(EcoRI PstI）
Outer membrane porin protein C from Escherichia coli BL21(DE3)
pSB1C3-RTS(EcoRI PstI）
Cyn operon in Escherichia coli BL21(DE3) (cynR+cynT+cynS) is about cyanate detoxification.
pSB1C3-FLA(EcoRI PstI）
Fluorinase enzyme from Streptomyces cattleya catalyzing the formation of a C–F bond by combining S-adenosyl-L-methionine (SAM) and F- to generate 5’-fluoro-5’-deoxyadenosine (5’-FDA) and L-methionine.
Worker System
Instructor System
Kill Switch
Growth Curve
Standardization
E-mail: byl.hust.china@gmail.com
HUST, China
Retrieved from " http://2014.igem.org/Team:HUST-China/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
