"
Page
Discussion
View source
History
teams
Log in
 
Team:HUST-China/Worker
From 2014.igem.org
oo
Overview
Project
Background
Design
Toolkit
Future Work
Experiment
Protocol
Results
Modeling
Human Pratice
Team
Worker
1. Function Description
1.1  E.Worker
E.Worker is an engineering bacteria to chelate copper ions, degrade the cyanide and detoxify the fluoride in the sewage at the same time. The E.Worker is consisted of two expression plasmids, and they are co-transformed into our E.coli BL21(DE) strain.
Figure 1: The genetic circuit of E. worker
1.2  E.Instructor
E.Instructor is an engineering bacteria to detect the concentration of Cu2+ in the sewage. The instructor is consisted of two expression plasmids, and they are co-transformed into our E.coli BL21(DE) strain.
Figure 2: The genetic circuit of E.Instructor
2. Genetic  Circuit  Design
In our main bacteria, E.Worker, one of the vector (pET-28a) carries ompC/oprF-His/CBP(coding a improved protein that can display on the cell's surface and chelate Cu2+), flA(coding an enzyme that catalyzes the combination of SAM and F- into 5'-FDA and L-methionine ) and RTS ( a operon control can degrade cyanide). OmpC/oprF-His/CBP and flA are under the regulation of promoter PpcoA, which is activated by copper Cu2+.
The other vector ACYCDuet-1, carries CI under the regulation of promoter PL lac and toxin under the regulation of promoter PR. Promoter PL is inhibited by CI. CI can be degraded by RecA protease which can be activated by ultraviolet; and then the toxin can be expressed , in order to kill the E.Worker.
When the concentration of Cu2+ in the sewage is above the standard limit, E.Instructor will express mRFP, indicating that the sewage needs to be purified. At the same time, in E.worker, promoter pcoA will activate the expression those gene, the E.Worker will start to disorp Cu2+, fluoride and cyanide. When the sewage has been purified, the expression of CII in E.Instructor will reduced while the expression of mRFP was inhibited and the expression of mGFP was activated.
3. Killing Switch
To prevent our E.Kungfu become a new threat to the environment, we planned to add a killing switch in the gene circuit, controlling the livelihood and death of the E.coli. Whenever we need, E.Workerwill express toxin protein and kill itself under the exposure of UV.
CI protein is an inhibitor that can bind with OL, inhibit the contact of RNA polymerase with promoter. flA is a key enzyme of fluoride degradation which can detoxify fluoride significantly. RTS is used for cyanide degradation. pACYCDuet-1 carries CI under the regulation of promoter PL lac and toxin under the regulation of promoter PR. In our E.Worker, PpcoA can be activated by copper ion, and PL can be inhibited by CI. CI can be degraded by RecA protease which can be activated by UV; and then the toxin can be expressed to kill the E. Worker.
OL has three CI proteins binding sites, named OL1, OL2 and OL3. Each of them is 17bp. They facilitate the CI protein binding process through synergistic effect.
It’s because the strong inhibition effect of CI proteins to PL promoter that the killing switch is on the off state. Since there is no expression of toxin, the bacteria can stay alive. But how to remove this inhibitive effect to turn on this switch?
With moderate degree UV existed, DNA was damaged. Since the replication progress is inhibited, there are lots of conglomerate ssDNA and SOS repairing process occurs. SsDNA can recruit RecA-forming nucleoprotein filaments and activate RecA.（using RecA* to represent the activated state）Then, with the recognition of CI proteins by RecA*, CI proteins can be degraded rapidly. Thus, the inhibitive effect is removed, and numerous toxin proteins can express to kill the bacteria.
A brief introduction to surface display
The strategy of cell surface display in bacterial has many advantages over traditional method of producing and using enzyme. First of all, cell surface display can cut down the cost of enzyme purification and restoration. Secondly, the method can eliminate the mass transfer barrier in transportation of substrates across the cell membrane.
Bacterial surface display systems use anchoring motifs to functional display proteins on the cell surface. Different protein scaffolds such as flagella, porins and virulence factors have been employed to display target proteins (enzyme). The system has been developed for various applications, such as protein engineering, biological synthesis, biosensing and biofuel cells.
The introduction of oprF-CBP system
Cell surface display technology has made possible a wide range of applications in the biotechnological and industrial fields, such as the recovery of harmful chemicals and heavy metals.Outer membrane proteins including OmpA,OprF, OmpS, FadL, LamB, PhoE, OmpC, and Lpp–OmpA have been successfully used as anchoring motifs for displaying various peptides and proteins.
The OprF is a major outer membrane protein of  Psns as a nonspecific porin to allow the passage of small hydrophilic molecules, plays a structural role in maintaining cell shape and outer membrane integrity, and is required for growth under low osmolality.The structure ofeudomonas aeruginosa. This protein functio OprF has been proposed to consist of three domains, the N-terminal forming h-barrel structure, a loop or hinge region, and the C-terminal associated with peptidoglycan. Here is the proposed secondary structure of OprF.
Based on the predicted secondary structure and information found in the literature, we chose Val188, Ala196 as potential fusion sites for displaying CBP.CBP is the abbreviation of copper binding peptide made up of seven amino acid.In order to reduce the effect on the CBP activity，we added (G4S) linker between OprF and CBP.
The introduction of flA
Fluorinase from Streptomyces cattleya catalyzing the formation of a C-F bond by combining S-adenosyl-L-methionine (SAM) and F- to generate5’-fluoro-5’-deoxyadenosine (5’-FDA) and L-methionine.
The enzyme’s molecular mass is 34402 and it has a catalytic rate constant (kcat) of 0.07 min21. The Michaelis constant (Km) for F— is 2 mM, the Km for SAM  is 74mM.
4. Judging Criteria
We deserve a Gold Medal Prize in view of the following reasons:
1. We completed safety form, judging form and team wiki before the deadline. It is certain that we are going to present a poster and give a presentation at the iGEM Jamboree.
2. We documented four newly standard BioBrick Part (oprF-GS-CBP/oprF-CBP/ompC/RTS) used in our project and submitted them to the iGEM Registry adhering to guidelines.
3. We improved a function of an existing BioBrick Part(BBa_K1172501). We changed the sequence and the protein expressed could surface display Cu2+.
4. Our work aims at dealing with sewage containing copper ions produced by some industries, which is a new application of environmental microbiology.
5. We did plenty of experiment to validate that two of BioBrick Part of our own design and construction works as expected.
6. We share information and material with WHU and HZAU .We help the HZAU-CHINA construct a part of the oscillator and they help us to sequence and test the function of PpcoA.
We deserve a Best Model Prize in view of the following reasons:
1. We simulated the biological process of our E. Worker to test the feasibility of the idea.
2. We analyzed robustness and sensitivity of this biological system to well understand its function.
3. We provided some suggestions to the web lab to improve the circuits.
4. We considered some environmental factors to make our simulations closer to the reality.
5. Future Work
Although we have spent almost a whole year on this project and fulfilled many achievements, we still have a long way to go in the future:
1. Culturing the transgenic bacterial in culture medium containing Cu2+, F- and CN- and measuring the change of the concentration of those ions
2. Examining instructor system
3. Co-transforming the worker system and the instructor system into one bacteria and testing the composite system
4. Recycling the copper via firing the bacteria
5. Testing the pollutants treatment capacity of the kit covered with the engineering bacterial film
6. Writing a handbook to help factories to find the most suitable rotating speed based on the RBC they established
7. Presenting our idea and data to some factories and trying to transform our project into real products
1.Function Description
2. Genetic  Circuit  Design
3. Killing Switch
E-mail: byl.hust.china@gmail.com
HUST, China
Retrieved from " http://2014.igem.org/Team:HUST-China/Worker "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
