<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HUST-China_Design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HUST-China/Design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HUST-China/Design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-China/Design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-China/Design&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HUST-China/Design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HUST-China/Design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="animat"><SPAN><P>oo</P></SPAN></DIV><DIV align="center" class="content_body"><DIV class="navigater" align="center"><UL><LI class="column" id="OVERVIEW"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Overview">Overview</A></SPAN></LI><LI class="column" id="PROJECT"><SPAN>Project</SPAN></LI><LI><A href="https://2014.igem.org/Team:HUST-China/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Toolkit">Toolkit</A></LI></UL><LI class="column" id="EXPERIMENT"><SPAN>Experiment</SPAN></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Protocol">Protocol</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Result">Result</A></LI><LI class="column" id="MODELING"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Modeling">Modeling</A></SPAN></LI><LI class="column" id="HUMANPRACTICE"><SPAN><A href="https://2014.igem.org/Team:HUST-China/HumanPractice">Human Pratice</A></SPAN></LI><LI class="column" id="TEAM"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Team">Team</A></SPAN></LI></DIV><DIV id="cont_column"><DIV class="chapter"><SPAN><FONT size="6px">Design</FONT></SPAN><H1 align="left" id="h2_0"> 1. Engineering Bacteria</H1><H3 align="left">1.1  <EM>E. worker</EM></H3><P><EM>E. worker</EM> is the bacteria chelating copper ions, degrading the cyanide and detoxifying the fluoride in the sewage at the same time. It is consisted of two expression plasmids-the worker system and the kill switch, and they are co-transformed into the <EM>E. coli</EM> BL21 (DE) strain.</P><H3 align="left">1.2  <EM>E. instructor</EM></H3><P><EM>E. instructor</EM> is the bacteria to detect the concentration of Cu<SUP>2+</SUP> in the sewage. It is consisted of two expression plasmids-the worker system to survive it in sewage and the instructor system, and they are co-transformed into our <EM>E. coli</EM> BL21(DE) strain. </P><H1 align="left" id="h2_1">2.Worker System </H1><H4 align="left">Surface Display System</H4><P><B>Surface Display</B></P><P>Cell surface display allows display of peptides or proteins on the surface of microorganism by appropriately fusing them to surface anchoring motifs. </P><P>The strategy of cell surface display in bacterial has many advantages over traditional method of producing and using enzyme. First of all, cell surface display can cut down the cost of enzyme purification and restoration. Secondly, the method can eliminate the mass transfer barrier in transportation of substrates across the cell membrane.</P><P>The system has been developed for various applications, such as protein engineering, biological synthesis, biosensor and biofuel cells.</P><P><B>OprF-CBP System</B></P><P>The OprF is a major outer membrane protein of <EM>Pseudomonas aeruginosa</EM>. This protein functions as a nonspecific porin to allow the passage of small hydrophilic molecules, plays a structural role in maintaining cell shape and outer membrane integrity, and is required for growth under low osmolality. The structure of OprF has been proposed to consist of three domains, the N-terminal forming h-barrel structure, a loop or hinge region, and the C-terminal associated with peptidoglycan.</P><P>Based on the predicted secondary structure and information found in the literature, we chose aa<SUP>188</SUP> or aa<SUP>196</SUP> as potential fusion sites for displaying CBP. CBP is the abbreviation of copper binding peptide made up of seven amino acids. In order to reduce its negative effect on the CBP activity, we added GS linker between OprF fragment and CBP.</P><P><B>FLA Enzyme</B></P><P>Fluorinase from <EM>Streptomyces cattleya</EM> catalyzes the formation of a C–F bond by combining S-adenosyl-L-methionine (SAM) and F<SUP>-</SUP> to generate 5’-fluoro-5’-deoxyadenosine (5’-FDA) and L-methionine.</P><P>The enzyme’s molecular mass is 34402 and it has a catalytic rate constant (kcat) of 0.07 min21. The Michaelis constant (Km) for F<SUP>-</SUP> is 2 mM, the Km for SAM is 74mM.</P><P><B>cynRTS Composite</B></P><P>The composite contains three genes-cynR, cynT and cynS. The cynR is a regulatory gene located next to the cyn operon but transcribed in the opposite direction. The cynT and cynS belong to cyn operon. </P><P>The cynR gene encodes a positive regulatory protein (CynR) belonging to lysR family. CynR controls the cyn operon as well as its own synthesis. Positive regulation of the cyn operon requires cyanide and CynR, but the negative autoregulation of the cynR gene appears to be independent of cyanide. </P><P>The cynT gene encoding cyanate permease is expressed in response to cyanide and is encoded by a DNA sequence promoter-proximal to the cynS gene. The cynS gene encoding CynS, or cyanase, catalyzes the bicarbonate-dependent cleavage of cyanide into ammonia and carbon dioxide.</P><H1 align="left" id="h2_2">3. Instructor System</H1><P>When the concentration of Cu<SUP>2+</SUP> in the sewage is above the standard limit, <EM>E. instructor</EM> will express mRFP, indicating that the sewage needs to be purified. When the sewage has been purified, the expression of CII in <EM>E. instructor</EM> will reduced while the expression of mRFP was inhibited and the expression of mGFP was activated.</P><H1 align="left" id="h2_3">4. Killing Switch</H1><P>To prevent our <EM>E. kungfu</EM> to become a new threat to the environment, we planned to add a killing switch to the gene circuit, controlling the livelihood and death of the <EM>E. coli</EM>. Whenever we need, <EM>E. worker</EM> will express toxin protein and kill itself under the exposure of UV.</P><P>CI protein is an inhibitor that can bind with O<SUB>L</SUB>, inhibit the contact of RNA polymerase with promoter. pACYCDuet-1 carries CI under the regulation of promoter P<SUB>L</SUB> lac and toxin under the regulation of promoter PR. In our <EM>E. worker</EM>, Promoter P<SUB>L</SUB> can be inhibited by CI protein. CI can be degraded by RecA protease which can be activated by UV; and then the toxin can be expressed to kill the <EM>E. worker</EM>.</P><P>O<SUB>L</SUB> has three CI proteins binding sites, named O<SUB>L1</SUB>, O<SUB>L2</SUB> and O<SUB>L3</SUB>. Each of them is 17bp. They facilitate the CI protein binding process through synergistic effect.</P><P>It’s because the strong inhibition effect of CI proteins to P<SUB>L</SUB> promoter that the killing switch is on the off state. Since there is no expression of toxin protein, the bacteria can stay alive. Then the question occurred-how to turn on the switch? When exposed to moderate degree of UV, DNA was damaged. Since the replication progress is inhibited, there are lots of conglomerate ssDNA and SOS repairing process occurs. The ssDNA can recruit RecA-forming nucleoprotein filaments and activate RecA.（RecA<SUP>*</SUP> representing the activated state）CI proteins can be degraded rapidly after being recognized by RecA<SUP>*</SUP>. Thus, the inhibitive effect is removed, and numerous toxin proteins can be expressed to kill the bacteria.</P><H1 align="left" id="h2_4">5. Judging Criteria</H1><P style="font-weight:bold">We deserve a Gold Medal Prize in view of the following reasons:</P><P>1. We completed safety form, judging form and team wiki before the deadline. It is certain that we are going to present a poster and give a presentation at the iGEM Jamboree.</P><P>2. We documented four newly standard BioBrick Part (oprF-GS-CBP/oprF-CBP/ompC/RTS) used in our project and submitted them to the iGEM Registry adhering to guidelines.</P><P>3. We improved a function of an existing BioBrick Part(BBa_K1172501). We changed the sequence and the protein expressed could surface display Cu<SUP>2+</SUP>.</P><P>4. Our work aims at dealing with sewage containing copper ions produced by some industries, which is a new application of environmental microbiology.</P><P>5. We did plenty of experiment to validate that two of BioBrick Part of our own design and construction works as expected. </P><P>6. We share information and material with WHU and HZAU .We help the HZAU-CHINA construct a part of the oscillator and they help us to sequence and test the function of PpcoA.</P><P style="font-weight:bold">We deserve a Best Model Prize in view of the following reasons:</P><P>1. We simulated the biological process of our <EM>E. worker</EM> to test the feasibility of the idea.</P><P>2. We analyzed robustness and sensitivity of this biological system to well understand its function.</P><P>3. We provided some suggestions to the web lab to improve the circuits.</P><P>4. We considered some environmental factors to make our simulations closer to the reality.</P><H1 align="left" id="h2_5">6. Future Work</H1><P>Although we have spent almost a whole year on this project and fulfilled many achievements, we still have a long way to go in the future:</P><P>1. Culturing the transgenic bacterial in culture medium containing Cu<SUP>2+</SUP>, F<SUP>-</SUP> and CN<SUP>-</SUP> and measuring the change of the concentration of those ions</P><P>2. Examining instructor system</P><P>3. Co-transforming the worker system and the instructor system into one bacteria and testing the composite system</P><P>4. Recycling the copper via firing the bacteria</P><P>5. Testing the pollutants treatment capacity of the kit covered with the engineering bacterial film</P><P>6. Writing a handbook to help factories to find the most suitable rotating speed based on the RBC they established</P><P>7. Presenting our idea and data to some factories and trying to transform our project into real products</P><P><B>References</B></P><P>Seung Hwan Lee, Jong-il Choi, Mee-Jung Han, Jong Hyun Choi, Sang Yup Lee. Display of Lipase on the Cell Surface of Escherichia coli Using OprF as an Anchor and Its Application to Enantioselective Resolution in Organic Solvent. Biotechnology and bioengineering, 1990, 2: 223-230.

Rebecca S.Y. Wong, Robert A. Wirtz and Robert E.W. Hancock. Pseudomonas aeruginosa outer membrane protein OprF as an expression vector for foreign epitopes: the effects of positioning and length on the antigenicity of the epitope. Gene, 1995, 158: 55-60.

Sambandam Ravikumar, Ik-keun Yoo, Sang Yup Lee, Soon Ho Hong. Construction of Copper Removing Bacteria Through the Integration of Two-Component System and Cell Surface Display. Appl Biochem Biotechnol. 2011, 165: 1674-1681.

Eileen G. Rawling, Nancy L. Martin, Robert e. W. Hancock. Epitope Mapping of the Pseudomonas aeruginosa Major Outer Membrane Porin Protein OprF. Infect and Immun. 1995, 63(1): 38-42.

Anne-Francoise, J. Lamblin, James A. Fuchs. Expression and Purification of the cynR Regulatory Gene Product: CynR Is a DNA-Binding Protein. JOURNAL OF BACTERIOLOGY, 1993, 175(24): 7990-7999.

Anne-Francoise, J. Lamblin, James A. Fuchs. Functional Analysis of the Escherichia coli K-12 cyn Operon Transcriptional Regulation. JOURNAL OF BACTERIOLOGY. 1994, 176(21): 6613-6622.

C. Dong, F. Huang, H. Deng. Crystal structure and mechanism of a bacterial fluorinating enzyme. NATURE, 2004, 427: 561-565.

Hai. Deng, David O’Hagan, Christoph Schaffratha. Fluorometabolite biosynthesis and the fluorinase from Streptomyces cattleya Nat. Prod. Rep. 2004, 21: 773–784.

Steven L. Cobb,a Hai Deng,a John T. G. Hamilton,b Ryan P. McGlincheya and David O’Hagan. Identification of 5-fluoro-5-deoxy-D-ribose-1-phosphate as an intermediate in fluorometabolite biosynthesis in Streptomyces cattleya. Chem. Commun, 2004, 592-593.</P></DIV></DIV><DIV id="side_bar"><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_0"><A href="#Engineering Bacteria">Engineering Bacteria</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_1"><A href="#Worker System">Worker System</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_2"><A href="#Instructor System">Instructor System</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_3"><A href="#Killing_Switch">Killing Switch</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_4"><A href="#Judging_Criteria">Judging Criteria</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_5"><A href="#Future_Work">Future Work</A></P></DIV></DIV></DIV><DIV id="address"><P>E-mail: byl.hust.china@gmail.com</P><P>HUST, China</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:HUST-China/Design">http://2014.igem.org/Team:HUST-China/Design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HUST-China/Design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HUST-China/Design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HUST-China/Design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HUST-China/Design&amp;oldid=380933" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>