<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HUST-China_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HUST-China/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HUST-China/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-China/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HUST-China/Protocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HUST-China/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HUST-China/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="animat"><SPAN><P>oo</P></SPAN></DIV><DIV align="center" class="content_body"><DIV class="navigater" align="center"><UL><LI class="column" id="OVERVIEW"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Overview">Overview</A></SPAN></LI><LI class="column" id="PROJECT"><SPAN>Project</SPAN></LI><LI><A href="https://2014.igem.org/Team:HUST-China/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Toolkit">Toolkit</A></LI></UL><LI class="column" id="EXPERIMENT"><SPAN>Experiment</SPAN></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Protocol">Protocol</A></LI><LI><A href="https://2014.igem.org/Team:HUST-China/Result">Result</A></LI><LI class="column" id="MODELING"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Modeling">Modeling</A></SPAN></LI><LI class="column" id="HUMANPRACTICE"><SPAN><A href="https://2014.igem.org/Team:HUST-China/HumanPractice">Human Pratice</A></SPAN></LI><LI class="column" id="TEAM"><SPAN><A href="https://2014.igem.org/Team:HUST-China/Team">Team</A></SPAN></LI></DIV><DIV id="cont_column"><DIV class="chapter"><SPAN><FONT size="6px">Protocol</FONT></SPAN><H1 align="left" id="h2_0">Part 1: The Construction of the Worker System</H1><H3 align="left">Step 1: Gene Cloning</H3><P>To find the optimal temperature for oprF and <EM>cyn</EM>RTS composite amplification, we set a gradient of annealing temperature. The result shows that 58℃ is suitable to this PCR. The PCR product was stored in -20℃. The PCR reaction system was listed in table 1-1.</P><H3 align="center">Table 1-1 Gradient PCR System</H3><TABLE border="1" cellpadding="1" cellspacing="0" width="800"><TBODY><TR><TD>Components(50μl)</TD><TD>Volume(ml)</TD></TR><TR><TD>PrimerStar Buffer</TD><TD>10</TD></TR><TR><TD>dNTPs(2.5mM)</TD><TD>5</TD></TR><TR><TD>Primer-F(10μM)</TD><TD>1.5</TD></TR><TR><TD>Primer-R(10μM)</TD><TD>1.5</TD></TR><TR><TD>Template</TD><TD>1.5</TD></TR><TR><TD>PrimerStar</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>30</TD></TR></TBODY></TABLE><H3 align="left">Step 2: Optimization of oprF</H3><P>We choose the 188aa or 196aa of OprF as the CBP fusion points. Primers were designed (Table1-2) to clone the target fragment, and then the GS linker and/or the gene fragment coding CBP were added at the C-terminal of the target fragment.</P><P>The protein with GS linker, using the 188aa as CBP fusion point, was named as OprF-1. While the protein without GS linker, using the 196aa as CBP fusion point, was named as OprF-2.</P><H3 align="center">Table 1-2  Primers of Optimization oprF <TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="94">Primer	</TD><TD width="519">Sequence(5'→3')</TD></TR><TR><TD>oprF-CBP-F</TD><TD>CCGGAATTCAACTGAAGAACACCTT</TD></TR><TR><TD>oprF-CBP-R1</TD><TD>CCAGCCGCCATGATGCGGGGAAACCGGTTCCGGAGCCGGAGCGGC</TD></TR><TR><TD>oprF-CBP-R2</TD><TD>ATAGTTTAGCGGCCGCCGGCCAGCCGCCATGATGCGGGGA</TD></TR><TR><TD>oprF-GS-F</TD><TD>CCGGAATTCAACTGAAGAACACCTT</TD></TR><TR><TD>oprF-GS-R1</TD><TD>TGAACCTCCGCCACCTTTCGAACCACCGAAGTTGAAG</TD></TR><TR><TD>oprF-GS-R2</TD><TD>CCAGCCGCCATGATGCGGGGATGAACCTCCGCCACC</TD></TR><TR><TD>oprF-GS-R3</TD><TD>ATAGTTTAGCGGCCGCCGGCCAGCCGCCATGATGCGGGGA</TD></TR></TBODY></TABLE></H3><H3 align="left">Step 3: Construction of Vectors</H3><P>pET28a vector and the PCR product were digested with EcoRI and NotI from Takara, and then retrieved and purified with DNA retrieve kits from Omega.</P><P>Then, pET28a(+) vector, gene oprF-1/oprF-2/<EM>cyn</EM>RTS composite were linked together with T4 ligase to form a new vector: pET28a(+)-oprF-1/oprF-2/RTS. The reaction system for digestion and conjunction were listed in the table 1-3, 1-4 and the digestion results were displayed in fig1-1.</P><H3 align="center">Table 1-3  Reaction System for Digestion</H3><TABLE border="1" cellpadding="1" cellspacing="0" width="800"><TBODY><TR><TD width="131">Components(50μl)</TD><TD width="88">Volume(μl)</TD></TR><TR><TD>10×H Buffer</TD><TD>5</TD></TR><TR><TD>BSA </TD><TD>5</TD></TR><TR><TD>EcoRI</TD><TD>2.5</TD></TR><TR><TD>NotI </TD><TD>2.5</TD></TR><TR><TD>Fragment</TD><TD>25</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>10</TD></TR><TR><TD>Conditions</TD><TD>37℃ 1h</TD></TR></TBODY></TABLE><H3 align="center">Table 1-4  for Gene Conjunction</H3><TABLE border="1" cellpadding="1" cellspacing="0" width="800px"><TBODY><TR><TD>Components</TD><TD>System components(10μl)</TD></TR><TR><TD>Buffer(with T4 ligase)</TD><TD>5</TD></TR><TR><TD>Insert</TD><TD>4.5</TD></TR><TR><TD>Vector</TD><TD>0.5</TD></TR><TR><TD>Conditions</TD><TD>16℃ 1h</TD></TR></TBODY></TABLE><H3 align="center">Fig 1-1 Digested gene and plasmid</H3><H3 align="left">Step 4: Transforming the Plasmids into <EM>E. coli</EM> BL21 Strain </H3><P>Step 4: Transforming the Plasmids into <EM>E. coli </EM>BL21 Strain </P><P>Plasmids pET28a(+)-oprF-1 or pET28a(+)-oprF-2 or pET28a(+)-flA was transformed into <EM>E. coli</EM> BL21(DE3) and protein expression was analyzed.The strains were grown in LB medium containing 100ug/ml kanamycin at 37℃, 250rpm until OD600 reached 0.4–0.6. We then added IPTG to 0.5mM and continued the incubation at 28℃ overnight to induce the overexpression of OprF-CBP or OprF-GS-CBP or flA. The cells were collected, re-suspended with 10mM imidazole containing 0.1mM protease inhibitor PMSF and then disrupted using Selecta Sonopuls. After centrifugation, the sediment was treated with 1xSDS gel loading buffer and kept in boiling water for 5 minutes and applied to SDS-PAGE. The result was displayed in Fig 1-4.</P><H3 align="center">Table 1-5  Reaction System of Colonial PCR validation</H3><TABLE border="1" cellpadding="1" cellspacing="0" width="800"><TBODY><TR><TD>Components(10μl)</TD><TD>Volume(ml)</TD></TR><TR><TD>2×Ex Taq Mix </TD><TD>5</TD></TR><TR><TD>Primer-F(10μmol/L) </TD><TD>0.3</TD></TR><TR><TD>Primer-R(10μmol/L) </TD><TD>0.3</TD></TR><TR><TD>Template</TD><TD>0.3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>4.1</TD></TR></TBODY></TABLE><H3 align="center">Table 1-6  Reaction System of Double Digestion (37 ℃ 1h)</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Components(10μl)</TD><TD>Volume(μl)</TD></TR><TR><TD>10×H Buffer </TD><TD>1</TD></TR><TR><TD>BSA</TD><TD>1</TD></TR><TR><TD>EcoRI </TD><TD>0.5</TD></TR><TR><TD>NotI</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7</TD></TR></TBODY></TABLE><H3 align="center">Fig 1-2 Gel images of novel plasmids with OprF-CBP/GS-CBP testified results</H3><H3 align="center">Fig 1-3 Gel images of novel plasmids with <EM>cyn</EM>RTS composite testified results</H3><P>The <EM>E. coli</EM> BL21(DE3) strain transformed with pET28a(+)-flA was kindly donated by Prof. David O'Hagan and Prof. James H. Naismith from University of St Andrews, Saint Andrews, Scotland, United Kingdom.  </P><H3 align="left">Step 5: Expression of Protein</H3><P>
  Plasmid pET28a(+)-oprF-1/oprF-2/<EM>cyn</EM>RTS composite/flA was transformed into <EM>E. coli</EM> BL21(DE3) and we get for protein expression analysis. The strains were grown in Luria broth containing 100ug/ml kanamycin at 37℃, 250rpm until an absorbance of 0.4–0.6 at 600 nm was reached. We then added IPTG to 0.5mM and continued the incubation at 28℃ overnight to induce the overexpression of OprF-CBP/OprF-GS-CBP. The cells were collected, suspended with 10mM imidazole containing 0.1mM protease inhibitor PMSF and then disrupted using Selecta Sonopuls. After centrifugation, the sediment was treated with 1*SDS gel loading buffer and kept in boiling water for 5 minutes and applied to SDS-PAGE. </P><H3 align="left">Step 6: Surface Displaying Copper Ions</H3><P>
  To identify that whether our OprF has anchored on the cell membrane of <EM>E. coli</EM>, we performed immunofluorescence assay. HA tag was added to the N-terminal of OprF-CBP so that the recombinant protein OprF-CBP-HA can be specifically recognized by anti-HA antibody. When FITC labeled anti-IgG antibody was used as the secondary antibody and interacted with the primary antibody, green fluorescence could be observed in the cell membrane of <EM>E. coli</EM> under the fluorescent microscope.</P><H1 align="left" id="h2_1">Part 2: The Construction of the Instructor System</H1><P>All the parts except PpcoA we used to construct <EM>E. instructor</EM> are from kits in Distribution 2013.They are listed in table2-1.</P><H3 align="center">Table 2-1 Parts from Kits in Distribution 2013</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Name</TD><TD>Parts</TD><TD>Well</TD><TD>Short Description</TD></TR><TR><TD>CII</TD><TD>BBa_P0153</TD><TD>9A(plate3,2013)</TD><TD>Protein</TD></TR><TR><TD>CI</TD><TD>BBa_C0051</TD><TD>3A(plate3,2013)</TD><TD>Protein</TD></TR><TR><TD>RFP</TD><TD>BBa_E1010</TD><TD>18F(plate5,2013)</TD><TD>engineered mutant of red fluorescent protein</TD></TR><TR><TD>GFP</TD><TD>BBa_E0044</TD><TD>14G(plate5,2013)</TD><TD>mGFP mut3b+AAV</TD></TR><TR><TD>PR</TD><TD>BBa_R0051</TD><TD>6K(plate5,2013)</TD><TD>CI regulated promoter</TD></TR><TR><TD>PRE</TD><TD>BBa_R0053</TD><TD>6M(plate5,2013)</TD><TD>CII regulated promoter</TD></TR></TBODY></TABLE><H3>Assemble all the elements via standard restrict sites:</H3><H4 align="left">1. PCR: PpcoA (add EcoRI, XbaI, SpeI and PstI restriction sites)</H4><H4 align="left">2. Double digestion: PpcoA PCR products and rbs+CII(LVA)+ter+ter/pSB1C3(BBa_P0153):</H4><H4 align="left">3. Colony PCR Validation (primer PpcoA F and PpcoA R):</H4><H5>The plasmid pPcoA-rbs+CII+LVA/pSB1C3 was extracted and sequenced.</H5><H4 align="left">4. Double digestion: PpcoA-rbs+CII+LVA/pSB1C3 and rbs+CI+tag-dT:</H4><H4 align="left">5. Colony PCR Validation(primer PpcoA F and cI R):</H4><H5 align="center">The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT/pSB1C3 was extracted and sequenced.</H5><H4 align="left">6. Double digestion: PCR product of rbs+mRFP+tag and dT/pSB1C3:</H4><H4 align="left">7. Colony PCR Validation (primer mRFP F1 and mRFP R1):</H4><P>The plasmid rbs+mRFP+tag-dT/pSB1C3 were extracted and sequenced.</P><H4 align="left">8. Double digestion: PRE/pMD-19T and rbs+mRFP+tag-dT/pSB1C3:</H4><H4 align="left">9. Colony PCR Validation (primer PRE F and mRFP R1):</H4><P h5="" align="left">The plasmid PRE-rbs+mRFP+tag-dT/pMD19-T was extracted and sequenced.</P><H3>step 2: Transformation of bacteria and colony PCR(primer VF2 and VR)</H3><H4 align="left">1. Promoter lambda (cI regulated) with GFP (+LVA) reporter in pSB1A3:</H4><H4 align="left">The plasmid PR-rbs+GFP+LVA/pSB1C3 was extracted and sequenced.</H4><H4 align="left">2. Double digestion:（3A assembly）</H4><H4 align="left"> PRE- rbs+mRFP+tag-dT/pMD19-T</H4><H4 align="left">3. PpcoA-rbs+CII+LVA-rbs+CI+tag-dT/pSB1C3</H4><H3 align="left">4. Colony PCR validation (primer cI F and mRFP R1):</H3><H4 align="left">The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT/pSB1C3 was extracted and sequenced using primer VF2 and VR.</H4><H3>5. Double digestion PR-rbs+GFP+LVA/pSB1C3</H3><H4 align="left">6. PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT/pSB1C3</H4><H4 align="left">7. Colony PCR Validation (primer PRE F and GFP R):</H4><H4 align="left">The plasmid PpcoA-rbs+CII+LVA-rbs+CI+tag-dT-PRE-rbs+mRFP+tag-dT-PR-rbs+GFP+LVA/pSB1C3 was extracted and sequenced using primer VF2 and VR.</H4><H1 align="left" id="h2_1">Part 3: The Construction of the Kill Switch</H1><H3>Plasmid construction:</H3><P>PCR: CI repressor from <EM>E. coli</EM> phage lambda (+LVA) in pSB1C3（BBa_C0051）was amplified to add RBS using primer:</P><P>cI F: 5'–TATGAATTCTCTAGATAAGGAGATATAATGAGCACAAAAAAG-3' and cI R: 5'–TAATCTGCAGACTAGTGCGATCTACACTAGCACTATC-3'</P><H3>Double digestion:</H3><H3><P> PCR product of cI+rbs obtained using gel extraction kit and double terminator (dT) in pSB1C3 were applied to double digestion. The reaction systems were listed in table 3-1, 3-2.</P></H3><H3 align="center">Table3-1 Digestion of Fragment (cI+rbs):</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Reaction system</TD><TD>50 μl</TD></TR><TR><TD>DNA solution</TD><TD>41 μl</TD></TR><TR><TD>EcoRI</TD><TD>2μl</TD></TR><TR><TD>SpeI</TD><TD>2 μl</TD></TR><TR><TD>10×H</TD><TD>5 μl</TD></TR></TBODY></TABLE><H3 align="center">Table 3-2 Digestion of pSB1C3-dT:</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Reaction system</TD><TD>50 μl</TD></TR><TR><TD>plasmid</TD><TD>41 μl</TD></TR><TR><TD>EcoRI</TD><TD>2μl</TD></TR><TR><TD>SpeI</TD><TD>2 μl</TD></TR><TR><TD>10×M</TD><TD>5 μl</TD></TR></TBODY></TABLE><P>The mixture was incubated in 37℃ for 2h , then gel electrophoresis and gel extraction were performed:</P><P>Then, we linked them together at room temperature for 1h. The reaction system listed in table 3-3. </P><H3 align="center">Table 3-3 Reaction System of  Ligation</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Ligation system</TD><TD>10μl</TD></TR><TR><TD>cI+rbs</TD><TD>4μl</TD></TR><TR><TD>Vector</TD><TD>1μl</TD></TR><TR><TD>T4 Ligase(contain 10x T4 Ligase Buffer)</TD><TD>5μl</TD></TR></TBODY></TABLE><H3>Colony PCR validation:</H3><P>Select monoclonal and applied to PCR:</P><P>The positive colone was cultured in LB overnight for plasmid extraction and sequencing.</P><P>Assemble all elements one by one using the same method:</P><P>Double digestion: </P><H3>P<SUB>L</SUB> lacI (BBa_R0011) in pSB1A3 and pSB1C3-cI+rbs-dT</H3><H3>Colony PCR validation(primer VF2 and cI R):</H3><H3>The plasmid PL lacI-rbs+cI-dT/pSB1C3 was extracted and sequenced.</H3><H3>Double digestion:</H3><H3>P<SUB>L</SUB>/pMD-19T and rbs+mRFP+tag PCR products:</H3><P>Colony PCR validation(primer VF2 and mRFP R1):</P><H3>Colony PCR validation(primer VF2 and mRFP R1):</H3><H3>The plasmid PL -rbs+mRFP+tag/pSB1C3 was extracted and sequenced.</H3><H3>Double digestion:</H3><H3>P<SUB>L</SUB> -rbs+mRFP+tag/pMD19-T and PL lacI-rbs+cI-dT/pSB1C3:</H3><H3>Colony PCR validation (primer VF2 and cI R):</H3><P>The plasmid P<SUB>L</SUB> lacI-rbs+cI-dT- PL -rbs+mRFP+tag/pMD-19T was extracted and sequenced.</P><H1 align="left" id="h2_1">Part 4: Measuring of Growth Curve</H1><H3 align="left">Step 1:<P> 200μL overnight-cultured bacterial was added to vials containing newly prepared culture medium (containing 100μg/ml Kanamycin). Set one vial with culture medium as blank. All the samples were cultured in the shaking incubator with rotational speed at 200 rpm and temperature at 37℃.</P></H3><H3 align="left">Step 2:<P> OD<SUB>600</SUB> values was measured using the nucleic acid analyzer. When the OD<SUB>600</SUB> values of culture reached 0.6, 200μL culture was added to a new vial containing 5mL LB, 100μg/ml Kanamycin, 0.1M IPTG and CN-/F-/Cu2+ with different concentration. Set one vial without bacteria as blank. All these samples were cultured in the shaking incubator with rotational speed at 200 rpm and temperature at 37℃.</P></H3><H3 align="left">Step 3:<P> OD600 of samples from the vials were measured every 30 min. </P><P h1="" align="left" id="h2_1">Part 5: The Standardization of Parts</P></H3><H2>Materials (used in this part)</H2><H4>The materials we used in this part are listed in table 5-1 and table 5-2</H4><H3 align="center">Table 5-1: Bacterial strains and plasmids</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Strains and vectors</TD><TD>Relevant genotype and characteristics</TD><TD>Originate</TD></TR><TR><TD><EM>E. coli</EM> DH5α</TD><TD>Strains</TD><TD>conserved in the lab</TD></TR><TR><TD>pMD18T</TD><TD>Vectors</TD><TD>TaKaRa Biotechnology (DaLian)Co.,Ltd.</TD></TR><TR><TD>pSB1C3</TD><TD>Vectors</TD><TD>iGEM package</TD></TR></TBODY></TABLE><H3 align="center">Table 5-2: Primers' Name and Sequence</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Primer</TD><TD>Sequence(5’→3’)</TD></TR><TR><TD>oprF-F</TD><TD>G GAATTC GCGGCCGC T TCTAGAG ATGAAACTGAAGAACACC</TD></TR><TR><TD>oprF-R</TD><TD>TTT CTGCAG CGGCCGC T ACTAGT ACCAGCCGCCATGAT</TD></TR><TR><TD>ompC-F</TD><TD>G GAATTC GCGGCCGC T TCTAGA G ATGCGTCTTGGCTT</TD></TR><TR><TD>ompC-R</TD><TD>TGCA CTGCAG CGGCCGC T ACTAGT ATTAGAACTGGTAAACC</TD></TR><TR><TD>RTS-F</TD><TD>G GAATTC GCGGCCGC T TCTAGA G TCAGAACGGTTTGG</TD></TR><TR><TD>RTS-R</TD><TD>TGCA CTGCAG CGGCCGC T ACTAGT ACTACCGTGATTCATTTC</TD></TR><TR><TD>flA-F</TD><TD>G GAATTC GCGGCCGC T TCTAGA G ATGGCTGCCAACAGCAC</TD></TR><TR><TD>flA-R</TD><TD>TGCA CTGCAG CGGCCGC T ACTAGT ATCAGCGGGCCTCGACCC</TD></TR><TR><TD>ompC-m-A</TD><TD>CCGCTTCTAGAGATGCGTCTTGGCTT</TD></TR><TR><TD>ompC-m-B</TD><TD>GGTGTCACCACCGAACTCTGGCAGT</TD></TR><TR><TD>ompC-m-C</TD><TD>ACTGCCAGAGTTCGGTGGTGACACC</TD></TR><TR><TD>ompC-m-D</TD><TD>GTTACGCCACTAGTAAAGCCTTCAC</TD></TR><TR><TD>ompC-m-E</TD><TD>ATTGCAGGTAAGCCAGGGACGGACG</TD></TR><TR><TD>ompC-m-F</TD><TD>GTGAAGGCTTTGCTAGTGGCGTAAC</TD></TR><TR><TD>ompC-m-G</TD><TD>CGTCCGTCCCTGGCTTACCTGCAAT</TD></TR><TR><TD>ompC-m-H</TD><TD>CCGCTACTAGTATTAGAACTGGTAAACC</TD></TR></TBODY></TABLE><H3 align="left">Step 1: Gene Standardized</H3><H3>P4R amplification with the primers set as the template (pET28a-OprF-CBP/OprF-GS-CBP/<EM>cyn</EM>RTS composite/FLA). The conditions of the reaction were listed in table 5-3. </H3><H3 align="center">Table 5-3: The Reaction System to harvest target genes</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Components(50μl)</TD><TD>Volume(μl)</TD></TR><TR><TD>Template</TD><TD>1.25</TD></TR><TR><TD>dNTPs</TD><TD>2.5</TD></TR><TR><TD>10×LA PCR Buffer</TD><TD>5</TD></TR><TR><TD>LA Tag</TD><TD>0.5</TD></TR><TR><TD>Primer-F(10μmol/L)</TD><TD>2.5</TD></TR><TR><TD>Primer-R(10μmol/L)</TD><TD>2.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O </TD><TD>35.75</TD></TR></TBODY></TABLE><H3 align="left">Step 2: Obliteration of the Illegal Restriction Sites in the Gene<P>It is necessary to add standard restriction enzyme sites-EcoRI/XbaI/SpeI/PstI to both terminals of each gene. However, the sequence of ompC contains EcoRI, SpeI and PstI restriction site. So we obliterated the restriction sites by site-directed mutagenesis based on overlap extension PCR. Briefly, target mutation (GAATTC→GAGTTC, ACTAGT→GCTAGT , CTGCAG→CTCCAA, ) was introduced into primers (ompC-m-A/B/C/D/E/F/G/H in table 5-2), and the four previous PCR products(ompC-AB, ompC-CD, ompC-EF, ompC-GH) were used as template for the second PCR, and the two previous PCR products(ompC-AD, ompC-EH) were used as template for the third PCR. The final PCR segment with target mutation sites was then cloned into pMD18-T vector for sequencing. The three PCR systems were listed in table 5-4, 5-5, 5-6. </P></H3><H3 align="center">Table 5-4: Reaction System of the 1st PCR</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Components(50μl)</TD><TD>Volume(μl)</TD></TR><TR><TD>2×Ex Taq Mix</TD><TD>25</TD></TR><TR><TD>ompC-m-A/C/E/G(10μM)</TD><TD>1.5</TD></TR><TR><TD>ompC-m-B/D/F/H(10μM)</TD><TD>1.5</TD></TR><TR><TD>Template</TD><TD>1.5</TD></TR><TR><TD>H<SUB>2</SUB>O </TD><TD>20.5</TD></TR></TBODY></TABLE><H3 align="center">Table 5-5: Reaction System of the 2nd PCR</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Components(50μl)</TD><TD>Volume(μl)</TD></TR><TR><TD>2×Ex Taq Mix</TD><TD>25</TD></TR><TR><TD>ompC-A/E(10μM)</TD><TD>2.5</TD></TR><TR><TD>ompC-D/H(10μM)</TD><TD>2.5</TD></TR><TR><TD>ompC-AB/EF</TD><TD>5</TD></TR><TR><TD>ompC-CD/GH</TD><TD>5</TD></TR><TR><TD>H2O </TD><TD>10</TD></TR></TBODY></TABLE><H3 align="center">Table 5-6: Reaction System of 3rd PCR</H3><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Components(50μl)</TD><TD>Volume(μl)</TD></TR><TR><TD>2×Ex Taq Mix</TD><TD>25</TD></TR><TR><TD>ompC-F(10μM)</TD><TD>2.5</TD></TR><TR><TD>ompC-R(10μM)</TD><TD>2.5</TD></TR><TR><TD>ompC-AD</TD><TD>5</TD></TR><TR><TD>ompC-EH</TD><TD>5</TD></TR><TR><TD>H<SUB>2</SUB>O </TD><TD>10</TD></TR></TBODY></TABLE><H3 align="left">Step 3: Insertion of the Standard Parts into plasmid backbone-pSB1C3<P>pSB1C3 backbone and the standardized parts was digested with EcoRI and PstI and then retrieved and purified with kits from Omega. pSB1C3 and the standardized parts were then linked together to from new Biobricks.</P></H3><H3 align="left">Step 4 : Submission of New Standardized Parts<P>After completing all the validation, the four parts were submitted to iGEM official organization in the early October and arrived in New York in 8th Oct. The information was listed in the table 5-7.</P><TABLE border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD>Standard biobricks</TD><TD>description</TD></TR><TR><TD>pSB1C3-oprF-1(EcoRI PstI）</TD><TD>A major outer membrane protein of <EM>Pseudomonas aeruginosa</EM> functions as a nonspecific porin to allow the passage of small hydrophilic molecules. We cut the protein at 188aa and add a CBP tag to C-terminal with a GS linker between them.</TD></TR><TR><TD>pSB1C3-oprF-2(EcoRI PstI）</TD><TD>A major outer membrane protein of <EM>Pseudomonas aeruginosa<EM> functions as a nonspecific porin to allow the passage of small hydrophilic molecules. We cut the protein at 196aa and add a CBP tag to C-terminal.</EM></EM></TD></TR><TR><TD>pSB1C3-ompC(EcoRI PstI）</TD><TD>Outer membrane porin protein C from <EM>Escherichia coli</EM> BL21(DE3)</TD></TR><TR><TD>pSB1C3-RTS(EcoRI PstI）</TD><TD>Cyn operon in <EM>Escherichia coli</EM> BL21(DE3) (cynR+cynT+cynS) is about cyanate detoxification.</TD></TR><TR><TD>pSB1C3-FLA(EcoRI PstI）</TD><TD>Fluorinase enzyme from Streptomyces cattleya catalyzing the formation of a C–F bond by combining S-adenosyl-L-methionine (SAM) and F- to generate 5’-fluoro-5’-deoxyadenosine (5’-FDA) and L-methionine.</TD></TR></TBODY></TABLE><DIV id="side_bar" background-color:transparent=""><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_0"><A href="#Part_1">Worker System</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_1"><A href="#Part_2">Instructor System</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_2"><A href="#Part_3">Kill Switch</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_2"><A href="#Part_4">Growth Curve</A></P></DIV><DIV class="anchor-h2" id="h2num_1"><P style="text-align:right" class="h2_2"><A href="#Part_5">Standardization</A></P></DIV></DIV><DIV id="address"><P>E-mail: byl.hust.china@gmail.com</P><P>HUST, China</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:HUST-China/Protocol">http://2014.igem.org/Team:HUST-China/Protocol</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HUST-China/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HUST-China/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HUST-China/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HUST-China/Protocol&amp;oldid=384893" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></H3></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>