<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row splashala"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-12"><UL class="nav nav-pills nav-justified"><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Background"> Background </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Notebook"> Notebook </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Collaborations">  Collaborations</A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Attributions">  Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Basic_Part">  Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Composite_Part">  Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Part_Collection">  Part Collection </A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI></DIV></DIV></DIV><H3> Background on EUTs: </H3><P class="main">
Ethanolamine-utilizing microcompartments (EUTs) are endogenous to <I>Salmonella enterica, Clostridium difficile </I>, and <I>Escherichia coli </I> (Held et al., 2016). EUT bacterial microcompartments (BMCs) house enzymes responsible for isolation and degradation of cytotoxic ethanolamine, and the genomic EUT operon genes are expressed only in the presence of ethanolamine and vitamin B12 (Choudhary et al., 2012). The 17-gene operon encodes for five shell-protein subunits (EutSMNLK), and a number of ethanolamine-catabolizing enzymes (2012). These preliminary findings on the EUT BMC’s structure and function indicate that it’s among the simplest of the known proteinaceous BMCs, whereas the formation of propanediol-utilizing microcompartments (PDUs) and carboxysomes require orchestrated expression of larger genomic operons (2012). The most prominent aspect of EUTs lending them to utilization as model protein nanocages is the capacity for a single gene’s product (EutS) to homohexamerize and further oligomerize into polyhedral protein nanocages. As Choudhary et alii illustrated, EutS and EutSMNLK shell proteins don’t merely form hydrophobically agglomerated inclusion bodies; they’re shown to form hydrophilic semi-regular polyhedrons of ~50nm and 100-150nm, respectively. As previously indicated, amino acids 1-19 of the EutC gene product appear to target covalently bound proteins to the interior of the forming bacterial microcompartment, as evidenced by the fluorescent puncta (points) observed in <I>E. coli </I> coexpressing EutS and EutC:eGFP, or EutSMNLK and EutC:eGFP. 1-19AA-EutC therefore targets peptides to the interior of the EUT shell scaffold, allowing for compartmentalization of potential cargos or components. To better characterize this targeting peptide, Choudhary et alii used 1-19AA-EutC to successfully target the bulky 464kDa homotetrameric ß-galactosidase to the interior of EutS and EutSMNLK protein shells: ß-galactosidase retained its functionality, hydrolyzing 5-bromo-4-chloro-3-indolyl- ß-D-galactopyranoside (X-gal) into the insoluble, blue colored 5,5’-dibromo-4,4’-dichloro-indigo (2012). In E. coli cultures coexpressing EutSMNLK and 1-19AA-EutC: ß-galactosidase in the presence of X-gal, accumulations of the vibrant indole indicator were observed within the BMCs, whereas otherwise equivalent cultures without EutS displayed cytosolically diffuse localization of the indigo indicator. These results indicate that the quasi-polyhedral EutS shells may exhibit greater permeability of sequestered products or small molecule cargo than their five-component EutSMNLK counterparts (2012). As Held et alii allude, the contemporary understanding of EUT BMCs’ evolutionarily origins as selectively segregating nano-bioreactors lends to myriad synthetic applications of EUT functionality (Held, 2016).   </P><FIGURE align="center"><FIGCAPTION align="center">Fig. I: (A). EUT-operon diagram (asterisks indicate genes with proposed N-termini targeting peptide sequences—EutG N-terminal sequence was not shown to target assembling EUTs); (B). EUT BMC functional schematic indicating the putative biochemical pathway for the catabolism of cytotoxic ethanolamine substrate into biologically-inert products including ethyl alcohol, acetyl-phosphate, and acetyl-CoA. Figure courtesy Choudhary et alii research team; cited from pp. 3 of <I> Engineered Protein Nano-Compartments for Targeted Enzyme Localization </I> in PlosOne.  </FIGCAPTION><H3> Background on Photoisomerizing Amino Acid Substitutions: </H3><P class="main">
Hoersch et alii’s research on the incorporation of azobenzene into protein nanocompartments to impart light-inducible conformational modularity has granted utilitarian advantage to synthetic and molecular biologists alike. Their successful efforts to photoisometrically manipulate the open/closed conformation of ATPase group II (BiP) chaperonins pioneers a path toward precise molecular mechanization, and consequently confers greater abilities unto researchers to construct efficient, novel biosynthetic pathways. While the barrel-and-lid structure of BiP chaperonins isn’t directly analogous with BMCs like EUTs, we can glean invaluable insights from Hoersch et alii’s investigations: photoisomerizing noncanonical amino acids, when substituted into ideal loci of proteins, can allosterically manipulate tertiary and quaternary protein structures. Thiol-reactive azobenzene-dimaleimide was introduced to selectively crosslink solvent-accessible cysteine residues on the translated group II chaperonins (Hoersch et al., 2013). ∆C/K87C/S199C mutants of the chaperonin were azobenzene-crosslinked as such, and exhibited bidirectional light-inducible conformational change when examined on a native gel assay (2013). Further investigation of the conformational modularity of these chaperonins was carried out with cryo-electron microscopy; results corroborated the native gel findings (2013). In light of future applications of photoswitchable molecular mechanization, Hoersch et alii note that protein complexes have been evolutionarily winnowed to exhibit deep energy wells in conformational landscapes that reflect thermodynamic ideals (in the case of chaperonins, energy wells exist for the open and closed states) (2013). To properly photo-mechanize protein structures with crosslinkers like azobenzene-dimaleimide, researchers should computationally verify that the conformational energy well(s) to overcome don’t exceed the minute forces allosterically imbued by the photoisomerizing moieties. To address these biochemical and thermodynamic constraints, we’ve employed molecular modeling engines like PyMol and Rosetta to run predictions on the energetic feasibility of proposed amino acid substitutions.</P><FIGURE align="center"><FIGCAPTION align="center">Fig. II: Diagram showing wavelengths responsible for azobenzene cis-trans and trans-cis conformational changes, and corresponding allosteric modulation of tertiary protein structure (i.e. open-closed or closed-open conformational change) in the group II chaperonin. Figure courtesy Hoersch et alii’s research team; cited from pp. 929 of <I>Reprogramming an ATP-driven protein machine into a light-gated nanocage </I> in Nature Nanotechnology. </FIGCAPTION><H3> Background on Computational Predictive Molecular Modelling: </H3><P class="main">
In an effort to avoid laborious high-throughput assays of potential azobenzene-containing EutS mutants, we’ve opted to employ powerful modeling software to compute answers to the investigation’s biochemical constraints. As previously indicated, Hoersch’s research team used PyMol and Rosetta to find all cysteine residue pairs within 5-14Å in the closed conformation, and 16.6-19.5Å in the cytosolically-accessible state. Of the nine cysteine residue candidates for crosslinking in the chaperonin, three residue pairs were investigated, all of which appeared to impart some degree of photoswitchable conformational mechanization when crosslinked with azobenzene-dimaleimide (2013). In our fundamentally equivalent endeavors to photoisometrically mechanize the assembly and disassembly of EutS shell proteins, PyMol and Rosetta have been employed to incandesce potential candidate loci for noncanonical amino acid substitution(s). In light of Choudhary et alii’s experimental observation that substitution of the conformationally labile glycine with valine at the 39th residue of EutS resulted in a mutant isoform ∆V/G39V-EutS incapable of punctually sequestering coexpressed eGFP:1-19AA-EutC (ergo incapable of forming complete, functional BMCs), we conjecture that an amber stop codon substitution for an azobenene moiety containing amino acid at or near 39AA will serve as an ideal candidate for photoisomerization-driven molecular mechanization of EUTs. </P><H3> References: </H3><P class="main">
Tsoy, Olga, Dmitry Ravcheev, and Arcady Mushegian. &quot;Comparative Genomics of Ethanolamine Utilization.&quot; <I>Journal of Bacteriology </I>191.23 (2009): 7157-164. <I>American Society for Microbiology. Web. 03 July 2016. </I></P><P class="main">
Choudhary, Swati, Maureen B. Quin, Mark A. Sanders, Ethan T. Johnson, and Claudia Schmidt-Dannert. &quot;Engineered Protein Nano-Compartments for Targeted Enzyme Localization.&quot; <I> PLoS ONE </I> 7.3 (2012): 1-11. Web. </P><P class="main">
Held, Mark, Alexander Kolb, Sarah Perdue, Szu-Yi Hsu, Sarah E. Bloch, Maureen B. Quin, and Claudia Schmidt-Dannert. &quot;Engineering Formation of Multiple Recombinant Eut Protein Nanocompartments in E. Coli.&quot; <I>Nature Sci. Rep. Scientific Reports </I>6 (2016): 1-15. Web. </P><P class="main">
Renfrew, P. Douglas, Eun Jung Choi, Richard Bonneau, and Brian Kuhlman. &quot;Incorporation of Noncanonical Amino Acids into Rosetta and Use in Computational Protein-Peptide Interface Design.&quot; <I> PLoS ONE </I>7.3 (2012): 1-15. Web. </P><P class="main">
Bonacci, Walter, Poh K. Teng, Bruno Afonso, Henrike Niederholtmeyer, Patricia Grob, Pamela A. Silver, and David F. Savage. &quot;Modularity of a Carbon-fixing Protein Organelle.&quot; <I> Proceedings of the National Academy of Sciences </I>109.2 (2011): 478-83. Web.
 </P><P class="main">
Smith, Colin A., and Tanja Kortemme. &quot;Predicting the Tolerated Sequences for Proteins and Protein Interfaces Using RosettaBackrub Flexible Backbone Design.&quot; <I> PLoS ONE </I>6.7 (2011): 1-11. Web. </P><P class="main">
Hoersch, Daniel, Soung-Hun Roh, Wah Chiu, and Tanja Kortemme. &quot;Reprogramming an ATP-driven Protein Machine into a Light-gated Nanocage.&quot; <I> Nature Nanotech Nature Nanotechnology </I>8.12 (2013): 928-32. Web.
 </P><P class="main">
Kofoid, Eric, Chad Rappleye, Igor Stojiljkovik, and John Roth. &quot;The 17-Gene Ethanolamine (eut) Operon of Salmonella Typhimurium Encodes Five Homologues of Carboxysome Shell Proteins.&quot; <I> Journal of Bacteriology </I> 181.17 (1999): 5317-329. Web.</P><P class="main"><I>BioBrick® Assembly Kit </I> | NEB. Digital image. <I> New England Biolabs. </I> N.p., n.d. Web. 3 July 2016. www.neb.com. </P></FIGURE></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>