Team:CU-Boulder/Experiments
Home
Project
Description
Background
Experiments
Results
Notebook
Team
Team
Collaborations
Attributions
Parts
Parts
Basic Parts
Composite Parts
Part Collection
Safety
Human Practices
Experimental Overview
Bacterial microcompartments represent fascinating opportunities for molecular level control. We thought to take the EutS component of EutSMNLK and use it to make single component microcompartments. We would visualize these microcompartments using a EutC1-19 tagged eGFP, and then test a variety of expression levels to see what levels would be most effective, and so decided using a two-plasmid system to enable us to swap these two components as we saw fit would help us characterize the compartments. To see what formed the best compartments, we would take images via fluorescent microscopy of the localization of our EutC-eGFP.
The second goal of our project was to make a light-sensitive compartment that we could break at will using light by incorporation of an azo-phenylalanine compound containing non-canonical amino acid. In order to accomplish this, we explored the possible mutation sites using molecular modelling, and then introduced a third plasmid to our two-plasmid system containing the necessary expression of a non-canonical tRNA synthetase. We then used fluorescent microscopy to test whether we could construct EutS compartments incorporating azo-phenylalanine, and further test if irradiation with certain light wavelengths could break the cages, distributing the EutC-eGFP throughout the cell.
The last goal we had was to characterize the Eut compartment natively found in our NEB-5alpha, and, assuming that the EutC tag still worked, mutate the genome-encoded Eut compartment via CRISPR to introduce lactose induction and later the same amber loci, which we could then use to produce light-sensitive nanocompartments natively in our E.coli.
eGFP Visualization of Bacterial Microcompartment
In order to characterize our microcompartment and ensure that it could still form upon transition to BioBrick, we needed a way to measure formation. For this, we used EutC1-19, a portion of the amino acid sequence for the EutC protein, a member of the Eut operon. Past research has shown that by attaching these 19 amino acids to the sequence of eGFP, you can localize the eGFP inside of EutSMNLK cages. This idea would enable us to test our EutS compartments at various levels of expression and see how they differed in formation, size, and localization efficacy.
In order to use this method of characterization, we first needed to create a two-plasmid system which we could co-transform with our NEB-5alpha E.coli. We used PCR with BioBrick prefix and suffix extended primers to take EutS from BBa_K311004, the EutSMNLK sequence, and then use similar primers with a EutC 1-19 extension to tag our eGFP. These two PCR products can then be ligated into different plasmid backbones, transformed, and then grown with the two necessary antibiotics. Using pre-prepared expression cassettes with different promoters and binding sites, we explored which expression levels produced the best results under fluorescent microscopy.
EutS eGFP Compartmentalization
Two-Plasmid System localizes fluorescent proteins
In order to characterize our microcompartment and ensure that it could still form upon transition to BioBrick, we needed a way to measure formation. For this, we used EutC1-19, a portion of the amino acid sequence for the EutC protein, a member of the Eut operon. Past research has shown that by attaching these 19 amino acids to the sequence of eGFP, you can localize the eGFP inside of EutSMNLK cages. This idea would enable us to test our EutS compartments at various levels of expression and see how they differed in formation, size, and localization efficacy.
In order to use this method of characterization, we first needed to create a two-plasmid system which we could co-transform with our NEB-5alpha E.coli. We used PCR with BioBrick prefix and suffix extended primers to take EutS from BBa_K311004, the EutSMNLK sequence, and then use similar primers with a EutC 1-19 extension to tag our eGFP. These two PCR products can then be ligated into different plasmid backbones, transformed, and then grown with the two necessary antibiotics. Using pre-prepared expression cassettes with different promoters and binding sites, we explored which expression levels produced the best results under fluorescent microscopy.
Experimental Design of EutS Visualization
Light Activation of Bacterial Microcompartment
Visualization of EutS hexamers coming together (one monomer from each hexamer shown).
In order to produce a bacterial microcompartment capable of light sensitive breakage, we would first need to produce a non-canonical amino acid incorporating azo-phenylalanine, the light-sensitive compound we wanted to use. To accomplish this, we first needed to come to an understanding of how the EutS compartment monomers assembled into a full compartment using modelling of their hexameric structures. Once we understood the superstructure, we then chose sites which could interfere with the association between individual EutS proteins.
Wild type EutS residue with mutation candidate residues colored red.
To do this, we used PyRosetta, a Python-based interface to the Rosetta molecular modeling suite. It enables a user to create custom molecular modeling algorithms with Rosetta sampling and scoring functions using Python scripting. Using the basic PyRosetta scoring function, the wild type EutS protein was scored to find a reference value. Then, one by one, each residue was replaced with a tryptophan to simulate the large size of the non-canonical residue. These mutations were scored and the mutations with energy values which were 75% or greater of the wild type were chosen as possible mutation candidates.
The next step was to use the molecular modeling program PyMOL to visualize the docking of the hexamers to better select residues to mutate. Residues which were located where two hexamers came together and which also had energy values similar to the wild type were chosen as the final targets, so that the stereochemical change of the non-canonical amino acid would be theoretically disrupting inter-hexamer interactions, but not intra-hexamer.
Once these sites were chosen, we performed PCR mutagenesis on our EutS-containing plasmid to change our target residues to Amber stop codons. Once we accomplished these mutations, we introduced a third plasmid into our system, containing a tRNA synthetase that recognizes AzoPhe, which with its other associated proteins could incorporate AzoPhe at the amber stop codons we introduced to EutS. After growing these cells with pre-irradiated AzoPhe, we were able to see whether the mutant EutS could still form a microcompartment, and whether irradiating the compartment with blue light would be able to cause it to dissociate and disperse the localized eGFP throughout the cell.
EutS Mutation Concept
Genome Integration of Light Activated Bacterial Microcompartment
CRISPR editing was performed using an arabinose inducible Cas9 (pX2-Cas9) and a temperature inducible lambda red system to confer recombineering capabilities in E. coli, as described in Bassalo et al 2016. Cas9 from Streptococcus pyogenes was cloned into the broad host range vector pBTX-2 (Prior et al. 2010) and the lambda red system genes was expressed from the pSIM5 plasmid (Datta et al. 2006). The target E. coli strain previously transformed with pX2-Cas9 and pSIM5 was grown to mid-log phase, followed by an incubation at 42˚C for 15 minutes to induce the lambda red system. Induced cells were made electrocompetent and transformed with 200 ng of the gRNA vector, followed by outgrown in LB media supplemented with 0.2% arabinose for 3 hours. The gRNA vector also contained the corresponding homology cassette to repair the double strand break and introduce the designed mutation, as described in Garst et al 2016.The library design for genomic EutS saturation mutagenesis was designed as described in Garst et al 2016.
Bassalo MC, Garst AD, Halweg-Edwards AL, Grau WC, Domaille DW, Mutalik VK, Arkin AP, Gill RT. Rapid and Efficient One-Step Metabolic Pathway Integration in E. coli. ACS Synthetic Biology 2016. 5, 561-568.
Prior, J. E., Lynch, M. D., and Gill, R. T. (2010) Broad-host- range vectors for protein expression across gram negative hosts. Biotechnol. Bioeng. 106, 326−332.
Datta, S., Costantino, N., and Court, D. L. (2006) A set of recombineering plasmids for gram-negative bacteria. Gene 379, 109− 115.
Garst, A., Bassalo, M.C., Pines, G., Lynch, S.A., Halweg-Edwards, A.L., Liu, R., Liang, L., Wang, Z., Zeitoun, R., Alexander, W.G., Gill, R.T. (2016). Genome-wide mapping of mutations at single nucleotide resolution for protein, metabolic and genome engineering. Nature Biotechnology. In press.
Protocols
LB Culture
500 mL Deionized Distilled water
5g tryptone
2.5g yeast extract
2.5g NaCl
Autoclave for 30 minutes
Making Plate
Take out the cooled Agar and LB mixture and heat it up in the microwave
If making LB plates just pour them out in sterile plates on the counter top
If making antibiotic plates pour agar into graduated cylinder and add appropriate amount of antibiotic, this is again 1:1000 so say you make 100 mL you add 100µL
Stab culture recovery
Place 3 mL of LB in a 15 mL tube, careful not to keep the LB open for too long or to breath into the LB
Add appropriate antibiotic as a 1:1000 dilution into culture
Ex 3 mL add 3µL of antibiotic, all of the antibiotics are at the appropriate concentration
Stab the slant with a p10 pipet tip and place immediately in your LB solution.
Grow at 37 degrees Celsius and rotate
Making an Agar Gel
1% agar powder by mass with Tris buffer.
For example, 60 mL plate, .6 grams of agar and 59.4 grams of Tris buffer (or 59.4 mL of Tris buffer).
Mix in clean beaker and microwave until the solid is dissolved, move to the fume hood.
While heating, clean the cassette and comb with water and then DI water and dry with paper towel and move to the fume hood with the clamp.
Add 3 mL of ethidium bromide to the agarose mixture in the hood.
Set up the plate with the comb inside in the clamp and pour the solution into the plate and let cool for an hour.
Wash everything after use.
Running an Agar Gel
Place wells in the electrophoresis machine with the wells on the negative side for DNA.
Fill the device with 50 mL of Tris buffer or until the wells are covered with Tris.
Remove comb vertically, if the gel comes out of the cassette just push it gently back down with your finger.
Carefully load each sample by placing the pipet tip in the well and ejecting the sample, this should push the TRIS out and keep the sample in the well.
Each sample of DNA should be 5 µL of DNA to 1 µL of purple loading dye (6x)  (kept in freezer).
On the end of the gel place 5 µL of the ladder (kept in freezer), this already has the dye in it.
Lock lid on, this should be pushed all the way down connecting the plastic.
Settings for machine, 120 volts, 30 min, press run.
Bubbles should form and you should wait.
PCR Positive
1.25 µL Fwd Primer (diluted 1:10 from stock).
1.25 µL Reverse Primer (diluted 1:10 from stock).
1 µL of DNA (diluted 1:10 from stock).
Fill up to 12.5 with Distilled deionized water.
Add 12.5 Master Mix once all other solutions are ready. This mix should be kept in the freezer and taken out in the blue container to be kept cold.
PCR Negative (control)
1.25 µL Fwd Primer (diluted 1:10 from stock).
1.25 µL Reverse Primer (diluted 1:10 from stock).
1 µL of DNA (diluted 1:10 from stock).
Fill up to 25µL with Distilled deionized water.
DNA Digestion
To create a 30 µL mixture:
Add 25 µL of DNA.
Add 3 µL Cut Smart Buffer.
Add 0.5 µL of each enzyme.
Add 1uL (or enough to create total mixture volume) of dH20.
DNA Ligation
To create a 20 µL mixture:
*Volumes that produces ratios of equimolar amounts of 1:1, 1:2, 1:3 (backbone to insert)*
5 µL of insert.
5 µL of Backbone plasmid.
Ligase Buffer (1 µL per 10 µL reaction for 10X buffer).
0.5-1μL T4 DNA Ligase.
dH20 to fill up to total volume.
Thermo-cycle for 2 hours.
Heat up to 65C for 10 minutes.
DNA Chemical Transformation
50 ul of competent cells with 1ul of DNA.
Keep on ice for 20 minutes.
45 seconds at 42 degrees celsius.
Keep on ice for 2 minutes.
Add 300 uL of LB and grow for an hour.
Plate on selective media.
DNA Electroporation
Grow cells to midlog phase and chill on ice for 20 minutes.
Pellet and resuspend in cold water twice.
Ressupend in 1 mL of chilled water.
Take chilled electroporation cuvette and add 200 ng of DNA with 50 uL of cells.
Set electroporator to 1.8 volts and press pulse twice with cuvette in machine.
Immediately add 1 mL of LB and grow for an hour.
Plate on selective media.
Freezing Cells
30 percent solution of glycerol and water needs to be steril, should be filter sterilized.
Use a saturated solution of cells and mix it with the glycerol solution 1:1 and freeze in -80.
Microscopy
Making 2% agarose gel pads: 1 g agarose + 49 mL water, 5 uL smashed between two slides.
Cells are grown overnight, spun down, and then resuspended in PBS. 1:100 works.
OR cells are grown to mid log from an overnight culture.
Add 2uL on the agarose pad and place cover slip on top.
And View.
CRISPR Gene Editing
The target E. coli strain previously transformed with pX2-Cas9 and pSIM5 was grown to mid-log phase.
Incubate cells at 42˚C for 15 minutes to induce the lambda red system.
Induced cells were made electrocompetent by washing with water then transformed with 200 ng of the gRNA vector.
Grown in LB media supplemented with 0.2% arabinose for 3 hours.
The gRNA vector also contained the corresponding homology cassette to repair the double strand break and introduce the designed mutation.
Light Activated Microcompartment Experiment
Grow transformed culture at 30 degrees C to decrease the chance of trans state azobenzene.
Irradiate at 430nm before growth then grow overnight in LB.
Grow a second culture of cells overnight without irradiation in LB.
Follow microscopy protocol and compare compartment formation.
