<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row splashala"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-12"><UL class="nav nav-pills nav-justified"><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Background"> Background </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Notebook"> Notebook </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Collaborations">  Collaborations</A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Attributions">  Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Basic_Part">  Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Composite_Part">  Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Part_Collection">  Part Collection </A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI></DIV></DIV></DIV><DIV style=" margin-left: 70px; float:left"><TABLE><TBODY><TR><TH colspan="7">May</TH></TR><TR><TD><A href="#0801">1</A></TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD><A href="#0808">8</A></TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD></TR><TR><TD><A href="#0815">15</A></TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD></TR><TR><TD><A href="#0822">22</A></TD><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD></TR><TR><TD><A href="#0829">29</A></TD><TD>30</TD><TD>31</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">June</TH></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD><A href="#0905">5</A></TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD></TR><TR><TD><A href="#0912">12</A></TD><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD></TR><TR><TD><A href="#0919">19</A></TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD></TR><TR><TD><A href="#0919">26</A></TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD><TD>31</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">July</TH></TR><TR><TD>1</TD><TD>2</TD></TR><TR><TD><A href="#1003">3</A></TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD></TR><TR><TD><A href="#1010">10</A></TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD></TR><TR><TD><A href="#1017">17</A></TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD></TR><TR><TD><A href="#1024">24</A></TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD></TR><TR><TD><A href="#1031">31</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">August</TH></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD></TR><TR><TD><A href="#1107">7</A></TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD></TR><TR><TD><A href="#1114">14</A></TD><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD></TR><TR><TD><A href="#1121">21</A></TD><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD></TR><TR><TD><A href="#1128">28</A></TD><TD>29</TD><TD>30</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">September</TH></TR><TR><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TD><A href="#1204">4</A></TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD></TR><TR><TD><A href="#1211">11</A></TD><TD>12</TD><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD><TD>17</TD></TR><TR><TD><A href="#1218">18 </A></TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD><TD>24</TD></TR><TR><TD><A href="#1225">25</A></TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">October</TH></TR><TR><TD>1</TD></TR><TR><TD><A href="#1302">2</A></TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD></TR><TR><TD><A href="#1309">9</A></TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD><TD>15</TD></TR><TR><TD><A href="#1316">16</A></TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD></TR><TR><TD><A href="#1323">23</A></TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD></TR><TR><TD><A href="#1330">30</A></TD><TD>31</TD></TR></TBODY></TABLE></DIV><B><H3 id="top"> Notebook </H3><H2> Week 1: 5/8/16-5/14/16</H2><P class="main">For the first week, the team participated in “iGEM boot camp” in which we quickly went through the procedures and equipment that we would be using throughout the summer. This entailed the procedures for PCR amplification, gel electrophoresis, cloning, and DNA sequencing. We extracted the Eut SMNLK via an agar stab provided from the Schmidt- Dannert lab and PCR amplified the Eut S gene using the biobrick primers and cloned on the high, low, and lac promoters onto the 1C3 backbone prior to the ligation of the Eut S gene onto these modified backbones. The 1C3 backbone was taken from the 2016 iGEM registry. Finally, we ligated the Eut S- high, low, and lac onto PSB1C3, the bio-brick standard backbone. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 2: 5/15/16-5/21/16</H2><P class="main"> Performed the ligation of the Eut- SMNLK, provided by the Schmidt- Dannert lab, onto the 6A1 backbone. We transformed the eGFP: Eut C low: 1C3 and eGFP: Eut C lac: 1C3 and Eut SMNLK: 6A1 and sequenced confirmed Eut S low: 1C3 and the eGFP high: 1C3.
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 3: 5/22/16-5/28/16 </H2><P class="main"> Colonies successfully grew of the transformed Eut S low and the Eut S high constructs on the 6A1 backbone. We PCR amplified the Eut SMNLK as well as the Eut S lac and ran overnight ligations of the Eut SMNLK, Eut S lac, Eut S hi, and Eut S lo all onto 6A1. The Eut S lac insert continued to be problematic to clone onto the 6A1 backbone because the insert and the backbone are almost identical in size. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 4: 5/29/16-6/4/16 </H2><P class="main"> Performed the cotransfoms of the eGFP Eut C low &amp; Eut S low: 6A1, eGFP Eut C low &amp; Eut S high:6A1, eGFP Eut C high &amp; Eut S low: 6A1, eGFP Eut C high &amp; Eut S high: 6A1. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 5: 6/5/16-6/11/16 </H2><P class="main"> Used the k808000 (pBad modified with an RBS that includes a GFP protein) from the iGEM registry and sequence confirmed our modification to the protein. We continued to try and get Eut S lac onto 6A1 by repeating the ligation of this. We used Electron Microscopy to image all combinations of eGFP and Eut S as well as the inducible pBad with various level of arabinose. We next began to move all fluorescent proteins over to either a Tetracyclin or Kanamyicin backbone because these are compatible with the non-canonical plasmid that we will be using in subsequent weeks. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 6: 6/12/16-6/18/16 </H2><P class="main"> Sequenced confirmed the pBads eGFPs. EM imaged the co-transforms but with a mid-log growth phase of the cultures. Checked our 5 alpha NEB cells for any phages, which we did not find, and did a genomic PCR amplification of the Eut S. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 7: 6/19/16-6/25/16 </H2><P class="main"> Ligation of the Neptune onto high and low cassettes and imaged these using electron microscopy. The nep high showed inclusion bodies but the nep low resulted in good images so we decided to use the nep low construct for all further cotransforms to be imaged.
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 8: 6/26/16-7/2/16 </H2><FIGURE style="float:right"><FIGCAPTION style="padding: 10px 10px 10px; margin-top:460px; margin-right:60px;  width: 24em">Gel.2 - Lane 2: bad cut with E Lane 3: Nep-Lo 1k3 Lane 4: Nep-Lo 1k3 Lane 5: Ladder Lane 6: eGFP-Hi 1k3 Lane 7: eGFP-Hi 1k3</FIGCAPTION></FIGURE><P class="main"> Performed cotransforms of the Eut S high and low both with nep low but failed to see any turbidity in overnight cultures. </P><P class="main" style="margin-bottom: 550px"><A href="#top">Go to top</A></P><H2> Week 9: 7/3/16-7/9/16 </H2><P class="main"> Repeated cotransforms of the Eut S high nep low, Eut S low nep lo, Eut S lac nep low, pBad eGPF Eut S lac. Plates grew but overnights continued to fail and no turbidity was seen after days of growth. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 10 7/3/16-7/9/16</H2><P class="main"> Realized that the 6A1 and the 1K3 backbone have the same origin of replication which explained why the cotransforms failed continuously. To remedy this, we will move all inserts from the 6A1 backbone onto the 4A5 backbone that is in the iGEM registry and repeat all cotransforms. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 11: 7/10/16-7/16/16</H2><P class="main"> We confirmed successful sequencing data from the Eut S low construct and imaged the cotransforms of this with Neptune low, pBad eGFP induced with 0.5% arabinose, and pBad eGFP with no induction. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 12: 7/17/16-7/23/16 </H2><FIGURE style="float:right"><FIGCAPTION style="padding: 10px 10px 10px; margin-top:460px; margin-right:60px;  width: 24em">Gel.3 - Lane 2: eGFP-Hi 4A5 Lane 3: eGFP-Hi 4A5 Lane 4: Ladder Lane 5: eGFP-Hi 4A5 Lane 6: eGFP-Hi 4A5. Lane 3 and Lane 5 look good</FIGCAPTION></FIGURE><P class="main"> Made ethanolamine plates to image the endogenous Eut S compartments, but no growth was found initially. We will try these next with the addition of thiamine (vitamin B1) to promote growth of the DH5 alpha NEB cells in the media. </P><P class="main" style="margin-bottom: 500px"><A href="#top">Go to top</A></P><H2> Week 13: 7/24/16-7/30/16 </H2><FIGURE style="float:right"><FIGCAPTION style="padding: 10px 10px 10px; margin-top:460px; margin-right:60px;  width: 24em">Gel.4 - Lane 1: Ladder Lane 2: EutS- Hi 6A1 Lane 3: EutS-Hi 4A5 Lane 4: EutS-Hi 4A5 Lane 5: Ladder Lane 6: eGFP-Hi Lane 7: eGFP-Hi. eGFP is good</FIGCAPTION></FIGURE><P class="main"> The Eut S lac: 4A5 construct has continued to show insert ligating to the 6A1 backbone on gels as well as not sequencing correctly. We redid the entire construct by repeating the gel insert of the 4A5 backbone as well as the ligation of the Eut S lac insert. Due to the previous failure of the insert onto the 6A1 backbone, we once again employed the piecewise method of cloning the Eut S insert onto the 4A5 followed by a separate cloning of the lac insert onto the backbone. The final construct appeared successful on a gel and was sent in for sequencing. 
</P><P class="main" style="margin-bottom: 450px"><A href="#top">Go to top</A></P><H2> Week 14: 7/31/16-8/6/16 </H2><FIGURE style="float:right"><FIGCAPTION style="padding: 10px 10px 10px; margin-top:460px; margin-right:60px;  width: 24em">Gel.5 - Lane 1: Ladder Lane 2: EutS-Hi Lane 3: EutS-Hi Lane 4: EutS-Hi Lane 5: Ladder Lane 6: EutS-Lac 6A1 Lane 7: EutS-Lac 4A5 Lane 8: EutS-Lac 4A5</FIGCAPTION></FIGURE><P class="main">Sequence data of the Eut S high showed that the Eut S low insert was used accidently and data of the Eut S lac showed little to no priming. New sequence verification primers for each construct were ordered to obtain better results. Eut S lac cotransforms with the eGPF hi: 1K3, eGFP low:1k3, and Neptune low: 1K3 were carried out regardless of sequencing data. The microscopy images were inconclusive as to if the induced overnights formed better localization than the non-induced due to a strong background fluorescence. </P><P class="main" style="margin-bottom: 500px"><A href="#top">Go to top</A></P><H2> Week 15: 8/7/16-8/13/16 </H2><P class="main"> Sequenced the Eut S Mid overnights prior to PCR mutagenesis and imaging of this construct. Chose mutations at interfacial positions of the Eut S hexamer to best break apart the compartment without destroying the structure forming compartment entirely, as an intra- mutation would likely do. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 16: 8/14/16-8/20/16 </H2><P class="main"> Performed PCR mutagenesis to change Low promoter cassette to a medium expression level, which came back successful. Took more images via fluorescent microscopy to characterize EutS with medium expression, and the expression level seems perfect. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 17: 8/21/16-8/27/16 </H2><P class="main"> Attempted PCR mutagenesis of 6 loci in EutS protein via QuikChange protocol and Q5 polymerase. After processing &amp; sequencing, one reaction was successful at EutS residue 38. Troubleshooting of mutagenesis protocol, then second attempt. One successful mutant at G26 in second batch.
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 18: 8/28/16-9/3/16 </H2><P class="main"> GENOME TEAM: Put the IPTG inducible promoter into the E. Coli genome using Gibson assembly and transformed into cells. </P><P class="main"> PLASMID TEAM:  More PCR mutagenesis with Q5 polymerase, two rounds total with 6 loci each. No successful mutants in first batch. One successful mutant at A63 in second batch. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 19: 9/4/16-9/10/16 </H2><P class="main">  GENOME TEAM: Cloned the CREATE libraries from muse and used electroporation to incorperate into the cell and sent for sequencing confirmation. 
</P><P class="main"> PLASMID TEAM:  More PCR mutagenesis with Q5 polymerase, 5 loci total. No successful mutants. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 20: 9/11/16-9/17/16</H2><P class="main"> Attempted PCR mutagenesis in a different lab using Phusion polymerase. After processing and sequencing, two mutants proved successful, for five total sequence confirmed mutants. Purchased Phusion polymerase for main lab, as this was the suspected culprit, with primer length and affinity the suspected problem with the other 3 loci. Suspending mutations to focus on characterizing mutated compartments.
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 21: 9/18/16-9/24/16 </H2><P class="main"> Took microscopy data for three mutants (A23, A38, A63) without phenylalanine-4’-azobenzene (AzoPhe). None show localization, which is a good sign! Finished synthesis of AzoPhe for culturing EutS mutants.
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 22: 9/25/16-10/1/16  </H2><P class="main"> Cultured mutants with AzoPhe and took microscopy data. 
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 23: 10/2/16-10/8/16 </H2><P class="main"> Attempted different culture techniques with light-protected culture tubes and pre-irradiated AzoPhe (to encourage cis- conformation), and took more data. Attempted yet more cultures with low temperature incubation &amp; some cultures with pre-heated AzoPhe (to encourage trans- conformation).
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 24: 10/9/16-10/15/16 </H2><P class="main"> GENOME TEAM:Realized the libraries were designed for Samonella cells instead of E. coli cells. Re-cloned the libraries as well as re-cloned in the promoter using Gibson assembly and used electroporation to transform into E. coli cells. Sent in samples for sequencing.
</P><P class="main"> PLASMID TEAM:Took final data for cultures of EutS mutants with AzoPhe. We suspect that two of our mutants (A38, K61) may be wild type, so we are attempting to resequence them by isolating the single plasmid from our frozen mutant cell stock. </P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 25: 10/16/16-10/22/16 </H2><P class="main"> GET THE WIKI DONE!!!
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 26: 10/23/16-10/29/16 </H2><P class="main"> GET  THOSE PRESENTATIONS LOOKING GOOD!!!
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P><H2> Week 27: 10/30/16-11/05/16 </H2><P class="main"> GETTING PUMPED FOR JAMBOREE
</P><P class="main" style="margin-bottom: 300px"><A href="#top">Go to top</A></P></B></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>