<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row splashala"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-12"><UL class="nav nav-pills nav-justified"><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Background"> Background </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Notebook"> Notebook </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Collaborations">  Collaborations</A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Attributions">  Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Basic_Part">  Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Composite_Part">  Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Part_Collection">  Part Collection </A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI></DIV></DIV></DIV><DIV class="column full_size judges-will-not-evaluate"><H3> Project Description </H3><P class="main">Bacterial microcompartments (BMCs) are endogenous platforms ideally suited for synthetic biology, as modular protein structures of relatively simple construction. Of the three known BMCs (carboxysomes, PDUs, and Euts), Euts were chosen as candidates for photo-mechanization due to their comparatively straightforward assembly. While carboxysomes and PDUs require precise ratios of coexpressed protein subunits to assemble, the ethanolamine utilizing microcompartments’ shell can form in vivo from a single subunit: EutS. Our research focused on the incorporation of azophenylalanine-sidechain noncanonical amino acids into the EutS protein, which was hypothesized to confer the nanocages with a photo-switchable function for assembly and disassembly. </P><FIGURE align="center"><FIGCAPTION align="center">EutS Functionality</FIGCAPTION><P class="main">Photo-mechanized protein nanocages could serve as adaptable tools for a broad spectrum of synthetic biologists and engineers; applications include compartmentally isolated biocatalyses, and targeted cargo transport and delivery (e.g. precise drug delivery). As such, our research has emphasized general optimization of a light-induced nanocage, although avenues to specific applications have been partially paved in the process. </P><P class="main">Protein nanocages like the Eut are attractive subjects for engineered modifications, since their shells’ structural uniformity reduces the influence of unknown variables on our understanding of the BMC system. Although bacterially-encoded Eut complexes contain several shell protein constituent subunits (including EutS, M, N, L, &amp; K), the EutS component is sufficient for complete shell formation in vivo. EutS homohexamerizes, then further complexes with additional homohexamers to form polyhedral nanocages with predictable subunit interfaces and vertices. Rosetta and PyMol were employed in analyzing the thermodynamically favorable interfacing of subunits, and identifying corresponding residues at which azophenylalanine substitutions wouldn’t disrupt nanocage assembly.</P><FIGURE align="center"><FIGCAPTION align="center">The Big Idea</FIGCAPTION><P class="main">Several candidate residue sites were tested (i.e. AA-encoding codon replaced with amber stop codon; Schultz lab tRNA incorporated azo-phenylalanine-moiety amino acids at these sites), and some cages were observed to form uninterrupted with the incorporated azo-phenylalanine AAs. While in situ assembly/disassembly has yet to be accomplished, our observations involving pre-irradiated azo-phenylalanine indicate that cis-trans isomerization of the noncanonical amino acid does incur subunit conformational changes which overtly affect nanocage assembly.</P><H3> References </H3><P class="main">
Tsoy, Olga, Dmitry Ravcheev, and Arcady Mushegian. &quot;Comparative Genomics of Ethanolamine Utilization.&quot; <I>Journal of Bacteriology </I>191.23 (2009): 7157-164. <I>American Society for Microbiology. Web. 03 July 2016. </I></P><P class="main">
Choudhary, Swati, Maureen B. Quin, Mark A. Sanders, Ethan T. Johnson, and Claudia Schmidt-Dannert. &quot;Engineered Protein Nano-Compartments for Targeted Enzyme Localization.&quot; <I> PLoS ONE </I> 7.3 (2012): 1-11. Web. </P><P class="main">
Held, Mark, Alexander Kolb, Sarah Perdue, Szu-Yi Hsu, Sarah E. Bloch, Maureen B. Quin, and Claudia Schmidt-Dannert. &quot;Engineering Formation of Multiple Recombinant Eut Protein Nanocompartments in E. Coli.&quot; <I>Nature Sci. Rep. Scientific Reports </I>6 (2016): 1-15. Web. </P><P class="main">
Renfrew, P. Douglas, Eun Jung Choi, Richard Bonneau, and Brian Kuhlman. &quot;Incorporation of Noncanonical Amino Acids into Rosetta and Use in Computational Protein-Peptide Interface Design.&quot; <I> PLoS ONE </I>7.3 (2012): 1-15. Web. </P><P class="main">
Bonacci, Walter, Poh K. Teng, Bruno Afonso, Henrike Niederholtmeyer, Patricia Grob, Pamela A. Silver, and David F. Savage. &quot;Modularity of a Carbon-fixing Protein Organelle.&quot; <I> Proceedings of the National Academy of Sciences </I>109.2 (2011): 478-83. Web.
 </P><P class="main">
Smith, Colin A., and Tanja Kortemme. &quot;Predicting the Tolerated Sequences for Proteins and Protein Interfaces Using RosettaBackrub Flexible Backbone Design.&quot; <I> PLoS ONE </I>6.7 (2011): 1-11. Web. </P><P class="main">
Hoersch, Daniel, Soung-Hun Roh, Wah Chiu, and Tanja Kortemme. &quot;Reprogramming an ATP-driven Protein Machine into a Light-gated Nanocage.&quot; <I> Nature Nanotech Nature Nanotechnology </I>8.12 (2013): 928-32. Web.
 </P><P class="main">
Kofoid, Eric, Chad Rappleye, Igor Stojiljkovik, and John Roth. &quot;The 17-Gene Ethanolamine (eut) Operon of Salmonella Typhimurium Encodes Five Homologues of Carboxysome Shell Proteins.&quot; <I> Journal of Bacteriology </I> 181.17 (1999): 5317-329. Web.</P><P class="main"><I>BioBrick® Assembly Kit </I> | NEB. Digital image. <I> New England Biolabs. </I> N.p., n.d. Web. 3 July 2016. www.neb.com. </P></FIGURE></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>