<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row splashala"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-12"><UL class="nav nav-pills nav-justified"><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Background"> Background </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Notebook"> Notebook </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Collaborations">  Collaborations</A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Attributions">  Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Basic_Part">  Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Composite_Part">  Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:CU-Boulder/Part_Collection">  Part Collection </A></LI><LI role="presentation"><A href="https://2016.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI></DIV></DIV></DIV><H3> Results </H3><P class="main">
We set out at the beginning of the summer with the aim of consistently producing a bacterial microcompartment from a single protein, with the objective of adding a non-canonical amino acid at a point within its sequence that enables us to break these self-forming compartments apart using light. We have produced the following results: </P><P class="main">1. We have successfully produced a one-protein BioBrick compatible part that forms microcompartments (EutS, BBa_K2129001)</P><P class="main">2. We have tested and demonstrated the results of different expression levels of our EutS compartment with variable levels of tagged eGFP using a variable two-plasmid system and a variety of promoters (BBa_K2129003-K2129007)</P><P class="main">3. We successfully mutated amber stop codons at at least three loci in the EutS gene</P><P class="main">4. We introduced a 3-plasmid system using the AzoPhe pEVOL plasmid and then visualized the results of adding irradiated phenylalanine-4’-azobenzene to the medium of cells with the mutated plasmids</P><P class="main">5. We attempted to produce similar results using genome modification of NEB-5alpha e.coli cells, and produced a library of genomic edits towards this end.</P><H3> Construction of the EutS BioBrick compatible part </H3><P class="main">
Using the part EutSMNLK (BBa_K311004), we PCR amplified out the S protein of the operon and cloned it alone onto a plasmid backbone. Testing using our two-plasmid system has demonstrated that EutS does indeed produce localization of EutC tagged fluorescent proteins, as described in Schmidt-Dannert et al. 2016 (PMID: 27063436). Our system is fully BioBrick compatible and extremely simple, producing localization and compartmentalization with a minimum of necessary protein expression. </P><P class="main"><FIGURE align="center"><FIGCAPTION align="center">EutS eGFP Compartmentalization</FIGCAPTION></FIGURE></P><H3> Two-Plasmid System localizes fluorescent proteins </H3><FIGURE align="center"><FIGCAPTION align="center">Two-Plasmid System</FIGCAPTION><P class="main">
We used standard BioBrick plasmids pSB4A5 and and introduced high (BBa_314100), low (BBa_314100), and lactose (BBa_K314103) promoters with the EutS compartmental structure protein (BBa_2129001), along with a EutC1-19 tagged enhanced GFP fluorescent protein (BBa_2129003) on pSB1K3 with the same high and low promoters as well as an arabinose inducible promoter (BBa_2129006, 007). Comparative localization was demonstrated by comparison to fluorescent microscopy of non-EutS expressing cells. </P><P class="main">
Fluorescent microscopy demonstrates increased localization of fluorescent proteins when the EutS plasmid is present. We have demonstrated and characterized this using three different EutS promoter systems and found that of the three, EutS Mid and Low (BBa_2129004,005) produce the best results. Some characterization was attempted using an IPTG-inducible cassette, however due to the medium no particular difference in expression was observed in non-IPTG media. </P><P class="main">
Localization of the EutC1-19 - eGFP system was tested with low, high, and arabinose cassettes on pSB1K3. Localization was always noticeable with good levels of EutS expression, however, cells would frequently overexpress eGFP resulting in sporadic cells being extremely fluorescent and making visualization of surrounding cells more difficult. The best results can be seen with EutS with a low cassette (BBa_2129006) and araC-Pbad (BBa_K808000)under non-arabinose added medium using Luria-Bertani broth (this is documented behavior on the parts page for BBa_K808000). </P><FIGURE align="center"><FIGCAPTION align="center">Two-Plasmid System localizes fluorescent proteins</FIGCAPTION><P class="main">
These results demonstrate that the standard BioBrick promoters and plasmids are able to produce similar results to those previously discovered, with some possibility of reduced fidelity and frequency of ‘misbehavior’ by individual cells. </P><H3> Mutation of EutS Sequence to introduce Amber stop codons </H3><P class="main">
Using site directed mutagenesis, we changed the sequence of our EutS Low plasmid construct to instead use a Mid promoter, and we introduced amber stop codons into the sequence of EutS at residues that we selected based on predictive protein modelling with Rosetta and Pymol. Sequencing confirmed that several of these plasmids were successfully mutated at the loci we selected, including the part EutS Mutant C (BBa_2129002) which we characterized more fully with our three plasmid system and fluorescent microscopy. </P><P class="main"><FIGURE align="center"><FIGCAPTION align="center">Amber EutS Mutant C</FIGCAPTION></FIGURE></P><H3> Introduction of azo-phenylalanine amino acids to EutS structure </H3><P class="main">
Using QuikChange PCR mutagenesis, we introduced Amber (UAG) stop codons to the EutS sequence at residues predictively modelled to produce super and substructure altering effects. Once sequenced and confirmed for incorporation of amber codons at desired sites,  we performed a co-transformation of eGFP Low (BBa_2129006) and the pEVOL AzoPhe plasmid provided by the Shultz lab, and then introduced our mutated EutS Mid plasmids via electroporation. 
 </P><FIGURE align="center"><FIGCAPTION align="center">3 plasmid system</FIGCAPTION><P class="main">
Fluorescent microscopy of the tri-plasmid system without mutation produced similar results to the two plasmid system with successful growth in tri-antibiotic media, confirming the viability of the three-plasmid protocol. Amber codon mutation into the sequence of the EutS gene was sequence confirmed at five loci, all of which were transformed with the prepared two-plasmid e.coli and viewed under fluorescent microscopy. With reduced efficiency from the control, eGFP localization was still notable in the mutant-plasmid transformed cells, suggesting successful incorporation of phenylalanine-4’-azobenzene (AzoPhe) into the EutS structure. Under repeated exposure to blue-wavelength light, localization showed significant diminishment, raising the possibility of successful cage disruption under trans-AzoPhe conditions. 
 </P><FIGURE align="center"><FIGCAPTION align="center">EutS Mutants with and without AzoPhe</FIGCAPTION><H3> Genome modification of NEB 5(alpha) cells for IPTG-inducible Eut expression </H3><P class="main">
Genome sequencing of the NEB-5alpha cells used for all our cloning was performed to confirm the presence of a functional e.coli Eut operon. Upon confirmation of gene presence and function, we performed high-throughput mutation to attempt to reproduce our two-plasmid expression system with genomic transcription, introducing an IPTG-inducible promoter in lieu of the normal ethanolamine operon induction into the genome. </P><P class="main">
Due to the considerable time-latency this procedure requires, we did not successfully produce functional mutants at the time of the competition, but we did produce a mutation library and transformed our cells.  </P><H3> Future Directions! </H3><P class="main">
The project will continue as we explore different residue locations for amber mutation. We hope to soon be able to produce a compartment capable of consistent breakage and formation with light irradiation. Once we have that compartment, we hope to develop a procedure to isolate it, and maybe in the farther future, explore the possibilities this compartment may offer in manufacturing and drug delivery. We will also continue to attempt to create these compartments on the native EutS compartment  of our NEB-5alpha E.Coli using our mutation library; this would enable easier growth of compartments at scale, as well as prevent the cells from losing the plasmid in media.</P><H3> Considerations for Replication of the Experiment </H3><P class="main">
Several issues occurred during our first attempt at the three plasmid system, which we should like to avoid if we were to repeat the experiment. First of all, the usage of an Ampicillin resistance plasmid may have not been the best choice. As is evidenced by our microscopy, many E.Coli cells would grow far past the length of a healthy E.Coli, which we believe is largely due to the use of Ampicillin. Additionally, several weeks were spent troubleshooting our two-plasmid system before we moved from the pSB6A1 to pSB4A5 backbones. Always double check that your plasmids are compatible!  </P><P class="main">
Another concern arose from our use of eGFP as a fluorescent localization marker. eGFP fluoresces at a wavelength around 500, which is not ideal for visualization of a system where we want to break the cages with a wavelength of ~460. We did attempt to use another fluorescent protein (Neptune, a far red fluorescent protein), but it proved to form aggregates on its own, which rendered it a poor choice for viewing localization. </P><FIGURE align="center"><FIGCAPTION align="center">Fluorescence from Neptune (No EutS Added)</FIGCAPTION><P class="main">
Additionally, we would explore the possibility of incorporating EutS and the fluorescent protein we switch to onto the same plasmid now that we have a good idea of which expression levels are most efficient. A three plasmid system has the potential to cause problems with successful transformation of E.Coli, so switching to a single plasmid expressing both would cut down on potential failures and also reduce the need to use as many antibiotics. </P></FIGURE></FIGURE></FIGURE></FIGURE></FIGURE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>