"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/notebook/c0051debug.html
From 2010.igem.org
Characterization of BBa_C0051, one of the most widely used and studied parts
Bba_C0051 cI-Lamba, the promoter in disguise During the construction of one of our larger pieces of the spatial oscillator, the ON switch, we stumbled upon an unexpected phenotype.
The RBS-RFP in this picture is actually RBS-RFP-F1610, but for simplicity purposes and purposes of explaining what's going on, we have illustrated it as RBS-RFP
The Problem
Our scanned image of our plate with colonies containing RBS-C0051 ligated to RBS-RFP-F1610.  Click above for the full-screen image.
Our intermediate ligation of RBS-C0051 to RBS-RFP-F1610 produced a red phenotype but it has no promoter! Sequencing results available here.
Since Bba_C0051 is a highly studied and widely used part in the registry, it presents a huge problem not only in the context of the spatial oscillator but also in the 76 parts that utilize it throughout the registry.
This exciting new find causes many problems in our design and must cause malfunctions in a huge number of parts entered and used in the registry.
The Investigation
We began by sequencing the DNA of the red colonies from the ligation of RBS-C0051 to RBS-RFP-F1610 in order to rule out the presence of any promoter or mutation. The sequence analysis revealed that we did indeed have the correct sequence.
Further investigation into Bba_C0051 showed that the part not only contained the coding region, cI, but it is also followed by an LVA degradation tag and a barcode. When looking further into the RBS (Bba_E1010), we found that it also contained an LVA degradation tag and a barcode after the RFP coding region.
We then conducted a number of experiments in order to determine which part or combination of parts contained in the construct caused this unwanted promoter-like behavior.
After determining the cause of the unwanted transcription, we continued to conduct a number of experiments in order to measure the results.
A Starting Point
Our scanned image of our plate with colonies containing RBS-RFP-F1610. No abnormalities here! Click to enlarge.
The previous step in the construction of the on switch was RBS-RFP-F1610. Sequencing results available here.
This construct clearly exhibited a white phenotype. No abnormalities here!
Test #1
To determine whether Bba_C0051 played a role in the unexpected transcription, we ligated RBS-RFP to RBS-RFP in order to see if transcription occurred.
The construct exhibited a white phenotype. Thus, we concluded that transcription did not start in any part of the RFP. This helped to narrow our focus down to the possibility of the cI coding region in Bba_C0051 or a possible artifact in the plasmid as the cause for the unwanted transcription.
After the results were in, we diligently sent them out for sequencing so that we would be sure that our results were accurately reflecting the parts that we believed we were analyzing.
Test #2
Our scanned image of our plate with colonies with C0051 into Chris Anderson's promoter screening plasmid.  Click to enlarge.
Next, we sought to determine whether the presence of an artifact in the plasmid might be the cause or contribute to the promoter function.
We ligated RBS-C0051 to Chris Anderson's Promoter Screening Plasmid (Bba_J23006)
By changing the context in which the parts were placed, we were able to removing the possibility of the artifact causing transcription
Even after placing this part into a new context, we obtained red colonies! Therefore, we know that the transcription initiation has something to do with the Bba_C0051 region.
Sequenced? Yes! For sequencing data, click here .
Test #3
Our scanned image of our plate with colonies containing RBS-RFP in Chris Anderson's PSP.  Click above for the full-screen image.
Now, although the presence of an artifact from the original plasmid has been determined to be negligible, it is also important to verify the same for Chris Anderson's Promoter Screening Plasmid (Bba_J23006)
We ligated RBS-RFP into Chris Anderson's Promoter Screening Plasmid (Bba_J23006)
A white phenotype was expressed. Therefore, it is clear that we must focus down on the Bba_C0051 part in order to determine where a promoter function is being expressed.
Once again, of course this part was sequenced!
In Silico Promoter Scanning of C0051 Coding Region
While the cI lambda phage repressor is one of the best studied and most used proteins in synthetic biology we wanted to start our search for potential promoters by asking whether or not we could detect any putative promoters in the coding region of cI by in silico methods.  We tried a number of on-line tools for automated promoter detection but did not come up with any meaningful results.  Next we contacted Drew Endy at Stanford to tap into his knowledgebase of phage biology.  He forwarded this request to Thomas Schneider at the NIH who used the Delia programs to search for sigma70 binding sites.  Surprisingly, a few potential "good" scoring candidates were found.  Armed with this data, we decided to see if we could experimentally confirm one of these sites as a potential transcript start site using the technique called 5'-RACE.  Initial results follow.
Figure: Putative sigma 70 promoters detected by Thomas Schneider using Delia programs.  Click to enlarge!!
5' Race PCR
In order to locate the transcript start site of "mystery C0051" promoter, we conducted a 5'-RACE experiment on three cell lines.  (1) C0051-PSP (C0051 in a promoter screening plasmid - red cells), (2) E1010-PSP (RFP in promoter screening plasmid - white cells) and (3) DH5alpha cells.  Using RFP specific primers, we were unable to generate any cDNA from either constructs 2 or 3 - our negative controls.  We were able to retrieve transcript from the C0051-PSP strain indicating that RFP was indeed transcribed.  When we sequenced the resulting cDNA (see figure below) we noticed that the poly-A tail (added during sample prep for amplification) was located just upstream of the scar between the barcode and the ribosome binding site (Bba_B0034).  Therefore, our initial 5'-RACE experiment indicates that the transcript likely initiates within the bar code region rather than within the cI coding region.  A second sample/clone verified this result
Figure legend: Sequencing chromatogram of 5'-RACE experiment.
As shown, the transcription that we are seeing clearly starts at the beginning of the barcode region
Test #4
Modified C0051 in PSP exhibited white phenotype.  Click above for the full-screen image.
From all the test so far, it is clear that a certain interaction of the cI coding region and the barcode are acting together to produce some kind of promoter function.
In order to experimentally provide data to prove this point, we removed the barcode from the RBS-C0051 construct and placed it into Chris Anderson's Promoter Screening Plasmid (Bba_J23006).
Surely enough, the phenotype was white!
We finally narrowed it down and were able to create a part in which the cI lambda coding region can be utilized without creating an unwanted and detrimental promoter function.
This part was sequenced and submitted! Sequence is available in the zip file below.
The Fix
By removing the barcode and submitting this part to the registry, we have greatly improved a fundamental part by modifying it so that it works as advertised.
As you can imagine, an unwanted constant transcription can really set you back and cause major malfunctions in your system.
With this necessary fix implemented, you can now feel confident that this fundamental part will work just as you intend!
For a complete archive of all relevant DNA sequences to these experiments (in .ab1 format), CLICK TO DOWNLOAD HERE !
RESULTS AND CONCLUSIONS
We have clearly demonstrated that the bar code that was attached to C0051 and other parts in the registry (as a marker that might help determine the origin of the part) contributes to the "invention" of promoter-like activity in the context of the cI lambda phage repressor coding sequence and LVA proteolytic degradation tag.
It is critical to recognize that this function only seems to arise (at least for now) in the context of the cI coding region and not from the bar code alone - as seen in the E1010 experiment where the presence of the LVA tag and the bar code in the context of RFP do not create a promoter function.  We would like to eventually try other coding regions upstream.
The 5'-RACE experiment seems to suggest that the transcript start site is within the bar code.  This is further supported by the experiment in which the bar code is removed and the cells no longer produce RFP.  We are also repeating this experiment several more times to verify the result.
The combination of results 2 and 3 suggest that there is some functional coordination between sequence in cI and the bar code that is ultimately responsible for promoter activity.  We are testing this hypothesis by generating various truncations of cI that retain the bar code and LVA tag.  This should allow us to "zoom in" on the sequence in Bba_C0051 that is responsible for this phenomenon. We expect to have this data for our presentation - please see our talk.
In addition to data from the previous point, we also intend to have some measure of relative promoter strength in time to present.  Again, please see our talk.
While we don't completely understand the mechanism of this promoter, we have made great strides to characterize this critical registry part and contributed a "corrected" promoter-free version of the part in the registry ( BBa_K327018 ).  Given the broad use of this part we expect this to have a relatively large impact.
Finally, a critical extension of this result is the conclusion that we must be prepared to accept the idea that the assembly of even well-standardized parts may be subject to unintended consequences that arise from the nature of the substrate we're working with (DNA).  That is, while we might understand the isolated function of each part, specific combination of these parts may invent novel function that were unexpected - the assembly of a device may lead to functions that are more than the sum of its parts.
More to Come!
We are currently working to clearly pinpoint the place in the cI-lambda coding region that interacts with the barcode and caused transcription.
In order to achieve this, we have designed primers, which are used to carefully truncate Bba_C0051. After these truncations are made, we will observe their phenotype in order to determine our results.
Also, we are in the process of characterizing and measuring the relative strength of the promoter present in the system containing a barcode.
We look forward to sharing the results of these quantitative tests at the iGEM Jamboree ‘10.
The Problem
The Investigation
A Starting Point
Test #1
Test #2
Test #3
In Silico Promoter Scanning
5' Race PCR
Test #4
The Fix
Results and Conclusions
More to Come
We would like to take a moment to thank all of our sponsors for their very generous donations, as we could not have done this without your help!
We would also like to thank and acknowledge:
Our Advisors
Marc Facciotti
Ilias Tagkopoulos
Technical Guidance
David Larsen
Andrew Yao
Visiting iGEMer
Jia Li of Zhejiang University (TEAM ZJU-China)
cI Promoter Screen
Drew Endy - Stanford
Thomas Schneider - NIH
Want to sponsor us?  Send an email to mtfacciotti@ucdavis.edu to discuss various ways you can help! :)
Retrieved from " http://2010.igem.org/Team:UC_Davis/notebook/c0051debug.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
