<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Constructs_design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Constructs design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col-md-8"><UL class=" text-left wtf-menu "><LI class="parent main-item"><A href="#">Project  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Production">Production</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Active_Testing">Testing </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Active_Testing">Active Testing</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Biocompatibility_Testing">Biocompatibility Testing</A></LI></UL><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Optimization">Optimization</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Public_Engagement">Public Engagement</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Software">Software  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMPDesigner_Overview">AMP Designer Overview</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMP_Forest">AMP Forest</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMP_Evolver">AMP Evolver</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Result">Result</A></LI><LI class="parent main-item"><A href="#">Judging </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Medal_Criteria">Medal Criteria </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Results">Key Results</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/BioBricks">BioBricks</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/InterLab">InterLab</A></LI><LI class="parent main-item"><A href="#">Team  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Lab_Notebook">Lab Notebook</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="logolist2"><DIV class="sidelogos top5"><DIV id="twitapp"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV class="textbody h1"><H1>Constructs design</H1></DIV><DIV class="textbody"><P>In 2016, Lam et al., described SNAPPS (Structurally Nanoengineered Antimicrobial Peptide Polymers). SNAPPs are a unique class of star-shaped antimicrobial peptides with activity against a range of Gram-negative pathogens.</P></DIV><DIV class="textbody table tr td"><TABLE boarder="5" cellpassing="0" cellpadding="12" width="300"><TBODY><TR><TH allign="center">Box 1: What are SNAPPs?</TH></TR><TR height="100"><TD aligne="centre">SNAPPS are built from short chains of lysine and valine, 10-30 amino acids long. These peptide arms are conjugated at one end to a core of poly(amido amine). Each core may be linked to 16-32 separate arms. The result is positively charged protein star, about 20 nm across.</TD></TR></TBODY></TABLE></DIV><DIV class="textbody"><P>The goal of this project is to replace chemically synthesized SNAPPs with fully synthetic biological fusion proteins. We reasoned that the key features of SNAPP geometry could be reproduced by fusing short, positively charged AMPs to the N- and C- termini of highly multimeric proteins. In this way, the naturally multimeric protein subunits form the core of the structure and the fused AMPs extend outward to form the arms of the star-shape. We called the resulting proteins StarCores.</P></DIV><DIV class="textbody h2"><H2>Architectural Principles of Star Cores</H2></DIV><DIV class="textbody"><P> Following Shirbin (2018), we identified some key features needed for effective StarCores.</P><P><B>1) Arm number between 4-32.</B> If each monomer of protein is fused to an AMP at both ends, then we want to use proteins that form 2- to 16-mer homomultimeric complexes.</P><P><B>2) Star size smaller than 30 nm.</B> SNAPPs are most effective when they are dense, presenting many AMPs to the membrane.</P><P><B>3) Strong Positive Charge.</B> Bacterial membranes are negatively charged and the SNAPP interaction and pore formation is charge-mediated.</P></DIV><DIV class="textbody h2"><H2>Combinatorial Design of StarCores</H2></DIV><DIV class="textbody"><P>Protein engineering is challenging. A good synthetic protein needs to express efficiently, fold tightly, avoid self-aggregation and then function with high activity. We knew it was unlikely that we would design a perfect StarCore on the first try. Therefore, we decided to design many StarCores and screen them systematically.</P><P>First, we selected 15 known AMPs to function as the arms of the StarCore. The purpose of these AMPs is to functionally substitute for the lysine-valine chains in the original SNAPP design. Therefore, we chose AMPs with a high density of positive charges, reasoning that they would interact with bacterial membranes by a similar mechanism. Our second criterion for AMP selection was simply diversity. We chose natural AMPs from many organisms as well as synthetic AMPs.</P></DIV><DIV class="textbody table tr td"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>Peptide Name</TH><TH>Class</TH><TH>Source</TH><TH>Activity</TH></TR></THEAD><TBODY><TR><TD>Cg-Def</TD><TD>Defensin</TD><TD>Crassostrea gigas (Oyster)</TD><TD>Antibacterial, Antifungal</TD></TR><TR><TD>Alyteserin-2a</TD><TD>Alyteserin</TD><TD>Alytes obstetricans (Toad)</TD><TD>Antibacterial, Cytotoxicity</TD></TR><TR><TD>Pardaxin P-1</TD><TD>Pardaxin</TD><TD>Pardachirus pavoninus (Fish)</TD><TD>Cytotoxic</TD></TR><TR><TD>ɑ-1-purothionin</TD><TD>Thionin</TD><TD>Triticum aestivum (Wheat)</TD><TD>Cytotoxic</TD></TR><TR><TD>Arenicin-1</TD><TD>Arenicin</TD><TD>Arenicola marina (Lugworm)</TD><TD>Antibacterial, Antifungal, Cytotoxic</TD></TR><TR><TD>Ovispirin-1</TD><TD>Sheep-derived</TD><TD>Synthetic</TD><TD>Antibacterial, Cytotoxic</TD></TR><TR><TD>V6 peptide</TD><TD>Cyclic cationic</TD><TD>Synthetic</TD><TD>Antibacterial</TD></TR><TR><TD>PolyVK-11</TD><TD>Poly-VK</TD><TD>Synthetic</TD><TD>Antibacterial</TD></TR><TR><TD>PolyVK-12</TD><TD>Poly-VK</TD><TD>Synthetic</TD><TD>Antibacterial</TD></TR><TR><TD>PolyVK-21</TD><TD>Poly-VK</TD><TD>Synthetic</TD><TD>Antibacterial</TD></TR><TR><TD>Bactofencin A</TD><TD>IId</TD><TD>Lactobacillus</TD><TD>Antibacterial</TD></TR><TR><TD>Aureocin A5</TD><TD>IId</TD><TD>Staphylococcus</TD><TD>Antibacterial</TD></TR><TR><TD>Bacteriocin E50-52</TD><TD>IIa</TD><TD>Enterococcus</TD><TD>Antibacterial</TD></TR><TR><TD>Enterocin A</TD><TD>IIa</TD><TD>Enterococcus</TD><TD>Antibacterial</TD></TR><TR><TD>Pediocin PA-1</TD><TD>IIa</TD><TD>Pediococcus</TD><TD>Antibacterial</TD></TR></TBODY></TABLE></DIV><DIV class="textbody"><P> Next, we selected 14 proteins to use as the core of the StarCore. There are many potential multimeric proteins found in nature. We filtered them by a variety of criteria, favoring convenience and diversity (Box 2).</P><P> For each core protein, we used the PDB structure to determine if the N- or C-termini, or both, were free to accept protein fusions. We only constructed fusion proteins using ends that were determined to be free.</P></DIV><DIV class="textbody table tr td"><TABLE boarder="5" cellpassing="0" cellpadding="12" width="500"><TBODY><TR><TH allign="center">Box 2: Criteria for Selecting Core Proteins</TH></TR><TR height="100"><TD>- The monomer must be expressible in <I>E. coli</I>:<LI>Previous literature showing expression,</LI><LI>No post-translational modifications,</LI><LI>Bacterial origin preferred.</LI>- The monomer must self-assemble into a homomultimeric complex.- The compex must have a size between 4-25 nm.- The complex must have known structure deposited in the PDB.- The N- and/or C- termini of the monomers should be free.- The set of complexes should be of diverse shapes.</TD></TR></TBODY></TABLE></DIV><DIV class="textbody"><P>Following these criteria, we selected the following proteins from RCSB Protein data bank.</P></DIV><DIV class="textbody table tr td"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>NAME</TH><TH>Ferritin</TH><TH>MS2</TH><TH>Qbeta</TH><TH>Small  HSP</TH></TR></THEAD><TBODY><TR><TD>PDB reference</TD><TD>4XGS</TD><TD>7msf</TD><TD>5vlz</TD><TD>1SHS</TD></TR><TR><TD>Antimicrobial peptide fused to</TD><TD>N-terminal</TD><TD>N-terminal 
C-terminal</TD><TD>C-terminal</TD><TD>C-terminal</TD></TR><TR><TD>Image</TD><TD>4XGS</TD><TD>7MSF</TD><TD>5vlz</TD><TD>1SHS</TD></TR></TBODY></TABLE></DIV><DIV class="textbody table tr td"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>NAME</TH><TH>ConcavalinA</TH><TH>Pyruvate dehydrogenase</TH><TH>Designed protein lyase cage</TH><TH>Isoaspartyl Dipeptidase</TH></TR></THEAD><TBODY><TR><TD>PDB reference</TD><TD>2g4i</TD><TD>1EAA</TD><TD>4QCC</TD><TD>5LP3</TD></TR><TR><TD>Antimicrobial peptide fused to</TD><TD>N-terminal
C-terminal</TD><TD>N-terminal</TD><TD>N-terminal</TD><TD>N-terminal</TD></TR><TR><TD>Image</TD><TD>2g4i</TD><TD>1EAA</TD><TD>4QCC</TD><TD>5LP3</TD></TR></TBODY></TABLE></DIV><DIV class="textbody table tr td"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>NAME</TH><TH>GTP_Cyclo-hydrolase</TH><TH>FucU isomerase</TH><TH>Pizza_3</TH><TH>Holo- synthase</TH></TR></THEAD><TBODY><TR><TD>PDB reference</TD><TD>1GTP</TD><TD>2WCV</TD><TD>3WW8</TD><TD>5xu7</TD></TR><TR><TD>Antimicrobial peptide fused to</TD><TD>N-terminal</TD><TD>C-terminal</TD><TD>N-terminal 
C-terminal</TD><TD>N-terminal 
C-terminal</TD></TR><TR><TD>Image</TD><TD>1GTP</TD><TD>2WCV</TD><TD>3WW8</TD><TD>5xu7</TD></TR></TBODY></TABLE></DIV><DIV class="textbody table tr td"><TABLE class="table table-striped table-bordered"><THEAD><TR><TH>NAME</TH><TH>Pizza_2</TH><TH>RADA self-assembling peptide</TH></TR></THEAD><TBODY><TR><TD>PDB reference</TD><TD>3WW7</TD><TD>–</TD></TR><TR><TD>Antimicrobial peptide fused to</TD><TD>N-terminal
 C-terminal</TD><TD>N-terminal
 C-terminal</TD></TR><TR><TD>Image</TD><TD>3WW7</TD><TD>–</TD></TR></TBODY></TABLE></DIV><DIV class="textbody"><P>In total, we designed 210 StarCore proteins as the combinatorial fusion of 15 AMPs to 14 cores.</P></DIV><DIV class="textbody h1"><H1>Results</H1></DIV><DIV class="textbody h2"><H2>Construction of the StarCore Fusion Proteins</H2></DIV><DIV class="textbody"><P>StarCore proteins were constructed as translational fusions using GoldenGate assembly.</P></DIV><DIV class="textbody h1"><H1>Discussion</H1></DIV><DIV class="textbody"><P>Here will be your text.</P></DIV><DIV class="textbody h1"><H1>Methods</H1></DIV><DIV class="textbody h2"><H2>T7 Expression Vector</H2></DIV><DIV class="textbody"><P>StarCores were cloned into the pACYCDuet-1 Vector (Novagen). The cloning strategy made use of the manufacturer-suggested start codon, downstream of a T7 promoter. The final construct omitted protein purification tags included in the original vector.</P><P>For cloning, the plasmid was first linearized by digestion with AvrII and NcoI. Then we performed PCR on the linearized vector with Golden Gate cloning sites added as primer tags.</P></DIV><DIV class="textbody h2"><H2>DNA Synthesis</H2></DIV><DIV class="textbody"><P>Coding sequences for AMPs and core proteins were generously synthesized by team sponsor IDT (Coralville, USA). Codon usage was optimized by the supplier for expression in E. coli. Sequences were free of BioBrick restriction enzyme cut sites, as well as common type IIS restriction sites.</P><P>Golden Gate cloning sites were added to the ends of each coding sequence by PCR with tagged primers.</P></DIV><DIV class="textbody h2"><H2>Golden Gate Cloning</H2></DIV><DIV class="textbody"><P>We created translational fusions of the AMP peptides to the core proteins using standard Golden Gate cloning. Golden Gate Assembly Mix was supplied by NEB (#E1600S). Cloning reactions made use of the the BsaI enzyme and 4 custom cloning overhangs.</P><LI>GG1: 5′-GGTCTCNCATG-3′</LI><LI>GG2: 5′-GGTCTCNATCA-3′</LI><LI>GG3: 5′-GGTCTCNTTCG-3′</LI><LI>GG4: 5′-GGTCTCNTGAC-3′</LI><P>The AMP, core and plasmid backbones were assembled in one step.</P><P>In cases where the N-terminus of the core protein could not be tagged, the core sequence was amplified with GG1 and the N-terminal AMP was omitted. The C-terminus was treated similarly. No additional amino acids were added to the coding sequences of core termini deemed unsuitable for tagging.</P></DIV><DIV class="textbody h2"><H2>Transformation and Construct Validation</H2></DIV><DIV class="textbody"><P>Golden Gate asseumbly products were transformed into E. coli DH5a. Because this strain lacks T7 polymerase, it does not express the potentially antimicrobial StarCore peptides.</P><P>Successful clones were verified by colony PCR.</P></DIV><DIV id="footerdiv"><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France<A href="mailto:paris-bettencourt-2018@cri-paris.org">paris-bettencourt-2018@cri-paris.org</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>