Team:Paris Bettencourt/Constructs design
Project  
Human Practices
Applied Design
Design
Production
Testing 
Active Testing
Biocompatibility Testing
Modeling
Optimization
Public Engagement
Software  
AMP Designer Overview
AMP Forest
AMP Evolver
Result
Judging 
Medal Criteria
Key Results
BioBricks
InterLab
Team  
Members
Collaborations
Attributions
Lab Notebook
Acknowledgements
@iGEM_Paris
Constructs design
In 2016, Lam et al., described SNAPPS (Structurally Nanoengineered Antimicrobial Peptide Polymers). SNAPPs are a unique class of star-shaped antimicrobial peptides with activity against a range of Gram-negative pathogens.
Box 1: What are SNAPPs?
SNAPPS are built from short chains of lysine and valine, 10-30 amino acids long. These peptide arms are conjugated at one end to a core of poly(amido amine). Each core may be linked to 16-32 separate arms. The result is positively charged protein star, about 20 nm across.
The goal of this project is to replace chemically synthesized SNAPPs with fully synthetic biological fusion proteins. We reasoned that the key features of SNAPP geometry could be reproduced by fusing short, positively charged AMPs to the N- and C- termini of highly multimeric proteins. In this way, the naturally multimeric protein subunits form the core of the structure and the fused AMPs extend outward to form the arms of the star-shape. We called the resulting proteins StarCores.
Architectural Principles of Star Cores
Following Shirbin (2018), we identified some key features needed for effective StarCores.
1) Arm number between 4-32. If each monomer of protein is fused to an AMP at both ends, then we want to use proteins that form 2- to 16-mer homomultimeric complexes.
2) Star size smaller than 30 nm. SNAPPs are most effective when they are dense, presenting many AMPs to the membrane.
3) Strong Positive Charge. Bacterial membranes are negatively charged and the SNAPP interaction and pore formation is charge-mediated.
Combinatorial Design of StarCores
Protein engineering is challenging. A good synthetic protein needs to express efficiently, fold tightly, avoid self-aggregation and then function with high activity. We knew it was unlikely that we would design a perfect StarCore on the first try. Therefore, we decided to design many StarCores and screen them systematically.
First, we selected 15 known AMPs to function as the arms of the StarCore. The purpose of these AMPs is to functionally substitute for the lysine-valine chains in the original SNAPP design. Therefore, we chose AMPs with a high density of positive charges, reasoning that they would interact with bacterial membranes by a similar mechanism. Our second criterion for AMP selection was simply diversity. We chose natural AMPs from many organisms as well as synthetic AMPs.
Peptide Name
Class
Source
Activity
Cg-Def
Defensin
Crassostrea gigas (Oyster)
Antibacterial, Antifungal
Alyteserin-2a
Alyteserin
Alytes obstetricans (Toad)
Antibacterial, Cytotoxicity
Pardaxin P-1
Pardaxin
Pardachirus pavoninus (Fish)
Cytotoxic
ɑ-1-purothionin
Thionin
Triticum aestivum (Wheat)
Cytotoxic
Arenicin-1
Arenicin
Arenicola marina (Lugworm)
Antibacterial, Antifungal, Cytotoxic
Ovispirin-1
Sheep-derived
Synthetic
Antibacterial, Cytotoxic
V6 peptide
Cyclic cationic
Synthetic
Antibacterial
PolyVK-11
Poly-VK
Synthetic
Antibacterial
PolyVK-12
Poly-VK
Synthetic
Antibacterial
PolyVK-21
Poly-VK
Synthetic
Antibacterial
Bactofencin A
IId
Lactobacillus
Antibacterial
Aureocin A5
IId
Staphylococcus
Antibacterial
Bacteriocin E50-52
IIa
Enterococcus
Antibacterial
Enterocin A
IIa
Enterococcus
Antibacterial
Pediocin PA-1
IIa
Pediococcus
Antibacterial
Next, we selected 14 proteins to use as the core of the StarCore. There are many potential multimeric proteins found in nature. We filtered them by a variety of criteria, favoring convenience and diversity (Box 2).
For each core protein, we used the PDB structure to determine if the N- or C-termini, or both, were free to accept protein fusions. We only constructed fusion proteins using ends that were determined to be free.
Box 2: Criteria for Selecting Core Proteins
- The monomer must be expressible in E. coli:
Previous literature showing expression,
No post-translational modifications,
Bacterial origin preferred.
- The monomer must self-assemble into a homomultimeric complex.
- The compex must have a size between 4-25 nm.
- The complex must have known structure deposited in the PDB.
- The N- and/or C- termini of the monomers should be free.
- The set of complexes should be of diverse shapes.
Following these criteria, we selected the following proteins from RCSB Protein data bank.
NAME
Ferritin
MS2
Qbeta
Small  HSP
PDB reference
4XGS
7msf
5vlz
1SHS
Antimicrobial peptide fused to
N-terminal
N-terminal C-terminal
C-terminal
C-terminal
Image
4XGS
7MSF
5vlz
1SHS
NAME
ConcavalinA
Pyruvate dehydrogenase
Designed protein lyase cage
Isoaspartyl Dipeptidase
PDB reference
2g4i
1EAA
4QCC
5LP3
Antimicrobial peptide fused to
N-terminal C-terminal
N-terminal
N-terminal
N-terminal
Image
2g4i
1EAA
4QCC
5LP3
NAME
GTP_Cyclo-hydrolase
FucU isomerase
Pizza_3
Holo- synthase
PDB reference
1GTP
2WCV
3WW8
5xu7
Antimicrobial peptide fused to
N-terminal
C-terminal
N-terminal C-terminal
N-terminal C-terminal
Image
1GTP
2WCV
3WW8
5xu7
NAME
Pizza_2
RADA self-assembling peptide
PDB reference
3WW7
–
Antimicrobial peptide fused to
N-terminal C-terminal
N-terminal C-terminal
Image
3WW7
–
In total, we designed 210 StarCore proteins as the combinatorial fusion of 15 AMPs to 14 cores.
Results
Construction of the StarCore Fusion Proteins
StarCore proteins were constructed as translational fusions using GoldenGate assembly.
Discussion
Here will be your text.
Methods
T7 Expression Vector
StarCores were cloned into the pACYCDuet-1 Vector (Novagen). The cloning strategy made use of the manufacturer-suggested start codon, downstream of a T7 promoter. The final construct omitted protein purification tags included in the original vector.
For cloning, the plasmid was first linearized by digestion with AvrII and NcoI. Then we performed PCR on the linearized vector with Golden Gate cloning sites added as primer tags.
DNA Synthesis
Coding sequences for AMPs and core proteins were generously synthesized by team sponsor IDT (Coralville, USA). Codon usage was optimized by the supplier for expression in E. coli. Sequences were free of BioBrick restriction enzyme cut sites, as well as common type IIS restriction sites.
Golden Gate cloning sites were added to the ends of each coding sequence by PCR with tagged primers.
Golden Gate Cloning
We created translational fusions of the AMP peptides to the core proteins using standard Golden Gate cloning. Golden Gate Assembly Mix was supplied by NEB (#E1600S). Cloning reactions made use of the the BsaI enzyme and 4 custom cloning overhangs.
GG1: 5′-GGTCTCNCATG-3′
GG2: 5′-GGTCTCNATCA-3′
GG3: 5′-GGTCTCNTTCG-3′
GG4: 5′-GGTCTCNTGAC-3′
The AMP, core and plasmid backbones were assembled in one step.
In cases where the N-terminus of the core protein could not be tagged, the core sequence was amplified with GG1 and the N-terminal AMP was omitted. The C-terminus was treated similarly. No additional amino acids were added to the coding sequences of core termini deemed unsuitable for tagging.
Transformation and Construct Validation
Golden Gate asseumbly products were transformed into E. coli DH5a. Because this strain lacks T7 polymerase, it does not express the potentially antimicrobial StarCore peptides.
Successful clones were verified by colony PCR.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
paris-bettencourt-2018@cri-paris.org
