<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Bettencourt/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col-md-8"><UL class=" text-left wtf-menu "><LI class="parent main-item"><A href="#">Project  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Production">Production</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Active_Testing">Testing </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Active_Testing">Active Testing</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Biocompatibility_Testing">Biocompatibility Testing</A></LI></UL><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Optimization">Optimization</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Public_Engagement">Public Engagement</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Software">Software  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMPDesigner_Overview">AMP Designer Overview</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMP_Forest">AMP Forest</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/AMP_Evolver">AMP Evolver</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Result">Result</A></LI><LI class="parent main-item"><A href="#">Judging </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Medal_Criteria">Medal Criteria </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Results">Key Results</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/BioBricks">BioBricks</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/InterLab">InterLab</A></LI><LI class="parent main-item"><A href="#">Team  </A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Lab_Notebook">Lab Notebook</A></LI><LI class="parent main-item"><A href="https://2018.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="logolist2"><DIV class="sidelogos top5"><DIV id="twitapp"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV class="textbody h1"><H1>Modeling</H1></DIV><DIV class="textbody"><P>Our project is based on the consequences of a conformational change of antimicrobial peptides. Results generated by the testing group showed that MIC is not a reliable criteria to understand the activity of our StarCores while it has been previously used for species. It was crucial to have some models to:</P></DIV><DIV class="textbody"><P><B>1.</B>Determine which constructs would be interesting.</P><P><B>2.</B>Interpret our experiments results.</P></DIV><DIV class="textbody h2"><H2>RESULTS</H2></DIV><DIV class="textbody h4"><P>Our Modelling workflow could be summed up as :</P></DIV><DIV class="textbody"><P><B> a. </B>Two hackathons to screen the most promising cores and AMPs based on rationnal criteria.</P></DIV><DIV class="textbody"><P><B> b. </B>Homology modeling of the 210 constructs with Modeller on Pymol.</P></DIV><DIV class="textbody"><P><B> c. </B> Molecular Dynamic simulations to test the stability of the predicted structures using Yasara and Inserm Cluster.</P></DIV><DIV class="textbody h2"><H2>I. Homology Modelling</H2></DIV><DIV class="textbody"><P>Followed by cores and AMPs selection, we were interested in determing the model of the fusion. This led us to the modeling expert, Antoine Tally, who guided us to standardise a protocol for our analysis.<B> Aim: To construct an atomic-resolution model of Star-core monomer via comparative homology modelling.</B></P></DIV><DIV class="textbody"><P><B>1.</B> Making fusion protein of 2 known PDB files using CHIMERA.</P></DIV><DIV class="textbody"><P><B>2.</B> Monomer construct on Chimera using Modeller graphic interface.</P></DIV><DIV class="textbody"><P><B>3.</B> Construct optimization on Chimera.</P></DIV><DIV class="textbody"><P>Superpozition on Pymol thanks to handmade script called<B> -&gt; </B><A href="http://Superpoz.py" target="_blank">Superpoz.py</A><B>&lt;-</B></P></DIV><DIV class="textbody h2"><H2>II. Molecular dynamic simulation</H2></DIV><DIV class="textbody"><P><B>OBJECTIVE:</B></P></DIV><DIV class="textbody"><P><B>-</B>Study behaviour of fusion molecule in vicinity of cell.</P></DIV><DIV class="textbody"><P><B>-</B>Assess the stability of Star cores in terms of protein folding, structural characteristics and energy.</P></DIV><DIV class="textbody"><P><B>-</B>Define constraints and parameters for simulations.</P></DIV><DIV class="textbody"><P><B>-</B>Use Yasara to analyze MDS data.</P></DIV><DIV class="textbody h2"><H2>CONCLUSION</H2></DIV><DIV class="textbody"><P>In this project, we combinatorially fused a set of known AMPs to structurally diverse, self-assembling protein cores to produce star-shaped complexes.</P><P>We selected 14 cores that already exist and 15 AMPs, hence over 200 fusions were designed and expressed in a cell-free system, then screened for activity, biocompatibility, and membrane selectivity.</P><P>To study the behaviour of in-vitro synthesised molecules we designed the fusion molecules for all constructs using Chimera, Modeller and Pymol. We visualized the assembly and fusion monomer (core + AMP) and studied their behavioural attributes, changes in folding of alpha helix and beta sheath using Yasara as it mimics atom behaviour in real life. Using visualization and MDS studies we confirm that proteins folds well and on an average maintain constant RMSF (Root Mean Square Fluctuations) with all amino acid residues, also expressed well in cell free system.</P></DIV><DIV class="textbody h2"><H2>METHODS</H2></DIV><DIV class="textbody"><P><B>These protocols were defined under certain parameters:</B></P></DIV><DIV class="textbody"><P>AMPs are relatively small, so we assumed that structural changes would be minimal, enabling us to perform Homology modeling.</P><P>AMPs are only fused on the N/C terminals thus oriented outside the homomultimeric self-assembling protein, the modelisation should keep the geometry of the nude core, displaying the fusionned peptides on the surface of the core. It enabled us to structure superposition.</P><P>However we verified this hypothesis with our stability MD assay developped with Marc Baaden.</P></DIV><DIV class="textbody h2"><H2>Homology Modelling</H2></DIV><DIV class="textbody"><P><B>What is homology modelling?</B></P></DIV><DIV class="textbody"><P>Homology modelling is comparative modelling of proteins. It is a comparative protein modelling method designed to find the most probable structure for a sequence given its alignment with related structures. The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability density functions (pdfs) for the features restrained. For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equivalent residue in a related protein, and the local similarity between the two sequences. Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures. The pdfs restrain C alpha-C alpha distances, main-chain N-O distances, main-chain and side-chain dihedral angles. A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database. The 3D model of a protein is obtained by optimization of the molecular pdf such that the model violates the input restraints as little as possible. The molecular pdf is derived as a combination of pdfs restraining individual spatial features of the whole molecule. The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms. The method is automated. We used modeller to predict all our models. The steps are: </P></DIV><DIV class="textbody"><P><B>1.</B> Obtain reference PDB structures representing the core and antimicrobial peptide protein monomers</P><P><B>2.</B> Use MODELLER via CHIMERA interface for homology modelling</P><P><B>3.</B> Choose the best fusion protein model that represents the Star core monomer</P></DIV><DIV class="textbody"><P>For this workflow we will use:</P><P><B>UCSF-CHIMERA</B>: download link</P><P><B>MODELLER</B>: online or installed</P><P>Core: Ferritin: PDB= 4XGS</P><P>AMP:	Cg-Defensin: PDB= 2B68</P><P>AA seq in FASTA format: download</P><P>pDB.001_translation</P><P>MGFGCPGNQLKCNNHCKSISCRAGYCDAATLWLRCTCTDCNGKKESSHLKPEMIEKLNEQMNLELYSSLL</P><P>YQQMSAWCSYHTFEGAAAFLRRHAQEEMTHMQRLFDYLTDTGNLPRINTVESPFAEYSSLDELFQETYKLEQ</P><P>LITQKINELAHAAMTNQDYPTFNFLQWYVSEQHEEEKLFKSIIDKLSLAGKSGEGLYFIDKELSTLDTQN</P></DIV><DIV class="textbody h2"><H2>Part 1: Making fusion protein of 2 known PDB files
using CHIMERA &amp; AMP MODELLER</H2></DIV><DIV class="textbody"><P>Open the fasta file in Chimera.  Then from sequence window menu: Info… Blast Protein to search the PDB for matching structures.  In the Blast Protein results, find 4XGS and 2B68 and choose both lines (click, ctrl-click in the results dialog), then click the “Show in MAV” and “Load Structure” buttons at the bottom of the dialog.</P><P>Now you will have a new sequence alignment window with 3 sequences in it: the fusion protein and the two protein structures, and in the main Chimera window, the two structures.</P><P>In Chimera, delete all extra protein chains. In other words, if there are extra copies of those structures, just delete them so that you have only one copy to use as the template. Make sure that the remaining copy of each is associated with its sequence in the alignment (sequence alignment window menu: Structures… Associations…)</P><P>Position the two structures so that the termini are in a somewhat reasonable place relative to each other to template the fusion protein. In our case, the C-terminal of Defensin monomer attaches with the N-terminal of Ferritin monomer. You can “freeze” one in place by deactivating it and move just the other with the mouse as described here:</P></DIV><DIV class="textbody"><P>From the sequence alignment window menu choose:  Structure… Modeller (homology) to show the Modeller dialog.  Choose the query as the target and both structures as the template, etc. as in the modeling tutorials.  You may also want to turn on “Use thorough optimization” in the Advanced Options section.</P></DIV><DIV class="textbody"><P>NOW YOU CAN SAVE ONE OF THE STRUCTURE AS A PDB FILE AND USE THIS AS A MONOMER FOR FURTHER STEPS.</P></DIV><DIV class="textbody h2"><H2>Part 2: Make a multimer of the fusion protein using reference PDB structure in Pymol</H2></DIV><DIV class="textbody"><P>Open core assembly ( 4XGS) and fusion monomer (4XGS + 2B68) in pymol.</P><P>Use python based code, <B> -&gt; </B><A href="http://superpoz.py" target="_blank">superpoz.py</A><B>&lt;-</B>.</P><P> We wrote a small script to generate the assembly structure of our StarCores based on the reference biological assembly of the nude core.</P></DIV><DIV class="textbody h2"><H2>Result</H2></DIV><DIV class="textbody"><P>Fusion monomers for all the constructs are developed for molecular dynamic simulation studies and assembly of core with monomer for visualization of scaffold proteins.</P></DIV><DIV class="textbody h2"><H2>Molecular dynamic simulation</H2></DIV><DIV class="textbody"><P>MD studies are based on Newton second law of motion:</P><P><B>F = ma</B></P><P>Where <B>F</B> is the force of an atom, <B>m</B> is the mass of the particle and <B>a</B> is the acceleration.</P><P>As, per universal concept of energy</P><P><B>F(x) = −∇U(x)</B></P><P>Where <B>x</B> represents coordinates of all atoms, and <B>U</B> is the potential energy function, Velocity is the derivative of position, and acceleration is the derivative of velocity.</P><P>We can thus write the equations of motion as:</P><P><B>dx/dt = v</B></P><P><B>dv / dt = F(x)/m</B></P><P>This is a system of ordinary differential equations. For &lt;<B>n</B> atoms, we have <B>3n</B> position coordinates and <B>3n</B> velocity coordinates. Calculation of  “Analytical” (algebraic) solution is impossible but Numerical solution can be given by,</P><P><B>xi+1 = xi +δ t vi</B></P><P><B>vi+1 = vi +δ t F(xi )/m</B></P><P>This is layman representation of principle used for molecular dynamic simulation studies. Since, in real life and MD also, atoms are in constant motion then motion is used to understand the probability of observing a particular arrangement of atoms as a function of the potential energy.</P></DIV><DIV class="textbody h2"><H2>Objective</H2></DIV><DIV class="textbody"><P><B>Open YASARA</B></P><P><B>1. </B>Go to options &gt; Macro &amp; Movie &gt; Set Target pdb &gt; md_runfast.mcr</P><P><B>2. </B>Wait for the structure to run upto 15ns.</P><P><B>3. </B>Record a video if required.</P><P><B>4. </B>Analyse the result using MD_analyze.mcr</P><P><B>5. </B>Analyse each residue  using MD_analyzeres.mcr</P></DIV><DIV class="textbody"><P>Figure :  Simulation of fusion monomer ferritin + ovispirin (4XGS + 1HU5)</P></DIV><DIV class="textbody h2"><H2>Parameters</H2></DIV><DIV class="textbody"><P> -pH at which the simulation should be run, by default physiological pH 7.4</P><P>The ion concentration as a mass fraction, here we use 0.9% NaCl (physiological solution)ions=‘NaCl,0.9’</P><P> -Simulation temperature, 298K</P><P> -Water density = 0.997</P><P> -Duration of the simulation = 15ns</P><P> -Extension of the cell on each side of the protein ‘10’ means that the cell will be 20 A larger than the protein.</P><P>Shape of the simulation cell: ‘Cube’.</P><P> -Forcefield : ForceField AMBER14</P><P> -Cell boundary : Boundary periodic</P><P>The simulation speed, ‘fast’ (maximize performance with 2*2.5 fs timestep and constraints)
The save interval for snapshots. Normally you don’t need more than 500-1000 snapshots of your simulation.
Solute from diffusing around and crossing periodic boundaries. Disable that for simulations of crystals.</P></DIV><DIV id="footerdiv"><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France<A href="mailto:paris-bettencourt-2018@cri-paris.org">paris-bettencourt-2018@cri-paris.org</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>