<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Shenzhen_SFLS_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Shenzhen SFLS/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Shenzhen SFLS/Demonstrate - 2017.igem.org</TITLE></P><DIV id="header"><NAV class="sfls-nav" role="navigation"><DIV class="container"><DIV class="row"><DIV class=" text-center menu-1 "><UL><LI class="has-dropdown"><A href="https://2017.igem.org/Team:Shenzhen_SFLS">Project</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Description ">Description</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Design ">Design</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Experiments ">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Demonstrate ">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Collaborations ">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Achievement ">Achievement</A></LI></UL><LI class="has-dropdown"><A href="#">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Basic_Part ">basic part</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Composite_Part ">composite part</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Model">Modeling</A></LI><LI class="has-dropdown"><A href="#">Human Practice</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/HP/Gold_Integrated ">GOLD</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/HP/Silver ">SILVER</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Publicity ">Publicity</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Biosafety ">Biosafety</A></LI><LI class="has-dropdown"><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Notebook ">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Notebook ">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Protocols ">protocols</A></LI><LI class="has-dropdown"><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Team">About us</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Attributions ">Attributions</A></LI></DIV></DIV></DIV></NAV></DIV><TITLE>Team:Shenzhen SFLS/Demonstrate - 2017.igem.org</TITLE><DIV class="img_head_bar text-center"><DIV class="display-t" style="background-image:url(https://static.igem.org/mediawiki/2017/c/c4/Sfls-title-modeling.svg); background-repeat:no-repeat; background-attachment:fixed"><DIV class="display-tc"><H1>Modeling</H1></DIV></DIV></DIV><DIV class="marin_top"><DIV class="marin_top_content"><DIV id="tbox"><UL class="ul_t"><LI class="taoyu" href="#index_0" id="BigTitle"><DIV class="taoyu_title"><A href="#index_0" title="0">Off-target Prediction</A></DIV></LI><LI class="taoyu" href="#index_1"><DIV class="taoyu_title"><A href="#index_1" title="1">Introduction</A></DIV></LI><LI class="taoyu" href="#index_2"><DIV class="taoyu_title"><A href="#index_2" title="2">Methods</A></DIV></LI><LI class="taoyu" href="#index_3"><DIV class="taoyu_title"><A href="#index_3" title="3">Results</A></DIV></LI><LI class="taoyu" href="#index_4"><DIV class="taoyu_title"><A href="#index_4" title="4">Conclusion</A></DIV></LI><LI class="taoyu" href="#index_5" id="BigTitle"><DIV class="taoyu_title"><A href="#index_5" title="5">Drug Designing</A></DIV></LI><LI class="taoyu" href="#index_6"><DIV class="taoyu_title"><A href="#index_6" title="6">Introduction</A></DIV></LI><LI class="taoyu" href="#index_7"><DIV class="taoyu_title"><A href="#index_7" title="7">Diffusion part</A></DIV></LI><LI class="taoyu" href="#index_8"><DIV class="taoyu_title"><A href="#index_8" title="8">Reaction part</A></DIV></LI><LI class="taoyu" href="#index_9"><DIV class="taoyu_title"><A href="#index_9" title="9">Future work</A></DIV></LI></UL></DIV><DIV class="main_context" id="main"><DIV class="row"><DIV class="mainpage mainpage5"><H3 id="index_0">Prediction of Off-target Effect</H3></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage1"><H5 id="index_1">INTRODUCTION</H5><P>Since the CRISPR/Cas9 system was first used in genetic engineering, the researches on its off-target effects have never stopped. The methods of Hsu-Zhang scoring (1) and CCTop (2) are two widely used algorithms for designing a single guide
              RNA (sgRNA) sequence and finding potential off-target locus. Last year, a new algorithm named CFD (Cutting Frequency Determination) scoring method was developed to evaluate potential off-target sits with 240 parameters (Fig. 1) (3). All
              of these three methods (Hsu-Zhang scoring, CCTop, and CFD) take  different weight coefficients of different mismatch position into consideration, however, only CFD scoring method considers mismatch types as a factor as well.</P></DIV><P>Fig.1 The values of CFD scores change over mismatch positions and types (Data derived from Doench 2016). Mismatch position is counted from 5’ end of gene, position 20 represents the nucleotide nearest to protospacer adjacent motif (PAM), and
            position 1 represent the nucleotide farthest from PAM.</P></DIV><DIV class="row"><DIV class="mainpage mainpage2"><H5 id="index_2">Methods</H5><P>We chose to use CFD scoring method instead of Hsu-Zhang scoring or CCTop for the following reasons: First, it is reported that the CFD method has higher Pearson correlation (3), compared with Hsu-Zhang scoring method and CCTop, especially
              when the number of mismatched bases is large; Second, Computing the scores by using CFD method is much easier than the other two methods. </P><P>In order to obtain the CFD score of a certain DNA locus, we multiply all the scores of single base mismatch together. If the DNA loci and sgRNA have mismatched bases at position α, β, γ… with mismatch types rA-dC, rC-dC, rU-dT…(Fig. 1B), its
              CFD score is calculated as:</P><P>It is reported that about 60% of melanomas contain a mutation in the v-raf murine sarcoma viral oncogene homolog B (BRAF), and V600E (1799T&gt; A) variation (Fig. 2) in BRAF is the main type of mutations in the cancer tissues, which plays a critical
              role in carcinogenesis of melanoma.</P><P>Fig. 2 The graphic of BRAF 1799T&gt;A (V600E) mutation</P><P>In our project, we try to disrupt the mutant BRAF in the two melanoma cell lines (A375 and G361) by CRISPR/Cas9 technology. A typical PAM is ‘NGG’. However, we didn’t find it. It has been reported that alternative PAM sequence ‘NAG’ has rather
              high cleavage efficiency while ‘NTG’ shows no tendency of cutting (3). To meet the goal of specific cleavage, we arranged the mutant base on the three PAM bases as shown in Fig 3. </P><P>Fig.3 The sgRNA for targeting mutant BRAf in melanoma cells</P><P>After setting the sgRNA sequence, we searched for the potential off-target locus. The potential off-target locus must meet the following conditions: 1) Having a PAM sequence (‘NGG’, ‘NAG’, ‘NCG’, or ‘NGA’) at 3’ end; 2) More than 13 base identities are
               identified by <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM =blastn&amp;PAGE_TYPE=BlastSearch&amp;LINK_LOC=blasthome">MegaBLAST</A>; 3) CFD score is greater than 5%.</P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage3"><H5 id="index_3">RESULTS</H5><P>Using Megablast, we find that over 500 alignments have potential off-target effects, and 62 of them have a PAM sequence (‘NGG’, ‘NAG’, ‘NCG’, or ‘NGA’). Seven of them are scored higher than 5% (Fig. 4). Since ‘NGG’ PAM has much higher efficiency
              of cleavage than ‘NAG’ (Fig.5)(3), the off-target probability of Seq 2, 3, 4, 5, 6 and 7 may be higher than its scores.</P><P>Fig.4 Potential off-target sites of our sgRNA. PAM is marked in red. Compared with the sgRNA, the different bases are marked in yellow.</P></DIV><P>Fig.5 Proportion of active sgRNAs with different PAM on targeting region(Data derived from Doench 2016)</P></DIV><DIV class="row"><DIV class="mainpage mainpage2" id="reference"><H5 id="index_4">Conclusion</H5><P>Our sgRNA sequence has high cleavage efficiency on the mutated BRAF gene, as well as a high risk of off-target effect. To avoid the off-target effect, we designed an artificial microRNA complementary to SAMMSON gene, which is specifically
              expressed in human melanomas. The CRISPR/CAS9 system is only activated in cancer cells, having no any effects on normal cells.</P><P>View our regulatory system at <A href="https://2017.igem.org/Team:Shenzhen_SFLS/Design#index_3">https://2017.igem.org/Team:Shenzhen_SFLS/Design</A></P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage2" id="reference"><H5>References</H5><LI>http://crispr.mit.edu/</LI><LI> https://crispr.cos.uni-heidelberg.de/ </LI><LI> Doench, J. G., Fusi, N., Sullender, M., Hegde, M., Vaimberg, E. W., Donovan, K. F., ... &amp; Virgin, H. W. (2016). Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nature biotechnology, 34(2), 184-191.</LI><LI> Kleinstiver, B. P., Prew, M. S., Tsai, S. Q., Topkar, V. V., Nguyen, N. T., Zheng, Z., ... &amp; Aryee, M. J. (2015). Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Nature, 523(7561), 481-485.</LI></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage5"><H3 id="index_5">Drug Designing</H3></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage6"><H5 id="index_6">Introduction</H5><P>At first, we intended to use liposome as genetic vector. However, experiment data shows that liposome (has blank plasmid in it) has a high cytotoxic effect. When the liposome concentration is 4μg/mL, the number of cells died from cytotoxic
              effect is even greater than the number of survival cells (G361). Because the malignant degree of A375 is higher than G361, cytotoxic effect seems much weaker in A375.</P><P>Fig.1 Growth curves of the two cell lines: A375 and G361 (Stannard fitting)</P><P>Fig.2 A comparison of R-Squares of exponential curve and different sigmoid curves. “⭐” marks the best fitting equation.</P><P>Therefore, we planned to use lentivirus as gene vector, which is reported to have lower cytotoxic effect and higher transduction efficiency than liposome vector. Lentiviruses enter into cells while most kinds of viruses bind to cellular membrane and inject their genetic materials into cells when transducing.</P><P>Fig.3 Lentiviruses diffusing and entering cells in vivo (Illustration by <I>Tumblr: rainalv </I>)</P><P>As the picture shows, the lentiviruses behave two ways in vivo: Diffusion and reaction (entering cells). The concentration of drug is a function of time (since injection of drug) and position (the distance to drug injection site).</P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage7"><H5 id="index_7">Diffusion part</H5><P>We assume that lentiviruses' diffusion in extracellular substance is unsteady-state diffusion, which follows Fick’s Second Law:</P><P>C<SUB>diff</SUB> is the concentration of drug participating diffusion, C stands for the concentration of drug at certain time and position. t stands for time, D is the diffusion coefficient of lentivirus diffusing in melanoma tissue, ∇
              is the Nabla operator.
              </P><P>We assume the diffusion coefficient D is a constant, which does not change over time or space. Thus, the equation can be written as:</P><P>Δ is the Laplace operator.</P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage8"><H5 id="index_8">Reaction part</H5><P>It is obviously that at each moment, the concentration of lentiviruses entering cells is proportional to the current concentration of lentivirus.</P><P>C<SUB>reac</SUB> stands for the concentration of drug participating reaction,C stands the concentration of drug, k is a constant.</P><P>Combining the two terms together, we have:</P><P>We assume that the volume of each dose is V, and the drug present a spherical shape at the moment of its injection into skin. R is the radius of the spherical drug system.</P><P>The partial differential equation has the following boundary conditions:</P><P>I At the moment of injection, the drug has not diffused or entered into cells, so the concentration inside the spherical drug system is the initial drug concentration C<SUB>0</SUB>.</P><P>II&amp;III The concentration is approaching zero when time or distance is long enough.</P><P>Fig.5 Forecast results of spherical drug diffusion and reaction in vivo</P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage9"><H5 id="index_9">Future work</H5><P>Since the time is limited, we cannot put our genetic circuit into drugs. We also failed to find data from the existing articles with which the values of parameters can be perfectly determined.</P><P>The parameters D and k can be determined using the method mentioned in Minchinton 2006(4).</P><P>We can know the concentration at any time and position, when the initial concentration and the volume of each dose are set. Thus, the drug release is under control.</P><P>When the drug gets to the border of the dermis and subcutaneous tissue (where the distance to injection site is x<SUB>s</SUB> x_s), it will be taken away through the complex network of blood vessels. To avoid doing harm to normal cells,
C_max is the concentration which results in the maximal tolerable cytotoxic effect., the concentration at the border should be lower than the concentration resulting in the maximal tolerable cytotoxic effect, which is to be determined using the future experiment data.</P></DIV></DIV><DIV class="row"><DIV class="mainpage mainpage2" id="reference"><H5>References</H5><LI>Zwietering M H, Jongenburger I, Rombouts F M, et al. Modeling of the bacterial growth curve[J]. Applied and environmental microbiology, 1990, 56(6): 1875-1881.</LI><LI>https://en.wikipedia.org/wiki/Fick%27s_laws_of_diffusion</LI><LI>Siepmann J, Siepmann F. Modeling of diffusion controlled drug delivery[J]. Journal of Controlled Release, 2012, 161(2): 351-362.</LI><LI>Minchinton A I, Tannock I F. Drug penetration in solid tumours[J]. Nature Reviews Cancer, 2006, 6(8): 583-592.</LI></DIV></DIV></DIV></DIV><DIV class="img_bottom_bar  text-center"><DIV class="display-t"><DIV class="display-tc"><H1> </H1></DIV></DIV></DIV><FOOTER id="sfls-footer" role="contentinfo"><DIV class="container"><DIV class="row row-pb-md"><DIV class="col-md-4 sfls-widget"><H3><A href="home.html">SFLS.</A></H3><P id="ours">Shenzhen Foreign Language School, Yantian Road No.1, Yantian District, Shenzhen, Guangdong, China</P></DIV><DIV class="col-md-2 col-sm-4 col-xs-6 col-md-push-1"><UL class="sfls-footer-links"><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS">Home</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Description ">Description</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Design ">Design</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Experiments ">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Demonstrate ">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Achievement">Achievement</A></LI></UL></DIV><DIV class="col-md-2 col-sm-4 col-xs-6 col-md-push-1"><UL class="sfls-footer-links"><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Basic_Part ">Basic Part</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Composite_Part ">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Model ">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/HP/Silver ">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/HP/Gold_Integrated ">HP-Gold</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Biosafety ">Biosafety</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Protocols">Protocols</A></LI></UL></DIV><DIV class="col-md-2 col-sm-4 col-xs-6 col-md-push-1"><UL class="sfls-footer-links"><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Team ">About us</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Attributions ">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Notebook ">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Publicity ">Publicity</A></LI><LI><A href="https://2017.igem.org/Team:Shenzhen_SFLS/Improve">Improve</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>