Team:Shenzhen SFLS/Experiments
Team:Shenzhen SFLS/Demonstrate - 2017.igem.org
Project
Description
Design
Experiment
Demonstrate
Collaboration
Achievement
Parts
basic part
composite part
Modeling
Human Practice
GOLD
SILVER
Publicity
Biosafety
Notebook
Notebook
protocols
Team
About us
Attributions
Team:Shenzhen SFLS/Experiment - 2017.igem.org
Experiment
Plasmid construction
Functional test of sgRNA
Cell culture
Transfection, TA cloning and Sequencing
Cell proliferation assay
Cell migration assay
Flow cytometry
Plasmid construction
To generate a plasmid that expresses a sgRNA targeting mutant BRAF in melanoma cells, a plasmid (pHS-ACR-ZQ170) containing a sgRNA driven by U6 promoter (Part: BBa_K2200010) and a Cas9 protein driven by phEF1a promoter was constructed. In order to test the function and specificity of the sgRNA, two plasmids containing a wild-type BRAF sequence (pHS-ACR-ZQ190) and mutant BRAF (pHS-ACR-ZQ191)were constructed. The two plasmids both contained prematurely terminated mKate gene (a new type of dark red fluorescent protein originated from TagRFP) driven by phEF1a promoter.
Functional test of sgRNA
The function and specificity of our sgRNA were tested by gRNA ActivityAssay Test Kit (Beijing Syngentech Co., LTD.). The mKate gene in pHS-ACR-ZQ191 and pHS-ACR-ZQ190 was terminated prematurely by a terminator. In order to test the activity of our gRNA, the target sequence mutant BRAF was inserted after the terminator. After the action of Cas9 and sgRNA, the double-strand DNA at the target site was cleaved to form a double strand breaks(DSB), and the mKate were activated though cell homologous recombination.
HEK293 cells were transfected with pHS-ACR-ZQ170 and pHS-AVC-ZQ190as negative control group (NC group) (Fig.1A), and transfected with pHS-ACR-ZQ17 and pHS-AVC-ZQ190 as treatment group(Fig.1B). The gRNA activityassay demonstrated that the CRISPR-Cas9 system specifically cleaved the mutant BRAF V600E gene (Fig.1B), but no effects on wild type BRAF (Fig.1A).
Cell culture
We selected A375 and G361 cells to demonstrate the function of our circuit in human melanoma cells. A375 is a melanoma cell line with BRAF V600E homozygous mutation while G361 is with heterozygous mutation in BRAF gene. A375 and G361 cells were maintained in DMEM medium (Invitrogen, Carlsbad, CA, USA) supplemented with 10% FBS at 37°C in an atmosphere of 5%CO2.
Transfection, TA cloning and Sequencing
The plasmid containing the CRISPR/Cas9 system with a sgRNA for mutant BRAF gene was transfected to A375 and G361 cells. To clarify the genomic sequence alterations of the BRAF genomic region around V600E site in melanoma cells with or without the sgRNA plasmid pHS-ACR-ZQ170 guided editing by CRISPR/Cas9, DNA fragments containing genomic region around BRAF V600 site were amplified by PCR with primers set TA-BARF(V600) F/R (F: 5'- GCTGTGGATCACACCTGCCTTAAA-3' — R: 5'- TCGCCCAGGAGTGCCAAGAGA-3'). Ligation of PCR products with pEASY-Blunt Cloning Vector were conducted by a TOPO TA Cloning KIT(CB101, TransGene®, Beijing) followed by transforming to Trans1-T1 competent cells and plating onto an LB plate containing ampicillin as well as X-gal and IPTG for the case of Blue/White screening. After 16h of incubation, 10 to 14 white colonies randomly picked from each plate were cultured at 37 °C overnight and then the bacteria culture(s) were sent to Sanger sequencing in Sangon Biotech (Shanghai) Co., Ltd.
Cell proliferation assay
After 12h of transfection, A375 and G361 cells were plated in 96-well plates. Cell counting Kit (CCK-8) was used to detect the effects of the CRISPR/Cas9 system on the cell proliferation. About 104 cells per well were seeded in a 96-well plate and pre-incubated for 12 h. After 24h, 48h, and 72h, 10ul of Cell Counting Kit solution (TransGen, Beijing, China) were added to each well. After 30 minutes’ incubation, the OD value of each well was calculated by the CCK-8 reader machine (Bio-Rad, Hercules, CA, USA). The experiments were performed at least three times.Line charts were plotted according to these data.
Cell migration assay
Cell migration was measured by Transwell assay. Having been transferred and cultured in the 10% fetal bovine serum (FBS) in DMEM medium for 24h, A375 and G361 cells were seeded in triplicate in 24-well plates (5x105 cells per well).The upper chambers contained serum-free medium, which was separated from the lower chambers containing DMEM with 10% FBS medium by polycarbonate membrane. After incubated for 24 h, cells that had passed through the membrane were stained with (0.5%) crystal violet and then counted under a microscope.
Flow cytometry
In order to detect effects of the CRISPR/Cas9 system on cell apoptosis, the flow cytometry was used. A375 and G361 cells were placed in a 6-well plate. Then cells were harvested using trypsin without EDTA and collected. According to the manufacturer’sprotocols，cells were double-strained with FITC-Annexin and PI, which could mark early apoptotic cells and late apoptotic cells respectively. A flow cytometer (EPICS, XL-4, Beckman, CA, USA) was used to observe cell apoptosis. Cells were divided into living cells, early apoptotic cells, late apoptotic cells and dead cells. The ratio of early apoptotic cells and late apoptotic cells in the negative control group and the experimental group was used as an index for comparison.
 
SFLS.
Shenzhen Foreign Language School, Yantian Road No.1, Yantian District, Shenzhen, Guangdong, China
Home
Description
Design
Experiment
Demonstrate
Collaboration
Achievement
Basic Part
Composite Part
Modeling
HP-Silver
HP-Gold
Biosafety
Protocols
About us
Attribution
Engagement
Contribution
Notebook
Publicity
Improve
