<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUIS_Shanghai_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUIS Shanghai/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menudiv"><DIV class="singleoption" id="so1"><DIV class="singlemainop"><A class="mlink" href="https://2018.igem.org/Team:SUIS_Shanghai"><P>HOME</P></A></DIV></DIV><DIV class="singleoption" id="so2"><DIV class="singlemainop"><P>TEAM</P></DIV><DIV class="singlesubop" id="ssop1"><A href="https://2018.igem.org/Team:SUIS_Shanghai/Team"><NOBR>Team Members</NOBR></A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Collaborations">Collaborations</A></DIV></DIV><DIV class="singleoption" id="so3"><DIV class="singlemainop"><P>PROJECT</P></DIV><DIV class="singlesubop" id="ssop2"><A href="https://2018.igem.org/Team:SUIS_Shanghai/Description">Description</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Design">Design</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Experiments">Experiments</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Notebook">Notebook</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/InterLab">InterLab</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Model">Model</A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Attributions">Attributions</A></DIV></DIV><DIV class="singleoption" id="so4"><DIV class="singlemainop"><A class="mlink" href="https://2018.igem.org/Team:SUIS_Shanghai/Parts"><P>PARTS</P></A></DIV></DIV><DIV class="singleoption" id="so5"><DIV class="singlemainop"><A class="mlink" href="https://2018.igem.org/Team:SUIS_Shanghai/Safety"><P>SAFETY</P></A></DIV></DIV><DIV class="singleoption" id="so6"><DIV class="singlemainop"><P>HUMAN PRACTICES</P></DIV><DIV class="singlesubop" id="ssop4"><A href="https://2018.igem.org/Team:SUIS_Shanghai/Human_Practices"><NOBR>Human Practices</NOBR></A><A href="https://2018.igem.org/Team:SUIS_Shanghai/Public_Engagement">Education&amp;Engagement</A></DIV></DIV><DIV class="singleoption" id="so7"><DIV class="singlemainop"><A class="mlink" href="https://2018.igem.org/Team:SUIS_Shanghai/Applied_Design"><P>AWARDS</P></A></DIV></DIV><DIV class="singleoption" id="so8"><DIV class="singlemainop"><A class="mlink" href="https://igem.org/2018_Judging_Form?team=SUIS_Shanghai"><P>JUDGING FORM</P></A></DIV></DIV></DIV><DIV class="wrapersbody"><DIV class="wrapers"><DIV class="titledivs">Experiments</DIV><H3 id="expe1">Summary</H3><H4>We genetically engineered E.coli cells to perform heterologous expression of a biosynthetic gene cluster for the production and secretion of the siderophore vibrioferrin. This was achieved with the synthesis and transformation of an expression vector containing our genes of interest. Cells were cultured in our lab and expression was achieved through an inducible promoter. We measured siderophore activity in cell cultures by use of the Chrome Azurol S (CAS) assay. Finally we wanted to confirm any positive CAS results were caused by the presence of vibrioferrin and hence, the exogenous gene cluster. </H4><DIV id="expopcontain"><DIV id="expoption1div"><DIV class="expoptionwords">Heterologous Expression of Gene Cluster</DIV></DIV><DIV id="expoption2div"><DIV class="expoptionwords">Detection and identification of Vibrioferrin activity</DIV></DIV></DIV><DIV id="expsection1" class="expsections"><DIV class="titledivs">Heterologous Expression of Gene Cluster</DIV><H3 id="expe2">DNA synthesis:</H3><H4>The sequence of our target gene cluster was obtained from NCBI. Prior to synthesis onto plasmids by Genscript™ we edited the sequence to remove all illegal restrictions sites to ensure our biobricks would be RFC10 compatible. Table 1 outlines the locations of edits that were made to the original sequence from Vibrio parahaemolyticus. </H4><H5>Table 1 - Identification of illegal restriction enzyme sites and their location within our original gene sequences FASTA files. Illegal sites were made to the sequence through silent mutations prior to sequencing.</H5><H3 id="expe3">Transformation of pET30a(+) vector</H3><H4>Transformation of the synthesized expression vector into BL21(DE3) high efficiency competent cells was achieved through Heat Shock method. Protocol from NEB. 
Successful transformation was observed on LB agar plates containing Kanamycin antibiotic marker (working condition 50µl/ml). Positive control (BL21 cells plated on LB agar plates w/o Kanamycin) showed cells were viable, while Negative control (BL21 cells w/o plasmid plated on LB agar plates containing Kanamycin) yielded no colonies. All plates were incubated for 16 hours overnight at 37 C. </H4><H5>Fig 1A &amp; 1B - LB agar plates containing Kanamycin antibiotic selection marker showing successful transformation of the pET-30a(+) expression plasmid, containing part BBa_K269000, in BL21(DE3) cells (1A) and positive and negative controls showing effectiveness of antibiotic and viability of cells (1B).</H5><H3 id="expe4">Expression of Biosynthetic Gene Cluster</H3><H4>Single colonies were inoculated in 40 ml LB broth containing Kanamycin in a 1000:1 ratio in Falcon tubes and cultured at 37 C while shaking at 220 rpm until OD600 = 0.5 (figure 2). 5 ml of cell culture broth was added to each of six 15 ml falcon tubes. IPTG (Thermofisher) was added to five of the tubes in the following concentrations:</H4><H5>Table 2 - Inducing of gene of interest on pET-30a(+) vector by use of IPTG. Final Contents of each experiment group is described in column 3. </H5><H4>Tubes were incubated at 37 C @ 220 rpm for 3 hours to allow for biosynthesis and secretion of vibrioferrin.</H4><H3>References:</H3><H4>Maitra, &amp; Dill. (2014). Bacterial Growth and Division: Theory. Biophysical Journal, 106(2), 378a-379a.
Rosano, G., &amp; Ceccarelli, E. (2014). Recombinant protein expression in Escherichia coli : Advances and challenges. Frontiers in Microbiology, 5, 172.</H4></DIV><DIV id="expsection2" class="expsections"><DIV class="titledivs">Detection and identification of Vibrioferrin activity</DIV><H3 id="expe5">Siderophore detection assay using CAS agar plates</H3><H4>After induction with IPTG, 1.0 ml of each cell culture broth was transferred to 1.5 ml sterile centrifuge tubes and centrifuged at 11,000 g for 15 minutes (Hu 2011). An assay was conducted to determine what IPTG concentration yielded the highest productivity of siderophore. 5 mm plugs were made on 9 cm CAS agar plates using the large size of a micropipette tube (Louden et al., 2011). 20 µl of the cell free supernatant was extracted and placed in the plug. Plates were left in the dark for 8 hours at room temperature and detection of siderophore activity was indicated by orange halos forming around the plate plugs (Louden et al., 2011). This qualitative test was used to determine siderophore activity, and thus expression of genes at different concentrations of IPTG. </H4><H5>Fig 1 - CAS agar plate qualitative test. 5mm diameter plugs were made in the CAS agar and filled with cell free supernatant. Yellow/orange halos around the plug indicate siderophore activity due to a strong iron chelator such as a siderophore removing iron from the dye complex within the agar (Louden et al., 2011).</H5><H4>It must be noted however that some reports found that bacteria produced substantial amounts of siderophore in liquid solution did not always produce halos on the CAS plate. (Alexander &amp; Zuberer, 1991)</H4><H3 id="expe6">Quantitative analysis of CAS liquid Assay.</H3><H4>For the quantitative method of estimation of siderophore production, 0.5 ml of the supernatant of the cultures was transferred to a glass centrifuge and mixed with 0.5 ml of CAS liquid reagent. CAS liquid reagent was made following the protocol described in. Siderophore production was calculated using the equation below and measured in psu (percent siderophore units) (Payne 1993;  Arora &amp; Verma 2017). Absorbances were measured using a spectrophotometer at 630 nm. </H4><H5>Table 1 - Absorbance and percentage siderophore units of each group in CAS liquid assay. Control was taken as Ar value as a reference.</H5><H5>Fig 2 - CAS liquid on right compared to 0.5 ml of CAS liquid + 0.5 ml Cell free supernatant containing siderophore excreted from cells. Color change shows positive result for siderophore.</H5><H5>Fig 3 - Absorbance at 630nm. Control = CAS + LB broth, A = CAS liquid + Cell free supernatant uninduced, B = CAS liquid + Cell free supernatant induced with 0.4 mM IPTG, C = CAS liquid + Cell free supernatant induced with 0.8 mM IPTG, CAS liquid + Cell free supernatant induced with 1.0 mM IPTG). CAS liquid and Cell free supernatant were added in a 1:1 ratio. </H5><H5>Fig 4 - CAS liquid assay results. From left to right: CAS liquid + Cell free supernatant induced with 0.4 mM IPTG; CAS liquid + Cell free supernatant induced with 0.8 mM IPTG; CAS liquid + Cell free supernatant induced with 1.0 mM IPTG). </H5><H5>Fig 5 - Percentage Siderophore Units (PSU) for each of the five groups (Control = CAS + LB broth, A = CAS liquid + Cell free supernatant uninduced, B = CAS liquid + Cell free supernatant induced with 0.4 nM IPTG, C = CAS liquid + Cell free supernatant induced with 0.8 nM IPTG, CAS liquid + Cell free supernatant induced with 1.0 nM IPTG). CAS liquid and Cell free supernatant were added in a 1:1 ratio. Values are the mean of 3 replicates. </H5><H4>CAS liquid assay results indicate siderophore activity in all groups. Siderophore activity was greatest when our cells were induced with 0.8 mM IPTG. When we increased the concentration to 1.0 mM siderophore activity decreased. We attribute this to an increased metabolic load affecting cell survival (Baneyx, 1999). Our results indicate that Group A also yielded siderophore activity in all three groups, despite IPTG inducer not having been added to the mixture. We attribute this to leaky expression by the T7 promoter on the pET-30a(+) vector. Our results suggest that heterologous expression of our system is best achieved with induction with 0.8 mM IPTG, when BL21(DE3) cells are induced at OD600 = 0.5. Further assays would need to be conducted to investigate the effect of cell density, incubation time, or temperature on expression levels. </H4><H3 id="expe7">Identification of Vibrioferrin</H3><H4>In order to verify that the siderophore produced by our cells was indeed vibrioferrin, and to demonstrate our cells work as intended we decided to identify the presence of vibrioferrin my mass spectrometry. Samples of cell free supernatant were prepared by centrifugation (11,000g for 15 minutes) and frozen with dry ice. Samples were sent to Shanghai Sensichip Biotechnology™ for LC-MS analysis. Protocol used is as follows:</H4><H4>1. 100μL samples were taken, adding 300 μL of methanol and 10μL of internal standard (2.9 mg/mL, DL-o-Chlorophenylalanine)
2. All samples were ultrasonicated for 30 min, by 40KHz, then let stand for 1 hour at -20 ℃.
3. The samples were centrifuged at 12000 rpm and 4 °C for 15 min.
4. 200 μL of supernatant was transferred to vial for LC-MS analysis.</H4><H4>1.Analysis platform: LC-Q/TOF-MS (Agilent,1290 Infinity LC, 6530 UHD and Accurate-Mass Q-TOF /MS)
2.Column: Agilent, 100 mm× 2.1 mm, 1.8 μm
3.Chromatographic separation conditions: Column temperature: 40 °C; Flow rate: 0.35 mL/min; Mobile phase A: water+0.1% formic acid; Mobile phase B: acetonitrile+0.1% formic acid; Injection volume:4 μL; Automatic injector temperature: 4 °C.</H4><H3 id="expe8">MS parameters</H3><H4>ESI+: capillary voltage: 4 kV; Sampling cone: 35 kV; Source temperature: 100 °C; Desolvation temperature: 350°C; Cone gas flow: 50 L/h; Desolvation gas flow: 600 L/h; Extraction cone: 4 V.
ESI-: capillary voltage: 3.5 kV; Sampling cone: 50 kV; Source temperature: 100 °C; Desolvation temperature: 350°C; Cone gas flow: 50 L/h; Desolvation gas flow: 700 L/h; Extraction cone: 4 V.
Scan time:0.03s; Inter scan time:0.02s; scan range:50-1000 m/z
Lock Mass: leu-enkephalin, [M+H]+: 556.2771 Da; [M-H]-: 554.2615 Da
</H4><H5>Figure 7 - MS data provided shows signal for vibrioferrin (plus adduct ion [M+Na]-  In the cell culture samples indicating successful and synthesis and excretion of vibrioferrin. </H5><H4>Shanghai Sensichip Biotechnology™ reported to us that they measured t a positive signal for vibrioferrin. Vibrioferrin, which has a molecular weight of 434.3223 amu, was detected. Figure 6 shows a mass to charge ratio of 456.3361 suggesting that a signal was detected but with the presence of an adduct ion (Keller et al., 2008).
Raw data provided from the three samples (Cell free supernatant after induction with 0.8 mM IPTG) sent to Shanghai Sensichip Biotechnology™ shows a m/z signal for vibrioferrin, mass = 434.3223 (see table 3) and attachment 1. Although no These results confirmed that vibrioferrin was present in the cell culture broth and therefore we achieved successful heterologous expression of the biosynthetic gene cluster and transport protein in E.coli. </H4><H5>Table 2 - MS data showing compound 2806 (Vibrioferrin) detection in counts per second (cps). </H5><H3>References:</H3><H4>Qing-Ping Hu. (2011). A simple double-layered chrome azurol S agar (SD-CASA) plate assay to optimize the production of siderophores by a potential biocontrol agent Bacillus. African Journal of Microbiology Research, 5(25), 4321-4327.
Louden, B., Haarmann, D., &amp; Lynne, A. (2011). Use of Blue Agar CAS Assay for Siderophore Detection. Journal of Microbiology &amp; Biology Education, 12(1), 51-3.
Alexander, D., &amp; Zuberer, B. (1991). Use of chrome azurol S reagents to evaluate siderophore production by rhizosphere bacteria. Biology and Fertility of Soils, 12(1), 39-45.
Payne, S. (1993). Iron acquisition in microbial pathogenesis. Trends in Microbiology, 1(2), 66-69.
Arora, N., &amp; Verma, K. (2017). Modified microplate method for rapid and efficient estimation of siderophore produced by bacteria. 3 Biotech, 7(6), 1-9.
Baneyx, F. (1999). Recombinant protein expression in Escherichia coli. Current Opinion in Biotechnology, 10(5), 411-421.
Keller, Sui, Young, &amp; Whittal. (2008). Interferences and contaminants encountered in modern mass spectrometry. Analytica Chimica Acta, 627(1), 71-81.</H4></DIV></DIV><DIV id="footerline"><DIV id="footer1"><DIV id="footer1mid"><DIV id="foormidword">IGEM   |   WANYUAN   |   SUIS</DIV><DIV id="foorrightdiv"><P>Website: wanyuan.suis.com.cn</P><P>Mail Box: suisigem@outlook.com</P><P>No.509, Pingji Rd, Minhang District, Shanghai</P></DIV></DIV></DIV><DIV id="footer2">Copyright © 2018 Shanghai United International School. All Rights Reserved.</DIV></DIV></DIV><DIV id="navbardiv"><DIV id="navlist"><DIV class="singlenav" id="expenav1">Summary</DIV><DIV class="singlenav" id="expenav2">(1) DNA synthesis</DIV><DIV class="singlenav" id="expenav3">(1) Transformation</DIV><DIV class="singlenav" id="expenav4">(1) Expression</DIV><DIV class="singlenav" id="expenav5">(2) CAS plates</DIV><DIV class="singlenav" id="expenav6">(2) Quantitative CAS</DIV><DIV class="singlenav" id="expenav7">(2) LC-MS</DIV></DIV></DIV><DIV id="upbtndiv"><DIV id="upbtnword">Back To Top</DIV></DIV><DIV id="homebtndiv"><DIV id="homebtnword">Home Page</DIV></DIV><DIV id="phonebtndiv"><DIV id="phonebtnword">Contact</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>