Team:USTC/Design
Modeling
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Navigation
A Prion's Life
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Design
Together we stand
Leaders
Ya Jiang
Yin Wu
Kaiyue Ma
Chengle Zhang
Xuefeng Meng
Introduction of Sup35
Sup35 is one kind of yeast prions which is found in [PSI+] Saccharomyces cerevisiae. [PSI+] is the prion phenotype which causes nonsense suppression of all three types of stop codons.
As we know, Prion-forming ability is not conserved, revealing that it is a disease. Prion domains have non-prion functions, explaining some conservation of sequence. Yeast prions [PSI] and [URE3] induce a cellular stress response (Hsp104 and Hsp70 induction), suggesting the cells are not happy about being infected[1]. Under three quarters of growth conditions tested that a difference between the two states of protein [PSI+] and [psi-] existed, if the protein are in the state [psi-], the cells are healthier[2]. The yeast prion are much milder than animal prion and the huge difference in the genome sequences between human beings and yeast creates barriers to the infection from yeast to human beings.
The [PSI+] has a strong ability to propagate by inducing the newly produced Sup35 to switch to the same state as [PSI+], resulting in the aggregation[3]. Therefore the chance of reading through the stop coden or the nonsense mutation enhanced, forming a new phenotype[4]. The acatastatic read-through change the physicochemical properties of the mRNA and protein, and then the stability of the mRNA and the function of the protein are changed[5]. Because of the sequence diversity downstream the stop coden, the uncertanity of phenotypes is remarkable in the hosts. These phenotypes are heritable when the prion comes into progeny from cytoplasm[6],[7],[8].
Sup35 is a 685-amino-acid protein, which contains three regions that are distinguished by their function and amino acid composition: the amino-terminal region Sup35N(amino acids 1 to 123), the middle region Sup35M(amino acids 124 to 253) and the carbon-terminal region Sup35C(amino acids 254 to 685)[9].
[Figure 1] Crystal structure of Sup35(PDB code: 1R5B)
[Crystal structure and functional analysis of the eukaryotic class II release factor eRF3 from S. pombe             Kong, C., Ito, K., Walsh, M.A., Wada, M., Liu, Y., Kumar, S., Barford, D., Nakamura, Y., Song, H.           (2004) Mol.Cell 14: 233-245 ]
The character of Sup35 is importantly decided by the character of the amino acids sequence which is quite different from other normal proteins. Sup35N is unusually rich in  Gln(28%) and Asn(16%) resides, but has few aliphatic amino acids. By comparison, the average protein has just 9% Gln and Asn and 29% aliphatic residues. Sup35N also contains several imperfect oligopeptide PQGGYQQ_YN repeats that are similar in character to PHGGGWGQ repeats present in the mammalian prion protein, PrP. These repeats are the only immediately obvious similarity between Sup35 and PrP protein sequences. Sup35M is highly charged, unlike the rest of the protein. Forty-one percent of the residues in this region are lysine, glutamic acid, or aspartic acid. SupN is sufficient to support yeast prion induction and propagation, and Sup35M plays a role in [PSI+] stability and Sup35 solubility. So the Sup35NM is the prion-determining region(PrD) which enables the proteins to act as prions[9].
Sup35 has two states: nonprion and prion states, and they can transform into each other. When it's in its nonprion state, Sup35 is the translation release factor of Saccharomyces cerevisiae which binds to Sup45 and ends the process of translation.
[Figure 2] Sup35 as translation release factor
When Sup35 is in prion state, it can propagate and transform nonprion-state Sup35 into prion state, so it can lead to a chain reaction and many yeast prions aggregate to form amyloid fiber. The gathering of Sup35 is an important property that can be used in our project.
Sup35's gathering is with the help of Heat Shock Proteins (HSP). HSP is family of proteins that are produced by cells in response to exposure to stressful conditions, like hest shock[10].
[Figure 3] The function of HSP104 on Sup35 assembling
When the yeast is exposed to heat shock, HSP104 will be produced. HSP104 can convert soluble nonprion-state Sup35 into prion-state Sup35 that nucleates over time. This nucleus then rapidly seeds further conversion and form amyloid fiber[11]. However, overexpression of Hsp104 resolves Sup35, eliminating [PSI+]. It implies that Sup35 aggregates at a temperature range. And we find this range, around 37℃~42℃[12]. What's more, protein denaturant Guanidine Hydrochloride(GDNHCL) can scatter aggregated Sup35, eliminating [PSI+][13]. So Sup35 is controlled by both heat shock and GDNHCL.
Thus, Sup35 has such three important properties: assembling, thermal control and reversibility. We take a fancy to these properties and plan to do something with that. We use the prion-determining region of Yeast Prion Sup35, that is Sup35NM, to construct an open tool box. We link Sup35NM to a gene of interest so that this expressed fusion protein gets the property of yeast prions. The gene linked to Sup35NM can be any genes of interest which are available in our system, so this tool box can be used to realize many functions.
References
[1] Wickner RB. [URE3] as an altered URE2 protein: evidence for a prion analog in S. cerevisiae. Science 1994; 264:566-9.
[2] True HL, Lindquist SL. A yeast prion provides a mechanism for genetic variation and phenotypic diversity. Nature 2000; 407:477-83.
[3] Sparrer HE, Santoso A, Szoka FC Jr, Weissman JS (2000) Evidence for the prion hypothesis: Induction of the yeast [PSI+] factor by in vitro- converted Sup35 protein. Science 289(5479):595–599.
[4] Chernoff YO (2004) Amyloidogenic domains, prions and structural inheritance: rudiments of early life or recent acquisition? Curr Opin Chem Biol 8: 665–671.
[5] Von der Haar T, Tuite MF (2007) Regulated translational bypass of stop codons in yeast. Trends Microbiol 15: 78–86.
[6] Eaglestone SS, Cox BS, Tuite MF (1999) Translation termination efﬁciency can be regulated in Saccharomyces cerevisiae by environmental stress through a prion-mediated mechanism. EMBO J 18: 1974–1981.
[7] Nakayashiki T, Kurtzman CP, Edskes HK, Wickner RB (2005) Yeast prions [URE3] and [PSIþ] are diseases. Proc Natl Acad Sci U S A 102: 10575–10580.
[8] Chernoff YO, Galkin AP, Lewitin E, Chernova TA, Newnam GP, et al. (2000) Evolutionary conservation of prion-forming abilities of the yeast Sup35 protein. Mol Microbiol 35: 865–876.
[9] Uptain S M, Lindquist S. Prions as protein-based genetic elements[J]. Annual Reviews in Microbiology, 2002, 56(1): 703-741.
[10] Ritossa F (1962). "A new puffing pattern induced by temperature shock and DNP in drosophila". Experientia 18 (12): 571–573. doi:10.1007/BF02172188. ISSN 0014-4754. Retrieved 2014-04-27.
[11] McGuire J. Prions: in [PSI+] from yeast[J]. Eukaryon, 2005, 1(1): 8.
[12] Johnny M. Tkach and John R. Glover. Amino Acid Substitutions in the C-terminal AAA+ Module of Hsp104 Prevent Substrate Recognition by Disrupting Oligomerization and Cause High Temperature Inactivation. The Journal of Biological Chemistry. Vol. 279, No. 34, Issue of August 20, pp. 35692-35701, 2004.
[13] Ferreira PC, Ness F, Edwards SR, Cox BS, Tuite MF. 2001. The elimination of the yeast [PSI+] prion by guanidine hydrochloride is the result of Hsp104 inactivation. Mol. Microbiol. 40:1357–69.
Propri-ontein
Introduction of Yeast two-hybrid system
Our first gene circuit is based on Yeast two-hybrid system.
Yeast two-hybrid system(Y2H) is a molecular biology technique used to discover protein–protein interactions (PPIs)[1] and protein–DNA interactions[2],[3] by testing for physical interactions (such as binding) between two proteins or a single protein and a DNA molecule, respectively.
The premise behind the test is the activation of downstream reporter gene(s) by the binding of a transcription factor onto an upstream activating sequence (UAS). For yeast two-hybrid system, the transcription factor is split into two separate fragments, called the binding domain (BD) and activating domain (AD). The BD can bind to the UAS and the AD can activate the transcription of downstream reporter gene. The Y2H is thus a protein-fragment complementation assay.
[Figure 1] Yeast two-hybrid system
History
Y2H was originally designed to detect protein–protein interactions using the GAL4 transcriptional activator of the yeast Saccharomyces cerevisiae. The GAL4 protein activated transcription of a protein involved in galactose utilization, which formed the basis of selection[4]. Since then, the same principle has been adapted to describe many alternative methods, including some that detect protein–DNA interactions or DNA-DNA interactions, as well as methods that use Escherichia coli instead of yeast[3]. The key to the Y2H is that in most eukaryotic transcription factors, the activating and binding domains are modular and can function in proximity to each other without direct binding[5]. This means that even though the transcription factor is split into two fragments, it can still activate transcription when the two fragments are indirectly connected.(https://en.wikipedia.org/wiki/Two-hybrid_screening#cite_note-young-1)
Application of Y2H
In application, this system often utilizes a genetically engineered strain of yeast in which a reporter gene (GFP, nutrient genes or others) is inserted, which can be only activated by certain transcriptional activator. This activator can be split into two fragments: DNA-binding domain (BD) fragment and activation domain (AD) fragment. Two plasmids are engineered to produce a protein product in which the BD fragment is fused onto a protein while another plasmid is engineered to produce a protein product in which the AD fragment is fused onto another protein. The protein fused to the BD may be referred to as the bait protein, and is typically a known protein the investigator is using to identify new binding partners. The protein fused to the AD may be referred to as the prey protein and can be either a single known protein or a library of known or unknown proteins.
If the bait and prey proteins interact (bind), then the AD and BD of the transcription factor are indirectly connected, bringing the AD in proximity to the transcription start site and transcription of reporter gene(s) can occur. If the two proteins do not interact, there is no transcription of the reporter gene. In this way, a successful interaction between the fused protein is linked to a change in the cell phenotype[1].
The reporter genes are usually GFP, nutrient genes and others. If using GFP, when AD binds to BD, GFP will express to show green fluorescence. If using nutrient gene, this system often utilizes a genetically engineered strain of yeast in which the biosynthesis of certain nutrients (usually amino acids or nucleic acids) is lacking. When grown on media that lacks these nutrients, the yeast fail to survive. So only when AD binds to BD can the nutrient be expressed so that the yeast can survive.
Yeast two-hybrid system in iGEM
Y2H has been used in iGEM for several times:
iGEM12_TU_Munich uses Y2H to construct a yeast light-switchable promoter system based the protein interaction between phytochrome B and phytochrome interacting factor 3.
iGEM15_HUST-China uses the interaction of CIB1, a basic helix-loop-helix (bHLH) protein and cryptochrome 2 (CRY2), a blue light stimulated photoreceptor, to regulate DNA transcription by blue light.
The Tianjin iGEM09 team utilizes yeast Two-hybrid system to detect Microcystins(MCs, induce liver cancer) in waters .
iGEM Göttingen 2014 uses Y2H screen to find the interaction partners(peptides) of their surface proteins.
All those show that yeast two-hybrid system has many applications in synthetic biology. This year, our team introduce yeast prion to this system, and we will see the wonderful cooperation among these elements.
propri-ontein in yeast
[Figure 2] propri-ontein
In this circuit, we combine the prion-determining region (PrD) of Sup35 and the AD, BD fragments in Y2H system. We have designed two fusion proteins: AD-PrD and BD-PrD. AD-PrD is inserted into Uracil labeled plasmid YEplac195, and BD-PrD is inserted into Tryptophan labeled plasmid PYeScGAp. For our downsteam reporter gene, we insert pGAL1 promoter and GFP into Leucine labeled plasmid YEplac181.
We predict the structure of our fusion protein: PrD-AD and PrD-BD by Phyre2[6].
[Figure 3] strcuture of PrD-AD(by Phyre)]
[The Phyre2 web portal for protein modeling, prediction and analysis
Kelley LA et al.
Nature Protocols 10, 845-858 (2015).]
[Figure 4] structure of PrD-BD(by Phyre)
[The Phyre2 web portal for protein modeling, prediction and analysis
Kelley LA et al.
Nature Protocols 10, 845-858 (2015).]
Then we transform these three plasmids into Saccharomyces cerevisiae. We expect this designded system is sensitive to temperature and concentration of protein denaturant GdnHCl. When fusion proteins AD-PrD and BD-PrD are produced in yeast, as Sup35 has some good properties (assembling, thermal control and reversibility), the fusion protein PrD-AD and PrD-BD also have such properties. When we give heat shock to yeast, because of the aggregation of prion, fusion protein AD-PrD-PrD-BD can be generated to activate downstream gene expression. However, protein denaturant Guanidine Hydrochloride (GdnHCL) can inhibit the aggregation of prion, and the expression of downstream gene. So we will realize the thermal control and GdnHCL control of gene expression. When the temperature is high enough to give yeast heat shock, expression of reporter gene is induced,showing green fluorescence. On the contrary, if our rebuilt yeasts exist in relatively low temperature, green fluorescence is invisible.
Propri-ontein looks like an open tool box, because the downstream gene can be any genes of interest and we can use this circuit to control the expression of many genes by giving it heat shock or GdnHCL. In our experiments, we use GFP as downstream reporter gene to test the function of our circuits.
References
[1] Young K H. Yeast two-hybrid: so many interactions, (in) so little time. [J]. Biology of Reproduction, 1998, 58(2):302-11.
[2] Joung J K, Ramm E I, Pabo C O. A bacterial two-hybrid selection system for studying protein-DNA and protein-protein interactions[J]. Proceedings of the National Academy of Sciences.
[3] Jessica A. Hurt, Stacey A. Thibodeau, Andrew S. Hirsh, Carl O. Pabo, J. Keith Joung. Highly specific zinc finger proteins obtained by directed domain shuffling and cell-based selection[J]. Proceedings of the National Academy of Sciences of the United States of America, 2003, 100(21):12271-6.
[4] Fields S, Song O. A novel genetic system to detect protein-protein interactions.[J]. Nature, 1989, 340(6230):245-6.
[5] Verschure P J, Visser A E, Rots M G. Step out of the Groove: Epigenetic Gene Control Systems and Engineered Transcription Factors[J]. Advances in Genetics, 2006, 56(56):163-204.
[6] The Phyre2 web portal for protein modeling, prediction and analysis           Kelley LA et al. Nature Protocols 10, 845-858 (2015).
Pro Priontein
Design
[Figure 1] Pro Priontein
This circuit derives from GFP splitting and protein detection technique. GFP can be split into two fragments: GFP1-10 and GFP11. GFP11 is linked to an X protein. When two fragments connect to each other, green florescence is visible; when the X protein is inaccessible, green florescence will disappear[2]. However, existing variants of GFP often misfold when expressed as fusions with other proteins, so we decide to use a robustly folded version of GFP, “superfolder” GFP, which can fold well even when fused to poorly folded polypeptides[6].
We use the plasmid designed by group iGEM07_Cambridge (2008) which containing superfolder GFP (sfGFP) and split it into two parts, sfGFP1-10 and sfGFP11. In this circuit, sfGFP11 is linked to PrD (prion-determining region). Prion aggregates at a certain temperature range which is around 37℃~42℃[7] and protein denaturant Guanidine Hydrochloride (GdnHCL) can eliminate the aggregation[8]. So it’s easy to imagine that green fluorescence is invisible when heat shock is added because of prion assembling, and visible when Guanidine Hydrochloride is added.
Thus, this circuit can be used as a biological temperature control kill switch and a biological temperature indicator (BTI). We also hope this circuit can be used in scientific research. If a certain protein A can be split into two fragments A1 and A2, which can spontaneously bind to each other and perform function, then we can use this circuit to control the function of this protein in the protein level. It may be a useful tool in protein engineering.
What is GFP & How to make use of it
I wonder the one who is looking at this website must know about the green fluorescent protein (GFP), anyway, I will make a brief introduction.
GFP is a protein composed of 238 amino acid residues (26.9 kDa) which exhibits bright green fluorescence when exposed to light in the blue to ultraviolet range. The kind of GFP scientists usually use traditionally refers to the protein first isolated from the jellyfish Aequorea victoria. In cell and molecular biology, the GFP gene is frequently used as a reporter of expression (so it is in our project!). Scientists Roger Y. Tsien, Osamu Shimomura, and Martin Chalfie were awarded the 2008 Nobel Prize in Chemistry for their discovery and development of the green fluorescent protein[9].
Normal application of GFP is introducing the GFP gene to animals or other species and then measuring its expression in a given organism, in selected organs, or in cells of interest. Based on that, here we create a novel usage of GFP, splitting and reassembling.
GFP splitting
If we split GFP into two parts, it’s of high possibility that both parts won’t show bright green fluorescence, and if they can reunite into a “intact” GFP in some conditions, the fluorescence will appear. That is our primary idea. We are surprised to find that several research groups have been working on projects related to that, thanks to their experience and achievements, we can quickly get our idea started.
GFP splitting has two roles in modern bioresearch, one is detecting protein solubility, and the other is defining cell contacts and synapses in living nervous systems.
[Figure 2] GFP splitting & protein detection
GFP is first split into two frgments, and to have a better result, two fragments are not of the same size. A protein of interest (X) is fused to the smaller GFP fragment via a flexible linker. The complementary GFP fragment is expressed separately. Neither fragment alone is fluorescent. When mixed, the small and large GFP fragments spontaneously associate, resulting in GFP folding and formation of the fluorophore. Processes that make the small GFP tag inaccessible, such as misfolding or aggregation, which often happens when X is insoluble, can prevent complementation, thus there’s still no fluorescence[2].
[Figure 3] membrane contacts and synapses
Scientists have developed a system to label membrane contacts and synapses between two cells in living animals based on the proximity of the presynaptic and the postsynaptic plasma membranes, it is called GRASP (GFP Reconstitution Across Synaptic Partners). When the complementary fragments of GFP are fused to ubiquitous transmembrane proteins, GFP fluorescence appears uniformly along membrane contacts between the two cells. When one or both GFP fragments are fused to synaptic transmembrane proteins, GFP fluorescence is tightly localized to synapses. GRASP has the potential to greatly increase the ease of synaptic mapping, not only in simple and straightforward systems, but also in more complex systems, such as vertebrate brains, because GRASP should be immediately applicable in dissociated cells or slice cultures[3].
GFP reassembling
The two steps of using GFP in a novel way is splitting and reassembling, so if we can’t figure out whether the two fragments of GFP will self-reassemble or under what condition they will reassemble, we can’t start the experiment.
[Figure 4] GFP(PDB code: 1GFL)[4]
[The molecular structure of green fluorescent protein.             Yang, F., Moss, L.G., Phillips Jr., G.N.           (1996) Nat.Biotechnol. 14: 1246-1251].
To get the best effect of reassembling, we need to know the structure of GFP first and find out the best splitting point. The picture above shows the structure of the Aequorea victoria green fluorescent protein. According to that, GFP has a beta barrel structure consisting of eleven β-strands, with an alpha helix containing the covalently bonded chromophore 4-(p-hydroxybenzylidene)imidazolidin-5-one (HBI) running through the center. Five shorter alpha helices form caps on the ends of the structure[1].
So it will be beneficial to the reassembling when the splitting point, also the junction point corresponds to loops or turns between β-strands. Previous experiments done by Cabantous S, Terwilliger T C and Waldo G S showed that one feasible pair of GFP fragments are amino acids 1-214 and 214-238. To further reduce the size of the small GFP fragment, they tested 1–214 (GFP 1–10) for complementation with 214–230 (GFP 11), eliminating the residues 231-238 from the small fragment[2]. Their experiments results show that these two fragments can bind with each other spontaneously, which is really delight news for our project.
GFP --> sfGFP
At this point, we start our experiment to acquire two GFP fragments from complete GFP, simultaneously, we continue reading papers and learning experience from others. Fortunately, we find a variant of GFP, superfolder GFP (sfGFP), which gives us great help in the following experiments. sfGFP carries the following amino acid changes with respect to mut3 GFP (E0040), the currently most commonly used GFP in iGEM registry: S30R, Y39N, F64L, G65T, F99S, N105T, Y145F, M153T, V163A, I171V, A206V[5].
[Figure 5] sfGFP(PDB code: 2B3P)[9]
[Engineering and characterization of a superfolder green fluorescent protein.             Pedelacq, J.D., Cabantous, S., Tran, T., Terwilliger, T.C., Waldo, G.S.           (2006) Nat.Biotechnol. 24: 79-88].
Existing variants of green fluorescent protein (GFP) often misfold when expressed as fusions with other proteins (this is exactly we are going to do in our project), while a robustly folded version of GFP, called sfGFP, can fold well even when fused to poorly folded polypeptides. sfGFP shows improved tolerance of circular permutation, greater resistance to chemical denaturants and improved folding kinetics[6]. Other properties of sfGFP is similar with GFP, so as we planned before, we decide to split sfGFP into sfGFP1-10 (1-214) and sfGFP11 (214-230), then connect sfGFP11 with PrD. The protein structure we simulated are shown below.
[Figure 6] sfGFP1-10(by PyMOL)
[The Phyre2 web portal for protein modeling, prediction and analysis
Kelley LA et al.
Nature Protocols 10, 845-858 (2015)].
[Figure 7] PrD+sfGFP11(by PyMOL)
[The Phyre2 web portal for protein modeling, prediction and analysis
Kelley LA et al.].
Acquisition of sfGFP1-10 and sfGFP11
Finally we make our experiment plan and set out to acquire two sfGFP fragments: sfGFP1-10 and sfGFP11. The gene sequence of sfGFP1-10 is the same as 1-642 of sfGFP sequence, so after getting the pSB1C3 plasmid containing the gene of sfGFP (Part BBa_I746916) in 2016 Kit Plate 6, we get sfGFP1-10 by PCR. First we want to get sfGFP11 in the same way as sfGFP1-10, however, the gene sequence of sfGFP11 contains several mutations from sfGFP, also we need to add a linker (a flexible polypeptide chain) to joint sfGFP11 and PrD (prion-determining region), so we decide to acquire sfGFP11, together with the linker directly by gene synthesis.
Pro Priontein in yeast
We design and construct a two-plasmid system in yeast. Fusion protein sfGFP11-PrD is inserted into Tryptophan labeled plasmid PYeScGAP and sfGFP1-10 is inserted into Uracil labeled plasmid YEplac195. We expect this system is sensitive to temperature and concentration of GdnHCl. Contrary to Propri-ontein, this system performs function in opposite way. When our rebuilt yeasts exist in relatively low temperature, two fragements of sfGFP can bind to each other spontaneously because of non-aggregation state of PrD. On the contrary, if we give it heat shock, green fluorescence is invisible for the aggregation of Sup35. On the other hand, introduction of GdnHCl eliminates [PSI+], inhibiting the aggregation of Sup35. Thus green fluorescence is visible even when the system is heat shocked.
Like Propri-ontein system, this Pro Priontein system can also be an open tool box, because the sfGFP can be replaced by other proteins of interest and we can use this circuit to control the function of these proteins by giving it heat shock or GdnHCL.
References
[1] https://en.wikipedia.org/wiki/Green_fluorescent_protein. Wikipedia: Green fluorescent protein.
[2] Cabantous S, Terwilliger T C, Waldo G S. Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein[J]. Nature biotechnology, 2005, 23(1): 102-107.
[3] Feinberg E H, VanHoven M K, Bendesky A, et al. GFP Reconstitution Across Synaptic Partners (GRASP) defines cell contacts and synapses in living nervous systems[J]. Neuron, 2008, 57(3): 353-363.
[4] http://www.rcsb.org/pdb/explore/explore.do?structureId=1GFL.
[5] http://parts.igem.org/Part:BBa_I746916. iGEM part: BBa_I746916.
[6] Pédelacq J D, Cabantous S, Tran T, et al. Engineering and characterization of a superfolder green fluorescent protein[J]. Nature biotechnology, 2006, 24(1): 79-88.
[7] Tkach J M, Glover J R. Amino acid substitutions in the C-terminal AAA+ module of Hsp104 prevent substrate recognition by disrupting oligomerization and cause high temperature inactivation[J]. Journal of Biological Chemistry, 2004, 279(34): 35692-35701.
[8] Ferreira P C, Ness F, Edwards S R, et al. The elimination of the yeast [PSI+] prion by guanidine hydrochloride is the result of Hsp104 inactivation[J]. Molecular microbiology, 2001, 40(6): 1357-1369.
[9] http://www.rcsb.org/pdb/explore/explore.do?structureId=2B3P.
Contact us
e-mail
Facebook
Weibo
Sponsors
Designed by 2016 iGEM Team:USTC
Under CC License
Based on Semantic-UI
