Team:USTC/Description
Modeling
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Navigation
A Prion's Life
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Description
See our project in detail
Leader
Yin Wu
Motivation – Start from Here
When talking about prions, an extremely dangerous pathogen will come to most people’s mind. That’s true, because prions usually cause serious diseases such as bovine spongiform encephalopathy (mad cow disease) in bovine and Creutzfeldt–Jakob disease in human beings. It sounds horrible, but actually this is what attracts us most and where we begin our project.
How do these diseases initiate and develop by the infection of such simple proteins?
Prion proteins have two states: normal folding state PrPc and misfolding state PrPsc, it’s known that PrPc can misfold to transform into PrPsc.     And then, PrPsc can go on to convert more PrPc into PrPsc, leading to a chain reaction resulting in large amounts of PrPsc. These PrPsc can polymerize into an aggregate called amyloid fiber. Finally, clumps of high polymer form, causing a series of diseases.           Without any change in genes, prions achieve the goal of changing its structure and function. This ability allows prions to control the host cell in a more direct way, skipping long and complicated processes of transcription and translation. So it’s natural to imagine that, if scientists can regulate the transformation between PrPc and PrPsc, it’s of high possibility for them to develop a method with which they can control the host cell or the function of other proteins at protein level.
However, high pathogenicity of prions prevent people from using it in experiments. Luckily, several yeast prions have been found in recent sicentific reseaches. These proteins are harmless to animals, so can be used in our experiments. Sup35 is one kind of yesat prions which is found in [PSI+] Saccharomyces cerevisiae. [PSI+] is the prion phenotype which causes nonsense suppression of all three types of stop codons. Sup35 is the translation release factor of Saccharomyces cerevisiae,           which has three domains: N, M and C. Among them, NM domain is the Prion-Determining Region (PrD), which decides the property of yeast prion. It’s incredible that when PrD is fused with another protein, the fusion protein behaves just like prions. We’d like to describe it as a mysterious power with which one can determine the structure and function, in other words, destiny of a given protein.
According to some published article, Sup35 has several properties. It aggregates at a certain temperature range which is around 37℃~42℃, and protein denaturant Guanidine Hydrochloride (GdnHCL) can eliminate the aggregation. So by changing temperature and concentration of GdnHCL, one can decide whether or not Sup35 aggregates.
Circuits – Make the Most of PrD of Sup35
In traditional Chinese thought, everything is in the balance of Yin and Yang, that is to say, it can show two opposing characters. When reaches extreme in one side, it will turn to the opposite side. Aggregation is the most salient character of Sup35, so it must has the opposite character, separation. Based on this seemingly weird thought, two circuits are designed, one shows aggregation, and the other shows separation.
The first circuit is based on Yeast two-hybrid system. In this system, BD is binding domain and fused with X protein, AD is activating domain and fused with Y protein. When X binds to Y, the final fusion protein can activate the expression of downstream gene.
[Figure 1] The Yeast two-hybrid system(Y2H)
This is our first circuit shown in figure 2. The Prion-Determining Region, shown as PR, replace the protein X and Y in Y2H. We give it heat shock and fusion protein can be generated to activate downstream gene. Meanwhile, GdnHCL can inhibit the formation of this fusion protein, and inhibit the expression of downstream gene (GFP in our experiments). Reporter genes can be any genes of interest, so we can use this circuit to realize many functions by controlling the expression.
[Figure 2] Propri-ontein
Our second circuit derives from GFP splitting and protein detection technique. GFP can be split into two fragments: GFP1-10 and GFP11. GFP11 is linked to protein X. When two fragments of GFP connect to each other, green fluorescence is visible. When protein X is inaccessible, for example, aggragating, green fluorescence will disappear. To avoid misfolding of GFP when expressed as a fusion with other protein, we use a robustly folded version of GFP, called “superfolder” GFP (sfGFP), that folds well even when fused to poorly folded polypeptides.
[Figure 3] GFP splitting & protein detection
Then we construct circuit 2 like shown in figure 4, sfGFP11 is linked to PrD. In this circuit, green fluorescence is invisible when heat shock is added because of Prions assembling, and visible when GdnHCl is added. Similar to circuit 1, sfGFP can be altered by any genes of interest.
[Figure 4] Pro Priontein
We name the first circuit 'Propri-ontein', which means connecting two proteins (AD and BD) by prion, in other words, aggregation; and the second circuit 'Pro Priontein', which means stopping the reassembling of two proteins by prion, in other words, separation.
Achievements – Enjoy What We Do
With these two circuits (systems), controlling proteins in a quick and direct way at protein level, won’t be only a dream. Since PrD is sensitive to temperature, both circuits can be used as biological temperature control switches and biological temperature indicators.
With modeling as an useful tool (temperature and concentration of GdnHCL as two variables), quantitative purposes can be reached. In Propri-ontein system, the expression level of reporter protein can be regulated quantitatively. In Pro priontein system, the amount of reassembled protein can also be regulated quantitatively.
We are confident of and looking forward to the practical utilizations of our systems in future industry field and lab work!
Contribution – Improvement to Parts
Part: BBa_J63006 Our part is Part: BBa_K2009363 ,designed by Kaiyue Ma. We construted this composite by ligating GAL1 promoter + Kozak sequence (Part: BBa_J63006 ) and GFP (Part: BBa_E0040 ). However, the ATG inside the KOZAK sequence won't be in the same reading frame as the ATG of the downstream coding sequence, so a frame-shift mutation is inevitable. To solve this problem, we add a base pair after the Kozak sequence so that the ATG can be in the same reading frame and the GFP can be expressed properly. What’s more, this part become “ready to use”, which means the GFP sequence can be directly altered by other functional parts and the sequence will be expressed properly.
Part: BBa_K1739000 Our part is Part: BBa_K2009357 .This part's designer is Kaiyue Ma.The Prion-determining region of Sup35 we used is originally provided by Dong Men, PhD of Wuhan Insititue of Virology of Chinese Academy of Sciences, and the sequence is not quite the same as the existing sequence in the Parts Registry, for a lot of mutations have been done. The standardization of this gene is completed by ourselves by adding the Biobrick prefix and suffix to its ends and the introduction of site mutation to eliminate PstI cutting site inside it. Compared to the sequence submitted by other teams, our Sup35 gene is shorter, which means lower expressing pressure and the possibility to express more protein in E.coli.
References and Sources
[1] Wickner RB. [URE3] as an altered URE2 protein: evidence for a prion analog in S. cerevisiae. Science 1994; 264:566-9.
[2] True HL, Lindquist SL. A yeast prion provides a mechanism for genetic variation and phenotypic diversity. Nature 2000; 407:477-83.
[3] Sparrer HE, Santoso A, Szoka FC Jr, Weissman JS (2000) Evidence for the prion hypothesis: Induction of the yeast [PSI+] factor by in vitro- converted Sup35 protein. Science 289(5479):595–599.
[4] Chernoff YO (2004) Amyloidogenic domains, prions and structural inheritance: rudiments of early life or recent acquisition? Curr Opin Chem Biol 8: 665–671.
[5] Von der Haar T, Tuite MF (2007) Regulated translational bypass of stop codons in yeast. Trends Microbiol 15: 78–86.
[6] Eaglestone SS, Cox BS, Tuite MF (1999) Translation termination efﬁciency can be regulated in Saccharomyces cerevisiae by environmental stress through a prion-mediated mechanism. EMBO J 18: 1974–1981.
[7] Nakayashiki T, Kurtzman CP, Edskes HK, Wickner RB (2005) Yeast prions [URE3] and [PSIþ] are diseases. Proc Natl Acad Sci U S A 102: 10575–10580.
[8] Chernoff YO, Galkin AP, Lewitin E, Chernova TA, Newnam GP, et al. (2000) Evolutionary conservation of prion-forming abilities of the yeast Sup35 protein. Mol Microbiol 35: 865–876.
[9] https://en.wikipedia.org/wiki/Prion
[10] Uptain S M, Lindquist S. Prions as protein-based genetic elements[J]. Annual Reviews in Microbiology, 2002, 56(1): 703-741.
[11] McGuire J. Prions: in [PSI] t from yeast[J]. Eukaryon, 2005, 1(1): 8.
[12] Ferreira P C, Ness F, Edwards S R, et al. The elimination of the yeast [PSI+] prion by guanidine hydrochloride is the result of Hsp104 inactivation[J]. Molecular microbiology, 2001, 40(6): 1357-1369.
[13] Tkach J M, Glover J R. Amino acid substitutions in the C-terminal AAA+ module of Hsp104 prevent substrate recognition by disrupting oligomerization and cause high temperature inactivation[J]. Journal of Biological Chemistry, 2004, 279(34): 35692-35701.
[14] https://en.wikipedia.org/wiki/Two-hybrid_screening
[15] Cabantous S, Terwilliger T C, Waldo G S. Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein[J]. Nature biotechnology, 2005, 23(1): 102-107.
[16] Pédelacq J D, Cabantous S, Tran T, et al. Engineering and characterization of a superfolder green fluorescent protein[J]. Nature biotechnology, 2006, 24(1): 79-88.
[17] http://www.dxy.cn/bbs
[18] https://en.wikipedia.org/wiki/Sup35p
[19] Feinberg E H, VanHoven M K, Bendesky A, et al. GFP Reconstitution Across Synaptic Partners (GRASP) defines cell contacts and synapses in living nervous systems[J]. Neuron, 2008, 57(3): 353-363.
[20] Cabantous S, Waldo G S. In vivo and in vitro protein solubility assays using split GFP[J]. Nature methods, 2006, 3(10): 845-854.
[21] Patterson G H, Knobel S M, Sharif W D, et al. Use of the green fluorescent protein and its mutants in quantitative fluorescence microscopy[J]. Biophysical journal, 1997, 73(5): 2782.
[22] Ritossa F. A new puffing pattern induced by temperature shock and DNP in Drosophila[J]. Experientia, 1962, 18(12): 571-573.
[23] Stemmer W P. DNA shuffling by random fragmentation and reassembly: in vitro recombination for molecular evolution[J]. Proceedings of the National Academy of Sciences, 1994, 91(22): 10747-10751.
Contact us
e-mail
Facebook
Weibo
Sponsors
Designed by 2016 iGEM Team:USTC
Under CC License
Based on Semantic-UI
