Team:USTC/Notebook/7 15
Modeling
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Navigation
A Prion's Life
Home
Project
Description Design Proof of Concept Demonstrate Results Notebook Spin-off
Parts
Basic Parts Composite Parts Part Collection
Safety Attributions
Human Practices
Silver Gold Integrated Practices Engagement
Awards
Model
Team
Members Sponsor About USTC Acknowledgements Gallery Collaborations
Notebook
Together we stand
Performers
Everybody
Date: 7.15
Recorder：Kaiyue Ma
Agarose gel electrophoresis of pGAD-T7 and pGBK-T7
Material:           The plasmids were get from Plasmid Extraction.
Add 1 g agarose to 100 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 1 μL 6× loading buffer to 5 μL plasmids of each tube.
Loading：DNA marker 6 μL（Trans 2K Plus II）, sample(*6) 6 μL. From left to right: AD1, AD2, AD3, AD4, BD1, BD2, BD3, BD4.
Electrophoresis gel：110 V 30 min.
Dissolve about 2 μL Gelred to a container with TAE buffer and put the agarose gel in.
Autoradiography(UV).
Result
Recorder:Tianshu Liu Transformation of TEF2 1.  Find the pSB1A3 plasmid containing the gene of TEF2 yeast constitutive promoter (Part Name BBa_K165037) in 21H well of 2016 Kit Plate 3.             2.  Punch a hole through the foil cover into the corresponding well by a pipette tip. (Do not remove the foil cover, as it could lead to cross contamination between the wells.)             3.  Pipette 10 µL of ddH2O (distilled water) into the well. Pipette up and down a few times and lett for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension is red, as the dried DNA has cresol red dye.             4.  Put competent cells on ice. Then pre-chill by placing the tubes on ice.             5.  Pipet 1 µL of plasmid into one of the competent cell tubes and pipet 2 µL of plasmid into the other competent cell tube.             6.  Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.             7.  Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.             8.  Immediately transfer the tubes back to ice, and incubate on ice for 2 minutes. This helps the cells recover.             9.  Add 900 µL of SOC media per tube, and incubate at 37°C for 2 hours.             10. Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.             11. Discard 600 μL supernatant liquid and resuspend the bacteria.           12. Coat plate: add two tubes of solution (400 μL each) in two 1/1000 Cam plates and spread it, respectively, then cultivate it overnight.
Recorder：Xuefeng Meng、Jianyi Wang PCR of AD and BD
Experimental materials 1. Plasmid: pGAD-T7 and pGBK-T7(5 μL for each) from laboratory 436 of USTCSLS, Mr Liao;             2. Primer: AD 1, AD 2, BD 1, BD 2. Designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μL). All bought from TaKaRa, Code No. R010A.
About primers
primer
AD 1
AD 2
BD 1
BD 2
sequence
5'-GAATTCGCGGCC
GCTTCTAGATGG
CCAATTTTAATC
AAAGTGGGAATA
TTGCTG-3'
5'-CTGCAGCGGCCG
CTACTAGTACTA
CTCTTTTTTTGG
GTTTGGTGGGGT
ATC-3'
5'-GAATTCGCGGCC
GCTTCTAGATGA
TGAAGCTACTGT
CTTCTATCGAAC-3'
5'-CTGCAGCGGCCG
CTACTAGTACTA
CGATACAGTCAA
CTGTCTTTGACC-3'
lenth (nt)
54
51
48
48
GC%
44.44
50.98
47.92
52.08
molecular weight (Da)
16690.78
15716.06
14733.50
14663.45
Tm (℃)
71.26
73.40
71.53
73.24
Procedure:
1.Prepare 6 PCR tubes and sequentially add：
sample
AD*3
BD*3
Sterilized ddH2O
33 μL
33 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
10 μL
dNTP Mixture (2.5 mM each)
4 μL
4 μL
Template(plasmid pGAD-T7)
0.5 μL
0
Template(plasmid pGBK-T7)
0
0.5 μL
AD Primer-1
1 μL
0
AD Primer-2
1 μL
0
BD Primer-1
0
1 μL
BD Primer-2
0
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
0.5 μL
total
50 μL
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
55
15 s
step 4
72
30 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder：Yin Wu Transformation of Superfolder GFP
Find the pSB1C3 plasmid containing the gene of superfolder GFP(Part Name BBa_I746916) in 9A well of 2016 Kit Plate 6.
With a pipette tip, punch a hole through the foil cover into the corresponding well.(Do not remove the foil cover, as it could lead to cross contamination between the wells.)
Pipette 10 uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension is red, as the dried DNA has cresol red dye.
Thaw competent cells on ice. Then pre-chill by placing the tubes on ice.
Pipet 1 µL of plasmid into the competent cell tube.
Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.
Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.
Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.
Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.
Discard 150 μL supernatant liquid and resuspend the bacteria .
Coat plate: add the solution in a Cm plate and spread it , then cultivate it overnight.
Recorder:Yinchenguang Lyu, Xinfu Qin, Kaiyue Ma Transformation of yeast GAL1 promoter
Procedure:             1. Find the BBa_J63010 plasmid(Ampicillin resistence) containing the gene of yeast GAL1 promoter(Part Name BBa_J63006) in 7D well of 2013 Kit Plate 5.             2. With a pipette tip, punch a hole through the foil cover into the corresponding well.(Do not remove the foil cover, as it could lead to cross contamination between the wells.)             3. Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension is red, as the dried DNA has cresol red dye.             4. Put competent cells on ice. Then pre-chill by placing the tubes on ice.             5. Pipeted uncertain amount of each plasmid separately into two competent cell tubes, which is less than 1 μL.             6. Flick the tube gently with your finger to mix. Incubate on ice for an hour. Pre-heat waterbath now to 42℃.             7. Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.             8. Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.             9. Add 900 μL of SOC media per tube, and incubate at 37℃ for 2 hours.             10. Centrifuge the tube at 4000 rpm for 2 minutes, make the bacteria deposit at the bottom of the tube.             11. Discard 600 μL supernatant liquid and resuspend the bacteria.           12. Coat plate: add the solution 400 μL in a plate which contains LB media and ampicilin and spead it , then cultivate it overnight.(cultivation starts on 13:30 at 7.15)
Date: 7.16
Recorder: Yinchenguang Lyu
Result of transformation yesterday We have enough colonies on two plates to work on.
Colony picking 1.Pick the colonies of the plasmid we transformed yesterday into 4 tubes.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Recorder: Kaiyue Ma Transformation of Superfolder GFP 1. Thaw competent cells on ice. Then pre-chill by placing the tubes on ice.             2. Pipet 1 µL of plasmid into the competent cell tube.             3. Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.             4. Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.             5. Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.             6. Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.             7. Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.             8. Discard 150 μL supernatant liquid and resuspend the bacteria .           9. Coat plate: add the solution in a Cm plate and spread it , then cultivate it overnight.
Recorder：Xuefeng Meng、Jianyi Wang PCR of AD and BD Agarose gel electrophoresis of PCR products.
Add 0.8 g agarose to 80 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading： Plate 1: DNA marker 6 μL（Trans 2K Plus II）, sample(6) 30 μL. From left to right: AD1, AD1, AD2, AD2, AD3, AD3; Plate 2: DNA marker 6 μL（Trans 2K Plus II）, sample(6) 30 μL. From left to right: BD1, BD1, BD2, BD2, BD3, BD3.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result
Succeed!
PCR products:
Marker:
Tips:             1. Tm depends on the amount of nucleotides which can bind complementarily to the template;           2. Better to use PrimeSTAR HS DNA Polymerase, bought from TaKaRa.
Recorder: Xuefeng Meng、Chengle Zhang Gel Extraction of AD and BD
Procedure:             1. Cut off the Gel part containing the target band. Get 6 tubes of AD and 5 tubes of BD.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 300 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
sample
AD 1
AD 2
AD 3
AD 4
AD 5
AD 6
BD 1
BD 2
BD 3
BD 4
BD 5
A260/A280
1.99
1.92
1.84
1.88
1.91
1.90
1.86
2.00
1.95
1.90
1.89
Density(ng/μl)
82.4
56.8
69.1
60.3
64.3
61.4
67.5
47.7
51.7
56.9
62.5
Date: 7.17
Recorder：Kaiyue Ma
Colony picking of sfGFP 1.Pick the colonies of the plasmid we transformed yesterday into 2 tubes.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Plasmid Extraction for pGAL1, sfGFP and TEF2
Procedure:             1. Centifuge 1.5 mL bacterium solution at 11000 rpm, few sediment getted. Remove the  supernatant. Repeat twice.             2. Add 250 μL Buffer P1, resuspend cells.             3. Add 250 μL Buffer P2, mix well, 3 min's standing.             4. Add 350 μL Buffer P3, mix well.             5. 13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.             7. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             8. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
sfGFP-1
sfGFP-2
TEF2-1
TEF2-2
TEF2-3
TEF2-4
pGAL1-1
pG4AL1-2
pGAL1-3
pGAL1-
A260/A280
-0.63
1.80
1.71
1.83
1.81
1.74
1.81
1.84
1.84
1.84
Density(ng/μl)
0.6
178.7
175.4
105.5
136.1
159.2
422.9
414.9
416.4
456.9
Date: 7.18
Recorder: Guanchao Ding&Chengle Zhang
Double Digestion of pGal1 and GFP Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. GFP gene in plasmid PSB1A2(from iGEM kit plate)             3. Restriction enzyme SpeI,PstI,XbaI and 10×Tango Buffer(from Thermo Fisher Scientific)           4. Nuclease-free water
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
GFP-1
GFP-2
GFP-4
A260/A280
1.81
1.84
1.84
1.84
1.86
1.88
1.86
Density(ng/μL)
422.9
414.9
416.4
456.9
177.6
111.2
102.8
volume(μL)
2.4
2.4
2.4
2.2
5.6
9.0
9.7
10×Tango Buffer(μL)
2
2
2
2
2
2
2
nuclease-free water(μL)
12.6
12.6
12.6
12.8
6.4
3.0
2.3
PstI(μL)
2
2
2
2
2
2
2
SpeI(μL)
1
1
1
1
-
-
-
XbaI(μL)
-
-
-
-
2
2
2
Mix gently and incubate at 37 degree Celsius for 3 hours.
Recorder：Guanchao Ding and Chengle Zhang Agarose gel electrophoresis of digestion products Materials:             1. 1×TAE buffer             2. DNA marker （Trans 2K Plus II）           3. 6× loading buffer
Procedure:             1. Add 0.5 g agarose to 50 mL 1*TAE buffer.             2. Dissolved by heating.             3. Cool down.             4. Pour into electrophoresis tank.             5. Add 5 μL 6× loading buffer to 20 μL digestion product of each tube.             6. Loading：             Plate: DNA marker 6 μL（Trans 2K Plus II）, sample(×7) 20 μL. From left to right: pGAL1-1,pGAL1-2,pGAL1-3,pGAL1-4,GFP-4,GFP-2,GFP-1.             7. Electrophoresis gel：110 V 30 min.           8. Autoradiography(UV).
Results:             The picture shows the digestion failed.
Agarose gel electrophoresis of digestion of pGal1 and GFP
Recorder：Guanchao Ding and Chengle Zhang Second time for Double Digestion of pGal1 and GFP Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. GFP gene in plasmid PSB1A2(from iGEM kit plate)             3. Restriction enzyme SpeI,PstI,XbaI and 10×Tango Buffer(from Thermo Fisher Scientific)           4. Nuclease-free water
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-0
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
GFP-0
GFP-1
GFP-2
GFP-4
A260/A280
1.81
1.81
1.84
1.84
1.84
1.86
1.86
1.88
1.86
Density(ng/μL)
422.9
422.9
414.9
416.4
456.9
177.6
177.6
111.2
102.8
volume(μL)
2.4
2.4
2.4
2.4
2.2
5.6
5.6
9.0
9.7
10×Tango Buffer(μL)
2
2
2
2
2
2
2
2
2
nuclease-free water(μL)
15.6
12.6
12.6
12.6
12.8
10.4
6.4
3.0
2.3
PstI(μL)
-
2
2
2
2
-
2
2
2
SpeI(μL)
-
1
1
1
1
-
-
-
-
XbaI(μL)
-
-
-
-
-
-
2
2
2
Mix gently and incubate at 37 degree Celsius for 3 hours.
Recorder：Guanchao Ding and Chengle Zhang Agarose gel electrophoresis of digestion products Materials:             1. 1×TAE buffer             2. DNA marker （Trans 2K Plus II）           3. 6× loading buffer
Procedure:             1. Add 0.8 g agarose to 80 mL 1*TAE buffer.             2. Dissolved by heating.             3. Cool down.             4. Pour into electrophoresis tank.             5. Add 5 μL 6× loading buffer to 20 μL digestion product of each tube.             6. Loading：             Plate 1: DNA marker 6 μL（Trans 2K Plus II）, sample(×5) 20 μL. From left to right: pGAL1-0,pGAL1-1,pGAL1-2,pGAL1-3,pGAL1-4.             Plate 2:DNA marker 6 μL（Trans 2K Plus II）, sample(×4) 20 μL. From left to right: GFP-0,GFP-1,GFP-2,GFP-4.             7. Electrophoresis gel：110 V 30 min.           8. Autoradiography(UV).
Results:             The picture shows the digestion failed again.
Agarose gel electrophoresis of digestion of pGal1
Agarose gel electrophoresis of digestion of  GFP
Recorder：Yin Wu, Xuefeng Meng PCR of GFP1-10
Experimental materials 1. Plasmid: pSB1A2 containing GFP sequence;             2. Primer: GFP1-10-p, GFP1-10-r, designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μl). All bought from TaKaRa, Code No. R010A.
About primers
primer
GFP1-10-p
GFP1-10-r
sequence
5'-GAATTCGCGGCC
GCTTCTAGATGC
GTAAAGGAGAAG
AACTTTTCA-3'
5'-CTGCAGCGGCCG
CTACTAGTATTA
TTACTTTTCGTT
GGGATCTTTCGA
AAGG-3'
lenth (nt)
45
52
GC%
47
46
molecular weight (Da)
13909.1
16006.5
Tm (℃)
55
60
Procedure:
1.Prepare 4 PCR tubes and sequentially add：
sample
1*2
2*2
Sterilized ddH2O
32.5 μL
32 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
10 μL
dNTP Mixture (2.5 mM each)
4 μL
4 μL
Template(plasmid pSB1A2)
1 μL
1.5 μL
GFP1-10-p
1 μL
1 μL
GFP1-10-r
1 μL
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
0.5 μL
total
50 μL
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
53
15 s
step 4
72
43 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder: Kaiyue Ma Colony picking of GFP 1.Pick the colonies of the plasmid we transformed yesterday into 2 tubes.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Recorder：Xuefeng Meng, Yin Wu PCR of GFP1-10 Agarose gel electrophoresis of PCR products.
Add 0.4 g agarose to 40 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker 6 μL（Trans 2K Plus II）, sample(*8) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Marker:
Recorder: Xuefeng Meng、Yin Wu Gel Extraction of GFP1-10
Procedure:             1. Cut off the Gel part containing the target band. Get 8 tubes of GFP1-10.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 300 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of GFP1-10.
sample
1
2
3
4
5
6
7
8
A260/A280
2.21
1.98
2.05
1.82
1.81
1.95
1.89
2.04
Density(ng/μl)
37.6
44.0
43.2
52.4
39.1
72.2
55.0
36.3
Date: 7.19
Recorder: Chengle Zhang
First time of Double Fast Digestion of pGal1 Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)           3. Nuclease-free water
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-0
pGAL1-1
Density(ng/μL)
403.6
403.6
volume(μL)
5
5
10×FastDigest Green Buffer(μL)
2
2
nuclease-free water(μL)
13
11
FastDigest PstI(μL)
0
1
FastDigest SpeI(μL)
0
1
Mix gently and incubate at 37 degree Celsius for 2 hours.
Recorder：Guanchao Ding and Chengle Zhang Agarose gel electrophoresis of digestion products Materials:             1. 1×TAE buffer             2. DNA marker （Trans 2K Plus II）           3. 6× loading buffer
Procedure:             1. Add 0.4 g agarose to 40 mL 1*TAE buffer.             2. Dissolved by heating.             3. Cool down.             4. Pour into electrophoresis tank.             5. Add 5 μL 6× loading buffer to 20 μL digestion product of each tube.             6. Loading：             Plate: DNA marker 6 μL（Trans 2K Plus II）, sample(×2) 20 μL. From left to right: pGAL1-0,pGAL1-1.             7. Electrophoresis gel：110 V 30 min.           8. Autoradiography(UV).
Results:             The picture shows the digestion failed.
Agarose gel electrophoresis of digestion of pGal1
Recorder: Yinchenguang Lyu Plasmid Extraction for GFP             Procedure:             1.Centifuge 1.5 mL bacterium solution at 11000 rpm, few sediment getted. Remove the supernatant. Repeat twice.             2.Add 250 μL Buffer P1, resuspend cells.             3.Add 250 μL Buffer P2, mix well, 3 min's standing.             4.Add 350 μL Buffer P3, mix well.             5.13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6.Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.             7.Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             8.12000 rpm centifuge 1 min.             9.Lying for 10 min. 10.Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
Recorder：Yin Wu, Xuefeng Meng PCR of sfGFP1-10
Experimental materials 1. Plasmid: pSB1C3 containing sfGFP sequence;             2. Primer: sfGFP1-10-p, sfGFP1-10-r, designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μl). All bought from TaKaRa, Code No. R010A.
About primers
primer
sfGFP1-10-p
sfGFP1-10-r
sequence
5'-CCGGAATTCGCG
GCCGCTTCTAGa
tgcgtaaaggcg
aagagctgttca-3'
5'-TGCACTGCAGCG
GCCGCTACTAGT
ATTATTAtttct
cgttcggatctt
tagacagaacg-3'
lenth (nt)
48
59
GC%
56.25
45.76
molecular weight (Da)
14833.58
18117.65
Tm (℃)
74.95
72.59
Procedure:
1.Prepare 4 PCR tubes and sequentially add：
sample
*4
Sterilized ddH2O
32.5 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
dNTP Mixture (2.5 mM each)
4 μL
Template(plasmid pSB1C3)
1 μL
sfGFP1-10-p
1 μL
sfGFP1-10-r
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
total
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
55
15 s
step 4
72
43 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder：Ya Jiang, Yin Wu Agarose gel electrophoresis of PCR products.
Add 0.4 g agarose to 40 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker 6 μL（Trans 2K Plus II）, sample(*8) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Marker:
Recorder: Ya Jiang、Yin Wu Gel Extraction of sfGFP1-10
Procedure:             1. Cut off the Gel part containing the target band. Get 8 tubes of sfGFP1-10.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 300 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of sfGFP1-10.
sample
1
2
3
4
5
6
7
8
A260/A280
2.19
2.05
2.24
2.96
2.09
2.03
1.86
2.03
Density(ng/μl)
13.7
22.6
19.4
26.5
30.9
24.8
40.9
27.5
Recorder: Chengle Zhang
Second time of Double Fast Digestion of pGal1 Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-0
pGAL1-1
Density(ng/μL)
403.6
403.6
volume(μL)
30
30
10×FastDigest Green Buffer(μL)
5
5
nuclease-free water(μL)
15
13
FastDigest PstI(μL)
0
1
FastDigest SpeI(μL)
0
1
Mix gently and incubate at 37 degree Celsius for 2 hours.
Agarose gel electrophoresis Results:             The picture shows the digestion failed.
Agarose gel electrophoresis of digestion of pGal1(from left to right:marker,pGAL1-0,pGAL1-0,pGAL1-1,pGAL1-1)
Recorder: Kaiyue Ma Colony picking of pGAL 1.Pick the colony of the plasmid we transformed yesterday into a tube.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Transformation of pGAL 1. Thaw competent cells on ice. Then pre-chill by placing the tubes on ice.             2. Pipet 1 µL of plasmid (we extracted before) into the competent cell tube.             3. Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.             4. Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.             5. Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.             6. Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.             7. Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.             8. Discard 150 μL supernatant liquid and resuspend the bacteria .           9. Coat plate: add the solution in a Ap plate and spread it , then cultivate it overnight.
Date: 7.20
Recorder：Kaiyue Ma
Colony picking of pGAL 1.Pick the colonies of the plasmid we transformed yesterday into 4 tubes.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Amplification of sup35
We got bacteria containing PET28a-sup35-GFP plasmid from Dr. Dong Men.
We pipetted 2 μL bacteria to 5ml LB liquid medium, parallelling 4 tubes. (Cultivate at 37°C, shock at 250 rpm)
We also coated 20 μL bacteria on a 10 μg/ml ka+ LB plate. (Cultivate at 37°C)
Plasmid Extraction for pGAL
Procedure:             1. Centifuge 1.5 mL bacteria solution at 11000 rpm, few sediment getted. Remove the  supernatant. Repeat twice.             2. Add 250 μL Buffer P1, resuspend cells.             3. Add 250 μL Buffer P2, mix well, 3 min's standing.             4. Add 350 μL Buffer P3, mix well.             5. 13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.             7. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             8. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
pGAL
A260/A280
1.88
Density(ng/μl)
490.4
recorder：Xuefeng Meng Coating plate of bacteria containing sup35
We coated 200 μL 4-hours cultivated bacteria on a 10 μg/ml ka+ LB plate, 4 plates. (Cultivate at 37°C)
Recorder：Yin Wu, Chengle Zhang PCR of sfGFP1-10
Experimental materials 1. GFP1-10 PCR product (7.19 sample 8);             2. Primer: sfGFP1-10-p, sfGFP1-10-r, designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μl). All bought from TaKaRa, Code No. R010A.
About primers
primer
sfGFP1-10-p
sfGFP1-10-r
sequence
5'-CCGGAATTCGCG
GCCGCTTCTAGa
tgcgtaaaggcg
aagagctgttca-3'
5'-TGCACTGCAGCG
GCCGCTACTAGT
ATTATTAtttct
cgttcggatctt
tagacagaacg-3'
lenth (nt)
48
59
GC%
56.25
45.76
molecular weight (Da)
14833.58
18117.65
Tm (℃)
74.95
72.59
Procedure:
1.Prepare 4 PCR tubes and sequentially add：
sample
1
2
3
4
Sterilized ddH2O
28.5 μL
30.5 μL
32.5 μL
33 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
10 μL
10 μL
10 μL
dNTP Mixture (2.5 mM each)
4 μL
4 μL
4 μL
4 μL
Template (PCR product)
5 μL
3 μL
1 μL
0.5 μL
sfGFP1-10-p
1 μL
1 μL
1 μL
1 μL
sfGFP1-10-r
1 μL
1 μL
1 μL
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
0.5 μL
0.5 μL
0.5 μL
total
50 μL
50 μL
50 μL
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
55
15 s
step 4
72
43 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder: Haoyu Liu, Shuai Shao. Plasmid Extraction for pGAL1
Procedure:             1. Centrifuge 1.5 mL bacteria solution at 11000 rpm, few sediment got. Remove the  supernatant. Repeat twice.             2. Add 250 μL Buffer P1, suspend cells.             3. Add 250 μL Buffer P2, mix well, 3 min's standing. 4. Add 350 μL Buffer P3, mix well.             5. 13400 rpm centrifuge 10 min, move all supernatant to adsorption column, 11000 rpm centrifuge 30 s, discard filtrate.             6. Add 500 μL Buffer DW1, 12000 rpm centrifuge 30 s, discard filtrate.             7. Add 500 μL Wash Solution, 12000 rpm centrifuge 30 s, discard filtrate. Repeat once.             8. 12000 rpm centrifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centrifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
Result:
sample
1
2
3
4
A260/A280
1.85
1.87
1.87
1.86
Density(μg/ml)
303.3
219.2
319.4
377.9
Recorder: Yin Wu Agarose gel electrophoresis of PCR products.
Add 0.4 g agarose to 40 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker 6 μL（Trans 2K Plus II）, sample(*8) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Marker:
The color of the bands isn't apparent, so we don't cut the gel.
Recorder：Yin Wu, Xuefeng Meng PCR of sfGFP1-10
Experimental materials 1. Plasmid: pSB1C3 containing sfGFP sequence;             2. Primer: sfGFP1-10-p, sfGFP1-10-r, designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μl). All bought from TaKaRa, Code No. R010A.
About primers
primer
sfGFP1-10-p
sfGFP1-10-r
sequence
5'-CCGGAATTCGCG
GCCGCTTCTAGa
tgcgtaaaggcg
aagagctgttca-3'
5'-TGCACTGCAGCG
GCCGCTACTAGT
ATTATTAtttct
cgttcggatctt
tagacagaacg-3'
lenth (nt)
48
59
GC%
56.25
45.76
molecular weight (Da)
14833.58
18117.65
Tm (℃)
74.95
72.59
Procedure:
1.Prepare 8 PCR tubes and sequentially add：
sample
1*8
Sterilized ddH2O
32.5 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
dNTP Mixture (2.5 mM each)
4 μL
Template(plasmid pSB1C3)
1 μL
sfGFP1-10-p
1 μL
sfGFP1-10-r
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
total
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
50 or 55
15 s
step 4
72
43 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)             note: The temperature of step3:             tube 1~4: 50           tube 5~8: 55
Recorder: Chengle Zhang Third time of Double Fast Digestion of pGal1 Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1
Density(ng/μL)
403.6
volume(μL)
30
10×FastDigest Green Buffer(μL)
5
nuclease-free water(μL)
13
FastDigest PstI(μL)
1
FastDigest SpeI(μL)
1
Mix gently and incubate at 37 degree Celsius for 5 hours.
Agarose gel electrophoresis Results:             The picture shows the digestion failed because of high star activity.
Agarose gel electrophoresis of digestion of pGal1(from left to right:marker,control,test,test)
Recorder: Shuai Shao and Chengle Zhang Agarose gel electrophoresis of pGal1 plasmid Materials:             1. plasmid 1:pGal1 gene in plasmid BBa-J63010(from iGEM kit plate) with concentration of 403.6 ng/μL(extracted in two days before)             2. plasmid 2:pGal1 gene in plasmid BBa-J63010(from iGEM kit plate) with concentration of 490.4 ng/μL(extracted today)           3. marker:Trans 2K Plus II
Before agarose gel electrophoresis,add the following materials to the plasmid:
sample
2 μL plasmid 1
2 μL plasmid 2
nuclease-free water(μL)
3
3
6×loading buffer(μL)
1
1
Agarose gel electrophoresis Results:             The picture shows plasmid has nomal quality.
Agarose gel electrophoresis of digestion of  pGal1 plasmid(from left to right:marker,plasmid 1,plasmid 2)
Recorder：Yin Wu and Chengle Zhang **Double Fast Digestion of pGal1 ** Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate) with concentration of 490.4 ng/μL(extracted today)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
Density(ng/μL)
490.4
490.4
volume(μL)
8.2
1
Nuclease-free water(μL)
0
7
10×FastDigest Green Buffer(μL)
1
1
FastDigest EcoRI(μL)
0.4
0.5
FastDigest SpeI(μL)
0.4
0.5
Mix gently and incubate at 37 degree Celsius for 4 hours.
Agarose gel electrophoresis Results:
Agarose gel electrophoresis of digestion of  pGal1(from left to right:marker,control,pGAL1-1,pGAL1-2,others,others,others,others,others)           The picture shows that double fast digestion of pGal1 is successful.It indicates that the material is OK.
Recorder：Xuefeng Meng and Chengle Zhang Double Digestion of GFP Materials:             1. GFP gene in plasmid PSB1A2(from iGEM kit plate)with concentration of 153.8 ng/μL             2. GFP gene in plasmid PSB1A2(from iGEM kit plate)with concentration of 112.0 ng/μL             3. Restriction enzyme XbaI,EcoRI and 10×Tango Buffer(from Thermo Fisher Scientific)             4. Nuclease-free water           5. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
GFP-1
GFP-2
GFP-3
GFP-4
Density(ng/μL)
153.8
112.0
153.8
112.0
volume(μL)
16
16
13
13
10×Tango Buffer(μL)
2
2
4
4
EcoRI(μL)
1
1
1
1
XbaI(μL)
1
1
2
2
Mix gently and incubate at 37 degree Celsius for 15 hours.
After gel electrophoresis，bathed in extra gel red for 270 mins.             Agarose gel electrophoresis Results:
Agarose gel electrophoresis of pGAL1 &GFP(3)(from left to right:marker,pGAL1-1,2,3,4,GFP-1,2,3,4)
Recorder：Shuai Shao and Chengle Zhang **Double Digestion of pGal1 **             Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. Restriction enzyme SpeI,EcoRI and 10×Tango Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
Density(μg/ml)
303.3
219.2
319.4
377.9
volume(μL)
1
1
1
1
10×Tango Buffer(μL)
2
2
2
2
EcoRI(μL)
1
1
1
1
XbaI(μL)
1
1
1
1
Mix gently and incubate at 37 degree Celsius for 10 hours.
Agarose gel electrophoresis Results:             1.First time without extra gel red
Agarose gel electrophoresis of pGAL1 &GFP(1)           (from left to right:marker,pGAL1-1,2,3,4,GFP-1,2,3,4)
2.Third time with bathed in extra gel red for 90 mins
Agarose gel electrophoresis of pGAL1 &GFP(2)           (from left to right:marker,pGAL1-1,2,3,4,GFP-1,2,3,4)
3.Third time with bathed in extra gel red for 270 mins
Agarose gel electrophoresis of pGAL1 &GFP(3)(from left to right:marker,pGAL1-1,2,3,4,GFP-1,2,3,4)
Date: 7.21
Recorder：Yu Xie
Plasmid Extraction for pGAL and sup35
Procedure:             1. Centifuge 1.5 mL bacteria solution at 11000 rpm, few sediment getted. Remove the  supernatant. Repeat twice.             2. Add 250 μL Buffer P1, resuspend cells.             3. Add 250 μL Buffer P2, mix well, 3 min's standing.             4. Add 350 μL Buffer P3, mix well.             5. 13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.             7. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             8. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
pGAL1-1
pGAL1-2
sup35-1
sup35-2
A260/A280
1.86
1.84
1.90
1.91
Density(ng/μl)
368.7
381.8
70.4
74.1
Recorder: Yin Wu Agarose gel electrophoresis of PCR products.
Add 0.8 g agarose to 80 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker(2) 5 μL（Trans 2K Plus II）, sample(16) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Marker:
Recorder: Kaiyue Ma Colony picking of SUP35 1.Pick the colonies of the plasmid we transformed yesterday into 4 tubes.           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Recorder: Yin Wu, Shuai Shao Gel Extraction of sfGFP1-10
Procedure:             1. Cut off the Gel part containing the target band. Get 16 tubes of sfGFP1-10.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 3 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column (every 4 tubes of solution into 1 adsorption column), 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of sfGFP1-10.
sample
1
2
3
4
A260/A280
1.95
2.10
1.92
1.91
Density(ng/μl)
44.2
55.7
72.2
70.9
Recorder：Xuefeng Meng Get shuttle plasmid PYeScGAP We get shuttle plasmid PYeScGAP from Miss. Dan Wu.
Recorder：Kaiyue Ma Transformation of SUP35
Thaw competent cells (BL21) on ice. Then pre-chill by placing the tubes on ice.
Pipet 2 µL of plasmid (Extracted yesterday) into the competent cell tube.
Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.
Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.
Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.
Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.
Discard 150 μL supernatant liquid and resuspend the bacteria .
Coat plate: add the solution in a Ka+ plate and spread it , then cultivate it overnight.
Recorder：Yin Wu and Chengle Zhang **Double Digestion of pGal1 **             Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme PstI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
Density(μg/ml)
303.3
219.2
319.4
377.9
volume(μL)
1
1
1
1
Nuclease-free water(μL)
7
7
7
7
10×FastDigest Green Buffer(μL)
1
1
1
1
EcoRI(μL)
0.5
0.5
0.5
0.5
PstI(μL)
0.5
0.5
0.5
0.5
Mix gently and incubate at 37 degree Celsius for 1 hours.
After gel electrophoresis，bathed in extra gel red for 180 mins.             Agarose gel electrophoresis Results:
Agarose gel electrophoresis of digestion of  pGal1(from left to right:marker,others,others,others,pGAL1-1,pGAL1-2,pGAL1-3,pGAL1-4,control)           The picture shows that the double digestion of pGal1 is successful,while the efficiency is low.
Recorder：Xuefeng Meng、Yin Wu PCR of sup35
Experimental materials 1. Plasmid: PET28a-sup35-GFP plasmid, get from Dong Men.             2. Primer: sup35-p, sup35-r. Designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μL). All bought from TaKaRa, Code No. R010A.
About primers
primer
1
2
sequence
5'-ATGTCGGATTCA
AACCAGGGCAAC-3'
5'-CATTCACCACCT
CATCGTCAACTT
CC-3'
lenth (nt)
24
26
GC%
50
50
molecular weight (Da)
7370.9
7746.1
Tm (℃)
61
61
Procedure:
1.Prepare 4 PCR tubes and sequentially add：
sample
1,2
3,4
Sterilized ddH2O
32.5 μL
31.5 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
10 μL
dNTP Mixture (2.5 mM each)
4 μL
4 μL
Template(plasmid 0.741ng/μL)
1 μL
0
Template(plasmid 74.1ng/μL)
0
2 μL
sup35-p
1 μL
1 μL
sup35-r
1 μL
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
0.5 μL
total
50 μL
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
55
15 s
step 4
72
50 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder: Yin Wu, Xuefeng Meng Agarose gel electrophoresis of PCR products.
Add 0.4 g agarose to 40 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker(*2) 5 μL（Trans 2K Plus II）, sample(1,1; 2,2; 3,3; 4,4) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Marker:
Recorder：Yin Wu Double Digestion of pTEF2 Materials:             1. pTEF2 gene (BBa-K165037) is in iGEM kit plate3 21H             2. Restriction enzyme (Fastdigest) SpeI, EcoRI and 10×Green Buffer           3. Nuclease-free water
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
1
2
3
4
5
6
7
8
9
10
time/min
30
30
60
60
30
30
60
60
60
0
plasmid volume(μL)
5
3
5
3
3
3
3
3
2
2
10×Green Buffer(μL)
1
1
1
1
1
1
1
1
0
0
nuclease-free water(μL)
3
5
3
5
5.5
5.5
5.5
5.5
3
3
EcoRI(μL)
0.5
0.5
0.5
0.5
0.5
0
0.5
0
0
0
SpeI(μL)
0.5
0.5
0.5
0.5
0
0.5
0
0.5
0
0
Mix gently and incubate at 37 degree Celsius.
Recorder：Yin Wu Agarose gel electrophoresis of digestion products Materials:             1. 1×TAE buffer             2. DNA marker （Trans 2K Plus II）           3. 6× loading buffer
Procedure:             1. Add 0.4 g agarose to 40 mL 1*TAE buffer.             2. Dissolved by heating.             3. Cool down.             4. Pour into electrophoresis tank.             5. Add 2 μL 6× loading buffer to 10 μL digestion product of each tube.             6. Loading：             Plate: DNA marker 5 μL（Trans 2K Plus II), sample(×8) 12 μL, sample(×2) 6 μL.             7. Electrophoresis gel：110 V 30 min.           8. Autoradiography(UV).
Results:
Date: 7.22
Recorder：Dong Yan and Chengle Zhang **Double Digestion of pGal1 **             Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
Density(μg/ml)
303.3
219.2
319.4
377.9
volume(μL)
16
16
16
16
10×FastDigest Green Buffer(μL)
2
2
2
2
EcoRI(μL)
1
1
1
1
SpeI(μL)
1
1
1
1
Mix gently and incubate at 37 degree Celsius for 1 hours.
Agarose gel electrophoresis Results:
Agarose gel electrophoresis of digestion of  pGal1(from left to right:marker,pGAL1-1,pGAL1-2,pGAL1-3,pGAL1-4,control:2 μL pGAL1-3)           Double digestion of pGal1 failed again.
Recorder：Yu Xie
Plasmid Extraction for pGAL and sup35
Procedure:             1. Centifuge 1.5 mL bacteria solution at 11000 rpm, few sediment getted. Remove the  supernatant. Repeat twice.             2. Add 250 μL Buffer P1, resuspend cells.             3. Add 250 μL Buffer P2, mix well, 3 min's standing.             4. Add 350 μL Buffer P3, mix well.             5. 13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.             7. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             8. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 50 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of plasmids.
sup35-1
sup35-2
sup35-3
sup35-4
A260/A280
1.71
1.88
1.88
1.91
Density(ng/μl)
53.4
71.0
63.1
72.3
Recorder：Yin Wu Transformation of constitutive promoter family member and RBS
Find the pSB1C3 plasmid containing the gene of constitutive promoter family member and RBS (Part Name BBa_K081005) in 16E well of 2016 Kit Plate 3.
With a pipette tip, punch a hole through the foil cover into the corresponding well.(Do not remove the foil cover, as it could lead to cross contamination between the wells.)
Pipette 10 uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension is red, as the dried DNA has cresol red dye.
Thaw competent cells on ice. Then pre-chill by placing the tubes on ice.
Pipet 1 µL of plasmid into the competent cell tube.
Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.
Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.
Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.
Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.
Discard 150 μL supernatant liquid and resuspend the bacteria .
Coat plate: add the solution in a Cm plate and spread it , then cultivate it at 37°C.
Recorder：Kaiyue Ma PCR of sup35
Experimental materials 1. Plasmid: PET28a-sup35-GFP plasmid, get from Dong Men.             2. Primer: sup35-p, sup35-r. Designed by ourselves, synthesized by Sangon Biotech;             3. Sterilized ddH2O;           4. 5×PrimeSTAR Buffer (Mg2+Plus), dNTP Mixture (2.5 mM each), PrimeSTAR HS DNA Polymerase (2.5 U/μL). All bought from TaKaRa, Code No. R010A.
About primers
primer
1
2
sequence
5'-ATGTCGGATTCA
AACCAGGGCAAC-3'
5'-CATTCACCACCT
CATCGTCAACTT
CC-3'
lenth (nt)
24
26
GC%
50
50
molecular weight (Da)
7370.9
7746.1
Tm (℃)
61
61
Procedure:
1.Prepare 4 PCR tubes and sequentially add：
sample
1,2
3,4
Sterilized ddH2O
32.5 μL
31.5 μL
5×PrimeSTAR Buffer (Mg2+Plus)
10 μL
10 μL
dNTP Mixture (2.5 mM each)
4 μL
4 μL
Template(plasmid 0.741ng/μL)
0
2 μL
Template(plasmid 74.1ng/μL)
1 μL
0
sup35-p
1 μL
1 μL
sup35-r
1 μL
1 μL
PrimeSTAR HS DNA Polymerase (2.5 U/μl)
0.5 μL
0.5 μL
total
50 μL
50 μL
2.PCR reaction           Parameters setting：
stage
temperature
time
step 1
95
5 min
step 2
95
10 s
step 3
55
15 s
step 4
72
50 s
step 5
72
10 min
step 6
4
--
30 cycles(step 2 ~ step 4)
Recorder：Yin Wu, Chengle Zhang Transformation of Kozak sequence (yeast RBS)
Find the pSB1C3 plasmid containing the gene of Kozak sequence (yeast RBS) (Part Name BBa_K165002) in 12M well of 2016 Kit Plate 3.
With a pipette tip, punch a hole through the foil cover into the corresponding well.(Do not remove the foil cover, as it could lead to cross contamination between the wells.)
Pipette 10 uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension is red, as the dried DNA has cresol red dye.
Thaw competent cells on ice. Then pre-chill by placing the tubes on ice.
Pipet 1 µL of plasmid into the competent cell tube.
Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.
Heat-shock the cells by placing into the waterbath for 90 seconds. Be careful to keep the lids of the tubes above the water level, and keep the ice close by.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. This helps the cells recover.
Add 200 µL of SOC media in the tube, and incubate at 37°C for 1 hours.
Centrifuge the tube at 4000 rpm for 1 minutes, make the bacteria deposit at the bottom of the tube.
Discard 150 μL supernatant liquid and resuspend the bacteria .
Coat plate: add the solution in a Cm plate and spread it , then cultivate it at 37°C.
Recorder：Dong Yan and Chengle Zhang **Double Digestion of pGal1 **             Materials:             1. pGal1 gene in plasmid BBa-J63010(from iGEM kit plate)             2. FastDigest restriction enzyme SpeI,EcoRI and 10×FastDigest Green Buffer(from Thermo Fisher Scientific)             3. Nuclease-free water           4. marker:Trans 2K Plus II
Procedure:           Add the materials into new EP tubes in the order of following table respectively.
sample
pGAL1-1
pGAL1-2
pGAL1-3
pGAL1-4
Density(μg/ml)
303.3
219.2
319.4
377.9
volume(μL)
6
6
6
6
nuclease free water(μL)
10
10
10
10
10×FastDigest Green Buffer(μL)
2
2
2
2
EcoRI(μL)
1
1
1
1
SpeI(μL)
1
1
1
1
Mix gently and incubate at 37 degree Celsius for 1 hours.
Agarose gel electrophoresis Results:
Agarose gel electrophoresis of digestion of  pGal1(from left to right:marker,pGAL1-1,pGAL1-1&2&3&4,pGAL1-2,empty,pGAL1-3,pGAL1-4,pGAL1-1&2&3&4)
Recorder：Yin Wu Double Digestion of pTEF2 Materials:             1. pTEF2 gene (BBa-K165037) is in iGEM kit plate3 21H             2. Restriction enzyme (Fastdigest) SpeI, EcoRI and 10×Green Buffer           3. Nuclease-free water
Procedure:             In each sample, add             nuclease-free water  6 μL             10*Green Buffer  2 μL             plasmid  10 μL             EcoRI  1 μL             SpeI  1 μL           Mix gently and incubate at 37 degree Celsius, sample 1 and 2 for 20 min, sample 3-8 for 30 min.
Recorder：Yin Wu, Ya Jiang Agarose gel electrophoresis of digestion products Materials:             1. 1×TAE buffer             2. DNA marker （Trans 2K Plus II）           3. 6× loading buffer
Procedure:             1. Add 0.4 g agarose to 40 mL 1*TAE buffer.             2. Dissolved by heating.             3. Cool down.             4. Pour into electrophoresis tank.             5. Add 4 μL 6× loading buffer to 20 μL digestion product of each tube.             6. Loading：             Plate: DNA marker 5 μL（Trans 2K Plus II), sample(×8) 24 μL.             7. Electrophoresis gel：110 V 30 min.           8. Autoradiography(UV).
Results:
Recorder: Kaiyue Ma Agarose gel electrophoresis of PCR products of SUP35
Add 0.4 g agarose to 40 mL TAE buffer.
Dissolved by heating.
Cool down.
Pour into electrophoresis tank.
Add 10 μL 6× loading buffer to 50 μL PCR product of each tube.
Loading：             DNA marker(*2) 5 μL（Trans 2K Plus II）, sample(1,1; 2,2; 3,3; 4,4) 30 μL.
Electrophoresis gel：110 V 30 min.
Autoradiography(UV).
Result:             PCR products:
Gel extraction of SUP35 Procedure:             1. Cut off the Gel about the position of 750bp from the fourth lane.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 3 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column , 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density.
sample
SUP35
A260/A280
29.1
Density(ng/μl)
2.02
Recorder: Ya Jiang, Yin Wu Gel Extraction of pTEF2
Procedure:             1. Cut off the Gel part containing the target band. Get 8 tubes of pTEF2.             2. Weigh the cut off Gel part.             3. Add Buffer B2 which 3 times the weight of the Gel.             4. Add isopropanol which is 1/3 volume of the Buffer B2.             5. Move the solution to adsorption column (every 4 tubes of solution into 1 adsorption column), 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O, 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the density of pTEF2.
sample
1
2
A260/A280
2.39
2.21
Density(ng/μl)
5.8
10.2
Recorder: Kaiyue Ma Colony picking of pGAL, TEF2 and GFP 1.Pick the colonies of the plasmid we transformed before into 12 tubes (4 for eaach).           2.Cultivate the bacteria at 37°C and shock at 250 rpm/min overnight.
Recorder: Haoyu Liu,Dong Yan and Chengle Zhang Gel Extraction of pGal1
Procedure:             1. Cut off the Gel part containing the target band. Get 7 tubes of pGal1.             2. Weigh the cut off Gel part.             3. Add Buffer B2 whose weight is 3 times of the Gel.             4. Add isopropanol whose volume is 1/3 of the Buffer B2.             5. Move the solution to adsorption column , 11000 rpm centifuge 30 s, discard filtrate.             6. Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.             7. 12000 rpm centifuge 1 min.             9. Lying for 10 min.           10. Put the adsorption column in a new EP tube. Add 20 μL ddH2O which is pre-heated at 55 degree Celsius , 10 min's standing, 12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.
After extraction, we measured the OD A260/A280 and the concentration of pGal1.
sample
1
2
3
4
5
6
7
A260/A280
1.69
4.42
2.46
3.67
4.84
2.60
1.67
concentration(ng/μL)
14.2
2.7
7.2
5.5
7.0
2.4
11.0
The table indicates that we didn't get the pGal1 back.
Then we did agarose gel electrophoresis to test the extraction product.(electrophoresis system:2 μL sample,with 2 μL nuclease-free water and 2 μL 6× loading buffer)             Results:
The picture shows the agarose gel electrophoresis of gel extraction of pGal1 failed.We got nothing.
Contact us
e-mail
Facebook
Weibo
Sponsors
Designed by 2016 iGEM Team:USTC
Under CC License
Based on Semantic-UI
