<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USTC-Software_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USTC-Software/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:USTC-Software - 2016.igem.org</TITLE></P><DIV class="ar0"><UL class="dropdown menu row ar1" data-dropdown-menu=""><LI class="column small-4 large-2 ar0"><A href="https://2016.igem.org/Team:USTC-Software">Home</A></LI><LI class="column small-4 large-2 ar1"><A href="https://2016.igem.org/Team:USTC-Software/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Video">Video</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Technology">Technology</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Installation">Installation</A></LI></UL><LI class="column small-4 large-2 ar2"><A href="https://2016.igem.org/Team:USTC-Software/HumanPractices">Human Practices</A></LI><LI class="column small-4 large-2 ar3"><A href="https://2016.igem.org/Team:USTC-Software/Requirements">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Collaborations">Collaborations</A></LI><LI class="column small-4 large-2 ar4"><A href="https://2016.igem.org/Team:USTC-Software/Notebook">Notebook</A></LI><LI class="column small-4 large-2 ar5"><A href="https://2016.igem.org/Team:USTC-Software/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:USTC-Software/Attributions">Attributions</A></LI><DIV class="row ar2"><DIV class="column ar0 show-for-large large-3"><DIV id="navigation-bar"><UL class="menu vertical" data-magellan=""><LI><A href="#Overview">Overview</A></LI><LI><A href="#Module_Pano">Pano</A></LI><LI><A href="#Module_Pathway_finder">Pathway finder</A></LI><LI><A href="#Module_Forum">Forum</A></LI><LI><A href="#Module_Simulation">Simulation</A></LI><LI><A href="#Module_BLAST">BLAST</A></LI><LI><A href="#Module_ABACUS">ABACUS</A></LI><LI><A href="#Module_Biobrick_manager">Biobrick manager</A></LI><LI><A href="#Plugins">Plugins</A></LI><LI><A href="#Docker">Docker</A></LI><LI><A href="#Outlook">Outlook</A></LI></UL></DIV></DIV><DIV class="column ar1 small-12 large-9"><DIV class="row ar0" id="Overview" data-magellan-target="Overview"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Overview">Overview</A></H1></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								Welcome to BioHub! This is an open and fantastic platform with practical functions for synthetic biologists. For scientists whose goal is to make discoveries in biology, steps using bioinformatics can be seen as the most basic but rather troublesome part. To solve this, our team created BioHub.
							</P><P>
								BioHub is a platform with several plug-ins, which means that users can not only employ the functions we provide, but also develop new and useful ones which can meet their special needs, without caring for servers and network transmission. Others include Pathway finder, which means find the correlation among genes, Pano, meaning designing genetic circuits like an artist, Simulation, which can imitate real experimental circumstances, and other splendid functions.
							</P></DIV></DIV><DIV class="row ar1" id="Module_Pano" data-magellan-target="Module_Pano"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_Pano">Module:Pano</A></H1><P>
								Pano is a visualized editing interface which is designed to arrange the relationships between genes. It can be used to find pathways between genes and simulate related substances according to controlling ODEs.
							</P><P>
								The relationships of genes are represented by graphs, which include nodes and edges. Each node represents a gene while each edge represents a relationship. Users shall build up a graph in order to input relationships network between genes. Users will be capable to process the network further such as operating a simulation and so on.
							</P></DIV></DIV><DIV class="row ar2" id="Module_Pathway_finder" data-magellan-target="Module_Pathway_finder"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_Pathway_finder">Module:Pathway finder</A></H1><P>
								Pathway finder is one of the most powerful plugins of BioHub. In traditional circumstances, searching for the relationships among components in the genetic circuit can be rather arduous, so users must read plenty of papers and inquire several databases. To rescue biologists from this, Pathway finder is born now. Select any two components in genetic circuit, and by searching the database of gene routes in the server we can rapidly know whether they have strong relationship, and whether the feature is of interest. It combines metabolic network databases, cell signaling pathway databases and gene interaction databases, and the path between components means produce, combine, influence activation, promote degradation or others.
							</P><P>
								Pathway finder is a module integrated in the Pano to provide more convenient and faster ways to find paths from a specific source node to another target node, select appropriate paths, and add them to the Pano. Biologic scientific workers using the Pathway finder, nodes and edges will be added under the rules provided by the database of the backend. 
							</P><P>
								1. Select a node as a source node in the pano, and click “PATH FINDER… ” at the sidebar.
							</P></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								2. Select another node as a target, then a modal dialog will be opened.
							</P></DIV><DIV class="column small-12 large-12 ar4 textcontainer"><P>
								3. Change the amount and the maximum depth, then click “GO”.
							</P></DIV><DIV class="column small-12 large-12 ar6 textcontainer"><P>
								4. Wait for a while until the paths shows on the screen.
							</P></DIV><DIV class="column small-12 large-12 ar8 textcontainer"><P>
								5. Delete paths that you do not want.
							</P></DIV><DIV class="column small-12 large-12 ar10 textcontainer"><P>
								6. Then click “OK”. Cheers!
							</P></DIV><DIV class="column small-12 large-12 ar12 textcontainer"><P>
								7. “SMART MAP” can be used to make it better.
							</P></DIV></DIV><DIV class="row ar3" id="Module_Forum" data-magellan-target="Module_Forum"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_Forum">Module:Forum</A></H1><P>
								In order to make communication in BioHub easier, we designed the 'Forum' feature to provide users an area to communicate, view others' projects and write comments on them. They are also able to share different ideas and generate new thoughts.
							</P><P>
								In Forum, users can write, edit and remove his comments freely just like operating on any other forums. They will harvest enlightenment as well as fun.
							</P><H2>Project</H2><P>
								We prepared the Project Page for users to view their projects clearly and manage their projects properly. The Project Page serves as a demonstration of users’ projects and others’ comments.
							</P></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								On the page, you can see detailed information of the projects, such as name, author, state, thumbnail, update time, etc.
								
								The 'edit' button leads to the place where you edit your panos, do simulations, path-finding, doing BLAST, etc.
								
								The button with a 'plus' icon provides you with a convenient way to start a new project:
							</P></DIV><DIV class="column small-12 large-12 ar4 textcontainer"><P>
								The button with a 'mailbox' icon provides you a clear view of the comments that others give you:
							</P></DIV><DIV class="column small-12 large-12 ar6 textcontainer"><P>
								Interesting? Then use the Project Page as the start of all.
							</P><H2>Forum</H2><P>
								In order to make communications in BioHub easier, we designed the 'forum' feature to give users a space to communicate, view others’ projects, comment on them, share different ideas and form new thoughts.
							</P></DIV><DIV class="column small-12 large-12 ar8 textcontainer"><P>
								Notice, we provide a place to leave what you want to say as a comment on other users’ projects. Find the icon 'messages' under every projects.
							</P></DIV><DIV class="column small-12 large-12 ar10 textcontainer"><P>
								Type your word as below.
								
								Submit it and have fun:)
							</P></DIV></DIV><DIV class="row ar4" id="Module_Simulation" data-magellan-target="Module_Simulation"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_Simulation">Module:Simulation</A></H1><P>
								After you have run the module &quot;Pathway finder&quot;, you may get some nodes. Here you can use this module to simulate the concentration of the substances involved in this reaction. 
							</P><P>
								Simulation is a simulator designed to solve the concentration function with respect to time based on the differential equations given by the user. After the user has submitted control functions, this module will upload relative data to a high performance server, which will be able to return results in minutes or hours, based on the quantity of control functions and the width of time domain. The concentration of all nodes will be displayed visualized, meanwhile a proper notice will pop up if an instability is detected in order to warn the user to be cautious about the result after the critical point.
							</P><P>
								You can simulate concentration of genes’ products by entering controlling ODEs. There are two ways to access to “Simulation”: Clicking “Simulation” at sidebar, or entering pano project and selecting “Simulation” with some certain nodes. Normally, the second way is more common. Here is the instruction of “Simulation”.
							</P><P>
								1. Enter your pano project and click button 1.
							</P></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								2. On the popped dialog, click “START SIMULATION”.
							</P></DIV></DIV><DIV class="row ar5" id="Module_BLAST" data-magellan-target="Module_BLAST"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_BLAST">Module:BLAST</A></H1><P>
								BioHub provide users with convenient one-key BLAST. In graphical interface, the user select the components which he or she wants to align the sequence, and by just pressing one build-in button, BioHub will read the sequence and align it with sequences in online databases using BLAST interface provided by NCBI, and show the outcome simply and elegantly to the users.
							</P><P>
								Tips: BLAST is available only online. When using NCBI BLAST API, any user should obey their usage guidelines.
							</P></DIV></DIV><DIV class="row ar6" id="Module_ABACUS" data-magellan-target="Module_ABACUS"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_ABACUS">Module:ABACUS</A></H1><P>
								A plugin design to design amino acid sequence from the .pdb file provided by user. The uploaded .pdb file will be evaluated by several progress, with support of simulated annealing algorithm, then the server will generate a .zip file containing several .pdb files and .fasta files for user to download. It’s recommended to run this plugin on a high performance server, because the plugin will invoke computationally intensive programs. So please be patient while this plugin is running, it may take several hours to get results on a high performance server. You can press get status button to check if the calculation is done.
							</P><P>
								ABACUS can be a really useful function, because for synthetic biologists who want to design new protein, they may need to know the amino acid sequence for proteins in specific structure. For some researchers, they also need to design new protein which should in maximum degree resemble their original structure, which is needed to purify the protein. This steps can be really time-consuming, because if you make some mistakes in designing amino acid sequences, months of experiments can be of no use.
							</P><P>
								Users can design amino acid sequence from the provided structure. Just input the number of result files and tag name (optional), then upload the .pdb file(which represents the structure) and click UPLOAD AND CALCULATE. Then the server will generate amino acid sequence file you want. Just follow the steps below, then your file can be evaluated. And there is one thing you should pay attention to: Every time you upload pdb file, the server will clean old .pdb file and overwrite old results.
							</P></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								While server is calculating, you can click GET STATUS to know whether processes are done. If your file is still being processed, you will get a dialog below:
							</P></DIV><DIV class="column small-12 large-12 ar4 textcontainer"><P>
								If files are generated successfully, the 'GET STATUS' button will change to 'DOWNLOAD FILE', then you can get your result as a .zip file. 
							</P><P>
								There is possibility that something go wrong during designing, so if any error happens, you will get a dialog about error information.
							</P></DIV></DIV><DIV class="row ar7" id="Module_Biobrick_manager" data-magellan-target="Module_Biobrick_manager"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Module_Biobrick_manager">Module:BioBrick manager</A></H1><P>
								The iGEM Registry has over 20,000 documented parts organized by type, chassis, function and more. Recognizing that this is a great asset in the field of synthetic biology, we believe that it is necessary to let users explore BioBricks on BioHub. As a result, this function was implemented as an official plugin for BioHub. Users can search for BioBricks they desire and easily browse all the information about the searching results.
							</P><P>
								Users can directly visit BioBrick page from the 'BioBrick' button in the navigation bar of homepage, project manage Page or Forum’s right side when they are in need of searching some BioBricks. This page provides a search Widget in which user can input keywords to search for BioBricks.
							</P></DIV><DIV class="column small-12 large-12 ar2 textcontainer"><P>
								This user-friendly UI based on Google’s Material Design implemented from MaterializeCss. The neat and simple workspace can provide a more efficient work environment which is important for researchers.
							</P><P>
								All needed to be done is inputting keywords and hitting ENTER, and the result will appear in no time like this.
							</P></DIV><DIV class="column small-12 large-12 ar4 textcontainer"><P>
								In search result, we provide a hint according to the SBOL visual standard by analyzing the description of the brick. Also, if the user is interested in the BioBrick and want more intel, he or she can click on the link icon to jump to the official site of iGEM for more detailed information. Export of SBOL version xml file is provided, just click the SBOL download key, and download the SBOL form file.
							</P></DIV></DIV><DIV class="row ar8" id="Plugins" data-magellan-target="Plugins"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Plugins">Plugins</A></H1><P>
								The great thing about BioHub is that we have designed a unified, extensible plugin platform, which not only have the powerful tools provided by the development team, but also is easy to learn and use by other developers to provide better features. Each specific feature of BioHub is provided by a plugin, whose developer do not need to care about details such as server and network requests. Plugins can just focus on performing their own tasks and manage user's design results. With BioHub's plugin platform, once any of the latest tools or algorithms in the field of synthetic biology got programmed, developers can easily transform it into a BioHub plugin by studying the sample files we provide, making it a new blade of the powerful Swiss Saber - BioHub.
							</P></DIV></DIV><DIV class="row ar9" id="Docker" data-magellan-target="Docker"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Docker">Docker</A></H1><P>
								In order to make BioHub easy to install, configure and use, we carefully selected the program architecture and package format. The use of server -  client architecture makes it possible for labs to run the server on a high-performance computer, and easily access it from any device. Also, we use Docker to package the program, which makes the server very easy to start, update and shutdown. Normally, users may never encounter any compatibility issues. What's more, we also released a virtual machine image file for non-Linux platforms.
							</P></DIV></DIV><DIV class="row ar10" id="Outlook" data-magellan-target="Outlook"><DIV class="column small-12 large-12 ar0 textcontainer"><H1><A href="#Outlook">Outlook</A></H1><P>
								Since our project is a software designed for synthetic biologists to simplify their work as well as give them effective auxiliary, there are undoubtedly dozens of chances of this software to be improved, just like the previous work 'Biopano'. In the future, we will upgrade our program in order to implement more functions as well as enhance the current ones. For example, we may enable the 'Pathway finder' module to find the strength of the connection between genes, which our software cannot do this at present. Meanwhile, we will introduce our software and its background to our juniors to attract them to join us and to participate in the following iGEM competitions.
							</P></DIV></DIV></DIV></DIV></DIV><FOOTER class="row ar1"><DIV class="column small-12 large-2 large-offset-5 ar1"><H2>Links</H2><P><A href="http://www.ustc.edu.cn">USTC</A></P><P><A href="https://2014.igem.org/Team:USTC-Software">Biopano</A></P></DIV><DIV class="column small-12 large-2 large-offset-1 end ar2"><H2>Contact Us</H2><P>igemustc2016@gmail.com</P><P>No.96, JinZhai RD., Hefei, Anhui, PRC.</P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>