<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Semantic_containment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Semantic_containment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Semantic_containment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Semantic_containment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Semantic_containment&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Semantic_containment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Semantic containment</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Semantic containment</DIV><TABLE id="tableboxed"><TBODY><TR><TD><P><B>Aims</B></P><UL><LI> Creating a semantic containment system to prevent gene expression in natural organisms
</LI><LI> Characterize the system
</LI><LI> Use this system in all genes of the system, the critical genes first (e.g. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Suicide" class="external text" rel="nofollow">colicin</A>)
</LI></UL><P><B>System</B></P><UL><LI>An amber codon (stop codon) embedded in protein genes to prevent their expression and an amber suppressor system in our genetically engineered bacteria
</LI></UL><P><B>Achievements :</B></P><UL><LI> Construction and characterization of 2 biobricks :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914000" class="external text" rel="nofollow">K914000</A> : P<SUB>Lac</SUB>-supD-T : tRNA amber suppressor
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914009" class="external text" rel="nofollow">K914009</A> : P1003* Ser133-&gt;Amber Codon : kanamycin gene resistance with 1 amber mutation
</LI></UL><P>Both part were well <A href="#Results">characterized</A> and works well. For the second parts, we show that as expected, one mutation is quite leaky, although it works qualitatively, but one mutation is not enough if we want to release such parts in nature. Other reasons emphasize this observation, notably the weakness of being at one mutation to recover the protein functionality.
</P><UL><LI> Creation of a new category in the part registry : <A href="http://partsregistry.org/Biosafety" class="external text" rel="nofollow">Semantic containment</A>. The aim of this category is to let people improving each part by adding for instance other amber mutations to existing part to increase the containment.
</LI></UL><DIV id="boston"><P><B>Achievements : </B></P><UL><LI> Construction and characterization of 1 biobrick :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914018" class="external text" rel="nofollow">K914018</A> : P1003** Ser133 &amp; Ser203 -&gt;Amber Codon : kanamycin gene resistance with 1 amber mutation
</LI></UL><LI> Construction of 1 plasmid backbone : 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914012" class="external text" rel="nofollow">K914012</A> : pSB1A2 with one Amber Codon : Ampicillin gene resistance with 1 amber mutation
</LI></DIV></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Objectives"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#How_to_choose_between_serine_and_tyrosine_.3F"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">How to choose between serine and tyrosine ?</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Weakness_calculation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Weakness calculation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Can_the_tRNA_rescues_the_KanR_phenotype_.3F_.28Qualitative_experiment.29"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Can the tRNA rescues the Kan<SUP>R</SUP> phenotype ? (Qualitative experiment)</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Is_it_working_well.3F_Is_the_amber_mutation_leaky.3F_.28Quantitative_experiment.29"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Is it working well? Is the amber mutation leaky? (Quantitative experiment)</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Experiments_and_results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Experiments and results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Qualitative_characterization_of_K914000_.28supD.29_and_K914009_.28Kan.2A.29"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Qualitative characterization of K914000 (supD) and K914009 (Kan*)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Quantitative_characterization_of_K914000_.28supD.29_and_K914009_.28Kan.2A.29"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Quantitative characterization of K914000 (supD) and K914009 (Kan*)</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Experimental_setup"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Results"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Quantitative_characterization_of_K914018_.28Kan.2A.2A.29"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Quantitative characterization of K914018 (Kan**)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Comparison_between_one_and_two_mutation"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Comparison between one and two mutation</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Conclusion_.26_Perspectives"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Conclusion &amp; Perspectives</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#References_.26_Appendix"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References &amp; Appendix</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Appendix"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Appendix</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Genotype_of_strain_used"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">Genotype of strain used</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Other_amino-acids_weakness_score"><SPAN class="tocnumber">6.2.2</SPAN><SPAN class="toctext">Other amino-acids weakness score</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>We want to prevent our genetic construct from conferring an advantage to other organisms or alter them phenotypically. Minimizing horizontal gene transfer (HGT), either by conjugation, transduction, or transformation is thus our main concern. As these processes involve two parties, the genetically modified bacteria and some wild type population partners, and as we will not modify wild type populations, we cannot assume that HGT is fully avoidable. Semantic containment<SUP>[</SUP><SUP><A href="#References">1</A></SUP><SUP>]</SUP> means that our bacteria won't be able to &quot;speak&quot; with other organisms, since they don't speak the same &quot;language&quot;, the language being DNA. Our system will read the stop codon TAG as the amino-acid serine. It means that in our bacteria the stop codon will be translated into a serine, whereas in wild type bacteria this protein will be truncated and will not confer an advantage to these cells. The 'TAG' codon has been chosen because of its low frequency in <I>E. coli</I> genome (314 occurrences), and also because for further applications, Church Lab tries to remove all amber codon of an <I>E. coli</I> strain<SUP>[</SUP><SUP><A href="#References">2</A></SUP><SUP>]</SUP>. Although it has been demonstrate that the over-expression of a tRNA amber suppressor sole does not affect its growth rate nor the morphology of <I>E. coli</I><SUP>[</SUP><SUP><A href="#References">3</A></SUP><SUP>]</SUP>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 1 :</B> Semantic containment principle. The information carried by the DNA cannot be read by other organism</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><P>We want to create a new way to contain semantically genes, by replacing an amino-acid codon by a stop codon (amber codon, 'TAG'), and in our synthetic cell this stop codon will be read as the amino-acid. Here we want to show that this system works as expected. First we had to choose between two tRNA amber suppressor, either serine, or tyrosine that are available in the part registry. For that we calculate the abilities for the amber codon to reverse to a serine or tyrosine or related amino-acid that could conserve the function. Secondly we create a biobrick with a tRNA Amber suppressor (<A href="http://partsregistry.org/Part:BBa_K914000" class="external text" rel="nofollow">BBa_K914000</A>), in order to have a reliable biobrick, with characterization of it. Thirdly, to test the latter biobrick, we built a biobrick which is a kanamycin resistance gene (P1003) with one amber mutation added instead of the serine 133 (<A href="http://partsregistry.org/Part:BBa_K914009" class="external text" rel="nofollow">BBa_K914009</A>).
</P><P>K914000 is the construction P<SUB>Lac</SUB>-supD-T, and is named supD in the rest of the page. K914009 is the P1003 gene (kanamycin resistance gene) with the serine 133 which is replaced by a amber codon 'TAG'. Its name is Kan<SUP>*</SUP>.
</P><P>With more time we will try to increase the robustness of this system, which is null when the tRNA amber suppressor is transferred too. We will try to create a new library of plasmid backbones in the part registry, where all backbones have at least two amber mutations. The idea is that all the community will be able to improve this library, either by adding new contained backbones, or by adding amber mutations on the same backbone, or add semantic containment to any other gene.
</P><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><H3><SPAN class="mw-headline" id="How_to_choose_between_serine_and_tyrosine_.3F">How to choose between serine and tyrosine ? </SPAN></H3><P>At one mutation of the 'TAG' codon we can have one serine or two tyrosine (among others). Even if the serine seems more interesting by this simple observation, we still want to know which of these two amino-acids is the less robust to mutation from a 'TAG' codon (amber codon) including similarity property of amino-acids. We calculated a score of weakness. The weakness of an amino-acid is defined here by its abilities to not revert to the same amino-acid or any other similar, from the amber codon.
The score is calculated using the following formula :
</P><P>Where i is one of the nine amino-acids accessible after 1 mutation. Subst(AA,AAi) calculate the similarity score, using a BLOSUM100 matrix, between serine or tyrosine (AA) and one of the nine amino-acids around (AAi). The lowest score is the weakest. The BLOSUM100 (BLOck SUbstitution Matrix) is constructed using local alignment of sequenced less than 100% identical<SUP>[</SUP><SUP><A href="#References">4,5</A></SUP><SUP>]</SUP>, and is consequently adapted to appreciate the effect of a single mutation.
</P><H3><SPAN class="mw-headline" id="Weakness_calculation">Weakness calculation</SPAN></H3><P>We wrote a Python script that can calculate the score of weakness, with all amino-acids (not only serine or tyrosine), and will sort a list of score. Data shown in <A href="#Other_amino-acids_weakness_score">appendix</A> for other amino-acids.
</P><P>The scores are, with a mutation rate of 10<SUP>-9</SUP> for the serine and tyrosine:
</P><UL><LI>BLOSUM100 :
</LI><LI> Serine : Score<SUP>S</SUP><SUB>W</SUB> = -3.33e-09
</LI><LI> Tyrosine : Score<SUP>Y</SUP><SUB>W</SUB> = -1.00e-09
</LI></UL><P>The amber codon is less likely to revert into a serine or similar.
</P><P>Also, we favored serine replacement over tyrosine, because the frequency of serine in the <I>E. coli</I> genome is superior to the tyrosine one. It turns out that S has 57,88 codons over 1000 codons when Y has 28,59 codons over 1000 codons (<A href="http://openwetware.org/wiki/Escherichia_coli/Codon_usage" class="external text" rel="nofollow">Codon usage</A>). Therefor, it might be more convenient to replace a serine than a tyrosine, because proteins would more likely to contain many serines than tyrosines.
</P><H3><SPAN class="mw-headline" id="Can_the_tRNA_rescues_the_KanR_phenotype_.3F_.28Qualitative_experiment.29">Can the tRNA rescues the Kan<SUP>R</SUP> phenotype ? (Qualitative experiment)</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:SCsupD.gif" title="File:SCsupD.gif">File:SCsupD.gif</A><DIV class="thumbcaption"><B>Figure 3 :</B> Our hypothesis : supD can rescue the Kan<SUP>R</SUP> phenotype. <B>A</B> production of the mRNA from the Kan* gene. <B>B</B> The mRNA is translated only if there is a tRNA amber suppressor (supD)</DIV></DIV></DIV></DIV><P>To do so, we will transform a plasmid with a the Kan<SUP>*</SUP> gene into a MG1655 strain that contain either pSB1C3::supD, or pSB1C3::RFP. We plate them on Chloramphenicol and Kanamycin. The Kan<SUP>*</SUP> is supposed to be non functional without an amber suppressor.
</P><H3><SPAN class="mw-headline" id="Is_it_working_well.3F_Is_the_amber_mutation_leaky.3F_.28Quantitative_experiment.29">Is it working well? Is the amber mutation leaky? (Quantitative experiment)</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2 :</B> Schema of the four strains used for the quantitative experiments</DIV></DIV></DIV><P>
We used the following strains, all in the <I>E. coli</I> strain MG1655. To quantify the leakiness in terms of expression of the Kan<SUP>*</SUP> gene, we performed real time experiment, where we measured the growth rate (through OD<SUB>600</SUB> measurement) of each of these strains in different concentrations of kanamycin : 
</P><P>(1) <B>pSB1A1::Kan<SUP>R</SUP> + pSB1C3::supD</B>: positive control : Expresses constitutively kanamycine resistance gene.
</P><P>(2) <B>pSB1A1::Kan<SUP>*</SUP> + pSB1C3::supD</B>: construction : Is supD as efficient as the positive control?
</P><P>(3) <B>pSB1A1::Kan<SUP>*</SUP> + pSB1C3::RFP</B>: construction : Is Kan* as unefficient as the negative control?
</P><P>(4) <B>pSB1A2::P<SUB>Lac</SUB> + pSB1C3::supD</B>: negative control : No kanamycin resistance gene.
</P><P>In case of leakiness, the Kan* + RFP strain (3) will be able to grow in higher concentration of kanamycin than the negative control (4).
</P><H2><SPAN class="mw-headline" id="Experiments_and_results">Experiments and results</SPAN></H2><H3><SPAN class="mw-headline" id="Qualitative_characterization_of_K914000_.28supD.29_and_K914009_.28Kan.2A.29">Qualitative characterization of K914000 (supD) and K914009 (Kan*)</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3 :</B> Number of CFU/µg of plasmid after the different concentration</DIV></DIV></DIV><P>
After preparing electro-competent M1655 cells with either pSB1C3::supD or pSB1C3::RFP. We transform the plasmid pSB1A1::Kan<SUP>*</SUP> in both competent cells. After transformation, cells are plated on Cm+Kan.
</P><P>We can observe that without any plasmids transformed no cells grow, or when we transform another plasmid with no Kan resistance gene, but with an Amp gene resistance (and plated on Cm+Amp), colonies appear in the strain with RFP, unfortunately the other control (with supD) did not work this time, hence there is no picture of it. But for the quantitative experiment we need that control too (4), and we manage to do it that times. But the pSB1A1::Kan<SUP>*</SUP> can express the kanamycin resistance phenotype only in the strain containing the supD gene.
</P><P>We can conclude here that the supD gene can rescue the phenotype Kan<SUP>S</SUP> by allowing the correct expression of the kan gene P1003.
</P><CENTER><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>plates of the double transformation</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>MG1655 + pSB1C3::supD only   Selection : Cm+Kan
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>MG1655 + pSB1C3::RFP only   Selection : Cm+Kan
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>MG1655 + pSB1C3::supD + pSB1A1::Kan<SUP>*</SUP>   Selection : Cm+Kan
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>MG1655 + pSB1C3::RFP + pSB1A1::Kan<SUP>*</SUP>   Selection : Cm+Kan
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>MG1655 + pSB1C3::RFP + pSB1A2::pLac<SUP>*</SUP>    Selection : Cm+Amp
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H3><SPAN class="mw-headline" id="Quantitative_characterization_of_K914000_.28supD.29_and_K914009_.28Kan.2A.29">Quantitative characterization of K914000 (supD) and K914009 (Kan*)</SPAN></H3><P>Here we characterize both biobrick quantitatively. First, we are going to confirm the qualitative result for the part K914000 and then we will determine how a single amino-acid substitution is leaky.
</P><H4><SPAN class="mw-headline" id="Experimental_setup">Experimental setup</SPAN></H4><P>Once the construction is made (Fig. 2), we double transformed the plasmids into MG1655 strains, that does not contain any amber suppressor. We will work with three replicates of each strains, and for each strain we will be in 8 different conditions of antibiotic resistance. The antibiotic used is kanamycin, an aminoglycoside interfering with the translation. The range of concentration goes from 4 times as much as the usual concentration (100 µg/mL) to  8 times less. The 96 wells plate is then incubated in a plate reader that take measurements of OD<SUB>600</SUB> every 6 minutes. This measure is correlated to the number of cell in the well. 
Each well contains 200µL of LB (Lysogeny broth, aka Luria Bertani), chloramphenicol and ampicilin at their usual concentration, the dilution of cells and different amount of kanamycin. An overlay of 50µL of mineral oil is added on the top. The measurement lasted approximately 16 hours and 30 min.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 4 :</B> The over night (O/N) culture is diluted twice, first to normalize all samples, and then to start with low concentration of cells.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>First we observed that the qualitative result is reproduce here, the supD gene rescues the kanamycin resistance at any concentration of the antibiotic. It shows also that there is no disadvantage to use the supD amber suppressor compared to the wild type kanamycin gene resistance, since there is no difference of growth rate (Figure 4B and 4C). However the leakiness of the Kan* gene is higher than expected, and thus one mutation is not sufficient for the containment. Indeed, at usual concentration, Kan* gene manages to express an antibiotic resistance, even though lower than Kan<SUP>R</SUP> or supD (Figure 4A). Here we define the growth rate as the OD<SUB>600</SUB> observed at a given time (here t=8h20'), in order to overcome the fact that Kan*+RFP does not have a clear exponential phase. The fact that the strain (3) grows faster and can have higher OD<SUB>600</SUB> may be explain by the fact that supD is deleterious for the strain, by removing some stop codon of other genes for instance. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 4A and 4B:</B> The bar graph represent the OD<SUB>600</SUB> at time = 8h20' (black line Figure 4C) for the different strains at a given kanamycin concentration (black line on Figure 4B). On B, we observe the variations of the OD<SUB>600</SUB> at different kanamycin concentrations, at the same time.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 4C:</B> The variations of the OD<SUB>600</SUB> is observed in function of the time, for different concentrations of kanamycin (400µg/mL, 100µg/mL, 25µg/mL).</DIV></DIV></DIV></DIV><DIV id="boston"><H3><SPAN class="mw-headline" id="Quantitative_characterization_of_K914018_.28Kan.2A.2A.29">Quantitative characterization of K914018 (Kan**)</SPAN></H3><P>Here we perform the exact same experiment than the previous one, except that kanamycin resistant gene is contained with two amber mutations instead of one. It is observed that the positive and negative controls behave correctly, and similarly to the previous experiment, it is also the case for the complementation of Kan** with supD. That means that two mutation are not a problem for the cell. However, we notice that there is no more leakiness in the system at usual concentration, even at low concentration. Actually the double kanamycin resistant gene with two mutation behave exactly like the negative control (Plac + supD) which means that semantic containment is working well.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 5A and 5B:</B> The bar graph represent the OD<SUB>600</SUB> at time = 8h37' (black line Figure 5C) for the different strains at a given kanamycin concentration (black line on Figure 5B). On B, we observe the variations of the OD<SUB>600</SUB> at different kanamycin concentrations, at the same moment.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:803px;"><DIV class="thumbcaption"><B>Figure 5C:</B> The variations of the OD<SUB>600</SUB> is observed in function of the time, for different concentrations of kanamycin (400µg/mL, 100µg/mL, 25µg/mL and 3.125µg/mL).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Comparison_between_one_and_two_mutation"> Comparison between one and two mutation </SPAN></H3><P>Here is a graph that gather both results, to highlight the effect of the second mutation. The results are normalized intra group (experiment with the one-mutated gene, and with the two-mutated gene) in order to have the same scale to compare them. We took the average of the positive (KanR+supD) and the negative (Plac+supD) controls. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 6:</B> Variation of OD<SUB>600</SUB> in different concentrations of [Kan] (µg/mL) at t= 8,37h</DIV></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Conclusion_.26_Perspectives">Conclusion &amp; Perspectives</SPAN></H2><P>This work demonstrates that semantic containment can be achieved by changing an amino-acid into an amber codon. We saw that one codon replacement is not enough because of some leakiness, and demonstrated that upon the replacement of two codons, the system isn't leaky at all. It means that for a robust semantic containment system we need 3 codon replacements, because with two replacements we are only one mutation away of a leaky expression. The underlying idea is <I>in fine</I> to create a library of semantic-contained genes and backbones that would be improved by anybody, by either adding new semantic systems, or increasing the degree of containment by adding mutations on already existing parts. This library is already started with the previously described part and with a plasmid backbone that has only one mutation so far. But we also showed that tRNA amber suppressor might be deleterious for the cell. This trade-off between the leakiness and the toxicity has to be studied in order to optimize the system.
</P><P>Further experiments should improve the robustness of this system by adding a new security component that would ensure that as much as three genes would have to be transferred by HGT, to express synthetic gene in natural bacteria, thus cousing our containment system to fail. In our current design, the system fails if the tRNA amber suppressor gene is transferred with another semantic gene into a natural bacteria. The idea is to add a semantic containment for the tRNA. Since it is a tRNA, we cannot replace amino-acids because it is not translated. In order to fix that problem we will construct the following system. The tRNA supD gene will be under a T7 promoter which is orthogonal, meaning that it needs a special RNA polymerase (T7 RNA polymerase) to transcribe the gene (here it would be supD), but we would mutate the T7 RNA polymerase with several amber mutations. It means that we would need to transfer these two genes with the semantically contained gene in order to have something functional in the other organism, which is very unlikely to happen. Further experiments will elucidate the probability of such an event. A scheme of this system is depicted in Figure 5.
This system needs to be activated in the lab, for example by transforming a plasmid with the wild type T7 RNA polymerase gene with another antibiotic gene resistance, say X. Then we would remove antibiotic X and loose the plasmid with wild type T7 polymerase when the positive feedback loop starts. We could also activate the system by transforming a wild-type T7 RNA polymerase carrying plasmid with conditional origin of replication, such as temperature-sensitive plasmids.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 5 : </B>Once the mRNA of the T7 RNA polymerase is transcribed (A), it needs the tRNA amber suppressor (B) to let the ribosome translate the mRNA into a functional protein. Then the T7 RNA polymerase will be able to transcribe the supD gene into this tRNA amber suppressor (C). Then it is a positive feedback loop, we need to start this system by adding either tRNA in the medium or T7 RNA pol wild type gene.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References_.26_Appendix">References &amp; Appendix</SPAN></H2><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><A href="#Overview">1</A> - Marliere, P. The farther, the safer : a manifesto for securely navigating synthetic species away from the old living world. System and Synthetic Biology 3, 77-84 (2009). <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2759432/" class="external text" rel="nofollow">Paper</A></P><P><A href="#Overview">2</A> - Isaacs, F.J. et al. Precise manipulation of chromosomes in vivo enables genome-wide codon replacement. Science (New York, N.Y.) 333, 348-53 (2011). <A href="http://arep.med.harvard.edu/pdf/Isaacs_Sci_11.pdf" class="external text" rel="nofollow">Paper</A></P><P><A href="#Overview">3</A> - Anderson, J.C., Voigt, C. a &amp; Arkin, A.P. Environmental signal integration by a modular AND gate. Molecular systems biology 3, 133 (2007). <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1964800/" class="external text" rel="nofollow">Paper</A></P><P><A href="#Calculation_of_serine_and_tyrosine_weakness">4</A> - S Henikoff and J G Henikoff. Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci U S A. 1992 November 15; 89(22): 10915–10919. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453/pdf/pnas01096-0363.pdf" class="external text" rel="nofollow">Paper</A></P><P><A href="#Calculation_of_serine_and_tyrosine_weakness">5</A> - Jorja G. Henikoff, Steven Henikoff, Blocks database and its applications, Methods in Enzymology, Academic Press, 266, 88-105 (1996). <A href="http://www.sciencedirect.com/science/article/pii/S007668799666008X" class="external text" rel="nofollow">Paper</A></P><H3><SPAN class="mw-headline" id="Appendix">Appendix</SPAN></H3><H4><SPAN class="mw-headline" id="Genotype_of_strain_used">Genotype of strain used</SPAN></H4><UL><LI> MG1655 : F- λ- ilvG- rfb-50 rph-1 
</LI></UL><H4><SPAN class="mw-headline" id="Other_amino-acids_weakness_score"> Other amino-acids weakness score </SPAN></H4><UL><LI> Weakness score with a BLOSUM 100, mutation rate = 10<SUP>-9</SUP>, initial Codon : TAG
</LI></UL><P><A href="#Weakness_calculation">main result part</A></P><P>('A', [-4.000000002555556e-09])
</P><P>('C', [-6.777777782148149e-09])
</P><P>('D', [-5.2222222257407416e-09])
</P><P>('E', [-3.222222225407407e-09])
</P><P>('F', [-3.0000000040740746e-09])
</P><P>('G', [-6.555555560444445e-09])
</P><P>('H', [-2.888888891555555e-09])
</P><P>('I', [-5.111111115777778e-09])
</P><P>('K', [-2.6666666694074067e-09])
</P><P>('L', [-4.333333337555555e-09])
</P><P>('M', [-4.000000003185186e-09])
</P><P>('N', [-4.3333333354074076e-09])
</P><P>('P', [-5.888888893518519e-09])
</P><P>('Q', [-1.6666666689629633e-09])
</P><P>('R', [-3.888888891888888e-09])
</P><P>('S', [-3.333333334925926e-09])
</P><P>('T', [-4.22222222462963e-09])
</P><P>('V', [-4.888888893111112e-09])
</P><P>('W', [-2.4444444508518513e-09])
</P><P>('Y', [-1.0000000044444444e-09])
</P><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Semantic_containment">http://2012.igem.org/Team:Paris_Bettencourt/Semantic_containment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Semantic_containment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Semantic_containment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Semantic_containment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Semantic_containment&amp;oldid=293562" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>