<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Delay"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Delay">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Delay&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Delay&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Delay&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Delay" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Delay</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Delay system(s)</DIV><TABLE id="tableboxed"><TBODY><TR><TD><P><B>Aim :</B>
A programmed delay will allow the cell to perform its intended function before our DNA-degrading suicide machinery is expressed.
</P><P><B>Experimental system:</B>
We used two different approaches to create this delay. The <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Simple_delay_system" class="external text" rel="nofollow">first one</A> is based on the gradual dilution of a regulatory transcription factor. The <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#sRNA_delay_system" class="external text" rel="nofollow">second one</A> makes use of a stationary-phase specific promoter. Both systems eventually result in the expression of the restriction enzyme I-SceI. In the final design, I-SceI cleaves the antitoxin gene, ultimately dooming the cell. Each step in this causal sequence contributes to the overall delay in the system.
</P><P><B>Achievements :</B></P><UL><LI> Construction and characterization of the dilution delay system
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914004" class="external text" rel="nofollow">K914004</A> : P<SUB>BAD</SUB>-AraC-RBS-LacI ; 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914014" class="external text" rel="nofollow">K914014</A> : P<SUB>BAD</SUB>-AraC-RBS-LacI-P<SUB>lac</SUB>-RBS-GFP-double terminator ; <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Characterization" class="external text" rel="nofollow">characterization</A></LI><LI><A href="http://partsregistry.org/Part:BBa_K914015" class="external text" rel="nofollow">K914015</A> : P<SUB>BAD</SUB>-AraC-RBS-LacI-P<SUB>lac</SUB></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Refined_characterization_of_the_Yokobayashi_et_al._sRNA_repression_plasmidic_device" class="external text" rel="nofollow">Characterization</A> of the sRNA repression system of Yokobayashi <I>et al.</I></LI><LI> Cloning of the yiaGp stationary phase promoter
</LI><DIV id="boston"><UL><LI> Partially biobricked sRNA system : 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914017" class="external text" rel="nofollow">K914017</A> stationary phase promoter Yiagp
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914016" class="external text" rel="nofollow">K914016</A> coding sequence of Colicin E2
</LI></UL></DIV></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Dilution_delay_system"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Dilution delay system</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Objectives"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Design"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Characterization"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Characterization</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#sRNA_delay_system"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">sRNA delay system</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Objectives_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Design_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Cloning_of_the_YiaGp_promoter"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Cloning of the YiaGp promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Characterization_of_the_Yokobayashi_et_al._sRNA_repression_device"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Characterization of the Yokobayashi et al. sRNA repression device</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>The delay system serves to suppress the function of the suicide device and preserve the integrity of the host genome while the organism does its intended job in the environment. We experimented with two designs for producing a programmed delay before the expression of the DNA-degrading suicide machinery.
</P><P>The dilution delay system relies on the lab-specific expression of a transcriptional inhibitor. This inhibitor is induced by a specific compound found in the laboratory but not in the environment. When the cell enters the environment the inhibitor is gradually degraded or diluted by cell growth. Eventually, the repressor concentration falls below a critical threshold, releasing the suicide machinery.
</P><P>The sRNA delay system makes use of a stationary-phase specific promoter. Under laboratory conditions, the suicide machinery is repressed by an inducible sRNA. In the environment, the suicide machinery is expressed as soon as the cells reach stationary phase.
</P><H2><SPAN class="mw-headline" id="Dilution_delay_system">Dilution delay system</SPAN></H2><H3><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H3><P>Our cells need time to work in the environment before we degrade their genomes. A programmed delay circuit faces the following challenges:
</P><UL><LI>The delay must be programmable and long enough for our cells to perform a useful task.
</LI><LI>Once begun, the delay countdown should lead inevitably to death, and should not be reversible.
</LI><LI>Prior to induction, our suicide machinery must be very tightly repressed.
</LI></UL><H3><SPAN class="mw-headline" id="Design">Design</SPAN></H3><P>We chose to base our delay on a simple transcriptional network. The arabinose-activated promoter, pBAD, drives the production of the LacI gene. LacI represses a restriction enzyme at the pLac promoter. In the absence or arabinose, the restriction enzyme is eventually expressed, triggering irreversible cell death.
</P><P>We chose the pLac promoter because of its reputation for tight repression by the lacI protein. Our final design eventually expresses a restriction enzyme, because the DNA hydrolysis reaction is effectively irreversible. But for the purposes of characterization, we substituted GFP for the restriction enzyme at the end of our transcriptional cascade. 
</P><P>The system we created to characterize our delay is shown below:
</P><H3><SPAN class="mw-headline" id="Characterization">Characterization</SPAN></H3><P>We characterized the behavior of our transcriptional circuit by first growing cells in the presence of 1% arabinose induction. In this state, LacI expression is highly induced. The cells were then washed, diluted, and characterized in the presence of arabinose, IPTG, or no induction.
</P><P>The phase between the 'uninduced' and 'IPTG' curves is the delay from dilution of LacI. After removal of arabinose, LacI remains within the cells but soon becomes inefficient at inhibiting GFP expression as the cells divide. The delay is characterized by also adding IPTG to the system, which activates GFP expression despite presence of LacI.
Here, the delay is about 1 hour.
</P><H2><SPAN class="mw-headline" id="sRNA_delay_system">sRNA delay system</SPAN></H2><H3><SPAN class="mw-headline" id="Objectives_2">Objectives</SPAN></H3><P>As an alternative delayed trigger for our suicide machinery we sought to make use of the natural delay experienced by cells as they approach stationary phase. The design requirements for this system are similar to those described above.
</P><UL><LI>The delay must be programmable and sufficiently long.
</LI><LI>Death must follow inevitably one the countdown begins.
</LI><LI>The suicide machinery is highly toxic, and must not be expressed until needed.
</LI></UL><H3><SPAN class="mw-headline" id="Design_2">Design</SPAN></H3><P>To achieve tight repression, our system is augmented with an sRNA. The concept of our design is described below. Similar systems have been shown to respond in a treshold-linear way [2] 
</P><P>We adapted the construct of Yokobayashi <I>et al.</I> [1] described below.
</P><P>The final design of our construct is shown below.
</P><P>The transcription of the toxin is controlled twice: first by the stationary phase promoter of yiaG. It is recognized for transcription only by the sigma-S subunit of RNA polymerase, and not the exponential phase sigma-70 subunit. [3], [4].
</P><P>We have shown that the yiaGp promoter is indeed activated during the stationary phase in our construct, even though the level of trancription is rather low (see experiments)
</P><P>In order to achieve a complete toxin lockdown, we added a second post-transcriptional repression mechanism. We used and modified the constructs of Yokobayashi <I>et al.</I> to allow a repression of the translation using sRNA [1]. Our specific sRNA binds the leader sequence of its target mRNA and not its coding sequence, and allows a 20 fold repression of protein expression. The transcription of the sRNA is under the control of the pBAD promoter.
</P><P>We expect the final sRNA delay system will allow the DNA-degradation machinery to be expressed only by cells in stationary phase and lacking arabinose.
</P><H3><SPAN class="mw-headline" id="Cloning_of_the_YiaGp_promoter">Cloning of the YiaGp promoter</SPAN></H3><P>YiaGp is a late stationary phase promoter. It has been shown to be activated by 10 to 30 fold during to stationary phase, and is among the strongest promoters during this phase. Furthermore, its action has been reported to last for a longer time than most stationary phase promoters. [4]
We managed to replace the P33 promoter with the YiaGp promoter in the pKP33-aOmpF-GFPuv plasmid between the XhoI and EcoRI sites. The sequencing was consistent with the expected sequence. However, the way the original plasmids were designed made them difficult to biobrick (presence of restriction sites in the sequences). <B>We will try to biobrick the entire design in order to both facilitate the cloning and share our construct with the iGEM community.</B></P><P>We still need to characterize this promoter and compare its activity in exponential and stationary phase.
</P><H3><SPAN class="mw-headline" id="Characterization_of_the_Yokobayashi_et_al._sRNA_repression_device">Characterization of the Yokobayashi <I>et al.</I> sRNA repression device</SPAN></H3><P>We double transformed TOP10 cells with the GFP and pBAD-sRNA plasmids (see [1] for details, and click <A href="https://static.igem.org/mediawiki/2012/c/c4/Odparisbett.png" class="external text" rel="nofollow">here</A> for a network schematic). We chose the this sRNA because it shows a strong repression GFP expression, and binds specifically 5' leader sequence. It does not bind the GFP coding sequence, suggesting is could easily adapted for use with other genes.
</P><P>The cells were grown in LB until stationary phase, then diluted 100-fold in LB. After adding defined arabinose concentrations to the medium, we monitored the GFP fluorescence with a TECAN 96-well plate reader. The mean normalized fluorescence after 10 hours is reported below (4 replicates per concentration).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Normalized Fluorescence of GFP with different concentrations of arabinose</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P>1. Sharma, V., Yamamura, A., &amp; Yokobayashi, Y. (2012). Engineering Artificial Small RNAs for Conditional Gene Silencing in Escherichia coli, 6-13.
<A href="http://pubs.acs.org/doi/abs/10.1021/sb200001q|" class="external text" rel="nofollow">Paper</A></P><P>2. Levine, E., Zhang, Z., Kuhlman, T., &amp; Hwa, T. (2007). Quantitative characteristics of gene regulation by small RNA. PLoS biology, 5(9), e229. doi:10.1371/journal.pbio.0050229
<A href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.0050229|" class="external text" rel="nofollow">Paper</A></P><P>3. Sharma, U. K., &amp; Chatterji, D. (2010). Transcriptional switching in Escherichia coli during stress and starvation by modulation of sigma activity. FEMS microbiology reviews, 34(5), 646-57. doi:10.1111/j.1574-6976.2010.00223.x<A href="http://jb.asm.org/content/186/21/7112.long|" class="external text" rel="nofollow">Paper</A></P><P>4. Shimada, T., Makinoshima, H., Ogawa, Y., Miki, T., Maeda, M., &amp; Ishihama, A. (2004). Classification and Strength Measurement of Stationary-Phase Promoters by Use of a Newly Developed Promoter Cloning Vector, 186(21), 7112-7122. doi:10.1128/JB.186.21.7112 <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6976.2010.00223.x/abstract;jsessionid=8C98B1B383242B9DFF45429802F48428.d02t04|" class="external text" rel="nofollow">Paper</A></P><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Delay">http://2012.igem.org/Team:Paris_Bettencourt/Delay</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Delay" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Delay" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Delay&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Delay&amp;oldid=292849" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>