<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Achievements"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Achievements">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Achievements&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Achievements&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Achievements&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Achievements" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Achievements</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Achievements</DIV><TABLE id="tableboxed"><TBODY><TR><TD><B>Semantic containment </B><P><B>Aims</B></P><UL><LI> Creating a semantic containment system to prevent gene expression in natural organisms
</LI><LI> Characterize the system
</LI><LI> Use this system in all genes of the system, the critical genes first (e.g. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Suicide" class="external text" rel="nofollow">colicin</A>)
</LI></UL><P><B>System</B></P><UL><LI>An amber codon (stop codon) embedded in protein genes to prevent their expression and an amber suppressor system in our genetically engineered bacteria
</LI></UL><P><B>Achievements :</B></P><UL><LI> Construction and characterization of 2 biobricks :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914000" class="external text" rel="nofollow">K914000</A> : P<SUB>Lac</SUB>-supD-T : tRNA amber suppressor
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914009" class="external text" rel="nofollow">K914009</A> : P1003* Ser133-&gt;Amber Codon : kanamycin gene resistance with 1 amber mutation
</LI></UL><P>Both part were well <A href="https://2012.igem.org/Team:Paris_Bettencourt/Semantic_containment#Results" class="external text" rel="nofollow">characterized</A> and works well. For the second parts, we show that as expected, one mutation is quite leaky, although it works qualitatively, but one mutation is not enough if we want to release such parts in nature. Other reasons emphasize this observation, notably the weakness of being at one mutation to recover the protein functionality.
</P><UL><LI> Creation of a new category in the part registry : <A href="http://partsregistry.org/Biosafety" class="external text" rel="nofollow">Semantic containment</A>. The aim of this category is to let people improving each part by adding for instance other amber mutations to existing part to increase the containment.
</LI></UL><DIV id="boston"><P><B>Achievements : </B></P><UL><LI> Construction and characterization of 1 biobrick :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914018" class="external text" rel="nofollow">K914018</A> : P1003** Ser133 &amp; Ser203 -&gt;Amber Codon : kanamycin gene resistance with 1 amber mutation
</LI></UL><LI> Construction of 1 plasmid backbone : 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914012" class="external text" rel="nofollow">K914012</A> : pSB1A2 with one Amber Codon : Ampicillin gene resistance with 1 amber mutation
</LI></DIV></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B>Suicide system </B><P><B>Aims :</B>
Implement a kill-switch that features population-level suicide and complete genome degradation. 
</P><P><B>System :</B>
A synthetic toxin-anti-toxin system based on the wild type Colicin E2 operon.
</P><P><B>Achievements :</B>
We showed that Colicin E2 cells induce cell death in sensitive populations, and that these sensitive populations can be protected by providing them with our engineered immunity protein. 
</P><UL><LI> Construction of 2 biobricks :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914001" class="external text" rel="nofollow">K914001</A> : pLac-repressilator RBS-Colicin E2 immunity protein
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914002" class="external text" rel="nofollow">K914002</A> :repressilator RBS-Colicin E2 immunity protein
</LI></UL><P>Part K914001 is well characterized and provides immunity to sensitive cells against the Colicin E2 activity protein, but is leaky. Part K914002 is promoterless and allows users to easily plug in the appropriate promoter for their desired purpose. 
</P><UL><LI> Creation of a new category in the part registry : <A href="http://partsregistry.org/Biosafety" class="external text" rel="nofollow">XNase</A>. The aim of this category is to provide users with DNase/RNase parts that can be used for improved kill switches featuring the degradation of genomic material.
</LI></UL></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B> Restriction Enzyme System </B><P><B> Aim: </B></P><P>To design a plasmid self-digestion system.
</P><P><B>Experimental System:</B></P><P>We are testing different combinations of promoters and restriction enzymes. We have to characterize both the promoters (by measuring the expression of RFP) and the restriction enzymes (by measuring killed cells).
</P><P><B>Achievements :</B></P><UL><LI> Construction of 4 biobricks <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Design" class="external text" rel="nofollow">[Read more]</A>:
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914003" class="external text" rel="nofollow">K914003</A>: L-rhamnose-inducible promoter
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914005" class="external text" rel="nofollow">K914005</A>: Meganuclease I-SceI controlled by pLac
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914007" class="external text" rel="nofollow">K914007</A>: Meganuclease I-SceI controlled by pBad
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914008" class="external text" rel="nofollow">K914008</A>: Meganuclease I-SceI controlled by pRha
</LI></UL><LI> Demonstration that all 3 generators (K914005, K914007, K914008) work and express I-SceI meganuclease in cells. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Measuring_the_efficiency_of_I-SceI_.28Cloned_parts.29" class="external text" rel="nofollow">[Read more]</A></LI><DIV id="boston"><UL><LI><B><A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#New_Results" class="external text" rel="nofollow">[Updated for Boston]</A></B> : Characterization of 2 biobricks from TUDelft <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Measuring_the_I-SceI_efficiency_.28TUDelft_parts.29" class="external text" rel="nofollow">[Read more]</A>:
</LI><LI><A href="http://partsregistry.org/Part:BBa_K175041" class="external text" rel="nofollow">K175041</A>: p(LacI) controlled I-SceI homing endonuclease generator
</LI><LI><A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>: I-SceI restriction site
</LI></UL></DIV><UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Characterization_of_pRha" class="external text" rel="nofollow">Characterization </A> of the L-rhamnose-inducible promoter (<A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Design" class="external text" rel="nofollow">pRha</A>). 
</LI></UL></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B>Delay System </B><P><B>Aim :</B>
A programmed delay will allow the cell to perform its intended function before our DNA-degrading suicide machinery is expressed.
</P><P><B>Experimental system:</B>
We used two different approaches to create this delay. The <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Simple_delay_system" class="external text" rel="nofollow">first one</A> is based on the gradual dilution of a regulatory transcription factor. The <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#sRNA_delay_system" class="external text" rel="nofollow">second one</A> makes use of a stationary-phase specific promoter. Both systems eventually result in the expression of the restriction enzyme I-SceI. In the final design, I-SceI cleaves the antitoxin gene, ultimately dooming the cell. Each step in this causal sequence contributes to the overall delay in the system.
</P><P><B>Achievements :</B></P><UL><LI> Construction and characterization of the dilution delay system
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914004" class="external text" rel="nofollow">K914004</A> : P<SUB>BAD</SUB>-AraC-RBS-LacI ; <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Characterization" class="external text" rel="nofollow">characterization</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay#Refined_characterization_of_the_Yokobayashi_et_al._sRNA_repression_plasmidic_device" class="external text" rel="nofollow">Characterization</A> of the sRNA repression system of Yokobayashi <I>et al.</I></LI><LI> Cloning of the yiaGp stationary phase promoter
</LI><DIV id="boston"><P><B>Achievements :</B></P><UL><LI> Partially biobricked sRNA system : 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914017" class="external text" rel="nofollow">K914017</A> stationary phase promoter Yiagp
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914016" class="external text" rel="nofollow">K914016</A> coding sequence of Colicin E2
</LI></UL></DIV></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B>MAGE </B><P><B>Aims :</B></P><P>Removal of four FseI restriction sites from E. coli MG1655 genome.
</P><P><B>Experimental System:</B></P><P>Using multiplex automated genome engineering (MAGE) - a technique capable of editing the genome by making small changes in existing genomic sequences.
</P><P><B>Achievements:</B></P><P>Proof of concept by introducing a stop codon in the middle of the lacZ gene
</P></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B>Synthetic Import Domain </B><P><B>Aim :</B></P><P>Creation of a novel protein import mechanism in bacteria. 
</P><P><B>Experimental System:</B></P><P>Exploit the natural Colicin import domain fused to any protein at will, dubbed here: &quot;Synthetic Import Domain&quot;.
</P><P><B>Achievements:</B></P><UL><LI>Construction of colicin-like toxin by fusing Colicin D based &quot;Synthetic Import Domain&quot; with DNAse domain of colicin E2
</LI><LI>Constructon of FseI, I-SceI, LuxR active fragment, LacZ alpha fragment, PyrF and T7 RNA polymerase fused to the two types of &quot;Synthetic Import Domains&quot; from Colicin E2 and Colicin D
</LI><LI>Proof of concept with LacZ alpha fragment fused to &quot;Synthetic Import Domain&quot; from Colicin D
</LI></UL></TD></TR></TBODY></TABLE><TABLE id="tableboxed"><TBODY><TR><TD><B>Encapsulation </B><P><B>Aim:</B>
Harness bacteria-containing gel beads to assure cell containment and complement activity of genetic safety systems.
</P><P><B>Experimental system:</B>
Bacterial cells are encapsulated in alginate beads. We used a <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation#Cell_Containment_Assay" class="external text" rel="nofollow">cell containment assay</A> based on plating to assess the release of cells from alginate beads. In addition, we aimed at improving the entrapment of cells through stabilization by polyethyleneimine and covalent cross-linkage by glutaraldehyde. 
</P><P><B>Achievements:</B></P><UL><LI>Encapsulated cells achieved and their ability to propagate and express proteins within alginate beads demonstrated.
</LI><LI>Stabilized alginate beads by covalent cross-linkage achieved and their ability to entrap cells demonstrated.
</LI><LI>we performed <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation#Bristol_2010_Nitrate_Reporter" class="external text" rel="nofollow">additional characterization</A> of the Bristol 2010 nitrate reporter <A href="http://partsregistry.org/Part:BBa_K381001" class="external text" rel="nofollow">K381001</A></LI><LI> Efficient killing by colicin producing cells was achieved within the beads.
</LI></UL><DIV id="boston"><P><B>Achievements : </B></P><UL><LI>Covalently stabilized alginate beads by polyethyleneimine and glutaraldehyde <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation#Covalent_Stabilization_of_Alginate_Beads" class="external text" rel="nofollow">treatment</A>.
</LI><LI>Demonstrated cells' ability to <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation#Blue-White_Screening_Assay" class="external text" rel="nofollow">propagate and express proteins</A> within stabilized alginate beads.
</LI><LI>Performed <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation#Cell_Containment_Assay" class="external text" rel="nofollow">cell containment assay</A> that demonstrated stabilized beads' enhanced cell entrapment capacity in phosphate buffer.
</LI></UL></DIV></TD></TR></TBODY></TABLE><DIV id="boston"><B>Safety Assessment</B><P><B>Aims</B></P><P>Safety is an important issue in synthetic biology, especially for environmentally related projects. We started to answer the question, “how safe is safe enough?” by involving experts, the public and our fellow scientists, and also by building biosafety devices. However, to really answer the question, we need first to ask ourselves a more basic question, “how do we measure safety?”. As we see synthetic biology as an engineering approach to biology, we could think about the adaptation of safety engineering, a well studied engineering subset, that has been widely use to minimize risks in many fields of engineering, such as mechanical engineering, aircrafts, and manufactures. However, the risks they face are surely different from the risks of synthetic biology. 
</P><P><B>Objectives</B></P><UL><LI> Adapting existing safety assessment tools for synthetic biology
</LI><LI> Proposing new methods to assess safety in synthetic biology
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Human_Practice">Human Practice</SPAN></H2><TABLE id="tableboxed"><TBODY><TR><TD><P><B>Aims</B></P><P>Human concerns arose organically during the construction of the bWARE containment system, and human practices were intrinsic to every stage of our project. In designing and building our best genetic containment system, we often encountered limits on the ability of science alone to measure our performance. When is a biosafety system safe enough? The answer to this question is partially scientific, to the extent that horizontal gene transfer events can be observed and modeled. But the answer is also social, because ultimately the public will decide if a biosafety system works well enough to use. The only way for us to know if bWARE is a success is in conversation with experts and the community.
</P><P>We propose and implement new ways for iGEM to organize and present biosafety information, both for scientists and the public. We believe our reforms to the BioBrick registry will help synthetic biologists to find the best biosafety tools for their application. We also imagine the beginnings of a quantitative, context-specific biosafety database serving citizen scientists. Practical safety data will feed an informed public forum.
</P><P><B>Methodology</B></P><OL><LI><I><B>Interviews with experts</B></I> which enabled us to have a broad overview of the state of the art. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Interview" class="external text" rel="nofollow">Read More</A></LI><LI><I><B>Interaction with high-schoolers</B></I> to have first-hand appreciation of reactions from first exposure to synthetic biology
</LI><LI><I><B>We screened previous iGEM team’s wikis</B></I> to trace the evolution of biosafety concerns and devices in the iGEM community, focusing on proposed containment systems. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/WikiScreen" class="external text" rel="nofollow">Read More</A></LI><LI><I><B>We focused on horizontal gene transfer as the main generic risk factor</B></I>. 
</LI><LI><I><B>Comprehensive report</B></I> where we addressed the concerns raised by synthetic biology per se, that is, as a technique. Then, we analyzed the specific concerns that arise from synthetic biology’s potential applications in nature.  <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Report" class="external text" rel="nofollow">Read More</A></LI></OL><P><B>Main Conclusions</B></P><OL><LI> Societal interaction: 
</LI><LI><I><B>The need to raise awareness</B></I> of synthetic biology in the population so people can decide in the most enlightened way possible what they want from this new technology and of its applications (A),
</LI><LI><I><B>The need of a discussion</B></I> between society’s different protagonists to set goals, define what they would consider as benefits and acceptable risks (B),
</LI><LI> Best research practice:
</LI><LI><I><B>Zero risk is impossible to achieve</B></I> as no containment system can be 100% safe (bacteria can always escape by mutations)  (C),   
</LI><LI> There is a <I><B>lack of quantitative data evaluating the probability of failure of any synthetic biology engineered system, in particular containment systems</B></I> (D),
</LI><LI>  There is a <I><B>lack of quantitative data evaluating the risk of HGT assuming containment systems failed</B></I> (E),
</LI><LI> The compiling of the wiki screen shows that <I><B>no containment systems created in iGEM is robust</B></I>: they lack the above quantification and are mostly one mutation away from failure. We call for major effort of the iGEM community to quantify available containment systems and search for new solutions (F),
</LI><LI><I><B>The need for an INDEPENDENT cohort of scientists</B></I> to test experimentally any application of synthetic biology that requires releasing in the environment (G), 
</LI></OL><P>You can find the full list of conclusions <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Report" class="external text" rel="nofollow">here</A></P><P><B>Main Proposals</B></P><OL><LI> Societal interaction: 
</LI><LI><I><B>Organizing a workshop</B></I> on synthetic biology and a tour of our lab for 60 high school students, (addresses issue A and B) <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Workshop" class="external text" rel="nofollow">Read More</A>. First initiative for teaching synthetic biology in French high-school leading to a high-school iGEM team. Ultimately, we would like interaction with high school or middle school students to be a requirement for an iGEM gold medal. 
</LI><LI><I><B>Organizing a debate</B></I> with 10 non expert students from various background, and then opening the debate to the floor (the public), which was made up of both experts and non experts, (addresses issue A and B) <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Debate" class="external text" rel="nofollow">Read More</A>. 
</LI><LI><I><B>Creating a page to explain horizontal gene transfer</B></I> to non scientists. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/HGT" class="external text" rel="nofollow">Go to HGT page</A></LI><LI> Best research practice:
</LI><LI><I><B>Creating a system as robust as possible</B></I>, that is many mutations away from failure (this is what our <A href="https://2012.igem.org/Team:Paris_Bettencourt/Overview" class="external text" rel="nofollow">bench work</A> has been all about) (addresses issue C and F),
</LI><LI><I><B>Creating a safety page on the biobrick registry</B></I> where all the safety devices that exist are listed and characterized (included evaluation of their robustness) in order for iGEM teams to pick the most appropriate device to add to their newly created genetic circuit. Ultimately, we would like <I><B>the integration of safety modules and risks assessments to be part of of every synthetic biology project from the very start</B></I> (already listed in the safety page or created de novo by the team)   (addresses issue D, F), <A href="http://partsregistry.org/Biosafety" class="external text" rel="nofollow">Go to safety page</A></LI><LI>The community has to <I><B>build a collection of bio-safety devices for future engineers</B></I></LI><LI>Each synthetic biology application should <I><B>assess and disclose a list of application-specific risks and hazards</B></I>.
</LI><LI>Development and adoption of a <I><B>safety chasis for synthetic biology research and prototyping</B></I>.
</LI></OL><P>You can find the full list of proposals <A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/Report#III_Proposals" class="external text" rel="nofollow">here</A></P><DIV id="boston"><P><B>Achievements : </B></P><UL><LI> Team aWAREness
</LI></UL><P>During this summer, all of us gained knowledge in synthetic biology and learned lab skills, but that wasn't all. From the beginning of our brainstorming sessions, safety questions came up in our discussions. Our mutual interest in this topic lead us to center our project on safeguard systems and human practices related to public awareness and risk assesssment. This meant that we had to work hard not only on our wet lab project, but also on human practices. To our delight, this effort resulted not only in community outreach, but also changed our own opinion on biosafety in the context of synthetic biology. We feel that our Human Practice project changed each and every one of us.<A href="https://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/perception" class="external text" rel="nofollow">Here are our personal perceptions.</A></P></DIV></TD></TR></TBODY></TABLE><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Achievements">http://2012.igem.org/Team:Paris_Bettencourt/Achievements</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Achievements" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Achievements" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Achievements&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Achievements&amp;oldid=292738" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>