<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Human_Practice_WikiScreen"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Human_Practice/WikiScreen&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Human_Practice/WikiScreen&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Human_Practice/WikiScreen&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Human_Practice/WikiScreen" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Human Practice/WikiScreen</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Wiki Screen</DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Objectives"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Procedure"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Conclusion"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Appendix"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Appendix</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><OL><LI>To evaluate the evolution of biosafety concerns in iGEM.
</LI><LI>To look at the types of containment system that have been designed by previous iGEM teams.
</LI></OL><H2><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H2><P>We took the list of all iGEM teams since 2006 and looked at their wiki. If they had a containment system, we filled in the table that is below.
In a more rapid way, we used Google™ search option to screen occurrence of biosafety terms (safety or bio(-)safety) or the occurrence of biosafety devices terms (semantic containment, kill switch, DNase). To do so, we used the following syntaxe with year '2006' and word 'safety' as an example :
</P><PRE>safety site:2006.igem.org
</PRE><P>We took into account the number of results, and we divided it by the number of results with the word 'igem' to normalize with the number of page on all the servers. We assume that there is on average a bit more than one &quot;igem&quot; occurrence per page (in the name of the page).
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>We can see that the safety page rule creating in 2008 induced a increasing frequence of the word safety. This phenomenon started the kill switch trend. Deeper research through the manual wiki screen makes us realize that most of the kill switches were inefficient, or not even well characterized. In spite of the hard work of team Paris Bettencourt in 2009 on human practice, that raised many biosafety concerns, few teams made an effort to implement a serious biosafety device. Indeed we were able to put only about 15% of the 32 biosafety project that we selected in the wiki-screen in the biosafety catalog of the registry.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Evolution of the relative occurrence of biosafety terms on the iGEM server over 7 years of the iGEM existence. The black arrow indicates the year from which safety page is mandatory</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Evolution of the relative occurrence of biosafety devices terms on the iGEM server over 7 years of the iGEM existence. The black arrow indicates the year from which safety page is mandatory</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>Biosafety is not a side part of synthetic biology, and it needs to have a greater emphasis in iGEM in general. The new category Biosafety in the parts registry aims at improving this. We hope to see in future years more biosafety projects arising.
</P><H2><SPAN class="mw-headline" id="Appendix">Appendix</SPAN></H2><DIV id="grouptitle">Summary table </DIV><TABLE border="1" width="100%"><TBODY><TR><TH width="5%"> Team
</TH><TH width="5%"> Year
</TH><TH width="5%"> Project Name
</TH><TH width="25%"> Project Summary
</TH><TH width="30%"> Biosafety Idea
</TH><TH width="30%"> Efficiency
</TH></TR><TR><TD><CENTER> St. Andrews </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:St_Andrews" class="external text" rel="nofollow">Kill switch engage!</A></CENTER></TD><TD> Kill switch
</TD><TD><I><B>Kill switch.</B></I> &quot;Our kill switch is designed by inserting an antimicrobial peptide (AMP) gene into E.Coli&quot;
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:St_Andrews/biosafe" class="external text" rel="nofollow">Read More</A></P></TD><TD> Used the LIVE/DEAD Baclight Bacterial Viability kit, but don't have any quantitative results in terms of number or proportion of cell death. &quot;The relationship between the concentration of arabinose and the amount and rate of cell death seems linear in nature.&quot;
</TD></TR><TR><TD><CENTER> Imperial College </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Imperial_College_London" class="external text" rel="nofollow">Auxin</A></CENTER></TD><TD> Engineer bacteria to accelerate plant root development
</TD><TD><I><B>Toxin/antitoxin.</B></I> Consits of the insertion of the Holin + Endolysine and Anti Holin genes.
<UL><LI>Holin is a protein that forms pores in cell membranes 
</LI><LI> Anti-holin binds to holin, inhibiting it's action. 
</LI><LI>Once pores are formed by holin, lysozyme can access the periplasmic space and degrade the cell wall, causing cell lysis.
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Gene_Overview" class="external text" rel="nofollow">Read More</A></P></TD><TD> Anti-holin was expressed in cells, but no experiments for this system have been made. &quot;It seems very likely that our GM E. coli have been able to survive in soil and retain their plasmid for six weeks despite competition and selective pressure against the plasmid.&quot;
</TD></TR><TR><TD><CENTER> Bristol </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:BCCS-Bristol" class="external text" rel="nofollow">AgrEcoi</A></CENTER></TD><TD> Bacteria that detects and signals the presence of nitrates
</TD><TD><I><B>Encapsulation in a gel.</B></I> We encapsulated our bacteria in beeds made out of a non toxic gel.
 <P style="text-align:right;"><A href="https://2010.igem.org/Team:BCCS-Bristol/Wetlab/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> Due to the beads, &quot;bacteria are kept separate from the environment, reducing public safety fears.&quot; No quantification provided.
</TD></TR><TR><TD><CENTER> Colombia </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Colombia" class="external text" rel="nofollow">Defense aid for coffee plantations against fungi</A> (&quot;Detect and alert&quot; system) </CENTER></TD><TD> Detect chitin,and alert the plant by stimulating an early hypersensitive response against infection.
</TD><TD>A few ideas but no design:
<OL><LI> Amplification mostly from start to stop codon to avoid hidden pathogenicity + blast of sequences to confirm
</LI><LI> Cloning in vectors without a mobilizable origin or TRA region.
</LI><LI> Growth curve of the strain to confirm low dissemination.
</LI><LI> Develop a strategy to monitory the presence of the modified bacteria in the crops fields based on color markers (no design made).
</LI></OL><P style="text-align:right;"><A href="https://2011.igem.org/Team:Colombia/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> British Columbia</CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:British_Columbia" class="external text" rel="nofollow">iSynthase</A></CENTER></TD><TD> Optimize production of terpenes in Saccharomyces cerevisiae yeast (to help trees fight invading beetles and fungi)
</TD><TD> A few ideas (no design):
<OL><LI>Contain a tracking system to detect their presence e.g. a non-coding DNA code akin to a fingerprint
</LI><LI>Contain a suicide system to enable effective elimination of the organism when so desired
</LI></OL><P style="text-align:right;"><A href="https://2011.igem.org/Team:British_Columbia/Wild" class="external text" rel="nofollow">What experts say</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> Peking </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:Peking" class="external text" rel="nofollow">Heavy metal decontamination kit</A></CENTER></TD><TD> Heavy metal bioreporter and bioabsorbent engineering
</TD><TD> A few superficial ideas:
<OL><LI> Make the plasmid toxic for any bacteria it may transfer to.
</LI><LI> Plasmid can’t be replicated in another bacteria
</LI></OL><P style="text-align:right;"><A href="https://2010.igem.org/Team:Peking/Biosafety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> Brown-Stanford </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Brown-Stanford/REGObricks/Introduction" class="external text" rel="nofollow">REGObricks</A></CENTER><CENTER><A href="https://2011.igem.org/Team:Brown-Stanford/PowerCell/Introduction" class="external text" rel="nofollow">PowerCell</A></CENTER></TD><TD> Projects which could be useful for a space travelling:
<UL><LI> REGObricks brings the principle of In-Situ Resource Utilization to bear on the problem of constructing, maintaining and expanding shelter for human inhabitants on the desolate Martian landscape.
</LI><LI> By producing macromolecules essential to bacterial growth, PowerCell will form a metabolic foundation for the biological systems which will eventually enable a settlement on Mars.
</LI></UL></TD><TD> Not about biosafety, but:
<UL><LI><B>The team used GelRed, a DNA dye produced by Biotium Inc, as a non-toxic alternative to ethidium bromide (EtBr).</B> Let's use it too!</LI></UL><P>They do not believe that their BioBrick parts will have any negative effect on the environment. But they could be interested by our project!
</P><P style="text-align:right;"><A href="https://2011.igem.org/Team:Brown-Stanford/Lab/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> Not applicable
</TD></TR><TR><TD><CENTER> KULeuven </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:KULeuven" class="external text" rel="nofollow">E.D. FROSTI</A></CENTER></TD><TD> CONTROLLING ICE FORMATION.
<UL><LI>Bacterium E.D. Frosti induces ice crystallization, using the ice-nucleating protein (INP), or inhibits ice crystal formation, using the anti-freeze protein (AFP), depending on the given stimulus. 
</LI><LI>These proteins will be extracellularly anchored at E.D. Frosti’s cell membrane. 
</LI></UL></TD><TD><I><B>Suicide mechanism: DNA nuclease.</B></I> (ideas)
<UL><LI>Destruction of the DNA &gt; bacteria dies, but keeps its shape so it can still do its job (functional protein are at the surface of the membrane).
</LI><LI>The suicide mechanism activity is mediated by an “AND”-gate system: the cell death mechanism is only activated when one of the two stimuli is given, AND a sudden decrease in temperature (a cold-shock) occurs
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:KULeuven/Description" class="external text" rel="nofollow">Read More</A></P></TD><TD> No constructions, no results.
</TD></TR><TR><TD><CENTER> LMU-Munich </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:LMU-Munich" class="external text" rel="nofollow">Metal Sensor</A></CENTER></TD><TD> Biosensor to detect metals in waste water
</TD><TD> One idea (no design):
<UL><LI>Every potentially pathogenic or hazardous biobrick should be cloned in a special backbone containing the sequences of the present backbones AND an inducible killing gene cassette. 
</LI><LI>This killing cassette is induced by a normaly absent reagent that could easily be added in case of contamination
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:LMU-Munich/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results.
</TD></TR><TR><TD><CENTER> Lyon-INSA-ENS </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Lyon-INSA-ENS" class="external text" rel="nofollow">Cobalt Buster</A></CENTER></TD><TD> Decontamination of radioactive cobalt in water by a biofilm
</TD><TD><I><B>Biofilm &amp; Adherence</B></I> =
<UL><LI>confinement bacterias 
</LI><LI>easy purification of water
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:Lyon-INSA-ENS/Project/Context" class="external text" rel="nofollow">Read More</A></P></TD><TD> No quantitative results.
</TD></TR><TR><TD><CENTER> METU-Ankara </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:METU-Ankara" class="external text" rel="nofollow">MethanE.COLIc</A></CENTER></TD><TD> Sensing methane gas and converting it into methanol
</TD><TD><I><B>Kill switch = Toxin temperature induced</B></I><UL><LI>Holin + Endolysine 
</LI><LI>Induced when T°&gt;42°C
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:METU-Ankara#project5" class="external text" rel="nofollow">Read More</A></P></TD><TD> No quantitative results.
</TD></TR><TR><TD><CENTER> Caltech </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Caltech" class="external text" rel="nofollow">Bioremediation of Endocrine Disruptors Using Genetically Modified Escherichia Coli</A></CENTER></TD><TD> Endocrine-disrupting chemicals (EDCs) are chemicals that interact with the endocrine system by binding to hormone receptors, causing problems in sexual development and reproduction of organisms
</TD><TD> Biosafety ideas:
<UL><LI>Water filtration system for containment of microbes (filters all microbes, does not retain free DNA).
</LI><LI>Use the ccdB gene (BBa_P1016 and BBa_P1010)on a plasmid in conjunction with an E. coli strain such as DB3.1 (BBa_V1005). The plasmid will code for the “death gene” which will kill any cell that does not code for immunity in its genome. If a native microorganism would uptake this man-made plasmid, it would die, preventing the propagation of the recombinant DNA in the environment (Featured Parts: Cell Death).
</LI><LI>Another similar “suicide” containment system uses streptavidin (BBa_J36841) (Kaplan, Mello et al. 1999; Urgun-Demirtas, Stark et al. 2006). This protein binds very tightly to biotin, a required co-enzyme for many metabolic pathways. This makes biotin unavailable and causes cell death. Kaplan et al. reported cell counts were reduced 99.9% in eight hours after their system was activated by absence of pollutant to degrade.
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:Caltech/Biosafety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> Berkeley</CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:Berkeley" class="external autonumber" rel="nofollow">[1]</A></CENTER></TD><TD> Choa Choa's Delivery Service
</TD><TD> Clotho Framework:
<P>Clotho implements the current biosafety standards as outlined by the NIH and the CDC. Whenever a new part is instantiated, Clotho BLAST's its sequence against a databank of known virulence factors and pathogens and returns the RG number of the highest match. This framework ensures that every part in Clotho has a RG value associated with it. In addition, when composite parts are made by joining basic parts, the composite part is also assigned a BSL number. Finally, all strains in Clotho have a risk group.
</P><P style="text-align:right;"><A href="https://2010.igem.org/Team:Berkeley/Clotho" class="external text" rel="nofollow">Read More</A></P></TD><TD> Not applicable.
</TD></TR><TR><TD><CENTER> WITS-CSIR_SA </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:WITS-CSIR_SA" class="external text" rel="nofollow">Biotweet</A></CENTER></TD><TD> Engineer bacteria to allow them to transport packets of chemicals, and then form a network.
</TD><TD> Promoter responsive to synthetic chemicals not present in nature.
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> Not applicable.
</TD></TR><TR><TD><CENTER> ULB-Brussels </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:ULB-Brussels" class="external text" rel="nofollow">One-step gene insertion or deletion system</A></CENTER></TD><TD> An easy way to allowing the insertion and/or deletion of genes in the E. coli chromosome in a minimal number of steps
</TD><TD> FLP recombination to remove resistance gene, eg. FRT - Cm - FRT will remove the Cm resistance gene
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:ULB-Brussels/removing" class="external text" rel="nofollow">Read More</A></P></TD><TD> &quot;All 32 candidates that contained pINDEL grew on LB but none of them grew on LB Cm plates indicating that all of them have lost the antibiotic resistance cassette, sgowing the the DEL function of pINDEL functions properly.&quot;
</TD></TR><TR><TD><CENTER> Johns Hopkins </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Johns_Hopkins" class="external text" rel="nofollow">VitaYeast</A></CENTER></TD><TD> They want to make bread with yeast producing Vitamin
</TD><TD> their yeast strain lacks functional pathways for <B>seven</B> essential aa.
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No quantification
</TD></TR><TR><TD><CENTER> Kyoto </CENTER></TD><TD><CENTER> 2009 </CENTER></TD><TD><CENTER><A href="https://2009.igem.org/Team:Kyoto/GSDD" class="external text" rel="nofollow">p1 : Gene Switch Depending on Duplication</A></CENTER></TD><TD> How to express a gene after a certain lifespan. The use Linear DNA, in which a repressor will be degraded after few cell division.
</TD><TD> &quot;they want to control exact cell’s life time (or <B>death time</B>) depending on the number of cell division times.
<P>They get rid of the exonucelase problem by inserting multiple protein binding site that will be degraded when division occurs, but not with exonucleases. The repressor gene is degraded after a certain time.&quot;
</P><P style="text-align:right;"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Experiment" class="external text" rel="nofollow">Read More</A></P></TD><TD> No quantification
</TD></TR><TR><TD><CENTER> UNICAMP-EMSE_Brazil </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:UNICAMP-EMSE_Brazil" class="external text" rel="nofollow">stress wars</A></CENTER></TD><TD> Coli based production of anti-stress compounds in the organism to improve the quality of life
</TD><TD> &quot;no design but two suggestions :
<P>One suggestion in the biosafety of the researcher is to include in the iGEM parts kit the bacterial less endotoxicchassi (developed by Berkeley UC 2007), to avoid serious septic problems in the case of an accident that leads to an intense contact with the bacteria (eye or bloodstream contact, inhalation or ingestion).
</P><P>Another suggestion for population and environment safety is to (somehow) include in all biobricks an operating unit that detects if the bacteria is not in a culture media (detects some molecule produced through the metabolism of a specific media constituent) and express lysozyme to kill it if it's released in the environment (idea inspired by Team UNICAMP-Brazil 2009).&quot;
</P><P style="text-align:right;"><A href="https://2011.igem.org/Team:UNICAMP-EMSE_Brazil/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> Michigan </CENTER></TD><TD><CENTER> 2009 </CENTER></TD><TD><CENTER><A href="https://2009.igem.org/Team:Michigan" class="external text" rel="nofollow">The Toluene Terminator</A></CENTER></TD><TD> The Toluene Terminator is a Pseudomonas putida device that aims to:
<OL><LI>identify the toxic compound toluene in an environmental setting (e.g. a spill into soil from an underground petrol tank), 
</LI><LI>move to and uptake it, 
</LI><LI>metabolize it, 
</LI><LI>destroy itself when all of the toluene has been metabolized.
</LI></OL></TD><TD><I><B>A kill switch that operates through the Enterobacteria phage T4 Lysis Device (lysozyme, holin, and antiholin)</B></I> created by the Berkley 2008 team. We proposed two mechanisms for cell lysis:
<UL><LI>arabinose inducible suicide mechanism
</LI><LI>suicide mechanism with tunable repression.
</LI></UL><P style="text-align:right;"><A href="https://2009.igem.org/Team:Michigan/Project" class="external text" rel="nofollow">See the designs</A></P></TD><TD> No results
</TD></TR><TR><TD><CENTER> Queens Canada 2011 </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Queens_Canada" class="external text" rel="nofollow">Nemoremediation</A></CENTER></TD><TD> Engineering C.elegans for advanced bioremediation
</TD><TD><I><B>Kill switch.</B></I><UL><LI><I><B>MRT-2 mutant Kill Switch</B></I></LI></UL><P>This is an indirect kill switch. The teams plans to use a mutant strain of bacteria which has losts its molecules for immortality. It can reproduce for only a couple generation and will then become sterile, leading to the extinction of this bacteria population. 
</P><UL><LI><I><B>RNAi Kill Switch</B></I></LI></UL><P>RNAi knockout is a method of gene regulation where double stranded RNA (dsRNA) is introduced to the worm, binding to gene products such as specific RNAs (mRNA), thereby decreasing or eradicating RNA activity by recycling said products. Only a few dsRNA molecules per cell are required to produce effective interference.
</P><P style="text-align:right;"><A href="https://2011.igem.org/Team:Queens_Canada/Side/KillSwitch" class="external text" rel="nofollow">Read More</A></P></TD><TD><UL><LI><I><B>MRT-2 mutant Kill Switch</B></I></LI></UL><P>&quot;Given that the germ-line of the mrt-2 mutant has a limited number of generations in which it can reproduce, our created strain is timed for eventual extinction of a span between 2 months and 1 year, with the average being 6.25 months&quot;. However, no experiment is provided, and so we do not know how they obtained these numbers, but we could assume they come from the literature.
</P><UL><LI><I><B>RNAi Kill Switch</B></I></LI></UL><P>No results 
</P></TD></TR><TR><TD><CENTER> Unist Korea </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:UNIST_Korea" class="external text" rel="nofollow">CHOp-Coli-LATE</A></CENTER></TD><TD> Safety device that is off when bateria senses the that it is in its native environment (the fermentor) but that activates and leads to bacterial suicide when the bacteria escapes the fermentor (it senses that it is in a non native environment).
</TD><TD>Self killing device that will activate if the bacteria's environment changes and lyse the bacteria's DNA. Turns on in the presence of: low temperature, light, change of osmolality.
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:UNIST_Korea/project/modules" class="external text" rel="nofollow">Read More</A></P></TD><TD> The team managed to have the sensor part work (the bacteria can efficiently sense a change in light, temperature and osmolality. However they did not manage to get the lysis module to work. Therefore this system was never completed, so there are no results available in terms of its efficency to prevent bacterial escape from the fermentor.
</TD></TR><TR><TD><CENTER> Tokyo NoKoGen </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Tokyo-NoKoGen" class="external text" rel="nofollow">EcoLion</A></CENTER></TD><TD> E. coli collecting heavy metal ions
</TD><TD><UL><LI><I><B>Collecting E.coli</B></I> with phototaxis or aggregation 
</LI><LI><I><B>Kill switch:</B></I> Holin/Endolysin system (for the reamining bacteria that were excaped collecting)
</LI></UL><P style="text-align:right;"><A href="https://2011.igem.org/Team:Tokyo-NoKoGen/photocontrol" class="external text" rel="nofollow">Read More</A></P></TD><TD><UL><LI><I><B>Collection E.coli:</B></I> The team did not get conclusive results for phototaxis.  The team showed that bacteria with the aggregation module aggregaated more then the control, although they were not able to control this aggregation: their iPTG inducible promoter gave results expected if it was a constitutively active one, suggesting leakage.
</LI><LI><I><B>Kill switch:</B></I> Within 2 hours after addition of IPTG, the OD660 went down by 80%, indicating that the lysis genes were successfully induced by IPTG. However, we should wonder if 80% is good enough...
</LI></UL></TD></TR><TR><TD><CENTER> Newcastle </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:Newcastle" class="external text" rel="nofollow">BacillaFilla: Fixing Cracks in Concrete</A></CENTER></TD><TD> BacillaFilla, an engineered Bacillus subtilis, aims to repair cracks in concrete which can cause catastrophic structural failure.
</TD><TD><I><B>Kill switch.</B></I> &quot;We have considered a kill switch which in the absence of key nutrient would activate and help stop the spread of our bacteria.&quot;
 <P style="text-align:right;"><A href="https://2010.igem.org/Team:Newcastle/Non-target-environment_kill_switch" class="external text" rel="nofollow">Read More</A></P></TD><TD> They designed a a kill switch which uses the mazEF Toxin-Antitoxin system but <B>they didn't obtain any quantitative results</B>. &quot;In our construct a sucrose inducer is placed in the construct so that mazE and mazF can be switched off by a depletion of of sucrose in the media, be this running out of it after achieving its purpose in the microcrack, or if it is sprayed into the environment, where there is little or no sucrose (e.g in the soil, on human skin). The deactivation of expression will lead to a build up of mazF and the death of the cell&quot;.
</TD></TR><TR><TD><CENTER> Wisconsin-Madison </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:Wisconsin-Madison" class="external text" rel="nofollow">iDIET (Intelligent Delivery of Ingestible Enzyme Treatment)</A></CENTER></TD><TD> Design of an universal platform for polypeptide release within the small intestine of the human gastrointestinal tract.
</TD><TD><I><B>Encapsulation.</B></I> &quot;Our goals is to have each cell be surrounded by a protective 'capsule' to allow them to safely travel through the harsh acidic environment of the stomach to arrive in the small intestine for their main purpose.&quot;
<P><I><B>Timed Lysis.</B></I> &quot;To deliver the dose of enzymes to the small intestine, the bacteria will lyse. Lysis must occur in the small intestine, too early and the stomach could harm enzyme activity, too late and the enzyme isn't able to act on its target.&quot;
</P><P style="text-align:right;"><A href="https://2010.igem.org/Team:Wisconsin-Madison/delivery#Project" class="external text" rel="nofollow">Read More</A></P><P><I><B>Risk assessment.</B></I> &quot;We wanted to explore the potential hazardous ramifications of actually implementing bacterial delivery system into a mammalian host. Hazardous ramifications will be evaluated by the following by a calculation of the Risk involved, which is some function of Hazard and Probability&quot;
</P><P style="text-align:right;"><A href="https://2010.igem.org/Team:Wisconsin-Madison/safety#.282.29_Issues_of_Project_Ramifications:" class="external text" rel="nofollow">Read More</A></P></TD><TD> The two mechanisms, encapsulation and lysis, are not directly used for biosafety purposes but rather just as a way to administer the enzymes efficiently in a body which is a specific environment. The results they got are inconclusive and more testing should be done.
<P>The risk assessment part is more descriptive and subjective, it lacks a quantitative background.
</P></TD></TR><TR><TD><CENTER> Illinois-Tools </CENTER></TD><TD><CENTER> 2009 </CENTER></TD><TD><CENTER><A href="https://2009.igem.org/Team:Illinois-Tools" class="external text" rel="nofollow">Interactive Metabolic Pathway Tools (IMP Tools)</A></CENTER></TD><TD> Interactive Metabolic Pathway Tools (IMP Tools) is an open source, web based program that involves model-guided cellular engineering where new metabolic functions can be added to existing microorganisms.
</TD><TD><I><B>Limiting ATP production.</B></I> &quot; By limiting ATP produced we lessen the chances of a cell growing vigorously and being a potential danger to the environment. The actual means to prevent this is by creating a script that will analyze ATP consumption and production. With this information, you are able to adjust the ATP metabolism in whatever way you want. You will be able to keep the production low enough so cell processes can still occur and allow the cell to grow to an extent but not high enough that it can potentially grow out of control.&quot;
 <P style="text-align:right;"><A href="https://2009.igem.org/Team:Illinois-Tools/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results.
</TD></TR><TR><TD><CENTER> Harvard </CENTER></TD><TD><CENTER> 2009 </CENTER></TD><TD><CENTER><A href="https://2009.igem.org/Team:Harvard" class="external text" rel="nofollow">Optical communication</A></CENTER></TD><TD> A system that allows for interspecies, bacteria-to-yeast optical communication.
</TD><TD> A bacterial blackboard using a yeast two hybrid system and luciferase proteins. The blackboard can only be activated with a certain frequency of light, and must be erased with a different frequency of light.
 <P style="text-align:right;"><A href="https://2009.igem.org/Team:Harvard/Ethics" class="external text" rel="nofollow">Read More</A></P></TD><TD> No results.
</TD></TR><TR><TD><CENTER> Duke </CENTER></TD><TD><CENTER> 2009 </CENTER></TD><TD><CENTER><A href="https://2009.igem.org/Team:Illinois-Tools" class="external text" rel="nofollow">IMPtools</A></CENTER></TD><TD> &quot;Interactive Metabolic Pathway Tools (IMP Tools) is an open source, web based program that involves model-guided cellular engineering where new metabolic functions can be added to existing microorganisms.
<UL><LI>This program will assist in the design stage of synthetic biology research. 
</LI><LI> It takes a user-defined input compound, output compound, and weighting scheme and determines the ideal pathway from the starting to the ending compound.&quot;
</LI></UL></TD><TD> &quot;ATP is the energy within a cell and a majority of what it relies on to flourish. &quot;&quot;&quot;&quot;&quot;&quot;By limiting ATP produced we lessen the chances of a cell growing vigorously and being a potential danger to the environment&quot;&quot;&quot;&quot;&quot;&quot;.
<UL><LI>The actual means to prevent this is by creating a script that will analyze ATP consumption and production.
</LI><LI> With this information, you are able to adjust the ATP metabolism in whatever way you want. 
</LI><LI>You will be able to keep the production low enough so cell processes can still occur and allow the cell to grow to an extent but not high enough that it can potentially grow out of control. &quot;
</LI></UL><P style="text-align:right;"><A href="https://2009.igem.org/Team:Illinois-Tools/Safety" class="external text" rel="nofollow">Read More</A></P></TD></TR><TR><TD><CENTER> UCSF </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:UCSF" class="external text" rel="nofollow">Synthetic Cancer Killers</A></CENTER></TD><TD> &quot;Killer cells of the immune system identify cancer and pathogen-infected cells and kill them. These potent killers travel throughout the body, recognizing proteins and other molecules on the surface of cells. In order to differentiate between healthy and diseased cells, killer cells use a variety of receptors, which bind to specific ligands on the target cells’ surface. If the target cell is deemed potentially dangerous, the killer cell grips the target cell tightly and creates an immunological synapse at the site of adhesion. Within this immunological synapse, the killer cell releases cytotoxic granules to kill the target cell without harming nearby cells, triggering a directed apoptotic response.
<P>Our team will focus on improving killer cells’ specificity and killing efficiency towards cancerous target cells. By using tools of synthetic biology, we hope to create powerful killing bio-machines to fight cancer. Our newly engineered synthetic devices would have the potential to enhance current adoptive cell-based immunotherapy for cancer patients.&quot;
</P></TD><TD> We believe that as long as the proper precautions are taken and the safety guidelines are followed, most potential safety concerns can be prevented. None of our genes, parts, or devices are considered potentially oncogenic or pathogenic which would require a safety rating above BSL1. We specifically chose not to use materials from known pathogens. A possible extra precaution to make parts, devices, and systems safer would have been to put suicide genes into the sequences to prevent unintended introductions of them into the environment, but this was less necessary based upon the components and systems we used.
 <P style="text-align:right;"><A href="https://2010.igem.org/Team:UCSF/Safety" class="external text" rel="nofollow">Read More</A></P></TD></TR><TR><TD><CENTER>  University of Waterloo  </CENTER></TD><TD><CENTER> 2008 </CENTER></TD><TD><CENTER><A href="https://2008.igem.org/Team:Waterloo" class="external text" rel="nofollow">Genome-free Bacterial Bioproduct Factory: </A></CENTER></TD><TD>  A bacterial cell already containing a plasmid encoding bioproduct synthesis genes, will self-destruct by degrading its own genome and transiently produce the bioproduct until cell resources have been exhausted.
</TD><TD><P style="text-align:right;"><A href="https://2008.igem.org/Team:Waterloo/Project" class="external text" rel="nofollow">Read More</A></P></TD><TD> No result, bronze medal
</TD></TR><TR><TD><CENTER> Bielefeld-Germany </CENTER></TD><TD><CENTER> 2011 </CENTER></TD><TD><CENTER><A href="https://2011.igem.org/Team:Bielefeld-Germany" class="external text" rel="nofollow">the Bisphenol-A team</A></CENTER></TD><TD> &quot;The development of sensitive and selective biosensors is an important research field in synthetic biology. Biosensors can be applied in a wide range of uses - from the detection of environmental toxics up to clinical diagnostics. Because cells have to sense their surroundings, there are a lot of natural systems that are similar to a biosensor. Prejudicial cellular biosensors often show negative side effects that complicate any practical application. Common problems are the limited use outside of a gene laboratory due to the use of genetically engineered cells, the low durability because of the usage of living cells and the appearance of undesired signals induced by endogenous metabolic pathways.
<P>To solve these problems, the iGEM-Team Bielefeld 2011 aims to develope a cell-free bisphenol A (BPA) biosensor based on a coupled enzyme reaction fused to S-layer proteins for everyday use. Bisphenol A is a supposedly harmful substance which is used in the production of polycarbonate. To detect BPA it is degraded by a fusion protein under formation of NAD+ which is detected by an NAD+-dependent enzymatic reaction with a molecular beacon. Both enzymes are fused to S-layer proteins which build up well-defined nanosurfaces and are attached to the surface of beads. By providing these nanobiotechnological building blocks the system is expandable to other applications.&quot;
</P></TD><TD> Our approach is a cell free biosensor. One advantage is that no living organisms need to be used outside the lab for the application of our system. We want to provide S-layers as nanobiotechnological building blocks for cell free biosensors. By fusing different enzymes to these proteins, a variety of biosensors can be build. Further cell free applications are possible and we think that when more projects are focusing on cell free systems this is also a contribution to more safety and security. All GMOs can stay in the lab, therefore are grown under controlled conditions and only qualified personnel has access to them.
 <P style="text-align:right;"><A href="https://2011.igem.org/Team:Bielefeld-Germany/Safety" class="external text" rel="nofollow">Read More</A></P></TD></TR><TR><TD><CENTER> CHIBA </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:Chiba" class="external text" rel="nofollow">Double Click</A></CENTER></TD><TD> We’re inspired by double-click of computer’s mouse. It doesn’t react to the first click but does react when it is accompanied by the second one. This is one of the most accepted, familiarized, and proven mechanism to diminish the erroneous operation. This fail-safe technology should find various uses also in biotechnology.
</TD><TD> &quot;If there is only an input, nothing happens.
<P>Duration of the input is not the matter. The circuit cares only the number of input.
However, a certain time after the 1st input, it returns to the initial state.
Giving two inputs in the limited time the circuit get activated (gives output).&quot;
</P><P style="text-align:right;"><A href="https://2010.igem.org/Team:Chiba/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> System did not work
</TD></TR><TR><TD><CENTER> HKU-Hong Kong </CENTER></TD><TD><CENTER> 2010 </CENTER></TD><TD><CENTER><A href="https://2010.igem.org/Team:HKU-Hong_Kong" class="external text" rel="nofollow">The Bio-Safety Net</A></CENTER></TD><TD> &quot;By using different promoters, the system can respond to changes in environmental factors and control expression specific to a chosen factor. Such mechanism can be easily assembled and incorporated into bacteria through the use of biobricks.
<P>Our team’s project is a “bio-safety net” that limits the survival of bacteria according to tailored conditions.&quot;
</P></TD><TD> Killswitch using T4 holin ,T4 anti holin, and lysozyme system (or Gene E), under control of pBAD and pLAC
 <P style="text-align:right;"><A href="https://2010.igem.org/Team:HKU-Hong_Kong/Safety" class="external text" rel="nofollow">Read More</A></P></TD><TD> After 8 hours, cells implementing this circuit had an OD of 0.0309 versus an OD of 1.9414 and 1.9651 in cells where the circuit was suppressed.
</TD></TR></TBODY></TABLE><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/WikiScreen">http://2012.igem.org/Team:Paris_Bettencourt/Human_Practice/WikiScreen</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Human_Practice/WikiScreen" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Human_Practice/WikiScreen" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Human_Practice/WikiScreen&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Human_Practice/WikiScreen&amp;oldid=234846" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>