<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Suicide"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Suicide">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Suicide&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Suicide&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Suicide&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Suicide" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Suicide</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Suicide System</DIV><TABLE id="tableboxed"><TBODY><TR><TD><P><B>Aims :</B>
Implement a kill-switch that features population-level suicide and complete genome degradation. 
</P><P><B>System :</B>
A synthetic toxin-anti-toxin system based on the wild type Colicin E2 operon.
</P><P><B>Achievements :</B>
We showed that Colicin E2 cells induce cell death in sensitive populations, and that these sensitive populations can be protected by providing them with our engineered immunity protein. 
</P><UL><LI> Construction of 2 biobricks :
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914001" class="external text" rel="nofollow">K914001</A> : pLac-repressilator RBS-Colicin E2 immunity protein
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914002" class="external text" rel="nofollow">K914002</A> :repressilator RBS-Colicin E2 immunity protein
</LI></UL><P>Part K914001 is well characterized and provides immunity to sensitive cells against the Colicin E2 activity protein. Part K914002 is promoterless and allows users to easily plug in the appropriate promoter for their desired purpose. 
</P><UL><LI> Creation of a new category in the part registry : <A href="http://partsregistry.org/Biosafety" class="external text" rel="nofollow">XNase</A>. The aim of this category is to provide users with DNase/RNase parts that can be used for improved kill switches featuring the degradation of genomic material.
</LI></UL><DIV id="boston"><UL><LI> Partially biobricked sRNA system : 
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914017" class="external text" rel="nofollow">K914017</A> stationary phase promoter Yiagp
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914016" class="external text" rel="nofollow">K914016</A> coding sequence of Colicin E2
</LI></UL></DIV></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bigger_Picture"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Bigger Picture</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Objectives"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Experiments_and_results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Experiments and results</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Colicin_E2_Expression_Kills_Sensitive_Cells"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Colicin E2 Expression Kills Sensitive Cells</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Experimental_setup"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Results"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Expression_of_Immunity_Protects_Sensitive_Cells"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Expression of Immunity Protects Sensitive Cells</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Experimental_setup_2"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Results_2"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Quantitative_Characterization_of_the_Antitoxin_Plasmid"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Quantitative Characterization of the Antitoxin Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Experimental_Setup_3"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Experimental Setup</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Results_3"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Testing_the_effects_of_colicin-containing_protein_extract_on_E.coli"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Testing the effects of colicin-containing protein extract on E.coli</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Experimental_Setup_4"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">Experimental Setup</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Results_4"><SPAN class="tocnumber">4.4.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Persperctives"><SPAN class="tocnumber">4.4.3</SPAN><SPAN class="toctext">Persperctives</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Related_Parts_and_Links"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Related Parts and Links</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Appendix"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Appendix</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#ColicinE2_Plasmid_Map"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">ColicinE2 Plasmid Map</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Our goal is to engineer a synthetic toxin-anti-toxin system from the wild type Colicin E2 (Col E2) operon. This synthetic toxin-anti-toxin system is species specific, allows for population-level suicide, complete genome degradation, and will function on a <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay" class="external text" rel="nofollow">tunable delay</A>. The Col E2 toxin, called the activity protein, is a DNase, meaning that it cleaves DNA, which targets related species of <I>E.coli</I>. The Col E2 anti-toxin, called the immunity protein, binds the Col E2 activity protein with high affinity preventing the activity protein from acting on its own producing bacteria. Our idea is to clone the activity protein and the immunity protein on two different plasmids, called toxin and anti-toxin plasmids, so we can switch on the suicide mechanism by degrading the anti-toxin plasmid with the <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme" class="external text" rel="nofollow">restriction enzyme system</A>. To test our system, we cloned the immunity protein under the Lac promoter inside a medium copy plasmid. We tested our system by showing that this inducible system protects the cells from the Col E2 activity protein. The next step will be to place the activity protein under a constitutive promoter  inside a low copy plasmid; work which is ongoing.
</P><CENTER><A href="/File:Toxin3aPB12.gif" title="File:Toxin3aPB12.gif">400px</A></CENTER><H3><SPAN class="mw-headline" id="Background">Background</SPAN></H3><P><B>What are Colicins?</B>
Bacteria have developed mechanisms to kill other bacteria in order to reduce competition between themselves in the environment [1]. Some strains of <I>E. coli</I> produce <B>lethal proteins</B> called colicins which kill sensitive bacteria, including related species of E. coli. Colicinogenic cells contain <B>specific immunity</B> against their own toxin, and their colicins only effect cells containing specific receptors on their  outer membrane surface. Colicins are therefore classified by the receptor to which they bind to, for example, colicins E1-E9 bind to the outer membrane (OM) protein BtuB which mediates the entry of nucleosides, siderphores, and vitamin B12 in the cell. Colicins E1-E9 are also Group A colicins, meaning that they are translocated through the cell envelope by the Tol machinery, whereas Group B colicins are translocated through the cell envelope by the TonB machinery. Colicins have different modes of action, including <B>enzymatic activity</B> such as DNase (DNA cleaving) activity, and <B>pore-forming activity</B>. Colicins have three domains, the N-terminal domain functions in translocation through the membrane, the central domain is involved in binding to the OM receptor, and the C-terminal domain contains active (lethal) protein region [1]. We have taken advantage of the modularity of the colicin domains, generating a <A href="https://2012.igem.org/Team:Paris_Bettencourt/SID" class="external text" rel="nofollow">synthetic import domain</A> that will allow for new and exciting forms of <B>communication</B> in <I>E.coli</I>.
</P><P><CENTER> Kleanthous C Nature Reviews Microbiology 2010 8, 843-848 </CENTER></P><P>For our project we have selected Colicin E2, which has enzymatic DNase activity. Colicin E2 is ideal because it not only kills sensitive cells on a <B>population level</B>, but also <B>destroys genomic material</B>, preventing the spread of genetically modified parts via horizontal gene transfer. 
</P><P>In natural systems colicin operons are regulated under the SOS promoter (Stress response). Group A colicins contain type I plasmids. These plasmids are generally 6-10 kb, found in  about 20 copies per cell, and can be amplified and mobilized in the presence of a conjugative plasmid. The first gene of the colicin operon is the activity protein, named  Colicin <I>X</I> activity (i.e. cea for Colicin E2) . This is followed by the immunity protein, named Colicin <I>X</I> immunity (cei),  which, like the activity protein, is regulated by the LexA promoter, but also  has its own constitutive promoter. This separate promoter is located with in the coding sequence of the cea and allows for constant overproduction of the cei, thereby protecting the producing cell. The last gene codes for the lysis protein, named colicin <I>X</I> lysis protein (cel), which causes the release of colicin into medium and is responsible for the cell death of the producer. The SOS promoter can be induced by UV light, chemicals, and stress conditions[1]. 
</P><P>We would like to clone the cea and cei of Colicin E2 onto two separate plasmids, creating toxin and antitoxin plasmids.
</P><H3><SPAN class="mw-headline" id="Bigger_Picture">Bigger Picture</SPAN></H3><P>Our antitoxin plasmid can be combined with the <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme" class="external text" rel="nofollow">restriction enzyme system</A> to generate a toxin-antitoxin system that works on a tunable delay depending on the induction of the restriction enzyme. Once the degradation of the antitoxin plasmid containing the immunity protein is initiated, the cell will continue to produce the activity protein, which will digest the cells own genetic material as well as its neighbors and extracellular DNA via its DNase activity. 
</P><CENTER>Scroll through the thumbnails to see the step by step explanation of our project.</CENTER><DIV class="gallerycontainer"><CENTER><A class="thumbnail" href="#thumb"><SPAN><DIV id="txtOV">In the initial phase, the colicin activity protein and immunity protein are produced and form a heterodimer, protecting the cell from the activity proteins' DNase domain.</DIV></SPAN></A><A class="thumbnail" href="#thumb"><SPAN><DIV id="txtOV">In the second phase, the restriction enzyme is produced.</DIV></SPAN></A><A class="thumbnail" href="#thumb"><SPAN><DIV id="txtOV">In the third phase, the restriction enzyme recognizes and cleaves the restriction sites present in the antitoxin plasmid.</DIV></SPAN></A><A class="thumbnail" href="#thumb"><SPAN><DIV id="txtOV">In the fourth phase, the antitoxin plasmid is completely degraded rendering the cell vulnerable to the lethal activity protein.</DIV></SPAN></A><A class="thumbnail" href="#thumb"><SPAN><DIV id="txtOV">In the fifth phase, the activity protein kills the cell and digests its genetic contents.</DIV></SPAN></A></CENTER></DIV><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><P>Bacteria have developed mechanisms to kill other bacteria in order to reduce competition between themselves in their environment [1]. Some strains of <I>E. coli</I> produce <B>lethal proteins</B> called colicins which kill sensitive bacteria, including related species of E. coli. Colicinogenic cells contain <B>specific immunity</B> against their own toxin, and their colicins only effect cells containing specific receptors on their  outer membrane surface.
</P><P>We have exploited this naturally occurring system to construct a well characterized toxin-antitoxin system  (<A href="https://2012.igem.org/Team:Paris_Bettencourt/Suicide#Design" class="external text" rel="nofollow">design</A>):
</P><OL><LI>Design an antitoxin part from the colicin E2 immunity gene that confers immunity against the Col E2 activity protein
</LI><LI>Measure viability of immune and vulnerable cells in the presence of WT Col E2 cells
</LI><LI>Design a Col E2 activity protein part under the control of a constitutive promoter and establish in immune strains to generate synthetic Col E2 cells
</LI><LI>Measure viability of immune and vulnerable cells in the presence of synthetic Col E2 cells
</LI><LI>Combine our system with the Restriction Enzyme System, measure viability
</LI></OL><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><P>For our system we will generate two plasmids, one carrying the Col E2 activity protein and one carrying the Col E2 immunity protein. As the immunity protein is normally present in excess in natural colicinogenic cells, we chose a 10-12 copy vector pSB3C5 to carry the immunity protein, and a lower ~5 copy vector pSB4K5 to carry the toxin protein. We used pLac to drive expression of the immunity protein, however, different inducible promoters should be used depending on the overall design and application of the safety circuit. We chose the constitutive promoter BBa_J23108 from the Anderson Promoter Collection, which has a relatively moderate expression level, to drive the expression of the activity protein so that it would not overwhelm the cells until after the desired delay. We ultimately plan to integrate the activity protein cassette into the genome of our bacteria.
</P><P>We do not require the colicin lysis protein for our system, and while the activity protein may degrade extracellular DNA released from our genetically modified bacteria, it is preferable that the cells do not lyse and the genomic material is digested within the cells. A small percentage of cells will die naturally and release the activity protein, which will have a lethal action on any cells that escape our system via mutation of the activity protein gene.
</P><H2><SPAN class="mw-headline" id="Experiments_and_results">Experiments and results</SPAN></H2><H3><SPAN class="mw-headline" id="Colicin_E2_Expression_Kills_Sensitive_Cells">Colicin E2 Expression Kills Sensitive Cells</SPAN></H3><P>We performed a basic assay to characterize the toxicity of the Col E2 cells. The Col E2 cells secrete the lethal activity protein, which diffuses out from the Col E2 colonies spotted on the LB plates. This generates a region where vulnerable cells can not grow, visible as an empty ring (ZOI) around the Col E2 colonies.  Here we expect ZOI when Col E2 is spotted on vulnerable cells. 
</P><H4><SPAN class="mw-headline" id="Experimental_setup">Experimental setup</SPAN></H4><P><B>Cell Types</B></P><UL><LI>MG1655: These cells approximate wild type E.coli cells as they have undergone minimal genetic manipulation as compared to other laboratory strains. Genotype: F- λ- ilvG- rfb-50 rph-1
</LI><LI>MG1655.RFP: MG1655 cells transformed with constitutive RFP (from Anderson promoter library, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23102" class="external text" rel="nofollow">BBa_J23102</A>)
</LI><LI>Col E2: Wild Type Colicin E2 cells, containing pColE2-P9 plasmid [2]
</LI></UL><P><B>Protocol</B></P><OL><LI>Plate 10 uL of 0.1 OD lawn cells from liquid culture on plain LB plates
</LI><LI>Spot 5 uL of saturated Colicin E2 cells from liquid culture on plates
</LI><LI>Incubate plates overnight
</LI></OL><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>Col E2 cells produce ZOI in sensitive cells, as indicated bellow by red arrows. MG1655 and MG1655.RFP, cells which are not colicinogenic, do not produce ZOI when spotted on a lawn of sensitive cells. 
</P><P><B>Interpretation</B></P><P>The Col E2 cells secrete the lethal activity protein, which diffuses out from the Col E2 colonies spotted on the LB plates. This generates a region where vulnerable cells can not grow, visible as an empty ring (ZOI) around the Col E2 colonies. It is important to note that Col E2 cells do not need to be induced to produce sufficient activity protein to generate the ZOI. These results confirm the toxicity of Col E2 cells against MG1655 and MG1655.RFP, related species of <I>E.coli</I>. We also assayed Top10 cells, NEB turbo cells, MAGE cells, and MG1655 ZI.RFP cells, for more details look
<A href="http://openwetware.org/wiki/IGEM:Paris_Bettencourt_2012/Notebooks/suicide_group/day_by_day//2012/08/07" class="external text" rel="nofollow">here</A></P><H3><SPAN class="mw-headline" id="Expression_of_Immunity_Protects_Sensitive_Cells">Expression of Immunity Protects Sensitive Cells</SPAN></H3><P>To test the function of our antitoxin plasmid, we performed Colicin E2 toxicity assays on cells transformed with our plasmid. Here we expect zones of inhibition (ZOI) when Col E2 is spotted on vulnerable cells, and the absence of ZOI in the transformed immune cells.
</P><H4><SPAN class="mw-headline" id="Experimental_setup_2">Experimental setup</SPAN></H4><P><B>Cell Types</B></P><UL><LI>MG1655: These cells approximate wild type E.coli cells as they have undergone minimal genetic manipulation as compared to other laboratory strains. Genotype: F- λ- ilvG- rfb-50 rph-1
</LI><LI>MG1655.RFP: MG1655 cells transformed with constitutive RFP (from Anderson promoter library, BBa_J61002)
</LI><LI>Col E2: Wild Type Colicin E2 cells, containing pColE2-P9 plasmid [2] 
</LI><LI>NEB.Imm: NEB turbo cells transformed with the antitoxin plasmid (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K914001" class="external text" rel="nofollow">BBa_K914001</A> inside <A href="http://partsregistry.org/Part:pSB3C5" class="external text" rel="nofollow">pSB3C5</A>). This strain is commonly used in the lab as it grows rapidly (visible colonies on agar, ~6.5 hours; shaking liquid culture OD 600 = 2.0, ~4 hours). This strain is also T1 phage resistant and importantly it expresses the Lac repressor (lacR), which allows us to induce the production of the immunity protein the pLac promoter with IPTG. Genotype: F´ proA+B+ lacIq ∆ lacZ M15/ fhuA2 ∆(lac-proAB) glnV gal R(zgb-210::Tn10)TetS endA1 thi-1 ∆(hsdS-mcrB)5
</LI></UL><P>References:New England Biolabs, product catalogue number C2984H
</P><UL><LI>Z1.Imm: MG1655 Z1 cells transformed with the antitoxin plasmid (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K914001" class="external text" rel="nofollow">BBa_K914001</A> inside <A href="http://partsregistry.org/Part:pSB3C5" class="external text" rel="nofollow">pSB3C5</A>). MG1655 Z1 contain the Z1 cassette (lacR tetR SpR), which allows us to induce the production of the immunity protein under control of the pLac promoter with IPTG.
</LI></UL><P><B>Protocol</B></P><OL><LI>Grow cells in liquid culture containing the appropriate antibiotic
</LI><LI>Add IPTG (0.1mM) to appropriate liquid cultures after 2 hours (OD ~0.2) 
</LI><LI>Wash cells containing antibiotics/IPTG and re-suspend in plain LB
</LI><LI>Plate 10 uL of 0.1 OD lawn cells from liquid culture on plain LB plates or IPTG plates
</LI><LI>Spot 5 uL of saturated Colicin E2 cells from liquid culture on plates
</LI><LI>Incubate plates overnight
</LI></OL><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><P>Col E2 cells do not generate a ZOI in cells containing the antitoxin plasmid, in both induced and un-induced cells. However, Col E2 cells do produce a ZOI in vulnerable MG1655 cells as indicated by red arrows below. 
</P><P><B>Interpretation</B></P><P>These results indicate that our antitoxin plasmid indeed confers immunity to sensitive cells, however, immunity is not dependent on IPTG induction. This is not completely surprising given that the immunity protein has an incredibly high affinity for the activity protein and that pLac is known to be leaky. Even a small concentration of immunity protein could protect un-induced cells against the DNase activity of the colicin activity protein.
</P><H3><SPAN class="mw-headline" id="Quantitative_Characterization_of_the_Antitoxin_Plasmid">Quantitative Characterization of the Antitoxin Plasmid</SPAN></H3><P>We characterized the plasmid pLac + ColE2 immunity gene (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K914001" class="external text" rel="nofollow">BBa_K914001</A> inside <A href="http://partsregistry.org/Part:pSB3C5" class="external text" rel="nofollow">pSB3C5</A>) by testing the response of the cells when we mix them with colicin producing cells in various concentration with also various concentration of IPTG as the inducer of pLac. We counted the number of surviving colonies to see the effectiveness of the immunity protein in protecting the cells from the colicin activity protein.  
</P><H4><SPAN class="mw-headline" id="Experimental_Setup_3">Experimental Setup</SPAN></H4><P><B>Cell types</B></P><UL><LI>Col E2: Wild Type Colicin E2 cells, containing pColE2-P9 plasmid [2] 
</LI><LI>NEB.CmR: NEB turbo cells transformed with <A href="http://partsregistry.org/Part:pSB3C5" class="external text" rel="nofollow">pSB3C5</A>).
</LI><LI>NEB.Imm: NEB turbo cells transformed with the antitoxin plasmid (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K914001" class="external text" rel="nofollow">BBa_K914001</A> inside <A href="http://partsregistry.org/Part:pSB3C5" class="external text" rel="nofollow">pSB3C5</A>).
</LI></UL><P><B>Protocol</B></P><OL><LI>Grow overnight cultures of NEB.CmR, NEB.Imm each with antibiotic (Cm) and Col E2.
</LI><LI>Centrifuge and resuspend in LB.
</LI><LI>Prepare 5 tubes of 10ml LB and put 40uL of NEB.Imm cells into 4 tubes and NEB.CmR cells in 1 tube.
</LI><LI>Incubate 37C until the OD reaches 0.2.
</LI><LI>Put IPTG with different concentration in 3 NEB.Imm tubes (0.1mM, 0.05mM, 0.025mM).
</LI><LI>Incubate again until the OD reaches 1.0.
</LI><LI>Take each 0.5ml and mix with 0.5ml Colicin cells (saturated, saturated diluted 1000x, plain LB with corresponding amount of IPTG) in small 2ml eppendorf tubes.
</LI><LI>Incubate for 30 minutes.
</LI><LI>Dilute 10000x (100x then 100x) in MgSO4 e-2M.
</LI><LI>Plate 20uL in Cm to kill the Colicin cells and leave the immune cells.
</LI></OL><P>Note: 1ml of cells of OD 1.0 has 1e9 cells, so we expect if all cells survive we will get 1000 colonies at the end.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><P>
In this experiment we expected to see more survival if we induce the cells with more IPTG and/or mix them with less Colicin cells. Given that from the previous experiment the immunity protein turned out to be independent to IPTG induction, we treated all the NEB.Imm data as the same. The results showed that the immunity protein can indeed protect the cells from saturated colicin. If we do not put the Colicin cells in the mixture, the numbers of survival cells of NEB.Imm (BBa_K914001) and NEB.CmR (pSB3C5) are almost the same. But if we put the Colicin cells, even very diluted, the NEB.CmR cells could not survive.
</P><P>We noticed that the survival on the positive controls (mixture without Colicin) was somehow less than the one with Colicin cells. We guessed that the problems are
</P><UL><LI>The plates have more cells when they have more Colicin: the Colicin cells maybe took the Immunity/Cm-resistance plasmid
</LI><LI>Less cells on the plate without Colicin (and very few cells overall compared to the theoretical calculation): the cells may have died because they loose the Immunity/Cm-resistance plasmid during the incubation without antibiotic (which we did to avoid killing the Colicin cells).
</LI></UL><P>Therefore we still need to improve protocol to provide a more reliable quantitative data.
</P><DIV id="boston"><H3><SPAN class="mw-headline" id="Testing_the_effects_of_colicin-containing_protein_extract_on_E.coli">Testing the effects of colicin-containing protein extract on E.coli</SPAN></H3><P>We mix Z1 cells with the supernatant of centrifuged Colicin cells (exctract with the ColE2 protein). Therefore we can put the antibiotic during the whole experiment to avoid plasmid loss.
</P><H4><SPAN class="mw-headline" id="Experimental_Setup_4">Experimental Setup</SPAN></H4><P><B>Cell types</B></P><UL><LI>colicin-producing cells
</LI><LI>Z1 cells with immunity
</LI><LI>Z1 cells without immunity as a control
</LI></UL><P><B>Protocol</B></P><OL><LI>Grow overnight cultures of colicin-producing cells, Z1 cells with immunity and Z1 cells without immunity.
</LI><LI>For the colicin-producing cells (protocol adapted from <A href="http://www.piercenet.com/browse.cfm?fldID=06010401" class="external free" rel="nofollow">http://www.piercenet.com/browse.cfm?fldID=06010401</A> ):
</LI><LI> Centrifuge at 5000g for 10 mins
</LI><LI> Weigh the falcons with the cell pellet and add 4mL of B-PER Reagent per gram of cell pellet. Pipette the suspension up and down until it is homogeneous
</LI><LI>Incubate 10-15 minutes at room temperature
</LI><LI> Pour the lysate into eppendorff tubes and centrifuge it at 15,000 × g for 5 minutes to separate soluble proteins from the insoluble proteins. Keep the soluble fraction: this is the protein extract containing colicins.
</LI></OL><OL><LI>For the immune and non-immune cells:
</LI><LI> Dilute the overnight cultures 1/100 into 10-ml cultures (with antibiotics):
</LI><LI> 3 x Z1 cells without immunity
</LI><LI> 3 x Z1 cells with immunity, induced with IPTG
</LI><LI> 3 x Z1 cells with immunity, non-induced
</LI></OL><LI> Grow the cultures for at least 1h, make sure that the OD is more or less the same for all cultures (tip: you can keep a culture with a higher OD at 4°C to wait until the other cultures reach the same OD)
</LI><LI> Mix the immune/non-immune cells with the colicin extract, in different proportions (cells/colicin extract 9:1 and 5:5). For controls, mix immune/non-immune cells with the B-PER buffer, in same proportions
</LI><LI> Put the cells into a 96 well plate for Tecan, 2 wells per each of the cultures
</LI><LI> Put the plate into Tecan, and make OD measurements each 10mins
</LI><H4><SPAN class="mw-headline" id="Results_4">Results</SPAN></H4><P>Growth curves of cells with or without protein extract of colicin producing cells.
We compare the growth of cells that have immunity protein or not. Colicin-containing protein extract significantly impedes the growth of cells that do not have immunity (green curve), whereas it does not affect cells expressing immunity (blue curve), and they grow as well as positive controls (red and purple lines, respectively).
</P><P>Remarks: Immune cells grow even better in the presence of the colicin extract than in the presence of the buffer. This can be explained by the fact that the buffer used for protein extraction leads to cell lysis. In future, we will use the extract of cells that do not produce colicin, as a positive control. Also, in this setup, the step-like trend of the growth curve for the sensitive cells in the presence of colicin extract (blue) can be explained by the degradation of the colicin, and/or by the fact that it all colicin present could be used up at some point, and not affect further growth of cells. 
</P><P>In conclusion, these results show that colicin produced by cells prevents the growth or even kills sensitive cells, whereas the cells expressing the immunity protein survive.
</P></DIV><H4><SPAN class="mw-headline" id="Persperctives">Persperctives</SPAN></H4><P>We suggest in the future we could give stronger stress to the Colicin to induce more the SOS promoter; such as UV light.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Cloning the colicin E2 activity protein, given its lethal nature, is a great challenge for us. We took several approaches. We established immune strains expressing excess immunity protein in which we could transform the toxin plasmid. We tried cloning the toxin without a promoter and finally tried various vectors with and with out promoters. Although the immune strains are viable, transformable, and prove immune to wild type colicin E2 cells, we have yet to successfully transform a single toxin plasmid. 
</P><P>However, developing and characterizing the immune strains was not as straight forward as we imagined. Whether in NEB turbo or MG1655 Z1 cells, the immune strains grew incredibly slowly. Despite being under the control of pLac, their immunity proved somewhat independent of IPTG, as both un-induced and induced cells prove immune. In addition, we had some strange toxicity assay results, partially resolved by the explanation of colicin E2 contamination. 
</P><P>To see more details on the toxicity assays, click <A href="http://openwetware.org/wiki/IGEM:Paris_Bettencourt_2012/Notebooks/suicide_group/day_by_day//2012/09/08" class="external text" rel="nofollow">here</A></P><P><B>UPDATE:</B>
We are still struggling with cloning the colicin E2 activity protein, however we have managed to clone the coding sequence and have some more promising results using the sRNA approach described <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay" class="external text" rel="nofollow">here</A></P><H2><SPAN class="mw-headline" id="Related_Parts_and_Links">Related Parts and Links</SPAN></H2><UL><LI><A href="http://partsregistry.org/Part:BBa_K131000" class="external text" rel="nofollow">BBa_K131000</A> Colicin E2 operon, designed by Kevin McLeod, group: iGEM08_Calgary_Wetware (2008-07-22)
</LI><LI><A href="http://partsregistry.org/Part:BBa_R0011" class="external text" rel="nofollow">BBa_R0011</A> Promoter (lacI regulated, lambda pL hybrid), designed by Neelaksh Varshney, Grace Kenney, Daniel Shen, Samantha Sutton, group: Registry   (2003-01-31)
</LI><LI><A href="http://partsregistry.org/Part:BBa_R0011:Experience/iGEM10_Kyoto" class="external text" rel="nofollow">BBa_R0011:Experience/iGEM10 Kyoto</A> pLac model
</LI><LI><A href="http://partsregistry.org/pSB4K5" class="external text" rel="nofollow">pSB4K5 </A> Low copy plasmid containing kanamycin resistance
</LI><LI><A href="http://partsregistry.org/pSB3C5" class="external text" rel="nofollow">pSB3C5 </A> Low-medium copy plasmid containing chloramphenicol resistance
</LI><LI><A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23108" class="external text" rel="nofollow">BBa_J23108</A> Anderson Constitutive Promoter with relative strength 0.51 (RFP reporter)
</LI><LI><A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23102" class="external text" rel="nofollow">BBa_J23102</A> Anderson Constitutive Promoter with relative strength 0.86 (RFP reporter)
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914016" class="external text" rel="nofollow">K914016</A> coding sequence of Colicin E2
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI>Cascales E, et al. (2007) Colicin biology. Microbiol Mol Biol Rev 71:158–229. 
</LI><LI>Majeed G, Gillor O, Kerr B, Riley MA. (2011) Competitive interactions in Escherichia coli populations: the role of bacteriocins. The ISME Journal 5, 71-81.
</LI><LI>Pugsley AP. (1985) Escherichia coli K12 strains for use in the identification and characterication of colicins. J Gen Microbiol 131: 369-376.
</LI><LI>Kleanthous C. (2010) Swimming against the tide: progress and challenges in our understanding of colicin translocation. Nat Microbiol Rev 8:843-848.
</LI><LI>Torres B, et al. (2003) A dual lethal system to enhance containment of recombinant micr-organisms. Microbiol 149: 3595-3601.
</LI></OL><H2><SPAN class="mw-headline" id="Appendix">Appendix</SPAN></H2><H3><SPAN class="mw-headline" id="ColicinE2_Plasmid_Map">ColicinE2 Plasmid Map</SPAN></H3><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Suicide">http://2012.igem.org/Team:Paris_Bettencourt/Suicide</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Suicide" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Suicide" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Suicide&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Suicide&amp;oldid=294714" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>