<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="boston"><DIV id="grouptitle">Safety Assessment</DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Objectives"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#Methods_and_Results"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Methods and Results</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Hazard_Identification"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Hazard Identification</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Risk_Assessment"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Risk Assessment</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Controlling_hazards_and_risks"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Controlling hazards and risks</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Control_Checks"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Control Checks</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Physical_containment_failure"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Physical containment failure</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Genetic_failure"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Genetic failure</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Total_failure_probability"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Total failure probability</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Discussion"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Conclusion"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Perspectives"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Perspectives</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Safety is an important issue in synthetic biology, especially for environmentally related projects. We started to answer the question, “how safe is safe enough?” by involving experts, the public and our fellow scientists, and also by building biosafety devices. However, to really answer the question, we need first to ask ourselves a more basic question, “how do we measure safety?”. As we see synthetic biology as an engineering approach to biology, we could think about the adaptation of safety engineering, a well studied engineering subset, that has been widely use to minimize risks in many fields of engineering, such as mechanical engineering, aircrafts, and manufactures. However, the risks they face are surely different from the risks of synthetic biology. 
</P><P>According to Dana et al<SUP><A href="#References">1</A></SUP> there are four areas of risk research in environmental application of synthetic biology:
</P><UL><LI>Differences in the physiology of natural and synthetic organisms will affect how they interact with the surrounding environment,
</LI><LI>Escaped microorganisms have the potential to survive in receiving environments and to compete successfully with non-modified counterparts,
</LI><LI>Synthetic organisms might evolve and adapt quickly, perhaps filling new ecological niches, and
</LI><LI>Gene transfer.
</LI></UL><P>Knowing that there are different areas of risk that we need to take into account, we need to design our safety containment with different parts to deal with each of them. Identifying the relationship between one part and the others will help us to see the reliability of the overall system. Here we propose an approach to assess safety for environmental release of genetically engineered bacteria (GE bacteria).
</P><H3><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H3><UL><LI>Adapting existing safety assessment tools for synthetic biology.
</LI><LI>Proposing new methods to assess safety in synthetic biology.
</LI></UL><H2><SPAN class="mw-headline" id="Methods_and_Results">Methods and Results</SPAN></H2><P>We propose a method to assess hazards and risk<SUP><A href="#References">2</A></SUP> in releasing genetically engineered bacteria into the environment. As a case study, we want to release GE bacteria which will perform some function in the environment and we want to apply 3 sets of bWARE safety containment modules (alginate beads, bWARE killswitch, semantic containment) to control the hazards and risks. Here we focus more on the reliability of the safety containment system rather than the functional part, although a similar assessment can also be applied to assess the reliability of the functional part.
</P><H3><SPAN class="mw-headline" id="Hazard_Identification">Hazard Identification</SPAN></H3><P>Identifying hazards means that we need to find and understand the possible harm that may happen in the application of our system. Generally there are two potential hazards in environmental application of synthetic biology<SUP><A href="#References">1</A></SUP> which are the successful escape of the GE bacteria followed by successful competition with natural strains, and horizontal gene transfer from GEO to natural strains.
</P><H3><SPAN class="mw-headline" id="Risk_Assessment">Risk Assessment</SPAN></H3><P>Risk assessment provides an idea of what kind of risk we face in releasing the GEO in the environment and helps to design containment devices. Here we adapted a risk management method in workplaces complying with health and safety law <SUP><A href="#References">3</A></SUP>.
</P><P>To-do-list to assess risk in 5 steps:
</P><OL><LI>Identify the hazards
</LI><LI>Decide who might be harmed and how
</LI><LI>Evaluate the risks and decide on precaution
</LI><LI>Record your findings and implement them
</LI><LI>Review your assessment and update if necessary
</LI></OL><P>
Worksheet example:
</P><TABLE style="border-spacing:0;"><TBODY><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>What are the hazards?</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Who might be harmed and how?</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>What are we already doing?</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>What further action is necessary?</B></TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> GE bacteria outcompete natural strains
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> GE bacteria escaping the containment may outcompete natural strains if they have better fitness, creating natural imbalance
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> Using harmless strain or strain with low fitness compared to natural strain (standard E coli for lab)
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> Designing a safety containment to prevent the cells from reproducing themselves at some point and/or separating them from natural strains
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> Horizontal gene transfer (HGT)
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> Other strain/species may uptake engineered genes, and if the genes give advantage in fitness, it may outcompete other strains creating natural imbalance
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> -
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> Designing a safety containment to degrade DNA and/or separate the DNA from the environment and/or prevent natural strains from gaining advantages from modified DNA
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Controlling_hazards_and_risks">Controlling hazards and risks</SPAN></H3><P>In this step we decided what control elements we want to implement in our system to reduce the risks. We should choose the best suitable control elements depending on the system function and application. As we already know that we want to apply bWARE safety containment, here we describe the role of each safety part in risk reduction. 
</P><OL><LI>GE bacteria outcompete natural strains
</LI><LI>In order to prevent GE bacteria from escaping and outcompeting the natural strains, we will stop their reproduction by putting a self killing mechanism based on <A href="https://2012.igem.org/Team:Paris_Bettencourt/Suicide" class="external text" rel="nofollow">suicide</A> and <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme" class="external text" rel="nofollow">restriction enzyme</A> systems to kill the cells <A href="https://2012.igem.org/Team:Paris_Bettencourt/Delay" class="external text" rel="nofollow">after</A> they perform their function. 
</LI><LI>In case of the failure of the killing mechanism, we will put the cells inside a <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation" class="external text" rel="nofollow">physical containment</A> so there will be no physical interaction between them and the surrounding cells.
</LI><LI>HGT
</LI><LI>We will use a <A href="https://2012.igem.org/Team:Paris_Bettencourt/Suicide" class="external text" rel="nofollow">DNAse</A> to degrade DNA after the cells perform their function so they won’t leave any genetic material behind.
</LI><LI>In case of the failure due to inefficiency of the DNAse, the <A href="https://2012.igem.org/Team:Paris_Bettencourt/Encapsulation" class="external text" rel="nofollow">physical containment</A> will still protect the DNA from the environment.
</LI><LI>In case of the leakiness of the physical containment, the DNA has a special encryption system, i.e. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Semantic_containment" class="external text" rel="nofollow">semantic containment</A>, so the receiver cells will not be able to read GE genetic material.
</LI></OL><H3><SPAN class="mw-headline" id="Control_Checks">Control Checks</SPAN></H3><P>To check and predict the reliability of our system, we used fault tree analysis to assess multiple safety containment modules in one biodevice. This top-down approach allows us to see the relationship between each containment element and predict the overall failure probability. With this method we can see what basic events are the key elements in our system.
</P><P>AND gate represents independent events
</P><P>P(A and B) = P(A)P(B)
</P><P>OR gate corresponds to set of union
</P><P>P(A or B) = P(A) + P(B)
</P><P>so the total failure probability of the system is
</P><P>Ptotal = P1.P2.P3.P4.P5 + P1.P6.(P5+P7+P8+P9)
</P><P>List of the basic events
</P><TABLE style="border-spacing:0;"><TBODY><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Notation</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Failure component</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Failure mode</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Consequence</B></TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"><B>Method for determining the probability</B></TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P1
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> physical containment
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> leakage
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> DNA/cell escape
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> experiment
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P2
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> DNA uptake
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> DNA transferred into natural strain
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> HGT
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> transconjugant to donor ratio in HGT is typically &lt;10<SUP>-5</SUP><SUP><A href="#References">4</A></SUP>, assume this is the rate per generation
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P3
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> semantic containment
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> genetic failure
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> HGT
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> estimation
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P4
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> population suicide
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no surrounding cells with enough toxin
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no death
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> based on the cell density and the expectation number of surrounding non-mutant
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P5
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> toxin production
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> genetic failure
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no DNA degradation (w/o antitoxin), no death (with antitoxin)
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> estimation
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P6
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> mutant fitness
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> beneficial mutation
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> outcompetition of the mutant
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> -
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P7
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> delay system
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> genetic failure
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no antitoxin degradation
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> estimation
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P8
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> restriction enzyme
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> genetic failure
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no antitoxin degradation
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> estimation
</TD></TR><TR><TD style="border:0.035cm solid #000000;padding:0.176cm;"> P9
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> restriction site
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> genetic failure
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> no antitoxin degradation
</TD><TD style="border:0.035cm solid #000000;padding:0.176cm;"> estimation
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Physical_containment_failure">Physical containment failure</SPAN></H4><P>From the experiment we get the escape rate of the cells from the alginate beads is (escaping cells/total cells) is 10<SUP>-5</SUP> per 12 hours. Assuming 30 minutes of division time, we have 24 generations, and the escape rate per generation is 4 x 10<SUP>-6</SUP>.
</P><H4><SPAN class="mw-headline" id="Genetic_failure">Genetic failure</SPAN></H4><P>There are different processes that lead to a loss of function of a gene
</P><UL><LI> Mutation
</LI><LI> Recombination
</LI><LI> Plasmid loss
</LI></UL><P><B>Losing function because of mutation</B></P><P>Estimating the mutation rate which cause failure of the safety containment of the device
</P><UL><LI> Spontaneous mutation rate of a wild-type E coli strain growing on research medium is 3.3 x 10<SUP>-9</SUP> per nucleotide per generation <SUP><A href="#References">5</A></SUP></LI><LI> approximately ⅔ of nucleotide mutation will change amino acid (64 combination of nucleotides code only 20 amino acids)
</LI><LI> approximately 70% of amino acid change will lose the part function <SUP><A href="#References">6</A></SUP></LI><LI> mean length of genetic coding sequence for E coli (procaryotes) is 924 bp <SUP><A href="#References">7</A></SUP> ≈ 1000 bp 
</LI></UL><P>So the loss function rate of a gene per generation because of mutation is 
</P><P>P<SUB>L</SUB>= 3.3 x 10<SUP>-9</SUP> x ⅔ x 0.70 x 1000 = 1.54 x 10<SUP>-6</SUP></P><P><B>Recombination</B></P><P>Finding a definite rate of recombination is not easy. Guttman and Dykhuizen in 1994 <SUP><A href="#References">8</A></SUP> stated that recombination is believed to occur at such a low frequency, but their experiment showed that the rate could be as high as mutation rate (50 fold higher compared to the mutation rate believed at that moment, but actually it is 5.0 x 10<SUP>-9</SUP> changes per nucleotide per generation, a bit more than one fold to the value believed now). 
</P><P>Redoing the previous calculation with this number will lead us to a rate of 2.33 x 10<SUP>-6</SUP> per gene per generation.
</P><P><B>Plasmid loss</B></P><P>According to L Boe <SUP><A href="#References">9</A></SUP> plasmid loss rate is defined as the probability that a division of a plasmid-carrying individual results in the birth of one plasmid-free and one plasmid-carrying daughter cell. For the low copy plasmid this rate is about ~1% /division and for high copy plasmid is ~0.01% /division, neglecting the viable/non-viable factor of the cells of losing the plasmid (no selection). If a plasmid with a lost rate of 0.05 is stabilized with a 100% efficient postsegragational killing system, the 5% of the division will result in non viable daughter cell, and we can not see the effect except of increasing the appearing division time by ln 2/ln (1-0.05) = 1.04. Thus in our system we can neglect this plasmid loss assuming that we have antibiotic selection in the beads.
</P><P>Assuming only mutation and recombination could happen to delete a function of a gene, we have the total genetic failure rate approximately 4 x 10<SUP>-6</SUP> per gene per generation.
</P><H4><SPAN class="mw-headline" id="Total_failure_probability">Total failure probability</SPAN></H4><P>Having Ptotal = P1.P2.P3.P4.P5 + P1.P6.(P5+P7+P8+P9) and put 1 for unknown probability (P4 and P6) as a worst-case scenario, we come to the failure rate number 6.4 x 10<SUP>-11</SUP> per generation.
</P><P>A volume of one alginate bead is approximately 20 uL. So if the cells grow until they reach the stationary phase (4 x 10<SUP>9</SUP> cells/ml), we will have 8 x 10<SUP>7</SUP> cells. If we started from 100 cells per bead, the number of generations we have is <SUP>2</SUP>log (8 x 10<SUP>7</SUP> / 100) =19.6 generations. Therefore the total failure becomes 6.4 x 10<SUP>-11</SUP> x 19.6 = 1.25 x 10<SUP>-9</SUP>.
</P><P>With this method we can also see that the sense-HGT rate is much lower (6.4 x 10<SUP>-22</SUP>) , almost negligible compared to the outcompetition rate, because it has more back up/redundancy components.
</P><H2><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H2><P>We have proposed and implemented an example of assessing biosafety by adapting existing methods from safety engineering. However, it is very clear that the prediction is based on rough estimation and we need more experimental data to verify this estimation. For an example, Torres et al <SUP><A href="#References">10</A></SUP> tested a dual containment killing system and compared it with single containment system. They tested an EcoRI based system and a Colicin E3-based system and obtained the efficacy (based on the numbers of successful transformants) each 10<SUP>4</SUP> and 10<SUP>5</SUP> respectively but when they combine both system they only got efficacy 10<SUP>6</SUP>, while theoretically they could reach up to 10<SUP>9</SUP>. This could give us an idea that predicting the performance of a combination of genetic parts is not as easy as combining the performance of each genetic part. However, we can still conclude that having redundant parts is necessary to reduce the failure rate and make it as low as possible.
</P><P>Research in measuring the evolutionary stability in E coli was performed by Sean C Sleight et al <SUP><A href="#References">11</A></SUP> where they measured it of BioBrick-assembled genetic circuits in E coli over multiple generation by measuring the number of loss-functioning mutants. They concluded that to make a robust GE bacteria, one has to take into account 3 principles:
</P><OL><LI> High expression of genetic circuits comes with the cost of low evolutionary stability (for example induced vs non-induced system)
</LI><LI>Avoid repeated sequences because it will more likely be mutated
</LI><LI>Use of inducible promoters generally increases evolutionary stability compared to constitutive promoters
</LI></OL><P>To predict the robustness of the system from mutation, we have to take into account the duration (i.e. how many generations) we want our system to work. From their experiment for example, the AHL induced TetR-GFP cells lose their functions after 30 generations, while the uninduced ones lose their function slower (50% after 300 generations). A similar idea is also shown from our work where the mutation rate we got from literature research is in the unit of per generation. This is how reproduction differs biological system from other engineering systems. In engineering we also have failure probability as a function of time because of the life cycle of the system, but the system is not reproducing so it has no variability. 
</P><P>Analyzing key elements that cause failure in any system including synthetic biology systems is essential for system improvement and prediction of failure. Adaptation of classical safety engineering methods needs to take into account the unique properties of synthetic biology. Reproducibility and complexity are two examples of areas that need to be studied deeper in this context. We invite future iGEM teams to use and improve our assessment framework, build a solid assessment protocol towards a better understanding of biosafety and successful application of synthetic biology.
</P><H2><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H2><UL><LI>We can adapt methods in safety engineering to assess risk in biosafety
</LI><LI>We need more experimental data to confirm the risk assessing method
</LI></UL><H2><SPAN class="mw-headline" id="Perspectives"> Perspectives </SPAN></H2><UL><LI>We need a method to predict the evolutionary stability of circuits from the properties of their parts, but the emergent behaviours of circuits will likely make prediction difficult. Thus it will be very useful if each BioBricks parts is completed with evolutionary stability data sheet, so we can make prediction of the stability of more complex circuits.
</LI><LI>This safety assessment method is useful for identifying key elements of having failure in synthetic circuit systems before we go further to the real application. Thus we recommend for the future iGEM team to use our assessment method and give feedback to the community by improving this framework.
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P><A href="#Background">1</A> - Genya V Dana et al. Four steps to avoid a synthetic biology disaster. 2012. Nature vol 483
</P><P><A href="#Methods_and_Results">2</A> - WorkSafe Victoria. A handbook for workplaces, controlling OHS hazards and risks. 2007. Edition No.1
</P><P><A href="#Methods_and_Results">3</A> - “Five steps to risk assessment” from <A href="http://www.hse.gov.uk/risk/fivesteps.htm" class="external free" rel="nofollow">http://www.hse.gov.uk/risk/fivesteps.htm</A>. - risk management method in workplaces complying with health and safety law 
</P><P><A href="#Check_controls">4</A> - Soren J Sorensen et al. 2005. Studying plasmid horizontal gene transfer in situ: a critical review. Nature Reviews Microbiology Vol 3
</P><P><A href="#Check_controls">5</A> - Heewook Lee et al., Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. 2012. PNAS E2774-E2783
</P><P><A href="#Check_controls">6</A> - Stanley A. Sawyer et al. Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila. 2007. PNAS 104(16):6504-6510
</P><P><A href="#Check_controls">7</A> - Lin Xu et al. Average gene length is highly conserved in prokaryotes and eukaryotes and diverges only between two kingdoms. 2006. Mol. Biol. Evol. 23(6):1107-1108
</P><P><A href="#Check_controls">8</A> - David S Guttmand and Daniel E. Dykhuizen. 1994. Clonal divergence in Escherichia coli as a result of recombination, not mutation. Science Vol 266
</P><P><A href="#Check_controls">9</A> - L.Boe. 1996. Estimation of Plasmid Loss Rates in Bacterial Populations with a Reference to the Reproducibility of Stability Experiments. Plasmid 36, 161–167. Article No. 0043
</P><P><A href="#Discussion">10</A> - Torres B et al. 2003. A dual lethal system to enhance containment of recombinant micro-organisms. Microbiology. 2003 Dec;149(Pt 12):3595-601.
</P><P><A href="#Discussion">11</A> - Sean C Sleight et al., Designing and engineering evolutionary robust genetic circuits. 2010. Journal of Biological Engineering, 4:12
</P></DIV><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Modeling">http://2012.igem.org/Team:Paris_Bettencourt/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;oldid=296461" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>