<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Restriction_Enzyme"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Restriction_Enzyme">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Restriction_Enzyme&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Restriction_Enzyme&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Restriction_Enzyme&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Restriction_Enzyme" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Restriction Enzyme</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM Paris Bettencourt 2012</TITLE></P><DIV id="wrapper"><DIV id="menubarbleu" class=""><UL><LI class="first"><SPAN class="opener">Main</SPAN></LI><LI><A href="/Team:Paris_Bettencourt">Home</A></LI><LI><A href="/Team:Paris_Bettencourt/Parts">Parts Submitted</A></LI><LI><A href="/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:Paris_Bettencourt/Contact">Contact</A></LI><LI><A href="/Team:Paris_Bettencourt/Bonus">Bonus</A></LI></UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Team">Team</A></LI><LI><SPAN class="opener">Project</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Overview"> Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Delay"> Delay system</A></LI><LI><A href="/Team:Paris_Bettencourt/Semantic_containment"> Semantic containment</A></LI><LI><A href="/Team:Paris_Bettencourt/Restriction_Enzyme"> Restriction enzyme system</A></LI><LI><A href="/Team:Paris_Bettencourt/MAGE"> MAGE</A></LI><LI><A href="/Team:Paris_Bettencourt/Suicide"> Suicide system</A></LI><LI><A href="/Team:Paris_Bettencourt/Encapsulation"> Encapsulation</A></LI><LI><A href="/Team:Paris_Bettencourt/SID"> Synthetic import domain</A></LI><LI><A href="/Team:Paris_Bettencourt/Achievements">Achievements</A></LI><LI><SPAN class="opener2">Human Practice</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Overview">Overview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Interview">Interview</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Report">Report</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Debate">Debate</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/Workshop">Workshop</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/WikiScreen">Wiki Screen</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/HGT">HGT</A></LI><LI><A href="/Team:Paris_Bettencourt/Human_Practice/perception">Team Perception</A></LI><LI><SPAN class="opener2">Safety</SPAN></LI><LI><A href="/Team:Paris_Bettencourt/Safety"> Safety Questions</A></LI><LI><A href="/Team:Paris_Bettencourt/Modeling"> Safety Assessment</A></LI><LI><A href="/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="last"><A href="/Team:Paris_Bettencourt/Attributions">Attributions</A></LI></DIV><DIV id="page"><DIV id="grouptitle">Restriction Enzyme System</DIV><TABLE id="tableboxed"><TBODY><TR><TD><P><B> Aim: </B></P><P>To design a plasmid self-digestion system.
</P><P><B>Experimental System:</B></P><P>We are testing different combinations of promoters and restriction enzymes. We have to characterize both the promoters (by measuring the expression of RFP) and the restriction enzymes (by measuring killed cells).
</P><P><B>Achievements :</B></P><UL><LI> Construction of 4 biobricks <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Design" class="external text" rel="nofollow">[Read more]</A>:
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914003" class="external text" rel="nofollow">K914003</A>: L-rhamnose-inducible promoter
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914005" class="external text" rel="nofollow">K914005</A>: Meganuclease I-SceI controlled by pLac
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914007" class="external text" rel="nofollow">K914007</A>: Meganuclease I-SceI controlled by pBad
</LI><LI><A href="http://partsregistry.org/Part:BBa_K914008" class="external text" rel="nofollow">K914008</A>: Meganuclease I-SceI controlled by pRha
</LI></UL><LI> Demonstration that all 3 generators (K914005, K914007, K914008) work and express I-SceI meganuclease in cells. <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#I-SceI_is_expressed_and_active_in_eliminating_restriction-site_harbouring_plasmid" class="external text" rel="nofollow">[Read more]</A></LI><DIV id="boston"><UL><LI><B><A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#New_Results" class="external text" rel="nofollow">[Updated for Boston]</A></B> : Characterization of 2 biobricks from TUDelft <A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Measuring_the_I-SceI_efficiency_.28TUDelft_parts.29" class="external text" rel="nofollow">[Read more]</A>:
</LI><LI><A href="http://partsregistry.org/Part:BBa_K175041" class="external text" rel="nofollow">K175041</A>: p(LacI) controlled I-SceI homing endonuclease generator
</LI><LI><A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>: I-SceI restriction site
</LI></UL></DIV><UL><LI><A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Characterization_of_pRha" class="external text" rel="nofollow">Characterization </A> of the L-rhamnose-inducible promoter (<A href="https://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme#Design" class="external text" rel="nofollow">pRha</A>). 
</LI></UL></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Objectives"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Experiments_and_results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Experiments and results</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#I-SceI_is_expressed_and_active_in_eliminating_restriction-site_harbouring_plasmid"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">I-SceI is expressed and active in eliminating restriction-site harbouring plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Transformation_results"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Transformation results</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#The_first_combination_of_plasmids:"><SPAN class="tocnumber">4.1.1.1</SPAN><SPAN class="toctext">The first combination of plasmids:</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#The_second_combination_of_plasmids:"><SPAN class="tocnumber">4.1.1.2</SPAN><SPAN class="toctext">The second combination of plasmids:</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#The_third_combination_of_plasmids:"><SPAN class="tocnumber">4.1.1.3</SPAN><SPAN class="toctext">The third combination of plasmids:</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-10"><A href="#Control_for_plasmid_compatibility"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Control for plasmid compatibility</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Day_light_photos:"><SPAN class="tocnumber">4.1.2.1</SPAN><SPAN class="toctext">Day light photos:</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Photos_of_fluorescence:"><SPAN class="tocnumber">4.1.2.2</SPAN><SPAN class="toctext">Photos of fluorescence:</SPAN></A></LI><LI class="toclevel-4 tocsection-13"><A href="#Recovery_in_glucose"><SPAN class="tocnumber">4.1.2.3</SPAN><SPAN class="toctext">Recovery in glucose</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#The_first_combination_of_plasmids:_2"><SPAN class="tocnumber">4.1.2.4</SPAN><SPAN class="toctext">The first combination of plasmids:</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#The_second_combination_of_plasmids:_2"><SPAN class="tocnumber">4.1.2.5</SPAN><SPAN class="toctext">The second combination of plasmids:</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#The_third_combination_of_plasmids:_2"><SPAN class="tocnumber">4.1.2.6</SPAN><SPAN class="toctext">The third combination of plasmids:</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Results"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Measuring_the_I-SceI_efficiency_.28TUDelft_parts.29"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Measuring the I-SceI efficiency (TUDelft parts)</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Step_1"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Step 1</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Step_2"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Step 2</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Step_3"><SPAN class="tocnumber">4.2.3</SPAN><SPAN class="toctext">Step 3</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Results_2"><SPAN class="tocnumber">4.2.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-23"><A href="#The_combination_of_plasmids:"><SPAN class="tocnumber">4.2.4.1</SPAN><SPAN class="toctext">The combination of plasmids:</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#New_Results"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">New Results</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Characterization_of_pRha"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Characterization of pRha</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Experimental_setup"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Results_3"><SPAN class="tocnumber">4.4.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>We designed a plasmid self-digestion system. This synthetic system allows to digest plasmids into linear parts of DNA and thus disrupt the expression of the genes carried by this plasmid. In our specific containment module, this will result in extinguishing the expression of the antitoxin (Colicin immunity protein), and any plugged-in synthetic device. This will lead to activation of the colicin DNAse toxin thst will degrade  the cell's chromosomal DNA, leading to its death.
</P><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><OL><LI> Find appropriate restriction enzymes which have to match the following properties:
</LI><LI> The corresponding restriction site must not be found in the <I>E.Coli</I> genome;
</LI><LI> The enzyme has to have high specifity;
</LI><LI> It has to work in wide range of different conditions (pH, T°, etc)
</LI><LI> Choose a strong yet tightly repressible promoter to regulate the restriction enzyme expression;
</LI><LI> Clone circuits with different combinations of the chosen restriction enzymes and promoters;
</LI><LI> Measure the degradation efficiency of the restriction enzyme for each circuit;
</LI><LI> Based on the best combination, design a self-disruption plasmid.
</LI></OL><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><P>According to the first two of our objectives, we should find an appropriate restriction enzyme and to choose a strong yet tightly repressible promoter to regulate its expression.
</P><P><B>Restriction enzyme candidates:</B></P><OL><LI><B>Fse I</B> is a restriction endonuclease which recognizes an 8bp long DNA sequence: GGCCGG▽CC (CC△GGCCGG). The reason why we chose it is because it has the lowest number of restriction sites in  the <I>E.coli</I> genome: only 4 copies. We decided to use MAGE to remove those sites from the chromosome (<A href="https://2012.igem.org/Team:Paris_Bettencourt/MAGE" class="external text" rel="nofollow">see for more details</A>). However, MAGE did not have the expected yield, and we decided to freeze the work on this restriction enzyme and focus on the second candidate.</LI><LI><B>I-SceI</B> is an intron-encoded endonuclease. It is present in the mitochondria of <I>Saccharomyces cerevisiae</I> and recognises an 18-base pair sequence 5'-TAGGGATAA▽CAGGGTAAT-3' (3'-ATCCC△TATTGTCCCATTA-5') and leaves a 4 base pair 3' hydroxyl overhang. It is a rare cutting endonuclease. Statistically an 18-bp sequence will occur once in every 6.9*10^10 base pairs or once in 20 human genomes.</LI></OL><P><B>Promoter candidates:</B></P><OL><LI><B>pLac</B></LI><LI>
Standard pLac promoter from the Parts Registry: <A href="http://partsregistry.org/Part:BBa_R0011" class="external text" rel="nofollow">R0011</A>.<SUP>[</SUP><SUP><A href="#References">7</A></SUP><SUP>]</SUP></LI><LI><B>pBad</B></LI><LI>Standard pBad promoter from the Parts Registry: <A href="http://partsregistry.org/Part:BBa_I13453" class="external text" rel="nofollow">I13453</A>.<SUP>[</SUP><SUP><A href="#References">8</A></SUP><SUP>]</SUP></LI><LI><B>pRha</B> L-rhamnose-inducible promoter is capable of high-level protein expression in the presence of L-rhamnose, it is also tightly regulated in the absence of L-rhamnose and by the addition of D-glucose.

</LI><LI>pRha is probably the best candidate for us because of two reasons:

</LI><LI> It is a new promoter, and so it will be orthogonal to any other existing system. It supports the modularity idea of our project.</LI><LI> It was reported in the literature to be appropriate for the expression of toxic genes.</LI></OL><LI>L-rhamnose is taken up by the RhaT transport system, converted to L-rhamnulose by an isomerase RhaA and then phosphorylated by a kinase RhaB. Subsequently, the resulting rhamnulose-1-phosphate is hydrolyzed by an aldolase RhaD into dihydroxyacetone phosphate, which is metabolized in glycolysis, and L-lactaldehyde. The latter can be oxidized into lactate under aerobic conditions and be reduced into L-1,2-propanediol under unaerobic conditions.<SUP>[</SUP><SUP><A href="#References">7</A></SUP><SUP>]</SUP></LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><FONT size="1"><B>The <I>E. coli</I> rhaBRS locus.</B> In the presence of L-rhamnose, RhaR activates transcription of <I>rhaR</I> and <I>rhaS</I>, resulting in an accumulation of RhaS. RhaS then acts as the L-rhamnose-dependent positive regulator of the <I>rhaB</I> promoter.</FONT></DIV></DIV></DIV><LI>The genes rhaBAD are organized in one operon which is controlled by the rhaPBAD promoter. This promoter is regulated by two activators, RhaS and RhaR, and the corresponding genes belong to one transcription unit which is located in opposite direction of rhaBAD. If L-rhamnose is available, RhaR binds to the rhaPRS promoter and activates the production of RhaR and RhaS. RhaS together with L-rhamnose in turn binds to the rhaPBAD and the rhaPT promoter and activates the transcription of the structural genes. However, for the application of the rhamnose expression system it is not necessary to express the regulatory proteins in larger quantities, because the amounts expressed from the chromosome are sufficient to activate transcription even on multi-copy plasmids. Therefore, only the rhaPBAD promoter has to be cloned upstream of the gene that is to be expressed. Full induction of rhaBAD transcription also requires binding of the CRP-cAMP complex, which is a key regulator of catabolite repression.<SUP>[</SUP><SUP><A href="#References">11</A></SUP><SUP>]</SUP></LI><LI>The pRha sequence was containing an EcoRI restriction site, so we had to disrupt it in order to use pRha as a biobrick. In order to decide which base pair to modify, we used the <A href="http://microbes.ucsc.edu" class="external text" rel="nofollow">UCSC Microbial Genome Browser</A>. We compared the pRha sequence in E.coli and similar species, and identified that the at the position is sometimes replaced by a in some species, so we decided to replace it in a same way. We ordered a gBlock with the pRha sequence having the mutation, and this is the sequence we used and submitted.</LI><P>Considering these candidates, we decided to clone the following constructs in low-copy vector pSB3C5 to use it in our experiments, all with a medium RBS <A href="http://partsregistry.org/Part:BBa_B0032" class="external text" rel="nofollow">B0032</A>:
</P><CENTER><TABLE class="wikitable" width="90%" style="text-align: center;"><TBODY><TR><TD> pBad &amp; RBS &amp; I-SceI
</TD><TD> pRha &amp; RBS &amp; GFP
</TD><TD> pBad &amp; RBS &amp; RFP
</TD></TR><TR><TD> pLac &amp; RBS &amp; I-SceI
</TD><TD> pRha &amp; RBS &amp; GFP
</TD><TD> pLac &amp; RBS &amp; RFP
</TD></TR><TR><TD> pRha &amp; RBS &amp; I-SceI
</TD><TD> pRha &amp; RBS &amp; GFP
</TD><TD> pRha &amp; RBS &amp; RFP
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Experiments_and_results">Experiments and results</SPAN></H2><H3><SPAN class="mw-headline" id="I-SceI_is_expressed_and_active_in_eliminating_restriction-site_harbouring_plasmid">I-SceI is expressed and active in eliminating restriction-site harbouring plasmid</SPAN></H3><P>To measure the digestion efficiency of I-SceI, we transformed two plasmids with different antibiotic resistances into NEB Turbo <I>E.Coli</I> strain:
</P><OL><LI><B>First plasmid:</B> Low copy plasmid with encoded generator to express I-SceI meganucllease. Three version with different promoters was tested: I-SceI meganuclease controlled by pBad, pLac and pRha. For all version:
</LI><LI> Backbone: pSB3C5
</LI><LI> Resistance: Chloramphenicol
</LI><LI> Origin of Replication: p15a
</LI><LI><B>Second plasmid:</B> High copy plasmid with encoded I-SceI restriction site, <A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>. This biobrick was a kind gift of the <A href="https://2012.igem.org/Team:TU-Delft" class="external text" rel="nofollow">TUDelft iGEM team</A> which we further characterized.
</LI><LI> Backbone: pSB1AK3
</LI><LI> Resistance: Ampicillin and Kanamycin
</LI><LI> Origin of Replication: modified pMB1 derived from pUC19
</LI></OL><P>We expected to perform transformation with both plasmids, and plate with two antibiotics in order to select for double transformants. We would then induce I-SceI expression in those clones to measure  its efficiency.
</P><H4><SPAN class="mw-headline" id="Transformation_results">Transformation results</SPAN></H4><P>From the experiment we can clearly see that on plates with two antibiotics (Chloramphenicol,  Cm; &amp; Ampicillin, Amp) there are no colonies, while on plates with only one antibiotic Cm or Amp there are numerous colonies.
</P><H5><SPAN class="mw-headline" id="The_first_combination_of_plasmids:">The first combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pBad &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="The_second_combination_of_plasmids:">The second combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pLac &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="The_third_combination_of_plasmids:">The third combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pRha &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We suggested <B>two hypotheses</B> to explain the results:
</P><OL><LI><B>Those two plasmids are not compatible.</B> Plasmids could have different origins of replication. That might be the reason why double transformation is unsuccessful. </LI><LI><B>Our system works.</B> Our system perfectly works, but there is some leakage in the promoter leading to the expression of I-SceI meganuclease. In such case, it very efficiently cuts I-SceI restriction site, digesting the second plasmid with ampicillin resistance.</LI></OL><P>
First, we decided to check the hypothesis 1, and verify if two plasmids are compatible with each other.
</P><H4><SPAN class="mw-headline" id="Control_for_plasmid_compatibility">Control for plasmid compatibility</SPAN></H4><P>As control experiment, we decided to trasform two plasmids into NEB Turbo <I>E.Coli</I> strain. The first plasmid in this experiment is analogous to the one from the previous experiment, but with GFP insted of I-SceI meganuclease; the second plasmid is the same as in the previous experiment:
</P><OL><LI><B>First plasmid:</B> Low copy plasmid with encoded generator to express GFP meganucllease. Only the version with pLac promoter was tested, because they all have the same backbone plasmid, and consequently the same replication prigin.
</LI><LI> Backbone: pSB3C5
</LI><LI> Resistance: Chloramphenicol
</LI><LI> Origin of Replication: modified pMB1 derived from pUC19
</LI><LI><B>Second plasmid:</B> High copy plasmid with encoded I-SceI restriction site, <A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>.
</LI><LI> Backbone: pSB1AK3
</LI><LI> Resistance: Ampicillin and Kanamycin
</LI><LI> Origin of Replication: modified pMB1 derived from pUC19
</LI></OL><P>We expected to have colonies on both type of plates: firstly, on plates with one antibiotic (Chloramphenicol &amp; Ampicillin), secondly, on plates with both antibiotics.
</P><P>Our expectations became true, and our cells expressed GFP, so we conclude these two plasmids have compatible replication origins, and there should be nothing preventing the I-SceI from being expressed in the previous experiment. That means that our circuits work, but there is some leaky expression of I-SceI meganuclease that leads to a very efficient digestion of the plasmid carrying the Ampicillin antibiotic. 
</P><H5><SPAN class="mw-headline" id="Day_light_photos:">Day light photos:</SPAN></H5><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Photos_of_fluorescence:">Photos of fluorescence:</SPAN></H5><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
To avoid leakage, in the next experiment we tried to recover cells after transformation and plate it in the presence of glucose that represses the pLac promoter.
</P><H5><SPAN class="mw-headline" id="Recovery_in_glucose">Recovery in glucose</SPAN></H5><H5><SPAN class="mw-headline" id="The_first_combination_of_plasmids:_2">The first combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pBad &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="The_second_combination_of_plasmids:_2">The second combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pLac &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="The_third_combination_of_plasmids:_2">The third combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pRha &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Amp</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>Even if we recover the double transformants in the presence of Glucose to tightly repress the expression of the meganuclease, we still have no colonies on the plates containing both Amp and Cm. This is a sign that our construct is leaky, and the expression and function of I-SceI is so efficient that it digests all plasmids containing the corresponding restriction site, and the cells are no longer resistant to Amp.
</P><H3><SPAN class="mw-headline" id="Measuring_the_I-SceI_efficiency_.28TUDelft_parts.29">Measuring the I-SceI efficiency (TUDelft parts)</SPAN></H3><P>In 2009, <A href="https://2009.igem.org/Team:TUDelft/SDP_Overview" class="external text" rel="nofollow">TUDelft iGEM Team</A> has already tried to design a Self Destructive Plasmid based on I-SceI meganuclease. For their experiments, they designed a generator to produce I-SceI meganuclease and used the same pLac promoter, but they had two essential differences with our system:
</P><UL><LI> They used a <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0030" class="external text" rel="nofollow">strong RBS</A>.
</LI><LI> I-SceI meganuclease they used had an LVA tag.
</LI></UL><P>Moreover, they didn't submit the meganuclease alone as a biobrick, so it couldn't be used for constructing new composite biobricks controlled by other promoters, which would be very useful for the modularity of our system. That is why we started working on our own constructions of meganuclease first. However, we asked TUDelft to send us two plasmids that they designed, so that we can test and characterize them:
</P><OL><LI><B>First plasmid:</B> High copy plasmid with encoded generator to express I-SceI meganucllease, <A href="http://partsregistry.org/Part:BBa_K175041" class="external text" rel="nofollow">BBa_K175041</A>:
</LI><LI> Backbone: pSB1C3
</LI><LI> Resistance: Chloramphenicol
</LI><LI> Origin of Replication: modified pMB1 derived from pUC19
</LI><LI><B>Second plasmid:</B> High copy plasmid with encoded I-SceI restriction site, <A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>:
</LI><LI> Backbone: pSB1AK3
</LI><LI> Resistance: Ampicillin and Kanamycin
</LI><LI> Origin of Replication: modified pMB1 derived from pUC19
</LI></OL><H5><SPAN class="mw-headline" id="Step_1">Step 1</SPAN></H5><P>To mesure the efficiency of I-SceI from TUDelft parts, we proceeded in the same way as for the characterization of our own designs: we transformed two plasmids with different antibiotic resistances into NEB Turbo <I>E.Coli</I> strain.
</P><P>The transformation was successful.
</P><H5><SPAN class="mw-headline" id="Step_2">Step 2</SPAN></H5><P>We followed the following protocol:
</P><UL><LI> Pick a colony and start a liquid culture with both antibiotic resistances (Amp and Cm).
</LI><LI> Incubate at 37°C  until optical density (OD) reaches 0.5
</LI><LI> Pellet and wash cells to remove Amp and Cm.
</LI><LI> Re-dilute them in the same volume of LB.
</LI><LI> From each of those two tubes, start two liquid cultures:
</LI><LI> The first culture with Cm and w/o IPTG.
</LI><LI> The second culture with Cm and IPTG.
</LI><LI> Incubate overnight at 37°C
</LI></UL><H5><SPAN class="mw-headline" id="Step_3">Step 3</SPAN></H5><P>Plate colonies from each tube on two different plates:
</P><OL><LI> Selection with Cm and Amp.
</LI><LI> Selection with Cm.
</LI></OL><H4><SPAN class="mw-headline" id="Results_2"> Results </SPAN></H4><P>To analyze data, we counted and compared the number of colonies on four plates corresponding to 4 conditions.
</P><UL><LI>First, we compared the number of CFU formed by non-induced cells, plated with a single antibiotic and with both antibiotics. We expected that the plate with only Cm selection would have the biggest number of colonies. A smaller number would be on the plate with two antibiotics (Amp and Cm). It could be explained by the loss of the plasmid carrying Amp resistance.
</LI><LI>Next, we compared the number of CFUs formed by the cells where we induced the I-SceI expression, plated with a single antibiotic and with both antibiotics. We expect that there would be much less colonies on the double antibiotic plate, because the Ampicillin carrying plasmid would be lost not only due to the absence of selection during growth (as for the first two cells), but also due to the digestion by the restriction enzyme.
</LI></UL><H5><SPAN class="mw-headline" id="The_combination_of_plasmids:">The combination of plasmids:</SPAN></H5><UL><LI>First plasmid: pLac &amp; RBS &amp; I-SceI [Cm]</LI><LI>Second plasmid: I-SceI restriction site [Amp]</LI></UL><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm  Plated colonies: w/o IPTG</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm  Plated colonies: IPTG induced</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp  Plated colonies: w/o IPTG</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Selection: Cm &amp; Amp   Plated colonies: IPTG induced</DIV></DIV></DIV></DIV></TD></TR><TR><TD colspan="2"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">CFU number from 4 plates.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
From photos and bar graph above we can see that our expectation came true.
</P><H3><SPAN class="mw-headline" id="New_Results">New Results</SPAN></H3><DIV id="boston"><P>
Before going to Boston, we improved our protocol to characterize TUDelft BioBricks and showed the reproducibility of the results.
</P><P><B>New protocol:</B></P><P>Both TUDelft plasmids are transformed into the <I>E.coli</I> NEB Turbo strain:
</P><OL><LI><B>First plasmid:</B> High copy plasmid with encoded generator to express I-SceI meganuclease, <A href="http://partsregistry.org/Part:BBa_K175041" class="external text" rel="nofollow">BBa_K175041</A>:
</LI><LI> Resistance: Chloramphenicol
</LI><LI><B>Second plasmid:</B> High copy plasmid with encoded I-SceI restriction site, <A href="http://partsregistry.org/Part:BBa_K175027" class="external text" rel="nofollow">K175027</A>:
</LI><LI> Resistance: Ampicillin and Kanamycin
</LI></OL><UL><LI> Start a liquid culture with Amp and Cm to select bacteria which have both plasmids.
</LI><LI> Incubate at 37°C  overnight.
</LI><LI> Pellet and wash cells to remove Amp and Cm.
</LI><LI> Re-dilute them in the LB to have OD = 0.05.
</LI><LI> From the tube, start two liquid cultures:
</LI><LI> The first culture with Cm.
</LI><LI> The second culture with Cm and IPTG.
</LI><LI> Incubate bacteria until OD = 0.5
</LI><LI>Plate colonies from each tube on two different plates:
</LI><LI> Select with Cm and Amp.
</LI><LI> Select with Cm.
</LI></UL><P>The advantage of the new version of our protocol is that we add IPTG at the very low OD, so we expect to have better induction of the meganuclease.
</P><P><B>Experimental results:</B></P><P>In much the same way as with our previous experiment, we counted and compared the number of colonies on four plates corresponding to 4 conditions:
</P><UL><LI>First, we compared the number of CFUs formed by the cells where we induced the I-SceI expression, plated with a single antibiotic or with both antibiotics. We expected that there would be fewer colonies on the double antibiotic plates, because the Ampicillin carrying plasmid would be lost not only due to the absence of selection during growth and basal expression level of I-SceI (as for the control), but also due to digestion by the restriction enzyme.
</LI></UL><UL><LI>As a control, we compared the number of CFUs formed by non-induced cells, plated with a single antibiotic and with both antibiotics. We expected that the plates with only Cm selection would have the biggest number of colonies. A smaller number would be on the plate with two antibiotics (Amp and Cm). It could be explained by the loss of the plasmid carrying Amp resistance and basal expression level of the I-SceI meganuclease.
</LI></UL><P>We calculated the ratio between plates with only one or both antibiotics for both the induced (with IPTG) and uninduced (w/o IPTG) case according to the formula:
</P><P>The experiment was repeated four times. Average value with error bars for both cases are shown on the next diagram:
</P><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Proportion of cells with ampicillin resistant plasmids.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
From the diagram we can see two times the difference between induced and uninduced cells. The results suggest that the plasmids with AmpR which carry the I-SceI restriction site are digested by the I-SceI meganuclease.
</P><P><B>Future experiments:</B>
We suggest a few ways to improve the efficiency of Restriction Enzyme System Safeguard:
</P><UL><LI> Firstly, to increase the probability to cut ampicillin resistant plasmids, we could increase the number of restriction sites it contains;
</LI><LI> Secondly, as we have showed, I-SceI meganuclease without the LVA-tag cut ampicillin resistant plasmids very efficiently even without induction of the regulated promoter. So, we could try to clone our constructs with weaker RBS or weaker LVA tag, two find a balance between the two states we describes in our experiments.
</LI></UL></DIV><H3><SPAN class="mw-headline" id="Characterization_of_pRha">Characterization of pRha</SPAN></H3><P>We have submitted to the registry a new characterized promoter: pRha <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K914003" class="external text" rel="nofollow">K914003</A>.
</P><H4><SPAN class="mw-headline" id="Experimental_setup">Experimental setup</SPAN></H4><P>In order to characterize this promoter, we made a construct with a medium RBS (<A href="http://partsregistry.org/Part:BBa_B0032" class="external text" rel="nofollow">B0032</A>) and an RFP ([]) cloned downstream of the pRha, on the pSB3C5 plasmid. We induced the expression of RFP by adding L-Rhamnose. As a negative control, we used cells without the inducer, as well as cells repressed with Glucose.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><P>First, by simple observation under a fluorescence viewer, we have seen that the addition of 1% L-Rhamnose leads to a significant expression of RFP after 10hours. The negative controls, where no Rhamnose was added, or when the promoter was repressed by Glucose, did not show any visible fluorescence. 
Both photos are taken after we centrifuged a culture of NEB Turbo strain with transformed plasmid. For the fluorescent result, the same tubes were photographed under excitation light (540nm), through an emission filter (590nm). 
</P><TABLE align="center"><TBODY><TR valign="top"><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><I>E.Coli</I><B>The left tube:</B> cells were grown without L-rhamnose. <B>The right tube:</B> cells are grown in the present of 1% of L-rhamnose.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><I>E.Coli</I><B>The right tube</B> which was induced by L-rhamnose expresses RFP, while <B>the left tube</B> where we didn't add it, has no visible expression.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We quantified this result in a plate reader.
</P><TABLE align="center"><TBODY><TR valign="top"><TD colspan="2"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:532px;"><DIV class="thumbcaption">Quantification of the fluorescence after 10h of growth</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Next, we characterized the pRha promoter using a plate reader. We used different concentrations of L-Rhamnose (0.05%, 0.1%, 0.2%, 0.5% and 1%) and observed the resulting fluorescence over time. As negative controls, we used the non-induced cells, as well as cells repressed by 1% Glucose.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Characterization of the pRha promoter using a plate reader</DIV></DIV></DIV></DIV><P>As we can see from the graph above, pRha promoter works as expected and it could be well tuned by concentration of L-rhamnose.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI><I>«Fsel, a new type II restriction endonuclease that recognizes the octanucleotide sequence 5′ GGCCGGCC 3′»</I>, Janise Meyertons Nelson+, Sheila M. Miceli, Mary P. Lechevalier1 and Richard J. Roberts*, (1990)
</LI><LI><I>«Structure and activity of the mitochondrial intron-encoded endonuclease, I-SceIV»</I>, Wernette C. M. Biochem Biophys Res Commun, (1998)
</LI><LI> Yisheng Kang et al., <I>«Systematic Mutagenesis of E.coli K-12 MG1655 ORFs»</I>, Yisheng Kang, Tim Durfee, Jeremy D. Glasner, Yu Qiu, David Frisch, Kelly M. Winterberg, and Frederick R. Blattner, (2004)
</LI><LI><I>«On Spontaneous DNA Damage in Single Living Cells»</I>, Jeanine M. Pennington, Ph.D. thesis, Baylor College of Medicine, Houston (2006):
</LI><LI><I>«A switch from high-fidelity to error-prone DNA double-strand break repair underlies stress-induced mutation»</I>, Rebecca G. Ponder, Natalie C. Fonville, Susan M. Rosenberg, (2005)
</LI><LI><I>«Universal Code Equivalent of a Yeast Mitochondrial lntron Reading Frame Is Expressed into E. coli as a Specific Double Strand Endonuclease»</I>, L. Colleaux, L. d'Auriol, M. Betermier, G. Cottarel, A. Jacquier, F. Galibert†, B. Dujon, (1986)
</LI><LI><I>«Tightly regulated vectors for the cloning and expression of toxic genes»</I>,  Larry C. Anthony*, Hideki Suzuki, Marcin Filutowicz, (2004)
</LI><LI><I>«In Vivo Induction Kinetics of the Arabinose Promoters in Escherichia coli»</I> , Casonya M. Johnson And Robert F. Schleif*, (1995)
</LI><LI><I>«Toxic protein expression in Escherichia coli using a rhamnose-based tightly regulated and tunable promoter system»</I> Matthew J. Giacalone1, Angela M. Gentile2, Brian T. Lovitt2, Neil L. Berkley2,  Carl W. Gunderson1, and Mark W. Surber, (2006)
</LI><LI><I>«DNA-Dependent Renaturation of an insoluble DNA binding Protein. Identification of the RhaS Binding Site at rhaBAD»</I> Susan M.Egan and Robert F. Schleif, (1994)
</LI><LI><I>«Optimization of an E. coli L-rhamnose-inducible expression vector: test of various genetic module combinations»</I> Angelika Wegerer, Tianqi Sun and Josef Altenbuchner, (2008)
</LI></OL><DIV id="footer1">
	Copyright (c) 2012 igem.org. All rights reserved. Design by <A href="http://www.freecsstemplates.org">FCT</A>. 
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme">http://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Restriction_Enzyme" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Restriction_Enzyme" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Restriction_Enzyme&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Restriction_Enzyme&amp;oldid=291736" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>