Team:Westminster/Labnotebook
Team
Team Members
Attributions
Project
Application
Description
Experiments
Results
Biocontainment
Safety
Parts
Team Parts
Notebook
Lab Notebook
Outreach
Collaborations
iGEM UK MeetUp
Human Practices
Lab Notebook
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24 /31
25
26
27
28
29
30
September
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
Go Back To Top
08/06/15
LB AGAR
7g of agar powder
200ml of dH2O
Broth in 200ml flask and 10ml UB
Go Back To Top
09/06/15
TRANSFORMATION
Transforming double terminator and promoter biobricks into TOP 10 E. coli cells
Double terminator- BBa_B0015- 2013/Plate 3/4F chloramp
TetR repressible promoter- BBa_R0040- 2013/Plate5/6I chloramp
#32 and 20- Double Terminator- BBa_B0015
#29 and 12- TetR promoter- BBa_R0040
#1-Double promoter
#13-Promoter
#21- control
Inoculated 6 plates. 2 with double terminator (1 plate @ 20 µl and the other 230 µl), 2 with promoter (1 plate @ 20 µl and the other 230 µl) and 2 with control (1p plate @ 20 µl and the other 230 µl).
Also inoculated 10 ml LB broth with T6 and T7 promoters for overnight growth.
-Are the parts compliant with iGEM standards
-Smallest parts/largest parts
-What are the ends (Z/X)
ILLUSTRATION
forward primer
0.5 µl
1 µl
reverse primer
0.5 µl
1 µl
template DNA
1 µl
2 µl
master mix
23 µl
46 µl
TOTAL VOLUME
25 µl
50 µl
PCR
95oC for 3 mins
95oC for 30 secs- 30 cycles
55oC for 40 secs- 30 cycles
72oC for 1 min- 30 cycles
72oC for 10 mins
(1%) Gel electrophoresis 100U for 40 mins
Well no.
Material in well
1
2µl of DNA ladder
2
PCR A (1µl of dye and 5 µl of product)
3
PCR B (1µl of dye and 5 µl of product)
Go Back To Top
17/06/15
Making competent cells
Spectrophotometer set at OD 600nm
0.131 after 40 mins incubation need 0.3-0.4
0.238 after1 hr and 10mins
0.340 after 1 hr and 30 mins incubation
10ml overnight growth- 200ml LB broth
Go Back To Top
18/06/15
No growth
Go Back To Top
22/06/15
Make antibiotics
Make plates- melt agar
Transformation using control plasmid (check dilution of control plasmid)
Plate competent cells and check tomorrow
Go Back To Top
23/06/15
No growth
Go Back To Top
24/06/15
*KEEP STERILE
Once gets to 0.6 put flask on ice for 30 mins
Aliquot 50ml 4x50ml chilled falcon tubes
Centrifuge @ 3500rpm, @4 oC for 7 mins
-resuspend cells in 12.5ml 0.1M MgCl2
-centrifuge @ 3500rpm, @4 oC for 7 mins, discard supernatant
-resuspend each tube in 25ml of 0.1M CaCl2
-incubate for 30 mins then centrifuge
-resuspend each tube in 700 µl 0.1M CaCl2 and 300 µl of 50% glycerol to give final volume of 1ml in each tube.
-aliquot 50 µl into 1.5 ml sterile micro centrifuge tubes on ice.
-Store @ -80 oC
OD 600
0.125 after 1 hr- give an extra 45 mins
Rpm increased from 210 to 250
Want OD to get to 0.600
0.315 at 1 hr and 45mins
0.456 at 2 hrs and 15 mins
0.538 at 2 hours and 45 mins
Go Back To Top
25/06/15
Shewanella oneidensis
→
MtrD/ MtrE/ MtrF/ OmcA/ MtrC/ MtrA/ MtrB
966/ 2139/ 1920/ 2208/ 2016/ 1022/ 2094
-Illegal (5-pst1, 1-Spe1)
-Size of each fragment
Biomass and electrical potential
#8 (old) High str BBa_J23100—(125 µl on LB and 125 µl on AMP)
#2 (old) low str BBa_J23103—(125 µl on LB and 125 µl on AMP)
#16 (old) control—(250 µl on AMP)
#84 (new) High str BBa_J23100—(20 µl on LB and 230 µl on AMP)
BBa_J23103- 2013 plate 5 18I
BBa_J23100- 2013 plate 5 18C
Go Back To Top
01/07/15
Isolating high and low strength constitutive promoter from 2013 DNA plates.
BBa_J23100- High strength- 2013 plate 5 18C
BBa_J23103- Low strength- 2013 plate 5 18I
To test competent cells we will be using XL1-Blue competent cells (known to work) with an old batch (made in 2014) of TOP 10 and new batch (made 6/15) of TOP 10.
MtrB 56172-58265
MtrA 58278-59279
MtrC 59348-61363
OmcA 161693-63900
ILLUSTRATION
-TOP 10 competent cells recovered by iGEM
-To make TOP 10 recommended to use CCMB80 TEKNOVA
-Ask Mark bioinformatics work- where do we start?
- Before OmcA? End? Whole pathway?
-NEB Biolabs iGEM offer
-We want:
– HiFi DNA Assembly master mix to assemble gBlocks OR
– BioBrick assembly kit
Ampillicin (sodium salt) = M.W 371.40
1g in 10ml- 100mg/ µl
1g in 20ml- 50mg/ µl
For RDP illegal sites= BsaI and NotI
MtrC- PstI illegal site CTGCAG
Therefore mutate GCT (codes for alanine) to GCG (Ala) in order to remove illegal site
Go Back To Top
02/07/15
gBlocks ordered- MtrA, MtrB, MtrC, OmcA, CymA
MtrA- 1002bp= 1 gBlock
MtrB- 2094bp in size therefore split into 3 gBlocks as gBlocks only come in 1kb size.
MtrB 1=1-850 to allow for Gibson assembly approximately 40bp overhang
MtrB 2=810-1660
MtrB 3=1620- 2094 (494bp in size)
MtrC- 2016bp in size = 3gBlocks
MtrC 1=1-750 to allow for Gibson assembly approximately 40bp overhang
MtrC 2=710-1460
MtrC 3=1420- 2016 (597bp in size)
OmcA- 2208bp in size
OmcA 1=1-750 to allow for Gibson assembly approximately 40bp overhang
OmcA 2=710-1460
OmcA 3=1420- 2208 (789bp in size)
CymA=approximately 500bp therefore 1gBlock
Go Back To Top
07/07/15
Plasmid backbone
pSBIC3
pSBIA3
pSBIT3
Transformation using pUC19 as control plasmid using old (5) and new (111) competent cells
pUC19 10pg/µl
#5- old (50 µl) and (1 µl of pUC19)
#111- new (50 µl) and (1 µl of pUC19)
Will be adding 50 µl of transformed cells to ampicillin plates (100µg/ µl conc.) and storing excess cells in 4 oC
Go Back To Top
21/07/15
BIOBRICKS
PART#
DESCRIPTION
LOCATION
RESISTANCE
BBa_R0010
LacI regulated inducible promoter
P5-W4G
Chloramphenicol
BBa_J23100
High strength promoter
P4-W17D
Ampicillin
BBa_ J23101
Medium strength promoter
P4-W17F
Ampicillin
BBa_ J23103
Low strength promoter
P4-W17J
Ampicillin
BBa_B0034
Strong RBS (Elowitz)
P4-W1N
Ampicillin
BBa_B0030
Strong RBS(Weiss)
P4-W4G
Chloramphenicol
BBa_B0015
Double terminator
P4-W3F
Chloramphenicol
Go Back To Top
24/07/15
Double terminator #BBa_B0015- tube 1 (plate 3)
Strong RBS (Weiss) #BBa_B0030- tube 2 (plate 4)
LacI regulated inducible promoter #BBa_R0010- tube 3 (plate 5)
Tube 1- 230 µl- growth
-20 µl- growth
Tube 2- 230 µl- growth
-20 µl- no growth
Tube 3- 230 µl- growth
-20 µl- no growth
Added 10 µl (34mg/ml) chloramphenicol to 10ml LB broth for final concentration of 34µg/µl
Inoculated BBa_B0015, BBa_B0030 and BBa_R0010 and grew overnight (~16-17 hrs) at 37 oC
Go Back To Top
30/07/15
Plasmid prepM
Centrifuging UB bottles at 12000rpm for 10 mins to sediment cells
BBa_R0010- 155.6 ng/µl
BBa_B0030- 91.3 ng/µl
BBa_B0015- 57.0 ng/µl
Go Back To Top
07/08/15
Reagents received
-NEBuilder HiFi DNA assembly
-gBlock (NCR)
-Primers
-gBlock (mtrC, A, B, OmcA, CymA)
-Competent cells
-gBlock (ChiA, B, C)- from 2013 Westminster project in order to improve upon the parts as part of the Gold medal criteria
-PCR (high F) master mix
-DNA ladder
-Synbiota parts (His, Flg, Deg)
HiFi Assembly reaction
Joining MtrC-1 and MtrC-2
MtrB-1 and MtrB-2
OmcA-1 and OmcA-2suspending
Resuspending gBlocks
1. Centrifuge tube for 3-5 sec @ 3000xg
2. Add TE (for 24 months storage) or water (for 1 month storage) for final conc of 10ng/µl
3. Vortex
4. Incubate at 50 oC for 20 mins
5. Vortex and centrifuge
MtrC1 and MtrC2= 1000ng: Add 100 µl of TE
MtrB1 and MtrB2= 1000ng: Add 100 µl of TE
OmcA1 and OmcA2= 1000ng: Add 100 µl of TE
CymA= 500ng: Add 50 µl of TE
MtrA= 1000ng: Add 100 µl of TE
NEBuilder protocol
Need 0.03-0.2 pmols for 2-3 fragments
0.2-0.5 pmols for 4-6 fragments
50 µg of 1000bp dsDNAis about 0.3 pmols
Therefore 10 µg is around 0.06 pmols
MtrB
-2 µl of MtrB1 (0.12 pmols)
-2 µl of MtrB2 (0.12 pmols)
-10 µl master mix
-8 µl of dH20
TOTAL volume= 22 µl (this may be too much)
MtrC and OmcA
-2 µl of MtrC1/OmcA1
-2 µl of MtrC2/OmcA2
-10 µl master mix
-6 µl of dH20
TOTAL volume= 20 µl
Resuspending primers
Master Stock (100µM)
-25µmols primer + 250µl dH20
-10µmols primer + 1000µl dH20
Working Stock
1:9 or 10:90 = 10 µl of primer (Master): 90 µl of dH20
Go Back To Top
11/08/15
ILLUSTRATION
1 µl of template DNA (gBlocks)
0.5 µl of forward primer
0.5 µl of reverse primer
12.5 µl of Master mix
10.5 µl of dH20
Total volume= 25 µl
1- MtrA BioBrick
2- MtrB BioBrick x
3- MtrC BioBrick x
4- OmcA BioBrick
5- CymA BioBrick x
6- MtrA RDP
7- MtrB RDP
8- MtrC RDP
9- OmcA RDP
10- CymA RDP
Go Back To Top
12/08/15
Gel electrophoresis
Running two gels to check PCR results from yesterday/ First gel has BioBrick samples (1-5), and second gel has RDP samples (6-10)
Used 5 µl of PCR product (from when and which experiment) and 1 µl of purple 6x Loading dye. And 6 µl for DNA ladder.
Gel 1
DNA ladder
Sample 1
Sample 2
Sample 3
Sample 4
Sample 5
DNA Ladder
Empty
Gel 2
DNA ladder
Sample 6
Sample 7
Sample 8
Sample 9
Sample 10
DNA Ladder
Empty
All wells were loaded with 6 µl.
Sample 2, 3 and 10 did not work
PCR (round 2)
Due to these three PCR products not working we will be repeating the experiment
1- MtrB BioBrick – worked
2- MtrC BioBrick - worked
3- OmcA RDP- not worked x
Adding:
1 µl of template DNA
0.5 µl of forward primer
0.5 µl of reverse primer
12.5 µl of Master mix
10.5 µl of dH20
Go Back To Top
13/08/15
Ran gel with 3 samples that previously did not work. MtrB and C worked and are the correct size. However OmcA did not work, this may be due to gBlock assembly not working.
Lane 1- DNA ladder
Lane 2- MtrB
Lane 3- MtrC
Lane 4- OmcA
1- OmcA BioBrick (PCR see above)
2-OmcA RDP
Running another gel (100V for 40 mins)
Lane 1- DNA ladder
Lane 2- OmcA BioBrick
Lane 3- OmcA RDP
Go Back To Top
14/08/15
RDP construct
Example of MtrCAB
[Anchor- Pr-RBS-MtrC-His-ter-Pr-RBS- MtrA-His-ter- Pr- RBS-MtrB- His- Cap]
Used this method in order to produce constructs for each of our genes of interest- Mtr C, MtrA, MtrB, CymA and OmcA. Below is an illustration of how this will occur, in theory, with MtrA.
RESULTS
1. MtrA BioBrick- 15.1 ng/ µl
2. MtrB BioBrick- 8.9 ng/ µl
3. MtrC BioBrick- 9.61 ng/ µl
4. CymA BioBrick- 10.5ng/ µl
5. OmcA BioBrick- 6.3 ng/ µl
Go Back To Top
18/08/15
PCR Cleanup
Have 20µl of PCR product left from the PCR reaction (5µl used in gel electrophoresis). Protocol states to add 4 units of B2 to 1 unit of DNA: 20 µl of PCR product + 80 µl of B2 binding buffer.
1. MtrA (6)
2. MtrB (7)
3. MtrC (8)
4. CymA (9)
5. OmcA (10)
Go Back To Top
19/08/15
DNA Cleanup
Followed Synbiota protocol on how to clean DNA after Bsa1 digestion
Results were recorded using the Nanodrop.
MtrA- 11.6 ng/ µl = 0.02 pm/µl
MtrB- 3.7 ng/ µl = 0.012 pm/µl
MtrC- 5.5 ng/ µl = 0.018 pm/µl
CymA- 1.5 ng/ µl = 0.0013 pm/µl
OmcA- 5.7 ng/ µl = 0.018 pm/µl
Go Back To Top
20/08/15
Gel electrophoresis
Checking the RDP digestion from yesterday using gel electrophoresis. 1 µl of DNA to 9µl of loading dye (LM) so that the total volume in each well = 10 µl
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
Lane 7
Lane 8
DNA ladder (10 µl)
MtrA(1µl) LM(9µl)
MtrB(1µl) LM(9µl)
MtrC(1µl) LM(9µl)
CymA(1µl) LM(9µl)
OmcA(1µl) LM(9µl)
DNA ladder (10 µl)
Empty
PCR
Followed the Synbiota protocol
1. MtrA
2. MtrB
3. MtrC
4. CymA
5. OmcA
Go Back To Top
24/08/15
Running gel on PCR product (no cleanup)
Lane:
1- Synbiota ladder
2- MtrA (1µl) + LM (9µl)
3- MtrB (1µl) + LM (9µl)
4- MtrC (1µl) + LM (9µl)
5- CymA (1µl) + LM (9µl)
6- OmcA (1µl) + LM (9µl)
7- Empty
8- Empty
Next: PCR cleanup to remove contaminants
RESULTS
MtrA- 35.3 µg/ µl
MtrB- 18.3 µg/ µl
MtrC- 15.1 µg/ µl
OmcA- 26.9 µg/ µl
CymA- 21.8 µg/ µl
Go Back To Top
25/08/15
RESULTS
Nanodrop
MtrA- 35.6 ng/ µl (1.3 kb)
MtrB- 24.3 ng/ µl (2.2 kb)
MtrC- 11.0 ng/ µl (2.3 kb)
OmcA- 24.6 ng/ µl (2.3 kb)
CymA- 26.3 ng/ µl (0.6 kb)
1 pm/µl = 670ng/µl /length of part (kb)
Range needed: pm/µl – 0.02- 0.04
MtrA = 670/1.3kb = 515.4
35.6/515.4 = 0.069 pm/µl
MtrB = 670/2.2kb = 304.5
24.3/304.5 = 0.079 pm/µl
MtrC = 670/2.3kb = 291.3
11.0/291.3 = 0.037 pm/µl
OmcA = 670/2.3kb = 291.3
24.6/291.3 = 0.084 pm/µl
CymA = 670/0.6kb = 1116.6
26.3/1116.6 = 0.024 pm/µl (quite low but should still work)
GEL
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
Lane 7
Lane 8
DNA ladder (10 µl)
MtrA(1µl) LM(9µl)
MtrB(1µl) LM(9µl)
MtrC(1µl) LM(9µl)
CymA(1µl) LM(9µl)
OmcA(1µl) LM(9µl)
Empty
Empty
Go Back To Top
26/08/15
RDP CONSTRUCTS
MtrA
Blk= blocker in order to ensure that no other part can anneal to that specific part that it is anneal to.
MtrB
[Anchor-X-CPr-Z-RBS-X-MtrB-Z-His tag-X-Cap]
MtrC
[Anchor-X-CPr-Z-RBS-X-MtrC-Z-His tag-X-Cap]x2
OmcA
[Anchor-X-CPr-Z-RBS-X-OmcA-Z-His tag-X-Cap]
CymA
[Anchor-X-CPr-Z-RBS-X-CymA-Z-His tag-X-Cap]
Entry level RDP exercise
The Adp is added as the MtrA and GFP are not complimentary (MtrA ends in Z and GFP starts with X). Adp is an extra part that allows for complementarity of the parts for construct annealing.
RDP CONSTRUCTS
His-tag MtrA
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3-X   X-MtrA-Z   Z- His- X  X-Ori.2 dT20
GFP test construct
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3-X   X-MtrA-Z   Z-Adp-X  X-aGFP-Z  Z-Ori.2 dT20
Go Back To Top
27/08/15
RDP constructs
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3-X   X-MtrA-Z   Z-pLkr.Xa-X  X- His10- Z  Z-Ori.3 dT18
This construct with MtrA, MtrB, MtrC, CymA and OmcA
Linker pLkr is a cleavage protein linker containing Factor Xa which is useful for recovering cleavage.
T4 DNA ligase calculation
Need 0.2 Weiss unit/µl in 1xligase buffer
(1 Weiss unit=~70 CE Units)
70x0.2=14 CE units/µl
In 1 construct will need 35 µl of ligase (7 parts) therefore in 5 constructs will need 175µl of ligase
175x 14 CE units= 2450 CE units (total volume)
T4 DNA ligase = 2,000,000 U/ml- 2,000 U/µl
2450U/2000U= 1.225 µl
Add 1.3 µl of T4 DNA ligase to 173.7 µl of DNA ligation buffer 1x
Go Back To Top
01/09/15
Gel electrophoresis
The gel run last week had negative results. This was probably due to too little DNA added to the gel. So another gel was run adding 10 µl of DNA rather than 1 µl.
10 µl of construct DNA + 5 µl of loading dye
15 µl of ladder
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
Lane 7
Lane 8
DNA ladder (15 µl)
MtrA(10µl) LM(5µl)
MtrB(10µl) LM(5µl)
MtrC (10µl) LM(5µl)
CymA(10µl) LM(5µl)
OmcA (10µl) LM(5µl)
Empty
Empty
New construct:
The constructs made last week used the old RBS.3 which forms a hidden XbaI site. Therefore we are making a new construct using MtrC. Synbiota recommended not using the blockers after each step.
Anchor- Pr.2- RBS.3.1- MtrC- linker- His tag- Cap
T4 DNA ligase
T4 DNA ligase= 400,000 CE Units/µl- 4000U/µl
1 construct with 6 parts (@ 5µl per reaction) = 30µl
Need a final concentration of T4 DNA ligase of 14 U/µl in T4 DNA ligation buffer 1x
30µl x 14 units = 420 units in total
Therefore 420/400= 1.05µl
1.05µl of T4 DNA ligase to 28.95µl of 1x buffer
Turning 10x buffer to 1x
0.5 µl of buffer and 49.5 µl to get a final volume of 50 µl
Then we used 28.95 µl in making the T4 DNA ligase master mix.
TRANSFORMATION
No cell growth on the following plates:
9- MtrA
47-MtrC
68- CymA
Cell growth on the following TOP10 competent cells (used as a positive control):
1. MtrA- 8 colonies
2. MtrC- 100’s of colonies
3. CymA- ~ 50 colonies
Restriction digest of plasmid and part
Enzyme Master mix (for 5 runs - 30 µl) (for plasmid)
5 µl NEB Buffer 2 10x
0.5 µl BSA 10x
0.5 µl PstI
0.5 µl EcoRI
0.5 µl DpnI
23 µl dH2O
Add 4 µl linearized plasmid backbone
Add 6 µl of Enzyme master mix
Digest 370C/30mins- heat kill 800C/20mins
Master mix for part (total 30 µl)
5 µl NEB Buffer 2 10x
0.5 µl BSA 10x
0.5 µl PstI
0.5 µl EcoRI
23.5 µl dH2O
PCR tube
1-5= pSB1SC plasmid 4 µl
6-MtrA
7-MtrB
8-MtrC
9- CymA
10-OmcA
RDP plates- resuspending parts for RDP.
Magnetic holders attracting bead pellet whilst clearing the supernatant.
Tom preparing an RDP construct
Building RDP construct- MtrCAB
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X X-MtrC-Z  Z-RBS.3.1-X  X-MtrA-Z  Z-RBS.3.1-X  X- MtrB- Z  Z-Ori.3 dT18
After each step wash twice.
Ligation
-2.5 µl digested plasmid
-1 µl T4 DNA ligase buffer
- 0.5 µl T4 DNA ligas
e
-6 µl of BioBricks 16 oC/30mins, heat kill 80 oC/20mins
Go Back To Top
03/09/15
1. Plasmid prep
-Nanodrop (test DNA)
-Gel
2. Build RDP constructs (MtrA, MtrB, CymA and OmcA)
3. Transform mew RDP constructs and MtrCAB
RESULTS
Plasmid Prep Analysis 03/09/15
ng/ µl
260/280
260/230
MtrA (a)
65.8
1.92
1.97
MtrA (b)
65.7
1.94
1.93
MtrA (c)
50.8
1.90
1.91
MtrC (a)
55.6
1.97
1.98
MtrC (b)
60.3
1.99
1.94
MtrC (c)
43.5
1.93
2.06
CymA (a)
59.8
1.97
1.90
CymA (b)
75.5
1.93
1.90
CymA (c)
77.2
1.96
1.95
Transformations: BioBricks with pSB1C3
1-MtrA
2-MtrB
3- MtrC
4-CymA
5-OmcA
Go Back To Top
05/09/15
Making RDP constructs
1. MtrA
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X   X-MtrA-Z   Z-linker-X  X- His- Z  Z-Cap
2. MtrB
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X   X-MtrB-Z   Z-linker-X  X- His- Z  Z-Cap
3. CymA
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X   X-CymA-Z   Z-linker-X  X- His- Z  Z-Cap
4. OmcA
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X   X-OmcA-Z   Z-linker-X  X- His- Z  Z-Cap
5. MtrCAB
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X X-MtrC-Z  Z-RBS.3.1-X  X-MtrA-Z  Z-RBS.3.1-X  X- MtrB- Z  Z-Ori.3 dT18
6. OmcA-CymA construct
A18-ChlrR-X   X-Pr2-Z   Z-RBS.3.1-X X-OmcA-Z  Z-RBS.3.1-X  X-CymA-Z  Z-Ori.3 dT18
25 parts x 5 µl= 125 µl
4.4 µl of T4
1:9
121 µl of buffer
10:90
20:180
Go Back To Top
07/09/15
1. Run a gel and check with nanodrop the PCR product of BioBrick parts
1.1 If they are correct length and high enough concentration, need to digest and ligate to pSB1C3
1.2 Transform plasmid into TOP 10 competent cells
2. Check constructs made on 05/09/15 using gel to see if they are correct length.
Nanodrop (ng/µl )(checking concentration of PCR product)
MtrA- 138.4
MtrB- 122.3
MtrC- 124.5
OmcA-128.6
CymA-94.6
Gel electrophoresis (100V for 40mins). Checking BioBricks are correct size.
Lane:
1- DNA ladder 10 µl
2. MtrA BioBrick (9 µl LM and 1 µl DNA)
3. MtrB BioBrick (9 µl LM and 1 µl DNA)
4. MtrC BioBrick (9 µl LM and 1 µl DNA)
5. OmcA BioBrick (9 µl LM and 1 µl DNA)
6. CymA BioBrick (9 µl LM and 1 µl DNA)
Followed restriction enzyme protocol
-250ng DNA (2 µl)
Ligation protocol (1:3)
2 µl plasmid backbone
6 µl digested fragment
1 µl T4 ligation buffer
0.5 µl T4 ligase
0.5 µl dH2O
OmcA, MtrB and MtrA were streaked on plates.
Go Back To Top
08/09/15
1. Check ligation from yesterday
1.1 Check larger constructs and His-tag
DNA digestion (more DNA than protocol used yesterday)
1 µl DNA
5 µl 10x buffer
1 µl EcoRI
1 µl PstI
42 µl dH2O
Total volume= 50 µl
1.2 Checking RDP constructs- gel electrophoresis
Add 10 µl of DNA construct to 5 µl of dye.
Lane:
1. DNA ladder (10 µl)
2. MtrA- His (3.3kb expected size)
3. MtrB-His (4.3kb)
4. MtrC-His (4.3kb)
5. OmcA-His (4.6kb)
6. CymA-His (~3kb)
7. MtrCAB (7.5kb)
8. OmcA-CymA (5.2kb)
2. Transform BioBricks that didn’t work into TOP 10 (MtrC and CymA)
2.1 Transform MtrCAB and CymA-OmcA into BL21 competent cells
Transformation
Transforming MtrCAB and CymA-OmcA into BL21 competent cells
Transforming MtrC and CymA into TOP 10 competent cells
1. MtrCAB
2. CymA-OmcA
3. MtrC
4. CymA
Go Back To Top
09/09/15
PLASMID PREP
Purifying plasmids (protocol followed from Invitrogen)
STEP 1 Harvest- MtrA, MtrB and OmcA incubated overnight @37 oC. Centrifuged for 15mins @ 2820 rpm. Media (supernatant) was poured away from pellet and then centrifuged for a further 5 mins @ 2820rpm. OmcA had the most pellet whereas MtrA had the smallest.
STEP 2 Resuspend - resuspended in original LB bottles and transferred to Eppendorf tubes once resuspended.
STEP 3 Lyse- 250 µl lysis buffer added to each tube incubated for 5 mins @ room temperature (RT)
STEP 4 Precipitate- centrifuge for 10 mins @ 12000 x g. D ue to film on surface tubes were centrifuged for a further 5 mins @ 12000 x g, not before attempting to clear the film from the surface using pipette tips to gently stab the film away from the edge.
STEP 5 Bind- load onto spin column (MtrA, B, OmcA) and centrifuge for 1 min at same speed as step 4
STEP 6 Wash (optional) - incubate for 1 min and centrifuge at the same speed as step 4
STEP 7 Wash and ethanol removal- 700 µl wash buffer centrifuge for 1 min at the same speed as step 4
STEP 8 Elute- incubate for 1 min @ RT- TE buffer added to centre of each column
STEP 9 Recover- centrifuge for 2 mins at the same speed as step 4, discard column. Store plasmid at 4 oC.
Plasmids taken to be measured using nanodrop. Blanked with TE buffer. See results.
Double digest of plasmid DNA
After transforming and growing TOP 10 cells with inserted recombinant plasmids (MtrA, MtrB, OmcA) BioBricks we isolated a high yield of DNA. To test this plasmid DNA has the correct insert we will be digesting it with EcoRI and SpeI. Expected results: should see top band around 3.3kb (size of plasmid, and inserted BioBricks which range from 1- 2.6kb)
MtrA (1)
MtrB (2)
OmcA (3)
DNA 1 µl -x µl
7 µl
10 µl
5 µl
10x NEBuffer 5 µl
EcoRI 1 µl
PstI 1 µl
dH2O -x µl
36 µl
33 µl
38 µl
RDP- BioBrick (PCR- 2hr)
Digestion and ligation (3-6 hrs)
Transformation (1 ½ days)(3hrs)
Plasmid Prep (1 ½ days)
PCR reaction
2µl Fwd (Anchor X-end)
2µl Reverse (Cap Z-end)
23µl PCR Master mix
2 µl DNA (MtrA-His, MtrB-His, MtrC-His, OmcA-His, CymA-His, MtrCAB, OmcA-CymA)
1µl dH2O
1. MtrA- His
2. MtrB-His
3. MtrC-His
4. OmcA-His
5. CymA-His
6. MtrCAB
7. OmcA-CymA
Results:
MtrA- 260/280= 1.86 260/230= 2.03
DNA present in TE buffer= 173.8ng/µl
MtrB- 260/280= 1.86 260/230= 1.90
DNA present in TE buffer= 87.0ng/µl
OmcA- 260/280= 1.87 260/230= 1.97
DNA present in TE buffer= 209.1ng/µl
Go Back To Top
10/09/15
1. Plasmid prep on MtrC
2. Run a gel on PCR products (RDP constructs)
2.1 If positive result, digest/ligate into pSB1C3
The results from transformation were positive in TOP 10 MtrC and CymA, but negative in the BL21 cells. This could be due to the proteins being toxic to the cells.
Gel-checking PCR products of RDP constructs
Lane:
1. DNA ladder (10 µl)
2. MtrA- His
3. MtrB-His
4. CymA-His
5. MtrC-His
6. OmcA-His
7. MtrCAB
8. OmcA-CymA
Lane numbers 1, 2 and 4 worked
BioBricks
MtrA worked.
Inoculate MtrB, C, CymA, OmcA for plasmid prep tomorrow.
Digest and ligate MtrA-His, B-His, CymA-His and transform.
RDP constructs-MtrC-His, OmcA-His, MtrCAB, OmcA-CymA
1. MtrA-His BioBrick (add date on side)
2. MtrB-His BioBrick
3. CymA-His BioBrick
Nanodrop results of these three BioBricks
MtrA=104.1ng/µl
MtrB=102.2ng/µl
CymA=100.8ng/µl
Digestion protocol
MtrA (1)
MtrB (2)
CymA (3)
DNA (1 µl)
10 µl
10 µl
10 µl
10x NEBuffer (2.1)
5 µl
5 µl
5 µl
EcoRI
1 µl
1 µl
1 µl
PstI
1 µl
1 µl
1 µl
dH2O
33 µl
33 µl
33 µl
Total volume for digestion=50 µl
Ligation protocol
MtrA (1)
MtrB (2)
CymA (3)
plasmid backbone (25ng)
2 µl
2 µl
2 µl
digested fragment
6 µl
6 µl
6 µl
T4 ligation buffer
1 µl
1 µl
1 µl
T4 ligase
0.5 µl
0.5 µl
0.5 µl
dH2O
0.5 µl
0.5 µl
0.5 µl
Total volume for ligation= 20 µl
Go Back To Top
11/09/15
Plasmid prep
1. Harvest
6 LB bottles incubated overnight with BL21 cells and:
Bottle no. 1. OmcA pSB1C5 (1)
Bottle no. 2. OmcA pSB1C5 (2)
Bottle no. 3. CymA pSB1C5 (1)
Bottle no. 4. CymA pSB1C5 (2)
Bottle no. 5. MtrC pSB1C5 (1)
Bottle no. 6. MtrC pSB1C5 (2)
All 6 LB bottles centrifuged for 15 mins @ 2880rpm. And then respun for a further 5 mins @ 2880 rpm. Supernatant (media) discarded.
OmcA=most pellet
CymA= smallest pellet
2. Resuspend- Pellet resuspended in LB bottles and then transferred to labelled eppendorf tubes.
3. Lyse- invert cap 5 times and incubated for 5 mins.
4. Precipitate- due to scum on surface of supernatant tubes were centrifuged for an extra 5 mins. Scum was swirled prior to centrifugation in order to clear from the edge using pipette tip.
5. Bind- some scum was still present on the surface. Carefully collected the supernatant to prevent collection of scum.
6. Wash (optional)- incubate column for 1 min
7. Wash and Ethanol Removal 2x centrifuge and 2x discard of flow through
8. Elute
9. Recover
Plasmids taken to be measured using nanodrop. Blanked with TE buffer. See results.
Run a gel in order to plasmid and insert
Lane:
1. DNA ladder
2. MtrC (1)
3. MtrC (2)
4. OmcA (1)
5. OmcA (2)
6. CymA (1)
7. CymA (2)
All the lanes appeared to demonstrate the sizes expected for each plasmid and insert.
Transformation of MtrA-His, MtrB-His, OmcA-His
Transformed into TOP 10 cells
1. MtrA-His
2. MtrB-His
3. OmcA-His
Measure 110µl GBX- split in half (55 µl)
Make half up to 236 µl
12.5 µl working solution
=6.25 µl detection reagent 1 + 6.25 µl detection reagent 2
Nanodrop results from plasmid prep
OmcA (1) - 260/280= 1.87  260/230=2.10
TOTAL= 114.5 ng/µl
OmcA (2) - 260/280= 1.88  260/230=2.13
TOTAL= 77.9 ng/µl
CymA (1) - 260/280= 1.86  260/230=2.13
TOTAL= 183.5ng/µl
CymA (2) - 260/280= 1.88  260/230=2.06
TOTAL= 107.1ng/µl
MtrC (1) - 260/280= 1.88  260/230=2.15
TOTAL= 166.7ng/µl
MtrC (2) - 260/280= 1.89  260/230=2.18
TOTAL= 91.1 ng/µl
• Electrodes made up of carbon cloth were prepared into specific surface area (each having 25cm2 dimension) were soldered to connecting wires.
Go Back To Top
12/09/15
Yesterdays transformed cells had mixed results. OmcA-His grew 20-30 colonies whilst MtrA-His and MtrB-His had no growth.
Transformation
1. MtrC-His (TOP 10)
2. OmcA-His (TOP 10)
3. MtrCAB (BL21)
4. OmcA-CymA (BL21)
RDP constructs
1. MtrC-His construct
2. OmcA-His construct
3. OmcA-CymA construct
4. MtrCAB
28 parts x 5 µl= 140 µl
140x14=1960
1960/400=4.9 µl (round up to 5 µl)
135 µl of 1x buffer + 5 µl of T4 ligase
Go Back To Top
13/09/15
Transformation
1. Transform MtrCAB and OmcA-CymA into TOP 10 cells.
1.1 Re-streak BL21 transformed cells.
2. Plasmid prep on OmcA-His
Transform
1. MtrCAB
2. OmcA-CymA
Go Back To Top
14/09/15
Double digest
MtrA- 173.8 ng/µl
MtrB- 87 ng/µl
MtrC- 124.0, 166.7, 91.0 ng/µl
OmcA- 209.0, 114.5, 77.9 ng/µl
CymA- 183.5, 107.1 ng/µl
OmcA-His- 90.1 ng/µl
MtrB-His- 217.9 ng/µl
CymA-His- 234.5 ng/µl
MtrA-His- 150.8 ng/µl
MtrC-His- 46.5 ng/µl
Digestion protocol
MtrA
MtrB
MtrC (1)
MtrC (2)
MtrC (3)
DNA (1 µl)
6 µl
12 µl
8 µl
6 µl
11 µl
10x NEBuffer (2.1)
5 µl
5 µl
5 µl
5 µl
5 µl
EcoRI
1 µl
1 µl
1 µl
1 µl
1 µl
PstI
1 µl
1 µl
1 µl
1 µl
1 µl
dH2O
37 µl
31 µl
33 µl
37 µl
32 µl
Digestion protocol
OmcA (1)
OmcA (2)
OmcA (3)
CymA (1)
CymA (2)
DNA (1 µl)
5 µl
9 µl
13 µl
6 µl
10 µl
10x NEBuffer (2.1)
5 µl
5 µl
5 µl
5 µl
5 µl
EcoRI
1 µl
1 µl
1 µl
1 µl
1 µl
PstI
1 µl
1 µl
1 µl
1 µl
1 µl
dH2O
38 µl
34 µl
30 µl
37 µl
33 µl
Digestion protocol
OmcA-His
MtrB-His
CymA-His
MtrA-His
MtrC-His
DNA (1 µl)
11 µl
5 µl
5 µl
7 µl
21 µl
10x NEBuffer (2.1)
5 µl
5 µl
5 µl
5 µl
5 µl
EcoRI
1 µl
1 µl
1 µl
1 µl
1 µl
PstI
1 µl
1 µl
1 µl
1 µl
1 µl
dH2O
32 µl
38 µl
38 µl
36 µl
22 µl
• Preparation of Minimal salts medium based on 50mM phosphate buffer comprising:
0.46g/L - NH¬4Cl
0.23g/L- (NH4)2 SO¬4
7.8g/L – Na2HPO4.7H2O
0.12g/L – MgSO4.7H2O
2.87g/L – NaH2PO4 • Preparation of X100 Vitamin Stock Solution
50mg/L  - p-aminbenzoic acid (PABA)
100mg/L – L-Ascorbic acid
50mg/L – Folic acid
10mg/L – Riboflavin
100mg/L- Nicotinic acid
100mg/L – Partothenic acid
10mg/L- Thiamine hydrochloride
100mg/L – Biotin
• Preparation of x100 Mineral Stock Solution
1.5g/L – Nitriloacetic acid
0.1g/L – MnCl2.4H2O
0.3g/L – FeSO4.7H2O
0.12g/L – CoCl2.6H2O
0.04g/L – CuSO4.5H2O
0.1g/L – ZnCl2
0.05g/L – AlK(SO4)2.12H2O
0.09g/L – NaMoO4
0.12g/L – NiCl2
0.02 - NaWO4.2H2O
0.1 – Na2SeO4
Go Back To Top
15/09/15
Plasmid prep
1. Harvest OmcA and CymA (1)
OmcA and CymA (2)
Centrifuge for 15mins + another 5 mins after most of the medium had been discarded.
Step 4 (precipitate) film formed on the surface and was punctured and swirled using a pipette tip to remove from the edge. Then the tube was centrifuged again for a further 5 mins to encourage the scum to the pellet.
Step 6 optional wash was undertaken.
Nanodrop results from plasmid prep
Blank with TE buffer
OmcA/CymA (1) - 260/280= 1.94    260/230=2.25
Total= 344.7ng/µl
OmcA/CymA (2) - 260/280= 1.94    260/230=2.25
Total= 340.0ng/µl
Inoculating 3 LB bottles with 10µl ampicillin (concentration ??) and MtrCAB RDP TOP 10. MtrCAB was incubated and grown overnight (14/09/15) and one colony from the five present on the amp plates was used in order to inoculate the LB bottles. Another colony was taken in order to streak another ampicillin plate (Amp Res) plate, inoculated 12/09/15. The plate was placed in 37 oC incubator. The Amp Res plate was used as it was observed that MtrCAB was not growing initially on chloramphenicol plates. It was discovered that the anchor used was not infat chloramphenicol as originally thought. It was ampicillin resistance. Therefore as of 14/09/15 MtrCAB was grown on amp plate with what appears to be positive results- 5 colonies. The colonies have yet to be tested further for positive confirmation.
PCR
PCR was used to turn MtrB, MtrC, OmcA and CymA into BioBricks
2µl forward primer (working stock)
2µl reverse primer (working stock)
23µl Master mix
71µl dH2O
2µl gBlock
• Preparation of sodium pyruvate (2.2g/L)
• Preparation of Casein hydrolysate (500mg/L)
• Preparation of Potassium ferricyanide (0.1M)
• MFCs systems and setups:
Internal chambers diameter was 5cm
Internal chamber length was 5cm
MFCs dimension was 12.1cm X 10cm X 8cm
Hence, internal volume (v) was 22/7 X (5/2)2X 5= 98.22cm3
➢ MFCs setups
Two identical H-type anode and cathode chamber was connected together by placing O-rings into the grooves to make the system water tight and thereafter Nafion™ 117 cation membrane (7cm X 7cm) was placed between the two connecting chambers and held together by external metal clip.
• Sterilization of assembled MFCs systems, anolyte solution and with other materials needed for the experiment.
• Culturing of genetically modified Escherichia coli (Top 10) in serum bottles for 24 hours at 37oC. Each bottle contained modified individual E. coli of mtrA alone, mtrC alone, mtrCAB alone, two positive controls (444 gene and 408 gene, supplied by Caroline Ajo-Franklin).
Go Back To Top
16/09/15
Results from the PCR products (Mtr B, MtrC, OmcA, CymA and MtrCAB) that were run overnight:
Tube 1 260/280=1.83 260/230=0.86
Total= 321.2ng/µl
Tube 2 260/280=1.88 260/230=0.76
Total= 190.4ng/µl
Tube 3 260/280=1.81 260/230=0.76
Total= 180.9ng/µl
Tube 4 260/280=1.83 260/230=0.78
Total= 179.6ng/µl
Tube 5 260/280=1.82 260/230=0.73
Total= 181.9ng/µl
Tube 6 260/280=1.82 260/230=0.76
Total= 193.4ng/µl
Tube 7 260/280=1.87 260/230=0.71
Total= 175.8ng/µl
Tube 8 260/280=1.83 260/230=0.71
Total= 169.4ng/µl
MtrCAB 260/280=1.83 260/230=0.74
Total= 186.8ng/µl
In order to calculate the quantity of DNA required for our digest experiment the following calculation was used:
250ng/concentration of DNA from the nanodrop results
Therefore for
Tube 1=250/321.2= 0.78 µl
Tube 2=250/190.4= 1.31 µl
Tube 3=250/180.9= 1.38 µl
Tube 4=250/179.6= 1.39 µl
Tube 5=250/181.9= 1.37 µl
Tube 6= 250/193.4= 1.29 µl
Tube 7= 250/175.8= 1.42 µl
Tube 8= 250/169.4= 1.48 µl
MtrCAB= 250/186.8= 1.34 µl
Below are the overall volumes of each reagent. The total volume was 20µl.
PCR product
Tube 1
Tube 2
Tube 3
Tube 4
Tube 5
Tube 6
Tube 7
Tube 8
MtrCAB
DNA (250ng)
0.8 µl
1.3 µl
1.4 µl
1.4 µl
1.4 µl
1.3 µl
1.4 µl
1.5 µl
1.3 µl
NEBuffer 2.1
2.5 µl
2.5 µl
2.5 µl
2.5 µl
2.5 µl
2.5 µl
2.5 µl
2.5 µl
2.5 µl
BSA
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
EcoRI
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
PstI
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
0.5 µl
dH2O
15.2µl
14.7µl
14.6µl
14.6µl
14.6µl
14.7µl
14.6µl
14.5µl
14.7 µl
Nanodrop results of the digested PCR product
Tube 1 260/280=1.26 260/230=0.35
Total= 86.1ng/µl
Tube 2 260/280=1.21 260/230=0.26
Total= 40.4ng/µl
Tube 3 260/280=1.15 260/230=0.30
Total= 43.1ng/µl
Tube 4 260/280=1.32 260/230=0.34
Total= 74.8ng/µl
Tube 5 260/280=1.31 260/230=0.37
Total= 87.4ng/µl
Tube 6 260/280=1.31 260/230=0.30
Total= 69.6ng/µl
Tube 7 260/280=1.21 260/230=0.34
Total= 51.1ng/µl
Tube 8 260/280=0.93 260/230=0.19
Total= 35.0ng/µl
MtrCAB 260/280=1.31 260/230=0.34
Total= 80.8ng/µl
These measurements are required for the next stage of the experiment- ligation. The aim is to ligate each part into plasmid pSB1C3 (tube 2,4,5,7 and MtrCAB) or pSB1A3 (tube 1, 3, 6 and 8).
The protocol used for the ligation was taken from iGEM. Total volume = 20 µl.
• Experimental setup
Briefly: H-type MFCs were constructed with two identical Duran bottles and were held together with an external metal clip. The anode and cathode compartments were separated with a cation-exchange membrane (CMI-7000, membranes International USA). Two rubber gaskets were used to ensure a seal. The electrodes were constructed from carbon cloth.  The anolyte used was MSM supplemented with 500mgL-1 casein hydrolysate and 2.2gL-1 sodium pyruvate as the primary carbon source and the catholyte used was 50mM (pH 7) phosphate buffer containing 0.1M potassium ferricyanide, without aeration. During start-up operation, actively growing E. coli Top 10 (10% v/v of the total anolyte volume). The anolyte was purged with nitrogen gas for 10 minutes through a 0.22 µm pore size diameter filter prior to inoculation. The experiments were conducted in batch mode with a working volume of 200mL in each MFC compartment.
Go Back To Top
17/09/15 RESULTS:
2 Gels of PCR products extracted yesterday morning (16/09/15) (tube 1, 2, 3, 4, 5, 6, 7, 8, MtrCAB). Gel was run for 40 mins at 100V.
GEL 1
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
Lane 7
Lane 8
DNA ladder (10 µl)
MtrCAB (2 µl + 8 µl)
Tube 1 (2 µl + 8 µl)
Tube 2 (2 µl + 8 µl)
Tube 3 (2 µl + 8 µl)
Tube 4 (2 µl + 8 µl)
DNA ladder (10 µl)
Empty
Both ladders and MtrCAB were the only lanes with results.
GEL 2
Lane 1
Lane 2
Lane 3
Lane 4
Lane 5
Lane 6
Lane 7
Lane 8
DNA ladder (10 µl)
Tube 5 (2 µl + 8 µl)
Tube 6 (2 µl + 8 µl)
Tube 7 (2 µl + 8 µl)
Tube 8 (2 µl + 8 µl)
DNA ladder (10 µl)
Empty
Empty
Lane 3 was the only lane without results.
Plasmid prep
9 LB bottles were inoculated with amplicillin (100µg/ml) or chlorophenicol (34.5µg/ml) and plasmid pSB1C3 or pSB1A3 from yesterday’s transformation and left to incubate overnight (pSB1C3 =tube 2, 4, 5, 7 and MtrCAB; pSB1A3= tube 1, 3, 6, 8).
Steps
Tube 1,3,6,8 (Amp)
Tube 2,5,7 (all Chlr) MtrCAB,
Tube 4 Chlr
1. Harvest
Yes 15mins + extra 5 mins
Yes 15mins + extra 5 mins
Yes 15mins + extra 5 mins
2. Resuspend
yes
yes
yes
3. Lyse
yes
yes
yes
4. Precipitate
yes
yes
yes
5. Bind
yes
yes
yes
6. Wash (optional)
yes
yes
yes
7. Wash and Ethanol Removal
yes
yes
yes
8. Elute
yes
yes
yes
9. Recover
yes
yes
yes
Tube 3 (Amp) - resuspended with 340 µl rather than 250 µl
Tube 7 (Chlr) - 360 µl resuspension buffer added rather than 250 µl
Nanodrop results of plasmid prep
Amp 1- 260/280= 1.98 260/230=2.12
TOTAL= 83.0ng/µl
Chlr 2- 260/280= 1.89 260/230=2.11
TOTAL= 151.7ng/µl
Amp 3- 260/280= 1.91 260/230=2.12
TOTAL= 173.5ng/µl
Chlr 4- 260/280= 1.89 260/230=2.18
TOTAL= 92.1ng/µl
Chlr 5- 260/280= 1.93 260/230=2.23
TOTAL= 87.8ng/µl
Amp 6- 260/280= 1.92 260/230=2.15
TOTAL= 129.5ng/µl
Chlr 7- 260/280= 1.91 260/230=2.21
TOTAL= 80.4ng/µl
Chlr 8- 260/280= 1.94 260/230=2.11
TOTAL= 128.4ng/µl
MtrCAB- 260/280= 1.96 260/230=2.28
TOTAL= 38.3ng/µl
Below is the plate that the plasmid DNA that will be freeze dried
1
2
3
4
A
1
9
17
25
B
2
10
18
26
C
3
11
19
27
D
4
12
20
E
5
13
21
F
6
14
22
G
7
15
23
H
8
16
24
1. MtrA pSB1C3
2. MtrB pSB1C3
3. MtrC pSB1C3
4. MtrC (1)
5. MtrC (2)
6. CymA (1)
7. CymA (2)
8. OmcA
9. OmcA (1)
10. OmcA (2)
11. MtrA-His
12. MtrB-His
13. MtrC-His
14. CymA-His
15. OmcA-His
16- OmcA-CymA(1)
17. OmcA-CymA (2)
18. MtrCAB
19. Chlr 2
20. Chlr 4
21. Chlr 5
22. Chlr 7
• Analytical Procedure
➢ Electrochemical test (polarisation test, power density test, voltage and current)
Polarisation curves for measuring power density vs current density plots were constructed using a range of external resistance ranging from 10Ω to 1 MΩ. The closed external circuit of the MFC system for each test were opened to connect various external resistances when the system exhibited a stable voltage across the initial 1000Ω external resistor. The current flowing through each external load was calculated using  Ohm’s law.
I   =   E/R
where E is the potential across the resistor (mV), I is the current flowing through the load (mA) and R is the external resistance (Ω).
The power generated was calculated with the following expression (Fernando et al., 2012)
P = E*I
where P is the power produced (µW), E is the potential difference between anode and cathode (mV) and I is the current generated (mA).
The power density and current density values were calculated by normalising power and current values to the projected surface area of the anodic electrode (25cm2).
Coulombic efficiency (CE) was calculated by integrating the measured current over time based on the observed COD removal by using the criteria outlined in Zhao, et al. 2009. CE is a measure of the amount of electrons generated via substrate oxidation that are reflected as current.
