<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Westminster_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Westminster/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Team"><SPAN>Team Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Westminster/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Application"><SPAN>Application</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Description"><SPAN>Description</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Biocontainment"><SPAN>Biocontainment</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Westminster/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Parts</SPAN></A></LI><LI class=""><A href="https://2015.igem.org/Team:Westminster/Parts"><SPAN>Team Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Westminster/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Westminster/Labnotebook"><SPAN>Lab Notebook</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Westminster/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Westminster/iGEMUKMeetUp"><SPAN>iGEM UK MeetUp</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Westminster/Practices"><SPAN>Human Practices</SPAN></A></LI></DIV><DIV id="maincontainer" class="protocols"><H3>Experiments &amp; Protocols</H3><H3 id="top">Content:</H3><P class="content"><A href="#P1">1.Purification Procedure Using Centrifugation Introduction (PureLink Quick Plasmid DNA Miniprep Kits)</A><A href="#P2">2.Resuspension of gBlock Gene Fragments  (IDT)</A><A href="#P3">3.Ampicillin</A><A href="#P4">4.Chloramphenicol</A><A href="#P5">5.Gel electrophoresis 1%/2% Agarose gel preparation</A><A href="#P6">6.Preparing competent cells</A><A href="#P7">7.Transformation</A><A href="#P8">8. Digestion</A><A href="#P9">9. Ligation</A><A href="#P10">10.Making LB Agar plates </A><A href="#P11">11.Preparing competent cells – Top 10 stock</A><A href="#P12">12.PCR Protocols</A><A href="#P13">13.PCR MasterMix</A><A href="#P14">14.RDP Assembly Protocol</A><A href="#P15">15.PCR-Based RDP Parts Creation Protocol Bsa1 digestion protocol</A><A href="#P16">16.PCR-Based RDP Parts Creation Protocol PCR Protocol</A></P><A href="#top" class="top">Go Back To Top</A><H5 id="P1">1. Purification Procedure Using Centrifugation Introduction (PureLink Quick Plasmid DNA Miniprep Kits)</H5><P>
Follow this procedure to purify plasmid DNA using a centrifuge. Use a microcentrifuge capable of centrifuging at &gt;12,000 × g. <B>Isolating Miniprep Plasmid DNA</B></P><OL><LI>Harvest. Sediment the cells by centrifuging 1-5ml of overnight LB – culture. Remove all medium.
</LI><LI>Resuspend. Add 250ml of resuspension buffer with RNase A to the cell pellet and resuspend the pellet until it is homogeneous.
</LI><LI>Lyse. Add 250ml of Lysis Buffer. Mix gently by inverting the capped tube five times. Incubate the tube at room temperature for 5 minutes.
</LI><LI>Precipitate. Add 350ml of Precipitation Buffer. Mix immediately by inverting the tube, or for large pellets, vigorously shaking the tube, until the mixture is homogenous. Centrifuge the lysate at &gt;12,000 x g for 10 minutes.
</LI><LI>Bind. Load the supernatant from step 4 onto a Spin Column in a 2ml Wash Tube. Centrifuge at the column at 12,000 x g for 1 minute. Discard the flow-through and place column back into the Wash Tube.
</LI><LI>Wash and Ethanol Removal. Add 700ml of Wash Buffer with ethanol to the column. Centrifuge the column at 12.00 x g for 1 minute. Discard the flow through and place the column into the Wash Tube. Centrifuge the column at 12,000 x g for 1 minute. Discard the Wash Tube with the flow – through.
</LI><LI>Elute. Place the Spin Column in a clean 1.5-mL recovery tube. Add 75 µL of preheated TE Buffer (TE) to the centre of the column. Incubate the column for 1 minute at room temperature. 
</LI><LI>Recover. Centrifuge the column at 12,000 × g for 2 minutes. <I>The recovery tube contains the purified plasmid DNA. </I>Discard the column. Store plasmid DNA at 4°C (short-term) or store the DNA in aliquots at −20°C (long-term). 
</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P2">2. Resuspension of gBlock Gene Fragments  (IDT)</H5><OL><LI>Centrifuge the tube for 3-5 sec at a minimum of 3000 x g to ensure the material is in the bottom of the tube.
</LI><LI>Add TE to reach a final concentration of 10ng/ µL (100 µL of TE)
</LI><LI>Vortex briefly
</LI><LI>Incubate at 50°C for 20 minutes
</LI><LI>Briefly vortex and centrifuge.
</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P3">3. Ampicillin</H5><P><B>Stocks &amp; Usage</B></P><UL><LI>Stock Concentration - 50mg/ml in H<SUB>2</SUB>O
</LI><LI>Aliquots - 100µl and 500µl
</LI><LI>Working Concentration - 50µg/ml<B>Preparation of 80ml stock solution</B></LI><LI>Ampicillin is kept in the 4C fridge in 68-564D. It is light sensitive.</LI><LI>Weigh 4g of ampicillin into a small weigh boat.
</LI><LI>Add 80ml of milliQ to a 250ml bottle.
</LI><LI>Add the ampicillin to the milliQ
</LI><LI>Mix/vortex so all the ampicillin goes into solution.
</LI><LI>Filter sterilise the solution into a falcon tube using a 20ml syringe and a 200nm filter.
</LI><LI>Aliquot into pcr tubes and 1.7ml eppendorfs.
</LI><LI>Store at -20C.
</LI><LI>Store the small aliquots in the small box and the big aliquots in the larger box.
</LI></UL><A href="#top" class="top">Go Back To Top</A><H5 id="P4">4. 
Chloramphenicol</H5><UL><LI>Stock concentration - 34mg/ml in 100% Ethanol
</LI><LI>Aliquots - 1ml
</LI><LI>Working concentration = 25µ/ml (Stringent), 170µ/ml (relaxed)<B>Preparation of 80ml stock solution</B></LI><LI>Chloramphenicol is kept at room temperature. It is near our other chemicals
</LI><LI>Weight 2.72g of chloramphenicol sulfate into a small weight boat.
</LI><LI>Add 80ml 100% EtOH to the chloramphenicol
</LI><LI>95% EtOH probably works just fine, but I haven't tried it.
</LI><LI>Mix/vortex vigorously so all the chloramphenicol goes into solution.
</LI><LI>Aliquot into 1.7ml eppendorfs.
</LI><LI>N.B. There is no need to filter sterilize, as it is in EtOH.
</LI><LI>Store at -20C.
</LI></UL><A href="#top" class="top">Go Back To Top</A><H5 id="P5">
5. Gel electrophoresis 1%/2% Agarose gel preparation</H5><P><B>Materials</B></P><UL><LI>0.5g of agarose weighed and added to 50ml of TAE buffer
</LI><LI> Add 60ml ml of the 1X TAE + 1.2g agarose = 2% gel. Add 60ml 1X TAE + 0.6 agarose = 1% gel.

Dissolve the agarose in the TAE buffer in the microwave. Ensure all the agarose is completely dissolved. Allow to cool slightly (approx. 60°C) and add 1µl of ethidium bromide. Pour gel into gel tray and place comb into position. Ensure there are no bubbles in the gel. Bubbles may be moved to the bottom of the gel using a sterile tip.  Allow gel to set completely. Run gel for approx. 1hour at 100volts.
</LI></UL><OL><LI>Measure out the appropriate mass of agarose into a beaker with the appropriate volume of buffer (see below). The volume required depends on your gelbox / casting system -- 50mL makes a good, thick gel for a 7x10cm gelbox.
</LI><LI>Microwave until the agarose is fully melted. This depends strongly on your microwave, but a 90 seconds at full power or 3 minutes at half power seem to provide decent results. As long as you do not burn the agarose and nothing bubbles over, this step is robust.
</LI><LI>Let the agarose cool on your bench until touching the bottom of the beaker with your bare hand doesn't burn you (~5 minutes for a 50mL gel). At this point add your DNA stain, e.g., ethidium bromide. The beaker will cool unevenly (surface first), so you must be careful not to cause ripples and bubbles.
</LI><LI>While the solution is cooling, seal the open edges of your gel box with one long piece of masking tape on each side. Make sure it is sealed well or the gel will leak.
</LI><LI>Pour the agarose solution into the taped gelbox. Carefuly pop or shove to the side any bubbles, put in the comb, and let it cool for about 30 minutes, until the gel is solid.

Notes:
TAE  - buffer
SYBR green – loading dye</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P6">6. 

Preparing competent cells</H5><UL><LI>Inoculate 250 ml of SOB medium with 1 ml vial of seed stock and grow at 20°C to an OD600nm of 0.3
</LI><LI>This takes approximately 16 hours.
</LI><LI>Controlling the temperature makes this a more reproducible process, but is not essential.
</LI><LI>Room temperature will work. You can adjust this temperature somewhat to fit your schedule
</LI><LI>Aim for lower, not higher OD if you can't hit this mark
</LI><LI>Centrifuge at 3000rpm at 4°C for 10 minutes in a flat bottom centrifuge bottle.
</LI><LI>Flat bottom centrifuge tubes make the fragile cells much easier to resuspend
</LI><LI>It is often easier to resuspend pellets by mixing before adding large amounts of buffer
</LI><LI>Discard supernatant by pouring out slowly and pipeting remaining supernatent
</LI><LI>Gently resuspend in 80 ml of ice cold CCMB80 buffer
</LI><LI>sometimes this is less than completely gentle. It still works.
</LI><LI>Incubate on ice 20 minutes
</LI><LI>Centrifuge again at 4°C and discard supernatant as described above.
</LI><LI>Resuspend in 10 ml of ice cold CCMB80 buffer.
</LI><LI>Test OD of a mixture of 200 µl SOC and 50 µl of the resuspended cells.
</LI><LI>Add chilled CCMB80 to yield a final OD of 1.0-1.5 in this test.
</LI><LI>Aliquot to chilled screw top 2 ml vials or 50 µl into chilled microtiter plates
</LI><LI>Store at -80°C indefinitely.
</LI><LI>Flash freezing does not appear to be necessary
</LI><LI>Test competence (see below)
</LI><LI>Thawing and refreezing partially used cell aliquots dramatically reduces transformation efficiency by about 3x the first time, and about 6x total after several freeze/thaw cycles.
<B>Measurement of competence</B></LI><LI>Transform 50 µl of cells with 1 µl of standard pUC19 plasmid (Invitrogen)
</LI><LI>This is at 10 pg/µl or 10-5 µg/µl
</LI><LI>This can be made by diluting 1 µl of NEB pUC19 plasmid (1 µg/µl, NEB part number N3401S) into 100 ml of TE
</LI><LI>Hold on ice 0.5 hours
</LI><LI>Heat shock 60 sec at 42C
</LI><LI>Add 250 µl SOC
</LI><LI>Incubate at 37 C for 1 hour in 2 ml centrifuge tubes rotated
</LI><LI>using 2ml centrifuge tubes for transformation and regrowth works well because the small volumes flow well when rotated, increasing aeration.
</LI><LI>For our plasmids (pSB1AC3, pSB1AT3) which are chloramphenicol and tetracycline resistant, we find growing for 2 hours yields many more colonies
</LI><LI>Ampicillin and kanamycin appear to do fine with 1 hour growth
</LI><LI>Plate 20 µl on AMP plates using sterile 3.5 mm glass beads
</LI><LI>Good cells should yield around 100 - 400 colonies
</LI><LI>Transformation efficiency is (dilution factor=15) x colony count x 105/µgDNA
</LI><LI>We expect that the transformation efficiency should be between 5x108 and 5x109 cfu/µgDNA
</LI></UL><A href="#top" class="top">Go Back To Top</A><H5 id="P7">7. 
Transformation</H5><P><B>Materials</B></P><UL><LI> competent cells (1 vial per transformation and 1 for the control)
</LI><LI>antibiotic plates (2 per transformation)
</LI><LI>control plate without antibiotic
</LI><LI>control plasmid (PUC19) 
</LI></UL><OL><LI>Remove plates from fridge and allow to warm and dry in 37°C incubator. 
</LI><LI>Re-suspend plasmid DNA from iGEM plate by: (1) Pierce identified location on plate. (2) Add 10µl of sterile water and pipette up and down a few times. (3) Allow to stand for 5 mins. 
</LI><LI>Remove competent cells from -80°C and leave on ice for 5 mins.  
</LI><LI>Add 1µl of part DNA to competent cells and mix very gently. 
</LI><LI>Incubate preparation on ice for 5-20 mins. 
</LI><LI>Heat shock at 42°C in a heat block (or water bath) for 45 seconds and immediately transfer to ice. 
</LI><LI>Incubate on ice for 5 mins - for chloramphenicol parts, incubate on ice for 20 mins. 
</LI><LI>Add 200µl of SOC or sterile LB broth and incubate at 37°C in a shaking incubator for at least 1 hour. 
</LI><LI>Plate out 20µl and 230µl volumes to each LB antibiotic plate and spread using sterile beads. 
</LI><LI>Discard used beads into used beads waste, (beads are later cleaned and autoclaved). 
</LI><LI>Plate out control cells onto non-antibiotic plate to check viability of competent cells. 
</LI><LI>Incubate overnight at 37°C and check for individual colonies
</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P8">8. Digestion</H5><OL><LI>	Add 250ng of DNA to be digested, and adjust with dH20 for a total volume of 16ul.
</LI><LI>	Add 2.5ul of NEBuffer 2.
</LI><LI>	Add 0.5ul of BSA.
</LI><LI>	Add 0.5ul of EcoRI.
</LI><LI>	Add 0.5ul of PstI.
</LI><LI>	There should be a total volume of 20ul. Mix well and spin down briefly.
</LI><LI>	Incubate the restriction digest at 37C for 1 hour, and then 80C for 20min to heat kill the enzymes.
</LI><LI>	Run a portion of the digest on a gel (8ul, 100ng), to check that both plasmid backbone and part length are accurate.

</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P9">9. 
Ligation </H5><OL><LI>	Add 2ul of digested plasmid backbone (25 ng)
</LI><LI>	Add equimolar amount of EcoRI-HF SpeI digested fragment (&lt; 3 ul)
</LI><LI>	Add equimolar amount of XbaI PstI digested fragment (&lt; 3 ul)
</LI><LI>	1:3 - Digested backbone:DNA insert
</LI><LI>	Add 1 ul T4 DNA ligase buffer. Note: Do not use quick ligase
</LI><LI>	Add 0.5 ul T4 DNA ligase
</LI><LI>	Add water to 10 ul
</LI><LI>	Ligate RT/30 min
</LI><LI>	Transform with 1-2 ul of product

</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P10">10. 
Making LB Agar plates </H5><OL><LI>Weigh 7.0g of LB agar.
</LI><LI>Add 200ml of deionised water into a flask containing the weighed LB agar.
</LI><LI>Mix well to dissolve the LB agar powder.
</LI><LI>Autoclave the flask (remember to cover the flask with foil and sponge)
</LI><LI>After autoclaving, store it in a clean dry space.<B>
Re-heating agar to prepare plates.</B></LI></OL><OL><LI>Heat the agar in microwave till the LB agar has fully liquidised.
</LI><LI>Let it cool.
</LI><LI> Add required antibiotic in a sterile environment and pour the LB on plates.
</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P11">11. 
Preparing competent cells –Top10 stock </H5><OL><LI>Pick single colony of cells from LB agar plate into 10 ml of LB media containing specific antibiotic for the cell type. Grow the culture overnight at 37 ˚C with shaking at 250rpm.
</LI><LI>Inoculate 200ml of pre-warmed medium (no antibiotic or specific for the cell type) with 10ml of overnight cultures , and grow at 37 ˚C for 60 min, with vigorous shaking at 250rpm or until OD600 is 0.4-0.5
</LI><LI>Put the flask on ice for 30min. At the same time chill sterile falcon tubes.
</LI><LI>Aliquot culture into 50ml each 4 x 50ml chilled falcon tubes.
</LI><LI>Harvest the cells by centrifugation for 7 minutes at 3500rpm, at 4 ˚C and discard the supernatant.
</LI><LI>Re-suspend cells in each tube in 12.5ml of 0.1M MgCl2 
</LI><LI>Centrifuge for 7 min at 3500rpm, at 4˚C and discard supernatant.
</LI><LI>Re-suspend cells in each tube in 25ml of 0.1M CaCl2
</LI><LI>Incubate cells on ice for 30 min.
</LI><LI>Centrifuge for 7 min rpm, at 4˚C and discard supernatant.
</LI><LI>Re-suspend cells in each tube in 700µl of 0.1M CaCl2 and 300µl of 50% glycerol to give a final volume of 1ml in each tube.
</LI><LI>Aliquot 50µl into 1.5ml sterile micro centrifuge tubes on ice and store at -80˚C.

</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P12">12.
PCR Protocols </H5><P><B>
Mutagenisis of the Chitinase genes</B>

In two separate tubes make up 50µl PCR reaction with DNA (plasmid prep of chiA), MgCl<SUB>2</SUB> and dNTP and Pfu polymerase. 

Tube A –Fwd-Main and Rev-Muta

Tube B --  Fwd-Muta and Rev-Main


Thermal cycling conditions
95°C for 3 minutes followed by 
30 cycles of 
95°C for 30 seconds, 55 °C for 40 seconds, and 72 °C for
60 seconds (or 1min/kb), and a final step at 72 °C for 10 minutes.
After PCR – can run of gel to clean up – but may not be necessary. Set up a second PCR without any primers, but containing all other PCR components. Add 5µl from each reaction as DNA template and add 15µl of PCR mix (total volume of PCR =25µl. 

Run PCR cycle as above for 10 cycles to generate mutated template. 
After the 10 cycles –add a further 25µl of PCR mix – but this mix will contain the outer (flanking) primers to allow for amplification of the entire mutated sequence. 

Run PCR as before for 30 cycles.

Do restriction digest – then run on gel and then gel purify.

Ligate to pSB1C3  backbone and transform. 

Pick colonies and colony PCR and inoculate LB broth. 
Select cloned colonies (from colony PCR) and grow those samples overnight in shaker. 

Mini-prep and analyse by restriction analysis and send for sequencing 

</P><A href="#top" class="top">Go Back To Top</A><H5 id="P13">13. 
PCR MasterMix</H5><P>
Make a master mix   X6 (6 reactions in total)</P><OL><LI>5x Q5 reaction 30µl
</LI><LI>10mM dNTPs 3µl
</LI><LI>DNA pol  1.5µl
</LI><LI>dH2O 94.2µL.
</LI></OL><P>
For each reaction add:- 
</P><UL><LI> 21.5µl master mix
   </LI><LI> 1µl Template plasmid 
  </LI><LI>  1.25µl Forward primer 
   </LI><LI>1.25µl reverse primer 
</LI></UL><A href="#top" class="top">Go Back To Top</A><H5 id="P14">
14. 
RDP Assembly Protocol </H5><P>
Note: Before beginning, briefly centrifuge all parts and solutions contained in microfuge tubes and mix thoroughly. Tubes that have sat in the fridge for a while will evaporate and condense liquid up at the cap. This step restores the original volume and concentration of the sample. 
Organise the RDP parts</P><OL><LI>Consult the circuit design. Post a copy of your RDP circuit design so you know the right order of assembly.  
</LI><LI>Arrange your RDP parts. Put your DNA tubes in the right order.  
<B>
Bead Washing </B>
The Process of bead washing is illustrated below: Resuspend Bead pull-down pellet Remove Wash 
</LI><LI>Begin the assembly by gently mixing the bead-anchor stock solution by pipetting it slowly up and down (avoiding bubbles) until homogenous.  This method minimizes the splashes of other more vigourous methods that strand droplets up the side of the tube.  </LI><LI>Immediately transfer 5 µL of the mixture to a 1.5 mL microfuge tube.  </LI><LI>Add RDP Wash buffer (WB).  Add 50 µL of RDP Wash buffer and mix as above. Pulse the liquid down briefly in a low speed microfuge.  </LI><LI>Place the tube in a magnet cradle and wait for the solution to clear, by allowing the beads to collect on the tube wall next to the magnet. Wait 20-30 seconds until the solution fully clears. 
 </LI></OL><CENTER><VIDEO width="800" controls=""><SOURCE src="https://static.igem.org/mediawiki/2015/c/cd/Team_Westminster_RDPbeadwash.mp4" type="video/mp4">
Your browser does not support the video tag.
</SOURCE></VIDEO></CENTER><OL start="7"><LI>Remove and discard liquid.  Place the pipette tip at the bottom of the tube at the opposite side from the bead pellet. Remove all liquid, including any droplets on side of tube and discard.  </LI><LI>Perform a second wash by repeating steps 5-7.  
Couple the first part to the anchor part 
</LI><LI>Add first RDP part and ligation master mix.  Add 5 µL of the first part in your design to the bead pellet, then add 5 µL of the ligation master mix.  Mix gently as described above taking care to avoid splashing.  </LI><LI>Incubate for 8 minutes at room temperature.  
Block unreacted ends 
Blockers (Blk-X and Blk-Z) are used to deactivate unreacted ends that can lead to errors at subsequent stages of the assembly. 

</LI><LI> Add 5 µL of the appropriate Blocker to the existing reaction: 
</LI><LI>Blk-X, if you just added an X-Z’ part.  
</LI><LI>Blk-Z, if you just added a Z-X’ part. Note: Adding the wrong Blocker will kill the assembly!  
</LI><LI>Mix gently and incubate an additional 2 minutes. 
Wash 2 times 
</LI><LI>Repeat steps 3 through 8.  </LI><LI>Mark off the coupled part on your circuit diagram.  On your sketch, cross off the part you just coupled so you can keep track of what step you are on.  

Couple the second and all subsequent parts 
</LI><LI> Repeat steps 9 through 14 for each remaining part, finishing with the Cap. 
Elute the DNA from the magnetic bed 
</LI><LI>Resuspend the bead pellet in 20 µL of RDP Elution buffer (EB).  </LI><LI>Place in magnet cradle.  Let stand 5 minutes, and pull down the magnetic bead.  </LI><LI>Extract the assembled DNA.  With a pipette, extract the supernatant containing the assembled DNA.  </LI><LI>Add RDP Storage buffer.  Add 20 µL of storage buffer to neutralize the sample for transformation and gel analysis.  
</LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P14">14. 
PCR-Based RDP Parts Creation Protocol Bsa1 digestion protocol </H5><OL><LI>BsaI Digestion 

This step creates the RDP ends in the RDP orientation that you specified in GENtle in the New RDP Part tool. 
Add 10µL of NEB CutSmart Buffer and 2µL of BsaI-HF (40 units) to the cleaned up DNA (75 µL). Mix and centrifuge briefly to pull down any splashes and eliminate any surface bubbles. Incubate at 37OC for 3 hours 
</LI><LI> DNA Cleanup 
This step inactivates and removes BsaI and the small flanking fragments that are released on cleavage. 
I)  Add 0.5mL of DNA binding buffer (PureLink B2 binding buffer) and vortex.  
II)  Place a Synbiota spin column into a 1.5mL microfuge tube, then load the entire sample and let stand for one minute. Centrifuge for 1 minute at 13K rpm, then discard the flow-through.  
III)  Add 200µL of wash buffer (W1) to the column. Centrifuge for 1 minute at 14K rpm, then discard the flow-through. Repeat the wash step, and discard the flow-through.  
IV)  Place column into a clean microfuge tube. Add 100µL of Elution buffer (Qiagen  EB). Let stand 1 minute, then centrifuge for 1 minute at 14K rpm. Do not discard the column. It can be reused for steps v) and vi). 

V)  Repeat steps i) to iii) by adding 0.5mL of PB to the DNA eluted in step iv).  
vi)  For the final elution step, add 25µL of TE buffer to the column. Let stand 1 minute, then centrifuge for 1 minute. Repeat elution with another 25µL collected into the same tube.  
At this stage your part is almost assembly-ready. The next step is to adjust its concentration appropriately to a value expressed as picomoles per microliter (pm/µL) as described below 
</LI><LI> Measuring and adjusting the part concentration 
In the RDP assembly protocol, PCR parts are used at a concentration of 0.04 pm/µL per reaction, for a total of 0.2 pm in 5µL. Concentrations of 0.02 pm will work but doubling the ligation time is advised. Concentrations greater than 0.04 pm pose no problem and may be advantageous. However adding more means less sample for subsequent builds so why not stick with 0.04 pm/µL? 

I)  If you have access to a Nano Drop or equivalent, first determine the concentration of your part in ng/µL using 2µL of the recovered sample. Zero the instrument using the same TE buffer that was used to elute your sample.  
II)  Determine the remaining volume of your sample.  
III)  To convert the concentration of your part from ng/µL to pm/µL(pm/µL<SUB>Part</SUB>), use the very close approximation: 1 pm/ µL= 670 ng/µL divided by the length of the part (kb).  
IV)  If the concentration significantly exceeds 0.04 pm/µL, then adjust the recovered volume of your sample (µL <SUB>recovered</SUB>) with TE buffer using the formula below:
  Added vol (µL) = [25 x pm/µLPart x µL recovered] - µL recovered  
As an alternative, the gel-based method described below, can be used to approximate a workable concentration. 
v)  Set up an agarose gel as described above, and prepare a gel sample of the digested DNA, add 1µL of the DNA to 9µL of load mix (LM).  
vi)  Load the entire sample into a centrally located well.  
vii)  Load 10µL of the DNA ladder to one side of your sample and 15µL to the other side.  
viii)  Following electrophoresis, photograph the gel at the lowest possible exposure where all bands remain visible but where gel background appears as dark as possible.  
ix)  To determine the minimum concentration of your sample, compare your product band to the nearest sized ladder band in either or both ladder lanes that has an intensity that is closest or lower than yours.  For the 10 µL load, all bands correspond to 20 ng of DNA except for the 500, 1000 and 3000 bp bands, which correspond to 60 ng. For the 15 µL load, all bands are at 30 ng of DNA except for the 500, 1000 and 3000 bp bands, which are at 90 ng.  
x)  The value that you decide on represents the minimum concentration of your sample expressed as µg/µL.  
xi) Use the formula described in step iii) to convert the concentration to pm/ µL. 
xii)  If the minimum concentration of your sample significantly exceeds 0.04 pm/µL. then dilute it as described in section step iv).  
xiii)  Store the remaining RDP Part DNA at ~4oC and remember to centrifuge it briefly before each use. </LI><LI>Part Validation 

To determine how well your part works in RDP assembly, incorporate you part into a simple RDP assembly that begins with an Anchor part and ends with a Cap part. If your part is in the X-Z’ format then start with dA<SUB>20</SUB>-ChlrR-X’ and finish with Z-Ori.3-dT<SUB>20</SUB>. If your part is in the Z-X’ format then start with dA20-ChlrR-Z’and finish with X-Ori.3-dT<SUB>20</SUB>. Follow the protocol described in the RDP Assembly section. 
The plasmid that results from this build is very similar to iGEM’s pSB1C3. Therefore this exercise will not only establish your part’s quality but will also provide with a plasmid where the part can be both sequenced and archived. </LI></OL><A href="#top" class="top">Go Back To Top</A><H5 id="P16">16. 
PCR-Based RDP Parts Creation Protocol PCR Protocol </H5><P><B>PCR Amplification </B></P><OL><LI>Primer Preparation 
Upon receipt of your RDP part primers, briefly centrifuge each tube to bring the solid material to the tube bottom. Resuspend each primer to a final concentration of 25nm/mL (25µM) in TE buffer using the formula: 
<B>Resuspension volume (µL)= nm (from supplier label) x 40 </B>
Vortex at maximum setting for 10 seconds, let stand 10 min then repeat twice. Briefly centrifuge and freeze until needed. 
Note: Keep track of your PCR primers, since you may be able to use some of them to sequence RDP parts in assembled circuits. Store frozen. 
</LI><LI> Template preparation 
If the starting template for the final RDP part is contained within a circular vector such as a plasmid, then linearize it by choosing an appropriate restriction endonuclease whose site is located in the vector backbone but not within the part sequence. PCR amplification from a linear template increases the final yield of the PCR product while reducing the level of lower- band contaminants. 

Since the PCR reaction uses 2µL of template @10ng/µL, digest 100ng of DNA in a final volume of 10µL. Add 2µL of the digest to the PCR reaction just prior to amplification as described below. 
</LI><LI> Reaction setup 
The parts creation protocol requires a minimum of 5µg of PCR product that should be easily obtained from a 100µL reaction as described below: 

Keep all reagents on ice
 Using a thin-walled 0.2 mL PCR tube, combine in the following order: 
2µL forward primer 
2µL reverse primer
 20µL 5x HF buffer  (Master Mix 23 µL (QS High – fidelity Master Mix – NEB))
2µL dNTPs (@10mM)  (Master Mix 23 µL (QS High – fidelity Master Mix – NEB))
71µL H<SUB>2</SUB>O 
2µL template
1µL Phusion polymerase (@2u/µL)  (Master Mix 23 µL (QS High – fidelity Master Mix – NEB))

Mix the reaction by pipetting up and down gently to avoid bubbles. Centrifuge (pulse) briefly. Then keep on ice until amplification. 
</LI><LI> PCR amplification 
Amplify using the cycling program shown below: <TABLE border="1" style="width:100%" id="protocolsTable"><TBODY><TR><TH>Step</TH><TH>Temp</TH><TH>Time</TH></TR><TR><TD>1.</TD><TD>Initial denaturation</TD><TD>98°C</TD><TD>30 sec</TD></TR><TR><TD>2.</TD><TD>Denaturation</TD><TD>98°C</TD><TD>10 sec</TD></TR><TR><TD>3.</TD><TD>Annealing</TD><TD>64°C</TD><TD>45 sec</TD></TR><TR><TD>4.</TD><TD>Extension</TD><TD>72°C</TD><TD> 30 sec/1kb (1.20min) </TD></TR><TR><TD>5.</TD><TD colspan="3"> - - - Repeat steps 2-4 for 25 cycles - - - </TD></TR><TR><TD>6.</TD><TD>Final extension   </TD><TD>72°C</TD><TD>10 min</TD></TR><TR><TD>7.</TD><TD>Hold</TD><TD>4°C</TD></TR><TR><TD colspan="4">
*Determined by multiplying the length of the template (in kilobases) by 30 sec.</TD></TR></TBODY></TABLE></LI><LI> Cleanup 
Use the standard Qiagen spin-column cleanup procedure (or equivalent), noting the procedural modifications described below. Do not use the spin columns provided with the RDP kit. They have a special cleanup role following the BsaI digestion step. 

When the Qiagen PB buffer mix is added to the column, let stand one minute before centrifuging (or filtration if using a vacuum manifold).  
Following the Qiagen PE buffer wash step, elute the DNA from the column into a 1.5mL microfuge tube using 50µL TE buffer, then repeat with a second 50 µL elution into the same collection tube yielding a final volume of ~85-90µL.  
</LI><LI> Determining the quality and quantity of the PCR Product 
For a more accurate measure, use a NanoDrop spectrophotometer (or equivalent small-volume device) if available. Use 2 µL of the recovered sample. Remember to zero the instrument using the same TE buffer that was used to elute your sample. 
If a NanoDrop spectrophotometer is not available, use the procedure below. 
Product size and purity 
i)  For products with an expected size of 0.2-10 kb, set up a 1% agarose gel using a 1:50 dilution of 50x TAE stock buffer (Tris-Acetate-EDTA @ 2M ) in deionized H2O for both the gel and running buffer.  
ii)  To prepare a gel sample of your PCR product, add 1µL of the DNA to 9µL of load mix (LM). Note: In order to minimize the interference of dyes with subsequent band quantification, only bromophenol blue has been included at considerably lower than standard concentrations to mitigate the masking of bands in the 300-600 bp range.  Store remainder of PCR product at 4°C.  
iii)  Selecting two of the most central wells of the gel, load 10µL of the provided DNA markers (ladder) and the full 10µL sample prepared in step ii) above. A gel figure of the DNA ladder, including band sizes and amounts (ng) is provided below.  
iv)  Following electrophoresis, stain the gel for 15 min with ethidium bromide, SYBR Green, or RED Safe, using whichever method is standard to your lab.  
</LI></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>