<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jilin_China_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jilin China/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><DIV class="navcontent"><UL class="navbottom" id="navbottom"><LI><A>Background</A></LI><LI><A href="/Team:Jilin_China/Background">Thermoregulation</A></LI><LI><A href="/Team:Jilin_China/Background2">Thermosensor</A></LI></UL><LI><A>Project</A></LI><LI><A href="/Team:Jilin_China/Project/Overview">Overview</A></LI><LI><A href="/Team:Jilin_China/Design">Design</A></LI><LI><A href="/Team:Jilin_China/Measurement">Measurement</A></LI><LI><A href="/Team:Jilin_China/Construction">Construction</A></LI><LI><A href="/Team:Jilin_China/Improve">Improve</A></LI><LI><A>Achievement</A></LI><LI><A href="/Team:Jilin_China/Result/Version_1">Results</A></LI><LI><A href="/Team:Jilin_China/Result/Version_1">Version 1.0</A></LI><LI><A href="/Team:Jilin_China/Result/Version_2">Version 2.0</A></LI><LI><A href="/Team:Jilin_China/Result/Version_3">Version 3.0</A></LI><LI><A href="/Team:Jilin_China/Part/Search_Engine">Toolkit Explorer</A></LI><LI><A href="/Team:Jilin_China/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Jilin_China/For_Judgement">For Judgement</A></LI><LI><A>Model</A></LI><LI><A href="/Team:Jilin_China/Model">Overview</A></LI><LI><A href="/Team:Jilin_China/Model/Curve_Fitting">Curve Fitting</A></LI><LI><A href="/Team:Jilin_China/Model/Screening_System">Screening System</A></LI><LI><A href="/Team:Jilin_China/Model/For_Practices">Thermol Loss</A></LI><LI><A>Lab</A></LI><LI><A href="/Team:Jilin_China/Notebook">Notebook</A></LI><LI><A href="/Team:Jilin_China/Experiments">Protocol</A></LI><LI><A href="/Team:Jilin_China/InterLab">Interlab</A></LI><LI><A>Parts</A></LI><LI><A href="/Team:Jilin_China/Parts">Overview</A></LI><LI><A href="/Team:Jilin_China/Basic_Part">Basic Part</A></LI><LI><A href="/Team:Jilin_China/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Jilin_China/Part_Collection">Part Collection</A></LI><LI><A>Practices</A></LI><LI><A href="/Team:Jilin_China/Human_Practices">Integrated HP</A></LI><LI><A href="/Team:Jilin_China/Public_Engagement">Engagement</A></LI><LI><A href="/Team:Jilin_China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Jilin_China/Meetup">Meetup</A></LI><LI><A href="/Team:Jilin_China/Collaborations">Collaborations</A></LI><LI><A>Team</A></LI><LI><A href="/Team:Jilin_China/Team">Team</A></LI><LI><A href="/Team:Jilin_China/Attributions">Attributions</A></LI></DIV></DIV><SECTION class="s1"><DIV class="title_con"><P>IMPROVE</P></DIV><DIV class="title_nav"><H2>Improvement</H2></DIV></SECTION><DIV class="bodycontent" style="padding:0;"><SECTION class="s2"><UL><LI class="paragraph_1 start full-width" id="paragraph_1"><DIV><H2>Improvement</H2><P>This year, we choose Golden Gate assembly especially for construction. (Learn about our <B>Construction</B>? See our <A href="/Team:Jilin_China/Construction">Construction Page</A>) BsaI and BbsI are used as Type IIs restriction endonucleases in our construction device. At the same time, we decided to use sfGFP as our reporter protein due to its faster folded feature and higher fluorescence intensity. However, sfGFP in iGEM part registry (<A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A>) has a BbsI recognition site, which could be affected during Golden Gate assembly. Hence, first coming into our mind was to do a site mutagenesis, BbsI recognition site GAAGAC was altered into GAGGAT while amino acids sequence kept the same. Then we got our sfGFP(BbsI free) part(<A href="http://parts.igem.org/Part:BBa_K2541401">BBa_K2541401</A>), a BbsI-free sfGFP and especially for Golden Gate assembly(<B>figure 1.</B>).  </P><P class="figure">Figure 1. Site mutagenesis for sfGFP BbsI free.</P><P>Because we cannot guarantee the mutated sfGFP has the same fluorescence expression as the previous one, after further study, we found a codon optimized sfGFP<SUP>[1,2]</SUP> (<B>sfGFP_optimism</B>, <A href="HTTP://parts.igem.org/Part:BBa_K2541400">BBa_K2541400</A>), which removed the BbsI site and had a functional improvement upon an existing superfolder GFP(BBa_I746916).</P><H3>Nucleic Acid Electrophoresis</H3><P>Free of the BbsI cleavage site was confirmed by nucleic acid electrophoresis(<B>figure 2.</B>). </P><P class="figure">Figure 2. L1: 1kb DNA marker; L2: BBa_I746916; L3: BBa_I746916+BbsI; L4: BBa_K2541401; L5: BBa_K2541401+BbsI; L6: BBa_K2541400; L7: BBa_K2541400+BbsI; L8: 1kb DNA marker.</P><H3>Emission and Excitation Spectra</H3><P>Next, we got three types of sfGFP emission and excitation spectra. We have set the excitation and emission wavelength range from 400nm to 600nm. For these three types of sfGFP, the excitation wavelength is around 495nm and the emission wavelength is around 510nm. (<B>figure 3.</B>)</P><P class="figure">Figure 3. Three types of sfGFP emission and excitation spectra</P><H3>Fluorescence Intensity</H3><P>For further characterization, we detected the expression intensity of these three types of sfGFP. According to the results(<B>figure 4.</B>), we found out that the expression intensity of sfGFP_optimism is nearly 2.6 as much as the other two types. As a result, we finally choose sfGFP_optimism as our reporter protein.</P><P class="figure">Figure 4. Three Types of sfGFP  expression in <I>E.coli</I>.</P></DIV></LI><LI class="pragraph_3" id="pragraph_3"><DIV><H2>Reference</H2></DIV></LI><LI>[1]Segall-Shapiro, Thomas H., Eduardo D. Sontag, and Christopher A. Voigt. &quot;Engineered promoters enable constant gene expression at any copy number in bacteria.&quot; Nature biotechnology 36.4 (2018): 352.</LI><LI>[2]Overkamp, Wout, et al. &quot;Benchmarking various GFP variants in Bacillus subtilis, Streptococcus pneumoniae and Lactococcus lactis for live cell imaging.&quot; Applied and Environmental Microbiology (2013): AEM-02033.</LI></UL></SECTION></DIV></DIV><FOOTER><DIV class="footcontent"><DIV class="school"><DIV class="jlulogo"><DIV><P><A href="http://www.jlu.edu.cn/">Jilin University</A></P><P><A href="http://jwc.jlu.edu.cn/">Office of Undergraduate Education</A></P><P><A href="http://http://oic.jlu.edu.cn/">Office of Global Engagement</A></P></DIV></DIV><DIV class="lifelogo"><DIV><P><A href="http://life.jlu.edu.cn/">School of Life Science</A></P><P><A href="http://nlv.jlu.edu.cn/">National Engineering Laboratory for AIDS Vaccine</A></P><P><A href="http://www.mee.jlu.edu.cn/">Key Laboratory forMolecular Enzymology and Engineering</A></P></DIV></DIV></DIV><DIV class="contacts"><P>CONTACT US</P><P class="mail"><A href="mailto:jilin_china@outlook.com">Jilin_China@Outlook.com</A></P></DIV><DIV class="flogos"><P>ACKNOWLEDGEMENTS</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>