Team:Jilin China/Design
Background
Thermoregulation
Thermosensor
Project
Overview
Design
Measurement
Construction
Improve
Achievement
Results
Version 1.0
Version 2.0
Version 3.0
Toolkit Explorer
Demonstrate
For Judgement
Model
Overview
Curve Fitting
Screening System
Thermol Loss
Lab
Notebook
Protocol
Interlab
Parts
Overview
Basic Part
Composite Part
Part Collection
Practices
Integrated HP
Engagement
Safety
Meetup
Collaborations
Team
Team
Attributions
DESIGN
Toolkit Design
Foreword
Heat-inducible Thermosensors
Heat-repressible Thermosensors
Cold-repressible Thermosensors
Cold-inducible Thermosensors
This year, we aim to design a thermosensor toolkit, whose members could sense different temperatures. After updating and modifying the toolkit for three times, our present toolkit consists of four types of thermosensors, which are heat-inducible, heat-repressible, cold-inducible and cold-repressible RNA-based thermosensors. We name it as SynRT toolkit, Synthetic RNA-based thermosensors. Naturally occurring RNA-based thermomsensors exhibit complicated secondary structures which are demonstrated to undergo a series of gradual structural changes in response to temperature shifts. However, they lack standardization and only respond to temperatures in a narrow range，which impedes application process[1]. Therefore, we focus on more rational and engineering design after understanding the principles of natural RNA-based thermosensors response. Because SD sequence is buried in a stem-loop structure, and it will be melted at elevated temperatures, so the synthetic RNA-based thermosensors mediate the temperature-controlled access of the ribosome to the SD sequence[2].
Figure 1. Base paring is destroyed as temperature rising
Stem-loop intramolecular base pairing is a pattern that can occur in single-stranded DNAs or, more commonly, in RNAs. A base pair (bp) is a unit consisting of two nucleobases bound to each other by hydrogen bonds, such as A-U or G-C.Based on natural RNA-based thermosensors’ stem-loop sequence, we would like to modify the stem-loop region in order to amplify our toolkit with different melting temperatures.  With the increasing of base pairing or hydrogen bonds in the stem-loop structure, the free energy required for conformation transitions would be higher. According to this, we could infer that increasing base pairing or GC content could increase the melting temperature, which means the temperature for unfolded mRNA molecules occupying the half of mRNA molecules. Even so,we need further quantitative parameters of thermosensors before our wet experiment, such as the detailed free energy of conformation transition. Thus, we apply mfold web server (Mfold) to help us to engineer our designs. Mfold describes a number of closely related software applications available on the World Wide Web for the prediction of the secondary structure of single stranded nucleic acids[3]. Secondary structure of RNA is predicted by energy minimization using nearest neighbor energy parameters. After prediction and selection, desired RNA-based thermosensors have been obtained and they have been measured in E.coli.
Figure 2. Schematic representation of our design workflow inculding design, constructs and measurement.
Heat-inducible RNA-based thermosensors
1.What is heat-inducible RNA-based thermosensors?
Natural heat-inducible RNA-based thermosensors form a complex structure in the 5'-untranslated  region of mRNA[4], which modulate the expression of upstream genes. There is evidence showing that heat-inducible RNA-based thermosensors act in an RNA-only manner, without the aid of additional factors[5]. Increasing the temperature gradually shifts the closed conformation to the open structure in a zipper-like manner, thereby increasing the efficiency of translation initiation. At high temperatures, RBS could be accessed to ribosome, and translation proceeds efficiently. At low temperatures, mRNA folds in such a manner so as to prevent ribosome access and inhibit translation[6].
Figure 3.Mechanism of Heat-inducible RNA-based thermosensors.
Nevertheless, natural heat-inducible RNA-based thermosensors have a relatively complicated secondary structure with multiple stems, loops and bulges, which bring difficulty to engineering toolkit. So our project focuses on designing simpler heat-inducible RNA-based thermosensors with only a single stem-loop structure hiding the SD sequence.ev
Figure 4. Difference between natural and synthetic RNA-based thermosensors.
2. How to design it?
The temperature response of these thermosensors was designed on the basis of the melting temperature of the minimum free energy structure. The 5’-UTR consists of anti-SD sequence(ASD sequence), loop sequence, consensus SD sequence (5’-AAGGAG-3’) and 8-nt spacer. Different synthetic 5’-UTR constructs differ in loop size and/or in the extent of complementarity between ASD and SD. To quickly assessed estimate the efficiency of thermosensors in E.coli, We selected a constitutive Anderson promoter J23104 as the promoter and choose superfolder Green Fluorescent Protein (sfGFP) as the reporter gene. To optimize the thermosensors for the melting temperature, three structural parameters have been considered: stem length, loop size and the presence of mismatches or bulges in the stem. The prediction of the secondary structure formation and the calculation of their free energies were performed using the algorithms of the Mfold web server.
Stem length can increase the melting temperature of heat-inducible RNA-based thermosensors, while decreasing stem length has the oppsite effect. Stem length should be in medium length to fit desired temperature range so stem length range from four-base pairing to ten-base pairing.
Loop size can moderate thermosensors melting temperature to suitable temperature. In our design, three kinds of loop are chosen: AAUAA, AAAUAUAAA, AAAAUAUAUAUAAAA .
Furthermore, we change base composition in ASD sequence in order to make mismatch or bulges in stem which can decrease the melting temperature.
Finally, we get a series of heat-inducible RNA-based theromsensors sequence.
Additionally, these theromsensors are predicted by computational methods (Mfold) to identify optimum thermosensors structures and adjust the switch properties to the desired temperature range.
Figure 5. (A) Sensing temperature optimization of heat-inducible RNA-based thermosensors (B) The ASD sequence and loop sequence used in design
Learn more results about the heat-inducible RNA-based thermosensors, click here!
Heat-repressible RNA-based thermosensors
Most naturally-occurring RNA-based thermosensors are heat-inducible[5] and they perform function by sequestering the ribosome binding site (RBS) in a stem-loop structure at low temperatures and exposing the RBS upon stem-loop destabilization at high temperatures. In order to acquire more kinds of engineered thermosensors and broaden our toolkit application, we designed the heat-repressible thermosensors based on RNase E which also follow the zipper-like manner[7].
1. What is RNase E?
RNase E is an endoribonuclease, with a preference for regions of single-stranded RNA[8]. This allows for targeted degradation of RNA in its unfolded form, which occurs at higher temperatures.  What is more, RNase E is native to E.coli, alleviating the need for the expression of a heterologous protein. Finally, both RNase E and its homologue RNase G are common in E.coli and other organisms[9]. So our heat-repressible thermosensors can be applied to a variety of chassis.
Figure 6. (A)RNA Fragments Connect RNase E Adjacent Tetramers within the Crystal Structure and (B)RNase E cleavage pathways.
2. How do heat-repressible RNA-based thermosensors work?
Each heat-repressible RNA-based thermosensor sequence was inserted between downstream of the transcription start site and upstream of the SD sequence. At high temperatures, the RNase E cleavage site (RC) is exposed, thus mRNA was cleaved by RNase E, leading to low expression. At low temperatures, the RC binds to the anti-RNase E cleavage site (ARC) and forms a stable stem-loop structure. This structure sequesters the RC, leading to high expression.
Figure 7. Heat-repressible RNA-based thermosensors mechanism
3. How to design heat-repressible RNA-based thermosensors?
In order to design heat-repressible RNA-based thermosensores with different melting temperatures, intensity and sensitivity, we change the ARC sequence because of the conserved RC sequence. Three structural parameters come into consideration: stem length, loop size and mismatches or bulges in the stem.
Stem length is determined by ARC sequence. Increasing stem length can optimize heat-repressible RNA-based thermosensors to higher melting temperature, while decreasing stem length has the opposite effect. Each heat-repressible RNA-based thermosensors have common RBS. Upstream of the RBS sequence is RC sequence (UCUUCC). We also change ARC sequence to have different stem length. Several types of loop size (AAUAA, AAAUAA, AAAAUAUAAA, AAAAAAUAUAAA, AAAAAUAUAUAUAAAA) are chosen to moderate thermosensors to desired melting temperature. Mismatches or bulges in stem loop can decrease the melting temperature so we also change the base composition in ARC to make mismatches or bulges in ARC.
Figure 8. Sensing temperature optimization of heat-repressible RNA-based thermosensors
Heat-repressible RNA-based thermosensors structures and melting temperature were estimated by the Mfold Web Server. Based on results, we identify optimum thermosensors structures and select desired RNA-based thermosensors.
Learn more design about the heat-repressible RNA-based thermosensor, click here!
Cold-repressible RNA-based thermosensors
In order to extend temperature sensing range of thermosensors, we further design cold-repressible RNA thermosensor which can make response in the lower temperature range. Its design principle also follows the zipper-manner based on RNase III, an enzyme native to E.coli . RNase III cleaves double-strained RNA (dsRNA) specifically[10].
1. What is RNase III?
In bacterial cells, RNase III cleavage of dsRNA is a key step in the maturation and degradation of coding and noncoding RNAs, which can regulate gene expression by controlling mRNA stability and translational efficiency. The processing reactivities of RNase III substrates in E.coli are determined in part by the sequence content of two discrete double-helical elements, termed the distal box (db) and proximal box (pb). The db and pb function as the positive recognition determinant. The pb is also a site of catalytic anti-determinant action. A db+pb set is sufficient to specify a cleavage site[10].
Figure 9．Structure of RNase III and distal box (db) and proximal box (pb) for RNase III cleavage.
2. How do Cold-repressible RNA-based thermosensors work?
Based on it, we utilized a minimal RNase III cleavage site in E.coli as our cold-repressible RNA-based thermosensors. Each thermosensor part was inserted between downstream of the transcription start site and upstream of the RBS. At low temperatures, the stem-loop structure formed, allowing RNase III to cleave the double stranded mRNA, resulting the mRNA degradation, and the gene expression turns 'OFF'. At high temperatures, the stem-loop structure is destabilized. RNase III can’t recognize the single stranded mRNA so the ribosome can access the RBS and gene expression turns 'ON' .
Figure 10．Mechanism for cold-repressible RNA-based theromsensor
3. How to design Cold-repressible RNA-based thermosensors?
We keep the db and pb sequence conserved because it is necessary for RNase III to recognize and cleave mRNA. And we change their adjacent base pairs to increase or decrease the stem length to design cold-repressible RNA-based thermosensors with different melting temperatures, intensity and sensitivity. Moreover, changing adjacent base pairs may also influence RNase III catalytic efficiency.
Considering the RNase III catalytic efficiency, stem length is from 11-base to 13-base. And two different loop sequences are chosen: GAGA, GCAA.
Thermosensors' structures and parameters are also estimated by the Mfold Web Server to identify optimum thermosensors structures and adjust the switch properties to the desired temperature range.
Figure 11. Optimization of the sensing temperature cold-repressible RNA-based thermosensors
Learn more result about the cold-repressible RNA-based thermosensor, click here!
Cold-inducible RNA-based Thermosensors
Besides cold-repressible RNA-based thermosensors，We design a series of cold-inducible RNA-based thermosensors to completely meet users’ need, which design is based on cspA (cold shock protein A) mRNA 5’UTR.
1. What is the cspA ?
In nature, as temperature shifting, E.coli enters the cold-acclimation phase and a set of cold-shock protein (Csp) is transiently expressed. CspA is the representative member in Csp family, which has been quite extensively studied for the mechanism of its cold response[11]. The expression of cspA is regulated by temperature. CspA is hardly detectable at 37 ℃ , but its expression is significantly enhanced after cold shock, when its level reaches 2% of the total proteins[12].
2. How does it response to temperature?
The cspA mRNA is key for cspA temperature sensing, adopting functionally distinct structures at different temperatures, even without the aid of trans-acting factors. Unlike other RNA-based thermosensors, these structural rearrangements do not result from melting of stem-loop structures. cspA mRNA has different conformation at 10℃ and 37℃. The 10℃ conformation of cspA mRNA can make translation proceeds efficiently. Conversely, 37℃ conformation limit translational efficiency[13] .
Let's find the pseudoknot!!
Pseudoknot structure cannot be formed because the potential pseudoknot site are separated.
SD sequence and AUG are buried in a huge and stable stem-loop structure, which can not be access to ribosome.
Pseudoknot structure is formed and various RNase could not catalyze and degrade mRNA.
SD sequence and AUG are exposed, which is beneficial to translation initation.
Figure 12.cspA mRNA 5’UTR has different conformation in different temperature.
The structural rearrangement is due to pseudoknot which is formed by long-distance base pairing [14]. At low temperatures, mRNA contains pseudoknot structure resulting in translating initiation region (TIR) to a conformation that facilitates ribosome recruitment. But at high temperatures, because of its thermodynamic instability, pseudoknot is unfolded so mRNA rearranges conformation which TIR is buried in double-stranded RNA, limiting translation initiation. Furthermore, the 37℃ conformation of cspA mRNA contains various of high sensitive RNase recognize sites, such as RNase V1, RNase V2, which makes mRNA easier to be degraded, leading to further translation inhibition[15]. In consequence, the formation of RNA pseudoknot is vitally crucial to cspA mRNA conformation transformation, and its sensitivity to temperature gives cspA mRNA different translating capacity at different temperatures.
Figure 13. The position of pseudoknot in cspA mRNA
3. How to design?
Based on pseudoknot structure, we design a series of cold-inducible RNA-based thermosensors with different temperature sensing range and different sensitivity. Several structural parameters come into consideration to optimize cold-inducible RNA-based thermosensors: base pairing, base pair position, GC content in pseudoknot region. Increasing base pairing can raise the melting temperature of pseudoknot, thus enlarging the temperature range of open conformation. We also add the same base pair at the different position of pseudoknot region. Increasing GC content can also raise the melting temperature of pseudoknot. What is more, we delete the cold box which is at the upstream of cspA 5’UTR. Cold box is a conservative region where CspA itself interferes with pseudoknot formation to repress CspA expression at the end of the cold acclimation phase.
Figure 14. Sensing temperature optimization of the cold-inducible RNA-based thermomsensors
Learn more results about the cold-inducible RNA-based thermosensor, click here!
References
[1]Neupert, J., Karcher, D., and Bock, R. Design of simple synthetic RNA thermometers for temperature-controlled gene expression in Escherichia coli Nucleic Acids Res.
[2]Narberhaus, F., Waldminghaus, T. & Chowdhury, S. RNA thermometers. FEMS Microbiol. Rev. 30, 3–16 (2006).
[3]Zuker, M. Mfold web server for nucleic acid folding and hybridization prediction Nucleic Acids Res. 31, 3406– 3415(2003)
[4]M V, Jones K, Barillari C, et al. Targeting HSP70: the second potentially druggable heat shock protein and molecular chaperone?[J]. Cell Cycle, 2010, 9(8):1542-1550.
[5]Chowdhury, S., Maris, C., Allain, F. H. & Narberhaus, F.Molecular basis for temperature sensing by an RNA thermometer. EMBO J. 25, 2487–2497 (2006).
[6]Rinnenthal, J., Klinkert, B., Narberhaus, F.& Schwalbe, H. Direct observation of the temperature-induced melting process of the Salmonella fourU RNA thermometer at base-pair resolution. Nucleic Acids Res. 38, 3834–3847
[7]Hoynes-O'Connor A, Hinman K, Kirchner L, et al. De novo design of heat-repressible RNA thermosensors in E.coli[J]. Nucleic Acids Research, 2015, 43(12):6166-6179.
[8]Bouvier,M. and Carpousis,A.J. (2011) A tale of two mRNA degradation pathways mediated by RNase E. Mol. Microbiol., 82,1305–1310.
[9]Cameron,J.C., Gordon,G.C. and Pfleger,B.F. (2015) Genetic and genomic analysis of RNases in model cyanobacteria. Photosynth.Res., doi:10.1007/s11120–11015–10076–11122.
[10]Pertzev A V, Nicholson A W. Characterization of RNA sequence determinants and antideterminants of processing reactivity for a minimal substrate of Escherichia coli ribonuclease III[J]. Nucleic Acids Research, 2006, 34(13):3708-3721.
[11]Eshwar A K, Guldimann C, Oevermann A, et al. Cold-Shock Domain Family Proteins (Csps) Are Involved in Regulation of Virulence, Cellular Aggregation, and Flagella-Based Motility in Listeria monocytogenes[J]. Frontiers in Cellular & Infection Microbiology, 2017, 7.
[12] Yamanaka K, Mitta M, Inouye M. Mutation Analysis of the 5′ Untranslated Region of the Cold Shock cspA mRNA of Escherichia coli[J]. Journal of Bacteriology, 1999, 181(20):6284.
[13] D G, Azar I, Oppenheim A B. Differential mRNA stability of the cspA, gene in the cold-shock response of Escherichia coli[J]. Molecular Microbiology, 1996, 19(2):241.
[14]Giuliodori A M, Pietro F D, Marzi S, et al. The cspA, mRNA Is a Thermosensor that Modulates Translation of the Cold-Shock Protein CspA[J]. Molecular Cell, 2010, 37(1):21-33.
[15] Brierley I, Pennell S, Gilbert R J C. Viral RNA pseudoknots: versatile motifs in gene expression and replication[J]. Nature Reviews Microbiology, 2007, 5(8): 598.
Jilin University
Office of Undergraduate Education
Office of Global Engagement
School of Life Science
National Engineering Laboratory for AIDS Vaccine
Key Laboratory for
Molecular Enzymology and Engineering
CONTACT US
Jilin_China@Outlook.com
ACKNOWLEDGEMENTS
