Team:Jilin China/Background2
Background
Thermoregulation
Thermosensor
Project
Overview
Design
Measurement
Construction
Improve
Achievement
Results
Version 1.0
Version 2.0
Version 3.0
Toolkit Explorer
Demonstrate
For Judgement
Model
Overview
Curve Fitting
Screening System
Thermol Loss
Lab
Notebook
Protocol
Interlab
Parts
Overview
Basic Part
Composite Part
Part Collection
Practices
Integrated HP
Engagement
Safety
Meetup
Collaborations
Team
Team
Attributions
BACKGROUND
Thermosensor
Thermosensors
Principles
Engineering
Toolkit
Toolkits Design
Thermosensors
Principles
Engineering
Toolkit
RNA-based thermosensors mediate more effective feed-forward controls
What’s the feed-forward control? In engineering, feed-forward control describes that disturbances are measured     before they have time to affect the system[1] (figure 1). Concerning intracellular temperature sensing, not only could RNA-based thermosensors respond to physiological temperature in a direct and rapid way before temperature-induced damages occurring, but they also reduce the time lag of transcriptional thermoregulation. Therefore, RNA-based thermoregulation performs in a more immediate and more effective manner[2].
Figure 1．The opening system and feed-foward system (A) The opening system  (B) The feed-forward system．The input is temperature and the output is gene expression. The feed-forward control is based on the RNA-based thermosensors.
How do RNA-based thermosensors work?
The translation rate of mRNAs depends on many biochemical factors. Two evident examples of them are mRNA stability and ribosome-mRNA interaction.
mRNAs are undergoing degradation induced by RNA ribonucleases while being synthesized at the same time. Therefore, the translation rate is determined by the mRNA concentration when it gets equilibrium. Additionally, the translation of a mRNA starts from its translation initiation region (TIR), which contains RBS within Sine-Dalgarno sequence (SD sequence) and start codon AUG. The 30S ribosomal subunit recognize and bind to SD sequence, consequently promoting the translation initiation. Thus the ribosome accessibility also plays an important role in translation process[3].
However, how do cis-acting RNA elements influence mRNA stability and ribosome accessibility? RNA-based thermosensors employ a common but efficient way -- conformational change[4]. The secondary or tertiary structure of mRNA such as stem-loops or pseudoknots could adopt different conformations at different temperatures[5,6]. The mRNA molecules in different conformations have different free energy. Since temperature can be described as the energy availability of mRNA molecules, there is a very different probability to form the same conformation that could facilitate the translation efficiently at the same temperatures (figure 2). There are two mechanisms on conformational change responding to temperature and control translation efficiency[7] (figure 3):
Figure 2. The relationship among temperatures, mRNA conformations and translation efficiency.
1.Zipper-like mechanism
This RNA element is in equilibrium between closed and open conformations. At low temperatures, the closed conformation hides the SD sequence by base-paring with vicinal anti-SD sequence. By contrast, stem-loop melts gradually at elevated temperature, leading the full liberation of the SD sequence and start codon. This conformational change promotes the entire ribosome accessibility of RBS.
Furthermore, the hidden region at low temperatures can also be RNase recognition site, which could induce the variations of mRNA stability in different RNase accessibility.
2.Switch-like mechanism
Switch mode is also thought as a two-state system. It consists of two mutually exclusive structures that depend on temperature shifts. These two structures have different mRNA stability and ribosome accessibility, thereby regulating the translation rate at different temperatures.
Figure 3. The zipper-like mechanism and switch-like mechanism (A) In zipper-like manner the stem-loop in mRNA would melt gradually when the temperature rises, which induce different Ribosome or RNase accessibility. (B) In switch-like mechanism, mRNA holds different conformations at different temperatures, which facilitate the translation efficiency in different degree.
From natural to synthetic
What I cannot create, I do not understand.
-----Prof. Richard Feynman
In many emerging fields of synthetic biology, it has been a persistent pursuit of scientists and engineers to rationally engineer biomolecular systems for biosensing[8] (figure 4). With respect to biological thermosensing, engineered thermosensors would be a huge boost in foundation and application, such as periodical drug delivery, diagnostics in vivo, temperature-induced gene expression and purification, fermentation and other large-scale processes[7]. The simplicity, accuracy and controllability of RNA-based thermosensors are required. Since natural RNA-based thermosensors adopts complex structures such as ROSE thermometer in rpoH and ibpAB mRNA[9], eliciting the minimal temperature-sensing element is critical to design synthetic and standard RNA-based thermosensors.
Figure 4. Synthetic biology extends the study of bilogical systems beyond that exsist[10]. Traditional biologists seek to understand how their molecular circuitry gives rise to observed behaviour. Synthetic biologists seem to do the opposite. They forward engineer new behaviour using well-understood genetic components and as simple a design as possible.
In 2008, a group attempted to de novo design of simple one-stem for manipulation of bacterial gene expression in response to temperature[5]. This modular work illustrated that the stability and loop size of heat-inducible RNA-based thermosensors were systematically altered and the regulatory consequences were monitored. In 2015, another group attempted to de novo design of simple heat-repressible RNA-based thermosensors, which was based on the interaction between RNase E and mRNA[9]. It modulates mRNA degradation and stability, and these thermosensors allow a response in the physiological temperature range. This suggests that there are numerous ways in which temperature-responsive RNA sensors can be set up. Inspired by this, we designed and demonstrated standard heat-inducible and heat-repressible RNA-based thermosensors with different sensing temperatures, and also designed new types of cold-inducible and cold-repressible RNA-based thermosensors, which are based on existing or hypothesized principles. After that we created an integrated and standard RNA-based thermosensors toolkit successfully.
SynRT: The collection and toolkit
The four types of synthetic and standard RNA-based thermosensors we designed are shown as follows:
1)Heat-inducible RNA-based thermosensors
2)Heat-repressible RNA-based thermosensors
3)Cold-inducible RNA-based thermosensors
4)Cold-repressible RNA-based thermosensors
>Learn more on our design page<
Firstly, we designed and standardized them based on free-energy method, which could broaden the temperature sensing range of these engineered thermosensors. It would adapt to a variety of possible situations of application. Secondly, we predicted their theoretical sensing temperature through software and detected the practical threshold by experimenting setting temperature gradient. Thirdly we used biophysical mathematical model to analyse their abilities to sense temperature and modulate the expression intensity, such as critical temperature, maximal expression intensity and sensitivity. Then we constructed evaluation model based on random forest, which we use to judge whether the thermosensor is desirable or undesirable.
After these systematic analytical works, we created a toolkit and matched search engine for potential users in numerous foundational and applied areas. We name this as SynRT toolkit, and the latest version is SynRT toolkit 3.0.
>See our project page<
References
[1]Haugen, F. (2009). Basic Dynamics and Control. ISBN 978-82-91748-13-9
[2]Kemmer C, Neubauer P. Antisense RNA based down-regulation of RNaseE in E. coli[J]. Microbial Cell Factories, 2006, 5(1):38-38.
[3]Narberhaus F, Waldminighaus T, Chowdhury S. RNA thermometers[J]. Fems Microbiology Reviews, 2006, 30(1):3–16.
[4]Riesner D, Steger G, Zimmat R, et al. Temperature-gradient gel electrophoresis of nucleic acids: analysis of conformational transitions, sequence variations, and protein-nucleic acid interactions[J]. Electrophoresis, 2010, 10(5-6):377-389.
[5]Giuliodori A M, Di P F, Marzi S, et al. The cspA mRNA is a thermosensor that modulates translation of the cold-shock protein CspA.[J]. Molecular Cell, 2010, 37(1):21-33.
[6]Kortmann J, Narberhaus F. Bacterial RNA thermometers: molecular zippers and switches.[J]. Nature Reviews Microbiology, 2012, 10(4):255-65.
[7]Dy A J, Collins J J. Engineering Models to Scale[J]. Cell, 2016, 165(3):516-517.
[8]Gaubig L C, Waldminghaus T, Narberhaus F. Multiple layers of control govern expression of the Escherichia coli ibpAB heat-shock operon[J]. Microbiology, 2011, 157(1):66-76.
[9]Hoyneso'Connor A, Hinman K, Kirchner L, et al. De novo design of heat-repressible RNA thermosensors in E. coli[J]. Nucleic Acids Research, 2015, 43(12):6166-6179.
[10]Elowitz M, Lim W A. Build life to understand it[J]. Nature, 2010, 468(7326):889.
Jilin University
Office of Undergraduate Education
Office of Global Engagement
School of Life Science
National Engineering Laboratory for AIDS Vaccine
Key Laboratory for
Molecular Enzymology and Engineering
CONTACT US
Jilin_China@Outlook.com
ACKNOWLEDGEMENTS
