<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Caltech/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Caltech/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="font-size:24pt;"><FONT face="georgia" style="color:#BB4400"><CENTER>Caltech iGEM 2011
</CENTER></FONT></DIV><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><CENTER><A href="/Team:Caltech" title="Team:Caltech"><FONT face="verdana" style="color:#BB4400"><B>Home</B></FONT></A><P><STRONG class="selflink"><FONT face="verdana" style="color:#BB4400"><B>Project</B></FONT></STRONG><A href="/Team:Caltech/Data" title="Team:Caltech/Data"><FONT face="verdana" style="color:#BB4400"><B>Data</B></FONT></A><A href="/Team:Caltech/Parts" title="Team:Caltech/Parts"><FONT face="verdana" style="color:#BB4400"><B>Parts</B></FONT></A><A href="/Team:Caltech/Team" title="Team:Caltech/Team"><FONT face="verdana" style="color:#BB4400"><B>Team</B></FONT></A><A href="/Team:Caltech/Notebook" title="Team:Caltech/Notebook"><FONT face="verdana" style="color:#BB4400"><B>Notebook</B></FONT></A><A href="/Team:Caltech/Biosafety" title="Team:Caltech/Biosafety"><FONT face="verdana" style="color:#BB4400"><B>Biosafety</B></FONT></A><A href="/Team:Caltech/Human_Impact" title="Team:Caltech/Human Impact"><FONT face="verdana" style="color:#BB4400"><B>Human Impact</B></FONT></A><A href="/Team:Caltech/References" title="Team:Caltech/References"><FONT face="verdana" style="color:#BB4400"><B>References</B></FONT></A><A href="/Team:Caltech/Acknowledgments" title="Team:Caltech/Acknowledgments"><FONT face="verdana" style="color:#BB4400"><B>Support</B></FONT></A></P></CENTER></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Bioremediation_of_Endocrine_Disruptors_Using_Genetically_Modified_Escherichia_Coli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bioremediation of Endocrine Disruptors Using Genetically Modified Escherichia Coli</SPAN></A></LI><LI class="toclevel-2"><A href="#Introduction:"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction:</SPAN></A></LI><LI class="toclevel-2"><A href="#BisdA_and_BisdB"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">BisdA and BisdB</SPAN></A></LI><LI class="toclevel-2"><A href="#Gibson_Assembly"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-2"><A href="#DDT_Dehydrochlorinase"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">DDT Dehydrochlorinase</SPAN></A></LI><LI class="toclevel-2"><A href="#Cytochrome_p450s"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Cytochrome p450s</SPAN></A></LI><LI class="toclevel-2"><A href="#Selection_of_EDC-Degrading_Organisms"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Selection of EDC-Degrading Organisms</SPAN></A></LI><LI class="toclevel-2"><A href="#Biofilm_Columns"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Biofilm Columns</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Bioremediation_of_Endocrine_Disruptors_Using_Genetically_Modified_Escherichia_Coli"> Bioremediation of Endocrine Disruptors Using Genetically Modified <I>Escherichia Coli</I></SPAN></H2><P>Endocrine disruptors, or substances that mimic estrogen in the body, have detrimental biological effects on the reproduction of several species of fish and birds; the Caltech team focuses on bioremediation of these toxins. Our goal is to create a system housed in <I>E. coli</I> that can be used to process water and remove endocrine disruptors on a large scale. We focus on isolating degradation systems for the common endocrine disruptors bisphenol A (BPA), DDT, nonylphenol and 17a-ethynylestradiol.  We synthesized known degradation enzymes DDT dehydrochlorinase, BisdA and BisdB, and characterized the behavior of these enzymes when acting on our target endocrine disruptors. In addition, we explored the potential of certain cytochrome p450s to initiate degradation of these chemicals, focusing on WT-F87A degradation of BPA. Finally, we characterized the functionality of <I>E. coli</I> protein processing when <I>E. coli</I> is deployed as an easily containable biofilm on various substances in aqueous environments.
</P><H3><SPAN class="mw-headline" id="Introduction:">Introduction:</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Sources of Endocrine Disruptors</DIV></DIV></DIV><P>Endocrine-disrupting chemicals (EDCs) are chemicals that interact with the endocrine system by binding to hormone receptors, causing problems in sexual development and reproduction of organisms. These chemicals are introduced to the environment from improper disposal of plastic wastes, hormonal medications remaining in human waste, and pesticides. Areas with high concentrations of estrogen in water have been shown to correlate with a higher percentage of intersex fish, and pesticides such as DDT have been shown to impact the development of the female reproductive tract in birds. Many EDCs are persistent organic pollutants, and even though regulations have been put in place for pesticide use and industrial production of endocrine disruptors, many of these chemicals continue to pollute bodies of water in significant concentrations.
Synthetic biology involves the engineering of genetic networks to create modified organisms that can address a problem or perform a task. We focus on modifying the genetic code of <I>E. coli</I> to remove endocrine-disrupting chemicals from water. To do this, we created constructs containing previously discovered BisdA and BisdB enzymes that have been shown to degrade BPA and we are testing their efficacy. We also assembled a gene for DDT Dehydrochlorinase, an enzyme previously shown to degrade DDT, and expressed this gene in<I> E. coli</I> to characterize its properties. We were able to establish that our synthesized gene expresses a protein that consistently degrades DDT. This enzyme may also be useful in degrading other EDCs. 
In addition to testing enzymes known to degrade EDCs, we also conducted a search for cytochrome p450s, enzymes known to initiate degradation of many compounds through an oxidation-reduction process. Using p450s selected from the Arnold lab library, we conducted reactions with BPA, DDT, nonylphenol, and 17a-ethynylestradiol. We analyzed the products of these reactions with HPLC.  We found that BPA showed evidence of degradation when combined with the p450 WT-F87A, so the gene for this p450 could be useful for bioremediation.
To further explore the possibilities for bioremediation of EDCs, we performed a selection experiment on biological samples from the Los Angeles River. Since the Los Angeles River is highly polluted with plastics and located in an urban area, it is likely that some selection has already occurred to allow organisms to consume EDCs. We grew these samples on minimal media with various EDCs as carbon sources, and demonstrated the presence of live organisms after several weeks of serial selection, indicating that these organisms can degrade EDCs as a primary carbon source. 
In order to test the feasibility of an<I> E. coli</I>-housed system for bioremediation, we conducted experiments with biofilms prepared on glass beads in a column. As a model system, we used the degradation of X-gal with beta-galactosidase. We also conducted an evaluation of local water plants to determine typical purification systems for large bodies of water and see if a bioremediation unit can integrate into typical water treatment plants.</P><H3><SPAN class="mw-headline" id="BisdA_and_BisdB">BisdA and BisdB</SPAN></H3><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Plasmid map of BisdA under the lac promoter <A href="http://partsregistry.org/Part:BBa_K620002" class="external text" rel="nofollow">K620002</A> in <A href="http://partsregistry.org/Part:pSB4A5" class="external text" rel="nofollow">pSB4A5.</A></P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Plamid map of BisdB under the tet promoter <A href="http://partsregistry.org/Part:BBa_K620003" class="external text" rel="nofollow">K6230003</A> in <A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3.</A></P></DIV></DIV></TD></TR></TBODY></TABLE><P>We sourced genetic material for BisdA <A href="http://partsregistry.org/Part:BBa_K123000" class="external text" rel="nofollow">(K123000)</A> and BisdB <A href="http://partsregistry.org/Part:BBa_K123001" class="external text" rel="nofollow">(K123001)</A> from the Registry of Standard Biological Parts. BisdA, a ferrodoxin, and BisdB, a cytochrome p450, both isolated from a strain of <I>Sphingomonas bisphenolicum</I>, are known to degrade bisphenol A in the presence of each other. The sequence listed for these parts had the wrong codons listed, so we sequenced the parts to determine the correct codons. We then designed genetic constructs with inducible promoters, ribosome binding regions, and terminators in plasmid backbones for each gene. We accessed these parts by transforming DNA from the Registry of Standard Biological Parts into chemically competent <I>E. coli</I> for construction, and used PCR to extract our desired components. We attempted several methods of assembly for these pieces, including traditional assembly, Gibson assembly, and a combination of PCR assembly for the coding construct and standard assembly to insert the coding construct into a backbone. After assembly of these components was complete, we combined the two coding constructs into one vector and expressed this vector in <I>E. coli</I> for experimentation. We are now inducing production of <A href="/File:PNT001map.png" title="File:PNT001map.png">BisdA</A> and <A href="/File:Pnt002map.png" title="File:Pnt002map.png">BisdB</A> so that we can investigate their ability to degrade EDCs.
</P><H3><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">A gel showing Gibson assembly of GFP, promoter, terminator and pSB4A5 and BisdA, lac promoter, terminator and pSB4A5 (pNT001) stopped at 0, 5, 10, 15, 30, 60 minutes. Generally, over time, the smaller bands decrease in brightness and larger bands increase in brightness</DIV></DIV></DIV><P>We attempted to use Gibson Assembly for creating composite BioBrick plasmids including inducible BisdA and BisdB. After many weeks of doing Gibson reactions and screening for colonies, we have observed that Gibson assembly may not be the most efficient and reliable method of assembling BioBricks. Gibson assembly may be a quicker method of cloning than using restriction enzymes and ligase, but this work indicates that quality and quantity of plasmids produced using this method are not as easily reproducible as standard assembly for assembly of multiple BioBrick parts is. If iGEM teams wish to use this method for assembly, we have shown that limiting the total DNA in the Gibson reaction, limiting the number of parts being combined in the reaction and using extremely competent cells improves the ratio of experimental colonies to negative control colonies. These can then be screened using colony PCR or sequencing. However, the high complementarity between the BioBrick prefix and suffix could contribute to the high numbers of self-ligation we observed. Due to the enzymes in the Gibson reaction, phosphatase cannot be used, a common procedure in standard assembly.</P><P>We recommend PCR assembly, if possible, of composite BioBrick inserts rather than multi-step standard assembly to increase the efficiency of assembling of BioBrick parts. Future iGEM teams are advised to try different methods of assembly and cloning in parallel, as the parts are not as modular as stated and each combination behaves differently than others.
</P><H3><SPAN class="mw-headline" id="DDT_Dehydrochlorinase">DDT Dehydrochlorinase</SPAN></H3><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Plasmid map of <A href="http://partsregistry.org/Part:BBa_K620000" class="external text" rel="nofollow">K620000</A> in <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> (submission plasmid)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Plasmid map of <A href="http://partsregistry.org/Part:BBa_K620000" class="external text" rel="nofollow">K620000</A> in pET11-a to amplify and purify the protein.
</P></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">The band indicating DDT dehydrochlorinase is visible in both elution columns, between the 21,500 and 31,000 g/mol bands. The DDT protein is 23,500 g/mol.</DIV></DIV></DIV><P>We found DDT Dehydrochlorinase in the literature as an enzyme discovered to degrade DDT. We found the amino acid code for this enzyme on GenBank, used <A href="http://helixweb.nih.gov/dnaworks/" class="external text" rel="nofollow">DNAWorks</A> to design oligos for assembling this enzyme optimized in E. coli, and assembled this gene using PIPE cloning. We then inserted this gene in a pET vector with a his-tag and overexpressed it in <I>E. coli</I>. We purified the protein and ran it on a gel, indicating that this gene can be expressed in <I>E. coli</I> as shown in the <A href="/File:DDTdechlorinase.jpg" title="File:DDTdechlorinase.jpg">DDT dehydrochlorinase gel image</A>. </P><P>We next conducted degradation experiments with DDT. We set up an experiment in which cell lysate of cells producing DDT dehydrochlorinase was prepared in a reaction with DDT and some buffers and reacted overnight. Then the reaction mixture was analyzed using electrospray GCMS, along with a control of DDT without cell lysate. As shown in the <A href="/File:DDT_before_degradation.png" title="File:DDT before degradation.png">GCMS of DDT without degradation enzymes</A>, there is a clear band at 235 g/mol. DDT's molar mass is 354.5. This indicates that the original <A href="/File:Ddt.png" title="File:Ddt.png">structure</A> loses a carbon and three chlorines to form a <A href="/File:MS1DDT235.png" title="File:MS1DDT235.png">new structure</A> with this mass during the GCMS process. The <A href="/File:DDT_after_degradation.png" title="File:DDT after degradation.png">12.5 minute GC peak and the 16.5 minute peak</A> in the DDT-DDT dehydrochlorinase reaction show a further degradation of DDT. The 16.5 minute peak showed an MS peak at 206 g/mole. The degradation that could result in this weight involves the loss of <A href="/File:MSDDT205.png" title="File:MSDDT205.png">two chlorines and a phenyl group</A>. 
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>GCMS of DDT showing a clear peak at 235g/mol
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>DDT reaction with DDT Dehydrochlorinase analyzed with GCMS-12.5 minute peak. There is a peak at 105 g/mol and a peak at 133 g/mol
</P></DIV></DIV></TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>DDT Molecular Structure
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>DDT structure after GCMS with a molecular weight of 206g/mol
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>DDT structure after reaction with DDT Dehydrochlorinase and GCMS with a molecular weight of 206 g/mol
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>There is another mass spec band in this reading at 191 g/mol, which we are still working to identify. In the 12.5 minute GC peak, there are MS peaks at 133 g/mol and at 105 g/mol, which we are also still working to identify.  However, these two GC peaks are not present in analysis of our reaction buffer with DDT and our reaction buffer with enzyme, indicating that they result from a degradation reaction between DDT dehydrochlorinase and DDT. 
</P><H3><SPAN class="mw-headline" id="Cytochrome_p450s">Cytochrome p450s</SPAN></H3><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>HPLC of BPA, without enzyme added
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>HPLC of BPA degraded, with p450 Wt-F87A
</P></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Plasmid map of <A href="http://partsregistry.org/Part:BBa_K620001" class="external text" rel="nofollow">K620001</A> in <A href="http://partsregistry.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> (submission plasmid).</DIV></DIV></DIV><P>Cytochrome p450s are known to be initiators of degradation for several compounds. We analyzed the structures of the cytochrome p450s which the Arnold Lab possessed in their genetic library and selected four highly promiscuous p450s. We then conducted reactions of these p450s with BPA, DDT, 17a-ethinylestradiol, and nonylphenol. We analyzed the product with HPLC. As shown in the HPLC results, the<A href="/File:Blank_BPA.jpg" title="File:Blank BPA.jpg"> plain BPA HPLC</A> has only one peak, while the <A href="/File:WT-F87A.jpg" title="File:WT-F87A.jpg">HPLC analysis of BPA degraded with the WT-F87A cytochrome p450</A> has two. We also analyzed this reaction using electrospray GCMS. This showed that the p450 degraded BPA from its <A href="/File:BPA_electrospray.jpg" title="File:BPA electrospray.jpg">original MS reading</A> of 227 g/mol to a <A href="/File:BPAdegra.png" title="File:BPAdegra.png">MS reading</A> of 205g/mole. We are working to determine what degradation steps would result in this mass. However, since the GC analysis shows a wide band of molecules with a mass of 205 g/mol, this indicates that there are likely many different structural compounds with this molecular mass created when the WT-F87A p450 degrades BPA. This result establishes another degradation pathway for Bisphenol A.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Gas chromatography of BPA before degradation
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Mass spectrometry of BPA showing characteristic peak at 227g/mol
</P></DIV></DIV></TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Gas chromatography of BPA after reaction with the WT-F87A P450. The peak is wider, indicating that multiple species are present
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Mass spectrometry of BPA after reaction with the WT-F87A P450. The peak at 227g/mol is diminished, and there is a new large peak at 205g/mol
</P></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Selection_of_EDC-Degrading_Organisms">Selection of EDC-Degrading Organisms</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Our gene fishing method</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Minimal media cultures</DIV></DIV></DIV><P>In order to identify an organism capable of degrading our EDCs, we collected dry and wet samples from the Los Angeles river and used these samples to inoculate liquid minimal media cultures in which the sole source of carbon was the EDC.  (We used media with and without a vitamin mix to account for any compounds the organisms might not be capable of manufacturing themselves, but the presence or absence of the vitamin mix seems to have made little difference in the growth of the organisms; cultures exhibited equivalent growth both with and without the vitamin mix.)  We sequentially used these cultures to inoculate new cultures every two to three days over an eight-week period in order to further isolate organisms.  
</P><P>At the third week we plated the cultures on LB plates and observed <A href="/File:Plates_wk_3.png" title="File:Plates wk 3.png">significant growth of many different types of organisms</A>, indicating that our cultures contained organisms capable of surviving on our EDCs.  To further isolate these organisms, colonies from each culture were resuspended in liquid minimal media; as before, new liquid minimal media cultures were inoculated from each of these every two to three days.  The cultures were plated again on LB in the eighth week and again, <A href="/File:Plates_wk_8.png" title="File:Plates wk 8.png">significant growth was observed</A>.
</P><P>At this stage, we attempted to plate our cultures on solid minimal media; however, all of the compounds we chose are insoluble in water, which makes traditional plating impossible. We attempted several alternate methods of plating the EDCs together with the minimal media and cultures, but all these attempts were unsuccessful. Thus, although we demonstrated the presence of EDC-degrading organisms in our cultures, we were unable to fully isolate any organisms for further study.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Figure 1: Liquid minimal media cultures plated on LB; week 3
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Figure 2: Liquid minimal media cultures plated on LB; week 8
</P></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Biofilm_Columns">Biofilm Columns</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Our plan to grow and test biofilms</DIV></DIV></DIV><P>We cultured top10 <I>E. coli</I> biofilms on glass beads and stained with crystal violet to show growth the bacteria. To measure the thickness of the biofilms, we washed the beads with water to remove free-floating crystal violet and bacteria, used a set amount of ethanol to wash off the crystal violet stain from the beads, then ran the wash through a spectrophotometer- the higher the absorbance, the thicker the biofilm. Our next step was to create a construct within the top10 bacteria which constitutively expressed the lacZ gene. Since lacZ cleaves X-gal, a clear chemical, into its component dye and galactose, the lacZ-infused bacteria was able to change a solution of X-gal bright blue in under 10 minutes. The final part of this experiment would be to grow the lacZ-infused bacteria into a biofilm onto glass beads, then flow the X-gal solution through a column of the biofilm beads. The effluent flow should be bright blue. However, this last bit hasn't been completed as of the iGEM  due date.</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Flask of bacteria modified with lac-z
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Flask of bacteria modified with lac-z after addition of x-gal
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Visible layers of biofilm as visualized by crystal violet
</P></DIV></DIV></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Caltech/Project">http://2011.igem.org/Team:Caltech/Project</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Project&amp;oldid=259251" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>