<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNICAMP-Brazil_Coliguard_Recognition"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNICAMP-Brazil/Coliguard/Recognition">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNICAMP-Brazil/Coliguard/Recognition&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Recognition&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Recognition&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNICAMP-Brazil/Coliguard/Recognition" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNICAMP-Brazil/Coliguard/Recognition</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TABLE width="100%" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="menuL"><A class="branco" title="Home" href="https://2009.igem.org/Team:UNICAMP-Brazil">Home</A></TD></TR><TR><TD class="menuL"><A class="branco" title="iGEM" href="https://2009.igem.org/Team:UNICAMP-Brazil/Igem">iGEM</A></TD></TR><TR><TD class="menuL">Project</TD></TR><TR><TD class="menuLS"><A class="branco" title="The Coliguard" href="https://2009.igem.org/Team:UNICAMP-Brazil/Coliguard">The Coliguard</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="The Yeastguard" href="https://2009.igem.org/Team:UNICAMP-Brazil/Yeastguard">The Yeastguard</A></TD></TR><TR><TD class="menuL">Lab</TD></TR><TR><TD class="menuLS"><A class="branco" title="Protocols" href="https://2009.igem.org/Team:UNICAMP-Brazil/Protocols">Protocols</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Notebooks" href="https://2009.igem.org/Team:UNICAMP-Brazil/Notebooks">Notebooks</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Safety" href="https://2009.igem.org/Team:UNICAMP-Brazil/Safety">Safety</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Sandbox" href="https://2009.igem.org/Team:UNICAMP-Brazil/Sandbox">Sandbox</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Team" href="https://2009.igem.org/Team:UNICAMP-Brazil/Team">Team</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Unicamp &amp; Brazil" href="https://2009.igem.org/Team:UNICAMP-Brazil/UnicampBrasil">Unicamp &amp; Brazil</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Human practices" href="https://2009.igem.org/Team:UNICAMP-Brazil/HumanPractices">Human practices</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Gallery" href="https://2009.igem.org/Team:UNICAMP-Brazil/Gallery">Gallery</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Sponsors &amp; Support" href="https://2009.igem.org/Team:UNICAMP-Brazil/Sponsors">Sponsors &amp; Support</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Press" href="https://2009.igem.org/Team:UNICAMP-Brazil/Press">Press</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Contact" href="https://2009.igem.org/Team:UNICAMP-Brazil/Contact">Contact</A></TD></TR></TBODY></TABLE></P><H1><SPAN class="mw-headline" id="The_Coliguard_-_Recognition">The Coliguard - Recognition</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>We aim to create bacteria that can protect themselves against contaminants. So, our bacteria need to be able to kill other organisms which may appear in the culture medium. 
</P><UL><LI>Why not create a continuous mechanism of eliminating contaminants?
</LI></UL><P>A simple way to do this would be to create bacteria that express a toxin constitutively. However, if the bacteria produce the toxin continuously, they will compromise their metabolic energy for the production of the toxic compound and, in addition, the production of large amounts of these antibiotic substances may contribute to the problem of antibiotic resistance. Moreover, the toxin itself would be a type of contaminant in the industrial process, requiring resources to be allocated in order to remove the toxin from the medium and purify the compound of interest. 
</P><UL><LI>Therefore, we need a contaminant-sensitive recognition system!
</LI></UL><P>We need to create a contaminant-sensitive mechanism that will allow the detection of the contaminants and trigger the killing mechanism. In this case, the toxin will be produced only when the contaminants are present. 
</P><UL><LI>How will our bacterial guards recognize the contaminants? 
</LI></UL><P>Our idea is based on the premise that the engineered <I>E. coli</I> must be able to recognize contaminants in the culture medium as non-self. Thus, our bacteria need to recognize a specific signal that they don’t produce, but which are broadly produced or presented by potential contaminants. Thinking about the potential signals, we found two candidates: the AI2 (auto inducer 2) and the conjugation recognition mechanism. 
</P><UL><LI>Using AI-2 as a signal. How?
</LI></UL><P>As most bacterial species produces AI2 as a secondary metabolite(1; 2; 3), we decided to use this compound as a recognition factor. Our <I>E. coli</I> will be an AI2- strain and won´t produce native AI2. The presence of AI2 in the culture medium will then indicate the presence of contaminants, which will be recognized by an AI2 sensitive promoter present in our <I>E. coli</I>. This promoter will control the differentiation and killing mechanisms (see the links for more information).
</P><UL><LI>But, what if the contaminants do not produce AI-2? 
</LI></UL><P>In these situations, our <I>E.coli</I> will use another recognition mechanism, based in conjugation.  In conjugation, a donor bacterium only conjugates with organisms that don’t have the same conjugative plasmid. Bacteria that already have the plasmid display membrane proteins that prevent the conjugation with other bacteria that have the same conjugative plasmid (4). Therefore, if our <I>E.coli</I> has a conjugative plasmid, they will only conjugate with different organisms, which will be the contaminants. 
</P><P>Nevertheless, conjugation is a process that does not occur between <I>E. coli</I> and any other organism. There is a type of specificity to this process. A possible alternative to reduce this specificity is using multiple replication origins in the same conjugative plasmid (see Recognition by conjugation – lab strategy and results). However, this solution fails to address the whole problem. 
</P><P>We think that a better way to improve the recognition system is to combine the conjugation signal with the AI-2 recognition mechanism.
</P><H2><SPAN class="mw-headline" id="Our_recognition_system:_a_combination_of_conjugation_and_the_AI-2_recognition_mechanism.21">Our recognition system: a combination of conjugation and the AI-2 recognition mechanism!</SPAN></H2><P>To improve our recognition system, we will use a signal produced at the beginning of the conjugation process in order to stimulate the production of AI-2 in our <I>E.coli</I>. This signal will induce the promoter pY, which controls the expression of the conjugation-related genes. This promoter will then control the expression of the <I>AI-2</I> gene in our <I>E.coli</I>. As a result, when conjugation begins the promoter will be induced and thus cause an up-regulation in <I>AI-2</I> gene expression. In this way, the conjugation process will activate the killing and differentiation mechanisms in a way that is AI-2-dependent.  
</P><P>Combining these two recognition systems, our <I>E. coli</I> will be able to recognize a vast group of contaminants.
</P><H2><SPAN class="mw-headline" id="Recognition_by_conjugation">Recognition by conjugation</SPAN></H2><P>	As mentioned in the introduction, we will use conjugation as a signal to recognize contaminants. During conjugation, a donor bacterium only conjugates with organisms that don’t have the same conjugative plasmid. Bacteria that already have the plasmid display membrane proteins that prevent conjugation with other bacteria that have the same conjugative plasmid. 
	We chose the F plasmid to use in our project because it is the best-known conjugative plasmid. When looking for a recognition factor that could be used in our mechanism, scientific papers concerning the F plasmid mentioned the <I>Py</I> promoter, which controls the expression of conjugation-related genes in the <I>tra-operon</I> of the F-plasmid. This operon encodes the majority of the proteins involved in DNA transfer by conjugation in <I>E.coli</I> (5; 6). However, information concerning how the <I>Py</I> promoter works is lacking, and, therefore, we will need to conduct experiments to verify its action (see “Py promoter cloning and characterization” section). 
	Our idea is to use the <I>Py</I> promoter to act as a second signal (in addition to AI-2) to recognize contaminants, controlling the expression of the <I>AI-2</I> gene and, consequently, the killing and the differentiation mechanisms that are AI-2-dependent. In order to do that, we first need to isolate the promoter and then, characterize its operation.
In parallel, we will improve F plasmid to use in our engineered <I>E. coli</I> by, removing uninteresting parts in order to make it smaller and easier to copy. Moreover, we will remove the insertion sequences, which are the portions of the F plasmid DNA that mediate the integration into the bacterial chromosome, thus avoiding possible problems caused by F plasmid integration. (7). Moreover, we will add different replication origins to the plasmid, thus increasing the range of contaminants that may be recognized (8).
</P><H2><SPAN class="mw-headline" id="Py_promoter_isolation">Py promoter isolation</SPAN></H2><P><B>Strategy</B></P><P>The strategy consists of PCR amplification of the <I>Py</I> promoter region by using primers that contain Biobrick prefix and Biobrick suffix, thus allowing us to clone the amplicon as Biobrick parts. Before designing primers, we verified that none of the restriction sites of the prefix or sufixe are contained in the amplified sequence.
</P><P><B>What we have done</B></P><P>We designed primers from the F plasmid sequence (5) to isolate the <I>Py</I> promoter by PCR. We used information on scientific literature to determine the region of the promoter (5; 6). We found that there is an intrinsic curvature, upstream of the promoter region, which could interfere in the promoter regulation. We decided, therefore, to design two pairs of primers: PyF1 - PyR1 and PyF2 - PyR1. These primer sets have the same reverse primers but different forward primers. The PyF1 – PyR1 amplicon has 133bp and includes the two closer bends upstream of the promoter region. The PyF2 - PyR1 amplicon has 75bp and doesn’t include any bend (see Notebooks – September 4th for more details).
We managed to amplify the pY promoter amplicons through PCR from the E. coli F plasmid, creating two new Biobrick parts: BBa_K284007 (PyF2 - PyR1 amplicon) and  BBa_K284008 (PyF1 - PyR1 amplicon).
</P><H2><SPAN class="mw-headline" id="Py_promoter_cloning_and_characterization">Py promoter cloning and characterization</SPAN></H2><P><B>Strategy</B></P><P>After isolation of the <I>Py</I> promoter, it must be cloned in the BioBrick vector according to the standard assembly strategy. Afterwards, to characterize the promoter activity, we will make a device, with a reporter gene under the control of the <I>Py</I> promoter. We decided to use a fluorescent protein as a reporter gene for this characterization, such as the GFP (Green Fluorescent Protein). This reporter system will allow us to use the fluorescence signal in order to quantify promoter activity in a spectrofluorometer (Hitachi F-4500).
The characterization experiment consists of a conjugation experiment (9) to be conducted using our engineered <I>E. coli</I> as the donor bacterium and another <I>E. coli</I>, without the F plasmid and containing a marker of resistance different from the F plasmid, as the receiver bacterium. After the beginning of the conjugation process, we will take samples after 1h, 2h, 3h, 6h, 9h, 12h and 24h. We will analyze these samples in a spectrofluorometer to quantify the fluorescence emitted. In addition, we will inoculate a portion of the sample in Petri dishes containing LB medium with the appropriated antibiotics to select only the receiver bacteria with the F plasmid. The plates will be incubated overnight at 37°C and then we will quantify the conjugation events through the observation of bacteria growing in the plates. Through this experiment, we will be able to analyze the <I>Py</I> promoter activity data in comparison with the conjugation events. If the measured Py promoter activity and the conjugation process begin coincide, we may conclude that the experiment worked and that the <I>Py</I> promoter can be used as a signal for conjugation and could thus be adapted into our recognition mechanism. 
</P><P>Literature reports that the <I>Py</I> promoter has a basal activity (5; 6). However, this fact won’t be a problem because its function will be adapted to the control of the production of AI-2 in the presence of contaminants. AI-2 is a type of signal that needs to exceed a threshold in order to stimulate the promoter sensitive to AI-2 (10), which will be in turn coupled to the differentiation and the killing mechanisms. Therefore, the basal activity of the <I>Py</I> promoter should not be enough to trigger the differentiation and killing mechanisms.
</P><P><B>What we have done</B></P><P>At first we intended to use the GFP as a reporter gene. We took the BBa_E0040 part, which contains the GFP, and started to work with it. Unfortunately, it didn’t work as expected. We did some restriction analyses and the bands expected after the electrophoresis running didn’t appear. 
After that, we decided to use another reporter gene. Searching for it in the Registry of Standard Parts, we found a device which is used to measure promoter activity, the exact construction that we need! So, we got this part (Bba_J23100) and, according to the standard assembly strategy, we substituted the promoter which already was there and put the <I>Py</I> promoter amplicons, resulting in BBa_K284008 (see Figure 1). We were only able to clone the PyF1 - PyR1 amplicon, the larger one, in this device, even after several repetitions. Therefore, we decided to continue experiments with this device only, in order to have time to perform the characterization experiment. 
</P><P>
In the characterization experiment, we will use a donor bacterium that carries an F plasmid with the ampicillin resistance gene. Thus, we can’t select transformed bacterial colonies with our device in a medium supplemented with ampicillin. Therefore, we cloned the BBa_K284008 part into the pSB1AK3 vector, which carries the ampicillin and kanamycin resistance genes. Then, we could select the colonies in a medium supplemented with kanamycin.
</P><H2><SPAN class="mw-headline" id="Engineering_the_F_plasmid_.E2.80.93_removing_parts_and_adding_new_replication_origins">Engineering the F plasmid – removing parts and adding new replication origins</SPAN></H2><P><B>Strategy</B></P><P>First of all, we will reduce the size of the F plasmid by removing uninteresting parts and the insertion sequences, which are the portions of the F plasmid DNA that mediate the integration into the bacterial chromosome (generating Hfr strains). As Willetts and Johnson reported (7), we will remove a part from the F plasmid by single digestion with HindIII followed by recircularization. This process creates pED100, a conjugative F plasmid derivative without insertion sequences. Through this simple procedure we will reduce considerably the possibility of plasmid integration into the chromosome. 
After that, we will add multiple replication origins to our F plasmid (8). In consequence, we will reduce conjugation specificity, thus increasing the range of contaminant recognition.Our strategy is to use the replication origins of plasmids from different groups of bacteria, in an effort to reduce as much as we can the conjugation specificity.  
</P><P><B>What we have done</B></P><P>	We managed to remove the insertions sequences and the uninteresting parts from the F plasmid. At this way, our plasmid is now easier to copy and to work with.
</P><H2><SPAN class="mw-headline" id="The_recognition_by_AI-2">The recognition by AI-2</SPAN></H2><P><B>Strategy</B></P><P>A well-known mechanism of cell-cell communication is called quorum sensing. In that process, bacteria communicate via secreted signaling molecules called autoinducers, which contribute to the regulation of the expression of particular genes. This is found in both  gram-negative and gram-positive bacteria (11; 12). The most widespread quorum-sensing system is the luxS system (13). This inducer has been found in a large range of gram-negative and gram-positive bacteria, including some pathogens (14). Because this autoinducer and its biosynthetic pathway are the same among all bacterial species that possess luxS, it has been proposed that this system could be used in interspecies communication (15). AI-2 is produced from S-adenosyl-L-methionine (SAM) in three enzymatic steps. First, SAM acts as a methyl donor in SAM-dependent methyltransferase reactions, which lead to the formation of S-adenosyl-L-homocysteine (SAH). The next step is catalyzed by a nucleosidase (Pfs) encoded by the pfs gene that cleaves the glycosidic bond in both 50-methylthioadenosine (MTA) and (SAH). When SAH serves as a substrate, adenine and S-ribosyl-L-homocysteine (SRH) are formed. LuxS finally converts SRH into homocysteine and 4,5-dihydroxy-2,3-pentanedione (DPD). DPD is very unstableand, reacts with water and cyclizes forming autoinducer-2 (AI-2) (16).
</P><P>As explained in the overview, another recognition mechanism is the AI-2 molecule itself. This process allows the identification of bacteria that are unable to conjugate, but able to produce the auto-inducer AI-2. 
Therefore, we need a mechanism in which both recognition systems can be detected.  In order to achieve that, the production of AI-2 has to be controlled by <I>Py</I> promoter.  The LuxS gene, which encodes for the LuxS synthase, produce DPD (4,5-dihydroxy-2,3-pentanedione) that spontaneously forms the AI-2 molecule. This part was already designed by Davidson Missouri Western in 2008 and we will use it to produce AI-2 under the control of the <I>Py</I> promoter.  In case of conjugation, <I>Py</I> will be activated and the LuxS synthase will be produced.  On the other hand, microorganisms that naturally produce the AI-2 molecule skip that step. Once the recognition is achieved, it triggers the differentiation process through the expression of the CRE-recombinase. The Cre-recombinase will be under the control of the <I>QseA</I> promotor, which is activated in the presence of AI-2.  The <I>QseA</I> also is known as the quorum-sensing <I>E. coli</I> regulator A and an array study showed that this regulator is regulated by AI-2 (10). The CRE-recombinase was already designed by  Arkin Lab and will be utlized in this project. 
</P><P><B>Constructions:</B></P><P><B>1)</B> BBa_K284027 - <I>QseA</I> promoter 
This part will be constructed using the standard assembly strategy. After the amplification of the <I>QseA</I> promoter the part will be joined with the Bba_J23100 plasmid for characterization.
</P><P><B>2)</B> BBa_K284028 – LuxS under the control of <I>Py</I> promoter
This device will be composed of BBa_K284008 (Py promoter), BBa_B0034 (a RBS), BBa_K091109 (LuxS) and BBa_0014 (a double terminator) and constructed following the standard assembly strategy.
</P><P><B>3)</B> BBa_K284029 – CRE-recombinase under the control of <I>QseA</I> promoter
</P><P>This device will be composed of BBa_K28402 (QseA promoter), BBa_B0034 (a RBS), BBa_J61047 (CRE-recombinase) and BBa_0014 (a double terminator) and constructed according to the standard assembly strategy.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>1. Cao, J.G. &amp; Meighen, E.A. J Biol Chem. 1989, 264, 21670-21676. 
</P><P>2. Miller, S.T. et al.  Mol Cell. 2004, 15, 677-687. 
</P><P>3. Miller, M. B.; Bassler, B. L.  Annu. Rev. Microbiol. 2001, 55, 165-199.
</P><P>4. Helinski, D. R.  Plasmids in bacteria. 1985,995. 
</P><P>5. Frost, L. S. et al.  Microb Rev. 1994,58(2), 162-210. 
</P><P>6. Will, W. &amp; Frost, L.  J Bacteriol. 2006, 188(2), 507-514.
</P><P>7. Willets, N &amp; Johnson, D.  Mol Gen Genet. 1981, 182, 520-522.
</P><P>8. Trieu-Cuot, P. et al. Microbiol. Let. 1997, 48, 289-294. 
</P><P>9. Balbás, P. &amp; Lorence, A.  Methods in Molecular Biology : Recombinant Gene Expression. 2004, 67, 225-240.
</P><P>10. Sperandio, V. et al. Infect. Immun. 2002, 70, 3085-3093.
</P><P>11. Fuqua C., Parsek M.R., Greenberg E.P. Rev. Genet. 2001. 35, 439–468.
</P><P>12. Miller M.B., Bassler, B.L. Annu. Rev. Microbiol. 2001. V. 55, 165–199.
</P><P>13. Schauder S., Shokat S., Surette M.G., Bassler B.L. Mol. Microbiol. 2001. 41, 463–476.
</P><P>14. Surette, M. G., M. B. Miller, and B. L. Bassler. Proc. Natl. Acad. Sci. USA. 1999. 96, 1639–1644.
</P><P>15. Mok, K. C., N. S. Wingreen, and B. L. Bassler. EMBO J., 2003, 22, 870–881
</P><P>16. Winzer,K et al.  Advances in Applied Microbiology. 2003, 53.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UNICAMP-Brazil/Coliguard/Recognition">http://2009.igem.org/Team:UNICAMP-Brazil/Coliguard/Recognition</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNICAMP-Brazil/Coliguard/Recognition" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNICAMP-Brazil/Coliguard/Recognition" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Recognition&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Recognition&amp;oldid=167671" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>