<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNICAMP-Brazil_Coliguard_Differentiation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNICAMP-Brazil/Coliguard/Differentiation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNICAMP-Brazil/Coliguard/Differentiation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Differentiation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Differentiation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNICAMP-Brazil/Coliguard/Differentiation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNICAMP-Brazil/Coliguard/Differentiation</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TABLE width="100%" border="0" cellpadding="0" cellspacing="0"><TBODY><TR><TD class="menuL"><A class="branco" title="Home" href="https://2009.igem.org/Team:UNICAMP-Brazil">Home</A></TD></TR><TR><TD class="menuL"><A class="branco" title="iGEM" href="https://2009.igem.org/Team:UNICAMP-Brazil/Igem">iGEM</A></TD></TR><TR><TD class="menuL">Project</TD></TR><TR><TD class="menuLS"><A class="branco" title="The Coliguard" href="https://2009.igem.org/Team:UNICAMP-Brazil/Coliguard">The Coliguard</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="The Yeastguard" href="https://2009.igem.org/Team:UNICAMP-Brazil/Yeastguard">The Yeastguard</A></TD></TR><TR><TD class="menuL">Lab</TD></TR><TR><TD class="menuLS"><A class="branco" title="Protocols" href="https://2009.igem.org/Team:UNICAMP-Brazil/Protocols">Protocols</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Notebooks" href="https://2009.igem.org/Team:UNICAMP-Brazil/Notebooks">Notebooks</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Safety" href="https://2009.igem.org/Team:UNICAMP-Brazil/Safety">Safety</A></TD></TR><TR><TD class="menuLS"><A class="branco" title="Sandbox" href="https://2009.igem.org/Team:UNICAMP-Brazil/Sandbox">Sandbox</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Team" href="https://2009.igem.org/Team:UNICAMP-Brazil/Team">Team</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Unicamp &amp; Brazil" href="https://2009.igem.org/Team:UNICAMP-Brazil/UnicampBrasil">Unicamp &amp; Brazil</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Human practices" href="https://2009.igem.org/Team:UNICAMP-Brazil/HumanPractices">Human practices</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Gallery" href="https://2009.igem.org/Team:UNICAMP-Brazil/Gallery">Gallery</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Sponsors &amp; Support" href="https://2009.igem.org/Team:UNICAMP-Brazil/Sponsors">Sponsors &amp; Support</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Press" href="https://2009.igem.org/Team:UNICAMP-Brazil/Press">Press</A></TD></TR><TR><TD class="menuL"><A class="branco" title="Contact" href="https://2009.igem.org/Team:UNICAMP-Brazil/Contact">Contact</A></TD></TR></TBODY></TABLE></P><H1><SPAN class="mw-headline" id="The_Coliguard_-_Differentiation">The Coliguard - Differentiation</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Cell differentiation process can be defined as a “Progressive restriction of the developmental potential and increasing specialization of function that leads to the formation of specialized cells, tissues, and organs”, according to NCBI’s MeSH Database(1). Generally, it is a process by which a cell acquires a new morphological/functional type, capable of performing different and brand new tasks. In most multicellular organism, however, we might find a kind of differentiation that results in a type of cell providing support for another one(2). The differentiation mechanism chosen for our project relies on this perspective, and the first attempt at its implementation as a synthetic biology tool was made by the Paris 2007 iGEM team(3).</P><P>The differentiation system adopted for our project was designed focusing the need of creating two subpopulations with distinct characteristics and, mainly, the need of controlling the proportions between both subpopulations. Therefore, we need a differentiation system whose rate could be controlled.</P><P>Accordingly to our project objectives, the final results of this differentiation system must be:</P><P>•	<B>A Worker subpopulation</B>
This lineage is the one responsible for producing the valuable products, being unable to recognize and destroy contaminants present in the culture medium. In an attempt to maximize productivity, it would be interesting to keep the largest proportion of this subpopulation in the culture media.
</P><P>•	<B>A Killer subpopulation</B>
This lineage is the one responsible for recognizing and destroying contaminants present in the culture media. This subpopulation is also able to produce the compound of interest, but since its metabolism must be relocated for maintaining the recognition and destruction systems, it should produce the compound in a much lower concentration. To guarantee maximum efficiency of our system, this subpopulation must remain in a basal low proportion, which would be greatly increased in the presence of contaminants. Once they are fully eliminated, our basal proportions must be automatically restored.
</P><H2><SPAN class="mw-headline" id="The_mechanism">The mechanism</SPAN></H2><P>	In order to design the suggested differentiation system, we will need to modify the expression pattern of several genes, introduce new constructions, and associate two different systems previously proposed by iGEM teams: the Slippage random mechanism by team Caltech 2008(4) and the Cre-recombinase mechanism proposed by team Paris 2007(3).
</P><P><B>The slippage mechanism controls the basal proportions of both lineages</B></P><P>	Slippage is just one of the numerous errors that can be made by DNA Polymerase, and in which the number of tandem repeats of a short sequence (AGTC) could be changed during the transcriptional process.
	Our construction will consist of 10 repetitions of the sequence AGTC, with an ATG start codon on the upstream position and the Cre-recombinase coding gene on the downstream position, everything under control of a constitutive promoter. In most cases, the slippage error won’t occur and the 10 repetitions of AGTC will leave the CRE-recombinase out of frame from the ATG, thus resulting in no expression of this recombinase. The absence of this expression in most cases will result in the expression of the Worker lineage characteristics, through mechanism that will be explained later. In the few cases in which the slippage error occurs and leaves a number of repeats that can be divided by three, i.e. from 10 to 9, the ATG and CRE-recombinase gene will be in the same reading frame, thus resulting in CRE-recombinase’s expressions and lead to Killer cells differentiation.
</P><H2><SPAN class="mw-headline" id="How_will_CRE-recombinase.E2.80.99s_expression_result_in_Killer_cell_differentiation.3F">How will CRE-recombinase’s expression result in Killer cell differentiation?</SPAN></H2><P>	CRE-recombinase is an enzyme that promotes an unidirectional recombination event between the lox66 and lox71 recombination sites(5). Therefore, we intend to adapt Paris 2007’s construction cassettes flanked by both lox sites made by team Paris 2007 by inserting our genes of interest into it.
	According to the characteristics of our two subpopulations, the following genes might be necessary in order to promote cell differentiation and must be included in this construction:
</P><P>•	GFP and Resistance Genes: as reporter, necessary only to assure whether or not the construction is inserted into genomic DNA.
</P><P>•	<I>ftsK</I>: essential gene for the maintenance of the full cell cycle. The importance of the cell cycle to our project will be explained later.
</P><P>•	<I>finOP</I>: conjugation inhibition system, related with the recognition and destruction system of the killer lineage.
</P><P>The general idea is that in the absence of CRE-recombinase expression (most cases), the resulting Worker cell has this construction fully operational, resulting in a cell capable of maintaining its cell cycle but unable to conjugate due to the presence of the <I>finOP</I> genes (Worker cell’s characteristics). On the other hand, the expression of the CRE-recombinase will result in the excision of this entire construction, resulting in a cell unable to maintain a complete cell cycle, but capable of conjugation (Killer cell’s characteristics).
</P><H2><SPAN class="mw-headline" id="Why_is_this_difference_related_to_conjugation_capacity.3F">Why is this difference related to conjugation capacity?</SPAN></H2><P>	Conjugation plays an important role in recognition and destruction systems (see the specific sections for further information). In view of this, it’s very important to inhibit conjugation of the Worker cells, since they must concentrate on metabolite production and be unable to recognize and eliminate contaminants.
	The conjugation inhibition will involve a system knows as <I>finOP</I>. Briefly, this system consists of an antisense RNA (<I>finP</I>) and a small protein (<I>finO</I>)(9). The antisense RNA prevents <I>traJ</I> transcription, which lies immediately upstream of the operon and, in turn, is essential in activating the entire tra operon transcription(10). As for <I>finO</I>, it binds to <I>finP</I> and <I>traJ</I>, thereby allowing the duplex formation(11). Thus, for the correctly function of this system, both <I>finO</I> and <I>finP</I> must be expressed.
</P><H2><SPAN class="mw-headline" id="Why_can.E2.80.99t_Killer_cells_complete_their_cell_cycle.3F">Why can’t Killer cells complete their cell cycle?</SPAN></H2><P>	The inability of completing cell cycle works as a security device, as it wouldn’t let the Killer cells to proliferate and take over the entire culture, thus resulting in loss of productivity.
	This security system works in two ways:
(1)	It maintains a low basal proportion of Killer cells, in the absence of contaminants.
(2)	It guarantees a quickly return to the basal proportion after the elimination of contaminants.
</P><P>As discussed previously, it’s very important that the proportion of Killer cells could be increased in the presence of contaminants. On the other hand, it’s even more important that this proportion can be quickly restored to the basal levels after contaminant elimination. The inability of completing cell cycles makes the Killer cells proportion directly and almost exclusively dependent to the specific stimulus trigged by the presence of contaminants. Thus, as soon as this stimulus ceases, the proportion of Killer cells would be drastically reduced, given that they would be unable to reproduce and proliferate.
</P><H2><SPAN class="mw-headline" id="How_will_the_basal_proportion_change_in_the_presence_of_contaminants.3F">How will the basal proportion change in the presence of contaminants?</SPAN></H2><P>	The answer is AI2 (auto-inducer 2). AI2 is a member of signaling molecules used in bacteria quorum sensing, produced by both gram-positive and gram-negative bacteria(6, 7 and 8). We thought that, since we could silence the <I>AI2</I> coding gene on our lineage, any detectable presence of this molecule would be due to the presence of contaminants, thereby working as a trigger signal.
	Thus, we decided to place the Cre-recombinase coding gene under the control of a promoter with sensibility to AI2. We will use <I>AI2</I> self promoter for that purpose, since there is a negative feedback control related to AI2 production.
	In short, when contaminants are present they are likely to produce AI2, which will trigger CRE-recombinase expression. This expression, as previously discussed, is responsible for Killer cells differentiation.
</P><H2><SPAN class="mw-headline" id="Strategy">Strategy</SPAN></H2><P>	As we previously explained, the model that we´ve developed to the differentiation subpart of our project involves the modification and coupling of two strategies previously used by the teams of Paris 2007 and Caltech 2008.
The Paris 2007 team used the <I>FX58 E. coli</I> strain, which has the construction <I>Lox71-FSTK-Lox66</I> inserted in its genome. The <I>FSTK</I> gene is essential to cell division; its disruption stops cell division. The <I>Lox71</I> and <I>Lox66</I> sites are cleaved by a CRE recombinase, excising them and  all the DNA sequences located between them. In Paris model, the Cre recombinase gene is under the control of <I>dapA</I> promoter. (for more information about Paris 2007 project, access the link: <A href="https://2007.igem.org/Paris" class="external free" rel="nofollow">https://2007.igem.org/Paris</A>)  
The Caltech 2008 team inserted the repetitive sequence AGCT10 between the start codon of the <I>GFP</I> gene and the rest of its ORF.  In this case, the ORF is not in frame with its start codon, so the <I>GFP</I> is not corrected translated. However, DNA polymerase may slip when it replicates repetitions of small nucleotide polymers. So, during the replication of this construction, in some cases the AGCT10 is replaced by AGCT9. In this case, the <I>GFP</I> ORF is in frame with its start codon, and the protein is corrected translated.
In our model, we aimed to insert two constructions flanking the <I>FSTK</I> gene; a) an upstream construction: <I>LOX71-GFP</I>; and b) a downstream construction: <I>FinO-FinP-Lox66</I>. The presence of <I>finO</I> and <I>FinP</I> genes impede the cell to conjugate. Moreover, we aimed to insert the repetition AGCT10 between the Cre recombinase start codon and the rest of its ORF; when Cre recombinase ORF is in frame with the start codon, it will be translated and act on the <I>Lox</I> based construction, excising the <I>FSTK</I>, <I>FinO</I> and <I>FinP</I> genes; so, the cell will lose its division skills, however it will be able to conjugate.
</P><H2><SPAN class="mw-headline" id="Constructions">Constructions</SPAN></H2><P>The constructions we´ve developed to this part of the project are described below.
</P><P>1)  <A href="/wiki/index.php?title=BBa_K284043&amp;action=edit&amp;redlink=1" class="new" title="BBa K284043 (page does not exist)">BBa_K284043</A>: AGTC repetition and <I>CRE</I> recombinase under constitutive promoter
	After the amplification of <I>CRE</I> recombinase by PCR, we cloned it into the PSB1A3 vector, according to the standard assembly strategy, resulting in the biobrick BBa_K284031. The following steps were also based on the standard assembly strategy, joining the biobricks BBa_R0040 (pTet promoter), BBa_284040 (AGCT repetition), BBa_284031 (Cre recombinase without the start codon) and BBa_B0015 (double terminator), resulting in BBa_K284043.
</P><P>2) BBa_K284090: Upstream construction flanking <I>FSTK</I> gene
	This construction was developed to be inserted in the upstream region of the <I>FSTK</I> gene. Lox 71 is a cleavage site for <I>CRE</I> recombinase. This biobrick was assembled according to the standard protocol, joining the previously existing biobricks BBa_I718017 (Lox 71), BBa_R0040 (pTet promoter), RBS (BBa_B0030), BBa_E0040 (green fluorescent protein ORF) and BBa_B0015 (double terminator).
</P><P>
3) Downstream construction flanking the <I>FSTK</I> gene
</P><P>This construction is a composite part formed by the biobricks BBa_K284060, BBa_K284071 and BBa_I718017.
</P><P>3.1) <A href="/wiki/index.php?title=BBa_K284060&amp;action=edit&amp;redlink=1" class="new" title="BBa K284060 (page does not exist)">BBa_K284060</A>: <I>FinP</I> under constitutive promoter
</P><P>
After the amplification of <I>FinP</I> (from <I>E. coli</I> F plasmid) by PCR, we cloned it into the PSB1A3 vector, according to the standard assembly strategy, resulting in the biobrick BBa_K284033. The following steps were also based on the standard assembly strategy, joining the biobricks BBa_R0040 (pTet promoter), BBa_B0030 (RBS), BBa_284033 (<I>FinP</I> form F plasmid) and BBa_B0015 (double terminator), resulting in BBa_K284060.
</P><P>3.2) <A href="/wiki/index.php?title=BBa_K284071&amp;action=edit&amp;redlink=1" class="new" title="BBa K284071 (page does not exist)">BBa_K284071</A>: FinO under constitutive promoter
After the amplification of FinO (from E. coli R100 plasmid) by PCR, we cloned it into the PSB1A3 vector, according to the standard assembly strategy, resulting in the biobrick BBa_K284032. The following steps were also based on the standard assembly strategy, joining the biobricks BBa_R0040 (<I>pTet</I> promoter), BBa_B0030 (<I>RBS</I>), BBa_284032 (<I>FinO</I> form R100 plasmid) and BBa_B0015 (double terminator), resulting in BBa_K284071.
</P><P>
3.3) <A href="/wiki/index.php?title=BBa_K284080&amp;action=edit&amp;redlink=1" class="new" title="BBa K284080 (page does not exist)">BBa_K284080</A>: Downstream construction flanking <I>FSTK</I> gene
This construction was developed to be inserted in the downstream region of the <I>FSTK</I> gene. <I>Lox 66</I> is a cleavage site for <I>CRE</I> recombinase. This biobrick was assembled according to the standard protocol, joining the biobrick BBa_K284060 (<I>FinP</I> under constitutive promoter), BBa_K284071 (<I>FinO</I> under constitutive promoter) and BBa_I718016 (<I>Lox 66</I>).
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>1. <A href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=mesh" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/sites/entrez?db=mesh</A></P><P>2. Weismann, A. The Germ Plasm. 1892
</P><P>3. <A href="https://2007.igem.org/Paris" class="external free" rel="nofollow">https://2007.igem.org/Paris</A></P><P>4. <A href="https://2008.igem.org/Team:Caltech" class="external free" rel="nofollow">https://2008.igem.org/Team:Caltech</A></P><P>5. Zhang, Z. and Lutz, B. Nucleic Acid Research. 2002, 30 (17), 1-5.
</P><P>6. Cao, J.G. &amp; Meighen, E.A. J Biol Chem. 1989, 264, 21670-21676. 
</P><P>7. Miller, S.T. et al. Mol Cell. 2004, 15, 677-687. 
</P><P>8. Miller, M. B.; Bassler, B. L. Annu. Rev. Microbiol. 2001, 55, 165-199.
</P><P>9. Gubbins, M.J et al. J. Biol Chem. 2003, 278, 27663–27671.
</P><P>10. Van Biesen, T. et al. Mol Microbiology. 1993, 10(1), 35-43.
</P><P>11. Koraimann, G. et al. Mol. Microbiology. 1996, 21, 811-821.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UNICAMP-Brazil/Coliguard/Differentiation">http://2009.igem.org/Team:UNICAMP-Brazil/Coliguard/Differentiation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNICAMP-Brazil/Coliguard/Differentiation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNICAMP-Brazil/Coliguard/Differentiation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Differentiation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNICAMP-Brazil/Coliguard/Differentiation&amp;oldid=167017" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>