Team:William and Mary/Measurement
Toggle navigation
Project
Background
Results
Temperature Systems
Methods and Data
3G Assembly
Method Overview
Collaborations
Parallel Circuit Testing
3G Parts
Protocol
Modeling
Human Practices
Overview
Incorporating New Communities
A Statewide Standard
Continuing the Conversation
IHP/Silver
Judging
Medal Requirements
Measurement
Demonstrate
Best Part Collection
Parts
Improve
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Page Title
Measurement
Overview
This year, our project focused on measuring the dynamical outputs of our decoding circuit. In the process we performed single cell time series measurements and developed novel protocols for the removal of small molecule inducers . Although single cell measurements using flow cytometry are our preferred way to obtain data, as our project developed, it became clear that much higher throughput would be required. As such, we spent a major portion of our project creating methods to test qualitative circuit behavior using a plate reader.  Throughout the many design build test cycles our decoding system required, we have created many advances in measurement protocols that we hope will be useful to other teams working on complex and dynamic circuits. Additionally, we attempted to follow best practices for data availability and measurement by displaying our graphs in a manner that shows the underlying data, as well as providing full data (in .csv format) and experimental methods for every plot shown on the wiki .
Measurement Techniques
We contributed several novel measurement techniques to iGEM this year. Firstly, we introduced to iGEM 3G Assembly , a new method of DNA assembly that will allow teams to easily create circuits with a wide variety of different parameters. This should not only serve to enhance future team's abilities to clone constructs, but should increase the ability of teams to characterize the properties of existing registry parts. Further, since this method is mostly compatible with CIDAR MoClo [1], we do not complicate characterization and standardization efforts by adding an additional and unnecessary standard. Indeed, we demonstrated this potential by collaborating with Virginia iGEM to characterize two commonly used ribosome binding sites.  Building off of this work, we also devised a way to utilize 3G to test different circuit design variants in a low cost, high throughput manner that requires no specialized equipment outside of a plate reader. Secondly, we have innovated in measurement by validating and using a protocol that allows for the ability to dynamically change the concentration of a small molecule inducers .
Lastly, we realized that many teams were unable to perform flow cytometry measurements due to lack of equipment or due to the prohibitive hourly costs associated with the use of their institutions' flow cytometer. This issue is further exacerbated during time course measurements, as experiments often last multiple hours, which results in large financial and time costs. To ameliorate this situation, we designed, tested, and validated a flow cytometry protocol that preserves cells for later measurement without changing their fluorescence data (Figure 1). This procedure can be performed with nothing more than a -80C freezer and glycerol, enabling teams with limited resources to partner with more resourced teams to obtain single cell characterization of their projects, as well as enabling teams performing time course measurements to have a lower cost way to obtain their crucial single cell measurements.
Figure 1: Single cell fluorescence measurements of BBa_K2333428 time course live (NonFrozen) and 3 days later (Frozen). Each dot represents the geometric mean of at least 10,000 single cell measurements each of 3 biological replicates, the shaded region represents one geometric standard deviation above and below the geometric mean.
Data Visualization and Availability
One crucial aspect of engineering disciplines is that data is displayed openly and in an unbiased fashion. To that end, we displayed data in the appropriate fashion, using univariate scatter plots instead of bar graphs, and clearly displayed our data in a manner that allows the reader to see the level of variation present in our graphs [2]. As gene expression is log normally distributed [3], we made sure to use the geometric, rather than regular, means and standard deviations on our graphs, and made sure to use the appropriate y axis scale. Lastly, we attempt to provide a model for data availability by providing both the full data (in .csv or .fcs) format as well as detailed experimental methods for every graph on our wiki on a single page .
References
[1] CIDAR MoClo: Improved MoClo Assembly Standard and New E. coli Part Library Enable Rapid Combinatorial Design for Synthetic and Traditional Biology. Sonya V. Iverson, Traci L. Haddock, Jacob Beal, and Douglas M. Densmore. ACS Synthetic Biology 2016 5 (1), 99-103 DOI: 10.1021/acssynbio.5b00124
[2] Weissgerber TL, Milic NM, Winham SJ, Garovic VD. Beyond bar and line graphs: time for a new data presentation paradigm. PLoS biology. 2015 Apr 22;13(4):e1002128.
[3] Beal J. Biochemical complexity drives log-normal variation in genetic expression. Engineering Biology. 2017 Jul 11;1(1):55-60.
Top
Contact us at igem@wm.edu
Sponsors
2018 William & Mary iGEM
