<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_3GProtocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/3GProtocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="dropdown"><NAV style="background-color: #FFFFFF" class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header" style="float: left;"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse"><UL id="wnm-top-nav" class="nav navbar-nav navbar-left navbar-uppercase dropdown"><LI><A href="https://2018.igem.org/Team:William_and_Mary/Description">
                Project
              </A></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Description"> 
                 Background
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Results"> 
                 Results
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Heat"> 
                 Temperature Systems
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Experiments"> 
                 Methods and Data
                </A></CENTER></LI></UL><LI><A href="https://2018.igem.org/Team:William_and_Mary/3G"> 
              3G Assembly
            </A></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/3G"> 
                 Method Overview
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Collaborations"> 
                Collaborations
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/3G_Mixed"> 
                 Parallel Circuit Testing
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Part_Collection"> 
                3G Parts
               </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/3GProtocols"> 
                Protocol
                </A></CENTER></LI><LI><A href="https://2018.igem.org/Team:William_and_Mary/Model">
                Modeling
              </A></LI><LI><A href="https://2018.igem.org/Team:William_and_Mary/Public_Engagement"> 
              Human Practices
            </A></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Public_Engagement"> 
                  Overview
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Human_Practices/Incorporating_new_communities"> 
                  Incorporating New Communities 
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Human_Practices/A_Statewide_Standard"> 
                  A Statewide Standard
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Human_Practices/Continuing_the_Conversation"> 
                  Continuing the Conversation
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Human_Practices"> 
                  IHP/Silver
                </A></CENTER></LI><LI><A href="#"> 
                Judging
              </A></LI><LI style="font-family: &quot;Optima&quot;;"><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Medal Requirements"> 
                  Medal Requirements
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Measurement"> 
                  Measurement
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Demonstrate"> 
                  Demonstrate
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Part_Collection"> 
                  Best Part Collection
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Parts"> 
                  Parts
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Improve"> 
                 Improve
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/InterLab"> 
                  Interlab
                </A></CENTER></LI><LI><A href="https://2018.igem.org/Team:William_and_Mary/Team"> 
              Team
            </A></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Team"> 
                  Meet the Team
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Attributions"> 
                  Attributions
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Notebook"> 
                  Notebook
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Protocols"> 
                 Protocols
                </A></CENTER></LI><LI><CENTER><A style="color: black" href="https://2018.igem.org/Team:William_and_Mary/Safety"> 
                  Safety
                </A></CENTER></LI></DIV></DIV></NAV></DIV><TITLE>Page Title</TITLE><H1 style="color:black;text-align:center;">3G Assembly Protocols</H1><DIV style="padding-left: 8%; padding-bottom: 10px;font-size: 25px"><B>Overview</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
3G assembly is a new and cutting edge hybrid method of DNA assembly first described in 2018 by A. D. Halleran, A. Swaminathan and R. M, Murray [1] which combines Golden Gate and Gibson assembly techniques to allow for modular assembly of multi-part circuits in a single day. In this method circuits are composed of modular transcriptional units which are constructed using Golden Gate Assembly. These transcriptional units are amplified with PCR, purified via gel extraction, then combined into a circuit using Gibson Assembly. 
</DIV><CENTER><FIGURE style="padding-left: px;"><FIGCAPTION><DIV style="padding-left: 20%;padding-right:20%; padding-top: 15px; color: #808080; font-size: 14px;">
Figure 1: Overview of 3G workflow
</DIV></FIGCAPTION></FIGURE></CENTER><DIV style="padding-left: 8%; padding-bottom: 10px;font-size: 25px"><B>Design</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
A circuit is composed of transcriptional units. Each transcriptional unit consists of a promoter, a 5’ untranslated region (UTR), a coding sequence (CDS), and a terminator in that order. Each of these four parts have distinct sticky ends according to the Cidar Modular Cloning System, which can be seen in the image below:
</DIV><CENTER><FIGURE style="padding-left: px;"><FIGCAPTION><DIV style="padding-left: 20%;padding-right:20%; padding-top: 15px; color: #808080; font-size: 14px;">
Figure 2: Schematic of MoClo Sticky Ends of promoters, 5' UTRs, CDSs and terminators
</DIV></FIGCAPTION></FIGURE></CENTER><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
During the Golden Gate stage, the restriction enzyme BsaI cuts inside of its recognition site to reveal each part’s sticky ends so that they can be ligated together in the correct order. In addition, unique nucleotide sequences (UNS) are attached at the 5’ and 3’ end of the transcriptional unit using oligo adapters. The UNS on the 5’ end must have an A sticky end so that it can attach to the promoter, and the UNS on the 3’ end must have an E sticky end to attach to the terminator. 
</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
There are a variety of UNS sequences that can be used. In order for the transcriptional units to attach to the backbone in the Gibson stage, the first unit to go on the circuit must begin with UNS 1, and the last unit on the circuit must end with UNS 10. When creating transcriptional units that are intended to end up on a circuit together, it is important to ensure that their UNSs overlap. For instance, if the first unit is flanked by UNS 1 and UNS 3, the second must be flanked with UNS 3 and UNS 10. 
</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
In the next step, transcriptional units are amplified with PCR. The forward primer will anneal to the 5’ UNS and the reverse primer will anneal to the 3’ UNS. The PCR products are then gel extracted. 
</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
In the Gibson stage, transcriptional units are attached to each other and to a backbone. The backbone must have UNS 1 and UNS 10 inside the BioBrick Prefix and Suffix in order for the transcriptional units to attach. As mentioned earlier, the UNSs on each transcriptional unit determine the order they end up in, as shown below. Up to six transcriptional units can be put on a backbone in 3G assembly. 
</DIV><CENTER><FIGURE style="padding-left: px;"><FIGCAPTION><DIV style="padding-left: 20%;padding-right:20%; padding-top: 15px; color: #808080; font-size: 14px;">
Figure 3: Gibson stage of 3G: Transcriptional units attach to the backbone at UNS 1 and UNS 10 and to each other at overlapping UNS
</DIV></FIGCAPTION></FIGURE></CENTER><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
Transcriptional units are either supplied from the iGEM registry or can be designed. They may need to be cloned on a backbone in order to increase volume. Oligo adapters are then added. 
</DIV><DIV style="padding-left: 8%; padding-bottom: 10px;font-size: 25px"><B>Protocol</B></DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Preparation of 50 nM Oligo Adapters (UNS)</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
Single stranded lyophilized oligos (see table in Additional Information) are resuspending in IDT duplex buffer to a concentration of 100 uM. Top and bottom oligos are combined in a 1:1 molar ratio in duplex buffer to a final concentration of 1µM and then annealed by heating to 98C for 1 minute and then cool to room temperature on the benchtop. 50 nM working stocks are created by a 1:20 dilution of the 1µM stock in NFW. 
</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Preparation of UNS1/10 iGEM Backbones</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
The UNS1/10 iGEM backbones can be prepared by a standard PCR. To a 0.2 mL tube, add:</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>1.25 µL primer BBa_K2680513</LI><LI>1.25 µL primer BBa_K2680514 </LI><LI>9 µl NFW, 1 ul of a miniprep of either pSB1C3 or pSB3K3</LI><LI>12.5 µl of Q5 2x Master Mix.</LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
Program the thermocycler with standard PCR cycles with a 55 C annealing temperature and a 1 minute extension time for pSB1C3, or a 1 minute 30 second extension time for  pSB3K3. Gel extract the PCR product.</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Preparation of 30 nM 3G parts</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
Quantify the concentration of the miniprepped 3G part with a nanodrop, convert to molarity, then dilute to 30 nM in nuclease free water. 
</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Golden Gate Assembly</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
To a 0.2 mL tube, add the following:</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>0.5 µL promoter 30 nM miniprep stock</LI><LI>0.5 µL 5’ UTR 30 nM miniprep stock</LI><LI>0.5 µL CDS 30 nM miniprep stock</LI><LI>0.5 µL terminator 30 nM miniprep stock</LI><LI>0.5 µL 50 nM left UNS adapter</LI><LI>0.5 uL 50 nM right UNS adapter</LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
In addition, either add:</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>0.5 µL 10x DNA ligase buffer</LI><LI>0.25 µL BsaI</LI><LI>0.25 µL T4 DNA Ligase</LI><LI>1 µL NFW</LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
OR</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>0.5 µL Golden Gate Buffer</LI><LI>0.25 µL NEB Golden Gate Master Mix</LI><LI>1.25 µL NFW</LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
Note: if performing multiple Golden Gate reactions, we recommend making a <B>master mix</B> with a slight upscale, then add 2 µL of the master mix to reaction.
Cycle the reaction in the thermocycler. For the Golden Gate Master Mix perform 30 cycles of incubating at 37 for 1 minute then incubating at 16 C for one minute. For the T4 ligase and BsaI, perform 9 cycles at 37C for 3 minutes followed by incubating at 16 C for 4 minutes. For both options, hold at 37 C for five minutes, then hold at 4 C. 
For information on creating multiple circuit variants at once, see <A href="https://2018.igem.org/Team:William_and_Mary/3G_Mixed" style="color:green">parallel 3G Assembly</A>.
</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>PCR Amplification</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
Amplify the product with a 50 uL PCR as according to the PCR protocol. Add: </DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>1.5 µL Golden Gate assembly product</LI><LI>18.5 µL NFW</LI><LI>2.5 µL of the appropriate 10uM UNS_foward Primer (that will anneal to the UNS on the 5’ end of your transcriptional unit)</LI><LI>2.5 µL 10 uM of the appropriate UNS_reverse primer (that will anneal to the UNS on the 3’ end of your transcriptional unit)</LI><LI>25 µL 2x Hot Start Master Mix </LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
Incubate reactions using the standard 2x Q5 Hot Start Master Mix protocol with 27 cycles, using 30 seconds per kilobase pair, and an annealing temperature of 64 C. Conduct a gel extraction on the PCR product. (Note that 3G often produces off target bands as well as an extremely bright on target band. This is normal, and is why gel extraction rather than PCR purification is recommended). 
</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Gibson Assembly</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
Perform a 5 uL Gibson assembly using the purified transcriptional units and vectors. Add to a .2 mL tube:</DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 0px;line-height: 25px;"><UL><LI>.015 pmolar each fragment</LI><LI>.015 pmolar desired backbone</LI><LI>Add NFW to a total volume of 2.5 µL.</LI><LI>2.5 ul 2X HiFi DNA Assembly master mix</LI></UL></DIV><DIV style="padding-right: 14%; padding-left: 14%; line-height: 25px;">
Ensure the total fragment volume of the PCR product and backbone does not exceed 2.5 µl. Place tube in a thermocycler, run for 50 C for an hour, and hold at 4 C. Proceed to a transformation fairly quickly as Gibson products are only stable for a few hours.
</DIV><DIV style="padding-left: 8%; padding-bottom: 10px;font-size: 25px"><B>Additional Information</B></DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B>Designing New 3G Parts</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
To design a new 3G part, use Gene Block to design a new sequence of double stranded DNA. Include a 40 bp pad sequence on either end, along with a BsaI site and sticky ends appropriate for the part type (for more information on pad sequences, see <A href="https://2018.igem.org/Team:William_and_Mary/3G#PAD" style="color:green">3G Assembly</A>). This allows the new part to be cloned onto a William and Mary pad backbone. 
</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B> Troubleshooting</B></DIV><DIV style="padding-right: 14%; padding-left: 14%; text-indent: 50px;line-height: 25px;">
In general, when 3G assembly fails, it will fail at the gel extraction phase. If correctly sized DNA is obtained from the gel, the Gibson reaction works very well. For circuits that fail to amplify at the PCR step, we recommend running an increased number of golden-gate cycles (100 or 30 for NEB and BsaI respectively). Additionally, use of longer (40 bp), full length UNS amplification primers may be helpful for amplification. 

</DIV><DIV style="padding-left: 14%; padding-bottom: 10px;font-size: 25px"><B> Sequences</B></DIV><CENTER><TABLE style=""><CENTER><CAPTION>Oligo Adapter Sequences</CAPTION></CENTER><TBODY><TR><TH style="background-color: #65A686;column-width: 150px;"><CENTER>Oligo Adapter</CENTER></TH><TH style="background-color: #65A686;column-width: 150px;"><CENTER>Sequence</CENTER></TH></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">pOSIP A Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCCCCGGGTACCGAGCTCGAATTCCGATCCCCAATTGGCGTC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">pOSIP A Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GACGCCAATTGGGGATCGGAATTCGAGCTCGGTACCCGGGGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">pOSIP E Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">AGGCGCCATGCATCTCGAGGCATGCCTGCAGCGGCCGCTAAAGCAGAGACCCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">pOSIP E Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGGGTCTCTGCTTTAGCGGCCGCTGCAGGCATGCCTCGAGATGCATGGCGCCT</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS1_A_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCGAGACGAGACGAGACAGCCTGAGAATGGATGCGAGTAATG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS1_A_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CATTACTCGCATCCATTCTCAGGCTGTCTCGTCTCGTCTCGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3_A Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCCGACCTTGATGTTTCCAGTGCGATTGAGGACCTTCAGTGC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3_A_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GCACTGAAGGTCCTCAATCGCACTGGAAACATCAAGGTCGGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3_E_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CGACCTTGATGTTTCCAGTGCGATTGAGGACCTTCAGTGCAAGCAGAGACCCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3_E_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGGGTCTCTGCTTGCACTGAAGGTCCTCAATCGCACTGGAAACATCAAGGTCG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4_A_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCGACTTTGCGTGTTGTCTTACTATTGCTGGCAGGAGGTCAG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4_A_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTGACCTCCTGCCAGCAATAGTAAGACAACACGCAAAGTCGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4_E_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GACTTTGCGTGTTGTCTTACTATTGCTGGCAGGAGGTCAGAAGCAGAGACCCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4_E_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGGGTCTCTGCTTCTGACCTCCTGCCAGCAATAGTAAGACAACACGCAAAGTC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5_A_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCCTCTAACGGACTTGAGTGAGGTTGTAAAGGGAGTTGGCTC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5_A_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GAGCCAACTCCCTTTACAACCTCACTCAAGTCCGTTAGAGGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5_E_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTCTAACGGACTTGAGTGAGGTTGTAAAGGGAGTTGGCTCAAGCAGAGACCCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5_E_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGGGTCTCTGCTTGAGCCAACTCCCTTTACAACCTCACTCAAGTCCGTTAGAG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS6_A_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CCGGTCTCTCTCCGTATGTGACCGTAGAGTATTCTTAGGTGGCAGCGAACGAG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS6_A_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTCGTTCGCTGCCACCTAAGAATACTCTACGGTCACATACGGAGAGAGACCGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS10_E_Bottom</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGTGGAAGGGCTCGGAGTTGTGGTAATCTATGTATCCTGGAAGCAGAGACCCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS10_E_Top</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGGGTCTCTGCTTCCAGGATACATAGATTACCACAACTCCGAGCCCTTCCACC</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE style=""><CENTER><CAPTION>Primers for Amplification of Transcriptional Units</CAPTION></CENTER><TBODY><TR><TH style="background-color: #65A686;column-width: 150px;"><CENTER>UNS Amplification Primer</CENTER></TH><TH style="background-color: #65A686;column-width: 150px;"><CENTER>Sequence</CENTER></TH></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS1 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CATTACTCGCATCCATTCTCAGGC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GCACTGAAGGTCCTCAATCG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTGACCTCCTGCCAGCAATAGT</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GAGCCAACTCCCTTTACAACCT</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS6 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTCGTTCGCTGCCACCTAAGAA</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS7 Forward</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CAAGACGCTGGCTCTGACATTT</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS3 Reverse</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CGACCTTGATGTTTCCAGTGCG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS4 Reverse</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GACTTTGCGTGTTGTCTTACTAT</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS5 Reverse</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">CTCTAACGGACTTGAGTGAGGTTG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS6 Reverse</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GTATGTGACCGTAGAGTATTCTTAGGTGG</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">UNS10 Reverse</TD><TD style="background-color: #96B69C;column-width: 150px; font-size: 15px;">GGTGGAAGGGCTCGGAGTTG</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE style=""><CENTER><CAPTION>Primers for Creation of UNS Backbones</CAPTION></CENTER><TBODY><TR><TH style="background-color: #65A686;column-width: 150px;"><CENTER>Primer</CENTER></TH><TH style="background-color: #65A686;column-width: 150px;"><CENTER>Sequence</CENTER></TH></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">Biobrick Suffix Fwd, UNS10 Overhang </TD><TD style="background-color: #96B69C;column-width: 300px; font-size: 15px;">CCAGGATACATAGATTACCACAACTCCGAGCCCTTCCACCTACTAGTAGCGGCC</TD></TR><TR><TD style="background-color: #A2CBAB;column-width: 150px; font-size: 15px;">Biobrick Prefix Rev, UNS1 Overhang</TD><TD style="background-color: #96B69C;column-width: 300px; font-size: 15px;">GAGACGAGACGAGACAGCCTGAGAATGGATGCGAGTAATGCTAGAAGCGGCC</TD></TR></TBODY></TABLE></CENTER><DIV class="references"><DIV style="padding-left: 8%; padding-bottom: 10px;font-size: 25px"><B>References</B></DIV><DIV style="padding-right: 190px; padding-left: 190px; text-indent: 0px;line-height: 25px;">[1] Single Day Construction of Multigene Circuits with 3G Assembly
Andrew D. Halleran, Anandh Swaminathan, and Richard M. Murray. ACS Synthetic Biology 2018 7 (5), 1477-1480. DOI: 10.1021/acssynbio.8b00060
</DIV></DIV><FOOTER class="footer footer-color-black" data-color="black"><DIV class="container" style="padding-bottom: 0px;"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
            Top
          </A></LI><LI><A>
            Contact us at igem@wm.edu
          </A></LI></UL></NAV><NAV><DIV class="social-area pull-right"><FONT size="3" face="Helvetica"><CENTER><FONT color="grey">Sponsors</FONT></CENTER></FONT></DIV></NAV><DIV class="copyright">

      2018 William &amp; Mary iGEM
    </DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>