Team:William and Mary/Experiments
Toggle navigation
Project
Background
Results
Temperature Systems
Methods and Data
3G Assembly
Method Overview
Collaborations
Parallel Circuit Testing
3G Parts
Protocol
Modeling
Human Practices
Overview
Incorporating New Communities
A Statewide Standard
Continuing the Conversation
IHP/Silver
Judging
Medal Requirements
Measurement
Demonstrate
Best Part Collection
Parts
Improve
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Page Title
Methods & Data
Overview
In the interest of promoting reproducible science, and enabling others to build off of our work, we have included both the precise methods used as well as the raw data (in csv format) for every experiment mentioned on our wiki. Experiments are split up by the page on which they are mentioned. For experiments performed with flow cytometry, .fcs files for both measurements and calibration beads (Spherotek RCP-30-5A, lot number: AB01) are included in the file. In all cases at least 10,000 single cell measurements were collected for flow cytometry data.  Whenever used, the term biological replicates refers to a distinct colony from a transformation of the same miniprep. Whenever plate reader fluorescence data is provided, it is given in arbitrary units of fluorescence normalized to cell density (recorded in either Absorbance 600 or Absorbance 700).
For all experiments, unless stated otherwise, the following is true:
Circuits used were on psb1C3
The E. coli strain used was JS006 (K-12 derivative with ΔAraCΔLacI)
The media used was M9 minimal media with 0.4% Casamino Acids and 0.4% Glucose (see protocols)
Antibiotics in minimal media were used in the following concentrations [Chloramphenicol: 7.5mg/mL, Kanamycin: 35mg/mL]
Transformations were plated on selective LB plates with either 25mg/mL Chloramphenicol, 50mg/mL Kanamycin or both
96 well plates used with the microplate reader (Synergy H1 [Biotek]) were Black Griener Fluotrack 200 flat bottom plates with the matching lid
Microplate reader was set to linear shaking at max speed and gradient temperatures were used to prevent condensation
Excitation/emission used for plate reader experiments was 565/593
The channel used for flow cytometry experiments was Fl2 (S3e cell sorter [Biorad]).
Chemical Inducible System
Figure 1: Fluorescence/OD600 (AU) measurements of the chemically induced circuits BBa_K2333427 and BBa_K2333434 when grown at 37C and then chemically induced. Dots represent the geometric mean of 3 distinct biological replicates (colonies) and the blue shaded region represents one geometric standard deviation above and below the mean. Data
Figure 2: Frozen experiment. Data
Figure 3: In both cases colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 37C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then subjected to their conditions. For the data shown, those conditions were 50ng/mL ATC and 0.1mM IPTG. Data
Figure 4: Cells were grown at 37C in 50ng/mL ATC to induce fluorescence. At timepoint 120, the cells were split into two groups. One group was spun down and washed in media with 50ng/mL ATC, and the other group was washed in media without ATC. The cells were then left to grow again at 37C Data
Figure 5: In both cases colonies were were grown at 37 C in m9 medium with 50ng/mL ATC and 0.1mM IPTG. At timepoint 40, the cells were spun down and washed in fresh m9 medium 3 times. At timepoint 80, the cells were spun down and resuspended in 50ng/mL ATC and 0.1mM IPTG m9 medium. Timepoints were taken every 20 minutes by pipetting cells into glycerol and freezing them in liquid nitrogen. Data
Heat Inducible System
Figure 2: In both cases colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 29C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then subjected to their conditions. For A, this was a 37C for the entire period, whereas for B this was 37C from 160 minutes to 280 minutes and 29C elsewhere. Data
Figure 3: In both cases colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 29C until cultures showed OD700s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then subjected to their conditions. For A, this was a 37C for the entire period, whereas for B this was 37C from 140 minutes to 280 minutes and 29C elsewhere. Data
Figure 4: In both cases colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 29C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then grown at 37C for 1100 minutes. Data
Figure 5: In all 3 cases colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 30C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then grown at 39C until OD600 plateau was reached. Data
Figure 6: Different pdt constructs (1C3) were co-transformed with BBa_K2680061 on psb3K3. Colonies were  were picked into 200µL of media into a 96 well plate and cultures were grown at 30C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then grown at 39C until OD600 plateau was reached. Data
Figure 7: K2333428 (1C3) was co-transformed with different minipreps of K2680060 and K2680061 on psb3K3. Colonies were  were picked into 3mL of media and grown overnight at 30C. Cultures were then diluted 1:100 in the morning, grown for 3 hours and then 50ng/mL ATc was added. Cells were grown 4 hours and measured via flow cytometry. At the same time, cells were 1:100 diluted into fresh media and grown for 2 hours at 39C before being measured. Part1 and Part2 .
Figure 8: K2333428 (1C3) was transformed into JS006 cells containing the integrated constructs. 6 colonies of each transformation were then picked into 200µL of media into a 96 well plate and cultures were grown at 30C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05, and ATc at 10ng/mL and 25ng/ml was added to 3 colonies of each construct. Cells were then grown at 39C until OD600 plateau Data .
Figure 9: 3 colonies were picked into 200µL of media into a 96 well plate and cultures were grown at 30C until cultures showed OD600s consistent with mid-log growth. Cultures were then diluted to an OD600 of ~.05 and then grown at 39C until OD600 plateau was reached. Data .
Collaboration
mScarlet-I RBS characterization: Circuits composed of J23107 B0034/B0032 mScarlet-I B0015 as well as a J23107 B0034 LacI B0015 control were cloned via 3G assembly onto psb1C3. 3 colonies of each circuit were picked into LB media, grown for 3 hours at 37C, diluted to OD600 0.1 and grown 4 hours. Then circuits were measured on UVAs plate reader (with emission/excitation 565/593 and absorbance 600). Data was then normalized with respect to cell density. Data
Flow Cytometry Protocol
Frozen vs Non Frozen cell tests: 3 colonies of cells with BBa_K2333428 (pTet mScarlet-I pdtA) were picked into 4mL of M9 media and incubate at 37C overnight. In the morning, cells were diluted 1:100 and grown for 3 hours at 37C. Cells were diluted to a final OD600 of ~.05 in 5mL of 37C M9 and ATc was added to a final concentration of 50ng/mL. Cells were grown at 37C and 200µL time points were taken every 20 minutes (t0= immediately following inducer being added) onto ice. 50µL was filtered into ice cold PBS and measured on the FACSed immediately, while the remaining 150µL had glycerol added to a final concentration of 15%, and were then flash frozen in liquid nitrogen. Experiment was continued for 160 minutes to ensure cells remained in midlog growth. Frozen samples were stored at -80C for 3 days and then thawed on ice, 1:10 diluted into ice cold PBS and then FACSed. Data . FACS files ( Day1 and Day2 )
Mixed 3G Assembly
Different transcriptional units were assembled using the parts listed on the mixed 3G page in equimolar volumes. Circuit mixture was then transformed into NEB 5-alpha cells and allowed to grow overnight at 30C. In the morning, 1 colony was picked into each well of a 96 well plate in 200µL of media. The plate was incubated at 30C until midlog growth was reached and colonies were then diluted 1:4 and grown at 37C for the remainder of the experiment. The full plate reader data can be found here
Sequencing
All Sanger sequencing was performed by Epoch Life Sciences. All constructs submitted to the registry as well as many non submitted constructs were sequenced. A key for William and Mary circuit names as well as .ab1s can be found in the two part file here and here .
Top
Contact us at igem@wm.edu
Sponsors
2018 William & Mary iGEM
