Team:William and Mary/3GProtocols
Toggle navigation
Project
Background
Results
Temperature Systems
Methods and Data
3G Assembly
Method Overview
Collaborations
Parallel Circuit Testing
3G Parts
Protocol
Modeling
Human Practices
Overview
Incorporating New Communities
A Statewide Standard
Continuing the Conversation
IHP/Silver
Judging
Medal Requirements
Measurement
Demonstrate
Best Part Collection
Parts
Improve
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Page Title
3G Assembly Protocols
Overview
3G assembly is a new and cutting edge hybrid method of DNA assembly first described in 2018 by A. D. Halleran, A. Swaminathan and R. M, Murray [1] which combines Golden Gate and Gibson assembly techniques to allow for modular assembly of multi-part circuits in a single day. In this method circuits are composed of modular transcriptional units which are constructed using Golden Gate Assembly. These transcriptional units are amplified with PCR, purified via gel extraction, then combined into a circuit using Gibson Assembly.
Figure 1: Overview of 3G workflow
Design
A circuit is composed of transcriptional units. Each transcriptional unit consists of a promoter, a 5’ untranslated region (UTR), a coding sequence (CDS), and a terminator in that order. Each of these four parts have distinct sticky ends according to the Cidar Modular Cloning System, which can be seen in the image below:
Figure 2: Schematic of MoClo Sticky Ends of promoters, 5' UTRs, CDSs and terminators
During the Golden Gate stage, the restriction enzyme BsaI cuts inside of its recognition site to reveal each part’s sticky ends so that they can be ligated together in the correct order. In addition, unique nucleotide sequences (UNS) are attached at the 5’ and 3’ end of the transcriptional unit using oligo adapters. The UNS on the 5’ end must have an A sticky end so that it can attach to the promoter, and the UNS on the 3’ end must have an E sticky end to attach to the terminator.
There are a variety of UNS sequences that can be used. In order for the transcriptional units to attach to the backbone in the Gibson stage, the first unit to go on the circuit must begin with UNS 1, and the last unit on the circuit must end with UNS 10. When creating transcriptional units that are intended to end up on a circuit together, it is important to ensure that their UNSs overlap. For instance, if the first unit is flanked by UNS 1 and UNS 3, the second must be flanked with UNS 3 and UNS 10.
In the next step, transcriptional units are amplified with PCR. The forward primer will anneal to the 5’ UNS and the reverse primer will anneal to the 3’ UNS. The PCR products are then gel extracted.
In the Gibson stage, transcriptional units are attached to each other and to a backbone. The backbone must have UNS 1 and UNS 10 inside the BioBrick Prefix and Suffix in order for the transcriptional units to attach. As mentioned earlier, the UNSs on each transcriptional unit determine the order they end up in, as shown below. Up to six transcriptional units can be put on a backbone in 3G assembly.
Figure 3: Gibson stage of 3G: Transcriptional units attach to the backbone at UNS 1 and UNS 10 and to each other at overlapping UNS
Transcriptional units are either supplied from the iGEM registry or can be designed. They may need to be cloned on a backbone in order to increase volume. Oligo adapters are then added.
Protocol
Preparation of 50 nM Oligo Adapters (UNS)
Single stranded lyophilized oligos (see table in Additional Information) are resuspending in IDT duplex buffer to a concentration of 100 uM. Top and bottom oligos are combined in a 1:1 molar ratio in duplex buffer to a final concentration of 1µM and then annealed by heating to 98C for 1 minute and then cool to room temperature on the benchtop. 50 nM working stocks are created by a 1:20 dilution of the 1µM stock in NFW.
Preparation of UNS1/10 iGEM Backbones
The UNS1/10 iGEM backbones can be prepared by a standard PCR. To a 0.2 mL tube, add:
1.25 µL primer BBa_K2680513
1.25 µL primer BBa_K2680514
9 µl NFW, 1 ul of a miniprep of either pSB1C3 or pSB3K3
12.5 µl of Q5 2x Master Mix.
Program the thermocycler with standard PCR cycles with a 55 C annealing temperature and a 1 minute extension time for pSB1C3, or a 1 minute 30 second extension time for  pSB3K3. Gel extract the PCR product.
Preparation of 30 nM 3G parts
Quantify the concentration of the miniprepped 3G part with a nanodrop, convert to molarity, then dilute to 30 nM in nuclease free water.
Golden Gate Assembly
To a 0.2 mL tube, add the following:
0.5 µL promoter 30 nM miniprep stock
0.5 µL 5’ UTR 30 nM miniprep stock
0.5 µL CDS 30 nM miniprep stock
0.5 µL terminator 30 nM miniprep stock
0.5 µL 50 nM left UNS adapter
0.5 uL 50 nM right UNS adapter
In addition, either add:
0.5 µL 10x DNA ligase buffer
0.25 µL BsaI
0.25 µL T4 DNA Ligase
1 µL NFW
OR
0.5 µL Golden Gate Buffer
0.25 µL NEB Golden Gate Master Mix
1.25 µL NFW
Note: if performing multiple Golden Gate reactions, we recommend making a master mix with a slight upscale, then add 2 µL of the master mix to reaction.
Cycle the reaction in the thermocycler. For the Golden Gate Master Mix perform 30 cycles of incubating at 37 for 1 minute then incubating at 16 C for one minute. For the T4 ligase and BsaI, perform 9 cycles at 37C for 3 minutes followed by incubating at 16 C for 4 minutes. For both options, hold at 37 C for five minutes, then hold at 4 C.  For information on creating multiple circuit variants at once, see parallel 3G Assembly .
PCR Amplification
Amplify the product with a 50 uL PCR as according to the PCR protocol. Add:
1.5 µL Golden Gate assembly product
18.5 µL NFW
2.5 µL of the appropriate 10uM UNS_foward Primer (that will anneal to the UNS on the 5’ end of your transcriptional unit)
2.5 µL 10 uM of the appropriate UNS_reverse primer (that will anneal to the UNS on the 3’ end of your transcriptional unit)
25 µL 2x Hot Start Master Mix
Incubate reactions using the standard 2x Q5 Hot Start Master Mix protocol with 27 cycles, using 30 seconds per kilobase pair, and an annealing temperature of 64 C. Conduct a gel extraction on the PCR product. (Note that 3G often produces off target bands as well as an extremely bright on target band. This is normal, and is why gel extraction rather than PCR purification is recommended).
Gibson Assembly
Perform a 5 uL Gibson assembly using the purified transcriptional units and vectors. Add to a .2 mL tube:
.015 pmolar each fragment
.015 pmolar desired backbone
Add NFW to a total volume of 2.5 µL.
2.5 ul 2X HiFi DNA Assembly master mix
Ensure the total fragment volume of the PCR product and backbone does not exceed 2.5 µl. Place tube in a thermocycler, run for 50 C for an hour, and hold at 4 C. Proceed to a transformation fairly quickly as Gibson products are only stable for a few hours.
Additional Information
Designing New 3G Parts
To design a new 3G part, use Gene Block to design a new sequence of double stranded DNA. Include a 40 bp pad sequence on either end, along with a BsaI site and sticky ends appropriate for the part type (for more information on pad sequences, see 3G Assembly ). This allows the new part to be cloned onto a William and Mary pad backbone.
Troubleshooting
In general, when 3G assembly fails, it will fail at the gel extraction phase. If correctly sized DNA is obtained from the gel, the Gibson reaction works very well. For circuits that fail to amplify at the PCR step, we recommend running an increased number of golden-gate cycles (100 or 30 for NEB and BsaI respectively). Additionally, use of longer (40 bp), full length UNS amplification primers may be helpful for amplification.
Sequences
Oligo Adapter Sequences
Oligo Adapter
Sequence
pOSIP A Bottom
CCGGTCTCTCTCCCCCGGGTACCGAGCTCGAATTCCGATCCCCAATTGGCGTC
pOSIP A Top
GACGCCAATTGGGGATCGGAATTCGAGCTCGGTACCCGGGGGAGAGAGACCGG
pOSIP E Bottom
AGGCGCCATGCATCTCGAGGCATGCCTGCAGCGGCCGCTAAAGCAGAGACCCC
pOSIP E Top
GGGGTCTCTGCTTTAGCGGCCGCTGCAGGCATGCCTCGAGATGCATGGCGCCT
UNS1_A_Bottom
CCGGTCTCTCTCCGAGACGAGACGAGACAGCCTGAGAATGGATGCGAGTAATG
UNS1_A_Top
CATTACTCGCATCCATTCTCAGGCTGTCTCGTCTCGTCTCGGAGAGAGACCGG
UNS3_A Bottom
CCGGTCTCTCTCCCGACCTTGATGTTTCCAGTGCGATTGAGGACCTTCAGTGC
UNS3_A_Top
GCACTGAAGGTCCTCAATCGCACTGGAAACATCAAGGTCGGGAGAGAGACCGG
UNS3_E_Bottom
CGACCTTGATGTTTCCAGTGCGATTGAGGACCTTCAGTGCAAGCAGAGACCCC
UNS3_E_Top
GGGGTCTCTGCTTGCACTGAAGGTCCTCAATCGCACTGGAAACATCAAGGTCG
UNS4_A_Bottom
CCGGTCTCTCTCCGACTTTGCGTGTTGTCTTACTATTGCTGGCAGGAGGTCAG
UNS4_A_Top
CTGACCTCCTGCCAGCAATAGTAAGACAACACGCAAAGTCGGAGAGAGACCGG
UNS4_E_Bottom
GACTTTGCGTGTTGTCTTACTATTGCTGGCAGGAGGTCAGAAGCAGAGACCCC
UNS4_E_Top
GGGGTCTCTGCTTCTGACCTCCTGCCAGCAATAGTAAGACAACACGCAAAGTC
UNS5_A_Bottom
CCGGTCTCTCTCCCTCTAACGGACTTGAGTGAGGTTGTAAAGGGAGTTGGCTC
UNS5_A_Top
GAGCCAACTCCCTTTACAACCTCACTCAAGTCCGTTAGAGGGAGAGAGACCGG
UNS5_E_Bottom
CTCTAACGGACTTGAGTGAGGTTGTAAAGGGAGTTGGCTCAAGCAGAGACCCC
UNS5_E_Top
GGGGTCTCTGCTTGAGCCAACTCCCTTTACAACCTCACTCAAGTCCGTTAGAG
UNS6_A_Bottom
CCGGTCTCTCTCCGTATGTGACCGTAGAGTATTCTTAGGTGGCAGCGAACGAG
UNS6_A_Top
CTCGTTCGCTGCCACCTAAGAATACTCTACGGTCACATACGGAGAGAGACCGG
UNS10_E_Bottom
GGTGGAAGGGCTCGGAGTTGTGGTAATCTATGTATCCTGGAAGCAGAGACCCC
UNS10_E_Top
GGGGTCTCTGCTTCCAGGATACATAGATTACCACAACTCCGAGCCCTTCCACC
Primers for Amplification of Transcriptional Units
UNS Amplification Primer
Sequence
UNS1 Forward
CATTACTCGCATCCATTCTCAGGC
UNS3 Forward
GCACTGAAGGTCCTCAATCG
UNS4 Forward
CTGACCTCCTGCCAGCAATAGT
UNS5 Forward
GAGCCAACTCCCTTTACAACCT
UNS6 Forward
CTCGTTCGCTGCCACCTAAGAA
UNS7 Forward
CAAGACGCTGGCTCTGACATTT
UNS3 Reverse
CGACCTTGATGTTTCCAGTGCG
UNS4 Reverse
GACTTTGCGTGTTGTCTTACTAT
UNS5 Reverse
CTCTAACGGACTTGAGTGAGGTTG
UNS6 Reverse
GTATGTGACCGTAGAGTATTCTTAGGTGG
UNS10 Reverse
GGTGGAAGGGCTCGGAGTTG
Primers for Creation of UNS Backbones
Primer
Sequence
Biobrick Suffix Fwd, UNS10 Overhang
CCAGGATACATAGATTACCACAACTCCGAGCCCTTCCACCTACTAGTAGCGGCC
Biobrick Prefix Rev, UNS1 Overhang
GAGACGAGACGAGACAGCCTGAGAATGGATGCGAGTAATGCTAGAAGCGGCC
References
[1] Single Day Construction of Multigene Circuits with 3G Assembly Andrew D. Halleran, Anandh Swaminathan, and Richard M. Murray. ACS Synthetic Biology 2018 7 (5), 1477-1480. DOI: 10.1021/acssynbio.8b00060
Top
Contact us at igem@wm.edu
Sponsors
2018 William & Mary iGEM
