Team:William and Mary/3G
Toggle navigation
Project
Background
Results
Temperature Systems
Methods and Data
3G Assembly
Method Overview
Collaborations
Parallel Circuit Testing
3G Parts
Protocol
Modeling
Human Practices
Overview
Incorporating New Communities
A Statewide Standard
Continuing the Conversation
IHP/Silver
Judging
Medal Requirements
Measurement
Demonstrate
Best Part Collection
Parts
Improve
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
3G Assembly
Motivation
The creation of genetic circuits is core to synthetic biology. However, with traditional DNA assembly methods, this process can be both time consuming and expensive. Additionally, current methods allow for only one circuit variant to be made at a time. This is inefficient when specific, but unknown, parameters are needed in the circuit.
Golden Gate-Gibson (3G) is a new, hybrid method of DNA assembly that addresses these issues. In 3G assembly, many variants of multi-part circuits can be constructed in a single day with high accuracy and efficiency [1]. When implemented in our iGEM lab, 3G assembly greatly improved our productivity and enabled us to spend less time on circuit construction and more time on experiments. With a simple protocol and low costs, 3G would be an invaluable tool for the iGEM program as whole. In order to make this method of DNA assembly accessible to all iGEM teams, we created a library of compatible parts to add to the registry.
Mechanism
Mechanistically, 3G assembly is a hybrid of Golden-Gate and Gibson assembly. In the first stage, transcriptional units are built using Golden-Gate. In the second stage, transcriptional units are combined on to a universal backbone.
Figure 1: Overview of 3G workflow
Golden Gate Stage
In Golden-Gate Assembly, type IIS restriction enzymes are used to cut DNA. Type IIS restriction enzymes are useful in that they cut outside their recognition sites, creating fragments of DNA with no unwanted base pair scars. We use the restriction enzyme BsaI, which recognizes specific DNA sequences (BsaI sites) and cuts outside of these sites, leaving sticky ends that can be ligated together with T4 DNA ligase.
In this stage of assembly, 3G takes advantage of the Cidar MoClo system, in which specific part types are distinguished by their sticky ends. After being cut, each type of part reveals a distinct sticky end on either side. The standard parts used in most synthetic circuits are promoters, 5’ untranslated regions, coding sequences, and terminators. Their MoClo sticky ends are shown in the image below:
Figure 2: Schematic of MoClo Sticky Ends of promoters, 5' UTRs, CDSs and terminators
The unique sticky ends allow for the parts to line up in the correct sequence before being ligated together. In this way, a full transcriptional unit can be created. Unique nucleotide sequences (UNS) are attached to both ends of the transcriptional unit. The UNS on the 5’ end of the transcriptional unit must have a sticky end A so that it can anneal to the promoter’s sticky end. The UNS on the 3’ end has a sticky end E so that it can anneal to the terminator’s sticky end. These sequences serve as a landing pad for primers in the next stage of PCR. They will also be used when combining the transcriptional units on to a backbone in the final stage of 3G assembly.
There are multiple 5’ UNSs and multiple 3’ UNSs, denoted by numbers (e.g., UNS 1, UNS 3, UNS 10). This allows us to combine multiple fragments in the Gibson step.
Figure 3: UNSs attached to transcriptional unit at sticky end A and sticky end E
Amplification
Next, Golden-Gate products are amplified by PCR. The forward primer will anneal to the 5’ UNS and the reverse primer will anneal to the 3’ UNS. The PCR products are then gel extracted.
Gibson Stage
In the final step, a Gibson assembly is performed. Here, multiple transcriptional units are combined on to a backbone. The backbone must have UNS 1 on the 5’ end and UNS 10 on the 3’ end. This can easily be achieved by PCRing your vector with primers that add the the UNSs.
Figure 4: Backbone with UNS 1 and UNS 10 adapters
In Gibson assembly, areas of homology are connected to form a circular plasmid. An exonuclease chews at the 5’ end of every fragment, creating compatible sticky ends that allow DNA fragments to align. DNA polymerase fills in the gaps and DNA ligase seals the bases together.
In 3G, the homologous regions between the vector and the transcriptional unit are the UNSs. To assemble a circuit with multiple transcriptional units, the first fragment should have UNS 1 on its 5’ end and the last fragment should have UNS 10 on its 3’ end. Adjacent fragments should have overlapping UNSs. For instance, the first transcriptional unit may begin with UNS 1 and end with UNS 3. The second transcriptional unit then begins with UNS 3 and ends with UNS 10. The UNS 1 on the first transcriptional unit will anneal to the UNS 1 on the vector. The UNS 3 sequences will anneal to each other, connecting the two units. Finally, the UNS 10 on the second unit will anneal to the UNS 10 on the vector, creating a circular plasmid. Of course, these steps are actually happening simultaneously, but they are outlined in this way for clarity.
Figure 5: Gibson stage of 3G: Transcriptional units attach to the backbone at UNS 1 and UNS 10 and to each other at UNS 3
Advantages
Modularity
3G assembly offers a variety of benefits that are specifically useful for iGEM teams. This system allows for remarkable modularity at multiple levels. During the Golden Gate stage, different variants of the same part can be added to the tube to produce a variety of transcriptional units. For example, if you do not know how strong an RBS will make your circuit perform optimally, you can try several types in one reaction, then screen your circuits for the output you want. The best performing circuit can then be sequenced. For a more detailed description, see here .
Figure 6: Modularity: RBS variants of the same circuit
The modularity of 3G assembly can also be seen at the Gibson stage, during which different transcriptional units can be easily combined. Once a transcriptional unit has been made, it can be combined with any other transcriptional unit with compatible UNSs. So, a unit flanked with UNS 1 and UNS 3 can be combined with any unit flanked with UNS 3 and UNS 5, and additionally any unit flanked with UNS 5 and UNS 10. Together, these units can go on any UNS compatible backbone. Up to six transcriptional units can be combined on to a vector at once.
Speed
The entire process of 3G assembly takes only about eight hours (including down time) followed by an overnight transformation. Furthermore, as explained above, numerous circuit variants can be created at once. Therefore, 3G is significantly more efficient than hierarchical cloning methods like ordinary Golden Gate.
Cost
Because standard adapter sequences are used, there is no need to redesign primers every time you create a circuit. This substantially cuts the cost of primers associated with Gibson assembly. In addition, the reactions in 3G are very small in volume, which saves on reagent costs. Our collaborators at UVA specifically cited this as an advantage they appreciated.
Simplicity
3G is a simple procedure that can be carried out with ease by any molecular biology lab. To demonstrate this as well as to ensure that our protocol was clear and repeatable, we collaborated with University of Pittsburgh and UVA. We sent these teams parts to assemble and test. Their success can be seen in the following gel images, which show Golden Gate PCR products of the expected lengths:
Figure 7: UVirginia Golden Gate gel extraction
Figure 8: UPittsburgh Golden Gate gel extraction
In addition, UVA conducted tests on a circuit they constructed. As expected, fluorescence levels were higher in the circuit with BBa_B0034 than BBa_B0032, the observed relative expression of B0032/B0034 (0.23) was roughly in line with the registry value of 0.3.
Figure 9: UVA circuit testing results: Each dot represents the relative fluorescence of an independent biological replicate compared to the geometric mean of B0034. The line represents the geometric mean of 3 independent biological replicates.
Because our collaborators were able to carry out 3G assembly, we expect that other iGEM teams should be able to as well. Thus, 3G can be practically implemented in iGEM labs, enabling teams to rapidly and inexpensively produce countless circuit variants. This leaves more time and resources for experimental work. In effect, 3G will allow for an advanced rate of scientific advances within iGEM and SynBio in general.
Within Our Project
Part One: Pad Sequences on Parts
One of our first goals was to create a library of parts that can easily be used in 3G assembly. To begin this process, we designed two sequences: WM_Pad1 and WM_Pad2. These pad sequences are orthogonal to the genome of E. coli and to the UNS sequences used later in 3G assembly. We designed primers that add the pad sequences to our parts outside of the BsaI sites and inside the BioBrick Prefix and Suffix. This leads to a part that looks like the following:
Figure 10: Schematic of the format of a 3G part in the WM Pad system
The sticky ends will, of course, vary, depending on which part we are working with. However, the pad sequences will remain the same. We put the same pad sequences on the high copy vector we commonly work with: 1C3. Then, we can easily use Gibson assembly to put our parts on the backbone and clone them. We created about seventy composite parts through this method. These parts can now be used by any team for 3G assembly, and can be viewed here .
Part Two: General Cloning
Whenever our team needed to construct multipart circuits, we used 3G assembly. Our results were generally excellent. We designed and created 45 circuits, only some of which we are submitting to the registry. An example of a transcriptional unit we created can be seen on this gel:
Figure 11: Gel extraction of WM18_03_10_19, expected length=1043
Part Three: Mixed Assembly
Our team’s project, like many iGEM projects, required us to create circuits with very specific desired behaviors. However, we were unable to perfectly anticipate what parameters would lead to the behavior we were looking for. Utilizing the modularity of 3G assembly, we took multiple variants of each part and combined them during the Golden Gate stage. This created an enormous number of transcriptional units that could be tested once put on circuits. We then screened for circuits that operated in the way we were looking for. The full plate reader experiment can be found here
Figure 11: Construction of multiple circuit variants in a single reaction
Overall, 3G assembly has provided boundless opportunities for our lab, and we are excited to share this powerful tool with the iGEM community.
References
[1] Single Day Construction of Multigene Circuits with 3G Assembly. Andrew D. Halleran, Anandh Swaminathan, and Richard M. Murray, ACS Synthetic Biology 2018 7 (5), 1477-1480. DOI: 10.1021/acssynbio.8b00060
Top
Contact us at igem@wm.edu
Sponsors
2018 William & Mary iGEM
