<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Design_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Design/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV><HEADER class="design-header"><DIV class="container"><H1 class="display">Design</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Design">Overview</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Animation">Animation</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Theory">Theory</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Design">Project Design</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Circuit">Circuit</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Design/Experiments">Experiments</A></LI><LI role="presentation"><A href="/Team:Lund/Design/References">References</A></LI></UL></DIV><DIV class="content-wrapper under-nav text-layout"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><SECTION><H3 class="section-heading">Overview</H3><P>

            In order to design a cohesive experimental plan, we begun by defining the parameters of interest. This was done with the intention of
            finding what constructs were necessary to not only generate parameter-data to fit the model but evaluate our parts both qualitatively and,
            when applicable, quantitatively.

          </P></SECTION><SECTION><H3 class="section-heading"><EM>In silico</EM> indications </H3><P>

            Through simulations of our intended construct with parameters defined a priori, we were able to draw some conclusions as to the validity
            of our system as a microplastic biosensor and generated some insight into the expected behavior which we could apply in the wet-lab
            setting.

            The model reflected very high dose sensitivity with regards to both ligands, which serves as an indicator that our system ought to work as
            a microplastic sensor. While we naturally expect lower affinity than that of the simulated natural ligands, it should still display
            somewhat similar characteristics. This is important as the concentrations of measured homologues will most likely be low. Thus, we can
            pass up on introducing positive autoregulatory motifs to our intended network topology.

            The rise-time of the GFP concentration suggests a slowly forming output signal, with over 10 hours before it reaches 90% of steady-state
            concentration (for varying concentrations of ligands). Thus, our fluorescence readings will have to be conducted first after several hours -
            most likely 10 hours post induction. Furthermore, improving the kinetics of a single node in our network will not particularly change the
            association time of GFP. That is to say, we do not need to ‚Äúimprove‚Äù any genes, for example through mutagenesis or simply changing some
            sequence to one with similar function, in an attempt to decrease the time to output signal as it will most likely not work.

            The IPTG concentrations will affect the overall gene transactivation and the associated GFP concentration. Thus, to achieve measureable results, we will use a relative high concentration corresponding to that of at least 1 mM.
          </P></SECTION><SECTION><H3 class="section-heading">Control construct design</H3><P>
            In this section we introduce the constructs intended to be used as controls. The gene architecture is designed to best generate parameters to fit the model and evaluate individual gene behaviors.
          </P>

          The pETDuet-1 vector was chosen for all desired protein expressions in this project as recommended by our advisor Dr. N√©lida Leiva Eriksson. Genes were
          synthesized at Eurofins Genomics and cloned into the pETDuet-1 vector using the restriction site pairs NcoI/BamHI and NdeI/AatII. The restriction
          sites were chosen based on their location downstream the two T7 promoters. In addition, it was required that these restriction enzymes were single
          site cutters which was ensured by using the <I>RestrictionMapper</I> online tool. All the gene sequences used were codon optimized for <EM>E.
          coli</EM> by using Eurofins‚Äô and IDT‚Äôs codon optimization tools prior to ordering the constructs. Furthermore, all genes encoding a protein were chosen to be ended with the double stop codon, TAA TAA.
          </SECTION><SECTION><H3 class="section-heading">NahR</H3><P>
            The control plasmid carries the <I>nahR gene</I> with a histidine tag, located under the T7 promoter. The purpose of this control plasmid was to
            see if it is possible to induce expression of NahR, using the drawn-up design illustrated in <I>fig. 1</I>. By utilizing the His-tag,
            the protein could be purified in order to determine its concentration.
          </P><P>
            The DNA sequence for the NahR protein was obtained from the BioBrick part <A href="http://parts.igem.org/Part:BBa_K228004">BBa_K228004</A>.
          </P><FIGURE><FIGCAPTION>Figure 1: nahR sequence with His-tag located under the T7 promoter.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">sfGFP </H3><P>
            The construct with <EM>superfolder green fluorescent protein (sfGFP)</EM> located under the T7 promoter, as depicted in <EM>fig. 2</EM>, was constructed to be used as a positive reference for the fluorescence signal.
          </P><FIGURE><FIGCAPTION>Figure 2: <I>sfGFP</I> located under the T7 promoter.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">GFP1-9</H3><P>

            The purpose of expressing the largest fragment of the tripartite split-GFP was to examine if the protein emit any fluorescence signal and
            if so, how large the fluorescence signal is compared to one from the sfGFP. A His-tag with 6 histidine residues was added to enable
            purification and measurements of protein concentration. Construct illustrated in <I>fig. 3</I>.
          </P><P>
            The DNA sequence for the tripartite split GFP was obtained from a study by Cabantous <I>et al.</I> (2013) [40].
          </P><FIGURE><FIGCAPTION>Figure 3: GFP1-9 located under the T7 promoter.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">NahR with sfGFP under Psal promoter</H3><P>
            This control construct, illustrated in <I>fig. 4</I> was designed in order to investigate if added salicylate increases the transcription rate of
            the reporter protein. Furthermore, the purpose was to examine the level of leaky expression obtained with no salicylate added. As
            explained in the theory section, NahR binds to and activates the <I>Psal</I> promoter in presence of salicylate resulting in expression of the
            reporter protein, in this case sfGFP.
          </P><P>
            The DNA sequences for sfGFP and <I>Psal</I> promoter were obtained from <A href="http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/superfolder_GFP/">SnapGene‚Äôs Plasmid file</A>
            and <A href="https://www.ncbi.nlm.nih.gov/nuccore/31249535?report=genbank">NCBI‚Äôs database</A>, respectively.
          </P><FIGURE><FIGCAPTION>Figure 4: <I>nahR</I> located under the T7 promoter, inducible by IPTG. After induced expression of the NahR protein, the latter binds to <I>Psal</I> and the sfGFP is expressed.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">NahR with GFP1-9 under Psal promoter</H3><P>

            Construct with <I>NahR</I> expressed under the T7 promoter and GFP1-9 located under the <I>Psal</I> promoter, as illustrated in <I>fig. 5</I>. The purpose of
            this control was to confirm the expression of GFP1-9 as a result from activation of the <I>Psal</I> promoter.
          </P><FIGURE><FIGCAPTION>Figure 5: The <I>NahR</I> gene located under the T7 promoter, inducible by IPTG. After induced expression of the NahR protein, the latter binds to <I>Psal</I> promoter (in presence of salicylate) and the GFP1-9 is expressed.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">GFP10-hER-ùõÇ LBD-GFP11</H3><P>
            The control was constructed to confirm expression of the fusion protein GFP10-hER-Œ± LBD-GFP11, illustrated in <I>fig. 6</I>. The protein is
            attached to a 6 histidine residue tag to facilitate purification and consequently allow concentration measurements.   The DNA sequence for
            the hER-Œ± LBD was kindly provided by Dr. Huimin Zhao upon request.
          </P><FIGURE><FIGCAPTION>Figure 6: GFP10-hER-Œ± LBD-GFP11 located under the T7 promoter with His-tag.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">GFP1-9 + GFP10-hER-Œ± LBD-GFP11 device</H3><P>
            Construct with GFP1-9 and GFP10-hER-Œ± LBD-GFP11, each located under a T7 promoter. This control was designed to investigate the reassembling of the tripartite split GFP and measure its fluorescence.
          </P><FIGURE><FIGCAPTION>Figure 7: Construct with genes encoding GFP1-9 and GFP10-hER-Œ± LBD-GFP11 each placed under their own T7 promoter. Note that the <I>lac</I> operators have been omitted for clarity.</FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">Intended experiments</H3><P><UL><LI> Induce cell culture with different IPTG concentrations at several OD<SUB>600</SUB> values to find an optimal condition for protein expression. SDS-PAGE would be performed for rough estimation of protein amounts. </LI><LI> Purify His-tagged proteins - NahR (<I>fig. 1</I>), GFP1-9 (<I>fig. 3</I>) and GFP10-hER-Œ± LBD-GFP11 (<I>fig. 6</I>), with Immobilized Metal ions Affinity Chromatography (IMAC) and thereafter perform protein concentration measurements using spectrophotometer.</LI><LI> Using fluorometer to measure fluorescence signal of sfGFP (<I>fig. 2</I>) and GFP1-9 (<I>fig. 3</I>).</LI><LI> Investigate the expression rate of sfGFP (<I>fig. 4</I>) and GFP1-9 (<I>fig. 5</I>) by using several concentrations of inducer salicylate as well as examine the leaky expression in absence of salicylate. </LI><LI> Use tamoxifen as an antagonistic ligand to investigate whether the conformational change of the GFP10-hER-Œ± LBD-GFP11 construct happens. The conformational change causing reassembly of the tripartite split GFP would result in fluorescence signal, measured with fluorometer. The same experiment would be performed without added tamoxifen in order to see whether the conformational change is needed for the tripartite split GFP to reassemble.</LI></UL></P></SECTION></DIV></DIV></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        S√∂lvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">¬©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>