<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Modeling_ModelDerivation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Modeling/ModelDerivation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV></DIV><HEADER class="modeling-landing"><DIV class="container"><H1 class="display">Modeling</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Model">Results</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL></DIV><DIV id="modeling" class="content-wrapper under-nav text-layout"><DIV class="container"><DIV class="row"><DIV class="col-lg-10"><SECTION><H3 class="section-heading">Introduction</H3><P> In this section we derive a model for the whole cell biosensor based on a system of ordinary differential equations. We choose to
          model one single cell deterministic and consider the dynamics of larger cell populations the same as the dynamics of a single cell, but scaled
          to the population size. This is reasonable although stochastic effects are present at the cellular level if one assumes that each cell is
          independent. The choice of modeling can then be justified by the law of large numbers: if there is a large number of identical and independent
          random variables (in this case cells), their average behaviour tend to converge to an expected behaviour.  Thus, as the expected behaviour is
          obtained from the deterministic scheme, this properly reflects our biosensor system as we can assume a very large population. </P><P>In addition, we present our reasoning behind the reactions and the corresponding assumptions. We start by modeling the regulation of the
          pETDuet-1 plasmid which consists of one gene encoding the <I>lac</I> repressor and two <I>lac</I> regulated genes which constitute the
          respective pathway in our AND-logic. The biomolecular network is then presented for each pathway followed by the final GFP complementation.
          At the end, a deterministic scheme representing the system in terms of ordinary differential equations is presented along with the
          associated assumptions and boundary conditions. </P></SECTION><SECTION><H3 class="section-heading">Overview</H3><P>A simplified description of the biochemical
          network for the implemented whole cell biosensor in the pETDuet-1 plasmid is illustrated in <I>fig. 1</I>. The vector has one <I>lac</I>
          operator regulating each T7-promoter and one corresponding <I>lacI</I> which constituitively expresses the <I>lac</I> repressor for both
          operators. Upon IPTG induction, the proteins NahR and GFP10-ER-GFP11 are expressed. These correspond to the respective pathway in the
          AND-logic. The former protein, NahR, is a transcription factor which binds to the <I>Psal</I> promoter located upstreams of <I>GFP1-9</I>.
          This activates the transcription of <I>GFP1-9</I> under the presence of salicylate, which serves as the first input into the gate. The
          GFP10-ER-GFP11 fusion protein can undergo a conformational change in the presence of estrogen, the second input. This brings the GFP10 and
          GFP11 linker bound fragments into close proximity to allow for a subsequent association with GFP1-9. This complementation yields a fully
          restored chromophore which can be detected by fluorometric methods, consequently closing the circuit. See the <A href="/Team:Lund/Design">design section</A> for more information.</P><FIGURE><FIGCAPTION><I>Figure 1: Diagram of the key biochemical reactions in the biosensor. The system starts with an induction of NahR and GFP10-ER-GFP11. In
      the presence of salicylate, NahR induces the transcription of GFP1-9. In the presence of estrogen, the GFP10-ER-GFP11 fusion protein can undergo a
      conformational change upon binding, allowing for the GFP10 and GFP11 linker bound fragments to come into close proximity. This situation allows GFP1-9
      to associate with the fragments, resulting in a fully restored fluorophore. </I></FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading"><I>lac</I> regulation</H3><P>The system starts with an IPTG induction of two <I>lac</I> operators. We choose to model this in a greater detail than the standard
          Michaelis-Menten scheme as we now have two operators being repressed by one <I>lacI</I>. Thus, we have chosen to include the repressor
          dynamics with the operators but also chosen an intermediate route where we dont take into account of leak transcriptions or other second order
          effects. Here we provide a description of the derivation based on a study by M. Stamatakis and N. V. Mantzaris who used mass
          action and stochastic kinetic models to study the autoregulation and bistability of the <I>lac</I> operon [2].</P><P>Starting with the transcriptional pathway of the <I>lac</I> repressor, the <I>lacI</I> gene $\ce{O_R}$ which codes for the dimeric monomer
          $\ce{R}$ is first transcribed to yield the mRNA $\ce{M_R}$. The transcript is then translated into the protein $R$ which dimerises into the repressor $R_2$ in a
          reversible reaction.</P><P> $$\ce{O_R -&gt;[k_\text{MR}] O_R + M_R}\tag{1}$$
              $$\ce{M_R -&gt;[k_\text{R}] M_R + R}\tag{2}$$
              $$\ce{2R &lt;--&gt;[k_\text{2R}][k_\text{-2R}] R_2}\tag{3}$$ </P><P> The repressor can subsequently bind to the operators $\ce{O_N}$ and $\ce{O_E}$ which regulates the expression of NahR and GFP10-ER-GFP11 to
          yield the corresponding repressed operators $\ce{R_2O_N}$ and $\ce{R_2O_E}$. This is modeled as two reversible reactions where the repressed
          operator can dissociate back into the repressor and free operator. Note that the reaction rates are equal for both equilibriums as we assume that
          the dynamics is the same for both operators. </P><P> $$\ce{R_2 + O_N &lt;--&gt;[k_r][k_{-r}] R_2O_N}\tag{4}$$
              $$\ce{R_2 + O_E &lt;--&gt;[k_r][k_{-r}] R_2O_E}\tag{5}$$
          </P><P> The induction of the repressed operons proceeds via the addition of an inducer $\ce{I}$ which binds to the repressor, allowing
          transcription to proceed. This is modeled as a third order reaction where two entities of $\ce{I}$ binds to $\ce{R_2}$ to yield $\ce{I_2R_2}$. The reaction
          can take place between both free and operator bound repressors and the complex $\ce{I_2R_2}$ can dissociate back and rerepress the operators. </P><P> $$\ce{2I + R_2 &lt;--&gt;[k_{dr1}][k_{-dr1}] I_2R_2}\tag{6}$$
              $$\ce{2I + R_2O_N &lt;--&gt;[k_{dr2}][k_{-dr2}] I_2R_2 + O_N}\tag{7}$$
              $$\ce{2I + R_2O_E &lt;--&gt;[k_{dr2}][k_{-dr2}] I_2R_2 + O_E}\tag{8}$$
          </P><P> The inducer $\ce{I}$ is distinguished from the extracellular inducer $\ce{I_{ex}}$. The flux over the cell membrane is modeled as free diffusion. </P><P> $$\ce{I_{ex} &lt;--&gt;[k_{dI}][k_{dI}] I}\tag{9}$$ </P><P> The transcription and translations are modeled as catalytic first order reactions. The genes regulated by $\ce{O_N}$ and $\ce{O_E}$ yield the
          NahR and GFP10-ER-GFP11 transcripts $\ce{M_N}$ and $\ce{M_E}$ respectively. These are further translated into the respective proteins, $\ce{N}$ and $\ce{E}$.
          </P><P> $$\ce{O_N -&gt;[k_\text{MN}] O_N + M_N}\tag{10}$$
          $$\ce{O_E -&gt;[k_\text{ME}] O_E + M_E}\tag{11}$$ $$\ce{M_N -&gt;[k_\text{N}] M_N + N}\tag{12}$$ $$\ce{M_E -&gt;[k_\text{E}] M_E + E}\tag{13}$$ </P><P> All
          degradations are modeled as first order reactions. It is assumed that all mRNA:s and proteins degrade with the exception of the operator bound
          repressor. Note that it is also assumed that the inducer does not decompose. </P><P> $$\ce{M_R -&gt;[\lambda_\text{MR}] \phi}\tag{14}$$
              $$\ce{M_N -&gt;[\lambda_\text{MN}] \phi}\tag{15}$$
              $$\ce{M_E -&gt;[\lambda_\text{ME}] \phi}\tag{16}$$
              $$\ce{R -&gt;[\lambda_\text{R}] \phi}\tag{17}$$
              $$\ce{R_2 -&gt;[\lambda_{\text{R}_2}] \phi}\tag{18}$$
              $$\ce{N -&gt;[\lambda_\text{N}] \phi}\tag{19}$$
              $$\ce{E -&gt;[\lambda_\text{E}] \phi}\tag{20}$$
              $$\ce{I_2R_2 -&gt;[\lambda_{\text{I}_2\text{R}_2}] 2I}\tag{21}$$
          </P></SECTION><SECTION><H3 class="section-heading">NahR</H3><P> It has been shown that NahR binds to the <I>Psal</I> promoter as a transcription factor regardless of the presence of
          salicylate. The binding affinity has previously been thought to be unaffected of salicylate, but recent in vitro studies with surface plasmon
          resonance indicate that it in fact has a slight inhibiting effect on the binding to <I>Psal</I>. There has also been indications that NahR binds
          as a tetramer to the promoter. However, it has also recently been shown that NahR binds sequentially as a monomer to <I>Psal</I> and then
          undergoes multimerization on site. The degree of multimerization is to our knowledge not known and seems to vary with the concentration of NahR.
          It has been shown that immobilized <I>Psal</I> can bind up to eight monomers for high concentrations of NahR (&gt; 6.8 µM), while at low
          concentrations (&lt; 0.42 µM) only one monomer is bound [1]. Furthermore, many aspects of the underlying dynamics remains unclear. For example,
          we do not know whether one salicylate has to bind to each NahR before transcription can occur or whether the transcription rate is cumulative.
          Neither do we know how the transcription factor affects the promoter with respect to the amount of monomers bound. Taking all the intermediate
          steps into account in the model would lead to a complicated system with many species, yielding a large set of equations with a big parameter
          space. Consequently, as the amount of data needed to estimate a model increases exponentially with the amount of parameters, we choose to model
          the regulation in two steps. </P><P> The reactions proceed by NahR binding to <I>Psal</I> followed by salicylate binding to the <I>NahR-Psal</I> complex in two reversible second order
          reactions, allowing subsequent transcription of <I>GFP1-9</I>. Thus, we have chosen to not take into account of the effect of salicylate on the binding
          of NahR to <I>Psal</I>. Furthermore, we have not taken into account of the binding stoichiometries or possible transcriptionally active intermediates and
          instead estimated the reactions as being of second order between the fully active and inactive states. Worth noting is that despite these assumptions,
          the model manages to fit experimental dose-response data obtained from the iGEM parts registry page, see the <A href="/Team:Lund/Modeling/ParameterEstimation">parameter estimation</A>
          page. Thus, the assumptions seem sufficient, at least for a high level description. By denoting the promoter as $\ce{\text{Psal}}$, salicylate as $\ce{S}$
          and the induced promoter $\ce{\text{SNPsal}}$, we get the following set of reactions: </P><P>
            $$\ce{N + P\text{sal} &lt;--&gt;[k_\text{aNP}][k_\text{dNP}] NP\text{sal}}\tag{22}$$
            $$\ce{S + NPsal &lt;--&gt;[k_\text{aSNP}][k_\text{dSNP}] SNPsal}\tag{23}$$
          </P><P> Once the the promoter is induced, the transciption of the <I>GFP1-9</I> can start. The transcript is denoted as $\text{M}_\text{G}$. This is followed by a translation
          which yields the GFP1-9 fragment denoted as $\text{G}$. </P>
            $$\ce{SNPsal -&gt;[k_\text{MG}] SNPsal + M_\text{G}}\tag{24}$$
            $$\ce{M_\text{G} -&gt;[k_\text{G}] M_\text{G} + G}\tag{25}$$
          <P>
            In addition, we distinguish intracellular salicylate from extracellular and model the flux as free difussion over the cell membrane. </P><P>
            $$\ce{S_\text{ex} &lt;--&gt;[k_{dS}][k_{dS}]S}\tag{26}$$
          </P><P> All degradations are of first order and we assume that salicylate does not decompose. </P><P>
            $$\ce{M_G -&gt;[\lambda_\text{MG}] \phi}\tag{27}$$
            $$\ce{G -&gt;[\lambda_\text{G}] \phi}\tag{28}$$
          </P></SECTION><SECTION><H3 class="section-heading">GFP10-ER-GFP11</H3><P>We assume equivalency between the kinetic mechanism of antagonistic and agonistic binding to the estrogen receptor. For sake of simplicity,
          antiestrogen is used interchangeably with estrogen throughout the text. The binding of estrogen to the GFP10-ER-GFP11 fusion protein is modeled as
          a reversible second order reaction where a ligand bound fusion protein is generated. By denoting estrogen as $\ce{Es}$ and the ligand bound
          protein $\ce{EEs}$, we get the following reaction:</P><P>
            $$\ce{E + Es &lt;--&gt;[k_{aEEs}][k_{dEEs}] EE\text{s}}\tag{29}$$
          </P><P> We distinguish intracellular estrogen from extracellular and model the flux as free diffusion over the cell membrane. In addition, the ligand
          bound estrogen receptor fusion protein degrades in a first order reaction (note that we already defined the degradation for $\ce{E}$).</P><P>
            $$\ce{E\text{s} &lt;--&gt;[k_\text{dEs}][k_\text{dEs}] E\text{s}_\text{ex}}\tag{30}$$
            $$\ce{EE\text{s} -&gt;[\lambda_E] \phi}\tag{31}$$
          </P></SECTION><SECTION><H3 class="section-heading">GFP association</H3><P>There are many ways in which one could model the tripartite association. For example, one could imagine the GFP10 and GFP11 fragments first
          establishing a hairpin loop prior to complementing with GFP1-9. Another mechanism would be that one of the estrogen receptor bound fragments first
          binds to GFP1-9 followed by an association of the second receptor bound fragment. Nevertheless, we choose to model the association as a direct
          bimolecular complementation as it allowed us to estimate the association and dissocation rate constants with ease due to the simple kinetic
          expression, see the <A href="/Team:Lund/Modeling/ParameterEstimation">parameter estimation section</A> for more details. We thus have the GFP1-9 and active GFP10-ER-GFP11 associating to
          yield a restored GFP denoted as $\ce{GFP}$. As before, the protein decomposes in a first order reaction.</P><P>
            $$\ce{G + EE\text{s} &lt;--&gt;[k_\text{aGFP}][k_\text{dGFP}] GFP}\tag{32}$$
            $$\ce{GFP -&gt;[\lambda_\text{GFP}] \phi}\tag{33}$$
          </P><TABLE class="table"><CAPTION>Table 1: List of species</CAPTION><THEAD><TR><TH>Notation</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>$\ce{O_\text{R}}$</TD><TD><I>lacI</I> gene</TD></TR><TR><TD>$\ce{M_\text{R}}$</TD><TD>LacI mRNA</TD></TR><TR><TD>$\ce{R}$</TD><TD>LacI monomer</TD></TR><TR><TD>$\ce{R_2}$</TD><TD><I>lac</I> repressor</TD></TR><TR><TD>$\ce{O_\text{N}}$</TD><TD><I>lac</I> operator regulating NahR </TD></TR><TR><TD>$\ce{O_\text{E}}$</TD><TD><I>lac</I> operator regulating GFP10-ER-GFP11</TD></TR><TR><TD>$\ce{R_2O_\text{N}}$</TD><TD>Repressed <I>lac</I> operator regulating NahR</TD></TR><TR><TD>$\ce{R_2O_\text{E}}$</TD><TD>Repressed <I>lac</I> operator regulating GFP10-ER-GFP11</TD></TR><TR><TD>$\ce{I_\text{ex}}$</TD><TD>Extracellular IPTG</TD></TR><TR><TD>$\ce{I}$</TD><TD>Intracellular IPTG</TD></TR><TR><TD>$\ce{I_2R_2}$</TD><TD>Inhibited <I>lac</I> repressor</TD></TR><TR><TD>$\ce{M_\text{N}}$</TD><TD>NahR mRNA</TD></TR><TR><TD>$\ce{M_\text{E}}$</TD><TD>GFP10-ER-GFP11 mRNA</TD></TR><TR><TD>$\ce{N}$</TD><TD>NahR</TD></TR><TR><TD>$\ce{E}$</TD><TD>GFP10-ER-GFP11</TD></TR><TR><TD>$\ce{P\text{sal}}$</TD><TD>Psal promoter</TD></TR><TR><TD>$\ce{NP\text{sal}}$</TD><TD>NahR-Psal complex</TD></TR><TR><TD>$\ce{SNP\text{sal}}$</TD><TD>Salicylate-NahR-Psal complex</TD></TR><TR><TD>$\ce{S}$</TD><TD>Intracellular salicylate</TD></TR><TR><TD>$\ce{S_\text{ex}}$</TD><TD>Extracellular salicylate</TD></TR><TR><TD>$\ce{M_\text{G}}$</TD><TD>GFP1-9 mRNA</TD></TR><TR><TD>$\ce{G}$</TD><TD>Matured GFP1-9 </TD></TR><TR><TD>$\ce{EEs}$</TD><TD>Salicylate bound GFP10-ER-GFP11</TD></TR><TR><TD>$\ce{Es}$</TD><TD>Intracellular estrogen</TD></TR><TR><TD>$\ce{Es_\text{ex}}$</TD><TD>Extracellular estrogen</TD></TR><TR><TD>$\ce{GFP}$</TD><TD>Complemented GFP</TD></TR></TBODY></TABLE></SECTION><SECTION><H3 class="section-heading">Determinstic equations</H3><P> For the sake of completeness, we present a determinstic representation of the network in terms of ordinary differential equations. It is
            assumed that all reactions follow mass action kinetics, that there is no cellular net growth and that each cell is chemically
            homogeneous and well stirred. In addition, we assume that the cells do not affect their environment so the concentrations of all extracellular
            species can be considered as constant boundary conditions. The reaction rates of (1)-(33) are then given by:
          </P><P><BUTTON id="ratebutton" class="btn btn-default" type="button" data-toggle="collapse" data-target="#rates" aria-expanded="false" aria-controls="rates">
              Show reaction rates
            </BUTTON></P><DIV class="collapse" id="rates"><P>
                $$v_1 = k_{MR}O_R \tag{34}$$
                $$v_2 = k_RM_R\tag{35}$$
                $$v_{3,f} = k_{2R}R^2 \tag{36}$$
                $$v_{3,r} = k_{-2R}R_2 \tag{37}$$
                $$v_{4,f} = k_rR_2\cdot O_N \tag{38}$$
                $$v_{4,r} = k_{-r}R_2O_N\tag{39}$$
                $$v_{5,f} = k_rR_2\cdot O_E\tag{40}$$
                $$v_{5,r} = k_{-r}R_2O_E\tag{41}$$
                $$v_{6,f} = k_{dr1}I^2\cdot R_2\tag{42}$$
                $$v_{6,r} = k_{-dr1}I_2R_2\tag{43}$$
                $$v_{7,f} = k_{dr2}I^2\cdot R_2O_N\tag{44}$$
                $$v_{7,r} = k_{-dr2}I_2R_2 \cdot O_N\tag{45}$$
                $$v_{8,f} = k_{dr2}I^2\cdot R_2O_E\tag{46}$$
                $$v_{8,r} = k_{-dr2}I_2R_2 \cdot O_E\tag{47}$$
                $$v_{9,f} = k_{dI}I_{ex}\tag{48}$$
                $$v_{9,r} = k_{dI}I\tag{49}$$
                $$v_{10} = k_{MN}O_N\tag{50}$$
                $$v_{11} = k_{ME}O_E\tag{51}$$
                $$v_{12} = k_{N}M_N\tag{52}$$
                $$v_{13} = k_{E}M_E\tag{53}$$
                $$v_{14} = \lambda_{MR}M_R\tag{54}$$
                $$v_{15} = \lambda_{MN}M_N\tag{55}$$
                $$v_{16} = \lambda_{ME}M_E\tag{56}$$
                $$v_{17} = \lambda_{R}R\tag{57}$$
                $$v_{18} = \lambda_{R_2}R_2\tag{58}$$
                $$v_{19} = \lambda_{N}N\tag{59}$$
                $$v_{20} = \lambda_{E}E\tag{60}$$
                $$v_{21} = \lambda_{I_2R_2}I_2R_2 \tag{61}$$
                $$v_{22,f} = k_{aNP}N\cdot Psal\tag{62}$$
                $$v_{22,r} = k_{dNP}NPsal\tag{63}$$
                $$v_{23,f} = k_{aSNP}S\cdot NPsal\tag{64}$$
                $$v_{23,r} = k_{dSNP}SNPsal\tag{65}$$
                $$v_{24} = k_{MG}SNPsal\tag{66}$$
                $$v_{25} = k_{G}M_G\tag{67}$$
                $$v_{26,f} = k_{dS}S_{ex}\tag{68}$$
                $$v_{26,r} = k_{dS}S\tag{69}$$
                $$v_{27} = \lambda_{MG}M_G\tag{70}$$
                $$v_{28} = \lambda_GG\tag{71}$$
                $$v_{29,f} = k_{aEEs}E\cdot Es\tag{72}$$
                $$v_{29,r} = k_{aEEs}EEs\tag{74}$$
                $$v_{30,f} = k_{dEs}Es\tag{75}$$
                $$v_{30,r} = k_{dEs}Es_{ex}\tag{75}$$
                $$v_{31} = \lambda_EEEs\tag{76}$$
                $$v_{32,f} = k_{aGFP}G\cdot EEs\tag{77}$$
                $$v_{32,r} = k_{dGFP}GFP\tag{78}$$
                $$v_{33} = \lambda_{GFP}GFP\tag{79}$$
              </P></DIV><P> By balancing the masses, we get the set of differential equations (80-105). Note that the rate equations for the extracellular inducers are
          set to zero, as they are considered as boundary conditions under the assumption that the cell does not affect it's environment.
        </P><P><BUTTON id="odebutton" class="btn btn-default" type="button" data-toggle="collapse" data-target="#odes" aria-expanded="false" aria-controls="odes">
            Show differential equations
          </BUTTON></P><DIV class="collapse" id="odes"><P>
              $$\frac{dO_R}{dt} = 0 \tag{80}$$
              $$\frac{dM_R}{dt} = v_1 - v_{14} \tag{81}$$
              $$\frac{dR}{dt} = v_2 - 2v_{3,f} + 2v_{3,r} - v_{17} \tag{82}$$
              $$\frac{dR_2}{dt} = v_{3,f} - v_{3,r}  - v_{4,f} + v_{4,r} - v_{5,f} + v_{5,r} - v_{6,f} + v_{6,r} - v_{18} \tag{83}$$
              $$\frac{dO_N}{dt} = -v_{4,f} + v_{4,r} + v_{7,f} - v_{7,r} \tag{84}$$
              $$\frac{dO_E}{dt} = -v_{5,f} + v_{5,r} + v_{8,f} - v_{8,r} \tag{85}$$
              $$\frac{dR_2O_N}{dt} = v_{4,f} - v_{4,r} - v_{7,f} + v_{7,r} \tag{86}$$
              $$\frac{dR_2O_E}{dt} = v_{5,f} - v_{5,r} - v_{8,f} + v_{8,r} \tag{87}$$
              $$\frac{dI_{ex}}{dt} = 0 \tag{88}$$
              $$\frac{dI}{dt} = v_{9,f} - v_{9,r} + 2v_{21} \tag{89}$$
              $$\frac{dI_2R_2}{dt} = v_{6,f} - v_{6,r} + v_{7,f} - v_{7,r}  + v_{8,f} - v_{8,r} - v_{21} \tag{90}$$
              $$\frac{dM_N}{dt} = v_{10} - v_{15} \tag{91}$$
              $$\frac{dM_E}{dt} = v_{11} - v_{16} \tag{92}$$
              $$\frac{dN}{dt} = v_{12} - v_{19} - v_{22,f} + v_{22,r} \tag{93}$$
              $$\frac{dE}{dt} = v_{13} - v_{20} - v_{29,f} + v_{29,r} \tag{94} $$
              $$\frac{dPsal}{dt} = -v_{22,f} + v_{22,r} \tag{95}$$
              $$\frac{dNPsal}{dt} = v_{22,f} - v_{22,r} - v_{23,f} + v_{23,r} \tag{96}$$
              $$\frac{dSNPsal}{dt} = v_{23,f} - v_{23,r} \tag{97}$$
              $$\frac{dS}{dt} = -v_{23,f} + v_{23,r} + v_{26,f} - v_{26,r} \tag{98}$$
              $$\frac{dS_{ex}}{dt} = 0 \tag{99}$$
              $$\frac{dM_G}{dt} = v_{24} - v_{27} \tag{100}$$
              $$\frac{dG}{dt} = v_{25} - v_{28} - v_{32,f} + v_{32,r} \tag{101}$$
              $$\frac{dEEs}{dt} = v_{29,f} - v_{29,r} - v_{31} - v_{32,f} + v_{32,r} \tag{102}$$
              $$\frac{dEs}{dt} = - v_{29,f} + v_{29,r} - v_{30,f} + v_{30,r} \tag{103}$$
              $$\frac{dEs_ex}{dt} = 0 \tag{104}$$
              $$\frac{dGFP}{dt} = v_{32,f} - v_{32,r} - v_{33} \tag{105} $$
            </P></DIV></SECTION><SECTION><H3 class="section-heading">
            References
          </H3><BUTTON id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
            Show
          </BUTTON><DIV class="collapse" id="ref"><OL class="references"><LI>[1] Park, H., Lim, W. and Shin, H. (2005). In vitro binding of purified NahR regulatory protein with promoter Psal. <I>Biochimica et Biophysica Acta (BBA) - General Subjects</I>, 1725(2), pp.247-255.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>