Team:Lund/Results
Toggle navigation
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Results
InterLab
Parts
Protocols
Notebook
Model
Safety
Practices
Results
Results and Discussion
The control constructs in pETDuet-1 vector, described in experimental design , were ordered at the end of June to start performing the             experiments during the summer. Unfortunately, several logistical problems and other unforeseen issues arose and severely delayed the intended             schedule. In addition, full access to lab space and necessary equipment were not provided until the beginning of September. Due to the delays,             the constructs were received in mid September and throughout October, resulting in limited time to run the planned experiments. Priorities had             to be made and the focus shifted to assembling BioBricks. Further, a backup plan was executed through modifying the constructs to suit BioBrick             standards and order them as gBlocks. Several attempts to assemble these gBlocks, which arrived prior to the pETDuet-1 constructs, into BioBricks             were performed. However, assembling and cloning successfully takes a great amount of time which we were not fortunate to have.
One frequently tested part was GFP1-9, see parts . Cells were transformed with the ligation product and the resulting colonies were cultured overnight. After cell harvest, plasmids carrying the insert were isolated and             agarose gel electrophoresis was run. The result is displayed in fig. 1 with two distinct bands visualized in the corresponding well. The band with a             size of 1.6 kbp is believed to be supercoiled and the other band (with a size of >3 kb) is thought to be nicked plasmid. The other two wells are             displaying PCR products of the plasmid carrying GFP1-9 mentioned earlier. A band corresponding to a size around 300 bp is most likely the result of             an empty plasmid since that is the length of the sequence between the primers. A plasmid containing the gene of interest would have a size of             approximately 900 bp. Furthermore, the obtained smearing could be due to wrong annealing temperature, which was further investigated using a             temperature gradient. It could also be the result of using too many amplification cycles, however, this does not explain why the DNA ladder also             appears as smeared. Similar results were obtained with the other parts as well.
Figure 1. Amplified fragments show a band around 300 bp and mini-prepped plasmid shows two distinct bands.
Various sources of error were encountered on the way and had to be investigated and handled, as described in the upcoming section. Due to these complications and restricted time to troubleshoot and redo the necessary experiments, we are sad to say that no BioBrick samples could be submitted to the registry.
Troubleshooting
Cell competency
All cells used in the experiments were in-house made competent cells, which could have affected the competency of the cells. However, after each batch, cell competency was tested showing a sufficient transformation efficiency.
Digestion
Possible sources of error during the digestion could be due to too few additional nucleotides at the end of synthesized gBlocks. This could have lead to limited performance of restriction enzymes. Hence, it could be theorized that no thorough cleavage could be performed, resulting in poor production of sticky ends. During the first experiments, the digestion time could have been too short. In the following experiments the digestion time was extended, however, no improvements were obtained. Another possibility of poor digestion could be due to addition of an insufficient amount of DNA.
Ligation
At first, the ligation time was considered too short and was later extended to an overnight ligation in order to improve the outcome. It is also possible that the optimal ratio of insert to backbone was not used, although three different ratios were tested each time. Problems with the ligation could also be a result of poor digestion.
Transformation
The resulting low transformation yields could be due to low competency of cells, non-gentle handling of cells during the transformation process or a result of unsuccessful ligation. Although several bacterial strains and DNA concentrations were tested, the result remained the same.
We did extensive troubleshooting and changed the protocols multiple times to no avail. The changes that were carried out were all discussed with             faculty members to ensure relevancy. With more time, or possibly as recommendations for future work, we would of course be interested in testing the             constructs according to our experimental design. As we never got the opportunity to try the complete circuit, it is not possible to say whether this             design would have worked or not. The original construct comprised of genes encoding three different proteins could pose a risk of putting too much             pressure on the cell when inducing protein expression. Instead, an option could be attempting double transformation using compatible plasmids. Another             interesting experiment would be to purify the proteins from the biosensor device to see how they interact ex situ. Finally, it would of course be             desirable to test the biosensor using natural ligands - organic pollutants and phthalates.
Home
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Results
InterLab
Parts
Protocol
Notebook
Model
Results
Model derivation
Parameter estimation
Implementation
Safety
Safety
Practices
Overview
Integrated
Engagement
Media
Communication
Contact
Sölvegatan 22 A
223 62 Lund
Sweden
igemlund@gmail.com
Stay connected
Facebook
Instagram
Twitter
©2017 iGEM Lund
