<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV><HEADER class="experiments-landing"><DIV class="container"><H1 class="display">Protocols</H1></DIV></HEADER><DIV class="content-wrapper"><DIV class="container"><SECTION id="experiments"><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#agarplates"><H5>
             Preparation of agar plates
           </H5></A><DIV id="agarplates" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>10 g Bacto tryptone</LI><LI>5 g Bacto yeast extract </LI><LI>10 g NaCl</LI><LI>15 g Agar </LI><LI>1 L dH2O </LI><LI>12.5 mg/mL stock solution chloramphenicol </LI><LI>100 mg/mL sock solution ampicillin</LI></UL><U>Equipment</U><UL><LI> 1L beaker</LI><LI> Magnetic stirrer</LI><LI> Petri dishes</LI><LI> Autoclave</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Add 500 mL dH2O to beaker</LI><LI>Add all dry reagents</LI><LI>Add another 500 mL dH2O, stir and autoclave</LI><LI>Let cool and add 1 mL chloramphenicol stock solution or 0.5 mL ampicillin stock solution </LI><LI>Stir and add 20 mL to petri dishes</LI></OL></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#lb"><H5>
             Preparation of LB medium
           </H5></A><DIV id="lb" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>10 g tryptone</LI><LI>5 g yeast extract </LI><LI>10 g NaCl</LI><LI>1000 mL dH2O </LI></UL><U>Equipment</U><UL><LI> Beaker</LI><LI> Magnetic stirrer</LI><LI> Autoclave</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Add all dry reagents in a beaker</LI><LI>Add 1000 mL dH2O and stirr</LI><LI>Autoclave</LI></OL></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#plasmidisolation">
            Plasmid isolation
          </A></H5><DIV id="plasmidisolation" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>NucleoSpin® Plasmid (NoLid) Plasmid DNA purification Kit</LI><LI>Sterile MQ water</LI><LI>Cell culture</LI></UL><U>Equipment</U><UL><LI>Centrifuge</LI><LI>Vortex (optionally)</LI><LI>Lab timer</LI><LI>Pipettes and Tips (1000 µL and 500 µL recommended)</LI><LI>1.5 mL microcentrifuge tubes</LI><LI>Nanophotometer</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Add 1 mL of Buffer A1 to the RNase A vial and vortex. Transfer the solution back into the Buffer A1 bottle and mix thoroughly. Indicate date of RNase A addition. Store Buffer A1 containing RNase A at 4 °C. The solution will be stable at this temperature for at least six months.</LI><LI>Add 100 mL of 96–100 % ethanol to Buffer A4.</LI><LI><B>Cultivate and harvest bacterial cells.</B> Use 1–5 mL of a saturated E.coli LB culture, pellet cells in a standard benchtop microcentrifuge for 30 s at 11,000 x g. Discard the supernatant and remove as much of the liquid as possible.</LI><LI><B>Cell lysis.</B> Add 250 μL Buffer A1. Resuspend the cell pellet completely by vortexing or pipetting up and down. Make sure no cell clumps remain before addition of Buffer A2!</LI><LI>Add 250 μL Buffer A2. Mix gently by inverting the tube 6–8 times. Do not vortex to avoid shearing of genomic DNA. Incubate at room temperature for up to 5 min or until lysate appears clear. </LI><LI>Add 300 μL Buffer A3. Mix thoroughly by inverting the tube 6–8 times until blue samples turn colorless completely! Do not vortex to avoid shearing of genomic DNA!</LI><LI>Make sure to neutralize completely to precipitate all protein and chromosomal DNA. LyseControl should turn completely colorless without any traces of blue.</LI><LI><B>Clarification of lysate.</B> Centrifuge for 5 min at 11,000 x g at room temperature. Repeat this step in case the supernatant is not clear.</LI><LI><B>Bind DNA.</B> Place a NucleoSpin® Plasmid/Plasmid (NoLid) Column in a Collection Tube (2 mL) and decant the supernatant from step 3 or pipette a maximum of 750 μL of the supernatant onto the column. Centrifuge for 1 min at 11,000 x g. Discard flow-through and place the NucleoSpin® Plasmid/Plasmid (NoLid) Column back into the collection tube. Repeat this step to load the remaining lysate.</LI><LI><B>Wash silica membrane.</B> Add 600 μL Buffer A4 (supplemented with ethanol, see section 3). Centrifuge for 1 min at 11,000 x g. Discard flow-through and place the NucleoSpin® Plasmid / Plasmid (NoLid) Column back into the empty collection tube.</LI><LI><B>Dry silica membrane.</B> Centrifuge for 2 min at 11,000 x g and discard the collection tube. Note: Residual ethanolic wash buffer might inhibit enzymatic reactions.</LI><LI><B>Elute DNA.</B> Place the NucleoSpin® Plasmid / Plasmid (NoLid) Column in a 1.5 mL microcentrifuge tube (not provided) and add 50 μL Buffer AE. Incubate for 1 min at room temperature. Centrifuge for 1 min at 11,000 x g.</LI><LI>Measure the concentration of DNA using nanophotometer. </LI></OL>
              Source: <A href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/Plasmid%20DNA%20Purification/UM_pDNA_NS.pdf ">Macherey-Nagel Plasmid DNA purification Manual </A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#transformation">
            Transformation
          </A></H5><DIV id="transformation" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>Resuspended DNA to be transformed</LI><LI>10 pg/µL positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the <A href="http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit">Competent Cell Test Kit</A>.)</LI><LI>Competent cells (50 µL per sample)</LI><LI>1.5 mL microtubes</LI><LI>SOC media (950 µL per sample)</LI><LI>Petri plates w/ LB agar and antibiotic (2 per sample)</LI></UL><U>Equipment</U><UL><LI>Ice and ice bucket</LI><LI>Lab Timer</LI><LI>42°C water bath</LI><LI>37°C incubator</LI><LI>Sterile spreader</LI><LI>Pipettes and tips (10 µL, 20µl, 200 µL, 1000 µL recommended)</LI><LI>Microcentrifuge</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Resuspend DNA in selected wells in the Distribution Kit with 10 µL dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.</LI><LI>Label 1.5 mL tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5 mL tubes (one tube for each transformation, including your control) in a floating foam tube rack.</LI><LI><B>Thaw competent cells on ice:</B> This may take 10-15 min for a 260 µL stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.</LI><LI><B>Pipette 50 µL of competent cells into 1.5 mL tube:</B> 50 µl in a 1.5 mL tube per transformation. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice. Don’t forget a 1.5 mL tube for your control.</LI><LI><B>Pipette 1 µL of resuspended DNA into 1.5mL tube:</B> Pipette from well into appropriately labeled tube. Gently pipette up and down a few times. Keep all tubes on ice.</LI><LI><B>Pipette 1µl of control DNA into 2ml tube:</B> Pipette 1µl of 10pg/µl control into your control transformation. Gently pipette up and down a few times. Keep all tubes on ice.</LI><LI><B>Close 1.5ml tubes, incubate on ice for 30min:</B> Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.</LI><LI><B>Heat shock tubes at 42°C for 45 sec:</B> 1.5ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are submerged. Timing is critical.</LI><LI><B>Incubate on ice for 5min:</B> Return transformation tubes to ice bucket.</LI><LI><B>Pipette 950µl SOC media to each transformation:</B> SOC should be stored at 4°C, but can be warmed to room temperature before use. Check for contamination.</LI><LI><B>Incubate at 37°C for 1 hours, shaking at 200-300rpm</B></LI><LI><B>Pipette 100µL of each transformation onto petri plates:</B> Spread with sterilized spreader or glass beads immediately. This helps ensure that you will be able to pick out a single colony.</LI><LI><B>Spin down cells at 6800g for 3mins and discard 800µL of the supernatant. Resuspend the cells in the remaining 100µL, and pipette each transformation onto petri plates.</B> Spread with sterilized spreader or glass beads immediately. This increases the chance of getting colonies from lower concentration DNA samples.</LI><LI><B>Incubate transformations overnight (14-18hr) at 37°C:</B> Incubate the plates upside down (agar side up). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.</LI></OL>
              Source: <A href="http://parts.igem.org/Help:Protocols/Transformation">iGEM homepage</A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#transformationquick">
            Transformation - quick protocol
          </A></H5><DIV id="transformationquick" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>Resuspended DNA to be transformed</LI><LI>10 pg/µl positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the <A href="http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit">Competent Cell Test Kit</A>.)</LI><LI>Competent cells (50µl per sample)</LI><LI>1.5 mL microtubes</LI><LI>SOC Media (950µL per sample)</LI><LI>Petri plates with LB agar and antibiotic (2 per sample)</LI></UL><U>Equipment</U><UL><LI>Ice and ice bucket</LI><LI>Lab timer</LI><LI>37 °C dry bath/heat block</LI><LI>37 °C incubator</LI><LI>Sterile spreader</LI><LI>Pipettes and tips (10µl, 20µl, 200µl, 1000µl recommended)</LI><LI>Microcentrifuge</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Resuspend DNA in selected wells in the Distribution Kit with 10 µL dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.</LI><LI>Label 1.5 mL tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5 mL tubes (one tube for each transformation, including your control) in a floating foam tube rack. Pre-heat the agar plates by putting them in the 37 °C incubator.</LI><LI><B>Thaw competent cells on ice:</B> This may take 10-15 min for a 260 µL stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.</LI><LI><B>Pipette 50 µL of competent cells into 1.5 mL tube:</B> 50 µL in a 1.5 mL tube per transformation. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice. Don’t forget a 1.5 mL tube for your control.</LI><LI><B>Pipette 1 µL of resuspended DNA into 1.5 mL tube:</B> Pipette from well into appropriately labeled tube. Tap the tube gently to mix. Keep all tubes on ice.</LI><LI><B>Pipette 1 µl of control DNA into 1.5ml tube:</B> Pipette 1µl of 50 pg/µl control into your control transformation. Tap the tube to mix.. Keep all tubes on ice.</LI><LI><B>Add 200 uL of SOC medium to the 1.5ml tubes and incubate on 37 °C for 30 min.</B></LI><LI><B>Pipette 75 uL of the incubated solution to pre-warmed agar plates</B> containing chloramphenicol. Spread with sterilized spreader or glass beads immediately. This helps ensure that you will be able to pick out a single colony.</LI><LI><B>Spin down 100 uL of the remaining cell solution at 6800 g for 3mins and discard 25 µL of the supernatant. Resuspend the cells in the remaining 75 µL, and pipette a transformation onto a petri plate.</B> Spread with sterilized spreader or glass beads immediately. This increases the chance of getting colonies from lower concentration DNA samples.</LI><LI><B>Incubate transformations overnight (14-18 hr) at 37 °C:</B> Incubate the plates upside down (agar side up). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.</LI><LI><B>After finished incubation put the plates in the fridge</B> unless you want to continue with creating the stock media right away.</LI></OL>
              Source: <A href="http://parts.igem.org/Help:Protocols/Transformation">iGEM homepage</A>, and <A href="http://www.zymoresearch.com/downloads/dl/file/id/166/t3001i.pdf">here (PDF)</A>.
            </DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#digestionandligation">
            Digestion and ligation
          </A></H5><DIV id="digestionandligation" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>linearised plasmid backbone pSB1C3</LI><LI>DNA to be transformed</LI><LI>gBlock fragments (25 ng/uL)</LI><LI>10pg/µl Positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the <A href="http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit">Competent Cell Test Kit</A>.)</LI><LI>1.5mL Microtubes</LI></UL><U>Equipment</U><UL><LI>Ice &amp; ice bucket</LI><LI>Lab Timer</LI><LI>80 °C heat block</LI><LI>37 °C heat block</LI><LI>Sterile spreader </LI><LI>Pipettes and Tips (2µl, 10µl, 20µl, 200µl recommended)</LI><LI>Microcentrifuge</LI></UL></DIV><DIV class="col-md-6"><U>Digestion</U><UL><LI>Enzyme Master Mix for Plasmid Backbone (25ul total, for 5 rxns)</LI><UL><LI>5 ul NEB Buffer 2</LI><LI>0.5 ul EcoRI-HF</LI><LI>0.5 ul PstI</LI><LI>19 ul dH20</LI></UL><LI>Digest Plasmid Backbone</LI><UL><LI>Add 4 ul linearized plasmid backbone (25ng/ul for 100ng total)</LI><LI>Add 4 ul of Enzyme Master Mix</LI><LI>Digest 37C/30 min, heat kill 80C/20 min</LI></UL></UL><U>Ligation</U><UL><LI>Add 2ul of digested plasmid backbone (25 ng)</LI><LI>Add equimolar amount of EcoRI-HF PstI digested fragment (6 ul)</LI><LI>Add 1 ul T4 DNA ligase buffer. <B>Note:</B> Do not use quick ligase</LI><LI>Add 0.5 ul T4 DNA ligase</LI><LI>Add water to 10 ul</LI><LI>Ligate 16C/over night, heat kill 80C/20 min</LI><LI>Transform with 1-2 ul of product, see transformation protocol</LI></UL></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#gelelectro">
            Agarose gel electrophoresis
          </A></H5><DIV id="gelelectro" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>70% ethanol</LI><LI>TBE Buffer</LI><LI>Agarose powder</LI><LI>Samples</LI><LI>Loading dye</LI><LI>Molecular Ladder</LI><LI>Gel Red</LI></UL><U>Equipment</U><UL><LI>Microwave</LI><LI>Pipettor</LI><LI>Pipette tips</LI><LI>Gel Box with Gel Tray</LI><LI>Microwaveable Flask</LI><LI>Measuring Glass</LI><LI>Scale</LI><LI>Weighing ship</LI><LI>Electrodes and Power Source</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Clean your working area by wiping down with 70% ethanol.</LI><LI>Weigh up 0.5g of agarose in a weighing ship for a 1% agarose gel.</LI><LI>Measure up 50mL of TBE buffer in a measuring glass.
                  <B>Note:</B> Make sure to use the same buffer as the one in the gel box (do not mix different buffers and do not use water).</LI><LI>Add the TBE and the agarose powder onto a microwaveable flask and swirl to mix.</LI><LI>Microwave the mixture for 1-3 min and check for clarity and complete dissolvation while swirling.
                  <B>Note:</B> Do not overboil the solution, as some of the buffer will evaporate and thus alter the final percentage of agarose in the gel. Many people prefer to microwave in pulses, swirling the flask occasionally as the solution heats up.</LI><LI>Once clear, let the agarose cool cown for some time (to not damage the gel trays), approximately 5 min.</LI><LI>Add 5uL of Gel Red. Swirl to mix. </LI><LI>Pour the agarose carefully onto the gel try with the well comb well attached and in place.
                  <B>Note:</B> Pour slowly to avoid bubbles which will disrupt the gel. Any bubbles can be pushed away from the well comb or towards the sides/edges of the gel with a pipette tip.</LI><LI>Let it solidify for approximately 30 min.</LI><LI>Remove the well comb from the solid gel and place it in the gel box (electrophoresis unit).</LI><LI>Fill up the gel box with running buffer to the maximum marking, or until the gel is completely covered.</LI><LI>Prepare the samples with loading dye.
                  <B>Note:</B> Loading dye serves two purposes: 1) it provides a visible dye that helps with gel loading and will also allows you to gauge how far the gel has run while you are running your gel; and 2) it contains a high percentage of glycerol, so it increases the density of your DNA sample causing it settle to the bottom of the gel well, instead of diffusing in the buffer.</LI><LI>Pipett a molecular ladded onto the first well and then continue with adding the samples onto the following wells.
                  <B>Note:</B> When loading the sample in the well, maintain positive pressure on the sample to prevent bubbles or buffer from entering the tip. Place the very top of the tip of the pipette into the buffer just above the well. Very slowly and steadily, push the sample out and watch as the sample fills the well. After all of the sample is unloaded, push the pipettor to the second stop and carefully raise the pipette straight out of the buffer.</LI><LI>Run the gel at 80-150 V until the dye line is approximately 75-80% of the way down the gel.
                  <B>Note:</B> Black is negative, red is positive. (The DNA is negatively charged and will run towards the positive electrode.) Always Run to Red.</LI><LI>Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.</LI><LI>Use any device that has UV light to visualize the DNA fragments on the gel.
                  <B>Note:</B> If you will be purifying the DNA for later use, use long-wavelength UV and expose for as little time as possible to minimize damage to the DNA.
                  <B>Note:</B> The fragments of DNA are usually referred to as 'bands’ due to their appearance on the gel.</LI></OL>
                Source: <A href="https://www.addgene.org/protocols/gel-electrophoresis/ ">Addgene Agarose Gel Electrophoresis protocol</A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#agarosednaextraction"><H5>
            DNA extraction from agarose gels
           </H5></A><DIV id="agarosednaextraction" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>NucleoSpin® Gel and PCR Clean-up Kit</LI><LI>Sterile MQ water</LI><LI>agarose gel with the DNA sample to be extracted</LI></UL><U>Equipment</U><UL><LI>Centrifuge</LI><LI>Scalpel</LI><LI>Vortex</LI><LI>Lab timer</LI><LI>Balance</LI><LI>Pipettes and Tips (1000 µL and 500 µL recommended)</LI><LI>1.5 mL microcentrifuge tubes</LI><LI>Nanophotometer</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI><B>Excise DNA fragment / solubilize gel slice.</B> Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose. Determine the weight of the gel slice and transfer it to a clean tube. </LI><LI>For each 100 mg of agarose gel &lt;  2 % add 200 μL Buffer NTI. For gels containing &gt; 2 % agarose, double the volume of Buffer NTI. Incubate sample for 5–10 min at 50 °C. </LI><LI>Vortex the sample briefly every 2–3 min until the gel slice is completely dissolved!</LI><LI><B>Bind DNA.</B> Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 mL) and load up to 700  μL sample. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube. Load remaining sample if necessary and repeat the centrifugation step.</LI><LI><B>Wash silica membrane.</B> Add 700 μL Buffer NT3 to the NucleoSpin® Gel  and PCR Clean-up Column. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube. Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A230</LI><LI><B>Dry silica membrane.</B> Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube. Note: Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70 °C prior to elution.</LI><LI><B>Elute DNA.</B> Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 mL microcentrifuge tube (not provided). Add 15–30 μL Buffer NE and incubate at room temperature (18–25 °C) for 1 min. Centrifuge for 1 min at 11,000 x g.</LI><LI>Measure the DNA concentration using nanophotometer</LI></OL>
              Source: <A href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/DNA%20clean-up/UM_PCRcleanup_Gelex_NSGelPCR.pdf"> Macherey-Nagel NucleoSpin® Gel and PCR Clean-up Manual</A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#cellprep">
            Preparation of competent <I>E. coli</I> cells
          </A></H5><DIV id="cellprep" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>DMSO</LI><LI>5 M PIPES pH 6.7 (adjust with KOH or HCl)</LI><LI>Inoue transformation buffer chilled to 0°C </LI><LI>LB media</LI><LI>1.5mL Microtubes</LI></UL><U>Equipment</U><UL><LI>Ice &amp; ice bucket</LI><LI>Lab Timer</LI><LI>liquid nitrogen</LI><LI>42°C water bath</LI><LI>18°C incubator</LI><LI>Pipettes and Tips (10µl, 20µl, 200µl recommended)</LI><LI>Pipettor</LI><LI>Pipettes (10 mL, 50 mL)</LI><LI>Microcentrifuge</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><B>Day 1: Growing Bacterial Cultures</B><OL><LI>Pick a single bacterial colony (2-3 mm in diameter) from a plate that has been incubated for 16-20 hours at 37°C.</LI><LI>Transfer colony into 25 mL of LB broth in 250 mL flask. </LI><LI>Incubate culture of 6-8 hours at 37°C with vigorous shaking (250-300 rpm).</LI><LI>Inoculate three 500 mL flasks of 100 mL LB using the below volumes of this starter culture.</LI><LI>Incubate all three flasks overnight at room temperature (18-22°C) with moderate shaking (180 rpm).</LI><TABLE style="width:100%;"><TBODY><TR><TH>Flask </TH><TH>Vol. of starter culture</TH></TR><TR><TD>1</TD><TD>100 uL</TD></TR><TR><TD>2</TD><TD>20 uL</TD></TR><TR><TD>3</TD><TD>10 uL</TD></TR></TBODY></TABLE></OL><B>Day 2: Harvesting Cells and Freezing Competent Cells</B><OL><LI>Read the OD600 of all three cultures. Continue to monitor evey 45 min until reading is at 0.55</LI><LI>Transfer the culture vessel to ice water bath for 10 min.</LI><LI>Harvest cells by centrifugation at 3900 rpm for 10 min at 4°C in 50 mL falcon tube.</LI><LI>Pour off medium and dry the tube (inverted) on paper towels for 2 min (use vacuum aspirator to remove any drops of remaining medium adhering to walls of the centrifuge bottle or trapped in its neck) </LI><LI>Resuspend the cells gently (by swirling) in 32 mL of ice-cold (0°C) Inoue transformation buffer. </LI><LI>Harvest cells by centrifugation at 2500 g for 10 min at 4°C. </LI><LI>Pour off the medium and dry the tube on paper towels for 2 min (use vacuum aspirator to remove any drops of remaining medioum adhering to walls of the centrifuge bottle or trapped in its neck).</LI><LI>Resuspend cells GENTLY in 8 mL of Inoue transformation buffer (0°C).</LI><LI>Add 0.6 mL of DMSO (swirl to mix bacterial suspension).</LI><LI>Store on ice for 10 min.</LI><LI>Quickly dispense 50 uL aliquots of suspensions into chilled, sterile microcentrifuge tubes (20 mL of suspension equals 400 tubes of 50 uL).</LI><LI>Snap freeze competent cells in liquid nitrogen (store stock at -80°C).</LI><LI>When needed, remove tube of competent cells from freezer, use immediately.</LI></OL>
              Source: <A href="https://www.princeton.edu/genomics/mcclean/protocols/Inoue-Method-for-Preparation-of-Competent-E-Coli.pdf">McClean Lab Protocol &quot;The Inoue Method for Preparation of Competent <I>E.Coli</I> “Ultra‐competent” Cells”, Princeton. </A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><H5><A data-toggle="collapse" data-parent="#experiments" href="#pcr">
            PCR
          </A></H5><DIV id="pcr" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>70% ethanol</LI><LI>Milliq Water</LI><LI>Q5 High-Fidelity 2X Master Mix</LI><LI>10uM Forward Primer</LI><LI>10uM Reverse Primer</LI><LI>Template DNA</LI></UL><U>Equipment</U><UL><LI>PCR Machine</LI><LI>Pipettor</LI><LI>Pipette tips</LI><LI>Vortexer</LI><LI>Table Top Centrifuge</LI><LI>PCR Tubes</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Clean your working area by wiping down with 70% ethanol.</LI><LI>Label one PCR tube for each desired reaction. </LI><LI>Prepare a PCR reaction for each colony. Easiest is to prepare a Master Mix with Q5 High-Fidelity 2X Master Mix, Milliq Water and possibly primers and then aliquot into the reaction tubes. Start by adding the lowest amount (the template DNA).
                  <TABLE style=""><TBODY><TR style="border-bottom: 1px solid black"><TH>Component</TH><TH>12.5 uL reaction</TH><TH>25 uL reaction</TH><TH>50uL reaction</TH><TH>Final Concentration</TH></TR><TR><TD>Q5 High-Fidelity 2X Master Mix</TD><TD>6.25 uL</TD><TD>12.5 uL</TD><TD>25 uL</TD><TD>1X</TD></TR><TR><TD>10uM Forward Primer</TD><TD>0.75 uL</TD><TD>1.25uL</TD><TD>2.5 uL</TD><TD>0.5 uM</TD></TR><TR><TD>10uM Reverse Primer</TD><TD>0.75 uL</TD><TD>1.25uL</TD><TD>2.5 uL</TD><TD>0.5 uM</TD></TR><TR><TD>Milliq Water</TD><TD>3.75 uL</TD><TD>9.5 uL</TD><TD>19 uL</TD></TR><TR><TD>Template DNA</TD><TD>1 uL</TD><TD>0.5 uL</TD><TD>1 uL</TD><TD>&lt;1,000 ng</TD></TR></TBODY></TABLE></LI><LI>Mix throuroghly and spin down the PCR reactions.</LI><LI>Place the tubes in the PCR Machine and use the following settings:
                  </LI><LI>Initial denaturation - 98°C for 30 sec.. </LI><LI>Separation of strands - 98°C for 10 sec.</LI><LI>Annealing of primers – Tm of primer (approx. 55-62°C) for 30 sec.</LI><LI>Elongation – 72°C for 30 sec/kb. </LI><LI>Repeat step 1-4 for 30 cycles.</LI><LI>Final extension - 72°C for 10 min. </LI><LI>Hold - 4°C for infinite time. </LI></OL><I>Note: If smearing appears on gel, it could be due to too many cycles – optimize between 25-35 cycles. If no correct bands appear on gel, try doing a temperature gradient PCR to optimixe the annealing temperature of the primers.</I><LI>Once the PCR tubes are taken out of the PCR Machine, turn it off and leave the lid closed until it has reached room temperature, then open it and leave the lid open for the next user.</LI>
              Source: <A href="https://www.neb.com/protocols/2012/12/07/protocol-for-q5-high-fidelity-2x-master-mix-m0492">Protocol for Q5® High-Fidelity 2X Master Mix by Neb</A>.
            </DIV></DIV></DIV></DIV><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#cpcr"><H5>
            Colony PCR
           </H5></A><DIV id="cpcr" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>70% ethanol</LI><LI>Transformed cells on plate</LI><LI>Milliq Water</LI><LI>Q5 High-Fidelity 2X Master Mix</LI><LI>10uM Forward Primer</LI><LI>10uM Reverse Primer</LI><LI>Template DNA</LI><LI>1% Agarose Gel</LI></UL><U>Equipment</U><UL><LI>PCR Machine</LI><LI>Pipettor</LI><LI>Pipette tips</LI><LI>Vortexer</LI><LI>Table Top Centrifuge</LI><LI>PCR Tubes</LI><LI>Agarose Gel Equipment</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Clean your working area by wiping down with 70% ethanol.</LI><LI>Label one PCR tube for each colony, at least three colonies should be picked for screening per plate.</LI><LI>Add 10uL of Milliq water to each labelled PCR tube.</LI><LI>Pick the colonies carefully using a pipett tip and resuspend them into the prepared PCR tubes. </LI><LI>Prepare a 12.5uL PCR reaction for each colony, where the addition of template DNA is taken from the PCR tubes with the resuspended colonies and 1uL should be added. </LI><LI>Mix throuroghly and spin down the PCR reactions.</LI><LI>Place the tubes in the PCR Machine and use the following settings:
                  </LI><LI>Initial denaturation - 98°C for 10 min.</LI><LI>Separation of strands - 98°C for 10 sec.</LI><LI>Annealing of primers – Tm of primer (approx. 55-62°C) for 30 sec.</LI><LI>Elongation – 72°C for 30 sec/kb. </LI><LI>Repeat step 1-4 for 30 cycles.</LI><LI>Final extension - 72°C for 10 min. </LI><LI>Hold - 4°C for infinite time. </LI></OL><I>Note: If smearing appears on gel, it could be due to too many cycles – optimize between 25-35 cycles. If no correct bands appear on gel, try doing a temperature gradient PCR to optimixe the annealing temperature of the primers. Also, cPCR is not enough accurate – so sending for sequencing could still be an idea even if no bands appear on the gel.</I><LI>Once the PCR tubes are taken out of the PCR Machine, turn it off and leave the lid closed until it has reached room temperature, then open it and leave the lid open for the next user.</LI><LI>Prepare a 1%-agarose gel with the number of wells corresponding to the number of colonies screened.</LI>
              Source: <A href=" https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492">here</A></DIV></DIV></DIV></DIV><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#freeze"><H5>
            Freeze-thaw protocol for protein extraction
           </H5></A><DIV id="freeze" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>Dry ice </LI>
                Sterile MQ water
                <LI>Overnight cell culture expressing the protein of interest</LI><LI>Lysis buffer</LI></UL><U>Equipment</U><UL><LI>Centrifuge</LI><LI>Vortex</LI><LI>Lab timer</LI><LI>Pipettes and Tips (100 µL and 1000 µL recommended)</LI><LI>1.5 mL microcentrifuge tubes</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Take as many 1 mL samples from the cell culture at OD<SUB>600</SUB> of 1 as experiments you would like to carry out. If the OD of the cell culture is not 1 you have to correct the sample volume to obtain an equivalent size pellet.</LI><LI>Spin down the cells for 5 min at 6000 rpm in a centrifuge.</LI><LI>To each cell pellet, add 100 uL of lysis buffer.</LI><LI>Vortex to resuspend the cells.</LI><B>Freeze-thaw</B><LI>Freeze quickly on ice and leave for 3 min</LI><LI>Thaw immediately at 42 °C and voretx vigorously to mix well. Repeat the steps 5. and 6. for three more times (4 freeze-thaw-vortex cycles in total)</LI><LI>Spin the tubes for 5 min at maxiumum speed in a centrifuge</LI><LI>Separate the supernatant (contains soluble protein) from the pellet (contains insoluble protein) by pipetting out the supernatant to a clean tube</LI><LI>Label the tubes accordingly</LI><LI>To each supernatant, add 1 mL acetone and vortex. Freeze or leave on ice for 15 min. </LI><LI>Spin 5 min at maximum speed</LI><LI>Remove acetone by pipetting it out, being careful not to disturb the pellet</LI><LI>Dry at 37 °C</LI><LI>To the acetone-treated pellet, add 30 uL SDS-PAGE loading buffer</LI><LI>To the cell pellet, add 40 uL SDS-PAGE loading buffer</LI><LI>Heat all samples to 95 °C (or greater) for 5 min</LI><LI>Vortex and centrifuge 5 min at maximum speed</LI><LI>Load 5-10 uL on a SDS-PAGE gel, taking sample from the top and avoiding any pellet</LI></OL>
              Source: Based on the protocol from Jeanne Perry, Molecular Biology Institute, UCLA, Los Angeles, USA
            </DIV></DIV></DIV></DIV><DIV class="well experiment-container"><A data-toggle="collapse" data-parent="#experiments" href="#sds"><H5>
            SDS-PAGE electrophoresis
           </H5></A><DIV id="sds" class="collapse"><DIV class="row"><DIV class="col-md-6"><U>Material</U><UL><LI>Bio-Rad precasted 4-20% polyacrylamide gels (10 wells, 50µL sample/well)</LI><LI>SDS destaining solution</LI><LI>Protein ladder</LI><LI>SDS Loading buffer</LI><LI>Cell pellet</LI><LI>10 x Running buffer (Tris base 30.3g, glycine 144g, SDS 10g, MQ H2O to 1L)</LI><LI>MQ H2O</LI></UL><U>Equipment</U><UL><LI>Heat block</LI><LI>1.5mL micro tubes</LI><LI>Lab timer</LI><LI>Pipettes and Tips (100µL and 500µL recommended)</LI><LI>Gel cassette and electrical leads</LI><LI>Orbital shaker/rocker</LI></UL></DIV><DIV class="col-md-6"><U>Procedure</U><OL><LI>Add 10µL 2x SDS-PAGE loading buffer to 10µL of cell pellet and boil in a heat block at 95°C for 5 min (or at 70°C for 10 min) .</LI><LI>Assemble the BioRad apparatus by putting in the precasted gel and add 500 mL of 1X running buffer (dilute 50mL 10X stock with 450mL MQ H2O) so that the gel is completely covered.</LI><LI>Remove the comb from the top of the gel. </LI><LI>Load the samples on the gel slowly, so that they have time to settle evenly. The first well should be filled with 5µL of protein ladder. </LI><LI>Run the gel at RT for 50-60 min on 100V or until the blue dye reaches the end of the gel. Note. Make sure the electrical leads are in the right position (when correct, the bubbles will start to form).</LI><LI>After electrophoresis pop open the gel cassette and slide the gel into the MQ water. </LI><LI>Staining of the gel. Wash the gel three times in 200mL MQ H2O for 5 min.</LI><LI>Remove the water from the staining container and add 50mL of staining solution to cover the gel.</LI><LI>Agitate at RT for 1 hour.</LI><LI>Destaining. Rinse in 200 mL destaining solution for minimum 30 min or until destained. Filtered destaining solution can be reused. After destaining the gel can be stored in water.</LI></OL></DIV></DIV></DIV></DIV></SECTION></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>