Team:Lund/Design/Design
Toggle navigation
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Results
InterLab
Parts
Protocols
Notebook
Model
Safety
Practices
Design
Overview
Animation
Theory
Project Design
Circuit
Experiments
References
Designing our system
A novel biosensor for the detection of microplastics was realized using a top-down black-box approach. Microplastics were identified as the target input and a reporter protein was designated as the output, as depicted in fig. 1. Due to their unreactive nature and varied composition, microplastics were quickly deemed a difficult set of molecules to identify using biological sensing elements, calling for an indirect method of identification if tools of synthetic biology were to be utilized. As discussed in the project description , successfully developing such a device would offer a rapid and cost-effective solution in contrast to the arduous filtration processes that are used today.
Expanding on the groundwork put forward by the UCL’12 iGEM team, establishing the presence of molecules associated with water-bound microplastics was determined to be a viable approach [9]. Two classes of molecules were identified as being commonly associated with microplastics – plasticizers and organic pollutants. They are both lipophilic compounds, allowing trans-membrane diffusion and subsequent intracellular recognition. Escherichia coli was chosen as a functional chassis for the biosensor; not only due to its compatibility with previously developed parts and devices but also as facilitated transmembrane transport of phthalates and organic pollutants has been reported through the FadL and subsequent FadD channels [33] [34]. The system updates are reflected in fig. 2.
Homology analysis of the two nominated compounds and natural ligands to existing sensory elements generated two transcriptional regulators – NahR with affinity for the most common moieties found in organic pollutants and the human estrogen receptor alpha with noted antagonistic interaction with plasticizers [9] [24]. Both NahR and hER-α serve as transactivators in presence of respective cofactors [15] [27]. Both the full-length hER-α and truncated versions have been successfully transformed into E. coli with adequate expression rates, even if the former has shown toxicity at higher expression rates, and NahR has been utilized as an inducible transcriptional factor in multiple iGEM projects [9] [35] [36] [37] [38] [39]. Thus, they were chosen as the sensory elements of our circuit, as shown in fig. 3.
To increase the selectivity of the sensor and thus the accuracy of the output readings, the input signals corresponding to the sensing of phthalates and organic pollutants must transform according to that of a logic AND-gate, as illustrated in fig. 4. That is to say, the reporter will only be measureable in the instance that phthalates and organic pollutants are both present in the sample. In an attempt to counteract the potential toxicity of introducing and expressing the full-length hER-α in E. coli, the truncated version hER-α LBD was finalized as the phthalate-sensing element to be applied in a complementation assay through domain insertion, in accordance with previous prosperous hER-α biosensor studies as noted in the theory section [29] [30] [31].
Employing the conformational change of the hER-α LBD in a genetic AND-gate can be achieved through a plethora of different approaches. However, in each instance a fusion protein needs to be constructed that capitalizes on the ligand-induced convergence of the termini. Furthermore, rather than connecting the NahR and the hER-α LBD in series in the circuit architecture, implementing their expression concurrently would decrease the reporter synthesis response time. These conditions were met using a tripartite split GFP-protein, as schematically shown in fig. 5. Two GFP gene residues, corresponding to beta-sheet 10 and 11 of the GFP beta barrel motif, were attached to each respective side of the hER-α LBD with appropriate linkers. The remaining GFP, GFP1-9, was put downstream Psal – under regulation of NahR. Cabantous et al. iteratively mined the optimal split sites of the GFP for spontaneous self-assembly upon association [40]. Thus  , upon expression of the GFP10- hER-α LBD-GFP11 and GFP1-9, and subsequent antagonistic ligand interaction with the hER-α LBD, self-assembly of the fluorophore ought to occur. Similar results have been reported with a two-split GFP hER-α LBD fusion protein [30] [41]. The added benefit of utilizing a tripartite split GFP with smaller GFP-tags affixed to the hER-α LBD is that somewhat equidistantly split proteins have generally shown either high levels of aggregation or high background noise [30].
To alleviate the host of constant expression of the heterologous proteins, both nahR and the GFP10- hER-α LBD-GFP11 were put downstream lac o with the lac repressor lacI constitutively expressed. Thus, the system remains repressed until addition of a ligand inducing an allosteric shift and subsequent disassociation of the repressor [42]. The final system model can be found in fig. 6.
Design of fusion protein linkers
The choice of linkers was evaluated using the open-source software PyMOL. Crystal structures of the human estrogen receptor alpha were acquired at the NCBI gene bank [43] [44]. The sequence of ligand-free hER-α LBD and antagonist-hER-α LBD complex were analyzed and the distance between the two termini was measured using the PyMOL wizard tool. The former measured a total distance of approximately 38 Å with clear steric hindrance while the latter approximately 21 Å with limited to no steric hindrance. Thus, only linkers longer than 11 Å were considered adequate candidates. Ultimately, glycine-serine linkers were chosen due to their flexibility, which was needed for mobility of the two GFP fragments [45] [46]. Two different linker sizes were picked, corresponding to 6 a.a and 12 a.a respectively.
Figure 7: A model of the hER-α LBD without an associated ligand. The 3D protein structure was drawn using the PyMOL software. As depicted, there is a clear steric hindrance between the N- and the C-terminal with a total distance of approximately 38 Å.
Figure 8: A model of the hER-α LBD without an associated ligand. The 3D protein structure was drawn using the PyMOL software. There is no obvious steric hindrance between the N- and the C-terminal with a total distance of approximately 21 Å.
Fig. 1: System model 1, input signal defined.
Fig. 2: System model 2, input characteristics specified, chassi chosen.
Fig. 3: System model 3, sensory components specified.
Fig. 4: System model 4, output characteristics specified. For the sake of simplicity, ER-α will be used synonymously with ER-α-LBD.
Fig. 5: System model 5, sensory component relation specified.
Fig. 6: System model 6, complete genetic circuit.
Home
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Results
InterLab
Parts
Protocol
Notebook
Model
Results
Model derivation
Parameter estimation
Implementation
Safety
Safety
Practices
Overview
Integrated
Engagement
Media
Communication
Contact
Sölvegatan 22 A
223 62 Lund
Sweden
igemlund@gmail.com
Stay connected
Facebook
Instagram
Twitter
©2017 iGEM Lund
