<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Design_References skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Design/References</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV><HEADER class="design-header"><DIV class="container"><H1 class="display">Design</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Design">Overview</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Animation">Animation</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Theory">Theory</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Design">Project Design</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Circuit">Circuit</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Experiments">Experiments</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Design/References">References</A></LI></UL></DIV><DIV class="content-wrapper under-nav text-layout"><DIV class="container"><SECTION><H3 class="section-heading">References</H3><OL class="references"><LI id="ref1">
          [1] Mehrotra, P. (2016) Biosensors and their applications - A review. <EM>Journal of Oral Biology and Craniofacial Research, </EM>6(2), pp.153-159.
        </LI><LI id="ref2">
          [2] Bhalla, N., Jolly, P., Formisano, N. and Estrela, P. (2016) Introduction to biosensors. <EM>Essays in Biochemistry,</EM> 60(1), pp.1-8.
        </LI><LI id="ref3">
          [3] Sørensen, S. J., Burmølle, M., Hansen, L. H. (2006) Making bio-sense of toxicity: new developments in whole-cell biosensors. <EM>Current Opinion in Biotechnology.</EM> 17(1), pp.11-16
        </LI><LI id="ref4">
          [4] Ma, S., Tang, N., and Tian, J. (2012) DNA Synthesis, Assembly and Applications in Synthetic Biology. <EM>Current Opinion in Chemical Biology, </EM>16(3-4), pp.260-267.
        </LI><LI id="ref5">
          [5] Carlson, R. (2009) The changing economics of DNA synthesis. Nature Biotechnology, 27(12), pp.1091-1094.
        </LI><LI id="ref6">
          [6] Cameron, D. E., Bashor, C. J. and Collins, J. J. (2014) A brief history of synthetic biology. <EM>Nature Reviews Microbiology</EM>, 12(5), pp.381-390.
        </LI><LI id="ref7">
          [7] Wright, S. and Kelly, F. (2017). Plastic and Human Health: A Micro Issue? <EM>Environmental Science &amp; Technology</EM>, 51(12), pp.6634-6647.
        </LI><LI id="ref8">
          [8] Anderson, J., Palace, V. and Park, B. (2016) Microplastics in aquatic environments: Implications for Canadian ecosystems. <EM>Environmental Pollution</EM>, 218, pp.269-280.
        </LI><LI id="ref9">
          [9] iGEM team University College London 2012 (2012). Welcome to Plastic Republic. [online] [Accessed 1 Sep. 2017].
        </LI><LI id="ref10">
          [10] Li, Q. Q., Loganath, A., Chong, Y. S., Tan, J. and Obbard, J. P. (2006) Persistent organic pollutants and adverse health effects in humans. <EM>Journal Of Toxicology And Environmental Health. Part A, </EM>69(21), pp.1987-2005.
        </LI><LI id="ref11">
          [11] Stockholm Convention ‘Status of ratification’ (2017) [online] [Accessed 2 oct. 2017]
        </LI><LI id="ref12">
          [12] Viktor Shatalov, Knut Breivik, Torunn Berg, Sergey Dutchak and Jozef Pacyna (2004) Convention on Long-range Transboundary Air Pollution EMEP Assessment Part I. [online] [Accessed 1 sep. 2017].
        </LI><LI id="ref13">
          [13] Schell, M. (1986) Homology between nucleotide sequences of promoter regions of nah and sal operons of NAH7 plasmid of Pseudomonas putida. <EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, 83(2), pp.369-373.
        </LI><LI id="ref14">
          [14] Maddocks, S. E., Oyston, P. C. F. (2008)  Structure and function of the LysR-type transcriptional regulator (LTTR) family proteins. <EM>Microbiology-sgm</EM>, 154, pp.3609-3623.
        </LI><LI id="ref15">
          [15] Park, H. H., Lim, W. K. and Shin H. J. (2005) In vitro binding of purified NahR regulatory protein with promoter <EM>Psal. BBA - General Subjects, </EM>1725(2), pp.247-255.
        </LI><LI id="ref16">
          [16] Schell, M. and Poser, E. (1989). Demonstration, characterization, and mutational analysis of NahR protein binding to nah and sal promoters. <EM>Journal of Bacteriology</EM>, 171(2), pp.837-846.
        </LI><LI id="ref17">
          [17] Grimm, A. C.,  Harwood, C. S. (1999) NahY, a Catabolic Plasmid-Encoded Receptor Required for Chemotaxis of Pseudomonas putida to the Aromatic Hydrocarbon Naphthalene. <EM>Journal of Bacteriology</EM>, 181(10), pp.3310-3316.
        </LI><LI id="ref18">
          [18] Cebolla, A., Sousa, C. and de Lorenzo, V. (1997). Effector Specificity Mutants of the Transcriptional Activator NahR of Naphthalene Degrading Pseudomonas Define Protein Sites Involved in Binding of Aromatic Inducers. <EM>Journal of Biological Chemistry</EM>, 272(7), pp.3986-3992.
        </LI><LI id="ref19">
          [19] Bui, T., Giovanoulis, G., Cousins, A., Magnér, J., Cousins, I. and de Wit, C. (2016). Human exposure, hazard and risk of alternative plasticizers to phthalate esters. <EM>Science of The Total Environment</EM>, 541, pp.451-467.
        </LI><LI id="ref20">
          [20] J.S Harisson Piezoelectric polymers vol. 3.
        </LI><LI id="ref21">
          [21] Mathieu-Denoncourt, J., Wallace, S., de Solla, S. and Langlois, V. (2015). Plasticizer endocrine disruption: Highlighting developmental and reproductive effects in mammals and non-mammalian aquatic species. <EM>General and Comparative Endocrinology</EM>, 219, pp.74-88.
        </LI><LI id="ref22">
          [22] Bhatia, H., Kumar, A., Du, J., Chapman, J. and McLaughlin, M. (2013). Di-n-butyl phthalate causes antiestrogenic effects in female murray rainbowfish (Melanotaenia fluviatilis). <EM>Environmental Toxicology and Chemistry</EM>, 32(10), pp.2335-2344.
        </LI><LI id="ref23">
          [23] Zhang, Z., Hu, Y., Zhao, L., Li, J., Bai, H., Zhu, D. and Hu, J. (2011) Estrogen agonist/antagonist properties of dibenzyl phthalate (DBzP) based on <EM>in vitro</EM> and <EM>in vivo</EM> assays. Toxicology Letters,  207(1), pp.7-11.
        </LI><LI id="ref24">
          [24] Okubo, T., Suzuki, T., Yokoyama, Y., Kano, K. and Kano, I. (2003). Estimation of Estrogenic and Anti-estrogenic Activities of Some Phthalate Diesters and Monoesters by MCF-7 Cell Proliferation Assay in Vitro. <EM>Biological &amp; Pharmaceutical Bulletin</EM>, 26(8), pp.1219-1224.
        </LI><LI id="ref25">
          [25] Oehlmann, J., Schulte-Oehlmann, U., Kloas, W., Jagnytsch, O., Lutz, I., Kusk, K., Wollenberger, L., Santos, E., Paull, G., Van Look, K. and Tyler, C. (2009). A critical analysis of the biological impacts of plasticizers on wildlife. <EM>Philosophical Transactions of the Royal Society B: Biological Sciences</EM>, 364(1526), pp.2047-2062.
        </LI><LI id="ref26">
          [26] Kumar, R, Zakharov, M, Miki, S, Miki, R, Jang, H, Toraldo, G, Singh, R, Bhasin, S, and Jasuja, R. (2011) <EM>The Dynamic Structure of the Estrogen Receptor</EM>. Journal Of Amino Acids, pp. 1-7.
        </LI><LI id="ref27">
          [27] Brzozowski, A. M. and Pike, A. C. W. (1997) Molecular basis of the agonism and antagonism of the oestrogen receptor. <EM>Nature</EM>, 389(6652), pp.753-758.
        </LI><LI id="ref28">
          [28] Allegretto E.A. (2002) Ligand-Induced Conformational Changes in Estrogen Receptors-α and -β. In: Manni A., Verderame M.F. (eds) Selective Estrogen Receptor Modulators. <I>Contemporary Endocrinology</I>. Humana Press, Totowa, NJ.
        </LI><LI id="ref29">
          [29] Paulmurugan, R. and Gambhir, S. (2006). An intramolecular folding sensor for imaging estrogen receptor-ligand interactions. <EM>Proceedings of the National Academy of Sciences</EM>, 103(43), pp.15883-15888.
        </LI><LI id="ref30">
          [30] McLachlan, M., Katzenellenbogen, J. and Zhao, H. (2011). A new fluorescence complementation biosensor for detection of estrogenic compounds. <EM>Biotechnology and Bioengineering</EM>, 108(12), pp.2794-2803.
        </LI><LI id="ref31">
          [31] Makhlynets, O. and Korendovych, I. (2014). Design of Catalytically Amplified Sensors for Small Molecules. <EM>Biomolecules</EM>, 4(2), pp.402-418.
        </LI><LI id="ref32">
          [32] Sekar, T., Foygel, K., Massoud, T., Gambhir, S. and Paulmurugan, R. (2016). A transgenic mouse model expressing an ERα folding biosensor reveals the effects of Bisphenol A on estrogen receptor signaling. <EM>Scientific Reports</EM>, 6(1).
        </LI><LI id="ref33">
          [33] Academic Press Encyclopedia of Microbiology [2009] : “Applied microbiology – industrial” p 447.
        </LI><LI id="ref34">
          [34] van den Berg, B. (2010). Going Forward Laterally: Transmembrane Passage of Hydrophobic Molecules through Protein Channel Walls. <EM>ChemBioChem</EM>, 11(10), pp.1339-1343.
        </LI><LI id="ref35">
          [35] Zhang, C., Glenn, K., Kuntz, M. and Shapiro, D. (2000). High level expression of full-length estrogen receptor in Escherichia coli is facilitated by the uncoupler of oxidative phosphorylation, CCCP. <EM>The Journal of Steroid Biochemistry and Molecular Biology</EM>, 74(4), pp.169-178.
        </LI><LI id="ref36">
          [36] Mader, S., Chambon, P. and White, J. (1993). Defining a minimal estrogen receptor DNA binding domain. <EM>Nucleic Acids Research</EM>, 21(5), pp.1125-1132.
          </LI><LI id="ref37">
          [37] Gangloff, M., Ruff, M., Eiler, S., Duclaud, S., Wurtz, J. and Moras, D. (2001). Crystal Structure of a Mutant hERα Ligand-binding Domain Reveals Key Structural Features for the Mechanism of Partial Agonism. <EM>Journal of Biological Chemistry</EM>, 276(18), pp.15059-15065.
        </LI><LI id="ref38">
          [38] iGEM team Peking (2013). Aromatic Scouts. [online] [Accessed 5 oct. 2017].
        </LI><LI id="ref39">
          [39] iGEM team NYMU Taipei (2015). Fight the Blight. [online] [Accessed 5 oct. 2017].
        </LI><LI id="ref40">
          [40] Cabantous, S., Nguyen, H., Pedelacq, J., Koraïchi, F., Chaudhary, A., Ganguly, K., Lockard, M., Favre, G., Terwilliger, T. and Waldo, G. (2013). A New Protein-Protein Interaction Sensor Based on Tripartite Split-GFP Association. <EM>Scientific Reports</EM>, 3(1).
        </LI><LI id="ref41">
          [41] Kodama, Y. and Hu, C. (2010). An improved bimolecular fluorescence complementation assay with a high signal-to-noise ratio. <EM>BioTechniques</EM>, 49(5), pp.793-805.
        </LI><LI id="ref42">
          [42] Santillan, M. and Mackey, M. (2008). Quantitative approaches to the study of bistability in the <EM>lac</EM> operon of <EM>Escherichia coli</EM>. <EM>Journal of The Royal Society Interface</EM>, 5(Suppl 1), pp.S29-S39.
        </LI><LI id="ref43">
          [43] Eiler, S, Gangloff, M, Duclaud, S, Moras, D, and Ruff, M. (2001) Overexpression, purification, and crystal structure of native ER alpha LBD. <I>Protein Expression And Purification</I>, 22(2), pp.165-173
        </LI><LI id="ref44">
          [44] Fanning, S, Green, B, Panchamukhi, S, Greene, G, Mayne, C, Tajkhorshid, E, Carlson, K, Martin, T, Katzenellenbogen, J, Dharmarajan, V, Novick, S, Griffin, P, Toy, W, Chandarlapaty, S, Katzenellenbogen, B, and Shen, Y. (2016) Estrogen receptor alpha somatic mutations Y537S and D538G confer breast cancer endocrine resistance by stabilizing the activating function-2 binding conformation. Elife, 5.
        </LI><LI id="ref45">
          [45] Chen, X., Zaro, J. and Shen, W. (2013). Fusion protein linkers: Property, design and functionality. <EM>Advanced Drug Delivery Reviews</EM>, 65(10), pp.1357-1369.
        </LI><LI id="ref46">
          [46] Kim, J, Lee, J, Song, D, Youn, S, Kim, J, Cho, G, Kim, S, Lee, H, &amp; Jin, M. (2016) Crystal structures of mono- and bi-specific diabodies and reduction of their structural flexibility by introduction of disulfide bridges at the Fv interface. <EM>Scientific Reports</EM>, 6.
        </LI><LI id="ref47">
          [47] AS Rose, AR Bradley, Y Valasatava, JM Duarte, A Prlić and PW Rose. Web-based molecular graphics for large complexes. <EM>ACM Proceedings of the 21st International Conference on Web3D Technology (Web3D '16)</EM>: 185-186, 2016.
        </LI><LI id="ref48">
          [48] AS Rose and PW Hildebrand. NGL Viewer: a web application for molecular visualization. <EM>Nucl Acids Res</EM> (1 July 2015) 43 (W1): W576-W579 first published online April 29, 2015.
        </LI><LI id="ref49">
          [49] Guex, N., Peitsch, M.C. Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. <EM>Electrophoresis</EM>, 30(S1), S162-S173.
        </LI><LI id="ref50">
          [50] Bienert S, Waterhouse A, de Beer TA, Tauriello G, Studer G, Bordoli L, Schwede T (2017). The SWISS-MODEL Repository - new features and functionality <EM>Nucleic Acids Res.</EM> 45(D1):D313-D319.
        </LI><LI id="ref51">
          [51] Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. <EM>Nucleic Acids Res.</EM> 37, D387-D392.
        </LI><LI id="ref52">
          [52] Kopp J, and Schwede T (2006). The SWISS-MODEL Repository: new features and functionalities. <EM>Nucleic Acids Res.</EM>,34, D315-D318.
        </LI></OL></SECTION></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>