<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV><HEADER class="modeling-landing"><DIV class="container"><H1 class="display">Modeling</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation" class="active"><A href="/Team:Lund/Model">Results</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL></DIV><DIV id="modeling" class="content-wrapper under-nav text-layout"><DIV class="container"><DIV class="row"><DIV class="col-lg-10"><SECTION><H3 class="section-heading">Overview</H3><P>In this section we present the results of our model. A dose-response relationship between the salicylate and estrogen inputs and the GFP
          output is provided, and the gene regulatory components are reviewed. The GFP rise time is investigated using global sensitivity analysis and
          we show that the model indicate a sensitive biosensor with long response times. In addition, the rise time was shown to be highly
          insensitive with respect to the first order interactions between the model parameters. However, as higher order interactions were shown to
          be significant, this led to the conclusion that multiple parameter has to be modified in order to improve the responsiveness. Finally, the
          section ends with a discussion on the validity on the presented model and procedures in which it could be improved. </P></SECTION><SECTION><H3 class="section-heading">Purpose</H3><P>An important part in engineering is the simulation of systems in order to yield an insight of the expected behaviours before actual
          implementations are carried out. This process can be both resource and time saving as practical implementations are often expensive and tedious to
          carry out. The same principle can be applied in synthetic biology where biological constructs are tested <I>in silica</I> prior to wet lab
          implementations. By simulating the behaviours, it is possible in an early stage of the development to evaluate whether the construct will suffice
          the intended application. This is especially important for complex systems where basic heuristics no longer can give neither sufficient
          predictions or mechanistic understandings of the underlying components. </P><P>The purpose of this model is to study the time dynamics of a <I>lac</I> regulated AND-gate based on a tripartite association of green
          fluorescent proteins induced by salicylate and estrogen. The intentions are to study the dose-response behaviours with respect to <I>two</I>
          inputs. Interesting parameters are the dose-response sensitivities and the dose-response times as these play an important part in real life
          applications of biosensors. Additionally, it is interesting to study the underlying biochemical mechanics in order to establish an understanding
          of the behaviours. Also, would a biosensor of this kind be sufficient for the detection of two concurrently present molecules?</P></SECTION><SECTION><H3 class="section-heading">Dose-response analysis</H3><H5>Detection limit</H5><P>In this section we study the biosensors dose-response characteristics. <I>Fig. 1</I> illustrates the dose-response landscape for the system.
          The surface was generated by letting the system evolve until steady state (5000 min) without any inducers, followed by the simultaneous addition
          of IPTG, salicylate and estrogen. A concentration of 1 mM IPTG was added and the salicylate and estrogen concentrations were varied  in the range 0-1000 nM and
          0-1000 nM for 40 steps respectively. The system was then further simulated until equilibrium (5000 min) and the steady state concentrations
          of the associated GFP were saved. From the figure, it can be seen that for low concentrations of salicylate and estrogen, a low response
          is obtained. As the concentrations are increased, the corresponding response is increased until reaching a plateau were the response is saturated for
          increased concentrations of pollutants, giving the landscape the characteristic mountain look. This is reasonable as the amount of GFP expressed
          is limited by the amount of GFP1-9 and GFP10-ER-GFP11. For instance, adding more salicylate will eventually make all the <I>Psal</I>
          transcriptionally active and the addition of more will thus not increase the expression of GFP1-9. In a similar manner, if all GFP10-ER-GFP11 has
          been activated and associated with GFP1-9, the addition of estrogen will not cause more GFP10-ER-GFP11 to be expressed. Consequently, if either
          one of proteins are abscent, a complementation cannot occur and the response remains unchanged. </P><FIGURE><FIGCAPTION><I>Figure 1: Dose-response landscape for the system. The data was generated by running simulations with salicylate and estrogen  concentrations in
              the range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using 1 mM IPTG. A characteristic mountain-like landscape
              can be observed where the response reaches a plateau for high concentrations of salicylate and estrogen.</I></FIGCAPTION></FIGURE><P><I>Fig. 2</I> shows the dose-response heat map for the system obtained by looking at the surface level of <I>fig. 1</I> in the
          salicylate-estrogen plane. From the figure, the sensitivity with respect to salicylate and estrogen can be studied. It can be seen that no response
          is generated for low concentrations of both inputs. In addition, in the abscence of either molecule, no output is generated regardless if the
          other exists in excess or not. This clearly illustrates the AND-behaviour and it can be seen that the detection limits are approximately 1 nM and
          100 nM for salicylate and estrogen respectively. Concentrations lower than those yield either no or a weak response relative to the maximal
          response. Clearly, this indicates a very sensitive system with respect to the inputs. Since the detection limit of extracellular salicylate is 1
          nM and since the model assumes free diffusion, this would correspond to roughly 1 nM intracellular salicylate which is almost equal to one
          molecule. This means that it would be possible to obtain a response based of one single molecule entering the cell. This seem to be an overestimate of
          sensitivity since we have not found any literature on whole cell biosensors where subnanomolar detection limits have been attained. Nevertheless,
          the results indicate that the sensor could be highly sensitive in the codetection of salicylate and estrogen. </P><P>Moreover, since the model manages to measure quantities in the order of single molecules, one might question the validity on using a continuous
          scheme. For example, there is a very weak response obtained for a salicylate concentration of 0.5 nM, corresponding to one half molecule. It might
          thus seem more reasonable to model the biosensor using discrete and stochastic methods in the single cell level by using the stochastic simulation algorithm. However,
          since we are considering a deterministic scheme interpreted as the average response over a large population, we make no assumption as to the
          homogenity of the molecular distribution among the cells. That is, even though the concentration of a molecule in one cell corresponds to one half
          molecule, it does not mean that the cell has a half molecule in the cytoplasm, but rather that the average amount of molecules over the population
          is one half salicylate. This means that we are studying the deterministic limit for one cell and the results should be interpreted as an expected
          average of a large population. We thus consider the choice to model the cell deterministic reasonable as we do no expect a biosensor to consist of
          one single cell. </P><FIGURE><FIGCAPTION><I>Figure 2: Dose-response heatmap for the system. The data was generated by running simulations with salicylate and estrogen concentrations in the
              range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using 1 mM IPTG. The yellow region in the top left corner
              illustrates the detection region where the system is able to respond on salicylate and estrogen.</I></FIGCAPTION></FIGURE><H5><I>lac</I> regulation </H5><P> With the use of <I>lac</I> inducible promoters, it is possible to have an on/off switch regulating the AND-gate. The dose-response landscape
          and heat map obtained without any IPTG inducer but otherwise same conditions as in <I>fig. 1</I> and <I>fig. 2</I> is shown in <I>fig. 3</I>.
          A slight leak response can be observed which is because the <I>lac</I> repression does not inactivate the expression entirely. Since the repressing
          dynamics is modeled as a sequence of reversible reactions, there will always be a small amount of unbound operators allowing for transcription to
          occur. Nevertheless, when comparing the expression levels with the induced case in <I>fig. 1</I> we see that the difference is at least four
          orders of magnitude, clearly indicating a working switch. </P><FIGURE><FIGCAPTION><I>Figure 3: Dose-response landscape (upper) and heatmap (lower). The data was generated by running simulations with salicylate and estrogen
              concentrations in the range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using no IPTG. It can be seen that
              although there exist a slight leakage, the response is still at least four orders magnitude lower than in the IPTG induced case. </I></FIGCAPTION></FIGURE></SECTION><SECTION><H3 class="section-heading">Time characteristics</H3><H5>Sample simulations</H5><P><I>Fig. 4</I> illustrates four simulations showing the time evolution of the associated GFP, GFP1-9 and estrogen bound GFP10-ER-GFP11. The
          simulations were run as previously described, using salicylate and estrogen concentrations in the range 1-100 nM and 10-300 nM respectively. It can be seen that
          the time evolutions are smooth as the species converges to steady state concentrations and that no oscillations are present. Thus, there are no
          observed undershoots or overshoots as a response to salicylate and estrogen. The rise time for the associated GFP, defined as the time it takes to
          go from 10% to 90% of the steady state value, was measured at approximately 650 min for all realisations. The settling time, defined as the time it
          takes to fall within 2% of the steady state concentration, was measured at roughly 1200 min for all simulations. </P><FIGURE><FIGCAPTION><I>Figure 4:
          Four simulations showing the time evolution of the associated GFP (GFP), GFP1-9 (GFP1-9) and estrogen bound GFP10-ER-GFP11 (GFP10-ER-GFP11). The
          simulations were carried out as usual and it can be seen that the time evolution for all simulations are smooth and free from oscillations.</I></FIGCAPTION></FIGURE><H5>Gene regulation</H5><P> The dynamics of the <I>lac</I> operators prior to the addition of IPTG, salicylate and estrogen is shown in <I>fig. 5</I>. The upper part of
          the figure shows the repression of the <I>lac</I> operators regulating the expression of NahR and GFP10-ER-GFP11. It can be seen that all
          operators are repressed within a couple of minutes. However, within this time frame both proteins are transcribed until completely repressed. This
          is noted in the lower part of the figure where expressed NahR binds to the <I>Psal</I> promoter, inverting the ratio between the protein bound
          promoter <I>NPsal</I> and <I>Psal</I>. However, as the operators are repressed, the expression of NahR alters and the equilibrium is displaced
          back to <I>Psal</I> as the NahR is decomposed. Worth noting is that the entire process takes almost 1000 min before reaching equilibrium. This can
          be traced to the half-life of NahR which was set to 2 hours and the slow <I>Psal</I> dynamics. </P><FIGURE><FIGCAPTION><I>Figure 5: Illustration of the gene regulating dynamics prior to the addition of inducers. The simulation was generated as usual. In the upper
          part the <I>lac</I> dynamics are shown and it can be seen that all operators are repressed at equilibrium. In the lower part the <I>Psal</I>
          dynamics are shown. It can be seen that the equilibrium is established first after 1000 min once the transients has died off. </I></FIGCAPTION></FIGURE><P>The dynamics of the <I>lac</I> operators and the <I>Psal</I> promoter after the addition of IPTG, salicylate and estrogen is shown in <I>fig.
          6</I>. Here, 100 nM and 300 nM of salicylate and estrogen was used and the IPTG concentration was set at 1 mM. In the upper part of the figure, it
          can be seen that all operators are repressed at time zero, completely inhibiting the transcription of NahR and GP10-ER-GFP11. After the addition
          of IPTG, the operators are derepressed which consquently increases the transcription rates. It can be seen that not all repressors are inhibited
          at steady state, indicating that higher expression levels can be achieved by the addition of more IPTG. However, we chose to limit the
          extracellular concentration to 1 mM to keep the simulation within realistic boundaries set by wet-lab working conditions [1]. In addition, it is
          also worth mentioning that we did have some issues with numerical instability for higher concentrations of IPTG. This can be traced to the shock
          effect which occurs when a high concentration of IPTG is added momentaneously at time zero. However, it was observed that this problem could be
          circumvented by gradually increasing the concentration over a short time span instead of adding everything simultaneously.</P><P> The dynamics of the <I>Psal</I> promoter is illustrated in the lower part of <I>fig. 6</I>. At time zero, only <I>Psal</I> is present due to the
          abscence of NahR and salicylate. As time progresses, NahR is expressed, forming the intermediate NahR bound promoter <I>NPsal</I>. This is
          followed by salicylate simultaneously diffusing over the cell membrane and binding to <I>NPsal</I>. What is observed is then the <I>Psal</I> promoter
          slowly decreasing in concentration while <I>NPsal</I> is formed, followed by a lagged increase of the active promoter <I>SNPsal</I>. It can be
          seen that all <I>Psal</I> is converted into the active form <I>SNPsal</I> within the range of approximately 200 min. This indicates that the
          expression of GFP1-9 does not reach the maximal rate until 200 min after induction, which corresponds to almost a third of the total rise time.</P><FIGURE><FIGCAPTION><I>Figure 6: Illustration of the gene regulating dynamics after the addition of inducers. The simulation was generated as usual  with an IPTG
          concentration of 1 mM and salicylate and estrogen concentration of 100 nM and 300 nM respectively. The upper part shows the <I>lac</I> dynamics
          and it can be seen that not all operons are free after the addition of IPTG. In the lower part the <I>Psal</I> dynamics are shown. It can be seen
          that an equilibrium is established after approximately 200 min where equilibrium is shifted towards <I>SNPsal</I>. </I></FIGCAPTION></FIGURE><H5>Response time sensitivity analysis</H5><P> From the simulations carried out in the preceeding section, it was noted that rise times in the order of 650 min were observed for a variety
          of doses and it was seen that the time varied little with respect to the applied doses. Due to the long rise time of almost 11 hours, it would be
          of great interest to reduce the time to allow for a rapid detection for practical uses. Hence, to investigate the role of the parameters we
          chose to do global sensitivity analysis with respect to the rise time. Global sensitivity analysis is a method used to study the effect that
          large variations in the input parameters have on the output. This is slightly different from local sensitivity analysis which studies the effect
          on the output caused by small pertubtations on the input. One can view the local variant as a gradient of the output with respect to the parameter
          space at the point in the parameter space considered, whereas the global variant gives an overall effect on the output from each parameter with
          respect to a specified domain in the parameter space [2].</P><P> The parameters for the sensitivity analysis were chosen with respect to their expected effect on the rise time as well as their capability of
          being modified in the lab and are illustrated in <I>table 1</I>. We chose to investigate the transcription and translation rates for NahR, GFP1-9
          and GFP10-ER-GFP11, the <I>Psal</I> dynamics, the ligand binding dynamics of estrogen to GFP10-ER-GFP11 and the tripartite GFP association rate.
          The parameter space was chosen by logarithmic sampling around the estimated values within two orders of magnitude, that is the interval with
          bounds specified by one hundred times and one hundredth times the parameters. These intervals were chosen somewhat arbitrarly based on the belief
          that it is not possible (or at least very hard) to alter the parameters more than one hundred times their original values. In total, 42 000
          samples from the 13-dimensional parameter space were generated using Saltelli's sampling scheme (see implementation). The sensitivities can be
          found in <I>table 1</I> and <I>fig. 7</I>. We chose to only consider the first and total order sensitivity indices and present them along with 95% confidence
          intervals. Here, S<SUB>1</SUB> denotes the first order sensitivity index which is a measure of a single parameters contribution to the total
          variance in the output. The total sensitivity index, S<SUB>T</SUB> is similarly a measure of a parameters contribution to the output variance, but
          including both the first order effect and all higher interactions with other parameters. We interpret it as the contribution to the output variance
          where interparameter dependencies are included. We think it's worth mentioning that we initially had issues when sampling over the normal
          parameter space as it resulted in very small sensitivities for all parameters. This was solved by sampling over the corresponding logarithmic
          parameter space.</P><P> It can be seen from <I>table 1</I> and <I>fig. 7</I> that almost all first order sensitivity indices are weak. This means that most parameters
          in the pathway has a weak individual effect on the rise time. In addition, the confidence intervals indicate that almost all first order
          sensitivities are insignificant which implies that we cannot rule out the possibility that the parameters do not affect the output individually at
          all. In fact, the only strong first order sensitive parameter is $\text{k}_\text{aSNP}$, which also is the only significant one. Thus, this is
          the only parameter that strongly affects the rise time individually. However, when looking at the total sensitivity indices, it can be seen that
          their values are higher than almost all first order indices. In addition, most of them are significant in contrast to the first orders. This
          indicates a strong coupling in the system where parameters must co-vary in order to significantly affect the rise time of the biosensor. Clearly,
          this is just a rediscovery of the principle of the rate determining step commonly seen in chemical kinetics and we have simply found the parameter
          governing the rise time, namely $\text{k}_\text{aSNP}$. It is worth noting that in retrospect, this result might seem highly trivial and that this
          can simply be seen by observing $\text{k}_\text{aSNP}$ being the smallest rate constant in the network. However, when we carried out this analysis
          we did not intuitively realize this until after analyzing the sensitivity results. In fact, our hypothesis was that there would exist some key
          parameters strongly affecting the rise time and we were thus looking to find these in order to optimize for rapid detection. Nevertheless, the
          analysis at least gave a quantitative justification of the rate determining step principle and we are very happy to have performed such an
          analysis. </P><TABLE class="table"><CAPTION>Table 1: Sensitivity indices for 13 parameters with respect to the rise time generated from the global sensitivity analysis. The sensitivities were
          obtained by using the Sobol method by sampling 42000 points in the parameter space using Saltelli's sampling scheme. The parameter space used is
          defined by the values within two orders of magnitude from each estimated parameter. S<SUB>1</SUB> = first order sensitivity index, S<SUB>T</SUB> =
          total order sensitivity index, 95% CI = 95% confidence interval. </CAPTION><THEAD><TR><TH>Parameter</TH><TH>Value</TH><TH>S<SUB>1</SUB></TH><TH>95% CI</TH><TH>S<SUB>T</SUB></TH><TH>95% CI</TH></TR></THEAD><TBODY><TR><TD>$\ce{k_\text{aGFP}}$</TD><TD>$1.8 \cdot 10^{-5}$</TD><TD>$0.0035$</TD><TD>$0.011$</TD><TD>$0.036$</TD><TD>$0.013$</TD></TR><TR><TD>$\ce{k_\text{aEEs}}$</TD><TD>$0.0078$</TD><TD>$0.0013$</TD><TD>$0.0022$</TD><TD>$0.00075$</TD><TD>$0.00037$</TD></TR><TR><TD>$\ce{k_\text{dEEs}}$</TD><TD>$0.072$</TD><TD>$0.0011$</TD><TD>$0.0019$</TD><TD>$0.00044$</TD><TD>$0.00019$</TD></TR><TR><TD>$\ce{k_\text{aNP}}$</TD><TD>$4.7 \cdot 10^{-3}$</TD><TD>$0.019$</TD><TD>$0.031$</TD><TD>$0.18$</TD><TD>$0.034$</TD></TR><TR><TD>$\ce{k_\text{dNP}}$</TD><TD>$11.5$</TD><TD>$0.029$</TD><TD>$0.039$</TD><TD>$0.22$</TD><TD>$0.044$</TD></TR><TR><TD>$\ce{k_\text{aSNP}}$</TD><TD>$4.61 \cdot 10^{-4}$</TD><TD>$0.24$</TD><TD>$0.070$</TD><TD>$0.55$</TD><TD>$0.067$</TD></TR><TR><TD>$\ce{k_\text{dSNP}}$</TD><TD>$3.81 \cdot 10^{-6}$</TD><TD>$0.0010$</TD><TD>$0.0027$</TD><TD>$0.00082$</TD><TD>$0.00039$</TD></TR><TR><TD>$\ce{k_\text{MN}}$</TD><TD>$3.09$</TD><TD>$0.0077$</TD><TD>$0.037$</TD><TD>$0.18$</TD><TD>$0.033$</TD></TR><TR><TD>$\ce{k_\text{ME}}$</TD><TD>$3.18$</TD><TD>$0.00094$</TD><TD>$0.019$</TD><TD>$0.083$</TD><TD>$0.026$</TD></TR><TR><TD>$\ce{k_\text{MG}}$</TD><TD>$0.168$</TD><TD>$0.031$</TD><TD>$0.036$</TD><TD>$0.28$</TD><TD>$0.055$</TD></TR><TR><TD>$\ce{k_\text{N}}$</TD><TD>$3.0$</TD><TD>$0.027$</TD><TD>$0.038$</TD><TD>$0.21$</TD><TD>$0.040$</TD></TR><TR><TD>$\ce{k_\text{E}}$</TD><TD>$2.86$</TD><TD>$-0.0018$</TD><TD>$0.019$</TD><TD>$0.074$</TD><TD>$0.022$</TD></TR><TR><TD>$\ce{k_\text{G}}$</TD><TD>$4.59$</TD><TD>$0.035$</TD><TD>$0.037$</TD><TD>$0.23$</TD><TD>$0.043$</TD></TR></TBODY></TABLE><FIGURE><FIGCAPTION><I>Figure 7: Graphical illustration of the sensitivities from table 1. The purple bars represent the
              sensitivity indices and the blue bars the corresponding 95% confidence intervals. </I></FIGCAPTION></FIGURE><H5>Parameter sweeps</H5><P>To illustrate the effect of the sensitivite and insensitive parameters within the scanned ranges, we performed four parameter sweeps.
          All simulations were carried out under normal conditions as explained previously, using 1 mM IPTG and 100 nM and 300 nM salicylate and estrogen
          respectively. A total of 30 realisations were computed for each parameter in a range of two orders around the estimated value. <I>Fig. 8</I>
          illustrates the results. It can be seen that the only parameter affecting the rise time strongly is $\text{k}_\text{aSNP}$ which is in line with
          the sensitivity analysis. However, what is interesting is that tuning the parameter does not improve the rise time. In fact, the high sensitivity affects
          the output by drastically increasing the rise time when lowering the parameter value, whereas increasing it does not impose any effect. We have thus found that $\text{k}_\text{aSNP}$ is not only
          the rate determining parameter, but also the  time limiting parameter. It is here important to make the statement that the sensitivities
          only give the contribution to the total variance in the output, but does not give any information on the direction of the change. Similarly,
          studying the rest of the parameters reveal that they have a relatively small effect on the rise time, with the association rate
          $\text{k}_\text{aGP}$ being the only one improving the rise time in a noteworthy way. </P><FIGURE><FIGCAPTION><I>Figure 8: Rise times for the associated GFP with respect to parameter sweeps for various rate constants. The simulations were carried out under
          normal conditions, using 1 mM IPTG and 100 nM and 300 nM salicylate and estrogen respectively. It can be seen that the rise time is insensitive to
          changes in parameters for all parameters except $\text{k}_\text{aSNP}$, which is the time limiting parameter. </I></FIGCAPTION></FIGURE><H5> Practical interpretation of the sensitivities and parameter sweeps </H5><P> Generally, sensitivity indices can be interpreted in two ways depending on the design phase. If the purpose of the model is to recreate a real
          system and describe it as carefully as possible, then the indices gives the parameters that are the most important to estimate accurately. This is
          because an error on these parameters will give a larger contribution to the output than less sensitive parameters. On the other hand, if the
          system has been constructed and the parameters are estimated to a satisfactory precision, the sensitivities gives the most important parameters to
          modify in order to achieve a certain behaviour.</P><P> In our case, the observations on the sensitivities yielded important information for avoiding
          unnecessary work in the wet lab. Since almost all parameters are first order insensitive, individual parameter optimizations will not yield any
          significant effect on the performance in terms of waiting times. In addition, the only significant first order sensitive paramater turned out to be
          the rate limiting step. It was seen that an increase in the parameter value would not give any improved rise time, whereas a decrease would
          rapidly increase it. The only possible way left to efficiently lower the rise time would then be to consider higher order interactions. But this
          means modifying many parameters simulatenously and such procedures would be very tiresome and thus not feasible in reality. The conclusion based
          on the model is thus that the biosensor, albeit very sensitive with regards to the inputs, is very slow at generating an output. The model thus
          indicates that if a sensitive biosensor is desired, this kind of system might work whereas if a quick and responsive biosensor is needed, a
          different biochemical network topology should probably be considered. </P></SECTION><SECTION><H3 class="section-heading">Conclusion</H3><P> To conclude, the model indicates:
            <UL><LI>High dose sensitivity </LI><LI>Long response times </LI><LI>Well defined on/off states</LI></UL>
            This was done by fitting the model to
            <UL><LI>iGEM registry data</LI><LI>Experimental results from published material</LI><LI>Average kinetic parameters</LI></UL>
            We provided
            <UL><LI> A full derivation of the network topology</LI><LI> A determenistic representation of ordinary differential equations </LI><LI> Implementation details </LI></UL>
            The contribution to our project was
            <UL><LI> An insight into the biochemical mechanics </LI><LI> Order of magnitudes of response times and dose sensitivities </LI><LI> An indication of the unfeasability of improving response times </LI></UL></P></SECTION><SECTION><H3 class="section-heading"> A note on the validity of the model</H3><P>With mathematical models, abstractions of real world systems can be constructed in order to infer information about reflected behaviours. A
          lot can go wrong under such processes and it is practically impossible to include all details in a model. This makes the process of
          questioning, experimentalily validating and proposing  alternative models ever more important before final conclusions about the real world
          behaviour can be infered. In this section we want to draw attention on some ambiguitities in the presented model.</P><P>The first ambiguity is the estimation of the reaction rates in the binding of salicylate to <I>NPsal</I>. This was estimated by using the
          induction ratios from a dose-response diagram of salicylate induced GFP from the registry. Here, we proposed an <I>ad hoc</I> base level response of 100 nM
          GFP and assumed linearity in the induction ratios. Clearly, there is no reason to believe that the concentration of GFP is exactly 100 nM
          at base level, more than it seemed reasonable. However, despite this, the model managed to catch the temporal dose-response behaviour, so although
          the absolute concentrations might be off, the time to equilibrium should at least be within the right order.</P><P>The estimation of the association rates between GFP1-9 and GFP10-ER-GFP11, and NahR and <I>Psal</I> were based on experimental <I>in vivo</I>
          measurments from published litterature. There is obviously no reason to believe that the behaviour <I>in vitro</I> gives a complete description of the <I>in vivo</I> behaviour
          as intracellular conditions are usually much more complex. Nevertheless, they should still be indicative and hint towards a true behavour. Other
          parameters based on average reaction rates are only valid under the assumption that the cell is working in an &quot;average mode&quot;. This is not always
          true however. For example, overexpressing a protein will alter the expression rates of other proteins in the cell [3]. In our case, we do not know the
          effect that multiple genetic modules have on the total cell behaviour. Thus, the system should ideally be implemented in the wet lab and all
          parameters reestimated based on experimental observations. </P><P>Finally, it is not obvious that the proposed network topology is the most informative one. Reactions of higher order with more side reactions
          or intermediate steps could give a better description of the behaviour. Clearly, a comparision using models with other networks topologies should
          be made to find the best description, balancing intricacy and accuracy.</P><P>Nevertheless, despite all of this, we believe the model at least gave us
          indications of the real world behaviour in our biosensor.</P></SECTION><SECTION><H3 class="section-heading">References</H3><BUTTON id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
            Show
          </BUTTON><DIV class="collapse" id="ref"><OL class="references"><LI>[1] Larentis, A., Nicolau, J., Esteves, G., Vareschini, D., de Almeida, F., dos Reis, M., Galler, R. and Medeiros, M. (2014). Evaluation of pre-induction temperature, cell growth at induction and IPTG concentration on the expression of a leptospiral protein in E. coli using shaking flasks and microbioreactor. <I>BMC Research Notes</I>, 7(1), p.671.</LI><LI>[2] Zi, Z. (2011). Sensitivity analysis approaches applied to systems biology models. <I>IET Systems Biology</I>, 5(6), pp.336-346.</LI><LI>[3] Kafri, M., Metzl-Raz, E., Jona, G. and Barkai, N. (2016). The Cost of Protein Production. <I>Cell Reports</I>, 14(1), pp.22-31.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>