<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Design_Theory skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Design/Theory</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV><HEADER class="design-header"><DIV class="container"><H1 class="display">Design</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Design">Overview</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Animation">Animation</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Design/Theory">Theory</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Design">Project Design</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Circuit">Circuit</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Experiments">Experiments</A></LI><LI role="presentation"><A href="/Team:Lund/Design/References">References</A></LI></UL></DIV><DIV class="content-wrapper under-nav text-layout"><DIV class="container"><SECTION><DIV class="row"><DIV class="col-md-8"><H3 class="section-heading">Sensing organic pollutants</H3></DIV></DIV><DIV class="row"><DIV class="col-md-8"><P>Persistent organic pollutants are harmful organic compounds of anthropogenic origins that have been, and in some cases still are, utilized as industry chemicals and pesticides. Due to their ecotoxicity and their propensity of bioaccumulation, the United Nations Environment Programme (UNEP) adopted and implemented proposed regulation of these chemicals as put forward by the Stockholm Convention on Persistent Organic Pollutants in 2001 [10]. As of 2017, 181 countries have ratified the convention [11]. While there is no explicit organic backbone, most organic pollutants share certain characteristics. They have one or more cyclic ring structures, most often of the aromatic nature, they are highly lipophilic and they are, to a varied extent, halogenated [12]. Consequently, there are different moieties that can be exploited when searching for homologues that act as natural ligands to gene regulatory elements. The 2012 UCL iGEM team successfully determined and utilized the homology between the NahR inducing ligand salicylate and the aromatic backbone found in most organic pollutants [9]. </P><P>The <EM>NahR</EM> gene encodes a Lys-R type transcription regulator found in the naphthalene degrading operon of the NAH7 plasmid of various <EM>Pseudomas</EM> [13]. As with all Lys-R type regulators, NahR has a conserved N-terminal DNA-binding helix-turn-helix motif and a C-terminal co-inducer-binding domain [14]. The catabolic genes are organized in two operons, <EM>fig. 1</EM>, <EM>nah</EM> (<EM>nah A-F) </EM>and <EM>sal</EM> (<EM>sal G-M</EM>) that encode enzymes for metabolism of naphthalene to salicylate, and salicylate to the TCA-intermediates pyruvate and acetaldehyde respectively [13]. NahR regulates the expression of both <EM>sal </EM>and <EM>nah</EM> through binding 60 bp upstream of each respective promoter, <EM>Psal </EM>and<EM> Pnah</EM>, and is constitutively expressed upstream <EM>nah-G</EM> [15]<EM>.</EM> It binds regardless of the presence of the inducer salicylate. However, the transcriptional induction rate is approximately 20 times higher upon association with salicylate, as this induces a conformational change in the DNA-NahR complex that relieves DNA-bending at the site and subsequently allows better DNA-RNAP interaction [16]. The conformational change springs from an additional interaction between the NahR and the DNA 35 bp upstream of the RNAP binding site [15].</P><P>There lies some uncertainty in the exact configuration of the DNA-NahR association. Some reports suggest that NahR binds as a monomer, while other research indicates multimerization at the binding site. Moreover, it remains unknown whether the binding is cooperative or not [15]. For further elaboration on the issue, see <A href="/Team:Lund/Model">modeling</A>.</P></DIV><DIV class="col-md-4"><FIGURE><FIGCAPTION>Figure 1: Schematic representation of the organization of the upper and lower degradation pathway of naphthalene found in the Nah7 plasmid of <EM>Pseudomas</EM>. Both pathways are under regulation of the constitutively expressed regulator NahR [17].</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 2: 3D structure of the NahR, a transcriptional regulator spanning 306 amino acids. The protein structure is organized as follows; the N-terminal consists of a highly conserved DNA-binding helix-turn-helix motif. The DNA association is furthermore dependent on a multimerization site located along the C-terminal. Through iterative mutagenesis, the ligand-binding pocket has been determined to lie close to equidistantly between the termini [18] [47] [48] .</FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row"><DIV class="col-md-8"><H3 class="section-heading">Sensing plasticizers</H3></DIV></DIV><DIV class="row"><DIV class="col-md-8"><P>Plasticizers are very common additives found in most commercial plastic products. They provide the polymers with durability, elasticity and flexibility, offering increased malleability and strength [19]. The plasticizer action on the polymeric material has generally thought to be lubrication between the polymeric chains, but recent thermodynamic simulations have shown that the situation might be more complex [20]. The most predominant class of plasticizers used today is that of low-weight phthalate esters (PEs), in particular DEHP (bis-2-ethylhexyl phthalate) [19]. While PEs offer a cost-effective solution to maintaining flexibility, there is cause for concern as they are not covalently bonded to the plastic and will readily leach into the environment. Multiple sources have reported severe adverse effects in vertebrates; among other things, toxic action on the human endocrine system has been noted [21] [22] [23] [24] [25]. In particular, low-weight PEs have demonstrated some antagonistic affinity for the ligand-binding domain of the human estrogen-alpha receptor (hER-α) [23] [24], making it a viable candidate for detecting plasticizers.</P><P>The estrogen receptor is a member of the intracellular receptor family and exists in two different forms, hER-α and hER-β, with significant overlapping sequence homology. Each receptor shares a common architecture consisting of five domains; the N-terminal <EM>A/B</EM> region able to regulate gene transcription in absence of a ligand, the <EM>C</EM>-domain able to bind to a designated DNA sequence (DBD) and a hinge region <EM>D</EM> (DBD) that connects the C-domain and the ligand binding domain (LBD) <EM>E</EM> [26]. The mechanism of gene transactivation by the estrogen receptor has been studied in detail through analysis of the ligand-induced conformational change of the LBD. The LBD has a similar organization to that of the other nuclear receptor LBDs, with a three-layered antiparallel α-helical sandwich motif, for a total of 11 α-helices (H1-11) with one additional anti-parallel β-sheet flanking the arrangement (S1-2). The central core, located between the two outer helical layers, constitutes a wedge-shaped molecular scaffold to bind the ligand [27]. Upon interaction between the binding site and a ligand, a sizeable conformational change occurs at the C-terminal helix H12 to modify the action at the DNA-interface. An agonistic ligand spatially displaces H12 to fit snugly over the cavity whereas an antagonistic ligand prevents such confirmation and instead forces the helix to reposition through a 130° degree rotation toward the N-terminus [27][28]. Such displacement of H12 has been utilized in complementation assays as a <EM>molecular switch</EM>. Complementation of a split protein fused to each respective side of a truncated hER-α LBD is made possible upon antagonist association with the binding site as the termini are subsequently brought in close proximity to one another [29] [30] [31].</P></DIV><DIV class="col-md-8 col-md-4"><FIGURE><FIGCAPTION>Figure 3: 3D structure of the human estrogen receptor alpha ligand-binding domain (in complex with estradiol), a domain found in the transcriptional transactivator hER-α spanning 300 amino acids. The ligand-binding domain consists of a deep and promiscuous ligand-binding pocket consisting of a three-layered antiparallel α-helical sandwich motif. Upon recognition of a ligand, helix H12 will change conformation in accordance with the nature of the ligand; antagonists will displace the helix and bring the two termini close to one another whereas agonists will induce helix H12 to transverse the protein and stabilize over the pocket [27] [49] [50] [51] [52] .</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4: The hER-a ligand binding has been implemented and utilized successfully in complementation assays through capitalizing on the conformational change upon ligand recognition. [32]</FIGCAPTION></FIGURE></DIV></DIV></SECTION></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>