<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Modeling_ParameterEstimation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Modeling/ParameterEstimation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_safety"><A href="/Team:Lund/Safety">Safety</A></LI><LI id="li_hp"><A href="/Team:Lund/HP/Silver">Practices</A></LI></DIV></DIV></DIV><HEADER class="modeling-landing"><DIV class="container"><H1 class="display">Modeling</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Model">Results</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI role="presentation"><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL></DIV><DIV id="modeling" class="content-wrapper under-nav text-layout"><DIV class="container"><DIV class="row"><DIV class="col-lg-10"><SECTION><H3 class="section-heading">Overview</H3><P>In this section we provide the parameters used in the model. The values were estimated using tabulated kinetic parameters, iGEM
          registry data and experimental results from published literature. An overview can be found in <I>table 1</I>.</P></SECTION></DIV></DIV><DIV class="row"><DIV class="col-lg-10"><SECTION><H3 class="section-heading">Parameters</H3><TABLE class="table"><CAPTION>Table 1: List of all parameters used in the model. For a description, see the corresponding description section further down.</CAPTION><THEAD><TR><TH>Parameter</TH><TH>Value</TH><TH>Unit</TH><TH>Description</TH><TH>Reference</TH></TR></THEAD><TBODY><TR><TD>$\ce{C_\text{p}}$</TD><TD>$40$</TD><TD>-</TD><TD>Copy number</TD><TD>[1]</TD></TR><TR><TD>$\ce{V}$</TD><TD>$8 \cdot 10^{-16}$</TD><TD>L</TD><TD>E. coli cell volume</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{MR}}$</TD><TD>$0.11$</TD><TD>$\text{min}^{-1}$</TD><TD>MR transcription rate constant</TD><TD>[2]<SUP>a)</SUP></TD></TR><TR><TD>$\ce{k_\text{R}}$</TD><TD>$15$</TD><TD>$\ce{min}^{-1}$</TD><TD>R translation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{2R}}$</TD><TD>$50$</TD><TD>$\ce{\text{nM}^{-1} min^{-1}}$</TD><TD>R dimerization rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{-2R}}$</TD><TD>$10^{-3}$</TD><TD>$\ce{min}^{-1}$</TD><TD>R$_2$ dissociation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{r}}$</TD><TD>$960$</TD><TD>$\text{nM}^{-1} \text{min}^{-1}$</TD><TD>Operator repression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{-r}}$</TD><TD>$2.4$</TD><TD>$\text{min}^{-1}$</TD><TD>Operator derepression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{dr1}}$</TD><TD>$3\cdot 10^{-7}$</TD><TD>$\text{nM}^{-2}\text{min}^{-1}$</TD><TD>Free R$_2$ repression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{-dr1}}$</TD><TD>$12$</TD><TD>$\text{min}^{-1}$</TD><TD>Free R$_2$ derepression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{dr2}}$</TD><TD>$3\cdot 10^{-7}$</TD><TD>$\text{nM}^{-2} \text{min}^{-1}$</TD><TD>Operator bound R$_2$ repression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{-dr2}}$</TD><TD>$4.8 \cdot 10^{3}$</TD><TD>$\text{nM}^{-1} \text{min}^{-1}$</TD><TD>Operator bound R$_2$ derepression rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{MN}}$</TD><TD>$3.09$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_R}$ transcription rate constant</TD><TD>[3]<SUP>b)</SUP></TD></TR><TR><TD>$\ce{k_\text{ME}}$</TD><TD>$3.18$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_E}$ transcription rate constant</TD><TD>[3]<SUP>b)</SUP></TD></TR><TR><TD>$\ce{k_\text{MG}}$</TD><TD>$0.168$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_G}$ transcription rate constant</TD><TD>Estimated<SUP>g)</SUP></TD></TR><TR><TD>$\ce{k_\text{N}}$</TD><TD>$3.0$</TD><TD>$\text{min}^{-1}$</TD><TD>NahR translation rate constant</TD><TD>[3]<SUP>c)</SUP></TD></TR><TR><TD>$\ce{k_\text{E}}$</TD><TD>$2.86$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP10-ER-GFP11 translation rate constant</TD><TD>[3]<SUP>c)</SUP></TD></TR><TR><TD>$\ce{k_\text{G}}$</TD><TD>$4.59$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP1-9 translation rate constant</TD><TD>[3]<SUP>c)</SUP></TD></TR><TR><TD>$\ce{k_\text{aEEs}}$</TD><TD>$0.078$</TD><TD>$\text{nM}^{-1}\text{min}^{-1}$</TD><TD>GFP10-ER-GFP11, estrogen association rate constant</TD><TD>[4]<SUP>e)</SUP></TD></TR><TR><TD>$\ce{k_\text{dEEs}}$</TD><TD>$0.072$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP10-ER-GFP11, estrogen dissociation rate constant</TD><TD>[4]<SUP>e)</SUP></TD></TR><TR><TD>$\ce{k_\text{aNP}}$</TD><TD>$4.7\cdot 10^{-3}$</TD><TD>$\text{nM}^{-1}\text{min}^{-1}$</TD><TD>NahR-Psal association rate constant</TD><TD>[5]<SUP>f)</SUP></TD></TR><TR><TD>$\ce{k_\text{dNP}}$</TD><TD>$11.46$</TD><TD>$\text{min}^{-1}$</TD><TD>NahR-Psal dissociation rate constant</TD><TD>[5]<SUP>f)</SUP></TD></TR><TR><TD>$\ce{k_\text{aSNP}}$</TD><TD>$4.61 \cdot 10^{-4}$</TD><TD>$\text{nM}^{-1}\text{min}^{-1}$</TD><TD>Salicylate-[NahR-Psal] association rate constant</TD><TD>Estimated<SUP>g)</SUP></TD></TR><TR><TD>$\ce{k_\text{dSNP}}$</TD><TD>$3.81 \cdot 10^{-6}$</TD><TD>$\text{min}^{-1}$</TD><TD>Salicylate-[NahR-Psal] dissociation rate constant</TD><TD>Estimated<SUP>g)</SUP></TD></TR><TR><TD>$\ce{k_\text{aGFP}}$</TD><TD>$1.8 \cdot 10^{-5}$</TD><TD>$\ce{\text{n}M^{-1}min^{-1}}$</TD><TD>Tripartite GFP association rate constant</TD><TD>[6]<SUP>h)</SUP></TD></TR><TR><TD>$\ce{k_\text{dGFP}}$</TD><TD>$0.0$</TD><TD>$\ce{min^{-1}}$</TD><TD>Tripartite GFP dissociation rate constant</TD><TD>[6]<SUP>h)</SUP></TD></TR><TR><TD>$\ce{k_\text{dI}}$</TD><TD>$0.92$</TD><TD>$\text{min}^{-1}$</TD><TD>IPTG diffusion constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{k_\text{dS}}$</TD><TD>$0.281$</TD><TD>$\text{min}^{-1}$</TD><TD>Salicylate diffusion rate constant</TD><TD>[7]<SUP>i)</SUP></TD></TR><TR><TD>$\ce{k_\text{dE}}$</TD><TD>$1.44$</TD><TD>$\text{min}^{-1}$</TD><TD>Estrogen diffusion rate constant</TD><TD>[8]<SUP>j)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{MR}}$</TD><TD>$0.462$</TD><TD>$\ce{min^{-1}}$</TD><TD>MR degradation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{\lambda_\text{MN}}$</TD><TD>$0.2$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_N}$ degradation rate constant</TD><TD>[3]<SUP>k)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{ME}}$</TD><TD>$0.2$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_E}$ degradation rate constant</TD><TD>[3]<SUP>k)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{MG}}$</TD><TD>$0.2$</TD><TD>$\text{min}^{-1}$</TD><TD>$\ce{M_G}$ degradation rate constant</TD><TD>[3]<SUP>k)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{R}}$</TD><TD>$0.2$</TD><TD>$\ce{min}^{-1}$</TD><TD>R degradation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{\lambda_{\text{R}_2}}$</TD><TD>$0.2$</TD><TD>$\ce{min}^{-1}$</TD><TD>R$_2$ degradation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{\lambda_{\text{I}_2\text{R}_2}}$</TD><TD>$0.2$</TD><TD>$\ce{min}^{-1}$</TD><TD>I$_2$R$_2$ degradation rate constant</TD><TD>[2]</TD></TR><TR><TD>$\ce{\lambda_\text{N}}$</TD><TD>$0.00578$</TD><TD>$\text{min}^{-1}$</TD><TD>NahR degradation rate constant</TD><TD>Assumed<SUP>l)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{E}}$</TD><TD>$0.00578$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP10-ER-GFP11 degradation rate constant</TD><TD>[3, 9]<SUP>m)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{EEs}}$</TD><TD>$0.00578$</TD><TD>$\text{min}^{-1}$</TD><TD>Ligand bound GFP10-ER-GFP11 degradation rate constant</TD><TD>[3, 9]<SUP>m)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{G}}$</TD><TD>$0.0347$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP1-9 degradation rate constant</TD><TD>[Bionumbers]<SUP>n)</SUP></TD></TR><TR><TD>$\ce{\lambda_\text{GFP}}$</TD><TD>$0.0347$</TD><TD>$\text{min}^{-1}$</TD><TD>GFP degradation rate constant</TD><TD>[Bionumbers]<SUP>n)</SUP></TD></TR></TBODY></TABLE></SECTION></DIV></DIV><DIV class="row"><DIV class="col-lg-10"><SECTION><H3 class="section-heading">Description</H3><P>a)  The rate constant is 0.23 nM/min for an operator content of 2.08 nM [2]. This yields the rate constant $\ce{k_\text{MR} = 0.11}$ min<SUP>-1</SUP></P><P>b)  The average transcription rate is 50 nucleotides per second in <I>E. coli</I> [3]. The NahR transcript has approximately 900 nucleotides which
            yields $\ce{k_\text{MN}=3.33}$ min<SUP>-1</SUP>. The GFP10-ER-GFP11 transcript has approximately 1800 nucleotides which
            yields $\ce{k_\text{ME}=1.67}$ min<SUP>-1</SUP>. The GFP1-9 transcript has approximately 600 nucleotides which yields $\ce{k_\text{MG} = 5}$
            min<SUP>-1</SUP>. </P><P>c)  The average translation rate is 15 amino acids per second in <I>E. coli</I> [3]. NahR has approximately 300 amino acids which
            yields $\ce{k_\text{MN}=3}$ min<SUP>-1</SUP>. GFP10-ER-GFP11 has approximately 600 amino acids which
            yields $\ce{k_\text{ME}=1.5}$ min<SUP>-1</SUP>. The GFP1-9 transcript has approximately 200 amino acids which yields $\ce{k_\text{MG}=4.5}$
            min<SUP>-1</SUP>. </P><P>e) The rate constants were taken for 17β-estradiol [4].</P><P>f) The NahR-Psal association and dissociation rates were taken as $k_\text{aNP} = 4.7 \cdot 10^{-3}$ nM<SUP>-1</SUP>min<SUP>-1</SUP> and
            $k_\text{dNP}=11.46$ min<SUP>-1</SUP> after unit conversions [5].</P><P>g) A dose-response scheme for the salicylate inducible promoter was taken from the <A href="http://parts.igem.org/Part:BBa_J61051"><I>iGEM parts
          registry documentation page</I></A>, see figure 13. In the experimental setup, eight doses of salicylate were added every 30 minutes, activating
          the transcription of repressed <I>GFP</I>. The response was measured as an induction ratio of the fluorescence with respect to the fluorescence at the
          uninduced state. <I>Fig. 1</I> illustrates the dose-response curve.</P><FIGURE><FIGCAPTION><I>Figure 1:
            Dose response scheme for the NahR system taken from the <A href="http://parts.igem.org/Part:BBa_J61051"><I>iGEM parts registry documentation
            page</I></A>. The data points are represented as red dots linearly interpolated with black lines.</I></FIGCAPTION></FIGURE><P>Implementing the entire system with all parameters except for $\text{k}_\text{MG}$, $\text{k}_\text{aSNP}$ and $\text{k}_\text{dSNP}$ (see <A href="/Team:Lund/Modeling/ModelDerivation"><I>Model derivation</I></A> for the equations and <A href="/Team:Lund/Modeling/Implementation"><I>Implementation</I></A> for implementation details), we fit the model to
            the dose-response scheme to obtain the parameters. In the setup we assume an induction base level of 100 nM which corresponds to approximately 100
            proteins per cell at the uninduced state and assume a linearity in the concentration with respect to the induction ratio. We further assume that a
            maximal induction ratio is obtained at a dose of 0.001 M salicylate.</P><P>Doses of salicylate were added every 30 min and the parameters were fit as follows: we defined a penalty function
            $f(\boldsymbol{\theta})=f(k_\text{aSNP}, k_\text{dSNP}, k_\text{MG})$ corresponding to the root mean square difference between the models
            output of folded GFP1-9 and the corresponding response data. This function is then a measure of how good the model fits the measured data with
            respect to the three paramters $k_\text{aSNP}$, $k_\text{dSNP}$, and $k_\text{MG}$ given that all other parameters are set.  The objective is
            then to minimize the penalty $f(\theta)$ under the stated conditions. This was done in MATLAB 2017a using <I>LSQNONLIN</I> which is a nonlinear least
            squares solver which uses the Levenberg-Marquardt algorithm. <I>Fig. 2</I> shows the resulting fit and one can see that the model manages to catch the
            structure from the dose-response data. The resulting parameters were estimated as $\text{k}_\text{aSNP} = 4.61 \cdot 10^{-4}$
            nm<SUP>-1</SUP>min<SUP>-1</SUP>, $\text{k}_\text{dSNP} = 3.81 \cdot 10^{-6}$ min<SUP>-1</SUP> and $\text{k}_\text{MG} = 0.168$ min<SUP>-1</SUP>.</P><FIGURE><FIGCAPTION><I>Figure 2: Fitting the model to the dose response scheme for the NahR system taken from the <A href="http://parts.igem.org/Part:BBa_J61051"><I>iGEM parts registry documentation
                  page</I></A>. As indicated by the figure, the model manages to capture the sigmoidal appeareance.</I></FIGCAPTION></FIGURE><P>Regarding the validity of this estimation, it might seem ambiguous due to the ansatz of 100 nM induction base level. However, at the maximal
            induction level, this yields an expression of approximately 9000 nM of protein per cell. Assuming a molecular weight of 30kDa (a GFP has about 27
            kDa) and a cell concentration of $2 \cdot 10^9$ cells per milliliter for an overnight culture [3], this would correspond to a yield of $0.432$
            µg/mL, which is regarded as reasonable. In addition, the transcription rate constant was estimated as $\text{k}_\text{MG}=0.168$
            min<SUP>-1</SUP> which is slightly higher than the transcription rate $\text{k}_\text{MR}$ for the <I>lac</I> repressor but lower than the ones
            estimated for NahR and GFP10-ER-GFP11. This indicates that although the exact value might not be correct as the proposed model is coarse, it is
            likely to at least lie within the right order of magnitude.</P><P>To conclude, the <I>ad hoc</I> proposition of 100 nM base induction level seem reasonable as it yields plausible expression levels. In addition,
            the estimated transcription rate of the GFP1-9 transcript conforms with the other transcription rate constants. Finally, the model managed to fit
            the sigmoidal shape indicating that the reaction scheme proposed describe the system well. </P><P>h) In a paper from 2013, a tripartite split GFP-association was studied by complementing GFP1-9 with GFP10-SR-GFP11 (SR = sulfite reductase)
            fusion protein [6]. The construct was studied <I>in vitro</I> under equimolar amounts of GFP1-9 and GFP10-SR-GFP11 to yield a time-intensity
            fluorescence plot. We chose to estimate the association and dissociation rate parameters from the experiments carried out. Since the reactions were
            carried out <I>in vitro</I>, we assume that the time frame in which the measurement was carried out (12 hours)  is too short for the proteins to
            decompose. In addition, we assume that once a GFP has been associated, it cannot dissociate back into it's respective fragments (this is reasonable
            as one does not expect a GFP under mild conditions to spontaneously dissociate into three parts). Also assuming that no side reactions occur and
            that everything that can react eventually reacts, the following relation for the concentration $c(t)$ of the complemented protein was derived (see
            <A href="#mathstuff">appendix</A>),

            $$c(t) = \begin{cases} G_0  - \frac{G_0}{1 + G_0kt}, &amp; \mbox{if } G_0=E_0 \\
            \frac{G_0E_0 - G_0E_0e^{k(G_0-E_0)t}}{E_0 - G_0e^{k(G_0-E_0)t}}, &amp; \mbox{if } G_0\ne E_0 \end{cases} \tag{1}$$

            where $G_0$ and $E_0$ are the concentrations of GFP1-9 and GFP10-SR-GFP11 at time $t=0$. At figure 3 in the article, a steady state defined as 95 %
            of the asymptotic concentration is reached after approximately 9 hours. As equimolar amounts of proteins were used in the paper ($G_0 = E_0 = 3.5$  µM), we get the expression

            $$0.95G_0 = G_0  - \frac{G_0}{1 + 9G_0k}\tag{2}$$

            which yields $\ce{k_\text{aGFP}=1.0 \cdot 10^{-5}}$ nM<SUP>-1</SUP>min<SUP>-1</SUP>.</P><P>i) The constant was estimated from the permeability $p$ using the relation $k = p A / V$ where $A$ is the surface area and $V$ the cell volume.
            We assume that the cell is spherical, yielding the constant $\ce{k_\text{dS}=0.281}$ min<SUP>-1</SUP> from the permeability $\ce{p=9 \cdot 10^{-8}}$
            cm s<SUP>-1</SUP> [7].</P><P>j) The constant was estimated from the diffusion constant $D$ using the relation $k = D/A$ where $A$ is the surface area. Assuming that the cell
          is spherical, we obtained $\ce{k_\text{dE}=1.44}$ min<SUP>-1</SUP> from the diffusion constant $\ce{D=10^{7}}$ Å<SUP>2</SUP>/s. [8]</P><P>k) We set the lifetime of the mRNAs to 10 min, which yields the degradation rate $\lambda=0.2$ [3].</P><P>l) Protein degardation rates in <I>E. coli</I> vary a lot [3] and to our knowledge there exist no estimated value of the NahR degradation rate.
            We choose to set the half-life of NahR to 2 hours so that it takes a similar degradation rate as the other species in the model.</P><P>m)  The half-life for the estrogen receptor ligand binding domain in mammalian cells has been shown to be more than 3 hours in the unbound state
             and 1-3 hours for the ligand bound state. Usually, protein lifetimes in <I>E. coli</I> cells vary a lot [3] and in the absence of information and
             experimental data we chose a half-life of 2 hours for both the ligand bound and unbound state.</P><P>n) The half-life of GFPs in <I>E. coli</I> range from a couple of minutes to hours [Bionumbers]. We set the half-life to 20 minutes.</P></SECTION><SECTION><H3 class="section-heading">Initial values</H3><P> As the solutions to first order ordinary differential equations are not uniquely quantified unless proper initial
            conditions is provided, these must be specified prior to solving the system. For our system we have chosen to set the concentrations of all species
            to zero except for the gene content which is set at 83.2 nM. This can be calculated by the relation $c =
            \frac{\text{C}_\text{P}}{\text{N}_\text{A}V}$ where $\text{N}_\text{A}$ is Avogadros number, $\text{C}_\text{P}$ the copy number and $V$ the cell
            volume. The latter two parameters can be found in <I>table 1</I>.
          </P></SECTION><SECTION><H3 id="mathstuff" class="section-heading">Math stuff</H3><P>

            To derive relation (1) we start by setting up the differential equations. We have that two species $a$ and $b$ can react irreversibly to yield
            $c$. We get the following reactions:

            $$\ce{a + b -&gt;[k] c}\tag{3}$$
            $$\ce{a -&gt;[\lambda_1] \phi}\tag{4}$$
            $$\ce{b -&gt;[\lambda_2] \phi}\tag{5}$$
            $$\ce{c -&gt;[\lambda_3] \phi}\tag{6}$$

            where $\text{a}$ corresponds to the GFP1-9 fragment and $\text{b}$ to GFP10-SR-GFP11 (SR = sulfite reductase). The product $\text{c}$ is the associated GFP. Since the
            reactions were carried out in vitro, we assumed that no degradation occur, i.e $\lambda_i = 0$. By balancing the masses we get the following
            equations:

            $$\frac{dc}{dt} = -\frac{db}{dt} = -\frac{da}{dt} =  kab\tag{7}$$

            These can be reduced into a single equation by noticing that $\frac{dc}{dt} + \frac{db}{dt} = 0$ and $\frac{dc}{dt} + \frac{da}{dt} = 0$. It can now be seen that
            the sum of $c$ and $b$, and $c$ and $a$ is constant. These relations can be solved to yield

            $$b = b_0 - c\tag{8}$$
            $$a = a_0 - c\tag{9}$$
            $$\frac{dc}{dt} = k(a_0-c)(b_0-c)\tag{10}$$

            Here, $a_0$ and $b_0$ are the initial concentrations of $a$ and $b$. This is a separable differential equation which can be solved by separating the
            time and concentration terms. There are two cases, either $a_0 = b_0$ or $a_0 \neq b_0$. Starting with the former case, one gets

            $$\frac{dc}{dt} = k(a_0-c)^2\tag{11}$$

            which can be separated and solved via the following operations:

            $$\frac{dc}{k(a_0-c)^2} = dt\tag{12}$$
            $$\int_{0}^{c} \frac{dx}{k(a_0-x)^2} = \int_{0}^{t}dx\tag{13}$$
            $$\frac{1}{k}(\frac{1}{a_0-c}-\frac{1}{a_0}) = t\tag{14}$$
            $$c(t) = a_0 - \frac{a_0}{a_0kt+1}\tag{15}$$

            where we have set $c(0)=0$. Similarly, one can solve the case $a_0=b_0$ by separating the varibles, partial fraction decompose the expression
            and solving the integrals. One then yields:

            $$c(t) = \frac{a_0b_0 - a_0b_0e^{k(a_0-b_0)t}}{b_0 - a_0e^{k(a_0-b_0)t}}\tag{16}$$

            The entire solution to the problem is then:

            $$c(t) = \begin{cases} a_0  - \frac{a_0}{1 + a_0kt}, &amp; \mbox{if } a_0=b_0 \\
            \frac{a_0b_0 - a_0b_0e^{k(a_0-b_0)t}}{b_0 - a_0e^{k(a_0-b_0)t}}, &amp; \mbox{if } a_0\ne b_0 \end{cases}\tag{17}$$
          </P></SECTION><SECTION><H3 class="section-heading">References</H3><BUTTON id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
            Show
          </BUTTON><DIV class="collapse" id="ref"><OL class="references"><LI>[1] Merck Millipore pETDuet-1 docuementation. <A href="http://www.merckmillipore.com/SE/en/product/pETDuet%E2%84%A2-1-DNA---Novagen,EMD_BIO-71146#documentation">Link here</A></LI><LI>[2] Michaeil Stamatakis and Nikos V. Mantzaris. &quot;Comparision of Determinstic and Stochastic Models of the <I>lac</I> Operon Genetic Network&quot;. <I>Biophysical Journal Volume 96</I>. Feb 2009 887-906</LI><LI>[3] Milo, R., Phillips, R. and Orme, N. (2016). Cell biology by the numbers. New York, NY: Garland Science. </LI><LI>[4] Rebecca L. Rich, a, Lise R. Hoth, a, Kieran F. Geoghegan, a, Thomas A. Brown, a, Peter K. LeMotte, a, Samuel P. Simons, a, Preston Hensley, a, &amp; David G. Myszka, a 2002, 'Kinetic Analysis of Estrogen Receptor/Ligand Interactions', <I>Proceedings Of The National Academy Of Sciences Of The United States Of America</I>, 13, p. 8562.</LI><LI>[5] Park, H., Lim, W. and Shin, H. (2005). In vitro binding of purified NahR regulatory protein with promoter Psal. <I>Biochimica et Biophysica Acta (BBA) - General Subjects</I>, 1725(2), pp.247-255.</LI><LI>[6] Cabantous, S., Nguyen, H., Pedelacq, J., Koraïchi, F., Chaudhary, A., Ganguly, K., Lockard, M., Favre, G., Terwilliger, T. and Waldo, G. (2013). A New Protein-Protein Interaction Sensor Based on Tripartite Split-GFP Association. <I>Scientific Reports</I>, 3(1).</LI><LI>[7] John Gutknecht. &quot;Aspirin, acetaminophen and proton transport through phospholipid bilayers and mitochondrial membranes&quot;. <I>Molecular and Cellular Biochemistry</I> 114: 3-8, 1992</LI><LI>[8] Idit Oren, Sarel J. Fleishman, Amit Kessel, Nir Ben-Tal. &quot;Free Diffusion of Steroid Hormones Across Biomembranes: A Simplex Search with Implicit Solvent Model Calculations&quot;. <I>Biophysical Journal</I> Vol. 87, August 2004, p. 768-779.</LI><LI>[9] Kocanova, S., Mazaheri, M., Caze-Subra, S. and Bystricky, K. (2010). Ligands specify estrogen receptor alpha nuclear localization and degradation. <I>BMC Cell Biology</I>, 11(1), p.98.</LI><LI>[Bionumbers] The bionumbers databse. Milo et al. Nucl. Acids Res. (2010) 38 (suppl 1): D750-D753</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Results</A></LI><LI><A href="/Team:Lund/Modeling/ModelDerivation">Model derivation</A></LI><LI><A href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</A></LI><LI><A href="/Team:Lund/Modeling/Implementation">Implementation</A></LI></UL><H5>Safety</H5><UL><LI><A href="/Team:Lund/Safety">Safety</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/HP/Silver">Overview</A></LI><LI><A href="/Team:Lund/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Lund/Engagement">Engagement</A></LI><LI><A href="/Team:Lund/HP/Media">Media</A></LI><LI><A href="/Team:Lund/Communication">Communication</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4>
        Sölvegatan 22 A
        223 62 Lund
        Sweden<A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2017 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>