<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SYSU-Software_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SYSU-Software/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>SYSU-Software:Project</TITLE></P><DIV id="for_en"><DIV id="myBody"><DIV id="navigator"><DIV id="menuContainer"><DIV id="menu"><UL><LI><A href="https://2016.igem.org/Team:SYSU-Software/Project"><DIV class="interlab_Project">Project</DIV></A><A href="https://2016.igem.org/Team:SYSU-Software/Project"><P>Overview</P></A><DIV class="Pro_class" class="" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Description"><P>Description</P></A></DIV><DIV class="Pro_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Design"><P>Design</P></A></DIV><DIV class="Pro_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Feature"><P>Features</P></A></DIV><DIV class="Pro_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Proof"><P>Proof</P></A></DIV><DIV class="Pro_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Demonstrate"><P>Demonstrate</P></A></DIV></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Model"><DIV class="interlab_Modeling">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Human_Practices"><DIV class="interlab_Human">Human Practices</DIV></A><A href="https://2016.igem.org/Team:SYSU-Software/Human_Practices"><P>Overview</P></A><DIV class="HP_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/HP/Silver"><P>Silver</P></A></DIV><DIV class="HP_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/HP/Gold"><P>Gold</P></A></DIV><DIV class="HP_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Integrated_Practices"><P>Integrated Practices</P></A></DIV><DIV class="HP_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Engagement"><P>Engagement</P></A></DIV></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Collaborations"><DIV class="interlab_Colla">Collaborations</DIV></A></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Safety"><DIV class="interlab_Safety">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Attributions"><DIV class="interlab_Attr">Attributions</DIV></A></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Interlab"><DIV class="interlab_Interlab">Interlab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Team"><DIV class="interlab_Team">Team</DIV></A><A href="https://2016.igem.org/Team:SYSU-Software/Team"><P>Overview</P></A><DIV class="Team_class" class="" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Team/Members"><P>Team Members</P></A></DIV><DIV class="Team_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Team/Instructors"><P>Instructors and Advisors</P></A></DIV><DIV class="Team_class" style="display:block"><A href="https://2016.igem.org/Team:SYSU-Software/Team/Notebook"><P>Notebook</P></A></DIV></LI><LI><A href="https://2016.igem.org/Team:SYSU-Software/Medals"><DIV class="interlab_Medals">Medals</DIV></A></LI></UL></DIV></DIV></DIV><DIV id="mybanner1"><DIV id="mybackimg"><DIV id="overview_DE"><DIV><P>OVERVIEW</P></DIV></DIV><DIV id="overview_P"><UL><LI><P>The whole model is comprised of 4 parts.</P></LI><LI><P>PATH SEARCHER: a large-scale chemicals network and a tool to find the path to generate the target product.</P></LI><LI><P>COMMUNITY SIMULATOR: a comprehensive static and dynamic mathematical model for analyzing microbial communities.</P></LI><LI><P>RBS DESIGNER: a predictive model for designing a ribosome binding site sequence according to a specific translation initiation rate.</P></LI><LI id="jumpforcon"><P>PROMOTER PREDICTOR: a model to predict the strength of a full-length bacterial promoter.</P></LI></UL></DIV></DIV></DIV><DIV class="mid-navigator-wrapper"><DIV id="jumpforcon" class="mid-navigator-container"><A href="#jumpforcon">DATA SOURCES</A><A href="#jump_1">PATH SEARCHER</A><A href="#jump_2">COMMUNITY SIMULATOR</A><A href="#jump_3">RBS DESIGNER</A><A href="#jump_4">PROMOTER PREDICTOR</A></DIV></DIV><DIV id="body_word"><DIV class="model_data_in_model"><DIV style="display:inline-block"><P>Data sources</P></DIV></DIV><DIV class="model_div_p"><UL><LI><P>The raw data used in this model come from <A href="http://www.metacyc.org/">MetaCyc</A>, <A href="http://www.rhea-db.org/">Rhea</A>, and <A href="http://parts.igem.org/">iGEM Registry</A>. We eliminate the microorganisms that</P></LI><LI><P>belong to <A href="https://www.nih.gov/">risk group 3</A> or a higher one and chemicals that are controlled for security. After preprocessing, the data</P></LI><LI id="jump_1"><P>cover more than 7,000 species, nearly 10,000 kinds of chemicals, more than 3,660,000 biochemical reactions and more</P></LI><LI><P>than 10 GB of protein coding sequence.</P></LI></UL></DIV><DIV class="model_path_searcher"><DIV style="display:inline-block"><UL><LI class="model_path_searcher1"><P>PATH SEARCHER</P></LI><LI class="model_path_searcher2"><P>Find appropriate reaction paths based on graph theory.</P></LI></UL></DIV></DIV><DIV class="model_div_p"><UL><LI><P>To find appropriate pathways to generate the target products, we develop a model based on the graph theory.We </P></LI><LI><P>considered metabolites as vertexes on a 2-dimensions plane. We firstly built a circle, which contains the conserved</P></LI><LI><P>compounds in most of microorganisms and the basic compounds that lives need (such as Glucose,PEP,PAG,etc.).</P></LI><LI><P>We then add other chemicals outside the circle to the graph.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Example:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Given the vertices of the graph, we should define the edges of the graph as reactions. To find the resultant</P></LI><LI><P>corresponding to a specific reactant in a reaction, we developed a creative model based on an existing concept</P></LI><LI><P>“reaction pair” to distinguish which product is corresponding to which reactant in the sake of carbon skeleton changing.</P></LI></UL></DIV><DIV class="model_gcm"><DIV class="model_gcm_p" style="display:inline-block"><P>Group contribution model</P></DIV></DIV><DIV class="model_div_p"><UL><LI><P>We introduce a method to estimate the standard Gibbs energies of biochemical compounds and of biochemical reactions</P></LI><LI><P>by adding up the energy contributions of groups.</P></LI><LI><P>For most biochemical reactions, experimental data on equilibrium constants or Gibbs energies are not available.</P></LI><LI><P>However, the standard Gibbs energy of reaction is related to the standard Gibbs energies of formation of its reactants</P></LI><LI><P>and products. And the difference of the standard Gibbs energies between the reactants and products represent the</P></LI><LI><P>total energy absorb(or release)while the reaction occurs.</P></LI></UL></DIV><DIV class="model_step"><DIV class="model_step_p" style="display:inline-block"><P>STEPS</P></DIV></DIV><DIV class="model_div_p"><UL><LI><P>The thermodynamic analysis can be carried out using the standard Gibbs energy of reaction, -ΔG0. which is closely</P></LI><LI><P>related to the equilibrium constant, K :</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>where R is the ideal-gas constant and T is the temperature. The standard state for -ΔG0 is a dilute aqueous solution</P></LI><LI><P>at T = 25°C, pH 7, and concentrations of compounds (other than H+, OH-, and H20) equal 1M.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Letting Vi be the stoichiometric coefficient of compound Si, we can write a reaction (or any transformation with</P></LI><LI><P>known stoichiometry) as:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Where Vi&gt; 0 for products and Vi&lt; 0 for reactants. Let  -ΔG0 be the Gibbs energy of formation of Si.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The Gibbs energy of reaction, -ΔG0, is then given by the equation mentioned above.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The contribution of each group to the thermodynamic must also be provided, along with the origin, a </P></LI><LI><P>starting value that is used in the estimation (and is constant for all compounds).</P></LI><LI><P>Let C0 be the origin for the property C, and let Ci be the contribution of group gi,which is used Ni times in the</P></LI><LI><P>compound. The property C for the whole compound is calculated as:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Within this method, we view the compound as composed of functional groups and sums amounts contributed by</P></LI><LI><P>each group to the overall value of the property, or a suitable mathematical function of the property.</P></LI><LI><P>Each group corresponding to a standard initial Gibbs energy. By adding the group contributions of a set of</P></LI><LI><P>compounds, one can calculate the standard Gibbs energies for any chemical or biochemical transformation in aqueous</P></LI><LI><P>solution.</P></LI><LI><P>The contributions were estimated via multiple linear regression, using screened and weighted literature data. For</P></LI><LI><P>most of the data employed, the error is less than 2 kcal/mol.</P></LI><LI><P>According to the standard data, the error is less than 5 kcal/mol. Thus, the method provides an acceptable first</P></LI><LI><P>approximation to Gibbs energies and equilibrium constants in biochemical systems.</P></LI></UL></DIV><DIV class="model_gcm"><DIV class="model_gcm_p" style="display:inline-block"><P>Reaction Pair</P></DIV></DIV><DIV class="model_div_p"><UL><LI><P>As we have processes of filtering the candidate pathways, which require evaluating the Gibbs Energies of the</P></LI><LI><P>chemical reaction. Yet, the change of Gibbs energy within a certain reaction (ΔG) is nearly impossible to be estimated</P></LI><LI><P>directly. In this part, we introduce a new concept “reaction pair” to distinguish which product is corresponding to which</P></LI><LI><P>reactant in the sake of carbon skeleton.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>In this way, we make it clear to identify the corresponding reaction pair in a certain chemical reaction equation, while</P></LI><LI><P>adding the group contributions for standard Gibbs Energies of biochemical compounds in aqueous solution, we are able</P></LI><LI><P>to get the standard Gibbs Energy of the whole reaction.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>When given the name of the target product, we construct a converse graph of our directed graph and find paths</P></LI><LI><P>from the target product to the circle. If more than one paths are found, the software will choose an appropriate path</P></LI><LI><P>for the user.</P></LI></UL></DIV><DIV class="model_path_searcher"><DIV style="display:inline-block"><UL><LI class="model_path_searcher1"><P>COMMUNITY SIMULATOR</P></LI><LI class="model_path_searcher2"><P>Metabolic Modeling and Analysis of Microbial Communities based on linear bi-level programming.</P></LI></UL></DIV></DIV><DIV class="model_div_p"><UL><LI><P>The original FBA model (Flux Balance Analysis) simulates the reactions in an individual microorganism. The flux of re-</P></LI><LI><P>actions is used as variables,and the steady-state mass balance and a lower and upper bound for each flux are used as</P></LI><LI><P>constraint condition. The model is built based on linear programming.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>To simulate the flux balance of a community, we improved the original FBA model. Based on linear bi-level</P></LI><LI><P>programming, we developed a new model for the metabolic modeling and analysis of microbial communities[6]. In the</P></LI><LI><P>new model, we consider the transport process of a metabolite as a reaction.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Where ui is the uptake or export flux of the metabolite in the microorganism,trani is the uptake or export flux</P></LI><LI><P>of the metabolite  outside the microorganism.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>According to the dual theory of linear programming, the dual gaps of the inner problems are added to the outer</P></LI><LI><P>objective function. The model is then transformed into a nonlinear programming. We use OpenBLAS, an open</P></LI><LI><P>source implementation of the BLAS (Basic Linear Algebra Subprograms) API, and IPOPT, a software library for large</P></LI><LI><P>scale nonlinear optimization of continuous systems, to solved the non-linear programming.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The dual gap:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The non-linear programming:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>d-COMMUNITY SIMULATOR: A dynamic multi-level and multi-objective metabolic modeling of microbial</P></LI><LI><P>communities based on COMMUNITY SIMULATOR.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>d-COMMUNITY SIMULATOR is a dynamic multi-level and multi-objective metabolic modeling of microbial</P></LI><LI><P>communities. Based on COMMUNITY SIMULATOR, d-COMMUNITY SIMULATOR formulation has its unique features,</P></LI><LI><P>which enables not only capturing the dynamic behavior of microbial communities, but also taking both biomass and</P></LI><LI><P>shared metabolite into account.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P><STRONG>Symbol:</STRONG></P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P><STRONG>Unique features and accompanying shortcomings</STRONG></P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>To improve COMMUNITY SIMULATOR, d-COMMUNITY SIMULATOR uses two ordinary differential equations to</P></LI><LI><P>precisely calculate varieties of concentrations, so we can capture the dynamic behavior of microbial communities.</P></LI><LI><P>Thanks to this, our software can display concentration-time graph to help users catch information in a second and</P></LI><LI id="jump_3"><P>hence, be more user-friendly. Unavoidably, the ODEs have its side effect. That is, this advanced model is difficult to</P></LI><LI><P>solve due to the presence of ODEs. Luckily, by applying Backward Euler method, it is converted to a linear bi-level opti-</P></LI><LI><P>mization problem.</P></LI></UL></DIV><DIV class="model_path_searcher"><DIV style="display:inline-block"><UL><LI class="model_path_searcher1"><P>RBS DESIGNER</P></LI><LI class="model_path_searcher2"><P>Automatic design of synthetic ribosome binding sites based on dynamic programming.</P></LI></UL></DIV></DIV><DIV class="model_div_p"><UL><LI><P>The equilibrium statistical thermodynamic model quantifies the strengths of the molecular interactions between an</P></LI><LI><P>mRNA transcript and the 30S ribosome complex—which includes the 16S rRNA and the tRNA—to predict the result-</P></LI><LI><P>ing translation initiation rate:</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>where is Boltzmann constant, is the difference in Gibbs free energy between the initial state, the folded mRNA</P></LI><LI><P>transcript and the free 30S complex, and the final state, the assembled 30S pre-initiation complex bound on an mRNA</P></LI><LI><P>transcript.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P><STRONG>Symbol</STRONG></P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Starting from 3’ and 5’ of a mRNA strand,  and  is calculated by using dynamic</P></LI><LI><P>programming recursions to evaluate the most stable secondary structure (minimum free energy structure) of the</P></LI><LI><P>mRNA sub-sequence as shown in the following figures. [9]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Other parts of are shown in the form below: [8,10,11]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI id="jump_4"><P>According to the formulations and model mentioned above, we can design a target sequence by traversal algorithm when</P></LI><LI><P>given a target translation initiation rate, selecting the optimal solution according to some biological standard and the deviation</P></LI><LI><P>between the target rate and the real one. [8]</P></LI></UL></DIV><DIV class="model_path_searcher"><DIV style="display:inline-block"><UL><LI class="model_path_searcher1"><P>PROMOTER PREDICTOR</P></LI><LI class="model_path_searcher2"><P>Predicting promoters strength based on regression analysis</P></LI></UL></DIV></DIV><DIV class="model_div_p"><UL><LI><P>To evaluate the contribution of sequence to promoter strength and function, we used modular position weight</P></LI><LI><P>matrix models comprised of each promoter motif and a penalty score for suboptimal motif location. The scores of the</P></LI><LI><P>promoter sequences are used to predict the promoter strength. We choose a set of promoters of which the</P></LI><LI><P>promoter strength is known to fit a promoter strength predicting model. [1,2]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>PROMOTER PREDICTOR were constructed for each core-promoter motif with a penalty term applied for non-op-</P></LI><LI><P>timal spacer and discriminator lengths. The up-element is also scored.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>For the core promoter [1]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>where</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The spacer and discriminator length penalty scores (S + D) were applied to promoters with suboptimal spacing be-</P></LI><LI><P>tween the +1, −10, and −35 motifs based on the observed spacing frequency for promoters.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>For the up-element [2]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>For the accuracy of the prediction, a total score was calculated for each promoter based on six modules by sum-</P></LI><LI><P>ming the scores that have a positive correlation with the logarithm of promoter strength: [1,2]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>For the score of the promoter , we have known that: [1,2]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Thus, the function between PWM score and log2(promoter strength) was then fitted by using least square regression.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Problem rises concerning the prediction of the strength of the promoter, that is, promoters contenting in registry of</P></LI><LI><P>iGEM have no annotations, yet we need to distinguish -35box、spacer、-10box、disc、and up-element intervals for the</P></LI><LI><P>calculation of the strength prediction. Therefore, we built a tool to identify the specific intervals in unaligned sequences</P></LI><LI><P>by identifying the consensus pattern based on weight matrix too. [4,12]</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>Given a set of sequences that have annotations of the specific intervals (-35, -10, discriminator, up-element,spacer), we</P></LI><LI><P>constructed weight matrices of every intervals.</P></LI><LI><P><STRONG>Example weight matrix: [13]</STRONG></P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The score of a new sequence is given by the weight matrices by summing the number of every positions of the sequence.</P></LI></UL></DIV><DIV class="model_div_p"><UL><LI><P>The positions of the intervals are determined by finding the highest score when the weight matrices are sliding on the</P></LI><LI><P>sequences and the order of determining the intervals is “-10”, “-35”, “far-proximal”, “proximal”, “distal”. “spacer” is between “-10” and “-35”. “disc” is between</P></LI><LI><P>“-10” and “-1”. “-1” and “+1” is “start”</P></LI></UL></DIV><DIV class="model_data_in_model"><DIV style="display:inline-block"><P>REFERENCE</P></DIV></DIV><DIV class="model_div_p"><UL><LI><P>1. Rhodius VA, Mutalik VK. Predicting strength and function for promoters of the Escherichia coli alternative sigma factor,</P></LI><LI><P>sigmaE. Proc Natl Acad Sci U S A. 2010;107(7):2854-2859. doi:10.1073/pnas.0915066107.</P></LI><LI><P>2. Rhodius VA, Mutalik VK, Gross CA. Predicting the strength of UP-elements and full-length E. coli sigmaE promoters.</P></LI><LI><P>Nucleic Acids Res. 2012;40(7):2907-2924. doi:10.1093/nar/gkr1190.</P></LI><LI><P>3. Berg OG, von Hippel PH. Selection of DNA binding sites by regulatory proteins. Trends Biochem Sci.</P></LI><LI><P>1988;13(6):207-211. doi:10.1016/0968-0004(88)90085-0.</P></LI><LI><P>4. Hertz GZ, Stormo GD. Identifying DNA and protein patterns with statistically significant alignments of multiple</P></LI><LI><P>sequences. Bioinformatics. 1999;15(7-8):563-577. doi:10.1093/bioinformatics/15.7.563.</P></LI><LI><P>5. Estrem ST, Gaal T, Ross W, Gourse RL. Identification of an UP element consensus sequence for bacterial promoters.</P></LI><LI><P>Proc Natl Acad Sci. 1998;95(17):9761-9766. doi:10.1073/pnas.95.17.9761.</P></LI><LI><P>6. Zomorrodi AR, Maranas CD. OptCom: A multi-level optimization framework for the metabolic modeling and analysis of</P></LI><LI><P>microbial communities. PLoS Comput Biol. 2012;8(2). doi:10.1371/journal.pcbi.1002363.</P></LI><LI><P>7. Zomorrodi AR, Islam MM, Maranas CD. D-OptCom: Dynamic Multi-level and Multi-objective Metabolic Modeling of</P></LI><LI><P>Microbial Communities. ACS Synth Biol. 2014;3(4):247-257. doi:10.1021/sb4001307.</P></LI><LI><P>8. Salis HM, Mirsky EA, Voigt CA. Automated design of synthetic ribosome binding sites to control protein expression. Nat</P></LI><LI><P>Biotechnol. 2009;27(10):946-950. doi:10.1038/nbt.1568.</P></LI><LI><P>9. Dirks RM, Bois JS, Schaeffer JM, Winfree E, Pierce NA. Thermodynamic Analysis of Interacting Nucleic Acid Strands. SIAM</P></LI><LI><P>Rev. 2007;49(1):65-88. doi:10.1137/060651100.</P></LI><LI><P>10. Znosko BM, Silvestri SB, Volkman H, Boswell B, Serra MJ. Thermodynamic parameters for an expanded nearest-neighbor</P></LI><LI><P>model for the formation of RNA duplexes with single nucleotide bulges. Biochemistry. 2002;41(33):10406-10417.</P></LI><LI><P>doi:10.1021/bi025781q.</P></LI><LI><P>11. Mathews DH, Sabina J, Zuker M, Turner DH. Expanded sequence dependence of thermodynamic parameters improves</P></LI><LI><P>prediction of RNA secondary structure. J Mol Biol. 1999;288(5):911-940. doi:10.1006/jmbi.1999.2700.</P></LI><LI><P>12. Huerta AM, Collado-Vides J. Sigma70 promoters in Escherichia coli: Specific transcription in dense regions of overlapping</P></LI><LI><P>promoter-like signals. J Mol Biol. 2003;333(2):261-278. doi:10.1016/j.jmb.2003.07.017.</P></LI><LI><P>13. Huerta AM, Francino MP, Morett E, Collado-Vides J. Selection for unequal densities of  sigma70 promoter-like signals in</P></LI><LI><P>different regions of large bacterial genomes. PLoS Genet. 2006;2(11):1740-1750. doi:10.1371/journal.pgen.0020185.</P></LI><LI><P>14. Mavrovouniotis ML. Group contributions for estimating standard Gibbs energies of formation of biochemical compounds</P></LI><LI><P>in aqueous solution. Biotechnol Bioeng. 1990;36(10):1070-1082. doi:10.1002/bit.260361013.</P></LI><LI><P>15. Henry CS, Broadbelt LJ, Hatzimanikatis V. Thermodynamics-Based Metabolic Flux Analysis. Biophys J.</P></LI><LI><P>2007;92(5):1792-1805. doi:10.1529/biophysj.106.093138.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>