<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_Notebook_Protocols_Primers skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/Notebook/Protocols/Primers</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>UBC iGEM 2015</TITLE></P><DIV class="UBCheader"><DIV class="UBCbod"><DIV id="UBCheader"><DIV id="UBCmenu"><DIV id="proj"><A href="https://2015.igem.org/Team:British_Columbia/Description"><H4>Project</H4></A><UL><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/Growing">Genetic Toolbox</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Screening">Screening</A></LI><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/Imidacloprid">Degrading Imid</A></LI><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/6CNA">6-CNA Detox</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Design">Prototype Testing</A></LI></UL></DIV><DIV id="parts"><A href="https://2015.igem.org/Team:British_Columbia/Parts"><H4>Parts</H4></A><UL><LI><A href="https://2015.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Part_Collection">Part Collection</A></LI></UL></DIV><DIV id="model"><A href="https://2015.igem.org/Team:British_Columbia/Modeling"><H4>Modeling</H4></A></DIV><DIV id="hp"><A href="https://2015.igem.org/Team:British_Columbia/Practices"><H4 style="margin-bottom:-20px;">Policy &amp; Practices</H4></A><UL><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/BeeFarm">Bee Farm</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Booth">Science World Booth</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Booth2">Farmer's Market Booth</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Report">CCD Report</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/RiskReport" style="font-size:10px;">Regulation &amp; Risk Assessment Report</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Infographic">Bee Infographs</A><P>A primer is a strand of nucleic acid that serves as a starting point for DNA synthesis. It is required for DNA replication because the enzymes that catalyze this process, DNA polymerases, can only add new nucleotides to an existing strand of DNA. Many of the laboratory techniques of molecular biology, such as DNA sequencing and the polymerase chain reaction (PCR), require DNA primers. These primers are usually short, chemically synthesized oligonucleotides, with a length of about twenty bases.</P><EM>This document will give you an overview on the proper methods of dealing with primers. What to do when they arrive, how to dilute them, and how to store them.</EM><H2>Protocol </H2><STRONG>
Dealing with newly arrived primers:</STRONG> <P>Primers arrive in screw-top tubes, usually with a blue lid, containing freeze dried oligonucleotides. To use these primers, follow the steps below:</P></LI><LI>
    Arrival: 
    Use of high quality, purified DNA templates greatly enhances the success of PCR:

   </LI><LI>Primers: 
    Oligonucleotide primers are generally 20–40 nucleotides in length and ideally have a GC content of 40–60%. Computer programs such as Primer3 (<A re_target="_blank" target="_blank" href="http://frodo.wi.mit.edu/primer3">http://frodo.wi.mit.edu/primer3</A>) can be used to design or analyze primers. The final concentration of each primer in a reaction may be 0.05–1 μM, typically 0.1–0.5 μM.</LI><LI>Mg<SUP>++ </SUP>and additives: 
    Mg<SUP>++ </SUP>concentration of 1.5–2.0 mM is optimal for most PCR products generated with <EM>Taq </EM>DNA Polymerase. The final Mg<SUP>++</SUP> concentration in 1X Standard <EM>Taq </EM>Reaction Buffer is 1.5 mM. This supports satisfactory amplification of most amplicons. However, Mg<SUP>++</SUP> can be further optimized in 0.5 or 1.0 mM increments using MgCl<SUB>2</SUB>. 
    Amplification of some difficult targets, like GC-rich sequences, may be improved with additives, such as DMSO (3) or formamide (4).</LI><LI>Deoxynucleotides:
    The final concentration of dNTPs is typically 200 μM of each deoxynucleotide.</LI><LI><EM>Taq </EM>DNA Polymerase Concentration: 
    We generally recommend using <EM>Taq</EM> DNA Polymerase at a concentration of 25 units/ml (1.25 units/50 μl reaction). However, the optimal concentration of <EM>Taq</EM> DNA Polymerase may range from 5–50 units/ml (0.25–2.5 units/50 μl reaction) in specialized applications.</LI><LI>Denaturation: 
    An initial denaturation of 30 seconds at 95°C is sufficient for most amplicons from pure DNA templates. For difficult templates such as GC-rich sequences, a longer initial denaturation of 2–4 minutes at 95°C is recommended prior to PCR cycling to fully denature the template. With colony PCR, an initial 5 minute denaturation at 95°C is recommended. 
    During thermocycling a 15–30 second denaturation at 95°C is recommended.</LI><LI>Annealing: 
    The annealing step is typically 15–60 seconds. Annealing temperature is based on the T<SUB>m</SUB> of the primer pair and is typically 45–68°C. Annealing temperatures can be optimized by doing a temperature gradient PCR starting 5°C below the calculated T<SUB>m</SUB>.  The NEB <A target="_blank" href="/tools-and-resources/interactive-tools/tm-calculator">Tm Calculator </A>is recommended to calculate an appropriate annealing temperature.
    When primers with annealing temperatures above 65°C are used, a 2-step PCR protocol is possible (see #10). </LI><LI>Extension: 
    The recommended extension temperature is 68°C. Extension times are generally 1 minute per kb. A final extension of 5 minutes at 68°C is recommended.</LI><LI>Cycle number: 
    Generally, 25-35 cycles yields sufficient product. </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>