<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_British_Columbia_Imidacloprid skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:British Columbia/Imidacloprid</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>UBC iGEM 2015</TITLE></P><DIV class="UBCheader"><DIV class="UBCbod"><DIV id="UBCheader"><DIV id="UBCmenu"><DIV id="proj"><A href="https://2015.igem.org/Team:British_Columbia/Description"><H4>Project</H4></A><UL><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/Growing">Genetic Toolbox</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Screening">Screening</A></LI><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/Imidacloprid">Degrading Imid</A></LI><LI style="font-size:15px;"><A href="https://2015.igem.org/Team:British_Columbia/6CNA">6-CNA Detox</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Design">Prototype Testing</A></LI></UL></DIV><DIV id="parts"><A href="https://2015.igem.org/Team:British_Columbia/Parts"><H4>Parts</H4></A><UL><LI><A href="https://2015.igem.org/Team:British_Columbia/Basic_Part">Basic Part</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Composite_Part">Composite Part</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Part_Collection">Part Collection</A></LI></UL></DIV><DIV id="model"><A href="https://2015.igem.org/Team:British_Columbia/Modeling"><H4>Modeling</H4></A></DIV><DIV id="hp"><A href="https://2015.igem.org/Team:British_Columbia/Practices"><H4 style="margin-bottom:-20px;">Policy &amp; Practices</H4></A><UL><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/BeeFarm">Bee Farm</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Booth">Science World Booth</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Booth2">Farmer's Market Booth</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Report">CCD Report</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/RiskReport" style="font-size:10px;">Regulation &amp; Risk Assessment Report</A></LI><LI><A href="https://2015.igem.org/Team:British_Columbia/Practices/Infographic">Bee Infographs</A> Imidacloprid<DIV id="two">Â </DIV><DIV id="UBCbody"><P align="justify">
Imidacloprid is a neonicotinoid commonly used in pesticides around the world [1]. 
Studies have shown that the use of this pesticide has adverse effects on insects since it acts as an neurotoxin by binding and blocking the nicotinergic neuronal pathway [2].  As a result, honeybees become paralyzed and eventually die [3]. Studies have shown that there are three cytochrome P450s (CYPs) capable of modifying imidacloprid into less toxic substances [4-6]. In light of such findings, experiments were conducted in order  to demonstrate that CYP6G1, CYP2D6, and CYP6CM1 could modify imidacloprid into less toxic compounds. In order to test for such modifications, constructs were created to code for: 1) a cytochrome P450 (CYP), 2) a targeting signal sequence, pelB, and 3) an NADPH-cytochrome P450 oxidoreductase (CPR).
</P><P align="justify">
In an attempt to improve CYP expression, pelB was added to the constructs. pelB targets the CYPs to the periplasm of the cell, where the CYPs are fully functional [7]. In addition, a P450 oxidoreductase (CPR) is also required to make the CYPs fully functional. AgCPR, a P450 oxidoreductase from <I>Anopheles gambiae</I>, reduces the CYPs by transferring two electrons from NADPH, a reaction necessary for the hydroxylation of imidacloprid by the CYPs [8].  As such, AgCPR was included in the constructs.
</P><P align="justify">
Final constructs were inserted into <I>E. coli</I> and tests were conducted to test the expression of the CYPs in <I>E. coli</I> as well as to test the degradative ability of the CYPs on imidacloprid. Two methods were used in the construction of the final constructs: 1) megaprimer mutagenesis PCR to insert pelB and 2) standard assembly to assemble the CYPs and the AgCPR. Gene expression was tested via SDS-PAGE electrophoresis and imidacloprid metabolism assays were conducted using HPLC.

</P><H2>Goals</H2><P align="justify">
Three cytochrome P450s, CYP6CM1, CYP6G1 and CYP2D6, have been found to modify imidacloprid (IMI) into less toxic compounds, primarily 4- and 5-hydroxyimidacloprid. Our project goals were to:</P></DIV></LI><LI> Optimize the CYPs for heterologous expression in <I>E. coli</I> through codon optimization. </LI><LI> Create three <I>E. coli</I> strains, each expressing either CYP6CM1, CYP6G1 or CYP2D6 by using a BioBrick compatible expression system (pSCB1C3) to enable <I>E. coli</I> to modify imidacloprid.</LI><LI> Test for protein expression and characterize <I>in vivo</I> detoxification of imidacloprid in <I>E. coli</I> through SDS-PAGE and High Performance Liquid Chromatography (HPLC), respectively.<H2>Construct Development</H2><P align="justify">
All constructs were made using restriction digest and ligation methods (standard assembly). pelB was inserted upstream of the CYPs and the AgCPR using megaprimer mutagenesis PCR. Please refer to &quot;Table 1&quot; for a complete list of construct designs. Refer to <A href="https://static.igem.org/mediawiki/2015/2/2f/Restriction_Digest_UBC.pdf"> Restriction Digest Protocol </A> for standard assembly methods. Further, refer to the <A href="https://static.igem.org/mediawiki/2015/f/fc/Mutagenesis_PCR_UBC.pdf"> Megaprimer Mutagenesis Protocol </A> used to insert pelB upstream of the CYPs and the AgCPR.
</P><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThree"><H4>Table 1: Construct List</H4></DIV><DIV style="height: 0px;" aria-expanded="false" id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>iGEM Part Number</TH><TH>Type</TH><TH>Description</TH><TH>LacI Repressor</TH><TH>pelB</TH></TR><TR><TD>BBa_K1813003</TD><TD>Coding</TD><TD>CYP6CM1</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813004</TD><TD>Coding</TD><TD>CYP6G1</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813005</TD><TD>Coding</TD><TD>HUMCYPDB1</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813010</TD><TD>Coding</TD><TD>A. gamb NADPH Reductase</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813011</TD><TD>Composite</TD><TD>ptac CYP6CM1 term</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813012</TD><TD>Composite</TD><TD>ptac CYP6G1 term</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813013</TD><TD>Composite</TD><TD>ptac CYP2D6 term</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813018</TD><TD>Composite</TD><TD>ptac AgambNADPH term</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813024</TD><TD>Composite</TD><TD>LacI Reversed HUMCYPDB1</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813025</TD><TD>Composite</TD><TD>LacI Reversed CYP6G1</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813027</TD><TD>Composite</TD><TD>LacI Reversed CPY6CM1</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813028</TD><TD>Composite</TD><TD>LacI Reversed AgamNADPH Reductase</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813032</TD><TD>Composite</TD><TD>LacIR AgCPR CYP2D6</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813033</TD><TD>Composite</TD><TD>LacIR AgCPR CYP6G1</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813034</TD><TD>Composite</TD><TD>LacIR AgCPR CYP6CM1</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813046</TD><TD>Coding</TD><TD>pelB CYP6G1</TD><TD>No</TD><TD>No</TD></TR><TR><TD>BBa_K1813050</TD><TD>Composite</TD><TD>ptac pelB CYP6G1</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813051</TD><TD>Coding</TD><TD>pelB CYP6CM1</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813052</TD><TD>Composite</TD><TD>ptac pelB CYP6CM1</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813053</TD><TD>Coding</TD><TD>pelB AgCPR</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813054</TD><TD>Composite</TD><TD>ptac pelB AgCPR</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813055</TD><TD>Coding</TD><TD>pelB CYP2D6</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813056</TD><TD>Composite</TD><TD>ptac pelB CYP2D6</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813057</TD><TD>Composite</TD><TD>LacIR pelB CYP6G1</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813058</TD><TD>Composite</TD><TD>LacIR pelB CYP6CM1</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813059</TD><TD>Composite</TD><TD>LacIR pelB AgCPR</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813060</TD><TD>Composite</TD><TD>LacIR pelB CYP2D6</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813061</TD><TD>Composite</TD><TD>ptac pelB AgCPR pelb CYP6G1</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813062</TD><TD>Composite</TD><TD>ptac pelB AgCPR pelb CYP6CM1</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813063</TD><TD>Composite</TD><TD>ptac pelB AgCPR pelb CYP2D6</TD><TD>No</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813064</TD><TD>Composite</TD><TD>LacIR pelB AgCPR ptac pelB-CYP6G1</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813065</TD><TD>Composite</TD><TD>LacIR pelB AgCPR ptac pelB CYP6CM1</TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>BBa_K1813066</TD><TD>Composite</TD><TD>LacIR pelB AgCPR ptac pelB CYP2D6</TD><TD>Yes</TD><TD>Yes</TD></TR></TBODY></TABLE></P></DIV></DIV></DIV><H2>Results</H2><P align="justify">
Please refer to &quot;Table 2&quot; for protein expression results and &quot;Table 3&quot; for imidacloprid degradation results. Further, refer to 
<A href="https://static.igem.org/mediawiki/2015/c/c7/SDS_Page_IMI_UBC.pdf"> SDS-PAGE Protocol </A> and <A href="https://static.igem.org/mediawiki/2015/a/a5/HPLC_UBC.pdf"> HPLC Protocol </A> for the methods used to run gene expression and metabolism assays, respectively. Protein expression was tested using SDS-PAGE methods and imidacloprid metabolism assays were tested using High-Performance Liquid Chromatography (HPLC). No detectable protein levels were seen through expression experiments. Further, no detectable levels of imidacloprid metabolites were visible on HPLC traces.
</P><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFour"><H4>Table 2: Protein Expression Results</H4></DIV><DIV style="height: 0px;" aria-expanded="false" id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>iGEM Part Number</TH><TH>Type</TH><TH>Description</TH><TH>LacI Repressor</TH><TH>pelB</TH><TH>Sequenced Confirmed</TH><TH>Visible Expression</TH></TR><TR><TD>BBa_K1813011</TD><TD>Composite</TD><TD>ptac CYP6CM1 term</TD><TD>No</TD><TD>No</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813012</TD><TD>Composite</TD><TD>ptac CYP6G1 term</TD><TD>No</TD><TD>No</TD><TD>Yes - Mutation Occurred</TD><TD>No</TD></TR><TR><TD>BBa_K1813013</TD><TD>Composite</TD><TD>ptac CYP2D6 term</TD><TD>No</TD><TD>No</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813018</TD><TD>Composite</TD><TD>ptac AgambNADPH term</TD><TD>No</TD><TD>No</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813052</TD><TD>Composite</TD><TD>ptac pelB CYP6CM1</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813050</TD><TD>Composite</TD><TD>ptac pelB CYP6G1</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813056</TD><TD>Composite</TD><TD>ptac pelB HUMCYPDB1</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813054</TD><TD>Composite</TD><TD>ptac pelB AgCPR</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813057</TD><TD>Composite</TD><TD>LacIR pelB CYP6G1</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813058</TD><TD>Composite</TD><TD>LacIR pelB CYP6CM1</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813059</TD><TD>Composite</TD><TD>LacIR pelB AgCPR</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813060</TD><TD>Composite</TD><TD>LacIR pelB HUMCYPDB1</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR></TBODY></TABLE></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFive"><H4>Table 3: Imidacloprid Degradation Results</H4></DIV><DIV style="height: 0px;" aria-expanded="false" id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive"><DIV class="panel-body"><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>iGEM Part Number</TH><TH>Type</TH><TH>Description</TH><TH>LacI Repressor</TH><TH>pelB</TH><TH>Sequenced</TH><TH>Detectable Degradative Ability</TH></TR><TR><TD>BBa_K1813052</TD><TD>Composite</TD><TD>ptac pelB CYP6CM1</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813050</TD><TD>Composite</TD><TD>ptac pelB CYP6G1</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>BBa_K1813054</TD><TD>Composite</TD><TD>ptac pelB AgCPR</TD><TD>No</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR></TBODY></TABLE></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTen"><H4>HPLC Traces</H4></DIV><DIV style="height: 0px;" aria-expanded="false" id="collapseTen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTen"><DIV class="panel-body"><P align="justify">Trace 1 was obtained after running HPLC with construct &quot;BBa_K1813050&quot; (ptac pelB CYP6G1). Imidacloprid has a retention rate of 5.128 minutes. An NADPH peak is seen at 0.8 minutes. A metabolite peak was expected to appear between 4-5 minutes. Seeing that there is almost no cell background noise between 4-5 minutes, the expected metabolite peak would be clear. Trace 2 was run with the same construct however no imidacloprid was added.  </P><DIV style="width:900px; margin:auto;"><SMALL> Trace 1: HPLC curve. </SMALL></DIV><DIV style="width:900px; margin:auto;"><SMALL> Trace 2: HPLC curve (without IMI). </SMALL></DIV></DIV></DIV></DIV><H2>Protocol Optimization</H2><P align="justify">
Several aspects of the SDS-PAGE and HPLC protocols could be modified in order to optimize gene expression and imidacloprid degradation. For SDS-PAGE, delta-aminolevulinic acid was added during the growth phase of the bacteria [6]. The addition of this heme precursor would âlightenâ the burden of the cells by allowing for the more efficient production of heme, a cofactor required by the CYPs. In addition, overnight culture times could be extended in order to acquire more cell mass. With regards to HPLC, and in addition to the SDS-PAGE optimizations, several aspects of the HPLC protocol could be modified in order to optimize for gene expression and the degradation of imidacloprid. Such modifications would include adding more or different protease inhibitor mixes, incubating the CYP/AgCPR/imidacloprid/NADPH mixture for longer times, and incubating the mixtures with different concentrations of imidacloprid.
</P><H2>References</H2></LI><LI>Elbert, A., Becker, B., Hartwig, J., Erdelen, C., 1991. Imidacloprid â a new systemic insecticide. Pflanzenschutz-Nachrichten Bayer 44, 113â135</LI><LI>Liu, M.Y., Casida, J.E., 1993. High affinity binding of [3 H] imidacloprid in the insect acetylcholine receptor. Pestic. Biochem. Physiol. 46, 40â46.
</LI><LI>Ishaaya, I. (2001). Biochemical Sites of Insecticide Action and Resistance. Springer. ISBN 3540676252.</LI><LI>JouÃen, N., Heckel, D.G., Haas, M., Schuphan, I., Schmidt, B., 2008. Metabolism of imidacloprid and DDT by P450 CYP6G1 expressed in cell cultures of Nicotiana tabacum suggests detoxification of these insecticides in Cyp6g1-overexpressing strains of Drosophila melanogaster, leading to resistance. Pest Manag. Sci. 64, 65â73</LI><LI>Schulz-Jander, D.A., Casida, J.E., 2002. Imidacloprid insecticide metabolism: human cytochrome P450 isozymes differ in selectivity for imidazolidine oxidation versus nitroimine reduction. Toxicol. Lett. 132, 65â70.</LI><LI>Karunker, Iris, Evangelia Morou, Dimitra Nikou, Ralf Nauen, Rotem Sertchook, Bradley J. Stevenson, Mark J.I. Paine, Shai Morin, and John Vontas. &quot;Structural Model and Functional Characterization of the Bemisia Tabaci CYP6CM1vQ, a Cytochrome P450 Associated with High Levels of Imidacloprid Resistance.&quot; Insect Biochemistry and Molecular Biology 39.10 (2009): 697-706. Elsevier. Web.</LI><LI>Pritchard, Michael P., Lesley McLaughlin, and Thomas Friedberg. &quot;Establishment of Functional Human Cytochrome P450 Monooxygenase Systems in Escherichia Coli.&quot; Ed. Ian R. Phillips and Elizabeth A. Shephard. 320 (2006): 19-29. Springer Link. Web.</LI><LI>Nikou, Dimitra, Hilary Ranson, Janet Hemingway. âAn adult-specific CYP6 P450 gene is overexpressed in a pyrethroid-resistant strain of the malaria vector, Anopheles gambiaeâ Gene. 318 (2003): 91-102. Elsevier. Web.<DIV id="footer" style="height:60px; padding-top:5px;"><H6><P style="font-size: 1.5em"><B style="position:absolute; right:50px; bottom:12px"> UBC iGEM 2015 </B></P></H6></DIV></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>