<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_HS_Israel_Modelling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion HS Israel/Modelling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE> Technion 2015 HS Team's Wiki</TITLE></P><DIV class="content" style="width:100%;top:150px;position:absolute"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/MedalFulfillment"> Medal Fulfillment</A><A href="https://2015.igem.org/Team:Technion_HS_Israel"> Home</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Description"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN><SPAN class="sdt_descr">More about our project</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Description">Description</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Project/Results">Results</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Parts">Parts</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Documents</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Experiments">Experiments</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Notebook">Notebook</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/safety">safety</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Protocols">Protocols</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modelling</SPAN><SPAN class="sdt_descr">We take our math seriously</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling">Model Overview</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/PlasmidLoss">Plasmid Loss</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Equations">Equations</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Parameters">Parameters</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Results">Results</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Documents">Documents</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Software</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software">Software Overview</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/SearchEngine"> Search Engine</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Constants_Database">Constants Database</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software/Simulation">Simulation </A><A href="https://2015.igem.org/Team:Technion_HS_Israel/LINK_TO_EXPLANATION">About the Database </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN><SPAN class="sdt_descr">We are a bunch of nerdy nerds</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Team">Members</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Attributions">Attributions and contributions</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/teamGallery">Gallery</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Practices</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/collaborations">Collaborations</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices">Practices</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/SearchEngine">Search Engine</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/modelingTutorials ">Modeling Tutorials </A></DIV></LI></UL></DIV><DIV id="whiteBox" style="width: 80%; background-color: white;  padding: 25px; height: auto; margin-left: auto;
    margin-right: auto; position: absolute; top: 285px; left: 10%; right: 10%; z-index:-1;"><H1><FONT color="#008080">Model Overview</FONT></H1><H2>1. Abstract</H2><DIV class="Unindented"><P>In modeling, we obtain mathematical relationships between quantities in the system. Then, one can easily obtain the initial conditions that would lead to the desired outcome. The main variation in the inital condition is variation in the initial concentration of the inducer (AHL - Acyl-Homoserine-lactone), which can (mainly) vary the total bacteria living time. Because of the complexity of our model, we use an ODE (Ordinary Differential Equations) numerical solver we’ve written in matlab for obtaining final and graphical results. We present our system briefly and reactions, and then we compare the model and the experimental results for validation and conclusions.</P></DIV><H2>2.Introduction</H2><DIV class="Unindented"><P>
Our model has 2 main aspects: 1. Gene expression; 2. Plasmid loss. For the first aspect, we obtained the equations by applying enzyme kinetics and mass action on our reactions. For the second aspect, we obtain the equations from a typcial bacterial population growth and plasmid loss models.</P></DIV><H3 class="Subsection">
2.1. Why Do We Need a Model?
</H3><DIV class="Unindented"><P>
Our model is an integral part of our biological system. In order to use our kill switch properly, one needs to know exactly which initial conditions match the expected result. Using our model, one can easily obtain the initial concentrations to put in the kill switch activate it for the appropriate time. </P></DIV><DIV class="Indented"><P>
The plasmid loss aspect of our model is an additional model that takes into account the fact that plasmid loss can occur and cause unexpected results. For example, Satellite Colonies (see “Plasmid Loss” page for more details), which can hurt the functioning of the kill switch.</P></DIV><H3 class="Subsection">
2.2. Methods
</H3><DIV class="Unindented"><P>
In this model, we described all the reactions and processes by ODEs (Ordinary Differential Equations). This method is suitable for describing dynamic systems and for easy simulation. This is the common procedure for describing dynamic systems in science, especially in biology. It enables predicting the behaviour of very complex dynamic systems, as long as we can describe how the system change at any given time. Our system is for sure complex and dynamic, so we use a set of ODE to describe it.</P></DIV><DIV class="Indented"><P>
In order to achive a quantitive description of the reactions and processes, we use mainly the Law of Mass Action. This law, or principle, states that the rate of a reaction is proportional to the product of the masses (hence the name) or concentrations of the reactant. This law holds for system in a steady state and since we assume that are system is in a quasi steady state, i.e. the changes in it are relatively slow, we can use it. There are a few reaction for which we use other kinetic laws.</P></DIV><H3 class="Subsection">
2.3. Our System
</H3><H4>
2.3.1. In General
</H4><DIV class="Unindented"><P>
Our system consists of an inducer (AHL from the Homoserine-Lactones group) and the genetic circuit that is inserted in a plasmid. In the circuit, there is a “death protein” (in this case, ccdB) which is responsible for the actual death. The precence of the inducer in the bacteria represses the expression of the death protein. Therefore, after the inducer is degraded, there is nothing that can repress the expression of the death gene, and the bacteria die.</P></DIV><H4> 
2.3.2. In Detail
</H4><DIV class="Unindented"><P>
The AHL (inducer) binds to the protein LuxR and they form a complex. This complex binds to the pLux promoter and activates it. Then, the tetR protein is expressed, binds to the pTet promoter and deactivates it. As a result, the ccdb (“death protein”) will not be expressed. When the AHL is fully degreaded, there will be not AHL-LuxR complex to activate the pLux promoter, thus no tetR will be expressed, thus ccdB will be expressed, which will kill the bacteria.</P></DIV><H4>
2.3.3. Note About Notations
</H4><DIV class="Unindented"><P>
In this page we’ll use the (relatively) full  names for the substances and complexes. In the equations page we’ll use abbreviations which will be explained there. Please note that we sometime call the TetR repressor TRLV (TetR with tag LVA). It’s so because during the process of modelling our system we didn’t knew for sure which version of TetR we’ll finally use. In other words, for any purposes of the modelling TRLV and TetR are equivalent and are used interchangeably in this wiki. As you've seen in the description page, in the end we used TetR in our biological circuit.</P></DIV><DIV class="Indented"><P>
Furthermore, the notations of in, out and sum are used in this page without much explanation, although they aren’t crucial for understanding our model. Thorough explanation of them available in the Documents section</P></DIV><H3 class="Subsection">
3. Reactions
</H3><P class="MsoListParagraphCxSpFirst" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">i.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">                  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">AHL self-degradation. </SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each molecule of AHL has a certain probability to degrade,
hence the corresponding change rate in the amount of the AHL is proportional to
the amount of AHL in all the cells. The coefficient is noted by C2 for cell
internal AHL and C2' for cell external AHL. </SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">ii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">                </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Diffusion of AHL</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Simple passive diffusion</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Will be explained in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:
Change in external AHL concentration:
Change in total amount inside of AHL inside all the cells:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">iii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">             </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">AHL degradation by AiiA</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Michaelis Menten</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: AiiA is an enzyme, and simple Mass Action doesn't work well
for enzymatic reactions. The reason for it is the fact that the enzyme and the
substrate form a complex, which is then converted to a product and the original
enzyme. Therefore, two mass actions are required to describe this process, but
under quasi-steady-state assumption we can derive a single equation, which is
the Michaelis Menten law. It has two parameters, the maximal reaction rate and
the turnover number.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">iv.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">             </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Pairing of AHL and LuxR into AHL-LuxR complex</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: The chance of a molecule of AHL to meet a molecule of LuxR
is proportional to both the concentration of AHL and LuxR (the more AHL you
have, the higher the chance for reaction between AHL and LuxR). We get that the
reaction rate is proportional to the product of the concentrations of AHL and
LuxR.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">v.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">                </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Disassociation of the AHL-LuxR complex to its components</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each AHL-LuxR complex has a certain probability to
disassociate, hence the corresponding change rate in the amount of the AHL is
proportional to the amount of AHL-LuxR. The coefficient is denoted by C4.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">vi.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">             </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Pairing of 2 AHL-LuxR complexes into the dimer (AHL-LuxR)<SUB>2</SUB></SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: The chance of a molecule of AHL-LuxR complex to meet another
one is proportional, again, to the product of their concentrations, (which this
time are equal and we get [AHL-LuxR]^2).</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">vii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">           </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Disassociation of (AHL-LuxR)<SUB>2</SUB> to its components</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each (AHL-LuxR)<SUB>2</SUB> dimer has a certain probability
to disassociate, hence the corresponding change rate in the amount of the AHL
is proportional to the total amount of AHL-LuxR in the cells. The coefficient
is denoted by C6.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">viii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">(AHL-LuxR)<SUB>2</SUB> binds to the pLuxR promoter</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: The activation rate is proportional to the product of the
concentrations of the dimer and the number of plasmid plasmids with pLuxR
promoters. It will be explained later in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">ix.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">              </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">(AHL-LuxR)<SUB>2 </SUB>unbinds from the pLuxR promoter</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each activated promoter has a certain probability to
deactivate and to release its (AHL-LuxR)<SUB>2</SUB>, and therefore the rate of
this process is proportional to the number of activated promoters. It will be
explained later in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">x.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">                </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Transcription of RNA<SUB>TRLV</SUB> by pLuxR promoter without the complex</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each inactivated promoter transcripts mRNA in a certain rate.
This rate is called leakiness. We multiply it by the number of inactivated LuxR
promoters to get the total rate. It will be explained further in the processes
section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xi.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">              </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Transcription of RNA<SUB>TRLV</SUB> by pLuxR promoter with the complex</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each activated promoter transcripts mRNA in a certain rate.
We multiply it by the number of activated LuxR promoters to get the total rate.
It will be explained further in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">           </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Translation of TRLV from RNA<SUB>TRLV</SUB></SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: mRNA of TetR translates to the protein TetR in a certain
rate. We multiply it by the concentration of RNA to get the total rate. It will
be explained further in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xiii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">         </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">TRLV self-degradation</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each molecule of TRLV has a certain probability to degrade,
hence the corresponding change rate in the amount of the TRLV is proportional
to the amount of TRLV all the cells.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xiv.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">         </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">TRLV binds to the pTetR repressor </SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: The activation rate is proportional to the product of the
concentrations of the dimer and the number of plasmid plasmids with pTetR promoters.
It will be explained later in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xv.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">           </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">TRLV unbinds from the pLuxR repressor</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each activated promoter has a certain probability to
deactivate and to release its TRLV, and therefore the rate of this process is proportional
to the number of activated promoters. It will be explained later in the
processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xvi.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">         </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Transcription of RNA<SUB>ccdB</SUB> by ptetR promoter without the TRLV</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: For each inactivated repressor there is transcription to mRNA
in a certain rate. We multiply it by the number of inactivated ptetR promoters
to get the total rate. It will be explained further in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xvii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">      </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Transcription of RNA<SUB>ccdB</SUB> by ptetR promoter with the TRLV</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: For each activated repressor there is still transcription to
mRNA in a certain rate. We multiply it by the number of activated ptetR promoters
to get the total rate. It will be explained further in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xviii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Translation of ccdB from RNA<SUB>ccdB</SUB></SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: mRNA of ccdb translates to the protein ccdb in a certain
rate. We multiply it by the concentration of RNA to get the total rate. It will
be explained further in the processes section.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xix.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">         </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">ccdB self-degradation</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each molecule of ccdB has a certain probability to degrade,
hence the corresponding change rate in the amount of the ccdB is proportional
to the amount of ccdB all the cells.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xx.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">            </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">producing of the desired protein x by the pCONST promoter</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass Action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each plasmid with our circuit produces the desired enzyme, X,
in a certain amount.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xxi.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">         </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">producing of LuxR by the pCONST promoter</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass Action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each plasmid with our circuit produces LuxR in a certain amount.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xxii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">       </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">producing of AiiA by the pCONST promoter</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Law: Mass Action</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Explanation: Each plasmid with our circuit produces AiiA in a certain amount.</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:54.0pt;text-align:left;text-indent:-36.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%">xxiii.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">plasmid loss</SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">We model the effect of plasmid loss on the system. It affects the
equations by having both bacteria with plasmids (N+) and without plasmid (N-).</SPAN></P><P class="MsoListParagraphCxSpLast" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Further explanation is <A href="https://2015.igem.org/Team:Technion_HS_Israel/PlasmidLoss">here</A>. </SPAN></P><P class="MsoListParagraphCxSpMiddle" dir="LTR" style="margin-top:0cm;margin-right:
0cm;margin-bottom:10.0pt;margin-left:72.0pt;text-align:left;text-indent:-18.0pt;
direction:ltr;unicode-bidi:embed"><SPAN style="font-size:16.0pt;line-height:
115%;font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN style="font-size:16.0pt;line-height:
115%">Results:</SPAN></P><H2>
4. Processes
</H2><H3 class="Subsection">
4.1. AHL Diffusion
</H3><DIV class="Unindented"><P>
Our system is based on controlling the initial amount of AHL inside the cells in order to control the life span of the bacteria. However, we can’t put the AHL directly inside the cells. We put it in the test tube together with the bacteria. Therefore, it’s important to model how this process, of AHL entering the cell, accure and how it affects our system. It is part of seeing our system as an integrated complex, whose parts interect with each other and affect the result: the external AHL diffuse inside the cells, inside the cells the AHL is degraded by the AiiA, and as AHL is decreasing inside the cells, the external AHL diffuses accordingly into the cells in order to maintain equilibrium.</P></DIV><DIV class="Indented"><P>
As AHL diffuse rather quickly through the membrane of E. Coli, we use a rather simplified model. We assume that each seperated part of our system (external environment, inside each cell, etc) is homogenous, which is quite true as AHL diffuses quickly, and it’s suffeciant to our needs.</P></DIV><DIV class="Indented"><P>
We base our diffusion model on <A class="URL" href="http://www.tiem.utk.edu/~gross/bioed/webmodules/diffusion.htm">this document</A>. Essentially, it says that the rate of the diffusion is proportional to the concetration gradient.</P></DIV><H3 class="Subsection">
4.2. Transcription and Translation
</H3><DIV class="Unindented"><P>
We have two proteins whose production is a part of our biological circuit: the TetR and the ccdb (or the RFP in the experiments). In our model, each of this processes is described using 4 equations. The first two describe the number of plasmids with an activated promoter (or repressor, depends on the processes) using regular Mass Action. In the third we assume that RNA is transcripted in a certain constant rate for each actiivated binding site and the same for unactivated, and get an expression for the change in the concentration of RNA strands. In the last one we desscribe the translation rate of the protein, using the Mass Action law, as a function of the RNA. We assume that the translation rate is proportional to the concentration of the RNA. </P></DIV><H3 class="Subsection">
4.3. Plasmid Loss
</H3><DIV class="Unindented"><P>
Read the full explanation and description of the plasmid loss model <A class="URL" href="https://2015.igem.org/Team:Technion_HS_Israel/PlasmidLoss">here</A>.</P></DIV><H2 class="Section">
5. Documents
</H2><DIV class="Unindented"><P>
During our work, we’ve made a documents describing different aspects and parts of our model. We put them here in hope that whomever want to dive into our model will be able to do so and will find these documents interesting.</P></DIV><UL><LI><A class="URL" href="https://static.igem.org/mediawiki/2015/8/8c/Technion_HS_2015_Scopes_ofModelling.pdf">Scopes in biological systems modelling.</A></LI><LI><A class="URL" href="https://static.igem.org/mediawiki/2015/9/9b/Simulation.pdf">Solving dynamic systems numerically</A></LI><LI>
Partial modelling notebook: <A class="URL" href="https://static.igem.org/mediawiki/2015/9/9c/Technion_HS_2015_Summary_Modelling1.pdf">1</A><A class="URL" href="https://static.igem.org/mediawiki/2015/2/2c/Technion_HS_2015_Summary_Modelling2.pdf">2</A><A class="URL" href="https://static.igem.org/mediawiki/2015/8/84/Technion_HS_2015_Summary_Modelling3.pdf">3</A>.
</LI></UL><H2 class="Section">
6. Simulation
</H2><DIV class="Unindented"><P>
We simulated our system a few times using our numerical solver. Description and documentation of our solver can be found <A class="URL" href="https://2015.igem.org/Team:Technion_HS_Israel/Software/Simulation">here</A>. The results we’ve got are in the <A class="URL" href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Results">Modelling Results page</A>.</P><H3>Parameters</H3><P>In order to describe our biological system, we need the right parameters. We looked up in literature and in iGEM projects from previous years (using our <A href="https://2015.igem.org/Team:Technion_HS_Israel/Constants_Database"> Constant Database</A>!) and found almost all the 40 required parameters and constants, and estimated the rest. Check our parameter table <A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Parameters">here</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>