Team:Technion HS Israel/Modelling
Technion 2015 HS Team's Wiki
Home
Medal Fulfillment Home
Project More about our project
Description Results Parts
Documents
Experiments Notebook safety Protocols
Modelling We take our math seriously
Model Overview Plasmid Loss Equations Parameters Results Documents
Software
Software Overview Search Engine Constants Database Simulation About the Database
Team We are a bunch of nerdy nerds
Members Attributions and contributions Gallery
Practices
Collaborations Practices Search Engine Modeling Tutorials
Model Overview
1. Abstract
In modeling, we obtain mathematical relationships between quantities in the system. Then, one can easily obtain the initial conditions that would lead to the desired outcome. The main variation in the inital condition is variation in the initial concentration of the inducer (AHL - Acyl-Homoserine-lactone), which can (mainly) vary the total bacteria living time. Because of the complexity of our model, we use an ODE (Ordinary Differential Equations) numerical solver we’ve written in matlab for obtaining final and graphical results. We present our system briefly and reactions, and then we compare the model and the experimental results for validation and conclusions.
2.Introduction
Our model has 2 main aspects: 1. Gene expression; 2. Plasmid loss. For the first aspect, we obtained the equations by applying enzyme kinetics and mass action on our reactions. For the second aspect, we obtain the equations from a typcial bacterial population growth and plasmid loss models.
2.1. Why Do We Need a Model?
Our model is an integral part of our biological system. In order to use our kill switch properly, one needs to know exactly which initial conditions match the expected result. Using our model, one can easily obtain the initial concentrations to put in the kill switch activate it for the appropriate time.
The plasmid loss aspect of our model is an additional model that takes into account the fact that plasmid loss can occur and cause unexpected results. For example, Satellite Colonies (see “Plasmid Loss” page for more details), which can hurt the functioning of the kill switch.
2.2. Methods
In this model, we described all the reactions and processes by ODEs (Ordinary Differential Equations). This method is suitable for describing dynamic systems and for easy simulation. This is the common procedure for describing dynamic systems in science, especially in biology. It enables predicting the behaviour of very complex dynamic systems, as long as we can describe how the system change at any given time. Our system is for sure complex and dynamic, so we use a set of ODE to describe it.
In order to achive a quantitive description of the reactions and processes, we use mainly the Law of Mass Action. This law, or principle, states that the rate of a reaction is proportional to the product of the masses (hence the name) or concentrations of the reactant. This law holds for system in a steady state and since we assume that are system is in a quasi steady state, i.e. the changes in it are relatively slow, we can use it. There are a few reaction for which we use other kinetic laws.
2.3. Our System
2.3.1. In General
Our system consists of an inducer (AHL from the Homoserine-Lactones group) and the genetic circuit that is inserted in a plasmid. In the circuit, there is a “death protein” (in this case, ccdB) which is responsible for the actual death. The precence of the inducer in the bacteria represses the expression of the death protein. Therefore, after the inducer is degraded, there is nothing that can repress the expression of the death gene, and the bacteria die.
2.3.2. In Detail
The AHL (inducer) binds to the protein LuxR and they form a complex. This complex binds to the pLux promoter and activates it. Then, the tetR protein is expressed, binds to the pTet promoter and deactivates it. As a result, the ccdb (“death protein”) will not be expressed. When the AHL is fully degreaded, there will be not AHL-LuxR complex to activate the pLux promoter, thus no tetR will be expressed, thus ccdB will be expressed, which will kill the bacteria.
2.3.3. Note About Notations
In this page we’ll use the (relatively) full  names for the substances and complexes. In the equations page we’ll use abbreviations which will be explained there. Please note that we sometime call the TetR repressor TRLV (TetR with tag LVA). It’s so because during the process of modelling our system we didn’t knew for sure which version of TetR we’ll finally use. In other words, for any purposes of the modelling TRLV and TetR are equivalent and are used interchangeably in this wiki. As you've seen in the description page, in the end we used TetR in our biological circuit.
Furthermore, the notations of in, out and sum are used in this page without much explanation, although they aren’t crucial for understanding our model. Thorough explanation of them available in the Documents section
3. Reactions
i.                   AHL self-degradation.
o   Law: Mass action
o   Explanation: Each molecule of AHL has a certain probability to degrade, hence the corresponding change rate in the amount of the AHL is proportional to the amount of AHL in all the cells. The coefficient is noted by C2 for cell internal AHL and C2' for cell external AHL.
o   Results:
ii.                 Diffusion of AHL
o   Law: Simple passive diffusion
o   Explanation: Will be explained in the processes section.
o   Results:
Change in external AHL concentration:
Change in total amount inside of AHL inside all the cells:
iii.              AHL degradation by AiiA
o   Law: Michaelis Menten
o   Explanation: AiiA is an enzyme, and simple Mass Action doesn't work well for enzymatic reactions. The reason for it is the fact that the enzyme and the substrate form a complex, which is then converted to a product and the original enzyme. Therefore, two mass actions are required to describe this process, but under quasi-steady-state assumption we can derive a single equation, which is the Michaelis Menten law. It has two parameters, the maximal reaction rate and the turnover number.
o   Results:
iv.              Pairing of AHL and LuxR into AHL-LuxR complex
o   Law: Mass action
o   Explanation: The chance of a molecule of AHL to meet a molecule of LuxR is proportional to both the concentration of AHL and LuxR (the more AHL you have, the higher the chance for reaction between AHL and LuxR). We get that the reaction rate is proportional to the product of the concentrations of AHL and LuxR.
o   Results:
v.                 Disassociation of the AHL-LuxR complex to its components
o   Law: Mass action
o   Explanation: Each AHL-LuxR complex has a certain probability to disassociate, hence the corresponding change rate in the amount of the AHL is proportional to the amount of AHL-LuxR. The coefficient is denoted by C4.
o   Results:
vi.              Pairing of 2 AHL-LuxR complexes into the dimer (AHL-LuxR)2
o   Law: Mass action
o   Explanation: The chance of a molecule of AHL-LuxR complex to meet another one is proportional, again, to the product of their concentrations, (which this time are equal and we get [AHL-LuxR]^2).
o   Results:
vii.            Disassociation of (AHL-LuxR)2 to its components
o   Law: Mass action
o   Explanation: Each (AHL-LuxR)2 dimer has a certain probability to disassociate, hence the corresponding change rate in the amount of the AHL is proportional to the total amount of AHL-LuxR in the cells. The coefficient is denoted by C6.
o   Results:
viii.         (AHL-LuxR)2 binds to the pLuxR promoter
o   Law: Mass action
o   Explanation: The activation rate is proportional to the product of the concentrations of the dimer and the number of plasmid plasmids with pLuxR promoters. It will be explained later in the processes section.
o   Results:
ix.               (AHL-LuxR)2 unbinds from the pLuxR promoter
o   Law: Mass action
o   Explanation: Each activated promoter has a certain probability to deactivate and to release its (AHL-LuxR)2, and therefore the rate of this process is proportional to the number of activated promoters. It will be explained later in the processes section.
o   Results:
x.                 Transcription of RNATRLV by pLuxR promoter without the complex
o   Law: Mass action
o   Explanation: Each inactivated promoter transcripts mRNA in a certain rate. This rate is called leakiness. We multiply it by the number of inactivated LuxR promoters to get the total rate. It will be explained further in the processes section.
o   Results:
xi.               Transcription of RNATRLV by pLuxR promoter with the complex
o   Law: Mass action
o   Explanation: Each activated promoter transcripts mRNA in a certain rate. We multiply it by the number of activated LuxR promoters to get the total rate. It will be explained further in the processes section.
o   Results:
xii.            Translation of TRLV from RNATRLV
o   Law: Mass action
o   Explanation: mRNA of TetR translates to the protein TetR in a certain rate. We multiply it by the concentration of RNA to get the total rate. It will be explained further in the processes section.
o   Results:
xiii.          TRLV self-degradation
o   Law: Mass action
o   Explanation: Each molecule of TRLV has a certain probability to degrade, hence the corresponding change rate in the amount of the TRLV is proportional to the amount of TRLV all the cells.
o   Results:
xiv.          TRLV binds to the pTetR repressor
o   Law: Mass action
o   Explanation: The activation rate is proportional to the product of the concentrations of the dimer and the number of plasmid plasmids with pTetR promoters. It will be explained later in the processes section.
o   Results:
xv.            TRLV unbinds from the pLuxR repressor
o   Law: Mass action
o   Explanation: Each activated promoter has a certain probability to deactivate and to release its TRLV, and therefore the rate of this process is proportional to the number of activated promoters. It will be explained later in the processes section.
o   Results:
xvi.          Transcription of RNAccdB by ptetR promoter without the TRLV
o   Law: Mass action
o   Explanation: For each inactivated repressor there is transcription to mRNA in a certain rate. We multiply it by the number of inactivated ptetR promoters to get the total rate. It will be explained further in the processes section.
o   Results:
xvii.       Transcription of RNAccdB by ptetR promoter with the TRLV
o   Law: Mass action
o   Explanation: For each activated repressor there is still transcription to mRNA in a certain rate. We multiply it by the number of activated ptetR promoters to get the total rate. It will be explained further in the processes section.
o   Results:
xviii.     Translation of ccdB from RNAccdB
o   Law: Mass action
o   Explanation: mRNA of ccdb translates to the protein ccdb in a certain rate. We multiply it by the concentration of RNA to get the total rate. It will be explained further in the processes section.
o   Results:
xix.          ccdB self-degradation
o   Law: Mass action
o   Explanation: Each molecule of ccdB has a certain probability to degrade, hence the corresponding change rate in the amount of the ccdB is proportional to the amount of ccdB all the cells.
o   Results:
xx.             producing of the desired protein x by the pCONST promoter
o   Law: Mass Action
o   Explanation: Each plasmid with our circuit produces the desired enzyme, X, in a certain amount.
o   Results:
xxi.          producing of LuxR by the pCONST promoter
o   Law: Mass Action
o   Explanation: Each plasmid with our circuit produces LuxR in a certain amount.
o   Results:
xxii.        producing of AiiA by the pCONST promoter
o   Law: Mass Action
o   Explanation: Each plasmid with our circuit produces AiiA in a certain amount.
o   Results:
xxiii.     plasmid loss
o   We model the effect of plasmid loss on the system. It affects the equations by having both bacteria with plasmids (N+) and without plasmid (N-).
o   Further explanation is here .
o   Results:
4. Processes
4.1. AHL Diffusion
Our system is based on controlling the initial amount of AHL inside the cells in order to control the life span of the bacteria. However, we can’t put the AHL directly inside the cells. We put it in the test tube together with the bacteria. Therefore, it’s important to model how this process, of AHL entering the cell, accure and how it affects our system. It is part of seeing our system as an integrated complex, whose parts interect with each other and affect the result: the external AHL diffuse inside the cells, inside the cells the AHL is degraded by the AiiA, and as AHL is decreasing inside the cells, the external AHL diffuses accordingly into the cells in order to maintain equilibrium.
As AHL diffuse rather quickly through the membrane of E. Coli, we use a rather simplified model. We assume that each seperated part of our system (external environment, inside each cell, etc) is homogenous, which is quite true as AHL diffuses quickly, and it’s suffeciant to our needs.
We base our diffusion model on this document . Essentially, it says that the rate of the diffusion is proportional to the concetration gradient.
4.2. Transcription and Translation
We have two proteins whose production is a part of our biological circuit: the TetR and the ccdb (or the RFP in the experiments). In our model, each of this processes is described using 4 equations. The first two describe the number of plasmids with an activated promoter (or repressor, depends on the processes) using regular Mass Action. In the third we assume that RNA is transcripted in a certain constant rate for each actiivated binding site and the same for unactivated, and get an expression for the change in the concentration of RNA strands. In the last one we desscribe the translation rate of the protein, using the Mass Action law, as a function of the RNA. We assume that the translation rate is proportional to the concentration of the RNA.
4.3. Plasmid Loss
Read the full explanation and description of the plasmid loss model here .
5. Documents
During our work, we’ve made a documents describing different aspects and parts of our model. We put them here in hope that whomever want to dive into our model will be able to do so and will find these documents interesting.
Scopes in biological systems modelling.
Solving dynamic systems numerically
Partial modelling notebook: 1 2 3 .
6. Simulation
We simulated our system a few times using our numerical solver. Description and documentation of our solver can be found here . The results we’ve got are in the Modelling Results page .
Parameters
In order to describe our biological system, we need the right parameters. We looked up in literature and in iGEM projects from previous years (using our Constant Database !) and found almost all the 40 required parameters and constants, and estimated the rest. Check our parameter table here
