<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_HS_Israel_Constants_Database skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion HS Israel/Constants Database</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE> Technion 2015 HS Team's Wiki</TITLE></P><DIV class="content" style="width:100%;top:150px;position:absolute"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/MedalFulfillment"> Medal Fulfillment</A><A href="https://2015.igem.org/Team:Technion_HS_Israel"> Home</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Description"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN><SPAN class="sdt_descr">More about our project</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Description">Description</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Project/Results">Results</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Parts">Parts</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Documents</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Experiments">Experiments</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Notebook">Notebook</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/safety">safety</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Protocols">Protocols</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modelling</SPAN><SPAN class="sdt_descr">We take our math seriously</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling">Model Overview</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/PlasmidLoss">Plasmid Loss</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Equations">Equations</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Parameters">Parameters</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Results">Results</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Modelling/Documents">Documents</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Software</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software">Software Overview</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/SearchEngine"> Search Engine</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Constants_Database">Constants Database</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Software/Simulation">Simulation </A><A href="https://2015.igem.org/Team:Technion_HS_Israel/LINK_TO_EXPLANATION">About the Database </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN><SPAN class="sdt_descr">We are a bunch of nerdy nerds</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Team">Members</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Attributions">Attributions and contributions</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/teamGallery">Gallery</A></DIV></LI><LI><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Practices</SPAN></SPAN></A><DIV class="sdt_box"><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/collaborations">Collaborations</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices">Practices</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/SearchEngine">Search Engine</A><A href="https://2015.igem.org/Team:Technion_HS_Israel/Practices/modelingTutorials ">Modeling Tutorials </A></DIV></LI></UL></DIV><DIV id="whiteBox" style="width: 80%; background-color: hsla(0,0%,85%,0.91);  padding: 25px; height: auto; margin-left: auto;
    margin-right: auto; position: absolute; top: 285px; left: 10%; right: 10%; z-index:-1;"><H1><FONT color="#008080"> The Constants Database</FONT></H1><DIV style="position:absolute;right:20px;font-size:30px;top:15;font-family:arial"><A href="LINK_TO_EXPLANATION">Explanation and Documentation</A></DIV><DIV class="datagrid"><TABLE><THEAD><TR class="tableizer-firstrow"><TH style="width:20px">name</TH><TH>description</TH><TH>value</TH><TH>unit</TH><TH>Source</TH><TH>url</TH><TH>team</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>k1A</TD><TD>Transcription Rate of Lac I gene</TD><TD>21</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>2</TD><TD>k1C</TD><TD>Transcription Rate of E7 + Imm gene</TD><TD>2.470588</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>3</TD><TD>d1A</TD><TD>Degradation Rate of Lac I mRNA</TD><TD>0.76246</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>4</TD><TD>d1C</TD><TD>Transcription Rate of E7 + Imm mRNA</TD><TD>0.0897</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>5</TD><TD>k2A</TD><TD>Translation Rate of Lac I mRNA</TD><TD>36</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>6</TD><TD>k2C</TD><TD>Translation Rate of E7 + Imm mRNA</TD><TD>4.23539</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>7</TD><TD>d2A,d2C</TD><TD>Protein Degradation Rate</TD><TD>0.03465</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>8</TD><TD>nC</TD><TD>Hill coefficeint for E7 + Imm</TD><TD>1</TD><TD> </TD><TD>This is obtained on the assumption that one Repressor Protein binds to one Lactose molecule complex</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>9</TD><TD>KC</TD><TD>Dissociation Constant for E7 + Imm</TD><TD>0.8</TD><TD> </TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">[1]</A></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>10</TD><TD>aC</TD><TD>Constitutive Portion for E7 + Imm</TD><TD>0.5</TD><TD> </TD><TD>Estimate since a is between 0 and 1 Implication that Lactose may not be  a very strongly Regulated Promoter</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>11</TD><TD>k3AB</TD><TD>Complex Formation Rate Between  Lac I Repressor and Lactose</TD><TD>1</TD><TD> </TD><TD>Estimate. This is based on the assumption  that the complex formation is only  dependent on the concentrations of  Lac I repressor and Lactose</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>12</TD><TD>kPF</TD><TD>Phosphorylation Rate of Ai-2</TD><TD>1</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>13</TD><TD>kPB</TD><TD>Dephosphorylation Rate of Ai-2</TD><TD>1</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>14</TD><TD>k1C</TD><TD>Transcription Rate of LsrR gene</TD><TD>4.402517</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>15</TD><TD>k1D</TD><TD>Transcription Rate of SupD gene</TD><TD>46.667</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>16</TD><TD>k1E</TD><TD>Transcription Rate of t7pTag gene</TD><TD>1.5556</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>17</TD><TD>k1F</TD><TD>Transcription Rate of Lysis gene</TD><TD>28</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>18</TD><TD>d1C</TD><TD>Degradation Rate of LsrR mRNA</TD><TD>0.159845</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>19</TD><TD>d1D</TD><TD>Degradation Rate of SupD mRNA</TD><TD>1.694359</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>20</TD><TD>d1E</TD><TD>Degradation Rate of t7pTag mRNA</TD><TD>0.056478</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>21</TD><TD>d1F</TD><TD>Degradation Rate of Lysis mRNA</TD><TD>1.0166</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>22</TD><TD>k2C</TD><TD>Translation Rate of LsrR Protein</TD><TD>7.54716</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>23</TD><TD>k2E</TD><TD>Translation Rate of t7 pTag Protein</TD><TD>2.6667</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>24</TD><TD>k2F</TD><TD>Translation Rate of Lysis Protein</TD><TD>48</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>25</TD><TD>d2C,d2E,d2F</TD><TD>Protein Degradation Rate</TD><TD>0.03465</TD><TD> </TD><TD>Made using Earlier assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>26</TD><TD>nD</TD><TD>Hill coefficeint for SupD</TD><TD>50</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisnD" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>27</TD><TD>nE</TD><TD>Hill coefficeint for t7 pTag</TD><TD>1</TD><TD> </TD><TD>Estimate. It is assuming that  one molecule of iron ion is required  to activate the production of one t7mRNA</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>28</TD><TD>nF</TD><TD>Hill coefficeint for Lysis</TD><TD>1</TD><TD> </TD><TD>Estimate. It is assumed that  one molecule of t7 is required  for activation of one Lysis mRNA</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>29</TD><TD>KD</TD><TD>Dissociation Constant for SupD</TD><TD>15</TD><TD> </TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">[2]</A></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>30</TD><TD>KE</TD><TD>Dissociation Constant for t7 pTag</TD><TD>1</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisKE" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>31</TD><TD>KF</TD><TD>Dissociation Constant for Lysis</TD><TD>1</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisKF" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>32</TD><TD>aD</TD><TD>Constitutive Portion for SupD</TD><TD>0.01</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaD" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>33</TD><TD>aE</TD><TD>Constitutive Portion for t7 pTag</TD><TD>0.01</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaE" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>34</TD><TD>aF</TD><TD>Constitutive Portion for Lysis</TD><TD>0.0001</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaF" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>35</TD><TD>k3BC</TD><TD>Complex Formation Rate Between  LsrR Repressor and Ai-2-Phosphorylated</TD><TD>0.01</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/Lysisk3BC" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>36</TD><TD>k3DE</TD><TD>Complex Formation Rate Between  SupD tRNA and t7 pTag mRNA</TD><TD>0.000000001</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/Lysisk3DE" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>37</TD><TD>S</TD><TD>Amount of Protein Kinase</TD><TD>28.747</TD><TD> </TD><TD>From Earlier Assumptions</TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>38</TD><TD>kf</TD><TD>Forward Reaction rate of Complex</TD><TD>1.2</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/kF" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>39</TD><TD>kr</TD><TD>Reverse Reaction rate of Complex</TD><TD>0.5</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/kr" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>40</TD><TD>r</TD><TD>Rate of Logistic Growth</TD><TD>0.01</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/r_logistic" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>41</TD><TD>A &amp; B</TD><TD>Varying Capacity</TD><TD>1 &amp; 0.9</TD><TD> </TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/A_B" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD><TD><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A></TD><TD>NTU-Singapore</TD></TR><TR><TD>42</TD><TD><FONT size="3" face="Cambria">k<SUB>n</SUB></FONT></TD><TD>Population and culture</TD><TD><FONT size="3" face="Cambria">3.85e-3</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">[1] observed division rate</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>43</TD><TD><FONT size="3" face="Cambria">N<SUB>max</SUB></FONT></TD><TD>Population and culture; carrying capacity</TD><TD><FONT size="3" face="Cambria">1e8</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>44</TD><TD><FONT size="3" face="Cambria">V</FONT></TD><TD>Population and culture;</TD><TD><FONT size="3" face="Cambria">1e-11</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>45</TD><TD><FONT size="3" face="Cambria">k<SUB>dil</SUB></FONT></TD><TD>Population and culture; dilution rate</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>46</TD><TD><FONT size="3" face="Cambria">k<SUB>assoc</SUB></FONT></TD><TD>IP binding to receptor</TD><TD><FONT size="3" face="Cambria">5e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>47</TD><TD><FONT size="3" face="Cambria">k<SUB>dissoc</SUB></FONT></TD><TD>IP binding to receptor</TD><TD><FONT size="3" face="Cambria">4.55e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>48</TD><TD><FONT size="3" face="Cambria">k<SUB>s1</SUB></FONT></TD><TD>Synthesis of CRE1, YPD1, SKN7 species</TD><TD><FONT size="3" face="Cambria">6.16e-5</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>49</TD><TD><FONT size="3" face="Cambria">k<SUB>s2</SUB></FONT></TD><TD>Synthesis of CRE1, YPD1, SKN7 species</TD><TD><FONT size="3" face="Cambria">6.00e-4</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>50</TD><TD><FONT size="3" face="Cambria">k<SUB>s3</SUB></FONT></TD><TD>Synthesis of CRE1, YPD1, SKN7 species</TD><TD><FONT size="3" face="Cambria">2.46e-4</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>51</TD><TD><FONT size="3" face="Cambria">k<SUB>dimer</SUB></FONT></TD><TD>Synthesis of CRE1, YPD1, SKN7 species</TD><TD><FONT size="3" face="Cambria">20</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>52</TD><TD><FONT size="3" face="Cambria">k<SUB>p1</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">0.1</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>53</TD><TD><FONT size="3" face="Cambria">k<SUB>p2</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">1243</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>54</TD><TD><FONT size="3" face="Cambria">k<SUB>p3</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">56</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>55</TD><TD><FONT size="3" face="Cambria">k<SUB>p-3</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">4.8</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>56</TD><TD><FONT size="3" face="Cambria">k<SUB>dp1</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">5.33e-2</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>57</TD><TD><FONT size="3" face="Cambria">k<SUB>dp3</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">2.89e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>58</TD><TD><FONT size="3" face="Cambria">k<SUB>dp3</SUB></FONT></TD><TD>Phosphorylation/Dephosphorylation reactions;</TD><TD><FONT size="3" face="Cambria">4.80e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>59</TD><TD><FONT size="3" face="Cambria">k<SUB>d1</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">5.33e-2</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>60</TD><TD><FONT size="3" face="Cambria">k<SUB>d2</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">2.89e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>61</TD><TD><FONT size="3" face="Cambria">k<SUB>d3</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">4.80e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>62</TD><TD><FONT size="3" face="Cambria">k<SUB>dip</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">5e-4</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>63</TD><TD><FONT size="3" face="Cambria">k<SUB>dgfp</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">5.77e-3</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[1]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>64</TD><TD><FONT size="3" face="Cambria">k<SUB>dtet</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">3.85e-3</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">Cell division rate (assumed stable)</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>65</TD><TD><FONT size="3" face="Cambria">k<SUB>dckx</SUB></FONT></TD><TD>Decay constants;</TD><TD><FONT size="3" face="Cambria">&lt;=3.85e-3</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">Cell division rate, or lower if diffusion is significant.</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>66</TD><TD><FONT size="3" face="Cambria">k<SUB>bgfp</SUB>, k<SUB>btet</SUB>, k<SUB>bckx</SUB></FONT></TD><TD>Basal expression;</TD><TD><FONT size="3" face="Cambria">6e-6</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">adapted from [1]</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>67</TD><TD><FONT size="3" face="Cambria">k<SUB>tdim</SUB></FONT></TD><TD>Basal expression;</TD><TD><FONT size="3" face="Cambria">1e3</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">arbitrary</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>68</TD><TD><FONT size="3" face="Cambria">k<SUB>gfp</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">6e-3</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">adapted from [1]</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>69</TD><TD><FONT size="3" face="Cambria">k<SUB>tet</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>70</TD><TD><FONT size="3" face="Cambria">k<SUB>ckx</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>71</TD><TD><FONT size="3" face="Cambria">K<SUB>g,</SUB> K<SUB>t, </SUB> K<SUB>c</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria" align="center">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>72</TD><TD><FONT size="3" face="Cambria">α<SUB>g, </SUB>α<SUB>t</SUB>α<SUB>c</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>73</TD><TD><FONT size="3" face="Cambria">K<SUB>i</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">5e-6</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">K<SUB>ass</SUB> = 2e11 M<SUP>-1</SUP> [2], [3]</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>74</TD><TD><FONT size="3" face="Cambria">β</FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>75</TD><TD><FONT size="3" face="Cambria">k<SUB>ipt4</SUB></FONT></TD><TD>Induced expression;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>76</TD><TD><FONT size="3" face="Cambria">k<SUB>sip</SUB></FONT></TD><TD>IPT4 enzyme kinetics;</TD><TD><FONT size="3" face="Cambria">varied</FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">-</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>77</TD><TD><FONT size="2" face="Cambria"><B>CKX enzyme kinetics</B></FONT></TD><TD>IPT4 enzyme kinetics;</TD><TD> </TD><TD> </TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>78</TD><TD><FONT size="3" face="Cambria">K<SUB>m</SUB></FONT></TD><TD>IPT4 enzyme kinetics;</TD><TD><FONT size="3" face="Cambria">40 ?µM</FONT></TD><TD> </TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters"><FONT size="3" face="Cambria">[4]</FONT></A></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>79</TD><TD><FONT size="3" face="Cambria">k<SUB>cat</SUB></FONT></TD><TD>IPT4 enzyme kinetics;</TD><TD><FONT size="3" face="Cambria">0.5-1000 s<SUP>-1</SUP></FONT></TD><TD> </TD><TD><FONT size="3" face="Cambria">[5], [6]</FONT></TD><TD><A href="https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters">https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters</A></TD><TD>University of Ottawa</TD></TR><TR><TD>80</TD><TD>Length of e.coli</TD><TD>2μm</TD><TD> </TD><TD>&quot;<A href="" http:="">University of Alberta</A> Justification: Values come from the University of Alberta?€™s datasheet on MG1655, 
 produced to aid modelling. There is variability in size between strains -
 for instance, AW405 length varies between 1.5?±0.2μm. But University of 
 Alberta datasheet is specifically for MG1655.&quot;</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>81</TD><TD>Diameter e. coli</TD><TD> </TD><TD>0.8μm</TD><TD> </TD><TD><A href="http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi" class="external text" rel="nofollow">University of Alberta</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>82</TD><TD>Shape e. coli</TD><TD>Actually rod-like. A circle with r= 0.714μm will have equivalent surface area to rod-like.</TD><TD>Circle r =0.714μm</TD><TD> </TD><TD><A href="http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi" class="external text" rel="nofollow">University of Alberta</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>83</TD><TD>Mass e. coli</TD><TD>Given 1x10<SUP>-12</SUP>g for cell wet weight. Dividing this by gravity (=9.81) gives mass.</TD><TD>1.02x10<SUP>-13</SUP>g</TD><TD> </TD><TD><A href="http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi" class="external text" rel="nofollow">University of Alberta</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>84</TD><TD>Swimming Speed e. coli</TD><TD>&quot;
 &quot;</TD><TD>50μms<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi" class="external text" rel="nofollow">University of Alberta</A><A href="http://www3.interscience.wiley.com/cgi-bin/fulltext/71003069/PDFSTART" class="external text" rel="nofollow">A Method for Measuring Bacterial Chemotaxis Parameters in a Microcapillary</A> Justification: University Alberta's datasheet gives 50μms<SUP>-1</SUP>. However, Swimming speed is affected by: <UL><LI>Viscosity (as viscosity increases the speed increases to some maximum, then decreases as the viscosity increases further. E.coli (strain:KL227 of length: 1.0μm and diameter: 0.5μm) maximum speed occurs at viscosity 8cp. </LI><LI>Temperature </LI><LI>Culture medium </LI><LI>Vary strain to strain. </LI><LI>Experimental methods </LI></UL><P>Various papers give different speeds for E. coli (most papers provide information on AW405 with a speed of ~20μms<SUP>-1</SUP>). The speed itself is nearly uniform during the run. The wet lab may need to measure this experimentally as we are unaware of the conditions that the speed for MG1655 was obtained. Alberta's value is higher than other values, but this probably because MG1655 is a motile strain. </P></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>85</TD><TD>Aspartate concentration detected by E. coli</TD><TD>Related to Run Tumble Motion.</TD><TD>Over ~5 orders of magnitude, 10nM up to 10mM. Can detect changes of as little as ~0.1%</TD><TD> </TD><TD><A href="http://www.jbc.org/cgi/reprint/281/41/30512" class="external text" rel="nofollow">Competitive and Cooperative Interactions in Receptor Signalling Complexes</A> Justification: E. coli detect small changes in concentration of 0.1% via temporal comparisons (4s) over a large range ( 10<SUP>-8</SUP> to 10<SUP>-3</SUP> ). Most computer simulations of chemotaxis are based on experimentally determined rates and concentrations. As a result they predict that the minimum detectable concentration of Aspartate is at ~200 nM. Experiments performed by Segall et al. in 1986, exposed tethered E. coli cells to iontophoretically delivered quantities of chemoattractant. These experiments indicated that a change in receptor occupancy of as little as 1/600 could produce an detectable change in swimming behaviour. With a K<SUB>d</SUB> of 1 ?µM, this corresponds to a minimum detectable concentration of about 2 nM Aspartate. Wild type E. coli cells can detect &lt;10nM of Asp and respond to Asp concentrations of upto 1mM,(responding to over ~5 orders of magnitude). M)</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>86</TD><TD>Temporal comparison of chemotactic gradient</TD><TD>Related to Run Tumble Motion.</TD><TD>4 seconds</TD><TD> </TD><TD><A href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=387059&amp;blobtype=pdf" class="external text" rel="nofollow">Temporal comparisons in bacterial chemotaxis</A><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6TBN-4CVRC68-2&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=470c7fd73fb9ebf4ca43342f365e221f#sec5.1" class="external text" rel="nofollow">Quantitative analysis of signalling networks</A><A href="http://web.mit.edu/biophysics/papers/PNAS2003b.pdf" class="external text" rel="nofollow">Motility of Escherichia coli cells in clusters formed by chemotactic aggregation</A> Justification: The past second has positive weighting, the previous 3 seconds have negative weighting. E coli compares past and present concentrations by comparing the average occupancy of the receptors over the 4s. Models reflecting this system have been developed by Segall et al and Schnitzer, cells compare their average receptor occupancy between 4 and 1 s ago c<SUB>1-4</SUB> to the average receptor occupancy during the last second c<SUB>0-1</SUB> . Hence b= c<SUB>0-1</SUB> - c<SUB>1-4</SUB> . If b&gt;0, the cell reduces the tumbling rate to T<SUB>tumbling</SUB> from the ambient value T<SUB>0</SUB> , 1s<SUP>-1</SUP> e.g. b&gt;0 don't tumble. b&lt; 0, tumble at a rate of 1s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>87</TD><TD>Tumbling angle</TD><TD>Related to Run Tumble Motion.</TD><TD>Shape parameter 4 Scale parameter 18.32 Location parameter -4.6</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. coli anaylsed by three-dimensions</A><A href="http://bioinformatics.oxfordjournals.org/cgi/reprint/21/11/2714" class="external text" rel="nofollow">AgentCell: a digital single-cell assay for bacterial chemotaxis</A><A href="http://jb.asm.org/cgi/reprint/189/5/1756" class="external text" rel="nofollow">On Torque and tumbling in swimming Escherichia coli</A> Justification: The tumble angle appears not to be dependant on the concentration gradient of chemoattractants/repellents. Nor is there correlation between the length of the run and the change in direction. The program uses a gamma distribution that fits the data collected by Berg and Brown. Several groups though, have observed that the tumble angle is not noramlly distributed but suggest that non-normality was only due to the experimental methods used e.g. in the capillary tube. Tumbling can cause a change in direction when as few as one flagella moves out of the bundle.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>88</TD><TD>Tumble angle direction</TD><TD>Related to Run Tumble Motion.</TD><TD>Bidirectional</TD><TD> </TD><TD>Justification: Personal communication with Howard Berg. 'The direction is random, more or less, but there is a slight forward bias. It varies from tumble to tumble. The turn-angle distribution peaks at 68?° rather than 90?°. Tumbles turn out to be more complex than believed in 1972. Motors switch independently, and a tumble can occur if one or just a few motors change their directions of rotation. Tumbles are short, as judged by the tracking microscope, because they involve filament physics rather than motor physics: a transformation in polymorphic form, following motor reversal, from normal to semi-coiled. See Darnton, N.C., Turner, L., Rojevsky, S. and Berg, H.C. On torque and tumbling in swimming Escherichia coli, J. Bacteriol. 189, 1756-1764 (2007).'</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>89</TD><TD>Tumbling time</TD><TD>Related to Run Tumble Motion.</TD><TD>0.14?±0.19s</TD><TD> </TD><TD>&quot;<A href="" http:="">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A> Justification: Exponential distribution fitted (stated to be exponential
 by Berg and Brown) using only the mean tumble length (not STDEV).&quot;</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>90</TD><TD>Relationship between tumbling angle and time</TD><TD>Related to Run Tumble Motion.</TD><TD> </TD><TD> </TD><TD> </TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>91</TD><TD>Speed while Tumbling</TD><TD>Related to Run Tumble Motion.</TD><TD>0μm.s<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A> Justification: Berg and Brown noted that AW405 slowed/stopped while tumbling.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>92</TD><TD>Drift during run</TD><TD>Related to Run Tumble Motion.</TD><TD>23?±23?°</TD><TD> </TD><TD>&quot;<A href="" http:="">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A><A href="" http:="">Persistence of direction increases the drift velocity of run and tumble chemotaxis</A><A href="" http:="">Bray computer modelling</A> Justification: Drift was observed. It is what would be expected from
 rotational diffusion. (at 2.7cp at 32?C drift was 23?±23?°). Rotational 
 Brownian motion cause the cell to veer off course, so that in between 
 tumbles the probability density function f of the swimming direction e 
 evolves according to the Fokker-Planck equation. Drift velocity in steep
 gradient of attractant ~7 ?µm.s<SUP>-1</SUP>(Berg &amp; 
 Turner, 1990. Note our model did not include the effects of drift&quot;</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>93</TD><TD>Thrust</TD><TD>Related to Run Tumble Motion.</TD><TD>Down an Asp gradient 0.41pN, Up an Asp gradient 0.4387pN</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A><A href="http://jb.asm.org/cgi/reprint/189/5/1756.pdf" class="external text" rel="nofollow">On Torque and Tumbling in Swimming E. coli</A><A href="http://www.pnas.org/content/103/37/13712.full.pdf+html" class="external text" rel="nofollow">Swimming efficiency of bacterium E. coli.</A> Justification: Average thrust =0.41pN. In the Berg and Brown paper it states that the speed of the bacteria up an aspartate chemotactic gradient increases by 7%. Therefore in our model we shall use the following; thrust DOWN the Asp gradient =0.41pN, up the Asp gradient = 0.4387pN. Data was obtained from 32 AW405s, a strain which has provided the majority of our previous parameters but is not as motile as MG1655. The value was obtained at 23?C in viscosity 0.93 and 3.07 cP for motility buffer and motility buffer with 0.18% methylcellulose, respectively. The standard deviation is not used as the speed is fixed at 50?µm.s<SUP>-1</SUP> . 0.57pN is the average thrust generated in strain HCB30 (a non tumbling strain). The thrust value was obtained when the imposed flow (U) U=0 at 23?C. O.41pN was calculated using the resistance force theory treating the flagellar bundle as a single filament. The body was assumed to be prolate elipsoid using values roughly similar to ours, 2μm for length and 0.86μm for diameter.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>94</TD><TD>Isotropic run lengths</TD><TD>Related to Run Tumble Motion.</TD><TD>0.86?±1.18s</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A> Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.4 Berg and Brown) The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>95</TD><TD>Run length UP Aspartate gradient</TD><TD>Related to Run Tumble Motion.</TD><TD>1.07?±1.80s</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A><A href="http://parts2.mit.edu/wiki/index.php/University_of_California_San_Francisco_2006" class="external text" rel="nofollow">UCSF wiki</A> Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.6, Berg and Brown). The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>96</TD><TD>Run length DOWN Aspartate gradient</TD><TD>Related to Run Tumble Motion.</TD><TD>0.8?±1.38s</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v239/n5374/pdf/239500a0.pdf" class="external text" rel="nofollow">Chemotaxis in E. Coli anaylsed by three-dimensional tracking</A> Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.6, Berg and Brown) The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>97</TD><TD>Viscosity</TD><TD>Related to Run Tumble Motion.</TD><TD>Viscosity of water is 1.002cP at 20?°C</TD><TD> </TD><TD><A href="http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.biochem.72.121801.161737" class="external text" rel="nofollow">The rotary motor of bacterial flagella.</A>, <A href="http://jb.asm.org/cgi/reprint/189/5/1756.pdf" class="external text" rel="nofollow">On Torque and Tumbling in swimming Escherichia coli</A> Justification: At present the medium being used by the lab is still be discussed. Currently though the medium most resembles water and therefore the water's viscosity value can be used. This allows us to assume that the medium is Newtonian (dilute aqueous medium that doesn?€™t contain long unbranched molecules such as methylcellulose or polyvinylpyrrolidone. Note that methlycellulose does not alter the run and tumble statistics, only bundle and motor rotation rates are affected by the addition of methylcellulose). If agar were to be used then the medium would be Non-Newtonian. Even though it would be Non- Newtonian John Hogan in passing said that we could assume it is Newtonian.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>98</TD><TD>C<SUB>pMax</SUB></TD><TD>Maximal CpxR protein concentration</TD><TD>Unknown (varied in the program)</TD><TD> </TD><TD> </TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>99</TD><TD>k<SUB>Cp</SUB></TD><TD>Maximal transcription rate of pCpxR promoter</TD><TD>0.075min<SUP>-1<SUP> ESTIMATED </SUP></SUP></TD><TD> </TD><TD><A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11830644" class="external text" rel="nofollow">Surface Sensing and Adhesion of Escherichia Coli controlled by the cpx-signalling pathway.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>100</TD><TD>theta<SUB>Cpx</SUB></TD><TD>Threshold for pCpxR promoter Hill Function</TD><TD>1 x 10<SUP>-9</SUP> M  ESTIMATED</TD><TD> </TD><TD><A href="https://2008.igem.org/team:kuleuven" class="external text" rel="nofollow">iGEM 2008 KULeuven </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>101</TD><TD>m<SUB>Cpx</SUB></TD><TD>Co-operativity of pCpxR promoter Hill function</TD><TD>1</TD><TD> </TD><TD> </TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>102</TD><TD>d<SUB>Im</SUB></TD><TD>Degradation rate of GFP mRNA</TD><TD>3.6 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>103</TD><TD>k<SUB>Ip</SUB></TD><TD>Rate of LuxI protein translation</TD><TD>9.6 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>104</TD><TD>d<SUB>Ip</SUB></TD><TD>Degradation rate of LuxI protein</TD><TD>1.67 x 10<SUP>-2</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>105</TD><TD>d<SUB>Gm</SUB></TD><TD>Degradation rate of GFP protein</TD><TD>1.65 x 10<SUP>-3</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T36-42HFN5W-11&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_acct=C000010018&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=38be996fd7eb7dc4e71179e9d721b298" class="external text" rel="nofollow">Efficient GFP mutations profoundly affect mRNA transcription and translation rates </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>106</TD><TD>k<SUB>Gp</SUB></TD><TD>Rate of GFP protein translation</TD><TD>2.4 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.springerlink.com/content/p14488p56p37n602/" class="external text" rel="nofollow">Quantitative measurement of green fluorescent protein expression</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>107</TD><TD>d<SUB>Gp</SUB></TD><TD>Degradation rate of GFP protein</TD><TD>2.14 x 10<SUP>-4</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>108</TD><TD>A<SUB>prod</SUB></TD><TD>AHL production rate per LuxI enzyme</TD><TD>3.6 min<SUP>-1</SUP></TD><TD> </TD><TD>[<A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>109</TD><TD>d<SUB>A</SUB></TD><TD>Degradation rate of AHL molecule</TD><TD>1 x 10<SUP>-2</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">A synthetic multicellular system for programmed pattern formation </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>110</TD><TD>D<SUB>A</SUB></TD><TD>Diffusion coefficient of AHL</TD><TD>0.23s<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>111</TD><TD>k<SUB>Tp</SUB></TD><TD>Maximal Transcription rate of ptetR promoter</TD><TD>0.08min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2007.igem.org/Imperial/imperial/cell-free/characterisation" class="external text" rel="nofollow">iGEM 2007Imperial College London</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>112</TD><TD>d<SUB>Rm</SUB></TD><TD>Degradation rate of LuxR mRNA</TD><TD>3.6 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>113</TD><TD>k<SUB>Rp</SUB></TD><TD>Rate of Lux protein translation</TD><TD>9.6 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>114</TD><TD>d<SUB>Rp</SUB></TD><TD>Degration rate of LuxR protein</TD><TD>2.31 x 10<SUP>-2</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T2K-4H4T39N-1&amp;_user=121739&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=121739&amp;md5=08a37acb41420b0e80d3cde6ead4a347" class="external text" rel="nofollow">Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.</A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>115</TD><TD>k<SUB>LuxR</SUB></TD><TD>Maximal transcription rate of LuxR promoter</TD><TD>0.11 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2008.igem.org/team:kuleuven" class="external text" rel="nofollow">iGEM 2008 KULeuven </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>116</TD><TD>theta<SUB>LuxR</SUB></TD><TD>Threshold for LuxR pR promoter Hill function</TD><TD>1.5 x 10<SUP>-9</SUP> M</TD><TD> </TD><TD><A href="https://2008.igem.org/team:kuleuven" class="external text" rel="nofollow">iGEM 2008 KULeuven </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>117</TD><TD>m<SUB>LuxR</SUB></TD><TD>Co-operativity of LuxpR promoter Hill function</TD><TD>1.6</TD><TD> </TD><TD><A href="https://2008.igem.org/team:kuleuven" class="external text" rel="nofollow">iGEM 2008 KULeuven </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>118</TD><TD>rho<SUB>C</SUB></TD><TD>LuxR/AHL dimerisation</TD><TD>0.5 micro M<SUP>-3</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">A synthetic multicellular system for programmed pattern formation </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>119</TD><TD>d<SUB>C</SUB></TD><TD>Degradation rate of mCherry mRNA</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">A synthetic multicellular system for programmed pattern formation </A></TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>120</TD><TD>d<SUB>Mm</SUB></TD><TD>Degradation rate of LuxR/AHL complex</TD><TD>1.65 x 10<SUP>-3</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD>Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>121</TD><TD>k<SUB>Mp</SUB></TD><TD>Rate of mCherry protein translation</TD><TD>2.4 x 10<SUP>-1</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD>Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>122</TD><TD>d<SUB>Mp</SUB></TD><TD>Degradation rate of mCherry protein</TD><TD>2.14 x 10<SUP>-4</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD>Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.</TD><TD><A href="https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters">https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters</A></TD><TD>BCCS-Bristol</TD></TR><TR><TD>123</TD><TD>Rate of production of HSL from LuxI</TD><TD>0.45</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>124</TD><TD><SUB>HSL</SUB></TD><TD>Rate of diffusion of HSL in/out of the cell</TD><TD>0.4</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>125</TD><TD><SUB>IPTG</SUB></TD><TD>Rate of diffusion of IPTG in/out of the cell</TD><TD>0.014</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[3]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>126</TD><TD>p<SUB>production</SUB></TD><TD>Number of plasmids</TD><TD>10</TD><TD> </TD><TD>medium copy plasmid number; decided within the team</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>127</TD><TD>p<SUB>latch</SUB></TD><TD>Number of plasmids</TD><TD>10</TD><TD> </TD><TD>medium copy plasmid number; decided within the team</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>128</TD><TD>p<SUB>lysis</SUB></TD><TD>Number of plasmids</TD><TD>10</TD><TD> </TD><TD>medium copy plasmid number; decided within the team</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>129</TD><TD>p<SUB>anti-lysis</SUB></TD><TD>Number of plasmids</TD><TD>10</TD><TD> </TD><TD>medium copy plasmid number; decided within the team</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>130</TD><TD>p<SUB>QS</SUB></TD><TD>Number of plasmids</TD><TD>10</TD><TD> </TD><TD>medium copy plasmid number; decided within the team</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>131</TD><TD><SUB>max_production</SUB></TD><TD>Maximal production rate of lux box promoter</TD><TD>0.44</TD><TD>pops</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>132</TD><TD><SUB>min_production</SUB></TD><TD>Minimal production rate of lux box promoter</TD><TD>0.013</TD><TD>pops</TD><TD>estimated</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>133</TD><TD><SUB>latch</SUB></TD><TD>Maximal production rate of latch promoter</TD><TD>0.28</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>134</TD><TD><SUB>lysis</SUB></TD><TD>Maximal production rate of lysis promoter</TD><TD>0.0426</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>135</TD><TD><SUB>anti-lysis</SUB></TD><TD>Maximal production rate of anti-lysis promoter</TD><TD>0.0066</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>136</TD><TD><SUB>QS</SUB></TD><TD>Maximal production rate of QS promoter</TD><TD>0.018</TD><TD>pops</TD><TD><A href="#promotor">See Below</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>137</TD><TD>K<SUB>LacI</SUB></TD><TD>Dissociation constant for LacI to LacO</TD><TD>700</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>138</TD><TD>K<SUB>LacI-IPTG</SUB></TD><TD>Dissociation constant for IPTG to LacI</TD><TD>1200</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>139</TD><TD>K<SUB>TetR</SUB></TD><TD>Dissociation constant for TetR to TetO</TD><TD>7000</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>140</TD><TD>K<SUB>cI</SUB></TD><TD>Dissociation constant for cI to operon</TD><TD>7000</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>141</TD><TD>K<SUB>P</SUB></TD><TD>Dissociation constant for P to lux box</TD><TD>700</TD><TD>m</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">See Explanation</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>142</TD><TD>n<SUB>LacI</SUB></TD><TD>Hill coefficient</TD><TD>2</TD><TD> </TD><TD>[10]</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>143</TD><TD>n<SUB>cI</SUB></TD><TD>Hill coefficient</TD><TD>2</TD><TD> </TD><TD>[11]</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>144</TD><TD>n<SUB>TetR</SUB></TD><TD>Hill coefficient</TD><TD>3</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>145</TD><TD>n<SUB>P</SUB></TD><TD>Hill coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>146</TD><TD><SUB>mRNA</SUB></TD><TD>Degradation of mRNA</TD><TD>0.00288</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[2]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>147</TD><TD><SUB>X</SUB></TD><TD>Degradation of X</TD><TD>0.00000802</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[5]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>148</TD><TD><SUB>Y</SUB></TD><TD>Degradation of Y</TD><TD>0.00000802</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[5]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>149</TD><TD><SUB>λ-CI</SUB></TD><TD>Degradation of lambda CI</TD><TD>0.002888</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[5]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>150</TD><TD><SUB>LacI</SUB></TD><TD>Degradation of LacI</TD><TD>0.001155</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[8]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>151</TD><TD><SUB>TetR</SUB></TD><TD>Degradation of TetR</TD><TD>0.00288811</TD><TD>1/s</TD><TD>tagged; half-life of 4 min</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>152</TD><TD><SUB>Holin</SUB></TD><TD>Degradation of Holin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estmated; half-life of an hour</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>153</TD><TD><SUB>Endolysin</SUB></TD><TD>Degradation of Endolysin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estimated; half-life of an hour</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>154</TD><TD><SUB>Antiholin</SUB></TD><TD>Degradation of Antiholin</TD><TD>0.0002</TD><TD>1/s</TD><TD>estimated; half-life of an hour</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>155</TD><TD><SUB>LuxI</SUB></TD><TD>Degradation of LuxI</TD><TD>0.002888</TD><TD>1/s</TD><TD>tagged; half-life of 4 min</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>156</TD><TD><SUB>LuxR</SUB></TD><TD>Degradation of LuxR</TD><TD>0.0002</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>157</TD><TD><SUB>HSL</SUB></TD><TD>Degradation of HSL</TD><TD>0.00016667</TD><TD>1</TD><TD>[8]; value for AHL</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>158</TD><TD><SUB>Protein</SUB></TD><TD>Translation rate of Protein</TD><TD>0.1</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[2]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>159</TD><TD><SUB>P</SUB></TD><TD>Rate of formation of the HSL-LuxI complex</TD><TD>0.0001</TD><TD>1/ms</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>160</TD><TD><SUB>-P</SUB></TD><TD>Rate of dissociation of the HSL-LuxI complex</TD><TD>0.003</TD><TD>1/s</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">[1]</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters">https://2009.igem.org/Team:Aberdeen_Scotland/parameters</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>161</TD><TD>k_1</TD><TD>Max Transcription rate of tRNA</TD><TD>46.67</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.19</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>162</TD><TD>k_2</TD><TD>Synthesis rate of Aa-tRNA</TD><TD>0.08</TD><TD>min^-1</TD><TD>Sensitivity: 0.09</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>163</TD><TD>k_3</TD><TD>Max Transcription rate of T7RNAP</TD><TD>1.5625</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>164</TD><TD>k_4</TD><TD>Max Translation rate of T7RNAP</TD><TD>2.68*0.05</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>165</TD><TD>k_5</TD><TD>Max Transcription rate of trigger CI</TD><TD>5.6</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>166</TD><TD>k_6</TD><TD>Transcription rate of bistable CI</TD><TD>5.6</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>167</TD><TD>k_6'</TD><TD>Transcription rate of bistable CI</TD><TD>1</TD><TD>nM/min</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>168</TD><TD>k_7</TD><TD>Transcription rate of T3RNAP</TD><TD>1.75</TD><TD>nM/min</TD><TD>Assumption, sensitivity:1.34</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>169</TD><TD>k_7'</TD><TD>Transcription rate of T3RNAP</TD><TD>1</TD><TD>nM/min</TD><TD>Sensitivity: Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>170</TD><TD>k_8</TD><TD>Translation rate of trigger CI</TD><TD>9.6*0.045</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>171</TD><TD>k_8'</TD><TD>Translation rate of bistable CI</TD><TD>9.6*0.3</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>172</TD><TD>k_9</TD><TD>Max Transcription rate of CI434</TD><TD>5.92</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>173</TD><TD>k_10</TD><TD>Transcription rate of CI434</TD><TD>10.14*0.5</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>174</TD><TD>k_11</TD><TD>Max Translation rate of T3RNAP</TD><TD>3*0.15</TD><TD>min^-1</TD><TD>Assumption, sensitivity:1.34</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>175</TD><TD>k_12</TD><TD>Max Transcription rate of GFP from Sal</TD><TD>5.25</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>176</TD><TD>k_12'</TD><TD>Max Trasncription rate of GFP from T3RNAP</TD><TD>5.25</TD><TD>nM/min</TD><TD>Assumption, sensitivity:1.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>177</TD><TD>k_13</TD><TD>Translation rate of GFP</TD><TD>9*0.6</TD><TD>min^-1</TD><TD>Assumption, sensitivity:1.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>178</TD><TD>k_s</TD><TD>rate of AND Gate 1</TD><TD>0.3</TD><TD>nM^-1</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>179</TD><TD>k_s'</TD><TD>rate of AND Gate 2</TD><TD>0.3</TD><TD>nM^-1</TD><TD>Sensitivity: 0.18</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>180</TD><TD>K_1</TD><TD>dissociation constant of AraC,tRNA</TD><TD>14</TD><TD>nM</TD><TD>Sensitivity: 0.03</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>181</TD><TD>K_3</TD><TD>dissociation constant of Sal,T7RNAP</TD><TD>0.5</TD><TD>nM</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>182</TD><TD>K_5</TD><TD>dissociation constant of T7RNAP,trigger CI</TD><TD>3</TD><TD>nM</TD><TD><A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=y&amp;id=103592&amp;ver=1" class="external text" rel="nofollow">Ref</A></TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>183</TD><TD>K_6</TD><TD>dissociation constant of CI,bistable CI</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>184</TD><TD>K_6'</TD><TD>dissociation constant of CI434,bistable CI</TD><TD>50</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>185</TD><TD>K_7</TD><TD>dissociation constant of CI,T3RNAP</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>186</TD><TD>K_7'</TD><TD>dissociation constant of CI434,T3RNAP</TD><TD>50</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>187</TD><TD>K_9</TD><TD>dissociation constant of CI,CI434</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>188</TD><TD>K_12</TD><TD>dissociation constant of Sal,GFP</TD><TD>0.5</TD><TD>nM</TD><TD>Sensitivity: Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>189</TD><TD>K_12'</TD><TD>dissociation constant of T3RNAP,GFP</TD><TD>55</TD><TD>nM</TD><TD>Ref: The FEBS journal 2006,273:17</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>190</TD><TD>n_1</TD><TD>Hill co-effiency of AraC,tRNA</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>191</TD><TD>n_3</TD><TD>Hill co-effiency of Sal,T7RNAP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>192</TD><TD>n_5</TD><TD>Hill co-effiency of T7RNAP,CI</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>193</TD><TD>n_6</TD><TD>Hill co-effiency of CI,bistable CI</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>194</TD><TD>n_6'</TD><TD>Hill co-effiency of CI434,bistable CI</TD><TD>2</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>195</TD><TD>n_7</TD><TD>Hill co-effiency of CI,T3RNAP</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>196</TD><TD>n_7'</TD><TD>Hill co-effiency of CI434,T3RNAP</TD><TD>2</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>197</TD><TD>n_9</TD><TD>Hill co-effiency of CI,CI434</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>198</TD><TD>n_12</TD><TD>Hill co-effiency of Sal,GFP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>199</TD><TD>n_12'</TD><TD>Hill co-effiency of T3RNAP,GFP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>200</TD><TD>γ_1</TD><TD>Degradation rate of tRNA</TD><TD>1/30+1/60</TD><TD>min^-1</TD><TD>Since half life of tRNA is very long,we decided to use 60 mins instead</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>201</TD><TD>γ_2</TD><TD>Degradation rate of Aa-tRNA</TD><TD>1/30+1/40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>202</TD><TD>γ_2'</TD><TD>Real Degradation rate of Aa-tRNA</TD><TD>ינו-40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>203</TD><TD>γ_3</TD><TD>Degradation rate of T7RNAP mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>204</TD><TD>γ_4</TD><TD>Degradation rate of T7RNAP</TD><TD>1/30+1/40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>205</TD><TD>γ_5</TD><TD>Degradation rate of trigger CI mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>206</TD><TD>γ_6</TD><TD>Degradation rate of bistable CI mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>207</TD><TD>γ_7</TD><TD>Degradation rate of T3RNAP mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>208</TD><TD>γ_8</TD><TD>Degradation rate of CI</TD><TD>1/30+1/44</TD><TD>min^-1</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>209</TD><TD>γ_9</TD><TD>Degradation rate of CI434 mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>210</TD><TD>γ_10</TD><TD>Degradation rate of CI434</TD><TD>1/30+1/11</TD><TD>min^-1</TD><TD>Ref: iGEM 2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>211</TD><TD>γ_11</TD><TD>Degradation rate of T3RNAP</TD><TD>1/30+1/30</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>212</TD><TD>γ_12</TD><TD>Degradation rate of GFP mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>213</TD><TD>γ_13</TD><TD>Degradation rate of GFP</TD><TD>1/30+1/60</TD><TD>min^-1</TD><TD>Since half life of GFP is very long,we use 60 mins instead</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>214</TD><TD>k_1</TD><TD>Max Transcription rate of tRNA</TD><TD>46.67</TD><TD>nM/min</TD><TD>Assumption, sensitivity:1.41</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>215</TD><TD>k_2</TD><TD>Synthesis rate of Aa-tRNA</TD><TD>0.08</TD><TD>min^-1</TD><TD>Sensitivity: 0.81</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>216</TD><TD>k_3</TD><TD>Max Transcription rate of T7RNAP</TD><TD>1.5625</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>217</TD><TD>k_4</TD><TD>Max Translation rate of T7RNAP</TD><TD>2.68*0.05</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>218</TD><TD>k_5</TD><TD>Max Transcription rate of trigger CI</TD><TD>5.6</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>219</TD><TD>k_6</TD><TD>Transcription rate of bistable CI</TD><TD>5.6</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>220</TD><TD>k_6'</TD><TD>Transcription rate of bistable CI</TD><TD>1</TD><TD>nM/min</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>221</TD><TD>k_7</TD><TD>Transcription rate of P2</TD><TD>16.8</TD><TD>nM/min</TD><TD>Assumption, sensitivity:1.23</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>222</TD><TD>k_7'</TD><TD>Transcription rate of P2</TD><TD>1</TD><TD>nM/min</TD><TD>Sensitivity: Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>223</TD><TD>k_8</TD><TD>Translation rate of trigger CI</TD><TD>9.6*0.05</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>224</TD><TD>k_8'</TD><TD>Translation rate of bistable CI</TD><TD>9.6*0.5</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>225</TD><TD>k_9</TD><TD>Max Transcription rate of CI434</TD><TD>5.92</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>226</TD><TD>k_10</TD><TD>Transcription rate of CI434</TD><TD>10.14*1</TD><TD>min^-1</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>227</TD><TD>k_11</TD><TD>Max Translation rate of P2</TD><TD>28.8*0.0045</TD><TD>min^-1</TD><TD>Assumption, sensitivity:1.23</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>228</TD><TD>k_12</TD><TD>Max Transcription rate of GFP from Sal</TD><TD>5.25</TD><TD>nM/min</TD><TD>Assumption, sensitivity:0.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>229</TD><TD>k_12'</TD><TD>Max Trasncription rate of GFP from P2</TD><TD>5.25</TD><TD>nM/min</TD><TD>Assumption, sensitivity:1.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>230</TD><TD>k_13</TD><TD>Translation rate of GFP</TD><TD>9*0.6</TD><TD>min^-1</TD><TD>Assumption, sensitivity:1.00</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>231</TD><TD>k_s</TD><TD>rate of AND Gate 1</TD><TD>0.3</TD><TD>nM^-1</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>232</TD><TD>k_s'</TD><TD>rate of AND Gate 2</TD><TD>0.01</TD><TD>nM^-1</TD><TD>Sensitivity: 1.29</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>233</TD><TD>K_1</TD><TD>dissociation constant of AraC,tRNA</TD><TD>14</TD><TD>nM</TD><TD>Sensitivity: 0.2</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>234</TD><TD>K_3</TD><TD>dissociation constant of Sal,T7RNAP</TD><TD>0.5</TD><TD>nM</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>235</TD><TD>K_5</TD><TD>dissociation constant of T7RNAP,trigger CI</TD><TD>3</TD><TD>nM</TD><TD><A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=y&amp;id=103592&amp;ver=1" class="external text" rel="nofollow">Ref</A></TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>236</TD><TD>K_6</TD><TD>dissociation constant of CI,bistable CI</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>237</TD><TD>K_6'</TD><TD>dissociation constant of CI434,bistable CI</TD><TD>50</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>238</TD><TD>K_7</TD><TD>dissociation constant of CI,P2</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>239</TD><TD>K_7'</TD><TD>dissociation constant of CI434,P2</TD><TD>50</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>240</TD><TD>K_9</TD><TD>dissociation constant of CI,CI434</TD><TD>40</TD><TD>nM</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>241</TD><TD>K_12</TD><TD>dissociation constant of Sal,GFP</TD><TD>0.5</TD><TD>nM</TD><TD>Sensitivity: 0</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>242</TD><TD>K_12'</TD><TD>dissociation constant of P2,GFP</TD><TD>35</TD><TD>nM</TD><TD>Sensitivity: 1.29</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>243</TD><TD>n_1</TD><TD>Hill co-effiency of AraC,tRNA</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>244</TD><TD>n_3</TD><TD>Hill co-effiency of Sal,T7RNAP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>245</TD><TD>n_5</TD><TD>Hill co-effiency of T7RNAP,CI</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>246</TD><TD>n_6</TD><TD>Hill co-effiency of CI,bistable CI</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>247</TD><TD>n_6'</TD><TD>Hill co-effiency of CI434,bistable CI</TD><TD>2</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>248</TD><TD>n_7</TD><TD>Hill co-effiency of CI,P2</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>249</TD><TD>n_7'</TD><TD>Hill co-effiency of CI434,P2</TD><TD>2</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>250</TD><TD>n_9</TD><TD>Hill co-effiency of CI,CI434</TD><TD>4</TD><TD> </TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>251</TD><TD>n_12</TD><TD>Hill co-effiency of Sal,GFP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>252</TD><TD>n_12'</TD><TD>Hill co-effiency of P2,GFP</TD><TD>2</TD><TD> </TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>253</TD><TD>γ_1</TD><TD>Degradation rate of tRNA</TD><TD>1/30+1/60</TD><TD>min^-1</TD><TD>Since half life of tRNA is very long,we decided to use 60 mins instead</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>254</TD><TD>γ_2</TD><TD>Degradation rate of Aa-tRNA</TD><TD>1/30+1/40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>255</TD><TD>γ_2'</TD><TD>Real Degradation rate of Aa-tRNA</TD><TD>ינו-40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>256</TD><TD>γ_3</TD><TD>Degradation rate of T7RNAP mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>257</TD><TD>γ_4</TD><TD>Degradation rate of T7RNAP</TD><TD>1/30+1/40</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>258</TD><TD>γ_5</TD><TD>Degradation rate of trigger CI mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>259</TD><TD>γ_6</TD><TD>Degradation rate of bistable CI mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>260</TD><TD>γ_7</TD><TD>Degradation rate of P2 mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>261</TD><TD>γ_8</TD><TD>Degradation rate of CI</TD><TD>1/30+1/44</TD><TD>min^-1</TD><TD>Ref: iGEM2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>262</TD><TD>γ_9</TD><TD>Degradation rate of CI434 mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>263</TD><TD>γ_10</TD><TD>Degradation rate of CI434</TD><TD>1/30+1/11</TD><TD>min^-1</TD><TD>Ref: iGEM 2007 PKU Team</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>264</TD><TD>γ_11</TD><TD>Degradation rate of P2</TD><TD>1/30+1/30</TD><TD>min^-1</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>265</TD><TD>γ_12</TD><TD>Degradation rate of GFP mRNA</TD><TD>1/30+1/4.4</TD><TD>min^-1</TD><TD>Assumption</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>266</TD><TD>γ_13</TD><TD>Degradation rate of GFP</TD><TD>1/30+1/60</TD><TD>min^-1</TD><TD>Since half life of GFP is very long,we use 60 mins instead</TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters">https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters</A></TD><TD>PKU Beijing</TD></TR><TR><TD>267</TD><TD>c<SUB>pLac</SUB></TD><TD>maximum transcription rate (M/min)</TD><TD>5E-10</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[1]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>268</TD><TD>c<SUB>pTet</SUB></TD><TD>maximum transcription rate (M/min)</TD><TD>0.00000015</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[2]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>269</TD><TD>c<SUB>pλ</SUB></TD><TD>maximum transcription rate (M/min)</TD><TD>0.00000015</TD><TD> </TD><TD>estimate</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>270</TD><TD>K<SUB>50</SUB><SUP>IPTG</SUP></TD><TD>dissociation constant (M)</TD><TD>0.0000013</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>271</TD><TD>K<SUB>50</SUB><SUP>LacI</SUP></TD><TD>dissociation constant (M)</TD><TD>0.0000008</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>272</TD><TD>K<SUB>50</SUB><SUP>TetR</SUP></TD><TD>dissociation constant (M)</TD><TD>1.79E-10</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>273</TD><TD>K<SUB>50</SUB><SUP>CI</SUP></TD><TD>dissociation constant (M)</TD><TD>0</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>274</TD><TD>n<SUB>IPTG</SUB></TD><TD>Hills coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>275</TD><TD>n<SUB>LacI</SUB></TD><TD>Hills coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>276</TD><TD>n<SUB>TetR</SUB></TD><TD>Hills coefficient</TD><TD>3</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>277</TD><TD>n<SUB>CI</SUB></TD><TD>Hills coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>278</TD><TD>d<SUB>LacI</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.1386</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>279</TD><TD>d<SUB>TetR</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.1386</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>280</TD><TD>d<SUB>CI</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.042</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>281</TD><TD>d<SUB>RFP</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.0063</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>282</TD><TD>d<SUB>GFP</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.0063</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[3]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>283</TD><TD>d<SUB>mRNA</SUB></TD><TD>degradation rate (M/min)</TD><TD>0.029</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[4]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>284</TD><TD>α</TD><TD>translation rate (translations/min/mRNA), depends on growth rate (a default value of 30 is used)</TD><TD>16 - 57</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[5]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>285</TD><TD>k<SUB>IPTG</SUB></TD><TD>rate constant for IPTG diffusion into cell</TD><TD>0.92</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[6]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>286</TD><TD>A</TD><TD>Surface area available (mm<SUP>2</SUP>)</TD><TD>1735</TD><TD> </TD><TD>Area of 0.2μm filter used in conjugation tests.</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>287</TD><TD>r<SUB>0</SUB></TD><TD>initial colony radius (μm)</TD><TD>0.8</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[7]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>288</TD><TD>g<SUB>n</SUB></TD><TD>specific growth rate (1/hr)</TD><TD>0.99</TD><TD> </TD><TD>Determined experimentally for R751 containing cells.</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>289</TD><TD>g<SUB>r</SUB></TD><TD>colony radial specific growth rate (μm/hr)</TD><TD>30</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_References" title="Team:TUDelft/Modeling References">[8]</A></TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>290</TD><TD>N<SUB>d</SUB></TD><TD>initial number of donors</TD><TD>10000</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>291</TD><TD>N<SUB>r</SUB></TD><TD>initial number of donors</TD><TD>10000</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>292</TD><TD>λ</TD><TD>intensity (CFU/mm<SUP>2</SUP>)</TD><TD>5.76</TD><TD> </TD><TD>Calculated using N<SUB>r</SUB>/A.</TD><TD><A href="https://2009.igem.org/Team:TUDelft/Modeling_Parameters">https://2009.igem.org/Team:TUDelft/Modeling_Parameters</A></TD><TD>TUDelft</TD></TR><TR><TD>293</TD><TD>K<SUB>LacI</SUB></TD><TD>LacI repressor dissociation constant</TD><TD>0.1 - 1 [pM] OR 800 [nM]</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>294</TD><TD>K<SUB>IPTG</SUB></TD><TD>IPTG-LacI repressor dissociation constant</TD><TD>1.3 [?µM]</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>295</TD><TD>K<SUB>tetR</SUB></TD><TD>TetR repressor dissociation constant</TD><TD>179 [pM]</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>296</TD><TD>K<SUB>cI</SUB></TD><TD>cI repressor dissociation constant</TD><TD>8 [pM] OR 50 [nM]</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>297</TD><TD>K<SUB>HSL</SUB></TD><TD>HSL-LuxR activator dissociation constant</TD><TD>0.09 - 1 [?µM]</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>298</TD><TD>K<SUB>LacI</SUB></TD><TD>Dissociation constant for LacI to LacO DNA site</TD><TD>~1*10 -12 M OR ~1.8*10-12 M</TD><TD> </TD><TD>Mitchel Lewis (2005) The Lac repressor. C. R. Biologies 328 (2005) 521?€“548; Falcon C.M and Matthews K.S. (2000) Operator DNA sequence Variation Enhances High Affinity Binding by Hinge Helix Mutants of Lactose Repressor Protein. Biochemistry. 39, 11074-11084</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>299</TD><TD>K<SUB>IPTG</SUB></TD><TD>Dissociation constant for IPTG to LacI</TD><TD>1*10-6 M</TD><TD> </TD><TD>&quot;Uri Alon, An introduction to systems Biology, p244</TD></TR><TR><TD>&quot;</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>300</TD><TD>K<SUB>tetR</SUB></TD><TD>Dissociation constant for TetR to TetO</TD><TD>(5.6 ?± 2) ?— 10-9 M OR 1.53*10-8 M</TD><TD> </TD><TD>Nucleic Acids Res. 2004; 32(2): 842?€“847. Two mutations in the tetracycline repressor change the inducer anhydrotetracycline to a corepressor Annette Kamionka, Joanna Bogdanska-Urbaniak, Oliver Scholz, and Wolfgang Hillen; Volume 272, Number 11, Issue of March 14, 1997 pp. 6936-6942, The Role of the Variable Region in Tet Repressor for Inducibility by Tetracycline, Christian Berens , Dirk Schnappinger and Wolfgang Hillen</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>301</TD><TD>K<SUB>cI</SUB></TD><TD>Dissocitation constant for cI to DNA site</TD><TD>50 * 10-9 M</TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD><TD>Aberdeen Scotland</TD></TR><TR><TD>302</TD><TD>α<SUB>TetR</SUB></TD><TD>TetR max. production rate</TD><TD>3.93 μM/min, updated to 1 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref5" title="Team:ETH Zurich/Modeling/References">[5]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>303</TD><TD>α<SUB>C</SUB></TD><TD>CI max. production rate</TD><TD>1 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>304</TD><TD>α<SUB>L1</SUB></TD><TD>LacI max. production rate</TD><TD>1 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>305</TD><TD>α<SUB>L2</SUB></TD><TD>LacI<SUB>M1</SUB> max. production rate</TD><TD>1 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>306</TD><TD>α<SUB>G</SUB></TD><TD>GFP max. production rate</TD><TD>2 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>307</TD><TD>α<SUB>LuxI</SUB></TD><TD>LuxI max. production rate</TD><TD>1 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>308</TD><TD>k<SUB>s1</SUB></TD><TD>AHL<SUB>i</SUB> production rate</TD><TD>0.01 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>309</TD><TD>ρ<SUB>R</SUB></TD><TD>LuxR-AHL dimerization rate</TD><TD>0.5 1/(μM<SUP>3</SUP>*min)</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>310</TD><TD>α<SUB>RFP</SUB></TD><TD>RFP max. production rate</TD><TD>1 μM/min</TD><TD> </TD><TD>assumption</TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>311</TD><TD>β<SUB>AcA</SUB></TD><TD>Acetaldehyde repression coefficient</TD><TD>7.124 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref5" title="Team:ETH Zurich/Modeling/References">[5]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>312</TD><TD>β<SUB>T</SUB></TD><TD>TetR repression coefficient</TD><TD>0.01 μM, updated to 0.1 μM/min</TD><TD> </TD><TD>assumption</TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>313</TD><TD>β<SUB>C</SUB></TD><TD>CI repression coefficient</TD><TD>0.008 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>314</TD><TD>β<SUB>L</SUB></TD><TD>LacI repression coefficient</TD><TD>0.8 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>315</TD><TD>θ<SUB>R</SUB></TD><TD>LuxR-AHL repression coefficient</TD><TD>0.01 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>316</TD><TD>γ<SUB>TetR</SUB></TD><TD>TetR degradation rate</TD><TD>0.0692 1/min</TD><TD> </TD><TD>assumption</TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>317</TD><TD>γ<SUB>C</SUB></TD><TD>CI degradation rate</TD><TD>0.0692 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>318</TD><TD>γ<SUB>L</SUB></TD><TD>LacI degradation rate</TD><TD>0.0231 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>319</TD><TD>γ<SUB>G</SUB></TD><TD>GFP degradation rate</TD><TD>0.0692 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>320</TD><TD>γ<SUB>LuxI</SUB></TD><TD>LuxI degradation rate</TD><TD>0.0167 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref3" title="Team:ETH Zurich/Modeling/References">[3]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>321</TD><TD>k<SUB>s0</SUB></TD><TD>AHL<SUB>i</SUB> degradation rate</TD><TD>1 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>322</TD><TD>k<SUB>se</SUB></TD><TD>AHL<SUB>e</SUB> degradation rate</TD><TD>1 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>323</TD><TD>γ<SUB>R</SUB></TD><TD>LuxR-AHL degradation rate</TD><TD>0.0231 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>324</TD><TD>γ<SUB>RFP</SUB></TD><TD>RFP degradation rate</TD><TD>0.0041 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref4" title="Team:ETH Zurich/Modeling/References">[4]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>325</TD><TD>n<SUB>1</SUB></TD><TD>LacI cooperativity coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>326</TD><TD>n<SUB>2</SUB></TD><TD>CI cooperativity coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>327</TD><TD>n<SUB>3</SUB></TD><TD>TetR cooperativity coefficient</TD><TD>2</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>328</TD><TD>n<SUB>4</SUB></TD><TD>LuxR-AHL cooperativity coefficient</TD><TD>1</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref1" title="Team:ETH Zurich/Modeling/References">[1]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>329</TD><TD>n<SUB>5</SUB></TD><TD>AlcR-AcA cooperativity coefficient</TD><TD>1.352</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref5" title="Team:ETH Zurich/Modeling/References">[5]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>330</TD><TD>n<SUB>7</SUB></TD><TD>XylR cooperativity coefficient</TD><TD>5</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref5" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>331</TD><TD>η</TD><TD>Diffusion rate across the cell membrane</TD><TD>2</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>332</TD><TD>η<SUB>ext</SUB></TD><TD>Average diffusion rate for all cells</TD><TD>1</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref2" title="Team:ETH Zurich/Modeling/References">[2]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>333</TD><TD>N</TD><TD>Number of cells</TD><TD>1</TD><TD> </TD><TD>assumption</TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>334</TD><TD>a<SUB>XylR</SUB></TD><TD>XylR max. production rate</TD><TD>95.5 μM/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref7" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>335</TD><TD>γ<SUB>XylRi</SUB></TD><TD>XylRi degradation rate</TD><TD>0.03553 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref7" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>336</TD><TD>γ<SUB>XylRa</SUB></TD><TD>XylRa degradation rate</TD><TD>0.04527 1/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref7" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>337</TD><TD>K<SUB>XylR</SUB></TD><TD>XylR activation coefficient</TD><TD>1.419*10-3 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>338</TD><TD>r<SUB>XylR</SUB></TD><TD>XylRi oligomerization constant</TD><TD>0.04315e-3 1/(μM*min)</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>339</TD><TD>r<SUB>RXylR</SUB></TD><TD>XylRa dissociation constant</TD><TD>0.1301e-3 1/(μM*min)</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref6" title="Team:ETH Zurich/Modeling/References">[6]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>340</TD><TD>z<SUB>end</SUB></TD><TD>Microfluidics channel length</TD><TD>5 cm</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Cell_Density" title="Team:ETH Zurich/Modeling/Microfluidics">Choice in diffusion model</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>341</TD><TD>R<SUB>Channel</SUB></TD><TD>Microfluidics channel radius</TD><TD>1 mm</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Cell_Density" title="Team:ETH Zurich/Modeling/Microfluidics">Choice in diffusion model</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>342</TD><TD>V<SUB>eColi</SUB></TD><TD><I>E. coli</I> average volume</TD><TD>2 μm<SUP>3</SUP></TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref13" title="Team:ETH Zurich/Modeling/References">[13]</A><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref14" title="Team:ETH Zurich/Modeling/References">[14]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>343</TD><TD>cd<SUB>OD600, undiluted</SUB></TD><TD><I>E. coli</I> Cell Density at OD<SUB>600</SUB> = 1</TD><TD>1e9 1/ml</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref13" title="Team:ETH Zurich/Modeling/References">[13]</A><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref16" title="Team:ETH Zurich/Modeling/References">[16]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>344</TD><TD>c<SUB>Dilution</SUB></TD><TD>Dilution of cell culture in microfludics channel</TD><TD>01-אוג</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Cell_Density" title="Team:ETH Zurich/Modeling/Microfluidics">Choice in diffusion model</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>345</TD><TD>cdRel<SUB>OD600, undiluted</SUB></TD><TD>Relative <I>E. coli</I> Cell Density at OD<SUB>600</SUB> = 1, undiluted</TD><TD>0.001</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Cell_Density" title="Team:ETH Zurich/Modeling/Microfluidics">Derived from V<SUB>eColi</SUB>, cd<SUB>OD600, undiluted</SUB></A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>346</TD><TD>cdRel<SUB>OD600, diluted</SUB></TD><TD>Relative <I>E. coli</I> Cell Density at OD<SUB>600</SUB> = 1, diluted</TD><TD>0.000125</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Cell_Density" title="Team:ETH Zurich/Modeling/Microfluidics">Derived from cdRel<SUB>OD600, undiluted</SUB>, c<SUB>Dilution</SUB></A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>347</TD><TD>C<SUB>Agarose</SUB></TD><TD>Scaling coefficient for diffusion of small molecules in agarose instead of water</TD><TD>0.9</TD><TD> </TD><TD>assumption</TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>348</TD><TD>D<SUB>AcAl</SUB></TD><TD>Acetaldehyde Diffusion Constant</TD><TD>1.23e-5 cm<SUP>2</SUP>/s</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref8" title="Team:ETH Zurich/Modeling/References">[8]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>349</TD><TD>m<SUB>AcAl</SUB></TD><TD>Acetaldehyde Molecular Weight</TD><TD>44.05316 u</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref8" title="Team:ETH Zurich/Modeling/References">[8]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>350</TD><TD>K<SUB>cat,AcAl</SUB></TD><TD>Specific enzyme activity of acetaldehyde dehydrogenase in <I>E. coli</I></TD><TD>14.1 μmol/(min*mg)</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref10" title="Team:ETH Zurich/Modeling/References">[10]</A><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>351</TD><TD>[E]<SUP>T</SUP></TD><TD>Mean acetaldehyde dehydrogenase enzyme concentration in <I>E. coli</I></TD><TD>1.6659 kg/m<SUP>3</SUP></TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Estimation_of_vmax.2CAcAl" title="Team:ETH Zurich/Modeling/Microfluidics">Estimation from cell extract</A>, <A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>352</TD><TD>v<SUB>max,AcAl</SUB></TD><TD>Maximum reaction rate for acetaldehyde dehydrogenase degrading acetaldehyde</TD><TD>0.02349 M/min</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Estimation_of_vmax.2CAcAl" title="Team:ETH Zurich/Modeling/Microfluidics">Estimation from cell extract</A>, <A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>353</TD><TD>K<SUB>M,AcAl</SUB></TD><TD>Michaelis-Menten constant for acetaldehyde dehydrogenase degrading acetaldehyde</TD><TD>10 μM</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref10" title="Team:ETH Zurich/Modeling/References">[10]</A><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>354</TD><TD>m<SUB>ADH</SUB></TD><TD>Molecular mass of acetaldehyde dehydrogenase</TD><TD>33442 u</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref12" title="Team:ETH Zurich/Modeling/References">[12]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>355</TD><TD>n<SUB>ADH</SUB></TD><TD>Mean number of acetaldehyde dehydrogenase protein molecules per <I>E. coli</I> cell</TD><TD>28512.2, rounded to 30000</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Microfluidics#Estimation_of_vmax.2CAcAl" title="Team:ETH Zurich/Modeling/Microfluidics">Estimation from cell extract</A>, <A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>356</TD><TD>[ADH]<SUB>Extract</SUB></TD><TD>acetaldehyde dehydrogenase concentration in <I>E. coli</I> cell extract</TD><TD>19 mg/ml</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>357</TD><TD>dil<SUB>Extract</SUB></TD><TD><I>E. coli</I> cell extract dilution</TD><TD>01-דצמ</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref11" title="Team:ETH Zurich/Modeling/References">[11]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>358</TD><TD>D<SUB>AHL</SUB></TD><TD>AHL Diffusion Constant</TD><TD>4.9e-6 cm<SUP>2</SUP>/s</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref9" title="Team:ETH Zurich/Modeling/References">[9]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>359</TD><TD>m<SUB>AHL</SUB></TD><TD>AHL Molecular Weight</TD><TD>297.38990 u</TD><TD> </TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref15" title="Team:ETH Zurich/Modeling/References">[15]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>360</TD><TD>γ<SUB>AHL,ext</SUB></TD><TD>AHL cell-external degradation</TD><TD>8.0225e-006/s</TD><TD> </TD><TD>Derived from 1 day half-life at pH 7 <A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/References#Ref7" title="https://2011.igem.org/Team:ETH Zurich/Modeling/References">[7]</A></TD><TD><A href="https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>361</TD><TD>Concentration of X</TD><TD>NA</TD><TD>molecules per cell</TD><TD>Notation convention</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>362</TD><TD>Delay due to protein X production and maturation</TD><TD>NA</TD><TD>s</TD><TD>Notation convention</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>363</TD><TD>Maximal production rate of pVeg promoter (constitutive)</TD><TD>0.02</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD>Estimated, see the <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Promoter_strengths">justification</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>364</TD><TD>Maximal production rate of pHyperSpank promoter</TD><TD>0.02</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD>Estimated, see the <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Promoter_strengths">justification</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>365</TD><TD>Maximal production rate of pT7 promoter</TD><TD>0.02</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD>Estimated, see the <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Promoter_strengths">justification</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>366</TD><TD>Maximal production rate of pComK promoter</TD><TD>0.049</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>367</TD><TD>Maximal production rate of pComS promoter</TD><TD>0.057</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>368</TD><TD>Maximal production rate of pComG promoter</TD><TD>0.02</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD>Estimated, see the <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Promoter_strengths">justification</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>369</TD><TD>Basal production rate of pComK promoter</TD><TD>0.0028</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>370</TD><TD>Basal production rate of pComG promoter</TD><TD>0.028</TD><TD>molecules.s<SUP>-1</SUP>or pops</TD><TD>Is roughly one order of magnitude higher than production rate of pComK<A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[4]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>371</TD><TD>Dissociation constant for IPTG to LacI</TD><TD>1200</TD><TD>molecules per cell</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">Aberdeen 2009 wiki</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>372</TD><TD>Dissociation constant for LacI to LacO (pLac)</TD><TD>700</TD><TD>molecules per cell</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">Aberdeen 2009 wiki</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>373</TD><TD>Dissociation constant for T7 RNA polymerase to pT7</TD><TD>10</TD><TD>molecules per cell</TD><TD>We used the classic assumption 1nM=1 molecule per cell and <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[1]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>374</TD><TD>Dissociation constant for ComK to pComK</TD><TD>110</TD><TD>molecules per cell</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>375</TD><TD>Dissociation constant for ComK to pComS</TD><TD>100</TD><TD>molecules per cell</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>376</TD><TD>ComK concentration for half maximal degradation</TD><TD>500</TD><TD>molecules per cell</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>377</TD><TD>ComS concentration for half maximal degradation</TD><TD>50</TD><TD>molecules per cell</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>378</TD><TD>Hill coefficient for LacI/IPTG interaction</TD><TD>2</TD><TD>NA</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">Aberdeen 2009 wiki</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>379</TD><TD>Hill coefficient for LacI/pHyperSpank interaction</TD><TD>2</TD><TD>NA</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">Aberdeen 2009 wiki</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>380</TD><TD>Hill coefficient for ComK/pComK and ComK/pComG (positive feedback) interaction</TD><TD>2</TD><TD>NA</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>381</TD><TD>Hill coefficient for ComK/pComS (negative feedback) interaction</TD><TD>5</TD><TD>NA</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>382</TD><TD>Translation rate of proteins</TD><TD>0.9</TD><TD>s<SUP>-1</SUP></TD><TD>Estimated, see the <A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Protein_translation_rate_justification">justification</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>383</TD><TD>Dilution rate in exponential phase</TD><TD>2.88x10<SUP>-4</SUP></TD><TD>s<SUP>-1</SUP></TD><TD>Calculated with a 40 min generation time. See explanation</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>384</TD><TD>Unrepressed degradation rate of ComK</TD><TD>1.4x10<SUP>-3</SUP></TD><TD>s<SUP>-1</SUP></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>385</TD><TD>Unrepressed degradation rate of ComS</TD><TD>1.4x10<SUP>-3</SUP></TD><TD>s<SUP>-1</SUP></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>386</TD><TD>Degradation rate of mRNA</TD><TD>2.88x10<SUP>-3</SUP></TD><TD>s<SUP>-1</SUP></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[4]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>387</TD><TD>Degradation rate of GFP</TD><TD>10<SUP>-4</SUP></TD><TD>s<SUP>-1</SUP></TD><TD><A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=y&amp;id=105188&amp;ver=2&amp;hlid=56398">BioNumbers</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>388</TD><TD>Degradation rate of RFP</TD><TD>10<SUP>-4</SUP></TD><TD>s<SUP>-1</SUP></TD><TD>Estimated equal to GFP degradation rate</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>389</TD><TD>Delay due to CFP production and maturation</TD><TD>360</TD><TD>s</TD><TD>Estimated equal to GFP delay (similar molecules)</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>390</TD><TD>Delay due to YFP production and maturation</TD><TD>360</TD><TD>s</TD><TD>Estimated equal to GFP delay (similar molecules)</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>391</TD><TD>Delay due to ComK production and maturation</TD><TD>300</TD><TD>s</TD><TD>Arbitrary</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>392</TD><TD>Delay due to ComS production and maturation</TD><TD>300</TD><TD>s</TD><TD>Arbitrary</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>393</TD><TD>Delay for ComS repression by ComK</TD><TD>714</TD><TD>s</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[3]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>394</TD><TD>Delay due tT7 RNA polymerase production and maturation</TD><TD>300</TD><TD>s</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters#references">[2]</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>395</TD><TD>Delay due GFP production and maturation</TD><TD>360</TD><TD>s</TD><TD><A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=y&amp;id=102972&amp;ver=8">BioNumbers</A></TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>396</TD><TD>Delay due RFP production and maturation</TD><TD>360</TD><TD>s</TD><TD>Estimated equal to GFP delay (similar molecules)</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>397</TD><TD>Delay due to mRNA production</TD><TD>30</TD><TD>s</TD><TD><A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=y&amp;id=104902&amp;ver=5&amp;hlid=58815 2kb">BioNumbers</A> with an approximation: all our contructs are around 1-2kb</TD><TD><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters</A></TD><TD>Paris Bettencourt</TD></TR><TR><TD>398</TD><TD>L</TD><TD>tail length</TD><TD>0.8</TD><TD>cm</TD><TD><A href="#atlas">[1]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>399</TD><TD>l</TD><TD>max tail width</TD><TD>0.07</TD><TD>cm</TD><TD><A href="#atlas">[1]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>400</TD><TD>r</TD><TD>head radius</TD><TD>0.3</TD><TD>cm</TD><TD><A href="#atlas">[1]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>401</TD><TD>T<SUB>skin</SUB></TD><TD>skin thickness</TD><TD>35</TD><TD>μm</TD><TD><A href="#atlas">[1]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>402</TD><TD>V<SUB>tail</SUB></TD><TD>tail volume</TD><TD>0.001</TD><TD>cm<SUP>3</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/volume#alas">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>403</TD><TD>V<SUB>head</SUB></TD><TD>head volume</TD><TD>0.1131</TD><TD>cm<SUP>3</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/volume#attlas">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>404</TD><TD>V</TD><TD>tadpole volume</TD><TD>0.1141</TD><TD>cm<SUP>3</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/volume#atttlas">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>405</TD><TD>V<SUB>skin</SUB></TD><TD>skin volume</TD><TD>0.0041</TD><TD>cm<SUP>3</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/volume#attttlas">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>406</TD><TD>V<SUB>receptor</SUB></TD><TD>receptor compartment volume</TD><TD>xxxx</TD><TD>cm<SUP>3</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/volume#atttttlas">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>407</TD><TD>P<SUB>X&lt;-&gt;Y</SUB></TD><TD>permeability between tissues X and Y</TD><TD>10<SUP>-5</SUP></TD><TD>cm.min<SUP>-1</SUP></TD><TD><A href="#perm">[3]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>408</TD><TD>S<SUB>tadpole</SUB></TD><TD>sum of the external surfaces of the tail and the head</TD><TD>1.219</TD><TD>cm<SUP>2</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/surface#tad">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>409</TD><TD>S<SUB>skin</SUB></TD><TD>skin internal surface</TD><TD>1.1839</TD><TD>cm<SUP>2</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/surface#skin">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>410</TD><TD>S<SUB>receptor</SUB></TD><TD>blood-receptor exchange surface</TD><TD>0.0781</TD><TD>cm<SUP>2</SUP></TD><TD><A href="https://2012.igem.org/Team:Evry/surface#blood">here</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>411</TD><TD>D<SUB>auxin</SUB></TD><TD>auxin diameter</TD><TD>0.766</TD><TD>nm</TD><TD><A href="#keggaux">[2]</A></TD><TD><A href="https://2012.igem.org/Team:Evry/parameters">https://2012.igem.org/Team:Evry/parameters</A></TD><TD>Evry</TD></TR><TR><TD>412</TD><TD>sun</TD><TD>natural sun light</TD><TD> </TD><TD> </TD><TD>n/a</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>413</TD><TD>room</TD><TD>sun*0.3, (UV&lt;350nm)*0.05, (UV&gt;=350nm)*0,90</TD><TD> </TD><TD> </TD><TD>n/a</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>414</TD><TD>bulb200W</TD><TD>Incandescent light bulb</TD><TD> </TD><TD> </TD><TD>1 m</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>415</TD><TD>k_UVR8_hv</TD><TD>Light dependent dissociation rate UVR8 dimer</TD><TD>2.08?·10<SUP>-3</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>from gels</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>416</TD><TD>k_UVR8_decay</TD><TD>Dimerization rate UVR8 monomer</TD><TD>8.4?·10<SUP>-10</SUP> nM<SUP>-1</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>estimate</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>417</TD><TD>KM_TetR</TD><TD>TetR repression coefficient</TD><TD>100 nM</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>418</TD><TD>n_TetR</TD><TD>TetR cooperativity coefficient</TD><TD>1</TD><TD> </TD><TD><SPAN class="eth_reference">[GarciaOjalvo2004]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>419</TD><TD>k_Ptet</TD><TD>Tet promoter expression strength</TD><TD>1.1 nM s<SUP>-1</SUP></TD><TD> </TD><TD>optimised</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>420</TD><TD>A</TD><TD>Basal expression fraction</TD><TD>0.05</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>421</TD><TD>n</TD><TD>Hill-like pABA cooperativity coefficient</TD><TD>10<SUP>-5</SUP></TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>422</TD><TD>k_deg</TD><TD>Protein degradation rate</TD><TD>3.85?·10<SUP>-5</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>423</TD><TD>KM_PabAB</TD><TD>PabAB Michaelis constant</TD><TD>960?·10<SUP>3</SUP> nM</TD><TD> </TD><TD><SPAN class="eth_reference">[Roux1992]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>424</TD><TD>k_cat</TD><TD>PabAB catalysis rate</TD><TD>0.65 s<SUP>-1</SUP></TD><TD> </TD><TD><SPAN class="eth_reference">[Roux1992]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>425</TD><TD>Chor0</TD><TD>Intracellular chorismate concentration</TD><TD>100 mM</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>426</TD><TD>k_out</TD><TD>pABA outflux rate</TD><TD>3.85?·10<SUP>-4</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>427</TD><TD>k_Cph8_hv</TD><TD>Light dependent activation rate</TD><TD>s<SUP>-1</SUP></TD><TD> </TD><TD>from photoinduction model</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>428</TD><TD>KM_LOV</TD><TD>LOV repression coefficient</TD><TD>142 nM</TD><TD> </TD><TD><SPAN class="eth_reference">[Strickland2007]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>429</TD><TD>KM_Cph8</TD><TD>Cph8 activation coefficient</TD><TD>1000 nM</TD><TD> </TD><TD>estimate</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>430</TD><TD>KM_LacI</TD><TD>LacI repression coefficient</TD><TD>800 nM</TD><TD> </TD><TD><SPAN class="eth_reference">[Basu2005]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>431</TD><TD>KM_cI</TD><TD>cI repression coefficient</TD><TD>8 nM</TD><TD> </TD><TD><SPAN class="eth_reference">[Basu2005]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>432</TD><TD>KM_TetR</TD><TD>TetR repression coefficient</TD><TD>100 nM</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>433</TD><TD>n_LacI</TD><TD>LacI cooperativity coefficient</TD><TD>2</TD><TD> </TD><TD><SPAN class="eth_reference">[Basu2005]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>434</TD><TD>n_cI</TD><TD>cI cooperativity coefficient</TD><TD>2</TD><TD> </TD><TD><SPAN class="eth_reference">[Basu2005]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>435</TD><TD>n_TetR</TD><TD>TetR cooperativity coefficient</TD><TD>1</TD><TD> </TD><TD><SPAN class="eth_reference">[GarciaOjalvo2004]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>436</TD><TD>n_Cph8</TD><TD>Cph8 cooperativity coefficient</TD><TD>1</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>437</TD><TD>n_LOV</TD><TD>LOV cooperativity coefficient</TD><TD>1</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>438</TD><TD>k_LOV_decay</TD><TD>Dark decay rate of active LOV</TD><TD>5.8?·10<SUP>-3</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD><SPAN class="eth_reference">[Drepper2007]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>439</TD><TD>k_Cph8_decay</TD><TD>Dark decay rate of active Cph8</TD><TD>5.8?·10<SUP>-3</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>estimate</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>440</TD><TD>k_Ptrp</TD><TD>Trp promoter expression strength</TD><TD>2.23 nM s<SUP>-1</SUP></TD><TD> </TD><TD>optimized</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>441</TD><TD>k_PompC</TD><TD>OmpC promoter expression strength</TD><TD>3.454?·10<SUP>-1</SUP> nM s<SUP>-1</SUP></TD><TD> </TD><TD>optimized</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>442</TD><TD>k_P_R</TD><TD>Lambda P_R expression strength</TD><TD>4.21?·10<SUP>-2</SUP> nM s<SUP>-1</SUP></TD><TD> </TD><TD>optimized</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>443</TD><TD>k_P_L</TD><TD>Lambda P_L expression strength</TD><TD>3.0?·10<SUP>-2</SUP> nM s<SUP>-1</SUP></TD><TD> </TD><TD>optimized</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>444</TD><TD>A</TD><TD>Basal expression fraction</TD><TD>0.05</TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>445</TD><TD>k_deg</TD><TD>Protein degradation rate</TD><TD>1.9?·10<SUP>-3</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD>assumed</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>446</TD><TD>ecoli_v</TD><TD>Volume of E.coli</TD><TD>2.0e-18 m3</TD><TD> </TD><TD><A href="http://bionumbers.hms.harvard.edu/Includes/KeyNumbersLinks.pdf" class="external text" rel="nofollow">Bionumbers</A></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>447</TD><TD>ecoli_extcoeff</TD><TD>Extinction coefficient of E.coli at 600nm</TD><TD>6.022e10 m2 mol-1</TD><TD> </TD><TD>Computed via OD600</TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>448</TD><TD>ecoli_absorption</TD><TD>Absorption spectrum E.coli</TD><TD> </TD><TD><SPAN class="eth_reference">[Kiefer2010]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>449</TD><TD>pABA_extcoeff</TD><TD>Extinction coefficient of pABA at 290nm</TD><TD>1.9e3 m2 mol-1</TD><TD> </TD><TD><SPAN class="eth_reference">[Quinlivan2003]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>450</TD><TD>pABA_absorption</TD><TD>Absorption spectrum pABA</TD><TD> </TD><TD><SPAN class="eth_reference">[EC2006]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>451</TD><TD>pABA_molarweight</TD><TD>Molar weight of pABA</TD><TD>137.14 g mol-1</TD><TD> </TD><TD><A href="http://www.sigmaaldrich.com/catalog/product/sigma/a9878?lang=de?®ion=DE" class="external text" rel="nofollow">Datasheet</A></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>452</TD><TD>layer_height</TD><TD>Height of sunscreen layer</TD><TD>2e-5 m</TD><TD> </TD><TD><SPAN class="eth_reference">[Vainio2001]</SPAN></TD><TD><A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>453</TD><TD>\(\alpha_{TetR} \)</TD><TD>TetR max. production rate</TD><TD>\(0.8~\mu \text{M}\cdot\text{min}^{-1} \)</TD><TD> </TD><TD>[5] Wilfried Weber, Markus Rimann, Manuela Spielmann, Bettina Keller, Marie Daoud-El Baba, Dominique Aubel, Cornelia C Weber &amp; Martin Fussenegger, Gas-inducible transgene expression in mammalian cells and mice, Nature Biotechnology, volume 22, number 11, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>454</TD><TD>\(\alpha_{FadR} \)</TD><TD>FadR max. production rate</TD><TD>\(100~\mu\text{M}\cdot\text{min}^{-1} \)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>455</TD><TD>\(\alpha_{GLP-1} \)</TD><TD>GLP-1 max. production rate</TD><TD>\(1.23~\mu\text{M}\cdot\text{min}^{-1} \)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>456</TD><TD>\(\alpha_{LacI} \)</TD><TD>LacI? max. production rate</TD><TD>\(0.8~\mu\text{M}\cdot\text{min}^{-1} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>457</TD><TD>\(k_{s1} \)</TD><TD>AHL <SUB>i</SUB> production rate</TD><TD>\(0.01~\text{min}^{-1} \)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>458</TD><TD>\(\rho_R \)</TD><TD>LuxR-AHL dimerization rate</TD><TD>\(0.5~\mu\text{M}^{-3}\cdot \text{min}^{-1} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>459</TD><TD>\(\beta_{TetR} \)</TD><TD>TetR repression coefficient</TD><TD>\(0.13~\mu\text{M} \)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>460</TD><TD>\(\beta_{FA} \)</TD><TD>FA repression coefficient</TD><TD>\(10~\mu\text{M} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>461</TD><TD>\(\beta_{FadR} \)</TD><TD>FadR repression coefficient</TD><TD>\(0.13~\mu\text{M} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>462</TD><TD>\(\beta_{LacI} \)</TD><TD>LacI repression coefficient</TD><TD>\(0.8~\mu\text{M} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>463</TD><TD>\(\beta_R \)</TD><TD>LuxR-AHL repression coefficient</TD><TD>\(0.01~\mu\text{M} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>464</TD><TD>\(\gamma_{TetR} \)</TD><TD>TetR degradation rate</TD><TD>\(0.0692~\text{min}^{-1} \)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>465</TD><TD>\(\gamma_{LacI} \)</TD><TD>LacI degradation rate</TD><TD>\(0.0231~\text{min}^{-1} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>466</TD><TD>\(\gamma_{GLP-1} \)</TD><TD>GLP-1 degradation rate</TD><TD>\(0.0731~\text{min}^{-1} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>467</TD><TD>\(\gamma_{LuxI} \)</TD><TD>LuxI degradation rate</TD><TD>\(0.0167~\text{min}^{-1} \)</TD><TD> </TD><TD>[3] MIT igem 2010</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>468</TD><TD>\(k_{s0} \)</TD><TD>AHL<SUB>i</SUB>? degradation rate</TD><TD>\(1~\text{min}^{-1} \)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>469</TD><TD>\(k_{se} \)</TD><TD>AHL<SUB>e</SUB>? degradation rate</TD><TD>\(1~\text{min}^{-1} \)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>470</TD><TD>\(\gamma_R \)</TD><TD>LuxR-AHL degradation rate</TD><TD>\(0.0231~\text{min}^{-1} \)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>471</TD><TD>\(\gamma_{RFP} \)</TD><TD>RFP degradation rate</TD><TD>\(0.0041~\text{min}^{-1} \)</TD><TD> </TD><TD>[4] Michael Halter, Alex Tona, Kiran Bhadriraju, Anne L. Plant, John T. Elliott, Automated Live Cell Imaging of Green Fluorescent Protein Degradation in Individual Fibroblasts, Cytometry Part A, Volume 71A Issue 10, 2007</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>472</TD><TD>\(n_1\)</TD><TD>FadR cooperativity coefficient</TD><TD>\(3\)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>473</TD><TD>\(n_2\)</TD><TD>FA cooperativity coefficient</TD><TD>\(2\)</TD><TD> </TD><TD>a.</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>474</TD><TD>\(n_3\)</TD><TD>LuxR-AHL cooperativity coefficient</TD><TD>\(3\)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>475</TD><TD>\(n_5\)</TD><TD>TetR cooperativity coefficient</TD><TD>\(2\)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>476</TD><TD>\(n_6\)</TD><TD>LacI cooperativity coefficient</TD><TD>\(4\)</TD><TD> </TD><TD>[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>477</TD><TD>\(\eta\)</TD><TD>AHL Diffusion rate across the cell membrane</TD><TD>\(2\)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>478</TD><TD>\(\eta\)<SUB>ext</SUB></TD><TD>Average diffusion rate for all cells</TD><TD>\(1\)</TD><TD> </TD><TD>[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>479</TD><TD>\([E]_t \)</TD><TD>Total active enzyme</TD><TD>\(50~\text{U}\text{mL}^{-1} \)</TD><TD> </TD><TD>[8] Bengt Borgstrom, Luminal Digestion of Fats, Handbook of Physiology, The Gastrointestinal System, Intestinal Absorption and Secretion, 1991 [9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973[11] M Lingumsky, E Granot, D Branski, H Stankiewicz, R Goldstein, Isolated lipase and colipase deficiency in two brothers, Gut, 1990</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>480</TD><TD>\(K_m \)</TD><TD>Reaction rate constant</TD><TD>\(47.9\)</TD><TD> </TD><TD>[6] Sulaiman Al-Zuhair, Masitah Hasan, K.B. Ramachandran, Kinetics of the enzymatic hydrolysis of palm oil by lipase, Process Biochemistry Volume 38, Issue 8, 2003 [7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>481</TD><TD>\(D_{FA} \)</TD><TD>FA Diffusion Constant</TD><TD>\( 6.46 \times 10^{-10}~\text{m}^2\text{s}^{-1} \)</TD><TD> </TD><TD>[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>482</TD><TD>\(d\)</TD><TD>Thickness of the unstirred water layer</TD><TD>\( 190~\mu\text{m} \)</TD><TD> </TD><TD>[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>483</TD><TD>\( K_{cat}\)</TD><TD>Catalytic Rate constant</TD><TD>\(1.3\times10^{-5}\text{min}^{-1}\)</TD><TD> </TD><TD>[6] Sulaiman Al-Zuhair, Masitah Hasan, K.B. Ramachandran, Kinetics of the enzymatic hydrolysis of palm oil by lipase, Process Biochemistry Volume 38, Issue 8, 2003 [7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>484</TD><TD>\( cd \)</TD><TD>Relative E. coli Cell Density</TD><TD>\( 0.1 \)</TD><TD> </TD><TD> </TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>485</TD><TD>\( D_{FA} \)</TD><TD>FA Diffusion Constant</TD><TD>\( 6.46 \times 10^{-10}\text{m}^2\text{s}^{-1} \)</TD><TD> </TD><TD>[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>486</TD><TD>\(D_{AHL} \)</TD><TD>AHL Diffusion Constant</TD><TD>\( 4.9\times10^{-6}~\text{cm}^2\text{s}^{-1} \)</TD><TD> </TD><TD>[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>487</TD><TD>\( \gamma_{AHL,ext} \)</TD><TD>AHL cell-external degradation</TD><TD>\( 8.0225\times10^{6}~\text{s}^{-1} \)</TD><TD> </TD><TD>Derived from 1 day half-life at pH 7. [7] [7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004</TD><TD><A href="https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters">https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters</A></TD><TD>NTU-Taida</TD></TR><TR><TD>488</TD><TD>D<SUB>AHL</SUB></TD><TD>AHL diffusion constant</TD><TD>4.9 x 10<SUP>-6</SUP> cm<SUP>2</SUP>/s</TD><TD> </TD><TD> Stewart P.S., 2003</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>489</TD><TD>C<SUB>agar</SUB></TD><TD>Reduced diffusion coefficient</TD><TD>0.9</TD><TD> </TD><TD>Fatin-Rouge <I>et al.</I>, 2004</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>490</TD><TD>α<SUB>AHL</SUB></TD><TD>AHL synthesis rate</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD> Garcia-Ojalvo <I>et. al.</I>, 2004</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>491</TD><TD>d<SUB>AHL</SUB></TD><TD>AHL degradation rate (intracellular)</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD>Basu <I>et al.</I>, 2005</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>492</TD><TD>d<SUB>AHL,e</SUB></TD><TD>AHL extracellular decay</TD><TD>4.8135 x 10<SUP>-4</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD>Horswill <I>et al.</I>, 2007</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>493</TD><TD>η<SUB>AHL</SUB></TD><TD>Diffusion rate across the cell membrane</TD><TD>2</TD><TD> </TD><TD> Garcia-Ojalvo <I>et. al.</I>, 2004</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>494</TD><TD>η<SUB>ext</SUB></TD><TD>Average diffusion rate for all cells</TD><TD>1.3333</TD><TD> </TD><TD> Garcia-Ojalvo <I>et. al.</I>, 2004</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>495</TD><TD>α<SUB>LuxI</SUB></TD><TD>LuxI synthesis rate</TD><TD>1 μM/min</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>496</TD><TD>d<SUB>LuxI</SUB></TD><TD>LuxI degradation rate</TD><TD>0.0167 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2010.igem.org/Team:MIT_tmodel"> MIT iGEM 2010</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>497</TD><TD>k</TD><TD>Cell growth rate</TD><TD>0.888 h<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL#growthRate">estimated from experimental data</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>498</TD><TD>α<SUB>LuxR</SUB></TD><TD>LuxR synthesis rate</TD><TD>0.005 μ M/min</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005 </A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>499</TD><TD>d<SUB>LuxR</SUB></TD><TD>LuxR degradation rate</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Manefield2002">Manefield <I>et al.</I>, 2002</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>500</TD><TD>ρ<SUB>R</SUB></TD><TD>LuxR/AHL dimerization</TD><TD>0.5 μM<SUP>-3</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>501</TD><TD>d<SUB>R</SUB></TD><TD>Dimer LuxR/AHL degradation rate</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>502</TD><TD>K<SUB>R</SUB></TD><TD>LuxR/AHL activation coefficient</TD><TD>0.013 nM</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5">estimated</A> from experimental data (scaled)</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>503</TD><TD>n</TD><TD>Hill coefficient</TD><TD>1</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>504</TD><TD>α<SUB>GFP</SUB></TD><TD>GFP synthesis rate</TD><TD>2 μM min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Basu2005">Basu <I>et al.</I>, 2005</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>505</TD><TD>d<SUB>GFP</SUB></TD><TD>GFP degradation rate</TD><TD>4.4432 x 10<SUP>-4</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Corish1999">Corish and Tyler-Smith, 1999 </A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>506</TD><TD>α<SUB>GusA</SUB></TD><TD>GusA synthesis rate</TD><TD>1 μM min<SUP>-1</SUP></TD><TD> </TD><TD>estimated</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>507</TD><TD>d<SUB>GusA</SUB></TD><TD>GusA degradation (half-life 55<SUP>o</SUP>C 2 hr)</TD><TD>9.6270<SUP>-5</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Jefferson1995">Jefferson, 1995</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>508</TD><TD>K<SUB>R1</SUB></TD><TD>AHL activation coefficient</TD><TD>4.45 nM</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5">estimated from experimental data</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>509</TD><TD>n<SUB>1</SUB></TD><TD>Hill coefficient</TD><TD>1.7</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5">estimated from experimental data</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>510</TD><TD>k<SUB>leaky</SUB></TD><TD>GusA basal expression</TD><TD>0.0375</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>511</TD><TD>α<SUB>AES</SUB></TD><TD>AES synthesis rate</TD><TD>1 μM min<SUP>-1</SUP></TD><TD> </TD><TD>estimated</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>512</TD><TD>d<SUB>AES</SUB></TD><TD>AES degradation (half-life 55<SUP>o</SUP>C 2 hr)</TD><TD>9.6270<SUP>-5</SUP> s<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter#Jefferson1995">Jefferson, 1995</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>513</TD><TD>K<SUB>R2</SUB></TD><TD>AHL activation coefficient</TD><TD>12555 nM</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5">estimated from experimental data</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>514</TD><TD>n<SUB>2</SUB></TD><TD>Hill coefficient</TD><TD>0.8</TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5">estimated from experimental data</A></TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>515</TD><TD>k<SUB>GFP</SUB></TD><TD>Hydrolases basal expression</TD><TD>0.0375</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">https://2013.igem.org/Team:ETH_Zurich/Parameter</A></TD><TD>ETH Zurich</TD></TR><TR><TD>516</TD><TD>Transcription rate of cI</TD><TD>4200/Gene Length (nM/min)</TD><TD>5.6</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>517</TD><TD>Translation rate of cI</TD><TD>2400RBS/Protein Length</TD><TD>9.6</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>518</TD><TD>Transcription rate of VIP</TD><TD>4200/Gene Length (nM/min)</TD><TD>1.74129353</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>519</TD><TD>Translation rate of VIP</TD><TD>2400RBS/Protein Length</TD><TD>2.985075</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>520</TD><TD>Transcription rate of GFP</TD><TD>4200/Gene Length (nM/min)</TD><TD>5.53359684</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>521</TD><TD>Translation rate of GFP</TD><TD>2400RBS/Protein Length</TD><TD>9.486166</TD><TD> </TD><TD><A href="https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters"><FONT color="blue"> https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters </FONT></A></TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>522</TD><TD>Degradation rate of cI (mRNA</TD><TD>Half life = 6.8 min, Division time = 30 min</TD><TD>0.18063836</TD><TD> </TD><TD>(Selinger, GW, et al., 2003)</TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>523</TD><TD>Degradation rate of cI (protein)</TD><TD>Half life &gt; 10 h; division time = 30 min</TD><TD>0.03885825</TD><TD> </TD><TD>(Varshavsky, (1997) and Tobias et al., 1991)</TD><TD><A href="https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters">https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters</A></TD><TD>CIDEB-UANL Mexico</TD></TR><TR><TD>524</TD><TD>α<SUB>LuxR</SUB></TD><TD>Production rate of LuxR</TD><TD>0.005 μMmin<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[20]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>525</TD><TD>k<SUB>RLux</SUB></TD><TD>Rate of formation of RLux from LuxAHL and LuxR</TD><TD>0.1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[19]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>526</TD><TD>k<SUB>-RLux</SUB></TD><TD>Dissociation rate of RLux</TD><TD>10 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[19]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>527</TD><TD>K<SUB>mLux</SUB></TD><TD>Lumped parameter for the Lux system</TD><TD>10 nM</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs#Parameters" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>528</TD><TD>d<SUB>LuxAHL</SUB></TD><TD>External degradation rate of LuxAHL (30C6HSL)</TD><TD>0.004 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs#Degradation" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>529</TD><TD>d<SUB>LuxR</SUB></TD><TD>Degradation rate of LuxR</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[21]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>530</TD><TD>d<SUB>RLux</SUB></TD><TD>Degradation rate of RLux</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[20]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>531</TD><TD>d<SUB>mRNABxb1</SUB></TD><TD>Degradation rate of mRNA<SUB>Bxb1</SUB></TD><TD>0.2773 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[22]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>532</TD><TD>d<SUB>Bxb1</SUB></TD><TD>Degradation rate of Bxb1</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>533</TD><TD>L<SUB>PLux</SUB></TD><TD>Leakiness after using riboswitch for P<SUB>lux</SUB></TD><TD>0.01463 nMmin<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>534</TD><TD>K<SUB>mRNABxb1</SUB></TD><TD>Rate of transcription of Bxb1</TD><TD>5 nMmin<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>535</TD><TD>k<SUB>Bxb1</SUB></TD><TD>Rate of formation of Bxb1</TD><TD>0.1 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>536</TD><TD>α<SUB>LasR</SUB></TD><TD>Production rate of LasR</TD><TD>0.005 μMmin<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[20]</A></SUP>(Assumed to be the same as Lux system)</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>537</TD><TD>k<SUB>RLas</SUB></TD><TD>Rate of formation of RLas from LasAHL and LasR</TD><TD>0.1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[19]</A></SUP> (Assumed to be the same as Lux system)</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>538</TD><TD>k<SUB>-RLas</SUB></TD><TD>Dissociation rate of RLas</TD><TD>10 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[19]</A></SUP>(Assumed to be the same as Lux system)</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>539</TD><TD>K<SUB>mLas</SUB></TD><TD>Lumped parameter for the Las system</TD><TD>0.45 nM</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>540</TD><TD>d<SUB>LasAHL</SUB></TD><TD>Degradation rate of LasAHL (30C12HSL)</TD><TD>0.004 min<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>541</TD><TD>d<SUB>LasR</SUB></TD><TD>Degradation rate of LasR</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[21]</A></SUP> (Assumed to be the same as Lux system)</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>542</TD><TD>d<SUB>RLas</SUB></TD><TD>Degradation rate of RLas</TD><TD>0.0231 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[20]</A></SUP> (Assumed to be the same as <PRE>Lux system)</PRE></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>543</TD><TD>d<SUB>mRNAϕc31</SUB></TD><TD>Degradation rate of mRNA<SUB>ϕc31</SUB></TD><TD>0.2773 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[22]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>544</TD><TD>d<SUB>ϕc31</SUB></TD><TD>Degradation rate of ϕC31</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>545</TD><TD>L<SUB>PLas</SUB></TD><TD>Leakiness after using riboswitch for P<SUB>las</SUB></TD><TD>0.02461 nMmin<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>546</TD><TD>K<SUB>mRNAϕc31</SUB></TD><TD>Rate of transcription of ϕc31</TD><TD>5 nMmin<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>547</TD><TD>k<SUB>ϕc31</SUB></TD><TD>Rate of formation of ϕc31</TD><TD>0.1 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>548</TD><TD>k<SUB>DBxb1</SUB></TD><TD>Dimerization rate of Bxb1</TD><TD>1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>549</TD><TD>k<SUB>-DBxb1</SUB></TD><TD>Dissociation rate of DBxb1</TD><TD>10<SUP>-6</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>550</TD><TD>k<SUB>SABxb1</SUB></TD><TD>Rate of formation of SA<SUB>Bxb1</SUB> from DBxb1 and SI<SUB>Bxb1</SUB></TD><TD>1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>551</TD><TD>k<SUB>-SABxb1</SUB></TD><TD>Dissociation rate of SA<SUB>Bxb1</SUB></TD><TD>10<SUP>-6</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>552</TD><TD>d<SUB>DBxb1</SUB></TD><TD>Degradation rate of DBxb1</TD><TD>0.02 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>553</TD><TD>k<SUB>Dϕc31</SUB></TD><TD>Dimerization rate of ϕc31</TD><TD>1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>554</TD><TD>k<SUB>-Dϕc31</SUB></TD><TD>Rate of dissociation of Dϕc31</TD><TD>10<SUP>-6</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>555</TD><TD>k<SUB>SAϕc31</SUB></TD><TD>Rate of formation of SA<SUB>ϕc31</SUB> from Dϕc31 and SI<SUB>ϕc31</SUB></TD><TD>1 nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>556</TD><TD>k<SUB>-SAϕc31</SUB></TD><TD>Rate of dissociation of SA<SUB>ϕc31</SUB></TD><TD>10<SUP>-6</SUP> min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">Fitted</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>557</TD><TD>d<SUB>Dϕc31</SUB></TD><TD>Degradation rate of Dϕc31</TD><TD>0.02 min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>558</TD><TD>k<SUB>ToffBxb1</SUB></TD><TD>Rate of flipping of T<SUB>on,i</SUB> to T<SUB>offBxb1</SUB></TD><TD>0.1 nM<SUP>-2</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>559</TD><TD>k<SUB>-ToffBxb1</SUB></TD><TD>Rate of flipping of T<SUB>offBxb1</SUB> to T<SUB>on,f</SUB></TD><TD>0.1 nM<SUP>-2</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>560</TD><TD>k<SUB>Toffϕc31</SUB></TD><TD>Rate of flipping of T<SUB>on,i</SUB> to T<SUB>offϕc31</SUB></TD><TD>0.1 nM<SUP>-2</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>561</TD><TD>k<SUB>-Toffϕc31</SUB></TD><TD>Rate of flipping of T<SUB>offϕc31</SUB> to T<SUB>on,f</SUB></TD><TD>0.1 nM<SUP>-2</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD>Assumed</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>562</TD><TD>k<SUB>mRNAGFP</SUB></TD><TD>Production rate of mRNA<SUB>GFP</SUB></TD><TD>5 nMmin<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>563</TD><TD>k<SUB>GFP</SUB></TD><TD>Rate of formation of folded GFP</TD><TD>1 min<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>564</TD><TD>d<SUB>mRNAGFP</SUB></TD><TD>Degradation rate of mRNA<SUB>GFP</SUB></TD><TD>0.2773 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[22]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>565</TD><TD>d<SUB>GFP</SUB></TD><TD>Degradation rate of GFP</TD><TD>0.0049 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs#Retrieving_degradation_rates" class="external text" rel="nofollow">Fitted to experimental data</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>566</TD><TD>k<SUB>mRNALasI</SUB></TD><TD>Production rate of mRNA<SUB>LasI</SUB></TD><TD>5 nMmin<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>567</TD><TD>k<SUB>LasI</SUB></TD><TD>Rate of formation of LasI</TD><TD>20 min<SUP>-1</SUP></TD><TD> </TD><TD>Estimated</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>568</TD><TD>d<SUB>mRNALasI</SUB></TD><TD>Degradation rate of mRNA<SUB>LasI</SUB></TD><TD>0.2773 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[22]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>569</TD><TD>d<SUB>LasI</SUB></TD><TD>Degradation rate of LasI</TD><TD>0.0167 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[21]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>570</TD><TD>k<SUB>LasAHL</SUB></TD><TD>Production rate of LasAHL (30C12HSL) from the LasI</TD><TD>0.04 min<SUP>-1</SUP></TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[19]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>571</TD><TD>θ</TD><TD>K<SUB>m</SUB> value for the production of mRNA<SUB>GFP</SUB> and mRNA<SUB>LasI</SUB></TD><TD>0.01 μM</TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[20]</A></SUP> (approximation)</TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>572</TD><TD>D<SUB>AHLext</SUB></TD><TD>Diffusion coefficient of extracellular AHL in liquid</TD><TD>4.9 10<SUP>-6</SUP> cm<SUP>2</SUP>/s</TD><TD> </TD><TD>Literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references#Stewart" title="Team:ETH Zurich/project/references">[27]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>573</TD><TD>D<SUB>m</SUB></TD><TD>Diffusion rate of AHL through the membrane</TD><TD>100 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel#Estimation" class="external text" rel="nofollow">Estimated</A> from literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[27]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>574</TD><TD>r</TD><TD>Growth rate of <I>E. coli</I> in our alginate beads</TD><TD>0.006 min<SUP>-1</SUP></TD><TD> </TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>575</TD><TD>α</TD><TD>Ratio of <I> E. coli</I> volume to the volume of one bead</TD><TD>100 min<SUP>-1</SUP></TD><TD> </TD><TD>V<SUB><I> E. coli</I></SUB> from literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references#Kaplan" title="Team:ETH Zurich/project/references">[28]</A></SUP>, bead volume from <A href="https://2014.igem.org/Team:ETH_Zurich/expresults#Diffusion" class="external text" rel="nofollow">experimental setup</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>576</TD><TD>N<SUB>0</SUB></TD><TD>Initial number of cells per bead</TD><TD>10<SUP>7</SUP> cells</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/expresults#Diffusion" class="external text" rel="nofollow">Experimental setup</A></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>577</TD><TD>N<SUB>m</SUB></TD><TD>Maximum number of cells per bead</TD><TD>8 10<SUP>7</SUP> cells</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel#Estimation" class="external text" rel="nofollow">Estimated</A> from literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references#Lars" title="Team:ETH Zurich/project/references"> [29]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>578</TD><TD>C<SUB>beads</SUB></TD><TD>Correction factor (a priori) for diffusion of LuxAHL in alginate beads</TD><TD>1</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel#Estimation" class="external text" rel="nofollow">Estimated</A> from literature <SUP><A href="https://2014.igem.org/Team:ETH_Zurich/project/references#Cronenberg" title="Team:ETH Zurich/project/references"> [30]</A></SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">https://2014.igem.org/Team:ETH_Zurich/modeling/parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>579</TD><TD>\(K_{d,\text{LuxRAHL}}\)</TD><TD>Dissociation constant between luxR and AHL</TD><TD>100 nM</TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Weber2013">Weber, 2013</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>580</TD><TD>\(\text{LuxR}_\text{tot}\)</TD><TD>Total concentration of LuxR</TD><TD>0.0025 μM</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>581</TD><TD>\(a_\mathrm{LuxI}\)</TD><TD>Maximal production rate of LuxI</TD><TD>1 μM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>582</TD><TD>\(a_\mathrm{LuxI,ribo}\)</TD><TD>Maximal production rate of LuxI</TD><TD>0.1 μM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich">ETHZ 2014</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>583</TD><TD>\(k_\mathrm{leaky}\)</TD><TD>Coefficient for leakiness dependency on LuxR concentration of P<SUB>LuxR</SUB> promoter</TD><TD>0.0375 μM<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter"> ETHZ 2013 </A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>584</TD><TD>\(K_\mathrm{a,LuxRAHL}\)</TD><TD>Activation coefficient of LuxRAHL</TD><TD>0.45 nM</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich">ETHZ 2014</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>585</TD><TD>\(K_\mathrm{LuxRAHL,ribo}\)</TD><TD>Activation coefficient of LuxRAHL in case of a riboregulated LuxR responsive promoter</TD><TD>285 nM</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich">ETHZ 2014</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>586</TD><TD>\(L_\mathrm{lux,ribo}\)</TD><TD>Leakiness after using riboswitch for P<SUB>lux</SUB></TD><TD>0.01463 nM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich">ETHZ 2014</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>587</TD><TD>\(n_\mathrm{lux}\)</TD><TD>Hill coefficient for LuxRAHL activation</TD><TD>1.7</TD><TD> </TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich">ETHZ 2014</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>588</TD><TD>\(d_\mathrm{LuxI}\)</TD><TD>Degradation rate of LuxI</TD><TD>0.0167 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2010.igem.org/Team:MIT_tmodel">MIT 2010</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>589</TD><TD>\(a_\mathrm{AHL}\)</TD><TD>Production rate of AHL</TD><TD>0.04 μM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Weber2013">Weber, 2013</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>590</TD><TD>\(d_\mathrm{AHL}\)</TD><TD>Degradation rate of AHL</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>591</TD><TD>\(v_\mathrm{AiiA}\)</TD><TD>Maximal conversion rate of AiiA</TD><TD>\(k_\mathrm{cat} \cdot C_\mathrm{AiiA} \)</TD><TD> </TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>592</TD><TD>\(k_\mathrm{cat}\)</TD><TD>Turnover number of AiiA</TD><TD>1.63 10<SUP>3</SUP>min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Wang2004">Wang, 2004</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>593</TD><TD>\(C_\mathrm{AiiA}\)</TD><TD>Concentration of AiiA</TD><TD>varied</TD><TD> </TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>594</TD><TD>\(K_\mathrm{M,AiiA}\)</TD><TD>Half-maximal rate substrate concentration of AiiA</TD><TD>2.95 10<SUP>3</SUP>μM</TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Wang2004">Wang, 2004</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>595</TD><TD>\(a_\mathrm{GFP}\)</TD><TD>Maximal production rate of GFP</TD><TD>2 μM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>596</TD><TD>\(d_\mathrm{GFP}\)</TD><TD>Degradation rate of GFP</TD><TD>0.01 min<SUP>-1</SUP></TD><TD> </TD><TD>estimated from doubling time of <I>E. coli</I></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>597</TD><TD>\(N_{d}\)</TD><TD>Number of <I> E. coli </I> in the doughnut</TD><TD>150</TD><TD> </TD><TD>Maximal number of <I>E. coli </I> that would fit on the surface</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>598</TD><TD>\(N_{b,max}\)</TD><TD>Maximum number of <I> E. coli </I> in the bulk</TD><TD>12798</TD><TD> </TD><TD>Considering the maximal OD is 2</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>599</TD><TD>\(V_{cell,d}\)</TD><TD>Volume around an <I> E. coli </I> in the doughnut</TD><TD>6 μm<SUP>3</SUP></TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>600</TD><TD>\(V_{cell,b,worst}\)</TD><TD>Volume around an <I> E. coli </I> in the bulk</TD><TD>78 μm<SUP>3</SUP></TD><TD> </TD><TD>Worst case, estimated from \(N_{b,max}\)</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>601</TD><TD>\(V_{cell,b,norm}\)</TD><TD>Volume around an <I> E. coli </I> in the bulk</TD><TD>1000 μm<SUP>3</SUP></TD><TD> </TD><TD>Normal case</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>602</TD><TD>\(\text{B}\)</TD><TD>\(\frac{Lac_\mathrm{ini}^2}{K_\mathrm{d,DLL}}\)</TD><TD>0.000001  -  4</TD><TD> </TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>603</TD><TD>\(\text{Lac}_{\text{ini}}\)</TD><TD>Initial concentration of lactate in the medium</TD><TD>0.1 μM  -  2 μM</TD><TD> </TD><TD>Low concentration of lactate in the medium</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>604</TD><TD>\(K_\mathrm{d,DLL}\)</TD><TD>Dissociation constant between the dimer of Lldr and Lactate</TD><TD>10 μM<SUP>2</SUP>  -  10000 μM<SUP>2</SUP></TD><TD> </TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>605</TD><TD>\(\alpha\)</TD><TD>Multiplication factor between the initial concentration of Lactate and Production of normal cells</TD><TD>1  -  150</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>606</TD><TD>\(F_\mathrm{C}\)</TD><TD>Fold change between Lactate production by cancer and normal cells</TD><TD>2  -  4</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>607</TD><TD>\(a_1\)</TD><TD>\(\frac{a_\mathrm{LacI}}{d_\mathrm{LacI}\cdot K_{RLacI}}\)</TD><TD>0.05  -  1000</TD><TD> </TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>608</TD><TD>\( a_\mathrm{LacI}\)</TD><TD>Maximal production rate of LacI</TD><TD>0.05 μM.min<SUP>-1</SUP>  -  1 μM.min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>609</TD><TD>\( d_\mathrm{LacI}\)</TD><TD>Degradation rate of LacI</TD><TD>0.01 min<SUP>-1</SUP>  -  0.1 min<SUP>-1</SUP></TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>610</TD><TD>\( K_\mathrm{R,LacI}\)</TD><TD>Repression coefficient of LacI</TD><TD>0.1 μM  -  10 μM</TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Basu2005">Basu, 2005</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>611</TD><TD>\( \gamma_1\)</TD><TD>\( \frac{L_\mathrm{2tot}}{K_\mathrm{R,L}}\)</TD><TD>5  -  10000</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>612</TD><TD>\( L_\mathrm{2tot}\)</TD><TD>Total concentration of LldR dimer</TD><TD>0.5 μM  -  10 μM</TD><TD> </TD><TD>estimated from paxdb</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>613</TD><TD>\( K_\mathrm{R,L}\)</TD><TD>Repression coefficient of LldR</TD><TD>0.001 μM  -  0.1 μM</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>614</TD><TD>\( \gamma_2\)</TD><TD>\(\frac{IPTG_{tot}}{K_{IL}}\)</TD><TD>0  -  500</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>615</TD><TD>\( \frac{a_1}{\gamma_2+1}\)</TD><TD> </TD><TD>0.001  -  1000</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>616</TD><TD>\( n_1\)</TD><TD>Hill coefficient of LldR</TD><TD>0.5  -  2.5</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>617</TD><TD>\( n_2\)</TD><TD>Hill coefficient of LacI</TD><TD>1.5  -  2.5</TD><TD> </TD><TD>estimated</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>618</TD><TD>\(D_\text{AHL,agar}\)</TD><TD>Diffusion coefficient of AHL in agar</TD><TD>\(3.0\times 10^{-10} m^2/s\)  -  \(4.41\times 10^{-10} m^2/s\)</TD><TD> </TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/References#Trovato2014">Trovato, 2014</A><A href="https://2015.igem.org/Team:ETH_Zurich/References#Fatin2004">Fatin-Rouge, 2004</A></TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters</A></TD><TD>ETH Zurich</TD></TR><TR><TD>619</TD><TD>c<SUB>1</SUB><SUP>max</SUP></TD><TD>0.01 [mM/h]</TD><TD>max. transcription rate of constitutive promoter (per gene)</TD><TD> </TD><TD>promoter no. J23105; Estimate</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>620</TD><TD>c<SUB>2</SUB><SUP>max</SUP></TD><TD>0.01 [mM/h]</TD><TD>max. transcription rate of LuxR-activated promoter (per gene)</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>621</TD><TD>l<SUP>hi</SUP></TD><TD>25</TD><TD>high-copy plasmid number</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>622</TD><TD>l<SUP>lo</SUP></TD><TD>5</TD><TD>low-copy plasmid number</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>623</TD><TD>a</TD><TD>0.01</TD><TD>basic production levels</TD><TD> </TD><TD>Estimate</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>624</TD><TD>d<SUB>LacI</SUB></TD><TD>2.31e-3 [1/s]</TD><TD>degradation of LacI</TD><TD> </TD><TD>Ref. <A href="http://www.biophysj.org/cgi/content/full/89/6/3873?maxtoshow=&amp;HITS=10&amp;hits=10&amp;RESULTFORMAT=&amp;searchid=1&amp;FIRSTINDEX=0&amp;volume=89&amp;firstpage=3873&amp;resourcetype=HWCIT" class="external text" rel="nofollow">[10] Tuttle et al.</A><I>&quot;Model-Driven Designs of an Oscillating Gene Network&quot;</I>, Biophys J 89(6):3873-3883, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>625</TD><TD>d<SUB>TetR</SUB></TD><TD>1e-5 [1/s] 2.31e-3 [1/s]</TD><TD>degradation of TetR</TD><TD> </TD><TD><A href="http://www.nature.com/nature/journal/v405/n6786/abs/405590a0.html" class="external text" rel="nofollow">[9] Becskei A and Serrano L</A><I>&quot;Engineering stability in gene networks by autoregulation&quot;</I>, Nature 405: 590-593, 2000<A href="http://www.biophysj.org/cgi/content/full/89/6/3873?maxtoshow=&amp;HITS=10&amp;hits=10&amp;RESULTFORMAT=&amp;searchid=1&amp;FIRSTINDEX=0&amp;volume=89&amp;firstpage=3873&amp;resourcetype=HWCIT" class="external text" rel="nofollow">[10] Tuttle et al.</A><I>&quot;Model-Driven Designs of an Oscillating Gene Network&quot;</I>, Biophys J 89(6):3873-3883, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>626</TD><TD>d<SUB>LuxR</SUB></TD><TD>1e-3 - 1e-4 [1/s]</TD><TD>degradation of LuxR</TD><TD> </TD><TD>Ref: <A href="http://dx.doi.org/10.1016/j.biosystems.2005.04.006" class="external text" rel="nofollow">[6] Goryachev AB et al.</A><I>&quot;Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants&quot;</I>, Biosystems 83(2-3):178-187, 2004</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>627</TD><TD>d<SUB>CI</SUB></TD><TD>7e-4 [1/s]</TD><TD>degradation of CI</TD><TD> </TD><TD>Ref. <A href="http://www.genetics.org/cgi/content/abstract/149/4/1633" class="external text" rel="nofollow">[7] Arkin A et al.</A><I>&quot;Stochastic kinetic analysis of developmental pathway bifurcation in phage λ-Infected Escherichia coli cells&quot;</I>, Genetics 149: 1633-1648, 1998</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>628</TD><TD>d<SUB>P22CII</SUB></TD><TD> </TD><TD>degradation of P22CII</TD><TD> </TD><TD> </TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>629</TD><TD>d<SUB>YFP</SUB></TD><TD>6.3e-3 [1/min]</TD><TD>degradation of YFP</TD><TD> </TD><TD>suppl. mat. to Ref. [8] corresponding to a half life of 110min. <A href="http://download.cell.com/supplementarydata/cell/107/6/739/DC1/index.htm" class="external text" rel="nofollow">[8] Colman-Lerner A et al.</A><I>&quot;Yeast Cbk1 and Mob2 Activate Daughter-Specific Genetic Programs to Induce Asymmetric Cell Fates&quot;</I>, Cell 107(6): 739-750, 2001 (supplementary material)</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>630</TD><TD>d<SUB>GFP</SUB></TD><TD>6.3e-3 [1/min]</TD><TD>degradation of GFP</TD><TD> </TD><TD>in analogy to YFP</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>631</TD><TD>d<SUB>RFP</SUB></TD><TD>6.3e-3 [1/min]</TD><TD>degradation of RFP</TD><TD> </TD><TD>in analogy to YFP</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>632</TD><TD>d<SUB>CFP</SUB></TD><TD>6.3e-3 [1/min]</TD><TD>degradation of CFP</TD><TD> </TD><TD>in analogy to YFP</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>633</TD><TD>K<SUB>LacI</SUB></TD><TD>0.1 - 1 [pM] 800 [nM]</TD><TD>LacI repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/content/full/100/13/7702?ck=nck" class="external text" rel="nofollow">[2] Setty Y et al.</A><I>&quot;Detailed map of a cis-regulatory input function&quot;</I>, P Natl Acad Sci USA 100(13):7702-7707, 2003 Ref. [12]</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>634</TD><TD>K<SUB>IPTG</SUB></TD><TD>1.3 [?µM]</TD><TD>IPTG-LacI repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/content/full/100/13/7702?ck=nck" class="external text" rel="nofollow">[2] Setty Y et al.</A><I>&quot;Detailed map of a cis-regulatory input function&quot;</I>, P Natl Acad Sci USA 100(13):7702-7707, 2003</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>635</TD><TD>K<SUB>TetR</SUB></TD><TD>179 [pM]</TD><TD>TetR repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/content/abstract/104/8/2643" class="external text" rel="nofollow">[1] Weber W et al.</A><I>&quot;A synthetic time-delay circuit in mammalian cells and mice&quot;</I>, P Natl Acad Sci USA 104(8):2643-2648, 2007</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>636</TD><TD>K<SUB>ATC</SUB></TD><TD>893 [pM]</TD><TD>ATC-TetR repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/content/abstract/104/8/2643" class="external text" rel="nofollow">[1] Weber W et al.</A><I>&quot;A synthetic time-delay circuit in mammalian cells and mice&quot;</I>, P Natl Acad Sci USA 104(8):2643-2648, 2007</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>637</TD><TD>K<SUB>LuxR</SUB></TD><TD>55 - 520 [nM]</TD><TD>LuxR activator dissociation constant</TD><TD> </TD><TD>Ref: <A href="http://dx.doi.org/10.1016/j.biosystems.2005.04.006" class="external text" rel="nofollow">[6] Goryachev AB et al.</A><I>&quot;Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants&quot;</I>, Biosystems 83(2-3):178-187, 2004</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>638</TD><TD>K<SUB>AHL</SUB></TD><TD>0.09 - 1 [?µM]</TD><TD>AHL-LuxR activator dissociation constant</TD><TD> </TD><TD>Ref:<A href="http://dx.doi.org/10.1016/j.biosystems.2005.04.006" class="external text" rel="nofollow">[6] Goryachev AB et al.</A><I>&quot;Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants&quot;</I>, Biosystems 83(2-3):178-187, 2004</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>639</TD><TD>K<SUB>CI</SUB></TD><TD>8 [pM] 50 [nM]</TD><TD>CI repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">[12] Basu S et al.</A><I>&quot;A synthetic multicellular system for programmed pattern formation&quot;</I>, Nature 434:1130-1134, 2005 starting with values of Ref.<A href="http://dx.doi.org/10.1016/j.biosystems.2005.04.006" class="external text" rel="nofollow">[6] Goryachev AB et al.</A><I>&quot;Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants&quot;</I>, Biosystems 83(2-3):178-187, 2004 and using Ref. <A href="http://ieeexplore.ieee.org/iel5/9711/30654/01416417.pdf" class="external text" rel="nofollow">[3] Braun D et al.</A><I>&quot;Parameter Estimation for Two Synthetic Gene Networks: A Case Study&quot;</I>, ICASSP 5:769-772, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>640</TD><TD>K<SUB>P22CII</SUB></TD><TD>0.577 [?µM]</TD><TD>P22CII repressor dissociation constant</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/reprint/99/2/679" class="external text" rel="nofollow">[11] McMillen LM et al.</A><I>&quot;Synchronizing genetic relaxation oscillators by intercell signaling&quot;</I>, P Natl Acad Sci USA 99(2):679-684, 2002</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>641</TD><TD>n<SUB>LacI</SUB></TD><TD>1;2</TD><TD>LacI repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://dx.doi.org/10.1016/j.jbiotec.2005.08.030" class="external text" rel="nofollow">[5] Iadevaia S and Mantzais NV</A><I>&quot;Genetic network driven control of PHBV copolymer composition&quot;</I>, J Biotechnol 122(1):99-121, 2006; Ref. [12]</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>642</TD><TD>n<SUB>IPTG</SUB></TD><TD>2</TD><TD>IPTG-LacI repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://dx.doi.org/10.1016/j.jbiotec.2005.08.030" class="external text" rel="nofollow">[5] Iadevaia S and Mantzais NV</A><I>&quot;Genetic network driven control of PHBV copolymer composition&quot;</I>, J Biotechnol 122(1):99-121, 2006</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>643</TD><TD>n<SUB>TetR</SUB></TD><TD>3</TD><TD>TetR repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://ieeexplore.ieee.org/iel5/9711/30654/01416417.pdf" class="external text" rel="nofollow">[3] Braun D et al.</A><I>&quot;Parameter Estimation for Two Synthetic Gene Networks: A Case Study&quot;</I>, ICASSP 5:769-772, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>644</TD><TD>n<SUB>ATC</SUB></TD><TD>2 (1.5-2.5)</TD><TD>ATC-TetR repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://ieeexplore.ieee.org/iel5/9711/30654/01416417.pdf" class="external text" rel="nofollow">[3] Braun D et al.</A><I>&quot;Parameter Estimation for Two Synthetic Gene Networks: A Case Study&quot;</I>, ICASSP 5:769-772, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>645</TD><TD>n<SUB>LuxR</SUB></TD><TD>2</TD><TD>LuxR activator Hill cooperativity</TD><TD> </TD><TD>Ref: <A href="http://dx.doi.org/10.1016/j.biosystems.2005.04.006" class="external text" rel="nofollow">[6] Goryachev AB et al.</A><I>&quot;Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants&quot;</I>, Biosystems 83(2-3):178-187, 2004</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>646</TD><TD>n<SUB>AHL</SUB></TD><TD>1</TD><TD>AHL-LuxR activator Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://ieeexplore.ieee.org/iel5/9711/30654/01416417.pdf" class="external text" rel="nofollow">[3] Braun D et al.</A><I>&quot;Parameter Estimation for Two Synthetic Gene Networks: A Case Study&quot;</I>, ICASSP 5:769-772, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>647</TD><TD>n<SUB>CI</SUB></TD><TD>2</TD><TD>CI repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html" class="external text" rel="nofollow">[12] Basu S et al.</A><I>&quot;A synthetic multicellular system for programmed pattern formation&quot;</I>, Nature 434:1130-1134, 2005</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR><TR><TD>648</TD><TD>n<SUB>P22CII</SUB></TD><TD>4</TD><TD>P22CII repressor Hill cooperativity</TD><TD> </TD><TD>Ref. <A href="http://www.pnas.org/cgi/reprint/99/2/679" class="external text" rel="nofollow">[11] McMillen LM et al.</A><I>&quot;Synchronizing genetic relaxation oscillators by intercell signaling&quot;</I>, P Natl Acad Sci USA 99(2):679-684, 2002</TD><TD><A href="https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters">https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters</A></TD><TD>ETHZ</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>