Team:Technion HS Israel/Constants Database
Technion 2015 HS Team's Wiki
Home
Medal Fulfillment Home
Project More about our project
Description Results Parts
Documents
Experiments Notebook safety Protocols
Modelling We take our math seriously
Model Overview Plasmid Loss Equations Parameters Results Documents
Software
Software Overview Search Engine Constants Database Simulation About the Database
Team We are a bunch of nerdy nerds
Members Attributions and contributions Gallery
Practices
Collaborations Practices Search Engine Modeling Tutorials
The Constants Database
Explanation and Documentation
name
description
value
unit
Source
url
team
1
k1A
Transcription Rate of Lac I gene
21
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
2
k1C
Transcription Rate of E7 + Imm gene
2.470588
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
3
d1A
Degradation Rate of Lac I mRNA
0.76246
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
4
d1C
Transcription Rate of E7 + Imm mRNA
0.0897
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
5
k2A
Translation Rate of Lac I mRNA
36
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
6
k2C
Translation Rate of E7 + Imm mRNA
4.23539
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
7
d2A,d2C
Protein Degradation Rate
0.03465
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
8
nC
Hill coefficeint for E7 + Imm
1
 
This is obtained on the assumption that one Repressor
Protein binds to one Lactose molecule complex
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
9
KC
Dissociation Constant for E7 + Imm
0.8
 
[1]
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
10
aC
Constitutive Portion for E7 + Imm
0.5
 
Estimate since a is between 0 and 1
Implication that Lactose may not be
a very strongly Regulated Promoter
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
11
k3AB
Complex Formation Rate Between
Lac I Repressor and Lactose
1
 
Estimate. This is based on the assumption
that the complex formation is only
dependent on the concentrations of
Lac I repressor and Lactose
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
12
kPF
Phosphorylation Rate of Ai-2
1
 
Estimate
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
13
kPB
Dephosphorylation Rate of Ai-2
1
 
Estimate
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
14
k1C
Transcription Rate of LsrR gene
4.402517
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
15
k1D
Transcription Rate of SupD gene
46.667
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
16
k1E
Transcription Rate of t7pTag gene
1.5556
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
17
k1F
Transcription Rate of Lysis gene
28
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
18
d1C
Degradation Rate of LsrR mRNA
0.159845
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
19
d1D
Degradation Rate of SupD mRNA
1.694359
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
20
d1E
Degradation Rate of t7pTag mRNA
0.056478
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
21
d1F
Degradation Rate of Lysis mRNA
1.0166
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
22
k2C
Translation Rate of LsrR Protein
7.54716
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
23
k2E
Translation Rate of t7 pTag Protein
2.6667
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
24
k2F
Translation Rate of Lysis Protein
48
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
25
d2C,d2E,d2F
Protein Degradation Rate
0.03465
 
Made using Earlier assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
26
nD
Hill coefficeint for SupD
50
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
27
nE
Hill coefficeint for t7 pTag
1
 
Estimate. It is assuming that
one molecule of iron ion is required
to activate the production of one t7mRNA
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
28
nF
Hill coefficeint for Lysis
1
 
Estimate. It is assumed that
one molecule of t7 is required
for activation of one Lysis mRNA
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
29
KD
Dissociation Constant for SupD
15
 
[2]
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
30
KE
Dissociation Constant for t7 pTag
1
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
31
KF
Dissociation Constant for Lysis
1
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
32
aD
Constitutive Portion for SupD
0.01
 
Trial and Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
33
aE
Constitutive Portion for t7 pTag
0.01
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
34
aF
Constitutive Portion for Lysis
0.0001
 
Trial and Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
35
k3BC
Complex Formation Rate Between
LsrR Repressor and Ai-2-Phosphorylated
0.01
 
Trial and Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
36
k3DE
Complex Formation Rate Between
SupD tRNA and t7 pTag mRNA
0.000000001
 
Trial and Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
37
S
Amount of Protein Kinase
28.747
 
From Earlier Assumptions
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
38
kf
Forward Reaction rate of Complex
1.2
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
39
kr
Reverse Reaction rate of Complex
0.5
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
40
r
Rate of Logistic Growth
0.01
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
41
A & B
Varying Capacity
1 & 0.9
 
Trial And Error
https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter
NTU-Singapore
42
kn
Population and culture
3.85e-3
 
[1] observed division rate
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
43
Nmax
Population and culture; carrying capacity
1e8
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
44
V
Population and culture;
1e-11
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
45
kdil
Population and culture; dilution rate
varied
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
46
kassoc
IP binding to receptor
5e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
47
kdissoc
IP binding to receptor
4.55e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
48
ks1
Synthesis of CRE1, YPD1, SKN7 species
6.16e-5
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
49
ks2
Synthesis of CRE1, YPD1, SKN7 species
6.00e-4
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
50
ks3
Synthesis of CRE1, YPD1, SKN7 species
2.46e-4
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
51
kdimer
Synthesis of CRE1, YPD1, SKN7 species
20
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
52
kp1
Phosphorylation/Dephosphorylation reactions;
0.1
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
53
kp2
Phosphorylation/Dephosphorylation reactions;
1243
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
54
kp3
Phosphorylation/Dephosphorylation reactions;
56
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
55
kp-3
Phosphorylation/Dephosphorylation reactions;
4.8
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
56
kdp1
Phosphorylation/Dephosphorylation reactions;
5.33e-2
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
57
kdp3
Phosphorylation/Dephosphorylation reactions;
2.89e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
58
kdp3
Phosphorylation/Dephosphorylation reactions;
4.80e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
59
kd1
Decay constants;
5.33e-2
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
60
kd2
Decay constants;
2.89e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
61
kd3
Decay constants;
4.80e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
62
kdip
Decay constants;
5e-4
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
63
kdgfp
Decay constants;
5.77e-3
 
[1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
64
kdtet
Decay constants;
3.85e-3
 
Cell division rate (assumed stable)
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
65
kdckx
Decay constants;
<=3.85e-3
 
Cell division rate, or lower if diffusion is significant.
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
66
kbgfp, kbtet, kbckx
Basal expression;
6e-6
 
adapted from [1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
67
ktdim
Basal expression;
1e3
 
arbitrary
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
68
kgfp
Induced expression;
6e-3
 
adapted from [1]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
69
ktet
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
70
kckx
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
71
Kg, Kt, Kc
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
72
αg, αt αc
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
73
Ki
Induced expression;
5e-6
 
Kass = 2e11 M-1 [2], [3]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
74
β
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
75
kipt4
Induced expression;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
76
ksip
IPT4 enzyme kinetics;
varied
 
-
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
77
CKX enzyme kinetics
IPT4 enzyme kinetics;
 
 
 
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
78
Km
IPT4 enzyme kinetics;
40 ?µM
 
[4]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
79
kcat
IPT4 enzyme kinetics;
0.5-1000 s-1
 
[5], [6]
https://2008.igem.org/Team:University_of_Ottawa/Modeling/Parameters
University of Ottawa
80
Length of e.coli
2μm
 
" University of Alberta Justification: Values come from the University of Alberta?€™s datasheet on MG1655,   produced to aid modelling. There is variability in size between strains -  for instance, AW405 length varies between 1.5?±0.2μm. But University of   Alberta datasheet is specifically for MG1655."
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
81
Diameter e. coli
 
0.8μm
 
University of Alberta
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
82
Shape e. coli
Actually rod-like. A circle with r= 0.714μm will have equivalent surface area to rod-like.
Circle r =0.714μm
 
University of Alberta
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
83
Mass e. coli
Given 1x10-12g for cell wet weight. Dividing this by gravity (=9.81) gives mass.
1.02x10-13g
 
University of Alberta
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
84
Swimming Speed e. coli
"  "
50μms-1
 
University of Alberta
A Method for Measuring Bacterial Chemotaxis Parameters in a Microcapillary Justification: University Alberta's datasheet gives 50μms-1. However, Swimming speed is affected by:
Viscosity (as viscosity increases the speed increases to some maximum, then decreases as the viscosity increases further. E.coli (strain:KL227 of length: 1.0μm and diameter: 0.5μm) maximum speed occurs at viscosity 8cp.
Temperature
Culture medium
Vary strain to strain.
Experimental methods
Various papers give different speeds for E. coli (most papers provide information on AW405 with a speed of ~20μms-1). The speed itself is nearly uniform during the run. The wet lab may need to measure this experimentally as we are unaware of the conditions that the speed for MG1655 was obtained. Alberta's value is higher than other values, but this probably because MG1655 is a motile strain.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
85
Aspartate concentration detected by E. coli
Related to Run Tumble Motion.
Over ~5 orders of magnitude, 10nM up to 10mM. Can detect changes of as little as ~0.1%
 
Competitive and Cooperative Interactions in Receptor Signalling Complexes Justification: E. coli detect small changes in concentration of 0.1% via temporal comparisons (4s) over a large range ( 10-8 to 10-3 ). Most computer simulations of chemotaxis are based on experimentally determined rates and concentrations. As a result they predict that the minimum detectable concentration of Aspartate is at ~200 nM. Experiments performed by Segall et al. in 1986, exposed tethered E. coli cells to iontophoretically delivered quantities of chemoattractant. These experiments indicated that a change in receptor occupancy of as little as 1/600 could produce an detectable change in swimming behaviour. With a Kd of 1 ?µM, this corresponds to a minimum detectable concentration of about 2 nM Aspartate. Wild type E. coli cells can detect <10nM of Asp and respond to Asp concentrations of upto 1mM,(responding to over ~5 orders of magnitude). M)
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
86
Temporal comparison of chemotactic gradient
Related to Run Tumble Motion.
4 seconds
 
Temporal comparisons in bacterial chemotaxis
Quantitative analysis of signalling networks
Motility of Escherichia coli cells in clusters formed by chemotactic aggregation Justification: The past second has positive weighting, the previous 3 seconds have negative weighting. E coli compares past and present concentrations by comparing the average occupancy of the receptors over the 4s. Models reflecting this system have been developed by Segall et al and Schnitzer, cells compare their average receptor occupancy between 4 and 1 s ago c1-4 to the average receptor occupancy during the last second c0-1 . Hence b= c0-1 - c1-4 . If b>0, the cell reduces the tumbling rate to Ttumbling from the ambient value T0 , 1s-1 e.g. b>0 don't tumble. b< 0, tumble at a rate of 1s-1
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
87
Tumbling angle
Related to Run Tumble Motion.
Shape parameter 4 Scale parameter 18.32 Location parameter -4.6
 
Chemotaxis in E. coli anaylsed by three-dimensions
AgentCell: a digital single-cell assay for bacterial chemotaxis
On Torque and tumbling in swimming Escherichia coli Justification: The tumble angle appears not to be dependant on the concentration gradient of chemoattractants/repellents. Nor is there correlation between the length of the run and the change in direction. The program uses a gamma distribution that fits the data collected by Berg and Brown. Several groups though, have observed that the tumble angle is not noramlly distributed but suggest that non-normality was only due to the experimental methods used e.g. in the capillary tube. Tumbling can cause a change in direction when as few as one flagella moves out of the bundle.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
88
Tumble angle direction
Related to Run Tumble Motion.
Bidirectional
 
Justification: Personal communication with Howard Berg. 'The direction is random, more or less, but there is a slight forward bias. It varies from tumble to tumble. The turn-angle distribution peaks at 68?° rather than 90?°. Tumbles turn out to be more complex than believed in 1972. Motors switch independently, and a tumble can occur if one or just a few motors change their directions of rotation. Tumbles are short, as judged by the tracking microscope, because they involve filament physics rather than motor physics: a transformation in polymorphic form, following motor reversal, from normal to semi-coiled. See Darnton, N.C., Turner, L., Rojevsky, S. and Berg, H.C. On torque and tumbling in swimming Escherichia coli, J. Bacteriol. 189, 1756-1764 (2007).'
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
89
Tumbling time
Related to Run Tumble Motion.
0.14?±0.19s
 
" Chemotaxis in E. Coli anaylsed by three-dimensional tracking Justification: Exponential distribution fitted (stated to be exponential  by Berg and Brown) using only the mean tumble length (not STDEV)."
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
90
Relationship between tumbling angle and time
Related to Run Tumble Motion.
 
 
 
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
91
Speed while Tumbling
Related to Run Tumble Motion.
0μm.s-1
 
Chemotaxis in E. Coli anaylsed by three-dimensional tracking Justification: Berg and Brown noted that AW405 slowed/stopped while tumbling.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
92
Drift during run
Related to Run Tumble Motion.
23?±23?°
 
" Chemotaxis in E. Coli anaylsed by three-dimensional tracking
Persistence of direction increases the drift velocity of run and tumble chemotaxis
Bray computer modelling Justification: Drift was observed. It is what would be expected from  rotational diffusion. (at 2.7cp at 32?C drift was 23?±23?°). Rotational   Brownian motion cause the cell to veer off course, so that in between   tumbles the probability density function f of the swimming direction e   evolves according to the Fokker-Planck equation. Drift velocity in steep  gradient of attractant ~7 ?µm.s-1(Berg & Turner, 1990. Note our model did not include the effects of drift"
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
93
Thrust
Related to Run Tumble Motion.
Down an Asp gradient 0.41pN, Up an Asp gradient 0.4387pN
 
Chemotaxis in E. Coli anaylsed by three-dimensional tracking
On Torque and Tumbling in Swimming E. coli
Swimming efficiency of bacterium E. coli. Justification: Average thrust =0.41pN. In the Berg and Brown paper it states that the speed of the bacteria up an aspartate chemotactic gradient increases by 7%. Therefore in our model we shall use the following; thrust DOWN the Asp gradient =0.41pN, up the Asp gradient = 0.4387pN. Data was obtained from 32 AW405s, a strain which has provided the majority of our previous parameters but is not as motile as MG1655. The value was obtained at 23?C in viscosity 0.93 and 3.07 cP for motility buffer and motility buffer with 0.18% methylcellulose, respectively. The standard deviation is not used as the speed is fixed at 50?µm.s-1 . 0.57pN is the average thrust generated in strain HCB30 (a non tumbling strain). The thrust value was obtained when the imposed flow (U) U=0 at 23?C. O.41pN was calculated using the resistance force theory treating the flagellar bundle as a single filament. The body was assumed to be prolate elipsoid using values roughly similar to ours, 2μm for length and 0.86μm for diameter.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
94
Isotropic run lengths
Related to Run Tumble Motion.
0.86?±1.18s
 
Chemotaxis in E. Coli anaylsed by three-dimensional tracking Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.4 Berg and Brown) The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
95
Run length UP Aspartate gradient
Related to Run Tumble Motion.
1.07?±1.80s
 
Chemotaxis in E. Coli anaylsed by three-dimensional tracking
UCSF wiki Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.6, Berg and Brown). The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
96
Run length DOWN Aspartate gradient
Related to Run Tumble Motion.
0.8?±1.38s
 
Chemotaxis in E. Coli anaylsed by three-dimensional tracking Justification: Exponential distribution fitted, this is only an approximate and does not fit exactly (see fig.6, Berg and Brown) The standard deviation is the standard deviation of the mean and has not been used in the exponential distribution
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
97
Viscosity
Related to Run Tumble Motion.
Viscosity of water is 1.002cP at 20?°C
 
The rotary motor of bacterial flagella. , On Torque and Tumbling in swimming Escherichia coli Justification: At present the medium being used by the lab is still be discussed. Currently though the medium most resembles water and therefore the water's viscosity value can be used. This allows us to assume that the medium is Newtonian (dilute aqueous medium that doesn?€™t contain long unbranched molecules such as methylcellulose or polyvinylpyrrolidone. Note that methlycellulose does not alter the run and tumble statistics, only bundle and motor rotation rates are affected by the addition of methylcellulose). If agar were to be used then the medium would be Non-Newtonian. Even though it would be Non- Newtonian John Hogan in passing said that we could assume it is Newtonian.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
98
CpMax
Maximal CpxR protein concentration
Unknown (varied in the program)
 
 
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
99
kCp
Maximal transcription rate of pCpxR promoter
0.075min-1
ESTIMATED
 
Surface Sensing and Adhesion of Escherichia Coli controlled by the cpx-signalling pathway.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
100
thetaCpx
Threshold for pCpxR promoter Hill Function
1 x 10-9 M
ESTIMATED
 
iGEM 2008 KULeuven
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
101
mCpx
Co-operativity of pCpxR promoter Hill function
1
 
 
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
102
dIm
Degradation rate of GFP mRNA
3.6 x 10-1 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
103
kIp
Rate of LuxI protein translation
9.6 x 10-1 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
104
dIp
Degradation rate of LuxI protein
1.67 x 10-2 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
105
dGm
Degradation rate of GFP protein
1.65 x 10-3 min-1
 
Efficient GFP mutations profoundly affect mRNA transcription and translation rates
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
106
kGp
Rate of GFP protein translation
2.4 x 10-1 min-1
 
Quantitative measurement of green fluorescent protein expression
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
107
dGp
Degradation rate of GFP protein
2.14 x 10-4 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
108
Aprod
AHL production rate per LuxI enzyme
3.6 min-1
 
[ Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
109
dA
Degradation rate of AHL molecule
1 x 10-2 min-1
 
A synthetic multicellular system for programmed pattern formation
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
110
DA
Diffusion coefficient of AHL
0.23s-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
111
kTp
Maximal Transcription rate of ptetR promoter
0.08min-1
 
iGEM 2007Imperial College London
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
112
dRm
Degradation rate of LuxR mRNA
3.6 x 10-1 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
113
kRp
Rate of Lux protein translation
9.6 x 10-1 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
114
dRp
Degration rate of LuxR protein
2.31 x 10-2 min-1
 
Systems Analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
115
kLuxR
Maximal transcription rate of LuxR promoter
0.11 min-1
 
iGEM 2008 KULeuven
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
116
thetaLuxR
Threshold for LuxR pR promoter Hill function
1.5 x 10-9 M
 
iGEM 2008 KULeuven
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
117
mLuxR
Co-operativity of LuxpR promoter Hill function
1.6
 
iGEM 2008 KULeuven
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
118
rhoC
LuxR/AHL dimerisation
0.5 micro M-3 min-1
 
A synthetic multicellular system for programmed pattern formation
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
119
dC
Degradation rate of mCherry mRNA
0.0231 min-1
 
A synthetic multicellular system for programmed pattern formation
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
120
dMm
Degradation rate of LuxR/AHL complex
1.65 x 10-3 min-1
 
Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
121
kMp
Rate of mCherry protein translation
2.4 x 10-1 min-1
 
Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
122
dMp
Degradation rate of mCherry protein
2.14 x 10-4 min-1
 
Due to difficulty in finding mCherry modelling parameters, exsisting GFP parameters have been used.
https://2008.igem.org/Team:BCCS-Bristol/Modeling-Parameters
BCCS-Bristol
123
Rate of production of HSL from LuxI
0.45
1/s
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
124
HSL
Rate of diffusion of HSL in/out of the cell
0.4
1/s
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
125
IPTG
Rate of diffusion of IPTG in/out of the cell
0.014
1/s
[3]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
126
pproduction
Number of plasmids
10
 
medium copy plasmid number; decided within the team
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
127
platch
Number of plasmids
10
 
medium copy plasmid number; decided within the team
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
128
plysis
Number of plasmids
10
 
medium copy plasmid number; decided within the team
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
129
panti-lysis
Number of plasmids
10
 
medium copy plasmid number; decided within the team
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
130
pQS
Number of plasmids
10
 
medium copy plasmid number; decided within the team
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
131
max_production
Maximal production rate of lux box promoter
0.44
pops
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
132
min_production
Minimal production rate of lux box promoter
0.013
pops
estimated
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
133
latch
Maximal production rate of latch promoter
0.28
pops
See Below
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
134
lysis
Maximal production rate of lysis promoter
0.0426
pops
See Below
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
135
anti-lysis
Maximal production rate of anti-lysis promoter
0.0066
pops
See Below
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
136
QS
Maximal production rate of QS promoter
0.018
pops
See Below
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
137
KLacI
Dissociation constant for LacI to LacO
700
m
See Explanation
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
138
KLacI-IPTG
Dissociation constant for IPTG to LacI
1200
m
See Explanation
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
139
KTetR
Dissociation constant for TetR to TetO
7000
m
See Explanation
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
140
KcI
Dissociation constant for cI to operon
7000
m
See Explanation
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
141
KP
Dissociation constant for P to lux box
700
m
See Explanation
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
142
nLacI
Hill coefficient
2
 
[10]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
143
ncI
Hill coefficient
2
 
[11]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
144
nTetR
Hill coefficient
3
 
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
145
nP
Hill coefficient
2
 
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
146
mRNA
Degradation of mRNA
0.00288
1/s
[2]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
147
X
Degradation of X
0.00000802
1/s
[5]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
148
Y
Degradation of Y
0.00000802
1/s
[5]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
149
λ-CI
Degradation of lambda CI
0.002888
1/s
[5]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
150
LacI
Degradation of LacI
0.001155
1/s
[8]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
151
TetR
Degradation of TetR
0.00288811
1/s
tagged; half-life of 4 min
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
152
Holin
Degradation of Holin
0.0002
1/s
estmated; half-life of an hour
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
153
Endolysin
Degradation of Endolysin
0.0002
1/s
estimated; half-life of an hour
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
154
Antiholin
Degradation of Antiholin
0.0002
1/s
estimated; half-life of an hour
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
155
LuxI
Degradation of LuxI
0.002888
1/s
tagged; half-life of 4 min
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
156
LuxR
Degradation of LuxR
0.0002
1/s
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
157
HSL
Degradation of HSL
0.00016667
1
[8]; value for AHL
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
158
Protein
Translation rate of Protein
0.1
1/s
[2]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
159
P
Rate of formation of the HSL-LuxI complex
0.0001
1/ms
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
160
-P
Rate of dissociation of the HSL-LuxI complex
0.003
1/s
[1]
https://2009.igem.org/Team:Aberdeen_Scotland/parameters
Aberdeen Scotland
161
k_1
Max Transcription rate of tRNA
46.67
nM/min
Assumption, sensitivity:0.19
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
162
k_2
Synthesis rate of Aa-tRNA
0.08
min^-1
Sensitivity: 0.09
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
163
k_3
Max Transcription rate of T7RNAP
1.5625
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
164
k_4
Max Translation rate of T7RNAP
2.68*0.05
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
165
k_5
Max Transcription rate of trigger CI
5.6
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
166
k_6
Transcription rate of bistable CI
5.6
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
167
k_6'
Transcription rate of bistable CI
1
nM/min
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
168
k_7
Transcription rate of T3RNAP
1.75
nM/min
Assumption, sensitivity:1.34
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
169
k_7'
Transcription rate of T3RNAP
1
nM/min
Sensitivity: Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
170
k_8
Translation rate of trigger CI
9.6*0.045
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
171
k_8'
Translation rate of bistable CI
9.6*0.3
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
172
k_9
Max Transcription rate of CI434
5.92
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
173
k_10
Transcription rate of CI434
10.14*0.5
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
174
k_11
Max Translation rate of T3RNAP
3*0.15
min^-1
Assumption, sensitivity:1.34
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
175
k_12
Max Transcription rate of GFP from Sal
5.25
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
176
k_12'
Max Trasncription rate of GFP from T3RNAP
5.25
nM/min
Assumption, sensitivity:1.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
177
k_13
Translation rate of GFP
9*0.6
min^-1
Assumption, sensitivity:1.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
178
k_s
rate of AND Gate 1
0.3
nM^-1
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
179
k_s'
rate of AND Gate 2
0.3
nM^-1
Sensitivity: 0.18
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
180
K_1
dissociation constant of AraC,tRNA
14
nM
Sensitivity: 0.03
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
181
K_3
dissociation constant of Sal,T7RNAP
0.5
nM
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
182
K_5
dissociation constant of T7RNAP,trigger CI
3
nM
Ref
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
183
K_6
dissociation constant of CI,bistable CI
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
184
K_6'
dissociation constant of CI434,bistable CI
50
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
185
K_7
dissociation constant of CI,T3RNAP
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
186
K_7'
dissociation constant of CI434,T3RNAP
50
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
187
K_9
dissociation constant of CI,CI434
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
188
K_12
dissociation constant of Sal,GFP
0.5
nM
Sensitivity: Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
189
K_12'
dissociation constant of T3RNAP,GFP
55
nM
Ref: The FEBS journal 2006,273:17
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
190
n_1
Hill co-effiency of AraC,tRNA
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
191
n_3
Hill co-effiency of Sal,T7RNAP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
192
n_5
Hill co-effiency of T7RNAP,CI
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
193
n_6
Hill co-effiency of CI,bistable CI
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
194
n_6'
Hill co-effiency of CI434,bistable CI
2
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
195
n_7
Hill co-effiency of CI,T3RNAP
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
196
n_7'
Hill co-effiency of CI434,T3RNAP
2
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
197
n_9
Hill co-effiency of CI,CI434
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
198
n_12
Hill co-effiency of Sal,GFP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
199
n_12'
Hill co-effiency of T3RNAP,GFP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
200
γ_1
Degradation rate of tRNA
1/30+1/60
min^-1
Since half life of tRNA is very long,
we decided to use 60 mins instead
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
201
γ_2
Degradation rate of Aa-tRNA
1/30+1/40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
202
γ_2'
Real Degradation rate of Aa-tRNA
ינו-40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
203
γ_3
Degradation rate of T7RNAP mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
204
γ_4
Degradation rate of T7RNAP
1/30+1/40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
205
γ_5
Degradation rate of trigger CI mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
206
γ_6
Degradation rate of bistable CI mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
207
γ_7
Degradation rate of T3RNAP mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
208
γ_8
Degradation rate of CI
1/30+1/44
min^-1
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
209
γ_9
Degradation rate of CI434 mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
210
γ_10
Degradation rate of CI434
1/30+1/11
min^-1
Ref: iGEM 2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
211
γ_11
Degradation rate of T3RNAP
1/30+1/30
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
212
γ_12
Degradation rate of GFP mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
213
γ_13
Degradation rate of GFP
1/30+1/60
min^-1
Since half life of GFP is very long,
we use 60 mins instead
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
214
k_1
Max Transcription rate of tRNA
46.67
nM/min
Assumption, sensitivity:1.41
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
215
k_2
Synthesis rate of Aa-tRNA
0.08
min^-1
Sensitivity: 0.81
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
216
k_3
Max Transcription rate of T7RNAP
1.5625
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
217
k_4
Max Translation rate of T7RNAP
2.68*0.05
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
218
k_5
Max Transcription rate of trigger CI
5.6
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
219
k_6
Transcription rate of bistable CI
5.6
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
220
k_6'
Transcription rate of bistable CI
1
nM/min
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
221
k_7
Transcription rate of P2
16.8
nM/min
Assumption, sensitivity:1.23
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
222
k_7'
Transcription rate of P2
1
nM/min
Sensitivity: Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
223
k_8
Translation rate of trigger CI
9.6*0.05
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
224
k_8'
Translation rate of bistable CI
9.6*0.5
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
225
k_9
Max Transcription rate of CI434
5.92
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
226
k_10
Transcription rate of CI434
10.14*1
min^-1
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
227
k_11
Max Translation rate of P2
28.8*0.0045
min^-1
Assumption, sensitivity:1.23
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
228
k_12
Max Transcription rate of GFP from Sal
5.25
nM/min
Assumption, sensitivity:0.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
229
k_12'
Max Trasncription rate of GFP from P2
5.25
nM/min
Assumption, sensitivity:1.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
230
k_13
Translation rate of GFP
9*0.6
min^-1
Assumption, sensitivity:1.00
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
231
k_s
rate of AND Gate 1
0.3
nM^-1
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
232
k_s'
rate of AND Gate 2
0.01
nM^-1
Sensitivity: 1.29
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
233
K_1
dissociation constant of AraC,tRNA
14
nM
Sensitivity: 0.2
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
234
K_3
dissociation constant of Sal,T7RNAP
0.5
nM
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
235
K_5
dissociation constant of T7RNAP,trigger CI
3
nM
Ref
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
236
K_6
dissociation constant of CI,bistable CI
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
237
K_6'
dissociation constant of CI434,bistable CI
50
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
238
K_7
dissociation constant of CI,P2
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
239
K_7'
dissociation constant of CI434,P2
50
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
240
K_9
dissociation constant of CI,CI434
40
nM
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
241
K_12
dissociation constant of Sal,GFP
0.5
nM
Sensitivity: 0
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
242
K_12'
dissociation constant of P2,GFP
35
nM
Sensitivity: 1.29
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
243
n_1
Hill co-effiency of AraC,tRNA
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
244
n_3
Hill co-effiency of Sal,T7RNAP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
245
n_5
Hill co-effiency of T7RNAP,CI
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
246
n_6
Hill co-effiency of CI,bistable CI
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
247
n_6'
Hill co-effiency of CI434,bistable CI
2
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
248
n_7
Hill co-effiency of CI,P2
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
249
n_7'
Hill co-effiency of CI434,P2
2
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
250
n_9
Hill co-effiency of CI,CI434
4
 
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
251
n_12
Hill co-effiency of Sal,GFP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
252
n_12'
Hill co-effiency of P2,GFP
2
 
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
253
γ_1
Degradation rate of tRNA
1/30+1/60
min^-1
Since half life of tRNA is very long,
we decided to use 60 mins instead
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
254
γ_2
Degradation rate of Aa-tRNA
1/30+1/40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
255
γ_2'
Real Degradation rate of Aa-tRNA
ינו-40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
256
γ_3
Degradation rate of T7RNAP mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
257
γ_4
Degradation rate of T7RNAP
1/30+1/40
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
258
γ_5
Degradation rate of trigger CI mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
259
γ_6
Degradation rate of bistable CI mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
260
γ_7
Degradation rate of P2 mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
261
γ_8
Degradation rate of CI
1/30+1/44
min^-1
Ref: iGEM2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
262
γ_9
Degradation rate of CI434 mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
263
γ_10
Degradation rate of CI434
1/30+1/11
min^-1
Ref: iGEM 2007 PKU Team
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
264
γ_11
Degradation rate of P2
1/30+1/30
min^-1
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
265
γ_12
Degradation rate of GFP mRNA
1/30+1/4.4
min^-1
Assumption
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
266
γ_13
Degradation rate of GFP
1/30+1/60
min^-1
Since half life of GFP is very long,
we use 60 mins instead
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
PKU Beijing
267
cpLac
maximum transcription rate (M/min)
5E-10
 
[1]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
268
cpTet
maximum transcription rate (M/min)
0.00000015
 
[2]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
269
cpλ
maximum transcription rate (M/min)
0.00000015
 
estimate
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
270
K50IPTG
dissociation constant (M)
0.0000013
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
271
K50LacI
dissociation constant (M)
0.0000008
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
272
K50TetR
dissociation constant (M)
1.79E-10
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
273
K50CI
dissociation constant (M)
0
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
274
nIPTG
Hills coefficient
2
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
275
nLacI
Hills coefficient
2
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
276
nTetR
Hills coefficient
3
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
277
nCI
Hills coefficient
2
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
278
dLacI
degradation rate (M/min)
0.1386
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
279
dTetR
degradation rate (M/min)
0.1386
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
280
dCI
degradation rate (M/min)
0.042
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
281
dRFP
degradation rate (M/min)
0.0063
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
282
dGFP
degradation rate (M/min)
0.0063
 
[3]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
283
dmRNA
degradation rate (M/min)
0.029
 
[4]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
284
α
translation rate (translations/min/mRNA), depends on growth rate (a default value of 30 is used)
16 - 57
 
[5]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
285
kIPTG
rate constant for IPTG diffusion into cell
0.92
 
[6]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
286
A
Surface area available (mm2)
1735
 
Area of 0.2μm filter used in conjugation tests.
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
287
r0
initial colony radius (μm)
0.8
 
[7]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
288
gn
specific growth rate (1/hr)
0.99
 
Determined experimentally for R751 containing cells.
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
289
gr
colony radial specific growth rate (μm/hr)
30
 
[8]
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
290
Nd
initial number of donors
10000
 
Estimate
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
291
Nr
initial number of donors
10000
 
Estimate
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
292
λ
intensity (CFU/mm2)
5.76
 
Calculated using Nr/A.
https://2009.igem.org/Team:TUDelft/Modeling_Parameters
TUDelft
293
KLacI
LacI repressor dissociation constant
0.1 - 1 [pM] OR 800 [nM]
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
294
KIPTG
IPTG-LacI repressor dissociation constant
1.3 [?µM]
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
295
KtetR
TetR repressor dissociation constant
179 [pM]
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
296
KcI
cI repressor dissociation constant
8 [pM] OR 50 [nM]
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
297
KHSL
HSL-LuxR activator dissociation constant
0.09 - 1 [?µM]
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
298
KLacI
Dissociation constant for LacI to LacO DNA site
~1*10 -12 M OR ~1.8*10-12 M
 
Mitchel Lewis (2005) The Lac repressor. C. R. Biologies 328 (2005) 521?€“548; Falcon C.M and Matthews K.S. (2000) Operator DNA sequence Variation Enhances High Affinity Binding by Hinge Helix Mutants of Lactose Repressor Protein. Biochemistry. 39, 11074-11084
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
299
KIPTG
Dissociation constant for IPTG to LacI
1*10-6 M
 
"Uri Alon, An introduction to systems Biology, p244
"
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
300
KtetR
Dissociation constant for TetR to TetO
(5.6 ?± 2) ?— 10-9 M OR 1.53*10-8 M
 
Nucleic Acids Res. 2004; 32(2): 842?€“847. Two mutations in the tetracycline repressor change the inducer anhydrotetracycline to a corepressor Annette Kamionka, Joanna Bogdanska-Urbaniak, Oliver Scholz, and Wolfgang Hillen; Volume 272, Number 11, Issue of March 14, 1997 pp. 6936-6942, The Role of the Variable Region in Tet Repressor for Inducibility by Tetracycline, Christian Berens , Dirk Schnappinger and Wolfgang Hillen
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
301
KcI
Dissocitation constant for cI to DNA site
50 * 10-9 M
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Aberdeen Scotland
302
αTetR
TetR max. production rate
3.93 μM/min, updated to 1 μM/min
 
[5]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
303
αC
CI max. production rate
1 μM/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
304
αL1
LacI max. production rate
1 μM/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
305
αL2
LacIM1 max. production rate
1 μM/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
306
αG
GFP max. production rate
2 μM/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
307
αLuxI
LuxI max. production rate
1 μM/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
308
ks1
AHLi production rate
0.01 1/min
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
309
ρR
LuxR-AHL dimerization rate
0.5 1/(μM3*min)
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
310
αRFP
RFP max. production rate
1 μM/min
 
assumption
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
311
βAcA
Acetaldehyde repression coefficient
7.124 μM
 
[5]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
312
βT
TetR repression coefficient
0.01 μM, updated to 0.1 μM/min
 
assumption
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
313
βC
CI repression coefficient
0.008 μM
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
314
βL
LacI repression coefficient
0.8 μM
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
315
θR
LuxR-AHL repression coefficient
0.01 μM
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
316
γTetR
TetR degradation rate
0.0692 1/min
 
assumption
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
317
γC
CI degradation rate
0.0692 1/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
318
γL
LacI degradation rate
0.0231 1/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
319
γG
GFP degradation rate
0.0692 1/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
320
γLuxI
LuxI degradation rate
0.0167 1/min
 
[3]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
321
ks0
AHLi degradation rate
1 1/min
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
322
kse
AHLe degradation rate
1 1/min
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
323
γR
LuxR-AHL degradation rate
0.0231 1/min
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
324
γRFP
RFP degradation rate
0.0041 1/min
 
[4]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
325
n1
LacI cooperativity coefficient
2
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
326
n2
CI cooperativity coefficient
2
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
327
n3
TetR cooperativity coefficient
2
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
328
n4
LuxR-AHL cooperativity coefficient
1
 
[1]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
329
n5
AlcR-AcA cooperativity coefficient
1.352
 
[5]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
330
n7
XylR cooperativity coefficient
5
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
331
η
Diffusion rate across the cell membrane
2
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
332
ηext
Average diffusion rate for all cells
1
 
[2]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
333
N
Number of cells
1
 
assumption
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
334
aXylR
XylR max. production rate
95.5 μM/min
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
335
γXylRi
XylRi degradation rate
0.03553 1/min
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
336
γXylRa
XylRa degradation rate
0.04527 1/min
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
337
KXylR
XylR activation coefficient
1.419*10-3 μM
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
338
rXylR
XylRi oligomerization constant
0.04315e-3 1/(μM*min)
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
339
rRXylR
XylRa dissociation constant
0.1301e-3 1/(μM*min)
 
[6]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
340
zend
Microfluidics channel length
5 cm
 
Choice in diffusion model
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
341
RChannel
Microfluidics channel radius
1 mm
 
Choice in diffusion model
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
342
VeColi
E. coli average volume
2 μm3
 
[13] [14]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
343
cdOD600, undiluted
E. coli Cell Density at OD600 = 1
1e9 1/ml
 
[13] [16]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
344
cDilution
Dilution of cell culture in microfludics channel
01-אוג
 
Choice in diffusion model
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
345
cdRelOD600, undiluted
Relative E. coli Cell Density at OD600 = 1, undiluted
0.001
 
Derived from VeColi, cdOD600, undiluted
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
346
cdRelOD600, diluted
Relative E. coli Cell Density at OD600 = 1, diluted
0.000125
 
Derived from cdRelOD600, undiluted, cDilution
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
347
CAgarose
Scaling coefficient for diffusion of small molecules in agarose instead of water
0.9
 
assumption
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
348
DAcAl
Acetaldehyde Diffusion Constant
1.23e-5 cm2/s
 
[8]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
349
mAcAl
Acetaldehyde Molecular Weight
44.05316 u
 
[8]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
350
Kcat,AcAl
Specific enzyme activity of acetaldehyde dehydrogenase in E. coli
14.1 μmol/(min*mg)
 
[10] [11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
351
[E]T
Mean acetaldehyde dehydrogenase enzyme concentration in E. coli
1.6659 kg/m3
 
Estimation from cell extract , [11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
352
vmax,AcAl
Maximum reaction rate for acetaldehyde dehydrogenase degrading acetaldehyde
0.02349 M/min
 
Estimation from cell extract , [11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
353
KM,AcAl
Michaelis-Menten constant for acetaldehyde dehydrogenase degrading acetaldehyde
10 μM
 
[10] [11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
354
mADH
Molecular mass of acetaldehyde dehydrogenase
33442 u
 
[12]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
355
nADH
Mean number of acetaldehyde dehydrogenase protein molecules per E. coli cell
28512.2, rounded to 30000
 
Estimation from cell extract , [11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
356
[ADH]Extract
acetaldehyde dehydrogenase concentration in E. coli cell extract
19 mg/ml
 
[11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
357
dilExtract
E. coli cell extract dilution
01-דצמ
 
[11]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
358
DAHL
AHL Diffusion Constant
4.9e-6 cm2/s
 
[9]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
359
mAHL
AHL Molecular Weight
297.38990 u
 
[15]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
360
γAHL,ext
AHL cell-external degradation
8.0225e-006/s
 
Derived from 1 day half-life at pH 7 [7]
https://2011.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
361
Concentration of X
NA
molecules
per cell
Notation convention
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
362
Delay due to protein X production and maturation
NA
s
Notation convention
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
363
Maximal production rate of pVeg promoter (constitutive)
0.02
molecules.s-1
or pops
Estimated, see the justification
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
364
Maximal production rate of pHyperSpank promoter
0.02
molecules.s-1
or pops
Estimated, see the justification
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
365
Maximal production rate of pT7 promoter
0.02
molecules.s-1
or pops
Estimated, see the justification
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
366
Maximal production rate of pComK promoter
0.049
molecules.s-1
or pops
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
367
Maximal production rate of pComS promoter
0.057
molecules.s-1
or pops
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
368
Maximal production rate of pComG promoter
0.02
molecules.s-1
or pops
Estimated, see the justification
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
369
Basal production rate of pComK promoter
0.0028
molecules.s-1
or pops
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
370
Basal production rate of pComG promoter
0.028
molecules.s-1
or pops
Is roughly one order of magnitude higher than production rate of pComK [4]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
371
Dissociation constant for IPTG to LacI
1200
molecules
per cell
Aberdeen 2009 wiki
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
372
Dissociation constant for LacI to LacO (pLac)
700
molecules
per cell
Aberdeen 2009 wiki
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
373
Dissociation constant for T7 RNA polymerase to pT7
10
molecules
per cell
We used the classic assumption 1nM=1 molecule per cell and [1]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
374
Dissociation constant for ComK to pComK
110
molecules
per cell
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
375
Dissociation constant for ComK to pComS
100
molecules
per cell
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
376
ComK concentration for half maximal degradation
500
molecules
per cell
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
377
ComS concentration for half maximal degradation
50
molecules
per cell
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
378
Hill coefficient for LacI/IPTG interaction
2
NA
Aberdeen 2009 wiki
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
379
Hill coefficient for LacI/pHyperSpank interaction
2
NA
Aberdeen 2009 wiki
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
380
Hill coefficient for ComK/pComK and ComK/pComG (positive feedback) interaction
2
NA
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
381
Hill coefficient for ComK/pComS (negative feedback) interaction
5
NA
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
382
Translation rate of proteins
0.9
s-1
Estimated, see the justification
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
383
Dilution rate in exponential phase
2.88x10-4
s-1
Calculated with a 40 min generation time. See explanation
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
384
Unrepressed degradation rate of ComK
1.4x10-3
s-1
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
385
Unrepressed degradation rate of ComS
1.4x10-3
s-1
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
386
Degradation rate of mRNA
2.88x10-3
s-1
[4]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
387
Degradation rate of GFP
10-4
s-1
BioNumbers
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
388
Degradation rate of RFP
10-4
s-1
Estimated equal to GFP degradation rate
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
389
Delay due to CFP production and maturation
360
s
Estimated equal to GFP delay (similar molecules)
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
390
Delay due to YFP production and maturation
360
s
Estimated equal to GFP delay (similar molecules)
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
391
Delay due to ComK production and maturation
300
s
Arbitrary
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
392
Delay due to ComS production and maturation
300
s
Arbitrary
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
393
Delay for ComS repression by ComK
714
s
[3]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
394
Delay due tT7 RNA polymerase production and maturation
300
s
[2]
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
395
Delay due GFP production and maturation
360
s
BioNumbers
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
396
Delay due RFP production and maturation
360
s
Estimated equal to GFP delay (similar molecules)
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
397
Delay due to mRNA production
30
s
BioNumbers with an approximation: all our contructs are around 1-2kb
https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters
Paris Bettencourt
398
L
tail length
0.8
cm
[1]
https://2012.igem.org/Team:Evry/parameters
Evry
399
l
max tail width
0.07
cm
[1]
https://2012.igem.org/Team:Evry/parameters
Evry
400
r
head radius
0.3
cm
[1]
https://2012.igem.org/Team:Evry/parameters
Evry
401
Tskin
skin thickness
35
μm
[1]
https://2012.igem.org/Team:Evry/parameters
Evry
402
Vtail
tail volume
0.001
cm3
here
https://2012.igem.org/Team:Evry/parameters
Evry
403
Vhead
head volume
0.1131
cm3
here
https://2012.igem.org/Team:Evry/parameters
Evry
404
V
tadpole volume
0.1141
cm3
here
https://2012.igem.org/Team:Evry/parameters
Evry
405
Vskin
skin volume
0.0041
cm3
here
https://2012.igem.org/Team:Evry/parameters
Evry
406
Vreceptor
receptor compartment volume
xxxx
cm3
here
https://2012.igem.org/Team:Evry/parameters
Evry
407
PX<->Y
permeability between tissues X and Y
10-5
cm.min-1
[3]
https://2012.igem.org/Team:Evry/parameters
Evry
408
Stadpole
sum of the external surfaces of the tail and the head
1.219
cm2
here
https://2012.igem.org/Team:Evry/parameters
Evry
409
Sskin
skin internal surface
1.1839
cm2
here
https://2012.igem.org/Team:Evry/parameters
Evry
410
Sreceptor
blood-receptor exchange surface
0.0781
cm2
here
https://2012.igem.org/Team:Evry/parameters
Evry
411
Dauxin
auxin diameter
0.766
nm
[2]
https://2012.igem.org/Team:Evry/parameters
Evry
412
sun
natural sun light
 
 
n/a
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
413
room
sun*0.3, (UV<350nm)*0.05, (UV>=350nm)*0,90
 
 
n/a
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
414
bulb200W
Incandescent light bulb
 
 
1 m
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
415
k_UVR8_hv
Light dependent dissociation rate UVR8 dimer
2.08?·10-3 s-1
 
from gels
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
416
k_UVR8_decay
Dimerization rate UVR8 monomer
8.4?·10-10 nM-1 s-1
 
estimate
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
417
KM_TetR
TetR repression coefficient
100 nM
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
418
n_TetR
TetR cooperativity coefficient
1
 
[GarciaOjalvo2004]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
419
k_Ptet
Tet promoter expression strength
1.1 nM s-1
 
optimised
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
420
A
Basal expression fraction
0.05
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
421
n
Hill-like pABA cooperativity coefficient
10-5
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
422
k_deg
Protein degradation rate
3.85?·10-5 s-1
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
423
KM_PabAB
PabAB Michaelis constant
960?·103 nM
 
[Roux1992]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
424
k_cat
PabAB catalysis rate
0.65 s-1
 
[Roux1992]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
425
Chor0
Intracellular chorismate concentration
100 mM
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
426
k_out
pABA outflux rate
3.85?·10-4 s-1
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
427
k_Cph8_hv
Light dependent activation rate
s-1
 
from photoinduction model
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
428
KM_LOV
LOV repression coefficient
142 nM
 
[Strickland2007]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
429
KM_Cph8
Cph8 activation coefficient
1000 nM
 
estimate
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
430
KM_LacI
LacI repression coefficient
800 nM
 
[Basu2005]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
431
KM_cI
cI repression coefficient
8 nM
 
[Basu2005]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
432
KM_TetR
TetR repression coefficient
100 nM
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
433
n_LacI
LacI cooperativity coefficient
2
 
[Basu2005]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
434
n_cI
cI cooperativity coefficient
2
 
[Basu2005]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
435
n_TetR
TetR cooperativity coefficient
1
 
[GarciaOjalvo2004]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
436
n_Cph8
Cph8 cooperativity coefficient
1
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
437
n_LOV
LOV cooperativity coefficient
1
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
438
k_LOV_decay
Dark decay rate of active LOV
5.8?·10-3 s-1
 
[Drepper2007]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
439
k_Cph8_decay
Dark decay rate of active Cph8
5.8?·10-3 s-1
 
estimate
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
440
k_Ptrp
Trp promoter expression strength
2.23 nM s-1
 
optimized
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
441
k_PompC
OmpC promoter expression strength
3.454?·10-1 nM s-1
 
optimized
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
442
k_P_R
Lambda P_R expression strength
4.21?·10-2 nM s-1
 
optimized
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
443
k_P_L
Lambda P_L expression strength
3.0?·10-2 nM s-1
 
optimized
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
444
A
Basal expression fraction
0.05
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
445
k_deg
Protein degradation rate
1.9?·10-3 s-1
 
assumed
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
446
ecoli_v
Volume of E.coli
2.0e-18 m3
 
Bionumbers
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
447
ecoli_extcoeff
Extinction coefficient of E.coli at 600nm
6.022e10 m2 mol-1
 
Computed via OD600
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
448
ecoli_absorption
Absorption spectrum E.coli
 
[Kiefer2010]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
449
pABA_extcoeff
Extinction coefficient of pABA at 290nm
1.9e3 m2 mol-1
 
[Quinlivan2003]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
450
pABA_absorption
Absorption spectrum pABA
 
[EC2006]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
451
pABA_molarweight
Molar weight of pABA
137.14 g mol-1
 
Datasheet
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
452
layer_height
Height of sunscreen layer
2e-5 m
 
[Vainio2001]
https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
453
\(\alpha_{TetR} \)
TetR max. production rate
\(0.8~\mu \text{M}\cdot\text{min}^{-1} \)
 
[5] Wilfried Weber, Markus Rimann, Manuela Spielmann, Bettina Keller, Marie Daoud-El Baba, Dominique Aubel, Cornelia C Weber & Martin Fussenegger, Gas-inducible transgene expression in mammalian cells and mice, Nature Biotechnology, volume 22, number 11, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
454
\(\alpha_{FadR} \)
FadR max. production rate
\(100~\mu\text{M}\cdot\text{min}^{-1} \)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
455
\(\alpha_{GLP-1} \)
GLP-1 max. production rate
\(1.23~\mu\text{M}\cdot\text{min}^{-1} \)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
456
\(\alpha_{LacI} \)
LacI? max. production rate
\(0.8~\mu\text{M}\cdot\text{min}^{-1} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
457
\(k_{s1} \)
AHL i production rate
\(0.01~\text{min}^{-1} \)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
458
\(\rho_R \)
LuxR-AHL dimerization rate
\(0.5~\mu\text{M}^{-3}\cdot \text{min}^{-1} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
459
\(\beta_{TetR} \)
TetR repression coefficient
\(0.13~\mu\text{M} \)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
460
\(\beta_{FA} \)
FA repression coefficient
\(10~\mu\text{M} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
461
\(\beta_{FadR} \)
FadR repression coefficient
\(0.13~\mu\text{M} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
462
\(\beta_{LacI} \)
LacI repression coefficient
\(0.8~\mu\text{M} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
463
\(\beta_R \)
LuxR-AHL repression coefficient
\(0.01~\mu\text{M} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
464
\(\gamma_{TetR} \)
TetR degradation rate
\(0.0692~\text{min}^{-1} \)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
465
\(\gamma_{LacI} \)
LacI degradation rate
\(0.0231~\text{min}^{-1} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
466
\(\gamma_{GLP-1} \)
GLP-1 degradation rate
\(0.0731~\text{min}^{-1} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
467
\(\gamma_{LuxI} \)
LuxI degradation rate
\(0.0167~\text{min}^{-1} \)
 
[3] MIT igem 2010
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
468
\(k_{s0} \)
AHLi? degradation rate
\(1~\text{min}^{-1} \)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
469
\(k_{se} \)
AHLe? degradation rate
\(1~\text{min}^{-1} \)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
470
\(\gamma_R \)
LuxR-AHL degradation rate
\(0.0231~\text{min}^{-1} \)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
471
\(\gamma_{RFP} \)
RFP degradation rate
\(0.0041~\text{min}^{-1} \)
 
[4] Michael Halter, Alex Tona, Kiran Bhadriraju, Anne L. Plant, John T. Elliott, Automated Live Cell Imaging of Green Fluorescent Protein Degradation in Individual Fibroblasts, Cytometry Part A, Volume 71A Issue 10, 2007
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
472
\(n_1\)
FadR cooperativity coefficient
\(3\)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
473
\(n_2\)
FA cooperativity coefficient
\(2\)
 
a.
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
474
\(n_3\)
LuxR-AHL cooperativity coefficient
\(3\)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
475
\(n_5\)
TetR cooperativity coefficient
\(2\)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
476
\(n_6\)
LacI cooperativity coefficient
\(4\)
 
[1] Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold & Ron Weiss.A synthetic multicellular system for programmed pattern formation, Nature Vol. 434, 2005
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
477
\(\eta\)
AHL Diffusion rate across the cell membrane
\(2\)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
478
\(\eta\)ext
Average diffusion rate for all cells
\(1\)
 
[2] Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz. Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing, PNAS vol. 101 no. 30, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
479
\([E]_t \)
Total active enzyme
\(50~\text{U}\text{mL}^{-1} \)
 
[8] Bengt Borgstrom, Luminal Digestion of Fats, Handbook of Physiology, The Gastrointestinal System, Intestinal Absorption and Secretion, 1991
[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973
[11] M Lingumsky, E Granot, D Branski, H Stankiewicz, R Goldstein, Isolated lipase and colipase deficiency in two brothers, Gut, 1990
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
480
\(K_m \)
Reaction rate constant
\(47.9\)
 
[6] Sulaiman Al-Zuhair, Masitah Hasan, K.B. Ramachandran, Kinetics of the enzymatic hydrolysis of palm oil by lipase, Process Biochemistry Volume 38, Issue 8, 2003
[7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
481
\(D_{FA} \)
FA Diffusion Constant
\( 6.46 \times 10^{-10}~\text{m}^2\text{s}^{-1} \)
 
[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
482
\(d\)
Thickness of the unstirred water layer
\( 190~\mu\text{m} \)
 
[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
483
\( K_{cat}\)
Catalytic Rate constant
\(1.3\times10^{-5}\text{min}^{-1}\)
 
[6] Sulaiman Al-Zuhair, Masitah Hasan, K.B. Ramachandran, Kinetics of the enzymatic hydrolysis of palm oil by lipase, Process Biochemistry Volume 38, Issue 8, 2003
[7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
484
\( cd \)
Relative E. coli Cell Density
\( 0.1 \)
 
 
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
485
\( D_{FA} \)
FA Diffusion Constant
\( 6.46 \times 10^{-10}\text{m}^2\text{s}^{-1} \)
 
[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
486
\(D_{AHL} \)
AHL Diffusion Constant
\( 4.9\times10^{-6}~\text{cm}^2\text{s}^{-1} \)
 
[9] Sallee VL, Dietschy JM., Determinants of intestinal mucosal uptake of short- and medium-chain fatty acids and alcohols, Journal of Lipid Research, 1973
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
487
\( \gamma_{AHL,ext} \)
AHL cell-external degradation
\( 8.0225\times10^{6}~\text{s}^{-1} \)
 
Derived from 1 day half-life at pH 7. [7] [7] Ho-Shing Wu, Ming-Ju Tsai, Kinetics of tributyrin hydrolysis by lipase, Enzyme and Microbial Technology, Volume 35, Issues 6?€“7, 2004
https://2012.igem.org/Team:NTU-Taida/Modeling/Parameters
NTU-Taida
488
DAHL
AHL diffusion constant
4.9 x 10-6 cm2/s
 
Stewart P.S., 2003
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
489
Cagar
Reduced diffusion coefficient
0.9
 
Fatin-Rouge et al., 2004
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
490
αAHL
AHL synthesis rate
0.01 min-1
 
Garcia-Ojalvo et. al., 2004
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
491
dAHL
AHL degradation rate (intracellular)
0.01 min-1
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
492
dAHL,e
AHL extracellular decay
4.8135 x 10-4 min-1
 
Horswill et al., 2007
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
493
ηAHL
Diffusion rate across the cell membrane
2
 
Garcia-Ojalvo et. al., 2004
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
494
ηext
Average diffusion rate for all cells
1.3333
 
Garcia-Ojalvo et. al., 2004
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
495
αLuxI
LuxI synthesis rate
1 μM/min
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
496
dLuxI
LuxI degradation rate
0.0167 min-1
 
MIT iGEM 2010
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
497
k
Cell growth rate
0.888 h-1
 
estimated from experimental data
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
498
αLuxR
LuxR synthesis rate
0.005 μ M/min
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
499
dLuxR
LuxR degradation rate
0.01 min-1
 
Manefield et al., 2002
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
500
ρR
LuxR/AHL dimerization
0.5 μM-3min-1
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
501
dR
Dimer LuxR/AHL degradation rate
0.0231 min-1
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
502
KR
LuxR/AHL activation coefficient
0.013 nM
 
estimated from experimental data (scaled)
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
503
n
Hill coefficient
1
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
504
αGFP
GFP synthesis rate
2 μM min-1
 
Basu et al., 2005
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
505
dGFP
GFP degradation rate
4.4432 x 10-4 min-1
 
Corish and Tyler-Smith, 1999
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
506
αGusA
GusA synthesis rate
1 μM min-1
 
estimated
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
507
dGusA
GusA degradation (half-life 55oC 2 hr)
9.6270-5 s-1
 
Jefferson, 1995
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
508
KR1
AHL activation coefficient
4.45 nM
 
estimated from experimental data
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
509
n1
Hill coefficient
1.7
 
estimated from experimental data
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
510
kleaky
GusA basal expression
0.0375
 
estimated
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
511
αAES
AES synthesis rate
1 μM min-1
 
estimated
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
512
dAES
AES degradation (half-life 55oC 2 hr)
9.6270-5 s-1
 
Jefferson, 1995
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
513
KR2
AHL activation coefficient
12555 nM
 
estimated from experimental data
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
514
n2
Hill coefficient
0.8
 
estimated from experimental data
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
515
kGFP
Hydrolases basal expression
0.0375
 
estimated
https://2013.igem.org/Team:ETH_Zurich/Parameter
ETH Zurich
516
Transcription rate of cI
4200/Gene Length (nM/min)
5.6
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
517
Translation rate of cI
2400RBS/Protein Length
9.6
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
518
Transcription rate of VIP
4200/Gene Length (nM/min)
1.74129353
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
519
Translation rate of VIP
2400RBS/Protein Length
2.985075
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
520
Transcription rate of GFP
4200/Gene Length (nM/min)
5.53359684
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
521
Translation rate of GFP
2400RBS/Protein Length
9.486166
 
https://2009.igem.org/Team:PKU_Beijing/Modeling/Parameters
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
522
Degradation rate of cI (mRNA
Half life = 6.8 min, Division time = 30 min
0.18063836
 
(Selinger, GW, et al., 2003)
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
523
Degradation rate of cI (protein)
Half life > 10 h; division time = 30 min
0.03885825
 
(Varshavsky, (1997) and Tobias et al., 1991)
https://2013hs.igem.org/Team:CIDEB-UANL_Mexico/Math-Parameters
CIDEB-UANL Mexico
524
αLuxR
Production rate of LuxR
0.005 μMmin-1
 
Literature [20]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
525
kRLux
Rate of formation of RLux from LuxAHL and LuxR
0.1 nM-1min-1
 
Literature [19]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
526
k-RLux
Dissociation rate of RLux
10 min-1
 
Literature [19]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
527
KmLux
Lumped parameter for the Lux system
10 nM
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
528
dLuxAHL
External degradation rate of LuxAHL (30C6HSL)
0.004 min-1
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
529
dLuxR
Degradation rate of LuxR
0.0231 min-1
 
Literature [21]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
530
dRLux
Degradation rate of RLux
0.0231 min-1
 
Literature [20]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
531
dmRNABxb1
Degradation rate of mRNABxb1
0.2773 min-1
 
Literature [22]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
532
dBxb1
Degradation rate of Bxb1
0.01 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
533
LPLux
Leakiness after using riboswitch for Plux
0.01463 nMmin-1
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
534
KmRNABxb1
Rate of transcription of Bxb1
5 nMmin-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
535
kBxb1
Rate of formation of Bxb1
0.1 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
536
αLasR
Production rate of LasR
0.005 μMmin-1
 
Literature [20] (Assumed to be the same as Lux system)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
537
kRLas
Rate of formation of RLas from LasAHL and LasR
0.1 nM-1min-1
 
Literature [19] (Assumed to be the same as Lux system)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
538
k-RLas
Dissociation rate of RLas
10 min-1
 
Literature [19] (Assumed to be the same as Lux system)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
539
KmLas
Lumped parameter for the Las system
0.45 nM
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
540
dLasAHL
Degradation rate of LasAHL (30C12HSL)
0.004 min-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
541
dLasR
Degradation rate of LasR
0.0231 min-1
 
Literature [21] (Assumed to be the same as Lux system)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
542
dRLas
Degradation rate of RLas
0.0231 min-1
 
Literature [20] (Assumed to be the same as
Lux system)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
543
dmRNAϕc31
Degradation rate of mRNAϕc31
0.2773 min-1
 
Literature [22]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
544
dϕc31
Degradation rate of ϕC31
0.01 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
545
LPLas
Leakiness after using riboswitch for Plas
0.02461 nMmin-1
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
546
KmRNAϕc31
Rate of transcription of ϕc31
5 nMmin-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
547
kϕc31
Rate of formation of ϕc31
0.1 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
548
kDBxb1
Dimerization rate of Bxb1
1 nM-1min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
549
k-DBxb1
Dissociation rate of DBxb1
10-6 min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
550
kSABxb1
Rate of formation of SABxb1 from DBxb1 and SIBxb1
1 nM-1min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
551
k-SABxb1
Dissociation rate of SABxb1
10-6 min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
552
dDBxb1
Degradation rate of DBxb1
0.02 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
553
kDϕc31
Dimerization rate of ϕc31
1 nM-1min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
554
k-Dϕc31
Rate of dissociation of Dϕc31
10-6 min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
555
kSAϕc31
Rate of formation of SAϕc31 from Dϕc31 and SIϕc31
1 nM-1min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
556
k-SAϕc31
Rate of dissociation of SAϕc31
10-6 min-1
 
Fitted
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
557
dDϕc31
Degradation rate of Dϕc31
0.02 min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
558
kToffBxb1
Rate of flipping of Ton,i to ToffBxb1
0.1 nM-2min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
559
k-ToffBxb1
Rate of flipping of ToffBxb1 to Ton,f
0.1 nM-2min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
560
kToffϕc31
Rate of flipping of Ton,i to Toffϕc31
0.1 nM-2min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
561
k-Toffϕc31
Rate of flipping of Toffϕc31 to Ton,f
0.1 nM-2min-1
 
Assumed
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
562
kmRNAGFP
Production rate of mRNAGFP
5 nMmin-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
563
kGFP
Rate of formation of folded GFP
1 min-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
564
dmRNAGFP
Degradation rate of mRNAGFP
0.2773 min-1
 
Literature [22]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
565
dGFP
Degradation rate of GFP
0.0049 min-1
 
Fitted to experimental data
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
566
kmRNALasI
Production rate of mRNALasI
5 nMmin-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
567
kLasI
Rate of formation of LasI
20 min-1
 
Estimated
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
568
dmRNALasI
Degradation rate of mRNALasI
0.2773 min-1
 
Literature [22]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
569
dLasI
Degradation rate of LasI
0.0167 min-1
 
Literature [21]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
570
kLasAHL
Production rate of LasAHL (30C12HSL) from the LasI
0.04 min-1
 
Literature [19]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
571
θ
Km value for the production of mRNAGFP and mRNALasI
0.01 μM
 
Literature [20] (approximation)
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
572
DAHLext
Diffusion coefficient of extracellular AHL in liquid
4.9 10-6 cm2/s
 
Literature [27]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
573
Dm
Diffusion rate of AHL through the membrane
100 min-1
 
Estimated from literature [27]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
574
r
Growth rate of E. coli in our alginate beads
0.006 min-1
 
 
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
575
α
Ratio of E. coli volume to the volume of one bead
100 min-1
 
V E. coli from literature [28] , bead volume from experimental setup
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
576
N0
Initial number of cells per bead
107 cells
 
Experimental setup
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
577
Nm
Maximum number of cells per bead
8 107 cells
 
Estimated from literature [29]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
578
Cbeads
Correction factor (a priori) for diffusion of LuxAHL in alginate beads
1
 
Estimated from literature [30]
https://2014.igem.org/Team:ETH_Zurich/modeling/parameters
ETH Zurich
579
\(K_{d,\text{LuxRAHL}}\)
Dissociation constant between luxR and AHL
100 nM
 
Weber, 2013
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
580
\(\text{LuxR}_\text{tot}\)
Total concentration of LuxR
0.0025 μM
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
581
\(a_\mathrm{LuxI}\)
Maximal production rate of LuxI
1 μM.min-1
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
582
\(a_\mathrm{LuxI,ribo}\)
Maximal production rate of LuxI
0.1 μM.min-1
 
ETHZ 2014
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
583
\(k_\mathrm{leaky}\)
Coefficient for leakiness dependency on LuxR concentration of PLuxR promoter
0.0375 μM-1
 
ETHZ 2013
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
584
\(K_\mathrm{a,LuxRAHL}\)
Activation coefficient of LuxRAHL
0.45 nM
 
ETHZ 2014
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
585
\(K_\mathrm{LuxRAHL,ribo}\)
Activation coefficient of LuxRAHL in case of a riboregulated LuxR responsive promoter
285 nM
 
ETHZ 2014
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
586
\(L_\mathrm{lux,ribo}\)
Leakiness after using riboswitch for Plux
0.01463 nM.min-1
 
ETHZ 2014
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
587
\(n_\mathrm{lux}\)
Hill coefficient for LuxRAHL activation
1.7
 
ETHZ 2014
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
588
\(d_\mathrm{LuxI}\)
Degradation rate of LuxI
0.0167 min-1
 
MIT 2010
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
589
\(a_\mathrm{AHL}\)
Production rate of AHL
0.04 μM.min-1
 
Weber, 2013
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
590
\(d_\mathrm{AHL}\)
Degradation rate of AHL
0.01 min-1
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
591
\(v_\mathrm{AiiA}\)
Maximal conversion rate of AiiA
\(k_\mathrm{cat} \cdot C_\mathrm{AiiA} \)
 
 
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
592
\(k_\mathrm{cat}\)
Turnover number of AiiA
1.63 103min-1
 
Wang, 2004
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
593
\(C_\mathrm{AiiA}\)
Concentration of AiiA
varied
 
 
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
594
\(K_\mathrm{M,AiiA}\)
Half-maximal rate substrate concentration of AiiA
2.95 103 μM
 
Wang, 2004
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
595
\(a_\mathrm{GFP}\)
Maximal production rate of GFP
2 μM.min-1
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
596
\(d_\mathrm{GFP}\)
Degradation rate of GFP
0.01 min-1
 
estimated from doubling time of E. coli
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
597
\(N_{d}\)
Number of E. coli in the doughnut
150
 
Maximal number of E. coli that would fit on the surface
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
598
\(N_{b,max}\)
Maximum number of E. coli in the bulk
12798
 
Considering the maximal OD is 2
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
599
\(V_{cell,d}\)
Volume around an E. coli in the doughnut
6 μm3
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
600
\(V_{cell,b,worst}\)
Volume around an E. coli in the bulk
78 μm3
 
Worst case, estimated from \(N_{b,max}\)
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
601
\(V_{cell,b,norm}\)
Volume around an E. coli in the bulk
1000 μm3
 
Normal case
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
602
\(\text{B}\)
\(\frac{Lac_\mathrm{ini}^2}{K_\mathrm{d,DLL}}\)
0.000001  -  4
 
 
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
603
\(\text{Lac}_{\text{ini}}\)
Initial concentration of lactate in the medium
0.1 μM  -  2 μM
 
Low concentration of lactate in the medium
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
604
\(K_\mathrm{d,DLL}\)
Dissociation constant between the dimer of Lldr and Lactate
10 μM2 -  10000 μM2
 
 
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
605
\(\alpha\)
Multiplication factor between the initial concentration of Lactate and Production of normal cells
1  -  150
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
606
\(F_\mathrm{C}\)
Fold change between Lactate production by cancer and normal cells
2  -  4
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
607
\(a_1\)
\(\frac{a_\mathrm{LacI}}{d_\mathrm{LacI}\cdot K_{RLacI}}\)
0.05  -  1000
 
 
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
608
\( a_\mathrm{LacI}\)
Maximal production rate of LacI
0.05 μM.min-1 -  1 μM.min-1
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
609
\( d_\mathrm{LacI}\)
Degradation rate of LacI
0.01 min-1 -  0.1 min-1
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
610
\( K_\mathrm{R,LacI}\)
Repression coefficient of LacI
0.1 μM  -  10 μM
 
Basu, 2005
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
611
\( \gamma_1\)
\( \frac{L_\mathrm{2tot}}{K_\mathrm{R,L}}\)
5  -  10000
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
612
\( L_\mathrm{2tot}\)
Total concentration of LldR dimer
0.5 μM  -  10 μM
 
estimated from paxdb
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
613
\( K_\mathrm{R,L}\)
Repression coefficient of LldR
0.001 μM  -  0.1 μM
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
614
\( \gamma_2\)
\(\frac{IPTG_{tot}}{K_{IL}}\)
0  -  500
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
615
\( \frac{a_1}{\gamma_2+1}\)
 
0.001  -  1000
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
616
\( n_1\)
Hill coefficient of LldR
0.5  -  2.5
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
617
\( n_2\)
Hill coefficient of LacI
1.5  -  2.5
 
estimated
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
618
\(D_\text{AHL,agar}\)
Diffusion coefficient of AHL in agar
\(3.0\times 10^{-10} m^2/s\)  -  \(4.41\times 10^{-10} m^2/s\)
 
Trovato, 2014
Fatin-Rouge, 2004
https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters
ETH Zurich
619
c1max
0.01 [mM/h]
max. transcription rate of constitutive promoter (per gene)
 
promoter no. J23105; Estimate
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
620
c2max
0.01 [mM/h]
max. transcription rate of LuxR-activated promoter (per gene)
 
Estimate
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
621
lhi
25
high-copy plasmid number
 
Estimate
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
622
llo
5
low-copy plasmid number
 
Estimate
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
623
a
0.01
basic production levels
 
Estimate
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
624
dLacI
2.31e-3 [1/s]
degradation of LacI
 
Ref. [10] Tuttle et al. "Model-Driven Designs of an Oscillating Gene Network", Biophys J 89(6):3873-3883, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
625
dTetR
1e-5 [1/s] 2.31e-3 [1/s]
degradation of TetR
 
[9] Becskei A and Serrano L "Engineering stability in gene networks by autoregulation", Nature 405: 590-593, 2000
[10] Tuttle et al. "Model-Driven Designs of an Oscillating Gene Network", Biophys J 89(6):3873-3883, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
626
dLuxR
1e-3 - 1e-4 [1/s]
degradation of LuxR
 
Ref: [6] Goryachev AB et al. "Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants", Biosystems 83(2-3):178-187, 2004
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
627
dCI
7e-4 [1/s]
degradation of CI
 
Ref. [7] Arkin A et al. "Stochastic kinetic analysis of developmental pathway bifurcation in phage λ-Infected Escherichia coli cells", Genetics 149: 1633-1648, 1998
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
628
dP22CII
 
degradation of P22CII
 
 
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
629
dYFP
6.3e-3 [1/min]
degradation of YFP
 
suppl. mat. to Ref. [8] corresponding to a half life of 110min. [8] Colman-Lerner A et al. "Yeast Cbk1 and Mob2 Activate Daughter-Specific Genetic Programs to Induce Asymmetric Cell Fates", Cell 107(6): 739-750, 2001 (supplementary material)
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
630
dGFP
6.3e-3 [1/min]
degradation of GFP
 
in analogy to YFP
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
631
dRFP
6.3e-3 [1/min]
degradation of RFP
 
in analogy to YFP
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
632
dCFP
6.3e-3 [1/min]
degradation of CFP
 
in analogy to YFP
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
633
KLacI
0.1 - 1 [pM] 800 [nM]
LacI repressor dissociation constant
 
Ref. [2] Setty Y et al. "Detailed map of a cis-regulatory input function", P Natl Acad Sci USA 100(13):7702-7707, 2003
Ref. [12]
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
634
KIPTG
1.3 [?µM]
IPTG-LacI repressor dissociation constant
 
Ref. [2] Setty Y et al. "Detailed map of a cis-regulatory input function", P Natl Acad Sci USA 100(13):7702-7707, 2003
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
635
KTetR
179 [pM]
TetR repressor dissociation constant
 
Ref. [1] Weber W et al. "A synthetic time-delay circuit in mammalian cells and mice", P Natl Acad Sci USA 104(8):2643-2648, 2007
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
636
KATC
893 [pM]
ATC-TetR repressor dissociation constant
 
Ref. [1] Weber W et al. "A synthetic time-delay circuit in mammalian cells and mice", P Natl Acad Sci USA 104(8):2643-2648, 2007
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
637
KLuxR
55 - 520 [nM]
LuxR activator dissociation constant
 
Ref: [6] Goryachev AB et al. "Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants", Biosystems 83(2-3):178-187, 2004
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
638
KAHL
0.09 - 1 [?µM]
AHL-LuxR activator dissociation constant
 
Ref: [6] Goryachev AB et al. "Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants", Biosystems 83(2-3):178-187, 2004
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
639
KCI
8 [pM] 50 [nM]
CI repressor dissociation constant
 
Ref. [12] Basu S et al. "A synthetic multicellular system for programmed pattern formation", Nature 434:1130-1134, 2005
starting with values of Ref. [6] Goryachev AB et al. "Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants", Biosystems 83(2-3):178-187, 2004
and using Ref. [3] Braun D et al. "Parameter Estimation for Two Synthetic Gene Networks: A Case Study", ICASSP 5:769-772, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
640
KP22CII
0.577 [?µM]
P22CII repressor dissociation constant
 
Ref. [11] McMillen LM et al. "Synchronizing genetic relaxation oscillators by intercell signaling", P Natl Acad Sci USA 99(2):679-684, 2002
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
641
nLacI
1;2
LacI repressor Hill cooperativity
 
Ref. [5] Iadevaia S and Mantzais NV "Genetic network driven control of PHBV copolymer composition", J Biotechnol 122(1):99-121, 2006
; Ref. [12]
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
642
nIPTG
2
IPTG-LacI repressor Hill cooperativity
 
Ref. [5] Iadevaia S and Mantzais NV "Genetic network driven control of PHBV copolymer composition", J Biotechnol 122(1):99-121, 2006
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
643
nTetR
3
TetR repressor Hill cooperativity
 
Ref. [3] Braun D et al. "Parameter Estimation for Two Synthetic Gene Networks: A Case Study", ICASSP 5:769-772, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
644
nATC
2 (1.5-2.5)
ATC-TetR repressor Hill cooperativity
 
Ref. [3] Braun D et al. "Parameter Estimation for Two Synthetic Gene Networks: A Case Study", ICASSP 5:769-772, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
645
nLuxR
2
LuxR activator Hill cooperativity
 
Ref: [6] Goryachev AB et al. "Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants", Biosystems 83(2-3):178-187, 2004
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
646
nAHL
1
AHL-LuxR activator Hill cooperativity
 
Ref. [3] Braun D et al. "Parameter Estimation for Two Synthetic Gene Networks: A Case Study", ICASSP 5:769-772, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
647
nCI
2
CI repressor Hill cooperativity
 
Ref. [12] Basu S et al. "A synthetic multicellular system for programmed pattern formation", Nature 434:1130-1134, 2005
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
648
nP22CII
4
P22CII repressor Hill cooperativity
 
Ref. [11] McMillen LM et al. "Synchronizing genetic relaxation oscillators by intercell signaling", P Natl Acad Sci USA 99(2):679-684, 2002
https://2007.igem.org/wiki/index.php?title=ETHZ/Parameters
ETHZ
