<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Project_Problem skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Project/Problem</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="center"><DIV id="blockNav"><NAV><DIV align="center"><UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas">HOME</A></LI><LI>PROJECT
             </LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Problem">PROBLEM: GENETIC INSTABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Plasmid_Study">OBSERVING FLUORESCENT GENE STABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Strain_Study">STUDYING STABILITY IN DIFFERENT STRAINS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Caffeine">REDESIGNING DECAFFEINATION PLASMIDS </A></LI></UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Interlab_Study">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Attributions">ATTRIBUTIONS</A></LI></DIV></NAV></DIV></DIV></DIV><DIV style="padding: 0px 25px;" align="justify"><H1><SPAN class="mw-headline" id="Problem:_Genetic_Instability">Problem: Genetic Instability</SPAN></H1><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Evolutionary stability of T9002 (black) and T9002 (color) re-engineered circuits with AHL - Image credit to Sean C. Sleight <I>et al.</I></DIV></DIV></DIV><P>One of the major issues facing genetic engineering is the longevity of genetic devices once inserted into organisms. An organism can be modified, but that does not ensure that the modification will last.<B> Subsequent generations of modified organisms often lose or “break” the genetic device through mutations.</B> Breaking a plasmid or an inserted device often decreases the metabolic load of that organism, giving it a competitive advantage by enabling it to allocate more resources toward replication. This allows them to dominate a population over time.
</P><P>Previous research has identified several factors that contribute to the breaking of genetic devices (1). One major factor is the relative fitness of organisms without the device over those with it. Strongly expressed devices will result in a broken population faster than weakly expressed devices, since devices of the former type place a greater metabolic load on the organism. Similarly, genes that code for 'costly' proteins that may interfere with other cellular functions in ways that are toxic are more likely to have a broken device sweep through the populaiton. 
</P><P>Another major factor that contributes to device stability is the rate of mutations leading to device failure. If a device includes DNA sequences that act as mutational hotspots, it may be very unstable. The <A rel="nofollow" class="external text" href="http://barricklab.org/django/efm">Evolutionary Failure Mode calculator</A> (developed by the Barrick lab) identifies DNA sequences that are prone to breaking, such as simple sequence repeats and long stretches of a single repeated nucleotide. The EFM calculator gives sequences a relative instability prediction (RIP) score, and higher RIP scores indicate sequences that are more prone to breaking.
</P><H2><SPAN class="mw-headline" id="Project_and_Motivation">Project and Motivation</SPAN></H2><P>Our projects focused on monitoring device stability and determining what kinds of sequences and sequence features contribute to device instability. <B>We wish to identify and better characterize sequences that contribute to the relative instability of genetic devices so they can be used to predict device longevity, as well as be taken into account when creating and modifying such devices</B>. In the spring of 2015, UT iGEM team members, assisted by the UT Austin Freshman Research Initiative stream &quot;Hijacking Cell Factories for Synthetic Biology,&quot; utilized the EFM calculator to identify sequences within fluorescent protein coding genes that were prone to mutation. We used this information to determine which coding sequences were most and least evolutionary stable. Then, over the summer we chose one of the unstable coding sequences and tested its stability in different strains of <I>E. coli</I>. Further, we redesigned the pDCAF3 plasmid (<A rel="nofollow" class="external text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K734000">BBa_K734000</A>) made by the 2012 Austin UTexas iGEM team in order to replace potentially unstable sequences while retaining the original functionality. In addition, we designed a set of plasmids to expand on the use of the pDCAF3 plasmid for measuring caffeine levels in beverages.
</P><H2><SPAN class="mw-headline" id="A_Computational_Approach">A Computational Approach</SPAN></H2><P>In the course of our experiments with genetic device stability, we also investigated ways to use the power of computation to study our devices. Results gathered using these methods provided similar data, yet offered efficiency improvements and stronger part verification over human analysis.  Such advances are key for scaling up the project to higher throughput technologies such as flow cytometry and Illumina sequencing.  Higher throughput will allow us to generate better statistics for determining how quickly a device breaks.
</P><DL><DD><B><A href="/Team:Austin_UTexas/Failure_Analysis" title="Team:Austin UTexas/Failure Analysis"> Learn more about our computational work here.</A></B></DD></DL><P><SPAN style="font-size:130%"><B><A href="/Team:Austin_UTexas/Project/Plasmid_Study" title="Team:Austin UTexas/Project/Plasmid Study"> Continue to Part 1: OBSERVING FLUORESCENT GENE STABILITY</A></B></SPAN></P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL><LI> Sleight et al.: Designing and engineering evolutionary robust genetic circuits. Journal of Biological Engineering 2010 4:12.</LI><LI> “EFMCalculator.” tool from the Barrick Lab: <A rel="nofollow" class="external free" href="http://barricklab.org/django/efm/">http://barricklab.org/django/efm/</A></LI><LI> Jack et al.: Predicting the genetic stability of engineered DNA sequences with the EFM calculator. Print. Sept. 2015</LI></OL></DIV></DIV></DIV></DIV></BODY></HTML>