<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Failure_Analysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Failure Analysis</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="center"><DIV id="blockNav"><NAV><DIV align="center"><UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas">HOME</A></LI><LI>PROJECT
             </LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Problem">PROBLEM: GENETIC INSTABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Plasmid_Study">OBSERVING FLUORESCENT GENE STABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Strain_Study">STUDYING STABILITY IN DIFFERENT STRAINS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Caffeine">REDESIGNING DECAFFEINATION PLASMIDS </A></LI></UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Interlab_Study">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Attributions">ATTRIBUTIONS</A></LI></DIV></NAV></DIV></DIV></DIV><DIV style="padding: 0px 25px;" align="justify"><H2><SPAN class="mw-headline" id="A_Computational_Approach:_fri-failure-analysis_and_gdparse">A Computational Approach: fri-failure-analysis and gdparse</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Diagram of fri-failure-analysis work flow. Dashed boxes represent collections of files.</DIV></DIV></DIV></DIV><P><B><I>fri-failure-analysis</I></B> is a computational pipeline that identifies mutations that occur in bacterial plasmids. The program analyzes alignments composed of a sample read (generated by Sanger sequencing) and its ancestor strain (template). By identifying homology between samples and mobile elements, the pipeline is able to correctly identify a host of mutations while minimizing false positives. The program streamlines the process of mutation analysis and offers large efficiency gains over human analysis.
</P><P>The program is divided into phases. The initial phase creates FASTA template files that will be used to create alignments. These each contain two sequences, either a template and a sample, or a sample and a mobile element. The next phase employs the MAFFT program to generate alignments between each sequence. The final phase consists of two steps; first, determine if a sample exhibits homology to a mobile element; then, identify mutations in the sample using this information. These mutations are stored in the Genomediff filetype, which you can read about  <A rel="nofollow" class="external text" href="http://barricklab.org/twiki/pub/Lab/ToolsBacterialGenomeResequencing/documentation/gd_format.html">here.</A></P><H3><SPAN class="mw-headline" id="gdparse">gdparse</SPAN></H3><P><B><I>gdparse</I></B> is a tool designed to process Genomediff files by counting up mutations and categorizing them. Since the terminal output of <B>fri-failure-analysis</B> is in this format, the program works well as a post-processing step in our &quot;Breaking is Bad&quot; experiment. The program currently outputs a CSV file that tabulates mutations based on their type (e.g. insertion or deletion). Combined with <B>fri-failure-analysis</B>, this program offers further efficiency gains over hand counting of mutations.
</P><P>This program works by creating dictionaries that map a category (in this case, a BioBrick name) onto other dictionaries that are organized by the type of mutation, the location of the mutation on the reference sequence, or both. Since reference sequences are stored along with each mutation in a Genomediff file, <B>gdparse</B> simply finds the reference sequence in a separate folder and determines what annotation contains the given mutation.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:575px;"><DIV class="thumbcaption">Output of gdparse program, using Genomediff files generated by fri-failure-analysis as input.</DIV></DIV></DIV></DIV><P>Combining the two programs, we obtain a set of mutations similar to those gathered through hand analysis. Currently, we still check data by hand to remove some false positives, such as those gathered from poor alignments or low-quality samples. Even so, <B>the programs demonstrate substantial efficiency improvements over manual analysis.</B> Additionally, the programs minimize human-introduced errors by enforcing stringent naming constraints on template annotations, sequence files, and part labels.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>All code maintained by Tyler Camp. <B><I>fri-failure-analysis</I></B> and <B><I>gdparse</I></B> are released under the GNU General Public License and are free software. Links to the repositories are included below.
</P><UL><LI><A rel="nofollow" class="external text" href="https://github.com/stationarysalesman/fri-failure-analysis">fri-failure-analysis</A></LI><LI><A rel="nofollow" class="external text" href="https://github.com/stationarysalesman/gdparse">gdparse</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>