<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Project_Plasmid_Study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Project/Plasmid Study</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="center"><DIV id="blockNav"><NAV><DIV align="center"><UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas">HOME</A></LI><LI>PROJECT
             </LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Problem">PROBLEM: GENETIC INSTABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Plasmid_Study">OBSERVING FLUORESCENT GENE STABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Strain_Study">STUDYING STABILITY IN DIFFERENT STRAINS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Caffeine">REDESIGNING DECAFFEINATION PLASMIDS </A></LI></UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Interlab_Study">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Attributions">ATTRIBUTIONS</A></LI></DIV></NAV></DIV></DIV></DIV><DIV style="padding: 0px 25px;" align="justify"><H1><SPAN class="mw-headline" id="Observing_Fluorescent_Gene_Stability">Observing Fluorescent Gene Stability</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>One of the major issues facing genetic engineering is the overall longevity of genetic devices once inserted into organisms. An organism can be modified, but that does not ensure that the modification will last. Subsequent generations of modified organisms often lose or “break” the genetic device through mutations. Mutations that cause a device to break (or otherwise lose part of their functionality) decrease the metabolic load of that organism. This allows the mutant and its descendants to replicate and grow faster than the originally modified organism, allowing the “broken” mutant to overtake a population. 
</P><P>The stability of the sequence was the focus of our research over the course of the summer. We sought to identify sequences that contribute to the relative instability of genetic devices so they can be used to predict the longevity of a genetic device as well as be taken into account when creating and modifying such devices. In the spring of 2015, <I>E. coli</I> was transformed using various fluorescent genes and monitored for breakage, with varying results depending on the plasmid used. Five fluorescent proteins: Yellow fluorescent protein (BBa_K592101), Super-folder Yellow fluorescent protein (BBa_K864100),  Engineered Yellow fluorescent protein (BBa_E0030), Enhanced Cyan fluorescent protein (BBa_E0020), and Blue fluorescent protein (BBa_592100).
</P><P>Understanding what leads to genetic instability could lead to more stable devices and thus could increase the longevity of genetically modified organisms.  Since genetic stability can be problematic for any work involving synthetic biology, we wanted to start by determining what sequence motifs make a genetic device unstable. So, in the spring, we began an experiment to study genetic instability of different plasmids in <I>E. coli</I>. We designed and transformed 14 plasmids into TOP10 <I>E. coli</I>. Then, we propagated the strains until either fluorescence was lost or the experiment was concluded. 
</P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_Construction">Plasmid Construction</SPAN></H3><P>Each plasmid comprised of a <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> backbone, which included a gene for chloramphenicol resistance and has a copy number of 100-300. Further, each plasmid received a promoter, ribosome-binding site, and coding sequence for a fluorescent protein. Plasmids were constructed with standard BioBrick cloning procedures and enzymes. Composite BioBricks were constructed using the following unique BioBrick combinations.
</P><P><B>Fig.1:</B></P><TABLE style="width:90%"><TBODY><TR><TD><B><U>Plasmid</U></B></TD><TD><B><U>Fluorescent Protein BioBrick</U></B></TD><TD><B><U>Fluorescent Protein</U></B></TD><TD><B><U>Promoter/RBS BioBrick</U></B></TD><TD><B><U>Promoter;RBS Strength</U></B></TD></TR><TR><TD>1</TD><TD>BBa_K592100</TD><TD>BFP</TD><TD>BBa_K608004</TD><TD>Strong; Weak</TD></TR><TR><TD>2</TD><TD>BBa_K592100</TD><TD>BFP</TD><TD>BBa_K608007</TD><TD>Medium; Weak</TD></TR><TR><TD>3</TD><TD>BBa_E0020</TD><TD>ECFP</TD><TD>BBa_K608007</TD><TD>Medium; Weak</TD></TR><TR><TD>4</TD><TD>BBa_E0020</TD><TD>ECFP</TD><TD>BBa_K608003</TD><TD>Strong; Medium</TD></TR><TR><TD>5</TD><TD>BBa_E0020</TD><TD>ECFP</TD><TD>BBa_K608002</TD><TD>Strong; Strong</TD></TR><TR><TD>6</TD><TD>BBa_K592101</TD><TD>YFP</TD><TD>BBa_K608002</TD><TD>Strong; Strong</TD></TR><TR><TD>7</TD><TD>BBa_K592101</TD><TD>YFP</TD><TD>BBa_K608006</TD><TD>Medium; Medium</TD></TR><TR><TD>8</TD><TD>BBa_K592101</TD><TD>YFP</TD><TD>BBa_K608007</TD><TD>Medium; Weak</TD></TR><TR><TD>9</TD><TD>BBa_K864100</TD><TD>SYFP2</TD><TD>BBa_K608002</TD><TD>Strong; Strong</TD></TR><TR><TD>10</TD><TD>BBa_K864100</TD><TD>SYFP2</TD><TD>BBa_K314100</TD><TD>Strong; Very Strong</TD></TR><TR><TD>11</TD><TD>BBa_K864100</TD><TD>SYFP2</TD><TD>BBa_K608006</TD><TD>Medium; Medium</TD></TR><TR><TD>12</TD><TD>BBa_E0030</TD><TD>EYFP</TD><TD>BBa_K608007</TD><TD>Medium; Weak</TD></TR><TR><TD>13</TD><TD>BBa_EE030</TD><TD>EYFP</TD><TD>BBa_K608006</TD><TD>Medium; Medium</TD></TR><TR><TD>14</TD><TD>BBa_E0030</TD><TD>EYFP</TD><TD>BBa_K314100</TD><TD>Strong; Very Strong</TD></TR></TBODY></TABLE><P>Several fluorescent protein/promoter+RBS combinations were studied in more than one experiment, which is why each plasmid above is classified as a 'unique' combination.
</P><P>The BioBricks in each pair were ligated together using T4 DNA Ligase, and transformed into TOP10 <I>E. coli</I>. Prepared cultures were streaked onto LB/chloramphenicol (CAM) agar plates, and were placed in a 37°C incubator overnight. After at least 20 hours, the plates were retrieved, and a single colony was grown overnight in 5 mL of LB/CAM media. After sequence confirming the plasmid in each culture, we froze a fraction of these cells in glycerol at -80° C for storage.
</P><H3><SPAN class="mw-headline" id="Fluorescence_Readings">Fluorescence Readings</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 1:</B> Procedure for moving fluorescent cultures forwards. The cultures were moved forward a maximum of 10 days, which is ~125 generations.  At the end of Day 1, the each culture has had ~35 generations of growth, each day after this adds an additional 10 generations.</DIV></DIV></DIV><P>After we transformed the plasmids into <I>E. coli</I>, we were almost ready to begin the propagation phase of our stability experiment. 
</P><P>First, we streaked transformed cells for each plasmid onto LB/CAM plates and let them incubate overnight. The next day, we selected a number of independent colonies that were still fluorescing for each plasmid. Then, we placed the colony in 5 mL of LB/CAM media and grew each overnight in the shaker at 37° C and 215 RPM. We refer to these cultures as 'Day 1'. The next day, we began the propagation phase of our experiment.
</P><P>Moving forward, each day we retrieved the overnight cultures from the shaker. We used 5 μl to start a fresh 5 mL culture with LB/CAM media, which was then grown overnight. Next, we measured fluorescence using a 96 well plate reader by pipetting 200μL from each culture and into a well on a 96-well plate. Finally, the day's cultures were placed in the cold room for storage. This process was repeated until either fluorescence was lost or ten days had past.
</P><P>After the propagation phase, we sequenced the final cultures and compared this sequence to the original to determine which mutations had dominated the cultures. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Table.1</B> Summary of observed mutations in plasmid study.  Asterisk indicates one plasmid that had two separate mutations.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H2><P>The plasmids created had a great amount of variance in their stability and longevity over the course of the experiment depending on the fluorescent protein that was used in the plasmid. 
</P><P>Plasmids that bore the SYFP2 gene tended to be unstable. Strains with SYFP2 gene frequently broke down within three days, going from very high fluorescence to no observed fluorescence. Furthermore, some samples of SYFP2 lost functionality after merely a day, showing the SYFP2 gene is very unstable across generations. The SYFP2 plasmids most often accumulated IS (insertion sequence) element inserts, which completely prevented the expression of the fluorescent protein. The most common IS elements observed were IS10R and IS10L. Furthermore, S10L/R expressed preference for a particular nucleotide location (GGCGTAGTACC) within the SYFP2 gene. Two point mutations were also independently observed in two samples of SYFP2.  One mutation converted resulted in the loss of a start codon.
</P><P>Plasmids that contain the YFP gene also tended to display instability. Samples with the YFP gene often broke after approximately five days, and a myriad of mutations appeared to have contributed to this instability. IS elements were another common feature among the YFP group. Point mutations in various locations also appeared in the YFP group, such as in the promoter and ribosome binding site. Two samples also featured a large deletion from the plasmid. 
</P><P>The BFP group of plasmids also had high instability, similar to the SYFP2 group of plasmids. This group of plasmids lasted around only a few days before their devices broke. IS elements were very common in the BFP plasmid group along with promoter deletions.
</P><P>Interestingly, the EYFP and ECFP plasmid groups were incredibly stable, remaining fluorescent for over ten days. Only one sample in the entire EYFP group displayed a mutation, which was a promoter deletion. Only two mutations were shown in the entire ECFP group, a large deletion and an In-del mutation. Overall, these two fluorescent proteins have shown to be very stable (see <B>Table 1</B> for full results of mutations). 
</P><P>Based on the patterns of mutations, it is clear that IS elements are a major source of instability in plasmids in this strain of <I>E. coli</I>, Top10. Furthermore, because of the common location of insertion for IS elements, they must have a sequence preference. By changing the sequence, it it may be possible to increase the stability of plasmids by eliminating a primary source of mutation. Furthermore, by elucidating the contribution that IS elements make towards the instability, we can anticipate these events and account for such instabilities when designing plasmids for future experiments.
</P><P>One of our team members also developed a <A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Austin_UTexas/Failure_Analysis">Computational Approach</A> to categorizing the mutations.  We believe this will be of great help to us as we begin to analyze our larger data set(s).
</P><P><SPAN style="font-size:130%"><B><A href="/Team:Austin_UTexas/Project/Strain_Study" title="Team:Austin UTexas/Project/Strain Study"> Continue to Part 3: STUDYING STABILITY IN DIFFERENT STRAINS</A></B></SPAN></P></DIV></DIV></DIV></DIV></BODY></HTML>