<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Project_Spring_Mutations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Project/Spring Mutations</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="center"><DIV id="blockNav"><NAV><DIV align="center"><UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas">HOME</A></LI><LI>PROJECT
             </LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Problem">PROBLEM: GENETIC INSTABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Plasmid_Study">OBSERVING FLUORESCENT GENE STABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Strain_Study">STUDYING STABILITY IN DIFFERENT STRAINS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Caffeine">REDESIGNING DECAFFEINATION PLASMIDS </A></LI></UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Interlab_Study">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Attributions">ATTRIBUTIONS</A></LI></DIV></NAV></DIV></DIV></DIV><DIV style="padding: 0px 25px;" align="justify"><H3><SPAN class="mw-headline" id="Spring_2015_Data_and_Observations">Spring 2015 Data and Observations</SPAN></H3><P>Data from the first experiment gave a strong indication as to whether or not a coding sequence was stable. Specifically, plasmids containing coding sequences for BFP, SYFP2 and YFP broke swiftly while coding sequences for EYFP and ECFP remained mostly intact by the experiment’s conclusion.
Several types of mutations were observed: IS element insertions*, point mutations, various deletions, and phage insertions. IS element insertions were the most common mutation, and occurred exclusively in the ‘unstable’ plasmids (BFP, SYFP2 and YFP). 23 of 40 independent cultures for these coding sequences exhibited a loss of fluorescence due to mobile element insertions.  The next most common mutation were single nucleotide polymorphisms; 6 point mutations were observed. Four of these point mutations occurred in a promoter (which?), while one was located in an RBS (which?) and one altered a start codon. Further, five samples incurred promoter deletions (which promoters?), and five additional samples developed miscellaneous large and small deletions. Finally, three samples with coding sequences for BFP contained the same phage element, suggesting that this event occurred before independent colonies were selected. (possibly add some discussion about promoters—repeat mediated?)
Our results displayed an array of mutations, and potential experiments for follow up. We decided to focus on IS element insertions, which were most prevalent. These insertion elements are found in the genome of Top10 E. coli. They replicate ~independently*? and insert themselves into different DNA locations. If this DNA location happens to be within the inserted plasmid, the genetic device breaks, granting a depreciated metabolic load in the mutant and reducing fluorescence. A number of mobile elements in E. coli have been identified. In particular, we observed mobile elements IS10L, IS10R and IS1, with the latter occurring only once*. Interestingly, in plasmids with YFP and BFP, these IS elements exhibited only moderate insertion preference. In these sequences, IS elements were found throughout both promoters and coding sequences. However, in SYFP2 plasmids, IS10L/R expressed preference for a particular nucleotide location (GGCGTAGTACC) near the start of the SYFP2 plasmid. (same nucleotide sequence in each plasmid?)*
Based on this preliminary data, we consider coding sequences for BFP, SYFP2 and YFP to be unstable while we consider coding sequences for EYFP and ECFP to be stable. Consequently, we decided to focus on SYFP2 as we continued studying instability.
</P><P><B><FONT color="017e70">(WILL BE MOVED TO FIRST DATA SET)</FONT></B><FONT face="Courier New">Observations and Discussion</FONT></P><P><FONT color="017e70">insert mutations in SFYFP
EYFP, CYFP stable
what contributes to stability
what does this mean about what sequences are stable&lt;/br&gt;
next steps</FONT></P></DIV></DIV></DIV></DIV></BODY></HTML>