<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Project_Strain_Study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Project/Strain Study</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="center"><DIV id="blockNav"><NAV><DIV align="center"><UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas">HOME</A></LI><LI>PROJECT
             </LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Problem">PROBLEM: GENETIC INSTABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Plasmid_Study">OBSERVING FLUORESCENT GENE STABILITY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Strain_Study">STUDYING STABILITY IN DIFFERENT STRAINS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Project/Caffeine">REDESIGNING DECAFFEINATION PLASMIDS </A></LI></UL><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Interlab_Study">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Austin_UTexas/Attributions">ATTRIBUTIONS</A></LI></DIV></NAV></DIV></DIV></DIV><DIV style="padding: 0px 25px;" align="justify"><H1><SPAN class="mw-headline" id="Studying_Stability_in_Different_Strains">Studying Stability in Different Strains</SPAN></H1><H2><SPAN class="mw-headline" id="Assessing_a_Yellow_Fluorescent_Protein_Gene_for_Stability">Assessing a Yellow Fluorescent Protein Gene for Stability</SPAN></H2><P>
Four strains of <I>E. coli</I> were selected to be transformed with our new biobrick, <A href="“http://parts.igem.org/Part:BBa_K1627007”">  BBa_K1627007 </A>. The device was composed of <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K608006">BBa_K608006</A>(composed of a medium promoter and ribosome binding site) and the Super Yellow Fluorescent Protein 2 <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K864100">BBa_K864100</A>. We selected strains TOP10, MDS42, BL21 (DE3), and BW25113. These strains were all selected because they are commonly used strains in synthetic biology research. The MDS42 strain was selected due to the fact that is had had its characterized IS elements removed from its genome (Umenhoffer et al. 2010). Our earlier experiments found that a major cause of mutation in genetic devices (and specifically in this genetic device) were transposable elements that were inserted into the devices. We hypothesized that cultures of MDS42 cells with the Super Yellow Fluorescent Protein gene (<A href="“http://parts.igem.org/wiki/index.php?title=Part:BBa_K864100”">  BBa_K864100 </A>) would exhibit greater fluorescence longevity than the three other strains. 

</P><P>
All four strains were transformed with the plasmid <A href="“http://parts.igem.org/Part:BBa_K1627007”">  BBa_K1627007 </A>, which was designed and constructed in the spring of 2015. The SYFP2 was selected because of its great instability in our initial experiments. After each strain was transformed with the plasmid, they were grown in LB overnight and preserved as glycerol stocks.

</P><P>To determine the relative stability of the SYFP2 genetic device in each of the four strains, we grew six replicates of each strain containing the genetic device in LB media. There were 24 cultures in all. Cultures grew overnight and became &quot;Day 1&quot; cultures. A Day 1 culture contains approximately 35 generations of <I>E. coli</I> cells.  
</P><P>A sample of Day 1 culture was used to carry forward to the next day via a 1:1000 dilution, yielding an additional 10 generations on Day 2.  A sample of Day 1 culture was also taken to measure fluorescence using flow cytometry.  The remaining culture was used to create frozen stock for the future.  This frozen stock was kept for making for use as reference samples.
</P><H2><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H2><P>
To continue studying evolutionary stability, we began a variation of our spring experiment. Instead of studying only Top10 <I>E. coli</I>, we expanded our experiment to include other common strains, specifically: BL21 (DE3), BW25113, and MDS42. The MDS42 cells had the added benefit of containing a minimal genome. Furthermore, (<A href="“http://parts.igem.org/Part:BBa_K1627007”">  BBa_K1627007 </A>) was the plasmid used in each of the four strains of <I>E. coli. </I>.

</P><P>Before propagating the cell cultures, for each strain, we streaked transformed cells from frozen stocks on to an LB/CAM plate and let them incubate overnight. The next day, we chose six independent and fluorescent colonies from each strain. To ensure all cells were retrieved (and thus a more accurate generation time), we pierced the full depth of the agar with a micropipette tip large enough to encompass the entire colony. We then placed the colony in 10 mL of LB/CAM media and grew all 24 cultures overnight in the shaker at 37° C and 215 RPM. We refer to these cultures as 'Day 1' in our results. The next day, we began the propagation/mutation* phase of our experiment.
</P><P>From this point forward, each morning we retrieved the overnight cultures from the shaker and checked for fluorescence using a blue light, recording any observations. Then, we used a vortex machine on each culture to homogenize the liquid. We used 10 μl to start a fresh 10 mL overnight culture with LB/CAM media. Next, we froze 3 mL of culture in 15% glycerol at -80° C for storage and spun down 4.5 mL of culture for minipreps. In the spring, we used a dark reader to determine a single fluorescence value. However, over the summer we switched to using flow cytometry as a more accurate measure of fluorescence.  So, with the remaining 2 mL of culture, every few days we would use the flow cytometer to determine the proportion of cells which were still fluorescent. 
</P><P>Re-suspension of each culture using PBS preceded the use of the flow cytometer. Each sample was then pipetted into a well in a 96-well plate, with every six samples separated by a well filled with PBS only. The flow cytometer reads to fluorescent value of each well by sipping each well automatically using a syringe. The media flows from the syringe and into cytometer to be passed through a laser which counts the number of cells (called events) and the intensity of the fluorescence and repeats this for each filled well. The first four days of samples that were read using flow cytometry are seen in Figure 1.
</P><P>After a culture appeared to stabilize at either a complete or partial loss of fluorescence, we stopped moving the culture forward.  
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Figures 1-4 display the day to day fluorescence of each strain given by the flow cytometer, which informs us about the stability of the device in each strain and how each strain might impact the device's stability. Max fluorescent cell intensity typically centers around 10^6, while control media (with zero fluorescence) centers around 10^4.
</P><P>In Figure 1, flow cytometry data is presented for 6 replicate cultures using Top10 cells.  On Day 1, only five out of the six sample display high fluorescence – centered around 10^6 - indicating that the SYFP gene is still functioning is the sample population. Sample #3 meanwhile already displays a large population that has broken its fluorescent gene – shown by fluorescence around 10^4 -although some cells in the population still have retained their fluorescent protein production.  However, by the second day, all six samples of Top10 display fluorescent values that show that the the cells in each sample population have ended the production of SYFP. The persistence of low fluorescence continues into the day 3 and 4, proving that the population has broken the genetic device and this strain was no longer carried forward after.
</P><P>BW25113 behaved similarly to Top10 (See Figure 2). On the second day,  less than 20% of the cells are considered to be still fluorescent. However, several of the samples appeared to maintain a small proportion of the cells (20%) with reduced fluorescence between 10^4 and 10^5. This reduced fluorescence, which is especially seen on the 4th day for BW25113, is indicative of sequence changes that do not end protein production. 
</P><P>Most BL21 (DE3) cultures experienced a similar reduction in fluorescence (See Figure 3). Every sample population on day one carried non-fluorescent cells. By the third day, the majority of the cells were non-fluorescent However, two of the six replicates maintained a reduced fluorescent population, at approximately 10^5, on day three. This reduced fluorescent populace decreased in size the following day as more cells continued to completely break the genetic device. 
</P><P>The MDS42 (Figure 4) samples displayed a varied and unique pattern of fluorescence across eight days, making the MDS42 strain the most stable set of samples. The first two days display patterns typical of the previous three groups. On the first day, while all six samples emanated high fluorescence, many samples already contained a large portion of the sample populace that had broken their genetic device.  By the second day, much more of the sample population in each sample had shifted to little or no fluorescence. The remaining population in each sample, while not broken, are emanating a reduced fluorescent value at around 10^5. Oddly, this remained a stable pattern across the remainder of the eight days.  Three of the samples contained populations that mostly presented reduced fluorescence at around 10^5.  The remaining three samples, though with a large portion displaying no fluorescence, had some portion displaying a similar, reduced fluorescence. 
</P><P>In summary, Top10 cells were the most unstable, with nearly all fluorescence being lost by Day 2.  As this is our most common cloning strain, this was rather depressing.  In contrast, MDS42 cultures exhibited the best stability but also the most variance. Half of the samples had a population that was mostly broken with some moderately fluorescent. The other half of the MDS42 samples persistently remained at a moderate fluorescence value. BW25113 and BL21 (DE3) samples experienced moderate fluorescence stability, with trends less extreme than Top10 or MDS42.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 1 (TOP10) - Each colored line corresponds to a sample replicate of transformed Top10, while the gray area is a blank. The x-axis represents the intensity of fluorescence on a logarithmic scale while the y-axis represents the number of cells that exhibited a particular fluorescent value. Each blank (grey area) is the fluorescent value of a reading of PBS (Phosphate-buffered saline). Peaks to the left of the end of the blank represent cells that have ceased producing fluorescence proteins while cells past this point have continued production. </B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 2 (BW-25113) - Each colored line corresponds to a sample replicate of transformed BW-25113, while the gray area is a blank. The x-axis represents the intensity of fluorescence on a logarithmic scale while the y-axis represents the number of cells that exhibited a particular fluorescent value. Each blank (grey area) is the fluorescent value of a reading of PBS (Phosphate-buffered saline). Peaks to the left of the end of the blank represent cells that have ceased producing fluorescence proteins while cells past this point have continued production.</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 3 (BL-21 (DE3)) - Each colored line corresponds to a sample replicate of transformed BL-21 (DE3), while the gray area is a blank. The x-axis represents the intensity of fluorescence on a logarithmic scale while the y-axis represents the number of cells that exhibited a particular fluorescent value. Each blank (grey area) is the fluorescent value of a reading of PBS (Phosphate-buffered saline). Peaks to the left of the end of the blank represent cells that have ceased producing fluorescence proteins while cells past this point have continued production.</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 4 (MDS-42) Each colored line corresponds to a sample replicate of transformed MDS-42, while the gray area is a blank. The x-axis represents the intensity of fluorescence on a logarithmic scale while the y-axis represents the number of cells that exhibited a particular fluorescent value. Each blank (grey area) is the fluorescent value of a reading of PBS (Phosphate-buffered saline). Peaks to the left of the end of the blank represent cells that have ceased producing fluorescence proteins while cells past this point have continued production.</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>The Top10 group showed the expected patterns based on previous research, further indicating that the super-yellow fluorescent protein is an unstable genetic device that quickly breaks down once transformed, resulting in a non-functioning mutant after merely two days (~45 generations or less). 
</P><P>The BL21 and BW25113 also displayed rapid destabilization and breakdown after three or four days, similar to the Top10 group. However, both groups also displayed a section of cells that seem to display a reduced, moderate value of fluorescence  of about 10^5, like on the third day of the BL21 strain. This is indicative of a different mutant that has mutated to reduced the intensity of the fluorescent protein without stopping production.
</P><P>This is also seen in the MDS42 strain of bacteria. The MDS42 strain maintained the genetic device the longest and seemed to stabilize at the previously mentioned moderately fluorescent value. The fact that this strain lasted so much longer than the other three strains could be from the lack of IS elements in the MDS42 genome. Without these large transposons inserting into the plasmid and breaking the fluorescent genetic device, the genetic device is much more likely to remain than in other strains. This could be an explanation to the longevity of this strain, which is consistent with our hypothesis. 
</P><P>The lack of transposons in the genome could also be an explanation for the persistence of the reduced and moderate fluorescence. Because large insertions by IS elements are not occurring, smaller mutations like point mutations are more likely to occur. Such point mutations could change the amino acid composition of the fluorescent protein without stopping production. This change in composition could reduce the intensity of fluorescence and reduce the cost of protein production, decreasing metabolic load. The decrease in metabolic load, would then allow the bacteria to remain competitive in the population and persist for longer.
</P><P>Further experiments in sequencing can confirm this explanation. Future experiments will include next-gen sequencing (Illumina) to confirm the presence of point mutations in the SYFP2 gene. Sequencing of the Top10, BL21, and BW25113 can also confirm the presence of IS elements in the breakdown of plasmids and the presence of a consistent binding site of IS elements. 
</P><P><SPAN style="font-size:130%"><B><A href="/Team:Austin_UTexas/Project/Caffeine" title="Team:Austin UTexas/Project/Caffeine"> Continue to Part 4: REDESIGNING DECAFFEINATION PLASMIDS</A></B></SPAN></P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL><LI>Umenhoffer, Kinga et al. “Reduced Evolvability of <I>Escherichia Coli</I> MDS42, an IS-Less Cellular Chassis for Molecular and Synthetic Biology Applications.” <I>Microbial Cell Factories</I> 9 (2010): 38. <I>PMC</I>. Web. Sept. 2015.</LI></OL></DIV></DIV></DIV></DIV></BODY></HTML>