<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BABS_UNSW_Australia_overview skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BABS UNSW Australia/overview</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/overview">Vision</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/endosymbionts">Endosymbionts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/phlow">pHlow system</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/pseudoknots">Pseudoknots</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/containment">Containment</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futureapps">Applications</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futurework">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices/Biosafety">Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Protocols">Protocols</A></LI></DIV></DIV></DIV><DIV class="container"><P>Synthetic biology and the iGEM competition have at their foundation the core concepts of chassis and genetic construct. Our project, Endosynbio, aims to expand this premise with a new core domain built around symbiogenesis and the use of chassis as synthetic organelles. 'Endosynbiology', as we call it, is a novel mechanism for in vivo expression of synthetic proteins.

Past drug design was centred on simple molecules capable of interacting with complex biological systems. As molecular biology has developed, however, modern medicine has expanded into more complex therapies - notably the administration of proteins. These proteins have been used as supplements, such as insulin injections for diabetics, as well as highly specific drugs, such as monoclonal antibodies in a range of neoplastic, autoimmune and inflammatory disorders. The use of nucleic acids to treat disease is also emerging, with siRNA technology being trialled as a tool to silence proteins in pancreatic cancer. We believe that synthetic bacterial organelles designed to stably produce these substances could serve as an important aspect of modern drug administration and delivery. This is our vision.


Our project aims to generate the tools and conceptual framework necessary to underpin this new paradigm. We have re-designed and optimised an old system for induced endocytosis based on two genes (known as invasin and listeriolysin O) into our new pHlow system, which integrates with the host cell biology to create a dynamic, elegant cellular entry system. Our plasmid design ensures the entry-enabling genes are de-activated once invasion has occurred, using delay-timer RNA pseudoknots and Cre recombination. A safety-switch toxin-antitoxin system, maintained on two plasmids and integrated with the pHlow system, will ensure no bacteria which leave the lab remain invasive and prevents horizontal gene transfer. <FIGURE><FIGCAPTION>Plasmid design of the pHlow system</FIGCAPTION></FIGURE>

Much of our research has involved optimising, characterising and gathering data on these systems to provide a foundation for future research. We had aimed to assay a range of bacterial and eukaryotic cells combinations to determine the most compatible organisms. However, we encountered significant safety risks and decided to refrain until our safety mechanims were more conclusively characterised. Our test subjects included three bacteria: the model laboratory strain <I>Esherichia coli</I> K12, a small, food-grade bacteria, <I>Lactococcus lactis</I> and a slow-growing cyanobacteria, <I>Synechocystis</I> PCC6803. Each of these are exciting or useful in their own way, as will be discussed in later pages of our wiki. Our planned host cells were three mammalian cell lines: HeLa T cells, HEK-329 cells and CHO7 cells.</P>

The use of an intracellular bacterium to supplement host cell function is not an old idea. According to the theory of symbiogenesis, which is supported by a range of phylogenetic and cell biology proofs, the mitochondria generating energy within each and every animal cell were originally ancient bacteria. A similar endosymbiosis is believed to underlie eukaryotic plant cells, with their carbon-fixing chloroplasts derived from proto-oxygenic photosynthetic bacteria. 

These symbioses are so old and ingrained within the cells, that there is no longer any functional distinction between them. Other endosymbionts, however, are far more recognisable. Indeed, inspiration for the endosynbio project was derived from the interaction between pea aphids and their obligate intracellular bacteria, <I>Buchnera</I>. Due to an amino acid-starved diet of tree sap, pea aphids are reliant on their endosymbionts for amino acid synthesis. <I>Buchnera</I> species contain the genetic code for synthesis of all twenty amino acids and thus function as an intracellular factory without which the pea aphid could not survive. Observing this relationship opened our eyes to the possibility of intracellular, biological drugs, and led us to our ultimate question - is an artificial symbiogenesis possible?

</DIV></DIV></DIV></DIV></BODY></HTML>