<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BABS_UNSW_Australia_Notebook2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BABS UNSW Australia/Notebook2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/overview">Vision</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/endosymbionts">Endosymbionts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/phlow">pHlow system</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/pseudoknots">Pseudoknots</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/containment">Containment</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futureapps">Applications</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futurework">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices/Biosafety">Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Protocols">Protocols</A></LI></DIV></DIV></DIV><NAV class="col-xs-3 bs-docs-sidebar"><UL id="sidebar" class="nav nav-stacked"><LI><A href="#2014">November 2014</A></LI><LI><A href="#february">February 2015</A></LI><LI><A href="#march">March 2015</A></LI><LI><A href="#april">April 2015</A></LI><LI><A href="#may">May 2015</A></LI><LI><A href="#june">June 2015</A></LI><LI><A href="#july">July 2015</A></LI><LI><A href="#august">August 2015</A></LI><LI><A href="#september">September 2015</A></LI></UL></NAV><DIV class="col-xs-9"><P>A brief summary of our trials, tribulations and fleeting moments of joy.</P><H3 id="2014">Late November - Early December 2014</H3><P>Following a lecture recruitment hustle by Rob, team members applied and were selected for the iGEM team. We all started reading up on synthetic biology and dreaming potential projects ranging from far-fetched to science fiction to milk.</P><H3 id="17february">17-19 February 2015</H3><P>The team attended a three day bootcamp run by Rob, Daniel and Chris. We learned the basics of molecular biology techniques for the iGEM competition - cloning, enzyme digests, ligations and transformations. We participated in many sessions (some lead by team members) about other aspects of the competition - wiki design, primer design, actual design design. We also started presenting and discussing project ideas, and brainstorming ways to make each of them better.</P><H3 id="2march">2-6 March 2015</H3><P>The uni semester starts, and we had our first meeting on the first day. We discussed communication strategies (online labbooks, OneNote vs Evernote) and role allocation. Most importantly, we started seriously talking about various projects ideas and their respective advantages and disadvantages.Projects discussed included:</P><P><LI>recombinant dehalogenases
</LI><LI>dynamic gene editing
</LI><LI>type 3 secretion systems for protein delivery
</LI><LI>biophotovoltaic cells
</LI><LI>synthetic endosymbiotic relationships for therapeutic use (endosynbio!)
</LI></P><H3 id="9march">9-13 March 2015</H3><P>Despite our 8am Monday morning meeting, we enthusiastically discussed a narrowed down list of projects, and set down pros and cons for each. After some discussion and a preferential vote, we settled on Endosynbio! Despite lying on the sci-fi end of the spectrum, we found some seemingly achievable goals to achieve with our project and decided it was worth the risk. The three parts as we decided were:</P><P><LI>invasin/listeriolysin
</LI><LI>colA/minC control system
</LI><LI>xenophagy
</LI></P>
We also started the iGEM registration process and that was exciting.
<H3 id="16march">16-20 March 2015</H3><P>At this Monday morning meeting, we discussed a wide range of topics. We started getting organised in terms of funding, outreach, wiki, team roles and labwork. We were all still spending a lot of time reading up and dreaming up project and outreach ideas. 
We had a separate design meeting (Kris, Mac, Daniel) where several ideas, including the DNA rainbow serpent, were borne (and rapidly buried).
Not satisfied with our one-hour meetings, the team (minus advisors) planned to meet all day on saturday. It was surprisingly fun, productive, and everyone left feeling optimistic. 
</P><H3 id="23march">23-27 March 2015</H3><P>Monday morning meeting: more project discussion. What symbionts/host cell lines should we use? Synechocystis seemed to be the favourite. We also decided on the importance of a very effective kill switch.</P><H3 id="30march">30 March - 3 April 2015</H3><P>Another early Monday morning meeting. Prior art searches showed significant similarity of our planned project to two previous iGEM projects (Warsaw 2009-2010). The main concern of this meeting was finding ways to differentiate our project from those, and make it novel. Ideas included a focus on population control within the cell and making the host reliant on the endosymbiont.
 We finished writing our first grant application, for a grant offered by NSW Trade &amp; Innovation. 
 We investigated flights to Boston in September. Australia is very far from the rest of civilisation, so this was important.
</P><H3 id="13april">13-17 April 2015</H3><P>A difficult week for the team, as we dealt with the loss of one our team members. The project took a hiatus as we mourned Ben. Words cannot express the shock and deep sadness we all felt. Though this wiki notebook seems like a strange forum to discuss such a tragic event, to gloss over it would be impossible. His absence was deeply felt. Several team members attended his funeral this week, and this further emphasised the tragedy of his passing. He was surrounded by loving family and friends. His ambition and passion for science and helping others has driven and informed the rest of our project. Rest in peace, Ben. Words on a wiki cannot do you justice, but I feel like you would appreciate it.
</P><H3 id="20april">20-24 April 2015</H3><P>We had our weekly Monday morning meeting, and a separate wiki and kill switch design meeting.
</P><H5><B>NEW! Wet lab:</B></H5>We started lab work: making media (LB and SOC) and pouring agar plates. Also, the distribution kit arrived.
<H3 id="27april">27 April-May 1 2015</H3><P>This was an assessment-heavy week for all of us, so not much progress was made. However, we managed to write and distribute our first monthly newsletter. 

</P><H5><B>Wet lab:</B></H5>This week consisted of familiarising ourselves with the lab and with using the iGEM distribution kit. Four BioBrick parts related to our project were resuspended and transformed into E. coli. All successful transformants were miniprepped to isolate their plasmid DNA, which was then quantified via NanoDrop/Qubit and visualised via PCR and electrophoresis. Stocks of LB and LB agar were prepared as well as TAE buffer for future gels. Our culture of Synechocystis has also been incubating this week, with significant green growth now evident.  
<H3 id="4may">4-8 May</H3><P>Many things starting to progress on the outreach front - collaboration discussions with the BIOMOD team, Izzy working on SmartSparrow programming and getting feedback from high school teacher surveys, and the conception of a UNSW-hosted Aussie iGEM meet up.
We were all starting to feel a bit lost with our project due to the lack of clear goals, and many problems becoming apparent. 
For example:
</P><P><LI>Not different enough from past projects. 
</LI><LI>Synechocystis issues such as long doubling time, transformation and lack of expression tools. Furthermore we lack a strong validation for why we want to use Synechocystis. 
</LI><LI>It is unclear what we can reasonably achieve with the project as in its current form. If Synecho fails or is too slow we won't have much. 
</LI></P>
This is when we started to consider using Lactococcus and E. coli symbionts in addition to Synechocystis. There was too much to talk about, so we scheduled in another mega-meeting. We spent several hours, and came up with many ideas that formed the final project, such as the pHlow system and the idea of trialling several symbionts in several different mammalian cell lines in several conditions/concentrations. 

<H5><B>Wet lab:</B></H5>
More competent DH5a cells were prepared (19x200 ul). The previous weeks PCR products (2.7P, 2.24B, 2.3I and 4.17F) were run on a gel, no bands were detected. The 10X TAE stock was used to produce a 1X solution that did not set. This was retested the following day and did set. The failed PCR was retried and ran once again on a gel, this time it worked! New parts were cloned, they include 1.30 (strong constitutive promoter, strong RBS), 2.20L (promoter activity reporter), 4.5E (AHL inducible H2O2 production), 1.4H (apple fragrance), 3.4K (lox site for Cre recombinase) and 2.3I (AHL). Of the plated samples 4 were successful and prepared for mini-prep. These were then mini-prepped and concentrations were read using the nanodrop. They were very low. These were cultured overnight and re-prepared. The following nanodrop yielded some great concentrations. Unfortunately once again no bands were registered on the gel, probably due to the use of the incorrect elution buffer. LB media (500ml) and LB agar (500ml) were prepared and the lab space was tidied. 
<H3 id="11may">11-15 May</H3><P>During our Monday morning meeting we discussed the results of our mega-meeting. The key goals of our project are now:
<OL><LI>invasion: Invasin and LLO ph induced expression. Improvement on previous systems as it responds to the environment and does not produce LLO unless it needs it. This system has been named phLOW. It will work in partner with the cre/lox safety switch. 
</LI><LI>Safety/Kill switch: Lox sites flank the invasion genes with Cre downstream and under the influence of the same promoters. This should result in the invasion genes excision and subsequent degradation after they have been expressed to invade a cell. We may need to play around with slowing down the transcription via rare codon elements or pseudoknots. If possible we aim to make the kill switch be positively selected for, one suggestion is using an auxotrophic strain and amino acid synthesis on the plasmid. Another is transferring an essential gene from the chromosome to the plasmid. 
</LI><LI>Population control: AHL mediated population density sensor that alters gene expression to produce an antimicrobial peptide. This would result in a dynamic, oscillating population density.
</LI></OL></P><H5><B>Wet lab:</B></H5>
We amplified the biobricks from the mini-preps from last week with PCR. These also ran very poorly on the gel. These minipreps were sent to the Ramachotti centre for sequencing to confirm part identity, but the signals from the sanger sequencing was also very poor. 
<H3 id="18may">18-22 May 2015</H3><P>Very, very busy assessment week for all. We even called off our weekly meeting.
</P><H5><B>Wet lab:</B></H5>
Our first 3A assembly was attempted this week. We digested a linerized backbone, 1:3O and 2:2OL and ligated them together.  
<H3 id="25may">25-29 May 2015</H3><P>We discovered a recently-published paper very similar to our project (Lee, Tae J., et al. &quot;A Power-Law Dependence of Bacterial Invasion on Mammalian Host Receptors.&quot; (2015): e1004203). After briefly panicking, we decided to view this as a positive - it helped us select cell lines, and assured us our research was topical. Parts we'd ordered from the iGEM registry were quarantined as they arrived in Australia. We were worried they'd be destroyed, but eventually they were passed on to us. The details of the Aussie meet up were firmed up, with other teams agreeing on a date. 
</P><H5><B>Wet lab:</B></H5>
The synechocystis culture was subcultured this week! We transformed the ligated plasmid from last week, but no colonies were observed after 48 hours :( . Other ordered parts arrived from the registry, were plated and prepared for minipreps.
<H3 id="1june">1-5 June 2015</H3><P>Lots happening. We organised details of the gathering, started planning a UNSW debate about ethics in synthetic biology, and booked accommodation for several Boston nights. We ordered more parts from the registry, 
</P><H5><B>Wet lab:</B></H5>
Lactococcus expression vectors arrived from Addgene and were plated on ampicillin plates in preparation for minipreps 
M17 media was prepared. 
Gel was run with various parts, including ligation reactions, PCR-amplified parts, and minipreps of registry parts. Still troubleshooting. We are looking for a new gel tank to see if that was an issue.
<H3 id="8june">8-12 June 2015</H3><P>We were all off this week to study for exams. Instead we spent our time doing iGEM. On Monday, we had a day long meeting. We managed to decide on the final parts of our project and plan how we were going to get them done. Following this, we started designing all the DNA parts, primers, and G-blocks we would need. Rhys worked on lactococcus signal peptides and anchoring domains, Mac worked on designing primers for synechocystis plasmids, Izzy designed the Tse2 extraction primers, the pseudoknot G-blocks - all with Daniel's help. We also updated our rapidly growing DNA parts/biobricks database. 
</P><H5><B>Wet lab:</B></H5>
We cloned and miniprepped synechocystis addgene plasmids. However, this was not enough DNA to transform synechocystis so we grew more overnight cultures and did 5x minipreps for each plasmid. We also got ready to culture mammalian cells by ordering media and sourcing them within the school.  
<H3 id="15june">15-19 June 2015</H3><H5><B>Wet lab:</B></H5>
This was the first week of exams, but we still managed to make progress in the lab. Several previously digested parts were ligated to begin assembling the pHlow plasmid. Transformations were were not successful. The next round of ligations week was also not successful - there was growth on the positive control, but none on the ligated plates. For the third round of digests/ligations, we used NEB buffer instead of CutSmart buffer for the digests and transformations were successful. Kill switch design was progressed, and the two plasmid toxin-anti-toxin system was decided on. Primers for extraction of Tse2 from composite part were received, and resuspended. Synechocystis PCC6803 was transformed with Addgene plasmids over 5 days. Because of the slow growth time, we will not know if it was successful for 2-3 weeks. Because of a stuff up, cells were not in log phase when transformed. Lactococcus was subcultured on new GM17 plates but no growth was observed.  We received new Biobricks (our third shipment of extra bricks) and these were plated, cloned, and miniprepped.  
<H3 id="22june">22-26 June 2015</H3><P>This was the second week of exam period so we decided to start studying and attend our exams. Sorry iGEM lab.
</P><H3 id="29june">29 June-3 July</H3><P>We all finished exams this week and 3 out of 5 team members were away. 
</P><H5><B>Wet lab:</B></H5>
Shout out to Manan and Izzy for smashing out an impressive amount of labwork. Colony PCRs of E. coli transformed with ligated parts were run. Acid shock promoters were adapted for g-block synthesis and ordered. Pseudoknot g-blocks were received in the mail, and these were digested and ligated into chloramphenicol biobrick backbone. Constructs with pseudoknots and an IPTG-inducible promoter and LacZ were also created for a future characterisation assay. Kill switch assembly was commenced (more digests, ligations and transformations!). These appeared successful, but later investigation showed that due to mix ups with the backbones they were in, they may not have been. More ligations were done were done, using the same digested parts. These were successful, and the RFP (reporter) was observed. Constructs with IPTG-inducible promoter, Tsi2 and GFP were created for characterisation. All 4 standard biobrick backbones were successully amplified using PCR, and all backbones were purified and nanodropped. More digests and ligations were carried out, to continue assembly of pHlow plasmid. All were ligated into the psblA3 backbone and transformed.  Chromoproteins were extracted from the kit, cloned and miniprepped. In addition, antibiotic stocks were replenished  and various medias and buffers were prepared. In addition, Izzy started a standardised protocol sheet adapted from the Melbourne 2014 iGEM team. 
<H3 id="6july">6-10 July 2015</H3><P>There are now three people in the iGEM lab. Practically a crowd. 
</P><H5><B>Wet lab:</B></H5>
More digests, ligations and tranformations to get parts together. Starting to realise the serious downsides of 3A assembly. We have also started enforcing the use of both positive and negative controls for all transformations, as many transformations last week now seem to have not worked.  
We received ONPG, IPTG and arabinose required for kinetic assays and inducible promoters. The new gel tank did not have the miraculous effect we hoped - the bands are very blurry and low resolution. Also, the gel seems very soft and breaks easily. We tried with a new kind of agarose, and there is no improvement.  All negative controls for transformations this week contain growth. After investigating the effectiveness of our antibiotic stocks, we realised we probably had a serious contamination problem. We also figured out the problem with the lactococcus media, and transformed both lactococcus strains (L. lactis subsp. lactis LM0230 and L. lactis subsp. cremoris MG1363) using RFP in a kanamycin backbone. The method was complex, and optimised for very high efficiency. However, after doing it we decided that it was unneccessarily complex and will use a simplified method next time.
<H3 id="13july">13-17 July 2015</H3><H5><B>Wet lab:</B></H5>
We observed the result of lactococcus transformation - there was growth on all plates, including the negative control. The GM17 media was contaminated, based on visual observation.  
We tried to salvage some transformations by screening colonies in media with new antibiotics, but again there was growth in the negative controls. We also run an experiment to see if our plates were contaminated, and there was growth on negative controls. We made up all new stocks of antibiotics (Km is our KING), as well as started using filter tips and that appeared to have solved the problem. We were then able to redo many of the transformations from last week. We have been trying out our IPTG induction protocol, however there is no observable GFP expressed after induction. Additionally we have been PCR amplifying our gBLOCKS (we had to wait for new primers that bind to the prefix and suffix) so that we can retry our unsuccessful transformations.  This week saw us host the Australian iGEM meetup, with teams from both Macquarie and University of Sydney coming to our campus. We had an afternoon of presentations and casual beverages and a BBQ.
<H3 id="20july">20-24 July 2015</H3><H5><B>Wet lab:</B></H5>
 More digestions, ligations and transformations in an attempt to assemble testing constructs for our pseudoknots. We prepared agar plates for synechocystis transformations. Gblocks for our lactococcus optimised invasion device arrived. Overlap PCR was attempted to begin assembling the 5 and 7 part constructs. It was unsuccessful, and primers were ordered to amplify individual gblocks.. Synechocsytis agar plates were prepared.
<H3 id="27july">27-31 July 2015</H3><H5><B>Wet lab:</B></H5>
Primers arrived and a few gblocks were successfully amplified (checked with gel electrophoresis). Protocols were optimised, several PCRs were run and most gblocks were amplified. New primers were ordered for the unsuccessful gblocks. Synechocystis plasmid primers were assembled by overlap PCR. Our pseudoknot testing constructs don't seem to be ligating well, with very few colonies and bands that are at the wrong sizes. Something is afoot. We have started using tertacyclin as a 4th antibiotic. It seems very successful in its job of killing all bacteria (not so good for us though).
<H3 id="3august">3-7 August 2015</H3><H5><B>Wet lab:</B></H5>
New glbock primers arrived, and after two PCRs all parts were successfully amplified. New synechocystis plates were poured according to new recommendations (washing agar and addition of sodium thiosulfate). Midipreps of Synechocystis plasmids were attempted and failed (again). We received a new Synechocystis strain (from Dr. Rob Willows’ group at Macquarie Uni) with a higher transformation efficiency than our existing strain.  IPTG inductions of Tsi2 + GFP, Listerolysin + GFP and Invasin + GFP were undertaken and run on an SDS page gel. There seems to be no observable difference between the induced and uninduced cultures, though all 6 were expressing bands that weren't observed in the DH5-alpha reference strain. When the cutures were induced and run for a second time these bands were not visible. Tetracylin has yet again been successful at killing all the bacteria, however Amp and Cm have both had growth on the negative control.
<H3 id="10august">10-14 Augsut 2015</H3><H5><B>Wet lab:</B></H5>
Overlap PCR to assemble gblocks and synechocystis plasmids was attempted with mixed results. A flaw in the initial gblock design was fixed with a cheeky primer. We are now purifying PCR reactions in between each step. We attempted to clone the standard insert as we get ready to submit parts and backbones to the registry. Based on gel electrophoresis images, it seems likely that synechocystis plasmids are as desired. There were more transformations with growth on the negative. We have taken to colony PCRing all ligations to confirm inserts with some level of success.
<H3 id="17august">17-23 August 2015</H3><H5><B>Wet lab:</B></H5>
Notice how we now have a seven-day week. We are urgently trying to get labwork done and several team members were in the lab all day on Sunday. Gblock overlap PCR assembly is ongoing - troubleshooting involving annealing temperatures, denaturation times etc. Synechocystis inserts are being assembled using our modified method of restriction cloning. We now have a positive control for our IPTG induction-so we know our protocol is working, but not our constructs :( .
<H3 id="24august">24-30 August 2015</H3><H5><B>Wet lab:</B></H5>
We reamplified all our gblocks, in an attempt to troubleshoot non-overlapping parts. The war against lactococcus gblocks continue. The pressure is on the get Synechocystis plasmids assembled - if we didn’t have them ready to transform our cells by midway through this week we have very little chance of having successfully transformed synechocystis before the Jamboree. Ligations were unsuccessful and we were unable to transform synecho with our designed parts. Instead, we transformed with unmodified Addgene plasmids as well as a plasmid gifted to us by the Macquarie team. Despite not having our genetic constructs, this will allows us to check our optimised protocol (based off many discussions with experts) and streamline the process for future work. We've started using commercial competent cells (partially out of laziness, but also due to contamination).
<H3 id="1september">31 August-6 September</H3><H5><B>Wet lab:</B></H5>
We conducted a PCR screen of all our attempted to overlaps to see if we had any successes. There were none. We decided to do a last minute Gibson assembly attempt. We digested pSB1C3 with XbaI and SpeI, hoping that the remainder of the prefix and suffix would be sufficient as overlaps. 
We also continued the process of transforming synechocystis.  We seem to have our CP44 testing construct and PAR constructs with ASR and one of the lacto-promoters! YAY-We can prepare for lacZ assays next week. We have done a test run with ASR+PAR and no colour change was observed-maybe we need to optimise the protocol before doing a high through-put assay. We are doing lots of ligations to try and get things into Cm for submission.
<H3 id="7september">7-13 September</H3><H5><B>Wet lab:</B></H5>
We got decent DNA yields from our Gibson minipreps, everything is looking up Millhouse:). We have been doing a tonne of colony PCR on our ligations and getting parts for submission. pH altered LB agar was made to try and observe RFP production in our constructs. New fact, agar won't gel at 1.5% below pH 4, it is like a gross custard instead. After the custard incident, no RFP was observed on any of our plates anyway (not even for CP44). The LacZ assays likewise are having poor results-very little data has been produced. We are sleeping less and less and our caffeine intake is slowly increasing.

<H3 id="14september">14-19 September</H3><H5><B>Wet lab:</B></H5><B>FINAL COUNTDOWN</B>
WE HAVE LACZ DATA THAT CAN ACTUALLY MAKE A GRAPH-Izzy may have cried with joy at this moment in time. We also successfully have a whole heap of pseudoknot constructs, lactoccoccus optimized parts and other odds and ends. While this isn't nearly as much as we wanted to produce, for a first time team it is a pretty good effort. We managed to get everything in the plate and to the post office by 4:55 pm (only 5 minutes before it closed), just in time to start adding stuff to the wiki before the freeze. We fly out at 7:30 on the 19th (so before the wiki freeze happens for us) so we are in a rush to finish it before we have to leave. As of 2 am Iz is not yet packed. 
 Thank you all so much for being part of our iGEM adventure and following us through to the end. Those who have been unlucky enough do be with us from day one will know that words can't describe the level of effort that the team has put in.  Til iGEM 2016 this is UNSW out!!!

</DIV></DIV></DIV></DIV></BODY></HTML>