Team:BABS UNSW Australia/pseudoknots
Toggle navigation
Home
Project
Vision
Endosymbionts
pHlow system
Pseudoknots
Containment
Applications
Future Work
Human Practices
Biosafety
Team
Parts
Attributions
Notebook
Lab Work
Protocols
Background
Our Project
IBV Pseudoknot
MMTV Pseudoknot
HCV Pseudoknot
TGV Pseudoknot
References
Background
RNA pseudoknots are RNA structures with diverse functions [1], usually formed by the interaction of the loop of an RNA stem-loop with nucleotides downstream of the stem. These are referred to as H-type pseudoknots. Alternatively, two RNA stem-loops can interact via their loops, a type of pseudoknot known as kissing stem-loops. Among their diverse functions, certain pseudoknots can induce ribosome pausing during translation. In viral mRNA molecules, this mechanism is often combined with a heptanucleotide sequence upstream of the pseudoknot to induce a translation frameshift, which is necessary for the proper expression of certain viral ORFs [2].
Sequence and Structures of some RNA pseudoknots [1].
Our Project
TGV psudoknot illustrated by pseudoviewer
For our project, we decided to exploit the ribosome-pausing capabilities of viral RNA pseudoknots to delay the translation of genes transcribed in a polycistronic mRNA molecule. To achieve this, we placed the pseudoknots in between genes rather than within the ORFs of our composite parts.
We have designed a number of BioBricks containing one or more pseudoknots. The rationale of the design was the following: the sequences of the chosen pseudoknots were synthesised, including the 20 bases upstream and downstream of the pseudoknot, based on the original viral genomes. This is to give enough space in between pseudoknots, should one wish to string several pseudoknots together to further delay the translation of a gene. We also included the standard BioBrick prefix and suffix to the parts. Since the goal is to delay the ribosome before it reaches the next gene of the polycistronic mRNA, it is important to avoid creating any new ORFs in the pseudoknot BioBricks. Thus, when necessary, we removed any potential START codons from the parts in our design.  In the cases where potential ATG codons existed within the pseudoknot structure, we inverted any of the three base pairs in which the ATG triplet participated in order to keep the new sequence as close to the original as possible. We then conducted a few RNA pseudoknot predictions to verify that the new sequences yielded similar structures. The criteria chosen were to keep the estimated free energy of the new pseudoknots as close as possible to the original, while keeping the same stems and loops as the reported structures. Pseudoknot predictions were achieved with DotKnot [3, 4], while structure visualisation was carried out with PseudoViewer [5].
DotKnot was able to predict the correct stems and loops as reported in [2] for the wild type sequences. One exception was the kissing stem-loops of HCV229E, where DotKnot was unable to predict the full length of the second stem, probably due to a series of mismatches reported in this stem. However, the interaction of the loops was accurately predicted. This increased our confidence in the prediction of the new structures after removal of the START codons.
We chose to synthesise four known pseudoknots to test which, alone or in combination, could cause sufficient delay of the ribosome: three H-type pseudoknots, from the Mouse Mammary Tumor Virus (MMTV), the Infectious Bronchitis Virus (IBV) and the Transmissible Gastroenteritis Virus (TGV); and one kissing stem-loops type, from the Human Coronavirus 229E (HCV229E). These pseudoknots vary in length (34 nucleotides in the MMTV pseudoknot to over 200 nucleotides in the kissing stem-loops) and in the number of paired nucleotides. For example, for the H-type structures, the stem in the TGV pseudoknot is longer and contains no mismatches, whereas the IBV pseudoknot is shorter and contains one mismatch. As such, we expect these different structures to pause the ribosome with varying efficiencies.
Further, an interesting aspect of pseudoknots is the combinatorial approach that could be used to delay translation. Simply, do two pseudoknots in a row have a linear increase in the delay or will larger secondary structure? Additionally if a larger secondary structure forms will it cause an increase or a decrease based off the linear model or will it cause the ribosome to fall off the mRNA?
IBV (BBa_K1677391)
The IBV pseudoknot is found in the Infectious Bronchitis Virus. The 1x IBV structure is pictured below along with the predicted structure and data from DotKnot. There are several proposed models for the 2x and 3x IBV pseudoknots, however none of them appear to be an additive composition of 2 (or 3) 1x.
MMTV (BBa_K1677369)
The MMTV pseudoknot is found in the Mouse Mammary Tumour Virus A 1x structure can be observed below, along with some of the proposed 2x and 3 x structures. The global free energy minimum for both the 2x and 3x knots appear to consist of several discrete MMTV pseudoknots, however the local minimums propose a contradictory model.
HCV229E (BBa_K1677666)
The HCV229E pseudoknot originated from the human coronavirus 229E genome. HCV229E is the largest pseudoknot in the suite. Dotknot could not correctly predict the 1x HCV229E structure and the size of the 2x and 3x pseudoknots was too large to be predicted by the current programs. The nature of the HCE229E combinations will need to be elucidated by experiments, or additional prediction software.
TGV (BBa_K1677743)
Pseudoknot TGV was derived from the genome of the transmissible gastroenteritis virus. Part of the 1x structure is retained in the global 2x prediction, however the first pseudoknot and part of the second fuse together into a larger structure. The 3x TGV retains most of the 2x structure, with an additional formation towards the end that doesnâ€™t resemble a 1x knot.
References
Staple, D. W., Butcher, S. E., Pseudoknots: RNA structures with diverse functions. PLoS biology 2005, 3, e213.
Baranov, P. V., Henderson, C. M., Anderson, C. B., Gesteland, R. F., et al., Programmed ribosomal frameshifting in decoding the SARS-CoV genome. Virology 2005, 332, 498-510.
Sperschneider, J., Datta, A., DotKnot: pseudoknot prediction using the probability dot plot under a refined energy model. Nucleic acids research 2010, 38, e103.
Sperschneider, J., Datta, A., Wise, M. J., Heuristic RNA pseudoknot prediction including intramolecular kissing hairpins. Rna 2011, 17, 27-38.
Byun, Y., Han, K., PseudoViewer: web application and web service for visualizing RNA pseudoknots and secondary structures. Nucleic acids research 2006, 34, W416-422.
