<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BABS_UNSW_Australia_project_phlow skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BABS UNSW Australia/project/phlow</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/overview">Vision</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/endosymbionts">Endosymbionts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/phlow">pHlow system</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/pseudoknots">Pseudoknots</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/containment">Containment</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futureapps">Applications</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futurework">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices/Biosafety">Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Protocols">Protocols</A></LI></DIV></DIV></DIV><NAV class="col-xs-3 bs-docs-sidebar"><UL id="sidebar" class="nav nav-stacked"><LI>Phlow system
            </LI><LI><A href="#overview">Overview</A></LI><LI><A href="#invasin">Invasin</A></LI><LI><A href="#llo">Listerolysin</A></LI><LI><A href="#acid">pH promoters</A></LI><LI><A href="#eco"> E. Coli ASR Promoter</A></LI><LI><A href="#lacto">Lactococcus Promoters</A></LI><LI><A href="#asrref">References</A></LI></UL></NAV><DIV class="col-xs-9"><H2>Overview</H2><P>The two proteins essential for bacterial invasion of mammalian cytoplasm, invasin and listeriolysin, were already in the registry, and were previously shown to work in a very simple genetic circuit - promoter-rbs-protein-rbs-protein. Our aim with the ‘pHlow’ system was to more finely tune the expression of these proteins, as well as improve the biosafety of modified organisms.
</P><P> The story of the pHlow plasmid begins with the induction of Invasin, leading to beta-integrin mediated invasion of the mammalian host. The future endosynbiont is engulfed in the lysosome, experiencing a drop in the external pH and triggering the induction of Listeriolysin. Cre-recombinase, while under the same promoter as Listeriolysin, is delayed by the RNA secondary structures until the endosynbiont has escaped into the host’s cytoplasm. The Cre-recombinase then acts on the lox sites, removing the Invasin, Listeriolysin and Cre-recombinase from the plasmid so that the endosynbiont cannot escape and invade other cells. Follow us on the pHlow journey through the various components. </P><H2>Invasin</H2><P>Invasin is a surface protein found in a range of pathogenic bacteria, in particular <I>Yersinia</I> and <I>Listeria</I> species. Invasin facilitates the phagocytosis of the bacteria via interaction with multiple β1-integrins on the incumbent cells surface. This invasion process can be modelled by a power law dependence, as this accounts for the large discrepancies in reported invasion efficiency [1]. Our particular brand of invasin was originally sourced from <I>Yersinia enterocolitica</I>. The protein itself consists of 5 chain-link like domains to which the integrin region is affixed. 
<FIGURE><FIGCAPTION>A representation of Invasin D1-D5 [2]</FIGCAPTION></FIGURE></P><H2>Listeriolysin</H2><P>Listeriolysin O (LLO) is a pore-forming toxin that is commonly produced by <I>Listeria Monocytogenes</I> the pathogen responsible for listeriosis. When <I>Listera</I> becomes engulfed by a phagosome in a host cell, LLO is thiol-activated and proceeds to produce holes in the phagosome. Once the phagosome has been lysed the cells are free to inhabit the cytoplasm where LLO's effect is reduced due to its lower concentration and the higher pH of the cytosolic environment. By incorporating both Invasin and LLO into the pHlow plasmid we hope to optimise the invasion process.
</P><FIGURE><FIGCAPTION>Crystal structure of listeriolysin [3].</FIGCAPTION></FIGURE><H2>Acid shock response promoter</H2><P>An essential component of the pHlow plasmid is listeriolysin only being expressed once the cell in within the lysosome. Bacteria often are subjected to environmental stressors in the form of pH change, requiring differential gene expression, and hence have pH specific promoters. We have had these endogenous promoters synthesized and have characterised them for use in the pHlow plasmid.</P><H4><B> E. Coli ASR Promoter</B></H4><P>The acid shock response protein is present in the genome of E. Coli K12 and was first submitted into the registry by IIT Delhi 2013 (<A href="http://parts.igem.org/Part:BBa_K1170000">http://parts.igem.org/Part:BBa_K1170000</A>) . This part was not available from the registry so we had it synthesised as a gBLOCK from IDT and biobricked it so that future years can access the part without synthesis.
</P><H4><B>Lactococcus Promoters</B></H4><P>Lactococcus lactis produce lactate in their own environment and have several genes that are upregulated due to the low pH they create.The two promoters we are investigating are p170 (<A href="http://parts.igem.org/Part:BBa_K1677300">BBa_K1677300</A>) and p170-CP25 (<A href="http://parts.igem.org/Part:BBa_K1677301">BBa_K1677301</A>).The reasons for attempting to characterise both promoters is due to their relative levels of expression;  p170 has a higher level of expression, but is a more leaky promoter, whereas p170-CP25 is more tightly regulated. P170 has been characterised to be induced at pH 6.5 and pH 6.0 but only when the culture is in stationary phase [4]. It is important to note that all characterization of these promoters occurred in E. Coli, not L. lactis and that further characterisation needs to be undertaken to conclusively determine their results. 
</P><P>As can be seen here, there is a slight increase in promoter activity at lower pH’s for both ASR and p170. For the purposes of the pHlow plasmid we ideally desire a promoter that will have minimal expression under normal conditions, as can be seen below the overall activity of either promoter is significantly less than CP44. Further characterization attempts should look at using a weaker promoter for the standard comparison. 
</P><P> It can be observed that CP44 has decreased activity at lower pH values despite being normalised for OD600 (conferring cell density). </P><P> While there is evidence that both ASR and p170 can be used to upregulate expression under acidic conditions, they still have expression in a neutral environment. While this may be helpful for future iGEM projects the pHlow plasmid relies on a tightly regulated and timed expression of listeriolysin and cre-recombinase, something that these two promoters don’t appear to confer. </P><H4><B> References </B></H4><P><OL><LI>Lee, Tae J. et al. 'A Power-Law Dependence Of Bacterial Invasion On Mammalian Host Receptors'. PLOS Computational Biology 11.4 (2015): e1004203. Web.
      </LI><LI>Hamburger, Z. A. 'Crystal Structure Of Invasin: A Bacterial Integrin-Binding Protein'. Science 286.5438 (1999): 291-295. Web.
      </LI><LI>Köster, Stefan et al. 'Crystal Structure Of Listeriolysin O Reveals Molecular Details Of Oligomerization And Pore Formation'. Nature Communications 5 (2014): n. pag. Web.</LI><LI>Madsen, S. M., Arnau, J., Vrang, A., Givskov, M., &amp; Israelsen, H. (1999). Molecular characterization of the pH‐inducible and growth phase‐dependent promoter P170 of Lactococcus lactis. Molecular microbiology, 32(1), 75-87 
      </LI></OL></P></DIV></DIV></DIV></DIV></BODY></HTML>