<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BABS_UNSW_Australia_endosymbionts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BABS UNSW Australia/endosymbionts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia">Home</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/overview">Vision</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/endosymbionts">Endosymbionts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/phlow">pHlow system</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/pseudoknots">Pseudoknots</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/containment">Containment</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futureapps">Applications</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/project/futurework">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Practices/Biosafety">Biosafety</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Attributions">Attributions</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Notebook2">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:BABS_UNSW_Australia/Protocols">Protocols</A></LI></DIV></DIV></DIV><NAV class="col-xs-3 bs-docs-sidebar"><UL id="sidebar" class="nav nav-stacked"><LI>Endosymbionts
            </LI><LI><A href="#synecho"><I>Synechocystis</I></A></LI><LI><A href="#lacto"><I>Lactococcus</I></A></LI><LI><A href="#ecoli"><I>E.coli</I></A></LI></UL></NAV><DIV class="col-xs-9"><H2><I>Synechocystis </I>PCC6803</H2><H4><STRONG>Introduction</STRONG></H4><I>Synechocystis </I> PCC6803 is a freshwater cyanobacterial species. It is a model organism for photosynthetic bacteria. It is capable of both autotrophic (light-dependent) and heterotrophic (organic carbon source-dependent) lifestyles.
<H4>Why?</H4><P>We chose <I>Synechocystis</I> for the same reason it is so difficult to work with in the lab – it has a 1-2 day doubling time. This means it is unlikely to overwhelm any mammalian cell it is inhabiting. Additionally, it has been previously shown [1] to survive stably inside the cytoplasmic environment. We were also inspired by the sci-fi connotations of having photosynthesis-capable organisms living in our cells – could we eventually create human chloroplasts and solve the world’s food shortage? Finally, the strain we used did not produce any toxic secondary metabolites and was completely non-pathogenic. <I>Synechocystis</I> has actually been shown to be part of the healthy human microbiome .</P><H4><STRONG>Design</STRONG></H4><UL><LI>We decided to use the same invasion device used for E. coli (i.e. invasion and listeriolysin), as the unmodified genes were previously used and seemed to work for mammalian cell invasion [1]. Therefore, the bulk of our work was in designing a biobrick compatible integrative plasmid.
   </LI><LI>Using the open source plasmid registry Addgene, we were able to source previously tested integrative plasmids, psbA2-PHLS (b) and cpc-PHLS (g) (Addgene accession numbers: 52307 and 52311, respectively) [2].
   </LI><LI>Using long primers containing annealing regions and tails for extension containing biobrick prefixes and suffixes, we were able to tailor these plasmids for use with Biobrick standards. The plasmids were originally designed to test PHLS (beta-phellandrene) under endogenous promoters in <I> Synechocystis </I>. Our modified version of the plasmids allow integration of any Biobrick compatible gene construct, assembled through 3A assembly. 
   </LI><LI>The two 500 bp recombination sites with sequence homology to chromosome regions enable double recombination, and allow the gene of interest to be inserted into the chromosome. 
   </LI><LI>The plasmids carry an ampicillin resistance gene on the portion lost after integration, and a chloramphenicol resistance gene in between recombination suites. Ampicillin is used for selection during cloning of plasmid in <I>E. coli</I>, while chloramphenicol is used for selection during integrative transformation of <I>Synechocystis</I>.
   </LI><LI>The terminator following the first recombination site prevents read-through transcription of the protein the insert gene is interrupting.
   </LI></UL><H4><STRONG>Challenges</STRONG></H4><P>Despite being a model organism, there is a lack of well-characterised genetic parts for use in <I>Synechocystis</I>. It is very different functionally and phylogenetically to <I>E. coli</I>, and standard parts often do not function in <I>Synechocystis</I>. For example, there are currently no effective inducible promoters, and no effective terminators. This made it challenging to effectively implement our biosafety strategy. Next, plasmids do not exist stably inside <I>Synechocystis</I> cells - transformants are created through use of integrative plasmids. These homologously recombine into the chromosomes. <I>Synechocystis</I> has 12 chromosomes, therefore gradually increasing antibiotic selection pressure is required to ensure integration into all chromosome copies. In the lab, the 2-4 week transformation time (due to slow doubling and gradual application of antibiotics) was also prohibitive, especially in the relatively short timeframe of the competition. Finally, <I>Synechocystis</I> PCC6803 is generally naturally competent – transformations involve simply exposing the organism to high concentrations of desired plasmid. However, we found out halfway through the competition that the strain we were using was most likely non-competent and therefore fundamentally not transformable. We then had to source a transformable strain. 
</P><FIGCAPTION><I>Synechocystis</I> transformation two weeks post-innoculation</FIGCAPTION><H4><STRONG>References</STRONG></H4><P> [1] Agapakis, C. M., Niederholtmeyer, H., Noche, R. R., Lieberman, T. D., Megason, S. G., Way, J. C., &amp; Silver, P. A. (2011). Towards a synthetic chloroplast.PLoS One, 6(4), e18877.
   </P><P> [2] Formighieri, C., &amp; Melis, A. (2014). Regulation of β-phellandrene synthase gene expression, recombinant protein accumulation, and monoterpene hydrocarbons production in Synechocystis transformants. Planta, 240(2), 309-324.
   </P><H2><I>Lactococcus lactis</I> LMO230</H2><H4><STRONG>Background</STRONG></H4><P><I>L. lactis</I> is a gram positive bacterium commonly used in the food industry. Most strains are hosts to large numbers of plasmids, which encode industry-relevant functions such as extracellular proteases required for lactose digestion and fermentation. Strain LM0230 is a plasmid-free strain, and is thus well-suited for molecular biology applications. It allows for easier addition and detection of engineered plasmids, and ensures that any recombinant proteins expressed will be intact if excreted (no protease activity).
</P><H4><STRONG>Why?</STRONG></H4><P>L. lactis strains are already found in high volumes inside humans, as essential components of the gut microbiome. They are a GRAS (Generally Regarded As Safe) organism, which may make it easier to convince people to have them introduced to their cells. In addition, L. lactis strain was the first ever live bacteria used for the treatment of human disease (in 2006, for the treatment of Chron’s disease) [1]. It is a facultative aerobe, and its fermentative metabolism may have interesting implications for intracellular survival. Additionally, many well-characterised genetic parts exist and are available in the iGEM registry and elsewhere. Like E. coli, it is quite hardy and easy to work with in the lab. Finally, <I>Lactococcus</I> survival in mammalian cytoplasm has never been assayed. Contrasting our rationale for <I>Synechocystis</I>, we wanted to explore unchartered territory. 
</P><H4><STRONG>Design</STRONG></H4><P>Because invasin is a membrane protein that binds to mammalian membrane beta-integrin receptors, it is essential for it to be localised to the bacterial outer membrane. We exchanged the localisation domains found on invasin (originally derived from Yersinia Enterocolitica) for membrane localisation peptides endogenous to <I>L. lactis</I>. These were protein A peptide USP45, and the LysM protein anchor. We also codon optimised invasion and listeriolysin for best translation efficiencies in <I>L. lactis</I>. Many well-characterised constitutive promoters from <I>Lactococcus</I> species exist in the registry – however, most were characterised in <I>E. coli</I>, not in <I>Lactococcus</I>.
</P><H4><STRONG>Challenges</STRONG></H4><P>Because <I>L. lactis</I> LM0230 is lacking all plasmids, several intrinsic abilities are removed. For example, it cannot utilise lactose as a carbon source. Therefore, we added glucose to the media as a substrate.  The main challenge, however arose from our use of the Invitrogen g-block system. Certain design features (such as limitations on repeated codons) forced modifications on optimal codons, and their limited length means you cannot synthesise whole genes. To assemble our constructs required overlap PCR (we designed parts to have ~20bp overlaps). This required primer design for amplification of all g-blocks, and several attempts to optimise PCR protocols for assembling overlapping part. After a multitude of failed overlap PCR attempts, we attempted Gibson assembly on the same parts... and it worked. 
</P><H4><STRONG>References</STRONG></H4><P> [1] Braat, H., Rottiers, P., Hommes, D. W., Huyghebaert, N., Remaut, E., Remon, J. P., ... &amp; Steidler, L. (2006). A phase I trial with transgenic bacteria expressing interleukin-10 in Crohn’s disease. Clinical gastroenterology and hepatology, 4(6), 754-759. </P><H2><I>Escherichia coli</I></H2><P>We used <I>E. coli</I> for the same reason everyone uses <I>E. coli</I> - it’s cheap and easy like a Sunday morning.</P><H4><STRONG>Design</STRONG></H4><P>The invasin and lysteriolysin parts were biobricked by Warsaw in 2009 and 2010 and hence llittle design effort was needed on this front. The flagship theoretical biobrick that has driven our project, the pHlow plasmid, was initially planned to be used in an <I>E. Coli</I> chassis.
</P><H4><STRONG>Challenges</STRONG></H4><P>However due to its rapid rate of replication, <I>E. Coli</I> would never be an ideal endosymbiont as it would rapidly overwhelm the cell. 
</P></DIV></DIV></DIV></DIV></BODY></HTML>