"
Page
Discussion
View source
History
teams
Log in
 
Team:Tsinghua/Notebook/Protocol/Molecular cloning
From 2014.igem.org
MAIN PAGE
INTRODUCTION
Overview
Diabetes Type I
Gene Therapy
PROJECT
Overview
The Virus
The Drug
The Microbe
The Cocktail
Modeling
Parts
SAFETY
NOTEBOOK
OVERVIEW
PROTOCOL
LABLOG
HUMAN PRACTICE
OVERVIEW
GALLERY
OUTREACH
VIDEO
TEAM
Notebook: Protocol
Return to: Notebook -> Protocol
Molecular Cloning
 
Polymerase Chain Reaction (PCR)
1. The master mix for reactions with FastPfu DNA polymerase contained:
Reagent
Volume
Final Concentration
5x FastPfu Buffer
10 ul
1X
10 mM
1 ul
0.2 mM
Primer 1 (25 pmol/ul)
1 ul
0.5 pmol/ul
Primer 2 (25 pmol/ul)
1 ul
0.5 pmol/ul
FastPfu (5U/ul)
0.4 ul
2U/50ul
Template DNA
variable
50 pg – 1 ug
ddH2O
To 50 ul
2. The master mix for reactions with Phusion DNA polymerase contained:
Reagent
Volume
Final Concentration
Phusion DNA Polymerase
0.5 ul
1.0 units/50 ul PCR
DMSO (optional)
(1.5 ul)
3%
Template DNA
variable
250 ng
10 uM Forward Primer
2.5 ul
0.5 uM
10 uM Reverse Primer
2.5 ul
0.5 uM
10 mM dNTPs
1 ul
200 uM
5X Phusion HF Buffer
10 ul
1X
ddH2O
To 50 ul
3. The master mix for reactions with Taq DNA polymerase contained:
Reagent
Volume
Final Concentration
10X Standard Taq Buffer
5 ul
1X
10 mM dNTPs
1 ul
200 uM
10 uM Forward Primer
1 ul
0.2 uM
10 uM Reverse Primer
1 ul
0.2 uM
Template DNA
Variable
Variable
Taq DNA polymerase
0.25 ul
1.25 units/50 ul PCR
ddH2O
To 50 ul
4. All temperature profiles were optimized according to manufacturer’s  protocol, the melting temperature of primers, and the length of the desired PCR  products.
Basic temperature profiles:
Step
Temperature
Time
Initial Denaturation
95℃
3 min
30 Cycles
95℃
30 sec
55℃
30 sec
72℃
1 min
Final Extension
72℃
5 min
Hold
4℃
 
Gel purification of  DNA Fragment
Agarose Electrophoresis:
1. A mixture of various sized DNA fragments were separated in an agarose gel (from 0.8 to 1.5% agarose in 1x TAE buffer ethidium bromide) at a constant voltage of 150 V.
2. UV light (λ = 254 nm) was used to visualize DNA with intercalated ethidium bromide.
Gel Purification of DNA fragment:
1. The band with the desired DNA fragments were excised from the gel, using a clean scalpel.
2. DNA was isolated from the gel slice with Gel Extraction Kit according to the manufacturer’s protocol.
3. Purity and amount of DNA was determined using NanoDrop.
Restriction Digestion
1. To digest the desired DNA restriction reactions were prepared as follows:
For analysis of cloned DNA
2µl of the appropriate restriction buffer (10X)
0.5 µL restriction enzyme
Bring volume to 20 µL with nuclease-free water.
Or
For isolation of specific DNA
2µl of the appropriate restriction buffer (10X)
Up to 2 µL restriction enzyme
Bring volume to 50 µL with nuclease-free water.
2. The sample was incubated at optimal temperature for the restriction enzymes.
3. Analysis of fragmented DNA was done by gel electrophoresis.
4. Desired DNA fragment was excised and purified using suitable DNA purification kit.
Ligation
T4 ligase ligates the 5' phosphate and the 3'-hydroxyl groups of DNA.
Vector and insert concentrations were estimated and insert and vector  fragments joined in a molar ratio of 3:1 (100-150ng Vector DNA).
A ligation mixture was prepared:
1X ligase buffer (10X) 
1 µL T4 ligase (3 U/µL) 
Bring volume to 10 µL with nuclease-free water. 
Reactions were incubated at 17 °C for 4 to 18 hours. 
After  incubation part of the ligation mixture was used for the transformation of  bacterial cells (see: transformation of bacteria).
Culturing bacteria
For plasmid DNA propagation two bacterial strains were used: DH5alpha and  TransT1.
Growth media for bacteria:
Luria Broth (LB) : 10 g/L tryptone, 5 g/L yeast extract, 10 g/L NaCl,  media is supplemented with suitable antibiotics depending on the selection  marker on the transfected plasmid: ampicilin 100 mg/L or kanamycin 50 mg/L.
LB agar plates: LB with 1.5% agar, media is supplemented with suitable  antibiotics depending on the selection marker on the transfected plasmid.
 
Bacteria transformation
E. coli DH5alpha and TransT1 competent cells were used for the propagation  of plasmid DNA.
100  µL of competent cells were thawed on ice. 
50 – 400 ng DNA solution was added to competent bacterial cells (depending on  the concentration of the DNA solution). 
A mixture of cells and DNA solution was incubated on ice for 30-60 minutes. 
The mixture was heat-shocked for 3 minutes at 42 °C. 
Cooled for 3 minutes on ice. 
500 µL of preheated antibiotic free LB-medium was added and incubated for one  hour at 37 °C with agitation for the purpose of inducing antibiotic resistance. 
The selection for plasmid containing and therefore antibiotic resistant  bacteria was conducted by plating them on antibiotic containing LB-agar plates.
 
Plasmid DNA isolation
MINI PREPs for analysis and sequencing:
A single colony was picked from a LB-agar plate or glycerol stock and  inoculated in 10 mL of LB-medium with the appropriate antibiotic for selection  (100 mg/L ampicillin, 50 mg/L kanamycin, 35 mg/L chloramphenicol). 
Bacteria were grown over night at 37 °C with agitation. 
Plasmid DNA was isolated from 6-10 mL of over-night culture with plasmid miniprep  kit according to the manufacturer's protocol. 
Amounts ranging from 6-10 µg of plasmid DNA were obtained. 
The purity and concentration of the isolated DNA was analyzed using NanoDrop.
 
Return to: Notebook -> Protocol
 
Retrieved from " http://2014.igem.org/Team:Tsinghua/Notebook/Protocol/Molecular_cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
