<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#000000;background-color:#ffffff;" cellpadding="3" cellspacing="1" border="3" width="80%" align="center"><TBODY><TR><TH align="center"><A href="/Team:UNIPV-Pavia" title="Team:UNIPV-Pavia">Home</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Team" title="Team:UNIPV-Pavia/Team">The Team</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Project" title="Team:UNIPV-Pavia/Project">The Project</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Safety" title="Team:UNIPV-Pavia/Safety">Biological Safety</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Parts" title="Team:UNIPV-Pavia/Parts">Parts Submitted to the Registry</A></TH></TR><TR><TH align="center"><A href="/Team:UNIPV-Pavia/Dry_Lab" title="Team:UNIPV-Pavia/Dry Lab">Dry Lab</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Wet_Lab" title="Team:UNIPV-Pavia/Wet Lab">Wet Lab</A></TH><TH align="center"><STRONG class="selflink">Modeling</STRONG></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Protocols" title="Team:UNIPV-Pavia/Protocols">Protocols</A></TH><TH align="center"><A href="/Team:UNIPV-Pavia/Notebook" title="Team:UNIPV-Pavia/Notebook">Activity Notebook</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Mathematical_modeling_page"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Mathematical modeling page</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Why_writing_a_mathematical_model.3F"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Why writing a mathematical model?</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Equations_for_gene_networks"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Equations for gene networks</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Binding_of_a_ligand_to_a_molecule:_Hill_equation"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Binding of a ligand to a molecule: Hill equation</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Regulated_transcription"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Regulated transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#ODE_for_activated_transcription"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">ODE for activated transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#ODE_for_inhibited_transcription"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">ODE for inhibited transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#ODE_for_constitutive_transcription"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">ODE for constitutive transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Transcription_factors_in_active_form"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Transcription factors in active form</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#ODE_for_protein_production"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">ODE for protein production</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-11"><A href="#Our_model"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Our model</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Mux"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Mux</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Demux"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Demux</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Simulations"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Mux_2"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Mux</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Demux_2"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Demux</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Mathematical_modeling_page"><B>Mathematical modeling page</B></SPAN></H1><P>In this section we explain two dynamic models that can be used to describe the gene networks in our project. After a brief overview about the motivation of a mathematical model, we will illustrate the general formulas we used, we will show the complete ODE models for Mux and Demux gene networks, and then we will report results of some simulations performed with Matlab and Simulink.
</P><H2><SPAN class="mw-headline" id="Why_writing_a_mathematical_model.3F"><B>Why writing a mathematical model?</B></SPAN></H2><P>The purposes of writing mathematical models for gene networks can be:
</P><UL><LI><B>Prediction</B>: a good and well identificated model can be used in simulations to predict real system behavior. In particular we could be interested in system output in response to never seen inputs. In this way, the system can be tested 'in silico', without performing real experiments 'in vitro' or 'in vivo'.
</LI><LI><B>Parameter identification</B>: we already wrote that it is very important to estimate all the parameters involved in the model, in order to perform realistic simulations. Another goal that can be reached with parameter identification is 'network summarization', in fact estimated parameters can be used as 'behavior indexes' for the network (or a part of it). These indexes can be very useful for synthetic biologists to choose and compare BioBrick standard parts for genetic circuits design, just like electronic engineers choose, for example, a Zener diode, knowing its Zener voltage.
</LI></UL><H2><SPAN class="mw-headline" id="Equations_for_gene_networks"><B>Equations for gene networks</B></SPAN></H2><P>In this paragraph, mathematical modeling for gene and protein interactions will be described.
</P><H3><SPAN class="mw-headline" id="Binding_of_a_ligand_to_a_molecule:_Hill_equation"><B>Binding of a ligand to a molecule: Hill equation</B></SPAN></H3><P>The fraction of a molecule saturated by a ligand ( = the probability that the molecule is bound to a ligand) can be expressed as a function of ligand concentration using Hill equation:
</P><UL><LI><B>Prob:</B> the probability that the molecule is bound to the ligand
</LI><LI><B>[L]:</B> concentration of the ligand
</LI><LI><B>K50:</B> dissociation constant. It is the concentration producing a probability of 0.5
</LI><LI><B>n:</B> Hill coefficient. It describes binding cooperativity
</LI></UL><H3><SPAN class="mw-headline" id="Regulated_transcription"><B>Regulated transcription</B></SPAN></H3><P>Transcription of a mRNA molecule can be regulated by transcription factors in active form. These factors can be activators or inhibitors: they respectively increase and decrease the probability that RNA polymerase binds the promoter.
This probability must be function of active transcription factor concentration and can be modeled using Hill equation.
If factor in active form activates transcription, we can write:
</P><UL><LI><B>Prob:</B> probability that the gene is transcripted
</LI><LI><B>[T]:</B> concentration of the transcription activator in active form
</LI><LI><B>K50:</B> dissociation constant. It is the concentration producing a probability of 0.5
</LI><LI><B>n:</B> Hill coefficient (n&gt;0)
</LI></UL><P>while, if factor in active form inhibits transcription, we are interested in unbound promoter and so we can write:
</P><UL><LI><B>Prob:</B> probability that the gene is transcripted
</LI><LI><B>[T]:</B> concentration of the transcription inhibitor in active form
</LI><LI><B>K50:</B> dissociation constant. It is the concentration producing a probability of 0.5
</LI><LI><B>n:</B> Hill coefficient (n&gt;0)
</LI></UL><P>The equations show that:
</P><UL><LI>In activation formula, if [T]=0 the trascription probability is Prob=0, while the maximum probability, Prob=1, is reached asymptotically for [T]-&gt;Inf
</LI><LI>In inhibition formula, if [T]=0 the trascription probability is Prob=1, while the minimum probability, Prob=0, is reached asymptotically for [T]-&gt;Inf
</LI></UL><P>These formulas can be included in a differential equation that can describe the dynamic behavior of a mRNA molecule as a function of the factor that regulates its transcription.
</P><H3><SPAN class="mw-headline" id="ODE_for_activated_transcription"> ODE for activated transcription </SPAN></H3><UL><LI><B>Vmax:</B> maximum transcription rate
</LI><LI><B>[m]:</B> concentration of mRNA molecule
</LI><LI><B>[T]:</B> concentration of the transcription activator in active form
</LI><LI><B>K50:</B> dissociation constant
</LI><LI><B>n:</B> Hill coefficient (n&gt;0)
</LI><LI><B>delta:</B> mRNA degradation constant
</LI><LI><B>a:</B> leakage factor that can modellize promoter basic activity. It is a percentage of Vmax
</LI></UL><H3><SPAN class="mw-headline" id="ODE_for_inhibited_transcription"> ODE for inhibited transcription </SPAN></H3><UL><LI><B>Vmax:</B> maximum transcription rate
</LI><LI><B>[m]:</B> concentration of mRNA molecule
</LI><LI><B>[T]:</B> concentration of the transcription inhibitor in active form
</LI><LI><B>K50:</B> dissociation constant
</LI><LI><B>n:</B> Hill coefficient (n&gt;0)
</LI><LI><B>delta:</B> mRNA degradation rate
</LI><LI><B>a:</B> leakage factor that can modellize promoter basic activity. It is a percentage of Vmax
</LI></UL><H3><SPAN class="mw-headline" id="ODE_for_constitutive_transcription"> ODE for constitutive transcription </SPAN></H3><UL><LI><B>c:</B> constant transcription rate
</LI><LI><B>[m]:</B> concentration of mRNA molecule
</LI><LI><B>delta:</B> mRNA degradation rate
</LI></UL><H3><SPAN class="mw-headline" id="Transcription_factors_in_active_form"><B>Transcription factors in active form</B></SPAN></H3><P>Formulas described so far are dependent on the concentration of transcription factor in active form. But how can we calculate the amount of transcription factor in active form?
Assuming that the transcription factor can be activated or inhibited by a <I>inducer</I> factor (that can be an exogenous input), we can use Hill equation again to calculate the concentration of bound and unbound transcription factor as a function of the inducer factor:
</P><UL><LI><B>[I]:</B> concentration of the inducer
</LI><LI><B>[T]t:</B> total concentration of the transcription factor
</LI><LI><B>[T]u:</B> concentration of unbound transcription factor
</LI><LI><B>[T]b:</B> concentration of bound transcription factor
</LI><LI><B>K50:</B> dissociation constant
</LI><LI><B>n:</B> Hill coefficient (n&gt;0)
</LI></UL><P>If active form is bound transcripion factor, we are interested in Tb, while if active form is unbound transcription factor, we are interested in Tu.
</P><H3><SPAN class="mw-headline" id="ODE_for_protein_production"><B>ODE for protein production</B></SPAN></H3><P>We have to modellize translation of mRNA molecules. We chose to describe it as a differential equation in which protein production and protein degradation are linear:
</P><UL><LI><B>[P]:</B> concentration of protein
</LI><LI><B>[m]:</B> concentration of mRNA molecule
</LI><LI><B>alpha:</B> translation rate
</LI><LI><B>beta</B> protein degradation rate
</LI></UL><H2><SPAN class="mw-headline" id="Our_model"><B>Our model</B></SPAN></H2><P>We considered Tetracycline, IPTG, Arabinose and GFP respectively as CH0, CH1, SEL and OUT signals for Mux, while we considered IPTG, Tetracycline, RFP and GFP respectively as IN, SEL, OUT0 and OUT1 signals for Demux. For all these signals, logic 1 corresponds to the presence of the molecule.

According to the equations described so far, every protein and mRNA is a state variable for the model. Mux and Demux models are described in the next paragraphs.

We considered full ODE models, in which transcription and translation processes are separated. In many contexts, anyway, we can write only one differential equation to describe a &quot;regulated protein production&quot;, in which trascription process is hidden. In this section we preferred to be general and we considered the transcription dynamic because our aim is to provide a mathematical model for future parameter identification. We also decided to neglect AHL synthesis dynamic, considering luxI protein a &quot;direct&quot; activator for luxR.

In the following paragraphs, the complete ODE systems and the Simulink block diagrams will be reported.
</P><H3><SPAN class="mw-headline" id="Mux"><B>Mux</B></SPAN></H3><H3><SPAN class="mw-headline" id="Demux"><B>Demux</B></SPAN></H3><H2><SPAN class="mw-headline" id="Simulations"><B>Simulations</B></SPAN></H2><P>The written models have been simulated in Matlab and Simulink, in order to perform a preliminary heuristic study of the system behavior and to search for critical parameters. For simulations, we used dimensionless ODEs to avoid numeric errors due to small parameter values, so results are reported in arbitrary units. We defined ODE systems in Simulink (block diagrams showed above) and then solved them using ode15s, a multistep ODE solver for stiff problems.

Parameters have been calibrated considering the values reported in literature and in some other iGEM wiki pages. In some cases values were missing, so we fixed these missing values at 1*10^om, where &quot;om&quot; is the (median) order of magnitude of the parameters having the same meaning (e.g. protein degradation rates).
In other cases we found different values for the same parameter, so we computed the mean value.

An important feature that will have to be studied is systems' basic activity. In other words, can genetic Mux discriminate the output in response to a low input signal and the output in response to a high input signal? Moreover, does Demux have a non negligible baseline activity that turns on the output channel that should be unactive? One of the parameters that plays a very important role in basic activities is the leakage factor (&quot;a&quot;, that is the probability that a mRNA is transcribed randomly). This parameter depends on the specific promoter. Future estimation of &quot;a&quot; for all the promoters will be useful to predict &quot;false positive&quot; system outputs. High &quot;a&quot; values give high basic activities, while for a-&gt;0 transcriptional regulation becomes &quot;ideal&quot;.

At first, we wrote the ODE models with only one &quot;a&quot; parameter, but then we decided to consider different leakage factors for the promoters in the network.

Here we report some simulation examples. The initial states (t=0) have been fixed to 0, but here we are interested into steady state values.
</P><H3><SPAN class="mw-headline" id="Mux_2"><B>Mux</B></SPAN></H3><P>We distinguished lux and las system contributions for output GFP.
</P><P><B>SIMULATION 1</B></P><UL><LI>Tetracycline = 10
</LI><LI>IPTG = 0
</LI><LI>Arabinose = 0
</LI><LI>Tetracycline channel (CH0) is selected and Tetracycline is present, so we expect to find a high GFP(lux) value and a low GFP(las) value.
</LI></UL><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Mux SIMULATION 1 - tetR, lacI, cI and GFPs</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Mux SIMULATION 1 - luxI, luxR, lasI and lasR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Comments: GFP(las) in low, but not null, because of leakage factor. Posing &quot;a&quot;=0, GFP(las) becomes null. Mux output is the sum of GFP(lux) and GFP(las). If we sum these values at steady state, we have 870.2330 (arbitrary units).
</LI></UL><P><B>SIMULATION 2</B></P><UL><LI>Tetracycline = 10
</LI><LI>IPTG = 0
</LI><LI>Arabinose = 10
</LI><LI>IPTG channel (CH1) is selected, but IPTG is not present, so we expect to find low steady state GFP values for both lux and las systems.
</LI></UL><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Mux SIMULATION 2 - tetR, lacI, cI and GFPs</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Mux SIMULATION 2 - luxI, luxR, lasI and lasR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Comments: total GFP is low after a transient time. If we sum GFP(lux) and GFP(las) values, we have 14.2292 (arbitrary units), that is much lower than the output total value in SIMULATION 1, confirming that the chosen parameter set gives low leakage.
</LI></UL><H3><SPAN class="mw-headline" id="Demux_2"><B>Demux</B></SPAN></H3><P><B>SIMULATION 1</B></P><UL><LI>Tetracycline = 0
</LI><LI>IPTG = 0
</LI><LI>IPTG signal (INPUT) is absent, so we expect to find a low RFP and GFP values.
</LI></UL><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Demux SIMULATION 1 - tetR, lacI, cI, GFP and RFP</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Demux SIMULATION 1 - luxI, luxR, lasI and lasR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Comments: RFP and GFP values are not null, because of leakage. The numeric steady state values are 0.869 for GFP and 2.8753 for RFP, in arbitrary units.
</LI></UL><P><B>SIMULATION 2</B></P><UL><LI>Tetracycline = 0
</LI><LI>IPTG = 10
</LI><LI>IPTG signal (INPUT) is present and Tetracycline signal is not. So, INPUT is transferred into RFP channel (OUT1).
</LI></UL><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Demux SIMULATION 2 - tetR, lacI, cI, GFP and RFP</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE cellpadding="12px" align="center"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Demux SIMULATION 2 - luxI, luxR, lasI and lasR</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>Comments: GFP steady state value (0.8709 arbitrary units) is comparable to GFP value in SIMULATION 1. This value is much lower that RFP steady state value (362.11 arbitrary units), confirming that the selected output channel has a significantly higher output value.
</LI></UL><P><B>CONCLUSIONS</B>
The reported simulations show the qualitative behavior of our two systems, predicted by our mathematical model in response to specific inputs. Results are compatible with systems' expected trends. Amplitude and time scales have no physical meaning because we considered dimensionless equations, but our goal was to study relative values at steady state, in order to check on simulated curves whether system on/off values could be distinguished or not.

Moreover, the mathematical model described so far is a good scaffold for future parameter identification, that will have to be performed in order to characterize our physical devices. To reach this goal, ad-hoc experiments will have to be performed on both final and intermediate devices to collect as many data as possible and an optimization algorithm will have to be used to compute parameter estimations, given the model and the acquired data.

Finally, sensitivity analysis can be performed to highlight the parameters that play the most critical role and to study the impact of their variability on systems behavior.
</P><H2><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H2><P>[1] <I>Modelli multiscala per studi di proteomica e genomica funzionale</I>, S. Cavalcanti, S. Furini, E. Giordano. &quot;Genomica e proteomica computazionale&quot;, GNB Book, Publisher: Pàtron, 2007
</P><P>[2] ETH Zurich iGEM 2007 Wiki - Modeling basics page
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UNIPV-Pavia/Modeling">http://2008.igem.org/Team:UNIPV-Pavia/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Modeling&amp;oldid=72912" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>