<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_Experiments_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Modeling/Experiments Model</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer"><H1>Experiments Meet Modeling</H1><P> Thanks to the excellent cooperation between our wetlab biologists and our modellers, we successfully characterized our lactate promoter library, as well as the effect of the AHL-degrading enzyme AiiA. </P><OL><DIV class="highlightBox"><H3> Goals</H3><OL><LI>Investigate the biological mechanism of LldR, after finding that literature on the system is incomplete. During the course of this project, we first made a model based on an <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module#How_did_we_derive_the_model_">assumption on its functioning</A>. We evaluated it against the results, and found that the model could not explain the data. We then studied the results and established an <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module#A_closer_look_at_the_mechanism_of_lldR">improved model</A>. </LI><LI><A href="#Characterization_of_the_effect_of_AiiA">Characterize</A> the influence of our AHL-degrading enzyme AiiA construct.  </LI></OL></DIV><DIV class="expContainer"><H2> Understanding the biological mechanism of LldR</H2><H3><SPAN style="color: #E90092">Nerd:</SPAN>
To model our system we need to have parameters from the lldR system? </H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Let's have a closer look at the mechanism of LldR! </H3></DIV><H3>After a few days in the lab ...</H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Here are your parameters: </H3><H4> Experiment 1): Characterization of lldR sytem in LB</H4><P><U>Equations</U></P><P> Because, we consider that only diffusion is happening, then we have </P>
\begin{align*}
\frac{d[GFP]}{dt}&amp;=\frac{a_\mathrm{GFP} \cdot \left(\frac{[Lact_{in}]}{K_\mathrm{A,Lact}}\right)^{n_1}}{1+\left(\frac{[Lact_{in}]}{K_\mathrm{A,Lact}}\right)^{n_1}}-d_\mathrm{GFP}[GFP]\\
\end{align*}
<P id="Parameter_fitting"><U>Parameter Fitting</U></P><P> We fitted our model using the Least Absolute Residual method, using the fitting toolbox of Matlab. 
We <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection#Design_of_synthetic_lldR_promoters">designed different constructs</A> of the LldR responsive promoters.We were thus able to compare all the promoters thanks to their ON/OFF ratio and K<SUB>M</SUB> values. For a more detailed description of the experiment and the characterization, go to the registry, by clicking on the different links provided below. </P><TABLE><TBODY><TR><TH>Promoter </TH><TH><A href="http://parts.igem.org/Part:BBa_J23114">Promoter Strength</A></TH><TH>ON/OFF ratio</TH><TH>K<SUB>M</SUB> (μM)</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K822000">K822000</A></TD><TD>unknown (natural)</TD><TD> 10.35 </TD><TD>955    </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847008">K1847008</A></TD><TD>162</TD><TD> 15.26 </TD><TD>  1075</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847009">K1847009</A></TD><TD>  1429</TD><TD>1.56    </TD><TD>977.5    </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847007">K1847007</A></TD><TD>2547</TD><TD>1.34  </TD><TD> 697.7 </TD></TR></TBODY></TABLE><P> We can observe that our construct <A href="http://parts.igem.org/Part:BBa_K1847008">K1847008</A> has the best ON/OFF compared to the other synthetic promoters which are very leaky. Also, during all these experiments, the levels of LldR inside the cell were kept constant. The two binding sites of LldR were also conserved (same distance to the promoter and same sequences). It is therefore nice to see that the K<SUB>M</SUB> values do not vary a lot depending on the construct. </P><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Have you seen lldR does not only act as an repressor it also activates gene expression?! </H3><H3><DIV class="imgBox" style="float:right;width:40%;margin: 10px 0px 10px 20px !important;"><P><B> Figure 1:</B> Biophysical model of the Lldr operon</P></DIV><P>After looking at <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Bacterial_sensor">puzzling results</A>, we realized that our <A href="#How_did_we_derive_the_model_">first model </A>of the mechanism of action of LldR was not realistic. Searching literature, we finally found a compatible explanation, depicted in Figure 1. In the paper by <A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">Aguilera et al. (2008)</A>, they suggest that LldR may be required for the transcription machinery. Hence, instead of having only repression by LldR, LldR might play a <B>dual role as a repressor and an activator</B>.  It suggests that when lactate is present, it destabilizes the DNA loop and induces a conformational change of LldR.This results in the transcription of the gene of interest (goi).  This mechanism is consistent with our results. In the following, we will describe the mathematical equations corresponding to this mechanism.</P></H3><H3><SPAN style="color: #E90092">Nerd:</SPAN>
Therefore we have to update our model, let's include a activation term?</H3><H4> Reactions</H4><TABLE><TBODY><TR><TD style="width:60%">
\begin{align} \label{eq:1}
\varnothing&amp;\mathop{\xrightarrow{\hspace{4em}}}^{a_{\mathrm{LldR}}} \text{LldR}\\
2 \cdot \text{LldR} &amp;\mathop{\mathop{\xrightarrow{\hspace{4em}}}^{\xleftarrow{\hspace{4em}}}}^{K_{\mathrm{d,1}}} \text{LldR}_2\\
2 \cdot \text{Lact}+\text{LldR}_2 &amp;\mathop{\mathop{\xrightarrow{\hspace{4em}}}^{\xleftarrow{\hspace{4em}}}}^{K_{\mathrm{d,2}}}2 \cdot  \text{LactLldR} \hspace{2em}\\
\text{LldR} &amp;\mathop{\xrightarrow{\hspace{4em}}}^{d_{\mathrm{lldR}}} \varnothing\\
\text{LldR}_2 &amp;\mathop{\xrightarrow{\hspace{4em}}}^{d_{\mathrm{lldR_2}}} \varnothing\\
\text{LactLldR} &amp;\mathop{\xrightarrow{\hspace{4em}}}^{d_{\mathrm{LactLldR}}} \varnothing\\
\end{align}
</TD><TD><OL><LI>Production of LldR.</LI><LI>Dimerization of LldR in solution.</LI><LI>Unbinding of the dimer of LldR with lactate.</LI><LI>Degradation of all the species.</LI></OL></TD></TR></TBODY></TABLE><H4> Mathematical model</H4><P> According to the previous description, the gene of interest is activated by LactLldR and repressed by LldR dimer. Hence, if <I>gfp</I> is the gene of interest, we get the following ODE: </P>
\begin{align*}
\frac{d[GFP]}{dt}&amp;=\frac{a_{GFP}}{1+\left(\frac{[LldR_2]}{K_R}\right)^{n_r}}\cdot \frac{\left(\frac{[LactLldR]}{K_A}\right)^{n_a}}{1+\left(\frac{[LactLldR]}{K_A}\right)^{n_a}}-d_\mathrm{GFP}[GFP]
\end{align*}
<P> Assuming mass conservation for LldR, the total amount of LldR is given by: </P>
\begin{align*}
\text{LldR}_{tot}&amp;=[LldR]+[LactLldR]+2 \cdot [LldR_2]
\end{align*}
<H4> Simplification</H4><P>This equation is unidentifiable and we simplified the system by approximating the transcription of <I>gfp</I> by an Hill activation function: </P>
\begin{align*}
\frac{d[GFP]}{dt}&amp;=\frac{a_\mathrm{GFP} \cdot \left(\frac{[Lact]}{K_\mathrm{A,Lact}}\right)^{n}}{1+\left(\frac{[Lact]}{K_\mathrm{A,Lact}}\right)^{n}}-d_\mathrm{GFP}[GFP]\\
\end{align*}

<H3><SPAN style="color: #E90092">Nerd:</SPAN>
We have a problem, the model predicts that the sensitivity of the lldR system is to low. Can we change sensitivity?</H3><H3>After digging into literature...</H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Maybe there is a solution, we could add a lactate importer to increase internal lactate concentration...</H3><H3>Characterization of the lactate importer in LB</H3><DIV class="imgBox" style="float:right;width:40%;margin: 10px 0px 10px 20px !important;"><P><B> Figure 2:</B> Experiments for the characterization of the LldR promoters</P></DIV><P> To the extent of our knowledge, no characterization of  the lldPRD operon is available in literature, nor in the iGEM registry. We characterized our synthetic promoters and the natural promoter in two lactate titration experiments: one with overexpressed LldP, and one with the natural expression of LldP. For a further description of this experiment, <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Characterization_of_the_natural_LldR_promoter">click here</A>. 
However, we encountered several problems. In this setup, the lldPRD operon is not knocked out in our <I> E. coli </I> strains. Thus, we don't know how much LldP is expressed by the  <I> E. coli </I>. We only know that the natural promoter is weak. Therefore, we assumed that when LldP is not overexpressed, there is no active transport of lactate by the permease. As the lactate molecule is small, we consider lactate passes through the <I> E. coli</I> only thanks to diffusion mechanisms, and then: 
\([Lact_{in}]=[Lact_{out}]\). When LldP is overexpressed, we consider that the lactate concentration is <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion#Diffusion_and_transport_of_chemical_species">higher inside the <I> E. coli </I></A> than in the extracellular space.  </P><P> To see the details about the characterization, <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Resultsa">click here</A>.</P><H4> Experiment 2): Overexpressed LldP in LB</H4><P> We then designed other constructs including the symporter LldP. We expected an increase in lactate import such that the <I>E. coli </I> cells became more sensitive. However, the designed promoters show completely different LldR levels compared to the previous experiments. That is why we can not extract the parameters for LldP symporter using both experiments, because the data sets are not comparable. We will therefore use the same fitting function as before. Below, the promoter levels computed with a <A href="https://salislab.net/software/doReverseRBS">RBS calculator</A> and the <A href="http://parts.igem.org/Part:BBa_J23114">registry</A> are indicated.  </P><TABLE><TBODY><TR><TH> Construct </TH><TH> Expression of LldR(A. U.)</TH><TH> Expression of LldP(A. U.)</TH></TR><TR><TD> LldR </TD><TD> 51100</TD></TR><TR><TD> High LldP- lldR</TD><TD> 664</TD><TD> 8400</TD></TR><TR><TD> Low LldP- lldR</TD><TD> 12</TD><TD> 23.4</TD></TR></TBODY></TABLE><P><U>Parameter Fitting</U></P><P> We then fitted our model as explained before,and we obtained the following values for ON/OFF ratio and K<SUB>M</SUB> values. </P><DIV style="width:100%"><TABLE style="float:left;width:50%"><CAPTION><P><B><U> Low LldP-LldR</U></B></P></CAPTION><TBODY><TR><TH>Promoter </TH><TH><A href="http://parts.igem.org/Part:BBa_J23114">Promoter Strength</A></TH><TH>ON/OFF ratio</TH><TH>K<SUB>M</SUB> (μM)</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K822000">K822000</A></TD><TD>unknown (natural)</TD><TD> 1.16</TD><TD> 720.2 </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847008">K1847008</A></TD><TD>162</TD><TD>1.42  </TD><TD> 337.7 </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847009">K1847009</A></TD><TD>  1429</TD><TD>0.96   </TD><TD> 459.8    </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847007">K1847007</A></TD><TD>2547</TD><TD> 1.29 </TD><TD>1337  </TD></TR></TBODY></TABLE><TABLE style="float:right;width:50%"><CAPTION><P><B><U> High LldP-LldR</U></B></P></CAPTION><TBODY><TR><TH>Promoter </TH><TH><A href="http://parts.igem.org/Part:BBa_J23114">Promoter Strength</A></TH><TH>ON/OFF ratio</TH><TH>K<SUB>M</SUB> (μM)</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K822000">K822000</A></TD><TD>unknown (natural)</TD><TD>8.04 </TD><TD>    1930</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847008">K1847008</A></TD><TD>162</TD><TD> 23.96 </TD><TD> 1751   </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847009">K1847009</A></TD><TD>  1429</TD><TD>24.34   </TD><TD> 2361  </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1847007">K1847007</A></TD><TD>2547</TD><TD> 3.85 </TD><TD>1977    </TD></TR></TBODY></TABLE></DIV><H4> Observations </H4><P> It is difficult to make a correct explanation here, since, both the levels of LldP and lldR change. However, in the first construct, we can clearly see that the leakiness is increased for small amounts of LldP/LldR. Consistent with our model, it is probably due to the insufficient levels of LldR. Since LldR is thought to repress the transcription, this could explain the leakiness. </P><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
The lldP helps to have a more sensitive system but it is not enough.</H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Maybe it is just the LB interfering with our system, let's try M9 media or better try our mammalian cell media!</H3></DIV><H3>Characterization of the lactate responsive promoter in mammalian medium</H3><P> Below is displayed the response of one synthetic promoter in <A href="https://2015.igem.org/Team:ETH_Zurich/Materials">RPMI</A>- serum free medium. The promoter harboring the permease  and LldR displays a much higher sensitivity than the one only harboring LldR. In the following, these parameters are going to be used to simulate the fold-change sensor. </P><DIV class="imgBox"><TABLE><TBODY><TR><TH>Parameter </TH><TH> value</TH></TR><TR><TD> K<SUB>M</SUB></TD><TD>173 μM </TD></TR><TR><TD>ON/OFF </TD><TD>7 </TD></TR><TR><TD> n</TD><TD>1.7 </TD></TR></TBODY></TABLE><P><B>Figure 3:</B> Characterization of the low engineered promoter</P></DIV><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
Cool this works much better!</H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Yeah, we also cocultured <I>E.coli</I> with cancer cells and they responded to the lactate produced by the cancer cells!</H3></DIV><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
We can detect CTCs by their lactate production!</H3></DIV><DIV class="expContainer"><H2> Geometry of the chip and design of the microfluidic chip </H2><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
But how should we set up our final system? It has to be compact and easy to handle, but still needs to allow single cell analysis...</H3><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
I have an idea for that!</H3></DIV><P><B>Single-cell analysis</B> is a pre-requisite for our cancer detection system. Circulating tumor cells are really <B>scarce</B> in the blood compared to healthy cells. Implementing a microfluidic <A href="https://2015.igem.org/Team:ETH_Zurich/Chip">chip</A> was then the ideal solution because it enables <B> high-throughput single-cell analysis</B>. However, having small reaction volumes was very challenging. Indeed, the natural quorum sensing system is meant to <B>work on large length scales</B>, as proven by the<A href="https://2014.igem.org/Team:ETH_Zurich"> last year’s ETHZ IGEM team</A>.  The signalling molecule AHL diffuses then almost instantaneously in the chip.  Biologically, we thought first that including AiiA degradation in the design or tightening the leakiness of the P<SUB>Lux </SUB> promoter would solve the problem. It turned out that implementing a fold-change sensor solved the problem, since a pulse of LuxR will limit over-activation of the system. </P><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
Very elegant, congratulations!</H3></DIV><DIV class="expContainer"><H2>Characterization of the effect of AiiA</H2><DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
So in the end, is AHL degradation beneficial for our system or not?</H3></DIV><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
Hm, we need more date to estimate that...</H3><P>To characterize the effect of AiiA on our system, we ran two experiments. In one of them, AiiA was constituvely produced and in the other one, AiiA was not present. Below we compare both responses. The first visible thing is that AiiA drastically shifts the curve until lower sensitivities. As AiiA degrades AHL, this phenomenon is expected. In addition, since AiiA is on a medium copy plasmid and under the control of a strong constitutive promoter, the large <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Quorum_sensing_module__influence_of_AHL_degradation">effect</A> seen in our experiments is expected. </P><H3><SPAN style="color: #E90092">Nerd:</SPAN> 
Ok, that should help us decide.</H3><P> From the parameters, we know AiiA acts with Michaelis Menten Kinetics. The K<SUB>M </SUB> value and turnover number are already known from literature,<A href=""> see parameters</A>. However, for our model, we need to know how much AiiA is produced by the cell in order to characterize the behaviour of the system. </P><H3> Dose response curves and apparent K<SUB> M </SUB> values</H3><P> Here we compared the fitted curves of AiiA and the K<M> values for both situations. AiiA drastically shifts the curve towards less sensitivities. We calculated the apparent half substrate maximum concentration for both cases. </M></P><DIV class="imgBox"><TABLE><TBODY><TR><TH>
K<SUB>M</SUB></TH><TD>
9.983 nM
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>
K<SUB>M</SUB></TH><TD>
10<SUP>6</SUP> nM
</TD></TR></TBODY></TABLE><P><B> Figure 4:</B> Experimental and fitted curves for the plasmid: A) Without AiiA, B) With expressed AiiA. </P></DIV><H3>Concentration of AiiA inside the <I> E. coli </I></H3><P> Below we fitted two ordinary differential equations thanks to MEIGO toolbox. The equations are below.  </P><H4>Equations</H4><P> Here, AHL is the input. There is no amplification by LuxI. </P>
\begin{align*}
\frac{d[AHL]}{dt}&amp;=- \frac{v_{AiiA} \cdot [AHL]}{K_{M,AiiA}+[AHL]} \\
\frac{d[GFP]}{dt}&amp;=\frac{a_\mathrm{GFP} \cdot \left(\frac{[AHL]}{K_\mathrm{Lux}}\right)^{n_1}}{1+\left(\frac{[AHL]}{K_\mathrm{Lux}}\right)^{n_1}}-d_\mathrm{GFP}[GFP]\\
\end{align*}

<H4>Parameter fitting</H4><P> Due to the practical identifiability problem, we could not estimate a value for K<M>. We then re-used the value from the literature, in the millimolar range, which is consistent with the rest of the equations. </M></P><P> We find: </P>
\begin{align*}
v_{AiiA}= 9.46 \cdot 10^4 nM min^{-1} \\
\end{align*}

<DIV align="right"><H3><SPAN style="color: #006EB4">Lab rat: </SPAN>
Ok, but just to be clear now...</H3></DIV><H3>Can AiiA be used in our system?</H3><P> As already mentioned, AiiA shifts the active curve until lower sensitivities. This means that higher concentrations of AHL will be required to activate the construct. In our case, the leakiness is necessary to initiate the signal. The amount of AiiA present here, will prevent any form of activation. We will therefore, <B>not use AiiA in our final design</B>.

</P></DIV></DIV></DIV><DIV id="footerContainer"><H2>We would like to thank our sponsors</H2></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>