<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Sandbox skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Sandbox</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer"><H1>Sandbox page</H1></DIV><DIV class="expContainer"><H3>Establishing a system for display of Annexin V on <I>E. coli</I> outer membran</H3><H4>Overview</H4><P>We conceptualize our Microbeacon system to rely on bacteria binding to cancer cells as a detection signal. The solution that we propose is to use Annexin V expressed on the outer membrane to bind cancer cells. Annexin V binds to phosphatidylserine (<A href="https://2015.igem.org/Team:ETH_Zurich/References#Leventis2010">Leventis, 2010</A>), which is a membrane lipid usually found in the inner part of the cell, but in cells undergoing apoptosis it is exposed to the outer membrane due to the activity of flipases (<A href="https://2015.igem.org/Team:ETH_Zurich/References#Leventis2010">Leventis, 2010</A>). </P><P>Annexin V is a soluble protein which has an unknown role in mammalian cell except its binding properties to membrane components(<A href="https://2015.igem.org/Team:ETH_Zurich/References#Gerke2002">Gerke <I>et al</I>, 2002</A>). As we want to express Annexin V in the outer membrane we need a protein that will export it. For this purpose we chose INP and changed the YFP, present in the part (<A href="”http://parts.igem.org/Part:BBa_K523013”">BBa_K523013</A>), for a human Annexin V protein optimized for <I>Escherichia coli</I> codon usage (ThermoFisher Scientific GeneOptimizer(R)). We used a strong RBS (<A href="”http://parts.igem.org/Part:BBa_B0034”">BBa_B0034</A>), a low expression promoter (<A href="”http://parts.igem.org/Part:BBa_J23114”">BBa_J23114</A>) and a low copy plasmid with a pSC101 origin of replication to avoid an excessive stress for the cell, as it is a membrane protein.</P><H4>Results</H4><P>To test our construct we used:</P><UL><LI>J23114-B0032-INP-Annexin V in pSEVA371 (low copy plasmid)expressing RFP in TOP10 cells</LI><LI>J23114 expressing RFP in TOP10 cells</LI></UL><P>As a first test, we did an assay with beads. We used <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Dynabeads_for_verification_of_Annexin_externalization">magnetic beads</A> coated with Protein A, which we incubated with monoclonal mouse anti-Annexin V antibodies (Affimetrix) for 30 min and then with the bacteria with different constructs. Unfortunately, we got no results from them.  We also did a Western blot to analize the expression of our construct using our anti-Annexin V antibody (Affimetrix) and we observed no band at the height where Annexin V is expected (35.9 kDa). </P><P>We wondered what could have gone wrong, so we checked with an RBS calculator (<A href="https://salislab.net/software/">Salis <I>et al</I>, 2015</A>) whether the expression would be high enough with our design. There, we realized that Annexin V showed a very low calculated translation initiation rate, which might indicate low or no expression. Using B0032 with INP-Annexin V we were getting a translation initiation rate of 410 a.u., while if we changed the RBS to B0034, we would get 12 000 a.u. Therefore, we changed the RBS of INP-Annexin V and Annexin V to B0034. As can be seen in Figure 5 we can see a band at the expected 35.9 kDa. This confirms the expression of Annexin V in our cells. Even though the band is weak, which we expected for low expressed membrane proteins, it indicates that INP-Annexin V is also expressed, which can be seen with a band at the corresponding 70.4 kDa. To confirm that INP-Annexin V was on the membrane fraction we run both the unclarified cell free extract (called crude from now on) of each sample and the supernatant of the centrifuged sample. Annexin V is present in both fractions, while INP-Annexin V is only visible in the crude fraction (Figure 5), likely to be part of the membrane containing cell debris.</P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 5.</B> Western blot image of B0034-Annexin V and B0034-INP-Annexin. Samples of Annexin V and INP-Annexin V were loaded both with a low copy plasmid (pSEVA371) and a high copy plasmid (pSB1C3). Each time a sample of the crude and of the supernatant were loaded on the gel. Detection was performed with a monoclonal mouse anti-Annexin antibody with a secondary goat anti-mouse antibody labelled with Alexa 488. </P></DIV><P>For the visualization of the Western Blot, we were forced to increase the sensitivity of the film a lot, which might indicate that the concentrations of the protein inside the cell are very low or that our primary antibody is not functioning too well.</P><H3>INP-Annexin V externalization</H3><H4>Overview</H4><P>After proving expression of our INP-Annexin V construct it is also crucial to test for it's functionality. We did so by using Protein G coated Dynabeads loaded with an anti-Annexin V antibody. The <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Dynabeads_for_verification_of_Annexin_externalization">detailed protocol</A> can be found in the experiments section.</P><P>We prepared two possible datasets:</P><UL><LI>Beads without loaded antibody. Five different conditions were compared: </LI><UL><LI>INP-Annexin V-RFP in a high copy plasmid</LI><LI>INP-Annexin V-RFP in a low copy plasmid</LI><LI>RFP in a high copy plasmid</LI><LI>Annexin-Alexa 488 purified</LI><LI>PBS</LI></UL><LI>Beads with loaded antibody. </LI><UL><LI>INP-Annexin V-RFP in a high copy plasmid</LI><LI>INP-Annexin V-RFP in a low copy plasmid</LI><LI>RFP in a high copy plasmid</LI><LI>Annexin-Alexa 488 purified</LI><LI>PBS</LI></UL></UL><P>After incubation of the antibody coated beads with the respective bacterial strain or control, we observed our beads using the microscope.</P><H4>Results</H4><P>First thing we did was to check the positive (Alexa-488 and beads with antibody and Alexa-488) and negative controls (beads without antibody). We found out that there was no green fluorescence in the positive control, <A href="#INP-Annexin_V_expression">confirming</A> again our fear that the <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Anti-Human-AnnexinV">anti-Annexin antibody</A> we are using is not binding as expected. Neither of our samples presented fluorescence from the secondary antibody either. We therefore concluded that the antibody was not very effective to detect annexin bound to another molecule. We came back to our Western blot (see Figure <B class="TODO">x</B>), where we used the same antibody, and saw that there were some unspecific bands at about 55 kDa. Also, the expected bands were very weak.</P><P>As can be seen in Figure <B class="TODO">x</B>, no difference can be seen in fluorescence levels of the beads. Moreover, bacteria have a similar size to bacteria (2.8 µm and 2 µm, respectively), so they would be seen in the microscopy picture if they were there. </P><DIV class="imgBox" style="width:50%;margin: 20px auto 20px auto !important;"><P><B> Figure x.</B> Experiment for binding beads to anti-Annexin V antibody with four different conditions: INP-Annexin V-RFP in high copy plasmid, INP-ANnexin V-RFP in low copy plasmid, RFP in high copy plasmid and a negative control (no Annexin V).</P></DIV><H3>INP-Annexin V functionality</H3><H4>Overview</H4><P>A properly expressed and exported Annexin V protein is the first step towards a bacterial apoptosis detection system. However, the validation of the functionality of Annexin V in this construct is also crucial. We therefore tried several approaches to prove functionality of this construct regarding binding to apoptotic cells. For easy monitoring of our bacteria, we introduced an RFP construct into the same plasmid carrying INP-Annexin V. We published this construct in the registry to allow future iGEM teams to use our <A href="parts.igem.org/Part:BBa_K1847015">bacterial apoptosis detector (BBa_K1847015)</A> for their project.</P><H4>Results and Discussion</H4><P>The first atempt we took were <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Phosphatidylserine-coated_beads_for_verification_of_Annexin_functionality">beads coated in phosphatidyl serine</A>, which would simulate the surface of an apoptotic mammalian cell. Unfortunately, we were not able to detect binding of Annexin-Alexa 488 or bacteria to these beads.</P><P>However, these results cannot be taken as a total disproval of our system, since it is not clear how well the phosphatidylserine (PS)-coated beads simulate an apoptotic cell. We came up with the following possible explanations as to why this setup might not be ideal:</P><UL><LI>It is possible that too high concentrations of PS (85 pM) on the beads interferes with the Annexin V-PS interaction.</LI><LI>The <A href="http://lipidlibrary.aocs.org/Primer/content.cfm?ItemNumber=39360">hydrophobic tails of PS</A> might form clumps if they are not embedded into a membrane, thereby hindering interactions.</LI><LI>The <A href="https://echelon-inc.com/index.php?module=Products&amp;func=detail&amp;id=760">biotin moiety</A> required to bind PS onto the beads might be positionned on the lipid such that the interaction with Annexin V is no longer possible</LI></UL><P>But this initial backset could not stop us! We decided to try a more realistic setup, testing the binding of our <I>E. coli</I> cells expressing the INP-Annexin V construct to actual cancer cells. The results of this experiment can be found <A href="#Detection_of_apoptotic_cells_by_our_engineered_bacteria">here</A>.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>