<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Glossary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Glossary</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer"><H1>Glossary</H1><H2>AND-Gate</H2><DIV class="imgBox" style="float:right;width:30%;margin: 0px 0px 20px 20px !important;"><P><B>Figure 1.</B> AND-gate logic.</P></DIV><P>The AND gate is a basic digital logic gate that implements logical conjunction - it behaves according to the truth table to the right. A high output results only if both the inputs to the AND gate are high. If neither or only one input to the AND gate is high, a low output results. </P><H2>Circulating Tumor Cell (CTC)</H2><P>CTCs are thought to originate from the primary tumor from which they escape. They travel in the blood stream and eventually colonize new tissues. Thus CTCs are believed to be the cause of metastasis. For this reason, they can potentially be used as an early marker for metastatic cancer [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Caceres2015">CÃ¡ceres, 2015</A>]. CTCs can be detected in the blood before the formation of metastasis, therefore allowing early intervention. However, CTCs are very scarce and detection methods have to be accordingly sensitive.</P><H2>Epithelial-mesenchymal transition</H2><P>The Epithelial-mesenchymal transition (EMT) takes place when epithelial cells detach from the walls of blood or lymphatic vessels and enter the circulatory system. These cells develop into multipotent stromal cells during normal embryonal development. However, a similar process occurs when cancerous cells invade the circulatory system to spread all over the body [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Kalluri2009">Kalluri, 2009</A>]. </P><H2>Fold-change sensor</H2><DIV class="imgBox"><P><B>Figure 2.</B> Concept of fold-change sensor. Scheme from [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Goentoro2009" style="display:inline;">Goentoro 2009</A>]</P></DIV><P> A fold-change sensor generates a response to fold-changes in inputs levels rather than a response to absolute values. The system is based on an incoherent feed-forward loop. An incoherent feed-forward loop is a system in which the intermediate pathways have opposite functional roles (i.e. at least one activation and one inhibition), leading to the adaptation of the output. The <B> delay </B> introduced by the additional step in one of the pathways generates a pulsed response. In the image below you can view the different parameters that influence this response. </P><DIV class="imgBox"><P><B>Figure 3.</B> Incoherent feed-forward loop and fold-change sensor. Scheme from [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Goentoro2009" style="display:inline;">Goentoro 2009</A>]</P></DIV><H2 id="LldPRD">LldPRD Operon</H2><P>LldPRD is an operon in <I>E. coli</I> that is involved in lactate metabolism [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">Aguilera, 2008</A>]. It consists of a promotor region followed by three genes:</P><DIV class="imgBox"><P><B>Figure 4.</B> The <I>E.coli</I> LldPRD operon [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008" style="display:inline;">Aguilera, 2008</A>]</P></DIV><H4>LldP</H4><P>L-lactate Permease, a membrane transporter which partcipates in the <B>active transport</B> of L-Lactate, D-lactate and glycolate by using a proton force [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Nunez2002">Nunez, 2002</A>].</P><H4>LldR</H4><P>L-lactate Regulator, a transcription factor that is responsive to L-lactate. In the absence of lactate it localizes to the operator regions O1/O2 and gene transcription is blocked. When lactate binds to LldR it detaches from the promotor region and gene expression is activated.</P><H4>LldD</H4><P>L-lactate Dehydrogenase, an enzyme that metabolizes L-lactate. Its expression is controlled by LldR acting on the upstream promoter. Replace this gene with any gene you want to upregulate in response to lactate. </P><H2>Logistic equation</H2><P>A logistic function is a smooth (i.e. infinitely differentiable) function to implement a step from one value to another. In population ecology, it models the size of a population starting at an initial size and growing until it reaches carrying capacity, forming a characteristic S-curve. It is defined by the equation
$$P(t) = \frac{n_0 K e^{Rt}}{K + n_0(e^{Rt}-1)}$$
where \(n_0\), \(K\), and \(R\) are the initial population size, carrying capacity (maximum population size), and growth rate, respectively.
</P><H2>Michaelis-Menten Kinetics</H2><P>This is a basic model to describe the rate of a (potentially reversible) enzymatic reaction like the following:</P><P>Enzyme <B>E</B> binds to Substrate <B>S</B> and converts it into product <B>P</B>. Each conversion happens at a given rate characterized by the rate constants <B>k</B>.
The rate at which product <B>P</B> is formed is given by:</P><P>The reaction rate <B>v</B> is the change in product concentration <B>[P]</B> over time. It increases with increasing substrate concentration <B>[S]</B>.</P><P>The maximum rate <B>V<SUB>max</SUB></B> is reached when every Enzyme <B>E</B> is in action and working on a Substrate molecule <B>S</B>. It is therefore proportional to the enzyme concentration <B>[E]<SUB>0</SUB></B> and determined by the turnover number <B>k<SUB>cat</SUB></B>, which describes the &quot;work enthusiasm&quot; of the enzyme and can vary greatly depending on the enzyme at hand. In mathematical terms this is represented by <B>V<SUB>max</SUB> = k<SUB>cat</SUB>[E]<SUB>0</SUB></B>.</P><P><B>K<SUB>M</SUB></B> (called Michaelis constant) is the substrate concentration at which the reaction rate has reached half the maximum rate <B>V<SUB>max</SUB></B>. It measures the affinity of an enzyme to the substrate. The lower <B>K<SUB>M</SUB></B> (the quicker the half-maximum rate is reached) the higher the affinity. This measure can vary greatly as well.</P><H2 id="phosphatidylserine">Phosphatidylserine</H2><P><B>Phosphatidylserine</B> is an important phospholipid membrane component (i.e. component of the cell membrane), which plays a key role in cell cycle signaling, specifically in relationship to apoptosis. Phosphatidylserine is normally exclusively located on the inner membrane. It is <B>flipped to the outer membrane by flipases when apoptosis is induced</B>. That is why phosphatidylserine is thought to be an early marker of apoptosis.</P><H2>Quorum sensing</H2><P>Quorum sensing is a system by which bacteria &quot;sense each other&quot;. This cell to cell communication is mediated by chemical signals such as <B>AHL</B> (N-acyl-L-homoserine lactone). This is a means to tune gene expression depending on the density of the population [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Miller2001">Miller, 2001</A>]. <B>LuxR</B> is the transcription factor that mediates the quorum sensing response. If <B>AHL</B> is present at sufficient concentrations it binds and activates <B>LuxR</B>, whereupon downstream gene expression is induced.
Gene expression might become <B>leaky</B> if too much <B>AHL</B> accumulates over time. We therefore introduced a constitutively expressed AHL-Lactonase (<B>AiiA</B>) to degrade <B>AHL</B> and keep the concentration low.</P><H2 id="TRAIL">TRAIL</H2><P> The Tumor Necrosis Factor Related Apoptosis Inducing Ligand, <B>TRAIL</B>, is an important protein of our immune system which induces apoptosis in cells expressing the required death receptors, DR4 and DR5, thereby inactivating the target cell. Binding of <B>TRAIL</B> to DR4 or DR5 on the surface of a target cell leads to <B>Caspase-8 dependent induction of apoptosis</B>. Cancer cells often express elevated levels of the DR4 and DR5 death receptors and are therefore especially susceptible to treatment with <B>TRAIL</B>, whereas healthy cells are generally not harmed by <B>TRAIL</B>. In nature, <B>TRAIL</B> exists as a membrane-bound and a soluble version. In our system we are using the soluble form, referred to as <B>sTRAIL</B> [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Johnstone2008">Johnstone, 2008</A>].</P><H2>Warburg Effect</H2><P>Malignant tumors often tune their metabolism towards highly elevated rates of glycolysis, followed by lactic acid fermentation, rather than pyruvate hydrolysis in mitochondria. This gives the tumor cells a comparative advantage over normal cells due to the rapid intake of glucose for which the cells are competing. This effect is referred to as the Warburg effect since it was first described by Otto Warburg. It is not clear what the ultimate cause of this effect is. The transition to lactic acid fermentation occurs also in environments with plenty of oxygen, and once it occurred, tumor cells keep this feature even during transition to metastasis [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Vander2009">Vander Heiden <I>et al</I>, 2009</A>].
</P><DIV class="imgBox" style="width:70%;margin: 20px auto 20px auto !important;"><P><B> Figure 5.</B> Warburg effect scheme. Figure adapted from [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Vander2009" style="display:inline;">Vander Heiden 2009</A>] </P></DIV></DIV></DIV><DIV id="footerContainer"><H2>We would like to thank our sponsors</H2></DIV></DIV></DIV></DIV></DIV></BODY></HTML>