<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Interlab_study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Interlab study</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer" style="position:relative"><H1>InterLab Study </H1><H2>Introduction</H2><P>Since 2014, teams participating in iGEM have been invited to collaborate in the InterLab study, with the objective of comparing the variability of measurements among different research groups.</P><H2>Goal</H2><P> The participating labs were tasked with measuring the fluorescence of three different devices consisting of three promoters (strong, medium and weak) followed by a GFP gene, with the aim of comparing the results among different iGEM teams all around the world. The recommended standard method for measuring is a plate reader setup. However, teams were invited to use other methods. In our case, we measured the devices using the plate reader and FACS. </P><H2>Methodology</H2><H3>Protocol</H3><P>For this experiment, we followed the <A href="https://2015.igem.org/Tracks/Measurement/InterLab_Protocol"> protocol available in the iGEM website</A>. </P><P>Organisms were prepared by BioBrick assembly using the parts indicated in Table 1. Once the parts were ligated, the plasmids containing devices 1-3 and the positive and negative controls were transformed into TOP10 <I>Escherichia coli</I>. The three devices, the positive control (I20270), and the first negative control (B0012) were cultured on LB plates with the addition of chloramphenicol, while TOP10 cells without the addition of any plasmid were cultured in LB-Streptomycin plates. The transformed cells were incubated in this manner overnight for 16 h at 37ÂºC. </P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Table 1.</B> Summary of used elements</P><TABLE><TBODY><TR><TH rowspan="3">Devices</TH><TD>Device 1</TD><TD>J23101 + I13504</TD></TR><TR><TD>Device 2</TD><TD> J23106 + I13504</TD></TR><TR><TD>Device 3</TD><TD>J23117 + I13504</TD></TR><TR><TH rowspan="3">Controls</TH><TD>Positive control</TD><TD>I20270</TD></TR><TR><TD rowspan="2">Negative control</TD><TD>no plasmid</TD></TR><TR><TD> B0012 </TD></TR></TBODY></TABLE></DIV><P>Ten colonies were picked from each plate. The presence of the correct device in the colonies was checked first by colony PCR and, if the size of the DNA fragment corresponded to the expected size, were then sent for sequencing. The colonies containing the correct plasmid were then incubated overnight for 16 h at 37ÂºC in 12 mL conical flasks in 5 mL LB medium with chloramphenicol or streptomycin, respectively. The flasks were shaken at 200 rpm at an angle. </P><H4>Protocol for the plate reader</H4><P>The overnight bacteria culture was diluted in Lysogenic Broth (LB) to OD600 of 0.2Â±0.04 in a 96-well plate. OD600 was monitored during the measurement for verification of the concentration. A unique measurement of the devicesâ fluorescence was performed at an excitation wavelength of 488 nm and emission was monitored at a wavelength of 530 nm 10 min after the dilution.</P><H4>Protocol for FACS</H4><P>Overnight cultures in LB were diluted 1:10 before the measurement. FACS was calibrated with PBS filtered with a 0.2 Âµm filter. Samples were diluted to have no more than 4,000 events/s prior to the measurement. TOP10 cells without the plasmid were used to establish the gating for GFP positive and negative cells, respectively. For the measurement of the three devices and all controls, 100,000 events were recorded in biological, as well as technical triplicates. </P><H3>Machines</H3><H4>Incubator</H4><P>The incubator used for growing the cells was the KuhnerShakerX, model ClimoShaker ISF1-X. The throw was orbital with 25mm of diameter. </P><H4>FACS</H4><P>The FACS machine used was a BD LSRFortessa Cell Analyzer. Cells were excited at a wavelength of 488nm and the filter used was a 505 LP. The SSC threshold was 400. Data is presented in absolute units.</P><P> The last calibration before our use was done on the 7th July 2015 by Ms JÃ¤ggin. </P><P>Data was analyzed using Diva, FlowJo 10.0.8 and Microsoft Excel 2013.</P><H4>Plate reader</H4><P>The plate reader we used is the Tecan Infinite M200 Pro. The excitation wavelength was set to 488 nm and the emission was at 530 nm. Fluorescence was normalized with the optical density, using fluorescence units/absorbance units. Optical density was determined at 600 nm. </P><P>The last calibration of the plate reader was done on the 17th March 2015 by the official Tecan Service (Tecan Sales Switzerland AG). </P><P>Data was analysed using Microsoft Excel 2013. </P><H2>Results</H2><H3>FACS measurement </H3><DIV class="imgBox" style="width:100%; margin: 20px auto 20px auto !important;"><P><B>Figure 1.</B> A) Comparison in the fluorescence of TOP10 cells without plasmid (black), with B0012 (light grey), with I20270 (dark grey), device 1 (orange), device 2 (blue) and device 3 (red). B) Mean values of fluorescence for each device. n=3 Â± SD</P></DIV><P>The median value was calculated for FACS as the distribution was skewed. We measured three different colonies (biological replicates) three times each (technical replicates). Data for each of them are summarized in Table 2. </P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Table 2.</B> Summary of the data obtained with FACS</P></DIV><H3>Plate reader</H3><DIV class="imgBox" style="width:80%;margin: 20px auto 20px auto !important;"><P><B> Table 3.</B> Comparison of the fluorescence of the controls and devices analysed by the plate reader.</P></DIV><P>As can be seen from Table 2 and Figure 2, the data correspond to the expected results: almost no fluorescence is detected in controls with TOP10 and TOP10 with B0012 plasmid, whereas cells transformed with I20270 present intermediate fluorescence. Device 1 has the strongest fluorescence, followed by device 2, and finally device 3 which is the weakest.</P><DIV class="imgBox" style="width:70%;margin: 20px auto 20px auto !important;"><P><B> Figure 2.</B> Fluorescence signal of different devices and controls. Values are mean n=3 Â± SD</P></DIV><H2>Other data</H2><H3>Who participated in the study?</H3><P>Lisa Baumgartner, Anna Fomitcheva, Anja Michel and Michael Meier prepared the transformants.</P><P>Verena JÃ¤ggin showed us how to properly manipulate the FACS machine and use the software.</P><P>Lisa Baumgartner and Anna Fomitcheva did the readings with FACS.</P><P>Lisa Baumgartner and Michael Meier did the plate reader experiment.</P><P>Data was processed by Lisa Baumgartner, Anna Fomitcheva and Michael Meier. </P><H3>Dates in which the study took place</H3><P>The measurements for the InterLab study took place on the 10th July 2015 for FACS and on 21st July 2015 for the plate reader. </P></DIV></DIV><DIV id="footerContainer"><H2>We would like to thank our sponsors</H2></DIV></DIV></DIV></DIV></DIV></BODY></HTML>