<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer" style="position:relative"><H1>Results</H1><P>We are excited to present a lot of interesting results, including the characterization of parts of the natural <I>E. coli</I><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#LldPRD">LldPRD operon</A> on which there is only a very limited amount of information present in the literature up to date, as well as a <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">library of promoters</A>, in which we present a <A href="http://parts.igem.org/Part:BBa_K1847008">better alternative</A> to the wild-type promoter. We also increased the sensitivity of the system by the addition of <A href="#Increased_sensitivity_for_single_cell_lactate_output">LldP</A>. In addition, we developed a <A href="http://parts.igem.org/Part:BBa_K1847010">hybrid promoter</A> responsive to LacI and LldR which shows an <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#AND-Gate">AND gate</A> behaviour.</P><P>We also characterized the behaviour of our system with <A href="http://parts.igem.org/Part:BBa_C0160">aiiA</A>; and explored the possibility of <A href="#Establishing_a_system_for_display_of_Annexin_V_on_E__coli_outer_membrane">Annexin V expression</A> on the outer membrane. Last but not least, we validated our lactate sensor in a <A href="https://2015.igem.org/Team:ETH_Zurich/Chip">nanoliter plate</A>.  </P><P>For a better understanding of the results of our project, we divided this part into three parts:</P><OL><LI><A href="#Bacterial_sensor">Bacterial sensor</A></LI><LI><A href="#_Experiments_involving_mammalian_cells_">Experiments involving mammalian cells</A></LI><LI><A href="#Mammalian_cells_and_bacteria_combined">Mammalian cells and bacteria combined</A></LI></OL><DIV class="highlightBox" style="margin-top: 40px"><H3>Goals</H3><P>Our main aims during this project were:</P><OL><LI>A highly sensitive lactate sensor
</LI><LI>the characterization of the <A href="#Characterization_of_the_natural_LldR_promoter">natural lldR-promoter</A> (<A href="http://parts.igem.org/Part:BBa_K822000">K822000</A>)</LI><LI>the characterization of our 8 newly designed <A href="#Characterization_of_synthetic_promoter_library">syntehtic promoters controlled by the regulator LldR</A></LI><LI>the characterization of 2 newly designed <A href="#Characterization_of_the_lacI-lldR_promoter">hybrid promoters</A>, combining repression by LldR and LacI</LI><LI>the characterization of the impact of the <A href="#Increased_sensitivity_for_single_cell_lactate_output">L-lactate permease LldP</A> on the sensitivity of our system.</LI><LI>the detection of <A href="#Increased_sensitivity_for_single_cell_lactate_output">CTC based on their elevated lactate output</A></LI><LI>Quorum sensing with low levels of leakiness
</LI><LI>the characterization of the influence of <A href="#Quorum_sensing_module__influence_of_AHL_degradation">AHL degradation by AiiA</A> on the sensitivity of the LuxR quorum sensing system</LI><LI>the design and validation of a tight <HTTPS:>AND gate with a clear binary behavior</HTTPS:></LI><LI>Apoptosis detection by <I>E. coli</I> binding to mammalian cells
</LI><LI>the induction of <A href="#Sensitivity_to_TRAIL">apoptosis in cancer cells</A> using sTRAIL</LI><LI>establishment of a system for <A href="#Establishing_a_system_for_display_of_Annexin_V_on_E__coli_outer_membrane">display of Annexin V</A> on the <I>E. coli</I> outer membrane</LI><LI>the investigation of <A href="#Mammalian_cells_and_bacteria_combined">binding of bacteria to apoptotic mammalian cells</A> via Annexin V and phosphatidylserine</LI><DIV class="expContainer"><H2>Bacterial sensor</H2><P>We performed a series of experiments to validate the expression of our constructs as well as the functionality of our system. The results are presented in the following sections.</P><H3 class="inMenu">Characterization of the natural LldR promoter</H3><H4>Overview</H4><DIV class="imgBox" style="float:right;width:40%;margin: 10px 0px 10px 20px !important;"><P><B> Figure 1.</B> The mechanism of repression of the lldR-promoter by LldR is thought to rely on DNA looping, which prevents the RNA Polymerase from transcription. The repression is removed in the presence of lactate.</P></DIV><P>For the sensing of Lactate inside our system we have a constitutive expression of the <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#LldPRD_Operon">L-lactate dehydrogenase regulator, LldR</A>. LldR is a protein which <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#LldPRD_Operon">binds to two operator sites</A>, operator1 (O1) and operator2 (O2), that flank the natural LldR promoter. This interaction forms a DNA loop, which does not allow the polymerase to transcribe the respective gene. Once Lactate is added to this system, it sequesters LldR, which can then no longer repress the promoter. Thus the gene of interest can be expressed <A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">[Aguilera 2008]</A>.</P><P>An improvement to our system, which was shown by the modeling to be important for enhanced sensitivity to low levels of lactate, is the addition of <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#LldPRD_Operon">the L-lactate permease, LldP</A>, a protein which is known to insert into the membrane and increase the import rate of lactate into the cell [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">Aguilera <I>et al</I>, 2008</A>].</P><P>We characterized the natural version of the lldR-promoter regarding its response to a large concentration range of L-lactate in a <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Characterization_of_the_lldR-promoter's-response_to_lactate">microtiter plate reader setup</A>.</P><H4>Results</H4><P>Surprisingly, what we found was not the expected release from the repression of LldR in the presence of Lactate. Indeed, if no LldR is present in the cell, according to the previous model, the levels of fluorescence should be equivalent as when there is high Lactate and LldR. We can clearly observe additional activation of GFP production! Especially for <A href="http://parts.igem.org/Part:BBa_K1847007">BBa_K1847007</A> and <A href="http://parts.igem.org/Part:BBaK1847009">BBa_K1847009</A> the levels of normalized fluorescence upon introduction of lactate into the system containing LldR rises clearly above levels observed without LldR<SUP><A href="#footnote1">1</A></SUP>. This interesting finding leads us to reinterpret the present model. Instead of having only repression by LldR, it seems that LldR is required for the transcription machinery. LldR would play <B> a dual role as an activator and as a repressor</B>, as discussed previously by <A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">[Aguilera 2008]</A> without experimental support. That made us think that we need to <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model#_Understanding_the_biological_mechanism_of_LldR">adapt our model</A> to accomodate this interesting and previously unexplored finding.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:562px;"><DIV class="thumbcaption"><B>Figure 2.</B> The wild type version of the LldR-promoter shows that LldR mediates repression, as well as activation of the gene controlled by the promoter</DIV></DIV></DIV></DIV><P>As mentioned above, we then introduced the lactate symporter lldP into our system. Expression of LldP and LldR are present in the natural operon structure, the result of which may be rather low levels of LldR in constructs also containing LldP, when compared to the experiment shown above in which there is no lldP (see the table below).</P><DIV class="imgBox" style="float:right;width:38%;margin: 10px 0px 10px 20px !important;"><P><B>Figure 3.</B> In the presence of L-Lactate, LldR mediates the activation of the gene of interest controlled by the lldR-promoter. The exact mechanism of this process has not been characterized in the literature.</P></DIV><P><B>Table 1:</B> Estimation of the translation initiation rates (TIR) of LldR and LldP in our different test constructs. Levels were calculated using the <A href="https://salislab.net/software/">Salis Lab Ribosomoe Binding Site Calculator</A> in combination with the comparative promoter strength provided with the <A href="http://parts.igem.org/Part:BBa_J23110">Anderson promoter collection.</A> For simplification, when the promoter is <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23114</A> we will refer to it as &quot;low&quot;, if it is <A href="http://parts.igem.org/Part:BBa_J23118">BBa_J23118</A> it will be referred to as &quot;medium&quot;, and if it is <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>, it will be &quot;high&quot;. </P><TABLE><TBODY><TR><TH> Construct </TH><TH> TIR of LldR(A. U.)</TH><TH> TIR of LldP(A. U.)</TH></TR><TR><TD> LldR </TD><TD> 51100</TD><TD>-</TD></TR><TR><TD> Low LldP- lldR</TD><TD> 664</TD><TD> 8400</TD></TR><TR><TD> High LldP- lldR</TD><TD> 12</TD><TD> 23.4</TD></TR></TBODY></TABLE><H3 class="inMenu">Characterization of synthetic promoter library</H3><H4>Overview</H4><P>To find the optimal regulatory system to integrate in our genetic circuit, we designed <A href="https://2015.igem.org/Team:ETH_Zurich/Parts#Part_Collection">eight versions</A> of the lldR-promoter that would be released from repression by LldR upon addition of Lactate. With our synthetic promoters we are planning to achieve higher sensitivity and reduced leakiness in respect to the natural version. We will remove the ArcA binding site to avoid unspecific regulation and will add LldR and LldP with higher expression levels than in the natural promoter, helping us to achieve the desired levels of expression. More information about the design process for these promoters can be found in our <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection#Design_of_synthetic_LldR_promoters">Part Collection</A></P><P><B>Table 2.</B> Design of the promoter regulated by LldR based on the natural version (<A href="http://parts.igem.org/Part:BBa_K822000">BBa_K822000</A>) by introduction of different <A href="http://parts.igem.org/Part:BBa_J23118">Anderson promoters</A>. The columns represent different topologies of the promoters with their operator sites and the rows show the different Anderson promoters used. .</P><TABLE><TBODY><TR><TH>O1-promoter-O2</TH><TH>promoter-O1-O2</TH><TH>promoter-O1-spacer-O2</TH></TR><TR><TH>J23117</TH><TD><A href="http://parts.igem.org/Part:BBa_K1847008">K1847008</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1847005">K1847005</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1847003">K1847003</A></TD></TR><TR><TH>J23118</TH><TD><A href="http://parts.igem.org/Part:BBa_K1847009">K1847009</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1847004">K1847004</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1847006">K1847006</A></TD></TR><TR><TH>J23100</TH><TD><A href="http://parts.igem.org/Part:BBa_K1847007">K1847007</A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1847002">K1847002</A></TD><TD>-</TD></TR><TR><TH>pLldR</TH><TD><A href="http://parts.igem.org/Part:BBa_K822000">K822000</A></TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P>For characterization of the promoter, we cloned all versions in front of a reporter-GFP. We then measured the sensitivity of our designed promoters to a range of Lactate concentrations by monitoring the GFP produced by our bacteria expressing constant LldR along with the nine reporter plasmids. Further information on the experimental setup can be found <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#E__coli_experiments">here</A>.</P><P>We tested the following combinations of plasmids:</P><OL><LI>a medium copy plasmid containing our designed lldR-promoters followed by a reporter GFP (<A href="http://parts.igem.org/Part:BBa_K1847008">lldRO1-J23117-lldRO2</A>-<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>-sfGFP)</LI><LI>the described medium copy plasmid and a low copy plasmid containing constant lldR (<A href="http://parts.igem.org/Part:BBa_J23118">promoter (J23118)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-<A href="http://parts.igem.org/Part:BBa_K1847001">lldR (K1847001)</A>-<A href="http://parts.igem.org/Part:BBa_B0015">terminator (B0015)</A>)</LI><LI>the described medium copy plasmid and a low copy plasmid containing constant lldR and constant lldP (low levels) (<A href="http://parts.igem.org/Part:BBa_J23114">promoter (J23114)</A>-<A href="http://parts.igem.org/Part:BBa_B0032">RBS (B0032)</A>-<A href="http://parts.igem.org/Part:BBa_K1847016">lldP-lldR (K1847016)</A>-<A href="http://parts.igem.org/Part:BBa_B0015">terminator (B0015)</A>)</LI><LI>the described medium copy plasmid and a low copy plasmid containing constant lldR and constant lldP (high levels)(<A href="http://parts.igem.org/Part:BBa_J23118">promoter (J23118)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-<A href="http://parts.igem.org/Part:BBa_K1847016">lldP-lldR (K1847016)</A>-<A href="http://parts.igem.org/Part:BBa_B0015">terminator (B0015)</A>)</LI></OL><P>For reasons of simplification, in the following we will refer to the genomic levels of LldR without overexpression from a plasmid as &quot;no lldR&quot;.</P><H4>Results</H4><P>Initial experiments with all nine promoters showed a clear bias of functionality towards the three versions resembling the architecture of the natural lldR-promoter (data not shown). For further experiments we therefore focussed on these versions:</P><UL><LI>natural version for reference <A href="http://parts.igem.org/Part:BBa_K822000">BBa_K822000</A></LI><LI>engineered P<SUB>lldR</SUB> with high promoter strength <A href="http://parts.igem.org/Part:BBa_K1847007">BBa_K1847007</A></LI><LI>engineered P<SUB>lldR</SUB> with medium promoter strength <A href="http://parts.igem.org/Part:BBa_K1847009">BBa_K1847009</A></LI><LI>engineered P<SUB>lldR</SUB> with low promoter strength <A href="http://parts.igem.org/Part:BBa_K1847008">BBa_K1847008</A></LI></UL><P>We were lucky to establish a <A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations#Stockholm">collaboration</A> with Stockholm, where they offered to characterize the <A href="https://2015.igem.org/Team:Stockholm/Collaborations#ETH_Zürich">less favorable designs</A> for us. In the following we will only focus on the four most promising versions.</P><P>We continued to characterize our reduced set of promoters by recording dose response curves amongst a broad range of lactate concentrations. The first thing to test was whether or not our promoters actually respond to lldR <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module#How_did_we_derive_the_model">as we expected</A> them to, i.e. if GFP intensity would be properly repressed in the presence of lldR. The resulting data is shown in Figure 4.</P><DIV class="imgBox" style="float:right;width:100%;margin: 10px 0px 10px 20px !important;"><P><B>Figure 4.</B> Comparison of the three candidate synthetic promoters and the wild type version regarding their dose response to L-lactate in the presence or absence of LldR and LldP. A is <A href="http://parts.igem.org/Part:BBa_K1847009">BBa_K1847009</A> (low), B is <A href="http://parts.igem.org/Part:BBa_K1847007">BBa_K1847007</A> (high) and C is <A href="http://parts.igem.org/Part:BBa_K1847008">BBa_K1847008</A> (medium). The dose-response curves for our synthetically designed LldR-promoters over a four orders of magnitude range of lactate concentration (1 uM to 335 mM) was determined 7 hours after induction. Due to apparent toxicity of L-lactate indicated by slower growth rates, only the values below 100 mM were analyzed. Mead values are calculated with n=3 ± SD.</P></DIV><P> Comparison of low levels of LldR and LldP, to Figure 4B, representing high levels of LldR, can suggest either of two things:</P><UL><LI>introduction of lldP into the system leads to leakiness due to uptake of possible traces of lactate in the medium</LI><LI>the levels of LldR are too low to properly suppress the LldR-promoter even without presence of lactate</LI></UL><P>To discriminate these two possibilities we tested the system, as mentioned, with a higher expression of LldR and LldP obtained due to a medium strength Anderson promoter (<A href="http://parts.igem.org/Part:BBa_J23118">BBa_J23118</A>). The reduced level of leakiness despite higher levels of lldP suggests that the problem before was not import of lactate from the medium (which should not be present anyways) but due to insufficient levels of LldR and therefore lack of proper repression in absence of lactate.</P><P>Based on these results we decided to use <A href="http://parts.igem.org/Part:BBa_K1847008">BBa_K1847008</A> in our final system since its high ON/OFF ratio with a value of 15.27 allows for sensitivity in the range in which we expect the lactate output of our cancer cells to be located. Additionally we showed that introducing the permease lldP increased sensitivity of our system to lactate. Further validation showed that <A href="#Increased_sensitivity_for_single_cell_lactate_output">lactate from mammalian cell</A> can be sensed in presence of lldP. All in all, we designed a promoter which is repressed by LldR and activated by L-lactate and responds with an increased sensitivity compared to the wild-type promoter.</P><H3 class="inMenu">Increased sensitivity for single cell lactate output</H3><H4>Overview</H4><P>We wanted to test if the lactate levels produced by our mammalian cells would be high enough to trigger GFP expression from our reporter plasmid, and whether or not there would be crosstalk between wells containing mammalian cells and wells without them. We combined bacteria and Jurkat or 3T3 cells, respectively, in our chip and recorded the bacterial growth along with the GFP fluorescence in 16 wells for 7 hours. We did the same experiment in absence of mammalian cells as a negative control.</P><H4>Results</H4><P>The results obtained from the lactate detection measurements within our nanowellplate show a clear increase of normalized fluorescence over time.</P><DIV class="imgBox"><P><B> Figure 5. </B><I>E.coli</I> sensing lactate produced Jurkat cells in the chip in RPMI medium: <I>E.coli </I> harboring the same promoter P<SUB>LldR</SUB> and either LldR or LldPR were introduced in the chip with and without Jurkat cells. </P></DIV><P>To compare the responses of <I> E. coli </I> in RPMI or DMEM medium, we performed the same experiments as described <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Characterization_of_the_lldR_promoter_s_response_to_lactate">previously</A> with for our bacteria in these two media in a 96-well plate using our microtiter plate reader (see Figure 6).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 6: Dose response curve of low P<SUB>LldR</SUB> construct to lactate in mammalian culture medium. Red and black curves represent growth in RPMI, the medium used for Jurkat cells. Green and blue curves represent growth in DMEM, the medium used for 3T3 cells. It is very clear from this data that the introduction of LldP is beneficial for both the ON/OFF ratio as well as the K<SUB>M</SUB> value of the promoter.</DIV></DIV></DIV></DIV><P> Regarding the cross-reaction of bacteria in wells without mammalian cells we found that lactate will easily diffuse in the medium covering the chip. In further experiments, the coverslip covering the filled nanowellplate should be sealed more tightly in order to prevent a continuous phase of medium covering and therby interlinking the different wells. Nevertheless, contol experiments without mammalian cells in the chip clearly showed that there is no self activation by bacteria within a chip that is devoid of mammalian cells (see Figure 5). This is consistent with our finding that <A href="#lactateresults"><I>E. coli</I> cells do not produce significant levels of lactate</A> themselves.</P><H3 class="inMenu" id="Characterization_of_the_lacI-lldR_promoter">Characterization of the lacI-lldR promoter</H3><H4>Overview</H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:266px;"><DIV class="thumbcaption">Figure 7. Response expected from the hybrid promoter, resulting in an AND-gate.</DIV></DIV></DIV><P>For the establishment of a <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Fold-change_sensor">fold change</A> sensor topology, a gateway integrating a positive and a delayed negative signal is required. For our system we engineered a hybrid promoter integrating repression by LacI and LldR. The two signals come together to influence our <B>newly</B> designed <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection#Double_responsive_promoter">hybrid promoter</A> in which we combined the Lac promoter with its operator (lacO) in the LldR operator promoter. We designed two hybrid promoters using different versions of the <I>lac</I> promoter. A closer description of the <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection#Double_responsive_promoter">design process</A> for these promoters can be found in our <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A>.</P><P>We tested both of our synthetic hybrid promoters for their leanikess and response to L-lactate, as well as IPTG. The setup of this experiment was a strain containing the following constructs.</P><UL><LI>a reporter GFP under the control of our hybrid promoter (<A href="http://parts.igem.org/Part:BBa_K1847010">lldRO1-plac-lacO-lldRO2</A>-<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>-sfGFP or <A href="http://parts.igem.org/Part:BBa_K1847011">lldRO1-plac-lacO-lldRO2</A>-<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>-sfGFP</LI><LI>constantly expressed lldR and lldP (<A href="http://parts.igem.org/Part:BBa_J23118">J23118</A>-<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>-<A href="http://parts.igem.org/Part:BBa_K1847000">lldP-lldR</A>-<A href="http://parts.igem.org/Part:BBa_B0015">B0015</A>)</LI><LI>constantly expressed LacI (<A href="http://parts.igem.org/Part:BBa_J23109">J23109</A><A href="http://parts.igem.org/Part:BBa_B0034">B0034</A><A href="http://parts.igem.org/Part:BBa_C0012">LacI</A>)</LI></UL><H4>Results</H4><P>The promoter responded to both lactate and IPTG, as can be seen in Figure 8. </P><DIV class="imgBox" style="float:right;width:100%;margin: 10px 0px 10px 20px !important;"><P><B>Figure 8.</B> Heat map showing the fluorescence intensity depending on the concentration of the lactate and IPTG for <A href="http://parts.igem.org/Part:BBa_K1847010">P<SUB>lac-lldR</SUB></A></P></DIV><DIV class="imgBox" style="float:right;width:100%;margin: 10px 0px 10px 20px !important;"><P><B>Figure 9.</B> Heat map showing the fluorescence intensity depending on the concentration of the lactate and IPTG for <A href="http://parts.igem.org/Part:BBa_K1847011">P<SUB>lacUV5-lldR</SUB></A>.</P></DIV><P>These results show that one of our combined promoters, P<SUB>lac-lldR</SUB> (<A href="http://parts.igem.org/Part:BBa_K1847010">K1847010</A>), reacts in an AND gate fashion to a combination of lactate and IPTG. Only in presence of both input signal there is high expression of the reporter GFP. We hereby <B>show that our synthetically engineered hybrid promoter P<SUB>lac-lldR</SUB> exhibits the desired behavior</B> which will be favourable for the establishment of an AND gate in our <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Fold_change_sensor">fold change sensor</A>, with an ON/OFF ratio of 15. The second hybrid promoter we designed, P<SUB>lacUV5-lldR</SUB> (<A href="http://parts.igem.org/Part:BBa_K1847011">K1847011</A>), does not show the desired behavior in response to IPTG. A possible reason for this could be that the optimized LacUV5 promoter is changed in respect to P<SUB>lac</SUB> in a way that relies on free accessibility of the operator, which might no longer be given if there is LldR interacting with the LldR-operators close by.</P><P>What can be observed in Figure <B class="TODO">xx</B> is that the repression due to LldR is much tighter in the absence of lactate, than the repression due to LacI in the absence of IPTG. A possible explanation for this may be found in the mechanism, which is thought to mediate the formation of a DNA loop <A href="https://2015.igem.org/Team:ETH_Zurich/References#Aguilera2008">[Aguilera 2008]</A>. It is also possible that lactate acts as an additional activator for the responding <A href="#Increased_sensitivity_for_single_cell_lactate_output">promoter</A>. Additionally, upon addition of lactate, the loop is resolved, which might be necessary for LacI to excert its inhibitory effect. Further experiments including the reaction of the hybrid promoters to LacI in absence of LldR are required for more detailed conclusions.</P><H3 class="inMenu">Quorum sensing module: influence of AHL degradation</H3><H4>Overview</H4><P>The basis of <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Quorum_sensing">quorum sensing</A> is the leaky production of <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Quorum_sensing">N-acyl homoserine lactone (AHL)</A> in absence of an external cue which is crucial to allow for basal levels of AHL needed to trigger quorum sensing upon colocalization of several bacteria. Unfortunately, this leakiness interferes greatly with our system, since the main point of a binary signal like we want it, is the tightness in absence of a trigger. We thought of several strategies to deal with this issue. One option was the introduction of AHL degradation which would help us eliminate basic production levels of AHL. We therfore included constitutive expression of the <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">AHL Lactonase AiiA</A> in our design. To validate our design, we tested the influence of AHL degradation on the sensitivity to a large range of AHL concentrations.</P><P>We tested the response to AHL for the following genetic setups:</P><UL><LI>a reporter GFP under control of Plux (<A href="http://parts.igem.org/Part:BBa_R0062">pluxR (R0062)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-sfGFP)</LI><LI>a reporter GFP under control of PluxR along with constitutive LuxR (<A href="http://parts.igem.org/Part:BBa_R0062">pluxR (R0062)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-sfGFP-<A href="http://parts.igem.org/Part:BBa_B0012">terminator (B0012)</A>, <A href="http://parts.igem.org/Part:BBa_J23118">promoter (J23118)</A>-<A href="http://parts.igem.org/Part:BBa_B0032">RBS (B0032)</A>-<A href="http://parts.igem.org/Part:BBa_C0062">LuxR (C0062)</A>-<A href="http://parts.igem.org/Part:BBa_B0012">terminator (B0012)</A>)</LI><LI>a reporter GFP under conrol of Plux along with constitutive LuxR and constitutive AiiA (<A href="http://parts.igem.org/Part:BBa_R0062">plux (C0062)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-sfGFP-<A href="http://parts.igem.org/Part:BBa_B0012">terminator (B0012)</A>, <A href="http://parts.igem.org/Part:BBa_J23118">promoter (J23118)</A>-<A href="http://parts.igem.org/Part:BBa_B0032">RBS (B0032)</A>-<A href="http://parts.igem.org/Part:BBa_C0062">LuxR (C0062)</A>-<A href="http://parts.igem.org/Part:BBa_B0012">terminatr (B0012)</A>, <A href="http://parts.igem.org/Part:BBa_J23118">promoter (J23118)</A>-<A href="http://parts.igem.org/Part:BBa_B0034">RBS (B0034)</A>-<A href="http://parts.igem.org/Part:BBa_C0160">aiiA (C0160)</A>)</LI></UL><H4>Results</H4><P>Our <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#AHL_response_experiments">experiments</A> comparing the dose-response of our system to AHL in the presence or absence of AiiA provided us with valuable results and parameters for the model, especially regarding apparent K<SUB>M</SUB> values:</P><TABLE><TBODY><TR><TH>
K<SUB>M</SUB></TH><TD>
9.983 nM
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>
K<SUB>M</SUB></TH><TD>
10<SUP>6</SUP> nM
</TD></TR></TBODY></TABLE><P><B> Figure 10.</B> Comparison of the dose-response curve A: in absence and B: in presence of AiiA. Experimental data points and solid line for fitted curves of AiiA. GFP fluorescence was measured 7 hours after induction with various AHL concentrations. Values for the apparent K<SUB>M</SUB> values are given for both cases as mean fluorescence value with n=3 ± SD.</P><P>AHL degradation by AiiA shifts the curve by<B>five orders of magnitude</B> towards lower sensitivity, i.e. a higher K<SUB>M</SUB> value. We calculated the apparent half maximum substrate concentration for both cases: with (10<SUP>6</SUP> nM) and without (9.983 nM) AiiA. The <A href="http://parts.igem.org/Part:BBa_C0160:Experience">characterization of AiiA</A> provided by these results indicate that the K<SUB>M</SUB> of Plux can be tuned over a big range of concentration by adaption of the expression levels of AiiA, since in our experimental setup we  are comparing the absence of AiiA with a <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module#Concentration_of_AiiA_inside_the_E._coli">high level of AiiA</A>, leaving a lot of freedom to adaption of the levels of expression.</P><P>Upon <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module#_Characterization_of_the_effect_of_AiiA">comprehensive modeling</A> of our system including the parameters gained from this study we could show that AHL degradation may solve our problem of leakiness of AHL production encountered in our system. However, to overcome the AHL degradation at high levels of AiiA, our cells would have to produce very high levels of AHL as well, which will be impossible in practice due to the limited number of bacteria in our <A href="https://2015.igem.org/Team:ETH_Zurich/Chip">singel cell analysis setup</A>. Lowering the levels of AiiA would allow for sensitivity in the desired range of AHL. However, it will prevent the buildup of high enough gradient in AHL levels between bacteria colocalized on a cancer cell as opposed to <I>E. coli</I> cells floating freely in the bulk. We therefore decided not to rely on AHL degradation for our final system. Instead, we focused on the optimization of our <A href="#Tightness_of_the_AND_gate_between_lactate_module_and_AHL_module">AND gate</A>.</P><H3>Tightness of the AND gate between lactate module and AHL module</H3><H4>Overview</H4><P>The ultimate goal of our system's setup is to receive a binary signal combining two independent indicators of CTC presence. The biggest hurdle in this was the leakiness of the <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Quorum_sensing">quorum sensing</A> system around LuxR. We therefore decided to arrange our two modules (the <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Sensor</A> and the <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL module</A>) in a sequential way in our <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model#AnD_gate">system</A>.</P><H4>Results</H4><P>We are working hard on last minute results!</P><H3>Establishing a system for display of Annexin V on <I>E. coli</I> outer membrane</H3><H4>Overview</H4><P>We conceptualize our Microbeacon system to rely on bacteria binding to cancer cells as a detection signal. The solution that we propose is to use Annexin V expressed on the outer membrane to bind cancer cells. Annexin V binds to phosphatidylserine (<A href="https://2015.igem.org/Team:ETH_Zurich/References#Leventis2010">Leventis, 2010</A>), which is a membrane lipid usually found in the inner part of the cell, but in cells undergoing apoptosis it is exposed to the outer membrane due to the activity of flipases (<A href="https://2015.igem.org/Team:ETH_Zurich/References#Leventis2010">Leventis, 2010</A>). </P><P>Annexin V is a soluble protein which has an unknown role in mammalian cell except its binding properties to membrane components(<A href="https://2015.igem.org/Team:ETH_Zurich/References#Gerke2002">Gerke <I>et al</I>, 2002</A>). As we want to express Annexin V in the outer membrane we need a protein that will export it. For this purpose we chose INP and changed the YFP, present in the part (<A href="http://parts.igem.org/Part:BBa_K523013">BBa_K523013</A>), for a human Annexin V protein optimized for <I>Escherichia coli</I> codon usage (ThermoFisher Scientific GeneOptimizer(R)). We used a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), a low expression promoter (<A href="http://parts.igem.org/Part:BBa_J23114">BBa_J23114</A>) and a low copy plasmid with a pSC101 origin of replication to avoid an excessive stress for the cell, as it is a membrane protein.</P><H4>Results</H4><P>To test our construct we used:</P><UL><LI>J23114-B0032-INP-Annexin V in pSEVA371 (low copy plasmid)expressing RFP in TOP10 cells</LI><LI>J23114 expressing RFP in TOP10 cells</LI></UL><P>As a first test, we did an assay with beads. We used <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Dynabeads_for_verification_of_Annexin_externalization">magnetic beads</A> coated with Protein A, which we incubated with monoclonal mouse anti-Annexin V antibodies (Affimetrix) for 30 min and then with the bacteria with different constructs. Unfortunately, we were not able to detect binding of our cells.  We also did a Western blot to analize the expression of our construct using our anti-Annexin V antibody (Affimetrix) and we observed no band at the height where Annexin V is expected (35.9 kDa). </P><P>We wondered what could have gone wrong, so we checked with an RBS calculator (<A href="https://salislab.net/software/">Salis <I>et al</I>, 2015</A>) whether the expression would be high enough with our design. There, we realized that Annexin V showed a very low calculated translation initiation rate, which might indicate low or no expression. Using B0032 with INP-Annexin V we were getting a translation initiation rate of 410 a.u., while if we changed the RBS to B0034, we would get 12 000 a.u. Therefore, we changed the RBS of INP-Annexin V and Annexin V to B0034. As can be seen in Figure 5 we can see a band at the expected 35.9 kDa. This confirms the expression of Annexin V in our cells. Even though the band is weak, which we expected for low expressed membrane proteins, it indicates that INP-Annexin V is also expressed, which can be seen with a band at the corresponding 70.4 kDa. To confirm that INP-Annexin V was on the membrane fraction we run both the unclarified cell free extract (called crude from now on) of each sample and the supernatant of the centrifuged sample. Annexin V is present in both fractions, while INP-Annexin V is only visible in the crude fraction (Figure 5), likely to be part of the membrane containing cell debris.</P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 11.</B> Western blot image of B0034-Annexin V and B0034-INP-Annexin. Samples of Annexin V and INP-Annexin V were loaded both with a low copy plasmid (pSEVA371) and a high copy plasmid (pSB1C3). Each time a sample of the crude and of the supernatant were loaded on the gel. Detection was performed with a monoclonal mouse anti-Annexin antibody with a secondary goat anti-mouse antibody labelled with Alexa 488. </P></DIV><P>For the visualization of the Western Blot, we were forced to increase the sensitivity of the film a lot, which might indicate that the concentrations of the protein inside the cell are very low or that our primary antibody is not functioning too well.</P><H3>INP-Annexin V externalization</H3><H4>Overview</H4><P>After proving expression of our INP-Annexin V construct it is also crucial to test for it's functionality. We did so by using Protein G coated Dynabeads loaded with an anti-Annexin V antibody. The <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Dynabeads_for_verification_of_Annexin_externalization">detailed protocol</A> can be found in the experiments section.</P><P>We prepared two possible datasets:</P><UL><LI>Beads without loaded antibody. Five different conditions were compared: </LI><UL><LI>INP-Annexin V-RFP in a high copy plasmid</LI><LI>INP-Annexin V-RFP in a low copy plasmid</LI><LI>RFP in a high copy plasmid</LI><LI>Annexin-Alexa 488 purified</LI><LI>PBS</LI></UL><LI>Beads with loaded antibody. </LI><UL><LI>INP-Annexin V-RFP in a high copy plasmid</LI><LI>INP-Annexin V-RFP in a low copy plasmid</LI><LI>RFP in a high copy plasmid</LI><LI>Annexin-Alexa 488 purified</LI><LI>PBS</LI></UL></UL><P>After incubation of the antibody coated beads with the respective bacterial strain or control, we observed our beads using the microscope.</P><H4>Results</H4><P>First thing we did was to check the positive (Alexa-488 and beads with antibody and Alexa-488) and negative controls (beads without antibody). We found out that there was no green fluorescence in the positive control, <A href="#Establishing_a_system_for_display_of_Annexin_V_on_E__coli_outer_membrane">confirming</A> again our fear that the <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Anti-Human-AnnexinV">anti-Annexin antibody</A> we are using is not binding as expected. Neither of our samples presented fluorescence from the secondary antibody either. We therefore concluded that the antibody was not very effective to detect annexin bound to another molecule. We came back to our Western blot (see Figure 11), where we used the same antibody, and saw that there were some unspecific bands at about 55 kDa. Also, the expected bands were very weak.</P><P>As can be seen in Figure 12, no difference can be seen in fluorescence levels of the beads. Moreover, bacteria have a similar size to bacteria (2.8 µm and 2 µm, respectively), so they would be seen in the microscopy picture if they were there. </P><DIV class="imgBox" style="width:50%;margin: 20px auto 20px auto !important;"><P><B> Figure 12.</B> Experiment for binding beads to anti-Annexin V antibody with four different conditions: INP-Annexin V-RFP in high copy plasmid, INP-ANnexin V-RFP in low copy plasmid, RFP in high copy plasmid and a negative control (no Annexin V).</P></DIV><H3>INP-Annexin V functionality</H3><H4>Overview</H4><P>A properly expressed and exported Annexin V protein is the first step towards a bacterial apoptosis detection system. However, the validation of the functionality of Annexin V in this construct is also crucial. We therefore tried several approaches to prove functionality of this construct regarding binding to apoptotic cells. For easy monitoring of our bacteria, we introduced an RFP construct into the same plasmid carrying INP-Annexin V. We published this construct in the registry to allow future iGEM teams to use our <A href="http://parts.igem.org/Part:BBa_K1847015">bacterial apoptosis detector (BBa_K1847015)</A> for their project.</P><H4>Results and Discussion</H4><P>The first atempt we took were <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Phosphatidylserine-coated_beads_for_verification_of_Annexin_functionality">beads coated in phosphatidyl serine</A>, which would simulate the surface of an apoptotic mammalian cell. Unfortunately, we were not able to detect binding of Annexin-Alexa 488 or bacteria to these beads.</P><P>However, these results cannot be taken as a total disproval of our system, since it is not clear how well the phosphatidylserine (PS)-coated beads simulate an apoptotic cell. We came up with the following possible explanations as to why this setup might not be ideal:</P><UL><LI>It is possible that too high concentrations of PS (85 pM) on the beads interferes with the Annexin V-PS interaction.</LI><LI>The <A href="http://lipidlibrary.aocs.org/Primer/content.cfm?ItemNumber=39360">hydrophobic tails of PS</A> might form clumps if they are not embedded into a membrane, thereby hindering interactions.</LI><LI>The <A href="https://echelon-inc.com/index.php?module=Products&amp;func=detail&amp;id=760">biotin moiety</A> required to bind PS onto the beads might be positionned on the lipid such that the interaction with Annexin V is no longer possible</LI></UL><P>But this initial backset could not stop us! We decided to try a more realistic setup, testing the binding of our <I>E. coli</I> cells expressing the INP-Annexin V construct to actual cancer cells. The results of this experiment can be found <A href="#Detection_of_apoptotic_cells_by_our_engineered_bacteria">here</A>.</P></DIV><DIV class="expContainer"><H2> Experiments involving mammalian cells </H2><P>Our first step needed to check the viability of our experimental set-up was to confirm the data found in the literature regarding the sensitivity to TRAIL [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Thomas1998">Thomas, 2008</A>] and the elevated lactate production observed in cancer cells, but not in non-cancer cells.</P><H3>Sensitivity to TRAIL</H3><H4>Background</H4><P>Apoptosis can be induced through a protein called TNF-related Apoptosis Inducing Ligand (TRAIL). TRAIL is a transmembrane protein, which can be found also in solubilised form (sTRAIL)[<A href="https://2015.igem.org/Team:ETH_Zurich/References#Bryune2013">de Bruyne <I>et al</I>, 2013 </A>], that recognizes the death receptors DR4, DR5, DCR1, and DCR2 [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Johnstone2008">Johnstone <I>et al</I>, 2008</A>] and has been used in cancer treatment inducing the apoptosis of cancerous cells but with low cytotoxicity for healthy ones [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Johanstone2008">Johnstone <I>et al</I>, 2008</A>; <A href="https://2015.igem.org/Team:ETH_Zurich/References#Zhang2005">Zhang <I>et al</I>, 2005</A>]. 

</P><H4>Experiment overview</H4><P>The first signal in our system is based on making cancer cells apoptotic while non-cancer cells remain untouched. Due to the properties of TRAIL explained before, we used it for a selective apoptosis induction in cancer cells. </P><P>The aim of this experiment was to check if cancer cells effectively undergo apoptosis after incubation with TRAIL and how long the incubation should last.</P><P>To test the predisposition of cancer cells to TRAIL-induced apoptosis we chose Jurkat cells, which are an immortalized human T lymphocyte cell line; and HL-60 cells, a line derived from acute myeloid leukemia. For a control with healthy cells we used a murine fibroblast cell line (3T3). </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:403px;"><DIV class="thumbcaption"><B> Scheme 1.</B> Scheme of the experiment for TRAIL sensitivity.</DIV></DIV></DIV></DIV><H4>Methods and Results</H4><P>Our first experiment consisted in incubating Jurkat and 3T3 cells with TRAIL for 4h and 6h (see <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Apoptosis_susceptibility_experiment_with_sTRAIL">protocol about apoptosis sensibility</A>). Concentrations of 0, 50 and 100 ng/mL of TRAIL were used with each cell line. A positive control for Jurkat cells was applied by incubating these cells with PMA and ionomycin overnight <A href="https://2015.igem.org/Team:ETH_Zurich/References#Park2001"> Park <I>et al</I>, 2001 </A>. After that, FACS analysis was performed. To differentiate between apoptosis and necrosis, PI dye was used to stain necrotic cells <A href="https://2015.igem.org/Team:ETH_Zurich/References#Rieger2011"> Rieger <I>et al</I>, 2011 </A>. Apoptotic cells were detected by using <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#AnnexinV-Alexa">Annexin V labelled with Alexa 488</A><SUP>1)</SUP>.</P><P>Unfortunately, despite doing several trials in which we incremented the number of hours for incubation (up to 48h) and tried different concentrations of <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#TRAIL">TRAIL</A>, we were not able to replicate the results we found in the literature [<A href="https://2015.igem.org/Team:ETH_Zurich/References#Johanstone2008">Johnstone <I>et al</I>, 2008</A>; <A href="https://2015.igem.org/Team:ETH_Zurich/References#Zhang2005">Zhang <I>et al</I>, 2005</A>]. To see where the problem was, we decided to incubate the cells with 1000 ng/mL of TRAIL over a period of 48 h. If our TRAIL was functional, we would expect to see a change in the cell's morphology after this time, indicating that our cells are apoptotic but our Annexin V-based detection system is not working, despite apoptosis being properly induced. If TRAIL was not functional, we expect not to see morphological differences. After the 48h we saw that there were no differences in the morphology of the cells, which lead us to suppose that probably the sTRAIL we bought was not working as expected. </P><P>Our next step was to purchase another TRAIL called <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#SuperKillerTRAIL™">SuperKillerTRAIL</A>, for which we found promising entries in the literature <A href="https://2015.igem.org/Team:ETH_Zurich/References#Mandal2014">[Mandal <I>et. al</I> (2014)]</A>. To check whether this time our experiments were working, we added 100 ng/mL of TRAIL to Jurkat cells and measured the apoptosis at 2 h and 4 h. We observed that in 2 h apoptosis increased from about 2% of the population in non-treated samples to almost 70% in cells incubated with TRAIL. After 4 h the apoptotic cell number increased up to 75% (see Figure 1). As the number of apoptotic cells is increasing slowly after 2 h, we decided to use this as our standard time for TRAIL incubation.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B> Figure 13.</B> Jurkat cells with 0 and 100 ng/mL of TRAIL after 2 and 4 h. Apoptosis was detected in all cases with Annexin V (Q3).</DIV></DIV></DIV></DIV><P>Once we confirmed that TRAIL was effectively inducing apoptosis in Jurkat cells, we wanted to do the complete experiment, in which we determine the concentration of TRAIL we will use in future experiments and contrast Jurkat cells with 3T3 (a non-cancerous cell line). </P><P>Jurkat cells and 3T3 were incubated for 2 h in 0, 10, 50 and 100 ng/mL of TRAIL. Then, we followed the <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Apoptosis_susceptibility_experiment_with_sTRAIL">protocol detailed in Experiments</A>. Annexin V was added to detect apoptosis, as it binds to phosphatidylserine which is found in higher proportion in apoptotic cells. To differentiate apoptotic and necrotic cells, PI dye was added. Annexin V and PI were detected by an emission filter at 488 and 561 nm, respectively.</P><P>Our data shows that 3T3 are not affected by TRAIL after 2h, whereas Jurkat cells are (as have been shown in the previous experiment). For Jurkat cells we decided to use a concentration of 100 ng/mL, as it allows to have a higher number of apoptotic cells in a relatively short amount of time (2 h). </P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 14.</B> 3T3 (upper image) and Jurkat cells (lower image) with 0, 10, 50 and 100 ng/mL of TRAIL after 2 h incubation. Annexin V-Alexa488 (488 nm) was used to detect apoptotic cells. To differentiate necrotic cells, PI was added (561 nm). </P></DIV><P>In Figure 14 in the upper image, for 0 and 50 ng/mL one can see that there is no necrotic cells (Q2). This is probably due to a mistake in manipulation, and these sample lacked PI.</P><P>Finally, to check if it was also possible to see apoptotic cells labelled with Annexin V in the microscope, we did an experiment with four conditions. One sample had no TRAIL and no Annexin V; the other had no TRAIL and Annexin V; the next had TRAIL and no Annexin V; and the final had both TRAIL and Annexin V.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 15. Jurkat cells with and without incubation with 100 ng/mL of TRAIL for 2 h, with or without Annexin V-Alexa488 staining. Annexin-Alexa488 positive cells are apoptotic. Magnification: 40x.</DIV></DIV></DIV></DIV><P>As we expecteed, the only sample that had fluorescence at 488 nm was the one with both TRAIL and Annexin V.</P><H3>Lactate production</H3><H4>Experiment overview</H4><P>A lot of cancer cells ungergo a metabolic transition, called Warburg effect, where they do the aerobic glycolysis to obtain ATP. A subproduct they generate in this kind of energy production id L-Lactate (Vander <I>et al</I>, 2009). Therefore, the excretion of Lactate by cancer and healthy cells is different enough to be used as an indicator of cancer.  </P><P>First, we wanted to check the how the production rates of lactate would vary in healthy and cancer cells. To this end, we analysed the medium in which our cells of interest had been incubated for a defined time in a defined seeding density. For detection of lactate in a sample, we used a <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Lactate_Detection_Kit">L-lactate kit (Megazyme)</A>, in which two enzymatic reactions lead to a measurable concentration of NADH proportional to the Lactate content of the sample, which can then be measured as optical density at 340 nm (OD340). The kit was used applying the provided protocol. </P><H4 id="lactateresults">Methods and Results</H4><P>We used Jurkat cells as our cancer cells, 3T3 as our control for non-cancer cells and we also measured the concentration of lactate in bacteria. We expected the Jurkat cells to produce more lactate than the 3T3 and the bacteria to produce none.</P><P>The first problem we came across measuring the absorbance was the phenol red of the medium of the mammalian cells, which absorbs at the same wavelength as the lactate kit. Once the phenol red was removed, there was still another problem: a component of the medium seemed to react with our kit. After a bibliographic search, we decided that removing FBS from the medium was our best chance to get reliable calibration curves and measurements <A href="">(researchgate.net)</A>.  We also incorporated a centrifugation step after the collection of the sample to remove any cell that could stay in suspension. </P><P>In the case of bacteria, we wanted to take into consideration their fast growth compared to mammalian cells, so we measured the OD<SUB>600</SUB> at each time point to have a correct normalization.</P><P>Lactate concentrations were first measured per well, and then normalized by the number of cells present in the well (see <A href="https://2015.igem.org/Team:ETH_Zurich/Experiments#Mammalian_cell_experiments_">protocol</A> about the measurement of Lactate in mammalian cells). The results can be seen in Figure 4.</P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 16.</B> Total concentration of lactate in a well of a 96-well plate (upper image) and normalized values of lactate per cell(lower image). </P></DIV><P>As can be seen in Figure 16, in the case of bacteria our results confirm the first hypothesis according to which bacteria do not produce lactate. However, they seem to contradict the presence of Warburg effect, as the concentration of lactate per cell is higher in 3T3 than in Jurkat cells. We came across some possible explanations for this phenomenon:</P><UL><LI>3T3 is a murine cell line, while Jurkat cells are a human cell line. It could be possible that basal murine lactate levels are higher than in humans. For checking this hypothesis, we wanted to use human fibroblasts. Unfortunately, none of the labs of our department had any adequate cell line for our purposes and due to time constrains we could not ask other departments.</LI><LI>3T3 are adherent cells, while Jurkat cells are non-adherent. Although both cell lines were seeded for not reaching confluency in the next two days it is still possible that the different cell density influenced the metabolism and therefore the lactate production.</LI></UL><P>In conclusion, we consider that the choice of the 3T3 as a control line was not our best option, but was done due to availability of the cell line and our inexperience using mammalian cells. For future experiments, we want to change our control cell line to produce more conclusive data.</P><P><A href="">1) </A>During the process of necrosis, parts of the inner cell membrane leaflet are exposed to the environment, including phosphatidylserine. Therefore we also expect some Annexin-positive cells in the absence of apoptosis.

</P></DIV><DIV class="expContainer"><H2>Mammalian cells and bacteria combined</H2><H3>Co-culture of mammalian cells with bacteria</H3><H4>Overview</H4><P>One issue that was concerning us was the possibility that the mammalian cells would behave in unusual ways in presence of bacteria. To exclude this possibility we decided to co-culture mammalian cells with bacteria for 3 h which is the time we expect that our test will last. </P><P>We used <I>E. coli</I> expresssing GFP and for the mammalian cell lines we used 3T3, HL-60 and Jurkat lines. </P><H4>Results</H4><P>We could observe that after the set time point (3 h) the 3T3 cells incubated with GFP-expressing bacteria had the same morphology as in the control. However, in Jurkat cells, it was possible to see some unspecific binding of bacteria to the cells.</P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 17.</B> Co-culture of 3T3 and Jurkat cells without and with bacteria expressing GFP after 3h of incubation. </P></DIV><H3>Detection of apoptotic cells by our engineered bacteria expressing Annexin V</H3><H4>Overview</H4><P>The ultimate goal of the introduction of our INP-Annexin V construct into <I>E. coli</I> is to make our MicroBeacons bind to apoptotic cancer cells. In order to prove this, we induced apoptosis in Jurkat and HL60 cells by incubating them with 100 ng/mL for 2 hours before combining them with our engineered bacteria. The final analysis was performed using microscopy. We verified apoptosis using Annexin V-Alexa488 staining.</P><H4>Results</H4><P>Unfortunately we were unable to show binding of our <I>E. coli</I> cells to apoptotic cells. On the contrary, it seemed that our bacteria bound preferrentially to non-apoptotic cells. We peformed an extensive series of experiments to achieve better results, adapting incubation time with the bacteria (1 h, 2 h, 3 h), calcium concentration of the <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Annexin_V_binding_buffer">Annexin binding buffer</A> (0.25 mM, 2.5 mM, 25 mM) and incubation temperature with the bacteria (4ºC, 25ºC, 37ºC). Still, it seems that our bacteria (both with and without the INP-Annexin construct (<A href="http://parts.igem.org/Part:BBa_K1847015">BBa_K1847015</A>) are only bining to healthy cells and only under low concentrations of calcium. Our previous <A href="#Co_culture_of_mammalian_cells_with_bacteria">coculture experiments</A>, however, showed that in complete absence of calcium the bacteria are not sticking to mammalian cells.</P><P>We are currently looking into alternative ways to get Annexin V exposed on the surfcae of our <I>E. coli</I>.</P><H3>Detection of apoptotic cells by our engineered bacteria using autodisplay</H3><H4>Overview</H4><P>As we could not prove the externalization of Annexin V by INP, we changed the strategy to use an Autodisplay system. Autodisplay consists in a beta-barrel that folds on the cell membrane putting outside of the cell the construct of interest. </P><H4>Results</H4><P>Results with Autodisplay were more promising than with INP. The bacteria that contained the Autodisplay system bound to the cells when compared with bacteria containing only RFP. Binding did not seem to be specific to cancer cells. However, we consider that due to the manipulation of the sample (incubation of the cells without a CO<SUB>2</SUB> pressure and at room temperature) could trigger apoptosis in the samples treated and not treated with TRAIL. More experiments will be needed to see if specific bnding to cancer cells can be finally achieved.  </P><DIV class="imgBox" style="width:90%;margin: 20px auto 20px auto !important;"><P><B> Figure 18.</B> Cells expressing Annexin V with an Autodisplay system compared to cells containing only RFP. </P></DIV></DIV><DIV id="footerContainer"><H2>We would like to thank our sponsors</H2></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>