<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mobileMenu"><UL class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV><DIV id="mainContainer"><DIV id="topContainer"><DIV id="menuContainer"><UL id="mainMenu" class="hideMenu"><LI class="sub">Project
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Project_Description">Project Description</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Glossary">Glossary</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/References">References</A></LI></UL><LI class="sub">Modeling
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module">Lactate Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/AHL_Module">AHL Module</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Single-cell_Model">Compartment Model</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Reaction-diffusion">Reaction-Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Experiments_Model">Experiments Meet Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">Parameters</A></LI><LI class="sub">Lab
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Lab">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip Design</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Interlab_study">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Materials">Materials</A></LI><LI class="sub ignoreSwitch">Human Practices
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Medicine">Medicine</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Awareness">Awareness</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Business">Business</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Practices/Patents">Patents</A></LI><LI class="sub">Parts
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Parts">Overview</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Basic_Part">Basic</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Description">Characterization</A></LI><LI class="sub">About Us
    </LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1847">iGEM Team Roster</A></LI></DIV></DIV><DIV id="sidemenuContainer"><DIV id="sidemenu" class="smtop smmid smbot"><I>&quot;What I cannot create I do not understand.&quot;</I> - Richard Feynmann
</DIV></DIV><DIV class="expContainer" style="position:relative"><H1>Experiments &amp; Protocols</H1><P>In the course of our project we performed various experiments, ranging from construction and transformations of plasmids into <I>E. coli</I> over the testing of their functionality and response measurements after induction of our circuit, up to experiments with mammalian cells, which are the subjects of our analysis. In addition, we designed and tested a <B>microfluidic setup</B> for our final system</P><P>Below you can find the protocols for all our experiments, grouped into experiments with <A href="#E__coli_experiments">bacteria</A>, <A href="#Mammalian_cell_experiments">mammalian cells</A>, the <A href="#Bacteria_and_mammalian_cells">combination of both</A>, and some <A href="#General_useful_things">general useful basics</A>.</P></DIV><DIV class="expContainer"><H2><I>E. coli</I> experiments</H2><DIV class="highlightBox"><H3>Overview</H3><P>The first important feature of our engineered <I>E. coli</I> is their <A href="https://2015.igem.org/Team:ETH_Zurich/Description">binding capability to apoptotic cells</A> via our <A href="http://parts.igem.org/Part:BBa_K1847015">INP-Annexin V</A> construct which makes them display Annexin V on their outer membrane. We performed experiments to show <A href="#Western_Blot">expression</A>, <A href="#Antibody_staining_for_Annexin_V">externalization</A>, and <A href="#Phosphatidylserine-coated_beads_for_verification_of_Annexin_functionality">functionality</A> of this construct.</P><P>In our system, the actual detection of <I>E. coli</I> binding to apoptotic cancer cells happens via <A href="https://2015.igem.org/Team:ETH_Zurich/Description#Quorum_sensing">quorum sensing</A> upon colocalisation of bound bacteria. We therefore designed experiments to test the <A href="#AHL_response_experiments">quorum sensing properties</A> of our final and working constructs.</P><P>The second signal which will be integrated by our bacteria is the <A href="https://2015.igem.org/Team:ETH_Zurich/Description">elevated lactate production</A> by cancer cells when compared to non-cancer cells. The main part of the <A href="https://2015.igem.org/Team:ETH_Zurich/Design">lactate sensor</A> we implemented in our system consists of <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">synthetically designed promoters</A> that is regulated by LldR and lactate, as well as by LacI. We characterized the <A href="#Characterization_of_the_lacI-lldR-hybrid_promoter's_response_to_lactate_and_IPTG">sensitivity</A> of these promoters to lactate and IPTG to find the <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Characterization_of_synthetic_promoter_library">best solution</A> to fit our <A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Lactate_Module#Parameter_fitting">model</A> with a good ON/OFF-ratio. Going one step further, we also probed our <I>E. coli</I> cells for their reaction to the actual lactate output of mammalian cells, the <A href="#Lactate_sensing_of_cancer_cells">protocol</A> of which you can find in the section of <A href="#Bacteria_and_mammalian_cells">experiments combining mammalian cells with bacteria</A>. </P></DIV><H3>Antibody staining for Annexin V</H3><P>To find out whether or not our INP-Annexin V construct leads to proper exposition of Annexin V on the outer membrane of our <I>E. coli</I> we performed an antibody assay where we stained the strains listed below with a primary antibody against Annexin V and YFP, respectively. Using flow cytometry and confocal microscopy we would then <A href="https://2015.igem.org/Team:ETH_Zurich/Results#INP-Annexin_V_externalization">detect binding</A> of this antibody by staining with a secondary antibody (antimouse) which is coupled to Alexa488.</P><TABLE><TBODY><TR><TH><B>strain</B></TH><TH><B>expected staining?</B></TH></TR><TR><TD>INP-Annexin</TD><TD>yes</TD></TR><TR><TD>INP-Annexin + rfp</TD><TD>yes</TD></TR><TR><TD>INP-Annexin (high copy)</TD><TD>yes</TD></TR><TR><TD>Annexin</TD><TD>no</TD></TR><TR><TD>INP-YFP</TD><TD>yes</TD></TR><TR><TD>no construct</TD><TD>no</TD></TR></TBODY></TABLE><P><B>Table 1.</B> Six strains were tested. We expect staining only for the strains expressing an INP-AnnexinV or INP-YFP construct, respectively, since without INP, Annexin should not be visible on the surface of our bacteria. Negative controls therefore include cells without any construct as well as cells expressing Annexin V but lacking INP. As a positive control we used an existing BioBrick <A href="http://parts.igem.org/Part:BBa_K523013">(BBa_K523013)</A> as well as our construct expressed on a high copy plasmid. One construct includes red fluorescent protein which will make it possible to probe our cells for colocalization of RFP and Alexa488, as well as colocalization of RFP-expressing <I>E. coli</I> with CTCs.</P><DIV class="info" style="max-height:30px"><OL><LI>Dilute overnight cultures (biological triplicates) to an OD600 of 0.5 in a final volume of 20 µL.</LI><LI> Add primary antibody to a final concentration of 5 µg/mL and incubate for 2 h at 37 ºC.</LI><LI>Centrifuge at low speed for 2 min to precipitate cells. Discard supernatant to remove excess primary antibody.</LI><LI>Resuspend cells in LB medium including antibiotics and add secondary antibody to a concentration of 1 µg/mL and incubate for 1 h at 37 ºC.</LI><LI>Centrifuge at low speed for 2 min to precipitate cells. Discard supernatant to remove excess secondary antibody.</LI><LI>Resuspend cells in PBS and observe fluorescence by FACS or Confocal microscopy.</LI></OL></DIV><H3>Dynabeads for verification of Annexin V externalization</H3><P>Dnyabeads bind to any kind of antibody you expose them to. With dynabeads coated with anti-Annexin V antibody we are probing our cells for proper expression of INP-Annexin V.</P><DIV class="info" style="max-height:30px"><OL><LI>Wash beads twice in PBS.</LI><LI>Dilute stock of beads 6x and use 12.5 µL per experiment.</LI><LI>Incubate 1  µg anti-Annexin V antibody on beads for 30 min on ice.</LI><LI>Wash beads twice in PBS using a magnet.</LI><LI>Incubate beads with bacteria expressing Annexin V on their surface for 30-60 min on ice. As a control Annexin V-Alexa488 can be used.</LI><LI>Wash beads in PBS at least twice.</LI><LI>Analyze beads by flow cytometry or microscopy.</LI></OL></DIV><H3>Phosphatidylserine-coated beads for verification of Annexin V functionality</H3><P><A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Streptavidin_Beads">Magnetic beads</A> coated in Streptavidin can be coated with biotinylated phosphatidylserine to simulate an apoptotic cell. After validation of Annexin V expression by our genetically engineered bacteria we thereby test the functionality of the <A href="http://parts.igem.org/Part:BBa_K1847015" target="_blank">INP-Annexin construct</A>.</P><DIV class="info" style="max-height:30px"><OL><LI>Wash beads twice in PBS using a magnet.</LI><LI>Use 5 µL beads suspension per experiment and incubate them with biotinylated phosphatidylserine for 30 min at RT.</LI><LI>Wash beads twice in PBS and once in <A href="#Annexin_V_binding_buffer">Annexin V binding buffer</A>.</LI><LI>Incubate beads with bacteria expressing Annexin V on their surface for 30-60 min. Use 25 µL of a bacterial preculture. To support binding of Annexin V to phosphatidylserine, add 25 µL of Annexin V binding buffer. As a control, beads without PS coating, as well as bacteria without the INP-Annexin V construct can be used.</LI><LI>Wash beads in PBS three times.</LI><LI>Analyze beads by flow cytometry or microscopy.</LI></OL></DIV><DIV class="info"><H3>Western Blot</H3><OL><LI>Make overnight cultures of all samples and controls.</LI><LI>Make quick extract or emulsification of overnight cultures at OD600 = 2.0 by spinning them down and resuspending the pellet in 50 µL of 1x SDS loading buffer and boiling it for 10 min. Then centrifuge at max speed for 10 min.</LI><LI>Take 8 mL of the supernatant and load onto SDS-PAGE gel.</LI><LI>Run according to standard SDS-PAGE procedure according to gel supplier <A href="http://www.genscript.com/tech_guide/TM0210.pdf">manual</A>.</LI><LI>Equilibrate gel in transfer buffer for 15 min at RT. Pre-wet 2 fibre pads, 2 pieces of filter paper of appropriate size, and 1 piece of nitrocellulose in transfer buffer.</LI><LI>Set up transfer sandwich on the black side of the cassette, in a plastic tray, in the order filter pad, filter paper, nitrocellulose, gel, filter paper, fibre pad, black side of cassette. Add buffer between each layer and avoid air bubbles! Fold the white half of the cassette over the sandwich and close.</LI><LI>Insert cassette into the transfer apparatus in the correct orientation.</LI><LI>Insert ice block (filled with ice or with water), fill box with transfer buffer.</LI><LI>Transfer at 250 mA (or 100 V) for 45 min.</LI><LI>Disassemble the transfer apparatus. Shake the membrane in water for 5 min.</LI><LI>Block the blot in 10 ml of 5% non-fat dry milk in 1x PBS + 0.05% Tween 20 for 1 hour at RT.</LI><LI>Remove block, add primary antibody in 5 ml of block, incubate for 1 hour at RT. Typical concentrations of primary antibodies are 1:5000, 0.2-2 µg/mL.</LI><LI>Remove primary antibody and wash 3x with 10 ml PBS, 5 minutes each at RT.</LI><LI>Add secondary antibody in appropriate concentration. Incubate for 1 hour at RT.</LI><LI>Remove secondary antibody and wash four times with 10 mL PBS, 5 min each at RT.</LI><LI>Detect on LI-COR Odyssey® CLx infrared Western Blot Scanner.</LI></OL></DIV><DIV class="info"><H3>Characterization of the lldR-promoter's response to lactate</H3><OL><LI>Dilute overnight culture in fresh medium plus antibiotics by adding 40  µL of overnight culture to 2.96 mL of medium.</LI><LI>Add diluted cultures into 96-well plate.</LI><LI>Incubate for 90 to 120 min at 37ºC.</LI><LI>In the meantime prepare 300 µL of desired concentrations of lactate. Careful: prepare 20x stock!</LI><LI>Add 10 µL of the lactate solution to the samples in the 96-well plate.</LI><LI>Set up kinetic cycle in plate reader:
                       </LI><LI>Shaking for 300 s, orbital, 6 mm amplitude.</LI><LI>Measure OD600.</LI><LI>Shaking for 5 s, orbital, 6 mm amplitude.</LI><LI>Measure fluorescence at excitation at 488 and emission at 530. Set gain manually to 70.</LI></OL><LI>Run kinetic cycle for 7 hours.</LI><P>Alternatively, the reaction of bacteria to any sample of interest containing lactate can be determined in this manner. Variation of the nutrient content of the medium used for these experiments may be considered for further characterization.</P><P>Characterize the Fold Change Sensor in the same way. Over time, adaption is expected due to delayed expression of LacI.</P></DIV><DIV class="info"><H3>Characterization of the lacI-lldR-hybrid promoter's response to lactate and IPTG</H3><OL><LI>Dilute overnight culture in new medium plus antibiotics by adding 40  µL of overnight culture to 2.96 mL of medium.</LI><LI>Add diluted cultures into 96-well plate.</LI><LI>Incubate for 90 to 120 min at 37ºC.</LI><LI>In the meantime prepare 300 µL of desired concentrations of lactate and IPTG. Careful: prepare 20x stock!</LI><LI>Add 10 µL of the lactate solution to the samples in the 96-well plate, creating rising a gradient from line 1 to 11.</LI><LI>Add 10 µL of the IPTG solution to the samples in the 96-well plate, creating rising a gradient from row A to H.</LI><LI>Set up kinetic cycle in plate reader:
                       </LI><LI>Shaking for 300 s, orbital, 6 mm amplitude.</LI><LI>Measure OD600.</LI><LI>Shaking for 5 s, orbital, 6 mm amplitude.</LI><LI>Measure fluorescence at with the settings according to the expressed fluorescent protein, here GFP: excitation at 488 nm and emission at 530 nm. Set gain manually to the same value for every experiment for comparability.</LI></OL><LI>Run kinetic cycle for 7 hours.</LI></DIV><DIV class="info"><H3>AHL response experiments</H3><OL><LI>Dilute overnight culture in fresh medium plus antibiotics by adding 40  µL of overnight culture to 2.96 mL of medium.</LI><LI>Add diluted cultures into 96-well plate.</LI><LI>Incubate for 90 to 120 min at 37ºC.</LI><LI>In the meantime prepare 20x stock solutions of desired concentrations of AHL.</LI><LI>Add 10 µL of the AHL solution to the samples in the 96-well plate.</LI><LI>Set up kinetic cycle in plate reader:
                       </LI><LI>Shaking for 300 s, orbital, 6 mm amplitude.</LI><LI>Measure OD600.</LI><LI>Shaking for 5 s, orbital, 6 mm amplitude.</LI><LI>Measure fluorescence at with the settings according to the expressed fluorescent protein, here GFP: excitation at 488 nm and emission at 530 nm. Set gain manually to the same value for every experiment for comparability.</LI></OL><LI>Run kinetic cycle for 7 hours.</LI></DIV><DIV class="info"><H3>Charactreization of the AND gate between lactate and quorum sensing</H3><OL><LI>Dilute overnight culture in fresh medium plus antibiotics by adding 40  µL of overnight culture to 2.96 mL of medium.</LI><LI>Add diluted cultures into 96-well plate. Include blank samples in row 12.</LI><LI>Incubate for 90 to 120 min at 37ºC.</LI><LI>In the meantime prepare 20x stock solutions of desired concentrations of AHL (here 3OC6-HSL), along with 20x stock solutions of desired concentrations of L-lactate.</LI><LI>Add 10 µL of the lactate solution to the samples in the 96-well plate, forming a rising gradient from row 1 to row 11.</LI><LI>Add 10 µL of the AHL solution to the samples in the 96-well plate, forming a rising gradient from line A to line H.</LI><LI>Set up kinetic cycle in plate reader:
                       </LI><LI>Shaking for 300 s, orbital, 6 mm amplitude.</LI><LI>Measure OD600.</LI><LI>Shaking for 5 s, orbital, 6 mm amplitude.</LI><LI>Measure fluorescence at with the settings according to the expressed fluorescent protein, here GFP: excitation at 488 nm and emission at 530 nm. Set gain manually to the same value for every experimetn for comparability.</LI></OL><LI>Run kinetic cycle for 7 hours.</LI></DIV></DIV><DIV class="expContainer"><H2>Mammalian cell experiments</H2><DIV class="highlightBox"><H3>Overview</H3><P>The detection of CTC with our engineered MicroBeacon bacteria relies on the detection of two cancer markers, <B>susceptibility to sTRAIL</B> and <B>elevated lactate production</B> by cancer cells. To reproduce literature data on these two points we performed several experiments with several mammalian cell lines, optimizing our protocols until we reached the most reliable results. We perormed all our experiments with Jurkat cells and 3T3 cells, representing a cancerogenic and non-cancerogenic cell line, respectively. We performed our experiments also for HL60 cells, another cancerogenic cell line, and also made some experience with HeLa cells. For more clarity, in our <A href="https://2015.igem.org/Team:ETH_Zurich/Results">Results page</A> we will however focus only on Jurkat and 3T3 since the other lines did not lead to significantly different results.</P><P>Below you can find the protocols we applied for measurements of lactate output of mammalian cells and their sTRAIL susceptibility.</P></DIV><H3>Apoptosis susceptibility experiment with sTRAIL</H3><P>sTRAIL is a sinaling molecule activating the TNF pathway in cancer cells, specifically inducing apoptosis in these cells, as opposed to the case of non-cancer cells which are not susceptible to sTRAIL <A href="https://2015.igem.org/Team:ETH_Zurich/References#Srivastava2001">[Srivastava 2001]</A>. With the following experiment we are determining the <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Methods_and_results">optimal incubation time and sTRAIL concentration</A> for induction of apoptosis in our cancer cell lines of interest.</P><DIV class="info" style="max-height:30px"><OL><LI>Count cells and prepare a mastermix to have 80’000 cells or 600’000 cells per well for adherent and suspension cells, respectively. Prepare enough mastermix for all desired controls, concentrations, incubation periods, and replicates. </LI><LI>Distribute the cells into the wells of a 96-well plate, 200 µL per well.</LI><LI>Adherent cells must be allowed to settle for 2 hours previous to the start of the experiment.</LI><LI>Apply 5 µL of the desired dillutions of sTRAIL to the appropriate wells and incubate for 2 to 6 hours at 30ºC.</LI><LI>After incubation, separate cells from their medium (spin down suspension cells, remove medium from wells with adhesive cell.</LI><LI>Wash 3 times with DPBS.</LI><P>For detection of apoptosis, Annexin V-Alexa488 can be used. It will bind to externalizes phosphatidylserine on apoptotic cells.</P><LI>Resuspend cells in <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Annexin_V_binding_buffer">Annexin binding buffer</A>  and add 5 µL per sample. Incubate at RT for 15 min.</LI><LI>Dillute Annexin V-Alexa488 by addition of 50 µL of Annexin binding buffer. Alternatively, engineered <I>E. coli</I> bacteria expressing our INP-Annexin V construct along with RFP <A href="http://parts.igem.org/Part:BBa_K1847015">(BBa_K1847015)</A> can be used for detection.</LI><LI>Analyse cells by flow cytometry.</LI><LI>PI staining can be used to distinguish apoptotic from necrotic cells. Apply 5 µL of PI to the respective samples 5 min prior to analysis using flow cytometry.</LI></OL><P>Application of the appropriate positive controls (eg. Phorbol myristate acetate (PMA) for Jurkat cells) and control staining (PI for necrotic cells) will lead to more detailed and reliable results.</P><H4>Alternative induction of apoptosis using PMA</H4><P>Susceptible cell lines can be incubated with PMA in appropriate incubation period and concentrations as a positive control to sTRAIL induced apoptosis. The following protocol is designed for Jurkat cells.</P><OL><LI>Incubate cells with 5 µg/mL PMA and 1  µM Ionomycin overnight.</LI><LI>Wash cells 3 times with DPBS and resuspend in Annexin V binding buffer for detection with Annexin V-Alexa488.</LI></OL></DIV><H3>Measurement of the Lactate output of mammalian cell lines.</H3><P>Our project relies on the phenomenon of elevated lactate output of cancer cells upon metabolic reprogramming referred to as the <A href="https://2015.igem.org/Team:ETH_Zurich/Glossary#Warburg_effect">Warburg effect</A>. To validate this effect in our <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#Bacterial_strains_and_Mammalian_Cell_Lines">cell lines of interest</A> we determined the lactate level of the cell’s culture medium after cell culture within the timeframe in which our system will be applied. Based on this, the lactate production by single cells can be <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Lactate_production">estimated</A>.</P><P>The experiment can be carried out in any volume of interest. We performed our measurement in 24-well plates.</P><P>Click <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Lactate_production">here</A> to see the results of this experiment.</P><DIV class="info" style="max-height:30px"><OL><LI>Count cells and prepare a mastermix to have 80’000 cells or 600’000 cells per well for adherent and suspension cells, respectively. Prepare enough mastermix for all desired time points and replicates. Counting must be very accurate if the determination of the lactate output per cell is the aim of the experiment. Consider using a cell counter.</LI><LI>Distribute the cells into the wells of a 24-well plate, 500 µL  per well.</LI><LI>Adherent cells must be allowed to settle for 2 hours previous to the start of the experiment.</LI><LI>At the beginning of the measurement, the medium has to be exchanged to start at the minimal level. To this end, spin down the cells at 250 rcf for 3 min and resuspend the pellet in 500 µL  of fresh, prewarmed medium. The sample for time zero has to be taken right after the exchange of the medium.</LI><LI>Samples are collected in regular time intervals by transferring the medium into an Eppendorf tube. Centrifuge samples at 250 rcf for 3 min and transfer the cell free medium into a new Eppendorf tube.</LI><LI>Samples can be stored at 4 to 8 ºC for analysis at a later time point.</LI><LI>The lactate concentration of the collected samples can be determined using the <A href="Lactate_Detection_Kit">L-Lactate Kit</A> according to its instructions. In order to get good results with this kit, it is essential to use medium free of serum and phenol red.</LI></OL></DIV></DIV><DIV class="expContainer"><H2>Bacteria and mammalian cells</H2><DIV class="highlightBox"><H3>Overview</H3><P>A rather uncommon thing in mammalian cell culture is the central part of our system: the coculture with bacteria. To anticipate unexpected side effects of such a coculture in our final setup, we performed initial experiments combining mammalian cells and <I>E. coli</I>. In a next step, we tested the funcionality of our engineered cells in respect to binding to apoptotic cells. The <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Mammalian_cells_and_bacteria_combined">results</A> we achieved from these tests were obtained applying the following protocols.</P></DIV><H3>Coculture</H3><P>Since it is a rather unusual experimental setup wanting to combine mammalian cells with bacteria, we performed preliminary tests to find out if anything undesired would happen if we mixed our bacteria with mammalian cells.</P><DIV class="info" style="max-height:30px"><OL><LI>Introduce mammalian cells of interest into the wells of a 12-well plate. Adherent cells must be allowed to settle for 2 hours previous to any experiment.</LI><LI>Add 500 µL of a bacterial preculture to the mammalian cells. For analysis using microscopy it is best to use bacteria expressing a fluorophore.</LI><LI>Incubate for 4 hours. Do not incubate in a mammalian cell incubator to avoid contamination of neighbouring cultures!</LI><LI>Analyze samples using a microscope. Check for colocalization or possible internalization of bacteria by mammalian cells, as well as viability of mammalian cells with or without bacteria.</LI></OL></DIV><H3>Binding assay</H3><P>Our engineered bacteria are designed to bind to apoptotic mammalian cells. To verify this interaction we performed the following experiment.</P><DIV class="info" style="max-height:30px"><OL><LI>Apoptosis can be induced in sTRAIL susceptible cells as described <A href="#Apoptosis_susceptibilit_ experiment_with_sTRAIL">elsewhere</A> using the optimal incubation time with sTRAIL using the optimal concentration of this agent.</LI><LI>Detection of apoptotic cells can be performed by applying 50 µL of INP-Annexin V and RFP expressing <I>E. coli</I> at an OD600 between 0.6 and 0.8 to the samples followed by 30 min incubation.</LI><LI>Analyse samples by flow cytometry.</LI></OL><P>Variations of the conditions, for example incubation temperature or Calcium content of the buffer were adjusted to find the best setup. We found that incubation on ice at a Calcium concentration of 2.5 mM is optimal.</P></DIV></DIV><DIV class="expContainer"><H2>Chip</H2><DIV class="highlightBox"><H3>Overview</H3><P>The final setup of our CTC-detection system will be a water in oil emulsion within a microfluidic chip. To approximate this environment in the limited amount of time we had, we invested a lot of thoughts into our <A href="https://2015.igem.org/Team:ETH_Zurich/Chip">Chip design</A>. We were able to test some components of our final system in the nanowell plate that resulted from this design process. We used our test setup to investigate <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Increased_sensitivity_for_single_cell_lactate_output">lactate detection</A> by our bacteria in a <B>single cell analysis</B> setup.</P></DIV><H3>Fabrication of the chip</H3><P>  The construction of a microfluidic chip relies on three steps: </P><OL><LI>Designing a transparency mask of the chip architecture. </LI><LI> Generating a negative mold of the chip on a Si-wafer using a negative photoresist and standard photolitography technique. </LI><LI> Cast PDMS over the produced mold to get a positive image of the chip. </LI></OL><P> Many thanks to the D-BSSE Clean Room facility for making the mold, and the Tay group PDMS room facility for making the chip for us.</P><DIV class="info" style="max-height:80px"><OL><LI> Draw the design on Autocad software.</LI><LI>Order the transparency mask from <A href="http://www.outputcity.com/"> CAD/Art Services, Inc</A></LI><LI> Use a negative photoresist (SU-8-3025 ) for mold making</LI><LI> Non-stick functionalize mold: first, place mold in closed wafer carrier box with a beaker containing a few drops of TMCS. Incubate for 15-60 minutes.</LI><LI> Prepare the PDMS: Mix 10:1 PDMS: Weigh in 66g base PDMS + 6.6g curing agent. Then mix and defoam PDMS in Thinky mixer (3 min at 2000 RPM mix + 3 min at 2200 RPM defoam).</LI><LI> Cast chips: Pour 30 g of the mixed PDMS over wafer in a glass petri dish, degas wafers in excitator for 60 minutes and place in oven on level surfaces. Cure 45 minutes at 80 °C</LI><LI>Finalize chips: Carefully chips from wafers and cut the chips into the desired shape. </LI></OL></DIV><H3>Preparation of the chip for the experiments</H3><DIV class="info" style="max-height:80px"><OL><LI> Clean chip with adhesive tape to remove  all dust particles </LI><LI> Plasma treat chip for 50 s at 40% power in plasma machine (Diener FEMTO).</LI><LI> Submerse chip in double distilled H<SUB> 2 </SUB>O </LI><LI> Incubate the chip in Fibronectin solution (0.1 mg/mL) for at least one hour. </LI><LI> Wash chip with PBS (2 times). </LI><LI> Seed cells as explained in Loading of Mammalian cells and bacteria. </LI><LI> After chip usage, clean by incubation in 1M NaOH, no more than 5 minutes </LI></OL></DIV><H3>Loading of Mammalian Cells and bacteria</H3><P>To allow for single cell analysis, we need to reach the stage where most wells in our chip contain only one mammalian cell. We tried several techniques of administration of mammalian cells and bacteria to our chip. The optimal protocol proved to be:</P><DIV class="info" style="max-height:80px"><OL><LI>Dilute the mammalian cells to a concentration of 15 cells/µL. </LI><LI>Load 3 mL of the cell suspension into the 6-well plate containing the chip. </LI><LI>Add bacteria to have a concentration of 15 cells/nL into the chip, after measuring the OD thanks to<A href="http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp">an OD600-bacterial density converter</A>. </LI><LI>Mix the solution with a pipette carefully. </LI><LI>Let the cells adhere for 15 minutes. </LI><LI>Optionally, seal the chip with a coverslip. </LI></OL></DIV><H3>Lactate sensing of cancer cells</H3><P>As a prove that the lactate levels produced by cancer cells are enough to trigger GFP expression from our lldR-promoters, we introduced our bacteria into a nanowell plate, along with Jurkat cells. Fluorescence and  bacterial cell count were monitored using ImageJ and lead to data presented <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Lactate_detection">here</A>.</P><DIV class="info" style="max-height:30px"><OL><LI>Dilute the mammalian cells to a concentration of 15 cells/µL. </LI><LI>Load 3 mL of the cell suspension into the 6-well plate containing the chip. </LI><LI>Add bacteria to have a concentration of 15 cells/nL into the chip . </LI><LI>Mix the solution with a pipette carefully. </LI><LI>Let the cells adhere for 30 minutes. </LI><LI>Observe chip for 6 h using the we used an inverted epi-fluorescence microscope (Nikon Ti-Eclipse) with magnification 20X. We imaged the bright field and gfp fluorescence every 15 minutes. </LI></OL></DIV></DIV><DIV class="expContainer"><H2>General useful protocols</H2><DIV class="highlightBox"><H3>Overview</H3><P>A substantial amout of our time in the lab was of course also spent on tasks like cloning, PCRs, transfromations, and many other things. The protocols we applied for these general duties can be found here.</P></DIV><DIV class="info"><H3>Cryostock of bacteria</H3><OL><LI>Add 1 mL of 40% glycerol to a cryogenic vial.</LI><LI>Add 1 mL sample from the bacterial culture.</LI><LI>Vortex the vial.</LI><LI>Store in -80ºC freezer.</LI></OL></DIV><DIV class="info"><H3>Preparation of chemically competent cells</H3><OL><LI>Take a trace of <I>Escherichia coli</I> from a glycerol stock vial and put it on a LB plate with the correspondent antibiotic. </LI><LI>Incubate overnight at 37ºC.</LI><LI>Pick a colony and inoculate 10 mL LB medium containing the relevant antibiotic. Grow overnight at 37ºC.</LI><LI>Add 1mL overnight culture to 100mL prewarmed LB medium containing the relevant antibiotic in a 500mL flask, and shake at 37ºC until an OD<SUB>600</SUB> of 0.5 is reached (usually 90-120 min).</LI><LI>Cool the culture on ice 5min, and transfer the culture to a sterile, round-bottom centrifuge tube.</LI><LI>Collect the cells by centrifugation at low speed (5 min, 4000 xg, 4ºC).</LI><LI>Resuspend the cells gently in cold (4ºC) <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#TFB1">TFB1 buffer</A> (30 mL for 100 mL culture) and keep the suspension on ice for 90min.</LI><LI>Collect the cells by centrifugation (5min, 4000 xg, 4ºC).</LI><LI>Discard the supernatant carefully. Always keep the cells on ice.</LI><LI>Resuspend the cells carefully in 4 mL ice-cold <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#TFB2">TFB2 buffer</A>.</LI><LI>Prepare aliquots of 100-200µL in sterile microcentrifuge tubes and freeze in dry-ice-ethanol mix. Store the competent cells at -80ºC</LI></OL></DIV><DIV class="info"><H3>Transformation of chemically competent cells</H3><OL><LI>Take 1 µL DNA to be transformed and put into a cold sterile 1.5 mL microcentrifuge tube, and keep on ice. </LI><LI>Thaw an aliquot of frozen competent cells on ice. </LI><LI>Resuspend the cells and transfer 50  of the cell suspension into the microcentrifuge tube with the plasmid DNA, mix carefullu, and keep on ice for 20 min. </LI><LI>Transfer the tube to a 42ºC heat block for 90 s.</LI><LI>Add 500 µL <A href="https://2015.igem.org/Team:ETH_Zurich/Materials#SOC_medium">SOC medium</A> to the cells and incubate for 60-90 min at 37ºC. Shaking increases transformation efficiency.</LI><LI>Plate 100 µL on LB-agar plates containing antibiotics. Incubate the plates overnight at 37ºC.</LI></OL></DIV><DIV class="info"><H3>Electrocompetent cells</H3><OL><LI>Dillute overnight cultures and let them grow to an OD600 of 0.6 to 0.8. Use approximately 2 mL of culture per transformation.</LI><LI>Spin down the cells at 1600 rcf for 10 min and remove the supernatant.</LI><LI>Wash cells three times in 1 mL ice cold water to remove all residual salts of the LB medium.</LI><LI>Resuspend cells in 50 µL ice cold water and add 1 µL of DNA.</LI><LI>Pipett the cells into a clean cuvette and apply an electric pulse. Check the pulse time. If it is above 5 ms you are fine.</LI><LI>Immediately resuspend the cells in 1 mL fresh medium and transfer them to an eppi. Incubate at 37ºC for 1 h.</LI><LI>Plate 100 µL and 300 µL onto LB plates containing the appropriate antibiotics.</LI></OL></DIV><DIV class="info"><H3> Digest with restriction enzymes </H3><OL><LI>Produce the following master mix:</LI><OL><LI>Water up to 50µL</LI><LI>1µg of DNA</LI><LI>5µL CutSmart buffer (bought in BioLabs)</LI><LI>1µL of each restriction enzyme</LI></OL><LI>Incubate 1 hour at 37ºC.</LI><LI>Inactivate the enzyme with the correspondent temperature during 20 minutes.  </LI><LI>Purify the product using an agarose gel.</LI></OL></DIV><DIV class="info"><H3>Ligation </H3><OL><LI>Produce the following master mix on ice:</LI><OL><LI>Water up to 20µL</LI><LI>1 equivalent of vector DNA</LI><LI>3 equivalents of insert DNA</LI><LI>2µL T4 DNA ligase buffer (NEB)</LI></OL><LI>Incubate overnight at 16ºC or 1 h at RT. </LI><LI>Heat inactivate at 65ºC for 10 minutes. </LI><LI>Put on ice and transform. </LI></OL></DIV><DIV class="info"><H3>Gibson Assembly</H3><OL><LI>Calculate the concentration needed of the correspondent fragments:</LI><OL><LI><I>For 2 fragments</I>: 2-3x molar excess of the insert. </LI><LI><I>For 2-3 fragments</I>: total molarity 0.02-0.5 pmol.</LI><LI><I>For 4-6 fragments</I>: total molarity 0.2-1 pmol.</LI></OL><LI>Put 5 µL of DNA with 15 µL of Gibson Assembly mix. 6mL of this mix consist in:</LI><OL><LI>3 mL of 1M Tris-HCl pH 7.5</LI><LI>150 µ of 2 M MgCl<SUB>2</SUB></LI><LI>60 µl of 100 mM dGTP</LI><LI>60 µl of 100 mM dATP</LI><LI>60 µl of 100 mM dTTP</LI><LI>60 µl of 100 mM dCTP</LI><LI>300 µL of 1M DTT</LI><LI>1.5 g PEG-800</LI><LI>300 µl of 100 mM NAD</LI></OL><LI>Incubate at 50ºC for 60 minutes.</LI><LI>Transform the correspondent microorganism. </LI></OL></DIV><DIV class="info"><H3>Colony PCR </H3><OL><LI>Take 20 µL of LB and put one colony inside. </LI><LI>Prepare master mix for PCR containing: </LI><OL><LI>Water up to 25µL</LI><LI>1µL of resuspended cells</LI><LI>10µL ThermoPol buffer (bought in BioLabs)</LI><LI>10 mM dNTPs</LI><LI>10 µM forward primer</LI><LI>10 µM reverse primer</LI><LI>0.125 µL of <I>taq</I> DNA polymerase</LI></OL><LI>Use the thermocycler with the following conditions:</LI><OL><LI>Initial denaturation at 98ºC for 30 seconds.</LI><LI>35 cycles, with a denaturation at 98ºC for 10 seconds; primer hybridization temperature depending on the primer for 25 seconds; and an elongation of the sequence at 72ºC for 30 seconds per each kb. </LI><LI>Final extension at 72º for 10 minutes.</LI></OL></OL></DIV><DIV class="info"><H3>Phusion PCR </H3><OL><LI>Prepare master mix for PCR containing: </LI><OL><LI>Water up to 50 µL</LI><LI>1 µL of resuspended cells</LI><LI>10 µL Phusion HF buffer (bought in BioLabs)</LI><LI>10 mM dNTPs</LI><LI>10 µM forward primer</LI><LI>10 µM reverse primer</LI><LI>0.5 µL of phusion DNA polymerase</LI></OL><LI>Use the thermocycler with the following conditions:</LI><OL><LI>Initial denaturation at 98ºC for 30 seconds.</LI><LI>35 cycles, with a denaturation at 98ºC for 10 seconds; primer hybridization temperature depending on the primer for 25 seconds; and an elongation of the sequence at 72ºC for 30 seconds per each kb. </LI><LI>Final extension at 72º for 10 minutes</LI></OL></OL></DIV><H3>Spheroblast Protocol</H3><P>In order to bind to their target cells, the bacteria designed by the iGEM team of <A href="https://2015.igem.org/Team:Stockholm">Stockholm</A> have to be turned into Spheroblast, since their construct is located in the inner membrane.</P><DIV class="info" style="max-height:30px"><OL><LI>5 mL overnight culture in Terrific Broth (TB) 2% glucose</LI><LI>7 mL culture in 100 mL Erlenmeyer flast (1/100 dilluted overnight culture) for 2 h at 37ºC</LI><LI>Cool down samples 20 min at RT</LI><LI>Spin down at 14'000 rpm for 2 min</LI><LI>Wash 2x with 1 mL of Tris-HCL (pH 8.0)</LI><LI>Resuspend cells in 1 mL od STE</LI><LI>Incubate on roto-mixer for 30 min at 37ºC</LI><LI>Spin down at 14'000 rpm for 1 min</LI><LI>Wash with 1 mL of freshly made Solution A. Centrifuge at 14'000 rpm for 1 min</LI><LI>Prepare lysozyme solution (50 mg/mL) with cold water</LI><LI>Resuspend in 1 mL freshly made Solution A and add 20 µL of lysozyme solution</LI><LI>Rota-incubate for 15 min at 37ºC</LI><LI>Centrifuge at 14'000 for 1 min</LI><LI>Resuspend pellet in 1 mL PBS</LI></OL>	
   

</DIV><H3>Mammalian cell culture: splitting cells</H3><P>Depending on the preferences of certain cell lines, split cells at different levels of confluency as described in the following.</P><DIV class="info" style="max-height:30px"><H4>Adherent cell lines</H4><OL><LI>Remove the medium by aspirating it using the vacuum pump.</LI><LI>Wash cells with 10 mL of DPBS to remove traces of medium.</LI><LI>Remove the DPBS by aspirating it using the vacuum pump.</LI><LI>Incubate the cells for 2-5 min with Accutase or Trypsin. Use one or two mL for a 25 cm<SUP>2</SUP> flask and 75 cm<SUP>2</SUP> flask, respectively.</LI><LI>Check if the cells are detached after the incubation using the microscope or checking by eye. If they are detached resuspend the cells in approximately 7mL of fresh prewarmed medium to neutralize Trypsin or Accutase. Transfer the resuspended cells into a 15 mL falkon tube.</LI><LI>Spin down the cells for 3 min at 250 rcf.</LI><LI>Remove the medium by aspirating it using the vacuum pump.</LI><LI>Resuspend the cells in 3 to 5 mL depending on size of the pellet.</LI><LI>Count cells using a Neubauer chamber.</LI><LI>Prepare new cultures in 5 or 10 mL in 25 or 75 cm<SUP>2</SUP> flasks, respectively. Seed 10'000 or 5'000 cells per cm<SUP>2</SUP> to reach confluency after 2 or 3 days, respectively.</LI></OL><H4>Suspension cells</H4><OL><LI>Transfer the whole volume of the culture into a 15 mL falkon tube.</LI><LI>Spin down the cells for 3 min at 250 rcf.</LI><LI>Remove the medium by aspirating it using the vacuum pump.</LI><LI>Resuspend the cells in 3 to 5 mL depending on size of the pellet.</LI><LI>Count cells using a Neubauer chamber.</LI><LI>Prepare new cultures in 5 or 10 mL in 25 or 75 cm<SUP>2</SUP> flasks, respectively. Seed 100'000 cells per mL to reach confluency after 2.</LI></OL></DIV></DIV></DIV><DIV id="footerContainer"><H2>We would like to thank our sponsors</H2></DIV></DIV></DIV></DIV></DIV></BODY></HTML>