<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NCTU_Formosa_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NCTU_Formosa/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NCTU_Formosa/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NCTU_Formosa/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NCTU_Formosa/project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NCTU_Formosa/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NCTU Formosa/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-wrapper"><DIV id="header"><DIV id="title"><A href="https://2013.igem.org/Team:NCTU_Formosa"><P>NCTU_Formosa - E.colightuner</P></A></DIV><DIV id="igemlogo"><A href="https://2013.igem.org/"><P>iGEM 2013</P></A></DIV><NAV id="nav" class="navwrapper"><BUTTON id="navb" class="dl-trigger">Open Menu</BUTTON><UL class="nav"><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project">Project</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Introduction">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Light-regulated_system">Light-regulated system</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Temperature-regulated_system">Temperature-regulated system</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Small_RNA-regulated_system">Small RNA-regulated system</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Mechanism_of_our_project">Mechanism of our project</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#i-Culture">i-Culture</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/project#Future_Works">Future Works</A></LI></UL><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/results">Results</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/results#Light-regulated_System">Light-regulated System</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/results#Temperature-regulated_System">Temperature-regulated System</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/results#Small_RNA-regulated_System">Small RNA-regulated System</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/modeling#MATLAB_Introduction">MATLAB Introduction</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/modeling#ANFIS_Introduction">ANFIS Introduction</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/modeling#Single_Unit">Single Unit</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/modeling#E.colightuner_Simulation">E.colightuner Simulation</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/biobricks">Biobricks</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/notes">Notes</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/notes#Material_and_Method">Material and Method</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/notes#March_2013">Detailed records</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice#i-Culture_Popularization">i-Culture Popularization</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice#Conference">Conference</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice#Biocamp">Biocamp</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice#iGEM_Chinese_Guiding_Book">iGEM Chinese Guiding Book</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/human_practice#Surveys">Surveys</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/team">Team</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/team#Attribution">Attribution</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/team#Members_and_Advisers">Members and Advisers</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:NCTU_Formosa/achievement">Achievements</A></LI><LI style="display:none"><A href="https://2013.igem.org/Team:NCTU_Formosa">Home</A></LI></NAV><UL id="back-top"><LI><A href="#top">Top</A></LI></UL></DIV></DIV><DIV id="main-wrapper"><DIV id="main"><DIV id="cover"><DIV id="coverbox"><DIV id="ctitle">Project</DIV><P>A multiple regulated-system was built using three different regulation mechanisms including red light, temperature, and sRNA. In other words, it is multitasking genetic engineered machine that can express a variable genes depending on the different command given.
</P></DIV></DIV><DIV id="changeFont"><SPAN class="changeFontText">Change the font size right here</SPAN></DIV><DIV id="content-wrapper"><DIV id="realcontent"><DIV id="crw"><DIV id="cr"><DIV class="cont"><DIV class="li"><DIV class="card"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Light-regulated_system"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Light-regulated system</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Introduction_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Design_of_Red_promoter"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design of Red promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Reference"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Temperature-regulated_system"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Temperature-regulated system</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Introduction_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Mechanism"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Mechanism</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Design"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Reference_2"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">Reference</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Small_RNA-regulated_system"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Small RNA-regulated system</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Introduction_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Mechanism_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Mechanism</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Design_2"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Reference_3"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Reference</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Mechanism_of_our_project"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Mechanism of our project</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Introduction_5"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Red_Light_Induced_Circuit"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Red Light Induced Circuit</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Dark_Response_Circuit"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Dark Response Circuit</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Summary"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#i-Culture"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">i-Culture</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Future_Works"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Future Works</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Application"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Application</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#The_next_BIG_thing"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">The next BIG thing</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Reference_4"><SPAN class="tocnumber">7.2.1</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>The majority of the previous iGEM projects focused on expressing certain genes to achieve a specific task such as sensing a certain phenomenon. This year, we took a step forward in exploring the potential of <I>Escherichia coli</I>, thus building a system that is capable of <B>regulating multiple genes</B>. The system, so-called E.colightuner, is built by three regulated-systems including <B>light regulated-system</B>, <B>temperature regulated-system</B>, and <B>sRNA regulated-system</B>. We used noninvasive factors such as red light and temperature to create different conditions under which <I>E. coli</I> can express different genes. We also introduced a new sRNA regulated-system that hasn't been widely applied in iGEM before. </P><P>E.colightuner is just like a machine that can process different tasks depending on the commands given. In this case, the gene expression of E.colightuner is the task and its environmental condition is the command. By changing the environmental condition, E.colightuner can express different genes. The advantage of building such a machine is to increase the convenience in expressing multiple genes. Instead of expressing different genes with different bacteria, we can express the genes with E.colightuner. This may ultimately lower the cost in expressing different genes.</P><P>We tend to apply E.colightuner to agriculture. By expressing different hormones required by different stages of plant growth, E.colightuner can aid plant growth. With that said, we built a device that can flower the desired hormones to plants. This device can be applied to any plantation, and it is simple to manipulate. With more efforts, we believe that E.colightuner can raise the potential values of <I>E.coli</I>.</P></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Light-regulated_system">Light-regulated system</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H4><P>Light is an easily accessible energy that can be applied to various fields. Of course, it can be applied to synthetic biology as well. Bacteria can sense light of a defined wavelength through a small light-absorbing ligand called chromophore. By employing this chromophore, light can be used to trigger a certain reaction and even regulate gene expressions.
</P><P>In E.colightuner, light serves as an activator that can regulate the function of a promoter. Since red light can be easily turned on and off, the activation and repression of the promoter can also be easily achieved. Substrate activators, on the other hand, are harder to manipulate. Substrates can activate their corresponding promoters efficiently, but they might be hard to remove, making it difficult to repress the promoters when desired. 
</P><P>In order to make <I>E. coli,</I> sense and respond to light, we installed a <B>&quot;two component system&quot; (cph8)</B> that can respond to <B>red light</B>. </P><P>The chimera cph8 is consisted of a phytochrome cph1 and a histidine kinase domain, envz-ompR, that includes a response regulator. cph1 is the first member of the plant photoreceptor family that has been identified in bacteria. 
The functional expression of a phytochrome domain (cph1) in <I>E. coli</I> requires the biosynthesis of the respective bilin chromophore <B>PCB</B>. Two phycocyanobilin(PCB)-biosynthesis genes (ho1<SUP>1</SUP> and pcya<SUP>2</SUP>) from Synechocystis encoding a heme oxygenase and bilin reductase, respectively, is essential to convert heme into PCB.  Figure 2 depicts the phycocyanobilin-biosynthesis pathway.<SUP>3</SUP> With the biosynthesis of PCB, cph1 serve as a red-light absorbing chromophore that is inactivated under red light and activated without red light . <SUP>4,5</SUP></P><P>envZ-ompR, a dimeric osmosensor, is a multidomain transmembrane protein and one of the best characterized two-component histidine kinases from <I>E.coli</I>. Besides the periplasmic receptor domain that is flanked by two transmembrane helices, it possesses a C-terminal 228-residue histidine kinase domain that is located in the cytoplasm<SUP>6</SUP>.
Upon changes of extracellular osmolarity, envZ specifically phosphorylates its cognate response regulator ompR, which, in turn, regulates the <B>P<SUB>ompc</SUB></B><SUP>7</SUP>.</P><P>With the two components just mentioned, cph8 can serve as a photoreceptor that regulates gene expression through P<SUB>ompc</SUB>. Without red light, cph1 is activated and it enables envZ-ompR to autophosphorylate which in turn activates P<SUB>ompc</SUB>. Under the exposure of red light, however, cph1 is deactivated, inhibiting the autophosphorylation, thus turning off gene expression. This mechanism is depicted in Figure 3<SUP>8</SUP> and the experimental data that support this mechanism is shown in Figure 4.</P><H4><SPAN class="mw-headline" id="Design_of_Red_promoter">Design of Red promoter</SPAN></H4><P>We designed a new biobrick, P<SUB>red</SUB>, to <B>positively</B> regulate gene expression using red light. Compared to the negative regulation provided by cph8, positive regulation is more direct and convenient to operate. In addition, cph8 relies on P<SUB>ompc</SUB> to express downstream gene, but our design relies on a stronger promoter, P<SUB>lac</SUB>, that can amplify gene expression, further making sure that the level of expression can meet our needs.
</P><P>In the presence of red light, P<SUB>ompc</SUB> is shut down and lacI cannot be expressed to repress P<SUB>lac</SUB>. As a result, P<SUB>lac</SUB> is activated and all the downstream genes can be transcribed. In the absence of red light, whatsoever, P<SUB>ompc</SUB> is activated as autophosphorylation is allowed. LacI can be expressed to repress P<SUB>lac</SUB> and the genes downstream cannot be expressed, consequently. By using P<SUB>red</SUB>, we can achieve positive control using red light.
</P><H6><SPAN class="mw-headline" id="Reference">Reference</SPAN></H6><DIV class="ref"><OL start="1"><LI>part BBa_I15008;MIT Registry of Standard Biological Parts</LI><LI>part BBa_I15009;MIT Registry of Standard Biological Parts</LI><LI>Levskaya, A. et al .(2005). Engineering Escherichia coli to see light. Nature, 438(7067), 442.</LI><LI>Kehoe DM, Grossman AR (1996) Similarity of a chromatic adaptation sensor to phytochrome and ethylene receptors. Science 273(5280):1409–1412</LI><LI>Yeh KC, Wu SH, Murphy JT, Lagarias JC (1997) A cyanobacterial phytochrome two-component light sensory system. cience 277 (5331):1505–1508</LI><LI>Dutta R, Qin L, Inouye M (1999) Histidine kinases: diversity of domain organization. Mol Microbiol 34(4):633–640</LI><LI>Forst SA, Roberts DL (1994) Signal transduction by the EnvZ–OmpR phosphotransfer system in bacteria. Res Microbiol 45(5–6):363–373</LI><LI>Thomas Drepper, Ulrich Krauss,Sonja Meyer zu Berstenhorst, Jörg Pietruszka, Karl-Erich Jaeger.(2011).Lights on and action! Controlling microbial gene expression by light. Appl Microbiol Biotechnol, 90:23–40 DOI:10.1007/s00253-011-3141-6</LI></OL></DIV></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Temperature-regulated_system">Temperature-regulated system</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_3">Introduction</SPAN></H4><P>It is not a dream anymore that &quot;making the impossible possible&quot; with the increase of modern technology. It is also a serious problem that what human has brought to Earth  along with our progressing civilization. Drastic environmental changes are inevitable due to the fact of massive developing industries. Temperature changing are one of the problems we need to confront.
</P><P>Cells are constantly subjected to changing environmental conditions. Comparing with chemical substances, temperature is more accessible, convenient and lower costing. Moreover, temperature is also less burden to the organisms. Therefore, few organisms use temperature as a regulating factor. A mechanism found in different organisms that makes the cell respond to thermal changes, for example, is the <B>RNA thermometer</B>.
</P><P>An RNA thermometer is a temperature-sensitive non-coding RNA molecule which regulates gene expression. The expression of heat-shock, cold-shock and some virulence genes are coordinated in response to temperature changes. There are several systems suggested in references that are based on RNA secondary structure. This structural transition can then expose or occlude important regions of RNA such as a <B>ribosome binding site</B>, which then affects the translation rate of a nearby protein-coding gene, so we can say that the RNA thermometers are just like riboswitches.
</P><P>Apart from protein-mediated transcriptional control mechanisms, translational control by RNA thermometers is also a widely used regulatory strategy.
</P><H4><SPAN class="mw-headline" id="Mechanism">Mechanism</SPAN></H4><P>RNA thermometers are identified in the 5' UTR of messenger RNA, upstream of a protein-coding gene. 
</P><P>Unlike normal RBS, the <B>37 °C RBS</B> has a unique or rather stable <B>hairpin</B> structure which shaped like a lock. If the temperature drops below 37 °C, the base-pairing neucleotides on the RNA will form a stable secondary hairpin structure, occluding the <B>Shine-Dalgarno sequence</B> (SD sequence) thus disabling the ribosome to bind. The base-pairing of this RNA region will block the expression of the protein encoded within. Since the structure is sustained by base pairing, heat can be employed to break the hydrogen bonds. So when it reaches 37 °C, the bonds would be broken and the hairpin would be unfolded, causing the SD sequence to be exposed and permitting the binding of the ribosome, which then starts the translation. This means that by raising temperature to a certain threshold, the 37 °C RBS can function as a normal RBS. In this way, gene expression can be regulated on the RNA level by temperature.
</P><H4><SPAN class="mw-headline" id="Design">Design</SPAN></H4><P>The specific ribosome binding site (RBS) <A href="http://parts.igem.org/Part:BBa_K115002" class="external text" rel="nofollow">BBa_ K115002</A> is a great RNA thermometer. It has high translation activity at high temperature (&gt; 37 °C) and low translation activity at room temperature. It was designed for the temperature-dependent genetic circuit in <I>E. coli</I>, with a green fluorescent protein (GFP) used as the reporter protein.
</P><P>Originally, it is in the 5'-UTR of the Salmonella <I>agsA</I> gene, which codes for a small heat shock protein that can be used for temperature sensitive post-transcriptional regulation, initiating translation at 37 °C. Transcription of the <I>agsA</I> gene is controlled by the alternative sigma factor σ32. Additional translational control depends on a stretch of four uridines that pair with the SD sequence, so they are named &quot;FourU&quot;. They repress translation of this protein by base-pairing with the Shine-Dalgardo sequence of the mRNA. This prevents ribosomes from binding the start codon.
</P><P>In our project, we use the 37 °C RBS in both red light induced circuit and dark response circuit and it is related to the production of GFP and BFP. We use this  37 °C RBS to design our project because we want to use different conditions to produce different substances. Therefore, the characteristic and sensitivity of 37 °C RBS perfectly fits our purpose which we want to regulate our system through temperature.
</P><H6><SPAN class="mw-headline" id="Reference_2">Reference</SPAN></H6><DIV class="ref"><OL start="9"><LI>Torsten Waldminghaus, Nadja Heidrich, Sabine Brantl and Franz Narberhaus .(2007). FourU: a novel type of RNA thermometer in Salmonella . Molecular Microbiology , 65(2): 413–424 DOI:10.1111/j.1365-2958.2007.05794.x</LI><LI>part BBa_K115002;TUDelft Registry of Standard Biological Parts</LI></OL></DIV></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Small_RNA-regulated_system">Small RNA-regulated system</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_4">Introduction</SPAN></H4><P>Base pairing offers a powerful way for one RNA to control the activity of another. Both prokaryotes and eukaryotes have many cases which a single-stranded RNA base pairs with a complementary region of an mRNA. As a result, it prevents expression of the mRNA.</P><P><B>RNA interference (RNAi)</B>, also called post transcriptional gene silencing, is a process in which RNA molecules inhibit gene expression by destroying specific mRNA molecules. In 2006, Andrew Fire and Craig C.Mello shared the Nobel Prize in Physiology or Medicine because of their study on RNA interference<SUP>11</SUP>. This powerful gene silencing tool in eukaryotes has been used in many research. As their counterparts in bacteria, <B>small non-coding RNAs (sRNAs)</B> are important regulatory roles.</P><P>Small RNAs (sRNAs) have become increasingly significant in playing the role of bacterial gene regulation. Most sRNAs interact with the targeted mRNAs by imperfect base pairing, reducing the translation efficiency and recruiting chaperones such as Hfq for translation termination. The sRNAs would bind to the target with its hairpin-like structure which wound by some of it's own sequences, and the chaperons would stuck between the hairpins in order to protect the sRNA-mRNA combination from degrading. In the end, the sRNAs regulate gene expression by forestalling translation. Since this regulated-system has already been employed in vivo, it is necessary to design an artificial sRNA that targets specifically to a desired genes, so that we can prevent the sRNA from affecting other undesired genes.</P><P>Unlike the regulated-system popular used in iGEM projects (e.g. P<SUB>tet</SUB>&amp;<I>tetR</I>, P<SUB>lux</SUB>&amp;<I>LuxR</I>, etc.), sRNA regulated-system seems <B>more efficient</B>. Because the inhibition works under RNA level, which means the energy wasted in <I>E. coli</I> is less than other system (producing proteins to regulate).</P><P>Figure 8 depicts how the small RNA chaperone, Hfq, associates with the small RNA and represses a target mRNA.<SUP>12</SUP></P><H4><SPAN class="mw-headline" id="Mechanism_2">Mechanism</SPAN></H4><P>How do small RNAs regulate gene expression? 
Figure 9 shows the mechanism of repressing one target mRNA.
The small RNA has three stem-loop double stranded RNA structures, 
and the loop which is closest to the 3’ terminus is complementary 
to the sequence preceding the initiation codon of the target mRNA.
Base pairing between small RNA and the target mRNA prevents the ribosome 
from binding to the initiation codon, so the translation would be repressed.<SUP>13</SUP><SUP>,14</SUP></P><H4><SPAN class="mw-headline" id="Design_2">Design</SPAN></H4><P>The following is the process of our idea in designing the sRNA.
</P><P>We have designed two artificial sRNAs, each modified from different paper. The sRNA-1 is based on afsRNA ARlacZ1, which is created by Hongmarn Park and his colleagues.<SUP>15</SUP><I>SibC</I> is a small RNA sequence which is already found in <I>E.coli</I>. Hongmarn Park and his colleagues changed few base pairs of <I>SibC</I> to make ARlacZ1, purposed to make the secondary structure more stable. After that, they tried to change 9 different locations of the target-recognition sequence to test the effect on gene silencing. Our design is to modify from the RNA sequence which has the best efficiency : ARlacZ7. In accordance with afsRNA ARlacZ7, we changed the target-recognition sequence to finally complete our sRNA-1.
</P><P>The other sRNA sequence we produced is called sRNA-2. It is employed in this project selected from a library of artificial sRNA that was constructed by fusing a randomized antisense domain of Spot42, the scaffold that is known for recruiting the RNA chaperons. The sRNA we picked contains a consensus sequence, 5’-CCCUC-3’, which can base pair with the SD sequence due to complementary binding. This sRNA, as expected, effectively regulates gene expression by reducing translation efficiency and recruiting Hfq. In addition, this sRNA shows high specificity against its targeted gene, <I>OmpF</I>, as it doesn't hold significant activity against other genes from <I>E. coli</I> genome (Sharma and others, 2011). 
</P><P>The sRNA picked is competent, but we hoped that it can target any desired gene. Therefore, we designed a RBS by employing the sRNA targeting region from <I>OmpF</I> and making the AUG codon sufficiently apart from the SD sequence for ribosome binding. By adding this RBS to the upstream of any desired gene, the gene can be regulated by sRNA.
</P><H6><SPAN class="mw-headline" id="Reference_3">Reference</SPAN></H6><DIV class="ref"><OL start="11"><LI>Xu, S.; Montgomery, M.; Kostas, S.; Driver, S.; Mello, C. (1998). &quot;Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans&quot;. Nature 391 (6669): 806–811 DOI:10.1038/35888</LI><LI>Jörg Vogel , Ben F. Luisi.(2011). Hfq and its constellation of RNA. Nature Reviews Microbiology, 9:578-589</LI><LI>E.K. Jocelyn, S.G. Elliott , T.K. Stephen, &quot;Lewin's Genes X.-10th ed.&quot;, Jones &amp; Bartlett, Sudbury, MA, 2011.</LI><LI>Karen M. Wassarman.(2002). &quot;Small RNAs in Bacteria: Diverse Regulators of Gene Expression in Response to Environmental Changes&quot;. Cell, 109:141–144</LI><LI>Hongmarn Park, Geunu Bak, Sun Chang Kim &amp; Younghoon Lee.(2013). &quot;Exploring sRNA-mediated gene silencing mechanisms using artificial small RNAs derived from a natural RNA scaffold in <I>Escherichia coli</I> &quot;. Nucleic Acids Research,Vol. 41, No. 6, 3787-3804 DOI:10.1093</LI><LI>Vandana Sharma, Asami Yamamura &amp; Yohei Yokobayashi.(2011). &quot;Engineering Artificial Small RNAs for Conditional Gene Silencing in E. coli&quot;. ACS Synthetic Biology</LI></OL></DIV></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Mechanism_of_our_project">Mechanism of our project</SPAN></H3><H4><SPAN class="mw-headline" id="Introduction_5">Introduction</SPAN></H4><P>This is the pathway we designed by using the three different regulating factors. We tend to set up four conditions for the <I>E.coli</I> to work by paring the light and the temperature controls. Distinguished by whether red light exists or not, the pathway is divided into two parts, the <B>Red Light Induced Circuit</B> and the <B>Dark Response Circuit</B>.  We use four different fluorescent proteins as reporter genes to test that whether the switch of our pathway is successful.
</P><H4><SPAN class="mw-headline" id="Red_Light_Induced_Circuit">Red Light Induced Circuit</SPAN></H4><P>At 25 °C and under red light, P<SUB>red</SUB> is activated for translation proceeding. However, ribosomes can only bind to the normal RBS, as 37 °C RBS forms a hairpin structure, forestalling ribosomes from binding. The only gene downstream of the normal RBS is RFP, so only RFP would be expressed. 
</P><P>On the other hand, with red light, 37 °C RBS unfolds at 37 °C , resulting in the expression of both LuxR and GFP. LuxR would bind with AHL to form a complex that activates P<SUB>lux</SUB>. The activation of P<SUB>lux</SUB> produces sRNA that binds to the normal RBS on mRNA level, blocking it from ribosomes. As a result, RFP would not be expressed and only GFP remains at 37 °C.
</P><P>Without red light, P<SUB>red</SUB> would not be activated, and therefore, the red light induced circuit is completely shut down in the dark.
</P><H4><SPAN class="mw-headline" id="Dark_Response_Circuit">Dark Response Circuit</SPAN></H4><P>P<SUB>red</SUB> activates in the presence of red light, producing tetR that represses P<SUB>tet</SUB>. In other words, the dark induced part is inactive in the exposure to red light and active without red light. At 25 °C in the dark, only normal RBS functions to express YFP while 37 °C RBS remains as a hairpin structure. 
</P><P>At 37 °C, however, 37 °C RBS unfolds to express LuxR and BFP. Just like the mechanism employed in the light induced part, LuxR forms a complex with AHL to activate P<SUB>lux</SUB> that produces the sRNA to block normal RBS. This way, the YFP downstream of rRBS cannot be expressed and BFP is expressed at 37 °C.
</P><H4><SPAN class="mw-headline" id="Summary">Summary</SPAN></H4><P>This is the pathway summary of the E.colightuner. There is one kind of fluorescent protein in each condition we set. It means that we can control four gene expressions by using the E.colightuner. The only thing we have to do is changing the different environmental conditions for specifically expression.
</P></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="i-Culture">i-Culture</SPAN></H3><P>After a year of hard working, we have manage to accomplish our project modeling, assembling our E.colightuner as well as tissue culturing. The most significant event is we've also combined these three dependent systems and created a magnificent tool, called i-Culture, to achieve the goal of synchronizing different plant growth stages for exportation. We can control our E.colightuner by giving input data into the computer programs, regulating it to secrete the corresponding hormones under specific sets of conditions.
</P><P>For Example, Taiwan is one of the main exporters of orchid. From a business point of view, the orchid need to blossom precisely at the moment the customers receive it, so the flower growers need to control the blossoms manually during transporting. By using our i-Culture device, we can adjust the blossoming time automatically to reduce the laboring cost. In conclusion, our system has a great development potential in business.
</P></DIV></DIV><DIV class="li"><DIV class="card"><H3><SPAN class="mw-headline" id="Future_Works">Future Works</SPAN></H3><P>The multiple regulated-system we have created in this project consists of two different parts : the light induced part and the dark induced part. Each part regulates two different genes, resulting in a total of four genes that can be regulated. We tend to regulate more genes by employing more light sensing promoters. By adding each light sensor, we would be able to regulate two more different genes by employing regulation mechanism of 37 °C RBS and sRNA.
</P><H4><SPAN class="mw-headline" id="Application">Application</SPAN></H4><P>Taiwan is renowned for Phalaenopsis, producing more than 35 million Phalaenopsis each year. There is no doubt that Taiwan has become &quot;The Kingdom of Phalaenopsis&quot;. The technique we usually use to grow Phalaenopsis is plant tissue culture, thus we intend to apply it to our system. 
</P><P><B>Tissue culture</B> is the growth of tissues or cells separated from the organism. This is typically facilitated via use of a liquid, semi-solid or solid growth medium, such as broth or agar. Moreover, <B>hormones</B> are added to the growth medium purposed to what plants need during growth period. However, the hormones needed during each growing state are not the same. So, we need to figure out a way to generate multiple hormones. By adding our engineered <I>E.coli</I> to the growth medium, the plants can gain multiple hormones secreted by <I>E. coli</I> under different circumstances. We chose four common hormones as our genes, Auxin, Cytokinin, Gibberellin, and Florigen. 
</P><P>Since <B>Florigen</B> (known as the <I>FT</I> protein) is the protein which isn't produced by <I>E.coli</I> naturally, there aren't any secretion systems in wild type <I>E. coli</I> to transport Florigen out of the cell. Though secretion through cell lysis may seem as a possible method to solve this problem, we still seek for other ways in order not to let our <I>E. coli</I> die during the secretion process. Thus, we ought to create a secretion system which can help Florigen pass through the inner and outer membrane of <I>E. coli</I>. We have found a system from the 2012 Kyoto iGEM team called Twin Arginate Translocation (Tat) pathway, which can help send Florigen from the inner membrane to periplasm. In the Tat pathway, a Tat transporter recognizes <I>TorA</I> signal. Proteins possessing a <I>TorA</I> signal at their N terminals can permeate into the periplasm. Moreover, the 2012 Kyoto iGEM team also found out that the <I>kil</I> protein from λ phage are capable of making holes on the outer membrane, which then transports Florigen out of the cell. In other words, with the <I>kil</I> protein gene fused with the <I>torA</I> and <I>FT</I> protein gene in our biobrick, we should be able to successfully transport Florigen out of the <I>E. coli</I>. 
</P><P>The use of <B>Auxin</B> is to induce the growth of shoot, but an excess amount of Auxin would likely kill the plant. We want to reach the maximum amount that will promote the plant’s growth but without killing the plant. After researching, the Imperial College London 2011 iGEM team also had the same problem. They chose the native Auxin indole-3-acetic acid (IAA) as their solution. Compared to the former, the synthetic Auxin is more stable since it can be preserved for weeks. However, an excess amount of IAA could be toxic. To ensure the efficiency but at the same time not excessive, they built a model to reach the maximum amount.
</P><P>We intend to carry out our application with the device beneath. Cultured in the growth medium circulation, our engineered <I>E. coli</I> can absorb enough nutrition to express the desired genes properly. We also use a <B>filter</B>, which allows hormones but not bacteria themselves to pass through, separating the bacteria from plant tissues. Thus, we keep the tissue culture medium isolated.
</P><P>Under red light and 37 °C, <B>Cytokinin</B> will be produced.
Cytokinin is a plant growth substance which can primarily induce cell growth and differentiation in plant roots and shoots. It can decrease apical dominance which will make the main stem grow better compared with other stems. In this way, all the stems are able to become stronger. Cytokinin can also signal lateral bud growth, which will make the plant bushier.
</P><P>Under red light and 25 °C, Auxin will be produced.
Auxin can coordinate development from cellular, organs to the whole plant. Auxin molecules in cells may cause direct responses by stimulating or inhibiting certain gene expressions. In the cellular level, Auxin is important for cell growth. It induces axial elongation in roots and also lateral expansion which can make roots swell. In the whole plant level, it affects the shape of the plants. Because of the different functions, each organ has uneven Auxin concentration and causes specific shapes.
</P><P>Under dark and 25 °C, <B>Gibberellin</B> will be produced.
Gibberellin is important in the elongation of cells. It causes the extension of stems by inducing cell divisions and elongations. A special characteristic of it is that it can produce more mass when the plant is exposed to cold temperature.
</P><P>Under dark and 37 °C, Florigen will be produced.
Florigen is the key hormone to trigger plant blossoming. We use a promoter that can promote transcription of the FT gene, which then translate into FT proteins. FT proteins will then be transported via the phloem to the shoot apical meristem, where it will interact with the FD protein (transcription factor) to activate floral identity, thus inducing flowering.
</P><H4><SPAN class="mw-headline" id="The_next_BIG_thing">The next BIG thing</SPAN></H4><P>On the other hand, we tend to optimize our system to achieve subtle control of the four gene expressions. We might be able to do this by <B>taking the values between the limits we have set</B>. Instead of 25 °C and 37 °C , we tend to take the values between, so we can create more conditions under which different level of expressions can be achieved. We can also vary the intensity of red light to control the level of expression. For instance, we can gain different pigment outputs by tuning the temperature and varying the intensity of red light. The ultimate goal is to precisely control the level expression of each gene.
</P><P>So, we designed and built a device which is able to control the intensity of light. This device is used in the incubator and created for the purpose of covering all the light source outside. So the only light source is the red light LED we put in. In our first design, the device has a cuboid body and a thread top. This thread top is capable to change the light intensity, simply by changing different top with different number of LEDs.
</P><P>We use corrugated paper to make a prototype of our light control device. It has one red light LED and the device is covered by several layers of black plastic bag, preventing any possibility of light leaking.   
</P><P>As mentioned above in the beginning, using red light as an input condition is an default environment in our project. Our ultimate goal is to regulate multiple genes as more as possible. So, in the future, by employing more different conditions, such as <B>different lights</B>, we would be able to regulate more genes.(one light sensor would result in controlling two more different genes)
We hope that our project could eventually serve as a standardized model and set up with a standard operating procedure which offers future iGEMers to manipulate the control of multiple genes by simply changing the environmental conditions.
</P><H6><SPAN class="mw-headline" id="Reference_4">Reference</SPAN></H6><DIV class="ref"><OL start="17"><LI>Team : Imperial_College_London -2011.igem</LI><LI>Team : Kyoto -2012.igem</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV id="footer-text"><P>2013 NCTU_Formosa</P><P class="author">Website designed by Calvin Hue.</P><P>Cover image credit: <A href="http://www.dvq.co.nz/" target="_blank">DVQ</A></P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NCTU_Formosa/project">http://2013.igem.org/Team:NCTU_Formosa/project</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NCTU_Formosa/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NCTU_Formosa/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NCTU_Formosa/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NCTU_Formosa/project&amp;oldid=361711" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>