Team:UC Davis/Parts
Cyantific: UC Davis iGEM 2016
Home
Project
Overview
Protein Discovery
Novel Expression
Production Optimization
Expression in a GRAS Organism
Collaboration
Experiments
Notebook
Safety
Software
CBCR Model
iGEMMatch.org
People
Team
Attributions
Human Practices
Overview
Safe By Design
Seeing Blue: Scenarios for SYNENERGENE
How to Conduct Ethical Interventions in iGEM Surveys
“Beyond Artificial” Policy Paper
Adventures in Human Practices: Integration from Day 1
Judging
Novel Expression
As a result of our search for homologs to blue GAF proteins, our team selected 13 GAF domain to transform into E.coli. We were able to successfully transform 4 of them, 3 of which were blue and 1 red.
Motivation
A main focus of our project was to engineer a replacement for the dye Blue #1. While there are some known GAF proteins which absorb only 10 nm away from Blue #1, this actually represents large visual differences in color (25). For example, the difference in color from Spirulina to Blue #1 is only 10nm, and that is clearly visible by eye (26). In an effort to match Blue #1’s peak spectra better, our team chose 13 blue GAF domain homologs in an effort to expand the known blue spectra of GAF domains, and ultimately get closer to ideal absorption of 630nm (8).
Research
After carefully selecting our 13 GAF proteins we began their transformation into E.coli. For easy induction, the DNA for the coding sequences were synthesized with a pBAD promoter (K206000) and strong RBS (B0034) so cultures can be induced with arabinose to produce the GAF protein, also known as the apoprotein.
Due to cloning difficulties, we resorted to using standard digestion/ligation methods to insert the protein-coding sequences into pSB1C3 sourced from J04450. Since untransformed plasmid colonies appear red, correctly transformed cells were screened by selecting white colonies. These were sequenced and co-transformed into DH5-alpha cells along with a plasmid containing the PCB pathway enzymes. The co-transformed cells were screened using a double-antibiotic plate, and a correctly co-transformed colony was used to innoculate 50ml TB cultures induced with arabinose. These cultures were harvested after overnight shaking for use in protein purification.
We were able to create the following succesfully expressed parts
BBa_K1916000
NpF2164g5
http://parts.igem.org/Part:BBa_K1916000
BBa_K1916002
Cyan7427
http://parts.igem.org/Part:BBa_K1916002
BBa_K1916004
Ga0100955_11928g3
http://parts.igem.org/Part:BBa_K1916004
BBa_K1916018
CBCR8
http://parts.igem.org/Part:BBa_K1916018
BBa_K1916100
pBAD+RBS+NpF
http://parts.igem.org/Part:BBa_K1916100
BBa_K1916102
pBAD+RBS+Cyan
http://parts.igem.org/Part:BBa_K1916102
BBa_K1916104
pBAD+RBS+G3
http://parts.igem.org/Part:BBa_K1916104
BBa_K1916118
pBAD+RBS+CBCR8
http://parts.igem.org/Part:BBa_K1916118
After cell lysis by sonication, the proteins were further purified using the intein mediated purification with an affinity chitin-binding tag (21). This is a purification method utilizes the inducible self-cleaving inteins to separate the protein from the affinity tag without the use of the protease (21). This purification method was chosen because it allows complete removal of the intein tag from the protein, meaning no foreign DNA is remaining on the protein after purification. Our team felt this is an important step in relation to the labeling scheme our dye may undergo in which having even a piece of non-native DNA could mean dramatic differences. A more thorough discussion of the labeling possibilities can be found in the human practices tab.
Our team discovered the purification of these proteins ultimately had a noticeable effect on the final absorption spectra. The main difference being the purified protein has a much clearer and refine absorbance spectra and does not have other residues around the 400 nm range. However, one downside that is that the purification does lead to a loss in product, as evidenced by the reduction in peak height. This loss of product can ultimately be improved in the future more efficient eluting methods.
Below is the spectra of Blue #1 compared to our positive GAF control and a novel GAF protein. Our positive GAF control, labeled here as NpF2164g5, is a previoulsy characterized GAF protein, but is new  to iGEM and built into a new plasmid construct by our team. This acts as a positive control that our data is reproducible and supported by published peer reviewed literature. Similar to other findings for the absorbance of the native NpF2164g5 15Z (-), our data shows the max peak absorbance at 640nm, only 10nm away from Blue #1 (13). The novel GAF protein, labeled here as G3 is a untested GAF protein sequence donated to us by our collaborators in the Clark Lagarias laboratory which we determined to have a peak around 740nm. This means the protein appears green to the human eye. Unfortunately this new GAF proteins is substaintially farther away from Blue #1, it still adds a new data point to the GAF absorbance spectra and also helps validate our predictive model. This protein was correctly predicted to absorb at higher than 600nm by our predictive model. Unfortunately the spectra of the remaining four novel GAF proteins which are correctly transformed and sequence verified are not available; however, our team hopes to bring the spectra to the iGEM compeitition.
Future Directions
In the future, our team would like to work towards expanding the GAF domain spectra by both continuing the search for GAF homologs and also by mutations. By continuing the search for GAF homologs we feel the spectra can be significantly more filled; however, given that small changes in peak absorption visually have a significant difference in color,  we also think forcing mutations in known GAF domains is the next step. Similar to mutations performed in Roger Y. Tsien’s lab to which established that green fluorescent protein (GFP) spectral characteristics could be dramatically changed by single point mutations, our team hypothesizes that this may be the case for GAF domains as well (22). This was the case when a single point mutation in a conserved cysteine within the GAF domain transformed the phytochrome to be fluorescent, meaning that it is possible that GAF proteins spectral characteristics could be altered by intentional mutations in protein sequences. (27)
Next: Production Optimization
Trotter, Greg. "Food Companies Are Phasing out Artificial Dyes, but Not Fast Enough for Some." Food Companies Are Phasing out Artificial Dyes, but Not Fast Enough for Some. Chicago Tribune, 24 June 2016. Web. 18 Oct. 2016.
"Konica Minolta - Products." Instrument Systems - Sensing Americans. Konica Minolta Sensing Americans Inc., Oct. 2016. Web. 18 Oct. 2016.
"Our Proposition on Artificial Colors - Colors Policy." MARS Incorporated, Oct. 2016. Web. 18 Oct. 2016.
"Kraft Mac & Cheese Says Goodbye to the Dye." Decision - 2016. NBC News, 20 Apr. 2015. Web. 18 Oct. 2016.
"Taste of General Mills." A Big Commitment for Big G Cereal. General Mills Cereal Incorporated, 22 June 2015. Web. 18 Oct. 2016.
"Nestlé USA Commits to Removing Artificial Flavors and FDA-Certified Colors from All Nestlé Chocolate Candy by the End of 2015." 150 Years of Good Food, Good Life. Nestle Incorporated, 17 Feb. 2015. Web. 18 Oct. 2016.
Alexandratos, Nikos, and Jelle Bruinsma. "World Agriculture Towards 2030/2050." Agricultural Development Economics Division - Food and Agriculture Organization of the United Nations. Global Perspective Studies Team, June 2012. Web. 18 Oct. 2016.
"Compound Summary for CID 19700." Brilliant Blue FCF. Pub Chem - Open Chemistry Database, 15 Oct. 2016. Web. 18 Oct. 2016.
"Summary of Color Additives for Use in the United States in Foods, Drugs, Cosmetics, and Medical Devices." For Industry - Color Additives - Color Additive Inventories. U.S. Food & Drug Administration, May 2016. Web. 18 Oct. 2016.
http://link.springer.com/article/10.1007/s00217-004-1062-7
http://www.instructables.com/id/Blue-Foods-Colorful-cooking-without-artificial-dy/
A Brief History of Phytochrome Nathan C. Rockwell and J. Clark Lagarias
Hirose, Yuu, et al. "Green/red cyanobacteriochromes regulate complementary chromatic acclimation via a protochromic photocycle."Proceedings of the National Academy of Sciences 110.13 (2013): 4974-4979.
Rockwell, Nathan C., and J. Clark Lagarias. "A brief history of phytochromes." ChemPhysChem 11.6 (2010): 1172-1180.
Bussell, Adam N., and David M. Kehoe. “Control of a Four-Color Sensing Photoreceptor by a Two-Color Sensing Photoreceptor Reveals Complex Light Regulation in Cyanobacteria.” Proceedings of the National Academy of Sciences of the United States of America 110.31 (2013): 12834–12839. PMC. Web. 7 Oct. 2016.
http://www.nutritionaloutlook.com/food-beverage/coloring-spirulina-blue
Nature’s Palette: The Search for Natural Blue Colorants
https://www.novozymes.com/en
http://www.dsm.com/corporate/home.html
http://prodata.swmed.edu/promals3d/promals3d.php
https://www.neb.com/products/e6901-impact-kit
Heim R, Cubitt A, Tsien R (1995). "Improved green fluorescence" (PDF). Nature. 373 (6516): 663–4. doi : 10.1038/373663b0 . PMID 7854443 .
https://www.ncbi.nlm.nih.gov/genome/browse/
http://pfam.xfam.org/
Red/Green Cyanobacteriochromes: Sensors of Color and Power - lagarias
https://en.wikipedia.org/wiki/Brilliant_Blue_FCF#/media/File:Blue_smarties.JPG
Fischer, A. J. and J. C. Lagarias (2004). "Harnessing phytochrome's glowing potential." Proceedings of the National Academy of Sciences of the United States of America 101(50): 17334-17339.
Stability of phycocyanin extracted from Spirulina sp.: Influence of temperature, pH and preservatives - R Chaiklahan, N Chirasuwan, B Bunnag - Process Biochemistry, 2012 - Elsevier
http://www.lightboxkit.com/Assay_dye.html
"Your Brain - A User's Guide - 100 Things You Never Knew." National Geographic Time INC. Specials 12 Feb. 2016: Print.
Sensing, Konica Minolta. "How Color Affects Your Perception of Food." Konica Minolta Color, Light, and Display Measuring Instruments. Konica Minolta Sensing Americans Inc., 2006. Web. 10 Oct. 2016.
"Center for Disease Control and Prevention - General Information Escherichia Coli." CDC 24/7: Saving Lives, Protecting People. Centers for Disease Control and Prevention, 06 Nov. 2015. Web. 10 Oct. 2016.
Olmos J, Paniagua-Michel J (2014) Bacillus subtilis A Potential Probiotic Bacterium to Formulate Functional Feeds for Aquaculture. J Microb Biochem Technol 6: 361-365. doi:10.4172/1948-5948.1000169
Zeigler, Daniel R. "Bacillus Genetic Stock Center." Bacillus Genetic Stock Center Website. The Bacillus Genetic Stock Center - Biological Sciences, Oct. 2016. Web. 10 Oct. 2016.
Vellanoweth, Robert Luis, and Jesse C. Rabinowitz. "The Influence of Ribosome binding-sites Elements on Translational Efficiency in Bacillus Subtilis and Escherichia Coli in Vivo." Molecular Microbiology 1105-1114, 15 Jan. 1992. Web. 9 Oct. 2016.
"Pveg (Plasmid #55173)." Addgene - The Nonprofit Plasmid Repository. Synthetic Biology; Bacillus BioBrick Box, Aug. 2016. Web. 10 Oct. 2016.
Gambetta, Gregory A., and Clark J. Lagarias. "Genetic Engineering of Phytochrome Biosynthesis in Bacteria." Section: Molecular and Cellular Biology. Proceedings of the National Academy of Sciences of the United States of America, 19 July 2001. Web. 9 Oct. 2016.
"Registry of Standard Biological Parts - B0015." Part: BBa_B0015. International Genetically Engineered Machine Competition (iGEM), 17 July 2003. Web. 9 Oct. 2016.
"Bacillus Subtilis - Indiamart." IndiaMART InterMESH Ltd., n.d. Web. 9 Oct. 2016.
Contact us at: ucdigem@gmail.com
