<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_Davis_Optimization skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC Davis/Optimization</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cyantific: UC Davis iGEM 2016</TITLE></P><DIV class="container-fluid"><NAV class="navbar"><DIV class="row row-centered"><DIV class="col-xs-12"><UL class="nav navbar-nav" id="nav"><LI class="navBtn homeLink"><A class="headLink" href="https://2016.igem.org/Team:UC_Davis">Home</A></LI><LI role="presentation" class="dropdown navBtn projectLink"><A class="dropdown-toggle headLink" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
                Project </A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Discovery">Protein Discovery</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Novel">Novel Expression</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Optimization">Production Optimization</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/GRAS">Expression in a GRAS Organism</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Safety">Safety</A></LI></UL><LI role="presentation" class="dropdown navBtn"><A class="dropdown-toggle headLink" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
                Software </A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Discovery">CBCR Model</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/iGEMMatch">iGEMMatch.org</A></LI><LI role="presentation" class="dropdown navBtn"><A class="dropdown-toggle headLink" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
                People </A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Attributions">Attributions</A></LI><LI role="presentation" class="dropdown navBtn projectLink"><A class="dropdown-toggle headLink" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
                Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Safe">Safe By Design</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Syn">Seeing Blue: Scenarios for SYNENERGENE</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Interview">How to Conduct Ethical Interventions in iGEM Surveys</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/Beyond">“Beyond Artificial” Policy Paper</A></LI><LI><A href="https://2016.igem.org/Team:UC_Davis/HP_Journal">Adventures in Human Practices: Integration from Day 1</A></LI><LI class="navBtn homeLink"><A class="headLink" href="https://2016.igem.org/Team:UC_Davis/Judging">Judging</A></LI></DIV></DIV></NAV></DIV><DIV class="container-fluid"><DIV class="row row-centered body_odd"><DIV class=" col-centered col-xs-10"><H1>Product Optimization</H1><P>Given that our project focuses on how to bring GAF proteins into the food colorant industry and have large scale production, a primary concern is how GAF protein production can be optimized. Our team first focused on apoprotein production using our GAF protein constructs that were regulated by a pBAD promoter. We induced cultures with different concentrations of arabinose, and after harvesting we assessed pellet volume and color vibrancy to see if there was an optimal inducer concentration.</P><P>Next our team focused on optimizing the entire genetic circuit, now including the cofactor production pathway along with GAF protein expression, to produce the highest protein expression. With the results of our previous induction experiment, we hypothesized that changing the expression mechanism of the cofactor could affect overall production of the complete functional protein. While our team was not able to complete our genetic circuit work, we provide a detailed experimental design and our work thus far.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row row-centered"><DIV class="col-centered col-xs-12"><H3>Motivation</H3><P>A main concern with biomanufacturing is how to maximize production of profitable product to offset manufacturing costs. By engineering the pathway at the cellular level, we can infer how we can optimize each step of the pathway to produce more protein using reasonable amounts of resources.</P><P>For our project, we take the first step by experimenting with and characterizing GAF protein production in E. coli. Although E. coli is not the optimal chassis to use for large scale production, our simpler, small scale experiments can produce sufficient data from E. coli. In addition, our parts can be used by future teams who wish to continue taking the next proper steps in advancing our protein towards large scale production.</P><H3>Induction Experiment</H3><P>After expressing our GAF protein our team observed visual differences in cell pellets depending on the amount of arabinose added to the cultures. In our construct arabinose induces the expression of the GAF. While it is widely known that the photosensory properties of the CBCR are dependant on the availability of the bilin cofactor (8), our team wanted to quantify that in regards to our genetic circuit and determine if there is a saturation point in which the arabinose is either 1) no longer effective or 2) harmful.</P><P>First we made 20mM, 10mM, and 1mM (following from left to right) cultures of our NpF2164g5 GAF construct. The 1mM culture had the largest cell pellet, but also lacked any vibrancy. The 10mM culture had a medium sized pellet, and appeared to be the most vibrant and the 20mM arabinose concentration culture was roughly as vibrant as the 10mM, but had a much smaller cell volume. Since the 10 mM solution produced the best results, our team repeated the experiment, but this time with 1mM, 3mM, 10mM, 33mM, and 100mM arabinose concentration in growth media. </P><P>After lysing the cells by sonication the lysate was used to take the absorbance spectra. Cells did not grow at 33mM and 100mM arabinose concentrations, likely because either the environment was no longer suitable for growth, or the cell was allocating more resources to GAF synthesis, but there was not enough cofactors to associate with the protein. Of the remaining three media concentrations that did produce cell pellets, the cells were lysed and the lysate used to take spectra. Surprisingly, the 3mM had a higher peak absorbance than the 10mM. This lack of correlation indicates that the higher induction levels do not correlate to higher peak absorbance levels meaning that there is an optimal production of the GAF. Based on our results our team estimates this is around 3mM arabinose concentrations of cell culture. While the induction of GAF synthesis obviously affected the protein vibrance and production efficiency, there could also be ways to improve the genetic circuit itself. For this reason, our team went on to design genetic circuit library that we would like to complete in the future.</P><H3>Genetic Circuit Experiment Design</H3><P>Because the GAF protein must associate with a bilin cofactor in order to be photoactive, an engineered cell must also be able to express the necessary enzymes that convert heme into the cofactor. Such an expression system that requires multiple coding genes has room for optimization in order to control the flux of the protein production. For example, upregulating expression of one of the heme-converting enzymes would theoretically increase bilin production and consequently affect overall production rate of the complete protein. Optimizing the pathway would be the next key step in advancing to industrial-scale mass production.</P><P>Our next step will be to design and test various operon structures for the key genes for protein production. From the iGEM registry, there are collections of promoters and ribosome binding sites of various strengths. We can design an operon library in which we swap promoters and RBS’s used to regulate expression of each gene. Monocistronic and polycistronic variants can also be considered, as well as gene order. Such a library would have thousands of variations. Experimentally measuring overall protein production will provide insight of what structure, gene order, and regulatory components will produce the most efficient genetic circuit for protein production.</P><P>From the current registry catalog, we can use the Anderson collection of constitutive promoters since they span a wide range of relative strength. The community RBS collection can also be considered for their popularity. A single strong terminator (B0015) will be used throughout since we decided termination rate is not worth varying in our libraries.</P><P>We have designed three overarching hierarchical operon structures for our phycocyanobilin-dependent GAF proteins. These proteins require three genes: heme oxygenase (HO) to convert heme to biliverdin, PcyA to convert biliverdin to the PCB cofactor, and the coding for the actual apoprotein. HO and PcyA are already in the registry, as they are the oldest parts ever submitted from the very first iGEM (I15008 and I15009, respectively) and are available for all future teams. Our goal with the operon will be to provide insight to future iGEM projects on how HO and PcyA should be expressed in conjunction with the CBCR protein.</P><P>Our first design is completely monocistronic, with each of these three proteins under control of separate promoters.</P><P>Our second design is partially polycistronic, with HO and PcyA expressed under the same promoter while the apoprotein is under separate regulation. Our reasoning for this split is because PCB production is an independent metabolic pathway and it is worth exploring separately.</P><P>Our third design is completely polycistronic. This provides the most flexibility in gene order, as the three genes can have swapped positions with different strength RBS’s while being regulated together under the same promoter.</P><H3 class="inline">Future Directions</H3><P>In the future, our team would like to do a more thorough investigation into induction quantity. We found our results that different GAF proteins required different amounts of the cofactor to be quite interesting and would like to expand our experiment to include other GAF proteins and also GAF proteins which use biliverdin as their cofactor, unlike the ones used here which only use the PCB cofactor. If we find the trend that cofactor availability affects different GAF proteins differently we would like to investigate what three dimensional aspects to the protein are influencing this.</P><P>As for the genetic circuit design, our team would like to finish the last bit of the operon library to find if there is an optimal genetic circuit that significantly upregulates or downregulates GAF protein expression. Unfortunately, our team was not able to finish the operon construction over the summer but was able to get the almost the entire construct with only the first promoter,rbs, and plasmid not assembling.</P></DIV></DIV><DIV class="col-xs-6 col-centered projLinks centered"><A href="https://2016.igem.org/Team:UC_Davis/GRAS" class="navBtn">Next: Expression in a GRAS Organism</A></DIV><DIV class="row row-centered footNotes"><DIV class="col-xs-12 col-centered"><OL><LI>Trotter, Greg. &quot;Food Companies Are Phasing out Artificial Dyes, but Not Fast Enough for Some.&quot; <EM>Food Companies Are Phasing out Artificial Dyes, but Not Fast Enough for Some</EM>. Chicago Tribune, 24 June 2016. Web. 18 Oct. 2016.</LI><LI><A href="http://sensing.konicaminolta.us/products/">&quot;Konica Minolta - Products.&quot; <EM>Instrument Systems - Sensing Americans</EM>. Konica Minolta Sensing Americans Inc., Oct. 2016. Web. 18 Oct. 2016.</A></LI><LI><A href="http://www.mars.com/global/about-us/policies-and-practices/color-policy">&quot;Our Proposition on Artificial Colors - Colors Policy.&quot; MARS Incorporated, Oct. 2016. Web. 18 Oct. 2016.</A></LI><LI><A href="http://www.nbcnews.com/business/consumer/kraft-mac-cheese-says-goodbye-dye-n345006">&quot;Kraft Mac &amp; Cheese Says Goodbye to the Dye.&quot; <EM>Decision - 2016</EM>. NBC News, 20 Apr. 2015. Web. 18 Oct. 2016.</A></LI><LI>&quot;Taste of General Mills.&quot; <EM>A Big Commitment for Big G Cereal</EM>. General Mills Cereal Incorporated, 22 June 2015. Web. 18 Oct. 2016.</LI><LI><A href="http://www.nestleusa.com/media/pressreleases/nestl%C3%A9-usa-commits-to-removing-artificial-flavors-and-fda-certified-colors-from-all-nestl%C3%A9-chocolate-candy-by-the-end-of-20">&quot;Nestlé USA Commits to Removing Artificial Flavors and FDA-Certified Colors from All Nestlé Chocolate Candy by the End of 2015.&quot; <EM>150 Years of Good Food, Good Life</EM>. Nestle Incorporated, 17 Feb. 2015. Web. 18 Oct. 2016.</A></LI><LI><A href="http://www.fao.org/docrep/016/ap106e/ap106e.pdf">Alexandratos, Nikos, and Jelle Bruinsma. &quot;World Agriculture Towards 2030/2050.&quot; <EM>Agricultural Development Economics Division - Food and Agriculture Organization of the United Nations</EM>. Global Perspective Studies Team, June 2012. Web. 18 Oct. 2016.</A></LI><LI><A href="https://pubchem.ncbi.nlm.nih.gov/compound/Acid_Blue_9#section=Stability">&quot;Compound Summary for CID 19700.&quot; <EM>Brilliant Blue FCF</EM>. Pub Chem - Open Chemistry Database, 15 Oct. 2016. Web. 18 Oct. 2016.</A></LI><LI><A href="http://www.fda.gov/ForIndustry/ColorAdditives/ColorAdditiveInventories/ucm115641.htm">&quot;Summary of Color Additives for Use in the United States in Foods, Drugs, Cosmetics, and Medical Devices.&quot; <EM>For Industry - Color Additives - Color Additive Inventories</EM>. U.S. Food &amp; Drug Administration, May 2016. Web. 18 Oct. 2016.</A></LI><LI><A href="http://link.springer.com/article/10.1007/s00217-004-1062-7">http://link.springer.com/article/10.1007/s00217-004-1062-7</A></LI><LI><A href="http://www.instructables.com/id/Blue-Foods-Colorful-cooking-without-artificial-dy/">http://www.instructables.com/id/Blue-Foods-Colorful-cooking-without-artificial-dy/</A></LI><LI>A Brief History of Phytochrome Nathan C. Rockwell and J. Clark Lagarias</LI><LI>Hirose, Yuu, et al. &quot;Green/red cyanobacteriochromes regulate complementary chromatic acclimation via a protochromic photocycle.&quot;<EM>Proceedings of the National Academy of Sciences</EM> 110.13 (2013): 4974-4979.</LI><LI>Rockwell, Nathan C., and J. Clark Lagarias. &quot;A brief history of phytochromes.&quot; <EM>ChemPhysChem</EM> 11.6 (2010): 1172-1180.</LI><LI>Bussell, Adam N., and David M. Kehoe. “Control of a Four-Color Sensing Photoreceptor by a Two-Color Sensing Photoreceptor Reveals Complex Light Regulation in Cyanobacteria.”<EM>Proceedings of the National Academy of Sciences of the United States of America</EM> 110.31 (2013): 12834–12839. <EM>PMC</EM>. Web. 7 Oct. 2016.</LI><LI><A href="http://www.nutritionaloutlook.com/food-beverage/coloring-spirulina-blue">http://www.nutritionaloutlook.com/food-beverage/coloring-spirulina-blue</A></LI><LI>Nature’s Palette: The Search for Natural Blue Colorants</LI><LI>https://www.novozymes.com/en</LI><LI><A href="http://www.dsm.com/corporate/home.html">http://www.dsm.com/corporate/home.html</A></LI><LI><A href="http://prodata.swmed.edu/promals3d/promals3d.php">http://prodata.swmed.edu/promals3d/promals3d.php</A></LI><LI><A href="https://www.neb.com/products/e6901-impact-kit">https://www.neb.com/products/e6901-impact-kit</A></LI><LI>Heim R, Cubitt A, Tsien R (1995).<A href="http://tsienlab.ucsd.edu/Publications/Heim%201995%20Nature%20-%20Improved%20GFP.PDF">&quot;Improved green fluorescence&quot;</A> (PDF). <EM>Nature</EM>. 373 (6516): 663–4.<A href="https://en.wikipedia.org/wiki/Digital_object_identifier">doi</A>:<A href="https://dx.doi.org/10.1038%2F373663b0">10.1038/373663b0</A>.<A href="https://en.wikipedia.org/wiki/PubMed_Identifier">PMID</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/7854443">7854443</A>.</LI><LI><A href="https://www.ncbi.nlm.nih.gov/genome/browse/">https://www.ncbi.nlm.nih.gov/genome/browse/</A></LI><LI><A href="http://pfam.xfam.org/">http://pfam.xfam.org/</A></LI><LI>Red/Green Cyanobacteriochromes: Sensors of Color and Power - lagarias</LI><LI><A href="https://en.wikipedia.org/wiki/Brilliant_Blue_FCF#/media/File:Blue_smarties.JPG">https://en.wikipedia.org/wiki/Brilliant_Blue_FCF#/media/File:Blue_smarties.JPG</A></LI><LI> Fischer, A. J. and J. C. Lagarias (2004). &quot;Harnessing phytochrome's glowing potential.&quot; Proceedings of the National Academy of Sciences of the United States of America 101(50): 17334-17339.</LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S1359511312000268">Stability of phycocyanin extracted from Spirulina sp.: Influence of temperature, pH and preservatives</A>- R Chaiklahan, N Chirasuwan, B Bunnag - Process Biochemistry, 2012 - Elsevier</LI><LI>http://www.lightboxkit.com/Assay_dye.html</LI><LI>&quot;Your Brain - A User's Guide - 100 Things You Never Knew.&quot; National Geographic Time INC. Specials 12 Feb. 2016: Print.</LI><LI>Sensing, Konica Minolta. &quot;How Color Affects Your Perception of Food.&quot; <EM>Konica Minolta Color, Light, and Display Measuring Instruments</EM>. Konica Minolta Sensing Americans Inc., 2006. Web. 10 Oct. 2016.</LI><LI>&quot;Center for Disease Control and Prevention - General Information Escherichia Coli.&quot; <EM>CDC 24/7: Saving Lives, Protecting People</EM>. Centers for Disease Control and Prevention, 06 Nov. 2015. Web. 10 Oct. 2016.</LI><LI>Olmos J, Paniagua-Michel J (2014) Bacillus subtilis A Potential Probiotic Bacterium to Formulate Functional Feeds for Aquaculture. J Microb Biochem Technol 6: 361-365. doi:10.4172/1948-5948.1000169</LI><LI>Zeigler, Daniel R. &quot;Bacillus Genetic Stock Center.&quot; <EM>Bacillus Genetic Stock Center Website</EM>. The Bacillus Genetic Stock Center - Biological Sciences, Oct. 2016. Web. 10 Oct. 2016.</LI><LI>Vellanoweth, Robert Luis, and Jesse C. Rabinowitz. &quot;The Influence of Ribosome binding-sites Elements on Translational Efficiency in Bacillus Subtilis and Escherichia Coli in Vivo.&quot; Molecular Microbiology 1105-1114, 15 Jan. 1992. Web. 9 Oct. 2016.</LI><LI>&quot;Pveg (Plasmid #55173).&quot; <EM>Addgene - The Nonprofit Plasmid Repository</EM>. Synthetic Biology; Bacillus BioBrick Box, Aug. 2016. Web. 10 Oct. 2016.</LI><LI>Gambetta, Gregory A., and Clark J. Lagarias. &quot;Genetic Engineering of Phytochrome Biosynthesis in Bacteria.&quot; <EM>Section: Molecular and Cellular Biology</EM>. Proceedings of the National Academy of Sciences of the United States of America, 19 July 2001. Web. 9 Oct. 2016.</LI><LI>&quot;Registry of Standard Biological Parts - B0015.&quot; <EM>Part: BBa_B0015</EM>. International Genetically Engineered Machine Competition (iGEM), 17 July 2003. Web. 9 Oct. 2016.</LI><LI>&quot;Bacillus Subtilis - Indiamart.&quot; IndiaMART InterMESH Ltd., n.d. Web. 9 Oct. 2016.</LI></OL></DIV></DIV></DIV></DIV><DIV class="container-fluid footer"><DIV class="row row-centered"><DIV class="col-xs-8 col-centered centered"><P>Contact us at: ucdigem@gmail.com </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>