Team:UC Davis/Notebook
Cyantific: UC Davis iGEM 2016
Home
Project
Overview
Protein Discovery
Novel Expression
Production Optimization
Expression in a GRAS Organism
Collaboration
Experiments
Notebook
Safety
Software
CBCR Model
iGEMMatch.org
People
Team
Attributions
Human Practices
Overview
Safe By Design
Seeing Blue: Scenarios for SYNENERGENE
How to Conduct Ethical Interventions in iGEM Surveys
“Beyond Artificial” Policy Paper
Adventures in Human Practices: Integration from Day 1
Judging
Welcome to our lab notebook
Below is a chronological journey through our experimental progress.
April
4/1
First iGEM meeting - Discussion of team progress and possible projects to perform for iGEM.
4/15
Meeting with potential on campus collaborators to discuss CBCR background and field.
4/29
Team agreement to base our summer research focus on CBCR expression in E. Coli and Bacillus. We decided that this would be an ideal project for our group because there are already a few experts at UC Davis on CBCR’s, among the feasibility and compatibility of using CBCR’s as an iGEM summer research project.
May
5/10
Meeting with on-campus collaborators to learn more about CBCR background information. Met with a researcher named Nathan C. Rockwell who is an expert on CBCR’s and have published multiple papers on how CBCR’s work through our institution, UC Davis. l
5/11
Meeting with MARS scientist to discuss industry needs and concerns with dyes as food colorants. In the process of these interviews, we discovered that we do not need to worry about having our cells autolyse because the MARS industry has specific machines already in place to control for this. The industry standard for organism-produced dyes has been Spirulina, therefore we decided to keep our workflow in Bacillus and E. Coli similar to Spirulina so our fits into the existing industry framework .
5/13
Explored iGEM prior art and discovered that only 20 teams have ever utilized the CBCR complex, 11 of which were biosensors, and 9 were to influence cellular function. No teams have ever adapted CBCRs for any form of use in food. Previous iGEM teams have also not attempted expression in Bacillus, let alone, produce CBCRs in a concentrated form.
5/14
First UCSC collaboration meeting (over skype). Met to discuss potential routes of collaboration on the human practices frontier and for science. Due to strong overlap in our scope of work, we determined that we would be able to establish meaningful collaboration on the human practices frontier to help both teams. UCSC is also using Bacillus subtilis as an expression chassis, as is our team. We therefore planned to share materials, protocols, and experimental successes in the course of the summer to establish our collaboration.
5/20
Project plan and Gantt chart (timeline) developed for summer iGEM experimentation. We chose to focus on four main projects:
Characterization of novel CBCRs
Operon optimization in E. coli
CBCR expression in Bacillus
Investigate regulatory framework surrounding a project like ours and current consumer concerns.
5/20 - Brent
Preliminary research was performed to identify non-sporulating, protease deficient, Bacillus subtilis strains for research use. Dr. Daniel Zeigler, from the Bacillus Genetic Stock Center was contacted for more information on this. Dr. Zeigler recommended the following Bacillus strains, which we ended up using for our Bacillus studies.
B. subtilis K07 - BGSC Accession # 1A1134– Generated from the commonly used laboratory host PY79. This strain has all it's seven associated proteases knocked out and is free from antibiotic resistance or integrated plasmids. This strain does produce spores however.
B. subtilis PY79 - BGSC Accession # 1A747 - which is the wildtype strain of the K07 protease knockout. We are selecting this strain so we have a control for comparison.
B. subtilus spo0A3 - BGSC Accession # 1S1 - This strain comprises spo0A mutants, therefore they do not produce alkaline or neutral proteases (most abundant). They also do not sporulate. Wall-associated proteases are still present. This strain is the result of mutations in the Spo0A locus and have normal antibiotic sensitivities. Note that Spo0A have reduced genetic competence, so it may be advisable to use electroporation to transform them.
June
6/10- Samir
iGEMmatch.org functional and running. We plan on using the UCSF symposium to add more teams and get feedback.
6/13
Our iGEM team attended the UCSF Bioengineering symposium and had an iGEM meet-up with the UCSD, UCSC, and the UC Berkeley team to discuss our projects.
6/17
12 Novel CBCR sequences from microbial samples selected and ordered
6/23 - Brent
pHCMC04, an 8,000 bp vector, was chosen as our integration chassis for our Bacillus experimentation. pHCMC04 is compatible in both E. Coli and in Bacillus for transformation. We annotated this vector for a region to insert our Bacillus DNA of interest in. We also prepared vectors to remove the Bsal site present on this vector and submitted this order to IDT.
6/29 - Brent
PcyA and HO were mini-preped and eluted with water. A nanodrop test was run on these samples to determine concentration and purity. The results illustrated 132.29 ng/μl for PcyA i15004 and 212.32 ng/μl for HO.
A gel electrophoresis sample was next performed with these constructs, and SYBR safe was used as a visualizing agent for the DNA. 0.375 grams of Agrose in 50 ml of 1x TAE Buffer was heated to a boil to produce the homogenous gel mixture, 3 μl of SYBR safe visualizing gel agent was added, and this solution was cooled to produce the solidified gel.
6/30- Brent
PcyA and HO from 6/30 were PCR amplified using a 55℃ annealing temperature. For each PCR, 10 μl of HF buffer was added, 1 μl of 10 nM DNTP’s, 0.5 μl of DNA Polymerase, 23.5 μl of milliQ H20, and 9 additional microliters of milliQ H20. 2.5 μl of a forward primer was added, 2.5 μl of a reverse primer was added, and 1 μl of template DNA.
PCR was implemented with a ProFlex PCR system with the following parameters:
Initial Denaturation at 98 deg. C. for 30 seconds
Denaturation at 98 deg. C. for 10 seconds
Primer Annealing at 55 deg. C. for 30 seconds
Extension at 72 deg. C. for 30 seconds per 1,000 bp of DNA
Final extension at 72 deg. C. for 5 minutes
Held at 4 deg. C. until used
Gel electrophoresis was used to verify appropriate band sizes and amplification of the DNA. HO = 720 bp and PcyA = 750 bp. Bands appeared in appropriate places with amplified DNA, indicating successful amplification of the PcyA and HO constructs.
July
7/5- Jon
PCR amplified PcyA(bc), PcyA(dg), HO(bc), HO(dg), NPF (h beta) [a CBCR], and PSB1C3 with an annealing temp of 55 ℃ was used. Electrophoresis of these samples indicated that these amplifications were successful.
7/6 - Amanda
All 13 CBCR gblocks were amplified with the promoter (Pbad), RBS (B0034), and chitin binding domain. An annealing temperature of 55℃ was used. This amplification was unsuccessful as the CBCRs should be about 500bp, however the gel electrophoresis run on these samples did not indicate any bands in this region.
7/8 - Jon
Nanodropped PCR amplified PcyA(bc), PcyA(dg), HO(bc), HO(dg), NPF (h beta) [a CBCR], to determine concentrations in preparation for a Golden Gate Reaction.
7/11 - Amanda
Tried to optimize PCR amplification of CBCR gblock since the experimental PCR amplification conducted on 7/6 was unsuccessful. The amplification was performed on CBCR2 at an annealing temp of 55, 60, and 65℃. These results were successful. It is hypothesized that the issue encountered on 7/6 was DNA polymerase staying active, due to the previous annealing temperatures, among the 13 CBCR samples.
7/12- Samir
PCR amplified CBCR3 and CBCR4 but changed the extension period from 30sec to 50sec. This was successful for CBCR3 and not CBCR4. This leads us to believe that some CBCRs amplify better than others.
7/13- All
To date, PCR amplification of CBCR’s have only been successful with select CBCR’s. An attempt was made to PCR amplify the previously unsuccessful CBCR’s to determine if the extension period was not long enough. CBCR 4, CBCR 6, CBCR 8, CBCR 9, CBCR 11, OSC 7112, GAG 2, GAG 3, Cyan, Cha, and NPF CBCR samples were tested by PCR Amplification. All sequences amplified successfully, with the exception of GAG 2 and GAG 3, when the amplification was performed with an extension period of 55 seconds.
On the Bacillus side of the experimentation, the CBCR - NPF, and the terminator B0015, was amplified with the appropriate primers. The standard PCR protocol was implemented, with a primer annealing time set to 55 seconds and a temperature of 55 deg. C.
The construct below, which is the gene designation for Bacillus experimentation, contains 3 RBS sites as designated below:
Each RBS is comprised of two primers which were designed to be annealed together. For each of the RBS sites, two sets of primers were combined in equal concentration and annealed with 98 deg. C. for 10 minutes, and then cooled in a temperature ramping setup to room temperature (21 deg. C.).
PCR Amplifications of the Bacillus CBCR - NPF and the terminator B0015 were imaged by gel electrophoresis, the basepairs of the amplified segments were juxtaposed to the ladder, and acknowledged as successful after comparison.
7/14- Samir
The 7 CBCR gblocks that were successfully amplified (NPF, CBCR 4, CBCR 6, CBCR 9, CBCR 11, OSC, Cha, and Cyan 2) were gel extracted and Nanodropped to establish concentrations.
7/15- Amanda
On 7/13, CBCR 4, CBCR 6, CBCR 8, CBCR 9, CBCR 11, OSC 7112, GAG 2, GAG 3, Cyan, Cha, and NPF CBCR were PCR amplified. From these samples, GAG 2 and GAG 3 did not PCR amplify, however the others amplified successfully and was verified by gel electrophoresis juxtoposition to a DNA ladder. The extension time for these samples were increased by 20 seconds. Upon this adjustment, amplification appeared successful upon implementation of a Gel Electrophoresis on these samples. All other reagent concentrations remained unchanged from the prior experimentation.
7/15-Brent
Made colonies of Bacillus subtillus by placing stock colonies on LB media and glycerol stock-- allowing them to grow for multiple days. Prepared PCY79, Spo0A3, and KO7 Bacillus strains for the UC Santa Cruz iGEM team to use. These strains were propagated and distributed to the UCSC iGEM team. We asked Dr. Daniel Zeigler for permission to distribute these strains and obtained written permission for this distribution on July 14th. Competence Preparation and Transformation protocols for Bacillus subtilis was also shared with the UCSC iGEM team as part of our collaboration.
A Cryofreeze Procedure was implemented as follows:
A single colony from Bacilus subtilis was isolated from each strain type: Spo0A3, PCY79, and KO7.
Single colonies were placed into a 3 ml LB media solution with no antibiotics within a test tube. This culture was grown up at 37 deg. C. at approximately 180 rpm, in a MaxQ 8000 thermal cycler.
A control was prepared with LB media only (no bacteria)
Each culture was grown up until the LB media appeared to be sufficiently cloudy.
Cryofreeze tubes were obtained from each culture, ensuring 2 ml total of LB-Bacillus culture and Glycerol combined made it into each Cryofreeze tube. 312.5 microliters of 80% glycerol stock was combined with 1,687.5 microliters of LB-Bacillus culture such that glycerol would comprise 12.5% of the final 2 ml solution.
7/15
Team completed CITI (Collaborative Institutional Training Initiative) training and is certified to conduct consumer surveys.
7/18- Jon
Initial Golden Gate Reaction on (date )looked incomplete but, we decided to transform it anyways. Reaction product transformed into bHS alpha and left overnight.
7/19- Jon
Seven colonies grew on plate from 7/18. Need to colony PCR to confirm success. We are skeptical of success and believe it is ultimately a failure.
7/19- Brent
Amplified RBS (ab), RBS (cd), and RBS (kl)
Preformed gel electrophoresis on Bacillus amplification from 7/13 and today’s amplifications. Npf and B0015 appear to be correct (___bp) ; however, the RBS fall below ladder minimum at 16bp so the successfulness cannot be determined.
(Insert gel)
7/20- Amanda & Jon
Attempted first Golden Gate reaction on CBCR 4, 9, and 11 with ho, pcyA, RBS (34AB and 34BC), terminator, and PSB1C3.
Ran gel to find that gel shows 4-5 bands. Need to hypothesize reasons for why reaction is inefficient.
(Insert gel)
7/20- Amanda
Gel purified the previously amplified CBCRs 2,3,and 8
7/21 - Jon
Transformed the four CBCRs from 7/20 into DHS alpha anyways. Plates turned out negative -- need to reevaluate golden gate methods.
7/21 - All
Met with Monsanto at their Monsanto Seminis site in Woodland. We were hosted on a tour by the company in order to hear about the molecular and genetic biology they perform with plant research. We were provided with $1,000 for our iGEM team, no strings attached, from the company and were provided with opportunities to continue engaging with the company for further research and/or assistance with our iGEM project. We were taken on a day-tour of their facilities to learn about their operations and draw connections to the scope of iGEM.
7/23- Jon
Freezer stocks of collaborators colony strains were prepped and put in the freezer.
A lot of colonies formed on plates from 7/21 which makes us suspicous there is contamination. We decided to run a colony PCR on npf colony to try and find results. We discovered it was once again a false positive. Need to brainstorm why this could be.
7/24- Samir
Repeat PCR amplification on CBCR gblocks so we can have a higher concentration of DNA. All amplifications were unsuccessful-- hypothesized to be due to denatured DNA polymerase.
(Insert gel)
7/25- Amanda & Jon
Prepared cell culture of collaborators cells so that we can mini prep them tomorrow.
7/25 - Jon
Third Golden Gate reaction performed on npf, pcyA, RBS1, RBS2, and the plasmid. Allowed to incubate overnight at 16℃.
7/25- Samir & Brent
PCR amplified CBCR 3,4,6, and 8. All were successful and determined to PCR clean-up on CBCR 6 and 8 because they were cleaner bands and perform a gel extraction on CBCR 3 and 4.
(Insert gel)
7/26- Samir and Brent
PCR amplified CBCR 9, 11, ocs, ga_2, ga_3, cyan, cha, and npf. All successful-- determined PCR cleanup for npf and gel extraction for the rest.
(Insert gel)
7/26- Jon
Gel results from golden gate reaction on 7/25 hardly show any ligation products. We believe this is because the low DNA concentrations. We still transformed DHSalpha and plated to look for results tomorrow.
Also, minipreped collaborators plasmids containing PCB+BV for Amanda
Also, sent false positive colony from (date) for sequencing.
(Insert gel)
7/27-Amanda
To grow collaborators plasmids, we made the plate broth including the relevant antibiotics (carbenicillin and kanamycin). Left overnight to set up.
7/27- Brent & Samir
Preformed gel extraction on CBCR 6, 8, 9 and 11 from the amplification on 7/26.
7/28- Jon
Made a broth with antibiotics containing:
475 ml H20
5 g Tryptone
5 g NaCl
2.5 g Yeast Extract
7.5 g Agar
The broth was stirred using a stir bar until dissolved. H20 was added until the total volume reached a volume of 50 ml and a pH of ~ 7.0 was verified. This solution was covered with foil and tape and autoclaved at 121 degrees Celcius on a ‘sterilize and dry’ cycle.
LB plates with Kanamycin and Carbenicillin were made.
HO and PcyA colonies were produced for miniprep.
A transformation protocol for E. coli called the Denter protocol was implemented as follows:
0.5  μl DNA for transformation
1 μl T4 buffer
0.5 μl BSA1
0.5 μl T4 Ligase
1 μl 10x BSA
Addition of H20 intil 10 μl volume reached.
Cycling Protocol:
40 deg. C for 2 minutes
20 deg. C for 5 minutes
Repeat (1+2) 25x
Then 60 deg. C for 10 minutes
Finally 85 deg. C for 10 minutes
Results: Transformation successful - colonies grew on all plates and were resistant to all antibiotics used, indicating successful transformation. Negative controls were plated with each of Nathan’s individual strains onto the Kanamycin and Carbenicillin prepared plates.
7/29- Brent
Preformed gel extraction on ___
7/29- Amanda
Model finished and developed functional web app to run it.
August
8/1- Amanda
Prepared 25mL culture of npf colony from (date) in TB with kanamycin and carb.. . After 2 hrs of incubation, 2mL of 1M arabinose was added to induce cell growth.
8/6 - All
Our UC Davis iGEM Team completed all of our IRB approved Human Practice surveys at the city of Davis Farmers Market.
8/6 - Brent
Performed a variety of Golden Gate techniques in an effort to synthesize the Bacillus Genetic Construct. For reference, the complete DNA assembly should appear as follows (excluding the insertion plasmid pHCMC04):
Golden gates were performed to construct pVEG through HO, HO through PcyA, PcyA through the first terminator B0015, and the last RBS through the last terminator B0015.
The results of a Gel Electrophoresis illustrate that the HO through PcyA and PcyA through the first terminator B0015 was successful. An amplification was performed with each and turned our successful.
8/10- Amanda
Prepared five buffer solutions in preparation for protein purification.
Also, prepared plates containing kanamycin and chloramphenicol for upcoming transformations of CBCR plasmids and PCB+BV plasmid.
8/11 - Brent
The following assembly for Bacillus subtilis was attempted:
Gel electrophoresis indicates that this Golden Gate reaction worked, thus completing the setup:
Later results from this month illustrate that pVEG did not assemble with this complex appropriately, this only the first RBS through the first terminator B0015 was assembled appropriately.
8/12/2016
For Bacillus subtilis the following Golden Gate reactions were performed:
Results were not promising from the Gel Electrophoresis as to if these assembles appropriately came together:
pHCMC04 was chosen as the plasmid for our Bacillus genetic construct to be inserted into. This vector yields compatibility with both E. Coli and Bacillus. This vector contained a Bsal site which was corrected for. A PCR cleanup was performed on this vector after performing site-directed-mutagenesis with specially designed primers. 1 μl of DPN1 and 2.2 μl of cutsmart was added to get rid of methylated DNA (containing the Bsal site). The vector was incubated with these reagents at 37 degrees celsius for 1 hour.
8/12 - Brent
A variety of golden gate reactions were performed for Bacillus subtils in an attempt to continue dna assembly with amplified segments as follows:
Gel electrophoresis results indicated below:
8/15 - Brent
A pHT43 vector was order and primers were designed to clone out IPTG from this vector. PCR was performed, along with a gel electrophoresis to verify that the IPTG inducible promoter was cloned out of this vector. PCR cleanup was implemented to purify this clone out.
A variety of Bacillus transformation protocols were researched for potential bacillus genetic transformation routes.
The following protocols were obtained through literature search:
Transformation of Bacillus subtilis - Klein et al. 1992
Transformation of Bacillus subtilis by Electroporation - Michele Stephenson and Paul Jarrett (Institute of Horticultural Research)
High transformation efficiency of Bacillus subtilis with integrative DNA using glycene betaine as osmoprotectant - Fatma Meddeb-Mouelhi, Carlos Dulcey, Marc Beauregard.
A modified electro-transformation method for Bacillus subtilis and its application in the production of antimicrobial lipopeptides - Richard et al.
(Groningen Method) Preparation of competent B. subtilis cells and transformation with plasmid DNA - the ‘Gronigen Method’)
(Paris Method) Preparation of competent B. subtilis cells and transformation with plasmid DNA - the ‘Paris Method’)
8/17 - Brent
An issue was identified with the RBS of Bacillus subtilis - a base pair sequence was missing which would affect the golden gate assembly of the final RBS in the segment:
This error was corrected by ordering a new sequence from IDT. This issue, which was identified through examination of the basepair sequences on the ordering sheets, may explain the failed assemblies encountered from the previous weeks.
8/22 - Brent
The following Golden Gate reactions were performed:
Golden Gate Reaction B: (CBCR @ 1,451 bp + B0015 @ 155 bp) = 1,606 bp
Golden Gate Reaction ε : (CBCR @ 1,451 bp + B0015 @ 155 bp) = 1,606 bp
Golden Gate Reaction B-1: (IPTG @ 107 bp + RBS [KL] @ 21 bp + CBCR @ 1,451 bp + B0015 @ 155 bp) = 1,734 bp
Golden Gate Reaction ε-1 : (*Cloned IPTG @ 162 bp + RBS [KL] @ 21 bp + CBCR @ 1,451 bp + B0015 @ 155 bp) = 1,789 bp
Results indicate successful Golden Gate assembly.
8/25- Brent
A variety of unsuccessful Golden Gate assemblies were performed - specifics omitted for brevity:
September
9/01 - Brent
The following Golden Gate reactions were performed, isolated, amplified, and successfully extracted:
An image of the gel-electrophoresis is indicated here:
9/02 - Brent
For Bacillus subtilis, amplification of the following construct was performed:
Amplification Golden Gate Reaction Part 1: (pVEG @ 263 bp + RBS[AB] @ 21 bp + HO @ 720 bp + RBS[BC] @ 21bp + Pcya @ 750 bp + B0015 @ 155 bp) = 1,930 bp
This segment would not amplify, contrary to gel electrophoresis results which suggested that the 1,930 base-pair construct was appropriately assembled. Something went wrong with the assembly since pVEG through B0015 would not amplify. A new route of synthesis was performed in an effort to correct this and achieve expression:
Amplifications of these segments were named: CAS, FBI, FDA, FOR, GEM, GRE, HAT, and SAT arbitrarily. Each name denotes the assembly technique above, such as CAS being an amplification of GG rxn 006, and FDA being an amplification of GG rxn 777, for example. The following gel was obtained:
Gel extractions were performed with the following: FBI, FOR, and GEM where FBI includes an amplification of the Golden Gate of the first RBS to the first terminator, and FOR/GEM includes an amplification of HO through the first terminator.
Effectively, the following was obtained:
9/09 - Brent
A variety of primers were designed in order to amplify the Bacillus gene construct and in anticipation of it’s insertion in the pHCMC04 vector. The designs were performed as follows:
A Golden Gate assembly was performed using the DNA amplifications of previous constructs. The assembly was performed with alternate variations of the following:
These sequences all came together successfully and were PCR amplified.
9/15 - Brent
Protocol from UCSC - Bacillus subtilis competent preparation:
The following protocol was implemented, as shared by the UCSC iGEM team. Protocol No. 4308 915.504 - 08/2003
A solution of 0.5 M sorbitol, 0.5 M mannitol, and 10% glycerol was made. This medium serves as the washing solution and the electroporation solution.
Competent preparation of Bacillus subtilis was performed as follows:
Overnight cultures of Bacillus subtilis, K07, PY79, and spo0A3 were produced with an optical density of 0.85-0.95.
Cells were chilled on ice-water for 10 minutes and harvested by centrifugation at 4 degrees celcius and 5000 x g for 5 minutes.
Cells were washed four times on ice-cold electroporation medium.
Cells were suspended in glycerol-LB medium for storage in the -80 degree Celcius freezer.
Competent cells, in theory, were thus produced.
9/26 - Brent
A variety of PCRs were performed on the E. Coli CBCR’s 2, 3, 4, 8, NPF, G2, and G3. All PCR’s were successful as illustrated below. A gel extraction and PCR cleanup was performed with all of these CBCR’s.
October
Contact us at: ucdigem@gmail.com
