<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AFCM-Egypt_Bioinformatics_Page_link skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AFCM-Egypt/Bioinformatics Page link</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:AFCM-Egypt" style="background-color: #585761;
">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Team" style="
    background-color: #585761;
">TEAM
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Team">Meet The Team</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Description" style="
    background-color: #585761;
">Project 
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Description">Project Description</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Bioinformatics_Page_link">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Metagenomics">Metagenomics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Machine Learning">Machine Learning</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Human_Practices" style="
    background-color: #585761;
">Human Practices
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/HP/Gold_Integrated">Integrated</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Judging" style="
    background-color: #585761;
">Judging
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Medal">Medals</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Parts" style="
    background-color: #585761;
">Parts
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Contributions">Contributions</A></LI></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="banner"><DIV class="banner-heading"><H1>Bioinformatics</H1></DIV></DIV><DIV class="column full_size"><P style="text-align: center;"> </P><DIV class="content"><H4>Computational Anti-Cancer Immune stimulation using Tumor Cell RNAs CAIS-TR</H4><P>Manual Curation of miRNA networks and Network analysis</P><P>We have constructed a miRNA-disease network for colorectal cancer through mirnet [1] tool which integrates data from eleven other miRNA databases according to the latest update of exosomal and tissue-specific miRNAs. The constructed network [figure-1] has been analysed using cytoscape [2] for significant interaction according to the curated attributes of statistical significance.
The analysis included curation of published evidence as filtering attribute in addition to statistical significance which suggested miRNAs (mir-134 and mir-370) to be of very significant connection to colorectal cancer network and by further literature research we found a potential tumor suppressor role for both RNA molecules in CRC carcinogenesis by regulating EGFR signaling cascade through targeting PIK3CA (Phosphatidylinositol-4,5-Bisphosphate 3-Kinase Catalytic Subunit Alpha) and EGFR (Epidermal growth factor receptor) [3].
 (Figure 1).
</P><DIV class="center"><DIV>Figure 1: Colorectal cancer constructed network, analyzed using cytoscape</DIV></DIV><P>MiRNA-mRNA interaction networks
String database [4] has been mined using RAIN tool [5] in order to construct interaction networks for both miRNAs (mir-370-3p and mir-134-5p) to be used for further pathway analysis [Figure-2, Figure-3].</P><DIV class="center"><DIV>Figure 2: String database interaction network of mir-134-5p and target proteins</DIV></DIV><DIV class="center"><DIV>Figure 2: String database interaction network of mir-370-3p and target proteins</DIV></DIV><DIV class="center"><DIV>Figure 2: Cytoscape constructed networks for pathway analysis of miRNAs target mRNA list</DIV></DIV><DIV><P style="text-align: center;">Table-1 Significant pathways enriched in miRNAs target list from KEGG database</P><TABLE style="margin-left: auto; margin-right: auto;" width="0"><TBODY><TR><TD width="71"><P><STRONG>Index</STRONG></P></TD><TD width="169"><P><STRONG>Name</STRONG></P></TD><TD width="106"><P><STRONG>P-value</STRONG></P></TD><TD width="98"><P><STRONG>Adjusted p-value</STRONG></P></TD><TD width="72"><P><STRONG>Z-score</STRONG></P></TD><TD width="108"><P><STRONG>Combined score</STRONG></P></TD></TR><TR><TD width="71"><P>1</P></TD><TD width="169"><P>Colorectal cancer_Homo sapiens_hsa05210</P></TD><TD width="106"><P>0.00004363</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-1.96</P></TD><TD width="108"><P>19.65</P></TD></TR><TR><TD width="71"><P>2</P></TD><TD width="169"><P>ErbB signaling pathway_Homo sapiens_hsa04012</P></TD><TD width="106"><P>0.00003766</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-1.90</P></TD><TD width="108"><P>19.35</P></TD></TR><TR><TD width="71"><P>3</P></TD><TD width="169"><P>Pathways in cancer_Homo sapiens_hsa05200</P></TD><TD width="106"><P>0.00008735</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-2.00</P></TD><TD width="108"><P>18.72</P></TD></TR><TR><TD width="71"><P>4</P></TD><TD width="169"><P>MAPK signaling pathway_Homo sapiens_hsa04010</P></TD><TD width="106"><P>0.00008799</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-1.87</P></TD><TD width="108"><P>17.47</P></TD></TR><TR><TD width="71"><P>5</P></TD><TD width="169"><P>Bladder cancer_Homo sapiens_hsa05219</P></TD><TD width="106"><P>0.00006857</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-1.82</P></TD><TD width="108"><P>17.42</P></TD></TR><TR><TD width="71"><P>6</P></TD><TD width="169"><P>Renal cell carcinoma_Homo sapiens_hsa05211</P></TD><TD width="106"><P>0.00006887</P></TD><TD width="98"><P>0.002610</P></TD><TD width="72"><P>-1.81</P></TD><TD width="108"><P>17.36</P></TD></TR><TR><TD width="71"><P>7</P></TD><TD width="169"><P>Chronic myeloid leukemia_Homo sapiens_hsa05220</P></TD><TD width="106"><P>0.0001214</P></TD><TD width="98"><P>0.002701</P></TD><TD width="72"><P>-1.71</P></TD><TD width="108"><P>15.40</P></TD></TR><TR><TD width="71"><P>8</P></TD><TD width="169"><P>HIF-1 signaling pathway_Homo sapiens_hsa04066</P></TD><TD width="106"><P>0.0001108</P></TD><TD width="98"><P>0.002701</P></TD><TD width="72"><P>-1.68</P></TD><TD width="108"><P>15.32</P></TD></TR><TR><TD width="71"><P>9</P></TD><TD width="169"><P>Proteoglycans in cancer_Homo sapiens_hsa05205</P></TD><TD width="106"><P>0.001447</P></TD><TD width="98"><P>0.02575</P></TD><TD width="72"><P>-1.78</P></TD><TD width="108"><P>11.64</P></TD></TR><TR><TD width="71"><P>10</P></TD><TD width="169"><P>MicroRNAs in cancer_Homo sapiens_hsa05206</P></TD><TD width="106"><P>0.001268</P></TD><TD width="98"><P>0.02509</P></TD><TD width="72"><P>-1.72</P></TD><TD width="108"><P>11.44</P></TD></TR></TBODY></TABLE></DIV><DIV><P style="text-align: center;"> </P><P style="text-align: center;">Table-2 Significant pathways enriched in miRNAs target list from PANTHER database</P><TABLE style="margin-left: auto; margin-right: auto;" width="0"><TBODY><TR><TD width="71"><P><STRONG>Index</STRONG></P></TD><TD width="187"><P><STRONG>Name</STRONG></P></TD><TD width="88"><P><STRONG>P-value</STRONG></P></TD><TD width="98"><P><STRONG>Adjusted p-value</STRONG></P></TD><TD width="72"><P><STRONG>Z-score</STRONG></P></TD><TD width="108"><P><STRONG>Combined score</STRONG></P></TD></TR><TR><TD width="71"><P>1</P></TD><TD width="187"><P>p53 pathway feedback loops 2_Homo sapiens_P04398</P></TD><TD width="88"><P>0.001279</P></TD><TD width="98"><P>0.05882</P></TD><TD width="72"><P>-1.66</P></TD><TD width="108"><P>11.04</P></TD></TR><TR><TD width="71"><P>2</P></TD><TD width="187"><P>EGF receptor signaling pathway_Homo sapiens_P00018</P></TD><TD width="88"><P>0.006002</P></TD><TD width="98"><P>0.1034</P></TD><TD width="72"><P>-1.67</P></TD><TD width="108"><P>8.54</P></TD></TR><TR><TD width="71"><P>3</P></TD><TD width="187"><P>p53 pathway_Homo sapiens_P00059</P></TD><TD width="88"><P>0.006743</P></TD><TD width="98"><P>0.1034</P></TD><TD width="72"><P>-1.33</P></TD><TD width="108"><P>6.64</P></TD></TR><TR><TD width="71"><P>4</P></TD><TD width="187"><P>Interleukin signaling pathway_Homo sapiens_P00036</P></TD><TD width="88"><P>0.01305</P></TD><TD width="98"><P>0.1339</P></TD><TD width="72"><P>-1.32</P></TD><TD width="108"><P>5.73</P></TD></TR><TR><TD width="71"><P>5</P></TD><TD width="187"><P>Angiogenesis_Homo sapiens_P00005</P></TD><TD width="88"><P>0.01746</P></TD><TD width="98"><P>0.1339</P></TD><TD width="72"><P>-1.18</P></TD><TD width="108"><P>4.79</P></TD></TR><TR><TD width="71"><P>6</P></TD><TD width="187"><P>Toll receptor signaling pathway_Homo sapiens_P00054</P></TD><TD width="88"><P>0.01494</P></TD><TD width="98"><P>0.1339</P></TD><TD width="72"><P>-1.04</P></TD><TD width="108"><P>4.36</P></TD></TR><TR><TD width="71"><P>7</P></TD><TD width="187"><P>CCKR signaling map ST_Homo sapiens_P06959</P></TD><TD width="88"><P>0.03086</P></TD><TD width="98"><P>0.2028</P></TD><TD width="72"><P>-1.23</P></TD><TD width="108"><P>4.29</P></TD></TR><TR><TD width="71"><P>8</P></TD><TD width="187"><P>Ras Pathway_Homo sapiens_P04393</P></TD><TD width="88"><P>0.03649</P></TD><TD width="98"><P>0.2055</P></TD><TD width="72"><P>-0.95</P></TD><TD width="108"><P>3.14</P></TD></TR><TR><TD width="71"><P>9</P></TD><TD width="187"><P>T cell activation_Homo sapiens_P00053</P></TD><TD width="88"><P>0.04202</P></TD><TD width="98"><P>0.2055</P></TD><TD width="72"><P>-0.80</P></TD><TD width="108"><P>2.55</P></TD></TR><TR><TD width="71"><P>10</P></TD><TD width="187"><P>Integrin signalling pathway_Homo sapiens_P00034</P></TD><TD width="88"><P>0.08346</P></TD><TD width="98"><P>0.2953</P></TD><TD width="72"><P>-0.63</P></TD><TD width="108"><P>1.58</P><P> </P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>