Team:AFCM-Egypt/Bioinformatics Page link
Toggle navigation
Home
TEAM
Meet The Team
Attributions
Project
Project Description
Bioinformatics
Modeling
Results
Metagenomics
Machine Learning
Human Practices
Education & Public Engagement
Integrated
Collaborations
Judging
Medals
Collaborations
Parts
Parts
Improved Parts
Contributions
Bioinformatics
 
Computational Anti-Cancer Immune stimulation using Tumor Cell RNAs CAIS-TR
Manual Curation of miRNA networks and Network analysis
We have constructed a miRNA-disease network for colorectal cancer through mirnet [1] tool which integrates data from eleven other miRNA databases according to the latest update of exosomal and tissue-specific miRNAs. The constructed network [figure-1] has been analysed using cytoscape [2] for significant interaction according to the curated attributes of statistical significance. The analysis included curation of published evidence as filtering attribute in addition to statistical significance which suggested miRNAs (mir-134 and mir-370) to be of very significant connection to colorectal cancer network and by further literature research we found a potential tumor suppressor role for both RNA molecules in CRC carcinogenesis by regulating EGFR signaling cascade through targeting PIK3CA (Phosphatidylinositol-4,5-Bisphosphate 3-Kinase Catalytic Subunit Alpha) and EGFR (Epidermal growth factor receptor) [3].  (Figure 1).
Figure 1: Colorectal cancer constructed network, analyzed using cytoscape
MiRNA-mRNA interaction networks String database [4] has been mined using RAIN tool [5] in order to construct interaction networks for both miRNAs (mir-370-3p and mir-134-5p) to be used for further pathway analysis [Figure-2, Figure-3].
Figure 2: String database interaction network of mir-134-5p and target proteins
Figure 2: String database interaction network of mir-370-3p and target proteins
Figure 2: Cytoscape constructed networks for pathway analysis of miRNAs target mRNA list
Table-1 Significant pathways enriched in miRNAs target list from KEGG database
Index
Name
P-value
Adjusted p-value
Z-score
Combined score
1
Colorectal cancer_Homo sapiens_hsa05210
0.00004363
0.002610
-1.96
19.65
2
ErbB signaling pathway_Homo sapiens_hsa04012
0.00003766
0.002610
-1.90
19.35
3
Pathways in cancer_Homo sapiens_hsa05200
0.00008735
0.002610
-2.00
18.72
4
MAPK signaling pathway_Homo sapiens_hsa04010
0.00008799
0.002610
-1.87
17.47
5
Bladder cancer_Homo sapiens_hsa05219
0.00006857
0.002610
-1.82
17.42
6
Renal cell carcinoma_Homo sapiens_hsa05211
0.00006887
0.002610
-1.81
17.36
7
Chronic myeloid leukemia_Homo sapiens_hsa05220
0.0001214
0.002701
-1.71
15.40
8
HIF-1 signaling pathway_Homo sapiens_hsa04066
0.0001108
0.002701
-1.68
15.32
9
Proteoglycans in cancer_Homo sapiens_hsa05205
0.001447
0.02575
-1.78
11.64
10
MicroRNAs in cancer_Homo sapiens_hsa05206
0.001268
0.02509
-1.72
11.44
 
Table-2 Significant pathways enriched in miRNAs target list from PANTHER database
Index
Name
P-value
Adjusted p-value
Z-score
Combined score
1
p53 pathway feedback loops 2_Homo sapiens_P04398
0.001279
0.05882
-1.66
11.04
2
EGF receptor signaling pathway_Homo sapiens_P00018
0.006002
0.1034
-1.67
8.54
3
p53 pathway_Homo sapiens_P00059
0.006743
0.1034
-1.33
6.64
4
Interleukin signaling pathway_Homo sapiens_P00036
0.01305
0.1339
-1.32
5.73
5
Angiogenesis_Homo sapiens_P00005
0.01746
0.1339
-1.18
4.79
6
Toll receptor signaling pathway_Homo sapiens_P00054
0.01494
0.1339
-1.04
4.36
7
CCKR signaling map ST_Homo sapiens_P06959
0.03086
0.2028
-1.23
4.29
8
Ras Pathway_Homo sapiens_P04393
0.03649
0.2055
-0.95
3.14
9
T cell activation_Homo sapiens_P00053
0.04202
0.2055
-0.80
2.55
10
Integrin signalling pathway_Homo sapiens_P00034
0.08346
0.2953
-0.63
1.58
 
