<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AFCM-Egypt_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AFCM-Egypt/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:AFCM-Egypt" style="background-color: #585761;
">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Team" style="
    background-color: #585761;
">TEAM
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Team">Meet The Team</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Description" style="
    background-color: #585761;
">Project 
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Description">Project Description</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Bioinformatics_Page_link">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Metagenomics">Metagenomics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Machine Learning">Machine Learning</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Human_Practices" style="
    background-color: #585761;
">Human Practices
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/HP/Gold_Integrated">Integrated</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Judging" style="
    background-color: #585761;
">Judging
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Medal">Medals</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Parts" style="
    background-color: #585761;
">Parts
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Contributions">Contributions</A></LI></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="banner"><DIV class="banner-heading"><H1>Model</H1></DIV></DIV><DIV class="column full_size"><P style="text-align: center;"> </P><DIV class="content"><H4>Structural Modelling</H4><P>To evaluate the structural fitting of miRNAs into TLR-9 We started by constructing structural models for both miRNAs, we have used SimRNA [5] Tool for simulating tertiary structure of miRNA mir-134 nad mir-370 as SimRNA generates a circular starting conformation with the 5’ and 3’ ends close to each other as a starting structure for simulation. By building the internal list of restraints on dots-and brackets format, the default parameters for simulation energy and simulations steps have been set to 500 steps and 1% of the lowest energy frames taken to clustering, while the TLR-9 structure was cleaned out of PDB structures of code 5Y3K,5Y3J to build a model of human TLR-9 through Swiss-model.
</P><DIV class="center"><DIV>Figure 1: Simulated Model of miR-134 through SimRNA visualized using Pymol</DIV></DIV><DIV class="center"><DIV>Figure 2: Simulated Model of miR-370 through SimRNA visualized using Pymol</DIV></DIV><P>Nucleic acid Modelling</P><P> </P><P>For verification of our proposal for colibactin interference in carcinogenic strains of e.coli we tried to investigate the binding of dCas9 into the target DNA of colibactin genes [clbN Escherichia coli colibactin non-ribosomal peptide synthetase ClbN, Escherichia coli precolibactin peptidase ClbP and Colibactin hybrid non-ribosomal peptide synthetase/type I polyketide synthase ClbB] along with its designed guide RNA using benchling to interfere with the transcription of the whole gene cluster of colibactin. To build a final model of complementary guide RNA binded to target DNA of colibactin -studied through EBI samples of UCI121 strain- , we used   3D-DART along with Rosetta FARFAR package to generate a 3D structure of the target DNA representing the cleavage site and PAM of cas9 as well as PAM flexibility.</P><P> </P><P> </P><P> </P><P> </P><P>Docking protocol </P><P> </P><P>We have followed HADDOCK docking protocol, consisting of randomized orientations and rigid body energy minimization, calculating 1,000 complex structures, with HADDOCK scoring performed according to the weighted sum (HADDOCK score) of different energy terms which includes van der Waals energy, electrostatic energy, distance restraints energy, direct RDC restraint energy, intervector projection angle restraints energy, diffusion anisotropy energy, dihedral angle restraints energy, symmetry restraints energy, binding energy, desolvation energy and buried surface area. This lowest energy structure displayed no AIR restraint violations within 0.3 A˚  threshold and was accepted as the final docked structure for the complex.</P><P> </P><P> </P><P> </P><P>Results analysis</P><P>The graphs below represents water-refined models generated by HADDOCK where clusters are calculated based on the interface-ligand RMSDs calculated by HADDOCK, with the interface defined automatically based on all observed contacts. The various structural analysis (FCC, i-RMSD and l-RMSD) are made with respect to the best HADDOCK model (the one with the lowest HADDOCK score).</P><DIV><P style="text-align: center;"> </P><P style="text-align: center;"><SPAN style="text-decoration: underline;"><STRONG>Table-2 Docking scores of both complexes using HADDOCK</STRONG></SPAN></P><P style="text-align: center;"> </P><TABLE style="margin-left: auto; margin-right: auto;" width="0"><TBODY><TR><TD width="139"><P>Complex</P></TD><TD width="122"><P>TLR9-miR-134</P></TD><TD width="117"><P>TLR9-miR-370</P></TD><TD width="106"><P>dCas9/grna 1</P></TD><TD width="101"><P>Dcas9 grna2</P></TD><TD width="101"><P>Dcas9 grna3</P></TD></TR><TR><TD width="139"><P>HADDOCK score</P></TD><TD width="122"><P>-126.5+/- 6.8</P></TD><TD width="117"><P>-142.4 +/- 5.6</P></TD><TD width="106"><P>-119.9+/- 19.2</P></TD><TD width="101"><P>-87.7+/- 16.1</P></TD><TD width="101"><P>-83.8+/- 23.6</P></TD></TR><TR><TD width="139"><P>clusters</P></TD><TD width="122"><P>13</P></TD><TD width="117"><P>12</P></TD><TD width="106"><P>4</P></TD><TD width="101"><P>5</P></TD><TD width="101"><P>15</P></TD></TR><TR><TD width="139"><P>Total Interaction energy(Kcal mol-1)</P></TD><TD width="122"><P>0.5 +/- 0.3</P></TD><TD width="117"><P>1.0 +/- 0.1</P></TD><TD width="106"><P>3.8 +/- 0.1</P></TD><TD width="101"><P>1.3 +/- 0.1</P></TD><TD width="101"><P>13.1 +/- 0.1</P></TD></TR><TR><TD width="139"><P>Van der Waals energy(Kcal mol-1)</P></TD><TD width="122"><P>-57.3 +/- 5.6</P></TD><TD width="117"><P>-74.6 +/- 5.5</P></TD><TD width="106"><P>-85.2 +/- 2.3</P></TD><TD width="101"><P>-84.8+/- 6.7</P></TD><TD width="101"><P>-60.3 +/- 3.7</P></TD></TR><TR><TD width="139"><P>Electrostatic Energy(Kcal mol-1)</P></TD><TD width="122"><P>-498.0 +/- 13.2</P></TD><TD width="117"><P>-419.6 +/- 3.1</P></TD><TD width="106"><P>-718.1+/-24.8</P></TD><TD width="101"><P>-637.3+/-34.6</P></TD><TD width="101"><P>-597.7+/-40.3</P></TD></TR><TR><TD width="139"><P>Desolvation energy(Kcal mol-1)</P></TD><TD width="122"><P>22.2 +/- 4.4</P></TD><TD width="117"><P>13.9 +/- 7.6</P></TD><TD width="106"><P>61.2 +/- 18.7</P></TD><TD width="101"><P>62.7+/- 17.9</P></TD><TD width="101"><P>39.4+/- 21.2</P></TD></TR><TR><TD width="139"><P>Restraints violation energy(Kcal mol-1)</P></TD><TD width="122"><P>82.1 +/- 15.55</P></TD><TD width="117"><P>22.1 +/- 19.67</P></TD><TD width="106"><P>476.7+/- 24.14</P></TD><TD width="101"><P>618.6+/- 116.90</P></TD><TD width="101"><P>566.8+/- 147.70</P></TD></TR><TR><TD width="139"><P>Buried Surface area</P></TD><TD width="122"><P>1734.1 +/- 39.0</P></TD><TD width="117"><P>2116.0 +/- 87.2</P></TD><TD width="106"><P>2390.1 +/- 163.2</P></TD><TD width="101"><P>2276.8 +/- 223.2</P></TD><TD width="101"><P>1791.7 +/- 106.9</P></TD></TR><TR><TD width="139"><P>z-score</P></TD><TD width="122"><P>-1.6</P></TD><TD width="117"><P>-2.2</P><P> </P></TD><TD width="106"><P>-1.8</P></TD><TD width="101"><P>-1.1</P></TD><TD width="101"><P>-1.3</P></TD></TR></TBODY></TABLE></DIV><P>Generated Poses of Docking </P><DIV class="center"><DIV>Figure 4: Figure shows the best docking pose of miR-134 into TLR-9 (left) and its HADDOCK clusters RMSD graph (right)</DIV></DIV><P> </P><P> </P><P> </P><DIV class="center"><DIV>Figure 5: Figure shows the best docking pose of miR-370 into TLR-9 (left) and its HADDOCK clusters RMSD graph (right)</DIV></DIV><P> </P><P> </P><P> </P><P>Insights from structural docking of TLR-9 and miRNAs</P><P>Residues of large contact with both miRNAs constituted a large contact area to bind nucleic acids of both miRNAs at CpG rich motifs including residues; 248Val, 290Val, 312Val, 314Asp, 337Arg, 338Lys, 340Asn, 364Val, 365Ala, 367Lys, 392Met, and 397Arg.</P><DIV class="center"><DIV>Figure 6: Figure shows the best docking pose of dCas9 into target e.coli DNA of colibactin B  (left) and its HADDOCK clusters RMSD graph (right)</DIV></DIV><P> </P><P> </P><P> </P><DIV class="center"><DIV>Figure 8: Figure shows the best docking pose of dCas9 into target e.coli DNA of colibactin N (left) and its HADDOCK clusters RMSD graph (right)</DIV></DIV><P> </P><P> </P><P> </P><P>Cellular Transfection Modelling</P><P>The aim of this model is to provide us with mathematical representation of the proposed kinetics of our proposed lentiviral delivery system, deSolve R package was used for solving ordinary differential equations while FME package was used for fitting parameters based on our proposed model of interactions and dynamics between molecular species in the modeling environment represented in the following table and diagram. </P><DIV><P> </P><P style="text-align: center;"><STRONG>Table shows parameters and constants of our lentiviral transfection model</STRONG></P><TABLE style="margin-left: auto; margin-right: auto;" width="0"><TBODY><TR><TD width="208"><P>Specie/Parameter</P></TD><TD width="208"><P>Description</P></TD><TD width="208"><P>Unit</P></TD></TR><TR><TD width="208"><P>C</P></TD><TD width="208"><P>Uninfected CRC cell</P></TD><TD width="208"><P>Cells uL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>Clent</P></TD><TD width="208"><P>Lentiviral Infected CRC cell</P></TD><TD width="208"><P>Cells uL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>Cmir</P></TD><TD width="208"><P>Mir Transfected CRC cells</P></TD><TD width="208"><P>Cells uL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>Cap</P></TD><TD width="208"><P>Apoptotic CRC cells to mIRNA transfection</P></TD><TD width="208"><P>Cells uL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>Lent</P></TD><TD width="208"><P>Lentiviral vector</P></TD><TD width="208"><P>Focus Forming unit (FFU) uL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>miR</P></TD><TD width="208"><P>miRNA specie</P></TD><TD width="208"><P>Ng mL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>1</SUB></P></TD><TD width="208"><P>Intrinsic division rate of CRC cells</P></TD><TD width="208"><P>Day <SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>2</SUB></P></TD><TD width="208"><P>Rate of conversion of CRC cells to lentiviral transfected cells</P></TD><TD width="208"><P>Ng Day <SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>3</SUB></P></TD><TD width="208"><P>Rate of reversion from lentiviral transfected cells to mIRNA-expressing cell</P></TD><TD width="208"><P>Day <SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>4</SUB></P></TD><TD width="208"><P>Rate of CRC cells RNA induced-apoptosis</P></TD><TD width="208"><P>Day <SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>5</SUB></P></TD><TD width="208"><P>Rate of lentiviral infection</P></TD><TD width="208"><P>ul Day<SUP>-1</SUP> FFU<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>K<SUB>6</SUB></P></TD><TD width="208"><P>Lentivirus secretion from infected CRC cells</P></TD><TD width="208"><P>FFU cells Day<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>K<SUB>7</SUB></P></TD><TD width="208"><P>Lentivirus decay rate</P></TD><TD width="208"><P>day<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>K<SUB>8</SUB></P></TD><TD width="208"><P>Saturated rate of miRNA expression</P></TD><TD width="208"><P>Ng mL<SUP>-1</SUP></P></TD></TR><TR><TD width="208"><P>k<SUB>9</SUB></P></TD><TD width="208"><P>mIRNA specie decay rate</P></TD><TD width="208"><P>Day<SUP>-1</SUP></P></TD></TR></TBODY></TABLE></DIV><P>System of ordinary differential equations</P><DIV class="center"><DIV>This Circuit diagram represents our Model of lentiviral transfection</DIV></DIV><P> </P><P> </P><P> </P><P>System of ordinary differential equations</P><DIV class="center"><DIV>This plot shows the kinetics of our model reaching equilibrium after 2.5 time units on 0.01 framing rate</DIV></DIV><P> </P><P> </P><P> </P><P>Model Conclusion:</P><P>The time to equilibrium is highly dependent on the rate of transformation and not on the initial value of untransfected cells in the experiment. The value of miRNA expressing cells to miRNA reaches a simultaneous level of equilibrium with equilibration of vector production out of transfected cell. On successful trials of transfection the rate of apoptotic induction is dependent on the success of expressing the apoptotic mIRNA as well as its regulatory effect.</P><P>References</P><DIV><P>Cyril Dominguez, Rolf Boelens and Alexandre M.J.J. Bonvin. HADDOCK: a protein-protein docking approach based on biochemical and/or biophysical information. J. Am. Chem. Soc. 125, 1731-1737 (2003).</P><P> </P><P>Boniecki MJ, Lach G, Dawson WK, et al. SimRNA: a coarse-grained method for RNA folding simulations and 3D structure prediction. Nucleic Acids Research. 2016;44(7):e63.</P><P>Arnold, K., Bordoli, L., Kopp, J. and Schwede, T. (2006) The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. Bioinformatics, 22, 195-201.</P><P> Benkert, P., Kunzli, M. and Schwede, T. (2009) QMEAN server for protein model quality estimation. Nucleic Acids Res, 37, W510-514.</P><P>Das, R., Karanicolas, J., and Baker, D. (2010), &quot;Atomic accuracy in predicting and designing noncanonical RNA structure&quot;. Nature Methods 7:291-294.</P><P>Lyskov S, Chou FC, Conchúir SÓ, Der BS, Drew K, Kuroda D, et al. &quot;Serverification of Molecular Modeling Applications: The Rosetta Online Server That Includes Everyone (ROSIE)&quot;. PLoS One. 2013 May 22;8(5):e63906</P><P>Van Dijk M, Bonvin AMJJ. 3D-DART: a DNA structure modelling server. Nucleic Acids Research. 2009;37(Web Server issue):W235-W239.</P><P>Hospital A, Faustino I, Collepardo-Guevara R, González C, Gelpí JL, Orozco M. NAFlex: a web server for the study of nucleic acid flexibility. Nucleic Acids Research. 2013;41(Web Server issue):W47-W55.</P><P>Cyril Dominguez, Rolf Boelens and Alexandre M.J.J. Bonvin. HADDOCK: a protein-protein docking approach based on biochemical and/or biophysical information. J. Am. Chem. Soc. 125, 1731-1737 (2003).</P><P>Karline Soetaert, Thomas Petzoldt, R. Woodrow Setzer (2010). Solving Differential Equations in R: Package deSolve. Journal of Statistical Software, 33(9), 1--25.</P><P>Browne EP, Letham B, Rudin C (2016) A Computational Model of Inhibition of HIV-1 by Interferon-Alpha. PLoS ONE 11(3): e0152316.</P><P> </P><P>Soetaert, K. &amp; Petzoldt, T. (2010): Inverse modelling, sensitivity and Monte Carlo analysis in R using package FME. Journal of Statistical Software 33(3), 1-28.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>