Team:AFCM-Egypt/Metagenomics
Toggle navigation
Home
TEAM
Meet The Team
Attributions
Project
Project Description
Bioinformatics
Modeling
Results
Metagenomics
Machine Learning
Human Practices
Education & Public Engagement
Integrated
Collaborations
Judging
Medals
Collaborations
Parts
Parts
Improved Parts
Contributions
CRC Metagenomics
 
Metagenomics Analysis of colorectal cancer publicly available datasets
Using MGnify (EBI Metagenomics) analysis pipelines, our team managed to study the analysis of 4 studies about microbiota of colorectal cancer in the following table:
Analysis Summary
The 4 datasets have been analyzed using EBI metagenomics analysis pipeline 4.1 studying and revisualizing the analysis summary taxonomic assignments and phylum level taxonomies of small subunit rRNA as well as functional analysis through GO slim ontology and interpro matches.
A.  Analysis summary of all runs of study A “EMG produced TPA metagenomics assembly of the Reproducibility of associations between the human gut microbiome and colorectal cancer assessed in a patient population from Washington, DC, USA (Colorectal cancer and the human gut microbiome) data set.”
Figure: shows GO slim analysis BP, CC and MF of study A
Figure shows phylum taxonomy analysis plot of study A
Figure shows Korna plot of Taxonomy Abundances of Study A SSU rRNA
B. Analysis summary of all runs of study B “EMG produced TPA metagenomics assembly of the Metagenomic analysis of fecal microbiome as a tool towards targeted non-invasive biomarkers for colorectal cancer (Metagenomic analysis of fecal microbiome as a tool towards targeted non-invasive biomarkers for colorectal cancer) data set”
Figure shows GO slim analysis BP, CC and MF of study B
Figure shows phylum taxonomy analysis plot of study B
Figure shows Korna plot of Taxonomy Abundances of Study B LSU rRNA
C. Analysis summary of all runs of study C “Human gut environment Targeted loci environmental”
Figure shows phylum taxonomy analysis plot of study C
Figure shows Korna plot of Taxonomy Abundances of Study C SSU rRNA
D. Analysis summary of all runs of study D “DNA from FIT can replace stool for microbiota-based colorectal”
Figure shows phylum taxonomy analysis plot of study D
Figure shows Korna plot of Taxonomy Abundances of Study D SSU rRNA
6.Measuring the expression of miR-134 in transformed RKO cell line
To confirm our findings, we measured the expression of miR-134 in transformed cells (with the transfer vector carrying mi!-134 as well as the empty vector backbone). After total RNA extraction, reverse transcription and RealTime PCR, we found that our miRNA showed significant overexpression in the transformed cell lines indicating the success of our experiment.
Conclusions of Metagenomics Analysis
This extensive meta-analysis provided us with extremely important data about colorectal cancer regarding changes of microbiota environment of the colon and its excretory products. For instance, we have identified the abundance of  Some bacteria, such as Butyricicoccus, E. coli, and Fusobacterium, which can be used as potential biomarkers for normal, adenoma, and cancer groups, respectively.
Regarding E.coli, we have found key signatures that denotes lower levels of E. coli in cancer patients (6.76%) than in health controls (18.80%) and adenoma patients (22.33%), which made E.coli a potential candidate for adenoma-associated biomarker which could be explained by altered host immunogenicity to bacterial products, mutagenesis or gene transfer in which all these pathogenic acts could lead to a typical driver-passenger model for CRC by colonizing-related adenomas that could favor carcinomatous transformation and by altering the microenvironment of the tumor then fade away by passenger bacteria which explains lower levels of E.coli in carcinoma samples versus adenomas.
References
EBI Metagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies ,Mitchell AL, Scheremetjew M, Denise H, Potter S, Tarkowska A, et al. 2017 46 (PMID: 29069476) (DOI: 10.1093/nar/gkx967)
https://www.ebi.ac.uk/ena/data/view/PRJNA318004
Gut mucosal microbiome across stages of colorectal carcinogenesis, Nakatsu G, Li X, Zhou H, Sheng J, Wong SH, et al. 2015 6 (PMID: 26515465 ) (DOI: 10.1038/ncomms9727 )
Analysis of Mucosa-Associated Microbiota in Colorectal Cancer, Xu K, Jiang B. 2017 23 (PMID: 28904330 )
https://www.ebi.ac.uk/metagenomics/
