<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AFCM-Egypt_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AFCM-Egypt/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:AFCM-Egypt" style="background-color: #585761;
">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Team" style="
    background-color: #585761;
">TEAM
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Team">Meet The Team</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Description" style="
    background-color: #585761;
">Project 
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Description">Project Description</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Bioinformatics_Page_link">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Metagenomics">Metagenomics</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Machine Learning">Machine Learning</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Human_Practices" style="
    background-color: #585761;
">Human Practices
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Public_Engagement">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/HP/Gold_Integrated">Integrated</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Judging" style="
    background-color: #585761;
">Judging
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Medal">Medals</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-hover="dropdown" href="https://2018.igem.org/Team:AFCM-Egypt/Parts" style="
    background-color: #585761;
">Parts
						</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:AFCM-Egypt/Contributions">Contributions</A></LI></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="banner"><DIV class="banner-heading"><H1>Results, Protocols and Lab Notes</H1></DIV></DIV><DIV class="column full_size"><P style="text-align: center;"> </P><DIV class="content"><H4>1.	Parts Synthesis, Restriction and Ligation</H4><P>We began building our construct by first synthesizing our parts and digesting them using IGEM plasmid pS1BC3 compatible restriction enzymes XbaI and PstI.as shown in figures 1-3. </P><DIV class="center"><H4>Ligation protocol</H4><P> 
XbaI  Order R0145

PstI  Order R0140

Restriction Enzyme Double Digestion
Steps
Set up reaction as follows:
COMPONENT	50 µl REACTION
DNA	1 µg
10X NEBuffer 3.1	5 µl (1X)
XbaI	1.0 µl (or 10 units)
PstI	1.0 µl (or 10 units)
Nuclease-free Water	to 50 µl
Incubate at 37°C for 5-15 minutes as both enzymes are Time-Saver qualified.</P><DIV class="center"><DIV>Figure 1: Figure (1): Agarose gel electrophoresis of IDT synthesized parts after digestion with IGEM plasmid pS1BC3-compatible restriction enzymes XbaI and PstI</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2a: Part BBa_K2534023 (CME E CMV P) gblock undigested fragment</P><P>Lane 2b: Part BBa_K2534023 (CME E CMV P) gblock digested fragment</P><P>Lane 3: Part BBa_K2534007 ( AMPp+AMPR )gblock digested fragment</P><P>Lane 4a: Part BBa_K2534011 (B glob intron + vsv G + poly a) gblock undigested fragment</P><P>Lane 4b: Part BBa_K2534011 (B glob intron + vsv G + poly a) gblock digested fragment</P><P>Lane 5a: Part BBa_K2534012 (HIV gag) gblock undigested fragment</P><P>Lane 5b: Part BBa_K2534012 (HIV gag) gblock digested fragment</P><P>Lane 6a: Part BBa_K2534013 (HIV pol) gblock undigested fragment</P><P>Lane 6b: Part BBa_K2534013 (HIV pol) gblock digested fragment</P><P>Lane 7a: Part BBa_K2534014 (RRE) gblock undigested fragment</P><P>Lane 7b: Part BBa_K2534014 (RRE) gblock digested fragment</P><P>Lane 8a: Part BBa_K2534015 (SV40p) gblock undigested fragment</P><P>Lane 8b: Part BBa_K2534015 (SV40p) gblock digested fragment</P></BLOCKQUOTE></DIV><DIV class="center"><DIV>Figure 1: Figure (1): Agarose gel electrophoresis of IDT synthesized parts after digestion with IGEM plasmid pS1BC3-compatible restriction enzymes XbaI and PstI</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 9a: Part BBa_K2534017 (WRPE) gblock undigested fragment</P><P>Lane 9b: Part BBa_K2534017 (WRPE) gblock digested fragment</P><P>Lane 10a: Part BBa_K2534018 (KOZAK EGFP) gblock undigested fragment</P><P>Lane 10b: Part BBa_K2534018 (KOZAK EGFP) gblock digested fragment</P><P>Lane 11a: Part BBa_K2534022 (TetR) gblock undigested fragment</P><P>Lane 11b: Part BBa_K2534022 (TetR) gblock digested fragment</P></BLOCKQUOTE></DIV><DIV class="center"><DIV>Figure (3): Agarose gel electrophoresis of IDT synthesized parts after digestion with IGEM plasmid pS1BC3-compatible restriction enzymes XbaI and PstI</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2a: Part BBa_K2534016 (HIV LTR) gblock undigested fragment</P><P>Lane 2b: Part BBa_K2534016 (HIV LTR) gblock digested fragment</P><P>Lane 3a: Part BBa_K2534020 (CCP/PPt) gblock undigested fragment</P><P>Lane 3b: Part BBa_K2534020 (CCP/PPt) gblock digested fragment</P><P>Lane 4a: Part BBa_K2534019 (HV PSI) gblock undigested fragment</P><P>Lane 4b: Part BBa_K2534019 (HIV PSI) gblock digested fragment</P><P>Lane 5a: Part BBa_K2534021 (Tet op) gblock undigested fragment</P><P>Lane 5b: Part BBa_K2534021 (Tet op) gblock digested fragment</P></BLOCKQUOTE></DIV><H4>2.MiniPrep, Ligation and characterization of Parts ligated to IGEM Plasmid pS1BC3</H4><P>We then performed our plasmid MiniPrep to ligate and characterize our parts to the plasmid pS1BC3 (figures 4-6).</P><H4>MiniPrep protocol</H4><P> The QIAprep Spin Miniprep Kit (cat. nos. 27104 and 27106) can be stored at room
temperature (15–25°C) for up to 12 months.
For more information, please refer to the most recent version of the QIAprep Miniprep
Handbook, which can be found at: www.qiagen.com/handbooks.
For technical assistance, please call toll-free 00800-22-44-6000, or find regional phone
numbers at: www.qiagen.com/contact.
Notes before starting
- Optional: Add LyseBlue reagent to Buffer P1 at a ratio of 1 to 1000.
- Add the provided RNase A solution to Buffer P1, mix and store at 2–8°C.
- Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume).
- All centrifugation steps are carried out at 13,000 rpm (~17,900 x g) in a conventional
table-top microcentrifuge.

1. Pellet 1–5 ml bacterial overnight culture by centrifugation at &gt;8000 rpm (6800 x g) for 3
min at room temperature (15–25°C).
2. Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a microcentrifuge
tube.
3. Add 250 μl Buffer P2 and mix thoroughly by inverting the tube 4–6 times until the
solution becomes clear. Do not allow the lysis reaction to proceed for
more than 5 min. If using LyseBlue reagent, the solution will turn blue.

4. Add 350 μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6
times. If using LyseBlue reagent, the solution will turn colorless.
5. Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
6. Apply 800 μl supernatant from step 5 to the QIAprep 2.0 spin column by pipetting. For
centrifuge processing, follow the instructions marked with a triangle (). For vacuum
manifold processing, follow the instructions marked with a circle ().  Centrifuge for
30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the
solution through the QIAprep 2.0 spin column and switch off the vacuum source.
7. Recommended: Wash the QIAprep 2.0 spin column by adding 0.5 ml Buffer PB. 
Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold
to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum
source.
Note: This step is only required when using endA+ strains or other bacteria strains with
high nuclease activity or carbohydrate content.
8. Wash the QIAprep 2.0 spin column by adding 0.75 ml Buffer PE.  Centrifuge for
30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the
solution through the QIAprep 2.0 spin column and switch off the vacuum source. Transfer
the QIAprep 2.0 spin column to the collection tube.
9. Centrifuge for 1 min to remove residual wash buffer.
10.Place the QIAprep 2.0 column in a clean 1.5 ml microcentrifuge tube. To elute DNA,
add 50 μl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAprep 2.0
spin column, let stand for 1 min, and centrifuge for 1 min.
11.If the extracted DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5
volumes of purified DNA. Mix the solution by pipetting up and down before loading the
gel.</P></DIV><DIV class="center"><H4>Ligation protocol</H4><P>Set up the following reaction in a microcentrifuge tube on ice.
(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) Use NEBioCalculator to calculate molar ratios.
COMPONENT	20 μl REACTION
T4 DNA Ligase Buffer (10X)*	2 μl
Vector DNA (4 kb)	50 ng (0.020 pmol)
Insert DNA (1 kb)	37.5 ng (0.060 pmol)
Nuclease-free water	to 20 μl
T4 DNA Ligase	1 μl* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10 minute ligation).
Heat inactivate at 65°C for 10 minutes.
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.</P>&lt;&gt;/div
<DIV class="center"><DIV class="center"><DIV>Figure (4): Agarose gel electrophoresis of IDT synthesized parts after ligation with IGEM plasmid pS1BC3 compatible restriction enzymes XbaI and PstI restriction enzymes</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2: Part BBa_K2534023 (CME E CMV P) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 3: Part BBa_K2534007 (AMPp+AMPR) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 4: Part BBa_K2534011 (B glob intron + vsv G + poly a) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 5: Part BBa_K2534012 (HIV gag) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 6: Part BBa_K2534018 (KOZAK EGFP) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 7: Part BBa_K2534022 (TetR) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 8: Part BBa_K2534014 (RRE) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 9: Part BBa_K2534011 (B glob intron + vsv G + poly a) gblock religated to IGEM plasmid pS1BC3</P><P>Lane 10: Part BBa_K2534011 (B glob intron + vsv G + poly a) gblock religated to IGEM plasmid pS1BC3</P><P>Lane 11: Part BBa_K2534012 (HIV gag) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 12: Part BBa_K2534012 (HIV gag) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 13:  IGEM plasmid backbone pSB1C3</P></BLOCKQUOTE></DIV><DIV class="center"><DIV>Figure (5): Agarose gel electrophoresis of IDT synthesized parts after ligation with IGEM plasmid pS1BC3 compatible restriction enzymes XbaI and PstI restriction enzymes</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2: Part BBa_K2534012 (HIV gag) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 3: Part BBa_K2534012 (HIV gag) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 4: Part BBa_K2534013 (HIV pol) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 5: Part BBa_K2534014 (RRE) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 6: Part BBa_K2534014 (RRE) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 7: Part BBa_K2534015 (SV40p) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 8: Part BBa_K2534016 (HIV LTR) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 9: Part BBa_K2534016 (HIV LTR) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 10: Part BBa_K2534017 (WRPE) gblock ligated to IGEM plasmid pS1BC3</P><P>Lane 11: Part BBa_K2534014 (RRE) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 12: Part BBa_K2534014 (RRE) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 13: Part BBa_K2534014 (RRE) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 14: Part BBa_K2534014 (RRE) gblock re-ligated to IGEM plasmid pS1BC3</P><P>Lane 15: DNA 1000 bp Ladder MW marker</P></BLOCKQUOTE></DIV><DIV class="center"><DIV>Figure (6): Agarose gel electrophoresis of IDT synthesized parts after ligation with IGEM plasmid pS1BC3-compatible restriction enzymes XbaI and PstI restriction enzymes</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2: IGEM plasmid backbone pSB1C3</P><P>Lane 3: Part BBa_K2534016 (HIV LTR) ligated to pSB1C3</P><P>Lane 4: Part BBa_K2534020 (CCP/PPt) ligated to pSB1C3</P><P>Lane 5: Part BBa_K2534019 (HIV PSI) ligated to pSB1C3</P><P>Lane 6: Part BBa_K2534021 (Tet op) ligated to pSB1C3</P></BLOCKQUOTE></DIV><H4>3.miRNA Ligation to IGEM pS1BC3 Assembly after being synthesized using IDT offer</H4><P>We then ligated our selected miRNA’s (miR-134) parts to the pS1BC3 backbone (figures 7-9).</P><DIV class="center"><DIV>Figure (7): Agarose gel electrophoresis of IDT synthesized miRNAs after ligation with IGEM plasmid pS1BC3</DIV><BLOCKQUOTE><P>Lane 1: DNA 100 bp Ladder MW marker</P><P>Lane 2: miR-134 ligated to IGEM plasmid pS1BC3</P><P>Lane 3: miR-370 ligated to IGEM plasmid pS1BC3</P></BLOCKQUOTE></DIV><DIV class="center"><H4>4.Competent E.coli Transformation</H4><P>Our first transformations were a success! We managed to transform our competent cell colonies with our synthesized lentiviral transfer vector carrying our miRNA of interest (miR-134). With these cell we will be able to make more copies of each part in preparation for transformation (figures 8 &amp; 9).
	</P></DIV><DIV class="center"><H4>Transformation protocol</H4><P> - Thaw competent cells on ice.
- Chill approximately (15 μl) of the ligation mixture in a 1.5 ml microcentrifuge tube.
- Add 50 µl of competent cells to the DNA. Mix gently by pipetting up and down or flicking
the tube 4–5 times to mix the cells and DNA. Do not vortex.
- Place the mixture on ice for 30 minutes. Do not mix.
- Heat shock at 42°C for 90 seconds. Do not mix.
- Add 950 µl of room temperature media* to the tube.
- Place tube at 37°C for 2h. Shake vigorously (250 rpm) or rotate.
- Warm selection plates to 37°C.
- Spread 50–100 µl of the cells and ligation mixture onto the plates.
- Incubate overnight at 37°C.</P></DIV><DIV class="center"><DIV>Figure (8): Competent E.coli cells after transformation of synthesized transfer vector plasmids of a 3rd generation lentiviral delivery</DIV></DIV><DIV class="center"><DIV>Figure (9): Competent E.coli cells after transformation of synthesized transfer vector plasmids of a 3rd generation lentiviral delivery ligated with miR-134 that has been selected through bioinformatics analysis.</DIV></DIV><H4>5.Transfection and Viability Results</H4><P>48 h after transfection, marked decreases in RKO cell counts (figure 10) and viability (figure 11) were observed by inverted microscopy. All the experimental conditions induced a cell-death phenotype that could be easily distinguished from control, indicating cell death after transfection with our transfer vector carrying miR-134 vs. the empty vector backbone showing highly significant efficacy of our construct (p &lt;0.001) (Table 1).</P><DIV class="center"><H4>Transfection protocol</H4><P> For mirRNA lentiviral transfection, RKO cells (2 × 105) were cultured in six-well plates for 24 h and transfected with miRNA transfer vector using Oligofectamine (Invitrogen) at the indicated concentrations. Cells were lysed after 30 h for RNA isolation and after 72 h for protein levels. shRNA lentivirus production and infection conditions for RKO cells were optimized in six-well plates. Cells were seeded at a density of 2 × 105 per well in a six-well plate, incubated for 24 h, infected using 1 mL of shRNA lentiviral supernatant, and incubated for 7 d and more using 2 μg/mL puromycin selection. Viral supernatants for all other cell lines were produced in RKO cells using a modified version of the three plasmid transfection protocols described. Briefly, RKO cells were transfected with a mixture of two helper plasmids, pVSV-G, packaging vectors, and the specific miRNA expressing vector along with Transit-293 lipid transfection reagent (Mirus). After 18 to 24 h, the medium was replaced, and after an additional 72h, supernatants containing viral particles were harvested, filtered through 0.45 μmol/L cellulose-acetate and filtrates frozen down at −80°C until use. Cells to be infected were plated in black-sided clear-bottomed 96-well plates at densities ranging from 500 to 4,000 cells per well depending on the particular cell type 18 to 24 before infection with virus. Infections were done in 200 μL of media supplemented with 8 μg/mL polybrene at 37°C and 5% CO2 for 4 h using between 0.1 and 2.0 μL of titer-normalized viral supernatant/well depending on the particular cell type. Medium was replaced 12 to 24 h later with 100 μL fresh medium and puromycin added to the wells after an additional 12 to 24 h (final concentration, 0.5-2.0 μg/mL)</P></DIV><DIV class="center"><H4>Viability assay protocol</H4><P>RKO cells were seeded into 96-well plates at a density of 3000 cell/well overnight. The next daycells were transfected using effectene (Qiagen) transfection reagent with P-EGFPn1 or 134-5pmiRNA at different concentration (2 or 3 µg) Cells were incubated at 37 °C for 72 h then the viability was assessed using the cell-titer blue kit (Promega). The viability was quantified as fluorescence intensity using a microplate reader, FLUOstar Omega (BMG LABTECH). Results are presented on a semi-log scale and represent the average of 3 biological replicates ± s.e.m. Asterisks denote statistical difference.</P><DIV>Figure (10): Viability results from RKO cells before and after transfection with the different concentrations of miR-134 expression vector and empty vector</DIV></DIV><DIV class="center"><DIV>Figure (11): Cell counts of RKO cells during the viability assay.</DIV></DIV><DIV class="center"><DIV>Figure (12): Table 1</DIV></DIV><H4>6.GFP and DAPI staining</H4><P>We confirmed the transformation of the cells and their apoptotic features by measuring GFP fluorescence and DAPI staining of RKO cells. This was after transfection with miR-134 expression vector and Co-expression with the GFP expression vector in a ratio of 3:1. This was to compensate for any ligation errors in the auto-GFP expression by transfected plasmids. Apoptotic features were strongly represented as brightening DAPI indicating condensed DNA (figure 13). 
	</P><DIV class="center"><DIV>Figure (13): GFP fluorescence (left) and DAPI (right) staining of RKO cells after transfection with miR-134 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>