Team:AFCM-Egypt/Description
Toggle navigation
Home
TEAM
Meet The Team
Attributions
Project
Project Description
Bioinformatics
Modeling
Results
Metagenomics
Machine Learning
Human Practices
Education & Public Engagement
Integrated
Collaborations
Judging
Medals
Collaborations
Parts
Parts
Improved Parts
Contributions
Project Description
Â 
miRNAs: A master key to cancer cell apoptosis (Computational & SynBio Approach)
The Bioinformatics:
In extension to our previous work in IGEM about non-coding RNAs, we thought of using miRNAs as master keys for restoring the balance of dysregulated pathways in cancer. We started with constructing a disease-miRNA network by selecting potential miRNAs that have apoptotic effect against colorectal cancer. Selected miRNAs have been subjected to target study and pathway analysis to get insights about the anticancer pathways enriched in the network.
Another aspect we considered was the possible action of miRNAs as modulators for TLRs using the ligand effect to elicit immune stimulatory effect against cancer.  To investigate the microbiome-related immune modulation of colorectal cancer, we have run a metagenomics analysis for publicly available literature datasets to come out with the linked species like colibactin-producing E.coli and their taxonomy for further study. Consequently, we found through literature mining a significant role of Toll like Receptors in this process by catching bacteria related toxins and genetic signatures. We then found specific oligonucleotides that can modulate the immunity through binding to TLR-9 thanks to specific features such as CpG content and binding motifs included.
Afterwards, we thought of an efficient delivery mechanism for our system. Thus, we went for lentiviral delivery as a method of cellular transfection to offer a solution to the important query of delivering non-coding RNAs as a therapeutic option. We started designing the required vectors by choosing a third generation delivery system supported by a second generation packaging and envelope vectors to increase the efficiency of transfection and minimize the number of required plasmids. We also synthesized a part collection for transfection of miRNAs through the lentiviral system to help future IGEM teams.
The lab:
The lab work went in parallel with the computational work through preparing the synthesized parts for transfection in colorectal cancer cell line and ligation to IGEM registry plasmids for part submission. We measured the viability of cancer cells after expressing the selected miRNA, proving the apoptotic effect and characterizing essential IGEM registry parts to our design as well as characterizing our new parts through fluorescence, gel and culture techniques.
We hoped to prove the concept of delivery through transfer plasmids rather than a complete viral transfection which had previously raised some concerns about safety regarding the risk group labs required and associated hazards (such as replicative competent lentivirus (RCL) or insertional mutagenesis). For this reason, we simply synthesized a suitable part collection for lentiviral delivery for future reference. Non-coding RNA transfection required a modular framework with strong promoters and optimized design for miRNAs stem loops to ensure the transcription process.
The modelling:
Our Modeling section has been run to provide structural evidence for TLR binding of miRNAs as well as future designs for dCas9 interference with colibactin gene clusters in toxigenic bacteria. This evidence of miRNA-TLR9 binding would support the regulatory and antitumor effect of miRNAs. Lab measurement of associated immune targets would certainly support structural data provided by RNA-Protein docking, however, the need for co-culture of cancer cells with immune cellular rich media is evident.
Studying the kinetics of cellular transfection was crucial to investigate the dynamics of the lentivirus systems and its interaction with cancer cells. We built a system of ordinary differential equations describing the proposed design. We started to do parameter fitting based on literature models and essential parameters starting from cellular transfection up to the point of equilibrium to go through all the critical points
Finally, we wanted to develop an artificial intelligence to classify DNA oligonucleotides according to their sequence features (TLR-binders or non-binders). This classification is based upon structural features of TLR-ligand binding and previous data of required features of DNA capable of binding to TLRs. We think this deep learning model could improve further immunogenicity studies of DNA oligonucleotides and facilitate the design process of specific immunogenic DNA sequences for immune modulation designs.
