<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:TUDelft"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:TUDelft/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:TUDelft/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=TUDelft"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Case13a</TITLE><HEADER><DIV class="navbar-fixed"><NAV class="row purple"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Description" class="purple-text">Description</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design" class="purple-text dropdown-button" data-hover="hover" data-activates="dropdown-design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Results" class="purple-text">Results</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Demonstrate" class="purple-text">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="purple-text">Entrepreneurship</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Notebook" class="purple-text">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Protocols" class="purple-text">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Safety" class="purple-text">Safety</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Parts" class="purple-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Basic_Parts" class="purple-text">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Composite_Parts" class="purple-text">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Part_Collection" class="purple-text">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Improved_Part" class="purple-text">Improved BioBrick</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Primers" class="purple-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="purple-text">Integrated Human Practices (IHP)</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="purple-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="purple-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Outreach" class="purple-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Newsletter" class="purple-text">Newsletter</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Meet_Up" class="purple-text">European iGEM Meet Up</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Attributions" class="purple-text">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Sponsors" class="purple-text">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Crowdfunding" class="purple-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Contribution-Overview" class="purple-text">Contribution Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Interlab" class="purple-text">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="purple-text">Software</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Plot_Grow_Curves" class="purple-text">Plot Growth Curves</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" class="purple-text">Find correct CRISPR targets</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Team" class="purple-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Team#Supervisors" class="purple-text">Supervisors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/TeamBuilding" class="purple-text">Team Building</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Collaborations" class="purple-text">Collaborations</A></LI></UL><UL id="dropdown-awards" class="dropdown-content nested menu-even-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="purple-text">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="purple-text">Software tool</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Measurement" class="purple-text">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="purple-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="purple-text">IHP</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="purple-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Modeling" class="purple-text dropdown-button" data-hover="hover" data-activates="dropdown-fullmodel">Modeling</A></LI></UL><UL id="dropdown-fullmodel" class="dropdown-content nested"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="purple-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="purple-text">Motif finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="purple-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="purple-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="purple-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="purple-text">Rate kinetics</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="purple-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="purple-text">Motif Finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="purple-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="purple-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="purple-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="purple-text">Rate Kinetics</A></LI></UL><UL id="dropdown-design" class="dropdown-content nested menu-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" class="purple-text">Cas13a</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" class="purple-text">Sample Preperation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" class="purple-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#TDP" class="purple-text">TDPs</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" class="purple-text">Vesicles</A></LI></UL><UL id="dropdown-achievements" class="dropdown-content"><LI><A href="#!" class="purple-text dropdown-button" data-hover="hover" data-activates="dropdown-awards">Special Tracks</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Medals" class="purple-text">Medals</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Awards_Nominations" class="purple-text">Awards &amp; Nominations</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2017.igem.org/Team:TUDelft" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Giant_Jamboree" class="waves-effect waves-light block">Giant Jamboree</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A href="https://www.facebook.com/TUDelft.iGEM2017/" class="menu-hover waves-effect waves-light block"><SPAN> TUDelft.iGEM2017</SPAN></A></LI><LI title="Click to copy our phonenumber"><A onclick="copyToClipboard('+31 152 78 77 51')" class="menu-hover waves-effect waves-light block" id="safari-phone-hack"><SPAN style="overflow: hidden; display:inline-block;">  +31 152 78 77 51</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light purple" id="menu-btn"><I class="material-icons">menu</I></A></HEADER><MAIN class="row lab"><DIV class="col m2"><DIV id="timeline"><DIV id="round24"><A href="#week24-scroll"><P>June</P></A></DIV><DIV id="round27"><A href="#week27-scroll"><P>July</P></A></DIV><DIV id="round32"><A href="#week32-scroll"><P>August</P></A></DIV><DIV id="round36"><A href="#week36-scroll"><P>September</P></A></DIV><DIV id="round40"><A href="#week40-scroll"><P>October</P></A></DIV></DIV></DIV><DIV class="col s12 m9"><DIV class="section" id="Notebook"><DIV id="notebook-buttons"><P>Make a selection:</P><A href="#!" class="btn deep-purple active waves-effect waves-purple"> Cas13a</A><A href="#!" class="btn deep-purple  waves-effect waves-purple"> TDPs</A><A href="#!" class="btn deep-purple  waves-effect waves-purple"> Detection</A><A href="#!" class="btn deep-purple  waves-effect waves-purple"> Sample Preparation</A><A href="#!" class="btn deep-purple  waves-effect waves-purple"> Vesicles</A></DIV><P class="white-text-hack">-------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- -------------------- -------------------- -----------------------</P><UL class="collapsible popout" data-collapsible="expandable" id="notebook-wrapper"><LI><DIV class="collapsible-header scrollspy" id="week24-scroll">Week 24</DIV></LI><LI><DIV class="collapsible-header">Wednesday 14 June</DIV><DIV class="collapsible-body" id="d20170614"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170614"><H3>Sample prep liquid culture of JW0729</H3><H6>Amária and Fiona</H6><P>The JW0729 strain was grown overnight in 10 mL LB according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. 5 µL
        kanamycin was added to the 10 mL LB. </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 15 June</DIV><DIV class="collapsible-body" id="d20170615"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170615"><H3>Sample prep plating</H3><H6>Amária and Fiona</H6><P>We made dilutions of the overnight culture of JW0729 (made on the 14<SUP>th</SUP> of June) by prepairing tubes with 900 µL LB, then adding 100 µL of the overnight culture to the first tube, mixing by pipetting and then transferring
        100 µL of the mix with a clean pipette to the next tube. We continued until we had a 10<SUP>4</SUP> X dilution. We spread out 100 µL of the 10<SUP>-3</SUP> and 10<SUP>-4</SUP> dilutions with a hockeystick plater. The plates
        used where made by the <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">pouring agar plates</A> protocol .We let them grow overnight at 37 °C. The used antibiotic was kanamycin. </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 16 June</DIV><DIV class="collapsible-body" id="d20170616"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170616"><H3>Sample prep plating results</H3><H6>Amária and Fiona</H6><P> The cells did not grow because our stove was at 50 °C instead of the 37 °C we wanted it to be. </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week25-scroll">Week 25</DIV></LI><LI><DIV class="collapsible-header">Monday 19 June</DIV><DIV class="collapsible-body" id="d20170619"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170619"><H3>Sample prep liquid culture of JW0729</H3><H6>Amária and Fiona</H6><P> We repeated making the Liquid culture of JW0729 as on the 14<SUP>th</SUP> of June.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 20 June</DIV><DIV class="collapsible-body" id="d20170620"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170620"><H3>Sample prep plating</H3><H6>Amária and Fiona</H6><P> We repeated the protocol of <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">pouring agar plates</A>  as on the 15<SUP>th</SUP> of June.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 21 June</DIV><DIV class="collapsible-body" id="d20170621"><DIV class="C"><DIV class="D"><DIV class="row notebook detection" id="D20170621"><H3>Final design coacervation method trial 1</H3><H6>Kasper</H6><P>The full received amount of 10 mg of polyuridylic acid potassium salt (from hereon denoted as polyU) was dissolved in 90 µL of nuclease free water to make a 10 wt% polyU stock solution. This was aliquotted into 15 tubes of 5 µL each and stored at -20 °C. </P><P> A 100 mM HEPES solution was made and pH adjusted to pH 7.6 using NaOH. </P><P> A 100 mM stock solution of spermidine was received from Christophe Danelon Lab.</P><P> Stocks of 1 M and 100 mM MgCl<SUB>2</SUB> were prepared.</P><P> Three tubes were prepared in an attempt to form visible coacervates. The first tube contained 5 mM HEPES pH 7.6, 1 mM MgCl<SUB>2</SUB>, 25 mM spermidine and 0.05 wt% polyU. The second tube contained 5 mM HEPES pH 7.6, 2 mM MgCl<SUB>2</SUB>, 25 mM spermidine and 0.1 wt% polyU. The third tube contained 48 mM HEPES pH 7.6, 1 mM MgCl<SUB>2</SUB>, 25 mM spermidine and 0.1 wt% polyU. All three were left at 37 °C for 30 minutes. No increase in turbidity of solution was observed.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20170621"><H3>Sample prep DNA extraction through heat</H3><H6>Amária and Fiona</H6><P> We made 7 different concentrations of colonies with 2,1,0.5,0.25,0.125,0.0625 colonies and one with an undifined but high dosage of colonies. We did this by putting 2 colonies (from the plate made on the 20<SUP>th</SUP> of June) in 2 mL milli
        Q, mixing and then transferring 1 mL to a new tube with 1 mL milliQ. We repeated this until we had 0.0625 of a colony in 1ml. The high dosage we got by scraping an inauculation loop over the plate with colonies and adding this to 1 mL
        milli Q. We repeated this to have a duplicate of every concentration. We extracted the DNA of 1 of the duplicates with the microwave method for DNA isolation and the other duplicate with the boiling method. Both are described by this protol:
        <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A>. We measured the DNA concentration with the nanodrop.</P><H6>Boiling method:</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample name</TH><TH>Colony concentration </TH><TH>Amount DNA (ng/µL)</TH><TH>A260</TH><TH>260/230 </TH><TH>260/280 </TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1</TD><TD> 2</TD><TD>3.89</TD><TD>0.078</TD><TD>0.74</TD><TD>-51.54</TD></TR><TR><TD class="firstcolumn">B2</TD><TD>1</TD><TD>1.05</TD><TD>0.021</TD><TD>0.68</TD><TD>-0.48</TD></TR><TR><TD class="firstcolumn">B3</TD><TD>0.5</TD><TD>0.87</TD><TD>0.017</TD><TD>8.05</TD><TD>-0.45</TD></TR><TR><TD class="firstcolumn">B4</TD><TD>0.25</TD><TD>-0.20</TD><TD>-0.004</TD><TD>-0.24</TD><TD>0.07</TD></TR><TR><TD class="firstcolumn">B5</TD><TD>0.125</TD><TD>-0.35</TD><TD>-0.007</TD><TD>1.87</TD><TD>0.10</TD></TR><TR><TD class="firstcolumn">B6</TD><TD>0.0625</TD><TD>0.22</TD><TD>0.004</TD><TD>-0.50</TD><TD>-0.08</TD></TR><TR><TD class="firstcolumn">BHC</TD><TD>high concentration</TD><TD>59.03</TD><TD>1.181</TD><TD>0.78</TD><TD>2.67</TD></TR></TBODY></TABLE><H6>Microwave method:</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample name </TH><TH>Colony concentration </TH><TH>Amount DNA (ng/µL)</TH><TH>A260</TH><TH>260/230 </TH><TH>260/280 </TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">M1</TD><TD>2 in 500 µL</TD><TD>8.02</TD><TD>0.160</TD><TD>0.61</TD><TD>2.18</TD></TR><TR><TD class="firstcolumn">M2</TD><TD> 2 in 1 mL</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">M3</TD><TD>1</TD><TD>0.46</TD><TD>0.009</TD><TD>0.28</TD><TD>-0.35</TD></TR><TR><TD class="firstcolumn">M4</TD><TD>0.5</TD><TD>2.64</TD><TD>0.053</TD><TD>0.78</TD><TD>19.85</TD></TR><TR><TD class="firstcolumn">M5</TD><TD>0.25</TD><TD>-1.34</TD><TD>-0.027</TD><TD>2.48</TD><TD>0.93</TD></TR><TR><TD class="firstcolumn">M6</TD><TD>0.125</TD><TD>-1.94</TD><TD>-0.039</TD><TD>0.80</TD><TD>0.55</TD></TR><TR><TD class="firstcolumn">M7</TD><TD>0.0625</TD><TD>-2.82</TD><TD>-0.056</TD><TD>1.34</TD><TD>0.69</TD></TR><TR><TD class="firstcolumn">MHC</TD><TD>high concentration</TD><TD>168.94</TD><TD>3.379</TD><TD>0.54</TD><TD>1.96</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 22 June</DIV><DIV class="collapsible-body" id="d20170622"><DIV class="C"><DIV class="D"><DIV class="row notebook detection" id="D20170622"><H3>Final design coacervation method trial 2</H3><H6>Kasper</H6><P> New 100 mM HEPES buffer was prepared but not pH adjusted. A 10 wt% stock solution of spermine was made in nuclease free water and stored at 4 °C.</P><P> Two tubes of 50 µL were prepared in an attempt to form visible coacervates. The first tube contained 5 mM HEPES, 1 mM MgCl<SUB>2</SUB>, 0.5 wt% spermine, 0.05 wt% polyU and MilliQ. The second tube contained 5 mM HEPES, 1 wt% spermine, 0.2 wt% polyU and MilliQ.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20170622"><H3>Sample prep PCR on DNA extracted via heat</H3><H6>Amária and Fiona</H6><P>We did PCR on the positive nanodrop concentrations of extracted DNA of the 21<SUP>th</SUP> of June, by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol. For the amount
        of Template DNA we adjusted the volume so that 10 ng was present in the PCR mix. The PCR program was as following:</P><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>94</TD><TD>120</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>94</TD><TD>30</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>60</TD><TD>30</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>25</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>300</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170622"><H3>TDPs Backbone PCR amplification</H3><H6>Guillermo and Fiona</H6><P>Primers IG0010 and IG0011 were dissolved according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#primerworkingstockpreparation" target="_blank">primer working stock preparation</A> protocol. Next, we followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol, with 0.5 µL of DNA template (linearized backbone pSB1C3) and 36 µL of milli-Q; Only 1 µL of each of the primers was used. The PCR program was run with an extension time of 2 minutes.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 23 June</DIV><DIV class="collapsible-body" id="d20170623"><DIV class="C"><DIV class="D"><DIV class="row notebook detection" id="D20170623"><P>To eliminate the effect that other components of the Bibdo I solution could have on the formation of coacervates, the same test was ran with only RNase A. 45 µL solutions (C &amp; D) were prepared as follows</P><TABLE class="notebook table table-style-1"><TBODY><TR><TD class="firstcolumn">C</TD><TD>D</TD></TR><TR><TD class="firstcolumn">20 mM HEPES</TD><TD>same as C</TD></TR><TR><TD class="firstcolumn">0.1 wt% polyU</TD><TD> same as C</TD></TR><TR><TD class="firstcolumn">0.05 wt% RNase A</TD><TD> same as C</TD></TR><TR><TD class="firstcolumn"> Nuclease free water</TD><TD> same as C</TD></TR></TBODY></TABLE><P>Both C &amp; D were incubated for 30 minutes at 37 °C. Subsequently, 5 µL of 10 wt% spermine was added to a final concentration of 1 wt%. Clear difference between C and D was observed. C was significantly less turbid than D indicating that the RNase A activity was successfully demonstrated using the coacervation method.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20170623"><H3>Sample prep PCR clean up</H3><H6>Amária and Fiona</H6><P>
        Purified the PCR products of the 22 <SUP>th</SUP> of June by the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol
      </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170623"><H3>TDPs Electrophoresis</H3><H6>Guillermo and Fiona</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol for the amplified sample. </P><P>The expected band of ~2,000  in lane 2 is absent, indicating a failed PCR reaction. Therefore, it was decided to do the amplification again. </P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week26-scroll">Week 26</DIV></LI><LI><DIV class="collapsible-header">Monday 26 June</DIV><DIV class="collapsible-body" id="d20170626"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170626"><H3>Sample prep liquid culture of JW0729</H3><H6>Amária and Fiona</H6><P>The JW0729 strain was grown overnight in 10 mL LB according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. 5 µL
        kanamycin was added to the 10 mL LB. </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170626"><H3>TDPs Backbone PCR amplification 2.0</H3><H4>Amplification</H4><H6>Guillermo and Fiona</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol with primers IG0010 and IG0011 and 0.5 µL of DNA template (linearized backbone pSB1C3) and 33 µL of sterile milli-Q. The PCR program was run with a denaturation time of 5 seconds and an extension time of 1  minute. Furthermore, the annealing temperature was decreased to 55 °C and the annealing time was set to 10 seconds. The resulting adjusted program is the following:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>98</TD><TD>30</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>98</TD><TD>5</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>55</TD><TD>10</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>60</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>300</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><H4>Results</H4><H6>Guillermo and Fiona</H6><P>The expected band size is 2,070 bp. The bright band in the second lane (S) matches this size. It was therefore decided to continue with this sample. Nevertheless, we wanted to do another backbone amplification to get more product.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 27 June</DIV><DIV class="collapsible-body" id="d20170627"><DIV class="C"><DIV class="row notebook cas13a" id="C20170627"><H3>Cas13a Backbone preparation</H3><H6>Jeroen and Fiona</H6><P>Agar plates with ampicillin were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">agar plate</A> protocol, except only 1 µL per mL of medium was used. Cells with pSB4A5 (low copy number plasmid with ampicillin resistance) were obtained from our supervisor Dominik. The plasmid contains an RFP insert. The cells were plated out and incubated overnight at 37 °C in the stove.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20170627"><H3>Effect of HEPES concentration on coacervates</H3><H6>Kasper</H6><P> To examine the effect of the HEPES concentration on the formation of coacervates was tested by preparing coacervates (0.1 wt% polyU, 1 wt% spermine and MilliQ) in six tubes of 50 µL with 0, 10, 20, 50, 75 and 89 mM of HEPES. No significant difference in turbidity (to the naked eye) between each of those tubes was observed, indicating that, at least to the naked eye, the concentration of HEPES does not significantly influence the turbidity of coacervate solutions.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20170627"><H3>Sample prep plating</H3><H6>Amária and Fiona</H6><P> We repeated the protocol of Plating as on the 15<SUP>th</SUP> of June.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170627"><H3>TDPs DNA purification</H3><H4>Wizard SV PCR Clean-Up</H4><H6>Guillermo and Fiona</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol for the amplified sample. We used 45 µL of sample and 45 µL of Membrane Binding Solution for the first step. 50 µL of sterile milli-Q was used for the elution.</P><H4>Third backbone PCR amplification</H4><H6>Guillermo</H6><P>The PCR program of the 26<SUP>th</SUP> of June was repeated, with the adaptation that the annealing temperature was increased to 57 °C.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170627"><H3>Vesicles Parts amplification </H3><H4>Amplification</H4><H6>Aafke and Gabriella</H6><P> Primers IG0023 and IG0022 were dissolved according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#primerworkingstockpreparation" target="_blank">primer working stock preparation</A> protocol. 
    GFP-TorA and TolR were dissolved according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gresuspension" target="_blank">gBlock resuspension</A> protocol. 
    Next, we followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol, with 1 µL of DNA template (GFP-TorA and TolR) and 35.5 µL of milli-Q; Only 1 µL of each of the primers was used. The PCR program was run with an extension time of 9 seconds for TolR and 28 seconds for GFP-TorA.</P><H4>Gel electrophoresis</H4><H6>Aafke and Gabriella</H6><P> The PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below. </P><P>
    The expected band size of GFP-TorA is 1,250 bp. The indicated band in the second lane matches this size. The expected band size of TolR is 564 bp. The bright band in the third lane matches this size. It was decided to purify both fragments from the gel, to get rid of the DNA-fragments with different sizes.
    </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 28 June</DIV><DIV class="collapsible-body" id="d20170628"><DIV class="C"><DIV class="row notebook cas13a" id="C20170628"><H3>Cas13a Grow overnight cultures</H3><H6>Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol was followed, with in total a volume of ~50µL of cell culture.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170628"><H3>Sample prep DNA Heat Extraction</H3><H6>Amária and Fiona</H6><P>
        We measured the OD600 of 1,2and 3 colonies taken from the 10<SUP>-6</SUP> Dilution and diluted in 1 mL(with ultraspec 10 from amersham biosciences). Results:
      </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Replicate # </TH><TH># of colonies diluted</TH><TH>Absorbance</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1</TD><TD>0.03</TD></TR><TR><TD class="firstcolumn">2</TD><TD>1</TD><TD>0.05</TD></TR><TR><TD class="firstcolumn">3</TD><TD>1</TD><TD>0.05</TD></TR><TR><TD class="firstcolumn">1</TD><TD>2</TD><TD>0.05</TD></TR><TR><TD class="firstcolumn">2</TD><TD>2</TD><TD>0.02</TD></TR><TR><TD class="firstcolumn">3</TD><TD>2</TD><TD>0.06</TD></TR><TR><TD class="firstcolumn">1</TD><TD>3</TD><TD>0.05</TD></TR><TR><TD class="firstcolumn">2</TD><TD>3</TD><TD>0.03</TD></TR><TR><TD class="firstcolumn">3</TD><TD>3</TD><TD>0.02</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170628"><H3>TDPs Electrophoresis of third backbone amplification</H3><H4>Electrophoresis and purification of amplification product</H4><H6>Guillermo and Fiona</H6><P>The PCR sample was purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol. Next, the sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol. The concentration of the new and old sample was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. However, the samples were not measured in triplo.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">27-06</TD><TD>48-63<SUP>*</SUP></TD></TR><TR><TD class="firstcolumn">28-06</TD><TD>67</TD></TR></TBODY></TABLE><P><SUP>*</SUP>This sample was measured twice, because the first time the machine was blanked incorrectly (with air instead of water). The actual DNA concentration should be in de indicated range. We calculated with 60 ng/µL.</P><H4>Results and discussion</H4><H6>Guillermo and Fiona</H6><P>The expected band size is 2,070 bp. The bright band in the first lane (S) matches this size. It was therefore decided to continue with this sample.</P><H4>Pouring plates for transformation</H4><H6>Guillermo and Fiona</H6><P>15 agar plates with Chloramphenicol antibiotics were poured according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">agar plate</A> protocol.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170628"><H3>Vesicles Parts amplification</H3><H4> Gel purification </H4><H6>Aafke and Gabriella</H6><P> The remainder of the PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol. 
    The results can be seen in the picture below. </P><P>The indicated bands were cut out and purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" target="_blank">gel product purification (Promega Wizard™ Kit)</A> protocol. 
     
    The concentration of the purified sample was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR</TD><TD>11.38</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>4.46</TD></TR></TBODY></TABLE><H4>Amplification</H4><H6>Aafke and Gabriella</H6><P>We tried to amplify the parts again by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol. This time with 2 µL of GFP-TorA (and 1 µL TolR).
    The result can be seen in the picture below.</P><P>
     Still, multiple bands are visible for TolR and even a smear for GFP-TorA. This indicates that the PCR-mix cannot be used directly, but that the right DNA-fragments should be isolated from the gel.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 29 June</DIV><DIV class="collapsible-body" id="d20170629"><DIV class="C"><DIV class="row notebook cas13a" id="C20170629"><H3>Cas13a Midiprep of pSB4A5</H3><H6>Fiona and Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#midiprep" target="_blank">plasmid midiprep (GeneJET plasmid midiprep kit)</A> protocol was followed and the purified sample was NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>), but only measured twice.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Measurement</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>8.67</TD></TR><TR><TD class="firstcolumn">2</TD><TD>8.20</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170629"><H3>Sample prep PCR on DNA extracted via Heat</H3><H6>Amária and Fiona</H6><P>We measured the nanodrop concentration of the extracted DNA on the 28<SUP>th</SUP> of June.</P><H6>Microwave extraction:</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample </TH><TH>Amount DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">10<SUP>7</SUP></TD><TD>1.76</TD></TR><TR><TD class="firstcolumn">10<SUP>6</SUP></TD><TD>0.7</TD></TR><TR><TD class="firstcolumn">10<SUP>5</SUP></TD><TD>-0.53</TD></TR><TR><TD class="firstcolumn">10<SUP>4</SUP></TD><TD>-0.38</TD></TR><TR><TD class="firstcolumn">10<SUP>3</SUP></TD><TD>0.50</TD></TR><TR><TD class="firstcolumn">10<SUP>2</SUP></TD><TD>-0.64</TD></TR><TR><TD class="firstcolumn">10<SUP>1</SUP></TD><TD>-0.03</TD></TR><TR><TD class="firstcolumn">10<SUP>0</SUP></TD><TD>-0.14</TD></TR></TBODY></TABLE><H6>Longer time microwave extraction:</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample </TH><TH>Amount DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">10<SUP>7</SUP></TD><TD>2.28</TD></TR><TR><TD class="firstcolumn">10<SUP>6</SUP></TD><TD>-0.42</TD></TR><TR><TD class="firstcolumn">10<SUP>5</SUP></TD><TD>-0.69</TD></TR><TR><TD class="firstcolumn">10<SUP>4</SUP></TD><TD>0.03</TD></TR><TR><TD class="firstcolumn">10<SUP>3</SUP></TD><TD>-0.57</TD></TR><TR><TD class="firstcolumn">10<SUP>2</SUP></TD><TD>-0.84</TD></TR><TR><TD class="firstcolumn">10<SUP>1</SUP></TD><TD>-0.72</TD></TR><TR><TD class="firstcolumn">10<SUP>0</SUP></TD><TD>-0.24</TD></TR></TBODY></TABLE><H6>Boiling extraction:</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample </TH><TH>Amount DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">10<SUP>7</SUP></TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">10<SUP>6</SUP></TD><TD>1.1</TD></TR><TR><TD class="firstcolumn">10<SUP>5</SUP></TD><TD>0.2</TD></TR><TR><TD class="firstcolumn">10<SUP>4</SUP></TD><TD>-0.1</TD></TR><TR><TD class="firstcolumn">10<SUP>3</SUP></TD><TD>0.2</TD></TR><TR><TD class="firstcolumn">10<SUP>2</SUP></TD><TD>0.5</TD></TR><TR><TD class="firstcolumn">10<SUP>1</SUP></TD><TD>0.3</TD></TR><TR><TD class="firstcolumn">10<SUP>0</SUP></TD><TD>0.6</TD></TR></TBODY></TABLE><P>We did PCR on the samples by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol. We made a mastermix for 24 samples so added 25 times all the components. We used 10 µL
    Template DNA in the PCR mix, exept for sample 10<SUP>7</SUP> of the microwave extraction, from this we added 6 µL and from the sample 10<SUP>7</SUP> of the longer microwave method and boiling method we added 3 µL. The PCR program
    was as following:</P><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>94</TD><TD>120</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>94</TD><TD>30</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>60</TD><TD>30</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>25</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>300</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><P>
    For the DNA gel electrophoresis we made two 0.8 % agarose gels, one big one (100 mL) and one small one(50 mL). We loaded 5 µL per samples on the gel and 6 µL 1 kb benchtop ladder in a slot. We run the gels for
    30 minutes at 100V. We ran gel 2 (the small one) for an additional 10 minutes. The following pictures shows the resulting DNA gels(loaded wrong so the question marks are where we did not know which sample it was but we loaded them again in the still
    empty slots):
    </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170629"><H3>TDPs Gibson assembly</H3><H6>Guillermo and Kelly</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gresuspension" target="_blank">gBlock resuspension</A> protocol for the 6 ordered tardigrade proteins and the T7 promoter, according to the table below.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>gBlock</TH><TH>Amount (ng)</TH><TH>Milli-Q (µL)</TH><TH>Final concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">T7 promoter</TD><TD>250</TD><TD>25</TD><TD>10</TD></TR><TR><TD class="firstcolumn">SAHS_33020</TD><TD>500</TD><TD>10</TD><TD>50</TD></TR><TR><TD class="firstcolumn">SAHS_68234</TD><TD>500</TD><TD>10</TD><TD>50</TD></TR><TR><TD class="firstcolumn">All four CAHS genes</TD><TD>1000</TD><TD>20</TD><TD>50</TD></TR></TBODY></TABLE><P>The vector and inserts were used in a 1:3 ratio. Following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">Gibson assembly</A> protocol, the ligation calculater was employed to calculate the necessary volumes of the insert solutions. An overview of the different parts and their sizes and volumes is displayed in the table below.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Part</TH><TH>Amount needed (ng)</TH><TH>Size (bp)</TH><TH>Conc. (ng/µL)</TH><TH>Volume needed (µL)</TH><TH>Size + T7 promoter</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">T7 promoter</TD><TD>186</TD><TD>128</TD><TD>10</TD><TD>1.9</TD><TD>-</TD></TR><TR><TD class="firstcolumn">CAHS_94205_2</TD><TD>129</TD><TD>891</TD><TD>50</TD><TD>2.6</TD><TD>1019</TD></TR><TR><TD class="firstcolumn">CAHS_94205</TD><TD>130</TD><TD>894</TD><TD>50</TD><TD>2.6</TD><TD>1022</TD></TR><TR><TD class="firstcolumn">CAHS_106094</TD><TD>130</TD><TD>894</TD><TD>50</TD><TD>2.6</TD><TD>1022</TD></TR><TR><TD class="firstcolumn">CAHS_107838</TD><TD>130</TD><TD>900</TD><TD>50</TD><TD>2.6</TD><TD>1028</TD></TR><TR><TD class="firstcolumn">SAHS_33020</TD><TD>104</TD><TD>719</TD><TD>50</TD><TD>2.1</TD><TD>847</TD></TR><TR><TD class="firstcolumn">SAHS_68234</TD><TD>107</TD><TD>735</TD><TD>50</TD><TD>2.1</TD><TD>863</TD></TR></TBODY></TABLE><P>The Gibson assembly samples were prepared according to the following table.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Thawed mastermix</TD><TD>7.5</TD></TR><TR><TD class="firstcolumn">Backbone pSB1C3 (PCR product)</TD><TD>1.7</TD></TR><TR><TD class="firstcolumn">T7 promoter<SUP>*</SUP></TD><TD>1.9</TD></TR><TR><TD class="firstcolumn">gBlock of gene<SUP>*</SUP></TD><TD>x</TD></TR><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>3.9-x</TD></TR></TBODY></TABLE><P><SUP>*</SUP>For these volumes, see the table before.</P><P>TOP10 cells that were made competent during the iGEM practice week were transformed according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. The cells were plated out on the prepared chloramphenicol plates.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170629"><H3>Vesicles Gibson assembly</H3><H4> Assembly mix </H4><H6>Aafke and Gabriella</H6><P>The vector and inserts were used in a 1:3 ratio. Following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">Gibson assembly </A> protocol, the ligation calculater was employed to calculate the necessary volumes of the insert solutions. The Gibson assembly samples were prepared according to the following tables. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Thawed mastermix</TD><TD>15</TD></TR><TR><TD class="firstcolumn">Backbone pSB1C3</TD><TD>2</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>2</TD></TR></TBODY></TABLE><P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Thawed mastermix</TD><TD>30</TD></TR><TR><TD class="firstcolumn">Backbone pSB1C3</TD><TD>2</TD></TR><TR><TD class="firstcolumn">TolR</TD><TD>7.2</TD></TR><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>0.8
    </TD></TR></TBODY></TABLE></P><H4> Transformation </H4><H6>Aafke and Gabriella</H6><P>TOP10 cells that were made competent during the iGEM practice week were transformed according to 
    the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. 
    The cells were plated out on chloramphenicol plates.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 30 June</DIV><DIV class="collapsible-body" id="d20170630"><DIV class="C"><DIV class="row notebook cas13a" id="C20170630"><H3>Cas13a Linearization of pSB4A5</H3><H4>Phusion PCR 1</H4><H6>Fiona and Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed with 5 µL of the midiprepped sample from 29-06-2017. Primers IG0010 and IG0011 were used. Annealing temperatures of 55 and 60 °C were tried out. The extension time was set at 1 minute. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 35 minutes.</P><P>It was concluded from the gel that the PCR was not successful. Faint bands are visible around 4,000 bp, which is too large. The expected size of the linearized backbone was 3395 bp. It was concluded that the bands are most likely caused by the circular template DNA.A new PCR was necessary.</P><H4>Phusion PCR 2 (with DMSO)</H4><H6>Fiona</H6><P>The Phusion PCR (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A>) was repeated with more template DNA (5 µL). Two samples using the midiprepped backbone and two samples using the PCR product as template DNA were prepared. For each template, one sample was run with DMSO and one without. Again the products were checked on gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A>).</P><P>The two PCR products (with and without DMSO) have a band of the correct size only, while the midiprepped samples have resulted in by-products.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20170630"><H3>Spermine dependence of spermine/polyU coacervates</H3><H6>Kasper</H6><P> How the concentration of spermine affects the turbidity of spermine/polyU coacervates, was determined using a Synergy H1 microplate reader (BioTek) in the Stan Brouns lab. Wells were filled with varying concentrations of spermine and polyU and the absorbance of 500 nm light was tracked over the course of 30 minutes. This is still a preliminary experiment to get familiarized with the method.</P><P>Six wells of 50 µL were prepared containing 0.1 wt% polyU, 20 mM HEPES and varying concentrations of spermine (0, 0.05, 0.1, 0.5, 1.0 and 5.0 wt%) in milliQ. Furthermore a blank was prepared containing only 20 mM HEPES and milliQ.</P><P>After inserting the plate into the platereader 37 °C, the samples were left for 5 minutes to equilibriate temperature and then the absorbance of 500 nm light was measured every minute.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20170630"><H3>Sample prep DNA electrophoresis of Digested samples</H3><H6>Amária</H6><P>
    We performed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol for a 0.8 % agarose gel. We mixed 3 µL loading buffer with 15 µL sample (from the digestion
    on the 27<SUP>th</SUP> of June) on parafilm. 6 µL 1kb benchtop ladder was used. We ran the gel at 100V for 40 min.The following picture shows the resulting DNA gel:
    </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170630"><H3>TDPs Plate counts and colony PCR</H3><H4>Transformation results</H4><H6>Guillermo, Gabriella and Kelly</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Gene</TH><TH>Index</TH><TH>Colonies plate 1<SUP>*</SUP></TH><TH>Colonies plate 2<SUP>*</SUP></TH><TH>Colonies picked</TH><TH># of colonies unpicked</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">CAHS_94205_2</TD><TD>1</TD><TD>5</TD><TD>0</TD><TD>1.1-1.3</TD><TD>2</TD></TR><TR><TD class="firstcolumn">CAHS_94205</TD><TD>2</TD><TD>4</TD><TD>2</TD><TD>2.1-2.3</TD><TD>3</TD></TR><TR><TD class="firstcolumn">CAHS_106094</TD><TD>3</TD><TD>10</TD><TD>6</TD><TD>3.1-3.3</TD><TD>13</TD></TR><TR><TD class="firstcolumn">CAHS_107838</TD><TD>4</TD><TD>2</TD><TD>2</TD><TD>4.1-4.3</TD><TD>1</TD></TR><TR><TD class="firstcolumn">SAHS_33020</TD><TD>5</TD><TD>3</TD><TD>1</TD><TD>5.1-5.3</TD><TD>1</TD></TR><TR><TD class="firstcolumn">SAHS_68234</TD><TD>6</TD><TD>18</TD><TD>15</TD><TD>6.1-6.3</TD><TD>30</TD></TR></TBODY></TABLE><P><SUP>*</SUP>For comparison, the negative control plate (Gibson assembly mix without an insert, transformed in TOP10) had four colonies. The number of positive colonies on the transformed plates is thus relatively low (except for gene 6). The picked colonies were checked by colony PCR, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol. However, the PCR program was run with an annealing temperature of 53 °C instead of 55 °C, because a pre-set program was used. The extension time was set to be 60 seconds for all the genes.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170630"><H3>Vesicles Colony PCR </H3><H4>Results transformation</H4><H6>Aafke and Gabriella</H6><P>
    No growth was detected on the plates with the cells transformed with TolR. The cells transformed with GFP-TorA did grow on the plates and 5 colonies were picked. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypickingcontrol" target="_blank">colony picking</A> protocol)  </P><H4>Colony PCR GFP-TorA</H4><H6>Aafke and Gabriella</H6><P>The picked colonies were checked by colony PCR, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol. 
    The first step was set to be 2.5  minute. The extension time was set to be 2 minutes. The annealing temperature of the primers was 55°C;.</P><H4>Transformation TolR</H4><H6>Aafke and Gabriella</H6><P>50 µL TOP10 cells that were made competent during the iGEM practice week were transformed with 15 µL of TolR-plasmid according to 
    the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. 
    The cells were plated out on chloramphenicol plates.</P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week27-scroll">Week 27</DIV></LI><LI><DIV class="collapsible-header">Monday 3 July</DIV><DIV class="collapsible-body" id="d20170703"><DIV class="C"><DIV class="row notebook cas13a" id="C20170703"><H3>Cas13a PCR product purification</H3><H6>Jeroen</H6><P>The two PCR products that were amplified from the previous PCR product were added together and purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard SV™ Kit)</A> protocol.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20170703"><H3>Thorough spermine dependence of spermine/polyU coacervates</H3><H6>Kasper</H6><P> To more thoroughly examine the dependence of turbidity caused by coacervation on the spermine concentration, solutions of 100 µL were prepared containing 20 mM HEPES 0.05 wt% polyU and varying concentrations of spermine (0, 0.05, 0.1, 0.5, 1.0, 5.0) wt%. Also a blank was prepared containing only milliQ. All wells were mixed by gentle pipetting, and subsequently 50 µL of each well was taken out and pipetted into another well to make technical duplicates. The temperature in the platereader was set at 27 °C and all wells were left to equilibriate temperature for five minutes. Then for 5 minutes the absorbance of 500 nm light was measured every minute. After two such rounds, the platereader was set to shake the samples for 1 minute and the protocol was repeated including the equilibriation step. </P></DIV></DIV><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170703"><H3>Colony PCR gel and new colony PCR</H3><H4>Colony PCR gel</H4><H6>Guillermo, Isabell and Kelly</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol for the amplified samples. The results for 14 out of the 18 samples are displayed below.</P><P>The expected band size is around 1,150 bp, as the used primers add an extra ~160 bp to the insert size. In every lane, two bands are present around this size. It is unclear if either of these is insert, or that both of them are an undesired amplified product. However, the negative control that was run by Aafke and Gabriella (Vesicles module) also shows these two bands, while there should not be an insert present in this sample. It was therefore decided to screen more colonies, especially since more colonies were now visible on the plates.</P><H4>New colony picking and PCR</H4><H6>Guillermo and Kelly</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Index</TH><TH>New colonies picked<SUP>*</SUP></TH><TH># picked</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1.4</TD><TD>1</TD></TR><TR><TD class="firstcolumn">2</TD><TD>2.4-2.6</TD><TD>3</TD></TR><TR><TD class="firstcolumn">3</TD><TD>3.4-3.9</TD><TD>6</TD></TR><TR><TD class="firstcolumn">4</TD><TD>4.4-4.5</TD><TD>2</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.4-5.7</TD><TD>4</TD></TR><TR><TD class="firstcolumn">6</TD><TD>6.4-6.7</TD><TD>4</TD></TR></TBODY></TABLE><P>The colony PCR was carried out as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol, with the exception that the number of cycles was increased to 45.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170703"><H3>Vesicles Results transformation and colony PCR </H3><H4>Results transformation TolR</H4><H6>Aafke and Gabriella</H6>
    This second time the transformation with TolR worked and 5 colonies were picked. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypickingcontrol" target="_blank">colony picking</A> protocol)
                       <H4>Results colony PCR GFP-TorA</H4><H6>Aafke and Gabriella</H6><P> The PCR samples were run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below. </P><P>Colony 2 deviates from the other colonies and might be interesting to sequence. 5 new colonies were picked.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypickingcontrol" target="_blank">colony picking</A> protocol) </P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 4 July</DIV><DIV class="collapsible-body" id="d20170704"><DIV class="C"><DIV class="row notebook cas13a" id="C20170704"><H3>Cas13a Cleaning and concentration of the backbone</H3><H6>Fiona and Jeroen</H6><P>The cleaned PCR products from 03-07-2017 were added together and DpnI digested <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digestion</A> in 70 µL in total (only 1 µL of milli-Q and 7 µL of CutSmart buffer were used). The resulting sample was again purified (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A>) and NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>). The concentration was determined to be 79.71 ng/µL.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20170704"><H3>RNase Titration</H3><H6>Kasper</H6><P> First the following stock solutions of RNase A (ThermoFisher, 12091021): 2 wt%, 0.2 wt% 0.02 wt%, 0.002 wt%. </P><P> Seven solutions of 100 µL were prepared, each containing 20 mM HEPES, 0.05 wt% polyU, 0.5 wt% spermine and varying amounts of RNase A (0, 10<SUP>-5</SUP>, 10<SUP>-4</SUP>, 10<SUP>-3</SUP>, 10<SUP>-2</SUP> and 10<SUP>-1</SUP> wt%) in nuclease free water, and mixed by gentle pipetting. Also a blank was prepared containing only 20 mM HEPES and nuclease free water. Subsequently 50 µL was taken out of each prepared well and pipetted into another well to make technical duplicates. Then the samples were equilibriated for 5 minutes at 27 °C and the absorbance (500 nm) was measured over 30 minutes every 2 minutes.</P></DIV></DIV><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170704"><H3>TDPs Colony PCR gel</H3><H6>Guillermo and Kelly</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol for the product of colony PCR of colonies that were picked on 03-08-2017 (except for the very last one, 6.8).</P><P>The gel shows a DNA ladder and 19 out of the 20 new colonies (in ascending order) that were picked (only 6.8 is not shown). 45 PCR cycles was too many; unspecific binding results have now been amplified too much. Some of the bands seem to be of the right size (~1,000 bp), but it is very hard to conclude anything from this gel. Due to the unspecifity of the primers, it was decided to miniprep the colonies that were transformed with gene 1 (CAHS_94205_2) and test them with digestion assays.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170704"><H3>Vesicles Colony PCR</H3><H4>Colony PCR</H4><H6>Aafke and Gabriella</H6><P>The 5 picked colonies of TolR of the second transformation and the additional 5 picked colonies of GFP-TorA of the first transformation were checked by colony PCR, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol. 
    The first step was set to be 2.5 minute. The extension time was set to be 2 minutes. The result can be seen in the picture below.</P><P>No bands can be seen on this gel, indicating that there was not enough DNA to run the PCR-reaction on. Based on this gel, we decided to change the first step of the colony PCR protocol to 5 minutes, to boil the bacteria. </P><H4>GFP-TorA plates</H4><H6>Aafke and Gabriella</H6><P>To be able to select for the right insert, we decided to select for GFP activity. The result can be seen in the picture below.</P><P>No GFP activity was detected. Colony 2 was grown in a liquid culture overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 5 July</DIV><DIV class="collapsible-body" id="d20170705"><DIV class="C"><DIV class="row notebook cas13a" id="C20170705"><H3>Cas13a Gibson assembly 1</H3><H6>Jeroen, Kelly and Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#gresuspension" target="_blank">gBlock resuspension</A> protocol was followed for theresuspension of part_1, part_2 and the gene_art_spacer (50 ng/µL). Furthermore, before the Gibson assembly, the purified backbone was checked on gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol).</P><P>Since the backbone looks to be fine, the Gibson assembly (<A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">Gibson assembly</A> protocol) was started. The information about the sizes and volumes of the parts (1:3 ratio of vector to inserts) can be found in the table below.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Part</TH><TH>Amount needed (ng)</TH><TH>Size (bp)</TH><TH>Conc. (ng/µL)</TH><TH>Volume needed (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pSB4A5</TD><TD>100</TD><TD>3395</TD><TD>79.71</TD><TD>1.3</TD></TR><TR><TD class="firstcolumn">Part_1</TD><TD>224</TD><TD>2540</TD><TD>50</TD><TD>4.48</TD></TR><TR><TD class="firstcolumn">Part_2</TD><TD>142</TD><TD>1608</TD><TD>50</TD><TD>2.84</TD></TR><TR><TD class="firstcolumn">Gene_art_spacer</TD><TD>27</TD><TD>309</TD><TD>50</TD><TD>0.54</TD></TR></TBODY></TABLE><P><U>NOTE</U>: because the total volume of the sample would exceed 20 µL if the volumes above were added, the 'DNA mix' with the backbone and the three inserts was partially evaporated to a volume of approximately 5 µL. A control (without any of the inserts) was also incubated. The Gibson mix was transformed in chemically competent Dh5α cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170705"><H3>TDPs Liquid cultures for digestion</H3><H6>Guillermo and Kelly</H6><P>Colonies 1.1-1.4 were grown in a liquid culture overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol. Chloramphenicol was used to select for correct clones.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 6 July</DIV><DIV class="collapsible-body" id="d20170706"><DIV class="C"><DIV class="row notebook cas13a" id="C20170706"><H3>Cas13a Transformation 1 results</H3><H6>Fiona and Jeroen</H6><P>The plates with the transformed cells contained a lot of satellite colonies<SUP>*</SUP>. Below one of the four plates as an example.</P><P>The control plates where also covered with colonies, that were assumed to be non-digested backbone. It was therefore decided to do the transformation again. <SUP>*In hindsight this is due to the too low concentration of ampicillin that was used, but we were not aware of this at that time.</SUP></P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170706"><H3>TDPs Miniprep and digestion</H3><H6>Guillermo and Kelly</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol was followed for the prepared liquid cultures.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Colony #</TH><TH>Concentration (ng/µL)</TH><TH>Average concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1.1</TD><TD>469.98</TD><TD>488.52</TD><TD>500</TD><TD>486</TD></TR><TR><TD class="firstcolumn">1.2</TD><TD>281.14</TD><TD>264.23</TD><TD>258.08</TD><TD>268</TD></TR><TR><TD class="firstcolumn">1.3</TD><TD>226.17</TD><TD>203.82</TD><TD>213.50</TD><TD>215</TD></TR><TR><TD class="firstcolumn">1.4</TD><TD>335.16</TD><TD>336.10</TD><TD>336.28</TD><TD>336</TD></TR></TBODY></TABLE><P>A digestion assay was carried out using PstI and EcoRI in CutSmart buffer, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol.</P><P>The fragments between 1,000 and 1,250 bp match the expected band size for the insert. The bands around 2,000 bp match the expected size for the backbone fragments. It remained unclear why the insert bands were brighter than the backbone bands. It was decided to send the miniprepped sample from colonies 1.2-1.4 for sequencing. These samples were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, with the exception that we only aimed at ~200 ng of DNA. Primer IG0012 was used.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170706"><H3>Vesicles GFP-TorA sequencing</H3><H6>Aafke and Gabriella</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol was followed for the liquid culture of GFP-TorA colony 2.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>232.14</TD></TR></TBODY></TABLE><P>The minipreped sample was prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, with the forward primer IG0012.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 7 July</DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week28-scroll">Week 28</DIV></LI><LI><DIV class="collapsible-header">Monday 10 July</DIV><DIV class="collapsible-body" id="d20170710"><DIV class="C"><DIV class="row notebook cas13a" id="C20170710"><H3>Cas13a Transformation 2</H3><H6>Fiona and Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A>was repeated, this time with TOP10 cells and ampicillin plates that were prepared by the kitchen.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170710"><H3>Culture preparation for digestion</H3><H6>Guillermo and Kelly</H6><P>Extra colonies were picked to check in a digestion assay. We used the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol for the following colonies: </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Gene</TH><TH>Index</TH><TH>Colonies picked</TH><TH>Colonies not picked</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">CAHS_94205</TD><TD>2</TD><TD>2.1-2.4</TD><TD>2.5-2.6</TD></TR><TR><TD class="firstcolumn">CAHS_106094</TD><TD>3</TD><TD>3.1-3.4</TD><TD>3.5-3.9</TD></TR><TR><TD class="firstcolumn">CAHS_107838</TD><TD>4</TD><TD>4.1-4.4</TD><TD>4.5</TD></TR><TR><TD class="firstcolumn">SAHS_33020</TD><TD>5</TD><TD>5.1-5.4</TD><TD>5.5-5.7</TD></TR><TR><TD class="firstcolumn">SAHS_68234</TD><TD>6</TD><TD>6.1-6.4</TD><TD>6.5-6.7</TD></TR><TR><TD>Total #</TD><TD>20</TD><TD>14</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170710"><H3>Vesicles TolR-AU and GFP </H3><H4>Resuspension TolR-AU plasmid</H4><H6>Floor and Gabriella</H6><P>
    TolR-AU plasmid was dissolved according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gresuspension" target="_blank">gBlock resuspension</A> protocol. 
    </P><H4>Sequencing GFP-TorA</H4><H6>Floor and Gabriella</H6><P>
    The sequencing result of GFP-TorA was not alligning with the GFP-TorA sequence. We sent it again with a higher concentration of DNA and the same forward primer IG0012. 
    (<A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol.) 
    </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 11 July</DIV><DIV class="collapsible-body" id="d20170711"><DIV class="C"><DIV class="row notebook cas13a" id="C20170711"><H3>Cas13a Transformation 2 results and backbone optimization</H3><H4>Transformation 2 results</H4><H6>Fiona and Jeroen</H6><P>The plates were again full of colonies, although not satellite colonies this time. One of the four plates is displayed as an example below.</P><P>There were many colonies on both the plates with the inserts and without. The control plates should however be empty, since the DpnI digestion should have cleared the solution with pSB4A5 from all the circular template plasmids. However, the gel from 05-07-2017 has a fat band, indicating the presence of a lot of DNA. This might have caused the DpnI digestion to fail. Therefore, it was decided to pick two times 96 colonies and screen them. Additionally, the backbone PCR and the Gibson assembly should be repeated with less DNA template.</P><H4>Backbone PCR</H4><H6>Fiona and Jeroen</H6><P>The Phusion PCR (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A>) was repeated with less template DNA (1 µL). The annealing temperature was set to be 57 °C and the extension time was 105 seconds. <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> resulted in the following gel:</P><P>The PCR product (2<SUP>nd</SUP> lane) was of the correct size and was thus purified (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification</A>), <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digested</A>, purified again and <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">NanoDropped</A>. The concentration of the final product was determined to be 8.52 ng/µL.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170711"><H3>TDPs New sequencing samples</H3><H4>Preparation of sequencing samples</H4><H6>Guillermo</H6><P>The results from the sequencing samples from 06-07-2017 came back in. However, the chromatograms were of very bad quality. This could be due to either the lower concentration of DNA or a bad sequencing primer (IG0012). Therefore, the samples 1.2-1.4 were prepared again, following the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol. We used both the forward (IG0012) and the backward primer (IG0013) for these three samples, making six samples in total.</P><H4>Miniprep of extra colonies</H4><H6>Kelly and Isabell</H6><P>The 20 colonies that were picked on 10-07-2017 were miniprepped using the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170711"><H3>Vesicles Colony PCR GFP-TorA </H3><H6>Floor and Gabriella</H6><P>The result of the colony PCR of GFP-TorA can be seen in the picture below.</P><P>The positive control gives the expected bands. #13 and #15 clearly show the expected band at ~1200 bp. Colony #11 and #12 show a vague band at the expected size.</P><P>Colony 13 and 15 Colony 2 were grown in a liquid culture overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 12 July</DIV><DIV class="collapsible-body" id="d20170712"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170712"><H3>TDPs Sequencing results and conclusions</H3><H6>Guillermo, Isabell and Kelly</H6><P>The chromatograms of the samples with the forward primer (IG0012) were of very bad quality, indicating that this primer is not suitable as a sequencing primer for backbone pSB1C3. The chromatograms of the samples with primer IG0013 were of better quality, but they showed the insert is absent in all three samples. The sequencing results did however indicate an extra (forbidden) PstI and EcorI restriction site in the sequence of colony 1.3 and 1.2, respectively. This (partially) explains the results obtained with the gel on 06-07-2017, as the backbone is thus cut in two halves of approximately 1,000 bp.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170712"><H3>Vesicles Verification GFP-TorA</H3><H4>Sequencing preparation</H4><H6>Floor and Gabriella</H6><P>The overnight cultures were minipreped according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA #13</TD><TD>99.40</TD><TD>1.72</TD></TR><TR><TD class="firstcolumn">GFP-TorA #15</TD><TD>89.94</TD><TD>2.12</TD></TR></TBODY></TABLE><P>The sample of colony 15 was prepared for sequencing with the reverse primers IG0013.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol.)</P><H4>Digestion GFP-TorA</H4><H6>Floor and Gabriella</H6><P>The GFP-TorA samples #13 and #15 were digested with EcoRI and PstI.The samples were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 25 µL total. 
    The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol).The result can be seen on the picture below. (with ND standing for no digestion, E for EcorI and P for PstI) </P><P>The insert should give a band at ~1250 bp. We found out that the primers used weren't optimal, explaining these non-matching results.</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 13 July</DIV><DIV class="collapsible-body" id="d20170713"><DIV class="C"><DIV class="row notebook cas13a" id="C20170713"><H3>Cas13a Gibson assembly 2</H3><H6>Fiona and Jeroen</H6><P>A new <A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">Gibson assembly</A> reaction was set up, using the new purified backbone. For this assembly, vector and inserts were used in a 1:1 ratio, according to the table below.</P><TABLE><THEAD><TR><TH>Part</TH><TH>Amount needed (ng)</TH><TH>Size (bp)</TH><TH>Conc. (ng/µL)</TH><TH>Volume needed (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pSB4A5</TD><TD>50</TD><TD>3395</TD><TD>8.52</TD><TD>5.88</TD></TR><TR><TD class="firstcolumn">Part_1</TD><TD>74.81</TD><TD>2540</TD><TD>50</TD><TD>1.5</TD></TR><TR><TD class="firstcolumn">Part_2</TD><TD>27.36</TD><TD>1608</TD><TD>50</TD><TD>0.95</TD></TR><TR><TD class="firstcolumn">Gene_art_spacer</TD><TD>9.10</TD><TD>309</TD><TD>50</TD><TD>0.18</TD></TR></TBODY></TABLE><P>A control reaction (without any inserts) was also prepared. </P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170713"><H3>TDPs Backbone check and digestion assays</H3><H6>Guillermo, Isabell and Kelly</H6><P>Based on the sequencing results from 12-07-2017, we wanted to ensure that the backbone in our PCR product was not mutated. Therefore, the two PCR samples from 27-06-2017 and 28-06-2017 were both NanoDropped according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">NanoDrop</A> protocol. The miniprepped samples from colonies 3.1-3.4 and 6.1-6.4 were NanoDropped in the same manner. These genes (colonies) were chosen as the plates of these genes had the most colonies and thus the highest chance of a correct clone. The NanoDrop results are displayed in the table below.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Gene index/Sample</TH><TH>Colony</TH><TH>Concentration (ng/µL)</TH><TH>Average concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">3</TD><TD>3.1</TD><TD>140.55</TD><TD>146.35</TD><TD>151.05</TD><TD>145.98</TD></TR><TR><TD>3.2</TD><TD>90.02</TD><TD>37.01</TD><TD>99.69</TD><TD>75.57</TD></TR><TR><TD>3.3</TD><TD>110.10</TD><TD>109.67</TD><TD>108.74</TD><TD>1109.50</TD></TR><TR><TD>3.4</TD><TD>130.03</TD><TD>189.35</TD><TD>188.24</TD><TD>169.21</TD></TR><TR><TD class="firstcolumn">6</TD><TD>6.1</TD><TD>127.50</TD><TD>127.11/td&gt;</TD><TD>128.05</TD><TD>127.55</TD></TR><TR><TD>6.2</TD><TD>85.38</TD><TD>84.65</TD><TD>86.98</TD><TD>85.67</TD></TR><TR><TD>6.3</TD><TD>140.51</TD><TD>140.06</TD><TD>-</TD><TD>140.29</TD></TR><TR><TD>6.4</TD><TD>122.81</TD><TD>122.18</TD><TD>121.51</TD><TD>122.17</TD></TR><TR><TD class="firstcolumn">PCR 1 (27-06-2017)</TD><TD>-</TD><TD>41.78</TD><TD>44.69</TD><TD>45.90</TD><TD>44.12</TD></TR><TR><TD class="firstcolumn">PCR 2 (28-06-2017)</TD><TD>-</TD><TD>65.93</TD><TD>64.88</TD><TD>72.08</TD><TD>67.30</TD></TR></TBODY></TABLE><P>The backbone samples (PCR 1 and PCR 2) were digested with PstI; the other samples (gene 3 and 6) were digested with both PstI and EcoRI. The samples were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 20 µL total. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol).</P><P>It is unclear what happened to the gel. Nevertheless, no conclusions could be drawn from this result. The digestion of the miniprepped samples and the gel electrophoresis were repeated by our supervisor Sebastian, leading to the gel results below.</P><P>From this gel, it could be concluded that all samples contained an intact backbone (~2,000 bp, the red band). However, none of the samples contained the insert (~1,000 bp). Therefore, it was concluded to do the Gibson assembly again. </P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170713"><H3>Vesicles Colony PCR</H3><H6>Floor and Gabriella</H6><P>As we found out that the primers weren't optimal for our DNA-template, we did the colony PCR again with primers IG0022 and IG0013 (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol). However, no bands could be seen on the gel.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 14 July</DIV><DIV class="collapsible-body" id="d20170714"><DIV class="C"><DIV class="row notebook cas13a" id="C20170714"><H3>Cas13a Colony picking and transformation 3</H3><H4>Colony picking</H4><H6>Jasper and Jeroen</H6><P>96 colonies were picked from the 2<SUP>nd</SUP> round of transformation (10-07-2017). This was done as described in the first steps of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol, but then for many samples at once (using two 96-wells plates).</P><H4>PCR</H4><H6>Jasper and Jeroen</H6><H4>Transformation</H4><H6>Jeroen</H6><P>Another <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation</A> with Gibson assembly mix 2 (from 13-07-2017) was carried out. Chemical competent TOP10 cells that were prepared during the lab training week were used. The cells were plated out on ampicillin plates that were prepared 12-07-2017.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170714"><H3>TDPs Backbone amplification</H3><H6>Guillermo and Kelly</H6><P>To ensure that the backbone used in the Gibson assembly was correct and to get more of the backbone, we did another round of backbone amplification. We used a sample containing pSB1C3 with RFP insert from the 2010 team. It was called CsgA-HA. The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was carried out, with an annealing temperature of 57 °C and an extension time of 1 minute. Next, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR purification</A> and the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digestion</A> protocol were executed. After the digestion, the sample was again purified and run on gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol).</P><P>The band around 2,000 bp (red-lined rectangle) in lane 2 and 3 is very vague, but nonetheless indicates product of the correct size. Therefore, it was decided that this sample could be used for Gibson assembly. The concentration of the sample was determined (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">NanoDrop</A> protocol) to be 167 ng/µL.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170714"><H3>Vesicles Verification TolR-AU </H3><H4>Digestion TolR-AU plasmid</H4><H6>Aafke and Floor</H6><P>The TolR-AU samples (1.6, 1.10, 2.1-2.3) were minipreped according to the </P><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol was followed for the liquid culture of GFP-TorA colony 2.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU 1.6</TD><TD>304.51</TD></TR><TR><TD class="firstcolumn">TolR-AU 1.10</TD><TD>277.30</TD></TR><TR><TD class="firstcolumn">TolR-AU 2.1</TD><TD>252.86</TD></TR><TR><TD class="firstcolumn">TolR-AU 2.2</TD><TD>274.31</TD></TR><TR><TD class="firstcolumn">TolR-AU 2.3</TD><TD>390.48</TD></TR></TBODY></TABLE><P>The samples were digested with XbaI and NdeI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 25 µL total. 
    The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol). The result can be seen on the picture below. </P><P>A band of ~450 bp was expected. Based on this gel, it might be possible that the digestion didn't work. Next time, include a non-digested control as well. </P><H4>GFP activity check</H4><H6>Aafke, Floor and Gabriella</H6>
    To check for GFP activity , the cells transformed with GFP-TorA are plated on a new plate with 1 mM IPTG.
    
    
    </DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week29-scroll">Week 29</DIV></LI><LI><DIV class="collapsible-header">Monday 17 July</DIV><DIV class="collapsible-body" id="d20170717"><DIV class="C"><DIV class="row notebook cas13a" id="C20170717"><H3>Cas13a Colony PCR and digestion Gootenberg plasmid</H3><H4>Colony PCR</H4><H6>Jasper and Jeroen</H6><P>The colonies that were picked first (14-07-2017; 96-wells plate) were used as a template in <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank">colony PCR (GoTaq)</A> with primers IG0022 and IG0023. Additionally, in the same round of colony PCR 8 colonies from the third transformation (14-07-2017) were used. The results are displayed in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> photos below.</P><P>In the results, we should be seeing a product of around 4000 base pairs. This is clearly not the case. After a discussion with a supervisor, we realised that the geneArt fragment came in circulairzed form, and we had not linearized it before the Gibson assembly. This explains why the experiment was not successful.</P><H4>Restriction Digestion</H4><H6>Jeroen</H6><P>The Gootenberg plasmid was used as the template in a double digestion with enzymes XhoI and XbaI. We used the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank"> digestion assay protocol</A>. The result was visualised on a <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> as shown below. In the first and second lane, two samples of the digestion are shown. In the third lane, a positive control is shown</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170717"><H3>TDPs Second round of Gibson assembly</H3><H6>Guillermo and Kelly</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">agar plate</A> protocol. was followed to prepare plates for transformed cells. Next, the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">Gibson assembly</A> protocol was carried out, with a 1:2 ratio of vector:inserts.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Part</TH><TH>Amount needed (ng)</TH><TH>Size (bp)</TH><TH>Conc. (ng/µL)</TH><TH>Volume needed (µL)</TH><TH>Size + T7 promoter</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">T7 promoter</TD><TD>124</TD><TD>128</TD><TD>10</TD><TD>1.3</TD><TD>-</TD></TR><TR><TD class="firstcolumn">CAHS_94205_2</TD><TD>86.1</TD><TD>891</TD><TD>50</TD><TD>1.7</TD><TD>1019</TD></TR><TR><TD class="firstcolumn">CAHS_94205</TD><TD>86.4</TD><TD>894</TD><TD>50</TD><TD>1.7</TD><TD>1022</TD></TR><TR><TD class="firstcolumn">CAHS_106094</TD><TD>86.4</TD><TD>894</TD><TD>50</TD><TD>1.7</TD><TD>1022</TD></TR><TR><TD class="firstcolumn">CAHS_107838</TD><TD>87.0</TD><TD>900</TD><TD>50</TD><TD>1.7</TD><TD>1028</TD></TR><TR><TD class="firstcolumn">SAHS_33020</TD><TD>69.5</TD><TD>719</TD><TD>50</TD><TD>1.4</TD><TD>847</TD></TR><TR><TD class="firstcolumn">SAHS_68234</TD><TD>71</TD><TD>735</TD><TD>50</TD><TD>1.4</TD><TD>863</TD></TR></TBODY></TABLE><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol was followed for TOP10 cells, with the exception that 10 µL of Gibson assembly mix was used (which is slightly more than indicated in the protocol).</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170717"><H3>Vesicles Digestions </H3><H4>Results plates GFP-TorA</H4><H6>Aafke and Floor</H6><P>No fluorescence was detected on the plates </P><H4>Digestion TolR-AU</H4><H6>Aafke and Floor</H6><P>The samples were digested with XbaI and NdeI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 25 µL total. 
    The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol). The result can be seen on the picture below. </P><P>Both digested and non-digested samples display the same bands. It might be that the restriction enzymes didn't work.
    
    </P><H4>Check restriction enzymes</H4><H6>Aafke and Floor</H6><P>To check the activity of the used restriction enzymes, we were provided with a sample (pBH), which was digested with XbaI and NdeI. If both restriction enzymes worked, we would get the following bands:
    -5365 bp
    -3673 bp
    -1231 bp
    -241 bp
    -164 bp
    -33 bp</P><P>The sample was prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with 25 µL total.
    The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol). The result can be seen in the picture below. </P><P>NdeI worked for sure. Based on this gel we cannot conclude if XbaI is still active, so we decided to use NdeI and SphI for the next digestion.
    
    </P><H4>Digestion TolR-AU</H4><H6>Aafke and Floor</H6><P>The sample (TolR-AU 1.3) was digested with SphI and NdeI.The sample was prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with 25 µL total and left overnight.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 18 July</DIV><DIV class="collapsible-body" id="d20170718"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170718"><H3>TDPs Backbone (and GFP) amplification</H3><H6>Kelly</H6><P>The colonies from yesterday were still very small. Therefore, colony picking was postponed until Wednesday 19-07-2017. Due to the vesicle module changing tactics from Gibson assembly to restriction-ligation, they needed more pSB1C3. Therefore, another backbone amplification was carried out. The same steps as on 14-07-2017 were followed; <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> with an annealing temperature of 57 °C and an extension time of 1 minute; <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR purification</A> and <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digestion</A>; purification and <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A>. The GFP insert was amplified at the same time, but this was done by the vesicle module themselves. The agarose gel shows the expected band (~2 kbp) for 3 out of the 4 prepared samples, so the deviating sample was thrown away. </P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170718"><H3>Vesicles Digestion and parts amplification</H3><H4>Results digestion TolR-AU</H4><H6>Aafke and Floor</H6><P>The digested sample was loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol).
     The result can be seen on the picture below. </P><P> No clear bands are visible for the TolR-AU sample, meaning that the plasmid is not yet verified.  </P><H4>Amplification of pSB1C3 and GFP-TorA</H4><H6>Aafke and Floor</H6><P>We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol. 
    For GFP-TorA 1µL of DNA template was used and 1µL of the primers IG0023 and IG0022. 
    0.5 µL of pSB1C3 was used together with 1 µL of the primers IG0010 and IG0011. 
    The PCR program was run with an extension time of 28 seconds for GFP-TorA and 60 seconds for pSB1C3.</P><P> The PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below (with lane 5 and 7 empty). </P><P>As no other bands are significantly visible on the gel, both GFP-TorA and pSB1C3 are pure enough to be purified from the PCR mixture itself. Only #2 of pSB1C3 is discarded.
    </P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 19 July</DIV><DIV class="collapsible-body" id="d20170719"><DIV class="C"><DIV class="row notebook cas13a" id="C20170719"><H3>Amplification of parts 1 and 2</H3><H4>Resuspension of primers</H4><H6>Jeroen</H6><P>Primers IG0029 - IG0032 were resuspended by adding an amount of sterile miliQ to bring the solutions to a concentration of 100mM.</P><TABLE><THEAD><TR><TH>Primer </TH><TH>MiliQ added (uL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">IG0029</TD><TD>312</TD></TR><TR><TD class="firstcolumn">IG0030</TD><TD>237</TD></TR><TR><TD class="firstcolumn">IG0031</TD><TD>213</TD></TR><TR><TD class="firstcolumn">IG0032</TD><TD>215</TD></TR></TBODY></TABLE><P>After adding the appropriate amount of miliQ, the tubes were incubated for 5 minutes at 37° and subsequently vortexed. After this, 10mM solutions were made by making a 10x solution with miliQ.</P><H4>Phusion PCR</H4><H6>Jeroen</H6><P>To amplify part 1, primers IG0029 and IG0022 were used. For part 2, primers IG0030 and IG0031 were used. The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">Phusion PCR protocol</A> was followed.</P><H4>Results and discussion</H4><P>An <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> was made to visualise the result, as shown below. In the first lane we see part 1 and in the second lane we see part 2. The sizes are not what we would expect, so we concluded the PCR to be unsuccesful. We decided to adjust the Tm to 55° and the extension time to 1.5 minutes.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170719"><H3>TDPs Colony counting, picking and PCR</H3><H4>Colony counting and picking</H4><H6>Guillermo and Kelly</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>75 µL plate</TH><TH>175 µL plate</TH><TH>Total # of white colonies</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Gene index</TD><TD>Red <SUP>*</SUP></TD><TD>White</TD><TD>Red <SUP>*</SUP></TD><TD>White</TD></TR><TR><TD class="firstcolumn">1</TD><TD>6</TD><TD>4</TD><TD>3</TD><TD>14</TD><TD>18</TD></TR><TR><TD class="firstcolumn">2</TD><TD>0</TD><TD>4</TD><TD>1</TD><TD>4</TD><TD>8</TD></TR><TR><TD class="firstcolumn">3</TD><TD>2</TD><TD>2</TD><TD>0</TD><TD>7</TD><TD>9</TD></TR><TR><TD class="firstcolumn">4</TD><TD>7</TD><TD>5</TD><TD>4</TD><TD>32</TD><TD>37</TD></TR><TR><TD class="firstcolumn">5</TD><TD>2</TD><TD>7</TD><TD>4</TD><TD>13</TD><TD>20</TD></TR><TR><TD class="firstcolumn">6</TD><TD>2</TD><TD>4</TD><TD>5</TD><TD>9</TD><TD>13</TD></TR><TR><TD class="firstcolumn">Negative control</TD><TD>0</TD><TD>1</TD><TD>0</TD><TD>1</TD><TD>2</TD></TR></TBODY></TABLE><P><SUP>*</SUP>These colonies most likely originate from non-digested motherstrand. </P><P>The photo shows the plate of gene 2 of the 175 µL plate. Since the (white) cells looked healthy and there was a great enough number of them (especially compared to the control plates) we decided to pick white colonies and PCR them.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TD class="firstcolumn">Gene index</TD><TD>75 µL plate</TD><TD>175 µL plate</TD><TD>Picked</TD><TD>#</TD></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1.1-1.4</TD><TD>1.5-1.18</TD><TD>1.1-1.2,1.5-1.8</TD><TD>6</TD></TR><TR><TD class="firstcolumn">2</TD><TD>2.1-2.4</TD><TD>2.5-2.8</TD><TD>2.1-2.3, 2.5-2.6</TD><TD>5</TD></TR><TR><TD class="firstcolumn">3</TD><TD>3.1-3.2</TD><TD>3.3-3.9</TD><TD>3.1-3.5</TD><TD>5</TD></TR><TR><TD class="firstcolumn">4</TD><TD>4.1-4.5</TD><TD>4.6-4.37</TD><TD>4.1, 4.6-4.10</TD><TD>6</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.1-5.7</TD><TD>5.8-5.20</TD><TD>5.1-5.2, 5.8-5.11</TD><TD>6</TD></TR><TR><TD class="firstcolumn">6</TD><TD>6.1-6.4</TD><TD>6.5-6.13</TD><TD>6.1-6.2, 6.5-6.8</TD><TD>6</TD></TR><TR><TD class="firstcolumn">Negative control</TD><TD>x1</TD><TD>x2</TD><TD>x1</TD><TD>1</TD></TR></TBODY></TABLE><H4>Primer resuspension</H4><H6>Kelly</H6><P>Primers IG0006, IG0026, IG0027 and IG0028 were dissolved according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#primerworkingstockpreparation" target="_blank">primer working stock preparation</A> protocol.</P><H4>Colony PCR</H4><H6>Kelly</H6><P>The picked colonies were used in a colony PCR (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR</A> protocol). Primers IG0006 and IG0013 were used. The annealing temperature used was 55 °C and the extension time was 66 seconds.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170719"><H3>Vesicles Digestion and ligation</H3><H4>Digestion GFP-TorA and pSB1C3</H4><H6>Aafke and Floor</H6><P>Prior to digestion, a PCR clean-up was performed according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol.
    The DNA concentration was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>212.61</TD><TD>1.83</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>53.6</TD><TD>1.91</TD></TR></TBODY></TABLE><P>The digestion was performed with EcoRI and PstI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 50 µL total.</P><P>As PstI could not be inactivated by a temperature change, a clean-up was performed according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol. </P><H4>Ligation GFP-TorA and pSB1C3</H4><H6>Aafke and Floor</H6><P>The vector and inserts were used in a 1:3 ratio. 
    To be able to calculate this, first the concentration of both vector and inserts was determined using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>17.35</TD><TD>2.18</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>29.21</TD><TD>2.02</TD></TR></TBODY></TABLE><P>The ligation was performed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation (sticky-end)</A> protocol.The samples were incubated at roomtemperature overnight.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>3</TD></TR><TR><TD class="firstcolumn">Backbone pSB1C3</TD><TD>4</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>10</TD></TR><TR><TD class="firstcolumn">Buffer</TD><TD>2</TD></TR><TR><TD class="firstcolumn">Ligase</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 20 July</DIV><DIV class="collapsible-body" id="d20170720"><DIV class="C"><DIV class="row notebook cas13a" id="C20170720"><H3>Amplification of parts 1, 2 and the GeneArt spacer</H3><H4>Phusion PCR</H4><H6>Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed with: 1 µL of C6 (a 1 µg/µL dilution of the part1), 1 µL of C7 (a 1 µg/µL dilution of the part2) and 1 µL of C8 (a 1 µg/µL dilution of the GeneArt spacer). Primers IG0022 and IG0029 were used for C6, primers IG0030 and IG0031 were used for C7 and primers IG0032 and IG0023 were used for C8. Annealing temperatures of 55  °C were used for all templates. The extension time was set at 1 minute and 30 seconds for templates C6 and C7 and 20 seconds for template C8. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 50 minutes.</P><P>It was concluded from the gel that the PCR amplification of the GeneArt spacer was succesful since it the gel shows a thick band above the 250 bp, which corresponds with the expected 304 bp. Furthermore the PCR product of C7 shows three bands: one clear band around 1500 bp, which corresponds with the expected 1608 bp, one faint band below the 1500 bp and one band at 1000 bp. The latter two could be a product of aspecific binding of the primers. Also a smear is present. Increasing the annealing temperature could increase the binding specificity and might result in a better result. At last, we concluded that the PCR amplification of C6 failed. Optimizing the annealing temperature could solve this problem. The new annealing temperature for C6 and C7 were calculated with the <A calculator="" href="http://tmcalculator.neb.com/#!" neb="" tm="">NEB Tm calculator</A>. The new annealing temperaturs are: </P></DIV></DIV></DIV></LI><LI>For C6 (part1): 72 °C</LI><LI>For C7 (part2): 62 °C</LI></UL></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170720"><H3>TDPs Colony PCR check and new colony PCR</H3><H4>Colony PCR check</H4><H6>Isabell and Kelly</H6><P>An agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol) was run to check the colony PCR's from yesterday.</P><P>The expected size without an insert was 261 bp. A band of this size is visible in the lane with the negative control (x1) and a lot of the other lanes. The expected size for gene 1-4 was ~1,200 bp and for gene 5-6 ~1,000 bp. A band of this size is present in some of the lanes. Therefore, the following colonies were grown in a liquid culture (<A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol): 1.5, 3.2, 3.5, 4.6, 4.8, 4.9, 5.1, 6.1, 6.2, 6.5-6.8.</P><H4>New colony PCR</H4><H6>Isabell and Kelly</H6><P>The following new colonies were picked:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TD class="firstcolumn">Gene index</TD><TD>Colonies picked</TD><TD>#</TD></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1.3, 1.4, 1.9-1.11</TD><TD>5</TD></TR><TR><TD class="firstcolumn">2</TD><TD>2.4, 2.7-2.8</TD><TD>3</TD></TR><TR><TD class="firstcolumn">3</TD><TD>3.6-3.9</TD><TD>4</TD></TR><TR><TD class="firstcolumn">4</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.3-5.4, 5.12-5.14</TD><TD>5</TD></TR><TR><TD class="firstcolumn">6</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">Negative control</TD><TD>x2</TD><TD>1</TD></TR></TBODY></TABLE><P>The colony PCR was carried out as on 19-07-2017 (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR</A> protocol).</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170720"><H3>Vesicles Transformation GFP-TorA and TolR-AU</H3><H4>Electrophoresis TolR-AU</H4><H6>Aafke and Floor</H6><P>The electrocompetent cells were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol with TOP10-cells. Next, the cells were transformed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>Heat-shock GFP-TorA</H4><H6>Aafke and Floor</H6><P>50 µL TOP10 cells that were made competent during the iGEM practice week were transformed with 15 µL of GFP-TorA plasmid according to 
    the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. 
    The cells were plated out on chloramphenicol plates.</P></DIV></DIV></DIV><LI><DIV class="collapsible-header">Friday 21 July</DIV><DIV class="collapsible-body" id="d20170721"><DIV class="C"><DIV class="row notebook cas13" id="C20170721"><H3>Amplification of parts 1 and 2 and cleaning the PCR product of the GeneArt spacer</H3><H4>Cleaning and concentration of the GeneArt spacer</H4><H6>Jeroen</H6><P>The PCR product of the GeneArt spacer from 20-07-2017 is DpnI digested <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digestion</A> in an total volume of 40 μL (7 μL CutSmart buffer, 1 μL milli-Q and 2 μL DpnI). The product was purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" tarhet="_blank"> PCR product purification (Promega Wizard SV™ Kit) protocol</A>. The purified sample was NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>) and labelled as C11. The concentration was 49.04 ng/μL with a 260/280 ratio of 2.63. This ratio is higher than desirable, so we decided to repeat the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">purification step</A> and measure the concentration again (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>). This time the concetration of C11 is 39.87 ng/μL with a 260/280 ratio of 1.67, which is better.</P><H4>Phusion PCR</H4><H6>Jeroen</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed with: 1 µL of the PCR product of C6 from 20-07-2017 (part1) and the PCR product of C7 from 20-07-2017 (part2). Primers IG0022 and IG0029 were used for part1 and primers IG0030 and IG0031 were used for part2. Two gradients were used for the annealing temperatures:</P></DIV></DIV></DIV></LI><LI>For part1:
                </LI><LI>65 °C</LI><LI>70 °C</LI><LI>72 °C</LI><LI>For part2:
                </LI><LI>55 °C</LI><LI>62 °C</LI><LI>65 °C</LI><P>The extension time was set at 1 minute and 45 seconds. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 50 minutes.</P><P>The gradient was for both part not a succes. While the PCR on part1 gave no result at all, there is a faint band at the right height for part 2, but it is still not really specific.</P></DIV></MAIN></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170721"><H3>TDPs Miniprep, sequencing and colony PCR check</H3><H4>Miniprep and sequencing</H4><H6>Isabell and Kelly</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol was carried out (followed by <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">NanoDropping</A>).</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Conc. (ng/µL)</TH></TR></THEAD><THEAD><TR><TH>Gene index</TH><TH>Colony</TH><TH>Run 1</TH><TH>Run 2</TH><TH>Run 3</TH><TH>Average</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1.5</TD><TD>148.67</TD><TD>148.27</TD><TD>150.28</TD><TD>149.07</TD></TR><TR><TD class="firstcolumn">3</TD><TD>3.2</TD><TD>197.23</TD><TD>191.69</TD><TD>190.83</TD><TD>191.25</TD></TR><TR><TD>3.5</TD><TD>230.88</TD><TD>232.24</TD><TD>234.75</TD><TD>232.62</TD></TR><TR><TD class="firstcolumn">4</TD><TD>4.6</TD><TD>221.87</TD><TD>224.01</TD><TD>226.74</TD><TD>224.21</TD></TR><TR><TD>4.8</TD><TD>225.61</TD><TD>227.69</TD><TD>226.82</TD><TD>226.71</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.1</TD><TD>243.62</TD><TD>245.65</TD><TD>245.97</TD><TD>245.08</TD></TR><TR><TD class="firstcolumn">6</TD><TD>6.1</TD><TD>212.62</TD><TD>216.66</TD><TD>217.45</TD><TD>215.58</TD></TR><TR><TD>6.2</TD><TD>206.76</TD><TD>207.45</TD><TD>206.94</TD><TD>207.05</TD></TR></TBODY></TABLE><P>These samples were prepared to send for sequencing (<A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol) with primer IG0006.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Colony</TH><TH>Label</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1.5</TD><TD>18A1ZAG130</TD></TR><TR><TD class="firstcolumn">3.2</TD><TD>18A1ZAG129</TD></TR><TR><TD class="firstcolumn">3.5</TD><TD>18A1ZAG128</TD></TR><TR><TD class="firstcolumn">4.6</TD><TD>18A1ZAG127</TD></TR><TR><TD class="firstcolumn">4.8</TD><TD>18A1ZAG126</TD></TR><TR><TD class="firstcolumn">5.1</TD><TD>18A1ZAG125</TD></TR><TR><TD class="firstcolumn">6.1</TD><TD>18A1ZAG124</TD></TR><TR><TD class="firstcolumn">6.2</TD><TD>18A1ZAG123</TD></TR></TBODY></TABLE><H4>Colony PCR gel and discussion</H4><H6>Isabell and Kelly</H6><P>It can be concluded from this gel (for expected sizes, see 20-07-2017 entry) that only 5.13 is a positive clone. Furthermore, clones 1.3, 1.4, 5.4 and 3.6 did not show any bands at all. Therefore, it was decided to check these on gel again. Moreover, since there were no new positive clones for gene 1, 2 and 3, more colonies were picked.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TD class="firstcolumn">Gene index</TD><TD>Colonies picked (new)</TD><TD>Colonies on gel (old)</TD><TD>#</TD></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>12-18</TD><TD>3, 4</TD><TD>9</TD></TR><TR><TD class="firstcolumn">2</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">3</TD><TD>-</TD><TD>6</TD><TD>1</TD></TR><TR><TD class="firstcolumn">4</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">5</TD><TD>15-20</TD><TD>4</TD><TD>7</TD></TR><TR><TD class="firstcolumn">6</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">Negative control</TD><TD>x2</TD><TD>-</TD><TD>1</TD></TR></TBODY></TABLE><P>The sample preparation and the program (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol) were the same as on 19-07-2017.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170721"><H3>Vesicles Colony PCR TolR-AU and Ligation GFP-TorA</H3><H4>Results transformation TolR-AU</H4><H6>Aafke and Floor</H6><P>The transformation was succesful, two colonies were picked.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypickingcontrol" target="_blank">colony picking</A> protocol)</P><H4>Colony PCR TolR-AU</H4><H6>Aafke and Floor</H6><P>The colony PCR was carried out with 1 colony from transformation #1, 1 colony from transformation #2 and 2 from tranformation #3 as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol, with an annealing temperature of 54 °C.</P><P>The results of the colony PCR can be seen in the picture below.</P><P>The gel seems fine, but we are going to verify with digestion to be sure.</P><H4>Ligation GFP-TorA</H4><H6>Aafke and Floor</H6><P>The transformation was not succesful, as there was no growth on the plates. The ligation was performed again following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation (sticky-end)</A> protocol.</P>
    The samples were incubated at 4 °C over the weekend.
    </DIV></DIV></DIV></DIV><LI><DIV class="collapsible-header scrollspy" id="week30-scroll">Week 30</DIV></LI><LI><DIV class="collapsible-header">Monday 24 July</DIV><DIV class="collapsible-body" id="d20170724"><DIV class="C"><DIV class="row notebook cas13" id="C20170724"><H3>Amplification of the backbone and parts, Gibson assembly and transformation</H3><H4>Phusion PCR</H4><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed with: 1 µL of C5 (psB4A5) in 10 μL milli-Q. Seven tubes were made. The used annealing temperature used is 57 °C, with an extension time of 1 minute and 45 seconds. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 50 minutes.</P><P>A part from well 4 there are no clear bands. We concluded to clean this up anyway.</P><H4>Cleaning and concentration of psB4A5</H4><H6>Jeroen</H6><P>The PCR product of psB4A5 from 24-07-2017 is purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" tarhet="_blank"> PCR product purification (Promega Wizard SV™ Kit) protocol</A>. The purified sample was NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>) and labelled as C17 (tubes 1-4) and C18 (tube 5-7). The concentrations and 260/280 ratios were:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Tube</TH><TH>Concentration (ng/µL)</TH><TH>260/280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">C17</TD><TD>241.24</TD><TD>1.81</TD></TR><TR><TD class="firstcolumn">C18</TD><TD>231.65</TD><TD>1.83</TD></TR></TBODY></TABLE><H4>Phusion PCR</H4><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed for amplifying part1 and part2. For part1 1 µL of C6 is used with primers IG0022 and IG0029. We used a gradient for the annealing temperature with three tubes (65 °C, 70 °C and 72 °C). The used extension time is 1 minute and 45 seconds and instead of repeating steps 2-4 30 times, it is only repeated 12 times. For part2 1 µL of C7 is used with primers IG0030 and IG0031. We used a gradient for the annealing temperature with three tubes (55 °C, 57 °C and 62 °C). The used extension time is 1 minute and 45 seconds and steps 2-4 is repeated 30 times. The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 50 minutes.</P><P>P1 and P2 stand for part1 and part2. The number that follows is the used annealing temperature. There are bands on the right heights for part1 (~2500 bp) and part2 (~1600 bp), however on the lanes from part1 we can see a second fainter band at around 2000 bp. We decided to gel isolate P1 65 and P1 70 from the gel. P2 55, P2 57 and P2 62 can cleaned immediately.</P><H4>Cleaning and concentration of part1 and part2</H4><H6>Fiona</H6><P>Part1 (P1 65 and P1 70) were isolated from the agarose gel and purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" tarhet="_blank"> Gel isolation and purification (Promega Wizard™ Kit) protocol</A>. Part2 (P2 55, P2 57 and P2 62) was purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" tarhet="_blank"> PCR product purification (Promega Wizard SV™ Kit) protocol</A>. The purified part1 is labelled as C22 and the purified part2 is labelled as C19. The samples were NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>). The concentrations and 260/280 ratios were:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Tube</TH><TH>Concentration (ng/µL)</TH><TH>260/280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">C22</TD><TD>6.51</TD><TD>2.93</TD></TR><TR><TD class="firstcolumn">C19</TD><TD>70.48</TD><TD>-</TD></TR></TBODY></TABLE><H4>Cas13a Gibson assembly
            </H4><H6>Jeroen</H6><P>The Gibson assembly (<A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly" target="_blank">NEB Gibson assembly</A> protocol) was used. The information about the sizes and volumes of the parts (1:3 ratio of vector to inserts) can be found in the table below.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Part</TH><TH>Amount needed (ng)</TH><TH>Size (bp)</TH><TH>Conc. (ng/µL)</TH><TH>Volume needed (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pSB4A5 (C17)</TD><TD>50</TD><TD>3395</TD><TD>241.24</TD><TD>0.5</TD></TR><TR><TD class="firstcolumn">Part_1</TD><TD>74.81</TD><TD>2540</TD><TD>50</TD><TD>1.5</TD></TR><TR><TD class="firstcolumn">Part_2 (C19)</TD><TD>47.36</TD><TD>1608</TD><TD>70.48</TD><TD>0.7</TD></TR><TR><TD class="firstcolumn">Gene_art_spacer (C11)</TD><TD>9.1</TD><TD>309</TD><TD>39.87</TD><TD>0.5</TD></TR></TBODY></TABLE><P>6.8 μL sterile milli-Q was added. For the control o.5 μL psB4A5 (C17) was used with 9.5 μL sterile milli-Q. The mixes were incubated 1 h at 37∇°C. The Gibson mix was transformed in chemically competent top10 cells (made in February) according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. The Gibson mix with insert was plated on two Amp plates (20 μL and 100 μL) and 100 μL of the control Gibson mix was plated on one Amp plate. The remaining cell were stored at 4 °C as back up.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170724"><H3>sample prep DNA gel electrophoresis of RPA with control primers and -DNA</H3><H6>Amária and Kasper</H6><P>
    We purified the DNA Amplified on the 24<SUP>th</SUP> of July by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol. We used 6&amp;nsbp;µL 100bp ex load biorad ladder and  instead of 1kb a 500bp ez load biorad ladder (it is faulty depicted on the picture from the gel) Then we performed the
    <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol. We prepaired a 1 % agarose gel. We ran it at 100V for 40 min.The following picture shows the resulting DNA gel:
      </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170724"><H3>TDPs Colony PCR check and transformation</H3><H4>Colony PCR gel and discussion</H4><H6>Isabell and Kelly</H6><P>The gel shows that colonies 1.3, 1.16, 1.18, 5.4, 5.16 and 5.19 are positive clones. Therefore, these colonies and colony 5.13 were prepared for miniprepping (following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol).</P><H4>Transformation of BL21(DE3)</H4><H6>Isabell and Kelly</H6><P>Chemically competent E.coli BL21(DE3) cells from New England Biolabs (C2527H) were transformed with the miniprepped plasmids from colony 1.5, 3.5, 4.6, 5.1 and 6.2, according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol. This was done, because the sequencing results (21-07-2017) looked promising. </P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170724"><H3>Vesicles verification TolR-AU</H3><H4>Miniprep TolR-AU</H4><H6>Aafke and Floor</H6><P>The liquid samples of colony 1 and colony 2 from transformation #3 from TolR-AU wer minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>
    The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU 3.1</TD><TD>58.16</TD><TD>1.83</TD></TR><TR><TD class="firstcolumn">TolR-AU 3.2</TD><TD>47.93</TD><TD>1.90</TD></TR></TBODY></TABLE><H4>Restriction assay TolR-AU</H4><H6>Aafke and Floor</H6><P>The digestion was performed with NdeI and SphI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 50 µL total.
    After an hour of incubation, both digested samples and undigested samples were checked on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.</P><P>The indicated bands verify the TolR-AU plasmid, so we decided to send two samples (TolR-AU 3.1 and 3.2) for sequencing.
    Both samples were prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, using the forward primer IG0028 .</P><H4>Transformation GFP-TorA into TOP10</H4><H6>Aafke and Floor</H6><P>The transformation was performed with 2µL of the ligation sample from 21/07 following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.
    The transformed cells were plated on IPTG and chloramphenicol.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 25 July</DIV><DIV class="collapsible-body" id="d20170725"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170725"><H3>Sample prep DNA gel electrophoresis of RPA with DNA from purified PSB1T3</H3><H6>Amária and Kasper</H6><P>
    We purified the amplified DNA on the 25<SUP>th</SUP> of July by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol. Then we performed the
    <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol. We prepaired a 1 % agarose gel. We ran it at 100V for 40 min.The following picture shows the resulting DNA gel:
      </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170725"><H3>TDPs Miniprep and sequencing</H3><H4>Miniprep</H4><H6>Isabell and Kelly</H6><P>The same samples (21-07-2017; except for two that did not look good) were send for sequencing again, this time with the reverse primer (IG0013; a <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Colony</TH><TH>Label</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1.5</TD><TD>18A1ZAG117</TD></TR><TR><TD class="firstcolumn">3.5</TD><TD>18A1ZAG118</TD></TR><TR><TD class="firstcolumn">4.6</TD><TD>18A1ZAG119</TD></TR><TR><TD class="firstcolumn">5.1</TD><TD>18A1ZAG120</TD></TR><TR><TD class="firstcolumn">6.1</TD><TD>18A1ZAG121</TD></TR><TR><TD class="firstcolumn">6.2</TD><TD>18A1ZAG122</TD></TR></TBODY></TABLE><H4>Sequencing</H4><H6>Isabell and Kelly</H6><P>Additionally, the 10 mL cultures of the new positive clones for genes 1 and 5 (1.3, 1.16, 1.18, 5.4, 5.13, 5.16 and 5.19) were miniprepped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A>) and NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>):</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Conc. (ng/µL)</TH></TR></THEAD><THEAD><TR><TH>Gene index</TH><TH>Colony</TH><TH>Run 1</TH><TH>Run 2</TH><TH>Run 3</TH><TH>Average</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1</TD><TD>1.3</TD><TD>134.72</TD><TD>151.03</TD><TD>152.65</TD><TD>146.13</TD></TR><TR><TD>1.16</TD><TD>152.27</TD><TD>148.45</TD><TD>150.26</TD><TD>150.33</TD></TR><TR><TD>1.18</TD><TD>219.61</TD><TD>217.09</TD><TD>228.70</TD><TD>221.80</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.4</TD><TD>77.66</TD><TD>78.65</TD><TD>75.96</TD><TD>77.40</TD></TR><TR><TD>5.13</TD><TD>151.00</TD><TD>158.26</TD><TD>153.41</TD><TD>154.22</TD></TR><TR><TD class="firstcolumn">5</TD><TD>5.16</TD><TD>86.80</TD><TD>89.41</TD><TD>88.94</TD><TD>88.38</TD></TR><TR><TD class="firstcolumn">6</TD><TD>5.19</TD><TD>122.74</TD><TD>126.29</TD><TD>127.69</TD><TD>125.57</TD></TR></TBODY></TABLE><H4>Colony counting</H4><H6>Isabell</H6><P>The colonies of the transformed BL21(DE3) (24-07-2017) were counted:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Colony index</TH><TH>75 µL plate</TH><TH>75 µL plate</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1.5</TD><TD>142</TD><TD>374</TD></TR><TR><TD class="firstcolumn">3.5</TD><TD>16</TD><TD>312</TD></TR><TR><TD class="firstcolumn">4.6</TD><TD>89</TD><TD>97</TD></TR><TR><TD class="firstcolumn">5.1</TD><TD>134</TD><TD>267</TD></TR><TR><TD class="firstcolumn">6.2</TD><TD>1</TD><TD>0</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170725"><H3>Transformation GFP-TorA</H3><H4>Transformation GFP-TorA in TOP10 and NEB5-α</H4><H6>Aafke and Floor</H6><P>No growth on the plates of the tranformation of GFP-TorA in TOP10 cells.</P><P>Two ligated samples of GFP-TorA were again transformed, but this time in commercial available NEB5-α competent cells following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol.
    However, the heat-shock was 30 seconds and after the heatshock the cells were stored on ice for 5 minutes before adding the enriched SOC-medium.</P><H4>Digestion GFP-TorA &amp; pSB1C3</H4><H6>Aafke and Floor</H6><P>In case the transformation didn't work, a new digestion ligation step is required before we can transform again. For this purpose we initiated a digestion of GFP-TorA and pSB1C3 with EcoRI and PstI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, with samples of 50 µL total.The sample was incubated at room temperature overnight.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 26 July</DIV><DIV class="collapsible-body" id="d20170726"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170726"><H3>TDPs Culture inoculation</H3><H6>Isabell and Kelly</H6><P>~20 mL cultures were inoculated with two BL21(DE3) colonies of 1.5 and two colonies of 5.1. They will be used for protein production and SDS-PAGE. The cells were incubated overnight at 37 °C in the shaking incubator (~150 rpm).</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170726"><H3>Vesicles Transformation GFP-TorA</H3><H4>Results transformation GFP-TorA in NEB5-α</H4><H6>Aafke and Floor</H6><P>Both transformations were succesful. The plates were checked for fluorescent activity. From each transformation 4 fluorescent colonies and 1 non-fluorescent colony were picked.</P><H4>Colony PCR GFP-TorA </H4><H6>Aafke and Floor</H6><P>The primers used were IG0006 and IG0013, with an annaeling temperature of 55 °C. The 10 picked colonies of GFP-TorA were checked by colony PCR, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol. 
    The first step was set to be 1 minute. The extension time was set to be 2 minutes. The result can be seen in the picture below.</P><P>The primers anneal 260 bp around the insert and the insert has a size of 1250 bp, making the expected band size around 1500 bp. All colonies except the control and the blank display this band. L1.2 and L1.3 were inoculated in 5 mL of LB to grow overnight. </P><H4>Sequencing results TolR-AU</H4><H6>Aafke and Floor</H6><P>The sequence was well aligned with the pET-DUET backbone and the TolR-sequence. TolR is inserted at bp 74 of pET-DUET, bp 74-168 are deleted from the backbone where TolR is inserted.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 27 July</DIV><DIV class="collapsible-body" id="d20170727"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170727"><H3>TDPs Large culture preparation and induction</H3><H6>Isabell and Kelly</H6><P>The OD<SUB>600</SUB> of the overnight cultures was determined to inoculate fresh 200 mL cultures at an OD<SUB>600</SUB> of 0.05. These flasks were incubated at 37 °C in the shaking incubator (~150 rpm). To two out of the four flasks IPTG was added after 100 minutes.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Culture</TH><TH>OD<SUB>600</SUB> at t=100 min.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1.5<SUB>IPTG</SUB></TD><TD>0.23<SUP>*</SUP></TD></TR><TR><TD class="firstcolumn">1.5</TD><TD>0.34</TD></TR><TR><TD class="firstcolumn">5.1<SUB>IPTG</SUB></TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">5.1</TD><TD>0.41</TD></TR></TBODY></TABLE><P><SUP>*</SUP>This induction was a mistake; induction preferably takes place when an OD<SUB>600</SUB> between 0.4 and 0.6 is reached.
                    4 hours after induction, the cells were pelleted using a centrifuge at maximum speed for 20 minutes. The supernatant was discarded and the pelleted cells were stored at -20 °C.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170727"><H3>Vesicles Verfification GFP-TorA and transformation TolR-AU</H3><H4>Sequence GFP-TorA</H4><H6>Aafke and Floor</H6><P>The liquid samples of L1.2 and L1.3 were minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA L1.2</TD><TD>214.05</TD><TD>2.04</TD></TR><TR><TD class="firstcolumn">GFP-TorA L1.3</TD><TD>83.30</TD><TD>1.95</TD></TR></TBODY></TABLE><P>Both samples were prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, using the forward primer IG0013 .</P><H4>Transformation TolR-AU in KEIO</H4><P>As the plasmid is verified now, we can transform the plasmid into the hypervesiculation strain (KEIO). First the KEIO-cells were made electrocompetent following <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol.
    Next, 50 µL of electrocompetent cells were transformed with 2 µL DNA according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.
    The cells were plated on both Ampicillin and Kanamycin and on Kanamycin alone as control. </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 28 July</DIV><DIV class="collapsible-body" id="d20170728"><DIV class="C"><DIV class="row notebook cas13" id="C20170728"><H3>New plan for assembling the cas13 construct</H3><P>The Amp plates containing transformed cell from 27-07-2017 has no colonies. Conclusion: the Gibson assembly and/or transformation was unsuccesful.</P><H4>Gel electrophoresis of Becca's PCR</H4><H6>Jeroen and Fiona</H6><P>The samples were loaded on an agarose gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol), which was run for 50 minutes.</P><P>BB stands for backbone (psB4A5) and P1 stands for part1.
            There are clear bands for part1, so we decided to proceed with the purifiction of the PCR product. The psB4A5 amplification was less succesful.</P><H4>Cleaning and concentration of part1</H4><H6>Fiona and Jeroen</H6><P>The cleaned PCR products of part1 from 27-07-2017 were added together and DpnI digested <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI digestion</A> in 70 µL in total (only 1 µL of milli-Q and 7 µL of CutSmart buffer were used). The resulting sample was again purified (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A>) in a total volume of 20 μL and NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>). The concentration was determined to be 358 ng/µL with an 260/280 ratio of 1.84. The tube is lablled as C26.</P><H3>Transformation of Gibson assembly</H3><H6>Jeroen and Fiona</H6><H4>New plan for assembling the cas13a construct: Plan A and Plan B</H4><H6>Jeroen and Fiona</H6><P>we designed a new approach for assembling our construct, since the Gibson assemblies and transformations kept failling. Both plans consist of a serie of Gibson assemblies that will create the construct step by step. The difference between the two aproaches lies in the order in which some parts are assembled. See below for a schematic overview of both approaches.</P><P>For these assemblies new primers had been ordered (IG0035-IG0050). The primers are designed in SnapGene by doing virtual Gibson Assemblies.
            We worked as follows:</P></DIV></DIV></DIV></LI><LI>Select vector
                </LI><LI>Actions</LI><LI>Gibson assembly</LI><LI>Insert fragment</LI><LI>Select parts of the vector you do not wish to include in the construct</LI><LI>Go to &quot;fragment&quot;</LI><LI>Select parts of the insert that you wish to include in the construct</LI><LI>Go to &quot;product&quot;</LI><LI>Click on &quot;create PCR overhangs&quot;</LI><LI>Select 55 °C as Tm and 35' overhangs</LI><LI>Click on &quot;assemble&quot;</LI></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170728"><H3>TDPs SDS-PAGE and BL21 AI transformation</H3><H4>Preparation for SDS-PAGE</H4><H6>Isabell and Kelly</H6><P>By adding 20 mL of 100 mM HEPES buffer, 19.6 mL milliQ and 0.4 mL of 5M NaCl, 40 mL of 50 mM HEPES, 50 mM NaCl buffer was prepared. The cell pellets stored at -20 °C the previous day (27-07-2017) were resuspended in the HEPES/NaCl buffer and then prepared for SDS-PAGE following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspageprecast" target="_blank">SDS-PAGE for precast gels</A> protocol. The samples were loaded in undiluted and diluted form (5x in sample buffer) and the marker used for the gel was <I>Precision Plus Protein™ Unstained</I> (#161-0363).</P><H4>Transformation of SAHS plasmids into BL21AI</H4><P>To have a tighter control on expression of the two SAHS genes, both SAHS plasmids were transformed into a BL21 AI strain. The BL21AI strain was plated on 27-07-2017 and the transformation followed the following protocol.</P><OL><LI>Pick three colonies of BL21 AI, inoculate in SOB medium and incubate in a shaking incubator at 37 °C for 3.5 hours.</LI><LI>Centrifuge cells at 15000 g for 10 min. Discard supernatant.</LI><LI>Resuspend cells in 1 mL of milliQ, wash with 25 mL milliQ and centrifuge at 15000 g for 10 min</LI><LI>Repeat washing procedure with milliQ.</LI><LI>Resuspend cells in 1 mL of glycerol and keep on ice.</LI><LI>Mix 1 mL plasmid solution and 100 µL cells in a sterile microcentrifuge tube and incubate on ice for 30 min.</LI><LI>Transfer mixture into an electroporation cuvette and pulse.</LI><LI>Immediately add SOC medium to the cuvettes and transfer to another sterile microcentrifuge tube.</LI><LI>Incubate cells in a shaking incubator at 37 °C for 60 min, and plate undiluted and in 10x and 100x dilutions.</LI></OL><P>The plates were left in the stove at 37 °C, taken out the following day (29-07-17) and moved to the fridge.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170728"><H3>Vesicles Results sequence GFP-TorA and transformation TolR-AU</H3><H4>Sequence GFP-TorA</H4><H6>Aafke and Floor</H6><P>Both the GFP-sequence and the TorA-sequence allign perfectly. However, some mismatches were found at the beginning of the sequence in the prefix and the Lac-promoter.</P><P>To be able to verify the sequence of the Lac-promoter and the prefix, L1.2 was prepared again for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol. 
    This time we were provided with the reverse primer BN156 that anneals on GFP.</P><H4>Transformation TolR-AU</H4><H6>Aafke and Floor</H6><P>The transformation was succesful, growth on all the plates as expected. </P></DIV></DIV></DIV></DIV><LI><DIV class="collapsible-header scrollspy" id="week31-scroll">Week 31</DIV></LI><LI><DIV class="collapsible-header">Monday 31 July</DIV><DIV class="collapsible-body" id="d20170731"><DIV class="C"><DIV class="row notebook cas13" id="C20170731"><H3>Colony picking and colony PCR of transformed colonies</H3><P>The plates with the transformed cells from 28-07-2017 looked as follows:</P><P>This is not the desired result, since ther are way more colonies on the control plate then on the insert plates. This could indicate that there is a lot of &quot;background noise&quot; consisting of empty self ligated backbones. We decided to still pick some colonies from the insert plates to check if there were some good colonies with colony PCR.</P><H4>Colony picking</H4><H6>Kasper and Jeroen</H6><P>We picked 20 colonies from the insert plates from 28-09-2017 where 20 μL, 100 μL and the rest of the transformed were plated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypickingcontrol" target="_blank">colony picking</A> protocol.</P><H4>Colony PCR</H4><H6>Jeroen</H6><P>A colony PCR was done on the 20 picked colonies following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> protocol. Primers IG0007 and IG0006 were used (IG0007 was resuspended and a working stock was made). We made postive control by addind 2 μL C1 (psB4A5) to the GoTaq mix. As annealing temperature (calculated with the <A href="http://tmcalculator.neb.com/#!/" target="_blank">NEB Tm calculator</A>) we set 52 °C with an extension time of 45 seconds. Step 2-4 was repeated 35 times.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170731"><H3>Culture Preparation and Sequencing Results and Conclusions</H3><H4>Preparation of Starter Cultures for Protein Purification</H4><H6>Guillermo, Isabell and Kelly</H6><P>50 mL cultures were inoculated with two BL21(DE3) colonies of 1.5 (further referred to as B1) and two BL21(AI) colonies of 5.1 (further referred to as A5). They will be used for protein production and SDS-PAGE. The cells were incubated overnight at 37 °C in the shaking incubator (~150 rpm).</P><H4>Preparation of cultures for miniprep</H4><H6>Guillermo, Isabell and Kelly</H6><P>10 mL cultures were inoculated with two BL21(DE3) colonies each of 1.5, 3.5 and 4.6 and two BL21(AI) colonies each of 5.1 and 6.2. They will be used for a miniprep. The cells were incubated overnight at 37 °C in the shaking incubator (~150 rpm).</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170731"><H3>Vesicles Transformations</H3><H6>Aafke and Floor</H6><P>For the coming experiments we needed to prepare different transformants. The following table shows the plasmid and the strain it is transformed in, including the corresponding antibiotics used for the plates (with WT standing for Wild Type of the KEIO-strain).</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU</TD><TD>WT</TD><TD>Ampicillin</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>Chloramphenicol</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>Chloramphenicol</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR></TBODY></TABLE><P>All transformations were performed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170801"><H3>Vesicles Results of transformations</H3><H6>Aafke and Floor</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU</TD><TD>WT</TD><TD>growth</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>Growth</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>No growth</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>Growth</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>No growth</TD></TR></TBODY></TABLE><P>GFP in KEIO and GFP-TorA in KEIO were plated again. </P></DIV><DIV><DIV class="row notebook vesicles" id="V20170802"><H3>Vesicles Preparing pSB1C3</H3><H6>Aafke and Floor</H6><P>pSB1C3 in TOP10 was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA L1.2</TD><TD>348.56</TD><TD>2.04</TD></TR></TBODY></TABLE><P>The minipreped sample was prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, using the forward primer IG0006.</P><DIV><DIV class="row notebook vesicles" id="V20170803"><H3>Vesicles DLS measurements</H3><H6>Aafke and Floor</H6><P>TolR-AU in KEIO and TolR-AU in WT were grown in 5 mL SOC-medium in triplo and induced when the cells reached an OD of ~0.5.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT (#1)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#2)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#3)</TD><TD>induced with 1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#4)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#5)</TD><TD>induced with 0.1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#6)</TD><TD>induced with 1 mM IPTG</TD></TR></TBODY></TABLE><P>DLS measurements were performed 45 minutes after induction with samples #3, #5 and #6.</P><P>To try out different protocols to isolate vesicles, after 2 hours 3 fractions of sample #3 were prepared as follows:
    1.without centrifugation
    2.centrifugation of 3 minutes at 2000g
    3.centrifugation of 20 minutes at 6000g followed by a 0.2 µm filter</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170804"><H3>Vesicles DLS measurements and transformations</H3><H4>DLS measurements</H4><H6>Aafke and Gabriella</H6><P>To try out different protocols to isolate vesicles, sample #5 and #6 were prepared as follows:
    1.no centrifugation
    2.no cenrifugation + addition of 25 mg liposomes
    2.centrifugation for 3 minutes at 1400<SMALL> x </SMALL><I>g</I>.
    3.centrifugation for 20 minutes at 3300<SMALL> x </SMALL><I>g</I>.</P><H4>Transformation GFP-TorA and GFP into KEIO</H4><H6>Aafke and Gabriella</H6><P>GFP and GFP-TorA were transformed into KEIO-cells following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170807"><H3>Vesicles Preparation Widefield samples</H3><H6>Aafke and Floor</H6><P>The transformatin for GFP-TorA in KEIO was succesful.</P><P>Overnight cultures were prepared for GFP-TorA in WT and in KEIO and for GFP in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170808"><H3>Vesicles Widefield microscopy</H3><H6>Aafke and Floor</H6><P>For the samples of GFP-TorA in WT and in KEIO a coverglass was prepared for Widefield microscopy following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#microscopecoverglass" target="_blank">Preparing coverglass for microscope</A> protocol.</P><P>The following picture was obtained for GFP-TorA in WT .</P><P>The following pictures were obtained for GFP-TorA in KEIO.</P><P>You can clearly see a difference in the morphology of KEIO compared to the WT. The KEIO-cells are longer and might even appaer filamentous. Above that, growth and protein production seems to be impaired, leading to less GFP-activity and less but longer cells.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170809"><H3>Vesicles Digestion and ligation of GFP-TorA and pSB4A5</H3><H6>Aafke</H6><P>To be able to control the expression of GFP, GFP-TorA will be cloned onto a low copy number plasmid.pSB4A5 was digested with EcoRI and PstI following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, in 50 µL total.GFP-TorA was already digested on 25/07. </P><P>Both samples were cleaned-up according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>18.00</TD><TD>2.03</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pSB4A5</TD><TD>9.40</TD><TD>1.83</TD></TR></TBODY></TABLE><P>Next, the digested DNA-fragments were ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation (sticky-end)</A> protocol and incubated at roomtemperature.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>1.5</TD></TR><TR><TD class="firstcolumn">Backbone pSB4A5</TD><TD>10</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>5.5</TD></TR><TR><TD class="firstcolumn">Buffer</TD><TD>2</TD></TR><TR><TD class="firstcolumn">Ligase</TD><TD>1</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170811"><H3>Vesicles Widefield microscopy</H3><H6>Gabriella</H6><P>As part of the Widefield microscope training, new pictures were taken of GFP-TorA in KEIO with the Widefield microscope. The pictures are shown below.</P><P>Again, the expression of GFP is not that high. Also, the morphology of the cells differs from the WT as seen before.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170814"><H3>Vesicles Preparation pET-Duet1</H3><H6>Floor and Gabriella</H6><P>To obtain the empty backbone pET-Duet1 without the TolR-fragment, the backbone will be amplified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol, with 1 µL of DNA template (TolR-AU) and 35.5 µL of milli-Q; Only 1 µL of each of the primers (IG0051 and IG0052) was used. The PCR program was run with an extension time of 81 seconds for TolR-AU.</P><P> The PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below (with 1 and 2 being the empty backbone and B the blank). </P><P>The gel was run again and the indicated band was cut out of the gel.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170815"><H3>Vesicles Transformations</H3><H4>Ligation pET-Duet1</H4><H6>Aafke and Floor</H6><P>The band containing pET-Duet1 was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" target="_blank">gel product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the purified sample was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>21.07</TD><TD>1.92</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #2</TD><TD>25.5</TD><TD>2.02</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #3</TD><TD>30.5</TD><TD>1.91</TD></TR></TBODY></TABLE><P>The obtained linearized backbone was ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated for 1 hour at room temperature.</P><P><I>!We forgot to phosphorylate first!</I></P><H4>Transformation</H4><H6>Aafke and Floor</H6><P>Cells from WT and KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    The following transformations were performed following the  <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>WT</TD><TD>Ampicillin</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>WT</TD><TD>Chloramphenicol</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>WT</TD><TD>Ampicillin</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170816"><H3>Vesicles Ligation pET-Duet1</H3><H6>Aafke and Gabriella</H6><P>The blunt-end ligation of linearized pET-Duet1 was performed again following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated overnight at 4 °C;.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170817"><H3>Vesicles Transformation &amp; preparation 96-wells experiment</H3><H4>Transformation</H4><H6>Aafke and Gabriella</H6><P>Transformation of pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol.
    </P><H4>preparation 96-wells experiment</H4><H6>Aafke and Gabriella</H6><P>Starter cultures were prepared following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P></DIV><DIV><DIV class="row notebook vesicles" id="V20170818"><H3>Vesicles 96-wells experiment GFP</H3><H6>Aafke and Floor</H6><P>First, 230 µL LB was pipetted into each well and inoculated with 10 µL, 18 wells per culture for the following culture:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P><P>An hour after inoculation, the cells are induced in triplo with the following final IPTG-concentrations:
    -0 mM
    -0.1 mM
    -0.3 mM
    -0.5 mM
    -0.7 mM
    -1 mM</P><P>After induction, the growth and fluorescence was monitored with a plate-reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170821"><H3>Overnight cultures</H3><H6>Gabriella</H6><P>Overnight cultures were prepared for pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170822"><H3>Vesicles Miniprep GFP-TorA on pSB4A5</H3><H6>Gabriella</H6><P>The sample with GFP-TorA on pSB4A5 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P>
    The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. <P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA #1</TD><TD>258</TD></TR><TR><TD class="firstcolumn">GFP-TorA #2</TD><TD>251</TD></TR></TBODY></TABLE></P></DIV><DIV><DIV class="row notebook vesicles" id="V20170823"><H3>Vesicles Transformation pET-Duet1</H3><H4>Miniprep pET-Duet1</H4><H6>Aafke</H6><P>pEt-Duet1 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>46.71</TD><TD>1.88</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>62.27</TD><TD>1.94</TD></TR></TBODY></TABLE><H4>Transformation pET-Duet1 into KEIO &amp; WT</H4><H6>Aafke</H6><P>4 µL of pET-Duet1 #2 was transformed into 50 µL KEIO and WT following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170824"><H3>Vesicles Transformations and preparation osmoshock</H3><H4>Transformations</H4><H6>Aafke</H6><P>Electrocompetent cells of TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    2 µL of GFP-TorA and GFP were transformed into TolR-AU in KEIO.
    4 µL of pET-Duet1 #2 was transformed again into 50 µL KEIO and WT. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>100 mL of osmoshock-solution was prepared following the description in <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    The following cells were grown in 20 mL LB medium and induced with 0.1 mM IPTG at an OD between 0.5 and 0.7 and grown overnight.
    
                                        </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>0.59</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>0.62</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>0.63</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170825"><H3>Vesicles Osmoshock and DLS-measurements</H3><H4>Osmoshock</H4><H6>Aafke</H6><P>The periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm.</P><H4>DLS-measurements</H4><H6>Floor and Gabriella</H6><P>TolR-AU in WT and pET-Duet1 in WT were inocculated in 10 mL LB and the growth was monitored; pET-Duet1 in WT did not grow. TolR-AU in WT #1 was induced at an OD of 0.45 and TolR-AU in WT #2 at an OD of 0.41. 
    After 2 hours, the samples of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. </P><P>pET-Duet1 was transformed again into KEIO and WT.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170828"><H3>Vesicles Transformations</H3><H6>Aafke and Floor</H6><P>pET-Duet1 was transformed again into KEIO and WT. pSB1C3 was transformed into TolR-AU in KEIO. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170829"><H3>Vesicles DLS-measurements</H3><H6>Aafke and Floor</H6><P>10 mL samples of TolR-AU in WT and pET-Duet1 in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the purification only after 2 hours. <I>The cells were grown on LB without antibiotics.</I> The samples were analysed with the DLS. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT #1</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT #2</TD><TD>0.46</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #1</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #2</TD><TD>0.55</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170830"><H3>Vesicles Transformations and DLS-measurements</H3><H4>Transformation pET-Duet1 in KEIO</H4><H6>Aafke and Floor</H6><P>pEt-Duet1 in WT was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>43.57</TD><TD>1.85</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>60.81</TD><TD>2.22</TD></TR></TBODY></TABLE><P>5 µL of pET-Duet1 #2 was transformed into 150 µL KEIO following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>DLS-measurements</H4><H6>Aafke and Floor</H6><P>10 mL of TolR-AU in KEIO were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.45 for both samples and the purification only after 2 hours. </P>
    10 mL of non-induced pET-Duet1 in WT was used as a positive control, with 1.25 µM liposomes added before purification.<I>The cells were grown in LB without antibiotics.</I> The samples were analysed with the DLS.</DIV><DIV><DIV class="row notebook vesicles" id="V20170831"><H3>Vesicles Preparation osmoshock and DLS-measurements</H3><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>KEIO and WT were inoculated in 20 mL LB medium in duplo and grown overnight. Above that, the following cells were inoculated in 20 mL LB medium, induced with 0.1 mM IPTG and grown overnight.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in KEIO #1</TD><TD>0.45</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO #2</TD><TD>0.33</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #1</TD><TD>0.67</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #2</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #2</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP in WT #2</TD><TD>0.49</TD></TR></TBODY></TABLE><H4>DLS measurements pET-Duet1 in KEIO</H4><H6>Floor</H6><P>10 mL of pET-Duet1 in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.31 and 0.34 and the measurement only after 2 hours. <I>The cells were grown in LB without antibiotics.</I>
    Next, 200 µL of the purified samples was analysed with the DLS.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170901"><H4>Osmoshock</H4><H6>Floor</H6><P>Both the cytoplasmic and the periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    <I>In step 6 and 9, 4 mL milli-Q was used instead of 2.5 mL.</I> With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm. Prior to the osmoshock, the pellets were weighted and used for normalization of the data:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>Weight (g) No Induction</TH><TH>Weight (g) Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA WT 1</TD><TD>0.0831</TD><TD>0.0901</TD></TR><TR><TD class="firstcolumn">GFP-TorA WT 2</TD><TD>0.0474</TD><TD>0.0662</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 1</TD><TD>0.0638</TD><TD>0.0511</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 2</TD><TD>0.0610</TD><TD>0.0568</TD></TR><TR><TD class="firstcolumn">GFP WT 1</TD><TD>0.0757</TD><TD>0.0658</TD></TR><TR><TD class="firstcolumn">GFP WT 2</TD><TD>0.0658</TD><TD>0.0839</TD></TR><TR><TD class="firstcolumn">GFP KEIO 1</TD><TD>0.1094</TD><TD>0.0678</TD></TR><TR><TD class="firstcolumn">GFP KEIO 2</TD><TD>0.0533</TD><TD>0.0655</TD></TR><TR><TD class="firstcolumn">KEIO</TD><TD>0.0649</TD><TD>-</TD></TR><TR><TD class="firstcolumn">WT</TD><TD>0.0575</TD><TD>-</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170905"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT and of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    TolR-AU in WT was induced at an OD of 0.57 and 0.58. pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170906"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT, pET-Duet1 in KEIO and TolR-AU in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in KEIO was induced at an OD of 0.35 and 0.45. TolR-AU in KEIO was induced at an OD of 0.45 and 0.40. Again, pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight. The overnight cultures of TolR-AU in WT were purified as well
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170907"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.47 and 0.49. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction. The overnight cultures of TolR-AU in KEIO and pET-Duet1 in KEIO were purified as well.
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P><I>It turned out that pET-Duet1 in KEIO was inocculated instead of pET-Duet1 in WT.</I></DIV><DIV><DIV class="row notebook vesicles" id="V20170908"><H3>Vesicles TEM</H3><H6>Floor</H6><P>The stored samples of TolR-AU in WT and TolR-AU in KEIO 5&amp;ndsp;hours after induction were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    However, the samples contained a lot of noise, making analysis impossible.</P><I>This might be because 0.2 µm filter was not used.</I></DIV><DIV><DIV class="row notebook vesicles" id="V20170912"><H3>Vesicles TEM</H3><H6>Gabriella</H6><P>5 mL of pET-Duet1 in KEIO and TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in WT was induced at an OD of 0.65 and TolR-AU in KEIO at an OD of 0.55. Next, the samples were prepared for TEM analysis following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    Pictures can be seen below.</P><P>TolR-AU in KEIO
    </P><P>pET-Duet in KEIO
    </P><P>You can see that vesicles are produced, regardless of whether or not the strain contains the TolR-insert.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170913"><H3>Vesicles Platereader GFP in vesicles</H3><H6>Gabriella</H6><P>5 mL of GFP-TorA &amp; TolR-AU in KEIO and GFP-TorA in KEIO were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    250 µL of the prepared samples were measured in the platereader at an exc./em. of 475/510 nm.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170914"><H3>Preparation DLS-Measurements</H3><H6>Floor</H6><P>Starter cultures were prepared for pET-Duet1 in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170915"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.42 and 0.39. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170916"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>The overnight cultures of pET-Duet1 in WT were purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    200 µL of the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170919"><H3>Vesicles Transformations</H3><H6>Aafke</H6><P>TolR-AU in KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    pSB1C3, TDP #4 and TDP#6 were transformed into the cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170920"><H3>Vesicles Membrane staining</H3><H4>Transformations</H4><H6>Aafke and Gabriella</H6><P>TDP #4 was transformed into WT, GFP-TorA into TolR-AU in KEIO and GFP-TorA into KEIO.</P><H4>Membrane staining</H4><H6>Aafke and Gabriella</H6><P>The remainder of the samples that were used for the DLS-measurements, were prepared as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="_blank">Membrane staining</A> protocol. 
    The samples prepared 3 hours after induction and the overnight samples were used. Different dilutions of stained liposomes were measured first, to be able to create a calibration curve. Next, the stained sample were measured in the plate reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170921"><H3>Vesicles Transformations</H3><H6>Aafke and Gabriella</H6><P><I>Previously used GFP was not controlled by a promoter.</I> We decided to use GFP E0840 with constitutive promoter J23108 with moderate expression on pSB1C3, as the new control for the untagged GFP.
    GFP was transformed into TolR-AU in KEIO and into WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV><DIV><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 1 August</DIV><DIV class="collapsible-body" id="d20170801"><DIV class="C"><DIV class="row notebook cas13" id="C20170801"><H4>Transformation with Zhang plasmids</H4><H6>Kasper</H6><P>Plasmids received from Feng Zhang lab were <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformed</A> into BL21 competent cells (NEB BL21(DE3), #C2527M). The cells were plated on selective ampicillin plates.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170801"><H3>Sequencing Preparation, Miniprep and Protein Purification (Day 0)</H3><H4>Sequencing Preparation</H4><H6>Guillermo and Isabell</H6><P>These samples were prepared to send for sequencing (<A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol) with primer IG0013.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Colony</TH><TH>Label</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">4.6</TD><TD>18A1ZAG108</TD></TR><TR><TD class="firstcolumn">6.1</TD><TD>18A1ZAG109</TD></TR><TR><TD class="firstcolumn">6.2</TD><TD>18A1ZAG110</TD></TR></TBODY></TABLE><H4>Miniprep of liquid cultures</H4><H6>Guillermo and Isabell</H6><P>The liquid cultures inoculated the previous day (31-07-2017) of the colonies 1.5, 3.5 and 4.6 in BL21(DE3) and 5.1 and 6.2 in BL21(AI) were miniprepped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A>).</P><H4>Protein Purification (Day 0)</H4><H6>Guillermo and Isabell</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A> was carried out with the two overnight cultures prepared the previous day (31-07-2017), B1 and A5. Two 1 L flasks were prepared per protein and the protein production only induced in one of the two flasks, respectively (B1<SUB>IPTG</SUB>, B1<SUB>x</SUB>, A5<SUB>IPTG</SUB> (1), A5<SUB>x</SUB>).</P><P>N.B.: The BL21(AI) strain is IPTG-induced as well as arabinose-induced to activate T7 polymerase. In A5<SUB>IPTG</SUB> 10 mL of arabinose were only added 2 hours after induction with IPTG. For this reason, protein production of A5 was repeated on the next day (02-08-17)</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170801"><H3>Vesicles Results of transformations</H3><H6>Aafke and Floor</H6><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU</TD><TD>WT</TD><TD>growth</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>Growth</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>No growth</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>Growth</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>No growth</TD></TR></TBODY></TABLE><P>GFP in KEIO and GFP-TorA in KEIO were plated again. </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 2 August</DIV><DIV class="collapsible-body" id="d20170802"><DIV class="C"><DIV class="row notebook cas13" id="C20170802"><H3>Transformation results + Miniprep</H3><H4>Miniprep and sequencing</H4><H6>Fiona</H6><P>We were curious what could possibly have been transformed into the cells on 31-07-2017, so we decided to purify plasmids from two picked colonies (31-07-2017) and send these for sequencing. The <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">plasmid isolation (PureYield™ Plasmid Miniprep System)</A> protocol was followed. The plasmid isolation was done on two an overnight culture that was made the day before (5 mL LB medium with 5 μL Amp) from the colonies that were picked on 31-07-2017 (sample 9 and 10). The purified samples were NanoDropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>) and labelled.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>Labelled as</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">9</TD><TD>C29</TD><TD>~300</TD></TR><TR><TD class="firstcolumn">10</TD><TD>C30</TD><TD>~180</TD></TR></TBODY></TABLE><P>The isolated plasmids were prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing protocol from Macrogen</A>. The samples were sent with primer IG0006.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170802"><H3>Protein Purification (Day 0)</H3><H6>Guillermo and Isabell</H6><P> The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A> with A5 as well as a control of BL21 AI without TDP plasmid (A5<SUB>IPTG</SUB> (2), AI) was repeated with the exception of adding 10 mL of arabinose in addition to IPTG upon induction. The liquid cultures from the 31-07-17 were used for inoculation of the 1 L culture.</P><P>N.B.: The A5 1 L cultures was centrifuged in two parts of ~500 mL.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170802"><H3>Vesicles Preparing pSB1C3</H3><H6>Aafke and Floor</H6><P>pSB1C3 in TOP10 was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA L1.2</TD><TD>348.56</TD><TD>2.04</TD></TR></TBODY></TABLE><P>The minipreped sample was prepared for sequencing following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing sample (Macrogen)</A> protocol, using the forward primer IG0006.</P><DIV><DIV class="row notebook vesicles" id="V20170803"><H3>Vesicles DLS measurements</H3><H6>Aafke and Floor</H6><P>TolR-AU in KEIO and TolR-AU in WT were grown in 5 mL SOC-medium in triplo and induced when the cells reached an OD of ~0.5.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT (#1)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#2)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#3)</TD><TD>induced with 1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#4)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#5)</TD><TD>induced with 0.1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#6)</TD><TD>induced with 1 mM IPTG</TD></TR></TBODY></TABLE><P>DLS measurements were performed 45 minutes after induction with samples #3, #5 and #6.</P><P>To try out different protocols to isolate vesicles, after 2 hours 3 fractions of sample #3 were prepared as follows:
    1.without centrifugation
    2.centrifugation of 3 minutes at 2000g
    3.centrifugation of 20 minutes at 6000g followed by a 0.2 µm filter</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170804"><H3>Vesicles DLS measurements and transformations</H3><H4>DLS measurements</H4><H6>Aafke and Gabriella</H6><P>To try out different protocols to isolate vesicles, sample #5 and #6 were prepared as follows:
    1.no centrifugation
    2.no cenrifugation + addition of 25 mg liposomes
    2.centrifugation for 3 minutes at 1400<SMALL> x </SMALL><I>g</I>.
    3.centrifugation for 20 minutes at 3300<SMALL> x </SMALL><I>g</I>.</P><H4>Transformation GFP-TorA and GFP into KEIO</H4><H6>Aafke and Gabriella</H6><P>GFP and GFP-TorA were transformed into KEIO-cells following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170807"><H3>Vesicles Preparation Widefield samples</H3><H6>Aafke and Floor</H6><P>The transformatin for GFP-TorA in KEIO was succesful.</P><P>Overnight cultures were prepared for GFP-TorA in WT and in KEIO and for GFP in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170808"><H3>Vesicles Widefield microscopy</H3><H6>Aafke and Floor</H6><P>For the samples of GFP-TorA in WT and in KEIO a coverglass was prepared for Widefield microscopy following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#microscopecoverglass" target="_blank">Preparing coverglass for microscope</A> protocol.</P><P>The following picture was obtained for GFP-TorA in WT .</P><P>The following pictures were obtained for GFP-TorA in KEIO.</P><P>You can clearly see a difference in the morphology of KEIO compared to the WT. The KEIO-cells are longer and might even appaer filamentous. Above that, growth and protein production seems to be impaired, leading to less GFP-activity and less but longer cells.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170809"><H3>Vesicles Digestion and ligation of GFP-TorA and pSB4A5</H3><H6>Aafke</H6><P>To be able to control the expression of GFP, GFP-TorA will be cloned onto a low copy number plasmid.pSB4A5 was digested with EcoRI and PstI following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, in 50 µL total.GFP-TorA was already digested on 25/07. </P><P>Both samples were cleaned-up according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>18.00</TD><TD>2.03</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pSB4A5</TD><TD>9.40</TD><TD>1.83</TD></TR></TBODY></TABLE><P>Next, the digested DNA-fragments were ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation (sticky-end)</A> protocol and incubated at roomtemperature.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>1.5</TD></TR><TR><TD class="firstcolumn">Backbone pSB4A5</TD><TD>10</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>5.5</TD></TR><TR><TD class="firstcolumn">Buffer</TD><TD>2</TD></TR><TR><TD class="firstcolumn">Ligase</TD><TD>1</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170811"><H3>Vesicles Widefield microscopy</H3><H6>Gabriella</H6><P>As part of the Widefield microscope training, new pictures were taken of GFP-TorA in KEIO with the Widefield microscope. The pictures are shown below.</P><P>Again, the expression of GFP is not that high. Also, the morphology of the cells differs from the WT as seen before.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170814"><H3>Vesicles Preparation pET-Duet1</H3><H6>Floor and Gabriella</H6><P>To obtain the empty backbone pET-Duet1 without the TolR-fragment, the backbone will be amplified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol, with 1 µL of DNA template (TolR-AU) and 35.5 µL of milli-Q; Only 1 µL of each of the primers (IG0051 and IG0052) was used. The PCR program was run with an extension time of 81 seconds for TolR-AU.</P><P> The PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below (with 1 and 2 being the empty backbone and B the blank). </P><P>The gel was run again and the indicated band was cut out of the gel.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170815"><H3>Vesicles Transformations</H3><H4>Ligation pET-Duet1</H4><H6>Aafke and Floor</H6><P>The band containing pET-Duet1 was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" target="_blank">gel product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the purified sample was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>21.07</TD><TD>1.92</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #2</TD><TD>25.5</TD><TD>2.02</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #3</TD><TD>30.5</TD><TD>1.91</TD></TR></TBODY></TABLE><P>The obtained linearized backbone was ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated for 1 hour at room temperature.</P><P><I>!We forgot to phosphorylate first!</I></P><H4>Transformation</H4><H6>Aafke and Floor</H6><P>Cells from WT and KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    The following transformations were performed following the  <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>WT</TD><TD>Ampicillin</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>WT</TD><TD>Chloramphenicol</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>WT</TD><TD>Ampicillin</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170816"><H3>Vesicles Ligation pET-Duet1</H3><H6>Aafke and Gabriella</H6><P>The blunt-end ligation of linearized pET-Duet1 was performed again following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated overnight at 4 °C;.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170817"><H3>Vesicles Transformation &amp; preparation 96-wells experiment</H3><H4>Transformation</H4><H6>Aafke and Gabriella</H6><P>Transformation of pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol.
    </P><H4>preparation 96-wells experiment</H4><H6>Aafke and Gabriella</H6><P>Starter cultures were prepared following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P></DIV><DIV><DIV class="row notebook vesicles" id="V20170818"><H3>Vesicles 96-wells experiment GFP</H3><H6>Aafke and Floor</H6><P>First, 230 µL LB was pipetted into each well and inoculated with 10 µL, 18 wells per culture for the following culture:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P><P>An hour after inoculation, the cells are induced in triplo with the following final IPTG-concentrations:
    -0 mM
    -0.1 mM
    -0.3 mM
    -0.5 mM
    -0.7 mM
    -1 mM</P><P>After induction, the growth and fluorescence was monitored with a plate-reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170821"><H3>Overnight cultures</H3><H6>Gabriella</H6><P>Overnight cultures were prepared for pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170822"><H3>Vesicles Miniprep GFP-TorA on pSB4A5</H3><H6>Gabriella</H6><P>The sample with GFP-TorA on pSB4A5 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P>
    The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. <P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA #1</TD><TD>258</TD></TR><TR><TD class="firstcolumn">GFP-TorA #2</TD><TD>251</TD></TR></TBODY></TABLE></P></DIV><DIV><DIV class="row notebook vesicles" id="V20170823"><H3>Vesicles Transformation pET-Duet1</H3><H4>Miniprep pET-Duet1</H4><H6>Aafke</H6><P>pEt-Duet1 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>46.71</TD><TD>1.88</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>62.27</TD><TD>1.94</TD></TR></TBODY></TABLE><H4>Transformation pET-Duet1 into KEIO &amp; WT</H4><H6>Aafke</H6><P>4 µL of pET-Duet1 #2 was transformed into 50 µL KEIO and WT following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170824"><H3>Vesicles Transformations and preparation osmoshock</H3><H4>Transformations</H4><H6>Aafke</H6><P>Electrocompetent cells of TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    2 µL of GFP-TorA and GFP were transformed into TolR-AU in KEIO.
    4 µL of pET-Duet1 #2 was transformed again into 50 µL KEIO and WT. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>100 mL of osmoshock-solution was prepared following the description in <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    The following cells were grown in 20 mL LB medium and induced with 0.1 mM IPTG at an OD between 0.5 and 0.7 and grown overnight.
    
                                        </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>0.59</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>0.62</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>0.63</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170825"><H3>Vesicles Osmoshock and DLS-measurements</H3><H4>Osmoshock</H4><H6>Aafke</H6><P>The periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm.</P><H4>DLS-measurements</H4><H6>Floor and Gabriella</H6><P>TolR-AU in WT and pET-Duet1 in WT were inocculated in 10 mL LB and the growth was monitored; pET-Duet1 in WT did not grow. TolR-AU in WT #1 was induced at an OD of 0.45 and TolR-AU in WT #2 at an OD of 0.41. 
    After 2 hours, the samples of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. </P><P>pET-Duet1 was transformed again into KEIO and WT.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170828"><H3>Vesicles Transformations</H3><H6>Aafke and Floor</H6><P>pET-Duet1 was transformed again into KEIO and WT. pSB1C3 was transformed into TolR-AU in KEIO. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170829"><H3>Vesicles DLS-measurements</H3><H6>Aafke and Floor</H6><P>10 mL samples of TolR-AU in WT and pET-Duet1 in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the purification only after 2 hours. <I>The cells were grown on LB without antibiotics.</I> The samples were analysed with the DLS. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT #1</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT #2</TD><TD>0.46</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #1</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #2</TD><TD>0.55</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170830"><H3>Vesicles Transformations and DLS-measurements</H3><H4>Transformation pET-Duet1 in KEIO</H4><H6>Aafke and Floor</H6><P>pEt-Duet1 in WT was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>43.57</TD><TD>1.85</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>60.81</TD><TD>2.22</TD></TR></TBODY></TABLE><P>5 µL of pET-Duet1 #2 was transformed into 150 µL KEIO following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>DLS-measurements</H4><H6>Aafke and Floor</H6><P>10 mL of TolR-AU in KEIO were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.45 for both samples and the purification only after 2 hours. </P>
    10 mL of non-induced pET-Duet1 in WT was used as a positive control, with 1.25 µM liposomes added before purification.<I>The cells were grown in LB without antibiotics.</I> The samples were analysed with the DLS.</DIV><DIV><DIV class="row notebook vesicles" id="V20170831"><H3>Vesicles Preparation osmoshock and DLS-measurements</H3><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>KEIO and WT were inoculated in 20 mL LB medium in duplo and grown overnight. Above that, the following cells were inoculated in 20 mL LB medium, induced with 0.1 mM IPTG and grown overnight.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in KEIO #1</TD><TD>0.45</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO #2</TD><TD>0.33</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #1</TD><TD>0.67</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #2</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #2</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP in WT #2</TD><TD>0.49</TD></TR></TBODY></TABLE><H4>DLS measurements pET-Duet1 in KEIO</H4><H6>Floor</H6><P>10 mL of pET-Duet1 in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.31 and 0.34 and the measurement only after 2 hours. <I>The cells were grown in LB without antibiotics.</I>
    Next, 200 µL of the purified samples was analysed with the DLS.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170901"><H4>Osmoshock</H4><H6>Floor</H6><P>Both the cytoplasmic and the periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    <I>In step 6 and 9, 4 mL milli-Q was used instead of 2.5 mL.</I> With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm. Prior to the osmoshock, the pellets were weighted and used for normalization of the data:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>Weight (g) No Induction</TH><TH>Weight (g) Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA WT 1</TD><TD>0.0831</TD><TD>0.0901</TD></TR><TR><TD class="firstcolumn">GFP-TorA WT 2</TD><TD>0.0474</TD><TD>0.0662</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 1</TD><TD>0.0638</TD><TD>0.0511</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 2</TD><TD>0.0610</TD><TD>0.0568</TD></TR><TR><TD class="firstcolumn">GFP WT 1</TD><TD>0.0757</TD><TD>0.0658</TD></TR><TR><TD class="firstcolumn">GFP WT 2</TD><TD>0.0658</TD><TD>0.0839</TD></TR><TR><TD class="firstcolumn">GFP KEIO 1</TD><TD>0.1094</TD><TD>0.0678</TD></TR><TR><TD class="firstcolumn">GFP KEIO 2</TD><TD>0.0533</TD><TD>0.0655</TD></TR><TR><TD class="firstcolumn">KEIO</TD><TD>0.0649</TD><TD>-</TD></TR><TR><TD class="firstcolumn">WT</TD><TD>0.0575</TD><TD>-</TD></TR></TBODY></TABLE></DIV><DIV><DIV class="row notebook vesicles" id="V20170905"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT and of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    TolR-AU in WT was induced at an OD of 0.57 and 0.58. pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170906"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT, pET-Duet1 in KEIO and TolR-AU in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in KEIO was induced at an OD of 0.35 and 0.45. TolR-AU in KEIO was induced at an OD of 0.45 and 0.40. Again, pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight. The overnight cultures of TolR-AU in WT were purified as well
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170907"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.47 and 0.49. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction. The overnight cultures of TolR-AU in KEIO and pET-Duet1 in KEIO were purified as well.
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P><I>It turned out that pET-Duet1 in KEIO was inocculated instead of pET-Duet1 in WT.</I></DIV><DIV><DIV class="row notebook vesicles" id="V20170908"><H3>Vesicles TEM</H3><H6>Floor</H6><P>The stored samples of TolR-AU in WT and TolR-AU in KEIO 5&amp;ndsp;hours after induction were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    However, the samples contained a lot of noise, making analysis impossible.</P><I>This might be because 0.2 µm filter was not used.</I></DIV><DIV><DIV class="row notebook vesicles" id="V20170912"><H3>Vesicles TEM</H3><H6>Gabriella</H6><P>5 mL of pET-Duet1 in KEIO and TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in WT was induced at an OD of 0.65 and TolR-AU in KEIO at an OD of 0.55. Next, the samples were prepared for TEM analysis following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    Pictures can be seen below.</P><P>TolR-AU in KEIO
    </P><P>pET-Duet in KEIO
    </P><P>You can see that vesicles are produced, regardless of whether or not the strain contains the TolR-insert.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170913"><H3>Vesicles Platereader GFP in vesicles</H3><H6>Gabriella</H6><P>5 mL of GFP-TorA &amp; TolR-AU in KEIO and GFP-TorA in KEIO were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    250 µL of the prepared samples were measured in the platereader at an exc./em. of 475/510 nm.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170914"><H3>Preparation DLS-Measurements</H3><H6>Floor</H6><P>Starter cultures were prepared for pET-Duet1 in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170915"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.42 and 0.39. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170916"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>The overnight cultures of pET-Duet1 in WT were purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    200 µL of the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170919"><H3>Vesicles Transformations</H3><H6>Aafke</H6><P>TolR-AU in KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    pSB1C3, TDP #4 and TDP#6 were transformed into the cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A>protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170920"><H3>Vesicles Membrane staining</H3><H4>Transformations</H4><H6>Aafke and Gabriella</H6><P>TDP #4 was transformed into WT, GFP-TorA into TolR-AU in KEIO and GFP-TorA into KEIO.</P><H4>Membrane staining</H4><H6>Aafke and Gabriella</H6><P>The remainder of the samples that were used for the DLS-measurements, were prepared as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="_blank">Membrane staining</A> protocol. 
    The samples prepared 3 hours after induction and the overnight samples were used. Different dilutions of stained liposomes were measured first, to be able to create a calibration curve. Next, the stained sample were measured in the plate reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170921"><H3>Vesicles Transformations</H3><H6>Aafke and Gabriella</H6><P><I>Previously used GFP was not controlled by a promoter.</I> We decided to use GFP E0840 with constitutive promoter J23108 with moderate expression on pSB1C3, as the new control for the untagged GFP.
    GFP was transformed into TolR-AU in KEIO and into WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV><DIV><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 3 August</DIV><DIV class="collapsible-body" id="d20170803"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170803"><H3>Protein Purification (Day 1)</H3><H6>Guillermo and Isabell</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out using the cell pellets of the induced and uninduced cultures from the 01-08-2017 and 02-08-2017 (B1<SUB>IPTG</SUB>, B1<SUB>x</SUB>, A5<SUB>IPTG</SUB> (1), A5<SUB>IPTG</SUB> (2), A5<SUB>x</SUB>, AI).</P><P>N.B.: Transfer of the lysates to dialysis tubing (<A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> step 7 and 8) was delayed and carried out on 04-08-2017.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170803"><H3>Vesicles DLS measurements</H3><H6>Aafke and Floor</H6><P>TolR-AU in KEIO and TolR-AU in WT were grown in 5 mL SOC-medium in triplo and induced when the cells reached an OD of ~0.5.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT (#1)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#2)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT (#3)</TD><TD>induced with 1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#4)</TD><TD>not induced, no growth detected</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#5)</TD><TD>induced with 0.1 mM IPTG</TD></TR><TR><TD class="firstcolumn">TolR-AU in KEIO (#6)</TD><TD>induced with 1 mM IPTG</TD></TR></TBODY></TABLE><P>DLS measurements were performed 45 minutes after induction with samples #3, #5 and #6.</P><P>To try out different protocols to isolate vesicles, after 2 hours 3 fractions of sample #3 were prepared as follows:
    1.without centrifugation
    2.centrifugation of 3 minutes at 2000g
    3.centrifugation of 20 minutes at 6000g followed by a 0.2 µm filter</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 4 August</DIV><DIV class="collapsible-body" id="d20170804"><DIV class="C"><DIV class="row notebook cas13" id="C20170804"><H4>Medium preparation</H4><H6>Kasper</H6><P>Today, 5 L of terrific broth (TB) medium was made. The composition of this medium can be found in the full<A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170804"><H3>Protein Purification (Day 1) - continued</H3><H6>Guillermo and Isabell</H6><P>Step 7 and 8 of the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> were executed with the samples B1<SUB>IPTG</SUB>, B1<SUB>x</SUB>, A5<SUB>IPTG</SUB> (1), A5<SUB>IPTG</SUB> (2), A5<SUB>x</SUB> and AI.</P><P>N.B.: It was later discovered, that we only dialysed against monobasic sodium phosphate (50 mM) instead of a mixture of mono- and dibasic phosphate (to reach pH = 7.4).</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170804"><H3>Vesicles DLS measurements and transformations</H3><H4>DLS measurements</H4><H6>Aafke and Gabriella</H6><P>To try out different protocols to isolate vesicles, sample #5 and #6 were prepared as follows:
    1.no centrifugation
    2.no cenrifugation + addition of 25 mg liposomes
    2.centrifugation for 3 minutes at 1400<SMALL> x </SMALL><I>g</I>.
    3.centrifugation for 20 minutes at 3300<SMALL> x </SMALL><I>g</I>.</P><H4>Transformation GFP-TorA and GFP into KEIO</H4><H6>Aafke and Gabriella</H6><P>GFP and GFP-TorA were transformed into KEIO-cells following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV></DIV></DIV></DIV></DIV></LI></DIV><LI><DIV class="collapsible-header scrollspy" id="week32-scroll">Week 32</DIV></LI><LI><DIV class="collapsible-header">Monday 7 August</DIV><DIV class="collapsible-body" id="d20170807"><DIV class="C"><DIV class="row notebook cas13" id="C20170807"><H4>Starter culture preparation</H4><H6>Kasper</H6><P>10 mL of ampicillin stock was prepared, with a concentration of 50 mg/mL. The concentration in medium will be 100 mg/mL. The stock was prepared by dissolving 500 mg of ampicillin in 10 mL milli-Q. Subsequently, the solution was filtered (200 nm) and stored at -20 °C. 
            Additionally, a starter culture for protein production was prepared. This was done by scraping one colony of the grown plates and inoculating them in 20 mL TB medium with 40 µL ampicillin. This starter culture was left overnight at 37 °C and 200 rpm.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170807"><H3>Protein Purification (Day 2)</H3><H6>Guillermo and Isabell</H6><P>The TDP purification was completed with the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday2" target="blank">TDP purification protocol (Day 2)</A> and the samples B1<SUB>IPTG</SUB>, B1<SUB>x</SUB>, A5<SUB>IPTG</SUB> (1), A5<SUB>IPTG</SUB> (2), A5<SUB>x</SUB> and AI. However, only the induced samples B1<SUB>IPTG</SUB>, A5<SUB>IPTG</SUB> (1) and (2) were lyophilized.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170807"><H3>Vesicles Preparation Widefield samples</H3><H6>Aafke and Floor</H6><P>The transformatin for GFP-TorA in KEIO was succesful.</P><P>Overnight cultures were prepared for GFP-TorA in WT and in KEIO and for GFP in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 8 August</DIV><DIV class="collapsible-body" id="d20170808"><DIV class="C"><DIV class="row notebook cas13" id="C20170808"><H4>Induction of Cas13a expression</H4><H6>Kasper</H6><P>The OD600 of the starter culture was found to be 1.56, relative to empty TB medium. 4 mL of this starter culture was inoculated into each 1 L flask of TB medium and subsequently 2 mL of the ampicillin stock solution was added. All four erlenmeyers were put innto the shaking incubator (37 °C, 180 rpm). 
    5 g of IPTG was received from Anna (Stan Brouns lab) and a stock solution with a concentration of 500 mM was made. For this purpose, 1.2 g of IPTG was dissolved in 10 mL of milli-Q and subsequently filtered through 0.2 µm. The IPTG was stored at -20 °C. 
    In the afternoon, after approximately five hours, the OD600 of the cultures was found slightly above 0.6, so expression was induced by addition of 1 mL of IPTG solution. Subsequently, cultures were put in the shaking incubator (18 , 180 rpm) overnight.
    </P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170808"><H3>Protein Purification (Day 0) and Preparation of Purification Samples for SDS-PAGE</H3><H4>Protein Purification (Day 0)</H4><H6>Guillermo</H6><P>A new purification with the colonies 3.5, 4.6 and 6.2 (B3, B4 and A6, respectively) following the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday2" target="blank">TDP purification protocol (Day 0)</A> was performed, with the exception of induction of A6 with 10 mL arabinose in addition to the standard induction with IPTG. All cultures were induced when they reached the exponential growth phase (B3 and A6 at OD = =0.55 and B4 at OD = 0.54).</P><P>N.B.: The induction of the 1 L cultures was not adjusted to the starter culture OD, instead they were simply induced with 12 mL of the culture medium.</P><H4>Pouring of SDS-PAGE gel</H4><H6>Guillermo</H6><P>A 12 % gel was poured following the protocol for <A href="https://2017.igem.org/Team:TUDelft/Notebook#poursdspage" target="blank">pouring a SDS-PAGE gel</A>.</P><H4>Preparation of Purification Samples for SDS-PAGE</H4><H6>Guillermo</H6><P>During the protein purification procedure started on 02-08-2017 aliquots from all samples were taken from the lysates before (BD) and after (AD) dialysis and prepared for SDS-PAGE (12 %) with the previously poured gel. The following volumes were used after the concentration was measured by <A href="https://2017.igem.org/Team:TUDelft/Notebook#proteinnanodrop" target="blank">nanodrop</A>:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>Purification Status</TH><TH>Concentration [g/L]</TH><TH>Loading Volume [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1<SUB>IPTG</SUB></TD><TD>before dialysis (BD)</TD><TD>44</TD><TD>5</TD></TR><TR><TD class="firstcolumn">B1<SUB>x</SUB></TD><TD>before dialysis (BD)</TD><TD>25</TD><TD>5</TD></TR><TR><TD class="firstcolumn">A5<SUB>IPTG</SUB> (1)</TD><TD>before dialysis (BD)</TD><TD>28.23</TD><TD>5</TD></TR><TR><TD class="firstcolumn">A5<SUB>IPTG</SUB> (2)</TD><TD>before dialysis (BD)</TD><TD>34</TD><TD>5</TD></TR><TR><TD class="firstcolumn">A5<SUB>x</SUB></TD><TD>before dialysis (BD)</TD><TD>13</TD><TD>5</TD></TR><TR><TD class="firstcolumn">B1<SUB>IPTG</SUB></TD><TD>after dialysis (AD)</TD><TD>0.7</TD><TD>15</TD></TR><TR><TD class="firstcolumn">B1<SUB>x</SUB></TD><TD>after dialysis (AD)</TD><TD>0.67</TD><TD>15</TD></TR><TR><TD class="firstcolumn">A5<SUB>IPTG</SUB> (1)</TD><TD>after dialysis (AD)</TD><TD>0.9</TD><TD>15</TD></TR><TR><TD class="firstcolumn">A5<SUB>IPTG</SUB> (2)</TD><TD>after dialysis (AD)</TD><TD>0.9</TD><TD>15</TD></TR><TR><TD class="firstcolumn">A5<SUB>x</SUB></TD><TD>after dialysis (AD)</TD><TD>0.9</TD><TD>15</TD></TR></TBODY></TABLE><P>The samples were prepared and run on the prepared SDS-PAGE gel following the <A href="https://2017.igem.org/Team:TUDelft/Notebook#sdspage" target="blank">SDS-PAGE protocol</A>.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170808"><H3>Vesicles Widefield microscopy</H3><H6>Aafke and Floor</H6><P>For the samples of GFP-TorA in WT and in KEIO a coverglass was prepared for Widefield microscopy following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#microscopecoverglass" target="_blank">Preparing coverglass for microscope</A> protocol.</P><P>The following picture was obtained for GFP-TorA in WT .</P><P>The following pictures were obtained for GFP-TorA in KEIO.</P><P>You can clearly see a difference in the morphology of KEIO compared to the WT. The KEIO-cells are longer and might even appaer filamentous. Above that, growth and protein production seems to be impaired, leading to less GFP-activity and less but longer cells.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 9 August</DIV><DIV class="collapsible-body" id="d20170809"><DIV class="C"><DIV class="row notebook cas13" id="C20170809"><H4>Pellet storage</H4><H6>Kasper</H6><P>In the morning, cultures were transferred into centrifugation tubes amd were centrifuged at 4 °C and 5200<SMALL> x </SMALL><I>g</I> for 15 minutes. Then the supernatant was discarded and resuspended into 30 mL of 1xPBS. This was put into falcon tubes (50 mL) and centrifuged at 3000<SMALL> x </SMALL><I>g</I> at 4 ° for 5 minutes. The supernatant was again discarded and pellets were stored at -80 °C. <U>NOTE</U>: The resuspension step into 1xPBS deviates from the original protocol by Gootenburg et al., 2017.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170809"><H3>Protein Purification (Day 1)</H3><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out using the cell pellets of the cultures B3, B4 and A6 from the previous day (08-08-2017).</P><P>N.B.: The sodium phosphate solution was not prepared to be at pH = 7.4 previously. From this purification on, the sodium phosphate solution was prepared correctly.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170809"><H3>Vesicles Digestion and ligation of GFP-TorA and pSB4A5</H3><H6>Aafke</H6><P>To be able to control the expression of GFP, GFP-TorA will be cloned onto a low copy number plasmid.pSB4A5 was digested with EcoRI and PstI following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> protocol, in 50 µL total.GFP-TorA was already digested on 25/07. </P><P>Both samples were cleaned-up according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>18.00</TD><TD>2.03</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pSB4A5</TD><TD>9.40</TD><TD>1.83</TD></TR></TBODY></TABLE><P>Next, the digested DNA-fragments were ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation (sticky-end)</A> protocol and incubated at roomtemperature.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Sterile milli-Q</TD><TD>1.5</TD></TR><TR><TD class="firstcolumn">Backbone pSB4A5</TD><TD>10</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>5.5</TD></TR><TR><TD class="firstcolumn">Buffer</TD><TD>2</TD></TR><TR><TD class="firstcolumn">Ligase</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 10 August</DIV></LI><LI><DIV class="collapsible-header">Friday 11 August</DIV><DIV class="collapsible-body" id="d20170811"><DIV class="C"><DIV class="row notebook cas13" id="C20170811"><H4>Buffer preparation</H4><H6>Kasper</H6><P>The following buffers were prepared: Lysis buffer (The composition can be found in the full<A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol. However, without the imidazole.) and SUMO digest buffer. The latter buffer is composed as follows: </P></DIV></DIV></DIV></LI><LI>30 mM Tris-HCL</LI><LI>500 mM NaCl</LI><LI>1 mM DTT</LI><LI>0.15  Tergitol (70% in H<SUB>2</SUB>O stock, borrowed from Dogtrom lab)</LI><P>The pH of the solution was adjusted to 8 by addition of 10 M NaOH.</P></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170811"><H3>Protein Purification (Day 2) and SDS-PAGE</H3><H4>Protein Purification (Day 2)</H4><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday2" target="blank">TDP purification protocol (Day 2)</A> was carried out using the lysates of the cultures B3, B4 and A6 from the 09-08-2017.</P><H4>SDS-PAGE</H4><H6>Guillermo</H6><P>Again, aliquots of the lysates were taken before and dialysis and prepared and run on the prepared SDS-PAGE gel following the <A href="https://2017.igem.org/Team:TUDelft/Notebook#sdspage" target="blank">SDS-PAGE protocol</A>.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170811"><H3>Vesicles Widefield microscopy</H3><H6>Gabriella</H6><P>As part of the Widefield microscope training, new pictures were taken of GFP-TorA in KEIO with the Widefield microscope. The pictures are shown below.</P><P>Again, the expression of GFP is not that high. Also, the morphology of the cells differs from the WT as seen before.</P></DIV></DIV></DIV></DIV><LI><DIV class="collapsible-header scrollspy" id="week33-scroll">Week 33</DIV></LI><LI><DIV class="collapsible-header">Monday 14 August</DIV><DIV class="collapsible-body" id="d20170814"><DIV class="C"><DIV class="row notebook cas13" id="C20170814"><H3>Cas13a Gibson assembly check</H3><H4>Phusion PCR</H4><H6>Kelly</H6><P>The Gibson assembly mix from 11-08-2017 was diluted (7.5 μL mix with 12.5 μL milli-Q). The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed to amplify check which ligations in the Gibson assembly mix took place. GC buffer was used with the following primers:
            <TABLE><THEAD><TR><TH>Part</TH><TH>Primers</TH><TH>Annealing temperature (°C)</TH><TH>Expected size (bp)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Spacer to backbone (SPBB)</TD><TD>IG0007 and IG0026</TD><TD>63</TD><TD>335</TD></TR><TR><TD class="firstcolumn">part1 to part2 to backbone (P1P2BB)</TD><TD>IG0006 and IG0025</TD><TD>63</TD><TD>3,327</TD></TR><TR><TD class="firstcolumn">part1 to backbone (P1BB)</TD><TD>IG0006 and IG0024</TD><TD>63</TD><TD>1,977</TD></TR><TR><TD class="firstcolumn">part1 to part2 (P1P2))</TD><TD>IG0001 and IG0025</TD><TD>68</TD><TD>2,508</TD></TR><TR><TD class="firstcolumn">part2 to spacer to backbone (P2SPBB)</TD><TD>IG0004 and IG0007</TD><TD>63</TD><TD>1,439</TD></TR><TR><TD class="firstcolumn">part1 to part2 to spacer to backbone (circular) (P1P2SPBB)</TD><TD>IG0006 and IG0007</TD><TD>63</TD><TD>4,500</TD></TR><TR><TD class="firstcolumn">part1 to backbone</TD><TD>IG0010 and IG0029</TD><TD>71</TD><TD>5,913</TD></TR><TR><TD class="firstcolumn">part2 to spacer (P2SP)</TD><TD>IG0030 and IG0023</TD><TD>68</TD><TD>1,851</TD></TR><TR><TD class="firstcolumn">part1 to spacer to backbone</TD><TD>IG0030 and IG0011</TD><TD>68</TD><TD>5,226</TD></TR></TBODY></TABLE></P><P>The PCR products were loaded on an agarose gel and ran for 45 minutes.</P><P>The gel shows that SPBB, P1P2BB, P1BB, P1P2 and P2SPBB are present.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170814"><H3>Vesicles Preparation pET-Duet1</H3><H6>Floor and Gabriella</H6><P>To obtain the empty backbone pET-Duet1 without the TolR-fragment, the backbone will be amplified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol, with 1 µL of DNA template (TolR-AU) and 35.5 µL of milli-Q; Only 1 µL of each of the primers (IG0051 and IG0052) was used. The PCR program was run with an extension time of 81 seconds for TolR-AU.</P><P> The PCR sample was run on gel according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A> protocol.
    The result can be seen in the picture below (with 1 and 2 being the empty backbone and B the blank). </P><P>The gel was run again and the indicated band was cut out of the gel.</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 15 August</DIV><DIV class="collapsible-body" id="d20170815"><DIV class="C"><DIV class="row notebook cas13" id="C20170815"><H4>SDS-PAGE</H4><H6>Kasper</H6><P>The resulting overnight Falcon tubes were spinned down (2 minutes at 2000<SMALL> x </SMALL>) at 4 °C and 50 µL was taken and ran on a 12% SDS-PAGE. </P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170815"><H3>Sample prep RNA gel electrophoresis of RPA with in vitro transcription</H3><H6>Amária and Kasper</H6><P>
     We performed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol.We used the high range riboruler from Thermo Fisher, so we used 2 µL ladder with 2 µL loading dye. we put the samples and ladder at 70 °C for 10 min and imediately transfered to ice for 5 min.We prepaired a 1 % agarose gel. We ran it at 80V for 40 min.The following picture shows the resulting RNA gel:
      We took 1 µL RNA and diluted it with 9 µL nuclease free water, this was measured with the the nanodrop following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaconcentration" target="_blank">RNA concentration measurement (NanoDrop)</A> protocol.The concentration was 45.2 ng/ µL, so the stock solution had a concentration of 452 ng/ µL. We adden 10 µL RNA to 45,2 µL nuclease free water for a 100 ng/ µL working stock. Then we aliquoted the RNA for further use into 5 tubes of 11 µL. </P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170815"><H3>Vesicles Transformations</H3><H4>Ligation pET-Duet1</H4><H6>Aafke and Floor</H6><P>The band containing pET-Duet1 was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" target="_blank">gel product purification (Promega Wizard™ Kit)</A> protocol.The concentration of the purified sample was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>21.07</TD><TD>1.92</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #2</TD><TD>25.5</TD><TD>2.02</TD></TR></TBODY><TBODY><TR><TD class="firstcolumn">pET-Duet1 #3</TD><TD>30.5</TD><TD>1.91</TD></TR></TBODY></TABLE><P>The obtained linearized backbone was ligated according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated for 1 hour at room temperature.</P><P><I>!We forgot to phosphorylate first!</I></P><H4>Transformation</H4><H6>Aafke and Floor</H6><P>Cells from WT and KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    The following transformations were performed following the  <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>Antibiotics</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">GFP-TorA on pSB4A5</TD><TD>WT</TD><TD>Ampicillin</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>KEIO</TD><TD>Chloramphenicol and Kanamycin</TD></TR><TR><TD class="firstcolumn">pSB1C3</TD><TD>WT</TD><TD>Chloramphenicol</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>KEIO</TD><TD>Ampicillin and Kanamycin</TD></TR><TR><TD class="firstcolumn">pET-Duet1</TD><TD>WT</TD><TD>Ampicillin</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 16 August</DIV><DIV class="collapsible-body" id="d20170816"><DIV class="C"><DIV class="row notebook cas13" id="C20170816"><H4>Cas13a elution</H4><H6>Kasper</H6><P>The supernatant of the Streptactin resin was run through FPLC (AKTA) and subsequently eluted over a NaCl gradient of elution buffer. The elution buffer consisted of 20 mM TRIS-HCl, 1 mM DTT, 5% glycerol and 0-2 M NaCl. 
    There was no elution of Cas13a, indicating that the protein was probably never really bound to the column. To check this hypothesis, the initial flowthrough was loaded on a 10% SDS-PAGE gel.
    
    Lane 2 again contains the supernatant of the resin. Lane 3 contains the flowthrough from the FPLC. 
    The flowthrough was centrifuged over a filter (100 kDa, 4 °C, 3220<SMALL> x </SMALL>) in rounds of 20 minutes each until all flowthrough was filtered. The resulting solution was stored in a tube labelled &quot;1608 flowthrough filtered&quot; and stored for the next SDS-PAGE gel. 
                    Two colonies from the BL21(DE3) cells in which the Zhang lab plasmid was transformed were plated on fresh plates.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170816"><H3>Vesicles Ligation pET-Duet1</H3><H6>Aafke and Gabriella</H6><P>The blunt-end ligation of linearized pET-Duet1 was performed again following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">Ligation (blunt-end)</A> protocol and incubated overnight at 4 °C;.</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 17 August</DIV><DIV class="collapsible-body" id="d20170817"><DIV class="C"><DIV class="row notebook cas13" id="C20170817"><H3>Cas13 amplification of 'P1-P2-Sp'</H3><H4>Amplification</H4><H6>Kelly</H6><P>Since it seemed that indeed the right primers were used, it was more likely that #8 from 14/08/2017 was in fact the P1-P2-Spacer insert complete with prefix and suffix. The PCR product at least had the right size. Therefore, this part was amplified in two parallel PCRs.</P><P>PCR programme 1: Used primers IG0006 and IG0007. Expected size of product: 4500 bp. Tm used: 63.5°C.</P><P>PCR programme 2: Used primers IG0022 and IG0023. Expected size of product: 4350 bp. Tm used: 72°C.</P><P>Furthermore, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed.</P><P>Results: For both primer sets the sample containing DMSO (lane 3 and 5) gives a more specific result than the sample without DMSO. Primer set IG0006 and IG0007 seems to give a more specific result. Conclusion: sample should be gel extracted.</P><H4>Gel Isolation</H4><H6>Kelly</H6><P>The remainder of C45 was loaded on gel (~21uL + 3.5uL loading dye). It was cut from gel and purified. Finally, the concentration was measured on the Nanodrop, yielding a concentration of 4.76 ng/uL. The gel used was an 0.6% agarose gel.</P><H4>PCR amplificication</H4><H6>Kelly</H6><P>Both the gel extracted sample and the PCRed samples with and without DMSO and primers IG00006-IG0007 were used as template for another round of PCR. For this PCR, the same programsas before were used (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase) protocol</A>). We used GC buffer and DMSO for all reactions. The following templates and primers were used:</P><TABLE><THEAD><TR><TH>Sample</TH><TH>Containing</TH><TH>Pirmers</TH></TR></THEAD><TBODY><TR><TD>G6</TD><TD>Gel extracted &quot;8&quot; (C46)</TD><TD>IG0006-IG0007</TD></TR><TR><TD>C23</TD><TD>Gel extracted &quot;8&quot; (C46)</TD><TD>IG0023-IG0022</TD></TR><TR><TD>6+</TD><TD>PCRed &quot;8&quot; with DMSO</TD><TD>IG0006-IG0007</TD></TR><TR><TD>6-</TD><TD>PCRed &quot;8&quot; without DMSO</TD><TD>IG0006-IG0007</TD></TR></TBODY></TABLE><P>The PCR products were loaded on an agarose gel an ran for 45 minutes.</P><P>We drew the following conclusions:</P><P> Both gel extracted samples barely have a yield and the product is the wrong size.</P><P>The samples were very diluted (for &quot;8&quot;), but at least they had a vague of the right size (even though only 0.4 μL was used as template).</P><P>All samples were purified (from another gel).</P><P>The prodcuts were Nanodropped (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A>), which gave th following results:</P><TABLE><THEAD><TR><TH>Sample</TH><TH>Concentration (ng/μL)</TH><TH>260/280</TH></TR></THEAD><TBODY><TR><TD>6-</TD><TD>141.8</TD><TD>1.92</TD></TR><TR><TD>6+</TD><TD>192.6</TD><TD>1.90</TD></TR><TR><TD>22-</TD><TD>237.2</TD><TD>1.85</TD></TR><TR><TD>22+</TD><TD>250.4</TD><TD>1.83</TD></TR></TBODY></TABLE><H4>Sequencing samples</H4><H6>Kelly</H6><P>The gel extracted &quot;8&quot; and the PCRed &quot;8&quot; +DMSO were sent for sequencing with primer IG0003 (code: 18A1ZAG096) following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing protocol from Macrogen</A> was followed.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170817"><H3>Vesicles Transformation &amp; preparation 96-wells experiment</H3><H4>Transformation</H4><H6>Aafke and Gabriella</H6><P>Transformation of pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation of chemical competent cells</A> protocol.
    </P><H4>preparation 96-wells experiment</H4><H6>Aafke and Gabriella</H6><P>Starter cultures were prepared following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 18 August</DIV><DIV class="collapsible-body" id="d20170818"><DIV class="C"><DIV class="row notebook cas13" id="C20170818"><H4>Preparation of ampicillin stock</H4><H6>Kasper</H6><P>A new ampicillin stock was made, following the same procedure as before (07-08-2017).</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170818"><H3>RNA Assay Preparation</H3><H6>Guillermo</H6><P>The lyophilized sample B1 from 07-08-2017 was resuspended in 1 mL nuclease free water and its concentration measured by nanodrop (c = 10 mg/mL. The solution was aliquoted into aliquots of 100 µL and stored at -80 °C.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170818"><H3>Vesicles 96-wells experiment GFP</H3><H6>Aafke and Floor</H6><P>First, 230 µL LB was pipetted into each well and inoculated with 10 µL, 18 wells per culture for the following culture:
    *GFP on pSB1C3 in WT 
    *GFP-TorA on pSB1C3 in WT 
    *GFP on pSB1C3 in KEIO 
    *GFP-TorA on pSB1C3 in KEIO </P><P>An hour after inoculation, the cells are induced in triplo with the following final IPTG-concentrations:
    -0 mM
    -0.1 mM
    -0.3 mM
    -0.5 mM
    -0.7 mM
    -1 mM</P><P>After induction, the growth and fluorescence was monitored with a plate-reader.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week34-scroll">Week 34</DIV></LI><LI><DIV class="collapsible-header">Monday 21 August</DIV><DIV class="collapsible-body" id="d20170821"><DIV class="C"><DIV class="row notebook cas13" id="C20170821"><H4>Culture induction</H4><H6>Kasper</H6><P>The starter culture from the previous day had an OD600 of 1.96 relative to empty TB medium. Both 1 L flasks were inoculated with 4.5 mL of starterculture and subsequently 2 mL of ampicillin stock solution. Both were incubated at 37 °C and 180 rpm for approximately four hours. 
                 The OD600 was then found to be 0.68. Both erlenmeyers were put on ice for 30 minutes. Then 1 mL was taken from each culture and stored at 4 °C. Subsequently, to both erlenmeyers 1 mL of IPTG stock solution was added and they were put at 18 °C, 180 rpm overnight.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170821"><H3>Overnight cultures</H3><H6>Gabriella</H6><H6><P>Overnight cultures were prepared for pET-Duet1 and GFP-TorA on pSB4A5 into DH5α following <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P></H6></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 22 August</DIV><DIV class="collapsible-body" id="d20170822"><DIV class="C"><DIV class="row notebook cas13" id="C20170822"><H4>SDS-PAGE of Cas13a samples</H4><H6>Kasper</H6><P>Both cultures were taken out of the incubator and of both 1 mL was taken. Then, both flasks were centrifuged at 4 °C and 5200<SMALL> x </SMALL><I>g</I> for 15 minutes. Then the supernatant was discarded and resuspended into 30 mL of 1xPBS. This was put into falcon tubes (50 mL) and centrifuged at 3000<SMALL> x </SMALL><I>g</I> at 4 ° for 10 minutes. The supernatant was again discarded and pellets were stored at -80 °C. 
                 Sample after induced: 3.2. Before induction it was 0.7. Both were spinned down (4 °C, 2 minutes, 6000<SMALL> x </SMALL><I>g</I>) and the supernatant was discarded. Since 100 µL of the sample after induction was already taken out, the pellet of the induced sample was resuspended into 288 µL of 1xPBS. The pellet before induction was resuspended into 70 µL of 1xPBS. 10 µL of both resulting suspensions were mixed with 2 µL of SDS-PAGE buffer and incubated at 95 °C for 10 minutes. It was then loaded on a 10% SDS-PAGE gel. </P><P>Lane 1 contains the protein ladder. The third and fourth lane contain the sample before and after induction, respectively. The band around 140 kDa is significantly fatter in lane 4, indicating cas13a expression. However, there is also leaky expression. </P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170822"><H3>RNA Assay</H3><H6>Guillermo</H6><P>An <A href="https://2017.igem.org/Team:TUDelft/Notebook#rnaassay" target="blank">RNA Assay</A> was carried out with the samples prepared on 18-08-2017. Varations on the protocol were executed to determine if all steps are necessary. It included the following samples:</P></DIV></DIV></DIV></DIV></DIV></LI><LI>B1 (1) (resuspended protein B1, incubated at 30 °C for 30 minutes and purified with RNeasy MiniElute Kit)</LI><LI>B1 (2) (resuspended protein B1, incubated at 30 °C for 30 minutes and not purified.</LI><LI>B1 (3) (resuspended protein B1, not incubated, but purified with RNEasy MiniElute Kit)</LI><LI>C1 (control without B1, incubated at 30 °C for 30 minutes and purified with RNeasy MiniElute Kit)</LI><LI>C2 (control without B1, not incubated or purified)</LI><DIV class="V"><DIV class="row notebook vesicles" id="V20170822"><H3>Vesicles Miniprep GFP-TorA on pSB4A5</H3><H6>Gabriella</H6><H6><P>The sample with GFP-TorA on pSB4A5 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P>
    The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. <P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA #1</TD><TD>258</TD></TR><TR><TD class="firstcolumn">GFP-TorA #2</TD><TD>251</TD></TR></TBODY></TABLE></P></H6></DIV></DIV><LI><DIV class="collapsible-header">Wednesday 23 August</DIV><DIV class="collapsible-body" id="d20170823"><DIV class="C"><DIV class="row notebook cas13" id="C20170823"><H4>Buffer preparation</H4><H6>Kasper</H6><P>Lysis buffer was prepared as indicated in the full<A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol (without the imidazole). Furthermore, an elution buffer with the following composition was prepared: 20 mM TRIS-HCl, 1 mM DTT, 250 mM imidazole.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170823"><H3>Sample prep transformation PSB1T3 in DH5-alfa</H3><H6>Amária</H6><P>
    We transformed PSB1T3 into the commercial DH5-alpha of NEB, we followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationDH5alpha" target="_blank">Transformation NEB DH5-alpha</A> protocol. We used 3 µL of 100 ng/ µL PSB1T3. We added to much DH5-alha, more than 50 µL.
    </P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170823"><H3>Vesicles Transformation pET-Duet1</H3><H4>Miniprep pET-Duet1</H4><H6>Aafke</H6><P>pEt-Duet1 in DH5α was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>46.71</TD><TD>1.88</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>62.27</TD><TD>1.94</TD></TR></TBODY></TABLE><H4>Transformation pET-Duet1 into KEIO &amp; WT</H4><H6>Aafke</H6><P>4 µL of pET-Duet1 #2 was transformed into 50 µL KEIO and WT following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 24 August</DIV><DIV class="collapsible-body" id="d20170824"><DIV class="C"><DIV class="row notebook cas13" id="C20170824"><H4>Nickel-Hisselect column purification</H4><H6>Kasper</H6><P>Cell pellet was thawn on ice and dissolved into 25 mL lysis buffer. It was subsequently French pressend and spinned down (1600 rpm, 4 °C, 45 minutes) in the ultracentrifuge. Supernatant was fitered through a 0.45 µm fiter. 
                 500 mL of 50% (v/v) Hisselect Nickel affinity gel was received from Anna (Stan Brouns lab) and this was washed in 5 mL lysis buffer three times. The filtered supernatant was added to this column material and incubated at 4 °C with rotation for 30 minutes. The mix was spinned down and the unbound fraction was collected in a clean falcon tube. The resin was loaded onta a gravity column and three wash fractions of 2 mL with lysis buffer were collected and stored at 4 °C. Elution was done by loading and collecting 300 µL elution buffer (5x) into five elution fractions (E1-E5). With the nanodrop the A280 for the samples E1-E5 was determined to be: 0.9270, 2.1992, 0.7050, 0.3045, 0.2085, respectively.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170824"><H3>Sample prep transformation PSB1T3 in DH5-alfa result</H3><H6>Amária</H6><P>
    The transformation of the 23<SUP>th</SUP> August appeared to be working because colonies grew and were red (RFP from the plasmid).
    </P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170824"><H3>Vesicles Transformations and preparation osmoshock</H3><H4>Transformations</H4><H6>Aafke</H6><P>Electrocompetent cells of TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    2 µL of GFP-TorA and GFP were transformed into TolR-AU in KEIO.
    4 µL of pET-Duet1 #2 was transformed again into 50 µL KEIO and WT. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>100 mL of osmoshock-solution was prepared following the description in <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    The following cells were grown in 20 mL LB medium and induced with 0.1 mM IPTG at an OD between 0.5 and 0.7 and grown overnight.
    
                                        <TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid</TH><TH>Strain</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA</TD><TD>KEIO</TD><TD>0.59</TD></TR><TR><TD class="firstcolumn">GFP-TorA</TD><TD>WT</TD><TD>0.62</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>KEIO</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>WT</TD><TD>0.63</TD></TR></TBODY></TABLE></P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 25 August</DIV><DIV class="collapsible-body" id="d20170825"><DIV class="C"><DIV class="row notebook cas13" id="C20170825"><H4>SDS-PAGE and dialysis</H4><H6>Kasper</H6><P>An SDS-PAGE gel was ran with a ladder, His-select supernatant, wash 1, 2 and 3, and all five elutions. The elution fractions were diluted to a concentration of 0.21 mg/mL on the gel.</P>
            The elutions were mixed and buffer exchanged by dialysis, into a buffer of 30 mM TRIS-HCl, 500 mM NaCl 1 mM DTT at pH 8. In total 1.5 mL of elution was dipped into a dialysis bag of 6 cm and left for 22 hours in 750 mL, giving a 500x dilution of the original. The concentration of protein was found to be 0.640 mg/mL, with an A260/A280 of 1.22. The mix was stored at 4 °C for further purification and cleavage assays.
        </DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170825"><H3>RNA Assay</H3><H6>Guillermo</H6><P>An <A href="https://2017.igem.org/Team:TUDelft/Notebook#rnaassay" target="blank">RNA Assay</A> was carried out with the aliquots of B1 prepared on 18-08-2017. It included the following samples:</P></DIV></DIV></DIV></DIV></DIV></LI><LI>1 (resuspended protein B1)</LI><LI>2 (resuspended protein B1 with Proteinase K)</LI><LI>C1 (control without B1)</LI><LI>C2 (control without B1, with Proteinase K)</LI><DIV class="V"><DIV class="row notebook vesicles" id="V20170825"><H3>Vesicles Osmoshock and DLS-measurements</H3><H4>Osmoshock</H4><H6>Aafke</H6><P>The periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm.</P><H4>DLS-measurements</H4><H6>Floor and Gabriella</H6><P>TolR-AU in WT and pET-Duet1 in WT were inocculated in 10 mL LB and the growth was monitored; pET-Duet1 in WT did not grow. TolR-AU in WT #1 was induced at an OD of 0.45 and TolR-AU in WT #2 at an OD of 0.41. 
    After 2 hours, the samples of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. </P><P>pET-Duet1 was transformed again into KEIO and WT.(<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV></DIV><LI><DIV class="collapsible-header scrollspy" id="week35-scroll">Week 35</DIV></LI><LI><DIV class="collapsible-header">Monday 28 August</DIV><DIV class="collapsible-body" id="d20170828"><DIV class="C"><DIV class="row notebook cas13" id="C20170828"><H3>Cas13 PCR amplifications of the plasmid and the insert</H3><P>This day we decided on a different strategy: we decided to do the whole plasmid PCR again, but in more tubes, so we could perform a gel extraction.</P><P>In parallel, we decided to run a Taq amplification of the insert (on the Gibson assembly mix) to produce a high yield of product. If this works, we could then ligate the product to the pGEM cloning plasmid, so we can sequence properly.</P><H4>Phusion PCR</H4><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase) protocol</A> was followed. Instead of HF-buffer we used GC-buffer and DMSO.</P><P> As template we used the Gibson assembly mix with the primers IG0055 and IG0056.The used annealing temperature was 66 °C with an extension time of 231 seconds.</P><P>Three tubes were made from i1 and 3 of i2.</P><H4>GoTaq PCR</H4><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank">Colony PCR (GoTaq) protocol</A> was followed.</P><P>For the template C48 was used (purified PCR product of &quot;8&quot;. Primers IG0053 and IG0054 were used. A gradient was used for the annealing temperature with 10 tubes in total:</P></DIV></DIV></DIV></LI><LI>50 °C</LI><LI>51 °C</LI><LI>52 °C</LI><LI>53 °C</LI><LI>54 °C</LI><LI>55 °C</LI><LI>49 °C</LI><LI>48 °C</LI><LI>47 °C</LI><LI>46 °C</LI><P>The extension time was set to 4.5 minutes.</P><P>Both the products from the Phusion PCR and the products from the GoTaq PCR were load on an agaros gel and ran for 45 mintues (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaeletrophoresis" target="_blank">DNA electrophoresis</A>)</P><P>The whole plasmid PCR did not work. We will try it again with a diluted template.</P><P>The Taq amplification of the insert did work, but there is a big smear. We will try it again with diluted template and DMSO to enahnce the specificity.</P><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170828"><H3>Vesicles Transformations</H3><H6>Aafke and Floor</H6><P>pET-Duet1 was transformed again into KEIO and WT. pSB1C3 was transformed into TolR-AU in KEIO. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P></DIV></DIV></DIV><LI><DIV class="collapsible-header">Tuesday 29 August</DIV><DIV class="collapsible-body" id="d20170829"><DIV class="C"><DIV class="row notebook cas13" id="C20170829"><H4>Preparation of overnight culture and TB medium</H4><H6>Kasper and Jeroen</H6><P>An overnight culture was prepared from plat into TB medium. Additionally, 2 L of TB medium was prepared, with the composition stated in the full<A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170829"><H3>Vesicles DLS-measurements</H3><H6>Aafke and Floor</H6><P>10 mL samples of TolR-AU in WT and pET-Duet1 in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the purification only after 2 hours. <I>The cells were grown on LB without antibiotics.</I> The samples were analysed with the DLS. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">TolR-AU in WT #1</TD><TD>0.66</TD></TR><TR><TD class="firstcolumn">TolR-AU in WT #2</TD><TD>0.46</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #1</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">pET-Duet1 in WT #2</TD><TD>0.55</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 30 August</DIV><DIV class="collapsible-body" id="d20170830"><DIV class="C"><DIV class="row notebook cas13" id="C20170830"><H4>Induction of protein expression</H4><H6>Kasper</H6><P>The OD600 of the starter culture was 2.00. So, 4 mL starter culture was added to two flasks with 1 L TB medium and 2 mL ampicillin. This was incubated at 37 °C, 180 rpm. After approximately 4 hours, OD600s of 0.52 and 0.57 were reached. We applied a cold shock for 30 minutes, after which 1 mL of IPTG was added to the cultures. They were then incubated overnight at 18 °C and 180 rpm.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170830"><H3>Preparation of overnight cultures for protein purification</H3><H6>Gabriella, Guillermo</H6><P>An overnight culture of the strain B1 was induced and grown overnight in a shaking incubator at 37 °C and 300 rpm.</P><P>NB: The culture only gre to an OD of 1.6 even after two days in the shaking incubator. Even induction of a second culture did not yield a higher OD overnight.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170830"><H3>Vesicles Transformations and DLS-measurements</H3><H4>Transformation pET-Duet1 in KEIO</H4><H6>Aafke and Floor</H6><P>pEt-Duet1 in WT was minipreped following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. </P><P>The concentration of the DNA was measured using the the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaconcentration" target="_blank">DNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>dsDNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">pET-Duet1 #1</TD><TD>43.57</TD><TD>1.85</TD></TR><TR><TD class="firstcolumn">pET-Duet2 #2</TD><TD>60.81</TD><TD>2.22</TD></TR></TBODY></TABLE><P>5 µL of pET-Duet1 #2 was transformed into 150 µL KEIO following <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><H4>DLS-measurements</H4><H6>Aafke and Floor</H6><P>10 mL of TolR-AU in KEIO were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.45 for both samples and the purification only after 2 hours. </P>
    10 mL of non-induced pET-Duet1 in WT was used as a positive control, with 1.25 µM liposomes added before purification.<I>The cells were grown in LB without antibiotics.</I> The samples were analysed with the DLS.</DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 31 August</DIV><DIV class="collapsible-body" id="d20170831"><DIV class="C"><DIV class="row notebook cas13" id="C20170831"><H4>Pellet storage</H4><H6>Kasper</H6><P>Two 1 mL-samples were taken from the cultures. These samples were spinned down at 4 °C and 5200<SMALL> x </SMALL><I>g</I> for 15 minutes. Then the supernatant was discarded and resuspended into 40 mL of 1xPBS. This was put into falcon tubes (50 mL) and centrifuged at 3220<SMALL> x </SMALL><I>g</I> at 4 ° for 10 minutes. The supernatant was again discarded and pellets were stored at -80 °C.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170831"><H3>Protein Purification (Day 0) and Transformation of plasmids</H3><H4>Protein Purification (Day 0)</H4><H6>Gabriella, Guillermo</H6><P>The overnight culture of strain B1 from 30-08-2017 was used to induce a 1 L culture according to the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A>.</P><H4>Transformation of plasmids into BL21 DE3</H4><H6>Gabriella, Isabell</H6><P>Chemically competent E.coli BL21(DE3) cells from New England Biolabs (C2527H) were transformed with the miniprepped plasmids from colony 1.5, 3.5, 4.6, 5.1 and 6.2, according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation</A> protocol, because cell growth had been stunted previously. Since all plasmids were transformed into BL21 (DE3), they will be referred to as B1, B3, B4, B5 and B6 from now on.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170831"><H3>Vesicles Preparation osmoshock and DLS-measurements</H3><H4>Preparation osmoshock</H4><H6>Aafke</H6><P>KEIO and WT were inoculated in 20 mL LB medium in duplo and grown overnight. Above that, the following cells were inoculated in 20 mL LB medium, induced with 0.1 mM IPTG and grown overnight.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>OD at induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in KEIO #1</TD><TD>0.45</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO #2</TD><TD>0.33</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #1</TD><TD>0.67</TD></TR><TR><TD class="firstcolumn">GFP in KEIO #2</TD><TD>0.48</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP-TorA in WT #2</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP in WT #1</TD><TD>0.51</TD></TR><TR><TD class="firstcolumn">GFP in WT #2</TD><TD>0.49</TD></TR></TBODY></TABLE><H4>DLS measurements pET-Duet1 in KEIO</H4><H6>Floor</H6><P>10 mL of pET-Duet1 in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol, with the induction at an OD of 0.31 and 0.34 and the measurement only after 2 hours. <I>The cells were grown in LB without antibiotics.</I>
    Next, 200 µL of the purified samples was analysed with the DLS.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 1 September</DIV><DIV class="collapsible-body" id="d20170901"><DIV class="C"><DIV class="row notebook cas13" id="C20170901"><H3>Cas13 colony PCR</H3><H4>Colony PCR</H4><H6>Fiona</H6><P>From the plates with colonies, 50 colonies were picked and a colony PCR was done following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq) protocol</A>. Primers IG0053 and IG0054 were used. The annealing temperature was 49 °C and the extension time was 4.5 minutes.</P><P>The results are displayed in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> photos below.</P><P>The PCR seemed unsuccessful. We decided to do a double digest, to see if we could confirm the presence of the plasmid. We used the restriction enzymes XbaI and SpeI, which could cut the gene out, resulting in bands of around 3,000 bp and 4,500 bp.</P><H4>Double digestion</H4><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay protocol</A> was followed.</P><P>The results are displayed in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> photo below.</P><P>The digest did not seem to be successful.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170901"><H3>Protein Purification (Day 1)</H3><H6>Gabriella, Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out with cell pellets of culture B1 prepared on 31-08-2017.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170901"><H4>Osmoshock</H4><H6>Floor</H6><P>Both the cytoplasmic and the periplasmic fraction of the overnight cultures was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="_blank">osmoshock</A> protocol. 
    <I>In step 6 and 9, 4 mL milli-Q was used instead of 2.5 mL.</I> With the plate reader, the fluorescence was measured at an excitation/emission of 475/510 nm. Prior to the osmoshock, the pellets were weighted and used for normalization of the data:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Cells</TH><TH>Weight (g) No Induction</TH><TH>Weight (g) Induction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA WT 1</TD><TD>0.0831</TD><TD>0.0901</TD></TR><TR><TD class="firstcolumn">GFP-TorA WT 2</TD><TD>0.0474</TD><TD>0.0662</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 1</TD><TD>0.0638</TD><TD>0.0511</TD></TR><TR><TD class="firstcolumn">GFP-TorA KEIO 2</TD><TD>0.0610</TD><TD>0.0568</TD></TR><TR><TD class="firstcolumn">GFP WT 1</TD><TD>0.0757</TD><TD>0.0658</TD></TR><TR><TD class="firstcolumn">GFP WT 2</TD><TD>0.0658</TD><TD>0.0839</TD></TR><TR><TD class="firstcolumn">GFP KEIO 1</TD><TD>0.1094</TD><TD>0.0678</TD></TR><TR><TD class="firstcolumn">GFP KEIO 2</TD><TD>0.0533</TD><TD>0.0655</TD></TR><TR><TD class="firstcolumn">KEIO</TD><TD>0.0649</TD><TD>-</TD></TR><TR><TD class="firstcolumn">WT</TD><TD>0.0575</TD><TD>-</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></LI></DIV><LI><DIV class="collapsible-header scrollspy" id="week36-scroll">Week 36</DIV></LI><LI><DIV class="collapsible-header">Monday 4 September</DIV><DIV class="collapsible-body" id="d20170904"><DIV class="C"><DIV class="row notebook cas13" id="C20170904"><H4>Making crDNA</H4><H6>Kasper</H6><P>We prepared <A href="https://2017.igem.org/Team:TUDelft/Protocols#primerworkingstockpreparation" target="_blank">primer stock solutions of</A> DNA oligo's IG0057-IG0063 in nuclease-free water. Next, 10 tubes were prepared, each containing 8 µL of nuclease-free. Five contained the T7 promoter (IG0057) and a target. The other five contained the T7 promoter ligated to the hairpin complement (IG0058). All tubes were then annealed according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#crannealing" target="_blank">crDNA annealing</A> protocol from Christobal (Stan Brouns lab) so that the target sequences bound to their T7 promoter (and optionally the hairpin).
                                         After annealing, an 8% acrylamide gel was ran to confirm annealing. Specifications of this gel were: 330 V, 500 mA, 45 minutes, SYBR gold staining.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170904"><H3>Sample prep calculate cell concentration</H3><H6>Amária</H6><P> We followed the protocol for <A href="https://2017.igem.org/Team:TUDelft/Protocols#agarplate" target="_blank">Pouring agar plates</A>. 15 ng/ mL Tetracycline was used.
    From the dilutions 10<SUP>-3</SUP>, 10<SUP>-4</SUP> and 10<SUP>-5</SUP> made on the 4<SUP>th</SUP> of September, we plated 100 µL with a hockeystick plater. We let them grow overnight at 37 °C.
    </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170904"><H3>Protein Purification (Day 0) and Preparation for Mass Spectrometry</H3><H4>Protein Purification (Day 0)</H4><H6>Gabriella, Guillermo</H6><P>The overnight culture of strains B1 and B5 from 03-09-2017 was used to induce a 1 L culture according to the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A>.</P><H4>Preparation for Mass Spectrometry</H4><H6>Gabriella</H6><P>Mass Spectrometry samples of cultures B1, B3, B4, B5 and B6 were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Notebook#massspecprep1" target="blank">Mass Spectrometry Preparation Protocol 1</A>.</P><P>NB: The cells did not grow well, so the preparation was repeated on the next day.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 5 September</DIV><DIV class="collapsible-body" id="d20170905"><DIV class="C"><DIV class="row notebook cas13" id="C20170905"><H4>In vitro transcription of crDNA</H4><H6>Kasper</H6><P>All ten tubes were in vitro transcribed using the following procedure: 
                                                 The remaining liquid in each tube was incubated with 0.5 µL of each of the nucleotides ATP/GTP/UTP/CTP; 1 µL Murine RNase Inhibitor; 5 µL 1x Rna Polymerase buffer; 1 µL T7 polymerase mix; 36 µL nuclease free water. Incubation took place at 37 °C for 3 hours. Subsequntly, all tubes except for the ones of target 5 (5) and target 5 wih the hairpin (5H) were frozen at -80 °C. The tubes of 5 and 5H were cleaned up using an RNA MinElute kit. The resulting 14 µL of presumably clean RNA was measured in the Nanodrop and values around 20 ng/µL were obtained. Of both, 3 µL was loaded on an RNA gel.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170905"><H3>Sample prep RPA with in vitro transcription on samples from our microwave DNA extraction method</H3><H6>Amária and Kasper</H6><P>
      We did recombinase polymerase amplification (RPA) on the DNA extracted with the microwave method from the dilution serie duplicate #1 with different concentrations of PSB1T3 in DH5-alpha made on the 4<SUP>th</SUP> of August. We followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA + in vitro transcription</A> protocol. We made a matermix for 6 samples so multiplied all the ingredients by 7. We added 3,95 µL of Target DNA to the reaction mix and no Milli Q. We used forward primer IG0016 and reverse primer IG0017. We purified the RNA by following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnacleanandconcentrate" target="_blank">RNA clean and concentrate (Qiagen RNeasy MinElute clean up kit)</A> protocol.While doing the RNA clean up, during step 8 of the kit, the pipette tip fell into the column in sample 10<SUP>-3</SUP>, a small drop fell off, but not in the middle. We pipetted 14 µL more into the center of the column.
    </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170905"><H3>Protein Purification (Day 1), Preparation for Mass Spectrometry and Bradford Assay</H3><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out with cell pellets of cultures B1 and B5 prepared on 04-09-2017.</P><P>NB: cOmplete EDTA-free mini tablets were used instead of the usual cOmplete EDTA-free tablets.
            
            </P><H4>Preparation for Mass Spectrometry</H4><H6>Gabriella</H6><P>The preparation of the samples for mass spectrometry was modified to <A href="https://2017.igem.org/Team:TUDelft/Notebook#massspecprep2" target="blank">Mass Spectrometry Preparation Protocol 2</A> and carried out with cultures prepared with colonies of the transformation plates from 31-08-2017.</P><P>NB: After step 4, two solutions were pinkish while the others were yellow (colour of LB medium).</P><H4>Bradford Assay</H4><H6>Gabriella, Guillermo</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Notebook#bradfordassay" target="blank">Bradford Assay</A> of the proteins B3 (from 09-08-2017) and B5 (from 05-09-2017) resuspended in 1 mL MQ was carried out to determine their concentration:</P></DIV></DIV></DIV></DIV></LI><LI>c[B3] = 1.34 g/L</LI><LI>c[B5] = 1.7 g/L</LI></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170905"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT and of TolR-AU in WT were prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    TolR-AU in WT was induced at an OD of 0.57 and 0.58. pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV></DIV><LI><DIV class="collapsible-header">Wednesday 6 September</DIV><DIV class="collapsible-body" id="d20170906"><DIV class="C"><DIV class="row notebook cas13" id="C20170906"><H4>crRNA check</H4><H6>Kasper</H6><P>To check the compound loaded on the RNA gel yesterday (05-09-2017), an assay was done with both RNase A and DNase I. Firstly, a stock soltion of 0.02 wt% DNase I was made by taking 80 µL nuclease-free water and 20 µL of 1 wt% DNase I solution. Nevertheless, this was a miscalculatin, as actually only 2 µL should have been taken. However, for the assay this does not matter. 
                                                 The following six solutions were made:</P><TABLE><THEAD><TR><TH>Sample</TH><TH>Volume (µL)</TH><TH>Compounds</TH></TR></THEAD><TBODY><TR><TD rowspan="2">C</TD><TD>2</TD><TD>20 ng/µL crRNA5</TD></TR><TR><TD>1</TD><TD>nuclease-free water</TD></TR><TR><TD rowspan="2">RNase</TD><TD>2</TD><TD>20 ng/µL crRNA5</TD></TR><TR><TD>1</TD><TD>0.02 wt% RNase A</TD></TR><TR><TD rowspan="2">DNase</TD><TD>2</TD><TD>20 ng/µL crRNA5</TD></TR><TR><TD>1</TD><TD>0.02 wt% DNase I</TD></TR><TR><TD rowspan="2">C<SUP>H</SUP></TD><TD>2</TD><TD>19 ng/µL crRNA5H</TD></TR><TR><TD>1</TD><TD>nuclease-free water</TD></TR><TR><TD rowspan="2">RNase<SUP>H</SUP></TD><TD>2</TD><TD>19 ng/µL crRNA5H</TD></TR><TR><TD>1</TD><TD>0.02 wt% RNase A</TD></TR><TR><TD rowspan="2">DNase<SUP>H</SUP></TD><TD>2</TD><TD>19 ng/µL crRNA5H</TD></TR><TR><TD>1</TD><TD>0.02 wt% DNase I</TD></TR></TBODY></TABLE><P>All samples were left at 37 °C for 20 minutes and 3 µL of RNA loading dye was added. An RNA gel was ran using 1 µL of ladder.</P><P>The first lane contains the ladder. Lanes 2-7 contain the samples in the order that they are listed in in the table.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170906"><H3>Sample prep RNA gel electrophoresis of RPA with in vitro transcription on samples from our microwave DNA extraction method</H3><H6>Amária and Kasper</H6><P>
     We performed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol.We used the high range riboruler from Thermo Fisher, so we used 2 µL ladder with 2 µL loading dye. we put the samples and ladder at 70 °C for 10 min and imediately transfered to ice for 5 min.We prepaired a 1 % agarose gel. We ran it at 80V for 40 min.The following picture shows the resulting RNA gel:
      </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170906"><H3>Protein Purification (Day 2) and LDH assay preparation</H3><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday2" target="blank">TDP purification protocol (Day 2)</A> was carried out with cell lysates of cultures B1 and B5 prepared on 05-09-2017.</P><H4>LDH assay preparation</H4><H6>Gabriella, Guillermo</H6><P>Several buffers were prepared for the LDH assay the following day:</P></DIV></DIV></DIV></DIV></LI><LI>Tris/HCl (pH = 7), c = 2.5 M</LI><LI>Sodium phosphate dibasic, c = 1 M</LI><LI>Sodium phosphate monobasic, c = 1 M</LI><DIV class="V"><DIV class="row notebook vesicles" id="V20170906"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT, pET-Duet1 in KEIO and TolR-AU in KEIO was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in KEIO was induced at an OD of 0.35 and 0.45. TolR-AU in KEIO was induced at an OD of 0.45 and 0.40. Again, pET-Duet1 in WT didn't grow and was discarded. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight. The overnight cultures of TolR-AU in WT were purified as well
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV></DIV><LI><DIV class="collapsible-header">Thursday 7 September</DIV><DIV class="collapsible-body" id="d20170907"><DIV class="C"><DIV class="row notebook cas13" id="C20170907"><H4>Checking the crRNA's</H4><H6>Kasper</H6><P>The in vitro transcription mixtures of target 1-4 with and without hairpin were purified using the RNeasy MinElute kit. The concentrations were measured afterwards, namely:</P><TABLE><THEAD><TR><TH>crDNA target</TH><TH>A260</TH><TH>A260/A230</TH><TH>A260/A280</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>0.747</TD><TD>0.30</TD><TD>2.48</TD><TD>30.0</TD></TR><TR><TD>1H</TD><TD>0.943</TD><TD>0.70</TD><TD>02.17</TD><TD>37.8</TD></TR><TR><TD>2</TD><TD>0.903</TD><TD>0.45</TD><TD>2.25</TD><TD>36.1</TD></TR><TR><TD>2H</TD><TD>0.859</TD><TD>0.69</TD><TD>2.39</TD><TD>34.0</TD></TR><TR><TD>3</TD><TD>0.938</TD><TD>0.52</TD><TD>2.26</TD><TD>37.5</TD></TR><TR><TD>3H</TD><TD>1.000</TD><TD>0.61</TD><TD>2.15</TD><TD>40.0</TD></TR><TR><TD>4</TD><TD>0.710</TD><TD>0.12</TD><TD>2.22</TD><TD>28.4</TD></TR><TR><TD>4H</TD><TD>0.770</TD><TD>0.31</TD><TD>2.20</TD><TD>30.8</TD></TR><TR><TD>5</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>20.0</TD></TR><TR><TD>5H</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>19.0</TD></TR></TBODY></TABLE><P>3 µL of each sample was mixed with 3 µL of RNA loading dye and incubated at 70 °C for 10 minutes. Samples were quickly cooled on ice for 5 minutes and centrifuged and subsequently run on a 10% PAGE gel, stained with SYBR Gold (55 minutes, 333 V, 500 mA).</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170907"><H3>Sample prep RNA gel electrophoresis of RPA with in vitro transcription</H3><H6>Amária and Kasper</H6><P>
     We repeated the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol as on the 6<SUP>th</SUP> of September. We wanted to see if the smudge with sample 10<SUP>-5</SUP> was a result of faulty running the RNA gel. We used the samples RPAéd and in vitro transcribed on the 5<SUP>th</SUP> of September.The following picture shows the resulting RNA gel:
      </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170907"><H3>LDH assay (controls)</H3><H6>Gabriella, Guillermo</H6><P>Samples for an <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> were prepared for the proteins B3 and A5 in concentrations of 0.5, 1 and 1.5 g/L. The LDH assay was then carried out for the half of the samples that was not to be dried.</P><P>NB: Instead of 1 µL of LDH-TDP solution, as stated in the protocol, 2 µL were used. This was deemed to result in too high activity, which could no longer be observed well enough in the data generated by the platereader, so the amount was adjusted .</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170907"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.47 and 0.49. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction. The overnight cultures of TolR-AU in KEIO and pET-Duet1 in KEIO were purified as well.
    200 µL of all purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P><I>It turned out that pET-Duet1 in KEIO was inocculated instead of pET-Duet1 in WT.</I></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 8 September</DIV><DIV class="collapsible-body" id="d20170908"><DIV class="C"><DIV class="row notebook cas13" id="C20170908"><H4>Checking for collateral cleavage</H4><H6>Kasper</H6><P>To assess the quality of Cas13a (Zhang lab; purified with a Hisselect gravity column and buffer exchange) a fluorescence assay was done with RNase Alert. Increase of fluorescence of a mix containing RNase Alert is idicative of RNase activity. 
                    The following reaction buffer (10x RB) was prepared: 400 mM TRIS-HCl; 600 mM NaCl; 60 mM MgCl<SUB>2</SUB>; pH = 7.3. This buffer was used in sample preparation on a 96-wells plate. 10 samples were prepared as indicated in the table. <U>NOTE</U>: All values indicate volumes in µL. Cas13a denotes the protein mix purified with Hisselect and buffer exchange. This solution 
                (~0.2 mg/mL) was left at 4 °C for a week. TcR denotes the target RNA, using RPA and in vitro transcription. This target is a part of the tetracyclin antibiotic resistance gene from pSB1T3. crRNA is the 1H crRNA from 06-09-2017 and 07-09-2017. NF H<SUB>2</SUB>O is nuclease-free water.</P><TABLE><THEAD><TR><TH>Wells</TH><TH>10x RB</TH><TH>Cas13a</TH><TH>RI<SUP>*</SUP></TH><TH>TcR</TH><TH>crRNA</TH><TH>RNase Alert</TH><TH>NF H<SUB>2</SUB>O</TH></TR></THEAD><TBODY><TR><TD>A1 + A2</TD><TD>10</TD><TD>3.65</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>72.35</TD></TR><TR><TD>A3 + A4</TD><TD>10</TD><TD>36.5</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>39.5</TD></TR><TR><TD>A5 + A6</TD><TD>10</TD><TD>3.65</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>10</TD><TD>73.35</TD></TR><TR><TD>A7 + A8</TD><TD>10</TD><TD>36.5</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>10</TD><TD>40.5</TD></TR><TR><TD>A9 + A10</TD><TD>10</TD><TD>0</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>76.0</TD></TR></TBODY></TABLE><P>RI<SUP>*</SUP> is murine RNase inhibitor. 
                    The mixes were left in the platereader for 3&amp;nbs;hours, with measurements every 2 minutes and 1.5 minutes shaking.</P><P>The data was used as follows:</P></DIV></DIV></DIV></LI><LI>Blue line: Mean(A1,A2)-Mean(A9,A10)</LI><LI>Grey line: Mean(A3,A4)-Mean(A9,A10)</LI><LI>Orange line: Mean(A5,A6)-Mean(A9,A10)</LI><LI>Yellow line: Mean(A7,A8)-Mean(A9,A10)</LI><P>These results indicate that for the reaction with 3.65 µL Cas13a, the fluorescence increases faster with TcR target than without TcR target. This can be interpreted as collateral cleavage activation.</P><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170908"><H3>LDH assay (dried samples)</H3><H6>Gabriella, Guillermo</H6><P>An <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with the dried samples of concentrations 0.5, 1 and 1.5 g/L of proteins B3 and A5 prepared on 07-09-2017.</P><P>NB: Instead of 1 µL of LDH-TDP solution, as stated in the protocol, 2 µL were used. This was deemed to result in too high activity, which could no longer be observed well enough in the data generated by the platereader, so the amount was adjusted.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170908"><H3>Vesicles TEM</H3><H6>Floor</H6><P>The stored samples of TolR-AU in WT and TolR-AU in KEIO 5&amp;ndsp;hours after induction were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    However, the samples contained a lot of noise, making analysis impossible.</P><I>This might be because 0.2 µm filter was not used.</I></DIV></DIV></DIV></DIV><LI><DIV class="collapsible-header scrollspy" id="week37-scroll">Week 37</DIV></LI><LI><DIV class="collapsible-header">Monday 11 September</DIV><DIV class="collapsible-body" id="d20170911"><DIV class="C"><DIV class="row notebook cas13" id="C20170911"><H4>Induction of protein expression</H4><H6>Jeroen</H6><P>The OD600 of the starter culture was determined to be 1.1. Subsequently, 4.5 mL of the starter culture was inoculated in 2x1 L TB medium. Next, the cells were incubated at 37 °C, 180 rpm. After approximately six hours, the OD600s were around 0.56 and a cold-shock was applied. After the cold-shock, a 1 mL sample was taken from both cultures. The samples were stored at 4 °C. Afterwards, the cultures were induced with 1 mL 500 mM IPTG. The cultures were incubated overnight at 18 °C and 180 rpm.</P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 12 September</DIV><DIV class="collapsible-body" id="d20170912"><DIV class="C"><DIV class="row notebook cas13" id="C20170912"><H4>SDS-PAGE</H4><H6>Jeroen</H6><P>1 mL samples were taken from both overnight cultures. They were stored at 4 °C. The OD600 of the cultures was determined to be 1.80 and 1.72. They were centrifuged at 4 °C and 5200<SMALL> x </SMALL><I>g</I> for 15 minutes. Then the supernatant was discarded and resuspended into 40 mL of 1xPBS. This was put into falcon tubes (50 mL) and centrifuged at 3220<SMALL> x </SMALL><I>g</I> at 4 ° for 10 minutes. The supernatant was again discarded and pellets were stored at -80 °C.
                An <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspageprecast" target="_blank">SDS-PAGE gel</A> was run.</P><P>Lanes 1 and 6 contain a protein ladder; lanes 2 and 3 contain samples from culture one before and after induction; lanes 4 and 5 contain samples from culture two before and after induction. The conclusion is that there is no difference at the expected cas13a size between the samples taken before and after induction. However, one can see a slight difference at around 25 kDa. 
                                 <U>NOTE</U>: Later sequencing results showed two mutations in our construct, resulting in a frameshift and an early stopcodon. The product would be around 25.8 kDa, which corresponds to this gel. 
                                 It was concluded that we should first repair the mutations in the construct before setting up cultures with it again.</P><H4>Buffer preparation</H4><H6>Jeroen</H6><P>An elution and storage buffer were prepared. Their composition can be found in the full<A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol. Additionally, a washing buffer (20 mM Tris-HCl; 25 mM imidazole; pH 8.0) was prepared.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170912"><H3>Sample prep Liquid culture</H3><H6>Amária</H6><P>pSB1T3 in DH5-α was grown in 10 mL LB overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. The used antibiotics was tetracyclin.</P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170912"><H3>Vesicles TEM</H3><H6>Gabriella</H6><P>5 mL of pET-Duet1 in KEIO and TolR-AU in KEIO were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    pET-Duet1 in WT was induced at an OD of 0.65 and TolR-AU in KEIO at an OD of 0.55. Next, the samples were prepared for TEM analysis following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="_blank">Preparing sample for TEM</A> protocol and analysed with the TEM. 
    Pictures can be seen below.</P><P>TolR-AU in KEIO
    </P><P>pET-Duet in KEIO
    </P><P>You can see that vesicles are produced, regardless of whether or not the strain contains the TolR-insert.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 13 September</DIV><DIV class="collapsible-body" id="d20170913"><DIV class="C"><DIV class="row notebook cas13" id="C20170913"><H4>Column elution</H4><H6>Kasper</H6><P><U>NOTE</U>: for the following steps, cell pellet from 16-08-2017 was used. 
                                 The pellet was thawed under running tap water until completely defrosted. Next, 40 mL 10 mM TRIS-HCl was added with 1 tablet protease inhibitor. This was frozen again at -80 °C. The proces of thawing and freezing was then repeated twice. Subsequently, the pellet was French pressed three times at 1 kbar. Then the pellet was centrifuged (16000 rpm for 45 minutes at 4 °C). The resulting supernatant was collected and filtered with a 0.45 µm filter. 
                                 After Hisselect was cleaned three times in 10 mM TRIS-HCl, the supernatant was added and incubated for 30 minutes at 4 °C while rotating. This was then centrifuged and the supernatant was discarded. The Hisselect was transferred to a column and washed four times with the washing buffer. Next, the column was eluted five times with elution buffer. Fractions E2-E5 were dialysed in the storage buffer at 4 °C overnight.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170913"><H3>Sample prep DNA isolation</H3><H6>Amária</H6><P>With the overnight culture, a dilution range was prepared from -1 until and including -7, both in milk and in Mili-Q in a total volume of 4.5 mL. For -3, -5 and -7, each sample was divided into three different samples. 500 µL was prepared following the microwave method described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. 1 mL was prepared following the boiling method described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. Lastly, 1 mL was prepared with the Norgen Milk Bacterial DNA Isolation Kit following the protocol for Gram Positive bacteria. All samples were stored at -20 °C.</P><P>After the DNA isolation, all the dilutions (-1 uptill and including -7) were each plated out.</P><I>The overnight culture that was used didn't grow, so no results were obtained.</I></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170913"><H3>Bradford Assay and LDH assay preparation</H3><H4>Bradford Assay</H4><H6>Gabriella, Guillermo</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Notebook#bradfordassay" target="blank">Bradford Assay</A> of the proteins B1 (from 05-09-2017), B4 and B6 (from 11-08-2017) resuspended in 1 mL MQ was carried out to determine their concentration:</P></DIV></DIV></DIV></DIV></LI><LI>c[B1] = 3.24 g/L</LI><LI>c[B4] = 1.25 g/L</LI><LI>c[B6] = 0.91 g/L</LI><H4>LDH assay preparation</H4><H6>Gabriella, Guillermo</H6><P>Multiple samples of different concentrations were prepared for an LDH assay. Additionally, LDH was freeze-dried in a solution containing 1 µL of LDH, 1 µL of Tris/HCl and 98 µL of MQ.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>TDP</TH><TH>concentration [g/L]</TH><TH>V(TDP) [µL]</TH><TH>V(LDH) [µL]</TH><TH>V(Tris/HCl) [µL]</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1</TD><TD>0.1</TD><TD>3.1</TD><TD>1</TD><TD>1</TD><TD>94.9</TD></TR><TR><TD>0.5</TD><TD>15.5</TD><TD>1</TD><TD>1</TD><TD>82.5</TD></TR><TR><TD>1</TD><TD>31</TD><TD>1</TD><TD>1</TD><TD>67</TD></TR><TR><TD>1.5</TD><TD>46.3</TD><TD>1</TD><TD>1</TD><TD>51.7</TD></TR><TR><TD>3</TD><TD>92.6</TD><TD>1</TD><TD>1</TD><TD>5.4</TD></TR><TR><TD class="firstcolumn">B4</TD><TD>0.1</TD><TD>8</TD><TD>1</TD><TD>1</TD><TD>90</TD></TR><TR><TD>0.5</TD><TD>40</TD><TD>1</TD><TD>1</TD><TD>58</TD></TR><TR><TD>1</TD><TD>80</TD><TD>1</TD><TD>1</TD><TD>18</TD></TR><TR><TD>1.2</TD><TD>96</TD><TD>1</TD><TD>1</TD><TD>2</TD></TR><TR><TD class="firstcolumn">B6</TD><TD>0.1</TD><TD>11</TD><TD>1</TD><TD>1</TD><TD>87</TD></TR><TR><TD>0.5</TD><TD>56</TD><TD>1</TD><TD>1</TD><TD>42</TD></TR><TR><TD>1</TD><TD>97</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR></TBODY></TABLE><DIV class="V"><DIV class="row notebook vesicles" id="V20170913"><H3>Vesicles Platereader GFP in vesicles</H3><H6>Gabriella</H6><P>5 mL of GFP-TorA &amp; TolR-AU in KEIO and GFP-TorA in KEIO were prepared according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    250 µL of the prepared samples were measured in the platereader at an exc./em. of 475/510 nm.</P></DIV></DIV><LI><DIV class="collapsible-header">Thursday 14 September</DIV><DIV class="collapsible-body" id="d20170914"><DIV class="C"><DIV class="row notebook cas13" id="C20170914"><H4>SDS-PAGE</H4><H6>Kasper</H6><P>The dialysis samples were collected and an <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspageprecast" target="_blank">SDS-PAGE gel</A> was run.</P><P>The conclusion is that it seems Cas13a has been degraded. NaCl should be added to the buffer to prevent this. Additionally, a T7 promoter is considered to produce more protein.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170914"><H3>Sample prep Liquid culture</H3><H6>Amária</H6><P>pSB1T3 in DH5-α was grown in 10 mL LB overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. The used antibiotics was tetracyclin.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170914"><H3>LDH assay</H3><H6>Guillermo</H6><P>An <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with the dried and liquid samples prepared on 13-09-2017. Additionally, the activity of a freeze-dried sample was measured.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170914"><H3>Preparation DLS-Measurements</H3><H6>Floor</H6><P>Starter cultures were prepared for pET-Duet1 in WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170915"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.42 and 0.39. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170916"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>The overnight cultures of pET-Duet1 in WT were purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    200 µL of the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170919"><H3>Vesicles Transformations</H3><H6>Aafke</H6><P>TolR-AU in KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    pSB1C3, TDP #4 and TDP#6 were transformed into the cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170920"><H3>Vesicles Membrane staining</H3><H4>Transformations</H4><H6>Aafke and Gabriella</H6><P>TDP #4 was transformed into WT, GFP-TorA into TolR-AU in KEIO and GFP-TorA into KEIO.</P><H4>Membrane staining</H4><H6>Aafke and Gabriella</H6><P>The remainder of the samples that were used for the DLS-measurements, were prepared as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="_blank">Membrane staining</A> protocol. 
    The samples prepared 3 hours after induction and the overnight samples were used. Different dilutions of stained liposomes were measured first, to be able to create a calibration curve. Next, the stained sample were measured in the plate reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170921"><H3>Vesicles Transformations</H3><H6>Aafke and Gabriella</H6><P><I>Previously used GFP was not controlled by a promoter.</I> We decided to use GFP E0840 with constitutive promoter J23108 with moderate expression on pSB1C3, as the new control for the untagged GFP.
    GFP was transformed into TolR-AU in KEIO and into WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV><DIV><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 15 September</DIV><DIV class="collapsible-body" id="d20170915"><DIV class="C"><DIV class="row notebook cas13" id="C20170915"><H4>Bradford assay</H4><H6>Kasper</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Protocols#bradfordassay" target="_blank">Bradford assay</A> was carried out. This resulted in the following table:</P><TABLE><THEAD><TR><TH>Sample</TH><TH>M1</TH><TH>M2</TH><TH>M3</TH><TH>Mean</TH></TR></THEAD><TBODY><TR><TD>BSA 2 mg/mL</TD><TD>0.529</TD><TD>0.517</TD><TD>0.520</TD></TR><TR><TD>BSA 1 mg/mL</TD><TD>0.282</TD><TD>0.271</TD><TD>0.280</TD></TR><TR><TD>BSA 0.5 mg/mL</TD><TD>0.152</TD><TD>0.147</TD><TD>0.165</TD></TR><TR><TD>BSA 0.25 mg/mL</TD><TD>0.63</TD><TD>0.053</TD><TD>0.078</TD></TR><TR><TD>Raw</TD><TD>-0.002</TD><TD>-0.006</TD><TD>0.018</TD></TR><TR><TD>4x concentrated</TD><TD>0.006</TD><TD>-0.003</TD><TD>0.021</TD></TR><TR><TD>FT</TD><TD>-0.009</TD><TD>-0.0012</TD><TD>0.005</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170915"><H3>Vesicles DLS-Measurements</H3><H6>Floor</H6><P>15 mL of pET-Duet1 in WT was prepared for DLS-measurements following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The samples were induced at an OD of 0.42 and 0.39. The purification was performed 2 hours, 3 hours, 4 hours and 5 hours after induction.The remaining culture was kept to grow overnight.
    200 µL of all the purified samples were analysed with the DLS. The remainder was stored in the -20 freezer for later TEM-analysis.</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week38-scroll">Week 38</DIV></LI><LI><DIV class="collapsible-header">Monday 18 September</DIV><DIV class="collapsible-body" id="d20170918"><DIV class="C"><DIV class="row notebook cas13" id="C20170918"><H4>Annealing crDNA</H4><H6>Kasper</H6><P>Four tubes were filled with 8 µL of nuclease-free water and 1 µL of both IG0058 and IG0059 were added. All four tubes were annealed using the <A href="https://2017.igem.org/Team:TUDelft/Protocols#crannealing" target="_blank">crDNA annealing</A> protocol. All tubes were stored at -80 °C.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170918"><H3>Sample prep Transformation pSB1T3 in DH5-α and BL21</H3><H6>Amária</H6><P>As the bacteria are not really growing, pSB1T3 is again transformed into cells. Commercial competent cells were used, DH5-α and BL21. The transformation was performed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationDH5alpha" target="_blank">Transformation NEB competent cells</A> protocol.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 19 September</DIV><DIV class="collapsible-body" id="d20170919"><DIV class="C"><DIV class="row notebook cas13" id="C20170919"><H4>Making crRNA</H4><H6>Kasper</H6><P>One tube of annealed crDNA was thawn on ice and mixed as follows for the in vitro transcription: </P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD>crDNA</TD><TD>10</TD></TR><TR><TD>ATP</TD><TD>2</TD></TR><TR><TD>CTP</TD><TD>2</TD></TR><TR><TD>GTP</TD><TD>2</TD></TR><TR><TD>UTP</TD><TD>2</TD></TR><TR><TD>Murine RNase inhibitor</TD><TD>2</TD></TR><TR><TD>10x RNase polymerase buffer</TD><TD>10</TD></TR><TR><TD>T7 polymerase mix</TD><TD>2</TD></TR><TR><TD>Nuclease-free water</TD><TD>68</TD></TR></TBODY></TABLE><P>The sample was left at 37 °C for three hours. Next, the reaction mixture was cleaned up using the RNeasy MinElute kit. 
                            6 µL of the cleaned sample was then divided over three tubes. Either 1 µL of nuclease-free water, DNase I or RNase A was added to a tube. The tubes were then incubated for one hour at 37 °C. The samples were then run on a RNA gel and it was found that RNA was produced.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170919"><H3>Sample prep Liquid culture</H3><H6>Amária</H6><P>Both transformations were succesful. 5 colonies of each plate were inoculated in 10 mL LB each according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. The used antibiotics was tetracyclin.</P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20170919"><H3>Vesicles Transformations</H3><H6>Aafke</H6><P>TolR-AU in KEIO were made electrocompetent following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cellselectro" target="_blank">making electrocompetent cells</A> protocol. 
    pSB1C3, TDP #4 and TDP#6 were transformed into the cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170920"><H3>Vesicles Membrane staining</H3><H4>Transformations</H4><H6>Aafke and Gabriella</H6><P>TDP #4 was transformed into WT, GFP-TorA into TolR-AU in KEIO and GFP-TorA into KEIO.</P><H4>Membrane staining</H4><H6>Aafke and Gabriella</H6><P>The remainder of the samples that were used for the DLS-measurements, were prepared as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="_blank">Membrane staining</A> protocol. 
    The samples prepared 3 hours after induction and the overnight samples were used. Different dilutions of stained liposomes were measured first, to be able to create a calibration curve. Next, the stained sample were measured in the plate reader.</P></DIV><DIV><DIV class="row notebook vesicles" id="V20170921"><H3>Vesicles Transformations</H3><H6>Aafke and Gabriella</H6><P><I>Previously used GFP was not controlled by a promoter.</I> We decided to use GFP E0840 with constitutive promoter J23108 with moderate expression on pSB1C3, as the new control for the untagged GFP.
    GFP was transformed into TolR-AU in KEIO and into WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV><DIV><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 20 September</DIV><DIV class="collapsible-body" id="d20170920"><DIV class="C"><DIV class="row notebook cas13" id="C20170920"><H4>Buffer preparation and Hisselect column treatment</H4><H6>Kasper</H6><P>The lysis buffer from 19-09-2017 was supplemented with lysozyme, benzonase and protease inhibitor tablets from Roche, as explained in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank"> Cas13a purification</A> protocol. Additionally, 1 L storage buffer was prepared. This buffer consists of 50 mM TRIS-HCl; 600 mM NaCl; 5% glycerol and 2 mM of DTT. The pH was adjusted to 7.5. 
                         The pellet from 31-08-2017 was resuspended in 30 mL lysis buffer. Next, the French press was rinced with lysine buffer (from 19-09-2017, without lysozyme, benzonase and protein inhibitor) at 1 kbar. The resuspended pellet was french pressed three times at 1 kbar. The resulting slution was centrifuged in the ultacentrifuge (16000 rpm, 45 minutes at 4 °C). Following this, the supernatant was filtered with a 0.45 µm filter. At the same time, 1 mL Hisselect resin (from Stan Brouns lab) was washed three times with the unsupplemented lysis buffer and spinned down for 1 minute at 13500 rpm. <U>NOTE</U>: This should be done at around 2000 rcf. 
                         The Hisselect was added to the filtered supernatant and was incubated for 30 minutes at 4 °C while rotating. The resulting solution was spinned down for 1 minute at 2000 rcf and the supernatant was topped off and stored at 4 °C. 
                         Unsupplemented lysis buffer was added to the column until no air was left in the column. Then the filter was pressed down. <U>NOTE</U>: All following steps are done at 4 °C.
                         The loaded Hisselect is added to the column and washed five times with 2 mL unsupplemented lysis buffer. Next, the Hisselect is eluted five times with 300 µL elution buffer. This lead to fractions with the following Nanodrop values:</P><TABLE><THEAD><TR><TH>Fraction</TH><TH>Concentration</TH><TH>260/280</TH><TH>A280</TH></TR></THEAD><TBODY><TR><TD>E1</TD><TD>0.30</TD><TD>0.27</TD><TD>0.201</TD></TR><TR><TD>E2</TD><TD>2.58</TD><TD>1.38</TD><TD>1.719</TD></TR><TR><TD>E3</TD><TD>2.40</TD><TD>1.54</TD><TD>1.6</TD></TR><TR><TD>E4</TD><TD>1.35</TD><TD>1.81</TD><TD>0.897</TD></TR><TR><TD>E5</TD><TD>0.69</TD><TD>2.17</TD><TD>0.459</TD></TR></TBODY></TABLE><P>280 µL of E2-E5 was put through dialysis at 4 °C and 100 rpm overnight, in storage buffer.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170920"><H3>Sample prep Liquid culture</H3><H6>Amária</H6><P>Again, the overnight cultures didn't grow. This could be because the Lac promoter is causing RFP to be overexpressed. This time the liquid culture was prepared with 1% glucose, to repress the Lac promoter. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol)</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170920"><H3>Bradford Assay and LDH assay preparation</H3><H4>Bradford Assay</H4><H6>Isabell, Guillermo</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Notebook#bradfordassay" target="blank">Bradford Assay</A> of the proteins B1 (from 09-09-2017), B4 and B6 (from 08-09-2017) resuspended in 1 mL MQ was carried out to determine their concentration:</P></DIV></DIV></DIV></DIV></LI><LI>c[B1] = 3.36 g/L</LI><LI>c[B5] = 0.96 g/L</LI><H4>LDH assay preparation</H4><H6>Guillermo, Isabell</H6><P>Multiple samples of different concentrations were prepared for a longterm LDH assay. Volumes of 300 µL were prepared and split into 15 aliquots of 20 µL, that would be measured over the coming four weeks. Five aliquots were stored in the fridge at 4 °C in liquid form (referred to as L4), five aliquots were dried and stored in the fridge at 4 °C (referred to as D4) and five aliquots were stored at room temperature (referred to as D).</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>TDP</TH><TH>concentration [g/L]</TH><TH>V(TDP) [µL]</TH><TH>V(LDH) [µL]</TH><TH>V(Tris/HCl) [µL]</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1</TD><TD>0.5</TD><TD>45</TD><TD>3</TD><TD>3</TD><TD>249</TD></TR><TR><TD class="firstcolumn">B1</TD><TD>1</TD><TD>90</TD><TD>3</TD><TD>3</TD><TD>204</TD></TR><TR><TD class="firstcolumn">B5</TD><TD>0.5</TD><TD>150</TD><TD>3</TD><TD>3</TD><TD>144</TD></TR><TR><TD class="firstcolumn">B5</TD><TD>1</TD><TD>294</TD><TD>3</TD><TD>3</TD><TD>0</TD></TR></TBODY></TABLE><DIV class="V"><DIV class="row notebook vesicles" id="V20170920"><H3>Vesicles Membrane staining</H3><H4>Transformations</H4><H6>Aafke and Gabriella</H6><P>TDP #4 was transformed into WT, GFP-TorA into TolR-AU in KEIO and GFP-TorA into KEIO.</P><H4>Membrane staining</H4><H6>Aafke and Gabriella</H6><P>The remainder of the samples that were used for the DLS-measurements, were prepared as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="_blank">Membrane staining</A> protocol. 
    The samples prepared 3 hours after induction and the overnight samples were used. Different dilutions of stained liposomes were measured first, to be able to create a calibration curve. Next, the stained sample were measured in the plate reader.</P></DIV></DIV><LI><DIV class="collapsible-header">Thursday 21 September</DIV><DIV class="collapsible-body" id="d20170921"><DIV class="C"><DIV class="row notebook cas13" id="C20170921"><H4>SDS-PAGE</H4><H6>Kasper</H6><P>The solution was pipetted out of the dialysis tubing into a 2 mL tube. Subsequently, an <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspageprecast" target="_blank">SDS-PAGE gel</A> was run.</P><P>The conclusion is that Cas13a is present in large quantities (the fattest band in all lanes). However, it is not pure.</P><H4>Bradford assay</H4><H6>Kasper</H6><P>A Bradford assay with a dilution series of BSA and with Cas13a samples was performed. UV-Vis at 595 nm with a 1 mm cuvette was used. The BSA samples were used to create a calibration curve from which to read the Cas13a concentration in the samples. </P><TABLE><THEAD><TR><TH>Sample</TH><TH>M1</TH><TH>M2</TH><TH>M3</TH><TH>Mean</TH></TR></THEAD><TBODY><TR><TD>BSA 2 mg/mL</TD><TD>0.542</TD><TD>0.54</TD><TD>0.545</TD><TD>0.54233</TD></TR><TR><TD>BSA 1 mg/mL</TD><TD>0.291</TD><TD>0.291</TD><TD>0.291</TD><TD>0.291</TD></TR><TR><TD>BSA 0.5 mg/mL</TD><TD>0.173</TD><TD>0.174</TD><TD>0.176</TD><TD>0.17433</TD></TR><TR><TD>BSA 0.25 mg/mL</TD><TD>0.0.77</TD><TD>0.075</TD><TD>0.076</TD><TD>0.076</TD></TR><TR><TD>Cas13a S1</TD><TD>0.165</TD><TD>0.165</TD><TD>0.165</TD><TD>0.165</TD></TR><TR><TD>Cas13a S1</TD><TD>0.161</TD><TD>0.161</TD><TD>0.161</TD><TD>0.161</TD></TR></TBODY></TABLE><P>From the assay and the calibration curve, the protein concentration in the cas13a samples was determined to be approximately 0.5 mg/mL. There was roughly 0.4 mL of Cas13a solution left, which was aliquotted (10 µ/tube), snapfreezed and stored at -80 °C.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170921"><H3>Sample prep plating dilutions</H3><H6>Amária</H6><P>This time, the cells grew. To be able to know the number of cells in the culture, dilutions were prepared fom -3 until and including -6. 100 µL of each dilution was plated and incubated at 37 °C overnight.
    </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170921"><H3>LDH assay (Longterm Day 1)</H3><H6>Guillermo, Isabell</H6><P>The first iteration of the longterm <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with an L4, D4 and D sample of both TDPs (B1 and B5) prepared on 20-09-2017.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170921"><H3>Vesicles Transformations</H3><H6>Aafke and Gabriella</H6><P><I>Previously used GFP was not controlled by a promoter.</I> We decided to use GFP E0840 with constitutive promoter J23108 with moderate expression on pSB1C3, as the new control for the untagged GFP.
    GFP was transformed into TolR-AU in KEIO and into WT following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationelectro" target="_blank">transformation of electrocompetent cells</A> protocol.</P><DIV><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV><DIV><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 22 September</DIV><DIV class="collapsible-body" id="d20170922"><DIV class="C"><DIV class="row notebook cas13" id="C20170922"><H4>Checking Cas13a for collateral cleavage</H4><H6>Kasper</H6><P>The new batch of Cas13a (purified 20-09-2017 and 21-09-2017) needed to be checked for collateral cleavage activity. The concentration of Cas13a in the aliquots was calculated to be 0.5 g/L÷146.47 g/mol = 3.4 µM. 50 nM Cas13a would be required for the assay, so 1.5 µL was needed. For the crRNA, ~20 nM was used. The sample preparation was again done in a 96-wells plate, according to the table below. 
                            TcR denotes the target RNA, using RPA and in vitro transcription. This target is a part of the tetracyclin antibiotic resistance gene from pSB1T3. crRNA is the 1H crRNA from 06-09-2017 and 07-09-2017. NF H<SUB>2</SUB>O is nuclease-free water.</P><TABLE><THEAD><TR><TH>Wells</TH><TH>10x RB</TH><TH>Cas13a</TH><TH>RI<SUP>*</SUP></TH><TH>TcR</TH><TH>crRNA</TH><TH>RNase Alert</TH><TH>NF H<SUB>2</SUB>O</TH></TR></THEAD><TBODY><TR><TD>A1 + A2</TD><TD>10</TD><TD>1.5</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>74.5</TD></TR><TR><TD>A3 + A4</TD><TD>10</TD><TD>1.5</TD><TD>2</TD><TD>1</TD><TD>0</TD><TD>10</TD><TD>75.5</TD></TR><TR><TD>A5 + A6</TD><TD>10</TD><TD>1.5</TD><TD>2</TD><TD>0</TD><TD>1</TD><TD>10</TD><TD>75.5</TD></TR><TR><TD>A7 + A8</TD><TD>10</TD><TD>1.5</TD><TD>2</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>76.5</TD></TR><TR><TD>A9 + A10</TD><TD>10</TD><TD>0</TD><TD>2</TD><TD>0</TD><TD>0</TD><TD>10</TD><TD>78.0</TD></TR></TBODY></TABLE><P>RI<SUP>*</SUP>This is murine RNase inhibitor. 
                    The mixes were left in the platereader for 3&amp;nbs;hours, with measurements every 2 minutes and 1.5 minutes shaking.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170922"><H3>Sample prep DNA isolation</H3><H6>Amária</H6><P>22 colonies grew on the -6 dilution plate. 5 µL was used for the teardrop assay, meaning that ~22 x 10^7 cells will be in the culture of 1 mL and ~11 x 10^7 in the culture of 500 µL. The dilutions that were prepared yesterday were used to isolate DNA from. Each of the dilutions was prepared following different methods:
    
            <TABLE class="notebook table table-style-1"><THEAD><TR><TH>Volume(µL)</TH><TH>Dilutions prepared in</TH><TH>Proteinase K</TH><TH>Protocol</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">1000</TD><TD>Milli-Q</TD><TD>No</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Boiling method</A></TD></TR><TR><TD class="firstcolumn">1000</TD><TD>Milli-Q</TD><TD>2 µL</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Boiling method</A></TD></TR><TR><TD class="firstcolumn">1000</TD><TD>Milk</TD><TD>No</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Boiling method</A></TD></TR><TR><TD class="firstcolumn">1000</TD><TD>Milk</TD><TD>2 µL</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Boiling method</A></TD></TR><TR><TD class="firstcolumn">500</TD><TD>Milli-Q</TD><TD>No</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Microwave method</A></TD></TR><TR><TD class="firstcolumn">500</TD><TD>Milli-Q</TD><TD>1 µL</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Microwave method</A></TD></TR><TR><TD class="firstcolumn">500</TD><TD>Milk</TD><TD>No</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Microwave method</A></TD></TR><TR><TD class="firstcolumn">500</TD><TD>Milk</TD><TD>1 µL</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">Microwave method</A></TD></TR><TR><TD class="firstcolumn">1000</TD><TD>Milk</TD><TD>No</TD><TD><A href="https://2017.igem.org/Team:TUDelft/Protocols#" target="_blank">Milk isolation kit</A></TD></TR></TBODY></TABLE></P><P> All samples were stored at -20 °C.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week39-scroll">Week 39</DIV></LI><LI><DIV class="collapsible-header">Monday 25 September</DIV><DIV class="collapsible-body" id="d20170925"><DIV class="C"><DIV class="row notebook cas13" id="C20170925"><H4>New annealing protocol</H4><H6>Kasper</H6><P>A new annealing protocol was tried to anneal the crDNA with the heating block and annealing buffer. crDNA 1H was made by mixing 2.5 µL of IG0057 and IG0058 with 5 µL annealing buffer (10 mM Tris-HCl pH 7.5; 50 mM NaCl; 1 mM EDTA). This was incubated for 5 minutes at 95 °C and subsequently slowly cooled to room temperature. 
                 In vitro transcription was done as follows:</P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD>crDNA</TD><TD>10</TD></TR><TR><TD>ATP</TD><TD>2</TD></TR><TR><TD>CTP</TD><TD>2</TD></TR><TR><TD>GTP</TD><TD>2</TD></TR><TR><TD>UTP</TD><TD>2</TD></TR><TR><TD>Murine RNase inhibitor</TD><TD>2</TD></TR><TR><TD>10x RNase polymerase buffer</TD><TD>10</TD></TR><TR><TD>T7 polymerase mix</TD><TD>2</TD></TR><TR><TD>Nuclease-free water</TD><TD>68</TD></TR></TBODY></TABLE><P>The sample was left at 37 °C for three hours. Next, the mixture was incubated at the same temperature for another hour, after addition of 1 µL DNase I. 
                 Following this, the reaction mixture was cleaned with the RNeasy MinElute Clean-up kit. A concentration of 32.33 ng/µL was found using the Nanodrop, which is comparable to the concentrations obtained with the previous method.</P><DIV class="row notebook cas13" id="C20171004"><H4>Making crRNA</H4><H6>Kasper</H6><P>Three tubes of 1.5 mL were prepared with each containing 10 µL annealing buffer (10 mM Tris-HCl pH 7.5; 50 mM NaCl; 1 mM EDTA), 5 µL IGOO57 and 5 µL IGOO58. The DNA was annealed by heating at 95 °C in the heatblock and slowly cooling down to room temperature.</P></DIV><DIV class="row notebook cas13" id="C20171005"><H4>In vitro transcription</H4><H6>Kasper</H6><P>The following sample was prepared:</P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD>crDNA</TD><TD>20</TD></TR><TR><TD>ATP</TD><TD>4</TD></TR><TR><TD>CTP</TD><TD>4</TD></TR><TR><TD>GTP</TD><TD>4</TD></TR><TR><TD>UTP</TD><TD>4</TD></TR><TR><TD>Murine RNase inhibitor</TD><TD>4</TD></TR><TR><TD>10x RNase polymerase buffer</TD><TD>20</TD></TR><TR><TD>T7 polymerase mix</TD><TD>4</TD></TR><TR><TD>Nuclease-free water</TD><TD>136</TD></TR></TBODY></TABLE><P>The sample was left at 37 °C for three hours. Next, the mixture was incubated at the same temperature for another hour, after addition of 1 µL DNase I. 
                 Following this, the reaction mixture was cleaned with the RNeasy MinElute Clean-up kit. A concentration of ~13 ng/µL was found using the Nanodrop, which is relatively low.</P></DIV><DIV class="row notebook cas13" id="C20171009"><H4>Testing other crRNA</H4><H6>Kasper</H6><P>To test the other crRNAs (2, 3 and 4) a platereader experiment with RNase Alert was carried out. For this purpose, wells B1/B2, B3/B4 and B5/B6 were filled with 1 µL crRNA 2, 3 and 4, respectively. Additionally, 1 µL of target (TcR) was added. Furthermore, 1 µL Cas13a, 10 µL of 1250 nM RNase Alert and 10 µL of 1x Reaction buffer was added. A control was made containing everything except TcR and crRNA.</P><P>As can be derived from the figure, crRNA 3 was found to be the best one.</P></DIV></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20170925"><H3>Making coacervates and Cas13a compatible</H3><H6>Kasper</H6><P>To test the compatibility of coacervates (0.1 wt% polyU, 1 wt% spermine) with the reaction buffer, of Cas13a, referred to as 1xRb (40 mM Tris-HCl, 60 mM NaCl, 6 mM MgCl<SUB>2</SUB>, pH 7.3), the following mixes were prepared in 50 µL.</P><TABLE class="notebook table table-style-1"><TBODY><TR><TD class="firstcolumn">A</TD><TD>B</TD></TR><TR><TD class="firstcolumn">1xRb</TD><TD>1xRb</TD></TR><TR><TD class="firstcolumn">0.1 wt% polyU</TD><TD> 0.1 wt% polyU</TD></TR><TR><TD class="firstcolumn">1.0 wt% spermine</TD><TD> 1.0 wt% spermine</TD></TR><TR><TD class="firstcolumn"> -</TD><TD>0.05 wt% RNase A</TD></TR><TR><TD class="firstcolumn"> Nuclease free water</TD><TD> Nuclease free water</TD></TR></TBODY></TABLE><P>Both were incubated at 37 °C for 10 minutes. Tube A did show coacervation, whereas B did not. This means that coacervates are compatible with 1xRb.</P><P> Given that coacervates can form in the reaction buffer of Cas13a, it was directly tested whether the coacervation method worked, by preparing two tubes of which only one with crRNA 1, and both containing 1xRb, Cas13a, target RNA, Murine RNase inhibitor, spermine and polyU.</P><TABLE class="notebook table table-style-1"><TBODY><TR><TD class="firstcolumn">C</TD><TD>D</TD></TR><TR><TD class="firstcolumn">1xRb</TD><TD>1xRb</TD></TR><TR><TD class="firstcolumn">0.1 wt% polyU</TD><TD> 0.1 wt% polyU</TD></TR><TR><TD class="firstcolumn">1.0 wt% spermine</TD><TD> 1.0 wt% spermine</TD></TR><TR><TD class="firstcolumn">~50 nM Cas13a*</TD><TD> ~50 nM Cas13a*</TD></TR><TR><TD class="firstcolumn"> 0.3 ng/µL TcR target RNA</TD><TD>0.3 ng/µL TcR target RNA</TD></TR><TR><TD class="firstcolumn"> 0.3 ng/µL crRNA 1</TD><TD>-</TD></TR><TR><TD class="firstcolumn"> 1 µL Murine RNase inhibitor</TD><TD>1 µL Murine RNase inhibitor</TD></TR><TR><TD class="firstcolumn"> Nuclease free water</TD><TD> Nuclease free water</TD></TR></TBODY></TABLE><P>*The stock concentration of Cas13a (~0.05 wt%) was measured using a Bradford assay, but no correction was made for impurity of the protein. Therefore the concentration is only a rough estimation.</P><P>Both tubes (C &amp; D) were incubated for 10 minutes at 37 °C. Both showed coacervation and kept showing coacervation for over 5 hours. Taking into account that the lifetime of the active Cas13a is roughly 3 hours at 37 °C and given buffer conditions, this is an indication that Cas13a does not cleave the polyU in the coacervate reaction mixture. </P></DIV></DIV><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170925"><H3>LDH assay (longterm Day 4)</H3><H6>Guillermo, Isabell</H6><P>The second iteration of the longterm <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with an L4, D4 and D sample of both TDPs (B1 and B5) prepared on 20-09-2017.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170925"><H3>Vesicles Preparation experiments</H3><H4>Preparation growth curve experiment</H4><H6>Aafke</H6><P>Starter cultures were made according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">Liquid (starter) culture (10 mL)</A> protocol for the following cells.</P> 
    -TolR-AU in WT
    -TolR-AU in KEIO
    -pET-Duet1 in WT
    -pET-Duet1 in KEIO<H4>Preparation GFP in vesicles experiment</H4><H6>Aafke</H6><P><I>We decided to compare the whole insert with the empty backbone.</I>The following glycerol stocks were streaked on a plate.</P>
    -GFP-TorA in WT
    -GFP-TorA in KEIO
    -GFP-TorA and TolR-AU in KEIO
    -pSB1C3 in WT
    -pSB1C3 in KEIO
    -pSB1C3 and TolR-AU in KEIO</DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 26 September</DIV><DIV class="collapsible-body" id="d20170926"><DIV class="C"><DIV class="row notebook cas13" id="C20170926"><H3>Cas13a Start second plan: repairing mutations</H3><H6>Fiona</H6><P>Since the first plan had failed and the primers for the second plan (repairing the mutations in plasmid 'J') had finally arrived, our plan B was started with PCR amplification of J with correcting primers. For this purpose, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was followed with primers IG0069 and IG0070. A 7.7 kb product would be expected and was indeed obtained, as can be seen in the agarose gel picture below.</P><P>The PCR product from the reaction with DMSO was <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank"> cleaned and concentrated</A>, leading to a concentration of 271 ng/µL. The sample was split in two; one sample was phosphorylated. The other sample would be used as a negative control in the subsequent blunt-end ligation step. For the phosphorylation, the following sample was prepared:</P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD>Ligase buffer</TD><TD>2.5</TD></TR><TR><TD>ATP</TD><TD>2.5</TD></TR><TR><TD>T4 PNK</TD><TD>1</TD></TR><TR><TD>DNA</TD><TD>2.5</TD></TR><TR><TD>Milli-Q</TD><TD>16.5</TD></TR><TR><TD>Total</TD><TD>25</TD></TR></TBODY></TABLE><P>The sample was incubated for 30 minutes at 37 °C. Then the T4 PNK was heat inactivated at 65 °C for 20 minutes. 1 µL of ligase was then added, after which the sample was left to incubate on the bench for two hours. The negative control (non-phosphorylated PCR fragment) was taken along for the ligation.  After the ligation, the sample was treated with <A href="https://2017.igem.org/Team:TUDelft/Protocols#dpn1digestion" target="_blank">DpnI</A> and subsequently <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank"> cleaned and concentrated.</A> Heat-shock competent commercial BL21 cells were<A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformed</A> with cleaned ligation mixture. </P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170926"><H3>Sample prep RPA and in vitro transcription</H3><H6>Amária</H6><P>The samples containing the DNA were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. Next, the RNA was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were prepared and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. We incubated the samples for 10 minutes at 70 °C and left them 5 minutes on ice. Next, 4 µL of all the samples was transferred to the slots. The gel can be seen in the following pictures.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170926"><H3>Transformation into BL21 (DE3) and Preparation of Starter Cultures for Protein Purification</H3><H4>Transformation into BL21 (DE3)</H4><H6>Guillermo</H6><P>Chemically competent E.coli BL21(DE3) cells from New England Biolabs (C2527H) were transformed with the miniprepped plasmid of an empty backbone, according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#transformationchemical" target="_blank">transformation</A> protocol as a control.</P><P>NB: Transformation did not yield colonies.</P><H4>Preparation of Starter Cultures for Protein Purification</H4><H6>Guillermo</H6><P>Starter cultures of colonies B1, B3 (2x) and B5 (2x) were incubated in a shaking incubator at 37 °C and 300 rpm for protein purification.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170926"><H3>Vesicles Preparation GFP in vesicles experiment</H3><H6>Aafke</H6><P>The following cells were inoculated from the plate in LB and prepared following the first steps in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol.
    pSB1C3 and TolR-AU in KEIO didn't grow and was discarded.</P><DIV><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="2"> OD at induction</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>0.56</TD><TD>0.49</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>0.48</TD><TD>0.37</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>0.43</TD><TD>0.42</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>0.38</TD><TD>0.43</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>0.43</TD><TD>0.69</TD></TR></TBODY></TABLE><P><I>To be able the number of viable cells of WT and KEIO at a certain OD, TolR-AU in KEIO and in WT were checked with a teardrop assay described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. 
    <I>We decided to discard our experiments planned with TDP #4.</I></I></P><DIV><DIV class="row notebook vesicles" id="V20170928"><H3><I>Vesicles Growth curves</I></H3><H6><I>Aafke</I></H6><P><I>The number of viable cells appeared to be in the same range for both the KEIO as the WT.</I></P><P><I>TolR-AU in KEIO and in WT and pET-Duet1 in KEIO and in WT were grown in dubplo in 5 mL LB-medium for 6 hours. Next, 20 µL of cells were transferred to 230 µL LB-medium, with technical duplicates.
    The growth was monitored with the platereader till an OD between 0.3-0.7. Half of the cells was induced with 10 µL of 2.6 mM IPTG to a final concentration of 0.1 mM. The growth was monitored overnight with the platereader.</I></P></DIV></DIV><TABLE class="notebook table table-style-1"><THEAD><TR><TH colspan="4"> OD before purification</TH></TR></THEAD><THEAD><TR><TH>Cells</TH><TH>1</TH><TH>1 ind.</TH><TH>2</TH><TH>2 ind.</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GFP-TorA in WT</TD><TD>4.1</TD><TD>4.5</TD><TD>3.7</TD><TD>3.8</TD></TR><TR><TD class="firstcolumn">GFP-TorA in KEIO</TD><TD>3.9</TD><TD>3.2</TD><TD>3.4</TD><TD>3.2</TD></TR><TR><TD class="firstcolumn">GFP-TorA and TolR-AU in KEIO</TD><TD>3.2</TD><TD>3.3</TD><TD>2.8</TD><TD>2.9</TD></TR><TR><TD class="firstcolumn">pSB1C3 in WT</TD><TD>4.5</TD><TD>5.1</TD><TD>4.0</TD><TD>5.0</TD></TR><TR><TD class="firstcolumn">pSB1C3 in KEIO</TD><TD>3.4</TD><TD>3.7</TD><TD>3.4</TD><TD>3.7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 27 September</DIV><DIV class="collapsible-body" id="d20170927"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20170927"><H3>Protein Purification (Day 0) and LDH Assay (longterm2) Preparation</H3><H4>Protein Purification (Day 0)</H4><H6>Guillermo</H6><P>The overnight cultures of colonies B1, B3 (2x) and B5 (2x) from 26-09-2017 were used to induce one or two 1 L culture each according to the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A>.</P><H4>LDH Assay (longterm) Preparation</H4><H6>Guillermo, Isabell</H6><P>Multiple samples of different concentrations were prepared for additional data for the previously started longterm LDH assay. Volumes of 100 µL were prepared and split into 12 aliquots of 8 µL, that would be measured over the coming three weeks. Four aliquots were stored in the fridge at 4 °C in liquid form (referred to as L4), four aliquots were dried and stored in the fridge at 4 °C (referred to as D4) and four aliquots were stored at room temperature (referred to as D). Addtionally, a 100 µL solution of LDH with 1 µL of Tris/HCl was lyophilized in aliquots of 25 µL to serve as a control in the assay.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>TDP</TH><TH>concentration [g/L]</TH><TH>V(TDP) [µL]</TH><TH>V(LDH) [µL]</TH><TH>V(Tris/HCl) [µL]</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1</TD><TD>0.1</TD><TD>3</TD><TD>1</TD><TD>1</TD><TD>95</TD></TR><TR><TD class="firstcolumn">B5</TD><TD>0.1</TD><TD>10</TD><TD>1</TD><TD>1</TD><TD>88</TD></TR><TR><TD class="firstcolumn">None</TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>1</TD><TD>98</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20170927"><H3>Vesicles GFP in vesicles experiment</H3><H6>Aafke and Gabriella</H6><P>The samples were further purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="_blank">purification of vesicles</A> protocol. 
    The OD was measured before purification to be able to normalize the measurements. <I>The OD was measured of 10 x diluted samples, to be able to measure it accurately. The OD displayed in the table is the measured OD times 10.</I></P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 28 September</DIV><DIV class="collapsible-body" id="d20170928"><DIV class="C"><DIV class="row notebook cas13" id="C20170928"><H3>Cas13a New ligation and transformation</H3><H6>Jasper</H6><P>Due to a lack of colonies from the previous transformation, all the steps (starting from the phosphorylation) were repeated again. This time two variations of the phosphorylation were tried out: one with CutSmart buffer and one with PNK buffer. Furthermore, the samples were now <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank"> cleaned and concentrated </A> before the ligation as well. Nevertheless, these transformations did not results in colonies either. At this point, taking into account the time left for the project, it was decided to not focus on obtaining the full construct anymore. Instead, we focussed on obtaining basic parts, so other teams might use them successfully to create the composite part.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170928"><H3>Sample prep Liquid culture</H3><H6>Amária</H6><P>pSB1T3 in DH5-α was grown in 10 mL LB overnight according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#liquidculture10mL" target="_blank">liquid (starter) culture (10 mL)</A> protocol, in a 50 mL tube. The used antibiotics was tetracyclin and 0.5 mL of 20% glucose solution was added to get a 1% solution. </P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170928"><H3>Protein Purification (Day 1) and LDH Assay (longterm2 Day 1)</H3><H4>Protein Purification (Day 1)</H4><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out with cell pellets of culture B1, B3 (2x) and B5 (2x) prepared on 27-09-2017.</P><H4>LDH Assay (longterm2 Day1)</H4><H6>Guillermo, Isabell</H6><P>The first iteration of the longterm2 <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with an L4, D4 and D sample of both TDPs (B1 and B5) at concentration 0.1 g/L, as well as a sample of LDH dried without TDPs and LDH freeze-dried prepared on 27-09-2017.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 29 September</DIV><DIV class="collapsible-body" id="d20170929"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20170929"><H3>Sample prep DNA isolation</H3><H6>Aafke and Amária</H6><P>The overnight culture was analysed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. The measured OD was 3.9. The dilutions were prepared in 2 mL commercial milk ranging from -3 to -9. Next, the samples were prepared following the boiling method of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20170929"><H3>Protein Purification (Day 2)</H3><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday2" target="blank">TDP purification protocol (Day 2)</A> was carried out with cell lysates of culture B1, B3 (2x) and B5 (2x) prepared on 28-09-2017.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week40-scroll">Week 40</DIV></LI><LI><DIV class="collapsible-header">Monday 2 October</DIV><DIV class="collapsible-body" id="d20171002"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171002"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>The samples containing the DNA were transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. Per sample, two different approaches were used: incubated 3 hours at 37 °C according to protocol and 1 hour at roomtemperature. Next, the RNA was purified according to the<A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were prepared and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. We incubated the samples for 10 minutes at 70 °C and left them 5 minutes on ice. Next, 4 µL of all the samples was transferred to the slots. The gel can be seen in the following picture.</P><P>The marker that we used is not suitable for the size of the RNA-fragments we were expecting.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171002"><H3>LDH Assay (longterm Day 11, longterm2 Day 5)</H3><H6>Guillermo, Isabell</H6><P>The third iteration of the longterm and the second iteration of the longterm2 <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with an L4, D4 and D sample of both TDPs (B1 and B5) at concentration 0.5 and 1 g/L prepared on 20-09-2017, as well as samples of B1 and B5 at concentration 0.1 g/L, LDH dried without TDPs and LDH freeze-dried prepared on 27-09-2017.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 3 October</DIV><DIV class="collapsible-body" id="d20171003"><DIV class="C"><DIV class="D"><DIV class="row notebook detection" id="D20171003"><H3>Cas13a titration with coacervates</H3><H6>Kasper</H6><P> To determine the upper limit up to which addition of Cas13a still leads to coacervation of polyU and spermine, six tubes were prepared of 50 µL, containing 0.1 wt% polyU and 1.0 wt% spermine in 1xRb and nuclease free water. All tubes were incubated for 10 minutes at 37 °C, and subsequently varying concentrations of Cas13a were added to to final concentrations of 100, 10, 1, 0.1, 0.01 and 0 nM. Coacervation was only observed if the Cas13a concentration was lower than 100 nM, and in presence of 10 nM Cas13a, only slightly higher turbidity was observed.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20171003"><H3>Sample prep Gel electrophoresis</H3><H6>Aafke and Amária</H6><P>The samples were prepared again and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. This time the Low range ssRNA marker was used and all the samples were incubated for 10 minutes at 70 °C and for 5 minutes on ice. The gel can be seen in the following picture.</P><P>The marker seems very vague. We decided to try it again, but this time following the protocol described for the marker. We also prepared -3, -2 and -1 dilutions of the ladder to check if the marker was degraded or not.</P><P>The samples were prepared again and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 2 µL of ladder was added to 38 µL of 2X RNA loading dye. Next the dilutions for the marker were prepared and all the samples (included the RNA-samples) were incubated for 5 minutes at 90 °C and for 2 minutes on ice. Next, 4 µL of all the samples was transferred to the slots. The gel can be seen in the following picture.</P><P>Based on this gel we can conclude that the ladder is not degraded. Also, this new protocol seems to work best.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171003"><H3>Preparation of Starter Cultures for Protein Purification</H3><H6>Guillermo</H6><P>Starter cultures of colonies B1, B3, B4, B5 and B6 (2x) were incubated in a shaking incubator at 37 °C and 300 rpm for protein purification.</P><P>NB: B3 did not grow, so a new overnight culture was started the next day.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 4 October</DIV><DIV class="collapsible-body" id="d20171004"><DIV class="C"><DIV class="row notebook cas13" id="C20171004"><H4>Making crRNA</H4><H6>Kasper</H6><P>Three tubes of 1.5 mL were prepared with each containing 10 µL annealing buffer (10 mM Tris-HCl pH 7.5; 50 mM NaCl; 1 mM EDTA), 5 µL IGOO57 and 5 µL IGOO58. The DNA was annealed by heating at 95 °C in the heatblock and slowly cooling down to room temperature.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20171004"><H6>Kasper</H6><P>To check if the coacervate based detection method works if the Cas13a is activated before polyU and spermine are added, two solutions were prepared of 50 µL, containing 20 nM Cas13a, 0.3 ng/µL, 1xRb. To one of the tubes, also 0.3 ng/µL crRNA 1 was added. Both tubes were incubated at 37 °C for 1 hour to activate the Cas13a. Subsequently 0.1 wt% polyU was added, and both tubes were again left for 3 hours at 37 °C. Finally, 1.0 wt% spermine was added, and it was observed that both tubes showed coacervation. Possible explanations are that the Cas13a was not properly activated because either crRNA or TcR target RNA was degraded, that Cas13a is not able to collaterally cleave polyU or that the Cas13a concentration was too low to cleave all polyU within the 3 hours. Please note that 3 hours is also the time after which Cas13a loses its activity, as determined by fluorescence assays.</P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20171004"><H3>Sample prep Gel electrophoresis</H3><H6>Aafke and Amária</H6><P>The samples were prepared again and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 4 µL of all the samples and 10 µL of the marker were loaded on the gel. Also a positive control was loaded on the gel; the target used in the Cas13a assays. The gel can be seen in the following picture.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171004"><H3>Protein Purification (Day 0)</H3><H6>Guillermo</H6><P>New purifications were started by using the overnight cultures of colonies B1, B4, B5 and B6 from 03-10-2017 to induce one 1 L culture each according to the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A>.</P><P>NB: B3 did not grow, so a new overnight culture was started.</P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 5 October</DIV><DIV class="collapsible-body" id="d20171005"><DIV class="C"><DIV class="row notebook cas13" id="C20171005"><H4>In vitro transcription</H4><H6>Kasper</H6><P>The following sample was prepared:</P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD>crDNA</TD><TD>20</TD></TR><TR><TD>ATP</TD><TD>4</TD></TR><TR><TD>CTP</TD><TD>4</TD></TR><TR><TD>GTP</TD><TD>4</TD></TR><TR><TD>UTP</TD><TD>4</TD></TR><TR><TD>Murine RNase inhibitor</TD><TD>4</TD></TR><TR><TD>10x RNase polymerase buffer</TD><TD>20</TD></TR><TR><TD>T7 polymerase mix</TD><TD>4</TD></TR><TR><TD>Nuclease-free water</TD><TD>136</TD></TR></TBODY></TABLE><P>The sample was left at 37 °C for three hours. Next, the mixture was incubated at the same temperature for another hour, after addition of 1 µL DNase I. 
                 Following this, the reaction mixture was cleaned with the RNeasy MinElute Clean-up kit. A concentration of ~13 ng/µL was found using the Nanodrop, which is relatively low.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20171005"><H3>RNase A titration</H3><H6>Kasper &amp; Jeroen</H6><P>An RNase A titration was done to answer the question: what concentration of RNase A is capable of cleaving enough of the 0.1 wt% polyU within 15 minutes. Assuming that the collateral cleavage activity of active Cas13a is of the same order of magnitude as that of RNase A, this gives an estimate of what order of magnitude of Cas13a should be present. Seven mixes were prepared (final volume 50 µL) in nuclease free water, containing 1xRb, 0.1 wt% polyU, and varying concentrations of RNase A (10<SUP>-1</SUP>, 10<SUP>-2</SUP>, 10<SUP>-3</SUP>, 10<SUP>-4</SUP>, 10<SUP>-5</SUP>, 10<SUP>-6</SUP>, 0) wt%. All were incubated for exactly 15 minutes, after which spermine was added to a final concentration of 1.0 wt%. The solutions containing 10<SUP>-1</SUP> to 10<SUP>-5</SUP> wt% RNase A showed no coacervation upon addition of spermine, whereas the solutions containing 10<SUP>-6</SUP> and 0 wt% RNase A did. This means that at least a concentration of 10<SUP>-5</SUP> wt% of RNase A is necessary to cleave 0.1 wt% polyU in 15 minutes, so that it does not coacervate with spermine anymore. Cas13a is roughly 10x heavier as RNase A, so 10 times as many wt% of Cas13a is required to have similar molar concentration as RNase A. To reach that concentration, at least 10<SUP>-4</SUP> Cas13a is at minimum required. With our 0.05 wt% stock, this means that in principle 0.1 µL of Cas13a is required per 50 µL reaction, but since our stock solution of Cas13a is not completely pure, we will use 1 µL per 50 µL reaction volume in the experiments that follow.
    </P></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20171005"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>The samples containing the DNA of the -7, -8 and -9 dilutions (positioned on the left side of the second ladder) were transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. The samples were inducated at 22 °C for 3 hours. Next, the RNA was purified according to the<A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were prepared and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 4 µL of all the samples and 10 µL of the marker were loaded on the gel. The results can be seen in the picture below.</P><P>You can see that the RPA-reaction worked at room temperature. The RNA polymerase on the other hand, does not work under these conditions.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171005"><H3>Protein Purification (Day 1) and Protein Purification (Day 0)</H3><H4>Protein Purification (Day 1)</H4><H6>Guillermo</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday1" target="blank">TDP purification protocol (Day 1)</A> was carried out with cell pellets of culture B1, B4, B5 and B6 prepared on 04-10-2017.</P><H4>Protein Purification (Day 0)</H4><H6>Guillermo</H6><P>A new purification was started by using the overnight cultures of colony B3 from 05-10-2017 to induce one 1 L culture according to the <A href="https://2017.igem.org/Team:TUDelft/Notebook#tdppurificationday0" target="blank">TDP purification protocol (Day 0)</A>.</P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 6 October</DIV><DIV class="collapsible-body" id="d20171006"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171006"><H3>Sample prep RNA concentration</H3><H6>Aafke and Amária</H6><P>To check the quality of our target RNA, we gave the RNA transcribed on the -8 DNA to the Cas13a model, to perform a cleavage assay with. Cas13a showed no activity, which might be because only 1 µL of our target was added to the mixture.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDPs" id="T20171006"><H3>PCRs for biobrick construction</H3><H6>Fiona</H6><P>For the construction of the TDP/vesicle basic part biobricks, many PCR reactions were carried out. They were prepared according to the following scheme: 
                5 reactions (5 different parts to be constructed) at 8 different melting temperatures (gradient PCR) = 40 tubes. So 40 tubes are labelled. A mastermix without primer template is prepared, containing the following compounds: </P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL) in 1 reaction</TH><TH>Volume (µL) in 42<SUP>*</SUP> reactions</TH></TR></THEAD><TBODY><TR><TD>HF buffer</TD><TD>10</TD><TD>420</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>42</TD></TR><TR><TD>DMSO</TD><TD>1,5</TD><TD>63</TD></TR><TR><TD>Milli-Q</TD><TD>31.0</TD><TD>1302</TD></TR><TR><TD>Total</TD><TD>43.5</TD><TD>1827</TD></TR></TBODY></TABLE><P><SUP>*</SUP>It is better to prepare slightly more mastermix, so that you are sure you will have enough in the end (taking into account pipetting error etc.). The total volume of this mix (1827 µL) is divided over five new tubes (~348 µL per tube). To each of the five tubes, add: 20 µL forward primer; 20 µL reverse primer; 8 µL DNA template and 4 µL of Phusion HF polymerase.<U>NOTE</U>: Keep the tubes on ice, especially after addition of the protein! 
                    The following basic parts were PCRd with the indicated primers: </P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD></TR></TBODY></TABLE><P>Then the following PCR programs were run for the different basic parts, with the number in the first row corresponding to the index of the biobrick.</P><TABLE><THEAD><TR><TH colspan="2">1</TH><TH colspan="2">2</TH><TH colspan="2">3</TH><TH colspan="2">4</TH><TH colspan="2">5</TH></TR></THEAD><THEAD><TR><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD></TR><TR><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD></TR><TR><TD>58-66</TD><TD>30</TD><TD>59-67</TD><TD>30</TD><TD>59-67</TD><TD>30</TD><TD>58-66</TD><TD>30</TD><TD>59-67</TD><TD>30</TD></TR><TR><TD>72</TD><TD>14</TD><TD>72</TD><TD>9</TD><TD>72</TD><TD>8</TD><TD>72</TD><TD>11</TD><TD>72</TD><TD>11</TD></TR><TR><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD></TR><TR><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><P>The PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below.</P><P>The first figure shows an overview of all 40 samples; the first eight are torA-GFP, the second eight SAHS_68234, etc. (in the order of the table). To get a better idea of the sizes on the bottom wells of the big gel, these were still run on gel separately. This is shown in the second figure. All the products were of the correct size (550-900 bp). </P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20171006"><H3>PCRs for biobrick construction</H3><H6>Fiona</H6><P>For the construction of the TDP/vesicle basic part biobricks, many PCR reactions were carried out. They were prepared according to the following scheme: 
                5 reactions (5 different parts to be constructed) at 8 different melting temperatures (gradient PCR) = 40 tubes. So 40 tubes are labelled. A mastermix without primer template is prepared, containing the following compounds: </P><TABLE><THEAD><TR><TH>Compound</TH><TH>Volume (µL) in 1 reaction</TH><TH>Volume (µL) in 42<SUP>*</SUP> reactions</TH></TR></THEAD><TBODY><TR><TD>HF buffer</TD><TD>10</TD><TD>420</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>42</TD></TR><TR><TD>DMSO</TD><TD>1,5</TD><TD>63</TD></TR><TR><TD>Milli-Q</TD><TD>31.0</TD><TD>1302</TD></TR><TR><TD>Total</TD><TD>43.5</TD><TD>1827</TD></TR></TBODY></TABLE><P><SUP>*</SUP>It is better to prepare slightly more mastermix, so that you are sure you will have enough in the end (taking into account pipetting error etc.). The total volume of this mix (1827 µL) is divided over five new tubes (~348 µL per tube). To each of the five tubes, add: 20 µL forward primer; 20 µL reverse primer; 8 µL DNA template and 4 µL of Phusion HF polymerase.<U>NOTE</U>: Keep the tubes on ice, especially after addition of the protein! 
                    The following basic parts were PCRd with the indicated primers: </P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD></TR></TBODY></TABLE><P>Then the following PCR programs were run for the different basic parts, with the number in the first row corresponding to the index of the biobrick.</P><TABLE><THEAD><TR><TH colspan="2">1</TH><TH colspan="2">2</TH><TH colspan="2">3</TH><TH colspan="2">4</TH><TH colspan="2">5</TH></TR></THEAD><THEAD><TR><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH><TH>T (°C)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30</TD></TR><TR><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10</TD></TR><TR><TD>58-66</TD><TD>30</TD><TD>59-67</TD><TD>30</TD><TD>59-67</TD><TD>30</TD><TD>58-66</TD><TD>30</TD><TD>59-67</TD><TD>30</TD></TR><TR><TD>72</TD><TD>14</TD><TD>72</TD><TD>9</TD><TD>72</TD><TD>8</TD><TD>72</TD><TD>11</TD><TD>72</TD><TD>11</TD></TR><TR><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD><TD>72</TD><TD>300</TD></TR><TR><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><P>The PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below.</P><P>The first figure shows an overview of all 40 samples; the first eight are torA-GFP, the second eight SAHS_68234, etc. (in the order of the table). To get a better idea of the sizes on the bottom wells of the big gel, these were still run on gel separately. This is shown in the second figure. All the products were of the correct size (550-900 bp). </P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week41-scroll">Week 41</DIV></LI><LI><DIV class="collapsible-header">Monday 9 October</DIV><DIV class="collapsible-body" id="d20171009"><DIV class="C"><DIV class="row notebook cas13" id="C20171009"><H4>Testing other crRNA</H4><H6>Kasper</H6><P>To test the other crRNAs (2, 3 and 4) a platereader experiment with RNase Alert was carried out. For this purpose, wells B1/B2, B3/B4 and B5/B6 were filled with 1 µL crRNA 2, 3 and 4, respectively. Additionally, 1 µL of target (TcR) was added. Furthermore, 1 µL Cas13a, 10 µL of 1250 nM RNase Alert and 10 µL of 1x Reaction buffer was added. A control was made containing everything except TcR and crRNA.</P><P>As can be derived from the figure, crRNA 3 was found to be the best one.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171009"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>The concentration of the RNA samples was measured following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaconcentration" target="_blank">RNA concentration measurement (NanoDrop)</A> protocol. </P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>ssRNA concentration (ng/µL)</TH><TH>A260/A280</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">-3</TD><TD>42.12</TD><TD>1.91</TD></TR><TR><TD class="firstcolumn">-4</TD><TD>37.88</TD><TD>1.94</TD></TR><TR><TD class="firstcolumn">-5</TD><TD>26.49</TD><TD>1.93</TD></TR><TR><TD class="firstcolumn">-6</TD><TD>23.06</TD><TD>2.17</TD></TR><TR><TD class="firstcolumn">-7</TD><TD>13.33</TD><TD>0.22</TD></TR></TBODY></TABLE><P>We didn't have enough -8 left to measure the concentration. Above that, the A260/A280 of -7 indicates a low purity of that sample. 1 &amp;micrp;L of -3 is tested as target for cas13a in a cleavage assay. Cas13a was activated by our -3 target!</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171009"><H3>Bradford Assay and LDH assay preparation</H3><H4>Bradford Assay</H4><H6>Aafke, Guillermo</H6><P>A Bradford Assay of the proteins B1, B5 and B6 (from 07-10-2017) and B3 (from 08-10-2016) resuspended in 1 mL MQ was carried out to determine their concentration:</P></DIV></DIV></DIV></DIV></LI><LI>c[B1] = 2.21 g/L</LI><LI>c[B4] = 1.23 g/L</LI><LI>c[B5] = 1.33 g/L</LI><LI>c[B6] = 1.01 g/L</LI><H4>LDH assay preparation</H4><H6>Guillermo, Isabell</H6><P>Multiple samples of different concentrations were prepared for an LDH assay. Volumes of 100 µL were prepared and split into 2 aliquots of 50 µL each. One aliquot was stored in the fridge overnight, serving as a control, and the other was dried.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>TDP</TH><TH>concentration [g/L]</TH><TH>V(TDP) [µL]</TH><TH>V(LDH) [µL]</TH><TH>V(Tris/HCl) [µL]</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">B1</TD><TD>0.1</TD><TD>5</TD><TD>1</TD><TD>1</TD><TD>93</TD></TR><TR><TD>0.5</TD><TD>23</TD><TD>1</TD><TD>1</TD><TD>75</TD></TR><TR><TD>1</TD><TD>45</TD><TD>1</TD><TD>1</TD><TD>53</TD></TR><TR><TD class="firstcolumn">B3</TD><TD>0.1</TD><TD>8</TD><TD>1</TD><TD>1</TD><TD>90</TD></TR><TR><TD>0.5</TD><TD>41</TD><TD>1</TD><TD>1</TD><TD>57</TD></TR><TR><TD>1</TD><TD>81</TD><TD>1</TD><TD>1</TD><TD>17</TD></TR><TR><TD class="firstcolumn">B5</TD><TD>0.1</TD><TD>8</TD><TD>1</TD><TD>1</TD><TD>90</TD></TR><TR><TD>0.5</TD><TD>38</TD><TD>1</TD><TD>1</TD><TD>60</TD></TR><TR><TD>1</TD><TD>75</TD><TD>1</TD><TD>1</TD><TD>23</TD></TR><TR><TD class="firstcolumn">B6</TD><TD>0.1</TD><TD>10</TD><TD>1</TD><TD>1</TD><TD>88</TD></TR><TR><TD>0.5</TD><TD>50</TD><TD>1</TD><TD>1</TD><TD>48</TD></TR><TR><TD>1</TD><TD>98</TD><TD>1</TD><TD>1</TD><TD>0</TD></TR></TBODY></TABLE><DIV class="V"><DIV class="row notebook vesicles" id="V20171009"><H3>New PCR of backbone for biobrick construction</H3><H6>Fiona</H6><P>There were no colonies from the transformation on 09-10-2017. However, there was no more backbone to do a restriction/ligation with. Therefore, the pSB1C3 backbone was PCR amplified. Thus, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was carried out with primers IG0010 and IG0011. The PCR was carried out at eight different melting temperatures, ranging between 64 and 71 °C. An extension time of 2 minutes was used.</P></DIV></DIV><LI><DIV class="collapsible-header">Tuesday 10 October</DIV><DIV class="collapsible-body" id="d20171010"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20171010"><H3>LDH assay and Cas13a drying assay</H3><H4>LDH assay (longterm Day 18, longterm2 Day 12)</H4><H6>Aafke, Guillermo</H6><P>An <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with the dried and liquid samples prepared on 09-10-2017. Additionally, the fourth iteration of the longterm and the third iteration of the longterm2 <A href="https://2017.igem.org/Team:TUDelft/Notebook#ldhassay" target="blank">LDH assay</A> was carried out with an L4, D4 and D sample of both TDPs (B1 and B5) at concentration 0.5 and 1 g/L prepared on 20-09-2017, as well as samples of B1 and B5 at concentration 0.1 g/L, LDH dried without TDPs and LDH freeze-dried prepared on 27-09-2017.</P><H4>Cas13a drying assay</H4><H6>Gabriella, Isabell</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Notebook#cas13atdpassay" target="blank">Cas13a TDP drying assay</A> was executed with the TDP CAHS 94205 in a concentration of 1 g/L.
            </P><P>Several samples were prepared to be dried, as well as several to serve as a liquid control:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>V(Cas13a) [µL]</TH><TH>V(TDP) [µL]</TH><TH>V(Tris/HCl)</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Cas13a + CAHS 94205 (dried)</TD><TD>4</TD><TD>50</TD><TD>1</TD><TD>45</TD></TR><TR><TD class="firstcolumn">Cas13a (dried)</TD><TD>4</TD><TD>0</TD><TD>1</TD><TD>95</TD></TR><TR><TD class="firstcolumn">CAHS 94205 (dried)</TD><TD>0</TD><TD>50</TD><TD>1</TD><TD>49</TD></TR><TR><TD class="firstcolumn">Cas13a + CAHS 94205 (liquid)</TD><TD>4</TD><TD>50</TD><TD>1</TD><TD>45</TD></TR><TR><TD class="firstcolumn">Cas13a (liquid)</TD><TD>4</TD><TD>0</TD><TD>1</TD><TD>95</TD></TR></TBODY></TABLE><P>All measurements were performed in duplicate.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20171010"><H3>Restriction and ligation for biobrick construction</H3><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank"> digestion assay</A> protocol was followed to cut both the pSB1C3 backbone and the PCR products (07-10-2017) with EcoRI and PstI. Next, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">sticky-end ligation</A> protocol was followed to ligate the inserts in the new pSB1C3 backbone. The aim was to have a 1:3 ratio of vector:inserts initially. However, due to a limited quantity if available backbone, each ligation was carried out with 4 µL of backbone and 4 µL of insert The subsequent ligation mixes were transformed in electrocompetent Dh5α-cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#heatshocknew" target="_blank">new heat-shock</A> protocol.p&gt;</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 11 October</DIV><DIV class="collapsible-body" id="d20171011"><DIV class="C"><DIV class="row notebook cas13a" id="C20171011"><H3>Cas13a parts amplification for biobrick construction</H3><H6>Fiona</H6><P>To amplify the Cas13a Part1+Part2 (which contains the cas13a gene with the biobrick prefix) with a suffix, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrphusion" target="_blank">PCR (Phusion polymerase)</A> protocol was carried out with primers IG0085 and IG0086. This was done at eight melting temperatures, between 60 and 68 °C. The extension time was 2 minutes. Additionally, the spacer sequence was amplified with a prefix and suffix following the same protocol. For this reaction, IG0023 and IG0087 were used. The same range of melting temperatures was tested. This PCR program was run with an extension time of 15 seconds. Finally, to amplify the whole construct (gene plus spacer sequence), the PCR protocol was carried out with primers IG0006 and IG0007, with temperatures ranging between 57 and 67 °C and an extension time of 150 seconds.</P><P>The PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below. The first 8 samples are the gene, the second 8 samples are the spacer and the final 8 samples are the composite part.</P><P>Especially the PCR for the composite part has unspecific bands. <A href="https://2017.igem.org/Team:TUDelft/Protocols#gelpurification" target="_blank"> Gel isolation and purification (Promega Wizard™ Kit)</A> was thus performed. This only resulted in a concentration of 4 ng/µL. Another round of PCR to amplify this failed and it was decided to focus our efforts on the basic parts instead of the composite part.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171011"><H3>Sample prep RPA and in vitro transcription and inocculation</H3><H4>RPA and in vitro transcription</H4><H6>Aafke and Amária</H6><P>-8 DNA and pSB1T3 were transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpa+invitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol.
            <I>pSB1T3 was left overnight on the bench, instead of in the -20 freezer.</I> Next, the RNA was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were stored in the -20 freezer. </P><H4>Isolates from mastitis infection</H4><H6>Aafke and Amária</H6><P>From the veterinary institute in Lelystad we recieved isolates from a mastitis infection, which contain the BlaZ gene. In this way we can test our sample preparation and detection method on a more relevant sample. We inocculated 5 mL of Tryptic Soy Broth with the three isolates (S. aureus, CNS1 &amp; CNS2).</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDP" id="T20171011"><H3>Coacervation assay</H3><H6>Guillermo</H6><P>TDPs were to e mixed with RNA in a mass ratio of 7 (TDP/RNA) with RNA 0.1 % and 19 % glycerol. 39 µL of each TDP were mixed with 9.5 µL glycerol (100 %), 1 µL of Ammonia Acetate (c = 1 M) and 0.5 µL Poly (U) (10 %). A series of controls were prepared without Poly (U) for each of the TDPs and checked. No coacervation could be observed in any of the samples.</P><P>The samples were then incubated at 37 °C for 5 minutes and 10 minutes, but no coacervation was visible.</P></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Thursday 12 October</DIV><DIV class="collapsible-body" id="d20171012"><DIV class="C"><DIV class="row notebook cas13a" id="C20171012"><H3>Restriction and ligation for biobrick construction</H3><H6>Fiona</H6><P>The amplified parts were digested with PstI and EcoRI according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank"> digestion assay</A> protocol. Next, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">sticky-end ligation</A> protocol was followed to ligate the inserts in the new pSB1C3 backbone. Two reactions were carried out simultaneously: one with a 1:1 ratio of vector:insert and one with the remaining backbone and an equal volume of insert. The subsequent ligation mixes were transformed in electrocompetent Dh5α-cells according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#heatshocknew" target="_blank">new heat-shock</A> protocol.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20171012"><H3>Discrete timepoing coacervation Cas13a</H3><H6>Kasper &amp; Jeroen</H6><P>Two mixes were prepared as follows, in nuclease free water:
    </P><TABLE class="notebook table table-style-1"><TBODY><TR><TD class="firstcolumn">Active mix</TD><TD>Inactive mix</TD></TR><TR><TD class="firstcolumn">1xRb</TD><TD>1xRb</TD></TR><TR><TD class="firstcolumn">0.1 wt% polyU</TD><TD> 0.1 wt% polyU</TD></TR><TR><TD class="firstcolumn">0.3 ng/µL TcR target</TD><TD>-</TD></TR><TR><TD class="firstcolumn">0.3 ng/µL crRNA 3</TD><TD>-</TD></TR></TBODY></TABLE><P>Both mixes were divided over 5 different tubes each, of 44 µL. To each of the 10 tubes, 1 µL of Cas13a (0.05 wt%, or 3.4 µM stock) was added. All tubes were left at 37 °C. At different timepoints, to one tube of the active and one of the inactive mix, spermine was added to a final concentration of 1.0 wt% was added and the tubes were checked for coacervation. The result:</P><TABLE class="notebook table table-style-1"><TBODY><TR><TD class="firstcolumn">Time (min)</TD><TD>Coacervation in Active mix?</TD><TD>Coacervation in Inactive mix?</TD></TR><TR><TD class="firstcolumn">0 </TD><TD>yes</TD><TD>yes</TD></TR><TR><TD class="firstcolumn">15 min</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD class="firstcolumn">30 min</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD class="firstcolumn">45 min</TD><TD>no</TD><TD>yes</TD></TR><TR><TD class="firstcolumn">60 min</TD><TD>no</TD><TD>yes</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="S"><DIV class="row notebook sampleprep" id="S20171012"><H3>Sample prep Gel electrophoresis and DNA isolation</H3><H4>DNA isolation isolates</H4><H6>Aafke and Amária</H6><P>Three different methods were used to isolate the DNA from the isolates. 1 mL of all three isolates were prepared following the boiling method of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. 3 mL of all three isolates were prepared following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A> protocol. Another 1 mL of all three isolates was prepared with the Norgen Milk Bacterial DNA Isolation Kit following the protocol for Gram Positive bacteria. The two commercial DNA isolation kits were used as a control. The samples were stored in the -20 freezer.</P><H4>RNA Gel electrophoresis </H4><H6>Aafke and Amária</H6><P>The stored samples of pSB1T3 and -8 were loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 4 µL of all the samples and 10 µL of the marker were used. The results can be seen in the picture below.</P><P>This time, the RPA and in vitro transcription on the -8 dilution DNA did not work.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDPs" id="T20171012"><H3>Colony PCRs for biobrick construction</H3><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCRs (GoTaq)</A> protocol was followed on the colonies resulting from the Dh5α heat-shock transformations. 24 colonies were picked for each of the TDP proteins (2 CAHS, 2 SAHS) and the vesicle TorA-GFP construct. The primers, the melting temperatures and extensions times used for each part can be found in the table below.</P><P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH><TH>Melting temperature (°C)</TH><TH>Extension time (s)</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD><TD>59</TD><TD>55</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD><TD>59</TD><TD>30</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD><TD>59</TD><TD>30</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD><TD>54</TD><TD>45</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD><TD>55</TD><TD>45</TD></TR></TBODY></TABLE></P><P>For each biobrick, 13/24 of the PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below. The first 13 are for TorA-GFP, the next 13 for SAHS_68234 etc. in the order of the table.</P><P>Since these results looked promising, some samples were <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">miniprepped</A> and sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. Two samples (with two different primers) were sent for each biobrick. The table below provides an overview:</P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH><TH>Colony</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD><TD>2</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD><TD>5</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD><TD>2</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD><TD>11</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD><TD>9</TD></TR></TBODY></TABLE><P>Sequencing results confirmed that SAHS_68234 and CAHS_94205 were biobricked successfully.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20171012"><H3>Colony PCRs for biobrick construction</H3><H6>Fiona</H6><P>The <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCRs (GoTaq)</A> protocol was followed on the colonies resulting from the Dh5α heat-shock transformations. 24 colonies were picked for each of the TDP proteins (2 CAHS, 2 SAHS) and the vesicle TorA-GFP construct. The primers, the melting temperatures and extensions times used for each part can be found in the table below.</P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH><TH>Melting temperature (°C)</TH><TH>Extension time (s)</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD><TD>59</TD><TD>55</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD><TD>59</TD><TD>30</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD><TD>59</TD><TD>30</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD><TD>54</TD><TD>45</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD><TD>55</TD><TD>45</TD></TR></TBODY></TABLE><P>For each biobrick, 13/24 of the PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below. The first 13 are for TorA-GFP, the next 13 for SAHS_68234 etc. in the order of the table.</P><P>Since these results looked promising, some samples were <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">miniprepped</A> and sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. Two samples (with two different primers) were sent for each biobrick. The table below provides an overview:</P><TABLE><THEAD><TR><TH>Index</TH><TH>Basic part</TH><TH>Primer set</TH><TH>Colony</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>torA-GFP</TD><TD>IG0073 + IG0074</TD><TD>2</TD></TR><TR><TD>2</TD><TD>SAHS_68234</TD><TD>IG0075 + IG0076</TD><TD>5</TD></TR><TR><TD>3</TD><TD>SAHS_33020</TD><TD>IG0077 + IG0078</TD><TD>2</TD></TR><TR><TD>4</TD><TD>CAHS_106094</TD><TD>IG0081 + IG0082</TD><TD>11</TD></TR><TR><TD>5</TD><TD>CAHS_94205</TD><TD>IG0083 + IG0084</TD><TD>9</TD></TR></TBODY></TABLE><P>Sequencing results confirmed that SAHS_68234 and CAHS_94205 were biobricked successfully.</P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 13 October</DIV></LI><LI><DIV class="collapsible-header scrollspy" id="week42-scroll">Week 42</DIV></LI><LI><DIV class="collapsible-header">Monday 16 October</DIV><DIV class="collapsible-body" id="d20171016"><DIV class="C"><DIV class="row notebook cas13a" id="C20171016"><H3>Colony PCRs for biobrick construction</H3><H6>Fiona</H6><P>The colonies were subjected to <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A>. For the gene, primers IG0085 and IG0086 were used. The PCR program had a melting temperature of 53 °C and an extension time of 2 minutes. The spacer biobrick PCRs used primers IG0087 and IG0023, a melting temperature of 52 °C and an extension time of 10 seconds. 
            PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> picture below. </P><P>The top row of wells corresponds to colonies with the gene; the bottom row corresponds to colonies with the spacer. A lot of unspecific bands are visble (also due to the low annealing temperature). Nevertheless, colony 11 of the gene and colonies 1, 11 and 14 of the spacer were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. The gene sample was sent with IG0005. The spacer samples were sent with IG0023.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDPs" id="T20171016"><H3>Sequencing for biobrick construction</H3><H6>Fiona</H6><P>Additional colonies were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. For SAHS_33020, colony 6 and 7 were sent in to be sequenced with primer IG0077. For CAHS_106094, colony 9 and 13 were sent in for sequencing with primer IG0081.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20171016"><H3>Sequencing for biobrick construction</H3><H6>Fiona</H6><P>Additional colonies were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A> with primer IG0073, namely colony 10 and 11.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Tuesday 17 October</DIV><DIV class="collapsible-body" id="d20171017"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="T"><DIV class="row notebook TDP" id="T20171017"><H3>Cas13a drying assay</H3><H6>Gabriella, Isabell</H6><P>A <A href="https://2017.igem.org/Team:TUDelft/Notebook#cas13atdpassay" target="blank">Cas13a TDP drying assay</A> was executed with the TDP SAHS33020 in a concentration of 1 g/L.
            </P><P>Several samples were prepared to be dried, as well as several to serve as a liquid control:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Sample</TH><TH>V(Cas13a) [µL]</TH><TH>V(TDP) [µL]</TH><TH>V(Tris/HCl)</TH><TH>V(MQ) [µL]</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Cas13a + SAHS 33020 (dried)</TD><TD>4</TD><TD>67</TD><TD>1</TD><TD>28</TD></TR><TR><TD class="firstcolumn">Cas13a (dried)</TD><TD>4</TD><TD>0</TD><TD>1</TD><TD>95</TD></TR><TR><TD class="firstcolumn">SAHS 33020 (dried)</TD><TD>0</TD><TD>67</TD><TD>1</TD><TD>32</TD></TR><TR><TD class="firstcolumn">Cas13a + SAHS 33020 (liquid)</TD><TD>4</TD><TD>67</TD><TD>1</TD><TD>28</TD></TR><TR><TD class="firstcolumn">Cas13a (liquid)</TD><TD>4</TD><TD>0</TD><TD>1</TD><TD>95</TD></TR></TBODY></TABLE><P>All measurements were performed in duplicate.</P></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Wednesday 18 October</DIV><DIV class="collapsible-body" id="d20171018"><DIV class="C"><DIV class="row notebook cas13a" id="C20171018"><H3>Colony PCRs for biobrick construction</H3><H6>Fiona</H6><P>Because of the previous, inconclusive colony PCR, another <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> colony PCR (GoTaq)</A> was run on the gene. This time, primers IG0006 and IG0007 were used. PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> picture below. </P><P>This time, the results were more specific. However, there were no bright bands at the right size. Nevertheless, colony 6 of the gene and colony 11 for the spacer were<A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">miniprepped</A> and sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. For both samples primer IG0006 was used.</P></DIV></DIV><DIV class="D"><DIV class="row notebook detection" id="D20171018"><H3>Repeat of spermine dependence at 0.1 wt% polyU</H3><H6>Kasper &amp; Jeroen</H6><P>A repeat of the spermine dependence assay from 03-Jul-2017 was done but now with three important changes:
    </P></DIV></DIV></DIV></LI><LI>The polyU concentration was adjusted to 0.1 wt%</LI><LI> It was done in the Cas13a reaction buffer (1xRb) instead of 20 mM HEPES.</LI><LI> It was done at 37 °C.</LI><P>Seven mixtures of 300 µL were prepared, of which one (the blank) contained only 1xRb and the rest contained 0.1 wt% polyU, 1xRb, and varying amounts of spermine (0, 0.05, 0.1, 0.5, 1.0, 5.0) wt%. The mixtures were mixed by gentle pipetting and divided over three wells in a 96 well plate. The plate was insterted into the platereader and mixed for 1 minute. Subsequently, it was left to equilibriate temperature for 5 minutes, before the absorbance at 500 nm was measured.
    </P><DIV class="S"><DIV class="row notebook sampleprep" id="S20171018"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>The isolated DNA from the BlaZ isolates were amplified and transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpa+invitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. As a negative control we used both TS-broth and Mili-Q. <I>The top of the blank was switched with the top of sample 3.1, contaiminating the blank.</I> Next, the RNA was purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were prepared and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 4 µL of all the samples and 10 µL of the marker were loaded on the gel. The results can be seen in the picture below. (1=S.aureus, 2=CNS1, 3=CNS2 and .1=boiling method, .2=miniprep, .3=norgen isolation kit)</P><P>As our blank was contaiminated, we cannot conclude anything from this gel.</P></DIV></DIV><DIV class="T"><DIV class="row notebook TDPs" id="T20171018"><H3>Sequencing for biobrick construction</H3><H6>Fiona</H6><P>The previous reads came back incomplete, which is why new samples where sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. For SAHS_33020, colony 6 was sent in. For CAHS_106094, colony 13 was sent in. Primer IG0006 was used in both cases. This time, the sequencing results came back positive and the TDPs were confirmed.</P></DIV></DIV><DIV class="V"><DIV class="row notebook vesicles" id="V20171018"><H3>Sequencing for biobrick construction</H3><H6>Fiona</H6><P>Additional colonies were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A> with primer IG0007, namely colony 2, 10 and 11.</P></DIV></DIV><LI><DIV class="collapsible-header">Thursday 19 October</DIV><DIV class="collapsible-body" id="d20171019"><DIV class="C"><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171019"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>The overnight culture was analysed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A> protocol. Again, the isolated DNA from the BlaZ isolates were amplified and transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpa+invitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. Besides that, also a dilution range of -3 until and including -8 was prepared in milk and the DNA was isolated following the boiling method of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. The isolated DNA from the dilution range were amplified and transcribed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpa+invitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. Next, the all the RNA samples were purified according to the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol. After purification of the RNA, the samples were prepared and loaded on the gel as described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol. 4 µL of all the samples and 10 µL of the marker were loaded on the gel. The results can be seen in the picture below. (1=S.aureus, 2=CNS1, 3=CNS2 and .1=boiling method, .2=miniprep, .3=norgen isolation kit. -3 to -8 indicating the different dilutions)</P><P>None of the dilutions shows a band.</P></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">Friday 20 October</DIV><DIV class="collapsible-body" id="d20171020"><DIV class="C"><DIV class="row notebook cas13a" id="C20171020"><H3>Colony PCRs and sequencingfor biobrick construction</H3><H6>Fiona</H6><P>Since the sequencing results came back negative, new colonies were <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> PCRd (GoTaq)</A> to find a positive clone. Primers IG0006 and IG0024 were used for the gene, with an melting temperature of 55 °C and an extension time of 2 minutes. For the spacer, primers IG0007 and IG0087 were used. This reaction was carried out with a T<SUB>M</SUB> of 53 °C and an extension time of 15 seconds.
            PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> pictures below. The first gel shows the colony PCR for the colonies with the gene; the second gel shows the colony PCR for the colonies with the spacer.</P><P>Per biobrick, five additional colonies were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. For the gene, colonies 5, 10, 14, 15 and 18 were sent. For the spacer, colonies 5, 6, 14, 17 and D were sent. Primer IG0006 was used in all cases. The sequencing results for the spacer came back positive.</P></DIV></DIV><DIV class="D"><DIV class="S"><DIV class="row notebook sampleprep" id="S20171020"><H3>Sample prep RPA and in vitro transcription</H3><H6>Aafke and Amária</H6><P>As none of the dilutions gave a band yesterday, but the undiluted samples did, we are trying it again today with a dilution of -1 and -2. The dilutions are prepared both in milk and in LB-medium. Besides that, we used as a negative control a sample containing the DH5-α with pSB1T3. The DNA was isolated following the boiling method of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. the -7 droplet from the teardrop assay performed on the 19<SUP>th</SUP> of november had a countable amount, it had 6 colonies. This means that in our original overnight culture (from the 18<SUP>th</SUP> of November) 12x10<SUP>9</SUP> cells where present. 
      </P></DIV></DIV><DIV class="T"><DIV class="V"><DIV class="row notebook vesicles" id="V20171020"><H3>Colony PCR for biobrick construction</H3><H6>Fiona</H6><P>Since the sequencing results came back negative, new colonies were <A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> PCRd (GoTaq)</A> to find a positive clone. Primers IG0073 and IG0074 were used, with an melting temperature of 53 °C and an extension time of 1 minute.
            PCR results are visible in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">agarose gel</A> picture below. </P><P>Per biobrick, five additional colonies were sent for <A href="https://2017.igem.org/Team:TUDelft/Protocols#seqsam" target="_blank">sequencing (Macrogen)</A>. In this case, colonies 11-15 were sent, with primer IG0006. Now sequencing results came back positive and the part was successfully biobricked.</P></DIV></DIV></DIV></DIV></DIV></LI><FOOTER><H4 class="center">We thank our sponsors:</H4><DIV class="sponsors center"><DIV><A href="https://www.tudelft.nl/en/faculty-of-applied-sciences/">Faculty of Applied Sciences</A></DIV></DIV></FOOTER></BODY></HTML>