<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Main-Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Main-Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col s12 m10 l8 center-margin"><DIV align="center"><H1 class="cyan-text"> An introduction to Case13a</H1></DIV><P>According to the World Health Organization (WHO), &quot;antibiotic resistance is one of the biggest threats to global health, food security and development today&quot; (<A href="#references">World Health Organization 2016</A>). In the United States alone, over 2 million illnesses and 23,000 deaths are estimated to be caused by infections of antibiotic resistant pathogens annually (<A href="#references">Centers for Disease Control and Prevention 2017</A>). Our goal as this year's  TU Delft iGEM team, was to tackle the antibiotic resistance problem by using synthetic biology. The first step towards this goal is to reduce the use of unnecessary antibiotics. We decided to focus on the dairy industry, which is responsible for 14% of the total revenue of the Dutch food production (<A href="#references">Statistics Netherlands (CBS) 2015</A>).</P><DIV class="row align-center"><DIV class="col s6"><P class="">Through our interactions with farmers, veterinarians and health and safety experts we found out that the most common disease in the dairy industry today is mastitis (see our <A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" target="_blank">Integrated Human Practices</A>). Mastitis is an infection of the cow-udder, causing over 100 million euros of damage annually in the Netherlands alone (<A href="#references">GDDiergezondheid 2017</A>). Testing whether an infection for a particular cow with mastitis is caused by antibiotic resistant bacteria is currently an expensive and laborious process. Typically, a farmer will first treat the cow with standard antibiotics. Only if these do not work will the farmer send milk samples to a company specialized in diagnostics. This company will then analyze the sample and indicate what antibiotics can be used. This 'down time' is unpleasant for both the farmer and the cow, which is why we focussed our efforts to engineer a system that can be used to detect antibiotic resistance genes easily and quickly. We have interacted with various stakeholders to make sure our system is both a relevant and responsible solution. With this in mind, we identified several engineering principles needed for our detection method. First of all, our detection method should be fast and accurate. Additionally, our design should be easy to use, safe, and allow for on-site detection.</P></DIV></DIV><DIV align="left"><H3 class="cyan-text">Cas13a</H3></DIV><P>For our detection method, we were inspired by recent advances in CRISPR-Cas technology. The CRISPR-Cas system is a bacterial adaptive immune system that utilizes CRISPR-associated (Cas) proteins. Some of these Cas proteins have the ability to search for a specific RNA or DNA sequence with small piece of guide RNA, denoted as crRNA. Once these Cas proteins find their target (matching its guide), the target is cleaved. Examples of these Cas proteins that have this 'search-and-destroy' function are Cas9, Cpf1 and Cas13a (<A href="#references">Jinek et al. 2012; Zetsche et al. 2015; Abudayyeh et al. 2016</A>, respectively). The first two target DNA, whereas the latter targets RNA. We utilized the recently characterized variant Cas13a (<A href="#references">Abudayyeh et al. 2016; Gootenberg et al. 2017; Liu et al. 2017</A>) which targets RNA. Besides only finding and cleaving its specified RNA target, Cas13a has the unique ability that once it has found its target, it undergoes a conformational change and starts to engage in collateral cleavage. This means that it will also cleave all other RNA it encounters. This feature makes Cas13a perfect for detection of specific RNA sequences: it only needs to be activated by a single target molecule, after which collateral cleavage is initiated that can be used to generate a detectable signal.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 1: Collatoral cleavage.</B> Cas13a recognizes its target. Cas13a gets activated. Once activated Cas13a starts cleaving all other RNA in its surroundings.</P></DIV></DIV><DIV align="right"><H3 class="cyan-text">Motif finder</H3></DIV><DIV class="row align-center"><DIV class="col m6 s12"><P><B>Figure 2: Schematic overview of our software tool.</B>  First our tool extracts a certain gene family from the NCBI database. Secondly, our newly developed software tool alignes the genes and screens for conserved regions. These conserved regions can serve as templates to design primers that could be used to amplify part of the genes.</P></DIV><DIV class="col m8 s12"><P class=""> There is a well-documented group of genes that is responsible for resistance against specific antibiotics (<A href="#references">Jia et al. 2017; McArthur and Wright 2015; McArthur et al. 2013</A>). By detecting these genes, we are able to predict whether or not a certain antibiotic will be effective. Thus stimulating a targeted use of antibiotics, preventing the further spread of antibiotic resistance. Ideally, we would only target a single sequence for each type of antibiotic resistance, but the genes encoding for the same antibiotic resistance exhibit a lot of variation. We therefore developed a software tool that can identify conserved regions among the variants of a certain resistance gene. These conserved regions were subsequently screened for suitable target sequences. Finally the reliability of the Cas13a-mediated detection system was assessed by employing a biophysical model that predicts the off-target activity of the protein when it utilizes a certain guide. Determining the probability of acquiring a false positive output in a sample containing non-target RNA indicates how reliable our system is. Our modeling was essential to support the lab work as we utilized the found target sequences to design the crRNA sequences that direct Cas13a towards a desired target.</P></DIV></DIV><DIV align="left"><H3 class="cyan-text">Sample prep</H3></DIV><P>For Cas13a to be able to detect antibiotic resistance genes, it needs an RNA target. Where current methods only detect active genes, we want our method to be able to identify potential resistant pathogens as well. Therefore, DNA needs to be isolated from a biological sample and the target DNA needs to be amplified and transcribed into RNA. To ensure that our design is easy to use, we experimented with and mapped out simplified protocol for the sample preparation. </P><DIV class="row"><DIV class="col s12"><P><B>Figure 3: Overview of our easy sample preparation.</B> DNA isolation by boiling and the subsequent DNA amplification and transcription into RNA without the use of any advanced laboratory devices.</P></DIV></DIV><DIV align="right"><H3 class="cyan-text">Coacervation</H3></DIV><DIV class="row align-center"><DIV class="col m6 s12"><P class=""> Once our Cas13a has found its target sequence and is activated, we need to convert the collateral cleavage activity to a read-out visible to the naked eye. We apply a completely novel detection method (coined CINDY Seq) that relies on the ability of oppositely charged macro-molecules to associate (coacervate) and form a cloudy mixture. In this way, we were able to visualize the cleavage of RNA by Cas13a through the turbidity of the solution. We applied a model to better understand the physics behind this phase transition. </P></DIV><DIV class="col m7 s9"><P><B>Figure 4: oppositely charged macro-molecules forming coacervates.</B></P></DIV></DIV><DIV align="left"><H3 class="cyan-text">TDPs</H3></DIV><DIV class="row align-center"><DIV class="col m2 s12"><P><B>Figure 5: dried and rehydrated Tardigrade.</B></P></DIV><DIV class="col m8 s12"><P class="">To allow for on-site detection, it is important that our device has a long shelf life. To be able to store our product for multiple days, we take inspiration from nature. Tardigrades are micro-animals that can survive severe desiccation and even the extreme conditions that occur in outer space (<A href="#references">Sloan et al. 2017</A>). The biomolecules that mediate desiccation tolerance in the tardigrade are Tardigrade-specific intrinsically Disordered Proteins (TDPs) (<A href="#references">Boothby et al., 2017</A>). To see if integration of these TDPs into our device was feasible, and which TDP would provide best results, we developed a lattice model to mimic the scaffold TDPs form that protect the protein structure upon drying. Subsequently, we applied the TDPs in our device and managed to dry and store our proteins, while retaining their functionality. </P></DIV></DIV><DIV align="right"><H3 class="cyan-text">Vesicles</H3></DIV><P>Finally, to achieve responsible usage of our detection system outside specialized lab facilities, we needed to engineer a cell-free system. Of course, this could be achieved by purification. However, to make optimal use of synthetic biology we decided to try a different approach. By letting our bacteria produce the proteins in vesicles, we can take advantage of our bacteria as little cell-factories and easily purify the proteins from the broth (<A href="#references">Schwechheimer and Kuehn, 2015</A>). We modeled the kinetics of such a system to decide at what time these vesicles need to be harvested. In this way we ensure a clean, safe and cell-free design. </P><DIV align="center"><H2 class="cyan-text">Project overview</H2></DIV><DIV style="text-align:center"><P><B>Figure 6: Overview of all the different aspects of our project.</B></P></DIV><P>The specific case we approached in our project was the detection of antibiotic resistance in bacteria that cause mastitis. Our detection method uses a guide RNA selected specifically for this purpose. This guide RNA can easily be changed to target any other sequence, allowing for much broader applications. In this way our tool can provide an easy-to-use alternative to already existing detection methods for specific RNA. </P><DIV id="references" class="section card"><DIV class="card-content"><SPAN class="card-title">References:</SPAN><OL><LI><A href="https://www.biorxiv.org/content/early/2016/05/21/054742">Abudayyeh, O.O. et al., 2016. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science 353, aaf5573.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S1097276517301338">Boothby, T.C., Tapia, H., Brozena, A.H., Piszkiewicz, S., Smith, A.E., Giovannini, I., Rebecchi, L., Pielak, G.J., Koshland, D., and Goldstein, B., 2017. Tardigrades use intrinsically disordered proteins to survive desiccation. Mol Cell 65, 975ñ984.</A></LI><LI><A href="https://www.cdc.gov/drugresistance/index.html"> Centers for Disease Control and Prevention, 2017. Antibiotic/Antimicrobial Resistance.</A></LI><LI><A href="http://www.gddiergezondheid.nl/diergezondheid/dierziekten/mastitis">GDDiergezondheid, 2017. Mastitis (uierontsteking).</A></LI><LI><A href="http://www.sciencemag.org/lookup/doi/10.1126/science.aam9321"> Gootenberg, J.S. et al., 2017. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science 356, 438ñ442. </A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210516/">Jia, B. et al., 2017. CARD 2017: Expansion and model-centric curation of the comprehensive antibiotic resistance database. Nucleic Acids Research 45(D1), D566ñD573.</A></LI><LI><A href="http://science.sciencemag.org/content/337/6096/816.long">Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J.A., Charpentier, E., 2012. A Programmable Dual-RNA-Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science 337, 816-821.</A></LI><LI><A href="http://dx.doi.org/10.1016/j.cell.2017.06.050"> Liu, L. et al., 2017. The molecular architecture for RNA-guided RNA cleavage by Cas13a. Cell 170, 714ñ726.e10.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697360/">McArthur, A.G. et al., 2013. The comprehensive antibiotic resistance database. Antimicrobial Agents and Chemotherapy 57, 3348ñ3357.</A></LI><LI><A href="http://dx.doi.org/10.1016/j.mib.2015.07.004"> McArthur, A.G. and Wright, G.D., 2015. Bioinformatics of antimicrobial resistance in the age of molecular epidemiology. Current Opinion in Microbiology 27, 45ñ50.</A></LI><LI><A href="https://www.nature.com/articles/nrmicro3525">Schwechheimer, C., and Kuehn, M.J., 2015. Outer-membrane vesicles from Gram-negative bacteria: biogenesis and functions. Nature Reviews: Microbiology 13, 605-619.</A></LI><LI><A href="https://www.nature.com/articles/s41598-017-05796-x">Sloan, D., Batista, A., and Loeb, A., 2017. The Resilience of Life to Astrophysical Events. Scientific Reports 7, 5419-5424.</A></LI><LI><A href="http://www.zuivelnl.org/wp-content/uploads/2016/09/Dutch-dairy-in-figures-2015.pdf"> Statistics Netherlands, 2015. Dutch dairy in figures. </A></LI><LI><A href="http://www.who.int/mediacentre/factsheets/antibiotic-resistance/en"> World Health Organization, 2016. Antibiotic resistance. </A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0092867415012003?via%3Dihub">Zetsche, B, Gootenberg, J.S., Abudayyeh, O.O., Slaymaker, I.M., Makarova, K.S., Essletzbichler, P., Volz, S.E., van der Oost, J., Regev, Aviv, Koonin, E.V., Zhang, F., 2015. Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System. Cell 163, 759-771. </A></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>