<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:TUDelft"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:TUDelft/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:TUDelft/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=TUDelft"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Case13a</TITLE><HEADER><DIV class="navbar-fixed"><NAV class="row cyan"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Description" class="cyan-text">Description</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Results" class="cyan-text">Results</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Demonstrate" class="cyan-text">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Notebook" class="cyan-text">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Protocols" class="cyan-text">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Safety" class="cyan-text">Safety</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Parts" class="cyan-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Basic_Parts" class="cyan-text">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Composite_Parts" class="cyan-text">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Part_Collection" class="cyan-text">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Improved_Part" class="cyan-text">Improved BioBrick</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Primers" class="cyan-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">Integrated Human Practices (IHP)</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Outreach" class="cyan-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Newsletter" class="cyan-text">Newsletter</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Meet_Up" class="cyan-text">European iGEM Meet Up</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Attributions" class="cyan-text">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Sponsors" class="cyan-text">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Crowdfunding" class="cyan-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Contribution-Overview" class="cyan-text">Contribution Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Interlab" class="cyan-text">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Plot_Grow_Curves" class="cyan-text">Plot Growth Curves</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" class="cyan-text">Find correct CRISPR targets</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Team" class="cyan-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Team#Supervisors" class="cyan-text">Supervisors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/TeamBuilding" class="cyan-text">Team Building</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Collaborations" class="cyan-text">Collaborations</A></LI></UL><UL id="dropdown-awards" class="dropdown-content nested menu-even-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software tool</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Measurement" class="cyan-text">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">IHP</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Modeling" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-fullmodel">Modeling</A></LI></UL><UL id="dropdown-fullmodel" class="dropdown-content nested"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate kinetics</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif Finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate Kinetics</A></LI></UL><UL id="dropdown-design" class="dropdown-content nested menu-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" class="cyan-text">Cas13a</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" class="cyan-text">Sample Preperation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#TDP" class="cyan-text">TDPs</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" class="cyan-text">Vesicles</A></LI></UL><UL id="dropdown-achievements" class="dropdown-content"><LI><A href="#!" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-awards">Special Tracks</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Medals" class="cyan-text">Medals</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Awards_Nominations" class="cyan-text">Awards &amp; Nominations</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2017.igem.org/Team:TUDelft" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Giant_Jamboree" class="waves-effect waves-light block">Giant Jamboree</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A href="https://www.facebook.com/TUDelft.iGEM2017/" class="menu-hover waves-effect waves-light block"><SPAN> TUDelft.iGEM2017</SPAN></A></LI><LI title="Click to copy our phonenumber"><A onclick="copyToClipboard('+31 152 78 77 51')" class="menu-hover waves-effect waves-light block" id="safari-phone-hack"><SPAN style="overflow: hidden; display:inline-block;">  +31 152 78 77 51</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light cyan" id="menu-btn"><I class="material-icons">menu</I></A></HEADER><MAIN class="row project"><DIV class="col s12 m10 l8"><MENU id="tocc" class="hide-on-med-and-down"><H5>Design:</H5><UL><LI><A href="#Intro-scroll">Introduction</A></LI><LI><A href="#cas13a-scroll">Cas13a</A></LI><LI><A href="#sampleprep-scroll">Sample Preparation</A></LI><LI><A href="#coacervation-scroll">Visible Readout</A></LI><LI><A href="#TDP-scroll">TDPs</A></LI><LI><A href="#vesicles-scroll">Vesicles</A></LI></UL></MENU><P id="Intro-scroll" class="scrollspy">The main aim of our project is to develop a simple and reliable tool for the detection of antibiotic resistance genes from agricultural pathogens. This means finding both a detection tool and a way to make our project work outside lab conditions. In designing our project we adhered to three over-arching engineering principles: Our tool should be fast and accurate, deployable on-site and have a visible readout. We aimed to fulfill these principles by using the Cas13a protein as our RNA detection tool and developing a readout method visible by eye through a new technique, coined CINDY Seq. Additionally, we equipped our system with a long shelf life by employing Tardigrade-specific intrinsically Disordered Proteins (TDPs). To make optimal use of synthetic biology we envision providing an easy purification method by letting the cell produce vesicles containing both proteins. We divided our project into five different modules which were designed to tackle the different aspects.</P><H3 id="cas13a-scroll" class="scrollspy">Cas13a - Accurate detection of specific genes</H3><P>The recently characterized protein Cas13a is an RNA-guided, RNA-targeting protein from the CRISPR-Cas system (<A href="#references">Gootenberg et al. 2017</A>; <A href="#references">East-Seletsky et al. 2016</A>). The protein has a very important feature: once it finds its target, it undergoes a conformational change and from then on, engages in collateral cleavage. In this state, it cleaves all RNA that it encounters unspecifically. This property allows amplification of an otherwise small signal and can easily be translated into a detectable read-out.</P><P>
                The CRISPR guide RNA (crRNA) is an  RNA that CRISPR uses as a guide to find its target, and is easily interchanged. Adequate crRNA design is important to allow the detection of specific antibiotic resistance genes and its variants with a single crRNA, while not yielding false positive results on different antibiotic resistance gene types. Furthermore, we derived six crRNA requirements, which we incorporate into a <A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" target="_blank"> spacer designer</A> that finds the best spacers to target specific groups of antibiotic resistance genes. We also testing these spacers with our <A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" target="_blank">Cas13a off-targeting prediction tool</A>. 
    In this module, we worked on getting an active Cas13a protein containing the desired crRNA that can accurately detect an RNA target of choice. The Cas13a proteins we purified are expressed from a plasmid (pC013) received from the authors of <A href="#references">Gootenberg et al. 2017</A>. We did this in parallel with the assembly of our own construct. The activity of Cas13a was tested with fluorescence activity assays, which are explained in the section Characterisation of Cas13a.
    </P><DIV class="col s12 center-margin"><H6><B>Figure 1: After binding to the target RNA, Cas13a will collaterally cleave RNA</B></H6></DIV><UL class="collapsible popout" data-collapsible="expandable"><LI><DIV class="collapsible-header">
                        BioBrick Design
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>In this section the design of our Cas13a construct is explained. The separate parts are explained in the following unfoldables. See Figure 2 for a schematic overview of the entire construct.</P><P class=""><B>Figure 2: Schematic overview of the most important features of our Cas13a BioBrick.</B> The backbone (<A href="http://parts.igem.org/Part:pSB4A5" target="_blank">pSB4A5</A>) consists of an pSC101 (yellow) origin and an Ampicillin resistant gene (light green). The Cas13a construct consists of the codon optimized LwCas13a gene for E. coli (orange) with a His tag and Twin strep tag for purification (pink). Downstream of the Cas13a gene is the CRISPR array with an adaptable spacer (blue) for the crRNA. </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                                    Coding sequence
                                </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P class="">
                                                For the Cas13a coding sequence we used the Cas13a ortholog of <I>Leptotrichia wadei</I> (LwCas13a) from the plasmid <A href="https://benchling.com/s/seq-66CfLwu7sLMQMbcXe7Ih/edit" target="_blank">pC013</A> described by <A href="#references">Gootenberg et al. 2017</A>. Since the gene was codon optimized for mammalian expression, we codon optimized the sequence for <I>E. coli</I> K12 expression using the online codon optimization tool from <A href="https://eu.idtdna.com/site/order/gblockentry" target="_blank">IDT</A>. Additionally, to make it a valid BioBrick, some forbidden restriction sites were removed by manually introducing silent mutations with the same tool. To determine whether the translation of the resulting gene would still produce the same protein, we did a protein blast of our gene and the gene from <A href="#references">Gootenberg et al. 2017</A> with the online <A href="http://web.expasy.org/translate/" target="_blank">Translate tool from Expasy</A> and the online <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch&amp;LINK_LOC=blasthome" target="_blank">Blast tool from NCBI</A>. A last modification was made: the TGA stop codon was changed into a double TAATAA stop codon. The LwCas13a and the purification tags are registered as basic part <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2306012" target="_blank">BBa_K2306012</A>.
                                            </P></DIV><DIV class="col s12"><P class=""><B>Figure 3: Schematic overview of the entire insert of the backbone. </B> This consists of the LwCas13a gene and the CRISPR array with adaptable spacer. </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                                    Promoter and Ribosome Binding Site
                                </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><H4>Promoter</H4><P>We replaced the lac promoter used by <A href="#references">Gootenberg et al. 2017</A> with lacUV5 (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_I732021" target="_blank">BBa_I732021</A>), a lac promoter inducible by IPTG commonly used in CRISPR research. The BioBrick of this promoter contains two nonsense regions, one on each side of the promoter (see Figure 3). One of these nonsense region contained a BsaI restriction site. However, BsaI is used in the Cas13a construct to replace the spacer sequence (see CRISPR array). Therefore a point mutation is introduced in the nonsense region to remove the BsaI restriction site in the promoter BioBrick. </P><H4>Ribosome Binding Site</H4><P>The ribosome binding site (RBS) that we used is a viral modified version of the Shine Dalgarno (SD) site, which has a high expression rate. The RBS is modified so that the expression rate is lowered. There is a 7 bp long spacer between the SD site and the start codon.
                                            </P></DIV><DIV class="col s12"><P><B>Figure 4: Schematic overview of the prefix (blue), promoter BioBrick (white) and the RBS (purple).</B></P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                                    Purification tags
                     </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>Before the gene sequence a His tag, a thrombin site, a strep tag and a SUMO cleavage site were added to enable purification (see Figure 4). These tags were also used by <A href="#references">Gootenberg et al. 2017</A> (<A href="https://benchling.com/s/seq-66CfLwu7sLMQMbcXe7Ih/edit" target="_blank">plasmid pC013</A>). The His-tag is used to purify the Cas13a protein. The thrombin site is a cleavage site that ensures the removal of the His tag once it is no longer necessary. If Cas13a is not pure enough after a His tag purification, the strep tag can be used for a second purification step. Finally, the SUMO cleavage site can be used to remove all tags if desired. We included BamHI restriction sites flanking the sequence for the tags, so that it can be removed if desired.
                                            </P></DIV><DIV class="col s12"><P><B>Figure 5: Schematic overview of the purification tags (His tag and Strep tag) and SUMO tag upstream of the Cas13a gene.</B></P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                                    CRISPR array and Terminator
                                </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><H4>CRISPR array</H4><P>The plasmid <A href="https://benchling.com/s/seq-Vyk8qK2fyhzegfNgLJHM/edit" target="_blank">pC011</A> from <A href="#references">Gootenberg et al. 2017</A> was used as a template for the CRISPR array. It consists of a spacer flanked by two direct repeats (DRs). The transcription is promoted by the constitutive promoter J23119 (<A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">BBa_J23119</A>). We optimised the spacer sequence to ensure that the spacer could be easily changed, to accommodate a different target. For this purpose we designed a “nonsense spacer”, which has two BsaI cutting sites, one at each end,with random nucleotides in between. In this way, the spacer can easily be removed by using the BsaI restriction enzyme creating overhangs which allow easy ligation of a new spacer sequence, with matching overhangs, into the array.
                                            </P><H4>Terminator</H4><P>For both the Cas13a gene and the CRISPR array a terminator was needed. The double terminator from iGEM (<A href="http://parts.igem.org/Part:BBa_B0014" target="_blank">BBa_B0014</A>), which contain a bidirectional terminator, was used for both parts.The CRISPR array was inverted so the double terminator would serve both the gene and the array.
                                            </P></DIV><DIV class="col s12"><P><B>Figure 6: Schematic overview of the CRISPR array downstream of the Cas13a gene, consisting of direct repeats (DR) (orange) and changeable spacer (dark blue).</B> The array is flipped so the same terminator (green) could be used as the gene.</P></DIV></DIV></DIV></LI></UL></DIV><LI><DIV class="collapsible-header">
                        crRNA design
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>We wrote a computer program that designs the crRNA spacer sequences according to six design principles to guide the Cas13a towards the desired target. Literature study revealed that potential crRNA targets are 1. 28 nucleotides in size, 2. may not contain a guanine (G) nucleotide next to the 3’ end of the target sequence and 3. preferably contain a 40-60% GC content (see our <A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" target="_blank">spacer design tool</A>). 4. Furthermore, the designed crRNA must match (i.e. be the reverse complement sequence) of the target. Moreover, 5. the designed crRNA must target a specific antibiotic resistance gene and its variants, 6. while not yielding off-target results on other genes. These requirements were incorporated into a <A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" target="_blank">off-targetting model</A> and a <A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" target="_blank">motif finder</A> that we employed to design 5 different crRNA spacer sequences that target a tetracycline resistance gene after amplification and in vitro transcription. The crRNA spacer sequences are thus the reverse complement of the in vitro transcribed RNA target to allow binding and subsequent activation of Cas13a. In addition, the first 30 nucleotides at the 5’ end of the crRNAs, preceding the spacer sequence, are identical in all crRNAs and form the hairpin that is unique to the Cas13a-variant we use.
                                </P><P>To save costs and allow the production of large amounts of crRNA, we designed and ordered DNA oligos containing a T7 promoter site (acquired from <A href="https://www.neb.com/faqs/2015/01/30/what-is-the-promoter-sequence-of-t7-rna-polymerase" target="_blank">NEB</A>) to allow in vitro transcription of the DNA oligos into the crRNAs. As such, we ordered suitable DNA oligos preceded by a T7 promoter, as schematically represented in Figure 6.
                                </P><P>After annealing of these sequences, T7 polymerase is added, allowing transcription of the DNA oligos into the desired crRNA sequences, see Figure 6 for a schematic overview.</P></DIV><DIV class="center-margin col m10 s12"><P><B>Figure 7: Schematic of gene detection with Cas13a.</B> The guide is in vitro transcribed crRNA, displaying 4 out of 6 design principles, while the remaining principles are relevant for targeting gene variants and preventing off-target activity. The target gene is amplified and a T7 promotor site is added. Subsequent in vitro transcription provides RNA target to which the crRNA, which is itself in vitro transcribed from two DNA oligos, must be complementary.
                                </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Characterisation of Cas13a
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>To characterize the activity of Cas13a RNase alert, cleavage assays will be employed. These assays are similarly used by <A href="#references">Gootenberg et al. 2017</A>. This method is based on the collateral cleavage activity of Cas13a after activation. RNase alert consists of both fluorophores and quenchers connected to RNA polymers in such a way that the quencher absorbs the emitted photons of the fluorophore. Once the RNA polymers are cleaved, the emitted photons by the fluorophore will no longer be absorbed by the quencher and the solution will fluoresce (<A href="#references">Thermo Fischer Scientific 2017</A>). The different crRNAs designed with our computational <A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" target="_blank">model</A> will be synthesized following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">crRNA synthesis protocol</A> and then be tested with these assays. This will give information on the efficiency of each of the crRNAs.
                              </P><DIV class="row align-center"><DIV class="col m6 s12"><P><B>Figure 8: Rotating surface image of Cas13a with crRNA bound.</B> Find a detailed description on our <A href="https://2017.igem.org/Team:TUDelft/Modeling#cas13a" target="_blank">model page</A>.</P></DIV><DIV class="col m8 s12"><P>One of our design requirements is that our final product, containing Cas13a, should be storable. For this purpose, Tardigrade Intrinsically Disordered Proteins (TDPs) will be used. These proteins give tardigrades the ability to survive desiccation by preserving protein activity. If Cas13a is dried in the presence of TDPs, it will have a longer shelf life. Cas13a will be tested after being dried with and without TDPs with fluorescent activity assays to test the activity and specificity of Cas13a after being dried and rehydrated. See the <A href="https://2017.igem.org/Team:TUDelft/Design#TDP" target="_blank">TDPs design page</A> for more information.
                               
Another design requirement of the final product is that it should give a visible readout to the naked eye. Therefore, a new detection method is designed that can translate the collateral cleaving of Cas13a into a visible output. This method uses the ability of long polymers to form coacervates in solution, see the <A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" target="_blank">Detection design page</A> for more information.
</P></DIV></DIV></DIV></DIV></DIV></LI><H3 id="sampleprep-scroll" class="scrollspy">Sample Preparation - From milk to RNA</H3><P>Because of interaction with different stakeholders, we chose to focus on a local problem: mastitis, a bacterial udder infection of the dairy cow (see our <A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" target="_blank">Integrated Human Practices</A>). For Cas13a to be able to detect antibiotic resistance genes, it needs an RNA target. This means we needed to get from an infected milk sample to RNA. To be able to detect not only active resistance genes but also the presence of the DNA encoding for antibiotic resistance, we put effort into the isolation of DNA and in vitro transcription into RNA. DNA isolation usually requires commercial kits, which are expensive, time-consuming and require specialized lab equipment. Furthermore, to be able to transcribe DNA into RNA, a T7-polymerase promoter site is required. This can be achieved by a DNA amplification step with specially designed primers. However, the commonly used Polymerase Chain Reaction (PCR) method requires a thermocycler. In this module, we focused on simplifying both the DNA isolation and the consecutive transcription step to provide a way to make lab access unnecessary without using expensive material. DNA can be isolated by either boiling the cells or by the use of a microwave described by <A href="#references">Dashti et al. 2009</A>. By centrifuging the samples, the cell debris is spun down and the DNA is left in the supernatant. We made two possible cheap centrifuges that will make DNA isolation possible at home or on a farm. Next, the DNA needs to be amplified and transcribed into RNA. Amplification of DNA is possible by Recombinase Polymerase Amplification (RPA). This isothermal alternative of a PCR reaction can be applied at constant temperature (optimal at 37-39 °C). The primers are designed such that a T7-polymerase promoter is added to the target-DNA in the RPA reaction. By adding T7-polymerase, the amplified target DNA can be transcribed into RNA. In this way, RNA can be synthesized at a constant temperature on-site, without using a thermocycler.</P><DIV class="col s12 center-margin"><P><CENTER><B>Figure 9: Schematic overview of sample preparation.</B></CENTER></P></DIV><UL class="collapsible popout" data-collapsible="expandable"><LI><DIV class="collapsible-header">
                        DNA isolation
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>
                                   We dove into literature and found that easy DNA extraction methods had been described by <A href="#references">Dashti et al. 2009</A>. Their research states that with the use of a microwave, you can burst open the cells. In the same way, boiling can be used to lyse the cells. By centrifuging it at 94 rcf, the cell debris forms a pellet and the DNA can be taken from the supernatant. We decided to try out both the boiling and the microwave method for DNA isolation.</P></DIV><DIV class="col s10 center-margin"><P><CENTER><B>Figure 10: Boiling milk samples for DNA isolation.</B></CENTER></P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Trial run of easy isolation method
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>
                                    To see if we can use the easy DNA extraction method described by <A href="#references">Dashti et al. 2009</A>, we repeated the control steps of the paper on a bacterial strain that was available in our lab at that time (KEIO-strain). To test for the presence of our DNA after the extraction step, we designed primers for PCR that anneal on the Kanamycin resistance gene. Our designed primers (<A href="https://2017.igem.org/Team:TUDelft/Primers">IG0014 and IG0015</A>) were 20 bp long,  had a GC content of 50 %, melting temperatures of 62 °C and 61 °C and the PCR product was expected to be 750 bp. To test if our PCR-mixture contained our expected product we did a digestion with BsrBI, to obtain DNA-fragments of sizes approximately 250 bp and 500 bp.</P></DIV></DIV><BLOCKQUOTE class="no-quote">Forward primer: CGTGGTGGTTGAGAAAGATC  Reverse primer: CAGTCATAGCCGAATAGCCT</BLOCKQUOTE></DIV></LI><LI><DIV class="collapsible-header">
                        DNA amplification and transcription
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>The amplification and transcription of the target DNA at a constant temperature was explained by <A href="#references">Gootenberg et al. 2017</A>. We used Recombinase Polymerase Amplification (RPA) to amplify a specific part of a gene at constant temperature (without the use of a thermocycler), as schematically depicted in Figure 7. To be able to transcribe the amplified fragment into RNA, a T7-polymerase binding site needed to be added to the amplified DNA-fragments. For this purpose, special primers were designed.</P></DIV><DIV class="col s12 center-margin"><P><B>Figure 11: Schematic of RPA and in vitro transcription with primers with T7 polymerase tail.</B></P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Trial run of easy DNA amplification and transcription
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12">
    
                               As a first step towards making our design work for real world applications, we tried our sample preparation with an antibiotic more common in the lab environment. As a proof of concept, we decided to try and detect tetracycline resistance in <I>E.coli</I>. Using the NCBI Primer-BLAST tool, we designed the primers (<A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0016 and IG0017</A>) for amplification and in vitro transcription of the tetracycline resistance gene. We used the default settings, only changing amplification product size (between 100 and 140 bp), primer melting temperatures (between 54 °C and 67 °C), and primer size (between 30 and 35 bp) as described by (<A href="#references">Gootenberg et al. 2017</A>). On the forward primer of the resulting primer pair, we added the T7-promoter sequence and on the 5’ end some arbitrary bases so that the RPA enzyme has enough space to bind. 
    
                            </DIV></DIV><BLOCKQUOTE class="no-quote">Forward primer: AATTCTAATACGACTCACTATAGGGATGCCCTTGAGAGCCTTAAC  Reverse primer: CCTCGCCGAAAATGACCCA </BLOCKQUOTE></DIV></LI><LI><DIV class="collapsible-header">
                        Demonstration of sample preparation in real-world applications
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>The difference between the previously explained detection of tetracycline resistance and a real world application like mastitis is that with a biological sample, you do not know the resistance (and its sequence) beforehand. Therefore, it is important that our system can detect many different antibiotic resistance genes. However, these genes convey a lot of variations, meaning that the number of primers and crRNAs required would be endless. We need to be able to find conserved regions within each antibiotic resistance group. In this way, we can tackle multiple variations in one go. We went to our modelers and they came up with a motif finder, with which we can find those conserved regions. According to veterinarian experts (<A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated">see Integrated Human Practices page</A>), detecting multi resistance genes such as <I>bla</I>Z and <I>mec</I>A would be most relevant. We used the <A href="https://2017.igem.org/Team:TUDelft/Model#motiffinder" target="_blank">motif finder</A> to find conserved regions within all the variations of the <I>bla</I>Z gene. Based on these regions, we were able to design primers for the RPA-reaction. We used the NCBI Primer-BLAST tool with some modified parameters (as described under “Trial run of easy DNA amplification and transcription”) to design primers (<A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0088 and IG0089</A>) that now amplified the <I>bla</I>Z gene instead of tetracycline. </P></DIV></DIV><BLOCKQUOTE class="no-quote">Forward primer:  AATTCTAATACGACTCACTATAGCTTCTAGAAATGATGTTGCTTTTGTTTATCCTAAG  Reverse primer: TTATCATTTGGCTTATCACTTTTATTGTCTTTATT</BLOCKQUOTE><DIV class="col s12 center-margin"><P><B>Figure 12: Schematic overview of our software tool.</B> First we extracted the <I>bla</I>Z gene family from the NCBI database. Secondly, our newly developed software tool aligned the genes and screened for conserved regions. These conserved regions served as templates to design primers that could be used to amplify part of <I>bla</I>Z genes.</P></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Design of the kit
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>We aimed to make all our steps as easy as possible, because the end user (a farmer) does not have access to specialized lab facilities and equipment. It is very important that the end-user is provided with all the necessary devices and materials. To make our kit accessible and user-friendly, we decided to buy all the materials in the local shopping centre instead of buying it from brands specialized in lab facilities. Products in stores are already developed to be user-friendly and usable by everyone.</P></DIV><DIV class="col s8 center-margin"><P><B>Figure 13: Team member shopping for materials at local stores</B></P></DIV><DIV class="col s12"><P>With a thermos flask and an immersion heater, water can be boiled. With a meat thermometer, the temperature of the water can be put to any desired temperature and with the thermos flask, kept around the same temperature for a few hours.</P></DIV><DIV class="col s12"><P><B>Figure 14: (Left) Thermos flask with immersion heater. (Right) Thermosflask with thermometer.</B></P><P>For the DNA isolation, a centrifuge step is necessary to spin down the cell debris. The centrifuge needs to go as slow as 94 rcf. For the DNA amplification and transcription, a centrifuge is used to spin down the cells. Centrifuges are mostly only found within lab areas and are quite expensive because they can reach high speeds. As these high speeds are not needed for our tool, we decided to put some effort into making a centrifuge ourselves. </P><P><B>Hard drive centrifuge:</B></P><P>We had the hard drive of an old computer lying around. As a hard drive has the ability to spin, we thought we might be able to employ it as a centrifuge with a few modifications. We used the tutorial posted by GaudiLabs to build a centrifuge ourselves. After connecting the hard drive short circuit to the power supply (connect the green wire to the black, this can easily be done with a paperclip) the hard drive began to spin. </P></DIV><DIV class="col s6 center-margin"><P><B>Figure 15: Connecting the hard drive short circuit to the power supply, by connecting the green wire to the black with a paperclip.</B></P></DIV><DIV class="col s12"><P><B>Hand-powered centrifuge:</B></P><P>The hard-drive centrifuge is electricity powered. For on-site detection, it would be a great advantage if we could make our centrifuge decentralized and battery-free. Inspired by the ‘paperfuge’ of <A href="#references">Bhamla et al. 2016</A>, we decided to design a similar centrifuge. To be able to fix tubes in the frame, we made it from hard-plastic instead of paper. The plastic can fit four 0.5 mL tubes and has a diameter of 8 cm.  </P></DIV><DIV class="col s6 center-margin"><P><B>Figure 16: Team member making our hand powered centrifuge.</B></P></DIV></DIV></DIV></LI></UL><H3 id="coacervation-scroll" class="scrollspy">CINDY Seq - Generating a visible readout</H3><P>We want to offer the perspective that the result of our synthetic biology research will be available for household usage. To achieve this, a detection method was developed that is cheap, safe and easy to use for all end-users. We set out to invent a method to detect the presence of very specific RNA sequences in a biological sample and came up with a method that meets all these requirements. The method, coined CINDY Seq (Coacervate Induced Nucleotide Detection of Your Sequence) makes use of a biochemical phenomenon called coacervation. Long negatively charged polymers including RNA can, in combination with positively charged polymers, phase separate into polymer-rich regions called coacervates. These coacervates increase the turbidity of the solution to a point at which it can be seen by the naked eye. When Cas13a engages in collateral cleavage upon target recognition, it will degrade all long RNA, thereby preventing coacervation and will become clear. If the Cas13a is not activated (that is, it does not find its RNA target), the solution will remain turbid. In the presence of the target gene, the solution will be transparent.</P><P>
    To achieve experimental proof of principle, experiments were designed and separated into three parts: formation and visualization of coacervates, proof of principle with a non-specific RNase and the proof of principle with Cas13a.</P><DIV class="col m4 s12 center-margin"><H6><B>Figure 17: Our detection method is based on the formation of coacervates.</B></H6></DIV><UL class="collapsible popout"><LI><DIV class="collapsible-header">Principle of CINDY Seq</DIV><DIV class="collapsible-body"><P>Solutions containing mutually attracting polymers can phase separate into polymer-rich regions, coined coacervates, and polymer-poor regions if the right conditions (e.g. pH, temperature, salinity) are provided. These coacervates can be seen by the naked eye as the solution becomes turbid. The increased turbidity is caused by the fact that the dense coacervates absorb and scatter light in the visible spectrum. Many types of coacervates with different polymers have been demonstrated, but for our purposes it is most important that (negatively charged) RNA is able to coacervate with positively charged polymers (<A href="#references">Aumiller et al. 2016</A>; <A href="#references">Aumiller &amp; Keating 2015</A>). Coacervate formation is critically dependent on the chain length of the polymers: only ‘long’ polymers form coacervates (<A href="#references">Qin et al. 2014</A>; <A href="#references">Qin &amp; De Pablo 2016</A>; <A href="#references">Veis 2011</A>). To learn why this is the case, check out our coacervation modelling page. The fact that only long polymers (of RNA in our case) coacervate, and that activated Cas13a degrades RNA forms the basis of our novel detection method.</P><DIV class="row"><DIV class="col s10 center-margin"><P><B>Figure 18: Schematic description of the Coacervate Induced Nucleotide Detection of Your Sequence (CINDY Seq) method.</B> The method works as follows: Cas13a with a guide of choice and some long RNA (collateral RNA) are mixed in an appropriate buffer. Then, the biological sample of interest is added to this mix. Only when this sample contains the target RNA, will the Cas13a be activated and will engage in collateral cleavage and degrade all the long collateral RNA. Without activation of Cas13a, the collateral RNA remains intact. After sufficient time waiting for this reaction to occur (~an hour), a positively charged polymer is added to the mix. The collateral RNA will only coacervate with the positively charged polymer if it still has its original length. Since coacervates are visible to the naked eye, observing the turbidity of the solution directly yields the answer to the question: “was the target RNA inside in our sample?”
    </P></DIV><P>To achieve experimental proof of principle, experiments were designed and separated into three parts: formation and visualization of coacervates, proof of principle with a non-specific RNase and the proof of principle with Cas13a. On the <A href="https://2017.igem.org/Team:TUDelft/Results#coacervation">results page</A> the experiments are described in more detail.</P></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Experiment design</DIV></LI><LI><B>Formation and visualization of coacervates</B><P> Following previous work on coacervation of RNA we decided to make coacervates from polyuridylic acid (polyU) and spermine, a tetravalent polycation at physiological pH, following (<A href="#references">Aumiller et al. 2016</A>). We made attempts to optimize the conditions so that solutions with coacervates are optimally visible by the naked eye, while limiting the required materials.</P></LI><LI><B>Proof of principle with a non-specific RNase</B><P>To show that the cleavage of RNA can be visualized to the naked eye, we showed that polyU/spermine coacervates can be dissolved by addition of RNase A. This RNase degrades RNA by cleaving the 3’OH-phosphate of pyrimidine nucleotides (<A href="#references">Sigma Aldrich 2017</A>), and thus is also able to cleave polyU.</P></LI><LI><B>Proof of principle with activated Cas13a</B><P>The last and most important part of this module was to show that the coacervation method works with activated Cas13a as nuclease. As described in the Cas13a design page, this requires Cas13a, a crRNA as guide and an RNA target.</P></LI></UL><H3 id="TDP-scroll" class="scrollspy">TDPs - Designing for on-site usage</H3><P>Another part of providing a system suitable for household usage is ensuring that it can easily be transported and stored without impeding its functionality. Tardigrades, also known as water-bears, are micro-animals able to survive a wide range of extreme conditions, amongst which is complete dehydration. Recently, a group of Tardigrade-specific intrinsically Disordered Proteins (TDPs) was found to improve desiccation tolerance in other organisms as well as in non-native proteins (<A href="#references">Boothby et al. 2017</A>). By using this property, we aimed to simplify the storage of Cas13a and reduce the necessary reagents in our detection tool, so that the protein could be dried and stored at room temperature. This would enable using our detection system on-site and consequently fulfill our goal in accordance with the aforementioned design principles.</P><P>Based on the characterisation by <A href="#references">Boothby et al. 2017</A> we chose four TDPs to preserve Cas13a:  two <U>c</U>ytosolic and two <U>s</U>ecretory <U>a</U>bundant <U>h</U>eat-<U>s</U>oluble proteins (CAHS and SAHS, respectively). To be able to optimise the production of TDPs for the different stages of our project, we designed our gene fragments in a modular way that allowed switching promoters fairly easily. The purified proteins were confirmed by SDS-PAGE and mass spectrometry. To characterize the TDPs’ effect on protein activity after desiccation and subsequent rehydration, we adapted the lactate dehydrogenase (LDH) activity assay previously carried out by <A href="#references">Boothby et al. 2017</A>. In later stages, we dried TDPs with Cas13a and confirmed its activity with an RNase alert assay.</P><DIV class="col m9 s12 center-margin"><H6><B>Figure 19: TDPs will preserve the activity of Cas13a after drying</B></H6></DIV><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Choosing the proteins
                    </DIV><DIV class="collapsible-body"><P>Desiccation tolerance can be conferred by multiple substances, among which are intrinsically disordered proteins. Tardigrade-specific intrinsically disordered proteins (TDPs) are a set of proteins found in tardigrades, that lack a tertiary structure. They are produced by the tardigrade upon drying and are capable of forming a glass-like matrix that protects cellular components within the micro-animal (<A href="#references">Sloan et al. 2017</A>). Two families of TDPs, cytosolic and secretory abundant heat-soluble proteins (CAHS and SAHS, respectively), were identified to be responsible for tardigrade’s desiccation tolerance by <A href="#references">Boothby et al. 2017</A>. They were named after their occurrence in the tardigrades: while CAHS proteins were most commonly found in the cytosol, SAHS proteins were found in the culture medium during the desiccation process (<A href="#references">Yamaguchi et al. 2012</A>). </P><P>Our choice of proteins was inspired by the characterization of <A href="#references">Boothby et al. 2017</A>. They performed experiments to observe the influence of disrupting the function of several genes encoding for TDPs on the survival of tardigrades under dry and hydrated conditions. We chose the three proteins with the highest impact on survival rate for our project: CAHS 94205, SAHS 68234 and SAHS 33020.</P><P>Additionally, the authors transformed a collection of CAHS proteins into a BL21 (DE3) strain and observed the increased cells’ desiccation tolerance. The fourth protein we chose, CAHS 106094, improved the desiccation tolerance of bacteria by almost a hundred percent (<A href="#references">Boothby et al. 2017</A>). 
    
    </P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Plasmid design
                    </DIV><DIV class="collapsible-body"><P>To be able to adjust the promoter for different purposes as required in our project, we designed our BioBrick in two parts, which can be easily linked via Gibson Assembly (<A href="#references">Gibson, 2009</A>). One part contains the gene coding sequence in combination with a strong ribosome binding site, preceded by a spacer sequence. The ribosome binding site consists of a g10-L sequence upstream of a Shine-Dalgarno sequence (<A href="#references">Olins and Rangwala, 1989</A>). The other part features a promoter followed by a spacer sequence identical to that of the previous part. An overview of the design can be found in Figure 11. Thanks to this design, different promoter-inducer combinations could be easily assembled before the coding sequences and be tested throughout our project, thus enabling an optimisation of protein expression.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 20: SnapGene overview of promoter and gene designs for the promoter P<SUB>BAD</SUB> and the gene for the tardigrade-specific protein CAHS 94205.</B></P></DIV></DIV><P>We prepared the three different promoters lacI, p<SUB>BAD</SUB> and T7. While the T7-promoter was to be used in the first stages of the project for protein purification, we planned on using either LacI or pBAD for integration of multiple plasmids into one organism. Since the ultimate goal to our project was the manufacture of a cell featuring the developed parts for the production of vesicles, Cas13a and TDPs; we prepared both the lacI and PBAD promoters in advance, as it remains unknown whether the different promoters involved can be separately induced in the same cell. </P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Experiment design
                    </DIV><DIV class="collapsible-body"><P>After the isolation of the different TDPs according to the protocol described by <A href="#references">Boothby et. al. 2017</A>, the proteins were identified by SDS-PAGE and mass spectrometry. To assess their influence on desiccation tolerance of proteins, we chose to first use an established protocol by <A href="#references">Goyal et al. 2005</A>, in which the protective effect is quantified as a function of the preservation of enzymatic activity in lactate dehydrogenase (LDH). <A href="#references">Boothby et al. 2017</A> previously reported that the activity of the enzyme lactate dehydrogenase (LDH) was conserved by TDPs after desiccation and subsequent rehydration. When LDH reduces pyruvate to lactate, it oxidizes NADH to NAD+, which can be observed by measuring the absorbance of the sample at 340 nm (<A href="#references">Moran et al. 1996</A>). To characterize the TDPs, LDH was dried with the four different TDPs at concentrations ranging from 0.1 g/L to 1 g/L. </P><P>Moreover, the stabilization of LDH was studied by long-term assays, in which TDP-LDH solutions were dried and stored at room temperature. Both freshly prepared LDH solutions, as well as TDP-LDH solutions, were stored over time at 4  °C to be used as controls. </P><P>After this first set of experiments, we continued drying Cas13a with TDPs and compared the changes in Cas13a activity with an RNase alert assay similar to that used to assess Cas13a's activity (<A href="https://2017.igem.org/Team:TUDelft/Design#cas13a">see Cas13a design section</A>).</P></DIV></LI></UL><H3 id="vesicles-scroll" class="scrollspy">Vesicles - Exploring the cell's potential as a (future) factory</H3><P>In the mind of synthetic biology we decided to pursue the vision of letting our bacteria produce all our necessary proteins at once and package them in Outer Membrane Vesicles (OMVs), which presents one way of achieving a responsible, cell-free system. After separating the vesicles from the cells, we would be left with little parcels containing all our system components in one go. This way, we can exploit our bacteria as little cell factories, resulting in an elegant synbio purification method (Schwechheimer and Kuehn, 2015). </P><P>It has been shown that under specific conditions, some bacteria produce OMVs. This secretory process has been linked to endogenous stress (see the first unfoldable!). These OMVs can be engineered to package proteins that were transported into the periplasm. Previous work has shown that mutations in the genes encoding protein complexes that stabilize the bacterial cell wall (e.g. the Tol-Pal complex) destabilize the outer membrane. In strains with these mutations,  strongly increased OMV production (hypervesiculation) is observed. Previous iGEM teams already made BioBricks to achieve this and we improved them by further characterization. Transporting proteins to the periplasm is possible with several pathways and methods, however, after considering the pros and cons we decided to use the twin arginine translocation (Tat) pathway in our design. In order to determine whether Cas13a and TDPs could be transported to the periplasm, we designed an assay using GFP as a primary experiment.</P><DIV class="col m8 s12 center-margin"><H6><B>Figure 21: Illustration of TDPs being encapsulated into vesicles</B></H6></DIV><UL class="collapsible popout" data-collapsible="expandable"><LI><DIV class="collapsible-header">
                        Hypervesiculation
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>
                                    Engineering vesicle production has become a profound technique in the past years for specific applications such as drug delivery (<A href="#references">Allen et al. 2013</A>). The production of outer membrane vesicles has been achieved by mutating genes of the five cell envelope proteins encoded by TolA, TolQ, TolR, TolB and Pal comprising the Tol-pal complex, shown in Figure 12. These proteins anchor in the outer and inner membrane, forming a bridge which regulates the outer membrane integrity. The most studied mutation is the overexpression of the TolR gene, which has shown to increase vesicle production in <I>E. coli</I> (<A href="#references">Baker et al. 2014</A>).</P></DIV><DIV class="col s10 center-margin"><P><B>Figure 22: Illustration of Tol-pal complex.</B> This protein complex comprises the proteins TolA, TolQ, TolR, TolB and Pal, which anchor the outer and inner membrane through the peptidoglycan (PG) layer (<A href="#references">Scwechheimer et al. 2015</A>).</P></DIV><P>
                                Last year, the iGEM team of the University of New South Wales (UNSW) <A href="https://2016.igem.org/Team:UNSW_Australia/Description">Australia 2016</A> implemented several mutations in an <I>E.coli</I> strain and assessed which combinations had the most favourable impact on hypervesiculation. Furthermore, they made a <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2022002">BioBrick</A> to overexpress the TolR gene and observed an increase in vesicle production with sizes ranging from 80 to 115 nm in diameter (d.nm). While this modification already causes the strain to hypervesiculate, they discovered that adding a knockout of the TolA gene destabilizes the cell envelope, resulting in an even higher vesicle production with sizes ranging from 130 to 300 d.nm.
                            </P></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Experimental Design hypervesiculation
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>
                                    Before performing assays to characterize hypervesiculation in our design, we first wanted to know whether the vesicles are big enough to encapsulate multiple Cas13a proteins together with TDPs. With the first <A href="https://2017.igem.org/Team:TUDelft/Model#ratekinetics">modeling</A> of this part, we determined that Cas13a is bigger than TDPs, which was why we used Cas13a to make a rough estimation on how many of these proteins could fit into vesicles. Furthermore, we made a <A href="https://2017.igem.org/Team:TUDelft/Model#ratekinetics">model</A> to estimate how many proteins could fit in vesicles of different sizes. With the model, we calculated that vesicles with a minimum size of 80 d.nm would be able to contain roughly 20 Cas13a proteins. Bigger vesicles of 300 d.nm would encapsulate around 1000 Cas13a proteins. With this in mind, we performed several assays to characterize hypervesiculation. These assays were carried out with the plasmid pTolR (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2022002">mutation in the TolR gene</A>) and the knockout TolA <I>E.coli</I> BW25133 (Keio collection, see <A href="#references">Baba et al. 2006</A>). We used different combinations of the plasmid and the strain to achieve accurate and extensive characterization, as depicted in Figure 13. Additionally, we performed experiments using plasmids with and without inserts in the Keio strain (KEIO), as a control. As blank for the experiment, we used the parental strain from the Keio collection (WT). To verify vesicle production we utilized three different methods: (1) Dynamic Light Scattering (DLS) to detect small particles in a solution, (2) Transmission Electron Microscopy (TEM) to verify whether the particles measured by DLS were, in fact, vesicles and (3) Fluorescence measurements using a plate-reader to detect the fluorescence of membrane dye FM4-64 to quantify the vesicle concentration</P></DIV><DIV class="col s12 center-margin"><P><B>Figure 23: Overview of different plasmids that were transformed into the parental strain of the Keio collection is denoted as WT, and the Keio collection itself is abbreviated to KEIO.</B> TolR-pET-Duet is the plasmid with the insert that overexpresses protein TolR which resuslts in hypervesiculation of the  bacteria. pET-Duet is the backbone in which TolR was placed into. </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Transport to the periplasm
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P> Exporting proteins to the periplasm has already been researched in the iGEM competition. There are several methods in <I>E.coli</I> for transporting proteins into the periplasm before they can be encapsulated into OMVs. Several transport mechanisms exist that facilitate the export of either folded or unfolded proteins, and typically they require an export tag. There are different export pathways, but the most common ones are the twin arginine translocation (Tat) pathway and the general secretory (Sec) pathway (<A href="#references">Baker et al. 2014</A>). In addition to these two, there is another method with which one can encapsulate the protein of interest into vesicles. In this method, the protein is attached to a surface protein with a so-called Ice Nucleation Protein (INP) (<A href="#references">Van Blooise et al. 2011</A>). </P><H3>
                                    Tat Pathway
                                </H3><P>
                                    The Tat pathway transports folded proteins across the cytoplasmic membrane into the periplasm. This pathway, shown in Figure 14, consists of three components: TatA, TatB and TatC. The transport tag, which is located on the N terminus of the protein sequence, will interact with the initial docking site TatC, after which TatB will act as a mediator and translocate the protein to TatA. TatA will form a ring structure to make a pore through which the protein can be translocated. Passage through the cytoplasmic membrane into the periplasm occurs after polymerization of TatA. Once in the periplasm, the tag will be cleaved off. A key characteristic of this pathway is that the pore sizes of TatA range from 100 kDa to over 500 kDa (<A href="#references">Oates et al. 2004</A>). The proteins that we want to transport into the periplasm are Cas13a and TDPs. Cas13a is a recently characterized protein where the exact size of the protein was still unknown. Before making the decision which pathway to use, we started the <A href="https://2017.igem.org/Team:TUDelft/Model#ratekinetics">modeling</A> by determining the size of Cas13a, which was 140 kDa. For TDPs, with the sequence of the proteins and the aid of an online <A href="https://web.expasy.org/protparam/">tool</A> we calculated that the size of the proteins are roughly 26 kDa.
                                </P></DIV><DIV class="col s12 center-margin"><P><B>Figure 24: Schematic outline of the Tat-pathway.</B> (1) The tag at the N terminus of the protein will dock at site TatC. (2) TatB, which forms a complex with TatC, will act as a mediator and translocate protein P to TatA. TatA forms a ring structure which functions as a pore through the cytoplasmic membrane. (3) Protein P will translocate through TatA pore. (4) When translocated into the periplasm, the tag at the N terminus will be cleaved off, therefore releasing protein P in the periplasm.</P></DIV><DIV class="col s12"><H3>SEC pathway </H3><P>
                                    The Sec pathway, depicted in Figure 15, is utilized by cells to translocate unfolded proteins (pre proteins) into the periplasm.There are several mechanisms that prevent the folding of the protein (<A href="#references">Schierle et al. 2003</A>). However, transporting the preprotein generally occurs with the three components: SecA, SecB and SecYEG. SecB will bind to the preprotein after its translation in the cytoplasm. This protein is a sec system-specific chaperone which mediates the preprotein to SecA. SecA, in return, binds strongly to a SecA specific binding site where the NH2 end of the preprotein will bind to the SecYEG and prepare it for translocation. The final step is the translocation of the preprotein through a pore formed by the heterotrimeric SecYEG complex where the folding of the protein will occur in the periplasm (<A href="#references">Mori et al. 2001</A>). As the environment of the cytoplasm is different to that of the periplasm, folding of the peptides might be different depending on these conditions. With a complicated protein such as Cas13a, this transport method might not be ideal, since it may alter the folding of the protein.
                                </P></DIV><DIV class="col s12 center-margin"><P><B>Figure 25: Illustration of the Sec pathway.</B> (1) Sec B binds to the preprotein to guide it to SecA, (2) when bound to SecA the NH2 end of the preprotein will bind to SecYEG. (3) Subsequently, the preprotein will be translocated into the periplasm. (4) Once in the periplasm, the NH2 end will be cleaved off whereafter the entire preprotein can move to the periplasm. (5) Here the protein will undergo its posttranslational folding (<A href="#references">Mori et al. 2001</A>).</P></DIV><DIV class="col s12"><H3>INP </H3><P>
                                    Another possible method is to make use of ice nucleation protein (INP). This carrier protein functions as an anchoring motif which displays our protein of interest on the surface of the outer membrane as is illustrated in Figure 16. INP has a hydrophobic N-terminal domain which tethers the protein to the outer membrane and a hydrophilic C-terminal domain which is exposed to the medium. Expressing this protein at a high level appears to decrease the viability of the bacteria (<A href="#references">Van Blooise et al. 2011</A>). This method, combined with hypervesiculation, would result in the proteins of interest being attached to the surface of the vesicles. The downside of this method is that the protein of interest will remain attached to the surface of the vesicle. This anchoring to the surface might affect the activity of the protein of interest.
                                </P></DIV><DIV class="col s8 center-margin"><P><B>Figure 26:  Illustration of INP: a schematic of INP tethering into the outer membrane on the N-terminal domain for protein P.</B> Here INP functions as anchoring motif for protein P (<A href="#references">Van Blooise et al. 2011</A>). </P></DIV><DIV class="col s12"><P>In short, we were doubtful about the folding of the proteins in the periplasm in the SEC pathway due to the difference in composition of the cytoplasm and periplasm and effects on the protein activity. The INP pathway seemed a reasonable method to transport the protein out of the cytoplasm by anchoring it onto the surface of the vesicles. However, this method is likely to affect the activity of the protein when still attached to the surface of the vesicles. For these reasons, we chose to use the Tat pathway to translocate proteins from the cytoplasm across the inner membrane and into the periplasm. In addition, this pathway was already assessed in combination with GFP (<A href="#references">Van Blooise et al. 2011</A>). It remains to be tested whether our protein Cas13a can be translocated into the periplasm. However, our modeling indicated that Cas13a is small enough to be transported through the pores, so theoretically this should be possible. The iGEM team from the Norwegian University of Science and Technology (NTNU) <A href="https://2013.igem.org/Team:NTNU-Trondheim">Trondheim</A> of 2013 also refer to the above mentioned paper as a reference and tried to reproduce the results by fusing the transport tag with GFP and RFP. We built on their <A href="http://parts.igem.org/Part:BBa_K1082001">work</A> by making our own <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2306000">BioBrick</A> in which we fuse the transport tag with GFP to transport it into the periplasm and eventually into the vesicles as a proof of concept of our design.
                                </P></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">
                        Experimental Design translocation of protein
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P>
                                    With the fusion protein TorA-GFP, we wanted to evaluate whether the folded GFP is transported into vesicles. We combined the basic part encoding TorA-GFP sequence <A href="http://parts.igem.org/Part:BBa_K2306000">(BBa_K230600)</A> with a LacI promoter (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A>), a ribosome binding site (<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>) and a double terminator (<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>) to control its expression. Figure 17 visualizes the combination of GFP-TA in WT and in KEIO. Here we expect that most of the GFP will be translocated to the periplasm and in KEIO strain into vesicles. Before starting with experiments, we wanted to <A href="https://2017.igem.org/Team:TUDelft/Model#ratekinetics">model</A> when the GFP concentration in the periplasm reaches its maximum. For this, we needed the OD curve of both strains. For the characterization of TorA-GFP, we assessed the presence of GFP in the periplasm with an osmotic shock assay. Here, we expected that there is more GFP in the periplasm than in the cytoplasm when the transport tag is attached to the protein. In addition, we also visualised this observation with Widefield Microscopy. Subsequently, we checked for the localisation of GFP in vesicles by transforming the fusion TorA-GFP into the hypervesiculating strain with a plate reader.
                                </P></DIV><DIV class="col s12 center-margin"><P><B>Figure 27: Overview of the parental strain of Keio (WT) and the Keio strain (KEIO) and GFP-TA.</B> In combination with the GFP-TA plasmid, there will be more GFP fluorescence in the periplasm in WT. In KEIO the GFP fluorescence from the periplasm will go into the vesicles. </P></DIV></DIV></DIV></LI></UL><DIV id="references-scroll" class="section card"><DIV class="card-content"><SPAN class="card-title">References:</SPAN><OL><LI><A href="https://ac.els-cdn.com/S0169409X12002980/1-s2.0-S0169409X12002980-main.pdf?_tid=45dd1140-bdc1-11e7-84c6-00000aab0f6b&amp;acdnat=1509402525_439c97a9df333bfbc44c0571b7bda414">Allen, T.M., &amp; Cullis, P.R., 2013. iposomal drug delivery systems: From concept to clinical applications. Adv Drug Deliv Rev. 65(1), pp 36-48. </A></LI><LI><A href="http://pubs.acs.org/doi/abs/10.1021/acs.langmuir.6b02499">Aumiller, W.M. et al., 2016. RNA-Based Coacervates as a Model for Membraneless Organelles: Formation, Properties, and Interfacial Liposome Assembly. Langmuir, 32(39), pp.10042–10053.</A></LI><LI><A href="https://www.nature.com/articles/nchem.2414?WT.ec_id=NCHEM-201602&amp;spMailingID=50516411&amp;spUserID=ODkwMTM2NjM5NgS2&amp;spJobID=842812717&amp;spReportId=ODQyODEyNzE3S0">Aumiller, W.M. &amp; Keating, C.D., 2015. Phosphorylation-mediated RNA/peptide complex coacervation as a model for intracellular liquid organelles. Nature Chemistry, 8(2), pp.129–137</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1681482/">Baba, T., et al, 2006. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol. 2.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0958166914000494">Baker, J.L. et al, 2014. Microbial biosynthesis of designer outer membrane vesicles. Curr Opin Biotechnol. 29, pp 76-84.</A></LI><LI><A href="https://www.nature.com/articles/s41551-016-0009?error=cookies_not_supported">Bhamla, M.S., Benson, B., Chai, C., Katsikis, G., Johri, A., and Prakash, M. Hand-powered ultralow-cost paper centrifuge. Nature Biomedical Engineering 1, 1-7.</A></LI><LI><A href="https://ac.els-cdn.com/S016777991000199X/1-s2.0-S016777991000199X-main.pdf?_tid=9cab875a-bdc0-11e7-8cc2-00000aab0f26&amp;acdnat=1509402242_3d6f00d6a7b875e8429d9f1523c73f60">van Blooise, E., et al, 2011. Decorating microbes: surface display of proteins on Escherichia coli. Trends Biotechnol. 29(2), pp 79-86.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pubmed/28306513"> Boothby T.C., Tapia H., Brozena A.H., Piszkiewicz S., Smith A.E., Giovannini I., Rebecchi L., Pielak G.J., Koshland D. and Goldstein B. 2017. Tardigrades use intrinsically disordered proteins to survive desiccation. Mol Cell 65, 975–984.e5.</A></LI><LI><A href="http://www.academia.edu/14010095/Heat_Treatment_of_Bacteria_A_Simple_Method_of_DNA_Extraction_for_Molecular_Techniques">
Dashti, A. A., Jadaon, M. M., Abdulsamad, A. M., &amp; Dashti, H. M., 2009. Heat Treatment of Bacteria: A Simple Method of DNA Extraction for Molecular Techniques. Kuwait Medical Journal 41, 117–122.</A></LI><LI><A href="https://www.nature.com/articles/nature19802">East-Seletsky, A. et al., 2016. Two distinct RNase activities of CRISPR-C2c2 enable guide-RNA processing and RNA detection. Nature, 538(7624), pp.270–273.</A></LI><LI><A href="https://www.nature.com/protocolexchange/protocols/554#/procedure">Gibson, D. 2009. “One-step enzymatic assembly of DNA molecules up to several hundred kilobases in size”, Protocol Exchange.</A></LI><LI><A href="http://science.sciencemag.org/content/356/6336/438">Gootenberg, J.S. et al., 2017. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, 356(6336), pp.438–442.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pubmed/15631617"> Goyal, K., Walton,  L.J. and Tunnacliffe, A. 2005. “LEA proteins prevent protein aggregation due to water stress”. Biochem. J., 388, pp. 151–157</A></LI><LI><A href="http://www.gaudi.ch/GaudiLabs/?page_id=328"> Hard drive Centrifuge, Gaudilabs (2017) </A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1868505/pdf/mmi0063-0545.pdf">McBroom, A.J., Kuehn, M.J., 2007. Release of outer membrane vesicles by Gram-negative bacteria is a novel envelope stress response. Mol Microbiol. 63(2), pp 545:558.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pubmed/8872918"> Moran Jeffery H, Schnellmann Rick G (1996)  “A Rapid beta-NADH-Linked Fluorescence Assay for Lactate Dehydrogenase in Cellular Death”. Journal of Pharmacological and Toxicological Methods 36, 41-44.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0966842X01021746/pdfft?md5=575c247c486b0e9fde30afcb9b66162b&amp;pid=1-s2.0-S0966842X01021746-main.pdf ">Mori, H., &amp; Ito, K., 2001. The Sec protein-translocation pathway. Trends Microbiol. 9(10), pp 494-500.</A></LI><LI><A href="https://ac.els-cdn.com/S0022283604015062/1-s2.0-S0022283604015062-main.pdf?_tid=82498688-bdbf-11e7-9df9-00000aacb360&amp;acdnat=1509401773_fefdbaeb8bee1ad8d503edc4e29628e0">Oates, J., et al, 2004. The Escherichia coli twin-arginine translocation apparatus incorporates a distinct form of TatABC complex, spectrum of modular TatA complexes and minor TatAB complex. J Mol Biol. 346(1), pp 295-305.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pubmed/2676996"> Olins, P. O. and S. H. Rangwala (1989). &quot;A novel sequence element derived from bacteriophage T7 mRNA acts as an enhancer of translation of the lacZ gene in Escherichia coli.&quot; Journal of Biological Chemistry 264(29): 16973-16976.</A></LI><LI><A href="http://pubs.acs.org/doi/abs/10.1021/mz500190w">Qin, J. et al., 2014. Interfacial tension of polyelectrolyte complex coacervate phases. ACS Macro Letters, 3(6), pp.565–568.</A></LI><LI><A href="http://pubs.acs.org/doi/abs/10.1021/acs.macromol.6b02113">Qin, J. &amp; De Pablo, J.J., 2016. Criticality and connectivity in macromolecular charge complexation. Macromolecules, 49(22), pp.8789–8800.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC193964/pdf/0521.pdf">Schierle, C.F., et al, 2003. The DsbA signal sequence directs efficient, cotranslational export of passenger proteins to the Escherichia coli periplasm via the signal recognition particle pathway. J Bactoriol. 185(19), pp 5706-13.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5308417/pdf/nihms778256.pdf ">Schwechheimer, C., &amp; Kuehn, M.J., 2015. Outer-membrane vesicles from Gram-negative bacteria: biogenesis and functions. Nat Rev Microbiol. 13(10), pp 605-9</A></LI><LI><A href="http://www.sigmaaldrich.com/catalog/product/sigma/r6513?lang=en&amp;region=NL">Sigma Aldrich, 2017. Ribonuclease A from bovine pancreas. Available at: http://www.sigmaaldrich.com/catalog/product/sigma/r6513?lang=en&amp;region=NL [Accessed October 13, 2017].</A></LI><LI><A href="">Singh, Reema, Akansha Saxena, and Harpreet Singh. &quot;Identification of group specific motifs in Beta-lactamase family of proteins.&quot; Journal of biomedical science 16.1 (2009): 109.</A></LI><LI><A href="https://www.nature.com/articles/s41598-017-05796-x">Sloan, David; Batista, Alves; Loeb, Abraham. 2017. &quot;The Resilience of Life to Astrophysical Events&quot;. Scientific Reports. 7 (5419). doi:10.1038/s41598-017-05796-x. </A></LI><LI><A href="https://www.thermofisher.com/order/catalog/product/AM1964">Thermo Fischer Scientific, 2017. RNaseAlert Lab Test Kit. Available at: https://www.thermofisher.com/order/catalog/product/AM1964 [Accessed October 23, 2017]</A></LI><LI><A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2001.02253.x/epdf">Thomas, J.D., et al, 2001. Export of active green fluorescent protein to the periplasm by the twin-arginine translocase (Tat) pathway in Escherichia coli. Mol Microbiol. 39(1), pp 47-53. </A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0001868611000169?via%3Dihub">Veis, A., 2011. A review of the early development of the thermodynamics of the complex coacervation phase separation. Advances in Colloid and Interface Science, 167(1–2), pp.2–11.</A></LI><LI><A href="https://doi.org/10.1371/journal.pone.0044209">Yamaguchi A., Tanaka S., Yamaguchi S., Kuwahara H., Takamura C., et al. 2012. Two Novel Heat-Soluble Protein Families Abundantly Expressed in an Anhydrobiotic Tardigrade. PLOS ONE 7(8): e44209.  </A></LI></OL></DIV></DIV></MAIN></DIV><FOOTER><H4 class="center">We thank our sponsors:</H4><DIV class="sponsors center"><DIV><A href="https://www.tudelft.nl/en/faculty-of-applied-sciences/">Faculty of Applied Sciences</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>