<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Model_Lattice_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Model/Lattice Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:TUDelft"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:TUDelft/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:TUDelft/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=TUDelft"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Case13a</TITLE><HEADER><DIV class="navbar-fixed"><NAV class="row cyan"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Description" class="cyan-text">Description</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Results" class="cyan-text">Results</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Demonstrate" class="cyan-text">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Notebook" class="cyan-text">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Protocols" class="cyan-text">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Safety" class="cyan-text">Safety</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Parts" class="cyan-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Basic_Parts" class="cyan-text">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Composite_Parts" class="cyan-text">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Part_Collection" class="cyan-text">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Improved_Part" class="cyan-text">Improved BioBrick</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Primers" class="cyan-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">Integrated Human Practices (IHP)</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Outreach" class="cyan-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Newsletter" class="cyan-text">Newsletter</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Meet_Up" class="cyan-text">European iGEM Meet Up</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Attributions" class="cyan-text">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Sponsors" class="cyan-text">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Crowdfunding" class="cyan-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Contribution-Overview" class="cyan-text">Contribution Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Interlab" class="cyan-text">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Plot_Grow_Curves" class="cyan-text">Plot Growth Curves</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" class="cyan-text">Find correct CRISPR targets</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Team" class="cyan-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Team#Supervisors" class="cyan-text">Supervisors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/TeamBuilding" class="cyan-text">Team Building</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Collaborations" class="cyan-text">Collaborations</A></LI></UL><UL id="dropdown-awards" class="dropdown-content nested menu-even-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software tool</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Measurement" class="cyan-text">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">IHP</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Modeling" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-fullmodel">Modeling</A></LI></UL><UL id="dropdown-fullmodel" class="dropdown-content nested"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate kinetics</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif Finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate Kinetics</A></LI></UL><UL id="dropdown-design" class="dropdown-content nested menu-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" class="cyan-text">Cas13a</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" class="cyan-text">Sample Preperation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#TDP" class="cyan-text">TDPs</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" class="cyan-text">Vesicles</A></LI></UL><UL id="dropdown-achievements" class="dropdown-content"><LI><A href="#!" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-awards">Special Tracks</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Medals" class="cyan-text">Medals</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Awards_Nominations" class="cyan-text">Awards &amp; Nominations</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2017.igem.org/Team:TUDelft" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Giant_Jamboree" class="waves-effect waves-light block">Giant Jamboree</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A href="https://www.facebook.com/TUDelft.iGEM2017/" class="menu-hover waves-effect waves-light block"><SPAN> TUDelft.iGEM2017</SPAN></A></LI><LI title="Click to copy our phonenumber"><A onclick="copyToClipboard('+31 152 78 77 51')" class="menu-hover waves-effect waves-light block" id="safari-phone-hack"><SPAN style="overflow: hidden; display:inline-block;">  +31 152 78 77 51</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light cyan" id="menu-btn"><I class="material-icons">menu</I></A></HEADER><MAIN class="row"><DIV class="col s12 m10"><H1>Model - Lattice Model</H1><P>One of the goals of our project was providing our detection tool with a long shelf-life under simple storage conditions. For this reason, we investigated the use of Tardigrade-specific intrinsically Disordered Proteins (TDPs) to preserve Cas13a activity upon desiccation and storage at room temperature. These TDPs originate from the tardigrade, an extremely resilient micro-animal (<A href="#references" target="_blank">Sloan et al. 2017</A>), and mediate the tardigrade’s desiccation tolerance. The mechanism behind this has been hypothesized to be a vitrification process (<A href="#references" target="_blank">Boothby et al. 2017</A>). By forming a glass-like matrix upon desiccation, TDPs protect cellular structures and proteins within the cells from degradation. As hydrophilic, disordered proteins, they can form dynamic scaffolds upon desiccation, while they have mutual repulsive interactions.</P><P>This matrix formation implies that during dehydration, the TDPs rearrange themselves around, in our case, Cas13a, in a process that could also be compared to phase separation. To simulate the dynamics of this phase separation, we decided to utilize a lattice model, which is well established for modelling these kinds of problems. With our model, we were able to determine which of the four TDPs CAHS 94205, CAHS 106094, SAHS 33020 and SAHS 68234 was most suitable for preserving Cas13a activity after drying and storage at room temperature. Additionally, we studied the TDP behaviour at different concentrations and estimated in what concentrations TDPs have to be used for drying.</P><DIV class="row"><DIV class="center-align"><A class="btn" href="https://static.igem.org/mediawiki/2017/a/a0/TDPLatticeModel.zip" style="color: white">DOWNLOAD THE LATTICE MODEL SCRIPT</A></DIV></DIV><H3>Proteins in the lattice model
            </H3><P>To keep our model simple, we wanted to take only next neighbour interactions on the lattice into account. Therefore, to be sure that no long-range interactions were neglected in the model, we calculated the Debye length, the length over which electrostatic interactions decline exponentially, with equation (debyelength)
            </P><P>$$\lambda_D = \sqrt{\frac{\epsilon_0 \cdot\epsilon_r \cdot k_B\cdot T}{2\cdot N_A\cdot e^2 \cdot I}}\tag{1}$$</P><P>where $\epsilon_0$ is the vacuum permittivity, $\epsilon_r$ is the relative permittivity, $k_B$ the Boltzmann constant, $N_A$ the avogadro constant and I the ionic strength of the solution (<A href="#references" target="_blank">Russel and Saville 1989</A>). With an electrolyte containing Tris-HCl (c = 27 mM) and NaCl (c = 24 mM), we found that the Debye length of the solution we were drying is 0.0024 nm. Since this length is smaller than the average size of an amino acid, we were free in selecting the grid size of the lattice model.
            </P><P>To effectively model protein-protein interactions, rather than interactions between individual amino acids, we decided to discretize our proteins into blocks of five amino acids per unit.</P><P>
                As it is not straightforward to manually derive a protein model using protein structure files, we wrote a program that converts the commonly used &quot;protein database&quot; file extension (.pdb) for Cas13a into a 'LEGO-block' model (in the form of a matrix) (Figure 1). The hydrophobicity and charge (isoelectric point) scales applied (<A href="#references" target="_blank">Biro, 2006</A>) can be easily adapted.
            </P><DIV class="row"><DIV class="col s12 center-margin"><DIV class="col s10 center-margin"><P><B>Figure 1: LEGO-block model of Cas13a to input into the lattice model.</B> A) Model of Cas13a hydrophobicity distribution. For visualization purposes, a colour threshold was applied: red areas are hydrophobic while grey areas are not. B) Model for Cas13a charge (isoelectric point) distribution. For visualization purposes, a colour threshold was applied: blue patches are positively charged while gray patches are not charged or negatively charged.
                        </P></DIV></DIV></DIV><P>
                We provide the Matlab <A href="https://static.igem.org/mediawiki/2017/0/02/T--TUDelft--PDB_for_Lattice_Model.zip">script</A>, enabling anyone with basic Matlab knowledge to import a structure, which can be converted to a coarse-grained 'LEGO-block' model with a resolution that can be set by the user. In addition, the code intuitively includes the desired hydrophobicity and charge scales to consider, which can be adapted as desired.
            </P><P>However, since TDPs are intrinsically disordered proteins, this method could not be applied to them. Thus, we simply settled on averaging the properties of five neighbouring amino acids in the sequence, instead of in space. Similar to protein folding, there are two main driving forces behind protein-protein interactions which are electrostatic and hydrophobic interactions (<A href="#references" target="_blank">Vagedes et al. 2002</A>). To be able to calculate the interaction energies between different protein units, the hydrophobic moment and isoelectric point (<A href="#references" target="_blank">Biro 2006</A>) of each amino acid were averaged for each unit during the discretization.

            </P><H3>Generating the simulation environment
            </H3><P>Besides the discretizations of the proteins used in the lattice model, several other aspects were simplified. In the model, we wanted to observe the interaction of a single Cas13a protein with TDPs at a certain concentration. To realize this, we fixed Cas13a at a specific point in the simulation environment and employed periodic boundary conditions, emulating a larger volume occupied by Cas13a and TDPs.
            </P><P>To generate the simulation environment, the concentration of Cas13a was set to 0.5 g/L, the concentration of Cas13a isolated with our <A href="https://2017.igem.org/Team:TUDelft/Protocols">protein purification protocol</A>. For the planned Cas13a drying assays, 4 µL Cas13a would be dried in a final volume of 100 µL with a given concentration of TDP. From a given TDP concentration and its molar mass, we could then calculate the number of TDPs that needed to be generated to obtain the right ratio of TDPs to Cas13a.
            </P><P>Subsequently, we calculated an average volume occupied by one amino acid of 128,50
                Å^3 from the volumes of individual amino acid residues (<A href="#references" target="_blank">Counterman and Clemmer 1999</A>) and multiplied this by the number of amino acids in one unit cell to determine its volume. This volume was then used to calculate the lattice dimensions for the model. Since we were investigating a drying process, we opted to study phase separation behaviour at volume fractions of the initial reaction volume. With preliminary runs we determined that a volume fraction of 0.1 was still large enough to reasonably assume that no protein degradation had occurred, but also small enough to allow for efficient simulation.
            </P><H3>Interaction Energies
            </H3><P>Finally, to be able to model the protein-protein interactions, we also needed to assign energies to different protein conformations. As we were studying both electrostatic and hydrophobic interactions, we wanted to calculate energy values based on both types. While a variety of hydrophobicity scales exist, they usually only describe relative hydrophobicity between amino acids (<A href="#references" target="_blank">Wimley and White 1996, Wolfenden et al. 1981, Rose et al. 1985</A>). While actual free energy values were estimated for hydrophobic interactions between amino acid residues (<A href="#references" target="_blank">Miyazawa and Jernigan 1985</A>), they are based on the distance between amino acid residues observed in protein x-ray structures. This means that they are not applicable for the units in our models, which contain five amino acid residues each.
            </P><P>For this reason we decided to use the hydrophobic compatibility index (HCI) established by (<A href="#references" target="_blank">Biro 2006</A>), along with the charge compatibility index (CCI) described in the same paper for consistency. Both indices give a maximal index of 20 for favourable interactions and a minimal index of 1 for unfavourable interactions. To convert these indices into energies, we subtracted 10 from the indices and multiplied by -1 to assign negative energies (-10 - 0) to favourable and positive energies (0 - 10) to unfavourable interactions. Furthermore, electrostatic interactions have been found to have a slightly higher free energy contributions than hydrophobic interactions, 1.4 kcal/mol (<A href="#references" target="_blank">Berg 2002</A>) versus 0.6 kcal/mol for small (36 residues) and 1.6 kcal/mol for large (341 residues) proteins (<A href="#references" target="_blank">Pace et al. 2011</A>). Since TDPs have ~210-230 residues, we estimated a hydrophobic interaction energy to be 0.9 kcal/mol. Thus, we came up with the following equations for hydrophobic and electrostatic interaction energies scaled to their free energy contributions.
            </P><P>
                $$ E_{HCl} = -\left( 10 - \left| [HM(A) - HM(B)]\frac{19}{10.6}\right| \right)\frac{0.9}{1.4} \tag{2} $$ $$ E_{CCL} = - \left(1- [PI(A) - 7][PI(B)-7]\frac{19}{33.8} \right) \tag{3}$$
            </P><H3>Model Conditions</H3><P>In the simulation, Cas13a stays fixed at the center. However, a range of movements is possible for the TDPs surrounding it as illustrated in the figures below . Both movements of single units, at the end or the middle of a chain, and movements of multiple units including, snake movements, rotations of chain ends, and translations are possible.
            </P><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 2:</B> 2D illustration of the process behind the movement of a single unit at the end of a chain. Possible new positions of the unit are determined and one is selected at random.
                    </P></DIV><DIV class="col m6 s12"><P><B>Figure 3:</B> 2D illustration of the process behind the movement of a single unit in the middle of a chain. Possible new positions of the unit are determined by overlaying the possible open sites of its neighbours and one is selected at random.
                    </P></DIV></DIV><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 4:</B> 2D illustration of the process behind a snake movement. A possible new position of one end of the chain is determined and the following units moved along the chain.
                    </P></DIV><DIV class="col m6 s12"><P><B>Figure 5:</B> 2D illustration of a rotational movement. A unit in the chain and on end of the chain is picked and the angle and axis (in 3D) by which the chosen part moves is generated randomly.
                    </P></DIV></DIV><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 6:</B> 2D illustration of a translation move with a shift of (-1, -2). A chain is picked and moved by a randomly generated shift.

                    </P></DIV></DIV><P>The simulation generates random moves and calculates the energy difference between the new and old chain conformation. As stated earlier, only nearest neighbour interactions are taken into account for this.
            </P><P>We employ the metropolis monte carlo algorithm in the simulation, accepting and rejecting moves with a probability of:</P><P>This prevents the system from falling into a local energy minimum, because it allows moves with unfavourable interactions. The probability of accepting unfavourable interactions decreases exponentially for interaction energies higher than the system’s energy. The value of</P><P> represents the energy present in the system and was chosen to be 1/20 for our simulation, after several trial runs.
            </P><H3>Results</H3><P>With our lattice model, we wanted to study the interactions between TDPs and Cas13a upon drying. For this purpose, we first had to assess if the model we were using was suitable for modelling protein interactions. Afterwards, we investigated which TDP would confer desiccation tolerance to Cas13a the best, and at which concentration we should use our TDPs.
            </P><H3>Modelling General Protein Interactions
            </H3><P>In our lattice model, we convert the hydrophobicity and charge compatibility indices established by <A href="#references" target="_blank">Biro 2006</A>. We tested if the generated energy values had the expected impact on the protein behaviour, by running the model without any favourable interactions, by generating only positive energy values (0-20), and with exclusively favourable interactions, by generating only negative energy values (-20-0) between the chains. By observing the behaviour of the chains after a given number of iterations, we confirmed that the model can provide information on the behaviour of polymers in solution.
            </P><P>In Figure 7 below, the polymer arrangement does not change significantly over 50000 iterations. While the chains have changed positions and it is clear that movements were generated, the conformations and the localization over the environment are similar. On the other hand, Figure 8 clearly shows a clustering of chains after the same amount of iterations, which would be expected if all chain interactions are favourable. As the energy input in both cases yields the expected results, we can conclude that the lattice model can be used to simulate the phase separation of polymers in solution and that it can be used to model the interactions between Cas13a and TDPs.
            </P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 7:</B> Two states of a simulation without favourable protein interactions. To the left, the initial state of the simulation is shown immediately after it is generated, to the right, the environment is shown after 50 000 iterations.
                    </P></DIV></DIV><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 8:</B> Two states of a simulation with exclusively favourable protein interactions. To the left, the initial state of the simulation is shown immediately after it is generated, to the right, the environment is shown after 50 000 iterations. Clusters of TDPs can be recognized, confirming the hypothesized protein-protein interactions.
                    </P></DIV></DIV><H3>Preservation of Cas13a activity by different types of TDPs</H3><P>To determine which TDP preserves Cas13a activity upon drying best, we investigated the behaviour of CAHS 94205, CAHS 106094, SAHS 330 20 and SAHS 68234 with Cas13a at a concentration of 1 g/L and a volume fraction of 0.1, since <A href="#references" target="_blank">Boothby et al. 2017</A> showed that LDH activity was already well preserved at a TDP concentration of ~ 1 g/L. The simulations were run over 300000 iterations. One of these simulations (SAHS 33020, 1 g/L, volume fraction 0.1) is shown in Figure 9.
            </P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 9:</B> Simulation of Cas13a with SAHS 33020 at a concentration of 1 g/L and in a volume fraction of 0.1.
                    </P></DIV></DIV><P>Both the migration of proteins towards Cas13a and formation of protein clusters is visible. However, protein clusters that form further away from Cas13a tend to remain in place and attract other chains. This is also visible in Figure 10, which plots the number of SAHS 33020 proteins at different distances from Cas13a over the number of iterations. During the simulation the area close to Cas13a clearly attracts other proteins as visible in the lower bottom part of the figure. However, while there is a general movement towards Cas13a, tend to stay in clusters for an extended amount of time, as is for example visible at around 150000 iterations. To observe migration of more TDPs towards Cas13a, the simulation would have to be run for a longer amount of time.</P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 10:</B> Number of SAHS 33020 proteins at different distances from Cas13a over 300000 iterations. Data was extracted from a Simulation of Cas13a with SAHS 33020 at a concentration of 1 g/L and in a volume fraction of 0.1.
                    </P></DIV></DIV><P>To determine which TDP would preserve Cas13a activity best, we analyzed two parameters: the coverage of Cas13a during the lattice model simulation (see Figure 11) and the development of the energy over time (see Figure 12).</P><P>The coverage of Cas13a is measured by counting the number of immediate neighbours to all units in the Cas13a protein. If the protein is covered with TDPs more quickly, they can form the glass-like matrix supporting its structure earlier, preventing degradation of the protein. In Figure 11, the number of immediate neighbours of Cas13a is plotted against the number of iterations per chain. All TDPs display similar development of coverage, but SAHS 33020 and CAHS 94205 display a steeper slope than the other proteins in their respective families and a higher coverage at 800 iterations per chain. This suggests that they are better suited for drying Cas13a.
            </P><DIV class="row"><DIV class="col m7 s12 center-margin"><P><B>Figure 11:</B> Number of immediate neighbours to Cas13a against the number of iterations per chain for TDPs CAHS 94205, CAHS 106094, SAHS 33020 and SAHS 68234 at a concentration of 1 g/L and in a volume fraction of 0.1.

                    </P></DIV></DIV><P>This result is partly reflected in the interaction energies calculated in the different TDP simulations. A steeper slope of the energy graph indicates that the phase separation occurs faster and that the steady state is reached at an earlier stage. Thus, Figure 12 implies that the two SAHS proteins and out of these SAHS 33020 would confer the highest desiccation tolerance to Cas13a. However, when comparing within the protein families, CAHS  904205 and SAHS 33020 show the best results again.
            </P><DIV class="row"><DIV class="col m7 s12 center-margin"><P><B>Figure 12:</B> Normalised energy of the system against the number of iterations per chain for TDPs CAHS 94205, CAHS 106094, SAHS 33020 and SAHS 68234 at a concentration of 1 g/L and in a volume fraction of 0.1.
                    </P></DIV></DIV><P>While the energy and Cas13a coverage development showed slightly different results, we decided to use CAHS 94205 and SAHS 33020 to dry Cas13a in
                <A href="https://2017.igem.org/Team:TUDelft/Results#TDP" target="_blank">further experiments</A>, to have TDPs of both families to test in the lab. Additionally, we experimentally determined that these two proteins also had the largest effect on desiccation tolerance of LDH, which further supported our choice. All in all however, we expect that SAHS 33020 performs best in Cas13a drying assays.
            </P><H3>Preservation of Cas13a activity at different concentrations</H3><P>As CAHS 94205 and SAHS 33020 were chosen to dry Cas13a, we simulated the behaviour of the two proteins at concentrations ranging from 0.1 to 1 g/L. An example on how the difference in concentration affects the number of chains is shown in Figure 13 for SAHS 33020. We again chose a volume fraction of 0.1 and analyzed the development of Cas13a coverage (see Figure 14) as well as the energy over time (see Figure 15).
            </P><DIV class="row"><DIV class="col m10 s12 center-margin"><P><B>Figure 13:</B> Initial state of lattice model simulation for SAHS 33020 concentrations ranging from 0.1 to 1 g/L at a volume fraction of 0.1.
                    </P></DIV></DIV><P>In Figure 14 the number of direct neighbours to Cas13a is plotted against the TDP concentration at the same time in each simulation. <A href="#references" target="_blank">Boothby et al. 2017</A> had previously observed a logarithmic behaviour of LDH activity towards TDP concentrations as high as 10 g/L. For this reason, we fitted a logarithmic curve over all points with function (4).</P><P>$$ y = m\cdot \ln(x) +b\tag{4} $$</P><P>The coefficient of determination of the resulting curve is between 0.7 and 0.8 for both fits, indicating that it complies with the data reasonably well. The system might not have reached its saturation point, but the curve evens out towards higher TDP concentrations as would be expected. This means that, up to a concentration of 1 g/L at least, a higher Cas13a coverage can be achieved by increasing the TDP concentration. Better preservation of Cas13a activity may be possible to obtain with higher TDP concentrations, but our results indicate that the difference in the resulting desiccation tolerance will become less pronounced at higher concentrations. This complies with observations by <A href="#references" target="_blank">Boothby et al. 2017</A>, who observed that the curve was flattening out towards the saturation point after around 1 g/L.
            </P><DIV class="row"><DIV class="col m10 s12 center-margin"><P><B>Figure 14:</B> a) Number of direct neighbours to Cas13a over a) CAHS 94205 and b) SAHS 33020 concentrations ranging from 0.1 to 1 g/L at 800 moves per chain at a volume fraction of 0.1. A logarithmic function was fitted to the data. The resulting R^2 of the fits is between 0.7 and 0.8 for both.
                    </P></DIV></DIV><P>The normalised energy of CAHS 94205 and SAHS 33020 over concentration shown in Figure 15, shows a similar trend of a steeper slope and therefore better protection of Cas13a for higher TDP concentrations. Furthermore, we could observe in the development of the system’s energy over time, that the energy of SAHS 33020 simulations generally displayed a steeper slope than that of CAHS 94205 simulations for all concentrations except 0.1 g/L. This implies again, that of all TDPs SAHS 33020 is best suited to preserve Cas13a activity upon drying. It also means that SAHS 33020 will likely show better activity preservation at lower concentrations than CAHS 94205.
            </P><DIV class="row"><DIV class="col m10 s12 center-margin"><P><B>Figure 15:</B> a) Normalised energy of the system against the number of iterations per chain for simulations with concentrations ranging from 0.1 to 1 g/L for a) CAHS 94205 and b) SAHS 33020 at 800 moves per chain and in a volume fraction of 0.1.

                    </P></DIV></DIV><P>While we cannot give recommendations for higher concentrations, it appears that a concentration of at least 1 g/L of either CAHS 94205 or SAHS 33020 is advisable for drying Cas13a. However, SAHS 33020 seems overall best suited for drying Cas13a and may even confer enough desiccation tolerance to Cas13a at lower concentrations. This is important to keep undesired side effects, such as the activation of Cas13a through TDPs as observed in the experiments, at a minimum.
            </P><DIV id="#references-scroll" class="section card"><DIV class="card-content"><SPAN class="card-title">#References:</SPAN><OL><LI><A href="https://www.ncbi.nlm.nih.gov/books/NBK22567/" target="_blank">Berg JM, Tymoczko JL, Stryer L. Biochemistry. 5th edition. New York: W H Freeman; 2002. Section 1.3, Chemical Bonds in Biochemistry.</A></LI><LI><A href="https://tbiomed.biomedcentral.com/articles/10.1186/1742-4682-3-15" target="_blank">Biro JC. Amino acid size, charge, hydropathy indices and matrices for protein structure analysis. Theoretical Biology and Medical Modelling. 2006; 3:15. doi: 10.1186/1742-4682-3-15.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S1097276517301338" target="_blank"> Boothby TC, Tapia H, Brozena AH, Piszkiewicz S, Smith AE, Giovannini I, Rebecchi L, Pielak GJ, Koshland D and Goldstein B (2017) Tardigrades use intrinsically disordered proteins to survive desiccation. Mol Cell 65, 975–984.e5.</A></LI><LI><A href="http://pubs.acs.org/doi/abs/10.1021/ja984344p" target="_blank"> Counterman, A.E., Clemmer, D. E. Volumes of Individual Amino Acid Residues in Gas-Phase Peptide Ions, J. Am. Chem. Soc., 1999
                            </A></LI><LI><A href="http://go.galegroup.com/ps/anonymous?id=GALE%7CA3910629&amp;sid=googleScholar&amp;v=2.1&amp;it=r&amp;linkaccess=fulltext&amp;issn=00368075&amp;p=AONE&amp;sw=w&amp;authCount=1&amp;isAnonymousEntry=true" target="_blank"> G. Rose, A. Geselowitz, G. Lesser, R. Lee and M. Zehfus, Hydrophobicity of Amino Acid Residues in Globular Proteins, Science 229(1985)834-838.
                            </A></LI><LI><A href="http://pubs.acs.org/doi/pdf/10.1021/ma00145a039" target="_blank"> Miyazawa, S., Jernigan, R., Estimation of Effective Interresidue Contact Energies from Protein Crystal Structures: Quasi-Chemical Approximation, Macromolecules 1985, 18, 534.
                            </A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0006349502753113?via%3Dihub" target="_blank">P. Vagedes, W. Saenger, E.W. Knapp, Driving forces of protein association: the dimer-octamer equilibrium in arylsulfatase A, Biophys J, 83 (2002), pp. 3066-3078
                            </A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0022283611002324?via%3Dihub" target="_blank">Pace CN, et al. Contribution of hydrophobic interactions to protein stability. J. Mol. Biol. 2011;408:514–528. doi: 10.1016/j.jmb.2011.02.053.
                            </A></LI><LI><A href="http://pubs.acs.org/doi/pdf/10.1021/bi00507a030" target="_blank">R. Wolfenden, L. Andersson, P. Cullis and C. Southgate, Affinities of Amino Acid Side Chains for Solvent Water, Biochemistry 20(1981)849-855.
                            </A></LI><LI><A href="https://books.google.nl/books?hl=nl&amp;lr=&amp;id=3shp8Kl6YoUC&amp;oi=fnd&amp;pg=PR13&amp;dq=Russel,+W.B.,+Saville,+D.A.+and+Schowalter&amp;ots=bHg3oTr6l5&amp;sig=hPS-8IDFwWCdJ1Kyz0bemfhS-Lw#v=onepage&amp;q=Russel%2C%20W.B.%2C%20Saville%2C%20D.A.%20and%20Schowalter&amp;f=false" target="_blank">Russel, W.B., Saville, D.A. and Schowalter, W. R. Colloidal Dispersions, Cambridge University Press, 1989
                            </A></LI><LI><A href="https://www.nature.com/articles/s41598-017-05796-x.pdf" target="_blank">Sloan, D.; Batista, A.; Loeb, A. (14 July 2017). &quot;The Resilience of Life to Astrophysical Events&quot;. Scientific Reports. 7 (5419). doi:10.1038/s41598-017-05796-x. Retrieved 14 July 2017.
                            </A></LI><LI><A href="https://www.researchgate.net/publication/14369494_Wimley_W_C_White_S_H_Experimentally_determined_hydrophobicity_scale_for_proteins_at_membrane_interfaces_Nat_Struct_Biol_3_842-848" target="_blank">Wimley, W.C., White, S.H., Experimentally determined hydrophobicity scale for proteins at membrane interfaces, Nature structureal biology, 3, 10.
                            </A></LI></OL></DIV></DIV></DIV></MAIN><FOOTER><H4 class="center">We thank our sponsors:</H4><DIV class="sponsors center"><DIV><A href="https://www.tudelft.nl/en/faculty-of-applied-sciences/">Faculty of Applied Sciences</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>