<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Main-Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Main-Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col s12 m10"><P>From <A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" target="_blank">Integrated Human Practices</A> we learned that there is high demand for a simple, safe and cheap method by which farmers can test for antibiotic resistant pathogens on-site. Developing a method that meets these requirements was precisely the main goal of our project. Our method of detection includes Cas13a, which is able to recognise an RNA target and subsequently collaterally cleave unspecific RNA sequences. This collateral cleavage, allows the amplification of a small signal into a visible read out. Starting with a biological sample, we need to extract DNA. The target DNA needs to be amplified and transcribed into the target RNA. To be able to target as many variations of a certain gene at once, it is important to determine the conserved regions and target those. All in all, our developed method starts at the extraction of a biological sample and ends at the detection visible to the naked eye. All the design requirements are optimized to achieve a simple, safe and cheap product. This page is dedicated to show that we have, in fact, <B>achieved these goals</B> of detecting antibiotic resistance genes by naked eye. We have divided the full procedure in three steps:</P><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        1. In situ sample processing
                    </DIV><DIV class="collapsible-body"><P>We successfully adapted a straightforward <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">protocol</A> for extracting bacterial antibiotics resistance genes that can be performed without the use of specialized lab facilities and requires minimal lab skills. The DNA extraction is as simple as boiling the sample and spinning down the cell debris. The subsequent step, which is the amplification of the target DNA and conversion into RNA, can be done without the use of any advanced laboratory devices. We <A href="https://2017.igem.org/Team:TUDelft/Results#sampleprep" target="_blank">demonstrated</A> and <A href="https://2017.igem.org/Team:TUDelft/Applied_Design#validation" target="_blank">validated</A> that this protocol can be used successfully to prepare a sample that can be amplified to enhance reliability of the test.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 1: Overview of our easy sample preparation</B> DNA isolation by boiling and the subsequent DNA amplification and transcription into RNA without the use of any advanced laboratory devices.</P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        2. Target recognition
                    </DIV><DIV class="collapsible-body"><P>We developed a <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">software tool</A> to find conserved regions in the genes that are responsible for antibiotic resistance. We tested this software tool on the <I>bla</I>Z family (Figure 2) mostly <A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" target="_blank">relevant</A> for the udder-disease mastitis. These genes, originally from the pathogen <I>Staphylococcus aureus</I>, encode for β-lactamases that degrade the most commonly used antibiotics from the penicillin group, including the first choice amoxicillin, ampicillin and benzylpenicillin.
                             The newly developed software tool was able to predict primers (see <A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" target="_blank">design page sample prep</A>) that would allow amplification of a <I>bla</I>Z gene extracted from agricultural pathogens, without knowing the exact sequence of the gene. This was based on predicting conserved regions in the <I>bla</I>Z gene family. We have <A href="https://2017.igem.org/Team:TUDelft/Results#sampleprep" target="_blank">proven experimentally</A> that the primers designed using the software did in fact allow amplification of an unknown <I>bla</I>Z gene in plasmid DNA extracted from an actual agricultural pathogen.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 2: Our software tool finds conserved regions in the <I>bla</I>Z gene family</B> (A) Schematic overview of our software tool. First we extracted the <I>bla</I>Z gene family from the NCBI database (yellow arrow). Secondly, our newly developed software tool aligned the genes and screened for conserved regions (red arrow). These conserved regions served as templates to design primers (see <A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" target="_blank">design page sample prep</A>) that could be used to amplify part of blaZ genes (blue arrow). (B) On the left hand side of the arrow, the general structure of penicillin antibiotics is depicted. The R indicates the groups that distinguishes the unique members of this group. The <I>bla</I>Z gene family encodes proteins called β-lactamases that degrade the most frequently used members of the penicillin family.</P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">                      3. Detection
                    </DIV><DIV class="collapsible-body"><DIV class="row"><P>We invented a novel method that allows for a readout visible to the naked eye. This method, coined CINDY Seq (Coacervation Induced Nucleotide Detection of Your Sequence), eliminates the need of any expensive lab equipment and trained personnel required to analyse the output, as would be the case for fluorescence, for example. This readout was <A href="https://2017.igem.org/Team:TUDelft/Results#coacervation" target="_blank">successfully implemented</A> in combination with Cas13a, visualising the target recognition and consecutive collateral cleavage. This patent-pending method is based on the physical phenomenon of coacervation, and is more elaborately described on the <A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" target="_blank">detection design page</A>. Theoretical analysis and modelling has also given more insight in developing and optimizing this method, as is shown on our <A href="https://2017.igem.org/Team:TUDelft/Model#coacervation" target="_blank">modelling page</A>.</P><DIV class="center-margin col m5 s8"><P><B>Figure 3: Our readout visible to the naked eye.</B> (Left) target recognition, clear solution. (Right) no target recognition, turbid solution.</P></DIV></DIV></DIV></LI></UL><H3>Validation of the kit</H3><P>To conclude, we were able to develop a method to detect antibiotic resistance in an easy and cheap way with a signal that is visible to the naked eye. As our design was intended to work in the field, we even had our design checked by a potential end-user.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>