Team:TUDelft/Main-Applied Design
Applied Design:
Problem
Our vision
Current situation
Design
Our solution
Validation
Societal impact
Future vision
What is the problem?
Annually, over 400,000 US residents fall ill due to infections caused by foodborne antibiotic resistant bacteria. The antibiotic resistance in these bacteria typically originate from the misuse of antibiotics in agriculture. Unfortunately, in the animal sector antibiotics are still overused, accelerating the further development of antibiotic resistance ( Physicians Committee 2017 ).
How is it regulated now?
Currently, there are 3 strategies to combat antibiotic resistance: prevention, treatment and new therapeutics ( Netherlands Center for One Heath 2016 ). The problem with prevention strategies and new therapeutics is that they depend on very extensive clinical trials. These trials will take longer than the validation of a detection device, that can be employed for the treatment strategies. Therefore, we focused on developing a tool that could contribute to frontline diagnostics and new treatment methods to stop the increasing use of antibiotics, which we exploit to fight antibiotic resistance in agriculture ( van Mierlo 2016 ).
Our vision
In our project, we developed a versatile, RNA-based detection tool that can reveal antibiotic resistance genes in bacteria. Once you know that a bacterium is resistant to a certain antibiotic, you can develop a more responsible treatment plan, excluding the antibiotics that you know for sure will not work. This approach reduces the development of more antibiotic resistance, thus contributing to the fight against this huge problem.
Which infection?
In the Netherlands, the dairy industry contributes to a large amount of the total revenue of the Dutch food production  ( Statistics Netherlands (CBS) 2015 ). Through our interactions with farmers, veterinarians and health and safety experts we found out that the most common disease in the dairy industry today is mastitis (check out our Integrated Human Practices page for more information). Mastitis is an infection of the cow-udder, causing over 100 million euros of damage annually in the Netherlands alone ( GD Diergezondheid 2017 ). Like most infections, mastitis is treated with antibiotics. As mastitis is a very common disease, responsible treatment is essential to minimise the spread of antibiotic resistance. We did this by testing for whether the infecting bacteria is already resistant to a certain type of antibiotics.
Mastitis is one of the two most common diseases on the farm.
Tjerkje Poppinga (farmer)
Tsjerkje is studying higher professional education livestock farming. She helps her dad on his dairy farm.
Current situation
Testing whether an infection of a particular cow with mastitis is caused by antibiotic resistant bacteria is currently an expensive and laborious process ( WVAB 2015 ). Typically, a farmer will first treat the cow with standard antibiotics. If this does not work, the farmer turns to lab analysis: he/she sends a milk sample to a company specialized in diagnostics, which then analyse what antibiotics are appropriate for treatment. This whole process, from sending in a milk sample to finding out what antibiotics to use, can take up to 3 days [refers to integrated human practices ]. This process adheres to a guideline ( Formularies ) that is followed by veterinarians when prescribing antibiotics.
Formularies
Formularies are national guidelines for the veterinarians, by the WVAB (Workgroup Veterinary Antibiotics Regulations) - KNMvD (Royal Dutch Society of Vetenary Medicine). These documents prioritize the antibiotics that can be given to treat a disease. Veterinarians are allowed to prescribe first choice antibiotics. The prescription of second- or third choice antibiotics need legitimate reason depending on laboratory results or monitoring activities.
Putting relevance into our design
When initialising mastitis treatment, it is of great importance to know that the infection is commonly caused by 6 different species of pathogens, the most harmful being Staphylococcus aureus (SAU) ( GD Diergezondheid, 2017 ). This pathogen forms the biggest threat to the cow’s health and is highly contagious. To treat mastitis, first choice antibiotics are first used. These antibiotics are shown in the table below.
Ranking prescription antibiotics  ( WVAB 2015 ):
First choice antibiotics: are used for empirical therapies. These antibiotics can be included in the farm treatment plan without further reasoning.
Second choice antibiotics: cannot be used except if there is enough reasoning to prescribe these antibiotics. These antibiotics can be limited included in the farm treatment plan.
Third choice antibiotics: can be used in exceptional cases. These antibiotics cannot be included in the farm treatment plan.
Fourth choice antibiotics: forbidden to use on food producing animals.
Category
Antibiotic
First choice
1. Procaïnebenzylpenicilline
1. Cloxacilline
Second choice
1. Amoxicilline / clavulaanzuur
1. Ampicilline / benzathinecloxacilline
1. Cefalexine
1. Cefapirine
2. Cefalexine / kanamycine
2. Lincomycine / neomycine
2. Procaïnebenzylpenicilline / neomycine
3. Procaïnebenzylpenicilline / nafcilline / dihydrostreptomycine
Third choice
1. Cefoperazone
1. Cefquinome
Resistance to one of the first and two of the second choice antibiotics, namely benzylpenicillin, ampicillin and amoxicillin, is caused by the β-lactamase (blaZ) gene (see unfoldable) in SAU and coagulase-Negative Staphilococci (CNS) ( GD Diergezondheid, 2017 ). Therefore, detection of this gene in SAU can directly lead to a more responsible antibiotics treatment plan.
BlaZ
BlaZ produces small-spectrum penicillinases. This means that penicillin based antibiotics, the most common first choice antibiotics that are empirically used by the farmer to treat mastitis, can not be used. Rapid detection of this gene could prevent the misuse of penicillins by the farmer and gives reason to directly prescribe other treatments.
Design Requirements
Besides finding a relevant gene to detect, we developed a number of design requirements (also see our Design and Demonstrate pages) for our device that came out of stakeholder interactions (see our Integrated Human Practices page) and discussions with experts.
Values
Design Requirements
Fast
There is a demand for a detection to not take longer than 8 hours and preferably less than 3 hours, according to stakeholders and according to research less than 8 hours ( Griffioen et al. 2016 )
Simple and user-friendly
Reduce the amount of steps and make them simple. It should be clear for the farmer what the next steps are after finishing the experiment
On-site
Current detection is done in the lab, new detection is only relevant if it can be done in the field. Sample preparation should not be a limiting factor for the on-site detection
Sensitivity
The sample preparation should give a suitable amount of RNA with minimal contamination. For this we would like to refer to our Sample Preparation and Cas13a pages
Limiting costs
The detection method should be cheaper than current detection methods. A test in the lab (antibiogram and cultivation for sensitivity) is around 25 euros
Biosafety
The pretreatment should be safe for the user and the environment after disposal
Storability
The ease of use on-site increases when the method can be stored without freeze-drying. For this we would like to refer to our TDP Design page
Accuracy
It is important that our detection method does not give false positives. To see how we researched this, we would like to refer to our Modeling page
Clear readout
To be able to obtain a readout visible by the naked eye, we developed a modular detection method which can be revised on the Coacervation Design page
Our solution
We incorporated all these design requirements in our end-product: CASE13A (see our Project Description ). To apply our design in the field, we looked at our design from the perspective of a potential end-user, in this case a farmer. We developed a detection protocol with simplified methods (see our Sample Preperation page). Due to the simplicity of the steps in our protocol, the detection can be done outside a lab environment and without special training. Check out our Demonstration page to see how we achieved this. We also managed to limit the costs to around 10 euros per detection, which is much less than the current method.
Costs
Sample preperation
€
Amount
Make
Used per detection
Cost per reaction(€)
RPA kit
405
96 reactions
Twistdx
1
4,22
T7 polymerase
232
500µL
NEB
1µL
0,46
Murine RNAse inhibitor
235
375µL
NEB
1µL
0,62
Primers
~30
6mL
IDT
4,8µL
0,024
NTP's
104
4x250µL
ThemoFisher
4x1µL
0,42
MgCl2
318,5
500ml (1M)
Merck
0,25µL
0,00017
RNeasy mini Kit
1142,5
250µL
Qiagen
1
4,57
Cas13a
His-select Ni affinity column
4475
500mL
Sigma Aldrich
1/500 mL
0,018
Protease inhibitor
172
30 tablets
Roche
4/500 tablet
0,046
DTT
1190
100 g
Sigma Aldrich
3.36 mg
0,04
TB medium
18.38
2 L
2/500 L
0,018
Detection
PolyU
724
100 mg
Merck
1 µg/µL
0,36
Spermine
2.047,03
100 g
Merck
500µg
0,01
crRNA (IDT)
0,65
1 µg
IDT
15 ng
0,01
Consumables
Disposable pasteur pipette
26,18
500
Servoprax
20,1
0,1
Disposable mohr Pipette 10 microliter
19,4
250
Servoprax
2
0,16
Disposable mohr Pipette 50 microliter
19,4
250
Servoprax
1
0,08
Disposable coloured microcentrifuge tubes 1,5 ml
21
600
Biorad
3
0,11
Disposable coloured microcentrifuge tubes 0,5 ml
28,59
1000
USA scientific
1
0,03
Total:
11,29617
Box Contents
Our product will be packaged with instructions for on-site usage.
1x 1.5mL clean green tube for the fresh milk
1x 0.5mL clean red tube
2x pasteur pipettes 1mL
1x White tube with dried compound (T7 polymerase and RPA)
1x Mixture A (Rehydration buffer and MgAC)
1x Spermine stock
1x Mohr Pipette 50 microliter
2x Mohr Pipette 10 microliter
1x Thermos Flask
1x Immersion heater
1x Thermometer
1x Centrifuge
1x Tube with positive control
1x Tube with negative control
1x Detection holder
Protocol
Protocol Scheme
Detailed protocol
Validation of our detection toolbox
Once we developed the device, we took it to a farm to see if it was, indeed, simple to use. We visited Paul Oosthoek, who confirmed that the steps in the protocol can be followed easily. He also affirmed that the test can be implemented within current regulations on his farm. Furthermore, he said that our do it yourself toolbox has potential as farmers are willing to invest time in detecting resistance genes themselves.
Feedback from farmer Paul
Furthermore, we received feedback on how we can improve our device. Paul stated that it would be better if the test really showed which specific antibiotic should be used for treatment, instead of knowing which antibiotics should NOT be used. We completely agree with this statement; finding out which antibiotic is appropriate to use is more effective in helping to solve the worldwide antibiotic resistance problem. Moreover, he advised us to use larger test tubes for the detection. Currently, we use 0.5 and even 0.2mL eppendorfs in which the detection is done. Paul recommended to perform the detection in larger tubes to increase the visibility and to be more user-friendly.
Lastly, in the protocol we developed there is a step in which the farmer gets external help to purify RNA and measure its concentration. It would, of course, be ideal if the farmer would be able to perform these steps himself. This would require some extra training for the farmer and an improved centrifuge. Also, a way to detect the purity of the RNA has to be developed without having to use expensive equipment. We envision this to be a really cool future iGEM project!
Comparison with other potential solutions
A different solution to this problem, is that the antibiotics test that is currently performed by a diagnostics lab, is performed by the farmer himself. This, however, means that the farmer still has a waiting time of at least 12 hours and that he/she would have to receive special training to be able to handle micro organisms. Comparing this to our solution, we see that we offer a more rapid test that, for the most part, can be performed by the farmer him/herself and does not require an ML-1 environment at the farm.
To what extent does our solution solve the problem?
Two major challenges are identified considering the development of a diagnostic tool. There is low commercial attractiveness as diagnostic tools are not reimbursed in our current healthcare system, which means that they will have to be paid for by farmers or doctors themselves. Also, the use of rapid diagnostics limits the amount of antibiotics used. Therefore, the commercial interests of pharmaceutical companies is low ( van Mierlo 2016 ). Concluding, the commercialization of diagnostic tools is challenging.
A perfect new diagnostic tool would answer the following questions ( O’Neill Commission ):
Is the infection causing the illness bacterial or viral?
If it is a bacterial infection, what type of bacteria is causing the infection?
Are the bacteria causing the infection carry resistance genes against the prescribed drug?
Are the bacteria causing the infection susceptible to the prescribed drug?
In addition, our application focuses on the detection of resistance genes. However, in order to solve the resistance problem, future diagnostic tools should shed light on other questions, as mentioned above. Our versatile RNA-based detection tool can also be applied to answer many other questions (also see our Entrepeneurship page ).
Impact on society
The demand for rapid detection tools to help decision making on antibiotic treatment is not only valuable in the veterinary field, but also relevant in the rapid diagnosis of the well-known MRSA (Methicillin-Resistant Staphylococcus aureus) infection ( Wielders et al. 2002 ). This infection, also known as the ‘hospital disease’, is extremely harmful to humans because it is almost impossible to treat with antibiotics. High-level resistance to methicillin is caused by the mecA gene. Detection of mecA during infections will have impact on the decision-making and the evacuation of the patient to prevent the spread of the dangerous MRSA.
We developed a versatile detection tool to detect the most relevant genes causing multiple antibiotic resistance (see Cas13a Design oage ). This expands the impact of our product to achieve better animal and human treatment strategies that contribute to the fight against antibiotic resistance, together with the upcoming methods to prevent, treat, or take away resistance.
Future vision
It became clear that our method could have a wide range of applications besides RNA detection. One of these applications, namely SNP detection was mentioned by one of our stakeholders (also see our Entrepeneurship page ).
SNP detection
Cas13a has single-base specificity. This means that Cas13a has the ability to detect single bases differing between two sequences ( Gootenberg et al. 2017 ). Such a variation of a single base in a DNA sequence among individuals is called a single nucleotide polymorphism (SNP). Some SNPs are related to (the probability to develop) certain diseases. Being able to easily detect, study and evaluate SNPs can therefore give information about an individual’s predisposition to these diseases ( Nature Education 2014 ). Moreover, the detection of strain SNPs allows for the differentiation between different virus strains ( Gootenberg et al. 2017 ), which is important in choosing a treatment.
After the Giant Jamboree, we will meet with a multi-species animal breeding, genetics and technology companies to discuss the implementation of our technology to reduce the use of antibiotics in livestock farming.
References:
Physicians Committee for Responsible Medicine (2017). Antibiotic Resistance from Animal Agriculture: Foodborne Illness and Medical Care.
Netherlands Center for One Heath, 2016
Van Mierlo, H., 2016. Tackling Antimicrobial Resistance: The Role of Synthetic Biology.  National Institute for Public Health and the Environment. University of Groningen, Faculty of Mathematics and Natural Sciences (private access)
Statistics Netherlands (CBS) 2015
gddiergezondheid 2017
WVAB 2015
Formularies
GD Diergezondheid: S. aureus, 2017
GD Diergezondheid: CNS, 2017
Griffioen, K., Hop, G. E., Holstege, M. M., Velthuis, A. G., Lam, T. J., & 1Health4Food–Dutch Mastitis Diagnostics Consortium. (2016). Dutch dairy farmers’ need for microbiological mastitis diagnostics. Journal of dairy science, 99(7), 5551-556.
Wielders, C. L. C., Fluit, A. C., Brisse, S., Verhoef, J., & Schmitz, F. J. (2002). mecA Gene Is Widely Disseminated in Staphylococcus aureus Population. Journal of Clinical Microbiology, 40(11), 3970–3975.
Gootenberg, J.S. et al., 2017. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, 356(6336), pp.438–442.
Nature Education. 2014. SNP. Accessed 30-10-2017
