<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Main-Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Main-Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col s12 m10"><P>During the course of this iGEM edition, TU Delft embarked on an ambitious and challenging journey for the realization of its project. However, we believe that the iGEM experience is more than just carrying out complex microbiological lab work and developing complicated yet cutting-edge models: iGEM is also a worldwide community that continues to expand, with more than 300 teams participating from all around the world this year.
Consequently, TU Delft organized an <A href="https://2017.igem.org/Team:TUDelft/Meet_Up#!" target="_blank">European iGEM Meetup</A> in which teams from all over Europe were invited to join us in Delft, the Netherlands. Approximately 200 attendees from 9 different countries had the chance to get to know each other, engage in a debate about synthetic biology and its future prospects, and present and share their projects with all the other teams. Besides the fun experience, the presentations and synthetic biology discussions with the other teams inspired us to engage in several collaborations. We hope that the rest of the attending teams were also equally inspired to collaborate with each other! 
     </P><P>TU Delft iGEM 2017 is happy to have collaborated with the teams of <A href="https://2017.igem.org/Team:Munich" target="_blank">Munich</A>, <A href="https://2017.igem.org/Team:Wageningen_UR" target="_blank">Wageningen UR</A>, <A href="https://2017.igem.org/Team:NTHU_Taiwan" target="_blank">NTHU Taiwan</A> and <A href="https://2017.igem.org/Team:Hamburg" target="_blank">Hamburg</A> as well as have contributed to the surveys of <A href="https://2017.igem.org/Team:UNebraska-Lincoln" target="_blank">UNebraska-Lincoln</A> , <A href="https://2017.igem.org/Team:NTHU_Taiwan" target="_blank">NTHU Taiwan</A> and <A href="https://2017.igem.org/Team:BostonU_HW" target="_blank">BU-Hardware</A>.</P><UL class="collapsible popout"><LI><DIV class="collapsible-header">NTHU Taiwan</DIV><DIV class="collapsible-body"><P>After stumbling upon the description of our project, <A href="https://2017.igem.org/Team:NTHU_Taiwan" target="_blank">NTHU Taiwan</A> initiated conversations with us via Facebook. Their project also consisted in a biosensor device, in this case featuring a series of overexpressed receptors and enzymes capable of both sensing and degrading EDC’s (Endocrine Disruptor Chemicals) on-site in sewage waters. A series of Skype meetings followed, in which the usefulness of the inclusion of any of our accomplished BioBricks into their project was discussed. Initially, <A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" target="_blank">vesicles</A> and <A href="https://2017.igem.org/Team:TUDelft/Design#TDP" target="_blank">TDPs</A> were deemed potentially interesting for NTHU Taiwan. However, after a closer study of our vesicles' BioBrick by Taiwan, they came back to us with a series of points (such as the variability in the content and size of vesicles) that made us aware of  the weaknesses of our approach for packing <A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" target="_blank">Cas13a</A> and TDPs in our vesicles. Such a feedback resulted important for TU Delft iGEM 2017, as it made us aware of more aspects that should be addressed and optimised in our vesicles prior to their integration in our project.</P><P> However, the collaboration changed its direction when Taiwan reported difficulties in the transformation of their plasmids.</P><CENTER><DIV class="row"><DIV class="col s12"><B>Figure 1: Snapgene overview of NTHU Taiwan’s construct.</B></DIV></DIV></CENTER><P>TU Delft offered to revise their SnapGene files featuring their construct to discard a faulty design as the issue and shared a series of recommendations on how to make their design clearer in Snapgene and  achieve transformations with higher yields.</P><P>Finally, NTHU communicated TU Delft the accomplishment of their transformation!
We wish all the best to our fellow iGEMers of NTHU Taiwan and look forward to meeting them at the Giant Jamboree!</P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Wageningen UR</DIV><DIV class="collapsible-body"><P>This year’s project of <A href="https://2017.igem.org/Team:Wageningen_UR/Project" target="_blank">Wageningen UR</A>  consisted in a device for the diagnosis of tropical diseases, which featured sensitive perishable material in the form of engineered cells.</P><P>As TU Delft also aimed to manufacture a device intended for the broad audience, we were aware that one of the biggest challenges to both of our projects was the shelf-life or storability of the biological material. To circumvent the previous issue, our team dedicated a part of our project to research the applicability of <A href="https://2017.igem.org/Team:TUDelft/Design#TDP" target="_blank">tardigrade-specific intrinsically disordered proteins (TDPs)</A>in order to enable the stabilization of all fragile material with just a simple drying step.</P><P>Consequently, TU Delft shipped three of his developed plasmids encoding for the production of the TDPs that conferred the most protection (CAHS 94205, CAHS 106094 and SAHS 33020) to Wageningen, so that they could test it with their cells.</P><CENTER><DIV class="row"><DIV class="col s12"><B>Figure 2: Bacteria desiccation tolerance with TDPs.</B> Number of colonies remaining after desiccation and rehydration for the empty vector,  TU Delft’s BioBricks for the production of <B>CAHS 94205</B>, <B>CAHS 106094</B> and <B>SAHS 33020</B> in BL21(DE3). Experiments performed by iGEM Wageningen UR.
                    </DIV></DIV></CENTER><P>As depicted in Figure 1, the survival of the bacteria upon drying overnight and resuspending is greatly enhanced for all the samples featuring the BioBricks with respect to that of the empty vector. In addition, Wageningen resuspended the samples with both Milli-Q water and PBS buffer, achieving additional results that indicated that the survival of the cells producing CAHS 94205 was improved in PBS while that of the cells with SAHS 33020 was instead worsened.
</P><P>In return, TU Delft iGEM tested an additional preservation method researched by iGEM Wageningen UR, consisting of drying cells in combination with kaolin clay.
In the experiments, TU Delft dried BL21(DE3) along with different amounts of kaolin in a laminar flow cabinet overnight
</P><CENTER><DIV class="row"><DIV class="col s12"><B>Bacteria desiccation tolerance experiments with Kaolin.</B> Kaolin tube forwarded by iGEM Wageningen UR and resulting plates dried and resuspended with Kaolin.
                    </DIV></DIV></CENTER><P>After plating the resuspended kaolin and colonies and incubating overnight, the results were unfortunately inconclusive:  the plates were either overgrown or had no colonies at all. Furthermore, after performing Wageningen's  protocol, TU Delft shared a list of suggestions for its improvement as certain steps (such as scraping the dried kaolin containing the bacteria or counting the colonies in plates with kaolin) were both laborious and difficult to reproduce accurately. </P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Munich</DIV><DIV class="collapsible-body"><P>A few months before our European Symposium had taken place, TU Delft shared a project description on Facebook featuring a summary of all the science behind our project. Shortly after, the team of <A href="https://2017.igem.org/Team:Munich" target="_blank">Munich</A> contacted us indicating that, coincidentally enough, our two teams were to make use of the protein <A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" target="_blank">Cas13a</A> in our projects. Because of this, TU Delft and Munich saw a great window of opportunity opening, as the possibility of collaborations that could substantially help both projects was there.</P><B>Wet lab Collaboration iGEM Munich 2017</B><P>Both TU Delft and <A href="https://2017.igem.org/Team:Munich/Description" target="_blank">Munich's project</A> had a common goal: the realisation of a detection device featuring Cas13a. Consequently, the stability of their Cas13a in their device was also one of the challenges in their design, as it would determine the shelf life and storability of their final product. As a part of this year’s project, TU Delft was also researching and developing an alternative stabilization method by making use of the <A href="https://2017.igem.org/Team:TUDelft/Design#TDP" target="_blank">tardigrade-specific intrinsically disordered proteins (TDPs).</A> Therefore, TU Delft shipped a purified batch of TDPs to iGEM Munich, so that they could also assay and evaluate these; and, perhaps, eventually integrate them into their project.
</P><CENTER><DIV class="row"><DIV class="col s12"><B>TDPs in Munich!</B></DIV></DIV></CENTER><P>However, the collaboration with Munich evolved after TU Delft had realised the first experiments with their purified Cas13a and TDPs and had observed that whenever our Cas13a was dried in combination with the TDP CAHS 94205, the resuspended Cas13a+TDP solution would start cleaving RNA  with and without target RNA at a similar rate, thus losing its specificity (see Figure 1).</P><CENTER><DIV class="row"><DIV class="col s12"><B>Figure 3: Cas13a activity with CAHS 94205 in RNase Alert assay.</B> Fluorescence intensities over time triggered by the RNase activity of Cas13a before and after drying with TDP CAHS 94205, crRNA and with/without the target RNA.
                    </DIV></DIV></CENTER><P>We asked  Munich to assist us by repeating a similar drying experiment with the same CAHS protein and their Cas13a, in order corroborate our results and help us find the cause of such an unexpected finding.
As observed in Figure 1, the same unexpected trend is described when Munich’s Cas13a is dried with our CAHS 94205 , hence verifying our CAHS+Cas13a results. </P><CENTER><DIV class="row"><DIV class="col s12"><B>Figure 4: Cas13a activity with CAHS 94205 in RNase Alert Assay by Munich.</B> Fluorescence intensities over time triggered by RNase activity of Cas13a after drying with the TDP CAHS 94205, with and without crRNA and target RNA. Data from iGEM  Munich 2017.
                    </DIV></DIV></CENTER><P>Not only did their results help us discard a problem in our Cas13a or an experimental error as the issue, but also proved a similar behaviour of the TDPs in combination with our  Cas13a’s (which were actually based on the sequences of different organisms and purified according to different protocols) and therefore further substantiated the use of TDPs for the preservation of Cas13a.</P><B>Drylab Collaboration iGEM Munich</B><P>
                Both the iGEM TU Delft and iGEM <A href="https://2017.igem.org/Team:Munich" target="_blank">LMU-TUM Munich</A> teams realized that optimal crRNA is a prerequisite in utilizing Cas13a for their respective detection purposes. In addition to sensitively detecting a sequence of interest, modeling the off-target effects of a chosen crRNA is important in assessing the reliability of the system, as it provides information about the probability of a false positive output. Both teams took a different approach: we employed a kinetic model (<A href="#references">Klein et al. 2017</A>) while iGEM LMU-TUM Munich applied an alignment algorithm. Please find the full model <A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting">here </A>.
            </P><P>
                Biophysical modeling has the advantage of providing a unified targeting framework that is able to take into account tiny (local) off-target effects that together may add to affect the reliability of the system. However, it is computationally intensive as compared to homology search and requires the development of the software, which is already available from NCBI for homology search.
            </P><P>
                We ran 4 off-target simulations for 2 different crRNAs spacer sequences (crRNA1-3 and crRNA4) designed by iGEM LMU-TUM Munich for targeting the 16s ribosomal RNA of two different bacterial species (<I>E. coli</I> DH5α and <I>B. subtilis</I>). The on and off-target activities are modeled for sequences on all possible frames on the genome (Figure 4). From the simulation results, we deduce that the crRNAs specifically target the different bacterial species.
            </P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 4: On and off-target effects of Munich crRNAs on different bacterial targets.</B> On and off-target effects of designed crRNAs on different bacterial targets. A) crRNA 1-3 yield 7 on-target hits (blue) on the <I>E. coli</I> DH5α genome, while off-target probabilities (orange) remain below 1 in 10 million per site. B) crRNA 1-3 yield off-target probabilities below 1 in 10 million for all possible sequences on the <I>B. subtilis</I> genome. C) crRNA 2 yields off-target probabilities below 1 in a million for all possible sequences on the <I>E. coli</I> DH5α genome. D) crRNA 4 yields 10 on-target hits (blue) on the <I>B. subtilis</I> genome, while off-target probabilities (orange) remain below 1 in a million for all possible sequences on the <I>B. subtilis</I> genome.
                    </P></DIV></DIV><P>
                We establish that the two designed crRNA sequences target their designated bacterial species specifically. crRNA 1-3 contains 7 on-target hits on the <I>E. coli</I> DH5α genome, while crRNA 4 contains 10 on-target hits on the <I>B. subtilis</I> genome. From this it can be concluded conclude that the designed crRNA sequences allow differentiation between the (Gram-negative) <I>E. Coli</I> and (Gram-positive) <I>B. subtilis</I> species. Arguably, designing one crRNA that targets both strains could be beneficial in distinguishing between for example a viral and bacterial infection by requiring less experiments, however a Gram-positive or negative distinction could is already very helpful.
            </P><P>
                We also compute $P_{FP}$ (the probability of a false positive output, provided that there is a positive output, for the four simulations (Figure 5). For all simulations, the probability of acquiring a false-positive result was found to be low: in the order of one in a million times. Moreover, having multiple on-target sites on the genome of the bacterial species to be detected decreases $P_{FP}$ on the targeted genome. The $P_{FP}$ on the non-targeted genome, however, is unaffected. This implies that having more on-target sites intuitively also increases the sensitivity of the assay. However, the off-target probability on non-target species remains unaffected by this.
            </P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 5: False-positive probability of crRNAs on different bacterial targets.<I>E. Coli</I> DH5α and <I>B. Subtilis.</I></B> The probability of false-positive output for crRNA1-3 on <I>E. Coli</I> DH5α and crRNA 4 on <I>B. Subtilis.</I> (off-target genomes) is significantly lower than for the targeted genome, because there are multiple on-target sites.
                    </P></DIV></DIV><P>
                We conclude that the design of the crRNAs is reliable as only minute off-target effects in the order of one in a million are observed. In addition, we gain a deeper insight into the effect of having multiple on-targets in the genome: this increases the sensitivity of the detection method but does not affect the reliability. As such, our tool assesses the design of the crRNAs by iGEM LMU-TUM Munich to be clever and to allow reliable detection. We thank iGEM LMU-TUM Munich for the fun collaboration and look forward to meeting them again at the Giant Jamboree.
            </P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Hamburg</DIV><DIV class="collapsible-body"><P><A href="https://2017.igem.org/Team:Hamburg" target="_blank">iGEM Hamburg</A> is another team whose project’s goal is the fight against antimicrobial resistance, on their case, by developing novel antibiotic compounds capable of dealing with multiresistant bacteria.</P><P>It all started with a series of preliminary conversations prompted by the common goal of our projects and the presence of a former student from Hamburg in our team. During the poster presentation held at our European Meet-up, both of our teams had the chance to get to know each other better and become more familiar with the science to our projects. Ideas ranging from stabilizing the siderophores in Hamburg’s antibiotic with our <A href="https://2017.igem.org/Team:TUDelft/Design#TDP" target="_blank">TDPs,</A> to detecting a carbapenem-resistant strain with our device and eliminating it with Hamburg’s antibiotic turned into proposals. </P><P>Unfortunately, none of the aforementioned could be carried out due to time constraints and the prerequisite of our projects being in a stage as advanced as to enable the collaborations.
Many thanks to iGEM Hamburg for traveling all the way from Hamburg to our European Meet-up to share their interesting ideas to make this world a better place!
</P></DIV></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>