<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Main-Motif_Finder skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Main-Motif Finder</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="col s12 m10"><H1>Model - Motif finder</H1><P>There are many different antibiotics and there are many ways for bacteria to be resistant to an antibiotic. Despite this, our detector should detect as many variants as possible for every type of resistance to an antibiotic, not only for variants in the same gene, but also for variants of homologous genes. We therefore looked for different genes encoding for the same type of antibiotic resistance and as well as started looking for different variants of the same gene. After we found a way to collect all those genes, we looked for conserved regions in those genes. From these, we can make appropriate crRNA that fits most or even all bacterial strains containing antibiotic resistance genes.</P><P>Usually genes that confer a similar function will also share regions that are similar. You can expect that this will also be the case in antibiotic genes (<A href="#references">Singh et al. 2009</A>). Therefore we developed a <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">software tool</A> to find these regions.</P><H3>Building a database</H3><P>For the analysis of the groups of genes resistant to one type of antibiotic we used the (<A href="https://card.mcmaster.ca/" target="_blank">CARD</A>) (Comprehensive Antibiotic Resistance Database) database (<A href="#references">Jia et al. 2017</A>). We picked this database because it was a database that is still curated and should therefore contain the most recently discovered genes. However, this database is not that easily accessible in programmatic terms.</P><P>Therefore we decided to build a web crawler to extract the gene sequences. We wrote a script that is able to collect all the sequences and also preserves the tree structure of the database. You can download the resulting collection of sequences (<A href="2017.igem.org/wiki/images/b/ba/T--TUDelft--2017_antibiotic_molecule_36008_%28copy%29.zip" target="">here</A>) (this also includes the rough results), or create it yourself by using our tool from the <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">software page</A>.</P><P>The <A href="https://card.mcmaster.ca/" target="_blank">CARD database</A> is divided in a tree-like structure in which the larger groups of genes are subdivided into smaller groups. Statistics of the groups are represented in Figure 1. the largest group of antibiotics are the beta-lactamase antibiotics. This group also includes for example genes conferring resistance to penicillin.</P><DIV class="row center"><DIV class="col s15"><P><B>Figure 1: Stastitics of our database.</B> The number of genes with known or unknown sequence or subgroup present in our database per antibiotic.</P></DIV></DIV><H3>De novo motif finding - Showing were the genes are alike</H3><P>Motifs are short (2-50) sequences that repeatedly occur in a set of genes. Since our probe (the cRNA) is also short (24-30) we focused on finding motifs in all homologs of genes conferring resistance to one certain antibiotic. To find such motifs we used the (<A href="http://meme-suite.org/" target="_blank">meme</A>) program. We ran the program on multiple groups of genes and after that we did some post-processing to meet the <A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" target="_blank">requirements for CRISPR Cas13a</A>. These motifs need to meet three additional requirements. It requires a 40 to 60 percent GC content and no G is allowed at the 3’-end of the motif (see <A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" target="_blank">crRNA design</A>). The third requirement is about the number of mutations in the conserved region. This is a maximum of 2 and in the seed region a maximum of 1 (see the <A href="https://2017.igem.org/Team:TUDelft/Modeling#offtargeting" target="_blank">off-targeting model page</A>). The seed region is the part of the crRNA that is first bound to the CRISPR Cas13a system, which is estimated to be between base number 8 and 13. All this post-processing is automated and the scripts for this are documented on the <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">Software Tool Page</A>.</P><P>To visualise the results we plotted the coverage of the motifs against the amount of motifs. The motifs are ordered with the maximum coverage first. See Figure 2.</P><P>Here are some interesting graphs created from the results (they are not all included):</P><DIV class="row"><DIV class="col m8 s12 center-margin"><P><B>Figure 2: Coveraged of genes.</B> The coveraged of motifs that confer antibiotic resistance against Beta-lacatamase(a), Streptogramin (b),Fluoroquinolone (c) and Aminoglycoside (d). (Percentages 100x)</P></DIV></DIV><H5>Discussion</H5><P>As you can see in the figures, the maximum coverage does not exceed much beyond 50 percent. Genes differ significantly from each other, and also Cas13a is so sensitive that small differences are usually not allowed. These small differences might be often caused by synonymous mutations, i.e mutations in the DNA do not cause a mutation in the protein. So, even if a region is fully conserved as a protein, it might not be fully conserved in the DNA. We did, however, reduce the number of probes needed largely in comparison with a one probe per one gene strategy.</P><H3>Looking at the variants - a case study</H3><P>For the analysis of variants of the genes, we focused on Mastitis, a cow disease that can be caused by <I>S. aureus</I>. Antibiotic resistance in <I>S. Aureus</I> can be caused either by <I>mec</I>A or <I>bla</I>Z. <I>A is a multi-drug resistance indicator and <I>bla</I>Z is a gene that confers resistance a much smaller amount of antibiotics, but these are the most used ones, like penicillin.</I></P><P>Because we focused on one gene alone, we wanted to make absolutely certain that the region we had in mind for it was the correct region. If you do a broad spectrum analysis, you will find conserved regions that are likely to be conserved in variants of one certain gene as well. We also searched for  a conserved region in variants of one gene.</P><P>To retrieve variants of one gene we used the <A href="https://www.ncbi.nlm.nih.gov/gene" target="_blank">NCBI gene database</A>. This database contains multiple variants of one gene. We searched by the gene name and retrieved the sequences of the returned accession numbers. Sometimes the accession number did not yield a DNA sequence directly and some additional parsing was required. The script that handles this is called 'retrieve_from_ncbi.py' and can be found on our <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">Software Tool page</A>.</P><H3>Results</H3><P>You can find the variants of the <I>bla</I>Z gene <A href="https://static.igem.org/mediawiki/2017/6/63/Blaz.zip" target="_blank">here</A>.</P><P>From the 43 variants, we did a motif search and extracted the conserved regions. This process can be automated by using the function &quot;only_one&quot; from the file &quot;meme_functions.py&quot; (See also our <A href="https://2017.igem.org/Team:TUDelft/Software" target="_blank">software page</A>):</P><PRE class="DNA center-margin">
—AXAXTGXTXAXXXAXXAAXXXAXAAAATXXTAAXGXXA
TXGGTGXXXXXAAAAAAGTTAAACAACXTXTAAAAXXAX
TXGGAGATAAAGTAACAAATCCAGTTAGATATGAXATAG
AATAAATTAXTATTCXCCAAAGAGCAAAAAAGATACTTC
AACXCCTGCTGCTTTCGGXAAGACTTTAAATAAACTTAT
CGCAAATGGAAAATTAAGCAAAXAAAAXAAAAAXTTCTT
ACTTGATTTAATGTTXAATAATAAAAXCGGAGAXACTTT
AATTAAAGAXGGTXTTXCAAAAGACTXTAAXGTXGCTGA
TAAAAGTGGTCAAGCAATAACATATGCTTCTAGAAATGA
TGTTGTTTTGTTTATCCTAAGGGCCAATCTGAACCTATT
GTTTTAGTCATTTTTACGAATAAAGACAATAAAAGTGAT
AAGCCAAATGATAAGTTGATAAGTGAAACCGCCAAGAGT
GTAATGAAGGAATTTTA
</PRE><P>— indicates the beginning of the sequence which was not conserved in all sequences.</P><P>X indicates a base that is not conserved enough (it is not the same in the 42 sequences checked)</P><P>NB. There was one from 43 sequences that did not have any resemblance to the rest of sequences. So this is likely a faulty result from in the data and is therefore not included in the results.</P><H3>Link to the lab</H3><P>From the conserved regions we have also designed primers (<A href="https://2017.igem.org/Team:TUDelft/Primers" target="_blank">IG0088 and IG0089</A>). These primers have been tested on a real world sample with an unknown sequence. This test was successful and proved that this part of the software is working as it should be. See <A href="https://2017.igem.org/Team:TUDelft/Results#sampleprep" target="_blank">the sample prep results page</A> for the results.</P><DIV id="references-scroll" class="section card"><DIV class="card-content"><SPAN class="card-title">References:</SPAN><OL><LI><A href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp335" target="_blank">Bailey, Timothy L, Boden, Mikael, Buske, Fabian A, Frith, Martin, Grant, Charles E, Clementi, Luca, Ren, Jingyuan, Li, Wilfred W &amp; Noble, William S (2009). MEME SUITE: tools for motif discovery and searching. Nucleic acids research, 37, W202-W208.</A></LI><LI><A href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw1004" target="_blank">Jia, Baofeng, Raphenya, Amogelang R, Alcock, Brian, Waglechner, Nicholas, Guo, Peiyao, Tsang, Kara K, Lago, Briony A, Dave, Biren M, Pereira, Sheldon, Sharma, Arjun N &amp; others (2017). CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database. Nucleic acids research, 45, D566-D573.</A></LI><LI><A href="https://jbiomedsci.biomedcentral.com/articles/10.1186/1423-0127-16-109" target="_blank">Singh, Reema, Saxena, Akansha &amp; Singh, Harpreet (2009). Identification of group specific motifs in Beta-lactamase family of proteins. Journal of biomedical science, 16, 109.</A></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>