Team:TUDelft/Protocols
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Contribution
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Silver HP
Integrated and Gold
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Case13a
Description
Design
Results
Demonstrate
Entrepreneurship
Notebook
Protocols
Safety
Parts
Basic Parts
Composite Parts
Part Collection
Improved BioBrick
Primers
Integrated Human Practices (IHP)
Education & Public Engagement
Applied Design
Outreach
Newsletter
European iGEM Meet Up
Attribution
Sponsors
Crowdfunding
Contribution Overview
InterLab
Software
Plot Growth Curves
Find correct CRISPR targets
Team
Supervisors
Team Building
Collaborations
Entrepreneurship
Software tool
Measurement
Applied Design
IHP
Education & Public Engagement
Modeling
Overview
Motif finder
Off-targeting
Coacervation
Lattice Model
Rate kinetics
Overview
Motif Finder
Off-targeting
Coacervation
Lattice Model
Rate Kinetics
Cas13a
Sample Preperation
Coacervation
TDPs
Vesicles
Special Tracks
Medals
Awards & Nominations
home
Project
Lab
Model
Sequences
Human Practices
Contributions
Team
Acknowledgements
Giant Jamboree
TUDelft.iGEM2017
  +31 152 78 77 51
close
menu
-------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- ----------------------- -------------------- -------------------- -----------------------
Agar plate
Agar plate
Dissolve 14.24 g of LB-agar in 400 mL of milli-Q.
Autoclave the LB-agar solution.
Cool the agar down to room temperature. To facilitate the process, you can hold the bottle under the cold water tap.
Add the correct amount of the correct antibiotic. For example, for our antibiotic stock solutions we used 1 µL of antibiotic solution/mL of LB-agar.
Under sterile conditions, pour the LB-agar in empty Petri dishes.
Leave the Petri dishes near the flame with the lid partially off for approximately 5 minutes so the agar can solidify.
Blunt end ligation
Blunt end ligation
Thaw the ligase buffer on ice, to prevent damaging the ATP
For phosphorylation, prepare a sample a sample as follows:
Compound
Volume (µL)
DNA vector
(~250 ng is required)*
DNA insert
*
Ligase buffer
2.5
ATP
2.5
T4 PNK enzyme
1 µL
Sterile milli-Q
Up to 25 µL
*Volume depends on concentration of the sample.
Add 250 ng of linearized vector backbone.
Depending on the assembly, the other assembly pieces can be added in a predetermined ratio of 1:3. An online ligation calculator can subsequently be used to calculate the amount of the assembly pieces that is required.
Incubate for at one hour at 37 °C.
Cancel phosphorylation process by incubating for 30 minutes at 65 °C.
Add 1 µL of Ligase T4 and incubate for at least one hour at 4 °C.
Bradford Assay
Bradford Assay
To determine the concentration of protein in a purified protein mix, Bradford assays were used. This protocol describes this procedure in detail.
Take 10 µL of 2 mg/mL BSA (PierceTM Bovine Albumin Standard Ampules, ThermoFisher).
Make a dilution series in 1 mL tubes, so that you eventually have 5 µL of 2 mg/mL, 1 mg/mL, 0.5 mg/mL, 0.25 mg/mL and 0 mg/mL BSA.
Put 5 µL of your protein sample in another 1 mL tube. Depending on how accurate the protein concentration needs to be, multiple replicates can be taken.
Dilute Bradford reagent (sigma) to appropriate concentration and add 1 mL of this diluted reagent to each tube containing protein.
Leave all samples for at least 10 minutes, but no longer than one hour.
Transfer the samples in 1 mL cuvettes and measure the absorbance of light at 595 nm. Write down the absorbance values and make a plot of the absorbance of the samples containing BSA versus the BSA concentration.
This reference line can now be used to estimate the protein concentration in your samples.
Cas13a activity assay with RNase Alert
Cas13a activity assay with RNase Alert
With this protocol the Cas13a collateral cleaving activity can be measured using RNAse alert. RNase alert consists of both fluorophores and quenchers connected to RNA polymers in such a way that the quencher absorbs the emitted photons of the fluorophore. Once the RNA polymers are cleaved, the emitted photons by the fluorophore will no longer be absorbed by the quencher and the solution will fluoresce.
This protocol is based on the LwCas13a collateral detection protocol by Gootenberg et al. 2017 .
Resuspend a tube of RNase Alert with 40 μL Nuclease free water an keep it on ice. Do not expose the RNase Alert to light as much as possible.
Set up the following reaction in a 96 well plate:
Order of pipetting
Cas13a with crRNA and target
Cas13a with target
Cas13a with crRNA
Cas13a
2
10x Cas13a reaction buffer*
10 μL
10 μL
10 μL
10 μL
6
Cas13a***
2.3 μM
2.3 μM
2.3 μM
2.3 μM
3
crRNA
A final concentration of 20 nM
A final concentration of 20 nM
A final concentration of 20 nM
A final concentration of 20 nM
4
Target
5
RNas Alert**
10 μL of resuspended RNase Alert
10 μL of resuspended RNase Alert
10 μL of resuspended RNase Alert
10 μL of resuspended RNase Alert
1
Nuclease free water
Add up to a final volume of 100 μL
Add up to a final volume of 100 μL
Add up to a final volume of 100 μL
Add up to a final volume of 100 μL
*A 10x Cas13a reaction buffer consist of 400 mM Tris-HCl, 600 mM NaCl, 60 mM MgCl2, pH 7.3.
**Thermo **Fischer Scientific, 2017. RNaseAlert Lab Test Kit. Available at: https://www.thermofisher.com/order/catalog/product/AM1964 [Accessed October 23, 2017].
***Adding Cas13a will trigger the reaction.
Insert the plate in the plate reader and measure the fluorescence over time with the following protocol:
Shake the plate for 3 minutes double orbital.
Measure every 3 minutes the fluorescence in each well.
Repeat step 2. for 3 hours.
References:
Gootenberg, J.S. et al., 2017. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, 356(6336), pp.438–442.
Thermo Fischer Scientific, 2017. RNaseAlert Lab Test Kit. Available at: https://www.thermofisher.com/order/catalog/product/AM1964 [Accessed October 23, 2017]
Cas13a purification
Cas13a purification
This protocol describes how we finally purified Cas13a. It is divided into two parts, namely (i) the preparation of the cell pellet and (ii) the purification of Cas13a from this cell pellet.
For growing bacteria that express Cas13a, Terrific Broth (TB) medium was used, which contains 12 g/L tryptone, 24 g/L yeast extract, 9.4 K2HPO4 and 2.2 g/L KH2PO4.
For one pellet, prepare a starter culture by mixing 5 mL of TB medium with appropriate amount of anbitiotics. The antibiotics to be added depend on the antibiotic resistance properties of the bacteria in which the plasmid encoding Cas13a has been transformed.
Inoculate a single colony containing bacteria into which the plasmid enoding for Cas13a has been transformed, into this starter culture, and leave overnight at 37 °C with rotation (220 rpm).
The next morning, measure the OD600 of the starter culture to confirm growth (an OD600 of around 2 is to be expected). Inoculate the 4 mL into 1 L of TB medium with appropriate antibiotics. Leave this culture at 37 °C with rotation (180 rpm), until an OD600 of 0.6 is reached.
Prior to induction, put the cultures on ice for 30 minutes, and then take out 1 mL and store at 4 °C as a control later in the procedure.
Induce by adding IPTG to a final concentration of 500 μM.
Leave the culture for 16 hours at 18 °C with rotation (180 rpm).
Take another 1 mL sample of the culture and store for later control in the procedure.
Spin the liter culture down at 5200 g and 4 °C for 15 minutes, and discard supernatant.
Resuspend the pellet into 30 mL 1x PBS buffer, and transfer to a 50 mL tube.
Spin this down for 10 minutes at 3220 g and 4 °C, and again discard supernatant.
Store the pellet that remains at -80 °C for later purification.
Measure the OD600 of the sample before induction and after expression, and subsequently spin down both tubes at 1,6000 xg for 2 minutes.
Discard supernatant and rehydrate in 100 μL × OD600 as measured in the previous step, and load 10 μL of this concentrated resuspension onto an SDS-PAGE gel to confirm successful induction and expression of Cas13a.
Thaw the pellets from -80 °C on ice. All following steps in this purification protocol are done at 4 °C.
Resuspend this pellet in lysis buffer (20 mM Tris-HCL, 500 mM NaCl, 1 mM DTT, 25 mM imidazole, pH 8.0) and 100 u/ml benzonase, 0.25 mg/ml lysozyme and 1 protease inhibitor tablet per ml. Make sure pellet is fully resuspended.
Break the cells using French press (3 rounds at 100 kbar).
Spin down the resulting French pressed suspension for 45 min at 16.000 g. Note that the Cas13a is now dissolved in the supernatant, and the pellet contains cell debris.
Filter the supernatant using a 0.45 μm filter.
Take 1 mL of 50% Hisselect Nickel Affinity gel, and wash this by dissolving it into 2 mL of lysis buffer and spinning down at 3220 g for 1 min. Do this three times.
Add the Hisselect column material to the filtered supernatant and incubate for one hour with rotation.
Spin down the mixture, for 1 min at 2000 g. This mix now contains Hisselect Nickel column bound to Cas13a.
Discard the supernatant by pipetting and load the resin onto a gravity column.
Wash the resin five times in 2 mL of lysis buffer and collect the flowthrough.
Elute the Cas13a from the resin 5 fractions of 300 μL of elution buffer (20 mM Tris-HCl, 500 mM NaCl, 1 mM DTT and 250 mM imidazole, pH 8.0), and collect in differently labeled tubes.
Measure the A280 of all five elution fractions at the Nanodrop, to roughly estimate the protein concentrations.
Load the wash and elution fractions onto an SDS-PAGE gel to confirm presence of the protein.
Prepare 1 L of storage buffer (50 mM Tris-HCl, 600 mM NaCl, 5% v/v glycerol, 2 mM DTT and pH 7.5).
Buffer exchange by dialysis of the mixed fractions (max 1.5 mL) overnight in the full 1 L storage buffer.
Take the Cas13a solution out of the dialysis tubing and measure the concentration using a Bradford assay .
Cas13a TDP Assay
Cas13a TDP Assay
Add 4 µL Cas13a (c = 0.5 g/L) and 1 µL Tris/HCl (c = 2.5 M) to 95 µL solution containing the TDP of choice in the concentration of choice.
Dry overnight in Speedvac.
Resuspend the next day in 100 µL of nuclease free water and use 25 µL in RNase alert assay .
Colony PCR (GoTaq)
Colony PCR (GoTaq)
Under sterile conditions, pick a colony and dilute it in 20-25 µL of milli-Q water.
NOTE: After you pick the colony, it cannot be used again. It is therefore recommended to make a 'back-up'-plate where you grow the colonies again. This plate should be incubated overnight at 37 °C.
Incubate at 90 °C for 10 min.*
Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
Component
Volume (µL)
GoTaq 5x buffer
10
10 mM dNTPs
1
Primer VF2 (10µM)
1
Primer VR (10µM)
1
Sterile milli-Q
31.8
Gotaq polymerase (5u/µL)
0.2
Total
45
Pipette 45 µL of mix into each PCR tube (one per colony).
Centrifuge the colony mixture for 5 minutes at 16,000
x
g.
Add 5 µL of supernatant of colony mixture to each PCR tube.
Put the tubes in the PCR machine and apply the following program (it needs to be adjusted for primers annealing temperature and extension time):
Step
Temperature (°C)
Time (s)
Initial denaturation
98
150
Denaturation
94
60
x30 cycles
Annealing
55
60
Extension
72
60 /1 kb
Final extension
72
480
Hold
4
∞
The PCR product(s) can be checked on gel. In order to do so, cast a gel and prepare the samples according to the DNA electrophoresis protocol.
*Instead of separate 'cooking' of the cells before the PCR, this step can be incorporated in the PCR program. The initial denaturation step should then be prolonged to 5 minutes (300 seconds).
Colony picking protocol
Colony picking protocol
Add 25 μL sterile milli-Q to X tubes.
Touch with a clean pipette tip a free colony with no other colonies attached to from your plate containing the bacteria.
Stir with the pipette tip containing the bacteria in one of the tube with milli-Q.
Point carefully with the pipette tip on a clean plate with no antibiotic resistance.
Repeat this for X colonies.
NB: Work sterile.
crDNA annealing
crDNA annealing
For the annealing of crDNA a PCR protocol was used, which is as follows:
Step
Temperature (°C)
Time (m:ss)
1
95
2:00
2
90
0:10
3
85
0:10
4
80
0:10
5
75
0:10
6
70
0:10
7
65
0:10
8
60
0:10
9
55
0:10
10
50
0:10
11
30
0:10
12
12
∞
Digestion assay
Digestion assay
Decide on which enzyme(s) to cut with. Check online what buffer the enzyme(s) work(s) in. Often, SmartCut buffer can be used.
Prepare a sample a sample as follows:
Compound
Volume (µL)
DNA
(~1 µg is required)*
Buffer (CutSmart)
2
Restriction enzyme(s) (5u/µg)
1 µL each
Sterile milli-Q
Up to 20-25 µL
*Volume depends on concentration of the sample.
Incubate for one hour at 37 °C.
Inactivate the restriction enzyme(s) by heating to 65 °C for 10 minutes.
NOTE: This last step can be skipped if the sample is evaluated on gel immediately after.
DNA concentration measurement (NanoDrop)
DNA concentration measurement (NanoDrop)
Turn on the NanoDrop UV-VIS Spectrophotometer.
Press the button for dsDNA to measure the concentration of double-stranded DNA in your samples.
Clean the measurement surface with a piece of tissue and ethanol.
Use 1-1.5 µL of sterile milli-Q as a blank.
Clean the measurement surface with a piece of tissue and water.
Use 1-1.5 µL of sample to measure its concentration.
NOTE: It is best to measure the same sample in triplo and use the average value.
If you have multiple samples, clean the measurement surface in between measurements.
DNA electrophoresis
DNA electrophoresis
Work clean! Handle all material labelled as EtBr contaminated with gloves. Don’t take it outside of the EtBr area and don’t touch anything that is not labelled as EtBr contaminated with gloves.
Prepare TAE buffer: take the 10X concentrated TAE from the chemicals cabinet and dilute it 10 times with milli-Q. For 500 mL, add 50 mL to 450 mL of dH2O.
Weigh agarose for a 1% gel. For 200 mL, 2 g of agarose is necessary.
Mix the TAE solution with the agarose and heat the solution (in a microwave) until it is completely dissolved.
Add SYBR Safe to the gel mould. For a small gel (40 mL of the prepared TAE/agarose mixture) add 4 µL of SYBR Safe; for a large gel (100 mL) add 10 µL of SYBR Safe.
Pour the solution into the mould, making sure there are no bubbles and that the SYBR Safe is completely mixed. Add a comb to create wells for the samples. Let it solidify (approx. 20 minutes).
Transfer the gel to the electrophoresis cell minding the arrow that indicates the direction of DNA migration. Remove the combs and cover it with TAE.
Prepare the electrophoresis samples (on parafilm); 1 µL of Nucleic Acid Loading Buffer per 5 µL of sample.
Load the molecular weight marker (ladder) in the first well (check the appropriate volume for each marker, generally 5 µL works fine) and load 5-10 µL of the samples in the other wells, according to the order in your lab journal.
NOTE: Do not contaminate the loading buffer and ladder with SYBR Safe! Do not touch it while wearing a glove.
Connect the cables following the colour code and run at 100-130 V for 30-60 min.
DNA isolation
DNA isolation
Bioling method:
Transfer 1 mL of your sample to a tube and place in boiling water for 10 minutes.
Centrifuge for 5 minutes at 94
x
g.
Transfer 200 µL supernatant containing your DNA to a new tube.
Microwave method:
Transfer 500 µL of your sample to a tube and put in the microwave for 10 seconds.
Centrifuge for 2 minutes at 94
x
g.
Transfer 200 µL supernatant containing your DNA to a new tube.
DpnI digestion
DpnI digestion
NOTE: When a PCR product is used in this assay, it should be purified according to the PCR purification protocol first!
Prepare a sample in a 0.5 mL microcentrifuge tube as follows:
Compound
Volume (µL)
Purified PCR product
30
Sterile milli-Q
5
CutSmart Buffer (10x)
4
DpnI
1
Incubate for 1.5 hour at 37°C.
Follow the PCR purification protocol with 40 µL membrane binding solution.
gBlock resuspension (IDT)
gBlock resuspension (IDT)
Centrifuge the tube with your IDT gBlock fragment for 3−5 s at a minimum of 3,000
x
g to pellet the material to the bottom of the tube.
Add sterile milli-Q to the tube for your desired final concentration. The required volume of milli-Q can be read from the table below (and the label on the IDT tube that contains your fragment):
Final concentration
250 ng
500 ng
1000 ng
10 ng/µL
25
50
100
20 ng/µL
Not recommended
25
50
50 ng/µL
Not recommended
10
20
Incubate at 50 °C for 20 minutes.
Briefly vortex and centrifuge.
Store the resuspended gBlock at -20 °C.
Gel product purification (Promega Wizard™ Kit)
Gel product purification (Promega Wizard™ Kit)
Place gel slice in a 1.5 mL microcentrifuge tube.
Pre-warm an aliquot of sterile milli-Q at ~40°C.
Add 10 µL Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is completely dissolved.
Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute.
NOTE: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle's instructions).
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.
Transfer the SV Minicolumn to an empty, labelled 1.5 mL tube.
Add 50 µL of the pre-warmed water (30 µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
Incubate at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4 °C (for immediate use) or -20 °C (for storage).
Gibson Assembly
Gibson Assembly
This Gibson Assembly protocol was adapted from Gibson (2009).
Thaw 7.5 µL of Gibson Assembly mastermix on ice.
Add 100 ng of linearized vector backbone.
Depending on the assembly, the other assembly pieces can be added in a predetermined ratio. An online ligation calculator can subsequently be used to calculate the amount of the assembly pieces that is required.
Add the correct volume of the linearized backbone and the assembly pieces to the tube of the mastermix, not exceeding 15 µL.
Incubate the assembly reaction at 50 °C for 60  and then place on ice.
Transform competent cells according to the transformation protocol. Alternatively, you can run a diagnostic agarose gel according to the DNA electrophoresis protocol to check for successful assembly.
Heat-shock for DH5α-cells from Tobal
Heat-shock for Dh5α-cells from Tobal
Since the transformation of cells from Tobal did not work out very well at first, a new heat-shock protocol was used for these Dh5α-cells:
Add all DNA mix (GA/ligation) to 50 µL cells which have been thawed on ice.
Incubate on ice for 30 minutes.
Heat-shock for 1 minute at 42 °C.
Incubate on ice for 2 minutes.
Add 950 µL SOC-medium.
Incubate at 37 °C for 1 hour.
Plate out 100 µL on a selective medium plate.
Briefly centrifuge the remaining cell culture.
Plate out the pellet on selective medium.
Incubate the plates at 37 ° overnight.
LDH Assay
LDH Assay
Stock solutions:
Tris/HCl (pH = 7), c = 2.5 M
Sodium phosphate dibasic, c = 1 M
Sodium phosphate monobasic, c = 1 M
Procedure:
Prepare reaction buffer (100 mM sodium phosphate, 100 µM NADH, 2 µM pyruvate) by mixing
0.6 mL of sodium phosphate dibasic (stock solution)
4.4 mL of sodium phosphate monobasic (stock solution)
0.011 g of pyruvate
0.00325 g of NADH
and protect resulting solution from sunlight (should be done right before the measurements).
Prepare samples by mixing:
1 µL of Tris/HCl (pH = 7)
1 µL of LDH (L-Lactate Dehydrogenase)
x µL of TDP solution (to reach desired concentration in 100 µL)
98-x µL of MQ
adding up to a volume of 100 µL
Split samples in half and store one half in the fridge at 4 °C as a control and dry the remaining half overnight with a SpeedVac, with a setting for aqueous solutions without centrifugation.
Collect dry samples and store them in a dry place at room temperature until measurement.
Resuspend with a volume of 100 µL MQ, vortex for 10 s and spin down to ensure homogeneous resuspension and keep on ice.
For measuring each sample, add 1 µL of sample into a well of a 96-well plate, add 199 µL of reaction buffer and measure the absorption of the mix at 340 nm immediately with a microoplate reader (25 °C).
Activity can be determined by comparing the initial linear reaction rate of each dehydrated and rehydrated sample to that of its corresponding unstressed control.
Liquid (starter) culture (10 mL)
Liquid (starter) culture (10 mL)
Label as many 15 mL Falcon tubes as the number of colonies you want to grow.
Prepare 10 mL of selective medium (LB or SOC) per Falcon tube. Use 1 µL of antibiotic stock per mL of medium (for chloramphenicol) or 2 µL per mL of medium for ampicillin.
Under sterile conditions, pour the medium in the Falcon tubes.
Under sterile conditions, pick a colony with the inoculating loop.
NOTE: After you pick the colony, it cannot be used again. It is therefore recommended to make a 'back-up'-plate where you grow the colonies again. This plate should be incubated overnight at 37 °C.
Swirl the inoculating loop in the Falcon tube containing the medium. Often you can see the colony falling from the loop.
Grow the liquid cultures overnight, shaking at approximately 250 rpm and 37 °C.
Making electrocompetent cells
Making electrocompetent cells
Thaw your cells on ice
Streak the cells on a plate with the corresponding antibiotics and let it grow overnight at 37 °C
Inoculate 30 mL LB with picked colonies from the plate in a 50 mL Greiner tube.
Grow till an OD of ~0.5
Centrifuge for 10 minutes at 4 °C at 3900 rpm.
Discard the supernatant and dissolve pellet in 20 mL cold milli-Q.
NOTE: make sure all the cells are resuspended!
Centrifuge for 10 minutes at 4 °C at 3900 rpm.
Discard the supernatant and dissolve pellet in 20 mL cold milli-Q.
Centrifuge for 10 minutes at 4 °C at 3900 rpm.
Discard supernatant and resuspend in 200 µL glycerol
Prepare aliquots of 50 µL
Either transform straight away or store at -80 °C
Mass Spectrometry Preparation
Mass Spectrometry Preparation
Prepare overnight cultures of the required samples.
Measure OD of overnight cultures and dilute to an OD of 0.05 in 10 mL LB medium.
When the OD of the new cultures has reached 0.5, add 10 µL of IPTG (c = 1 M) and incubate for 2-3 hours.
Centrifuge samples at maximum speed for five minutes.
Grow bacteria in 4-5 mL of LB medium at 37 °C at 300 rpm.
Induce with 1M IPTG (1 µL in 1 mL) at OD = 0.5.
Incubate for 2-3 hours at 37 °C at 300 rpm.
Add 1 mL of solution to 250 µL of 10 %TCA.
Incubate on ice for 10 min.
Spin down at maximum speed and discard supernatant.
Add 200 µL of 1 % TCA to wash (just flip the tube, don't vortex).
Spin down at maximum speed and discard supernatant.
Store at -20 °C.
Mass Spectrometry Preparation 1
Mass Spectrometry Preparation 1
Prepare overnight cultures of the required samples.
Measure OD of overnight cultures and dilute to an OD of 0.05 in 10 mL LB medium.
When the OD of the new cultures has reached 0.5, add 10 µL of IPTG (c = 1 M) and incubate for 2-3 hours.
Centrifuge samples at maximum speed for five minutes.
Mass Spectrometry Preparation 2
Mass Spectrometry Preparation 2
Grow bacteria in 4-5 mL of LB medium at 37 °C at 300 rpm.
Induce with 1M IPTG (1 µL in 1 mL) at OD = 0.5.
Incubate for 2-3 hours at 37 °C at 300 rpm.
Add 1 mL of solution to 250 µL of 10 %TCA.
Incubate on ice for 10 min.
Spin down at maximum speed and discard supernatant.
Add 200 µL of 1 % TCA to wash (just flip the tube, don't vortex).
Spin down at maximum speed and discard supernatant.
Store at -20 °C.
Membrane staining
Membrane staining
Dissolve the dye in 100 µL milli-Q to a final concentration of 1.6 mM.
Add 1 µL dye to 99 µL sample and mix well.
Measure at an exc./em. of 515/640 nm.
Milk Bacterial DNA Isolation Kit
Milk Bacterial DNA Isolation Kit
Lysate Preparation (Unknown or Gram Positive Bacteria):
Transfer 1 mL of milk to a tube and centrifuge at 20,000
x
g. for 2 minutes.
Discard the supernatant and resuspend the pellet in 100 µL of resuspension solution A (with lysozyme).
NOTE: the provided lysozyme should be added to the resuspension solution prior to use.
Incubate at 37 ° for 45 minutes; mix occasionally by vortexing.
After incubation, add 300 µL of Buffer SK and 10 µL of Proteinase K to the digestion mixture and mix well by vortexing.
Incubate the lysate at 55 °C for 45 minutes; mix occasionally by vortexing.
DNA extraction:
Add 200 µL of 96-100 % ethanol to the lysis mixture and mix by vortexing.
Transfer the mixture to a column placed in a 2 mL collection tube. Centrifuge for 2 minutes at 20,000
x
g.
Discard the flow-through and add 500 µL of Buffer SK. Centrifuge for 2 minutes at 20,000
x
g.
Discard the flow-through and add 500 µL of Wash Solution A. Centrifuge for 1 minute at 20,000
x
g.
Discard the flow-through and add another 500 µL of Wash Solution A. Centrifuge for 1 minute at 20,000
x
g.
Discard the flow-though and centrifuge for 2 minutes at 20,000
x
g. to ensure the column is completely dry.
Discard the collection tube and transfer the column to the provided 1.7 mL elution tube.
Add 100 µL of Elution Buffer B and centrifuge for 2 minutes at 425
x
g.
Centrifuge for 1 minute at 20,000
x
g. to complete the DNA elution. Store your sample at -20 °C.
Osmoshock
Osmoshock
Preparation:
Grow your cultures for more than 6 hours.
Prepare osmoshock-mix:
- 20 % sucrose
- 1 mM EDTA
- 30 mM Tris-HCl
Store milli-Q at 4 °C and put on ice right before usage.
Osmoshock:
Centrifuge for 15 minutes at 3,000
x
g. to pellet the cells.
Centrifuge longer if supernatant is not clear
Discard supernatant and weigh the pellet.
Resuspend cells in the prepared osmo-shock mix.
Resuspend gently
Use 80 mL per 1 g wet weight of pellet
Shake gently for 10 minutes at roomtemperature.
Centrifuge for 10 minutes at 13,000
x
g.
Discard supernatant and resuspend cells in ice cold milli-Q.
Use the same volume as in step 3
Shake gently for 10 minutes at 4 °C.
Centrifuge for 10 minutes at 13,000
x
g. at 4 °C.
Collect the supernatant for further analysis of the periplasmic fraction. Dissolve the pellet in milli-Q to collect the cytoplasmic fraction.
Use the same volume as in step 3
NOTE: Do not freeze the cell pellet; this will cause cell lysis
PCR (Phusion® polymerase)
PCR (Phusion® polymerase)
Prepare the mix for all the samples in a single 1.5 mL tube (mind pipetting error!). For one sample:
Component
Volume (µL) for 50 µL reaction
5X Phusion HF Buffer
10
10 mM dNTPs
1
FW Primer
2.5
RV Primer
2.5
Phusion polymerase
0.5
Template DNA (up to 250 ng)
Variable
Nuclease-free H2O
Up to 50 µL
NOTE: The 250 ng of template DNA are approximate, choose a volume that works fine for all your samples and fill it in the excel sheet so you are able to prepare a mix for all samples at the same time.
Add the total volume per sample (without DNA) to each PCR tube.
Add the template DNA to each tube.
Put the tubes in the PCR machine and apply the following program (it needs to be adjusted for primers annealing temperature and extension time):
Step
Temperature (°C)
Time (s)
Initial denaturation
98
30
Denaturation
98
10
x30 cycles
Annealing
60
15
Extension
72
15-30/1 kb
Final extension
72
300
Hold
4
∞
Run a DNA electrophoresis following the DNA electrophoresis protocol.
PCR product purification (Promega Wizard™ Kit)
PCR product purification (Promega Wizard™ Kit)
Add an equal volume of Membrane Binding Solution to the volume of PCR product.
Pre-warm an aliquot of sterile milli-Q at ~40°C.
Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved PCR mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute.
NOTE: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle’s instructions).
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.
Transfer the SV Minicolumn to an empty, labelled 1.5 mL tube.
Add 50 µL of the pre-warmed water (30 µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
Incubate at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4 °C (for immediate use) or -20 °C (for storage).
Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)
Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)
Prepare liquid cultures according to the liquid (starter) culture (10 mL) protocol.
Under sterile conditions, take a 200 µL aliquot as a back-up and store at -80 °C.
Prepare Lysate
Centrifuge the Falcon tubes for 10 minutes at maximum speed.
Pour off the supernatant and resuspend in 600µL of TE buffer. If they are not already in a microcentrifuge tube, transfer the resuspended cells to a sterile 1.5 ml microcentrifuge tube(s).
Add 100 µL of Cell Lysis Buffer (blue), and mix by inverting the tube 6 times.
Add 350 µl of cold (4–8 °C) Neutralization Solution, and mix thoroughly by inverting.
Centrifuge at maximum speed in a microcentrifuge for 3 minutes.
Transfer the supernatant (~900 µl) to a PureYield™ Minicolumn without disturbing the cell debris pellet.
Place the minicolumn into a Collection Tube, and centrifuge at maximum speed in a microcentrifuge for 15 seconds.
Discard the flowthrough, and place the minicolumn into the same Collection Tube.
Wash
Add 200 µl of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 seconds.
Add 400 µl of Column Wash Solution (CWC) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds.
Elute
Transfer the minicolumn to a clean 1.5 ml microcentrifuge tube, then add 30 µl of sterile milli-Q directly to the minicolumn matrix. Let stand for 1 minute at room temperature.
Centrifuge for 15 seconds to elute the plasmid DNA.
Following the NanoDrop protocol, determine the concentration of the miniprepped samples. Store at -20 °C.
Plasmid midiprep (GeneJET plasmid midiprep kit)
Plasmid midiprep (GeneJET plasmid midiprep kit)
The original protocol by Thermo Scientific can be found here . We used protocol A: Plasmid DNA purification using low speed centrifuges.
NOTE: Harvest the bacterial culture by centrifugation at +4 °C. All other centrifugation steps should be carried out at room temperature.
Grow up to 50 mL of bacterial culture to an OD600 of 2-3.
Harvest the cells by centrifugation for 10 minutes at 5,000
x
g. Discard the supernatant.
Resuspend the pelleted cells in 2 mL of Resuspension Solution. The bacterial pellet should be resuspended by vortexing or pipetting up and down until no cell clumps remain.
NOTE: ensure that RNase A Solution has been added to the Resuspension Solution.
Add 2 mL of Lysis Solution and mix gently by inverting the tube 4-6 times until the solution becomes viscous and slightly clear. Incubate for 3 min. at room temperature.
NOTE: do not vortex to avoid shearing chromosomal DNA. Do not incubate for more than 3 min. to avoid denaturation of supercoiled plasmid DNA.
Add 2 mL of the Neutralization Solution and mix immediately by inverting the tube 5-8 times.
Add 0.5 mL of the Endotoxin Binding Reagent. Mix immediately by inverting the tube 5-8 times. Incubate for 5 min at room temperature.
NOTE: after the addition of the Neutralization Solution and Endotoxin Binding Reagent it is important to mix gently, but thoroughly, to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should appear cloudy and contain white precipitate.
Add 3 mL of 96% ethanol. Mix immediately by inverting the tube 5-6 times.
Centrifuge for 40 min. at 4,000-5,000
x
g to pellet cell debris and chromosomal DNA.
Transfer the supernatant to a 15 mL tube (not provided) by decanting or pipetting. Avoid disturbing or transferring the white precipitate. Add 3 mL of 96% ethanol. Mix immediately by inverting the tube 5-6 times.
Transfer part of the sample (~5.5 mL) to the supplied column pre-assembled with a collection tube (15 mL). Be careful not to overfill the column. Centrifuge for 3 min at 2,000
x
g in a swinging bucket rotor. Discard the flow-through and place the column back into the same collection tube.
Repeat the previous step to process any remaining lysate through the purification column.
Add 4 mL of Wash Solution I (diluted with isopropanol) to the purificationcolumn. Centrifuge for 2 min. at 3,000
x
g in a swinging bucket rotor. Discard the flow-through and place the column back into the same collection tube.
Add 4 mL of Wash Solution II (diluted with ethanol) to the purification column. Centrifuge for 2 min. at 3,000
x
g in a swinging bucket rotor. Discard the flow-through and place the column back into the same collection tube.
Repeat the column wash with Wash Solution II
Centrifuge for 5 min. at 3,000
x
g in a swinging bucket rotor to remove residual wash solution. Discard the collection tube containing the flow-through.
Transfer the column into a fresh 15 mL collection tube (provided). Add 0.35 mL of the Elution Buffer to the centre of the purification column membrane. Incubate for 2 min. at room temperature and centrifuge for 5 min at 3,000
x
g in a swinging bucket rotor to elute plasmid DNA.
NOTE: to increase the concentration of eluted DNA the volume of the Elution Buffer can be reduced to 0.25 mL. Be aware that lower volumes of Elution Buffer will decrease the overall yield of eluted DNA.
To increase the overall DNA yield by 20-30% an additional elution step (optional) with Elution Buffer (0.15 mL) may be used.
Discard the purification column. Use the purified plasmid DNA in downstream applications or store DNA at -20°C.
Plate-reader growth curve
Plate-reader growth curve
NOTE: Prepare starter cultures overnight ( starter culture protocol)
Add 230 µL of LB into each well
Use at least 2 wells for blank, add 250 µL LB in these.
Inoculate 18 wells per culture with 10 µL of ths starter culture
An hour after inoculation, the cells are induced in triplo with the following final IPTG-concentrations:
-0 mM
-0.1 mM
-0.3 mM
-0.5 mM
-0.7 mM
-1 mM
After induction, the growth and fluorescence was monitored with a plate-reader.
Preparing coverglass for Widefield-microscope
Preparing coverglass for Widefield-microscope
Concentrate your cells:
Take 1.5 mL from your liquid culture.
Centrifuge for 5 minutes at 3900 rcf to pellet your cells.
Discard part of the supernatant and resuspend the pellet in the remaining supernatant.
Prepare agarose solution:
Add 0.15 g of agarose to 10 mL water
If you want to keep the sample for a longer time, add the agarose to LB or M9-medium
Dissolve the agarose by heating in the microwave.
Fixate your cells:
Take a coverglass and add two strips of parafilm on it, in such a way as to create a channel.
Add ~50 µL of your agarose solution, top it off with your coverslip and let it settle.
Take of the coverslip, add 1 µL of your cells on top of the agarose drop and let it settle (~5 minutes).
Put the coverslip on top of the cells and your coverglass is ready for microscopic analysis.
Preparing sample for TEM
Preparing sample for TEM
Use a 0.2 µm filter to clean your sample.
Ionize your grid with an ionisator.
Add 3 µL of your sample and let it settle for ~1 minute.
Take the liquid of the grid.
Add 3 µL of uranilacetate to the grid and let it settle ~1 minute.
Take the liquid of the grid.
Wash the grid three times with milli-Q and your grid is ready for TEM analysis.
Primer working stock preparation
Primer working stock preparation
Add x  µL of sterile milli-Q to dissolve the DNA material. This creates a 100 µM stock solution.
Heat the primer stock solution to 65 °C for 20 minutes.
Centrifuge the primer stock solution at maximum speed (~17,000
x
g) for 2 minutes.
Prepare a 10x diluted work solution (10 µm) by dilution with sterile milli-Q.
RNA assay
RNA assay
Incubate 8 µL of RNase/nuclease free water with 1 µL RNA and 1 µL sample at 30 °C for 30 minutes.
Purify sample with RNeasy MiniElute Kit.
Mix 3 µL of the sample with 3 µL of loading dye.
Boil mix at 70 °C for 10 minutes and immediately transfer to ice.
Load samples on RNA gel.
RNA concentration measurement (NanoDrop)
RNA concentration measurement (NanoDrop)
Turn on the NanoDrop UV-VIS Spectrophotometer.
Press the button for ssRNA to measure the concentration of single-stranded RNA in your samples.
Clean the measurement surface with a piece of tissue and ethanol.
Use 1-1.5 µL of sterile milli-Q as a blank.
Clean the measurement surface with a piece of tissue and water.
Use 1-1.5 µL of sample to measure its concentration.
NOTE: It is best to measure the same sample in triplo and use the average value.
If you have multiple samples, clean the measurement surface in between measurements.
RNA detection with the coacervation detection method
RNA detection with the coacervation detection method
In the experiment described in detail in this document, we have given a proof of principle that the coacervate-based detection method for RNA cleavage using Cas13a works. This document contains a brief explanation of the coacervate-based detection method using Cas13a, and the experimental setup that was used.
Protocol
Set up the following reaction:
Solution with target
Negative control
10x Cas13a Reaction buffer*
5 μL
5 μL
LwCas13a**
1 μL of a 0.05 wt% stock
1 μL of a 0.05 wt% stock
crRNA
A final concentration of 0.3 ng/μL
-
Target RNA
A final concentration of 0.3 ng/μL
-
PolyU***
A final concentration of 0.1 wt%
A final concentration of 0.1 wt%
Nuclease free water
Up to a final volume of 45 μL
Up to a final volume of 45 μL
*10x Cas13a reaction buffer contains: 400 mM Tris-HCl, 600 mM NaCl, 60 mM MgCl₂, pH 7.3
**Add Cas13a as last, it will trigger the reaction.
***Polyuridylic acid potassium salt dissolved in nuclease free water. A 10 wt% stock was and divided over aliquots that were stored at -20 °C.
Mix gently and incubate both tubes at 37 °C for at least one hour.
Add 5 μL 10 wt% spermine to both tubes and shake thoroughly.
The negative control will start showing coacervates. Depending whether the RNA target is present or not in the other tube, coacervates will form or not. If the solution is as turbid as the negative control, one can conclude that Cas13a has not been active. If the solution is clear, Cas13a has been activated.
RNA electrophoresis
RNA electrophoresis
Preparation of the samples:
Work RNase-free! Treat your workspace with RNaseZAP prior to your experiment and work with gloves. Don't talk while handling your samples!
Prepare the electrophoresis samples in 0.2 mL PCR tubes; 2 µL of 2X RNA Loading Buffer per 2 µL of sample (check the appropriate preparation for each marker).
Put your samples at 90 °C for 5 minutes.
Leave your samples on ice for 2 minutes. Store your samples at 4 °C when not loaded directly on the gel
Preparation of the gel:
Work clean! Handle all material labelled as EtBr contaminated with gloves. Don't take it outside of the EtBr area and don't touch anything that is not labelled as EtBr contaminated with gloves.
Prepare TBE buffer: take the 10X concentrated TBE from the chemicals cabinet and dilute it 10 times with milli-Q.
Weigh agarose for a 2% gel. For 100 mL, 2 g of agarose is necessary.
Mix the TBE solution with the agarose and heat the solution (in a microwave) until it is completely dissolved.
Add 10,000 X SYBR Safe to the solution and mix well. Meaning that for a large gel of 100 mL you have to add 10 µL of SYBR Safe.
Pour the solution into the mould, making sure there are no bubbles. Add a comb to create wells for the samples. Let it solidify (~10 minutes).
RNA electrophoresis:
Transfer the gel to the electrophoresis cell minding the arrow that indicates the direction of DNA/RNA migration. Remove the combs and cover it with TBE-buffer.
Load the RNA ladder in the first well (check the appropriate volume for each marker) and load 4 µL of the samples in the other wells, according to the order in your lab journal.
Connect the cables following the colour code and run at 80 V for 60 min.
RNA isolation
RNA isolation
Adjust the sample to a volume of 100 µL with nuclease free water.
Add 250 µL of absolute ethanol and mix well by pipetting.
Immediately transfer the sample to an RNeasy MinElute spin column placed in a 2 mL collection tube. Centrifuge for 15 s at 8000
x
g.
Place the RNeasy MinElute spin column in a new 2 mL collection tube and add 500 µL RPE buffer. Centrifuge for 15 s at 8000
x
g.
Discard the flow-through and add 500 µL of 80% ethanol to the RNeasy MinElute spin column. Centrifuge for 2 minutes at 8000
x
g.
Place the RNeasy MinElute spin column in a new 2 mL collection tube and centrifuge for 5 minutes at full speed.
Place the RNeasy MinElute spin column in a new 1.5 mL collection tube and add 14 µL nuclease free water. Centrifuge for 1 minute at full speed, your RNA is now in your tube.
RPA
RPA
Prepare the following reaction mix in a 1.5 mL tube:
NOTE: the displayed amounts are for one reaction.
Compound
Volume (µL)
Primer 1 (10 µM)
2.4
Primer 2 (10 µM)
2.4
Rehydration buffer
29.5
DNA template
13.2 µL
Total volume
47.5 µL
Add the reaction mix to the 0.2 mL tubes containing the freeze-dried reaction components. Mix by pipetting.
Take of the lid and pipet 2.5 µL of 280 mM MgAc on it. Spin down the liquid to start the reaction. Mix by vortexing and finish by spinning down all the liquid.
Incubate for 20 minutes at 37 °C.
RPA + in vitro transcription
RPA + in vitro transcription
Thaw the NTP's on ice.
Prepare the following reaction mix in a 1.5 mL tube:
NOTE: the displayed amounts are for one reaction.
Compound
Volume (µL)
Primer 1 (10 µM)
2.4
Primer 2 (10 µM)
2.4
Rehydration buffer
29.5
nuclease free water
2.95
MgCl2 (1 mM)
0.25
NTP's (100 M)
4 X 1
DNA template
3.95
murine RNase inhibitor
4
T7 RNA polymerase
1
Total volume
47.5 µL
Add the reaction mix to the 0.2 mL tubes containing the freeze-dried reaction components. Mix by pipetting.
Take of the lid and pipet 2.5 µL of 280 mM MgAc on it. Spin down the liquid to start the reaction. Mix by vortexing and finish by spinning down all the liquid.
Incubate for 3 hours at 37 °C.
SDS PAGE Electrophoresis
SDS PAGE Electrophoresis
An SDS PAGE electrophoresis is used to seperate proteins on their size by using an electric current. This protocol describes how to prepare SDS PAGE gels, how to prepare samples to run on the gel and how to run an SDS PAGE electrophoresis.
Preparing of SDS PAGE gel
An SDS PAGE gel consist of a Stack gel and a resolving gel. In this protocol we use a 4% Stack gel and a 10% resolving gel (this is for protein samples between 20 kDa and 300 kDa).
Prepare the following gels:
4x stacking gel 4%
4x resolving gel 10%
40% Acrylamide/Bis (37.5:1)
1 mL
6.25 mL
0.5 M Tris-HCl pH 6.8/1.5 M Tris-HCl pH 8.8
2.52 mL (pH 6.8)
6.25 mL (pH 8.8)
10% SDS
0.1 mL
0.25 mL
10%APS
0.05 mL
0.125 mL
TEMED
0.01 mL
0.0125 mL
milli-Q
6.4 mL
12.11 mL
Total
10 mL
25 mL
This is enough for 4 gels of each.
Assemble the moulds and pour the resolving gel in the mould up to
When the gel is soldified,the stacking gel can be poured on top of it and place the gel comb on top of it.
Let it soldify, then remove the mould. The gels are ready for usage, or can be stored at 4 °C wrapped in moisted paper towels for later usage.
Preparing samples
Add 5 μL protein Loading Dye to 20 μL of protein sample.
Incubate the samples for 10 minutes at 95 °C.
Running SDS PAGE gel
Run the gel at 150 V and 25 mA.
If the blue line passed the border between the stacking and resolving gel, run it at 150 V and 35 mA.
The gel is ready when the blu line is almost at the end of the gel.
Processing SDS PAGE gel
Turn off the power.
Take out the gel from the buffer and gently lift one glass plate from the gel.
Remove the remaining gel from the wells.
Fill a small box with a layer of demi water.
Gently take of the gell from the second glass plate into the demi water.
Carefully shake the box a few times to clean the buffer from the gell.
Discard the water from the box.
Fill the box containing the gell with a layer of instant blue (available at Sigma Aldrich ) until the gell is entirely covered.
Let the gell stain on a moving plate for at least 20 minutes (The gell is ready if clearbands can be seen.).
Discard the instant blue and wash the gel a couple of times with demiwater until no instant blue is left on the gel.
Take a picture.
SDS-PAGE for precast gels
SDS-PAGE for precast gels
This SDS-PAGE protocol was used for precast gels by XXX.
Add 5 µL sample buffer (SDS) to 20 µL sample.
Heat at 95 °C for 10 min.
Run Gel at 150 V for 50 min
Remove gel from plastic casing, wash with milliQ and bring milliQ to boil by microwaving for 30 s
Repeat wash process.
Pour hot safeblue stain over the gel and leave gel on bellydancer for 10 min.
Discard stain, cover gel in demi-water and leave to destain for a couple of hours.
Sequencing samples (Macrogen)
Sequencing samples (Macrogen)
Prepare a sequencing sample in a 1.5 mL microcentrifuge tube, labelled with a Macrogen sequencing sticker (QR-code) as follows:
Compound
Volume (µL)
DNA
(~500 ng is required)*
Sequencing primer (10 µM)
2.5
Sterile milli-Q
Up to 10 µL
*Volume depends on concentration of the sample.
Make sure to paste the small sticker in your notebook so that you can trace back which sample your sequence belongs to!
Sticky-end ligation
Sticky-end ligation
Thaw the ligase buffer on ice, to prevent damaging the ATP
Prepare a sample a sample as follows:
Compound
Volume (µL)
DNA vector
(~100 ng is required)*
DNA insert
*
Ligase buffer
2
Ligase T4
1 µL
Sterile milli-Q
Up to 20 µL
*Volume depends on concentration of the sample.
Incubate for at least one hour at 4 °C
TDP purification
TDP purification
Measure the OD of the overnight starter cultures in 10x dilution.
Inoculate 1 L of LB medium supplemented with 1 mL Chloramphenicol to reach an OD of 0.05 and incubate in shaking incubator at 37 °C and 190 rpm.
Perform regular OD measurements to determine when cultures reach the exponential growth phase.
Induce the protein production by adding 1 mL IPTG to the medium when the cells reach the exponential growth phase (OD = 0.4-0.6) and incubate the cultures for four hours at 37 °C and 190 rpm.
Transfer the cultures into 1 L centrifuge flasks after four hours, centrifuge at 1000 g for 30 min at 10ºC  and discard supernatant.
Resuspend cell pellets in in PBS.
Transfer resuspended cells into 50 mL Falcon tubes, centrifuge at 1000 g for 30 min at 10ºC and discard supernatant.
Store pellets in the freezer at -20 °C.
Take cell pellets out of the -20 °C freezer and resuspend in 12.5 mL of 50 mM HEPES and 50 mM NaCl supplemented with half a Roche COmplete Tablet.
Lyse cells by heatshock at 95 °C for 15 min.
Let lysate cool at room temperature for 30 min.
Transfer lysate into 25 mL centrifugation tubes and remove insoluble components by centrifugation at 20.000 g at 10 °C for 30  min.
Add 25 µL of 1 M NaCl and 1250 units of Benzonase and digest the lysate at 37 °C for one hour.
Cool lysate to room temperature.
Transfer lysate to dialysis tubing.
Dialyse against 50 mM sodium phosphate buffer (ph = 7) overnight.
Dialyse lysates against milliQ three times every three hours .
Transfer lysates into 50 mL falcon tubes, snap freeze in liquid nitrogen and lyophilize for 48 hours.
TDP purification (Day 0)
TDP purification (Day 0)
Measure the OD of the overnight starter cultures in 10x dilution.
Inoculate 1 L of LB medium supplemented with 1 mL Chloramphenicol to reach an OD of 0.05 and incubate in shaking incubator at 37 °C and 190 rpm.
Perform regular OD measurements to determine when cultures reach the exponential growth phase.
Induce the protein production by adding 1 mL IPTG to the medium when the cells reach the exponential growth phase (OD = 0.4-0.6) and incubate the cultures for four hours at 37 °C and 190 rpm.
Transfer the cultures into 1 L centrifuge flasks after four hours, centrifuge at 1000 g for 30 min at 10 ºC and discard supernatant.
Resuspend cell pellets in in PBS.
Transfer resuspended cells into 50 mL Falcon tubes, centrifuge at 1000 g for 30 min at 10 ºC and discard supernatant.
Store pellets in the freezer at -20 °C.
TDP purification (Day 1)
TDP purification (Day 1)
Take cell pellets out of the -20 °C freezer and resuspend in 12.5 mL of 50 mM HEPES and 50 mM NaCl supplemented with half a Roche COmplete Tablet.
Lyse cells by heatshock at 95 °C for 15 min.
Let lysate cool at room temperature for 30 min.
Transfer lysate into 25 mL centrifugation tubes and remove insoluble components by centrifugation at 20,000 g at 10 °C for 30  min.
Add 25 µL of 1 M NaCl and 1250 units of Benzonase and digest the lysate at 37 °C for one hour.
Cool lysate to room temperature.
Transfer lysate to dialysis tubing.
Dialyse against 50 mM sodium phosphate buffer (ph = 7) overnight.
TDP purification (Day 2)
TDP purification (Day 2)
Dialyse lysates against milliQ three times every three hours.
Transfer lysates into 50 mL falcon tubes, snap freeze in liquid nitrogen and lyophilize for 48 hours.
Teardrop assay
Teardrop assay
Grow your cells as usual and measure your OD at the start of your assay.
Prepare dilutions up till -12.
Pipet 5 µL of each dilution on an agar-plate and tilt to make the drop run down.
The drop should not reach the edge of the plate
Count the colonies per dilution.
Only colony numbers ranging from 3 to 33 can be counted accurately!
You can now calculate how many viable cells were in your sample at the start of the assay.
Transformation of chemically competent cells
Transformation of chemically competent cells
Get as many aliquots of competent cells (50 µL) from the -80 °C freezer as transformations to be done and put them on ice for 10-15 min.
NOTE: Don’t forget positive and negative controls (no DNA). If commercial competent cells (highly efficient) are used, an aliquot of 50 µL can be split in two equal volumes of 25 µL and used for two transformations.
Add your DNA to the 50 µL of competent cells. 5-7.5 µL of Gibson assembly product should be enough. For miniprepped samples, 2 µL should already be enough.
Incubate on ice for 10-20 minutes.
Heat shock at 42 °C for exactly 45 seconds.
Add 200 µL of fresh, warmed (approx. 37 °C) LB-medium.
Incubate at 37°C with shaking for 1 hour.
Plate the cells out on LB-agar plates with the correct antibiotic to select. You can plate out 75 µL on one plate and the remaining liquid on another.
Incubate at 37 °C overnight. Alternatively, you can incubate over the weekend, leaving the plate on the bench.
Transformation of electrocompetent cells
Transformation of electrocompetent cells
Thaw your cells on ice
Add ~200 ng DNA to 50 µL cells
Transfer the 52 µL to an electro-shock cuvette
Electro-shock the cells with the Electro Cell Manipulator at 25 kV, 200  and 25 µF
!Immediately add 948 µL of recovery medium (eg. SOC-medium)!
Resuspend and transfer to a 1.5 mL tube
Incubate at 37 °C at 200 rpm for 1 hour.
Plate the cells and incubate overnight at 37 °C
Vesicles purification
Vesicles purification
Inocculate your cultures in LB with the corresponding antibiotcs and monitor the growth by measuring the OD.
If required, induce your cells at an OD between 0.3 and 0.7 with 0.1 mM IPTG.
Take an sample of >2 mL after 2 hours and after growth overnight (in the stationary phase) and measure the OD.
Centrifuge at 10,000
x
g. for 10 minutes
Use a 0.45 µm filter to harvest the vesicles.
An additional 0.2 µm filter could be used if a lot of scatter is obtained in your results
We thank our sponsors:
Faculty of Applied Sciences
