<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:TUDelft"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:TUDelft/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:TUDelft/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:TUDelft/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:TUDelft/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=TUDelft"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Case13a</TITLE><HEADER><DIV class="navbar-fixed"><NAV class="row cyan"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Description" class="cyan-text">Description</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-design">Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Results" class="cyan-text">Results</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Demonstrate" class="cyan-text">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Notebook" class="cyan-text">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Protocols" class="cyan-text">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Safety" class="cyan-text">Safety</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Parts" class="cyan-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Basic_Parts" class="cyan-text">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Composite_Parts" class="cyan-text">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Part_Collection" class="cyan-text">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Parts#Improved_Part" class="cyan-text">Improved BioBrick</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Primers" class="cyan-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">Integrated Human Practices (IHP)</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Outreach" class="cyan-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Newsletter" class="cyan-text">Newsletter</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Meet_Up" class="cyan-text">European iGEM Meet Up</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Attributions" class="cyan-text">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Sponsors" class="cyan-text">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Crowdfunding" class="cyan-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Contribution-Overview" class="cyan-text">Contribution Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Interlab" class="cyan-text">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Plot_Grow_Curves" class="cyan-text">Plot Growth Curves</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" class="cyan-text">Find correct CRISPR targets</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Team" class="cyan-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Team#Supervisors" class="cyan-text">Supervisors</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/TeamBuilding" class="cyan-text">Team Building</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Collaborations" class="cyan-text">Collaborations</A></LI></UL><UL id="dropdown-awards" class="dropdown-content nested menu-even-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Entrepreneurship" class="cyan-text">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Software" class="cyan-text">Software tool</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Measurement" class="cyan-text">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Applied_Design" class="cyan-text">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated" class="cyan-text">IHP</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Engagement" class="cyan-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Modeling" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-fullmodel">Modeling</A></LI></UL><UL id="dropdown-fullmodel" class="dropdown-content nested"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate kinetics</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2017.igem.org/Team:TUDelft/Model" class="cyan-text">Overview</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Motif_Finder" class="cyan-text">Motif Finder</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Off_Targeting" class="cyan-text">Off-targeting</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model" class="cyan-text">Lattice Model</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Model/Rate_Kinetics" class="cyan-text">Rate Kinetics</A></LI></UL><UL id="dropdown-design" class="dropdown-content nested menu-larger"><LI><A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" class="cyan-text">Cas13a</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" class="cyan-text">Sample Preperation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#coacervation" class="cyan-text">Coacervation</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#TDP" class="cyan-text">TDPs</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" class="cyan-text">Vesicles</A></LI></UL><UL id="dropdown-achievements" class="dropdown-content"><LI><A href="#!" class="cyan-text dropdown-button" data-hover="hover" data-activates="dropdown-awards">Special Tracks</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Medals" class="cyan-text">Medals</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Awards_Nominations" class="cyan-text">Awards &amp; Nominations</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2017.igem.org/Team:TUDelft" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI><LI><A href="https://2017.igem.org/Team:TUDelft/Giant_Jamboree" class="waves-effect waves-light block">Giant Jamboree</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A href="https://www.facebook.com/TUDelft.iGEM2017/" class="menu-hover waves-effect waves-light block"><SPAN> TUDelft.iGEM2017</SPAN></A></LI><LI title="Click to copy our phonenumber"><A onclick="copyToClipboard('+31 152 78 77 51')" class="menu-hover waves-effect waves-light block" id="safari-phone-hack"><SPAN style="overflow: hidden; display:inline-block;">  +31 152 78 77 51</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light cyan" id="menu-btn"><I class="material-icons">menu</I></A></HEADER><MAIN class="row project"><DIV class="col s12 m10 l8"><MENU id="tocc" class="hide-on-med-and-down"><H5>Results:</H5><UL><LI><A href="#cas13a-scroll">Cas13a</A></LI><LI><A href="#sampleprep-scroll">Sample Preparation</A></LI><LI><A href="#coacervation-scroll">Visible Readout</A></LI><LI><A href="#TDP-scroll">TDPs</A></LI><LI><A href="#vesicles-scroll">Vesicles</A></LI></UL></MENU><H2>Cas13a - Accurate detection of specific genes</H2><P id="cas13a-scroll" class="scrollspy">Cas13a (previously known as C2c2) is a Class 2 type VI-A CRISPR-Cas effector that specifically targets RNA (<A href="#references">O. O. Abudayyeh et al. 2016</A>). Due to the potential use of Cas13a as a broad range detection tool we attempted to turn the protein and its required guide sequence into a BioBrick. The construct was delivered in three fragments that had to be assembled with the backbone into a plasmid. In parallel, we expressed Cas13a from a plasmid (<A href="https://benchling.com/s/seq-66CfLwu7sLMQMbcXe7Ih/edit" target="_blank">pC013</A>) containing <I>Leptotrichia wadei</I> Cas13a (LwCas13a), received from the authors of <A href="#references">Gootenberg et al. 2017</A>.</P><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Multipurpose plasmid construction
                    </DIV><DIV class="collapsible-body"><P>In this section, the synthesis of a BioBrick encoding for Cas13a is explained. Firstly, the sequence for LwCas13a was extracted from <A href="#references">Gootenberg et al. 2017</A>. This DNA construct (7.7 kb), was ordered in three separate parts, denoted part 1 (2.5 kb), part 2 (1.6 kb) and part 3 (0.3 kb). See Figure 1 for an overview of how the assembled plasmid should look like (see the <A href="https://2017.igem.org/Team:TUDelft/Notebook" target="_blank">Cas13a notebook</A> for the entire process of assembling this plasmid). After Gibson Assembly (GA), a PCR was run on the resulting GA mixture and the product was loaded on a gel. This showed that part 2 and part 3 were successfully assembled. </P><DIV class="row center"><DIV class="col s12"><P><B>Figure 1: Schematic overview of our multipurpose Cas13a construct.</B> The insert was delivered in three parts: part 1 (green), part 2 (blue) and part 3 (red). The backbone (<A href="http://parts.igem.org/Part:pSB4A5" target="_blank">pSB4A5</A>) is depicted in yellow.</P></DIV></DIV><P> This part was PCR-amplified from the GA mix (see Figure 2). Subsequently, another Gibson Assembly was done with part 1, the ligated part 2 and part 3 and the backbone (<A href="http://parts.igem.org/Part:pSB4A5" target="_blank">pSB4A5</A>). We transformed the GA mixture several times unsuccessfully (see the <A href="https://2017.igem.org/Team:TUDelft/Notebook" target="_blank">Cas13a notebook</A>). When amplifying the ligation mixture we obtained a band corresponding to the size of 7.7 kb, which indicated the presence of the fully assembled plasmid (Figure 3), at least in linear form. The plasmid was linearly amplified by PCR and <A href="https://2017.igem.org/Team:TUDelft/Protocols#bluntendligation" target="_blank">blunt-end ligation</A> was performed. The ligation was then transformed into BL21 (DE3) cells. Colonies of this transformation were screened with colony PCR which gave a positive result for colony 1 (Figure 4). Plasmid DNA from colony 1 was purified and sent for sequencing, see Figure 5 for the results.
                        </P><DIV class="row"><DIV class="col m4 s12"><P><B>Figure 2: PCR amplification of the ligated part 2 (P2) and part 3 (P3).</B> This was achieved using primers <A href="https://2017.igem.org/Team:TUDelft/Primers" target="_blank">IG0030 and IG0023</A>. The sequence should be around 1800 bp, which is in line with the picture.</P></DIV><DIV class="col m4 s12"><P><B>Figure 3: PCR amplification of the whole plasmid (7.7 kb).</B> Primers<A href="https://2017.igem.org/Team:TUDelft/Primers" target="_blank"> IG0055 and IG0056</A> were used. The Gibson assembly mix was used in different dilutions and in duplo. The dilutions 1:10 have bands of the right size.</P></DIV><DIV class="col m4 s12"><P><B>Figure 4: Colony PCR of colony 1.</B>Primers <A href="https://2017.igem.org/Team:TUDelft/Primers" target="_blank">IG0007 and IG0004</A> (1.5 kb) were used. Colony 1 seemed to contain plasmids with an insert of the right size.</P></DIV></DIV><DIV class="row center"><DIV class="col s12"><P><B>Figure 5: The alignment of the sequencing results.</B>This image shows the alignment of the sequencing results of plasmids from colony 1 with the original insert design in SnapGene. The blue rectangles indicate deletion sites.</P></DIV></DIV><P>The construct was successfully assembled, however two deletions were present. These deletions that are present in the coding sequence of the Cas13a gene caused a frameshift, resulting in a corrupted transcription of the gene. For further experiments, the Cas13a was purified from a plasmid received from the authors of <A href="#references">Gootenberg et al. 2017</A>. This protein is structurally similar to our envisioned BioBrick, except that our BioBrick is codon optimized for bacterial expression instead of mammalian expression and that the CRISPR array lays downstream the Cas13a coding sequence. Please see the details of our construct on the <A href="https://2017.igem.org/Team:TUDelft/Design#cas13a" target="_blank">Cas13a design page</A>.
                        </P><P>The CRISPR array with the interchangeable spacer was made a composite and basic part (<A href="http://parts.igem.org/Part:BBa_K2306013" target="_blank">BBa_K2306013</A> and <A href="http://parts.igem.org/Part:BBa_K2306015" target="_blank">BBa_K2306015</A>). This array can serve as the template for future Cas13a BioBricks. Thanks to the two BsaI restriction sites at each end of the spacer, any guide can be easily be made by restriction ligation of a new spacer. See Figure 6 for a schematic overview of <A href="http://parts.igem.org/Part:BBa_K2306013" target="_blank">BBa_K2306013</A>.</P><DIV class="row center"><DIV class="col s12"><P><B>Figure 6: Schematic overview of the basic part <A href="http://parts.igem.org/Part:BBa_K2306013" target="_blank">BBa_K2306013</A>.</B> It consists of a changeable spacer (blue) flanked by two direct repeats (DR) (orange) and a constitutive promoter (J23119)(white).</P></DIV></DIV></DIV></LI><H5>Achievements</H5><LI><DIV class="collapsible-header">
                        Purification of Cas13a
                    </DIV><DIV class="collapsible-body"><P>The purification protocol we used to purify Cas13a is based on the protocol described in <A href="#references">Gootenberg et al. 2017</A> (find the Cas13a purification protocol <A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank">here</A>). During purification, <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspage" target="_blank">SDS-PAGE analysis</A> was done to check for presence and purity of Cas13a (Figure 7).</P><DIV class="row"><DIV class="center-margin col m12 s12"><DIV class="row align-center"><DIV class="col m5 s12"><P><B>Figure 7: The difference in expression between the bacteria cultures before and after induction with IPTG (BI and AI). </B>The intensity of the bands around 140 kDa show a clear difference before and after induction, which mean that the induction was successful and that the size of the translated protein is right.</P></DIV><DIV class="col m6 s12"><P><B>B: The presence of proteins in the different purification samples.</B> This gels shows the presence of proteins in the supernatant (SN), the first two washing steps (W1 and W2) and the five elution steps (E1-5). It can be seen that clear bands around 140 kDa are present in the elution steps. Also, some fainter bands at different heights can be seen in the elution steps.
                                        </P></DIV></DIV></DIV></DIV><P>From these results we can conclude that Cas13a is expressed in significantly higher quantities after induction with IPTG (although there is some leaky expression). Furthermore we can conclude that purity of the product is increased using the different purification steps that were done.
                        </P></DIV></LI><LI><DIV class="collapsible-header">
                        Proof of Cas13a collateral cleavage activity
                    </DIV><DIV class="collapsible-body"><P>To assess the collateral cleavage activity of the purified Cas13a, we first made use of the fluorescence assay as described by <A href="#references">Gootenberg et al. 2017</A>. For a description of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#RNaseAlert" target="_blank">RNase alert assay</A> (<A href="#references">Thermo Fischer Scientific, 2017</A>), see the Cas13a design page. Reactions with Cas13a were performed with and without target and crRNA (Figure 8).</P><DIV class="row center"><DIV class="col s12"><P><B>Figure 8: The fluorescence of RNase alert over time. </B>Fluorescence indicates for Cas13a activity. It is shown here that there is only activity when both crRNA and target RNA is added to the solution. Values reported here are the measured values of which the blank (only RNase alert) is subtracted.</P></DIV></DIV><P>From this figure we can conclude that Cas13a indeed gets activated by finding its target and shows collateral cleavage. There is relatively more fluorescence when Cas13a is present with crRNA and target, than without them.
                        </P></DIV></LI><LI><DIV class="collapsible-header">
                        Production and testing of different crRNAs
                    </DIV><DIV class="collapsible-body"><P>We employed our crRNA design to design 5 crRNAs that have been successfully synthesized following the
                            <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">

                                crRNA synthesis protocol</A>. The synthesis of crRNA is achieved by annealing two complementary DNA oligos (<A href="https://2017.igem.org/Team:TUDelft/Primers" target="_blank">oligos IG0057 - IG0063</A>) that serve as templates. In vitro transcription was conducted on these DNA oligos. As indicated in Figure 9, we observe bands of the expected length on a gel, confirming crRNA production.
                        </P><P>To test the functioning of these crRNAs, similar fluorescent assays as for testing the functioning of Cas13a were repeated for different crRNAs. The results in Figure 10 show that all tested crRNAs result in Cas13a activation as expected. However, the efficiency of binding the target differs between the crRNAs, with crRNA 3 exhibiting the best target binding.</P><DIV class="row"><DIV class="col m12 s12 center-margin"><DIV class="row align-center"><DIV class="col m6 s12"><P><B>Figure 9: crRNA production. </B>Electrophoresis of 10% PAGE stained with SYBR Gold. A Low Range ssRNA Ladder (LRL) is used. This gel contains the different designed crRNAs synthesized by in vitro transcription with (1-5) and without hairpin (1H-5H). The presence of bands of the expected length in all lanes indicates successful crRNA production.</P></DIV><DIV class="col m6 s12"><P><B>Figure 10: Testing the crRNAs. </B>This graph shows the activity of Cas13a in combination with different crRNAs for the same target designed with the crRNA model. The difference in activity demonstrates the difference in efficiency of the different crRNAs.</P></DIV></DIV></DIV></DIV></DIV></LI></UL><H5>Conclusions</H5><P>Although we did not manage to create our own composite BioBrick part containing Cas13a and the spacer sequence, we were still able to to construct a BioBrick composite part comprising the spacer sequence and a constitutive promoter that can be used in combination with other BioBricks encoding for Cas13a.</P><P>Furthermore, the Cas13a obtained from <A href="#references">Gootenberg et al. 2017</A> was successfully expressed and purified following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#cas13apurification" target="_blank">Cas13a protein purification protocol</A>. We confirmed presence of the protein by running on <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspage" target="_blank">SDS-PAGE</A>. An <A href="https://2017.igem.org/Team:TUDelft/Protocols#RNaseAlert" target="_blank">RNase alert fluorescence assay</A> from <A href="#references">Gootenberg et al. 2017</A> was used to show the functionality of Cas13a. Additionally, the fluorescence assay enabled us to test the efficiency of different guides (different crRNAs) designed using our novel <A href="https://2017.igem.org/Team:TUDelft/Useful_Utilities#Target_Designer" target="_blank">crRNA design tool</A>. </P><H2>Sample preparation - From milk to RNA</H2><P id="sampleprep-scroll" class="scrollspy">Cas13a needs an RNA target to detect. Therefore, the bacterial DNA present in a milk sample first needs to be isolated, amplified and transcribed to RNA. DNA can be isolated by either boiling the cells or by the use of a microwave described by <A href="#references">Dashti et al. 2009</A>. We decided to try out both with consecutive amplification and transcription. This included testing the sensitivity of our methods and testing the target in a cleavage assay using Cas13a. We also put effort into constructing a centrifuge ourselves. Lastly, we used our ‘<A href="https://2017.igem.org/Team:TUDelft/conservedregion" target="_blank"><I><U>motif finder</U></I></A>' to design primers that could be used to amplify resistance genes from mastitis pathogens (<I>bla</I>Z genes).
            </P><H5>Achievements</H5><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Easy DNA isolation
                    </DIV><DIV class="collapsible-body"><P>To see if we were able to use the easy DNA extraction method described by <A href="#references">Dashti et al. 2009</A>, we repeated the control steps of the paper. We tried the boiling- and microwave- method on bacteria of the KEIO-strain (JW0729). We used PCR primers that anneal to the kanamycin resistance gene to test our DNA extraction method.</P><P>We used the microwave and boiling method described in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation</A> protocol. Next, we amplified a specific DNA-sequence of the KEIO-strain with primers <A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0014 and IG0015</A> (<A href="https://2017.igem.org/Team:TUDelft/Protocols#colonypcrgotaq" target="_blank"> GoTaq PCR</A> ). If the primers annealed, a band of size ~750 bp was expected. To be sure that the amplified fragment was the right one, we did a digestion with BsrBI, to obtain DNA-fragments of sizes approximately 250 bp and 500 bp (<A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">Digestion</A>). We checked the sample on a gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaelectrophoresis" target="_blank">DNA electrophoresis</A>), as shown in Figure 11.</P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 11: Digestion of PCR product on DNA extracted by DNA isolation methods. </B> The number of cells are indicated in each lane. The microwave extraction entails using the microwave for 10 seconds, while the longer microwave extraction was for 20 seconds. </P></DIV></DIV><P>We can conclude that both methods are suitable to isolate DNA from gram-negative bacteria. However, the boiling method was more user-friendly. This is because the settings of different microwaves are not always the same. For example, when we tried our protocol with another microwave, the tubes containing the samples burst open.</P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Easy DNA amplification and transcription
                    </DIV><DIV class="collapsible-body"><P>Amplification of DNA, which follows the isolation, is possible by Recombinase Polymerase Amplification (RPA). This isothermal alternative of a PCR reaction, can be applied at constant temperature. In this way, no thermocycler is required.</P><P>The RPA reaction was tested on pSB1T3, a plasmid with a tetracycline resistance gene, following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpa" target="_blank">RPA</A> protocol. We used the designed primers <A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0016 and IG0017</A>, with an expected amplification product of 167 bp. After purification of the samples (<A href="https://2017.igem.org/Team:TUDelft/Protocols#pcrpurification" target="_blank">PCR product purification</A>), the samples were checked on a RNA-gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A>). These results can be found in Figure 12.</P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 12: Target DNA made from pSB1T3. </B> The target DNA was made by RPA. </P></DIV></DIV><P>We successfully amplified a part of the tetracycline resistance gene from pSB1T3. RPA is thus applicable for amplifying DNA at a constant temperature, even though pipetting and mixing all compounds together might require some practice.</P><P>During the RPA reaction, a T7-polymerase binding site can be added to the target DNA by special designed primers. By adding T7-polymerase the DNA can be transcribed into RNA simultaneously.The combined reaction of RPA and T7 polymerase was tested to amplify and transcribe a part of the tetracycline resistance gene present on pSB1T3. First, the DNA was amplified and transcribed into RNA following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol. We used the primers <A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0016 and IG0017</A>, designed with a T7 promoter tail. The expected RNA product was 145 bp. After purification of the sample (<A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> ), the sample was checked on an RNA gel. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A>), as can be seen in Figure 13.</P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 13: Target RNA made from pSB1T3.</B> The target RNA was made by RPA and in vitro transcription at 37 °C. </P></DIV></DIV><P>DNA amplification and in vitro transcription can be combined to obtain the required target RNA. Again, pipetting and mixing all compounds together requires some training, meaning that we need to come up with a solution to reduce the number of pipetting steps.</P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Combining DNA isolation, amplification and transcription
                    </DIV><DIV class="collapsible-body"><P>The next step was to combine the DNA isolation step with the consecutive amplification and transcription step. Therefore, pSB1T3 was transformed into DH5-α and the cells were diluted in milk.</P><P>To assess the sensitivity of the boiling method on gram-negative bacteria in milk, we prepared a dilution range of 10<SUP>-3</SUP> up to and including 10<SUP>-9</SUP> in milk. The DNA was isolated following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">boiling method</A> protocol. </P><P>After amplification and in vitro transcription of the isolated DNA, the RNA was purified and loaded on a gel to check the result (making use of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol and the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol). The results can be found in Figure 14.</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 14: Target RNA made by RPA and in vitro transcription from tetracycline resistant bacteria diluted in milk.</B> Both the isolation method and the RPA and in vitro transcription reaction were tested on pSB1T3 in DH5-α at 37°C. To assess the sensitivity of the method, different dilutions of DH5-α transformed with pSB1T3 were prepared in milk. The dilutions are indicated per lane. </P></DIV></DIV><P>With a <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="_blank">teardrop assay</A>, we estimated the number of cells in the sample we used. There were approximately ~3 x 10<SUP>8</SUP> cells. We were able to isolate, amplify and transcribe DNA up till a dilution of 10<SUP>-8</SUP> in milk, indicating that we could detect our target in the range of 1-10 cells per sample. However, one should keep in mind that the cells contained a high copy number plasmid, which means that our sample contained more DNA per cell than an average biological sample.</P><P>To make our device more user-friendly, we tried the amplification and in vitro transcription at 22 °C (room temperature) for 3 hours, instead of at 37 °C. The results are displayed in Figure 15. These results indicate that the RPA is able to work at room temperature. The T7 RNA polymerase we used on the other hand, has an optimal temperature of 37 °C, and is not functional at room temperature. This means that for our device, we need to come up with a way to regulate the incubation temperature for 3 hours.</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 15: Target RNA made by RPA and in vitro transcription at 22°C from DH5-α transformed with pSB1T3, diluted in milk.</B> Both the isolation method and the RPA and in vitro transcription reaction were tested on pSB1T3 in DH5-α at 22 °C. To assess the sensitivity of the method, different dilutions of DH5-α transformed with pSB1T3 were prepared in milk. The dilutions are indicated per lane. </P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Combining Cas13a with our sample preparation</DIV><DIV class="collapsible-body"><P>The sample preparation is required to obtain the target for Cas13a. Therefore, It is important to find out if, with the current sample preparation, Cas13a is able to recognise the target and will engage in collateral cleavage. To check whether or not Cas13a gets activated by our prepared sample, we used one of our purified RNA samples in the Cas13a activity assay. As can be seen in Figure 16, our Cas13a was activated by our target. This demonstrates that we successfully prepared the sample in such a way that the target became available for Cas13a to detect.</P><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 16: Cas13a assay with target RNA extracted with our boiling DNA isolation method.</B> (blue) Cas13a with crRNA and target isolated from artificially contaminated milk, containing bacteria with the tetracycline resistance gene. (yellow) Cas13a with crRNA and target isolated from milk without bacteria containing the tetracycline resistance gene. (red) Cas13a without crRNA and without target.</P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Applying the sample preparation to a biological sample
                    </DIV><DIV class="collapsible-body"><P>We wanted to test if our sample preparation could be applied on a biological sample, without knowing the sequence beforehand. We decided to validate our sample preparation on mastitis. While talking to veterinarian experts in our <A href="https://2017.igem.org/Team:TUDelft/HP/Gold_Integrated">Integrated Human Practices</A>, we discovered that detecting multi resistance genes such as <I>bla</I>Z and <I>mec</I>A would be most relevant. We used our ‘<A href="https://2017.igem.org/Team:TUDelft/conservedregion" target="_blank"><I><U>motif finder</U></I></A>' to find all the conserved regions of <I>bla</I>Z. On these regions, we designed primers for RPA. We wanted to test if our motif finder would work in practice; ie. that the designed primers would bind to the unknown genome of any pathogen containing the <I>bla</I>Z-gene. We were happy to receive three isolates from the Wageningen Bioveterinary Research in Lelystad; one was confirmed to contain the <I>bla</I>Z-gene, the other two isolates most likely contained the <I>bla</I>Z-gene as well. The exact sequences of the strains were not provided. </P><P>First of all, we wanted to test if our primers would anneal to the unknown genome of the isolates, which most likely contained a variant of the <I>bla</I>Z-gene. Three different DNA isolation methods were used: the boiling method (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation boiling</A>) and two commercial kits (<A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit</A> and <A href="https://2017.igem.org/Team:TUDelft/Protocols#milkdnaisolation" target="_blank">Milk Bacterial DNA Isolation Kit</A>).</P><P>To test the sensitivity of our method, we decided to prepare a dilution range in milk of 10<SUP>-3</SUP> up to and including 10<SUP>-9</SUP> of <I>S. aureus</I>. Next, we used the boiling method to isolate the DNA. (<A href="https://2017.igem.org/Team:TUDelft/Protocols#dnaisolation" target="_blank">DNA isolation boiling</A>). After DNA isolation, the samples were amplified, transcribed into RNA, purified and then loaded on a gel (<A href="https://2017.igem.org/Team:TUDelft/Protocols#rpainvitrotranscription" target="_blank">RPA+in vitro transcription</A> protocol, of the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaisolation" target="_blank">RNA  isolation</A> protocol and the <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnaelectrophoresis" target="_blank">RNA electrophoresis</A> protocol). The samples prepared in LB-medium all gave a positive output. However, all the dilutions prepared in milk did not show a band. We wanted to test if the samples were too diluted already, so we prepared the 10<SUP>-1</SUP> and 10<SUP>-2</SUP> dilutions both in milk and in LB. The results can be found in Figure 17.
                        </P><DIV class="row"><DIV class="col s12"><P><B>Figure 17: Target RNA of <I>bla</I>Z-gene made by RPA and in vitro transcription. </B>(A) To test if the primers we designed on the conserved regions of <I>bla</I>Z work, we extracted DNA from S.aureus (1.0), Coagulase Negative Staphylococcus 1 (2.0) and Coagulase Negative Staphylococcus 2 (3.0) prepared in Tryptic Soy Broth in 3 different ways, namely: the boiling method (0.1), the commercial promega miniprep plasmid isolation kit (0.2) and the norgen milk isolation kit (0.3). (B) To assess the sensitivity of the boiling method, different dilutions of S.aureus were prepared in milk. These dilutions are on indicated per lane. We prepared the 10<SUP>-1</SUP> and 10<SUP>-2</SUP> dilutions of S.aureus both in milk and in LB and performed RPA and in vitro transcription. We used DH5-α transformed with pSB1T3;(Tet) as negative control. </P></DIV></DIV><P>The samples in milk again did not show a band, in contrast to the samples prepared in LB. For the samples prepared in LB, the boiling was long enough to break open the cells. This indicates that the boiling method can be used for gram-positive bacteria in LB. For milk on the other hand, 10 minutes in boiling water was not sufficient. We dove into literature again and found out that milk interferes with DNA isolation methods. We did not experience this when we boiled gram-negative bacteria in milk. However, gram-positive bacteria have a thick peptidoglycan layer in the cell wall, making it harder to lyse the cells. Therefore, for future experiments, we would recommend to boil for at least 15 minutes and immediately put samples on ice for 15 minutes before centrifuging as described by <A href="#references">Ribeiro Junior et al. 2016</A>. Another possible method was explained by <A href="#references">Parayre et al. 2009</A>, which involves the resuspension of the pellet in lysis buffer and the consecutive incubation with proteinase K. This means however, that an extra hour of incubation time needs to be considered and extra pipetting steps.</P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Developing an inexpensive and user-friendly centrifuge
                    </DIV><DIV class="collapsible-body"><P>For the DNA isolation, a centrifuge step is required to spin down the cell debris. We tried to build a centrifuge out of a hard drive, by making it spin on its own. Additionally, we made a hand-powered centrifuge out of a string and around-shaped hard plastic. The hard drive has a default speed which cannot easily be overridden, so we wanted to find out whether the default speed exceeded our ideal speed. For this simple speed test, we broke open the hard drive and marked a certain part of the disk. We filmed the spinning and analysed the video with VLC media player. We subtracted the frame rate (which was 24 Hz) and tried to determine in how many frames the mark would make one round. </P><P>Unfortunately the hard drive spun so fast that the mark already made a whole round in one frame. This means that the hard drive would spin with a speed of at least 24 Hz $\times$ 60 = 1440 rpm, exceeding our ideal speed of 1000 rpm. To be able to control the speed of the hard drive, we need a sensor less brushless motor driver (ESC) to drive the special motor of the hard drive and control this speed with an Arduino. We decided to discard this idea and we continued with our hand-driven centrifuge. </P><P>We started with a preliminary test, to check whether or not we could spin down our suspension of cells by using our centrifuge. By centrifuging for less than a minute, we already made a cell pellet. This process can be seen in figure 18.</P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 18: Cell pellet formed using our hand-powered centrifuge</B></P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                        Developing a protocol for the farmer
                    </DIV><DIV class="collapsible-body"><P>We aimed to make all our steps as easy as possible, as the end user (a farmer) does not have access to specialized lab facilities and equipment. It is very important that the end-user is provided with all the necessary materials and protocols. To make our kit accessible and user-friendly, we decided to buy all the materials in the local shopping centre instead of buying it from brands specialized in lab facilities. Products in stores are already developed to be user-friendly and usable by everyone. Based on the materials provided in the <A href="https://2017.igem.org/Team:TUDelft/Design#sampleprep" target="_blank">kit</A> we worked out a protocol, in easy accessible language and steps. To validate our work we visited Paul at his farm, where he tried out whether or not he could handle the provided materials in our protocol. </P><P>Box content per detection:</P></DIV></LI><LI>- 1x 1.5mL clean green tube for the fresh milk</LI><LI>- 1x 0.5mL clean red tube</LI><LI>- 2x pasteur pipettes 1mL </LI><LI>- 1x White tube with dried compound </LI><LI>T7 polymerase, RPA mix and murine RNA inhibitor</LI><LI>- 1x Mixture A</LI><LI>Rehydration buffer and MgAC</LI><LI>- 1x Mixture B </LI><LI>Spermine stock solution</LI><LI>- 1x Mohr Pipette 50 microliter </LI><LI>- 2x Mohr Pipette 10 microliter</LI><LI>- 1x Thermos Flask</LI><LI>- 1x Immersion heater</LI><LI>- 1x Thermometer</LI><LI>- 1x hand-powered centrifuge</LI><LI>- 1x Tube with positive control</LI><LI>- 1x Tube with negative control </LI><LI>- 1x Detection holder</LI><LI>- Protocol </LI></UL><P>From our work we gained two insights we were able to implement in the protocol: (1) we need to reduce the number of pipetting steps in the protocol for the farmer and (2) in order to let RPA and in vitro transcription work we need to regulate temperature for 3 hours. To reduce the amount of pipetting, we envision in our protocol to provide all the enzymes, primers and NTPs dried in a tube. In this way only the rehydration buffer (with the activation reagent) and the DNA isolated from the milk sample need to be added. Furthermore, we came up with a creative way to regulate the temperature, by using the materials we already had in our toolbox thermos flask. By adding a meat thermometer that can measure up to 120 °C, we can bring the water to the right temperature and keep it around the optimum temperature for RPA an in vitro transcription. </P><P>For future research on DNA isolation from gram-positive bacteria in milk we advise to boil for at least 15 minutes and immediately put samples on ice for 15 minutes before centrifuging. We recommend testing the sensitivity of the whole tool by also testing the RNA prepared from all the different dilutions DNA with Cas13a. Lastly, it would be interesting to test an isothermal T7 RNA polymerase. In this way, not only the RPA-reaction works at room temperature, but the whole RPA and in vitro transcription can be done at this temperature, greatly increasing the user-friendliness of our tool. </P></DIV><H5>Conclusions</H5><P>We experimentally validated that boiling is an easy and effective method to isolate DNA from gram-negative bacteria in milk and gram-positive bacteria in LB. Also, that the combined RPA and in vitro transcription reaction can be performed at a constant temperature. Furthermore, we mapped out and tested all the sample preparation steps to get from gram-negative bacteria with our target gene to our target RNA, which activated Cas13a. We can state that we successfully prepared a biological sample for detection via Cas13a. We translated what we learned into a user-friendly protocol that can be followed by farmers to perform sample preparation on their farm.</P><H2>CINDY Seq - Generating a visible readout</H2><P id="coacervation-scroll" class="scrollspy">The goal of the detection module was to demonstrate our newly invented coacervation method, named Coacervate Inducing Nucleotide Detection of Your Sequence (CINDY Seq). CINDY Seq allows naked-eye detection of target recognition by Cas13a, exploiting the physical phenomenon called “coacervation”. This is the phenomenon that mutually attracting polymers phase-separate into polymer-rich regions (known as coacervates) and polymer-poor regions if the polymers are long enough and the conditions are right. More elaborate description of this method is provided on the <A href="https://2017.igem.org/Team:TUDelft/Design#coacervation">design page</A>, and a theoretical model describing the coacervation process is provided on the <A ref="https://2017.igem.org/Team:TUDelft/Model#coacervation">modelling page</A>.</P><H5>Achievements</H5><UL class="collapsible popout"><LI><DIV class="collapsible-header">Production of visible coacervates</DIV><DIV class="collapsible-body"><P>We started forming coacervates following <A href="#references">Aumiller et al. 2016</A>. They use “long” polymers of uracil (polyU) in combination with spermine, a small positively charged polyamine, to form coacervates. PolyU/spermine coacervates were formed with 0.1 wt% polyU and various spermine concentrations. The absorbance of 500 nm light was measured by UV/Vis spectrophotometry. The results are shown in Figure 19. An interesting observation that follows from these measurements is that there is an optimum of spermine in between 0.1 and 1.0 wt%. This is counterintuitive, as it is hard to imagine that addition of more spermine actually decreases the amount of coacervates. However, this is perfectly in line with theoretical considerations as we explain in the coacervate modelling page.</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 19: Absorbance of polyU/spermine coacervate solutions.</B> Barplot shows the absorbance (500 nm light) of coacervate solutions containing varying amounts of spermine and a fixed (0.1 wt%) polyU concentrations. The result indicates that there is an optimum and that addition of more spermine after 1 wt% in fact decreases the overall turbidity. This result, although counterintuitive, is predicted by theoretical models of coacervates. Error bars indicate standard deviation from the mean of three measurements. </P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Proof of the coacervation method with RNase A</DIV><DIV class="collapsible-body"><P>With coacervates formed, the next step was to test the coacervation method for an unspecific RNase to give a proof of principle. This RNase would eventually be substituted by Cas13a. Besides the proof of principle, the experiments with non-specific RNase (RNase A) would already give us information on timescales, concentrations and methods of measurement. In Figure 20 two tubes are shown, both containing polyU and spermine. To the left tube, RNase A is added. There is clear difference in turbidity between the two tubes, proving that coacervates do not form in the presence of RNase A.</P><P>Now that we were able to prove that RNase can inhibit the formation of coacervates, we did an RNase A titration to determine from which concentration of RNase A the formation of coacervates is inhibited. The results are shown in Figure 21. From these results we can conclude that the formation of coacervates is inhibited above an RNase A concentration of 10<SUP>-4</SUP> wt%. </P><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 20: PolyU/spermine coacervates with (left) and without (right) RNase A.</B> As is directly visible to the naked eye, coacervate solutions of 0.1 wt% polyU and 1.0 wt% spermine are cleared upon addition of 0.05 wt% RNase A. This serves as a first proof of principle that coacervates can be used to indicate RNase activity.</P></DIV><DIV class="col m6 s12"><P><B>Figure 21: Titration of RNase A.</B> Various amounts of RNase A were added to polyU/spermine coacervate solutions and after 5 minutes of equilibrating to 27 °C, the absorbance of 500 nm was measured over the course of 30 minutes. In the case where no or only 10-5 wt% of RNase was added, significant absorbance could be measured throughout the full period of time. For higher concentrations, the absorbance fell to that of the background within the equilibration time.
                                </P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Proof of the coacervation method with activated Cas13a</DIV><DIV class="collapsible-body"><P>In order to transfer the results obtained with RNase A, we tested the effect of spermine on Cas13a. A fluorescence assay with RNase Alert (as described on the Cas13a design page) was done with the addition of 1.0 wt% spermine, demonstrating that Cas13a did not show collateral cleavage with this condition. The effect of spermine was tested with and without the presence of the target RNA. The control consisted of a regular Cas13a assay with and without target RNA. Figure 22 shows the results. We can conclude that spermine inhibits the activity of Cas13a, whereas it does not inhibit RNase A activity. For our purpose, this means that spermine always has to be added after sufficient time has given for Cas13a to collaterally cleave enough RNA. </P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 22: Collateral cleavage reaction is not compatible with 1.0 wt% spermine.</B> The fluorescence assay with and without active Cas13a, and with and without 1.0 wt% spermine indicates that Cas13a does not show collateral cleavage activity in the presence of spermine.</P></DIV></DIV><P>
                            Finally, the <A href="https://2017.igem.org/Team:TUDelft/Results#cas13a" target="blank">functional Cas13a</A> had to be integrated with CINDY Seq, to achieve a full proof of principle. First tests revealed that polyU/spermine coacervates form in the reaction buffer of Cas13a (40 mM Tris-HCl, 60 mM NaCl, 6 mM MgCl2, pH 7.3), but as said, Cas13a does not show activity in presence of 1.0 wt% spermine. Therefore, the experiment was separated into two parts. First, Cas13a with and without target and crRNA was incubated with polyU for varying amounts of time, and subsequently spermine was added to a final concentration of 1.0 wt%. It was observed that after 60 minutes of incubation and subsequent spermine addition, no coacervates were formed in the tube containing activated Cas13a. On the other hand, if spermine was added to the tube with inactive Cas13a, coacervates did form (see Figure 23). </P><P>
                            We can conclude that it is possible to prove the presence or absence of a certain RNA target with the naked eye using the CINDY Seq method. This conclusion can be drawn after at least 45 minutes of incubation, but preferably after 60 minutes (see Figure 24).</P><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 23: CINDY Seq gives a result within the hour.</B> This figure shows the difference in turbidity between solution with and without RNA target after 0 minutes and after 60 minutes of incubation. It is directly visible to the naked eye that when activated Cas13a is given 60 minutes, it is able to cleave the polyU to such an extent that it no longer forms coacervates. As a control, also inactive Cas13a was incubated with polyU and did not show cleavage of polyU to the extent where coacervation no longer occurred. The tubes in which spermine was added after 0 minutes serves as a control.
                                </P></DIV><DIV class="col m6 s12"><P><B>Figure 24: Animated GIF showing the readout of CINDY seq at different timepoints.</B> Mixes with active and inactive Cas13a and 0.1 wt% polyU were kept at 37  °C and supplemented with spermine (to final concentration of 1.0 wt%) at various timepoints. Previous experiments have shown that presence of 1.0 wt% spermine does not allow Cas13a to become active. At the first three timepoints (0, 15 and 30 min), coacervation is still visible in both tubes, indicating that there is still sufficient long polyU to coacervate. However, after 45 min and beyond, a clear solution can be observed in the mix containing active Cas13a, meaning that enough polyU has been cleaved.</P></DIV></DIV></DIV></LI></UL><H5>Conclusions</H5><P>Here we have proven that the ability of Cas13a to collaterally cleave RNA can be combined with the formation of coacervates of long polymers into an easy and cheap detection method. By incubating Cas13a for one hour at 37 °C in solution with 0.1 wt% polyU and subsequently adding spermine to a final concentration of 1.0 wt% the presence or absence of the RNA target can be proven.</P><H2>TDPs - Designing for on-site usage</H2><P id="TDP-scroll" class="scrollspy">To preserve activity of Cas13a and provide our device with a longer shelf life, we investigated the use of Tardigrade intrinsically Disordered Proteins (TDPs), which were found to preserve protein activity after desiccation and subsequent rehydration (<A href="#references">Boothby et al. 2017</A>). We picked four different TDPs, two cytosolic abundant heat soluble (CAHS) and two secretory abundant heat soluble (SAHS) proteins, further referred to as CAHS 94205, CAHS 106094, SAHS 33020 and SAHS 68234.</P><H5>Achievements</H5><UL class="collapsible popout"><LI><DIV class="collapsible-header">Plasmid assembly</DIV><DIV class="collapsible-body"><P>The first step in using TDPs to preserve Cas13a activity was assembling the construct from the gBlocks synthesized by IDT. Since the first stage in the TDP module involved protein purification, we chose the T7 promoter. This strong promoter results in overexpression of the produced proteins. To assemble the parts, a Gibson Assembly (<A href="#references">Gibson et al. 2009</A>) was carried out following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#gibsonassembly">Gibson assembly protocol</A>, and cloned into a TOP10 strain. The resulting colonies were then screened for the inserts after extraction of their DNA and subsequent PCR amplification of the insert. The colonies chosen for sequencing, included in Figure 25, were purified using <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep">Miniprep</A> and subsequently sequenced.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 25: Gel electrophoresis of colony PCR products of the colonies resulting from the transformation of the assembled plasmids.</B>The gels feature ladders (L), negative controls (x1) and the different inserts (1 to 6, respectively CAHS94205_2, CAHS94205, CAHS). Bands with the expected size for the insert are marked with.</P></DIV></DIV><P>The primers used for the PCR were chosen to anneal upstream the prefix and downstream the suffix of the backbone pSB1C3 to screen for the insert. The expected size of the PCR product without insert was 261 bp, while the inserts carrying the CAHS proteins (1.x, and 3.x) and SAHS (5.x and 6.x) had an expected size of ~1000 bp and ~850 bp, respectively. As shown in Figure 25, bands of approximately 1000 bp were found in the colonies 1.5, 3.2, 3.5, 4.6, 4.8 and 4.9 while bands of ~850 bp were visible in the colonies 5.1, 6.1, 6.2, 6.5, 6.6, 6.7 and 6.8. Therefore, the assembly was concluded to have yielded properly assembled plasmids for all inserts, except insert 2. However, both assembly of insert 2 and insert 4 were discarded due to time constraints. Finally, the colonies 1.5, 3.5, 5.1 and 6.2 were picked for further sequencing, where all of them were confirmed to contain the exact desired sequence.
                        </P></DIV></LI><LI><DIV class="collapsible-header">Purification of four TDPs</DIV><DIV class="collapsible-body"><P>After the sequencing results had corroborated a successful cloning, the next step was to evaluate whether the resulting plasmids would produce the expected proteins. For protein purification, the assembled plasmids were transformed into the protein expression strain BL21 (DE3). Subsequently, the <A href="https://2017.igem.org/Team:TUDelft/Protocols#tdppurification">TDP purification protocol</A> was executed and the final protein concentration was measured with a <A href="https://2017.igem.org/Team:TUDelft/Protocols#bradfordassay">Bradford assay</A>. The concentrations of the resulting protein solutions ranged from 1 to 3 g/L.</P><P>We ran <A href="https://2017.igem.org/Team:TUDelft/Protocols#sdspage">SDS-PAGE</A> gels for all four proteins. Additionally, we included a protein solution resulting from the purification of the bacterial strain featuring the same backbone as our BioBricks. As depicted in Figure 26, the samples from the purified TDPs led to a series of additional bands absent in the control, indicating that we successfully produced and isolated all four tardigrade-specific proteins. </P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 26: SDS-PAGE gels of TDPs after protein purification.</B>Gel a features the ladder (L), CAHS 94205 (#1), SAHS33025 (#2) and the control with the backbone pSB1C3 (Blank) while gel b includes CAHS 106094 (#3), the ladder (L) and SAHS 68234 (#4).</P></DIV></DIV><P>In addition to SDS-PAGE of the purified protein solutions, the identity of the proteins was also confirmed by mass spectrometry. To do this, a sequence unique to each TDPs’ amino acid sequence was chosen and screened for their presence in the proteome of the expression strain, BL21 (DE3). Subsequently, four cell cultures expressing a different TDP each were used to screen for all chosen sequences by mass spectrometry (MS) <A href="https://2017.igem.org/Team:TUDelft/Protocols#massspecprep">(see samples preparation)</A>  and an additional culture only featuring the same backbone (pSB1C3) as a control. The peak areas of the resulting mass spectrographs shown in Figure 27, reflecting the occurrence of a given sequence in the sample.</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 27: Bar chart graph including the peak areas of the TDP samples analysed by mass spectrometry after purification. </B>For the control, the expression strain only contained the same backbone.</P></DIV></DIV><P>The unique peptides that were screened for were only present in each TDP expected to contain the sequence. The differences in peak height in Figure 27 can be attributed to the different lengths of the target peptides, which can influence the readout due to the variation of the mass/charge ratios. Hence, it can be concluded that the results were positive and the identity of the proteins could be verified by mass spectrometry.</P></DIV></LI><LI><DIV class="collapsible-header">Preserving lactate dehydrogenase activity with TDPs</DIV><DIV class="collapsible-body"><P>As the production of TDPs by our cloned BioBricks had been validated by different methods, we moved on to evaluate the protective capabilities of our proteins of interest. We performed a series of preliminary activity assays with the enzyme “Lactate Dehydrogenase” (LDH) to determine whether the TDPs conferred desiccation tolerance to non-native proteins following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#ldhassay">LDH assay protocol</A>. In this assay, LDH was dried with and without TDPs and rehydrated before their measurement, so that the protection TDPs offer can be established as the difference between the initial and final enzyme activity. Subsequently, the feasibility of long-term storage with TDPs was studied by measuring the activity of LDH+TDP samples dried and stored at room temperature for different amounts of time, up to a maximum of 18 days. All experiments were carried out in triplicate.
                        </P><P>To the end of assessing the protective effect of the TDPs, all four proteins were tested as buffer excipients (in concentrations ranging from 0.1 to 1 g/L) for the enzyme Lactate Dehydrogenase (LDH) in an enzymatic activity assay based on that described by (<A href="#references">Boothby et al. 2017</A>).</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 28: LDH activity after drying and subsequent rehydration for the different TDPs.</B></P></DIV></DIV><P>As depicted in Figure 28, the LDH activity is least preserved at the lowest TDP concentration for all TDPs and a maximum of activity preserved for the highest concentrations we assessed, with the exception of CAHS 94205, though the error bars should be considered. This indicates that the higher the protein concentration in the solution, the better the activity is preserved; and thus, the better the protection of the enzyme. Such a trend was also observed in our <A href="https://2017.igem.org/Team:TUDelft/Model#">model</A> and by <A href="#References">Boothby et. al. 2017</A>, hence substantiating the validity of our results. </P><P>Furthermore, the preserved enzymatic activity at the highest concentration studied (1 mg/mL), did not differ significantly among the four proteins. It is however notable that, unlike the rest of the proteins, CAHS 106094 did not show a significant preservation of activity at a concentration of 0.1 mg/mL and yielded a much lower preservation at a concentration of 0.5 g/L in comparison to the other TDPs.
                        </P><P>In view of the previous results, we adapted the LDH assay previously carried out to test the protective capability of CAHS 94205 and SAHS 33020 for 18 days and thus evaluate their use for long-term storage at room temperature. Samples similar to those of the previous procedure were prepared and stored dried at room temperature for different amounts of time before their resuspension and measurement.</P><DIV class="row"><DIV class="col s12"><P><B>Figure 29: LDH activity after freeze-drying and drying with TDPs. </B> LDH activity after freeze-drying (FD)/ drying with different concentrations for CAHS 94205 <B>(a) </B> and SAHS 33020 <B>(b)</B> and rehydrating as a function of time. Experiments were performed at room temperature.</P></DIV></DIV><P>As shown in Figure 29, LDH maintains its activity when stored dried with the cytosolic and secretory TDPs, while its activity was not preserved when freeze-dried or dried in the absence of TDPs. Furthermore, it can be seen in Figure 29 that for TDP concentrations of 0.1 mg/mL only SAHS 33020 could maintain a significant amount of activity after a day.
                            </P><P>Although both proteins successfully maintained LDH activity at higher TDP concentrations, the activity decrease over time was substantially lower for a concentration of 1 mg/mL compared to that observed for 0.5 mg/mL. Moreover, for a concentration of 1 mg/mL, it can be concluded that the variations in activity over time for SAHS 33020 are lower than those for CAHS 94205, suggesting that the mechanism by which the SAHS proteins confer their protection is more effective for long periods of time. Our assays reveal that LDH in combination with SAHS 33020 is still active after being stored dried for 18 days at room temperature.</P><P>Nevertheless, the activity drops in the data discussed above could also be due to other factors than just the limitations of the proteins. It should be noted that after drying the samples no additional measures to ensure the dryness of the samples were taken, as these were kept in a drawer instead of in a desiccator. Moreover, interactions with any air components should not be neglected either, as the samples were not purged with an inert gas prior to their storage.</P><P>In future experiments the long-term evolution of the Cas13a activity should be assessed for SAHS 33020 and other tardigrade-specific SAHS proteins as, in view of the long-term results for the TDPs with LDH, these proteins hold great potential in simplifying the storage, usage, and shipment of the many fragile chemicals and biological materials. </P></DIV></LI><LI><DIV class="collapsible-header">Preserving whole cells with TDPs</DIV><DIV class="collapsible-body"><P>As described by (<A href="#references">Boothby et al. 2017</A>), the production of tardigrade-specific proteins by heterologous organisms increases their desiccation tolerance significantly. Therefore, to the end of further characterizing our BioBricks, <A href="https://2017.igem.org/Team:Wageningen_UR">iGEM Wageningen UR</A> was asked to carry out experiments with our plasmids to test the increase in bacterial desiccation tolerance by CAHS94205, CAHS106094 and SAHS33020. Furthermore, for the sake of gaining a better understanding of the proteins’ function, Wageningen performed additional experiments by washing with PBS buffer. This was a procedure that had not been done before. The results are displayed in Figure 30. </P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 30: bacteria desiccation tolerance with TDPs.</B> Number of colonies remaining after desiccation and rehydration for the empty vector, TU Delft’s BioBricks for the production of CAHS 94205, CAHS 106094 and SAHS 33020 in BL21(DE3). Experiments performed by iGEM Wageningen UR.
                                </P></DIV></DIV><P>As seen in Figure 30, the number of colonies found after drying and rehydrating the bacteria (BL21(DE3)) with the TDPs is considerably larger than that for the empty vector of the same bacterial strain (1 vs. 10<SUP>3</SUP>-10<SUP>4</SUP>). This proves that the survival of bacteria is greatly enhanced by the presence of the three TDPs in the cells and complies with the results from <A href="#References">Boothby et al. 2017</A>.</P><P>However, a closer look at the differences in trend between the samples washed with water and PBS reveals a modest increase in the survival of bacteria with plasmids for the production of cytosolic abundant proteins (CAHS) and a decrease for the secretory abundant protein SAHS 33020, which proves that either the proteins or the mechanism by which the protection is conferred are affected by the media used to resuspend.</P></DIV></LI><LI><DIV class="collapsible-header">Preserving Cas13a activity with TDPs</DIV><DIV class="collapsible-body"><P>After evaluating the influence of TDPs on the desiccation tolerance of LDH, an established procedure, we wanted to assess their protective performance with Cas13a. Our <A href="https://2017.igem.org/Team:TUDelft/Model/Lattice_Model">lattice model</A> indicated that it was best to use CAHS 94205 and SAHS 33020 to preserve Cas13a activity at a concentration of 1 g/L or higher. Furthermore, it was observed previously that these two TDPs preserved LDH activity after drying and subsequent rehydration, even after several weeks. </P><P>Both CAHS 94205 and SAHS 33020 were dried with Cas13a in a concentration of 1 g/L, as the protein purification yielded low concentrations for SAHS 33020. Subsequently, Cas13a was rehydrated and its activity measured via an <A href="https://2017.igem.org/Team:TUDelft/Protocols#rnasealert">RNase alert fluorescence assay</A>. All measurements were performed in duplicate.</P><P><B>Figure 31: Cas13a activity with CAHS (94205) (a) and SAHS (33020) (b) in RNase Alert assay after drying and rehydration.</B> SAHS 33020 and CAHS 94205 were dried with Cas13a and then combined with the target. Additionally, the fluorescence activity of the same sample without target, as well as a Cas13a sample with target, but dried without TDPs was measured. Cas13a with the respective TDP and the target without previous drying were used as control.
                        </P><P>As shown in Figure 31a and b, the fluorescence intensity triggered by the RNase-like activity of Cas13a dried without any TDPs is very low in comparison to that of the Cas13a undried and stored frozen. However, while the fluorescence intensity and thus Cas13a activity was higher when Cas13a was dried with CAHS 94205, it also seemed to lose its specificity: the fluorescence intensity after one hour was the same with crRNA and target as without (Figure 31a). However, when the same experiments were repeated with SAHS 33020 instead (Figure 31b), Cas13a remained active after being dried and rehydrated, and the fluorescence intensity observed in the presence of the target and crRNA was clearly higher than in its absence. This indicates that SAHS 33020 can preserve Cas13a activity after desiccation and rehydration, thus keeping the protein functional and opening the possibility of the storage of Cas13a with TDPs.
                        </P><P>In view of the unexpected results observed for CAHS94205 and the fact that <A href="https://2017.igem.org/Team:Munich">iGEM  Munich 2017</A> was also working with Cas13a and similar fluorescence assays, we sent them a newly purified batch of CAHS 94205, to determine whether the same phenomenon would be observed in experiments under the same conditions (see Figure 32).</P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 32: Cas13a activity with CAHS 94205 in RNase Alert Assay by <A href="https://2017.igem.org/Team:Munich">iGEM  Munich 2017</A>. </B>Fluorescence intensities over time triggered by RNase activity of Cas13a after drying with the TDP CAHS 94205, with and without crRNA and target RNA. Data from <A href="https://2017.igem.org/Team:Munich">iGEM Munich 2017</A>.
                            </P></DIV></DIV><P>The results of <A href="https://2017.igem.org/Team:Munich">iGEM Munich 2017</A> also showed the unexpected RNase activity of Cas13a. We therefore hypothesize, that the conformational change in Cas13a, normally caused by the binding of the crRNA with the target, is induced as a result of an interaction between CAHS 94205 and Cas13a upon drying and resuspension. Consequently, CAHS 94205 was ruled out as a potential desiccation tolerance mediator for Cas13a.</P></DIV></LI></UL><H5>Conclusions</H5><P>After receiving gene fragments for the four chosen TDPs and promoters, we assembled the fragments into a pSB1C3 backbone with the T7 promoter and transformed the resulting plasmids into the protein expression strain BL21(DE3). Their successful production was confirmed by both SDS-PAGE and mass spectrometry.</P><P>From the different drying assays we performed, it was concluded that LDH activity was remarkably well preserved after 18 days in the samples containing LDH dried with SAHS 33020. Furthermore, we discovered that while both CAHS 94205 and SAHS 33020 preserved Cas13a’s RNase-like activity after drying, CAHS 94205 could not preserve its specificity; Cas13a dried with the latter CAHS showed RNase activity even in the absence of the target RNA. This was corroborated by <A href="https://2017.igem.org/Team:Munich">iGEM Munich 2017</A>, when the same trend was observed under similar experimental conditions with a Cas13a protein originating from a different strain. Nonetheless, the results were positive for the tardigrade-specific protein SAHS 33020, as both the activity and specificity of Cas13a could be considerably preserved after drying and rehydrating. We therefore believe that the SAHS proteins hold great potential in simplifying the storage, usage and shipment of the many fragile chemicals and biological materials. </P><H2> Vesicles - Exploring the cell's potential as a (future) factory</H2><P id="vesicles-scroll" class="scrollspy">As indicated on the <A href="https://2017.igem.org/Team:TUDelft/Design#vesicles" target="_blank">design page</A>, we wanted to let our bacteria produce all our necessary proteins at once and package them in Outer Membrane Vesicles (OMVs). For this purpose, we tried to establish and characterize hypervesiculation in a strain with a knock-out of an important membrane envelope protein, TolA. Furthermore, we conducted experiments in which another membrane protein, TolR, was overexpressed. It had been suggested that this would further enhance vesiculation.

                </P><H5>Achievements</H5><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                            Confirming the hypervesiculation plasmid
                        </DIV><DIV class="collapsible-body"><P>We received the plasmid with the mutation in the TolR gene from <A href="https://2016.igem.org/Team:UNSW_Australia" target="blank">UNSW Australia 2016</A>. In order to confirm the sequence, we transformed the plasmid into a TOP10 strain. Some colonies were picked and screened with colony PCR (with primers <A href="https://2017.igem.org/Team:TUDelft/Primers" target="blank">IG0028 and IG0027</A>). Simultaneously these colonies were grown and <A href="https://2017.igem.org/Team:TUDelft/Protocols#miniprep" target="_blank">miniprepped</A>, prior to performing a <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion</A> with NdeI and SphI. Colonies 3.1 and 3.2 were picked for sequencing (with primer <A href="https://2017.igem.org/Team:TUDelft/Primers" target="blank">IG0028</A>), where the sequence of both plasmids was further confirmed.</P><DIV class="row"><DIV class="col m10 s12 center-margin"><P><B>Figure 33: Gel electrophoresis of the colony PCR products and restriction assay.</B> (A) The colony PCR products from different transformations. (B) The restriction assay, including non-digested samples as control.</P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">Establishing and characterizing hypervesiculation</DIV><DIV class="collapsible-body"><P>We performed a series of tests to determine if we successfully induced hypervesiculation by the deletion of TolA in the E.coli BW25133 strain from the Keio collection (KEIO) and the overexpression of TolR (<A href="#references">Baba et al. 2006</A>). The following combinations of plasmid and cells were used (Figure 34): pET-Duet with and without the insert of TolR, both in the E. coli BW25133 strain with (KEIO) and without (WT) the deletion of TolA. </P><DIV class="row"><DIV class="col m6 s12 center-margin"><P><B>Figure 34: Scheme of hypervesiculation.</B> Scheme of plasmids and strains used in characterization of hypervesiculation. </P></DIV></DIV><P>
                                By measuring the size distribution of the vesicles with Dynamic Light Scattering (DLS), we wanted to confirm vesicle production and determine vesicle size. Dynamic Light Scatter (DLS) measurements were performed following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="blank">DLS protocol</A>. Vesicle size distribution in the KEIO strain and WT strain with either TolR or pET-Duet were compared at different time points after induction: three, four, five hours and overnight (approximately 20 hours). In Figure 35, the raw data of both TolR and pET-Duet in the KEIO strain clearly show a distribution of larger particles compared to the WT strain. Demonstrating that the WT strain produces a very low amount to no vesicles. Therefore, we decided to only focus on analysing the DLS data of the KEIO strain. </P><DIV class="row"><DIV class="col s12 center-margin"><P><B>Figure 35: Raw data of the DLS experiment. </B> Raw data of the size distributions of the pET-Duet and TolR in the KEIO (a and b) and WT (c and d) strain, measured with Dynamic Light Scatter (DLS). The samples are put against the size in nm and the colour represents the size distribution in percentages. </P></DIV></DIV><P>In Figure 36, the analysed data of the KEIO strain is shown. In this graph the mean and width of the size distribution is plotted per time for TolR and pET-Duet. Overall, neither time nor the presence of TolR shifted size distribution of the vesicles. The only exception is the time point after 4 hours, which might possibly be a statistical st. Therefore, the KEIO strain is mostly responsible for vesicle production. Pertaining to size, the average we found was around 18 nm in diameter (d.nm), which differs from the 80-100 d.nm sized vesicles found by the 2016 iGEM team of <A href="https://2016.igem.org/Team:UNSW_Australia" target="blank">University of New South Wales (UNSW) Australia</A>. Further, they showed that the size distribution shifts to larger vesicles when TolR is overexpressed in the KEIO strain, which was not evident in our data. A possible reason could be the low amount of IPTG we added. However, due to time constraints, we were not able to test a range of different IPTG concentrations.
                            </P><DIV class="row"><DIV class="col s12 m6 center-margin"><P><B>Figure 36: Analysed data of the DLS experiment. </B> Vesicles size after induction of TolR (orange) and pET-Duet (blue) in the KEIO strain obtained by DLS measurements. The mean for each measurement was represented with dots and the width of the distribution with bars. The time points in hours is set against the size in nm. </P></DIV></DIV><P>
                                As shown in previous experiments, both pET-Duet and TolR in KEIO seem to produce vesicles. To confirm that the objects measured by DLS are vesicles and not, for example, parts of the cell, we made negative stain Transmission Electron Microscopy (TEM) images following the <A href="https://2017.igem.org/Team:TUDelft/Protocols#TEMSample" target="blank">TEM protocol</A>. We expected that vesicles have a different shape than the cell debris, namely spherical. Furthermore, when vesicles are big enough you should be able the see the lipid bilayer of the membrane.
                            </P><P>Figure 37 shows PET-Duet in KEIO (a) and TolR in KEIO (b). Vesicles were identified in the images, indicating that the measured objects were not simply cell debris. As shown in the raw DLS data in Figure 35, the range of vesicules go up to approximately 70 d.nm. Due to the resolution limitations of the TEM, we could only visualize vesicles above the average size of 18 d.nm.
                            </P><DIV class="row"><DIV class="col s12 center-margin"><P><B>Figure 37: Transmission Electron Microscop (TEM) images of TolR (left) and pET-Duet (right) in the KEIO strain.</B> The red arrows point to vesicles.</P></DIV></DIV><P>The DLS and TEM experiments do not provide any information about the concentration of produced vesicles. Therefore, vesicle concentration was determined through staining the DLS samples with the membrane dye FM4-64 and subsequent fluorescence measurements in a plate reader. The experiment followed the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Membranestaining" target="blank">membrane staining protocol</A>. In order to calculate the concentration of vesicles from the measured intensity, a calibration curve of synthetic liposomes was made. Figure 38 shows the following linear function which was fitted through the measured data: $I = 4.7865 \cdot C + 950.0159$. In this formula, $I$ is the intensity of the fluorescence in arbitrary units and $C$, the concentration of the liposomes in mg/µL.</P><P>
                                In Figure 39 the concentration of vesicles is represented. It can be seen that little to no vesicles are produced in the wild type strain compared to the KEIO strain. Furthermore, the concentration of vesicles after growing overnight is much higher than 3 hours after induction, thus showing that more vesicles are produced over time. Also, the concentration of vesicles with and without induction differ only with around 2 mg/μL which is not that significant considering the error bars. The reason for this could be the use of a high copy plasmid in combination with a leaky promoter. Besides this, it is confirmed that the presence of the TolR plasmid does not result in large differences in vesicle production.
                            </P><DIV class="row"><DIV class="col m6 s12"><P><B>Figure 38: Liposome calibration curve. </B> Calibration curve of liposomes. The intensity (a.u.) is put against the concentration of liposomes. The liposomes were stained with the membrane dye FM4-64, which only fluoresce when it is bound to the membrane. A linear fit is made, with the formula $I = 4.7865 \cdot C + 950.0159$. </P></DIV><DIV class="col m6 s12"><P><B>Figure 39: Concentration of vesicles.</B> The concentration of vesicles present in the purified samples of TolR and pET-Duet in the KEIO strain and pET-Duet in the WT strain. The concentration, in mg/µL, plotted for the different samples at 3 hours and 20 hours after induction.
                                    </P></DIV></DIV></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                            TorA-GFP plasmid assembly
                        </DIV><DIV class="collapsible-body"><P>We ordered our TorA-GFP from IDT and assembled the part on pSB1C3 by <A href="https://2017.igem.org/Team:TUDelft/Protocols#digestionassay" target="_blank">digestion assay</A> with EcoRI and PstI and consecutive <A href="https://2017.igem.org/Team:TUDelft/Protocols#stickyendligation" target="_blank">ligation</A>. We transformed the construct into a TOP10 strain and screened some colonies with colony PCR (with primers <A href="https://2017.igem.org/Team:TUDelft/Primers" target="blank">IG0006 and IG0013</A>).</P><DIV class="row"><DIV class="col s12 m10 center-margin"><P><B>Figure 40: Gel electrophoresis of colony PCR products resulting for the transformation with the construct.</B> The gel includes a ladder (<B>L</B>), negative controls (<B>C</B>), a blank (<B>B</B>) and the different inserts (<B>1</B> to <B>4</B>) from different ligation. The indicated bands have the expected size.</P></DIV></DIV><P>The indicated bands have an expected band size of ~1500 bp. Colonies 1 and 2 from ligation 1 were picked for sequencing (with primer <A href="”https://2017.igem.org/Team:TUDelft/Primers”">IG0013</A>), where the sequence of both plasmids was further confirmed.</P></DIV></LI></UL><UL class="collapsible popout"><LI><DIV class="collapsible-header">
                            Transporting GFP into the vesicles
                        </DIV><DIV class="collapsible-body"><P>
                                Introducing vesicle production is only the first step in transporting proteins into vesicles. It also requires translocation of the proteins to the periplasm. To achieve this we placed a TorA export tag in front of the protein and this fusion protein was transformed into the KEIO and WT strain, see overview Figure 41. Widefield microscope images were made to visualize whether the addition of the transport tag impaired the GFP structure and therefore its fluorescence. Additionally, an osmo-shock was performed to harvest the periplasmic fraction of the cell. After that, the plate-reader was used to determine the GFP levels in the periplasm and cytoplasm. This will tell us whether the the construct is functional thereby translocating the GFP into the periplasm.
                            </P><DIV class="row"><DIV class="col s12 m10 center-margin"><P><B>Figure 41: Scheme of translocation. </B> Scheme of plasmids and strains used in characterization of TorA-GFP. </P></DIV></DIV><P>
                                We modeled the dynamics of the system to know at what time the concentration of GFP in the periplasm is maximal. For this modeling, we needed to know the OD curve of both the WT and KEIO strain. Due to the fact that we did not know the effect of TorA-GFP on the growth rate, we transformed the plasmid into the strains and measured the OD in the plate-reader. The experiment was performed with the <A href="https://2017.igem.org/Team:TUDelft/Protocols#Growthrate" target="blank">growth rate protocol</A>. In Figure 42, it is shown that the maximal OD is lower for the KEIO strain compared to the WT strain. Also, the growth rate of the KEIO strain in its exponential phase was smaller. With modeling we determined the maximal growth rate to be 1.4 per hour for WT and 0.9 per hour for KEIO.
                            </P><DIV class="row"><DIV class=""><P><B>Figure 42: OD curves.</B> Measured OD curves of the (a) WT and (b) KEIO strain with TorA-GFP, measured in the plate-reader. The OD<SUB>600</SUB> is put against time (hours).</P></DIV></DIV><P> To verify whether the addition of TorA tag to GFP did not impair the functionality of GFP, widefield images were taken with the <A href="https://2017.igem.org/Team:TUDelft/Protocols#teardropassay" target="blank">widefield protocol</A>. As shown in Figure 43, GFP fluorescence was both observed in the KEIO and WT strain. Consequently, It is shown that more GFP is present in WT. Furthermore, you can see that the shape of the KEIO strain cells is elongated. This is probably caused by the deletion of TolA, which destabilizes the membrane (<A href="#refereces">Baker et al. 2014</A>).
                            </P><DIV class="row"><DIV class=""><P><B>Figure 43: Widefield images. </B>Widefield images of GFP-TA in the KEIO (right) and WT (left) strain. </P></DIV></DIV><P>
                                For GFP to be able to be transported into vesicles, it needs to be present in the periplasm. Therefore, an osmoshock was performed using the <A href="https://2017.igem.org/Team:TUDelft/Protocols#osmoshock" target="blank">osmoshock protocol</A>, to check the periplasmic fraction of the KEIO and WT strain, both with and without TorA-GFP. After the osmoshock the fluorescence of the GFP in the periplasm and cytoplasm was measured in the plate-reader.
                            </P><P>
                                In Figure 44, the fluorescence intensity in the periplasm in WT is significantly higher than in the cytoplasm. However, this difference in intensity is much less pronounced in the KEIO strain. A possible explanation for this is the previously demonstrated vesicle production. Previously, it was shown in Figure 40, that the KEIO strain produces a large amount of vesicles, while the WT strain produces none. This result suggests that in the KEIO strain GFP is transported into the vesicles, which reduces the amount of GFP in the periplasm. Another possibility could be that both the growth and the protein production is greatly impaired in the KEIO-strain, due to its mutation. This means that the overall GFP production is lower, leading to a lower concentration in the periplasm as well. All in all, we see that GFP-TA is transported into the periplasm.
                            </P><DIV class="row"><DIV class=""><P><B>Figure 44: GFP fluorescence in the periplasm and cytoplasm. </B>Intensity of GFP in the periplasm and cytoplasm. The fluorescence intensity is measured for the samples TorA-GFP in the WT and KEIO strain and WT ad KEIO strain without plasmid.</P></DIV></DIV><P>
                                In Figure 45, the analysed data is shown. Evidently, GFP is present in the purified sample of GFP-TA in WT, even though previous experiments showed that WT produces little to no vesicles, suggesting that those values are a background from the protocol used, due to a cell destruction during the centrifugation step.To test this hypothesis, the protocol could be adjusted to see whether GFP is still present at lower centrifugation speeds. Regardless of these possible changes, the fluorescence intensity of both GFP-TA with and without TolR in KEIO is significantly higher than GFP-TA in WT. Taken together, we can conclude that GFP is transported into the vesicles.
                            </P><DIV class="row"><DIV class=""><P><B>Figure 45: GFP fluorescence in vesicles. </B>Intensity of GFP in vesicles. The fluorescence intensity of the samples TorA-GFP with TolR in the KEIO strain, TorA-GFP in the KEIO and WT strain and the empty backbone in the WT strain. </P></DIV></DIV></DIV></LI></UL><H5>Conclusions</H5><P>The goal of this module was to transport proteins into vesicles. One of the things we have shown through Dynamic Light Scatter (DLS) experiments is that hypervesiculation occurs in E.coli BW25113 of the Keio strain with a TolA deletion. Produced vesicles have a size of around 18 d.nm. Additionally, by measuring fluorescence intensity in the cytoplasm and periplasm, we have demonstrated that GFP with the TorA tag is transported to the periplasm. To determine if GFP was transported into vesicles, we combined the two plasmids and measured the intensity of GFP in vesicles. <A href="https://2017.igem.org/Team:TUDelft/Model#Ratekinetics" target="blank">Modeling</A> determined that after 25 minutes post induction the concentration of GFP in the vesicles did not increase anymore and reached its maximum. On top of this, it we had shown that the amount of vesicles produced was higher after 20 hours. Combining this, the vesicles we harvested after 20 hours post induction. The same purification method as in the <A href="https://2017.igem.org/Team:TUDelft/Protocols#vesiclespurification" target="blank">DLS experiments</A> was used, after which fluorescence was measured in the plate reader. With these results, we can conclude that the fusion of TorA and GFP is transported into the periplasm.
                </P><DIV id="references" class="section card"><DIV class="card-content"><SPAN class="card-title">References:</SPAN><OL><LI><A href="http://science.sciencemag.org/content/353/6299/aaf5573.long">Abudayyeh, O.O. et al., 2016. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science (New York, N.Y.), 353(6299), p.aaf5573.</A></LI><LI><A href="http://pubs.acs.org/doi/abs/10.1021/acs.langmuir.6b02499">Aumiller, W.M. et al., 2016. RNA-Based Coacervates as a Model for Membraneless Organelles: Formation, Properties, and Interfacial Liposome Assembly. Langmuir, 32(39), pp.10042–10053.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1681482/">Baba, T., et al, 2006. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol. 2.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0958166914000494?via%3Dihub">Baker, J.L., et al, 2014. Microbial biosynthesis of designer outer membrane vesicles. Curr Opin Biotechnol. 29, pp 76-84</A></LI><LI><A href="http://www.cell.com/molecular-cell/abstract/S1097-2765(17)30133-8">Boothby, T.C. et al., 2017. Tardigrades Use Intrinsically Disordered Proteins to Survive Desiccation. Molecular Cell, 65(6), p.975–984.e5.</A></LI><LI><A href="http://www.uel.br/revistas/uel/index.php/semagrarias/article/view/24630">Carlos, J., Junior, R., Tamanini, R., Soares, B. F., Oliveira, A. M. De, Silva, F. D. G., … Beloti, V. 2016. Efficiency of boiling and four other methods for genomic DNA extraction of deteriorating spore-forming bacteria from milk. Semina Ciencias  Agrarias 37, 3069–3078.</A></LI><LI><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5130931/">Dashti, A. A., Jadaon, M. M., Abdulsamad, A. M., and Dashti, H. M., 2009. Heat Treatment of Bacteria: A Simple Method of DNA Extraction for Molecular Techniques. Kuwait Medical Journal 41, 117–122.</A></LI><LI><A href="https://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html">Gibson, D.G. et al., 2009. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nature Methods, 6(5), pp.343–345.</A></LI><LI><A href="http://science.sciencemag.org/content/356/6336/438">Gootenberg, J.S. et al., 2017. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, 356(6336), pp.438–442.</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0167701207000607?via%3Dihub">Parayre, S., Falentin, H., Madec, M-N., Sivieri, K., Le Dizes, A-S., Sohier, D., and Lortal, S., 2007. Easy DNA extraction method and optimization of PCR-Temporal Temperature Gel Electrophoresis to identify the predominant high and low GC-content bacteria from dairy products. Journal of Microbiological Methods 69, 431-441.</A></LI><LI><A href="https://www.thermofisher.com/order/catalog/product/AM1964">Thermo Fischer Scientific, 2017. RNaseAlert Lab Test Kit. Available at: https://www.thermofisher.com/order/catalog/product/AM1964 [Accessed October 23, 2017]</A></LI></OL></DIV></DIV></MAIN></DIV><FOOTER><H4 class="center">We thank our sponsors:</H4><DIV class="sponsors center"><DIV><A href="https://www.tudelft.nl/en/faculty-of-applied-sciences/">Faculty of Applied Sciences</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>