<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bordeaux_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bordeaux/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="intro-loader"><DIV class="DNA_cont"><DIV style="position: relative; top: 140px; line-height: 50px;"><H2 style="color:#ff727d; font-size: 45px">Loading...</H2></DIV></DIV></DIV><DIV class="acontainer menupop"><A class="buttonpop dl-trigger" href="#popup">☰ Menu</A><DIV class="popup" id="popup"><DIV class="popup-inner"><DIV class="popup__text"><UL id="menupop" class="tree"><LI><A id="menu-home" href="https://2018.igem.org/Team:Bordeaux">HOME</A></LI><LI><A id="menu-team" class="clickbelow">TEAM &amp; FRIENDS</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Crowdfunding">Crowdfunding</A></LI></UL><LI><A id="menu-project" class="clickbelow">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Attributions">Attributions</A></LI><LI><A id="menu-parts" href="https://2018.igem.org/Team:Bordeaux/Parts">PARTS</A></LI><LI><A id="menu-safety" href="https://2018.igem.org/Team:Bordeaux/Safety">SAFETY</A></LI><LI><A id="menu-hp" class="clickbelow">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Bordeaux/Public_Engagement">Education &amp; Engagement</A></LI><LI><A id="menu-award" class="clickbelow">AWARDS</A></LI><LI><A href="#">Applied Design</A></LI><LI><A href="#">Measurement</A></LI><LI><A href="#">Model</A></LI><LI><A id="menu-somfy" href="https://2018.igem.org/Team:Bordeaux">JUDGING FORM</A></LI></DIV><A class="popup__close" href="#">✕</A></DIV></DIV></DIV><DIV class="container text_intro"><H1 class="text-longshadow">Results</H1></DIV><DIV class="bg-color2 section-padding separator"><DIV class="container"><DIV class="row"><DIV class="col-12"><H1 class="section-title">Our results</H1></DIV><DIV class="col-12"><P>
                        Unfortunately, due to unsuccessful cloning, no measurements were obtained. Despite the several protocols tested, no cloning
                        have been confirmed.
                    </P><P><B>We do not know why the cloning did not work.</B></P><P>With the AQUA cloning, we frequently obtained red colonies, which is a strong indication that pSB1C3
                        with the RFP was integrated in E. coli. The issue here is that we used the linear plasmid provided
                        by iGEM. The latter did PCR on pSB1C3 with RPF to provide us the linear plasmid. Some template should
                        have stay in the linear plasmid that they sent us. To counter this phenomenon we used Dpn1 to digest
                        the circular plasmid following the right protocol but it still remains some template.</P><P>AQUA cloning and enzymatic cloning were equally concerned about problem on analysis. As soon as colonies
                        were present, we performed colony PCR which gave nothing and then when we did enzymatic digestions
                        the results were inexplicable. It was no contamination because the negative control was always empty,
                        meaning that the issue was not with the water, the primers or the buffers. The stripes were never
                        the right length and yet we tried numerous enzymes and enzymes blends. It could be non specific hybridization
                        but we verified with the ApE software and did not find any hybridation area. Even when a single enzyme
                        was used and should have given us the length of the plasmid with the insert, sometimes several stripes
                        were present. It was like there was a contamination or that two empty plasmid bound. We do not understand
                        how it could be something else than our plasmid because it should have possessed the chloramphenicol
                        resistance.
                    </P><P>In a nutshell, the profiles did not correspond to the expected clones and the analysis by sequencing
                        did not help to understand what was going on with these clonings.</P><P>Regarding the chemistry part of the project, it is a real shame that so many problems with the HPLC occured
                        because it does not allow any conclusion. The issue with the acid hydrolysis is that initially it
                        is used to produce sugars and that HMF is just an unwanted co-product. The quantity of HMF is very
                        low and is insufficient to be quantified. Originally, we aim to use ionic liquids which are present
                        in a significant amount of scientific articles but we learn that they are very expensive and they
                        need chrome salts which are banned at the industry level. Since our purpose was to adapt our method
                        in an industrial way, there was not interest in using ionic liquid even if they worked well.</P></DIV></DIV></DIV></DIV><DIV class="bg-color1 section-padding separator"><DIV class="container"><DIV class="row"><DIV class="col-12"><H1 class="section-title">Toxicity results coming from IGEM Leiden</H1></DIV><DIV class="col-12"><P>
                        Hydroxymethylfurfural (HMF) is an organic compound of the furfural family formed by dehydration of sugars (1). It is a multifunctional
                        molecule because of its structure (Fig 1),It has a furan ring with an aromatic alcohol and an aromatic
                        aldehyde. HMF is a versatile intermediate allowing to obtain by a transformation some high value-added
                        chemical. It is a valuable and interesting molecular building block which can be catabolized in some
                        useful molecule such as 2,5-furandicarboxylic acid (FDCA) (2).
  </P><FIGURE class="figure col-md-6 offset-md-3"><FIGCAPTION class="figure-caption">Figure 1 : Structure of Hydroxymethylfurfural (HMF)
                        </FIGCAPTION></FIGURE><P>
                        Our project is about producing a bacterial strain that transforms hydroxymethylfurfural (HMF) into furandicarboxylic acid
                        (FDCA). HMF is a byproduct of the degradation of cellulose or other polysaccharides. FCDA is a platform
                        molecule that can be polymerized into plastics or biofuels.
                    </P><P>Our researches in the literature on the enzymes that can catalyse the reaction from HMF to FDCA, have
                        shown that in some cases and for some organisms it may be toxic.</P><P>Several studies in different strains of Saccharomyces cerevisiae show that cell growth was significantly
                        affected (3). Studies also show mutagenic effects on human cells (3). HMF is also known to cause
                        specific damage on microorganisms with DNA mutations in Salmonella typhimurium (7) or Escherichia
                        coli (8). But some studies also show that for some strains of E. coli or some others bacteria, there
                        is not effect of toxicity (9).</P><P>The effects come from a reduction in available cellular energy caused by the inhibition of enzymes such
                        as alcohol dehydrogenase, aldehyde dehydrogenase, pyruvate dehydrogenase (4), and two key glycolytic
                        enzymes (hexokinase and glyceraldehyde-3-phosphate dehydrogenase) (5). The reactive aldehyde groups
                        in furfural family increases accumulation of reactive oxygen species (ROS). Consequences of ROS are
                        well known for causing DNA damage, membrane damage and protein misfolding (6).</P><P>We decided to investigate the toxicity of HMF. We found out through social media, that the project of
                        iGEM Leiden involved the detection of bacterial cell stress. Because HMF is a known to be a toxin,
                        we suggested that it would be interesting for us to test this on iGEM Leiden’s bacterial stress detection
                        system, and they wholeheartedly agreed.
                    </P><P>Following a skype session to discuss the details of our collaboration, we send them 2mL of solution containing
                        5 g/L HMF.</P><P>It allowed them to test the compound at various concentrations on four of their stress detection strains:
                        pDnaK (10), pRpoE (11), pSoxS (12) and pCspA (13).
                    </P><P>Results of exposure to HMF show no stress reaction for the pCspA-GFP (figure 2A) and pRpoE-GFP (figure
                        2C) strains. CspA is involved in metabolism related stress and RpoE is involved in protein synthesis
                        stress. Because both strains did not invoke a stress reaction, there is no indication that HMF delivers
                        stress in these pathways. However, to firmly conclude these stresses are not being experienced, more
                        extensive testing is needed, since these are just two promoters involved in these stress pathways.
                    </P><P>Both the pDnaK-GFP (figure 2B) and pSoxS-GFP strains (figure 2D) do show some responses when exposed
                        to 0.5µg/mL HMF. DnaK is involved in the process of chromosomal DNA replication and SoxS is involved
                        in DNA damage repair. Therefore, these results indicate that HMF delivers stress to bacteria by damaging
                        their DNA and disturbing their replication process.</P><P> However, the stress being experienced at these HMF concentrations seems to be low. They shared these
                        results with us which help us with our project, provide us with insight into the effect of HMF on
                        bacterial cell processes. We are very thankful for this meaningful collaboration and look forward
                        to meeting their team again in Boston.
                    </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Figure 2 : pDnaK-GFP and pSoxS-GFP show a slight stress response to HMF Four stress detecting strains
                            were exposed to multiple concentrations of HMF. Figure 2A shows little to no response of the
                            pCspA-strain to all 5 concentrations of HMF. Figure 2B shows a reaction of the pDnaK-strain when
                            exposed to 0.5 µg/mL HMF. Figure 2C shows no reaction of the pRpoE-strain for all 5 HMF concentrations.
                            Figure 2D shows a reaction of the pSoxS-strain when exposed to 0.5µg/mL HMF
                        </FIGCAPTION></FIGURE><P>From their stress detection strains, pCspA-GFP and pRpoE which are involved in stress metabolism did
                        not show indication that HMF delivers stress. But, pDnaK-GFP and pSoxS-GFP (which are involved in
                        DNA damage) show some responses.
                    </P><P>It would be great to investigate more the toxicity of HMF with an other assay (like Minimal Inhibitory
                        Concentration Assay) (14), we could do more tests and add some statistics tests for more relevance.
                    </P><DIV id="ref" style="margin-top: 50px"><H4>References :</H4><OL type="1"><LI>Van Putten, Robert-Jan, Van der Waal, Jan C.; de Jong, Ed; Rasrendra, Carolus B.; Heeres, Hero
                                J.; de Vries, Johannes G. (2013). &quot;Hydroxymethylfurfural, A Versatile Platform Chemical Made
                                from Renewable Resources&quot;. Chemical Reviews. 113 (3): 1499–1597. doi:10.1021/cr300182k. ISSN
                                0009-2665.
                            </LI><LI>Bicker, M.; Hirth, J.; Vogel, H. Dehydration of fructose to 5-hydroxymethylfurfural in sub- and
                                supercriticalacetone.Green Chem.2003,5, 280–28
                            </LI><LI>Rosatella, A. A., Simeonov, S. P., Frade, R. F. M., &amp; Afonso, C. A. M. (2011). 5-Hydroxymethylfurfural
                                (HMF) as a building block platform: Biological properties, synthesis and synthetic applications.
                                Green Chemistry, 13(4), 754. doi:10.1039/c0gc00401d
                            </LI><LI>Modig T, Lidén G, Taherzadeh MJ (2002) Inhibition effects of furfural on alcohol dehydrogenase,
                                aldehyde dehydrogenase and pyruvate dehydrogenase. Biochem J 363:769–776</LI><LI>Banerjee N, Bhatnagar R, Viswanathan L (1981) Inhibition of glycolysis by furfural in Saccharomyces
                                cerevisiae. Appl Microbiol Biotechnol 11:226–228</LI><LI>Perrone GG, Tan SX, Dawes IW (2008) Reactive oxygen species and yeast apoptosis. Biochim Biophys
                                Acta 1783:1354–1368
                            </LI><LI>Zdzienicka M, Tudek B, Zielenska M, Szymczyk T (1978) Mutagenic activity of furfural in Salmonella
                                typhimurium TA100. Mutat Res 58:205–209</LI><LI>Khan QA, Hadi SM (1993) Effect of furfural on plasmid DNA. Biochem Mol Biol Int 29:1153–1160</LI><LI>Takahashi CM, Takahashi DF, Carvalhal ML, Alterthum F: Effects of acetate on the growth and fermentation
                                performance of Escherichia coli KO11. Appl Biochem Biotechnol 1999, 81: 193-203. 10.1385/ABAB:81:3:193</LI><LI><A href="https://www.uniprot.org/uniprot/P0A6Y8" style="color: white!important">https://www.uniprot.org/uniprot/P0A6Y8</A></LI><LI>Aaron M. Nuss,Jens Glaeser, and Gabriele Klug, RpoHII Activates Oxidative-Stress Defense Systems
                                and Is Controlled by RpoE in the Singlet Oxygen-Dependent Response in Rhodobacter sphaeroides
                                J Bacteriol. 2009 Jan; 191(1): 220–230. doi: 10.1128/JB.00925-08
                            </LI><LI>Wu, J., and Weiss, B. Two divergently transcribed genes, soxR and soxS, control a superoxide
                                response regulon of Escherichia coli. (1991) J. Bacteriol. 173,2864-2871.</LI><LI>Yamanaka K, Inouye M. Induction of CspA, an E. coli major cold-shock protein, upon nutritional
                                upshift at 37 degrees C. Genes Cells. 2001 Apr;6(4):279-90.</LI><LI>Joana P. C. Pereira, Peter J. T. Verheijen, and Adrie J. J. Straathof, Growth inhibition of S.
                                cerevisiae, B. subtilis, and E. coli by lignocellulosic and fermentation products, Appl Microbiol
                                Biotechnol. 2016; 100(21): 9069–9080 doi: 10.1007/s00253-016-7642-1</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>