<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Notebook_Molecular skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Notebook/Molecular</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model">Model </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Impact">Impact</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Activities">Activities</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Team">Members</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Contribution</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/InterLab">Interlab</A></LI><UL class="nav navbar-nav navbar-right hidden-sm hidden-md"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><DIV class="row"><DIV class="col-xs-10"><H1>

Molecular Experiment

        </H1><P>

Notebook

        </P></DIV></DIV></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#2015"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">2015</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Dec."><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Dec.</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#13th"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">13<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-4"><A href="#Geco_plasmid_construction"><SPAN class="tocnumber">1.1.1.1</SPAN><SPAN class="toctext">Geco plasmid construction</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-5"><A href="#15th"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">15<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-6"><A href="#Geco_plasmid_construction_2"><SPAN class="tocnumber">1.1.2.1</SPAN><SPAN class="toctext">Geco plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#16th"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">16<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Geco_plasmid_construction_3"><SPAN class="tocnumber">1.1.3.1</SPAN><SPAN class="toctext">Geco plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#17th"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">17<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Geco_plasmid_construction_4"><SPAN class="tocnumber">1.1.4.1</SPAN><SPAN class="toctext">Geco plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#18th"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">18<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Geco_plasmid_construction_5"><SPAN class="tocnumber">1.1.5.1</SPAN><SPAN class="toctext">Geco plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#29th"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">29<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Piezo_plasmid_construction"><SPAN class="tocnumber">1.1.6.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#30th"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">30<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Piezo_plasmid_construction_2"><SPAN class="tocnumber">1.1.7.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#31th"><SPAN class="tocnumber">1.1.8</SPAN><SPAN class="toctext">31<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#Piezo_plasmid_construction_3"><SPAN class="tocnumber">1.1.8.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#2016"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">2016</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Jan."><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Jan.</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#21th"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">21<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#Piezo_plasmid_construction_4"><SPAN class="tocnumber">2.1.1.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#22th"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">22<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#Piezo_plasmid_construction_5"><SPAN class="tocnumber">2.1.2.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-4 tocsection-25"><A href="#Piezo_plasmid_construction_6"><SPAN class="tocnumber">2.1.2.2</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-4 tocsection-26"><A href="#Materials_sorting:"><SPAN class="tocnumber">2.1.2.3</SPAN><SPAN class="toctext">Materials sorting:</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#25th"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">25<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-28"><A href="#Piezo_plasmid_construction_7"><SPAN class="tocnumber">2.1.3.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#26th"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">26<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-30"><A href="#Piezo_plasmid_construction_8"><SPAN class="tocnumber">2.1.4.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#27th"><SPAN class="tocnumber">2.1.5</SPAN><SPAN class="toctext">27<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-32"><A href="#Piezo_plasmid_construction_9"><SPAN class="tocnumber">2.1.5.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#28th"><SPAN class="tocnumber">2.1.6</SPAN><SPAN class="toctext">28<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-34"><A href="#Piezo_plasmid_construction_10"><SPAN class="tocnumber">2.1.6.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#29_th"><SPAN class="tocnumber">2.1.7</SPAN><SPAN class="toctext">29 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-36"><A href="#Piezo_plasmid_construction_11"><SPAN class="tocnumber">2.1.7.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#30th_2"><SPAN class="tocnumber">2.1.8</SPAN><SPAN class="toctext">30<SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-38"><A href="#Piezo_plasmid_construction_12"><SPAN class="tocnumber">2.1.8.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#31th_2"><SPAN class="tocnumber">2.1.9</SPAN><SPAN class="toctext">31<SUB>th</SUB></SPAN></A></LI><LI class="toclevel-4 tocsection-40"><A href="#Piezo_plasmid_construction_13"><SPAN class="tocnumber">2.1.9.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Feb."><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Feb.</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#01_st"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">01 <SUP>st</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-43"><A href="#Piezo_plasmid_construction_14"><SPAN class="tocnumber">2.2.1.1</SPAN><SPAN class="toctext">Piezo plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#14_th"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">14 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-45"><A href="#TRPC5_plasmid_construction"><SPAN class="tocnumber">2.2.2.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#15_th"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">15 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-47"><A href="#TRPC5_plasmid_construction_2"><SPAN class="tocnumber">2.2.3.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#16"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">16</SPAN></A></LI><LI class="toclevel-4 tocsection-49"><A href="#TRPC5_plasmid_construction_.3D"><SPAN class="tocnumber">2.2.4.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction =</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#17_th"><SPAN class="tocnumber">2.2.5</SPAN><SPAN class="toctext">17 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-51"><A href="#TRPC5_plasmid_construction_3"><SPAN class="tocnumber">2.2.5.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#18_th"><SPAN class="tocnumber">2.2.6</SPAN><SPAN class="toctext">18 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-53"><A href="#TRPC5_plasmid_construction_4"><SPAN class="tocnumber">2.2.6.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#19_th"><SPAN class="tocnumber">2.2.7</SPAN><SPAN class="toctext">19 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-55"><A href="#TRPC5_plasmid_construction_5"><SPAN class="tocnumber">2.2.7.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#20_th"><SPAN class="tocnumber">2.2.8</SPAN><SPAN class="toctext">20 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#21_th"><SPAN class="tocnumber">2.2.9</SPAN><SPAN class="toctext">21 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#22_th"><SPAN class="tocnumber">2.2.10</SPAN><SPAN class="toctext">22 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-59"><A href="#TRPC5_plasmid_construction_6"><SPAN class="tocnumber">2.2.10.1</SPAN><SPAN class="toctext">TRPC5 plasmid construction</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#24_th"><SPAN class="tocnumber">2.2.11</SPAN><SPAN class="toctext">24 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#April"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">April</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#13_th.7E24_th"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">13 <SUP>th</SUP>~24 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-63"><A href="#087.26NFAT_Plasmid_Construction"><SPAN class="tocnumber">2.3.1.1</SPAN><SPAN class="toctext">087&amp;NFAT Plasmid Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#July"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">July</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#0_1_st"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">0 1 <SUP>st</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#02_nd"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">02 <SUP>nd</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#03_th"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">03 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-68"><A href="#Non-green_colony_test_.28II.29"><SPAN class="tocnumber">2.4.3.1</SPAN><SPAN class="toctext">Non-green colony test (II)</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#04_th"><SPAN class="tocnumber">2.4.4</SPAN><SPAN class="toctext">04 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-70"><A href="#06_th"><SPAN class="tocnumber">2.4.5</SPAN><SPAN class="toctext">06 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#07_th-10_th"><SPAN class="tocnumber">2.4.6</SPAN><SPAN class="toctext">07 <SUP>th</SUP>-10 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-72"><A href="#11_th"><SPAN class="tocnumber">2.4.7</SPAN><SPAN class="toctext">11 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#12_th"><SPAN class="tocnumber">2.4.8</SPAN><SPAN class="toctext">12 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-74"><A href="#13_th"><SPAN class="tocnumber">2.4.9</SPAN><SPAN class="toctext">13 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-75"><A href="#16_th_-_21_th"><SPAN class="tocnumber">2.4.10</SPAN><SPAN class="toctext">16 <SUP>th</SUP> - 21 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-76"><A href="#22_th_-_23_th"><SPAN class="tocnumber">2.4.11</SPAN><SPAN class="toctext">22 <SUP>th</SUP> - 23 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-77"><A href="#24_th_-_25_th"><SPAN class="tocnumber">2.4.12</SPAN><SPAN class="toctext">24 <SUP>th</SUP> - 25 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-78"><A href="#26_th_-_30_th"><SPAN class="tocnumber">2.4.13</SPAN><SPAN class="toctext">26 <SUP>th</SUP> - 30 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Aug."><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Aug.</SPAN></A></LI><LI class="toclevel-3 tocsection-80"><A href="#01_st_-_02_nd"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">01 <SUP>st</SUP> - 02 <SUP>nd</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-81"><A href="#03_rd_-_04_th"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">03 <SUP>rd</SUP> - 04 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-82"><A href="#05_th"><SPAN class="tocnumber">2.5.3</SPAN><SPAN class="toctext">05 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-83"><A href="#06_th_-_07_th"><SPAN class="tocnumber">2.5.4</SPAN><SPAN class="toctext">06 <SUP>th</SUP> - 07 <SUP>th </SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-84"><A href="#09_th"><SPAN class="tocnumber">2.5.5</SPAN><SPAN class="toctext">09 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-85"><A href="#10_th"><SPAN class="tocnumber">2.5.6</SPAN><SPAN class="toctext">10 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#11_th_-_13_th"><SPAN class="tocnumber">2.5.7</SPAN><SPAN class="toctext">11 <SUP>th</SUP> - 13 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-87"><A href="#22_th_2"><SPAN class="tocnumber">2.5.8</SPAN><SPAN class="toctext">22 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-88"><A href="#23_th"><SPAN class="tocnumber">2.5.9</SPAN><SPAN class="toctext">23 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-89"><A href="#24_th_2"><SPAN class="tocnumber">2.5.10</SPAN><SPAN class="toctext">24 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-90"><A href="#25_th"><SPAN class="tocnumber">2.5.11</SPAN><SPAN class="toctext">25 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-91"><A href="#26_th"><SPAN class="tocnumber">2.5.12</SPAN><SPAN class="toctext">26 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-92"><A href="#28_th"><SPAN class="tocnumber">2.5.13</SPAN><SPAN class="toctext">28 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-93"><A href="#29_th_2"><SPAN class="tocnumber">2.5.14</SPAN><SPAN class="toctext">29 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-94"><A href="#30_th"><SPAN class="tocnumber">2.5.15</SPAN><SPAN class="toctext">30 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-95"><A href="#31_th"><SPAN class="tocnumber">2.5.16</SPAN><SPAN class="toctext">31 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#Sept."><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Sept.</SPAN></A></LI><LI class="toclevel-3 tocsection-97"><A href="#01_st_2"><SPAN class="tocnumber">2.6.1</SPAN><SPAN class="toctext">01 <SUP>st</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-98"><A href="#04_th_2"><SPAN class="tocnumber">2.6.2</SPAN><SPAN class="toctext">04 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-4 tocsection-99"><A href="#Gibson_Assembly.C2.AE_Master_Mix"><SPAN class="tocnumber">2.6.2.1</SPAN><SPAN class="toctext"><B>Gibson Assembly® Master Mix</B></SPAN></A></LI><LI class="toclevel-3 tocsection-100"><A href="#05_th_2"><SPAN class="tocnumber">2.6.3</SPAN><SPAN class="toctext">05 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-101"><A href="#06_th_2"><SPAN class="tocnumber">2.6.4</SPAN><SPAN class="toctext">06 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-102"><A href="#07_th"><SPAN class="tocnumber">2.6.5</SPAN><SPAN class="toctext">07 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-103"><A href="#08_th"><SPAN class="tocnumber">2.6.6</SPAN><SPAN class="toctext">08 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-104"><A href="#9_th"><SPAN class="tocnumber">2.6.7</SPAN><SPAN class="toctext">9 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-105"><A href="#13_th_2"><SPAN class="tocnumber">2.6.8</SPAN><SPAN class="toctext">13 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-106"><A href="#14_th_2"><SPAN class="tocnumber">2.6.9</SPAN><SPAN class="toctext">14 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-107"><A href="#15_th_2"><SPAN class="tocnumber">2.6.10</SPAN><SPAN class="toctext">15 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-108"><A href="#18_th_2"><SPAN class="tocnumber">2.6.11</SPAN><SPAN class="toctext">18 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-109"><A href="#19_th_2"><SPAN class="tocnumber">2.6.12</SPAN><SPAN class="toctext">19 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-110"><A href="#20_th_2"><SPAN class="tocnumber">2.6.13</SPAN><SPAN class="toctext">20 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-111"><A href="#21_th_2"><SPAN class="tocnumber">2.6.14</SPAN><SPAN class="toctext">21 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-112"><A href="#22_th_3"><SPAN class="tocnumber">2.6.15</SPAN><SPAN class="toctext">22 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-113"><A href="#23_th_2"><SPAN class="tocnumber">2.6.16</SPAN><SPAN class="toctext">23 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-114"><A href="#24_th_3"><SPAN class="tocnumber">2.6.17</SPAN><SPAN class="toctext">24 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-115"><A href="#25_th_2"><SPAN class="tocnumber">2.6.18</SPAN><SPAN class="toctext">25 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-116"><A href="#26_th_2"><SPAN class="tocnumber">2.6.19</SPAN><SPAN class="toctext">26 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-117"><A href="#27_th"><SPAN class="tocnumber">2.6.20</SPAN><SPAN class="toctext">27 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-118"><A href="#28_th_2"><SPAN class="tocnumber">2.6.21</SPAN><SPAN class="toctext">28 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-119"><A href="#29_th_3"><SPAN class="tocnumber">2.6.22</SPAN><SPAN class="toctext">29 <SUP>th</SUP></SPAN></A></LI><LI class="toclevel-2 tocsection-120"><A href="#Oct."><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Oct.</SPAN></A></LI><LI class="toclevel-3 tocsection-121"><A href="#01_st_3"><SPAN class="tocnumber">2.7.1</SPAN><SPAN class="toctext">01 <SUP>st</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-122"><A href="#02_nd_2"><SPAN class="tocnumber">2.7.2</SPAN><SPAN class="toctext">02 <SUP>nd</SUP></SPAN></A></LI><LI class="toclevel-3 tocsection-123"><A href="#03_rd"><SPAN class="tocnumber">2.7.3</SPAN><SPAN class="toctext">03 <SUP>rd</SUP></SPAN></A></LI></DIV></DIV><DIV class="col-md-9" id="document-body"><H1><SPAN class="mw-headline" id="2015">2015</SPAN></H1><H2><SPAN class="mw-headline" id="Dec.">Dec.</SPAN></H2><H3><SPAN class="mw-headline" id="13th">13<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Geco_plasmid_construction">Geco plasmid construction</SPAN></H4><P>1. Polymerase Chain Reaction (PCR) 
</P><P>Use <B>PBX-084 TRE-mCherry-2A-Bla</B> plasmid as template to get <B>blasticidin(Bla) resistance gene</B>.
</P><P>Primers: Age-Bla-S &amp; Bla-E2A-AS
</P><P>(without purification), 20μl, 30 cycles
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Buffer </TD><TD> 2
</TD></TR><TR><TD> Template </TD><TD> 0.5
</TD></TR><TR><TD> Polymerase(Taq)</TD><TD> 0.2
</TD></TR><TR><TD> dNTP </TD><TD> 1
</TD></TR><TR><TD> Primer </TD><TD> 0.5, 0.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15.4
</TD></TR></TBODY></TABLE><P>
2. PCR 
</P><P>Use <B>PBX-084 TRE-mCherry-2A-Bla</B> plasmid as template to get <B>2A</B> sequence.
</P><P>Primers: E2A-S &amp; E2A-Hind III-AS
</P><P>(without purification), 20μl, 30 cycles
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Buffer </TD><TD> 2
</TD></TR><TR><TD> Template </TD><TD> 0.5
</TD></TR><TR><TD> Polymerase(Taq)</TD><TD> 0.2
</TD></TR><TR><TD> dNTP </TD><TD> 1
</TD></TR><TR><TD> Primer </TD><TD> 0.5, 0.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15.4
</TD></TR></TBODY></TABLE><P>
3. PCR 
</P><P><B>Step 1</B>: run 10 cycles without primer to let Bla and 2A link together.
</P><TABLE><TBODY><TR><TD> 98℃ </TD><TD> 10s </TD><TD> \ </TD></TR><TR><TD> 55℃ </TD><TD> 5s  </TD><TD> x10 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 30s </TD><TD> / </TD></TR></TBODY></TABLE><P><B>Step 2</B>：add primers to run 30 cycles.
</P><P>Primers: AgeI-Bla-S 0.3μM &amp; E2A-HindIII-AS 0.3μM
</P><TABLE><TBODY><TR><TD> 98℃ </TD><TD> 10s </TD><TD> \ </TD></TR><TR><TD> 55℃ </TD><TD> 5s  </TD><TD> x30 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 30s </TD><TD> / </TD></TR></TBODY></TABLE><P>4. PCR product cycle purification by using <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit</B></A>.
</P><H3><SPAN class="mw-headline" id="15th">15<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Geco_plasmid_construction_2">Geco plasmid construction</SPAN></H4><P>1. Digest the PCR purification product with AgeI and HindIII restriction endonuclease(RE).(37 ℃ 2 hours)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> PCR purification product </TD><TD> 8.4 (~1ug)
</TD></TR><TR><TD> CutSmart buffer </TD><TD> 5
</TD></TR><TR><TD> AgeI</TD><TD> 1.5
</TD></TR><TR><TD> HindIII </TD><TD> 1.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 33.6
</TD></TR></TBODY></TABLE><P>2. Digest the Geco plasmid with BglII restriction endonuclease. (50℃  3 hours)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Geco plasmid </TD><TD> 9.5 (~1ug)
</TD></TR><TR><TD> 3.1 buffer(X10) </TD><TD> 5
</TD></TR><TR><TD> BglII </TD><TD> 1.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 34
</TD></TR></TBODY></TABLE><P>3.Cycle pure the RE digestion product with the kit “xxxxxx”.
</P><P>
4. Digest the purification product with HindIII restriction endonuclease. (37 ℃  3 hours)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> the purification product </TD><TD> 5 (~640ng)
</TD></TR><TR><TD> CutSmart (X10) </TD><TD> 5
</TD></TR><TR><TD> HindIII </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 39
</TD></TR></TBODY></TABLE><P>5. Digest the vector plasmid with AgeI restriction endonuclease. (37 ℃  3 hours)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> The vector plasmid </TD><TD> 10 (~2.5 ug)
</TD></TR><TR><TD> CutSmart(X10) </TD><TD> 5
</TD></TR><TR><TD> AgeI </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 34
</TD></TR></TBODY></TABLE><P>6.Add 1.5 μl BclII restriction endonuclease. (50 ℃ 3 hours)
</P><P><B>Note:</B> BclI and BglII have the same sticky end.
</P><P>7. Gel running and Gel extraction to get the insert and vector fragments.<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Gel Extraction Kit - Spin Protocol</B></A></P><H3><SPAN class="mw-headline" id="16th">16<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Geco_plasmid_construction_3">Geco plasmid construction</SPAN></H4><P>1. Geco,Vector,Bla+2A Ligation with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase</B></A>(Vector:Insert=1:3).
</P><P>Vector	(5.8 kbp):	5.5ng/μl    
</P><P>Bla+2A	(0.45kbp): 	16.8ng/μl         
</P><P>Geco	(1.25kbp):  5.6ng/μl 
</P><TABLE class="table table-striped"><TBODY><TR><TH> Kit </TH><TH> Add (ul)
</TH></TR><TR><TD> 10X T4 DNA Ligase Buffer</TD><TD>2ul </TD><TD> 2.5
</TD></TR><TR><TD> Vector DNA(5.8kb) </TD><TD> 0.02pmol </TD><TD> 12.7
</TD></TR><TR><TD> Bla+2A(0.45kb) </TD><TD> 0.06pmol </TD><TD> 1
</TD></TR><TR><TD> Geco(1.25kb) </TD><TD> 0.06pmol </TD><TD> 8.2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 0
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1ul </TD><TD> 1
</TD></TR><TR><TD> Total </TD><TD> 20ul </TD><TD> 25
</TD></TR></TBODY></TABLE><P>Room temperature for 10 minutes.
</P><P>2. Mix&amp;Go hyper-efficient competent cells <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>transformation</B></A>.
</P><DL><DD><UL><LI> Quickly thaw: remove &quot;Mix&amp;Go&quot; hyper-efficient competent cells from -80℃ freezer  and water flow till 50% of the cells melt.</LI><LI> Add DNA: DNA volume is less than 1/10 volume of competent cells. (blow and suck for mixing )</LI><LI> Heat shock: heat shock at 42℃ water bath for 45s.</LI><LI> Spread the plates: remove LB(Amp+) plates from 4℃refrigerator,and spread all the bacterial on the plates. Overnight culture at 37℃ incubator (12~16hours).</LI></UL></DD></DL><P>Vector+Bla 2A+Geco 3 μl + competent cells 30 μl
</P><H3><SPAN class="mw-headline" id="17th">17<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Geco_plasmid_construction_4">Geco plasmid construction</SPAN></H4><P>1. Vector+Bla 2A+Geco transformation results:
</P><P>2. Pick single clones and overnight culture.
</P><P>Pick 5 single clones and use 6 ml Amp<SUP>+</SUP> LB medium with 37℃ and 220 rpm shaker to overnight culture each clones.
</P><H3><SPAN class="mw-headline" id="18th">18<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Geco_plasmid_construction_5">Geco plasmid construction</SPAN></H4><P>1. Extraction of plasmid with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol</B></A>.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Geco plasmid </TH><TH> Conc. (ng/ul) </TH><TH> A260/280
</TH></TR><TR><TD> 1 </TD><TD>94.2  </TD><TD> 1.92
</TD></TR><TR><TD> 2 </TD><TD> 79.7 </TD><TD> 1.96
</TD></TR><TR><TD> 3 </TD><TD> 110.6 </TD><TD> 1.90
</TD></TR><TR><TD> 4 </TD><TD> 150.3 </TD><TD> 1.87
</TD></TR><TR><TD> 5 </TD><TD> 149.7 </TD><TD> 1.88
</TD></TR></TBODY></TABLE><P>2. Send 5 plasmids to sequence.
</P><H3><SPAN class="mw-headline" id="29th">29<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction">Piezo plasmid construction</SPAN></H4><P>1. Piezo backbone(PBX-123 plasmid from Prof.Huang’s lab) transformation.<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Mix.26Go_method." title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Mix&amp;Go method.</B></A></P><P>Piezo backbone plasmids 2μl + competent cells 20μl
</P><H3><SPAN class="mw-headline" id="30th">30<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_2">Piezo plasmid construction</SPAN></H4><P>1. Piezo backbone transformation results:
</P><P>2. Pick single clones and overnight culture.
</P><P>Pick 3 single clones and use 6 ml Amp<SUP>+</SUP> LB medium with 37℃ and 220 rpm shaker to overnight culture each clones.
</P><H3><SPAN class="mw-headline" id="31th">31<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_3">Piezo plasmid construction</SPAN></H4><P>1. Extraction of plasmid with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol</B></A>.
</P><TABLE class="table table-striped"><TBODY><TR><TH> piezo </TH><TH> Conc. (ng/ul) </TH><TH> A260/280
</TH></TR><TR><TD> 1 </TD><TD> 143.4  </TD><TD> 1.87
</TD></TR><TR><TD> 2 </TD><TD> 283.1 </TD><TD> 1.87
</TD></TR><TR><TD> 3 </TD><TD> 186.8 </TD><TD> 1.87
</TD></TR></TBODY></TABLE><P>2. Piezo GPA、pTRE-3G PCR.
</P><P>Piezo: 310ng/μl GPA: 444bp pTRE-3G: 376bp
</P><TABLE class="table table-striped"><TBODY><TR><TH> Protocol </TH><TH> Final conc. </TH><TH> Add (ul)
</TH></TR><TR><TD> PrimeSTAR Max(2X)</TD><TD> 25ul </TD><TD> 1X </TD><TD>25
</TD></TR><TR><TD> Primer1 </TD><TD> 10-15pmol </TD><TD> 0.2-0.3uM </TD><TD> 1.3
</TD></TR><TR><TD> Primer2 </TD><TD> 10-15pmol </TD><TD> 0.2-0.3uM </TD><TD> 1.3
</TD></TR><TR><TD> Template </TD><TD>&lt;200ng </TD><TD>0.7
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 21.7
</TD></TR><TR><TD> Total </TD><TD> 50ul </TD><TD> 50
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 98℃ </TD><TD> 10s </TD><TD> \
</TD></TR><TR><TD> 55℃ </TD><TD> 15s </TD><TD> X 35 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 30s </TD><TD> /
</TD></TR></TBODY></TABLE><P>3.Gel Running
</P><P>4.PCR purification.
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Conc. (ng/ul) </TH><TH> A260/280
</TH></TR><TR><TD> GPA </TD><TD> 122.7  </TD><TD> 1.85
</TD></TR><TR><TD> pTRE-3G </TD><TD> 102.5 </TD><TD> 1.84
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="2016">2016</SPAN></H1><H2><SPAN class="mw-headline" id="Jan.">Jan.</SPAN></H2><H3><SPAN class="mw-headline" id="21th">21<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_4">Piezo plasmid construction</SPAN></H4><P>1. Piezo plasmid construction design:
</P><P>2. Piezo plasmid(original) RE digestion: 37℃ 2hours
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> piezo DNA (576ng/ul) </TD><TD> 2
</TD></TR><TR><TD> AscI </TD><TD> 1
</TD></TR><TR><TD> NotI</TD><TD>  1
</TD></TR><TR><TD> 10X CutSmart buffer </TD><TD> 5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 41
</TD></TR></TBODY></TABLE><P>3. Backbone PBX-123 plasmid RE digestion: 37℃ 2hours
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> pBX123 DNA (312ng/ul) </TD><TD> 3
</TD></TR><TR><TD> MfeI </TD><TD> 1
</TD></TR><TR><TD> PmeI </TD><TD>  1
</TD></TR><TR><TD> 10X CutSmart buffer </TD><TD> 5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 40
</TD></TR></TBODY></TABLE><P>4. Gel running and gel extraction. 
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Gel Extraction Kit - Spin Protocol</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Gel Extraction concentration (ng/ul)
</TH></TR><TR><TD> piezo (original) </TD><TD> 18.6
</TD></TR><TR><TD> pBX123 </TD><TD> 40.7
</TD></TR></TBODY></TABLE><P>5. Gibson assembly:
</P><TABLE class="table table-striped"><TBODY><TR><TH> Gibson system (20ul) </TH><TH> Gibson Control system(10ul)
</TH></TR><TR><TD> piezo </TD><TD> 4 </TD><TD> 0
</TD></TR><TR><TD> pBX-123 </TD><TD> 2 </TD><TD>1
</TD></TR><TR><TD> pTRE-3G</TD><TD>  0.2 </TD><TD> 0.1
</TD></TR><TR><TD> GpA</TD><TD>  0.2 </TD><TD> 0.1
</TD></TR><TR><TD> Gibson mix (X2) </TD><TD> 10 </TD><TD>5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 3.6 </TD><TD> 3.8
</TD></TR></TBODY></TABLE><P>Incubate the mix for 1 hour at 50°C. (<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Gibson_Assembly.C2.AE_Master_Mix_.E2.80.93_Assembly_.28E2611.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gibson Assembly® Master Mix – Assembly (E2611) Method</B></A>)
</P><P>6. Gibson product transformation:
</P><P>[[Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery | <B>Transformation &amp; Competent cell recovery</B>]
</P><P>5μl gibson product + 50μl competent cells
</P><P>Spread plates and overnight cultrue.
</P><H3><SPAN class="mw-headline" id="22th">22<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_5">Piezo plasmid construction</SPAN></H4><OL><LI> Pick single colonies:
<DL><DD> pick 1 colony from control group, pick 5 colonies from piezo group.</DD><DD> Add into 6ml Amp<SUP>+</SUP> LB, shake at 37℃ and 220 rpm.</DD></DL></LI><LI> Make Amp LB and agar.
<DL><DD> make two 500ml Amp<SUP>+</SUP> LB medium and one 500ml Amp<SUP>+</SUP> LB agar.</DD><DD> pour 29 plates.</DD></DL></LI><LI> Sterilization.
<DL><DD> We put bunch of tips and tubes for sterilization.</DD></DL></LI></OL><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_6">Piezo plasmid construction</SPAN></H4><OL><LI> Extract the plasmids from the single colonies (1 control group and 5 PIEZO group):</LI></OL><TABLE class="table table-striped"><TBODY><TR><TH> Conc. (ng/ul)
</TH></TR><TR><TD> Control group </TD><TD> 389.6
</TD></TR><TR><TD> PIEZO group 1 </TD><TD> 343.5
</TD></TR><TR><TD> PIEZO group 2 </TD><TD> 376.3
</TD></TR><TR><TD> PIEZO group 3 </TD><TD> 371.6
</TD></TR><TR><TD> PIEZO group 4 </TD><TD> 388.5
</TD></TR><TR><TD> PIEZO group 5 </TD><TD> 245.2
</TD></TR></TBODY></TABLE><OL><LI> Gel electrophoresis of the six plasmids:</LI></OL><P>Sample 2 and Sample 4 have shown positive results, which should be tested further with PCR.
</P><H4><SPAN class="mw-headline" id="Materials_sorting:">Materials sorting:</SPAN></H4><P>We sorted the material and put all of them into the lowermost layer of the -20℃ fridge. 
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Restriction enzyme and buffer ↑</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PCR related materials ↑</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Primer of our group ↑</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Plasmid construction of our group ↑</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Inner look of plasmid construction, left row is the final product. Right parts are materials from previous steps. ↑</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Overall look ↑</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="25th">25<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_7">Piezo plasmid construction</SPAN></H4><OL><LI> Sterilization:
<DL><DD> We put a bottle of ddH2O and bunch of tips and tubes for sterilization.</DD></DL></LI><LI>PCR test for PIEZO group 2 and PIEZO group 4: <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Premix_TaqTM_.28RR902A.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Premix TaqTM (RR902A)</B></A></LI></OL><P>For each system
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Premix Taq </TD><TD> 10
</TD></TR><TR><TD> Sample </TD><TD> 0.5
</TD></TR><TR><TD> GpA primer1</TD><TD> 1
</TD></TR><TR><TD> GpA primer2 </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7.5
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 5min </TD><TD> 95℃ </TD><TD> 30s  </TD><TD> \ </TD><TD> 55℃ </TD><TD> 20s  </TD><TD> x35 cycles
</TD><TD> 72℃ </TD><TD> 40s  </TD><TD> / </TD></TR></TBODY></TABLE><P>3. Gel running: 
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">The result showed that GpA had linked with backbone and piezo.</DIV></DIV></DIV></DIV><P>4. Pick up single colonies and colony PCR
</P><P>As the result of gel electrophoresis of piezo Gibson group 2&amp;4(yesterday,2016.1.24) missed bands of 4000, we picked up 20 single colonies to test by colony PCR. (use primers of GpA and PTRE).
</P><P>The system of colony PCR (total 20 μl): <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Premix_TaqTM_.28RR902A.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Premix TaqTM (RR902A)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 2X Taq mix </TD><TD> 10
</TD></TR><TR><TD> Template </TD><TD> 05
</TD></TR><TR><TD> Primer1</TD><TD> 1
</TD></TR><TR><TD> Primer2 </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 3
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 5min </TD></TR><TR><TD> 95℃ </TD><TD> 30s  </TD><TD> \ </TD></TR><TR><TD> 55℃ </TD><TD> 20s  </TD><TD> x35 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 40s  </TD><TD> / </TD></TR></TBODY></TABLE><P>Finally, we got 40 groups for PCR colonies (20 for GpA and 20 for PTRE).
</P><P>5. Gel electrophoresis of 40 groups after PCR colonies.
</P><P>6.TRPC5 &amp; TRPC6 &amp;PBX-090 <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>transformation</B></A>.
</P><P>We transformed these plasmids into competent cell and coated. After that, incubated at 37℃ for 16 hours.（5 plasmids onto 5 Amp plates）
</P><H3><SPAN class="mw-headline" id="26th">26<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_8">Piezo plasmid construction</SPAN></H4><P>1. Pick up single colonies
</P><P>After TRPC5, TRPC6 and PBX-090 transformation, we found that there are no colony growing on PBX-090 plate. Later we knew that PBX-090 is Kanamycin resistant. We pick 1 colony from other 4 plates. Add into 5ml Amp LB, shake at 37℃.
</P><P>2.PCR test <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Premix_TaqTM_.28RR902A.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Premix TaqTM (RR902A)</B></A></P><P>For yesterday PIEZO group 2 and PIEZO group 4(GPA verified):
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Premix Taq </TD><TD> 10
</TD></TR><TR><TD> Sample </TD><TD> 0.5
</TD></TR><TR><TD> GpA primer1</TD><TD> 1
</TD></TR><TR><TD> GpA primer2 </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7.5
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 5min </TD></TR><TR><TD> 95℃ </TD><TD> 30s  </TD><TD> \ </TD></TR><TR><TD> 55℃ </TD><TD> 20s  </TD><TD> x30 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 40s  </TD><TD> / </TD></TR></TBODY></TABLE><P>For yesterday PIEZO colony PCR promoter(pTRE) group 9,11,15(pTRE verified):
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Premix Taq </TD><TD> 10
</TD></TR><TR><TD> Sample </TD><TD> 0.5
</TD></TR><TR><TD> GpA primer1</TD><TD> 1
</TD></TR><TR><TD> GpA primer2 </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7.5
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 5min </TD></TR><TR><TD> 95℃ </TD><TD> 30s  </TD><TD> \ </TD></TR><TR><TD> 55℃ </TD><TD> 20s  </TD><TD> x30 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 40s  </TD><TD> / </TD></TR></TBODY></TABLE><P>3.Gel running of PCR product
</P><P>4.Streak plate of colony PCR9&amp;11&amp;15
</P><P>We streaked 3 Amp plates using tips.
</P><P>5.RE Digestion of PIEZO(original plasmid), PIEZO group 2,and PIEZO group 4
PIEZO Total 20μl
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 1ug PIEZO(original plasmid, conc. 576ng/ul) </TD><TD> 1.7
</TD></TR><TR><TD> BamHI </TD><TD> 0.2
</TD></TR><TR><TD> CutSmart buffer(X10)</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 16.1
</TD></TR></TBODY></TABLE><P>PIEZO group 2(1) Total 20μl
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 1ug PIEZO group2(original plasmid, conc. 376.3ng/ul) </TD><TD> 2.6
</TD></TR><TR><TD> PmeI </TD><TD> 0.2
</TD></TR><TR><TD> CutSmart buffer(X10)</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15.2
</TD></TR></TBODY></TABLE><P>PIEZO group 2(2) Total 20μl
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 1ug PIEZO group2(original plasmid, conc. 376.3ng/ul) </TD><TD> 2.6
</TD></TR><TR><TD> PmeI </TD><TD> 0.2
</TD></TR><TR><TD> BamHI </TD><TD> 0.2
</TD></TR><TR><TD> CutSmart buffer(X10)</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15
</TD></TR></TBODY></TABLE><P>PIEZO group 4(1) Total 20μl
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 1ug PIEZO group4(original plasmid, conc. 388.5ng/ul) </TD><TD> 2.6
</TD></TR><TR><TD> PmeI </TD><TD> 0.2
</TD></TR><TR><TD> CutSmart buffer(X10)</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15.2
</TD></TR></TBODY></TABLE><P>PIEZO group 4(2) Total 20μl
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 1ug PIEZO group2(original plasmid, conc. 388.5ng/ul) </TD><TD> 2.6
</TD></TR><TR><TD> PmeI </TD><TD> 0.2
</TD></TR><TR><TD> BamHI </TD><TD> 0.2
</TD></TR><TR><TD> CutSmart buffer(X10)</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 15
</TD></TR></TBODY></TABLE><P>Gel running results:
</P><P>Since the results out to be blurred, we decide to run gel again tomorrow.
</P><H3><SPAN class="mw-headline" id="27th">27<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_9">Piezo plasmid construction</SPAN></H4><P>1.Plasmid purification <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol - Spin Protocol</B></A></P><P>Plasmid purification of TRPC5 and TRPC6 from the single colonies we picked yesterday
</P><P>The concentration of the plasmids:
</P><TABLE class="table table-striped"><TBODY><TR><TH> Conc. (ng/ul)
</TH></TR><TR><TD> TRPC5 IRES-GFP </TD><TD> 461.9
</TD></TR><TR><TD> TRPC5 IRES-GFP </TD><TD> 425.1
</TD></TR><TR><TD> TRPC5 IRES-GFP </TD><TD> 542.7
</TD></TR><TR><TD> TRPC5 IRES-GFP</TD><TD> 866.8
</TD></TR></TBODY></TABLE><P>2.Gel electrophoresis of Restriction Enzyme Digestion product:</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><I>Something must have been wrong!</I></DIV></DIV></DIV></DIV><P>3.Pick single colonies
After we strake plate of colony PCR9&amp;11&amp;15, we pick 1 colony from each plate. Add into 5ml Amp LB, shake at 37℃.
</P><P>4. TRPC5 T5-GFP PCR <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Q5.C2.AE_High-Fidelity_2X_Master_Mix.28M0492S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Q5® High-Fidelity 2X Master Mix(M0492S)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Q5 premix </TD><TD> 10
</TD></TR><TR><TD> Primer1 </TD><TD> 1
</TD></TR><TR><TD> Primer2 </TD><TD> 1
</TD></TR><TR><TD> TRPC5 T5-GFP</TD><TD> 0.8
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7.2
</TD></TR><TR><TD> Total </TD><TD> 20
</TD></TR></TBODY></TABLE><P>We make 2 groups of 50μL PCR solution.
</P><P>5. PCR purification <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>Accidentally make the centrifuge in step 8 30 seconds rather than 1 minute
</P><P>Use 50μl ddH2O to elute DNA in the last process.
</P><P>DNA concentration:
</P><P>Group1: 21.8ng/μL
</P><P>Group2: 31.6ng/μL
</P><P>6. TRPC5 T5-GFP PCR(2nd) <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Q5.C2.AE_High-Fidelity_2X_Master_Mix.28M0492S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Q5® High-Fidelity 2X Master Mix(M0492S)</B></A></P><P>Because the DNA concentration in the last step was too low to conduct the following experiment, we did the TRPC5 T5-GFP PCR for the second time.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Q5 premix </TD><TD> 25
</TD></TR><TR><TD> Primer1 </TD><TD> 2.5
</TD></TR><TR><TD> Primer2 </TD><TD> 2.5
</TD></TR><TR><TD> TRPC5 T5-GFP</TD><TD> 3
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 17
</TD></TR><TR><TD> Total </TD><TD> 50
</TD></TR></TBODY></TABLE><P>We make 5 groups of 50μL PCR solution, and run 40 cycles.
</P><H3><SPAN class="mw-headline" id="28th">28<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_10">Piezo plasmid construction</SPAN></H4><P>1. Plasmid purification and maintenance breeding
① Plasmid purification of colony PCR9&amp;11&amp;15 from the single colonies we picked yesterday
</P><P>Use 50μL ddH2O to elute DNA in the last process.
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol - Spin Protocol</B></A></P><P>The concentration of the plasmids:
</P><P>Colony PCR9 463.6ng/μl
</P><P>Colony PCR11 556.1ng/μl
</P><P>Colony PCR15 474.4ng/μl
</P><P>② Maintenance breeding
</P><P>Mix 500μL bacterial liquid with 500μL (volume fraction:50%)glycerol, and then shore at -80℃.
</P><P>2. PCR purification <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>PCR purification of TPRC5 T5-GFP(2nd)
</P><P>Use 80 μl Tris-HCl to elute DNA in the last process.
</P><P>Finally, the concentration of DNA:
</P><P>Group 1 55.8ng/μl
</P><P>Group 2 68.7ng/μl
</P><P>Group 3 69.2ng/μl
</P><P>Group 4 71.6ng/μl
</P><P>3. Mycoplasma detection
</P><P>We did mycoplasma detection for the supernatant from the cell culture (step 4. Cell freezing) we collected yesterday.
</P><P>a. PCR the supernatant by the primers of mycoplasma DNA.
</P><P>The system of PCR (total 50μL): <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Q5.C2.AE_High-Fidelity_2X_Master_Mix.28M0492S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Q5® High-Fidelity 2X Master Mix(M0492S</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Q5 hot start </TD><TD> 25
</TD></TR><TR><TD> Primer1 </TD><TD> 2.5
</TD></TR><TR><TD> Primer2 </TD><TD> 2.5
</TD></TR><TR><TD> DNA</TD><TD> 3
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 17
</TD></TR></TBODY></TABLE><P>Program setting:
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 30s </TD></TR><TR><TD> 98℃ </TD><TD> 20s </TD><TD> \ </TD></TR><TR><TD> 50℃ </TD><TD> 10s </TD><TD> x35 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 100s </TD><TD> / </TD></TR><TR><TD> 72℃  </TD><TD> 120s </TD></TR><TR><TD> 4℃ </TD><TD> ∞ </TD></TR></TBODY></TABLE><P>b. Gel electrophoresis of PCR product.
</P>
Obviously, positive control turns out to be positive result, while negative control and the samples are negative results, so that our freezing cells are not polluted and we can store them into liquid nitrogen (big liquid nitrogen cell storage tank 3-4).
<P>4.Restriction Enzyme Digestion(A)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> DNA (~50ng/uL)(actually, we use group 1&amp;2 from step 2. today) </TD><TD> 80
</TD></TR><TR><TD> Age I-HF </TD><TD> 1
</TD></TR><TR><TD> 10X CutSmart buffer </TD><TD> 9
</TD></TR></TBODY></TABLE><P><B>Digest the ends of TRPC5 for ligation.</B></P><P>Since we have the PBX123 with sticky ends of Age I and Bcl I, we should digest TRPC5 with Age I enzyme and Bcl I.
</P><P>Firstly, we digest TRPC5 with Age I (activity:100%).The system (total 90μL):
</P><P>Incubate at 37℃ for two hours.
</P><P>Then, we digest TRPC5 with Bcl I (activity:75%).
</P><P>We add Bcl I enzyme (1μL) into the final system of the first digestion.
Incubate at 50℃ for six hours.
</P><P><B>Digest the product of plasmid purification (step 1. today).</B></P><P>We digest the product of plasmid purification of pick up single colonies of streak plate of colony PCR group9&amp;11&amp;15 to test its linking.
</P><P>The system (total 20μL):
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> DNA(~50ng/ul) </TD><TD> 1
</TD></TR><TR><TD> Bam HI-HF </TD><TD> 10.2
</TD></TR><TR><TD> 10X CutSmart buffer </TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 16.8
</TD></TR></TBODY></TABLE><P>Incubate at 37℃ for two hours.
</P><P>5. Gel electrophoresis of the digestion of plasmid purification product of colonies PCR 9&amp;11&amp;15
</P><P>As we digest the plasmid with only one enzyme, we only get linear 15k bp DNA.
Obviously, group9&amp;15 are positive.
</P><P>6. Restriction Enzyme Digestion(B) and gel electrophoresis
</P><P>We decide to digest the product of plasmid purification of colony PCR group 9&amp;15 with Bam-HI and Bgl I to ensure they are exactly correct.
</P><P>The system (total 20μL):
</P><P>Bam HI-HF 0.2μL(activity:100%)
</P><P>Bgl I 0.2μL(activity:100%)
</P><P>DNA(~500ng/μL) 1.25μL
</P><P>10×NEBuffer 2μL(3.1 buffer)
</P><P>ddH2O 16.35μL
</P><P>Incubate at 37℃ for two hours.
</P><P>Gel electrophoresis of the product of digestion.
</P><P>7. Restriction Enzyme Digestion purification
<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>Use 30μL ddH<SUB>2</SUB>O to elute DNA in the last process.
</P><P>TRPC5 T5-GFP 1 72.6ng/Μl
</P><P>TRPC5 T5-GFP 2 71.3ng/μL
</P><P>8. Restriction Enzyme Digestion <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>We digest PBX-123 with Age I to get the sticky end which can connected with TPRC5.
</P><P>The system (total 50μL):
</P><P>Age I-HF 2μL
</P><P>DNA(~300ng/μL) 20μL(actually, we use PBX group 3.)
</P><P>10×NEBuffer 5μL(CutSmart buffer)
</P><P>ddH2O 23μL
</P><P>Incubate at 37℃ for two hours.
</P><P>9. Ligation (TRPC5 and PBX-123)
</P><P>The system (total 20μL):
</P><P>10×T4 DNA buffer 2μL
</P><P>Vector(PBX123) 10μL(43.26ng)
</P><P>TRPC5 1μL(55.26ng)
</P><P>ddH<SUB>2</SUB>O 6μL
</P><P>T4 DNA ligase 1μL
</P><P>10. <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A>:
</P><P>We transform the plasmid which we get in the step 9. Ligation into 25μL competent cell.
</P><P>We also help Dr. Huang (our mentor) transform 3 plasmids into competent cell (each 25μL).
</P><P>Then, we coat and incubate at 37℃for 16 hours.
</P><H3><SPAN class="mw-headline" id="29_th">29 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_11">Piezo plasmid construction</SPAN></H4><OL><LI> Restriction Enzyme Digestion(PBX-123, Age I&amp; Bcl I)</LI></OL><P>We digested PBX-123 with Age I to get the sticky end which can connected with TPRC5.
</P><P>Then we use Bcl1 to digest.
</P><P>The system (total 50μL):
</P><P>Bcl I 2μL
</P><P>DNA(~300ng/μL) 20μL (actually, we use PBX group 3.)
</P><P>10×NEBuffer 5μL(CutSmart buffer)
</P><P>ddH2O 23μL
</P><P>Incubate at 50℃ for two hours.
</P><P>The result is strange. The brightest band is larger than 10000.
</P><P>Still, we extract the plasmid from the brightest band. Concentration:61.8ng/μl
</P><P>Gel running of restriction enzyme digestion (PIEZO Gibson Ligation from colony PCR group 9&amp;15, BamHI Bgl II)
</P><P>The undigested ones showed more bands while digested one only showed one band which is really strange.
</P><P>We ran the gel for two times and still got the same results.
</P><P>3. Colony PCR (5 PIEZO Gibson colony &amp;2 PIEZO Gibson control colony&amp;Negative control ddH2O, PCR for GpA&amp; PTRE)
</P><P>Total 20μl
</P><P>2xTaq 10μl
</P><P>Template 5μl
</P><P>Primer 1 1μl
</P><P>Primer 2 1μl
</P><P>DdH2O 3μl
</P><P>Primers: GpA primers/PTRE primers
</P><P>From the results, GpA can still be seen in ddH2O. The result is so strange. We don’t know how to explain.
</P><P>4.Colony PCR (12 TRPC5 Ligation colonies&amp;dd H2O)
</P><P>Total 20μl
</P><P>2xTaq 10μl
</P><P>Template 5μl
</P><P>TRPC5 Primer 1 1μl
</P><P>TRPC5 Primer 2 1μl
</P><P>ddH2O 3μl
</P><P>The PCR results showed no correct one. We can’t see TRPC5 band.
</P><P>The rightmost one: Digested PBX123 backbone for TRPC5 is the gel extraction from step 1(Conc. 61.8ng/μl).The band should be around 8000bp. While the result is larger than 10000bp. This indicates that the extracted plasmid is not right.
&lt;/blockquote&gt;
5. Restriction enzyme digest(PBX123 backbone for PIEZO, PIEZO, both MfeI&amp;BamHI&amp;PmeI)
</P><UL><LI>PBX123 backbone for PIEZO</LI></UL><P>Total 50μl
</P><P>PBX123(~300ng/μl) 35 μl
</P><P>10x Buffer 5 μl
</P><P>DdH2O 7 μl
</P><P>MfeI 1μl
</P><P>BamHI 1μl
</P><P>PmeI 1μl
</P><UL><LI>PIEZO</LI></UL><P>Total 50μl
</P><P>PBX123(~570ng/μl) 20μl
</P><P>10x Buffer 5 μl
</P><P>DdH2O 22 μl
</P><P>MfeI 1μl
</P><P>BamHI 1μl
</P><P>PmeI 1μl
</P><P>6. Pick up single colonies
</P><P>Since the results from the Colony PCR of TRPC5 ligation was not very promising, we pick up 6 colony from the plate. Add into 5ml Amp LB, shake at 37℃.
</P><P>The signal colonies are picked for the PCR process to make sure that Colony PCR it self have nothing to do with the failure.
</P><H3><SPAN class="mw-headline" id="30th_2">30<SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_12">Piezo plasmid construction</SPAN></H4><P>1. Enzyme digestion of PBX123(Age I and Bcl I for TRPC5)
</P><P>PBX123(~300ng/μL) 30μL
</P><P>Age I 1μL
</P><P>CutSmart 5μL
</P><P>ddH<SUB>2</SUB>O 14μL
</P><P>Incubate in 37℃ for a whole night.
</P><P>Add in Bcl I at 11:00, and move to 50℃.
</P><P>Bcl I 1μL
</P><P>Incubate in 50℃ for a 4 hours.
</P><P>2. Gel electrophoresis of PBX123(Mfe I, BamH I and Pme I for piezo), PBX123(Age I and Bcl I for TRPC5) and Piezo
</P><P>The PBX123(MfeI BamHI PmeI for piezo) and Piezo(digested) on the other gel was cut off for gel extraction
</P><P>The concentration of the extraction:
</P><P>PBX-123 40.8ng/μL
</P><P>Piezo 28.4ng/μL
</P><P>3. Plasmid extraction <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol - Spin Protocol</B></A></P><P>Plasmid extraction from the single colonies we picked up yesterday.
</P><P>The concentration of the plasmids:
</P><P>TRPC5 ligation 1 378.2ng/μL
</P><P>TRPC5 ligation 2 347.8ng/μL
</P><P>TRPC5 ligation 3 343.3ng/μL
</P><P>TRPC5 ligation 4 441.5ng/μL
</P><P>TRPC5 ligation 5 438.0ng/μL
</P><P>TRPC5 ligation 6 431.0ng/μL
</P><P>4. TRPC5 PCR</P><P>We add in the primer of TRPC5 and try to make TRPC5 PCR from the ligation plasmid we extracted from the single colonies.
</P><P>We also make two control groups, positive control group from the original TRPC5 T5-GFP plasmid we got, and negative control group from ddH<SUB>2</SUB>O.
</P><P>rTaq premix 5μL
</P><P>Templet 1μL
</P><P>Primer1 0.5μL
</P><P>Primer2 0.5μL
</P><P>ddH<SUB>2</SUB>O 3μL
</P><P>5. Gel electrophoresis of the PCR products and the gel extraction products.
</P><P>Nothing, even with the positive control group from the original TRPC5 T5-GFP plasmid we got, have shown the right result (some brightness around 3000bp).
</P><P>6. Gibson assembly of Piezo and PBX123
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Gibson_Assembly.C2.AE_Master_Mix_.E2.80.93_Assembly_.28E2611.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gibson Assembly® Master Mix – Assembly (E2611)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Gibson group (10ul) </TH><TH> Control group1 (10ul) </TH><TH> Control group2 (10ul) </TH><TH> Control group3 (10ul)
</TH></TR><TR><TD>Piezo (28.4ng/ul)</TD><TD> 2.5ul
</TD></TR><TR><TD> pBX-123 (40.8ng/ul) </TD><TD> 1.5ul </TD><TD> 1.5ul </TD><TD> 1.5ul </TD><TD>  1.5ul
</TD></TR><TR><TD> pTRE-3G (102.5ng/ul) </TD><TD> 0.13ul </TD><TD> 0.13ul </TD></TR><TR><TD> GpA (122.7ng/ul) </TD><TD> 0.11ul </TD><TD> 0.11ul </TD></TR><TR><TD> Gibson mix </TD><TD> 5ul </TD><TD> 5ul </TD><TD> 2.5ul </TD><TD> 5ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD>  0.76ul</TD><TD> 3.26ul </TD><TD> 1ul </TD><TD> 1ul
</TD></TR></TBODY></TABLE><P>7. Enzyme digestion of PBX123(Age I and Bcl I for TRPC5)(2nd)
</P><P>PBX123(~300ng/μL) 30μL
</P><P>Bcl I 1μL
</P><P>3.1 5μL
</P><P>ddH<SUB>2</SUB>O 14μL
</P><P>Incubate in 50℃ for a 4 hours.
</P><P>Conduct DNA extraction with Cycle-Pure Kit, use 45μL ddH<SUB>2</SUB>O to elute the DNA.
</P><P>The DNA concentration was 92.5ng/μL.
</P><P>DNA extraction 45μL
</P><P>Age I 1μL
</P><P>CutSmart 5μL
</P><P>Incubate in 37℃ overnight.
</P><P>8. Transformation
</P><P>We transform the plasmids which we get in Gibson assembly into 33μL competent cell(1 gibson system and 4 control systems).
</P><P>We also transform PBX090 and PBX123 to get more plasmid for further use.
</P><H3><SPAN class="mw-headline" id="31th_2">31<SUB>th</SUB></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_13">Piezo plasmid construction</SPAN></H4><OL><LI> Gel electrophoresis of digested PBX123(Age I and Bcl I for TRPC5)(2nd)</LI><LI> Colony PCR</LI></OL><P>Before colony PCR, we remove the plate of PBX123 from incubator to cold room.
</P><P>We pick up 20 single colonies from the plate of Gibson group, 2 colonies from both control group 1 and 2. There are no colonies on plates of control group 3 and PBX090.
</P><P>The system of colony PCR(use primers of poly A and pTRE):
</P><P>Primer1 0.5μL
</P><P>Primer2 0.5μL
</P><P>Bacterial liquid 5μL
</P><P>rTaq premix 6μL
</P><P>We use original PBX123 to substitute the Bacterial liquid in positive control group 1.
</P><P>We use poly A or pTRE that we get from the original PBX123 by PCR process to substitute the Bacterial liquid in positive control group 2.
</P><P>We use ddH<SUB>2</SUB>O to substitute the Bacterial liquid in negative control group.
</P><TABLE><TBODY><TR><TD> 95℃ </TD><TD> 30s </TD></TR><TR><TD> 98℃ </TD><TD> 10s </TD><TD> \ </TD></TR><TR><TD> 50℃ </TD><TD> 10s </TD><TD> x30 cycles
</TD></TR><TR><TD> 72℃ </TD><TD> 100s</TD><TD> / </TD></TR><TR><TD> 72℃ </TD><TD> 120s </TD></TR><TR><TD> 16℃ </TD><TD> ∞ </TD></TR></TBODY></TABLE><P>3. Gel electrophoresis of colony PCR (step 2.)
</P><P>Obviously, every Gibson group are negative.
</P><P>4. Enzyme digestion of PBX133(Bcl I and Age I, Nhe I, Xba I )
</P><P>We suspect the availability of our Bcl I enzyme, so we digest this new backbone with another tube of Bcl I, and set a group digested by our enzyme to test its availability.
</P><P>The system of digestion (by Bcl I of HW) (total 100μL):
</P><P>Template(~200ng/μL) 50μL
</P><P>Bcl I 5μL
</P><P>NEBuffer(3.1) 10μL
</P><P>ddH<SUB>2</SUB>O 35μL
</P><P>Incubate in 50℃ for 1 hour.
</P><P>The system of digestion (by Bcl I of IGEM)(total 10μL):
</P><P>Template(~200ng/μL) 2.5μL
</P><P>Bcl I(IGEM) 0.25μL
</P><P>NEBuffer(3.1) 1μL
</P><P>ddH<SUB>2</SUB>O 6.25μL
</P><P>Incubate in 50℃ for 3.5 hours.
</P><P>Conduct DNA extraction with <I>Cycle-Pure Kit</I>, use 50μL ddH<SUB>2</SUB>O to elute the DNA.
</P><P>The DNA concentration was ng/μL.
</P><P>The system of digestion (by Age I, Nhe I, Xba I )(total 50μL):
</P><P>DNA extraction 42μL
</P><P>Age I 1μL
</P><P>Nhe I 1μL
</P><P>Xba I 1μL
</P><P>NEBuffer(CutSmart) 5μL
</P><P>Incubate in 37℃ overnight.
</P><P>5. Gel electrophoresis of PBX133 digested by Bcl I(after incubating for 1 hour) (step 4.)
</P><P>From the result we can ensure our enzyme is available.
</P><H2><SPAN class="mw-headline" id="Feb.">Feb.</SPAN></H2><H3><SPAN class="mw-headline" id="01_st">01 <SUP>st</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Piezo_plasmid_construction_14">Piezo plasmid construction</SPAN></H4><P>1. Gel running of 4 enzymes digestion
</P><P>Digestion sites: Bcl I,Age I,Nhe I,Xba I
</P><P>2. Gel Extraction <B>E.Z.N.A.® Gel Extraction Kit - Spin Protocol</B></P><P>We use the rest digested product to do gel extraction of cut PBX133(After 4 enzymes digested) Concentration:39.7 ng/μl
</P><P>3. <B>Transformation</B> of enzyme digested products
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">2.5μl PBX133 digested by 4 enzymes, 2.5μl PBX133 digested by BclI(iGEM), 1μl PBX133 original, 2.5μl PBX133 digested by BclI(HW) with 25μl competent cells each.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="14_th">14 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction">TRPC5 plasmid construction</SPAN></H4><P>1. Enzyme digestion of TRPC5(T5-GFP and IRES-GPF)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> TRPC5(~400ng/ul) </TD><TD> 0.5
</TD></TR><TR><TD> Not I </TD><TD> 0.2
</TD></TR><TR><TD> 10X CutSmart buffer </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 8.3
</TD></TR><TR><TD> Total </TD><TD> 10
</TD></TR></TBODY></TABLE><P>Incubate in 37℃ for 3 hours.
</P><P>2. TRPC5 PCR(T5-GFP and IRES-GPF)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Template </TD><TD> 1
</TD></TR><TR><TD> Primer1 </TD><TD> 2.5
</TD></TR><TR><TD> Primer2 </TD><TD> 2.5
</TD></TR><TR><TD> Q5 hotstart 2X Master Mix </TD><TD> 25
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 19
</TD></TR></TBODY></TABLE><P>40 cycles
</P><P>3. Make petro dishes.
</P><P>We make 10 LB K+ petro dishes,75 Amp+ petro dishes and 14 LB petro dishes while waiting.
</P><P>4. Gel electrophoresis of TRPC5
</P><P>The expected brightness (~3k bp) appeared with TRPC5 T5-GFP PCR.
</P><P>But TRPC5 PCR purification group 4, which we did with the same process as TRPC5 T5-GFP PCR. Did not have the same brightness. So we decided to do another Gel electrophoresis process tomorrow.
</P><P>5. PCR purification of TRPC5 PCR products (T5-GFP and IRES-GPF)
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>The concentration of the products (diluted with 40μL ddH<SUB>2</SUB>O):
</P><P>TRPC5 T5-GFP 105.2ng/μL
</P><P>TRPC5 IRES-GPF 38.8ng/μL
</P><P>6. Enzyme digestion of TRPC5 T5-GFP PCR purification product
</P><P>Since we have the expected brightness (~3k bp) appeared with TRPC5 T5-GFP PCR, we carry on with the experiment. And started the enzyme digestion. Because Bcl I only have 75% activity in CutSmart buffer. So we decided to leave it overnight.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> TRPC5 T5 GFP(~102ng/ul) </TD><TD> 30
</TD></TR><TR><TD> Bcl I </TD><TD> 2
</TD></TR><TR><TD> CutSmart </TD><TD> 5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 13
</TD></TR><TR><TD> Total </TD><TD> 50
</TD></TR></TBODY></TABLE><P>Incubate in 50℃ for overnight.
</P><H3><SPAN class="mw-headline" id="15_th">15 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_2">TRPC5 plasmid construction</SPAN></H4><P>1. Enzyme digestion of TRPC5(T5-GFP-Age 1)
</P><P>Add 2 μL Age 1 into the system of last-night enzyme digestion.
</P><P>Incubate in 37℃ for 2 hours.
</P><P>2. Enzyme digested product purification of TRPC5
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>The concentration of the products (diluted with 30μL ddH<SUB>2</SUB>O):
</P><P>TRPC5 T5-GFP 58.8 ng/μl
</P><P>3. Gel electrophoresis of used TRPC5
</P><P>As we can see, the expected brightness (~3k bp) also appeared. It means that our TRPC5 PCR purification product is no problem, while the other four T5 products which were did before all have problem. So the four can’t be used, we will go deep by using the new TRPC5 PCR purification product.
</P><P>4. Ligation of TRPC5 and PBX133 backbone <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> 10* T4 DNA Ligase Buffer </TD><TD> 2
</TD></TR><TR><TD> Vector DNA </TD><TD> 3
</TD></TR><TR><TD> Insert DNA </TD><TD> 2
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 12
</TD></TR><TR><TD> Total </TD><TD> 20
</TD></TR></TBODY></TABLE><P>Gently mix and let it in room temperature for 20 minutes.
</P><P>Heat inactive at 65℃ for 10 minutes and put it in -20℃ for 4 hours.
</P><P>5. <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A> of TRPC5 plasmid
</P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cells/ul </TH><TH> DNA/ul
</TH></TR><TR><TD> TRPC5-pBX133 Ligation 1 </TD><TD> 33 </TD><TD> 5
</TD></TR><TR><TD> TRPC5-pBX133 Ligation 2 </TD><TD> 33 </TD><TD> 5
</TD></TR><TR><TD> pBX133 backbone </TD><TD> 33 </TD><TD> 5
</TD></TR></TBODY></TABLE><P>Results :
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 2</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 1</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX133 backbone</DIV></DIV></DIV></DIV><P>The results are not so good. There are many kinds of passible reasons which we will go to confirm one by one tomorrow.
</P><H3><SPAN class="mw-headline" id="16">16</SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_.3D">TRPC5 plasmid construction =</SPAN></H4><OL><LI> Learn to design primer from Prof. Huang.</LI><LI> Make new Amp+ agar plate.
<DL><DD> Spread 12μl 50mg/ml Amp+ (sodium salt) to the plate made in Feb.24</DD><DD> Put the plate to 25℃ for 3 hours</DD></DL></LI><LI> Examination of experiment and propagation of 090</LI></OL><P><B>Amp+ New plate(2016.2.16):</B></P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cells/ul </TH><TH> DNA/ul
</TH></TR><TR><TD> TRPC5-pBX133 Ligation 1 </TD><TD> 25 </TD><TD> 5
</TD></TR><TR><TD> pBX133 backbone 1 </TD><TD> 25 </TD><TD> 5
</TD></TR><TR><TD> TRPC5-pBX133 Ligation 2 </TD><TD> 25 </TD><TD> 1
</TD></TR><TR><TD> pBX133 backbone 2 </TD><TD> 25 </TD><TD> 1
</TD></TR><TR><TD> Control </TD><TD> 25 </TD><TD> 0
</TD></TR></TBODY></TABLE><P><B>Amp+ Old plate(2016.2.14):</B></P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cells/ul </TH><TH> DNA/ul
</TH></TR><TR><TD> pBX133 backbone 1 </TD><TD> 25 </TD><TD> 5
</TD></TR><TR><TD> TRPC5-pBX133 Ligation 2 </TD><TD> 25 </TD><TD> 1
</TD></TR><TR><TD> pBX133 backbone 2 </TD><TD> 25 </TD><TD> 1
</TD></TR><TR><TD> Control </TD><TD> 25 </TD><TD> 0
</TD></TR></TBODY></TABLE><P><B>Amp+ Former plate(2015):</B></P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cells/ul </TH><TH> DNA/ul
</TH></TR><TR><TD> TRPC5-pBX133 Ligation 1 </TD><TD> 25 </TD><TD> 5
</TD></TR></TBODY></TABLE><P><B>K+ plate(2016.2.14):</B></P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cells/ul </TH><TH> DNA/ul
</TH></TR><TR><TD> 090 </TD><TD> 25 </TD><TD> 5
</TD></TR><TR><TD> Control </TD><TD> 25 </TD><TD> 0
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><B>Amp+ New plate(2016.2.16):</B></P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 1</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 2</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX133 backbone 1</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX133 backbone 2</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Control</DIV></DIV></DIV></DIV><B>Amp+ Old plate(2016.2.14):</B><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 1</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX133 backbone 1</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX133 backbone 2</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Control</DIV></DIV></DIV></DIV><P><B>Amp+ Former plate(2015):</B></P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">TRPC5-PBX133 Ligation 1</DIV></DIV></DIV></DIV><P><B>K+ plate(2016.2.14):</B></P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Control</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">090</DIV></DIV></DIV></DIV><P>Discussion:
</P><P>Compare new and old plate, it comes out that some is wrong with the plate for its antibiotic function.
</P><P>We thinks there may be two reason for this:
</P><P>1.the antibiotics is inactived.
</P><P>2.there is some distribution problem in the production of plate.
</P><P>3.there is some concentration issue with the antibiotic.
</P><P>There is also some problem with the ligation of TPRC6 and PBX133 backbone.
</P><H3><SPAN class="mw-headline" id="17_th">17 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_3">TRPC5 plasmid construction</SPAN></H4><OL><LI> To check whether Ampicillin and Kanamycin are effective
</LI><LI> Add 1 μl Ampicillin solution to each LB agar plate and spread.Add 1 μl Kanamycin solution to each LB agar plate and spread.</LI><LI> 2 Let them stay for 3 hours.</LI><LI> 3 Add 25 μl competent cell to each plate. Incubate for 16 hours in 37 ℃</LI><DL><DD> Result：There is no any colony in each plate. So Ampicillin and Kanamycin are effective.</DD></DL><LI> TRPC5 primer design</LI></OL><P>3. Make Amp<SUP>+</SUP> and K<SUP>+</SUP> LB agar plates.
</P><P>We made 2*500ml LB agar. Add 500μl 1000x Amp<SUP>+</SUP> in one and add 500 μl 1000x K<SUP>+</SUP> in the other. We poured 15 K<SUP>+</SUP> plates and 13 Amp<SUP>+</SUP> plates.
</P><H3><SPAN class="mw-headline" id="18_th">18 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_4">TRPC5 plasmid construction</SPAN></H4><OL><LI> Modify the previous designed primers,based on following defects
</LI><LI> Protection bases are too short,should be 4~6 bases.</LI><LI> The temperature of matching pieces is too low,should be 55~65℃,and the Tm of primer should be 75~85℃.</LI><LI> The number of bases between start codon and stop codon of final ligation product should be the fold of 3.</LI></OL><DL><DD> Modified primers:</DD></DL><P><B>J23100-sfgfp-pBX123-piezo-primer</B></P><P>primer1:5'-AGAC ACCGGT TCTAGA CTAGAGTCGCGGCCGCTTTACTTGTA- 3'
</P><P>AgeI XbaI Tm 62.6
</P><P>primer2:5'-TACG GGATCC TTA T GGCGCGCC TTGATATCGAGCTCTTGACGGCTAGC- 3' 
</P><P>BamHI AscI Tm 60.9
<B>J23100-sfgfp-pBX123-TRPC5-primer</B></P><P>primer1: 5'-AGAC ACCGGT GAATTC CTAGAGTCGCGGCCGCTTTACTTGTA- 3'
</P><P>AgeI EcoRI Tm 62.6
</P><P>primer2: 5'-TACG GGATCC TTA T GCGGCCGC TTGATATCGAGCTCTTGACGGCTAGC- 3' 
</P><P>BamHI NotI Tm 60.9
</P><P>2. Amplify PBX-123 backbone
</P><P>Backbone 1μL+competent cell 33μL (3 tubes)
</P><P>Results:All 3 culture dishes don’t grow colony.
</P><P>Discussion:Maybe the concentration of PBX-123 backbone is too low,so we increase the concentration.(2016.02.19)
</P><P>3. Enzyme digestion of PBX-123 backbone(previous extracted PBX-123)
2 tubes,each 3μL
</P><TABLE class="table table-striped"><TBODY><TR><TD> NotI/ul </TD><TD> 1
</TD></TR><TR><TD> CutSmart/ul </TD><TD> 5
</TD></TR><TR><TD> pBX123 /ul </TD><TD> 3
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 41
</TD></TR><TR><TD> Total/ul </TD><TD> 50
</TD></TR></TBODY></TABLE><P>Purification of PBX-123(30μL system,2 tubes)
</P><P>Results: 
</P><P>①13.7ng/μL A260/280: 1.6
</P><P>②20.6ng/μL A260/280: 1.4
</P><P>4. Add Klenow Fragment to create blunt ends(2 tubes)
</P><P>Klenow Fragment: 5units/μL
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#DNA_Polymerase_I.2C_Large_.28Klenow.29_Fragment.28M0210S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>DNA Polymerase I, Large (Klenow) Fragment(M0210S)</B></A></P><P>Protocol(combine the NEB and Chinese protocol):
</P><P>①
</P><TABLE class="table table-striped"><TBODY><TR><TH>  Total</TH><TH> 40 </TH><TH> Final
</TH></TR><TR><TD> 10X NEBuffer2 </TD><TD> 4 </TD></TR><TR><TD> dNTP(2.5mM) </TD><TD> 0.6 </TD><TD> 33uM
</TD></TR><TR><TD> pBX123 </TD><TD> 27.5 </TD><TD> 20~30ul(0.2~8ug)
</TD></TR><TR><TD> Klenow Fragment </TD><TD> 0.1 </TD><TD> 0.4~1.6ul (2~8unit) 1 unit per ugDNA
</TD></TR><TR><TD> Control </TD><TD> 7.8 </TD></TR></TBODY></TABLE><P>②Incubate 15min at 25℃
</P><P>③Incubate 30min at 75℃
</P><P>5. Gel electrophoresis of 2 tubes blunt-ended PBX-123
</P><P>Marker: DL10000
</P><P>Results: Both 2 bands are near 10000bp and at the same site,which means the two bands are both blunt ends or sticky ends,we need to confirm after extracting the plasmid.
</P><P>(We forgot to take a picture of the gel)
</P><H3><SPAN class="mw-headline" id="19_th">19 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_5">TRPC5 plasmid construction</SPAN></H4><P>1. Gel cutting and extraction of PBX-123 Klenow Fragment products
</P><P>(30μL system)
</P><P>Concentration:7.6ng/μL, A260/280: 1.55
</P><P>2. Ligation of PBX-123 blunt end <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> Total </TD><TD> 30
</TD></TR><TR><TD> 10X Buffer </TD><TD> 3
</TD></TR><TR><TD> DNA </TD><TD> 25
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 2
</TD></TR></TBODY></TABLE><P>Amplify PBX-123 backbone,transform,select single colony and shake colony(4 tubes)
</P><P><B>Results:</B></P><TABLE class="table table-striped"><TBODY><TR><TD><B>PBX-123/ μl</B></TD><TD><B>Competent cell/ μl</B></TD><TD><B>Colonies</B></TD></TR><TR><TD> 0
</TD><TD> 25
</TD><TD> 0
</TD></TR><TR><TD> 5
</TD><TD> 25
</TD><TD> 10+
</TD></TR><TR><TD> 15
</TD><TD> 25
</TD><TD> 2
</TD></TR><TR><TD> 25
</TD><TD> 25
</TD><TD> 1
</TD></TR></TBODY></TABLE><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Fig.1 PBX-123 0 μl</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Fig.2 PBX-123 5 μl</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Fig.3 PBX-123 15 μl</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Fig.4 PBX-123 25 μl</DIV></DIV></DIV></DIV><P>Discussion:
</P><P>The number of colonies decreases as the concentration of PBX-123
</P><P>backbone increases,which is strange,so we need to do a gel running after
</P><P>extracting the plasmid to confirm whether the PBX-123 has problem.
</P><P>4. Ligation product transform,select single colony and shake colony (1 tubes)
</P><P>Results: only one culture dish grows 1 colony
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX-123 Not I-cut ligation 3</DIV></DIV></DIV></DIV><P>---
</P><H3><SPAN class="mw-headline" id="20_th">20 <SUP>th</SUP></SPAN></H3><P>TRPC5 plasmid construction
</P><P>1. PBX123 and PBX123(NotI-cut ligation) plasmid extraction
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#AxyPrep_Plasmid_Miniprep_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>AxyPrep Plasmid Miniprep Spin Protocol</B></A></P><P>Final volume: 70 ul each tube
</P><TABLE class="table table-striped"><TBODY><TR><TD> PBX-123 NotI-cut </TD><TD> 249.3ng/ul
</TD></TR><TR><TD> PBX-123 plasmid1 </TD><TD> 299.1ng/ul
</TD></TR><TR><TD> PBX-123 plasmid2 </TD><TD> 213.6ng/ul
</TD></TR><TR><TD> PBX-123 plasmid3 </TD><TD> 252.9ng/ul
</TD></TR><TR><TD> PBX-123 plasmid4 </TD><TD> 360.3ng/ul
</TD></TR></TBODY></TABLE><P>2. Breed preservation :
</P><P>500ul bacterial : 500ul glycerol each tube
</P><TABLE class="table table-striped"><TBODY><TR><TD> PBX-123 NotI-cut </TD><TD> 1 ml </TD><TD> -8℃ 107
</TD></TR><TR><TD> PBX-123 plasmid1 </TD><TD> 1 ml </TD><TD> -8℃ 107
</TD></TR><TR><TD> PBX-123 plasmid2 </TD><TD> 1 ml </TD><TD> -8℃ 107
</TD></TR><TR><TD> PBX-123 plasmid3 </TD><TD> 1 ml </TD><TD> -8℃ 107
</TD></TR><TR><TD> PBX-123 plasmid4 </TD><TD> 1 ml </TD><TD> -8℃ 107
</TD></TR></TBODY></TABLE><P>3. NotI digest and transform (2 tubes)
</P><TABLE class="table table-striped"><TBODY><TR><TD>NotI/ul </TD><TD> 1
</TD></TR><TR><TD> CutSmart/ul </TD><TD> 5
</TD></TR><TR><TD> pBX-123/ul </TD><TD> 3
</TD></TR><TR><TD> ddH<SUB>2</SUB>0/ul </TD><TD> 41
</TD></TR><TR><TD> Total/ul </TD><TD> 50
</TD></TR></TBODY></TABLE><P>Purification: 40 ul system
</P><P>Results: ① 18.1 ng/μL A260/280: 1.64
</P><P>Transform 3 plates (Amp+)
</P><P>4. He Yuhao repeats PBX123 NotI-cut:
</P><P>NotI digest: 2 tubes
</P><TABLE class="table table-striped"><TBODY><TR><TD>NotI/ul </TD><TD> 1
</TD></TR><TR><TD> CutSmart/ul </TD><TD> 5
</TD></TR><TR><TD> pBX-123/ul </TD><TD> 3
</TD></TR><TR><TD> ddH<SUB>2</SUB>0/ul </TD><TD> 41
</TD></TR><TR><TD> Total/ul </TD><TD> 50
</TD></TR></TBODY></TABLE><P>Purification: 30 ul system each
</P><P>Results: ① 25 ng/μL A260/280: 1.70
</P><P>② 23 ng/μL A260/280: 1.66
</P><H3><SPAN class="mw-headline" id="21_th">21 <SUP>th</SUP></SPAN></H3><P>TRPC5 plasmid construction
</P><P>1.Select single colony
</P><P>All 3 culture dishes grow many colonies,but we still need to do colony
</P><P>PCR to confirm they are blunt-ended.Ling Shaohua has designed the primer
</P><P>for colony PCR,we need to ask Pro.Huang whether it’s right.
</P><P>So we select 10 colonies,each with 10ul Amp+ LB broth and store at cold
</P><P>room.When the primer is verified and synthesized,we will do colony PCR
</P><P>and extract plasmid from those colonies which are confirmed to be right.
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX-123 Not I cut 2.20 ①</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX-123 Not I cut 2.20 ②</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">PBX-123 Not I cut 2.20 ③</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="22_th">22 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="TRPC5_plasmid_construction_6">TRPC5 plasmid construction</SPAN></H4><P>1.Enzyme double digestion of HindIII-HF，Not I-HF. 37 ℃, 3hrs
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> pBX-123 Not I-cut plasmid(~249ng/ul) </TD><TD> 4
</TD></TR><TR><TD> CutSmart buffer(10X) </TD><TD> 5
</TD></TR><TR><TD> HindIII-HF </TD><TD> 1
</TD></TR><TR><TD> NotI-HF </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 39
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> pBX-123 Not I-cut plasmid(~252ng/ul) </TD><TD> 4
</TD></TR><TR><TD> CutSmart buffer(10X) </TD><TD> 5
</TD></TR><TR><TD> HindIII-HF </TD><TD> 1
</TD></TR><TR><TD> NotI-HF </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 39
</TD></TR></TBODY></TABLE><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">Fig.1 PBX-123 plasmid pattern</DIV></DIV></DIV></DIV>
2.Gel running, proof:
<DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">This shows we have successfully delete the Not-I RE site.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="24_th">24 <SUP>th</SUP></SPAN></H3><P>TRPC5 plasmid construction
</P><P>1. TRPC5 enzyme digestion
</P><P>We cut the TRPC5 PCR gel extraction from day 2.16 to give it a shot.
</P><P>Only 23μL DNA left inside the tube.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul
</TH></TR><TR><TD> DNA </TD><TD> 23
</TD></TR><TR><TD> CutSmart buffer(10X) </TD><TD> 5
</TD></TR><TR><TD> BclI </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 39
</TD></TR><TR><TD> Total </TD><TD> 50
</TD></TR></TBODY></TABLE><P>2. <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A>：
</P><P>We transform PBX133 PBX090 and Piezo to propagate them for further use.
</P><P>We use the micropipette to inhale and blow the liquid inside Piezo for the plasmid sample, and add it to 33μL competent cells.
</P><P>We add 2.5μL PBX133 and 2.5μL PBX090 to 33μL competent cells.
</P><P>We coat and incubate at 37℃for 16 hours.
</P><H2><SPAN class="mw-headline" id="April">April</SPAN></H2><H3><SPAN class="mw-headline" id="13_th.7E24_th">13 <SUP>th</SUP>~24 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="087.26NFAT_Plasmid_Construction">087&amp;NFAT Plasmid Construction</SPAN></H4><P><B>Restriction Enzyme Digestion for Verification</B></P><B>Gel Extraction</B><B>Sequencing Result</B>:
<P>Correct
</P><P><B>04.24~05.06 TRPC5 Plasmid Construction </B></P><P><B>06.15~06.30 Restriction Enzyme,Ligase and Transformation Efficiency Test sfgfp PCR and Gel Extraction</B></P><B>Transformation Result</B><H2><SPAN class="mw-headline" id="July">July</SPAN></H2><H3><SPAN class="mw-headline" id="0_1_st">0 1 <SUP>st</SUP></SPAN></H3><P><B>Efficiency Tests</B></P><P>Transformation efficiency (3 groups)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cell/ul </TH><TH> sfGFP plasmid/ng </TH><TH> sfGFP plasmid/ul
</TH></TR><TR><TD> 33 </TD><TD> 45.88 </TD><TD> 0.3
</TD></TR></TBODY></TABLE><P>Enzyme digestion efficiency (3 groups)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cell/ul </TH><TH> sfGFP digested/ng </TH><TH> sfGFP digested/ul
</TH></TR><TR><TD> 33 </TD><TD> 45.88 </TD><TD> 1.9
</TD></TR></TBODY></TABLE><P>1&gt;Prepare digestion system:
</P><TABLE class="table table-striped"><TBODY><TR><TH> Amount/ ul
</TH></TR><TR><TD> DNA </TD><TD> 2.4
</TD></TR><TR><TD> AflII </TD><TD> 1
</TD></TR><TR><TD> EcoRI </TD><TD> 1
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 18.1
</TD></TR><TR><TD> Total </TD><TD> 25
</TD></TR></TBODY></TABLE><P>Incubate at 37℃,6hrs.
</P><P>Ligation efficiency (3 groups)
</P><TABLE class="table table-striped"><TBODY><TR><TH> Competent cell/ul </TH><TH> sfGFP plasmid ligated/ng </TH><TH> sfGFP plasmid ligated/ul
</TH></TR><TR><TD> 33 </TD><TD> 41.67 </TD><TD> 3.3
</TD></TR></TBODY></TABLE><P>1&gt; Prepare ligation system <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TD> Vector </TD><TD> 0.02pmol
</TD></TR><TR><TD> Insert </TD><TD> 0.08pmol
</TD></TR><TR><TD> T4 ligase </TD><TD> 1ul
</TD></TR><TR><TD> T4 ligase buffer 10X </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20ul
</TD></TR><TR><TD> Total </TD><TD> 20ul
</TD></TR></TBODY></TABLE><P>Incubate at room temperature for 20mins
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation  &amp; Competent cell recovery</B></A></P><P>1. Get competent cell (DH5α) from -80℃，lay on ice for 15mins till melted.
</P><P>2. Add plasmids according to the table above, gently blow to mix well.
</P><P>3. Incubate on ice for 30mins.
</P><P>4. Incubate in 42℃ water bath for 90s.
</P><P>5. Lay on ice for 5mins.
</P><P>6. Add 300μl LB, 37℃, 220rpm for 20mins.
</P><P>7. Add 300μl Amp+ LB, 37℃, 220rpm for 20mins.
</P><P>8. Centrifuge at 5000rpm, 1min. Drop 400μl suspension.
</P><P>9. Mix the left 200μl gently, spread plate.
</P><P><B>NOTE:</B></P><OL><LI> For ligation products, subpackage of competent cell may decrease the final efficiency.</LI><LI> During transformation, after heat shock, competent cell should be spread on preheated agar plate (37℃) to reach a higher transformation efficiency.</LI></OL><H3><SPAN class="mw-headline" id="02_nd">02 <SUP>nd</SUP></SPAN></H3><P><B>Efficiency tests results:</B></P><TABLE class="table table-striped"><TBODY><TR><TH> Green colony number</TH><TH> Non-green colony number
</TH></TR><TR><TD> Transformation 1</TD><TD>&gt;1000 </TD><TD>24
</TD></TR><TR><TD> Transformation 2</TD><TD>&gt;1000 </TD><TD>32
</TD></TR><TR><TD> Transformation 3</TD><TD>&gt;1000 </TD><TD>21
</TD></TR><TR><TD> Digestion1 </TD><TD> 0 </TD><TD>382
</TD></TR><TR><TD> Digestion2 </TD><TD> 1 </TD><TD>370
</TD></TR><TR><TD> Digestion3 </TD><TD> 0 </TD><TD>400
</TD></TR><TR><TD> Ligation1 </TD><TD> 0 </TD><TD> 300
</TD></TR><TR><TD> Ligation2 </TD><TD> 4 </TD><TD> 302
</TD></TR><TR><TD> Ligation3 </TD><TD> 0 </TD><TD> 267
</TD></TR></TBODY></TABLE><P>NOTE:
</P><P>Too many non-target bacterial colonies(non-green).
</P><P>Efficiency tests failed.
</P><P><B>Non-green colony test (I)</B></P><P>Pick 1 non-green colony on each plate.
</P><P>Mix with 20μl ddH<SUB>2</SUB>O
</P><P>Get 10μl add into 5ml Amp+ LB to incubate at 37℃, 220rpm for &lt;16hrs.
</P><P>Get 10μl to do colony PCR:
</P><P>1&gt; Primer
</P><P>Forward -- AACGTATAAGCTTTAGGCGTGTACG
</P><P>Reverse -- TTGTGCACCAGTCATAGCCGAATAG
</P><P>2&gt; Polymerase Chain Reaction (PCR)system&lt;/ul&gt;</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Q5.C2.AE_High-Fidelity_2X_Master_Mix.28M0492S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Q5® High-Fidelity 2X Master Mix(M0492S)</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH>Amount/ul
</TH></TR><TR><TD> Q5 Master Mix 2X </TD><TD> 12.5
</TD></TR><TR><TD> Colony mixture </TD><TD> 10
</TD></TR><TR><TD> PrimerF </TD><TD> 1
</TD></TR><TR><TD> PrimerR </TD><TD> 1
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 0.5
</TD></TR><TR><TD> Total </TD><TD> 25ul
</TD></TR></TBODY></TABLE><P>3&gt; PCR setting
</P><TABLE class="table table-striped"><TBODY><TR><TH>STEP </TH><TH>TEMP </TH><TH>TIME
</TH></TR><TR><TD> Initial Denaturation </TD><TD> 98℃ </TD><TD>30s
</TD></TR><TR><TD> 30 Cycles </TD><TD> 98℃ </TD><TD>10s
</TD></TR><TR><TD> 30 Cycles </TD><TD> 50-72℃ </TD><TD>20s
</TD></TR><TR><TD> 30 Cycles </TD><TD> 72℃ </TD><TD>20s
</TD></TR><TR><TD> Final Extension </TD><TD> 72℃ </TD><TD>2mins
</TD></TR><TR><TD> Hold </TD><TD> 10℃ </TD><TD> ∞
</TD></TR></TBODY></TABLE><P>4&gt; Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="03_th">03 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Non-green_colony_test_.28II.29">Non-green colony test (II)</SPAN></H4><P>1. Use E.Z.N.A.<SUP>TM</SUP> Plasmid Mini Kit(Omega D6942-02) to purified plasmids. 
</P><P>According to <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>
Test the concentration ( Eppendorf BioSpectrometer)
</P><P>Use another pairs of primers to do PCR
</P><P>Primer Forward -- CTAGAGTCGCGGCCGCTTTACTT
</P><P>Primer Reverse -- TCGAGCTCTTGACAGCTAGCTCA
</P><P>4.Double-enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> Non-green plasmid </TD><TD> 200ng
</TD></TR><TR><TD> AflII </TD><TD> 1ul
</TD></TR><TR><TD> EcoRI </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20ul
</TD></TR><TR><TD> Total </TD><TD> 20ul
</TD></TR></TBODY></TABLE><P>
5. Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="04_th">04 <SUP>th</SUP></SPAN></H3><P><B>Non-green colony test (III)</B></P><P>1.Enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> DNA </TD><TD> 200ng
</TD></TR><TR><TD> ScaI </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 10ul
</TD></TR><TR><TD> Total </TD><TD> 10ul
</TD></TR></TBODY></TABLE><P>
2. Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="06_th">06 <SUP>th</SUP></SPAN></H3><P>1.Enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> Non-green plasmid </TD><TD> 200ng
</TD></TR><TR><TD> ScaI/XbaI/XhoI </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20ul
</TD></TR><TR><TD> Total </TD><TD> 20ul
</TD></TR></TBODY></TABLE><P>
2. Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="07_th-10_th">07 <SUP>th</SUP>-10 <SUP>th</SUP></SPAN></H3><P><B>New plasmids design:</B></P><UL><LI> TRPC5-Loxp</LI></UL><UL><LI> Sfgfp-pBX123</LI></UL><UL><LI> pBX097-neoloxp</LI></UL><H3><SPAN class="mw-headline" id="11_th">11 <SUP>th</SUP></SPAN></H3><P><B>TRPC5-Loxp Plasmid Construction(I)</B></P><P>1. TRPC5-R plasmid enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> TRPC5-R </TD><TD> 2ug
</TD></TR><TR><TD> DraIII </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20ul
</TD></TR><TR><TD> Total </TD><TD> 20ul
</TD></TR></TBODY></TABLE><P>
2. Gel electrophoresis for gel extraction
</P>
NOTE: Wrong length and faint bands
<P>DraIII digestion site:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="12_th">12 <SUP>th</SUP></SPAN></H3><P><B>TRPC5-Loxp Plasmid Construction(II)</B></P><P>1.TRPC5-L and TRPC5-R PCR
</P><P>Primer F -- TTGGCAAAGAATTCCTCGAGG
</P><P>Primer R -- GCCGATCATGCTGATATTGAGT
</P><P>2.Gel electrophoresis for gel extraction
</P><P>3.Gel Extraction (According to <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A> )
</P><P>4.TRPC5-L plasmid re-synthesis
</P><H3><SPAN class="mw-headline" id="13_th">13 <SUP>th</SUP></SPAN></H3><P><B>pBX097-neoloxp construction(I)</B></P><P>Neoloxp PCR from pBX123 plasmid
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A></P><P>3. Test product’s concentration
</P><P>4.Double-enzyme digestion and pBX097 plasmid enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> pBX097/neoLoxP </TD><TD> 3ug
</TD></TR><TR><TD> MfeI </TD><TD> 1ul
</TD></TR><TR><TD> BamHI </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 5ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 50ul
</TD></TR><TR><TD> Total </TD><TD> 50ul
</TD></TR></TBODY></TABLE><P>
5.Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="16_th_-_21_th">16 <SUP>th</SUP> - 21 <SUP>th</SUP></SPAN></H3><P><B>pBX097-neoloxp construction(II) </B></P><P>1.Ligate the pBX097 and neoloxp with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A> and <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Quick_Ligation.E2.84.A2_Kit_.28M2200S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Quick Ligation</B></A></P><P>2.DNA <I>transformation</I></P><P>Result: None colony
</P><P><B>TRPC5-Loxp Plasmid Construction(III)</B></P><P>TRPC5-R PCR product double-enzyme digestion (AflII; KpnI) overnight
</P><P>TRPC5-L synthesis products transformation
</P><P>TRPC5-L plasmid purification
</P><P>TRPC5-L plasmid enzyme digestion (AflII; KpnI) overnight
</P><P>Ligation of TRPC5 left and right (T4 DNA ligase)
</P><P>Spread plate, incubate at 37℃ for &lt;16hrs
</P><P>Pick single colony, incubate in 15ml Amp+ LB for 16hrs at 37℃
</P><P>Separate the culture into 5ml(to prove); 9.5ml(for endofree); 0.5ml (for 
</P><P>preservation in -80℃)
</P><P>Single-enzyme digestion (PvuI; BamHI) and gel electrophoresis
</P><P>Correct
</P><P><B>pBX123-sfGFP construction</B></P><OL><LI> sfGFP PCR</LI><LI> Pbx123 double-enzyme digestion</LI></OL><P>3. Double-enzyme digestion ( AgeI; BamHI ) of sfGFP Overnight
</P><P>4.<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A> and <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>5. Ligation of sfGFP and pBX123 backbone
</P><P>6. Spread plate incubate at 37℃, &lt;16hrs
</P><P>7. Pick 8 colonies incubated in 5ml Amp+ LB each, at 37℃ 220rpm for &lt;16hrs
</P><P>8. Plasmid purification
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol</B></A></P><P>9. Double enzyme digestion for proof (AflII; EcoRI)
</P><P>10.Gel electrophoresis
</P><H3><SPAN class="mw-headline" id="22_th_-_23_th">22 <SUP>th</SUP> - 23 <SUP>th</SUP></SPAN></H3><P><B>pBX097-neoloxp construction(II) </B></P><P>1.Ligate the pBX097 and neoloxp with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><P>2.DNA transformation
</P><P>Result: 7 colonies
</P><P>3. Pick colonies to 5ml Amp+ LB, incubate at 37℃, 220rpm for &lt;16hrs.
</P><P>4. Plasmid extraction with Plasmid Mini Kit (Omega)
</P><P>5.Single-enzyme digestion
</P><TABLE class="table table-striped"><TBODY><TR><TD> pBX097/neoLoxP </TD><TD> 200ng
</TD></TR><TR><TD> AflIII/ScaI </TD><TD> 1ul
</TD></TR><TR><TD> CutSmart(NEB) </TD><TD> 2ul
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20ul
</TD></TR><TR><TD> Total </TD><TD> 20ul
</TD></TR></TBODY></TABLE><P>Incubate at 37℃, overnight.
</P><H3><SPAN class="mw-headline" id="24_th_-_25_th">24 <SUP>th</SUP> - 25 <SUP>th</SUP></SPAN></H3><P><B>pBX097-neoloxp construction(II) </B></P><P>1.Gel electrophoresis
</P><P>Result: pBX097-neoloxp successfully constructed.
</P><H3><SPAN class="mw-headline" id="26_th_-_30_th">26 <SUP>th</SUP> - 30 <SUP>th</SUP></SPAN></H3><P><B>TRPC5-Loxp Plasmid Construction(IV)</B></P><P>1.TRPC5-Loxp <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Endo-Free_Plasmid_DNA_Mini_Kit_II_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>endofree plasmid purification.</B></A></P><P>2.pBX097-neoloxp and TRPC5 sequencing
</P><P>Primer for TRPC5---GCCGATCATGCTGATATTGAGT
</P><P>Primer for pBX097-neoloxp---GGCAACGTGCTGGTTATTGTGC
</P><H2><SPAN class="mw-headline" id="Aug.">Aug.</SPAN></H2><H3><SPAN class="mw-headline" id="01_st_-_02_nd">01 <SUP>st</SUP> - 02 <SUP>nd</SUP></SPAN></H3><P><B>pBX123-sfGFP Efficiency test</B></P><TABLE class="table table-striped"><TBODY><TR><TD> pBX123-sfGFP Efficiency Test(AflII, EcoRI) </TD><TD> Transformation
</TD></TR><TR><TD> pBX123-sfGFP Efficiency Test(AflII, EcoRI) </TD><TD> Enzyme digestion
</TD></TR><TR><TD> pBX123-sfGFP Efficiency Test(AflII, EcoRI) </TD><TD> Ligation
</TD></TR></TBODY></TABLE><P>
Repeat the experiments steps mentioned in 7.1
</P><H3><SPAN class="mw-headline" id="03_rd_-_04_th">03 <SUP>rd</SUP> - 04 <SUP>th</SUP></SPAN></H3><P><B>pBX123-sfGFP enzymes’ digestion sites ligation ability</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><TABLE class="table table-striped"><TBODY><TR><TH> Groups </TH><TH> Combination
</TH></TR><TR><TD> 1 </TD><TD> AgeI+BamHI
</TD></TR><TR><TD> 2 </TD><TD> AgeI+AflI
</TD></TR><TR><TD> 3 </TD><TD> EcoRI+BamHI
</TD></TR><TR><TD> 4 </TD><TD> EcoRI+AflI
</TD></TR></TBODY></TABLE><P>1&gt; Prepare digestion system, incubate overnight
</P><P>2&gt;<A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A> to get different fragments
</P><P>3&gt; Ligation of <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><P>4&gt; Transformation
</P><P>Results:
</P><TABLE class="table table-striped"><TBODY><TR><TH> Groups </TH><TH> Colony number
</TH></TR><TR><TD> AgeI+BamHI  </TD><TD> 0
</TD></TR><TR><TD>  AgeI+AflI </TD><TD> 0
</TD></TR><TR><TD>  EcoRI+BamHI </TD><TD>non-green
</TD></TR><TR><TD>  EcoRI+AflI </TD><TD> non-green
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="05_th">05 <SUP>th</SUP></SPAN></H3><P><B>TRPC5-loxp sequencing results:</B></P><TABLE class="table table-striped"><TBODY><TR><TD> TRPC5-(1) </TD><TD> 1 mismatch
</TD></TR><TR><TD> TRPC5-(2) </TD><TD> Correct
</TD></TR><TR><TD> TRPC5-(3) </TD><TD> 1 Failed
</TD></TR><TR><TD> TRPC5-(4) </TD><TD> 1 Correct
</TD></TR></TBODY></TABLE><P><B>pBX097-neoloxp sequencing results:</B></P><TABLE class="table table-striped"><TBODY><TR><TD> 097-neoLoxP-(1) </TD><TD> Correct
</TD></TR><TR><TD> 097-neoLoxP-(2) </TD><TD> Correct
</TD></TR><TR><TD> 097-neoLoxP-(3) </TD><TD> 3 mismatches
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="06_th_-_07_th">06 <SUP>th</SUP> - 07 <SUP>th </SUP></SPAN></H3><P>Non-green colony PCR
</P><H3><SPAN class="mw-headline" id="09_th">09 <SUP>th</SUP></SPAN></H3><P><B>Change NFAT-YFP to NFAT-EGFP(I)</B></P><P>Double enzyme digestion (AgeI, BamHI) pBX123 to get EGFP fragment
</P><P>Double enzyme digestion (AgeI, BamHI) NFAT-YFP to get backbone Overnight
</P><H3><SPAN class="mw-headline" id="10_th">10 <SUP>th</SUP></SPAN></H3><P><B>Change NFAT-YFP to NFAT-EGFP(II)</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A> for EGFP and NFAT backbone
</P><P>Ligation by <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A></P><P>Transformation
</P><H3><SPAN class="mw-headline" id="11_th_-_13_th">11 <SUP>th</SUP> - 13 <SUP>th</SUP></SPAN></H3><P><B>Change NFAT-YFP to NFAT-EGFP(II)</B></P><OL><LI> Pick single colony to culture in 15ml Amp+ LB for 16hrs</LI><LI><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Plasmid_DNA_Mini_Kit_I_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Plasmid DNA Mini Kit I Protocol</B></A> to get purified plasmid</LI><LI> Enzyme digestion to prove</LI><LI> Send to sequencing</LI></OL><H3><SPAN class="mw-headline" id="22_th_2">22 <SUP>th</SUP></SPAN></H3><P><B>TRPC5 Random mutagenesis(I)</B></P><TABLE class="table table-striped"><TBODY><TR><TD> TRPC5 Random mutagenesis</TD><TD> Megaprimer PCR
</TD></TR><TR><TD> TRPC5 Random mutagenesis</TD><TD> Gibson
</TD></TR><TR><TD> TRPC5 Random mutagenesis</TD><TD> Enzyme Digestion
</TD></TR></TBODY></TABLE><P><B>Megaprimer PCR</B></P><P>1&gt; Pre-experiments: T5 exonuclease digestion level according to time.
</P><P>a. Use PrimerStar to do PCR of TRPC5 mutation region (~640bp)
</P><P>b. Gel extraction to get TRPC5 mutation region
</P><P>c. Set a series gradients of T5 digestion time: 0, 5, 10, 20mins
</P><TABLE class="table table-striped"><TBODY><TR><TH>T5 exonuclease digestion time</TH><TH> groups
</TH></TR><TR><TD> 5 mins</TD><TD> 1,2
</TD></TR><TR><TD> 10 mins</TD><TD> 3,4
</TD></TR><TR><TD> 20 mins</TD><TD> 5,6
</TD></TR></TBODY></TABLE><P>d. EDTA preparation
</P><P>To get EDTA solution with final concentration of 11mM (Planing to add 1μl solution into
</P><P>50μl to stop reaction). We should get 540mM stock solution.
</P><UL><LI> Weigh 1.08g EDTA powder add into 5ml ddH<SUB>2</SUB>O
</LI><LI> Use magnetic stirring bar to help dissolve
</LI><LI> Add NaOH powder to adjust solution’s PH until dissolved completely
</LI><LI> Use 0.22μm filter to filtrate the solution
</LI><LI> store in -20℃</LI></UL><P>Note: To keep T5 exonuclease digestion time as precise as possible, according to its
protocol, we use EDTA to stop the digestion.
e. T5 digestion (according to <B>T5 exonuclease protocol</B>)
</P><TABLE class="table table-striped"><TBODY><TR><TD> T5 exonuclease</TD><TD> 3.4ul
</TD></TR><TR><TD> NEB4.0 Buffer</TD><TD> 5ul
</TD></TR><TR><TD> TRPC5 region </TD><TD> 3.4ug
</TD></TR><TR><TD> ddH<SUB>2</SUB>O</TD><TD> to 50ul
</TD></TR><TR><TD> Total</TD><TD> 50ul
</TD></TR></TBODY></TABLE><P>f. Take group①②③for gel extraction; ④⑤⑥for cycle pure.
</P><H3><SPAN class="mw-headline" id="23_th">23 <SUP>th</SUP></SPAN></H3><P><B>Megaprimer PCR</B>
1&gt; Send ④⑤⑥ to sequence
Primer F:ATCCGAATTCCCCTCCAAATC
Primer R:TATGTTAAGTTCCCAGCCCAG
2&gt; Megaprimer PCR:
</P><UL><LI> Use <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Q5.C2.AE_High-Fidelity_2X_Master_Mix.28M0492S.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Q5® High-Fidelity 2X Master Mix(M0492S)</B></A>, System 50μl, Tm=65℃
</LI><LI><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A> to get purified PCR products.
</LI><LI> Use DpnI to digest plasmids that come from bacteria (methylated sites)
</LI><LI> Transformation of digested products.</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="24_th_2">24 <SUP>th</SUP></SPAN></H3><P>Megaprimer PCR results of different digestion time and ratios of templet and 
</P><P>primer:
</P><P>TRPC5 mutation region PCR
</P><P>TPRC5 preserved bacteria plate streaking
</P><P>DpnI digestion time experiment: “Whether 100ng plasmid can be digested completely by 1μl DpnI in 25μl, for 1h”
</P><P>Result:
</P><H3><SPAN class="mw-headline" id="25_th">25 <SUP>th</SUP></SPAN></H3><P><B>Megaprimer PCR</B></P><OL><LI> To find out the best annealing temperature for megaprimer. Set a series of annealing temperatures: 55, 60,65,70℃</LI><LI> Repeat PCR</LI><LI><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A> to get purified PCR products.</LI><LI> DpnI digestion</LI><LI><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A></LI></OL><H3><SPAN class="mw-headline" id="26_th">26 <SUP>th</SUP></SPAN></H3><P>Megaprimer PCR results of different digestion time;PCR annealing temperatures and ratios of templet and primer:
</P><P>Colony number according to different annealing temperatures and ratios of templet and primer:
</P><TABLE class="table table-striped"><TBODY><TR><TH>PCR Annealing TEMP</TH><TH>Primer:Template= 1:1</TH><TH>Primer:Template= 1:2
</TH></TR><TR><TD> 55℃</TD><TD> 125 </TD><TD>44
</TD></TR><TR><TD> 60℃</TD><TD> 120 </TD><TD>100
</TD></TR><TR><TD> 65℃</TD><TD> 116 </TD><TD>58
</TD></TR><TR><TD> 70℃</TD><TD> 168 </TD><TD>76
</TD></TR></TBODY></TABLE><P>Sequencing results of T5 exonuclease digestion level according to time:
</P><TABLE class="table table-striped"><TBODY><TR><TH>T5 Exonuclease digestion time</TH><TH>5' end digest number/bp</TH><TH>Direction
</TH></TR><TR><TD> 5mins</TD><TD> 48 </TD><TD>Forward
</TD></TR><TR><TD> 5mins</TD><TD> 43 </TD><TD>Reverse
</TD></TR><TR><TD> 10mins</TD><TD> 176 </TD><TD>Forward
</TD></TR><TR><TD> 10mins</TD><TD> 166 </TD><TD>Reverse
</TD></TR></TBODY></TABLE><P>
Repeat T5 exonuclease digestion level according to time:
</P><P>Set series of time gradients: 0, 5, 7.5, 10mins
</P><P>a. T5 digestion (according to T5 exonuclease protocol)
</P><TABLE class="table table-striped"><TBODY><TR><TD>T5 Exonuclease</TD><TD> 3.4ul
</TD></TR><TR><TD> NEB4.0 Buffer</TD><TD> 5ul
</TD></TR><TR><TD> TRPC5 region </TD><TD>3.4ug
</TD></TR><TR><TD> ddH<SUB>2</SUB>O</TD><TD> to 50ul
</TD></TR><TR><TD> Total</TD><TD> 50ul
</TD></TR></TBODY></TABLE><P>
b. Take each group for Gel electrophoresis
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">{{{caption}}}</DIV></DIV></DIV></DIV><P>c. Send each group to sequence
</P><P>Primer F:ATCCGAATTCCCCTCCAAATC
</P><P>Primer R:TATGTTAAGTTCCCAGCCCAG
</P><H3><SPAN class="mw-headline" id="28_th">28 <SUP>th</SUP></SPAN></H3><P>Sequencing results of T5 exonuclease digestion level according to time:
</P><TABLE class="table table-striped"><TBODY><TR><TH>T5 Exonuclease digestion time</TH><TH> 5' end digest number/bp </TH><TH>Direction
</TH></TR><TR><TD>5mins</TD><TD> 42 </TD><TD>Forward
</TD></TR><TR><TD>5mins</TD><TD> 43 </TD><TD>Reverse
</TD></TR><TR><TD>7.5mins</TD><TD> 104 </TD><TD>Forward
</TD></TR><TR><TD>7.5mins</TD><TD> 96 </TD><TD>Reverse
</TD></TR><TR><TD>10mins</TD><TD> 188 </TD><TD>Forward
</TD></TR><TR><TD>10mins</TD><TD> 156 </TD><TD>Forward
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="29_th_2">29 <SUP>th</SUP></SPAN></H3><P><B>Culture medium Pollution proving:</B></P><P>Set 4 groups of experiments (use sfGFP plasmid):
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="30_th">30 <SUP>th</SUP></SPAN></H3><P>Culture medium Pollution proving results:
</P><P>Group 3 number &gt; Group 1 number ; with lots of non-green bacteria
</P><P>Group 4 has &gt;100 colonies of non-green bacteria
</P><P>Group 2 has no colony
</P><P>Conclusion: Culture medium has been polluted.
</P><P><B>Gibson assembly (pre-experiment)</B></P><P>1&gt; Directly cut the mutation region by AflII and EcoRI for 8hrs
</P><P>2&gt; Gel extraction to get the 640bp fragment
</P><P>3&gt; Use <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Gibson_Assembly.C2.AE_Master_Mix_.E2.80.93_Assembly_.28E2611.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gibson Assembly® Master Mix</B></A> to get whole plasmid
</P><P>4&gt; Transformation
</P><H3><SPAN class="mw-headline" id="31_th">31 <SUP>th</SUP></SPAN></H3><P>1.Gibson assembly result: NONE colony
</P><P>2.Random mutagenesis with Kit, according to <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#GeneMorph_II_Random_Mutagenesis_Kit.28Agilent_Technologies.2CCatalog_.23200550.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Random mutagenesis protocol</B></A></P><H2><SPAN class="mw-headline" id="Sept.">Sept.</SPAN></H2><H3><SPAN class="mw-headline" id="01_st_2">01 <SUP>st</SUP></SPAN></H3><P><B>Design qPCR Primers and Send for Synthesization</B></P><P>Sequences:
</P><P>1.097&amp;NeoLoxP qPCR-1st
</P><P>2. qPCR Internal Positive Control
</P><H3><SPAN class="mw-headline" id="04_th_2">04 <SUP>th</SUP></SPAN></H3><H4><SPAN class="mw-headline" id="Gibson_Assembly.C2.AE_Master_Mix"><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Gibson_Assembly.C2.AE_Master_Mix_.E2.80.93_Assembly_.28E2611.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gibson Assembly® Master Mix</B></A></SPAN></H4><P>Insert : vector=2 : 3(50~100ng vector)
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul(for 20bp) </TH><TH>Volume/ul(for 30bp)</TH><TH>fmol
</TH></TR><TR><TD>Vector</TD><TD> 1 </TD><TD>1</TD><TD>19.4
</TD></TR><TR><TD>Insert</TD><TD> 8.6 </TD><TD>3.1</TD><TD>58.2
</TD></TR><TR><TD>2X Master Mix</TD><TD> 10 </TD><TD>10</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 0.4 </TD><TD>5.9</TD></TR><TR><TD>Total</TD><TD> 20 </TD><TD>20</TD></TR></TBODY></TABLE><P><I><B>T5 Exonuclease Digestion</B></I></P><P>Temperature: 50℃
</P><TABLE class="table table-striped"><TBODY><TR><TH> 20bp) </TH><TH>30bp
</TH></TR><TR><TD>Digestion time/min</TD><TD> 40 </TD><TD>60
</TD></TR></TBODY></TABLE><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A><B>into Competent Cells</B></P><P>5μl product are transformed with 50ul competent cells.
</P><P><B>Error-prone PCR(low frequency)</B></P><P>1. Information supplied with <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#GeneMorph_II_Random_Mutagenesis_Kit.28Agilent_Technologies.2CCatalog_.23200550.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Random mutagenesis protocol</B></A></P><P>(Agilent Technologies,Catalog #200550):
</P><P>2. Primers sequence:
</P><P>Forward: ttactcaccatacagagatcgaattcc
</P><P>Reverse: ttttccactttgctcagctccttaag
</P><P>Target DNA mass: 500ng,corresponding to low frequency mutation
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 9.5
</TD></TR><TR><TD>dNTP mix</TD><TD> 1
</TD></TR><TR><TD>Mutazyme II DNA polymerase</TD><TD> 1
</TD></TR><TR><TD>Forward Primer</TD><TD> 1
</TD></TR><TR><TD>Reverse Primer</TD><TD> 1
</TD></TR><TR><TD>Template</TD><TD> 31.5
</TD></TR><TR><TD>10X Mutazyme II reaction buffer</TD><TD> 5
</TD></TR><TR><TD>Total</TD><TD> 50
</TD></TR></TBODY></TABLE><P>4. Procedure
</P><P>The annealing temperature is set to be 57℃.
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A><B>of ep-PCR Product</B></P><P>Concentration: 135ng/μl, 30μl
</P><P><B>qPCR Primer Test</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P>Kit: Power SYBR® Green PCR Master Mix and RT-PCR Reagents Kit
</P><P>Sample:
</P><P>CHO cell genomic DNA extracted from 4.5x10<SUP>6</SUP> cells.
</P><P>Concentration: 56.1ng/μl, 400μl
</P><P>Target:
</P><P>Neoloxp segment, 18sRNA, GAPDH
</P><P>Standard:
</P><P>097&amp;Neoloxp plasmid(length:6635bp)
</P><P>Concentration: 101.2ng/μl
</P><TABLE class="table table-striped"><TBODY><TR><TH>Stock Conc./ng*ul<SUP>-1</SUP></TH><TH> Volume/ul</TH><TH>Dilute Volume/ul
</TH></TR><TR><TD>c(101.2)</TD><TD> 1</TD><TD>99
</TD></TR><TR><TD>c/100</TD><TD>1</TD><TD>99
</TD></TR><TR><TD>c/200</TD><TD> 50</TD><TD>50
</TD></TR><TR><TD>c/400</TD><TD> 50</TD><TD>50
</TD></TR><TR><TD>c/800</TD><TD> 50</TD><TD>50
</TD></TR><TR><TD>c/1600</TD></TR></TBODY></TABLE><P>4.
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul</TH><TH>Final Conc. </TH><TH>Negative Control
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 6.4</TD><TD>8.4
</TD></TR><TR><TD>2XSYBR® Master Mix</TD><TD> 10</TD><TD>10
</TD></TR><TR><TD>Forward Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Reverse Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Template</TD><TD> 2</TD><TD>0
</TD></TR><TR><TD>Total</TD><TD> 20</TD><TD>20
</TD></TR></TBODY></TABLE><P>5. Procedure
</P><H3><SPAN class="mw-headline" id="05_th_2">05 <SUP>th</SUP></SPAN></H3><P><B>Transformation Result</B></P><P>No colony
</P><P>Possible reason: The transformation efficiency of self-made competent cells is quite low.
</P><H3><SPAN class="mw-headline" id="06_th_2">06 <SUP>th</SUP></SPAN></H3><P><B>TRPC5 PCR with</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#PrimeSTAR_Max_DNA_Polymerase_-_Clontech.28R045Q.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>PrimeStar</B></A></P><P>1.Primers sequence:
</P><P>Forward: ttactcaccatacagagatcgaattcc
</P><P>Reverse: ttttccactttgctcagctccttaag
</P><P>2.Total 8 tubes of PCR reaction:
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul</TH><TH> Final conc./uM
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 20.8</TD></TR><TR><TD>2X Premix</TD><TD> 25</TD></TR><TR><TD>Forward Primer</TD><TD> 1.5</TD><TD>0.3
</TD></TR><TR><TD>Reverse Primer</TD><TD> 1.5</TD><TD>0.3
</TD></TR><TR><TD>Template</TD><TD> 1.2(1ng)</TD></TR><TR><TD>Total</TD><TD> 50</TD></TR></TBODY></TABLE><P>3. Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>10</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 10</TD></TR><TR><TD>55</TD><TD> 10</TD></TR><TR><TD>72</TD><TD> 60</TD><TD>32
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>10</TD><TD> ∞</TD></TR></TBODY></TABLE><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A></P><P>Concentration: 180ng/μl, 30μl, total 4 tubes.
</P><P><B><I>T5 Digestion</I> for 3.5 minutes and</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><P>Concentration: 60ng/μl, 50μl
</P><P><B>Mega-primer PCR with</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#KOD_DNA_Polymerase.28Merck_Millipore.40Novagen.2C_TB320.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>KOD Polymerase</B></A></P><P>Template DNA: 0.1ng,
</P><P>Mega-primer:Template=10000:1
</P><TABLE class="table table-striped"><TBODY><TR><TH>Component</TH><TH> Volume/ul</TH><TH>Final Conc.
</TH></TR><TR><TD>10X buffer#1 for KOD DNA Polymerase</TD><TD> 5</TD><TD>1X
</TD></TR><TR><TD>dNTP (2mM each)</TD><TD> 5</TD><TD>0.2mM(each)
</TD></TR><TR><TD>MgCl<SUB>2<SUB>(10mM)</SUB></SUB></TD><TD> 5</TD><TD> 1mM
</TD></TR><TR><TD>Forward Primer(5pmol/ul)</TD><TD> 4</TD><TD> 0.4uM
</TD></TR><TR><TD>Reverse Primer(5pmol/ul)</TD><TD> 4</TD><TD> 0.4uM
</TD></TR><TR><TD>Template</TD><TD> Y</TD></TR><TR><TD>PCR Grade water</TD><TD> X</TD></TR><TR><TD>KOD DNA Polymerase(2.5U/ul)</TD><TD> 0.4</TD><TD>0.02U/ul
</TD></TR><TR><TD>Total</TD><TD> 50</TD></TR></TBODY></TABLE><P>Procedure:
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>30</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 20</TD></TR><TR><TD>60</TD><TD> 20</TD></TR><TR><TD>72</TD><TD> 180</TD><TD>30
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>10</TD><TD> ∞</TD></TR></TBODY></TABLE><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A><B>and Gel Running</B></P><P>Concentration: 30ng/μl, 50μl
</P>
No 6000bp band is appeared,it’s a problem!!
<P><B>DpnI Digestion for 1 hour</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A></P><P>5μl product are transformed with 100ul competent cells.
</P><P><B>qPCR Primer Test</B></P><P>Instrument: Bio-Rad
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Power_SYBR.C2.AE_Green_PCR_Master_Mix_Kit.28life_technologies.2C4368708.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Kit: TaKaRa-One Step SYBR PrimeScript RT-PCR Kit I</B></A></P><P>1. Sample:
</P><P>CHO cell genomic DNA extracted from 4.5x10<SUP>6</SUP> cells.
</P><P>Concentration: 56.1ng/μl, 400μl
</P><P>2. Target:
</P><P>Neoloxp segment, 18sRNA, GAPDH
</P><P>3. Standard:
</P><P>097&amp;Neoloxp plasmid(length:6635bp)
</P><P>Concentration: 101.2ng/μl
</P><TABLE class="table table-striped"><TBODY><TR><TH>Stock Conc./ng*ul<SUP>-1</SUP></TH><TH> Volume/ul</TH><TH>Dilute Volume/ul
</TH></TR><TR><TD>c(101.2)</TD><TD> 1</TD><TD>99
</TD></TR><TR><TD>c/100</TD><TD> 1</TD><TD>99
</TD></TR><TR><TD>c/10000</TD><TD> 2</TD><TD>18
</TD></TR><TR><TD>c/100000</TD><TD> 2</TD><TD>18
</TD></TR><TR><TD>c/1000000</TD><TD> 2</TD><TD>18
</TD></TR><TR><TD>c/10000000</TD><TD> 2</TD><TD>18
</TD></TR></TBODY></TABLE><P>4.
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul</TH><TH>Final Conc. </TH><TH>Negative Control
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 6.4</TD><TD>8.4
</TD></TR><TR><TD>2XSYBR® Master Mix</TD><TD> 10</TD><TD>10
</TD></TR><TR><TD>Forward Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Reverse Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Template</TD><TD> 2</TD><TD>0
</TD></TR><TR><TD>Total</TD><TD> 20</TD><TD>20
</TD></TR></TBODY></TABLE><P>Wrong: Forget to add ‘PrimeScript Enzyme Mix’ which contains ‘Taq enzyme’,thus no PCR reaction works.
</P><P>5. Procedure
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><A href="/File:T--SUSTech_Shenzhen--3323D013-B56F-429C-ADBB-38AD7C3EA412.png" title="File:T--SUSTech Shenzhen--3323D013-B56F-429C-ADBB-38AD7C3EA412.png">1000px</A></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="07_th">07 <SUP>th</SUP></SPAN></H3><P><B>Transformation Result</B></P><P>No colony
</P><P>Possible reason: The mega-primer PCR procedure didn’t work.
</P><P><B>Mega-primer PCR with <I>Q5</I></B></P><P>1.Template DNA: 0.1ng,
</P><P>Mega-primer:Template=10000:1
</P><TABLE class="table table-striped"><TBODY><TR><TH> Volume/ul</TH><TH> Volume/ul
</TH></TR><TR><TH>Reagent</TH><TH> No T5 Digestion</TH><TH> T5 digestion for 3.5min
<P>-
</P></TH><TD>ddH<SUB>2</SUB>O</TD><TD> 18.4</TD><TD>17.1
</TD></TR><TR><TD>2X Q5 mix</TD><TD> 25</TD><TD>25
</TD></TR><TR><TD>Mega Primer</TD><TD> 0.6</TD><TD>1.9
</TD></TR><TR><TD>Template</TD><TD> 6(0.1ng)</TD><TD>6
</TD></TR><TR><TD>Total</TD><TD> 50</TD><TD>50
</TD></TR></TBODY></TABLE><P>2. Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>30</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 10</TD></TR><TR><TD>60</TD><TD> 30</TD></TR><TR><TD>72</TD><TD> 300</TD><TD>35
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>4</TD><TD> ∞</TD></TR></TBODY></TABLE><P><B>qPCR Primer Test with <I>Premix</I> and Gel Running</B></P><P>1. Gel concentration: 2%(to better show the needed band)
</P><P>2.
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul</TH><TH>Mass/ng
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> X</TD></TR><TR><TD>2X Premix</TD><TD> 12.5</TD></TR><TR><TD>Forward Primer</TD><TD> 0.5</TD></TR><TR><TD>Reverse Primer</TD><TD> 0.5</TD></TR><TR><TD>Template</TD><TD> Y</TD><TD>1ng for plasmid,640ng for gDNA
</TD></TR><TR><TD>Total</TD><TD> 25</TD></TR></TBODY></TABLE><P>3. Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>60</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 20</TD></TR><TR><TD>60</TD><TD> 30</TD></TR><TR><TD>72</TD><TD> 20</TD><TD>30
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>10</TD><TD> ∞</TD></TR></TBODY></TABLE><P>4. Results
</P>
The gel running results show that primer 1~4 and primer GAPDH has been contaminated,we need to use new primer and be care of experiment operation.
<H3><SPAN class="mw-headline" id="08_th">08 <SUP>th</SUP></SPAN></H3><P><B>Gel Running with 30μl mega-primer PCR product</B></P><P>There is a band at around 6000bp position!
</P><P><B>DpnI Digestion for 1 hour</B></P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 3
</TD></TR><TR><TD>DNA</TD><TD> 20
</TD></TR><TR><TD>5X Buffer</TD><TD> 6
</TD></TR><TR><TD>DpnI</TD><TD> 1
</TD></TR><TR><TD>Total</TD><TD>30
</TD></TR></TBODY></TABLE><P><I><B>Transformation</B></I></P><P>10μl product are transformed with 100ul competent cells.
</P><H3><SPAN class="mw-headline" id="9_th">9 <SUP>th</SUP></SPAN></H3><P><B>Transformation Result</B></P><P>Little or no colony is appeared,maybe the transformation efficiency of competent cell is still too low.
</P><P>09.12 Pre-experiment of enzyme digestion and ligation
</P><P>TRPC5 PCR with PrimeStar
</P><P>Primers sequence
</P><P>Forward Primer:　atccGAATTCccctccaaatc
</P><P>Reverse Primer:　ATGATCTCCAGTTCTCTGGA
</P><P>2. Total 2 tubes PCR reaction
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 20.8
</TD></TR><TR><TD>Forward Primer</TD><TD> 1.5
</TD></TR><TR><TD>Reverse Primer</TD><TD> 1.5
</TD></TR><TR><TD>2X Premix</TD><TD> 25
</TD></TR><TR><TD>Template</TD><TD> 1.2(1ng)
</TD></TR><TR><TD>Total</TD><TD>50
</TD></TR></TBODY></TABLE><P>3. Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>10</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 10</TD></TR><TR><TD>55</TD><TD> 10</TD></TR><TR><TD>72</TD><TD> 60</TD><TD>32
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>10</TD><TD> ∞</TD></TR></TBODY></TABLE><P><I><B>Gel Extraction</B></I></P>
Concentration: 44.4ng/μl, 40μl
<P><B>EcoRI,SacI Restriction Enzyme Digestion</B></P><P>1. Insert
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 1
</TD></TR><TR><TD>DNA</TD><TD> 40
</TD></TR><TR><TD>10X Buffer</TD><TD> 5
</TD></TR><TR><TD>EcoRI</TD><TD> 2
</TD></TR><TR><TD>SacI</TD><TD> 2
</TD></TR><TR><TD>Total</TD><TD>50
</TD></TR></TBODY></TABLE><P>The digestion is from 00:20 to 09:00.
</P><P>2. Vector
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 11
</TD></TR><TR><TD>DNA</TD><TD> 30 (5ug)
</TD></TR><TR><TD>10X Buffer</TD><TD> 5
</TD></TR><TR><TD>EcoRI</TD><TD> 2
</TD></TR><TR><TD>SacI</TD><TD> 2
</TD></TR><TR><TD>Total</TD><TD>50
</TD></TR></TBODY></TABLE><P>The digestion is from 21:00 to 09:00.
</P><P><B>qPCR Primer Re-design and Send for Synthesization</B></P><P>Sequences:
</P><P>097&amp;NeoLoxP qPCR-2nd
</P><H3><SPAN class="mw-headline" id="13_th_2">13 <SUP>th</SUP></SPAN></H3><P><I><B>Transformation</B></I></P><P>Use the bought competent cell,with transformation efficiency of approximately 10^7.
</P><P><B><I>Gel Extraction</I> of Enzyme Digested Vector</B></P><P><A href="/File:Media/image165.png" title="File:Media/image165.png">360x360px</A></P><P>Concentration of vector: 44.8ng/μl,20μl
</P><P><B><I>Cycle-pure</I> of Enzyme Digested Insert</B></P><P>Concentration of insert: 25.3ng/μl, 40μl
</P><P><B>Ligation with <I>T4 DNA Ligase</I></B></P><TABLE class="table table-striped"><TBODY><TR><TH>Volume/ul </TH><TH>Mass/ng </TH><TH>pmol </TH><TH>Control
</TH></TR><TR><TD>2</TD><TD>2
</TD></TR><TR><TD>1.4</TD><TD> 62</TD><TD>0.02</TD><TD>1.4
</TD></TR><TR><TD>1.1</TD><TD> 28</TD><TD>0.06</TD><TD>0
</TD></TR><TR><TD>14.5</TD><TD>15.6
</TD></TR><TR><TD>1</TD><TD>1
</TD></TR><TR><TD>20</TD><TD>20
</TD></TR></TBODY></TABLE><P>The ligation is from 12:50 to 14:00.
</P><P><I><B>Transformation</B></I></P><H3><SPAN class="mw-headline" id="14_th_2">14 <SUP>th</SUP></SPAN></H3><P><B>Transformation Result</B></P><TABLE class="table table-striped"><TBODY><TR><TH>Method </TH><TH>Colonies  </TH><TH>Control
</TH></TR><TR><TD>Q5</TD><TD>191 </TD><TD>0
</TD></TR><TR><TD>Gibson</TD><TD> 38</TD><TD>7
</TD></TR><TR><TD>KOD</TD><TD> 0</TD><TD>0
</TD></TR><TR><TD>Ligation</TD><TD>1772 </TD><TD>330
</TD></TR><TR><TD>Positive Control</TD><TD>237 </TD></TR></TBODY></TABLE><P><B>Competent Cell Efficiency</B></P><P>1. Formula: Colonies on plate / ng of DNA plated x 1000 ng/μg
</P><P>2. Calculation:　237 / 0.02 x 1000=1.185x10<SUP>7</SUP> cfu/μg
</P><P>3. Actual meaning: 20pg,5640bp→3.457x10<SUP>6</SUP> copies
</P><P>4. 457x10<SUP>6</SUP> / 237=14586,which means 1 of 14586 vectors would actually grow on the plate.
</P><P><B>Ligation Efficiency</B></P><P>1. 62ng,4906bp→1.232x10<SUP>10</SUP> copies
</P>
2. Efficiency=<DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><P><B>Mutation Library</B></P><P>1.Q5: 2.782x10<SUP>6</SUP></P><P>2.Gibson: 5.54x10<SUP>5</SUP></P><P>3. Ligation:　2.102x10<SUP>7</SUP></P><P><B>Repeat the pre-experiment on 09.12</B></P><P>Use enzyme digestion and ligation method to construct mutation library,except that using error-prone PCR,and the target DNA is 500ng.
</P><H3><SPAN class="mw-headline" id="15_th_2">15 <SUP>th</SUP></SPAN></H3><P><B>qPCR Primer Test with Premix and Gel Running</B></P><P>1. Procedure is the same with the experiment on 09.07
</P><P>2. Results:
</P>
2. Conclusion:
<P>I. Primer 1~4 are not good enough,cause they all can match with CHO cell’s gDNA and have PCR products.
</P><P>II. Primer 2~4 are contaminated.
</P><P>III. Primer GAPDH has unspecific PCR product,so it can’t be used.
</P><P>Primer 18sRNA can be used.
</P><H3><SPAN class="mw-headline" id="18_th_2">18 <SUP>th</SUP></SPAN></H3><P><I><B>Transformation</B></I></P><P>10μl positive ligation product are transformed with 125ul competent cells.For control group,5ul ligation product are transformed with 50ul competent cells.
</P><P><B>qPCR Primer Test with Premix and Gel Running</B></P><P>1. We set 2 annealing temperature,which are 65℃ and 70℃ respectively.
</P><P>2. Results:
</P><P>3. Conclusion:
</P><P>I. Primer 1~4 are all not contaminated and have specific PCR band with annealing temperature of 65℃,while they don’t have PCR product band with annealing temperature of 70℃.
</P><P>II. Raising annealing temperature can reduce unspecific amplification,while it can also decrease the PCR efficiency and cause Ct increasement.Also we need to verify whether the product band is what we need.
</P><H3><SPAN class="mw-headline" id="19_th_2">19 <SUP>th</SUP></SPAN></H3><P><B>Transformation Result</B></P><TABLE class="table table-striped"><TBODY><TR><TH>Colonies  </TH><TH>Control
</TH></TR><TR><TD>10688 </TD><TD>113
</TD></TR></TBODY></TABLE><P>1. Mutation library(10ul ligation product)=10688
</P>
2. Ligation efficiency=<DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Center</DIV></DIV></DIV><P><B>Pick 20 Single Colony for sequencing</B></P><P>Sequencing Result:
</P><P>7 out of 20 plasmids have mutation(s) among specific region,in which one has 15 base mutations,one has 2 base mutations,and the rest five have 1 base mutation.
</P><P><B>Scrape the Colonies for</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Endo-Free_Plasmid_DNA_Mini_Kit_II_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>endofree plasmid purification.</B></A></P><P>The scraped colonies are cultured for around 9 hours till the OD achieves 2.879,with 150ml Amp.
</P><P>Concentration: 55ng/μl, 1.75ml
</P><H3><SPAN class="mw-headline" id="20_th_2">20 <SUP>th</SUP></SPAN></H3><P><B>qPCR Primer Test with Premix and Gel Running</B></P><P>1. We set 3 annealing temperature,which are 61℃,62℃ and 63℃ respectively.
</P><P>2. Results:
</P><H3><SPAN class="mw-headline" id="21_th_2">21 <SUP>th</SUP></SPAN></H3><P>CHO cell <I>Genomic DNA Extraction</I></P><P>Positive: GECO+NFAT+Neoloxp
</P><P>①d1-A10 ②d2-B5 ③d2-B9 ④d1-D3 ⑤d2-E9
</P><P>⑥d1-F5 ⑦d1-F11 ⑧d1-A9 ⑨d1-B3 ⑩d2-C1
</P><P>Control: GECO+NFAT
</P><P>①C4 ②E6 ③F8 ④H10
</P><H3><SPAN class="mw-headline" id="22_th_3">22 <SUP>th</SUP></SPAN></H3><P><B>qPCR Primer Test</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P>Kit: TaKaRa-One Step SYBR PrimeScript RT-PCR Kit II
</P><P>1. Sample:
</P><P>CHO cell genomic DNA
</P><P>Positive: transfected with GECO+NFAT+Neoloxp
</P><P>Negative Control: transfected with GECO+NFAT
</P><P>Wild Type Control: CHO cell genomic DNA
</P><P>2. Target:
</P><P>Neoloxp segment, 18sRNA
</P><P>3. Standard:
</P><P>097&amp;Neoloxp plasmid(length:6635bp)
</P><P>Concentration: 101.2ng/μl
</P><TABLE class="table table-striped"><TBODY><TR><TH>Stock Conc./ng*ul<SUP>-1</SUP></TH><TH> Volume/ul</TH><TH>Dilute Volume/ul
</TH></TR><TR><TD>c(101.2)</TD><TD> 10</TD><TD>90
</TD></TR><TR><TD>c/10</TD><TD> 10</TD><TD>90
</TD></TR><TR><TD>c/100</TD><TD> 10</TD><TD>40
</TD></TR><TR><TD>c/500</TD><TD> 10</TD><TD>40
</TD></TR><TR><TD>..</TD><TD> ..</TD><TD>..
</TD></TR><TR><TD>dilution point8(1.5)</TD><TD> 10</TD><TD>40
</TD></TR></TBODY></TABLE><P>4.
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul</TH><TH>Final Conc. </TH><TH>Negative Control
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 6.4</TD><TD>8.4
</TD></TR><TR><TD>2XSYBR® Master Mix</TD><TD> 10</TD><TD>10
</TD></TR><TR><TD>Forward Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Reverse Primer</TD><TD> 0.8</TD><TD>0.4</TD><TD>0.8
</TD></TR><TR><TD>Template</TD><TD> 2</TD><TD>0
</TD></TR><TR><TD>Total</TD><TD> 20</TD><TD>20
</TD></TR></TBODY></TABLE><P>5. Procedure
</P><P>6. Results
</P><TABLE class="table table-striped"><TBODY><TR><TH>Primer </TH><TH>Ct
</TH></TR><TR><TD>Negative Control-2</TD><TD>32
</TD></TR><TR><TD>Negative Control-3</TD><TD>29
</TD></TR><TR><TD>Negative Control-4</TD><TD>30
</TD></TR><TR><TD>Negative Control-18s</TD><TD>32
</TD></TR></TBODY></TABLE><P>7. Conclusion
</P><P>I.Can’t use ‘TaKaRa-One Step SYBR PrimeScript RT-PCR Kit II’,cause our sample is DNA.
</P><P>II.We forgot to add ‘PrimeScript Enzyme Mix’ at the beginning,which may cause inaccurate results.
</P><P>III.We forgot to change pipette tips during the dilution,which may bring huge deviation.
</P><P>IV.Two-step PCR has a low amplification efficiency in our experiment.To increase the efficiency,we need to use three-step PCR.
</P><H3><SPAN class="mw-headline" id="23_th_2">23 <SUP>th</SUP></SPAN></H3><P><B>qPCR Primer Test</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P>Kit: TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus)
</P><P>1. Sample,Target and Standard are the same with the experiment yesterday.
</P><P>2.
</P><TABLE class="table table-striped"><TBODY><TR><TH>Reagent</TH><TH> Volume/ul
</TH></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD> 6
</TD></TR><TR><TD>SYBR Premix Ex Taq II(2X)</TD><TD> 10
</TD></TR><TR><TD>Forward Primer</TD><TD> 0.8
</TD></TR><TR><TD>Reverse Primer</TD><TD> 0.8
</TD></TR><TR><TD>Template(&lt;100ng)</TD><TD> 2
</TD></TR><TR><TD>ROX Reference Dye(50X)</TD><TD> 0.4
</TD></TR><TR><TD>Total</TD><TD> 20
</TD></TR></TBODY></TABLE><P>3. Procedure
</P>
4. Results
<TABLE class="table table-striped"><TBODY><TR><TH>Primer </TH><TH>Ct
</TH></TR><TR><TD>Negative Control-2</TD><TD>27.8
</TD></TR><TR><TD>Negative Control-3</TD><TD>30.7
</TD></TR><TR><TD>Negative Control-4</TD><TD>32
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Standard </TH><TH>R<SUP>2</SUP></TH><TH>Efficiency
</TH></TR><TR><TD>Primer-2</TD><TD>0.9954</TD><TD>116%
</TD></TR><TR><TD>Primer-3</TD><TD>0.9993</TD><TD>104%
</TD></TR><TR><TD>Primer-4</TD><TD>0.9998</TD><TD>104%
</TD></TR></TBODY></TABLE><P>5. Conclusion
</P><P>Primer 3,4 are similarly efficient,combining with previous experiment result,we choose Primer 3 in the later experiment.
</P><H3><SPAN class="mw-headline" id="24_th_3">24 <SUP>th</SUP></SPAN></H3><P><B>Repeat the mutation experiment on 09.14</B></P><P>1. The targeted mutation segments are 1ng,10ng,100ng respectively.
</P><P>2. During the first PCR procedure,the annealing temperature is set to be 52℃,which is quite low,so it didn’t work.Then we change it to 57℃.
</P><P>3. Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TH>Temperature/℃</TH><TH> Duration/seconds</TH><TH>Number of cycles
</TH></TR><TR><TD>98</TD><TD>10</TD><TD>1
</TD></TR><TR><TD>98</TD><TD> 10</TD></TR><TR><TD>55</TD><TD> 10</TD></TR><TR><TD>72</TD><TD> 60</TD><TD>32
</TD></TR><TR><TD>72</TD><TD> 120</TD><TD>1
</TD></TR><TR><TD>10</TD><TD> ∞</TD></TR></TBODY></TABLE><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A><B>of PCR Product(as Insert)</B></P><TABLE class="table table-striped"><TBODY><TR><TH>Target Mass/ng</TH><TH> 1</TH><TH>10</TH><TH>100
</TH></TR><TR><TD>Conc./ng*ul<SUP>-1</SUP></TD><TD>215.2</TD><TD>208.5</TD><TD>259.5
</TD></TR></TBODY></TABLE><P>Each with volume of 30μl.
</P><P><B>EcoRI,SacI Restriction Enzyme Digestion for 9 hours</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH>Target Mass/ng</TH><TH> 1</TH><TH>10</TH><TH>100
</TH></TR><TR><TD>Conc./ng*ul<SUP>-1</SUP></TD><TD>105</TD><TD>107</TD><TD>113
</TD></TR></TBODY></TABLE><P><B>qPCR Primer Test with Premix and Gel Running</B></P><P>Annealing temperature: 62℃
</P><P>Conclusion:
</P><P>I.We can directly see the band’s brightness differences from the gel running result,which may indicate their plasmid copy number differences.Still,we need to do quantitative PCR to get more precise results.
</P><P>II.It’s strange as well as unreasonable that band appears in wild-type control PCR result,which may indicate that it has been contaminated.
</P><P><B>qPCR: 097-gDNA with Primer3</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P><I>Kit: TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus)</I></P><P>Sample: Positive ①②③⑤⑦⑧⑩
</P><P>Target: Neoloxp
</P><P>Standard: 097&amp;Neoloxp plasmid-③
</P><P>2. Reagent and Procedure are the same with the experiment on 09.23.
</P><P>3. Results
</P><TABLE class="table table-striped"><TBODY><TR><TH>Standard </TH><TH>R<SUP>2</SUP></TH><TH>Efficiency
</TH></TR><TR><TD>Primer-3</TD><TD>0.9998</TD><TD>109%
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Name </TH><TH>Ct
</TH></TR><TR><TD>WT-old</TD><TD>29
</TD></TR><TR><TD>WT-1 new</TD><TD>21.6
</TD></TR><TR><TD>NC-3</TD><TD>32
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Sample </TH><TH>Copy Number
</TH></TR><TR><TD>1</TD><TD>19
</TD></TR><TR><TD>2</TD><TD>8
</TD></TR><TR><TD>3</TD><TD>4
</TD></TR><TR><TD>5</TD><TD>5
</TD></TR><TR><TD>7</TD><TD>12
</TD></TR><TR><TD>8</TD><TD>7
</TD></TR><TR><TD>10</TD><TD>5
</TD></TR></TBODY></TABLE><P>4. Conclusion
</P><P>Wild type control(new) may be contaminated.
</P><H3><SPAN class="mw-headline" id="25_th_2">25 <SUP>th</SUP></SPAN></H3><P>We find that the synthesized TRPC5 has a problem,which may cause replacement during protein translation,so we need to re-construct TRPC5 plasmid.
</P><P><B>Primer Synthesization(send to Invitrogen)</B></P><P>Sequences
</P><P>Forward Primer: atccGAATTCccctccaaatc(we already have,don’t need to synthesize)
</P><P>Reverse Primer: tgc tctaga gtg gcccagctgtactacaag
</P><H3><SPAN class="mw-headline" id="26_th_2">26 <SUP>th</SUP></SPAN></H3><P><B>qPCR Primer Verification with Premix and Sequencing</B></P><P>Sequencing Results:
</P><P>The PCR results are what we need,and the wild-type genomic DNA which were recently extracted are indeed contaminated.
</P><H3><SPAN class="mw-headline" id="27_th">27 <SUP>th</SUP></SPAN></H3><P>Re-construct TRPC5 Plasmid
</P><P><B>TRPC5 PCR with modify-primer</B></P><P>Same procedure as before
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A></P><P><B>XbaI,SacI Restriction Enzyme Digestion for Vector and Insert</B></P><H3><SPAN class="mw-headline" id="28_th_2">28 <SUP>th</SUP></SPAN></H3><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A><B>for Vector</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Cycle_Pure_Kit_-_Centrifugation_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>E.Z.N.A.® Cycle Pure Kit - Centrifugation Protocol</B></A><B>for Insert</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#T4_DNA_Ligase.28M0202.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>T4 DNA Ligase(M0202)</B></A><B>for 20min</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A></P><P>After culture for 10 hours:
</P><P><A href="/File:Media/image192.jpeg" title="File:Media/image192.jpeg">584x279px</A></P><P><B>Pick 8 Single Colony and Culture for Sequencing</B></P><P>Sequencing Result:
</P><P>All correct.
</P><P><B>Culture Colonies for</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Endo-Free_Plasmid_DNA_Mini_Kit_II_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>endofree plasmid purification.</B></A></P><P><B>CHO Cell <I>Genomic DNA Extraction</I></B></P><P>Positive: GECO+NFAT+Neoloxp
</P><P>(11)d1-C1 (12)d1-C8 (13)d1-D12 (14)d1-E3 (15)d1-E4
</P><P>(16)d1-F3 (17)d1-F6 (18)d1-F9 (19)d2-C3 (20)d2-C8
</P><P>(21)d2-D5
</P><H3><SPAN class="mw-headline" id="29_th_3">29 <SUP>th</SUP></SPAN></H3><P><B>PvuI,NcoI Restriction Enzyme Digestion for Verification</B></P><P>Verification result: All correct.
</P><H2><SPAN class="mw-headline" id="Oct.">Oct.</SPAN></H2><H3><SPAN class="mw-headline" id="01_st_3">01 <SUP>st</SUP></SPAN></H3><P>Construct Mutation Library
</P><P>Vector: TRPC5 digested with EcoRI,SacI,AflII
</P><P>Insert: ep-PCR segments,with low&amp;medium&amp;high mutation frequency.
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Gel_Extraction_Kit_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Gel Extraction protocol</B></A><B>of Enzyme-digested Vector</B></P><P><B>Ligation for 1 hour</B></P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Transformation_.26_Competent_cell_recovery" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Transformation</B></A></P><P>5μl ligation product are transformed with 100ul competent cells.
</P><P><B>qPCR: 097-gDNA with Primer3</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#Power_SYBR.C2.AE_Green_PCR_Master_Mix_Kit.28life_technologies.2C4368708.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>Kit: TaKaRa-One Step SYBR PrimeScript RT-PCR Kit I</B></A></P><P>1. Sample: Positive (3),(11)~(21)
</P><P>Negative: wild type cho cell gDNA
</P><P>Target: Neoloxp
</P><P>Standard: 097&amp;Neoloxp plasmid（ Endofree-① ），289ng/μl
</P><P>The first and last dilution point are added with 10ng wild-type gDNA,to see its influence on standard curve.
</P><P>2. Reagent and Procedure are the same with the experiment on 09.23,except that the annealing temperature is set to be 64℃.
</P><P>3. Results
</P><TABLE class="table table-striped"><TBODY><TR><TH>Standard </TH><TH>R<SUP>2</SUP></TH><TH>Efficiency
</TH></TR><TR><TD>Primer-3</TD><TD>0.9998</TD><TD>97%
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Name </TH><TH>Ct
</TH></TR><TR><TD>WT-old</TD><TD>24
</TD></TR><TR><TD>NC-3</TD><TD>29
</TD></TR></TBODY></TABLE><P>
4. Conclusion
</P><P>I. The plasmid concentration differs each time,we need to standardize the method for concentration measurement.
</P><P>II. The primer is not good enough,which may cause inaccurate standard curve and copy number result.
</P><H3><SPAN class="mw-headline" id="02_nd_2">02 <SUP>nd</SUP></SPAN></H3><P><B>Scrape the Plate for</B><A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#E.Z.N.A..C2.AE_Endo-Free_Plasmid_DNA_Mini_Kit_II_Protocol_-_Spin_Protocol" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>endofree plasmid purification.</B></A></P><TABLE class="table table-striped"><TBODY><TR><TH>Colonies </TH><TH>Control
</TH></TR><TR><TD>10<SUP>4</SUP>~10<SUP>5</SUP></TD><TD>12
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Target Mass/ng</TH><TH> 1</TH><TH>10</TH><TH>100</TH><TH>500
</TH></TR><TR><TD>Conc./ng*ul<SUP>-1</SUP></TD><TD>295.7</TD><TD>271.5</TD><TD>167.6</TD><TD>200.4
</TD></TR></TBODY></TABLE><P>Each with volume of 100μl.
</P><P><B>CHO <I>Cell Genomic DNA Extraction</I></B></P><P>Positive: GECO+NFAT+Neoloxp
</P><P>d2-C6 (23)d2-A5 (24)d2-A8
</P><P>d2-A9 (26)d1-D10
Wild type: CHO cell’s gDNA
All the plasmids’ concentrations are re-measured using NanoDrop.
</P><H3><SPAN class="mw-headline" id="03_rd">03 <SUP>rd</SUP></SPAN></H3><P><B>qPCR: 097-gDNA with Primer3</B></P><P>Instrument: ABI StepOne Plus qPCR
</P><P>Kit: TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus)
</P><P>1. Sample: Positive (3),(22)~(26)
</P><P>Negative: wild type cho cell gDNA(newly extracted)
</P><P>Target: Neoloxp
</P><P>Standard: 097&amp;Neoloxp plasmid（ Endofree-① , 307.8ng/μl
</P><P>All the dilution points are added with 10ng wild-type gDNA.
</P><P>2. Reagent and Procedure are the same with the experiment on 09.23,except that the annealing temperature is set to be 64℃.
</P><P>3. Results
</P><TABLE class="table table-striped"><TBODY><TR><TH>Standard </TH><TH>R<SUP>2</SUP></TH><TH>Efficiency
</TH></TR><TR><TD>Primer-3</TD><TD>0.9998</TD><TD>140%
</TD></TR></TBODY></TABLE><TABLE class="table table-striped"><TBODY><TR><TH>Name </TH><TH>Ct
</TH></TR><TR><TD>WT-old</TD><TD>26
</TD></TR><TR><TD>NC-3</TD><TD>30
</TD></TR></TBODY></TABLE><P>4. Conclusion
</P><P>I. The standard curve efficiency is 140%,it may result from the adding of wild type gDNA in the preparation of standard point.
</P><P>II. The Ct of wild type gDNA is 26,which indicates that it has been contaminated.So the copy number results are not precise,we can only compare their differences.
</P></DIV></DIV><FOOTER class="footer bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Made by  from the iGEM team SUSTech_Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>©2016 SUSTech_Shenzhen</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>