<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Proof_Directed_Evolution skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Proof/Directed Evolution</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model">Model </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Impact">Impact</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Activities">Activities</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Team">Members</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Contribution</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/InterLab">Interlab</A></LI><UL class="nav navbar-nav navbar-right hidden-sm hidden-md"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><DIV class="row"><DIV class="col-xs-10"><H1>

Proof of Concept

        </H1><P>

Directed Evolution

        </P></DIV></DIV></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#I._Library_Construction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">I. Library Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#i._Mutation_Frequency"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">i. Mutation Frequency</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#.28i.29._Ideal_Mutation_Frequency"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">(i). Ideal Mutation Frequency</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#.28ii.29._Practical_Mutation_Frequency"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">(ii). Practical Mutation Frequency</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#.28iii.29._Discussion"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">(iii). Discussion</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#ii._Three_Methods_for_Mutation_Library_Construction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">ii. Three Methods for Mutation Library Construction</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#.28i.29._Restriction_Enzyme_Digestion_and_Ligation"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">(i). Restriction Enzyme Digestion and Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#.28ii.29._Whole_Plasmid_Mutagenesis"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">(ii). Whole Plasmid Mutagenesis</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#.28iii.29._Gibson_Assembly"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">(iii). Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#.28iv.29._Result"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">(iv). Result</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#.28v.29._Discussion"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">(v). Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#II._Screening"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">II. Screening</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#i._Principle_:"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">i. Principle :</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#ii._Primer_Design_and_Test"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">ii. Primer Design and Test</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#.28i.29._Primer_Design"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">(i). Primer Design</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#.28ii.29._Normal_PCR_Test"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">(ii). Normal PCR Test</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#.28iii.29._Gel_Extraction_and_Sequencing"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">(iii). Gel Extraction and Sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#.28iv.29._Primer_Efficiency_Test"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">(iv). Primer Efficiency Test</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#iii._Procedure"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">iii. Procedure</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#.28i.29._Overall_process"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">(i). Overall process</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#.28ii.29._Real-time_PCR"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">(ii). Real-time PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#iv._Result"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">iv. Result</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#.28i.29._Amplification_Plot"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">(i). Amplification Plot</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#.28ii.29._Standard_Curve"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">(ii). Standard Curve</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#.28iii.29._Melt_Curve"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">(iii). Melt Curve</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#.28iv.29._Copy_Number"><SPAN class="tocnumber">2.4.4</SPAN><SPAN class="toctext">(iv). Copy Number</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#iv._Discussion"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">iv. Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#III._Directed_Evolution"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">III. Directed Evolution</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV></DIV><DIV class="col-md-9" id="document-body"><H1><SPAN class="mw-headline" id="I._Library_Construction">I. Library Construction</SPAN></H1><H2><SPAN class="mw-headline" id="i._Mutation_Frequency">i. Mutation Frequency</SPAN></H2><H3><SPAN class="mw-headline" id=".28i.29._Ideal_Mutation_Frequency">(i). Ideal Mutation Frequency</SPAN></H3><P>We used the <A href="/Team:SUSTech_Shenzhen/Notebook/Protocol#GeneMorph_II_Random_Mutagenesis_Kit.28Agilent_Technologies.2CCatalog_.23200550.29" title="Team:SUSTech Shenzhen/Notebook/Protocol"><B>GeneMorph II Random Mutagenesis Kit</B></A> to fulfill the random mutation in the ankyrin repeats segment of TRPC5. According to the instruction manual (Agilent Technologies,Catalog #200550), mutation rates of 1–16 mutations per kb can be achieved using a single set of buffer conditions (MgCl2, balanced dNTPs) optimized for high product yield. The desired mutation rate can be controlled simply by varying the initial amount of target DNA in the reaction or the number of amplification cycles performed.
</P><P>Mutation frequency is the product of DNA polymerase error rate and number of duplications. In the GeneMorph II kit, a sufficiently high error rate is achieved through use of Mutazyme II DNA polymerase. A low, medium or high mutation frequency is produced by adjusting the initial target DNA amounts in the amplification reactions. For the same PCR yield, targets amplified from low amounts of target DNA undergo more duplications than targets amplified from high concentrations of DNA. The more times a target is replicated, the more errors accumulate. Therefore, higher mutation frequencies are achieved simply by lowering
input DNA template concentration. Conversely, lower PCR mutation frequencies can be achieved by using higher DNA template concentrations to limit the number of target duplications. Mutation rates can also be decreased by lowering the number of cycles to achieve fewer target duplications. For targets that produce high product yields after 30 cycles, lower mutation rates can be achieved by amplifying lower target amounts for 20–25 cycles. This table below shows the relationship between mutation frequency and initial target quantity.
</P><P>Table 1. Mutation Frequency vs. Initial Target Quantity
</P><H3><SPAN class="mw-headline" id=".28ii.29._Practical_Mutation_Frequency">(ii). Practical Mutation Frequency</SPAN></H3><P>1. Error-prone PCR Program
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Temperature/℃</B></TD><TD><B>Duration</B></TD><TD><B>Cycles</B></TD></TR><TR><TD> 1
</TD><TD> 95
</TD><TD> 2min
</TD><TD> 1
</TD></TR><TR><TD> 2
</TD><TD> 95
</TD><TD> 30s
</TD><TD> 32
</TD></TR><TR><TD> 57
</TD><TD> 30s
</TD></TR><TR><TD> 72
</TD><TD> 1min
</TD></TR><TR><TD> 3
</TD><TD> 72
</TD><TD> 10min
</TD><TD> 1
</TD></TR></TBODY></TABLE><P>Table 2. Error-prone PCR Program
</P><P>2. Sequencing Result
</P><P>To test the mutation frequency practically, we sequenced the mutated TRPC5 with initial <B>target(mainly the ankyrin repeats, with length of 734bp)</B> amount of 500ng, intending to get low mutation frequency product. These figures below show the sequencing result.
</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption">a</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption">b</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption">c</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption">d</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption">e</DIV></DIV></DIV></DIV><P>Figure 1. Sequencing Result of Mutated TRPC5 with Low Mutation Frequency
</P><H3><SPAN class="mw-headline" id=".28iii.29._Discussion">(iii). Discussion</SPAN></H3><P>This table below shows the correspondence between mutation(s) and sequenced sample numbers.
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Mutation(s)</B></TD><TD> 1
</TD><TD> 2
</TD><TD> 0
</TD></TR><TR><TD><B>Sample Numbers</B></TD><TD> 4
</TD><TD> 1
</TD><TD> 14
</TD></TR></TBODY></TABLE><P>Table 3. Practical Mutations and Sample Numbers
</P><P>We’ve sent 20 mutated samples for sequencing, with 5 samples detected to being have mutation(s). Since the mutated fragment was restriction enzyme digested, and the transformation result showed that the digestion may be incomplete (Table 3), the picked single colonies which were sequenced later may not be representative. Still, the data could offer us some information. Basically, the practical mutation frequency with initial target amount of 500ng was similar with the theoretical mutation frequency.
</P><P>To construct a large-scale mutation library with different mutation frequencies in the specific region, we also did error-prone PCR with different initial target amounts to get TRPC5 mutants with low, medium and high mutation rate.
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Colonies</B></TD><TD><B>Control</B></TD></TR><TR><TD> 10688
</TD><TD> 113
</TD></TR></TBODY></TABLE><P>Table 4. Transformation Result of Mutated TRPC5 Ligation Product vs Control
</P><H2><SPAN class="mw-headline" id="ii._Three_Methods_for_Mutation_Library_Construction">ii. Three Methods for Mutation Library Construction</SPAN></H2><H3><SPAN class="mw-headline" id=".28i.29._Restriction_Enzyme_Digestion_and_Ligation">(i). Restriction Enzyme Digestion and Ligation</SPAN></H3><P>Procedure:
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 2. Procedure for R.E.D and Ligation Method</B></DIV></DIV></DIV></DIV><P>( R.E.D means Restriction Enzyme Digestion )
</P><H3><SPAN class="mw-headline" id=".28ii.29._Whole_Plasmid_Mutagenesis">(ii). Whole Plasmid Mutagenesis</SPAN></H3><P>1. Principle:
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 3. Principle of Whole Plasmid Mutagenesis Method</B></DIV></DIV></DIV></DIV><P>2. Procedure:
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 4. Procedure for Whole Plasmid Mutagenesis</B></DIV></DIV></DIV></DIV><P>①We didn’t use GeneMorph II EZClone Domain Mutagenesis Kit mentioned in the figure above, instead we utilized ‘Q5 Hot Start High-Fidelity 2X Master Mix’ (NEB, M0494L) to realize mega-primer PCR. It took us quite a long time to explore the PCR conditions since the annealing temperature and initial template amount was difficult to determine.
</P><P>②Since the PCR products generated from mega-primer couldn’t act as PCR templates due to their nick, the producing mutation library may be rather small. So we utilized T5 exonuclease to digest approximately 30 base pairs on mega-primer(734bp), thus the generated products could act as PCR templates, resulting in exponential amplification. This table below shows the T5 exonuclease digestion results through sequencing.
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>T5 Digestion Time/min</B></TD><TD><B>Digested Base Pairs</B></TD></TR><TR><TD> 5
</TD><TD> 48
</TD></TR><TR><TD> 7.5
</TD><TD> 100
</TD></TR><TR><TD> 10
</TD><TD> 172
</TD></TR></TBODY></TABLE><P>Table 5. T5 Exonuclease Digestion Result (3.5 min)
</P><H3><SPAN class="mw-headline" id=".28iii.29._Gibson_Assembly">(iii). Gibson Assembly</SPAN></H3><P>1. Principle:
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 5. Principle of Gibson Assembly <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP></B></DIV></DIV></DIV></DIV><P>2. Procedure
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 6. Procedure of Gibson Assembly</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id=".28iv.29._Result">(iv). Result</SPAN></H3><P>1. Transformation Result
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">a</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">b</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">c</DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption">d</DIV></DIV></DIV></DIV><P>Figure 7. Transformation Results of Three Methods for Mutation Library Construction
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Method</B></TD><TD><B>Colonies</B></TD><TD><B>Control</B></TD></TR><TR><TD> R.E.D and Ligation
</TD><TD> 1772
</TD><TD> 330
</TD></TR><TR><TD> Whole Plasmid Mutagenesis
</TD><TD> 191
</TD><TD> 0
</TD></TR><TR><TD> Gibson Assembly
</TD><TD> 38
</TD><TD> 7
</TD></TR></TBODY></TABLE><P>Table 6. Transformation Results of Three Methods for Mutation Library Construction
</P><P>The competent cell positive control was transformed with 20pg TRPC5 plasmid. The rest six were all transformed with 5μl products. Transformation operations were all the same.
</P><P>2. Library Calculation
</P><P>Competent Cell Efficiency
</P><P>(1). Formula: <SPAN class="katex-block">\frac{\text{Colonies}}{\text{ng of DNA plated}}\times 1000 \text{ng}/\mu g</SPAN></P><P>(2). Calculation:　237 / 0.02 x 1000=1.185x10<SUP>7</SUP> cfu/μg
</P><P>(3). Actual meaning: 20pg,5640bp TRPC5 plasmid → 3.457x10<SUP>6</SUP> copies
</P><P>3.457x10<SUP>6</SUP> / 237=14586,which means 1 of 14586 plasmids would actually grow on the plate.
</P><P>Ligation Efficiency
</P><P>(1). 62ng,4906bp plasmid → 1.232x10<SUP>10</SUP> copies
</P><P>(2). Efficiency= <SPAN class="katex-block">\frac{(1772-330)\times 20 \div 5}{1.232 \times 10^10 \times \frac{1}{14586}}\times 100\% = 0.68\%</SPAN></P><P>Theoretical Mutation Library
</P><P>(1). R.E.D and Ligation: (1772-330)x14586= 2.1x10<SUP>7</SUP></P><P>(2). Whole Plasmid Mutagenesis: (191-0)x14586= 2.8x10<SUP>6</SUP></P><P>(3). Gibson Assembly: (38-7)x14586= 4.5x10<SUP>5</SUP></P><P>All were calculated based on the transformation results of 5μl products transformed with 100ul competent cell.
</P><H3><SPAN class="mw-headline" id=".28v.29._Discussion">(v). Discussion</SPAN></H3><OL><LI> We could see from the results above that, the theoretical mutation libraries for three methods were (1)&gt;(2)&gt;(3).</LI><LI> Still, we need to consider their effect on CHO-K1 cell if being directly transfected without transformation and plasmid construction. Since the ligation efficiency was 0.68%, we did an experiment comparing the survival rate of CHO-K1 cells after being transfected with linear DNA and circular DNA. After antibiotic screening, we luckily found that the former one were almost dead, while the latter one were good. Thus we could directly transfect the ligation products into cell without transformation, preventing the mutation library from decreasing.</LI></OL><H1><SPAN class="mw-headline" id="II._Screening">II. Screening</SPAN></H1><H2><SPAN class="mw-headline" id="i._Principle_:">i. Principle :</SPAN></H2><P>A real-time polymerase chain reaction is a laboratory technique of molecular biology based on the polymerase chain reaction (PCR). It monitors the amplification of a targeted DNA molecule during the PCR, i.e. in real-time, and not at its end, as in conventional PCR<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>. Thus, we could use real-time PCR to screen the CHO-K1 cells grown from single colony and were transfected with single or low copies of NeoLoxp.
</P><H2><SPAN class="mw-headline" id="ii._Primer_Design_and_Test">ii. Primer Design and Test</SPAN></H2><H3><SPAN class="mw-headline" id=".28i.29._Primer_Design">(i). Primer Design</SPAN></H3><P>The real-time PCR primers (Forward: TTGTCAAGACCGACCTGTCC, Reverse: TTCAGTGACAACGTCGAGCA) were designed by Primer-BLAST, a tool for finding specific primers, followed by BLAST specificity check.
</P><H3><SPAN class="mw-headline" id=".28ii.29._Normal_PCR_Test">(ii). Normal PCR Test</SPAN></H3><P>Since the designed primer has some base pair matches with the wild-type CHO-K1 cell’s genomic DNA, we need to select an appropriate annealing temperature to avoid unnecessary amplification as well as improving primer efficiency. This figure below shows the gel running result of temperature gradient normal PCR using Premix Taq.
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 8. Gel Running Result of Temperature Gradient Normal PCR</B></DIV></DIV></DIV></DIV><P>The result indicates that 62℃ may be the appropriate annealing temperature since wild-type gDNA shows no sign of PCR amplification at this condition, while it seems to still have PCR bands at 61℃.
</P><H3><SPAN class="mw-headline" id=".28iii.29._Gel_Extraction_and_Sequencing">(iii). Gel Extraction and Sequencing</SPAN></H3><P>The original annealing temperature of the designed primer was 60℃. Since we planed to increased it to 62℃ during real-time PCR, we still need to check whether the amplification product was accurate at 62℃. The sequencing result proved us to be correct (Figure 9).
</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption"><B>Figure 9. Sequencing Result of Amplification Product at 62℃</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id=".28iv.29._Primer_Efficiency_Test">(iv). Primer Efficiency Test</SPAN></H3><P>1. Principle:
</P><P>Real-time PCR can be used to calculate the primer efficiency of PCR amplification via Equation 1. In which <SPAN>X_n</SPAN> = PCR product quantity after n cycles, <SPAN>X_0</SPAN> = initial copy number, <SPAN>E</SPAN> = amplification efficiency, <SPAN>n</SPAN> = cycle number. 
</P><P><SPAN class="katex-block">X_n=x_0 \times (1+E)^n</SPAN></P><P>Equation 1
</P><P>Therefore, we can get Equation 2 below.
</P><P><SPAN class="katex-block">C_T=-k\log(X_0)+b</SPAN></P><P>Equation 2
</P><P>C<SUB>T</SUB> is threshold cycle number, then, we can get Equations 3 and 4 from Equations 1 and 2.
</P><P><SPAN class="katex-block">k=\frac{1}{\log(1+E)}</SPAN></P><P>Equation 3
</P><P><SPAN class="katex-block">E=10^{-\frac{1}{k}}-1</SPAN></P><P>Equation 4
</P><P>Thus, we can get the amplification efficiency of primer during real-time PCR from the slope of C<SUB>T</SUB> and it’s corresponding log(X<SUB>0</SUB>).
</P><P>2. Result
</P><TABLE class="table table-striped"><TBODY><TR><TD> Relative Quantity
</TD><TD> Log(Relative Quantity)
</TD><TD> Ct
</TD></TR><TR><TD> 1
</TD><TD> 0
</TD><TD> 15.43882656
</TD></TR><TR><TD> 0.2
</TD><TD> -0.698970004
</TD><TD> 17.44297981
</TD></TR><TR><TD> 0.04
</TD><TD> -1.397940009
</TD><TD> 19.87988091
</TD></TR><TR><TD> 0.008
</TD><TD> -2.096910013
</TD><TD> 22.02274132
</TD></TR><TR><TD> 0.0016
</TD><TD> -2.795880017
</TD><TD> 24.36558533
</TD></TR><TR><TD> 0.00032
</TD><TD> -3.494850022
</TD><TD> 26.80500603
</TD></TR><TR><TD> 0.000064
</TD><TD> -4.193820026
</TD><TD> 28.72155571
</TD></TR><TR><TD> 0.0000128
</TD><TD> -5.89279003
</TD><TD> 29.95066071
</TD></TR><TR><TD> 0.00000256
</TD><TD> -6.591760035
</TD><TD> 31.58642006
</TD></TR></TBODY></TABLE><P>Table 7. Log(Relative Quantity) vs. Ct
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 10. Fitting Standard Curve</B></DIV></DIV></DIV></DIV><P>Thus, efficiency of the primer = <SPAN>=10^{\frac{1}{3.222}}-1</SPAN></P><P>(within the usual margins of error, which is 90%~110% )
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 11. Melt Curve of Designed Primer</B></DIV></DIV></DIV></DIV><P>Since the melt curve has only one peak, the designed primer is quite specific to the transfected CHO-K1 cell’s genomic DNA.
</P><H2><SPAN class="mw-headline" id="iii._Procedure">iii. Procedure</SPAN></H2><H3><SPAN class="mw-headline" id=".28i.29._Overall_process">(i). Overall process</SPAN></H3><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 12. Overall Process of CHO-K1 Cells Screening</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id=".28ii.29._Real-time_PCR">(ii). Real-time PCR</SPAN></H3><P>1. Instrument: ABI StepOne Plus qPCR
</P><P>Kit: TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus)
</P><P>2. Solution Setups and PCR Procedure
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume/μl</B></TD><TD><B>Final conc/μ</B>M
</TD></TR><TR><TD> SYBR Premix Ex Taq II(2×)
</TD><TD> 10
</TD><TD> 1×
</TD></TR><TR><TD> Forward Primer(10 μM)
</TD><TD> 0.8
</TD><TD> 0.4
</TD></TR><TR><TD> Reverse Primer(10 μM)
</TD><TD> 0.8
</TD><TD> 0.4
</TD></TR><TR><TD> ROX Reference Dye(50×)
</TD><TD> 0.4
</TD><TD> 1×
</TD></TR><TR><TD> Template(&lt;100ng)
</TD><TD> 2
</TD></TR><TR><TD> ddH<SUB>2</SUB>O
</TD><TD> 6
</TD></TR><TR><TD> Total
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>Table 8. Solution Setups
</P><TABLE class="table table-striped"><TBODY><TR><TD><B>Temperature/℃</B></TD><TD><B>Duration</B></TD><TD><B>Cycles</B></TD><TD><B>Stage</B></TD></TR><TR><TD> 95
</TD><TD> 30s
</TD><TD> 1
</TD><TD> Holding
</TD></TR><TR><TD> 95
</TD><TD> 5s
</TD><TD> 40
</TD><TD> Cycling
</TD></TR><TR><TD> 62
</TD><TD> 30s
</TD></TR><TR><TD> 72
</TD><TD> 30s
</TD></TR><TR><TD> 95
</TD><TD> 15s
</TD><TD> 1
</TD><TD> Melt Curve
</TD></TR><TR><TD> 62
</TD><TD> 1min
</TD></TR><TR><TD> 95
</TD><TD> 15s
</TD></TR></TBODY></TABLE><P>Table 9. PCR Procedure
</P><H2><SPAN class="mw-headline" id="iv._Result">iv. Result</SPAN></H2><H3><SPAN class="mw-headline" id=".28i.29._Amplification_Plot">(i). Amplification Plot</SPAN></H3><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 13. Amplification Plot for Standard</B></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 14. Amplification Plot for Sample</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id=".28ii.29._Standard_Curve">(ii). Standard Curve</SPAN></H3><TABLE class="table table-striped"><TBODY><TR><TD><B>Relative Quantity</B></TD><TD><B>log(Relative Quantity)</B></TD><TD><B>Ct</B></TD></TR><TR><TD> 1
</TD><TD> 0
</TD><TD> 15.01047802
</TD></TR><TR><TD> 0.2
</TD><TD> -0.698970004
</TD><TD> 16.94664574
</TD></TR><TR><TD> 0.04
</TD><TD> -1.397940009
</TD><TD> 19.24198341
</TD></TR><TR><TD> 0.00032
</TD><TD> -3.494850022
</TD><TD> 25.79216194
</TD></TR><TR><TD> 0.000064
</TD><TD> -4.193820026
</TD><TD> 27.9509449
</TD></TR></TBODY></TABLE><P>Table 10. Log(Relative Quantity) vs. Ct
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 15. Fitted Standard Curve</B></DIV></DIV></DIV></DIV><P>(within the usual margins of error, which is 90%~110%)
</P><H3><SPAN class="mw-headline" id=".28iii.29._Melt_Curve">(iii). Melt Curve</SPAN></H3><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Figure 16. Melt Curve of NeoLoxp Transfected CHO-K1 Cell’s gDNA</B></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id=".28iv.29._Copy_Number">(iv). Copy Number</SPAN></H3><TABLE class="table table-striped"><TBODY><TR><TD><B>Mass/ng</B></TD><TD><B>Ct</B></TD><TD><B>Relative Quantity</B></TD><TD><B>10^-3 fmole</B></TD><TD><B>Copy Number</B></TD></TR><TR><TD> 78.8075
</TD><TD> 23.2236
</TD><TD> 0.002110101
</TD><TD> 0.0531425
</TD><TD> 4.054
</TD></TR><TR><TD> 40.7625
</TD><TD> 24.1791
</TD><TD> 0.001040142
</TD><TD> 0.0274875
</TD><TD> 3.863
</TD></TR><TR><TD> 19.0225
</TD><TD> 25.2316
</TD><TD> 0.000477161
</TD><TD> 0.0128275
</TD><TD> 3.797
</TD></TR><TR><TD><B>Average</B></TD><TD> 3.905
</TD></TR></TBODY></TABLE><P>Table 11. NeoLoxp Copy Number of CHO-K1 Cell
</P><P>Relative Quantity = <SPAN>10^{\frac{C_t\times 14.902}{3.1101}}</SPAN></P><P>Copy Number :
</P><P>Plasmid used in standard curve: 0.1021 fmol, Relative Quantity=1
</P><P><SPAN class="katex-block">c=\frac{\text{fmole}\cdot 10^3}{0.1201}\cdot c = \frac{\text{Quantity}}{1}</SPAN><SPAN class="katex-block">c=\frac{\text{Quantity}\cdot 102.1}{\text{fmole}}</SPAN></P><P>(c: copy number)
</P><H2><SPAN class="mw-headline" id="iv._Discussion">iv. Discussion</SPAN></H2><P>The efficiency of PiggyBac transposon system ranged from about 2% for one transposon to 0.5% for five transposons <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> So it’s quite possible for us to screen out the cells with single or low copies of NeoLoxp.
</P><P>In practical experiments, the selected CHO-K1 cell transfected with NeoLoxp by then had NeoLoxp copy number of around 4, which was not good enough. The reason may be that the designed primer was not so satisfying. Also, the contamination during real-time PCR was hard to avoid. To get cells with single copy of NeoLoxp, we still need to design better primers as well as paying attention to the experiment operations.
</P><H1><SPAN class="mw-headline" id="III._Directed_Evolution">III. Directed Evolution</SPAN></H1><P>After we’ve selected the CHO-K1 cell transfected with single copy of NeoLoxp, we could transfect mutated TRPC5 into the cell’s genomic DNA using Cre-Loxp recombination system. Then we would use fluorescence activated cell sorting(FACS) to select the cells with high intensity of GFP after long term stimulation. To realize high-throughput screening, we designed a device as shown in the figure below. Cells are seeded at the larger area side of the bottle, and the bottle is fixed above vortex through a holder and foam. Then we are going to place this device in cell culture incubator with vortex running for 20 hours, to fulfill long term mechanical force stimulation.   
</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3"><DIV class="thumbnail"><DIV class="caption"><B>Figure 17. Self-designed Device for Long Term Stimulation</B></DIV></DIV></DIV></DIV><P>After the first round of mutation and screening, we still need to continue mutation based on the previous result. We will extract genomic DNA of the screened-out CHO-K1 cell, performing polymerase chain reaction with the mutation primer and sending the PCR product for sequencing. Through alignment of the mutated TRPC5 with normal one, we could clearly see which base site has been mutated. Combing with the TRPC5 structure we’ve known, we may explain why the specific site mutation could induce channel’s greater sensitivity to mechanical force. Then we could specify the mutation region every time, repeating then huge-library mutation and high-throughout screening procedure, thus realizing directed evolution.
</P><P>Ideally, we would like to screen out then mutated TRPC5 with high sensitivity to mechanical force. As for the audiogenetics, we hope to screen out cells which are able to response to specific sound frequency, achieving precise and orthogonal audio regulation.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Vornholt, T. (n.d.). Illustration of Gibson assembly. Retrieved April 17,2015, from <A rel="nofollow" class="external free" href="https://commons.wikimedia.org/wiki/File:Gibson_assembly_overview.png">https://commons.wikimedia.org/wiki/File:Gibson_assembly_overview.png</A></SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text"> Reviews, C. (2016). Genetics and Genomics in Medicine. United states: Cram101 Textbook Reviews. </SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Lu, X. and W. Huang, <I>PiggyBac mediated multiplex gene transfer in mouse embryonic stem cell.</I> PLoS One, 2014. <B>9</B>(12): p. e115072.</SPAN></LI></OL></DIV></DIV><FOOTER class="footer bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Made by  from the iGEM team SUSTech_Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>©2016 SUSTech_Shenzhen</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>