<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model">Model </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Impact">Impact</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Activities">Activities</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Team">Members</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Contribution</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/InterLab">Interlab</A></LI><UL class="nav navbar-nav navbar-right hidden-sm hidden-md"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><DIV class="row"><DIV class="col-xs-10"><H1>

Basic Part

        </H1><P>

Contribution

        </P></DIV></DIV></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#1.TRPC5_collection"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">1.TRPC5 collection</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#2._mR-GECO"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">2. mR-GECO</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#3._pTRE-TetON3G"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">3. pTRE-TetON3G</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#4._Anti-antibiotics_gene"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">4. Anti-antibiotics gene</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#5._Mutated_sfGFP"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">5. Mutated sfGFP</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV></DIV><DIV class="col-md-9" id="document-body"><P>This year we constructed 15 basic Parts, including:
</P><UL><LI> The core elements used in our project,TRPC5 collection and R-GECO;</LI><LI> Eukaryotic regulation system, TetOn system;</LI><LI> Antibiotics-resistant genes used for selection during our cell experiments;</LI><LI> Coding sequence of mutated sfGFP.</LI></UL><P>Actually, compared with the number of BioBricks that can be used in bacteria, few Parts is able to be applied in eukaryotic cells. Our project used mammalian cell line to express our core protein, the sound-sensitive channel, TRPC5, and calcium indicator, R-GECO. In addition, our antibiotics Parts and TetOn system are commonly used in cell experiments for selection and regulation.
</P><TABLE class="table table-striped"><TBODY><TR><TH><B>Category</B></TH><TH><B>Name'</B></TH><TH><B>Description</B></TH><TH><B>Length</B></TH></TR><TR><TD> TRPC5
</TD><TD>&lt;partinfo&gt;BBa_K1943021&lt;/partinfo&gt;</TD><TD> TRPC5
</TD><TD> 2928bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943022&lt;/partinfo&gt;</TD><TD> TRPC5-7
</TD><TD> 2928bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943023&lt;/partinfo&gt;</TD><TD> TRPC5-12
</TD><TD> 2928bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943024&lt;/partinfo&gt;</TD><TD> TRPC5-15
</TD><TD> 2928bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943025&lt;/partinfo&gt;</TD><TD> TRPC5-18
</TD><TD> 2928bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943026&lt;/partinfo&gt;</TD><TD> TRPC5-20
</TD><TD> 2928bp
</TD></TR><TR><TD> R-GECO
</TD><TD>&lt;partinfo&gt;BBa_K1943020&lt;/partinfo&gt;</TD><TD> mR-GECO CDS
</TD><TD> 1257bp
</TD></TR><TR><TD>  TetOn
</TD><TD>&lt;partinfo&gt;BBa_K1943019&lt;/partinfo&gt;</TD><TD> SV40 promoter + puro anti-antibiotics gene+ 2A + TetOn 3G promoter
</TD><TD> 1774bp
</TD></TR><TR><TD> Antibiotics resistant gene
</TD><TD>&lt;partinfo&gt;BBa_K1943013&lt;/partinfo&gt;</TD><TD> Puro CDS
</TD><TD> 597bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943014&lt;/partinfo&gt;</TD><TD> Bleo CDS
</TD><TD> 369bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943015&lt;/partinfo&gt;</TD><TD> Bla CDS
</TD><TD> 402bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943016&lt;/partinfo&gt;</TD><TD> Bla+T2A
</TD><TD> 456bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943017&lt;/partinfo&gt;</TD><TD> Puro+T2A
</TD><TD> 651bp
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K1943018&lt;/partinfo&gt;</TD><TD> T2A+Bleo
</TD><TD> 429bp
</TD></TR><TR><TD> sfGFP
</TD><TD>&lt;partinfo&gt;BBa_K1943012&lt;/partinfo&gt;</TD><TD> msfGFP CDS
</TD><TD> 717bp
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="1.TRPC5_collection">1.TRPC5 collection</SPAN></H2><P>Transient receptor potential channel 5 (TRPC5) is a calcium ion permeable channel that can be activated by mechanic force.<SUP id="cite_ref-Zholos_1-0" class="reference"><A href="#cite_note-Zholos-1">[1]</A></SUP>  In our study, audiogenetics is defined as a method of gene regulation by applying sound precisely and safely and the external sound signal is transduced through TRPC5.
In order to get mutated TRPC5 with high sensitivity to mechanical stress, we need to find out the specific fragment of TRPC5 which is related to stress sensing. According to the structure of TRPC5, we finally chose the ankyrin repeats region as the mutated region.<SUP id="cite_ref-Beck_2-0" class="reference"><A href="#cite_note-Beck-2">[2]</A></SUP>
We used GeneMorph II Random Mutagenesis Kit (Provided by Agilent Technologies Company) to make a library of mutated ankyrin repeats by error-prone PCR. We submitted 5 TRPC5 from our library and we used site-directed mutation to mutate EcoRI and PstI restriction sites in the coding sequence based on codon degeneracy.
</P><H2><SPAN class="mw-headline" id="2._mR-GECO">2. mR-GECO</SPAN></H2><P>GECO family is a series of fluorescent proteins artificially engineered to response to changes in the concentration of calcium ion Ca2+.<SUP id="cite_ref-zhao_3-0" class="reference"><A href="#cite_note-zhao-3">[3]</A></SUP>
They can help to visualize intracellular signaling activity by the intensity. According to Zhaoâ€™s article, there is optimally 110-folds of emission change.
By recombining the GCaMPs and GFP, firstly they got the G-GECO1 (green). The indicator we used is R-GECO1 derived from mApple and G-GECO1.1. The results of directed evolution also included GEM-GECO and Blue GECO.(Fig. 1). <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig. 1 The Mechanism of Calcium Indicator</B></DIV></DIV><P>These various indicators are helpful in live cell image, and improve Ca2+ imagining in neurons and muscles as well. In our experiment, R-GECO1 was used to detect cell response to mechanical vibration.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig. 2 CHO-K1 cell transfected with plasmids expressing R-GECO without stimulation.</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig. 3 CHO-K1 cell transfected with plasmids expressing R-GECO under 25mM KCl stimulation.</B></DIV></DIV><P>In 2012, an iGEM team submitted its sequence (Part:&lt;partinfo&gt;BBa_K881000&lt;/partinfo&gt;). But, unfortunately, it did not meet the RFC10 standard because of the restriction sites in its original sequence. For the convenience, we mutated these regions to remove the repeated cloning sites without changing the amino acid sequence of the protein expressed.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig. 4 Original GECO sequence contains 2 Pstâ… </B></DIV></DIV><H2><SPAN class="mw-headline" id="3._pTRE-TetON3G">3. pTRE-TetON3G</SPAN></H2><P>We submitted an inducible expression systems, by which we regulated the expression of downstream genes in CHO-K1 cell line. 
TRE-3G promoter is a tetracycline-responsive promoter. The expression of downstream gene can be induced by the addition of doxycycline (DOX). It has been submitted by our predecessor, SUSTC-Shenzhen in 2014 (see the BioBrick at <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K1431301">http://parts.igem.org/Part:BBa_K1431301</A> ) 
They combined these 2 parts into one plasmid. It is useful to improve the success rate of transfection.  Considering the difficulty to cut the whole fragment as a submitted part, they only constructed the assembly with a part of the regulation system.
</P><P>The other part of the system includes a promoter SV40 and the tetracycline (Tet)-regulated transactivator Tet-On 3G. <SUP id="cite_ref-Zhou_5-0" class="reference"><A href="#cite_note-Zhou-5">[5]</A></SUP><SUP id="cite_ref-Low_6-0" class="reference"><A href="#cite_note-Low-6">[6]</A></SUP>  This year we submitted this part connected with 2A and pure gene, the insertion can be confirmed by antibiotic selection, using 3Î¼g/mL puromycin.
</P><P>The use of this part will be wide. Following teams can insert the any target gene after promoter TRE 3G to make it under the control of doxycycline. The regulate part (SV40 to TetON 3G) and the response part (pTRE and target gene) could be constructed into 2 separated plasmid or the same plasmid as we did. 
</P><H2><SPAN class="mw-headline" id="4._Anti-antibiotics_gene">4. Anti-antibiotics gene</SPAN></H2><P>This year we use mammalian cells as carriers. So in our plasmids, there are several kinds of anti-antibiotics genes which are used for eukaryotic cell screening. Some of them are very useful but no team has submitted related Parts before. We kindly hope these resistance gene can help the following team researching on mammalian cell.
</P><UL><LI> The gene bla express Blasticidin S deaminase. This gene is from Asepergillus terreus. It can inactivate blasticidin S HCl, the nucleoside antibiotic isolated from Streptomyces griseochromogenes which inhibits protein synthesis in both prokaryotic and eukaryotic cells. We commonly use 10Î¼g/mL in cell line selection.   </LI><LI> The gene zeo is the resistance gene to antibiotic phleomycin D1, a kind of bleomycin. It is primarily used in selection of eukaryotic cell lines.<SUP id="cite_ref-Benko_7-0" class="reference"><A href="#cite_note-Benko-7">[7]</A></SUP>  In our project we use the concentration of 400Î¼g/mL.</LI><LI> The gene puro is corresponding to the antibiotic puromycin, which is an antibiotic that is a protein synthesis inhibitor by inhibiting translation. Puromycin is selective for either prokaryotes or eukaryotes, but poorly active for E.coli. We usually use 3Î¼g/mL in cell culture.</LI></UL><P>Usually, we will not use anti-antibiotics genes alone. We often use them together with some core protein such as GECO. However, it is monocistronic in the eukaryotic cells. There is a DNA sequence called 2A which connects two coding sequence and then both two call be expressed by the same promoter. We design primers and do PCR of this anti-antibiotics genes coding sequence together with 2A sequence and ligate it to pSB1C3 backbone. These promoter can be directly linked to protein coding sequence if other groups need.
</P><H2><SPAN class="mw-headline" id="5._Mutated_sfGFP">5. Mutated sfGFP</SPAN></H2><P>In our lab, we usually use sfGFP as a reporter for molecular experiment. Some of our backbones contain recognition site, KpnI, a commonly used restriction enzyme. We need a single KpnI site, while sfGFP coding sequence contains a KpnI site. So we do site-directed mutagenesis of it. And then we design primer and do PCR of this msfGFP coding sequence and ligate it to pSB1C3 backbone.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-Zholos-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Zholos_1-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>Zholos AV., TRPC5., Handb Exp Pharmacol., Retrieved from <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/pubmed/24756705">https://www.ncbi.nlm.nih.gov/pubmed/24756705</A>. </CITE></SPAN></LI><LI id="cite_note-Beck-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Beck_2-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>Beck, A., et al., Conserved gating elements in TRPC4 and TRPC5 channels., J Biol Chem., Retrieved from <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/pubmed/23677990">https://www.ncbi.nlm.nih.gov/pubmed/23677990</A>. </CITE></SPAN></LI><LI id="cite_note-zhao-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-zhao_3-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>Zhao Y, A. et al., An Expanded Palette of Genetically Encoded Ca2+ Indicators., Science., Retrieved from <A rel="nofollow" class="external free" href="http://science.sciencemag.org/content/333/6051/1888">http://science.sciencemag.org/content/333/6051/1888</A>. </CITE></SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">â†‘</A></SPAN><SPAN class="reference-text"> Zhao, Y., et al., <I>An expanded palette of genetically encoded Ca(2)(+) indicators.</I> Science, 2011. <B>333</B>(6051): p. 1888-91.</SPAN></LI><LI id="cite_note-Zhou-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Zhou_5-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>Zhou, X. et al., Optimization of the Tet-On system for regulated gene expression through viral evolution., Gene Ther., Retrieved from <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/pubmed/16724096">https://www.ncbi.nlm.nih.gov/pubmed/16724096</A>. </CITE></SPAN></LI><LI id="cite_note-Low-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Low_6-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>LÃ¶w, R. et al., Improved Tet-responsive promoters with minimized background expression., BMC Biotechnology, Retrieved from <A rel="nofollow" class="external free" href="http://bmcbiotechnol.biomedcentral.com/articles/10.1186/1472-6750-10-81">http://bmcbiotechnol.biomedcentral.com/articles/10.1186/1472-6750-10-81</A>. </CITE></SPAN></LI><LI id="cite_note-Benko-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Benko_7-0">â†‘</A></SPAN><SPAN class="reference-text"><CITE>Benko Z., et al., Zeocin for selection of bleMX6 resistance in fission yeast., Biotechniques., Retrieved from <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/pubmed/21781055">https://www.ncbi.nlm.nih.gov/pubmed/21781055</A>. </CITE></SPAN></LI></OL></DIV></DIV><FOOTER class="footer bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Made by  from the iGEM team SUSTech_Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>Â©2016 SUSTech_Shenzhen</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>