Team:SUSTech Shenzhen/Proof/Directed Evolution
Team SUSTC-Shenzhen
Toggle navigation
Home
Project
Description
Design
Proof of Concept
Demonstrate
Notebook
Model
Measurement
Hardware
Model
Human Practices
Overview
Impact
Activities
Team
Members
Collaborations
Safety
Attributions
Contribution
Parts
Basic Parts
Composite Parts
Part Collection
Interlab
Search
Log in
Proof of Concept
Directed Evolution
Contents
1 I. Library Construction
1.1 i. Mutation Frequency
1.1.1 (i). Ideal Mutation Frequency
1.1.2 (ii). Practical Mutation Frequency
1.1.3 (iii). Discussion
1.2 ii. Three Methods for Mutation Library Construction
1.2.1 (i). Restriction Enzyme Digestion and Ligation
1.2.2 (ii). Whole Plasmid Mutagenesis
1.2.3 (iii). Gibson Assembly
1.2.4 (iv). Result
1.2.5 (v). Discussion
2 II. Screening
2.1 i. Principle :
2.2 ii. Primer Design and Test
2.2.1 (i). Primer Design
2.2.2 (ii). Normal PCR Test
2.2.3 (iii). Gel Extraction and Sequencing
2.2.4 (iv). Primer Efficiency Test
2.3 iii. Procedure
2.3.1 (i). Overall process
2.3.2 (ii). Real-time PCR
2.4 iv. Result
2.4.1 (i). Amplification Plot
2.4.2 (ii). Standard Curve
2.4.3 (iii). Melt Curve
2.4.4 (iv). Copy Number
2.5 iv. Discussion
3 III. Directed Evolution
4 References
I. Library Construction
i. Mutation Frequency
(i). Ideal Mutation Frequency
We used the GeneMorph II Random Mutagenesis Kit to fulfill the random mutation in the ankyrin repeats segment of TRPC5. According to the instruction manual (Agilent Technologies,Catalog #200550), mutation rates of 1–16 mutations per kb can be achieved using a single set of buffer conditions (MgCl2, balanced dNTPs) optimized for high product yield. The desired mutation rate can be controlled simply by varying the initial amount of target DNA in the reaction or the number of amplification cycles performed.
Mutation frequency is the product of DNA polymerase error rate and number of duplications. In the GeneMorph II kit, a sufficiently high error rate is achieved through use of Mutazyme II DNA polymerase. A low, medium or high mutation frequency is produced by adjusting the initial target DNA amounts in the amplification reactions. For the same PCR yield, targets amplified from low amounts of target DNA undergo more duplications than targets amplified from high concentrations of DNA. The more times a target is replicated, the more errors accumulate. Therefore, higher mutation frequencies are achieved simply by lowering
input DNA template concentration. Conversely, lower PCR mutation frequencies can be achieved by using higher DNA template concentrations to limit the number of target duplications. Mutation rates can also be decreased by lowering the number of cycles to achieve fewer target duplications. For targets that produce high product yields after 30 cycles, lower mutation rates can be achieved by amplifying lower target amounts for 20–25 cycles. This table below shows the relationship between mutation frequency and initial target quantity.
Table 1. Mutation Frequency vs. Initial Target Quantity
(ii). Practical Mutation Frequency
1. Error-prone PCR Program
Segment
Temperature/℃
Duration
Cycles
1
95
2min
1
2
95
30s
32
57
30s
72
1min
3
72
10min
1
Table 2. Error-prone PCR Program
2. Sequencing Result
To test the mutation frequency practically, we sequenced the mutated TRPC5 with initial target(mainly the ankyrin repeats, with length of 734bp) amount of 500ng, intending to get low mutation frequency product. These figures below show the sequencing result.
a
b
c
d
e
Figure 1. Sequencing Result of Mutated TRPC5 with Low Mutation Frequency
(iii). Discussion
This table below shows the correspondence between mutation(s) and sequenced sample numbers.
Mutation(s)
1
2
0
Sample Numbers
4
1
14
Table 3. Practical Mutations and Sample Numbers
We’ve sent 20 mutated samples for sequencing, with 5 samples detected to being have mutation(s). Since the mutated fragment was restriction enzyme digested, and the transformation result showed that the digestion may be incomplete (Table 3), the picked single colonies which were sequenced later may not be representative. Still, the data could offer us some information. Basically, the practical mutation frequency with initial target amount of 500ng was similar with the theoretical mutation frequency.
To construct a large-scale mutation library with different mutation frequencies in the specific region, we also did error-prone PCR with different initial target amounts to get TRPC5 mutants with low, medium and high mutation rate.
Colonies
Control
10688
113
Table 4. Transformation Result of Mutated TRPC5 Ligation Product vs Control
ii. Three Methods for Mutation Library Construction
(i). Restriction Enzyme Digestion and Ligation
Procedure:
Figure 2. Procedure for R.E.D and Ligation Method
( R.E.D means Restriction Enzyme Digestion )
(ii). Whole Plasmid Mutagenesis
1. Principle:
Figure 3. Principle of Whole Plasmid Mutagenesis Method
2. Procedure:
Figure 4. Procedure for Whole Plasmid Mutagenesis
①We didn’t use GeneMorph II EZClone Domain Mutagenesis Kit mentioned in the figure above, instead we utilized ‘Q5 Hot Start High-Fidelity 2X Master Mix’ (NEB, M0494L) to realize mega-primer PCR. It took us quite a long time to explore the PCR conditions since the annealing temperature and initial template amount was difficult to determine.
②Since the PCR products generated from mega-primer couldn’t act as PCR templates due to their nick, the producing mutation library may be rather small. So we utilized T5 exonuclease to digest approximately 30 base pairs on mega-primer(734bp), thus the generated products could act as PCR templates, resulting in exponential amplification. This table below shows the T5 exonuclease digestion results through sequencing.
T5 Digestion Time/min
Digested Base Pairs
5
48
7.5
100
10
172
Table 5. T5 Exonuclease Digestion Result (3.5 min)
(iii). Gibson Assembly
1. Principle:
Figure 5. Principle of Gibson Assembly [1]
2. Procedure
Figure 6. Procedure of Gibson Assembly
(iv). Result
1. Transformation Result
a
b
c
d
Figure 7. Transformation Results of Three Methods for Mutation Library Construction
Method
Colonies
Control
R.E.D and Ligation
1772
330
Whole Plasmid Mutagenesis
191
0
Gibson Assembly
38
7
Table 6. Transformation Results of Three Methods for Mutation Library Construction
The competent cell positive control was transformed with 20pg TRPC5 plasmid. The rest six were all transformed with 5μl products. Transformation operations were all the same.
2. Library Calculation
Competent Cell Efficiency
(1). Formula: \frac{\text{Colonies}}{\text{ng of DNA plated}}\times 1000 \text{ng}/\mu g
(2). Calculation: 237 / 0.02 x 1000=1.185x107 cfu/μg
(3). Actual meaning: 20pg,5640bp TRPC5 plasmid → 3.457x106 copies
3.457x106 / 237=14586,which means 1 of 14586 plasmids would actually grow on the plate.
Ligation Efficiency
(1). 62ng,4906bp plasmid → 1.232x1010 copies
(2). Efficiency= \frac{(1772-330)\times 20 \div 5}{1.232 \times 10^10 \times \frac{1}{14586}}\times 100\% = 0.68\%
Theoretical Mutation Library
(1). R.E.D and Ligation: (1772-330)x14586= 2.1x107
(2). Whole Plasmid Mutagenesis: (191-0)x14586= 2.8x106
(3). Gibson Assembly: (38-7)x14586= 4.5x105
All were calculated based on the transformation results of 5μl products transformed with 100ul competent cell.
(v). Discussion
We could see from the results above that, the theoretical mutation libraries for three methods were (1)>(2)>(3).
Still, we need to consider their effect on CHO-K1 cell if being directly transfected without transformation and plasmid construction. Since the ligation efficiency was 0.68%, we did an experiment comparing the survival rate of CHO-K1 cells after being transfected with linear DNA and circular DNA. After antibiotic screening, we luckily found that the former one were almost dead, while the latter one were good. Thus we could directly transfect the ligation products into cell without transformation, preventing the mutation library from decreasing.
II. Screening
i. Principle :
A real-time polymerase chain reaction is a laboratory technique of molecular biology based on the polymerase chain reaction (PCR). It monitors the amplification of a targeted DNA molecule during the PCR, i.e. in real-time, and not at its end, as in conventional PCR [2] . Thus, we could use real-time PCR to screen the CHO-K1 cells grown from single colony and were transfected with single or low copies of NeoLoxp.
ii. Primer Design and Test
(i). Primer Design
The real-time PCR primers (Forward: TTGTCAAGACCGACCTGTCC, Reverse: TTCAGTGACAACGTCGAGCA) were designed by Primer-BLAST, a tool for finding specific primers, followed by BLAST specificity check.
(ii). Normal PCR Test
Since the designed primer has some base pair matches with the wild-type CHO-K1 cell’s genomic DNA, we need to select an appropriate annealing temperature to avoid unnecessary amplification as well as improving primer efficiency. This figure below shows the gel running result of temperature gradient normal PCR using Premix Taq.
Figure 8. Gel Running Result of Temperature Gradient Normal PCR
The result indicates that 62℃ may be the appropriate annealing temperature since wild-type gDNA shows no sign of PCR amplification at this condition, while it seems to still have PCR bands at 61℃.
(iii). Gel Extraction and Sequencing
The original annealing temperature of the designed primer was 60℃. Since we planed to increased it to 62℃ during real-time PCR, we still need to check whether the amplification product was accurate at 62℃. The sequencing result proved us to be correct (Figure 9).
Figure 9. Sequencing Result of Amplification Product at 62℃
(iv). Primer Efficiency Test
1. Principle:
Real-time PCR can be used to calculate the primer efficiency of PCR amplification via Equation 1. In which X_n
= PCR product quantity after n cycles, X_0
= initial copy number, E
= amplification efficiency, n
= cycle number.
X_n=x_0 \times (1+E)^n
Equation 1
Therefore, we can get Equation 2 below.
C_T=-k\log(X_0)+b
Equation 2
CT is threshold cycle number, then, we can get Equations 3 and 4 from Equations 1 and 2.
k=\frac{1}{\log(1+E)}
Equation 3
E=10^{-\frac{1}{k}}-1
Equation 4
Thus, we can get the amplification efficiency of primer during real-time PCR from the slope of CT and it’s corresponding log(X0).
2. Result
Relative Quantity
Log(Relative Quantity)
Ct
1
0
15.43882656
0.2
-0.698970004
17.44297981
0.04
-1.397940009
19.87988091
0.008
-2.096910013
22.02274132
0.0016
-2.795880017
24.36558533
0.00032
-3.494850022
26.80500603
0.000064
-4.193820026
28.72155571
0.0000128
-5.89279003
29.95066071
0.00000256
-6.591760035
31.58642006
Table 7. Log(Relative Quantity) vs. Ct
Figure 10. Fitting Standard Curve
Thus, efficiency of the primer = =10^{\frac{1}{3.222}}-1
(within the usual margins of error, which is 90%~110% )
Figure 11. Melt Curve of Designed Primer
Since the melt curve has only one peak, the designed primer is quite specific to the transfected CHO-K1 cell’s genomic DNA.
iii. Procedure
(i). Overall process
Figure 12. Overall Process of CHO-K1 Cells Screening
(ii). Real-time PCR
1. Instrument: ABI StepOne Plus qPCR
Kit: TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus)
2. Solution Setups and PCR Procedure
Reagent
Volume/μl
Final conc/μM
SYBR Premix Ex Taq II(2×)
10
1×
Forward Primer(10 μM)
0.8
0.4
Reverse Primer(10 μM)
0.8
0.4
ROX Reference Dye(50×)
0.4
1×
Template(<100ng)
2
ddH2O
6
Total
20
Table 8. Solution Setups
Temperature/℃
Duration
Cycles
Stage
95
30s
1
Holding
95
5s
40
Cycling
62
30s
72
30s
95
15s
1
Melt Curve
62
1min
95
15s
Table 9. PCR Procedure
iv. Result
(i). Amplification Plot
Figure 13. Amplification Plot for Standard
Figure 14. Amplification Plot for Sample
(ii). Standard Curve
Relative Quantity
log(Relative Quantity)
Ct
1
0
15.01047802
0.2
-0.698970004
16.94664574
0.04
-1.397940009
19.24198341
0.00032
-3.494850022
25.79216194
0.000064
-4.193820026
27.9509449
Table 10. Log(Relative Quantity) vs. Ct
Figure 15. Fitted Standard Curve
(within the usual margins of error, which is 90%~110%)
(iii). Melt Curve
Figure 16. Melt Curve of NeoLoxp Transfected CHO-K1 Cell’s gDNA
(iv). Copy Number
Mass/ng
Ct
Relative Quantity
10^-3 fmole
Copy Number
78.8075
23.2236
0.002110101
0.0531425
4.054
40.7625
24.1791
0.001040142
0.0274875
3.863
19.0225
25.2316
0.000477161
0.0128275
3.797
Average
3.905
Table 11. NeoLoxp Copy Number of CHO-K1 Cell
Relative Quantity = 10^{\frac{C_t\times 14.902}{3.1101}}
Copy Number :
Plasmid used in standard curve: 0.1021 fmol, Relative Quantity=1
c=\frac{\text{fmole}\cdot 10^3}{0.1201}\cdot c = \frac{\text{Quantity}}{1}
c=\frac{\text{Quantity}\cdot 102.1}{\text{fmole}}
(c: copy number)
iv. Discussion
The efficiency of PiggyBac transposon system ranged from about 2% for one transposon to 0.5% for five transposons [3]  So it’s quite possible for us to screen out the cells with single or low copies of NeoLoxp.
In practical experiments, the selected CHO-K1 cell transfected with NeoLoxp by then had NeoLoxp copy number of around 4, which was not good enough. The reason may be that the designed primer was not so satisfying. Also, the contamination during real-time PCR was hard to avoid. To get cells with single copy of NeoLoxp, we still need to design better primers as well as paying attention to the experiment operations.
III. Directed Evolution
After we’ve selected the CHO-K1 cell transfected with single copy of NeoLoxp, we could transfect mutated TRPC5 into the cell’s genomic DNA using Cre-Loxp recombination system. Then we would use fluorescence activated cell sorting(FACS) to select the cells with high intensity of GFP after long term stimulation. To realize high-throughput screening, we designed a device as shown in the figure below. Cells are seeded at the larger area side of the bottle, and the bottle is fixed above vortex through a holder and foam. Then we are going to place this device in cell culture incubator with vortex running for 20 hours, to fulfill long term mechanical force stimulation.
Figure 17. Self-designed Device for Long Term Stimulation
After the first round of mutation and screening, we still need to continue mutation based on the previous result. We will extract genomic DNA of the screened-out CHO-K1 cell, performing polymerase chain reaction with the mutation primer and sending the PCR product for sequencing. Through alignment of the mutated TRPC5 with normal one, we could clearly see which base site has been mutated. Combing with the TRPC5 structure we’ve known, we may explain why the specific site mutation could induce channel’s greater sensitivity to mechanical force. Then we could specify the mutation region every time, repeating then huge-library mutation and high-throughout screening procedure, thus realizing directed evolution.
Ideally, we would like to screen out then mutated TRPC5 with high sensitivity to mechanical force. As for the audiogenetics, we hope to screen out cells which are able to response to specific sound frequency, achieving precise and orthogonal audio regulation.
References
↑ Vornholt, T. (n.d.). Illustration of Gibson assembly. Retrieved April 17,2015, from https://commons.wikimedia.org/wiki/File:Gibson_assembly_overview.png
↑ Reviews, C. (2016). Genetics and Genomics in Medicine. United states: Cram101 Textbook Reviews. 
↑ Lu, X. and W. Huang, PiggyBac mediated multiplex gene transfer in mouse embryonic stem cell. PLoS One, 2014. 9(12): p. e115072.
Made by from the iGEM team SUSTech_Shenzhen.
Licensed under CC BY 4.0 .
©2016 SUSTech_Shenzhen
