<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model">Model </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Impact">Impact</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Human_Practices/Activities">Activities</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Team">Members</A></LI><LI><A href="/Team:SUSTech_Shenzhen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Contribution</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:SUSTech_Shenzhen/InterLab">Interlab</A></LI><UL class="nav navbar-nav navbar-right hidden-sm hidden-md"><LI><FORM action="/Special:Search" id="searchform" class="navbar-form" role="search"><BUTTON type="submit" name="go" class="btn btn-default" id="mw-searchButton" value="Go" title="Search the pages for this text">Search</BUTTON></FORM></LI><LI><A id="nav-login-status">Log in</A></LI></UL></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><DIV class="row"><DIV class="col-xs-10"><H1>

Proof of Concept

        </H1><P>

Project

        </P></DIV></DIV></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#1._PiggyBac_System_Test"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">1. PiggyBac System Test</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#2._R-GECO_Function_Examination"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">2. R-GECO Function Examination</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#3._Piezo1_and_TRPC5_Sensitivity_Measurement"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">3. Piezo1 and TRPC5 Sensitivity Measurement</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Hypoosmolarity"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Hypoosmolarity</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Microfluidics"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Microfluidics</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#4._Validation_of_audiogenetics"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">4. Validation of audiogenetics</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Audible_sonic_stimulation"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Audible sonic stimulation</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Ultrasonic_Stimulation"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Ultrasonic Stimulation</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Long-term_Stimulation"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Long-term Stimulation</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#5._Selection_by_Cre-loxp_system"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">5. Selection by Cre-loxp system</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#6._Mutagenesis_of_TRPC5"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">6. Mutagenesis of TRPC5</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Mutation_frequency_calibration"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Mutation frequency calibration</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#1._Define_ideal_mutation_frequency"><SPAN class="tocnumber">7.1.1</SPAN><SPAN class="toctext">1. Define ideal mutation frequency</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#2._Calibration_of_mutation_frequency_by_sequencing"><SPAN class="tocnumber">7.1.2</SPAN><SPAN class="toctext">2. Calibration of mutation frequency by sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Library_construction"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Library construction</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#3._Calculate_competent_cell_efficiency_from_experimental_result"><SPAN class="tocnumber">7.2.1</SPAN><SPAN class="toctext">3. Calculate competent cell efficiency from experimental result</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#4._Compare_three_methods_for_mutation_library_construction"><SPAN class="tocnumber">7.2.2</SPAN><SPAN class="toctext">4. Compare three methods for mutation library construction</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#7._Cell_screening_for_audiogenetics_test"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">7. Cell screening for audiogenetics test</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV></DIV><DIV class="col-md-9" id="document-body"><H1><SPAN class="mw-headline" id="Overview">Overview</SPAN></H1><P>To implement audiogenetics, we chose two (Piezo1 and TRPC5) from hundreds of mechanosensitive(MS) channels for their sensitivity to mechanical stress and selective permeability to calcium ion. We used genetically encoded calcium indicator (R-GECO) to quantitatively examine the sensibility of Piezo1 and TRPC5 channels. We used hypoosmolarity as a positive control stimuli and validate the function of the MS channels and Calcium sensor. 
We designed a microfluidic chip to quantitatively tune the shear force exerted on cell membrane, and both channel are responsive to such mechical stress. Surprisingly, we discovered that Piezo1-expression CHO cell response to shear force in an biphasic manner and a maximum responsive of ~0.3 pa. We haven't found similar report in the past. 
We also examined the intracellular calcium response to various combination of frequency and intensity of sound.We discovered that Piezo1-expression CHO cell are very sensitive to 15khz sound and not to 10khz, which indicate the previous identified touching sensing Piezo1 channel could detect sound wave of particular frequency range. 
Finally, we engineered CHO cell with a GFP gene under the control of NFAT promoter to detect chronic  calcium signal). With which, we further developed tools for direct evolution of TRPC5 towarding mutants with high sensitivity to specific sound parameters.
All in all, our project is well organized. Each procedure is indispensable. We not only implemented audiogenetics with synthetic biology to sense mechanical stress and sound, but only use it as a research tool for discover novel mechanical properties of Piezo1. The whole project can strongly prove the possibility of using sound wave energy as the extracellular input signal to induce intracellular downstream signal.
</P><H1><SPAN class="mw-headline" id="1._PiggyBac_System_Test">1. PiggyBac System Test</SPAN></H1><P>In order to quantitatively test the function of PiggyBac system, we did two groups of transfection using LipofectamineÂ® 3000. One was a group of CHO-K1 cells transfected with R-GECO, and the other group was the cells transfected with R-GECO and PiggyBac.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.1 PiggyBac system test</B></DIV></DIV><P>a) Bright field image of the cells transfected with R-GECO. b) Fluorescence of the cells transfected with R-GECO. c) Bright field image of the cells transfected with R-GECO and PiggyBac. d) Fluorescence of the cells transfected with R-GECO and PiggyBac.All figures was captured by Nikon microscope (Nikon TS100) 24 hours after transfection.
</P><P>Comparing the fluorescence image of R-GECO (Fig. 1b) with R-GECO + PiggyBac transfected cells (Fig. 1d), we can observe that the group with PiggyBac has more fluorescent cells. Itâs proved that PiggyBac transposon system can help the insertion of R-GECO gene into CHO-K1 genomes.
</P><H1><SPAN class="mw-headline" id="2._R-GECO_Function_Examination">2. R-GECO Function Examination</SPAN></H1><P>To quantitatively test the function of R-GECO, two experiments, potassium chloride (KCl) induced calcium influx and ionomycin induced calcium influx, were conducted under the live cell imaging.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.2 KCl induced Ca2+ influx result</B></DIV></DIV><P>(Calcium influx due to depolarization of cell membrane was induced by different concentration of KCl. Different volume of KCl was added by drops into cell culture wells to reach the different final concentrations of KCl (control, 12.5mM, 50mM, 200mM, respectively) at 40s. Fluorescence intensity over time is shown as signal per pixel (the initial intensity has been subtracteed and normalized). 0 mM control, 12.5 mM, 50 mM, 200 mM.)
</P><P>There was no distinct fluorescent increase at 12.5mM KCl compared with the control group. Cells under 50mM and 200mM KCl both displayed higher fluorescence. Besides, the fluorescence of cells at 200mM was higher than cell at 50mM along with oscillation. It was proved quantitatively that R-GECO could be excited by KCl-induced calcium influx.
</P><B>Fig.3 Ionomycin induced Ca<SUP>2+</SUP> influx result</B><P>(Fluorescence intensity over time is shown as signal per pixel (the initial intensity has been normalized). Ionomycin was added at 20s to 10Î¼M final concentration.)
</P><P>To further validate our cells, we use ionomycin as another positive control for calcium influx. Ionomycin was dropped on one side of the culture well, the direction of fluorescence gradient (shown in the movie) was consistent with the direction of ionomycin diffusion.
</P><H1><SPAN class="mw-headline" id="3._Piezo1_and_TRPC5_Sensitivity_Measurement">3. Piezo1 and TRPC5 Sensitivity Measurement</SPAN></H1><P>We designed two experiments to quantitatively examine the sensibility of Piezo1 and TRPC5. Microfluidics and hypoosmolarity were employed to exert shear force and tension on our transgenic CHO-K1 cellsâ membrane separately.
</P><H2><SPAN class="mw-headline" id="Hypoosmolarity">Hypoosmolarity</SPAN></H2><P>Calcium influx induced by hypoosmolarity was indicated by fluorescence intensity increase of R-GECO. CHO-K1 cells with Piezo1 (Fig.4a) were much more sensitive to hypoosmolarity. The peak values of R-GECO+Piezo1 increased as osmolarity decreased. And there were distinct oscillations of fluorescence when osmolarity was below 70%. However, fluorescence of cells with R-GECO (Fig. 4b) only showed obvious increase and oscillation when the osmolarity was 60%, whose intensity was a lot less than R-GECO+Piezo1 (Fig. 4c). It was quantitatively proved that Piezo1 channel was very sensitive to osmolarity.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>A</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>B</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>C</B></DIV></DIV><P><B>Fig. 4 Hypoosmolarity actived cytosolic R-GECO sensor</B> a) CHO-K1 cells with R-GECO+Piezo1. b) CHO-K1 cells with R-GECO. c) Compare for these two cell lines. At 10 second (t=10s), distilled water (added with CaCl2) was added into wells to make final concentration of HHBSS into 100%, 90%, 80%, 70%, 60% or 50% for 3 groups.)
Calcium influx was indicated by R-GECO fluorescence intensity. Culture medium was replaced by HHBSS before using live cell imaging. Three groups of cells were seeded into 18 wells: 6 wells of R-GECO, 6 wells of Piezo1 (without doxycycline inducement) and 6 wells of Piezo1 (with doxycycline inducement). Distilled water (added with CaCl<SUB>2</SUB>) was added into wells to make concentration of HHBSS into 100%, 90%, 80%, 70%, 60% or 50% for 3 groups.Relative fluorescence intensity over time is shown as signal per pixel (the initial intensity has been subtracted and normalized.)
</P><H2><SPAN class="mw-headline" id="Microfluidics">Microfluidics</SPAN></H2><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.5 Schematic structure of microfluidics channel</B></DIV></DIV><P>CHO-K1 cells were cultured adherent on the bottom glass of PDMS tunnel. Stable and flexible shear force field was formed cells at surrounding, directly controlled by the micro syringe pump inflow rate.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption">B: 6.25Î¼L/min, GECO+Piezo1 cells</DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption">A: 6.25Î¼L/min, GECO cells</DIV></DIV><DIV class="row"><B>Fig. 6 Fluorescent intensity of microfluidics test</B><P>(A: 6.25Î¼L/min, GECO cells B: 6.25Î¼L/min, C: GECO+Piezo1 cells. CHO-K1 cell suspension was injected into gelatin treated microfluidics chips and incubated at 37Â°C, 5% CO2 incubator for 24 hours. Culture medium was then injected into the chip channels (with the inflow rate of 0Î¼L/min,6.25Î¼L/min and 50Î¼L/min and controlled by micro syringe pump). The pump of culture medium started at t=10s. Cellsâ response towards different shear force was measured by R-GECO fluorescence intensity under live cell imaging.)
</P><P>At the pumping velocity of 6.25Î¼L/min, GECO cells didnât display much fluorescence change. While for GECO+Piezo1 cells, it can sense shear stress thus causing the influx of Ca2+.
</P><P>See the design of microfluidics chip and the simulation of force field at <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model">Model</A></P><P>
CHO-K1 cell suspension was injected into gelatin treated microfluidics chips and incubated at 37Â°C, 5% CO2 incubator for 24 hours. Then the culture medium was injected into the chip channels (with the inflow rate of 25Î¼L/min, 50Î¼L/min and100Î¼L/min controlled by micro syringe pump). The pump of culture medium started at t=10s. Cellsâ response towards different shear force was measured by R-GECO fluorescence intensity under live cell imaging.
</P><P>See the result discussion at <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:SUSTech_Shenzhen/Model/Result">Result</A></P><H1><SPAN class="mw-headline" id="4._Validation_of_audiogenetics">4. Validation of audiogenetics</SPAN></H1><P>We designed both ultrasonic and audible sonic simulation hardware to quantitatively test stimulated cells.
</P><H2><SPAN class="mw-headline" id="Audible_sonic_stimulation">Audible sonic stimulation</SPAN></H2><P>Buzzers, balanced amatures and speakers were used as sonic stimulators. Only buzzers induced obvious cell response in high frequency conditions (15kHz and 20kHz). A buzzer was attached to the bottom of a cell dish. When the buzzer was working, cells around the buzzer boundary was observed under microscope. Strong fluorescence intensity increased immediately after the buzzer was started. Cell fluorescence intensity increased greatly when stimulated by 15K and 20kHz sound. While no obvious fluorescence intensity change when GECO cells (without MS channel) were stimulated (Fig.8 ).
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.7 B</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.7 A</B></DIV></DIV><P><B>Fig. 8 (A) R-GECO+Piezo1 cell fluorescence intensity increased greatly when stimulated by 15K and 20KHz sound. While no obvious fluorescence intensity changed when GECO cells (without MS channel) were stimulated (B).</B></P><H2><SPAN class="mw-headline" id="Ultrasonic_Stimulation">Ultrasonic Stimulation</SPAN></H2><P>Ultrasonic transducers can transform electric energy into ultrasonic radiant energy. There were four kinds of ultrasonic transducer tried. Pressure fields of 108kHz and 1.1MHz transducers were simulated for different power and distance. The pressure field distribution changes smoothly, while the pressure distribution was quite chaos in condition of 1.1MHz. Only strong cell response was observed in condition of 108kHz stimulation as shown in Fig9. No obvious cell response was observed for 1.1MHz stimulation.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig10. 1.1MHz, Adjustable Power Ultrasonic  Transducer</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig. 11 108KHz,4.1W Ultrasonic Stimulation Device</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig 12. A</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig 12. B</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.12 C</B></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.12 D</B></DIV></DIV><P><B>Fig. 12 (A) Pressure distribution of 108KHz ultrasonic transducer, the transducer was 1.0mm far above the cell layer. (B) Pressure distribution of 1.1MHz ultrasonic transducer. 3D pressure distribution for 108KHz condition (C) and 1.1MHz condition (D).</B></P><P>In experiment, 108KHz 4.1W ultrasonic device induced a strong cell response as is shown in the following. However, no cell response was observed when cells were stimulated by 1.1MHz transducer with the same power.
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Fig.14 Fluorescence response of R-GECO+Piezo1 cell when stimulated by 108KHz ultrasonic transducer from 10s to 30s.</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Long-term_Stimulation">Long-term Stimulation</SPAN></H2><P>NFAT part can start gene expression when triggered by Ca2+ influx. The NFAT+Piezo1 cell expresses GFP when its MS channels open inducing a Ca2+ influx. Long-term stimulation (21h, 108KHz, 4.1W ultrasonic stimulation) experiment on Piezo1+NFAT cells showed GFP express level increased after 21h stimulation. 108KHz ultrasonic device was programmed to stimulate cells for duty cycle of 5s on and 10s off.
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Fig.15 Chronic Ca2+ influx activates NFAT promoter and downstream GFP gene expression..</B></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="5._Selection_by_Cre-loxp_system">5. Selection by Cre-loxp system</SPAN></H1><P>Cre-loxp system played an important role in directed evolution, because we needed to screen a single cell colony only with one loxp site in the genome of each CHO-K1 cell for single copy insert of mutated TRPC5. Here we tested the function of cre-loxp system by making 3 different groups of transfection in the cells that already inserted with loxp gene on genomes using LipofectamineÂ® 3000.
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.15 Result of the test of Cre-loxp system</B></DIV></DIV> a) Bright field image of the cells transfected with neomycin-loxp plasmids. b) Bright field image of the cells transfected with Cre recombinase and neomycin-loxp plasmids (3:3). c) Bright field image of the cells transfected with Cre recombinase and neomycin-loxp plasmids (1:3). )
<P>Few cells survives (Fig15. a) after antibiotics screening without transfecting Cre recombinase gene, which means that there is a small possibility that loxp gene can insert into genomes by interior transposase in cells. To find out the optimal ratio of Cre recombinase gene to Neomycin-loxp gene in the process of transfection, we compared two different ratio, 3:3 (Fig15. b) and 1:3 (Fig15. c), with equal amount of Neomycin-loxp gene but different amount of Cre recombinase gene.(All pictures was captured by Nikon microscope (Nikon TS100) 120 hours after transfection, 600 Î¼g/ml neomycin were added 72hours  screening.
</P><P>From observation, we know that excess Cre recombinase is not appropriate in transfection process. The most possible reason is that surplus Cre recombinase is harmful for cells, so they may random cut and recombine genes in cellâs genome. Thus, we finally chose 1:3 as the ratio of Cre recombinase gene to Neomycin-loxp gene in transfection.
</P><H1><SPAN class="mw-headline" id="6._Mutagenesis_of_TRPC5">6. Mutagenesis of TRPC5</SPAN></H1><H2><SPAN class="mw-headline" id="Mutation_frequency_calibration">Mutation frequency calibration</SPAN></H2><H3><SPAN class="mw-headline" id="1._Define_ideal_mutation_frequency">1. Define ideal mutation frequency</SPAN></H3><P>Mutation frequency is the product of DNA polymerase error rate and number of duplications. It depends on initial target amount and fold amplification. Mutation frequencies of 1~4 amino acid changes (2~7 nucleotide changes) per gene are commonly employed.
</P><P>We have mutated the ankyrin repeats region (734bp length) of TRPC5 by employ 500ng initial tartget and 32 cycle amplification according to the protocol of GeneMorph II Random Mutagenesis Kit (Agilent Technologies,Catalog #200550).
</P><H3><SPAN class="mw-headline" id="2._Calibration_of_mutation_frequency_by_sequencing">2. Calibration of mutation frequency by sequencing</SPAN></H3><P>To calibrate the mutation frequency in experimental condition, we sequenced 20 mutated TRPC5 samples in the region of ankyrin repeats. The table below shows the correspondence between mutation(s) and sequenced sample numbers.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Mutation(s) </TH><TH> 1 </TH><TH> 2 </TH><TH> 0
</TH></TR><TR><TH> Sample Numbers </TH><TH> 4 </TH><TH> 1 </TH><TH> 14
</TH></TR></TBODY></TABLE><P><B>Table 1 Practical mutations and sample numbers</B></P><P>5 samples have been detected mutation(s). However, the transformation result showed that the digestion might be incomplete (Table 2). Still, the data could offer us some information. Basically, the practical mutation frequency with initial target amount of 500ng was similar with the theoretical mutation frequency.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Colonies </TH><TH> Control
</TH></TR><TR><TD> 10688 </TD><TD> 113
</TD></TR></TBODY></TABLE><P><B>Table 2. Transformation result of mutated TRPC5 ligation product vs control</B></P><P>To construct a large-scale mutation library with different mutation frequencies in the specific region, we also did error-prone PCR with different initial target amounts to get TRPC5 mutants with low, medium and high mutation rate.
</P><H2><SPAN class="mw-headline" id="Library_construction">Library construction</SPAN></H2><H3><SPAN class="mw-headline" id="3._Calculate_competent_cell_efficiency_from_experimental_result">3. Calculate competent cell efficiency from experimental result</SPAN></H3><P>(1). Formula: <SPAN class="katex-block">\frac{\text{Colonies}}{\text{ng of DNA plated}}\times 1000ng/\mu g</SPAN></P><P>(2). Calculation: 237 / 0.02 x 1000=1.185x107 cfu/Î¼g
</P><P>(3). Actual meaning: 20pg, 5640bp TRPC5 plasmid â 3.457x106 copies
</P><P>3.457x106 / 237=14586, which means 1 of 14586 plasmids would actually grow on the plate.
</P><H3><SPAN class="mw-headline" id="4._Compare_three_methods_for_mutation_library_construction">4. Compare three methods for mutation library construction</SPAN></H3><TABLE class="table table-striped"><TBODY><TR><TH> Method </TH><TH> Colonies </TH><TH> Control
</TH></TR><TR><TD> R. E. D and Ligation </TD><TD> 1772 </TD><TD> 330
</TD></TR><TR><TD> Whole Plasmid Mutagenesis </TD><TD> 191 </TD><TD> 0
</TD></TR><TR><TD> Gibson Assembly </TD><TD> 38 </TD><TD> 7
</TD></TR></TBODY></TABLE><P>Table 5. Transformation Results of Three Methods for Mutation Library Construction. All groups were transformed with 5Î¼l products after all experiment procedure with 100ul competent cell. Restriction Enzyme Digestion and Ligation method and Gibson assembly method used the same mole of TRPC5 backbone. Transformation operations were all the same.
</P><P>From the table, we can know Restriction Enzyme Digestion and Ligation method is the most efficient way of constructing library. 
</P><P>(1). 62ng, 4906bp plasmid â 1.232x1010 copies
</P><P>(2). Efficiency= <SPAN class="katex-block">\frac{(1772-330)\times 20 \div 5}{1.232 \times 10^10 \times \frac{1}{14586}}\times 100\% = 0.68\%</SPAN></P><P>We needed to consider their effect on CHO-k1 cell if being directly transfected without transformation and plasmid construction. Since the ligation efficiency was 0.68%, we did an experiment comparing the survival rate of CHO-k1 cells after being transfected with linear DNA and circular DNA. After antibiotic screening, we luckily found that the former one were almost dead, while the latter one were good. Thus we could directly transfect the ligation products into cell without transformation, preventing the mutation library from decreasing.
</P><P><A href="/Team:SUSTech_Shenzhen/Proof/Directed_Evolution" class="btn btn-default"> See Details</A></P><H1><SPAN class="mw-headline" id="7._Cell_screening_for_audiogenetics_test">7. Cell screening for audiogenetics test</SPAN></H1><P>1. Transfect the cells with loxp gene, pNFAT+GFP gene, R-GECO gene (calcium indicator) by PiggyBac system and sort them into single colonies by FACS (BD FACSAria)</P><DIV class="thumbnail pull-right-sm"><DIV class="caption"><B>Fig.16 The R-GECO and GFP fluorescence of wild type cells and cells transfecd with loxp site, pNFAT+GFP gene and R-GECO gene</B></DIV></DIV><P>The data was collected in the process of sorting by FACS, total cells count is 10000. Cells transfected with R-GECO gene will continually express R-GECO, because CAG promoter is a constitutive promoter. Thus, most cells have red fluorescence (Fig16. b) compare with wild type control (Fig16. a). Cells transfected with pNFAT+GFP should not express GFP because promoter NFAT need to be activated by calcium signal indirectly. However, most cells still can express GFP (Fig.16 d) compare with wild type control (Fig.16 c), which mean there is some leaky expression of promoter NFAT or background calcium activity.
</P><P>2. Select the single cell colony only with one copy number of loxp by qPCRâ¨
</P><P>To evaluate the efficiency of each mutated channels, single cell colony with only one copy number is required when testing, so we designed the following experiments to achieve it. â¨â¨
</P><P><B>2.1 Primer design and primer efficiency test</B></P><P>The real-time PCR primers (Forward: TTGTCAAGACCGACCTGTCC, Reverse: TTCAGTGACAACGTCGAGCA) were designed by Primer-BLAST, a tool for finding specific primers, followed by BLAST specificity check. Firstly, we selected an appropriate annealing temperature to avoid unnecessary amplification as well as improving primer efficiency by normal PCR test in wild type CHO-K1 genomes, the result indicates that 62â may be the most appropriate annealing temperature. In addition, we also confirmed that the amplification product was accurate at 62â when the target was cells genome that transfected with neomycin-loxp gene by sequencing the PCR product.
</P><P>Real-time PCR ( here we used TaKaRa-SYBR Premix Ex Taq II(Tli RNaseH Plus) ran in the ABI StepOne Plus qPCR instrument) can be used to calculate the primer efficiency of PCR amplification via Equation 1. In which <SPAN>X_n</SPAN> = PCR product quantity after n cycles, <SPAN>X_0</SPAN> = initial copy number, <SPAN>E</SPAN> = amplification efficiency, <SPAN>n</SPAN> = cycle number. 
</P><P><SPAN class="katex-block">X_n=x_0 \times (1+E)^n</SPAN></P><P>Equation 1
</P><P>The efficiency of the primer is 104% by calculating (Within the usual margin of error, which is 90%~110%). Also, the melt curve has only one peak, which means the designed primer is quite specific to the transfected CHO-K1 cellâs genomic DNA.
</P><P><B>â¨2.2 Select the cell colony with single copy number</B></P><P>In each qPCR screening experiment, we made a standard curve by employing a series of 5-fold dilution gradient Neomycin-loxp gene with known mole amount. Therefore, we could quantitatively examined the copy number of neomycin-loxp gene in single colony genomes by fitting the sample Ct data in the standard amplification plot.  Standard curve was made by using the following data.
</P><TABLE class="table table-striped"><TBODY><TR><TH> Relative Quantity </TH><TH> log(Relative Quantity) </TH><TH> Ct
</TH></TR><TR><TD> 1 </TD><TD> 0 </TD><TD> 15.01047802
</TD></TR><TR><TD> 0.2 </TD><TD> -0.698970004 </TD><TD> 16.94664574
</TD></TR><TR><TD> 0.04 </TD><TD> -1.397940009 </TD><TD> 19.24198341
</TD></TR><TR><TD> 0.00032 </TD><TD> -3.494850022 </TD><TD> 25.79216194
</TD></TR><TR><TD> -4.193820026 </TD><TD> 27.9509449
</TD></TR></TBODY></TABLE><P>Table 10. Log(Relative Quantity) vs. Ct
</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3"><DIV class="thumbnail"><DIV class="caption">Fig. 17 Fitted Standard Curve</DIV></DIV></DIV></DIV><P>(within the usual margins of error, which is 90%~110%)
</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3"><DIV class="thumbnail"><DIV class="caption">Figure 18. Melt Curve of Neomycin-loxp Transfected CHO-K1 Cellâs gDNA</DIV></DIV></DIV></DIV><P><B>Copy Number</B></P><TABLE class="table table-striped"><TBODY><TR><TD><B>Mass/ng</B></TD><TD><B>Ct</B></TD><TD><B>Relative Quantity</B></TD><TD><B>10^-3 fmole</B></TD><TD><B>Copy Number</B></TD></TR><TR><TD> 78.8075
</TD><TD> 23.2236
</TD><TD> 0.002110101
</TD><TD> 0.0531425
</TD><TD> 4.054
</TD></TR><TR><TD> 40.7625
</TD><TD> 24.1791
</TD><TD> 0.001040142
</TD><TD> 0.0274875
</TD><TD> 3.863
</TD></TR><TR><TD> 19.0225
</TD><TD> 25.2316
</TD><TD> 0.000477161
</TD><TD> 0.0128275
</TD><TD> 3.797
</TD></TR><TR><TD><B>Average</B></TD><TD> 3.905
</TD></TR></TBODY></TABLE><P>Table 11. Neomycin-loxp Copy Number of CHO-K1 Cell  
</P><P>Relative Quantity =<SPAN class="katex-block">10^\frac{\text{Ct}-14.902}{-3.1101}</SPAN></P><P><B>Copy NumberÂ :</B></P><P>Plasmid used in standard curve: 0.1021 fmol, Relative Quantity=1
</P><P><SPAN class="katex-block">c=\frac{\text{Quantity}\cdot 102.1}{\text{fmole} }</SPAN>
(c: copy number)
</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3"><DIV class="thumbnail"><DIV class="caption">Figure 19. Amplification Plot for Standard</DIV></DIV></DIV></DIV><DIV class="thumbnail pull-right-sm"><DIV class="caption">Figure 20. Amplification Plot for Sample</DIV></DIV><P>The efficiency of PiggyBac transposon system ranged from 2% for one transposon to 0.5% for five transposons <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. Consequently, itâs quite possible for us to screen out the cells with single or low copies of Neomycin-loxp.
</P><P>In practical experiments, we totally test 18 different single colony genomes. The lowest copy number of those selected CHO-K1 cell transfected with Neomycin-loxp is around 4, which was not good enough. The reason may be that the designed primer was not so satisfying. Also, the contamination during real-time PCR was hard to avoid. To get cells with single copy of Neomycin-loxp, we still need to design better primers as well as paying attention to the experiment operations. 
</P><P><B>3. Transfect the cells with mutated TRPC5 by Cre-loxp system</B></P><P>After weâve selected the CHO-K1 cell transfected with single copy of NeoLoxp, we could transfect mutated TRPC5 into the cellâs genomic DNA using Cre-Loxp recombination system. Then we would use fluorescence activated cell sorting(FACS) to select the cells with high intensity of GFP after long term stimulation. To realize high-throughput screening, we designed a device as shown in the figure below. Cells are seeded at the larger area side of the bottle, and the bottle is fixed above vortex through a holder and foam. Then we are going to place this device in cell culture incubator with vortex running for 20 hours, to fulfill long term mechanical force stimulation.   
</P><DIV class="thumbnail pull-right-sm"><DIV class="caption">Figure 21. Self-designed Device for Long Term Stimulation</DIV></DIV><P>After the first round of mutation and screening, we still need to continue mutation based on the previous result. We will extract genomic DNA of the screened-out CHO-K1 cell, performing polymerase chain reaction with the mutation primer and sending the PCR product for sequencing. Through alignment of the mutated TRPC5 with normal one, we could clearly see which base site has been mutated. Combing with the TRPC5 structure weâve known, we may explain why the specific site mutation could induce channelâs greater sensitivity to mechanical force. Then we could specify the mutation region every time, repeating then huge-library mutation and high-throughout screening procedure, thus realizing directed evolution.
</P><P>Ideally, we would like to screen out then mutated TRPC5 with high sensitivity to mechanical force. As for the audiogenetics, we hope to screen out cells which are able to response to specific sound frequency, achieving precise and orthogonal audio regulation.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">â</A></SPAN><SPAN class="reference-text">Lu, X. and W. Huang, PiggyBac mediated multiplex gene transfer in mouse embryonic stem cell. PLoS One, 2014. <B>9</B>(12): p. ell 5072</SPAN></LI></OL></DIV></DIV><FOOTER class="footer bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Made by  from the iGEM team SUSTech_Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>Â©2016 SUSTech_Shenzhen</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>