Team:Baltimore Bio-Crew/Experiments
Home Team Project Results Design Notebook Composite Part Safety HP Silver Experiment Attribution Collaborations Interlab Contributions
Experiment
Protocols
Gel Electrophoresis
Adapted from Addgene, (https://www.addgene.org/protocols/gel-electrophoresis/)
Pouring the Gel
Measure out agarose on scale slowly (for a standard 1% gel, use 1 g of agarose. For other concentrations, calculate accordingly). Stay within ~.05 g of 1 g.
Mix agarose powder with 100 mL 1xTAE in a microwavable flask.
Microwave for 1-3 min until the agarose is completely dissolved, stopping every 30 seconds to swirl the flask. Keep an eye on it to make sure it does not overboil. Wear oven mitts when handling the flask.
Place agarose solution on counter until the flask can be comfortably held
Set up the gel tray with the well comb in place, as one person slowly pours the agarose solution into the well. Avoid bubbles.
Let gel sit for 20-30 minutes until completely solidified (the gel will be milky white and opaque).
Loading Samples
Add equal amount of loading buffer to each of your samples.
Place the agarose gel into the electrophoresis unit.
Fill gel box with 1xTAE until the gel is covered and container is filled.
Carefully load a molecular weight ladder into the first lane of the gel. Place your elbow on the counter to prop yourself up so you can look at the gel from an angle to see the wells. Place the tip of the pipette tip just above the well and slowly push so the sample fills the entire well.
Load the rest of the samples in this manner.
Run the gel at 80-150 V until the dye line is approximately 75-80% of the way down the gel for about 1 hour. Note: Black is negative, red is positive. Always Run to Red.
Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.
Visualize your DNA fragments under UV light.
Plasmid Purification
All plasmid purifications were done using the QIAprep spin mini prep kit protocol.
Restriction Digest
Master mix:
76 uL H2O
20 uL 10x NEB2
1 uL EcoR1
1 uL Pst1
1 uL Dpn1
1 uL BSA
Mix together:
10 uL Sample DNA
10 uL Master Mix
Ligation
Mix together:
2 uL plasmid
3.5 uL insert
1 uL T4 Ligase buffer
0.5 uL T4 Ligase
4.5 uL H2O
Transformation
We used the iGEM Parts Registry Transformation Protocol: https://static.igem.org/mediawiki/parts/6/67/IGEM_Registry_-_Transformation_Protocol.pdf
Changes made: Recovery tubes were made using 500 uL SOC media instead of 200 uL
Colony PCR
Master Mix:
316.8ul dIH20
44ul Taq Buffer
22ul foreward primer
22ul reverse primer
33ul dNTPs
2.2ul Taq
Add 1ul DNA to 19ul Master Mix
Settings:
1 Cycle
95°C - 1min
30 Cycles
95°C - 30s
55°C - 45s
72°C - 45s
1 Cycle
72°C - 2min
Hold at 4°C
Ni-NTA Spin Column Purification of 6xHis-Tagged Proteins (E. coli Lysates)
Resuspend a pellet derived from 5ml cell culture volume in 630 ul Lysis buffer (NPl-10). Add 70ul Lysozyme Stock Solution (10mg/ml) and add 3 units/ml culture volume Benzonase Nuclease
Incubate for 15-30min
Centrifuge lysate at 12000 x g for 15-30 min at 4 degrees C. Collect the supernatant.
Save 20 ul of the lysate for SDS-PAGE analysis
Equilibrate the Ni-NTA spin column with 600 ul buffer NPl-10. Centrifuge for 2 minutes at 2900 rpm. Spin without lid to ensure it is done in two minutes
Load up to 600 ul of the cleared lysate containing the His-Tagged protein onto the (pre-equilibrated) Ni-NTA spin columns. Spin for 5 minutes at about 1600 rpm, and collect the flow-through. To ensure efficient binding, don’t go higher than 1600 rpm, though do it with an open lid
Wash the column twice with 600 ul Buffer NPl-20. Centrifuge for 2 min. at about 2900 rpm. Save the flow-through for analysis by SDS-PAGE
Elute the protein twice with 300ul Buffer NPl-500. Centrifuge for 2 minutes at about 2900, and collect the eluate. Most of the protein will be in the first eluate. The rest will be in the second. 300ul each time
Protein Resuspension and Protein Gel Electrophoresis Protocol
Resuspend protein sample in a 1:1 ratio of sample to cracking buffer
Place resuspended sample in boiling water for 5 minutes
Load samples into protein gel
Run at 100v
Run until the gel is about 75-80% of the way down the gel
Remove gel from electrodes and plastic casing
Wash gel with purified water about 3 times, agitating the vessel the gel and water are in slightly to wash thoroughly, while not tearing the gel
Stain the gel with Simplyblue dye
In Vitro Transcription Translation
Protocol using NEB’s PURExpress kit-
https://www.neb.com/protocols/0001/01/01/protein-synthesis-reaction-using-purexpress-e6800
Changes made: RNA Polymerase was added during step two. A control sample was made using no RNA Polymerase.
PCR Amplification
Mix together:
35.4 uL Water
15 uL 5x Phusion Buffer
1 uL 10 mM dNTPs
.8 uL 30 mM Forward Primer
.8 uL 30 mM Reverse Primer
.5 uL enzyme (Phusion taq)
2 uL Template
Settings
1 Cycle
98°C - 30s
35 Cycles
98°C - 10s
55°C - 30s
72°C - 2min
1 Cycle
72°C - 10min
Flourocine Diacetate Enzyme Activity Assay
Enzyme activity was assayed using a modification of the fluorocine diacetate assay for hydrolytic enzymes (Gamble and Muriana 2007). 5ul of in vitro transcription/translation product was mixed with 250ul of phospate buffered saline containing 1 mM flourescine diacetate  in wells of a Tecan black Tecan micro titer plate. Flouresence generated by cleavage of the acetate groups was measured using a Tecan Genios plate reader with excitation at 485nM and emission at 535 nM.
Allergenicity Testing Protocol
Getting protein Sequence:
Use http://www.fr33.net/translator.php to convert nucleotide sequence to protein sequences
Plug in output sequences into NCBI BLAST to verify which protein sequence is the intended sequence.
Examining Allergenicity
Copy nucleotide sequence for the gene that is to be tested.
Click on a new tab, and go onto Translator by clicking this link: http://www.fr33.net/translator.php . Paste the nucleotide sequence into the box and press translate. This will change your nucleotides into amino acids. Copy this amino acid chain UNTIL the first asterisk (leave out the asterick).Copy this sequence.
Then, click on a new tab, and go onto the NCIB Blast by clicking this link: https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins . Paste the amino acid chain in the box and press BLAST. This could take a couple seconds for the results to come up. This is used to verify the identity of the protein that is going to be used later in the protocol. The identity of the amino acid chain can be seen by comparing the first result of the NCIB Blast matches the same name as the biobrick name on the IGEM website. If it does match then continue to the next step. If it does not check another translation found in part 2.
Click new tab, and go onto AllergenOnline by following this link: http://www.allergenonline.org/index.shtml
Press Search for full-length alignments by FASTA, and copy and paste your amino acid chain (The same amino acid chain from step 3) into the box. Make sure the bubble that says "Full Fasta" is hit and press submit.
This will open up another window. Look at the % identity and % of similarity. If there is a % identity higher than 50% than, the biobrick potentially can be an allergen. If not, then your biobrick is less likely to be an allergen.
Now, we will go back to the main page of the AllergenOnline website by clicking this link: http://www.allergenonline.org/index.shtml . Then, press Search for 80 amino acid alignments by FASTFA.
Paste the amino acid chain (the same chain from steps 3, and 5) into the box and make sure the second bubble is filled in called sliding 80mer window and press submit.
The results will open up in a new tab again, and look at and record the % identity and % similarity in the same google document. If the % identity is less than 35%, then the biobrick is less likely to be allergens.
Sponsors
The Baltimore Bio-Crew thanks our sponsors for their generous support of our team that made our project and travel to the Jamboree possible. Thank you!
