<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Baltimore_Bio-Crew_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Baltimore Bio-Crew/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="baltimoreBiocrewNav"><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew">Home</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Team">Team</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Description">Project</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Results">Results</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Design">Design</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Notebook">Notebook</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Composite_Part">Composite Part</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Safety">Safety</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Experiments"> Experiment</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Attributions"> Attribution</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/InterLab">Interlab</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Contribution">Contributions</A></DIV><DIV class="topnav"><UL><LI><A href="#july">July</A></LI><LI><A href="#august">August</A></LI><LI><A href="#september">September</A></LI><LI><A href="#october">October</A></LI></UL></DIV><DIV class="content"><H3>NoteBook</H3></DIV><SECTION id="july" class="julio"><H4>July</H4><DIV class="container"><BUTTON class="dates">7/01/17</BUTTON><DIV class="accordionsCalender"><P>Math:</P><UL><LI>Worked on getting set up to start math modeling of plastic degradation. Ready to start next week</LI><LI>Found possible alternatives to our plastic degrading enzymes</LI><LI>Collaboration: College Park mentoring for plastic degrading simulation</LI></UL><P>Wiki:</P><UL><LI>Created HTML for website from template (not implemented into wiki yet)</LI><LI>Had difficulties getting the template code to work with the pre-set wiki code</LI><LI>Emailed UMaryland team to ask for help with wiki programming.</LI></UL><P>Gene Design:</P><UL><LI>Designed genes with Secretion Tag and Periplasmic Tag</LI><LI>Optimized codons</LI><LI>Waiting for genes to be shipped</LI></UL><P>Lab:</P><UL><LI>Ran 30 colony PCR reactions</LI><LI>Ready to run a gel on the PCR and send the successful transformations in for gene sequencing</LI></UL><P>Outreach and Fundraising:</P><UL><LI>Created template for emailing companies</LI><LI>Reached out to aquarium, Henry Hall program</LI><LI>Contacted local high school for class time discussion</LI></UL></DIV><BUTTON class="dates">7/06/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>Prepared all 30 colony PCR samples to run on a gel</LI><LI>Ran 16 samples, one possible success found</LI><LI>Other 14 samples ready to be run on Saturday</LI></UL></DIV><BUTTON class="dates">7/08/17</BUTTON><DIV class="accordionsCalender"><P>Interlab:</P><UL><LI>signed up</LI><LI>made 8 chloramphenicol plates</LI></UL><P>Linearized Backbone Protocol:</P><UL><LI>learned the anatomy of the different vectors</LI><LI>performed restriction digest on vectors</LI></UL><P>Fundraising:</P><UL><LI>found more companies to reach out to</LI><LI>emailed enzyme assay companies about giving us a discount on assaying materials</LI></UL><P>Our Constructs:</P><UL><LI>ran PCR samples 17-30 on gel. Currently waiting for results</LI></UL><P>Wiki:</P><UL><LI>learned how to use iGEM wiki templates to link to more code files</LI><LI>added navigation bar</LI><LI>experimented with the code to learn more about how the iGEM wikis function</LI></UL></DIV><BUTTON class="dates">7/13/17</BUTTON><DIV class="accordionsCalender"><P>Interlab:</P><UL><LI>Decided to use Kit Plate 7</LI><LI>Resuspended 8 DNA samples from plate</LI><LI>Samples used:
     </LI><LI>Positive Control (BBa_I20270): well 21B</LI><LI>Negative Control (BBa_R0040): well 21D</LI><LI>Test Device 1 (BBa_J364000): well 21F</LI><LI>Test Device 2 (BBa_J364001): well 21H</LI><LI>Test Device 3 (BBa_J364002): well 21J</LI><LI>Test Device 4 (BBa_J364003): well 21L</LI><LI>Test Device 5 (BBa_J364004): well 21N</LI><LI>Test Device 6 (BBa_J364005): well 21P</LI></UL><LI>Set up transformations for the 8 samples (transformations later found to be mostly unsuccessful. Did not produce many colonies)</LI></DIV><BUTTON class="dates">7/15/17</BUTTON><DIV class="accordionsCalender"><P>Math:</P><UL><LI>learned basics of enzyme kinetics</LI><LI>started downloading MATLab</LI></UL><P>Lab:</P><UL><LI>ligated new gBlocks with both vectors</LI><LI>transformed gBlocks</LI><LI>made more media for plates</LI></UL><P>Interlab:</P><UL><LI>re-transformed all 8 samples</LI></UL><P>Wiki:</P><UL><LI>Uploaded home page</LI><LI>worked on color theme and logo ideas</LI><LI>still need to get the rest of the pages up</LI></UL><P>Waxworms:</P><UL><LI>students came in and measured waxworms</LI></UL></DIV></DIV></SECTION><SECTION id="august" class="augusto"><H4>August</H4><DIV class="container"><BUTTON class="dates">8/03/17</BUTTON><DIV class="accordionsCalender"><P>Lab</P><UL><LI>Restriction digest of esterase, lipase, C3, and A3 were lost. SO, we started all over again (hopefully for the last time) and did a restriction digest of esterase and lipase again. Then, found C3 and A3 digested from a random previous restriction digestion.</LI><LI>ran this restriction digest on a gel, and it looked good</LI><LI>did a PCR of the esterase and lipase stock solutions to produce more.</LI></UL></DIV><BUTTON class="dates">8/05/17</BUTTON><DIV class="accordionsCalender"><P>Lab</P><UL><LI>Used nanodrop to determine the DNA concentrations of our restricted digest of esterase, lipase, A3, and C3. Data is in Shatera’s notebook. We did this to determine the ratio of microliters of the vector and DNA insert to use for ligation.</LI><LI>Worked out the ratio for ligations using stoichiometry</LI><LI>Ligated using ratios.</LI><LI>Transformed bacteria by heat shocking. We have two groups of the ligation and two groups of the transformed bacteria. After incubating the ligations in room temp for 30 min, the ligations were incubated with some of the bacteria (heat shocked for transformation). Then, after an hour, the bacteria were plated and grown. Then, with the other half of the ligations, we continued to incubate the ligations overnight at 44 degrees after sitting for 30 minutes in room temp. Then, we would transform the bacteria the next day. Then, after an hour, they should be plated to grow.</LI></UL><P>Presentation</P><UL><LI>worked on putting together an outline of the presentation</LI></UL></DIV><BUTTON class="dates">8/10/17</BUTTON><DIV class="accordionsCalender"><P>Lab</P><UL><LI>Bacteria Colony PCR- we finally got positives!</LI><LI>Isolated plasmid DNA from colonies to screen as well (in case the PCRs aren’t working).</LI></UL></DIV><BUTTON class="dates">8/12/17</BUTTON><DIV class="accordionsCalender"><UL><LI>Made DNA stocks of positive clones which means that we did plasmid isolation with the positive colonies and used nanodrop to determine concentrations a purity of plasmid. Then, ethanol and salts were added to the isolated plasmids to increase concentration. The plasmids were sent to sequencing to Johns Hopkins.</LI><LI>Worked on presentation</LI></UL></DIV><BUTTON class="dates">8/17/17</BUTTON><DIV class="accordionsCalender"><UL><LI>Took the tour of the inner harbor and the trash wheel, and learned about the efforts to clean the harbor</LI></UL></DIV><BUTTON class="dates">8/19/17</BUTTON><DIV class="accordionsCalender"><UL><LI>worked on completing the presentation slides and script</LI></UL></DIV><BUTTON class="dates">8/26/17</BUTTON><DIV class="accordionsCalender"><UL><LI>worked on completing the presentation slides and script</LI></UL></DIV></DIV></SECTION><SECTION id="september" class="septembre"><H4>September</H4><DIV class="container"><BUTTON class="dates">9/02/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>Recloning the Lipase:
    </LI><LI>PCR Amplified: lipase g-block, previously digested lipase, C3 vector from J04450 plasmid, and A3 vector from J04450 plasmid</LI><LI>Miniprepped A3 + esterase plasmids to be used as a control for the PCR gel</LI><LI>Ran PCR gel (1 ul loading dye in each sample). Gel Key:
     </LI><LI>Ladder (1 ul)</LI><LI>C3 vector (5 ul)</LI><LI>A3 vector (5 ul)</LI><LI>lipase from previous digest (5 ul)</LI><LI>lipase from g-block (5 ul)</LI><LI>PCR purified all samples</LI><LI>Ran Nanodrop on PCR purification results. Nanodrop results:
     </LI><LI>C3: 16.4ng/ul</LI><LI>A3: 19.1ng/ul</LI><LI>lipase digested: 48.1ng/ul</LI><LI>lipase g-block: 25.0 ng/ul</LI><LI>1st esterase A3 plasmid miniprep: 30.4ng/ul</LI><LI>2nd esterase A3 plasmid miniprep: 52.6ng/ul</LI><LI>Diluted all samples to 10ng/ul</LI><LI>Ran restriction digest on all samples</LI></UL><LI>Purified esterase enzymes</LI><LI>Ran protein gel of esterase enzymes. Gel key:
    </LI><LI>standard 20ul</LI><LI>un-induced 20ul</LI><LI>induced 20ul</LI><LI>un-induced 40ul</LI><LI>induced 40ul</LI><LI>Working Interlab- Must start over the initial protocol with the spectrometer and incubation for 2, 4, 6 hours.</LI><P>Presentation:</P><UL><LI>rehersed</LI></UL><P>Wiki:</P><UL><LI>worked on design and color scheme</LI></UL></DIV><BUTTON class="dates">9/06/17</BUTTON><DIV class="accordionsCalender"><P>Purification of Protein and Interlab:</P><UL><LI>Grew up transformed bacteria that we know have our esterase gene! We grew the bacteria in 5mL overnight cultures. We used PSB1A3 + number 4 and PSB1A3 + number 6. PSB1C3 + number 5 did not grow.</LI><LI>Then, we added 2mL of overnight culture into 50 mL of LB AMP and we let it grow. Then, we checked the OD of our cultures so it could get to .6:
    </LI><LI>A4
      </LI><LI>.140 - 10:15</LI><LI>.402 - 11:15</LI><LI>.67 - 12:15</LI><LI>A6
      </LI><LI>.144- 10:15</LI><LI>.385 - 11:15</LI><LI>.740 - 12:15</LI></UL><LI>We induced our four flasks. 2 were induced with 1 mM IPTG 62.5 microliter of .8 M stock. Then, 2 were not induced</LI><LI>After each time point of 0, 1, 2, 3, 4 in a shaker, we obtained 6 mL from all four flasks. We put one mL of bacteria in 4 microcentrifuge tubes from each flask. Then, we placed 5mL from all four flasks into four tubes. We centrifuges all eight tubes in centrifuge for 10 min at 400 speed to form pellets of cells. Then, the microcentrifuge tubes will be used in a gel. Then, the tubes will be frozen and later lysed and purified using a nickel column to verify the length of proteins; this will happen on another day. For now, the pellets are frozen. The group of samples that will be used for the gel will be ran on a gel another day as well.</LI></DIV><BUTTON class="dates">9/16/17</BUTTON><DIV class="accordionsCalender"><P>Interlab:</P><UL><LI>Started on cell measurement protocol</LI><LI>Protocol was followed incorrectly and interlab would need to be re-done later</LI></UL><P>Lab:</P><UL><LI>Ran protein gel of clones:
    </LI><LI>A3+Esterase #4</LI><LI>A3+Esterase #6</LI></UL></DIV><BUTTON class="dates">9/21/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>Ran colony PCR of Lipase+C3 plasmid</LI></UL></DIV><BUTTON class="dates">9/22/17</BUTTON><DIV class="accordionsCalender"><P>Learned more about micro plastics at a seminar at the Baltimore Underground Science Space</P></DIV><BUTTON class="dates">9/23/17</BUTTON><DIV class="accordionsCalender"><P>Started on interlab again:</P><UL><LI>Took several timepoints:
    </LI><LI>t0 = 10:30</LI><LI>t1 = 12:30</LI><LI>t2 = 2:30</LI><LI>t3 = 4:30</LI></UL><LI>measured florescence at each timepoint and recorded it in the provided spreadsheet</LI><P>Lab:</P><UL><LI>ran protein gel for Esterase+C3 #5. Loaded 10ul of each sample, 5ul ladder. Gel Key:
    </LI><LI>protein ladder (NEB Broadspectrum)</LI><LI>psb1C3+Esterase #5 uninduced timepoint 4hr (sample 2 test)</LI><LI>psb1C3+Esterase #5 uninduced timepoint 4hr (sample 3 test)</LI><LI>psb1C3+Esterase #5 uninduced timepoint 4hr (Elute 1)</LI><LI>psb1C3+Esterase #5 uninduced timepoint 4hr (Elute 2)</LI><LI>psb1C3+Esterase #5 induced timepoint 4hr (test sample 1)</LI><LI>psb1C3+Esterase #5 induced timepoint 4hr (test sample 2)</LI><LI>psb1C3+Esterase #5 induced timepoint 4hr (test sample 3)</LI><LI>psb1C3+Esterase #5 induced timepoint 4hr (Elute 1)</LI><LI>psb1C3+Esterase #5 induced timepoint 4hr (Elute2)</LI></UL></DIV><BUTTON class="dates">9/28/17</BUTTON><DIV class="accordionsCalender"><P>Interlab:</P><UL><LI>OD600 reference point data lost. Went through protocol again.</LI><LI>Submitted interlab data. not accepted because we didn't have time to do enough replicates.</LI></UL></DIV><BUTTON class="dates">9/30/17</BUTTON><DIV class="accordionsCalender"><P>Due to interlab being no longer an option to complete our requirements, we considered other options to meet these requirements.</P><UL><LI>for our contribution requirement, we decided to write a protocol for testing for allergenicity in different proteins. This would make it easy for other teams to see if their biobricks would produce proteins that could contain allergens, and allow us to characterize other parts by testing for allergens in them.</LI></UL><P>Lab:</P><UL><LI>Because our protein gels were consistently negative for esterase, we hypothesized that the enzyme was toxic to E.coli. To test this, we decided to try transcribing and translating our esterase constructs using a cell free kit. We also wanted to sequence our biobricks by themselves, so that we could be sure that there were no errors in them before we started. Because of this, we needed to PCR amplify our biobricks to get more DNA to work with. PCR amplified:
    </LI><LI>esterase biobrick (2017)</LI><LI>lipase biobrick (2017)</LI><LI>esterase biobrick (2016)</LI><LI>lipase biobrick (2016)</LI><LI>Ran gel of PCR</LI></UL></DIV></DIV></SECTION><SECTION id="october" class="octubre"><H4>October</H4><DIV class="container"><BUTTON class="dates">10/05/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>gel showed PCR to be unsuccessful. The wrong primers were used to amplify.</LI><LI>Re-did PCR with correct primers</LI><LI>Ran gel of PCR. Used 1ul of each ladder and 10ul of each sample. Gel key:
    </LI><LI>1kb ladder</LI><LI>X</LI><LI>Lipase 2017 1</LI><LI>Lipase 2017 2</LI><LI>X</LI><LI>Esterase 2017 1</LI><LI>Esterase 2017 2</LI><LI>X</LI><LI>Lipase 2016 1</LI><LI>Lipase 2016 2</LI><LI>X</LI><LI>Esterase 2016 1 </LI><LI>Esterase 2016 2</LI><LI>X</LI><LI>100bp ladder</LI></UL></DIV><BUTTON class="dates">10/07/17</BUTTON><DIV class="accordionsCalender"><P>Presented to the public and the UMaryland iGEM team for Baltimore Innovation Week. Received feedback on project and presentaion</P><P>Lab:</P><UL><LI>Gel purified samples L1, L2, E1, and E2</LI><LI>Ran a gel with gel purified samples. Ran 6ul of each sample. Gel key:
    </LI><LI>ladder</LI><LI>X</LI><LI>L1</LI><LI>L2</LI><LI>X</LI><LI>E1</LI><LI>E2</LI><LI>Took nanodrop concentrations of the gel purified samples.</LI><LI>Despite unreliable readings from the nano drop, all of the results for the gel were positive.</LI></UL></DIV><BUTTON class="dates">10/14/17</BUTTON><DIV class="accordionsCalender"><P>helped run a table at the FamFest event at the Y, and talked about synthetic biology with the public</P></DIV><BUTTON class="dates">10/19/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>The concentrations of three samples, were taken on a nano drop. The two unmodified biobricks from this year, L1 and E1, did not have a reliable measurement on the nano drop. However, the successful plasmid from this year, CE 5, did have a good nano drop concentration reading. Because of this, we decided to only run the cell free kit on the CE 5 plasmid.</LI><LI>Sample CE 5 was transcribed and translated in-vitro using a cell free kit.</LI></UL></DIV><BUTTON class="dates">10/21/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>Ran protein gel of the cell free protein expression samples and the time course samples for the positive control bacteria and induced engineered E.coli, some induced with lactose and some induced with IPTG.</LI><LI>Gel 1 Key:
    </LI><LI>ladder</LI><LI>control 2</LI><LI>control 3</LI><LI>control4</LI><LI>CE5-1 IPTG 2hr</LI><LI>CE5-2 IPTG 2hr</LI><LI>CE5-3 IPTG 2hr</LI><LI>CE5-1 lac 2hr</LI><LI>CE5-2 lac 2hr</LI><LI>CE5-3 lac 2hr</LI><LI>Gel 2 Key:
    </LI><LI>CE5-1 IPTG 5hr</LI><LI>CE5-2 IPTG 5hr</LI><LI>CE5-3 IPTG 5hr</LI><LI>ladder</LI><LI>CE5-1 lac 5hr</LI><LI>CE5-2 lac 5hr</LI><LI>CE5-3 lac 5hr</LI><LI>X</LI><LI>X</LI><LI>X</LI><LI>Gel 3 Key:
    </LI><LI>CE5-1 IPTG</LI><LI>CE5-2 IPTG</LI><LI>CE5-3 IPTG</LI><LI>CE5-1 lac</LI><LI>CE5-2 lac</LI><LI>CE5-3 lac</LI><LI>ladder</LI><LI>X</LI><LI>CE5 (in-vitro)</LI><LI>positive control</LI></UL></DIV><BUTTON class="dates">10/26/17</BUTTON><DIV class="accordionsCalender"><P>Wiki:</P><UL><LI>compiled a list of what still needed to be done</LI></UL><P>Completed Judging form and worked on finishing up poster</P></DIV><BUTTON class="dates">10/28/17</BUTTON><DIV class="accordionsCalender"><P>Lab:</P><UL><LI>Completed flouricine diacetate enzyme assay on the products of cell free expression by the Esterase and Lipase biobricks. Showed positive for enzymes!</LI></UL><P>Wiki:</P><UL><LI>worked on getting finished up</LI><LI>put together protocols page content, project description content, and notebook page</LI></UL></DIV></DIV></SECTION><SECTION id="Footer" class="footerSection"><H2>
	Sponsors
			</H2><H4>
				The Baltimore Bio-Crew thanks our sponsors for their generous support of our team that made our project and travel to the Jamboree possible. Thank you!
			</H4></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>