<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Baltimore_Bio-Crew_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Baltimore Bio-Crew/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="baltimoreBiocrewNav"><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew">Home</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Team">Team</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Description">Project</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Results">Results</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Design">Design</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Notebook">Notebook</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Composite_Part">Composite Part</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Safety">Safety</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/HP/Silver">HP Silver</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Experiments"> Experiment</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Attributions"> Attribution</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/InterLab">Interlab</A><A href="https://2017.igem.org/Team:Baltimore_Bio-Crew/Contribution">Contributions</A></DIV><SECTION id="intro" class="Intro"><DIV class="SlideContainer"><DIV class="slides"><DIV class="experiment"><H1 style="font-size: 100px; overflow: visible; color: #f2f2f2;"> Experiment</H1></DIV></DIV></DIV></SECTION><SECTION id="protocols" class="pureBlue"><H3>Protocols</H3><DIV class="container"><BUTTON class="dates">Gel Electrophoresis</BUTTON><DIV class="accordionsCalender"><P>Adapted from Addgene, (https://www.addgene.org/protocols/gel-electrophoresis/)</P><P>Pouring the Gel</P><UL><LI>Measure out agarose on scale slowly (for a standard 1% gel, use 1 g of agarose. For other concentrations, calculate accordingly). Stay within ~.05 g of 1 g.</LI><LI>Mix agarose powder with 100 mL 1xTAE in a microwavable flask.</LI><LI>Microwave for 1-3 min until the agarose is completely dissolved, stopping every 30 seconds to swirl the flask. Keep an eye on it to make sure it does not overboil. Wear oven mitts when handling the flask.</LI><LI>Place agarose solution on counter until the flask can be comfortably held</LI><LI>Set up the gel tray with the well comb in place, as one person slowly pours the agarose solution into the well. Avoid bubbles.</LI><LI>Let gel sit for 20-30 minutes until completely solidified (the gel will be milky white and opaque).</LI></UL><P>Loading Samples</P><UL><LI>Add equal amount of loading buffer to each of your samples.</LI><LI>Place the agarose gel into the electrophoresis unit.</LI><LI>Fill gel box with 1xTAE until the gel is covered and container is filled.</LI><LI>Carefully load a molecular weight ladder into the first lane of the gel. Place your elbow on the counter to prop yourself up so you can look at the gel from an angle to see the wells. Place the tip of the pipette tip just above the well and slowly push so the sample fills the entire well.</LI><LI>Load the rest of the samples in this manner.</LI><LI>Run the gel at 80-150 V until the dye line is approximately 75-80% of the way down the gel for about 1 hour. Note: Black is negative, red is positive. Always Run to Red.</LI><LI>Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.</LI><LI>Visualize your DNA fragments under UV light.</LI></UL></DIV><BUTTON class="dates">Plasmid Purification</BUTTON><DIV class="accordionsCalender"><P>All plasmid purifications were done using the QIAprep spin mini prep kit protocol.</P></DIV><BUTTON class="dates">Restriction Digest</BUTTON><DIV class="accordionsCalender"><P>Master mix:</P><OL><LI>76 uL H2O</LI><LI>20 uL 10x NEB2</LI><LI>1 uL EcoR1</LI><LI>1 uL Pst1</LI><LI>1 uL Dpn1</LI><LI>1 uL BSA</LI></OL><P>Mix together:</P><UL><LI>10 uL Sample DNA</LI><LI>10 uL Master Mix</LI></UL></DIV><BUTTON class="dates">Ligation</BUTTON><DIV class="accordionsCalender"><P>Mix together:</P><UL><LI>2 uL plasmid</LI><LI>3.5 uL insert</LI><LI>1 uL T4 Ligase buffer</LI><LI>0.5 uL T4 Ligase</LI><LI>4.5 uL H2O</LI></UL></DIV><BUTTON class="dates">Transformation</BUTTON><DIV class="accordionsCalender"><P>We used the iGEM Parts Registry Transformation Protocol: <A href="https://static.igem.org/mediawiki/parts/6/67/IGEM_Registry_-_Transformation_Protocol.pdf">https://static.igem.org/mediawiki/parts/6/67/IGEM_Registry_-_Transformation_Protocol.pdf</A></P><P>
  Changes made: Recovery tubes were made using 500 uL SOC media instead of 200 uL
  </P></DIV><BUTTON class="dates">Colony PCR</BUTTON><DIV class="accordionsCalender"><P>Master Mix:</P><OL><LI>316.8ul dIH20</LI><LI>44ul Taq Buffer</LI><LI>22ul foreward primer</LI><LI>22ul reverse primer</LI><LI>33ul dNTPs</LI><LI>2.2ul Taq</LI></OL><P>Add 1ul DNA to 19ul Master Mix</P><P>Settings:</P><P>1 Cycle</P><OL><LI>95°C - 1min</LI></OL><P>30 Cycles</P><OL><LI>95°C - 30s</LI><LI>55°C - 45s</LI><LI>72°C - 45s</LI></OL><P>1 Cycle</P><OL><LI>72°C - 2min</LI></OL><P>Hold at 4°C</P></DIV><BUTTON class="dates">Ni-NTA Spin Column Purification of 6xHis-Tagged Proteins (E. coli Lysates)</BUTTON><DIV class="accordionsCalender"><OL><LI>Resuspend a pellet derived from 5ml cell culture volume in 630 ul Lysis buffer (NPl-10). Add 70ul Lysozyme Stock Solution (10mg/ml) and add 3 units/ml culture volume Benzonase Nuclease</LI><LI>Incubate for 15-30min</LI><LI>Centrifuge lysate at 12000 x g for 15-30 min at 4 degrees C. Collect the supernatant.</LI><LI>Save 20 ul of the lysate for SDS-PAGE analysis</LI><LI>Equilibrate the Ni-NTA spin column with 600 ul buffer NPl-10. Centrifuge for 2 minutes at 2900 rpm. Spin without lid to ensure it is done in two minutes</LI><LI>Load up to 600 ul of the cleared lysate containing the His-Tagged protein onto the (pre-equilibrated) Ni-NTA spin columns. Spin for 5 minutes at about 1600 rpm, and collect the flow-through. To ensure efficient binding, don’t go higher than 1600 rpm, though do it with an open lid</LI><LI>Wash the column twice with 600 ul Buffer NPl-20. Centrifuge for 2 min. at about 2900 rpm. Save the flow-through for analysis by SDS-PAGE</LI><LI>Elute the protein twice with 300ul Buffer NPl-500. Centrifuge for 2 minutes at about 2900, and collect the eluate. Most of the protein will be in the first eluate. The rest will be in the second. 300ul each time</LI></OL></DIV><BUTTON class="dates">Protein Resuspension and Protein Gel Electrophoresis Protocol</BUTTON><DIV class="accordionsCalender"><OL><LI>Resuspend protein sample in a 1:1 ratio of sample to cracking buffer</LI><LI>Place resuspended sample in boiling water for 5 minutes</LI><LI>Load samples into protein gel</LI><LI>Run at 100v</LI><LI>Run until the gel is about 75-80% of the way down the gel</LI><LI>Remove gel from electrodes and plastic casing</LI><LI>Wash gel with purified water about 3 times, agitating the vessel the gel and water are in slightly to wash thoroughly, while not tearing the gel</LI><LI>Stain the gel with Simplyblue dye</LI></OL></DIV><BUTTON class="dates">In Vitro Transcription Translation</BUTTON><DIV class="accordionsCalender"><P>Protocol using NEB’s PURExpress kit-</P><P><A href="https://www.neb.com/protocols/0001/01/01/protein-synthesis-reaction-using-purexpress-e6800">https://www.neb.com/protocols/0001/01/01/protein-synthesis-reaction-using-purexpress-e6800</A></P><P>Changes made: RNA Polymerase was added during step two. A control sample was made using no RNA Polymerase.</P></DIV><BUTTON class="dates">PCR Amplification</BUTTON><DIV class="accordionsCalender"><P>Mix together:</P><OL><LI>35.4 uL Water</LI><LI>15 uL 5x Phusion Buffer</LI><LI>1 uL 10 mM dNTPs</LI><LI>.8 uL 30 mM Forward Primer</LI><LI>.8 uL 30 mM Reverse Primer</LI><LI>.5 uL enzyme (Phusion taq)</LI><LI>2 uL Template</LI></OL><P>Settings</P><P>1 Cycle</P><OL><LI>98°C - 30s</LI></OL><P>35 Cycles</P><OL><LI>98°C - 10s</LI><LI>55°C - 30s</LI><LI>72°C - 2min</LI></OL><P>1 Cycle</P><UL><LI>72°C - 10min</LI></UL></DIV><BUTTON class="dates">Flourocine Diacetate Enzyme Activity Assay</BUTTON><DIV class="accordionsCalender"><P>Enzyme activity was assayed using a modification of the fluorocine diacetate assay for hydrolytic enzymes (Gamble and Muriana 2007). 5ul of in vitro transcription/translation product was mixed with 250ul of phospate buffered saline containing 1 mM flourescine diacetate  in wells of a Tecan black Tecan micro titer plate. Flouresence generated by cleavage of the acetate groups was measured using a Tecan Genios plate reader with excitation at 485nM and emission at 535 nM. </P></DIV><BUTTON class="dates">Allergenicity Testing Protocol</BUTTON><DIV class="accordionsCalender"><P>Getting protein Sequence:</P><OL><LI>Use <A href="http://www.fr33.net/translator.php">http://www.fr33.net/translator.php</A> to convert nucleotide sequence to protein sequences</LI><LI>Plug in output sequences into NCBI BLAST to verify which protein sequence is the intended sequence.</LI></OL><P>Examining Allergenicity</P><OL><LI>Copy nucleotide sequence for the gene that is to be tested.</LI><LI> Click on a new tab, and go onto Translator by clicking this link: <A href="http://www.fr33.net/translator.php">http://www.fr33.net/translator.php</A>. Paste the nucleotide sequence into the box and press translate. This will change your nucleotides into amino acids. Copy this amino acid chain UNTIL the first asterisk (leave out the asterick).Copy this sequence.</LI><LI>Then, click on a new tab, and go onto the NCIB Blast by clicking this link: <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins</A>. Paste the amino acid chain in the box and press BLAST. This could take a couple seconds for the results to come up. This is used to verify the identity of the protein that is going to be used later in the protocol. The identity of the amino acid chain can be seen by comparing the first result of the NCIB Blast matches the same name as the biobrick name on the IGEM website. If it does match then continue to the next step. If it does not check another translation found in part 2.</LI><LI>Click new tab, and go onto AllergenOnline by following this link: <A href="http://www.allergenonline.org/index.shtml">http://www.allergenonline.org/index.shtml</A></LI><LI> Press Search for full-length alignments by FASTA, and copy and paste your amino acid chain (The same amino acid chain from step 3) into the box. Make sure the bubble that says &quot;Full Fasta&quot; is hit and press submit.</LI><LI>This will open up another window. Look at the % identity and % of similarity. If there is a % identity higher than 50% than, the biobrick potentially can be an allergen. If not, then your biobrick is less likely to be an allergen.</LI><LI>Now, we will go back to the main page of the AllergenOnline website by clicking this link: <A href="http://www.allergenonline.org/index.shtml">http://www.allergenonline.org/index.shtml</A>. Then, press Search for 80 amino acid alignments by FASTFA.</LI><LI>Paste the amino acid chain (the same chain from steps 3, and 5) into the box and make sure the second bubble is filled in called sliding 80mer window and press submit.
</LI><LI>The results will open up in a new tab again, and look at and record the % identity and % similarity in the same google document. If the % identity is less than 35%, then the biobrick is less likely to be allergens.</LI></OL></DIV></DIV></SECTION><SECTION id="Footer" class="footerSection"><H2>
	Sponsors
			</H2><H4>
				The Baltimore Bio-Crew thanks our sponsors for their generous support of our team that made our project and travel to the Jamboree possible. Thank you!
			</H4></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>