Team:Baltimore Bio-Crew/Notebook
Home Team Project Results Design Notebook Composite Part Safety HP Silver Experiment Attribution Collaborations Interlab Contributions
July
August
September
October
NoteBook
July
7/01/17
Math:
Worked on getting set up to start math modeling of plastic degradation. Ready to start next week
Found possible alternatives to our plastic degrading enzymes
Collaboration: College Park mentoring for plastic degrading simulation
Wiki:
Created HTML for website from template (not implemented into wiki yet)
Had difficulties getting the template code to work with the pre-set wiki code
Emailed UMaryland team to ask for help with wiki programming.
Gene Design:
Designed genes with Secretion Tag and Periplasmic Tag
Optimized codons
Waiting for genes to be shipped
Lab:
Ran 30 colony PCR reactions
Ready to run a gel on the PCR and send the successful transformations in for gene sequencing
Outreach and Fundraising:
Created template for emailing companies
Reached out to aquarium, Henry Hall program
Contacted local high school for class time discussion
7/06/17
Lab:
Prepared all 30 colony PCR samples to run on a gel
Ran 16 samples, one possible success found
Other 14 samples ready to be run on Saturday
7/08/17
Interlab:
signed up
made 8 chloramphenicol plates
Linearized Backbone Protocol:
learned the anatomy of the different vectors
performed restriction digest on vectors
Fundraising:
found more companies to reach out to
emailed enzyme assay companies about giving us a discount on assaying materials
Our Constructs:
ran PCR samples 17-30 on gel. Currently waiting for results
Wiki:
learned how to use iGEM wiki templates to link to more code files
added navigation bar
experimented with the code to learn more about how the iGEM wikis function
7/13/17
Interlab:
Decided to use Kit Plate 7
Resuspended 8 DNA samples from plate
Samples used:
Positive Control (BBa_I20270): well 21B
Negative Control (BBa_R0040): well 21D
Test Device 1 (BBa_J364000): well 21F
Test Device 2 (BBa_J364001): well 21H
Test Device 3 (BBa_J364002): well 21J
Test Device 4 (BBa_J364003): well 21L
Test Device 5 (BBa_J364004): well 21N
Test Device 6 (BBa_J364005): well 21P
Set up transformations for the 8 samples (transformations later found to be mostly unsuccessful. Did not produce many colonies)
7/15/17
Math:
learned basics of enzyme kinetics
started downloading MATLab
Lab:
ligated new gBlocks with both vectors
transformed gBlocks
made more media for plates
Interlab:
re-transformed all 8 samples
Wiki:
Uploaded home page
worked on color theme and logo ideas
still need to get the rest of the pages up
Waxworms:
students came in and measured waxworms
August
8/03/17
Lab
Restriction digest of esterase, lipase, C3, and A3 were lost. SO, we started all over again (hopefully for the last time) and did a restriction digest of esterase and lipase again. Then, found C3 and A3 digested from a random previous restriction digestion.
ran this restriction digest on a gel, and it looked good
did a PCR of the esterase and lipase stock solutions to produce more.
8/05/17
Lab
Used nanodrop to determine the DNA concentrations of our restricted digest of esterase, lipase, A3, and C3. Data is in Shatera’s notebook. We did this to determine the ratio of microliters of the vector and DNA insert to use for ligation.
Worked out the ratio for ligations using stoichiometry
Ligated using ratios.
Transformed bacteria by heat shocking. We have two groups of the ligation and two groups of the transformed bacteria. After incubating the ligations in room temp for 30 min, the ligations were incubated with some of the bacteria (heat shocked for transformation). Then, after an hour, the bacteria were plated and grown. Then, with the other half of the ligations, we continued to incubate the ligations overnight at 44 degrees after sitting for 30 minutes in room temp. Then, we would transform the bacteria the next day. Then, after an hour, they should be plated to grow.
Presentation
worked on putting together an outline of the presentation
8/10/17
Lab
Bacteria Colony PCR- we finally got positives!
Isolated plasmid DNA from colonies to screen as well (in case the PCRs aren’t working).
8/12/17
Made DNA stocks of positive clones which means that we did plasmid isolation with the positive colonies and used nanodrop to determine concentrations a purity of plasmid. Then, ethanol and salts were added to the isolated plasmids to increase concentration. The plasmids were sent to sequencing to Johns Hopkins.
Worked on presentation
8/17/17
Took the tour of the inner harbor and the trash wheel, and learned about the efforts to clean the harbor
8/19/17
worked on completing the presentation slides and script
8/26/17
worked on completing the presentation slides and script
September
9/02/17
Lab:
Recloning the Lipase:
PCR Amplified: lipase g-block, previously digested lipase, C3 vector from J04450 plasmid, and A3 vector from J04450 plasmid
Miniprepped A3 + esterase plasmids to be used as a control for the PCR gel
Ran PCR gel (1 ul loading dye in each sample). Gel Key:
Ladder (1 ul)
C3 vector (5 ul)
A3 vector (5 ul)
lipase from previous digest (5 ul)
lipase from g-block (5 ul)
PCR purified all samples
Ran Nanodrop on PCR purification results. Nanodrop results:
C3: 16.4ng/ul
A3: 19.1ng/ul
lipase digested: 48.1ng/ul
lipase g-block: 25.0 ng/ul
1st esterase A3 plasmid miniprep: 30.4ng/ul
2nd esterase A3 plasmid miniprep: 52.6ng/ul
Diluted all samples to 10ng/ul
Ran restriction digest on all samples
Purified esterase enzymes
Ran protein gel of esterase enzymes. Gel key:
standard 20ul
un-induced 20ul
induced 20ul
un-induced 40ul
induced 40ul
Working Interlab- Must start over the initial protocol with the spectrometer and incubation for 2, 4, 6 hours.
Presentation:
rehersed
Wiki:
worked on design and color scheme
9/06/17
Purification of Protein and Interlab:
Grew up transformed bacteria that we know have our esterase gene! We grew the bacteria in 5mL overnight cultures. We used PSB1A3 + number 4 and PSB1A3 + number 6. PSB1C3 + number 5 did not grow.
Then, we added 2mL of overnight culture into 50 mL of LB AMP and we let it grow. Then, we checked the OD of our cultures so it could get to .6:
A4
.140 - 10:15
.402 - 11:15
.67 - 12:15
A6
.144- 10:15
.385 - 11:15
.740 - 12:15
We induced our four flasks. 2 were induced with 1 mM IPTG 62.5 microliter of .8 M stock. Then, 2 were not induced
After each time point of 0, 1, 2, 3, 4 in a shaker, we obtained 6 mL from all four flasks. We put one mL of bacteria in 4 microcentrifuge tubes from each flask. Then, we placed 5mL from all four flasks into four tubes. We centrifuges all eight tubes in centrifuge for 10 min at 400 speed to form pellets of cells. Then, the microcentrifuge tubes will be used in a gel. Then, the tubes will be frozen and later lysed and purified using a nickel column to verify the length of proteins; this will happen on another day. For now, the pellets are frozen. The group of samples that will be used for the gel will be ran on a gel another day as well.
9/16/17
Interlab:
Started on cell measurement protocol
Protocol was followed incorrectly and interlab would need to be re-done later
Lab:
Ran protein gel of clones:
A3+Esterase #4
A3+Esterase #6
9/21/17
Lab:
Ran colony PCR of Lipase+C3 plasmid
9/22/17
Learned more about micro plastics at a seminar at the Baltimore Underground Science Space
9/23/17
Started on interlab again:
Took several timepoints:
t0 = 10:30
t1 = 12:30
t2 = 2:30
t3 = 4:30
measured florescence at each timepoint and recorded it in the provided spreadsheet
Lab:
ran protein gel for Esterase+C3 #5. Loaded 10ul of each sample, 5ul ladder. Gel Key:
protein ladder (NEB Broadspectrum)
psb1C3+Esterase #5 uninduced timepoint 4hr (sample 2 test)
psb1C3+Esterase #5 uninduced timepoint 4hr (sample 3 test)
psb1C3+Esterase #5 uninduced timepoint 4hr (Elute 1)
psb1C3+Esterase #5 uninduced timepoint 4hr (Elute 2)
psb1C3+Esterase #5 induced timepoint 4hr (test sample 1)
psb1C3+Esterase #5 induced timepoint 4hr (test sample 2)
psb1C3+Esterase #5 induced timepoint 4hr (test sample 3)
psb1C3+Esterase #5 induced timepoint 4hr (Elute 1)
psb1C3+Esterase #5 induced timepoint 4hr (Elute2)
9/28/17
Interlab:
OD600 reference point data lost. Went through protocol again.
Submitted interlab data. not accepted because we didn't have time to do enough replicates.
9/30/17
Due to interlab being no longer an option to complete our requirements, we considered other options to meet these requirements.
for our contribution requirement, we decided to write a protocol for testing for allergenicity in different proteins. This would make it easy for other teams to see if their biobricks would produce proteins that could contain allergens, and allow us to characterize other parts by testing for allergens in them.
Lab:
Because our protein gels were consistently negative for esterase, we hypothesized that the enzyme was toxic to E.coli. To test this, we decided to try transcribing and translating our esterase constructs using a cell free kit. We also wanted to sequence our biobricks by themselves, so that we could be sure that there were no errors in them before we started. Because of this, we needed to PCR amplify our biobricks to get more DNA to work with. PCR amplified:
esterase biobrick (2017)
lipase biobrick (2017)
esterase biobrick (2016)
lipase biobrick (2016)
Ran gel of PCR
October
10/05/17
Lab:
gel showed PCR to be unsuccessful. The wrong primers were used to amplify.
Re-did PCR with correct primers
Ran gel of PCR. Used 1ul of each ladder and 10ul of each sample. Gel key:
1kb ladder
X
Lipase 2017 1
Lipase 2017 2
X
Esterase 2017 1
Esterase 2017 2
X
Lipase 2016 1
Lipase 2016 2
X
Esterase 2016 1
Esterase 2016 2
X
100bp ladder
10/07/17
Presented to the public and the UMaryland iGEM team for Baltimore Innovation Week. Received feedback on project and presentaion
Lab:
Gel purified samples L1, L2, E1, and E2
Ran a gel with gel purified samples. Ran 6ul of each sample. Gel key:
ladder
X
L1
L2
X
E1
E2
Took nanodrop concentrations of the gel purified samples.
Despite unreliable readings from the nano drop, all of the results for the gel were positive.
10/14/17
helped run a table at the FamFest event at the Y, and talked about synthetic biology with the public
10/19/17
Lab:
The concentrations of three samples, were taken on a nano drop. The two unmodified biobricks from this year, L1 and E1, did not have a reliable measurement on the nano drop. However, the successful plasmid from this year, CE 5, did have a good nano drop concentration reading. Because of this, we decided to only run the cell free kit on the CE 5 plasmid.
Sample CE 5 was transcribed and translated in-vitro using a cell free kit.
10/21/17
Lab:
Ran protein gel of the cell free protein expression samples and the time course samples for the positive control bacteria and induced engineered E.coli, some induced with lactose and some induced with IPTG.
Gel 1 Key:
ladder
control 2
control 3
control4
CE5-1 IPTG 2hr
CE5-2 IPTG 2hr
CE5-3 IPTG 2hr
CE5-1 lac 2hr
CE5-2 lac 2hr
CE5-3 lac 2hr
Gel 2 Key:
CE5-1 IPTG 5hr
CE5-2 IPTG 5hr
CE5-3 IPTG 5hr
ladder
CE5-1 lac 5hr
CE5-2 lac 5hr
CE5-3 lac 5hr
X
X
X
Gel 3 Key:
CE5-1 IPTG
CE5-2 IPTG
CE5-3 IPTG
CE5-1 lac
CE5-2 lac
CE5-3 lac
ladder
X
CE5 (in-vitro)
positive control
10/26/17
Wiki:
compiled a list of what still needed to be done
Completed Judging form and worked on finishing up poster
10/28/17
Lab:
Completed flouricine diacetate enzyme assay on the products of cell free expression by the Esterase and Lipase biobricks. Showed positive for enzymes!
Wiki:
worked on getting finished up
put together protocols page content, project description content, and notebook page
Sponsors
The Baltimore Bio-Crew thanks our sponsors for their generous support of our team that made our project and travel to the Jamboree possible. Thank you!
