"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/ProjectChassis
From 2014.igem.org
2014 UBC iGEM
Toggle navigation
Home
Project
Overview
Evolving
Biomining
Achievements
Modelling
Mutagenesis
Adsorptions
Human Practices
BioBricks
Notebook
Protocols
Lab Book
Safety
Team
Our Team
Gallery
Sponsors
Gene Evolution Using an Orthogonal Replication System
Evolution is a powerful driving force in biology. It created all the phenomenal life-forms we can find on our planet. Constant mutation and selection processes of genes in living organisms eventually lead to proteins with adapted and improved characteristics.
Copying this process is a desirable laboratory application to generate genetic material or gene products with new or enhanced functions. Generation of enzyme variants is important in basic enzymology studies and is a powerful tool to engineer enzymes for biotechnology applications. However, evolution in organisms is generally slow because the mutation rate is inversely related to the genome size (1,2). Also, natural evolution is not targeted since mutations in the genome accumulate randomly (3).
We were interested in protein evolution because we aimed to improve our mineral-binding peptides for our biomining strategy. Laboratory evolution would allow us to develop peptides with novel binding properties to isolate various minerals for mining. Moreover, such a technique would be a powerful tool to engineer genes in the registry of biological parts.
There are laboratory techniques to bypass the natural evolution constraints and accelerate the mutagenesis of a single gene. This traditional directed evolution is an ex vivo method and is achieved using error-prone PCR (3). However, this process is labour-intensive as it includes multiple rounds of DNA extraction, mutation and transformation (3). Furthermore, it is random and by no means continuous, i.e. single mutations in a gene do not necessarily add up to a beneficial function. Therefore, a large number of clones need to be screened to identify a variant with the desired improvement. Targeted mutagenesis inside cells is advantageous because positive genetic selection eliminates defective mutants from the mutant pool while allowing the accumulation of mutations with a positive effect on the protein of interest and thereby dramatically increases the number of mutations that can be analyzed (4).
We constructed an orthogonal replication system in bacteria that allows the replication of a plasmid, containing the gene of interest, by an error-prone DNA polymerase (DNAP), which does not affect the genomic DNA. To ensure genomic DNA does not get affected, we used two separate plasmids to introduce the error prone replication system. The first plasmid contains a bacterial origin of replication along with T7 bacteriophage replication machinery while the second plasmid contains the gene we are trying to mutagenize our ‘gene of interest’ (GOI) and a viral origin of replication. Other in vivo mutagenesis strategies to evolve genes have been reported. But these techniques do not prevent mutagenesis of genomic DNA in the host or have been applied in other organisms like yeast (3, 4).
As a proof of principle we decided to test our system in E. coli and the same principle can be implemented in other bacteria, like our mineral-binding strain Caulobacter. We utilize the minimal parts of the T7 bacteriophage DNA replication machinery to enable orthogonal replication of a recombinant plasmid. This machinery includes the mentioned DNAP, a primase, a single-stranded DNA-binding protein (ssDNA-BP) and a RNA polymerase (RNAP). After transformation of the recombinant plasmid, containing the viral origin of replication, into the engineered E. coli strain (Fig. 1A + B) the T7 replication machinery can be induced and the mutation process of the GOI is initiated (Fig. 1C + D). The plasmid-containing E. coli can be cultivated under selective conditions. This allows for simultaneous mutation and selection. For example, in our biomining assay, strains expressing mineral-binding peptides on their surface can be cultivated in the presence of the mineral that is intended to be isolated. Multiple rounds of mutation will eventually lead to peptides that enable efficient selection of the mineral. The mutants that bind best can be isolated in a screen and the plasmid with the mutated GOI can be extracted from the cells (Fig. 1E).
Our proposed strategy to evolve genes saves the experimenter tremendous amount of time compared to traditional ex vivo methods and is superior to other methods reported for bacteria.
Figure 1. Schematic overview of the orthogonal replication process.
Results
For the construction of the T7 replication machinery, we cloned the DNAP, RNAP, ssDNA-BP and the primase into the pSB1C3 vector. All genes are under the control of the pBAD promoter to allow for titratable induction with arabinose. To test the expression of the T7 genes we transformed the plasmids into DH5α cells. Cloning of the primase, unfortunately, was troublesome and our expression experiments showed that the primase was defective. The DNAP and ssDNA-BP expressed to high levels (Fig 2). RNAP expression was not detectable on a SDS-Gel. We are planning to assemble the genes together with promoter and terminator on the pSB1C3 plasmids.  The JH-pSB2K3 with its T7 origin sequence will serve as vector for the GOI and will be transformed into cells with the T7 replication machinery.
Figure 2. Expression of T7 replication machinery genes.
References
1. Drake, J.W. Proc. Natl. Acad. Sci. USA 88, 7160–7164 (1991).
2. Nowak, M.A. Trends Ecol. Evol. 7, 118–121 (1992).
3. Ravikumar, A. Nature Chem. Biol. 10, 175-177 (2014).
4. Camps, M. Proc. Natl. Acad. Sci. USA 100, 9727–9732 (2003).
© 2014 UBC iGEM
Retrieved from " http://2014.igem.org/Team:British_Columbia/ProjectChassis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
