"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/Modelling/Mutagenesis
From 2014.igem.org
2014 UBC iGEM
Toggle navigation
Home
Project
Overview
Evolving
Biomining
Achievements
Modelling
Mutagenesis
Adsorptions
Human Practices
BioBricks
Notebook
Protocols
Lab Book
Safety
Team
Our Team
Gallery
Sponsors
Mutagenesis
The flow diagram shows the basic work flow for directed evolution of a gene. Bacteria continuously replicate and in every round of replication the DNA mutates slightly. After each round of replication, the cells are screened and those with undesirable phenotype are discarded. This is repeated many times to allow for the gene of interest to mutate then the cells are plated. After plating the cells, each cell will grow into a colony. During cell growth, the error prone polymerase is still active so the colony grown up for sequencing will also contain mutations which could interfere with sequencing. The aim of this model is to determine the error rate at which a standard size colony would become too diverse for the sequence of the initial cell plated to be determined.
A probabilistic matlab model was created to simulate the mutation process that occurs as a single cell grows into a colony. The model begins with the root of a binary tree which represents the single cell being grown into a colony. The basics of the model is illustrated below.
Afterwards for every base, the ratio of mutated to un-mutated cells is obtained. For our demonstration, a 20% cut off was used. When 20% of a base in the colony is mutated, the original base is determined to be retrievable from a sequencer. The graphs below plot the number of un-readable bases in a 1000bp gene against the polymerase error rate. The number of un-readable bases is averaged over 500 trials. A polymerase error rate of 0.1 would mean there is a 10% chance for a each and every base to be mutated on each cell replication.
Looking at the narrow range graph of error rates from 0 to 1.2E-4 which ranges from a normal polymerase with error rate of about 1e-5 to a mutated polymerase with error rate of about 1e-4. The graph shows that in a 1000bp sequence less than 1 base is estimated to be un-sequencable.  The conclusion is that sequence obtained this way is still representative of the initial cell plated for reasonable polymerase errors rates. To significantly mutate the gene of interest, many rounds of cell replication and screening is required. The number of rounds of cell replication from the plating is insignificant compared to the mutation process.
© 2014 UBC iGEM
Retrieved from " http://2014.igem.org/Team:British_Columbia/Modelling/Mutagenesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
