<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hangzhou-H14Z_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hangzhou-H14Z/Practices</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2015_IGEM_H14Zteam_Wiki</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z" class="scroll">Home</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Intro" class="scroll">intro</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Project" class="scroll">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Team" class="scroll">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Collaborations " class="scroll">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Attributions" class="scroll">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Practices" class="scroll">Publicity</A></LI><LI><A href="https://2015.igem.org/Team:Hangzhou-H14Z/Parts" class="scroll">Parts</A></LI></UL></DIV></DIV><DIV id="tf-home"><DIV class="overlay"><DIV class="container"><DIV class="content-heading text-center"><H1>The best way ever to preserve lithic relics </H1><P class="lead">an innovative biochemical method to resolve erosion. </P><A href="#tf-intro" class="scroll goto-btn text-uppercase">View Our Works</A></DIV></DIV></DIV></DIV><DIV id="tf-intro"><DIV class="container"><DIV class="row"><DIV class=" col-md-offset-2 col-lg-offset-0 col-lg-11"><P>The inscriptions on Precipice are one of the most common and fundamental forms of traditional Chinese relics. Due to having been carved in open areas plus the exposure to sunshine and raindrops, which is often the case for most of them,the Inscriptions are weathered readily. Currently, there are quite a few methods to resolve this phenomenon but some of them are not used properly which occasionally cause the damage for the Inscriptions. Our research project aims at using microorganism method, which is to put a plasmid with the ability to convert bio-oriented oxaloacetic acid to oxalate acid into the cell of E.coil, inducing the cell and the product will react with one component of the Inscriptions-- calcium carbonate-- to generate nearly insoluble calcium oxalate monohydrate to adhere to the Inscriptions for protection. Another E.coil with the ablility to secrete lipase has also been constructed; we aim to add this cell when we are cleansing the Inscriptions to resolve the oil substances on the surface of them. By constructing these products, we are able to protect the relics either from human errors as cleaning and protecting or from natural damnification.  </P></DIV></DIV></DIV><DIV class="container"><DIV class="section-header"><H2>some  <SPAN class="highlight"><STRONG><FONT color="#00C459">destroyed  </FONT></STRONG></SPAN>  inscriptions   on    Precipice</H2><H5><EM>photos were took within Hangzhou</EM></H5></DIV><DIV id="team" class="owl-carousel owl-theme text-center"><DIV class="item"><DIV class="hover-bg"><DIV class="team-detail text-center"><H3>The White Tower</H3><P class="text-uppercase">a famous religious symbol in Hangzhou</P></DIV></DIV></DIV><DIV class="item"><DIV class="hover-bg"><DIV class="team-detail text-center"><H3>Figure of Buddha</H3><P class="text-uppercase">a symbol of Buddhism</P></DIV></DIV></DIV><DIV class="item"><DIV class="hover-bg"><DIV class="team-detail text-center"><H3>Woman in palace</H3><P class="text-uppercase">stone inscription</P></DIV></DIV></DIV><DIV class="item"><DIV class="hover-bg"><DIV class="team-detail text-center"><H3>Statue of MAHAKALA</H3><P class="text-uppercase">Ancient monument</P></DIV></DIV></DIV><DIV class="item"><DIV class="hover-bg"><DIV class="team-detail text-center"><H3>Ancient chinese governor</H3><P class="text-uppercase">stone inscription</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="tf-services"><DIV class="container"><DIV class="section-header"><H2>Project's<SPAN class="highlight"><STRONG> Presentation</STRONG></SPAN></H2><H5><EM>photoes were took within Hangzhou</EM></H5></DIV></DIV><DIV id="feature" class="gray-bg"><DIV class="container"><DIV class="row" role="tabpanel"><DIV class="col-md-4 col-md-offset-1"><UL class="features nav nav-pills nav-stacked" role="tablist"><LI role="presentation" class="active"><A href="#f1" aria-controls="f1" role="tab" data-toggle="tab">
                                    Project's background </A></LI><LI role="presentation"><A href="#f4" aria-controls="f4" role="tab" data-toggle="tab">
                                    Principles of experiments</A></LI><LI role="presentation"><A href="#f5" aria-controls="f5" role="tab" data-toggle="tab">
                                    Procedures in detail           part I</A></LI><LI role="presentation"><A href="#f8" aria-controls="f8" role="tab" data-toggle="tab">
                                    Procedures in detail           part II</A></LI><LI role="presentation"><A href="#f9" aria-controls="f9" role="tab" data-toggle="tab">                          Results of Experiments &amp; Analysis</A></LI><LI role="presentation"><A href="#f6" aria-controls="f6" role="tab" data-toggle="tab">
                                   Mathimatical models</A></LI><LI role="presentation"><A href="#f7" aria-controls="f7" role="tab" data-toggle="tab">
                                  Notebook</A></LI></UL></DIV><DIV class="col-md-6"><DIV class="tab-content features-content"><DIV role="tabpanel" class="tab-pane fade in active" id="f1"><H4>Project's background</H4><P>    The area near Qiantang River has a profound history of wealth. Hangzhou, in the heart of this area, situated many astonishing lithic cultural relics. Most of them were made during The Ten Kingdoms (A.D. 891-A.D. 979) and Song Dynasty (A.D. 960- A.D. 1279). Not only do those lithic relics represent the cutting edge of stone carving technology in contemporary time, but also they provide historians and archaeologists with solid historical evidence. </P><P>    
    There are three main types of lithic relics in Hangzhou; they are stone sculptures, stone towers also known as Dhavaja, and stone inscriptions. The stone sculptures are primarily represented by Feilai Peak (飞来峰), Ciyun Ridge (慈云岭), and Statue of Mahakala in Baocheng Temple (宝成寺麻曷葛剌). Stone Towers are primarily represented by White Tower (白塔). Stone inscriptions are primarily represented by Hetingwuhe Fubei on Lonely Mountain (孤山放鹤亭舞鹤赋碑).</P><P>     Plus, Feilai Peak and White Tower are National Key Historical Relics. 
    Most lithic relics encounter physical, chemical and biological interaction, resulting in weathering corrosion, pulverisation lesion, cracking, contamination, and colour mottling. Those phenomena jeopardise the value of lithic relics and even lithic relics themselves, and the lithic relics in Hangzhou are not of exceptions. </P><P>    
    In 2002, some of the classic lithic relics were inspected and it was found that the surface of Feilai Peak Sculpture was greatly ruined (Figure 1), and there is leaking on Ciyun Ridge Sculpture (Figure 2)..</P><P>    Meanwhile, Cultural Relics Department of China has been trying to maintain and recover the lithic relics. Since 2002, Cultural Relics Department of China mainly has been removing surface deposit, depleting pounding, strengthening crag, and changing flora; the government washed Feilai Peak Sculpture for many times (Figure 3).Government built a top above the Ciyun Peak Sculpture, this relic no longer had to expose to sun and rain henceforward (Figure 4).</P><P>    White Tower was comprehensively cleaned and strengthened by silicone by Zhejiang University of Technology (Figure 5).</P><P>     The power plants and railways were removed. During the washing, the staff intentionally left part of this tower intact, and made an contrast for further research of anti-contamination and anti-weathering (Figure 5-3, 5-4)</P><P>    However, 2015 field research shows most external lithic murals, statue, and towers still suffer from damage. Some even have distraction in main stone body. Meanwhile, there grows moss. Indoor stone inscriptions though are visible; there are obvious and irremovable cracks. </P><P>    
    Today, most preservation of lithic relics employ physical, chemical, or engineer approaches no matter where. It has been proven by reality that each of those approaches has fatal disadvantages for long-standing preservation of lithic relics, and those disadvantages would even lead to the ruin of lithic relics.</P><P>    Since forever, there are two categories of methods to preserve-- organic and inorganic. Organic materials cannot reach the standard of time to preserve relics such as epoxy resin, acrylic resin, silicone, etc.; inorganic materials would form hard shell that is not compatible or even react with the lithic relics, such as lime, barium hydroxide, and sulphuric acid. </P><P>    
    In conclusion, finding a long-standing, inexpensive, and efficient method to preserve lithic relics is an impending issue. </P><DIV id="myCarousel" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>Figure 1</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 2</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 3-1</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 3-2</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 4-1</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 4-2</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 5-1</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 5-2</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 5-3</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 5-4</P></CENTER></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="f4"><H4>Principles of experiments</H4><P>    The documents cited shows that even after natural weathering, biological decomposition, and acid rain erosion for a time period of time, some stone inscriptions, which are on rock surface containing calcium, remain in good condition because there is a thin layer of naturally formed semitransparent mineralised film that is hydrophilic and compact on such rock surface.</P><P>     In some part under the fine film, knife scratches on the stone from over a thousand years ago are still visible indistinctly. According to solid analysis, natural mineralised film is primarily composed of calcium oxalate monohydrate [Ca(COO)2·H2O] whose process of composition is interfered by microorganism.</P><P>    
    There are some advantages of calcium oxalate being a film as follows,</P><P>        1) In water, especially in acidulous water, the solubility of calcium oxalate is one order of magnitude smaller than normal calcium carbonate.</P><P>        2) Organically oriented calcium oxalate crystal's particles are fine and compact, which prevent the infiltration of toxic and harmful materials. </P><P>        3) The system can't be easily broken by external forces in that there is no distinct boundary between calcium carbonate rock and calcium oxalate film.</P><P>        So, this research project has chosen E. coli that has been transcribed with lipase to clean some stone relics that are covered with oil; then, we use E. coli, with transcription, that can secrete oxalate acid upon the stone relics. Therefore, there forms calcium oxalate to be protection film in order to preserve lithic relics.</P><P>        If we use chemical methods to generate calcium oxalate, we have to immerse the lithic relics in the chemical liquid; however, it is not suitable for the preservation of large-scale outdoor lithic relics. In fact, the natural existing calcium oxalate comes from the reaction of oxalate acid generated by microorganism and calcium compound on the surface of stone. This research project will utilise genetic engineering methods to &quot;graft&quot; the gene that generates calcium oxalate into Escherichia coli (E. coil).</P><P>    Then oxalate acid reacts with calcium lithic relics to generate calcium oxalate, which stimulates its natural formation process. Finally, there forms a fine layer to protect the lithic relics. </P><DIV id="mlyCarouselt" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>[Ca(COO)2·H2O]'s crystal</P></CENTER></DIV><DIV class="item"><CENTER><P>[Ca(COO)2·H2O]'s SEM Picture</P></CENTER></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="f5"><H4>Procedures in detail</H4><P>1.1  Experiment reagents and equipment
</P><P>    1.1.1 Target genes</P><P>        (1) Lipase: lipase is the target gene segment of this experiment, and the gene sequence is synthesised by IDT Inc. </P><P>        (2)  Oxaloacetate acetylhydrolase, abbreviation OAH, is the target gene segment of these experiments. The main function of OAH is to transfer oxaloacetic acid to oxalate acid; thus the oxalate acid reacts with the calcium ion to form calcium oxalate that preserves the relics. The target gene segment is synthesised by IDT Inc. after found in gene pool. 
 </P><P>    1.1.2 Strain and Plasmid
This experiment uses plasmid pET-28a, and the E. coli is TOP-10 competent cell, provided by TIANGEN Biotech CO., LTD.
</P><P>    1.1.3 Enzyme and Reagents This experiment uses restriction enzyme provided by NEB; T4 DNA Ligase, TIANprep Mini Plasmid Kit, and TIANquick Midi Purification Kit are provided by Tiangen Biotech CO., LTD.
</P><P>    1.1.4 PCR Primer</P><P>        (1) Lipase:
    According to the sequences and the MCS (multiple cloning sites) to the destination vector of the Lipase, XhoI and NcoI are chosen to be the enzyme cutting cites for the inserting enzyme cutting cites. There form the following primers: 
</P><P>    Sense primer: 5’-AAGCAAGCCCATGGAATTTGTAA-3’ (Underlined proportion is Ncoi enzyme cutting cite)
</P><P>    Anti-sense primer: 5’-TTTTGTTGCTCGAGTTAATTCG-3’ (Underlined proportion is Xhoi enzyme cutting cite)
</P><P>        
(2) OAH:
  According to the sequence detected by OAH, the primer is synthesised by the Invireogen:</P><P>        Sense primer: 5’-ATCATCTTGCATATGGCGTCCACCATTGCTGTT-3’ (Underlined proportion is Ndei enzyme cutting cite)</P><P>         Anti-sense primer: 5’-TTTCCTTTTGCGGCCGCTTAG-3’ (Underlined proportion is Noti enzyme cutting cite)</P><P>    1.1.5  Substrate Medium</P><P>        (1) The experiment contains LB substrate media. Take the example of confecting 200mL; the materials needed are 2g of tryptone, 1g of yeast extract, 4g of sodium chloride, and 200mL of water with 1000× ratio of chloramphenicol. </P><P>        (2) The experiment contains LB substrate media. Take the example of confecting 200mL; the materials needed are 2g of tryptone, 1g of yeast extract, 4g of sodium chloride, and 200mL of water with 1000× ratio of Kanamycin Sulfate.</P><P>        (3) The experiment contains Kanamycin Sulfate substrate media. Take the example of confecting 400mL; the materials needed are 4g of tryptone, 2g of yeast extract, 4g of sodium chloride, 6g AGAR powder and 1.2mL 
</P></DIV><DIV role="tabpanel" class="tab-pane fade" id="f8"><H4>Procedures in detail</H4><P>1.2 Procedures of Experiments</P><P>     1.2.1 PCR Proliferation
</P><P>        Due to high content of G-C base-pairs in target gene segments, GC Buffer is required in PCR proliferation. Refer to Takara's protocols to prepare 200μL of system: 100μL of GC Buffer, 16μL of dntp mixture, 4μL of sense primer, 4μL of anti-sense primer, 4μL of DNA templet, 2μL of Taq, and 70μL of H2O. 60°C-70°C is used for Gradient PCR, the sixth bad is the brightest (Figure 9) thus the temperature of such is best for annealing. In new system, PCR is proceeded in that temperature; finally we get the gene segments with gene linkers. </P><P>    1.2.2 Gel electrophoresis, purification</P><P>    1.2.3 Digestion</P><P>        Put NotI and NdeI restriction endonuclease in gene segment and vector pET-28a respectively. </P><P>    1.2.4</P><P>         Linking and setting the groups Experimental group:plasmid after digestion + target gene + T4 DNA Ligase + buffer
Control group: plasmid after digestion *Experiment conducted under 16°C</P><P>    1.2.5 </P><P>        Use DNA purification kit to extract product after linking; transcribe in TOP-10 competent cell, put on Kanamycin Sulfate substrate media and then culture under 37°C. </P><P>    1.2.6</P><P>        According to the growth of strains, bacterium plasmid is extracted with digesting with two enzymes above. Recheck the correctness of position and size to assure that it has been linked. </P><P>    1.2.7  Sequencing by Sangon Shanghai, China</P><P>    1.2.8  Functional Test of Lipase</P><P>        (1) Fasten the AGAR Plane: 2.5g of AGAR powder, 6.25mL of tributyrin, 18.75mL 3g/dL of polyvinyl alcohol, 2.5mL 1g/dL Victorian Blue, 225mL of H2O. Heat and resolve the AGAR powder, pure into substrate media, chilling. 
  IPTG solution: weigh 1.2g of IPTG and put it into centrifugal tubes. Add 40mL of H2O, resolve, then to the constant volume of 50mL. Use 0.22μm strainer to filter out the other bacteria, then put into refrigerator of -20°C. </P><P>        (2) Transcribe the lipase enzyme plasmid into BL-21 competent cell of E. coli, culture. </P><P>        (3) Pick bacteria into small colonies from growing strains on the cultivation board, which has been configured with a medium, Kanamycin, and join the 0.1 mmol/L IPTG solution to carry on the induction, six hours later to take on the culture medium centrifugal supernatant, get with lipase protein nutrient solution.</P><P>        (4) From fixed AGAR plate with holes, pick bacteria. Burn alcohol burner against the bottom of AGAR to assure it is stick. Put 30μL into the holes, lie horizontally in 30°C incubator for a while and then observe the change of colour. </P><DIV id="mjyCarouselt" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>Figure 9  Gradient PCR Electrophoresis (red column is the brightest)</P></CENTER></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="f9"><H4>Results of Experiments and Analysis</H4><P>1.3 Results of Experiments and Analysis</P><P>     1.3.1	Digestion check
</P><P>        (1) Use NcoI and XhoI to digest plasmid on lipase after PCR. We know our result is correct because two bright parts of plasmid is in the same position and of the same size. 
</P><P>        (2) Use NdeI and NotI to digest plasmid on OAH after PCR. We know our result is correct because two bright parts of plasmid is in the same position and of the same size. (Figure 10)
</P><P>    1.3.2	Sequencing</P><P>        The results of t sequencing of Lipase and OAH are both correct, for more detail, check appendix 1. </P><P>    1.3.3	Functional Test of Lipase
 </P><P>        Let PET-28a-Lipase plasmid be transcribed into BL-21 E. coli (the procedure is the same as that of TOP-10 E. coli), then put on substrate media. Pick E. coli into shake flask for 3 hours. Join IPTG solution (0.1 mmol/L) for abduction for 6 hours. Put into centrifugal machine then supernatant liquid (with lipase) for lipase test. The result shows that the colour has changed besides plasmid with lipase (Figure 11), showing that LipB2 gene is correctly expressed in BL21 E. coli. 
</P><P>         Linking and setting the groups Experimental group:plasmid after digestion + target gene + T4 DNA Ligase + buffer
Control group: plasmid after digestion *Experiment conducted under 16°C</P><P>1.4 Result Analysis and Prospect</P><P>    According to the results of sequencing, actual sequence is identical to designed sequence, which testifies the validity of formulation (appendix 1). In lipase functional test, there is relatively obvious phenomenon under IPTG abduction (Figure 11), which testifies the function of gene. </P><P>    Hence, using this reformed lipase can erase the grease cover on relic surface, and we hope to use OAH reconstruct relic surface. 
</P><P>    Meanwhile, recombinant lipase can be widely used in any field for de-greasing, for example, the de-greasing process of steel surface, pre-treatment of high-fat food disposal, etc. </P><P>    This research project only recombines the lipase without optimising it according to temperature fluctuation, abduction concentration. We hope our project is helpful in every field. </P><DIV id="mtyCarouselt" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>Figure 10  Digestion result test</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 11  LipB2 gene in BL21 E. coli</P></CENTER></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="f6"><H4>Mathimatical models</H4><P>    Since there is no one had done the same experiment as ours before, we assumed the growing pattern is similar as E.coil CVCC249. </P><P>    We got the data of this kind of E.coil from a doctoral dissertation,Comparison of Growth Kinetics and Physiological Characteristics of E.coil CVCC249 under both Batch and Continuous Culture by Huaiqiang Zhang, at page34. We used the logistic function data in this form, which is K=1.937 and r=0.550, and we assumed that N0=0.1( since N0 is initial population of E.coil, we can just simply set this data). </P><P>    According to logistic function , we can get the bacteria population-time function:(see Figure 1) </P><P>        f(t) = 1.937/(1+(1.937-0.1)e^(-0.55t)/0.1)

        f(t) = 1.937/(1+18.37*e^(-0.55t))</P><P>    After that, we assumed that there will be x% of bacteria can present the property we need. Then we can get population of expressed bacteria:</P><P>        fex(t) = x%*1.937/(1+18.37*e^(-0.55t))</P><P>    Because we need the oxalic acid-time function, we assume the speed of oxalic acid secretion of per unit of bacteria is v. So the total speed should be: </P><P>        V(t) = v*x%*1.937/(1+18.37*e^(-0.55t))</P><P>    Since the secretion of oxalic acid should be accumulated, we take the integration of total speed of time. We used WolframAlpha program to find out this integration.</P><P>        ∫1.937/(1+18.37*e^(-0.55t)) dt=3.522*log(18.37+e^(0.55t))+C0</P>        ∫V(t) dt=v*x%*3.522*log(18.37+e^(0.55t))+C

<P>    Because the function should began at (0,0) point, we can find out constant C in function.</P><P>        0=v*x%*3.522*log(18.37+e^(0.55*0))+C


        C=-3.522*log(19.37)*v*x%

        C=-10.438*v*x%

        Substitute C=-10.438*v*x%</P><P>        Soa(t)=v*x%*(3.522*log(18.37+e^(0.55t))-10.438)</P><P>    This is oxalic acid-time function.</P><P>    We also need to find out lipid-time function. Obviously, it’s more complicated than last function, since the decomposition speed cannot be a constant.</P><P>    The fex(t) function should be also available in this part. Starting constructing process, we assumed that initial total lipid is N, lipid that decomposed is n, and the decomposing speed is relate to following factors:</P><P>    k: a constant representing efficiency of lipase

    (N-n): remain amount of lipid

    fex(t): quantity of expressed bacteria</P><P>    And there should be a function meaning that decomposing speed is proportional to the remain amount of lipid and quantity of expressed bacteria:</P><P>        dn/dt=k*(N-n)*fex(t)</P><P>        Solve that differential equation.</P><P>        1/(N-n) dn=k*fex (t) dt

        ∫1/(N-n) dn=k∫fex (t) dt

        -ln(N-n)=k∫f ex(t) dt + C</P><P>    Since we already know that ∫f ex(t) dt= x%*(3.522*log(18.37+e^(0.55t))-10.438) in this condition.</P><P>        -ln(N-n)=k*x%*(3.522*log(18.37+e^(0.55t))-10.438)

        (N-n)=e^(-k*x%*(3.522*log(18.37+e^(0.55t))-10.438))</P><P>    So the lipid-time function, Slp(t), were found:</P><P>        Slp(t)=(N-n)=e^(-k*x%*(3.522*log(18.37+e^(0.55t))-10.438))
</P><DIV id="mayCarouselt" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>Quantity of Oxalic Acid</P></CENTER></DIV><DIV class="item"><CENTER><P>Quantity of Bacteria</P></CENTER></DIV><DIV class="item"><CENTER><P>Figure 1</P></CENTER></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="f7"><H4>Notebook</H4><P> Our origin experiment records</P><DIV id="mpyCarouselt" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active"><CENTER><P>Page 1</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 2</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 3</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 4</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 5</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 6</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 7</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 8</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 9</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 10</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 2</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 2</P></CENTER></DIV><DIV class="item"><CENTER><P>Page 2</P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="tf-works"><DIV class="container"><DIV class="section-header"><H2>INFO   ABOUT  <SPAN class="highlight"><STRONG>Team   MEMBER</STRONG></SPAN></H2><H5><EM>suspend your mouse above each images to learn more about our team member.</EM></H5></DIV><DIV class="text-center"><UL class="list-inline cat"><LI><A href="#" data-filter="*" class="active">Team members</A></LI></UL></DIV></DIV><DIV class="container-fluid"><DIV id="itemsWork" class="row text-center"><DIV class="col-xs-12 col-sm-6 col-md-3 col-lg-3 nopadding brand others"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Zhengdong   Chen</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 col-lg-3 nopadding apps"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Yue Wu</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 col-lg-3 nopadding others brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Shuyang Zhou</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 col-lg-3 nopadding others web"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Haobo Shao</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web others"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Yunfeng Guo</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding app"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Minxing Chen</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Jingcheng Shi</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding app"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Juncheng Luo</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Wei He</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Yuning Gu</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Xiaomei Tang</H10></P></DIV></DIV></DIV></DIV><DIV class="col-xs-12 col-sm-6 col-md-3 nopadding web brand"><DIV class="item"><DIV class="hover-bg"><DIV class="hover-text off"><P><H10>Jie Zheng</H10></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="tf-process"><DIV class="container"><DIV class="section-header"><H2><SPAN class="highlight">Collaborations</SPAN></H2><H5><EM>Our team collaborated with various IGEM teams and other people in this project</EM></H5></DIV></DIV><DIV class="tab-content features-content "><P>                                                                                                                                                                 We collaborated with the collegiate team— Zhejiang University.</P><P>                                                                                                                                                                The students of ZJU helped us in the pre-programm training in the site of Zhejiang Association for Science and Technology.</P><P>                                                                                                                                                                They taught us the basic biology experimental techniques by dyeing colon bacillus in a bio-tech way.</P><P>                                                                                                                                                                Meanwhile, they invited the team instructor of Peking University to introduce us how to understand this competition. </P><P>                                                                                                                                                                Interview with the local experts </P><P>                                                                                                                                                                Since our research projects are interdisciplinary as well as socially-based,</P><P>                                                                                                                                                                 we consulted our ideas with professor in the field about the production of our experiment applying the other fields beyond the cultural artifacts protection. </P><P>                                                                                                                                                                We then communicate with Prof. Zhang, Zhejiang University, whose field is civil-engineering. </P><P>                                                                                                                                                                        Q1: How can the greasing removal apply in civil engineering?</P><P>                                                                                                                                                                        Q2: In the field of civil engineering, are there any other applications? </P><P>                                                                                                                                                                        Q3: How efficient is this method or else than biological method? </P><P>                                                                                                                                                                Based on these three question, Professor gave the following answer to all of these.
</P><P>                                                                                                                                                                Reinforced concrete structure has been widely used in construction engineering, and the corrosion of concrete will cause irreversible damage of the structure, </P><P>                                                                                                                                                                especially in the moist environment, or in the coastal areas; for example the basement engineering, coastal areas of reinforced concrete Bridges.</P><P>                                                                                                                                                                In the dampness and water environment, corrosion speeds faster. At present engineers mainly adopt anticorrosion coating materials, including organic coating, inorganic coating and coating,</P><P>                                                                                                                                                                 the widely application of polyurethane waterproof coating, polymer cement waterproof coating, cement based waterproof coating and organic silicon waterproof coating, etc.</P><P>                                                                                                                                                                However, there is rare application of biological coating in practical. According to the principle, it has certain feasibility, </P><P>                                                                                                                                                                because the principle of chemical coating is the main component of waterproof material, such as sodium silicate, can quickly and effectively concrete structure layer of calcium hydroxide, aluminium, calcium, calcium silicate, </P><P>                                                                                                                                                                to form the wool stoma of inert crystals embedded in concrete,</P><P>                                                                                                                                                                 to airtight micro cracks, thereby increasing the compactness of concrete surface layer and the compressive strength. In the study of the biological mechanism of action of anticorrosion materials and similar is feasible in theory, </P><P>                                                                                                                                                                but in the specific application, also need to further study of the role of timeliness, economy.</P></DIV></DIV><DIV id="tf-pricing"><DIV class="container"><DIV class="section-header"><H2>The Attributions  <SPAN class="highlight"><STRONG>Acknowledgements  </STRONG></SPAN></H2><H5><EM>This investigative project is supported by various well-known institutes, universities and corporations</EM></H5><P>
Professor Luo jianhong, Zhejiang University</P><P>Dr. Yan yinggang, Zhejiang University</P><P>Dr. Ling Zhang, Zhejiang University</P><P>Hangzhou #14 High School Bio-technology Laboratory</P><P>Zhejiang Association For Science and Technology</P><P>Hangzhou Multiscience Lianke Biotech Company (donator)</P><P>Hangzhou # 14 High School (donator)</P></DIV></DIV></DIV><DIV id="tf-propaganda"><DIV class="container"><DIV class="section-header"><H2>Human practices &amp;<SPAN class="highlight"><STRONG> Publicity</STRONG></SPAN></H2><H5><EM>We build a set of pages in our high school's official website to publicize the subject of synthetic biology...</EM></H5></DIV><DIV class="gray-bg"><DIV class="container"><DIV id="process" class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="media process"><DIV class="media-body"><A href="http://ap.h14z.com/igem/index.html"><H4 class="media-heading">The page of H14z's synthetic biology</H4></A><A href="http://ap.h14z.com/igem/index.html"><P>This page was built by Minxing Chen as well. There is a brief introduction about the synthetic biology and it was written in simplified chinese. Also, we put some useful knowledges there such as the bio-switch. Abundant people has visited this website and learnd a lot about synthetic biology and our school.</P></A></DIV></DIV><DIV class="media process"><DIV class="media-body"><A href="http://ap.h14z.com"><H4 class="media-heading">AP center's website</H4></A><A href="http://ap.h14z.com"><P>All of our team members came from Hangzhou #14 highschool's Ap center. And we also put the synthetic biology's link there. Consequently, more students and facilities can see it.</P></A></DIV></DIV><DIV class="media process"><DIV class="media-body"><A href="http://www.h14z.com"><H4 class="media-heading">The official website of Hangzhou #14 high school</H4></A><A href="http://www.h14z.com"><P>This is the offcial website of our school. And you can find aboved links there.</P></A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="gray-bg"><DIV class="container"><P>     1.Training for iGEM</P><P>
The iGEM program requires the interdisciplinary abilities. To approach this purpose, we designed a series of trainings including computer programming, biological experiment skills, etc. (The following photographs illustrates the process)
The collegiate team from Zhejiang University provides an opportunity which shows us how the biological experiments run. </P><P>     2. Pre-research of project</P><P>We distributed questionnaires and conducted survey in order to learn about the current knowledge of local peoples towards cultural artifacts in Hangzhou.
The following diagrams are analyzed by the data. According to the data, the current situation of protection and renovating towards cultural artifacts and synthetic biology does not have bright prospects. The lack of attentions in civilians can cause the enormous negative effects on research and the practical usage, so we organized a series of activities to publicize the cultural artifacts in Hangzhou, the synthetic biology.(more details in Publicity)</P><P>     3.Publicity</P><P>(1)To acquaint students synthetic biology, our team’s human practice group publicize this subject several times during the summer break in our city including the danger and benefits of transgenosis food that was familiar and relevant to the citizens. 
Furthermore, we upload the basic synthetic biology knowledge in our school’s official website for months and update the latest information or the experimental knowledge at a time period.</P><P>(2)We hold a presentation in our school to introduce the iGEM to our successors. It consists that the general ideas of iGEM, the specific parts which are totally different from the biology competition in China, the judgements, also Q&amp;A section and the advantages of taking this competition. Our team leader Andrew Luo gave an excellent lecture.</P><P>(3)As we mentioned before, to introduce local citizens the importance of cultural artifacts in this city,  we sent out the specialized post cards with 7 different kinds of landscape seals and our team seal in order to publicize our project and the local artifacts protection at the same time. </P></DIV></DIV></DIV></DIV><DIV id="tf-parts"><DIV class="container"><DIV class="section-header"><H2> <SPAN class="highlight"><STRONG>Parts </STRONG></SPAN></H2><H5><EM>About parts in our project</EM></H5><DIV align="center"><H2>BBa_K1671000</H2><P><B>OAH</B></P><P>Oxalacetate acetylhydrolase (OAH), </P><P>a member of the phosphoenolpyruvate mutase/isocitrate lyase superfamily,</P> catalyzes the hydrolysis of oxalacetate to oxalic acid and acetate. <P><B>Design Notes</B></P><P>The gene sequence of this target gene contains many inappropriate enzyme cutting site, </P><P>we can only constructing other cutting site with PCR by ourself. </P><P><B>Source</B></P><P>We found this sequence in the essay 'Structure of Oxalacetate Acetylhydrolase,</P><P> a Virulence Factor of the Chestnut Blight Fungus.' Its accession number in GenBank is GU932672.</P><H2>BBa_K1671001</H2><P><B>Lipase </B></P><P>Lipase, full name triacylglycerol acylhydrolase, is the target gene of this part. </P><P>It is an is an enzyme that catalyzes the hydrolysis of fats, which is able to solve the oil like pollution on the surface of relics.  </P><P><B>Design Notes</B></P><P>The gene sequence of this target gene contains many inappropriate enzyme cutting site, </P><P>we can only constructing other cutting site with PCR by ourself. </P><P><B>Source</B></P><P>We found the sequence of this gene from the essay</P><P> ' Induce Secretion Expression of Lipase Gene from Bacillus subtle in Escherichia Coli.' </P></DIV></DIV></DIV></DIV><DIV id="tf-footer"><DIV class="container"><P class="pull-left">© 2015 Hangzhou #14 highschool iGEM team. All rights reserved. Powered by Minxing Chen.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>