<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTU-Singapore_Modelling_Parameter"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NTU-Singapore/Modelling/Parameter">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTU-Singapore/Modelling/Parameter&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Modelling/Parameter&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Modelling/Parameter&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTU-Singapore/Modelling/Parameter" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTU-Singapore/Modelling/Parameter</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><TABLE style="position:absolute; top:0; width:100%;" cellpadding="0"><TBODY><TR><TD><CENTER><DIV id="header" style="padding=0px;"><DIV id="navigation" style="padding-right:0px; padding-left:13px;"><UL id="MenuBar1" class="MenuBarHorizontal"><LI><A href="https://2008.igem.org/Team:NTU-Singapore">NTU@iGEM</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Project">Project</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Project">Introduction</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Project/Approach">Methodology</A></LI></UL><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Team/Introduction" class="MenuBarItemSubmenu">Team</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Team/Introduction">Introduction</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Team/Team Background">Team background</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Notebook">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modeling">Dry Lab</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modeling">Introduction</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/ODE">ODEs used</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Constructs">Modeling Constructs</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">Parameters used</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Deterministic Modeling">Deterministic Modeling</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Sensitivity Analysis">Sensitivity Analysis</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Stochastic Modeling">Stochastic Modeling</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Wetlab" class="MenuBarItemSubmenu">Wetlab</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Wetlab">Introduction</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Wetlab/Materials and Equipment">Materials and Equipment</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Wetlab/Protocols">Protocols</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Wetlab/Experimental Results">Verification of Detection &amp; Lysis System</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2008&amp;group=NTU-Singapore">Our New Parts</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts">Characterization</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts">Introduction</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/Standard Promoters">Standard Promoters</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/Characterization of LacI-GFP">Characterization of pLacI-GFP</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/Fluorescence Microscope Image">pLacI-GFP images and clips</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/Characterization of pLsrA-YFP">Characterization of pLsrA-YFP</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/New Proposal">New characterization Proposal</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Parts/Parameter Estimation using Characterization Results">Parameter Estimation &amp; Correlations</A></LI><LI><A href="https://2008.igem.org/Team:NTU-Singapore/Acknowledgements">Acknowledgements</A></LI></DIV></DIV></CENTER></TD></TR></TBODY></TABLE></DIV><DIV id="maincontent" style="margin-top:200px;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Parameters"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Lactose_controlled_production_of_E7_.2B_Imm"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Lactose controlled production of E7 + Imm</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Iron_and_Ai2_controlled_production_of_Lysis"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Iron and Ai2 controlled production of Lysis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Parameter_Estimation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Parameter Estimation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#E7_production_system"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">E7 production system</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Detection_.26_Lysis_system"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Detection &amp; Lysis system</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#GFP_production_system_with_logistic_growth"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">GFP production system with logistic growth</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Parameters">Parameters</SPAN></H1><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">What Parameters Should We USE??</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Lactose_controlled_production_of_E7_.2B_Imm">Lactose controlled production of E7 + Imm</SPAN></H2><OL><LI>LacI = A
</LI><LI>Lactose =B
</LI><LI>E7 = C
</LI></OL><H2><SPAN class="mw-headline" id="Iron_and_Ai2_controlled_production_of_Lysis">Iron and Ai2 controlled production of Lysis</SPAN></H2><OL><LI>Ai-2			:	A
</LI><LI>ai-2-phos		: 	B
</LI><LI>LsrR			:	C
</LI><LI>SupD derivatives	: 	D
</LI><LI>T7ptag			:	E
</LI><LI>Lysis			:	F
</LI></OL><H1><SPAN class="mw-headline" id="Parameter_Estimation">Parameter Estimation</SPAN></H1><P><B>Estimation of different parameters</B></P><TABLE border="0"><TBODY><TR><TD># Transcription :
</TD><TD>70nt/s
</TD></TR><TR><TD># Translation:
</TD><TD>40aa/s
</TD></TR><TR><TD># Number of Essential Genes:
</TD><TD>297
</TD></TR><TR><TD># Number of mRNA per cell:
</TD><TD>4000
</TD></TR><TR><TD># Average mRNA half life:
</TD><TD>5min
</TD></TR><TR><TD># Average mRNA length:
</TD><TD>1100
</TD></TR></TBODY></TABLE><P><B>Assumptions</B></P><OL><LI>Rate of transcription is dependent on length of gene
</LI><LI>Number of amino acids is 1/3 of the number of nucleotides in a gene
</LI><LI>Rate of Translation is dependent on number of nucleotides
</LI><LI>For each gene mRNA = 10 at steady state
</LI><LI>Rate of degradation of average mRNA = 1100/ 5 min
</LI><LI>Rate of degradation of protein is equivalent to time for cell division i.e. 40 min
</LI></OL><H2><SPAN class="mw-headline" id="E7_production_system">E7 production system</SPAN></H2><TABLE align="center" border="1"><TBODY><TR><TD><B>Type</B></TD><TD><B>Parameter</B></TD><TD><B>Values</B></TD><TD><B>Comments</B></TD></TR><TR><TD>Transcription Rate of Lac I gene
</TD><TD>k1A
</TD><TD>21
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Transcription Rate of E7 + Imm gene
</TD><TD>k1C
</TD><TD>2.470588
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Degradation Rate of Lac I mRNA
</TD><TD>d1A
</TD><TD>0.76246
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Transcription Rate of E7 + Imm mRNA
</TD><TD>d1C
</TD><TD>0.0897
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Translation Rate of Lac I mRNA
</TD><TD>k2A
</TD><TD>36
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Translation Rate of E7 + Imm mRNA
</TD><TD>k2C
</TD><TD>4.23539
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Protein Degradation Rate
</TD><TD>d2A,d2C
</TD><TD>0.03465
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Hill coefficeint for E7 + Imm
</TD><TD>nC
</TD><TD>1
</TD><TD>This is obtained on the assumption that one Repressor Protein binds to one Lactose molecule complex
</TD></TR><TR><TD>Dissociation Constant for E7 + Imm
</TD><TD>KC
</TD><TD>0.8</TD><TD>[1]
</TD></TR><TR><TD>Constitutive Portion for E7 + Imm
</TD><TD>aC
</TD><TD>0.5
</TD><TD>Estimate since a is between 0 and 1 Implication that Lactose may not be  a very strongly Regulated Promoter
</TD></TR><TR><TD>Complex Formation Rate Between  Lac I Repressor and Lactose
</TD><TD>k3AB
</TD><TD>1
</TD><TD>Estimate. This is based on the assumption  that the complex formation is only  dependent on the concentrations of  Lac I repressor and Lactose
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Detection_.26_Lysis_system">Detection &amp; Lysis system</SPAN></H2><P>Some of the parameter values were obtained via trial and error as the values could not be found in literature. The values are chosen to produce the desired behaviour of the graph. More work has to be done to actually find out how much lysis protein quantity is really required for lysis of the cell as this is still an unknown. Readers may find some of the parameters unrealistic but the purpose of this exercise is to observe how changing certain parameters may affect the system overall output. We advise the reader to look more into the behaviour of the system rather than be focused upon the numerical values.
</P><TABLE align="center" border="1"><TBODY><TR><TD><B>Type</B></TD><TD><B>Parameter</B></TD><TD><B>Values</B></TD><TD><B>Comments</B></TD></TR><TR><TD>Phosphorylation Rate of Ai-2
</TD><TD>kPF
</TD><TD>1
</TD><TD>Estimate
</TD></TR><TR><TD>Dephosphorylation Rate of Ai-2
</TD><TD>kPB
</TD><TD>1
</TD><TD>Estimate
</TD></TR><TR><TD>Transcription Rate of LsrR gene
</TD><TD>k1C
</TD><TD>4.402517
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Transcription Rate of SupD gene
</TD><TD>k1D
</TD><TD>46.667
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Transcription Rate of t7pTag gene
</TD><TD>k1E
</TD><TD>1.5556
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Transcription Rate of Lysis gene
</TD><TD>k1F
</TD><TD>28
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Degradation Rate of LsrR mRNA
</TD><TD>d1C
</TD><TD>0.159845
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Degradation Rate of SupD mRNA
</TD><TD>d1D
</TD><TD>1.694359
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Degradation Rate of t7pTag mRNA
</TD><TD>d1E
</TD><TD>0.056478
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Degradation Rate of Lysis mRNA
</TD><TD>d1F
</TD><TD>1.0166
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Translation Rate of LsrR Protein
</TD><TD>k2C
</TD><TD>7.54716
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Translation Rate of t7 pTag Protein
</TD><TD>k2E
</TD><TD>2.6667
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Translation Rate of Lysis Protein
</TD><TD>k2F
</TD><TD>48
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Protein Degradation Rate
</TD><TD>d2C,d2E,d2F
</TD><TD>0.03465
</TD><TD>Made using Earlier assumptions
</TD></TR><TR><TD>Hill coefficeint for SupD
</TD><TD>nD
</TD><TD>50
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisnD" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Hill coefficeint for t7 pTag
</TD><TD>nE
</TD><TD>1
</TD><TD>Estimate. It is assuming that  one molecule of iron ion is required  to activate the production of one t7mRNA
</TD></TR><TR><TD>Hill coefficeint for Lysis
</TD><TD>nF
</TD><TD>1
</TD><TD>Estimate. It is assumed that  one molecule of t7 is required  for activation of one Lysis mRNA
</TD></TR><TR><TD>Dissociation Constant for SupD
</TD><TD>KD
</TD><TD>15
</TD><TD>[2]
</TD></TR><TR><TD>Dissociation Constant for t7 pTag
</TD><TD>KE
</TD><TD>1
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisKE" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Dissociation Constant for Lysis
</TD><TD>KF
</TD><TD>1
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisKF" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Constitutive Portion for SupD
</TD><TD>aD
</TD><TD>0.01
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaD" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD></TR><TR><TD>Constitutive Portion for t7 pTag
</TD><TD>aE
</TD><TD>0.01
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaE" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Constitutive Portion for Lysis
</TD><TD>aF
</TD><TD>0.0001
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/LysisaF" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD></TR><TR><TD>Complex Formation Rate Between  LsrR Repressor and Ai-2-Phosphorylated
</TD><TD>k3BC
</TD><TD>0.01
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/Lysisk3BC" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD></TR><TR><TD>Complex Formation Rate Between  SupD tRNA and t7 pTag mRNA
</TD><TD>k3DE
</TD><TD>0.000000001
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/Lysisk3DE" class="external text" rel="nofollow">Trial and Error</A></SPAN></TD></TR><TR><TD>Amount of Protein Kinase
</TD><TD>S
</TD><TD>28.747
</TD><TD>From Earlier Assumptions
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="GFP_production_system_with_logistic_growth">GFP production system with logistic growth</SPAN></H2><TABLE align="center" border="1"><TBODY><TR><TD><B>Type</B></TD><TD><B>Parameter</B></TD><TD><B>Values</B></TD><TD><B>Comments</B></TD></TR><TR><TD>Forward Reaction rate of Complex
</TD><TD>kf
</TD><TD>1.2
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/kF" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Reverse Reaction rate of Complex
</TD><TD>kr
</TD><TD>0.5
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/kr" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Rate of Logistic Growth
</TD><TD>r
</TD><TD>0.01
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/r_logistic" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR><TR><TD>Varying Capacity
</TD><TD>A &amp; B
</TD><TD>1 &amp; 0.9
</TD><TD><SPAN style="font-size:16px;"><A href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter/A_B" class="external text" rel="nofollow">Trial And Error</A></SPAN></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>1. <A href="https://2007.igem.org/title=ETHZ/Parameters" class="external free" rel="nofollow">https://2007.igem.org/title=ETHZ/Parameters</A>
2. Grass G, Otto M, Fricke B, et al., FieF (YiiP) from Escherichia coli mediates decreased cellular accumulation of iron and relieves iron stress, ARCHIVES OF MICROBIOLOGY,   Volume: 183   Issue: 1   Pages: 9-18   Published: JAN 2005
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">http://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTU-Singapore/Modelling/Parameter" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTU-Singapore/Modelling/Parameter" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTU-Singapore/Modelling/Parameter&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTU-Singapore/Modelling/Parameter&amp;oldid=78589" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>