"
Page
Discussion
View source
History
teams
Log in
 
Team:NTU-Singapore/Wetlab/Protocols
From 2008.igem.org
NTU@iGEM
Project
Introduction
Methodology
Team
Introduction
Team background
Notebook
Dry Lab
Introduction
ODEs used
Modeling Constructs
Parameters used
Deterministic Modeling
Sensitivity Analysis
Stochastic Modeling
Wetlab
Introduction
Materials and Equipment
Protocols
Verification of Detection & Lysis System
Our New Parts
Characterization
Introduction
Standard Promoters
Characterization of pLacI-GFP
pLacI-GFP images and clips
Characterization of pLsrA-YFP
New characterization Proposal
Parameter Estimation & Correlations
Acknowledgements
Contents
1 Protocols
1.1 Creation of New Biobrick parts
1.2 Digestion and Ligation of NTU@iGEM System
1.2.1 Case I:  Insert in front – Vector behind
1.2.2 Case II:  Vector in front – Insert behind
1.3 OD600 measurement of pLsrA-Lysis-containing LuxS mutant upon addition of AI-2
Protocols
Creation of New Biobrick parts
To create a new biobrick part, the first step is to identify the gene of interest. In this project, our genes of interest are the E7 production gene, Immunity gene, and Lysis gene that is obtainable from the plasmid ColE7-K317 in BW Escherichia Coli strain. Another gene of interest is the LsrA promoter that is obtained from MG1655 Escherichia Coli strain. Information on the genome of interests is extracted from relevant publications and websites such as NCBI net.
For example the gene of interest – Lysis gene:
1. A pair of DNA primers, 10 ~ 15 nucleotides in length are designed and synthesized to flank the forward 3' – 5' end and reverse 5' – 3' end of the gene of interests. The primers also contain blunt ends of the EcoRI, XbaI, SpeI & PstI restriction sites. These primers will hook on to the double strand DNA by complementary pairing and serve as initiation and termination site of DNA amplification.
2. Amplification of the lysis gene occurs by PCR amplification. A gel electrophoresis is run, followed by a gel extraction of the specific interest.
3. The lysis gene is then purified using the PCR purification kit. Purified DNA is tested for concentration and purity using NanoDrop.
4. Next, the purified DNA is digested using EcoRI and PstI restriction enzymes to generate a standard Biobrick part with the gene sequence:
EcoRI – XbaI - Lysis gene – SpeI – PstI
Creation of New BioBricks
Digestion and Ligation of NTU@iGEM System
To ligate 2 Biobrick parts together, firstly we identify one part to be "vector" and the second part to be "insert". There are 2 possibility of ligation:
Case I:  Insert in front – Vector behind
Case II: Vector in front – Insert behind
Case I:  Insert in front – Vector behind
1.Digest Insert gene with EcoRI & SpeI and Vector gene with SpeI & PstI.
Purify the digested products with PCR Purification Kit and test for concentration and purity using NanoDrop.
2.Perform a gel electrophoresis of the purified products, followed by a gel extraction using PCR Gel Extraction Kit.
3.Fuse the Insert gene to the Vector gene to form a circular plasmid DNA using T4 DNA Quick Ligase.
Transform the DNA plasmid into Top 10 Competent Cells and select using appropriate antibiotic treatment.
Vector gene contain antibiotic resistant gene that confers resistance to the cells that are successfully transformed.
4.Extract the ligated DNA using MiniPrep Kit and digest the resultant DNA using EcoRI & PstI.
5.Lastly, perform a gel electrophoresis for confirmation. Check if the nucleotide length from the gel run matches the hypothetical nucleotide length of the digested product. If the result matches, ligation is successful.
Case II:  Vector in front – Insert behind
1. Digest Insert gene with XbaI & PstI and Vector gene with EcoRI & XbaI. Purify the digested products with PCR Purification Kit and test for concentration and purity using NanoDrop.
2. Repeat Step 2 ~ 5 as from Case I
Summary of Digestion and Ligation Procedures
OD600 measurement of pLsrA-Lysis-containing LuxS mutant upon addition of AI-2
1) One colony of LuxS(-) W3110 strain containing pLsrA-lysis plasmid was inoculated overnight in 50ml LB inside 250ml flask at 37 degrees C and 225rpm.
2) After 16 hour inoculation, 0.5 ml of cell sample was diluted in 50ml fresh LB.
3) Diluted cell sample was distributed into the wells of the black-96-well microplate with an amount of 200 µL per well.
4) 50µl AI-2-containing supernatant was added into corresponding wells initially (set 1) and 1 hour after incubated inside absorbance-meter (set 2).
5) Five different supernatant solutions used, which correspond to the time points when they had been obtained during AI-2 preparation: 3, 4, 5, 6, 8 hours.
6) Control samples included
i) cell suspension alone   ii) cell suspension with 50µl LB added initially (set 1)   iii) cell suspension with 50µl LB after 1 hour incubation (set 2)   iv) cell suspension with 50µl water added initially (set 1)    v) cell suspension with 50µl water added after 1 hour incubation (set 2)
Retrieved from " http://2008.igem.org/Team:NTU-Singapore/Wetlab/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
