"
Page
Discussion
View source
History
teams
Log in
 
Team:NTU-Singapore/Modelling/Parameter
From 2008.igem.org
NTU@iGEM
Project
Introduction
Methodology
Team
Introduction
Team background
Notebook
Dry Lab
Introduction
ODEs used
Modeling Constructs
Parameters used
Deterministic Modeling
Sensitivity Analysis
Stochastic Modeling
Wetlab
Introduction
Materials and Equipment
Protocols
Verification of Detection & Lysis System
Our New Parts
Characterization
Introduction
Standard Promoters
Characterization of pLacI-GFP
pLacI-GFP images and clips
Characterization of pLsrA-YFP
New characterization Proposal
Parameter Estimation & Correlations
Acknowledgements
Contents
1 Parameters
1.1 Lactose controlled production of E7 + Imm
1.2 Iron and Ai2 controlled production of Lysis
2 Parameter Estimation
2.1 E7 production system
2.2 Detection & Lysis system
2.3 GFP production system with logistic growth
2.4 References
Parameters
What Parameters Should We USE??
Lactose controlled production of E7 + Imm
LacI = A
Lactose =B
E7 = C
Iron and Ai2 controlled production of Lysis
Ai-2   : A
ai-2-phos  :  B
LsrR   : C
SupD derivatives :  D
T7ptag   : E
Lysis   : F
Parameter Estimation
Estimation of different parameters
# Transcription :
70nt/s
# Translation:
40aa/s
# Number of Essential Genes:
297
# Number of mRNA per cell:
4000
# Average mRNA half life:
5min
# Average mRNA length:
1100
Assumptions
Rate of transcription is dependent on length of gene
Number of amino acids is 1/3 of the number of nucleotides in a gene
Rate of Translation is dependent on number of nucleotides
For each gene mRNA = 10 at steady state
Rate of degradation of average mRNA = 1100/ 5 min
Rate of degradation of protein is equivalent to time for cell division i.e. 40 min
E7 production system
Type
Parameter
Values
Comments
Transcription Rate of Lac I gene
k1A
21
Made using Earlier assumptions
Transcription Rate of E7 + Imm gene
k1C
2.470588
Made using Earlier assumptions
Degradation Rate of Lac I mRNA
d1A
0.76246
Made using Earlier assumptions
Transcription Rate of E7 + Imm mRNA
d1C
0.0897
Made using Earlier assumptions
Translation Rate of Lac I mRNA
k2A
36
Made using Earlier assumptions
Translation Rate of E7 + Imm mRNA
k2C
4.23539
Made using Earlier assumptions
Protein Degradation Rate
d2A,d2C
0.03465
Made using Earlier assumptions
Hill coefficeint for E7 + Imm
nC
1
This is obtained on the assumption that one Repressor
Protein binds to one Lactose molecule complex
Dissociation Constant for E7 + Imm
KC
0.8
[1]
Constitutive Portion for E7 + Imm
aC
0.5
Estimate since a is between 0 and 1
Implication that Lactose may not be
a very strongly Regulated Promoter
Complex Formation Rate Between
Lac I Repressor and Lactose
k3AB
1
Estimate. This is based on the assumption
that the complex formation is only
dependent on the concentrations of
Lac I repressor and Lactose
Detection & Lysis system
Some of the parameter values were obtained via trial and error as the values could not be found in literature. The values are chosen to produce the desired behaviour of the graph. More work has to be done to actually find out how much lysis protein quantity is really required for lysis of the cell as this is still an unknown. Readers may find some of the parameters unrealistic but the purpose of this exercise is to observe how changing certain parameters may affect the system overall output. We advise the reader to look more into the behaviour of the system rather than be focused upon the numerical values.
Type
Parameter
Values
Comments
Phosphorylation Rate of Ai-2
kPF
1
Estimate
Dephosphorylation Rate of Ai-2
kPB
1
Estimate
Transcription Rate of LsrR gene
k1C
4.402517
Made using Earlier assumptions
Transcription Rate of SupD gene
k1D
46.667
Made using Earlier assumptions
Transcription Rate of t7pTag gene
k1E
1.5556
Made using Earlier assumptions
Transcription Rate of Lysis gene
k1F
28
Made using Earlier assumptions
Degradation Rate of LsrR mRNA
d1C
0.159845
Made using Earlier assumptions
Degradation Rate of SupD mRNA
d1D
1.694359
Made using Earlier assumptions
Degradation Rate of t7pTag mRNA
d1E
0.056478
Made using Earlier assumptions
Degradation Rate of Lysis mRNA
d1F
1.0166
Made using Earlier assumptions
Translation Rate of LsrR Protein
k2C
7.54716
Made using Earlier assumptions
Translation Rate of t7 pTag Protein
k2E
2.6667
Made using Earlier assumptions
Translation Rate of Lysis Protein
k2F
48
Made using Earlier assumptions
Protein Degradation Rate
d2C,d2E,d2F
0.03465
Made using Earlier assumptions
Hill coefficeint for SupD
nD
50
Trial And Error
Hill coefficeint for t7 pTag
nE
1
Estimate. It is assuming that
one molecule of iron ion is required
to activate the production of one t7mRNA
Hill coefficeint for Lysis
nF
1
Estimate. It is assumed that
one molecule of t7 is required
for activation of one Lysis mRNA
Dissociation Constant for SupD
KD
15
[2]
Dissociation Constant for t7 pTag
KE
1
Trial And Error
Dissociation Constant for Lysis
KF
1
Trial And Error
Constitutive Portion for SupD
aD
0.01
Trial and Error
Constitutive Portion for t7 pTag
aE
0.01
Trial And Error
Constitutive Portion for Lysis
aF
0.0001
Trial and Error
Complex Formation Rate Between
LsrR Repressor and Ai-2-Phosphorylated
k3BC
0.01
Trial and Error
Complex Formation Rate Between
SupD tRNA and t7 pTag mRNA
k3DE
0.000000001
Trial and Error
Amount of Protein Kinase
S
28.747
From Earlier Assumptions
GFP production system with logistic growth
Type
Parameter
Values
Comments
Forward Reaction rate of Complex
kf
1.2
Trial And Error
Reverse Reaction rate of Complex
kr
0.5
Trial And Error
Rate of Logistic Growth
r
0.01
Trial And Error
Varying Capacity
A & B
1 & 0.9
Trial And Error
References
1. https://2007.igem.org/title=ETHZ/Parameters
2. Grass G, Otto M, Fricke B, et al., FieF (YiiP) from Escherichia coli mediates decreased cellular accumulation of iron and relieves iron stress, ARCHIVES OF MICROBIOLOGY,   Volume: 183   Issue: 1   Pages: 9-18   Published: JAN 2005
Retrieved from " http://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
