<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wisconsin-Madison_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Wisconsin-Madison/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wisconsin-Madison/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wisconsin-Madison/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wisconsin-Madison/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><CENTER><PRE><B>PROJECT</B></PRE></CENTER><TABLE class="wikitable" border="0" align="center" width="200px"><TBODY><TR><TD style="vertical-align:top"><UL><LI> Abstract
</LI><LI> Biofuels as a Possible Solution
</LI><LI> Experimental Approach
</LI><LI> Genes
</LI><LI> Theory
</LI></UL></TD></TR></TBODY></TABLE><CENTER><DIV style="width: 700px;"><PRE><B>ABSTRACT</B></PRE><P>Biofuels represent a potential solution to current world energy demands. Total crude oil replacement based on a 20% fuel titer and current fuel demands would require 5.6 trillion gallons of fresh water per year. Current fresh water supplies may not support this added demand. Alternatively, a sustainable approach may use a portion of the Earth’s 3.5x1020 gallons of ocean water. However, current fuel-producing organisms are unable to thrive in ocean-level osmolarities. Glycine betaine, a powerful osmoprotectant, shields organisms from salt-induced stress.  Wild-type Escherichia coli can acquire glycine betaine from their surroundings or synthesize it from environmental choline.  Two enzymes, glycine/sarcosine methyltransferase, and sarcosine/dimethyglycine methyltransferase, catalyze three successive methylations of glycine for de novo synthesis of glycine betaine.  Here, we demonstrate an engineered E. coli with an increased growth rate under salt induced stress.  We highlight utility by demonstrating the improved growth of fuel producing bacteria in ocean water.
</P><PRE><B>EXPERIMENTAL APPROACH</B></PRE><P>In a Hypertonic environment cells typically shrink because the osmolarity outside the cell is higher than that inside the cell and cause the cell to lose turgidity. Water then leaves the cell by osmosis through the semi permeable cell membrane. The concentration of solutes increases causing proteins to change conformation.
Certain small charged molecules can be characterized as osmoprotectants. Osomoprotectants typically help the cell overcome the hypertonic environments by raising the osmolarity inside the cell preventing water loss through osmosis.  Hydrophobic/hydrophilic interactions of proteins and membranes are also stabilized by these useful molecules. 
Glycine Betaine is one such osmoprotectant that is particularly common and powerful. Wildtype E. Coli can uptake glycine betain, also called betaine, directly from its environment or synthesize it from in a two step process from choline , which also mush bet exogeneously supplied. 
Other haliotolerant bacteria have the ability to synthesize betaine in a three step methylation from glycine.  Since Glycine is an amino acid whose concentrion is maintained in E. coli. we have chosen pursue this pathway. We cloned two genes, GSMT and SDMT, into E Coli in order to produce betaine from glycine.  
</P><PRE><B>GENES</B></PRE><P><B>GSMT</B></P><P>We obtained the Glycine Sarcosine Methyl Transferase Gene from codon optomization and synthesis of its coding region from Aphanothece halophytica, as detailed in &quot;Genes for direct methylation of glycine provide high levels of glycinebetaine and abiotic-stress tolerance in Synechococcus and Arabidopsis&quot; (Waditee et. al. 2004) The gene carries out two successive methylations of Glycine to Sarsine to Dimethyl Glycine. S-Adenosyl Methionine is thought to act as the primary methyl donator.  
</P><P><B>SDMT</B></P><P>We obtained Sarcoisine Dimethyl  Glycine Methyl Transferase from Galdieria sulphuraria as detailed by Brian Fox. This Eukaryote is capable of living in environments of up to 10% w/v salinity. The gene caries out two successive methylations of Sarcosine to dimethyl glycine to glycine betaine. 
</P><P><B>MetK</B></P><P>We obtained Methionine Kinase from Streptomyces spectabilis. E. Coli usually maintain basal levels of SAM to fuel the abundant processes that require methyl donatiions in the typical cell. These reserves may become depleted with the added stress of glycine methylation. Overexpression of Sam Synthase may boots  SAM levels circumventing the problem.
</P><P><B>ProU</B></P><P>The ProU promoter was obtained from wild type K12 E. Coli Genomic DNA. Pro U controls transcription of the Pro U operon which contains three sub-units instrumental in bringing choline and proline into the cell.  The promoter is most active during times of osmotic stress. FA CT (check proline and choline). Placing the proU promoter before genes for betaine productions would allow cell resources to diverted the the betaine pathway only during times of salt stress.  
</P><PRE><B>THEORY</B></PRE><P>A plate reader is an automated instrument that can take optical density readingins over time.  These growth curves were generated by taking absorbance readings at 600nm over a 24 hour period in a plate reader. Each curve corresponds to a different salt concentration ranging from 0 to 1 molar sodium chloride. These curves were generated with an unmodified laboratory strain, DH10B. To find the 50% inhibitory concentration for growth, the molarity at which half the maximum gowth occurs was calculated.  Essentially, this formula takes the highest growth rate and finds the salt concentration that results in half this growth rate. We found the IC50 to be 0.55 M.
Increasing the IC50 indicate better growth in high osmolarities. We tested optical densities under salt stress over time to establish growth curves and demonstrate improved growth of constructs in high osmolarities.
</P><P>&lt;center&gt;
For more detail on our BioBricks See our [<A href="https://2009.igem.org/Team:Wisconsin-Madison/Parts" class="external text" rel="nofollow">Parts Page</A>].
&lt;/center&gt;</P><B>For references, please visit <A href="/Team:Wisconsin-Madison/Notebook_References" title="Team:Wisconsin-Madison/Notebook References">Here</A></B></DIV></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Wisconsin-Madison/Project">http://2009.igem.org/Team:Wisconsin-Madison/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wisconsin-Madison/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wisconsin-Madison/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/Project&amp;oldid=136312" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>