<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGIC_China_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGIC China/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Ellz's Site</TITLE></P><DIV id="s-page-container"><DIV data-reactroot="" data-reactid="1" data-react-checksum="-2073444347"><DIV id="s-content" class="s-variation-default s-font-heading-default s-font-title-default s-font-body-default" data-reactid="3"><DIV class="social-thumbnail" data-reactid="4"><DIV id="lw-share-data" style="display:none;" data-reactid="7"><DIV class="lws-title" data-reactid="8">Ellz's Site</DIV><DIV class="lws-description" data-reactid="9">Website</DIV><DIV class="lws-image" data-reactid="10">http://assets.strikingly.com/assets/fb_images/default-715654470f8f9a657474bf68cefc382f.png</DIV></DIV></DIV><UL class="slides s-page-1" data-reactid="13"><LI class="slide s-section-1" id="section-f_3b3523a6-2671-4e7d-b310-e6dfeb26fc8d" data-reactid="14"><DIV class="lazyload s-block-section s-section" data-reactid="17"><DIV class="container" data-reactid="18"><DIV class="alignment-container " data-reactid="21"><DIV class="s-repeatable s-block s-component s-mh " data-reactid="22"><DIV class="s-block-item s-repeatable-item s-block-sortable-item s-block-title " data-reactid="23"><SPAN data-reactid="24"><DIV data-reactid="25"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="26"><DIV class=" columns sixteen" data-reactid="27"><DIV class="s-item-title" data-reactid="28"><DIV class="s-component s-text" data-reactid="29"><H3 class="s-component-content s-font-heading" data-reactid="30"><P style="font-size: 130%;">The Proof of Concept</P></H3></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="31"><SPAN data-reactid="32"><DIV data-reactid="33"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="34"><DIV class=" columns sixteen" data-reactid="35"><DIV class="s-component s-text" data-reactid="36"><DIV class="s-component-content s-font-body" data-reactid="37"><P style="text-align: left; font-size: 130%;"><STRONG>Design:</STRONG></P><P style="text-align: left; font-size: 130%;">Our original design includes two plasmids, one acts as a sensor module (Fig.1), another acts as an signal amplification module (Fig.2).</P><P style="text-align: left; font-size: 130%;">The sensor plasmid contains pCpxP promoter which is inducible by various carbon sources including glucose, glycerol and citrate acid. Our experiment data also proved that the increase in pH could enchance the expression of pCpxP.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="38"><SPAN data-reactid="39"><DIV data-reactid="40"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="41"><DIV class=" columns sixteen" data-reactid="42"><DIV class="s-component s-image" data-reactid="43"><SPAN data-reactid="44"><DIV class="s-component-content" data-reactid="45"><DIV data-reactid="46"><A href="https://static.igem.org/mediawiki/2016/c/c7/T--BGIC_China--%E5%B1%8F%E5%B9%95%E6%88%AA%E5%9B%BE%28940%29_%E5%89%AF%E6%9C%AC.png" target="_blank" data-reactid="47">&lt;div class=&quot;s-ratio-box&quot; style=&quot;max-width:1267px;max-height:1193px;&quot; data-reactid=&quot;48&quot;&gt;</A></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="56"><SPAN data-reactid="57"><DIV data-reactid="58"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="59"><DIV class=" columns sixteen" data-reactid="60"><DIV class="s-component s-text" data-reactid="61"><DIV class="s-component-content s-font-body" data-reactid="62"><P style="text-align: left;"><EM><STRONG>Fig.1 Sensor plasmid (pLV1C3-pCpxP-LacZW-Cre)</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="81"><SPAN data-reactid="82"><DIV data-reactid="83"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="84"><DIV class=" columns sixteen" data-reactid="85"><DIV class="s-component s-text" data-reactid="86"><DIV class="s-component-content s-font-body" data-reactid="87"><P style="text-align: left;"><EM><STRONG>Fig.2 Signal amplification plasmid (pET28a-LSL-LacZW-Cre)</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="88"><SPAN data-reactid="89"><DIV data-reactid="90"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="91"><DIV class=" columns sixteen" data-reactid="92"><DIV class="s-component s-text" data-reactid="93"><DIV class="s-component-content s-font-body" data-reactid="94"><P style="text-align: left; font-size: 130%;">Sensor plasmid uses pCpxP promoter to control the expression level of the downstream genes---genes coding for LacZ and Cre. Cre is a recombinase which recognise loxp sites and recombine the target gene based on different directions of two loxp sites which flank the target gene. When Cre recombinase has been expressed from the sensor plasmid, it will cut down two B0015 terminators on the signal amplification plasmid and recombine the plasmid (Fig.3).</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="113"><SPAN data-reactid="114"><DIV data-reactid="115"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="116"><DIV class=" columns sixteen" data-reactid="117"><DIV class="s-component s-text" data-reactid="118"><DIV class="s-component-content s-font-body" data-reactid="119"><P style="text-align: left;"><EM><STRONG>Fig.3 Plasmid recombination by Cre at loxp sites</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="120"><SPAN data-reactid="121"><DIV data-reactid="122"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="123"><DIV class=" columns sixteen" data-reactid="124"><DIV class="s-component s-text" data-reactid="125"><DIV class="s-component-content s-font-body" data-reactid="126"><P style="font-size: 130%;">When the B0015 terminators downstream of T7 promoter have been cut off, genes coding for LacZ and Cre could be expressed continuously given that IPTG is in excess supply. Chain reaction will be trigger because recombined signal amplification plasmid itself could express Cre recombinase that cut off B0015 terminators on intact signal amplification plasmids. Thus, the amount of LacZ will be accumulated in an exponential rate and the expression of LacZ will be sustained even pCpxP inducers are used up.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="127"><SPAN data-reactid="128"><DIV data-reactid="129"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="130"><DIV class=" columns sixteen" data-reactid="131"><DIV class="s-component s-text" data-reactid="132"><DIV class="s-component-content s-font-body" data-reactid="133"><P style="font-size:130%"><STRONG>Experiment:</STRONG></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="134"><SPAN data-reactid="135"><DIV data-reactid="136"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="137"><DIV class=" columns sixteen" data-reactid="138"><DIV class="s-component s-text" data-reactid="139"><DIV class="s-component-content s-font-body" data-reactid="140"><P style="font-size: 130%; text-align: center;"><EM>Plasmid assembly method</EM></P><P style="font-size: 130%;">Genes coding for LacZ and Cre are obtained by PCR from plasmids which contain these two gene sequences with our designed primers. pCpxP promoter was synthesized by using phosphorylated single stranded oligonucleotides purchased from Invitrogen(TM). LSL(loxp-B0015-B0015-loxp) was also obtained by PCR with specially design primers. We then assembled our plasmids by using standard MCS subcloning method.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="141"><SPAN data-reactid="142"><DIV data-reactid="143"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="144"><DIV class=" columns sixteen" data-reactid="145"><DIV class="s-component s-text" data-reactid="146"><DIV class="s-component-content s-font-body" data-reactid="147"><P style="font-size:130%; text-align:center"><EM>Obstacles and changes of the original design</EM></P><P style="font-size:130%">Due to technical and time issues, we are not able to follow our original design. Instead, we synthesized two relatively simple plamids(Fig.4 &amp; Fig.5) with pCpxP promoter to test for the viability of the cell-free paper-based biochip system.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="166"><SPAN data-reactid="167"><DIV data-reactid="168"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="169"><DIV class=" columns sixteen" data-reactid="170"><DIV class="s-component s-text" data-reactid="171"><DIV class="s-component-content s-font-body" data-reactid="172"><P><EM><STRONG>Fig.4 pLV1C3-pCpxP-GFP</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="191"><SPAN data-reactid="192"><DIV data-reactid="193"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="194"><DIV class=" columns sixteen" data-reactid="195"><DIV class="s-component s-text" data-reactid="196"><DIV class="s-component-content s-font-body" data-reactid="197"><P><EM><STRONG>Fig.5 pLV1C3-pCpxP-LacZW</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="198"><SPAN data-reactid="199"><DIV data-reactid="200"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="201"><DIV class=" columns sixteen" data-reactid="202"><DIV class="s-component s-text" data-reactid="203"><DIV class="s-component-content s-font-body" data-reactid="204"><P style="font-size: 130%; text-align: center;"><EM>pCpxP expression in cell</EM></P><P style="font-size:130%">We first transformed the plasmid pLV1C3-pCpxP-GFP into bacteria E.coli DH5α and test for the OD and RFU value under different concentrations of glucose and glycerol in order to characterize the pCpxP promoter. Based on a recent research, promoter CpxP was found to have potential in detecting a variaty of carbon sources in clinical samples[1]. However, our experimental results do not fit this research, the trend of our data generally cannot prove the inducibility of pCpxP by glucose or glycerol (Fig.6 &amp; Fig.7).</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="223"><SPAN data-reactid="224"><DIV data-reactid="225"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="226"><DIV class=" columns sixteen" data-reactid="227"><DIV class="s-component s-text" data-reactid="228"><DIV class="s-component-content s-font-body" data-reactid="229"><P><EM><STRONG>Fig.6 MRPU against glucose concentration (mg/dL)</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="248"><SPAN data-reactid="249"><DIV data-reactid="250"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="251"><DIV class=" columns sixteen" data-reactid="252"><DIV class="s-component s-text" data-reactid="253"><DIV class="s-component-content s-font-body" data-reactid="254"><P><EM><STRONG>Fig.7 MRPU against glycerol concentration (mg/dL)</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="255"><SPAN data-reactid="256"><DIV data-reactid="257"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="258"><DIV class=" columns sixteen" data-reactid="259"><DIV class="s-component s-text" data-reactid="260"><DIV class="s-component-content s-font-body" data-reactid="261"><P style="font-size: 130%;">MRPU (modified relative promoter activity unit) is a equation that we modified from RPU[2] in order to determine the inducibility of a particular promoter by comparing the expression unit of the promoter with the expression unit of a reference promoter --- J23119 constitutive promoter. MRPU is a relatively simple equation that use much fewer parameters than RPU and provides a much easier way to perform the calculation to give a general idea about the inducibility of a particular promoter by particular factors.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="280"><SPAN data-reactid="281"><DIV data-reactid="282"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="283"><DIV class=" columns sixteen" data-reactid="284"><DIV class="s-component s-text" data-reactid="285"><DIV class="s-component-content s-font-body" data-reactid="286"><P><EM><STRONG>Fig.8 Equations used to calculate MRPU </STRONG>Here we use pCpxP as promoter</EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="287"><SPAN data-reactid="288"><DIV data-reactid="289"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="290"><DIV class=" columns sixteen" data-reactid="291"><DIV class="s-component s-text" data-reactid="292"><DIV class="s-component-content s-font-body" data-reactid="293"><P style="font-size: 130%;">According to Wolfe’s study, Cpx signal transduction system, in conjunction with SigmaE and Sigma32, responds to a board spectrum of adverse environmental conditions including high pH[3]. So we then test for the expression of promoter CpxP in E.coli DH5α in a range of pH from 6 to 9. We cultured bacteria in different pH under 37℃ for 10 hours and record the RFU and OD values. Graph of MRPU against pH has been plotted (Fig.9).</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="312"><SPAN data-reactid="313"><DIV data-reactid="314"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="315"><DIV class=" columns sixteen" data-reactid="316"><DIV class="s-component s-text" data-reactid="317"><DIV class="s-component-content s-font-body" data-reactid="318"><P><EM><STRONG>Fig.9 Graph of MRPU against pH</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="319"><SPAN data-reactid="320"><DIV data-reactid="321"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="322"><DIV class=" columns sixteen" data-reactid="323"><DIV class="s-component s-text" data-reactid="324"><DIV class="s-component-content s-font-body" data-reactid="325"><P style="font-size: 130%;">The graph shows an logarithmic relationship between MRPU and pH, indicated the expression level of pCpxP increases when the pH rises. The MRPU at pH 9 is about three-folds the MRPU when pH is 6. Thus, the result shows that the expression level of pCpxP could be greatly increased when the pH has been increased in a certain range.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="326"><SPAN data-reactid="327"><DIV data-reactid="328"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="329"><DIV class=" columns sixteen" data-reactid="330"><DIV class="s-component s-text" data-reactid="331"><DIV class="s-component-content s-font-body" data-reactid="332"><P style="font-size:130%; text-align:center"><EM>pCpxP expression in S30 cell-free system</EM></P><P style="font-size:130%">Next, we put the pLV1C3-pCpxP-GFP and J23119-GFP plasmids into S30 cell-free system and record the RFU values of the mix after reaction for 1 hour to see if pCpxP promoter can express GFP protein in S30 cell-free system. The data proved that the pCpxP promoter can initiate the transcription in S30 cell-free system and has much higher expression level than J23119 promoter(Fig.10).</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="351"><SPAN data-reactid="352"><DIV data-reactid="353"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="354"><DIV class=" columns sixteen" data-reactid="355"><DIV class="s-component s-text" data-reactid="356"><DIV class="s-component-content s-font-body" data-reactid="357"><P><EM><STRONG>Fig.10 Comparison between the RFU value of pCpxP-GFP and J23119-GFP plasmids in S30 cell-free system after cultured for 1 hour</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="358"><SPAN data-reactid="359"><DIV data-reactid="360"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="361"><DIV class=" columns sixteen" data-reactid="362"><DIV class="s-component s-text" data-reactid="363"><DIV class="s-component-content s-font-body" data-reactid="364"><P style="font-size: 130%;">Due to time issue, we are unable to test for the expression level of pCpxP in different pH in S30 cell-free system. However, this data will be gained later and updated to our part pages.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="365"><SPAN data-reactid="366"><DIV data-reactid="367"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="368"><DIV class=" columns sixteen" data-reactid="369"><DIV class="s-component s-text" data-reactid="370"><DIV class="s-component-content s-font-body" data-reactid="371"><P style="font-size: 130%;"><STRONG>The expression of pCpxP promoter in S30 cell-free system</STRONG></P><P style="font-size: 130%;">We have putted plasmid with LacZ gene downstream of pCpxP promoter into the S30 cell-free system which contains X-gal and incubated for 10 hours under 37℃. The change in color indicated the successful expression of LacZ gene in the cell-free system.</P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="372"><SPAN data-reactid="373"><DIV data-reactid="374"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="375"><DIV class=" columns sixteen" data-reactid="376"><DIV class="s-component s-image" data-reactid="377"><SPAN data-reactid="378"><DIV class="s-component-content" data-reactid="379"><DIV data-reactid="380"><A href="https://static.igem.org/mediawiki/2016/d/d0/T--BGIC_China--206712064748063215_%E5%89%AF%E6%9C%AC.png" target="_blank" data-reactid="381"><DIV class="s-ratio-box" style="max-width:860px;max-height:624px;" data-reactid="382">&lt;/div&gt;</DIV></A></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="390"><SPAN data-reactid="391"><DIV data-reactid="392"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="393"><DIV class=" columns sixteen" data-reactid="394"><DIV class="s-component s-text" data-reactid="395"><DIV class="s-component-content s-font-body" data-reactid="396"><P><EM><STRONG>Fig.11 After incubated for 10 hours under 37℃</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV></LI><LI class="slide s-section-2" id="section-f_b028bf3e-fb2f-4738-a365-d6c1309bbe9c" data-reactid="397"><DIV class="lazyload s-block-section s-section" data-reactid="400"><DIV class="container" data-reactid="401"><DIV class="alignment-container " data-reactid="404"><DIV class="s-repeatable s-block s-component s-mh " data-reactid="405"><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="406"><SPAN data-reactid="407"><DIV data-reactid="408"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="409"><DIV class=" columns sixteen" data-reactid="410"><DIV class="s-button s-component" data-reactid="411"><SPAN data-reactid="412"><DIV class="s-component-content" data-reactid="413"><A class="s-action-button s-common-button s-font-body" href="https://2016.igem.org/Team:BGIC_China/Model" data-component="button" target="_blank" data-reactid="414">Mathematical Modelling</A></DIV></SPAN></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="415"><SPAN data-reactid="416"><DIV data-reactid="417"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="418"><DIV class=" columns sixteen" data-reactid="419"><DIV class="s-component s-text" data-reactid="420"><DIV class="s-component-content s-font-body" data-reactid="421"><P style="font-size: 130%;"><A target="_blank" href="https://2016.igem.org/Team:BGIC_China/Model">Click here to read our mathematical model of diffusion rate of test paper</A></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></DIV></LI><LI class="slide s-section-3" id="section-f_6fd0d778-4963-4ebf-9a0f-e303ad68e8fc" data-reactid="422"><DIV class="lazyload s-block-section s-section" data-reactid="425"><DIV class="container" data-reactid="426"><DIV class="alignment-container " data-reactid="429"><DIV class="s-repeatable s-block s-component s-mh " data-reactid="430"><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="431"><SPAN data-reactid="432"><DIV data-reactid="433"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="434"><DIV class=" columns sixteen" data-reactid="435"><DIV class="s-button s-component" data-reactid="436"><SPAN data-reactid="437"><DIV class="s-component-content" data-reactid="438"><A class="s-action-button s-common-button s-font-body" href="https://2016.igem.org/Team:BGIC_China/Demonstrate" data-component="button" target="_blank" data-reactid="439">Demonstration</A></DIV></SPAN></DIV></DIV></DIV></DIV></SPAN></DIV><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="440"><SPAN data-reactid="441"><DIV data-reactid="442"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="443"><DIV class=" columns sixteen" data-reactid="444"><DIV class="s-component s-text" data-reactid="445"><DIV class="s-component-content s-font-body" data-reactid="446"><P style="font-size: 130%;" class="s-text-color-blue"><A href="https://2016.igem.org/Team:BGIC_China/Demonstrate">Click here to read our demonstration page</A></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></DIV></LI><LI class="slide s-section-4 s-last-section-no-footer" id="section-f_cf699a11-f31b-493d-b50a-cbf5492a803d" data-reactid="447"><DIV class="lazyload s-block-section s-section" data-reactid="450"><DIV class="container" data-reactid="451"><DIV class="alignment-container " data-reactid="454"><DIV class="s-repeatable s-block s-component s-mh " data-reactid="455"><DIV class="s-block-item s-repeatable-item s-block-sortable-item " data-reactid="464"><SPAN data-reactid="465"><DIV data-reactid="466"><DIV class="s-block-item-inner clearfix" style="position:relative;" data-reactid="467"><DIV class=" columns sixteen" data-reactid="468"><DIV class="s-component s-text" data-reactid="469"><DIV class="s-component-content s-font-body" data-reactid="470"><P style="font-size:130%; text-align:center"><STRONG><EM>References</EM></STRONG></P><P style="font-size:130%"><STRONG><EM>1. Courbet, Alexis, et al. &quot;Detection of pathological biomarkers in human clinical samples via amplifying genetic switches and logic gates.&quot; Science translational medicine 7.289 (2015): 289ra83-289ra83. </EM></STRONG></P><P style="font-size:130%"><STRONG><EM>2. Kelly, Jason R., et al. &quot;Measuring the activity of BioBrick promoters using an in vivo reference standard.&quot; Journal of biological engineering 3.1 (2009): 1.​</EM></STRONG></P><P style="font-size:130%"><EM><STRONG>3. Wolfe, Alan J., et al. &quot;Signal integration by the two-component signal transduction response regulator CpxR.&quot; Journal of bacteriology 190.7 (2008): 2314-2322.</STRONG></EM></P></DIV></DIV></DIV></DIV></DIV></SPAN></DIV></DIV></DIV></DIV></DIV></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>