<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Toronto_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Toronto/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Toronto/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Toronto/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Toronto/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org" title="iGEM">            </A></LI></UL></DIV></DIV><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Toronto" title="Home">  HOME</A></LI><LI><A href="" title="Project">PROJECT</A></LI><LI style="margin-left:1px"><A href="https://2013.igem.org/Team:Toronto/Project/Background" title="Elements">BACKGROUND</A></LI><LI style="width:100px;"><A href="https://2013.igem.org/Team:Toronto/Project/Elementss" title="Elements">ELEMENTS</A></LI><LI style="width:100px;"><A href="https://2013.igem.org/Team:Toronto/Project/Assays" title="Assays">ASSAYS</A></LI><LI style="width:100px;"><A href="https://2013.igem.org/Team:Toronto/Project/Data" title="Data">DATA</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Toronto/Submitted_Parts" title="Submitted Parts">PARTS SUBMITTED</A></LI><LI><A href="https://2013.igem.org/Team:Toronto/Modeling" title="Modeling">MODELING</A></LI><LI><A href="https://2013.igem.org/Team:Toronto/Notebook" title="Notebook">NOTEBOOK</A></LI><LI><A href="https://2013.igem.org/Team:Toronto/Safety" title="Safety">SAFETY</A></LI><LI style="margin-left:1px"><A href="https://2013.igem.org/Team:Toronto/Safety/LSSE" title="LSSE">LSSE</A></LI><LI style="width:80px;"><A href="https://2013.igem.org/Team:Toronto/Safety/Form" title="EHS">FORM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Toronto/Team" title="Team">TEAM</A></LI><LI><A href="" title="Attributions">ATTRIBUTIONS</A></LI><LI style="margin-left:-20px"><A href="https://2013.igem.org/Team:Toronto/Attributions/Financial" title="Financial">FINANCIAL</A></LI><LI style="width:40px;"><A href="https://2013.igem.org/Team:Toronto/Attributions/Other" title="Other">OTHER</A></LI></DIV></DIV></DIV><DIV class="scroll"><H7><FONT color="black"><P style="text-align:center; font-size:35px;"><B>NOTEBOOK</B></P><P style="font-size:17px;"><B><U>05/26</U></B>
Prepared stock solution of LB and autoclaved empty bottles.</P><P style="font-size:17px;"><B><U>05/27</U></B>
Prepared antibiotic aliquots and practiced pouring a gel.
</P><P style="font-size:17px;"><B><U>05/28</U></B>
Practiced dilution plating and glass bead plating. Worked collaboratively on a miniprep 
and used Nanodrop to evaluate the DNA purity.
Met with Michael to talk about lambda red recombination. He is trying to make his own 
deletion strains with a pKD46 plasmid, recombinases, and PCR. Researched plasmid 
map of pBR322 to determine key characteristics like antibiotic resistance, origin of 
replication etc.</P><P style="font-size:17px;"><B><U>05/29</U></B>
Learned how to use the incubator with shaker and how to operate the Moraes lab 
Thermocycler and Spectrophotometer (details uploaded to Wiki). 
Also investigated the ideal rbs sequence for our genes. According to From Genes to 
Clones, the rbs should be 4-9bp upstream of ATG. It's stronger if you add adenines 
upstream of the RBS and it comes before minor genes in an operon. If we're interested 
in operon expression we must keep the stoichiometric balance of each minor gene by 
preserving the native rbs sequences.</P><P style="font-size:17px;"><B><U>05/30</U></B>
Project Milestones are established:
1. Primer design with pMal vector. 
2. Amplification and cloning of genes
3. Establishing selection and cultivation conditions with assays and stimuli
Sub-goals within each step were established as well. Prepared glycerol for inoculation 
and streaking. Lab members were assigned stimuli/assays to do and today we made a 
list of the ingredients from these protocols which we didn't have. 
Designed primers for CsgD after establishing RBS and other sequences.</P><P style="font-size:17px;"><B><U>06/01</U></B>
Crystal violet assay – Discussed the different procedures for this with Afiq and Seemi. </P><P style="font-size:17px;"><B><U>06/02</U></B>
First round of crystal violet protocols established.</P><P style="font-size:17px;"><B><U>06/03</U></B>
Edited primers.</P><P style="font-size:17px;"><B><U>06/05</U></B>
New cloning strategy introduced by Kristina: InFusion by Clontech. Worked to redesign 
primers for this purpose.15bp homology required between each insert.
Prepared M63 minimal media.</P><P style="font-size:17px;"><B><U>06/06</U></B>
Made a culture of MG1655 from a plate Michael brought over from BioZone for the 
purpose of making competent cells.
Edited primers.</P><P style="font-size:17px;"><B><U>06/07</U></B>
Learned to determine OD600 of growing MG culture by diluting.
Made agar and 10% glycerol from 40% stock. Poured plates and streaked the little tabs 
that the deletions came in on top of each plate by suspending in sterile LB medium. 
Strains were incubated at 7 PM.</P><P style="font-size:17px;"><B><U>06/09</U></B>
Made liquid cultures of each deletion strain so we can have glycerol stocks. Got trained 
on microscope.</P><P style="font-size:17px;"><B><U>06/12</U></B>
In order to wrap up primer design, Dr. Steipe instructed us on how to make the RBS 
overlap, checked over the PCR product schema, and suggested a double stop codon. 
Re-established goals for the project and reviewed stimuli and assays. 
Booked equipment training with Artur in the Division of Teaching Labs so we can start 
trying the ELIZA and spectrophotometer downstairs.</P><P style="font-size:17px;"><B><U>06/13</U></B>
Made a list of strains to be tested for biofilm proteins.
Incubated MG1655 at room temperature without shaking. Prepared for crystal violet 
assays with ethanol, temperature and salt stimuli on clear plates</P><P style="font-size:17px;"><B><U>06/14</U></B>
Artur trained us to use the ELIZA and Titrek Multiscan spectrophotometer. 
Made stock solutions of NaCl+LB at varying concentrations, 50mM-350mM.</P><P style="font-size:17px;"><B><U>06/16</U></B>
Pairing Stimuli with Assays – Organized a chart of stimuli variations matched with 
Assays for the different proteins of interest. Ran a sample crystal violet with MG1655 
and realized that readings are not noticeably different from each other. Made 400g/L 
PEG solution for osmolarity stimuli and ordered ingredients for yeast media. 
Grew WT cultures overnight at 250 RPM.</P><P style="font-size:17px;"><B><U>06/17</U></B>
Pipetted CV plates with varying salt concentrations. Made more salt-free medium and 
made stock CR. 
Amount of ethanol that needs to be pipetted for each stimulus treatment was 
calculated for 100% ethanol. The amount didn't work, since less than 1 microL 
was required for some. Thus, 70% ethanol was decided upon. Plate maps were 
drawn for osmolarity (using PEG-4000) and ethanol stress stimuli. CR solution 
was made.</P><P style="font-size:17px;"><B><U>06/18</U></B>
Prepared ethanol assay with MG1655 and BW25113. Prepared LB agar plates for 
colony morphology assay. Prepared yeast media (YPD) and 20% glucose solution for 
time to yeast agglutination assay.</P><P style="font-size:17px;"><B><U>06/19</U></B>
To prepare for CR assay, we diluted overnight culture to OD600=1 and did the CR 
protocol with the prepared culture. OD480 absorbance was taken for BW and MG.</P><P style="font-size:17px;"><B><U>06/20</U></B>
The PEG &amp; Ethanol crystal violet assay was conducted again.</P><P style="font-size:17px;"><B><U>06/21</U></B>
Made liquid overnight cultures of <I>E. coli</I> MG1655 and BW25113.</P><P style="font-size:17px;"><B><U>06/22</U></B>
Inoculated motility and Colony morphology plates. Made wet mounts of cultures to observe yeast (which seem to aggregate at bottom of 
tube). Made more YPD. Having trouble getting the spec to make sensible OD600 measurements when diluting 
culture.</P><P style="font-size:17px;"><B><U>06/23</U></B>
Prepared a 1:100 dilution of overnight culture and made motility plates. Used 
microscope to visualize bacteria using Koehler illumination. Made notes on colony 
morphology results i.e. the criteria on which to judge the colonies e.g. form, elevation, 
etc. Dr. Steipe suggests to validate CV by measuring OD600 throughout the growth stages, 
tapping hard into the bench for the washing step, putting more CV inside, and 
randomizing wells. He also suggests a swarm plate recipe with 'liquid agar'. 
We must first dilute our samples to OD&lt;1 as the instrument is less accurate at high 
optical densities.</P><P style="font-size:17px;"><B><U>06/24</U></B>
Meeting: CV protocol was improved on and data from the absorbance readings was 
analyzed using Excel. Looked up protocols for Ag43 assay. Yeast were obtained from the Meneghini lab and streaked onto YPD plates, then 
incubated O/N. The protocol for time to yeast agglutination was copied down.</P><P style="font-size:17px;"><B><U>06/25</U></B>
Prepared swarm/motility plates with tryptone broth agar (TBA). Prepared motility assay 
samples for BW25113.</P><P style="font-size:17px;"><B><U>06/26</U></B>
Prepared frozen stocks of yeast by inoculating them. Prepared swarming agar for 
motility assays. Pipetted another stimuli plate using OD600 = 1 cultures. Prepared cellulose assay 
(brilliant blue and congo red) plates. Made glycerol stocks of <I>E. coli</I> with pKD46, pKD3.</P><P style="font-size:17px;"><B><U>06/27</U></B>
Incubated plates for motility assay, prepares OD600=1 culture for yeast agglutination 
assay, and repeated CR assay. Finished pipetting CV sodium stimuli plates at 10 PM 
and covered from light; grew overnight cultures. The colony morphology assay was 
performed. The colony morphology assay protocol was written out. A fresh 96-well plate was 
pipetted into according to a new plate design for the osmolarity and sodium 
assay. M9 salts have NaCl, but the effects are likely small of the amount of salt 
present in M9 salts in regular LB.</P><P style="font-size:17px;"><B><U>06/28</U></B>
Made OD600=1 cultures of bacteria and subcultured yeast on YPD (still not cloudy). Did
CR and CV assay protocols and re-inoculated motility plates. Afiq contacted Lynne 
Howell about a budget-friendly PGA assay.</P><P style="font-size:17px;"><B><U>06/29</U></B>
Did CV assay on osmolarity, sodium and ethanol plates. 
Assayed for agglutination phenotype with yeast cells but there doesn't seem to be much
happening when bacteria and yeast mix.
Measured motility haloes of the two WT strains, which seem to indicate that BW has 
smaller motility radius than MG, and then returned the plates to darkness for 24 hours.</P><P style="font-size:17px;"><B><U>06/30</U></B>
Finished assay on sodium plates but there were some issues:
1. Having the lid on may not be a good idea. It fits the sterile and non-sterile plates 
differently.
2. Colour trend of resolubilized CV doesn't seem to follow that of the plate we 
pipette from.
3. Amount of liquid remaining in the plate that we pipetted from is not uniform at 
least partially due to the odd way tips stick to the multipipettor.
4. Does this assay measure biofilm or the lack of it? </P><P style="font-size:17px;"><B><U>07/02</U></B>
Data on empty 96- well plate with and without lid were taken from the 96-well 
plate reader.</P><P style="font-size:17px;"><B><U>07/03</U></B>
TBA plates were made, OD = 1 cultures were made, overnight cultures and 
sodium-less LB cultures were made for wild type strains.</P><P style="font-size:17px;"><B><U>07/04</U></B>
Overnight cultures were prepared with regular LB and sodium-less LB. TBA 
plates were also poured.
Started working on establishing stimuli protocols.
A meeting was held in preparation for the iGEMxDIYBio talk, and a general outline was 
prepared.</P><P style="font-size:17px;"><B><U>07/05</U></B>
Researched an adhesion assay that would pair with crystal violet for quantification.
Plated competent cells on LB agar, left to grow at 37C for use in colony PCR.</P><P style="font-size:17px;"><B><U>07/06</U></B></P><P style="font-size:17px;"><B><U>07/07</U></B>
Cloned csgD via colony PCR of MG1655 after resuspending lyophilized primers.</P><P style="font-size:17px;"><B><U>07/08</U></B>Ran gel electrophoresis of csgD PCR product from the day before (1% agarose TBE).
Results show no PCR product.</P><P style="font-size:17px;"><B><U>07/09</U></B>
A meeting was held with Dr. Steipe to discuss replacement of soytone with 
tryptone in TSB. A digest was performed on pMal by cutting HindIII and NdeI cut 
sites. A thermometer was broken by accident. It was cleaned up with AgNO3 
found in the Hg-spill kit.
Resuspended lyophilized primers for pofut2, csrA, fimB, mlrA, ydeH, and ompA.
Conducted colony PCR for the above 6 genes and repeated csgD PCR cloning.</P><P style="font-size:17px;"><B><U>07/10</U></B>
Regrew cultures.
Ran yesterdays PCR products on a 1% agarose TBE gel. Results show no PCR 
products except for pofut2. Note: pofut2 PCR template was template DNA whereas the 
others used genomic DNA from colony PCR.
pMAL vector was digested by other lab members with HindIII and NdeI.
NaCl assay sent by Seemi was checked over. Cultures were regrown. </P><P style="font-size:17px;"><B><U>07/11</U></B>
A dilution of overnight culture was made. 2 µL was pipetted onto LB plates. The 
plates were beaded and incubated at 37 degrees C. More LB was made.
Worked on establishing the phosphate buffered LB with Helia. Stimuli protocols were 
established as per the deadline.
iGEMxDIYBio talk was presented.
Conducted a nanodrop to measure concentration of double-digested pMAL.</P><P style="font-size:17px;"><B><U>07/12</U></B>A meeting was held with Dr. Steipe to discuss functional characterization of 
constructs. LB plates and LB kan plates were poured. Overnight cultures of 
deletion strains and MG &amp; BW was made using glycerol stocks. YPD medium 
was made and autoclaved. Phosphate buffered LB was made. A recipe for fixing 
cells was decided upon.</P><P style="font-size:17px;"><B><U>07/13</U></B></P><P style="font-size:17px;"><B><U>07/14</U></B>Overnight culture mini-experiment: do we actually need to take OD600 =1 every time? 
Colony counts indicate that there are negligible differences in the number of cells in 
culture. Monica grew cultures from glycerol stocks. 
Made an aspirator to suck up crystal violet fluids. </P><P style="font-size:17px;"><B><U>07/15</U></B>
PCR cleanup and subsequent nanodrop of pofut2 PCR product conducted by 
Samantha.
Gel of Kristina's pMAL digest (HindIII/NdeI) ran by Michael – appears to work fine.</P><P style="font-size:17px;"><B><U>07/16</U></B></P><P style="font-size:17px;"><B><U>07/17</U></B>
Colony PCR of csrA, fimB, csgD, mlrA, ydeH, and ompA using revised pre-PCR boiling 
of cells as one experimental replicant and the old protocol as another replicant.</P><P style="font-size:17px;"><B><U>07/18</U></B>
Gel ran of yesterday’s PCR products; “new” boiling colony PCR protocol found to be 
successful! Old protocol colony PCR only successful for fimB, ydeH, and ompA.</P><P style="font-size:17px;"><B><U>07/19</U></B></P><P style="font-size:17px;"><B><U>07/20</U></B></P><P style="font-size:17px;"><B><U>07/21</U></B></P><P style="font-size:17px;"><B><U>07/22</U></B></P><P style="font-size:17px;"><B><U>07/23</U></B>Got trained on Houry plate reader.
Tried CV, Adhesion, CR and CF on 3 dilutions of overnight culture, grown for 48h. 
HindIII/NdeI digest of pMAL followed by gel electrophoresis and gel extraction of the 
large (5444bp) fragment for use in in-fusion.</P><P style="font-size:17px;"><B><U>07/24</U></B></P><P style="font-size:17px;"><B><U>07/25</U></B></P><P style="font-size:17px;"><B><U>07/26</U></B>Transformed MG1655 with pMAL glycerol stock via electroporation. Plated on LB agar 
with ampicillin.</P><P style="font-size:17px;"><B><U>07/27</U></B></P><P style="font-size:17px;"><B><U>07/28</U></B>
The solutions for the stimuli were all prepared: indole, ethanol, sucrose, salt and the 
buffered LB. They were autoclaved separately to prevent precipitation and mixed together. This was done a couple of times and perfected based on the solubility and 
other properties of the solutions. Set up plate plans for the strains.</P><P style="font-size:17px;"><B><U>07/29</U></B></P><P style="font-size:17px;"><B><U>07/30</U></B>
2 WT overnight cultures in phosphate buffered LB, 50% EtOH solution was 
prepared using phosphate buffered LB as a solvent. 2 M sucrose, 1 M indole, 
and 1 M NaCl was also prepared in a similar way but the indole solution failed as 
solubility for this substance is only 0.19 g/100 mL. A graph was created 
comparing weight measurements of standard weights on the fine balance and 
the regular balance. It was discovered that both were reasonably accurate.
Made glycerol stocks of MG1655+pMAL.
Conducted miniprep of pMAL.
Conducted nanodrop of miniprepped pMAL.
HindIII/NdeI digest of pMAL.</P><P style="font-size:17px;"><B><U>07/31</U></B>
Gel electrophoresis of pMAL digestion and gel extraction of large fragment. Nanodrop of
large fragment (poor yield).
Developed a phases plan for the assays.
1. Validate the machine: does it read concentration or volume?
2. Validate working concentrations of dyes
3. Validate petri-dish based assays with dilutions
4. Validate assays themselves. </P><P style="font-size:17px;"><B><U>08/01</U></B>
Constant amount, not concentration, matters for absorbance. See CV data on July 31. 
Another digestion of pMAL, along with extraction and nanodrop. Poor yield once again.
Started another overnight digest for Kristina to gel extract.
Required amounts of crystal violet and calcofluor dye were pipetted into a 96 
well-plate as desired.</P><P style="font-size:17px;"><B><U>08/02</U></B>
Re-made phosphate buffered LB.
Excised 5444bp fragment of HindIII/NdeI pMAL digest and gave to Kristina to extract 
with her kit.
Used speedvac DNA concentrator to concentrate the gel extraction products.
Julianne and Helia used the 5444bp and pofut2 PCR product to attempt in-fusion 
construction of our construct (unsuccessful).</P><P style="font-size:17px;"><B><U>08/03</U></B>
Solutions were prepared but had to be remade since stimuli, LB and phosphate 
buffer solutions had to all be autoclaved separately (otherwise they would form 
precipitates).</P><P style="font-size:17px;"><B><U>08/04</U></B></P><P style="font-size:17px;"><B><U>08/05</U></B>
Started working on safety scenarios with Al. Worked on remaking the motility plates 
using a 0.3% agar in LB solution instead of TBA with Congo Red and Coomassie Blue.</P><P style="font-size:17px;"><B><U>08/06</U></B>
Overnight cultures from glycerol stocks were made for all 23 E. coli strains.</P><P style="font-size:17px;"><B><U>08/07</U></B></P><P style="font-size:17px;"><B><U>08/08</U></B>
Serial dilutions of CF, CV, CR and EtBr planned out. Need to find a cell lysis solution for
EtBr since it binds with DNA. Data will be analyzed with Excel. </P><P style="font-size:17px;"><B><U>08/09</U></B>
Microscope slides were made containing cultures from all 23 strains. The crystal 
violet and calcofluor dye plates were read on the fluorescence spec.
A serial dilution of 1:2 was done across 3 rows of the washed black plate for 
calcofluor, starting from 1:100 dilution of 1 g/L. A serial dilution of 1:5 of original 
CR solution was done. Both sets of data were read on the spec in the Houry Lab.
Graphs were prepared for the initial CV serial dilution, the recent CR and CF 
data.Worked on motility assay validation.</P><P style="font-size:17px;"><B><U>08/10</U></B>Overnight cultures of MG, BW, (delta)csgD, (delta)fimH and (delta)ompR were 
made.Worked on using different strains for validating motility assay.</P><P style="font-size:17px;"><B><U>08/11</U></B>A 1:2 serial dilution of a 1:100 dilution of 0.1% CV solution was done. 
Absorbance readings were taken. EtBr protocol was modified.
To get the standard dilution curve of EtBr, we placed cells inside and add increasing 
volumes of ethanol:acetic acid.Measured OD600 for MG1655 and B25113 and dropped 1 microlitre onto the motility 
plates.</P><P style="font-size:17px;"><B><U>08/12</U></B>
The EtBr protocol was performed over 4 h. 3 rows had a constant volume and 
concentration of EtBr, whereas 3 others had a constant of EtOH: acetic acid 
solution. 3 control wells had Milli-Q, 3 had EtBr and LB, 3 had EtOH: acetic acid 
and LB. Regular and kan CR plates were made. 2 replicates were made for each
type of plate with MG and BW to test whether the glycerol stocks of the WT 
strains are contaminated.</P><P style="font-size:17px;"><B><U>08/13</U></B>
Decided that motility assay needs to be repeated by just dipping the pipette tip into the 
overnight culture. Made a new batch of motility plates and made 70% EtOH.
The EtBr protocol was repeated with a 1:10 dilution of 0.5 mg/mL EtBr. CR + BB 
plates were used to replate glycerol stocks of wild type strains.</P><P style="font-size:17px;"><B><U>08/14</U></B>
Worked with Al on the safety scenarios guide. Re-made TSB and TB solutions.
EtBr protocol was repeated but only for increasing concentrations of EtOH: acetic
acid with constant volume of the 1:10 dilution of 0.5 g/L EtBr solution. Non-kan 
antibiotic plates were made to test contamination of some deletion strains. Insert 
region for our master plasmid was constructed through PCR extension of two 
complimentary template primers. In-fusion of this into the 5444bp fragment of 
HindIII/NdeI pMAL digest to create pEBS (<I>E. coli</I> biofilm system) expression plasmid 
and subsequent transformation of Stellar competent cells was carried out.</P><P style="font-size:17px;"><B><U>08/15</U></B>
Made Coomassie blue-Congo Red agar for Colony Morphology plates. Analyzed plate 
reader data. Isolated areas on the graph where the variation of fluorescence with 
concentration is linear to confirm with another pipetting scheme. Question: does 
background fluorescence interfere with that of bound EtBr? We need a lysis buffer that 
will make suspensions clear for the EtBr assay. Dr. Steipe recommends making a 
TBE-EtBr stock solution to avoid pipetting small volumes. 
Revision of protocol: should include sodium azide. In-fusion insert was verified to be 
correct on a 1% TBE agarose gel.</P><P style="font-size:17px;"><B><U>08/16</U></B>
Prepared MG1655, BW25113, and deletion strains for csgD, csgA, and bcsA with with 
stimuli.
PCR of fim operon, csgBA operon, and bcsA.</P><P style="font-size:17px;"><B><U>08/17</U></B>
In-fusion assembly of pEBS trial #2.</P><P style="font-size:17px;"><B><U>08/18</U></B>
No colonies from pEBS assembly trial #2.</P><P style="font-size:17px;"><B><U>08/19</U></B>
Made sodium azide and culture suspension in TBE. Also made more ethanol: acetone 
80:20 solution with anhydrous ethanol and acetone from the fumehood. Results show 
linear variation of RFU with the concentration of culture (treated with sodium azide.)</P><P style="font-size:17px;"><B><U>08/20</U></B></P><P style="font-size:17px;"><B><U>08/21</U></B>
Tried sodium azide protocol with 2x and 1/2x EtBr in comparison with 2 days ago.</P><P style="font-size:17px;"><B><U>08/22</U></B></P><P style="font-size:17px;"><B><U>08/23</U></B>Made a list of strains to be plated from glycerol stocks on Monday when we have dry 
ice. Corrected working concentration of Kan to 5 mL/L on wiki based on publications, 
and grew stimuli cultures thus.</P><P style="font-size:17px;"><B><U>08/24</U></B></P><P style="font-size:17px;"><B><U>08/25</U></B>
Grew 6 uncontaminated cultures in Phosphate buffered Kan LB and put in shaker.</P><P style="font-size:17px;"><B><U>08/26</U></B></P><P style="font-size:17px;"><B><U>08/27</U></B></P><P style="font-size:17px;"><B><U>08/28</U></B></P><P style="font-size:17px;"><B><U>08/29</U></B>
In-fusion assembly of pEBS trial #3. Transformed cells plated on ampicilin plates.</P><P style="font-size:17px;"><B><U>08/30</U></B>
Colonies found from pEBS assembly #3 and LB+Amp overnight cultures made.</P><P style="font-size:17px;"><B><U>08/31</U></B>
Miniprep of overnight cultures to extract supposed pEBS.Overnight analytical digest using BseRI and ScaI to confirm successful plasmid 
construction.</P><P style="font-size:17px;"><B><U>09/01</U></B>
Digested samples ran on 1% TBE agarose gel to confirm successful pEBS construction
Preparative digest of successful pEBS using BseRI. This linearized vector will be used 
to accept target genes amplified by PCR from MG1655.</P><P style="font-size:17px;"><B><U>09/01</U></B></P><P style="font-size:17px;"><B><U>09/02</U></B>
In-fusion of target genes (ydeH, fimB, csgD, ompA, mlrA) into pEBS.</P><P style="font-size:17px;"><B><U>09/03</U></B>
Additional analytical digests of pEBS, PCR products from 08/16.
pEBS sent for sequencing using M13 primers from TCAG facility at Sick Kids Hospital.
5 in-fusion colonies for each target gene started as overnight cultures.
MG1655 and BW25113 overnight cultures started from glycerol stocks</P><P style="font-size:17px;"><B><U>09/04</U></B>
Chemically competent MG1655 and BW25113 made.
Minipreps of in-fusion overnight cultures. 
Isolated plasmids digested with ScaI and NotI to confirm successful cloning of target 
genes into pEBS.
Gel ran to confirm this. Results negative. Another pEBS linearization/preparative digest 
conducted.</P><P style="font-size:17px;"><B><U>09/05</U></B>
Gel ran to confirm that insert/target genes were not degraded and are of correct length. 
fimB, csgD, ompA, ydeH, and mlrA intact and correct. Fim operon, bcsA, and csgBA 
were not properly amplified.
In-fusion of target genes (ydeH, fimB, csgD, ompA, mlrA) into pEBS trial #2.</P><P style="font-size:17px;"><B><U>09/06</U></B>
Successful colonies from target genes In-fusion trial #2 started as overnight cultures; 5 
separate colonies per gene along with linear backbone control.
pEBS sequenced at TCAG and confirmed correct.
Assays for (delta)rcsA, (delta)fimA and (delta)csgD were performed.</P><P style="font-size:17px;"><B><U>09/07</U></B>
Minipreps of pEBS+target genes overnight cultures.
Assays for (delta)bcsA, (delta)pgaB and (delta)ompR were performed.</P><P style="font-size:17px;"><B><U>09/08</U></B>
Digest of the day before’s minipreps using NotI to confirm correct insert’s (target gene’s)
presence. 1% TAE agarose gel ran to confirm this. Positive results found for all 5 genes.
Assays for (delta)fimA, and (delta)ompA were performed.</P><P style="font-size:17px;"><B><U>09/09</U></B>
Analytical digest of pSB1C3 to confirm integrity prior to transfer of target genes from 
pEBS to it for biobrick submission.</P><P style="font-size:17px;"><B><U>09/10</U></B>
Transformation of MG1655 and BW25113 (via heat shock) with pEBS+target genes. 
Constructs used to transform sent for sequencing.
Gel of pSB1C3 digest confirms it is intact.</P><P style="font-size:17px;"><B><U>09/11</U></B>
Analytical digests of yesterdays transformation plasmids identify them as being pEBS 
without target genes. Contamination or human error (using the wrong samples) to be 
blamed. Sequencing results confirm this. Therefore cannot use recently transformed 
strains.</P><P style="font-size:17px;"><B><U>09/12</U></B>
Other samples exist of “successful” pEBS+target gene plasmids. Digest all in 
preparation of ligating target genes into pSB1C3. Linearize pSB1C3.
Digested target genes taken from pEBS+target genes constructs confirmed to be of 
correct size by gel electrophoresis.
Ligation of target genes into pSB1C3 and subsequent transformation into DH5 α
competent cells.</P><P style="font-size:17px;"><B><U>09/13</U></B>
Colonies found from ligation reactions (target genes into pSB1C3) and 10 overnight 
cultures started per gene to allow screening for false positives.</P><P style="font-size:17px;"><B><U>09/14</U></B></P><P style="font-size:17px;"><B><U>09/15</U></B>
Minipreps of ligation overnight cultures and subsequent analytical digests.
Samples digested to screen for false positive ligations. Minimum one found per gene 
and prepared for submission to the registry.
Electrocompetent MG1655 and BW25113 prepared and transformed with pEBS+
(csgD/ydeH/ompA/fimB/mlrA) to be used in assays to characterize new biobricks under 
lac promoter control. Glycerol stocks of pEBS+target genes and pEBS in stellar E. coli 
made.</P><P style="font-size:17px;"><B><U>09/16</U></B>
pSB1C3+biobricks (target genes) shipped to the registry and sent for sequencing.</P><P style="font-size:17px;"><B><U>09/17</U></B></P><P style="font-size:17px;"><B><U>09/18</U></B></P><P style="font-size:17px;"><B><U>09/19</U></B>
0.3 g/L stock of IPTG in buffered LB. It was induced at 0.3 mM in each well at the 
beginning of stimuli inoculation.
The Biobrick inserts (mlrA, ompA, csgD and fimB) were transformed into BW previously.
Overnight cultures of these were inoculated with various stimuli.</P><P style="font-size:17px;"><B><U>09/20</U></B>
BW-WT and MG-WT were inoculated along with various stimuli.</P><P style="font-size:17px;"><B><U>09/21</U></B>
Overnight culture of BW-Y (containing ydeH insert) was inoculated with various stimuli.
The BW strains containing various Biobrick inserts that were inoculated 2 days earlier 
were assayed.</P><P style="font-size:17px;"><B><U>09/22</U></B>
Amount of agar plate ingredients were calculated for “media” plates (e.g., TB plates, 
TSB plates).</P><P style="font-size:17px;"><B><U>09/23</U></B>
BW and MG Wild type strains were assayed.</P><P style="font-size:17px;"><B><U>09/24</U></B>
Colony morphology plates were poured for various media. Overnight cultures of BW 
with Biobrick inserts were made.</P><P style="font-size:17px;"><B><U>09/25</U></B>
Overnight cultures made the day before were streaked onto dried colony morphology 
media plates.</P><P style="font-size:17px;"><B><U>09/26</U></B>
pSB1C3 biobrick constructs sequencing results analyzed. fimB and ompA biobricks 
appear to have mutations and require further investigation.</P><P style="font-size:17px;"><B><U>09/27</U></B></P></FONT></H7></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Toronto/Notebook">http://2013.igem.org/Team:Toronto/Notebook</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Toronto/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Toronto/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Toronto/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Toronto/Notebook&amp;oldid=239004" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>