"
Page
Discussion
View source
History
teams
Log in
 
Team:Toronto
From 2013.igem.org
                      
  HOME
PROJECT
BACKGROUND
ELEMENTS
ASSAYS
DATA
PARTS SUBMITTED
MODELING
NOTEBOOK
SAFETY
LSSE
FORM
TEAM
ATTRIBUTIONS
FINANCIAL
OTHER
BIOFILM ENGINEERING
What are biofilms?
Biofilms are communities of microbes where the cells aggregate onto a surface, and are bound together by secreted extramembranous materials made of a mixture of specialised carbohydrates and extramembrane proteins. The special environment in the biofilm allows cells to survive harsher environments.
What is the “biofilm response”?
Bacterial cells can exist in multiple physiological states, switching between them as a direct consequence of environmental factors. Common ones include heat shock, nutrient (carbohydrate or amino acid) starvation, metal micronutrient starvation, etc.
Gene expression in each of these physiological states is mediated by the changing the sigma (σ) factor involved in transcript synthesis. Each σ factor has its own consensus sequence; the σ factor on the RNA polymerase recognises and binds to promoter regions. When a σ factor is active, its effect is to shift which fractions of the genome are preferably expressed, because of the different promoter recognition consensus sequences. The sigma factor that is involved in the stationary phase (i.e. no motility, no reproduction) of E. coli strains is the σs factor. It directs the expression of genes necessary to induce biofilm and aggregation behaviour are a subset of those genes whose promoters are recognised by the σs factor.
Modulating and Measuring the Biofilm Response
The focus of our project is to characterise the physical and chemical manifestations of the biofilm response for cells in which key genes for both structural and regulatory aspects of the biofilm response  have been either overexpressed or deleted. The numerous assays that need to be done drove us to develop a standardised battery of assays, so that the cell can be studied as an entire system, as opposed to the methods used in past literature that only measured the effects of genetic engineering on just one manifestation of the biofilm response.
Defect in the BioBrick Paradigm
Due to the mechanics of the BioBrick Assembly method, the BioBrick paradigm lends itself to the construction of gene parts that are necessarily additive. Consequently, apart from using “additive” exotic methods such as RNAi to disrupt gene expression to achieve functional gene silencing, gene deletions from a genome (negative “additive” modifications) simply cannot be submitted to the Gene Parts Registry.
The spectacular success of the metabolic engineering field eminently shows the defect in this paradigm. Successful engineering of model organisms is based on using chemical kinetics and metabolic networks, where metabolite flux is directed by both gene up-regulation and deletion. Production of n-butanol and lycopene in metabolically engineered E. coli has needed strategic knockouts of genes to direct carbon flux in the metabolic pathways towards the more economically valuable compounds: see Alper (2005) and Atsumi (2008).
Since deletions from a cell's genome is another means to engineering the cell as a system, we have also characterised the biofilm response of a set of knockout E. coli. Unfortunately, these systems cannot be submitted to the BioBrick registry because they do not conform to the BioBrick paradigm, an issue we hope that the iGEM competition can address.
Works Cited
Alper, H. Miyaoku, K. Stephanopoulos, G. "Construction of lycopene-overproducing E. coli strains by combining systematic and combinatorial gene knockout targets". Nature Biotechnology 23, 612 - 616. 2005. Published online: 10 April 2005 | doi:10.1038/nbt1083
Atsumi, S. Cann, A. F. Connor, M. R., et. al. "Metabolic engineering of Escherichia coli for 1-butanol production". Metabolic Engineering. Volume 10, Issue 6, November 2008, Pages 305–311.
Retrieved from " http://2013.igem.org/Team:Toronto "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
