<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UIOWA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UIOWA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UIOWA"><DIV class="menu_item direct_link">
            HOME
        </DIV></A><DIV class="menu_item">
        TEAM
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UIOWA/Team"><DIV class="submenu_item">
                Team Members
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Collaborations"><DIV class="submenu_item">
                Collaborations
            </DIV></A></DIV><DIV class="menu_item">
        PROJECT
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UIOWA/Description"><DIV class="submenu_item">
                Description
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Design"><DIV class="submenu_item">
                Design
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Experiments"><DIV class="submenu_item">
                Experiments
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Notebook"><DIV class="submenu_item">
                Notebook
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/InterLab"><DIV class="submenu_item">
                InterLab
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Model"><DIV class="submenu_item">
                Model
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Results"><DIV class="submenu_item">
                Results
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Demonstrate"><DIV class="submenu_item">
                Demonstrate
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Improve"><DIV class="submenu_item">
                Improve
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Attributions"><DIV class="submenu_item">
                Attributions
            </DIV></A></DIV><DIV class="menu_item">
        PARTS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UIOWA/Parts"><DIV class="submenu_item">
                Parts Overview
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Basic_Part"><DIV class="submenu_item">
                Basic Parts
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Composite_Part"><DIV class="submenu_item">
                Composite Parts
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Part_Collection"><DIV class="submenu_item">
                Part Collection
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UIOWA/Safety"><DIV class="menu_item direct_link">
            SAFETY
        </DIV></A><DIV class="menu_item">
        HUMAN PRACTICES
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UIOWA/Human_Practices"><DIV class="submenu_item">
                Human Practices
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Public_Engagement"><DIV class="submenu_item">
                Education &amp; Engagement
            </DIV></A></DIV><DIV class="menu_item">
        AWARDS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UIOWA/Measurement"><DIV class="submenu_item">
                Measurement
            </DIV></A><A href="https://2018.igem.org/Team:UIOWA/Model"><DIV class="submenu_item">
                Model
            </DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=UIOWA"><DIV class="menu_item direct_link">
            JUDGING FORM ⇗
        </DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><H1>
    Experiments and Protocols
</H1><DIV class="collapsible_accordion"><H1> Agarose Gels and Agarose Gel Electrophoresis
        </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction:</B> Agarose gel electrophoresis is useful for evaluating DNA quality, separating linear DNA fragments by size,  and for estimating relative abundance of specific DNA fragments prior to conducting cloning procedures.
            </P><P><B>Materials</B><UL style="list-style-type:circle"><LI>Dedicated Flask</LI><LI>Agarose (depends on 1% agarose gel desired)</LI><LI>30mL 1X TAE buffer</LI><LI>Paper Towel Plug</LI><LI>Gel Red (keep dark - it's light sensitive)</LI></UL></P><P><B>Procedure</B></P><P><I>Assembling Gel Bed</I><OL type="1"><LI>Place Gel Bed in between the Gel Bed Holder and lock in place by pressing the Holder Walls together and locking them using the handle.</LI><LI>Make sure the Gel Bed Holder is stable by using the Level Tool. Adjust the Holder using the knobs until it is stable.</LI><LI>Place Well Comb.</LI></OL></P><P><I>Preparing the Agarose</I><OL type="1"><LI>Add 30mL of 1x TAE Buffer to the flask.</LI><LI>Measure and add agarose to the flask according to percentage of gel desired.</LI>
            1% agarose  = 0.300g agarose 
                <LI>Plug the flask with paper towel and microwave agarose and 1x TAE  buffer 1 min on High until just beginning to bubble
                    Swirl to aid in dissolving the agarose, and continue microwaving 10 seconds at a time, until all the agarose is in solution.
                </LI><LI>Discard paper plug and add 1 μl of gel red for every 10 mls of agarose (10 μl per 100 ml).....[to be efficient use 1.5 uL in 30mL]</LI><LI>Gel Red is light sensitive so do this step fast and keep the gel red covered.</LI><LI>Swirl solution.</LI></OL></P><P><I>Pour the Gel</I><OL type="1"><LI>Slowly pour into prepared gel beds (small 7 cm square) [ use 40 mls for the 10 cm rectangular gel beds.]  </LI><LI>Remove any bubbles (or move them away from the wells) created during pouring using a pipet tip.</LI><LI>Cover the gel from light and allow to set at room temperature. (Or store  in bags or boxes at 4oC in the dark and create a humidified environment by adding TAE-soaked paper towels to the bag or box.)</LI></OL></P><P><I>Gel Electrophoresis</I><OL type="1"><LI>Remove Well Comb and place the Gel Bed to the Electrophoresis Machine.  The wells in the gel should be closer to the Negative Terminal.</LI><LI>Mix 10-20 uL of DNA Sample with Loading dye (5 ul per 20 uL of sample)</LI><LI>Mix by pipetting in and out or by lightly vortexing.</LI><LI>Carefully load the Sample DNA+Loading Dye into the well. Load 2.5 uL of DNA Ladder into a well.
                    Keep track of what each well contains.</LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> Clean-up for PCR (and other enzymatic reactions)
        </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction:</B> The QIAquick system is suitable for fast cleanup of up to 10 μg of DNA fragments from enzymatic reactions. Enzyme contamination of DNA samples can interfere with subsequent downstream applications. QIAquick Kits can be used for highly efficient removal of a broad spectrum of enzymes widely used in molecular biology. The QIAquick system combines the convenience of spin-column technology with the selective binding properties of a uniquely designed silica membrane. Special buffers provided with each kit are optimized for efficient recovery of DNA and removal of contaminants in each specific application. DNA adsorbs to the silica membrane in the presence of high concentrations of salt, while contaminants pass through the column. Impurities are efficiently washed away, and pure DNA is eluted with Tris buffer or water.
                </P><P>Note 1: During the DNA adsorption step, unwanted primers and impurities, such as salts, enzymes, unincorporated nucleotides, agarose, dyes, ethidium bromide, oils, and detergents (e.g., DMSO, Tween® 20) do not bind to the silica membrane but flow through the column. Salts are quantitatively washed away by the ethanol-containing Buffer PE. Any residual Buffer PE, which may interfere with subsequent enzymatic reactions, is removed by an additional centrifugation step.</P><P>Note 2: Elution efficiency is strongly dependent on the salt concentration and pH of the elution buffer. Contrary to adsorption, elution is most efficient under basic conditions and low salt concentrations. DNA is eluted with 50 μl or 30 μl of the provided Buffer EB (10 mM Tris·Cl, pH 8.5). The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water to elute, make sure that the pH is within this range. In addition, DNA must be stored at –20°C when eluted with water since DNA may degrade in the absence of a buffering agent.</P><P>Note 3: DNA yield depends on the following three factors: the volume of elution buffer, how the buffer is applied to the column, and the incubation time of the buffer on the column. To completely cover the QIAquick membrane, use 100–200 μl elution buffer. This ensures maximum yield, even when not applied directly to the center of the membrane. Elution with ≤50 μl requires the buffer to be added directly to the center of the membrane, and if elution is done with the minimum recommended volume of 30 μl, an additional 1 minute incubation is required for optimal yield. DNA will be up to 1.7 times more concentrated if the QIAquick column is incubated for 1 minute with 30 μl of elution buffer, than if it is eluted in 50 μl without incubation (Figure 4, page 13).</P><P><B>Materials</B><UL style="list-style-type:circle"><LI>Spin columns</LI><LI>Buffer PE - check that ethanol has already been added (10mM Tris-HCl pH7.5, 80% ethanol)</LI><LI>Buffer PB - (5M Gu-HCl, 30% isopropanol) </LI><LI>Microfuge</LI><LI>Sterile Distilled H2O </LI><LI>EB (10 mM Tris·Cl, pH 8.5)</LI></UL></P><P><B>Procedure</B></P><P><I>Adsoption</I><OL type="1"><LI>Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
                For example, add 500 μl of Buffer PB to 100 μl PCR sample.</LI><LI>If pH Indicator I has been added to Buffer PB, check that the color of the mixture is yellow.
                If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. </LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.</LI><LI>To bind DNA, apply the sample to the QIAquick column and centrifuge (high or 13,000 rpm) for 30–60 s.</LI><LI>Discard flow-through. Place the QIAquick column back into the same tube.</LI></OL></P><P><I>Wash</I><OL type="1"><LI>To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge (high or 13,000 rpm) for 30–60 s.</LI><LI>Discard flow-through and place the QIAquick column back in the same tube. Centrifuge the column for an additional 1 min.
                IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.</LI><LI>Plug the flask with paper towel and microwave agarose and 1x TAE  buffer 1 min on High until just beginning to bubble
                        Swirl to aid in dissolving the agarose, and continue microwaving 10 seconds at a time, until all the agarose is in solution. </LI><LI>Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</LI></OL></P><P><I>Recover DNA</I><OL type="1"><LI>To elute DNA, add 30-50 μl EB (to the center of the QIAquick membrane and centrifuge the column (high or 13,000 rpm) for 1 min. For increased DNA concentration, use 30 μl rather than 50 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.
                IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 μl from 50 μl elution buffer volume, and 28 μl from 30 μl elution buffer.
                Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent.</LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.</LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> Extraction of DNA fragments from an Agarose Gel
                </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction:</B> Enzyme contamination of DNA samples can interfere with subsequent downstream applications.  Therefore DNA fragments from enzymatic reactions and agarose gels often must be &quot;cleaned&quot; or purified prior to further use. QIAquick Kits can be used for highly efficient removal of a broad spectrum of enzymes widely used in molecular biology.  The Qiagen Gel Extraction kit allows recovery of fragments as small as 70 bp and as large as 10 kb.   Buffer QG in the QIAquick Gel Extraction Kit solubilizes the agarose gel slice and provides the appropriate conditions for binding of DNA to the silica membrane in the purification column.  DNA adsorbs to the silica membrane in the presence of high concentrations of salt, while contaminants pass through the column. Impurities are efficiently washed away, and pure DNA is eluted with Tris buffer or water.
                    </P><P>
                        Buffer QG contains an integrated pH Indicator allowing easy determination of the optimal pH for DNA binding. DNA adsorption requires a pH ≤ 7.5, and the pH Indicator in the buffers will appear yellow in this range. If the pH is &gt;7.5, which can occur if during agarose gel electrophoresis, the electrophoresis buffer had been used repeatedly or incorrectly prepared, or if the buffer used in an enzymatic reaction is strongly basic and has a high buffering capacity, the binding mixture turns orange or violet. This means that the pH of the sample exceeds the buffering capacity of Buffer QG and DNA adsorption will be inefficient. In these cases, the pH of the binding mixture can easily be corrected by addition of a small volume of 3 M sodium acetate*, pH 5.0, before proceeding with the protocol.
                    </P><P><B>Materials</B><UL style="list-style-type:circle"><LI>Buffer QG (yellow at pH less than or equal to 7.5)</LI><LI>Buffer PE  (Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume).</LI><LI>Isopropanol (100%)  </LI><LI>Heating block or water bath at 50°C are required.</LI></UL></P><P><B>Procedure</B></P><P><I>Cut the fragment out of an agarose gel</I><OL type="1"><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel on a UV box.  Important:  use eye and skin protection while doing this; the UV radiation is harmful.</LI><LI> Weigh the gel slice in a colorless tube whose weight has been pre-determined. </LI></OL></P><P><I>Dissolve the agarose</I><OL type="1"><LI> Add 3 volumes Buffer QG to 1 volume gel
                            </LI><LI> Note 1:  100 mg gel ~ 100 μl</LI><LI>Note 2:  The maximum amount of gel per spin column is 400 mg.</LI><LI>Note 3:  For &gt;2% agarose gels, add 6 volumes Buffer QG.</LI><LI>Incubate at 50°C for 10 min or until the gel slice has completely dissolved.  Vortex the tube every 2–3 min to help dissolve gel.</LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 μl 3 M sodium acetate, pH 5.0, and mix. <I>Note 1:  If the DNA will be used for salt-sensitive applications (e.g., sequencing, blunt- ended ligation), let the column stand 2–5 min after addition of Buffer PE.</I></LI><LI>Centrifuge the QIAquick column in the provided 2 ml collection tube for 1 min to remove residual wash buffer. <I>Note 2:  This seems trivial and is easy to forget, but it is very important to remove all residual buffer.</I></LI></OL></P><P><I>Bind the DNA to a Qiaquick Column</I><OL type="1"><LI> Add 1 gel volume isopropanol to the sample and mix. </LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.  Apply the sample to the QIAquick column and centrifuge for 1 min.</LI><LI>Discard flow-through and place the QIAquick column back into the same tube. For sample volumes of &gt;800 μl, load and spin again.</LI><LI>To wash, add 750 μl Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube.</LI><LI>Centrifuge the QIAquick column in the provided 2 ml collection tube for 1 min to remove residual wash buffer.  <I>Note 2:  This seems trivial and is easy to forget, but it is very important to remove all residual buffer.</I></LI></OL></P><P><I>Elute the DNA from the column</I><OL type="1"><LI> Place QIAquick column into a clean 1.5 ml microcentrifuge tube. </LI><LI> To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min.
                        </LI><LI>Note 1:  For increased DNA concentration, add 30 μl Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. </LI><LI>Note 2:  After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time to up to 4 min can increase the yield of purified DNA.</LI></OL></P><P><I>Evaluate Recovery</I><OL type="1"><LI> Option 1: If purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.  This, of course, uses up your precious sample, but is sometimes necessary to determine whether your DNA has been recovered and whether what was recovered is high quality. </LI><LI> Option 2: Alternatively, use the nanospec to get a DNA concentration.
                           </LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> Ligation Protocol With T4 DNA Ligase (NEB Protocol)
                    </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Full protocol <A href="https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202">here!</A></B></P><P><B>Materials</B><UL style="list-style-type:circle"><LI>10X T4 DNA Ligase Reaction Buffer</LI><LI>T4 DNA Ligase</LI><LI>Vector DNA of known concentration  </LI><LI>Insert DNA of known concentration</LI><LI>Nuclease-free water</LI></UL></P><P><B>Procedure</B></P><P><I>Set up the T4 DNA Ligase Reaction</I><OL type="1"><LI>Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.</LI><LI>   Use <A href="http://nebiocalculator.neb.com/#!/ligation">NEBioCalculator</A> to calculate molar ratios for insert and vector DNA of specific sizes.  The desired molar ratio is 1:3 vector to insert </LI><LI>Set up the following reaction in a microcentrifuge tube on ice:</LI><TABLE><TBODY><TR><TH>Component</TH><TH>Volume (µL)</TH></TR><TR><TD>10x T4 DNA Ligase Buffer</TD><TD>2</TD></TR><TR><TD>Vector DNA: (x kb)</TD></TR><TR><TD>Insert DNA: (y kb)</TD></TR><TR><TD>Nuclease Free Water</TD><TD>17</TD></TR><TR><TD>T4 DNA Ligase (Add LAST!)</TD><TD>1</TD></TR><TR><TD><B>Total</B></TD><TD>20</TD></TR></TBODY></TABLE><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.</LI><LI>Heat inactivate at 65 degrees C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.</LI>
                        Use 25 uL DH5α cells, and add 2 uL of reaction mixture.
                        </OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> Qiagen Plasmid Mini-Prep
                        </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Materials</B><UL style="list-style-type:circle"><LI>Buffer P1:  50 mM Tris HCl (pH 8), 10 mM EDTA, 100, 100 micrograms/ml (DNase free - RNaseA);  store in the refrigerator</LI><LI>Buffer P2:  200 mM NaOH, 1% SDS (store at RT)</LI><LI>Buffer N3:  4.2 M Gu-HCl, 0.9M potassium acetate (N3 pH 4.8)  (store at RT)  </LI><LI>Buffer PB:  5M Gu-HCl (store at RT)</LI><LI>Buffer EB:  10 mM Tris, pH 8.5 (store at RT)</LI><LI>Qiaprep Spin Columns</LI><LI>Microfuge</LI><LI>Ethanol</LI></UL></P><P><B>Procedure</B></P><P><I>Prepare Buffer P1</I><OL type="1"><LI>(Optional) Add LyseBlue reagent to Buffer P1 at a ratio of 1 to 1000</LI><LI> Add the provided RNase A  100 μg/mL to Buffer P1, mix, and store at 2–8°C.</LI><LI>Add ethanol (100%) to Buffer PE before use (see bottle label for volume). </LI></OL></P><P><I>Prepare the Cell Lysate</I><OL type="1"><LI>RESUSPEND THE CELLS.  Pellet 1–5 ml bacterial overnight culture by centrifugation at &gt;8000 rpm</LI>
                            (6800 x g) for 3 min at room temperature (15–25°C).
                            <LI>Resuspend pelleted bacterial cells in 250 μl Buffer P1 and transfer to a microcentrifuge tube.</LI><LI>LYSE THE CELLS USING ALKALINE CONDITIONS.  Add 250 μl Buffer P2 and mix thoroughly by inverting the tube</LI>
                            4–6 times until the solution becomes clear.  Do not allow the lysis reaction to proceed for more than 5 min. Note:  If using LyseBlue reagent, the solution will turn blue.
                            <LI>NEUTRALIZE THE LYSATE.  Add 350 μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, the solution will turn colorless</LI><LI>CLEAR THE LYSATE.  Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a microcentrifuge. </LI></OL></P><P><I>Purify the DNA </I><OL type="1"><LI>Apply the supernatant from step 8 to the QIAprep spin column by decanting or pipetting. Centrifuge for 30–60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source.</LI><LI>Recommended: Wash the QIAprep spin column by adding 0.5 ml Buffer PB.  Centrifuge for 30–60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source.</LI>
                            Note: This step is only required when using endA+ strains or other bacteria strains with high nuclease activity or carbohydrate content.
                            <LI>Wash the QIAprep spin column by adding 0.75 ml Buffer PE.</LI>
                            Centrifuge for 30–60 s and discard the flow-through, or apply vacuum to the manifold to draw the solution through the QIAprep spin column and switch off the vacuum source. Transfer the QIAprep spin column to the collection tube.
                            <LI>Centrifuge for 1 min to remove residual wash buffer. Note:  Important - Do not skip this step.</LI><LI>Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of the QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.</LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> PCR with Q5 High Fidelity Polymerase
                            </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction:</B>Q5® High-Fidelity DNA Polymerase provides one of the highest fidelity amplification available (~280 times higher than Taq), with ultra-low error rates. Q5 DNA Polymerase is composed of a novel polymerase that is fused to the processivity-enhancing Sso7d DNA binding domain, improving speed, fidelity and reliability of performance.</P><P>Notes:
                                    </P><P>Note that protocols with Q5 High-Fidelity DNA Polymerase may differ from protocols with other polymerases.</P><P>Optimal annealing temperatures for Q5 High- Fidelity DNA Polymerase tend to be higher than for other PCR polymerases. The NEB Tm Calculator should be used to determine the annealing temperature when using this enzyme. Typically, use a 10–30 second annealing step at 3°C above the Tm of the lower Tm primer. A temperature gradient can also be used to optimize the annealing temperature for each primer pair.</P><P>Extension: The recommended extension temperature is 72°C. Extension times are generally 20–30 seconds per kb for complex, genomic samples, but can be reduced to 10 seconds per kb for simple templates (plasmid, E. coli, etc.) or complex templates &lt; 1 kb. Extension time can be increased to 40 seconds per kb for cDNA or long, complex templates, if necessary.</P><P>A nal extension of 2 minutes at 72°C is recommended.</P><P>Q5 High-Fidelity DNA Polymerase concentration:  We generally recommend using Q5 High-Fidelity DNA Polymerase at a nal concentration of 20 units/ml (1.0 unit/50 μl reaction). However, the optimal concentration of Q5 High-Fidelity DNA Polymerase may vary from 10–40 units/ml (0.5–2 units/50 μl reaction) depending on amplicon length and dif culty. Do not exceed 2 units/50 μl reaction.</P>Mg++ concentration of 2.0 mM is optimal for most PCR products generated with Q5 High- Fidelity DNA Polymerase. When used at a final concentration of 1X, the Q5 Reaction Buffer provides the optimal Mg++ concentration.<P><B>Materials</B><UL style="list-style-type:circle"><LI>2x Q5 Mastermix</LI><LI>Template</LI><LI>Forward primer</LI><LI>Reverse primer</LI><LI>Sterile water</LI><LI>Ice </LI><LI>Thermocycler </LI><LI>0.2 mL thin-walled PCR tubes</LI><LI>PCR tube racks</LI></UL></P><P><B>Procedure</B></P><P><I>Set up reactions on ice</I><OL type="1"><LI>All reaction components should be kept and mixed (gentle pipeting) on ice.  Use PCR tube racks in your ice bucket to keep your tubes organized but cold.</LI><LI> Label each tube.  Be sure your group number is also on the tube.</LI><LI>When the reactions are ready, quickly transfer reactions to a thermocycler preheated to the denaturation temperature (98°C). </LI><TABLE><TBODY><TR><TH>Component</TH><TH>50µL Reaction</TH><TH>Final Concentration </TH></TR><TR><TD>Q5 High Fidelity 2x Master Mix</TD><TD>25µL </TD><TD>1x</TD></TR><TR><TD>10µM Forward Primer</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>10µM Reverse Primer</TD><TD>2.5µL</TD><TD>0.5µM</TD></TR><TR><TD>Template DNA</TD><TD>Variable</TD><TD> At most 1000ng (depends on DNA type)</TD></TR><TR><TD>Nuclease Free Water</TD><TD>Add up to make 50µL Reaction</TD></TR></TBODY></TABLE><LI>Thermocycling conditions for routine PCR:</LI><TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time </TH></TR><TR><TD>Initial Denaturation</TD><TD>98 Degrees C</TD><TD>30 Seconds</TD></TR><TR><TD>25-35 Cycles</TD><TD>98 Degrees C</TD><TD>5-10 Seconds</TD></TR><TR><TD>50-72 Degrees C</TD><TD>10-30 Seconds</TD></TR><TR><TD>72 Degrees C</TD><TD> 2 Minutes</TD></TR><TR><TD>Hold</TD><TD>4-10 Degrees C</TD></TR></TBODY></TABLE></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> Plasmid Mini-Prep (Alkaline-Lysis with Silica Purification)
                                </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction:</B>A bacterial culture containing the plasmid of interest is centrifuged in order to concentrate cellular material (including DNA) into a pellet at the bottom of the containing vessel. The supernatant is discarded, and the pellet is then re-suspended in an EDTA-containing physiological buffer. The purpose of the EDTA is to chelate divalent metal cations such as Mg2+ and Ca2+, which are required for the function of DNA degrading enzymes (DNAses) and also serve to de-stabilise the DNA phosphate backbone and cell wall. Glucose in the buffer will maintain the osmotic pressure of the cell in order to prevent the cell from bursting. Tris in the buffer will retain the pH of the cell with 8.0 and RNase will remove the RNA which will disrupt the experiment.</P><P>Next, a strong alkaline solution consisting of the detergent sodium dodecyl sulfate (SDS) and a strong base such as sodium hydroxide (NaOH) is added. The detergent disrupts cell membranes and allows the alkali to contact and denature both chromosomal and plasmid DNA.  </P><P>Finally, potassium acetate is added. This acidifies the solution and allows the renaturing of plasmid DNA, but not chromosomal DNA, which is precipitated out of solution. Another function of the potassium is to cause the precipitation of sodium dodecyl sulfate and thus removal of the detergent. A final centrifugation is carried out, and this time the pellet contains only debris and can be discarded. The plasmid-containing supernatant is carefully removed and can be further purified or used for analysis, such as gel electrophoresis.</P><P><B>Materials</B><UL style="list-style-type:circle"><B>Stocks:</B><LI>1M 1 M Tris-HCl, pH8.0</LI><LI>0.5 M EDTA, pH 8.0</LI><LI>1 N NaOH </LI><LI>10% SDS</LI><LI>100% ethanol (200 proof)</LI><LI>PBS (phosphate buffered saline)</LI></UL></P><B>Solution I (100 ml)  --- store at 4oC or -20oC</B><LI>0.9 g of D-glucose</LI><LI>2.5ml of 1 M Tris-HCl, pH8.0</LI><LI>2.0 ml of 0.5 M EDTA, pH 8.0</LI><LI>DNAse free RNAse added to 100 μg/ml</LI><LI>Filter Sterilize and store at 4C</LI><B>Solution II (100 ml)  --- store at room temperature; make fresh weekly</B><LI>20ml (or 0.8 g) of 1 N NaOH</LI><LI>10ml of 10% SDS</LI><LI>70 ml of ddH2O</LI><B>Solution III (100 ml), 3M NaOAc, pH 4.8 --- store at room temperature</B><LI>Dissolve 24.6 g of NaOAc in 80 ml of ddH2O, adjust to pH  4.8 with glacial acetic acid.  Then bring the solution  to a total volume of 100 ml. </LI><B>6M guanidinium HCL - TE, pH 6.4  (100 ml) --- store at room temperature</B><LI>57.32 g guanidine-HCl (Sigma G-4505)</LI><LI>5 ml  of 1 MTris-HCl, pH 7.6</LI><LI>4 ml  of 0.5 M EDTA, pH 8.0</LI><LI>ddH2O to 100 ml (note:  this is not the same as 100 mls of ddH2O</LI><B>Silica (100 mg/ml in 6M guanidinium HCL - TE, pH 6.4) --- store at room temperature</B><LI>Suspend 10 g of Silica (Sigma S-5631) in 100 ml of PBS in a 100 ml graduated cylinder, and let it settle for 3-5 hrs. (There should be about 10ml of Silica) aspirate off the milky supernatant and resuspend the pellet in 100 ml of PBS let stand overnight remove the supernatant again and resuspend in 100ml of 6 M guanidine-HCl, pH 6.4 containing 50 mM Tris-HCl, 20 mM EDTA.</LI><B>Plasmid wash buffer (100 ml) (10 mM Tris-HCl, pH 8.0, 1 mM EDTA, pH 8.0, 50% ethanol in dH2O.) </B><LI>1 ml of 1 M Tris-HCl, pH 8.0</LI><LI>0.2 ml of 0.5 M EDTA, pH 8.0</LI><LI>50 ml of 100% EtOH</LI><LI>48.8 ml of ddH2O</LI><B>EB (100 ml) (10 mM Tris-HCl, pH 8.0-8.5)</B><LI>1 ml of 1 M Tris-HCl, pH 8.0 or 8.5)</LI><LI>99 ml of ddH2O</LI><P><B>Procedure</B></P><P><I>Preparing Plasmid</I><OL type="1"><LI>Centrifuge 1.5 mls cells in a microfuge tube at low speed for 2-3 minutes</LI><LI>Resuspend the pellet in 80 μl of Solution I by vigorous pipeting.</LI><LI>Add 80 ml of fresh (1 week shelf life) Solution II and mix by gentle inverstion of the tube.  Do not vortex. </LI><LI>Incubate the tube at room temperature for 3-5 minutes.</LI>
                                    The cell should lyse at this point as indicated by a clearing of the tube contents
                                    <LI>Add 80 μl of Solution III and mix by gently inversion.</LI><LI>Incubate on ice for 10 minutes.</LI><LI>Microfuge the tubes for 10 minutes at 12,000 rpm.</LI><LI>Transfer the supernatant to a fresh (labeled) tube.  Discard the old tube.</LI><UL style="list-style-type:circle"><LI>Avoid carrying over any of the pellet.</LI><LI>An alternative way to do this is to use a sterile toothpick to dislodge the pellet and remove it from the tube.</LI></UL></OL></P><P><I>Silica Purification</I><OL type="1"><LI>Add 80 μl of resuspended silica solution to the tube and invert the tube for 2-3 minutes.</LI>
                                    The plasmid will bind to the silica particles in the solution.
                                    <LI>Centrifuge the tubes for 1 min at 6K  in the microfuge or until silica has (just) pelleted. The centrifugation step may need to be adjusted for any given microfuge.  The goal is to pellet the silica such that it remains easy to resuspend during the subsequent wash step</LI><LI>Aspirate off (using a micropipette or an aspirator) the supernatant being careful not to dislodge the silica pellet.</LI><LI> Wash the pellet by adding 1 ml of DNA wash solution to each tube.  Mix well by vortexing or pipetting.</LI><LI>Repeat the centrifugation step (1 min at 6K). </LI><LI>Remove (micropipette or aspirate) the wash buffer and discard it.</LI><LI>Repeat the wash step (1 ml of DNA plasmid wash solution with mixing or pipetting)</LI><LI>Remove (micropipette or aspirate) the wash buffer and discard it. </LI><LI>Allow pellets to dry for 5-10 minutes by incubating the tubes with caps open at 65oC.</LI></OL></P><P><I>DNA Elution </I><OL type="1"><LI>Add 80 μl of sterile EB elution buffer (10 mM Tris pH8.5)  to each tube, vortexing or pipetting to mix.</LI><LI> Incubate the tubes (closed caps) for 5 min at 65oC, with occasional mixing by inversion of the tubes. </LI>
                                    The DNA will be eluted from the silica particles during this step and will now be in the supernatant
                                        <LI>Centrifuge the tubes for 30 seconds on high and remove the supernatant (containing the DNA) to a fresh, labeled tube. Discard the tubes with the silica pellets. KEEP the SUPERNATANT!</LI><LI>Store your DNA at -20oC until you are ready to determine its concentration (spectrophotometry) and evaluate its quality (agarose gel).</LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> iGEM Style Transformation
                                    </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction: </B> Transformations are essential to using BioBricks which are provided as dried down DNA in a well of a microtiter dish. After the desired DNA part has been reydrated, use it to transform the DNA into competent cells, and select single colonies.
                                        </P><P>Transformations can be one of the more fickle laboratory techniques. At iGEM HQ the following protocol works well for transforming a few samples at a time.</P><P> The original protocol can be found <A href="http://parts.igem.org/Help:Protocols/Transformation">here!</A></P><B>Materials</B><UL style="list-style-type:circle"><LI>Resuspended DNA to be transformed</LI><LI>Positive Transformation Control DNA</LI>
                                        10pg/µl Positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid.)
                                        <LI>Competent <I>E. coli</I> cells (typically the strain DH5alpha)</LI><LI>1.5mL Microtubes</LI><LI>LB or SOC Media for Outgrowth</LI><LI>Petri Plates with LB Agar and the appropriate Antibiotic</LI><LI>42 Degree Temperature Block</LI><LI>37 Degree Incubator</LI><LI>Sterile Glass Spreading Beads</LI><LI>Microcentrifuge</LI><LI>Micropipetters and Tips</LI><LI>Ice Bucket and Ice</LI><LI>Timer</LI></UL><P><B>Procedure</B></P><P><I>Transformation</I><OL type="1"><LI>Label 1.5ml tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5ml tubes (one tube for each transformation, including your control)</LI><LI>Thaw competent cells on ice: This may take 10-15min for a 260µl stock. </LI>
                                        Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.
                                        <LI>Pipette 25-50µl of competent cells into 1.5ml tube (one per transformation). Don’t forget a 1.5ml tube for your positive and negative controls. </LI><LI>Close 1.5ml tubes, incubate on ice for 30min: Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.</LI><LI>Heat shock tubes at 42°C for 45 sec: Place tubes into a temperature block with water in the wells and is already warmed to 42 Degrees C. Timing is critical.</LI><LI>Incubate on ice for 5min: Return transformation tubes to ice bucket.</LI><LI>Pipette 450µl SOC media to each transformation: SOC should be stored at 4°C, but can be warmed to room temperature before use. Check for contamination.</LI><LI>Incubate at 37°C for 1 hour, shaking at 200-300rpm. </LI><LI>Pipette 100µL of each transformation onto petri plates Spread with sterilized spreader or glass beads immediately. </LI><LI>Spin down cells at 6800g for 3mins and discard all but around 100µL of the supernatant. Resuspend the cells in the remaining 100µL, and pipette each transformation onto petri plates Spread with sterilized spreader or glass beads immediately. This increases the chance of getting colonies from lower concentration DNA samples.</LI><LI>Incubate transformations overnight (14-18hr) at 37°C: Incubate the plates upside down (agar side up). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.</LI></OL></P><P><I>Interpreting the Results</I><OL type="1"><LI>Count colonies for control transformation: Count colonies on the 100μl control plate and calculate your competent cell efficiency. Competent cells should have an efficiency of 1.5x10^8 to 6x10^8 cfu/µg DNA.
                                                </LI><LI>Pick single colonies: Use single colonies from the BioBrick transformations to (i) streak for singles, (ii) inoculate a liquid culture to amplify the plasmid and (iii) do a colony PCR to confirm the construction of the part. </LI></OL></P></DIV></DIV><DIV class="collapsible_accordion"><H1> iGEM Style Restriction Digest
                                        </H1><DIV class="collapsible_accordion_content hide_content"><DIV class="deadline_content"><P><B>Introduction: </B> The iGEM protocol for restriction digests is highly standardized. It uses a single buffer for all digests and a PCR machine to handle all temperature changes.</P><P> The original protocol can be found <A href="http://parts.igem.org/Help:Protocols/Restriction_Digest">here!</A></P><B>Materials</B><UL style="list-style-type:circle"><LI>(1) 8-tube strip, or (3) 0.6ml thin-walled tubes</LI><LI>Ice and Ice Bucket</LI><LI>BioBrick Part in BioBrick plasmid (Purified DNA, &gt;16ng/uL) or other DNA</LI><LI>Sterile H2O</LI><LI>NEB Buffer 2</LI><LI>Cutsmart for all High-Fidelity enzymes</LI><LI>BSA (for use with older buffers)</LI><LI>Restriction Enzymes: EcoRI, SpeI, XbaI, PstI</LI><LI>Thermocycler</LI></UL><P><B>Procedure</B></P><P><I>Set Up the Reaction (Note: If setting up several digests with a single DNA sample, make a single reaction mix on ice and aliqot it to individual tubes -- add enzymes last)</I><OL type="1"><LI>Add 250-500ng of DNA to be digested, and adjust with dH20 for a total volume of 16.5-17ul.</LI><LI>Add 2.0ul of NEBuffer 2 or Cutsmart buffer. </LI><LI>Add 0.5ul of BSA for older NEB buffers. </LI><LI>Add 0.5ul of EcoRI.</LI><LI>Add 0.5ul of PstI.</LI><LI>There should be a total volume of 20ul. Mix well and spin down briefly.</LI></OL></P><P><I>Carry Out the Digestion</I><OL type="1"><LI>Incubate the restriction digest at 37C for 30min. This can be done using temperature blocks or thermocycler.
                                                </LI></OL></P><P><I>Inactivate the Enzymes</I><OL type="1"><LI>80C for 20min to heat kill the enzymes.</LI></OL></P><P><I>Check the Products of the Reaction by Agarose Gel Electrophoresis</I><OL type="1"><LI>Run a portion of the digest on a gel (8ul, 100ng), to check that both plasmid backbone and part length are accurate.</LI></OL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>