<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNOTT_Design1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNOTT/Design1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-default navbar-static-top" role="navigation" style="padding-top: 0.5%"><DIV class="container" style="padding-top: 0.5%"><DIV class="navbar-header" style="padding-top: 10px"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="padding-top: 0.5%"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:UNOTT">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:UNOTT/Description" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Medal_Criteria">Medal Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">LAB</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Notebook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">MODELLING</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Modelling">Models </A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">OUTREACH</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Engagement">Public Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV id="icons" style="background-color:#02263e;"><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Brainstorms"><FIGCAPTION style="color: #ffffff;">Brainstorm</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design1"><FIGCAPTION style="color: #ffffff;">Design Process</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design2"><FIGCAPTION style="color: #ffffff;">Key. coli Plasmid Design</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design3"><FIGCAPTION style="color: #ffffff;">Key Transport Design</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design4"><FIGCAPTION style="color: #ffffff;">Future Improvements</FIGCAPTION></A></FIGURE></DIV><DIV class="contentmargin"><H1>Design Process</H1><H2> The Prerequisite</H2><P>The idea of how to make bacteria which could be used as a security system evolved over several weeks. The key requirement of the system is the ability to generate <B>many random</B> combinations of expression outputs to ensure security. 
</P><H2>Transposons</H2><P>One of the first ideas regarding how this could be achieved was with the use of transposons. 
Transposable elements are DNA fragments that can change their position within a genome. This creates mutations resulting in different levels/suppression of expression of certain genes. The use of transposons would yield bacteria with various phenotypes, which could be used in the <I>Key. coli</I> security system. A target site-specific Tn7 transposon could be used for this purpose. This bacterial mobile DNA segment inserts at high-frequency into a single specific site, called attTn7 in <I>E. coli</I><SUP>1</SUP>. Credit: Phillip Dumesic, UCSF (Adapted from Transposon by Lauren Solomon, Broad Institute
 </P><P>The Tn7 system has been commonly used to generate random mutation libraries within a broad range of organisms whereby the machinery of the transposon system is expressed by the host organism. Unfortunately, this is not a good option in our case; expression of the transposition machinery in the host organism will create an inevitable bias, with transposition events most likely resulting in configurations that are associated with a smaller metabolic cost. 

A solution to this problem would be to do the transposition in vitro. However, further research revealed several disadvantages with this idea. The mechanisms of Tn7 recombination are complicated, requiring many proteins for successful transposition. Although we thought this would be interesting to test, we came across difficulties with sourcing the TsnD subunit of the Tn7 transposon. As expression and purification of this subunit was not realistic within the timeframe we had, we decided to pursue a different method.</P><H2>CRISPRi (dcas9 mediated gene repression)</H2><P>The next idea on how to obtain different phenotypes of bacteria was the use of RNA interference. In this process RNA molecules inhibit gene expression on the transcriptional level. 

An extensive literature search revealed that the CRISPR interference system seems to be a reliable and predictable RNA interference mechanism. CRISPRi influences gene expression primarily at the transcriptional level and allows sequence-specific control of gene expression. This method utilises the CRISPR pathway and a catalytically inactive Cas9 (dCas9) protein. In the traditional type II CRISPR system, Cas9 introduces double-stranded breaks in specific genomic sequences, guided by a short guide RNA (sgRNA). dCas9 is engineered to lack nuclease activity enabling repurposing of the system for genomic DNA targeting without cleavage and therefore allowing precise transcription regulation<SUP>2</SUP>.
</P><P>As a result, we came up with an idea that a variety of reporters could be under the control of promoters which can be targeted by sgRNAs and dCas9. The sgRNA-dCas9 complex can be targeted to bind to the sequence of the promoter. By doing so, it physically interferes with translation, inhibiting the transcription machinery from recognising and binding to the promoter.
<SUP>1</SUP>Peters, J. E. (2014). Tn7. Microbiology Spectrum, 2(5). doi:10.1128/microbiolspec.mdna3-0010-2014<SUP>2</SUP>Dominguez, A. A., Lim, W. A., &amp; Qi, L. S. (2016). Beyond editing: repurposing CRISPR–Cas9 for precision genome regulation and interrogation. Nature Reviews. Molecular Cell Biology, 17(1), 5–15. http://doi.org/10.1038/nrm.2015.2

</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>