Team:UNOTT/Notebook
Toggle navigation
HOME
PROJECT
Description
Design
Demonstrate
Medal Criteria
LAB
Lab book
Experiments
Parts
InterLab
MODELLING
Overview
Models
Software
OUTREACH
HP Silver
Public Engagement
TEAM
Our Team
Collaborations
Sponsors
Attributions
LAB BOOK:
MARCH - JULY
Planning! We spent a long time umming and ahhing about potential projects. We only decided to put all our efforts into one just a week before we were due to start in the lab at the beginning of July!
WEEK 1
Discussing options for CRISPRi and transposons to give assortment of levels for products. We spent a lot of time in meetings and planning to make sure that we were sure about our project. We went bowling as a team this week and discovered that Natalia is an incredibly precise bowler... hopefully her precision transfers to her pipetting skills - all will be revealed later.
WEEK 2
This week we really got our heads down and the biochemists planned our constructs and designed primers. In the meantime the rest of the team worked on outreach (i.e. sending hundreds of e-mails!). Vikram, the computer scientist, worked on lots of complex computer stuff and worked endlessly on our game, a Portal mod. Jake made us a new logo and designed a front page for our wiki which looks super professional now.
WEEK 3
This week we started lab work! 5 of us (biotechnologists and biochemists) were in the lab learning the techniques needed for iGEM.
Primers and enzymes were ordered for one of our two constructs today so we started on the InterLab study until they arrive! We transformed the controls and devices into DH5a competent cells that were provided for us. We set up overnight cultures of TOP10 cells for making electrocompetent cells, and we also made O/N cultures of E.coli carrying the backbone we need for our pReporter which was donated by a lab member. This is called pSTLS.  Vikram released a teaser trailer for the game we have made. Vik's computer then died.
Vik's computer is alive again. Unfortunately our transformations only gave very small colonies on some plates, and none on others. We re-streaked those colonies that we did have. We extracted plasmid DNA from the O/N cultures of pSTLS and others for the lab to get some practice. We learnt to use the NanoDrop to quantify DNA. Outside of the lab, Georgette was busy creating our very first vlog which will be shared on the social media pages. Vik is looking at building hardware. Today we also skyped Edinburgh UG team to discuss possible collaborations. We waited for primers to arrive but NEB sent us some enzymes and competent cells so thanks to NEB! We also learnt to electroporate DH5a and TOP10 cells. We transformed them with either a plasmid containing dCas9 or our low copy backbone, pSTLS. We also set up overnight cultures for the InterLab Study. Next came DISASTER! Two of our overnight colonies didn't grow so we had to restart the overnights on Monday. Ah well, we are learning what science is like in reality. We went for a social today - laser quest! Georgette won by far, Ellie lost miserably (-8300 points). Georgette was appointed new team leader due to Ellie's shocking performance... just kidding!
WEEK 4
This week we carried on with the InterLab study and waited for our primers to be delivered. We set up new overnight cultures for the InterLab study today and all our cultures grew so we proceeded with InterLab GFP measurements... they weren't what we were expecting! Jake and Vik skyped Bristol iGEM to see whether we can help them improve the wiki design process. We analysed the data from the InterLab study today and e-mailed some important people for outreach purposes. Jake changed the nav bar and Chris made one of our many games.
We also took team photos, ate fish and chips and OUR PRIMERS ARRIVED! This means we can properly start in the lab on Monday!!
WEEK 5
The primers arrived so we had a long day in the lab doing 46 PCR reactions! We learnt how to set up PCR and load agarose gels, and how interesting waiting around for PCR is. We got lots of interesting results.  Our original PCRs were using 1ng template and touchdown from 70-60oC for 10 cycles followed by 25 at 65oC. These conditions were great for getting most of the components to amplify, however we had to optimise a few. For example the dCas9 and low copy backbone didn't amplify first time around, so we used a new strategy - using various amounts of template with duplicates placed at different annealing temperatures. One was placed in a touchdown setting as before but with annealing temperatures dropping from 60 to 50oC followed by 25 cycles at 65oC. The other was placed at 55oC for all cycles. We also didn't get good amplification of P1, P2, P3, P4, T1 or T2 so we repeated their PCR using touchdown from 70 to 60 as before but with 5 more cycles.
In the optimised PCR, we managed to get P1, P2, P3, P4 amplified, and T1 and T2 amplified but there were far too many bands to be certain we were amplifying the right thing. So we re-amplified T1 and T2 using PCR with touchdown 60-50oC as above. T1 was successfully amplified using this approach, however we got really strange bands for T2 so we went back to the samples from the previous PCR to see whether we could use those instead for T2. Our optimised PCR showed dCas9 was successfully amplified in all conditions - yay! The low copy backbone only amplified at 55oC using 1ng template so we repeated this PCR using those settings to generate enough DNA for later. This PCR was successful.
Now that we had most of the components, we could digest them. P1-5 + PE, the strong and weak RBS linked GFPs, and T2 were digested with BsaI. For the gRNA plasmid, J23119, gRNA 1-5 and gRNA0 as well as Tfdx were digested with BsaI. We digested 1ug for all of the components except the GFPs. 2ug of both backbones were digested too. The gRNA backbone was digested with SalI and AscI, whereas the low copy backbone was digested with SalI and BsteII. For the dCas9 brick, we digested dCas9 with Bsa1, T1 with BsaI and SalI, and PdCas9 with BsteII and BsaI. We PCR purified all of the digested before ligating together the bricks using T4 DNA ligase. We had problems digesting gRNA 4 as the DNA concentration dropped to 2ng/ul on two separate occasions, so the whole process was repeated for gRNA 4. 50ng of each brick was used for the brick ligations whereas 100ng of the backbones were used. 13.3ng of T1 was used and 7.2ng of PdCas9. Ligations were carried out for 2.5 hours.
Today we checked whether the ligations had worked. We tried to amplify using our brick forward and reverse primers for all bricks, however the results showed amplification of products much smaller than expected. We suspected a few problems so took a while to look through the primers using SnapGene and identified a few tweaks that could be made to the gRNA brick primers, so we ordered new primers for that. We were very confused by the promoter-GFP-terminator bricks as the products should be over 1kb whereas they were showing up at under 500bp. We decided to troubleshoot the PCR by using the brick forward primer and the sGFP reverse primer on P1, P2, P3, PE on the corresponding ligations. Today we also decided to ligate the sgRNAs with their promoter, terminator and the backbone in a 4 fragment ligation using T4 ligase overnight at room temperature.
Another late finish as we checked whether the PCRs worked with the brick forward primers and the sGFP reverse primer for P2, P3, P4 and PE "bricks" using our ligations as templates. They did! So that gave us hope that the promoter and sGFP are attached properly. We realised there was a problem with T2 amplification so we decided to switch to using T3 instead. So we created primer dimers of T3 and amplified it before PCR purification, ready for digestion on Monday. We noticed another problem with P1 which meant that it would not amplify using the primers we currently had, we made larger quantities of PCR of P2-5 + PE bricks using the brick forward primer and either strong and weakGFP reverse primers. Unfortunately the gel showed that the wGFP bricks did not amplify but the sGFP bricks did so we are a bit confused. We cut out the bands which had worked ready for gel extraction on Monday. Today we also dialysed our ligations which had been running overnight, and transformed these into TOP10 by electroporation and plated them on chloramphenicol plates to select for transformants. We also checked the transformation plates from yesterday which hopefully had the low copy backbone plus dCas9, PdCas9 and T1 in - at the end of Friday there were 2 colonies from being left in 30 degrees for around 24 hours - good news!
WEEK 6
WEEK 7
WEEK 8
WEEK 9
WEEK 10
Sequencing showed that our dCas9 vectors consistently had a frameshift only 40bp into the dCas9 gene so we looked for a new source of dCas9 template. We would like to say a huge thank-you to Dr. Bolt's lab from the University of Nottingham who gave us a small amount of PdCas9 from addgene to use as a template. The below gel shows our attempts to mutate dCas9 using this template.
WEEK 11
In order to test the random ligation idea, we constructed various controls where we used P4 as "on" or "+", or PE as "off" or "-". We constructed control plasmids where only GFP was on, RFP was on, or CFP was on. We also had a control where all three fluorescent proteins were on. Unfortuantely a control where we had all three fluorescent proteins off generated no colonies. We also used a random ligation where both the bricks for P4 and P0 bricks for each fluorescent protein were added to a mixture where the fluorescent protein bricks had both been digested with sets of enzymes that mean we can control where each fluorescent protein appears in the plasmid. In this way, the fluorescent protein can be either on or off at each position, randomly. We wanted to check that each colony had the plasmid containing a 3kb insert of all 3 fluorescent protien bricks, so we carried out colony PCR.
Sequencing results showed that the previous gRNA 3 and 5 colony plasmids were mutated so we decided to re-construct the gRNA 3 plasmid in the same way as before. We then used gRNAv_F_seq and ColE1_F1 to probe for positive colonies in colony PCR, as shown below.
WEEK 12
The random ligation controls generated some unexpected results - the colours didn't line up with what we were expected so we did some sequencing with M13F and p15a_F1 to troubleshoot the issue. The result of this is that we suspect some contamination somewhere along the line, so we went back to construction. Unfortunately the primers selected were not able to sequence across the whole 3 bricks so we don't know what happened to these random ligations and thought it would be safest to backtrack.
We set up 5ml overnights of the new gRNA 3 & 5 vectors. We extracted the plasmid from these cultures and sent them off for sequencing using gRNAv_F_seq and ColE1_F1. We also set up 5ml of the kit plate CFP, RFP and sfGFP transformants for re-construction of the bricks. We extracted plasmid from these the next day too.
We set up overnights of pMTL83211 from the SBRC culture collection and purified this in the morning before digesting 4μg with NotI and XbaI. We also amplified Pfdx donated by the lab with Pfdx-NdeI-BsaI_R and XbaI-Pfdx_F. dCas9 was amplified from the vector gifted by Ed Bolt's lab using cas9-R-NotI_OH and BsaI-cas9_F using KOD. dCas9 (4.1kb) was run on a gel and purified from there, and Pfdx was PCR purified.
Pfdx was then digested with XbaI and BsaI, while dCas9 was digested with BsaI and NotI at 37oC for 2 hours. The components of this new dCas9 strategy vector were ligated in a 1:5:3 (vector:Pfdx:dCas9) ratio using 100ng of vector at 4oC overnight.
The new dCas9 strategy vector ligations were dialysed then transformed into TOP10 before recovery and plating on LB + Em plates and growth at 30oC for 2 days. This generated only 1 colony on the negative control plates each and around 100 on each of the test ligations.
WEEK 13
gRNA tests work
8 colonies were picked from the new dCas9 strategy vector transformation and screened using cas9_scr_R1 and pCB102_R1. If present, the plasmid would give a 1kb amplicon. PCR settings: 55oC annealing temperature and 1 minute extension time, 30 cycles.
All colonies gave a amplicon of 1kb, which shows that all 8 colonies were positive. Colonies 1 and 3 were used to innoculate overnight cultures. Plasmids were extracted from overnight cultures and sequenced. Sequencing showed they were correct.
Ellie digested all of the RFP bricks that Chris had re-made, as well as digesting the new dCas9 strategy vector with AscI and SalI overnight at 37oC. RFP brick digests were purified using a PCR cleanup kit, whereas the vector digest was run on a gel and the band at 7656bp was separated from the 261bp insert by gel extraction.
The digested RFP bricks were ligated into 50ng of vector in a 1:3 (vector:insert) ratio using 20ng of insert. Ligations were done using T4 DNA ligase (Promega) at room temperature for 3 hours before dialysis using Millipore filters, and transformation into electrocompetent TOP10. The electroporated cells were recovered at 30oC for 2 hours in SOC before plating on LB + Cm + Em plates and grown at 30oC over a weekend.
Random ligation work
WEEK 14
The transformations of each RFP brick inside the new dCas9 vector resulted in 100s of colonies per plate, with only 10 on the negative control (backbone only ligation). We picked and screened 5 colonies from each transformations, as well as using the empty plasmid from before digestion as a control in the first lane. Primers used were: cas9_scr_R1 and pCB102_R1 which should give a product of around 1kb if empty, and 1.6kb if the RFP brick had been successfully inserted.
The positive colonies corresponding to the lanes labelled with a red arrow were used to set up overnight cultures in 10ml of LB (+ chloramphenicol and erythromycin). Plasmids were extracted from these cultures and sequenced using cas9_scr_R1 and pCB102_R1. These plasmids will be called the reporter plasmids henceforth when referring to the CRISPRi tests. Overnights were also set up of all gRNA vectors (1, 2, 4, and 0 from -80oC storage, and 3 & 5 from the plates of the new ligations done recently. Chris made competent cells ready for transformations in the next few weeks.
WEEK 15
100ng of one gRNA plasmid (targeting the promoter e.g. gRNA 4 targeting P4, or non-targeting gRNA 0) was co-transformed with 100ng of the corresponding reporter plasmid into TOP10 electocompetent cells by electroporation, recovered in SOC at 30oC for 2 hours, then plated onto LB with erythromycin (500µg/ml) and chloramphenicol (25µg/ml) and grown overnight at 30oC. 5 colonies were picked per transformation and resuspended in 10µl of water. 1µl of this resuspension was used for a colony PCR to confirm the presence of gRNA plasmid, and another 1µl used for confirmation of the reporter plasmid. The primers used and expected sizes are shown in the table below. Vik, the computer scientist, was convinced to learn how to load gels finally too! Learning new skills all round.
Plasmid to detect
Primer 1
Primer 2
Product for sRFP plasmids (bp)
Product for PE sRFP plasmid (bp)
Product for wRFP plasmids (bp)
Product for PE wRFP plasmid (bp)
gRNA vector
gRNAv_F_seq
ColE1_F1
495
New design reporter plasmid
cas9_scr_R1
pCB102_R1
 1647
1612
1649
1614
These gels show that the vast majority of colonies were positive for both the gRNA plasmid and the reporter plasmid. We chose the colonies that represent the first 3 positive lanes containing each of the strongRBS-RFP and innoculated an overnight culture of 10ml of LB with erythromycin(500ug/ml) and chloramphenicol (25ug/ml) to maintain the plasmids.
In the morning, the OD of these cultures were measured by photospectrometer and used to innoculate 15ml of fresh LB with Cm and Em to a starting OD of 0.02. OD and RFP fluorescence were measured from 100µl of each culture at 0, 2, 4, 6 and 24 hours after innoculation using a Clariostar (?) with the following settings (?). 3 biological replicates (3 colonies from each transformation) were measured, using duplicates for each to allow for technical errors. At 6h, some of the results went out of the range capable by the microplate reader (>260,000) so at the 24 hour timepoint the readings were done at a 1 in 5 dilution (20µl of culture mixed with 80ul LB in the 96 well plate). As the results at 6h had some replicates just below 260,000 we predict that these results that exceeded 260,000 were likely close to this value anyway. Due to this, we have included them in our results, however this must be considered during interpretation of the results.
As the overnight cultures were not visually red, 200µl of a representative colony was spun down and supernatant removed to see whether a visual result can be seen at a higher cell density. The following picture shows these pellets.
The image below shows colony resuspensions dotted onto LB+Cm+Em plates. Each quadrant contains the 5 colonies selected per transformation (a-e refers to the order of lanes in the colony PCR gel). Labelling of quadrants is as follows: the first number is the promoter attached to RFP, s/w corresponds to the strength of the RBS, and the second number is the gRNA plasmid that is present. An example is 1s1 where P1 is attached to the strong RBS RFP in the reporter plasmid, and has been co-transformed with gRNA 1 plasmid. ? on colonies were those whose results were negative or less clear than others on the gel. Circled colonies were those selected to be used for storage of the plasmid at -80oC.
A visual of the gRNA results is shown by the colonies below.
WEEK 16
This week we started to prepare our DNA for submission - we digested all our sRFP-containing plasmids with EcoR1 and PstI, as well as our gRNA plasmids and the pSB1C3 backbone at 37oC overnight. We ran these on the gel shown below and extracted the bands indicated for gel purification.
We also transformed pSB1C3 from kit plate 4 well 4B and set up overnight cultures from colonies for plasmid extraction. We then digested 4ug of this plasmid with EcoRI and PstI to allow us to ligate in the parts.
After ligation, we dialysed them and transformed them into TOP10 and plated for growth at 30oC over the weekend.
WEEK 17
This week we did colony PCR of the DNA submission transformants, as seen in the gel below.
We then set up overnight cultures of the colonies represented by the lanes indicated with red arrows. Plasmids were extracted the next day before sequencing. The remaining plasmid was diluted to a concentration of 25ng/ul so that we could dry them in the submission plates ready for shipping to Boston.
After the results of the sequencing came back, we put 10ul of each plasmid to be submitted into a 96 well plate as per instructions, and sent them off to iGEM! We had a wonderful summer and we feel lost without our DNA now!
In the spirit of iGEM, we also prepared everything for storage in the SBRC's central culture collection for future iGEM teams and the centre to use as they wish.
