<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNOTT_Design2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNOTT/Design2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-default navbar-static-top" role="navigation" style="padding-top: 0.5%"><DIV class="container" style="padding-top: 0.5%"><DIV class="navbar-header" style="padding-top: 10px"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="padding-top: 0.5%"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:UNOTT">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:UNOTT/Description" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Medal_Criteria">Medal Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">LAB</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Notebook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">MODELLING</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Modelling">Models </A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">OUTREACH</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Engagement">Public Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV id="icons" style="background-color:#02263e;"><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Brainstorms"><FIGCAPTION style="color: #ffffff;">Brainstorm</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design1"><FIGCAPTION style="color: #ffffff;">Design Process</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design2"><FIGCAPTION style="color: #ffffff;">Key. coli Plasmid Design</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design3"><FIGCAPTION style="color: #ffffff;">Key Transport Design</FIGCAPTION></A></FIGURE><FIGURE><A href="https://2017.igem.org/Team:UNOTT/Design4"><FIGCAPTION style="color: #ffffff;">Future Improvements</FIGCAPTION></A></FIGURE></DIV><DIV class="contentmargin"><H1>Key. coli Design</H1><H2>Choice of host</H2><P><I>Escherichia coli (E. coli)</I> was the chosen host and more specifically the TOP10 strain, a standard laboratory strain belonging to risk group 1. <I>E. coli</I> was deemed a suitable host for Key. coli given the vast number of genetic tools and information available, its ability to grow aerobically and the experience of our group regarding its use. In addition, the TOP10 strain is deficient in DNA recombination machinery. In our design, many parts are very similar on the genetic level, thus having a rec- host will prevent unwanted genetic rearrangements due to sequence homology of the different parts.
</P><H2>Choice of reporter</H2>
Fluorescent proteins were the reporters of choice; fluorescent proteins are routinely used in molecular biology, being well understood, safe to use and easily detected. Most importantly, the colour of fluorescent proteins is invisible to the naked eye and can only be ‘unlocked’ under the correct wavelength of light. 

In the present study we used 3 different fluorescent proteins to generate unique spectra:
<P style="margin-left: 50px">•  Green Fluorescent Protein (GFP) 
•  Red Fluorescent Protein (RFP)
•  Cyan Fluorescent Protein (CFP)</P><H2>Introducing randomness and variation</H2>
To extend the potential combinations of fluorescent signals that can be obtained we introduced variation in the following ways:
<P1> 1.  Varying the Ribosome Binding Site (RBS)</P1>
Two different versions of a suitable RBS were employed with the aim of introducing further variability in expression. Essentially, each fluorescent protein employed could have one of the two, increasing the reporter range from 3 to 6. To facilitate cloning, the RBS was amplified as part of the reporter.
<P style="margin-left: 50px">• Strong (s) RBS yielding higher expression
• Weak (w) RBS yielding lower expression </P><P><P1>2.  Promoter pool</P1>
Five promoters of <B>different strengths</B> were chosen from literature<SUP>1</SUP> to drive the expression of these fluorescent proteins. These were the following:
</P><P style="margin-left: 50px">• p1
• p2
• p3
• p4
• p5</P><P>
We also designed a 6<SUP>th</SUP> ‘empty’ promoter (pE) with no TATA boxes as a non-expressing (OFF) signal. 

In theory, any of the 6 promoters can be paired with any of the 6 available reporters (36 combinations).
<P1>3.  CRISPR interference</P1>
In Key. coli, dCas9 has been targeted to repress the promoters driving the expression of the fluorescent proteins. This is achieved by using sgRNAs with seed regions complementary to the sequences between the -10 and -35 regions of the given promoters. dCas9 presence inhibits access of the RNA polymerase to these regions via steric hindrance, thus inhibiting transcription. 

Five promoter-sgRNA sets were taken from literature<SUP>2</SUP> to modify the levels of protein expression: 
</P><P style="margin-left: 50px">• sgRNA1 targeting p1
• sgRNA2 targeting p2
• sgRNA3 targeting p3
• sgRNA4 targeting p4
• sgRNA5 targeting p5</P><P>
These sgRNAs have <B>different repression efficiencies</B>, introducing an additional layer of variability. 

We also designed a 6<SUP>th</SUP> ‘empty’ sgRNA (sgRNA0) with no matching sequence as a non-repressing (ON) signal.

dCas9 and the sgRNAs were driven by the constitutive promoter J23119. </P><P><P1>4. Modular design and random assembly</P1>
For the proof-of-concept experiments, we envisioned the use of a <B>two-plasmid system</B>, comprised of <B>pReporter</B> and <B>pgRNA</B>. An additional layer of variability and randomness can be introduced at the level of the plasmid ligation reaction. Ligation is driven by Brownian motion, a truly random process that cannot be predicted.

This requires that all bricks are modular as explained below: 
</P><P>
Each individual Promoter-Reporter-Terminator (P-R-T) brick contains interchangeable parts, linked together with BsaI sites. This method is also used for the construction of Promoter-sgRNA-Terminator (P-sgRNA-T) bricks. The bricks are flanked by a prefix and suffix, which are in turn flanked by restriction sites ABCD on either end. Digestion of bricks with A+B, B+C and C+D allows any brick to be placed in positions 1, 2 or 3 respectively within the plasmid. 

If all the bricks present in the ligation mixture are digested with all three sets of enzymes, any brick has equal chances of being placed in any position in the plasmid, allowing expansion of possibilities whilst maintaining randomness of insertion. 
</P><H2>Key. coli plasmid design</H2>
Ideally, all the promoter-reporter combinations and dCas9 could be integrated into the genome, yielding a stable ‘basic’ strain. Expression levels of the different reporters could then be influenced by the expression of different plasmid-based sgRNAs. 

However, for the proof-of-concept experiments, we decided to employ a dual plasmid-based system (comprised of pReporter and pgRNA) for the following reasons: 
<P style="margin-left: 50px">• given the available time-frame, a plasmid-based system would be faster to create than a genomically-integrated one
• promoter-reporter combinations would be easily interchangeable
• it would allow exemplification of the randomness of the ligation process</P><P>
The pReporter plasmid would contain: 
</P><P style="margin-left: 50px">• P-R-T bricks, randomly assembled
• P-dCas9-T
• low copy replicon (p15a), to minimise toxic effects of dCas9 and to maximise the difference upon sgRNA inhibition
• erythromycin resistance, for selection</P><P>
The pgRNA plasmid would contain:
</P><P style="margin-left: 50px">• P-sgRNA-T bricks targeting the promoters present in pReporter
• high copy replicon (ColE1), to have an excess of sgRNA expression to ensure full targeting of the promoters on the other plasmid
• chloramphenicol resistance, for selection</P><P>
Both plasmids would be transformed into <I>E. coli</I>, resulting in a randomly generated fluorescent pattern. 

The randomly selected expression level of each colony is like a random number generator; this process ensures that any bacterial colony used as a key will have unique characteristics increasing the safety of the Key. coli security system.
</P><H2>Experimental planning</H2><P1>1. Create a promoter library to assess the strength of promoters 1-5:</P1><P style="margin-left: 50px">• Creation of pReporter plasmids with 1 x P-sFP-T or 1 x P-wFP-T bricks
• Measure fluorescence at different time points</P><P><P1>2. Exemplify that random ligations are possible:</P1></P><P style="margin-left: 50px">• Random assembly of pReporter plasmids containing GFP, RFP and CFP 
• Select colonies from transformation plates
• Measure fluorescence at different time points</P><P><P1>3. Assess the level of repression of gRNAs 1-5:</P1></P><P style="margin-left: 50px">• Creation of psgRNA plasmids with 1 x P-sgRNA-T brick
• Creation of pReporter plasmids with 1 x P-sRFP-T or 1 x P-wRFP-T brick (already done in 1)
• Transformation of E. coli with the following combinations of plasmids:</P><P style="margin-left: 100px">pReporter 1 + pgRNA 1 (repressed)
pReporter 1 + pgRNA 0 (ON)
pReporter 2 + pgRNA 2 (repressed)
pReporter 2 + pgRNA 0 (ON)
pReporter 3 + pgRNA 3 (repressed)
pReporter 3 + pgRNA 0 (ON)
pReporter 4 + pgRNA 4 (repressed)
pReporter 4 + pgRNA 0 (ON)
pReporter 5 + pgRNA 5 (repressed)
pReporter 5 + pgRNA 0 (ON)
pReporter E + pgRNA 6 (OFF)</P><P style="margin-left: 50px">• Measure fluorescence at different time points</P><P><P1>4. Carry out random assemblies of pReporter and pgRNA plasmids with 3 x P-R-T and 3 x P-sgRNA-T bricks</P1></P><P style="margin-left: 50px">• Creation of psgRNA plasmids with 3 x P-sgRNA-T bricks
• Creation of pReporter plasmids with 3 x P-FP-T bricks
• Select colonies from transformation plates
• Measure fluorescence at different time points
• Assess ‘uniqueness’ of profiles generated
• Confirm ‘uniqueness’ by DNA sequencing</P><P><FONT size="-1"><SUP>1</SUP>Nielsen, A. A., &amp; Voigt, C. A. (2014). Multi-input CRISPR/Cas genetic circuits that interface host regulatory networks. Molecular Systems Biology, 10(11), 763. http://doi.org/10.15252/msb.20145735<SUP>2</SUP>Nielsen, A. A., &amp; Voigt, C. A. (2014). Multi-input CRISPR/Cas genetic circuits that interface host regulatory networks. Molecular Systems Biology, 10(11), 763. http://doi.org/10.15252/msb.20145735</FONT></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>