Team:UNOTT/Design1
Toggle navigation
HOME
PROJECT
Description
Design
Demonstrate
Medal Criteria
LAB
Lab book
Experiments
Parts
InterLab
MODELLING
Overview
Models
Software
OUTREACH
HP Silver
Public Engagement
TEAM
Our Team
Collaborations
Sponsors
Attributions
Brainstorm
Design Process
Key. coli Plasmid Design
Key Transport Design
Future Improvements
Design Process
The Prerequisite
The idea of how to make bacteria which could be used as a security system evolved over several weeks. The key requirement of the system is the ability to generate many random combinations of expression outputs to ensure security.
Transposons
One of the first ideas regarding how this could be achieved was with the use of transposons.
Transposable elements are DNA fragments that can change their position within a genome. This creates mutations resulting in different levels/suppression of expression of certain genes. The use of transposons would yield bacteria with various phenotypes, which could be used in the Key. coli security system. A target site-specific Tn7 transposon could be used for this purpose. This bacterial mobile DNA segment inserts at high-frequency into a single specific site, called attTn7 in E. coli1.
Credit: Phillip Dumesic, UCSF (Adapted from Transposon by Lauren Solomon, Broad Institute
The Tn7 system has been commonly used to generate random mutation libraries within a broad range of organisms whereby the machinery of the transposon system is expressed by the host organism. Unfortunately, this is not a good option in our case; expression of the transposition machinery in the host organism will create an inevitable bias, with transposition events most likely resulting in configurations that are associated with a smaller metabolic cost.
A solution to this problem would be to do the transposition in vitro. However, further research revealed several disadvantages with this idea. The mechanisms of Tn7 recombination are complicated, requiring many proteins for successful transposition. Although we thought this would be interesting to test, we came across difficulties with sourcing the TsnD subunit of the Tn7 transposon. As expression and purification of this subunit was not realistic within the timeframe we had, we decided to pursue a different method.
CRISPRi (dcas9 mediated gene repression)
The next idea on how to obtain different phenotypes of bacteria was the use of RNA interference. In this process RNA molecules inhibit gene expression on the transcriptional level.
An extensive literature search revealed that the CRISPR interference system seems to be a reliable and predictable RNA interference mechanism. CRISPRi influences gene expression primarily at the transcriptional level and allows sequence-specific control of gene expression. This method utilises the CRISPR pathway and a catalytically inactive Cas9 (dCas9) protein. In the traditional type II CRISPR system, Cas9 introduces double-stranded breaks in specific genomic sequences, guided by a short guide RNA (sgRNA). dCas9 is engineered to lack nuclease activity enabling repurposing of the system for genomic DNA targeting without cleavage and therefore allowing precise transcription regulation2.
As a result, we came up with an idea that a variety of reporters could be under the control of promoters which can be targeted by sgRNAs and dCas9. The sgRNA-dCas9 complex can be targeted to bind to the sequence of the promoter. By doing so, it physically interferes with translation, inhibiting the transcription machinery from recognising and binding to the promoter.
1Peters, J. E. (2014). Tn7. Microbiology Spectrum, 2(5). doi:10.1128/microbiolspec.mdna3-0010-2014
2Dominguez, A. A., Lim, W. A., & Qi, L. S. (2016). Beyond editing: repurposing CRISPR–Cas9 for precision genome regulation and interrogation. Nature Reviews. Molecular Cell Biology, 17(1), 5–15. http://doi.org/10.1038/nrm.2015.2
