<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNOTT_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNOTT/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-default navbar-static-top" role="navigation" style="padding-top: 0.5%"><DIV class="container" style="padding-top: 0.5%"><DIV class="navbar-header" style="padding-top: 10px"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="navbar-collapse collapse" style="padding-top: 0.5%"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:UNOTT">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:UNOTT/Description" data-toggle="dropdown">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Medal_Criteria">Medal Criteria</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">LAB</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Notebook">Lab book</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">MODELLING</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Modelling">Models </A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">OUTREACH</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Engagement">Public Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Team">Our Team</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:UNOTT/Attributions">Attributions</A></LI></DIV></DIV></DIV></DIV><DIV id="container" style="background-color: #02263E;"><P> </P><P> </P><H1 style="text-align:center;color:#ffffff;">DEMONSTRATE</H1><P><SPAN style="color: #ffffff;"> </SPAN></P></DIV><DIV class="contentmargin"><H2><I>Remember to celebrate milestones as you prepare for the road ahead</I></H2><H4> - Nelson Mandela</H4><P>In the given time frame, we achieved important milestones that will pave the long way that is ahead of us for the successful development of the <I>Key. coli</I> system. </P><H1>1. Promoter library with a wide expression range</H1><P>We employed 5 different promoters previously tested in <I>E. coli</I> (p1-5), alongside a promoter sequence with modified TATA boxes (pE) that would serve as a negative control. These were combined with strong and weak versions of suitable ribosome binding sites, expanding the library to 12 possible expression signals. The promoter library was tested with RFP. </P><P>The expression signals employed yielded different degrees of fluorescence, spanning a wide range over all time points assayed. The wide expression range is an advantage for the <I>Key. coli</I> system, introducing a desired degree of variability.</P><P>In future embodiments of the <I>Key. coli</I> system,  the promoter library can be further expanded to include more expression signals with the aim of pushing the boundaries of the expression range observed here even further. </P><H1>2. sgRNA-promoter pairs with varying repression efficiencies</H1><P>We employed 5 different sgRNAs (sgRNA1-5) with SEED sequences complementary to the chosen promoters (p1-5) and a sgRNA0 used as a control, unable to target any of the five selected promoters nor the <I>E. coli</I> genome.</P><P>We demonstrated that dCas9 RNA interference is feasible, efficient and specific as the constitutive expression of both dCas9 and each of the tested sgRNAs led to loss of the red colouration of colonies on the plate or reduction of red fluorescence in liquid culture; this is a qualitative proof of effective interference in RFP transcription. 
</P><P>Nevertheless the degree of  interference was different between different sgRNAs. As expected, there was no significant transcription interference when each sgRNA was combined with the null promoter pE, suggesting that the interference is specific to the target region. Interference was also quantified with a fluorescence assay. </P><P>The combined use of promoters of different strengths and sgRNAs of varying repression efficiencies has generated a greater range of expression profiles.</P><P>The proof-of-concept of CRISPR interference demonstrated here allows us to envision its permanent incorporation in the <I>Key. coli</I> system, which could even include a further expansion of the sgRNA pool.</P><H1>3. Random plasmid assembly via ligation is a feasible strategy</H1><P>The degree of randomness that can be achieved from a ligation reaction was put to the test in order to assess the suitability of this technique for the large scale generation of keys.</P><P>A strong and weak promoter was combined with all three fluorescent reporters: GFP, RFP and CFP. Ligations were carried out with controlled use of the DCBA restriction system, with the positions that each reporter could assume being predetermined. In theory, under this design, 2<SUP>3</SUP> possible versions of a ligated plasmid could be obtained. From a sample size of 12 colonies, 6/8 combinations were encountered with comparable distribution frequencies. It is logical to assume that upon screening a larger colony sample, all possible combinations could be encountered. </P><P>The above results provide strong evidence that assembly via ligation can generate a truly random combination set, thus being a suitable method for key generation in the future. Scaling-up the random ligation process would be possible with larger part pools, supported by the increasing availability of robotic platforms and relevant high-throughput, automated protocols for plasmid assembly, bacterial transformation, colony picking and screening. </P><H1>4. Freeze-drying is a suitable storage method that allows reproducible revival and expression output</H1><P>Two engineered <I>E. coli</I> strains expressing RFP under strong or weak promoters were cultured, aliquoted and freeze-dried. These aliquots were then split in three subgroups relative to their storage environment: -80°C, 4°C and room temperature (RT). At frequent and regular times, cells were revived in liquid medium. OD and fluorescence measurements were performed to assess the growth and fluorescence profile for each sample.</P><P>The results show that storage temperature does not have a significant impact on revival quality up to 2 weeks of storage. At 3 weeks however, the expression profile of samples stored at RT begins to deviate. Longer-term experiments would be able to provide more accurate information. </P><P>Based on the current data, -20oC would be a suitable storage temperature for the <I>Key. coli</I> device ensuring consistency of the expression output. Most importantly, this temperature is easily achieved using a conventional freezer.</P><P>Expression output is identifiable from 2-hrs onwards, implying that the system can also be used with a very small time delay, in cases where time is of essence. However, for more accurate use, we recommend that the ‘unlock’ attempt should take place from 4-hrs onwards.</P><P>Growth and expression profiles obtained elsewhere (e.g. Groningen) were also comparable to the ones obtained in Nottingham, pointing towards the potential universality of the &gt;i&gt;Key. coli system.</P><H1>5. Modelling and software programs underpinning <I>Key. coli</I></H1><P>To support the work done by the wet team, diverse and complementary models were developed.</P><P>First, constitutive gene expression was modeled to set the basis of protein synthesis in a bacterial cell. Then, CRISPR interference (CRISPRi) was superposed to this basic model by introducing different parameters such as dCas9 production rate and dCas9:gRNA association rate, simulating transcription interference on the expression of one, two or i fluorescent proteins (FPs). A third layer of modelisation linked the maximum expected fluorescence to the protein concentration available, allowing access to the latter directly via spectrophotometric measurements.</P><P>As fluorescence was at the heart of our project, an absorption and emission model summarised all useful values that could be used to excite and detect emission from each of the three selected FPs.</P><P>In addition to the above, a software was developed to assess fluorescence levels between <I>Key. coli</I> capsules and the mother strain. The software is able to translate spectrophotometric readings into image-like data using an image similarity algorithm, with predefined threshold values. Readings outside of the threshold range would result in “Denied Access” to the lock of interest.</P><P>Finally, we have generated a Random Number Generation tool and a user-friendly Excel sheet with editable Protein Concentration fields that can be used to create a customised strain and get the associated tweaked combination allowing for randomness and flexibility.</P><H1><SPAN style="color: #588BAE;">Conclusion</SPAN></H1><P><SPAN style="color: #588BAE;">In this study we have shown the first steps taken towards the development of a randomly assorting, fluorescent bacterial key: the <I>Key. coli</I>. The results of the proof-of-concept experiments undertaken provide us with confidence that <I>Key. coli</I>, albeit futuristic, is a feasible technology employing available know-how, such as large part pools, random plasmid assemblies and CRISPR interference. In addition to proving the scientific feasibility of <I>Key. coli</I>, we have also designed a safe and portable device for its storage and revival at a desired time point. Finally, we have paired <I>Key. coli</I> to a streamlined authentication procedure, tailored to be immune to current hacking frameworks. 
</SPAN></P><P><SPAN style="color: #588BAE;">We hope that this next generation biological security system will one day be commercialised so we can all protect our gems with germs! Until then, there is lots of exciting work to be done by team UNOTT...
</SPAN></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>