<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HFLS_H2Z_Hangzhou_sub_enzyme skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HFLS H2Z Hangzhou/sub/enzyme</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><SPAN class="error">Template loop detected: <A href="/Template:HFLS_H2Z_Hangzhou" title="Template:HFLS H2Z Hangzhou">Template:HFLS H2Z Hangzhou</A></SPAN>&lt;!DOCTYPE html&gt;</P><DIV class="main view"><DIV class="main nav"><A class="logo-name trex-go">Aquamade</A><SPAN class="link-set"><A href="/Team:HFLS_H2Z_Hangzhou" class="nav link-item home">Home</A><SPAN class="nav link-item achiev">Achievement
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/parts">Parts</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Collaborations">Collaborations</A></DIV></DIV></SPAN><SPAN class="nav link-item project">Project
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/overview">Overview</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/backg">Background</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Applied_Design">Design</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/enzyme">Enzyme</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/genecirc">Genetic Circuits</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Demonstrate">Performance</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/chassis">Chassis Selection</A></DIV></DIV></SPAN><SPAN class="nav link-item model">Modeling
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Model">Models</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Software">Software</A></DIV></DIV></SPAN><A href="/Team:HFLS_H2Z_Hangzhou/Safety" class="nav link-item safety">Safety</A><SPAN class="nav link-item team">Team
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/member">Member</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/Attributions">Attribution</A></DIV></DIV></SPAN><SPAN class="nav link-item note">Notebook
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/sub/protocol">Protocols</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/InterLab">Interlab</A></DIV></DIV></SPAN><SPAN class="nav link-item hp">Human Practices
					<DIV class="expand-item-set-cont"><DIV class="expand-item-set"><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/HP/Silver">Public Outreach</A><A class="sub-link-item" href="/Team:HFLS_H2Z_Hangzhou/HP/Gold_Integrated">Integrated</A></DIV></DIV></SPAN></SPAN></DIV><DIV class="main bottom-bar"><DIV class="block black-bg"><DIV class="wide article article-font" style="color: rgb(253, 253, 253);"><H2 style="color: rgb(253, 253, 253);">Team Members</H2><DIV style="font-family: Veneer;"><DIV class="member-badge">
							Jianan Li
						</DIV><DIV class="member-badge">
							Qingrui Sun
						</DIV><DIV class="member-badge">
							Yiming Rong
						</DIV><DIV class="member-badge">
							Jiayue Guo
						</DIV><DIV class="member-badge">
							Zhiyuan Lu
						</DIV><DIV class="member-badge">
							Meiqi Yuan
						</DIV><DIV class="member-badge">
							Zhengyao Lin
						</DIV><DIV class="member-badge">
							Caiyi Feng
						</DIV><DIV class="member-badge">
							Shuyun Zhang
						</DIV><DIV class="member-badge">
							Tenghao Huang
						</DIV><DIV class="member-badge">
							Yining Huang
						</DIV><DIV class="member-badge">
							Yanyue Zhu
						</DIV></DIV><H2 style="color: rgb(253, 253, 253);">Sponsors</H2></DIV></DIV></DIV><DIV class="main cont-wrap"><DIV class="page-enzyme comsec-main comsec-use-exnav" style="font-family: Veneer;"><DIV class="main-block fullscr main-title" style="font-size: 7rem; text-align: center;"><DIV class="vcenter" style="line-height: 1;"><SPAN>Enzyme</SPAN></DIV></DIV><DIV class="block white-bg"><H1>Overview</H1><DIV class="article article-font">
				To meet with the parameter we set in <A href="/Team:HFLS_H2Z_Hangzhou/Applied_Design">Design</A>, we first researched on possible ways to eliminate Nitrite ions. We settled down with three target enzymes to form a substrate channeling system(see <A href="/Team:HFLS_H2Z_Hangzhou/Applied_Design">Design</A> concerning goal one): NiR (nitrite reductase), NOR(nitric oxide reductase), nosZ(nitrous oxide reductase).
			</DIV></DIV><DIV class="block white-bg"><DIV class="wide article article-font"><DIV style="position: absolute; left: 0; top: 0;"><DIV class="ui secondary vertical menu float-menu sticky"><DIV class="ui secondary vertical menu"><A class="active item passlink" data-passage="consvec">Construct the Expression Vectors</A><A class="item passlink" data-passage="purehis">Purification of Proteins(His-tag)</A><A class="item passlink" data-passage="sdsana">SDS-PAGE Analysis</A></DIV><DIV class="header" style="margin-top: 1rem; margin-bottom: 0.5rem;">Enzyme Reactivity Assay</DIV><DIV class="ui secondary vertical menu"><A class="item passlink" data-passage="assayov">Overview</A><A class="item passlink" data-passage="aenzyme1">NiR (Nitrite Reductase)</A><A class="item passlink" data-passage="aenzyme2">NOR (Nitric oxide reductase)</A><A class="item passlink" data-passage="aenzyme3">NosZ (Nitrous Oxide Reductase)</A><A class="item passlink" data-passage="ref">Reference</A></DIV></DIV></DIV><DIV class="atext article-cont"><DIV class="passage passage-consvec shown"><H1>Construct the Expression Vectors</H1>
						Before we send out the order for the synthesis of our three enzymes' sequence, we added the restriction site BamHI before 5' of the sequence, we added the restriction site XhoI after 3' of the sequence.The sequence of BamHI:  ggatccThe sequence of XhoI:  ctcgag<DIV class="img-name">Construction of expression vectors</DIV>

						We also avoided the following restriction site in our synthesis order: NdeI,NheI,BamHI,EcoRI,EcoRV,HindIII,NotI,XhoI,SacI,SalI,NcoI, XbaI, SpeI, PstI. Among these, XhoI, EcoRI, SpeI, XbaI, NotI are avoided for the ease of RFC10 standardization.
						<DIV class="img-name">Target enzyme sequences after BamHI and XhoI digestion</DIV>

						We received our orders in plasmids. After transformation, culture and mini prep, we used BamHI XhoI to digest our target sequences from high copy vectors.
						
						The backbone(high copy cloning vector) is provided by DNA synthesis company.
						

						We also digested linearized pET28a with these two restriction enzymes.
						
						The primers for bulk PCR of pET28a are:
						
						Forward primer: TCGAATTCGGATCCGCGACCCATTTGCTGT Tm=68 °C
						<DIV class="img-name">pET28a afte BamHI and XhoI digestion</DIV>

						Reverse primer: GGCCGCACTCGAGCACCACCACCACCACCA Tm=74 °C

						<DIV class="img-name">Verification of our expression vectors using BamHI and XhoI restriction enzymes</DIV>

						Then, we ligated our target enzyme sequences with pET28a vector, and after verification and sequencing, we moved to the next stage.
					</DIV><DIV class="passage passage-purehis"><H1>Purification of Proteins with His-tag</H1><H2>Materials</H2>Cell cultures1M IPTG1xPBS bufferNi-NTAImidazoleGravity Chromatography Columns, 5mlUltrasonic Cell Disruptor
						
						<H2>Process</H2><TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no">1</DIV></TD><TD>Grow 30mL cultures of cells with LB for 3-4 hours.</TD></TR><TR><TD><DIV class="no">2</DIV></TD><TD>Measure OD of the cells.</TD></TR><TR><TD><DIV class="no">3</DIV></TD><TD>Add 3µL IPTG into cell cultures if OD is between 0.4-0.6. (Continue incubation if OD is still below 0.4)</TD></TR><TR><TD><DIV class="no">4</DIV></TD><TD>Incubate the cultures at 16°C, 150rpm overnight.</TD></TR><TR><TD><DIV class="no">5</DIV></TD><TD>Centrifuge the overnight cultures at 6,000rpm for 5 mins. </TD></TR><TR><TD><DIV class="no">6</DIV></TD><TD>Remove the supernatant(LB) after centrifugation. </TD></TR><TR><TD><DIV class="no">7</DIV></TD><TD>Add 20mL 1xPBS into the centrifuge tube, suspend the cell deposit. </TD></TR><TR><TD><DIV class="no">8</DIV></TD><TD>Centrifuge at 6,000rpm for 5 mins again to remove LB remains. </TD></TR><TR><TD><DIV class="no">9</DIV></TD><TD>Crack the cells using a ultrasonic cell disruptor for 20 minutes. (5s on / 10s off) </TD></TR><TR><TD><DIV class="no">10</DIV></TD><TD>Centrifuge the products at 12,000 rpm, 4°C for 5 mins. </TD></TR><TR><TD><DIV class="no">11</DIV></TD><TD>Collect the supernatant and add 100µL Ni-NTA.</TD></TR><TR><TD><DIV class="no">12</DIV></TD><TD>Place the mixture on a shaker for 1h.</TD></TR><TR><TD><DIV class="no">13</DIV></TD><TD>Add the protein solution into the gravity chromatography columns and collect the effluent. </TD></TR><TR><TD><DIV class="no">14</DIV></TD><TD>Add 20mM Imidazole into the columns to wash away impurities. </TD></TR><TR><TD><DIV class="no">15</DIV></TD><TD>Add 250mM Imidazole into the columns to wash off the proteins. Collect the effluent. </TD></TR></TBODY></TABLE></DIV><DIV class="passage passage-sdsana"><H1>SDS-PAGE Analysis</H1><DIV class="img-name">SDS-PAGE analysis</DIV>

						To test the expression and reactivity of each enzyme—NOR, NiR, and NosZ—we expressed them seperately on E.coli strain BL21(DE3). Please see our protocol Purification of Proteins with His-tag.   After purification, we examined each sample for the presence of enzymes using SDS-PAGE analysis. The results indicated that all three enzymes were successfully expressed and purified.
					</DIV><DIV class="passage passage-assayov"><H1>Overview</H1>

						To make sure our three enzymes' successful expression and reaction in $E. coli$ chassis, we conducted assays on three enzymes individually in vitro. Then, if the assay method can be applied to in vivo conditions, we perform the assay of enzyme activity in vivo.
						
						Throughout our experiments, we concluded there are three possible ways to assay enzymes in our case.
						
						<TABLE class="frameless ref-table"><TBODY><TR><TD>Measure the reduction in substrate concentration.</TD></TR><TR><TD>Measure the increase in product concentration.</TD></TR><TR><TD>Measure enzyme reactivity by monitoring the electron loss (specific to our case: all  three enzymes catalyze reduction—enzyme loses electrons while substrate gains electron hence reduced).</TD></TR></TBODY></TABLE>

						As you will see when going through our assay report, we successfully measured the reactivity of two enzymes: NiR(nitrite reductase) and nosZ(nitrous reductase).
						
						We measured the reactivity of NiR both in vivo and in vitro. This was because the method we leant from ZECCC was sensitive and accurate(see our <A href="/Team:HFLS_H2Z_Hangzhou/HP/Gold_Integrated">Integrated Human Practice</A>). For enzyme nosZ however, we only found one method which is investigating enzyme's electron loss rate using intermediate color reagent so we measured its reactivity in vitro. We simply cannot use this method in vivo because not only the reagent toxic to metabolic activity, but the intermediate color reagent is unstable. 
						
						For the other enzyme NOR(nitric oxide reductase), we tried several methods, the first two while feasible in theory, were unsuccessful because of the unavoidable limitations in practice. We did manage to found correct method but could not access the high-end Gas-Phase Molecular Absorption Spectrometry, so we cited the experiment results from the paper measuring the same enzyme expressed in same chassis $E.coli$.
					</DIV><DIV class="passage passage-aenzyme1"><H1>$\mbox{NiR}$ (Nitrite Reductase)</H1><H2>Background</H2>
						
						The first enzyme NiR is the product from expression of nirS gene amplified from Alcaligenes eutrophus DSM 530. The NiR catalyzes the reduction of nitrite (NO2-) to nitrogen monoxide(NO), as the first step of our substrate channeling system.
					
						<H2>Biology</H2><DIV class="img-name">Nitrite reductase's tertiary structure</DIV>

						We expressed the nitrite reductase with a His-tag in E.coli using a pET28a vector. The proteins are then extracted from the cells and purified using a HIS-trap column. After further purification, we ran electrophoresis to check the correct expression of NiR. The result of protein electrophoresis is shown below.

						<H2>Assay method</H2>
						We used N-(1-naphthyl)ethylene diamine dihydrochloride spectrophotometric method, a common approach in waste water treatment, to measure the concentration of nitrite in the solution to track the reaction process. The nitrite concentration was measured using a color-showing reagent consisting of p-aminobenzenesulfonamide and 1g n-(1-naphthyl)-ethylenediamine dihydrochloride.
						
						The reagent reacts with nitrite to form the redish pink diazo salt  in the solution, which has a maximum absorbance around 540nm. A standard curve was first made by measuring the absorbance at 540nm  of the reaction system under various nitrite concentration.
						
						Stand curve between Absorbance-540nm and Sodium Nitrite concentration is shown below.

						<DIV class="img-name">Standard curve between Absorbance-540nm and Sodium Nitrite concentration</DIV><H2>Reactivity assay in vitro</H2>
						To assay the reactivity of the protein. We put the enzyme, sodium nitrite and pbs buffer together  to establish a reaction system. We recorded the nitrite concentration over 24 hours, under temperature of 37 degree Celcius. A standard curve was first made(as seen above) by measuring the absorbance at 540nm of the reaction system under various nitrite concentration.
						
						According to our results, the absorbance dropped from 0.633 to 0.008 over 24 hours, which indicates a drop of nitrite concentration from 0.196 mg/L to roughly zero. Currently we are still testing samples with shorter and various reaction time-length to plot the closest curve of the reactivity.

						<DIV class="img-name">NiR reactivity assay in vitro</DIV>

						Since the application of these enzymes will be in vivo, we are satisfied with NiR's reactivity was verified. We are much more interested in NiR's ability to react in vivo with much greater concentration of nitrite.
					
						<H2>Reactivity assay in vivo</H2><H3>Protocols</H3><TABLE class="frameless ref-table"><TBODY><TR><TD>Add IPTG into prepared cultures of E.coli (IPTG concentration: 0.1mg/L)</TD></TR><TR><TD>Induce at 16 °C, 150 rpm overnight</TD></TR><TR><TD>Add sodium nitrite solution at various concentration (Target concentration: ~65mg/L)</TD></TR><TR><TD>Culture the bacteria at 37 °C,150 rpm</TD></TR><TR><TD>Take samples (500uL) of the cultures at various time points</TD></TR><TR><TD>Measure optical density(600nm) of the samples</TD></TR><TR><TD>Centrifugate at 6,000 rpm for 60s </TD></TR><TR><TD>Take 100uL samples from supernatant and dilute with 900uL ddH2O</TD></TR><TR><TD>Take 100uL samples from the diluted solution and dilute again with 900 uL ddH2O</TD></TR><TR><TD>Add 20uL of chromogenic agent to the solution</TD></TR><TR><TD>Stand the solution for 20 mins and wait for the color to develop. </TD></TR><TR><TD>Measure the absorbance(540nm) of the solutions </TD></TR><TR><TD>Record data in Excel forms</TD></TR></TBODY></TABLE>
						*Prepare the chromogenic agent as follows:

						<TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no">1</DIV></TD><TD>In 500mL beaker
										</TD></TR><TR><TD>50mL Phosphoric Acid (d=1.70g/m)</TD></TR><TR><TD>20.0g p-aminobenzenesulfonamide (C6H8N2O2S)</TD></TR><TR><TD>1g n-(1-naphthyl)-ethylenediamine dihydrochloride</TD></TR><TR><TD>250mL ddH2O</TD></TR></TBODY></TABLE><DIV class="no">2</DIV>Transfer the solution to a  500mL volumetric flask and dilute with ddH2O to standard volume.<B>NOTICE: We diluted the solution x100, so you have to time 100 the corresponding nitrite concentration using standard curve.</B><H3>Results</H3><DIV class="img-name">Nitrite conc. against reaction time</DIV><TABLE class="frameless ref-table"><TBODY><TR><TD>According to our results, the absorbance at 540nm dropped from 0.410 to 90.105 over 45.5 hours, which indicates a drop of NaNO2 concentration from 62.0(mg/L) to 14.9(mg/L).</TD></TR><TR><TD>According to the graph, the reactivity of NiR is the highest during 20-30hours. About 76% of total nitrite is degraded at the end of assay. Also, there is no obvious reduction in nitrite concentration in control group.</TD></TR></TBODY></TABLE></DIV><DIV class="passage passage-aenzyme2"><H1>$\mbox{NOR}$ (Nitric oxide reductase)</H1><H2>Background</H2>

						Cytochrome P450 55B1 from Chlamydomonas reinhardtii
						General Equation: $ \ce{2NO + NAD(P)H + H+ → N2O + NADP+ + H2O} $

						
												
						We found three assay methods to test the activity of nitric oxide reductase: using NO measurement Kit, titration and Gas-Phase Molecular Absorption Spectrometry. The first two methods were failed, and the details and reasons were recorded below. Due to limitations of lab equipment, we were not able to conduct the experiment using the third method.

						<DIV class="img-name">Nitric oxide reductase's tertiary structure</DIV><H2>Assay method 1: NO Measurement Kit</H2>
						
						The NO kit we planned to use was developed by Nanjing Jiancheng Bioengineering Institute, and the concentration of NO in the solution can be calculated by the concentration of $\ce{NO3-}$, because NO will be reduced to $\ce{NO3-}$ in blood plasma of chicken or mice. We prepared the solution according to the method as below:
						<TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no">1</DIV></TD><TD>Remove the air in the flask by passing through nitrogen gas for 30 minutes.</TD></TR><TR><TD><DIV class="no">2</DIV></TD><TD>Pass through nitric oxide gas for 45 minutes in Flask C that contains ddH2O.</TD></TR><TR><TD><DIV class="no">3</DIV></TD><TD>Flask A contains 30% H2O2 solution, and Flask B contains KMnO4 / NaOH solution, to oxidize and absorb the undissolved gas.[1]</TD></TR></TBODY></TABLE>
						However, although the NO solution we prepared using this method was saturated, the concentration was too low for the verification to proceed.
					
						<DIV class="img-name">Fig 1. Setup of equipments for preparing NO solution</DIV><DIV class="img-name">Fig 2. NO gas container</DIV><DIV class="img-name">Fig 3. Equipments setup</DIV><H2>Assay Method 2: Titration</H2>

						By measuring the concentration of nitrous oxide—the final product of the reduction of nitric oxide, the amount of nitric oxide reacted can be calculated, and therefore the activity of nitric reductase can be verified. When the reaction is complete, acidic potassium permanganate is added to the solution, and the highly oxidative potassium permanganate reacts with nitrous oxide. We then use oxalic acid to titrate the remaining potassium permanganate. Therefore, the amount of potassium permanganate used in the reaction with nitrous oxide can be calculated, and the amount of nitrous oxide produce can be found.

						

						However, even for the most stable solution of nitrous oxide—the standard solution of 1.00mg/ml, nitrous oxide will escape in gas form, and the escaping speed is not constant. As a result, the method of directly measuring the amount of nitrous oxide is not valid.

						<DIV class="img-name">Fig 4. Standard N2O solution</DIV><H2>Assay Method 3: Gas-Phase Molecular Absorption Spectrometry</H2>

						The turnover of the overall reaction of nitric oxide reductase was determined by monitoring the NADH consumption rate under various concentrations of NO, where the NO concentration was controlled by bubbling the NO/N2 mixed gas produced with the gas divider (ESTEC SGD-SC-0.5L). Unfortunately, since we could not find access to the required gas divider, we were not able to verify the results. The results from the original paper was shown in Fig 4.
						<H3>Protocol</H3><TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no">1</DIV></TD><TD>Prepare the buffer: 0.1 M sodium phosphate at pH 7.2.</TD></TR><TR><TD><DIV class="no">2</DIV></TD><TD>Add reduced form of NADH.</TD></TR><TR><TD><DIV class="no">3</DIV></TD><TD>Purge the air in the buffer solution by nitrogen gas to prevent formation of NO2 before adding NO.</TD></TR><TR><TD><DIV class="no">4</DIV></TD><TD>Introduce NO gas into the sample solution after passing 1 M KOH and the buffer solutions. </TD></TR><TR><TD><DIV class="no">5</DIV></TD><TD>Incubate the buffer solution containing NO (2.5, 2.0, 1.6, 1.5, 1.2, or 1.0 mM) and NADH (0.16 mM) in the optical cell at 10°C for 5 min.</TD></TR><TR><TD><DIV class="no">6</DIV></TD><TD>Measure the absorbance (340nm) of the solutions.</TD></TR></TBODY></TABLE><H2>Results</H2>

						Due to limited access to equipments, we could not directly measure the reactivity of NOR. However, experiments had been carried out for the same enzyme expressed also expressed in E.coli, with method listed above. For the continuation of our project, we had to draw conclusion from other's experiment results.

						<DIV class="img-name">Fig 5. Time course of the absorption change of the reduced form of NADH at 340 nm in the NO reduction reaction catalyzed by P450nor. [2]</DIV><H2>Conclusion</H2><B>We delightedly found out that the curve became smooth and parallel to the x-axis just within the course of five minutes. This represented nearly all the substrates had been converted to products in an extremely rapid rate: much more rapid than nitrite reductase. This means that whenever NO is released from nitrite reductase as product, NOR can utilize it as substrate quickly.</B><H2>Reference</H2><TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no noind">1</DIV></TD><TD>程得胜, 黄曜, 黄郁芳. 一氧化氮标准储备水溶液浓度的测定及稳定性分析[J]. 分析测试学报, 2007, 26(4):566-569.</TD></TR><TR><TD><DIV class="no noind">2</DIV></TD><TD>Shiro Y., Fujii M., Iizuka T., Adachi S., Tsukamoto K., Nakahara K., Shoun H. 1995. Spectroscopic and kinetic studies on reaction of cytochrome P450nor with nitric oxide. Implication for its nitric oxide reduction mechanism. J. Biol. Chem. 270, 1617–162310.1074/jbc.270.4.1617.</TD></TR><TR><TD><DIV class="no noind">3</DIV></TD><TD>Shoun H, Fushinobu S, Jiang L, et al. Fungal denitrification and nitric oxide reductase cytochrome P450nor[J]. Philosophical Transactions of the Royal Society of London, 2012, 367(1593):1186.</TD></TR></TBODY></TABLE></DIV><DIV class="passage passage-aenzyme3"><H1>$\mbox{NosZ}$ (Nitrous Oxide Reductase)</H1><H2>Background</H2><DIV class="img-name">Nitrous oxide reductase's tertiary structure</DIV>

						This enzyme catalyzes the reduction of nitrous oxide(N2O) to nitrogen(N2) and oxygen(O2). The enzyme comes from Agrobacterium tumefaciens. The enzyme contains two copper center, namely CuA and Cuz, which can transfer the electron.

						<H2>Assay Method</H2>

						We prepared a standard measuring mixture(2ml) consisting of 50mM Tris-HCL(pH9), 0.5mM methyl viologen, a proper amount of NosZ and 1mM sodium dithionite and 160uL of nitrous oxide solution.
						
						Methyl viologen and sodium dithionite will go on to react and form a blue intermediate product, with a relativelly stable absorbance at 600nm. The blue intermediate product serves as an electron donor for the reduction process of nitrous oxide. Therefore, as the reaction goes on, the blue substance starts to lose electrons and its color dims, resulting in a lower 600nm absorbance. Thus, by directly measuring the change of 600nm absorbance of the system, we are able to track the processing of the reaction.
					
						<H2>Results</H2><H2>Conclution</H2><B>
							We delightedly found out that for just 900 seconds, the abs600 value of the color reagent dropped 32%. This represented nearly all the substrates had been converted to products in an extremely rapid rate: much more rapid than nitrite reductase.
							
							And even though we cannot make the statement which among nosZ and NOR has a faster reactivity rate, we do know the fact that both enzymes react in a much faster rate than NiR.
						</B></DIV><DIV class="passage passage-ref"><H1>Reference</H1><TABLE class="frameless ref-table"><TBODY><TR><TD><DIV class="no noind">1</DIV></TD><TD>Braker, Gesche &amp; Fesefeldt, Andreas &amp; P Witzel, K. (1998). Development of PCR Primer Systems for Amplification of Nitrite Reductase Genes (nirK and nirS) To Detect Denitrifying Bacteria in Environmental Samples. Applied and environmental microbiology. 64. 3769-75.</TD></TR><TR><TD><DIV class="no noind">2</DIV></TD><TD>Gong, B., Liang, X., Li, Y., Xiao, Q., Yang, P., &amp; Wu, Y. (2017). Expression and purification of cytochrome p450 55b1 from chlamydomonas reinhardtii and its application in nitric oxide biosensing. Applied Biochemistry &amp; Biotechnology(10), 1-11.</TD></TR><TR><TD><DIV class="no noind">3</DIV></TD><TD>Henry, S., Bru, D., Stres, B., Hallet, S., &amp; Philippot, L. (2006). Quantitative detection of the nosz gene, encoding nitrous oxide reductase, and comparison of the abundances of 16s rrna, narg, nirk, and nosz genes in soils. Applied &amp; Environmental Microbiology, 72(8), 5181-5189. Mu, Yunlong, et al. Molecular cloning and expression of a gene encoding nitrite reductase from Alcaligenes faecalis TR126.</TD></TR><TR><TD><DIV class="no noind">4</DIV></TD><TD>Kristjansson, Jakob K., and Thomas C. Hollocher. “First Practical Assay for Soluble Nitrous Oxide Reductase of Denitrifying Bacteria and a Partial Kinetic Characterization.” The Journal of Biological Chemistry, vol. 255, 27 Aug. 1979.</TD></TR><TR><TD><DIV class="no noind">5</DIV></TD><TD>Liu, Xueqian, et al. &quot;The nos gene cluster from gram-Positive bacterium Geobacillus thermodenitrificans NG80-2 and functional characterization of the recombinant NosZ.&quot; FEMS Microciol Lett, vol. 289, no. 46, ser. 52, 2008. 52.</TD></TR><TR><TD><DIV class="no noind">6</DIV></TD><TD>Yeung, Natasha, et al. “Rational Design of a Structural and Functional Nitric Oxide Reductase.” Nature, 24 Dec. 2009.</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>