Team:UMaryland/Description
</div> </div>
Modeling
Fix
Projects
Parts
Modeling
Outreach
Notebook
About Us
Projects
Parts
Modeling
Outreach
Notebook
About Us
Methane Biosequestration
Creating Bacteria to Break Down Methane
Environmentally friendly strategy for ameliorating global climate change
Background
Design
Results
Conclusions
Abstract
Global climate change is the most profound threat facing society. Greenhouse gasses like carbon dioxide and methane are major contributors, with methane being 29 times more potent than carbon dioxide. Sources of methane include landfills and animal farms. Current methods to control methane emissions include combustion into carbon dioxide, which depletes methane but at the levels present naturally is inefficient for generating energy. We plan to create strains of E. coli that break down methane using methane monooxygenase from methanotrophs, organisms that use methane as an energy source. Methane will be oxidized into methanol, which could be extracted for industrial use, but we plan instead to introduce other metabolic pathways (perhaps in co-cultured bacteria) that convert methanol into cellular metabolites (biomass) or else carbon dioxide. Our biological removal of methane could be implemented into the piping of landfills as an environmentally friendly strategy for ameliorating global climate change.
Background
Global climate change is one of the most pertinent threats facing our world today. Greenhouse gases absorb solar energy and reemit it as heat in the Earth’s atmosphere, causing the temperature to rise. Most people are already aware that carbon dioxide is a major contributor to the greenhouse gas effect. Methane is another such greenhouse gas, and, though not as abundant as carbon dioxide, is actually more potent per molecule1. While most government agencies and companies are working to reduce carbon dioxide emissions, the reduction of methane emissions is often overlooked.
The three biggest contributors to methane emissions are fossil fuel production, livestock farming, and landfills. Several ways of managing and reducing emissions are employed at these sources. Legislation passed by the EPA requires oil and gas companies to monitor their methane emissions and forces them to operate under regulations that require emissions to be under a certain quantity2. Methane in landfills is managed by flaring or used in power generators that function as an alternative energy source (Peter Karasik, personal communication, June 17, 2016).
Livestock produce methane gas in their metabolism and manure produces methane as it decomposes. In attempt to control the amount of methane produced by the latter, farms can implement anaerobic digesters that use the biogases produced by anaerobic digestion and decomposition of manure to create an alternative energy or heat source3. These regulations, however, do not stop the problem. Flaring is not 100% effective4 and can produce air pollutants, and machines that are able to use methane as an energy or heat source can be expensive and difficult to maintain (Peter Karasik, personal communication, June 17, 2016). Despite these efforts, a lot of methane still escapes into the atmosphere.
Landfill electricity generator system using methane as seen in Gude Landfill, MD
In order to create a more foolproof method to tackle the issue of methane pollution, we selected a biological approach. Using genes from several organisms that have evolved to metabolize methane, we plan to create strains of bacteria that are capable of breaking down methanol. These bacteria would generate metabolites during the process, and will either accumulate biomass or release CO2.
Design
Our design combines the work of several previous iGEM teams, creating an integrative start to finish approach of tackling this immense global issue. To sequester methane, we will engineer E. coli to be able to metabolize methane. We have designed two metabolic pathways that involve three different plasmids, each converting methane into CO2 and energy, or biomass. Methane is not only removed in the process, but the cell gains metabolic advantages through these pathways. Energy can also be eventually harvested from the resulting generated biomass.
Both pathways start with the conversion of methane to methanol using the sMMO gene. This gene has been characterized by Team Braunschweig 2014, who assembled the gene into a polycistronic six subunit plasmid. sMMO oxidizes methane, converting it to methanol.
The sMMO gene used in our plasmid is extracted from Methylococcus capsulatus. This organism is an obligate methanotrophic bacteria, requiring methane as its carbon source5. M. capsulatus grows optimally at 45 C6 in environments with high methane concentrations, such as soils and inside live-animals as symbionts. Naturally, this strain of bacteria and many other methanotrophs are actively reducing methane emissions by metabolizing atmospheric methane.  M. capsulatus is the most extensively studied methanotroph and its genome has been sequenced7.
Once the methane is converted to methanol, another strain of recombinant E. coli will metabolize the methanol. This will be able to happen in two different pathways, dubbed the “fructose” and “formate” pathways.
Our “Fructose” plasmid consists of three enzymes found in Bacillus methanolicus. These enzymes provide a pathway for metabolizing methanol. B. methanolicus is a thermotolerant bacteria, growing optimally at 50 C that uses methanol as its carbon source8. It can be found in freshwater marsh soil9. This bacteria can be used to generate amino acids from methanol, thereby converting the toxic alcohol into protein building blocks.
This plasmid is an improvement based on work done by Team Aachen in 2015. The part they created was a composite of four enzymes - methanol dehydrogenase (MDH2), 3-hexulose-6-phosphate synthase (HPS),  6-phospho-3-hexuloisomerase (PHI), and D-Xylulose-5-phosphate-phosphoketolase (XPK). One of our modifications to their part includes replacing a constitutive promoter with a lac + pL promoter, allowing for us to selectively induce the genes. We also optimized all the codons, ensuring most efficient expression by E. coli.
Additionally, our plasmid did not incorporate D-Xylulose-5-phosphate-phosphoketolase (XPK). XPK is an enzyme that is used in the pentose phosphate pathway to eventually get a product used in glycolysis10. However, the product of D-Fructose-6-Phosphate, is already a molecule that is incorporated in glycolysis. Having the cell express XPK is unnecessary and will only tax the cell to make more enzyme than it requires, thereby slowing its growth.
Overall, this enzymatic pathway incorporates the carbon from methanol into fructose-6-phosphate. This transformation from a 5 carbon sugar to a 6 carbon sugar allows the product to be broken down further via the glycolysis pathway, providing energy to the cell. It is important to note that this pathway requires an input sugar source, since the 5 carbon derivative is essential for the complete reaction.
The second pathway utilizes the “Formate” plasmid, which takes. We designed the plasmid with the same MDH2 gene as the Fructose plasmid, but incorporated formaldehyde dehydrogenase (FALDH), and formate dehydrogenase (FDH), genes previously characterized by Team UESTC-China 2014. These enzymes together make a pathway that converts methanol to carbon dioxide by first using MDH2 to convert methanol to formaldehyde, using FALDH to convert formaldehyde to formate, and using FDH to convert formate to the final product. At each step in the pathway, energy is released in the form of reduction of NAD+ into NADH, serving as a source of energy for the cell.
The engineered bacteria will be co-cultured to provide the complete pathway of methane metabolism.This co-culture can be applied to the piping of landfills, where methane gas may escape.
Construction
sMMO
We attempted to recreate this plasmid using several different methods. The plasmid consists of a lac pL promoter, inducible by IPTG, six sMMO subunits, a ribosome binding site before each subunit, and a double terminator. The subunits are MMOB, MMOC, MMOD, MMOX, MMOY, and MMOZ. MMOB, MMOC, MMOD, and MMOX were all ordered from the registry. The lac + pL promoter and ribosome binding site were obtained from the kit. An RBS, MMOY,  another RBS, MMOZ, and the double terminator were designed into a G-Block provided by IDT. We designed the G-Block to have an overhang on both ends that corresponded to the ends of a linearized backbone and then performed a gibson assembly to insert that set of genes into a chloramphenicol resistant backbone provided by iGEM. We then attempted to combine a ribosome binding site with each subunit using 3A assembly.  The plan was then to sequentially combine the subunits together, combine them with the G-Block part, and then combine the promoter with that using more 3A assemblies. Unfortunately, we were not able to find success in this strategy and the length of the process proved to be a major setback so we looked to try a new method.
For this new method, we hoped to take advantage of the ease of gibson assemblies. We designed primers for each subunit. The forward primer would contain an overhang to correspond to the part it would attach to as well as a ribosome binding site. The reverse primer would simply anneal to the end of the subunit, excluding the final subunit, which would also contain an overhang that would correspond to the backbone. The products of the PCR reactions would then be put into a gibson reaction with a plasmid that had a chloramphenicol resistant backbone and an insert of the lac pL promoter. The proposed order would be MMOB, MMOC, MMOD, and MMOX, with MMOB having an overhang with the end of the lac pL promoter, MMOC having an overhang with MMOB, MMOD having an overhang with MMOC, and MMOX having an overhang with MMOD (in the forward primer) and the backbone (in the reverse primer). This method, too, was not providing us the results we were expecting.
Our final attempt at creating the sMMO plasmid involved purchasing two new G-Blocks, one with the promoter, a ribosome binding site, and MMOX, and another with ribosome binding sites in front of MMOB, MMOC, and MMOD. These G-Blocks were designed with overhangs to be combined with each other and be used in a gibson reaction with a chloramphenicol resistant backbone to create a plasmid. This plasmid was then to be combined with the MMOY/MMOZ plasmid using a 3A assembly. Unfortunately, despite screening over 20 Gibson assembly colonies, none of them yielded the correct sequence. As such, the sMMO plasmid has yet to be completed.
Fructose
To assemble this part we ordered two G-Blocks, one containing a lac pL promoter, a ribosome binding site, and MDH2, and another containing ribosome binding sites in front of HPS and PHI and a double terminator. The G-Blocks were designed so that they had overhangs that corresponded with each other as well as the ends of the backbone. We performed a gibson assembly to put all the pieces together and got our final product of the assembled plasmid.
Formate
To assemble this plasmid we ordered a G-Block with a lac pL promoter, a ribosome binding site and the MDH2 gene that had overhangs to be inserted into the chloramphenicol resistant backbone. We special ordered FDH and FALDH parts and attempted to use 3A to attach a ribosome binding site to each. When this step did not work for us, we decided to design primers that would have extensions to add in ribosome binding sites and would be able to anneal to create a plasmid with the two enzymes. We successfully performed a gibson after a PCR was run with the designed primers and parts, and did a 3A with this gibson product and the G-Block. Our final step was to do another 3A to attach a double terminator to the end.
After inspection of our created plasmids, the Fructose and the Formate plasmids, we noticed a few features of the plasmids that needed to be edited - Fructose had an extra EcoRI site and the FDH and FALDH genes did not contain stop codons. To alter our plasmids, we performed site directed mutagenesis using PCR with primers we designed.
An additional plasmid we attempted to create would be responsible for producing the protein folding chaperones GroES and GroEL which would assist in the particularly complicated folding mechanism of sMMO. Dr. George Lorimer kindly donated a plasmid already containing GroES and GroEL to us. We then sequenced this plasmid and identified the insert containing GroES and GroEL that we wanted to isolate. Overhang PCR was performed in order to create a PCR product containing GroES and GroEL flanked on either side by Gibson overhangs that matched up with either end of the standard iGEM chloramphenicol resistant backbone. The PCR product was then combined with the backbone via Gibson assembly. The resulting plasmid was then combined with the standard iGEM double terminator (BBa_B0015) through 3A assembly. Overhang PCR and another subsequent Gibson assembly was then used to attach a promoter and ribosome binding site to the GroESL construct. Sequencing of this plasmid, however, revealed several point mutations and deletions that would impact protein functionality. As such, this construct cannot be deemed ready for use at the moment.
References
Methane: The other important greenhouse gas. New York, New York.
Conservation Practices Minnesota Conservation Funding Guide.
Ismail, O. S.; Umukoro, G. E., Global Impact of Gas Flaring. Scientific Research 2012.
Jones, E., EPA Releases First-Ever Standards to Cut Methane Emissions from the Oil and Gas Sector. 2016.
Brock, T. D Methylococcus capsulatus is a methane-oxidising bacterium that has great potential in bioremediation Vol. 5 pp.9-33. June 1st 2002.
Kylie J. Walters, George T. Gassner, Stephen J, Lippard, Gerhard Wager. “Structure of 7. the soluble methane monooxygenase regulatory protein B” Vol. 96 pp. 7877-7882, July 1999
Martin Bender, Ralf Conrad. “Microbial Oxidation of Methane, Ammonium and Carbon Monoxide and Turnover of Nitrous Oxide and Nitric Oxide in Soils”Vol. 27. pp97-112, 1994
M., Irla. (2014, August 22). Complete genome sequence of Bacillus methanolicus MGA3, a thermotolerant amino acid producing methylotroph. Research Gate. Retrieved October 19, 2016.
Brautaset, T., Jakobsen, Ø.M., Josefsen, K.D. et al. Appl Microbiol Biotechnol (2007)
Meile, L.; Rohr, L. M.; Geissmann, T. A.; Herensperger, M.; Teuber, M., Characterization of the d-Xylulose 5-Phosphate/d-Fructose   6-Phosphate Phosphoketolase Gene (xfp) from Bifidobacterium lactis. Journal of Bacteriology 2001, 183(9), 29.
Results and Discussion
We verified that our constructs were assembled correctly through sequencing, which can be viewed in the pdf below.
https://static.igem.org/mediawiki/2016/3/34/T--UMaryland--sequences.pdf
Due to time constraints, we decided to focus most heavily on the testing of the fructose construct. To test protein expression, we ran an SDS-PAGE. Cells were grown at 37C to OD600=0.6-0.7 and induced with the appropriate concentration of IPTG. After induction, they were left to shake at 30C for six hours. They were then harvested, resuspended in 1mL PBS, and lysed with sonication. The supernatant and pellet of each lysate was applied to the appropriate lane.
Figure 1a: SDS-PAGE of fructose transformed DH5α cells. Culture was induced with (1,5) 0 mM IPTG (2,6) 0.2 mM IPTG (3,7) 0.1 mM IPTG and (4,8) 0.05 mM IPTG. Lanes 1-4 are the supernatant from cell lysate, and lanes 5-8 are the pallet from the cell lysate, diluted 1:20.
Figure 1b: SDS-PAGE of fructose transformed BL21 DE3 cells. Culture was induced with (1) 0 mM IPTG and (2) 0.5 mM IPTG at OD600~0.65. Lanes 1 and 2 are the supernate fractions of the sonicated cell lysate.
We did not see any overexpression of the desired proteins in DH5Alpha cells. For the BL21 cells, we did see the appearance of a ~45kDa band upon induction with IPTG. The MDH2 enzyme, the first protein in the fructose construct, has a size of 44.5 kDa. At also see the emergence of a ~100kDa band in lane 2. We predict that this is the T7 polymerase encoded by the DE3 strain of BL21. Although our construct has no use for the T7 polymerase for protein production, we used this cell line as it was made readily available to us from a collaborating lab. As an advantage, the expression of T7 polymerase, which depends on IPTG induction, also serves as an integrated positive control for induced expression.
We predict that the other two enzymes, HPS and PHI, were not seen in the expression assay because their relative expression levels were much lower than that of MDH2. As MDH2 was the first protein encoded in the polycistronic mRNA, it may be more likely that ribosomes bind to and translate the enzyme. The rbs that was used was also relatively weak. Future experiments would include swapping out ribosome binding sites, putting promoters in between each gene to have separate mRNA transcripts, all to determine the validity of our construct in terms of expression.
In Vitro Tests
To test the functionality of our fructose construct, we cultured cells in environments with methanol. To determine if methanol was a viable carbon source for our cultured bacteria, growth assays were done in M9 media where the carbon sources were controlled.
Figure 2: Growth rate in M9 media with controlled carbon sources. IPTG was added at hour 5.5, when the OD of all the cultures approximately reached 0.6, n=3 for each group.
Methanol was added at various concentrations to an M9 media containing 0.2% glucose. Half the traditional glucose concentration was chosen to put cells in less than ideal conditions, giving them incentive to pick up methanol as a carbon source once the plasmids were activated. Figure 2 shows that an increase of methanol from 1% to 2% impairs the growth rate of cells. A comparison with the same growth conditions against 0% methanol was also done. The results of methanol toxicity can be seen in Figure 1 of the supplemental data section. Aside from the impaired growth rate with increased methanol concentration, the OD appears to be higher from hours 6.5 to 9.5 for the cultures induced with IPTG. Although we do expect to see an increase in growth rate as the cells are now able to metabolize the methanol present in the media, this minor increase is not statistically significant. Further tests have to be done to validate the functionality of our plasmid.
In collaboration with the University of Maryland Bioprocess Scale-Up Facility , we were able to acquire access to the 10 liter bioreactor. This instrument allowed for us to control for all of the major factors that influence bacterial growth including temperature, gas-flow rate, gas-composition, and agitation.
The bioreactor used to test our constructs.
Two major tests were run on fructose in the bioreactor in order to test its functionality. The first test was designed to determine the growth rate of our fructose construct in 1.5% methanol. The experimental group had the construct induced with IPTG to stimulate protein production, while the control group had no induction, to serve as a comparison for the effects of the induced proteins.
The second test was to see if our fructose construct was capable of surviving the stress of induction, methanol exposure, and the atmospheric conditions associated with landfills. Typically, landfill gas composition is approximately 50% methane and 50% carbon dioxide with trace amounts of oxygen, nitrogen, and several other gases. Due to safety concerns and gas availability, we decided to simulate this landfill gas composition by using 99% nitrogen gas and 1% air. Since the fructose construct does not utilize methane nor carbon dioxide, this gas composition would effectively simulate the almost anaerobic environment of landfills.
For each test, one liter of LB broth was inoculated with 50 mL of starter culture of BL21-DE3 glycerol stock prepared the day before. Samples from the bioreactor were then taken on a regular basis and absorbance at lambda=600nm was taken and normalized to OD600. At OD600=0.6, the conditions within the reactor were changed to match the methanol content, gas composition, and IPTG induction levels outlined in the table below.
Bioreactor Run Number
Gas Input Composition
Amount of Methanol Added (mL)
Final Concentration of IPTG (mM)
1
100% air
15
0
2
100% air
15
0.5
3
99% Nitrogen 1% air
15
0.5
Following the variation in the tests, OD600 was measured on a regular basis. Figure 3 shows the optical density over time of the various reactor tests.
Figure 3. Growth Rate in Bioreactor
This data revealed that the fructose construct with IPTG induction grew at a faster rate in methanol than when uninduced. Meanwhile, the data gathered from the landfill simulation showed that the fructose construct was capable of growth, albeit slow growth, under these stressful conditions. In addition to testing growth rates, we also attempted to measure methanol concentration overtime via the use of a biochemistry analyzer kindly loaned to us by YSI. We experienced several inconsistencies on our second use of the machine, despite the reliability of the first run. We deemed our data for methanol concentration unreliable, but have included it in the supplemental data section. The sources of the measurement inconsistencies are unknown, but we hope to troubleshoot this problem or find another method of measuring methanol concentration for the future plans of this project.
Figure 4. Depiction of the growth rate of our fructose construct under landfill simulation conditions.
The culture was initially grown to an OD600 of 0.6 under aerobic conditions. Following this gas input composition was switched to 99% nitrogen and 1% air. One liter of gas per minute was pumped through the one liter of culture. The spike in OD600 seen in the middle of the graph accompanies a malfunction in the machinery in which greater than 1% of the gas being used was air. This data shows that the fructose construct was capable of growth, albeit slow growth, under these stressful conditions that mimic that of the intended site of implementation, a landfill piping system.
Supplemental Data
Supplemental Figure 1
Overnight cultures of the fructose construct grown in LB were used to inoculate M9 media with .2% glucose and 0% methanol (n=2), 1% methanol (n=6), and 2% methanol (n=6). At three time intervals before OD600 reached 0.6, the OD600 values were recorded. Data shows that as methanol concentration increases, the average OD of the culture does not increase as quickly over time. After 5.5 hours, some cells were induced with IPTG to stimulate protein production (Figure 3). The continued trend of methanol inhibiting cell growth can still be seen in Figure 3.
Supplemental Figure 2
To measure our methanol concentrations we used a YSI2900. This biochemistry analyzer is able to use a membrane containing alcohol oxidase that turns methanol into hydrogen peroxide. The hydrogen peroxide is then electrochemically oxidized at a platinum anode producing a signal current that can then be measured. To use the machine, you must first calibrate it with a sample of methanol at a concentration of 1.00 g/L.
To get our measurements, we took samples from the bioreactor every hour after methanol was added. We transferred the sample into three 2mL microcentrifuge tubes, all filled with 2mL of the sample, and then spun them at 8000G for 5 minutes. After the samples were spun down, we transferred 200µL of the supernatant into a microcentrifuge tube with 1800µL of water to create a 1:10 dilution for each original tube. We took measurements of each diluted sample and recorded the results.
Calibrate machine with 1.00 g/L standard provided.
Aliquot 2 mL of sample from the bioreactor into each of three microcentrifuge tubes.
Spin down tubes for 5 minutes at 8000 g.
Dilute each tube by transferring 200µL of the supernatant to 1800 µL of water in a new microcentrifuge tube.
Measure the methanol concentration using the YSI machine (in g/L).
On the first bioreactor run, the concentration of methanol decreased over time, and the readings were accurate. Methanol is a volatile liquid, so we expect the concentration to decrease until it reaches its equilibrium with the vapor state, which can be seen on the graph. On the second bioreactor run, however, the measurements were widely inconsistent. Machine calibration, measurement techniques, or a combination of various other factors could have caused the error, but sources are undetermined. We do see the methanol concentration reaching a lower equilibrium, but with the lack of precision and multiple trials, this data set is ultimately inconclusive.
Conclusions and Future Plans
Conclusion:
The data gathered from our various tests suggest that our fructose construct could be functional. Unfortunately, due to our inability to properly measure methanol concentration over time, we cannot definitively prove that methanol is being degraded. However, since cells with our construct post induction grew faster under methanol exposure than their uninduced counterparts, there is a high possibility that our transformed cells are metabolizing methanol and using it as an energy source. This would explain why induction results in an increased growth rate in the presence of methanol. Additionally, the simulated landfill gas scenario shows that our fructose construct is viable enough to survive in the conditions associated with a landfill environment. When compiled together, all of this data suggests that our fructose construct could be implemented into a landfill environment along with a construct expressing sMMO in order to assist in the metabolization of methane to carbon dioxide.
Future plans
There are a lot of potential avenues for this project in future years. First and foremost on our list of future plans is to complete all of the plasmids we set out to construct at the beginning of the summer. This would include finishing the sMMO plasmid as well as touching up the GroESL plasmid through several mutagenesis reactions. Upon the completion of all desired constructs we would run another set of even more rigorous tests on each individual construct. Specifically, we plan to definitively prove expression of all enzymes in each pathway with SDS-PAGE. This time around, we may plan on using constructs with proteins with affinity tags for purification. Running purified gels will increase the likelihood and evidence that our constructs are being expressed. Purification also allows us to perform functional assays with purified cell extract. More functional assays are planned to further prove that our constructs will work as expected.  For the formate construct we would run similar tests to those done on the fructose plasmid. For the sMMO construct we would plan to test for methanol production in the bioreactor when methane is included among the gases being pumped into the culture.
After completing several repetitions of these basic functionality tests, we plan to run several coculture experiments in the bioreactor. Here, a green fluorescent protein expressing variant of the sMMO construct would be grown in the bioreactor alongside a red fluorescent protein expressing variant of the fructose or formate construct. Growth rates of the constructs would be recorded and methanol levels upon the introduction of methane would be recorded. The hope is that the fructose/formate construct will degrade any methanol produced by the sMMO construct through its oxidation of methane.
Alongside our coculture experiments we would consider combining the sMMO plasmid with our fructose or formate plasmid. By doing so, we would have our entire system located in a single, large plasmid of about 10 to 11 thousand base pairs. Testing of this construct would be somewhat simpler as the ratios between the different enzymes would be easier control. The plasmid combination and coculture experiments would be supplemented by our mathematical modeling. Our model informs us of the concentrations of enzymes needed to sustain the metabolic reactions for a prolonged duration. As such, tuning the strength of ribosome binding sites or promoters as indicated by our modeling is critical in this step of integrating the plasmids together.
Upon the completion of these aspects of our project, we would need to decide on an exact method of implementation. As the major goal of our project is addressing methane build up in landfills, we are currently planning to incorporate our final enzymatic system including sMMO and the enzymes necessary for the subsequent degradation of methanol into custom made bioreactor. In our human practice visit we learned that landfills already have a pump and piping system to collect the gaseous emissions. Integrating a bioreactor into a piping system would thus not be so difficult. This bioreactor would be designed to uptake landfill gas under the ideal conditions to facilitate the degradation of methane by our final bacterial construct(s). Based on the testing we have done so far, this would entail the need for some of the same conditions used in previous bioreactor tests including a controlled temperature and the ability to agitate/shake cells for proper aeration.
Unlike the simulation of a landfill gas environment experiment, we would plan to incorporate a fair amount of oxygen into the gas input of our system for two reasons. First and foremost, oxygen is necessary to facilitate faster growth rates in our constructs. Additionally, oxygen is required for the oxidation of methane to methanol. As such, we plan to have a gas input into our system of about 20-50% oxygen and 50-80% landfill gas. The ideal oxygen to landfill gas ratio will need to be determined through testing of system efficiency.
The final integrated reactor system would involve the landfill pipe system bubbling a mixture of landfill gas and O2 through a liquid culture of cells. The cells will be maintained in continuous flow liquid culture, similar to industry standards of large scale protein production. The continuous flow culture would constantly be resupplied with fresh media containing necessary nutrients, likely standard tryptone powder and yeast extract, and possibly recycled biomass from the culture itself. Cells would be removed from the culture to maintain a stable culture OD, as well as to harvest the biomass for energy or to feed back into the culture system. Overall, implementation and maintenance of this system will definitely be costly, but the energy returns will help to offset the cost, and the product of almost entirely removing methane emissions from landfills is enticing enough to pursue this project.
The figure below details our proposed landfill bioreactor set up:
Contact Us
© University of Maryland iGEM 2016
