<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UMaryland_notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UMaryland/notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;/div&gt;&lt;/div&gt;<TITLE>About Our Team</TITLE><NAV><UL><A style="opacity: 0">Fix</A><LI><A href="https://2016.igem.org/Team:UMaryland/projects">Projects</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/Part_Collection">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/about">About Us</A></LI></UL></NAV></P><DIV id="div-navResized"><UL><LI><A href="https://2016.igem.org/Team:UMaryland/projects">Projects</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/Part_Collection">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:UMaryland/about">About Us</A></LI></UL></DIV><DIV data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2016/a/ae/T--UMaryland--notebook.jpg" class="div-scrollPic" style="height: 550px;"><DIV class="titleText"><H11>UMaryland iGEM Notebook</H11><H21>Recording Our iGEM Experience</H21></DIV></DIV><DIV id="div-text"><DIV id="div-content"><TABLE id="table-selector"><THEAD><TR><TH class="borderRight navigator" id="th-journal" data-select="journal"><P>Journal</P></TH><TH class="borderBoth navigator" id="th-protocol" data-select="protocol"><P>Protocol</P></TH><TH class="borderLeft navigator" id="th-safety" data-select="safety"><P>Safety</P></TH></TR></THEAD></TABLE><DIV id="div-journal" class="sectionClass"><DIV class="contentText blog"><DIV class="textBody"><P class="h3-heading" id="Week1">Week 1</P><EM class="date">June 6th - June 10th</EM><P class="p-text">Our first step this summer was to finalize the design of our three plasmids (sMMO, Fructose, and Formate). Many of the genes needed to create our plasmids are offered in the iGEM registry, but we realized that we would need to order some genes as GBlocks from IDT. We began to design GBlocks for the genes not offered in the registry (MMOY &amp; MMOZ subunits, MDH2, HPS, PHI), which was a new experience for most of our lab members, who had little experience creating parts for Gibson Assembly.</P><P class="p-text">This summer, we moved into a new lab space, so we had to set up the lab with all our equipment. Once that was completed, we were able to train our very young team in the basic skills of genetic engineering, like using micropipettes and performing transformations.</P><P class="tags">Week1, week, 1, wet, lab, set, up, lb, idt, gblocks, mmoy, mmoz, mdh2, hps, phi, smmo, fructose, formate, june, 6th, 10th, 6/6, 6/10, 6/7, 6/8, 6/9</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week2">Week 2</P><EM class="date">June 13th - June 17th</EM><P class="p-text">We received the iGEM kit this week! After opening the kit, we all cloned the biobricks we had requested (MMO subunits B,D,C,X; FDH; FALDH) to create stocks of them. After cloning those genes, we grew overnight cultures that we miniprepped to isolate the plasmids with the biobricks.</P><P class="p-text">During dry lab, we revised the design for our GBlocks (MMOY &amp; MMOZ subunits, MDH2, HPS, PHI), to include restriction sites, and promoters and terminators where needed. We also scavenged the building for old lab equipment that other scientists wanted to dispose. In addition to scavenging for lab equipment, we scavenged for DNA and asked team Braunshweig for a copy of the sMMO plasmid they constructed in 2015.</P><P class="tags">Week2, week, 2, wet, lab, dry, lab, gblocks, mmoy, mmoz, mdh2, hps, phi, restriction, sites, re, promoters, terminators, braunshweig, smmo, plasmid, clone, overight, miniprep, fdh, faldh, june, 13th, 17th, 6/13, 6/14, 6/15, 6/16, 6/17</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week3">Week 3</P><EM class="date">June 20th - June 24th</EM><P class="p-text">We started assembling parts together to reach our final goals of creating plasmids with their respective promoters, ribosome binding sites, enzymes, and terminators. We used the technique of 3A assembly - the digestion and ligation of an antibiotic resistant backbone with two inserts - to attach ribosome binding sites to each of the biobricks we requested.</P><P class="p-text">We then transformed our competent cells with the assembled plasmids (after which they grew successfully on our antibiotic plates) and grew them overnight to then extract the plasmid DNA by doing a miniprep. We also contacted a professor on campus who works with methane to ask her about her safety protocol when handling the gas. </P><P class="tags">Week3, week, 3, promoter, promoters, rbs, ribosome, binding, sites, terminator, terminators, 3A, 3A assembly, ligation, digestion, biobricks, transformation, transform, methane, professor, miniprep, june, 20th, 24th, 6/20, 6/21, 6/22, 6/23, 6/24</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week4">Week 4</P><EM class="date">June 27 - July 1st</EM><P class="p-text">After digesting our minipreps, we screened our resulting plasmids using a gel. Our results from the gel matched up with our expectations, so we continued our process of assembling the plasmids using 3A. This time, we attempted to combine the RBS and FDH plasmid with the RBS and FALDH, RBS and MMOB with RBS and MMOD, and RBS and MMOC with RBS and MMOX. Additionally, we finalized our g blocks and met with several university employees to discuss funding.</P><P class="tags">Week4, week, 4, miniprep, gel, 3A, rbs, ribosome, binding, sites, FALDH, MMOB, MMOD, MMOC, MMOX, g, blocks, GBlocks, funding, june, 27th, july, 1st, 6/27, 6/28, 6/29, 6/30, 7/1, 7/01</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week5">Week 5</P><EM class="date">July 4th - July 8th</EM><P class="p-text">
                            We spent the week continuing the 3A assembly we had started earlier. Because our supply of backbone was very limited, we digested one of the igem parts, ran a gel, and extracted the backbone to use for our assembly. After transforming competent cells with the ligations of the two parts and backbone, we, unfortunately, observed no growth on our plates.
                        </P><P class="tags">week, 5, july, 4th, 8th, 7/4, 7/5, 7/6, 7/7, 7/8, 3A, assembly, backgone, extraction, gel, extracted, ligation, plates, plating</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week6">Week 6</P><EM class="date">July 11th - July 15th</EM><P class="p-text">
                            Because our 3A assemblies from the prior week failed, we attempted to digest, ligate, and transform everything again. Though we did get growth on plates, a restriction enzyme digestion and gel to screen our resulting plasmids did not provide the results we were expecting. Additionally, Team Braunschweig’s plasmid had arrived, so we attempted to recover their plasmid of sMMO.
                        </P><P class="tags">week, 6, july, 11th, 15h, 7/11, 7/12, 7/13, 7/14, 7/15, 3A, assembly, digest, ligate, transformation, transform, gel, team, braunschweig, plasmid, smmo</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week7">Week 7</P><EM class="date">July 18th - July 22th</EM><P class="p-text">
                            We transformed E. coli with the plasmid that was extracted, but somehow had growth on both chloramphenicol and ampicillin resistant plates; the suspected backbone only had chloramphenicol resistance. When running a gel of a restricted digest of the plasmid, we observed bands that were not expected. Additionally, we tried sequencing all our DNA recovered over the past few weeks and were disappointed to see only a few of our samples matched the expected sequences (RBS with MMOX and RBS with FDH). While our cloning was proving to have its challenges, we successfully held our Second Annual Mid Atlantic Meetup.
                        </P><P class="tags">week, 7, july, 18th, 22th, 7/18, 7/19, 7/20, 7/21, 7/22, e, coli, chlor, amp, chloramphenicol, ampicillin, backgone, gel, restrition, digest, sequencing, rbs, mmox, fdh, mid, atlantic, meetup, second, annual</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week8">Week 8</P><EM class="date">July 25th - July 29th</EM><P class="p-text">
                            Due to time limitations and difficulty with traditional 3A assemblies, we decided to take a different approach on cloning. Instead of sequentially digesting and ligating, we chose to start Gibson assemblies. We designed primers for each part, the forward having a gibson overhang, ribosome binding site, and annealing sequence and the reverse containing just an annealing sequence. Our G-blocks also came in, which were designed to be ready for a Gibson assembly with a backbone. We created stocks by transforming our resulting Gibson assemblies. We also had our first experience with PCR, as we wanted to try an alternative way to extract backbone. At the end of the week, we volunteered with Building with Biology in Port Discovery and explained the significance and applications of synthetic biology.
                        </P><P class="tags">week, 8, july, 25th, 29th, 3A, assembly, gibson, overhang, ribosome, binding, site, rbs, annealing,reverse, g, blocks, g-blocks, pcr, backbone, port, discovery, biology, building, with, synthetic, biology</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week9">Week 9</P><EM class="date">August 1st - August 5th</EM><P class="p-text">
                            We ordered primers to construct our formate and fructose plasmids via Gibson assembly. Sequencing results showed that our fructose plasmid was successfully constructed and our formate plasmid has RBS with FALDH and RBS with FDH. To continue working on the formate plasmid, we began a 3A assembly with a promoter, RBS, and MDH2 (an ordered g block that was assembled into a chloramphenicol resistant backbone.
                        </P><P class="tags">week, 9, august, 1st, gibsone, formate, fructose, plasmid, rbs, faldh, fdh, ribisome, binding, sites, 3A, assembly, mhd2, g, block, blocks, g-blocks, chloramphenicol, chlor</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week10">Week 10</P><EM class="date">August 8th - August 12th </EM><P class="p-text">
                            We finished our 3A assembly and found promising results from sequences. Therefore, we moved on to adding the final piece to the formate plasmid - a double terminator. We also designed and ordered primers to create a Gibson assembly of a promoter and four subunits of the sMMO enzymes.
                        </P><P class="tags">week, 10, 3A, sequence, assembly, smmo, double, terminator, gibson, sub, unit, subunit, sequence, enzyme</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week11">Week 11</P><EM class="date">August 15th - August 19th </EM>We completed the interlab study to contribute data to standardizing fluorescence measurements across various laboratories. It will be a step forward in collaboration in synthetic biology. We also selected our track for the competition and submitted our title and abstract. We plan to test two of our constructs in a plating experiment with induction by IPTG to see out it affects growth. Then we intend to perform SDS PAGE to see if there are noticeable bands for our protein size
                        <P class="tags">week, 15th, 19th, 15, 19, track, title, abstract, interlab, il, measurements, SDS, PAGE, IPTG, plating, fructose, formate</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week12">Week 12</P><EM class="date">August 22th - August 26th </EM><P class="p-text">Our fructose and formate pathway plasmids were completed and sequenced and found the presence of the double terminator. Our growth test proved successful as many colonies were visible. We also distributed pamphlets about methane and interviewed farmers at the UMD Farmer’s Market and the Montgomery Agricultural Fair to gauge their knowledge and interest in methane, especially with regards to livestock.
                        </P><P class="tags">week, 22th, 26th, 22, 26, fructose, formate, plasmid, sequence, double, terminator, pamphlets, methane, farmers, farmer, Farmer's, Market, UMD, Montgomery, Agricultural, Fair, livestock</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week13">Week 13</P><EM class="date">August 29th - September 2nd </EM><P class="p-text">We ran SDS-PAGE on extracts from our fructose and formate pathway constructs along with positive and negative controls. We also discovered that FDH and FALDH genes were missing stop codons in the registry, which may have prevented us from seeing the bands on the gel. We are also continuing construction of the sMMO using the Gibson Assembly method using overhangs that were added using PCR. We also visited the Onley Farmer’s Market to continue to gather information about methane from farmers. 
                        </P><P class="tags">week, 29th, august, september, 2nd, SDS, PAGE, fructose, formate, constructs, positive, negative, controls, FDH, FALDH, codon, stop, sMMO, gibson, pcr, onley, farmer, farmer's, market, methane</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week14-Week15">Week 14 - Week 15</P><EM class="date">September 5th - September 16th</EM><P class="p-text">We re-ran the SDS-PAGE gel on our fructose and formate pathway constructs to detect overexpression of protein, we did not see clear overexpression and will perform site directed mutagenesis to add the missing stop codons. We also tested a growth rate curve of the BL21 strain of E. coli that we are using to test our constructs to practice for the bioreactor. 
                        </P><P class="tags">week, 14, 15, september, 5, 16, sds, page, fructose, formate,bl21, growth, rate, curve, bioreactor, site, directed, mutagenesis</P></DIV><DIV class="textBody"><P class="h3-heading" id="Week16-Week17">Week 16 - Week 17</P><EM class="date">September 16th - September 30th</EM><P class="p-text">Site directed mutagenesis was completed successfully on our constructs and a growth rate curve of bacteria transformed with the fructose plasmid constructs was gathered. The growth rate was favorable and ensured that the plasmid did not end of cell death. We also re-did the SDS-PAGE but still did not see a noticeable band where MDH2 or other proteins are located.
                        </P><P class="tags">week, 16, 17, september, 16, 30, site, directed, mutagenesis, construct, fructose, sds, page, mdh2</P></DIV></DIV></DIV><DIV id="div-protocol" class="sectionClass"><DIV class="contentText procedure"><DIV class="textBody"><P class="h3-heading" id="MakingLBBroth">Making LB Broth</P><P class="p-text">Note: Typical procedure calls for 25 g LB Broth powder per 1L of pure water. We have 500 mL bottles and we do not want to overfill them so we will do a quarter of that amount. This also helps in that if one bottle gets contaminated, then only a quarter of the LB broth batch will need to be discarded.</P><P class="tags">LB, broth, food, making, media</P><P class="p-text"><OL><LI>Mass 6.25 g of LB powder 4 times</LI><LI>Add to 4 bottles</LI><LI>Add 250 mL pure water in each</LI><LI>Swirl to mostly dissolve</LI><LI>Seal cap tightly and then unscrew the cap about two turns</LI><LI>Cover with aluminum foil</LI><LI>Place autoclave tape on at least one bottle</LI><LI>Place bottles in a basket</LI><LI>Autoclave for 20 minutes on liquid setting</LI><LI>Do not add antibiotics to pure broth. Only add to LB used to plate</LI></OL></P></DIV><DIV class="textBody"><P class="h3-heading" id="MakingLBPlates">Making LB Plates</P><P class="p-text"><OL><LI>Measure out 25 g of LB broth powder for every one litre of plating material desired. (divide as necessary; we typically do this in quarters)</LI><LI>Place in desired container</LI><LI>Add 15 g of bacto agar for every litre of plating material desired (divide as necessary)</LI><LI>Add 1 L of pure water to container (divide as necessary)</LI><LI>Swirl until mostly dissolved</LI><LI>Seal cap tightly and then unscrew the cap about two turns</LI><LI>Cover caps with aluminum foil</LI><LI>Add at least one piece of autoclave tape</LI><LI>Autoclave for 20 minutes on liquid setting. Do not set exhaust time. Ensure door is sealed and liquid media is selected</LI><LI>Allow to cool to about 50 degrees Celsius. Cool until it is warm to touch. Do not forget to add corresponding antibiotic</LI></OL></P><P class="p-text"><STRONG>Adding Antibiotics</STRONG></P><P class="p-text">
                            Note:Chloramphenicol is kept in 1000x stock Ampicillin is kept in 500x stock</P><P class="p-text"><OL><LI>Thaw antibiotic</LI><LI>Add 250 uL CMP to 250mL LB/agar or 500 uL AMP to 250mL LB/agar</LI></OL></P><P class="tags">plates, lb, food, making, chlor, chloramphenicol, amp, ampicillin, antibiotic</P></DIV><DIV class="textBody"><P class="h3-heading" id="RestrictionDigestandLigation(3AAssembly)">Restriction Digest and Ligation (3A Assembly)</P><P class="p-text">
                            Protocol modified from this source:<A href="https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf">https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf</A></P><P class="p-text"><OL><LI>Acquire all necessary information including nanodrop concentrations of any miniprepped samples as well as the concentration of the linearized plasmid backbone. Calculate the volume of miniprepped DNA necessary to total approximately 500 ng. Designate the DNA strands that are to be ligated together as either A or B.</LI><LI>Create the three master mixes. One will contain EcoRI and PstI, another will contain EcoRI and SpeI, and the last will contain XbaI and PstI. Combine with water and cutsmart buffer in the ratios directed below. The tables direct for 25 µL to be the final volume. This is enough for 5 digestions. Amounts of CutSmart buffer and water can be adjusted as necessary. Enzyme concentration should only be adjusted in the case of an increase in final volume due to micropipette limitations.</LI><LI>Prepare linear backbone for digest by measuring out 4 µL (100 ng) of the backbone of choice and 4 µL of the master mix for the linearized plasmid backbone for each ligation planned for the next step into one PCR tube. Mix well with a pipette. Total volume 8uL.</LI><LI>Prepare sample A’s by adding 500 ng worth of DNA to a labeled PCR tube. Supplement with water so that the total volume added is 5 µL. Add 5 µL of master mix A to each tube. Total volume 10 uL.</LI><LI>Prepare sample B’s by adding 500 ng worth of DNA to a labeled PCR tube. Supplement with water so that the total volume added is 5 µL. Add 5 µL of master mix B to each tube. Total volume 10 uL.</LI><LI>Place all samples in the incubator (set at 37 degrees Celsius) for at least one hour. Longer durations are acceptable. They can be left overnight.</LI><LI>Heat kill restriction enzymes at 80 degrees celsius in the the thermocycler (or another apparatus) for 20 minutes.</LI></OL></P><P class="p-text"><STRONG>Ligation</STRONG></P><P class="p-text"><OL><LI>Add 2 µL of backbone, 2 µL of enzyme A, and 2 µL of enzyme B to a labeled PCR tube.</LI><LI>Add 1 uL T4 DNA ligase buffer Add 2 uL ddH2O</LI><LI>Add T4 DNA ligase, total volume 10 uL</LI><LI>Allow to sit at room temperature for at least one hour. It can be left overnight at room temperature.</LI></OL></P><P class="tags">restriction, digest, and, ligation, (3A, Assembly), 3A, Assembly, master, mix, EcoRi, PstI, XbaI, heat, kill, pcr, incubator, thermocycler, dna, buffer, ligase, T4</P></DIV><DIV class="textBody"><P class="h3-heading" id="Transformation">Transformation</P><P class="p-text"><OL><LI>Set water bath to 42 C</LI><LI>Make sure SOC media is not contaminated. LB is also usable</LI><LI>Label test tubes and microcentrifuge tubes</LI><LI>Obtain competent cells. KEEP ON ICE</LI><LI>Thaw ON ICE until cells are thawed</LI><LI>If necessary, create 20 uL aliquots of competent cells in microcentrifuge tubes, keep on ice and return to the -80 as soon as possible</LI><LI>Add 2 uL of DNA to 20 uL of competent cells in a microcentrifuge tube</LI><LI>Pipette to resuspend, but do so gently so as to not harm the cells</LI><LI>Place back on ice for 5 minutes</LI><LI>Return any unused cells marking date thawed ON ICE</LI><LI>Heat shock at 42 C for 30 - 45s</LI><LI>Remove from heat then place in ice for 5 mins</LI><LI>Add 200 uL of SOC/rescue media. FLAME BOTTLE AND BOTTLE CAP THOROUGHLY </LI><LI>Rescue for one hour in incubator with shaker</LI><LI>Plate cells on plates with antibiotics overnight at 37 C</LI></OL></P><P class="p-text">
                            Optional-plate 50uL onto plate with differing antibiotic resistance for control
                        </P><P class="tags">transformation, water, bath, soc, rescue, media,ice,competent, cells, incubator, antibiotic, plate</P></DIV><DIV class="textBody"><P class="h3-heading" id="PreparingOvernights">Preparing Overnights</P><P class="p-text">
                            Note: Sterilize test tubes, lids, and micropipette tips before using, and perform procedure close to flame.
                        </P><P class="p-text"><OL><LI>Gather 5 mL culture tubes and label</LI><LI>Ignite bunsen burner</LI><LI>Remove plates from incubator (if not already done so)</LI><LI>Add 5 mL of LB Broth to test tubes. FLAME BOTTLE AND BOTTLE CAP THOROUGHLY </LI><LI>Add of (10 uL) ampicillin or (5 uL) chloramphenicol. (Change amount if not from stock solutions of 500X and 1000X)</LI><LI>Use a micropipette to obtain one colony from each plate then release colony in the media</LI><LI>Eject pipette tip in the test tube as well and close</LI><LI>FLAME CULTURE TUBE AND CAP</LI><LI>Place in shaker/incubator overnight</LI></OL></P><P class="tags">preparing, overnights, plasmid, cloning, amp, ampicillin, chlor, chloramphenicol, shaker, incubator, bunsen, burner, lb, broth, media</P></DIV><DIV class="textBody"><P class="h3-heading" id="Miniprep">Miniprep</P><P class="p-text"><OL><LI>Obtain overnight cultures</LI><LI>Label microcentrifuge tubes</LI><LI>Use pipette to transfer 2 x 0.63mL of culture to microcentrifuge tubes, leaving pipette tips inside the respective tubes for future use</LI><LI>Try to avoid the pipette tips already inside</LI><LI>Centrifuge for 5 minutes at 13000 g</LI><LI>Dispose of supernatant, being careful not to lose cells (tap sides gently but do not flick)</LI><LI>Repeat steps 3, 4, and 5 until all of the culture has been centrifuged</LI><LI>Add 250uL of P1 Buffer and mix with pipette</LI><LI>Add 250uL of P2 Buffer</LI><LI>You can use the same pipette tips as long as you don’t touch the inside of the pipette</LI><LI>Invert at least 4-6 times. This is the important step for DNA</LI><LI>Add 350 uL of P3 and invert 4-6 more times</LI><LI>Centrifuge for 10 minutes at highest setting (13500 x g; protocol calls for 17900 g, but centrifuge in use has max of 13500 g)</LI><LI>Remove all supernatant and place in QIAprep 2.0 spin column via pipette</LI><LI>Centrifuge for 30-60 seconds. Discard flow through</LI><LI>Wash spin column with 0.5 mL Buffer PB</LI><LI>Centrifuge for 30-60 seconds. Discard flow through</LI><LI>Wash spin column with 0.75 mL Buffer PE</LI><LI>Centrifuge for 30-60 seconds. Discard flow through</LI><LI>Centrifuge for one more minute and discard flow through</LI><LI>Place QIAprep 2.0 column in a clean 1.5 mL microcentrifuge tube</LI><LI>Elute DNA a with 25 uL pure water or Buffer EB</LI><LI>Set centrifuge to 7500 x g to keep caps from falling off</LI><LI>Centrifuge for one minute after letting sit for one minute. DO NOT DISCARD FLOW THROUGH</LI><LI>Repeat steps 20-22</LI><LI>Dispose of spin column</LI><LI>Take microcentrifuge tube and test for concentration in a nanodrop</LI></OL></P><P class="tags">miniprep, overnight, plasmid, centrifuge, p1, p2, buffer, p3, pe, pb, spin, column, QIAprep, 2.0</P></DIV><DIV class="textBody"><P class="h3-heading" id="GibsonAssemblywithGblocks">Gibson Assembly with G blocks</P><P class="p-text"><STRONG>Standard</STRONG></P><P class="p-text"><OL><LI>Acquire materials including NEB kit, G blocks, and linearized backbone</LI><LI>G blocks should be spun down in the centrifuge for 3-5 seconds at 3,000 g to reform pellets that may have been affected by shipping</LI><LI>Resuspend G blocks in water. To make each the same concentration of 50 ng/μl, 20 μl of TE Buffer should be added to the pelleted g block fragments. (adjust if starting amount is not 1000 ng)</LI><LI>If backbone is in solid form this should be resuspended as well and the resultant concentration should be noted</LI><LI>Measure out 10 μl of 2X Gibson Assembly Master Mix into container of choice.
                                    </LI><LI>Add in 100 ng of plasmid to the tube. About 2-3 times molar excess of g block should then be added. (volume needs to be calculated)</LI><LI>Pure water should then be added to reach a total volume of 20 μL</LI><LI>Incubate the container at 50 degrees Celsius for one hour</LI><LI>The plasmid is now ready for transformation and sequencing</LI></OL></P><P class="p-text"><STRONG>Combining Mdh2, HPS, and PHI Using NEB Gibson Kit</STRONG></P><P class="p-text"><OL><LI>Acquire materials including NEB kit, G blocks, and linearized pSB1A3 backbone</LI><LI>G blocks should be spun down in the centrifuge for 3-5 seconds at 3,000 g to reform pellets that may have been affected by shipping</LI><LI>Resuspend G blocks in water. To make each the same concentration of 50 ng/μl, 20 μl of TE Buffer should be added to the pelleted g block fragments</LI><LI>If backbone is in solid form this should be resuspended as well and the resultant concentration should be noted</LI><LI>Measure out 10 μl of 2X Gibson Assembly Master Mix into container of choice</LI><LI>Add in 100 ng (.07 pmol) of plasmid to the tube. About 2-3 times excess of g blocks should then be added. .2 pmol of both Mdh2 and HPS/PHI will be added. This equates to about .18 μg each or 3.6 μL of each solution. (The two fragments differ only by 70 bp in length. As such the difference in the number of moles is marginal here)</LI><LI>Pure water should then be added to reach a total volume of 20 μL. This will have to be adjusted for at the time of the procedure as the volume added in by the plasmid is not known yet</LI><LI>Incubate the container at 50 degrees Celsius for one hour</LI><LI>The plasmid is now ready for transformation and sequencing</LI></OL></P><P class="tags">gibson, assembly, with, g, blocks, neb, kit, new, england, bioscience, backbone, pellet, te buffer, mdh2, hps, phi, master, mix</P></DIV><DIV class="textBody"><P class="h3-heading" id="GelElectrophoresis">Gel Electrophoresis</P><P class="p-text"><STRONG>Gel Casting</STRONG></P><P class="p-text"><OL><LI>Measure out 0.75 g agarose for 1.5% gel, 0.5g for 1% gel</LI><LI>Add agarose to 50 mL SYBRsafe in .5x TBE in erlenmeyer flask</LI><LI>Microwave in 30 s intervals until agarose is dissolved, making sure to cover flask with paper towel</LI><LI>Pour into gel mold, add comb and let cool 20 min</LI><LI>Take gel out of mold and put into gel chamber</LI></OL></P><P class="p-text"><STRONG>Gel Electrophoresis</STRONG></P><P class="p-text"><OL><LI>Cover gel in 1x LB buffer</LI><LI>Add 5 uL of ladder to far left well</LI><LI>Add 1 uL dye to 5 uL DNA, then load 5uL into each well</LI><LI>Run gel at 150 V for 1 hr, until loading dye is about 75% the way to the end</LI></OL></P><P class="tags">gel, electrophoresis, casting, agarose, sybrsafe, tbe, chamber, comb,mold, ladder, V, volts, wells, buffer</P></DIV><DIV class="textBody"><P class="h3-heading" id="GelPurification">Gel Purification</P><P class="p-text">
                            Note: Taken from Zymo Research. Meant for use with ZymoClean kit. <A href="http://www.zymoresearch.com/downloads/dl/file/id/34/d4001i.pdf">http://www.zymoresearch.com/downloads/dl/file/id/34/d4001i.pdf</A></P><P class="p-text"><OL><LI>Excise the DNA fragment<SUP>1</SUP> from the agarose gel using a razor blade, scalpel or other device and transfer it into a 1.5 ml microcentrifuge tube</LI><LI>Add 3 volumes of ADB to each volume of agarose excised from the gel (e.g. for 100 µl (mg) of agarose gel slice add 300 µl of ADB)</LI><LI>Incubate at 37-55 °C for 5-10 minutes until the gel slice is completely dissolved<SUP>2</SUP></LI><LI>For DNA fragments &gt; 8 kb, following the incubation step, add one additional volume (equal to that of the gel slice) of water to the mixture for better DNA recovery (e.g., 100 µl agarose, 300 µl ADB, and 100 µl water)</LI><LI>Transfer the melted agarose solution to a Zymo-Spin Column in a Collection Tube</LI><LI>Centrifuge for 30-60 seconds. Discard the flow-through<SUP>3</SUP></LI><LI>Add 200 µl of DNA Wash Buffer to the column and centrifuge for 30 seconds. Discard the flow-through. Repeat the wash step</LI><LI>Add ≥ 6 µl DNA Elution Buffer<SUP>4</SUP> or water<SUP>5</SUP> directly to the column matrix. Place column into a 1.5 ml tube and centrifuge for 30-60 seconds to elute DNA</LI></OL></P><P class="p-text"><SMALL><SUP>1</SUP> The amount of agarose excised from the gel should be as small as possible.<SUP>2</SUP> Do not incubate above 60°C. It is important that the gel slice dissolve completely. This can be facilitated by gentle mixing during the incubation.<SUP>3</SUP> Remove the flow-through by aspiration. Avoid contamination of the collection tube rim.<SUP>4</SUP> DNA Elution Buffer: 10 mM Tris-HCl, pH 8.5, 0.1 mM EDTA.<SUP>5</SUP> Elution of DNA from the column is dependent on pH and temperature. If water is used, make sure the pH is &gt;6.0. Waiting 1 minute prior to elution may improve the yield of larger (&gt; 6 kb) DNA. For even larger DNA (&gt; 10 kb), the total yield may be improved by eluting the DNA with 60-70 C DNA Elution Buffer</SMALL></P><P class="tags">gel, purification, zymo, zymoclean, kit, excise, gel, slice, agarose, adb</P></DIV><DIV class="textBody"><P class="h3-heading" id="DNAPrecipitationfromFilterPaper">DNA Precipitation from Filter Paper</P><P class="p-text"><OL><LI>To recover the DNA, cut the marked circle area that contains the dried plasmid DNA</LI><LI>Using clean forceps, insert the filter paper into a 1.5 ml microcentrifuge tube. Add 100 µl of TE buffer or Ultrapure water, vortex briefly and incubate at room temperature for 5 minutes. Vortex again and briefly centrifuge</LI><LI>Add 20 µL of 3M sodium acetate (final conc. 0.3M) and mix well</LI><LI>Add 300 µL of cold 100% ethanol from -20 C (final conc. 70% EtOH), mix, and centrifuge on high for 10 min</LI><LI>Pour out supernatant and wash pellet with 1 mL 70% EtOH</LI><LI>Remove EtOH after washing with pipette</LI><LI>Dry under vacuum for 5 min</LI></OL></P><P class="p-text">
                            (revised for Braunschweig’s filter paper, which was too large and the conc of DNA was too high to use the procedure above)
                        </P><P class="p-text"><OL><LI>To recover the DNA, cut the marked circle area that contains the dried plasmid DNA</LI><LI>Using clean forceps, insert the filter paper into a 1.5 ml microcentrifuge tube. Add 750 µl of TE buffer or Ultrapure water, vortex briefly and incubate at room temperature for 5 minutes. Vortex again and briefly centrifuge</LI><LI>Split into two 1.5 ml microcentrifuge tubes, leaving filter paper in original</LI><LI>Add 75 µL of 3M sodium acetate (final conc. 0.3M) to each tube and mix well</LI><LI>Add 1125 µL of cold 100% ethanol from -20 C (final conc. 70% EtOH) to each tube, mix, and centrifuge on high for 10 min</LI><LI>Pour out supernatant and wash pellet with 7.5 mL 70% EtOH</LI><LI>Remove EtOH after washing with pipette</LI><LI>Air dry</LI></OL></P><P class="tags">dna, precipitation, from, filter, paper, plasmid, sodium acetate, ethanol, etoh, pellet, dried, te, buffer, vacuum</P></DIV><DIV class="textBody"><P class="h3-heading" id="AutoclavingWaste">Autoclaving Waste</P><P class="p-text"><OL><LI>Put biohazard bag in an autoclave tray</LI><LI>Put in autoclave in ‘wrapped’ setting for 1 hour heating and 2 hour drying
                                    </LI><LI>Make sure waste is not liquid and has dried</LI><LI>Put the autoclaved waste inside a regular trash bag and make sure the regular trash bag is tied</LI><LI>Take waste to the black dumpster behind the building</LI></OL></P><P class="tags">autoclaving, waste, autoclave, biohazard, trash</P></DIV></DIV></DIV><DIV id="div-safety" class="sectionClass"><DIV class="contentText risk"><DIV class="textBody"><P class="h3-heading">Personal Protective Equipment</P><P class="p-text">Every member of UMaryland iGEM was required to take various online and classroom safety training before performing experiments in the lab, including proper safety protocol. We made sure to wear gloves, closed toed shoes, and safety glasses when conducting experiments.</P></DIV><DIV class="textBody"><P class="h3-heading">Biocontainment</P><P class="p-text">Making sure that our transformed bacteria does not escape into the environment was a very important concern for our team. We constantly cleaned our surfaces, properly disposed of waste, and autoclaved frequently in order to prevent environmental harm.</P></DIV></DIV></DIV></DIV></DIV><DIV id="bot-nav"><DIV id="container"><DIV class="pull-left"><A href="mailto:umarylandigem@gmail.com"><P id="contact"> Contact Us </P></A></DIV></DIV><DIV class="copyright"><P id="copyright">© University of Maryland iGEM 2016</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>