Team:Stanford-Brown/Parts
Toggle navigation
SB iGEM
Home
Vision
biOrigami Projects
Overview
Polystyrene Synthesis
P(3HB) Synthesis
BioHYDRA
Cellulose Sheets
Plastic Folding
CRATER
Modeling
Collaborations
Cellulose Characterization
InterLab Measurement Study
Practices
BioBricks
Notebooks
About
Our Team
Attributions
Achievements
Biobricks
Our Biobricks
Polystyrene Synthesis BioBricks
Biobrick: BBa_K1692000
Ferulic Acid Decarboxylase Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We isolated this genetic part from Saccharomyces cerevisiae and inserted the protein-coding sequence into the pSB1C3 backbone. The original sequence in yeast contains an SpeI restriction site in the 999th nucleotide position. Thus, we performed site-directed mutagenesis in order to make our part BioBrick compatible. Gene sequencing analysis confirmed that our site-directed mutagenesis was successful.
Biobrick: BBa_K1692001
Ferulic Acid Decarboxylase with T7 promoter Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We isolated this genetic part from Saccharomyces cerevisiae and inserted the protein-coding sequence into the pSB1C3 backbone. The original sequence in yeast contains an SpeI restriction site in the 999th nucleotide position. Thus, we performed site-directed mutagenesis in order to make our part BioBrick compatible. Gene sequencing analysis confirmed that our site-directed mutagenesis was successful.  This part includes a T7 promoter, allowing for inducible expression.
Biobrick: BBa_K1692002
Codon-optimized Ferulic Acid Decarboxylase with T7 promoter and FLAG tag Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We codon-optimized the FDC gene from Saccharomyces cerevisiae for expression in E. coli. The decision to use FDC from S. cerevisiae was based on prior work in styrene biosynthesis, notably McKenna (2012). Our construct includes the FDC coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.
Biobrick: BBa_K1692003
PAL with T7 promoter Phenylalanine ammonia lyase (PAL) catalyzes the conversion of L-phenylalanine to trans-cinnamic acid. This part is a modification of University of British Columbia’s 2013 PAL biobrick part (BBa_K1129003) from Streptomyces maritimus. Specifically, our part contains a T7 promoter, allowing for inducible expression.
Biobrick: BBa_K1692004
Codon-optimized PAL with T7 promoter and FLAG tag Phenylalanine ammonia lyase (PAL) catalyzes the conversion of L-phenylalanine to trans-cinnamic acid. Our PAL construct is codon-optimized for expression in E. coli. The original sequence is derived from Anabaena variabilis. We chose the A. variabilis variant of PAL because the literature has characterized it as functioning well, in contrast to University of British Columbia’s 2013 PAL biobrick part (BBa_K1129003) from Streptomyces maritimus, which has much lower activity.  Our construct includes the PAL coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.
Biobrick: BBa_K1692005
UbiX UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. This part is contains the protein-coding UbiX sequence from E. coli.
Biobrick: BBa_K1692006
UbiX with T7 promoter UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. This part is contains the protein-coding UbiX sequence from E. coli. Additionally, this part includes a T7 promoter, allowing for inducible expression.
Biobrick: BBa_K1692007
UbiX with T7 promoter and FLAG tag UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. Our genetic construct includes the UbiX coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.
Biobrick: BBa_K1692008
Styrene Synthesis Operon This operon is a composite of three enzymes in the following order: FDC, UbiX, and PAL.  PAL converts phenylalanine to cinnamic acid.  FDC converts cinnamic acid to styrene.  UbiX modifies flavin mononucleotide to produce a cofactor that is required for FDC activity. The entire operon is controlled via an inducible T7 promoter. Each protein-coding sequence is preceded by a ribosome binding site and followed by a FLAG-tag peptide, enabling easy and efficient extraction.
P(3HB) Synthesis BioBricks
Biobrick: BBa_K1692020
S. aureus type II PanK (CoaA) Type II pantothenate kinase from Staphylococcus aureus, not sensitive to feedback inhibition from CoA.
Biobrick: BBa_K1692034
Pank + hybrid phaCAB + lysis Construct that was cloned to have P(3HB) producing E. coli cells lyse when induced with Arabinose. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part.
Biobrick: BBa_K1692023
Ptet + Luxl Construct that was cloned to have P(3HB) producing E. coli cells lyse after reaching a certain population density using a quorum sensing promoter. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part.
Biobrick: BBa_K1692024
Autolysis Construct that was cloned to have P(3HB) producing E. coli cells lyse after reaching a certain population density using a quorum sensing promoter. The difference between this brick BBa_K1692023 is that the basic parts composing this gene are in a different order, which is part of our testing to optimize this system. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part.
BioHYDRA BioBricks
Biobrick: BBa_K1692028
cotz-aeBlue-CipA a fusion protein consisting of a spore coat protein, cotZ (building off work done on Sporobeads by the LMU Munich 2012 iGEM team), and a cellulose binding domain (CipA). Additionally, we decided to add aeBlue, a chromogenic protein, between cotZ and CipA to be able to see with the naked eye whether Bacillus is in a vegetative or a spore state.
Cellulose Sheets BioBricks
Biobrick: BBa_K1692027
CipA Cellulose binding domain adapted from Imperial 2014 iGEM team. Last part of our composite part BBa_BBa_K1692028. We removed the illegal EcoR1 site that the Imperial 2014 iGEM team had in their construc.
CRATER Briobricks
Biobrick : BBa_K1692033
RFP gDNA This gene provides the intermediate for synthesizing gRNA targeting RFP (BBa_J04450). When used in conjuction with the CRISPR/Cas system, this gRNA will bind to and digest the RFP plasmid.
Biobrick: BBa_K1692030
amilGFP yellow chromoprotein, RBS and promoter This plasmid contains the amilGFP yellow chromoprotein gene from part BBa_K1033931 with the RBS and promoter from part BBa_K608002.
Biobrick: BBa_K1692031
meffBlue blue chromoprotein with RBS and promoter This plasmid contains the meffBlue blue chromoprotein gene found in the BBa_K1033902 part, with the RBS and promoter from part BBa_K608002.
Biobrick: BBa_K1692032
amilCP blue chromoprotein with RBS and promoter This plasmid contains the amilCP blue chromoprotein gene found in part BBa_K592009 and the RBS and promoter from part BBa_K608002.
Copyright © 2015 Stanford-Brown iGEM Team
