<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-material-light-blue navbar-fixed-top oriEdit"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Stanford-Brown">SB iGEM</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2015.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Vision">Vision</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">biOrigami Projects</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Projects">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PS">Polystyrene Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PHA">P(3HB) Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/BioHYDRA">BioHYDRA</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Cellulose">Cellulose Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Plastic">Plastic Folding</A></LI></UL><LI><A href="https://2015.igem.org/Team:Stanford-Brown/CRATER">CRATER</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Modeling">Modeling</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations">Cellulose Characterization</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Interlab">InterLab Measurement Study</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Notebook">Notebooks</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">About</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/JudgingAndMedalsCriteria">Achievements</A></LI></DIV></DIV></NAV></DIV><P><TITLE>Notebooks</TITLE></P><DIV class="jumbotron oridomiLeft"><DIV class="container"><H1>Modeling<SMALL><I>In silico</I> Studies of Bioplastic Production</SMALL></H1></DIV></DIV><DIV class="container pp"><DIV class="pull-right"><VIDEO poster="https://static.igem.org/mediawiki/2015/9/93/O_69aa34f17133dcbe-0.jpg" controls="" width="558" height="316"><P style="font-style:italic;color:red;border-style:solid;border-width:2px;border-color:red">Your browser either does not support HTML5 or cannot handle MediaWiki open video formats. Please consider upgrading your browser, installing the appropriate plugin or switching to a Firefox or Chrome install.</P></VIDEO></DIV><H2>Why Model?</H2><P>Experimentation makes up the backbone of any scientific discipline, and synthetic biology is no exception. It is only through experimental analysis that we can understand, test, and refine our biological creations in a rigorous manner. One might be tempted to ask why computation and simulation need even make appearances on our stage. We shall attempt to say a few words in the defense of mathematical modeling.</P><P>To the question “Why model?”, Joshua Epstein, the 2008 recipient of the NIH Director’s Pioneer Award, offers a simple retort: “You are a modeler,” by which he means to say that we all are constantly running implicit models inside our head often without realizing it. When a PCR yields unexpected results, for example, we may call upon an internal model of DNA replication in order to assess the points at which reality may have diverged from expectation: perhaps the primers bound to the template DNA non-selectively or maybe our chosen annealing temperature was too high. Whenever scientists generate hypotheses based on some internal picture in their head, those scientists are practicing the age old tradition of modeling.</P><P>The question then becomes, “Why use mathematical models?” Our PCR example from above offers an answer to this question as well. In determining whether nonspecificity lay at the root of our faulty PCR, we may resort to a read-alignment program that can computationally predict potential primer binding sites. And it is standard practice to use an annealing-temperature calculator before designing a set of PCR cycles. The existence and widespread use of tools such as these illustrates the inadequacy of our minds in solving certain problems without aid as well as the consequent need for explicit quantitative models and computing machinery.</P><P>In the case of our project, we used computational tools to help model the molecular dynamics of two of our biosynthetic reactions. These were instances where human intuition alone could not reliably answer a question that interested us, namely, which enzyme (PAL or FDC) or substrate (CoA or Acetate) exerts greater control on the flux or amount of product in our specified pathway? It is this question that we shall explore below.</P><DIV class="row"><DIV class="col-md-8"><H2>Constructing a Model for Styrene Synthesis</H2><P>We wanted to create a plasmid containing our three genes of interest, PAL, UbiX, and FDC, but we didn’t know which order to place the genes in our plasmid. Since the genes placed closer to the beginning of the operon are transcribed more than those closer to the end, we would want to place our most influential enzymes toward the promoter, and our less important enzymes toward the end. There are 3! = 6 orderings of these three genes so we would potentially make 6 plasmids, test their styrene productivity levels, and submit the one which yielded the most styrene. The described process would have cost us weeks of our time as well as lab resources and money. Instead of creating all 6 plasmids and conducting the wet lab experiments, we instead turned to our mathematical model. In order to model our enzymatic pathway, we created a system of ordinary differential equations based on the Michaelis Menten enzyme model shown to the right. Numerically simulating the model in MATLAB gave us curves that represented the species concentration as a function of time over a specified period of time.</P><P>Each FDC and PAL’s influence on the overall reaction flux was tested by varying each of their concentrations while keeping the other constant. It was found that changing the concentration of FDC changed the rate of the reaction much more than changing PAL. Specifically, PAL and FDC started out at 3 nM protein concentration and 1 nM of PAL concentration was added to one while keeping FDC constant. This process was repeated for the varying of FDC while keeping PAL constant. We then recorded the model’s prediction of styrene concentration after an hour of the reaction running and compared the two enzyme’s influence. The results can be seen below. Clearly, we can see that FDC had a much bigger effect on the overall reaction velocity.</P></DIV></DIV><P>We decided not to include UbiX in the model because we already knew that UbiX would correspond to active FDC in a 1 to 1 ratio, so we would never need more UbiX than FDC, and also because UbiX would always be right after FDC since UbiX is needed to activate FDC, our enzyme of most influence. In conclusion, our model helped us decide on the order FDC-UbiX-PAL for our combo plasmid without having to wastefully test out every combination possible.</P><H2>Constructing a Model for P(3HB) Synthesis</H2><DIV class="row"><DIV class="col-md-8"><P>Seeing that Coenzyme A (CoA) and acetate are two precursors to our bioplastic P(3HB), we hypothesized that by adding the PanK gene, we could upregulate CoA and thus significantly increase the amount of plastic produced. We did increase the amount of 3HB CoA monomer produced, but the difference wasn’t as large as we had hoped and expected. To explain our observations, we looked towards a mathematical model.</P><P>Using a similar approach to the one detailed above, we attempted to model the bioproduction of P(3HB) using the same construction as we did for styrene. Detailed on the right is our set of differential equations used to represent the dynamics of the reaction pathway producing our other bioplastic P(3HB).</P><P>In a similar fashion to our strategy for determining the most influential enzyme, we tried to determine the most influential substrate on the amount of produced P(3HB). We plotted the time profiles of our P(3HB) for varying CoA while holding the initial concentration of acetate constant, and also the reverse, varying acetate while keeping the initial concentration of CoA constant. We found that over long time periods, final concentration of P(3HB) grew linearly with initial concentration of acetate. In other words, as we increased the initial concentration of acetate by a constant 1 mM each simulation while holding initial concentration of CoA constant at 1 mM, the final concentration of P(3HB) increased each time by a constant amount. In contrast, as we increased the initial concentration of CoA by a constant 0.5 mM each simulation while holding the initial concentration of acetate fixed at 1 mM, we found that the corresponding changes in final concentration of P(3HB) decreased. So after a certain threshold initial concentration of CoA, any additional CoA added made little to no difference on the final concentration of our plastic produced by the pathway.</P></DIV></DIV><P>These results explain why we may have seen an underwhelming increase in amount of monomer produced due to the upregulation of the initial substrate CoA. According to the results of our simulations, future attempts at upregulation of 3HB CoA should focus on the upregulation of the other precursor, acetate.</P><P>Below is a table with our kinetic parameters taken from various kinetics databases and papers.</P><H3>References</H3><P>[6] Payne, Karl A. P., Mark D. White, Karl Fisher, Basile Khara, Samuel S. Bailey, David Parker, 
   Nicholas J. W. Rattray, Drupad K. Trivedi, Royston Goodacre, Rebecca Beveridge, Perdita Barran, Stephen E. J. Rigby, Nigel S. Scrutton, Sam Hay, and David Leys. &quot;New Cofactor Supports α,β-unsaturated Acid Decarboxylation via 1,3-dipolar Cycloaddition.&quot; Nature (2015): 497-501. </P><P>[7] Wang, Lin, Alejandra Gamez, Holly Archer, Enrique E. Abola, Christineh N. Sarkissian, Paul 
    Fitzpatrick, Dan Wendt, Yanhong Zhang, Michel Vellard, Joshua Bliesath, Sean M. Bell, 
    Jeffrey F. Lemontt, Charles R. Scriver, and Raymond C. Stevens. &quot;Structural and Biochemical Characterization of the Therapeutic Anabaena variabilis Phenylalanine Ammonia Lyase.&quot; Journal of Molecular Biology: 623-35.  </P><P>[9] Oeljeklaus, Silke, Kerstin Fischer, and Bernt Gerhardt. &quot;Glyoxysomal Acetoacetyl-CoA Thiolase and 3-oxoacyl-CoA Thiolase from Sunflower Cotyledons.&quot; Planta (2002): 597-607.</P><P>[10] Matsumoto, K., Y. Tanaka, T. Watanabe, R. Motohashi, K. Ikeda, K. Tobitani, M. Yao, I. Tanaka, and S. Taguchi. &quot;Directed Evolution and Structural Analysis of NADPH-Dependent Acetoacetyl Coenzyme A (Acetoacetyl-CoA) Reductase from Ralstonia eutropha Reveals Two Mutations Responsible for Enhanced Kinetics.&quot; Applied and Environmental Microbiology (2013): 6134-139. </P><P>[11] Yuan, Wei, Yong Jia, Jiamin Tian, Kristi D Snell, Ute Müh, Anthony J Sinskey, Ralph H Lambalot, Christopher T Walsh, and Joanne Stubbe. &quot;Class I and III Polyhydroxyalkanoate Synthases from Ralstonia Eutropha and Allochromatium Vinosum: Characterization and Substrate Specificity Studies.&quot; Archives of Biochemistry and Biophysics: 87-98.</P><P>[12] Yuan, Wei, Yong Jia, Jiamin Tian, Kristi D Snell, Ute Müh, Anthony J Sinskey, Ralph H Lambalot, Christopher T Walsh, and Joanne Stubbe. &quot;Class I and III Polyhydroxyalkanoate Synthases from Ralstonia eutropha and Allochromatium vinosum: Characterization and Substrate Specificity Studies.&quot; Archives of Biochemistry and Biophysics: 87-98. </P><P>[13] Brown, T. D. K., M. C. Jones-Mortimer, and H. L. Kornberg. &quot;The Enzymic Interconversion of Acetate and Acetyl-coenzyme A in Escherichia coli.&quot; Journal of General Microbiology (1977): 327-36. </P></DIV><FOOTER><DIV class="container"><DIV class="row"><H6>Copyright © 2015 Stanford-Brown iGEM Team</H6></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>