<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-material-light-blue navbar-fixed-top oriEdit"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Stanford-Brown">SB iGEM</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2015.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Vision">Vision</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">biOrigami Projects</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Projects">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PS">Polystyrene Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PHA">P(3HB) Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/BioHYDRA">BioHYDRA</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Cellulose">Cellulose Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Plastic">Plastic Folding</A></LI></UL><LI><A href="https://2015.igem.org/Team:Stanford-Brown/CRATER">CRATER</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Modeling">Modeling</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations">Cellulose Characterization</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Interlab">InterLab Measurement Study</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Notebook">Notebooks</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">About</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/JudgingAndMedalsCriteria">Achievements</A></LI></DIV></DIV></NAV></DIV><P><TITLE>Biobricks</TITLE></P><DIV class="jumbotron oridomiLeft"><DIV class="container"><H1>Our Biobricks</H1></DIV></DIV><DIV class="container"><H2>Polystyrene Synthesis BioBricks</H2><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692000" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692000</H4><P><B>Ferulic Acid Decarboxylase</B> Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We isolated this genetic part from Saccharomyces cerevisiae and inserted the protein-coding sequence into the pSB1C3 backbone. The original sequence in yeast contains an SpeI restriction site in the 999th nucleotide position. Thus, we performed site-directed mutagenesis in order to make our part BioBrick compatible. Gene sequencing analysis confirmed that our site-directed mutagenesis was successful.
 </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692001" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692001</H4><P><B>Ferulic Acid Decarboxylase with T7 promoter</B> Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We isolated this genetic part from Saccharomyces cerevisiae and inserted the protein-coding sequence into the pSB1C3 backbone. The original sequence in yeast contains an SpeI restriction site in the 999th nucleotide position. Thus, we performed site-directed mutagenesis in order to make our part BioBrick compatible. Gene sequencing analysis confirmed that our site-directed mutagenesis was successful.  This part includes a T7 promoter, allowing for inducible expression.</P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692002" class="btn" id="be1" target="_blank"><H4> Biobrick: BBa_K1692002</H4><P><B>Codon-optimized Ferulic Acid Decarboxylase with T7 promoter and FLAG tag</B> Ferulic acid decarboxylase (FDC) catalyzes the conversion of trans-cinnamic acid to styrene. We codon-optimized the FDC gene from Saccharomyces cerevisiae for expression in E. coli. The decision to use FDC from S. cerevisiae was based on prior work in styrene biosynthesis, notably McKenna (2012). Our construct includes the FDC coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.
 </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692003" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692003</H4><P><B>PAL with T7 promoter</B> Phenylalanine ammonia lyase (PAL) catalyzes the conversion of L-phenylalanine to trans-cinnamic acid. This part is a modification of University of British Columbia’s 2013 PAL biobrick part (BBa_K1129003) from Streptomyces maritimus. Specifically, our part contains a T7 promoter, allowing for inducible expression.
</P></A></DIV><DIV><DIV class="pull-right"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692004" class="btn" id="be1" target="_blank"><H4> Biobrick: BBa_K1692004</H4><P><B>Codon-optimized PAL with T7 promoter and FLAG tag</B> Phenylalanine ammonia lyase (PAL) catalyzes the conversion of L-phenylalanine to trans-cinnamic acid. Our PAL construct is codon-optimized for expression in E. coli. The original sequence is derived from Anabaena variabilis. We chose the A. variabilis variant of PAL because the literature has characterized it as functioning well, in contrast to University of British Columbia’s 2013 PAL biobrick part (BBa_K1129003) from Streptomyces maritimus, which has much lower activity.  Our construct includes the PAL coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.
</P></A></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692005" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692005</H4><P><B>UbiX</B> UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. This part is contains the protein-coding UbiX sequence from E. coli.
</P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692006" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692006</H4><P><B>UbiX with T7 promoter</B> UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. This part is contains the protein-coding UbiX sequence from E. coli. Additionally, this part includes a T7 promoter, allowing for inducible expression.
 </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692007" class="btn" id="be1" target="_blank"><H4> Biobrick: BBa_K1692007</H4><P><B>UbiX with T7 promoter and FLAG tag</B> UbiX is a flavin prenyltransferase that normally plays a role in ubiquinone biosynthesis in E. coli. UbiX transfers a prenyl group from dimethylallyl monophosphate (DMAP) to flavin mononucleotide (FMN), thereby creating a cofactor that happens to be essential to the functionality of FDC. Our genetic construct includes the UbiX coding sequence, a T7 inducible promoter, a ribosome binding site, and a FLAG-tag peptide sequence for easy and efficient protein purification. We have sequenced our construct and verified that all these components are indeed present.</P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692008" class="btn" id="be1" target="_blank"><H4> Biobrick: BBa_K1692008</H4><P><B>Styrene Synthesis Operon</B> This operon is a composite of three enzymes in the following order: FDC, UbiX, and PAL.  PAL converts phenylalanine to cinnamic acid.  FDC converts cinnamic acid to styrene.  UbiX modifies flavin mononucleotide to produce a cofactor that is required for FDC activity. The entire operon is controlled via an inducible T7 promoter. Each protein-coding sequence is preceded by a ribosome binding site and followed by a FLAG-tag peptide, enabling easy and efficient extraction.
</P></A></DIV></DIV><H2>P(3HB) Synthesis BioBricks</H2><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692020" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692020</H4><P><B>S. aureus type II PanK (CoaA)</B> Type II pantothenate kinase from <I>Staphylococcus aureus</I>, not sensitive to feedback inhibition from CoA. </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692034" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692034</H4><P><B>Pank + hybrid phaCAB + lysis</B> Construct that was cloned to have P(3HB) producing <I>E. coli</I> cells lyse when induced with Arabinose. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part.  </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692023" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692023</H4><P><B>Ptet + Luxl</B> Construct that was cloned to have P(3HB) producing <I>E. coli</I> cells lyse after reaching a certain population density using a quorum sensing promoter. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part. </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692024" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692024</H4><P><B>Autolysis</B> Construct that was cloned to have P(3HB) producing <I>E. coli</I> cells lyse after reaching a certain population density using a quorum sensing promoter. The difference between this brick BBa_K1692023 is that the basic parts composing this gene are in a different order, which is part of our testing to optimize this system. This would greatly facilitate P(3HB) extraction. We are currently still testing and characterizing this part. </P></A></DIV></DIV><H2>BioHYDRA BioBricks</H2><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692028" class="btn" id="be1" target="_blank"><H4>Biobrick: BBa_K1692028</H4><P><B>cotz-aeBlue-CipA</B>  a fusion protein consisting of a spore coat protein, cotZ (building off work done on Sporobeads by the LMU Munich 2012 iGEM team), and a cellulose binding domain (CipA). Additionally, we decided to add aeBlue, a chromogenic protein, between cotZ and CipA to be able to see with the naked eye whether <I>Bacillus</I> is in a vegetative or a spore state.  </P></A></DIV></DIV><H2>Cellulose Sheets BioBricks</H2><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692024" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692027</H4><P><B>CipA</B> Cellulose binding domain adapted from Imperial 2014 iGEM team. Last part of our composite part BBa_BBa_K1692028. We removed the illegal EcoR1 site that the Imperial 2014 iGEM team had in their construc. </P></A></DIV></DIV><H2>CRATER Briobricks</H2><DIV class="row"><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692033" class="btn" id="be1" target="_blank"><H4>Biobrick : BBa_K1692033</H4><P><B>RFP gDNA</B> This gene provides the intermediate for synthesizing gRNA targeting RFP (BBa_J04450). When used in conjuction with the CRISPR/Cas system, this gRNA will bind to and digest the RFP plasmid. </P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692030" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692030</H4><P><B>amilGFP yellow chromoprotein, RBS and promoter </B>This plasmid contains the amilGFP yellow chromoprotein gene from part BBa_K1033931 with the RBS and promoter from part BBa_K608002.</P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692031" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692031</H4><P><B>meffBlue blue chromoprotein with RBS and promoter</B> This plasmid contains the meffBlue blue chromoprotein gene found in the BBa_K1033902 part, with the RBS and promoter from part BBa_K608002.</P></A></DIV><DIV class="col-sm-6"><A href="http://parts.igem.org/Part:BBa_K1692032" class="btn" id="be2" target="_blank"><H4>Biobrick: BBa_K1692032</H4><P><B>amilCP blue chromoprotein with RBS and promoter</B> This plasmid contains the amilCP blue chromoprotein gene found in part BBa_K592009 and the RBS and promoter from part BBa_K608002.</P></A></DIV></DIV></DIV><FOOTER><DIV class="container"><DIV class="row"><H6>Copyright © 2015 Stanford-Brown iGEM Team</H6></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>