<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_PS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/PS</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-material-light-blue navbar-fixed-top oriEdit"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Stanford-Brown">SB iGEM</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2015.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Vision">Vision</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">biOrigami Projects</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Projects">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PS">Polystyrene Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PHA">P(3HB) Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/BioHYDRA">BioHYDRA</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Cellulose">Cellulose Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Plastic">Plastic Folding</A></LI></UL><LI><A href="https://2015.igem.org/Team:Stanford-Brown/CRATER">CRATER</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Modeling">Modeling</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations">Cellulose Characterization</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Interlab">InterLab Measurement Study</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Notebook">Notebooks</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">About</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/JudgingAndMedalsCriteria">Achievements</A></LI></DIV></DIV></NAV></DIV><P><TITLE>SB iGEM 2015</TITLE></P><DIV class="jumbotron oridomiLeft"><DIV class="container"><H1>Polystyrene<SMALL>Biologically Producing Self-Folding Plastics</SMALL></H1></DIV></DIV></DIV><DIV class="container pp"><DIV class="pull-right"><VIDEO poster="https://static.igem.org/mediawiki/2015/9/96/O_b7712019c922a549-0.jpg" controls="" width="558" height="316"><P style="font-style:italic;color:red;border-style:solid;border-width:2px;border-color:red">Your browser either does not support HTML5 or cannot handle MediaWiki open video formats. Please consider upgrading your browser, installing the appropriate plugin or switching to a Firefox or Chrome install.</P></VIDEO></DIV><DIV><H2>Introduction</H2><P>Polystyrene is one of the most widely used plastics today. As a synthetic polymer of styrene monomers, polystyrene exhibits self-folding properties when heated.  We endeavored to biologically synthesize styrene by genetically engineering E. coli to produce the enzymes required in the pathway from L-phenylalanine to trans-cinnamic acid to styrene. Using this pathway we aimed to produce the styrene monomer both in vivo and in vitro from renewable sources.  After producing styrene biologically, we characterized a method for polymerizing styrene into the polymer, polystyrene, which is then ready for folding.
  </P><A href="https://2015.igem.org/Team:Stanford-Brown/Parts" class="btn btn-success btn-lg">See our BioBricks</A><H2>Styrene Synthesis:  Engineering E. coli to produce the styrene monomer</H2><P>The first step in making biologically produced polystyrene is to create the monomer styrene.  After combing through the literature we found a group from Arizona State University that had successfully produced styrene in a cell via a two-step enzymatic pathway from the amino acid phenylalanine to trans-cinnamic acid to styrene.  The enzymes that catalyzed the first and second step were Phenylalnine Ammonia Lyase (PAL) and Ferulic Acid Decarboxylase (FDC) respectively [4].  However, as we read more papers about styrene synthesis we found that the two-step pathway was actually more complicated than it looked.  The second enzyme FDC, was regulated by a co-factor that was recently found to be a prenylated riboflavin formed from dimethylallyl monophosphate (DMAP) and flavin mononucleotide (FMN) by another enzyme known as UbiX [8].  We now had a complete pathway to produce styrene in E. coli.  </P><P><B>Step one:  Get the genes</B></P><P>In order to get the genes that encoded our three enzymes we extracted the gene directly from a host organism, synthesized the gene using Integrated DNA Technologies (IDT), and, when possible, ordered the preexisting part from the iGEM registry.   </P><P>Our first enzyme PAL was previously characterized by the University of British Columbia 2013 iGEM team.  We ordered their biobrick (<A href="http://parts.igem.org/Part:BBa_K1129003">BBa_K1129003</A>) from the registry.  However, this PAL gene was extracted from Streptomyces maritimus, which according to the literature had a very ineffective PAL enzyme [4].  So in addition to ordering UCB’s PAL biobrick we also ordered from IDT a codon optimized (for E. coli) version of the PAL gene from Anabaena variabilis.  We chose the PAL gene from Anabaena variabilis because it was well characterized in literature and had been shown to be the most effective version of the enzyme [7].  </P><P>For our second enzyme, FDC, we both extracted the gene directly from Saccharomyces cerevisiae and ordered this gene codon optimized (for E. coli) from IDT.    </P><P>For our last enzyme UbiX we both extracted the gene directly from E. coli and ordered this gene with a FLAG tag from IDT.  As we will explain later on, we added a FLAG tag sequence to the end of all of our synthesized genes in order to extract and purify the protein for in-vitro assays.  
  </P><DIV class="pull-left"><P><FONT size="1"><B>SDS PAGE Gel</B> with Mark 12 ladder</FONT></P></DIV><P><B>Step two:  Put the genes into plasmids</B></P><P>After obtaining our genes, we inserted them into a pSB1C3 plasmid from the registry that had a T7 promoter and RBS (<A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A>). We confirmed our gene insert through DNA sequencing with VF2 and VR primers.  </P><P>The FDC gene we extracted from yeast has an illegal SpeI restriction enzyme site within its sequence, rendering the plasmids with the FDC gene incompatible with biobrick assembly methods.  To solve this we successfully performed site directed mutagenesis on this site [3].</P><P><B>Step three:  Turn genes into proteins</B></P><P>Now that we had all our genes in plasmids with a promoter and RBS, we transformed our three synthesized genes (PAL, FDC, UbiX) into T7 expressing NEB E. coli separately and then initiated T7 polymerase gene expression by adding IPTG to our cultures.  Because all of our synthesized genes had a FLAG tag at the end of their sequence, we were able to purify our proteins from the cell lysate. Finally we ran all three of our purified enzymes on SDS PAGE to verify that our proteins were the correct molecular weight, which they were.  See below for all of the specific protocols used in this section of our project.</P><H3>Testing functionality in-vitro of enzymes and modeling pathway</H3><P><B>Step one:  Testing PAL</B></P><P>Once we were confident that we had successfully purified our enzymes from the T7 expressing E. coli, the next logical step was to test the in vivo functionality of our proteins.  In order to test PAL’s functionality, we made use of the fact that PAL’s reactant and product, namely phenylalanine and trans-cinnamic acid (tCA), have different characteristic absorbance spectra in the ultraviolet region [4,7]. Notably, tCA has a large and easily distinguishable peak at 268 nm, whereas phenylalanine displays a much less noticeable peak just below 260 nm.  In an initial assay, we took absorbance spectra from a reaction mixture of PAL with phenylalanine and noticed a large peak at 268 nm, suggesting the presence of tCA. </P><P>This initial experiment provided good evidence that our PAL was in fact functioning. Our next step was to perform a kinetic time course experiment in order to obtain new data on our enzymes kinetic parameters.  Using the Beer-Lambert relation, which states that, all other factors held constant, concentration is directly proportional to absorbance, we could spectrophotometrically track the increase in absorbance at 268 nm in real time. 



We created reaction mixtures of PAL along with 8 different concentrations of its substrate, phenylalanine, and tracked the reaction in real time over the course of about 4 hours using a spectrophotometer. We had to purchase special UV-transparent plates to run this experiment, because plastic well plates absorb in the UV range.  Not only did this experiment further demonstrate that our PAL was working, it also provided us with the necessary kinetic data to estimate PAL’s biochemical parameters.</P><P><B>Step two:  Testing FDC/UbiX</B></P><P>Given the success of the spectrophotometric approach in our PAL assays, we attempted to proceed in a similar manner with FDC and UbiX. Unfortunately, there were several complicating factors.</P><P>In the case of UbiX, the main problem lay in the impossibility of distinguishing UbiX’s reactant from its product. UbiX catalyzes the prenylation of flavin mononucleotide. Unfortunately, this chemical transformation does not result in a change in the overall absorbance of the reaction solution that we are able to detect. The reactant and the product are simply too chemically similar [6]. We believe that this is one reason why no isozyme of UbiX has ever been kinetically characterized.</P><P> In the case of FDC, we initially set out to use tCA’s unique absorbance spectrum to our advantage: whereas we measured an increase in tCA to track the activity of PAL (which produces tCA), we endeavored to measure a decrease in tCA to probe the activity of FDC (which consumes tCA). Unfortunately, FDC cannot function in the absence of its prenylated FMN cofactor, which is supplied by UbiX. This presents a challenge, since FMN has a strong absorbance peak in precisely the same region as tCA’s peak. Consequently, we could not apply the same spectrophotometric method to verify or quantify FDC activity.</P><P>As an alternative, we adapted a purely computational approach to obtain at least some of the results that we would have gained from the same genre of experimental analysis that we performed in our PAL assay. Although we could not perform a multiplexed time course experiment on FDC or UbiX, we carried out a sensitivity analysis to determine FDC’s role in the overall synthetic pathway, as described below.</P><P><B>Step three:  Modeling pathway</B></P><P>We had hoped that our in vitro assays would help us not only verify that our enzymes were functioning, but also assist us in determining each enzyme’s kinetic significance to the overall reaction.  This knowledge would allow us to optimize in vivo styrene production by preferentially expressing the enzymes with the most significance. However, without complete in vitro data we could not determine which enzymes were most significant so turned to a mathematical model of our pathway.  Using literature values of our enzymatic constants, we were able to model our enzymatic pathway from a system of ordinary differential equations based on the Michaelis Menten equations. Numerically simulating the model in MATLAB gave us curves that represented the species concentration as a function of time over a specified period. </P><P>FDC and PAL’s influence on the overall reaction flux was tested by varying each of their concentrations while keeping the other constant. It was found that changing the concentration of FDC significantly increase the rate of the reaction, while changing PAL concentration did little to the rate of reaction. </P><P>In conclusion, our in vitro testing and mathematical models helped us both verify our enzymes functionality and determine which enzymes were most significant to the rate of styrene production. See below for all of the specific protocols used in this section of our project.  See the modeling section for more information on our modeling approach.  </P><H3>In-vivo production and in-situ removal of Styrene</H3><P><B>Step one:  Making combo plasmid</B></P><DIV class="pull-left"><P><FONT size="1"><B>Styrene Synthesis Operon</B> made using Gibson Assembly</FONT></P></DIV><P>Now that we had determined the importance of our enzymes to styrene production we planned on creating an operon of our three genes of interest, PAL, UbiX, and FDC for in vivo testing.  The genes closest to the beginning of the operon are transcribed more than those to the end [1].  Therefore, we would place our most influential enzymes toward the beginning and our less influential enzymes toward the end. The order of our operon is FDC, UbiX then PAL.  We assembled the operon in a plasmid using Gibson Assembly and used DNA sequencing to confirm the order of the genes.  We then transformed our combo plasmid into T7 expressing cells to extract our protein as before.  Again, we ran our proteins sample on an SDS PAGE and confirmed that not only were all of our proteins expressing correctly, but that we had the most FDC and least PAL, as expected. </P><P><B>Step two:  In-situ removal</B></P><DIV class="pull-right"><P><FONT size="1"><B>Cell cultures</B> with and without solvent </FONT></P></DIV><P>A major issue with producing styrene in vivo is that styrene is actually toxic to the cell at concentrations as low as 300 mg/L [4].  So in order to make styrene production practical in vivo we would have to either make our cells more resistant to styrene or find a way to remove the styrene as it was being produced.  Because there is little known about why styrene is toxic to the cell, we decided to focus on the latter.  We found a paper that used an immiscible solvent, n-dodecane, to remove styrene in situ from cell cultures that produced styrene. 



The solvent did not impede cell growth and was able to more than double the total styrene produced by cell cultures [5].  To test this strategy we set up a cell growth assay with 48 liquid cell cultures. Half of the cultures were mixed with n-dodecane, while the other half had no solvent.  We than exogenously added pure styrene to the cultures at varying concentrations from 0 mg/L to 4,800 mg/L.  We let all of our cultures grow for 16 hours then measured their optical density.  We found that at high concentrations of styrene, the cultures without solvent had significantly lower absorbance levels than the cultures with solvent.  This result confirmed that in situ removal of styrene using n-dodecane would effectively increase the styrene toxicity threshold for our cells, making in vivo production more effective.  </P><P><B>Step three:  In-vivo production</B></P><P>We are currently in the process of verifying the presence of styrene in the cultures of our transformed combo plasmid we used for protein extraction.  We are analyzing these cultures using gas chromatography mass spectrometry to detect and quantify our styrene production in vivo. </P><H2>Styrene Polymerization:  Polymerizing biologically produced styrene into polystyrene</H2><DIV class="pull-right"><P><FONT size="1"><B>Free Radical Polymerization of Styrene Mechanism</B></FONT></P></DIV><P>Up until now we have focused on styrene synthesis.  However equally important is the polymerization of styrene into polystyrene, our final product for folding.  We researched many methods for styrene polymerization used both in labs and industry.  We found that a free radical mechanism would be the simplest and easiest to test.   </P><P><B>Step one:  Testing viability</B></P><P>Styrene has a conjugated pi-bond system between its alkene and phenyl group. Because of this, the molecule can stabilize an electron radical allowing it to polymerize during the propagation step of radical reactions.  

</P><DIV class="pull-right"><P><FONT size="1"><B>Azobisisobutyronitrile (AIBN)</B> decomposition at 60°C <A href="https://en.wikipedia.org/wiki/Azobisisobutyronitrile#/media/File:Formation_of_Radicals_from_AIBN.png"><I>(image source)</I></A></FONT></P></DIV>  

In order to initiate the radical reaction we used azobisisobutyronitrile (AIBN).  AIBN is an unstable molecule that when heated to 60°C will decompose into two radical species.  These radical species can initiate the polymerization of styrene.  We incubated pure styrene with AIBN at 60°C for one hour.   We then precipitated out polystyrene using a methanol wash, confirming that AIBN initiated polymerization of styrene to polystyrene works.  <H2>Conclusion</H2><DIV class="pull-right"><P><FONT size="1"><B>Polymerized Styrene</B></FONT></P></DIV><P>We successfully cloned all the genes for PAL, FDC, and UbiX into plasmids submitted as biobricks to the registry.  We purified our enzymes and confirmed the molecular weight on a SDS PAGE gel.  We tested PAL’s functionality in vitro, yielding quantitative information about the reaction velocity in the form of concentration curves that were used to determine reaction parameters. We used the Michaelis Menten equations to model our chemical pathway and make informed predictions about our specific enzymes and their influence on the overall pathway flux.  We used these predictions to design an operon of our genes that would yield the largest amount of styrene.  We used Gibson Assembly to create this operon in a plasmid.  We also refined procedures for extracting styrene as it is being produced bacterially and polymerize this styrene into polystyrene. </P><P>Some future work that could be done is in vivo testing of styrene production after transforming our operon containing plasmid into E. coli. Once styrene is identified, our extraction procedure would need to be carried out in order to isolate the styrene from the cell while maintaining non-toxic levels in the cell’s environment to allow the cells to continue producing styrene. Finally, our polymerization method would need to be applied to our isolated styrene and then our polystyrene would be ready for folding and property testing.</P><P>We showed that producing polystyrene from renewable sources using synthetic biology is possible!</P><H2>Protocols</H2><P><B>Device construction.</B> Parts for this project were obtained from the 2015 Distribution Plates, ordered from the Registry, or synthesized by IDT. BioBricks assembly was used to construct the devices.</P><P><B>Transforming bacteria.</B> Chemically competent DH5-alpha <I>E. coli</I> cells from New England Biolabs were used for all parts of this project. Plates containing successful transformants were stored at 4C.</P><P><B>Colony PCR.</B> We often needed to verify the success of our transformations into E. coli. We performed PCRs on multiple colonies using the OneTaq polymerase from New England Biolabs.</P><P><B>Site Directed Mutagenesis.</B> The native yeast sequence for FDC contains an SpeI restriction cut site at nucleotide position 999. We used Liu and Naismith’s technique for site-directed mutagenesis in order to change the codon to be compatible with BioBrick standards [3].</P><P><B>FLAG-tag purification.</B> Some of our genetic constructs included sequences coding for the FLAG octapeptide. We used Anti-FLAG M2 magnetic beads to easily extract protein products from the bacteria in which they were synthesized. Full protocol available from <A href="https://static.igem.org/mediawiki/2015/c/c5/M8823bul.pdf">Sigma-Aldrich</A>.</P><P><B>SDS PAGE.</B> To verify the identity of our protein products we ran sodium dodecyl sulfate polyacrylamide gels (SDS PAGEs). We fixed and visualized the gel using the SYPRO Ruby Gel Stain. Full protocol from <A href="https://static.igem.org/mediawiki/2015/2/24/Mp12000.pdf">Thermo Fisher</A>.</P><P><B>Spectramax Pro.</B> We used a Spectramax Pro Plate Reader to perform our spectrophotometric analysis, which included both full spectrum reads and kinetic time course assays. Many of our chemicals absorbed in the ultraviolet range, requiring specialized UV-transparent well plates. We used the Corning plates.</P><P><B>Gibson Assembly (NEB).</B> To create our composite operon consisting of PAL, FDC, and UbiX, we used Gibson Assembly to stitch together our three genes. Full protocol from <A href="https://static.igem.org/mediawiki/2015/f/fa/ManualE2611.pdf">New England Biolabs</A>.</P><P><B>Polymerization.</B> Polymerization of styrene into polystyrene proceeds by way of a radical mechanism. We used azobisisobutyronitrile (AIBN) as an initiator and methanol as a solvent. Full protocol <A href="https://static.igem.org/mediawiki/2015/c/cd/Untitleddocument_%281%29.pdf">here</A>.</P><A href="https://static.igem.org/mediawiki/2015/8/8e/LabNotebook-Styrene.pdf" class="btn btn-danger btn-lg">See our Lab Notebook!</A><H3>References</H3><P>[1] Lim, H. N., Y. Lee, and R. Hussein. &quot;Fundamental Relationship between Operon  Organization and Gene Expression.&quot; Proceedings of the National Academy of   Sciences (2011): 10626-0631. 
    </P><P>[2] Lin, Fengming, Kyle L. Ferguson, David R. Boyer, Xiaoxia Nina Lin, and E. Neil G.   Marsh. &quot;Isofunctional Enzymes PAD1 and UbiX Catalyze Formation of a Novel   Cofactor Required by Ferulic Acid Decarboxylase and 4-Hydroxy-3-  polyprenylbenzoic Acid Decarboxylase.&quot; ACS Chem. Biol. ACS Chemical Biology   (2015): 1137-144. 
    </P><P>[3] Liu, Huanting, and James H Naismith. &quot;An Efficient One-step Site-directed   Deletion, Insertion, Single and Multiple-site Plasmid Mutagenesis Protocol.&quot; BMC  Biotechnology BMC Biotechnol: 91. 
    </P><P>[4] Mckenna, Rebekah, and David R. Nielsen. &quot;Styrene Biosynthesis from Glucose by   Engineered E. Coli.&quot; Metabolic Engineering: 544-54. 
    </P><P>[5] Mckenna, Rebekah, Luis Moya, Matthew Mcdaniel, and David R. Nielsen.  &quot;Comparing in Situ Removal Strategies for Improving Styrene Bioproduction.&quot;   Bioprocess Biosyst Eng Bioprocess and Biosystems Engineering (2014): 165-74.  
    </P><P>[6] Payne, Karl A. P., Mark D. White, Karl Fisher, Basile Khara, Samuel S. Bailey, David  Parker, Nicholas J. W. Rattray, Drupad K. Trivedi, Royston Goodacre, Rebecca  Beveridge, Perdita Barran, Stephen E. J. Rigby, Nigel S. Scrutton, Sam Hay, and   David Leys. &quot;New Cofactor Supports α,β-unsaturated Acid Decarboxylation via   1,3-dipolar Cycloaddition.&quot; Nature (2015): 497-501.  
    </P><P>[7] Wang, Lin, Alejandra Gamez, Holly Archer, Enrique E. Abola, Christineh N.   Sarkissian, Paul Fitzpatrick, Dan Wendt, Yanhong Zhang, Michel Vellard, Joshua  Bliesath, Sean M. Bell, Jeffrey F. Lemontt, Charles R. Scriver, and Raymond C.  Stevens. &quot;Structural and Biochemical Characterization of the Therapeutic  Anabaena Variabilis Phenylalanine Ammonia Lyase.&quot; Journal of Molecular  Biology: 623-35. 
    </P><P>[8] White, Mark D., Karl A. P. Payne, Karl Fisher, Stephen A. Marshall, David Parker,   Nicholas J. W. Rattray, Drupad K. Trivedi, Royston Goodacre, Stephen E. J.  Rigby, Nigel S. Scrutton, Sam Hay, and David Leys. &quot;UbiX Is a Flavin  Prenyltransferase Required for Bacterial Ubiquinone Biosynthesis.&quot; Nature   (2015): 502-06.  
    </P></DIV><FOOTER><DIV class="container"><DIV class="row"><H6>Copyright © 2015 Stanford-Brown iGEM Team</H6></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>