<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_Cellulose skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/Cellulose</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-material-light-blue navbar-fixed-top oriEdit"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Stanford-Brown">SB iGEM</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2015.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Vision">Vision</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">biOrigami Projects</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Projects">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PS">Polystyrene Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/PHA">P(3HB) Synthesis</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/BioHYDRA">BioHYDRA</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Cellulose">Cellulose Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Plastic">Plastic Folding</A></LI></UL><LI><A href="https://2015.igem.org/Team:Stanford-Brown/CRATER">CRATER</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Modeling">Modeling</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations">Cellulose Characterization</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Interlab">InterLab Measurement Study</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Notebook">Notebooks</A></LI><LI class="dropdown"><A href="" data-target="#" class="dropdown-toggle" data-toggle="dropdown">About</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Stanford-Brown/JudgingAndMedalsCriteria">Achievements</A></LI></DIV></DIV></NAV></DIV><P><TITLE>SB iGEM 2015</TITLE></P><DIV class="jumbotron oridomiLeft"><DIV class="container"><H1>Welcome to Cellulose<SMALL> microbial sheets</SMALL></H1></DIV></DIV></DIV><DIV class="container pp"><DIV class="pull-right"><VIDEO poster="https://static.igem.org/mediawiki/2015/a/ad/O_6c8a88797bd8ec61-0.jpg" controls="" width="558" height="316"><P style="font-style:italic;color:red;border-style:solid;border-width:2px;border-color:red">Your browser either does not support HTML5 or cannot handle MediaWiki open video formats. Please consider upgrading your browser, installing the appropriate plugin or switching to a Firefox or Chrome install.</P></VIDEO></DIV><DIV class="row featurette"><DIV class="col-md-7"><H2 class="featurette-heading">Abstract</H2><P class="lead">We used the acetic acid bacterium <I>Gluconacetobacter hansenii </I> to produce bacterial cellulose. Because of its fibrous, tough, water-insoluble properties [1] bacterial cellulose is the perfect substrate for biOrigami. After making the cellulose, we refined it into a flat, paper-like sheet using a DIY paper-making protocol. One unique aspect of cellulose is the existence of a class of proteins known as cellulose binding domains (CBDs) that can attach to a cellulose sheet. We designed a universal CBD (uCBD) that allows for the attachment of any protein onto a cellulose sheet. Using our uCBD device, we can extend our self-folding system to serve additional functions, such a sensor for the detection of inorganic molecules, or binding enzymes to catalyze reactions on the surface of the sheet. We have contributed the BioBricks of our uCBD system to the registry. See our biobricks below: </P><A href="https://2015.igem.org/Team:Stanford-Brown/Parts" class="btn btn-success btn-lg">See our BioBricks</A></DIV></DIV><DIV class="row featurette"><DIV class="col-md-7 col-md-push-5"><H2 class="featurette-heading">Introduction</H2><P class="lead"> We used <I> G. hansenii </I> to produce cellulose as an alternative substrate for our biOrigami. Combine with cellulose binding domain and spore coat protein, cotZ, we could attach spores onto the cellulose surface for self-folding. Lastly, we collaborated with the Edinburgh 2015 IGEM team by providing them with cellulose sheets to test their protein sensors. You can learn more about the collaboration <A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations">here</A></P></DIV><DIV class="col-md-5 col-md-pull-7"><P> Fig. 1 Unprocessed cellulose before drying</P></DIV></DIV><P> Cellulose is a polysaccharide made up of the monomer, B-D glucose, linked by (1-&gt;4) glycosidic bond [1]. It is an important structural component of the plant cell walls and has been used in various textile applications such as paper production [2]. Cellulose is also produced by a Gram-negative bacterium, <I>Gluconacetobacter hansenii</I>, which secretes highly-crystalline cellulose [3]. Because of the few requirements needed to produce bacterial cellulose, <I> G. hansenii </I> is seen as a model organism for the study of cellulose synthesis in plants [3]. We grew a large culture of <I> G. hansenii </I> in growth medium to produce large batch of bacterial cellulose. After retrieving the cellulose from the medium, we used a paper-making protocol to turn them into cellulose sheets (see protocol below). The cellulose would be an important substrate for our biOrigami when combined with the spores to make bioHYDRAS <A href="https://2015.igem.org/Team:Stanford-Brown/Collaborations"> link here </A>. We can use CBDs to attach the spores to the cellulose sheet. This can be accomplished by using a DNA linker connecting the CBD sequence to a protein expressed on the spore coat, Cotz. Since Cotz localizes to the spore coat, the hybrid Cotz-CBD will allow the spores to attach to the sheet. Furthermore, we can expand on the unique properties of this CBD to allow attachment of any protein onto a cellulose surface. We call our CBD system uCBD to note its universal protein attachment abilities. 

    The system is inspired by the <A href="https://2014.igem.org/Team:StanfordBrownSpelman/Cellulose_Cross_Linker"> 2014 Stanford-Brown-Spelman iGEM team </A>; however, it has several improvements. We used the cellulosomal-scaffolding protein A (cipA) of <I> Clostridium thermocellum </I> as our CBD. We chose cipA because of its high cellulose binding affinity (tested by the Imperial 2014 IGEM team); furthermore, we improved on the Imperial cipA by removing the illegal EcoRI site within the gene. CipA is attached to a monomeric streptavidin instead of the wild-type tetramer streptavidin to prevent the chance of protein aggregation, which can inhibit protein function [4]. We thus, sacrifice a small decrease in binding affinity from the monomeric streptavidin for an increase in functional activity of the overall hybrid protein. Together, the monomeric streptavidin and cipa (mSA-cipA) gene made up the cellulose attachment part of our uCBD sytem (Part I). The second part of the system involves molecular cloning any protein of interest onto the bifunctional ligase/repressor (birA) protein attached to an acceptor peptide. BirA is found in <I> E. coli </I> and it catalyzes attachment of biotin onto the biotin acceptor peptide such as the Avitag [5]. Because the acceptor peptide is connected to the protein of interest, this protein can be extracted and purified from the cells and can then attach to Part I via streptavidin-biotin interaction. The overall second construct would be called protein of interest-birA (POI-birA) </P><DIV class="row featurette"><DIV class="col-md-7 col-md-push-5"><H2 class="featurette-heading">Data and Results </H2></DIV><DIV class="col-md-5 col-md-pull-7"><P> Fig. 2 Unprocessed cellulose after drying
        </P></DIV></DIV><P> We made bacterial cellulose using <I> G. hansenii </I> and processed them into flat cellulose sheet. We also designed and cloned the mSA-cipA and POI-birA into NEB-5 alpha competent <I> E. coli </I>. Sequencing result was consistent for mSA-cipA; however, our cloning for POI-birA was unsuccessful. This was because of a PstI site within the gene that caused difficulty in cloning. We also did protein extraction of the mSA-cipA construct; however, the gel  did not show the presence of the protein band. We believed that this was due to a problem with the Flag affinity purification. In conclusion, we made bacterial cellulose and processed it into flat sheet. We also managed to clone and verify the sequence for mSA-cipA. Our future goal is to successfully clone the POI-birA gene and extract proteins from these two genes. Lastly, we will do functional testing by doing binding affinity assay of the cellulose binding domain and the streptavidin-biotin complex. </P><P>Fig. 3 bottom: Processed Cellulose, top: Unprocessed</P><H2>Protocols</H2><H3> Cellulose Sheets </H3><P><B>Cellulose Production</B>: We produced cellulose using <I>Gluconobacter hansenii</I>, the bacterium known to provide the highest cellulose yield (Ross et al, 1991). To produce cellulose from <I>G. hansenii</I>, we used the 2014 Stanford-Brown-Spelman iGEM team’s protocol, as below, with one slight modification – the addition of fructose to promote cellulose production (Embuscado, Marks &amp; BeMiller, 1994).</P><P>Modified <I>Acetobacter xylinum</I> medium recipe (makes 500 mL media, see <A href="http://openwetware.org/wiki/Acetobacter_Xylinum_Culture">original source</A>):</P><UL><LI>25g Fructose</LI><LI>10g Glucose</LI><LI>2.5g Tryptone (or peptone)</LI><LI>2.5g Yeast Extract</LI><LI>1.35g Na2HPO4</LI><LI>0.75g Citric Acid</LI><LI>500mL Water</LI></UL><P>Inoculate with <I>G. hansenii</I> culture and incubate at room temperature for up to one week or until desired volume is achieved.</P><P><B>Troubleshooting tips:</B></P><P>Contamination - When producing cellulose in large vats, contamination by fungi or bacteria can become a major issue.</P><P>Recommendations - </P><UL><LI>Be certain to work with <I>G. hansenii</I> in the hood and to thoroughly seal the vats. 
          </LI><LI>You can also try adding an antifungal that will not act upon bacteria to the medium.
            </LI><LI>The strain of <I>G. hansenii</I> we are using has some resistance to kanamycin, so you could perhaps try adding that. Wild-type bacteria sometimes have natural resistance to certain antibiotics, so we ran a quick experiment to see if <I>G. hansenii</I> is resistant to any of the antibiotics in our lab. There was some growth in the kanamycin plates / cultures – however, it was too slow to merit the addition of kanamycin to all cultures.
              </LI><LI>You could also try transforming the <I>G. hansenii</I> via electroporation (see 2014’s adapted <I>Acetobacter</I> transformation protocol <A href="https://static.igem.org/mediawiki/2014/1/1e/Protocols.pdf">here</A>).
                </LI><LI>If all else fails, you could grow <I>G. hansenii</I> in a large (and we mean LARGE) Erlenmeyer flask. This significantly reduces risk of contamination, although surface area is limited.
                </LI></UL><P>No growth – The cellulose has stopped growing after a while.</P><P>Recommendation - If cellulose is no longer being produced, try adding extra glucose/fructose. Bacteria can’t make cellulose out of thin air!</P><P><B>Cellulose Processing</B>: By solving key issues in cellulose processing, we hope to expand the possibilities of microbial cellulose. We identified brittleness to be the major problem, given that it results in poor flexibility (especially bad for folding) and limited strength of material. Additionally, we theorized that breaking down residual cell components could improve cellulose binding and sheet color. After an investigation spanning from literature reviews to DIY paper making forums, we decided to use glycerol to greatly enhance flexibility and integrity, as well as acetic acid to gently remove extraneous cell structures and pigments from the cellulose.
                </P><P>Here is the protocol we developed to process cellulose sheets:</P><OL><LI>Soak in acetic acid for one hour.</LI><LI>Wash with ddH2O a few times.</LI><LI>Soak in glycerol for one hour.</LI><LI>Separate layers for thinner sheets, if desired. (Applying friction to the sheets with a gloved hand should lead to separation of sheets from each other. If not, and thinner sheets are desired, cut down one end of the sheet and then pull apart.)</LI><LI>Soak in glycerol again briefly (~5 min).</LI><LI>Dry sheets on cardstock and paper towels. Place heavy object on top to flatten sheet and speed absorption.</LI></OL><H3>Cellulose Binding Domains</H3><P><B>Transformation</B>: Chemically and electrocompetent NEB-5 alpha <I> E. coli </I> cells from New England Biolabs were used for all parts of this project. Protocols were followed according to the manufactur's instructions. Plates containing successful transformants were stored at 4C.  <B> Growing Culture </B>. Transformed cells were inoculated into 3 ml Lysogeny Broth (LB) medium supplemented with chloramphenicol and shaken at 37C for 24 hours. Growing culture for protein purification were inoculated in 100 ml LB medium instead. <B> Molecular Cloning </B> The parts were synthesized by IDT and cloning was done with the standard BioBrick cutsites. <B> Flag-tag Protein Purification </B> We used the FLAG octapeptide and Anti-FLAG M2 magnetic beads to extract protein products from the bacteria. Look here for the full <A href="https://static.igem.org/mediawiki/2015/c/c5/M8823bul.pdf"> protocol. </A></P><A href="https://static.igem.org/mediawiki/2015/0/09/SB2015_CelluloseLabNotebook2.pdf" class="btn btn-danger btn-lg">See our Lab Notebook!</A><H2>References</H2><P>[1] O’Sullivan, A.C. 1997. Cellulose: the structure slowly unravels. Cellulose 4(3): 173-207. 
                  [2] Senese, Fred. “What is Cellulose?” General Chemistry Online. N.p., n.d. Web. 18 Sept. 2015. 
                  [3] Lyer et al. 2010. Genome Sequence of a Cellulose-Producing Bacterium, Gluconacetobacter hansenii ATCC 23769. J Bacteriol 192(16): 4256-4257. 
                  [4] Lim et al. 2013. Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnology and Bioengineering 110(1): 57-67. 
                  [5] Kay et al. 2009. High-throughput Biotinylation of Proteins. Methods Mol Biol 498:185-196.
                </P></DIV><FOOTER><DIV class="container"><DIV class="row"><H6>Copyright © 2015 Stanford-Brown iGEM Team</H6></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>