<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jilin_China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jilin China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbox"><DIV class="logo"> </DIV><DIV class="nav"><UL><LI class="drop-menu-effect first"><A href="https://2017.igem.org/Team:Jilin_China"><SPAN>Home</SPAN></A></LI><LI class="drop-menu-effect"><A><SPAN>Project</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Description">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Design">Design</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Model">Model</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Improve">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Lab_Safety">Lab Safety</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Results</SPAN></A><DIV class="submenu" style="left: 4px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Geneguard">Geneguard</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Application">Application</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Experiments</SPAN></A><DIV class="submenu" style="left: 23px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Protocol">Protocol</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/InterLab">Interlab</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Notebook">Notebook</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Parts</SPAN></A><DIV class="submenu" style="left: -25px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Parts">Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Composite_Part">Composite Parts</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Practices</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Human_Practices">Human Practices</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Collaborations">Collaborations</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Team</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Team">Team</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Attributions">Attributions</A></DIV></DIV></DIV></LI></UL></DIV></DIV><DIV class="banner"><DIV class="menu">Design</DIV></DIV><DIV class="sec_box"><DIV class="h1_title">Geneguard: Toxin-Antitoxin (TA) system</DIV><DIV class="h1_content"><P>Toxin-antitoxin (TA) systems are ubiquitous modules existed in bacterial genomes. The transcription of a toxin gene is often coupled with its cognate antitoxin. Toxins are small polypeptides that negatively regulate cellular growth by inhibiting important cellular processes while antitoxins can neutralize them and bring cells back to normal[1].</P><P>TA systems are classified into six groups based on the type of antitoxin and mechanism of how antitoxin blocks toxin activity[2]. We choose the CbeA-CbtA TA pair. CbeA -CbtA is a novel type IV TA system for indirect interaction between toxin and antitoxin. Toxin CbtA can bind and inhibit the polymerization of MerB and FtsZ, leading to effect on cellular morphology and division. Antitoxin CbeA suppresses CbtA toxicity by stabilizing the CbtA-targeted proteins rather than by directly interacting with CbtA to suppress its toxicity. Specifically, CbeA directly binds to both MreB and FtsZ, enhancing the bundling of their filaments <I>in vitro</I>. Notably, this is a unique feature of the CbeA–CbtA system, distinguishing it from all the other TA systems, that toxin CbtA and antitoxin CbeA do not form a complex. Thus, toxicity of CbtA can be considered as reversible, which means even if cellular division has been repressed by CbtA, CbeA can still neutralize CbtA toxicity and enable cellular division.</P><DIV class="pic_box">
				Figure 1. The mechanism of CbeA and CbtA(type IV TA system). CbeA blocks the CbtA toxicity by promoting the assembly of FtsZ and MreB filaments, which is inhibited by the CbtA.
			</DIV></DIV><DIV class="h1_title">Trigger: WT-DmpR &amp; DmpR mutants</DIV><DIV class="h1_content"><P>DmpR, the product of the <I>Pseudomonas sp</I> Strain CF600 <I>dmpR</I> gene[3,4], mediates expression of the <I>dmp</I> operon to allow growth on simple phenols. Pr is a constitutive promoter allowing expression of DmpR. Transcription from P<SUB>0</SUB>, the promoter of the <I>dmp</I> operon, is activated when DmpR detects the presence of an inducing phenol[4]. It depends on a direct physical interaction between the sensor domain of DmpR and the inducing phenol. A productive association between the sensor domain and a phenolic molecule causes DmpR to undergo a conformational change that results in a polymerase-activating form of the protein[5,6], promoting the expression of gene downstream P<SUB>0</SUB>.</P><DIV class="pic_box">
				Figure 2. The mechanism of DmpR sensor.
			</DIV><P>Domain-swapping experiments to form XylR-DmpR hybrids demonstrated that the sensor activity of these regulatory proteins is localized in the amino-terminal region[4]. Modification of the sensor domain should allow the creation of novel proteins that respond to xenobiotics which remain undetected by the wild-type protein. Such altered proteins have the potential to extend the chemical target range of biosensors beyond that based on natural systems. Phenol and substituted phenols are common starting materials and waste by-products in the manufacture of chemical, industrial, and agricultural products. The natural interaction of DmpR with a subset of phenols suggested that modification of its sensor domain might create protein derivatives with the capacity to detect phenolic molecules commonly used in industry.</P></DIV><DIV class="h1_title">DmpR promoter</DIV><DIV class="h1_content"><P>It’s reported that high expression of DmpR would delay cell growth with longer incubation time required to culture the bacterial cells to early stationary phase but did not help to increase the sensitivity towards pollutants[7]. As such, promoters have different promoting strength belongs to Anderson Promoter Collection are in front of <I>dmpR</I> to certify it.</P></DIV><DIV class="h1_title">Enzyme- monooxygenase</DIV><DIV class="h1_content"><P>Naturally, the first two steps in phenolics degradation evolve two enzymes, ortho hydroxyl addition by monooxygenase and cleavage of aromatic ring by dioxygenase.</P><DIV class="pic_box">
				Figure 3. Intial two steps in phenolics degradation.
			</DIV><P>TfdB-JLU is a novel 2,4-dichlorophenol hydroxylase whose amino acid sequence exhibits less than 48% homology with other known TfdBs. The rate-limited step of phenolic degradation is the ortho hydroxylation[8]. Compared to wild-type TfdB, TfdB-JLU has a wilder substrate range and higher catalysis activity. Thus, the enzyme has advantages in efficient disposing of phenolic effluents[8].</P><P>CphA-I and CaO19_12036 are dioxygenase which from <I>A. chlorophenolicus A6</I> and <I>Candida albicans TL3</I> respectively[9,10]. These two enzymes have a similar optimum temperature with TfdB-JLU and have a high catalytic activity towards pyrocatechol.</P></DIV><DIV class="h1_title">Circuit</DIV><DIV class="h1_content"><P>DmpR and toxin are downstream of a constitutive promoter, antitoxin and enzyme are downstream of <I>pdmp</I> operon (Fig 4). When there are no phenolic components in environment, DmpR protein can be ready to activate <I>dmp</I> operon and toxin can be expressed to repress growth of our engineered bacteria. When aromatic substrates appear, DmpR protein can combine <I>pdmp</I> and trigger antitoxin and enzyme expression, leading to toxin neutralization and phenolic degradation, respectively.</P><DIV class="pic_box">
				Figure 4. The design of whole circuit.
			</DIV></DIV><DIV class="h1_title">Reference: </DIV><DIV class="h1_content"><P>[1] Hisako Masuda, Masayori Inouye.Toxins of Prokaryotic Toxin-Antitoxin Systems with Sequence-Specific Endoribonuclease Activity. Toxins, 2017, 9, 140;</P><P>[2] Masuda, Tan. YeeU enhances the bundling of cytoskeletal polymers of MreB and FtsZ, antagonizing the CbtA (YeeV) toxicity in Escherichia coli. Molecular Microbiology, 2012, 84(5), 979-989</P><P>[3] Shingler, V., M. Bartilson, and T. Moore. Cloning and nucleotide sequence of the gene encoding the positive regulator (DmpR) of the phenol catabolic pathway encoded by pVI150 and identification of DmpR as a member of the NtrC family of transcriptional activators. J. Bacteriol. 1993,175: 1596–1604.</P><P>[4] Shingler, V., and T. Moore. Sensing of aromatic compounds by the DmpR transcriptional activator of phenol-catabolizing Pseudomonas sp. strain CF600. J. Bacteriol. 1994, 176:1555–1560.</P><P>[5] Ng, L., E. O’Neil, and V. Shingler. Genetic evidence for interdomain regulation of the phenol-responsive s54-dependent activator DmpR. J. Biol. Chem. 1996, 271:17281–17286.</P><P>[6] Shingler, V., and H. Pavel. Direct regulation of the ATPase activity of the transcriptional activator DmpR by arromatic compounds. Mol. Microbiol.1995, 17:505–513.</P><P>[7] Huiqing Chong and Chi Bun Ching, Development of Colorimetric-Based Whole-Cell Biosensor for Organophosphorus Compounds by Engineering Transcription Regulator DmpR. ACS Synth. Biol. 2016, 5, 1290−1298.</P><P>[8] Yang Lu • Ying Yu • Rui Zhou, Cloning and characterisation of a novel 2,4 dichlorophenol hydroxylase from a metagenomic library derived from polychlorinated biphenyl-contaminated soil. Biotechnol Lett 2011, 33:1159–1167</P><P>[9] Seok H. Lee a, Sun H. Lee, Effective biochemical decomposition of chlorinated aromatic hydrocarbons with a biocatalyst immobilized on a natural enzyme support. Bioresource Technology 141 2013 89–96.</P><P>[10] San-Chin Tsai · Yaw-Kuen Li, PuriWcation and characterization of a catechol 1,2-dioxygenase from a phenol degrading Candida albicans TL3. Arch Microbiol 2007 187:199–206.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>