<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jilin_China_Application skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jilin China/Application</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbox"><DIV class="logo"> </DIV><DIV class="nav"><UL><LI class="drop-menu-effect first"><A href="https://2017.igem.org/Team:Jilin_China"><SPAN>Home</SPAN></A></LI><LI class="drop-menu-effect"><A><SPAN>Project</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Description">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Design">Design</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Model">Model</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Improve">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Lab_Safety">Lab Safety</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Results</SPAN></A><DIV class="submenu" style="left: 4px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Geneguard">Geneguard</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Application">Application</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Experiments</SPAN></A><DIV class="submenu" style="left: 23px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Protocol">Protocol</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/InterLab">Interlab</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Notebook">Notebook</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Parts</SPAN></A><DIV class="submenu" style="left: -25px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Parts">Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Composite_Part">Composite Parts</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Practices</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Human_Practices">Human Practices</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Collaborations">Collaborations</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Team</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Team">Team</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Attributions">Attributions</A></DIV></DIV></DIV></LI></UL></DIV></DIV><DIV class="banner"><DIV class="menu">Application</DIV></DIV><DIV class="sec_box"><DIV class="h1_title">Ⅰ. Overview</DIV><P>Water pollution is a serious problem all over the world. It takes a lot to handle with sewage every year. There are three mostly used methods of sewage treatment:</P><P>- Physical processing: separate and collect water-insoluble impurities in waste water.</P><P>- Chemical treatment: separate or transform some kinds of solutes and colloidal particles into non-hazard chemicals.</P><P>- Biological metabolism: transform some kinds of solutes and colloidal particles into stable non-hazard substances.  </P><P>Biological treatment is one kind of high efficiency and environment friendly method to purify sewage. Since many species of microorganisms could hardly be separated from natural environment and cultured in artificial condition, main focus is put on engineered microorganism.</P><DIV class="h1_title">Ⅱ. Circuit</DIV><P>DmpR and toxin are downstream products of a constitutive promoter, antitoxin and enzyme are downstream products of <I>pdmp</I> operon (Fig 1). When there is are no phenolic components in environment, DmpR protein can be ready to activate <I>dmp</I> operon and toxin can be expressed to repress growth of our engineered bacteria. When aromatic substrates appear, DmpR protein can combine <I>pdmp</I> and trigger antitoxin and enzyme expression, leading to toxin neutralization and phenolic degradation, respectively.
		</P><DIV class="pic_box center">
			Figure 1. The design of whole circuit.
		</DIV><DIV class="h1_title">Ⅲ. DmpR sensor</DIV><P style="text-indent: 0px;"><STRONG>Key achievements</STRONG>
		1. Comparing different promoting strength of several promoters.
		2. Picking out promoter with applicable promoting strength.<STRONG>Introduction</STRONG>  DmpR, the product of the <I>Pseudomonas sp</I>. Strain CF600 <I>dmpR</I> gene, mediates expression of the <I>dmp</I> operon to allow growth on simple phenols[1,2]. Pr is a constitutive promoter allows expression of DmpR. Transcription from P<SUB>0</SUB>, the promoter of the <I>dmp</I> operon, is activated when DmpR detects the presence of inducing phenols[3]. A productive association between the sensor domain and a phenolic molecule causes DmpR to undergo a conformational change that results in a polymerase-activating form of the protein, promoting the expression of gene downstream P<SUB>0</SUB>. It’s reported that high expression of DmpR will delay cell growth with longer incubation time required to culture the bacterial cells to early stationary phase but did not help to increase the sensitivity towards pollutants[4]. As such, promoters with different promoting strength belongs to Anderson Promoter Collection are put in the upstream of <I>dmpR</I> to certify it. We use value of GFP to visualize the expression level of DmpR protein. We put Renilla-luciferase downstream of <I>dmp</I> operon (P<SUB>0</SUB> promoter) to measure the response intensity of DmpR.
		</P><DIV class="pic_box center">
			Figure 2. Circuit for sensor function certification
		</DIV><P style="text-indent: 0px;"><STRONG>Results</STRONG>
		1.J23101 showed strongest promoting activity. </P><P>
		To figure out whether EGFP expression will have effects on bacteria growth, we tested the growth rate of different constructed bacteria containing different promoters. The growing tendency of bacteria expressing EGFP was similar with unloaded vector within 12 hours, which suggested that in 12 hours, the expression of EGFP wouldn’t have effects on the growing of bacteria.</P><P>
		We used the ratio of EGFP and value of OD600 to make comparisons on promoter strength. It turned out that promotor J23101 expressed strongest among all the promotors, and J23107 next. Except for these two promotors, others showed no difference with mock in expression level.</P><DIV class="pic_box center">
			Figure 3. Promoter strength certification. (A) Growth curve of different constructed bacteria. Constructions contained promoter J23101, J23107, J23114, wt-Pr and mock (B) ratio of EGFP/ OD600
		</DIV><P style="text-indent: 0px;">
		2.DmpR downstream of strongest promoter had the greatest sensitivity
	</P><P>
		To probe whether promoter strength will influence DmpR’s sensitivity toward phenolic pollutions, we chose weak promoter J23114, medium promoter J23107 and strong promoter J23101. Phenol, 2-CP, 4-CP and pyrocatechol are selected to test DmpR’s sensitivity. Comparisons are based on ratio of different promotors’ response and mock’s response to same phenolic component respectively.
	</P><P>
		Among three promotors, DmpR downstream of J23107 responded strongest towards phenol, 4-CP and pyrocatechol. Under J23107, DmpR’s response towards phenol was about 1.5 times of pyrocatechol's, and about eight times than 4-CP’s. under J23101, its response towards phenol was lower than pyrocatechol’s, in accordance with J23114. Besides, promotor J23101 and J23107 didn’t show any response to 2-CP. To sum up, promotor J23107 acts best in the response to phenolics.
		</P><DIV class="pic_box center">
			Figure 4. Response intensity of DmpR downstream of promoter with different strength★，p&lt;0.05</DIV><DIV class="h1_title">Ⅳ. Enzyme for phenolics degradation</DIV><P style="text-indent: 0px;"><STRONG>Key achievements</STRONG>
		1. Overexpression and purification of monooxygenase TfdB-JLU
		2. Determination of the substrate preference and activity of TfdB-JLU <I>in vitro</I>
		3. Certification of TfdB-JLU catalytic activity <I>in vivo</I><STRONG>Introduction</STRONG>  Phenolic components cause severe pollution in Yangtze River, Yellow River, Huaihe River and many other rivers in China. Biodegrading has drawn people’s attention over these years. Naturally, the first two steps in phenol degradation evolve two enzymes, ortho hydroxyl addition by monooxygenase and cleavage of aromatic ring by dioxygenase.
		</P><DIV class="pic_box center">
			Figure 5. Initial two steps in phenol degradation</DIV><P>
		Usually, monooxygenases have harrow substract preference. However, this year, the monooxygenase we use is TfdB-JLU, a novel 2,4-dichlorophenol hydroxylase whose amino acid sequence exhibits less than 48% homology with other known TfdBs[5,6]. Compared to wildtype TfdB, TfdB-JLU has a wilder substrate range and higher catalysis activity. To induce expression of TfdB-JLU, we constructed pET28a-JLU expression vector and transformed it into BL21(DE3)-pLysS.
	</P><P>
		Because substract and production of TfdB-JLU share close maximum absorbance, along with TfdB-JLU is a NADPH dependent enzyme, we determined enzyme activity by monitoring the decrease in absorbance at 340 nm (e340 = 6,220 M-1 cm-1) following the substrate-dependent oxidation of NADPH.
	</P><P>
		To certify <I>in vivo</I> activity, we use 4-AAP assay. Phenol can react with 4-AAP in alkaline medium (pH =10.0±0.2), with the oxidizer K<SUB>3</SUB>Fe(CN)<SUB>6</SUB>. Reaction product antipyrine dye appears orange, whose absorption peak is at 510 nm.
	</P><DIV class="pic_box center">
		Figure 6. Mechanism of 4-AAP assay</DIV><STRONG>Experiment Design</STRONG><P>
		pET28a-TfdB-JLU and mock (unloaded pET28a) vector were transformed into BL21. Then digestion assay and sequencing were done to confirm. Several experiments were designed to certify TfdB-JLU’s function, including <I>in vitro</I> and <I>in vivo</I> assay.
	</P><P style="text-indent: 0px;">
		1.Overexpression and purification of TfdB-JLU  To demonstrate the activity and function of TfdB-JLU in virto, the gene tfdB-JLU was recombined with the expression vector pET28a, overexpressed in <I>E. coli</I> BL21(DE3) pLysS with an N-terminal poly histidine tag, and then purified using Ni2+ affinity chromatography. Overexpression and purification protocol had been optimized during several experiments.
	</P><P style="text-indent: 0px;">
		2.Enzyme assay <I>in vitro</I>  To determine the substrate preference of TfdB-JLU, various homologues of 2,4-DCP were tested. The activities of chlorophenol hydroxylases were determined by monitoring the decrease in absorbance at 340 nm (e340 = 6,220 M-1 cm-1) following the substrate-dependent oxidation of NADPH. One unit of activity was defined as the amount of enzyme required to consume 1 μmol NADPH per min at 25℃.
	</P><P style="text-indent: 0px;">
		3.Enzyme assay in vivo  To certify TfdB-JLU have catalytic activity <I>in vivo</I>, HPLC and 4-AAP assay were performed. 
	</P><STRONG>Results</STRONG><P style="text-indent: 0px;">
		1. Overexpression and purification of TfdB-JLU
	</P><P>
		1.1	Digestion assay  To certify succeed in constructing pET28a-TfdB-JLU, digestion assay using BamHI and XbaI was performed. The product digested by BamHI and Xbal was around 1800 bp as predicted 1776 bp of TfdB-JLU.
	</P><DIV class="pic_box center">
		Figure 7. Digestion assay of pET28a-tfdB-JLU</DIV><P>
		1.2	Overexpression and purification of TfdB-JLU  We used optimized protocol to overexpress and purify enzyme, the purified enzyme migrated as a single band with an Mw of 63 kDa on SDS-PAGE, close to the predicted 66.9 kDa Mw of TfdB-JLU plus the 6xHis-tag.
	</P><DIV class="pic_box center"><DIV>
			Purified TfdB-JLU was eluted in 250 mM imidazole elution buffer (lane 7, lane 8)
			Lane 1: crude extract 
			Lane 2: flow through 
			Lane 3: 20 mM imidazole washing buffer 
			Lane 4: 50 mM imidazole washing buffer 
			Lane 5, 6: 100 mM imidazole washing buffer
			Lane 7, 8: 250 mM imidazole elution buffer
			Lane 9: 500 mM imidazole buffer</DIV>
		Figure 8. SDS–PAGE analysis of the expressed TfdB-JLU enzyme
	</DIV><P>
		1.3	substrate activity and preference of TfdB-JLU
		  TfdB-JLU has a wild substract preference among phenolic components. The activities of chlorophenol hydroxylases were determined by monitoring the decrease in absorbance at 340 nm (e340 = 6,220 M-1 cm-1) following the substrate-dependent oxidation of NADPH (Ledger et al. 2006). Unless otherwise indicated, standard enzyme activity assays were performed by incubating the purified enzyme with 0.1 mM 2,4-DCP and 0.2 mM NADPH in 50 mM sodium phosphate buffer (pH 7.5) at 25℃ in 1ml.
	</P><DIV style="padding-left: 26px;">
		Table1. substrate activity of TfdB-JLU<TABLE cellpadding="0" cellspacing="0" width="70%"><TBODY><TR><TH width="30%">substrate</TH><TH width="30%">Relative activity withFAD (%)</TH><TH>Specific activity(U/mg)</TH></TR><TR><TD>3-CP	</TD><TD>311	</TD><TD>1.9144</TD></TR><TR><TD>4-CP	</TD><TD>37	</TD><TD>0.2295</TD></TR><TR><TD>Phenol	</TD><TD>75	</TD><TD>0.4590</TD></TR><TR><TD>2,3-DCP	</TD><TD>110	</TD><TD>0.6717</TD></TR><TR><TD>2,4-DCP	</TD><TD>100	</TD><TD>0.6157</TD></TR><TR><TD>2,5-DCP	</TD><TD>55	</TD><TD>0.3470</TD></TR><TR><TD>2,6-DCP	</TD><TD>143	</TD><TD>0.8788</TD></TR><TR><TD>3,4-DCP	</TD><TD>16	</TD><TD>0.1008</TD></TR><TR><TD>3,5-DCP	</TD><TD>17	</TD><TD>0.1064</TD></TR><TR><TD>2,4,5-TCP	</TD><TD>31	</TD><TD>0.1903</TD></TR></TBODY></TABLE></DIV><P>
		For determining substrate specificity, the enzyme was incubated in 50 mM sodium phosphate buffer, pH 7.5, with 0.1 mM substrate under standard conditions. Relative activity is expressed as a percentage of the maximum enzyme activity towards 2,4-DCP with FAD.
	</P><P style="text-indent: 0px;">
		2. Phenol degradation assay in vivo
	</P><P>
		Among phenolic components, methods in phenol detection are easiest and widely used. Thus, we chose phenol as the substract for degradation experiments in vivo. We used two methods, one is 4-AAP assay, in which phenol can react with 4-AAP in alkaline medium (pH =10.0±0.2), with the oxidizer K3Fe(CN)6. Reaction product antipyrine dye appears orange, whose absorption peak is at 510nm. The other one is HPLC.
	</P><P>
		2.1	HPLC  We used HPLC to demonstrate our engineered bacteria can degrade phenol.
	</P><DIV class="pic_box center">
	Figure 9. HPLC analysis of phenol degradation Phenol-PBS solution before incubation(A)  and after incubation(B)  with engineered bacteria
		Figure 10. Phenol degradation potency in vivo★，p&lt;0.05</DIV><P>
		2.2	4-AAP assay  We used another method to further certify <I>in vivo</I> activity of TfdB-JLU. Firstly, we proved that 4-AAP assay system can be used to test phenol.
	</P><DIV class="pic_box center" style="line-height: 26px;">
		Figure 11. 4-AAP analysis of phenol
		(A) assay system without K3Fe(CN)6 (+ 4-AAP + phenol) 
		(B) assay system with K3Fe(CN)6 (+4-AAP + phenol) Phenol concentration from left to right: 0 mM、1 mM、2 mM、5 mM
	</DIV><DIV class="pic_box center" style="line-height: 26px;">
		Then we certified TfdB-JLU bacteria can degrade phenol after incubation.
		Figure 12. 4-AAP analysis for phenol degradation potency in vivo★，p&lt;0.05</DIV><STRONG>Discussion</STRONG><P>We successfully certified that TfdB-JLU can degrade phenolic components both <I>in vivo</I> and <I>in vitro</I>, but enzyme showed higher activity <I>in vitro</I> than <I>in vivo</I>. There are two possible reasons for this. Firstly, <I>E. coli</I> we use have a complicated intercellular environment, which can affect enzyme activity, also, the way phenol pass the cell wall is free diffusion, which may have lower efficiency in catalyzed by enzyme. Secondly, we found our medium appeared blue after 16 hrs induction. This is because TfdB-JLU can produce indigo through tryptophan, which has indolyl. Thus, for phenol, tryptophan may cause competitive inhibition. The phenomenon that medium appeared blue can further prove TfdB-JLU can work in vivo.</P><DIV class="h1_title">Ⅴ. Future work</DIV><P style="text-indent: 0px;">
		1.	Move circuit into Bacillus or other bacteria used widely for phenolics disposing.
		2.	Artificially constructed an efficient enzyme system which have wide substract range for phenolics degradation.
		3.	Find a novel application for combination with sensor and enzyme like Industrial raw material production.</P><STRONG>Reference：</STRONG><P>[1] V. L. Campos, Detection of Chlorinated Phenols in Kraft Pulp Bleaching Effluents Using DmpR Mutant Strains. Bull. Environ. Contam. Toxicol. 2004 73:666–673</P><P>[2] V. L. Campos, Monitoring Phenolic Compounds During Biological Treatment of Kraft Pulp Mill Effluent Using Bacterial Biosensors. Bull. Environ. Contam. Toxicol. 2006 77:383–390</P><P>[3] ARLENE A. WISE. Generation of Novel Bacterial Regulatory Proteins That Detect Priority Pollutant Phenols. APPLIED AND ENVIRONMENTAL MICROBIOLOGY, 0099-2240/00/$04.0010 Jan. 2000, p. 163–169</P><P>[4] Huiqing Chong, Development of Colorimetric-Based Whole-Cell Biosensor for Organophosphorus Compounds by Engineering Transcription Regulator DmpR. ACS Synth. Biol. 2016, 5, 1290?1298</P><P>[5] Ledger T, Pieper DH, Gonzalez B, Chlorophenol hydroxylases encoded by plasmid pJP4 differentially contribute to chlorophenoxyacetic acid degradation. Appl Environ Microbiol 2006 72:2783–2792</P><P>[6] Yang Lu, Cloning and characterisation of a novel 2,4-dichlorophenol hydroxylase from a metagenomic library derived from polychlorinated biphenyl-contaminated soil. Biotechnol Lett 2011 33:1159–1167</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>