<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Jilin_China_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Jilin China/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbox"><DIV class="logo"> </DIV><DIV class="nav"><UL><LI class="drop-menu-effect first"><A href="https://2017.igem.org/Team:Jilin_China"><SPAN>Home</SPAN></A></LI><LI class="drop-menu-effect"><A><SPAN>Project</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Description">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Design">Design</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Model">Model</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Improve">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Lab_Safety">Lab Safety</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Results</SPAN></A><DIV class="submenu" style="left: 4px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Geneguard">Geneguard</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Application">Application</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Experiments</SPAN></A><DIV class="submenu" style="left: 23px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Protocol">Protocol</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/InterLab">Interlab</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Notebook">Notebook</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Parts</SPAN></A><DIV class="submenu" style="left: -25px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Parts">Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Composite_Part">Composite Parts</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Practices</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Human_Practices">Human Practices</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Collaborations">Collaborations</A></DIV></DIV></DIV></LI><LI class="drop-menu-effect"><A><SPAN>Team</SPAN></A><DIV class="submenu" style="left: 0px;"><DIV class="mj_menu_pro_bg"><DIV><A href="https://2017.igem.org/Team:Jilin_China/Team">Team</A></DIV><DIV><A href="https://2017.igem.org/Team:Jilin_China/Attributions">Attributions</A></DIV></DIV></DIV></LI></UL></DIV></DIV><DIV class="banner"><DIV class="menu">Basic Parts</DIV></DIV><DIV class="thr_box"><STRONG style="color: #229d73;">1.wt-Pr</STRONG><P>wt-Pr is a constitutive promoter which is in the upstream of DmpR, a transcriptional factor of dmp operon from <I>Pseudomonas</I> sp. Strain CF600 encoded by <I>dmp</I>R gene[1]. </P><STRONG style="color: #229d73;">2.cphA-1</STRONG><P>CphA-1, a catechol 1,2-dioxygenase from <I>Arthrobacter chlorophenolicus</I> A6, is responsible for ring cleavage in aromatic compounds degrading process [2].</P><DIV class="pic_box center">
			Figure 2. Reaction of cphA-1
		</DIV><STRONG style="color: #229d73;">3.CaO19</STRONG><P>CaO19, a hydroxyquinol 1,2-dioxygenase from <I>Candida albicans</I> TL3, is responsible for ring cleavage in aromatic compounds degrading process[3].</P><DIV class="pic_box center">
			Figure 3. Reaction of CaO19
		</DIV><STRONG style="color: #229d73;">4.CbtA (toxin)</STRONG><P>CbtA is a protein found in crytic prophage CP4-44 in <I>Escherichia coli</I> K-12. CbtA could inhibit cell division and cell elongation via direct and independent interactions with FtsZ and MreB[4], so it is defined as a kind of toxin.</P><STRONG style="color: #229d73;">5.CbeA (antitoxin)</STRONG><P>CbeA is a protein found in crytic prophage CP4-44 in <I>Escherichia coli</I> K-12 which could suppress the effect of CbtA, so it is defined as a kind of antitoxin. Instead of interacting with CbtA, CbeA directly binds MreB and FtsZ and promotes the assembly of FtsZ and MreB filaments[4].</P><DIV class="pic_box center">
			Figure 4. TA system
		</DIV><P>
			This year we use CbtA and CbeA to build the Geneguard system in our project. For detailed information about toxin/antitoxin (TA) system, <A href="https://2017.igem.org/Team:Jilin_China/Description">please visit ...</A></P><STRONG style="color: #229d73;">6.DmpR</STRONG><P>DmpR is a σ54-dependent transcriptional factor of <I>dmp</I> operon from <I>Pseudomonas</I> sp. Strain CF600.[5] Transcription of Po, promoter of <I>dmp</I> operon, is activated when DmpR detects the presence of certain phenol compounds.[5-6] DmpR directly interacts with proper inducers at its effector-sensing domain, and the effector-sensor compound then binds to Po promoter and downstream transcription is initiated. The DmpR we use is based on the wild type DmpR used by 2013 Peking (<A href="http://parts.igem.org/Part:BBa_K1031211">BBa_K1031211</A>) but has 5 sites of nucleotides mutation[7] of which two are nonsense mutation and others lead to two amino acids change. The mutant type shows <A href="https://2017.igem.org/Team:Jilin_China/Application">high efficiency of transcription initiation</A>.
                </P><DIV class="pic_box center">
			Figure 5. The mechanism of DmpR sensor.
		</DIV><STRONG style="color: #229d73;">7.TfdB-JLU</STRONG><P>TfdB-JLU is a novel 2,4-dichlorophenol hydroxylase whose amino acid sequence exhibits less than 48% homology with other known TfdBs. The rate-limited step of phenolic degradation is the ortho hydroxylation[8]. Compared to wild-type TfdB, TfdB-JLU has a wilder substrate range and higher catalysis activity. Thus, the enzyme has advantages in efficient disposing of phenolic effluents[8].
                </P><DIV class="pic_box center">
			Figure 6. Reaction of TfdB-JLU  
                </DIV><STRONG>Reference：</STRONG><P>[1] Shingler, V., M. Bartilson, and T. Moore. Cloning and nucleotide sequence of the gene encoding the positive regulator (DmpR) of the phenol catabolic pathway encoded by pVI150 and identification of DmpR as a member of the NtrC family of transcriptional activators. J. Bacteriol. ( 1993) 175: 1596–1604.</P><P>[2] Seok H. Lee. Effective biochemical decomposition of chlorinated aromatic hydrocarbons with a biocatalyst immobilized on a natural enzyme support. Bioresource Technology. (2013) 141:89–96.</P><P>[3] Purification and characterization of a catechol 1,2-dioxygenase from a phenol degrading Candida albicans TL3. San-Chin Tsai · Yaw-Kuen Li Arch Microbiol. (2007) 187:199–206.</P><P>[4] Masuda, Tan. YeeU enhances the bundling of cytoskeletal polymers of MreB and FtsZ, antagonizing the CbtA (YeeV) toxicity in Escherichia coli. Molecular Microbiology. (2012) 84(5), 979–989.</P><P>[5] V. L. Campos. Detection of Chlorinated Phenols in Kraft Pulp Bleaching Effluents Using DmpR Mutant Strains Bull. Environ. Contam. Toxicol. (2004) 73:666–673</P><P>[6] V. L. Campos,1 Monitoring Phenolic Compounds During Biological Treatment of Kraft Pulp Mill Effluent Using Bacterial Biosensors Bull. Environ. Contam. Toxicol. (2006) 77:383–390</P><P>[7] ARLENE A. WISE AND CHERYL R. KUSKE*. Generation of Novel Bacterial Regulatory Proteins That Detect Priority Pollutant Phenols. APPLIED AND ENVIRONMENTAL MICROBIOLOGY,0099-2240/00/$04.0010 Jan. 2000, p. 163-169.</P><P>[8] Yang Lu • Ying Yu • Rui Zhou, Cloning and characterisation of a novel 2,4 dichlorophenol hydroxylase from a metagenomic library derived from polychlorinated biphenyl-contaminated soil. Biotechnol Lett 2011, 33:1159–1167</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>