"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Project/Methods
From 2014.igem.org
$~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ \newcommand{\MyColi}{{\small Mighty\hspace{0.12cm}Coli}} \newcommand{\Stabi}{\small Stabi}$ $\newcommand{\EColi}{\small E.coli} \newcommand{\SCere}{\small S.cerevisae}\\[0cm] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ \newcommand{\PI}{\small PI}$ $\newcommand{\Igo}{\Large\mathcal{I}} \newcommand{\Tgo}{\Large\mathcal{T}} \newcommand{\Ogo}{\Large\mathcal{O}} ~$
Example of a hierarchical menu in CSS
carousel slider
Home
Team
Members
Official Profile
Attributions
Human Practice
Introduction
Brussels Game Festival
Synbio Hours
Parts
Previous Years
2012 : InteGreator
2011 : Pindel
2010 : HydroColi
2009 : GluColi
- Université Libre de Bruxelles -
Materials and methods
WetLab
WetLab Structure
The design of the Mighty Coli system requires several intermediate constructions and experiments which will be explained on this page. For the results and the comments of each experiment, see Results .
I.  Search for an efficient 2A peptide
The preliminary version of Mighty Coli consists in 2 plasmids: one constitutively expressed carrying the gene of a toxin, the other inducible and carrying the gene of an antitoxin linked with the gene of the Green Fluorescent Protein (GFP) by the 2A peptide gene (Cf. Fig 1 in Modelling pages).
The 2A peptide sequence allows the post-transcriptional cleavage of 1 ARN sequence into 2 amino-acid sequences: one upstream and one downstream of p2A. This cleavage is done by ribosome skipping within the sequence of p2A. The construction of the peptidic bound between the two last amino-acids of p2A is interrupted, resulting in the termination of the translation of the upstream mRNA sequence. The ribosome can then either stop the translation of the mRNA or continue to translate the downstream sequence into a second, separated protein. The C-terminal extremity of the upstream protein is thus fused with the N-terminal extremity of p2A, and N-terminal extremity of the downstream protein is fused with the last amino-acid of p2A (a prolin).
We thus decided to separate our WetLab in two separate projects: on the first hand, we will try to find a 2A peptide that works in Escherichia Coli (F2A f.e.); on the other hand, we will build the Mighty Coli system in Saccharomyces Cerevisiae using the p2A peptide (P2A f.e.). We will use thus use two different TA Systems: ccdB-ccdA for E.Coli, and Kid-Kis for S.Cerevisiae (respectively Toxin-Antitoxin), as explained in the Introduction page of our project's description.
II. E.Coli chassis
A. Screening of different p2A-like sequences
In order to make an effective screening of different 2A peptides, we will need to design a plasmid containing 2 molecular markers (the red fluorescent protein (RFP) and the alkaline phosphatase (phoA)) separated by a 2A peptide (RFP::p2A::phoA). After cloning this plasmid in bacteria lacking the phoA gene in their genome and after growth on chromogenic and selective XP-medium, we should be able to observe 4 types of results:
1. Colourless colonies and medium
2. Green colonies and colourless medium
3. Colourless colonies and blue medium
4. Green colonies and blue medium.
In the first three cases, p2A would not work properly: there is some problem at the translational or post-translational level (case 1.: for both proteins; case 2.: for GFP only ; case 3.: for phoA only). In the 4th case, p2A would work as expected: GFP is separated from phoA during the translation, and both proteins remain active after the separation.
However, we are only interested in the translational problem, which are linked to the peptide 2A, and not in the post-translational problems, which are linked to the molecular markers we use (RFP and phoA). We must thus design another experiment in order to eliminate the impact that the post-traditional effects could have on the screening.
B. Construction and quantification of the Mighty Coli system
If we find 2A peptide which work within E.Coli, we will build our Mighty Coli system into it. It will be done by PCR amplification (construction of the RFP-p2A-ccdA and ccdB inserts), homologous recombination (ligation of each insert in a vector carrying a different resistance gene), electroporation of the recombinant vectors into E.Coli, and growth on selective medium.
However, in order to have a valid experiment, we must test the effect of the toxin (ccdB) alone on the bacteria – that is, the effect of the toxin without the antitoxin.
III. S.Cerevisiae chassis
A. P2A peptide cleavage rate - Modelling
The modelling team needs to know the cleavage rate of p2A in order to compute the effectiveness of Mighty Coli. It will also give us quantitative expectation of the empiric measurement, which could lead to interesting axis of research if the measurement is too different from the prediction.
It will be done by the construction of a insert linking the p2A to two molecular markers: GFP and RFP (GFP::p2A::RFP). After ligation, electroporation and growth on selective medium, we will thus be able to measure by spectrophotometry the GFP/RFP  ratio (1:1) as well as the production rate of both proteins, which should be a good indicator of those same data in Mighty Coli.
B. Quantification of the Mighty Coli system
The quantitative evaluation of Mighty Coli in S.Cerevisiae will be done in the same way than with E.Coli: we will compare the GFP production yield of a common yeast and the one of a yeast expressing the Mighty Coli system (one plasmid containing the Kid gene, and the other containing the construction GFP::p2A::Kis).
The measurement will be done with the collaboration of F. Delvigne from the ULg.
C. Quality control of the Mighty Coli system
To evaluate the improvement in the quality of the protein production, we will use Apol1 as protein of interest. Indeed, this protein possesses several isoforms, each of them the resulting of a mutation of the original Apol1 gene, and the concentration of each can be easily measured. We will thus compare the relative concentrations of the isoforms of Apol1 produced by a common yeast with those of a yeast expressing the Mighty Coli system (one plasmid containing the Kid gene, and the other containing the construction Apol1::p2A::Kis).
Since all the frameshift mutation affecting the plasmid of containing Apol1 will also disrupt the translation of the antitoxin, we expect the mutated forms of Apol1 to be far less produced by the Mighty Coli yeasts.
IV. Constructions and Biobricks summaries
In order to complete our project, we will have to build 11 recombinant plasmids (6 in E.Coli, 5 in S.Cerevisiae). Each chassis consists in an independent project, which should enable us to complete at least one of them at the end of the summer.
At the end of our project, we should have sent at least 7 biobricks, and maybe more if the screening of the different 2A peptides is positive.
Lab Protocols
Electroporation
Dyalisis (with 0,0250 µm filter) for 20 minutes of 12µl of ligation solution and 12µl of digested plasmid.
Place 50µl of electrocompetent bacteria in an cold electroporation cell (don't touch the electrodes). Inject the dyalisis product into the eletrocompetent cell.
Insert the electrocompetent cell into the electroporation machine, and electroporate at 250V . All is well f there is no spark and the time constant approximates 4,6.
PCR amplification
Miniprep: GenElute™ Plasmid Miniprep Kit
Bacterial cells are harvested via centrifugation, subjected to a modified alkaline-SDS lysis procedure and the DNA adsorbed onto silica in the presence of high salts. Contaminants are then removed by a simple wash step. Bound DNA is eluted in water or Tris-EDTA buffer.
Gel purification: GenElute™ Gel Extraction Kit
The GenElute Gel Extraction Kit combines silica-binding technology with the convenience of a spin or vacuum column format. DNA fragments of interest are extracted from slices of an agarose gel and are bound to a silica membrane. Contaminants are removed by a simple spin or vacuum wash. The bound DNA is then eluted.
The purified DNA is suitable for a variety of downstream applications, such as automated DNA sequencing, PCR, restriction digestion, cloning, and labeling.
Column purification: GenElute™ PCR Clean-Up Kit
The GenElute PCR Clean-Up Kit combines the advantages of silica binding with a microspin format. The DNA is bound on a silica membrane within the spin column. The bound DNA is washed and the clean, concentrated DNA is eluted in the buffer of choice. Each column can purify up to 100 μL or 10 μg of PCR amplified DNA and recover up to 95% of PCR products between 100 bp and 10 kb. More than 99% of the primers and most primer-dimers (<40 bp are removed).
PCR cloning:  Clontech™ In-Fusion HD Cloning Plus
The In-Fusion Enzyme premix fuses PCR-generated sequences and linearized vectors efficiently and precisely, utilizing a 15 bp overlap at their ends. This 15 bp overlap can be engineered by designing custom primers for amplification of the desired sequences. This method can be used to clone single or multiple fragments into a single vector without subcloning.
< Introduction
Results >
Retrieved from " http://2014.igem.org/Team:ULB-Brussels/Project/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
