Team:Jiangnan China/Protocols
Home
Team
Team Members Sponsors Attributions Collaborations
Project
Description Design Experiments Notebook Protocols Results Demonstrate Model
Safety
Training Protection Material
Human Pactices
Silver_Human Pactices Gold_Human Pactices
Awards
Bronzes Silver Gold Prizes
Parts
Basic Parts Composite parts
InterLab
1.  Media component:
    LB medium for E.coli
    1）  Weight the component: 5g/L Yeast Extract. 10g/L NaCl. 10g/L Tryptone
           Note: solid medium: add 20g/L agar
    2） The above materials are fully dissolved by water.
    3） Sterilize the medium for 20 minutes at 121 °C.
    M17 medium
    M17 medium is the commonly used growth medium for L. lactis. This medium is commercially available without carbon source.
    Addition of carbon source for growth:
    0.5% glucose or 0.5% lactose (all strains can grow on glucose; for growth on lactose a strain needs the lactose operon)
    Media and growth conditions for Lactococcus lactis
    Lactococcus lactis vector pNZ8149 harboring lacF as food grade selection marker is selected for the ability to grow on lactose. The host strain L. lactis NZ3900 has all genes involved in the lactose fermentation on the genome, with a deletion of the lacF gene (Enzyme III of the Lac-PTS). The strain can grow on glucose, but in the presence of the lacF gene on a plasmid, it can also grow on lactose.
    A special medium that can be used for selection of Lac+ colonies is Elliker medium. On this rich medium all cells can grow, Lac+ or Lac-, but when lactose is added as sole carbon source the lactose-fermenting cells give yellow colonies.
    Elliker medium
       20 g/l Tryptone
       5 g/l Yeast extract*
       4 g/l Sodium chloride
       1.5 g/l Sodium acetate (water free)
       0.5 g/l L(+) Ascorbic acid
       For agar: 15 g/l agar
       The pH is about 6.8, no adjustment required
       Sterilization: 15 min 121 °C
       After sterilization:
       + 0.5% lactose (stock: 20% solution)
       + 0.004% Bromocresolpurple (0.4% stock solution, filter sterilized)
       * For less background of lactose negative colonies, half the yeast extract concentration can be used.
    Making Agar Plates
       Methods:
       Once agar is molten, mix carefully. Add any other desired additives.
       Pour plates carefully.
       Allow to set for approximately 10 min.
       Invert plates and allow to dry for 20-30 min.
       Store plates at 4°C.
2.  PCR
    1)  Colony PCR
         System: 2×Taq mix:10ul; Forward primer: 1ul; Reverse primer: 1ul; Sterile water: 8ul;
         Total: 20ul;
         Procedure: 95°C 10min; 95°C 30s; (Tm-5)°C 30s; 72°C 1K/min; 72°C 7min; 16°C ∞.
    2)  High fidelity PCR
        System: prime star Buffer: 10ul; dNTP：4ul; forward primer: 1ul; reverse primer: 1ul; primer star enzyme: 0.5ul; template: 100---200ng (calculate the volume according to the concentration of the template); add sterile water to make the total volume reach 50ul;
         Total: 50ul;
         Procedure: 98°C 3min; 98°C 10s; 55°C 15s; 72°C 1K/min; 72°C 10min; 12°C ∞.
    3)  Inverse PCR
        System: prime star Buffer: 10ul; dNTP：4ul; forward primer: 1ul; reverse primer: 1ul; primer star enzyme: 0.5ul; template: 100---200ng (calculate the volume according to the concentration of the template); add sterile water to make the total volume reach 50ul;
         Total: 50ul;
         Procedure: 98°C 3min; 98°C 10s; 55°C 15s; 72°C 1K/min; 72°C 10min; 12°C ∞.
3.  Restriction enzyme cutting reaction
    System: the target gene 40 μl, FD-buffer 5 μl, Xba1 2.5ul, Nco1 2.5 μl.
    37 degree metal bath for 2 hours.
4. Gel extraction
    1) Preparation of agarose glue: 1% agarose gel (six-hole small comb, 0.3g agarose + 30ml 1 × TAE buffer, add 1.5 microliters of golden view dye after boiling).
    2) Agarose gel electrophoresis: DL 5000 marker 8 microliters, FD-DIGEST GREEN BUFFER 100 microliters, restriction fragment solution 5 microliters.
    3) Cut the glue: cut the glue under UV light.(Note: First, the blade should be wiped clean as much as possible. Second, the glue should not be cut as much as possible, and the cutting time under UV light should not be too long.)
    4) Glue extraction: follow the kit operation.
    5) Measure the concentration of the solution and store it in a refrigerator at 4 °C.
5. Ligation
    System: solution1: 5 microliters, vector x microliters, fragment y microliters (x+y=5, x and y should be determined by the concentration of the vector and fragment.)
    16 degrees Celsius metal bath overnight for ligation.
6. One step cloning
    The following reaction system was prepared in an ice water bath. If you accidentally stick the liquid to the tube wall, it can be sunk into the bottom of the tube by brief centrifugation.
    The optimal cloning vector for the ClonExpressTM II recombinant reaction system was 0.03 pmol; the optimal cloning vector to insert fragment molar ratio was 1:2, ie the optimal inserted fragement was used in an amount of 0.06 pmol.
    The DNA quality corresponding to these moles can be obtained by a rough calculation of the following formula:
    Optimum cloning vector usage = [0.02 × cloning vector base pair] ng (0.03 pmol)      Optimal insert usage = [0.04 × insert base pair] ng (0.06 pmol)
      System:
      ddH2O-----------------------------------------------------------Up to 20 μl
      5 × CE II Buffer-------------------------------------------------- 4 μl
      linearized cloning vector----------------------------------------50～200 ng
      Inserted fragment amplification product-----------------------20～200 ng
      ExnaseTM II------------------------------------------------------ 2 μl
    After the system is prepared, use a pipette to gently mix the components several times and avoid air bubbles (do not violently oscillate or vortex). The reaction was carried out at 37 ° C for 30 min. After the reaction is completed, place the reaction tube immediately in an ice water bath for 5 minutes. Thereafter, the reaction product can be directly converted; it can also be stored at -20 ° C and thawed and transformed as needed.
7. Genome Extraction
    1) Sample processing
    Take 1 mL of the overnight cultured bacterial solution, add to a 1.5 mL centrifuge tube, centrifuge at 8000 rpm for 1 hour at room temperature, discard the supernatant, and collect the cells. Add 180ul Buffer Digestion, 37 ° C water bath for 30 to 60 minutes, then add 20ul Proteinase K solution, shake and mix, 56 ° C water bath for 1 hour until the cells are completely lysed.
    2)Add 200ul Buffer BD, mix upside down thoroughly.
    3)Add 200ul of absolute ethanol and mix thoroughly by inverting.
    4)Put the adsorption column into the collection tube, add the solution and the translucent fibrous suspension to the
adsorption column with a pipette, let stand for 2 minutes, centrifuge at 12000 rpm for 1 minute at room temperature, and drain the waste liquid in the collection tube.
    5)Put the adsorption column back into the collection tube, add 500ul PW solution, centrifuge at 10000rpm for 30 seconds, and drain the filtrate from the collection tube.
    6)Put the adsorption column back into the collection tube, add 500ul Wash Solution, centrifuge at 10000rpm for
30 seconds, and drain the filtrate from the collection tube.
    7)Put the adsorption column back into the collection tube, centrifuge at 12000 rpm for 2 minutes, and centrifuge
to remove the residual Wash Solution.
    8)Open the lid of the adsorption column and place it in a 65 ° C metal bath for 7-8 minutes.
    9)Remove the adsorption column, place it in a new clean 1.5 mL centrifuge tube, add 50 μl of deionized water of
pH 8.0 for 3 minutes, centrifuge at 12000 rpm for 2 minutes, and collect the genomic DNA solution.
8. Plasmid Extraction
    Before the Buffer PW is used, add the appropriate amount of ethanol (96-100%) according to the bottle label of the Buffer PW.
    1) Place the spin column CP3 in a clean collection tube and add 500μl of Buffer BL to CP3. Centrifuge at 12,000 rpm
(~ 13,400 x g) for 1 minute in a benchtop microcentrifuge. After discarding the waste in the collection tube, turn the rotary column CP3 back into the collection tube. (Use the rotating column on the day).
    2) Centrifuge for 1 minute at 12,000 rpm (~ 13,400 x g) in a conventional benchtop microcentrifuge at room temperature (15-25°C), 1-5 ml of bacterial cells were harvested in a microcentrifuge tube, and then all traces The supernatant opens the centrifuge tube by reversing until all the media has drained (for large volumes of bacterial cells, collect a tube through several centrifugation steps.)
    3) Use 250μl of Buffer P1 (to ensure that the RNase A has been added) to resuspend the bacteria. The bacteria should be completely resuspended by up or down rotation or blowing until there is no cell mass. Note: After the suspension is suspended again, the cell mass is not visible, otherwise incomplete dissolution will reduce yield and purity.
    4) Add 250 μl of buffer solution P2 and mix gently with 6-8 inverting tubes. NOTE: Mix gently by pouring the tube. Do not whirl, as this will lead to cutting genomic DNA. If necessary, continue to turn the tube until the solution becomes viscous and slightly removed. Do not let the cracking reaction be carried out for more than 5 minutes. If the lysate is still unclear, reduce the bacterial precipitation.
    5) Add 350μl of buffer P3 and mix gently with the tube upside down 6-8 times immediately. The solution should be cloudy. Centrifuge at 12,000 rpm for 10 minutes (~ 13,400 x g) in a benchtop centrifuge. Note: To avoid localized precipitation, the solution is thoroughly mixed immediately after adding buffer P3. If there is still white precipitate in the supernatant, please centrifuge again.
    6) Transfer the supernatant from step 5 to spin column CP3 (place CP3 in the collection tube by decantation or pipetting). Centrifuge at 12,000 rpm (~ 13,400 x g) for 30-60 seconds. Dispose of the circulation and set the rotary column CP3 back to the header.Transfer the supernatant from step 5 to spin column CP3 (place CP3 in the collection tube by decantation or pipetting). Centrifuge at 12,000 rpm (~ 13,400 x g) for 30-60 seconds. Dispose of the circulation and set the rotary column CP3 back to the header.
    7) (Optional, practically we have almost never used) Rotate column CP3 by washing 500μl of buffer PD and centrifuging at
12,000 rpm (~ 13,400 × g) for 30-60 seconds. Release the circulation and return the rotary column CP3 to the collection tube. It is recommended to use endA + strains such as JM series, HB101 and its derivatives or any wild-type strain with high levels of nuclease activity or high carbohydrate content to remove trace nuclease activity.
    8) The rotary column CP3 was rinsed with 600μl of buffer PW (ensuring that ethanol was added (96% -100%)
and centrifuged at 12,000 rpm (~ 13,400 × g) for 30-60 seconds. Release the Spin Column CP3 and return it to the collection.
    9) Repeat step 8.
    10) Centrifuge at 12,000 rpm (~ 13,400 x g) for another 2 minutes to remove residual wash buffer PW. Note that residual
ethanol from buffer PW may inhibit subsequent enzymatic reactions. We recommend opening the CP3 lid and keeping it at room temperature for a period of time to remove residual ethanol.
    11) Place the rotary column CP3 in a clean 1.5 ml microcentrifuge tube. To elute the DNA, 50-100 μl of buffer EB was added to the center of the rotary column CP3 for 2 minutes and centrifuged at 12,000 rpm (~ 13,400 × g) for 2 minutes. Note: If the volume of the elution buffer is less than 50ul, the recovery efficiency may be affected. The pH of the elution buffer has a certain effect on the eluate; buffer EB or distilled water (pH 7.0-8.5) suggests elution of plasmid DNA. For long-term preservation of DNA, it is recommended to elute in buffer EB and store at -20 °C because the DNA stored in water undergo acid hydrolysis. Step 11 is repeated to improve the efficiency of the recovery of the plasmid.
9. Transformation of Lactococcus lactis
    1)  Preparation of the cells:
         Day 1: Inoculate 5 ml of G/L-SGM17B medium with L. lactis glycerol stock from -80 ℃ and grow at 30 ℃,
without aeration, overnight.
         Day 2: Inoculate 50 ml of G/L-SGM17B with pre-culture in a dilution of 1:100 and grow at 30 ℃, without
aeration, overnight.
         Day 3:
- Add 50 ml full-grown culture to 400 ml of G/L-SGM17B medium.
- Grow the culture until OD600 is 0.2-0.3 (ca. 3 h).
- Spin down cells for 20 min at 6000 x g, 4 ℃.
- Wash cells with 400 ml of 0.5 M sucrose, 10% glycerol (4 ℃) and spin down at 6000 x g (centrifugation speed may need to be increased during successive washing steps).
- Resuspend the cells in 200 ml of 0.5 M sucrose, 10% glycerol, 50 mM EDTA (4 ℃), keep the suspension on ice for 15 min and spin down.
- Wash cells with 100 ml of 0.5 M sucrose, 10% glycerol (4 ℃) and spin down (6000 x g) - Resuspend the cells in 4 ml of 0.5 M sucrose, 10% glycerol (4 ℃):
         Use 40 μl per electropopration (keep on ice).
         Or store the cells in small portions at -80 ℃, let them defreeze on ice before use.
    2) Electroporation:
- Place 40 μl cells in a pre-chilled electroporation cuvette with 1 μl DNA (100-500 ng vector DNA reconstituted in TE-, Tris-buffer, or distilled water; for transforming cells with ligation product use 500-1000 ng DNA) and keep the cuvette on ice.
- Use Biorad Genepulser with following adjustments:
2000 V
25 μF
200 Ω
- Pulse (normal reading is 4.5-5 msec).
- Add 1 ml of G/L-M17B + 20 mM MgCl2 + 2 mM CaCl2.
- Keep the cuvette for 5 min on ice and incubate 1-1.5 h at 30 ℃.
- Plate 10 µl, 100 µl, 900 µl on M17agar with glucose or lactose and antibiotics (depends on plasmid).
- Incubate 1-2 days at 30 °C.
10.  Measurement of survival rate
    1) Take 900 μl of bacteria solution in a 1.5mL EP tube, centrifuge and remove the stress medium.
    2) Resuspend the bacteria with 900 μl physiological saline and do it twice.
    3) Take 100 μl of the obtained bacteria solution from the previous step and add it to 900 μl of physiological saline to obtain a bacterial solution with a dilution of 10^(-1). Add 100 μl of the bacteria solution with a dilution of 10^(-1) to 900μl physiological saline, a bacterial solution having a dilution of 10^(-2) was obtained; and so on, the bacteria was diluted to 10^(-6).
    4) Divide an Eliker medium plate into 4 sections; mark the dilution gradient; and take 10 μl of each dilution gradient of the bacterial solution to the corresponding area, 3 samples for each dilution gradient were pointed to the area.
Stress Time
Dilution Gradients of which the samples were pointed to the plate
0h
10^(-3)
10^(-4)
10^(-5)
10^(-6)
0.5h
10^(-2)
10^(-3)
10^(-4)
10^(-5)
1h
10^(-0)
10^(-1)
10^(-2)
10^(-3)
1.5h
10^(-0)
10^(-1)
10^(-2)
10^(-3)
    5) Place the plate in a 30-degree incubator for 12 hours statically. Count the numbers of colonies from the plate and calculate the survival rate.
Copyright © jiangnan_China 2018
