<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_China_Tongji_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:China Tongji/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="head"><DIV class="nav"><DIV class="navOne" id="divHome"><A class="navOneA" id="Home" href="https://2015.igem.org/Team:China_Tongji">Home</A></DIV><DIV class="navOne" id="divProject"><A class="navOneA" id="Project" href="https://2015.igem.org/Team:China_Tongji/Project">Project </A><DIV class="navDrop"><LI class="navTwo" id="navOverview"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Overview">Overview</A></LI><LI class="navTwo" id="navBackground"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Background">Background</A></LI><LI class="navTwo" id="navDesign"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=ProjectDesign">Project Design</A></LI><LI class="navTwo" id="navProtocol"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Protocol">Protocol</A></LI><LI class="navTwo" id="navSummaryResult"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=SummaryResult">Summary and Result</A></LI><LI class="navTwo" id="navDesign"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Design">Design</A></LI><LI class="navTwo" id="navCharacterize"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Characterize">Characterize an existing part</A></LI></DIV></DIV><DIV class="navOne" id="divNotebook"><A class="navOneA" id="Notebook" href="https://2015.igem.org/Team:China_Tongji/Notebook">Notebook </A><DIV class="navDrop"><LI class="navTwo" id="navRecord"><A href="https://2015.igem.org/Team:China_Tongji/Notebook?locationId=Record">Record</A></LI><LI class="navTwo" id="navTimeline"><A href="https://2015.igem.org/Team:China_Tongji/Notebook?locationId=Timeline">Timeline</A></LI></DIV></DIV><DIV class="navOne" id="divAchivement"><A class="navOneA" id="Achivement" href="https://2015.igem.org/Team:China_Tongji/Achivement">Achivement </A><DIV class="navDrop"><LI class="navTwo" id="navParts"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Parts">Parts</A></LI><LI class="navTwo" id="navEquipment"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Equipment">Equipment</A></LI><LI class="navTwo" id="navModeling"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Modeling">Modeling</A></LI><LI class="navTwo" id="navSelfJudgement"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=SelfJudgement">Self Judgement</A></LI><LI class="navTwo" id="navJudgingForm"><A href="https://igem.org/2015_Judging_Form?id=1634" target="_blank">Judging Form</A></LI></DIV></DIV><DIV class="navOne" id="divTeam"><A class="navOneA" id="Team" href="https://2015.igem.org/Team:China_Tongji/Team">Team </A><DIV class="navDrop"><LI class="navTwo" id="navMembers"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Members">Members</A></LI><LI class="navTwo" id="navAdvisor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Advisor">Advisor</A></LI><LI class="navTwo" id="navInstructor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Instructor">Instructor</A></LI><LI class="navTwo" id="navAttributions"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Attributions">Attributions</A></LI><LI class="navTwo" id="navSponsor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Sponsor">Sponsor</A></LI><LI class="navTwo" id="navContact"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Contact">Contact</A></LI></DIV></DIV><DIV class="navOne" id="divOutreach"><A class="navOneA" id="Outreach" href="https://2015.igem.org/Team:China_Tongji/Outreach">Outreach </A><DIV class="navDrop"><LI class="navTwo" id="navHumanPractice"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=HumanPractice">Human Practice</A></LI><LI class="navTwo" id="navCollaboration"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=Collaboration">Collaboration</A></LI><LI class="navTwo" id="navQuestionare"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=Questionare">Questionare</A></LI></DIV></DIV><DIV class="navOne" id="divSafety"><A class="navOneA" id="Safety" href="https://2015.igem.org/Team:China_Tongji/Safety">Safety</A></DIV></DIV></DIV><DIV class="bigName"><P align="left">Project</P></DIV><DIV class="myContent"><DIV id="contentList"><LI class="listOne"><P id="listOverview">1. Overview</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first1').offsetTop)">1.1 Introduction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first2').offsetTop)">1.2 Molecular cloning and micro-injection</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first3').offsetTop)">1.3 Worms testing and tracks recording</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first4').offsetTop)">1.4 Tracks analysis and video edit</LI><LI class="listOne"><P id="listBackground">2. Background</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('second1').offsetTop)">2.1 What is optogenetics?</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('second2').offsetTop)">2.2 Why we use the C.elegans?</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('second3').offsetTop)">2.3 What proteins do we use?</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('second4').offsetTop)">2.4 Reference</LI><LI class="listOne"><P id="listProjectDesign">3. Project Design</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third1').offsetTop)">3.1 Introduction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third2').offsetTop)">3.2 General Design</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third3').offsetTop)">3.3 Plasmid Design</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third4').offsetTop)">3.4 Equipment Design</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third5').offsetTop)">3.5 Test Design</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('third6').offsetTop)">3.6 Reference</LI><LI class="listOne"><P id="listProtocol">4. Protocol</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth1').offsetTop)">4.1 Introduction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth2').offsetTop)">4.2 Taq PCR</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth3').offsetTop)">4.3 Pfu PCR</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth4').offsetTop)">4.4 AGE(agarose gel electrophoresis)</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth5').offsetTop)">4.5 Gel extraction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth6').offsetTop)">4.6 Digestion &amp; ligation</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth7').offsetTop)">4.7 Seamless cloning</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth8').offsetTop)">4.8 Transformation</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth9').offsetTop)">4.9 Plasmid Extraction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fourth10').offsetTop)">4.10	Microinjection</LI><LI class="listOne"><P id="listSummaryResult">5. Summary and Result</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fifth1').offsetTop)">5.1 Plasmids Construction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('fifth2').offsetTop)">5.2 Test Result</LI><LI class="listOne"><P id="listDesign">6. Design</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('sixth1').offsetTop)">6.1 Introduction</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('sixth2').offsetTop)">6.2 Medical treatment potential</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('sixth3').offsetTop)">6.3 Useful tool in scientific research</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('sixth4').offsetTop)">6.4 Conclusion</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('sixth5').offsetTop)">6.5 Reference</LI><LI class="listOne"><P id="listCharacterize">7. Characterize an existing part</P></LI></DIV><DIV id="mainContent"><P class="titleOne" id="Overview">1. Overview</P><P class="titleTwo" id="first1">1.1 Introduction </P><P class="contentP">In our project, we will use the optogenetic technology and the lights of different specific wavelength produced by the light source assembled by ourselves to control the movement of C.elegans and finally construct a movement controlling system.</P><P class="titleTwo" id="first2">1.2 Molecular cloning and micro-injection</P><P class="contentP">We construct the plasmids which are inserted our specific promotors and targeted light-sensitive ion channels genes .The specific promotors such as: AIY, pmyo2, pmyo3 and the opsin such as: ChR2,iC1C2, chETA, Blink are all founded on different papers and websites of worms. Among them, we find ChR2 (<A class="partLink" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K309007" target="_blank">BBa_K309007</A>) has been submitted to iGEM by iGEM10_Queens-Canada. The iGEM10_Queens-Canada focus on establishing some basic parts for later users to use in C.elegans. And we decide to make use of this part, not only establish our own project but also test this part and characterize it. However, we find it being defined as “Sample It's complicated” and is not available in distributions. So we choose to have the part synthesized according to the sequence submitted by iGEM10_Queens-Canada and construct ChR2-YFP. After we construct the plasmids, we inject the plasmids into C.elegans by using the micro-injection technology. By doing that, we can control the behaviours of C.elegans such as moving forwards or twisting more effectively.</P><P class="contentP">What's more,we will express GFP,YFP,mcherry in E.coli. By combining the color of microorgasims and C.elegans, we want to construct some interesting scenes to form a C.elegans' fancy world.</P><P class="titleTwo" id="first3">1.3 Worms testing and tracks recording</P><P class="contentP">We test the C.elegans with the fluorescence microscope . In the testing, we can select the C.elegans in which our target gene has expressed stably. Then, we observe the movement of worms under specific lengh of wave. Finally, we select out the worms which performas expected and recording their tracks in video.</P><P class="contentP">We next change the duration, the wave length and the intensity of the light we use so that we can grope how different conditions influence the movement of C.elegans in the form of table.</P><P class="titleTwo" id="first4">1.4 Tracks analysis and video edit</P><P class="contentP">We analyse the video according to the frame and draw the track lines of each movement. Then we draw the curve graph based on the different conditions and the response of worms. Then, we perfect our video and label the casting part on the worm. </P><P class="contentP">This technology will be helpful in the research on neuron's function and interaction. In the future, this technology may also be used in mechanical controlling system and the theraphy of movement defect.</P><P class="titleOne" id="Background">2. Background</P><P class="titleTwo" id="second1">2.1 What is optogenetics?</P><P class="contentP">Optogenetics involves the use of light to control cells in living tissue, typically <B>neurons</B>, that have been genetically modified to express <B>light-sensitive ion channels</B>. It is a neuromodulation method employed in neuroscience that uses a combination of techniques from optics and genetics to control and monitor the activities of individual neurons in living tissue—even within freely-moving animals—and to precisely measure the effects of those manipulations in real-time.<SUP>[1]</SUP> The rapid progression of such interdisciplinary &quot;optogenetic&quot; approaches has expanded capabilities for optical imaging and genetic targeting of specific cell types.</P><P class="contentP">The key reagents used in optogenetics are <B>light-sensitive proteins</B>. Spatially-precise neuronal control is achieved using optogenetic actuators like channelrhodopsin, halorhodopsin, and archaerhodopsin, while temporally-precise recordings can be made with the help of optogenetic sensors for calcium (Aequorin, Cameleon, GCaMP), chloride (Clomeleon) or membrane voltage (Mermaid).<SUP>[2]</SUP></P><P class="titleTwo" id="second2">2.2 Why we use the C.elegans?</P><P class="contentP">C. elegans(Caenorhabditis elegans) is a small individual,which only has a few cells(959 in the adult hermaphrodite; 1031 in the adult male) and <B>302 neurons</B>. Because of that,C.elegans become one of the <B>simplest organisms</B> with a nervous system. Besides, the body of C.elegans is  transparent and easy to <B>observe</B>. Based on the above, C. elegans is a convenient and effective animal model applied in the optogenetics.</P><P class="contentP">Based on the characteristic of C. elegans,we choose it as our experimental objective.On the one hand, we can easily controll it by casting different waves of light on it .On the other hand, we can also clearly observe it and recort it’s track under the fluorescence microscope.</P><P class="titleTwo" id="second3">2.3 What proteins do we use?</P><P class="contentP">Each opsin protein requires the incorporation of retinal, a vitaminA-related organic photon-absorbing cofactor, to enable lightsensitivity; this opsin-retinal complex is referred to as rhodopsin.The retinal molecule is covalently fixed in the binding pocketwithin the 7-TM helices and forms a protonated retinal Schiffbase (RSBH+; Figure 1) with a conserved lysine residue locatedon TM helix seven (TM7). The ionic environment of the RSBH+,heavily influenced by the residues lining the binding pocket, dictates the spectral characteristics of each individual protein; upon absorption of a photon, the retinal chromophore isomerizes and triggers a series of structural changes leading to iontransport, channel opening, or interaction with signaling transducer proteins.</P><P class="imgIntroduction"><B>Figure 2-1:</B> Light-mediated isomerization of the retinal Schiff base (RSB). Top: retinal in the all-transstate, as found in the dark-adapted state of microbial rhodopsins andin the light-activated forms of type II rhodopsins of higher eukaryotes. The absorption of a photon converts the retinal from the all-transto the 11-cisconfiguration. Bottom: 11-cisretinal is found only in type II rhodopsins, where it binds to the opsin in the dark state before isomerizing to the all-trans position after photonabsorption.</P><P class="contentP">Opsin genes are divided into two distinct superfamilies: microbial opsins (type I) and animal opsins (type II). Bucause we study C.elegans, we only introduction type II here. Type II opsin genes are present only in highereukaryotes and are mainly responsible for vision (Sakmar, 2002). A small fraction of type II opsins also play roles in circadian rhythm and pigment regulation (Sakmar, 2002; Shichidaand Yamashita, 2003). Type II opsins primarily function as Gprotein-coupled receptors (GPCRs) and appear to all use the11-cisisomer of retinal (or derivatives) for photon absorption(Figure 1, bottom)</P><H3>2.3.1 ChRs(ChR2)</H3><P class="contentP">The first known and described ChR, channelrhodopsin-1(ChR1), was identified as a light-gated ion channel inChlamydomonas reinhardtii, a green unicellular alga from temperate freshwater environments (Nagel et al., 2002). ChR1 has broad cation conductance, includingfor Na+,K+, and even Ca2+ions (Lin et al., 2009; Tsunoda andHegemann, 2009). Channelrhodopsin-2 (ChR2),was later characterized from the same organism.Similar to ChR1, ChR2 also conducts cations (Nagel et al.,2003; Tsunoda and Hegemann, 2009), and both ChRs exhibitfast on and off kinetics. When introduced into neurons, ChRscan insert into the plasma membrane and mediate membranepotential changes in response to blue light (Boyden et al.,2005; Ishizuka et al., 2006; Li et al., 2005). </P><P class="contentP">Indeed, the photocycle of ChR2 (Figure 2 and Figure 3(Yizhar et al.,2011b))has different spectral characteristics .In ChR2, adark-adapted state absorbing at 470 nm (D470) converts rapidlyupon illumination to the conducting state P520, via the shortlived photointermediates P500 and P390. Illumination of theopen channel at this step with green light terminates the photocurrent (Bamann et al., 2008; Berndt et al., 2009) by photochemically shifting the channel back into a closed state, which may bethe dark-adapted state D470 or the light-adapted state P480(Stehfest and Hegemann, 2010), effectively resetting the photocycle. This photocycle-shortcut pathway may be relevant only atvery high light intensities with wild-type ChR2.</P><P class="imgName" align="center"><B>Figure 2-2:</B> The working principle of ChR2.</P><P class="imgIntroduction"><B>Figure 2-3:</B> Simplified model for the photocycle of ChRs. The D470 dark state is converted by a light-induced isomerization of retinal via the early intermediate P500 andthe transient P390 intermediate to the conducting-state P520. The recovery of the D470 dark state proceeds either thermally via the nonconducting P480intermediate or photochemically via possible short-lived intermediates (green arrow). The late or desensitized P480 state can also be activated (blue arrow) toyield the early intermediate P500. Additional parallel cycles may be present (Yizhar et al., 2011b)</P><H3>2.3.2 chETA</H3><P class="contentP">Inanother approach addressing both desensitization and deactivation, considering the crystalstructure of BR led to modification of the counterion residue E123 of ChR2 to threonine oralanine; the resulting faster opsin is referred toas ChETA (Gunaydin et al. 2010).</P><P class="contentP">This substitution introduced two advantagesover wild-type ChR2. First, it reduced desensitization during light exposure, with the resultthat light pulses late in high-frequency trainsbecame as likely as early light pulses to drivespikes (a very important property referred to astemporal stationarity).</P><P class="contentP">Second, it destabilizedthe active conformation of retinal, speedingspontaneous isomerization to the inactive stateafter light-off and thus closing the channelmuch more quickly after cessation of light thanwild-type or improved ChR2 variants. Theresulting functional consequences of ChETAmutations are temporal stationarity, reducedextra spikes, reduced plateau potentials, andimproved high-frequency spike followingat 200 Hz or more over sustained trains, even within intact mammalian brain tissue(Gunaydin et al. 2010)</P><P class="imgIntroduction"><B>Figure 2-4:</B> Engineered channelrhodopsin-2 variant with faster deactivation kinetics, resulting in: (1) high-fidelity light-driven spiking over sustained trains at least up to 200 Hz; (2) reduced multiplets and plateau potentials; (3) faster recovery from inactivation, and (4) improved temporal stationarity of performance in sustained trains.</P><H3>2.3.3 iC1C2</H3><P class="contentP">Scientists have designed and characterized aclass of channelrhodopsins (originally cation-conducting) converted into chloride-conductinganion channels. These tools enable fast optical inhibition of action potentials and can beengineered to display step-function kinetics for stable inhibition, outlasting light pulses and fororders-of-magnitude-greater light sensitivity of inhibited cells.</P><P class="contentP">The engineered iC1C2 was designed based on the 2012 crystal structure of C1C2 to conduct chloride ions instead of cations, utilizing physiological chloride gradients to precisely inhibit action potentials in response to blue light. The resulting inhibition is much more light-sensitive than with prior optogenetic inhibitory tools and involves reversible input resistance changes. Light sensitivity of expressing cells is further improved. The channel pore is open and flow of chloride ions across the cell membrane is elevated between the blue and red light pulses, thereby greatly reducing spike probability in expressing neurons without the need for continuous light delivery.</P><P class="imgName" align="center"><B>Figure 2-5:</B> C1C2 structure, with the nineresidues mutated in C1C2_4x and C1C2_5x in orange.</P><P class="imgName" align="center"><B>Figure 2-6:</B> C1C2’s best reaction situation.</P><H3>2.3.4 Blink	</H3><P class="contentP">A blue-light-induced K(+) channel 1 (BLINK1) engineered by fusing the plant LOV2-Jα photosensory module to the small viral K(+) channel Kcv. BLINK1 exhibits biophysical features of Kcv, including K(+) selectivity and high single-channel conductance but reversibly photoactivates in blue light. Opening of BLINK1 channels hyperpolarizes the cell to the K(+) equilibrium potential. Ectopic expression of BLINK1 reversibly inhibits the escape response in light-exposed zebrafish larvae. BLINK1 therefore provides a single-component optogenetic tool that can establish prolonged, physiological hyperpolarization of cells at low light intensities.</P><P class="titleTwo" id="second4">2.4 References</P><P class="reference">[1] Deisseroth, K.; Feng, G.; Majewska, A. K.; Miesenbock, G.; Ting, A.; Schnitzer, M. J. (2006). &quot;Next-Generation Optical Technologies for Illuminating Genetically Targeted Brain Circuits&quot;. Journal of Neuroscience 26 (41): 10380–6. doi:10.1523/JNEUROSCI.3863-06.2006. PMC 2820367. PMID 17035522.</P><P class="reference">[2] Mancuso, J. J.; Kim, J.; Lee, S.; Tsuda, S.; Chow, N. B. H.; Augustine, G. J. (2010). &quot;Optogenetic probing of functional brain circuitry&quot;. Experimental Physiology 96 (1): 26–33. doi:10.1113/expphysiol.2010.055731. PMID 21056968.</P><P class="reference">[3] The Microbial Opsin Familyof Optogenetic Tools; Feng Zhang,Johannes Vierock, Ofer Yizhar, Lief E. Fenno, Satoshi Tsunoda, Arash Kianianmomeni, et al.(2011) Cell147,1446-1457.</P><P class="reference">[4] Lief Fenno,Ofer Yizharand Karl Deisseroth, 2011. The Development andApplication of Optogenetics ; Neurosci34:389–412.</P><P class="reference">[5] http://web.stanford.edu/group/dlab/optogenetics/sequence_info.html.</P><P class="reference">[6] Andre Berndt,Soo Yeun Lee,Charu Ramakrishnan, and Karl Deisseroth (2014); Structure-Guided Transformationof Channelrhodopsin into aLight-Activated Chloride Channel; SCIENCE 344,420-423.</P><P class="reference">[7] http://web.stanford.edu/group/dlab/optogenetics/sequence_info.html.</P><P class="titleOne" id="ProjectDesign">3. Project Design</P><P class="titleTwo" id="third1">3.1 Introduction</P><P class="contentP">In this part, we will illustrate how we designed our project in a <B>Q&amp;A </B>way.To help you understand our project better, this section will be divided into 3 parts: Plasmid Design, Equipment Design and Test Design.</P><P class="titleTwo" id="third2">3.2 General Design</P><H3>3.2.1 Q: WHY choose to control the locomotion of C.elegans?</H3><P class="contentP"><B>A:</B> At first we wanted to make out something FANCY for people can see, so controlling the locomotion of C.elegans become the first choice. Besides, this work has some potential in treating paralyzed animal, even maybe treat people in the future. Right now, there are already researchers successfully made paralyzed mouse move it leg muscle again. </P><P class="contentP">In all kinds of expressions of locomotion, the study of forward and reverselocomotion serves as an entry into understanding theworm’s motor circuit. And we also tried to control the turning of C.elegans. If we can make the worms go forward, go back and turn left or right, we may create something like Snakylines, which is fancy and attractive.</P><H3>3.2.2 Q: HOW to control the locomotion of C.elegans?</H3><P class="contentP"><B>A:</B> At first we have to find neurons or muscles which are related to the movement of C.elegans. Next step is trying to activate or restrain these neurons and muscles by expressing channelrhodopsin2 (ChR2) or its improved versions. As we have illustrated in project background, ChR2 is a channel which is located at the cell membrane. When use appropriate light toirradiate the worm, the particular tissue will be activated or restrained, and then the whole C.elegan will be controlled by the light.</P><P class="titleTwo" id="third3">3.3 Plasmid Design</P><H3>3.3.1 Q: How to express ChR2 at the certainneurons and muscles we want?</H3><P class="contentP"><B>A:</B> We use specific promoter to drive the ChR2 at the specific tissue. Besides, we also tried to use cre-loxp system for 2 promoters which can overlap at one single neuron. This may be a good way to express at single neuron. (Unfortunately, this experiment failed at last.) After reading papers, we choose 5 promoters at last.</P><P class="contentP"><B>Pmyo-2:</B> Encodes a muscle-type specific myosin heavy chain isoform. Myo-2 is expressed in pharyngeal muscle. We supposed that we can use pmyo-2 because it expresses specifically in pharyngeal muscle, which may lead worm turning when irradiated by appropriate light.</P><P class="contentP"><B>Pmyo-3:</B> Encodes MHC A, the minor isoform of MHC (myosin heavy chain) that is essential for thick filament formation, and for viability, movement, and embryonic elongation. Expressed in body muscle, the somatic sheath cell covering the hermaphrodite gonad, and also expressed in enteric muscle, vulval muscles of the hermaphrodite and the diagonal muscles of the male tail(from Wormatlas). 
        We decided to use pmyo-3 to construct a plasmid which can let our ChR2s express in worm’s body muscle which is directly related with worm movements.In this way, we may achieve our purpose.</P><P class="contentP"><B>Pttx-3:</B> Encodes a LIM homeodomain protein required for functions of the interneuron AIY. Expressed at AIY neuron only, in this case the targeted illumination system was used to stimulate AIY only when theworm’s head swung in a particular direction. This work provides new functional evidenceof the chemosensory circuit’s complexity and robustness, and is an example of ‘closed-loop’ optogeneticsstimulation based on behavior.</P><P class="contentP"><B>Pmec-3:</B> Encodes a founding member of the LIM (Lin-11, Isl-1, Mec-3) homeodomain family of transcriptional regulators. During C. elegans development, mec-3 activity is required for proper differentiation and maturation of the mechanosensory neurons. Mec-3 is expressed in the mechanosensory neurons(from Wormatlas). We hope that this may make C.elegans move backward when we irradiate the appropriate light.</P><H3>3.3.2 Q: WHY choose those rhodopsins?</H3><P class="contentP"><B>A: For ChR2: (Excitation)</B></P><P class="contentP">It is the most basic one,and at the same time is the easiest one for us to get. So we choose ChR2 to confirm that our experiment can be completed.</P><P class="contentP"><B>For iC1C2 : (Inhibition)</B></P><P class="contentP">Activated by brief blue light stimulation at low intensities, remains open in the dark for an extended period of time and gets deactivated by red light. Without the need for continuous light delivery.</P><P class="contentP"><B>For chETA:(Excitation)</B></P><P class="contentP">(1) Faster deactivation kinetics;</P><P class="contentP">(2) High-fidelity light-driven spiking over sustained trains at least up to 200 Hz;</P><P class="contentP">(3) Reduced multiples and plateau potentials;</P><P class="contentP">(4) Faster recovery from inactivation, improved temporal stationarity of performance in sustained trains;</P><P class="contentP">(5) Destabilized the active conformation of retinal, speeding spontaneous isomerization to the inactive state after light-off and thus closing the channel much more quickly after cessation of light than wild-type or improved ChR2 variants. </P><P class="titleTwo" id="third4">3.4 Equipment Design</P><H3>3.4.1 Q: Why should we choose LED light sources rather than ordinary light sources?</H3><P class="contentP"><B>A:</B> In this program we use LED light sources instead of using optical filters. </P><P class="contentP">Compared to other light sources, the LED light sources are easier to control. By using C4W cube, we can connect more than two different LEDs in one light path. So it means that we can change the light instantaneously without infecting the observation of our worms. </P><P class="contentP">At the same time, compared to the normal light sources, our light sources’ power is larger, which means that we can have a wider field of vision.</P><P class="contentP">LED has another advantage that LED is instant available, which means we needn’t to wait if we turn off it by accident. We can realize the flash mode (modulate pulse) due to this feature. </P><H3>3.4.2 Q: Why should we refit our LED from 1W to 5W?</H3><P class="contentP"><B>A:</B> The LED which we choose originally is 1W, whose power is larger than ordinary light sources. But we choose change our LED from 1W to 5W, which means that when we testing the reaction of our C.elegens, we can have a wider field of vision. So that we can observe it for a long time which benefit to our analyzation later. </P><P class="contentP">After the refit, we find that the heat dispersion is still very well, which means that it won’t affect the time we use of the LED.</P><H3>3.4.3 Q: Why should we choose DC2100 as our LED driver instead of normal LED driver?</H3><P class="contentP"><B>A:</B> The DC2100 is advanced version of LED driver. It has a current-limiting program to avoid the LED from being damaged. </P><P class="contentP">Compared to ordinary LED drivers, the DC2100 can control the current more accurately, which means that we can test the optimum light intensity to active or repress the worms.</P><P class="contentP">By using DC2100, we can modulate pulse which other LED drivers couldn’t realize.</P><P class="imgName" align="center"><B>Figure 3-1:</B> DC2100 VS traditional LED Driver</P><P class="titleTwo" id="third5">3.5 Test Design</P><H3>3.5.1 Q: why should we standardize our test method?</H3><P class="contentP"><B>A:</B> To evaluate the reaction of these gene modified worms, we find some different aspects to observe them which are the trace, the speed and its angle when the C.elegent makes a turn. So standardize the video is very important for us to analyze the speed and the trace.</P><P class="contentP">So we use 5-10-10 routine to make the video of the worms, so that it can benefit our analyzation later.</P><H3>3.5.2 Q: What is 5-10-10 routine? Why should we use this style?</H3><P class="contentP"><B>A:</B> The 5-10-10 routine means that the first 5 seconds leave the worm in white light, after that give it 10 seconds of LED light, at last leave it in white light for about 10 seconds or more. The 5-10-10 routine is better for us to analyze the speed of those worms. And the first 5 seconds white light is used to observe the normal behavior of the worms which can make comparison to the following period. The third period is use to observe how long the worm can get right. </P><H3>3.5.3 Q: Why should we analyze the trace of the C.elegens?</H3><P class="contentP"><B>A:</B> The trace of the c.elegens is very useful to our project. It can show the movement of these worms visually. We can find the worm keep going or turn left/right or stop even recede during we give the light. After combining the trace with the coordinate, we can change the graphic information into digital information which is easier for us to analyze. </P><H3>3.5.4 Q: What kind of software do we use to record the reaction of worms? Why?</H3><P class="contentP"><B>A:</B> To record the behavior of the worms, we use DP7200 camera and software called Biolife DP to make a video. Compared to those ordinary cameras, DP7200 can change the color temperature of the background. It means that no matter what color the background is, we can always change it to a white background relatively. This can make sure that we can have a high quality video to analyze.</P><H3>3.5.5 Q: Why should we use a red glassine paper to filtering the white light when testing the worms?</H3><P class="contentP"><B>A:</B> The white light contains all kinds of light qualities include the blue light or green light. Using the red glassine paper is to make sure our worms will not infect by the background lights.</P><P class="titleTwo" id="third6">3.6 Reference</P><P class="reference">[1]Steven J. Husson, Alexander Gottschalkand Andrew M. Leifer；Optogenetic manipulation of neuralactivity inC. elegans:Fromsynapseto circuits and behaviour；Biol. Cell (2013)105, 235–250DOI:10.1111/boc.201200069.</P><P class="reference">[2]Andre Berndt, Soo Yeun Lee, Charu Ramakrishnan, and Karl Deisseroth (2014); Structure-Guided Transformation of Channelrhodopsin into a Light-Activated Chloride Channel; SCIENCE 344,420-423.</P><P class="reference">[3]LiefFenno, OferYizhar and Karl Deisseroth, 2011. The Development and Application of Optogenetics ;Neurosci 34: 389–412.</P><P class="titleOne" id="Protocol">4. Protocol</P><P class="titleTwo" id="fourth1">4.1 Introduction</P><P class="contentP">Our project aims to control C.elegans’ movement by expressing chR2 in their muscle and neuron. In our plan, we will make over 20 parts of 3 kinds of channelrhodopsins with 5 different promoters.</P><P class="contentP">First of all, we need to design the PCR primers with primer 5. Then we run taq PCR or pfu PCR to get our parts out of the C.elegans’ genome or plasmids. After that, we do the digestion of gene parts and vector pPD95.77. Use traditional method to do the ligation and transformation. Besides, we also use seamless cloning to deal with some difficult ligations. The last step in molecular construction is plasmid extraction.</P><P class="contentP">Then we come to the C.elegans part, which includes microinjection, making NGM and ATR plates and seeding plates. All the details will show below.</P><P class="titleTwo" id="fourth2">4.2 Taq PCR</P><P class="contentP">(1) Put dNTP, primers, template, taqbuffer and taq enzyme on ice;</P><P class="contentP">(2) Prepare the mix liquid:</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Experimental Material</TH><TH>Dose</TH></TR><TR><TD>Template</TD><TD>1ul</TD></TR><TR><TD>Primer-Front</TD><TD>1ul</TD></TR><TR><TD>Primer-Reverse</TD><TD>1ul</TD></TR><TR><TD>dNTPs</TD><TD>4ul</TD></TR><TR><TD>Taq PCR buffer</TD><TD>5ul</TD></TR><TR><TD>taq enzyme</TD><TD>0.25ul</TD></TR><TR><TD>ddH2O</TD><TD>37.75ul</TD></TR><TR><TD>Total volume</TD><TD>50ul</TD></TR></TBODY></TABLE><P class="contentP">(3) Mix solution well;</P><P class="contentP">(4) Use the PCR machine and amplification the gene:</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Method</TH><TH colspan="2">Time</TH></TR><TR><TD>95℃ pre-denaturation</TD><TD colspan="2">10min</TD></TR><TR><TD>95℃ denaturation</TD><TD>30s</TD><TD rowspan="3">35cycles</TD></TR><TR><TD>60℃ anneal</TD><TD>30s</TD></TR><TR><TD>72℃ extend</TD><TD>1min</TD></TR><TR><TD>4℃ save</TD><TD colspan="2">end</TD></TR></TBODY></TABLE><P class="titleTwo" id="fourth3">4.3 Pfu PCR</P><P class="contentP">(1) Put dNTP, primers, template,pfubuffer andpfuenzyme on ice;</P><P class="contentP">(2) Prepare the mix liquid:</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Experimental Material</TH><TH>Dose</TH></TR><TR><TD>Template</TD><TD>1ul</TD></TR><TR><TD>Primer-Front</TD><TD>2.5ul</TD></TR><TR><TD>Primer-Reverse</TD><TD>2.5ul</TD></TR><TR><TD>dNTPs</TD><TD>5ul</TD></TR><TR><TD>5*Loading Buffer</TD><TD>10ul</TD></TR><TR><TD>Pfu DNA polymerase</TD><TD>1ul</TD></TR><TR><TD>ddH2O</TD><TD>27ul</TD></TR><TR><TD>Total volume</TD><TD>50ul</TD></TR></TBODY></TABLE><P class="contentP">(3) Mix solution well;</P><P class="contentP">(4) Use the PCR machine and amplification the gene:</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Method</TH><TH colspan="2">Time</TH></TR><TR><TD>95℃ pre-denaturation</TD><TD colspan="2">10min</TD></TR><TR><TD>95℃ denaturation</TD><TD>30s</TD><TD rowspan="3">35cycles</TD></TR><TR><TD>60℃ anneal</TD><TD>30s</TD></TR><TR><TD>72℃ extend</TD><TD>1min</TD></TR><TR><TD>4℃ save</TD><TD colspan="2">end</TD></TR></TBODY></TABLE><P class="titleTwo" id="fourth4">4.4 AGE ( agarose gel electrophoresis )</P><P class="contentP">(1) Make gel with 0.5g agarose and 50ml 10X TAE, add 2 drops of EB to dye the gel;</P><P class="contentP">(2) Mix the PCR sample with 10x loading buffer; </P><P class="contentP">(3) 120V 30min;</P><P class="contentP">(4) Use UV light to view the result.</P><P class="titleTwo" id="fourth5">4.5 Gel extraction</P><P class="contentP">(1) Excise the agarose gel slice containing the DNA fragment of interest with a clean, sharp scalpel under ultraviolet illumination. Briefly place the excised gel slice on absorbent toweling to remove residual buffer. Transfer the gel slice to a piece or plastic wrap or a weighing boat. Mince the gel into small pieces and weigh. In this application, the weight of gel is regarded as equivalent to the volume. For example, 100 mg of gel is equivalent to a 100 μl volume. Transfer the gel slice into a 1.5 ml microfuge tube. </P><P class="contentP">(2) Add a 3x sample volume of Buffer DE-A. </P><P class="contentP">(3) Resuspend the gel in Buffer DE-A by vortexing. Heat at 75°C until the gel is completely dissolved (typically, 6-8 minutes) Heat at 40°C if low-melt agarose gel is used. Intermittent vortexing (every 2-3 minutes) will accelerate gel solubilization. </P><P class="contentP">(4) Add 0.5x Buffer DE-A volume of Buffer DE-B, mix. If the DNA fragment is less than 400 bp, supplement further with a 1x sample volume of isopropanol. </P><P class="contentP">Example: For a 1% gel slice equivalent to 100 μl, add the following: </P><P class="contentP">• 300 μl Buffer DE-A </P><P class="contentP">• 150 μl Buffer DE-B </P><P class="contentP">If the DNA fragment is &lt; 400 bp, you would also add: </P><P class="contentP">• 100 μl of isopropanol. </P><P class="contentP">(5) Place a Miniprep column into a 2 ml microfuge tube (provided) Transfer the solubilized agarose from Step 4 into the column. Centrifuge at 12,000xg for 1 minute. </P><P class="contentP">(6) Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 500 μl of Buffer W1. Centrifuge at 12,000xg for 30 seconds.</P><P class="contentP">(7) Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 700 μl of Buffer W2. Centrifuge at 12,000xg for 30 seconds. </P><P class="contentP">(8) Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Add a second 700 μl aliquot of Buffer W2 and centrifuge at 12,000xg for 1 minute. </P><P class="contentP">(9) Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Centrifuge at 12,000xg for 1 minute. </P><P class="contentP">(10) Transfer the Miniprep column into a clean 1.5 ml microfuge tube (provided) To elute the DNA, add 25-30 μl of Eluent or deionized water to the center of the membrane. Let it stand for 1 minute at room temperature. Centrifuge at 12,000xg for 1 minute.</P><P class="titleTwo" id="fourth6">4.6 Digestion &amp; ligation</P><H3>4.6.1 Digestion</H3><P class="contentP">(1) Add enzyme A 1ul and enzyme B 1ul;</P><P class="contentP">(2) Add plasmid 4ul or gene 10ul;</P><P class="contentP">(3) Add buffer 2ul;</P><P class="contentP">(4) Add enough water;</P><P class="contentP">(5) 37℃ 2h;</P><P class="contentP">(6) Do agarose gel electrophoresis;</P><P class="contentP">(7) Gel extraction.</P><H3>4.6.2 Ligase reaction</H3><P class="contentP">(1) Add 1ul ligase;</P><P class="contentP">(2) Add 2ul ligase buffer;</P><P class="contentP">(3) Add 10ul gene that have digested;</P><P class="contentP">(4) Add 3ul digested plasmid;</P><P class="contentP">(5) Add water;</P><P class="contentP">(6) 12℃ 8h.</P><P class="titleTwo" id="fourth7">4.7 Seamless cloning</P><P class="contentP">The design of primers of PCR amplification for cloning of your sequence of interest is based on the same principles as the design of PCR primers for any sequence. The only difference is that simply add the 14-18 bases of vector sequence to the 5’end of your sequence-specific PCR primers when designing primers. After PCR clean up, the resulting PCR- amplified insert is ready for Fast Seamless Cloning.</P><P class="contentP">(1) Digest the vector with two enzymes;</P><P class="contentP">(2) Set up fast seamless gene cloning reaction: 7.5ul seamless cloning enzyme mix with 1ul linearizedvector and 1.5ul gene;</P><P class="contentP">(3) 42℃ 30min;</P><P class="contentP">(4) Transformation.</P><P class="titleTwo" id="fourth8">4.8 Transformation</P><P class="contentP">(1) Get competence E.coli from -80C fridge;</P><P class="contentP">(2) Add 15ul plasmid liquid into competence E.coli, put it in ice water for 15-30min. then give it a 42℃ heat shock for 90 sec. finally put it out of the 42℃ water bath as quick as possible. Put it into ice water for 5 min;</P><P class="contentP">(3) Add 500ul LB into competence E.coli;</P><P class="contentP">(4) At 37℃ we train them for 1h;</P><P class="contentP">(5) Add 100ul into a LB plate which has already added ampicillin. Put them in the 37C incubator for 14h-16h.</P><P class="titleTwo" id="fourth9">4.9 Plasmid Extraction</P><P class="contentP">(1) Put the bacterium liquid in the EP tube, Centrifuge at 12100rpm for 1 minute at room temperature;</P><P class="contentP">(2) Pour out the supernatant as clean as possible;</P><P class="contentP">(3) Add 150 ul P1 to the bacteria sediment, suspend the bacteria in P1 buffer;</P><P class="contentP">(4) Add 150 ul P2, and shake the tube gentle 6-8 times until the liquid become in clear purple color;</P><P class="contentP">(5) Add 350 ul P5, and shake it quickly for 12-20 times;</P><P class="contentP">(6) Centrifuge at 12100rpm for 2 minutes at room temperature;</P><P class="contentP">(7) Transfer 700 ul of the mixture(from Step 6) into a clean DNA Mini Column assembled in a 2ml collection tube(provided) Centrifuge at 12100rpm for 2 min at room temperature to pass solution through column;</P><P class="contentP">(8) Pour out the liquid;</P><P class="contentP">(9) Add 300ul PWT in the column, Centrifuge at 12100rpm for 2 min at room temperature to pass solution through column;</P><P class="contentP">(10) Pour out the liquid and centrifuge again at 12100rpm for 1 min at room temperature;</P><P class="contentP">(11) Place column into a new clean 1.5ml micro-centrifuge tube. Add 50ul 50℃ ddH2O directly onto the column matrix and centrifuge at 12100rpm for 2min to elute DNA;</P><P class="contentP">(12) Exam the OD.</P><P class="titleTwo" id="fourth10">4.10 Microinjection</P><H3>4.10.1 Equipment</H3><P class="contentP">(1) Injection table;</P><P class="contentP">(2) Inverted DIC microscope;( Zeiss Observe.A1)</P><P class="contentP">(3) Micromanipulator;(Zeiss)</P><P class="contentP">(4) Pressurized injection system with needle holder;</P><P class="contentP">(5) Needle puller. Sutter instruments MODEL P-1000 micropipette pullers.</P><H3>4.10.2 Materials</H3><P class="contentP">(1) Microinjection needles;</P><P class="contentP">(2) Injection pads.(Bring 2% agarose in water to a boil, mix well, and place in a heat block. Using a broken Pasteur pipette or a cut-off P200 tip, place a drop (~100ul) of hot agarose onto a #1, 50X22-mm glass coverslip. Quickly place a second coverslip on the drop and lightly tap it. Alternatively, place several drops on the first coverslip, which should merge and mostly cover the surface after adding the second coverslip;)</P><P class="contentP">(3) Injection oil. Series 700 Halocarbon oil;</P><P class="contentP">(4) Worm pick;</P><P class="contentP">(5) M9 buffer;</P><P class="contentP">(6) Worms; (Well-fed, young to middle-aged (≥1day old) gravid hermaphrodites with a full but single row of eggs.)</P><P class="contentP">(7) Needle-loading pipettes. </P><H3>4.10.3 Method</H3><P class="contentP">(1) Fill a needle-loading pipette by capillary action with ≥ 1 ul of DNA injection mix;</P><P class="contentP">(2) Insert the pipette tip in through the back of the injection needle, and expel injection mix onto the needle's internal filament;</P><P class="contentP">(3) Place a loaded needle into the needle holder and mount on the manipulator;</P><P class="contentP">(4) Position the needle so that the tip is in the center of the microscope's field of view using the 5X objective;</P><P class="contentP">(5) Place a drop of oil on an injection pad and place under a dissecting microscope on top of a small Petri plate cover;</P><P class="contentP">(6) Scoop one to several worms from a bacteria-free region of an NGM plate with a naked pick and transfer to the oil drop. Avoid contact with the worm's head. Alternatively, first touch the worm pick to the oil, and use the oil droplet to pick up the animals from a bacteria-free region. The idea is to minimize transfer of bacteria to the pad;</P><P class="contentP">(7) Flame then cool the worm pick and use it to position the worms in the oil drop, and to gently push them down onto the pad. Orient the worms in rows with their ventral sides facing the same direction (opposite the needle direction) If the worms fail to adhere to the pad, move to a new location or rub the bodies with the pick to remove water or bacteria droplets. If adherence is still a problem re-bake the pads or use thicker or higher concentration agarose pads (see above)</P><P class="contentP">(8) Transfer the slide face-up onto the microscope stage. Center the first worm to be injected and focus using the 5X objective. Move the needle down and in close proximity to the dorsal surface of the first animal. Switch to the 40X objective and focus on the worm;</P><P class="contentP">(9) First make sure the needle is flowing;</P><P class="contentP">(10) Insert the needle into the worm;</P><P class="contentP">(11) Inject the DNA solution;</P><P class="contentP">(12) Recover the worms: (Return the coverslip to the dissecting scope, and add a drop (~20 ul) of recovery buffer on the worms.)</P><H3>4.10.4 Preparation of NGM plates</H3><H4>4.10.4.1 Equipment and Reagents</H4><P class="contentP">• NaCl</P><P class="contentP">• Agar</P><P class="contentP">• Peptone</P><P class="contentP">• 5 mg/ml cholesterol in ethanol (Do not autoclave!)</P><P class="contentP">• 1 M KPO4 buffer pH 6.0 (108.3 g KH2PO4, 35.6 g K2HPO4, H2O to 1 litre)</P><P class="contentP">• 1M MgSO4</P><P class="contentP">• Petri plates</P><P class="contentP">• Peristaltic pump</P><H4>4.10.4.2 Methods</H4><P class="contentP">(1) Mix 3 g NaCl, 17 g agar, and 2.5 g peptone in a 2 litre Erlenmeyer flask. Add 975 ml H2O. Cover mouth of flask with aluminium foil. Autoclave for 50 min;</P><P class="contentP">(2) Cool flask in 55°C water bath for 15 min;</P><P class="contentP">(3) Add 1 ml 1 M CaCl2, 1 ml 5 mg/ml cholesterol in ethanol, 1 ml 1 M MgSO4
        and 25 ml 1 M KPO4 buffer.Swirl to mix well;</P><P class="contentP">(4) Using sterile procedures, dispense the NGM solution into petri plates using a peristaltic pump. Fill plates 2/3 full of agar;</P><P class="contentP">(5) Leave plates at room temperature for 2-3 days before use to allow for detection of contaminants, and to allow excess moisture to evaporate. Plates stored in an air-tight container at room temperature will be usable for several weeks.</P><H4>4.10.4.3 Seeding NGM plates</H4><P class="contentP">Using sterile technique, apply approximately 0.05 ml of E. coli OP50 liquid culture to small or medium NGM plates or 0.1 ml to large NGM plates using a pipet. If desired, the drop can be spread using the pipet tip or a glass rod. Spreading will create a larger lawn, which can aid in visualizing the worms. Take care not to spread the lawn all the way to the edges of the plate; keep the lawn in the center. The worms tend to spend most of the time in the bacteria. If the lawn extends to the edges of the plate the worms may crawl up the sides of the plate, dry out and die. Allow the E. coli OP50 lawn to grow overnight at room temperature or at 37°C for 8 hours (cool plates to room temperature before adding worms) Seeded plates stored in an air-tight container will remain usable for 2-3 weeks.</P><H3>4.10.5 Seeding ATR NGM plates</H3><P class="contentP">Add 1.75ul 5uM ATR into 1 ml E.coli OP50 liquid.1ml liquid can seed 10 small NGM plates.</P><P class="titleOne" id="SummaryResult">5. Summary and Result</P><P class="titleTwo" id="fifth1">5.1 Plasmids Construction</P><P class="chartName">Chart 5-1: plasmids we build in our project (We ligate them into PPD95.75 for use in C,elegans)</P><TABLE class="myTable" id="tableParts" align="center" border="3"><TBODY><TR><TH>Promoter/Protein</TH><TH>ChR2-YFP</TH><TH>ChETA-EYFP</TH><TH>ic1c2-TS-EYFP</TH><TH>blink</TH><TH>dsRed</TH></TR><TR><TH>Pmyo-2</TH></TR><TR><TH>Pmyo-3</TH></TR><TR><TH>Pttx-3</TH></TR><TR><TH>Ptwk-16</TH></TR></TBODY></TABLE><P class="chartName">( When you move your mouse on to the tick part, you may see our plasmids’ maps below. )</P><DIV class="equipmentPic"><DIV class="partsPicDiv" id="partsL1R1"><P class="imgWords">pmyo2-chR2-YFP</P></DIV><DIV class="partsPicDiv" id="partsL1R2"><P class="imgWords">pmyo2-chETA-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL1R3"><P class="imgWords">pmyo2-iC1C2-TS-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL1R4"><P class="imgWords">pmyo2-blink</P></DIV><DIV class="partsPicDiv" id="partsL1R5"><P class="imgWords">pmyo2-dsred</P></DIV><DIV class="partsPicDiv" id="partsL2R1"><P class="imgWords">pmyo3-chR2-YFP</P></DIV><DIV class="partsPicDiv" id="partsL2R2"><P class="imgWords">pmyo3-chETA-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL2R3"><P class="imgWords">pmyo3-iC1C2-TS-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL2R4"><P class="imgWords">pmyo3-blink</P></DIV><DIV class="partsPicDiv" id="partsL2R5"><P class="imgWords">pmyo3-dsred</P></DIV><DIV class="partsPicDiv" id="partsL3R1"><P class="imgWords">pttx3-chR2-YFP</P></DIV><DIV class="partsPicDiv" id="partsL3R2"><P class="imgWords">pttx3-chETA-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL3R3"><P class="imgWords">pttx3-iC1C2-TS-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL3R4"><P class="imgWords">pttx3-blink</P></DIV><DIV class="partsPicDiv" id="partsL3R5"><P class="imgWords">pttx3-dsred</P></DIV><DIV class="partsPicDiv" id="partsL4R1"><P class="imgWords">ptwk16-chR2-YFP</P></DIV><DIV class="partsPicDiv" id="partsL4R3"><P class="imgWords">ptwk16-iC1C2-TS-EYFP</P></DIV><DIV class="partsPicDiv" id="partsL4R5"><P class="imgWords">ptwk16-dsred</P></DIV></DIV><P class="titleTwo" id="fifth2">5.2 Test Result</P><P class="contentP">With meticulously designs and precise experiments, we build up the plasmid we want. By using microinjectiontechnology, we get six strains of worms which could stable inheritance. Then we could test their reactions by means of using our refitted microscopeto find out what functions our plasmids have. To improve the quality of the test, we improve our light sources and refit it from 1W to 5W.</P><P class="contentP">Here we show our results. </P><H3>5.2.1 Expression Pattern</H3><CENTER><P class="resultImgName">Figure 5-1: The expression pattern of pmyo2</P><P class="resultImgName">The pymo2 always express in the pharyngeal part and head of the worms.</P></CENTER><CENTER><P class="resultImgName">Figure 5-2: The expression pattern of pmyo3</P><P class="resultImgName">The pymo3 always express in the all parts of the worms.</P></CENTER><H3>5.2.2 Phenotype of each genotype</H3><P class="chartName">Chart 5-2: The phenotype of each genotype</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Pmyo2-ChR2-YFP</TH><TH>Pmyo2-ChETA-EYFP</TH><TH>Pmyo2-ic1c2-EYFP</TH><TH>Pmyo3-ChR2-YFP</TH><TH>Pmyo3-ChETA-EYFP</TH><TH>Pmyo3-ic1c2-EYFP</TH></TR><TR><TH>Speed Change</TH></TR><TR><TH>Direction Change</TH></TR><TR><TH>Bending Degree</TH></TR><TR><TH>Muscle Contraction</TH></TR></TBODY></TABLE><H3>5.2.3 Response index measuring </H3><P class="chartName">Chart 5-3: The response index of each genotype</P><TABLE class="myTable" align="center" border="3"><TBODY><TR><TH>Pmyo2-ChR2-YFP</TH><TH>Pmyo2-ChETA-EYFP</TH><TH>Pmyo2-ic1c2-EYFP</TH><TH>Pmyo3-ChR2-YFP</TH><TH>Pmyo3-ChETA-EYFP</TH><TH>Pmyo3-ic1c2-EYFP</TH></TR><TR><TH>response</TH><TD>67%</TD><TD>38%</TD><TD>26%</TD><TD>92%</TD><TD>42%</TD><TD>47%</TD></TR><TR><TH>no response</TH><TD>33%</TD><TD>62%</TD><TD>74%</TD><TD>8%</TD><TD>58%</TD><TD>53%</TD></TR><TR><TH>relative error</TH><TD>0.025</TD><TD>0.02</TD><TD>0.01</TD><TD>0.015</TD><TD>0.042</TD><TD>0.049</TD></TR></TBODY></TABLE><CENTER><P class="resultImgName">Figure 5-3: The response index of each genotype</P></CENTER><P class="contentP">All these strains are tested with blue LED (470nm, 5W, 1000mA). Comparing to the other strains, the pmyo3-chR2-YFP has the highest expression efficiency. While, having no response doesn’t mean the plasmids that we inject do not work. The expression quantity may be too low, so that the light we use couldn’t stimulate the worms.</P><H3>5.2.4 Behavior changes and track analysis</H3><DIV class="Group" id="ControlGroup"><DIV class="groupNameDiv"><SPAN class="groupName">Control group</SPAN></DIV><H4>5.2.4.1 Lite1 worm without any modified</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/c/c6/China-Tongji-Project-video1-lite1-control.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-4: The track of lite1 worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) This group is lite1 worm which is not sensitive to the lights, while other types of worms make have response to the light. So we choose lite1 worm to be our experimental subject to avoid unnecessary factors.</P><H4>5.2.4.2 Pmyo2-ChR2-YFP worm without ATR</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/2/20/China-Tongji-Project-video9.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-5: The track of pmyo3-ChR2-YFP with blue light (470nm, 5W, 1000mA, fostered without ATR)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) This group we use the mediums without ATR to foster the pmyo3-ChR2-YFP worms. We use this group to find out the effect of ATR. As a result, we find out that the ATR is necessary to our project. Only being fostered in the mediums which have ATR do the worms have response to the lights we gave. In our project, we also set up other control groups to each strain, and they all have the same phenomenon. We choose pmyo3-ChR2-YFP worms as an example for this kind of worm have the highest efficiency.</P><H4>5.2.4.3 Functional worms with green light</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/5/54/China-Tongji-Project-video8.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-6: The track of pmyo3-ChR2-YFP worm given green light (490nm, 3W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) This group we use green light to stimulate the pmyo3-ChR2-YFP worm. By doing this, we try to find out if the worm have response to all kinds of lights or not. At last, we find out that our worms would only have response to blue light (470nm, 5W, 1000mA). In our project, we also set up other control groups to each strain, and they all have the same phenomenon. We choose pmyo3-ChR2-YFP worms as an example for this kind of worm have the highest efficiency.</P></DIV><DIV class="Group" id="Pmyo2Group"><DIV class="groupNameDiv"><SPAN class="groupName">Pmyo2</SPAN></DIV><H4>5.2.4.4 pmyo2-ChR2-YFP </H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/2/2a/China-Tongji-Project-video4-pmyo2-chR2.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-7: pmyo2-ChR2-YFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) Reactions: We test this kind of worms by using 5W LED blue light (470nm) with 1000mA LED driver. When we give light to this kind of worms, we can find some obvious responses. First, after we giving light, the worm would change their direction in about 2 seconds in average. Their reactions are always step back. Secondly if we focus on the movement of their heads, we can find the turning angles change a lot during this time. It means the blue light can stimulate the muscle of their heads and as a result the worm will change the direction. But their behavior will turn to normal at the moment we turn off the light instantaneously. The speed of worm doesn’t have some apparent changes.</P><H4>5.2.4.5 pmyo2-chETA-EYFP</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/c/c2/China-Tongji-Project-video6-pmyo2-chETA.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-8: pmyo2-chETA-EYFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) We test this kind of worms by using 5W LED blue light (470nm) with 1000mA LED driver.When giving the light, this kind of worms has little responses. But you can see the behavior of the head have changed a lot. It means the blue light can still infect the muscle near the head. When the light is on, the behaviors of the worms become stiff compare to the normal worms.  At the same time, after the light is given, you can find the speed of the worm obviously slow down. It is very interested that the worm will stop or even recede when the light is turned off. It means it will take some time for the worm to turn to normal.</P><H4>5.2.4.6 pmyo2-iC1C2-EYFP</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/a/a4/China-Tongji-Project-video5-pmyo3-ic1c2.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-9: pmyo2-iC1C2-EYFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) We test this kind of worms by using 5W LED blue light (470nm) with 100mA LED driver. This kind of worms also has obvious reaction under the blue light. Firstly, you can see an apparent direction change when giving the light. Their reactions are always step back when they are about to changing directions, but the changes are not taken place instantaneously. It means it will be 5-7 seconds later when the worm changes direction. Secondly, you can see they twist their body when stimulated by the light. The turning angles of their head have changed a lot comparing to the normal worms.</P></DIV><DIV class="Group" id="Pmyo3Group"><DIV class="groupNameDiv"><SPAN class="groupName">Pmyo3</SPAN></DIV><H4>5.2.4.7 Pmyo3-chR2-YFP</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/4/49/China-Tongji-Project-video2-pmyo3-chR2.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-10: pmyo3-ChR2-YFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) Reactions: We test this kind of worms by using 5W LED blue light (470nm) with 1000mA LED driver.This kind of worms is one of the highest expression worms among all the strains we have, and its reaction is very obvious. First when the light is on, we can find the whole body of this worm contract, which means the blue light lead to the muscle contraction of the worm. Secondly the worms will stay still until we turn oof the lights. It means that the blue light prevent the worm from moving. When giving the light, the worm will stop move. When the light is off the worm will start move again. These mean taht the speed of the worm is changing. It is very interesting that all these reactions are taken place instantaneous.</P><H4>5.2.4.8 Pmyo3-ChETA-EYFP</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/7/7d/China-Tongji-Project-video3-pmyo3-chETA.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-11: pmyo3-ChETA-EYFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) Reactions: We test this kind of worms by using 5W LED blue light (470nm) with 1000mA LED driver.This kind of worm has apparent response to the blue light. Firstly, when we give it blue light we found their bodies seem to be loss of control. It is remarkable that the blue light affect the muscles of the worm a lot. After the light turn off, it still need about 10 to 20 seconds for the worms to turn to be normal. But there seem to be no obvious changes when we analyze the direction and speed.</P><H4>5.2.4.9 Pmyo3-iC1C2-EYFP</H4><P class="contentP">(1) Video: Please pay attention to the top right corner of the video, the appearance of blue point represents the change from white light to 5W LED blue light (470nm)</P><P class="contentP"><A href="https://static.igem.org/mediawiki/2015/a/a4/China-Tongji-Project-video5-pmyo3-ic1c2.mp4" target="_blank"><SPAN class="linkWords">Video Watching Link</SPAN></A><SPAN style="color:#F00"><B> ( Click the link, please! )</B></SPAN></P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-12: pmyo3-iC1C2-EYFP worm given blue light (470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><P class="contentP">(3) Reactions: We test this kind of worms by using 5W LED blue light (470nm) with 1000mA LED driver.This strain has an obvious reaction to the blue light. When we give light to this kind of worm, we can see it step back in about 2 seconds in average. It means this reaction is fast and instantaneously. It is remarkable that when the light is off, the worm turn to normal rapidly. But the speed and the movement of the worms will not be affected by blue light.</P></DIV><H3>5.2.5 Turning angle measuring</H3><P class="contentP">As we know pmyo2 is express in pharyngeal of C.elegents, so the light will stimulate the head of the worms directly to the head. As a result, observing the movement of their heads is very significative. As we all know, the head of the worm is always shaking, so the turning angle (the angle of each shake) is a very useful data which reflect the response of the head. In this part, we use turning angle of their heads to evaluate the reaction of their head. </P><P class="contentP">We choose pmyo2 worms as our experimental objects in this part. The results are showed below.</P><CENTER><P class="resultImgName">Figure 5-13: The turning angle measuring of pmyo2-ChR2-YFP (using blue light, 470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><CENTER><P class="resultImgName">Figure 5-14: The turning angle measuring of pmyo2-chETA-EYFP (using blue light, 470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><CENTER><P class="resultImgName">Figure 5-15: The turning angle measuring of pmyo2-iC1C2-EYFP (using blue light, 470nm, 5W, 1000mA)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><CENTER><P class="resultImgName">Figure 5-16: The turning angle measuring of pmyo2-ChR2-EYFP (given no light)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER><H4>Analysis:</H4><P class="contentP">From these figures we can get some useful information.</P><P class="contentP">(1) The pmyo2-ChR2-YFP worms have an obvious response. When we give the lights, we can see the amplitude of turning angles become larger. The change of the turning angle becomes drastic.</P><P class="contentP">(2) Compare to the pmyo2-ChR2-YFP worms, the other worm need a long time to be activated. And the time when the worm is activated has become longer.</P><P class="contentP">(3) We use the pmyo2 worms which are given no lights as the control group. We test all the strains of pmyo2 worms, and they have the same reactions. We can see the fluctuation of turning angle is more mild compare to those experimental groups. </P><H3>5.2.6 The relationship between light intensity and response index</H3><P class="contentP">By using DC2100 we can achieve the aim that controling the current of LED accurately. For our LEDs, there is a direct proportion relationship between light intensity and the current which move across it. To test which value is the best to stimulate the C.elegents, we design this part to help us. According to the limitation of DC2100, the largest current we can use is 1000mA. So we pick some worms of all strains which have obvious reactions as our experimental material (using 1000mA to test the reactions before). Because 0mA is needn’t to be tested, we choose to start from 50mA. We pick up 10 worms in each strain to test if it has reactions or not. After 50mA has been tested, we test the 100mA and then 150mA and so on. Until we finish the test of 1000mA, we calculate the ratio of having reactions. Here is the graph we get due to the records.The results are showed below. </P><CENTER><P class="resultImgName">Figure 5-17: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo2-ChR2-YFP worm)</P></CENTER><CENTER><P class="resultImgName">Figure 5-18: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo2-ChETA-EYFP worm)</P></CENTER><CENTER><P class="resultImgName">Figure 5-19: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo2-iC1C2-EYFP worm)</P></CENTER><CENTER><P class="resultImgName">Figure 5-20: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo3-ChR2-YFP worm)</P></CENTER><CENTER><P class="resultImgName">Figure 5-21: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo3-ChETA-EYFP worm)</P></CENTER><CENTER><P class="resultImgName">Figure 5-22: The changing trend of response index with the change of light intensity (blue LED, 470nm, 5W, using pmyo3-iC1C2-EYFP worm)</P></CENTER><H4>From these data, we can find out some conclusions.</H4><P class="contentP">(1) For all these strains, the response index is getting larger with the increase of the current. </P><P class="contentP">(2) For pmyo2-ChR2-YFP and pmyo3-ChR2-YFP, when the current increase to about 600, we can see the worm can be totally activated. </P><P class="contentP">(3) For pmyo2-iC1C2-EYFP and pmyo3-iC1C2-EYFP, when the current increase to about 800, we can see the worm can be totally activated.</P><P class="contentP">(4) For pmyo2-ChETA-EYFP and pmyo3-ChETA-EYFP, when the current increase to about 900, we can see the worm can be totally activated.</P><P class="contentP">(5) For all the strains of worms, 1000mA is the most suitable current. So in our project we use 1000mA blue light (470nm, 5W) to test our worms.</P><H3>5.2.7 Interval flash mode</H3><P class="contentP">As chETA has been optimized in active and recover speed, we also tried internally lighting partten to reappear this character on c.elegans. pmyo2-chETA-eYFP has showed impressive phenomenon.</P><TABLE border="2" class="tableTwoTd"><TBODY><TR><TD><H4>5.2.7.1 lite-1</H4><P class="contentP">(1) Video:</P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-23: lite-1 worm given blue light (470nm, 5W, 1000mA, 10Hz flash)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER></TD><TD><H4>5.2.7.2 pmyo2-chETA-eYFP</H4><P class="contentP">(1) Video:</P><P class="contentP">(2) Track:</P><CENTER><P class="resultImgName">Figure 5-24: pmyo2-chETA-eYFP worm given blue light (470nm, 5W, 1000mA, 10Hz flash)</P><P class="resultImgName">[ <SPAN style="color:#F00"><B>Red points</B></SPAN> represents the track under white light. ]</P><P class="resultImgName">[ <SPAN style="color:#00F"><B>Blue points</B></SPAN> represents the track under blue light (470nm) ]</P></CENTER></TD></TR><TR><TD colspan="2"><H4>Analysis:</H4><P class="contentP">The test is doing under 5W LED blue light (470nm) with 1000mA LED driverand 10 Hz flash.
                    Compared to the control group, pmyo2-chETA-eYFP showed a strong body binding tendency under internal light, mainly caused by the increasing head swing angle, which indicats the capability of chETA. To further explore its unique features, we also compared the results to those worms with same genotype, but given continuous stimulation (Figure 8). We found an enhanced influence on locomotion, as the turn is too sharp that it even changes their heading direction. A possible reason for this difference is that the switch between muscle activation state can be complete in this model. When the channelrhodopsin can’t reach the switching effiency, the tissue turned into a state of platform potential, thus subtle effect can be missing. But with chETA, the quick transforming is expressed.</P></TD></TR></TBODY></TABLE><H3>5.2.8 Muscle contraction </H3><P class="contentP">From the result we get, we find that pmyo3-ChR2-YFP have an obvious response that the muscle of worm contract. So we try to observe the reactions of this kind of worm in high power field, for the light intensity is larger in high power field. As a result, we can find the muscle of worm contract severely, the video is showed below.</P><H3>5.2.9 Surprise</H3><P class="contentP">Click the bug below, there will be a surprise. ( Just for fun! )</P><P class="titleOne" id="Design">6. Design</P><P class="titleTwo" id="sixth1">6.1 Introduction</P><P class="contentP">Our project focuses on using optogenetics technique to control the simple movement of C.elegans. We have successfully expressed different kinds of opsins in different tissues of the worm, and finally controlled it under specific wavelength LED lights. Actually, this technique has much more potential use in medical treatment and scientific research.</P><P class="titleTwo" id="sixth2">6.2 Medical treatment potential</P><H3>6.2.1 Treatment of paralysis</H3><P class="contentP">Right now, we could already make some simple control of C.elegans’ movement, so optogenetics has potential to help the disabled to gain their athletic ability again. After reading papers we found that this work has been done by professor Linda Green Smith, London University. They insert opsins into embryonic stem cell of mouse, and they add signal molecule to let the stem cell grow into neuron which can transport signal between spine and other parts of body. Next, they input these neurons into mouse whose ischiadic nerve has been cut. 5 weeks later, they used blue light to stimulate the nerve and they found that the mouse’s muscle on leg has shrink reaction. So as we can see, this technique will finally be used on human being. At the same time, right now the only way to help patient is using electric to stimulate the muscle, however, this method will bring extra pain to the patient. Once the optogenetics used on paralysis treatment, the stimulation will be more gently so that the patient will feel better.</P><H3>6.2.2 Treatment of neurodegenerative diseases</H3><P class="contentP">Optogenetics is a new thought to treat neurodegenerative diseases, for instance, parkinson’s disease is an illness which has strong impact on human brain health. Prof. Fan Yang’s group expressed chETA at culturing neuroglia. They found that under the stimulation of blue light, the neuron will differentiate to neuron, at the same time, the neuron cells they got has obvious molecular maker which owns by dopaminergic neuron. So we have great chance to use optogenetics into the treatment of neurodegenerative diseases.</P><H3>6.2.3 Treatment of depression</H3><P class="contentP">Last but no least, we know that it’s difficult to cure depression right now. So what if we try to use optogenetics into treatment of depression? One method today to treat depression is using electric stimulation to stimulate neuron in order to make it alive. However, this method may make some damages of brain tissue. With gentle technique like optogenetics, we may find a better way to solute this problem.We can stimulate the dopaminergic neuron and let it secret more dopamine so that the mental condition of patient will be remitted. Besides, there are research shows that when express activated opsins into mice brain, the anxiety behavior will decrease under the irradiation of light. This strongly proved that this technique can be used in treatment of depression one day in the future! </P><P class="titleTwo" id="sixth3">6.3 Useful tool in scientific research</P><H3>6.3.1 Research of neural circuit</H3><P class="contentP">As a quick operate tool, neither will it make extra damage on biological sample like electric stimulation, nor it would appear the inaccurate area like traditional stimulation method. At present, optogenetics can achieve ‘ms’ level optical control on intact mammal neural circuit. This offers wider approaches for research on neuron which has contact with specific cell.</P><H3>6.3.2 Research of animal behavior</H3><P class="contentP">Optogenetics can also be used in research of animal behavior. For instance, researchers use optogenetics to make dopamine secret inside drosophila’s body. This will accompany with obvious behavior. So they can figure out the relationship with the dopaminergic neuron’s actions and drosophila’s behavior, which will be very difficult to use traditional method to make position locate.</P><H3>6.3.3 Research of neuropharmacology experiment</H3><P class="contentP">Traditional neuropharmacology experiments rely on detection of behaviors, which is slow and inaccurate. Optogenetics let us can operate the behavior easier. Like what we have done in our project, we use this technique finding out the specific function of different muscle or neurons. Besides, traditional method should use couple of days to remove the drug completely, while optogenetics only need very little time.</P><P class="titleTwo" id="sixth4">6.4 Conclusion</P><P class="contentP">We can see from the material above that optogenatics has unlimited potential in real life’s medical care. We still need to work on this topic for a long time!</P><P class="titleTwo" id="sixth5">6.5 Reference</P><P class="reference">[1] KlingenbegrM, HuangSG. Structure and function of the uncoupling protein from brown adipose issue, Biochem, Biophy, 1999, 271-296.</P><P class="reference">[2] Study of optogenetics, YiZhang Chen, Frontier.</P><P class="reference">[3] Optogenetics technique can be used on repairing damaged dopaminergic neuron, Dan Wang, Health Daily.</P><P class="reference">[4] Optogenetics technique help regaining the function of paralyzed muscle, Dan Chen.Science and Technology Daily.</P><P class="titleOne" id="Characterize">7. Characterize an existing part</P><P class="contentP"><A href="http://parts.igem.org/Part:BBa_K309007:Experience" target="_blank"><SPAN class="linkWords">BBa_K309007</SPAN></A> (Please click the part number to see more details.)</P><P class="contentP">Since our project also focus on Optogenetics, we want to express channelrhodopsin in C.elegans. There are many kinds of channelrhodopsin now, but we want to start with the most traditional one, ChR1 or ChR2. Then we search in the iGEM database and we find this part submitted by iGEM10_Queens-Canada. The iGEM10_Queens-Canada focus on establishing some basic parts for later users to use in C.elegans. So we decide to make use of this part, not only establish our own project but also test this part and characterize it. However, we find it being defined as &quot;sample It's complicated&quot; by iGEM. What’s more, there are no samples of that part are available in distributions. So we choose to have the part synthesized according to the sequence submitted by iGEM10_Queens-Canada. And we have YFP in our lab in Tongji University, so we use overlapping PCR to link the sequence of YFP and ChR2, thus constructing a fusion protein, ChR2-YFP.</P><P class="contentP">We ligate pmyo2-ChR2-YFP and pmyo3-ChR2-YFP into PPD95.75. Then we micro-inject the PPD95.75-pmyo2-ChR2-YFP and PPD95.75-pmyo3-ChR2-YFP into the C.elegans. All the results of our testing on C.elegans( pmyo2-ChR2-YFP) and C,elegans( pmyo2-ChR2-YFP) are displayed on <A href="http://parts.igem.org/Part:BBa_K309007:Experience" target="_blank"><SPAN class="linkWords">BBa_K309007</SPAN></A>.</P></DIV></DIV><DIV class="foot"><DIV class="navFoot" align="center"><A href="https://2015.igem.org/Team:China_Tongji">Home</A>|<A href="https://2015.igem.org/Team:China_Tongji/Project">Project</A>|<A href="https://2015.igem.org/Team:China_Tongji/Notebook">Notebook</A>|<A href="https://2015.igem.org/Team:China_Tongji/Achivement">Achivement</A>|<A href="https://2015.igem.org/Team:China_Tongji/Team">Team</A>|<A href="https://2015.igem.org/Team:China_Tongji/Outreach">Outreach</A>|<A href="https://2015.igem.org/Team:China_Tongji/Safety">Safety</A></DIV><DIV class="copyright"><CENTER><P id="copyrightText">Copyright © China_Tongji iGEM 2015 | Developer: LiJunWu </P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>