<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_China_Tongji_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:China Tongji/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="head"><DIV class="nav"><DIV class="navOne" id="divHome"><A class="navOneA" id="Home" href="https://2015.igem.org/Team:China_Tongji">Home</A></DIV><DIV class="navOne" id="divProject"><A class="navOneA" id="Project" href="https://2015.igem.org/Team:China_Tongji/Project">Project </A><DIV class="navDrop"><LI class="navTwo" id="navOverview"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Overview">Overview</A></LI><LI class="navTwo" id="navBackground"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Background">Background</A></LI><LI class="navTwo" id="navDesign"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=ProjectDesign">Project Design</A></LI><LI class="navTwo" id="navProtocol"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Protocol">Protocol</A></LI><LI class="navTwo" id="navSummaryResult"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=SummaryResult">Summary and Result</A></LI><LI class="navTwo" id="navDesign"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Design">Design</A></LI><LI class="navTwo" id="navCharacterize"><A href="https://2015.igem.org/Team:China_Tongji/Project?locationId=Characterize">Characterize an existing part</A></LI></DIV></DIV><DIV class="navOne" id="divNotebook"><A class="navOneA" id="Notebook" href="https://2015.igem.org/Team:China_Tongji/Notebook">Notebook </A><DIV class="navDrop"><LI class="navTwo" id="navRecord"><A href="https://2015.igem.org/Team:China_Tongji/Notebook?locationId=Record">Record</A></LI><LI class="navTwo" id="navTimeline"><A href="https://2015.igem.org/Team:China_Tongji/Notebook?locationId=Timeline">Timeline</A></LI></DIV></DIV><DIV class="navOne" id="divAchivement"><A class="navOneA" id="Achivement" href="https://2015.igem.org/Team:China_Tongji/Achivement">Achivement </A><DIV class="navDrop"><LI class="navTwo" id="navParts"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Parts">Parts</A></LI><LI class="navTwo" id="navEquipment"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Equipment">Equipment</A></LI><LI class="navTwo" id="navModeling"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=Modeling">Modeling</A></LI><LI class="navTwo" id="navSelfJudgement"><A href="https://2015.igem.org/Team:China_Tongji/Achivement?locationId=SelfJudgement">Self Judgement</A></LI><LI class="navTwo" id="navJudgingForm"><A href="https://igem.org/2015_Judging_Form?id=1634" target="_blank">Judging Form</A></LI></DIV></DIV><DIV class="navOne" id="divTeam"><A class="navOneA" id="Team" href="https://2015.igem.org/Team:China_Tongji/Team">Team </A><DIV class="navDrop"><LI class="navTwo" id="navMembers"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Members">Members</A></LI><LI class="navTwo" id="navAdvisor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Advisor">Advisor</A></LI><LI class="navTwo" id="navInstructor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Instructor">Instructor</A></LI><LI class="navTwo" id="navAttributions"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Attributions">Attributions</A></LI><LI class="navTwo" id="navSponsor"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Sponsor">Sponsor</A></LI><LI class="navTwo" id="navContact"><A href="https://2015.igem.org/Team:China_Tongji/Team?locationId=Contact">Contact</A></LI></DIV></DIV><DIV class="navOne" id="divOutreach"><A class="navOneA" id="Outreach" href="https://2015.igem.org/Team:China_Tongji/Outreach">Outreach </A><DIV class="navDrop"><LI class="navTwo" id="navHumanPractice"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=HumanPractice">Human Practice</A></LI><LI class="navTwo" id="navCollaboration"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=Collaboration">Collaboration</A></LI><LI class="navTwo" id="navQuestionare"><A href="https://2015.igem.org/Team:China_Tongji/Outreach?locationId=Questionare">Questionare</A></LI></DIV></DIV><DIV class="navOne" id="divSafety"><A class="navOneA" id="Safety" href="https://2015.igem.org/Team:China_Tongji/Safety">Safety</A></DIV></DIV></DIV><DIV class="bigName"><P align="left">Notebook</P></DIV><DIV class="myContent"><DIV id="contentList"><LI class="listOne"><P id="listRecord">1. Record</P></LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first1').offsetTop)">1.1 Plasmid Part</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first2').offsetTop)">1.2 Worm Part</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first3').offsetTop)">1.3 Efficiency Part</LI><LI class="listTwo" onclick="window.scrollTo(0,document.getElementById('first4').offsetTop)">1.4 Equipment Part</LI><LI class="listOne"><P id="listTimeline">2. Timeline</P></LI></DIV><DIV id="mainContent"><P class="titleOne" id="Record">1. Record</P><P class="partTwo" id="first1">1.1 Plasmid Part</P><DIV class="slidePanel slidePanel1"><H3 class="slideTitle" id="slideTitle1dot1">1.1.1 JUNE -- Week 1~4 -- June 6~25<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock1dot1"><H4>1.1.1.1 Week1 -- June 6</H4><H5>June 6</H5><P>1, Design the PCR primers of chR2-YFP.</P><H4>1.1.1.2 Week2 -- June 8~14</H4><H5>June 8</H5><P>1, The amplification of CHR2-YFP (use taq PCR protocol)</P><P>2, AGE ( agarose gel electrophoresis )</P><P>3, Gel extraction of chR2-YFP.</P><P>4, Transformation of GFP, YFP, mcherry in E•coli DH5α.</P><H5>June 9</H5><P>1, Select a single clone of each plate. (GFP, YFP, mcherry) .Put the E.coli in 4ml LB buffer and train for one night at 37℃.</P><P>2, Transformation of vector with pmyo-3(ppd95.77)</P><H5>June 10</H5><P>1, Select a single clone of plate. (pmyo-3,ppd95.77) .Put the E.coli in 4ml LB buffer and train for one night at 37℃.</P><P>2, Plasmid extraction ofE.coli DH5αwith GFP, YFP and mcherry in it.</P><H5>June 11</H5><P>1, Plasmid extraction of pmyo-3(ppd95.77)</P><P>2, Digestion of ppd95.77 with pmyo-3 in it and chR2-YFP using BamHI and EcoRI.(digestion protocol)</P><H5>June 12</H5><P>1, AGE ( agarose gel electrophoresis ) of digested vector---ppd95.77 with pmyo-3.</P><P>2, Gel extraction of pmyo-3.</P><P>3, purification of chR2-YFP witch has been digested with BamHI and EcoRI.</P><H5>June 14</H5><P>1, Transform of GFP, YFP, mcherryin E•coli BL21.</P><H4>1.1.1.3 Week3 -- June 15~21</H4><H5>June 15</H5><P>1, Ligation of pmyo-2 and chR2-YFP. (ligation protocol)</P><P>2, Transformation of ligation product: pmyo3-chR2-YFP (in ppd95.77).</P><H5>June 17</H5><P>1, Select a single clone of plate. (pmyo-3-chR2-YFP, ppd95.77) .Put the E.coli in 4ml LB buffer and train for one night at 37℃.</P><P>2, Plasmid extraction of pmyo3-chR2-YFP.</P><H5>June 19</H5><P>1, Digestion of new plasmid: pmyo3-chR2-YFP using HindIII and EcoRI to check if the ligation step work or not. (it worked! Our first part is done!)</P><H5>June 20</H5><P>1, Transformation of pmyo3-chR2-YFP in order to get more plasmid.</P><P>2, Transformation of pmec3-chR2-YFP, pmec3-dsred and pmec4-chR2-YFP which are offered by professor Li’s lab.</P><P>3, Save the E.coli strain with glycerinum.</P><H5>June 21</H5><P>1, Theamplification of dsred and pmyo-2(use taq PCR protocol---to test the best temperature for the PCR).</P><P>2, the amplification of dsred and pmyo-2(use pfu PCR protocol).</P><P>3, AGE ( agarose gel electrophoresis ) of pmyo-2 and dsred.</P><P>4, Gel extraction of pmyo-2 and dsred.</P><H4>1.1.1.4 Week4 -- June 22~25</H4><H5>June 22</H5><P>1, digestion of pmyo3-chR2-YFP(using HindIII and BamHI)</P><P>2, AGE ( agarose gel electrophoresis ) of digested vector---ppd95.77 with chR2-YFP.</P><P>3, Gel extraction of pmyo-3(PPD95.77).</P><P>4, Digest of pmyo2 with HindIII and BamHI. Digest of dsred with BamHI and EcoRI. </P><P>5, gene purification for pmyo2 and dsred.</P><H5>June 24</H5><P>1, Ligation of pmyo2 with chR2-YFP (in ppd95.77).</P><P>2, Ligation of dsred with pmyo3 and pmyo2(in ppd95.77)</P><H5>June 25</H5><P>1, Digest of pmyo2-chR2-YFP and pmyo2-dsred and pmyo3-dsred.(usingHindIII and EcoRI) To check if we had ligated it successfully.</P><P>2, AGE ( agarose gel electrophoresis ) of digested products. Analyze the result.</P></DIV></DIV><DIV class="slidePanel slidePanel1"><H3 class="slideTitle" id="slideTitle1dot2">1.1.2 JULY -- Week 2~5 -- July 11~30<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock1dot2"><H4>1.1.2.1 Week2 -- July 11~14</H4><H5>July 11</H5><P>1, Design the PCR primers of Blink gene.</P><P>2, Design the PCR primers of pttx-3.</P><P>3, Design the PCR primers of ptrp4.</P><P>4, Prepare for competent cells</P><H5>July 14</H5><P>1, Transformation of the Blink plasmid which was kind offered by Anna’s lab.</P><P>2, Transformation of pmyo2-chR2-YFP, pmyo2-dsred and pmyo3-dsred in order to get more plasmids.</P><P>3, GFP, YFP, mcherry transform OP50 and PA14. (OP50 and PA14 are the food of C.elegans)</P><H4>1.1.2.2 Week3 -- July 15~21</H4><H5>July 15</H5><P>1,Select single clones of plate. (pmyo-2-chR2-YFP, pmyo2-dsred and pmyo3-dsred ppd95.77) . Put the E.coli in 4ml LB buffer and cultivate for one night at 37℃.</P><H5>July 16</H5><P>1, Plasmid extraction of pmyo-2-chR2-YFP, pmyo2-dsred and pmyo3-dsred.</P><H5>July 17</H5><P>1, Recovery of </P><P><B>pNP260( Pnmr-1::flox::ChR2::mCherry.)</B></P><P><B>pCoS2(pnhr-79::Cre):</B></P><P><B>pCoS13(posm-10::loxP::LacZ::STOP::loxP:: ChR2::mCherry::SL2::GFP)</B></P><P><B>pSH116 (pdes-2::Cre)</B></P><P>which are kind offered by Alexander Gottschalk’ lab. </P><P>2, Transformation of pNP260, pCoS2, pCoS13 and pSH116.</P><P>3, Design of the PCR primers of pttx-HindIII-F and pttx3-Xbal-R.</P><H5>July 18</H5><P>1, Select single clones of plate.(pNP260, pCoS2, pCoS13 and pSH116). Put the E.coli in 4ml LB buffer and cultivate for 12h at 37℃.</P><P>2, Plasmid extraction.</P><P>3, Make genome DNA from worms.</P><P>4, Make LB plate and LB liquid.</P><P>5, Repeat the experiment: Prepare for competent cells: GFP, YFP, mcherry transform OP50 and PA14.</P><P>6, The amplification of Blink.</P><P>7, AGE ( agarose gel electrophoresis ) of blink PCR products. Analyze the result.</P><P>8, Gel extraction. (all at 300ng/ul)</P><H5>July 19</H5><P>1, Use the C.elegans genomic DNA as template and got pttx-3 with the help of Pttx3-HindIII-F and pttx-3-XbaI-R.</P><P>2, AGE ( agarose gel electrophoresis ) of PCR products. Analyze the result.</P><H5>July 20</H5><P>1, Transformation of P Blue plasmid.(used to make the mixture of microinjection liquid)</P><P>2, Select single clone on the culture plate.Put the E.coli in 4ml LB buffer and cultivate for 12h at 37℃.</P><P>3, Select a single clone of each plate and put each one in a tube that contain the ampicillin and 1ml LB culture media. (OP50 and PA14)</P><P>4, At 37℃ we culture them for 12h.</P><H5>July 21</H5><P>1, Add the mixture to a new 15ml tube and add in 4ml LB culture media.</P><P>2, Culture them for 3h until the OD number near 0.8.</P><P>3, Add 1mg IPTG in the liquid.</P><P>4, Culture them for 3h in order to gain the protein.</P><P>5, Plasmid extraction of P blue.</P><P>6, Digestion: Use EcoR I and BamH I digest the pmyo2 and pmyo3 plasmid and the Blink. (use digestion protocol)</P><P>7, Ligase reaction. (use ligation protocol)( pmyo2-blink,pmyo3-blink)</P><H4>1.1.2.3 Week4 -- July 22~27</H4><H5>July 22</H5><P>1, Design the PCR plasmid of chETA and ic1c2 which we bought from addgene.</P><P>2, Cultivation of plasmid chETA and ic1c2 from Addgene.Pick a loop of bacteria from the sample, streaking on Amp+ plates, cultivate in 37℃ for 12h.</P><P>3, Transform DH5α(pmyo2-blink,pmyo3-blink). Label the plate and put it in the incubator about one night, 37℃.</P><H5>July 23</H5><P>1, Amplify of plasmid chETA and ic1c2 from Addgene.Pick 5 single clone from each plate, add in 4ml LB medium separately, and shaking in 37℃ for 14h.</P><P>2, Make LB plate.</P><P>3, Select a single clone and culture for 12h of each template. (pmyo2-blink,pmyo3-blink)</P><H5>July 24</H5><P>1, Plasmid extraction of plasmid chETA and ic1c2. (the results are all around 100ng/ul)</P><P>2, Taq PCR of chETA and ic1c2. (to test the best reaction tempareture)</P><P>3, AGE ( agarose gel electrophoresis ) of PCR products. Analyze the result. We found that 68℃ is the best temperature.</P><P>4, Plasmid Extraction. (pmyo2-blink,pmyo3-blink) the results are all at 200-300ng/ul.</P><P>5, Ligase reaction (again): pmyo2-blink, pmyo3-blink.</P><H5>July 25</H5><P>1, Pfu PCR of chETA and ic1c2.</P><P>2, AGE ( agarose gel electrophoresis ) of pfu PCR products.</P><P>3, Gel extraction and recycle the chETA and ic1c2.</P><P>4, Used the C.elegans genomic DNA as template and got pttx-3 with the help of Pttx3-HindIII-F and pttx-3-XbaI-R.</P><P>5, Algarose gel electrophoresis to test PCR result.</P><P>6, Gel extraction and recycle the pttx-3.</P><P>7, Transform DH5α(pmyo2-blink, pmyo3-blink)</P><P>8, Select single clone from culture plate. And culture for 12h.</P><H5>July 26</H5><P>1, Plasmid Extraction (pmyo2-blink, pmyo3-blink)</P><H5>July 27</H5><P>1, Repeat the experiment: used the C.elegans genomic DNA as template and got pttx-3 with the help of Pttx3-HindIII-F and pttx-3-XbaI-R.</P><P>2, Ran the Algarose gel electrophoresis to test my PCR result.</P><P>3, After confirming the accuracy of my PCR result, we did an gel extraction and recycle the pttx-3.</P><P>4, Transformation of pmyo2, pmyo3 plasmids to get more for the latter experiment.Culture at 37℃ for 16h.</P><P>5, Pfu PCR of chETA and ic1c2.</P><P>6, AGE ( agarose gel electrophoresis ) of pfu PCR products.(chETA and ic1c2)</P><P>7, Gel extraction and recycle the chETA and ic1c2.</P><P>8, Select single clone of AMP LB plate.And culture for 12h.</P><H4>1.1.2.4 Week5 -- July 28~31</H4><H5>July 28</H5><P>1, Plasmid Extraction(pmyo2, pmyo3---ppd95.77).</P><H5>July 29</H5><P>1, Make Backbone, transformation of backbone.Culture at 37℃ for 16h. (according by protocol offered by iGEM)</P><P>2, DigestchETA and ic1c2 genes, pmyo2 and pmyo3 vectors with BamHI and EcoRI.(digestion protocol)</P><P>3, Ligation of pmyo2-chETA, pmyo2-ic1c2, pmyo3-chETA and pmyo3-ic1c2. (ligation protocol)</P><H5>July 30</H5><P>1, Transformation of pmyo2-chETA, pmyo2-ic1c2, pmyo3-chETA and pmyo3-ic1c2.Culture at 37℃ for 16h.</P><P>2, Select single clone from culture plate (pmyo2-chETA, pmyo2-ic1c2).And culture for 12h.(there is no clone of pmyo3-chETA and pmyo3-ic1c2 )</P><P>3, Select single clone from culture plate.(Backbone)</P><H5>July 31</H5><P>1, Plasmid Extraction (pmyo2-chETA, pmyo2-ic1c2).</P><P>2, Plasmid Extraction (Backbone).</P><P>3, Digest of pmyo2-chETA and pmyo2-ic1c2 using BamHI and EcoRI. Make sure that the gene had been successfully ligated into the plasmids.</P><P>4, Digest of backbone with PstI and EcoRI. Make sure our backbone is made in the right way.</P></DIV></DIV><DIV class="slidePanel slidePanel1"><H3 class="slideTitle" id="slideTitle1dot3">1.1.3 AUGUST -- Week 1~5 -- August 1~31<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock1dot3"><H4>1.1.3.1 Week1 -- August 1~7</H4><H5>August 1</H5><P>1, Transformation of pmyo2-chETA and pmyo2-ic1c2 in order to get more plasmids.Culture at 37℃ for 16h.</P><P>2, Select single clone from culture plate (pmyo2-chETA and pmyo2-ic1c2.).And culture for 12h.</P><P>3, Try to ligate pmyo3-chETA and pmyo3-ic1c2 again as last time we failed. Digestion and ligation.</P><P>4, Transformation of pmyo3-chETA and pmyo3-ic1c2, Culture at 37℃ for 16h.</P><H5>August 2</H5><P>1, Select single clone from culture plate (pmyo3-chETA and pmyo3-ic1c2.).And culture for 12h.</P><P>2, Plasmid Extraction (pmyo2-chETA, pmyo2-ic1c2).</P><P>3, Plasmid Extraction (pmyo3-chETA, pmyo3-ic1c2).</P><H5>August 3</H5><P>1, Digest ofpmyo3-chETA, pmyo3-ic1c2 to check if we had ligated them right. (the result turn out that the pmyo3-chETA is right)</P><P>2, Transformation of pmyo3-chETA in order to get more plasmids.</P><P>3, Give pmyo2-chR2, pmyo2-chETA, pmyo2-ic1c2, pmyo3-chR2, pmyo3-chRTA and pmec4-dsred to company to test the sequences.</P><H5>August 4</H5><P>1, Select single clone from culture plate (pmyo3-chETA.).And culture for 12h.</P><P>2, Plasmid Extraction (pmyo3-chETA).</P><P>3, Pfu PCR of pttx-3 from C.elegans genome.</P><P>4, AGE ( agarose gel electrophoresis ) of pfu PCR products.(pttx3) However, nothing out.</P><P>5, Trying to ligate pmyo2-ic1c2 once again.Then transformation of it.Cultured in 37℃ for 16h.</P><H5>August 5</H5><P>1, Select single clone from culture plate (pmyo2-ic1c2).And culture for 12h.
                </P><P>2, Plasmid extraction.( pmyo2-ic1c2)</P><P>3, Give pmyo2-ic1c2 to company, and let it test the sequence.</P><P>4, Pfu PCR of pttx-3 from C.elegans genome again. (use different program and different temperature,) </P><P>5, AGE ( agarose gel electrophoresis ) of pfu PCR products.(pttx-3)</P><P>6, Gel extraction and recycle the pttx-3. (10ng/ul)</P><H5>August 6</H5><P>1, Digest of pttx-3 with SalI and BamHI. (using digestion protocol)</P><P>2, Digest of the new ppd95.77 vector with SalI and BamHI. (using digestion protocol)</P><P>3, Ligate of pttx-3 into ppd95.77. (using ligation protocol)</P><P>4, Transformation of pttx-3 in ppd95.77. culture in 37℃ for 16h.</P><H5>August 7</H5><P>1, Yesterday’s transformation has no clone grow.</P><P>2, Pfu PCR of pttx-3 again.</P><P>3, Design the primers of pmec-4. (add HindIII and BamHI)</P><P>4, AGE ( agarose gel electrophoresis ) of pfu PCR products.(pttx-3)</P><P>5, Gel extraction and recycle the pttx-3. (13ng/ul)</P><P>6, Ligate of pttx-3 into ppd95.77. (using ligation protocol)</P><P>7, Transformation of pttx-3 in ppd95.77. culture in 37℃ for 16h.</P><H4>1.1.3.2 Week2 -- August 8~13</H4><H5>August 8</H5><P>1, Yesterday’s transformation has still no clone grow.</P><P>2, Have a lab meeting about why our ligation has so many problems. We decided to try a new method---seamless cloning.</P><H5>August 9</H5><P>1, As we are going to do seamless cloning, we have to design new primers of pttx-3, chR2, chETA, dsRed and ic1c2, blink. </P><H5>August 10</H5><P>1, Make LB liquid.Make LB AMP plates.</P><P>2, Taq PCR of pttx-3, chR2, chETA, dsRed and ic1c2, blink to test the best temperature of PCR reaction.</P><P>3, AGE ( agarose gel electrophoresis ) of taq PCR products. (pttx-3, chR2, chETA, dsRed and ic1c2)</P><P>4, Pfu PCR of pttx-3.</P><H5>August 11</H5><P>1, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx-3)</P><P>2, Gel extraction and recycle the pttx-3. (17ng/ul)</P><P>3, Digest of ppd95.77 with SalI and BamHI.</P><P>4, Seamless clone of pttx-3 into nppd95.77. (use seamless clone protocol)</P><P>5, Transformation of pttx-3 ppd95.77. culture for 16h on AMP LB plate in 37℃.</P><H5>August 12</H5><P>1, Select single clone on AMP LB plate.(pttx-3 ppd95.77) culture in 37℃ for 12h.</P><P>2, Plasmid extraction of pttx-3 ppd95.77.</P><P>3, Digest of pttx-3 with SalI and BamHI to test the ligation result. (turns out to be right!)</P><P>4, Transformation of pttx-3 ppd95.77 in order to get more right plasmids.</P><P>5, Pfu PCR of chR2, chETA, dsRed and ic1c2, blink.</P><P>6, AGE ( agarose gel electrophoresis ) of pfu PCR products. (chR2, chETA, dsRed and ic1c2, blink)</P><P>7, Gel extraction and recycle the chR2, chETA, dsRed and ic1c2, blink. (around 80ng/ul)</P><P>8, Digest 1.5 ul of pttx-3 ppd95.77.</P><P>9, Seamless clone of pttx-3 with chR2, chETA, dsRed, ic1c2 and blink. (seamless clone protocol)</P><P>10, Transformation of pttx-3-chR2, PTTX-3-chETA, PTTX-3-dsRed, pttx-3-ic1c2 and pttx-3-blink.Cultured in 37℃ for 16h.</P><H5>August 13</H5><P>1, Select single clone of pttx-3-chR2, pttx-3-chETA, pttx-3-dsRed and pttx-3-ic1c2 (pttx-3-blink has no clone.). Cultured at 37℃ for 12h.</P><P>2, Plasmid extraction. (pttx-3-chR2, pttx-3-chETA, pttx-3-dsRed and pttx-3-ic1c2)</P><P>3, Digest of (pttx-3-chR2, pttx-3-chETA, pttx-3-dsRed and pttx-3-ic1c2 with BamHI and EcoRI to test if we had successfullyligate the gene into the vector, which turn out that these are all right.</P><P>4, Send them to company for a sequence test.</P><H4>1.1.3.3 Week3 -- August 15~21</H4><H5>August 15</H5><P>1, Pfu PCR of blink again.</P><P>2, AGE ( agarose gel electrophoresis ) of pfu PCR products. (blink)</P><P>3, Gel extraction and recycle the blink. (around 50ng/ul)</P><P>4, Digest of pttx-3 ppd95.77 with BamHI and EcoRI.</P><P>5, Seamless clone of pttx-3-blink.</P><P>6, Transformation of pttx3-blink. Culture in 37℃ for 16h.</P><P>7, Taq PCR of ptwk16 to test the best reaction situation.</P><P>8, AGE ( agarose gel electrophoresis ) of taq PCR products. (ptwk16)</P><P>9, Pfu PCR of ptwk16.</P><H5>August 16</H5><P>1, Select single clone of pttx-3-blink.Cultured at 37℃ for 12h.</P><P>2, Plasmid extraction of pttx-3-blink.</P><P>3, AGE ( agarose gel electrophoresis ) of pfu PCR products. (ptwk16)</P><P>4, Gel extraction and recycle the blink. (around33ng/ul)</P><P>5, Digest of ptwk16 with HindIII and SalI. Digest the vector ppd95.77 with HindIII and SalI.</P><P>6, Ligation of ptwk16 into ppd95.77 with T4 ligase. (12h)</P><P>7, AGE ( agarose gel electrophoresis ) of pttx-3-blink to test if we got the right result.</P><H5>August 17</H5><P>1, Transformation of ptwk16 ppd95.77. culture in 37℃ for 16h.</P><P>2, Transformation of the back bone we made in order to get more to prepare for the later backbone making.Culture in 37℃ for 16h.</P><H5>August 18</H5><P>1, Select single clone of ptwk16 ppd95.77. Cultured at 37℃ for 12h.</P><P>2, Select single clone of backbone.Cultured at 37℃ for 12h.</P><P>3, Plasmid extraction of ptwk16 ppd95.77 and backbone.</P><P>4, Digest of ptwk16 ppd95.77 with HindIII and SalI to test if we had ligated it in a right way.</P><H5>Augest 19</H5><P>1, Pfu PCR of blink, chR2, dsred, ic1c2 and chETA.</P><P>2, AGE ( agarose gel electrophoresis ) of pfu PCR products. (blink, chR2, dsred, ic1c2 and chETA)</P><P>3, Gel extraction and recycle the blink. (around 50ng/ul)</P><P>4, Digest of ptwk16 ppd95.75 with BamHI and HindIII.</P><P>5, Seamless clone of ptwk16-blink, ptwk16-chR2, ptwk16-dsred, ptwk16-ic1c2 and ptwk16-chETA.</P><P>6, Transformation of ptwk16-blink, ptwk16-chR2, ptwk16-dsred, ptwk16-ic1c2 and ptwk16-chETA.Cultured in 37℃ for 16h.</P><H5>August 20</H5><P>1, Select the single clone of ptwk16-blink, ptwk16-ic1c2 and ptwk16-chETA. (ptwk16-chR2, ptwk16-dsred has not been ligated successfully.) culture in 37℃ for 12h</P><P>2, plasmid extraction of ptwk16-blink, ptwk16-ic1c2 and ptwk16-chETA.</P><P>3, Digest of ptwk16-blink, ptwk16-ic1c2 and ptwk16-chETA with BamHI and EcoRI to test if we had ligated them in the right way. (It turns out to be right.)</P><P>4, Ligate of ptwk16-chR2, ptwk16-dsRed again.</P><P>5, Transformation of ptwk16-chR2, ptwk16-dsred.Culture in 37℃ for 16h.</P><H5>August 21</H5><P>1, Select the single clone of ptwk16-blink, ptwk16-ic1c2 and ptwk16-chETA. (ptwk16-chR2, ptwk16-dsred) culture in 37℃ for 12h</P><P>2, Plasmid extraction of ptwk16-chR2, ptwk16-dsred.</P><P>3,digest of ptwk16-chR2, ptwk16-dsred with BamHI and EcoRI to test if the result is right.</P><P>4, Start to make backbone which we are going to send to iGEM. Design the seamless clone PCR primers of pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA, pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA.</P><P>5, Make some chloramphenicol LB plates.</P><H4>1.1.3.4 Week4 -- August 22~28</H4><H5>August 22</H5><P>1, Taq PCR of pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA, pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA to test the best reaction situation.</P><P>2, AGE ( agarose gel electrophoresis ) of these taq PCR products.</P><P>3, Pfu PCR of pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA.</P><P>4, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA.)</P><P>5, Gel extraction and recycle the blink. (around 50ng/ul)</P><P>6, Digest of backbone with PstI and EcoRI.</P><P>7, AGE ( agarose gel electrophoresis ) of digestion products. </P><P>8, Gel extraction and recycle the blink. (around 30ng/ul)</P><H5>August 23</H5><P>1, Seamless clone of backbone---pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA.</P><P>2, Transformation of backbone---pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA.Culture on chloramphenicol LB plates in 37℃ for 16h.</P><P>3, Select the single clone of backbone---pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA. Culture onchloramphenicol LB plates in 37℃ for 12h.</P><P>4, Pfu PCR of pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA.</P><P>5, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA.) However, nothing came out.</P><H5>August 24</H5><P>1, Plasmid extraction of pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA.</P><P>2, Digest of pmyo2-chR2, pmyo2-dsred, pmyo2-blink, pmyo2-ic1c2, pmyo2-chETA, pmyo3-chR2, pmyo3-dsred, pmyo3-blink, pmyo3-ic1c2, pmyo3-chETA to check if we had ligate the right parts into plasmid.</P><P>3, Send them to company to test the sequence.</P><P>4, Pfu PCR of pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA again. (change the reaction temperature.)</P><P>5, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx3-chR2, pttx3-dsred, pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-chR2, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA.) pttx3-chR2, pttx3-dsred and ptwk16-chR2 show on the gel.</P><P>6, Gel extraction and recycle the pttx3-chR2, pttx3-dsred and ptwk16-chR2.</P><H5>August 25</H5><P>1, Digest the backbone with EcoRI and PstI.</P><P>2, Seamless clone of backbone--- pttx3-chR2, pttx3-dsred and ptwk16-chR2.</P><P>3, Transformation of backbone---pttx3-chR2, pttx3-dsred and ptwk16-chR2.Cultured on chloramphenicol LB plates in 37℃ for 19h.</P><H5>August 26</H5><P>1, Onlybackbone-ptwk16-chR2 grown some clones on the plate. Select single clone on the plate. Cultured it on chloramphenicol LB plates in 37℃ for 19h.</P><P>2, Plasmid extraction of backbone-ptwk16-chR2.</P><P>3, Digest of backbone-ptwk16-chR2 to make sure that we had made the right backbone. </P><H5>August 27</H5><P>1, Design new primers of pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA.</P><P>2, As we couldn’t have PstI, EcoRI, SpeI and BamHI in our backbone plasmid, we have to do some point mutation.</P><P>3, The overlap PCR of backbone---pmyo2-chR2, pmyo2-dsred, pmyo3-chR2 to mutate the PstI site in them.</P><H5>August 28</H5><P>1, The overlap PCR of backbone---pmyo2-chR2, pmyo2-dsred, pmyo3-chR2 to mutate the PstI site in them.</P><P>2, Pfu PCR of pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA using new primers to test the best reaction situation.</P><P>3, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA)</P><P>4, Pfu PCR of pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA using new primers.</P><H4>1.1.3.5 Week5 -- August 29~31</H4><H5>August 29</H5><P>1, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA) we got pttx3-blink and ptwk16-ic1c2.</P><P>2, Gel extraction and recycle pttx3-blink and ptwk16-ic1c2.</P><P>3, Digest of backbone with PstI and EcoRI.</P><P>4, Seamless clone of backbone-pttx3-blink and ptwk16-ic1c2.</P><P>5, Transfomation of backbone-pttx3-blink and ptwk16-ic1c2.Culture in 37℃ for 19h.</P><P>6, Point mutation of our backbone products.</P><H5>August 30</H5><P>1, Select the single clone of ptwk16-ic1c2 on chloramphenicol LB plates. (another has no clone) cultured in 37℃ for 19h.</P><P>2, Point mutation of our backbone products.</P><P>3, Point mutation of our backbone products.</P><H5>August 31</H5><P>1, Plasmid extraction of ptwk16-ic1c2.</P><P>2, Digest ptwk16-ic1c2 with PstIEcoRI to check our result.</P><P>3, Point mutation of our backbone products.</P></DIV></DIV><DIV class="slidePanel slidePanel1"><H3 class="slideTitle" id="slideTitle1dot4">1.1.4 SEPTEMBER -- Week 1~2 -- September 1~10<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock1dot4"><H4>1.1.4.1 Week1 -- September 1~7</H4><H5>September 1</H5><P>1, Pfu PCR of pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA for the last time.</P><P>2, Point mutation of our backbone products. (Some of them failed.)</P><H5>September 2</H5><P>1, AGE ( agarose gel electrophoresis ) of pfu PCR products. (pttx3-blink, pttx3-ic1c2, pttx3-chETA, ptwk16-dsred, ptwk16-blink, ptwk16-ic1c2, ptwk16-chETA) we got ptwk16-blink and ptwk16-chETA.</P><P>2, Gel extraction and recycleptwk16-blink and ptwk16-chETA.</P><P>3, Send all the backbone we have now to company to test the sequence.</P><H5>September 3</H5><P>1, Digest of backbone with PstI and EcoRI.</P><P>2, Seamless cloning of backbone-ptwk16-blink and ptwk16-chETA.</P><P>3, Transformation into chloramphenicol LB plates.Cultured in 37℃ for 19h. </P><P>4, Make more LB AMP plates.</P><H5>September 4</H5><P>1, Select single clone on the plate. Culture in 37℃ for 13h.</P><P>2, Plasmid extraction of backbone-ptwk16-blink and ptwk16-chETA.</P><P>3, Digest of backbone-ptwk16-blink and ptwk16-chETA to test. The result is strange which means that we may failed in this ligation.</P><H5>September 5</H5><P>1, Try a new method to make the failed backbones.Digest the backbone-pmyo2-blink and pmyo2-chETA with BamHI and SpelI.Digest the ptwk16 out of the plasmid with HindIII and SalI.</P><P>2, Pfu PCR of ptwk16.</P><P>3, AGE ( agarose gel electrophoresis ) of pfu PCR products. (ptwk16)</P><P>4, Gel extraction and recycleptwk16.</P><H5>September 6</H5><P>1, Traditional ligation of backbone-ptwk16-blink and ptwk16-chETA.</P><P>2, Transformation of backbone-ptwk16-blink and ptwk16-chETA.Culture in 37℃ for 13h</P><H5>September 7</H5><P>1, Select single clone on the plate of backbone-ptwk16-blink and ptwk16-chETA. Culture in 37℃ for 13h.</P><P>2, Plasmid extraction of backbone-ptwk16-blink and ptwk16-chETA.</P><P>3, Digest test.</P><H4>1.1.4.2 Week2 -- September 9~10</H4><H5>September 9</H5><P>1, Make our sending plasmid powder by freeze dryer.</P><H5>September 10</H5><P>1, Make our sending plasmid powder by freeze dryer.</P></DIV></DIV><P class="partTwo" id="first2">1.2 Worm Part</P><DIV class="slidePanel slidePanel2"><H3 class="slideTitle" id="slideTitle2dot1">1.2.1 JUNE -- June 20~29<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock2dot1"><H4>1.2.1.1 June 20~29</H4><P>Learn how to do the microinjection on C.elegans.</P></DIV></DIV><DIV class="slidePanel slidePanel2"><H3 class="slideTitle" id="slideTitle2dot2">1.2.2 JULY -- Week 2~5 -- July 9~31<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock2dot2"><H4>1.2.2.1 Week2 -- July 9~13</H4><H5>July 9</H5><P>1, Preparation of NGM plates.</P><P>2, Seed NGM plates.</P><H5>July 11</H5><P>1, Recover the L1 larva of lite-1 worm from -80 fridge.</P><H5>July 13</H5><P>1, Last time the recovery failed. Recover the L1 larva of lite-1 worm again from -80 fridge.</P><H4>1.2.2.2 Week3 -- July 15~21</H4><H5>July 15</H5><P>1, The recovery still failed. Recover the L1 larva of lite-1 worm once again!(this time we successes!)</P><H5>July 21</H5><P>1, Seed plates with OP50 and ATR (keep in dark place). For 10 3cm plateS(1.5ml agar EACH),seed 1000ul OP50 mixed with 1.75ul 100uM ATR. The OP50 E.coli should be shacked for 12 hours in the conical flask at 37℃(150ml LB added 4-5 single colonies. )</P><P>2, Microinjection pmyo-3::dsRed and Pmyo-3::ChR2 co-injection.(40 worms)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo3::dsrRed</TD><TD>20ng/ul</TD><TD>144.7ng/ul</TD><TD>1.38ul</TD></TR><TR class="evenTr"><TD>Pmyo-3::ChR2</TD><TD>80ng/ul</TD><TD>174.7ng/ul</TD><TD>4.58ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>162.5ng/ul</TD><TD>2.04ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><H4>1.2.2.3 Week4 -- July 23~28</H4><H5>July 23</H5><P>1, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred.Put the worm on a new NGM plate.</P><P>2, Preparation of some new NGM plates.</P><P>3, Seed NGM plates.</P><H5>July 24</H5><P>1, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred.Put the worm on a new NGM plate.</P><H5>July 25</H5><P>1, Subculture of C.elegans which had been injection of pmyo-3::dsRed and Pmyo-3::ChR2.</P><P>2, Subculture of lite1C.elegans.</P><H5>July 26</H5><P>1, Make ATR solution. (help chR2 work in C.elegan)</P><P>2, Subculture of C.elegans which had been injection of pmyo-3::dsRed and Pmyo-3::ChR2.</P><P>3, Subculture of lite1C.elegans.</P><H5>July 27</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Put some L4 pmyo3-dsred and pmyo3-chR2 C.elegans on ATR plate.</P><H5>July 28</H5><P>1, Microinjection.(pmyo-2::dsRed and Pmyo-2::ChR2 co-injection) for 40 worms.</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo2::dsrRed</TD><TD>20ng/ul</TD><TD>209.7ng/ul</TD><TD>0.95ul</TD></TR><TR class="evenTr"><TD>Pmyo2::ChR2</TD><TD>50ng/ul</TD><TD>549ng/ul</TD><TD>0.91ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331.2ng/ul</TD><TD>3.62ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>2.52ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><H4>1.2.2.4 Week5 -- July 29~31</H4><H5>July 29</H5><P>1, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo2 will express at the worm neck) Put the worm on a new NGM plate.</P><P>2, Subculture of all the C.elegans we have.</P><H5>July 30</H5><P>1, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo2 will express at the worm neck) Put the worm on a new NGM plate.</P><P>2, Subculture of all the C.elegans we have.</P><H5>July 31</H5><P>1, Microinjection.( PNP260and PSH116 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>PNP260</TD><TD>50ng/ul</TD><TD>471ng/ul</TD><TD>1.06ul</TD></TR><TR class="evenTr"><TD>PSH116</TD><TD>50ng/ul</TD><TD>516ng/ul</TD><TD>0.96ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>271ng/ul</TD><TD>4.42ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>1.56ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, subculture of all the C.elegans we have.</P><P>3, put some L4 pmyo-2::dsRed and Pmyo-2::ChR2 C.elegans on ATR plate.</P></DIV></DIV><DIV class="slidePanel slidePanel2"><H3 class="slideTitle" id="slideTitle2dot3">1.2.3 AUGUST -- Week 1~5 -- August 1~31<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock2dot3"><H4>1.2.3.1 Week1 -- August 1~6</H4><H5>August 1</H5><P>1, Microinjection.( pmyo-3::dsRed and Pmyo-3::ChR2 co-injection AGAIN! Last time we failed)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo2-dsred</TD><TD>20ng/ul</TD><TD>144.7ng/ul</TD><TD>1.38ul</TD></TR><TR class="evenTr"><TD>Pmyo2-chR2</TD><TD>50ng/ul</TD><TD>120.8ng/ul</TD><TD>4.41ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>271ng/ul</TD><TD>2.5ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Subculture of all the C.elegans we have.</P><P>3, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo3 will express at the worm’s whole body muscle) Put the right worm on a new NGM plate.</P><H5>August 2</H5><P>1, Microinjection.( pmec-3::dsRed and Pmec-3::ChR2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmec3-dsred </TD><TD>20ng/ul</TD><TD>159ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmec3-chR2</TD><TD>50ng/ul</TD><TD>190ng/ul</TD><TD>2.6ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.6ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Microinjection.(Pcos13 and Pcos2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>PCoS13</TD><TD>50ng/ul</TD><TD>191.5ng/ul</TD><TD>2.6ul</TD></TR><TR class="evenTr"><TD>PCoS2</TD><TD>50ng/ul</TD><TD>327.2ng/ul</TD><TD>1.5ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>271ng/ul</TD><TD>3.3ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.6ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>3, Subculture of all the C.elegans we have.</P><P>4, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo3 will express at the worm’s whole body muscle) Put the right worm on a new NGM plate.</P><P>5, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmec3 will express at the worm’s touch neuron) Put the right worm on a new NGM plate.</P><H5>August 3</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo3 will express at the worm’s whole body muscle, (pmec3 will express at the worm’s touch neuron, PCoS13 overlap PCos2 will express at AVA neuron, PNP260 overlap PSH116 will express at PVC neuron).Put the right worm on a new NGM plate.</P><P>3, Preparation of some new NGM plates.</P><P>4, Seed NGM plates.</P><P>5, Seed new ATR plates.</P><H5>August 4</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo3 will express at the worm’s whole body muscle, (pmec3 will express at the worm’s touch neuron, PCoS13 overlap PCos2 will express at AVA neuron, PNP260 overlap PSH116 will express at PVC neuron). Put the right worm on a new NGM plate.</P><H5>August 5</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected. We are supposed to select the worm with dsred. (pmyo3 will express at the worm’s whole body muscle, (pmec3 will express at the worm’s touch neuron, PCoS13 overlap PCos2 will express at AVA neuron, PNP260 overlap PSH116 will express at PVC neuron). Put the right worm on a new NGM plate.</P><H5>August 6</H5><P>1, Subculture of all the C.elegans we have.</P><H4>1.2.3.2 Week2 -- August 8~14</H4><H5>August 8</H5><P>1, Subculture of all the C.elegans we have.</P><H5>August 9</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Since the some of C.elegans we injected has not been passaged stalely, so we had to reinject some of them.</P><P>Microinjection.( pmec-3::dsRed and Pmec-3::ChR2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmec3-dsred</TD><TD>20ng/ul</TD><TD>159ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmec3-chR2</TD><TD>50ng/ul</TD><TD>190ng/ul</TD><TD>2.6ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.6ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( pmyo-2::dsRed and Pmyo-2::ChR2 co-injection) for 40 worms.</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo2::dsrRed</TD><TD>20ng/ul</TD><TD>209.7ng/ul</TD><TD>0.95ul</TD></TR><TR class="evenTr"><TD>Pmyo2::ChR2</TD><TD>50ng/ul</TD><TD>549ng/ul</TD><TD>0.91ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331.2ng/ul</TD><TD>3.62ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>2.52ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><H5>August 10</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H5>August 11</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H5>August 12</H5><P>1, Microinjection.( pmyo-3::dsRed and Pmyo-3::ic1c2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo3-dsred</TD><TD>20ng/ul</TD><TD>159ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmyo3-ic1c2</TD><TD>50ng/ul</TD><TD>187ng/ul</TD><TD>2.6ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.6ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Microinjection.( pmyo-3::dsRed and Pmyo-3::chETA co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo3-dsred</TD><TD>20ng/ul</TD><TD>159ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmyo3 chETA</TD><TD>50ng/ul</TD><TD>106ng/ul</TD><TD>2.9ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.3ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>3, Subculture of all the C.elegans we have.</P><P>4, Select the F1 generation of the worm we injected.</P><H5>August 13</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H5>August 14</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H4>1.2.3.3 Week3 -- August 15~21</H4><H5>August 15</H5><P>1, Microinjection.( pmyo-2::dsRed and Pmyo-2::chETA co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo2-dsred</TD><TD>20ng/ul</TD><TD>128ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmyo2-chETA</TD><TD>50ng/ul</TD><TD>106ng/ul</TD><TD>2.9ul</TD></TR><TR><TD>PBlue</TD><TD>120ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.3ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( pmyo-2::dsRed and Pmyo-2::ic1c2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo2-dsred</TD><TD>20ng/ul</TD><TD>156ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pmyo2-ic1c2</TD><TD>90ng/ul</TD><TD>178ng/ul</TD><TD>2.55ul</TD></TR><TR><TD>PBlue</TD><TD>70ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.75ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Subculture of all the C.elegans we have.</P><P>3, Select the F1 generation of the worm we injected.</P><H5>August 16</H5><P>1, Microinjection.(pttx-3::dsRed and Pttx-3::chR2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>pttx-3::dsRed</TD><TD>20ng/ul</TD><TD>128ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pttx-3::chR2</TD><TD>80ng/ul</TD><TD>200ng/ul</TD><TD>2.9ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.3ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( pttx-3::dsRed and Pttx-3::BLINK co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>pttx-3::dsRed</TD><TD>20ng/ul</TD><TD>128ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pttx-3::BLINK</TD><TD>80ng/ul</TD><TD>213ng/ul</TD><TD>2.8ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.4ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( pttx-3::dsRed and Pttx-3::ic1c2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>pttx-3::dsRed</TD><TD>20ng/ul</TD><TD>128ng/ul</TD><TD>1.6ul</TD></TR><TR class="evenTr"><TD>Pttx-3::ic1c2</TD><TD>70ng/ul</TD><TD>167ng/ul</TD><TD>2.8ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>331ng/ul</TD><TD>3.25ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.4ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Subculture of all the C.elegans we have.</P><P>3, Select the F1 generation of the worm we injected.</P><H5>August 18</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H5>August 19</H5><P>Microinjection.(Pcos13 and Pcos2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>PCoS13</TD><TD>50ng/ul</TD><TD>191.5ng/ul</TD><TD>2.6ul</TD></TR><TR class="evenTr"><TD>PCoS2</TD><TD>50ng/ul</TD><TD>327.2ng/ul</TD><TD>1.5ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>271ng/ul</TD><TD>3.3ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.6ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( PNP260and PSH116 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>PNP260</TD><TD>50ng/ul</TD><TD>471ng/ul</TD><TD>1.06ul</TD></TR><TR class="evenTr"><TD>PSH116</TD><TD>50ng/ul</TD><TD>516ng/ul</TD><TD>0.96ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>271ng/ul</TD><TD>4.42ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>1.56ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><H5>August 20~21</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H4>1.2.3.4 Week4 -- August 22~27</H4><H5>August 22</H5><P>1, Microinjection.( ptwk16-dsRed and ptwk16-blink co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>ptwk16-dsRed</TD><TD>50ng/ul</TD><TD>187ng/ul</TD><TD>1.06ul</TD></TR><TR class="evenTr"><TD>ptwk16-blink</TD><TD>90ng/ul</TD><TD>213ng/ul</TD><TD>0.96ul</TD></TR><TR><TD>PBlue</TD><TD>50ng/u</TD><TD>271ng/ul</TD><TD>4.42ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>1.56ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( ptwk16-dsRed and ptwk16-chR2 co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>ptwk16-dsRed</TD><TD>50ng/ul</TD><TD>197ng/ul</TD><TD>1.06ul</TD></TR><TR class="evenTr"><TD>ptwk16-chR2</TD><TD>90ng/ul</TD><TD>254ng/ul</TD><TD>0.93ul</TD></TR><TR><TD>PBlue</TD><TD>50ng/u</TD><TD>271ng/ul</TD><TD>4.42ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>1.59ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>2, Subculture of all the C.elegans we have.</P><P>3, Select the F1 generation of the worm we injected.</P><H5>August 23~24</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><H5>August 25</H5><P>1, Subculture of all the C.elegans we have.</P><P>2, Select the F1 generation of the worm we injected.</P><P>3, Microinjection.( pmyo3-dsRed and pmyo3-blink co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>Pmyo3-dsRed</TD><TD>50ng/ul</TD><TD>197ng/ul</TD><TD>1.06ul</TD></TR><TR class="evenTr"><TD>Pmyo3-blink</TD><TD>90ng/ul</TD><TD>254ng/ul</TD><TD>0.93ul</TD></TR><TR><TD>PBlue</TD><TD>50ng/u</TD><TD>271ng/ul</TD><TD>4.42ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>1.59ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Microinjection.( pttx-3::dsRed and Pttx-3::BLINK co-injection)</P><P><B>Mixture:</B></P><TABLE class="tableQuestion"><TBODY><TR><TH>Material</TH><TH>Final concentration</TH><TH>Real concentration</TH><TH>Real Volume</TH></TR><TR><TD>pttx-3::dsRed</TD><TD>20ng/ul</TD><TD>128ng/ul</TD><TD>1.25ul</TD></TR><TR class="evenTr"><TD>Pttx-3::BLINK</TD><TD>80ng/ul</TD><TD>213ng/ul</TD><TD>2.8ul</TD></TR><TR><TD>PBlue</TD><TD>90ng/ul</TD><TD>331ng/ul</TD><TD>3.6ul</TD></TR><TR class="evenTr"><TD>Rat Genomic DNA</TD><TD>10ng/ul</TD><TD>100ng/ul</TD><TD>1ul</TD></TR><TR><TD>10XTE</TD><TD>1ul</TD></TR><TR class="evenTr"><TD>ddH2O</TD><TD>0.4ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><H5>August 26~27</H5><P>1, Subculture of all the C.elegans we have.</P><H4>1.2.3.5 Week5 -- August 29~31</H4><P>1, Subculture of all the C.elegans we have.</P></DIV></DIV><DIV class="slidePanel slidePanel2"><H3 class="slideTitle" id="slideTitle2dot4">1.2.4 SEPTEMBER -- Week 1~2 -- September 1~15<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock2dot4"><P>1, Subculture of all the C.elegans we have.</P></DIV></DIV><P class="partTwo" id="first3">1.3 Efficiency Testpart</P><DIV class="slidePanel slidePanel3"><H3 class="slideTitle" id="slideTitle3dot1">1.3.1 AUGUST -- Week 1~5 -- August 3~31<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock3dot1"><H4>1.3.1.1 Week1 -- August 3~7</H4><H5>August 3</H5><P>1, Initial test of pmyo3-chR2-YFP C.elegans.since we don’t have enough worms yet, we just test for 2 worm. We found their behavior have obvious change when we turn on the blue light. (470nm)</P><H5>August 5</H5><P>1, Initial test of pmyo2-chR2-YFP C.elegan. Since the chR2 is expressed at the neck, we didn’t see any big change of movement when we turn on the light.</P><H5>August 6</H5><P>1, Intial test of Pcos13 and Pcos2 C.elegan.</P><H5>August 7</H5><P>1, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP and Pcos13 overlap with Pcos2 C.elegans again. </P><H4>1.3.1.2 Week2 -- August 9~14</H4><H5>August 9</H5><P>1, Trying to take some small video of the pmyo2-chR2-YFP, pmyo3-chR2-YFP and Pcos13 overlap with Pcos2 C.elegans.</P><H5>August 10</H5><P>1, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP and Pcos13 overlap with Pcos2 C.elegans.</P><H5>August 13</H5><P>1, Intial test of Pmyo-3-chETAC.elegan.</P><H5>August 14</H5><P>1, Intial test of Pmyo-3-ic1c2 C.elegan.</P><H4>1.3.1.3 Week3 -- August 15~21</H4><H5>August 15</H5><P>1, Try to take some small video of thePmyo-3-chETA and Pmyo-3-ic1c2 C.elegans.</P><P>2, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP, Pcos13 overlap with Pcos2, Pmyo-3-chETA and Pmyo-3-ic1c2. </P><H5>August 16</H5><P>1, Intial test of Pmyo-2-ic1c2 andPmyo-2-chETA.</P><H5>August 17</H5><P>1, Try to take some small video of thePmyo-2-chETA and Pmyo-2-ic1c2 C.elegans.</P><P>2, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP, Pcos13 overlap with Pcos2, Pmyo-3-chETA Pmyo-3-ic1c2, Pmyo-2-ic1c2 and Pmyo-2-chETA.</P><H5>August 19</H5><P>1, Intial test of pttx3-chR2.</P><P>2, Try to take some small video of thepttx3-chR2 C.elegans.</P><H5>August 20</H5><P>1, Lab meeting about how to take our video in a standard way.We decided to take 40sec each one: 10s for white light, 10s for 470nm light, 10 for white light and 10s for another wavelength light.In this way, we could use some video software to analyze the test result. </P><H5>August 21</H5><P>1, Intial test of ptwk16-blink.</P><P>2, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP, Pmyo-3-chETA Pmyo-3-ic1c2, Pmyo-2-ic1c2, Pmyo-2-chETA pttx3-chR2 and ptwk16-blink.</P><P>3, Take video of these genetypeC.elegans.</P><H4>1.3.1.4 Week 4~5 -- August 23~30</H4><H5>August 23,25,27,28,30</H5><P>1, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP, Pmyo-3-chETA Pmyo-3-ic1c2, Pmyo-2-ic1c2, Pmyo-2-chETA pttx3-chR2 and ptwk16-blink.</P><P>2, Take video of these genetypeC.elegans.</P></DIV></DIV><DIV class="slidePanel slidePanel3"><H3 class="slideTitle" id="slideTitle3dot2">1.3.2 SEPTEMBER -- Week 1~2 -- September 1~15<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock3dot2"><H4>1.3.2.1 Week1 -- September 1~7</H4><P>1, Test of pmyo2-chR2-YFP, pmyo3-chR2-YFP, Pmyo-3-chETA Pmyo-3-ic1c2, Pmyo-2-ic1c2, Pmyo-2-chETA pttx3-chR2 and ptwk16-blink.</P><P>2, Take video of these genetypeC.elegans.</P><H4>1.3.2.2 Week2 -- September 8~16</H4><P>1, Analyze the results with related video software.</P><P>2, Still test of our C.elegans.</P></DIV></DIV><P class="partTwo" id="first4">1.4 Equipment Part</P><DIV class="slidePanel slidePanel4"><H3 class="slideTitle" id="slideTitle4dot1">1.4.1 JULY -- July 1,28<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock4dot1"><H4>1.4.1.1 Week1 -- July 1</H4><P>1, We pick up the parts we need on the internet under the help of our instructor Wei Li. Ordered them on line.</P><H4>1.4.1.2 Week4 -- July 28</H4><P>1, We learn about how to build our LED light source together. </P></DIV></DIV><DIV class="slidePanel slidePanel4"><H3 class="slideTitle" id="slideTitle4dot2">1.4.2 AUGUST -- Week 1~2 -- August 1~8<SPAN class="expand">[Expand]</SPAN></H3><DIV class="slideBlock" id="slideBlock4dot2"><H4>1.4.2.1 Week1 -- August 1~7</H4><H5>August 1</H5><P>1, All our equipment has been arrived, we started to build it. We met some problems during the process, for instance, some of the parts did not fit in with each other.</P><H5>August 2~3</H5><P>1, Build the LED light source.</P><H5>August 4</H5><P>1, Adjustment of light path.</P><P>2, Ask for the help about how install and use the LED related software on the computer from microscope engineer.</P><H5>August 5</H5><P>1, After the first test, we found that our LED power is still not powerful enough. So we bought some new LED source on the internet.</P><H5>August 7</H5><P>1, Weld the new 5W LED light source up.</P><H4>1.4.2.2 Week2 -- August 8</H4><P>1, Replace the old LED source with the new one we bought.</P></DIV></DIV><P class="titleOne" id="Timeline">2. Timeline</P></DIV></DIV><DIV class="foot"><DIV class="navFoot" align="center"><A href="https://2015.igem.org/Team:China_Tongji">Home</A>|<A href="https://2015.igem.org/Team:China_Tongji/Project">Project</A>|<A href="https://2015.igem.org/Team:China_Tongji/Notebook">Notebook</A>|<A href="https://2015.igem.org/Team:China_Tongji/Achivement">Achivement</A>|<A href="https://2015.igem.org/Team:China_Tongji/Team">Team</A>|<A href="https://2015.igem.org/Team:China_Tongji/Outreach">Outreach</A>|<A href="https://2015.igem.org/Team:China_Tongji/Safety">Safety</A></DIV><DIV class="copyright"><CENTER><P id="copyrightText">Copyright © China_Tongji iGEM 2015 | Developer: LiJunWu </P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>