<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Moscow_RF_notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Moscow RF/notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Moscow_RF"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Moscow_RF/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT DESIGN
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Moscow_RF/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?id=2488"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Thanks"><DIV class="menu_button direct_to_page">
				THANKS
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="column notebook"><H1>Notebook</H1><H5>31/07/2017, Monday</H5><P>Filled two Petri dishes with two different antibiotics (chloramphenicol and kanamycin) to test the strains.As a result,the strains have grown in chloramphenicol successfully.</P><H5>01/08/2017, Tuesday</H5><P>Resuspended plasmids from the distribution kit in 20 μl distilled water.
<U>Resuspended plasmids:</U><UL><LI>BBa_K1351005(plate 5,2H)</LI><LI>BBa_K1415002(plate 5,13E)</LI><LI>BBa_K1321105(plate 5,16B)</LI><LI>BBa_K676002(plate 6,19N)</LI></UL></P><P>Transformed competent cells with 5 μl of resuspension(with heat shock).
Heat shock: <UL><LI>20 min - ice</LI><LI>45 s - 42°</LI><LI>2 min -  ice</LI></UL></P><P>
   Streaked chloramphenicol agar with 300 μl of transformation reaction.
Transformation reaction: 5μl plasmid + 110 μl cells + 300 μl LB.</P><H5>02/08/2017, Wednesday</H5><P>Put night cultures after adding nutrient medium (5 ml - LB) with chloramphenicol (add 1 colony).<U>Used plasmids:</U><UL><LI>BBa_K1351005(plate 5,2H)</LI><LI>BBa_K1415002(plate 5,13E)</LI><LI>BBa_K1321105(plate 5,16B)</LI><LI>BBa_K676002(plate 6,19N)</LI></UL></P><P>
Concentration of plasmids: 100 pg/μl.
</P><H5>03/08/2017, Thursday</H5><P>Night cultures of next plasmids:
<UL><LI>BBa_K1351005(plate 5,2H)</LI><LI>BBa_K1415002(plate 5,13E)</LI><LI>BBa_K1321105(plate 5,16B)</LI><LI>BBa_K676002(plate 6,19N)</LI></UL></P><P>I. Purification of plasmids for sequencing.
For alkaline lysis were used:<UL><LI>Tris HCL - 250 μl</LI><LI>lysing solution -  250 μl</LI><LI>potassium acetate - 350 μl</LI></UL></P><P>
For the rest plasmids’ purification:<UL><LI>isopropanol - 850 μl</LI><LI>ethanol 80% - 850 μl</LI><LI>buffer Tris HCL - 60 μl</LI><LI>reagents for agarose gel electrophoresis</LI></UL></P><P><U>Gel electrophoresis:</U></P><UL><LI>1. BBa_K1362051 (Intein protease with arabinose inducible regulatory promoter)</LI><LI>2. BBa_K1319004 (TEV protease with anti-self cleavage mutation S219V)</LI><LI>3. BBa_K1321090 (Phytochelatin (PC) EC20)</LI><LI>4. BBa_K300004 (Engineered pH-inducible intein (codon optimized for E. coli) – internal domain)</LI><LI>5. Ladder</LI><LI>6. Control, some plasmid</LI></UL><P>II. Resuspended plasmids from the distribution kit in 20 μl distilled water.
      Transformed competent cells with 5 μl of resuspension (with heat shock).
Heat shock:<UL><LI>20 min - ice </LI><LI>90 s - 42°</LI><LI>2 min -  ice</LI></UL></P><P>
After adding 200μl SOB - 15-20 min 37°
Streaked chloramphenicol agar with 300 μl of transformation reaction.</P><P><U>Resuspended plasmids:</U><UL><LI>BBa_K1362051(plate 5,23I)</LI><LI>BBa_K1321090(plate 5,12L)</LI><LI>BBa_K1319004(plate 5,9O)</LI><LI>BBa_K300004(plate 6,6C)</LI></UL></P><H5>07/08/2017, Monday</H5><P>  As our previous transformation had bad results, we decided to test other BioBricks with different mass values and ways of transformation.</P><P>I. Resuspended plasmids from the distribution kit in 20 μl distilled water.
  Transformed competent cells with 5 μl(plate 2,1D) and 10 μl(plate 2,1D and 1B) of resuspension with heat shock and 10 μl of resuspension (plate 2,1B) with electroporation.
  Streaked chloramphenicol agar with 300 μl of transformation reaction.
Transformation reaction: 5μl plasmid + 85 μl cells + 200 μl SOB.<U>Resuspended plasmids:</U><UL><LI>BBa_K909007 (plate 2,1B)</LI><LI>BBa_K777113 (plate 2,1D)</LI></UL></P><P>II.Put night cultures after adding nutrient medium (5-6 ml LB) with antibiotic (chloramphenicol), add 1 colony.<U>Used plasmids:</U><UL><LI>BBa_K1319004 (plate 5,9O)</LI><LI>BBa_K300004 (plate 6,6C)</LI></UL></P><H5>08/08/2017, Tuesday</H5><P><U>Used plasmids:</U><UL><LI>BBa_K1319004 (plate 5,9O)</LI><LI>BBa_K300004 (plate 6,6C)</LI></UL></P><P> Purification of plasmids for sequencing.
      For alkaline lysis were used:
<UL><LI>Tris HCL - 250 μl</LI><LI>lysing solution -  250 μl</LI><LI>potassium acetate - 350 μl</LI></UL></P><P>
   For the rest plasmids’ purification:
<UL><LI>isopropanol - 850 μl</LI><LI>ethanol 80% - 850 μl</LI><LI>buffer Tris HCL - 60 μl</LI><LI>reagents for agarose gel electrophoresis.</LI></UL></P><H5>09/08/2017, Wednesday</H5><P>Resuspended plasmids from the distribution kit in 20 μl distilled water.Transformed competent cells with 5 μl of resuspension(with electroporation).
After adding 800μl SOB - 40 min 37°.
Streaked chloramphenicol agar with 300 μl of transformation reaction.
Transformation reaction: 5μl plasmid + 85 μl cells + 200 μl SOB.</P><P><U>Resuspended plasmids:</U><UL><LI>BBa_K1362051 (plate 5,23I)</LI><LI>BBa_K1321090 (plate 5,12L)</LI><LI>BBa_K1319004 (plate 5,9O)</LI><LI>BBa_K300004 (plate 6,6C)</LI></UL></P><H5>10/08/2017, Thursday</H5><P> Put night cultures after adding nutrient medium (5-6 ml LB) with antibiotic (chloramphenicol), add 1 colony.<U>Used plasmids:</U><UL><LI>BBa_K1362051 (plate 5,23I)</LI><LI>BBa_K1321090 (plate 5,12L)</LI><LI>BBa_K1319004 (plate 5,9O)</LI><LI>BBa_K300004 (plate 6,6C)</LI></UL></P><H5>11/08/2017,Friday </H5><P> Night cultures of <U>plasmids</U>:
<UL><LI>BBa_K1362051(plate 5,23I)</LI><LI>BBa_K1321090(plate 5,12L)</LI><LI>BBa_K1319004(plate 5,9O)</LI><LI>BBa_K300004(plate 6,6C)</LI></UL></P><P>

    Purification of plasmids for sequencing.
      For alkaline lysis were used:
<UL><LI>Tris HCL - 250 μl</LI><LI>lysing solution - 250 μl</LI><LI>potassium acetate - 350 μl</LI></UL></P><P>
   For the rest plasmids’ purification:
<UL><LI>isopropanol - 850 μl</LI><LI>ethanol 70% - 850 μl</LI><LI>ethanol 96% - 850 μl</LI><LI>buffer Tris HCL - 60 μl</LI><LI>reagents for agarose gel electrophoresis</LI></UL></P><H5>17/08/2017, Thursday</H5><P> pOV-3Op plasmid
TTP-PAG plasmid

   Transformed 1 μl of pOV-3Op plasmid and 1 μl of TTP-PAG plasmid(with heat-shock).
Heat-shock: 
   Streaked ampicillin agar with 286 μl of TTP-PAG’s and pUV-3Op’s transformation reaction.
Transformation reaction: 1μl plasmid + 85 μl cells + 200 μl SOB.
</P><H5>18/08/2017, Friday</H5><P>I.The TTP-PAG’s transformation results were quite good, but pUV-3Op needed retransformation.
   Transformed 1 μl of pUV-3Op plasmid (with electroporation).
   Streaked kanamycin agar with 286 μl of pUV-3Op transformation reaction.
 Transformation reaction: 1μl plasmid + 85 μl cells + 200 μl SOB.</P><P>
   II.Put night cultures of TTP-PAG after adding nutrient medium with antibiotic to the colony.</P><H5>19/08/2017, Saturday</H5><P>  Put night cultures of pUV-30p after adding nutrient medium (5-6 ml) with antibiotic (kanamycin), add 1 colony.
Used plasmids:
<UL><LI>TTP-PAG</LI><LI>pUV-3Op</LI></UL></P><H5>20/08/2017, Sunday</H5><P> Purification of plasmids for sequencing.
For alkaline lycis were used:
<UL><LI>isopropanol - 850 μl</LI><LI>Tris HCL 250 μl</LI><LI>lysing solution 250 μl</LI><LI>potassium acetate - 350 μl</LI></UL></P><P>

For the rest plasmids’ purification:
<UL><LI>isopropanol - 850 μl</LI><LI>ethanol 70% - 850 μl</LI><LI>ethanol 96% - 850 μl</LI><LI>buffer Tris HCL - 60 μl</LI><LI>reagents for agarose gel electrophoresis</LI></UL></P><P>
Used plasmids:
<UL><LI>TTP-PAG</LI><LI>pUV-3Op</LI></UL></P><H5>28/08/2017, Monday</H5><P>I. Restriction:
<UL><LI>3 μl of plasmid</LI><LI>1 μl of restrictase BamHI</LI><LI>2 μl of 10x buffer </LI><LI>14 μl of water</LI></UL></P><P>
In the second eppendorf add 15  μl of water and no restrictase to compare two tracks and to see the impact of the enzyme on the plasmid.
Put the mixtures in a thermostat, temperature of 37 °C, 30 minutes.</P><P>
Used plasmids:
<UL><LI>pUV-3Op</LI></UL></P><P>
II. Agarose gel electrophoresis</P><H5>30/08/2017, Wednesday</H5><P>I. PCRMaterials:
<UL><LI>5x Phusion buffer 20 μl</LI><LI>nucleotides (final concentration=0.2 μM) 2 μl</LI><LI>primer BBGlucFor1:GCCCCAGATCTCAGGCCTGTTCTTCCGT (final concentration=1 μM) 10 μl</LI><LI>primer BBGlucRev1:GGCCGAGATCTGTAAAGACACTGGGAGT (final concentration=1 μM) 10 μl</LI><LI>matrix(final concentration=30 ng) 1 μl</LI><LI>polymerase 1.4 μl</LI><LI>water 60 μl</LI></UL></P><P>
Total:100 μl
</P><P>
   PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>29</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>95 °C</TD><TD>1) 95 °С2) 49 °C3) 72 °C</TD><TD>49 °C</TD><TD>72 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1) 30 sec2) 30 sec3) 40 sec</TD><TD>20 sec</TD><TD>3 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
II.Agarose gel electrophoresis
Cut the track with the DNA and put it to an eppendorf
</P><P>

   III.DNA gel Isolation:
<UL><LI>binder solution 600 μl x2(2 ependorfs)</LI><LI>isopropanol 200 μl x2</LI><LI>water 40 μl x2</LI><LI>gel with DNA 200 μg x2</LI><LI>the wash solution 800 μl x2</LI></UL></P><P>
  Result: concentration - 5 ng/μl
     as the concentration is too small we do a rePCR
</P><P>
   
   IV.rePCR:
<UL><LI>nucleotides (final concentration=0.2 μM) 1 μl</LI><LI>primer  BBGlucFor1:GCCCCAGATCTCAGGCCTGTTCTTCCGT (final concentration=1 μM) 5 μl</LI><LI>primer  BBGlucRev1:GGCCGAGATCTGTAAAGACACTGGGAGT (final concentration=1 μM) 5 μl</LI><LI>matrix(final concentration 3 ng) 3 μl</LI><LI>10x buffer 5 μl</LI><LI>polymerase 0.7 μl</LI><LI>water 35 μl</LI></UL></P><P>
total:59.7 μl
</P><P>

  V.DNA reprecipitation:
<UL><LI>mixture 59.7 μl</LI><LI>sodium acetate 5 μl</LI><LI>alcohol 96% 150 μl</LI><LI>centrifuge - 15 min (21 °C; 13.2 rpm)</LI><LI>in thermostat on 37 °C for 10-15 min</LI></UL></P><H5>31/08/2017, Thursday</H5><P>I. PCR for vector:
<UL><LI>10x buffer 5 μl</LI><LI>primer BBGlucFor1:GCCCCAGATCTCAGGCCTGTTCTTCCGT (final concentration=2 μM) 0.5 μl</LI><LI>primer BBGlucRev1:GGCCGAGATCTGTAAAGACACTGGGAGT (final concentration=2 μM) 0.5 μl</LI><LI>matrix 0.5 μl</LI><LI>polymerase 0.7 μl</LI><LI>nucleotides 1 μl</LI><LI>water 42 μl</LI></UL></P><P>
Total:50 μl</P><P>

   PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>17</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>95 °C</TD><TD>1) 95 °С2) 49 °C3) 72 °C</TD><TD>49 °C</TD><TD>72 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1) 30 sec2) 30 sec3) 40 sec</TD><TD>20 sec</TD><TD>3 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
II.DNA reprecipitation<UL><LI>mixture 50 μl</LI><LI>sodium acetate 5 μl</LI><LI>alcohol 96% 150 μl</LI><LI>centrifuge - 15 min (21 °C;13.2 rpm)</LI><LI>put in thermostat on 37 °C for 10-15 min</LI></UL></P><P>
DNA concentration=188,8 ng/μl
</P><P>

III.Agarose gel electrophoresis</P><P>

IV.Restriction</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>eppendorf</TD><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TD><UL><LI>restrictase BamH1 1.5 μl</LI><LI>vector 10 μl</LI><LI>10x buffer 5 μl</LI><LI>water 33.5 μl</LI></UL></TD><TD><UL><LI>restrictase BamH1 1.5 μl</LI><LI>insert 42 μl</LI><LI>10x buffer 5 μl</LI><LI>water 33.5 μl</LI></UL></TD><TD><UL><LI>restrictase BamH1 1.5 μl</LI><LI>vector with phosphatase 10 μl</LI><LI>10x buffer 5 μl</LI><LI>water 33.5 μl</LI></UL></TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P><UL><LI>put in thermostat on 37 °C for 30 min</LI></UL></P><P>


   V.DNA isolation:
<UL><LI>binder solution 250 μl x2(on two ependorfs)</LI><LI>isopropanol 125 μl x2</LI><LI>the reaction mixture 50 μl x2</LI><LI>the wash solution 700 μl x2</LI></UL></P><P>
DNA Concentration:
<UL><LI>vector-19.9 ng/μl</LI><LI>vector with phosphatase-13.5 ng/μl</LI><LI>insert-62.7 ng/μl</LI></UL></P><P>
VI.Agarose gel electrophoresis</P><P>

VII.Ligation:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>eppendorf</TD><TD>1(vector)</TD><TD>2(vector with phosphatase)</TD></TR><TR><TD>10x buffer</TD><TD>1 μl</TD><TD>1 μl</TD></TR><TR><TD>vector</TD><TD>0.5 μl</TD><TD>0.5 μl</TD></TR><TR><TD>insert</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>water</TD><TD>3.5 μl</TD><TD>3.5 μl</TD></TR><TR><TD>ligase</TD><TD>1 μl</TD><TD>1 μl</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P><UL><LI>leave the eppendorfs overnight in the thermostat on 14 °C</LI></UL></P><H5>01/09/2017, Friday</H5><P>I.Vector restriction check:
 agarose gel electrophoresis(150 w)
<UL><LI>marker 4 μl(concentration:25 ng/μl)</LI><LI>gel-1.5 %</LI></UL></P><P>
      
    II. Transformed competent cells(XL1Blue) with 3 μl of ligate
                                                  With electroporation:shock 3-5 sec
 
                        Wash  with SOB 1000 μl
Put eppendorfs to shaker on 1 hour
                        Transformation reaction - 1045  μl 
</P><P>
     III.Streaked with kanamycin agar with 50 μl of transformation reaction
                        
          Wring the solution, take 900 µl of supernatant ,mix it
          Streaked with kanamycin agar with the rest of the transformation reaction(50 μl)
                          
                        Put eppendorfs to the thermostat  for night</P><H5>03/09/2017, Sunday</H5><P>Nothing has grown
</P><H5>04/09/2017, Monday</H5><P>
I.Ligate reprecipitation:
<UL><LI>ligate of vector - 7 μl</LI><LI>co-precipitator 5xSatellite red - 1 μl</LI><LI>NaAC - 0.7 μl</LI><LI>EtOH 97% - 21 μl</LI><LI>centrifugation - 15 min on 4 °C</LI><LI>to the thermostat on 37 °C</LI></UL></P><P>
            II.Ligation:
<UL><LI>vector with phosphatase - 2.5 μl</LI><LI>10x buffer - 2 μl</LI><LI>ligase santific thermo fisher - 2 μl</LI><LI>water - 3.5 μl</LI></UL></P><P>
the total volume of the reaction:20 μl
               leave the eppendorfs overnight in thermostat on 14 °C

</P><H5>05/09/2017, Thuesday</H5><P>I.Transformed 45 μl of competent cells(XL1Blue) with vector,vector with phosphatase and for control:plasmid EVST yellow fluorescently(4.5 KB)
                                                                      With electroporation:shock - 3-5 sec
          Add 4 μl of concentrated ligate(after reprecipitation) and repeat electroporation
          Wash with SOB 1000 μl
          Put eppendorfs to shaker on 1 hour
</P><P>
     II.Streaked with kanamycin agar with 300 μl of transformation reaction(50 μl of cells and 250 μl of SOB)   -&gt;4 petri dishes

</P><H5>06/09/2017, Thursday</H5><P>
     I.PCR:</P><P>Control PCR with the cells from the medium (toothpick smear from agar without colonies).</P><P>Total amount of samples - 18, also negative and positive controls.</P><P>For each sample:</P><P>V = 10 μl</P><P>Evrogen Screen mix 5x = 2 μl</P><P>mQ = 7.6 μl</P><P>primers:</P><P>forward (10 mM) = 0.2 μl</P><P>reverse (10 mM) = 0.2 μl</P><P>
PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>18</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94 °C</TD><TD>1) 94 °С2) 49 °C3) 82 °C</TD><TD>49 °C</TD><TD>82 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1) 30 sec2) 20 sec3) 25 sec</TD><TD>20 sec</TD><TD>3 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
II.Agarose gel electrophoresis(160 w):
<UL><LI>gel-1.5 %</LI></UL></P><P>
Result:negative
</P><P>
   III.rePCR</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>4</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94 °C</TD><TD>1) 94 °С2) 49 °C3) 82 °C</TD><TD>49 °C</TD><TD>82 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1) 30 sec2) 20 sec3) 25 sec</TD><TD>20 sec</TD><TD>3 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
   IV.Agarose gel electrophoresis


Result:negative(ligase could die in the freezing)

We decided to develop a lot of vector and insert to put a restriction and ligation with a normal volume
</P><P>
    V.PCR of insert:
   materials:
<UL><LI>buffer - 10 μl</LI><LI>nucleotides - 2 μl</LI><LI>matrix (20 ng/μl) - 0.5 μl</LI><LI>primers - 10 μl x2 (10 μM)</LI><LI>MQ - 66 μl</LI><LI>Tersus of polymerase - 1.4  μl</LI></UL></P><P>
total:50 μl x2 (100 μl)
</P><P>

PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>17</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94 °C</TD><TD>1) 94 °С2) 49 °C3) 82 °C</TD><TD>49 °C</TD><TD>82 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1) 30 sec2) 20 sec3) 25 sec</TD><TD>20 sec</TD><TD>3 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
VI.Reprecipitation ligate:
<UL><LI>ligate - 20 μl</LI><LI>co-precipitator Satellite red - 1 μl</LI><LI>NaAC - 2 μl</LI><LI>EtOH 97% - 60 μl </LI><LI>centrifugation - 15 min</LI><LI>to the thermostat on 37 °C</LI></UL></P><P>
Used:
<UL><LI>ligate transformation - 4 μl</LI><LI>source vector for further developments - 2 μl</LI></UL></P><P>
Unfortunately,the results weren’t good again
</P><P>
VII.PCR screen(petri dish with biobrick C3)
     8 pieces - negative
</P><P>

VIII. Transformed <B>1 μl</B> of pUV-3Op plasmid<B>(with electroporation)</B>
   Streaked kanamycin agar with 286 μl of pUV-3Op transformation reaction.
 Transformation reaction: 1μl plasmid + 85 μl cells + 200 μl SOB.
  Put night cultures of pUV-3Op
</P><P>
IX.Plasmid purification:
<UL><LI>liquid medium - 50 μl</LI><LI>spin in tube(50   μl)</LI><LI>take away the supernatant</LI><LI>binder solution - 2 ml on 100 ml of cells</LI><LI>neutralizing solution - 750 μl  </LI><LI>the wash solution - 500 μl</LI><LI>centrifugation 15 min</LI></UL></P><H5>07/09/2017, Thursday</H5><P>
     I.Plasmid purification:

<UL><LI>liquid medium - 80 ml</LI><LI>spin in tubes</LI><LI>take away the supernatant</LI><LI>binder solution - 2ml on 100 ml of cells</LI><LI>neutralizing solution - 750 μl </LI><LI>the wash solution - 500 μl </LI><LI>centrifugation 15 min</LI><LI>ligat of each vector spin 2 times for 30 min</LI><LI>to wash 700 μl wash solution</LI><LI>spin</LI><LI>add 50 μl of water and leave for a minute</LI><LI>spin</LI></UL></P><P>
II.The PCR of 150 μl of  insert(result):</P><H5>12/09/2017, Tuesday</H5><P>
     I.the PCR products reprecipitation:<UL><LI>PCR-products - 150 μl</LI><LI>sodium acetate 10% - 15 μl </LI><LI>alcohol 3:1 volume - 450 μl</LI><LI>to spin for 15 min on 13.2 cph</LI><LI>to select supernatant</LI><LI>to put into the thermostat on 37 °C(10 min)</LI><LI>to dissolve in 50 μl of water</LI></UL></P><P>
1.PUV3Op after restriction
2.basic PUV3Op 
</P><P>II.Agarose gel electrophoresis:
<UL><LI>insert - 1 μl </LI><LI>1 Kb Ladder - 4 μl</LI><LI>100 b Ladder - 4 μl</LI></UL></P><P>Result: NanoDrop:
<UL><LI>insert - 550 ng/μl(total:50 μl)</LI><LI>vector - 340 ng/μl</LI></UL></P><P>III.Restriction </P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Restrictively DNA</TD><TD>vector(pUV3Op)</TD><TD>insert(phytase)</TD></TR><TR><TD>10x buffer FD</TD><TD>10 μl</TD><TD>10 μl</TD></TR><TR><TD>restrictase</TD><TD>BamH1 - 4 μl</TD><TD>Bgl2 - 4 μl</TD></TR><TR><TD>alkaline phosphatase</TD><TD>4 μl</TD><TD>-</TD></TR><TR><TD>DNA</TD><TD>30 μl</TD><TD>18 μl</TD></TR><TR><TD>MQ(deionized water)</TD><TD>52 μl</TD><TD>68 μl</TD></TR><TR><TD>total volume </TD><TD>100 μl</TD><TD>100 μl</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P><UL><LI>put into the thermostat on 37 °C for 30 min</LI></UL></P><P>IV.Agarose gel electrophoresis

</P><P> V.Restrict allocation on the columns
<UL><LI>5 volumes of binder solution to 1 volume of reaction mixture</LI><LI>isopropanol - 250 μl(to the restrict of insert)</LI><LI>put into the collection tube and centrifuge for 30 sec(maximum speed)</LI><LI>replace the filtrate </LI></UL></P><P>
(maximum volume of column-800  μl;after each application of an aliquot the eppendorf needs centrifugation)</P><P><UL><LI>700 μl of wash solution </LI><LI>centrifuge 30 sec</LI><LI>replace the filtrate</LI><LI>centrifuge empty column for 60 sec for the full replacement off the wash solution</LI><LI>put the column to another eppendorf</LI><LI>30 μ of water to the centre of membrane</LI><LI>centrifuge for 30 sec</LI><LI>replace the filtrate</LI><LI>re-apply the aluate on the column</LI><LI>centrifuge for 30 sec</LI></UL></P><P>The concentrations: of vector=77 ng/μl
                                 of insert=215 ng/μl</P><P>VI.The ligation of restriction product:
<UL><LI>vector- 1 μl</LI><LI>insert - 28 μl</LI><LI>buffer - 5 μl</LI><LI>MQ - 11 μl</LI></UL></P><P>
total:50 μl</P><H5>14/09/2017, Thursday</H5><P>
     I.Ligat reprecipation:<UL><LI>water - 30 μl</LI><LI>SattelineRed - 2 μl</LI><LI>PCR products - 50 μl</LI><LI>sodium acetate - 5 μl</LI><LI>EtOH - 150 μl</LI><LI>to spin for 15 min on 13.2 cph</LI><LI>to replace supernatant</LI><LI>to put into the thermostat for 10 min </LI><LI>to dissolve in 50 μl of water</LI></UL></P><P>
     I.Ligat transformation into cells:<UL><LI>ligat - 3 μl</LI><LI>competent cells XL1Blue - 45 μl</LI><LI>with electroporation</LI><LI>SOB - 1000 μl</LI></UL></P><P>
Streaked kanamycin agar with 50 μl of initial mixture(1 petri dish) and 50 μl of 100 times concentrated mixture.
</P><H5>15/09/2017, Friday</H5><P>     Result:on the 1 petri dish 20 colonies have grown,on the 2 - 300 colonies</P><P>     I.PCR with primers of insert:
1-40 colonies
K(-)  control - :4 sm touch of petri dish
K(+) control +:0.2 μl of ligat</P><P>V=10 μl
<UL><LI>Screen mix 5x-2 μl on reaction</LI><LI>water - 7.6 μl</LI><LI>primer 1 - 0.2 μl</LI><LI>primer 2 - 0.2 μl</LI></UL></P><P>     PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>22</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94 °C</TD><TD>1)94 °C
2)49 °C
3)72 °C</TD><TD>72 °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1)30 sec
2)20 sec
3)30 sec</TD><TD>1 min </TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>


     II.Plasmid purification:
<UL><LI>liquid medium - 80 μl </LI><LI>spin in tube - 50 ml </LI><LI>take away the supernatant</LI><LI>binder solution - 0.5 μl on 15 ml of cells</LI><LI>lysing solution - 500 μl</LI><LI>neutralizing solution - 750  μl</LI><LI>centrifugation for 15 min</LI><LI>spin every ligat 2 times for 30 min</LI><LI>wash 700  μl of neutralizing solution</LI><LI>replace liquid from the collection tubef</LI><LI>spin</LI><LI>change the collection tube</LI><LI>water - 15  μl</LI><LI>spin for 30 sec on maximum speed</LI><LI>apply the eluate to the column and re-centrifuge</LI></UL></P><P>
concentration of vector colonies:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>№</TD><TD>ng/ μl</TD></TR><TR><TD>6</TD><TD>1400</TD></TR><TR><TD>9</TD><TD>840</TD></TR><TR><TD>18</TD><TD>970</TD></TR><TR><TD>20</TD><TD>1300</TD></TR><TR><TD>32</TD><TD>1200</TD></TR><TR><TD>34</TD><TD>2250</TD></TR><TR><TD>36</TD><TD>880</TD></TR><TR><TD>38</TD><TD>1000</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>     III.Agarose gel electrophoresis:<UL><LI>samples - 2 μl</LI><LI>ladder - 4 μl</LI></UL></P><P>sequencing:
result:sequencing has shown that our insert has embedded in another vector(pTagYFP-C).It can be explained with the possibility of accidental interference of the cells with another vector of resistance to kanamycin during re-building plasmids</P><H5>28/09/2017, Thursday</H5><P>
     I.Obtaining vector:
The vector was re-established in XL1Blue competent cells and selected as described.</P><P>
     II.re-PCR of insert as described above</P><P>
     III.Restriction of vector and insert:</P><UL><LI>primer 1 -0.2 μl</LI><LI>primer 2 - 0.2 μl</LI><LI>control(agar without colonies)</LI><LI>Screen mix 5x - 2 μl for reaction</LI><LI>water - 7.6 μl</LI><LI>20 reaction volumes + 2 extra volumes</LI></UL></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Restricted DNA</TD><TD>Vector(PUV-3Op)</TD><TD>Insert(phytase)</TD></TR><TR><TD>10x buffer FD</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>Restrictases</TD><TD>2,5 μl(bamH1)</TD><TD>2,5 μl(Bgl2)</TD></TR><TR><TD>Alkaline phosphatase</TD><TD>2,5 μl</TD><TD>-</TD></TR><TR><TD>DNA</TD><TD>15 μl</TD><TD>15 μl</TD></TR><TR><TD>MQ</TD><TD>25 μl</TD><TD>27,5 μl</TD></TR><TR><TD>V total</TD><TD>50 μl</TD><TD>50 μl</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
     IV.Ligation:
<UL><LI>vector 200 ng(1 μl)</LI><LI>insert 2000 ng(7 μl)</LI><LI>ligase T4 DNA Ligase 1 U/μl(1 μl)</LI></UL></P><P>
V total:10 μl</P><P>Without DNA isolation </P><H5>03/10/2017, Tuesday</H5><P>
     I.Transformation:Transformed <B>3 μl</B> of ligat <B>(with electroporation)</B>
   Streaked kanamycin agar with 1048 μl of ligat transformation reaction.
   Transformation reaction: 3 μl of ligat + 45 μl cells + 1000 μl SOB.

<UL><LI>to put into the shaker on an hour</LI><LI>to spin for 15 sec at maximum speed</LI><LI>to take away the 1000 μl of supernatant</LI><LI>to spread the residue of the reaction on the Petri dish<B>(kanamycin agar)</B></LI></UL></P><H5>04/10/2017, Friday</H5><P>
     I.PCR-screening colonies:</P><P><UL><LI>1-20 colonies from the dish A </LI><LI>K+ - (+) control - 0,2 μl of ligat</LI></UL>
     V = 10 μl
</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>	For one reaction</TD><TD>	Master mix</TD></TR><TR><TD>Scrin mix 5x</TD><TD>2 μl</TD><TD>44 μl </TD></TR><TR><TD>mQ</TD><TD>7,6 μl</TD><TD>162,7 μl</TD></TR><TR><TD>Primers BBGlucFor1 and ConstrRev</TD><TD>0,2 μl + 0,2 μl</TD><TD>4,4 μl + 4,4 μl</TD></TR><TR><TD>V total </TD><TD>10 μl</TD><TD>220 μl</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>2</TD><TD>20</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94  °C</TD><TD>1)94  °C
2)46  °C
3)72  °C</TD><TD>1)94  °C
2)48  °C
3)72  °C</TD><TD>72  °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1)30 sec
2)20 sec
3)1 min 30 sec</TD><TD>1)30 sec
2)20 sec
3)1 min 30 sec</TD><TD>2 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>
     II.Agarose gel electrophoresis of PCR-products:
<UL><LI>samples - 2 μl</LI><LI>ladder - 4 μl</LI></UL></P><P>
colonies with a positive result:2,9,10
</P><H5>05/10/2017, Saturday</H5><P>
     I.Plasmid purification of the 2,9,10 colonies:
<UL><LI>liquid medium - 80 μl </LI><LI>spin in tubes  (50 ml) </LI><LI>take away the supernatant</LI><LI>binder solution - 0.5 μl on 15 ml of cells</LI><LI>lysing solution - 500 μl</LI><LI>neutralizing solution - 750  μl</LI><LI>centrifugation for 15 min</LI><LI>spin every ligat 2 times for 30 min</LI><LI>wash 700  μl of neutralizing solution</LI><LI>replace liquid from the collection tube</LI><LI>spin</LI><LI>change the collection tube</LI><LI>water - 30  μl</LI><LI>spin for 30 sec on maximum speed</LI><LI>apply the eluate to the column and re-centrifuge for 30 sec</LI></UL></P><H5>09/10/2017, Monday</H5><P>I.Additional PCR-screening colonies:
As we had only 3 colonies after the previous PCR-screening, we decided to repeat the procedure.</P><P><UL><LI>a1-a20 colonies from the dish A</LI><LI>K+ - (+) control - 0,2 μl of ligat</LI></UL></P><P>
     V = 10 μl</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>For one reaction</TD><TD>Master mix</TD></TR><TR><TD>Scrin mix 5x</TD><TD>2 μl</TD><TD>44 μl </TD></TR><TR><TD>MQ</TD><TD>7,6 μl</TD><TD>162,7 μl</TD></TR><TR><TD>Primers BBGlucFor1 and ConstrRev</TD><TD>0,2 μl + 0,2 μl</TD><TD>4,4 μl + 4,4 μl</TD></TR><TR><TD>V total</TD><TD>10 μl</TD><TD>220 μl</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>PCR machine programm:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>Segment</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Cycles</TD><TD>1</TD><TD>2</TD><TD>20</TD><TD>1</TD></TR><TR><TD>Temperature</TD><TD>94  °C</TD><TD>1)94  °C
2)46  °C
3)72  °C</TD><TD>1)94  °C
2)48  °C
3)72  °C</TD><TD>72  °C</TD></TR><TR><TD>Time</TD><TD>2 min</TD><TD>1)30 sec
2)20 sec
3)1 min 30 sec</TD><TD>1)30 sec
2)20 sec
3)1 min 30 sec</TD><TD>2 min</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><P>II.Agarose gel electrophoresis of PCR-products:
<UL><LI>samples - 2 μl</LI><LI>ladder - 4 μl</LI></UL></P><P>
         colonies with a positive result:a1,a13,a20</P><H5>10/10/2017, Tuesday</H5><P>I.Build Up colonies(a1,a13,a20):
            in 15 μl of LB <I>with kanamycin</I></P><H5>11/10/2017, Wednesday</H5><P>I.Plasmid purification of the a1,a13,a20 colonies:
<UL><LI>liquid medium - 80 μl </LI><LI>spin in tubes (50 ml) </LI><LI>throw away the supernatant</LI><LI>binder solution - 0.5 μl on 15 ml of cells</LI><LI>remove to the 2 μl tube</LI><LI>lysis buffer - 500 μl</LI><LI>neutralizing solution - 750  μl</LI><LI>centrifugation for 15 min</LI><LI>spin every ligat 2 times for 30 min</LI><LI>wash 700  μl of neutralizing buffer</LI><LI>replace liquid from the collection tube</LI><LI>spin</LI><LI>change the collection tube</LI><LI>water - 30  μl</LI><LI>spin for 30 sec on maximum speed</LI><LI>apply the eluate to the column and re-centrifuge for 30 sec</LI></UL></P><P>Determination of the concentration of selected plasmids on the spectrophotometer:</P></DIV><DIV class="column half_size"><TABLE><TBODY><TR><TD>№</TD><TD>ng/μl</TD></TR><TR><TD>1</TD><TD>1118</TD></TR><TR><TD>2</TD><TD>1044</TD></TR><TR><TD>3</TD><TD>1048</TD></TR></TBODY></TABLE></DIV><DIV class="column notebook"><H5>18/10/2017, Wednesday</H5><P>I.the DNA was given to the transformation in yeast Yarrowia lipolytica</P><P>II. Preparation of an agarose gel:

<UL><LI>phytate sodium, 2%(200 mM)</LI><LI>tris-acetate buffer (pH 5,5),with 1% of agarose</LI><LI>CaCl2 (calcium chloride)-5 %</LI><LI>pour indicator medium into the Petri dishes(25 μl of medium on Petri dish with 9 sm in diameter)</LI><LI>punch 15 holes in agarose gel with medium with a diameter of 5 mm </LI><LI>pour the control samples</LI></UL></P><P>the control: 4, 16, 64 times dilution and a sample without dilution</P><P><UL><LI>1,2,4 - sample 11</LI><LI>3,8,12 - sample 13</LI><LI>5,6,7 - sample 14</LI><LI>9,10,11 - sample 15</LI><LI>13,14,15 - sample 16</LI></UL></P><H5>23/10/2017, Monday</H5><P>I. Preparation of an agarose gel:
<UL><LI>phytate sodium, 2%(200 mM)</LI><LI>tris-acetate buffer(pH 5,5),with 1 % of agarose</LI><LI>CaCl2 (calcium chloride)-5 %</LI><LI>pour indicator medium into the Petri dishes(25 μl of medium on Petri dish with 9 sm in diameter)</LI><LI>punch 15 holes in agarose gel with medium with a diameter of 5 mm </LI><LI>pour the studied samples (with modified and original phytase, heat before and after cell destruction)</LI></UL></P><P>Control</P><P>Positive result:6 holes with studied samples</P><P>(For more details, see the page “Demonstration”)</P><H5>24/10/2017, Tuesday</H5><P>I.Cloning biobricks 2488000 and 2488002:</P><P>Biobricks were cloned with the restriction sites Spel and Pstl, as the instructions posted on the IGEM website were misunderstood.
Restriction sites were introduced into the structure with PCR using synthetic primers.</P><P>II.Cloning biobrick 2488001:</P><P>Biobrick was cloned using Gibson Assembly MasterMix and synthetic primers created especially for this purpose.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>