<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Moscow_RF_Contribution skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Moscow RF/Contribution</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Moscow_RF"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Moscow_RF/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT DESIGN
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Moscow_RF/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Moscow_RF/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?id=2488"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A><A href="https://2017.igem.org/Team:Moscow_RF/Thanks"><DIV class="menu_button direct_to_page">
				THANKS
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="column full_size"><H1>Contribution</H1></DIV><DIV class="column full_size"><H1> Characterization of part BBa_K842003 (FliA)  </H1><P>σ<SUP><SMALL>32</SMALL></SUP> and σ<SUP><SMALL>28</SMALL></SUP> are classified as group 3 sigma factors. Sigma factors of this group contain regions 2, 3 and 4 and are very diverse in their structure. σ<SUP><SMALL>28</SMALL></SUP> of E. coli is involved in synthesis of flagella (Paget, 2015). The function of this sigma factor is determined significantly by reduced promotor melting capability. It has been demonstrated that strict consensus in the promoter region is very important for σ<SUP><SMALL>28</SMALL></SUP>: this ensures strict specificity of recognition when transcription is initiated (Koo et al, 2009).</P><P>Demonstrated consensus:</P></DIV><DIV class="column full_size"><P><I>(doi:10.1093/nar/gkm456)</I></P><P>It may be attributed to the fact that this type of sigma factors can lack one or more amino acid residues required for the looping out of -11 А when promoter region melting is initiated (Feklístov <I>et al</I>, 2014).</P></DIV><DIV class="column full_size"><P>Fig 1. Amino acid sequences of alternative sigmas aligned along the basic one. The red box is region 3.2.<I>(10.1093/nar/gkt1384)</I></P><P>When performing transcription involving this sigma sub-unit it is necessary to consider the factors that may influence the product output (that is RNA transcript in our case). The main factors influencing the function of the enzyme are salt concentration in the reaction buffer and the temperature of the experiment. Standard conditions for transcription are considered to be 40-50 mM NaCl buffer. However, other salts and concentrations have been tested and it has been discovered that when using KCl instead of NaCl the reaction optimum is not offset and remains at the concentration rate of 40-50 mM. But when using sodium glutamate and sodium acetate instead of sodium chloride the reaction optimum shifts to high concentration regions (300-400 mM). Besides, the influence of temperature on reaction efficiency should be considered. Here it should be noted that the function optimum of RNA polymerase containing an s70 sub-unit is observed at 37°C while for an enzyme containing an s38 sub-unit this value goes down to 30°C (for achieving maximum transcription reaction efficiency). (Promoter Selectivity of Escherichia coli RNA Polymerase sF Holoenzyme Involved in Transcription of Flagellar and Chemotaxis Genes TAPAS KUMAR KUNDU,1 SHUICHI KUSANO,1,2 AND AKIRA ISHIHAMA1 *)  </P><P>As sigma subunits account for promoter binding specificity, for conducting experiments with RNAP it is important to understand different promoters’ strength. Therefore, we decided to examine and compare the activity of E. coli RNA polymerase with two different promoters characterized for this sigma - flgMp and modAp1. It was discovered that promoter complex stability for modAp1 was higher than for flgMp. The half-life of flgMp and modAp1 promoters was 38s and 1800s, accordingly. Standard deviations were plotted based on the results of three experiments.</P><DIV class="column full_size"><P>Besides, it was demonstrated that the efficiency of synthesis differed for two promoters as well: the efficiency of synthesis for flgMp was higher than that one for modAp1.</P><P><B>Experiment design:</B></P><P>Promoter sequences were taken from RegulonDB and amplified using PCR with genomic DNA of E. coli. </P><P>Highlights: -35 (yellow), -10 (green) and +1 base (purple).</P><P>To measure the kinetics of promoter complex dissociations, competitive inhibition of transcription by heparin was used. Full-length products were synthesized in the following conditions: RNAP core enzyme – 200 nmol, σ-subunit – 500 nmol. Heparin was being added to the reaction mixture containing an open RNAP promoter complex with a correspondent promoter until its concentration reached 10mcg/ml. At different intervals after that (20′′, 40&quot;, 1′, 3′, 10′, 30′) samples were taken from the mixture and transcription was initiated in them by adding a mixture of nucleotides: ATP, CTP and GTP in equal concentrations (200 µmol), 10 µmol of UTP, and 2.5 µCi of α-[32-P]-UTP (3000 Ci/mmol). Each sample was incubated for 3 minutes at 37°С (in BIOSAN WB-4MS water thermostat); after that the reaction was terminated by adding equal amounts of stop-buffer. In the beginning of the experiment a ‘null’ sample had been taken from the reaction mixture (control without heparin) to which the same mixture of nucleotides was added. The results were visualized using a phosphorimager; the quantitative analysis of the results was carried out with the help of ImageQuant</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>