<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Optogenetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Optogenetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Optogenetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Optogenetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Optogenetics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Optogenetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Optogenetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><NAV id="igemuwmenu-wrap"><UL id="igemuwmenu"><LI><A href="https://2012.igem.org/Team:Washington">Home</A></LI><LI><A href="https://2012.igem.org/Team:Washington">UW 2012</A></LI><LI><A href="https://2011.igem.org/Team:Washington">UW 2011</A></LI><LI><A href="https://2010.igem.org/Team:Washington">UW 2010</A></LI><LI><A href="https://2009.igem.org/Team:Washington">UW 2009</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Washington">UW 2008</A></LI><LI><A href="https://2012.igem.org/Main_Page">iGEM Homepage</A></LI></UL><LI><A href="https://2012.igem.org/Team:Washington/Plastics">Plastic Degradation</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#App">App</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#Results">Results Summary</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#Future">Future Directions</A></LI><LI><A href="http://students.washington.edu/cnt/pmwiki/pmwiki.php?n=Main.HomePage">Turbidostat Wiki</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Plastics#Parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu">Flu Binders</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#App">App</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#Results">Results Summary</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#Future">Future Directions</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Flu#Parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics">Optogenetics</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#App">App</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#Results">Results Summary</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#Future">Future Directions</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Optogenetics#Parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Outreach">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Protocols#General_Protocols">General</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Protocols#Plastic_Degradation">Plastic Degradation</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Protocols#Flu_Binders">Flu Binders</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Protocols#Optogenetics">Optogenetics</A></LI><LI style="border-right:0; box-shadow:0px 0 0;"><A href="https://2012.igem.org/Team:Washington/Team:Members">About</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Team:Members">The Team</A></LI><LI><A href="https://2012.igem.org/Team:Washington/Safety">Safety</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=Washington">Parts Registry</A></LI></NAV></P><H2><SPAN class="mw-headline" id="Optogenetics:_A_hands-free_approach_to_protein_regulation"><I>Optogenetics: A hands-free approach to protein regulation</I></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">An image of two peppers by shining red and green light on bacteria with light receptors.[3]</DIV></DIV></DIV><H1 id="Background"><SPAN class="mw-headline" id="Background">Background </SPAN></H1><P>Bacterial metabolic pathways constitute a foundation on which to build biological processes that perform useful tasks such as the production of drugs, biofuels, or the degradation of harmful compounds. Often, to achieve an optimally tuned system, expression levels of each component must be varied over a wide range. The burgeoning field of optogenetics affords researchers the ability to control gene expression with light. In addition to being low cost,  controlling of gene expression with light has a number of advantages over standard chemical methods of gene control. Among these are the ability to finely tune induction levels through changes in intensity, as well as the ability to quickly and completely remove the input. Further, many light-induced expression systems are reversible depending on the wavelength used for illumination. Thus one of the goals of the 2012 iGEM team is to implement a light inducible system which we hoped to apply to the tuning of multiple metabolic pathways. In addition, we wanted to make the tools to control optogenetic systems readily available and easier to access.
</P><H1 id="App"><SPAN class="mw-headline" id="Our_App:_E._colight.5BTop.5D">Our App: E. colight<A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><CENTER><A href="http://www.youtube.com/watch?v=8h4RbDjTDYg" class="external text" rel="nofollow"><B><FONT size="4">See our app in action!</FONT></B></A></CENTER><DIV class="thumb tleft"><DIV class="thumbinner" style="width:248px;"><DIV class="thumbcaption">We developed an <A href="https://play.google.com/store/apps/details?id=com.UWIgem.test2d" class="external text" rel="nofollow">app for the Android OS</A> that allows up to control gene expression with light.</DIV></DIV></DIV><P>At the beginning of our project we wanted to make a system for high-specificity light control without the common problems such systems came with: High cost, difficult assembly, and very little reproducibility. With these goals in mind, we worked with Max Gelb to create E. colight.
</P><P>E. colight was designed entirely from scratch in order to illuminate bacterial cultures with different types of light. The light source is on as either blue, green, or red. In the app, any given color of light can be shone between 0 and 60,000 milliseconds, which will illuminate the culture accordingly. Intensity of the light can be adjusted between 0 and 100%. All three colors are individually controllable, making high-specificity multichromatic tuning very easy.
</P><P>The app has two settings which allow it to control bacterial cultures: One is its petri dish setting, which is a single light source made for a petri dish with an 8.5mm diameter. The other is a 96-well plate, which means it has <I><B>96 individually-controllable light sources</B></I>, which lends the potential to run huge numbers of tests in tandem. The size of both the wells and the petri dish is flexible and can be adjusted between 0 and 100% of its original size.
</P><H1 id="Methods"><SPAN class="mw-headline" id="Methods.5BTop.5D">Methods<A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The basic operation of our light system.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_a_light_inducible_protein_expression_system_for_the_tuning_of_biological_pathways">Characterization of a light inducible protein expression system for the tuning of biological pathways</SPAN></H3><P>Chromatic tuning is based on light-induced protein interactions. The most basic of these systems is a monochromatic system, in which a specific wavelength of light changes a protein's conformation temporarily, causing it to interact differently with its environment. Our system involves the use of a photoreceptor CcaS attached to a chromophore and CcaR, a response regulator. The chromophore, when illuminated with green light (λ=535nm) will undergo a conformational change leading to autophosphorylation of the photoreceptor CcaS. Increased autophosphorylation leads to activation of the response regulator via phosphorylation. The activated response regulator will then bind to the promoter P<SUB>cpcG2</SUB>, activating transcription and producing beta-galactosidase. Beta-galactosidase will cleave the S-gal, creating a black precipitate and allowing us to know visually that our light sensor is working. Red light (λ=672nm), on the other hand, changes the protein conformation such that it resists phosphorylation, thus not binding to the promoter and not allowing gene expression. Any gene placed after the promoter can be theoretically controlled by the exposure of λ=535nm and λ=637nm light to the system.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">By inserting the light system into <I>E. coli</I>, we can test its functionality by measuring pigment density.</DIV></DIV></DIV><P>A blue light sensor (Lov_Tap BBa_K191003) was found in the registry and transformed to be used as one of our main components to create a functioning blue light sensor, activated at 470 nm wavelengths of light. The part taken from the registry was joined with a TetR inverter, since originally the blue light sensor repressed gene expression. The newly added TetR inverter then suppressed the regulator that came with Lov_Tap, allowing transcription to occur under stimulation of blue light. We also included sfGFP which is our readout protein, allowing the used to know whether or not the light sensor was properly functioning.  Near the end, once we had all the pieces we did a Gibson assembly to join our pieces. At that time we noticed that our blue light sensor had a point mutation that needed to be fixed. The point mutation was then fixed through several PCRs that retrieved two parts of the sensor without the mutation. Once those two parts were obtained, they were then fused together in the Gibson assembly along with being Gibson assembled together with the inverter and sfGFP. Future iGEM projects could be focused around detaching sfGFP as the output protein and attaching the light sensor to other parts of the e.coli gene, allowing light illuminated control of genetic pathways.
</P><H3><SPAN class="mw-headline" id="Building_Optogenetic_Tools">Building Optogenetic Tools</SPAN></H3><P>The current swath of tools available to illuminate bacterial cultures in a controlled manner often necessitate the purchase of specialized equipment or materials. Further, devices must often be both constructed, and fully characterized by the researcher prior to conducting any experiments to ensure reproducibility. To circumvent these problems, we sought to develop an application whose accessibility played off of modern technological conveniences. In recent years, mobile technology has become increasingly ubiquitous. Each year sees the release of many new tablets, phones, and small computers, including some that are relatively cheap. Tablets are excellently-sized, cost-effective tools for use with bacterial cultures. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">AMOLED displays have organic cathode layers that provide high-intensity light when stimulated by a TFT array.[1]</DIV></DIV></DIV>After a bit of research we decided to install our app on the Samsung Galaxy Tab 7.7. We chose the tablet because of it's relatively low cost and because of the brightness and contrast of the display. The Samsung Galaxy Tab 7.7 uses a Super AMOLED (Active-Matrix Organic Light-Emitting Diode) display, which emits brighter and more intense light than LCD displays used on other tablets. Contrast is also elevated under an AMOLED display, as the color black is represented by turning an LED off, instead of the dark-grey substitute LCD displays use. The downside to AMOLED displays is that the organic material in the screen decays over the course of years, which causes uneven color shifts and imprints in the screen, but we decided that the benefits of the AMOLED display outweighed this detriment. The app was programmed by Max Gelb.<DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">An example petri dish on top of the working app.</DIV></DIV></DIV><P>
Our solution was to design and build a software application for use on tablet devices that is able to shine light of different frequencies in different conformations (i.e. 96 well plate, petri dish) to enable controlled and reproducible characterization and testing of optogenetic pathways. The application that we designed can generate different wavelengths of light. To use the application we simply chose well or plates, and then chose wait time between flashes of colors in milliseconds of wait time and color intensities of each color from 0-100. The bacteria can then be grown overnight on top of the app. The app is currently available in the google play store for free and provides a convenient way for anyone interested in biological sciences to test their optogenetic systems. Because iGEM teams and labs will have a set budget, we hope that this tool becomes useful to all that want to pursue science.
</P><H3><SPAN class="mw-headline" id="Experimental_Description">Experimental Description</SPAN></H3><P>E. coli was transformed with pJT122 and pJT106b, which contained the lacZ gene, the green light sensor, and the system's chromophores.[3] In order to characterize the genes, we put them through three experiments: An assay of pigment concentration based on bacterial concentration, an assay of pigment concentration based on absorbed light, and a test of light leakage between wells.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">In the bacterial concentration assay, as the amount of E. coli in a well increased, so too did the expression of lacZ.</DIV></DIV></DIV>For the assay based on bacterial concentration, we grew an overnight culture of JT2 cells transformed with pJT122 and pJT106b overnight in buffered LB. Cultures were grown in tubes wrapped with aluminum foil to ensure no light interfered with gene expression during growth. We then diluted the cultures in a 1:5 serial dilution in a row on a 96-well plate, and exposed them only to green light from E. colight for a 15-hour incubation. As seen in the picture to the right, the cells visibly demonstrated an increase in pigment production (a sign of increased lacZ activity).<DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">lacZ is downregulated by red light in our modified bacteria. Layout of plates, from left to right: constant illumination under red light, green light, or grown in the dark.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Image analysis of the petri dishes above. The x axis indicates treatment (red light, green light, or darkness). The y axis indicates darkness. Red-treated cells are much lighter than the other treatments, indicating red light downregulation of lacZ expresion.</DIV></DIV></DIV><P>
In order to test the functionality of the green light activator and red light repressor, we plated the same cells onto three petri dishes. One petri dish was incubated in green light, one in red light, and one in total darkness for 15 hours. The dish exposed to red light demonstrated very strong downregulation, while the dish exposed to green light showed strong upregulation.
</P><P>When performing characterization tests on 96-well plates, we found that bacteria over a predominantly-green well but near predominantly-red wells had drastically decreased gene expression compared to predominantly-green wells that weren't near red light. After repeating trials, we decided this was due to light leaking from one well to another. An LED spreads light in all directions, unlike a laser which focuses only in a single area. This leak was causing the green light sensors to switch conformation and repress gene output, which meant that our tuning could lead to imprecise results. In order to compensate for this, we added a feature to the app, and made use of a rubber gasket. Hoping to keep our tools all electronic, we added a well-size modulator to the app in order to increase the gap spaces between wells and hopefully decrease the amount of light that moved from its generative well to another well. The rubber gasket was our physical fix, and simply involved placing a 96-well rubber cover on top of the app so that only light moving upwards would pass though.
</P><P>In order to make more precise comparisons between data, we used ImageJ to quantify lacZ expression. In ImageJ, the images were converted to grayscale, and the average black pixel density was calculated for each plate or well. The quantified data gives a more careful look at our results, and adds details to results that cannot be told apart by the naked eye.
</P><H1 id="Results"><SPAN class="mw-headline" id="Results_Summary_.5BTop.5D">Results Summary <A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Plate Experiment: Columns 1-5: constant exposure to green light. Columns 6-7: flashing red light. Columns 8-12: constant exposure to green light. Rows are replicates.</DIV></DIV></DIV>When originally characterizing the light sensor, we found that bacteria left in darkness turned the controlled gene on equal to or more than the green light-illuminated plate. This is possibly because P<SUB>cpcG2</SUB> is leaky, or possibly because the natural conformation of the green light sensor is its upregulation form. Either way, our tests demonstrated that only the red light repressor was truly functional. When dealing with light leakage, we found that even decreasing the size of the light source didn't help as much as simply covering the light with the rubber stop.<DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Image analysis of the plate experiment. The y axis is well darkness and the x axis is distance from the center of the plate. 0 is columns 6 and 7, 1 is 5 and 8, and so on. Wells closest to the center (red light, reduces lacZ expression) are lighter than wells that are farther away, indicating weaker gene expression.</DIV></DIV></DIV><P>As a proof of concept of both the light sensor and E. colight, we created a gradient of gene expression by making use of the light leakage. In two rows of a 96-well plate, we programmed all the well lights to display green constantly except for two lights in the middle, which periodically flashed red. The red light-induced downregulation of gene expression was demonstrated as the least amount of visible pigment was in the center of the gradient, and the greatest amount was at the ends. The well images were also put into ImageJ, which measured the average pixel value of each culture, as is shown in the graph on the right. The large uncertainty in the data is caused by uneven bacterial spreads in the agar, which can be solved by using soft agar.
</P><H1 id="Future"><SPAN class="mw-headline" id="Future_Directions.5BTop.5D">Future Directions<A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><P>We believe that light inducible gene expression represents a field that could be of great interest to synthetic biologists, and the iGEM community.  The work we carried out this summer has laid a foundation on which we hope future teams can build, but there is much more than can be done.  To that end, we are currently trying to standardize more the light sensors, and submit them to the registry so that they are readily available for all interested teams in the future. The light sensor used in this year's research appears to have a rather leaky promoter in our hands, leading to more transcription than we would normally expect in dark conditions. We think that the E.colight app developed over the summer could be used to address this problem.  In 96 well plate experiments we noticed what appeared to be leakiness between wells.  In order to carry out experiments using liquid cultures in the future, we may need to modify the assay plates.  We are currently exploring a variety of options to accomplish this. For example, a suggestion was made to use a rubber adaptor with appropriately placed holes that would fit under a 96 well plate.  Currently, the blue light sensor has not been characterized, and the only confirmation of a working sensor available right now are the sequencing files. We hope to continue to work with this sensor, testing and characterizing the sensor and seeking to fix any problems with the light sensor if problems are to be found. We hope to accomplish this kind of testing within the next couple of weeks. We would also like to work with the light sensors that we currently have in order to increase control of genetic pathways. The light sensors can also be attached to different genes in order to control production of certain substrates. Finally, Taking current light sensors available and applying them to e.coli is another aspiration of the Washington iGEM team.
</P><P>
The main idea of creating a light activated system would be to control gene transcription, either producing a wanted substrate or breaking substrates down. One of the ways we would like to work with our light systems in the future is to optimize it to control metabolism or production of needed substrates in e.coli. Doing that will allow us to optimally control the growth of our bacteria, only allowing it to grow under a certain wavelength of light and metabolism being repressed in the absent of light and slowed down without the right wavelength of light illuminating the bacteria. That way, if someone becomes contaminated with the bacteria and goes out into the environment, the bacteria will ideally not be able to thrive there. Therefore, by preventing bacteria growth through optimized gene control we can prevent the spread of bacteria therefore protecting the environment from potential contamination with mutant strains.
</P><P>
The app is readily available and can be downloaded onto any android OS system through the Google play store. Currently we have access to the tablet system with the app on it so we will want to use this app to further test our bacteria in the future to validate for functionality and to control genetic expression. 
</P><H1 id="Parts"><SPAN class="mw-headline" id="Parts_Submitted_.5BTop.5D">Parts Submitted <A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K892800" class="external text" rel="nofollow">BBa_K892800 <B>LovTAP with reporter gene</B></A></P><P>LovTAP system, with inverter and sfGFP output.
</P><H1 id="Sources"><SPAN class="mw-headline" id="Sources_.5BTop.5D">Sources <A href="#Background"><FONT size="3">[Top]</FONT></A></SPAN></H1><P>[1]<A href="http://upload.wikimedia.org/wikipedia/commons/9/96/AMOLED_en.svg" class="external free" rel="nofollow">http://upload.wikimedia.org/wikipedia/commons/9/96/AMOLED_en.svg</A>
[2]Levskaya, A. et al. Synthetic biology: Engineering Escherichia coli to see light. Nature 438, 441–442 (2005).

[3]Tabor, J. J., Levskaya, A. &amp; Voigt, C. a Multichromatic control of gene expression in Escherichia coli. Journal of molecular biology 405, 315–24 (2011).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Washington/Optogenetics">http://2012.igem.org/Team:Washington/Optogenetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Optogenetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Optogenetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Optogenetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Optogenetics&amp;oldid=268123" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>