Team:NTU-Singapore/Description
NTU SG iGEM 2015
NTU S'PORE
Home
Team
Attributions
Project
Parts
Notebook
Human Practices
Modeling
Jump To
Ribosomal
Binding Site
Introduction
Our Plan
Details
& Methods
Lactate
Metabolism
Introduction
Our Plan
Execution
Protocols
Description
Results
Design
Our Project
Ribosomal Binding Site
The one that glows
Read More
Lactate Metabolism
It goes "bzzzzz"
Read More
Ribosomal Binding Site
Introduction
After transcription of a gene, the mRNA will interact with the Ribosome to produce the protein coded in the mRNA in a process called translation. Like how a promoter initiates transcription by allowing RNA polymerase to bind, the ribosome binding site(RBS) provides a platform for ribosomes to bind and positions itself so that translation initiation can occur.
This binding interaction is similar to that of the base pairing of complementary nucleotides. As ribosomes are made out of a highly folded single strand rRNA and small ribosomal proteins, the 3’ end of the 16S rRNA in the 30S subunit has a short nucleotide sequence that complements the mRNA at the RBS. The consensus sequence of the RBS is known as the Shine-Dalgarno sequence, AGGAGG.
The RBS has two purposes in translation initiation, the first being the facilitation of ribosomal binding, the second is to position the ATG start codon of the protein at the peptidyl site of the ribosome. Hence, the distance between the RBS and the start codon plays a pivotal role in translation initiation. However, it’s effect is not explored in our project.
This summer we investigate the effect of substitution mutations on the strength of RBS to initiate translation. As more base pairing interactions would mean a stronger binding affinity to the mRNA, the right substitutions may bring about stronger binding to the RBS and hence, a higher rate of translational initiation.
Our Plan
As Shewanella oneidensis has been an important model organism for the making of the MFC, we would like to characterise a library of RBS mutants in this organism. For this summer, we will mutate a popular RBS, BBa_B0034 from the iGEM registry by making all three possible single base pair substitution for each of the nucleotide in the 12 base pair sequence. For example, switching the first base pair from A to C, G and T. Using eGFP, BBa_E0040 as our reporter, we will characterise the how strong can this RBS initiate translation in Shewanella oneidensis MR1 under a constitutive promoter for this strain, pLac BBa_R011. The final construct for our RBS charecterisation is pLac-RBSM-GFP-TT ligated into pHG101 vector.
Mutants of RBS BBa_B0034, the original sequence is shown at the top of each column.
We will then characterise these RBS mutants by measuring the GFP fluorescence intensity by using this construct conjugated into Shewanella oneidensis MR1 in the pHG101 vector.
Details and Methods
We first constructed the measurement construct in a pSB1AK3 plasmid and then transferred it into the pHG101 vector. This extra step of changing the plasmid backbone is to allow our reporter to be conjugated into our Shewanella via the pHG101 vector.
Then 36 different primers (3 X 12 bp) containing the RBS each with only one different nucleotide are designed to carry out PCR. The kit we used allows the whole plasmid to be replicated and the product can be directly transformed into DH5a. The plasmid is then extracted after overnight cuture and sent for sequencing to confirm sucessfull mutagenesis.
The plasmids are then transformed into an E. Coli strain WM3064 which is an diaminopimelic acid(DAP) auxotroph. This strain will be used to conjugate our reporter into Shewanella as plating the mixture of two organism in selective media would leave only sucessfull Shewanella conjugates.
After that we would measure the strength of the RBS in our chassis, by measuring the fluorescence intensity for each RBS mutant over 8 hours untill the OD600 reaches roughly 1. To ensure that all of the bacterias are metabolically active, we first culture the bacteria until mid-log phase around OD600= 0.5 and then diluting it to OD600 0.05 before starting our 8 hours long measurement.
Lactate Metabolism
Introduction
Under anaerobic conditions, S. Oneidensis MR1 strain is able to utilise lactate as a energy source via the oxidation of lactate to pyruvate. Two crucial enzymes for lactate metabolism is the Lactate Permease(LDP) and the Lactate Dehydrogenase(LDH). LDP will be the first player which transports lactate into the cell. Then LDH will catalyse the conversion of lactate to pyruvate using NAD+ as an oxidant. With NADH carrying the electrons to the plasma membrane, the electrons will be shuttled out to the external environment via the MTR pathway which involves membrane proteins in the outer and plasma membrane.
The plan
At the brainstorming stage of this project, Zhang Lei mentioned that she did some screening of LDH and LDP before. More specifically, her experiments characterized the performance of Shewanella oneidensis MR-1 over-expressing LDH and LDP originated from different bacteria such as Bacillus subtilis and Psedomonas aeruginosa. Characterized in both M1 and M9 medium, the results indicated that enzymes from Psedomonas aeruginosa work well in the chassis of MR-1, giving a high level of electrical output.
Considering LDP and LDH are both key players for the lactate metabolism of MR-1, we will mutate the enzyme which would facilitate lactate oxidation to be more efficiently. Hence we can obtain a higher electrical output when these two molecular machines work better. The strategy that we would employ is to randomly make base pair substitutions to the LDH gene so that missense mutations would occur.
We will carry out error-prone PCR with low fidelity DNA polymerases so that lots of mutants can be generated in a single reaction. The PCR fragments are then ligated to the back of pLac BBa_R0011. We then transform this intermediate construct into DH5a to separate all the mutant fragments.
After that we will complete the intermediate construct into an operon consisting of the mutant LDH and the wild type LDP, using BBa_B0015 as our transcription terminator and BBa_R0034 as our RBS. After transferring our operon into the pHG101 backbone, similar methods will be employed to conjugate the plasmid into Shewanella. Finally, we will measure the growth curves of the bacteria for carrying each mutant LDH under anaerobic conditions.
Execution
We used GeneMorph error-prone PCR Kit from Agilent Technologies. As we are able to control the frequency of mutations on the PCR products, we carried out a medium mutation rate PCR on LDH. The parameters manipulated are the amount of initial DNA template for the PCR. ~400ng of DNA template is used for the medium mutation rate PCR reaction.
The primers are designed to contain the ATG start codon and TAA stop codons so that the start and stop codons would not be changed. EcoR1, Xba1 and RBS(BBa_B0034)are designed into the forward primer while Spe1 and Pst1 cut sites are in the reverse primers. With these two primers, the PCR fragments can be digested with Xba1 and Pst1 and ligated in front of pLac promoter in a pSB1AK2 vector digested with Spe1 and Pst1. The ligation products are then transformed into DH5a and plated. 8 colonies are picked for the first bacth of mutant and sent for sequencing. Here are the results.
For the first batch, a simpler expression device which only express our mutant LDH is used to measure the growth curve. The anaerobic growth of Shewanella carrying the different LDH mutants were then measured in M1 media for 36 hours. Another second batch of 20 LDH mutants were picked and then ligated infront of a wild type LDP.
Again, the growth of the bacterias carrying the LDHM/LDPWT operon is then measure under the same conditions. However, the device used for measurement of the second batch was more automated as manual sampling of the first batch is too labour intensive. Please take a look at our protocols.
