Team:NTU-Singapore/Dairy
NTU SG iGEM 2015
NTU S'PORE
Home
Team
Attributions
Project
Parts
Notebook
Human Practices
Modeling
Calendar
May-June
Avengers Assemble
JUMP
July
Do This Do That
JUMP
August
Execution
JUMP
September
Time's Tickin'
JUMP
May
Overview
Wake me up when May starts. Most of our team members are still busy coping with final exams and the last few weeks of internship. However, we had short meetups at night to plan for the summer. Specifically, our activities in the month focuse on
Interviewing and recruiting team members
Reading of papers on synthetic biology and past iGEM projects
Brainstorming and exchanging ideas
June
Week 2
The whole iGEM 2015 NTU team finally met and have introduced ourselves to each other. We proceed immediately to our project outline and decideded on how our works are going to be done. A wiki team has been set up, Kean Hean and Hao Ting will be responsible on designing our wiki's layout. On the other hand, we are introduced to our mentor, Zhang Lei, who will be supervising and mentoring us during out wet lab experiments.
Wet Lab:
Zhang Lei guided us on the process of carrying out error prone PCR on a lactate dehydrogenase and a lactate permease gene to see whether any mutant protein with higher efficiency than the wild type protein can be produced when the products are expressed. This served as our training for general techniques required for carrying out our experiments like PCR, digestions, ligations and transformations.
Dry Lab:
Kean Hean and Hao Ting have decided and came out with a template on how our contents will be presented on our wiki. Moreover, they also have drafted few logos for our team.
Week 3
Wet Lab:
We will be working on two projects currently:
1. Carrying out medium and low level random mutagensesis of the LDH and LDP gene.
2. Make single base pair substitutions on the RBS part BBa_R0043 and charecterise it in our chasis, Shewanella oneidensis MR1.
For this week we will start with carrying our error prone PCR assays on the LDH and LDP gene. We carried out low and medium mutation frequency for LDH and LDP. Have a look at the protocols of our mutagenesis assay in our Experiments and Protocols page. Error prone PCR for the low level mutation frequency for LDH is unsucessfull while the rest shows positive results after gel electrophoresis.
Week 4
Wet Lab:
As the error-prone PCR products only contains the RBS and the mutant gene, we assemble the gene into an expression device. First, we ligated our PCR products in front of pLac and transformed it into DH5a. This step allows us to separate the PCR products as every fragment carries different mutations. For each mutation rate, only 8 colonies are picked for overnight culture.We continued on our LDH and LDP projects. We digested and ligated the mutants to a double terminator. It was very labourous as there are 24 samples in total. We then picked the colonies for overnight cuture. Plasmid extraction is carried out the next day and sent for sequencing. We listed down the mutations that have occured to check whether a missense mutation or a frameshift mutation have occured. Those that has frameshift occuring nearing the N-teminal of the gene is discared.
Moreover, we also constructed a reporter device for our RBS mutagenesis charecterisation. This will be used as the template for site directed mutaganesis on the RBS sequence. At the same time, 36 primers were orders for the the site directed mutagenesis of RBS.
July
Week 1
Wet Lab:
After sequence verification, we digested and ligated the expression construct of mutant LDP and LDH from pSB1A2 to pHG101. The ligation product is transformed into the E. Coli strain WM3064, reason for this step can be found here . We picked one colony for each mutant for overnight culture. After that we conjugated our expression construct of mutant LDP and LDH in pHG101 into S. Oneidensis.
On the other hand, site directed mutagenesis of the RBS is carried out upon arrival of our 36 primers. The PCR products are then directly transformed into DH5a. 29 out of 36 grew colonies(some has only one colony). For those that didn't grow, transformations was carried out agian.
Only one colony was picked from each of the 36 plates and inoculated in LB for overnight culture. Picture on the left shows colonies carrying the reporter with the 5th base pair of RBS substituted from A to T. It looked like the night sky with beautifull stars. Â  :)
Week 2
Wet Lab:
We picked colonies of S. Oneidensis for overnight culture and minipred the plasmid with our LDH and LDP mutants the next day. We sent the for sequencing and found that some acquired new mutations during conjugation.
We minipreped the all RBS mutant reporters and sent them for sequence verification. 7 out of 36 RBS mutants shows negative results while the rest have correct mutations i.e. the desired substitution has occured. We picked another colony for those that are wrong and sent again for sequencing. For those that are correct, we transformed them into WM3064 and picked colonies for overnight culture the next day. Then we conugated the RBS mutant reporter into S. Oneidensis.
In summary, we sucessfully created 35 out of 36 RBS single base pair mutants.  Substitution of the 6th base pair from G to T was carried out three times but was unsucessfull.
Week 3
We measured the growth curve of 6 LDH and 8 LDP mutants over a 36 hour period and took measurements at 2 hour intervals. Welcome to have a look at our method. We also picked all remaining colonies from our plates, which will be our second batch of mutants. However, for our second batch, we will ligate mutant LDH with wild type LDP and vice versa into an operon with pLac as a promoter. Hence, the resulting construct looks like this
We got all the the 7 mutants right for the second colony picked. Then we started to carry our characterisation of the RBS mutants which have been transferred into Shewanella.
Week 4
We started out construction of the LDPWT-TT by ligating the wild type LDP gene into the TT vector. and the same is done for the LDH wild type gene. We continued on to miniprep all the LDH mutants which is 14 in total for our second batch.
After conjugation, most of the plated mixtures of WM3064 and MR1 gave too much colonies and picking single colony out was very hard and was met with failures. Hence, we redid conjugation for plates that shows smears rather then single colonies. Moreover, we were also working out a protocol to measure the GFP intensity with the Tecan Plate Reader as our first attempt did not provide any conclusive results. The intensities were too high, exceeding detection levels of the reader even when our cultures were diluted. Hence, we adjusted the parameters of the photometer and tried different sampling volumes so that we can measure the fluorescence intensities even when the culture becomes saturated
August
Week 1
We sent our  LDPWT-TT and LDHWT-TT construct for sequencing and found that they were wrong. Hence, our progress on the LDH and LDP characterisation is stalled untill we got the constructs. We redo ligation for this two constructs.
We got a protocol that enables us to measure GFP intensities over an eight hour period and still remains within the detection limit of the plate reader. Do have a look are our protocols for more detials. Hence, we started right away with 4 mutants to test if our protocol worked. 10AG, 10AT, 8AC and 8AT were the first batch of RBS mutants tested. The process takes a full day and was very tiring. The conjugation of the reporters into Shewanella is still on going. We will start on another batch next week.
Week 2
Our construct of LDPWT-TT was correct for this time but LDHWT-TT is still wrong. As we do not have our LDPWT-TT construct we start to ligate out LDH mutants with the wild type LDP into one operon. We ligated 20 mutants in total, 6 from the first batch nd 14 for the second batch. Still, we carried on with the LDHWT-TT construct.
For the RBS, we characterised RBS mutants of base pair 1, 2 and 3 on Monday and and mutants of base pair4, 5 and 6 on Thursday. As we are still lacking the mutant with G switched to T on the sixth base pair, we tried mutagenesis again.
Week 3
As our ligation product was very long, >3kb, we had to use 2 sequencing primers to confirm if our ligations were sucessfull. Fortunately, our ligations were all sucessfull. We have conformation that an RBS is between the TAA stop codon of the LDH mutant and the ATG start codon of wild type LDP. Hence, we minipreped the constructs and transformed them into WM3064 for conjugation into Shewanella.
We decided to complete our RBS characterisation by this weekend. Hence, we characterised mutants of 7, 8, 9, 10, 11, 12 on the same day. On the other hand, we finally succeeded in having our desired mutant, 6GT. Hence, we started the process of conjugating this mutant RBS with the GFP reporter into Shewanella. Some results of our mutants were mistakenly left behind, 5AG and there are some(7GT and 7GC) that are showing fluorescence intensities which are too low(we are speculating our results). Hence, we decided to make another round of measurement along with 6GT.
Week 4
We transformed and plated our constructs into WM3064 but the plates dried up on the second day as the agar was too thin. Hence, we redid transformation into WM3064. As time is running out, we did not carry on to make the operon of LDPM/LDHWT.
The final batch of RBS mutant characterisation was done this week. Moreover, we started moving our constructs into pSB1C3 for shipment.
September
Week 1
After obtaining the WM3064 strains with our LDHM/LDPWT construct, started to conjugate them into Shewanella and picked for colonies on the second day.
Conrtucts were also being moved in to pSB1C3 for shipment.
Week 2
We set up the MFCs for our mutants as well as made growth curve measurements this week.
Week 3
Shipping of our constructs.
