<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>AMethods</TITLE></P><DIV class="container-fluid"><DIV id="top" class="row"><UL class="navy"><LI id="drop1" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Description">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Description" class="dropped">Description</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Results" class="dropped">Results</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Parts" class="dropped">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Composite_Part" class="dropped">Composites</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Part_Improvement" class="dropped">Gold Part</A></LI><LI><A href="https://2017.igem.org/Team:UCL/InterLab" class="dropped">Interlab</A></LI></UL><LI id="drop" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Human_Practices" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Engagement" class="dropped">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:UCL/HP/Gold_Integrated" class="dropped">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:UCL/HP/Silver" class="dropped">HP Silver</A></LI><LI id="drop5" class="navy-link"><A href="#">MODELLING</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model1" class="dropped">GOLIT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model2" class="dropped">Optoflux</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model4" class="dropped">LEGIT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model3" class="dropped">MOM</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Methods" class="dropped">Methods</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Glossary" class="dropped">Glossary</A></LI><LI id="drop3" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Entrepreneurship">ENTREPRENEURSHIP</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Entrepreneurship" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Enterpreneurship/Product" class="dropped">Product</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Enterpreneurship/Business" class="dropped">Business Model</A></LI><LI id="drop2" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Medals">OUR MEDALS</A></LI><LI id="drop6" class="navy-link"><A href="#about">PEOPLE</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Team" class="dropped">Team</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Collaborations" class="dropped">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Attributions" class="dropped">Attributions</A></LI><LI id="drop4" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Notebook">LAB BOOK</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Safety" class="dropped">Safety</A></LI></DIV></DIV><DIV class="container-fluid"><DIV class="row"><NAV id="side-index" class="col-sm-2"><UL class="nav nav-stacked" data-spy="affix" data-offset-top="205"><LI id="step1" class="navy-link side-link"><A href="#section1">Overview</A></LI><LI id="step2" class="navy-link side-link"><A href="#section2">Sensitivity Analysis</A></LI><LI id="step3" class="navy-link side-link"><A href="#section3">Parameter Sampling</A></LI></UL></NAV><DIV style="float: right !important" class="col-sm-10"><DIV id="section1" style="margin-top: 20px; border:none; margin-left:0!important; padding-left:0!important" class="text-section"><H1 class="title" style="font-size:50px!important; color:#E55190"> Cracking the Model</H1><H1 class="subtitle">Methods of parameter optimization</H1><P class="text-content-new"> Mathematical Modelling describes complex biological systems with mathematical
                    equations and concepts. Setting up the initial conditions in the model is not that straight forward.
                    If no experimental data is available, reasonable assumptions based on literature, logic and
                    quantitative analysis need to be made. We believe that the initial assumptions made are crucial
                    to crack the model. Once established, the parameters need to constantly be optimized with experimental
                    data and/or literature data in order to fit reality to its maximum. Parameter optimization adds all
                    the weight to a model's credibility, this is why we decided to approach this in with 2 methods:
					Sensitivity Analysis and Parameter Sampling.</P></DIV><DIV id="section2" class="row" style="border:none"><DIV class="text-section" style="border:none"><H1 class="title">Sensitivity Analysis</H1><P class="text-content-new">Sensitivity Analysis is a method which is conventionally used for parameter optimization.
						A range of independent variables are tested in this technique and their impact on the dependent variable is evaluated. The independent variables which have the greatest influence on the dependant variable are then identified. 
						In our case, we  want to optimize the impact the values of specific independent variables have on the rate limiting step of our cellular mechanisms.
						So, we are evaluating a range of values for each independent variable and analysing which ones optimize the process. We applied this method to
						the LEGIT and the MOM models.</P></DIV><DIV class="text-section" style="border:none"><H1 class="subtitle">Method</H1><OL class="text-content-new" style="width: 100%"><LI style="margin-left: 40px">Identify the dependent variable to be optimized (the rate limiting step in our process)</LI><LI style="margin-left: 40px">Identify the independent variables that have an impact on this step</LI><LI style="margin-left: 40px">From literature search/ experimental data, obtain a range of values to be tested for each independent variable</LI><LI style="margin-left: 40px">Run the model for the range of values tested and identify the optimal condition for the rate limiting step</LI><LI style="margin-left: 40px">Re-run the initial model to evaluate the impact of the optimized parameters on the overall process, i.e. to determine if the optimized step is prevailed the rate limiting step in the overall cellular mechanism</LI><LI style="margin-left: 40px">Determine the areas in which research could be performed to further optimize the parameters (Wet Lab contribution)</LI></OL></DIV><DIV id="show1" class="text-section" style="border:none"><H1 class="title" style="color:#E55190; font-size: 20px; padding-left:25px!important">Sensitivity Analysis method applied to LEGIT model</H1></DIV><DIV id="hidden-sanalysis"><P class="text-content-new" style="margin-left: 20px; padding:5px">
						Given that transport of intimin to the surface of the E.coli cells is the rate
						limiting step in the overall process, we are rewriting this ODE in order to simplify
						the Sensitivity Analysis on this step. </P></DIV></DIV><DIV id="section3" class="row" style="border:none"><DIV class="text-section" style="border:none"><H1 class="title">Parameter Sampling</H1><P class="text-content-new">Parameter Sampling is another technique used to tackle the issue of uncertain and incomplete knowledge of parameter values.
                    It is a method of parameter optimization, which relies on the random sampling of parameter values within a specified range.
                    A probability density function (pdf) determines the distribution of the continuous random variables evaluated in the function.
                    Parameter Sampling considers biological noise, making it a more realistic representation of the behaviour of a biological system.</P><H1 class="subtitle">Method</H1><OL class="text-content-new" style="width: 100%"><LI style="margin-left: 40px">Identify the dependent variable to be optimized (the rate limiting step in our process)</LI><LI style="margin-left: 40px">Identify the independent variables that have an impact on the step we are trying to optimize</LI><LI style="margin-left: 40px">Collect raw data from literature</LI><LI style="margin-left: 40px">Process raw data by assigning weights to the values depending on the similarity to the to-be-optimized parameter</LI><LI style="margin-left: 40px">Calculate the weighted mean for all the optimized parameters</LI><LI style="margin-left: 40px">Generate probability density functions (pdf);input the sample size, standard deviation and weighted mean</LI></OL><H1 class="subtitle">LEGIT model</H1><P class="text-content-new" style="padding-bottom:0!important">Assessing the impact of randomly generated discrete variables on the rate of
                    transport of intimin. For this the values of K<SUB>m</SUB> and V<SUB>max</SUB> are evaluated. </P></DIV><DIV class="text-section col-sm-10" style="border:none; padding-top:0%!important"><P class="text-content-new" style="padding-top:0%!important">Conditions for intimin</P><UL class="text-content-new"><LI style="margin-left: 40px">Size: 94 kDa </LI><LI style="margin-left: 40px">Optimal pH conditions: 7 </LI><LI style="margin-left: 40px">Native organism: E.coli </LI></UL><UL class="text-content-new"><LI style="list-style-type: none">Weighted mean of K<SUB>m</SUB>: 0.0147 µM</LI><LI style="list-style-type: none">Weighted mean of V<SUB>max</SUB>: 300000 s<SUP>-1</SUP>(1 s.f.) </LI></UL></DIV></DIV><DIV style="margin-top: 20px; border:none" class="text-section"><P class="text-hyperlink"><A href="https://2017.igem.org/Template:Team:UCL/Paramter">Click here to check out the data collected to optimize the value of K<SUB>m</SUB></A></P><P class="text-hyperlink"><A href="https://2017.igem.org/Template:Team:UCL/Paramter">Click here to check out the data collected to optimize the value of V<SUB>max</SUB></A></P></DIV><DIV class="text-section" col-sm-10=""><H1 class="subtitle">MOM model</H1><P class="text-content-new">Assessing the impact of randomly generated discrete variables on the rate of translation of RFP. For this the rate of nuclear export of the transcript and the rate of RFP degradation are evaluated.</P><DIV class="text-section" style="border:none; padding-top:0%!important"><P class="text-content-new" style="padding-top:0%!important">Conditions for RFP</P><UL class="text-content-new"><LI style="margin-left: 40px">Size: 27.9 kDa </LI><LI style="margin-left: 40px">Location of protein: Cytoplasm</LI><LI style="margin-left: 40px">Native organism: CHO cells</LI></UL><UL class="text-content-new"><LI style="list-style-type: none">Weighted mean of rate of nuclear export: 1.43 x 10<SUP>-4</SUP> s<SUP>-1</SUP></LI><LI style="list-style-type: none">Weighted mean of rate of degradation: 8.96 x 10<SUP>-1</SUP> s<SUP>-1</SUP></LI></UL></DIV></DIV><DIV style="margin-top: 20px; border:none" class="text-section"><P class="text-hyperlink"><A href="https://2017.igem.org/Template:Team:UCL/Paramter">Click here to check out the data collected to optimize the  rate of nuclear export</A></P><P class="text-hyperlink"><A href="https://2017.igem.org/Template:Team:UCL/Paramter">Click here to check out the data collected to optimize the rate of degradation</A></P></DIV><DIV class="row"><DIV class="text-section" style="border:none"><H1 class="title">Selecting our Programming Languages </H1><P class="text-content-new"> This summer we decided to run our models on two different programming languages:
                            MATLAB and Python. Both are high level languages that allowed us to create robust simulations.
                            Upon starting iGEM, everyone on the modelling team wanted to learn a new programming language.
                            Python is an open source software and it is relatively easy to find resources online to help
                            troubleshoot our codes. We also decided to use MATLAB for our models, as we were already familiar
                            with this language from our engineering degrees. </P></DIV></DIV><DIV class="row"><DIV class="text-section" style="border:none"><H1 class="title">Bibliography</H1><P class="subtitle" style="max-width: 85%">
										Thereza, C., Marina, I., Luciana.Cambricoli, d. and Olivia, C. (2004). Expression of green fluorescent protein (GFPuv) in Escherichia coli DH5-a, under different growth conditions. African Journal of Biotechnology, 3(1), pp.105-111.
										
										Wlab.ethz.ch. (2017). Cell Surface Protein Atlas. [online] Available at: http://wlab.ethz.ch/cspa/#abstract [Accessed 4 Sep. 2017].
										
										Sciencedirect.com. (2017). Membrane integration of E. coli model membrane proteins - ScienceDirect. [online] Available at: http://www.sciencedirect.com/science/article/pii/S0167488904000813 [Accessed 4 Sep. 2017].
										
										Arun KH e. Green fluorescent proteins in receptor: an emerging tool for drug discovery. – PubMed –NCBI [Internet]. Ncbi.nlm.gov.2017 [cited 21 August 2017]. Available from: https://www.ncbi.nlm.nih.gov/pubmed/15596111
										
										[internet] UVGermicidalTechnicalData. [cited 10 September 2017] http://www.pveducation.org/pvcdrom/effect-of-light-intensity (accessed 10 September 2017).
										
										Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E. and Loo Poh, C.  Blue light-mediated transcriptional activation and repression of gene expression in bacteria. NCBI. 2016;44(14):6994–7005.
										
										Philips, Ron. &quot;Cell Biology By The Numbers.&quot; Book.bionumbers.org. N.p., 2017. Web. 10 Sept. 2017.
										
										Kiparissides, Alexandros et al. &quot;Modelling The Delta1/Notch1 Pathway: In Search Of The Mediator(S) Of Neural Stem Cell Differentiation.&quot; PLoS ONE 6.2 (2011): e14668. Web.
										
										Carbonell-Ballestero M, Duran-Nebreda S, Montanez R, Sole R, Macia J, Rodriguez-Caso C. A bottom-up characterization of transfer functions for synthetic biology designs: lessons from enzymology. 2017.
										
										Km &amp; Vmax [Internet]. Mofetsrv.mofet.macam98.ac.il. 2017 [cited 21 September 2017]. Available from: http://mofetsrv.mofet.macam98.ac.il/~rafid/BIOLOGY/biochem/infopages/aminoacids/www-biol.paisley.ac.uk/courses/stfunmac/glossary/Km.html
										
										Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
										
										[Internet]. Karger.com. 2017 [cited 28 September 2017]. Available from: https://www.karger.com/Article/Pdf/131439
										
										HeLa cell volume - Human Homo sapiens - BNID 103725 [Internet]. Bionumbers.hms.harvard.edu. 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=103725&amp;ver=3
										
										Baumann N, Vidugiriene J, Machamer C, Menon A. Cell Surface Display and Intracellular Trafficking of Free Glycosylphosphatidylinositols in Mammalian Cells. 2017.
										
										Cortes L, Vainauskas S, Dai N, McClung C, Shah M, Benner J et al. Proteomic identification of mammalian cell surface derived glycosylphosphatidylinositol-anchored proteins through selective glycan enrichment. 2017.
										
										Key Numbers for Cell Biologists [Internet]. Bionumbers.org; 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/Includes/KeyNumbersLinks.pdf
						
										Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>