<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCL_Model3 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCL/Model3</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Description</TITLE></P><DIV style="margin:0 !important; padding:0 !important" class="container-fluid"><DIV id="top" class="row"><UL class="navy"><LI id="drop1" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Description">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Description" class="dropped">Description</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Results" class="dropped">Results</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Parts" class="dropped">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Composite_Part" class="dropped">Composites</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Part_Improvement" class="dropped">Gold Part</A></LI><LI><A href="https://2017.igem.org/Team:UCL/InterLab" class="dropped">Interlab</A></LI></UL><LI id="drop" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Human_Practices" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Engagement" class="dropped">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:UCL/HP/Gold_Integrated" class="dropped">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:UCL/HP/Silver" class="dropped">HP Silver</A></LI><LI id="drop5" class="navy-link"><A href="#">MODELLING</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model1" class="dropped">GOLIT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model2" class="dropped">Optoflux</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model4" class="dropped">LEGIT</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Model3" class="dropped">MOM</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Methods" class="dropped">Methods</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Glossary" class="dropped">Glossary</A></LI><LI id="drop3" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Entrepreneurship">ENTREPRENEURSHIP</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Entrepreneurship" class="dropped">Overview</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Applied_Design" class="dropped">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Enterpreneurship/Business" class="dropped">Business Model</A></LI><LI id="drop2" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Medals">OUR MEDALS</A></LI><LI id="drop6" class="navy-link"><A href="#about">PEOPLE</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Team" class="dropped">Team</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Collaborations" class="dropped">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Attributions" class="dropped">Attributions</A></LI><LI id="drop4" class="navy-link"><A href="https://2017.igem.org/Team:UCL/Notebook">LAB BOOK</A></LI><LI><A href="https://2017.igem.org/Team:UCL/Safety" class="dropped">Safety</A></LI></DIV></DIV><DIV class="container-fluid"><DIV class="row"><NAV id="side-index" class="col-sm-2"><UL class="nav nav-stacked" data-spy="affix" data-offset-top="205"><LI id="step1" class="navy-link side-link"><A href="#section1">Aims</A></LI><LI id="step2" class="navy-link side-link"><A href="#section2">Science behind MOM</A></LI><LI id="step3" class="navy-link side-link"><A href="#section3">Results</A></LI><LI id="step4" class="navy-link side-link"><A href="#section4">Cost Analysis</A></LI><LI id="step5" class="navy-link side-link"><A href="#section5">Alternative applications</A></LI><LI id="step6" class="navy-link side-link"><A href="#section6">Assumptions</A></LI></UL></NAV><DIV style="float: right !important" class="col-sm-10"><H1 class="title" style="font-size:50px!important; color:#E55190">MOM - From light to organs</H1><H2 class="subtitle" style="margin-left: 0!important; margin-top: 20px !important; margin-bottom:5px!important"> Mammalian Optogenetic Model - A mathematical model that optimizes the speed and precision with which HEK293T cells activate gene expression via light-induction</H2><H1 class="title" id="section1" style="padding-top: 30px!important;">Model Aims</H1><P class="text-content-new" style="padding: 0 12% 0 10%!important">1. Determine the rate limiting cellular mechanism</P><P class="text-content-new" style="padding: 0 12% 0 10%!important">2. Optimize the rate limiting parameters by using a conventional (Sensitivity Analysis) and an innovative (Parameter Sampling) approach </P><P class="text-content-new" style="padding: 0 12% 0 10%!important">3. Determine the range of optimal light intensities and light pulses our Wet Lab should test</P><P class="text-content-new" style="padding: 0 12% 0 10%!important">4. Conduct a Cost Analysis to evaluate the economic viability of our optogenetic tool</P><H1 class="title" id="section2">The Science behind MOM</H1><P class="text-content-new">  MOM models the light-activated expression of mCherry (an RFP) within genetically engineered HEK293T cells. This is a proof-of-concept model for precise control and direction for the expression of any protein of interest. The model starts with the single-illumination light-activated CRISPR-Cas9 effector (siLACE, BBa_K2332317) system embedded in the plasma membrane of our HEK293T cells. 
The siLACE system consists of a transmembrane domain, the photocleavable linker PhoCl, a dCas9 and a transcription factor (TF). The photocleavable linker anchors the dCas9-TF to the cytosolic leaflet of the plasma membrane. When we shine light onto our HEK293T cells the photocleavable linker is cleaved and the dCas9-TF is released into the cytoplasm of the cell. The dCas9 leads the TF to the promoter-region of the RFP reporter inside the cell nucleus. This mechanism could be adapted to induce the production of any target-protein after only a single-light exposure! As such, our technology and model could be used to guide cells with light to create highly specific mammalian tissues.</P><P class="text-content-new" style="padding-bottom:0!important; margin-bottom:0!important!">Breakdown of the cellular mechanisms involved in the expression of intimin:</P><TABLE style="border: 1px solid black; margin-top: 30px; margin-bottom: 15px; margin-left:110px!important; max-width: 80%!important;"><TBODY><TR><TD><P class="text-figure" style="max-width: 100%; text-align: left; padding-left:10px!important"> Legend:
                                <B>LACE<SUB>TF</SUB></B> = Photocleavable (PhoCl) linker attached to the outer membrane of the cells and the TF
                                <B>LACE</B> = Protein on cells' outer membrane attached to a part of the cleaved PhoCl linker
                                <B>TF<SUB>C</SUB></B> = Free TF in the cytoplasm of the cell
                                <B>TF<SUB>N</SUB></B> = Free TF in the nucleus of the cell
                                <B>Pr<SUB>RFP</SUB></B> = RFP Promoter
                                <B>Pr<SUB>RFP + TF<SUB>N</SUB></SUB></B> = TF in the nucleus bound to the RFP Promoter
                                <B>mRNA<SUB>N</SUB></B> = mRNA in the cell nucleus
                                <B>mRNA<SUB>C</SUB></B> = mRNA in the cell cytoplasm
                                <B>P<SUB>RFP</SUB></B> = RFP translated protein
                            </P></TD></TR></TBODY></TABLE><P class="text-content-new">In order to tackle the cellular mechanisms involved in RFP expression, we decided to divide our model into three main steps:</P><P class="text-content-new" style="padding: 0 12% 0 10%!important"> Step 1: Photocleavage and release of the dCas9-TF in the cytoplasm of the cell</P><P class="text-content-new" style="padding: 0 12% 0 10%!important">Step 2: Transcription</P><P class="text-content-new" style="padding: 0 12% 0 10%!important">Step 3: Translation</P><P class="text-figure" style="padding-top:15px; padding-bottom: 30px"> Figure 1: Demonstrates an overview of the modelling steps in our MOM model</P><P class="text-content-new" style="margin-top: 0; padding-top:0 ">In our MOM model the rate kinetics of our system are induced with photocleaving.
                This is a unique twist we incorporated in contrast to conventional mammalian cell expression systems. In addition, the attachment of a dCas9
                at the end of our TF, is another unique adaptation we incorporated into our model.
            </P><P class="text-hyperlink" style="margin-top: 0!important; margin-left:10px!important"><A href="https://github.com/LITiGEM/Simulation/tree/master/Barchitecture">Click here to check out our code on Github!</A></P><DIV class="text-section" style="border-bottom: none; border-left: none; border-right: none"><H1 class="title" style:="">Modelling Steps</H1><H1 class="subtitle " style="font-size: 22px!important">1) Photocleavage and release of the dCas9-TF in the cytoplasm of the cell</H1><P class="text-figure" style="padding-top:15px; padding-bottom: 0px!important"> Figure 2: Demonstrates an overview of photocleavage</P><P class="text-content-new" style="padding-top:0px!important; margin-top:0!important ">Location: Outer cell membrane </P><P class="text-content-new">The first step of our simulation is the photocleavage of PhoCl, which results in the release of the dCas9-TF in the cytoplasm of the cell. We introduced two degradation factors within our ODEs, in order to account for the half-life of the dCas9-TF and the
                    continual transport of the dCas9-TF from the cytoplasm to the nucleus of the cell.</P><P class="text-content-new" style="padding-bottom: 0"> The photocleavage is dependent on light intensity (L). The Hill equation is commonly used to
                    fit inducible promoter genetic circuits. We therefore decided to adapt the Hill equation in our ODE to evaluate
                    the effect different light intensities would have on the rate of photocleavage of the LACE<SUB>TF</SUB> complex. </P><H1 class="subtitle" style="padding-top: 15px; font-size: 22px!important">2) Transcription</H1><P class="text-figure" style="padding-top:15px; padding-bottom: 0px!important"> Figure 3: Demonstrates an overview of transcription</P><P class="text-content-new" style="margin-top: 0!important; padding-top:0!important "> Location: Nucleus</P><P class="text-content-new">The second step of our simulation is the production of mRNA from transcription.
                    In order for transcription to take place the dCas9-TF needs to be transported from the cytoplasm of the cell to the nucleus. The dCas9-TF is guided into the
                    nucleus of the cell to the RFP promoter with the help of dCAS9. The rate at which the dCas9-TF enters the nucleus of the cell and binds to the RFP promoter
                    affects the rate at which transcription takes place. We incorporated two degradation factors to account for the half-life of the mRNA and the transport of the mRNA from the nucleus of the cell
                    to the cytoplasm.</P><H1 class="subtitle" style="padding-top:15px; font-size: 22px!important">3) Translation</H1><P class="text-figure" style="padding-top:15px; padding-bottom: 0px!important"> Figure 4: Demonstrates an overview of translation</P><P class="text-content-new" style="margin-top: 0!important; padding-top:0!important ">Location: Cytoplasm</P><P class="text-content-new" style="padding-top:10px!important">The third step of our simulation is the production of the RFP protein from translation. The change in concentration of the RFP protein is dependent on three main factors. Firstly, the rate at which mRNA is transported from the nucleus to the cytoplasm, this is

                    represented with the letter 'β'. Secondly, the rate of translation, represented by 'k<SUB>3</SUB>', and thirdly the maximum amount of RFP protein a cell can express, represented by the letter 'η'. A degradation term was introduced to represent the degradation of
                    RFP over time because of dilution due to cell growth and the limited half-life of the RFP protein. </P><H1 class="text-hyperlink" style="padding-top:0!important; margin-left:10px!important"><A href="https://2017.igem.org/Team:UCL/Description">Not enough biology? Check out the project description page!</A></H1></DIV><DIV class="text-section" id="section3" style="border-bottom: none; border-left: none; border-right: none"><H1 class="title" style="margin-top:40px!important; padding-top:0!important; padding-bottom:10px!important">Results</H1><P class="text-content-new" style="padding-top:0!important; margin-top:0!important">From our MOM simulation we compiled our graphs, evaluated their results
                    and determined the optimal range of values our Wet Lab team should evaluate in order to optimize the rate-limiting parameters of the system.</P></DIV><H1 class="subtitle" style="font-size: 22px!important">Determining the rate limiting step in RFP expression</H1><P class="text-content-new">We used rate kinetics modelling to determine the rate limiting step in the cellular mechanisms involved in the light-activated translation of RFP.</P><P class="text-figure" style="margin-bottom: 10px!important; padding-top: 5px!important">Figure 5: Demonstrates the rate of the cellular mechanisms involved in the expression of RFP protein </P><P class="text-content-new">In Figure 5 the photocleavable expression system is induced with a 27 W/cm<SUP>2</SUP> light intensity. The concentration of the free dCas9 TF and mRNA in the nucleus appear to be much smaller than that of the RFP protein.
                Therefore, we decided to plot a second figure, in which we re-scaled the graph, in order to evaluate the rate of photocleaving and transcription. </P><P class="text-figure" style=" padding-top: 5px!important"> Figure 6: Depicts a re-scaled plot of Figure 5 in order to view the rate at which the TF is released in the cell cytoplasm</P><TABLE style="width: 60%; margin-left: 115px!important; padding-left:70px!important" class="text-content-duo-primary"><TBODY><TR><TH class="text-content" style="padding: 0">Step</TH><TH class="text-content" style="padding: 0">Time taken to plateau (hr) </TH></TR><TR><TD class="text-content" style="padding: 0">Cleavage of PhoCl Linker</TD><TH class="text-content" style="padding: 0">0.1</TH></TR><TR><TD class="text-content" style="padding-left: 25px;padding-top: 0; padding-bottom: 0">Transcription</TD><TH class="text-content" style="padding-left: 25px;padding-top: 0; padding-bottom: 0">0.1</TH></TR><TR><TD class="text-content" style="padding-left: 25px">Translation</TD><TH class="text-content" style="padding-left: 25px">3</TH></TR></TBODY></TABLE><P class="text-figure" style="margin-bottom: 0px!important; padding-top: 5px!important">Figure 7: Demonstrates the time taken for each cellular mechanisms involved in the expression of RFP protein to plateau</P><P class="text-content-new">A rate limiting step in a process is the step that takes the longest amount of time to reach its maximum value, essentially the step that takes the longest time to plateau.
                    We identified the rate of translation as the rate limiting step in this cellular mechanism.
                    The rapidity of our LIT technology is vital, therefore we decided to identify the rate limiting parameters within the ODE that describe the translation of mRNA to RFP protein.
                    We performed two parameter optimization techniques: <A class="text-hyperlink" href="https://2017.igem.org/Team:UCL/Methods"> Sensitivity Analysis </A> and <A class="text-hyperlink" href="https://2017.igem.org/Team:UCL/Methods"> Parameter Optimization </A>.
                </P><H1 class="subtitle" style="font-size: 22px!important">Optimizing Light Intensity using Sensitivity Analysis</H1><P class="text-content-new">Our MOM model is initiated with the photocleavage of a PhoCl Linker that leads to the release of the dCas9 TF in the cells cytoplasm. Therefore, we thought it would be interesting to evaluate the effect
                    different light intensities had on the concentration and rates of expression of RFP protein in the cell. The values we selected to test fell within the range of 0 W/cm<SUP>2</SUP> and
                    54 W/cm<SUP>2</SUP>, where 0 W/cm<SUP>2</SUP> represented our system in its off state and 54 W/cm<SUP>2</SUP> represented the maximum light intensity mammalian cells can withstand
                    before they started to die.</P><P class="text-figure" style="padding-top: 5px!important"> Figure 8: Evaluates the effect light intensity has on the concentration and rate of expression of RFP protein in the cell with the use of a Sensitivity Analysis </P><P class="text-content-new" style="margin-top: 0; padding-top:0">From Figure 8 it is evident that an increase in the light intensity used to photocleave PhoCl corresponds to an increase in the concentration of RFP expressed in the cells.</P><H1 class="subtitle" style="font-size: 22px!important">Optimizing the parameters in the rate-limiting step</H1><P class="text-content-new">We decided to optimize the rate-limiting parameters present in the ODE which describes the translation of RFP protein in the cell. The two rate-limiting parameters identified were: β (the rate of transport of mRNA from the nucleus to the cell cytoplasm) and d<SUB>3</SUB> (the degradation rate of the RFP protein).
                    We decided to optimize the values for each parameter by using two approaches, a Sensitivity Analysis and Parameter Sampling. The maximum and minimum values that could be assigned to each parameter were determined from literature. </P><H1 class="subtitle" style="padding-top: 15px; max-width: 80%; font-size: 22px!important">Optimizing the rate at which mRNA is transported to the cell nucleus (β) with a Sensitivity Analysis and Parameter Sampling</H1><P class="text-content-new">The rate of translation is dependent on the rate at which mRNA is transported from the nucleus to the cytoplasm of the cell. Ideally, we would want a high β value to increase the rate at which the mRNA is transported to the cytoplasm of the cell. This would increase the rate of translation.</P><P class="text-figure" style="padding-top: 5px!important"> Figure 9: Evaluates the effect β has on the rate of RFP expression in the cytoplasm of the cell with the use of a Sensitivity Analysis

                </P><P class="text-content-new"> Sensitivity Analyses determine the magnitude of the effect a particular parameter has on the output of a model. Therefore, we first decided to run a Sensitivity Analysis to determine/ validate that β has a large contribution
                    on the concentration of RFP expressed in the cells. In order to do this a range of β values, from other similar cellular mechanisms, were inputted into our model. From Figure 9 we determined the optimum β value to
                    operate our cellular mechanism at is 2.20Ε+00 1/s. We therefore, decided to conduct Parameter Sampling to ensure the value we select
                 to optimize our β to can realistically be achieved in our cellular mechanism.</P><P class="text-figure" style="padding-top: 5px!important"> Figure 10: Evaluates the effect β has on the rate of RFP expression in the cytoplasm of the cell with the use of Parameter Sampling </P><P class="text-content-new">Parameter Sampling allowed us to take a series of factors into consideration when optimizing the value of β.
                    The values of β represented in figure 10 are more realistic and significantly lower than
                    the ones obtained from the Sensitivity Analysis. This is because a series of factors were taken into consideration when
                    creating a normal distribution of the possible β values for our technology. These factors include: the size,
                    origin and operational pH of the cellular mechanism from which the data was sourced form. From figure 10 we determined the optimum
                    β value that would result in the highest concentration of RFP expressed in our cells was 1.3E-03 1/s.</P><P class="text-hyperlink" style="padding-top:0; margin-top:5px!important; margin-left:10px!important"><A href="https://2017.igem.org/Team:UCL/Methods">Interested in the steps and weighting system we used for our Parameter Sampling?</A></P><H1 class="subtitle" style="padding-top: 15px; padding-bottom: 15px; max-width: 80%; font-size: 22px!important">Optimizing the degradation rate (d<SUB>3</SUB>) of the RFP protein with the use of a Sensitivity Analysis and Parameter Sampling</H1><P class="text-content-new">The parameter d<SUB>3</SUB> represents the rate at which the concentration of RFP decreases due to protein degradation and dilution.
                    Therefore, we would want to minimize the value of d<SUB>3</SUB> so as to decrease the time it takes for the maximum amount of RFP to be expressed in the cell.</P><P class="text-figure" style="padding-top: 5px!important"> Figure 11: Evaluates the effect d<SUB>3</SUB> has on the rate of RFP expression in the cell with the use of a Sensitivity Analysis

                </P><P class="text-content-new"> In Figure 11 it is evident that from the Sensitivity Analysis d<SUB>3</SUB>  is a large contributor to the final concentration of
                    RFP that can be expressed in the cells. Therefore, we decided to progress and carry out a Parameter Analysis in order to optimize
                    our d<SUB>3</SUB> value.</P><P class="text-figure" style="padding-top: 5px!important"> Figure 12: Evaluates the effect d<SUB>3</SUB> has on the rate of RFP expression in the cell with the use of Parameter Sampling</P><P class="text-content-new">Parameter Sampling allowed us to take a series of factors into consideration when optimizing the value of d<SUB>3</SUB>.
                    From Figure 12 we determined the optimum d<SUB>3</SUB> value to maximize the concentration of RFP expressed in the
                    cells is 2.2E-05 1/s.</P><H1 class="subtitle" style="font-size: 22px!important">Determining the effect light pulsing has on the rate of RFP expression</H1><P class="text-content-new"> We determined increasing light intensity has a directly proportional increasing effect on the concentration
                    and the rate of expression of RFP in the cell. We therefore decided to evaluate the effect other light properties would have on our system. More specifically, we wanted to evaluate whether light pulsing had an effect on RFP expression. This would allow us to see whether by periodically activating and
                    deactivating our light induced system we would observe the expression of a larger concentration of RFP, at a faster rate, in the cells.</P><P class="text-content-new">We first ran a control simulation, where we introduced one long pulse of 7 hours of light, to ensure we received a curved response that would simulate the activation and deactivation of our cellular mechanisms over a defined period of time. </P><P class="text-figure" style=" padding-top: 5px!important">Figure 13: Evaluates the effect one long pulse has on the rate kinetics of the expression of RFP in the cell cytoplasm</P><P class="text-content-new"> From Figure 13 it is evident the behavior of all the species produced, after each step involved in the expression of RFP, are represented in a dumb bell shaped curve. This is justifiable, as when the light is turned off we expect the rate of product production to become smaller than the rate of product degradation. Therefore, as time passes the concentration of each product from each step is expected to decrease.
                    We then wanted to evaluate the effect pulsing, for periods of 2 hours, would have on the rate kinetics involved in the expression of RFP in the cytoplasm of the cell.</P><P class="text-figure" style="padding-top: 5px!important">Figure 14: Evaluates the effect pulsing light, in 2-hour intervals, has on the rate kinetics involved in the rate and concentration of RFP expressed in the cytoplasm of the cell

                </P><P class="text-content-new" style="padding-bottom:10px!important">From Figure 14 we discovered that if we pulse light on the cells for a period of 2 hours over a 24-hour period, we experienced a small decrease (14% decrease) in the concentration of RFP expressed in the cytoplasm of the cells.</P><DIV class="text-section" style="border:none"><H1 class="title" style="margin-top:10px!important; padding-top:10px!important">Optimized conditions </H1><P class="text-content-new"> The biggest increase in the rate of RFP expression in the cytoplasm of the cells occurs when the value of 'β’ increases and the values of ‘d<SUB>3</SUB>’ decreases. This is something we expected, as the larger the value of β the faster the speed at which mRNA is transported to the cytoplasm of the cell from the nucleus and therefore the faster the rate of translation. In addition, the lower the d<SUB>3</SUB> value, the slower the degradation of mRNA in the cell. Thus, the combination of both of these phenomena results in a faster and greater concentration of RFP expressed in the cytoplasm of the cell.</P><TABLE style="width: 60%; margin-left: 115px!important; padding-left:70px!important" class="text-content-duo-primary"><TBODY><TR><TH class="text-content" style="padding: 0">Optimized Parameter</TH><TH class="text-content" style="padding: 0">Value </TH></TR><TR><TD class="text-content" style="padding:0">β (1/s)</TD><TH class="text-content" style="padding: 0">2.2E+00</TH></TR><TR><TD class="text-content" style="padding:0">d<SUB>3</SUB> (1/s)</TD><TH class="text-content" style="padding:0">6.4E-05</TH></TR></TBODY></TABLE><P class="text-figure" style="margin-bottom: 0px!important; padding-top: 5px!important">Figure 15: Demonstrates a summary of the optimized conditions for the model</P><H1 class="subtitle" style="margin-top: 25px; font-size: 22px!important">Running the model with optimized parameters</H1><P class="text-content-new"> We ran our rate kinetics model with our newly optimized parameters and determined that with our optimized parameters a larger concentration of RFP was expressed in the cytoplasm of the cell. However, the same amount of time was taken for the RFP expression rate to plateau. Therefore, this
                        means that although the concentration of RFP expressed in the cytoplasm of the cell was optimized this step prevails as the rate-limiting step in the network of cellular mechanisms involved.</P><P class="text-figure" style="padding-top: 5px!important">Figure 16: Depicts the optimized and unoptimized rate kinetics for the translation of the RFP protein </P></DIV><DIV class="text-section" id="section4" style="border-bottom: none; border-left: none; border-right: none"><H1 class="title">Cost Analysis</H1><P class="text-content-new"> We ran a  Cost Analysis to ensure our LIT optogenetic tool was an affordable technology current labs could use to replace alternative technologies.
                        We focused on identifying the main operational costs a user would incur. We determined the most critical ones were: the light intensity they used to activate the HEK 293 cellular mechanism; and the frequency of light pulsing.
                        We created a function in Python that took costs incurred for both methods into consideration, where we attempted to identify the most cost-efficient operational conditions for our tool.</P><TABLE style="width: 60%; margin-left: 115px!important; padding-left:65px!important; padding-top:0; padding-bottom:0" class="text-content-duo-primary"><TBODY><TR><TH class="text-content" style="padding: 0">Light Intensity (W/cm<SUP>2</SUP>)</TH><TH class="text-content" style="padding: 0">Cost per hour of utility (£/hr) </TH></TR><TR><TD class="text-content" style="padding: 0">70</TD><TH class="text-content" style="padding: 0">0.014</TH></TR><TR><TD class="text-content" style="padding: 0">53</TD><TH class="text-content" style="padding: 0">0.011</TH></TR><TR><TD class="text-content" style="padding: 0">35</TD><TH class="text-content" style="padding: 0">0.007</TH></TR><TR><TD class="text-content" style="padding: 0">18</TD><TH class="text-content" style="padding: 0">0.0036</TH></TR><TR><TD class="text-content" style="padding: 0">0</TD><TH class="text-content" style="padding: 0">0</TH></TR></TBODY></TABLE><P class="text-figure" style="margin-top: 0; padding-top:0">Figure 17: Outlines the operating costs incurred for operating the optogenetic tool with a range of light intensities with no light pulsing present</P><TABLE style="width: 60%; margin-left: 115px!important; padding-left:65px!important; padding-top:0; padding-bottom:0" class="text-content-duo-primary"><TBODY><TR><TH class="text-content" style="padding: 0">Pulsing Duration (hr)</TH><TH class="text-content" style="padding:0 0 0 20px">Cost for total pulsing over 2 hour operation period (£) </TH></TR><TR><TD class="text-content" style="padding: 0">2</TD><TH class="text-content" style="padding: 0 0 0 20px">7E-03</TH></TR><TR><TD class="text-content" style="padding: 0">1.33</TD><TH class="text-content" style="padding:0 0 0 20px">4.E-03</TH></TR><TR><TD class="text-content" style="padding: 0">1</TD><TH class="text-content" style="padding:0 0 0 20px">3.5E-03</TH></TR><TR><TD class="text-content" style="padding: 0">0.5</TD><TH class="text-content" style="padding:0 0 0 20px">1.8E-03</TH></TR><TR><TD class="text-content" style="padding: 0">0.25</TD><TH class="text-content" style="padding:0 0 0 20px">8.8E-0.4</TH></TR><TR><TD class="text-content" style="padding: 0">0</TD><TH class="text-content" style="padding:0 0 0 50px">0</TH></TR></TBODY></TABLE><P class="text-figure">Figure 18: Outlines the operating costs incurred for pulsing the light source of the optogenetic tool over a range of frequencies at a light intensity of 27 W/cm<SUP>2</SUP></P><P class="text-figure" style="margin-top: 0; padding-top:0">Figure 19:  Comparison of the effects of pulsing and light intensity on the overall cost </P><P class="text-content-new"> We decided to plot both the costs incurred for different light intensities and the pulsing durations on the same graph to evaluate their synergistic effect.
                        The most optimum operating conditions for our optogenetic tool were defined as those at which the lines intersect. Beyond the point of intersection, it was evident that the operational costs
                        for the optogenetic tool were too high. Below the intersection point it was evident that the activity of the HEK 293 cell adhesion mechanism would
                        be too slow, and as one of the biggest selling points of our technology is its rapidity we decided it would not be practical to
                        operate at such a slow rate.
                        
                        Therefore, the best trade-off was identified when operating at a light intensity of 30 W/cm<SUP>2</SUP> with a 4.5 hour pulsing frequency.</P></DIV><DIV class="text-section" id="section5" style="border-bottom: none; border-left: none; border-right: none"><H6 class="title">Alternative application for our model</H6><DIV style="margin-top: 20px"><H1 class="subtitle">Pharmaceutical Industry</H1><P class="text-content-new" style="padding-top: 10%!important; margin-top:0!important">We envision our model could be adapted and used by pharmaceutical companies to optimize their
                                Downstream Processing conditions to obtain a high product yield. Focus could be placed on identifying the bottleneck
                                of the manufacturing process, or the stage at which the greatest concentration of product is lost.
                                Each differential equation could represent a batch mode taking place in one-unit operations, where the product concentration
                                changes over time. For each equation, we would include the loss of product in the form of degradation rates, as this would
                                signal a decrease in product concentration. For each of these conditions a discount factor for the product concentration
                                would need to be introduced. The product loss could be due to: the product getting stuck in crevices in the machinery;
                                product degradation due to a limited half-life; and/or product aggregation to other cellular debris. This model would help
                                companies determine the unit operation at which the largest product loss is experienced.
                            </P></DIV><DIV class="text-section" id="section6" style="border-left: none; border-right: none; border-bottom:none"><H1 class="title">MOM Assumptions</H1><UL class="text-content" style="margin-left:40px!important; max-width:80%"><LI><A class="text-hyperlink" href="glossary_o.html"> Mass action kinetics </A></LI><LI><A class="text-hyperlink" href="glossary_o.html"> Michaelis Menten </A></LI><LI>All reactions are taking place in cells that are plated in a petri dish</LI><LI>PhoCl always cleaves when exposed to violet light (400 nm)</LI><LI>LACE with the dCas9 TF attached to it is constitutively expressed in the cell</LI><LI>Every cell is activated by the same light intensity, regardless of its position in the petri dish</LI><LI>There is no light reflection or refraction from the cells</LI><LI>All cells are at their exponential growth phase</LI><LI>The cells are not going through the cell cycle when they are induced to photocleave and release the dCas9 TF in the cytoplasm of the cell</LI></UL><H1 class="title">MOM Species</H1><H1 class="title"> MOM Parameters</H1><H1 class="title">Initial Assumptions</H1></DIV><DIV class="text-section" style="border:none"><H1 class="title">Bibliography</H1><P class="subtitle" style="max-width: 80%">
                            Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
                            
                            Zitieren eine Webseite - Cite This For Me [Internet]. Karger.com. 2017 [cited 28 September 2017]. Available from: https://www.karger.com/Article/Pdf/131439
                            
                            HeLa cell volume - Human Homo sapiens - BNID 103725 [Internet]. Bionumbers.hms.harvard.edu. 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=103725&amp;ver=3
                            
                            Baumann N, Vidugiriene J, Machamer C, Menon A. Cell Surface Display and Intracellular Trafficking of Free Glycosylphosphatidylinositols in Mammalian Cells. 2017.
                            
                            Cortes L, Vainauskas S, Dai N, McClung C, Shah M, Benner J et al. Proteomic identification of mammalian cell surface derived glycosylphosphatidylinositol-anchored proteins through selective glycan enrichment. 2017.
                            
                            Key Numbers for Cell Biologists [Internet]. Bionumbers.org; 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/Includes/KeyNumbersLinks.pdf
                            
                            Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
                            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>