<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLouvain_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLouvain/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UCLouvain Team</TITLE><TITLE>iGEM UCLouvain Team</TITLE></P><DIV class="container"><DIV class="menu"><NAV class="nav"><UL class="first-level"><LI><A href="/Team:UCLouvain" class="animsition-link">Home</A></LI><LI><A href="/Team:UCLouvain/OurProject/Overview">our project</A></LI><LI><A href="/Team:UCLouvain/OurProject/Overview" class="animsition-link">overview</A></LI><LI><A href="/Team:UCLouvain/OurProject/Auxotrophs_Overview" class="animsition-link">auxotrophs</A></LI><LI><A href="/Team:UCLouvain/OurProject/ComRS_Overview" class="animsition-link">comR/S</A></LI><LI><A href="/Team:UCLouvain/Badge" class="animsition-link">badge</A></LI></UL><LI><A href="#">Results</A></LI><LI><A href="/Team:UCLouvain/OurProject/Approach1/Results" class="animsition-link">Auxotrophs</A></LI><LI><A href="/Team:UCLouvain/OurProject/Approach2/Results" class="animsition-link">Com R/S </A></LI><LI><A href="/Team:UCLouvain/HP/Silver" class="animsition-link">human practices</A></LI><LI><A href="/Team:UCLouvain/Safety" class="animsition-link">safety</A></LI><LI><A href="#">in the lab</A></LI><LI><A href="/Team:UCLouvain/InterLab" class="animsition-link">interlab</A></LI><LI><A href="/Team:UCLouvain/LN" class="animsition-link">Lab Notebook</A></LI><LI><A href="/Team:UCLouvain/Protocols" class="animsition-link">protocols</A></LI><LI><A href="/Team:UCLouvain/Basic_Part" class="animsition-link">Parts</A></LI><LI><A href="/Team:UCLouvain/Team">about us</A></LI><LI><A href="/Team:UCLouvain/Team" class="animsition-link">team</A></LI><LI><A href="/Team:UCLouvain/Attributions" class="animsition-link">attributions</A></LI><LI><A href="/Team:UCLouvain/OurUniversity" class="animsition-link">our university</A></LI><LI><A href="/Team:UCLouvain/OurEvents" class="animsition-link">our events</A></LI><LI><A href="/Team:UCLouvain/Aknowledgement" class="animsition-link">aknowledgement</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=UCLouvain">judging form</A></LI></NAV></DIV></DIV><DIV class="site-hero_2"><DIV class="page-title"><DIV class="big-title montserrat-text uppercase">In the Lab</DIV><DIV class="small-title montserrat-text uppercase"><MARINE style="margin-left:32px;font-family:'Brush Script MT';font-size:32px;text-transform:none;font-weight:bold;">Protocols</MARINE></DIV></DIV></DIV><SECTION><DIV class="container"><DIV class="row" style="margin-bottom: 40px;"><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>DNA Gel electrophoresis</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Agarose Powder</LI><LI style="color:#999999;margin-left:20px;">TAE buffer</LI><LI style="color:#999999;margin-left:20px;">Gel mould</LI><LI style="color:#999999;margin-left:20px;">Gel Tank</LI><LI style="color:#999999;margin-left:20px;">DNA ladder</LI><LI style="color:#999999;margin-left:20px;">DNA loading dye</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Prepare 0.8% agarose (w/v) solution in TAE buffer.</LI><LI style="color:#999999;margin-left:20px;">Heat until the agarose dissolve.</LI><LI style="color:#999999;margin-left:20px;">Pour into the gel mould and set a comb.</LI><LI style="color:#999999;margin-left:20px;">Wait for the gel to solidify.</LI><LI style="color:#999999;margin-left:20px;">Remove the comb and transfer into the gel tank. Fill with TAE buffer.</LI><LI style="color:#999999;margin-left:20px;">Fill the wells with DNA ladder and your samples mixed with corresponding amount of loading dye.</LI><LI style="color:#999999;margin-left:20px;">Run for 20-30 minutes at 120 V.</LI></OL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>T4 Ligation</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">PCR tubes</LI><LI style="color:#999999;margin-left:20px;">T4 DNA Ligase (NEB)</LI><LI style="color:#999999;margin-left:20px;">T4 DNA Ligase buffer 10x</LI><LI style="color:#999999;margin-left:20px;">ddH20</LI><LI style="color:#999999;margin-left:20px;">Insert DNA</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Calculate insert to vector ratio with NEB calculator. Most common is 3:1.</LI><LI style="color:#999999;margin-left:20px;">Add the vector plasmid, the insert DNA, 2 µL of buffer, 1 µL of T4 ligase and enough ddH20 to reach 20 µL.</LI><LI style="color:#999999;margin-left:20px;">Gently mix by pipeting up and down.</LI><LI style="color:#999999;margin-left:20px;">For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.</LI><LI style="color:#999999;margin-left:20px;">Heat inactivate at 65°C for 10 minutes.</LI></OL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>Selection of the sgRNA and modification of pTarget</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Introduction</P></DIV><P>
			This protocol cover the identification of a suitable N20 sequence and the modification of the pTarget plasmid. Protocol from S. Worms.
			</P><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">1) Selection of the N20 sequence</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Use Benchling's CRISPR tool, click on &quot;Design and analyse guide&quot;</LI><LI style="color:#999999;margin-left:20px;">Using the default settings (note: Doench et al.'s optimized scoring algorithm doesn't work for circular sequences such as bacterial chromosomes)</LI><LI style="color:#999999;margin-left:20px;">Select the target region and press (+)</LI><LI style="color:#999999;margin-left:20px;">Select the N20 with the higher On-target score.</LI></OL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">2) Modification of pTarget</P></DIV><P>
			pTarget is currently available in two version, pTargetF with spectinomycin resistance and pTargetTet_pyrF and its derivatives with a tetracyclin resistance. They should behave identically.</P><OL type="1"><LI style="color:#999999;margin-left:20px;">Design a forward oligo to use for a QuickChange-like insertion of the N20 sequence. The sequence of the primer should be the following:</LI><P>agctagctcagtcctaggtataatactagtNNNNNNNNNNNNNNNNNNNNgttttagagctagaaatagcaagttaaaa</P><P>where the Ns stands for your N20 sequence.</P><LI style="color:#999999;margin-left:20px;">Amplify the pTarget backbone of your choice using the designed forward primer and the appropriate reverse primer (actagtattatacctaggactgagctagctg) using Q5 polymerase and the following conditions:</LI></OL><TABLE><TBODY><TR><TH style="padding:10px;border:1px solid #999;">Temperature (°C)</TH><TH style="padding:10px;border:1px solid #999;">Time</TH></TR><TR><TD style="padding:10px;border:1px solid #999;">98</TD><TD style="padding:10px;border:1px solid #999;">30 s</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">98</TD><TD style="padding:10px;border:1px solid #999;">15 s</TD><TD style="padding:10px;border:1px solid #999;" rowspan="3">35 x</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">65</TD><TD style="padding:10px;border:1px solid #999;">30 s</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">72</TD><TD style="padding:10px;border:1px solid #999;">75 s</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">72</TD><TD style="padding:10px;border:1px solid #999;">120 s</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">4</TD><TD style="padding:10px;border:1px solid #999;">inf.</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>Media</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">NaCl</LI><LI style="color:#999999;margin-left:20px;">Tryptone</LI><LI style="color:#999999;margin-left:20px;">Yeast Extract</LI><LI style="color:#999999;margin-left:20px;">Na2HPO4</LI><LI style="color:#999999;margin-left:20px;">KH2PO4</LI><LI style="color:#999999;margin-left:20px;">NH4Cl</LI><LI style="color:#999999;margin-left:20px;">Glucose</LI><LI style="color:#999999;margin-left:20px;">Tyrosine</LI><LI style="color:#999999;margin-left:20px;">ddH20</LI><LI style="color:#999999;margin-left:20px;">CaSO5</LI><LI style="color:#999999;margin-left:20px;">MgSO4</LI><LI style="color:#999999;margin-left:20px;">Agar</LI><LI style="color:#999999;margin-left:20px;">Heat-resistant glass bottles</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure M9 medium</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Make 10xM9 salts.</LI><LI style="color:#999999;margin-left:20px;">Mix the following in 500 mL H20 and sterilize by autoclave.</LI><TABLE><TBODY><TR><TH style="padding:10px;border:1px solid #999;">Mass (g)</TH></TR><TR><TD style="padding:10px;border:1px solid #999;">Na2HPO4</TD><TD style="padding:10px;border:1px solid #999;">64 (Na2HPO4.7H2O) </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">KH2PO4</TD><TD style="padding:10px;border:1px solid #999;">15 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">NaCl </TD><TD style="padding:10px;border:1px solid #999;">2.5 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">NH4Cl </TD><TD style="padding:10px;border:1px solid #999;">5 </TD></TR></TBODY></TABLE><LI style="color:#999999;margin-left:20px;">Make 1M CaCl2 stock solution. The CaCl2 is sterilized separately to avoid formation of CaSO4 precipitate. Dissolve 5.592 g CaC2 to 50 mL H2O and sterilize by filtration or autoclaving.</LI><LI style="color:#999999;margin-left:20px;">Make 1M MgSO4 stock solution.</LI><LI style="color:#999999;margin-left:20px;">Make 20% glucose solution. Dissolve 10g of glucose in 50 mL H2O and sterilize by filtration. Do not autoclave. Dissolve 6.018 g of MgSO4 in 50 mL H2O and sterilize by filtration or autoclaving.</LI><LI style="color:#999999;margin-left:20px;">Make 5x tyrosine solution. Dissolve 0.25 g of tyrosine in 1L H2O and sterilize by filtration.</LI><LI style="color:#999999;margin-left:20px;">Sterilize water by autoclaving.</LI><LI style="color:#999999;margin-left:20px;">For 500 mL.</LI><TABLE><TBODY><TR><TH style="padding:10px;border:1px solid #999;">M9</TH><TH style="padding:10px;border:1px solid #999;">M9 + Tyr</TH><TH style="padding:10px;border:1px solid #999;">M9 Agar</TH><TH style="padding:10px;border:1px solid #999;">M9 Agar + Tyr</TH></TR><TR><TD style="padding:10px;border:1px solid #999;">10x M9 salts</TD><TD style="padding:10px;border:1px solid #999;">50 </TD><TD style="padding:10px;border:1px solid #999;">50 </TD><TD style="padding:10px;border:1px solid #999;">50 </TD><TD style="padding:10px;border:1px solid #999;">50 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Agar (g)</TD><TD style="padding:10px;border:1px solid #999;">0 </TD><TD style="padding:10px;border:1px solid #999;">0 </TD><TD style="padding:10px;border:1px solid #999;">7.5 </TD><TD style="padding:10px;border:1px solid #999;">7.5 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">H2O </TD><TD style="padding:10px;border:1px solid #999;">445 </TD><TD style="padding:10px;border:1px solid #999;">345 </TD><TD style="padding:10px;border:1px solid #999;">445 </TD><TD style="padding:10px;border:1px solid #999;">345 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Autoclave </TD><TD style="padding:10px;border:1px solid #999;">No </TD><TD style="padding:10px;border:1px solid #999;">No </TD><TD style="padding:10px;border:1px solid #999;">Yes </TD><TD style="padding:10px;border:1px solid #999;">Yes </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">M1 MgSO4 </TD><TD style="padding:10px;border:1px solid #999;">1 </TD><TD style="padding:10px;border:1px solid #999;">1 </TD><TD style="padding:10px;border:1px solid #999;">1 </TD><TD style="padding:10px;border:1px solid #999;">1 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">M1 CaSO4 </TD><TD style="padding:10px;border:1px solid #999;">0,1 </TD><TD style="padding:10px;border:1px solid #999;">0,01 </TD><TD style="padding:10px;border:1px solid #999;">0,01 </TD><TD style="padding:10px;border:1px solid #999;">0,1 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">20% glucose </TD><TD style="padding:10px;border:1px solid #999;">5 </TD><TD style="padding:10px;border:1px solid #999;">5 </TD><TD style="padding:10px;border:1px solid #999;">5 </TD><TD style="padding:10px;border:1px solid #999;">5 </TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Tyrosine </TD><TD style="padding:10px;border:1px solid #999;">0 </TD><TD style="padding:10px;border:1px solid #999;">100 </TD><TD style="padding:10px;border:1px solid #999;">0 </TD><TD style="padding:10px;border:1px solid #999;">100 </TD></TR></TBODY></TABLE></OL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure LB medium</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Mix the following.</LI><TABLE><TBODY><TR><TH style="padding:10px;border:1px solid #999;">Volume (1L)</TH></TR><TR><TH style="padding:10px;border:1px solid #999;"> LB</TH><TD style="padding:10px;border:1px solid #999;">LB agar</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Yeast Extract (g)</TD><TD style="padding:10px;border:1px solid #999;">8 </TD><TD style="padding:10px;border:1px solid #999;">8</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Tryptone (g)</TD><TD style="padding:10px;border:1px solid #999;">5 </TD><TD style="padding:10px;border:1px solid #999;">5</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">NaCl (g)</TD><TD style="padding:10px;border:1px solid #999;">5 </TD><TD style="padding:10px;border:1px solid #999;">5</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Agar (g)</TD><TD style="padding:10px;border:1px solid #999;">0 </TD><TD style="padding:10px;border:1px solid #999;">14</TD></TR></TBODY></TABLE><LI style="color:#999999;margin-left:20px;">Add ddH2O to the desired volume.</LI><LI style="color:#999999;margin-left:20px;">Sterilize by autoclaving.</LI></OL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>Genome modification of E. coli using recombineering and CRISPR/Cas9</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Introduction</P></DIV><P>
			This protocol is adapted from Jiang et al., 2015. It assumes a version of pTargetTet targeting the correct locus of the E. coli genome has already been created, as well as a donor DNA that include overlap regions and the gene to be inserted. For information on the design and creation of the sgRNA-producing pTarget, see Selection of the sgRNA and modification of pTarget.
			</P><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">E. coli strain containing the pCas plasmid.</LI><LI style="color:#999999;margin-left:20px;">pTargetTet targeting the locus to be modified.</LI><LI style="color:#999999;margin-left:20px;">Donor DNA: gene of interest flanked by two 500-bp overlaps, or an ssDNA for deletion.</LI><LI style="color:#999999;margin-left:20px;">LB, Kanamycin, Tetracyclin.</LI><LI style="color:#999999;margin-left:20px;">Arabinose 10%.</LI><LI style="color:#999999;margin-left:20px;">Chilled sterile ddH2O.</LI><LI style="color:#999999;margin-left:20px;">Sterile PEP tube.</LI><LI style="color:#999999;margin-left:20px;">IPTG 0,1 M.</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><P>
			pCas has a thermo-sensitive origin of replication. All steps until pCas curing should be incubated at 30°C.
			</P><P><B>Day 1</B></P><UL><OL type="1"><LI style="color:#999999;margin-left:20px;">Start a 5 ml pre-culture of TOP10 pCas in LB-Kanamycin. Incubate it overnight at 30°C without shaking.</LI></OL></UL><P><B>Day 2</B></P><P><I>Competent cells</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Start a culture for transformation by inoculating 40 ml of LB-Kan with 400 µl of pre-culture. Incubate in a 250 ml flask at 30°C until an OD600 between 0,4 and 0,6 is attained. </LI><LI style="color:#999999;margin-left:20px;">Add 600 µl of 10% arabinose to the culture to cause the induction of the λ-Red genes. Incubate for 15 minutes at 30°C.</LI><LI style="color:#999999;margin-left:20px;">Chill the culture in a water-ice bath. Transfer to a 50 ml centrifuge tube and centrifuge at 4600 x g for 7 minutes in a centrifuge cooled to 4°C.</LI><LI style="color:#999999;margin-left:20px;">Add 30 ml of ice-cold water to the pellet. Swirl gently to resuspend the cells. Centrifuge again as described in step 5.</LI><LI style="color:#999999;margin-left:20px;">Resuspend the pellet in 1 ml of cold ddH2O. Transfer it using a 1000 µl tip to a pre-chilled sterile EP tube. Centrifuge down at 10.000 x g for 4 seconds in a chilled tabletop centrifuge. (Note, due to speed-up time, doing a quickspin up to 10.000 x g and then letting it slow down is usually good enough) Carefully pipette out the supernatant.</LI><LI style="color:#999999;margin-left:20px;">Repeat step 7.</LI><LI style="color:#999999;margin-left:20px;">Resuspend cells in 200 µl of chilled ddH2O. Keep on ice until ready to transform.</LI></OL><P><I>Transformation</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Using a 1-mm transformation culture, electroporate 100 ng of pTargetTet plasmid and 400 ng of linear donor DNA in 50 µl of competent cells (Note: when using an ssDNA oligo instead of dsDNA linear donor DNA, transform 100 ng of the plasmid with 100 ng of oligo). Recover the cells at 30°C for 1 hour in SOC medium then plate on LB-Tet-Kan plates. Incubate the plate at 30°C overnight. (Note: it often takes up to 48 hours for colony to show up).</LI><LI style="color:#999999;margin-left:20px;">Screen transformant by colony PCR to identify correct clones.</LI></OL><P><I>Plasmid curing</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">To cure the plasmids, resuspend a colony in 2 ml of LB-Kan to which 10 µl of IPTG 0,1 M are added (Final concentration: 0,5 mM) and the culture is grown overnight at 30 °C. In the morning, the colonies are plated on LB plates and incubated at 37°C O/N to cure them of the pCas plasmid.</LI></OL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>RFP fluorescence test</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">400 W LEO / S-400-94-CR light</LI><LI style="color:#999999;margin-left:20px;">Five 12-wells plates</LI><LI style="color:#999999;margin-left:20px;"> M9, LB</LI><LI style="color:#999999;margin-left:20px;">Tyrosine, ortho-nitrobenzyl tyrosine, L-arabinose</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Precultures procedure</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Inoculate 10 mL of liquid LB and incubate at 37°C overnight with appropriate antibiotic</LI><LI style="color:#999999;margin-left:20px;">Bl21(DE3), TyrA- (not transformed), TyrA- + RFP</LI></OL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Fluorescence test procedure</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Pellet precultures at 3000 xg for 10 minutes at 4°C</LI><LI style="color:#999999;margin-left:20px;">Wash 3x with 5 mL liquid M9 to remove tyrosine present in medium</LI><LI style="color:#999999;margin-left:20px;">When OD600 ~ 0.6, concentrate 10X</LI><LI style="color:#999999;margin-left:20px;">Add the following in each well of a 12-wells plate (final volume 3 mL, completed withliquid M9). Make three replicas.</LI></OL><TABLE><TBODY><TR><TD style="padding:10px;border:1px solid #999;">Tyr + Ara (BL21)</TD><TD style="padding:10px;border:1px solid #999;">Tyr (BL21)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr + Ara (BL21)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr (BL21)</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Tyr + Ara (TyrA-)</TD><TD style="padding:10px;border:1px solid #999;">Tyr (TyrA-)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr + Ara (TyrA-)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr (TyrA-)</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">Tyr + Ara (TyrA-/RFP)</TD><TD style="padding:10px;border:1px solid #999;">Tyr (TyrA-/RFP)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr + Ara (TyrA-/RFP)</TD><TD style="padding:10px;border:1px solid #999;">ONB-Tyr (TyrA-/RFP)</TD></TR></TBODY></TABLE><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Concentrations</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Tyrosine: 0.05 mg/mL</LI><LI style="color:#999999;margin-left:20px;">ONB-Tyr: 0.084 mg/mL</LI><LI style="color:#999999;margin-left:20px;">L-arabinose: 0.1% (w/v)</LI><LI style="color:#999999;margin-left:20px;">KAN30, CM34</LI></UL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>Glucuronidase expression test with ComRS (E. coli)</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Na2HPO4</LI><LI style="color:#999999;margin-left:20px;">KCl</LI><LI style="color:#999999;margin-left:20px;">MgSO4</LI><LI style="color:#999999;margin-left:20px;">ComS	</LI><LI style="color:#999999;margin-left:20px;">Na2CO3</LI><LI style="color:#999999;margin-left:20px;"><I>para</I>-nitrophenyl glucuronide</LI><LI style="color:#999999;margin-left:20px;">FastPrep machine</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><P><I>ComS induction</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Dilute preculture to OD600 = 0.1</LI><LI style="color:#999999;margin-left:20px;">Incubate 2h at 37°C</LI><LI style="color:#999999;margin-left:20px;">Induce for 2h at 37°C with 8 µM of ComS</LI></OL><P><I>Gus reaction test</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Pellet cells at 13 000 RPM</LI><LI style="color:#999999;margin-left:20px;">Resuspend in Gus tampon (60 mM Na2HPO4, 40 mM NaH2PO4, 10 mM KCl, 1 mM MgSO4, pH 7)</LI><LI style="color:#999999;margin-left:20px;">Lyse with FastPrep for cycles of 30 sec each (see FastPrep protocol)</LI><LI style="color:#999999;margin-left:20px;">Recover supernatant after centrifugation</LI><LI style="color:#999999;margin-left:20px;">Optional: mesure total concentration of protein with Bradford test</LI><LI style="color:#999999;margin-left:20px;">Add 4 µL of substrat (12.5 mM <I>para</I>-nitrophenyl glucuronide) per 100 µL of supernatant and incubate 10 minutes at 37°C</LI><LI style="color:#999999;margin-left:20px;">Stop reaction by addition of a 1 M Na2CO3 solution (150 µL per 100 µL of supernatant)</LI><LI style="color:#999999;margin-left:20px;">Mesure absorbance at 405 nm</LI></OL><P>
			Note: the specific activity for Gus is defined in arbitrary units as the concentration of para-nitrophénol (calculated with a standard curve) produced per minute and per mg of total protein.
			</P></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>PCR from DNA template or colony</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Phusion HF DNA Polymerase (NEB) or GoTaq Polymerase (Promega)</LI><LI style="color:#999999;margin-left:20px;">5x Phusion HF buffer or GoTaq Flexi Green</LI><LI style="color:#999999;margin-left:20px;">Template DNA</LI><LI style="color:#999999;margin-left:20px;">ddH20</LI><LI style="color:#999999;margin-left:20px;">PCR tubes</LI><LI style="color:#999999;margin-left:20px;">Primers (forward and reverse, 10 mM)</LI><LI style="color:#999999;margin-left:20px;">Thermocycler</LI><LI style="color:#999999;margin-left:20px;">dNTP solution (5 mM each)</LI><LI style="color:#999999;margin-left:20px;">25 mM MgCl2 solution</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><P><I>PCR reaction mix</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">In PCR tubes, add the following:</LI><TABLE><TBODY><TR><TD style="padding:10px;border:1px solid #999;">A</TD><TD style="padding:10px;border:1px solid #999;">B</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">1</TD><TD style="padding:10px;border:1px solid #999;"><B>Component</B></TD><TD style="padding:10px;border:1px solid #999;"><B><B>Volume (µL)</B></B></TD></TR><TR><TD style="padding:10px;border:1px solid #999;">2</TD><TD style="padding:10px;border:1px solid #999;">Buffer</TD><TD style="padding:10px;border:1px solid #999;">10</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">3</TD><TD style="padding:10px;border:1px solid #999;">Primer FWD</TD><TD style="padding:10px;border:1px solid #999;">2.5</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">4</TD><TD style="padding:10px;border:1px solid #999;">Primer REV</TD><TD style="padding:10px;border:1px solid #999;">2.5</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">5</TD><TD style="padding:10px;border:1px solid #999;">MgCl2</TD><TD style="padding:10px;border:1px solid #999;">2 (only Go Taq)</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">6</TD><TD style="padding:10px;border:1px solid #999;">dNTPs</TD><TD style="padding:10px;border:1px solid #999;">1</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">7</TD><TD style="padding:10px;border:1px solid #999;">Polymerase</TD><TD style="padding:10px;border:1px solid #999;">0.5</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">8</TD><TD style="padding:10px;border:1px solid #999;">DNA Template</TD><TD style="padding:10px;border:1px solid #999;">10-200 ng</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">9</TD><TD style="padding:10px;border:1px solid #999;">ddH20</TD><TD style="padding:10px;border:1px solid #999;">up to 50 µL</TD></TR></TBODY></TABLE><LI style="color:#999999;margin-left:20px;">Mix with up-and-downs</LI><LI style="color:#999999;margin-left:20px;">Spin gently to collect the liquid at the bottom</LI></OL><P><I>Thermocycler</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">In a PCR machine run the following program.</LI><P>
			Note: use <A style="color:#0066cc;" href="http://tmcalculator.neb.com/#!/">NEB's Tm Calculator</A> to estimate the annealing temperature.
			</P><TABLE><TBODY><TR><TD style="padding:10px;border:1px solid #999;">A</TD><TD style="padding:10px;border:1px solid #999;">B</TD><TD style="padding:10px;border:1px solid #999;">C</TD><TD style="padding:10px;border:1px solid #999;">D</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">1</TD><TD style="padding:10px;border:1px solid #999;"><B>Step</B></TD><TD style="padding:10px;border:1px solid #999;"><B>Temperature</B></TD><TD style="padding:10px;border:1px solid #999;"><B>Time</B></TD></TR><TR><TD style="padding:10px;border:1px solid #999;">2</TD><TD style="padding:10px;border:1px solid #999;">Initial Denaturation</TD><TD style="padding:10px;border:1px solid #999;">98°C</TD><TD style="padding:10px;border:1px solid #999;">5 minutes</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">3</TD><TD style="padding:10px;border:1px solid #999;">Denaturation</TD><TD style="padding:10px;border:1px solid #999;">98°C</TD><TD style="padding:10px;border:1px solid #999;">30 seconds</TD><TD style="padding:10px;border:1px solid #999;" rowspan="3">25-35 cycles</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">4</TD><TD style="padding:10px;border:1px solid #999;">Annealing</TD><TD style="padding:10px;border:1px solid #999;">See note</TD><TD style="padding:10px;border:1px solid #999;">30 seconds</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">5</TD><TD style="padding:10px;border:1px solid #999;" rowspan="2">Elongation</TD><TD style="padding:10px;border:1px solid #999;" rowspan="2">72 °C</TD><TD style="padding:10px;border:1px solid #999;">1 minute/kb (GoTaq)</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">6</TD><TD style="padding:10px;border:1px solid #999;">1 minute/kb (GoTaq)</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">7</TD><TD style="padding:10px;border:1px solid #999;">Final Elongation</TD><TD style="padding:10px;border:1px solid #999;">72 °C</TD><TD style="padding:10px;border:1px solid #999;">7 minutes</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">8</TD><TD style="padding:10px;border:1px solid #999;">Soak</TD><TD style="padding:10px;border:1px solid #999;">4°C</TD><TD style="padding:10px;border:1px solid #999;">∞</TD></TR></TBODY></TABLE><LI style="color:#999999;margin-left:20px;">Load DNA on gel or purify for downstream applications.</LI></OL><P><I>Colony PCR</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Same protocol, pick a colony to add in the mix instead of DNA.</LI><LI style="color:#999999;margin-left:20px;">Plate out colonies on LB agar for safekeeping</LI></OL></DIV><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>Gibson Assembly Protocol</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Introduction</P></DIV><P>
			This is adapted from the protocol for Gibson Assembly using the Gibson Assembly® Cloning Kit (NEB).
			</P><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Gibson Assembly Mastermix (NEB)</LI><LI style="color:#999999;margin-left:20px;">DNA Polymerases (for generating PCR products)</LI><P>
			         Recommended: Q5® High-Fidelity DNA Polymerase, Q5 Hot Start High-Fidelity DNA Polymerase, or Q5 Hot Start High-Fidelity 2X Master Mix
			        </P><LI style="color:#999999;margin-left:20px;">ddH20</LI><LI style="color:#999999;margin-left:20px;">LB plates with appropriate antibiotic</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><P><I>Set up the following reaction on ice:</I></P><OL type="1"><LI style="color:#999999;margin-left:20px;">Reaction volumes:</LI><P>
			        NEB recommends a total of 0.02–0.5 pmols of DNA fragments when 1 or 2 fragments are being assembled into a vector and 0.2–1.0 pmoles of DNA fragments when 4–6 fragments are being assembled. Efficiency of assembly decreases as the number or length of fragments increases. To calculate the number of pmols of each fragment for optimal assembly, based on fragment length and weight, we recommend using NEB's online tool, <A style="color:#0066cc;" href="http://nebiocalculator.neb.com/#!/ligation">NEBioCalculator</A>.
			        </P><TABLE><TBODY><TR><TH style="padding:10px;border:1px solid #999;">A</TH><TH style="padding:10px;border:1px solid #999;">B</TH><TD style="padding:10px;border:1px solid #999;">C</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">1</TD><TD style="padding:10px;border:1px solid #999;"><B>2-3 Fragments Assembly</B></TD><TD style="padding:10px;border:1px solid #999;"><B>4-6 Fragments Assembly</B></TD></TR><TR><TD style="padding:10px;border:1px solid #999;">2</TD><TD style="padding:10px;border:1px solid #999;"><B>Concentration Range of DNA fragments</B></TD><TD style="padding:10px;border:1px solid #999;">0.2 - 0.5 pmols*</TD><TD style="padding:10px;border:1px solid #999;">0.2 - 1.0 pmols*</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">3</TD><TD style="padding:10px;border:1px solid #999;"><B>Total Volume of Fragments (μl)</B></TD><TD style="padding:10px;border:1px solid #999;">Depends on DNA concentration</TD><TD style="padding:10px;border:1px solid #999;">Depends on DNA concentration</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">4</TD><TD style="padding:10px;border:1px solid #999;"><B>Gibson Assembly Master Mix (2x) (μl)</B></TD><TD style="padding:10px;border:1px solid #999;">10</TD><TD style="padding:10px;border:1px solid #999;">10</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">5</TD><TD style="padding:10px;border:1px solid #999;"><B>ddH20 (μl)</B></TD><TD style="padding:10px;border:1px solid #999;">Completing to reach 20</TD><TD style="padding:10px;border:1px solid #999;">Completing to reach 20</TD></TR><TR><TD style="padding:10px;border:1px solid #999;">6</TD><TD style="padding:10px;border:1px solid #999;"><B>Total Volume (μl) **</B></TD><TD style="padding:10px;border:1px solid #999;">20</TD><TD style="padding:10px;border:1px solid #999;">20</TD></TR></TBODY></TABLE><P> 
                                *Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%.
                               </P><P>
                                ** If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required.
                                </P><LI style="color:#999999;margin-left:20px;">Incubate samples in a thermocycler at 50°C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled.</LI><P>
                                Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases 
                                </P><LI style="color:#999999;margin-left:20px;">Store samples on ice or at –20°C for subsequent transformation.</LI><DIV class="col-md-12"><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><SPAN>FastPrep cell lysis</SPAN></DIV><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Introduction</P></DIV><P>
			This is the protocol for cell lysis using the FastPrep technique. Protocol received from Nadia El Bakkali Taheri.
			</P><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Materials</P></DIV><UL><LI style="color:#999999;margin-left:20px;">Glass beads (Glasperlen 0,17-0,18 mm, suspended in 10 ml PBS pH 7.5).</LI><LI style="color:#999999;margin-left:20px;">PBS pH 7.5</LI><LI style="color:#999999;margin-left:20px;">FastPrep machine</LI></UL><DIV class="section-title" style="text-align:left;float:left;width:100%;margin-bottom:0"><P class="montserrat-text uppercase">Procedure</P></DIV><OL type="1"><LI style="color:#999999;margin-left:20px;">Centrifuge 2 ml of cell culture at 4000 rpm and 4°C for 10 minutes.</LI><LI style="color:#999999;margin-left:20px;">Discard the supernatant and resuspend the cells in 500 µl of PBS.</LI><LI style="color:#999999;margin-left:20px;">With a cut tip, add 500 µl of the beads in PBS. Keep agitating to keep the beads in suspension.</LI><LI style="color:#999999;margin-left:20px;">Lyse using the FastPrep machine for 60 sec.</LI><LI style="color:#999999;margin-left:20px;">Centrifuge 10 minutes at maximum speed.</LI><LI style="color:#999999;margin-left:20px;">Recover the supernatant that include the lysate. </LI></OL></DIV></OL></DIV></DIV></DIV></SECTION><SECTION class="blue-section wow fadeInUp" style="padding:50px 0"><DIV class="container"><DIV class="col-md-12 col-sm-12"><DIV class="row"><SPAN class="white-text montserrat-text uppercase" style="font-size:30px;display:block;">
						you think we're cool? so are our sponsors!
					</SPAN></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>