Team:UCL/Methods
AMethods
PROJECT
Description
Results
Basic Parts
Composites
Gold Part
Interlab
HUMAN PRACTICES
Overview
Public Engagement
Integrated Practices
HP Silver
MODELLING
Overview
GOLIT
Optoflux
LEGIT
MOM
Methods
Glossary
ENTREPRENEURSHIP
Overview
Product
Business Model
OUR MEDALS
PEOPLE
Team
Collaborations
Attributions
LAB BOOK
Safety
Overview
Sensitivity Analysis
Parameter Sampling
Cracking the Model
Methods of parameter optimization
Mathematical Modelling describes complex biological systems with mathematical                     equations and concepts. Setting up the initial conditions in the model is not that straight forward.                     If no experimental data is available, reasonable assumptions based on literature, logic and                     quantitative analysis need to be made. We believe that the initial assumptions made are crucial                     to crack the model. Once established, the parameters need to constantly be optimized with experimental                     data and/or literature data in order to fit reality to its maximum. Parameter optimization adds all                     the weight to a model's credibility, this is why we decided to approach this in with 2 methods:      Sensitivity Analysis and Parameter Sampling.
Sensitivity Analysis
Sensitivity Analysis is a method which is conventionally used for parameter optimization.       A range of independent variables are tested in this technique and their impact on the dependent variable is evaluated. The independent variables which have the greatest influence on the dependant variable are then identified.        In our case, we  want to optimize the impact the values of specific independent variables have on the rate limiting step of our cellular mechanisms.       So, we are evaluating a range of values for each independent variable and analysing which ones optimize the process. We applied this method to       the LEGIT and the MOM models.
Method
Identify the dependent variable to be optimized (the rate limiting step in our process)
Identify the independent variables that have an impact on this step
From literature search/ experimental data, obtain a range of values to be tested for each independent variable
Run the model for the range of values tested and identify the optimal condition for the rate limiting step
Re-run the initial model to evaluate the impact of the optimized parameters on the overall process, i.e. to determine if the optimized step is prevailed the rate limiting step in the overall cellular mechanism
Determine the areas in which research could be performed to further optimize the parameters (Wet Lab contribution)
Sensitivity Analysis method applied to LEGIT model
Given that transport of intimin to the surface of the E.coli cells is the rate       limiting step in the overall process, we are rewriting this ODE in order to simplify       the Sensitivity Analysis on this step.
Parameter Sampling
Parameter Sampling is another technique used to tackle the issue of uncertain and incomplete knowledge of parameter values.                     It is a method of parameter optimization, which relies on the random sampling of parameter values within a specified range.                     A probability density function (pdf) determines the distribution of the continuous random variables evaluated in the function.                     Parameter Sampling considers biological noise, making it a more realistic representation of the behaviour of a biological system.
Method
Identify the dependent variable to be optimized (the rate limiting step in our process)
Identify the independent variables that have an impact on the step we are trying to optimize
Collect raw data from literature
Process raw data by assigning weights to the values depending on the similarity to the to-be-optimized parameter
Calculate the weighted mean for all the optimized parameters
Generate probability density functions (pdf);input the sample size, standard deviation and weighted mean
LEGIT model
Assessing the impact of randomly generated discrete variables on the rate of                     transport of intimin. For this the values of Km and Vmax are evaluated.
Conditions for intimin
Size: 94 kDa
Optimal pH conditions: 7
Native organism: E.coli
Weighted mean of Km: 0.0147 µM
Weighted mean of Vmax: 300000 s-1(1 s.f.)
Click here to check out the data collected to optimize the value of Km
Click here to check out the data collected to optimize the value of Vmax
MOM model
Assessing the impact of randomly generated discrete variables on the rate of translation of RFP. For this the rate of nuclear export of the transcript and the rate of RFP degradation are evaluated.
Conditions for RFP
Size: 27.9 kDa
Location of protein: Cytoplasm
Native organism: CHO cells
Weighted mean of rate of nuclear export: 1.43 x 10-4 s-1
Weighted mean of rate of degradation: 8.96 x 10-1 s-1
Click here to check out the data collected to optimize the  rate of nuclear export
Click here to check out the data collected to optimize the rate of degradation
Selecting our Programming Languages
This summer we decided to run our models on two different programming languages:                             MATLAB and Python. Both are high level languages that allowed us to create robust simulations.                             Upon starting iGEM, everyone on the modelling team wanted to learn a new programming language.                             Python is an open source software and it is relatively easy to find resources online to help                             troubleshoot our codes. We also decided to use MATLAB for our models, as we were already familiar                             with this language from our engineering degrees.
Bibliography
Thereza, C., Marina, I., Luciana.Cambricoli, d. and Olivia, C. (2004). Expression of green fluorescent protein (GFPuv) in Escherichia coli DH5-a, under different growth conditions. African Journal of Biotechnology, 3(1), pp.105-111.
Wlab.ethz.ch. (2017). Cell Surface Protein Atlas. [online] Available at: http://wlab.ethz.ch/cspa/#abstract [Accessed 4 Sep. 2017].
Sciencedirect.com. (2017). Membrane integration of E. coli model membrane proteins - ScienceDirect. [online] Available at: http://www.sciencedirect.com/science/article/pii/S0167488904000813 [Accessed 4 Sep. 2017].
Arun KH e. Green fluorescent proteins in receptor: an emerging tool for drug discovery. – PubMed –NCBI [Internet]. Ncbi.nlm.gov.2017 [cited 21 August 2017]. Available from: https://www.ncbi.nlm.nih.gov/pubmed/15596111
[internet] UVGermicidalTechnicalData. [cited 10 September 2017] http://www.pveducation.org/pvcdrom/effect-of-light-intensity (accessed 10 September 2017).
Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E. and Loo Poh, C.  Blue light-mediated transcriptional activation and repression of gene expression in bacteria. NCBI. 2016;44(14):6994–7005.
Philips, Ron. "Cell Biology By The Numbers." Book.bionumbers.org. N.p., 2017. Web. 10 Sept. 2017.
Kiparissides, Alexandros et al. "Modelling The Delta1/Notch1 Pathway: In Search Of The Mediator(S) Of Neural Stem Cell Differentiation." PLoS ONE 6.2 (2011): e14668. Web.
Carbonell-Ballestero M, Duran-Nebreda S, Montanez R, Sole R, Macia J, Rodriguez-Caso C. A bottom-up characterization of transfer functions for synthetic biology designs: lessons from enzymology. 2017.
Km & Vmax [Internet]. Mofetsrv.mofet.macam98.ac.il. 2017 [cited 21 September 2017]. Available from: http://mofetsrv.mofet.macam98.ac.il/~rafid/BIOLOGY/biochem/infopages/aminoacids/www-biol.paisley.ac.uk/courses/stfunmac/glossary/Km.html
Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
[Internet]. Karger.com. 2017 [cited 28 September 2017]. Available from: https://www.karger.com/Article/Pdf/131439
HeLa cell volume - Human Homo sapiens - BNID 103725 [Internet]. Bionumbers.hms.harvard.edu. 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=103725&ver=3
Baumann N, Vidugiriene J, Machamer C, Menon A. Cell Surface Display and Intracellular Trafficking of Free Glycosylphosphatidylinositols in Mammalian Cells. 2017.
Cortes L, Vainauskas S, Dai N, McClung C, Shah M, Benner J et al. Proteomic identification of mammalian cell surface derived glycosylphosphatidylinositol-anchored proteins through selective glycan enrichment. 2017.
Key Numbers for Cell Biologists [Internet]. Bionumbers.org; 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/Includes/KeyNumbersLinks.pdf
Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.
