<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Georgia_State_Mambalgin skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Georgia State/Mambalgin</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM Team:GSU</TITLE></P><DIV class="wrapper"><HEADER class="header"><DIV class="container"><NAV class="navigation singlepage-nav"><UL><LI><A class="current-ele" href="#intro">Home</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/Team">Team</A></LI><LI><A href="#Project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/ASF">Atlanta Science Festival</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/legislative">Georgia Capitol</A></LI></UL><LI><A href="https://2015.igem.org/Team:Georgia_State/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/ASF">Atlanta Science Festival</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Georgia_State/ImageGallery">Image Gallery</A></LI><LI><A href="#contact-info">Contact</A></LI></NAV></DIV></HEADER><SECTION id="intro" class="full-screen-intro bg-img29"><DIV class="full-cap-wraper"><DIV class="content-caption light-color"><DIV class="container"><H2 class="h1">Mambalgin</H2></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: #0387A6;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><P class="lead" style="color: white;"><A class="tipped" data-title="Abstract" data-tipper-options="{&quot;direction&quot;:&quot;top&quot;}"><B> 
			Mambalgin is a potent analgesic polypeptide found in the venom of the Black Mamba snake (Dendroaspis polylepis). Mambalgin-1, the version found here, is a 3-finger toxin consisting of 57 amino acid residues forming 3 loops around a core in the shape of a hand. This protein has been found to inhibit acid-sensing ion channels (ASICs) in the central and peripheral nervous systems of mice through intraplanter and intrathecal injections. The inhibition of ASICs – important contributors to the pain pathway in both mice and humans – decreases sensitivity of nociceptive neurons to the perception of pain. The potency of mambalgin has been compared to the drug morphine. Unlike morphine, however, mambalgin has not shown an increase in tolerance over time (Diochot et al, 2012).
Because of its potency and non-addictive properties, the potential of mambalgin as a pain reducer in humans is enticing. Synthetically producing this protein in large quantities would invalidate the need to harvest and extract directly from snakes – a dangerous and costly process.  
</B></A></P></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H2 class="h3" style="text-align: left;color: black;">Mambalgin E. Coli</H2><H3 class="h4" style="text-align: left;color: black;">Construct Design</H3><DIV><P class="lead" style="color: rgb(0, 0, 0); left"><B>
		For the design, we first codon optimized the Mambalgin-1 sequence (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1110003">BBa_K1110003</A>) for Escherichia coli using the IDT codon optimization tool. Restriction sites were then removed from the coding sequence that were iGEM incompatible. We added RFC 10 prefix and suffix and also included a start codon in the beginning of the sequence. Also included in the construct was a prokaryotic RBS (<A href="http://parts.igem.org/Part:BBa_J61101">BBa_J61101</A>), IPTG inducible promoter (<A href="http://parts.igem.org/Part:BBa_R0011">BBa_R0011</A>), and Myc and 6xHis epitopes. This fusion protein was designed specifically for easy expression in E. coli.
	</B></P></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H3 class="h4" style="text-align: right;color: black;">Experimental Design</H3><DIV><P class="lead" style="color: rgb(0, 0, 0);"><B> Our experiment explores the possibility of synthetically producing the analgesic mambalgin-1 protein using E. coli and the yeast Pichia pastoris. Currently, the most effective form of extraction involves milking the venom from Dendroaspis polylepis (black mamba) – a dangerous and costly process. For E. coli, we inserted the mambalgin E. coli construct into the pSB1C3 vector to create a recombinant plasmid. This was then transformed into BL21 expression strain E. coli, which was IPTG induced in liquid cell culture.

		</B></P></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H3 class="h4" style="text-align: right;color: black;">Results</H3><DIV><P class="lead" style="color: rgb(0, 0, 0);"><B> The 2015 GSU iGEM team was able to successfully express and characterize the mambalgin protein in E. coli. After inducing with IPTG overnight, the cell cultures were centrifuged and disrupted with a French press. Mambalgin was isolated using affinity chromatography, utilizing the 6x his tag in the construct. A SDS-PAGE was done, followed by a Ponceau S. stain to visualize the proteins. A band indicating a protein of ~9 kda – the size of the mambalgin for E. coli construct – was seen in the elute fraction.

		</B></P></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H2 class="h3" style="text-align: left;color: black;">Mambalgin Yeast</H2><H3 class="h4" style="text-align: left;color: black;">Construct Design</H3><DIV><P class="lead" style="color: rgb(0, 0, 0);"><B> For the design, we took the coding sequence for mambalgin, removed the TAG stop codon, and added RFC 25 prefix and suffix in Snapgene software to visualize. RFC 25 was used because we were anticipating the use of this part in creating fusion proteins. Additional nucleotides were added before the biobrick prefix and after the biobrick suffix to enable more efficient restriction enzyme cutting at EcoRI and PstI sites. The mambalgin construct was then ordered from IDT and ligated into pSB1C3 backbone.
Using the pGAPzα expression system provided by ThermoFisher Scientific, the GSU team has been working to express the protein in Pichia Pastoris. The pGAPzα vector includes the constitutive glyceraldehyde-3-phosphate dehydrogenase (GAP) promoter, an alpha secretory signal peptide, and Myc and 6x His epitopes. The MCS, between the alpha secretion signal and epitopes, consists of restriction sites which will enable insertion of mambalgin. In-frame insertion into the MCS of pGAPza will be possible after using PCR to modify the construct - removing the RFC 25 prefix and suffix and adding restriction sites and nucleotides.

		</B></P></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H3 class="h4" style="text-align: right;color: black;">Experimental Design</H3><DIV><P class="lead" style="color: rgb(0, 0, 0);"><B> Our experiment explores the possibility of synthetically producing the analgesic mambalgin-1 protein using E. coli and the yeast Pichia pastoris. Currently, the most effective form of extraction involves milking the venom from Dendroaspis polyepis (black mamba) – a dangerous and costly process. With Pichia pastoris, we chose to use the shuttle vector pGAPza. This plasmid includes an alpha-factor secretion signal and epitopes enabling secretion of the protein into the culture and easier characterization. To make in-frame insertion into pGAPza possible, the mambalgin for Pichia construct was modified using extension PCR. Using primers, the biobrick prefix and suffix were effectively replaced with restriction sites and nucleotides for correct in-frame insertion. The recombinant pGAPza plasmid was then transformed into Pichia pastoris and grown for expression.

	</B></P></DIV></DIV></DIV></DIV></SECTION><SECTION id="about" class="wow fadeIn ptb ptb-sm-80" style="background-color: white;"><DIV class="container"><DIV class="row text-center"><DIV class="col-md-12"><H3 class="h4" style="text-align: right;color: black;">Results</H3><DIV><P class="lead" style="color: rgb(0, 0, 0);"><B> 
As of yet, we have not successfully been able to express mambalgin in Pichia pastoris.


	</B></P></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>