<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-Software_Models"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-Software/Models">Page               </A></LI><LI><A href="/Talk:Team:SYSU-Software/Models">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/Models&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/Models&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-Software/Models" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-Software/Models</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TITLE>无标题 1</TITLE></DIV> <DIV class="content-background"><DIV class="wrapper"><DIV id="menu"><UL id="nav"><LI class="mainlevel" id="mainlevel_01"><A href="https://2012.igem.org/Team:SYSU-Software"> Home</A></LI><LI class="mainlevel" id="mainlevel_02"><A href="https://2012.igem.org/Team:SYSU-Software/Project">Project</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Models">Models</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Tutorial">Tutorial</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Team">Team</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Notebook">Notebook</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Practice">
	Tips</A></LI></UL></DIV><DIV id="content"><DIV class="entry page clear"><H1 id="doctitle" class="pagetitle" style="line-height: 1.5em;">
		Algorithm&amp;Models</H1><TABLE class="doc-toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL id="toc-ul"><LI class="toclevel-1"><A href="#pp1"><SPAN class="toctext"> Flux Balance Analysis </SPAN></A></LI><LI class="toclevel-1"><A href="#pp2">SiRNA designer</A></LI><LI class="toclevel-1"><A href="#pp3">Riboswitch designer</A></LI></UL></TD></TR></TBODY></TABLE><DIV id="doc-contents"><DIV><H1 id="pp1">Flux Balance Analysis</H1><P>      Traditional approaches for metabolic network exploration are mainly based on physicochemical kinetics laws and principles. However,the difficulty for monitoring nutritional changes in metabolic biosystem neccecitates a more reductionist form of information especially available for computational processing. In 1992, Savinell and Palsson[1,2]proposed flux balance analysis(FBA) to simulate microbial metabolism and in 1993 FBA was applied in exploration of E.coli System[3,4,5].Thereafer, Palsson and his parters extend the uses of FBA even to genetic level[6,7].</P></DIV><DIV><P>      Here are the mathematical description of FBA[8]. Firstly, FBA assumes that metabolic networks will reach homeostasis constrained by stoichiometry without knowing material changes all along the process. The values in this stoichiometric matrix are the stoichiometric coefficients of each reaction in the system as the set of constraints for the optimization. The mathematical description can be interpreted in the simple example with graphs following.</P></DIV><DIV><P>      In the simple biosystem illustrated above, the compounds served as the nodes are connected by the reactions for conversion among compounds. The increase of Compound A is due to the converstion from Compound C and A, while the decrease of Compound A is the result of the conversion from Compound A to C. The graph below shows the overall change of the amount per time unit of Compound A, B, C. </P><P>      Since the whole system will reach homeostasis, which means a state that compound in the system will be constant, in other words, the flux-in and flux-out of the node(the compound) will be equal. That is why we call this method ‘Flux Balance Analysis’.</P><P>      Secondly, the results above can be translated into a stoichiometric matrix. In Matrix S below.</P><P>    Each col of Matrix S represents stoichiometric coefficients the related to each compound and each row of Matrix S refers to coefficients of each reaction according to the three equations above. The plus and minus signs are determined by the direction of reations according to the compound itself.</P><P>      While in the vector above, v1 and v2 represents the reaction rate namely fluxes of the two reactions in this sample system.</P><P>So you can see the multiplication of Matrix S and the vector can return to those three original equations above. As has been explained, the multiplication will be the zero vector.</P><P>      In the general case we can write:</P><P>       </P><P>      Thirdly,with stoichiometry prepared and the objective function(the biomass function) determined, linear programming can be performed for optimization.</P><P>     </P><P>      In many cases, constraints are set upon the values of fluxes based on some thermodynamic conditions.</P><P>   </P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-no-proof:yes">(All the graphs 
are from Wikipedia.)</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-family:&quot;Times New Roman&quot;,&quot;serif&quot;;
mso-no-proof:yes">Reference:</SPAN></P><P> 1.Savinell JM, Palsson BO: Optimal selection of metabolic fluxes for in vivo measurement. I. Development of mathematical methods. J Theor Biol 1992, 155:201-214.</P><P>2. Savinell JM, Palsson BO: Optimal selection of metabolic fluxes for in vivo measurement. II. Application to Escherichia coli and hybridoma cell metabolism. J Theor Biol 1992,155:215-242.</P><P>3.Varma A, Palsson BO: Metabolic capabilities of Escherichia coli:I. Synthesis of biosynthetic precursors and cofactors. J TheorBiol 1993, 165:477-502.</P><P>4. Varma A, Palsson BO: Metabolic capabilities of Escherichia coli:II. Optimal growth patterns. J Theor Biol 1993,165:503-522.</P><P>5. Varma A, Palsson BO: Stoichiometric flux balance models quantitatively predict growth and metabolic by-product secretion in wild-type Escherichia coli W3110. Appl Environ Microbiol 1994, 60:3724-3731.</P><P>6. Edwards JS, Palsson BO: The Escherichia coli MG1655 in silico metabolic genotype: its definition, characteristics, and capabilities. Proc Natl Acad Sci USA 2000, 97:5528-5533.</P><P>7.Edwards JS, Palsson BO: Robustness analysis of the Escherichia coli metabolic network. Biotechnol Prog 2000, 16:927-939.</P><P>8. Kenneth J Kauffman, Purusharth Prakash and Jeremy S Edwards: Advances in flux balance analysis, Current Opinion in Biotechnology 2003, 14:491–496</P><P> </P><P> </P><H1 id="pp2">SiRNA designer</H1><H3><SPAN lang="EN-US" style="background: white">Tom Tuschl's rule (for 
		cDNA)</SPAN></H3><P class="MsoNormal"><SPAN lang="EN-US">1. Select targeted region from a 
		given cDNA sequence beginning 50-100 nt downstream of start codon</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">2. Look for sequence motif 
		AA+N19+TT first. If there is no suitable sequence look 23-nt sequence 
		motif NA+N21 and convert the 3' end of the sense siRNA to TT</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">3. Or search for NAR+N17+YNN</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">PS: Target sequence should have 
		a GC content of around 50%</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">R means A/G, and Y means T/C.</SPAN></P><H3><SPAN lang="EN-US" style="background: white">Rational siRNA 
		design(for mRNA)</SPAN></H3><P class="MsoNormal"><SPAN lang="EN-US">Evaluate potential candidates 
		and assign scores to them, sequences with higher scores will have higher 
		chance of success in RNAi. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">The table below lists the 8 
		criteria and the methods of score assignment.</SPAN></P><TABLE class="MsoNormalTable" border="1" cellspacing="0" cellpadding="0" width="708" style="width:425.0pt;border-collapse:collapse;border:none;mso-border-top-alt:
 solid windowtext .5pt;mso-border-bottom-alt:solid windowtext .5pt;mso-yfti-tbllook:
 1184;mso-padding-alt:0cm 5.4pt 0cm 5.4pt;mso-border-insideh:.5pt solid windowtext"><TBODY><TR style="mso-yfti-irow:0;mso-yfti-firstrow:yes"><TD width="72" rowspan="2" style="width:42.9pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:solid windowtext 1.0pt;border-right:none;
  mso-border-top-alt:solid windowtext .5pt;mso-border-bottom-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" align="center"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Criteria</SPAN></P></TD><TD width="386" rowspan="2" style="width:231.85pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:solid windowtext 1.0pt;border-right:none;
  mso-border-top-alt:solid windowtext .5pt;mso-border-bottom-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" align="center"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Description</SPAN></P></TD><TD width="250" colspan="2" valign="top" style="width:150.25pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:solid windowtext 1.0pt;border-right:none;
  mso-border-top-alt:solid windowtext .5pt;mso-border-bottom-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" align="center"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Score</SPAN></P></TD></TR><TR style="mso-yfti-irow:1"><TD width="154" style="width:92.15pt;border:none;border-bottom:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-top-alt:solid windowtext .5pt;
  mso-border-bottom-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" align="center"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Yes</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;border-bottom:solid windowtext 1.0pt;
  mso-border-top-alt:solid windowtext .5pt;mso-border-top-alt:solid windowtext .5pt;
  mso-border-bottom-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" align="center"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">No</SPAN></P></TD></TR><TR style="mso-yfti-irow:2"><TD width="72" valign="top" style="width:42.9pt;border:none;mso-border-top-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;mso-border-top-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Moderate to 
				low (30%-52%) GC Content</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;mso-border-top-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;mso-border-top-alt:
  solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:3"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">2</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">At least 3 
				A/Us at positions 15-19 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point 
				/per A or U</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:4"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">3</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Melting 
				temperature-Tm*&lt;20</SPAN><SPAN style="mso-bidi-font-size:10.5pt;
  font-family:宋体;mso-ascii-font-family:&quot;Times New Roman&quot;;mso-hansi-font-family:
  &quot;Times New Roman&quot;">℃</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:5"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">4</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">A at 
				position 19 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:6"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">5</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">A at 
				position 3 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:7"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">6</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">U at 
				position 10 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1 point</SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD></TR><TR style="mso-yfti-irow:8"><TD width="72" valign="top" style="width:42.9pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">7</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">No G/C at 
				position 19 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">-1 point</SPAN></P></TD></TR><TR style="mso-yfti-irow:9;mso-yfti-lastrow:yes"><TD width="72" valign="top" style="width:42.9pt;border:none;border-bottom:solid windowtext 1.0pt;
  mso-border-bottom-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">8</SPAN></P></TD><TD width="386" valign="top" style="width:231.85pt;border:none;border-bottom:
  solid windowtext 1.0pt;mso-border-bottom-alt:solid windowtext .5pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">No G at 
				position 13 (sense)</SPAN></P></TD><TD width="154" valign="top" style="width:92.15pt;border:none;border-bottom:solid windowtext 1.0pt;
  mso-border-bottom-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P></TD><TD width="97" valign="top" style="width:58.1pt;border:none;border-bottom:solid windowtext 1.0pt;
  mso-border-bottom-alt:solid windowtext .5pt;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">-1 point</SPAN></P></TD></TR></TBODY></TABLE><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">The &quot;anti-sense&quot; 
		strand is the siRNA strand that is complementary to the target mRNA and 
		that will be binding to the mRNA.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">The melting 
		temperature’ of a siRNA candidate can be calculated by the formula 
		showed below:</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">wA, xT, yG, zC are 
		separately the number of A, T, G, C in a siRNA candidate.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size: 10.0pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black; background: white">
		All siRNA candidates scored higher than 6 are acceptable.</SPAN></P><P class="MsoNormal" align="left"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P><H1 id="pp3"><SPAN lang="EN-US">Riboswitch designer:</SPAN></H1><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">The designer is 
		based on a special promoter working in eukaryotes - internal ribosome 
		entry site (IRES). We design the upstream sequence that can bind to IRES 
		part so the second structure of IRES can be transformed by the ligand.</SPAN></P><H3><SPAN lang="EN-US">Up regulated:</SPAN></H3><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Target sequence is 
		assembled by five parts: MS-SL, aaIRES (anti-anti-IRES), aptamer, aIRES 
		(anti-IRES), IRES.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">The IRES, aIRES, 
		aaIRES parts are settled, we get aptamer sequence according to 
		customer’s quests from aptamer database and the MS-SL is paired to the 
		combination part of aaIRES and aptamer. Then adjust the MS-SL part’s 
		length to fit it’s free energy to -11.7 kcal/mol.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P><H3><SPAN lang="EN-US">Down regulated:</SPAN></H3><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">The differences 
		from down regulated to up regulated riboswitch are the order between 
		aptamer and aaIRES, the sequence of MS part. The new structure is showed 
		below:</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">We set MS to 
		“CCUCU” under experimental data and the aptamer still comes from aptamer 
		database.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P><H3><SPAN lang="EN-US">Algorithm about free energy calculation comes 
		from Turner’s team:</SPAN></H3><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:
normal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Hairpin loops:</SPAN></I></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">In this equation, n 
		(5) is the number of nucleotides in loop, the terminal mismatch 
		parameter is the sequence-dependent term for the first mismatch stacking 
		on the terminal base pair.</SPAN></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:
normal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Watson-Crick 
		Helices:</SPAN></I></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">Specific data can 
		be found on web site:</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US">
		http://rna.urmc.rochester.edu/NNDB/turner04/index.html</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt"><O:P> </O:P></SPAN></P><H2><SPAN lang="EN-US">References:</SPAN></H2><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">1.Elbashir SM et 
		al. (2001) Duplexes of 21-nucleotide RNAs mediate RNA interference in 
		cultured mammalian cells. Nature. 411:494-498.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">2.Elbahir SM et al. 
		(2001). Functional anatomy of siRNAs for mediating efficient RNAi in 
		Drosophila melanogaster embryo lysate. EMBO J. 20:6877-6888.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">3.Elbashir SM et 
		al. (2002). Analysis of gene function in somatic mammalian cells using 
		small interfering RNAs. Methods. 26:199-213.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">4.Reynolds A, Leake 
		D, Boese Q, Scaringe S, Marshall WS, Khvorova A. Rational siRNA design 
		for RNA interference. Nat Biotechnol. 2004 Mar;22(3):326-30.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">5.Ogawa, A. (2011). 
		Rational design of artificial riboswitches based on ligand-dependent 
		modulation of internal ribosome entry in wheat germ extract and their 
		applications as label-free biosensors. RNA (New York, N.Y.), 17(3), 
		478-88. doi:10.1261/rna.2433111</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">6.Ogawa, A. (2012). 
		Rational construction of eukaryotic OFF-riboswitches that downregulate 
		internal ribosome entry site-mediated translation in response to their 
		ligands. Bioorganic &amp; medicinal chemistry letters, 22(4), 1639-42. 
		Elsevier Ltd. doi:10.1016/j.bmcl.2011.12.118</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="mso-bidi-font-size:10.5pt">7.Turner, D. H. &amp; 
		Mathews, D. H.<SPAN style="mso-spacerun:yes"> </SPAN>(2009).<SPAN style="mso-spacerun:yes"> </SPAN>NNDB: The nearest neighbor parameter database for predicting 
		stability of nucleic acid secondary structure.<SPAN style="mso-spacerun:yes"> </SPAN>Nucleic Acids Research.<SPAN style="mso-spacerun:yes"> </SPAN>38, D280-D282.</SPAN></P></DIV></DIV></DIV></DIV><DIV id="sidebar"><DIV id="sidebar"><DIV id="doc-nav"><DIV><H5>IGEM-SYSU-Software </H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software">Main Page</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Team">Album</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/SecondEdition">Second Edition</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Accomplishment">Accomplishment</A></LI></UL></DIV></DIV><DIV><H5>User Guide</H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Download">
						Download</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Tutorial">Tutorial</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Video">Video</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Email</A></LI></UL></DIV></DIV><DIV><H5>FAQ </H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Faq">Questions</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Trouble">Troubleshooting</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Uninstalling">Uninstalling</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Contact Us</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><DIV class="wrapper clear"><DIV id="footer-about" class="footer-column"><H2>   About</H2><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Team">Developers</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Contact Us</A></LI></UL></DIV><DIV id="footer-flickr" class="footer-column"><H2>   Links</H2><UL><LI><A href="http://www.sysu.edu.cn">SUN YAT-SEN University</A></LI><LI><A href="http://lifescience.sysu.edu.cn/main/default/index.aspx">SYSU LifeScience</A></LI></UL></DIV><DIV id="footer-search" class="footer-column"><H2>    Facebook</H2></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SYSU-Software/Models">http://2012.igem.org/Team:SYSU-Software/Models</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-Software/Models" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-Software/Models" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-Software/Models&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-Software/Models&amp;oldid=228502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>