"
Page
Discussion
View source
History
teams
Log in
 
Team:SYSU-Software/Models
From 2012.igem.org
无标题 1
 
 Home
Project
Models
Tutorial
Team
Notebook
Tips
Algorithm&Models
Contents
Flux Balance Analysis
SiRNA designer
Riboswitch designer
Flux Balance Analysis
      Traditional approaches for metabolic network exploration are mainly based on physicochemical kinetics laws and principles. However,the difficulty for monitoring nutritional changes in metabolic biosystem neccecitates a more reductionist form of information especially available for computational processing. In 1992, Savinell and Palsson[1,2]proposed flux balance analysis(FBA) to simulate microbial metabolism and in 1993 FBA was applied in exploration of E.coli System[3,4,5].Thereafer, Palsson and his parters extend the uses of FBA even to genetic level[6,7].
      Here are the mathematical description of FBA[8]. Firstly, FBA assumes that metabolic networks will reach homeostasis constrained by stoichiometry without knowing material changes all along the process. The values in this stoichiometric matrix are the stoichiometric coefficients of each reaction in the system as the set of constraints for the optimization. The mathematical description can be interpreted in the simple example with graphs following.
      In the simple biosystem illustrated above, the compounds served as the nodes are connected by the reactions for conversion among compounds. The increase of Compound A is due to the converstion from Compound C and A, while the decrease of Compound A is the result of the conversion from Compound A to C. The graph below shows the overall change of the amount per time unit of Compound A, B, C.
      Since the whole system will reach homeostasis, which means a state that compound in the system will be constant, in other words, the flux-in and flux-out of the node(the compound) will be equal. That is why we call this method ‘Flux Balance Analysis’.
      Secondly, the results above can be translated into a stoichiometric matrix. In Matrix S below.
    Each col of Matrix S represents stoichiometric coefficients the related to each compound and each row of Matrix S refers to coefficients of each reaction according to the three equations above. The plus and minus signs are determined by the direction of reations according to the compound itself.
      While in the vector above, v1 and v2 represents the reaction rate namely fluxes of the two reactions in this sample system.
So you can see the multiplication of Matrix S and the vector can return to those three original equations above. As has been explained, the multiplication will be the zero vector.
      In the general case we can write:
       
      Thirdly,with stoichiometry prepared and the objective function(the biomass function) determined, linear programming can be performed for optimization.
     
      In many cases, constraints are set upon the values of fluxes based on some thermodynamic conditions.
   
(All the graphs  are from Wikipedia.)
Reference:
 1.Savinell JM, Palsson BO: Optimal selection of metabolic fluxes for in vivo measurement. I. Development of mathematical methods. J Theor Biol 1992, 155:201-214.
2. Savinell JM, Palsson BO: Optimal selection of metabolic fluxes for in vivo measurement. II. Application to Escherichia coli and hybridoma cell metabolism. J Theor Biol 1992,155:215-242.
3.Varma A, Palsson BO: Metabolic capabilities of Escherichia coli:I. Synthesis of biosynthetic precursors and cofactors. J TheorBiol 1993, 165:477-502.
4. Varma A, Palsson BO: Metabolic capabilities of Escherichia coli:II. Optimal growth patterns. J Theor Biol 1993,165:503-522.
5. Varma A, Palsson BO: Stoichiometric flux balance models quantitatively predict growth and metabolic by-product secretion in wild-type Escherichia coli W3110. Appl Environ Microbiol 1994, 60:3724-3731.
6. Edwards JS, Palsson BO: The Escherichia coli MG1655 in silico metabolic genotype: its definition, characteristics, and capabilities. Proc Natl Acad Sci USA 2000, 97:5528-5533.
7.Edwards JS, Palsson BO: Robustness analysis of the Escherichia coli metabolic network. Biotechnol Prog 2000, 16:927-939.
8. Kenneth J Kauffman, Purusharth Prakash and Jeremy S Edwards: Advances in flux balance analysis, Current Opinion in Biotechnology 2003, 14:491–496
 
 
SiRNA designer
Tom Tuschl's rule (for    cDNA)
1. Select targeted region from a    given cDNA sequence beginning 50-100 nt downstream of start codon
2. Look for sequence motif    AA+N19+TT first. If there is no suitable sequence look 23-nt sequence    motif NA+N21 and convert the 3' end of the sense siRNA to TT
3. Or search for NAR+N17+YNN
PS: Target sequence should have    a GC content of around 50%
R means A/G, and Y means T/C.
Rational siRNA    design(for mRNA)
Evaluate potential candidates    and assign scores to them, sequences with higher scores will have higher    chance of success in RNAi.
The table below lists the 8    criteria and the methods of score assignment.
Criteria
Description
Score
Yes
No
1
Moderate to      low (30%-52%) GC Content
1 point
 
2
At least 3      A/Us at positions 15-19 (sense)
1 point      /per A or U
 
3
Melting      temperature-Tm*<20℃
1 point
 
4
A at      position 19 (sense)
1 point
 
5
A at      position 3 (sense)
1 point
 
6
U at      position 10 (sense)
1 point
 
7
No G/C at      position 19 (sense)
 
-1 point
8
No G at      position 13 (sense)
 
-1 point
The "anti-sense" strand is the siRNA strand that is complementary to the target mRNA and    that will be binding to the mRNA.
The melting    temperature’ of a siRNA candidate can be calculated by the formula    showed below:
wA, xT, yG, zC are    separately the number of A, T, G, C in a siRNA candidate.
All siRNA candidates scored higher than 6 are acceptable.
 
Riboswitch designer:
The designer is    based on a special promoter working in eukaryotes - internal ribosome    entry site (IRES). We design the upstream sequence that can bind to IRES    part so the second structure of IRES can be transformed by the ligand.
Up regulated:
Target sequence is    assembled by five parts: MS-SL, aaIRES (anti-anti-IRES), aptamer, aIRES    (anti-IRES), IRES.
The IRES, aIRES,    aaIRES parts are settled, we get aptamer sequence according to    customer’s quests from aptamer database and the MS-SL is paired to the    combination part of aaIRES and aptamer. Then adjust the MS-SL part’s    length to fit it’s free energy to -11.7 kcal/mol.
 
Down regulated:
The differences    from down regulated to up regulated riboswitch are the order between    aptamer and aaIRES, the sequence of MS part. The new structure is showed    below:
We set MS to    “CCUCU” under experimental data and the aptamer still comes from aptamer    database.
 
Algorithm about free energy calculation comes    from Turner’s team:
Hairpin loops:
In this equation, n    (5) is the number of nucleotides in loop, the terminal mismatch    parameter is the sequence-dependent term for the first mismatch stacking    on the terminal base pair.
Watson-Crick    Helices:
Specific data can    be found on web site:
http://rna.urmc.rochester.edu/NNDB/turner04/index.html
 
References:
1.Elbashir SM et    al. (2001) Duplexes of 21-nucleotide RNAs mediate RNA interference in    cultured mammalian cells. Nature. 411:494-498.
2.Elbahir SM et al.    (2001). Functional anatomy of siRNAs for mediating efficient RNAi in    Drosophila melanogaster embryo lysate. EMBO J. 20:6877-6888.
3.Elbashir SM et    al. (2002). Analysis of gene function in somatic mammalian cells using    small interfering RNAs. Methods. 26:199-213.
4.Reynolds A, Leake    D, Boese Q, Scaringe S, Marshall WS, Khvorova A. Rational siRNA design    for RNA interference. Nat Biotechnol. 2004 Mar;22(3):326-30.
5.Ogawa, A. (2011).    Rational design of artificial riboswitches based on ligand-dependent    modulation of internal ribosome entry in wheat germ extract and their    applications as label-free biosensors. RNA (New York, N.Y.), 17(3),    478-88. doi:10.1261/rna.2433111
6.Ogawa, A. (2012).    Rational construction of eukaryotic OFF-riboswitches that downregulate    internal ribosome entry site-mediated translation in response to their    ligands. Bioorganic & medicinal chemistry letters, 22(4), 1639-42.    Elsevier Ltd. doi:10.1016/j.bmcl.2011.12.118
7.Turner, D. H. & Mathews, D. H.  (2009).  NNDB: The nearest neighbor parameter database for predicting    stability of nucleic acid secondary structure.  Nucleic Acids Research.  38, D280-D282.
IGEM-SYSU-Software
Main Page
Album
Second Edition
Accomplishment
User Guide
Download
Tutorial
Video
Email
FAQ
Questions
Troubleshooting
Uninstalling
Contact Us
   About
Developers
Contact Us
   Links
SUN YAT-SEN University
SYSU LifeScience
    Facebook
Retrieved from " http://2012.igem.org/Team:SYSU-Software/Models "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
