<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-Software_SecondEdition"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-Software/SecondEdition">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-Software/SecondEdition&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/SecondEdition&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/SecondEdition&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-Software/SecondEdition" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-Software/SecondEdition</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TITLE>无标题 1</TITLE></DIV> <DIV class="content-background"><DIV class="wrapper"><DIV id="menu"><UL id="nav"><LI class="mainlevel" id="mainlevel_01"><A href="https://2012.igem.org/Team:SYSU-Software"> Home</A></LI><LI class="mainlevel" id="mainlevel_02"><A href="https://2012.igem.org/Team:SYSU-Software/Project">Project</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Models">Models</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Tutorial">Tutorial</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Team">Team</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Notebook">Notebook</A></LI><LI class="mainlevel"><A href="https://2012.igem.org/Team:SYSU-Software/Practice">
	Tips</A></LI></UL></DIV><DIV id="content"><DIV class="entry page clear"><H1 id="doctitle" class="pagetitle">Second Edition</H1><TABLE class="doc-toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL id="toc-ul"><LI class="toclevel-1"><A href="#pp1"><SPAN class="toctext"> Overall</SPAN></A></LI><LI class="toclevel-1"><A href="#pp2">Genome Browser</A></LI><LI class="toclevel-1"><A href="#pp3">Biobrick</A></LI><LI class="toclevel-1"><A href="#pp4">Regulator Designer</A></LI><LI class="toclevel-1"><A href="#pp5">Simulator</A></LI><LI class="toclevel-1"><A href="#pp6">Trans-platform Realization</A></LI></UL></TD></TR></TBODY></TABLE><DIV id="doc-contents"><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN id="pp1" lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
Overall</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
The first version of <B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:
normal">BiArkit</I></B> is now completed and released. The second version,
<B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal">
BiArkit 2.0</I></B>, has been proposed and designed to be more competent to 
fulfill the needs of synthetic biology researchers.
<B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal">
BiArkit 2.0 </I></B>will still concentrate on how to deeply exploit 
bio-information for researchers to design and execute their further experiments. 
First, every kit will be optimized functionally and ameliorated to be more 
user-friendly. Second, the explosion of second generation sequencing motivates 
us to involve the next generation sequencing data to <B><I>BiArkit</I></B><B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal">2.0</I></B>. 
Third, for more information coming from biological systems, including 
reconstruction networks, molecular interactions, and artificial networks, we 
will make standards and integrate them into one section to provide the 
comprehensive prospects for researchers to employ. Fourth, as
<B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:
normal">BiArkit</I></B> was coden by <B><I>JAVA</I></B>, we plan to make this 
software run not only on <B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal">Win</I></B> but also on the platform of
<B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal">Mac</I></B> 
and<B style="mso-bidi-font-weight:normal"><I style="mso-bidi-font-style:normal"> 
Linux</I></B>. </SPAN></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN id="pp2" lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
GenomeBrowser</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
The updated GenoneBrowser will incorporate bio-information out of RNA-Seq, 
ChIP-Seq and tiling-array to illustrate the genome expression level under given 
conditions and locate molecular that interact with the genomes. With the known 
annotation, the genomes can be divided according to the products they 
transcribed. Besides, for make this kit more user friendly, we are going to 
refine it and allow researchers to screen their interested genomes graphically 
only by pasting the relevant web links to the interface. After pasting the 
links, this kit will download the xml or fasta from the original web 
automatically. </SPAN></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN id="pp3" lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
Biobrick</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
The updated Biobrick section will focus on three parts to be optimized. First, 
we will specifically classify the known biobricks. As lots of accomplishments by 
using biobricks have come out, some bricks have been tested to be adjustable to 
the specific kinds of researches. For example, a kind of biobricks are always 
used to implement the cell signals transportation and another kind are always 
applied to the cell colony communication etc. This
<B style="mso-bidi-font-weight:
normal"><I style="mso-bidi-font-style:normal">research-oriented</I></B> 
classification facilitate researchers get the useful information they need 
conveniently and efficiently. Second, we will refine the codes to make this kit 
updated immediately according to the official website. Third, we will try to 
improve the ways of restoring the data by documenting more graph and credit 
identification information for each biobricks. </SPAN></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;"><O:P> </O:P></SPAN></B></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN id="pp4" lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
Regulator Designer</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;
mso-bidi-font-weight:bold">Apart from Riboswitch and SiRNA, some other kinds of 
regulator will also be considered in the next version of the kit. One of the 
most common used in synthetic biology is transcriptional factors (TFs). We will 
develop a TFs matcher for researchers to find their desired TFs according to the 
experimental requirements. The selection process is executed based on the 
existent TFs database and previous common used TFs in synthetic biology. And our 
ultimate goal is to add the regulator designer, including riboswitch, siRNA, TFs 
and others to the </SPAN><B style="mso-bidi-font-weight:normal"><I><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
Clotho </SPAN></I></B><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-weight:
bold">frame for sharing among all the researchers. </SPAN></P><P class="MsoNormal"><B style="mso-bidi-font-weight:normal"><SPAN id="pp5" lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;">
Simulator</SPAN></B></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;
mso-bidi-font-weight:bold">The advanced version of Simulator will be improved 
from three prospects. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-fareast-font-family:
Georgia;mso-bidi-font-family:Georgia"><SPAN style="mso-list:Ignore">(1) </SPAN></SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-weight:
bold">One focuses on ameliorating the Algorithms of the original one. </SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:
&quot;Lucida Sans Unicode&quot;">The next version of Simulator, other methods 
for simulating will combined into this part according to the completeness of the 
information necessary for certain kind of analysis provided by the SBML files. 
In the first version, we use the </SPAN><SPAN lang="EN" style="font-size:14.0pt;
font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:&quot;Lucida Sans Unicode&quot;;
color:black;mso-ansi-language:EN"><A title="Constraint-based approach (page does not exist)" href="http://en.wikipedia.org/w/index.php?title=Constraint-based_approach&amp;action=edit&amp;redlink=1"><SPAN style="color:
black;text-decoration:none;text-underline:none">constraint-based </SPAN></A>FBA 
but in the next version algorithm with both constraint and kinetic approaches</SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:
&quot;Lucida Sans Unicode&quot;;color:black"> will</SPAN><SPAN lang="EN" style="font-size:
14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:&quot;Lucida Sans Unicode&quot;;
color:black;mso-ansi-language:EN"> be applied</SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:
&quot;Lucida Sans Unicode&quot;;color:black"> to enhance the reliability and 
sensitivity of simulation. </SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:
&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:&quot;Lucida Sans Unicode&quot;">
For program code To realize FBA analysis, the original version is realized based 
on Package ‘abcdeFBA’ of R. While in the second version of this part, 
programming will be supported directly by JAVA or C++, which will not only 
reduce the size of the software but also accelerate computation. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-fareast-font-family:
Georgia;mso-bidi-font-family:Georgia"><SPAN style="mso-list:Ignore">(2) </SPAN></SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:
&quot;Lucida Sans Unicode&quot;">One focuses on improving the user experiencing. 
Firstly, for model used in simulation, users will not only use SBML files 
documented in released database like Biomodels but also can edit and save the 
SBML in a user-friendly way. Information necessary for FBA including reactions 
and metabolites will be showed clearly and in the interface where user can 
revise model . The new model can be saved in a revised SBML file. Secondly, the 
metabolic network can be visualized in the interface. Users can drag the 
pathways joint to a right place in the screen like playing the building block 
game, which can add to the fun of this simulation.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-fareast-font-family:
Georgia;mso-bidi-font-family:Georgia"><SPAN style="mso-list:Ignore">(3) </SPAN></SPAN><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:
&quot;Lucida Sans Unicode&quot;">Inspired by the recent whole-cell model in 
predicting phenotype and giving insights in some undiscovered cellular 
behaviors, we plan to expand our models and simulation not only concentrating on 
metabolome information but also should involve genome annotation, transcriptome 
, proteome and molecular interaction into consideration. As cell processes have 
been divided into 28 submodels and synthetic biology have developed many typical 
artificial systems recent years, we hope to give out a new standard for 
documenting these cellular process and relevant bioinformation mentioned above 
in one file. Thus, researchers just need to edit such kinds of files, such as 
adding metabolites or deleting one interaction between two molecular, to 
simulate their designed systems before experiments. And the mentioned 
comprehensive bio-information can be applied to offset the difference between 
prediction and experiments, which often motivate the discovery of unobserved 
cellular process.</SPAN></P><P class="MsoNormal"><B><SPAN id="pp6" lang="EN-US" style="font-size:14.0pt;font-family:
&quot;Georgia&quot;,&quot;serif&quot;;mso-bidi-font-family:&quot;Lucida Sans Unicode&quot;">
Trans-platform Realization </SPAN></B></P><SPAN lang="EN-US" style="font-size:14.0pt;font-family:&quot;Georgia&quot;,&quot;serif&quot;;
mso-fareast-font-family:宋体;mso-bidi-font-family:&quot;Lucida Sans Unicode&quot;;
mso-font-kerning:1.0pt;mso-ansi-language:EN-US;mso-fareast-language:ZH-CN;
mso-bidi-language:AR-SA">As the first version of BiArkit is written by JAVA 
completely, it facilitate us to make the whole software transferable. BiArkit 
2.0 will be available on both of Mac and Linux and researchers can use it 
conveniently in terms of their own equipments.<SPAN style="mso-spacerun:yes">  </SPAN></SPAN></DIV></DIV></DIV><DIV id="sidebar"><DIV id="doc-nav"><DIV><H5>IGEM-SYSU-Software </H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software">Main Page</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Team">Album</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/SecondEdition">Second Edition</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Accomplishment">Accomplishment</A></LI></UL></DIV></DIV><DIV><H5>User Guide</H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Download">
						Download</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Tutorial">Tutorial</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Video">Video</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Email</A></LI></UL></DIV></DIV><DIV><H5>FAQ </H5><DIV><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Faq">Questions</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Trouble">Troubleshooting</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Uninstalling">Uninstalling</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Contact Us</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><DIV class="wrapper clear"><DIV id="footer-about" class="footer-column"><H2>   About</H2><UL><LI><A href="https://2012.igem.org/Team:SYSU-Software/Team">Developers</A></LI><LI><A href="https://2012.igem.org/Team:SYSU-Software/Contact">Contact Us</A></LI></UL></DIV><DIV id="footer-flickr" class="footer-column"><H2>   Links</H2><UL><LI><A href="http://www.sysu.edu.cn">SUN YAT-SEN University</A></LI><LI><A href="http://lifescience.sysu.edu.cn/main/default/index.aspx">SYSU LifeScience</A></LI></UL></DIV><DIV id="footer-search" class="footer-column"><H2>    Facebook</H2></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SYSU-Software/SecondEdition">http://2012.igem.org/Team:SYSU-Software/SecondEdition</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-Software/SecondEdition" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-Software/SecondEdition" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-Software/SecondEdition&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-Software/SecondEdition&amp;oldid=234131" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>