<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid content-main"><DIV class="row"><NAV class="col-md-3 bs-docs-sidebar navFont"><UL id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330"><LI><A href="#1">Introduction</A></LI><LI><A href="#2">Cloning</A></LI><LI><A href="#Standard">Standard</A></LI><LI><A href="#Chelators">Chelators</A></LI><LI><A href="#Other">Other Parts</A></LI></UL><LI><A href="#3">Promoter Characterisation</A></LI><LI><A href="#sub1">Experimental design</A></LI><LI><A href="#sub2">Results</A></LI><LI><A href="#subA">pCopA sfGFP</A></LI><LI><A href="#subB">pCopA sfGFP- divergent. CueR</A></LI><LI><A href="#subC">pCopA CueR sfGFP</A></LI><LI><A href="#subD">pCusC mKate</A></LI><LI><A href="#subE">pCusC CusR mKate</A></LI><LI><A href="#4">Copper Chelation</A></LI><LI><A href="#BCS">BCS Absorbance Assay</A></LI><LI><A href="#vivo">Assay with live cells</A></LI><LI><A href="#purification">Protein Purification</A></LI><LI><A href="#vitro">Assay with purified proteins</A></LI><LI><A href="#FLIM">FLIM</A></LI><LI><A href="#5">Alginate Bead Preparation</A></LI><LI><A href="#alginate">Alginate only</A></LI><LI><A href="#Chitosan">with Chitosan</A></LI><LI><A href="#6">Conclusions</A></LI><LI><A href="#PromoterSensitivity">Promoter Sensitivity</A></LI><LI><A href="#ChelatorCharacterisation">Chelator Characterisation</A></LI><LI><A href="#Delivery">Delivery</A></LI><LI><A href="#Ref">References</A></LI></NAV><DIV class="col-md-9 content-right" style="background-color: #fff;"><DIV class="pageTitle pageTitleBlue">Overview</DIV><SECTION id="1"><H2>Introduction</H2><P>
 In order to provide a long-term treatment for Wilson's disease we realised we needed to design components of a bacterial system that could detect copper and produce a copper chelators to prevent its absorption by the body.  These functions were tested separately to try to understand how to design a system able to operate on physiological copper concentrations.
</P><P>
Copper detection was investigated using promoter systems based upon the native <I>E. coli</I> copper regulator <A href="https://2016.igem.org/Team:Oxford/Parts#CueR">CueR</A> and the <A href="https://2016.igem.org/Team:Oxford/Parts#CusS-CusR">CusS/CusR</A> two component system. The two copper chelators we tested were <A href="https://2016.igem.org/Team:Oxford/Parts#Csp1H">Copper storage protein 1</A> (Csp1) from <I>Methylosinus trichosporium</I> OB3b and <A href="https://2016.igem.org/Team:Oxford/Parts#MymTH">mycobacterial metallothionein</A> (MymT) from <I>Mycobacterium tuberculosis.</I></P><P>
Throughout all of our experiments we tried to think about whether the conditions the bacteria were growing in were likely to be realistic of the gut environment, for instance in regards to pH and temperatures. 
</P><P>
 We spoke to patients with Wilson’s disease and realised that, whilst many were comfortable with the idea of a synthetic biology treatment, the administration method for the treatment was one of their priorities. Many complained that their current treatments involved pills that were too large, had to be taken too frequently and had to be kept refrigerated, making travelling difficult. From their feedback we investigated small alginate beads, layered with chitosan to allow survival in the stomach, as a method of probiotic delivery.
</P></SECTION><SECTION id="2"><H1 class="pageTitle pageTitleBlue">Cloning</H1><SECTION id="Standard"><H2>Standard Cloning</H2><P>
 All of our original part sequences were synthesised by IDT between the <A data-toggle="popover1" data-trigger="hover" title="Biobrick prefixes" data-content="GAATTCGCGGCCGCTTCTAGAG or GAATTCGCGGCCGCTTCTAG if the part begins with ATG" placement:="">appropriate Biobrick prefix</A> containing <A data-toggle="popover1" data-trigger="hover" title="EcoRI" data-content="G/AATTC" placement:="">EcoRI</A>/<A data-toggle="popover1" data-trigger="hover" title="XbaI" data-content="T/CTAGA" placement:="">XbaI</A> restriction sites and the universal biobrick suffix sequence containing <A data-toggle="popover1" data-trigger="hover" title="SpeI" data-content="A/CTAGT" placement:="">SpeI</A>/<A data-toggle="popover1" data-trigger="hover" title="PstI" data-content="CTGCA/G" placement:="">PstI</A> restriction sites. Short pre-preffix and post-suffix sequences were included for the binding of our amplification primers. All our chelators were synthesized with a C terminal sequence encoding a <A data-toggle="popover1" data-trigger="hover" placement:=""> hexa-histidine tag</A>. 
</P><P>
 The sequences were amplified using PCR and ligated into the <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> BioBrick shipping vector backbone before being cloned into <I>E. coli </I> DH5α strain. After growth on a plate containing <A data-toggle="popover1" data-trigger="hover" title="Chloramphenicol" data-content="An antibiotic that operates by blocking peptide chain elongation. Resistance is provided by the chloramphenicol acetyltransferase encoded by the shipping plasmid" placement:="">chloramphenicol</A>, colonies were then picked and grown overnight in 5ml of LB and plasmid DNA extracted via miniprep. A small volume of each these samples were then digested with <A data-toggle="popover1" data-trigger="hover" title="EcoRI" data-content="G/AATTC" placement:="">EcoRI</A> and <A data-toggle="popover1" data-trigger="hover" title="PstI" data-content="CTGCA/G" placement:="">PstI</A> and then run on an agarose gel. If these digests had the correct band sizes then a small volume of the undigested plasmid was sent off for sequencing. 
</P><P>
 Correctly sequenced parts were then kept for testing and deposition in the registry. All our promoter parts were tested in the shipping vector after being cloned into the <I> E. coli</I> K-12 MG1655 strain.
 </P></SECTION><SECTION id="Chelators"><H2>Chelators</H2><P>
 Our copper chelator parts (Copper storage protein 1 (Csp1), Csp1sfGFP, mycobacterial metallothionein (MymT) and MymTsfGFP) lacked a promoter so had to be ligated into the <A data-toggle="popover1" data-trigger="hover" title="Arabinose" data-content="a five-carbon sugar" placement:="">arabinose</A>-inducible pBAD His B commercial expression plasmid for characterisation. This was done by adding a <A data-toggle="popover1" data-trigger="hover" title="BspHI" data-content="T/CATGA" placement:="">BspHI</A> restriction site (an <A data-toggle="popover1" data-trigger="hover" title="Isocaudamer" data-content="Restriction enzymes with different recognition sequences but compatible sticky ends e.g. SpeI and XbaI" placement:="">isocaudamer</A> of <A data-toggle="popover1" data-trigger="hover" title="NcoI" data-content="C/CATGG" placement:="">NcoI</A>) into the part at the initiator methionine using PCR. The pBAD vector was digested with <A data-toggle="popover1" data-trigger="hover" title="NcoI" data-content="C/CATGG" placement:="">NcoI</A> and <A data-toggle="popover1" data-trigger="hover" title="PstI" data-content="CTGCA/G" placement:="">PstI</A> and the insert with <A data-toggle="popover1" data-trigger="hover" title="BspHI" data-content="T/CATGA" placement:="">BspHI</A> and <A data-toggle="popover1" data-trigger="hover" title="PstI" data-content="CTGCA/G" placement:="">PstI</A>. Once ligated these were then transformed into <I>E. coli </I> DH5α strain, extracted sequenced and correct parts were then transformed into <I> E. coli</I> K-12 MG1655. The pBAD expression vector contains a copy of β-lactamase gene allowing cells with the plasmid to have <A data-toggle="popover1" data-trigger="hover" title="Ampicillin" data-content="An β-lactam antibiotic that irreversibly inhibits the transpeptidase enzyme used to make bacterial cell walls" placement:="">ampicillin</A> resistance. Consequently ampicillin was used for all transformations using these parts.</P></SECTION><SECTION id="Other"><H2>Other Parts</H2><P>
One of our promoter parts (pCopA sfGFP with plasmid divergent CueR) was sequenced four times from separate PCR and transformation attempts and had the same two point mutations on each attempt. We attributed this to a synthesis issue and this part was abandoned.  However we wanted to get a similar sequence with the same design as this part so we amplified the promoter system and divergent CueR only from our part: pCopA MymT sfGFP with divergent CueR and then ligated the resulting sequence with prefix and suffix into the pSB1C3 shipping vector.
</P><P>
We also received from Tom Folliard, a PhD student in our lab, a plasmid containing a copper biosensor (pCusC mKate) he had done a small amount of work on previously. We characterised this part further and attempted to make it biobrick compatible. As the part had a Biobrick illegal <A data-toggle="popover1" data-trigger="hover" title="SpeI" data-content="A/CTAGT" placement:="">SpeI</A> restriction site in the ribosome-binding site we amplified the promoter region only so we could deposit this in the registry. We also wanted to deposit the entire testing component complete with promoter, ribosome binding site and RFP. After attempts at site directed mutagenesis failed, we ordered this part without this site from IDT.
</P></SECTION></SECTION><SECTION id="3"><H1 class="pageTitle pageTitleBlue">Promoter Characterisation</H1><SECTION id="sub1"><H2>Experimental Design</H2><P>
We decided that our copper chelating bacteria would require a promoter system able to detect dietary copper rather than just constitutive expression. This has three main advantages:
</P><LI class="textLiFont">
If the bacteria are made to produce the chelator constantly, even when it is not required, then this will increase use of cell resources such as amino acids. This increases selection pressure for the gene to mutate out and decreases the likelihood of our bacteria strain being able to successfully compete with other gut bacteria.
</LI><LI class="textLiFont"> If the copper chelator is expressed at all times then the rate of copper chelation is essentially uncontrolled and only a function of the number of bacteria present which could could feasibly vary slightly due to short-term factors such as food intake or illness. This unpredictable behaviour would be difficult to reconcile with other treatments.
</LI><LI class="textLiFont">
Constitutive chelator expression that is too low for a particular patient at a particular time will not treat their symptoms effectively. If the chelator expression is too high however they may experience copper deficiency. </LI><P>
We found two different copper sensitive promoter systems: CueR-linked and CusS/CusR-linked (see our <A href="https://2016.igem.org/Team:Oxford/Parts#Promoters">parts page</A>) and investigated rearranging the components to form feedback systems in order to improve sensitivity over the physiological range of copper concentrations. 
</P><P>
 We aimed to characterise all of our promoters using three different methods:
</P><LI class="textLiFont">
Plate reader
</LI><LI class="textLiFont">
Flow cytometry
</LI><LI class="textLiFont">
Microscopy
</LI><P>
The plate reader experiments were preformed to study how the expression of the fluorescent protein changes with copper concentration and changes over time. This data was used to parameter fit the models of each promoter system. 
</P><P>
Flow cytometry was performed to confirm the results of the plate reader and to provide a measure of the variance in expression within each population at different copper concentrations.
</P><P>
Microscopy was performed to qualitatively confirm the results of the previous two methods and to study the cellular localisation of the fluorescent proteins.
</P><P>
Further details of each of these can be found on our <A href="https://2016.igem.org/Team:Oxford/Experiments">experiments page</A> and full protocols on our <A href="https://2016.igem.org/Team:Oxford/Protocols">protocols page</A>.
</P></SECTION><SECTION id="sub2"><H2>Results</H2><P>
For all of our promoter systems the results of all three experimental methods were broadly consistent. 
</P><P>
We investigated three designs of a promoter system based upon the CueR-linked system:
</P><P>
 One (pCopA sfGFP) was the simplest but only responsive to copper at high concentrations. This promoter alone was therefore not considered suitable for our system.
</P></SECTION><SECTION id="subA"><H3>pCopA sfGFP</H3><FIGCAPTION>Plate reader data after four hours (top left), flow cytometry data (top right), microscopy images for differential interference contrast, fluorescence and composite channels (bottom)</FIGCAPTION></SECTION><SECTION id="subB"><P>
To investigate whether this insensitivity was due to there being only a single copy of the <I>cueR</I> gene expressing from the cell genome compared with the 500+ copies of pCopA on the high copy plasmid we wanted to test a version of the part with plasmid expressed CueR. We based this upon a part in the registry already (see our <A href="https://2016.igem.org/Team:Oxford/Parts#Promoters">parts page</A>) but never managed to clone it successfully possibly due to a synthesis issue. However we did produce two parts (BBa_K1980010 and BBa_K1980012) with our chelators in front of the sfGFP. These are expected to operate much like the part without the chelator if we consider the amount of copper bound by the chelator to be negligible. These both appeared more copper responsive than the pCopA promoter alone although the much larger TAT Csp1 appeared to have expression issues at higher copper concentrations forming <A data-toggle="popover1" data-trigger="hover" title="Inclusion bodies" data-content="Cytoplasmic aggregates of inactive, misfolded proteins" placement:="">inclusion bodies</A> seen in the microscopy images: </P><H3>pCopA MymT sfGFP with divergent CueR</H3><FIGCAPTION>Plate reader data after four hours (top left), flow cytometry data (top right), microscopy images for differential interference contrast, fluorescence and composite channels (bottom)</FIGCAPTION><H3>pCopA TAT Csp1 sfGFP with divergent CueR</H3><FIGCAPTION>Plate reader data after four hours (top left), flow cytometry data (top right), microscopy images for differential interference contrast, fluorescence and composite channels (bottom)</FIGCAPTION></SECTION><SECTION id="subC"><P>We wanted to experiment with the CueR-system further by adding in a feedback loop. CueR has an <A href="https://2016.igem.org/Team:Oxford/CueR_MOI">mechanism of action</A> meaning that it can operate both as a net activator or a net repressor depending on the copper concentration through interactions with <A data-toggle="popover1" data-trigger="hover" title="RNA polymerase" data-content="An enzyme that transcribes the DNA sequence of a gene into an mRNA sequence that is then translated into a protein sequence.">RNA polymerase</A>. We designed a part where CueR is expressed from pCopA from a plasmid in front of sfGFP hoping that this would act as a positive feedback system increasing the sensitivity of the part. Instead it appears to act as negative feedback system dampening the response to copper compared to the part with constitutive expression.  This also seems to have reduced the variation between cells as shown by the flow cytometer data. Under some circumstances this could be a useful feature being able to keep the chelator concentration more similar and predictable over a wider concentration range but this part and behaviour was not deemed to be useful over the physiological copper concentrations we were interested in.</P><H3>pCopA CueR sfGFP/ Feedback pCopA sfGFP</H3><FIGCAPTION>Plate reader data after four hours (top left), flow cytometry data (top right), microscopy images for differential interference contrast, fluorescence and composite channels (bottom)</FIGCAPTION></SECTION><SECTION id="subD"><P>We also received a part containing a copper biosensor: pCusC mKate, from Tom Folliard, a PhD student in our lab. Whilst he had some preliminary late reader data we characterised it further. While not directly comparable to our other parts due to the different fluorescent protein this promoter turned out to be much more sensitive and responsive than any of the CueR systems:  </P><H3>pCusC mKate</H3><FIGCAPTION>Plate reader data after four hours (top left), flow cytometry data (top right), microscopy images for differential interference contrast, fluorescence and composite channels (bottom)</FIGCAPTION></SECTION><SECTION id="subE"><P>We attempted to make a positive feedback system based upon pCusC. However after many attempts we only managed to obtain a version with a point mutation (Val to Ala) in the CusR DNA-binding domain. We tested this part to see if the mutation was tolerable but found no evidence of a more sensitive system.
</P><H3>pCusC CusR mKate</H3><FIGCAPTION>Plate reader data after four hours (left), flow cytometry data (right)</FIGCAPTION></SECTION><SECTION id="4"><H2 style="margin-top: 0px; padding-top: -120px;">Copper Chelation</H2><P>We attempted to show that our two copper chelating proteins chelated copper when expressed from<I> E. coli.</I> With limited time and budget we tried to develop a absorbance assay for copper using the reagent BCS.<SUP>(1)(2)</SUP> We used this to assay to test both live cells and purified proteins for the ability to reduce free copper concentration. Unfortunately the assay was either too insensitive to show an effect or our chelators were not operating as intended. With more time we hoped to develop the assay further.</P><P>We also came across a paper by  Hötzer et al<SUP>(3)</SUP> that detailed how it might be possible to use the fluorescence lifetime of a His tagged GFP as a copper assay. Lacking access to a <U>F</U>luorescence <U>L</U>ifetime <U>IM</U>agining (FLIM) microscope we contacted Cardiff iGEM who were kind enough to run a few of our samples in their <A href="“http://blogs.cardiff.ac.uk/bioimaging/flim/”">bioimaging unit</A>.</P><SECTION id="BCS"><H3>BCS Absorbance Assay</H3><FIGCAPTION>The range of BCS colours generated by Cu<SUP>+</SUP> (0 to 50μM)</FIGCAPTION><P>
We researched reagents that give a coloured solution with copper to form a spectrometric copper assay. These reagents detect copper in the 1+ oxidation state. When performing any experiments exposed to the air the Cu<SUP>+</SUP> is oxidised by oxygen to Cu<SUP>2+</SUP>. Consequently we needed a reducing agent to reduce Cu<SUP>2+</SUP> back to Cu<SUP>+</SUP>.
</P><P>
The two most useful papers recommended the reagent bathocuproine disulphonate (BCS) using the ascorbate as a reducing agent.<SUP>(1)(2)</SUP> When BCS forms a 2:1 complex with a Cu<SUP>+</SUP> ion it has an orange absorbance at 480nm.
</P><SECTION id="vivo"><H3>Assay <I>in vivo</I></H3><P>
Using ascorbate as the reducing agent we generated a standard curve for free copper for bacterial growth medium by growing up overnight cultures of <I>E. coli</I> MG1655 containing the pBAD His B  expression plasmid lacking an insert.
</P><FIGCAPTION>The copper standard curve for centrifuged pBAD cells grown in LB, assayed with BCS and ascorbate</FIGCAPTION><P>
Different copper concentrations were added to aliquots of the cells, the cultures allowed to incubate for 30 minutes and then the cultures centrifuged for three minutes at 13,000 rpm. The copper in the supernatant was assayed in a spectrometer using BCS and ascorbate. The assay was found to be linear only between 1 and 20µM. We repeated the same procedure with our MG1655 cells containing MymT, MymT-sfGFP, Csp1, Csp1-sfGFP in the pBAD expression system with 2mM arabinose but were unable to detect any difference between the cells and the pBAD standard curve. This was also the case for the parts pCopA MymT with plasmid constitutive CueR, pCopA MymT-sfGFP with plasmid constitutive CueR and pCopA Csp1-sfGFP with plasmid constitutive CueR.</P><P>
Mathematical modeling by our dry lab suggested that with cells producing a reasonable amount of chelator (10<SUP>5</SUP>/cell) at a normal cell density (10<SUP>8</SUP>/ml) then even assuming maximum chelator binding then the drop in copper concentration using this protocol would be below the detection limit of the assay. Modeling suggested that if we could increase the chelator concentration, by purifying the protein, then we should be able to get a measurable drop in copper concentration. </P></SECTION><SECTION id="purification"><H3>Protein Purification</H3><P> In order to test the protein <I>in vitro</I> we used affinity column chromatography to purify it from the cell lysate. Initially, small bacterial cell cultures of <I>E. coli</I> MG1655, containing our parts MymT-sfGFP and Csp1-sfGFP, (10 ml) were grown overnight with arabinose at a concentration that allowed maximal induction of the chelators' expression (1.5mM). The next day, 1ml of the bacterial cell culture was transferred into fresh LB in a larger container (1L) and was grown overnight, again, at the same conditions (liquid LB low salt medium and 1.5mM arabinose). The resulting bacterial solution was treated with multiple rounds of centrifugation and sonication, to break up the cells and release the protein. The protein was designed with a hexahistidine sequence at its C-terminus so Ni-NTA Agarose affinity chromatography was used to purify the protein from the cell lysate. The eluted protein was collected in 1.5ml Eppendorf tubes. <A data-toggle="popover1" data-trigger="hover" title="SDS-PAGE" data-content="A technique that uses the detergent Sodium Dodecyl Sulphate to denature proteins allowing them to run in PolyAcrylamide Gel Electrophoresis as a function of their mass" placement:="">SDS-PAGE</A> gels, a gel for fluorescence visualisation and a denaturation gel, were run containing the two chelators at different dilutions with a GFP standard as well. For the fluorescence gel, whilst protein structure is mostly denatured, the fluorophore of GFP remains intact. For the denaturation gel the whole protein was denatured. The gel was then stained with Coomassie Blue to visualise the protein bands. This was done to compare expression levels of each chelator (which is proportional to the fluorescence emitted since the constructs are linked to GFP). The two gels are shown below:</P><DIV class="div50"><FIGCAPTION>SDS-PAGE Denaturation gel.</FIGCAPTION></DIV><DIV class="div50"><FIGCAPTION>SDS-PAGE fluorescence gel.</FIGCAPTION></DIV><P>Both gels show that expression levels of MG (MymT-sfGFP) are higher than those of CG (Csp1-GFP). The remaining protein was stored at -20°C for further experimentation. </P><P> Despite the fact that it was visible from the protein solution (how green the solution was under light) and the SDS gels that MG was expressed at higher levels than CG, we wanted to give the amount a quantitative value. We used ImageJ to measure the pixel density (learn more about the process in the <A href="https://2016.igem.org/Team:Oxford/Protocols#Densitometry">Protocols</A> page) of the GFP standards, draw a curve and map the pixel density values of the chelator lanes. The picture and table below show how the picture was processed and the results showing the amount of protein in each lane.  

</P><DIV class="div50"><FIGCAPTION>The fluorescence gel with the lanes highlighted in yellow boxes for processing.</FIGCAPTION></DIV><DIV class="div50"><FIGCAPTION>Table showing the results for each lane. The area value refers to the area under the curve created as pixel density changes when you move from the top to the bottom of each yellow rectangle. The GFP masses were known and were used to relate the &quot;Area&quot; value from ImageJ to protein amount in the band.</FIGCAPTION></DIV></SECTION><SECTION id="vitro"><H3 style="clear: both;">Assay <I>in vitro</I></H3><P>
We attempted to perform a similar assay to the live cells using the purified extract using a sample of a his-tagged GFP as a control. Protein concentration of each of the samples was measured using the absorbance at 280nm and predicted extinction coefficients from <A href="http://web.expasy.org/protparam/">ProtParam</A>. 
After numerous attempts to show a measurable drop in copper concentration failed, we investigated alternative reducing agents, eventually concluding that L-Glutathione was better than ascorbate and Dithiothreitol because it was efficient at reducing Cu<SUP>2+</SUP> but mild enough to avoid damaging the proteins. </P><P>
After further attempts however we could not convincingly show copper chelation by our <I>in vitro</I> protein samples.</P><P>The chelators we used for purification had a C-terminal sfGFP-tagged connected via a short, hydrophilic linker to aid folding and a His tag was attached to the sfGFP. This means that if the linker was cleaved by proteases during purification (despite the addition of protease inhibitors) then we would purify sfGFP without the chelator attached. Then the actually chelator concentration would have been much lower than our measured protein concentration. This may be the cause of multiple bands seen on the SDS-PAGE gel corresponding to different oligomerisation states as well as degradation products. Using a C-terminal his tag was necessary with Csp1 so as to not disturb the N-terminal  
TAT sequence but MymT-sfGFP could have had an N-terminal His-tag so that we definitely purified the correct protein. We would design and test a part like this if we had more time.</P></SECTION></SECTION><SECTION id="FLIM"><H2>Fluorescence Lifetime Imaging</H2><P>
We discovered an innovative paper by Hotzer et al<SUP>(3)</SUP> that described how His-tagged GFP can be quenched by a copper ion binding to this His tag leading to a reduction in the fluorescence lifetime (the time the fluorophore spends in the excited state before returning to the ground state by emitting a photon). As the mean fluorescence lifetime, rather the sample fluorescence, is measured this should be independent of the level of GFP expression. They speculated that this could potentially be used as a <I>in vivo</I> copper assay.
</P><FIGCAPTION>A representation of the suggested quenching mechanism of the GFP fluorohore via His-tag bound copper ion</FIGCAPTION><P>
As we had His-tagged our chelator-sfGFP constructs we were curious to see if this technique could be applied to our parts to measure copper chelation <I>in vivo</I> by our parts. We believe that two possibilities were likely:
</P><OL><LI>Copper chelation by the chelator reduces the free copper concentration inside the cell meaning that less binds to the His tag and the fluorescence lifetime will be greater than a His-tagged sfGFP control</LI><LI>Copper chelation by the chelator would allow additional quenching if copper was bound within the quenching radius of the fluorophore leading to a reduction in fluorescence lifetime compare with a sfGFP control</LI></OL><P>
Lacking access to a fluorescence lifetime microscope ourselves we contacted <A href="https://2016.igem.org/Team:Cardiff_Wales">Cardiff iGEM</A> who had a FLIM machine in their <A href="“http://blogs.cardiff.ac.uk/bioimaging/flim/”">bioimaging unit</A>. They very kindly agreed to run a few samples for us taking up over five hours of microscope time. 
</P><P>
We sent Cardiff iGEM our parts Csp1-sfGFP, MymT-sfGFP (both in pBAD) and pCopA CueR sfGFP (as a control) in live MG1655 <I>E. coli</I> in agar tubes. Cardiff grew them overnight in 5ml of LB with 5μM copper with and without 2mM arabinose. 
</P><P>
The imaging unit spread each strain on slides and measured the fluorescence lifetime of three areas on each slide.</P><P> (Acquisition parameters: using the x63 water immersion objective with excitation at 483nm (71% intensity, pulse rate 40MHz) and emission via a BP500-550 filter. Scan resolution at 512 x 512 pixels at pixel size of 0.26 microns/pixel, 1AU pinhole.  Counts of &gt;1000 per lifetime recording.)
</P><P>FLIM images from one section of each slide:</P><FIGCAPTION>FLIM images captured from each of the three arabinose induced slides. The fluorescence lifetime is represented by colour.</FIGCAPTION><P>
As expected the pCopA CueR sfGFP control was fluorescent, with and without arabinose, with the mean fluorescence lifetime a consistent 2.6ns. </P><P>
When Csp1-sfGFP was induced the mean lifetime decreased to 2.5ns. This might be indicate that copper chelation has occurred or may be reflective of the expression problems of Csp1.</P><P>
When MymT-sfGFP was induced the mean lifetime decreased to 2.3ns. As MymT-sfGFP is was observed to be reliably expressed and because MymT is a small copper cluster separated form sfGFP by a small linker we believe that this represents additional quenching of the fluorphore by MymT-bound copper showing <I>in vivo</I> copper chelation.</P><FIGCAPTION>Fluorescence lifetime for sfGFP, Csp1-sfGFP and MymTsfGFP from iGEM Cardiff with three repeats. (No error bars shown when standard deviation=0.) </FIGCAPTION><H3>sfGFP:</H3><FIGCAPTION>Histogram of fluorescence lifetimes for sfGFP</FIGCAPTION><H3>Csp1-sfGFP:</H3><FIGCAPTION>Histogram of fluorescence lifetimes for Csp1sfGFP</FIGCAPTION><H3>MymT-sfGFP:</H3><FIGCAPTION>Histogram of fluorescence lifetimes for MymTsfGFP</FIGCAPTION><P>
From this preliminary data we believe that measuring GFP fluorescence lifetime could be used as a reliable method for measuring copper concentration <I>in vivo</I> and that MymT displays copper chelation activity in <I>E. coli</I>. If we could perform additional experiment we would measure the fluorescence lifetime of sfGFP with different overnight copper concentrations and use this to form a standard curve to which we could compare the chelator-sfGFP constructs.   
</P><P>Our most sincere gratitude to all those at <A href="“https://2016.igem.org/Team:Cardiff_Wales&quot;">Cardiff iGEM</A> in particular Rob Newman for setting up the collaboration, Geraint Parry for setting up the experiment and Anthony Hayes at the <A href="“http://blogs.cardiff.ac.uk/bioimaging/flim/”">bioimaging unit</A> measuring the our parts using the FLIM microscope.</P></SECTION></SECTION><SECTION id="5"><H1 class="pageTitle pageTitleBlue">Alginate Bead Preparation</H1><P>Based upon patient feedback into delivery systems we decided to investigate the feasibility of using <A data-toggle="popover1" data-trigger="hover" title="Alginate" data-content="An anionic polysaccharide produced by brown algae" placement:="">alginate</A> beads to encase our bacteria and successfully deliver them to the intestine, while maintaining their integrity in the stomach. As quantitative measure of bead stability we used the dye <A href="https://2016.igem.org/Team:Oxford/Chemicals#Crystal-Violet">crystal violet</A> which we could measure by it absorbance at 595nm.</P><SECTION id="alginate"><H2>Alginate only</H2><P>Our first experiment was to prepare a 2% (weight/volume) sodium alginate solution, which was done dissolving 2g of alginate in 100ml of warm MilliQ. We then added 2ml of the dye. This solution was then dropped with a pipette into a solution of 0.1mM CaCl<SUB>2</SUB> that solidifies and hardens the alginate. </P><P>
We found during their first 30 minutes in the CaCl<SUB>2</SUB> solution (the “stabilisation phase”) dye did leak from the beads. We then transferred the beads into a fresh CaCl<SUB>2</SUB> solution and found the beads no longer leaked dye. Hence, the beads had stabilised.
</P><P>The beads before hardening showing leakage of the dye:</P><FIGCAPTION>The beads with crystal violet dye before the stabilisation phase</FIGCAPTION><P>The beads after hardening in one calcium solution for 30mins then transferred to another for one hour with no signs of leakage:</P><FIGCAPTION>The beads with crystal violet dye after the stabilisation phase</FIGCAPTION><P> To investigate the stability of our beads within the body we decided to place the alginate-dye beads into stomach and intestine simulation solutions and measured the concentration of dye released into each solution using absorbance at 595nm.</P><LI>We simulated the conditions of the stomach by making up 0.2% (w/v) NaCl solution made up to pH 2.0 with 1M HCl. </LI><LI>We simulated the conditions of the intestine by making 0.68% (w/v) monobasic potassium phosphate solution, made up to pH 7.2 with 1M NaOH. </LI><P>The 2 solutions are placed into a incubator shaking at 37°C for 90 minutes. Every 10 minutes 100μl of solution is removed and placed into 1 well of a 96 well plate. Measure absorption of each well once 90 minutes is complete. </P><FIGCAPTION>The absorbance in the medium increases in both stomach and intestine simulated solutions due to breakdown of the beads and release of the dye</FIGCAPTION><P>The significant increase in fluorescence of the alginate-only beads in the stomach simulated solution (where we do not want our beads to disintegrate) is shown more clearly on the &quot;alginate-only stomach, pH 2.0&quot; graph:</P><FIGCAPTION>The absorbance in the medium increases in  stomach simulated solution showing that the beads are breaking down</FIGCAPTION><P>The results suggested alginate beads would not be able to carry out bacteria successfully into the intestine, as the stomach acid would disintegrate the beads and hence kill our bacteria. We then decided to re-design the beads.</P></SECTION><SECTION id="Chitosan"><H2>With Chitosan</H2><FIGCAPTION>The layered design of our beads</FIGCAPTION><P>Upon analysis of our results we decided to improve the stability of the beads using repeating layers of <A data-toggle="popover1" data-trigger="hover" title="Chitosan" data-content="A linear polymer of glucosamine and N-acetyl-glucosamine sugar units made by alkali digestion of shrimp shells" placement:="">chitosan</A> and alginate. This was suggested by our research into probiotic delivery. The layering was done by preparing the beads as before then after their 30 minutes hardening, filtering them and alternatively dipping into 0.4% (w/v) chitosan solution (in 0.1M acetic acid adjusted to pH 6.0 with 1M NaOH, 10ml) for 10 mins and 0.04% (w/v) alginate solution (10ml) for 10 min.</P><P>10ml of these beads were placed in flasks of 100ml stomach simulate and intestinal simulate solutions and incubated in a 37°C shaker for 90 minutes. Every 10 minutes, 100μl samples were taken and after the whole 90 minutes the absorbance of each of the samples was measure at 595nm with a plate reader. </P><FIGCAPTION>The absorbance increases in intestine but not stomach simulated conditions.</FIGCAPTION><P>We found that our the absorbance of the stomach solution did not increase over time indicating that layering the alginate with chitosan allowed the beads to maintain their structural integrity and retain the dye. Conversely, in the small intestine, the more alkaline pH causes the layered coating to degrade, steadily releasing dye into solution. </P><P>This suggests that this method would be a good way of delivering our bacteria to the small intestine, as our probiotic would be released along the small intestine.</P><P>If we had more time we would test this system again by encapsulating bacteria that constitutively express GFP into the original alginate matrix and measure the fluorescence at different time points. At the end of the experiment, test the viability of the bacteria by taking an aliquot of the small intestine solution and adding it to LB broth and leaving to incubate overnight. Bacterial growth would show the bacteria had survived the solution.</P></SECTION></SECTION><SECTION id="6"><H1 class="pageTitle pageTitleBlue">Conclusions</H1><P>Our experiments have focussed on three areas: promoter sensitivity, chelator characterisation, and delivery method. </P><SECTION id="PromoterSensitivity"><H2> Promoter Sensitivity </H2><P>
 This research encompasses the bulk of our experimentation. We have characterised all of our promoters of the basis of either GFP or RFP fluorescence using plate readers, flow cytometry and microscopy. With more time, we would like to experiment with a greater number of promoter systems including our feedback pCusC CusR RFP free from point mutations, to see whether an increased sensitivity to lower copper concentrations can be achieved. </P><SECTION id="ChelatorCharacterisation"><H2> Chelator Characterisation </H2><P>
We have attempted a number of copper assays using our proteins. Unfortunately, due to purification problems, we have been unable to quantitatively determine the degree of chelation of either of our proteins. In the case of Csp1, we believed that this is due to the formation of intracellular inclusion bodies. However, the Cardiff iGEM team managed to collect some promising FLIM data for us, which indicates that MymT may successfully chelate copper <I>in vivo</I>. With more time, we would like to properly purify and characterise our proteins quantitatively. </P><SECTION id="Delivery"><H2> Delivery </H2><P>
We have collected promising initial data regarding our beads; demonstrating that they are capable of maintaining structural integrity in pH 2.0 (simulating the stomach) and dissociating to release coloured dye in pH 7.2 (simulating the small intestine). Our next step would be to test this using bacteria constitutively expressing GFP to see whether they were able to survive. At this point, we would begin to think about testing our entire system. </P></SECTION><SECTION id="Ref"><H1 class="pageTitle pageTitleBlue">References</H1><H3>For copper assay:</H3><DIV class="referenceSize"><UL><LI>
(1) Poillon W. and Dawson C. (1963)&quot;ON THE NATURE OF COPPER IN ASCORBATE OXIDASE I. THE VALENCE STATE OF COPPER IN THE DENATURED AND NATIVE ENZYME&quot; 
Biochim Biophys Acta. 1963 Sep 3;77:27-36
</LI><LI>
(2) Rapisarda V., Volentini S., Ricardo N., Massa E. (2002) &quot;Quenching of bathocuproine disulfonate fluorescence by Cu(I) as a basis for copper quantification&quot; Analytical Biochemistry, 307(1)m pp. 105-109. doi: 10.1016/S0003-2697(02)00031-3
</LI><LI>
(3) Hötzer B., Ivanov R., Altmeier S., Kappl R., Jung G., (2011) &quot;Determination of copper(II) ion concentration by lifetime measurements of green fluorescent protein.&quot;  Journal of Fluorescence, 21(6), pp. 2143-2153. doi: 10.1007/s10895-011-0916-1
</LI></UL></DIV><H3>For alginate bead preparation:</H3><DIV class="referenceSize"><UL><LI>Cook, M.T., Tzortzis, G., Khutoryanskiy, V.V. and Charalampopoulos, D. (2013) ‘Layer-by-layer coating of alginate matrices with chitosan–alginate for the improved survival and targeted delivery of probiotic bacteria after oral administration’, J. Mater. Chem. B, 1(1), pp. 52–60. doi: 10.1039/c2tb00126h.
</LI><LI>
Chávarri, M., Marañón, I., Ares, R., Ibáñez, F.C., Marzo, F. and Villarán, M. del C. (2010) ‘Microencapsulation of a probiotic and prebiotic in alginate-chitosan capsules improves survival in simulated gastro-intestinal conditions’, International Journal of Food Microbiology, 142(1-2), pp. 185–189. doi: 10.1016/j.ijfoodmicro.2010.06.022.
</LI><LI>
Krasaekoopt, W., Bhandari, B. and Deeth, H. (2004) ‘The influence of coating materials on some properties of alginate beads and survivability of microencapsulated probiotic bacteria’, International Dairy Journal, 14(8), pp. 737–743. doi: 10.1016/j.idairyj.2004.01.004.
</LI></UL></DIV></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><DIV class="container-fluid" id="footer" style="padding-top:25px; padding-bottom:30px; border-top:1px solid #CDCCCE; font-family: avenir, arial">

Made with gooood stuff.  </DIV></DIV></DIV></DIV></DIV></BODY></HTML>