<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid navbarFluid content-main"><DIV class="row"><NAV class="col-md-3 bs-docs-sidebar navFont"><UL id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330"><LI><A href="#Parts">Parts</A></LI><LI><A href="#chelators">Copper Chelators</A></LI><LI><A href="#Csp1H">Csp1</A></LI><LI><A href="#Csp1">Csp1</A></LI><LI><A href="#Csp1sfGFP">Csp1sfGFP</A></LI></UL><LI><A href="#MymTH">MymT</A></LI><LI><A href="#MymT">MymT</A></LI><LI><A href="#MymTsfGFP">MymTsfGFP</A></LI><LI><A href="#Promoters">Copper Promoters</A></LI><LI><A href="#CueR">CueR-linked systems</A></LI><LI><A href="#pcg">pCopA GFP</A></LI><LI><A href="#pg">pCopA GFP- divergent. CueR</A></LI><LI><A href="#p">pCopA - divergent. CueR</A></LI><LI><A href="#fcg">pCopA CueR GFP</A></LI><LI><A href="#CusS-CusR">CusS/CusR systems</A></LI><LI><A href="#pCusC">pCusC RFP</A></LI><LI><A href="#fck">pCusC CusR RFP</A></LI><LI><A href="#Comps">Composite Parts</A></LI><LI><A href="#pm">pCopA MymT- divergent CueR</A></LI><LI><A href="#pmg">pCopA MymTGFP- divergent CueR</A></LI><LI><A href="#ptcg">pCopA Csp1GFP- divergent CueR</A></LI><LI><A href="#references">References</A></LI></NAV><DIV class="col-md-9 content-right" style="background-color: #fff;"><DIV class="pageTitle pageTitleNavy" id="Parts">Parts</DIV><P>Our project required a system able to detect and then chelate dietary copper. We decided that we should produce parts to test these two functions separately. 
</P><P>
Having characterised each we could have a better understanding of a system whereby dietary copper was detected and a <A data-toggle="popover1" data-trigger="hover" placement:="">chelator</A> produced until the free copper concentration is reduced to a stable level. 
</P><P>
Our copper chelators and copper-sensitive <A data-toggle="popover1" data-trigger="hover" placement:="">promoters</A> belong in our list of <A href="https://2016.igem.org/Team:Oxford/Basic_Part">basic parts</A> whilst our promoter-chelators systems belong in our <A href="https://2016.igem.org/Team:Oxford/Composite_Part">composite parts</A> range. All can be found in our complete <A href="https://2016.igem.org/Team:Oxford/Part_Collection">parts collection</A>.
</P><SECTION id="chelators"><H1 class="pageTitleNavy">Copper Chelators</H1><P> In order to chelate copper we searched for copper binding and storage proteins. 
We decided that the ideal copper chelator would have these properties:
</P><UL class="textLiFont"><LI>Should be able to bind multiple copper ions per peptide to increase the efficient use of cell resources. </LI><LI>They should be from the prokaryotic domain because eukaryotic proteins can have expression issues in <I>Escherichia coli.</I></LI><LI>As <I> E. coli </I>naturally deals with copper toxicity by binding copper in the <A data-toggle="popover1" data-trigger="hover" title="Periplasm" data-content="A space between the outer cell membrane and bacterial cell wall in gram negative bacteria" placement:="">periplasm</A> then exporting it, periplasmic proteins may reduce toxicity to the host. </LI></UL><P>Using these criteria we found two copper chelators that we thought would be useful. We designed gBlocks, codon optimised for <I>E. coli</I>, containing these parts both alone and linked to super-folder <A data-toggle="popover1" data-trigger="hover" title="Green Fluorescent Protein" data-content="A protein that emits green light when illuminated with blue to UV light">GFP</A>. This form of the fluorescent protein was used because the standard GFP doesn’t fold particularly well in the periplasm where one of our chelators is intended to travel to. 
The proteins, both alone and attached to sfGFP, were ordered with C-terminal <A data-toggle="popover1" data-trigger="hover" placement:=""> hexa-histidine tag</A> so we could purify them. A concern was raised that the His tag would also weakly bind copper potentially affecting the results. However we decided that increasing the copper-binding would only improve the proteins' intended function.  
</P></SECTION><H2 id="Csp1H">Copper Storage Protein 1</H2><P>
Copper storage protein 1 is a protein discovered in a methane-oxidizing alphaproteobacterium called<I> Methylosinus trichosporium OB3b. </I> (OB3b here stands for “oddball” strain 3b). This bacterium has a high demand for copper for use in its particular <A data-toggle="popover1" data-trigger="hover" title="Methane Monoxygenase" data-content="An enzyme that oxides methane">methane monoxygenase</A> enzyme. Vita <I>et al.</I><SUP>(1)</SUP> discovered Csp1 in 2015, characterised the protein’s copper affinity and obtained crystal structures with and without copper.
</P><H3 id="Csp1">TAT Csp1</H3><P>
Csp1 is a tetramer of four-helix bundle subunits. Each monomer can bind up to 13 Cu(I) ions meaning that the tetramer binds a maximum of 52 copper ions. Vita <I>et al.</I> crystallised Csp1 with and without copper bound. The copper is bound inside the pre-folded helical bundles by Cys residues in contrast to metallothioneins, which are unstructured until they fold around metal ion clusters. Vita <I>et al.</I><SUP>(1)</SUP> found an average copper affinity of approximately 1x10<SUP>17</SUP>M<SUP>-1</SUP>.
</P><P>
Csp1 has a signal peptide targeting it to the <A data-toggle="popover1" data-trigger="hover" title="TAT" data-content="An incompletely understood prokaryotic secretion system"> twin arginine translocation pathway</A> (TAT).  This means that it is likely a periplasmic protein. However they also found cytoplasmic homologues in many species challenging their and our assumption that copper storage occurs only in the periplasm due to copper toxicity.
</P><P>
We codon optimised Csp1 to <I>E. coli</I> and replaced the original TAT sequence with a TAT sequence from the <I>E. coli</I> protein <A data-toggle="popover1" data-trigger="hover" title="CueO" data-content="An E. coli multicopper oxidase enzyme that protects against copper toxicity by oxidising Cu(I) to the less toxic Cu(II)"> CueO</A>, which is also involved in copper regulation. To get Csp1 from the shipping vector to the pBAD expression system for testing the TAT sequence had to be modified by the addition of a serine residue after the initiator amino acid of methionine. Serine was chosen over other amino acid possibilities because other TAT sequences seemed to have serine in this location. 
</P><FIGCAPTION>A Csp1 monomer with chelated copper from Vita <I>et al.</I><SUP>(1)</SUP></FIGCAPTION><P>You can view a animation of the full Csp1 tetramer <A href="https://static.igem.org/mediawiki/2016/9/91/Csp1_with_copper_gif_Oxford_2016_Sam.gif">here</A>.</P><H3 id="Csp1sfGFP">TAT Csp1 sfGFP</H3><P>
Csp1 sfGFP is a form of Csp1 with a C-terminal sfGFP tag as well as the modified TAT sequence. The sfGFP has a C-terminal hexahistidine tag for purification using Ni-NTA agarose affinity chromatography. We used this version of Csp1 for purification and microscopy studies. The microscopy images unfortunately showed no evidence of the protein reaching the periplasm. Instead they likely formed inclusion bodies.
</P><FIGCAPTION>Composite, Differential Interference Contrast and fluorescence channel images for Csp1sfGFP expressed form the pBad expression plasmid with 2mM arabinose</FIGCAPTION><H2 id="MymTH">Mycobacterial Metallothionien (MymT)</H2><P>
MymT is a small prokaryotic <A data-toggle="popover1" data-trigger="hover" title="Metallothein" data-content="Very small Cysteine-rich proteins involved in transport and storage">metallothein</A> discovered in <I>Mycobacterium tuberculosis</I> by Gold <I>et al.</I><SUP>(2)</SUP> It is believed that the protein may help the bacterium survive copper toxicity. No crystal structure of MymT was obtained but there is an NMR structure of a eukaryotic homologue<SUP>(3)</SUP>: 
</P><FIGCAPTION>NMR average of yeast Cu Metallothein (PDB: 1AQR)</FIGCAPTION><H3 id="MymT">MymT</H3><P>
MymT is much smaller than Csp1 and, because we did not add in a secretion tag, is intended to be cytoplasmic. It can bind up to 7 copper ions but has a preference for 4-6.<SUP>(2)</SUP></P><H3 id="MymTsfGFP">MymT sfGFP</H3><P>
This was a form of MymT with a C-terminal sfGFP. We used this version of MymT for purification and microscopy studies. Unlike Csp1sfGFP it had few problems with expression.
</P><SECTION id="Promoters" style="padding-top: 75px; margin-top:-75px;"><H1 id="promoters" class="pageTitleNavy">Copper sensitive Promoters</H1><P>
 For our copper chelating bacteria to successfully compete in the gut we investigated copper biosensors so that the chelator was only produced when copper was present, maintaining cell resources. 
</P><P>
As copper is a <A data-toggle="popover1" data-trigger="hover" title="Redox metal" data-content="Redox metals are ions with multiple stable oxidation states and can gain or lose electrons from other compounds. Copper has the oxidations states +1 and +2">redox metal</A> it is both very important in cells enzymes but also very toxic because it leads to the formation of reactive oxygen species. Consequently <I>Escherichia coli</I> has two systems of detecting excess copper: the CueR-linked system that detects cytoplasmic copper and the CusS/CusR two component system that detects periplasmic copper. 
We investigated both systems and by arranging the components to form a <A data-toggle="popover1" data-trigger="hover" title="Positive feedback" data-content="As system whereby a small increase in the concentration of a substance (here activator) leads then to a greater increase in the concentration of the same substance">positive feedback loop</A> hoped to improve their sensitivity over the range of copper concentrations we were interested in.
</P><H2 id="CueR">CueR linked systems</H2><FIGCAPTION>The <I>E. coli</I> copper regulator CueR: monomer with copper (red) and without copper (cyan) attached to the pCopA promoter showing DNA bending. (cealigned in pymol from PDB 4wls and 4wlw)<SUP>(6)</SUP></FIGCAPTION><P><I>E. coli</I> cells use a protein called CueR to regulate the cytoplasmic copper concentration. CueR is a <A data-toggle="popover1" data-trigger="hover" title="MerR" data-content="An E. coli transcription factor with a helix-turn-helix motif that regulates the bacterial cells’ response to mercury ">MerR</A>-type regulator with an interesting <A href="https://2016.igem.org/Team:Oxford/CueR_MOI">mechanism of action</A> whereby it can behave as a net activator or a net repressor under different copper concentrations through interaction with <A data-toggle="popover1" data-trigger="hover" title="RNA polymerase" data-content="An enzyme that transcribes the DNA sequence of a gene into an mRNA sequence that is then translated into a protein sequence.">RNA polymerase</A>.<SUP>(7)</SUP> More information and an animated version of the above image can be found <A href="https://2016.igem.org/Team:Oxford/CueR_MOI">here</A>. CueR forms dimers consisting of three functional domains (a DNA-binding, a dimerisation and a metal-binding domain). The DNA binding domains bind to DNA <A data-toggle="popover1" data-trigger="hover" title="Inverted Repeat" data-content="A DNA sequence followed downstream by its reverse complement">inverted repeats</A> called CueR boxes with the sequence: 
</P><P>
CCTTCCNNNNNNGGAAGG.
</P><P>
This box is present at the promoter regions of the copper exporting ATPase CopA, some molybdenum cofactor synthesis genes and the periplasmic copper oxidase protein CueO.<SUP>(4)</SUP></P><P>
We constructed different promoter systems based upon the promoter for <I>copA</I>, pCopA:
</P><H3 id="pcg">pCopA sfGFP</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980005">BBa_K1980005</A></FIGCAPTION><P>
The simplest system we tested was simply the pCopA promoter in front of sfGFP. We found that this system was only weakly copper responsive. This may be because there were 500+ copies of the pCopA promoter located on the high copy plasmid compared with only a single copy of CueR expressed from the bacterial genome. If CueR is only weakly expressed then there would be insufficient protein to bind at all of the promoter binding sites and the genes could not be activated.</P><P>A BspH1 site was included at the start codon of the sfGFP so that other proteins with a compatible start (such as the copper chelators we ligated into our pBAD vector) could be ligated into this location.
</P><FIGCAPTION>Plate reader data for BBa_K1980005 after 4 hours </FIGCAPTION><H3 id="pg">pCopA sfGFP with divergent CueR</H3><FIGCAPTION>Our biosensor design. We unfortunately never managed to submit this part</FIGCAPTION><P>
This promoter system was based upon <A href="http://parts.igem.org/Part:BBa_K1758324">this part</A> in the registry from Team Bielefeld-CeBiTec in 2015. 
</P><P>
The team assembled a copper biosensor from two subparts which they then joined together: 
</P><P> 
The first part is a pCopA-RBS-sfGFP and the second part is the regulator CueR expressed from a constitutive promoter. The part is deposited in the registry and labelled to suggest the CueR is expressed divergent from the sfGFP (on the opposite strand and transcribed in the opposite direction): 
</P><FIGCAPTION>How BBa_K1758324 is labelled in the registry</FIGCAPTION><P>
However if you look at the sequence level this is clearly not the case. The constitutive promoter and the CueR start codon are at the 5’ end of the sfGFP coding strand and the CueR stop codon just upstream of pCopA. The part in fact has the constitutive promoter on the same strand as pCopA and sfGFP facing in the same direction and would be better represented like this: 
</P><FIGCAPTION>How BBa_K1758324 should be labelled in the registry based on its underlying sequence</FIGCAPTION><P>
As the two coding regions are not separated by a transcription terminator, there would be read through from the constitutive promoter to the sfGFP and sfGFP would be expressed even in the absence of copper. As no negative control is included in the plate reader graph they provide and no settings provided for their BioLector experiments in their <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">protocols</A>  it is unclear just how high the expression level at 0mM copper was for this part compared to a negative control strain.
</P><P>The CueR subpart (<A href="http://parts.igem.org/Part:BBa_K17583204">BBa_K1758320</A>) making up BBa_K1758324 is also incorrectly labelled.</P><P>
When we designed this part we flipped the CueR and the constitutive promoter to face the opposite direction on the opposite strand i.e. so they were divergent. We also had to remove the 5'UTR, which Bielefeld found to increase expression, because it was too AT rich to be synthesised.</P><P> Unfortunately, every attempt to amplify this part from the synthesised sequence we received from IDT resulted in the same two point mutations in the sfGFP region of this part making it non-functional. To compare this promoter system to the others we designed, we used our parts with chelator-sfGFP fusions instead of the sfGFP which we expected to have similar behaviour.
</P><H3 id="p">pCopA with divergent expressed CueR</H3><P>
To facilitate making any protein under control of a copper-responsive promoter we designed a part (<A href="http://parts.igem.org/Part:BBa_K1980006">BBa_K1980006</A>) similar to the above pCopA sfGFP with divergent expressed CueR but without the sfGFP. The RBS however is still present meaning that any protein with the ATG biobrick prefix and universal suffix can be inserted with biobrick standard assembly with the correct RBS-ATG distance.
</P><H3 id="fcg">pCopA CueR sfGFP/ Feedback pCopA sfGFP</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980008">BBa_K1980008</A></FIGCAPTION><P>
We designed a part similar to the above but with CueR expressed from the same pCopA promoter that it controls in order to study CueR operating under a feedback loop. If CueR is acting as a net activator, this should in theory act as a positive feedback system whereby a small amount of copper stimulation produces more CueR causing greater activation. Should CueR be acting as a net repressor this system would be less sensitive than our previous system and this appears to be what we discovered. </P><P>
A BspH1 site was included at the start codon of the sfGFP so that other proteins with a compatible start (such as the copper chelators we ligated into our pBAD vector) could be ligated into this location.
</P><FIGCAPTION>Plate reader data for BBa_K1980008 after 4 hours </FIGCAPTION><H2 id="CusS-CusR">CusS/CusR linked systems</H2><P>
The second system <I>E. coli</I> uses to respond to copper is the CusS/CusR two-component system. This consists of the transmembrane histidine kinase enzyme: CusS in the bacterial cytoplasmic membrane and a cytoplasmic response regulator: CusR. 
</P><P>
When CusS binds periplasmic copper it transfers a phosphate group from ATP to CusR aspartate residue 51 via CusS histidine residue 271. Phosphorylated CusR can bind to DNA <A data-toggle="popover1" data-trigger="hover" title="Inverted Repeat" data-content="A DNA sequence followed downstream by its reverse complement">inverted repeat</A> CusR boxes (AAAATGACAANNTTGTCATTTT) and activate gene expression. 
</P><P>
In <I>E. coli</I> this box is present between the promoters for CusCFBA operon which encodes a multi-protein pump that exports cytoplasmic and periplasmic copper from the cell and the CusRS operon which encodes the two component system (therefore acting as a positive feedback loop in vivo). <SUP>(4)</SUP></P><H3 id="pCusC">pCusC RFP</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980007">BBa_K1980007</A></FIGCAPTION><P>
We received a copy of pCusC <A data-toggle="popover1" data-trigger="hover" title="monomeric Kate2" data-content="A very bright far-red fluorescent protein with excitation/emission maxima at 588 and 633 nm">mKate</A> (a form of RFP) from Tom Folliard, one of the PhD students in our lab. This sequence had a biobrick-illegal Spe1 site in the RBS region meaning we could not deposit in the registry. We performed site directed mutagenesis to swap a C in the Spe1 site for G, and moved it into the shipping vector. This part was then deposited. We also amplified the promoter region only and deposited this separately.(<A href="http://parts.igem.org/Part:BBa_K1980004">BBa_K1980004</A>) 
</P><FIGCAPTION>Plate reader data for BBa_K1980007 after 4 hours </FIGCAPTION><H3 id="fck">pCusC CusR RFP</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980009">BBa_K1980009</A></FIGCAPTION><P>
We also designed a version of the part with CusR expressed before mKate forming a positive feedback loop. This sort of system was shown to be more responsive in a paper by Ravikumar S <I>et al.</I><SUP>(5)</SUP>. However after many attempts at cloning our only full length construct had a point mutation (Val to Ala) in the <I>CusR</I> gene. We tested this part to see if this mutation was tolerable, but found no evidence for a more sensitive system:
</P><FIGCAPTION>Plate reader data for BBa_K1980009 after 4 hours </FIGCAPTION><P>A BspH1 site was included at the start codon of the sfGFP so that other proteins with a compatible start (such as the copper chelators we ligated into our pBAD vector) could be ligated into this location.
</P></SECTION><SECTION id="Comps" style="padding-top: 75px; margin-top:-75px;"><H1 class="pageTitleNavy">Composite Parts</H1><P>
These are parts we produced with a copper chelator under control of a copper-sensitive promoter. If these behave as expected then, if a small concentration of copper is added to cells containing this part, copper will induce expression of the chelator, which would then chelate free copper. If free copper is reduced then chelator expression should be reduced until a stable equilibrium is reached. This copper buffering is dependent upon both the copper promoters behaviour and the copper chelator expression and affinity (see our <A href="https://2016.igem.org/Team:Oxford/Model">modelling page</A>). In practice the level of copper concentrations upon which these systems would operate was below our copper absorbance assay detection limits.</P><H3 id="pm">pCopA MymT with divergent expressed CueR</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980011">BBa_K1980011</A></FIGCAPTION><P>
This part contains the chelator MymT behind a copper sensitive promoter with divergent expressed CueR in the biobrick. 
</P><H3 id="pmg">pCopA MymT sfGFP divergent expressed CueR</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980012">BBa_K1980012</A></FIGCAPTION><P>
This part contains our chelator MymT with a C-terminal sfGFP tag behind the pCopA promoter with divergent expressed CueR in the biobrick. 
</P><FIGCAPTION>Plate reader data for BBa_K1980012 after 4 hours </FIGCAPTION><H3 id="ptcg">pCopA TAT Csp1 sfGFP constitutively expressed CueR</H3><FIGCAPTION>The design of part <A href="http://parts.igem.org/Part:BBa_K1980010">BBa_K1980010</A></FIGCAPTION><P>
This part contains our chelator Csp1 with a C-terminal sfGFP tag behind the pCopA promoter with constitutively expressed CueR in the biobrick. Likely due to the expression problems of Csp1 it responded worse than the similar part above with our MymTsfGFP fusion protein.
</P><FIGCAPTION>Plate reader data for BBa_K1980010 after 4 hours </FIGCAPTION></SECTION><H1 id="references" class="pageTitleNavy">References:</H1><DIV class="references"><UL><LI>
(1)	Nicolas Vita, Semeli Platsaki, Arnaud Basle, Stephen J. Allen, Neil G. Paterson, Andrew T. Crombie, J. Colin Murrell, Kevin J.Waldron &amp; Christopher Dennison (2015)
“A four-helix bundle stores copper for methane
oxidation”, Nature 525 issue 7567 pg. 140-143 doi:10.1038/nature14854
</LI><LI>
(2) Ben Gold, Haiteng Deng, Ruslana Bryk, Diana Vargas, David Eliezer, Julia Roberts,
Xiuju Jiang, &amp; Carl Nathan (2009) “Identification of a Copper-Binding Metallothionein in Pathogenic Mycobacteria” Nat Chem Biol. 
2008 October ; 4(10): 609–616. doi:10.1038/nchembio.109.
</LI><LI>
(3)
1AQS from Peterson, C.W., Narula, S.S., Armitage, I.M. “3D solution  structure of copper and silver-substituted yeast metallothioneins” (1996) FEBS Lett. 379: 85-93
</LI><LI>
(4) Yamamoto K, Ishihama A. (2005) “Transcriptional response of Escherichia coli to external copper.” Mol Microbiol. 2005 Apr;56(1):215-27.
</LI><LI>
(5) Sambandam Ravikumar, Van Dung Pham, Seung Hwan Lee, Ik-keun Yoo, Soon Ho Hong (2012) “Modification of CusSR bacterial two-component systems by the introduction of an inducible positive feedback loop” Journal of Industrial Microbiology &amp; Biotechnology June 2012, Volume 39, Issue 6, pp 861–868
</LI><LI>
(6) Philips, S.J., Canalizo-Hernandez M., Yildirim I., Schatz G., Mondragón A., O’Halloran T., (2015) &quot;Allosteric transcriptional regulation via changes in the overall topology of the core promoter.&quot; Science 349: 877-881
</LI><LI>
(7) Danya J. Martell, Chandra P. Joshi, Ahmed Gaballa, Ace George Santiago, Tai-Yen Chen, Won Jung, John D. Helmann, and Peng Chen (2015) “Metalloregulator CueR biases RNA polymerase’s kinetic sampling of dead-end or open complex to repress or activate transcription” Proc Natl Acad Sci U S A. 2015 Nov 3; 112(44): 13467–13472.
</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><DIV class="container-fluid" id="footer" style="padding-top:25px; padding-bottom:30px; border-top:1px solid #CDCCCE; font-family: avenir, arial">

Made with gooood stuff.  </DIV></DIV></DIV></DIV></DIV></BODY></HTML>