<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid content-main"><DIV class="row"><NAV class="col-md-3 bs-docs-sidebar navFont"><UL id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330"><LI><A href="#Overview">Overview</A></LI><LI><A href="#Problem">Wilson's Disease</A></LI><LI><A href="#Probiotics">Probiotics</A></LI><LI><A href="#Video">Video</A></LI><LI><A href="#Parts">Parts</A></LI><LI><A href="#Delivery">Delivery</A></LI><LI><A href="#Results">Results</A></LI><LI><A href="#Improve">Improving previous parts</A></LI><LI><A href="#Conc">Conclusion</A></LI><LI><A href="#Future">Future</A></LI><LI><A href="#References">References</A></LI></UL></NAV><DIV class="col-md-9 content-right" style="background-color: #fff;"><DIV class="pageTitle pageTitleOrange">Cure</DIV><P><CENTER><FONT size="5"> Our project was awarded a <FONT color="#FFD700"><B>GOLD</B></FONT> medal at the 2016 Jamboree.  We were nominated for: <B>Best Therapeutic Project</B>, 
<B>Best Wiki</B>,
<B>Best Presentation</B> and <B>Best Education &amp; Public Engagement!</B></FONT></CENTER></P><SECTION id="Overview"><H2>Overview</H2><P>
Rare, or “orphan”, diseases are frequently ignored by the pharmaceutical industry. They encompass a huge range of disorders from ALS to Tourette’s Syndrome, but individually have a relatively low number of patients. The low patient numbers mean that there is very little impetus for the pharmaceutical industry to research and produce novel, innovative therapeutics. This means that patients are often left with unsatisfactory treatments. Our goal is to produce a probiotic therapeutic to treat one such disorder: Wilson’s Disease. 
</P></SECTION><SECTION id="Problem"><H2>Wilson's Disease</H2><P>
Wilson’s Disease is a genetic disorder characterised by an inability of the body to metabolise copper. Normally, when copper is ingested, it is taken up by the small intestine, taken to the liver and subsequently transported into the blood or excreted into the bile. In Wilson’s Disease, there is a mutation in the gene ATP7B. ATP7B encodes a copper-transporting protein that is responsible for loading copper onto caeruloplasmin for transport in the blood, and into the bile for removal from the body. In the absence of a functional form of this protein, copper is unable to be removed from the liver after absorption. This results in toxic accumulation, as cuprous ions react with hydrogen peroxide to produce dangerous free radicals that damage tissue. This allows copper ions to leak into the blood and eventually accumulate in, and damage, other tissues, such as the kidneys and brain. You can read more about Wilson's disease <A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">here</A>.
</P><P>
Current treatments are regarded by patients as unsatisfactory. From discussions with patients, we have identified three main problems with current treatments:
</P><LI class="textLiFont">Side effects: sometimes so severe that treatment must be stopped</LI><LI class="textLiFont">Price: these drugs are very expensive for the UK National Health Service</LI><LI class="textLiFont">High dosage frequency: drugs must be taken daily and refrigerated</LI><P>
Our Cure aims to address these limitations.</P><FIGCAPTION>Wilson's disease causes distinctive copper deposits in the cornea of some patients, causing brown rings around the iris called Kayser-Fleischer rings.</FIGCAPTION></SECTION><SECTION id="Probiotics"><H2>Probiotics</H2><P>
A probiotic is a microorganism that is introduced into the body for its beneficial properties. The concept of a probiotic, meaning “for life”, was introduced by Elie Metchnikoff in 1907, when he hypothesised that replacing or diminishing the populations of ‘putrefactive’ bacteria in the gut with lactic acid bacteria could positively affect bowel health.
</P><P>
Products that are commonly sold as probiotics include foodstuffs, such as yoghurts and cheeses.  However, recently there has been an increase in the amount of research going into the use of probiotics as therapeutics, with the genetic engineering of organisms to produce useful substances. Currently there is limited legislation regarding probiotics, as probiotics sold as dietary supplements do not require FDA approval. A genetically-engineered probiotic therapeutic would require more stringent legislation and FDA approval to ensure, through clinical trials, that it works as expected.
</P><P>
Although advancing rapidly, the field of probiotics still requires significant research particularly in areas such as safety. Although regarded as safe for relatively healthy humans to consume, there have been some reports of probiotic-related side effects in people with serious underlying medical conditions. We carried out a comprehensive <A href="https://2016.igem.org/Team:Oxford/Safety">safety</A> review when completing our project.
</P></SECTION><SECTION id="Video"><H2>Video</H2></SECTION><SECTION id="Parts"><H2>Parts</H2><P>
In order to provide a long-term treatment for Wilson's disease we realised that we needed to design components of a bacterial system that could detect copper and produce a copper chelator to prevent its absorption by the body. 
</P><P>
We investigated copper detection using promoter systems based upon the native <I>E. coli</I> copper regulator <A href="https://2016.igem.org/Team:Oxford/Parts#CueR">CueR</A> and the <A href="https://2016.igem.org/Team:Oxford/Parts#CusS-CusR">CusS/CusR</A> two component system. The two copper chelators we tested were <A href="https://2016.igem.org/Team:Oxford/Parts#Csp1H">Copper storage protein 1</A> (Csp1) from <I>Methylosinus trichosporium</I> OB3b and <A href="https://2016.igem.org/Team:Oxford/Parts#MymTH">mycobacterial metallothionein</A> (MymT) from <I>Mycobacterium tuberculosis.</I></P><P>You can read about how we chose our parts <A href="https://2016.igem.org/Team:Oxford/Parts">here</A>.</P><FIGCAPTION>Our chelator Copper Storage Protein 1 (monomer) with 13 bound copper ions.</FIGCAPTION></SECTION><SECTION id="Delivery"><H2>Delivery</H2><P>
From discussion with patients and the public, and the work carried out by previous Oxford iGEM teams, we decided to investigate the use of a bead to deliver our bacteria to the small intestine. Our bacteria will initially be encapsulated in an alginate matrix, and then be alternately coated in layers of alginate and chitosan. The goal of the multiple polymer-coatings is to protect the bacteria from the harsh conditions of the stomach, whilst having the ability to degrade in the more alkaline pH of the small intestine. This degradation releases our bacteria into the favourable conditions of the small intestine, where they can colonise the area and chelate dietary copper.
</P><FIGCAPTION>Our bead design</FIGCAPTION></SECTION><SECTION id="Results"><H2>Results</H2><P>
Through our experimental work we have been able to obtain data suggesting the following:</P><UL><LI>Different arrangements of both our CueR-linked and CusS/CusR-linked promoter systems are sensitive over a range of copper concentrations, including at the lower concentrations mimicking the gut.</LI><LI>MymT is able to chelate a measurable amount of copper <I>in vivo.</I></LI><LI>Alternately layered chitosan-alginate beads release material in the small intestine, following passage through the stomach, in a simulation accounting for the pH change. </LI></UL><P>Please see our <A href="https://2016.igem.org/Team:Oxford/Experiments">experiments</A> and <A href="https://2016.igem.org/Team:Oxford/Results">results</A> pages for more discussion of our methods and the outcomes.</P></SECTION><SECTION id="Improve"><H2>Improving Registry Parts</H2><P>
As part of our project, we have improved the function/characterisation of three poorly documented parts.</P><P>
Please visit our <A href="https://2016.igem.org/Team:Oxford/Parts">parts</A> and <A href="https://2016.igem.org/Team:Oxford/Results">results</A> pages for further information.</P><H3>BBa_I760005:</H3><P>This <A href="http://parts.igem.org/Part:BBa_I760005">copy of pCusC</A> located in the registry had been previously used 11 times, but had no associated useful characterisation data. From discussions with a mentor, Tom Folliard, we came to the conclusion that this may have been because the promoter region was too short, and thus missing key binding sites. We decided to deposit an elongated form of this promoter, which we have achieved thorough characterisation. </P><FIGCAPTION>Plate reader data for our improved part (BBa_K1980004) with RFP: BBa_K1980007</FIGCAPTION><H3>BBa_K190020:</H3><P><A href="http://parts.igem.org/Part:BBa_K190020">MymT</A> can be found in the registry with no associated characterisation data. We codon-optimised the sequence for expression in <I>E. coli</I> and added a hexahistidine tag for purification. We also produced a form with a C-terminal sfGFP tag (also his-tagged) to aid purification, allowing us to view the cellular distribution of MymT and allow analysis by <U>F</U>luorescence <U>L</U>ifetime <U>IM</U>aging (FLIM). The fluorescence lifetime of GFP with a C-terminal His tag was shown to be reduced by copper by Hötzer et al., allowing FLIM to be used in an <I>in vivo</I> assay of copper binding. This has provided us with preliminary characterisation data for the protein, suggesting copper-binding activity <I>in vivo</I>. More details can be found on our <A href="https://2016.igem.org/Team:Oxford/Results#FLIM">results page</A>.  </P><FIGCAPTION>Fluorescence lifetime for sfGFP, Csp1-sfGFP and MymTsfGFP from iGEM Cardiff. (No error bars shown when standard deviation=0.) </FIGCAPTION><H3>BBa_K1758324:</H3><P>We were interested in using <A href="http://parts.igem.org/Part:BBa_K1758324">this part</A> from Team Bielefeld-CeBiTec in 2015 as a potential copper biosensor.
</P><P>
The team assembled a copper biosensor from two subparts which they then joined together: 
</P><P> 
The first part is a pCopA-RBS-sfGFP and the second part is the regulator CueR expressed from a constitutive promoter. The part is deposited in the registry and labelled to suggest the CueR is expressed divergent from the sfGFP (on the opposite strand and transcribed in the opposite direction): 
</P><FIGCAPTION>How BBa_K1758324 is labelled in the registry</FIGCAPTION><P>
However if you look at the sequence level this is clearly not the case. The constitutive promoter and the CueR start codon are at the 5’ end of the sfGFP coding strand and the CueR stop codon just upstream of pCopA. The part in fact has the constitutive promoter on the same strand as pCopA and sfGFP facing in the same direction and would be better represented like this: 
</P><FIGCAPTION>How BBa_K1758324 should be labelled in the registry based on its underlying sequence</FIGCAPTION><P>
As the two coding regions are not separated by a transcription terminator, there would be read through from the constitutive promoter to the sfGFP and sfGFP would be expressed even in the absence of copper. As no negative control is included in the plate reader graph they provide and no settings provided for their BioLector experiments in their <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">protocols</A>  it is unclear just how high the expression level at 0mM copper was for this part compared to a negative control strain.
</P><P>The CueR subpart (<A href="http://parts.igem.org/Part:BBa_K17583204">BBa_K1758320</A>) making up BBa_K1758324 is also incorrectly labelled.</P><P>
When we designed this part we flipped the CueR and the constitutive promoter to face the opposite direction on the opposite strand i.e. so they were divergent. We also had to remove the 5'UTR, which Bielefeld found to increase expression, because it was too AT rich to be synthesised.</P><P> Unfortunately, every attempt to amplify this part from the synthesised sequence we received from IDT resulted in the same two point mutations in the sfGFP region of this part making it non-functional. To compare this promoter system to the others we designed, we used our parts with chelator-sfGFP fusions instead of the sfGFP which we expected to have similar behaviour.
</P></SECTION><SECTION id="Conc"><H2>Conclusion</H2><P>Over the course of the summer we have successfully created and submitted 13 sequence-confirmed BioBrick parts, 8 of which have been characterised. Through our experiments, we have been able to examine the copper-sensitivity of a variety of promoters integral to our Cure. In addition, we have tested the chelating-ability of our copper chelators with a variety of different assays. Although these have not all been successful, through a collaboration with Cardiff we have been able to obtain preliminary data suggesting that MymT may be able to successfully lower the intracellular copper concentration. We have also obtained promising preliminary data relating to the delivery of substances to the small intestine via alginate-chitosan beads.</P><P>In conclusion, over the course of the summer we have successfully managed to produce an initial proof of concept for our idea and have contributed to our understanding of these systems, in addition to investigating a treatment that addresses the needs and concerns of the patients themselves.</P></SECTION><SECTION id="Future"><H2>Future</H2><P>To further develop our project, we would hope to carry out a number of further experiments to allow us to generate something more than a proof-of-concept model. Primarily, we would hope to properly characterise the copper-chelating ability of Csp1 and confirm its localisation in the periplasm by redesigning the TAT sequence. Additionally, we would carry out further experiments to refine our delivery system. </P><P>Once the system was able to successfully operate in <I>E. coli</I> DH5-α, we would hope to express it in a strain that more closely mimics the bacteria of the gut. More information on a potential future chassis choice can be found on our <A href="https://2016.igem.org/Team:Oxford/Safety">safety page</A>.</P><P>Finally, the design of our system went through many iterations over the summer. One of our most ambitious designs was based on logic gates that would provide an additional safety mechanism. We would love to test this system. We discuss this design further on our <A href="https://2016.igem.org/Team:Oxford/Safety">safety page</A>.</P></SECTION><FIGCAPTION>Post presentation photo with the whole team.</FIGCAPTION><H1 id="References" class="pageTitleOrange">References:</H1><DIV class="referenceSize"><UL><LI>
Roberts, E. (2011) ‘Wilson’s Disease’, Medicine, 39(10), pp. 602–604.
</LI><LI>
Schilsky, M.L., Roberts, E.A., Hahn, S. and Askari, F. (2015) ‘Costly choices for treating Wilson’s disease’, Hepatology, 61(4), pp. 1106–1108. doi: 10.1002/hep.27663.
</LI><LI>
Czlonkowska, A., Gajda, J. and Rodo, M. (1996) ‘Effects of long-term treatment in Wilson’s disease with d-penicillamine and zinc sulphate’, Journal of Neurology, 243(3), pp. 269–273. doi: 10.1007/bf00868525.
</LI><LI>
Duan, F.F., Liu, J.H. and March, J.C. (2015) ‘Engineered Commensal bacteria Reprogram intestinal cells into glucose-responsive Insulin-Secreting cells for the treatment of diabetes’, Diabetes, 64(5), pp. 1794–1803. doi: 10.2337/db14-0635.
</LI><LI>
Didari, T., Solki, S., Mozaffari, S., Nikfar, S. and Abdollahi, M. (2014) ‘A systematic review of the safety of probiotics’, Expert Opinion on Drug Safety, 13(2), pp. 227–239. doi: 10.1517/14740338.2014.872627.
</LI><LI>
Anukam K. C., Reid G. (2007). “Probiotics: 100 years (1907–2007) after Elie Metchnikoff’s Observation,” Communicating Current Research and Educational Topics and Trends in Applied Microbiology, ed. Méndez-Vilas A., editor. (Formatex.org) 466–474
</LI><LI>
Cook, M.T., Tzortzis, G., Khutoryanskiy, V.V. and Charalampopoulos, D. (2013) ‘Layer-by-layer coating of alginate matrices with chitosan–alginate for the improved survival and targeted delivery of probiotic bacteria after oral administration’, J. Mater. Chem. B, 1(1), pp. 52–60. doi: 10.1039/c2tb00126h.
</LI><LI>
Chávarri, M., Marañón, I., Ares, R., Ibáñez, F.C., Marzo, F. and Villarán, M. del C. (2010) ‘Microencapsulation of a probiotic and prebiotic in alginate-chitosan capsules improves survival in simulated gastro-intestinal conditions’, International Journal of Food Microbiology, 142(1-2), pp. 185–189. doi: 10.1016/j.ijfoodmicro.2010.06.022.
</LI><LI>
Krasaekoopt, W., Bhandari, B. and Deeth, H. (2004) ‘The influence of coating materials on some properties of alginate beads and survivability of microencapsulated probiotic bacteria’,International Dairy Journal, 14(8), pp. 737–743. doi: 10.1016/j.idairyj.2004.01.004.
</LI><LI>
Argüello, J.M., Raimunda, D. and Padilla-Benavides, T. (2013) ‘Mechanisms of copper homeostasis in bacteria’, Frontiers in Cellular and Infection Microbiology, 3. doi: 10.3389/fcimb.2013.00073.
</LI><LI>
Grey, B. and Steck, T.R. (2001) ‘Concentrations of copper thought to be toxic to Escherichia coli can induce the viable but Nonculturable condition’, Applied and Environmental Microbiology, 67(11), pp. 5325–5327. doi: 10.1128/aem.67.11.5325-5327.2001.
</LI><LI>
Lewis, K.O. (1973) ‘The nature of the copper complexes in bile and their relationship to the absorption and excretion of copper in normal subjects and in Wilson’s disease’, Gut, 14(3), pp. 221–232. doi: 10.1136/gut.14.3.221.
</LI><LI>
Rensing, C. and Grass, G. (2003) ‘Escherichia coli mechanisms of copper homeostasis in a changing environment’, FEMS Microbiology Reviews, 27(2-3), pp. 197–213. doi: 10.1016/s0168-6445(03)00049-4.
</LI><LI>
Zoetendal, E.G., Raes, J., van den Bogert, B., Arumugam, M., Booijink, C.C., Troost, F.J., Bork, P., Wels, M., de Vos, W.M. and Kleerebezem, M. (2012) ‘The human small intestinal microbiota is driven by rapid uptake and conversion of simple carbohydrates’, The ISME Journal, 6(7), pp. 1415–1426. doi: 10.1038/ismej.2011.212.
</LI><LI>
Franz, K.J. (2012) ‘Application of inorganic chemistry for non-cancer therapeutics’, Dalton Transactions, 41(21), p. 6333. doi: 10.1039/c2dt90061k.
</LI><LI>
Das, S.K. and Ray, K. (2006) ‘Wilson’s disease: An update’, Nature Clinical Practice Neurology, 2(9), pp. 482–493. doi: 10.1038/ncpneuro0291.
</LI><LI>
Yamamoto, K. and Ishihama, A. (2005) ‘Transcriptional response of Escherichia coli to external copper’, Molecular Microbiology, 56(1), pp. 215–227. doi: 10.1111/j.1365-2958.2005.04532.x.
</LI><LI>
Gold, B., Deng, H., Bryk, R., Vargas, D., Eliezer, D., Roberts, J., Jiang, X. and Nathan, C. (2008) ‘Identification of a copper-binding metallothionein in pathogenic mycobacteria’, Nature chemical biology, 4(10), pp. 609–616. doi: 10.1038/nchembio.109.
</LI><LI>
Outten, F.W. (2001) ‘The independent cue and cus systems confer copper tolerance during aerobic and anaerobic growth in Escherichia coli’, Journal of Biological Chemistry, 276(33), pp. 30670–30677. doi: 10.1074/jbc.m104122200.
</LI><LI>
Ravikumar, S., Pham, V.D., Lee, S.H., Yoo, I. and Hong, S.H. (2012) ‘Modification of CusSR bacterial two-component systems by the introduction of an inducible positive feedback loop’, Journal of Industrial Microbiology &amp; Biotechnology, 39(6), pp. 861–868. doi: 10.1007/s10295-012-1096-y.
</LI><LI>
Neupert, J., Karcher, D. and Bock, R. (2008) ‘Design of simple synthetic RNA thermometers for temperature-controlled gene expression in Escherichia coli’, Nucleic Acids Research, 36(19), pp. e124–e124. doi: 10.1093/nar/gkn545.
</LI><LI>
Vita, N., Platsaki, S., Baslé, A., Allen, S.J., Paterson, N.G., Crombie, A.T., Murrell, J.C., Waldron, K.J. and Dennison, C. (2015) ‘A four-helix bundle stores copper for methane oxidation’, Nature, 525(7567), pp. 140–143. doi: 10.1038/nature14854.
</LI><LI>
Balogha R., Gyurcsika B., Hunyadi-Gulyásb E., Christensenc H., Jancsóa A., (2016) “Advanced purification strategy for CueR, a cysteine containing copper(I) and DNA binding protein” Protein Expression and Purification Volume 123, July 2016, Pages 90–96.
</LI><LI>Danya J. Martell, Chandra P. Joshi, Ahmed Gaballa, Ace George Santiago, Tai-Yen Chen, Won Jung, John D. Helmann, and Peng Chen (2015) “Metalloregulator CueR biases RNA polymerase’s kinetic sampling of dead-end or open complex to repress or activate transcription” Proc Natl Acad Sci U S A. 2015 Nov 3; 112(44): 13467–13472.</LI><LI>Philips, S.J., Canalizo-Hernandez M., Yildirim I., Schatz G., Mondragón A., O’Halloran T., (2015) &quot;Allosteric transcriptional regulation via changes in the overall topology of the core promoter.&quot; Science 349: 877-881</LI><LI>
http://bionumbers.hms.harvard.edu/KeyNumbers.aspx
</LI><LI>
Selinger, D et al. (2003) Global RNA Half-Life Analysis in Escherichia coli Reveals Positional Patterns of Transcript Degradation, Genome Res. 13, pp. 216-223.
</LI><LI>
http://homepages.ulb.ac.be/~dgonze/BIONUMBERS/bionumbers.html
</LI><LI>
Changela, A et al. (2003) Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR, Science 301 (5638), pp. 1383-1387.
</LI><LI>
Stoyanov, J et al. (2001) CueR (YbbI) of Escherichia coli is a MerR family regulator controlling expression of the copper exporter CopA,  Molecular Microbiology 39(2), pp. 502-512.
</LI><LI>
Gudipaty, S et al (2014) The histidine kinase CusS senses silver ions through direct binding by its sensor domain, Biochim. Biophys. Acta 1844(9), pp. 1656-1661.
</LI><LI>
Hsiao, V et al. (2013) Design and implementation of a synthetic biomolecular concentration tracker, ACS Synth. Biol. 4(2), pp. 150-161.
</LI><LI>
Wang, B et al. (2013) A modular cell-based biosensor using engineered genetic logic circuits to detect and integrate multiple environmental signals, Biosensors and Bioelectronics 40, pp. 368-376. 
</LI><LI>
Vita, N et al. (2015) A four-helix bundle stores copper for methane oxidation, Nature 525, pp. 140-143.
</LI><LI>
http://physiologyplus.com/motility-in-the-small-intestine/
</LI><LI>
Moxon, T et al. (2016) In silico modelling of mass transfer &amp; absorption in the human gut, Journal of Food Engineering 176, pp. 110-120.
</LI><LI>
http://cds.ismrm.org/protected/10MProceedings/files/158_7021.pdf flow speed
</LI><LI>
http://trove.nla.gov.au/work/11977899?q&amp;versionId=45681698 other dimensions of gut
</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><DIV class="container-fluid" id="footer" style="padding-top:25px; padding-bottom:30px; border-top:1px solid #CDCCCE; font-family: avenir, arial">

Made with gooood stuff.  </DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>