<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Copper_Promoters skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Copper Promoters</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-9"><P>
 For our copper chelating bacteria to successfully compete in the gut we investigated copper biosensors so that the chelator was only produced when copper was present, maintaining cell resources. 
</P><P>
As copper is a redox metal copper is both very important in cells enzymes but also very toxic because it leads to the formation of reactive oxygen species. Consequently <I>Escherichia coli</I> have two system of detecting excess copper: the CueR-linked system that detect cytoplasmic copper and the CusS/CusR two component system that detects periplasmic copper. 
We investigated both systems and, by arranging the components to form a positive feedback loop, managed to improve their sensitivity over the range of copper concentrations we were interested in.
</P><H2> CueR linked systems:</H2><P><I>E. coli</I> cells use a protein called CueR to regulate the cytoplasmic copper concentration. CueR is a Mer type regulator with an interesting <A href="https://2016.igem.org/Team:Oxford/CueR_MOI">mechanism of action</A>. CueR forms dimers consisting of three functional domains (a DNA-binding, a dimerisation and a metal-binding domain). The DNA binding domains bind to DNA inverted repeats called CueR boxes with the sequence: CCTTCCNNNNNNGGAAGG.
</P><P>
This box is present at the promoter regions of the copper exporting ATPase CopA, some molybdenum cofactor synthesis genes and the periplasmic copper oxidase protein CueO.*
Transcriptional response of Escherichia coli to external copper
Authors Yamamoto, Ishihama
 </P><P>
We constructed three different promoter systems based upon pCopA.
</P><H2> pCopA sfGFP </H2><P>
The simplest system we tested was simply the pCopA promoter in front sfGFP. We found that this system was only weakly copper responsive likely because the CueR from the single gene on the genome was outweighed by the pCopA on a high copy number plasmid. 
</P><H2> pCopA sfGFP with constitutively expressed CueR </H2><P>
This is what we called pg.
</P><H2> pCopA CueR sfGFP/ Feedback pCopA </H2><P>
We designed a part similar to the above but with CueR expressed from the same pCopA promoter that it controls. This should in theory act as a positive feedback system whereby, if CueR is acting as a net activator, a small amount of copper stimulation produces more CueR causing greater activation. Obviously should CueR be acting as a net repressor this system would be less sensitive than our previous system. 
</P><H2> pCopA with constitutively expressed CueR </H2><P>
To facilitate making any protein under control of a copper-responsive promoter we designed a part with similar to the above pCopA sfGFP with constitutively expressed CueR but without the sfGFP. The RBS however is still present meaning that any protein with the ATG biobrick prefix and universal suffix can be inserted with biobrick standard assembly with the correct RBS-ATG distance.
</P><H2> pCopA Csp1-sfGFP with constitutively expressed CueR </H2><P>
This part contains our chelator Csp1 with a C terminal sfGFP tag behind the pCopA promoter with constitutively expressed CueR. Likely due to the expression problems of Csp1 it responded worse than the similar part below with MymT.
</P><H2> pCopA MymT with constitutively expressed CueR </H2><P>
This part was little used. 
</P><H2> pCopA MymT-sfGFP with constitutively expressed CueR </H2><P>
This part contains our chelator MymT with a C terminal sfGFP tag behind the pCopA promoter with constitutively expressed CueR. The expression of this part was unsurprisingly very similar to MymT.
</P><H2> CusS/CusR systems:</H2><P>
The second system E. coli uses to respond to copper is the CusS/CusR two component system. This consists of the membrane-bound histidine kinase enzyme CusS in the bacterial outer membrane and a cytoplasmic response regulator CusR. 
</P><P>
When CusS binds periplasmic copper it transfer a phage group from ATP to CusR aspartate residue 51 via CusS histidine residue 271. Phosphorylated CusR can bind to DNA inverse repeat CusR boxes (AAAATGACAANNTTGTCATTTT)  and activate gene expression. 
</P><P>
In <I>E. coli</I> this box is present between the promoters for CusCFBA operon which encodes a multiprotein pump that exports cytoplasmic and periplasmic copper from the cell and the CusRS operon which encodes the two component system (therefore acting as a positive feedback loop in vivo).
</P><H2>pCusC RFP</H2><P>
We received a copy of pCusC mKate (a form of RFP) from Tom Folliard, one of the PhD students in our lab. This sequence had a illegal Spe1 site in the RBS region meaning we could not deposit in the registry. As we had already got impressive plate reader data for this part we performed site directed mutagenesis to swap a C in the Spe1 site for G in order to make the smallest change to the DNA structure. This part was then deposited. We also amplified the promoter region only and deposited this separately. 
</P><H2>pCusC CusR RFP/Feedback pCusC RFP</H2><P>
We also designed a version of the part with CusR expressed before mKate forming a positive feedback loop. This sort of system was shown to be more responsive by Modification of CusSR bacterial two-component systems by the introduction of an inducible positive feedback loop.
Ravikumar S et al. 
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>