<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/InterLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid content-main"><DIV class="row"><NAV class="col-md-3 bs-docs-sidebar navFont"><UL id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330"><LI><A href="#Introduction-What-Is-Interlab">Introduction: What is InterLab</A></LI><LI><A href="#Overview">Overview</A></LI><LI><A href="#The-Constructs">The Constructs</A></LI><LI><A href="#Test-Devices">Test Devices</A></LI><LI><A href="#Experimental-Controls">Experimental Controls</A></LI></UL><LI><A href="#Methods-and-Materials">Methods and Materials</A></LI><LI><A href="#Plate-Reader">Plate Reader</A></LI><LI><A href="#Setup">Setup</A></LI><LI><A href="#Calibration-Protocols">Calibration Protocols</A></LI><LI><A href="#Resuspension">Resuspension</A></LI><LI><A href="#Transformation">Transformation</A></LI><LI><A href="#LB-Agar-Plate-Preparation">LB Agar Plate Preparation</A></LI><LI><A href="#Picking-Colonies">Picking Colonies</A></LI><LI><A href="#Extracting-the-plasmid-DNA">Extracting the plasmid DNA</A></LI><LI><A href="#In-silico-Cloning-and-Sequence-Alignment"><I>In silico</I> cloning and sequence
alignment</A></LI><LI><A href="#Cell-Measurement-Protocol">Cell Measurement Protocol</A></LI><LI><A href="#Results-and-Discussion-the-iGEM-Protocol">Results and discussion:the iGEM protocol</A></LI><LI><A href="#Improving-the-iGEM-protocol">Improving the iGEM protocol</A></LI><LI><A href="#Oxford-iGEM-teams-plate-reader-protocol">Oxford Team’s protocol</A></LI></NAV><DIV class="col-md-9" style="background-color: #fff;"><SECTION id="Introduction-What-Is-Interlab" style="padding-left: 60px; padding-right: 60px"><H1 class="pageTitleBlue">Introduction: What is Interlab?</H1><P>Standardisation is one of the main principles of synthetic biology. This includes a standard way of making measurements and expressing quantities in the same way across labs around the world. One such challenge are fluorescent measurements of genetic constructs. This is because different labs use different protocols for making constructs, for measuring, and for processing of fluorescence. Without having a standard way of expressing fluorescence it is extremely difficult to compare measurements for repeats of the same part from different labs or for different constructs. The iGEM InterLab study aims to standardise the way that fluorescent measurements are made, and therefore allow easier comparisons.
	This year, iGEM has provided a standard protocol for making measurements in a plate reader, a spectrophotometer or a flow cytometer. In addition, to remove variations in the production of the constructs, all biobricks to be tested have been sent in plasmid form. A standard way of data processing has also been devised by iGEM HQ by sending a preconstructed Excel data sheet where each team will input its data and fluorescence can be calculated automatically in a standardised way.</P></SECTION><SECTION id="Overview" style="padding-left: 60px; padding-right: 60px"><H1 class="pageTitleBlue">Overview</H1><H2 id="The-Constructs">The Constructs</H2><P>The Interlab study of 2016 aims to characterise the strength of 3 constitutive promoters from the <A href="parts.igem.org/Promoters/Catalog/Anderson">Anderson Collection</A> by measuring the fluorescence of the GFP encoded downstream of each of the promoters (promoter strength is proportional to GFP fluorescence). The three constructs; named Test Device 1,2 and 3 (TD1, TD2, TD3), include one of the promoters (<A href="http://parts.igem.org/Part:BBa_J23101">J23101</A>, <A href="http://parts.igem.org/Part:BBa_J23106">J23106</A> and <A heref="http://parts.igem.org/Part:BBa_J23117">J23117</A> respectively) and share the RBS (<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>), GFP (<A href="http://parts.igem.org/Part:BBa_E0040">E0040</A>) and the transcriptional terminator (<A href="http://parts.igem.org/Part:BBa_B0015">B0015</A>).</P><H3 id="Test-Devices">Test Devices</H3><CENTER> Test Device 1 (TD1) </CENTER><CENTER> Test Device 2 (TD2) </CENTER><CENTER> Test Device 3 (TD3) </CENTER><H3 id="Experimental-Controls">Experimental controls</H3><P>The InterLab Measurement Kit included 2 controls. The Positive Control (PC) consists of the constitutively expressed GFP device (<A href="http://parts.igem.org/Part:BBa_I20270">I20270</A>) and the Negative Control consists of the pTetR promoter (<A href="http://parts.igem.org/Part:BBa_R0040">R0040</A>) with no coding sequence downstream of it. All five constructs were already in the pSB1C3 plasmid, which carries chloramphenicol resistance.</P><CENTER> Negative Control (NC) </CENTER><CENTER> Positive Control (PC) </CENTER></SECTION><SECTION id="Methods-and-Materials" style="padding-left: 60px; padding-right: 60px"><H1 class="pageTitleBlue">Methods and materials</H1><H2 id="Plate-Reader">Plate reader</H2><P>For OD600nm and fluorescence measurements the BMG Labtech FLUOstar Omega plate reader was used.</P><P>2 plate reader scripts were created, one for OD600 and one for fluorescence. Costar 96-well plates were used to load our solutions.
	</P><H3 id="Setup">Setup</H3><H4>OD600 script:</H4><P>The excitation filter was 600nm and the number of flashes per well was set to 22. Pathlength correction was turned off</P><H4>Fluorescence script:</H4><P>The excitation filter was set to 485-12nm and the emission filter to 520nm. The number of flashes per well was set to 20 and the bottom optic was used for the measurements.</P><H3 id="Calibration-Protocols">Calibration protocols</H3><H4>OD600 reference point measurement</H4><P>LUDOX S30 from the InterLab Measurement Kit was thawed. 100μl of LUDOX S30 was added a COSTAR 96-well plate, in wells A1, B1, C1 and D1. The same was repeated with dH2O in wells A2, B2, C2 and D2. The OD600 script was used to measure the OD in each of the eight wells. The data was used to produce a correction factor. The correction factor would later be used to transform the measured OD600 raw data to standard OD600 measurements.</P><H4>FITC fluorescence and standard curve</H4><P>The FITC (black test tube) from the InterLab Measurement Kit was centrifuged (10 seconds, 13.0k rpm) to make sure that the pellet was at the bottom of the tube. 1ml of 1xPBS was added in the tube to make a 2x FITC stock (500μM). The solution was incubated for 4 hours at 42°C but FITC was not completely dissolved. Therefore the FITC solution was incubated overnight. The stock was then further diluted to 1x FITC (250μM). 200μl of the 1x FITC solution was added to wells A1, B1, C1 and D1 in a COSTAR 96-well plate. 100μl 1x PBS was added to wells A2-12, B2-12, C2-12, D2-12. 100μl from A1 was transferred to A2. The new dilution was mixed and the process was repeated until A11 was reached. 100μl out of the final 200μl in well A11 was discarded. The process was repeated for rows B, C and D. The fluorescence script on the plate reader was used to measure the fluorescence. Data was then exported to the Excel Data Sheet from iGEM and the calibration curve was created. </P><H2 id="Resuspension">Resuspension</H2><P>The five test tubes containing the plasmid constructs were taken out of the InterLab Measurement Kit and centrifuged at maximum speed (13.0k rpm) for 10 seconds in order to deposit the liquid at the bottom of the tube. 100μl of MilliQ (ultrapure water) was then added to the tubes to make a final concentration of 9.09ng/ml (initial volume and concentration were 10μl and 100pg/μL). The solutions were centrifuged again to make sure that everything in the tube was properly dissolved. The diluted plasmid DNA was stored at -4°C.</P><H2 id="Transformation">Transformation</H2><P>Competent <I>E. coli</I> (DH5α strain) cells were produced and used for transformation. Aliquots of 100μl were taken out of the -80°C freezer and thawed on ice (~-4°C). Plasmid DNA and chloramphenicol (stock solution of 30mg/ml) were put on ice as well for thawing. Once the 100μl of DH5α <I>E. coli</I> had thawed, 5μl of plasmid DNA was added. The mixture was left on ice for 30 minutes. </P><H2 id="LB-Agar-Plate-Preparation">LB agar plate preparation</H2><P>While the competent bacteria and DNA mixture was on ice, Lysogeny Broth (LB) plates were made. Low-salt LB agar was heated in a 900W Panasonic microwave oven. The agar was melted using the “Simmer” heating setting for 2 minute intervals. The heating was repeated until the agar was completely liquefied. This liquid LB was put in a 55°C water bath in order to cool down. After reaching the target temperature, the bottle containing the broth was allowed to cool below 55°C for 3-5 minutes in order to add chloramphenicol. Chloramphenicol was then added in a 1:1000 ratio to the LB (1μl of chloramphenicol per ml of LB). In a flow cupboard, the solution was poured into Thermo Scientific Sterilin Standard single-use petri dishes (25ml per dish) and allowed to resolidify (30-35 minutes). After 30 minutes, the bacterial/plasmid DNA solutions were heat shocked in a 42°C water bath for 45 seconds. They were put on ice again for 1 minute and then 800μl of the iGEM supplied SOC was added. This was then incubated at 37°C for 1 hour at 400rpm.
	After the end of incubation, 100μl of the solution for each part was added on separate LB agar plates to make the “dilute” plates. The rest of the solutions were centrifuged for 1 minute at 13.0k rpm in order to deposit any remaining bacteria at the bottom of the Eppendorf tubes. The supernatant was decanted and the bacteria were resuspended in 100μl of SOC. This second solution was used to make the “concentrated” LB plate for each construct. This was done in case the “dilute” plate wouldn’t produce any colonies. The solution in each plate was spread using sterile glass beads which were removed after the spreading. All the plates were taped and placed in a 37°C incubator overnight.</P><H2 id="Picking-Colonies">Picking colonies</H2><P>The following day, plates were taken out of the incubator and stored in the cold room, at -4°C. In the afternoon, 10ml of liquid LB was pipetted into sterile glass test tubes and 5μl of chloramphenicol was added to each solution. 2 test tubes were prepared for each construct (10 in total). Using <I>P200</I> pipette tips, 2 colonies for each construct were picked from their respective plates. Each tip was dropped in each test tube. The test tubes were put in a 37°C, 225rpm incubator overnight (16-18 hours).</P><H2 id="Extracting-the-plasmid-DNA">Extracting the plasmid DNA</H2><P>A QIAprep Spin Miniprep Kit (250) was used to extract the plasmid DNA from the overnight cultures (see protocol on the Protocols page). The concentration of the extracted DNA solution was measured using Thermo Scientific’s NanoDrop 1000 spectrophotometer by blanking it first with MilliQ and then adding 2μl of plasmid solution. The DNA solutions were diluted to a value close to 100μl. 10μl of each DNA construct was then aliquoted into fresh Eppendorf tubes and sent for sequencing (<A href="http://www.sourcebioscience.com/">SourceBioScience, UK</A>) to confirm the nature of the plasmids. Primers created for our project used to sequence our own constructs in pSB1C3 were also sent.</P><H2 id="In-silico-Cloning-and-Sequence-Alignment"><I>In silico</I> cloning and sequence alignment</H2><P>On the iGEM interlab webpage there are no files with the complete sequence of the five constructs in the pSB1C3 plasmid. Therefore, the Registry of Standard Biological Parts was used to find all five construct (PC, NC, TD1, TD2, TD3) and plasmid backbone (pSB1C3) sequences. Each construct was digested in silico using SnapGene, the molecular biology software. Digestion was done with EcoRI for the biobrick prefix and with PstI for the suffix. The same was done for the plasmid backbone. Each construct was then ligated into the plasmid to give the final recombinant sequence. After receiving the sequences back, we aligned each sequence with the target plasmid again. Once alignment was confirmed, colonies were picked in order to start experimentation.</P><H2 id="Cell-Measurement-Protocol">Cell Measurement Protocol</H2><P>Initially, the OD600 of the overnight cultures was measured. The data was imported into the Normalisation tab of the Excel Data Sheet provided by iGEM. The sheet calculated the amount of medium with antibiotic (liquid LB with chloramphenicol) to add in order to make the solutions reach a target OD600 of 0.02. Under a flame (to provide sterile conditions), the appropriate amount of medium and bacterial solution for each construct was added in 15ml falcon tubes to make a final volume of 10ml. The solutions were mixed and 100μl from each was aliquoted into pre-chilled Eppendorf tubes as a recording for 0h. The rest of the solution was put in a 37C 225rpm incubator. Aliquots were removed and frozen every hour at 1h, 2h, 3h, 4h, 5h and 6h. At the end all samples were loaded into the 96-well plate in the same layout as the one suggested by the iGEM plate reader protocol. A single measurement was then made in the plate reader. The data was recorded in the iGEM Data Excel Sheet and sent to measurement@igem.org.</P><H3 id="Results-and-Discussion-the-iGEM-Protocol">Results and discussion: the iGEM protocol</H3><CENTER> OD600 Measurement </CENTER><CENTER> Relative Fluoresence Measurement</CENTER><CENTER> Absolute Fluorescence Plot</CENTER><P>The OD600 measurements seem to be typical. There is exponential growth between 1h and 4h. Then, the rate of growth decreases. Eventually bacteria reach the stationary phase. All parts follow this except device 1, replicate 1 which seems to continue growing exponentially even at hour 6. As for the fluorescence measurements, a similar pattern is observed. Fluorescence also shows which test device contains the stronger promoter. Promoter strength is in the order of TD1, TD2 and TD3, with TD1 being the strongest. This is in agreement with the strength observed in the Anderson collection. The constitutively expressed GFP in the positive control shows that that promoter has a strength in between TD1 and TD2. As expected, the negative control shows little to no fluorescence.</P><P>The absolute fluorescence seems to conclude the same as relative fluorescence. For all measurements, the fluorescence per cell for TD1 is greater than that of TD2 which is greater than that of TD3. The unusual result is that for each construct, absolute fluorescence decreases over time. One would expect it to be almost constant (as the cells create the same amount of fluorescence as time passes, they just increase in number) or increase with time (initially, cells produce no fluorescence but as they enter the exponential phase, they start increasing their transcription output and so fluorescence increases). The exponential decrease seen by all parts might be due to cells not being able to handle the additional plasmid or due to the fact that the samples were contaminated with cells not carrying the plasmid, which outcompete our cells over time. This would mean that the proportion of cells carrying the fluorescent construct out of the whole population in the sample decreases, and so does the absolute fluorescence.</P><H3 id="Improving-the-iGEM-protocol">Improving the iGEM protocol</H3><P style="font-weight:bold;">Advantages of the iGEM protocol</P><UL class="textLiFont"><LI>Simple to follow</LI><LI>All teams follow this protocol so the results we get will be comparable to other teams' results.</LI><LI>Only required a single measurement in the plate reader which is advantageous in a busy laboratory when a lot of people are trying to use it</LI></UL><P style="font-weight:bold;">Disadvantages of this protocol</P><UL class="textLiFont"><LI>Putting cells on ice did not completely prevent their growth so the expression measured would be higher than it actually was at the time the cells were aliquoted and placed on ice</LI><LI>Very time and resource consuming as had to put everything into a separate testube.</LI><LI>Time delays made the process unreliable as not all cells could be removed simultaneously on the hour.</LI><LI>The transfer of the bacterial cultures from the falcon tube to the pre-chilled testube and subsequently from the testube into the plate can be a source of pipetting error.</LI><LI>Although the protocol encourages the production of two biological repeats to account for biological variation, there are no experimental repeats from the same culture.</LI></UL><P>We propose a similar protocol, which we believe addresses the drawbacks of the iGEM protocol. We encourage the use of our protocol for future interlab measurements as it does not require anything additional from teams that choose to measure the constructs using a plate reader.</P><H3 id="Oxford-iGEM-teams-plate-reader-protocol">Oxford iGEM team’s plate reader protocol</H3><P>The procedure for transformation of all devices and picking colonies is exactly the same as the one used for iGEM's protocol.</P><P>Our measurement protocol makes use of our plate reader’s ability to incubate plates at a set temperature and record OD600 and fluorescence at set intervals. We used the same plate plan as the original iGEM protocol but instead of adding a new sample each hour we did seven repeats from each colony and left them in the plate reader to record OD600 and fluorescence every hour.</P><P style="font-size: 100px"><I>(LEFT) iGEM's plate overview as specified by the provided protocol. (RIGHT) The plate overview for our own protocol. Notice that on the column axis we measure 7 repeats (0-6).For each repeat, measurements at hours 0-6 are made automatically by the plate reader and recorded in an excel spreadsheet.</I></P><P style="font-weight:bold;">Problems addressed by this protocol:</P><UL class="textLiFont"><LI>The plate is constantly incubated; the samples do not cool down as when they are taken out to be put on ice as in iGEM's protocol</LI><LI>We know that the data from the cells corresponds to the exact time the measurements are taken unlike in iGEM's protocol where there is still some cell growth after the cells had been put on ice</LI><LI>We can get many more data points because there are more repeats for each biological sample at each time.</LI><LI>We can also take measurements more often than once every hour</LI><LI>We only had to use one plate so it was easier and faster than filling up multiple testubes. This decreases the probability of pipetting errors and saves lab resources (no testubes used, less pipette tips due to less transfers, etc)</LI></UL><P style="font-weight:bold;">Possible downfalls:</P><UL class="textLiFont"><LI>Because the plate reader continuously monitors and records OD600 and fluorescence for the plate, the instrument would be continuously occupied for 6-7 hours. This might not be ideal if the instrument is extensively used by other lab groups</LI></UL><P style="font-weight:bold;">Results of iGEM Oxford’s protocol</P><P> OD600 Measurement: </P><P> Relative Fluoresence Measurement</P><P> Absolute Fluorescence Plot</P><P><I>OD and fluorescence data was processed in the same way as the formulae on the pre-made iGEM Excel spreadsheet. All graphs were illustrated using MatLab by Iain Dunn and Shu Ishida.</I></P><P style="font-weight:bold;">Results and discussion</P><P>The measurements were followed for 15 hours instead of 6. This allows us to see the stationary phase in both the OD600 and fluorescence measurements. The OD600 data is normal for all parts (exponential phase followed by the stationary phase). Fluorescence agrees with the promoter strength results from the iGEM protocol: TD1 contains the strongest promoter and TD3 the weakest.</P><P>Absolute fluorescence results are better than the interlab data. Again, TD1 shows the greatest fluorescence per cell for all times. It is followed by TD2 and then TD3. This trend is in agreement with the Anderson collection’s promoter strength values. For test devices 2 and 3 we observe what is expected for the absolute fluorescence (a sigmoidal-like trend). Test device 1 shows a sharp decrease but it is then followed by an exponential increase that slows and eventually flatlines.
	</P><P>In conclusion, iGEM Oxford's protocol seems to give better and more reliable results than the iGEM original protocol.</P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><DIV class="container-fluid" id="footer" style="padding-top:25px; padding-bottom:30px; border-top:1px solid #CDCCCE; font-family: avenir, arial">

Made with gooood stuff.  </DIV></DIV></DIV></BODY></HTML>