<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Copper_Chelators skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Copper Chelators</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-9"><SECTION id="GroupA"><H2>Chelator Characteristics</H2><P> As our project was to chelate dietary copper we searched for copper binding and storage proteins. 
We decided that the ideal copper chelator would have these properties:
</P><UL><LI>Should be able to bind multiple copper ions per peptide to increase the efficient use of cell resources. </LI><LI>They should be from the prokaryotic domain because eukaryotic proteins can have expression issues in <I>Escherichia coli.</I></LI><LI>As <I> E. coli </I>naturally deals with copper toxicity by binding copper in the <A href="#" data-toggle="popover1" data-trigger="hover" title="Periplasm" data-content="A space between the outer cell membrane and bacterial cell wall in gram negative bacteria">periplasm</A>, periplasmic proteins may reduce toxicity to the host. </LI></UL><P>Using these criteria we found two copper chelators that we though would be useful. We designed gBlocks, codon optimised for <I>E. coli</I>, containing these parts both alone and linked to super-folder <A href="#" data-toggle="popover1" data-trigger="hover" title="Green Fluorescent Protein" data-content="A protein that emits green light when illuminated with blue to UV light">GFP</A>. This form of the fluorescent protein was used because the standard GFP doesn’t fold particularly well in the periplasm where one of our chelators is intended to travel to. 
All the proteins, both alone and attached to sfGFP, were ordered with <A href="#" data-toggle="popover1" data-trigger="hover" title="His Tag" data-content="Six sequential Histidine residues that bind to NTA columns"> hexa-histidine tag</A> so we could purify them. A concern was raised that the his tag would also weakly bind copper potentially affecting the results. However we decided that increasing the copper-binding would only enhance the protein.  
</P></SECTION><SECTION id="GroupB"><H2>Copper Storage Protein 1</H2><P>
Copper storage protein 1 is a protein discovered in an methane-oxidizing alphaproteobacterium called<I> Methylosinus trichosporium OB3b. </I> (OB3b here stands for “oddball” strain 3b). This bacterium has a high demand for copper for use in its particular <A href="#" data-toggle="popover1" data-trigger="hover" title="Methane Monoxygenase" data-content="An enzyme that oxides methane">methane monoxygenase</A> enzyme. Vita et al* discovered Csp1 in 2015.
</P><DIV id="GroupBSub1"><H3>Csp1</H3><P>
Csp1 is a tetramer of four-helix bundles. Each monomer can bind up to 13 Cu(I) ions meaning that the tetramer binds a maximum of 52 copper ions. Vita et al crystallised Csp1 with and without copper bound. The copper is bound inside the pre-folded helical bundles by Cys residues in contrast to metallothioneins which are unstructured until they fold around metal ion clusters. They found an average copper affinity of approximately 1x10^-7M.
</P><P>
Csp1 has a signal peptide targeting it to the <A href="#" data-toggle="popover1" data-trigger="hover" title="TAT" data-content="An incompletely understood prokaryotic secretion system"> twin arginine translocation pathway</A> (TAT).  This means that it is likely as periplasmic protein. However they also found cytoplasmic homologues in many species challenging their and our assumption that only copper storage occurs in the periplasm due to copper toxicity.
</P><P>
We codon optimised Csp1 to <I>E. coli</I> and replaced the original TAT sequence with a TAT sequence from the <I>E. coli</I> protein <A href="#" data-toggle="popover1" data-trigger="hover" title="CueO" data-content="An E. coli multicopper oxidase enzyme that protects against copper toxicity by oxidising Cu(I) to the less toxic Cu(II)"> CueO</A>, which is also involved in copper regulation. To get Csp1 from the shipping vector to the pBAD expression system for testing the TAT sequence had to be modified but the addition of a serine residue after the initiator methionine. Serine was chosen over other amino acid possibilities because other TAT sequences seemed to have serine in this location. Unfortunately we were not able to demonstrate that Csp1 was transported to the periplasm in microscopy images.

</P></DIV><DIV id="GroupBSub2"><H3>Csp1 sfGFP</H3><P>
Csp1 sfGFP is a form of Csp1 with a C terminal sfGFP tag. The sfGFP has a C terminal hexahistidine tag for purifications.
</P></DIV></SECTION><SECTION id="GroupC"><H2>MymT</H2><DIV id="GroupCSub1"><H3>MymT</H3><P>
MymT is a small prokaryotic <A href="#" data-toggle="popover1" data-trigger="hover" title="metallothein" data-content="Very small Cysteine-rich proteins involved in transport and storage">metallothein</A> discovered in <I>Mycobacterium tuberculosis.</I> It can bind up to 7 Cu ions in a solvent-shielded core but apparently prefers 4-6. It is much smaller than Csp1 and exists as a monomer. It is cytoplasmic. No crystal structure exists but there is a structure of a eukaryotic homologue:
</P></DIV><DIV id="GroupCSub2"><H3>MymT sfGFP</H3><P>
MymT sfGFP is a form of MymT that has a C terminal sfGFP tag.
</P></DIV></SECTION></DIV><NAV class="col-xs-3 bs-docs-sidebar"><UL id="sidebar" class="nav nav-stacked fixed" data-spy="affix" data-offset-top="205"><LI><A href="#GroupA">Chelator Characteristics</A></LI><LI><A href="#GroupB">Copper Storage Protein 1</A></LI><LI><A href="#GroupBSub1">Csp1</A></LI><LI><A href="#GroupBSub2">Csp1 sfGFP</A></LI></UL><LI><A href="#GroupC">MymT</A></LI><LI><A href="#GroupCSub1">MymT</A></LI><LI><A href="#GroupCSub2">MymT sfGFP</A></LI></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>