<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Oxford 2016 - Cure for Copper</TITLE></P><DIV class="container-fluid"><DIV class="collapse navbar-collapse collapse" id="myNavbar" style="background-color: #fff;"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Wilson_Disease">Wilson's Disease</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Parts">Parts Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/CueR_MOI">CueR Mechanism</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Results">Overview and Results</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Chemicals">Chemicals</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sequences">Sequences</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/InterLab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dry lab </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model">Modelling Overview</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelA">Reaction Kinetics</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelB">Promoter Characterisation</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelC">Copper Advection</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Model#ModelD">Bacteria Distribution</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Silver">Silver Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/HP/Gold">Gold Criterion</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Questionnaires">Surveys</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Engagement">Engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">People </A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Oxford/Contact">Contact Us</A></LI><UL class="nav navbar-nav navbar-right"><LI><A href="https://oxfordigem2016.wordpress.com">Blog</A></LI></UL></DIV></DIV><DIV class="container-fluid content-main"><DIV class="row"><NAV class="col-md-3 bs-docs-sidebar navFont"><UL id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330"><LI><A href="#Cloning">Cloning</A></LI><LI><A href="#DNA-Suspension">DNA Suspension</A></LI><LI><A href="#Polymerase-Chain-Reaction-PCR">Polymerase Chain Reaction (PCR)</A></LI><LI><A href="#Agarose-Gel-Production">Agarose Gel Production</A></LI><LI><A href="#Agarose-Gel-Electrophoresis">Agarose Gel Electrophoresis</A></LI><LI><A href="#Agarose-Gel-Visualisation">Agarose Gel Visualisation</A></LI><LI><A href="#Gel-Extraction">Gel Extraction</A></LI><LI><A href="#DNA-Restriction-Enzyme-Digestion">DNA Restriction Enzyme Digestion</A></LI><LI><A href="#DNA-Ligation">DNA Ligation</A></LI><LI><A href="#Competent-Cell-Production">Competent Cell Production</A></LI><LI><A href="#Day-1">Day 1</A></LI><LI><A href="#Day-2">Day 2</A></LI></UL><LI><A href="#Agar-Plate-Production">Agar Plate Production</A></LI><LI><A href="#Antibiotic-Preparation">Antibiotic Preparation</A></LI><LI><A href="#Transformation">Transformation</A></LI><LI><A href="#Picking-Colonies">Picking Colonies</A></LI><LI><A href="#DNA-Plasmid-Extraction-from-Bacterial-Cultures-Miniprep">DNA Plasmid Extraction from Bacterial Cultures (“Miniprep”)</A></LI><LI><A href="#DNA-Sequencing">DNA Sequencing</A></LI><LI><A href="#Protein-Expression-Measurements">Protein Expression Measurements</A></LI><LI><A href="#Plate-Reading-Relative-and-Absolute-Fluorescence">Plate Reading</A></LI><LI><A href="#Day-1Plate">Day 1</A></LI><LI><A href="#Day-2Plate">Day 2</A></LI><LI><A href="#Flow-Cytometry">Flow Cytometry</A></LI><LI><A href="#Day-1Flow">Day 1</A></LI><LI><A href="#Day-2Flow">Day 2</A></LI><LI><A href="#Microscopy">Microscopy</A></LI><LI><A href="#Day-1Microscopy">Day 1</A></LI><LI><A href="#Day-2Microscopy">Day 2</A></LI><LI><A href="#Protein-Characterisation">Protein Characterisation</A></LI><LI><A href="#Protein-Purification">Protein Purification</A></LI><LI><A href="#Buffer-Preparation-ProtPurification">Buffer Preparation</A></LI><LI><A href="#Day-1ProtPurification">Day 1</A></LI><LI><A href="#Day-2ProtPurification">Day 2</A></LI><LI><A href="#Day-3ProtPurification">Day 3</A></LI><LI><A href="#Day-4ProtPurification">Day 4</A></LI><LI><A href="#Dialysis">Dialysis</A></LI><LI><A href="#Day-1Dialysis">Day 1</A></LI><LI><A href="#Day-2Dialysis">Day 2</A></LI><LI><A href="#Day-3Dialysis">Day 3</A></LI><LI><A href="#SDS-PAGE">SDS-PAGE</A></LI><LI><A href="#Buffer-Preparation-SDS-PAGE">Buffer Preparation</A></LI><LI><A href="#SDS-PAGE-Setup">SDS-PAGE Setup</A></LI><LI><A href="#Densitometry">Densitometry</A></LI><LI><A href="#BCS-Assay">BCS Assay</A></LI><LI><A href="#Version-1">Version 1</A></LI><LI><A href="#Version-2">Version 2</A></LI><LI><A href="#Version-3">Version 3</A></LI><LI><A href="#Version-4">Version 4</A></LI><LI><A href="#Version-5">Version 5</A></LI><LI><A href="#Version-6">Version 6</A></LI><LI><A href="#Version-7">Version 7</A></LI><LI><A href="#Version-8">Version 8</A></LI><LI><A href="#Version-9">Version 9</A></LI><LI><A href="#Version-10">Version 10</A></LI><LI><A href="#Capsule-Design-and-Testing">Capsule Design and Testing</A></LI><LI><A href="#Multi-layered-Microcapsule-Production">Multi-layered Microcapsule Production</A></LI><LI><A href="#Microcapsule-Integrity-in-Simulated-Gut-Environments">Microcapsule Integrity in Simulated Gut Environments</A></LI></NAV><DIV class="col-md-9 content-right" style="background-color: #fff;"><DIV class="pageTitle pageTitleBlue">Protocols</DIV><P>Here is a documentation of the protocols we have used in the wet lab for our project.</P><SECTION id="Cloning"><H1 class="pageTitleBlue">Cloning</H1><P>Cloning refers to the group of processes which will take a designed DNA part from powder form to a fully functional DNA sequence that can be read and/or expressed in a bacterial E. coli strain. The following protocols give a detailed account of the process.</P><H2 id="DNA-Suspension">DNA Suspension</H2><UL class="textLiFont"><LI>Open the folder from IDT (Company that constructed the DNA parts) and take out tube</LI><LI>Add MilliQ to make a solution of 10ng/μl (volume varies according to mass of powder)</LI><LI>Vortex to make sure that the power has been properly dissolved</LI><LI>Store in -20°C for future use</LI></UL><H2 id="Polymerase-Chain-Reaction-PCR">Polymerase Chain Reaction (PCR)</H2><UL class="textLiFont"><LI>Defrost DNA templates, primers and NEB Q5® High-Fidelity 2X Master Mix in an ice bucket</LI><LI>Produce the following reaction mix in PCR tubes</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Component</TH><TH>Volume(μl)</TH></TR></THEAD><TBODY><TR><TD>Q5 High-Fidelity 2X Master Mix</TD><TD>12.5</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1.25</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>1.25</TD></TR><TR><TD>1 ng/μl DNA template</TD><TD>1</TD></TR><TR><TD>MilliQ (ultrapure water)</TD><TD>9</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Set the reaction protocol on the PCR machine. This will vary according to the DNA template</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Stage</TH><TH>Temperature (°C)</TH><TH>Time (s)</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>120</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD></TR><TR><TD>Annealing</TD><TD>Annealing temp.*</TD><TD>15</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>30</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>30 per 1kb</TD></TR><TR><TD>HOLD</TD><TD>10</TD><TD>HOLD</TD></TR><TR><TD colspan="3">The protocol is set for x25 cycles</TD></TR></TBODY></TABLE><P>* Calculate using the <A href="http://tmcalculator.neb.com/#!/">NEB Tm calculator.</A></P><UL class="textLiFont"><LI>Remove the tubes from the machine after the procedure has ended</LI><LI>If no further experiments will be executed store the resulting solutions in the fridge (-20C)</LI></UL><H2 id="Agarose-Gel-Production">Agarose Gel Production</H2><P>For a small, 1% gel:</P><UL class="textLiFont"><LI>Weigh 1g of agarose using the electronic balance</LI><LI>Pour 100ml of ½x TBE Buffer into a 200ml Duran bottle</LI><LI>Pour the 1g of agarose into the buffer and mix</LI><LI>Heat the solution in the microwave the “High” power setting for 2 minutes. Make sure that the bottle cap is loosely screwed to relieve pressure build up. Place a large weighing boat underneath the bottle in case of boiling spillage</LI><LI>Take bottle out and swirl to make sure that all the agarose has dissolved</LI><LI>Put the bottle in the 55°C water bath to allow it to cool down for 15-20 minutes</LI></UL><H2 id="Agarose-Gel-Electrophoresis">Agarose Gel Electrophoresis</H2><UL class="textLiFont"><LI>Pour the warm agarose solution into a gel plate in a setting tray with the combs already in place and allow it to set for 20-25 minutes</LI><LI>Once set, remove the combs and transfer the gel plate into the electrophoresis chamber</LI><LI>Flood the chamber with 0.5x TBE buffer until the wells are covered. The gel should be positioned in such a way that the positive electrode is the farthest away from the gel wells, as the DNA is negatively charged and so will migrate towards the positive electrode</LI><LI>Defrost DNA samples to be electrophoresed (e.g. PCR products, plasmid DNA, etc)</LI><LI>Add 20μl of ladder to the first well and 20μl of the DNA in each subsequent well.</LI><LI>Run on 100V for 15-30 minutes, periodically checking the progress of the gel</LI></UL><DIV style=" float: right"><FIGCAPTION>An agarose gel for running PCR products or digested plasmids</FIGCAPTION></DIV><H2 id="Agarose-Gel-Visualisation">Agarose Gel Visualisation</H2><UL class="textLiFont"><LI>Remove the gel plate from the electrophoresis device and pour off any excess buffer</LI><LI>Slide the gel gently into the Ethidium Bromide vat, being careful to avoid any spillage (Dilute Ethidium Bromide is carcinogenic)</LI><LI>Set the vat to shake for 20 minutes so that the gel is stained</LI><LI>Once done, use a spatula and a plastic bowl to remove the gel from the vat. Rinse the gel with water to remove any residual ethidium bromide</LI><LI>Place the gel in the GeneSnap UV Transilluminator machine and open the GeneSnap software on the computer to produce an image</LI><LI>Choose the “agarose gel” option and the dimensions of the gel</LI><LI>Edit, save and/or print the image</LI><LI>If DNA band excision needs to be done place the gel in the desktop UV transilluminator. Wear the protective mask and make sure you don’t have any exposed skin. Turn off the lights and open the UV illuminator. You can see the orange bands (corresponding to intercalated ethidium bromide fluorescing)</LI></UL><H2 id="Gel-Extraction">Gel Extraction</H2><UL class="textLiFont"><LI>After putting the DNA gel on the desktop transilluminator, label 1.5ml plastic testubes with the names of the bands to be excised</LI><LI>Use a razor to cut the band and transfer it into the matching testube</LI></UL><P>We used a slightly modified version of the the QIAGEN gel extraction kit (catalogue no. 28704 or 28706) protocol provided with the kit.</P><UL class="textLiFont"><LI>Notes before starting the protocol</LI><UL><LI>Add ethanol (96-100%) to the PE buffer before its use</LI><LI>All centrifugation steps are carried out at 13000 rpm unless otherwise stated</LI></UL><LI>Excise the target DNA bands from the gel using a razor</LI><LI>Weight the gel slice in a plastic testube. Add 3 volumes of Buffer QG to 1 volume of gel (100mg gel ≈ 100μl)</LI><LI>Incubate at 50°C for 10min at 350-400rpm until the gel is completely dissolved. Every 2 minutes check the gel and vortex to speed up the dissolution. After the gel is dissolved the mixture should be yellow. If it is orange or violet, add 10μl of 3M sodium acetate, pH 5.0 and mix. The solution should now be yellow</LI><LI>Add 1 volume isopropanol to the sample and mix/vortex</LI><LI>Apply the DNA from the testube in a QIAquick column which is in a QIAquick spin tube</LI><LI>Centrifuge for 1 minute and discard flow-through</LI><LI>Add 500μl Buffer QG to the column and centrifuge for 1 min. Discard flow through</LI><LI>Add 750μl of Buffer PE (note: with ethanol dissolved in it) in the column and centrifuge for 1 minute</LI><LI>Take column out of the centrifuge and let it on the bench for 2-5 minutes so that any residual ethanol has evaporated</LI><LI>Spin again for 1 minute to make sure that all ethanol has been removed</LI><LI>Place the QIAquick column in a 1.5ml plastic testube</LI><LI>To elute DNA, add 30μl of dH<SUB>2</SUB>O to the QIAquick column. Let it settle for 1 minute so that DNA has dissolved</LI><LI>Centrifuge for 1 minute</LI><LI>Pipette the flowthrough from the bottom of the testube to the QIAquick column and centrifuge for a second time to make sure you have maximal DNA dissolution</LI><LI>DNA is ready of storage or further experimentation</LI></UL><H2 id="DNA-Restriction-Enzyme-Digestion">DNA Restriction Enzyme Digestion</H2><P>All the restriction enzymes we used are from New England Biolabs (NEB. If using two REs together we checked to make sure the buffer used was compatible with both enzymes (can use the <A href="https://www.neb.com/tools-and-resources/ interactive-tools/double-digest-finder">NEB website</A>to find out which buffer is best:).</P><UL class="textLiFont"><LI>Label testubes with the parts to be digested</LI><LI>Defrost DNA and enzymes on ice</LI><LI>Make the following reaction mixture:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Component</TH><TH>Quantity</TH></TR></THEAD><TBODY><TR><TD>DNA</TD><TD>25 μl for PCR products
					10 μl for plasmids</TD></TR><TR><TD>10 x buffer</TD><TD>5 μl</TD></TR><TR><TD>Enzyme</TD><TD>1 μl each</TD></TR><TR><TD>MilliQ</TD><TD>make up to 50 μl total volume</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>In the case that the gel to be run is diagnostic (i.e. it is just to check band sizes or to check if ligation has occurred correctly) use this reaction mixture:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Component</TH><TH>Quantity</TH></TR></THEAD><TBODY><TR><TD>Plasmid</TD><TD>5 μl</TD></TR><TR><TD>10x buffer</TD><TD>2 μl</TD></TR><TR><TD>Enzyme</TD><TD>0.5 μl each</TD></TR><TR><TD>MilliQ</TD><TD>make up to 20 μl total volume</TD></TR></TBODY></TABLE><H2 id="DNA-Ligation">DNA Ligation</H2><P>In ligation, the molar ratio between the insert and the plasmid should be around 3:1. The following reaction mixture is an example. To determine the concentration of both the plasmid and the insert, the ThermoScientific NanoDrop 1000 machine was used.</P><UL class="textLiFont"><LI>Prepare the following ligation mixture in an PCR tube</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Component</TH><TH>Quantity</TH></TR></THEAD><TBODY><TR><TD>Digested Plasmid</TD><TD>10μl</TD></TR><TR><TD>Digested Instert</TD><TD>24μl</TD></TR><TR><TD>T4 DNA Ligase (Invitrogen)</TD><TD>1μl</TD></TR><TR><TD>T4 DNA Ligase Buffer (Invitrogen)</TD><TD>10μl</TD></TR><TR><TD>MilliQ</TD><TD>5μl</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Prepare the following ligation mixture in an PCR tube</LI></UL><H2 id="Competent-Cell-Production">Competent Cell Production</H2><H3 id="Day-1">Day 1:</H3><UL class="textLiFont"><LI>Using a glass pipette, pour 5ml of liquid low salt LB in a glass testube with 10μl of the E. coli strain to be made competent</LI><LI>Grow overnight in shaking incubator at 37°C and 225 rpm</LI></UL><H3 id="Day-2">Day 2:</H3><UL class="textLiFont"><LI>Transfer 1ml of the overnight culture into 100ml of fresh liquid LB. Incubate into the shaking incubator at 37°C, 225rpm for 2-2.5 hours</LI><LI>Prechill 1.5ml plastic testubes. 20 testubes are needed per 100ml</LI><LI>Take out culture and measure its 600nm optical density (OD) in a spectrophotometer. Blank the apparatus using 1ml of liquid LB in a cuvette. The OD should be between 0.4-0.6. If not, leave in the incubator for longer and recheck the OD until it reaches the target value</LI><LI>Decant culture in two 50ml falcon tubes</LI><LI>Spin in a centrifuge for 20 minutes at 2000rpm and 4°C</LI><LI>Pre-chill buffers TFBI and TFBII</LI><LI>Decant supernatant and put on ice</LI><LI>Gently resuspend the pellets by adding 1ml TFBI (make sure the buffers are chilled!) and by occasionally flicking the bottom of the falcon tube </LI><LI>Add a further 16ml into each falcon tube and mix</LI><LI>Leave on ice for 20 minutes</LI><LI>Spin again at 2000rpm and 4°C for 10 minutes</LI><LI>Discard the supernatant</LI><LI>Gently resuspend pellets in 2ml TFBII</LI><LI>Aliquot 200μl of solution into each 1.5ml testube</LI><LI>Store in the freezer (-80°C)</LI></UL><H2 id="Agar-Plate-Production">Agar Plate Production</H2><P>Agar plates are produced for growth of bacterial colonies and selection of bacteria that have uptaken the target plasmid after transformation.</P><UL class="textLiFont"><LI>Estimate how many plates you will need for your experiment</LI><LI>Melt solid low salt LB agar by adding it in the microwave. Chose the “simmer” power option and heat for 1-2 minutes. Repeat until the gel has melted. Make sure the lid is loosely screwed to alleviate pressure build up</LI><LI>Move the agar in the 55°C water bath to cool down</LI><LI>After that put on the bench to cool down below 55°C for 5 minutes to prevent antibiotic degradation</LI><LI>Add the selection antibiotic to a ratio of 1:1000 (i.e. 1μl of antibiotic per ml of agar)</LI><LI>Mix gently</LI><LI>In the laminar flow hood, pour 25ml of agar in each plastic petri dish</LI><LI>With the lid open, allow the agar to solidify for 30 minutes in the laminar flow hood</LI><LI>Use plate. If extra plates have been produced, store in the cold room.</LI></UL><H2 id="Antibiotic-Preparation">Antibiotic Preparation</H2><UL class="textLiFont"><LI>Weigh appropriate amount of antibiotic powder</LI><LI>Add MilliQ to make a solution. Concentrations are:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Antibiotic</TH><TH>Concentration for <I>E. coli</I></TH></TR></THEAD><TBODY><TR><TD>Ampicillin</TD><TD>100 μg/ml</TD></TR><TR><TD>Chloramphenicol</TD><TD>30μg/ml</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix to ensure total dissolution</LI><LI>Filter sterilise the antibiotic solution using a plastic syringe and a 0.45μm syringe filter.</LI><LI>Store in fridge for future use</LI></UL><H2 id="Transformation">Transformation</H2><P>Plasmids are introduced into the E. coli through a cold/heat shock process. The E. coli cells are then grown in LB growth medium for one hour to allow cells with a plasmid to start making a protein that makes the cells antibiotic resistant. The cells are then spread on solid growth medium (LB-Agar) containing antibiotic and grown overnight at 37°C. Only cells with a plasmid will form colonies.</P><P>Different strains of E. coli are used for different purposes. This protocol has been used to transform NEB-alpha, DH5-alpha and MG1655 strains. If transforming a ligation add no more than 25 ml of ligation to 100 ml of cells.</P><UL class="textLiFont"><LI>Thaw aliquots of competent E. coli cells on ice. Need 100 μl per transformation (one 200 μl aliquot = 2 transformations)</LI><LI>Thaw plasmid DNA</LI><LI>Thaw appropriate antibiotic</LI><LI>Once cells are thawed split into 100 μl</LI><LI>Add 1 μl of plasmid DNA.</LI><LI>Incubate on ice for 30 mins.</LI><LI>(Now is a good time to melt the LB agar to make the plates you’ll need. Two plates per transformation.)</LI><LI>Heat shock (45 s in 42°C water bath, then on ice for 1 min.)</LI><LI>Add 800 μl SOC and incubate at 37°C for 1 hour</LI><LI>Spread 100 μl of the cells onto a plate using sterile glass beads, then spin down the remaining and resuspend in 100 μl. Spread this 100 μl onto a second plate. This should be done in a sterile environment so work near a flame or use the flow cupboard</LI><LI>Incubate plates at 37°C overnight</LI></UL><H2 id="Picking-Colonies">Picking Colonies</H2><UL class="textLiFont"><LI>Take plate with colonies out of the cold room</LI><LI>Using a glass pipette, transfer 5ml of liquid low salt LB into a glass testube and add the appropriate antibiotic to a 1:1000 ratio (5μl)</LI><LI>Using a yellow pipette tip, gently scrap a single colony from the petri dish. Drop the tip into the LB solution</LI><FIGCAPTION>Picked colonies in 5ml of LB</FIGCAPTION><LI>Lastly, incubate in the shaking incubator at 37°C, 225rpm overnight  </LI></UL><H2 id="DNA-Plasmid-Extraction-from-Bacterial-Cultures-Miniprep">DNA Plasmid Extraction from Bacterial Cultures (“Miniprep”)</H2><P>We used the QIAGEN Miniprep kit protocol (catalogue no. 27104 or 27106) provided in the kit.</P><UL class="textLiFont"><LI>Notes before starting the protocol
		</LI><LI>Add LyseBlue (Dye) to buffer P1 at a 1:1000 ratio</LI><LI>Add the RNase A provided in the kit to buffer P1 and mix. Store at 2-8°C after use</LI><LI>Add ethanol (96%-100%) to buffer PE before use</LI><LI>All centrifugation steps where revolutions are not mentioned are carried out at 13000 rpm </LI></UL><LI>Pour 1.5ml of the overnight bacterial solution in a plastic testube. Centrifuge and discard supernatant. Repeat to get a larger pellet and therefore extract more DNA</LI><LI>Resuspend pellet in 250μl of Buffer P1 and transfer the solution to a microcentrifuge tube</LI><LI>Add 250μl of P2 buffer and mix by inverting the tube 4-6 times until the solution becomes blue. Proceed to next step in less than 5 minutes</LI><LI>Add 350μl buffer N3 and mix immediately by inverting the tube again. The solution should turn colourless</LI><LI>Centrifuge for 10 minutes</LI><LI>Apply 800μl of the supernatant in a QIAprep 2.0 spin column by pipetting. Centrifuge for 1 minute and discard the flowthrough</LI><LI>Add 500μl of buffer PB. Spin for 1 minute and discard flow through</LI><LI>Wash the QIAprep 2.0 spin column by adding 750μl of buffer PE (with ethanol in it). Centrifuge for 1 minute and discard flowthrough</LI><LI>Leave on bench for 2-5 minutes so residual ethanol can evaporate. To ensure no residual ethanol is left centrifuge for a minute again and discard any possible flowthrough</LI><LI>Transfer the spin column in a fresh 1.5ml testube</LI><LI>Add 30μl of dH<SUB>2</SUB>O and leave it for a minute so that DNA can dissolve. Centrifuge for 1 minute. Pipette the flowthrough back into the spin column for a second time to maximize DNA dissolution</LI><LI>Use nano-drop to check DNA concentration if needed. Blank with dH<SUB>2</SUB>O first and then add 2μl of the DNA to be measured</LI><LI>Store at -20°C.</LI><H2 id="DNA-Sequencing">DNA Sequencing</H2><P>SourceBioscience was the company used to sequence our constructed plasmids. This is done to check that the end construct is what you intended it to be and there aren’t any mutated bases or unwanted insertions.</P><UL class="textLiFont"><LI>Measure the concentration of the DNA sample</LI><LI>Its concentration should be around 100ng/μl. If needed dilute it and remeasure its concentration</LI><LI>Aliquot 10μl of the DNA to be sequenced</LI><LI>Do the same for the forward and backward primers that are to be used for the sequencing. Their concentration needs to be 3.2ng/μl. (This might not be necessary if you have already sent primers with previous orders)</LI><LI>Make and online order and put the necessary testubes (DNA, primers) in a folder with the order number. Put the folder in the dropbox</LI></UL></SECTION><SECTION id="Protein-Expression-Measurements"><H1 class="pageTitleBlue">Protein Expression Measurements</H1><P>In order to show that our synthetic bacteria have a fully functional part that works, expression measurements were made. The construct was usually linked to superfold GFP or an RFP variant which fluoresces. The fluorescent intensity can be measured in different ways and is directly related to the expression frequency of the construct (the more intense the signal the more expression you have). For most parts plate reader, flow cytometry and microscopy experiments were done. </P><H2 id="Plate-Reading-Relative-and-Absolute-Fluorescence">Plate Reading (Relative and Absolute Fluorescence)</H2><P>A plate reader is able to take optical density (OD600nm) and fluorescent measurements over time. OD is a measure of bacterial growth over time and fluorescence a measure of protein expression over time.</P><H3 id="Day-1Plate">Day 1:</H3><UL class="textLiFont"><LI>Pick 4 colonies for the part(s) to be tested, positive control and negative control (12 in total or 16 if running two parts simultaneously)</LI><LI>Put in glass testubes with 5ml of liquid low salt LB and 5μl of appropriate antibiotic</LI><LI>Incubate overnight in 37°C, 225rpm incubator</LI></UL><H3 id="Day-2Plate">Day 2:</H3><UL class="textLiFont"><LI>Generate 200mM of Copper stock by dissolving 0.5g of copper salt (Copper (II) pentahydrate CuO<SUB>4</SUB>  : 5H<SUB>2</SUB>O, MW: 249.69g/mol) in MilliQ and mix thoroughly</LI><LI>Then make a 10mM stock by diluting 1ml of the 200mM stock in 19ml of MilliQ</LI><LI>Generate tubes of 12 different copper concentrations as follows:</LI></UL><TABLE style="margin: auto;"><TBODY><TR><TD>Tube concentration in mM</TD><TD>NC (0)</TD><TD>PC (200)</TD><TD>0</TD><TD>0.1</TD><TD>0.5</TD><TD>1</TD><TD>5</TD><TD>10</TD><TD>50</TD><TD>100</TD><TD>150</TD><TD>200</TD></TR><TR><TD>Volume of 200mM to add to tube (μl)</TD><TD>120</TD><TD>30</TD><TD>60</TD><TD>90</TD><TD>120</TD></TR><TR><TD>Volume of 10mM to add to tube (μl)</TD><TD>1.2</TD><TD>6</TD><TD>12</TD><TD>60</TD><TD>120</TD></TR><TR><TD>Volume of MilliQ to add to tube (μl)</TD><TD>120</TD><TD>120</TD><TD>118.8</TD><TD>114</TD><TD>108</TD><TD>60</TD><TD>90</TD><TD>60</TD><TD>30</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Transfer into 1.2ml deep-well plates:</LI><UL><LI>10μl of each copper concentration in the order seen above in each row (i.e. row 1 1st well will have NC, row 1 well 2 will have PC and so on)</LI><LI>10μl of overnight culture in each well. Each row should have a different repeat. Wells 1 and 2 will have NC and PC accordingly and the rest wells in the row will have the part to be tested</LI><LI>980μl of low salt liquid LB medium with antibiotic (ratio: 1μl of antibiotic per ml of LB) in each well</LI></UL><LI>Use a blank deep-well plate with the same weight as the plate prepared to centrifuge it so that the solutions in each well are properly mixed. Centrifuge for 5 seconds at 1000rpm</LI><LI>Take the plate out and use a multi-tip pipette to transfer 200μl from each well into a plate reader compatible plate (COSTAR 96 plates)  </LI><LI>Close the lid and transfer the plate into the plate reader</LI><LI>Adjust the temperature to 37°C and make a script with 2 protocols, one for OD and one for fluorescent measurements</LI><UL><LI>OD protocol: 600nm, 200rpm double orbital shaking</LI><LI>Fluorescence protocol: excitation filter 485-12nm, emission filter 520nm, bottom optic, 200rpm double orbital shaking</LI></UL><LI>Run the script for 12 hours taking measurements every 10 minutes</LI></UL><P>NOTE: The same protocol was used to measure protein expression in pBAD plasmids which require induction via L-arabinose and not copper. Arabinose concentrations used were the same.</P><H2 id="Flow-Cytometry">Flow Cytometry</H2><P>Flow cytometry takes fluorescent measurements of individual bacterial cells in a population. This is done to ensure that the fluorescence observed is due to each bacterium fluorescing at approximately the same intensity and not due to some bacteria fluorescing a lot and some not at all. </P><H3 id="Day-1Flow">Day 1:</H3><UL class="textLiFont"><LI>Pick 1 colony for the part to be tested, positive control and negative control (3 in total)</LI><LI>Put in glass testubes with 5ml of liquid low salt LB and 5μl of appropriate antibiotic</LI><LI>Incubate overnight in 37°C, 225rpm incubator </LI></UL><H3 id="Day-2Flow">Day 2:</H3><P>Obtain a COSTAR96 plate and prepare as follows:</P><TABLE style="margin: auto;"><TBODY><TR><TD>PC - MilliQ</TD><TD>NC - MilliQ</TD><TD>PC - 2.000mM</TD><TD>NC - 2.000mM</TD></TR><TR><TD>Test - MilliQ</TD><TD>Test – 0.001mM</TD><TD>Test – 0.005mM</TD><TD>Test – 0.010mM</TD><TD>Test – 0.050mM</TD><TD>Test – 0.075mM</TD><TD>Test – 0.100mM</TD><TD>Test – 0.250mM</TD><TD>Test – 0.500mM</TD><TD>Test – 1.000mM</TD><TD>Test – 1.500mM</TD><TD>Test – 2.000mM</TD></TR><TR><TD>Test 2 - MilliQ</TD><TD>Test 2 – 0.001mM</TD><TD>Test 2 – 0.005mM</TD><TD>Test 2 – 0.010mM</TD><TD>Test 2 – 0.050mM</TD><TD>Test 2 – 0.075mM</TD><TD>Test 2 – 0.100mM</TD><TD>Test 2 – 0.250mM</TD><TD>Test 2 – 0.500mM</TD><TD>Test 2 – 1.000mM</TD><TD>Test 2 – 1.500mM</TD><TD>Test 2 – 2.000mM</TD></TR><TR><TD>Test 3 - MilliQ</TD><TD>Test 3 – 0.001mM</TD><TD>Test 3 – 0.005mM</TD><TD>Test 3 – 0.010mM</TD><TD>Test 3 – 0.050mM</TD><TD>Test 3 – 0.075mM</TD><TD>Test 3 – 0.100mM</TD><TD>Test 3 – 0.250mM</TD><TD>Test 3 – 0.500mM</TD><TD>Test 3 – 1.000mM</TD><TD>Test 3 – 1.500mM</TD><TD>Test 3 – 2.000mM</TD></TR><TR><TD>Test 4 - MilliQ</TD><TD>Test 4 – 0.001mM</TD><TD>Test 4 – 0.005mM</TD><TD>Test 4 – 0.010mM</TD><TD>Test 4 – 0.050mM</TD><TD>Test 4 – 0.075mM</TD><TD>Test 4 – 0.100mM</TD><TD>Test 4 – 0.250mM</TD><TD>Test 4 – 0.500mM</TD><TD>Test 4 – 1.000mM</TD><TD>Test 4 – 1.500mM</TD><TD>Test 4 – 2.000mM</TD></TR><TR><TD>Test 5 - MilliQ</TD><TD>Test 5 – 0.001mM</TD><TD>Test 5 – 0.005mM</TD><TD>Test 5 – 0.010mM</TD><TD>Test 5 – 0.050mM</TD><TD>Test 5 – 0.075mM</TD><TD>Test 5 – 0.100mM</TD><TD>Test 5 – 0.250mM</TD><TD>Test 5 – 0.500mM</TD><TD>Test 5 – 1.000mM</TD><TD>Test 5 – 1.500mM</TD><TD>Test 5 – 2.000mM</TD></TR><TR><TD>Test 6 - MilliQ</TD><TD>Test 6 – 0.001mM</TD><TD>Test 6 – 0.005mM</TD><TD>Test 6 – 0.010mM</TD><TD>Test 6 – 0.050mM</TD><TD>Test 6 – 0.075mM</TD><TD>Test 6 – 0.100mM</TD><TD>Test 6 – 0.250mM</TD><TD>Test 6 – 0.500mM</TD><TD>Test 6 – 1.000mM</TD><TD>Test 6 – 1.500mM</TD><TD>Test 6 – 2.000mM</TD></TR><TR><TD>Test 7 - MilliQ</TD><TD>Test 7 – 0.001mM</TD><TD>Test 7 – 0.005mM</TD><TD>Test 7 – 0.010mM</TD><TD>Test 7 – 0.050mM</TD><TD>Test 7 – 0.075mM</TD><TD>Test 7 – 0.100mM</TD><TD>Test 7 – 0.250mM</TD><TD>Test 7 – 0.500mM</TD><TD>Test 7 – 1.000mM</TD><TD>Test 7 – 1.500mM</TD><TD>Test 7 – 2.000mM</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Well contents:</LI><UL><LI>2µl overnight bacterial culture (eg. positive control [PC], negative control [NC], test colony [Test] etc.)</LI><LI>2µl additive (Cu<SUP>2+</SUP> solution or Arabinose solution) to final concentration noted</LI><LI>196µl liquid low salt LB media w/ 1:1000 appropriate antibiotic</LI></UL><LI>Place into plate reader/incubator (FLUOstar Omega - BMG Labtech) and grow for 2-4hrs at 37°C with 200rpm double-orbital shaking. Monitor absorbance at 600nm (700nm for bacteria expressing RFP due to absorption overlap) every 4 mins</LI><LI>Turn on the flow cytometer (Attune NxT - Life Technologies) and create a new experiment</LI><LI>Run a 1.5mL Eppendorf tube containing 1.0mL PBS to ensure that the lines are not contaminated</LI><LI>Remove plate while bacteria are in exponential growth phase</LI><LI>Immediately place plate on ice</LI><LI>Prepare 1.5mL Eppendorf tubes with 1.0mL PBS</LI><LI>Place 2.0µL bacterial culture from each well into a separate Eppendorf tubes containing 1.0mL PBS</LI><UL><LI>After preparing each tube, immediately vortex and place in flow cytometer</LI><LI>Gate on SSC-H and FSC-H to select bacteria</LI><LI>Plot histogram of BL1-H for GFP fluorescence and YL2-H for RFP fluorescence</LI></UL><LI>Analyze .fcs data files in FlowJo</LI><UL><LI>Gate on SSC-H and FSC-H to select bacteria</LI><LI>Plot histogram of BL1-H for GFP fluorescence and YL2-H for RFP fluorescence</LI></UL><LI>Prepare half-offset histograms of all concentrations of the additive on your test cells and with MilliQ on NC cells</LI><UL><LI>Use modal normalization</LI></UL></UL><H2 id="Microscopy">Microscopy</H2><P>Microscopy is used for visual confirmation of the plate reader experiments and flow cytometer experiments and to view the protein cellular distribution.</P><H3 id="Day-1Microscopy">Day 1:</H3><UL class="textLiFont"><LI>Pick 1 colony for the part to be tested, positive control and negative control (3 in total)</LI><LI>Put in glass testubes with 5ml of liquid low salt LB and 5μl of appropriate antibiotic</LI><LI>Incubate overnight in 37°C, 225rpm incubator </LI></UL><H3 id="Day-2Microscopy">Day 2:</H3><UL class="textLiFont"><LI>Take 100ul of the overnight culture into 5ml of the LB with the correct copper/arabinose concentration.</LI><LI>Grow at 37°C until the OD 600 is between 0.4-0.6</LI><LI>Melt 100 ml of 1% agarose made up with MilliQ (microwave at high power for 2 minutes).</LI><LI>While it’s still hot add 200ul (using a 1ml pipette) onto a microscopy slide between tow coverslips. Top with another coverslip and press down.</LI><LI>When its set, (a few seconds) slide off the top coverslip, cut away excess agar then add 100ul of the bacterial culture. Top with a cover slip.</LI><LI>Turn on the filters and lamp on the microscope and ensure that the settings are correct.</LI><LI>Find the correct focal plane for the cells.  Move around the slide until you can find a reasonable number of cells. Focus on the cells then capture an image of of both the DIC and fluorescence channels. Save in .tif format.</LI><LI>For faintly fluorescent cells better contrast can be obtained by precipitating 1ml of the grown cells by centrifugation a 13.3 rpm for 3 mins then re-suspension in 200μl of PBS buffer. This also concentrates the cells making them easier to find.</LI><LI>Images were interpreted using the free ImageJ software then the images cropped to 250px by 250px. The images were saved as the composite image of DIC and fluorescence channels and the two separately.</LI></UL></SECTION><SECTION id="Protein-Characterisation"><H1 class="pageTitleBlue">Protein Characterisation</H1><P> This set of protocols describes the methods used to extract our protein from cells and to characterise chelation when exposed to copper. 
</P><H2 id="Protein-Purification">Protein Purification</H2><P>In order to test <I>in vitro</I> the efficacy of chelation of our proteins we had to purify the proteins from a bacterial culture. </P><H3 id="Buffer-Preparation-ProtPurification">Buffer preparation</H3><P>Buffers should be prepared on Day 1 of the protocol</P><UL class="textLiFont"><LI>Lysis buffer for 1 litre:</LI><UL><LI>100% Glycerol (100ml)</LI><LI>1M Tris (pH8) (50ml)</LI><LI>5M NaCl (30ml)</LI><LI>MilliQ (up to 1litre)</LI><LI>Add EDTA free protease inhibitors (1 tablet per 50ml of lysis buffer. Found in the small fridge at the back of the lab)</LI></UL></UL><UL class="textLiFont"><LI>Elution buffer for 1 litre</LI><UL><LI>Same as lysis buffer but with addition of imidiazole (34g for a 500mM solution)</LI></UL></UL><P>NOTES:</P><UL class="textLiFont"><LI>Lysis and elution buffers should be adjusted to pH8 after making them using the pH probe</LI><LI>Lysis and elution buffers should be stored in the cold room when they are not used. Also during use, keep on ice.</LI></UL><H3 id="Day-1ProtPurification">Day 1:</H3><UL class="textLiFont"><LI>Pick colonies and put in 5ml of LB and 5μl of appropriate antibiotic</LI><LI>Grow overnight in incubator (37°C, 225rpm)</LI></UL><H3 id="Day-2ProtPurification">Day 2:</H3><UL class="textLiFont"><LI>In a 1L flask add:</LI><UL><LI>2 large liquid LB (around 500ml each) bottles</LI><LI>1ml of appropriate antibiotic (1:1000 dilution)</LI><LI>2ml of overnight culture from Day 1</LI><LI>Any extra chemicals (E.g. for the pBAD plasmids you need arabinose to induce expression of your construct)</LI></UL></UL><H3 id="Day-3ProtPurification">Day 3:</H3><UL class="textLiFont"><LI>Take the 1L flask out of the 37°C 225rpm incubator</LI><LI>Take 2 plastic bottles 300ml each, and pour half in each</LI><LI>Balance them using the balance at the end of the lab</LI><UL><LI>Use water if the difference is small to balance the weight</LI><LI>Make sure that they have almost the same weight</LI></UL><LI>Put in large centrifuge</LI><UL><LI>4500 rpm</LI><LI>5°C</LI><LI>30 minutes</LI><LI>JLA 8.1000 rotor and lid (in cold room)</LI><LI>Make sure the centrifuge is safely plugged in</LI><LI>Seal with the lid and screw the top</LI></UL><LI>Take out of the centrifuge and decant liquid</LI><LI>Add 50ml of lysis buffer </LI><LI>Vortex until pellet has been resuspended</LI><LI>Pour in 50ml Falcon tubes</LI><LI>Balance  (add water)</LI><LI>Centrifuge again</LI><UL><LI>20-25 min</LI><LI>5°C</LI><LI>3100 rpm</LI></UL><LI>Decant liquid (carefully and slowly)</LI><LI>Store at -80°C</LI></UL><H3 id="Day-4ProtPurification">Day 4:</H3><DIV style="float: right;"><FIGCAPTION>An NTA collumn with bound MymTsfGFP illuminated with a UV torch</FIGCAPTION></DIV><UL class="textLiFont"><LI>Take Falcon tubes out of the freezer</LI><LI>Incubate at a heating block for 10 minutes at 37°C</LI><LI>Take out and put on an ice bucket</LI><LI>Sonicate each Falcon tube for (20s pulse at 100% power and 20s cooldown time) x 6; therefore 4 minutes per tube</LI><LI>Setup a nickel column</LI><UL><LI>Break white tip, replace with yellow tip</LI><LI>Add nickel beads in fridge </LI><LI>Allow them to settle, let liquid drip, but don’t allow beads to dry. Keep topping up lysis buffer</LI></UL><LI>Centrifuge for 20 minutes, 184krpm in JA-20 centrifuge.</LI><LI>Take out and pour supernatant in 2 new plastic tubes. Repeat the centrifugation with new tubes for another 20 minutes. </LI><LI>Pour the supernatant in syringe and filter sterilise in 2 Falcon tubes using the 0.45μm syringe filter</LI><LI>Wash the nickel column with lysis buffer for a final time</LI><LI>Add sample gently with plastic dripper on the top of the column</LI><LI>After the green line (fluorescent protein) reaches the bottom of the tube top up the column with lysis buffer. Repeat for a second time</LI><LI>Make dilutions of the 500mM stock elution buffer of 10mM, 25, 50, 75mM, 100 + 200mM solutions (dilute with lysis buffer and not water)</LI><LI>According to which dilution the protein starts eluting (around 75mM) start collecting protein in testubes and store in ice.</LI></UL><H2 id="Dialysis">Dialysis</H2><P>Dialysis of protein samples is done to remove the elution buffer from the protein solution. It is done after the protein has been eluted from the affinity chromatography column</P><H3 id="Day-1Dialysis">Day 1 <I>(same as Day 3 of the protein purification protocol):</I></H3><UL class="textLiFont"><LI>Set up 2x2L plastic containers with dialysis buffer (same as lysis buffer but double the amount) and put in cold room to cool down</LI><LI>Pour the testubes that fluoresce the most in a dialysis bag using a pipette tip</LI><LI>Seal the dialysis bag and put it in one of the dialysis buffer containers. Transfer the whole thing in the cold room on the magnetic stirrer. Leave the extra container in the cold room as well. Cover both with aluminium foil </LI><LI>Leave overnight</LI></UL><H3 id="Day-2Dialysis">Day 2:</H3><UL class="textLiFont"><LI>Change the dialysis buffer (the 2nd batch you have prepared)</LI><LI>Leave overnight</LI></UL><H3 id="Day-3Dialysis">Day 3:</H3><UL class="textLiFont"><LI>Pre chill some eppendorfs on ice</LI><LI>Aliquot the contents of the dialysis bag in the test tubes</LI><LI>Store at -20°C (fridge) or start experiments with protein</LI></UL><H2 id="SDS-PAGE">SDS-PAGE</H2><P>SDS-PAGE was used to check if protein samples were pure after purification.</P><H3 id="Buffer-Preparation-SDS-PAGE">Buffer Preparation</H3><UL class="textLiFont"><LI>20x RunBlue SDS Running buffer:</LI><UL><LI>71.7g of tricine (free base)</LI><LI>72.6g of Tris (free base)</LI><LI>10g of SDS</LI><LI>Make bottle upto 500ml with MilliQ</LI></UL></UL><H3 id="SDS-PAGE-Setup">SDS-PAGE Setup:</H3><UL class="textLiFont"><LI>Defrost protein(s) in ice</LI><LI>Go to SnapGene and copy the protein coding bases of the target protein (+linker and GFP if it is on the construct)</LI><LI>Translate using the ExPasy translate tool ( http://web.expasy.org/translate/ )</LI><LI>Copy the in-frame amino acid sequence and paste it into the ProtParam tool ( http://web.expasy.org/protparam/ )</LI><LI>Record:</LI><UL><LI>No. of amino acids</LI><LI>Molecular Weight</LI><LI>Extinction Coefficient</LI><LI>Abs 0.1% (1 g/l)</LI></UL><LI>Nanodrop the protein sample at A=280nm </LI><UL><LI>Blank it with lysis buffer</LI><LI>On the program choose: sample type 1Abs=1mg/ml (equivalent 0.1% and 1g/l)</LI><LI>Measure (2μl of protein) and record</LI><LI>Divide the value on screen by the recorded 0.1%Abs value from expasy</LI><LI>Then do: nanodrop value/expasy value to give you the amount of protein (mg/ml or ug/ul)</LI></UL><LI>Make up your protein solution (each well will hold 5-15μg of proteins so if the protein is too concentrated dilute with lysis buffer)</LI><LI>Add loading dye (called 4x LDS sample buffer)</LI><LI>The dye should make ¼ of your total solution (e.g. 15μl of protein and 5 of dye or 18μl of protein and 6 of dye)</LI><LI>Leave each protein sample on the bench (at 25°C) for ~1hr. This will allow the dye to unfold the protein but not the GFP fluorophore.</LI><LI>Dilute the 20x SDS buffer to 1x. You should have 800ml at the end (so 40ml of stock and 760 of milliQ)</LI><LI>Open a precast gel and rinse it with water to get rid of the liquid in the bag.</LI><LI>Put gel in SDS PAGE container. The taller side of the gel should be facing you and the shorter the inside of the SDS PAGE container</LI><LI>Make sure it is properly fitted depth wise and seal it with the 2 plastic clamps. The apparatus can hold 2 gels, in the other put a white plastic dummy sheet.</LI><LI>Add the 1x buffer first in the gel compartment. Make sure that it covers the wells. The buffer level should be above the wells but below the outer tall side of the gel.</LI><LI>Add the rest of the buffer in the outer compartment. It should be approximately half filled</LI><LI>Take a 2-20 pipette (yellow) and use the fine tips (blue square box on the gel electrophoresis bench).</LI><LI>Take some liquid from the top and then inject it in the inside of each well. This is done to rinse the inside of the wells with buffer.</LI><LI>Load the protein ladder (called BenchmarkTM protein ladder by). Load 10μl in well 1</LI><LI>Load your samples in the following wells (TIP: push the fine long tip of the pipette towards the “tall” side of the gel and then slowly move downwards. Fill the well slowly).</LI><LI>Close with lid. The setup should look like this:</LI><FIGCAPTION>The SDS-PAGE setup</FIGCAPTION><LI>Run at 180V until the bands reach the bottom of the container (30-40 mins, maybe a bit less)</LI><LI>Once done take gel out and clean the container.</LI><LI>Take a glass square dish and a pair of scissors.</LI><LI>Break the glass-plastic surrounding the gel by shoving the tip of one of the scissor on the side and pull apart with caution.</LI><LI>Once open put the gel in the square dish.</LI><LI>Throw the glass-plastic in the glass bin</LI><LI>Put the dish in the gel visualisation machine (the same as the one for DNA gels) and visualise the fluorescence.</LI><LI>Take out. Add 20ml of InstaBlue dye. Close the square dish and put it on the vat machine to spread the dye over the gel. Put it for 25-30 minutes.</LI><LI>Gel is visualised. Use the ladder reference to compare the bands.</LI></UL><H3>DENATURATION SDS GEL: </H3><UL class="textLiFont"><LI>Same procedure but you don’t need to see the fluorescence. So instead of leaving your proteins for 1hr at 25, you boil them at 70°C for 15 minutes. The loading dye is in the sample while you heat it up. This is used as a diagnosic gel, to see if the proteins you have collected after protein purification match the target proteins in your constructs.</LI></UL><H2 id="Densitometry">Densitometry</H2><P> Fluorescence in SDS-PAGE gels is converted to pixel density when saved as a picture. The pixel intensity is proportional to the intensity of the band so a high fluorescence band will have pixels with higher intensity than a low fluorescence band. Processing a picture of an SDS gel can therefore be used to compare fluorescence between different bands. This can give information about the amount of protein in each band. For image processing we used <A href="https://imagej.nih.gov/ij/download.html">ImageJ</A>which can be found online for free. The procedure is as follows:</P><UL class="textLiFont"><LI>Open Image J </LI><LI>Go to <I>File &gt; Open...</I> and choose the image you want to measure</LI><LI>Go to <I>Image &gt; Type &gt; 32-Bit</I></LI><LI>Chose the &quot;Rectangular&quot; selection on the toolbar</LI><LI>Draw a rectangle around the first band you want you want to measure</LI><LI>Go to <I>Analyze &gt; Gels &gt; Select First Lane</I> in order to tag your selected band. The rectangular area around it will be labelled &quot;1&quot;</LI><LI>Hover over &quot;1&quot;, click and drag. The box will be copied</LI><LI>Move the box to the next band you want analysed</LI><LI>Go to <I>Analyze &gt; Gels &gt; Select Next Lane</I></LI><LI>Repeat for all the lanes you want analysed</LI><LI>Go to <I>Analyze &gt; Gels &gt; Plot Lanes</I>. A graph for each selected band will be plotted.</LI><LI>Select the area under the peak in each of the graphs using the &quot;Straight&quot; button on the toolbar</LI><LI>Using the &quot;Wand&quot; tool on the toolbar click on the area under the graph. This will give it a value. All the data will be displayed in a new window</LI><LI>The data can be exported to Excel for further analysis</LI></UL><H2 id="BCS-Assay">BCS Assay</H2><P> The BCS Assay was one of the methods we used try to quantify copper chelation of our proteins <I>in vivo</I>. BCS absorbance was measured in the abscence of copper bound to it. A known copper concentration was made, a reductand was added to make all copper ions into the Cu<SUP>+1</SUP> variant and our proteins were put in it. Chelation would remove an amount of copper. BCS was added to the solution afterwards to chelate fully the remaining copper. Absorbance of BCS was remeasured. A difference in absorbance between the BCS<SUB>(free)</SUB> and the copper-chelated BCS to give the final copper concentration. This value compared to the initial copper concentration made would give the amound of copper absorbed by our protein. The protocols below are different versions (2-9) of an initial protocol (1) we devised. The differ in reductant used, copper concentrations and methods used to measure absorbance.</P><H3 id="Version-1">Version 1</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice.</LI><LI>Defrost proteins on ice</LI><LI>Prepare 10µM Bovine Serum Albumin solution in protein buffer from lyophilized powder.</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Prepare 2x200µL samples of Nickel affinity beads.</LI><UL><LI>Collect into Eppendorf</LI><LI>Add 1mL PBS.</LI><LI>Centrifuge to pellet beads and remove supernatant</LI><LI>Repeat three previous steps once more</LI></UL><LI>Make 6 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>BSA Control</TH><TH>Negative Control</TH><TH>Positive Control 1</TH><TH>Positive Control 2</TH><TH>Positive Control 3</TH><TH>Positive Control 4</TH><TH>Positive Control 5</TH><TH>Positive Control 6</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>300µL mg</TD><TD>300µL BSA</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>100µM</TD><TD>100µM</TD><TD>0µM</TD><TD>50µM</TD><TD>70µM</TD><TD>80µM</TD><TD>90µM</TD><TD>100µM</TD><TD>200µM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>300µL</TD><TD>300µL</TD><TD>1mL</TD><TD>1mL</TD><TD>1mL</TD><TD>1mL</TD><TD>1mL</TD><TD>1mL</TD><TD>1mL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 3 or 10µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Add 200µL protein solution to each Eppendorf containing beads</LI><LI>Vortex and place on ice for 15 mins</LI><LI>Centrifuge and fill 1mL cuvettes with</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein Solution</TD><TD>100µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>10µL</TD></TR><TR><TD>pH 7.0 Buffer 1M</TD><TD>900µL</TD></TR><TR><TD>Total Volume</TD><TD>1020µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm</LI></UL><H3 id="Version-2">Version 2</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice.</LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as mg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Prepare 2x200µL samples of Nickel affinity beads</LI><UL><LI>Collect into Eppendorf</LI><LI>Add 1mL PBS</LI><LI>Centrifuge to pellet beads and remove supernatant</LI><LI>Repeat a-c once more</LI></UL><LI>Make 9 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>GFP Control</TH><TH>Negative Control</TH><TH>Positive Control 1</TH><TH>Positive Control 2</TH><TH>Positive Control 3</TH><TH>Positive Control 4</TH><TH>Positive Control 5</TH><TH>Positive Control 6</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>250µL mg</TD><TD>250µL GFP</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>50µM</TD><TD>0µM</TD><TD>10µM</TD><TD>20µM</TD><TD>30µM</TD><TD>40µM</TD><TD>50µM</TD><TD>75µM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD><TD>250µL</TD></TR></TBODY></TABLE>&lt;<P style="text-align:right" class="footnote"><SUP>*</SUP>total 2.5µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Add 200µL protein solution to each Eppendorf containing beads</LI><LI>Vortex and place on ice for 15 mins</LI><LI>Centrifuge and fill 1mL cuvettes with</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein Solution</TD><TD>200µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>10µL</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>800µL</TD></TR><TR><TD>Total Volume</TD><TD>1020µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm</LI></UL><H3 id="Version-3">Version 3</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice</LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as mg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Make 7 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>Test Protein wo/ Cu</TH><TH>GFP Control</TH><TH>GFP Control wo/ Cu</TH><TH>Negative Control</TH><TH>Positive Control 1</TH><TH>Positive Control 2</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>210µL mg</TD><TD>210µL mg</TD><TD>210µL GFP</TD><TD>210µL GFP</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD><TD>0µM</TD><TD>10µM</TD><TD>50µM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>210µL</TD><TD>210µL</TD><TD>210µL</TD><TD>210µL</TD><TD>210µL</TD><TD>210µL</TD><TD>210µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 2.1µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Fill 1mL cuvettes with</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein Solution</TD><TD>200µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>10µL</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>800µL</TD></TR><TR><TD>Total Volume</TD><TD>1020µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm</LI></UL><H3 id="Version-4">Version 4</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice</LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as mg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Make 15 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>Test Protein wo/ Cu</TH><TH>GFP Control</TH><TH>GFP Control wo/ Cu</TH><TH>Negative Control</TH><TH>Positive Control 1</TH><TH>Positive Control 2</TH><TH>Positive Control 3</TH><TH>Positive Control 4</TH><TH>Positive Control 5</TH><TH>Positive Control 6</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>100µL mg</TD><TD>100µL mg</TD><TD>100µL GFP</TD><TD>100µL GFP</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD><TD>0µM</TD><TD>5µM</TD><TD>10µM</TD><TD>20µM</TD><TD>30µM</TD><TD>40µM</TD><TD>50µM</TD></TR><TR><TD>[DTT]<SUP>**</SUP></TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM</TD><TD>50mM50mM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Positive Control 7</TH><TH>Positive Control 8</TH><TH>Negative Control wo/DTT</TH><TH>Positive Control wo/DTT</TH></TR></THEAD><TBODY><TR><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>75µM</TD><TD>100µM</TD><TD>0µM</TD><TD>100µM</TD></TR><TR><TD>50mM</TD><TD>50mM</TD><TD>0mM</TD><TD>0mM</TD></TR><TR><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 1µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 1µL of 5M DTT stock or MilliQ for 0mM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Fill 1mL cuvettes with</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein Solution</TD><TD>100µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>10µL</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>900µL</TD></TR><TR><TD>Total Volume</TD><TD>1020µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm</LI></UL><H3 id="Version-5">Version 5</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on </LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as mg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Prepare solution of lysozyme with same concentration at mg sample by dilution with protein buffer from lyophilized powder</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Make 20 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>Test Protein wo/ Cu</TH><TH>GFP Control</TH><TH>GFP Control wo/ Cu</TH><TH>Lysozyme Control</TH><TH>Lysozyme Control wo/Cu</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>100µL mg</TD><TD>100µL mg</TD><TD>100µL GFP</TD><TD>100µL GFP</TD><TD>100µL lys</TD><TD>100µL lys</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>25µg/mL</TD><TD>25µg/mL</TD><TD>25µg/mL</TD><TD>25µg/mL</TD><TD>25µg/mL</TD><TD>25µg/mL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein 2</TH><TH>Test Protein wo/ Cu 2</TH><TH>GFP Control 2</TH><TH>GFP Control wo/ Cu 2</TH><TH>Lysozyme Control 2</TH><TH>Lysozyme Control wo/Cu 2</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>100µL mg</TD><TD>100µL mg</TD><TD>100µL GFP</TD><TD>100µL GFP</TD><TD>100µL lys</TD><TD>100µL lys</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>0µg/mL</TD><TD>0µg/mL</TD><TD>0µg/mL</TD><TD>0µg/mL</TD><TD>0µg/mL</TD><TD>0µg/mL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Negative Control 1</TH><TH>Negative Control 2</TH><TH>Positive Control 1 (1)</TH><TH>Positive Control 1 (2)</TH><TH>Positive Control 2 (1)</TH><TH>Positive Control 2 (2)</TH><TH>Positive Control 3 (1)</TH><TH>Positive Control 3 (2)</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>0µM</TD><TD>0µM</TD><TD>30µM</TD><TD>30µM</TD><TD>50µM</TD><TD>50µM</TD><TD>100µM</TD><TD>100µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>25µg/mL</TD><TD>0µg/mL</TD><TD>25µg/mL</TD><TD>0µg/mL</TD><TD>25µg/mL</TD><TD>0µg/mL</TD><TD>25µg/mL</TD><TD>0µg/mL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 1µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 1µL of 2.5mg/mL Ascorbate stock or MilliQ for 0µg/mL</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Fill 1mL cuvettes with</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein Solution</TD><TD>100µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>10µL</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>900µL</TD></TR><TR><TD>Total Volume</TD><TD>1020µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm</LI></UL><H3 id="Version-6">Version 6</H3><UL class="textLiFont"><LI>Make 14 1 mL cuvettes each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Ascorbate 1 w/ Cu</TH><TH>Ascorbate 2 w/ Cu</TH><TH>Ascorbate 3 w/ Cu</TH><TH>Glutathione 1 w/ Cu</TH><TH>Glutathione 2 w/ Cu</TH><TH>Glutathione 3 w/ Cu</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>10mM</TD><TD>5mM</TD><TD>1mM</TD><TD>0mM</TD><TD>0mM</TD><TD>0mM</TD></TR><TR><TD>[Glutathione]<SUP>***</SUP></TD><TD>0mM</TD><TD>0mM</TD><TD>0mM</TD><TD>10mM</TD><TD>5mM</TD><TD>1mM</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD></TR><TR><TD>Total Volume</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Ascorbate 1 wo/ Cu</TH><TH>Ascorbate 2 wo/ Cu</TH><TH>Ascorbate 3 wo/ Cu</TH><TH>Glutathione 1 wo/ Cu</TH><TH>Glutathione 2 wo/ Cu</TH><TH>Glutathione 3 wo/ Cu</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>10mM</TD><TD>5mM</TD><TD>1mM</TD><TD>0mM</TD><TD>0mM</TD><TD>0mM</TD></TR><TR><TD>[Glutathione]<SUP>***</SUP></TD><TD>0mM</TD><TD>0mM</TD><TD>0mM</TD><TD>10mM</TD><TD>5mM</TD><TD>1mM</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD><TD>900µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD><TD>10µL</TD></TR><TR><TD>Total Volume</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD><TD>1040µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Control w/ Cu </TH><TH>Control wo/ Cu</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>10µM</TD><TD>0µM</TD></TR><TR><TD>[Ascorbate]<SUP>**</SUP></TD><TD>0µM</TD><TD>0µM</TD></TR><TR><TD>[Glutathione]<SUP>***</SUP></TD><TD>0µM</TD><TD>0µM</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>900µL</TD><TD>900µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>10µL</TD><TD>10µL</TD></TR><TR><TD>Total Volume</TD><TD>1040µL</TD><TD>1040µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 10µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 10µL of Ascorbate stock 100x dilution or MilliQ for 0mM</P><P style="text-align:right" class="footnote"><SUP>***</SUP>total 10µL of Glutathione stock 100x dilution or MilliQ for 0mM</P><UL class="textLiFont"><LI>Mix by inversion</LI><LI>Immediately measure on spectrophotometer for absorbance at 480nm.</LI></UL><H3 id="Version-7">Version 7</H3><UL class="textLiFont"><LI>Prepare the following wells on a Costar96 plate:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Negative Control </TH><TH>Positive Control 1</TH><TH>Positive Control 2</TH><TH>Positive Control 3</TH><TH>Positive Control 4</TH><TH>Positive Control 5</TH><TH>Positive Control 6</TH><TH>Negative Control wo/Glutathione</TH><TH>Positive Control wo/Glutathione</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>0µM</TD><TD>1µM</TD><TD>5µM</TD><TD>10µM</TD><TD>15µM</TD><TD>20µM</TD><TD>30µM</TD><TD>0µM</TD><TD>30µM</TD></TR><TR><TD>[Glutathione]<SUP>**</SUP></TD><TD>pH 7.0 Buffer 0.2M</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD></TR><TR><TD>Total Volume</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Glutathione Negative Control </TH><TH>Glutathione Positive Control 1</TH><TH>Glutathione Positive Control 2</TH><TH>Glutathione Positive Control 3</TH><TH>Glutathione Positive Control 4</TH><TH>Glutathione Positive Control 5</TH><TH>Glutathione Positive Control 6</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD><TD>10µM</TD></TR><TR><TD>[Glutathione]<SUP>**</SUP></TD><TD>0µM</TD><TD>3µM</TD><TD>10µM</TD><TD>30µM</TD><TD>100µM</TD><TD>300µM</TD><TD>1000µM</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD></TR><TR><TD>Total Volume</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD></TR></TBODY></TABLE><TABLE style="margin: auto;"><THEAD><TR><TH>Glutathione Negative Control (2)</TH><TH>Glutathione Positive Control 1 (2)</TH><TH>Glutathione Positive Control 2 (2)</TH><TH>Glutathione Positive Control 3 (2)</TH><TH>Glutathione Positive Control 4 (2)</TH><TH>Glutathione Positive Control 5 (2)</TH><TH>Glutathione Positive Control 6 (2)</TH></TR></THEAD><TBODY><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD><TD>0µM</TD></TR><TR><TD>[Glutathione]<SUP>**</SUP></TD><TD>0µM</TD><TD>3µM</TD><TD>10µM</TD><TD>30µM</TD><TD>100µM</TD><TD>300µM</TD><TD>1000µM</TD></TR><TR><TD>pH 7.0 Buffer 0.2M</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD><TD>146µL</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD><TD>50µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD><TD>2µL</TD></TR><TR><TD>Total Volume</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD><TD>200µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 2µL of 100x conc. stock CuSO4 or MilliQ for 0µMt</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 2µL of Glutathione stock 100x dilution or MilliQ for 0mM</P><UL class="textLiFont"><LI>Mix in plate reader at 200rpm</LI><LI>Immediately measure absorbance at 480nm in the ClarioStar.</LI></UL><H3 id="Version-8">Version 8</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice</LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as mg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Make 6 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein</TH><TH>Test Protein wo/ Cu</TH><TH>GFP Control</TH><TH>GFP Control wo/ Cu</TH><TH>Negative Control</TH><TH>Positive Control</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>100µL mg</TD><TD>100µL mg</TD><TD>100µL GFP</TD><TD>100µL GFP</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>50µM</TD><TD>0µM</TD><TD>50µM</TD><TD>0µM</TD><TD>0µM</TD><TD>50µM</TD></TR><TR><TD>[Glutathione]<SUP>**</SUP></TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 1µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 1µL of 10mM Glutathione stock or MilliQ for 0mM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Prepare a Costar96 plate with 1 well for each Eppendorf as follows:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein/Test Solution</TD><TD>50µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>2µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>2µL</TD></TR><TR><TD>pH 7.0 Buffer 1M</TD><TD>146µL</TD></TR><TR><TD>Total Volume</TD><TD>200µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix in plate reader at 200rpm</LI><LI>Immediately measure absorbance at 480nm in the ClarioStar.</LI></UL><H3 id="Version-9">Version 9</H3><UL class="textLiFont"><LI>Procedure should be in cold room and all containers and solutions chilled on ice</LI><LI>Defrost proteins on ice</LI><LI>Prepare solution of His tagged GFP with same concentration as cg sample by dilution with protein buffer using Nanodrop to calculate concentration</LI><LI>Dilute mg sample with protein buffer to concentration of cg</LI><LI>Nanodrop mg sample to use and:</LI><UL><LI>A<SUB>nanodrop</SUB>/ε = [Protein] (in M)</LI></UL><LI>Make 6 Eppendorfs each containing:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Test Protein 1</TH><TH>Test Protein 1 wo/ Cu</TH><TH>Test Protein 2</TH><TH>Test Protein 2 wo/ Cu</TH><TH>GFP Control</TH><TH>GFP Control wo/ Cu</TH><TH>Negative Control</TH><TH>Positive Control</TH></TR></THEAD><TBODY><TR><TD>Protein</TD><TD>100µL mg</TD><TD>100µL mg</TD><TD>100µL cg</TD><TD>100µL cg</TD><TD>100µL GFP</TD><TD>100µL GFP</TD><TD>No</TD><TD>No</TD></TR><TR><TD>[Cu2+]<SUP>*</SUP></TD><TD>10µM</TD><TD>0µM</TD><TD>10µM</TD><TD>0µM</TD><TD>10µM</TD><TD>0µM</TD><TD>0µM</TD><TD>10µM</TD></TR><TR><TD>[Glutathione]<SUP>**</SUP></TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD><TD>100µM</TD></TR><TR><TD>pH 8.0 Buffer</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>no additional</TD><TD>yes</TD><TD>yes</TD></TR><TR><TD>Total Volume</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD><TD>100µL</TD></TR></TBODY></TABLE><P style="text-align:right" class="footnote"><SUP>*</SUP>total 1µL of 100x conc. stock CuSO4 or MilliQ for 0µM</P><P style="text-align:right" class="footnote"><SUP>**</SUP>total 1µL of 10mM Glutathione stock or MilliQ for 0mM</P><UL class="textLiFont"><LI>Lightly vortex after addition</LI><LI>Place on ice for 30 mins</LI><LI>Prepare a Costar96 plate with 1 well for each Eppendorf as follows:</LI></UL><TABLE style="margin: auto;"><THEAD><TR><TH>Solution</TH><TH>Volume (μl)</TH></TR></THEAD><TBODY><TR><TD>Protein/Test Solution</TD><TD>150µL</TD></TR><TR><TD>BCS 5mg/mL</TD><TD>2µL</TD></TR><TR><TD>Ascorbate 2.5mg/mL</TD><TD>2µL</TD></TR><TR><TD>pH 7.0 Buffer 1M</TD><TD>46µL</TD></TR><TR><TD>Total Volume</TD><TD>200µL</TD></TR></TBODY></TABLE><UL class="textLiFont"><LI>Mix in plate reader at 200rpm</LI><LI>Immediately measure absorbance at 480nm in the ClarioStar.</LI></UL><H3 id="Version-10">Version 10</H3><UL class="textLiFont"><LI>Prepare 6 test tubes containing 5mL of LB broth and 25µg/mL chloramphenicol</LI><LI>Add the inducer L-arabinose to 10mg/mL concentration in 3 tubes and 1mg/mL in the other 3 tubes</LI><LI>Pick 2 colonies of each negative control mg-1655 cells, and cg cells adding one of each to the 10mg/mL Arabinose tube and the other to the 1mg/mL Arabinose tube
</LI><LI>Incubate overnight at 37°C</LI><LI>Take 10µL of each overnight culture into Eppendorf tubes</LI><UL><LI>Add 1mL of fresh LB broth to each tube</LI><LI>Bring chloramphenicol and arabinose concentrations to those matching the overnight cultures in each tube</LI><LI>Add 1µL 10mM CuSO4 solution to each tube</LI></UL><LI>Incubate all 4 tubes for 5 hours</LI><LI>Centrifuge all tubes and take 0.9mL of each supernatant into 1mL cuvettes</LI><UL><LI>Create positive controls by creating 2 1mL LB solutions containing 25µg/mL chloramphenicol, 10mM CuSO4 with one containing 10mg/mL Arabinose tube and the other 1mg/mL Arabinose.  Add 0.9mL of each of these as well into 1mL cuvettes.</LI></UL><LI>Add 80µL 1.0M pH 7.0 Tris buffer, 10µL 10mM Glutathione and 10µL 5mg/mL BCS to each cuvette</LI><LI>Mix by inversion</LI><LI>Immediately measure absorbance at 480nm on a spectrophotometer</LI></UL></SECTION><SECTION id="Capsule-Design-and-Testing"><H1 id="Capsule-Design-and-Testing" class="pageTitleBlue">Capsule Design and Testing</H1><H2 id="Multi-layered-Microcapsule-Production">Multi-layered Microcapsule Production</H2><P>The beads had to be first designed using alginate and chitosan in alternating layers. This would allow transfer of the probiotic in the gut.</P><H3>Creating the alginate core and the multi-layered coating:</H3><UL class="textLiFont"><LI>Heat 100ml of MilliQ.</LI><LI>Weigh 2g of sodium alginate and add it very slowly to the heated MilliQ to prevent clumping. This creates a 2% (w/v) sodium alginate solution. </LI><LI>Once the alginate has fully dissolved stop heating. </LI><LI>Measure 2ml of crystal violet (1% (w/v)) and add to 18ml of alginate solution, stir together. </LI><LI>Take up the alginate-dye solution with a syringe that has a needle at the end. Add the solution to 0.1M of CaCl<SUB>2</SUB> solution. This will create the alginate core</LI><LI>Leave for 30 minutes in order for the beads to harden</LI><LI>Prepare a 0.4% (w/v) chitosan solution (in 0.1M acetic acid adjusted to pH 6.0 with 1M NaOH, 10mL) and a 0.04% (w/v) alginate solution (10mL).</LI><LI>Remove alginate core from calcium chloride solution by filtration</LI><LI>Dip the beads in the chitosan solution for 10 minutes. Then in the alginate solution for 10 minutes. Do this 3 times</LI></UL><H2 id="Microcapsule-Integrity-in-Simulated-Gut-Environments">Microcapsule Integrity in Simulated Gut Environments</H2><UL class="textLiFont"><LI>Make a simulated stomach solution (0.2% (w/v) NaCl solution made up to pH 2.0 with 1M HCl)</LI><LI>Make a simulated instestine solution (0.68% (w/v) monobasic potassium phosphate solution, made up to pH 7.2 with 1M NaOH)</LI><LI>Place 10ml of beads in each solution and place both containers in 37°C shaker for 90 minutes. As a negative control place 10ml of beads into CaCl<SUB>2</SUB> solution in the same shaker.  </LI><LI>Take a 100μl sample from each solution every 10 minutes and place it into a COSTAR96 well plate</LI><LI>Once 90 minutes have passed, measure the absorption in each well in the plate reader.</LI></UL><P>Schematic of the plate:</P><TABLE style="margin: auto;"><TBODY><TR><TD>Stomach, t = 0</TD><TD>Stomach, t = 10</TD><TD>Stomach, t = 20</TD><TD>Stomach, t = 30</TD><TD>Stomach, t = 40</TD><TD>Stomach, t = 50</TD><TD>Stomach, t = 60</TD><TD>Stomach, t = 70</TD><TD>Stomach, t = 80</TD><TD>Stomach, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Stomach, t = 0</TD><TD>Stomach, t = 10</TD><TD>Stomach, t = 20</TD><TD>Stomach, t = 30</TD><TD>Stomach, t = 40</TD><TD>Stomach, t = 50</TD><TD>Stomach, t = 60</TD><TD>Stomach, t = 70</TD><TD>Stomach, t = 80</TD><TD>Stomach, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Stomach, t = 0</TD><TD>Stomach, t = 10</TD><TD>Stomach, t = 20</TD><TD>Stomach, t = 30</TD><TD>Stomach, t = 40</TD><TD>Stomach, t = 50</TD><TD>Stomach, t = 60</TD><TD>Stomach, t = 70</TD><TD>Stomach, t = 80</TD><TD>Stomach, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Intestine, t = 0</TD><TD>Intestine, t = 10</TD><TD>Intestine, t = 20</TD><TD>Intestine, t = 30</TD><TD>Intestine, t = 40</TD><TD>Intestine, t = 50</TD><TD>Intestine, t = 60</TD><TD>Intestine, t = 70</TD><TD>Intestine, t = 80</TD><TD>Intestine, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Intestine, t = 0</TD><TD>Intestine, t = 10</TD><TD>Intestine, t = 20</TD><TD>Intestine, t = 30</TD><TD>Intestine, t = 40</TD><TD>Intestine, t = 50</TD><TD>Intestine, t = 60</TD><TD>Intestine, t = 70</TD><TD>Intestine, t = 80</TD><TD>Intestine, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Intestine, t = 0</TD><TD>Intestine, t = 10</TD><TD>Intestine, t = 20</TD><TD>Intestine, t = 30</TD><TD>Intestine, t = 40</TD><TD>Intestine, t = 50</TD><TD>Intestine, t = 60</TD><TD>Intestine, t = 70</TD><TD>Intestine, t = 80</TD><TD>Intestine, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Control, t = 0</TD><TD>Control, t = 10</TD><TD>Control, t = 20</TD><TD>Control, t = 30</TD><TD>Control, t = 40</TD><TD>Control, t = 50</TD><TD>Control, t = 60</TD><TD>Control, t = 70</TD><TD>Control, t = 80</TD><TD>Control, t = 90</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Control, t = 0</TD><TD>Control, t = 10</TD><TD>Control, t = 20</TD><TD>Control, t = 30</TD><TD>Control, t = 40</TD><TD>Control, t = 50</TD><TD>Control, t = 60</TD><TD>Control, t = 70</TD><TD>Control, t = 80</TD><TD>Control, t = 90</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><DIV class="container-fluid" id="footer" style="padding-top:25px; padding-bottom:30px; border-top:1px solid #CDCCCE; font-family: avenir, arial">

Made with gooood stuff.  </DIV></DIV></DIV></DIV></DIV></BODY></HTML>