Team:Oxford/Notebook
iGEM Oxford 2016 - Cure for Copper
Project
Description
Wilson's Disease
Timeline
Safety
Proof of Concept
Achievements
Parts
Parts Overview
Basic Part
Composite Part
Part Collection
CueR Mechanism
Wet lab
Overview and Results
Experiments
Protocols
Chemicals
Sequences
InterLab Study
Notebook
Dry lab
Modelling Overview
Reaction Kinetics
Promoter Characterisation
Copper Advection
Bacteria Distribution
Practices
Human Practices
Silver Criterion
Gold Criterion
Integrated Practices
Surveys
Engagement
People
Team
Collaborations
Attributions
Sponsors
Contact Us
Blog
Intro
Week 1
Day 1
Day 2
Day 3
Day 4
Day 5
Week 2
Day 1
Day 2
Day 3
Day 4
Day 5
Week 3
Day 1
Day 2
Day 3
Day 4
Day 5
Week 4
Day 1
Day 2
Day 3
Day 4
Day 5
Week 5
Day 1
Day 2
Day 3
Day 4
Day 5
Day 7
Week 6
Day 1
Day 2
Day 3
Day 4
Day 5
Week 7
Day 1
Day 2
Day 3
Day 4
Day 5
Week 8
Day 1
Day 2
Day 3
Day 4
Day 5
Day 7
Week 9
Day 1
Day 2
Day 3
Day 4
Day 5
Week 10
Day 1
Day 2
Day 3
Day 4
Day 5
Day 7
Week 11
Day 1
Day 2
Day 3
Day 4
Day 5
Day 6
Day 7
Week 12
Day 1
Day 2
Day 3
Day 4
Day 5
Day 7
Week 13
Day 1
Day 2
Day 3
Day 4
Day 5
Day 6
Day 7
Week 14
Day 1
Day 2
Day 3
Day 4
Day 5
Day 6
Day 7
Week 15
Day 1
Day 2
Day 3
Day 4
Day 5
Day 7
Week 16
Day 1
Day 2
Interlab Notebook
Notebook
Introduction
This page documents all the experiments we carried out in the wet lab as a part of our project. The details of the process we carried out can be found on our protocols page, and the chemicals we used can be found on the chemicals page. The interlab project was carried out alongside the rest of the wet lab but we have recorded it separately .
The acronyms we used throughout the wet lab diary correspond to the following parts:
TAT Copper Storage Protein 1 (BBa_K1980000): tc
TAT Copper Storage Protein 1 sfGFP (BBa_K1980001): cg
MymT (BBa_K1980002): m or mEx
MymT sfGFP (BBa_K1980003): mg
pCusC promoter (BBa_K1980004): pCusC
pCopA sfGFP (BBa_K1980005): pcg
pCopA sfGFP with plasmid constitutively expressed CueR (never deposited): pg
pCopA with plasmid constitutively expressed CueR (BBa_K1980006): p
pCusC RFP (BBa_K1980007): pCusC RFP
pCopA CueR sfGFP/ feedback pCopA sfGFP (BBa_K1980008): fcg
pCusC CusR RFP (BBa_K1980009): fck
pCopA TAT Csp1 sfGFP with plasmid constitutively expressed CueR (BBa_K1980010): ptcg
pCopA MymT with plasmid constitutively expressed CueR (BBa_K1980011): pm
pCopA MymT sfGFP with plasmid constitutively expressed CueR (BBa_K1980012): pmg
Our starting Gblock sequences, primers and plasmids can be found on our sequences page.
Week 1 20/06
Day 1
Preparation of Stock Solutions
gBlocks
3 of our gBlocks from IDT had arrived: pCopA MymT HH (pc), MymT sfGFP HH (mg) and TAT Csp1 HH (tc). These arrived in the form of a solid, white powder. We resuspended the gBlocks in different volumes of MilliQ, depending on the weight delivered, to give a stock solution of 10 ng/μl.
Primers. The forward and reverse primers from IDT came as 32.4 nmol and 23.8 nmol of solid respectively. Forward: 5’-CGACTTGATCACGTAGAATTC-3’. Reverse: 5’-ACACGATCGATATAACTGCAGC-3’. For our stock solutions, the primers were suspended in different volumes of MilliQ to give 100μM.
Preparation of reaction solutions. gBlocks: Following resuspension, 10μl of each DNA solution was added to 90μl MilliQ (1:10 dilution) in an Eppendorf to give a final concentration of 1 ng/μl and final solution volume of 100μl.
Primers: Following resuspension, 10μl of each primer solution was added to 90μl MilliQ (1:10 dilution) in an Eppendorf to give a final concentration of 10μM and final solution volume of 100μl.
Day 2
Polymerase Chain Reaction (PCR). 25μl reactions were run according to the NEB Q5 High-Fidelity 2X Master Mix PCR protocol. * 98°C denaturation temperature used due to the high stability of the Q5 polymerase. ** 60°C annealing temperature used on Chris’s recommendation. *** 30s elongation time used as maximum DNA length was 936bp and a PCR takes approximately 20-30s to extend a DNA sequence by 1000bp.
Gel Electrophoresis of PCR-Amplified gBlocks
A 1% agarose gel was prepared according to the agarose gel preparation protocol.  A DNA ladder mix was prepared in an eppendorf. The 25μl solutions of PCR-Amplifed gBlocks were mixed with 5μl loading dye in order to visualise their migration. 20μl of this solution was then loaded into each gel well. The gel was run with a potential difference of 100V applied across the electrodes for approx. 40 minutes. Following separation, the gel was transferred to a bath of EtBr for staining. It was left shaking for 20 minutes. Success! The bands produced corresponded to the expected DNA sizes. These were excised using a razor blade and placed in labelled eppendorfs for freezing overnight.
Our first agarose gel :0
Day 3
Gel extraction of PCR-Amplified gBlocks from agarose gel
The extraction was performed using the Qiagen QIAquick Gel Extraction Kit protocol. 540μl of Buffer QG was added to each tube in 2. In step 9, 30μl MilliQ was used rather than 50μl buffer EB on Chris’s recommendation. The tubes were spun with lids open, hence, they were spun with the lids positioned such that would not flip around during centrifugation, away from the direction of spinning.
Preparation of MymT HH Stock Solution. Another gBlock, MymT HH (m), arrived from IDT and was resuspended as described in Week 1: Day 1. 424ng was delivered, so 42.4μl MilliQ was added to give 10ng/μl.
Preparation of MymT HH reaction solution. Same procedure as Week 1: Day 1.
PCR of MymT. Same program as Week 1: Day 2 used, apart from using a 62°C annealing temperature. gBlocks from yesterday also run to see whether the higher annealing temperature eliminated the smears observed in the PCR from Day 2.
Less smeary gels
MymT sfGFP HH and TAT Csp1 ran slightly better under the new conditions. No result from pCopA MymT HH. No template controls run adjacent under same name as DNA produced no result, as expected. MymT HH produced a large smear. Will run again in future.
Day 4
PCR of MymT
Results: 1) produced a smear. 2) produced a smear. 3) produced nothing.
Restriction Digest of extracted PCR products. Digest of PCR-amplified pCopA MymT HH, MymT sfGFP HH, and TAT Csp1 HH performed using NEB EcoR1-HF and PstI-HF restriction enzymes. CutSmart buffer determined to be the best to use using NEB’s Double Digest Finder. Volume added to make reaction mixture up to 50μl. Incubation at 37°C for 2 hours in a ThermoMixer.
Day 5
Open day, no wet lab.
Week 2 27/06
Day 1
PCR of MymT. After no success last week, a new working reaction solution was made up in case the previous reaction solution had been contaminated. Run under new conditions but still a smear over most molecular weights. Carry out again.
PCR, gel extraction and restriction enzyme digest of shipping vector. Using shipping vector from the team last year for our plasmids. Ran a PCR using standard conditions to amplify amount of the vector. Gel Extraction of plasmid using the Qiagen QIAquick Gel Extraction Kit protocol. Restriction enzyme digest of extraction using same protocol as followed in Week 1: Day 4. 10μl of plasmid used instead of 25μl.
Gel extraction of restriction digest incubations. Upon completion of the restriction digest incubation, the gBlocks and the plasmid backbones were purified using the Qiagen QIAquick Gel Extraction Kit protocol. We quantified the amount of DNA using NanoDrop 1ul of water and tissue were initially used to clean the NanoDrop stage. A blank reading was made using 1μl of MilliQ. 1μl of each DNA sample was measured for concentration. The DNA was ligated with the shipping vector and left overnight at 16°C.
Day 2
Antibiotic stock made. Shipping plasmid contains gene conferring resistance to chloramphenicol, so chloramphenicol antibiotic stock made up. Reaction mixture composed of 300mg chloramphenicol, 0.5ml MilliQ, and 9.5ml ethanol (prevents freezing). After making solution, filter pipette tips used to produce 20 x 0.5ml aliquots of chloramphenicol solution for later use.
Set up agar plates. LB agar melted to produce a pourable liquid. After melting, chloramphenicol from stock solution added at 1:1000 dilution. Plating in flow cupboard. Plates left to set for 30 minutes.
Transformation of E. coli cells with ligated plasmid DNA. Competent E. coli cells removed from freezer at -80°C and thawed on ice. Transformation protocol followed. 2 plates for each DNA sample (MymT sfGFP HH, TAT Csp1 HH, pCopA MymT HH sample 1, and pCopA MymT HH sample 2). First plate produced by spreading 100μl of the cell solution onto one plate. Second plate produced after spinning down the remaining solution and resuspending in LB broth and plating, resulting in a more concentrated plate. Plates left to incubate overnight at 37°C.
Miniprep of CusC. Minipreparation: isolation of plasmid DNA from bacteria. Completed using the QIAprep Spin Miniprep Kit protocol. Resulting solution placed in freezer at -20°C
Day 3
Growth and Culture of bacteria. Plates incubated overnight all contain colonies. More are present on the concentrated plates. 2 colonies from each plate are selected (not too large or too small) and incubated in test tubes with 5ml LB broth and chloramphenicol in 37°C shaker overnight. Aim of the procedure is to increase the number of plasmids containing our biobricks.
PCR of MymT HH was unsuccessful again.
Preparation of pCopA TAT Csp1 sfGFP HH (ptcg) Stock Solution. Another gBlock, pCopA TAT Csp1 sfGFP HH, arrived from IDT and was resuspended as described in Week 1: Day 1. 1000ng was delivered, so 100μl MilliQ was added to give 10ng/μl.
Preparation of MymT HH reaction solution - Same procedure as Week 1: Day 1.
PCR of pCopA TAT Csp1 sfGFP HH - Unsuccessful – producing a large DNA smear over many different DNA sizes.
Day 4
PCRs of pCopA TAT Csp1 sfGFP HH and MymT HH - All unsuccessful.
Miniprep of pCopA MymT HH, MyMT sfGFP HH and TAT Csp1 HH completed using the QIAprep Spin Miniprep Kit protocol. Resulting solution placed in freezer at -20°C. Diagnostic gel of pCopA MymT HH, MyMT sfGFP HH and TAT Csp1 HH. Aim was to check that our biobrick inserts had been successfully incorporated into the plasmid, by digesting with the restriction enzyme. Gel set up.
Our first diagnostic gel
Day 5
PCRs of pCopA TAT Csp1 sfGFP HH and MymT HH - All unsuccessful.
Week 3 4/07
Day 1
Only interlab wet lab.
Day 2
Only interlab wet lab.
Day 3
PCRs of pCopA TAT Csp1 sfGFP HH and MymT HH - All unsuccessful. Negative control gave spear – possible risk of contamination Transformed parts: DspB, DspBx, DsbA-DspB, and DsbA-DspBx to send to XBU China.
Day 4
PCRs of TAT Csp1 HH (tc) and negative control to check no contamination (positive control and negative control). Successful! No DNA in negative control, bands where expected to be in positive control. Picked colonies from DspB, DspBx, DsbA-DspB, and DsbA-DspBx.
Day 5
Mini prepped sequences for XBU China (DspB, DspBx, DsbA-DspB and DsbA-DspBx). Isolated 2 colonies per part (just for back-up). Completed using the QIAprep Spin Miniprep Kit protocol. Resulting solution placed in freezer at -20°C. Sent parts off to China
Sent sequences off for sequencing: CSP1A – VF2, CSP1A – VR, CSP1B – VF2, CSP1B – VR, MGA – VF2, MGA – VR, MGB – VF2, MGB – VR, PMA – VF2, PMA – VR, PMB – VF2, PMB – VR, pCusC – CC forward, pCusC – CC reverse. Required concentrations: Plasmid DNA - 5μl of 100ng/l, Primers - 5μl of 3.2pmol/l per reaction. Sent excess. All failed except the pCusCs.
Week 4 11/07
Day 1
pCopA MymT sfGFP HH (pmg) and TAT Csp1 sfGFP HH (cg) arrive. Resuspend (both arrived as 1000ng sample), dilute to 10ng/μl in tube then take dilute by 1 in 10 to get a 1ng/ml stock then add 1μl of this to the PCR mixture.
PCR pmg and cg using standaed concentrations and volumes. 25 cycles, 95°C for 15s, 62°C for 15s, 72°C for 1 min. Results: both produced smears (NTC = no result)
Day 2
PCR pmg and cg. 64°C, 1 min elongation time and 66°C, 1 min elongation time. pmg64, pmg66 and cg64 all produced smears, cg66 showed nothing
PCR ptcg because new primers arrived. Ran at 60°C with new primers. Success! Gel extract ptcg. Gel volume = 50μl
PCR pmg (using new ptcg primers) and cg (using new ptcg reverse primer and usual forward primer). Run at 60°C and 62°C. All successful!
Day 3
Gel extract cg and pmg, gel volume = 40μl
PCR MymT using normal forward primer and new ptcg reverse primer. 60°C and 62°C, 1 min elongation time. m60 = smear, m62 = smear but band at 254bp
PCR MymT again at 63°C and 68°C with 15s elongation time. Both still produced a smear but IDT said this part’s mass spec was messy, band is present
Gel extract MymT HH, gel volume = 50μl
PCR ptcg and pmg using combination of old and new primers: Ptcg1 = old forward, new reverse, Ptcg2 = new forward, old reverse, Pmg1 = old forward, new reverse, Pmg2 = new forward, old reverse. 2s gave better results (used 2s for gel extraction and ligation)
Day 4
Ligation of cg (new forward and old reverse primers) and MymT (m) (new forward and old reverse primers), cut with EcoRI and SpeI and used CutSmart buffer.
Gel extract ptcg and pmg, gel volume = 50μl
Day 5
Digest and ligate ptcg2 and pmg2 (both using new forward and old reverse) using PstI and XbaI. Diagnostic gel for cg and Mym using EcoRI and SpeI
Getting things into pBad from the shipping plasmid. PCR amplify with specific primer pairs from plasmid. 4 reactions using 5 primers.
1. Csp1 into pBad, then cute this into BspHI and Pst1, cut pBad with NcoI and PstI Ex P TAT Csp1 Forward – Tm 64°C Ex P Rev – Tm 67°C
2. MymT sfGFP into pBad Ex P MymT Forw – Tm 64°C Ex P Rev – Tm 67°C
3. MymT sfGFP into MymT in pBAD Ex P MymT Forw – Tm 64°C MymT only cut Rev – Tm 63°C
4. MymT sfGFP into MymT in shipping vector – cut with EcoRI and PstI, cut shipping vector with EcoRI/ and PstI MymT shipping forward – Tm 61°C MymT only cut – Tm 63°C
PCR conditions,  95°C 02:00, Then 25x {95°C 00:15, 62°C 00:15, 72°C 00:30} Then 72°C 01:00 and hold at 10°C. Results: Ran off gel but then re-ran
Week 5 18/07
Day 1
pg arrived, band size is 1391 bp. Resuspended.
PCR pg. pg1 – old forward, new reverse primers, pg2 – new forward, old reverse primers. Both at 60°C, 1 min elongation. Ran gel of pg PCR products. pg1 – successful, cut from gel but pg2 – unsuccessful, multiple bands produced
Digest and ligate Csp1 into pBad. Cut parts with BspHI and PstI and cut plasmid (pBAD) with NcoI and PstI using CutSmart buffer. Digest and ligate MymT sfGFP ino pBad. Cut parts with BspHI and PstI and cut plasmid (pBAD) with NcoI and PstI using CutSmart buffer. Digest and ligate MymT sfGFP into MymT in pBad. Cut parts with BspHI and PstI and cut plasmid (pBAD) with NcoI and PstI using CutSmart buffer.
Day 2
Diagnostic gel for cg (16) and m (16)
Gel extract pg, gel weight = 10mg. Enzyme digestion of pg gel extract using EcoRI and SpeI and CutSmart buffer. Ligation of pg into shipping plasmid.
Day 3
PCR mg from sequenced plasmid (“3” and “4”). To get m out cut 3 with BspHI and PstI and cut 4 with EcoRI and PstI. Conditions = 95°C, 62°C annealing, 25x cycles
Digest ptcg and pmg using EcoRI and SpeI. Gel extract 3 and 4, Gel volume = 40mg  (used 2% gel)
Miniprep pmg (14) and ptcg (14), ran diagnostic gel on miniprep product. Unsuccessful.
Transformations of pg using chloroamphenicol and of tc and mg (into pBAD) using amp.
Day 4
Digest ptcg (1 and 2) and pmg (1 and 2), and 3 and 4. Digest shipping plasmid with M3 and M4 inside it. M3 with BspHI and PstI, M4 with EcoRI and PstI.
Re-digestion of unsuccessful transformation minipreps. Ptcg1: old forward primer, new reverse primer, use EcoRI and SpeI and CutSmart buffer. Ptcg2: new forward primer, old reverse primer, use PstI and XbaI and CutSmart buffer. Pmg1: old forward primer, new reverse primer, use EcoRI and SpeI and CutSmart buffer. Pmg2: new forward primer, old reverse primer use PstI and XbaI and CutSmart buffer. M3 into pBAD use BspHI and PstI and CutSmart buffer. M4 into shipping plasmid use EcoRI and PstI and CutSmart buffer
Colonies picked from tc, mg (amp – pBAD) and pg (chloroamphenicol – shipping).
Day 5
Digest shipping plasmid to use for ligation of M3 and M4. Digest one plasmid with BspI and PstI. Digest 2nd with EcoRI and PstI. Run gel of these digested shipping plasmids.
Ligation. Pmg1 – EcoRI and SpeI, Pmg2 – PstI and XbaI, Ptcg1 – EcoRI and SpeI, Ptcg2 – PstI and XbaI, M3 (pBAD) – BspHI and PstI, M4 (shipping plasmid) – EcoRI and PstI.
Miniprep tc, mg, cg and pg, run diagnostic digest and gel. tc and mg in pBAD – using BamHI and PstI. cg and pg in shipping plasmid – using EcoRI and SpeI
Day 7
Picked Mg1655 colonies.
Week 6 25/07
Day 1
Transform ptcg and pmg 2 plates each, therefore 4 plates
Feedback pCusC mKATE (fck) arrived 1000ng delivered → add 100μl to give 10ng/μl After resuspension, dilute 1 in 10 to give 1ng/μl
PCR fck fck,on 60 = old forward, old reverse primers, 60°C annealing fck,on 62 = old forward, old reverse primers, 62°C annealing fck,on 60 = old forward, new reverse primers, 60°C annealing fck,on 62 = old forward, new reverse primers, 62°C annealing Results: fck on 60°C and fck on 62°C worked
Gel extraction for fck
Day 2
Picked colonies from pmg and ptcg
Day 3
Miniprep ptcg, pmg and pm
Diagnostic gel for ptcg and pmg Using PstI and EcoRI Expected band sizes = 1824 and 1565, respectively
Sent ptcg and pmg for sequencing
Day 4
Sequencing results ptcg → didn’t work pmg → didn’t work cg (in shipping plasmid) → didn’t work tc → didn’t work pg → has promoter, no mismatches, truncated mg → good
Next steps: Re-transform fck, PCR ptcg, pmg, cg and tc, Pick M3 colonies, Transform pg and mg into MG1655
Old shipping plasmid possibly contaminated, digest new. Using part from last year (Art-175). Component: Art-175 5μl, 10xbuffer 5μl, EcoRI 1ul, SpeI 1μl and MilliQ 38μl. Set up digest at 37°C for 2 hours
Next steps: Heat inactivate at 80°C for 10 mins. Dephosphorylate with 1μl antarctic phosphatase 30 mins at 37°C. Run gel. Excise 2kb band.
PCR tc, cg, ptcg and pmg. Tc – 60°C annealing, old forward, old reverse. Cg – 62°C annealing, old forward, new ptcg reverse. Ptcg – 60°C annealing, new forward, old reverse. Pmg – 60°C, new forward, old reverse.
Gel for tc, cg, ptcg and pmg Expected band size = 545, 1277, 1824 and 1565 bp, respectively Results: tc = very smeary, other 3 to be extracted
Day 5
PCR ptcg and pmg.  ptcg = new forward, old reverse primers,60°C annealing temperature
Transform pg and mg into MG1655, pg uses chloroamphenicol, mg now uses ampicillin.
Week 7 1/08
Day 1
Measuring fluorescence in 96 well plate of pg
Transformation of mg(pBAD) into MG1655 using ampicillin.
Pick 4x colonies from pg plate using chloroamphenicol.
Pick 4x colonies from MG1655, no antibiotic.
Ligate ptcg, pmg, tc, cg and fck
Day 2
Miniprep M3 using ampicillin.
Digest and diagnostic gel using EcoRI and SpeI. Expected band size = 254bp Came back incorrect
Pick colonies for pCusC, MG1655 negative control, MG into pBAD using chloroamphenicol, no antibiotic and amphicillin, respectively. 4 colonies each
Transformations of fck, ptcg, pmg, tc and cg into DH5-α, chloroamphenicol need 10 plates. Not successful.
Day 3
Transformations of fck, ptcg, pmg, tc and cg into DH5-α. Results: unsuccessful for all except cg. Chloroamphenicol
Transformation of mg(pBAD) into MG1655, Ampicillin, Successful.
Digestion and diagnostic gel of M3 (1-6). On gel, expected band size = 287bp. Used 2% agrose gel.
PCR m out of mg shipping vector. Expected band size = 196bp Successful Extracted
Day 4
Miniprep cg
Diagnostic gel for cg, use EcoRI and SpeI. Expected band size = 1250bp
Plate reader experiment for pg and negative control, followed protocol. 50μl LB (not 40μl) OD 600
Transforms: Pmg – chlorophenicol into DH5α, Fck – chloroamphenicol into DH5α, Ptcg – chloramphenicol into DH5α, mEx – ampicillin into DH5α, pcg – chloramphenicol into DH5α, pBAD vector no insert = ampicillin MG1655.
Day 5
Miniprep cg . Diagnostic gel for cg. Digest using EcoRI and SpeI. Expected band size = 1250bp
Plate reader experiment for pg and negative control. Followed protocol. 50μl LB (not 40μl). OD 600
Transforms: Pmg – chlorophenicol into DH5α. Fck – chloroamphenicol into DH5α. Ptcg – chloroamphenicol into DH5α. mEx – amphicillin into DH5α. pcg – chloroamphenicol into DH5α. pBAD vector no insert = amphicillin MG1655
Week 8 8/08
Day 1
PCR ptcg, pmg, cg and fck. For ptcg – use old forward (EcoRI), new reverse (SpeI), 60°C annealing. For pmg – use old forward (EcoRI), new reverse (SpeI), 60°C annealing. For cg – use old forward (EcoRI), new reverse (SpeI), 60°C annealing. For fck – use old forward (EcoRI), new reverse (SpeI), 60°C annealing. All 1 min elongation time and ran 2 of each. Results: ptcg, pmg, cg and fck successful
Gel extract all 4 parts and mymT in expression (mEx)
Digestion for all 4 parts and mEx using EcoRI and SpeI for ptcg, pmg, cg and fck, XbaI and PstI for mEx
Ligation of ptcg, pmg, tc, cg, pcg, mEx and fck
Nanodrop cg and fck, 7.6 and 18.8, respectively. Start from PCR stage.
PCR m out of mg (retry) 62°C annealing – 15s, 72°C elongation – 15s
Day 2
Started using improved cloning procedures to attempt to get everything working
Starting from PCR with tc, cg, ptcg, pmg, pcg and fck. Old forward, new reverse primers for cg, ptcg, pmg, pcg and fck. Old forward, old reverse primer for tc.
Transform Fla-Art175 (part from last year) and tc(shipping) into DH5α, use chloroamphenicol. Fla-Art175 = 1120 bp. tc = 545 bp. PSB1C3C shipping plasmid backbone = 2070bp
Transform pmg, mEx and tc (pBAD) into DH5α, use chloroamphenicol for pmg and use ampicillin for mEx and tc.
Pick colonies of pBAD (no insert) to make pBAD stock. Pick x8
Day 3
Miniprep pBAD inset x6. Ran x2 samples from each picked colony. Digest with NcoI and PstI. Only digested using sample from 1a.
PCR tc, cg, ptcg, pcg 1, fcg 2. tc - use old forward, old reverse, 62o annealing. cg - use old forward, new reverse, 60°C annealing. ptcg - use old forward, new reverse, 62°C annealing. pcg 1 - use old forward, new reverse, 60°C annealing. fcg 2 - use old forward, new reverse, 62°C annealing.
Gel extract to make pBAD stock solution. Used 1a colony. Send undigested plasmid for sequencing. Put 1a gel in freezer. Gel extract if sequencing comes back correct.
Pick colonies of: Fla-Art175 (x4, chloroamphenicol), Tc (shipping) for vector (x4, chloroamphenicol), mEx (x6, amphicillin), Tc in pBAD (x6, amphicillin).
PCR tc out of shipping tc2 using ExP Rev and Exp Tat Csp1 primers, 62°C annealing. Ran gel → got one band on each so can do PCR purification rather than gel extraction
Day 4
Miniprep tc for shipping, Fla-Art175 shipping and mEx. Tc shipping = 1a, 1b, 2a, 2c, Fla-Art175 shipping = 3a, 3b, 4a, 4c, mEx = A, B, C, D, E
Digestion for above parts, 1a to 4b using EcoRI and SpeI. A to E using BamHI and PstI (use 2% gel)
Gel for Mex (A to E) came back successful, nanodropped A to E. C had highest value (35.5) → sent for sequencing
Gel for 1a-4a, 1a = tc, 2a = tc, 3a = Fla-Art175, 4a = Fla-art175. Kept 1a and 2a. Set up more to digest overnight
Pick colonies for mEx and pg (MG1655)
Day 5
Miniprep mEx (1-4), digest, diagnostic gel, sequence. Nandrop results: 1 = 95.8, 2 = 99.4,3 = 89.3, 4 = 93.5 Discarded 3.
Re-transform pg into MG1655
Run mEx gel, Ran mEx 1-4. Sending mEx2 for sequencing. Keep mEx1 just in case.
Day 7
Made more chloramphenicol
Pick colonies of fcg, cg, ptcg, fck, pcg, pmg and mEx, 6x of each, apart from pmg (only 1x). Chloramphenicol for all except mEx (ampicillin)
Week 9 15/08
Day 1
Miniprep  cg, fcg and ptcg
Digest cg, fcg and ptcg for diagnostic gel
Transform tc and pmg
Day 2
Pick colonies of mg and pmg, 2x of each, Chloramphenicol
Pick colonies of tc, 1x of each Chloramphenicol
Day 3
Miniprep 2x pmg and 1x tc Results: tc = 75.8, pmg a = 104.0, pmg b = 332.9
Digest tc and pmg a, using EcoRI and PstI. Ran gel with just pmg a One band at around 2000, probably shipping vector so hasn’t worked
Day 4
Send tc2 for sequencing  Successful!
Digest pmg a and b from Day 3 using EcoRI and PstI. Expected band size = 1533
Nanodrop pmg a and pmg b Results: 104.0 and 332.9, respectively  From gel, pmg b worked so sent off for sequencing
Digest fck 1-6 for diagnostic gel using EcoRI and PstI, both on 1% gel
PCR purify fck 1-6 1 volume 20μl, 3 volume 60ul 60ul QG 20μl isopropanol From gel – fck didn’t work
Day 5
Digest and diagnostic gel of mEx (Freezer has 1,3,4,5 and 6) using BamHI and PstI. Run diagnostic gel – expected size = 264bp
Week 10, 22/08
Day 1
Digest pg and fck using SpeI and no other enzyme (other half already done). Ran a gel, gel extracted and ligated pg and fck.
Day 2
Digest cg and m into shipping plasmid. Labelled c1-6 and m1-6. For cg use BamHI and PstI. For m use PstI and EcoRI. Run diagnostic gel for cg and m. Unsuccessful.Transformed pmg and fck into NEBα. Started cg, ptcg, m, pmg and fck from PCR again. Gel showed cg, pmg and ptcg were unsuccessful but fck and m were carried on and PCR purified, digested, purified again and ligated into the shipping vector. Plate reader for fcg and pcg, not a roaring success.
Day 3
Picked 3 fck colonies and 6 pmg colonies. PCRed pmg and ptcg again using correct restriction enzymes, ran gel, PCR purified, digested, purified and ligated into shipping vector. Transformed cg and fck into NEBɑ. SDS-PAGE of mg following protocol on protocols page. Picked colonies of fcg, pg and pcg for microscopy.
Day 4
Mini prepped colonies as per the protocol. Digested the miniprepped solutions using PstIHF and EcoRIHF. Ran diagnostic gels for pmg and fck. pmg gel smeared, inconclusive. fck3 had the correct band, sent for sequencing. Picked more colonies for pmg, cg and fck. Microscopy images of fcg, pg and pcg.
Day 5
Sequencing for fck unsuccessful. Mini prepped colonies as per the protocol. Digested the miniprepped solutions. Used PstIHF and EcoRIHF for fck and pmg. Used PstIHF and BamHI for cg. Ran diagnostic gels for pmg, fck and pmg. Sent cgG2, pmg4 and pmg9 for sequencing. Transformed ptcg and pmg again.
Day7
Picked colonies for ptcg and pmg and pcg and fcg for plate reader. Transformed pmg, cg and m into MG1655.
Week 11 29/09
Day 1
PCRed pCusC, purified, digested, purified and ligated. Re-ligated more fck. Plate reader for pcg and fcg. Microscopy of cg, pcg, mg and pCusC.
Day 2
Transformed pCusC into NEBɑ. Picked colonies for fck, pCusC and SDM. PCRed and SDMed pCusCRFP to remove illegal restriction site. Prepared mg and cg for protein purification.
Day 3
Mini prepped colonies as per the protocol  Digested the miniprepped solutions, used PstIHF and EcoRIHF for all colonies. Ran diagnostic gels for everything. Sent fckB4 for sequencing. Picked pCusC colonies. Transformed SDM colonies. Protein purification for cg and fcg.
Day 4
Miniprepped pCusC, digested, gel showed unsuccessful. SDM colonies picked. Ran SDS-PAGE of mg and cg. Bead Experiment, made pure alginate beads as a control.
Day 5
SDM miniprepped, digested, run on gel and sent for sequencing.
Day 6
Copper standard curve made.
Day 7
Made copper standard curve again
Week 12 5/09
Day 1
Started pCusC again from PCR, did PCR, diagnostic gel, PCR purified and digested with EcoRI and PstI. Transformed SDM into NEBɑ. Ran out of shipping vector so have to make more tomorrow. Initial copper chelation assays using BCS and ascorbate. Picked colonies of cg, mg, pBAD, NC, PC, fcg and pcg. Bead preparation with alginate and chitosan.
Day 2
Made new shipping vector digested with EcoRI and PstI. Ligated the digested pCusC with the new shipping vector. Picked SDM colonies. Plate reader of cg, mg, pBAD, NC, PC, fcg and pcg. Bead experiment to find a chitosan concentration that produces successful layers. Tested at different pHs.
Day 3
Transformed the ligated pCusC in NEBα. Miniprepped, digested and ran gel of SDM parts, all had a weird band so unsuccessful. Copper chelation assay for mg. Tested pH adjusted beads in the stomach pH solution.
Day 4
Picked pCusC colonies. Plate reader experiment for fck and pCusC. BCS standard assay attempt, failed. Picked colonies for ptcg, fck and pmg. Prepared reagents for bead experiments.
Day 5
Mini prepped colonies as per the protocol, digested the miniprepped solutions. Used PstIHF and EcoRIHF for all colonies. Ran diagnostic gels for everything. Nothing had the right band, it hasn’t worked. Microscopy of fck, pmg and ptcg. New bead experiments using 2% alginate
Day 7
Picked colonies for pCusC, fck and SDM Copper assasys
Week 13 12/09
Day 1
Mini prepped colonies as per the protocol. Digested the miniprepped solutions, used PstIHF and EcoRIHF for fck and pCusC and SDM. Ran diagnostic gels for everything fck3 and 5 have the right band, sent fck5 for sequencing. Successful bead experiment yay!
Day 2
Fck sequencing unsuccessful. Site directed mutagenesis of pCusC. Transformed SDMs into NEBα and fck and pg into MG1655 for plate reader experiments.
Day 3
Started pCusC from PCR again, diagnostic gel, PCR purified, digested with EcoRI and PstI, gel extracted again and then ligated into the shipping vector and left overnight. Transformed SDM into DH5α.
Day 4
Put transformed SDMs into the cold room. Transformed pCusC into DH5α. Miniprepped, digested and ran a gel for pg, sent for sequencing. Miniprepped, digested and ran a gel for fck, sent for sequencing.
Day 5
fck and pg sequencing failed. Miniprepped, digested and ran a gel for fck but all the gels were smears or in the wrong place. Left a plate reader running of fcg and pcg
Day 6
Ligated fck with the shipping vector digested with EcoRI and Spe.
Day 7
Transformed fck ligation into DH5α. Picked colonies for pCusC. Picked colonies for protein purification.
Week 14 19/09
Day 1
Miniprepped pCusC. Started p, pg and fck from PCR, diagnostic gel, PCR purified, digested, gel extracted and ligated. Picked more fck colonies. Protein purification day 2
Day 2
Miniprepped fck colonies, digested, diagnostic gel. Transformed fck, p, pg, pCusCRFP and pCusC. Protein purification day 3.
Day 3
Picked colonies from p, pg and pCusCRFP (pCusC and fck plates failed). Protein purification day 4.
Day 4
Miniprepped colonies, digested and ran a gel. All the p bands were in the correct place, sent one for sequencing. None of the other bands were in the right place. Protein purification 5. Plate reader for ptcg and pmg.
Day 5
p sequencing was successful. Gel extracted the new shipping vector digested with SpeI and EcoRI. Ligated the new shipping vector with more fck and pg repeats.
Day 6
Transformed fck and pg ligations. PCRed more pCusCRFP, ran on a gel, PCR purified, digested, cleaned up and then ligated into shipping vector.
Day 7
Picked colonies from fck and pg plates. Transformed pCusCRFP into DH5α.
Week 15 26/09
Day 1
Miniprepped pg and fck, ran a gel and digested. pg had expected bands so both sent for sequencing but fck did not have the expected bands. Picked more pCusCRFP and fck colonies. Ligated some more fck colonies. SDS-PAGE of mg and cg and gfp.
Day 2
Both pg sequences unsuccessful. Miniprepped fck and pCusCRFP colonies, digested and ran a gel but none of them had the expected bands. Transformed the new fck ligations.
Day 3
Picked colonies from the transformed fck plates. Started fck and pCusCRFP from PCR again because past performance suggests it won’t work anyway.
Day 4
Miniprepped fck, digested and run on a gel, ONE OF THEM HAD THE RIGHT BAND! Sent this for sequencing. pCusCRFP colonies did not grow so pCusCRFP was transformed again. Denaturation Gel.
Day 5
SEQUENCING FOR FCK WAS (very nearly: Val to Ala point mutation) SUCCESSFUL AT LAST! pCusCRFP plates were moved to the cold room.
Day 7
Colonies were picked from pCusCRFP
Week 16 3/10
Day 1
pCusCRFP was miniprepped, digested and sent for sequencing.
Day 2
Sequencing for pCusCRFP came back successfully. Both fck and pCusCRFP were transformed into MG1655.
Interlab
Day 1
Transformed the 5 interlab parts into DH5α as per the transformation protocol.
Day 2
Transformations mostly unsuccessful. Started making new competent cells.
Day 3
Second day of competent cell preparation, cells frozen and put in the freezer.
Day 4
LB media preparation and re-transformation of the interlab parts.
Day 5
Had colonies from all interlab parts except PC. Picked colonies from all successful plates. Retransformed PC.
Day 6
PC didn't transform again. After measuring the sample we were sent with nanodrop we found the sample has no DNA in it. Requested a second sample from iGEM HQ Nevertheless we did a third transformation for PC. NC and TD1-3 were miniprepped and sent for sequencing.
Day 7
TD1 sequencing successful, others had the wrong part in so retransformed these. PC colonies successful despite nanodrop so picked colonies for these.
Day 8
PC miniprepped, digested and sent for sequencing. Colonies picked for re-transformed TD2, TD3 and NC.
Day 9
PC sequencing unsuccessful. TD2, TD3 and NC miniprepped and sent for sequencing.
Day 10
TD2 sequencing successful, others not. PC, NC and TD3 transformed again
Day 11
PC, NC and TD3 colonies picked.
Day 12
PC, NC and TD3 miniprepped and sent for sequencing.
Day 13
NC and TD3 sequencing successful but PC wrong so retransformed again.
Day 14
Picked colonies for PC.
Day 15
PC was miniprepped and digested. Gel showed it contained the wrong part so we transformed the part from the distribution kit instead.
Day 16
Colonies picked for PC.
Day 17
PC miniprepped, digested and sent for sequencing. Produced more competent cells.
Day 18
PC sequencing correct! All 5 interlab parts were transformed into MG1655
Day 19
Made the calibration curves for the OD600 and the FITC standard curve Colonies picked for all the interlab parts.
Day 20
Tested iGEM's protocol for the OD600 and the fluorescence. Picked more colonies for all 5 parts.
Day 21
Repeated iGEM's protocol and started Oxford iGEM's protocol.
Day 22
Finished Oxford iGEM's protocol and submitted the data obtained using iGEM's method to iGEM
Made with gooood stuff.
 
 
