<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_Synthetic_Chromatin_Properties"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UCSF/Synthetic_Chromatin_Properties">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF/Synthetic_Chromatin_Properties&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Synthetic_Chromatin_Properties&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF/Synthetic_Chromatin_Properties&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF/Synthetic_Chromatin_Properties" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF/Synthetic Chromatin Properties</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TITLE>Untitled Document</TITLE></P><H1 align="center">Synthetic Chromatin Bit (Part II)</H1><P align="center"> </P><P><A href="https://2008.igem.org/Team:UCSF/Synthetic_Chromatin_Design">Design of our System (previous)</A></P><P> </P><BLOCKQUOTE><BLOCKQUOTE><H2 align="justify">Why use Chromatin as a Tool for Synthetic Biology?</H2><P align="justify">Transcriptional activators and repressors, the mainstays of current synthetic genetic circuits, work at the level of the promoter by affecting the recruitment of the basal transcription machinery. The eukaryotic cell, however, has a more potent mechanism to regulate gene expression: chromatin (gene silencing). Chromatin-based gene regulation has a number of interesting properties in vivo. We reasoned that it might be a powerful tool for synthetic biology. To explore this potential, we spent the summer developing a synthetic chromatin system, and demonstrating its unique features .</P><H2 align="justify">Analysis of our Data</H2><P align="justify">Single-cell analysis was done using flow-cytometry. Example cells positive for GFP are seen in the microscope images on the lower right panel  and their predicted population distribution is shown in the green curve in the ficticious graph in the left (below).</P><P align="justify"> </P><H2 align="justify">The Properties of Our Synthetic Chromatin Bit</H2><P align="justify">We tested our synthetic chromatin bit in a number of ways.</P><P align="justify"> </P><H3 align="justify">1. Targeting of Sir2 Leads to Complete Gene Silencing</H3><BLOCKQUOTE><P align="justify">In the eukaryotic cell, heterochromatin blocks gene expression completely. We targeted the silencing machinery to a transgenic locus and monitored reporter expression. </P><P align="justify"> </P></BLOCKQUOTE><BLOCKQUOTE><P align="justify"><STRONG>RESULT 1:</STRONG></P></BLOCKQUOTE><P align="justify"> </P><BLOCKQUOTE><P align="justify">After the addition of galactose (inducing LexA-Sir2 expression), we observed complete silencing of the GFP reporter. In this case, a medium constitutive promoter (Cyc1P) was used, but similar results were obtained for other promoters (e.g. Fig1 P, see below).</P></BLOCKQUOTE><H3 align="justify">2. Silencing in the chromatin bit is Dominant over Transcription Factors</H3><BLOCKQUOTE><P align="justify">Heterochromatin plays a primary role in the differentiation of the cells of higher eukaryotes, and therefore must be resistant to activation by transcription factors. We tested whether our synthetic chromatin bit, once closed (heterochromatin), could be activated. The GFP reporter in this case was driven by the pheromone-inducible Fig1 promoter. </P><P align="justify"> </P><P align="center"> </P></BLOCKQUOTE><P align="justify"> </P><BLOCKQUOTE><P align="justify"><STRONG>RESULT 2</STRONG>:</P></BLOCKQUOTE><P align="justify"> </P><BLOCKQUOTE><P align="justify">Transcriptional activation of the reporter gene was completely blocked in cells that were grown in galactose to induce silencing. Indeed, even the basal activity of the Fig1 promoter was blocked by silencing (compare the red and blue traces). </P></BLOCKQUOTE><H3 align="justify">3. Silencing in the Chromatin Bit is Regional/Bi-directional</H3><BLOCKQUOTE><P align="justify">Unlike typical transcriptional activators or repressors, silencing should be promoter independent, and thus should function equally well when initiated from upstream or downstream of a gene (i.e. silencing should be bi-directional). We used a dual reporter construct to test this possibility in our synthetic system.</P></BLOCKQUOTE><P align="justify"> </P><BLOCKQUOTE><P align="justify"><STRONG>RESULT 3:</STRONG></P><P align="left"> </P><P align="justify">The upstream mCherry and downstream GFP reporter were both silenced completely, indicating that silencing spreads (and functions) bi-directionally.</P></BLOCKQUOTE><H3 align="justify">4. Silencing in the Chromatin Bit Spreads From the Point of Initation</H3><BLOCKQUOTE><P align="justify">In S. cerevisiae, spreading from endogenous sites of initation at the telomere ends occurs to ~3 Kb. We tested the range of our synthetic system using a series of spacer constructs, where the LexA operators (initiation of silencing) were 250-3,000 bp downstream of the GFP reporter.</P><P align="justify"><STRONG>RESULT 4:</STRONG></P><P align="justify"> </P><P align="justify">Silencing was robust to 2,000 bp. At 3,000 bp distance, the cells were bi-modal for GFP expression, indicating that the limit is between 2,000 and 3000 bp. We are currently investigating &quot;boundary elements&quot;--DNA sequences that work at endogenous sites to limit the spread of heterochromatin. If modular, these elements would be useful for determining the size of the chromatin bit.</P></BLOCKQUOTE><H3 align="justify">5. Silencing in the Chromatin Bit is Binary/Switch-Like</H3><BLOCKQUOTE><P align="justify">Unlike standard transcriptional repressors which yield low to moderately cooperative repression, we noticed a distinct lack of intermediate GFP expression levels in our silencing experiments. To determine how cooperative the silencing transition is, we assayed silencing over a wide range of steady-state LexA-Sir2-mCherry expression levels. This required graded output from the Gal-inducible promoter driving the LexA-Sir2-mCherry construct, and we used a delta Gal2 strain (Hawkins and Smolke, 2006--see Materials and Methods). The mean GFP expression was plotted against the LexA-Sir2-mCherry expression.</P><P align="justify"><STRONG>RESULT 5:</STRONG></P><P align="justify"> </P><P align="justify">Our results show that silencing is ultra-cooperative, indicated by the &quot;sharpness&quot; of the transition in the the curve shown above. The data were fitted to a standard model for transcriptional repression. The Hill coefficient (n) calculated here is significantly higher than that seen for typical transcriptional repressors. However, we reasoned that this calculation from population level measurements might understate the actual cooperativity of silencing, and thus we replotted the data where each point represents a single cell. The top plot shows a composite graph of three different clones. We plotted the clones separately below, fitted to the same model, and then compared the Hill coefficients for each. </P><P align="justify"> </P><P align="justify">Indeed, we observed higher n values from these fits, indicating that population level measurements may underestimate the cooperativity of switching between ON and OFF states. The single cell analysis is preliminary--there are several caveats to fitting such noisy data. However, these experiments clearly show that silencing is ultra-cooperative.</P></BLOCKQUOTE><H3 align="justify"> </H3><H3 align="justify">6. Silencing in the Chromatin Bit Exhibits Short Term Memory</H3><BLOCKQUOTE><P align="justify">In the differentiating mammalian cell, changes in heterochromatin are &quot;locked-in&quot;, in some cases for the lifetime of the organism. Of course, yeast don't live for 80 years, but we tested the duration of memory after transient induction of silencing in the chromatin bit.</P><P align="justify"><STRONG>RESULT 6:</STRONG></P><P align="justify"> </P><P align="justify">After withdrawal of galactose, we observed persistence of silencing for six hours, corresponding to ~3 cell cycles in our yeast. This short-term memory might be improved by the use of additional feedback, something that we would like to explore in the future.</P><P align="right"> </P><P align="right"><A href="https://2008.igem.org/Team:UCSF/Modeling">Modeling (next)</A></P></BLOCKQUOTE></BLOCKQUOTE></BLOCKQUOTE><TABLE style="color:#333333;background-color:#cccccc;" cellpadding="3" cellspacing="3" border="0" width="99%" align="center"><TBODY><TR><TH align="center"><A href="/Team:UCSF" title="Team:UCSF">Home</A></TH><TH align="center"><A href="/Team:UCSF/Team" title="Team:UCSF/Team">The Team</A></TH><TH align="center"><A href="/Team:UCSF/Project" title="Team:UCSF/Project">The Project</A></TH><TH align="center"><A href="/Team:UCSF/Parts" title="Team:UCSF/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:UCSF/Modeling" title="Team:UCSF/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:UCSF/Human_Practices" title="Team:UCSF/Human Practices">Human Practices</A></TH><TH align="center"><A href="/Team:UCSF/Notebook" title="Team:UCSF/Notebook">Notebooks</A></TH></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UCSF/Synthetic_Chromatin_Properties">http://2008.igem.org/Team:UCSF/Synthetic_Chromatin_Properties</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF/Synthetic_Chromatin_Properties" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF/Synthetic_Chromatin_Properties" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF/Synthetic_Chromatin_Properties&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF/Synthetic_Chromatin_Properties&amp;oldid=102869" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>