"
Page
Discussion
View source
History
teams
Log in
 
Team:UCSF/Project
From 2008.igem.org
Project Summary
The goals of synthetic biology are 1) to gain a deeper understanding of native biological systems, and 2) to develop new ones.  In doing so, most devices and systems have been constructed by co-opting traditional genetic elements, where transcriptional activators and/or repressors modulate the expression of genes.  While this has proved useful and reliable for many synthetic systems, there are additional mechanisms for transcriptional regulation that synthetic biologists have yet to harness.
This year, our team is attempting to engineer epigenetic control of gene expression. In eukaryotic cells, DNA is wound around nucleosomes, protein "spools" that consist of an octamer of histones. The DNA and protein together, termed chromatin, can be tightly packgaged (heterochromatin) or more loosely arranged (eucharomatin). The density of nucleosomal packaging is signaled by a host of histone modifying enzymes, and enforced by chromatin remodeling complexes.  Euchromatin is accessible to the transcriptional machinery (active) while heterochromatin is inaccessible, and refractory to transcription (silenced).
In nature, the modulation of gene expression via the alteration of DNA structure is an incredibly powerful form of cellular memory. Indeed, epigenetic changes regulating genome-wide expression patterns can persist through multiple rounds of cell division and remain for the lifetime of the cell.  This mechanism allows embryonic stem cells to differentiate into myriad cell types in higher eukaryotes.
For our project, we are establishing, characterizing and standardizing methods to engineer epigenetic control in the eukaryotic yeast Saccharomyces cerevisiae.  To do so, we are taking endogenous proteins known to modify chromatin structure, such as Sir2, an NAD+ dependent histone deacetylase, and devising methods to control and direct their activity.  We are concentrating on generating a chromatin toolkit that can:
silence (or “close”) euchromatin
operate in a promoter/orientation-independent manner
operate regionally (spread to silence one or more genes)
retain memory (i.e. persistence past a transient stimulus)
initiate silencing or activation through various extracellular cues
link changes in heterochromatin to distinct biological outputs
We believe that the ability to control the structure of chromatin will allow synthetic biologists to engineer robust systems with novel and predictable behaviors.  We look forward to introducing and discussing these ideas in November!
More on Synthetic Chromatin
(Very) Selected Heterochromatin References:
Hecht, A., Strahl-Bolsinger, S., and M. Grunstein, Spreading of transcriptional repressor Sir3 from telomoeric heterochromatin. Nature, 1996. 383: p. 92-96.
Howitz, KT., Bitterman, KJ., Cohen, HY., Lamming, DW., Lavu, S., Wood, JG., Zipkin, R.E., Chung, P., Kisielewski, A., Zhang, LL., Scherer, B., and DA. Sinclair, Small molecule activators of sirtuins extend Saccharomyces cerevisiae lifespan. Nature, 2003. 425(6954): p.191-6.
Kornberg, R.D. and Y. Lorch, Twenty-five years of the nucleosome, fundamental particle of the eukaryote chromosome. Cell, 1999. 98(3): p. 285-94.
Jenuwein, T. and C.D. Allis, Translating the histone code. Science, 2001. 293(5532): p. 1074-80.
Rusche, L.N., A.L. Kirchmaier, and J. Rine, The establishment, inheritance, and function of silenced chromatin in Saccharomyces cerevisiae. Annu Rev Biochem, 2003. 72: p. 481-516.
Moazed, D., Common themes in mechanisms of gene silencing. Mol Cell, 2001. 8(3): p. 489-98.
Pirrotta, V. and D.S. Gross, Epigenetic silencing mechanisms in budding yeast and fruit fly: different paths, same destinations. Mol Cell, 2005. 18(4): p. 395-8.
Home
The Team
The Project
Parts Submitted to the Registry
Modeling
Human Practices
Notebooks
Retrieved from " http://2008.igem.org/Team:UCSF/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
