<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Szeged_SA_RMG_references skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Szeged SA RMG/references</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Szeged_SA_RMG/members">Members</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/mentors">Mentors</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/collaboration">Collaborations</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/pictures">Gallery</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Szeged_SA_RMG/summary">Summary</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/articles">Article</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/bioinfo">Bioinformatics</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/biobricks">Biobricks</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/presentation">Presentation</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/poster">Poster</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/references">References</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Human Practices</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Szeged_SA_RMG/survey">Survey</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/methane">Methane</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/highschooloutreach">High School Outreach</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/universityoutreach">University Outreach</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Laboratory</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Szeged_SA_RMG/methods">Methods</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/labequipments">Lab Equipments</A><A href="https://2017.igem.org/Team:Szeged_SA_RMG/safety">Safety</A></DIV></DIV></UL><UL class="nav navbar-nav navbar-right"><LI><A href="http:/www.methungeny.com">METHUNGENY</A></LI></UL></DIV></DIV><CENTER><H3>The Scientific Articles used by our Team</H3></CENTER><DIV class="w3-row-padding w3-light-grey w3-padding-64 w3-container"><DIV class="w3-content"><P>1.        Alit, H. &amp; Murrell, J. C. Development and validation of promoter-probe vectors for the study of methane monooxygenase gene expression in Methylococcus capsulatus Bath. Microbiology 155, 761–771 (2009).
</P><P>2.        Anthony, C. &amp; Williams, P. The structure and mechanism of methanol dehydrogenase. in Biochimica et Biophysica Acta - Proteins and Proteomics 1647, 18–23 (2003).</P><P>3.        Baani, M. &amp; Liesack, W. Two isozymes of particulate methane monooxygenase with different methane oxidation kinetics are found in Methylocystis sp. strain SC2. Proc. Natl. Acad. Sci. 105, 10203–10208 (2008).</P><P>4.        Boettcher, M. &amp; McManus, M. T. Choosing the Right Tool for the Job: RNAi, TALEN, or CRISPR. Molecular Cell (2015). doi:10.1016/j.molcel.2015.04.028</P><P>5.        Bowman, J. P., Sly, L. I., Nichols, P. D. &amp; Hayward,  a. C. Revised Taxonomy of the Methanotrophs: Description of Methylobacter gen. nov., Emendation of Methylococcus, Validation of Methylosinus and Methylocystis Species, and a Proposal that the Family Methylococcaceae Includes Only the Group I Methanotrophs. Int. J. Syst. Bacteriol. 44, 375–375 (1994).</P><P>6.        Brantl, S. Antisense-RNA regulation and RNA interference. Biochim Biophys Acta 1575, 15–25. (2002).</P><P>7.        Cardy, D. L. N., Laidler, V., Salmond, G. P. C. &amp; Murrell, J. C. The methane monooxygenase gene cluster of Methylosinus trichosporium: cloning and sequencing of the mmoC gene. Arch. Microbiol. 156, 477–483 (1991).</P><P>8.        Cardy, D. L. N., Laidler, V., Salmond, G. P. C. &amp; Murrell, J. C. Molecular analysis of the methane monooxygenase (MMO) gene cluster of Methylosinus trichosporium OB3b. Mol. Microbiol. 5, 335–342 (1991).
</P><P>9.        Carter, P. Site-directed mutagenesis. Biochem. J. 237, 1–7 (1986).</P><P>10.      Coufal, D. E. et al. Sequencing and analysis of the Methylococcus capsulatus (Bath) soluble methane monooxygenase genes. Eur. J. Biochem. 267, 2174–2185 (2000).</P><P>11.      Culpepper, M. A. &amp; Rosenzweig, A. C. Structure and Protein–Protein Interactions of Methanol Dehydrogenase from\n Methylococcus capsulatus\n (Bath). Biochemistry 53, 6211–6219 (2014).</P><P>12.      Csáki, R., Bodrossy, L., Klem, J., Murrell, J. C. &amp; Kovács, K. L. Genes involved in the copper-dependent regulation of soluble methane monooxygenase of Methylococcus capsulatus (Bath): Cloning, sequencing and mutational analysis. Microbiology (2003). doi:10.1099/mic.0.26061-0</P><P>13.      de Souza, N. Prokaryotic RNAi. Nat. Publ. Gr. 9, 220–221 (2012).</P><P>14.      Dietrich, P., Sanders, D. &amp; Hedrich, R. The role of ion channels in light-dependent stomatal opening. J. Exp. Bot. 52, 1959–1967 (2001).</P><P>15.      Farhan Ul Haque, M., Gu, W., DiSpirito, A. A. &amp; Semrau, J. D. Marker exchange mutagenesis of mxaF, encoding the large subunit of the Mxa methanol dehydrogenase, in Methylosinus trichosporium OB3b. Appl. Environ. Microbiol. 82, 1549–1555 (2016).</P><P>16.      Fire, A. et al. Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans. Nature 391, 806–811 (1998).</P><P>17.      Fodor, B. D. et al. Modular Broad-Host-Range Expression Vectors for Single-Protein and Protein Complex Purification. 70, 712–721 (2004).</P><P>18.      Foster, J. W. &amp; Davis, R. H. A methane-dependent coccus, with notes on classification and nomenclature of obligate, methane-utilizing bacteria. J. Bacteriol. 91, 1924–1931 (1966).</P><P>19.      Fox, B. G., Borneman, J. G., Wackett, L. P. &amp; Lipscomb, J. D. Haloalkene oxidation by the soluble methane monooxygenase from Methylosinus trichosporium OB3b: mechanistic and environmental implications. Biochemistry 29, 6419–6427 (1990).</P><P>20.      Giver, L. Methane and Methanotrophic Bacteria as a Biotechnological Platform Calysta overview. (2015).</P><P>21.      Good, L. &amp; Stach, J. E. M. Synthetic RNA silencing in bacteria - antimicrobial discovery and resistance breaking. Front. Microbiol. 2, 1–11 (2011).</P><P>22.      Guyton, A. C. &amp; Hall, J. E. Textbook of Medical Physiology. 2011 XXXIII, (2011).</P><P>23.      Hammond, S. M. Dicing and slicing: The core machinery of the RNA interference pathway. FEBS Letters 579, 5822–5829 (2005).</P><P>24.      Hanson, R. S. &amp; Hanson, T. E. Methanotrophic bacteria. Microbiol. Rev. 60, 439–471 (1996).</P><P>25.      Harwood, J. H., Williams, E. &amp; Bainbridge, B. W. Mutation of the Methane Oxidizing Bacterium, Methylococcus capsulatus. J. Appl. Bacteriol. 35, 99–108 (1972).</P><P>26.      Henard, C. A. et al. Bioconversion of methane to lactate by an obligate methanotrophic bacterium. Sci. Rep. 6, 21585 (2016).</P><P>27.      Henckel, T. et al. Molecular Analyses of the Methane-Oxidizing Microbial Community in Rice Field Soil by Targeting the Genes of the 16S rRNA , Particulate Methane Monooxygenase , and Methanol Dehydrogenase Molecular Analyses of the Methane-Oxidizing Microbial Community in R. Appl. Environ. Microbiol. 65, 1980–1990 (1999).</P><P>28.      Herrero, M., De Lorenzo, V. &amp; Timmis, K. N. Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in gram-negative bacteria. J. Bacteriol. (1990). doi:10.1128/jb.172.11.6557-6567.1990</P><P>29.      Kalyuzhnaya, M. G., Puri, A. W. &amp; Lidstrom, M. E. Metabolic engineering in methanotrophic bacteria. Metab. Eng. 29, 142–152 (2015).</P><P>30.      Keates, A. C., Fruehauf, J., Xiang, S. &amp; Li, C. J. TransKingdom RNA interference: a bacterial approach to challenges in RNAi therapy and delivery. Biotechnol. Genet. Eng. Rev. 25, 113–27 (2008).</P><P>31.      Kelly, W. J. et al. in Genome Announcements 4, e00232-16 (2016).</P><P>32.      Khalifa, A. Y. Z. Mutagenesis of a Copper P-Type ATPase Encoding Genein Methylococcus capsulatus (Bath) Results inCopper-Resistance. Int. J. Biosci. Biochem. Bioinforma. (2013). doi:10.7763/IJBBB.2013.V3.159</P><P>33.      Kitmitto, A., Myronova, N., Basu, P. &amp; Dalton, H. Characterization and structural analysis of an active particulate methane monooxygenase trimer from Methylococcus capsulatus (Bath). Biochemistry 44, 10954–10965 (2005).</P><P>34.      Larsen, ??ivind &amp; Karlsen, O. A. Transcriptomic profiling of Methylococcus capsulatus (Bath)during growth with two different methane monooxygenases. Microbiologyopen 5, 254–267 (2016).</P><P>35.      Lawton, T. J. &amp; Rosenzweig, A. C. Methane-Oxidizing Enzymes: An Upstream Problem in Biological Gas-to-Liquids Conversion. J. Am. Chem. Soc. 138, 9327–9340 (2016).</P><P>36.      Lemos, S. S., Perille Collins, M. L., Eaton, S. S., Eaton, G. R. &amp; Antholine, W. E. Comparison of EPR-visible Cu(2+) sites in pMMO from Methylococcus capsulatus (Bath) and Methylomicrobium album BG8. Biophys. J. 79, 1085–1094 (2000).</P><P>37.      Liu, T. ENZYMATIC SYNTHESIS OF POLY(LACTIC ACID) BASED POLYESTER CAPABLE OF FUNCTIONALIZATION. (2015).</P><P>38.      Lloyd, J. S., Bhambra, A., Murrell, J. C. &amp; Dalton, H. Inactivation of the regulatory protein B of soluble methane monooxygenase from Methylococcus capsulatus (Bath) by proteolysis can be overcome by a Gly to Gln modification. Eur. J. Biochem 248, 72–79 (1997).</P><P>39.      Lo, H. C., Haskel, A., Kapon, M. &amp; Keinan, E. TpPt(IV)Me(H)2 forms a σ-CH4 complex that is kinetically resistant to methane liberation. J. Am. Chem. Soc. 124, 3226–3228 (2002).</P><P>40.      M??ller, J. E. N. et al. Engineering Escherichia coli for methanol conversion. Metab. Eng. 28, 190–201 (2015).</P><P>41.      Macalady, J. L., McMillan, A. M. S., Dickens, A. F., Tyler, S. C. &amp; Scow, K. M. Population dynamics of type I and II methanotrophic bacteria in rice soils. Environ. Microbiol. (2002). doi:10.1046/j.1462-2920.2002.00278.x</P><P>42.      Makarova, K. S., Grishin, N. V, Shabalina, S. A., Wolf, Y. I. &amp; Koonin, E. V. A putative RNA-interference-based immune system in prokaryotes: computational analysis of the predicted enzymatic machinery, functional analogies with eukaryotic RNAi, and hypothetical mechanisms of action. Biol. Direct 1, 7 (2006).</P><P>43.      Manuscript, A., Thomason, M. K. &amp; Storz, G. Bacterial antisense RNAs: How many are there and what are they doing? Analysis 324, 167–188 (2011).</P><P>44.      Martin, H. &amp; Murrell, J. C. Methane monooxygenase mutants of Methylosinus trichosporium constructed by marker-exchange mutagenesis. FEMS Microbiol. Lett. 127, 243–248 (1995).</P><P>45.      Marx, C. J. &amp; Lidstrom, M. E. Development of improved versatile broad-host-range vectors for use in methylotrophs and other gram-negative bacteria. Microbiology 147, 2065–2075 (2001).</P><P>46.      Mocellin, S. &amp; Provenzano, M. RNA interference: learning gene knock-down from cell physiology. J. Transl. Med. 2, 39 (2004).</P><P>47.      Morita, T., Mochizuki, Y. &amp; Aiba, H. Translational repression is sufficient for gene silencing by bacterial small noncoding RNAs in the absence of mRNA destruction. Proc. Natl. Acad. Sci. U. S. A. 103, 4858–63 (2006).</P><P>48.      Murrell, J. C. Genetics and molecular biology of methanotrophs. FEMS Microbiol. Rev. 88, 233–248 (1992).</P><P>49.      Nakamura, T. et al. Soluble and particulate methane monooxygenase gene clusters in the marine methanotroph Methylomicrobium sp. strain NI. FEMS Microbiol. Lett. 277, 157–164 (2007).</P><P>50.      Neuwirth, G. &amp; Szemerszki, M. Tehetséggondozás a középiskolában, tehetségek a felsőoktatásban. Educatio 18, 204–218 (2009).</P><P>51.      Omay, D. &amp; Guvenilir, Y. Synthesis and characterization of poly(d,l-lactic acid) via enzymatic ring opening polymerization by using free and immobilized lipase. Biocatal. Biotransformation 31, 132–140 (2013).</P><P>52.      Park, D. &amp; Lee, J. Biological conversion of methane to methanol. Korean J. Chem. Eng. 30, 977–987 (2013).</P><P>53.      Patel, R. N. &amp; Hoare, D. S. Oxidation of C-1 Compounds capsulatus Physiological Studies of Methane and Methanol- Oxidizing Bacteria : Oxidation of C-1 Compounds by Methylococcus caps ulatus. J. Bacteriol. 107, 187–192 (1971).</P><P>54.      Pratt, A. J. &amp; MacRae, I. J. The RNA-induced silencing complex: A versatile gene-silencing machine. J. Biol. Chem. 284, 17897–17901 (2009).</P><P>55.      Puri, A. W. et al. Genetic tools for the industrially promising methanotroph Methylomicrobium buryatense. Appl. Environ. Microbiol. 81, 1775–1781 (2015).</P><P>56.      Rasmussen, L. C. V., Sperling-Petersen, H. U. &amp; Mortensen, K. K. Hitting bacteria at the heart of the central dogma: sequence-specific inhibition. Microb. Cell Fact. 6, 24 (2007).</P><P>57.      Rhee, M. S. et al. Complete Genome Sequence of a thermotolerant sporogenic lactic acid bacterium, Bacillus coagulans strain 36D1. Stand. Genomic Sci. (2011). doi:10.4056/sigs.2365342</P><P>58.      Róbert, C. et al. Szolubilis metán monooxigenáz rézfüggő szabályozásának vizsgálata Methylococcus capsulatus (Bath) törzsben. (2005).</P><P>59.      Sander, J. D. &amp; Joung, J. K. CRISPR-Cas systems for editing, regulating and targeting genomes. Nat. Biotechnol. 32, 347–55 (2014).</P><P>60.      Sander, J. D. &amp; Joung, J. K. CRISPR-Cas systems for editing, regulating and targeting genomes. Nat. Biotechnol. 32, 347–55 (2014).</P><P>61.      Sauer, N. J. et al. Oligonucleotide-mediated genome editing provides precision and function to engineered nucleases and antibiotics in plants. Plant Physiol. 170, pp.01696.2015 (2016).</P><P>62.      Sharpe, P. L. et al. Use of transposon promoter-probe vectors in the metabolic engineering of the obligate methanotroph Methylomonas sp. strain 16a for enhanced C 40 carotenoid synthesis. Appl. Environ. Microbiol. 73, 1721–1728 (2007).</P><P>63.      Sirajuddin, S. &amp; Rosenzweig, A. C. Enzymatic oxidation of methane. Biochemistry 54, 2283–2294 (2015).</P><P>64.      Stainthorpe, A. C., Lees, V., Salmond, G. P. C., Dalton, H. &amp; Murrell, J. C. The methane monooxygenase gene cluster of Methylococcus capsulatus (Bath). Gene 91, 27–34 (1990).</P><P>65.      Stolyar, S., Franke, M. &amp; Lidstrom, M. E. Expression of individual copies of Methylococcus capsulatus Bath particulate methane monooxygenase genes. J. Bacteriol. 183, 1810–1812 (2001).</P><P>66.      Stolyar, S., Costello, A. M., Peeples, T. L. &amp; Lidstrom, M. E. Role of multiple gene copies in particulate methane monooxygenase activity in the methane-oxidizing bacterium Methylococcus capsulatus Bath. Microbiology (1999). doi:10.1099/13500872-145-5-1235</P><P>67.      Strong, P. J., Xie, S. &amp; Clarke, W. P. Methane as a resource: Can the methanotrophs add value? Environmental Science and Technology 49, 4001–4018 (2015).</P><P>68.      Taguchi, S. et al. A microbial factory for lactate-based polyesters using a lactate-polymerizing enzyme. Proc. Natl. Acad. Sci. 105, 17323–17327 (2008).</P><P>69.      Timberlake, K. C. Chemistry - An Introduction to General, Organic and Biolgical Chemistry. (2015).</P><P>70.      University of Kentucky. Molecular Absorption Spectroscopy : Determination of Iron with 1,10-phenanthroline. 30–34
</P><P>71.      Unniyampurath, U., Pilankatta, R. &amp; Krishnan, M. N. RNA interference in the age of CRISPR: Will CRISPR interfere with RNAI? International Journal of Molecular Sciences 17, (2016).</P><P>72.      Wang, Y., Tseng, C., Chen, Y., Huang, D. &amp; Chan, S. I. Production of High-Quality Particulate Methane Monooxygenase in High Yields from. 185, 5915–5924 (2003).</P><P>73.      Ward, N. et al. Genomic insights into methanotrophy: The complete genome sequence of Methylococcus capsulatus (Bath). PLoS Biol. (2004). doi:10.1371/journal.pbio.0020303</P><P>74.      Wenyan Jiang, David Bikard, David Cox, Feng Zhang,  and L. A. M. CRISPR-assisted editing of bacterial genomes. Nat biotechnol 31, 233–239 (2013).</P><P>75.      West, C. A., Salmond, G. P. C., Dalton, H. &amp; Murrell, J. C. Functional expression in {Escherichia}-coli of protein-{B} and protein-{C} from soluble methane monooxygenase of {Methylococcus}-capsulatus (bath). J Gen Microbiol 138, 1301–1307 (1992).</P><P>76.      West, C. A., Salmond, G. P. C., Dalton, H. &amp; Murrell, J. C. Functional expression in Escherichia coli of proteins B and C from soluble methane monooxygenase of Methylococcus capsulatus (Bath). J. Gen. Microbiol. 138, 1301–1 (1992).</P><P>77.      Whittenbury, R., Phillips, K. C. &amp; Wilkinson, J. F. Enrichment , Isolation and Some Properties of Methane-utilizing Bacteria. J. Gen. Microbiol. 61, 205–218 (1970).</P><P>78.      Whittenbury, R., Phillips, K. C. &amp; Wilkinson, J. F. Enrichment, Isolation and Some Properties of Methane-utilizing Bacteria. J Gen Microbiol 61, 205–218 (1970).</P><P>79.      Williams, E. &amp; Bainbridge, B. W. Genetic transformation in Methylococcus capsulatus. J. Appl. Bacteriol. 34, 683–687 (1971).</P><P>80.      Yan, X., Chu, F., Puri, A. W., Fu, Y. &amp; Lidstrom, M. E. Electroporation-based genetic manipulation in type I methanotrophs. Appl. Environ. Microbiol. 82, 2062–2069 (2016).</P><P>81.      Yanagihara, K. et al. Effects of short interfering RNA against methicillin-resistant Staphylococcus aureus coagulase in vitro and in vivo. J. Antimicrob. Chemother. (2006). doi:10.1093/jac/dki416</P><P>82.      Yang, J. E., Choi, S. Y., Shin, J. H., Park, S. J. &amp; Lee, S. Y. Microbial production of lactate-containing polyesters. Microb. Biotechnol. 6, 621–636 (2013).</P><P>83.      Ye, R. W. et al. Construction of the astaxanthin biosynthetic pathway in a methanotrophic bacterium Methylomonas sp. strain 16a. J. Ind. Microbiol. Biotechnol. 34, 289–299 (2007).</P><P>84.      Zahn, J. A. &amp; Dispirito, A. A. Membrane-associated methane monooxygenase from Methylococcus capsulatus ( Bath ). Membrane-Associated Methane Monooxygenase from Methylococcus capsulatus ( Bath )†. J. Bacteriol. 178, 1018–1029 (1996).</P><P>85.      Zhang, H., Kolb, F. A., Jaskiewicz, L., Westhof, E. &amp; Filipowicz, W. Single processing center models for human Dicer and bacterial RNase III. Cell 118, 57–68 (2004).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>