<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hong_Kong_HKUST_characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Hong_Kong_HKUST/characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hong_Kong_HKUST/characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/characterization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hong_Kong_HKUST/characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hong Kong HKUST/characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="menu"><UL class="first"><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST">Home</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/members">Members</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/instructors">Instructors</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/attribution">Attribution</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/acknowledge">Acknowledgement</A></LI></UL><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/abstract">Abstract</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/modules">Modules Description</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/data">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/characterization">Characterization</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/results">Result</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/future">Future Work</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/Wetlab">Wetlab</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/achievement">Achievement</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/medals">Medal Requirements</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/prizes">Prizes</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/cp">Country Profile</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/blog">Blog</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/interview">Interviews</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/article/genet">Article</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/video">Videos</A></LI><LI><A href="https://2013.igem.org/Team:Hong_Kong_HKUST/hp/presentation">Presentations</A></LI></DIV><DIV id="slide"><CENTER><H3 class="title">Characterizations</H3></CENTER><H1>Mitochondrial Leader Sequence (<A href="http://parts.igem.org/Part:BBa_K1119000">BBa_K1119000</A>, <A href="http://parts.igem.org/Part:BBa_K1119001">BBa_K1119001</A>)</H1><H3>Introduction</H3><P id="yo">In our characterization, the coding DNA sequence (CDS) of MLS was assembled in frame with that of GFP reporter using Freiburg’s RFC25 format (<A href="http://parts.igem.org/Part:BBa_K648013">BBa_K648013</A>). The translation unit was driven by CMV promoter (<A href="http://parts.igem.org/Part:BBa_K1119006">BBa_K1119006</A>), and terminated by hGH polyA signal (<A href="http://parts.igem.org/Part:BBa_K404108">BBa_K404108</A>).
The aforementioned construct (<A href="http://parts.igem.org/Part:BBa_K1119009">BBa_K1119009</A>) was then transfected into HEK293FT cells. Mitochondria were stained with MitoTracker® Red CMXRos dye after transfection and co-localization between the GFP signal and that of the dye was determined as the area of signal overlap.
To provide a positive control, the CDS of EGFP from pEGFP-N1 (Clontech) was inserted downstream and in frame with the CDS of the MLS in the commercial plasmid pCMV/<I>myc</I>/mito (Invitrogen, Carlsbard, CA). Our negative control construct was the same as our experimental construct, but minus the MLS CDS. (<A href="http://parts.igem.org/Part:BBa_K1119008">BBa_K1119008</A>).
</P><P id="yo">MLS is submitted in RFC25 standard (<A href="http://parts.igem.org/Part:BBa_K1119001">BBa_K1119001</A>) to facilitate fusing with other CDS. MLS in RFC10 standard (<A href="http://parts.igem.org/Part:BBa_K1119000">BBa_K1119000</A>) is submitted as alternative but it cannot be fused directly to other CDS due to limitations in RFC10. Users who obtained the part in RFC10 standard can amplify the part by PCR and fuse it to other domains using overlapping PCR. 	
</P><P id="yo">To quantify the amount of signal overlapped between the GFP signal and the MitoTracker® dye, we adopted the method described by A.P. French et al. in “Colocalization of fluorescent markers in confocal microscope images of plant cells” (French et al., 2008). With the use of the Pearson-Spearman correlation colocalization plugin for ImageJ image processing, scatter plots of the green intensities (y-axis) and red intensities (x-axis), Pearson's correlation coefficient and Spearman's correlation coefficient were generated.</P><H3>Result</H3><P id="yo"><B>Figure 1. MLS directs GFP into mitochondria.</B> When MLS is added to the N-terminus of GFP, the GFP was directed to the mitochondria in the cells, giving patches of GFP signal that overlapped with the signals from MitoTracker®. When MLS is not added to the GFP, the GFP signal can be seen scattered all around inside the cell. Scale bar = 10µm.</P><P id="yo"><B>Figure 2. Scatter plots of fluorescence intensities of green (y-axis) and red (x-axis) by construct.</B> It showed that the BioBrick MLS-GFP and commercial MLS-GFP construct had a linear relationship of green intensities and red intensities while the GFP generator alone had no relationship. Pearson's correlation coefficient (rp) and Spearman's correlation coefficient (rs) were determined using the Pearson-Spearman correlation colocalization plugin (French et al., 2008) for ImageJ with a threshold of 0 and listed for each image.</P><P id="yo"><B>Figure 3. Calculated mean Pearson's (rp) and Spearman's (rs) correlation coefficients for each construct.</B> The coefficients were generated using ImageJ software and specific plugins. For every batch of transfected cells, four samples were used for quantification. Experimental BioBrick MLS-GFP and commercial MLS-GFP: coefficients were close to 1, good colocalization; GFP: Coefficients were close to 0, poor colocalization. Error bars show standard deviation.</P><H1>CMV Promoter (<A href="http://parts.igem.org/Part:BBa_K1119006">BBa_K1119006</A>)</H1><H3>Introduction</H3><P id="yo">For this promoter's characterization we assembled it with GFP reporter (<A href="http://parts.igem.org/Part:BBa_K648013">BBa_K648013</A>) and hGH polyA terminator (<A href="http://parts.igem.org/Part:BBa_K404108">BBa_K404108</A>).
The P<I>cmv</I>-GFP was then transfected into HEK293FT cells and the <I>in vivo</I> green fluorescence signal was observed under confocal microscope.
The positive control was pEGFP-N1 (Clontech) that contains CMV promoter and EGFP reporter. The negative control was the same as the experimental construct, but minus the promoter.
The <A href="https://2013.igem.org/Team:Hong_Kong_HKUST/protocols">detailed protocols</A> employed for our characterization work can be accessed through the link.</P><DIV class="row"><DIV class="nine columns"><H3>Result</H3><P id="yo"><B>Figure 1. CMV promoter drives expression of GFP.</B> HEK293FT cells transfected with P<I><SUB>cmv</SUB></I>-GFP gave GFP signals. HEK293FT cells transfected with the commercial pEGFP-N1 showed similar results, while the same construct without any promoter did not give any GFP signals. Scale bar = 10µm.</P></DIV></DIV><H1>EF-1alpha Promoter (<A href="http://parts.igem.org/Part:BBa_K1119010">BBa_K1119010</A>)</H1><H3>Introduction</H3><P id="yo">The constitutive human Elongation Factor-1alpha (EF-1alpha) Promoter regulates gene expression in mammalian cells. It is known that the CMV promoter is commonly used for constitutive expression, and here we introduce EF-1alpha promoter as an alternative mammalian promoter, which works in a wide range of cell types. The origin of this part is from <I>Homo sapiens</I> chromosome 6 genomic contig, GRCh37. p13.</P><P id="yo">In our characterization, the sequence of EF-1alpha Promoter was assembled in front of a GFP reporter (<A href="http://parts.igem.org/Part:BBa_K648013">BBa_K648013</A>)and hGH polyA terminator (<A href="http://parts.igem.org/Part:BBa_K404108">BBa_K404108</A>)using Freiburg’s RFC25 format. The EF-1alpha promoter-GFP was then transfected into HEK293FT cells and in vivo green fluorescence signal was observed under fluorescence microscope. The positive control was iDUET101a plasmid (<A href="http://www.addgene.org/17629/">Addgene Plasmid Number 17629</A>) that contains EGFP reporter driven by an EF-1alpha promoter. A negative control was made by GFP generator that does not contain the EF-1alpha promoter. As a side by side comparison, a CMV promoter driven GFP reporter was also transfected, though a quantitative comparison between the two was not conducted in our characterization.
<A href="https://2013.igem.org/Team:Hong_Kong_HKUST/protocols">Detailed protocols</A> for our characterization work can be accessed via the link.</P><H3>Result</H3><P id="yo"><B>Figure 1: GFP signal of EF-1alpha observed.</B> HEK293FT cells were transfected with iDUET101a (positive control), pEF-1alpha-GFP, pCMV-GFP (alternative mammalian constitutive promoter), and GFP without promoter. Cells transfected with pEF-1alpha-GFP showed weaker green signal compared to those with iDUET101a and pCMV-GFP. This result agreed with the result reported by Qin et al., that the EF-1alpha promoter gives weaker activity than CMV promoter in HEK293T cells. Our negative control, GFP without promoter did not give any GFP signal. Scale bar = 0.1mm</P><P id="yo"><I>At the time of regional jamboree, no GFP signal could be observed in cells transfected with GFP reporter driven by EF-1alpha promoter. Originally, we thought that the sequence of EF-1alpha promoter cloned from iDUET101a contained the full functional promoter region annotated in pBudCE4.1 (Invitrogen). We believed that EF-1alpha did trigger transcription but failed to translate the GFP coding sequence due to insufficient 5’ untranslated region (UTR). After the regional jamboree, the promoter was re-cloned with additional 3' sequences after the identified TATA box to allow a longer 5’ untranslated region before the GFP coding DNA sequence. From the the results above, we believed that translation of GFP is successful this time.</I></P><H3>Conclusion</H3><P id="yo">EF-1alpha promoter was observed to drive expression of GFP in HEK293FT cells and green fluorescence was observed under fluorescence microscope.</P><DIV class="row"><DIV class="nine columns"><H3>Reference</H3><P id="yo">Qin, Jane Yuxia, Li Zhang, et al. &quot;Systematic Comparison of Constitutive Promoters and the Doxycycline-Inducible Promoter.&quot; PLoS ONE. 5.5 (2010) <HTTP:>.</HTTP:></P><P id="yo">Zhou, B. Y., Ye, Z., Chen, G., Gao, Z. P., Zhang, Y. A., &amp; Cheng, L. (2007). Inducible and reversible transgene expression in human stem cells after efficient and stable gene transfer. Stem Cells, 25(3), 779-789. doi:10.1634/stemcells.2006-0128 </P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Hong_Kong_HKUST/characterization">http://2013.igem.org/Team:Hong_Kong_HKUST/characterization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hong_Kong_HKUST/characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hong_Kong_HKUST/characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/characterization&amp;oldid=361115" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>