<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_FrenchFrog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/FrenchFrog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/FrenchFrog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/FrenchFrog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/FrenchFrog&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/FrenchFrog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/FrenchFrog</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><CENTER><H1><B><I>Xenopus tropicalis</I>: A new multicellular chassis</B></H1></CENTER><H2>Establishment of a new chassis</H2><P>So far, synthetic biology has mostly focused on bacteria, since they are simple to engineer. iGEM teams and laboratories have met synthetic biology laboratories on unicellular organisms in order to understand the underlying biology and have developed an impressive database of molecular parts. Some work has also been done on engineering mammalian cells and a few iGEM teams have followed this trend. Synthetic biologists are now imagining the rational design of multicellular organisms with numerous applications ranging from gene therapy or drug production to environmental monitoring. This year, our team would like to be part of that challenge.</P><P>The arrival of <I>Xenopus</I> as a chassis in synthetic biology requires the creation of new standards and protocols that the community will be able to build on. We provided the registry with such tools that allow rapid construction and characterization of devices in vivo, and include debugging tools. We think they will be very useful for later iGEM teams and synthetic biologists who wish to work with <I>Xenopus</I> for building multicellular systems.</P><P>A multi-tissular systems allows testing protein effect into an animal. The expression/degradation of a protein (a protein fused to GFP in example) can be followed in the organism. <I>Xenopus</I> can be used as a biosensor, Organisation for Economic Co-operation and Development (OECD) plan to validate an assay capable of <A href="http://www.oecd.org/chemicalsafety/testingofchemicals/41620749.pdf">detecting thyroid disruptor using <I>Xenopus</I></A>. With our plasmid it is easy to test in 5 days a promoter or/and a reporter in the new chassis <I>xenopus</I> because it contains a working immune/vascular/neurologic/nephrologic/digestive systems.

		</P><P>This year, the Evry iGEM team is going to be the one of the first iGEM team to work on a vertebrate. Our work is focused both on developing a system for intercellular and intertissue communication, and creating the tools for the iGEM community to easily express genes in specific tissues. We believe the tadpole is a chassis of choice for iGEM on multicellular organisms, as experiments can be conducted in one week using microinjection methods. We hope to demonstrate the feasibility of engineering <I> Xenopus </I> in one summer for an iGEM project, and to create a great tool for multicellular synthetic biology: 
<A href="https://2012.igem.org/Team:Evry/AIDSystem">An orthogonal hormonal system</A>.
</P><A name="plasmid"><H3>The simple molecular strategy to build eukaryotic plasmid ready to use: </H3></A><P><I>Xenopus tropicalis</I> represents a challenge as it is a vertebrate but also because it's a new chassis in the iGEM competition. To make sure we meet iGEM’s expectations on time, we have had to develop a new biobrick plasmid backbone compatible with<I> Xenopus tropicalis</I> (but also others vertebrate and fish). The plasmid is produced in bacteria then purified and injected into <I>Xenopus</I>'s eggs. The plasmid can not replicate in eukaryotic cells. As origins of replication are cryptic in <I>Xenopus</I>, the plasmid does not replicate. Therefore, it is only active in the first two or three weeks of development, as the injected plasmid is passed on randomly to daughter cells. After that, it becomes too diluted. 
</P><UL><LI><B>Kozac</B> : The Kozak consensus sequence is essential for the initiation of the translation process in eukaryotes. The sequence is the following (gcc)gccRccAUGG, where upper case letters are highly-conserved bases, and lower case letter can vary</LI><LI><B>Promoter </B>: The promoter of a gene is located upstream of this gene and initiate its transcription. The promoter is surrounded by the enzyme restriction sites <B>SalI</B> and <B>HindIII</B> in order to be able to switch more easily to other promoters</LI><LI><B>5'UTR and 3’UTR</B> : UTRs are Untranslated Region. The B-globin 5' UTR is located upstream of the coding sequence and is involved in 5'RNA capping. The 3'UTR is located downstream of the coding sequence and may contain sequences for the regulation of translation efficiency, mRNA stability, and polyadenylation signals</LI><LI><B>SV40 PolyA signal</B> : The simian virus 40 polyadenylation signal is involved in the maturation of the mRNA for translation and is composed of a succession of adenine bases </LI><LI><B>Antibiotic resistance genes</B> : This sequence is necessary for the selection of transformed bacteria exposed to the antibiotic</LI><LI><B>Biobrick prefix and suffix</B></LI><LI><B>Origin of replication</B> : This origin of replication is bacterial, this sequence initiates the replication of DNA</LI>
By putting all the parts necessary for expression in eukaryotes, we have made plasmids where any coding biobrick (containing Kozak sequence) can be cloned in directly without having to transfer each parts individually. These plasmids can be used to rapidly test genetic constructions in<I> Xenopus tropicalis</I> after a single cloning. It is possible to easily change promoter with the restriction sites SalI and HindIII. 
The plasmid also contains tools to calibrate the system in combination with a model. For instance, it contains sites for in vitro transcription (sp6 sites) of genes to make RNA that can then be injected directly in the embryo, allowing a much finer control of the ratio between levels of different genes during construct testing. 
To test it, inject 2.3 nL of 100 ng.uL-1 plasmid solution into the one cell embryo following the <A href="https://2012.igem.org/Team:Evry/InjectionTuto" target="_blank">injecting tutorial</A>.For a plasmid of 4kb it represents approximately 45 million of plasmids per injection. As the cell divides, plasmids are shared between cells but not replicated so a high concentration of DNA is necessary to ensure there will be DNA in most of the organism. Once the transcription machinery turns on during the development, plasmids are transcribed and translated. Since the tadpole stage starts after a few days, we can work on a whole vertebrate with most organs formed within a week. With different tissues it is possible to diversify the type of expression with different promoters.
 
So far we have made 3 new plasmid backbones with different promoters :
<LI>With a <B>CMV promoter</B> for an ubiquitous expression :
(<A href="http://partsregistry.org/Part:BBa_K812000" target="_blank">BBa_K812000</A>), 
</LI><LI>With a <B>Hsp70 promoter</B> for an inducible expression :
(<A href="http://partsregistry.org/Part:BBa_K812300" target="_blank">BBa_K812300</A>),
</LI><LI>With a <B>Elastase promoter</B> for a tissue specific expression :
(<A href="http://partsregistry.org/Part:BBa_K812200" target="_blank">BBa_K812200</A>). </LI>

Our goal was to provide tools which would allow to rapidly build and characterize constructs in the embryos (along with a synthetic hormone to make them communicate), with tools for debugging (by injecting mRNA) and with ubiquitous, inducible and tissue specific promoters.
</UL><H2>Example of GFP expression in <I>Xenopus</I></H2><P>The <A href="https://2012.igem.org/Team:Evry/InjectionTuto">injection tutorial</A> explains very simply with diagram how we did injection and how take care about your embryos and tadpole. The experiment carries on 5 days, from the  unfertilized egg to a swimming tadpole at <A href="https://2012.igem.org/Team:Evry/Stages">stage</A> 48-50. The GFP (or other fluorescent protein) is expressed  few hours after the fertilization to the end of the week (see below).

		</P><H3>Plasmid injected: pCS2+ with CMV promoter and GFP-aid reporter</H3><P>pCS2+ GFP-aid: this plasmid contains the constitutive and ubiquitous promoter CMV and the aid sequenced of the aid system fusionned to GFP (Green Fluorescent Protein)(Nishimura et al., 2009), this Biobrick created by our team is <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K812010">BBa_K812010</A>, and it was integrated into our Eucaryotic plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K812000">BBa_K812000</A>.We injected about 3.78E+7 plasmids.</P><P>
GFP-aid expression from the embryo at <A href="https://2012.igem.org/Team:Evry/Stages">stage</A> 20 (one day after injection )to a tadpole at <A href="https://2012.igem.org/Team:Evry/Stages">stage</A> ~46 (four days after injection). For this tadpole the expression is localized in the skin.</P>

We characterize the promoter CMV and elastase, not yet for the inducible promoter HSP70.
Reporters characterized: sfGFP, mCitrine, mCFP and GFP-aid.<H4><B>The characterization of all reporter and promoters is <U><A href="https://2012.igem.org/Team:Evry/Tadpole_injection1">here</A></U>.</B></H4><H2>Conclusion</H2><P>This new BioBricked plasmid is ready to use in <I>Xenopus tropicalis</I>. As you can see above the plasmid pCS2+ with the CMV pomoter express the fluorescent protein GFP-aid, but this plasmid carry on  important things to express a gene into eukaryotic calls. The 5'UTR region and the 3' UTR region was needed for  gene expression in euklaryotic organism. The simplest way to do that was to use a plasmid already used in eukaryotic cells, vertebrate and especially  <I>Xenopus tropicalis</I>  as pCS2+. This pCS2+ was Biobricked to be compatible with the registry. 

The pCS2+ plasmid was characterized and different reporters were expressed under the control of different promoters: <A href="https://2012.igem.org/Team:Evry/Tadpole_injection1">here</A>.</P><P>
Nevertheless we expected a more uniform expression of the reporters with the CMV promoter. Within tadpole, fluorescent proteins were observed in one to four different tissues, and tissues were different between tadpoles.
</P><P>
Explanation: The plasmid DNA does not diffuse in the egg and stay in the same area around the injection site. This means that depending on the injection site, the plasmid will be inherited by a given set of cells within the tadpole. This question was raised in our <A href="https://2012.igem.org/Team:Evry/plasmid_splitting">model</A>. Another reason could be that the metabolism of each differentiated cell is different and changes during the tadpole's development.
</P><P>
Our experiment showed us that plasmids injected do not diffuse in the whole organism. To express a protein with a promoter tissue specific it is a problem, because of the very low the probability to have the plasmid into the expected tissue. To solve the problem it is possible to easily integrate promoters and genes into the <I>Xenopus</I>'s  chromosome with the REMI technic [3]. A new plasmid pCS2+ with I-SceI sites upstream  the promoter and downstream of the reporter could allow the integration of the sequence between this two I-SceI restriction sites. This plasmid could be useful for the eukaryotic community, they could change promoters easily with SalI and HindIII and the reporter is compatible with the registry (with BB prefix and suffix), and they would have the choice to test a construction by plasmid injection or DNA chromosome integration.
</P><P>
Moreover, the expression of reporters decreases during time, because plasmid DNA is subjected to a catabolic activity during development but also plasmid DNA gets diluted as cells proliferate and the quantity of plasmid DNA decreases for each cell. Integration into the chromosome could prevent it. 
</P><P>
Our project raised important ethics question because the team use tadpole, an animal. We reflect that working with tadpole involved new questions about the animal pain but also about using animals in iGEM and in Synthetic Biology. It made us think about these questions, our reflection is in <A href="https://2012.igem.org/Team:Evry/HumanPractice">Human Practice</A>.</P><DIV id="citation_box"><P id="references">References:</P><OL><LI><I>Inducible control of tissue-specific transgene expression in <I>Xenopus</I> tropicalis transgenic lines.</I>, Chae J., Zimmerman L.B., Grainger R.M., Mechanisms of development 117:1-2, 2002</LI><LI><I>Xenopus: a prince among models for pronephric kidney development.</I>, Jones E., JASN 16:2, 2005</LI><LI><I>REMI (Restriction Enzyme Mediated Integration) and its Impact on the Isolation of Pathogenicity Genes in Fungi Attacking Plants</I></LI> Kahmann R., Basse C., European Journal of Plant Pathology</OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Evry/FrenchFrog">http://2012.igem.org/Team:Evry/FrenchFrog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/FrenchFrog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/FrenchFrog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/FrenchFrog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/FrenchFrog&amp;oldid=229784" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>