"
Page
Discussion
View source
History
teams
Log in
 
Team:Utah State/Experiments
From 2009.igem.org
Untitled Document
HOME
ABOUT US
PROJECT Abstract
Introduction
Broad-Host Vectors
Secretion
Experiments
Future Work
References
BIOBRICKS
MODELING
NOTEBOOK
ETHICS
JUDGING
CONTACT
Experimental Section: Approach for BioBrick Compatibility
Converting Broad-Host Vectors into a BioBrick-Compatible Format
Two Broad-host range vectors were used in this study; pRL1383a and PCPP33. To convert these vectors into BioBrick-compatible format, the four standard BioBrick sites EcoRI, XbaI, SpeI, and PstI needed to be inserted into the multiple cloning site. For pRL1383a, common BioBrick primers VR and VF2 were also included to allow the use of PCR in amplifying the BioBrick parts.
Figure 1. Plasmid map of pRL1383a
Figure 2. Plasmid map of pCPP33
Apart from being shown effective in the Synechosystis PCC 6803 (Marraccini 1993), pRL1383a is an ideal candidate for use as a BioBrick-compatible broad-host range vector because the BioBrick restriction sites are absent within the vector sequence. To convert pRL1383a into a BioBrick format, the existing multiple cloning site, which is flanked by a BamHI site and a HindIII site, was utilized. First, modified primers were synthesized from BioBrick primers VR and VF2. These primers were modified by adding extra nucleotides to insert the desired restriction enzyme sites into the PCR product. A BamHI site was added to 5’ end of the forward primer (VF2) and a HindIII site was added to the 5’ end of the reverse primer (VR).  These primers were used to amplify an existing, tested BioBrick part by PCR. For this purpose, we selected BBa_I20260 because it does not contain BamHI or HindIII sites, and successful ligation is readily testable as it is a GFP -producing construct. The addition of IPTG is typically necessary to induce GFP production in this particular device. However, when using Top10 E. coli cells it is produced continuously because these cells lack a lac repressor (insert invitrogen link). After cutting the vector at the multiple cloning site using BamHI and HindIII, the BioBrick insert obtained by PCR with modified ends was ligated into the backbone. The vector was then transformed using Top10 One Shot® chemically competent E. coli and tested for successful insertion using PCR and restriction digests.
Another broad host range vector, pCPP33, previously shown effective in Pseudomonas Putida,was standardized using similar methods. While the complete sequence of this plasmid is not available, it was shown that there are no BioBrick restriction sites outside the multiple cloning site (Figure 2). The multiple cloning site of this vector is flanked by EcoRI and HindIII. This allowed the PCR product of BBa_I20260 to again be used by cutting with HindIII and EcoRI restriction enzymes. Restriction digests and gel analysis were used to test for the insert.
Broad Host Conjugation
In order to transfer a vector of interest using conjugation, the tra gene (contained in what we will refer to as a transfer plasmid, or helper plasmid) must be expressed in order to initiate the conjugation process. This plasmid codes for genes which, when expressed, form pili on the cell surface, which in turn initiate conjugation (Heinemann 1989). This plasmid may be present in one of three different procedures:
Hfr strain – The tra operon is many times contained in an episome, which can incorporate itself into the cell genome. These resultant Hfr strains will often begin the transfer of their own DNA, both plasmid and genomic. Due to the transfer of the genomic DNA, these strains are referred to as high frequency recombinant (Hfr) strains.
Biparental (normal) Conjugation – Cells containing the tra genes, often labeled as F-positive (F+) due to the F-plasmid, a well-known transfer plasmid, can express the transfer genes necessary for conjugation to occur. When a vector of interest and a transfer plasmid are of different incompatibility groups, they may both be transformed into the same cell, and conjugation may take place between the F+ donor cell and the recipient cell
Triparental Mating – In the case where the transfer plasmid and the vector of interest are of the same incompatibility group, the two plasmids may not stably coexist (Heinemann 1989). In this case, two separate cells containing the transfer gene (the helper cell) and the vector (the donor cell) must be used in conjugation. The helper cell will assist the donor cell in the transfer of its mobilizable plasmid to the recipient cell. This method circumvents some of the barriers that may prevent the transfer of plasmids.
For our project, we chose to use the triparental mating procedure for the transmission of our vector. While not being the most efficient method, it circumvents possible barriers and intermediate steps.
Because of the use of three different cells in our transformation procedure, the selection criteria for each component needed to be unique. In addition, we selected helper plasmids which had been known to work with the intended recipient cell.
Table 1 Components and selection criteria used in conjugation with the broad-host vector PCPP33
Table 2 Components and selection criteria used in conjugation with the broad-host vector PRL1383A
Results
Testing the ligation of pRL1383a and BBa_I20260 using PCR and restriction digests showed that the insert was not present in the vector, and the conversion to BioBrick format ultimately unsuccessful. The procedure as described above was repeated multiple times without success. Tri-parental conjugation of unmodified pRL 1383a was inconclusive in all target organisms.
In an effort to troubleshoot this vector, several different approaches were taken. First, the ligation was repeated with varying concentrations of insert (10X, 2X) in an attempt to account for the impact of the large vector size on the ligation reaction. These ligations yielded similar results to reactions done at calculated concentrations. A Blunt-end ligation using a Klenow fragment was also performed. This was repeated, both attempts without success. The BBa_I20260 PCR product with BamHI/HindIII ends was ligated into another vector in an attempt to test the insert’s ability to be cut with the restriction enzymes. This ligation did not indicate the presence of the insert, suggesting that the problem lies with the vector or primers. The primers were tested and found viable on another insert, with similar testing of restriction enzymes to show functionality. The primers and enzymes were operating as intended, but new enzymes were ordered for more experimental certainty. The insert was then digested only with HindIII, and left in a ligation reaction. The outcome of this ligation was not of the desired length. This was repeated, and the same result obtained. While there is some suggestion that the BioBrick insert may not be functioning, the ambiguous results of tri-parental mating with unmodified pRL1383a suggests that the vector may be damaged or misunderstood.
Testing the ligation of PCPP33 and BBa_I20260 also proved unsuccessful. Restriction digests using BioBrick standard pieces failed to yield an insert. Tri-parental mating of this vector proved successful in all organisms that we tested. All organisms yielded colonies on tetracycline plates, suggesting presence of the plasmid. Further testing by plasmid extraction and gel analysis will be done to conclusively determine presence of the plasmid.
Figure 3. Results of the tri-parental mating between pCPP33 and R. sphaeroides, P. putida, and Synechocystis sp., respectively. Each plate is shown alongside a negative control
Go to Top of Page
Experiments: Secretion
Methods for Constructing BioBrick Parts
One of the objectives of this project was to create a library of (link to silver fusion wiki) Silver-fusion compatible BioBrick signal peptides and protein-coding parts for secretion studies. The Silver-fusion assembly method was used because the standard BioBrick prefix and suffix do not facilitate fusion of two parts. The scar that forms from the overlap of compatible restriction enzyme sites XbaI and SpeI is not conducive to fusion because it contains a stop codon and is 8 nucleotides long. Because the scar is not a multiple of three, the sequence after the scar will be read out-of-frame. The Silver-fusion assembly method retains compatibility with the standard BioBrick assembly method, but fusion is allowed. A single nucleotide is removed from the prefix and suffix of Silver-fusion BioBricks so that the scar that forms from the ligation of XbaI and SpeI sites does not contain a stop codon and is 6 nucleotides in length.
Five signal sequences were selected for this study based on the secretion pathway that they represent and their prominence in literature. The selected sequences are presented in Table X. Two protein coding regions were obtained: phasin and GFP. All of these sequences were designed for Silver-fusion compatibility. Four different promoters with an attached ribosome binding site were designed and then synthesized by DNA 2.0, followed by ligation into a BioBrick vector. Composite devices were assembled piecewise by cutting one part typically with EcoRI and XbaI, and the part to be inserted with EcoRI and SpeI. Analysis by PCR with the Primers VF2 and VR was used to qualitatively determine whether successful ligation had taken place. Once partially confirmed, samples were sequence at the Utah State University Center of Integrated Biotechnology.
Signal Peptides:
To construct the OmpA, PelB, and GeneIII sequences, complimentary forward and reverse oligonucleotides were synthesized by Eurofins Operon. These strands were then annealed together. The oligonucleotides were designed so that the silver fusion prefix and suffix sequences were appended onto the end of each sequence. These parts were then cut with EcoRI and SpeI and ligated into a BioBrick vector. Each of these parts were successfully constructed and sequenced.
The TorA and HlyA signal peptides were synthesized by DNA 2.0 because these sequences are longer than the other signal peptides, which made the complimentary oligonucleotides method not ideal. The Silver-fusion prefix and suffix was added to each of these constructs. EcoRI and SpeI were used to cut the part out of the commercial vector. The DNA was isolated by gel electrophoresis and ligated into a BioBrick compatible vector, pSB3K3.
Phasin:
The phasin (PhaP) sequence was isolated from the genomic DNA of Cupriavidus necator (also known as Ralstonia eutropha). There are four different phasin genes in the genomic DNA of this organism. This particular phasin was selected based on references in literature, although no information was acquired that indicated that one phasin gene would yield better production over another. The primers were designed so that the Silver-fusion prefix and suffix were overhanging, thereby resulting in a final product that is Silver-fusion compatible. The 579 bp phasin sequence was found to contain a PstI site. The PstI site was mutated using site-directed mutagenesis. Sequencing confirmed that this site was successfully removed.
GFP:
Near the beginning of this project, a Silver-fusion compatible GFP BioBrick (BBa_K125500) derived from BBa_E0040 by the Hawaii 2008 iGEM team was obtained. However, upon further analysis it was determined that this part was modified so that the start codon of the sequence was removed. Although this should not affect the expression of GFP in composite parts with a signal peptide prior to the sequence, it is not ideal for this particular project. The lack of a start codon requires N-terminal fusion of another protein or signal peptide, and a functional GFP control without a signal sequence would not be functional. This control is important in our study to compare with composite parts containing signal peptide-protein fusion to determine whether the produced GFP is being transported.  Additionally, this part would not work with C-terminal signal peptide fusions. The HlyA signal peptide is recognized on the C-terminus of the target protein by the Type I secretion pathway (Mergulhão, 2005; Gentschev, 2003; Hess, 1990). The absence of the start codon inhibits study of this secretory pathway. Another disadvantage of this GFP part is its small Stokes shift (excitation 501 nm, emission 511 nm). An ideal GFP that fluorescence would have a shorter excitation wavelength so that GFP-positive samples can be detected visually using a UV transilluminator.
A new Silver-Fusion compatible GFP BioBrick part was constructed for this project via a similar mechanism as the phasin construct. This particular GFP was previously mutated for improved fluorescence photostability (Crameri, 1996). The excitation and emission wavelengths for this GFP are 395 nm and 501 nm, respectively. That being said, GFP-positive cells emit a bright green fluorescence when exposed to shorter-wavelength UV light, such as on a transilluminator. Primers were synthesized for isolation of the sequence and, like the phasin-specific primers, designed so that the Silver-fusion prefix and suffix were inserted on the ends of the sequence (see primers). Figure 4 shows GFP- Top10 E. coli colonies (left) and unfused GFP+ Top10 E. coli colonies (right). This figure shows that the GFP construct is functional and easily detectable.
Figure 4. Plate with GFP- cells (left) next to plate with GFP+ cells(right)
Bioplastic Production:
A plasmid harboring the genes for PHB production (pBHR68) was used in these experiments. This plasmid contains the sequence for ampicillin resistance and contains a ColE1 origin of replication. E. coli harboring pBHR68 were cultured according to methods outlined by Kang et al (2008) and production of PHB was verified using 1H NMR analysis. The spectrum obtained from this experiment is given as Figure 5. The observed peaks at 1.24 ppm, 2.54 ppm, and 5.2 ppm correspond with those observed in standard polyhydroxyalkanaote samples.
Figure 5. Proton NMR spectra for PHB production in recombinant E. coli
To maintain plasmid compatibility in E. coli transformed with both the pBHR68 and phasin plasmids, it was determined that the vector used for the phasin secretion device required a p15A ori site. BioBrick vector pSB3K3 was found suitable as the host for the secretion constructs. XL1-Blue E. coli were transformed with both a phasin device and the pBHR68 BioBrick plasmids, and these cells were cultured and tested for secretion.
SDS-PAGE Analysis
Sodium dodecyl sulfate polyacrylamide gel electrophoresis was used to analyze the protein content in transformed E. coli. As a positive control, E. coli containing the Lac/RBS/GFP/Terminator (BBa_K208045) construct were sonicated and centrifuged (see Figure 6). Additionally, E. coli cells containing an individual BioBrick part (BBa_B0015) were analyzed as a negative control. The resulting gel was stained with coomassie blue and is shown as Figure 6. The bright band at 27 kD in the GFP+ sample corresponds to the GFP protein (Bio-Rad). The absence of this band in the GFP- sample further reinforces the functionality of the GFP construct.
Figure 6. Protein gel showing a strong band corresponding to GFP
The geneIII secretion signal sequence fused to the phasin protein was expressed in E. coli cells. The E. coli cells were grown overnight in LB growth media and centrifuged to pellet the cells.  Supernatants (5ml) were then concentrated using a Centricon Centriplus concentrator (Amicon, Beverly MA).  This process concentrated proteins that were larger than 10kDa and removed molecules smaller than 10kDa.  Approximately 20ug of protein were then applied to a SDS polyacrylamide gel to separate the proteins according to size.  The gel was then stained with coomassie blue for protein detection, as shown in Figure 7. Following SDS polyacylamide gel electrophoresis (PAGE) and subsequent coomassie blue staining of the separated proteins, a protein with an approximate size of 22kDA is observed in the sample from the phasin-expressing E. coli cells that is not present in the control E. coli sample.  The phasin protein has been reported by others to migrate on SDS PAGE from 14-28kDa (Pötter, 2002; York, 2002). These results indicate that the GeneIII::phasin expression construct is being produced by the E. coli cells and is being secreted outside the cell into the media.
Figure 7. Protein gel showing the presence of phasin protein in supernatant samples (third well from left)
next to supernatant from an E. coli sample without a phasin-producing construct.
Western blotting with phasin-specific antibodies was performed to verify the observed band as phasin.  Phasin antibody was kindly provided by Anthony J. Sinskey at Massachusetts Institute of Technology. The results of the western blotting were inconclusive. Non-specific binding to larger constructs was observed. Additional testing is required to further reinforce preliminary findings and confirm the secretion of phasin. The secretion of phasin would provide evidence that PHA recovery via phasin secretion is possible. Addtionally, this would reinforce that the constructed BioBricks are not only functional, but would be beneficial for use in other studies.
Go to Top of Page
Retrieved from " http://2009.igem.org/Team:Utah_State/Experiments "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
