<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Introduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Utah_State/Introduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Introduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Introduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Introduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Introduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Introduction</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P class="class"><TITLE>Untitled Document</TITLE><TABLE width="100%" border="0"><TBODY><TR valign="top"><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State"><FONT size="4">HOME</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Team"><FONT size="4">ABOUT US</FONT></A></TD></TR><TR><TD width="172" id="ana"><SPAN class="currentPage"><FONT size="4">PROJECT</FONT></SPAN><A href="https://2009.igem.org/Team:Utah_State/Abstract">Abstract</A><A href="https://2009.igem.org/Team:Utah_State/Introduction">Introduction</A><A href="https://2009.igem.org/Team:Utah_State/Broad-HostVectors">Broad-Host Vectors</A><A href="https://2009.igem.org/Team:Utah_State/Secretion">Secretion</A><A href="https://2009.igem.org/Team:Utah_State/Experiments">Experiments</A><A href="https://2009.igem.org/Team:Utah_State/FutureWork">Future Work</A><A href="https://2009.igem.org/Team:Utah_State/References">References</A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Parts"><FONT size="4">BIOBRICKS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Modeling"><FONT size="4">MODELING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Notebook"><FONT size="4">NOTEBOOK</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/ETHICS"><FONT size="4">ETHICS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Achievements"><FONT size="4">JUDGING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Contact"><FONT size="4">CONTACT</FONT></A></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><B><I><FONT size="5" face="Century Gothic, Arial, San Serif" color="#000033">
Introduction: Why Break BioBrick Borders?
    </FONT></I></B></TD></TR></TBODY></TABLE></P><P class="class">
      Since the beginning of iGEM, BioBricks have chiefly been designed for use in <I>E.coli</I>.  This has primarily been due to the efficient growth rate of <I>E.coli</I> and its relatively thorough characterization.  However, the employment of the BioBrick system in host organisms other than <I>E. coli</I> would greatly enhance and expand the field of synthetic biology.  In order to investigate the BioBrick system in other organisms, it is imperative that a reliable broad-host-range vector be developed.  The 2009 Utah State iGEM team is building on the 2008 University of Hawaii team’s efforts to develop a broad-host-range BioBrick vector that would make possible the use of BioBrick parts, devices, and systems in organisms other than <I>E. coli</I>.  The organisms under investigation are <I>Pseudomonas putida</I> KT2440, <I>Synechocystis</I> PCC 6803, and <I>Rhodobacter sphaeroides</I>. Additionally, our project seeks to break borders in another way: through the construction of Silver-fusion compatible BioBrick parts for  secretion-based recovery of recombinant proteins and other compounds, like polyhydroxyalkanoates. </P><P class="class">
      One of the BioBrick borders we seek to break is that of <I>Pseudomonas putida</I>.  This bacterium would open the BioBrick doors to soil applications.  <I>Pseudomonas putida</I> is a non-pathogenic, gram-negative soil bacterium with optimal growth at room temperature.  The diversity of its metabolic pathways allows it to be used for bioremediation purposes; it can degrade many polluting aromatic hydrocarbons including toluene, benzene, xylene, naphthalene, and styrene. This organism can also act as a biocontrol agent (Lemanceau, 1992; Haas and Defago, 2005), suppressing the growth of fungi.  <I>P. putida</I> performs these functions while colonizing the rhizosphere of plant roots, enhancing the growth of the plant through these and other means (Albert and Anderson, 1987; Bakker et al., 1986).  The genome of <I>P. putida</I> KT2440 has been sequenced, allowing more extensive genetic analyses and contributing to this strain being the “preferred host for cloning and gene expression for Gram-negative soil bacteria” (Nelson et al., 2002).  Potential applications include BioBrick devices for enhancing the catabolism of environmental pollutants, the implementation of BioBrick devices used to protect plants (and its subsequent consumers) against pathogens not previously defended against, and the use of BioBrick devices to increase crop yields. </P><P class="class">
Another BioBrick border we'd like to break is that of cyanobacteria.  We have specifically been working with <I>Synechocystis</I> PCC 6803.  This bacterium would allow BioBricks to be used in photosynthetic applications.  <I>Synechocystis</I> PCC 6803 is a Gram-negative bacterium that can produce energy either through photosynthesis or respiration (Tabei et al., 2007). It also displays a circadian rhythm in several of its cellular functions (Kucho et al., 2005) and can take up foreign DNA (Williams 1988).  It can also grow in a variety of temperatures (Gombos et al., 1992).  Cyanobacteria in general play an important role in nitrogen fixation for crops and are a major player in rice cultivation (Irisarri et al., 2001).  Potential applications include the use of BioBrick devices in bioenergy, wastewater treatment, crop yields, and biomanufacturing processes that take advantage of the fact that a carbon source is not needed.</P><P class="class"> A third border that we aim to break is that of <I>Rhodobacter sphaeroides</I>, an organism usually found in the anaerobic mud of ponds and lakes where there is access to sunlight.  This is a very metabolically diverse organism that has potential for providing a myriad of BioBrick opportunities.  <I>Rhodobacter sphaeroides</I> can grow under a variety of conditions: aerobic or anaerobic respiration, photosynthesis, and fermentation; it has optimal growth in microaerophilic surroundings.  It can also fix dinitrogen as its sole nitrogen source (Mackenzie et al., 2007).  Similar to E. coli, this organism moves with a single flagellum.  <I>R. sphaeroides</I> has more membrane surface per cell than other organisms used to express membrane proteins, making it an ideal host for overexpressing and studying such proteins.  It is capable of making biofuels through the process of lithotrophy (Roy et al., 2008) and other pathways (Yokoi et al., 2002).  <I>R. sphaeroides</I> is also capable of tolerating and reducing at least 11 rare earth metal oxides and oxyanions, making it an excellent candidate for bioremediation and detoxification purposes (O’Gara et al., 1997).  Many of the above-listed characteristics place <I>R. sphaeroides</I> in the spotlight for use in biomanufacturing.  Possible BioBrick applications with <I>R. sphaeroides</I> include membrane protein studies (including secretion and protein overexpression studies), biomanufacturing, bioenergy, and bioremediation/detoxification.  </P><P class="class">To <I>even further</I> break down BioBrick borders, composite devices were constructed to investigate phasin and green fluorescent protein secretion. Secretion of phasin was studied to show that these PHA-associated proteins are targetable for export out of the cytoplasm, and that optimization of phasin expression and binding may facilitate bioplastic secretion.  Constructs for GFP translocation were made in parallel with the phasin secretion devices. These GFP constructs provide a visually or spectrofluorometrically detectable control due to a high level of fluorescent protein accumulation. Successful GFP translocation would reinforce the potential of phasin export, which is not as readily monitored. Beyond the scope of this project, the constructed signal peptides and GFP BioBricks can readily be used by other researchers for recombinant protein secretion studies.</P><B><I><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20">
Project Objectives
    </FONT></I></B><P class="class">The overall goal of this project is to demonstrate the concept of “BioBricks without Borders” by expanding the use of broad-host vectors for expression of BioBricks in multiple organisms and by demonstrating secretion  for simplified recovery of recombinant proteins using BioBrick constructs. More specific goals of this project are to:</P><UL class="circle"><LI>Determine how broad-host range vectors can be modified to comply with the BioBrick assembly standard.</LI><LI>Use broad-host range vectors to transform <I>Synechocystis</I> PCC6803, <I>R. sphaeroides</I>, and <I>P. putida</I> by triparental mating. </LI><LI>Create a BioBrick genetic library of Silver fusion-compatible signal peptides and coding regions for secretion studies.</LI><LI>Test the functionality of BioBrick devices and determine methods for detecting phasin and/or PHA secretion.</LI></UL><P class="class">The following sections provide more extensive details about these goals, experimentation and testing, and the results and conclusions from this project.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Utah_State/Introduction">http://2009.igem.org/Team:Utah_State/Introduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Introduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Introduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Introduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Introduction&amp;oldid=168114" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>