<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Utah_State/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><A name="top"><TITLE>Untitled Document</TITLE></A><TABLE width="100%" border="0"><TBODY><TR valign="top"><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State"><FONT size="4">HOME</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Team"><FONT size="4">ABOUT US</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Project"><FONT size="4">PROJECT</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Parts"><FONT size="4">BIOBRICKS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Modeling"><FONT size="4">MODELING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Notebook"><FONT size="4">NOTEBOOK</FONT></A></TD></TR><TR><TD width="172" id="ana"><SPAN class="currentPage"><FONT size="4">PROTOCOLS</FONT></SPAN><A href="#CTAB">Isolating DNA Plasmids</A><A href="#transformation">Bacterial Transformation</A><A href="#streakplates">Streak Plates &amp; Cultures</A><A href="#CTAB2">Preparation for DNA Separation</A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/ETHICS"><FONT size="4">ETHICS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Achievements"><FONT size="4">SUCCESS</FONT></A></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><A name="CTAB"><FONT size="4" face="Century Gothic, Arial, San Serif" color="#000033"><B><I>Isolating DNA Plasmids with CTAB</I></B></FONT></A></TD></TR><TR><TD><FONT size="2.5" face=" Helvetica, Tahoma, Arial" color="#231f20"><B>
This protocol needs a description
	</B><OL><LI>Prepare two water baths, one boiling and the other 68C.  </LI><LI>Centrifuge the 12 ml tubes containing the 5 ml cultures in the large centrifuge at 3K RPM for 20 min.  Discard supernatant.</LI><LI>Resuspend cells in 200 l of STET buffer. Transfer to 1.5 ml tubes.</LI><LI>Add 10 l (if older preparation) Lysozyme (50 mg/ml) and incubate at room temperature for 5 min.</LI><LI>Boil for 45 sec and centrifuge for 20 min at 13K RPM (or until pellet gets tight).</LI><LI>Use a pipette tip to remove the pellet.</LI><LI>Add 5 l RNase A (10 mg/ml) and incubate at 68C for 10 minutes.</LI><LI>Add 10 l of 5% CTAB and incubate at room temperature for 3 min.</LI><LI>Centrifuge for 5 min at 13K RPM, discard supernatant, and resuspend in 300 l of 1.2 M NaCl by vortexing.</LI><LI>Add 750 l of ethanol and centrifuge for 5 min at 13K RPM.</LI><LI>Discard supernatant, rinse pellet (which cannot be seen) in 80% ethanol, and let tubes dry upside down with caps open.</LI></OL></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><FONT size="3" face="Century Gothic, Arial, San Serif" color="#000033"><DIV align="center"><B><A href="#top">Go to Top of Page</A></B></DIV></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><A name="transformation"><FONT size="4" face="Century Gothic, Arial, San Serif" color="#000033"><B><I>Bacterial Transformation</I></B></FONT></A></TD></TR><TR><TD><FONT size="2.5" face=" Helvetica, Tahoma, Arial" color="#231f20"><B>
Once the target DNA has been successfully ligated into the plasmid vector, the plasmid must be transferred into the host cell for replication and cloning. In order to do this, the bacterial cells must first be made competent. The term competent is to describe a cell state in which there exist gaps or openings in the cell wall which will allow the plasmid containing the target genes to enter into the cell. Several methods to make bacterial cells competent exist, such as the calcium chloride method and electroporation. Competent cells may also be purchased commercially. The team at USU has purchased competent cells for all experiments. The following is the method used by the USU team to insert the plasmids containing the PHB biobricks into the cells. 	</B><OL><LI>Ensure the necessary antibiotic agar plates have been prepared or begin their preparation now.  Four plates per transformation will be necessary (two today, then two tomorrow for streaking).  Also ensure that 10 ml liquid media is made up per transformation (also for tomorrow).</LI><LI>Clean punchout tool by dipping in 10% chlorox, deionized water, deionized water again, then 80%aq ethanol.  Let dry for 2 minutes and repeat cleaning procedure between punchouts. </LI><LI>Punch out gene of choice with a twisting motion, allowing the metal to cut the paper.  Use the center part of the punchout tool to dislodge the paper into a 2.5 ml microcentrifuge tube. </LI><LI>Add 5 l TE buffer, place in a 42C water bath, and allow plasmids in the paper to elute for 20 minutes.</LI><LI>Centrifuge tube(s) at 15K RPM for 3 minutes.  Remove SOC media from the -20C freezer and leave out to thaw.</LI><LI>Take competent cells from the -80C freezer and place on an ice bath.</LI><LI>Pipette contents of tube up and down a few times then take 2 l of the DNA solution and add to the competent cells.  Ensure the pipetting is done directly into the cell solution.  Let cells incubate in the ice bath for 30 minutes.  Heat water bath to 42C.</LI><LI>Heat shock cells in the 42C water bath for 30 seconds.  Remove and place back in the ice bath for 2 minutes.</LI><LI>In the hood, add 250 l SOC media to each tube, bringing the total cell solution to 300 l.  Incubate at 37C for 1 hour.</LI><LI>Get out the antibiotic agar plates.  In the hood, add 200 l of each transformed cell solution to the appropriate antibiotic plate.  Use the Bunsen burner to create a hockey stick out of a glass pipette tip by holding over the flame until it bends.  Allow to cool.  Spread cell solution uniformly over the agar plate using the hockey stick, then before discarding, spread residual solution on the stick over a second plate to get more a more sparse colony distribution. </LI><LI>Parafilm all plates and place in 37C incubator 12-14 hours, or overnight if that is not possible.</LI></OL></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><FONT size="3" face="Century Gothic, Arial, San Serif" color="#000033"><DIV align="center"><B><A href="#top">Go to Top of Page</A></B></DIV></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><A name="streakplates"><FONT size="4" face="Century Gothic, Arial, San Serif" color="#000033"><B><I>Streak Plates and Liquid Cultures from Transformed Colonies</I></B></FONT></A></TD></TR><TR><TD><FONT size="2.5" face=" Helvetica, Tahoma, Arial" color="#231f20"><B>After bacterial cells have been transformed, successfully transformed cells must be selected. Because 100% of the cells do not receive the desired plasmid and target gene, it is essential to select for cells that do have the target genes. Several methods are used to accomplish this, such as incorporation of antibiotic resistance and also the lac operon. The USU team has used antibiotic resistance to select for successful transformations. To do this, an antibiotic resistance gene is also added to the plasmid vector that contains the target genes. By doing so, it is possible to know that a cell was successfully transformed based on its ability to grow on an agar plate with antibiotics added. Because the cell is able to grow, the antibiotic resistance gene must be present as well as the target gene. From the agar plates containing the antibiotics, a colony is picked off and transferred into a liquid culture for further analysis and cellular production. The following is the method used by USU to clone the DNA and select for the successful transformation of the PHB biobrick into the cells.</B><OL><LI>Prepare two 12 ml tubes per transformation, each with 5 ml media containing the appropriate antibiotic (if felt necessary).  Get out antibiotic agar plates.  Inspect plates from yesterday for colonies.  At least two colonies are preferred, but one will do.  Select two colonies and label them.
</LI><LI>Use a pipette tip to extract half of each colony and inoculate one agar plate per colony.  Using a pipette with a tip, extract the other half of each colony and inoculate one liquid media tube per colony.  Label all tubes and plates and place in the 37C incubator until tomorrow morning.
</LI></OL></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><FONT size="3" face="Century Gothic, Arial, San Serif" color="#000033"><DIV align="center"><B><A href="#top">Go to Top of Page</A></B></DIV></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><A name="CTAB2"><FONT size="4" face="Century Gothic, Arial, San Serif" color="#000033"><B><I>Preparation for DNA Separation</I></B></FONT></A></TD></TR><TR><TD><FONT size="2.5" face=" Helvetica, Tahoma, Arial" color="#231f20"><B>Following successful bacterial cloning and isolation, it is important to verify that the target gene is in the cell and that the plasmid is functional. To do this, it is a common practice to sequence the DNA plasmid. To obtain enough DNA for sequencing, the bacterial clones are grown in a liquid culture. The cells are harvested by centrifugation and then prepared for DNA plasmid extraction. DNA plasmid extraction can be done several ways, and the overall purpose is to lyse the cells and separate the plasmid DNA from all other cellular proteins, DNA, and debris. The following is the method used by the USU team to isolate the plasmid DNA containing the PHB biobricks.</B><OL><LI>Prepare two water baths, one boiling and the other 68C.     
</LI><LI>Centrifuge the 12 ml tubes containing the 5 ml cultures in the large centrifuge at 8K RPM for 20 min.  Discard supernatant.   
</LI><LI>Resuspend cells in 200 l of STET buffer.   
</LI><LI>Add 10 l (if older preparation) Lysozyme (50 mg/ml) and incubate at room temperature for 5 min.    
</LI><LI>Boil for 45 sec and centrifuge for 20 min at 13K RPM (or until pellet gets tight).  
</LI><LI>Use a pipette tip to remove the pellet.  
</LI><LI>Add 5 l RNase A (10 mg/ml) and incubate at 68C for 10 minutes.  
</LI><LI>Add 10 l of 5% CTAB and incubate at room temperature for 3 min.  
</LI><LI>Centrifuge for 5 min at 13K RPM, discard supernatant, and resuspend in 300 l of 1.2 M NaCl using a vortex mixer.  
</LI><LI>Add 750 l of ethanol and centrifuge for 5 min at 13K RPM.  
</LI><LI>Discard supernatant, rinse pellet (cannot see) in 80% ethanol, and let tubes dry upside down with caps open. 
</LI></OL></FONT></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><FONT size="3" face="Century Gothic, Arial, San Serif" color="#000033"><DIV align="center"><B><A href="#top">Go to Top of Page</A></B></DIV></FONT></TD></TR></TBODY></TABLE>







=== Restriction Enzyme Digestion and Electrophoresis ===

Restriction enzyme digestion is the process by which an insert DNA sequence is separated from the rest of the DNA molecule. Specific knowledge of the DNA insert is needed to determine which enzyme and conditions to use during the digestion reaction. Once the DNA sequence is known and the correct enzymes have been selected, the DNA may be digested. Listed below is the procedure used by USU to digest the cellular DNA. After enzyme digestion, electrophoresis is used to separate the plasmid from the insert. A gel is prepared and the respective reaction mixes are loaded into the gel. Using a DNA ladder, and knowing the size of the insert, the corresponding band can be seen and cut out of the gel. The insert may then be removed and isolated from the gel, thus yielding the desired DNA. The DNA from this may then be used in PCR reactions, sequencing, ligations for further experimentation, and many more. Listed below are the protocols used by the USU team to run the electrophoresis reaction. 

'''Method''' 
1. Resuspend DNA in 40 l water, vortex, and do a brief centrifuge to get solution to the bottom of the tube.  
2. Add components to the digestion solution in the following order:  DNA (23 l), 10X Tango buffer (3 l), Xba1 (2 l), and Pst1 (2 l).  The volume and restriction enzymes can be varied, but it should be ensured that the total volume is 10X the amount of Tango buffer added.  Tap tubes periodically and allow to digest while preparing electrophoresis gel.  
3. Prepare electrophoresis gel by adding 2 g agarose to 200 ml TAE (1% solution).  This is best done in an Erlenmeyer flask of adequate volume as swirling will need to be done.  Place in the microwave and microwave on high for 20 seconds at a time, pulling it out and swirling until solution is homogeneous again, then repeating (BE CAREFUL to watch the solution closely when swirling  it superheats and can boil over and cause severe burns).  Continue until solution is seen boiling in the microwave then gently swirl again.  
4. Add 20 l ethidium bromide to solution and swirl until dissolved evenly.  
5. Add 6 l of 6X loading dye to each tube of digested DNA solution.  
6. Prepare the electrophoresis unit by orienting the basin sideways with rubber gaskets firmly against the side.  Place desired well template in the basin.    
7. When the agarose solution is cool enough to comfortably touch the flask, pour into the basin until the solution is about  of the way to the top of the well template.  
8. When the gel is solidified (should look somewhat cloudy), remove the well template and change basin orientation to have the wells closest to the negative pole (as the DNA will flow towards the positive pole).  Pour 1X TAE buffer into both sides of the electrophoresis unit until it just covers the gel and fills the wells. 
9. By inserting the pipette tip below the TAE liquid and into the well, add 10 l of DNA ladder solution to first (and last if desired) well, skip one well, then begin adding the digested DNA solutions to the wells by adding about 2 l less than the total volume in the tubes to prevent air bubbles in the wells. 
10. Place the cover on the electrophoresis unit, plug into the power source, and turn on voltage to 70 V (this can be as high as 100 V if time is an issue), and press the start button.  Separation should take two to three hours.  The yellow dye shows the location of the smaller nucleotide lengths and the blue dye shows the location of the larger nucleotide lengths.  DNA separation can be observed as time goes on by turning off the power supply then gently removing the basin from the electrophoresis unit (be careful not to let the gel slip out of the basin) and placing on the UV transilluminator to see DNA bands.  The basin can then be placed back in the electrophoresis unit for further separation if desired.  Take care to not have the power supply on without the lid to the unit in place.   
11. When the desired level of separation is attained, the basin can be placed on the transilluminator for picture taking.  Place the cone-shaped cover over the transilluminator and place the digital camera in the top hole for pictures. 


=== Media Preparation ===

For all experimentation involving the need for bacterial biomass and experimentation, proper media is needed to grow the cells. The following is the mediFollowing successful bacterial cloning and isolation, it is important to verify that the target gene is in the cell and that the plasmid is functional. To do this, it is a common practice to sequence the DNA plasmid. To obtain enough DNA for sequencing, the bacterial clones are grown in a liquid culture. The cells are harvested by centrifugation and then prepared for DNA plasmid extraction. DNA plasmid extraction can be done several ways, and the overall purpose is to lyse the cells and separate the plasmid DNA from all other cellular proteins, DNA, and debris. The following is the method used by the USU team to isolate the plasmid DNA containing the PHB biobricks.a composition and methods used by USU to prepare the media.

1. Add 5 g yeast extract, 10 g NaCl, 10 g Bacto Tryptone, and 15 g agar (if desired) to a 2 L Erlenmeyer flask and bring the volume up to 1 L with ddH20.  Mix by swirling.  Cover top with foil. 
2. Autoclave for 45 minutes (liquid setting, 0 minutes drying time).  It will take an additional half hour for the autoclave to finish cooling then an additional 20 to 30 minutes until the media is cool enough to pour. 

=== 1X TAE Preparation ===
1. Add 40 ml 50X TAE solution to a 2 L flask and bring level up to 2 L with ddH20.   

=== Polymerase Chain Reaction (PCR) ===

PCR is used to amplify a desired DNA sequence. The reaction is first set up by designing primers that will bind only to the desired regions of the DNA sequence. Once the primer and polymerase have been selected, the reaction parameters of time and temperature must be optimized. When the reaction works properly only the target DNA will be amplified into large quantities that may then be isolated and used for further experimentation. The following is the procedure used by USU for PCR reactions to amplify the PHB synthesis and promotion genes.  
'''Method''' 
1. Obtain the following reagents from the freezer:  DNA template (cells or DNA), 10X Taq buffer (+KCl, -Mg/Cl<SMALL>2</SMALL>), MgCl<SMALL>2</SMALL>, 10 mM dNTP Mix, Taq polymerase (take out of freezer only immediately when needed and put back), and sterile distilled H<SMALL>2</SMALL>O.  Place all reagents on ice.  Also obtain PCR (either 0.2 or 0.5 ml) tubes. 
2.	Add the following reagents to a tube (50 l reaction) in the following volumes and order: 
	32 l sterile H<SMALL>2</SMALL>O, 
	5 l 10X buffer,  
	2 l dNTP Mix,  
	6 l MgCl<SMALL>2</SMALL> 
	3 l cells/DNA,  
	0.25 l Taq Polymerase 
	1 l primer 1 
	1 l primer 2 
MgCl<SMALL>2</SMALL> volume can be varied (lower to increase specificity  just ensure total volume is 50 ul with H<SMALL>2</SMALL>O).  If many reactions are to be constructed, a master mix can be made up to cut down on time and pipette tip usage (if this is done, ensure primers are added to the appropriate reaction, i.e. perhaps not to the master mix).  Tap or vortex tubes and take to the thermocyler.  Place all reagents back in the -20C freezer.   
3. Choose thermocycler temperatures.  The Eppendorf Mastercycler will cycle between three temperatures:  typical temperatures are 94C for denaturing, 50-60C for primer annealing, and 72C for polymerase extending.  Lowering the annealing temperature decreases DNA specificity; 55C is a good temperature to begin if no trials have been made with the sample.   
4. Turn on thermocycler with the switch in the back of the unit and open the lid.  The placement of the tubes depends on the size of the tube (0.2 or 0.5 ml) and whether or not a temperature gradient is to be used.   

* If no gradient will be used, tubes can be placed anywhere on the unit in the appropriately-sized hole. Select Files and press enter.  Select Load and then Standard.  If cells will be used in the reaction, include a 1-minute lysing step at the beginning (step 1); this will be followed by a 1-minute DNA denaturing step (step 2).  If purified DNA will be used, set step 1 to 1 second.   Set an annealing temperature for step 3.  Ensure the lid temperature is 105C and the extending temperature is 72C.  Press exit.  If prompted to save, save by pressing enter three times.  Press exit to return to the main menu.  Choose Start on the main menu and select Standard.  The program should begin.   

* If a gradient is to be used, temperature will vary according to column.  A 20C range is the maximum range that can be used (+/- 10C).  The range is made by setting a temperature for the middle column and then setting a +/- range.  To see what the temperatures will be if a gradient is used, select OPTIONS on the main menu, then select Gradient.  Select the size tube that is being used by pressing Sel, then press enter.  Choose a temperature for the center column, press enter, then select a +/- range and press enter.  The column number along with the corresponding temperature is shown.  Decide tube placement based on this information.  Press exit twice to return on the main menu.  Select Files then Load, then Gradient.  If cells are being used, set the cell lysing step (step 1) to 1 minute (1:00); if purified DNA is being used, set this time to 1 second (0:01).  Step 2 should be 94C, Step 4 should be 72C, and the lid temperature should be 105C.  Go to step 3 and set an annealing temperature for the center column.  Leave the next two lines as they are, and change the gradient setting (G) to the +/- the center temperature amount.  Press exit.  If prompted to save, press enter three times; if not prompted to save, press enter once.  Press exit to get back to the main menu.  To begin cycle, select Start, then select Gradient.  The program should begin.   

5. The thermocycler is set to store the completed reaction tubes at 4C when finished.  

=== Ligation ===

Ligation is the process by which the insert (target DNA gene) is inserted into a plasmid. Both the plasmid and insert have been digested and have the proper sticky or blunt ends which are compatible for joining the two DNA pieces together into one molecule. These two DNA pieces are placed in a reaction tube and the proper DNA ligase, buffer, and cofactors are added for the reaction to take place. When done properly, the ligation will result in a successful combination of the insert and plasmid into one plasmid. This newly formed plasmid may then be isolated using gel electrophoresis and then used for bacterial transformation or other experimentation. The following is the procedure used by USU to ligate PHB genes into the biobrick plasmids.  
'''Method''' 
1. Obtain the following reagents, some of which are in the -20C freezer:  DNA vector, DNA insert, 10X ligation buffer, T4 DNA ligase (take out only when needed, then return immediately to freezer), and sterile distilled water. 
2. Ideally, it is desirable to have the concentration of insert ends (or moles of insert) be two to three times the concentration of vector ends (or moles of vector), with a total DNA concentration of 50-400 ng/l in the reaction.  If determining the DNA concentration is not possible, place two to three times the volume of vector as the volume of insert in the reaction.  As this is often the case, place the following reagents in a thin-walled PCR tube in the following volumes: 

	10 l insert DNA 
	3 l vector DNA 
	2 l 10X ligation buffer 
	4 l H2O 
	1 l T4 DNA ligase 
= ''20 l total'' 

This could also be done in different volumes depending on DNA concentration/total volume desired. 
3. Gently mix the tube, and place the tube in the PCR thermocyler, turn on the machine, select Start, from the main menu, select 22 and press Start.  The thermocycler will keep the reaction at 22C. 
4. Incubate for 60 minutes.  Heat-inactivate by placing tubes in 68C water bath for 10 minutes.  Place in the freezer if storing for later use.   

===Western Blot===

[[I really don't know what the heck this protocol is for.  Better ask someone.]]

'''Method''' 
1.	Collect cells (confluent T-25) by trypsinization and spin.  
2.	Lyse the pellet with 100 l lysis buffer on ice for 10 min.
*For 500,000 cells, lyse with 20 l.  
3.	Spin at 14,000 rpm (16,000 g) in an Eppendorf microfuge for 10 min at 4C.  
4.	Transfer the supernatant to a new tube and discard the pellet.  
5.	Determine the protein concentration (Bradford assay, A280, or BCA)
*(We use the Bradford assay from Bio-Rad.)  
6.	Take x l (= y g protein) and mix with x l of 2x sample buffer.  
7.	Boil for 5 min.  
8.	Cool at RT for 5 min.  
9.	Flash spin to bring down condensation prior to loading gel.  
10.	Assemble gel in gel rig.  
11.	Prepare protein samples (10 g will suffice).  
12.	Use 5 l of Kaleidoscope standard.  
13.	Run at 200 V (constant voltage) for 30 min.  
14.	Cut a piece of PVDF membrane (Millipore Immobion-P #IPVH 000 10).  
15.	Wet for about 30 min in methanol on a rocker at room temp.  
16.	Remove methanol and add 1x Blotting buffer until ready to use.  
17.	Assemble &quot;sandwich&quot; for Bio-Rad's Transblot.  
18.	Prewet the sponges, filter papers (slightly bigger than gel) in 1x Blotting buffer. 
*        Sponge - filter paper - gel - membrane - filter paper - sponge  
19.	Transfer for 1 hr at 1 amp at 4C on a stir plate. 
*        Bigger proteins might take longer to transfer.
*        For the Mini-Transblot, it's 100 V for 1 hr with the cold pack and prechilled buffer.  
20.	When finished, immerse membrane in Blocking buffer and block overnight.  
21.	Incubate with primary antibody diluted in Blocking buffer for 60 min at room temp.  
22.	Wash 3 x 10 min with 0.05% Tween 20 in PBS.  
23.	Incubate with secondary antibody diluted in Blocking buffer for 45 min at room temp.  
24.	Wash 3 x 10 min with 0.05% Tween 20 in PBS.  
25.	Detect with Amersham ECL kit (RPN 2106).  
26.	Rinse blot off with 0.05% Tween 20 in PBS.  
27.	Put blot into Kapak bag cut to slightly bigger size than blot.  
28.	Add about 5 to 10 ml Stripping buffer.  
29.	Remove as much air as possible and seal bag.  
30.	Immerse into 80C water bath and incubate for 20 min.  
31.	Rinse blot off with 0.05% Tween 20 in PBS.  
32.	Block for about 1 hr with 5% BSA/Tween 20, or overnight with 3% BSA/Tween 20. 

===Total Protein Assay===

[[This protocol needs a description too.]]

'''Method'''  
1.	Prepare standard dilutions of BSA of 25, 50, 75 and 100  g/ml. Prepare appropriate serial dilutions of the sample to be measured.  
2.	Place 1.0 ml of each of the above into separate tubes. Add 100  l of sodium deoxycholate to each tube. 
3.	Wait 10 minutes and add 100  l of TCA to each tube.  
4.	Centrifuge each tube for 15 minutes at 3,000 xg and discard the supernatant.  
5.	Add 1.0 ml of water to each tube to dissolve the pellet. Add 1.0 ml of water to a new tube to be used as a blank.  
6.	Add 1.0 ml of CTC to each tube (including the blank), vortex and allow to set for 10 minutes.  
7.	Add 500  l Folin-Ciocalteu to each tube, vortex and allow it to set for 30 minutes.  
8.	Turn on and zero a spectrophotometer to a wavelength of 750 nm. Use the blank from Step 7 to adjust for 100% T.  
9.	Read each of the standards and samples at 750 nm.  
10.	Plot the absorbance of the standards vs. their concentration. Compute the extinction coefficient and calculate the concentrations of the unknown samples.


===Site-Directed Mutagenesis===


'''Mutant Strand Synthesis Reaction (Thermal Cycling)''' 

[[The original protocol had a table here.]] 

1. Synthesize two complimentary oligonucleotides containing the desired
mutation, flanked by unmodified nucleotide sequence. Purify these
oligonucleotide &quot;primers&quot; prior to use in the following steps (see
Mutagenic Primer Design). 

2. Prepare the control reaction as indicated below: 
*5 l of 10 reaction buffer (see Preparation of Media and Reagents)
*2 l (10 ng) of pWhitescript 4.5-kb control plasmid (5 ng/l)
*1.25 l (125 ng) of oligonucleotide control primer #1 [34-mer (100 ng/l)]
*1.25 l (125 ng) of oligonucleotide control primer #2 [34-mer (100 ng/l)]
*1 l of dNTP mix
*39.5 l of double-distilled water (ddH2O) to a final volume of 50 l
Then add
*1 l of PfuTurbo DNA polymerase (2.5 U/l) 

3. Prepare the sample reaction(s) as indicated below: 
Note: Set up a series of sample reactions using various
concentrations of dsDNA template ranging from 5 to 50 ng
(e.g., 5, 10, 20, and 50 ng of dsDNA template) while keeping
the primer concentration constant. 
*5 l of 10 reaction buffer
*X l (550 ng) of dsDNA template
*X l (125 ng) of oligonucleotide primer #1
*X l (125 ng) of oligonucleotide primer #2
*1 l of dNTP mix
*ddH2O to a final volume of 50 l 
Then add 
*1 l of PfuTurbo DNA polymerase (2.5 U/l)
*QuikChange Site-Directed Mutagenesis Kit 9

4. If the thermal cycler to be used does not have a hot-top assembly,
overlay each reaction with ~30 l of mineral oil. 

5. Cycle each reaction using the cycling parameters outlined in Table I.
(For the control reaction, use a 5-minute extension time and run the
reaction for 18 cycles.) 

6. Adjust segment 2 of the cycling parameters in accordance with the type
of mutation desired (see the following table): 
[[This thing needs some work too.]] 
Type of mutation desired 			Number of cycles 
Point mutations 				12 
Single amino acid changes 			16 
Multiple amino acid deletions or insertions 	18 

7. Following temperature cycling, place the reaction on ice for 2 minutes
to cool the reaction to =37C. 


If desired, amplification may be checked by electrophoresis of
10 l of the product on a 1% agarose gel. A band may or may not
be visualized at this stage. In either case proceed with Dpn I
digestion and transformation. 


'''Dpn I Digestion of the Amplification Products''' 

Note:  It is important to insert the pipet tip below the mineral oil overlay
when adding the Dpn I restriction enzyme to the reaction tubes
during the digestion step or when transferring the 1 l of Dpn Itreated
DNA to the transformation reaction. Using specialized
aerosol-resistant pipet tips, which are small and pointed,
facilitates this process. 

1. Add 1 l of the Dpn I restriction enzyme (10 U/l) directly to each
amplification reaction below the mineral oil overlay using a small,
pointed pipet tip. 

2. Gently and thoroughly mix each reaction mixture by pipetting the
solution up and down several times. Spin down the reaction mixtures in
a microcentrifuge for 1 minute and immediately incubate each reaction
at 37C for 1 hour to digest the parental (i.e., the nonmutated)
supercoiled dsDNA. 


'''Transformation of XL1-Blue Supercompetent Cells'''

Notes: Please read the Transformation Guidelines before proceeding with
the transformation protocol.
XL1-Blue cells are resistant to tetracycline. If the mutagenized
plasmid contains only the tetR resistance marker, an alternative
tetracycline-sensitive strain of competent cells must be used. 

1. Gently thaw the XL1-Blue supercompetent cells on ice. For each
control and sample reaction to be transformed, aliquot 50 l of the
supercompetent cells to a prechilled 14-ml BD Falcon polypropylene
round-bottom tube. 

2. Transfer 1 l of the Dpn I-treated DNA from each control and sample
reaction to separate aliquots of the supercompetent cells.
Note Carefully remove any residual mineral oil from the pipet tip
before transferring the Dpn I-treated DNA to the
transformation reaction. 
As an optional control, verify the transformation efficiency of the
XL1-Blue supercompetent cells by adding 1 l of the pUC18 control
plasmid (0.1 ng/l) to a 50-l aliquot of the supercompetent cells.
Swirl the transformation reactions gently to mix and incubate the
reactions on ice for 30 minutes. 

3. Heat pulse the transformation reactions for 45 seconds at 42C and then
place the reactions on ice for 2 minutes.
Note This heat pulse has been optimized for transformation in
14-ml BD Falcon polypropylene round-bottom tubes.
QuikChange Site-Directed Mutagenesis Kit 11 

4. Add 0.5 ml of NZY+ broth (see Preparation of Media and Reagents)
preheated to 42C and incubate the transformation reactions at 37C for
1 hour with shaking at 225250 rpm. 

5. Plate the appropriate volume of each transformation reaction, as
indicated in the table below, on agar plates containing the appropriate
antibiotic for the plasmid vector. 
For the mutagenesis and transformation controls, spread cells on
LBampicillin agar plates containing 80 g/ml X-gal and 20 mM IPTG
(see Preparing the Agar Plates for Color Screening). 

[[This is supposed to be a table.]] 

Transformation reaction plating volumes
Reaction Type Volume to Plate
pWhitescript mutagenesis control 250 l
pUC18 transformation control 5 l (in 200 l of NZY+ broth)*
Sample mutagenesis 250 l on each of two plates
(entire transformation reaction) 
Place a 200-l pool of NZY+ broth on the agar plate, pipet the 5 l of the
transformation reaction into the pool, then spread the mixture. 

6. Incubate the transformation plates at 37C for &gt;16 hours.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Utah_State/Protocols">http://2009.igem.org/Team:Utah_State/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Protocols&amp;oldid=137631" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>