<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Secretion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Utah_State/Secretion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Secretion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Secretion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Secretion&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Secretion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Secretion</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P class="class"><TITLE>Untitled Document</TITLE><TABLE width="100%" border="0"><TBODY><TR valign="top"><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State"><FONT size="4">HOME</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Team"><FONT size="4">ABOUT US</FONT></A></TD></TR><TR><TD width="172" id="ana"><SPAN class="currentPage"><FONT size="4">PROJECT</FONT></SPAN><A href="https://2009.igem.org/Team:Utah_State/Abstract">Abstract</A><A href="https://2009.igem.org/Team:Utah_State/Introduction">Introduction</A><A href="https://2009.igem.org/Team:Utah_State/Broad-HostVectors">Broad-Host Vectors</A><A href="https://2009.igem.org/Team:Utah_State/Secretion">Secretion</A><A href="https://2009.igem.org/Team:Utah_State/Experiments">Experiments</A><A href="https://2009.igem.org/Team:Utah_State/FutureWork">Future Work</A><A href="https://2009.igem.org/Team:Utah_State/References">References</A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Parts"><FONT size="4">BIOBRICKS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Modeling"><FONT size="4">MODELING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Notebook"><FONT size="4">NOTEBOOK</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/ETHICS"><FONT size="4">ETHICS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Achievements"><FONT size="4">JUDGING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Contact"><FONT size="4">CONTACT</FONT></A></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><B><I><FONT size="5" face="Century Gothic, Arial, San Serif" color="#000033">
Secretion: Bioplastics, Phasin, and GFP
    </FONT></I></B></TD></TR></TBODY></TABLE></P><P class="class">
Recovery of cellular products is often a difficult and expensive challenge. As much as 80% of protein production costs are attributable to downstream processing (Hearn and Acosta, 2001). Likewise, the separation and purification cost for non-protein products, like polyhydroxyalkanaotes (PHAs) are significant and commonly represent more than half of the total process expense (Ling, 1998; Jung, 2005).  </P><P class="class">
Polyhydroxyalkanoates comprise a class of polyesters that are generated by a variety of microorganisms (Anderson and Dawes, 1990; Doi, 1990). These bioplastic compounds are intracellularly accumulated and stored as a reserve of carbon, energy, and reducing power in response to an environmental stress or nutrient limitation (Lee, 1996). Polyhydroxybutyrate (PHB) is the most common form of PHA. PHAs have comparable material properties to conventional plastics, like polypropylene, but are fully biodegradable and renewable (Steinbüchel and Füchtenbusch, 1998). As a result, PHAs are of particular interest as a sustainable source of non-petrochemically derived thermoplastics for use in an assortment of commercial and medical applications (Madison and Huisman, 1999).</P><P class="class">Costs associated with the PHA manufacturing process have limited the widespread application of the bioplastic material (Lee, 1996). Economic analyses for industrial scale PHA production place the cost of PHAs at about $4-5/kg (Choi, 1997; Choi, 1999). In contrast, the average cost of petrochemically-derived plastic lies between $0.62-0.96/kg (Steinbüchel and Füchtenbusch, 1998). This significant discrepancy in expense is largely attributable to downstream processing. Traditional methods involving the use of solvents, enzymatic digestion, or mechanical disruption are expensive and impractical for industrial-scale recovery (Jung, 2005). As a result, the development of alternative methods for PHA recovery is necessary.</P><P class="class">Genetic engineering strategies have been used in attempts to simplify PHA recovery and eliminate the need for mechanical or chemical cellular disruption. Jung et al. (2005) used recombinant <I>E. coli</I> MG1655 harboring PHA biosynthesis genes from C. necator to instigate spontaneous autolysis of the cell wall. Up to 80% of the cells in culture released PHA granules, which were subsequently recovered using centrifugation and washing with distilled H2O (Jung, 2005). Resch et al. (1998) used recombinant PHA-producing <I>E. coli</I> transformed with the E-lysis gene of bacteriophage PhiX174 from plasmid pSH2. Amorphous PHB in is pushed out of the cell through an E-lysis tunnel structure, which is an opening in the cell envelope (Resch, 1998). In this procedure, the osmotic pressure difference between the cytoplasm and the culture medium provides the driving force for PHA movement into the extracellular medium. The PHA is then recovered by centrifugation or through the addition of divalent cations (Resch, 1998). Although these methods use genetic means to bring about cellular disruption, these mechanisms still require cellular death and fail to promote a continuous production system.  </P><P class="class">Recently, extracellular deposition of PHA granules was observed in a mutant strain of Alcanivorax borkumensis SK2, which is a marine bacterium that uses hydrocarbons as its source of carbon and energy (Sabirova, 2006). This finding by Sabirova et al (2006) is the first account of PHA accumulation outside of the cell (Prieto, 2007). However, the mechanism by which this deposition occurs is unknown (Sabirova, 2006; Prieto, 2007). A defined system for microbial excretion of PHAs has yet to be created. Such a system would be of value due to the potential to optimize and introduce the mechanism into other organisms with advantageous characteristics, such as fast-growing <I>E. coli</I> or photoautotrophic PHA-producers <I>R. sphaeroides</I> and <I>Synechocystis</I> PCC6803. </P><P class="class">PHA-associated proteins, called phasins, strongly interact with the PHA granule surface (York, 2001; Maehara, 1999). Accordingly, PHA recovery may be possible by tagging the phasin protein for translocation. Specifically, the Silver fusion Biobrick standard can be used to create constructs in which a targeting signal peptide sequence is genetically fused to the phasin protein (Phillips, 2006). Fusing a signal peptide to a protein promotes export of the complex out of the cytoplasm (Choi, 2004; Mergulhão, 2005). The interaction of phasin with PHA is required for secretion-based granule recovery because PHA is a non-proteinaceous compound produced by the action of three enzymes (Suriyanmongkol 2007; Verlinden 2007). Consequently, the signal peptide cannot be directly attached PHA granules. The phasin protein with attached signal peptide binds to PHA granules, thereby creating a PHA-phasin-signal peptide complex that may be recognized by the cell for export. Figure 1 depicts this export process in general terms. Green fluorescent protein (GFP) translocation has been documented (Barrett, 2003; Santini, 2001; Thomas, 2001). Due to its ease of detection, studying GFP in parallel with phasin secretion mechanisms could provide a framework for determining the functionality of secretion systems.</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 1.</B>  Schematic for bioplastic recovery by secretion
</FONT></DIV><P class="class">
Secretion-based product recovery mechanisms hold great potential to improve the economics of industrial-scale production systems. In addition to reduced downstream processing requirements, secretory production has additional benefits, such as potentially improved product stability and solubility (Mergulhão, 2005). Recombinant <I>E. coli</I> do not typically secrete high levels of proteins and functionality of proteins secretion is difficult to predict (Sandkvist, 1996; Choi, 2004). Accordingly, a trial-and-error approach with different combinations of signal peptides and promoters is recommended for any given protein, and will be discussed in more detail in subsequent sections (Choi, 2004). 
</P><B><I><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20">
Principles of Recombinant Protein Secretion
    </FONT></I></B><P class="class">
The functionality of protein secretion mechanisms is affected by the structural differences between gram-positive and gram-negative organisms (Desveaux, 2004; Sandkvist, 1996). Gram-positive species have a solitary cytoplasmic membrane, which effectively means that protein membrane translocation is equivalent to secretion in these species (Pugsley, 1993). Alternatively, gram-negative organisms have both an inner and outer membrane that proteins must cross for secretion.  Accordingly, proteins can either be exported into the periplasmic space or secreted fully into the extracellular medium (Pugsley, 1993). </P><P class="class">
There are five pathways observed for secretion of recombinant proteins in gram-negative prokaryotes, numbered I through V (Desvaux, 2004; Mergulhão, 2005). While all of these pathways differ mechanistically, they each promote secretion while maintaining the integrity of the cell structure (Koster, 2000). Types I and II are the most common pathways for recombinant protein secretion (Mergulhao, 2005) and will be discussed here. </P><P class="class">Type I secretion is a single-step translocation of protein across both inner and outer membranes. (Binet, 1997). The constituents of this system include inner membrane proteins HlyB and HlyD, as well as the TolC outer membrane protein (Mergulhão, 2005; Desveax, 2004). These three proteins interact to form a channel that spans the periplasm (Mergulhão, 2005). Appending the last 42-60 amino acids of the HlyA protein C-terminus to the C-terminus of a recombinant protein targets the protein for secretion (Mergulhão, 2005; Gentschev, 2003; Hess, 1990). The HlyA signal sequence binds to the channel complex, resulting in ATP hydrolysis by HlyB to drive protein secretion (Gentschev, 2003). Proteins as large as 4000 amino acids can be secreted through the type I channel, which has an internal diameter of 3.5 nm and a length of 14 nm (Sapriel, 2003; Fernandez and de Lorenzo, 2001).  Unlike in the Type II pathway, the signal peptides of Type I secretion remain attached to the protein after export out of the cytoplasm (Blight and Holland, 1994).  Figure 2 depicts the secretion of a protein with a C-terminal fused HlyA signal peptide by Type I secretion (Mergulão, 2005).  
</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 2.</B>  HlyA Type I Secretion Pathway
</FONT></DIV><P class="class">
The type II secretion pathway is a two-step process. The cytoplasmic protein must first be exported into the periplasm through the action of a translocase. Specifically, the Sec and Twin-arginine translocation (TAT) machinery facilitate protein movement across the inner membrane and will be discussed in detail in the next section.  After entering the periplasm, the protein can be translocated into the extracellular medium through the action of a secreton, which is a 12-16 core protein complex present in many gram-negative strains, such as <I>E. coli</I> K-12 (Cianciotto, 2005).  Although the secreton functionality is not completely understood, it is known that protein conformational changes are necessary for this process to be carried out (Mergulhão, 2005; Sandkvist, 2001).</P><P class="class">
Translocation of cellular products into the periplasm is advantageous over cytoplasmic production because recovery of periplasmic products is relatively simpler (Mergulhão, 2005). There are additional mechanisms for recovering periplasmic proteins if the secreton machinery is either not present in the host strain or incompatible with the protein of interest. These mechanisms are depicted in Figure 3. L-form and Q-cells are mutant strains that have a weakened outer membrane, which allows for some proteins to leak into the extracellular medium (Mergulhão, 2005).  However, these organisms have reduced growth rates and are not ideal candidates for general cellular production.  The permeability of the outer membrane may be enhanced mechanically, such as by application of ultrasound, or through chemical treatment, such as through addition of Triton X-100 or 2% glycine (Kaderbhai, 1997; Choi, 2004). As another example, enzymatic digestion with lysozyme breaks the outer membrane to release periplasmic proteins (Shokri, 2003).  Yet another alternative involves coexpression of genes, such as kil, out, and tolAIII, that cause cellular lysis and subsequent release of recombinant proteins (Choi, 2004; Mergulhão, 2005). The downside to these alternatives is the weakening of cell integrity.
</P><B><I><FONT size="3" face="Helvetica, Arial, San Serif" color="#000033">
Cytoplasmic Membrane Translocation in the Type II Pathway
    </FONT></I></B><P class="class">
Several membrane-associated components mediate translocation of proteins across the inner membrane of gram-negative <I>E. coli</I> (Luirink, 2004). This machinery includes translocases, ATPases, and accessory proteins (Luirink, 2004; Veenendaal, 2004). The Sec pathway and the TAT system are the two general mechanisms by which proteins are transported into the periplasm, with the Sec-translocon providing most common export route (Luirink, 2004; Veenendaal, 2004). Within the Sec-dependent category, proteins are exported either via the SecB-dependent pathway or by the action of the signal recognition particle (SRP). The attachment of a short sequence, called a signal peptide, to the N-terminus of a protein is generally necessary for targeting proteins to any of the three translocation pathways (Luirink, 2004; Choi, 2004; Mergulhão, 2005). </P><P class="class">In the Sec pathway, SecA is attached peripherally to the inner membrane and drives peptide translocation through ATPase activity (van der Does, 2004).  Integral membrane proteins SecY and SecE form the core of the Sec translocon, and SecG interacts with this core to form a multimeric protein complex, SecYEG (Veenendaal, 2004). This complex functions as a protein-conducting channel for both post-translational and co-translational protein export (Luirink, 2004; Veenendaal, 2004). Interestingly, the SecYEG translocon can be found in all domains of life, reiterating the prevalence and importance of this mechanism for protein export (Cao, 2002). </P><P class="class">A SecB-dependent mechanism is used by gram-negative species to target post-translational periplasmic and outer membrane proteins to the Sec-translocon (Luirink, 2004). Of the three translocation routes, the Sec-B pathway is the most common for recombinant protein export (Mergulhão, 2005). First, a trigger factor binds to the preprotein as it leaves a ribosome (Luirink, 2004; Mergulhão, 2005). Next, the unfolded protein is recognized and bound by the SecB chaperone protein and directed to SecA, where ATP hydrolysis provides the force to drive the protein through the SecYEG translocase into the periplasm (Mergulhão, 2005). In co-translational protein export, a signal recognition particle (SRP) identifies and interacts with the signal sequence of the nascent protein as it is exiting the ribosome to the Sec-translocon (Luirink, 2004; von Heijne, 1996; Mergulhão, 2005). </P><P class="class">
The TAT system is used to export folded proteins into the periplasmic space (Choi, 2004).  Like the Sec-dependent pathways, specific N-terminal signal peptide sequences target a protein for export by the TAT machinery. Although similar, TAT signal peptides differ from those that target proteins to the Sec machinery. TAT signal peptides contain a conserved sequence of seven amino acids, (S/T)-R-R-x-F-L-K, at the interface between the N- and H-regions, where x represents a polar amino acid (Berks, 2000; Palmer, 2004). The twin-arginine residues are consistently present in TAT signal peptides, and the occurrence of the other amino acids is greater than 50% (Berks 1996, Berks 2000, Palmer, 2004). Figure 3 illustrates the mechanism for protein export by the Sec and TAT pathways.</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 3.</B>  Mechanism of protein translocation by Sec and Tat
</FONT></DIV><P class="class">
Whether a protein is targeted to the SecB, SRP, or TAT pathways is largely dependent on the characteristics of the attached signal peptide (Mergulhão, 2005; van der Does, 2004; Luirink, 2004). For example, the hydrophobicity of the signal peptide plays a role in designating which route will be used for protein export (Berks, 2000; Luirink, 2004). The affinity of a signal sequence to the SRP increases as the number of hydrophobic residues in the H-domain of the signal peptide (Valent, 1997). The trigger factor of the SecB pathway recognizes slightly less hydrophobic sequences in the signal peptide and consequently prevents binding by the SRP.  Lastly, TAT pathway signal sequences are the most hydrophilic in the H-domain (Berks, 2000). Moreover, increasing H-domain hydrophobicity of TAT signal sequences can even divert a protein typically translocated via the TAT pathway to the Sec translocon (Berks, 2000; Cristobal, 1999).  The mature region of the protein may also play a role in pathway targeting, particularly in regard to the SecB mechanism (Luirink, 2004). </P><B><I><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20">
Signal Peptides
    </FONT></I></B><P class="class">
Signal peptides consist of about 15-30 amino acids and are generally required to direct a secretory protein to the translocons of the cytoplasmic membrane (Pugsley, 1993; Choi, 2004; Luirink, 2004). Despite overall sequence variability, structural similarities exist between different signal peptides, including a positively-charged 2-10 amino acid N-region, a hydrophobic core H-region, and a neutral C-domain of about 6 residues (Pugsley, 1993; Molhoj, 2004; Berks, 2000).  The C-domain conforms to the -3, -1 rule in which amino acids with short and neutral side-chains, such as alanine, are required in positions -3 and -1 of the sequence (Choi, 2004; von Heijne, 1984). A signal peptidase interacts with a cleavage recognition site within the C-domain to release the protein into the periplasmic space (Luiritz, 2004; Choi, 2004). The absence or mutation of the cleavage site can lead to the targeted protein remaining fixed to the inner membrane (Luiritz, 2004). Figure 4 shows the typical composition of a signal peptide sequence.</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 4.</B>  Typical signal peptide sequence
</FONT></DIV><P class="class">
A small signal sequence is typically necessary for all translocation pathways. However, certain protein-coding sequences can be secreted without having an attached signal sequence due to the presence of additional targeting information within the sequence (Luiritz, 2004). Additionally, an attached signal sequence does not guarantee export of a protein, which further suggests that information in the protein sequence itself can affect secretion efficiency (Luiritz, 2004).  However, the fusion of a signal sequence to a recombinant protein can lead to export of a previously non-secretable protein. There are many reported examples of recombinant protein translocation through signal sequence gene fusion. For example, fusion with the Tat-dependent signal peptide TorA allowed for export of folded GFP into the periplasm of <I>E. coli</I> (Palmer, 2004; Barrett, 2003; Santini, 2001; Thomas, 2001). </P><P class="class">
Two factors that affect protein export are the positive charge of the N-terminus of the signal peptide and the charge of the N-terminus of the recombinant protein (Akita 1990). Akita et. Al (1990) determined that increasing the positive charge of the signal peptide N-terminus not only enhances the interaction with SecA protein, but also reduces the requirements of SecA ATPase activity for translocation.  Therefore, a higher net positive N-terminus charge improves the rate of protein translocation (Mergulhão, 2005). For the recombinant protein, the charge of the N-terminus also affects protein secretion. A net positive charge within the first five amino acids near the C-domain cleavage site of the signal sequence can reduce protein export by as much as 50-fold because the charge inhibits the protein from entering the lipid bilayer (Schatz, 1990). </P><P class="class">
Although factors like hydrophobicity and charge are known to affect protein export, there are few available guidelines for selecting a proper signal peptide for any given protein (Choi, 2004). It is advised to carry out investigation of recombinant protein secretion by trial-and-error with different host strains and signal peptides (Choi, 2004).  The mechanisms of protein secretion are complicated and many obstacles can inhibit the process. Some commonly observed problems include incomplete translocation, degradation of recombinant protein by proteases, formation of inclusion bodies, and inefficiency of secretion machinery (Mergulhão, 2005; Choi, 2004). Optimization of the secretion efficiency requires balancing the promoter strength and gene copy number so as not to overwhelm the system (Mergulhão, 2005). Lastly, some proteins may simply be unsuitable for secretion due to their size or sequence (Koster, 2000). </P><B><I><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20">
Phasin
    </FONT></I></B><P class="class">
Phasin (PhaP) is a low-molecular weight protein that plays a role in PHA granule formation by physically binding to the PHA granule surface (York, 2001). The specific purpose of phasin production is not completely understood (York 2002), although some of the affects of the phasin/PHA interaction have been studied. York et al (2001) determined that the production of phasin is dependent on PHA accumulation. Specifically, it is suggested that phasin expression requires the presence of PHA synthase (York, 2001). Maehara et al (1999) observed that the level of PHA accumulation substantially decreases and the size of PHA granules increases when phasin is either absent or regulated by a repressor, PhaR.  Therefore, PHA production levels are enhanced in the presence of phasin due to an increased granule surface-to-volume ratio (York 2001; Maehara 1999). </P><P class="class">
In addition to reducing PHA granule size, other functions of phasin have been proposed. In the absence of phasin, other proteins can bind to the granule surface (Maehara, 1999). Therefore, phasins may function to inhibit attachment of other proteins to the PHA surface that could cause defects in granule formation (York 2001; Maehara, 1999). Lastly, it is suggested that phasins promote PHA synthesis through an interaction with PHA synthase (York, 2001). </P><P class="class">
Due to their physical interaction with the PHA granule, phasins can be used in recombinant protein purification (Banki, 2005), or PHA recovery as this project is investigating. For protein purification, genetic fusion of a protein product, a self-splicing element called an intein, and phasin can be used (Banki, 2005). The genetically-fused protein is produced in <I>E. coli</I> harboring the PHB production genes (Banki, 2005). The phasin protein binds to the surface of the PHB granule, and a cleavage-inducing buffer stimulates the release of the product protein into the soluble fraction of the solution (Banki, 2005). </P><P class="class">
For this procedure, PHB is released and proteins are recovered only after the cell lysed, which is not ideal. However, the system provides evidence that the phasin/PHA interaction may be exploited for improving production processes and that genetic fusion of other elements with phasin does not inhibit binding to PHA (Banki, 2005).  The fusion of phasin with a signal peptide, which is a sequence that tags a protein for secretion, could result in a signal peptide/phasin/PHA complex that is recognized by cell for transmembrane export. </P><P class="class">
The recovery of PHA granules via secretion of a signal peptide/phasin/PHA complex may be inhibited due to the size of PHA granules. However, the binding of phasins decreases PHA molecular weight and encourages the formation of numerous, small granules (Maehara, 1999). Though the actual size of PHA granules varies, Maehara et al (1999) observed spherical granules approximately 20 – 60 nm in diameter in the presence of phasin and absence of the PhaR repressor.  This indicates that enhanced production of phasin may further reduce granule size, which may make PHAs more suitable for export. </P><B><I><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20">
Green Flourescent Protein
    </FONT></I></B><P class="class">
GFP is a commonly used reporter of gene regulation.  It is expressed in many bioluminescent jellyfish naturally (Shimomura, 1962). Its value in the academic and biotechnology industry was recognized after successful cloning and expression in <I>E. coli</I> (Chalfie, 1994).  Purified GFP, composed of 238 amino acids, absorbs blue light (395 nm) and emits green light (Chalfie, 1994).  The detection of intracellular GFP is not limited by the availability of substrates, but requires only irradiation by near UV or blue light (Chalfie, 1994).  However, to ease the process of GFP detection for many organisms, a stronger whole cell fluorescence signal is desirable.  Figure 5 depicts the GFP barrel structure.</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 5.</B>  The GFP Barrel Structure
</FONT></DIV><P class="class">Many mutant forms of GFP have been created which improve fluorescence photostability and ultimately the ability of GFP to function as a practical reporter.  The cycle 3 mutant developed by Crameri et al. (1996) is of special interest because it produces a fluorescence signal 45-fold greater than wild-type GFP.  The developed GFP possesses three point mutations of the wild-type GFP.  These mutations do not affect the chromophore itself, but reside in the surrounding barrel of the GFP protein.  In <I>E. coli</I>, due to its hydrophobic nature, most of the wild-type GFP gathers to form inclusion bodies that limit the ability of blue light to provide the necessary excitation energy to activate fluorescence (Crameri , 1996).  The three point mutations in the cycle 3 mutant, have no effect on excitation and emissions maxima, but create a more hydrophilic GFP less prone to form inclusion bodies.  The soluble mutant is easily activated by a UV light box or light wand common in the laboratory creating an immediate, practical reporter protein. Furthermore, fusions onto amino- or carboxy-termini of GFP do not inhibit fluorescence, which makes GFP an ideal candidate for fusion studies (LaVallie, 1995).</P><FONT size="3" face="Century Gothic, Arial, San Serif" color="#000033"><B><A href="#top">Go to Top of Page</A></B></FONT></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Utah_State/Secretion">http://2009.igem.org/Team:Utah_State/Secretion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Secretion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Secretion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Secretion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Secretion&amp;oldid=169426" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>