"
Page
Discussion
View source
History
teams
Log in
 
Team:Utah State/Notebook
From 2009.igem.org
Untitled Document
HOME
ABOUT US
PROJECT
BIOBRICKS
MODELING
NOTEBOOK Meeting Notes
Lab Notebook
Protocols
ETHICS
JUDGING
CONTACT
Official Meetings
May 12
Introduction to team members and to iGEM.  Reviewed last year’s competition and last year’s team contribution.  Introduction also to the 2009 iGEM home page and our team wiki.
June 23
After researching previous projects and brainstorming since our last meeting, we discussed the possibility of continuing the project that University of Hawaii initiated last year.  We spoke with an advisor from their team and agreed that we could continue the project.  Our team agreed that it would be a good project foundation.
June 30
Small workshop on programming a wiki.  Further collaboration with team Hawaii has taken place and we are waiting for DNA constructs with which they worked.
August 20
Discussed progress made with broad-host vector construction.  The broad-host vector constructs that we have been working with from Team Hawaii and from the iGEM parts catalog do not appear to be functioning.  After PCR, attempted ligations, enzymatic digestions and electrophoretic gel observations, we’ve decided to move on and try to modify another known broad-host vector to be compatible with the BioBrick format.
September 3
Team discussion of PHAs, phasin, silver-fusion, and secretory pathways.  Further discussion of attempted broad-host vector modifications.
September 10
Reviewed judging criteria and reviewed our standing with the broad-host vector and the secretion pathways.  Discussed which tracks would be most applicable to our project.  Discussed titles for our project.  Finalization of team roster and travel information.
September 17
Made final decisions for our intended track, chose a final project title, and gave a final review of our abstract.  Discussed our wiki progress.
September 24
Took team picture.  Presentation of different team logo options and team shirt design options.  Flash animation presentation to be used potentially in wiki and presentation.
September 29
Team meeting with internet programming advisor.  Discussed final formatting options for wiki.
October 1
Instruction given on tri-parental mating.  Discussed selective plates and media for tri-parental mating.
October 6
Instruction given on Western Blot procedure and function.  Discussed modified GFP construct.  Final team logo was presented.
October 8
Met shortly and separated to work on different assignments for wiki.
October 13
Final T-shirt design presented to team.  Reviewed completed parts and discussed broad-host vector.  Presentation of completed secretion pathway models.
October 15
Discussed portions of the wiki that needed to be completed before the weekend.  Reviewed project components and iGEM basics in preparation for the jamboree.
October 20
Final meeting before wiki closure.  Discussed last minute assignments to ensure that the wiki is completed.
Go to Top of Page
Laboratory Notebook
Members of our team each had individual lab notebooks. Rather than outline each procedure that was run by each individual person, we have instead decided to present our wiki lab notebook as a weekly update of the progress that was made. These weekly updates are presented in our weekly meeting outlines, and much of the day-to-day happenings are presented in the various procedure details and specifics found below in the protocols section.
Figure 1. Some of our laboratory notebooks
Go to Top of Page
Protocols
Bacterial Transformation
Once the target DNA has been successfully ligated into the plasmid vector, the plasmid must be transferred into the host cell for replication and cloning. In order to do this, the bacterial cells must first be made “competent.” The term “competent” is to describe a cell state in which there exist gaps or openings in the cell wall which will allow the plasmid containing the target genes to enter into the cell. Several methods to make bacterial cells competent exist, such as the calcium chloride method and electroporation. Competent cells may also be purchased commercially. The team at USU has purchased competent ce lls for all experiments. The following is the method used by the USU team to insert the plasmids containing various biobricks into the cells.
Method
Ensure the necessary antibiotic agar plates have been prepared or begin their preparation now. Four plates per transformation will be necessary (two today, then two tomorrow for streaking). Also ensure that 10 ml liquid media is made up per transformation (also for tomorrow).
If using Biobrick parts from iGEM distribution, use registry to identify appropriate will containing plasmid of interest and proceed to step 3, if using other DNA proceed to step 5.
Add 10ul of sterile water to distribution well to dissolve DNA.  Remove 10ul and place in 0.5ml bullet tube.  Label tube with part number, use 2ul to transform and save the other 8ul in the BioBrick part box.
Take competent cells from the -80˚C freezer and place on an ice bath.
Add 2 μl of the DNA solution (or 4ul of ligation reaction) to the competent cells. Ensure the pipetting is done directly into the cell solution. Let cells incubate on ice for 30 minutes. Heat water bath to 42˚C.
Heat shock cells in the 42˚C water bath for 30 seconds. Remove and place back in the ice bath for 2 minutes.
In the hood, add 250 μl SOC media to each tube, bringing the total cell solution to 300 μl. Incubate at 37˚C for 1 hour.
Add 200 μl of each transformed cell solution to the appropriate antibiotic plate. Use the Bunsen burner to create a “hockey stick” out of a glass pipette tip by holding over the flame until it bends. Allow to cool. Spread cell solution uniformly over the agar plate using the “hockey stick,” then before discarding, spread residual solution on the “stick” over a second plate to get more a more sparse colony distribution.
Parafilm all plates and place in 37˚C incubator 12-14 hours, or overnight if that is not possible.
Streak Plates and Liquid Cultures from Transformed Colonies
After bacterial cells have been transformed, successfully transformed cells must be selected. Because 100% of the cells do not receive the desired plasmid and target gene, it is essential to select for cells that do have the target genes. The USU team uses antibiotic resistance to select for successful transformations. To do this, an antibiotic resistance gene is also added to the plasmid vector that contains the target genes. By doing so, it is possible to know that a cell was successfully transformed based on its ability to grow on an agar plate with antibiotics added. Because the cell is able to grow, the antibiotic resistance gene must be present as well as the target gene. From the agar plates containing the antibiotics, a colony is picked and transferred into a liquid culture for further analysis. The following is the method used by USU to clone the DNA and select for the successful transformation of various BioBricks in E.coli.
Method
Prepare two 15 ml tubes per transformation, each with 5 ml media containing the appropriate antibiotic.
Use a pipette tip to extract half of each colony and inoculate one agar plate per colony. Using a pipette with a tip, extract the other half of each colony and inoculate one liquid media tube per colony. Label all tubes and plates and place in the 37˚C incubator until the next morning.
Plasmid DNA Isolation
Following successful bacterial cloning and isolation, it is important to verify that the target gene is in the cell and that the resultant plasmid is correct. To do this, it is a common practice to sequence the plasmid DNA. To obtain enough DNA for sequencing, the bacterial clones are grown in a liquid culture. The cells are harvested by centrifugation and then prepared for DNA plasmid extraction. DNA plasmid extraction can be done several ways, and the overall purpose is to lyse the cells and separate the plasmid DNA from all other cellular proteins, DNA, and debris. The following is the method used by the USU team to isolate plasmid DNA containing the various biobricks.
Method
Prepare two water baths, one boiling and the other 68C.
Centrifuge bactrerial cultures (3 to 5 ml) at 3K RPM for 20 min. Discard supernatant.
Resuspend cell pellet in 200 μl of STET buffer. Transfer to 1.5 ml tubes.
Add 10 μl of lysozyme (50 mg/ml) and incubate at room temperature for 5 min.
Boil for 45 sec and centrifuge for 20 min at 13K RPM (or until pellet gets tight).
Use a pipette tip or toothpick to remove the pellet.
Add 5 μl RNase A (10 mg/ml) to supernatant and incubate at 68C for 10 minutes.
Add 10 μl of 5% CTAB and incubate at room temperature for 3 min.
Centrifuge for 5 min at 13K RPM, discard supernatant, and resuspend in 300 μl of 1.2 M NaCl by vortexing.
Add 750 μl of ethanol and centrifuge for 5 min at 13K RPM.
Discard supernatant, rinse pellet (which cannot be seen) in 80% ethanol, and let tubes dry upside down with caps open.
Resuspend pellet in either sterile water or TE buffer.
Restriction Enzyme Digestion and Electrophoresis
Restriction enzyme digestion is the process by which an insert DNA sequence is separated from the rest of the DNA molecule. Specific knowledge of the DNA insert is needed to determine which enzyme and conditions to use during the digestion reaction. Once the DNA sequence is known and the correct enzymes have been selected, the DNA may be digested. Listed below is the procedure used by USU to digest the plasmid DNA. After enzyme digestion, electrophoresis is used to separate the plasmid from the insert. A gel is prepared and the respective reaction mixes are loaded into the gel. Using a DNA ladder, and knowing the size of the insert, the corresponding band can be seen and cut out of the gel. The insert may then be removed and isolated from the gel, thus yielding the desired DNA. The DNA from this may then be used in PCR reactions, sequencing, ligations for further experimentation, etc. Listed below are example protocols used by the USU team for a restriction enzyme digestion and subsequent agarose gel electrophoresis.
Method
Resuspend DNA in 20 to 40 μl water, vortex, and do a brief centrifuge to get solution to the bottom of the tube.
Add components to the digestion solution in the following order: DNA (23 μl), 10X restriction enzyme buffer (3 μl), Xba1 (2 μl), and Pst1 (2 μl). The volume and restriction enzymes can be varied, but it should be ensured that the total volume is 10X the amount of RE buffer. Tap tubes periodically and allow to digest at appropriate temperature while preparing electrophoresis gel.
Prepare electrophoresis gel by adding 2 g agarose to 200 ml TAE (1% solution). This is best done in an Erlenmeyer flask of adequate volume as swirling will need to be done. Place in the microwave and microwave on high for 20 seconds at a time, pulling it out and swirling until solution is homogeneous again, then repeating (BE CAREFUL to watch the solution closely when swirling – it superheats and can boil over and cause severe burns). Continue until solution is seen boiling in the microwave then gently swirl again.
Add 20 μl ethidium bromide to solution and swirl until dissolved evenly.
Add 6 μl of 6X loading dye to each tube of digested DNA solution.
Prepare the electrophoresis unit by orienting the basin sideways with rubber gaskets firmly against the side. Place desired well template in the basin.
When the agarose solution is cool enough to comfortably touch the flask, pour into the basin until the solution is about ¾ of the way to the top of the well template.
When the gel is solidified (should look somewhat cloudy), remove the well template and change basin orientation to have the wells closest to the negative pole (as the DNA will flow towards the positive pole). Pour 1X TAE buffer into both sides of the electrophoresis unit until it just covers the gel and fills the wells.
By inserting the pipette tip below the TAE liquid and into the well, add 10 μl of DNA ladder solution to first (and last if desired) well, skip one well, then begin adding the digested DNA solutions to the wells by adding about 2 μl less than the total volume in the tubes to prevent air bubbles in the wells.
Place the cover on the electrophoresis unit, plug into the power source, and turn on voltage to 70 V (this can be as high as 100 V if time is an issue), and press the start button. Separation should take two to three hours. The yellow dye shows the location of the smaller nucleotide lengths and the blue dye shows the location of the larger nucleotide lengths. DNA separation can be observed as time goes on by turning off the power supply then gently removing the basin from the electrophoresis unit (be careful not to let the gel slip out of the basin) and placing on the UV transilluminator to see DNA bands. The basin can then be placed back in the electrophoresis unit for further separation if desired. Take care to not have the power supply on without the lid to the unit in place.
When the desired level of separation is obtained, the basin can be placed on the transilluminator for picture taking. Place the cone-shaped cover over the transilluminator and place the digital camera in the top hole for pictures.
Media Preparation
For all experimentation involving the need for bacterial biomass and experimentation, proper media is needed to grow the cells. The following is the media composition and methods used by USU to prepare the media.
Add 5 g yeast extract, 10 g NaCl, 10 g Bacto Tryptone, and 15 g agar (if desired) to a 2 L Erlenmeyer flask and bring the volume up to 1 L with ddH20. Mix by swirling. Cover top with foil.
Autoclave for 45 minutes (liquid setting, 0 minutes drying time). It will take an additional half hour for the autoclave to finish cooling then an additional 20 to 30 minutes until the media is cool enough to pour.
Polymerase Chain Reaction (PCR)
PCR is used to amplify a desired DNA sequence. The reaction is first set up by designing primers that will bind only to the desired regions of the DNA sequence. Once the primer and polymerase have been selected, the reaction parameters of time and temperature must be optimized. When the reaction works properly only the target DNA will be amplified into large quantities that may then be isolated and used for further experimentation. The following is the procedure used by USU for PCR reactions to amplify various biological parts.  A useful set of primers are the universal BioBrick primers VF2 and VR that can be used to amplify almost any BioBrick part.
Method
Obtain the following reagents from the freezer: DNA template (cells or DNA), 10X Taq buffer (+KCl, -Mg/Cl2), MgCl2, 10 mM dNTP Mix, Taq polymerase (take out of freezer only immediately when needed and put back), and sterile distilled H2O. Place all reagents on ice. Also obtain PCR (either 0.2 or 0.5 ml) tubes.
Add the following reagents to a tube (50 μl reaction) in the following volumes and order:
32 μl sterile H2O,
5 μl 10X buffer,
2 μl dNTP Mix,
6 μl MgCl2
3 μl cells/DNA,
0.25 μl Taq Polymerase
1 μl primer 1
1 μl primer 2
MgCl2 volume can be varied (lower to increase specificity – just ensure total volume is 50 ul with H2O). If many reactions are to be constructed, a master mix can be made up to cut down on time and pipette tip usage (if this is done, ensure primers are added to the appropriate reaction, i.e. perhaps not to the master mix). Tap or vortex tubes and take to the thermocyler. Place all reagents back in the -20˚C freezer.
Choose thermocycler temperatures. The Eppendorf Mastercycler will cycle between three temperatures: typical temperatures are 94˚C for denaturing, 50-60˚C for primer annealing, and 72˚C for polymerase extending. Lowering the annealing temperature decreases DNA specificity; 55˚C is a good temperature to begin if no trials have been made with the sample.
Turn on thermocycler with the switch in the back of the unit and open the lid. The placement of the tubes depends on the size of the tube (0.2 or 0.5 ml) and whether or not a temperature gradient is to be used.
If no temperature gradient will be used, tubes can be placed anywhere on the unit in the appropriately-sized hole. Select “Files” and press enter. Select “Load” and then “Standard.” If cells will be used in the reaction, include a 1-minute lysing step at the beginning (step 1); this will be followed by a 1-minute DNA denaturing step (step 2). If purified DNA will be used, set step 1 to 1 second. Set an annealing temperature for step 3. Ensure the lid temperature is 105˚C and the extending temperature is 72˚C. Press exit. If prompted to save, save by pressing enter three times. Press exit to return to the main menu. Choose “Start” on the main menu and select “Standard.” The program should begin.
If a temperature gradient is to be used, temperature will vary according to column. A 20˚C range is the maximum range that can be used (+/- 10˚C). The range is made by setting a temperature for the middle column and then setting a +/- range. To see what the temperatures will be if a gradient is used, select “OPTIONS” on the main menu, then select “Gradient.” Select the size tube that is being used by pressing “Sel,” then press enter. Choose a temperature for the center column, press enter, then select a +/- range and press enter. The column number along with the corresponding temperature is shown. Decide tube placement based on this information. Press exit twice to return on the main menu. Select “Files” then “Load,” then “Gradient.” If cells are being used, set the cell lysing step (step 1) to 1 minute (1:00); if purified DNA is being used, set this time to 1 second (0:01). Step 2 should be 94C, Step 4 should be 72˚C, and the lid temperature should be 105˚C. Go to step 3 and set an annealing temperature for the center column. Leave the next two lines as they are, and change the gradient setting (“G”) to the +/- the center temperature amount. Press exit. If prompted to save, press enter three times; if not prompted to save, press enter once. Press exit to get back to the main menu. To begin cycle, select “Start,” then select “Gradient.” The program should begin.
The thermocycler is set to store the completed reaction tubes at 4˚C when finished.
Ligation
Ligation is the process by which the insert (target DNA gene) is inserted into a plasmid. Both the plasmid and insert have been digested and have the proper “sticky” or blunt ends which are compatible for joining the two DNA pieces together into one molecule. These two DNA pieces are placed in a reaction tube and the proper DNA ligase, buffer, and cofactors are added for the reaction to take place. When done properly, the ligation will result in a successful combination of the insert and plasmid into one plasmid. This newly formed plasmid may then be isolated using gel electrophoresis and then used for bacterial transformation or other experimentation. The following is the procedure used by USU to ligate together various biobrick parts.
Method
1. Obtain the following reagents, some of which are in the -20˚C freezer: DNA vector, DNA insert, 10X ligation buffer, T4 DNA ligase (take out only when needed, then return immediately to freezer), and sterile distilled water.
2. Ideally, it is desirable to have the concentration of insert ends (or moles of insert) be two to three times the concentration of vector ends (or moles of vector), with a total DNA concentration of 50-400 ng/μl in the reaction. If determining the DNA concentration is not possible, place two to three times the volume of vector as the volume of insert in the reaction. As this is often the case, place the following reagents in a thin-walled PCR tube in the following volumes:        • 10 μl insert DNA        • 3 μl vector DNA        • 2 μl 10X ligation buffer        • 4 μl H2O        • 1 μl T4 DNA ligase        = 20 μl total        This could also be done in different volumes depending on DNA concentration/total volume desired.
3. Gently mix the tube, and place the tube in the PCR thermocyler, turn on the machine, select “Start,” from the main menu, select “22” and press “Start.” The thermocycler will keep the reaction at 22˚C.
4. Incubate for 60 minutes. Heat-inactivate by placing tubes in 68C water bath for 10 minutes. Place in the freezer if storing for later use.
Western Blot
Western blotting is a procedure that allows for the identification of proteins using a specific antibody after protein separation on an SDS polyacrylamide gel.
Method
Collect bacterial cells by centrifugation and lyse the cells using any of a variety of procedures.
Spin at 14,000 rpm (16,000 g) in a microfuge for 10 min at 4°C.
Transfer the supernatant to a new tube and discard the pellet.
Determine the protein concentration (Bradford assay, A280, or BCA)
Mix 20 µl of sample (20ug) with 10ul of 3x sample buffer.
Boil for 1 min, ool at RT for 5 min.
Flash spin to bring down condensation prior to loading gel.
Assemble pre-prepared polyacrylamide gel into gel running rig.
Load protein samples into individual wells.
Use 10 µl of Kaleidoscope standard.
Run at 35 mA (constant current) for approximately 2hrs.
Disassemble gel when done running.
Cut a piece of PVDF membrane (Millipore Immobion-P #IPVH 000 10).
Wet for about 10 min in methanol on a rocker at room temp.
Remove methanol and add 1x Blotting buffer until ready to use.
Assemble "sandwich" for Bio-Rad's Transblot.
Sponge - filter paper - gel - membrane - filter paper - sponge
Transfer for 1 hr at 100V at 4°C on a stir plate. Bigger proteins might take longer to transfer.
When finished, immerse membrane in blocking buffer (5%  nonfat dry milk)and block overnight.
Incubate with primary antibody diluted in 0.5% blocking buffer for 60 min at room temp.
Wash 3 x 10 min with 0.05% Tween 20 in PBS.
Incubate with secondary antibody diluted in 0.5% blocking buffer for 45 min at room temp.
Wash 3 x 10 min with 0.05% Tween 20 in PBS.
Detect with TMB stabilized substrate for HRP.
Site-Directed Mutagenesis
QuikChange II Site-Directed Mutagenesis Kit (Stratagene)
Synthesize two complimentary oligonucleotides containing the desired mutation, flanked by unmodified nucleotide sequence.
Prepare the control reaction as indicated below:
5 μl of 10× reaction buffer (see Preparation of Media and Reagents)
2 μl (10 ng) of pWhitescript 4.5-kb control plasmid (5 ng/μl)
1.25 μl (125 ng) of oligonucleotide control primer #1 [34-mer (100 ng/μl)]
1.25 μl (125 ng) of oligonucleotide control primer #2 [34-mer (100 ng/μl)]
1 μl of dNTP mix
39.5 μl of double-distilled water (ddH2O) to a final volume of 50 μl
Then add
1 μl of PfuTurbo DNA polymerase (2.5 U/μl)
Prepare the sample reaction(s) as indicated below:
Note: Set up a series of sample reactions using various concentrations of dsDNA template ranging from 5 to 50 ng (e.g., 5, 10, 20, and 50 ng of dsDNA template) while keeping the primer concentration constant.
5 μl of 10× reaction buffer
X μl (5–50 ng) of dsDNA template
X μl (125 ng) of oligonucleotide primer #1
X μl (125 ng) of oligonucleotide primer #2
1 μl of dNTP mix
ddH2O to a final volume of 50 μl
Then add
1 μl of PfuTurbo DNA polymerase (2.5 U/μl)
If the thermal cycler to be used does not have a hot-top assembly, overlay each reaction with ~30 μl of mineral oil.
Cycle each reaction using the cycling parameters as outlined in Table I of the Stratagene QuikChange II Site-Directed Mutagenesis Kit manual. We used an annealing temperature of 55C for 1 min and an extension temperature of 68C for 5 min and 18 cycles.
Following temperature cycling, place the reaction on ice for 2 minutes to cool the reaction to ≤37°C. If desired, amplification may be checked by electrophoresis of 10 μl of the product on a 1% agarose gel. A band may or may not be visualized at this stage. In either case proceed with Dpn I digestion and transformation.
Dpn I Digestion of the Amplification Products
Add 1 μl of the Dpn I restriction enzyme (10 U/μl) directly to each amplification reaction below the mineral oil overlay using a small, pointed pipet tip.
Gently and thoroughly mix each reaction mixture by pipetting the solution up and down several times. Spin down the reaction mixtures in a microcentrifuge for 1 minute and immediately incubate each reaction at 37°C for 1 hour to digest the parental (i.e., the nonmutated) supercoiled dsDNA. Transform into XL1-Blue Supercompetent Cells and proceed as previously described.
Go to Top of Page
Retrieved from " http://2009.igem.org/Team:Utah_State/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
