<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Utah_State/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Untitled Document</TITLE><TABLE width="100%" border="0"><TBODY><TR valign="top"><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State"><FONT size="4">HOME</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Team"><FONT size="4">ABOUT US</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Project"><FONT size="4">PROJECT</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Parts"><FONT size="4">BIOBRICKS</FONT></A></TD></TR><TR><TD width="172" id="ana"><SPAN class="currentPage"><FONT size="4">MODELING</FONT></SPAN><A href="#parameters">Parameters</A><A href="#simulations">Simulations</A><A href="#references">References</A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Notebook"><FONT size="4">NOTEBOOK</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/ETHICS"><FONT size="4">ETHICS</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Achievements"><FONT size="4">JUDGING</FONT></A></TD></TR><TR><TD id="nav"><A href="https://2009.igem.org/Team:Utah_State/Contact"><FONT size="4">CONTACT</FONT></A></TD></TR></TBODY></TABLE><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><FONT size="5" face="Century Gothic, Arial, San Serif" color="#000033"><B><I>Modeling Secretion Mechanisms</I></B></FONT></TD></TR></TBODY></TABLE></P><P class="class"> 
There are currently few guidelines for selecting which signal peptide (which determine the secretion pathway taken) should be used for a given recombinant protein (Choi, 2004). The construction of models that evaluate protein secretion may provide a useful framework for studying and attempting to predict factors that affect secretion efficiency. Models of the Sec and Tat translocation pathways were made using MATLAB’s Simbiology toolbox. They were evaluated using the embedded ordinary differential equation solver in the software. Assumptions were made for both models due to lack of time scale information for individual steps for these translocation mechanisms. However, both provide a flexible framework that can become more detailed as additional information is found in literature or discovered in the laboratory. Within both models, a protein is first generated and then carried out of the cytoplasm to the periplasm. Only protein species were tracked, as species involved with making the protein are currently not as useful to monitor.
   </P><A name="parameters"><B><I><FONT size="3" face="Helvetica, Arial, San Serif" color="#000033">
Model Parameters
    </FONT></I></B><P class="class"> 
In both models, parameters for the manufacture of protein were the same and found using averaged values for <I>E. coli</I>. The average length of an mRNA for <I>E. coli</I> is 1100 nucleotides and an <I>E. coli</I> cell transcribes at an average rate of 70 nucleotides per second. A simple rate calculation determines that a strand of mRNA is made approximately every 15.7 seconds. This value was then used to yield a first order reaction rate of (1/15.7) 1/second.  The translation rate was found in a similar manner. Given an average protein size in <I>E. coli</I> as 360 amino acids and the average translation rate is 40 amino acids per second, the first order translation rate constant is (1/9) 1/s (Institute for Biomolecular Design, 2008). To initiate the model, the concentration of the gene was set at 1, which was locked at that value over the course of the simulation. The median half life for mRNA was found to be 3.7 minutes, after which it will degrade (Milo, n. d.). For the Tat-dependent mechanism, the protein is fully folded prior to translocation (Mergulhão, 2005). For this folding time, a place holder value of 10 minutes was used as an approximate order of magnitude, although the value can easily be changed depending on parameters of the protein in question. Degradation of protein was modeled as the average time it takes the cell to reproduce, which is given as half an hour (Institute for Biomolecular Design, 2008).
   </P><P class="class"> 
In addition to the discussed values regarding the manufacture of protein, parameters for the Sec and Tat pathway are required. A literature review yielded no specific rate constants or time parameters for the individual steps involved with either process. However, one article stated that the Sec and Tat pathways take a few seconds and a few minutes, respectively, to translocate protein to the periplasmic space (Mergulhão, 2005). Accordingly, the Sec and Tat pathways were modeled to require 3 seconds and 3 minutes for protein translocation, respectively. First order rate constants were determined by taking the inverse of these values in seconds. Images of the model diagrams are seen below. The different pathways after the formation of protein can be activated and deactivated as more detailed information comes available about the translocation and secretion processes. The Sec secretion pathway also includes the SRP secretion pathway as a possible option. The constructed models for the Sec and Tat pathways are given as Figures 1 and 2, respectively.
</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 1.</B>  Sec Pathway Model Diagram
</FONT></DIV><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 2.</B>  Tat Pathway Model Diagram
</FONT></DIV></A><A name="simulations"><B><I><FONT size="3" face="Helvetica, Arial, San Serif" color="#000033">
Simulations
    </FONT></I></B><P class="class">Simulations for the Sec pathways were run for 5400 seconds (1.5 hours), which allowed the number of periplasmic to reach approximately steady state. The simulation for the Sec pathway secretion model is shown below as Figure 3. 
Protein generated by the <I>E. coli</I> cell reaches steady state quickly, as it is both produced and translocated in to the periplasm rapidly. The periplasmic protein reaches approximately 90 proteins upon achieving steady state. The long settling time and high number of periplasmic protein are a result of the long protein degradation time.</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 3.</B>  Simulation of protein production and translocation by the Sec pathway
</FONT></DIV><P class="class">Simulations for the Tat pathway were run for 3600 seconds (1 hour).  The shorter length of simulation relative to the Sec model is used despite having significantly longer protein translocation time because the time-consuming folding step limits the reaction. The results of the ordinary differential equation simulation are shown below in Figure 4. The amount of periplasmic protein is lower than cytoplasmic and folded protein values upon reaching steady state. This is due to the rate-limiting folding step, which also leads to an expected high concentration of folded protein as well.
</P><DIV align="center"><FONT size="2.5" face="Helvetica, Arial, San Serif" color="#231f20"><B>Figure 3.</B>  Simulation of protein production and translocation by the Sec pathway
</FONT></DIV><P class="class"> The accuracy of models, such as those constructed, will increase as more knowledge about the time step parameters for each of the mechanisms is determined. These improvements will allow for a better comparison of the two secretion methods, which can lead to better signal peptide selection (and corresponding pathway selection) for the protein in question. Future work could also include incorporating promoter strength into the model to affect transcription rates to provide a more accurate depiction of what is happening inside the cell. Better correlation between these parameters and the model will result in a more useful aid in when studying protein secretion. 
</P></A><A href="#top">Go to Top of Page</A></DIV><TABLE width="100%" style="background:#CCCCCC; padding:7px; border-style:none"><TBODY><TR><TD><B><I><FONT size="3" face="Helvetica, Arial, San Serif" color="#000033">
References
    </FONT></I></B><P><FONT size="2.5" face="Tahoma, Helvetica, Sans-Serif" color="#000000"><UL class="circle"><LI>Institute for Biomolecular Design (2008, August 1) CyberCell Database. Retrieved October 2009. http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
</LI><LI>Choi JH, Lee SY (2004) Secretory and extracellular production of recombinant proteins using <I>Escherichia coli</I>. Appl Microbiol Biotechnol 64:625-635</LI><LI>Mergulhão FJM, Summers DK, Montier GA (2005) Recombinant protein secretion in <I>Escherichia coli</I>. Biotechnol Adv 23:177-202</LI><LI>Milo R (n.d.) BioNumbers: The Database of Useful Biological Numbers. Retrieved October 2009. http://bionumbers.hms.harvard.edu/default.aspx</LI></UL></FONT></P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Utah_State/Modeling">http://2009.igem.org/Team:Utah_State/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;oldid=167270" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>