<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Slovenia_Protease_signaling_Orthogonality skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Slovenia/Protease signaling/Orthogonality</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Orthogonality</TITLE></P><DIV id="example"><DIV class="pusher"><DIV class="full height"><DIV class="banana"><DIV class="ui vertical sticky text menu"><A class="item" href="//2016.igem.org/Team:Slovenia/Protease_signaling/Reporters"><B>Reporters</B></A><A class="item" href="//2016.igem.org/Team:Slovenia/Protease_signaling/Orthogonality" style="color:#DB2828;"><B>Orthogonality</B></A><A class="item" href="#ach" style="margin-left: 10%"><B>Achievements</B></A><A class="item" href="#mot" style="margin-left: 10%"><B>Introduction</B></A><A class="item" href="#var" style="margin-left: 10%"><B>TEVp variants</B></A><A class="item" href="#hom" style="margin-left: 10%"><B>TEVp homologs</B></A><A class="item" href="//2016.igem.org/Team:Slovenia/Protease_signaling/Split_proteases"><B>Split proteases</B></A></DIV></DIV><DIV class="article" id="context"><DIV><DIV class="main ui citing justified container"><DIV><H1 class="ui left dividing header"><SPAN id="ach" class="section colorize"> </SPAN>Protease
                                orthogonality</H1><DIV class="ui segment" style="background-color: #ebc7c7; "><P><B><UL><LI>Two TEVp variants (TEVpE and TEVpH) were tested and proved to be mutually
                                            orthogonal.

                                        </LI><LI>Upon overexpression none of the tested proteases were toxic to mammalian
                                            cells, demonstrating that they do not interfere with essential cellular
                                            processes.

                                        </LI><LI>Additionally, three TEVp homologues (PPVp, SbMVp and SuMMVp) were tested and
                                            proved to be fully orthogonal.
                                     

                                    </LI></UL></B></P></DIV></DIV><DIV class="ui segment"><H4><SPAN id="mot" class="section colorize"> </SPAN></H4><P>The first challenge in construction of a new protease-based signaling cascade was the
                                selection of appropriate proteases. The candidate proteases should:
                                i) recognize defined target cleavage sequences, preferably as long as possible; ii) be
                                active in mammalian cells, but not toxic to them and iii) be inducible,
                                ideally through the reconstitution of split protein fragments. Most importantly, a large
                                number of proteases with similar properties but different
                                substrates should be available to allow for modular construction of signaling pathways
                                and logic functions. These proteases should be orthogonal to each
                                other, meaning that their specific cleavage sites should not be recognized by other
                                proteases in the system.
                            </P><P>We found that the tobacco etch virus protease (TEVp) was the only protease described in
                                the literature to match our criteria.</P><DIV class="ui styled fluid accordion"><DIV class="title">
                                    Further explanation ...
                                </DIV><DIV class="content"><P>TEV protease is a highly specific, 242 amino acids long, 27 kDa cysteine
                                        protease, that originates from the tobacco etch virus (TEV) of the Potyvirus
                                        genus.
                                        It has a target recognition sequence of seven amino acids, ENLYFQ-S/G, where
                                        cleavage occurs after the glutamine residue (denoted by the – symbol),
                                        and the final residue of the recognition sequence can be either S or G (denoted
                                        by the / symbol). This substrate sequence is scarcely represented in the
                                        proteome. TEV protease is therefore relatively non-toxic
                                        <X-REF>Parks1994</X-REF>
                                        and can be safely expressed in host cells. Due to this non-toxicity and
                                        its high cleavage specificity, TEVp is an attractive protease for use in several
                                        biotechnological applications, such as removal of affinity tags
                                        from recombinant proteins.
                                    </P><P>Despite its widespread use in biotechnology, TEVp also displays some
                                        shortcomings, the most prominent of them being self-cleavage. Substitution
                                        of Ser-219 with Val or Pro
                                        <X-REF>Cesaratto2015</X-REF>
                                        or a replacement of the C-terminal sequence MSELVYSQ with the sequence MNEGGGLE
                                        <X-REF>Cesaratto2015</X-REF>
                                        decreased the self-cleavage of TEVp and thereby increased its activity.
                                    </P></DIV></DIV><P>To overcome the lack of inducible orthogonal proteases, we looked for the characterized
                                TEVp mutants and naturally occurring proteases, closely related to
                                TEVp that might also be used to function as split proteases.</P></DIV><H1><SPAN class="section colorize"> </SPAN>Results</H1><DIV class="ui segment"><DIV><H3><SPAN id="var" class="section colorize"> </SPAN>TEVp variants</H3><P>Based on the sequence alterations described by Yi et al.
                                    <X-REF>Yi2013</X-REF>
                                    , our team decided to test two mutated variants of TEVp - TEVpE and TEVpH.
                                </P><DIV class="ui styled fluid accordion"><DIV class="title">
                                        Further explanation ...
                                    </DIV><DIV class="content"><P>Yi et al.
                                            <X-REF>Yi2013</X-REF>
                                            tackled the problem of acquiring novel orthogonal proteases by screening a
                                            library of TEVp mutants for orthogonal
                                            specificity. They designed a novel yeast ER sequestration screening assay
                                            that allowed them to identify variants of TEVp that recognize
                                            alternative substrates. The two most prominent mutants were TEVpE, which
                                            cleaves the substrate ENLYFE-S (TEVsE) and TEVpH, cleaving the substrate
                                            ENLYFH-S (TEVsH). Although the two variants were also able to cleave the
                                            wild type TEVp substrate
                                            ENLYFQ-S (TEVs), they displayed a high preference for their own variation of
                                            the substrate and were mutually orthogonal.
                                        </P></DIV></DIV><P>To test these two proteases we used a <A href="https://2016.igem.org/Team:Slovenia/Protease_signaling/Reporters#cle">cleavable
                                    firefly luciferase
                                    (fLuc) reporter</A> with an appropriate cleavage sequence inserted in a permissible
                                    site. We observed a significant decrease in luciferase activity upon
                                    co-expression of the reporters and their corresponding proteases, whereas
                                    co-expression of an orthogonal protease did not decrease the luciferase activity (<REF>1</REF>). These results were additionally confirmed by results from western blotting, where
                                    cleavage of the luciferase reporter was
                                    detected only in the presence of the corresponding protease, but not in the presence
                                    of orthogonal proteases (<REF>2</REF>).
                                </P><DIV style="float:left; width:100%"><FIGURE data-ref="1"><FIGCAPTION><B>Activity and orthogonality of TEVp variants.</B><P style="text-align:justify">(A) Activity of cleavable fLuc with the wild type TEVs cleavage sequence is reduced only in the presence of wild type TEVp. (B) Activity of cleavable fLuc with the TEVsE cleavage sequence is reduced only in the presence of TEVpE. (C) Activity of cleavable fLuc with the  TEVsH cleavage sequence is reduced only in the presence of TEVpH. HEK293T cells were transfected with the
                                                indicated fLuc:TEVs and TEVp variant constructs. Luciferase activity was
                                                determined 24 h after transfection. (D) Table of cleavage sequences for the wild type TEVp and its variants. The cleavage occurs before the final Ser and is indicated by -.
                                            </P></FIGCAPTION></FIGURE></DIV><P style="clear:left;">No data has previously been reported on TEVpE and TEVpH toxicity,
                                    therefore we expressed each TEVp variant in
                                    HEK293T cells and performed a viability assay. Even after transfection with a high
                                    amount of the plasmid for each respective protease, the cells showed high viability,
                                    with practically
                                    no difference when compared to cells transfected with a mock plasmid (<REF>3</REF>).
                                </P><DIV style="width:60%; float:left;"><FIGURE data-ref="2"><FIGCAPTION><B>Orthogonality and activity of TEVpE and TEVpH.</B><P style="text-align:justify">HEK293T cells were transfected with 2000 ng of
                                                the indicated protease and 500 ng of the indicated reporter. Cells
                                                were lysed and analyzed by western blotting with anti-AU1 antibodies.
                                                The cleaved reporter (55 kDa) was detected only in the presence of the
                                                corresponding
                                                TEVp variant.
                                            </P></FIGCAPTION></FIGURE></DIV><DIV style="width:40%; float:left;"><FIGURE data-ref="3"><FIGCAPTION><B>Different TEVp variants are not toxic to HEK293T cells.</B><P style="text-align:justify">HEK293T cells were transfected with plasmids
                                                coding for different TEVp variants. 48 hours after transfection, cells
                                                were stained with the trypan blue dye, which stains only dead cells. The
                                                ratio of live and dead cells was analysed with the Countess automated
                                                cell counter.
                                            </P></FIGCAPTION></FIGURE></DIV></DIV><DIV><H3 style="clear:both"><SPAN id="hom" class="section colorize"> </SPAN>TEVp homologs</H3><P>Introduction of two new TEVp variants expanded our repertoire of tools, demonstrating
                                    that we can use the results of mutational screenings for
                                    protease variants. However, a larger number of strictly orthogonal proteases would
                                    be required for modular design of complex logic circuits. We therefore decided to
                                    investigate the activity of de novo created split proteases from the potyviridae
                                    family.
                                </P><DIV class="ui styled fluid accordion"><DIV class="title">
                                        Further explanation ...
                                    </DIV><DIV class="content"><P>The NIa proteases from the potyviridae group of plant viruses in general
                                            recognize a seven amino acid sequence motif as their substrate. They are
                                            are classified as cysteine proteases with an active site closely related to
                                            eukaryotic serine proteases. The NIa proteases adopt a characteristic
                                            two-domain antiparallel β-barrel fold. The active site of the protease
                                            comprises a catalytic triad: His-46, Asp-81, Cys-151 (amino acids numbered
                                            according to the TEVp sequence) with a Gly-x-Cys-Gly motif around the active
                                            cysteine residue
                                            <X-REF>Yoon2000, Nunn2005</X-REF>
                                            .
                                        </P></DIV></DIV><P>We searched for different potyviruses available on UniProt (<REF>4</REF>), paying particular
                                    attention to any evidence of orthogonality among their target substrates.
                                    We decided to test the plum pox virus protease (PPVp), the soybean mosaic virus
                                    protease (SbMVp), and the sunflower mild mosaic virus protease (SuMMVp).
                                </P><DIV style="clear:right; width:70%"><FIGURE data-ref="4"><FIGCAPTION><B>Orthogonal proteases from the potyviridae family.</B><P style="text-align:justify">Homology models obtained from <A style="padding-right: 0" href="https://swissmodel.expasy.org/">SWISS-MODEL</A> using the
                                                crystal structure of TEVp (red) (<A style="padding-right: 0" href="http://www.rcsb.org/pdb/explore.do?structureId=1LVB">PDB 1LVB</A>) as template. The
                                                homology models of the potyviridae family proteases PPVp (blue), SbMVp
                                                (cyan) and
                                                SuMMVp (yellow) are shown in ribbons.
                                            </P></FIGCAPTION></FIGURE></DIV><DIV class="ui styled fluid accordion"><DIV class="title">
                                        Further explanation ...
                                    </DIV><DIV class="content"><P>                        PPVp is one of the most studied potyviral proteases after the TEV protease.
                                            Its substrate (PPVs) has an amino acid sequence NVVVHQ-A.
                                            In contrast to TEVp, it has been reported that PPVp is resistant to
                                            self-cleavage at the C-terminus
                                            <X-REF>Zheng2008, Garcia1991</X-REF>
                                            .SbMVp has been
                                            recently studied by Seo et al. as a tool for protein-protein interaction
                                            studies in the soybean plant. The substrate (SbMVs) has been determined to
                                            be the
                                            sequence ESVSLQ-S
                                            <X-REF>Seo2016, Yoon2000</X-REF>
                                            .Similarly, SuMMVp has been used by Fernandez-Rodriguez et al.
                                            <X-REF>Fernandez-Rodriguez2016</X-REF>
                                            . in a study of genetic circuits using potyviral proteases. The substrate
                                            (SuMMVs) has been defined as the sequence
                                            EEIHLQ-S/G
                                            <X-REF>Fernandez-Rodriguez2016</X-REF>
                                            .
                                        </P></DIV></DIV><P>All selected potyviral proteases were designed as synthetic genes and their activity
                                    was tested in mammalian cells using
                                    <A href="https://2016.igem.org/Team:Slovenia/Protease_signaling/Reporters#cyc">cyclic
                                        luciferase reporters</A>, which propagate luciferase activity only upon
                                    cleavage.
                                    We detected an increase of luciferase activity only in the corresponding
                                    protease-reporter pairs, confirming exquisite orthogonality of the selected
                                    proteases and
                                    their activity in the human cell chassis (<REF>5</REF>).
                                </P><DIV style="clear:right; width:70%"><FIGURE data-ref="5"><FIGCAPTION><B>Protease orthogonality.</B><P style="text-align:justify">(A) Table of cleavage sequences for TEVp and its homologues. The cleavage occurs before the final Ser or Ala and is indicated by -. (B) Heat map showing orthogonality of the TEVp homologues. Dark red corresponds to higher luciferase activity. (C) Orthogonality of TEVp homologues shown as bar graphs. HEK293T cells were transfected with the
                                                indicated cycLuc reporters and proteases. Luciferase activity was
                                                detected only in the presence
                                                of a protease and cycLuc reporter containing appropriate protease
                                                cleavage site.
                                            </P></FIGCAPTION></FIGURE></DIV></DIV></DIV><H3 class="ui left dividing header"><SPAN id="ref-title" class="section colorize"> </SPAN>References
                        </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>