<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Slovenia_Protease_signaling_Logic skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Slovenia/Protease signaling/Logic</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Logic</TITLE></P><DIV id="example"><DIV class="pusher"><DIV class="full height"><DIV class="banana"><DIV class="ui vertical sticky text menu"><A class="item" href="//2016.igem.org/Team:Slovenia/Protease_signaling/Light_dependent_mediator"><B>Light-dependent mediator</B></A><A class="item" href="//2016.igem.org/Team:Slovenia/Protease_signaling/Logic" style="color:#DB2828;"><B>Logic</B></A><A class="item" href="#ach" style="margin-left: 10%"><B>Achievements</B></A><A class="item" href="#mot" style="margin-left: 10%"><B>Introduction</B></A><A class="item" href="#ini" style="margin-left: 10%"><B>Initial testing</B></A><A class="item" href="#ant" style="margin-left: 10%"><B>Antiparallel CCs</B></A><A class="item" href="#cle" style="margin-left: 10%"><B>Cleavable constructs</B></A><A class="item" href="#des" style="margin-left: 10%"><B>Destabilized CCs</B></A><A class="item" href="#ind" style="margin-left: 10%"><B>Inducible logic</B></A><A class="item" href="//2016.igem.org/Team:Slovenia/Hardware"><B>Ultrasound controlling device</B></A></DIV></DIV><DIV class="article" id="context"><DIV class="main ui citing justified container"><DIV><H1 class="ui left dividing header"><SPAN id="ach" class="section colorize"> </SPAN>Protease-based signaling and logic</H1><DIV class="ui segment" style="background-color: #ebc7c7; "><P><UL><LI><B>New antiparallel and destabilized coiled coil pairs were designed and functionally characterized in mammalian cells.
                                </B></LI><LI><B>Coiled coils were combined with split luciferase fragments to design functions with logical negation.
                                </B></LI><LI><B>Light and chemically inducible proteases were used as mediators in a functional proof of concept for fast regulated logic gates.
                            </B></LI></UL></P></DIV></DIV><DIV class="ui segment"><H3 style="clear:both;"><SPAN id="mot" class="section colorize"> </SPAN></H3><P>As the main challenge of our project was to create fast responsive synthetic circuits in cells, we sought to implement logic operations based on protein post-translational modification, rather than slower transcriptional activation. Introduction of a protease targeting site has already been used to engineer linear signaling pathways that functioned <I>in vitro</I> using linked autoinhibitory peptides <X-REF>Stein2014</X-REF>. However our developed set of <A href="https://2016.igem.org/Team:Slovenia/Protease_signaling/Orthogonality">orthogonal
                                proteases</A> that could additionally be <A href="https://2016.igem.org/Team:Slovenia/Protease_signaling/Split_proteases">split</A>, provided the modules to implement logic functions, for which we had to design the appropriate framework.
							
								An inspiration
                            was provided by the study by Shekhawat <I>et al.</I> in which they presented an <I>in vitro</I> protease sensor using autoinhibited coiled coil <X-REF>Shekhawat2009</X-REF>.
                            The principle of their approach was that the two segments of a split reporter are linked to the coiled coil dimer forming peptides. Dimerization of the two chains is
                            prevented by the presence of an antiparallel coiled coil segment that inhibits the binding of its partner to other CC peptides. Reconstitution is enabled by the proteolytic
                            cleavage of the linker between the coiled coil fused to the split reporter and the autoinhibitory segment, which dissociates and can therefore be replaced by a second
                            coiled coil forming peptide with the second segment of the split reporter (<REF>4.12.0</REF>).</P><DIV style="margin-left:auto; margin-right:auto;width:70%" align="center"><FIGURE data-ref="4.12.0"><FIGCAPTION><B> Principle of the protease sensor based on autoinhibited coiled coil interactions. </B><P style="text-align:justify">Coiled coil segments can reconstitute the active split
                                        reporter after cleavage of the autoinhibitory segment.
                                    </P></FIGCAPTION></FIGURE></DIV><P style="clear:both">We realized that the same design could be adapted for our orthogonal proteases by replacing the cleavage sites with appropriate protease target motif, such as
                            for the orthogonal proteases PPVp and TEVp.</P></DIV><H1><SPAN class="section colorize"> </SPAN>Results</H1><DIV class="ui segment"><DIV><H3 style="clear:both;"><SPAN id="ini" class="section colorize"> </SPAN>Initial testing based on published coiled coil modules</H3><P>The constructs B:nLuc, cLuc:A, A’:TEVs:B:nLuc and cLuc:A:PPVs:B’2A, which do not possess autoinhibitory segments, were tested for CC binding by measuring luciferase
                                reconstitution. Constructs without protease cleavage sites (B:nLuc, cLuc:A ) were used as a control (<REF>4.12.1.</REF>). A’:TEVs:B:nLuc and cLuc:A:PPVs:B’2A were
                                tested in the presence of TEVp and PPVp, which cleave off the autoinhibitory CC, resulting in split luciferase reconstitution. Additionally, different ratios of
                                constructs were tested in order to obtain the best luciferase activity (<REF>4.12.1.</REF>).</P><DIV style="float:right; width:60%"><FIGURE data-ref="4.12.1."><FIGCAPTION><B> Interactions and protease activated AB coiled coil formation.</B><P style="text-align:justify">HEK293T cells were transfected with appropriate plasmids, 24 h after
                                            transfection cells were lysed and double luciferase assay was performed. (A) B:nLuc and cLuc:A coiled coils constructs fused with split firefly luciferase
                                            system spontaneously interact and reconstitute firefly luciferase. (B) A’:TEVs:B:nLuc and cLuc:A:PPVs:B’2A autoinhibitory CCs reconstitute activity of firefly
                                            luciferase upon cleavage by TEVp and PPV. Successive luciferase reconstitution is observed only when high amounts of both A’:TEVs:B:nLuc and cLuc:A:PPVs:B’2A.
                                        </P></FIGCAPTION></FIGURE></DIV><P>Results showed that very high amounts of the constructs based on same coiled coil sequences used by Shekhawat <I>et al.</I><X-REF>Shekhawat2009</X-REF> (i.e. 50 ng of each)
                                were needed to detect the firefly luciferase signal in mammalian cells (<REF>4.12.1.</REF>). Therefore, we decided to engineer designed coiled coils from a toolbox, used
                                by the <A href="https://2009.igem.org/Team:Slovenia/Orthogonal_coiled-coils.html"> 2009 Slovenian iGEM team </A><X-REF>Gradisar2011a</X-REF>. In order to design an
                                antiparallel coiled coil-based system applicable for logic operation in living cells we took into consideration the rules that establish the orientation and strength
                                of the affinity of the CCs and designed new coiled coils, expanding on the available collection of orthogonal CC and modeled the ratio of the affinities that are required to obtain the optimal response at low
                                leakage <A href="https://2016.igem.org/Team:Slovenia/CoiledCoilInteraction">(Coiled coil interaction model)</A>.</P><DIV class="ui styled fluid accordion" style="clear:both;"><DIV class="title">
                                    Further explanation ...
                                </DIV><DIV class="content"><H3>Coiled coils</H3><P>Alpha-helical segment interaction is a common feature in protein tertiary and quaternary structures, where helices form complexes of two or more
                                        coils (<REF>4.12.1.2</REF>). The most frequent interaction is between two alpha-helices, which form a dimeric coiled coil. Interactions can occur both in the two parallel
                                        or antiparallel orientation of the coil pairs <X-REF>Hadley2006</X-REF>. The interaction strength of different coiled coil pairs depends on their amino acid sequence and
                                        their structure, which determine the underlying noncovalent forces of attraction and repulsion the helices exert on each other. Understanding the rules that govern the
                                        interactions between coiled coils is thus inherently linked to understanding their amino acid sequences <X-REF>Woolfson2005</X-REF>. </P><P>Sequences of coiled coils that form interactions have a characteristic seven amino acid repeat, called heptad repeat. The position of each amino acid
                                        within a heptad is presented in a unified nomenclature (<I>a,b,c,d,e,f,g</I>). Interaction between two coils occurs on a continuous patch along the side of each
                                        alpha-helix with each patch facing the core of the dimer’s interface (<REF>4.12.1.2</REF>B). The amino acid residues which occupy this strip correspond to the
                                        <I>a</I> and <I>d</I> positions of the heptad; they are generally hydrophobic and represent the driving force behind dimerization <X-REF>Woolfson2005</X-REF>. Coiled
                                        coils are additionally stabilized by ionic interactions between polar amino acids (Asp, Glu, His, Lys, Asn, Gln, Arg, Ser or Thr) in positions e and g <X-REF>Woolfson2005,
                                            Gradisar2011a</X-REF>; while amino acids in positions <I>b, c</I> and <I>f</I>, which are less important in interactions between the helices; contribute to helix
                                        stability and solubility. </P><DIV style="float:right; width:80%"><FIGURE data-ref="4.12.1.2"><FIGCAPTION><B> Coiled coil structure and schematic representation of heptad repeats.</B><P style="text-align:justify">(A) Structure of coiled coil. Helical projection representing specific coiled coil interactions in
                                                    (B) parallel and (C) antiparallel orientation
                                                </P></FIGCAPTION></FIGURE></DIV><P>Two alpha-helices that form a coiled coil can interact either in a parallel or in an antiparallel orientation <X-REF>Oakley1998</X-REF>
                                        (<REF>4.12.1.2</REF>B and C). The orientation of coiled coils is largely determined through interactions between amino acid residues in
                                        positions <I>e</I> and <I>g</I><X-REF>Woolfson2005, Oakley1998</X-REF>. In coiled coils with a parallel orientation, electrostatic interactions form between position
                                        <I>g</I> on the first and position <I>e</I> on the second alpha-helix. In coiled coils with an antiparallel orientation, electrostatic interactions occur between
                                        <I>g:g’</I> and <I>e:e’</I> positions of the two helices <X-REF>Litowski2001</X-REF>. The repeating and predictable nature of these interactions can be used for the
                                        rational design of coiled coils <X-REF>Gradisar2011a</X-REF>. Antiparallel CC orientation allows for fusion of C-termini of N-part of split protein to N-termini of
                                        CC via a shorter linker, thereby likely resulting in more efficient reconstitution upon binding with appropriate CC partner.  As represented in the wheel helical
                                        projection in <REF>4.12.1.2</REF> parallel CC are stabilized by electrostatic interactions <I>g:e’</I> and <I>e:g’</I>, while interactions between <I>g:g’</I> and
                                        <I>e:e’</I> positions stabilize antiparallel CC. While CC orientation is mainly influenced by electrostatic interactions specific amino acid residues such as Asn
                                        inside CC core can contribute to the orientation as well. Due to polarity of the Asn residue two asparagines prefer interaction with each other rather than with other
                                        hydrophobic residues in vicinity such as Leu and Ile. These interactions stabilize the core of intended CC orientation and destabilize the core of CC in the opposite
                                        orientation.</P></DIV></DIV></DIV><DIV><H3 style="clear:both;"><SPAN id="ant" class="section colorize"> </SPAN>Antiparallel coiled coils</H3><P>In order to compare the reconstitution efficiency of split protein dictated by parallel or antiparallel coiled coil interaction, we prepared fusion proteins
                                with split firefly luciferase where we designed a new antiparallel peptide (AP4) and tested their activity in cells. Antiparallel coiled coils (AP4:P3) worked
                                significantly better than parallel coiled coils (P4:P3) (<REF>4.12.4</REF>), thus demonstrating that a shorter linker between reporters and dimerizing units
                                helps in the reconstitution of the split protein.</P><DIV style="float:left; width:50%"><FIGURE data-ref="4.12.4"><FIGCAPTION><B> Comparison of the efficiency of the split luciferase reconstitution by parallel and antiparallel coiled coils. </B><P style="text-align:justify">Reconstituted activity of the luciferase dictated by the parallel (left) and antiparallel coiled coil formation (right).
                                            HEK293T cells were transfected with genetic fusions of coiled coil forming peptides and split luciferase. 24 h after transfection luciferase activity was
                                            measured. Coiled coil orientation is represented by coloring of each helix form blue (N-terminus) to red (C-terminus). N and C termini of split luciferase
                                            are represented by N or C, respectively.
                                        </P></FIGCAPTION></FIGURE></DIV><P>To investigate whether the newly designed antiparallel CC is suited for implementation as logic unit into our system, the constructs nLuc:AP4 and P3:cLuc were compared
                                to the coiled coil cLuc:A and B:nLuc from Shekhawat <I>et al.</I><X-REF>Shekhawat2009</X-REF>. Measurement of the reconstituted firefly luciferase activity showed that our
                                designed coiled coils provided far higher (~50 fold) signal (<REF>4.12.5.</REF>), thus proposing the use of this new coiled coils for more sensitive logic gates that
                                functions well in the cellular milieu.</P><DIV style="float:right; width:50%"><FIGURE data-ref="4.12.5."><FIGCAPTION><B> Comparison of the split protein reconstitution based on two different sets of CCs. </B><P style="text-align:justify">HEK293T cells were transfected with different amounts of constructs. (A) Previously reported CCs were tested in different
                                            B:nLuc to cLuc:A ratios. Luciferase reconstitution can be observed at higher plasmid amounts. (B) nLuc:AP4 to P3:cLuc CCs were tested in different ratios.
                                            Luciferase activity was detected even with lower plasmid amounts used. Overall, the comparison between pairs of CCs B:nLuc and cLuc:A to nLuc:AP4 and
                                            P3:cLuc showed that our own CCs give a much higher signal, so lower amounts can be used for integration into our whole system.
                                        </P></FIGCAPTION></FIGURE></DIV><P style="clear:both">The system presented by Shekhawat <I>et al</I> is able to process AND or OR logic functions but not those including negation (such as NOR, NAND etc.) We realized that this type of logic functions could be accomplished by introducing an additional
                                cleavage site between the split reporter and coiled coil segment (<REF>4.12.6.1</REF>).</P></DIV><DIV><H3 style="clear:both;"><SPAN id="cle" class="section colorize"> </SPAN>Cleavable constructs</H3><P>Constructs were therefore modified by the addition of TEVp cleavage site (TEVs) between nLuc and AP4 and PPVp cleavage site (PPVs) between P3 and cLuc. This
                                represents a logic NOR gate based on the input signals, represented by TEVp and PPVp.</P><DIV style="float:left; width:50%"><FIGURE data-ref="4.12.6.1"><FIGCAPTION><B> Introduction of protease cleavage site between the reporter (effector) and coiled coil segment(s) </B><P style="text-align:justify">Cleavage sites in between CCs and reporter protein introduces logical negation.
                                        </P></FIGCAPTION></FIGURE></DIV><DIV style="float:left; width:50%"><FIGURE data-ref="4.12.6."><FIGCAPTION><B> Optimization of protease and substrate plasmid amounts.</B><P style="text-align:justify">HEK293T cells were transfected with plasmids for constructs with introduced TEVs and PPVs (substrates) and different plasmid
                                            amounts of either PPVp (left) or TEVp (right) protease. Results show that 1:5 ratio of substrate and protease, respectively, was needed to achieve adequate
                                            cleavage followed by the decrease in protease activity.
                                        </P></FIGCAPTION></FIGURE></DIV><P>Indeed the system performed nicely (<REF>4.12.6.</REF>). Using this type of cleavage sites enabled us to design protease-based logic gates NOR, NOT A and NOT B (<REF>4.12.7.</REF>).

                                </P><DIV style="float:left; width:100%"><FIGURE data-ref="4.12.7."><FIGCAPTION><B> Design of protease based logic operations NOT, NOT A and NOT B in HEK203 cells.</B><P style="text-align:justify">HEK293 cells were transfected plasmids for nLuc:TEVs:AP4, P3:PPVs:cLuc, nLuc:AP4, P3:cLuc, TEVp and/or PPV as indicated
                                in graphs. 24 h after transfection cells were lysed and double luciferase assay was performed. (A) Logic gate NOR, where the output signal is active only
                                when none of the input signals are present. (B) Logic gate NOT A, in which output signal is active when none or just B input signals (TEVp) is present. (C)
                                Logic gate NOT B, in which the output signal is active when none or just A input signal (PPV) is present.
                            </P></FIGCAPTION></FIGURE></DIV></DIV><DIV><H3 style="clear:both;"><SPAN id="des" class="section colorize"> </SPAN>Destabilized coiled coils</H3><P>For implementation of the system with additional logic operations further modifications on our own CCs collection were needed. Analysis of the equilibrium model reveals that the affinity of the autoinhibitory segment should not be too strong, otherwise the inhibition will remain; but should also not be too weak, otherwise the system would be leaky and active already without cleavage. Stability of the coiled coil interaction can be tuned by introduction of non-favorable interactions <I>e.g.</I> by introducing Ala residues at <I>a</I> and <I>d</I> positions <X-REF>Acharya2002</X-REF>.  We designed four different destabilized P3 coils by substituting <I>b</I> and <I>c</I> position with polar amino acids and <I>a</I> and <I>d</I> positions of different heptads with Ala residues (<REF>4.12.8.</REF>).</P><DIV style="float:left; width:50%"><FIGURE data-ref="4.12.8."><FIGCAPTION><B> Sequence alignment of different coils used to tune the affinity of antiparallel coiled coils.</B><P style="text-align:justify">We designed different destabilized coils from our coiled coil pair AP4 and P3; Ile and Leu were substituted with
                                        Ala at the <I>a</I> and/or <I>d</I> position of the first and/or second heptad of P3mS (a more soluble variant of the original P3).
                                    </P></FIGCAPTION></FIGURE></DIV><DIV style="float:right; width:50%"><SPAN id="autoinhibitory" class="section colorize"> </SPAN><FIGURE data-ref="4.12.9."><FIGCAPTION><B> P3mS-2A and P3mS were the best autoinhibitory coiled coil constructs.</B><P style="text-align:justify">(A) In the presence of TEVp the auto inhibitory coil is cleaved off, allowing P3 to dimerize with AP4 and reconstitute the
                                        split luciferase. (B) Normalized luciferase activity was compared between samples with and without added TEVp to calculate the fold change (black dots) of luciferase
                                        activity. Out of the four different constructs, the constructs which contained the inhibitory coils P3mS and P3mS-2A worked best, where we observed up to
                                        15 times fold increase with the addition of TEVp.
                                    </P></FIGCAPTION></FIGURE></DIV><P>Those variably destabilized peptides were used as autoinhibitory coiled coil forming segments to test the difference in activity between the uncleaved and TEVp cleaved forms.</P><P>To test which one of our four destabilized CCs worked best, all constructs were tested <I>in vivo</I> with and without the presence of TEVp (<REF>4.12.9.</REF>). We concluded that P3mS and P3mS-2A demonstrated the highest fold increase in the luciferase activity upon the addition of TEVp. The other two constructs showed little to no increase in luciferase activity upon the addition of TEVp, suggesting that the peptides were destabilized too much leading to the leakage in the uninduced form.</P></DIV><DIV><H3 style="clear:both;"><SPAN id="ind" class="section colorize"> </SPAN>Inducible logic</H3><P style="clear:both">The final test was to investigate if the system could indeed be controlled by two signals at the same time. In order to test this we constructed NOR gate with logic
                            processing (nLuc:TEVs:AP4 and P3:PPVs:cLuc) and inducible components (split PPVp and TEVp inducible by the rapamycin and light, respectively) (<REF>4.12.13.</REF>).</P><DIV style="float:left; width:50%"><FIGURE data-ref="4.12.13."><FIGCAPTION><B> Protease-based NOR logic gate regulated by light and rapamycin. </B><P style="text-align:justify">HEK293 cells were transfected with appropriate plasmids as indicated in the graph. 24 hours after the transfection the
                                        cells were induced with light and rapamycin for 15 min and after 4 hours, lysed and double luciferase assay was performed.
                                    </P></FIGCAPTION></FIGURE></DIV><DIV style="float:right; width:50%"><FIGURE data-ref="4.12.11."><FIGCAPTION><B> Logic gates A and A nimply B regulated by light (input A) and rapamycin (input B) based on protease processing produce correct and
                                    measurable output after 15 minutes. </B><P style="text-align:justify">HEK293 cells were transfected with appropriate plasmids as indicated in graph. 24 hours after the transfection the cells
                                        were induced with light and rapamycin for 15 minutes, lysed and double luciferase assay was performed. (A) Logic gate A, in which output signal has a
                                        value of 1 when A input signal (TEVp induced with light) is present. (B) Logic gate A, in which output signal has a value of 1 when only A input signal
                                        (TEVp induced with light) is present.
                                    </P></FIGCAPTION></FIGURE></DIV><P>The types building blocks that we developed made possible to construct all of the possible 16 two-input logic function based on the proteolysis.
                        To demonstrate this we tested additional logic operations (A and A nimply B), where the response was measured directly after 15 minutes of induction to demonstrate
                        the increased speed of protein-based processing, as shown on <REF>4.12.11.</REF>.</P><P style="clear:both">In both cases the system performed very well, producing clear difference between the active and inactive output states within 15 min after the stimulation by combinations of two signals. Logic function A nimply B is relatively difficult to implement but on the other hand it can be quite useful as for example the signal B may identify the cell type and trigger  activation by an external signal only in a selected cell types or cells in a selected state.</P><P style="clear:both">The availability of a set of <A href="https://2016.igem.org/Team:Slovenia/Protease_signaling/Orthogonality">orthogonal proteases</A> as well as an
                            <A href="https://2009.igem.org/Team:Slovenia/Orthogonal_coiled-coils.html"> orthogonal coiled coil dimers toolset</A> enabled the construction of a fast complex logic processing
                            circuits.  The previous CC toolbox has been further expanded with a strategy of generating antiparallel and destabilized CC. Furthermore, designed system based on split proteases can also be linked to many other input signals such as
                            <A href="https://2016.igem.org/Team:Slovenia/Mechanosensing/CaDependent_mediator"> intracellular calcium increase</A>. An important advance is the adaptation of the system to function <I>in vivo</I> in mammalian cells. Further, reporter as the output
                            signal could be substituted by a split protease, enabling multi-layered processing, or used as an trigger for other cellular processes, such as the
                            <A href="https://2016.igem.org/Team:Slovenia/Implementation/ProteaseInducible_secretion"> release of therapeutics</A>. Therefore, we believe that those results represent a valuable foundational advance in synthetic biology.</P></DIV></DIV><H3 class="ui left dividing header"><SPAN id="ref-title" class="section colorize"> </SPAN>References
                </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>