"
Page
Discussion
View source
History
teams
Log in
 
Team:Northwestern/detectpH
From 2013.igem.org
Team
Official Profile
Meet us!
Attribution
Project
Project
Parts Submitted
Methods
Modeling
Results: Buffers
Results: Fluorescence
Notebook
Notebook
References
Outreach
Human Practice
Safety
Safety
Detection of pH Levels
Why use a pH-inducible promoter?
In order to execute the alkalinity response only when it is needed,we required a promoter that would be activated at fairly low pH. It was first necessary to identify said pH-inducible promoters that are active at or near pH 5.5 At pH 5.5 or below, the rate of demineralization of the tooth exceeds that of the re-mineralization process provided by saliva. This results in erosion of the hard tissues of the tooth. Thus, a promoter induced at pH 5.5 provides useful transcriptional control over genes that might prevent the progression of tooth decay.
Identifying a pH-inducible promoter
The E. coli genome contains genes which demonstrate elevated levels of transcription at or near pH 5.5. Tucker, et al. performed a comprehensive study that identified acid-inducible genes contained within the E. coli genome. Our team compared the expression levels of genes in cells grown at pH 5.5 to cells grown in pH 7.4. They found that the transcription of the asr and gadA genes were considerably induced at pH 5.5. In fact, the asr gene was the most significantly pH-induced gene identified. Our team proceeded with isolating the promoters of the asr and gadA genes to be used as the pH-inducible promoters driving elevated levels of gene expression within our dual-state promoter.
GadA is a crucial component in the acid stress response of E. coli
The gadA gene is a part of the gad system, which is an acid-inducible glutamate decarboxylase-based acid resistance system that enables the survival of E. coli under acid stress conditions. GadA is a gene encoding a glutamate decarboxylase. The process of decarboxylating glutamate consumes protons that leach into the cell under acid stress. In this manner, the gad system manages protons that would otherwise drop the cellular pH below levels at which E. coli could survive.
The function of the gad system explains why these genes experience high transcription levels at low pH. Despite this functional explanation, modes of transcriptional control over the gad system are rather complex. Transcriptional factors RpoS, cyclic AMP receptor protein, HN-S and EvgA all play a role in transcriptional regulation. Transcriptional regulators GadW and GadX also affect induction of the gad system through intricate interactions both with the GadA promoter region, as well as with one another. Due to this complexity, our strategy was to characterize the gadA promoter empirically.
Transcriptional regulation of asr promoter
As of yet, the function of the asr gene, or “acid-shock RNA” gene, and the mechanism responsible for its induction are still unclear. However, Iien et al. have taken significant steps toward characterizing the gene. They propose that asr encodes a periplasmic or outer-membrane protein. Knockout experiments illustrated that the PhoBR operon plays a significant role in activating the asr gene. They demonstrated through mobility shift electrophoresis that the PhoB protein binds to the promoter region of asr.  By analyzing the sequence of the asr promoter region, they revealed that it contains a sequence similar to that of the Pho box, which is a consensus sequence known to bind the PhoB protein. The Pho box can be found in the promoter regions of other PhoB-regulated genes. This evidence suggests that the regulatory protein PhoB indeed exerts some transcriptional control over the asr gene.
Identification of gadA and asr promoter regions
In order to choose the correct sequence of the promoters, we examined both the start codon of the promoters, the base pairs AUG, and the Shine Dalgarno sequence within the promoters. For a ph-induced promoter to be combined with a constitutive promoter in a dual state, the promoter sequence could not contain the ribosomal binding site that exists before the start codon. This is called the Shine Dalgarno sequence, the collection of base pairs in the promoter before the start codon (about 8 base pairs upstream) where the ribosome will bind. If this was not excluded, then the ribosome will bind after the pH-induced promoter and would translate the spacer, constitute promoter and the gene instead of translating just the gene.  With that in mind the promoter sequences for both asr and gadA were selected.
The GadA promoter sequence was taken from the 2010 Madison-Wisconsin iGEM Team. It was adjusted to be used with the RBS and then also selected to exclude the RBS and end the sequence before the Shine Dalgarno sequence.
When working with the asr promoter, there ended up being two different lengths that we decided to move forward with. The start codon AUG, repeats itself and there is no  research that confirms which start codon is the true start codon. According to Šeputienė et al. the third AUG is the start codon as there is an Shine Dalgarno sequence preceding it and mutation in this codon leads to an stop in the translation of the asr gene. Off of this paper we formed an asr promoter which we labeled “asr long.”  However, the start codon is generally at +1 (instead of +21 for asr long) and because of this we formed an “Asr short” which had the sequence up to the Shine Dalgarno before the first AUG. Thus we had two different length promoters to account for the lack of knowledge of the start codon for the asr gene.
Retrieved from " http://2013.igem.org/Team:Northwestern/detectpH "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
