<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Northwestern_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Northwestern/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Northwestern/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Northwestern/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Northwestern/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><NAV id="main-nav"><UL id="nav-primary"><LI><A href="https://2013.igem.org/Team:Northwestern/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Northwestern">Official Profile</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Team">Meet us!</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Attributions">Attribution</A></LI></UL><LI><A href="https://2013.igem.org/Team:Northwestern/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Parts">Parts Submitted</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/methods"> Methods</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Results/PreFluorescence">Results: Buffers</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Results/Fluorescence">Results: Fluorescence </A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/biblio">References</A></LI><LI><A href="#"> Outreach </A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Human_Practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Northwestern/Safety">Safety</A></LI></NAV><H1> Team Notebook </H1><H3> Week 1 </H3><DIV id="week"><DIV class="day"><B>Day 1, June 17</B></DIV><P>Each team member prepared 5 ideas, and the team selected the top five ideas. These ideas include a biosensor for common parasites in water, a biosensor for UV damage, a bacteria that fights cavity, bacteria to produce diesel from food oil, and a composite biosensor that incorporates several biosensors. </P><DIV class="day"><B>Day 2, June 18</B></DIV><P>Each member focused on one idea. The goal for the day was to determine a possible mechanism. The goal for the day was to determine the feasibility of the idea. The team also considered several new ideas such as MSG detection, urine detection, and biofertilizer.</P><DIV class="day"><B> Day 3, June 19 </B></DIV><P>This day was a continuation of the previous day. Emphasis today was to research previous iGEM teams for inspiration and possible mechanisms and biobricks.</P><DIV class="day"><B> Day 4, June 20 </B></DIV><P>We tried to find one more idea that will be feasible. After meeting with advisors, we tried to focus more on the mechanistic details rather than a high level discussion of the project idea. We also took an inventory of the lab space and began sending out sponsorship emails.</P><DIV class="day"><B> Day 5, June 21 </B></DIV><P>After deciding most of the ideas we have come up with are far from feasible, we focused again on previous iGEM teams, hoping to build on a previous team's work. Each team member was tasked with coming up with a new idea for the next week.</P></DIV><H3> Week 2 </H3><DIV id="week"><DIV class="day"><B> Day 6, June 24 </B></DIV><P>The team decided on roles to improve efficiency and team organization. The team made SOB solution and obtained Top 10 cells from Dr. Leonard (one of our advisors). The team also talked quorum sensing of bacteria as a possible mechanism to detect parasites in water. </P><DIV class="day"><B> Day 7, June 25 </B></DIV><P>The team discovered that UV sensing is already done several times by different iGEM teams. The idea may lack novelty. To improve on the biofuel idea, different starting materials are considered, hoping to increase the financial sensibility by decreasing the value of the starting materials. </P><DIV class="day"><B> Day 8, June 26 </B></DIV><P>Today, the team focused primarily on oral health and tried to come up with several ways to attack biofilm formation in the mouth. The ideas that were discussed include engineering V. Parvula to upregulate lactic acid metabolism, introducing the CovR pathway to disrupt biofilm formation, and using the L-Arginine pathway to produce ammonia. The team also created CCMB solution. </P><DIV class="day"><B> Day 9, June 27 </B></DIV><P>The team learned how to make plates (created 40) using CB antibiotic. The team continued to talk about the three ideas, trying to discover anything that will make the project feasible. The primary hurdle is the mechanism to import lactic acid from outside the cell. </P><DIV class="day"><B> Day 10, June 28 </B></DIV><P>The transformation failed, and no cells grew on the plates. The professors pointed out that the upregulation of lactic acid uptake would disrupt the redox balance within the cell, causing the key enzyme lactate dehydrogenase to work in reverse. However, the team has decided that the project for the summer will involve oral health. </P></DIV><H3> Week 3 </H3><DIV id="week"><DIV class="day"><B> Day 11, July 1 </B></DIV><P>The team redid the transformation experiment again. This time, all glassware was washed to make sure they are free of detergent. The team prepared for the meeting with professors by trying to flesh out the idea as much as possible. </P><DIV class="day"><B> Day 12, July 2 </B></DIV><P>The cells did not grow once again. The team divided lab space in preparation of the next phase of the project where each member will have more independence. The team is focusing on pH sensor since that appears to be the common mechanism in all of the ideas that were brought up. To add an extra layer of complexity, the team will incorporate the idea of a &quot;dual promoter&quot; into the design. </P><DIV class="day"><B> Day 13, July 3 </B></DIV><P>The team redid the transformation once again. Most of the materials used is store bought to eliminate any possible errors. Prior to leaving for the break, the team is brainstorming on a list of possible mechanisms for the pH sensor. </P><DIV class="day"><B> Day 14, July 4 </B></DIV><P>The team came in just to check for the results of the transformation. Yet again, no cells grew. </P><DIV class="day"><B> Day 15, July 5 </B></DIV><P>July 4th weekend holiday! No meetings. </P></DIV><H3> Week 4 </H3><DIV id="week"><DIV class="day"><B> Day 16, July 8 </B></DIV><P>Over the long weekend, we spent time identifying genes that were activated at high levels due to a pH shift. We focused on the asr (acid-shock rna) gene and the gadA genes. Their promoters should be pH-inducible. Moreover, a past iGem team from Wisconsin-Madison has already isolated the gadA promoter.   

</P><P>Over the long weekend, no growth was shown on the 7/3 transformation, indicating that our SOB was not the problem. Three lab members independently performed a transformation to test competency of cells prepared on 7/2 using a pUC19 plasmid. The transformations were tested on both Cb and Lb plates. Additionally, our TA Jessica independently ran a transformation testing our cells, plates, and SOB. </P><DIV class="day"><B> Day 17, July 9 </B></DIV><P>Success! Our TA has indicated that our cells are competent comparable with her Dr. Jewett’s lab. She realized that the plates we thought were Amp were actually Cm, and this could be the reason why some of our transformations are failing. Our SOB and cells are fine. We also had an advisor’s meeting, where they suggested we use a low-copy plasmid for our construct, create a GANTT chart before next week, assign people to become specialists in each protocol we will be doing over the summer, and to practice these protocols over the next week or so. 

Afterward, Brendan showed the lab how to make a gel, and he and Nirmal performed a transformation on the correct Amp plates using both the pUC19 and pL plasmids. Viral inoculated the cells Jessica transformed, so we can practice a miniprep tomorrow. </P><DIV class="day"><B> Day 18, July 10 </B></DIV><P>We finally had a successful transformation! The cells containing pL grew on the Amp plates. The pUC19 cells did not grow, indicating that the plasmid, in conjunction with the mislabeled plates, was the source of our transformation errors.  We then practiced a miniprep, gel electrophoresis, and subsequent gel extraction. </P><DIV class="day"><B> Day 19, July 11 </B></DIV><P>We spent the entire day in the library, coming up with a schematic for how to construct our dual-state promoters. We struggled particularly with designing how to add spacers so that we need as few primers as possible to save money. Ultimately we decided to have one 50bp sequence with the same restriction enzyme site on both ends. We would add this 50bp sequence repeatedly to create 100bp and 150bp spacers. 

In the afternoon, we learned how to design primers. We designed a primer together, and then decided that we would go home and design primers on our own and compare the next day. This should help eliminate errors. </P><DIV class="day"><B> Day 20, July 12 </B></DIV><P>We had a meeting with Jessica for feedback on our schematic. She saw a problem with having the same RES on both ends of the spacer, because then the spacer could be added in either direction, and our sequences would not be consistent. Moreover, the sequence is very short, which will be difficult to insert into a plasmid in terms of efficiency. 

We realized overnight that there was an RES in the middle of the constitutive promoter-RBS-GFP construct we were going to use from Jessica. So we re-designed the schematic to work around this RES using site-directed mutagenesis. </P></DIV><H3> Week 5 </H3><DIV id="week"><DIV class="day"><B> Day 21, July 15 </B></DIV><P>We spent the first half of the day trouble-shooting on our design schematic, confirming the proper sequences of our parts, and then designing promoters one by one together. </P><DIV class="day"><B> Day 22, July 16 </B></DIV><P>We prepared for our meeting with our advisors by drawing a final schematic, and compiling a list of primers. Our advisors informed us that to simplify our design,we should construct primers with two restriction sites. This way we can just use the Biobrick methodology to construct our dual-state promoters. We set to work re-doing our primers in this manner in the afternoon. We also met with a representative from IDT, who was very helpful in providing us with $250 worth or free oligos, as well as t-shirts, lab notebooks, and nuclease-free water. </P><DIV class="day"><B> Day 23, July 17 </B></DIV><P>Jessica informed us that we should be checking the temperatures and GC contents of the portion of the primer that anneals to our target. Before this, we had been checking and adjusting the temperatures of the entire strand. This means we need to re-do all of our primers. </P><DIV class="day"><B> Day 24, July 18 </B></DIV><P>We performed a transformation with pSB4A1 from the kit (Kit Plate 5, Well 21D) to practice transforming a plasmid. Past iGem teams have had trouble properly extracting the DNA from the kit. </P><DIV class="day"><B> Day 25, July 19 </B></DIV><P>The 7/18 transformation did not work. So we ran the same plasmid alongside a positive control using pL plasmid to check our technique. </P></DIV><H3> Week 6 </H3><DIV id="week"><DIV class="day"><B> Day 26, July 22 </B></DIV><P>No Lab Today!</P><DIV class="day"><B> Day 27, July 23 </B></DIV><P>We ran into budget problems today as the primers cost more than the amount of donation we have. We also researched more to change pH in plate media through buffer. Lastly, we worked more on human practices by researching Dr. Zoloth's area of expertise. </P><DIV class="day"><B> Day 28, July 24 </B></DIV><P>We redid the transformation once again. We have controls to make sure that both the plasmids and the cells are good. In addition, we developed a rough draft of a powerpoint for human practices and looked up how to do PCR. We also ordered primers.</P><DIV class="day"><B> Day 29, July 25 </B></DIV><P>The primers arrived today, and we will PCR tomorrow. We also inoculated a colony today. We looked at how to make M9 (minimal media), but we were missing ammonium chloride. This will be postponed until Monday. The transformations worked well, proving again that our cells are competent. Two plasmids from the kit failed, but both plasmids obtained from our Dr. Leonard's lab worked well. </P><DIV class="day"><B> Day 30, July 26 </B></DIV><P> Today we started PCR! We suspended the primers in nuclease free water. We only performed PCR on ASR-RSB. We performed the assay under the guidance of our TA. In the afternoon, we tried PCR purification with last year's kit (our kits will arrive soon!), but the yields were low. </P></DIV><H3> Week 7 </H3><DIV id="week"><DIV class="day"><B> Day 31, July 29 </B></DIV><P> Since our yields were very low, we decided to perform another PCR. This time, we went ahead and used all of the primers we ordered. Afterwards, we purified the DNA with our new kits from Epoch, and the yields were acceptable. Today, we also tried to digest the PCR products from Friday, but due to faint bands, gel purification was aborted. We also transformed cells! </P><DIV class="day"><B> Day 32, July 30 </B></DIV><P> Today, we minipreped pSC101 and ended with acceptable yield. We also prepared digestion on all of the PCR products from yesterday. Lastly, we inoculated the transformed cells in preparation of the miniprep today. </P><DIV class="day"><B> Day 33, July 31 </B></DIV><P> We finished digestion that we started yesterday and started ligation. We have two protocols for ligation (one short and one long), so we will prepare for the longer ligation today. In addition, we minipreped pSB4A5 and got  surprisingly high concentration of DNA. Other lab work includes making plates and transforming new cells. </P><DIV class="day"><B> Day 34, August 1 </B></DIV><P> The team finished the ligation protocol and transformed our cells with the products. We will see the results tomorrow. In addition, we started a digestion and ligation cycle of the same parts (ASR-RBS, LPP-RBS, GadA-RBS) today. These are duplicates of the ligation we performed today. We will complete the protocol tomorrow. Lastly, we PCRed all of the primers we have (including new primers that just came in today). </P><DIV class="day"><B> Day 34, August 2 </B></DIV><P> The cells grew! This means the ligation worked since our negative control worked relatively well. We also minipreped these cells to prepare for sequencing. The duplicate ligation prtocol is also completed along with the transformation. The PCR products are also PCR purified. </P></DIV><H3> Week 8 </H3><DIV id="week"><DIV class="day"><B> Day 35, August 4 </B></DIV><P> We completed digestions, ligations, and 12-hr transformations from the weekend. Counted cells for the Lpp-RBS ligations and compared them with the negative control’s. Performed inoculations on 3 colonies from the plate with best ratios, which were 6:1 asr-RBS, 6:1 gadA-RBS, and 1:1 Lpp-RBS. </P><DIV class="day"><B> Day 36, August 5 </B></DIV><P> We miniprepped the inoculations from 8/4. Jessica discussed with us tips to increasing miniprep yields and concentration requirements for sequencing. Samson and Simon completed digestion and gel purification of PCR products from 8/1. Kevin made different acids and buffers for fluorescence assays. Simon made media to make AMP plates.
</P><DIV class="day"><B> Day 37, August 6 </B></DIV><P> Performed ligations on the PCR products that were digested and purified yesterday. These include gadA RBS, asr RBS, GFP, Tac RBS, Lpp RBS, asr short, asr long, and gadA. At advisor’s meeting, Prof. Tyo suggested 30ml inoculations and miniprepping to cope with our low copy plasmid.
 </P><DIV class="day"><B> Day 38, August 7 </B></DIV><P> Transformed ligations from yesterday. We tried doing 35mL inoculations and miniprepping with no avail because the we misused the rotor to spin overnight tubes. We reinoculated the failed ones. Made more SOB and inoculated Top10 cells to make more competent cells.</P><DIV class="day"><B> Day 39, August 10 </B></DIV><P> Samson and Simon digested, digest purified, and ligated cycle 3 redo (from 8/1).  </P></DIV><H3> Week 9 </H3><DIV id="week"><DIV class="day"><B> Day 40, August 11 </B></DIV><P> Miniprepped transformations from 8/10. </P><DIV class="day"><B> Day 41, August 12 </B></DIV><P> Performed diagnostics digests on minipreps from 8/12 (Asr-RBS and GadA-RBS and Lpp-RBS).</P><DIV class="day"><B> Day 42, August 13 </B></DIV><P> Redid diagnostic digests of Asr-RBS and Lpp-RBS. PCRed Asr-Short and Asr-Long from genomic DNA. </P><DIV class="day"><B> Day 43, August 14 </B></DIV><P> Checked for Asr-Short and Asr-Long on gel. Asr-RBS and Lpp-RBS diagnostic digests were redone again. Ligated and transformed Asr-Short and Asr-Long into pSB4A5. </P><DIV class="day"><B> Day 44, August 15 </B></DIV><P> PCRed GFP from iGEM biobrick parts. Performed diagnostic digests on Asr-Short and Asr-Long constructs that were placed in pSB4A5. Also performed diagnostic digest on Tac-RBS and GadA constructs in pSB4A5. Digested GFP.  </P><DIV class="day"><B> Day 45, August 16 </B></DIV><P> Performed yet another diagnostic digest on the same constructs listed above. Ligated and transformed GFP digests into pSB4A5. </P><DIV class="day"><B> Day 46, August 17 </B></DIV><P> Inoculated many cells overnight that contained several different constructs. </P></DIV><H3> Week 10 </H3><DIV id="week"><DIV class="day"><B> Day 47, August 19 </B></DIV><P> We miniprepped 3 overnight cultures of GFP and digested GFP (1, 2, 3) with X and P, Tac-RBS (1, 2, 3), Lpp-RBS (1, 2, 3) with S and P as well as X and P. We also worked on figuring out the Primer Annealing and Extensions. Afterwards, we ran gels on the digested products and performed PCR purification. </P><DIV class="day"><B> Day 48, August 20 </B></DIV><P>We stopped digesting on Tac-RBS and Lpp-RBS. Samson performed gel purification on GFP digests from 8/19. Viral ordered primers fro 128 bp spacer sequence. Nirmal and Samson ran confirmation gels on the purified digest after PCR of Tac-RBS as well as the digest after inserting into plasmid of Tac-RBS.Brendan performed transformation of ligations of GadA-RBS+GFP and ASR-RBS+GFP. 
</P><DIV class="day"><B> Day 49, August 21 </B></DIV><P> Ligations of ASR-RBS + GFP and GadA-RBS + GFP worked! We miniprepped the overnight cultures of GadA - 9 samples. We digested 64 bp spacer (8/19 45C) and pSB4A5 (8/9) with E and P. Ran a 60 mL 1% gel. We digested Lpp-RBS with S and P and PCR purified. We inoculated Asr-RBS2+GFP2 and GadA-RBS2+GFP2 in SOB and on amp plates.

 </P><DIV class="day"><B> Day 50, August 22 </B></DIV><P> We gel purified 64 bp spacer and pSB4A5 from yesterday’s digest. We then ligated the 64 bp spacer into pSB4A5, with 6:1 and 3:1 ratio. He also transformed. We prepared GadA-RBS, ASR-RBS, LPP-RBS, ASR-short, ASR-long, GadA, and GFP for sequencing. We ran PCR product of 128bp spacer on gel. Then she PCR purified. Samson re-made the M9 media, this time without cassamino acids and with thiamine. </P><DIV class="day"><B> Day 51, August 23 </B></DIV><P> Transformations from yesterday (64 bp spacer into pSB4A5 in 3:1 and 6:1 ratios) did not work. We believe that we did not deactivate at the appropriate temperature. Viral began making competent cells. We ligated 128 bp spacer into pSB4A5. We digested 64 bp spacer and GFP 1, 2, 3 again. Gel purified both digests. We digested pSB1C3 with E and P. Made digestion master mix.
 </P></DIV><H3> Week 11 </H3><DIV id="week"><DIV class="day"><B> Day 52, August 26 </B></DIV><P>We PCR purified the pSB1C3 digest from 8/23 and eluting only in 30 uL. We miniprepped weekend inoculations of Asr-RBS+GFP, GadA-RBS+GFP, Lpp-RBS+GFP, Tac-RBS, 128bp spacer, pSB4A5. We ligated Asr-RBS to pSB1C3, and 64bp (40C, 45C, 50C) to pSB4A5. We inoculated  ASR-RBS 1-3, Lpp-RBS 1-5, GadA-RBS 1-3, postive control (GFP plate from jessica), negative control ( just GFP, no RBS), and negative control (no cells). We transformed ligations into cells. We miniprepped pSB4A5. </P><DIV class="day"><B> Day 53, August 27 </B></DIV><P> Our ligations of Asr-RBS + pSB1C3 and 64bp (40C, 45C, 50C) + pSB4A5 looked good in terms of ratios. Diagnostic digests on Tac-RBS, Lpp-RBS+GFP, and 128bp failed. Samson’s inoculations in M9 media did not work. Simon made more Cb (AKA amp) plates. Sequencing results came back. Everything besides Tac-RBS worked! Inoculated Asr-RBS in pSB1C3 and 64bp spacer in pSB4A5 overnight. We started 128bp spacer over from the PCR step.

</P><DIV class="day"><B> Day 54, August 28 </B></DIV><P> Brendan miniprepped Asr-RBS in pSB1C3. Simon miniprepped 64bp spacer in pSB4A. 5. Samson made Knight M9 media, replacing glucose with glycerol and Coldspring Harbor M9 media. Used Jewett lab material for both media.Samson inoculated LRG , ARG, GRG,Tac (positive), GFP (negative), swirl (negative), nothing (negative) in Knight M9, Coldspring M9, and SOB. Samson and Brendan digested of 64bp. Viral digest LR3 and GFP1 (both sequence confirmed) and ran on gel. Viral also an transformation, testing competency of her competent cells made on 8/26. Simon tested his Cb plates.

 </P><DIV class="day"><B> Day 55, August 29 </B></DIV><P> Viral’s competent cells didn’t work. Samson’s Knight lab M9 media worked, but Coldsprings didn’t. Simon’s Cb plates worked. Brendan ran the diagnostic digests (again) of 128bp, 64bp, and LRG. 128bp seemed to work, but the other two didn’t. He also inoculated 10 more LRG, in addition to the 5 we already have, for sequencing. We digested GadA with E and S, 64bp with E and X, 128bp with E and X. We redigested LR and G and ligated them. We also ligated the LR and G digests from before. Samson performed fluorescence assays.
</P><DIV class="day"><B> Day 56, August 30 </B></DIV><P> We purified digests, ligated, and transformed 64bp &amp; 128bp. We also performed maxipreps. Samson’s redigestion didn’t work. Transformations of our ligations didn’t have much difference between positive and negative.  Samson worked on fluorescence assays.  </P></DIV><H3> Week 12 </H3><DIV id="week"><DIV class="day"><B> Day 57, September 3 </B></DIV><P> We thought the reason for failing digestions was the water. Samson redigested GadA (2, 5, 4) and also digested Asr (short) 2, Asr (long) 3  with old water. We redigested GadA (2, 5, 4) and Asr (short) 2, Asr (long) 3 with new water. E and S were used for the digestions.
 </P><DIV class="day"><B> Day 58, September 4 </B></DIV><P>We redigesting 64bp (1, 2, 3) and 128bp (1, 2, 3) with E and X with new water. We ran GadA and Asr (short/long) digests from yesterday on gel but results failed. We digested GadA-RBS with different combinations of old and new restriction enzymes. Samson redid the fluorescence assay. Sequences of LR1a, AS2, AL2, LR2a, and TR2 worked!

</P><DIV class="day"><B> Day 59, September 5 </B></DIV><P>We ran diagnostic gels on the digestions from yesterday but did not work. We made new TBE buffer. We redigested with 1000ng instead of 500ng. Digestions on GadA-RBS worked! Samson inoculated 3 colonies of Asr-RBS in pSB1C3.  </P><DIV class="day"><B> Day 60, September 6 </B></DIV><P> We gel purified Asr Short 2, Asr Long 2, GadA 4 digests with E and S from yesterday and maxiprepped Asr-RBS + pSB1C3 inoculations. We digested 64bp and 128bp with E and X and PCR purified.

</P></DIV><H3> Week 13 </H3><DIV id="week"><DIV class="day"><B> Day 61, September 9 </B></DIV><P> We digested LRG with E and X. We made more Cm plates. We PCRed GadA4, Asr Short 2, and Asr Long 2 and purified. We ligated Asr short 2 to 64bp and 128bp and transformed. Also, we ligated Asr long 2, GadA4 to 64bp and 128bp and transformed. We used newly made M9 for inoculation. 
 </P><DIV class="day"><B> Day 62, September 10 </B></DIV><P>We digested more pSB1C3 with E and P. We maxiprepped Asr-RBS+pSB1C3 for sequencing. We ligated Asr-RBS, GadA-RBS, Lpp-RBS (all from 8/1) to pSB1C3 (8/26) and transformed. Weperforming diagnostic PCR on Asr long 2, Asr short 2, GadA 4 + 64bp/128bp. We worked on fluorescence assays. 


</P><DIV class="day"><B> Day 63, September 11 </B></DIV><P>We sent in Asr-RBS+pSB1C3 maxipreps from yesterday to sequence. We gel purifying pSB1C3 that were cut with E and P. We performed diagnostic PCR of the AL, AS, and G + 64/128. Chose the ones that worked and Brendan miniprepped them. Some worked. AS-128 and AL-128 both didn’t work at all. We ran re-diagnostic PCR on other AS-128s and AL-128s. We redigested pSB1C3 with E and P using CIP to de-phosporylate the ends of the pSB1C3 so they don’t re-ligate to the insert. We performed diagnostic PCR on AR, GR, LR with pSB1C3. Inconclusive. </P><DIV class="day"><B> Day 64, September 12 </B></DIV><P> We nanodropped minipreps from yesterday. We sequence confirmed that ASR-RBS + pSB1C3 3 from 9/6 (sent on 9/10). We digested. We purified phosphotased digests of pSB1C3 (E + P). 

</P><DIV class="day"><B> Day 65, September 13 </B></DIV><P> We redigested AS2-64, AL2-64, G4-63, GR-128. We colony PCRed more AS2-128, AL2-128. Didn’t seem to work. We submitted AR-pSB1C3 2 to iGEM!</P></DIV><H3> Week 14 </H3><DIV id="week"><DIV class="day"><B> Day 66, September 16 </B></DIV><P> We maxipreped weekend cultures of AS2-64, AS2-128, AL2-64, AL2-128, G4-64, G4-128. We purified, ligated, and transformed AS2-64, AL2-64, G4-64, and G4-128 to LRG. We checked sequences of GR 1, 2, 4 and LR 1, 3, 5 that were in pSB1C3. We digested minipreps from above with E and X, digesting LRG 8 (8/30) with E and S.
 </P><DIV class="day"><B> Day 67, September 17 </B></DIV><P>We PCR purified products digested with E and X. We gel-purified products digested with E and S
</P><DIV class="day"><B> Day 68, September 18 </B></DIV><P>We colony PCRed all the dual-state constructs transformed yesterday: (AS2-64)8-LRG 4, (G4-128)2-LRG 1, (AS2-64)2-LRG 2, (AS2-64)8-LRG 3, (AS2-64)-LRG 1, (G4-128)2-LRG 2. We ligated and transformed LRG to phosphatased AS-128, G4-64, GR-128</P><DIV class="day"><B> Day 69, September 19 </B></DIV><P> Miniprepped inoculations from yesterday. Digested LRG, AS, AL, G for ligation and transformation tomorrow with no spacer. Colony PCRed transformations from yesterday to see if dual-states worked. None worked.

</P><DIV class="day"><B> Day 70, September 20 </B></DIV><P> Digest purified Asr Long 3, Asr Short 2, GadA4, LRG8 1,2,3.-Ligating and transforming AL3-LRG, AL2-LRG, G4-LRG, (G4-64)(2,4,7)-LRG, (AL2-64)8-LRG, (AL2-128)4-LRG, (AS2-128)4-LRG. Digested pSB1C3 with E and P. and ran it on gel. </P><DIV class="day"><B> Day 71, September 21 </B></DIV><P> Gel purified yesterday’s digests
</P></DIV>



You should make use of the calendar feature on the wiki and start a lab notebook.  This may be looked at by the judges to see how your work progressed throughout the summer.  It is a very useful organizational tool as well.
<DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Northwestern/Notebook">http://2013.igem.org/Team:Northwestern/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Northwestern/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Northwestern/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;oldid=242292" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>