"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Amsterdam/project/background/
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
Project Background Information
Contents
1 Project Background Information
1.1 Summary of the Mechanism
1.2 Methylation of Restriction Sites
1.3 Zinc Fingers
1.4 Gel Electrophoresis
Summary of the Mechanism
The Cellular Logbook is build on the natural occurring epigenetic phenomenon of methylation. Each sensor is able to expres a zinc finger and methyltransferase fusion protein (ZnF-Mtase). The ZnF-Mtase binds to its Memory Part (MP), which consists of a restriction site that is targeted by the methyltransferase, and two binding sites that are specific to the zinc finger. Upon binding, the methyltransferase will methylate the restriction site. The methylated restriction site is inaccessible to restriction enzymes. By digesting the Memory Part with the restriction enzyme and performing gel electrophoresis, it can be read out if the ZnF-Mtase has come to expression and as such if the related signal has been present.
Methylation of Restriction Sites
DNA methylation consists of the addition of a methyl group to cytosine or adenine. In bacteria DNA methylation is part of the restriction modification system. Specific DNA sequences are targeted by a methyltransferase to be methylated. A good number of methyltransferases target restriction sites, resulting in the prevention of restriction enzymes cutting a restriction site when it is methylated.
By taking a  promoter and replacing its gene with our methyltransferase gene, the methyltransferase will be expressed upon the activation of the promoter. The methyltransferase will then methylate its target sequence. Given a methyltransferase that targets a restriction site, we are able to read out if a promoter was active or not by use of restriction digestion. If the promoter has been active, the methyltransferase will be activated, and the restriction site is inhibited by a methyl group preventing it from being cut by a restriction enzyme. However, if the promoter has not been active, the restriction site is not inhibited by a methyl group and the plasmid will be cut by a restriction enzyme. Knowing this we are able to measure if a signal in front of a promoter has been present or not using digestion and gel electrophoresis.
Zinc Fingers
Zinc fingers are small protein structural motifs that get their name by the fact that they are stabilized by one or more zinc ions. The vast majority of these zinc fingers are able to recognize and bind specific DNA and RNA sequences. Zinc fingers are often only able to recognize a 3 nucleotide sequence, but when multiple zinc fingers are engineered into one zinc finger array the amount of nucleotides recognized is increased.
In our project, zinc finger arrays can be used by creating a zinc finger and methylase fusion protein. This way we are able to use the same methyltransferase combined with a unique zinc finger array for each new sensor, as the binding affinity of a specificly designed zinc finger array is usually significantly higher than that of a methyltransferase.
Gel Electrophoresis
Gel electrophoresis can be used to separate a mixed population of DNA fragments by length, so one can estimate the size of DNA fragments. An electrical charge is applied on the gel that causes the negatively charged DNA molecules to move through the gel pores towards the electrical signal. Smaller DNA fragments move faster because these shorter molecules move more easily through the gel pores. After a short amount of applying the electrical charge (~30 minutes) all different DNA fragments have moved separately through the gel and are positioned according to their fragment size.
We use gel electrophoresis to separate plasmid bands by size, by which we can read out whether or not a restriction site has been cut. In the case of multiple restriction sites associated with a different sensor, we can read out the fragment sizes to determine which restriction sites have been cut, and as such which sensor has been active.
Click the arrows to go to the previous or next page
Retrieved from " http://2012.igem.org/Team:Amsterdam/project/background/ "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
