"
Page
Discussion
View source
History
teams
Log in
 
Team:Amsterdam/project/mechanism/
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
The Molecular Mechanism
Contents
1 The Molecular Mechanism
1.1 The molecular design in a nutshell
1.2 Choosing a methyltransferase
1.3 Extending the idea to multiple bits
1.4 Read out mechanism for multiple bits
The molecular design in a nutshell
We’ve established in a very early stage of the design process that we were going to use DNA methylation as the molecular mechanism to create our storage mechanism. DNA methylation, which means enriching of a specific nucleotide with a methyl group, is performed by a group of proteins called the methyltransferases. In making this design choice, we were heavily influenced by the DamID technology ( 4 , 5 ). Van Steensel was interested in the binding sites in drosophila genomes of various eukaryotic transcription factor. By fusing a bacterial methyltransferase (MTase) to these transcription factors, he was able to infer the transcription factor binding sites by reading out which genomic regions were methylated.
We thought we could reverse this idea to create a memory unit, at the core constituted by an MTase that would methylate an especially designated genomic region but only in reaction to the sensing of a signal by the cell. The epigenetic status of the bit region could thus either be 1 (methylated or written) or 0 (ummethylated or unwritten), effectively forming a binary memory unit.
Finally, the methylation status of the bit region can be assessed using digestion of the plasmid extracts with the MTase-coupled RE, followed by analyzing the band lengths intensities of a gel electrophoresis of the product.
Choosing a methyltransferase
To create this system, we first required an MTase that is not already present in the chassis organism E.coli, has a binding motif of that is not methylated by any other MTases in the system and for which a restriction enzyme (RE) dependent upon the action of this methyltransferase has been identified. This last condition is always true for any methyltransferase which has been identified to be part of a bacterial Restriction/Modification system . We wrote a python script to mine the REBASE database of REs and MTases for a MTase that meets our needs. We found an ideal candidate: M.ScaI, originally from Streptomyces caespitosus. The only remaining design question was how to control the activity of the fusion protein as we would want it to only be active in the presence of the generic signal. This design question will be more elaborately discussed in the next section.
Extending the idea to multiple bits
Extending the design idea further, we realized that it should be possible to register and store the presence of multiple signals smultaneously in a single cell. By fusing DNA binding proteins to the methyltransferase and adding their corresponding DNA binding motifs to the bit regions of the DNA. Either traditional bacterial transcriptional regulators could be used for this purpose or the Zinc-Finger Array (ZFA) technology ( 1 ). ZFA allows the construction of highly specific artificial protein-DNA interactions ( 2 ).
By fusing the MTase to a Zinc Finger or traditional transcriptional regulator and extending the bit region with the transcriptional regulator’s binding motif, the binding affinity of the fusion protein for the DNA motif could increase about 18-fold ( 3 ) (a C5-cytosine MTase was studied here, not in the class of N4C-MTases to which M.ScaI belongs). This will heavily diminish the possible aspecific cross-talk between FPs and bits assigned to other FPs.
The wide range of available Zinc Fingers could also allows for the usage of the same MTase for multiple signal. This assumes that the site specificity of the Zinc Finger Array is much higher than the site specifity of the MTase, which is very plausible ( 3 ). In this setup there would be a very low chance of cross-talk between FPs and other signals’ bits.
Read out mechanism for multiple bits
Retrieved from " http://2012.igem.org/Team:Amsterdam/project/mechanism/ "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
