"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Amsterdam/project/diary/
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
Lab
26/04/2012
We arrive at the lab to get started! First items on the agenda are getting settled at the lab and acquiring our needed equipment. As a bonus we do a transformation need tot test the Gibson method we will be using throughout these months. We will do this by cloning a GFP into a pET28 vector using our transformation protocol.
27/04/2012
Succesfully transformed the plasmids. Ordered the primers for our gibson assembly and started to prepare things. Aliquoting our phusion and gisbon master mix.
07/05/2012
Preparing O/N cultures which can be taken out of 37C in the weekend by a friend
08/05/2012
Our friend former igem member Yuri of 2011 will take out our O/N cultures.
09/05/2012
Yuri forgot... Start again. Note; rely on ourselves
10/05/2012
The primers arrived. we reconstituded them and tried them on the plasmids after mini-prepping our O/N culture. Gel: Starting our first PCR.
Modelling
08/23/12 - pBAD as an alternative to pLac
It has been some time since I have updated this diary, but we haven't been idle! Extensive simulations with different parameter values for both the diffusion constant and the \(k_{cat}\) value of the fusion protein have been performed, of which the result will be uploaded shortly in the modelling section. The general conclusion out of these simulations: the leakiness of the Lac operon is much too high for our system to work robustly. Luckily, the experimental results gathered so far have been more optimistic, in the sense that there was a clear difference between the IPTG- and IPTG+ experiments (negative and positive control respectively). Much remains to be elucidated on what underlies the results we've gotten in the lab, however.
Will the pBAD promotor be a better choice? Unlike the Lac promotor, it is not inducible by an artifial inducer.  The following quote from BioTechniques is encouraging:
"The pBAD system, which relies on catabolite repression and positive induction, is tightly regulated, it produces a relatively small amount of recombinant protein"
Now on to modelling pBAD operon and the systems functioning when we let it control our fusion protein and let's see if we achieve desirable results in silico!
08/14/12 - Automatically convert Stochpy input to LaTeX Wouldn't it be nice if you could convert your model description files to LaTeX automatically, so when you want to update your Wiki or report with your latest models they can be published right away? In case you've been using the nice python program Stochpy to do your simulations you no longer have to be left craving: we've created a tool that parses model description files in MDL format that Stochpy uses and spits out a nicely formatted latex document. You can find it on github .
08/02/12 - Fusion protein activity
There are no \(k_{cat}\) values available for the methyltransferase that we use, M.ScaI. We will have to do a parameter scan of plausible \(k_{cat}\) values. Comparing the rates of other methyltransferases will hopefully yield sensible upper and lower boundaries for this parm scan.
07/31/12 - Stochastic model of the Lac operon We are using a glucose-insensitive lac operon to control the expression of our Zinc-Finger-methyltransferase fusion protein (FP), by replacing the three naturally controlled genes (LacY, LacZ and LacA) with our fusion protein. The signal to be registered in our proof of principle, lactose, controls the expression this operon. The lac operon has been extensively studied in the past fifty years and is known to display some 'leaky' expression of the three genes. This means that some operon expression will occur independent of the presence of the natural inducer lactose. How should we control for this leaky expression? Will the small amounts of leakily expressed fusion protein cause the majority of bits in our Cellular Logbook cell to be methylated? This would mean we need to adapt our genetic circuits, as we want methylation only to occur in reaction to the signal to be measured. Or is the dreaded leaky expression too low and/or are the diffusion time and the methylation rate of the FP too low to form a serious threat?
To answer these questions, we thought it would be valuable to model the behaviour of the lac operon stochastically using Gillespie's Stochastic Simulation Algorithm . Only this way, the activity of the low copies of FP can be accurately accounted for. This will also allow us to account for natural variation occurring between individuals in the population of cells and to estimate probability distribution functions of characteristics of interest. Performing and analyzing stochastic simulations are more challenging than setting up systems of differential equations. We are currently bundling some tips on how to do this effectively. They have been gathered 'the hard way' in an trial-and-error like fashion, hopefully they can save you some trouble should you decide to do stochastic simulations as well. Expect them in the modelling section shortly.
As a starting point, the stochastic model for the lac operon by Stamatakis & Mantzaris (2007 has been used along with parameter values they have gathered. From this we can gather a basic idea of the amounts of fusion protein in the cell. The next step would be to model the activity of the FP. Should we find that the leaky expression of the FP leads to too much activity, we still have various means of increasing the protein degradation rate. Alternatively we could lower the operon's leaky expression by overexpression of LacI. More on this to come!
Celullar Logbook Construct Designer
01-09-2012
The tool is online and working. During the next meeting it will be discussed with the advisors and the team members to see if any (minor) adjustments are required.
07-08-2012
The tool now also has the minor things that were not yet added but essential in store.
Creates a seperate memory plasmid gb file and a restriction site plasmid gb file.
Been debugging and cleaning up the code for hours and hours and hours
The tool should be in beta by the end of the week
Only thing essential still missing is an (automaticly generated) list of mtases and zf's
Next week we can start with dressing up the tool for extra features such as primer generation / cloning steps
29-07-2012
We have a working tool that generates a genbank file given a backbone plasmid. The next few days will have to be spent to automatically and correctly insert the features that are to go with our memory plasmid.
Adjusted the tool so it correctly makes all the objects and it's sub objects / functions
25-07-2012
Had a meeting for our plasmid design tool establishing the rules for the plasmid generation
Wiki
05-09-2012
Filling the glossary with data
07-08-2012
To catch up on what should have already been done the following will be worked on tomorrow:
Easier navigation
Dressing up of the wiki (adding pictures/animations/presentations)
Adding a glossary
Adding a calendar with project milestones
Adding medal requirements and a link to the respective data that fullfills these requirements
31-07-2012
Put the following stuff on the wiki:
Protocols, Applications, Safety - questions, Safety - components, Frequently Asked Questions
29-06-2012
Project Description:
Spent some time the past few days to write (read: Mostly put together already existing parts) a project description to put on the wiki. Somewhere in the coming days we will review it with the group and put it online afterwards.
25-07-2012
Had a meeting for our wiki
We are going to add the following stuff in the coming weeks:
Project description - Maarten R
Frequently Asked Questions - Glenn and Ernst
Project application(s) - Matias
Retrieved from " http://2012.igem.org/Team:Amsterdam/project/diary/ "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
