"
Page
Discussion
View source
History
teams
Log in
 
Team:Amsterdam/project/lab setbacks/
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
Lab setbacks
The lab life of any researcher is a series of unavoidable experimental flaws, setbacks and incessant trials and errors. Truth is, even though one executes scrupulously experimental protocols, the outcome will occasionally be void results. Troubleshooting and optimisation are the keys to a successful project.
Contents
1 Lab setbacks
1.1 Primer Design & PCR
1.2 To silently mutate it out, or not?
1.3 Competency lost
1.4 Counteracting the leakiness
1.5 Reference List
Primer Design & PCR
The Cellular Logbook project implied numerous experimental challenges. Brainstorming sessions led to the idea that several intermediary constructs should be designed to enable better characterisation of the biobrick. Hence, the project required a great deal of cloning. Since the classical digestion–gel extraction–ligation strategy is known for its inconsistency, we opted for the one step-isothermal Gibson assembly method (GM).1 A polymerase chain reaction (PCR) had to be performed prior to the GM, leading to the first setback; primer design. GM requires specifically optimised primers with overlapping sequences between the vector and the insert, therefore, adding constraints to the choice of primers. Great care was taken in designing highly specific primers with appropriate GC content and melting temperature. However, in combination with the Phusion® High-Fidelity polymerase, the PCR resulted in a heavy background noise, with the product of interest present in only small quantities. Optimisation of the PCR was attempted repeatedly using different PCR conditions, primer dilutions, addition of a PCR enhancer mix2, but we were eventually able to reduce the background only to some extent. Amplification of the polydactyl Zinc Finger (PZF) using overlapping-ends primers was the most difficult part of the project. Since the PZF contains a series of repeated motifs, the primers were designed in such a way that two-third of the primer would anneal outside of the PZF while the rest annealed inside the PZF sequence. This choice was made to reduce the probability of the primer annealing at different places inside the PZF sequence, but also to prevent insertion of too many nucleotides to the linker, thus hindering folding of the fusion protein. Finally, we opted for gel extraction of the appropriate-sized fragment, even though this approach should be avoided according to Daniel Gibson1. Any GM attempt with impure PCR products failed. It took about two months to obtain a pure PCR fragment for the PZF. Ultimately, we succeeded in cloning the PZF into order to make our final construct.
To silently mutate it out, or not?
A ScaI restriction site was accidentally introduced during synthesis of the Myc linker, methyltransferase M.ScaI and double terminator (BBa_B0014). Since the number of ScaI restriction sites is vital for the development of the methylation-based system, it was decided to mutate this site out using site-directed mutagenesis (Agilent Technologies). However, due to time pressure, most of the experiments were conducted using the non-mutated constructs. For this reason, both the mutated and the non-mutated versions of our constructs were submitted to the iGEM headquarters.
Competency lost
Another setback experienced during this project was the fact that the bacterial strain ordered for the transformation of our clones lost their competency during transportation. Considering the limited amount of time that we had for this project, a couple of weeks of waiting for the new bacterial strain to arrive can cause a consequent delay in the flow of experiments.
Counteracting the leakiness
It was found during the course of the project that the promoter chosen, the lacI regulated, lambda pL hybrid (BBa_R0011), showed leakiness. When it comes to methylation, even low expression will cause the sites to be methylated. Therefore, it was imperative to optimise the system and reduce the leakiness of the promoter. As a result, the LacIQ bacterial strain was chosen to achieve this goal, however unsuccessfully. Switching to the pBAD weak promoter (BBa_K206001) was not helpful either. Consequently, an idea emerged that the copy number of the plasmids used might have a strong influence on the expression of the methyltransferase. This option involved more cloning steps which proved to be hassle but we finally managed to clone only the final construct involving the LacH promoter, PZF, methyltransferase, memory part into the pSB3C5 (low to medium copy number plasmid) and pSB4C5 (low copy number plasmid).
“Success is stumbling from failure to failure with no loss of enthusiasm”
– Winston S. Churchill
Given the amount of setbacks encountered, the satisfaction of eventually succeeding most of the cloning steps and proving our concept was delightful. This experience enabled us to develop scientific creativity, troubleshooting skills and critical thinking.
Reference List
1.      Gibson,D.G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nat. Methods 6, 343-345 (2009).  2.      Ralser,M. et al. An efficient and economic enhancer mix for PCR. Biochem. Biophys. Res. Commun. 347, 747-751 (2006).
Retrieved from " http://2012.igem.org/Team:Amsterdam/project/lab_setbacks/ "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
