"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Amsterdam/outreach/handout
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
iGEM Amsterdam Handout
Contents
1 iGEM Amsterdam Handout
1.1 International Genetically Engineered Machine Competition (iGEM)
1.2 Team UvA/VU
1.3 Our project: The Cellulair Logbook
1.4 Applications
1.4.1 Medical applications
1.4.2 Environmental applications: Heavy Metals
1.4.3 Environmental applications: Using synthetic biology to protect against synthetic biology
1.5 The mechanism
1.6 Definitions
1.7 References
International Genetically Engineered Machine Competition (iGEM)
iGEM[1] is an international undergraduate 'synthetic biology'[2] competition. Participants are teams that represent one or multiple universities. Each team aims to develop genetically modified cells that contribute to science and/or society. The process touches all the activities that are required in a 'real' project: brainstorming for ideas, turning ideas to practice, acquirement of sponsorships and human outreach.
Each team is obliged to present their project on a wiki site, which will be judged between the 26th of September and the start of the regional jamborees. 5 regional jamborees are held at the beginning of October[3] where each team will have a poster about their project and hold a presentation in front of the other team and the judges. The highest scoring teams will go on to the finals in Boston hosted by MIT at the beginning of November.
This years (and last years) European Jamboree is held at the Vrije Universiteit, Amsterdam. Organized by the universities of all participating Dutch teams (Amsterdam, Groningen, Delft, Wageningen, Eindhoven).
Team UvA/VU
Amsterdam is represented by a joint team of the Vrije Universiteit and the Universiteit van Amsterdam. The team itself consists of six undergraduate (MSc) students that span the following studies: Biomoleculair Sciences, Systems Biology, Bioinformatics.
Our project: The Cellulair Logbook
Meet the Cellulair Logbook. It does what the name implies: A cell that keeps track of its cell variables and variables in its close environment. These variables can be anything that can be sensed by a cell. Some examples are: oxygen, light, toxics, heavy metals and nutritients. When a cell measures one of these variables it is stored in the cell by methylation of restriction sites (see definitions), literally storing data on the DNA. By having these restriction sites methylated, the DNA can not be cut and as such the read out will show that the the gene has been expressed, and as such we know that the variable has been present. Because in bacteria daughter cells do not inherit methylated sites from its parent, every reproduction cycle the amount of methylated cells is halfed. This way we are able to use mathematical models to trace back when something is methylated and when the variable has been present.
Applications
It is very important to realise that what we are doing is very fundamental research. This means that once we are done it can not be directly used for any of the applications below. However, when further build upon these ideas we believe that the applications listed below are possible. Some are a long way into the future (years) where others might be applicable after months of further research.
Medical applications
In the future we might be able to put our cell in the human body. The cell will then travel trough it and keep track of its surrounding variables. When the cell has travelled trough the human body it is excreted and the 'log' is read out.
If, for say, the cell has noted that at some point there was less oxygen present than normal, we can trace back when this has happened with mathematical models using the fact that bacterial cells do not inherit methylated sites. Knowing that an area around the tumor is less oxygen rich than normal areas in the human body, we can mark this region as potential tumor region.
Environmental applications: Heavy Metals
If we design our system in such a way that it is able to recognize multiple heavy metals (by having a sensor, fusion protein and restriction site for each) we can detect if water is polluted. This would be especially handy in third world countries, as this would be a cheap method to detect if water is drinkable or not.
Environmental applications: Using synthetic biology to protect against synthetic biology
One of the potential dangers of synthetic biology is that potentially harmful cells are released in the environment and can not be extracted as easily. We can use our system to detect these harmful cells and excrete them from the environment.
Note: Our cell does not pose a threat to the environment itself
More applications are given here .
The mechanism
A gene is expressed when certain requirements are met, which we call a 'sensor'. In our proof of concept that we are trying to achieve, we use a lactose sensor. In order to methylate our target restriction site we need to express a gene that translates into a methylase protein. We achieve this by replacing the gene activated by the sensor with a gene that expresses the methylase. Because we want to make sure that only the right site is methylated we also express a zinc finger in our gene, creating a zinc-finger and methylase fusion protein. By putting the site recognized by our zinc finger close to the restriction site that is recognized by our methyltransferase, we make sure it is much more likely that our methylates the right spot.
When we expose our cell to lactose it will methylate the restriction site associated to our fusion protein. If we then expose our cell to the restriction enzyme associated to our restriction site, the DNA will not be cut, where it would be cut if lactose would not be sensed. Using 'gel electrophoresis'[4] or an other method that is able to visualize the size and amount of DNA bands, we can identify if lactose has been sensed or not.
Definitions
Sensor: A piece of DNA that recognizes (directly or indirectly) a variable (molecule, protein, etc. etc.). Often a sensor is a switch for gene expression.
Gene: A piece of DNA that often can be transcripted and translated to a protein.
Plasmid: DNA that lives in a host cell, but is not part of the cells core DNA
Restriction enzyme: A protein that cuts DNA at its associated restriction site
Restriction site: A specific DNA sequence that can be cut by its associated restriction enzyme
Methyltransferase: A protein that methylates a specific DNA sequence
Methylase: A molecule that is the result of the methyltransferase. It sits on top of a methylated DNA sequence, making it inaccessible to other molecules and proteins. A restriction enzyme can not access a restriction site if it is methylated.
Zinc-finger: A protein that recognizes a specific sequence of DNA and binds to it.
Contact: igemamsterdam2012@gmail.com
References
1- http://www.igem.org
2- http://en.wikipedia.org/wiki/Synthetic_biology
3- https://2012.igem.org/Jamborees
4- http://en.wikipedia.org/wiki/Gel_electrophoresis
Retrieved from " http://2012.igem.org/Team:Amsterdam/outreach/handout "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
