<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Amsterdam_achievements_stochastic_model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Amsterdam/achievements/stochastic_model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Amsterdam/achievements/stochastic_model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/achievements/stochastic_model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/achievements/stochastic_model&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Amsterdam/achievements/stochastic_model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Amsterdam/achievements/stochastic model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="main"><DIV id="main-area"><DIV id="sidebar-area"><DIV class="content-block"><DIV id="navmenu"><UL><LI><A href="/Team:Amsterdam">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Amsterdam/team/members">Members</A></LI><LI><A href="/Team:Amsterdam/team/advisors">Advisors</A></LI></UL><LI><A href="/Team:Amsterdam/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Amsterdam/project/molecular_design">Molecular Design</A></LI><LI><A href="/Team:Amsterdam/project/biobricks">BioBricks</A></LI><LI><A href="/Team:Amsterdam/project/features_and_applications">Features and Applications</A></LI><LI><A href="#">Data</A></LI><LI><A href="/Team:Amsterdam/data/experimental">Experimental Results</A></LI><LI><A href="/Team:Amsterdam/data/time_inference_model">Time Inference Model</A></LI><LI><A href="/Team:Amsterdam/data/background_activity">Background activity</A></LI><LI><A href="/Team:Amsterdam/project/diary/">Logbook Designer</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/webtool">Webtool</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/setup">Setup</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/manual">Manual</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/future_perspective">Future Perspective</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Amsterdam/practices/overview">Overview</A></LI><LI><A href="/Team:Amsterdam/practices/methods">Methods</A></LI><LI><A href="/Team:Amsterdam/practices/results">Results</A></LI><LI><A href="/Team:Amsterdam/practices/conclusion">Conclusion</A></LI><LI><A href="#">Safety</A></LI><LI><A href="/Team:Amsterdam/safety/questions">Questions</A></LI><LI><A href="#">Extra</A></LI><LI><A href="/Team:Amsterdam/extra/protocols">Protocols</A></LI><LI><A href="/Team:Amsterdam/extra/diary">Lab Diary</A></LI><LI><A href="/Team:Amsterdam/extra/faq">FAQ</A></LI><LI><A href="/Team:Amsterdam/extra/software">Software</A></LI></DIV></DIV></DIV><DIV id="content-area"><DIV id="sub-menu" class="content-block"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Stochastic_Differential_Equations"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Stochastic Differential Equations</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Comparison_of_SSA_implementations"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Comparison of SSA implementations</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Leakiness_and_background_noise"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Leakiness and background noise</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#ODE_Model_definition"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">ODE Model definition</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Stochastic_model_definition"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Stochastic model definition</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Retrieving_sensible_parameter_values"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Retrieving sensible parameter values</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Leaky_expression_rate"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Leaky expression rate</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Fusion_protein_catalysis_rate"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Fusion protein catalysis rate</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Cell_growth_rate"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Cell growth rate</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Other_parameters"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Other parameters</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Steady_state_parameter_scanning"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Steady state parameter scanning</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Discussion"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><P>Here we will propose a solution to the the background noise observed in the experiments, and conclude what rates of leakiness are acceptable to the system.
In this, we model the system both using both ordinary differential equations and stochastically using the Gillespie SSA algorithm.
</P><P>We will focus on the effects of two rate constants in particular on the fraction of methylated plasmids in absence of the inducing signal: 
</P><UL><LI> the leaky transcription rate of the mRNA for the fusion protein (FP), denoted by $k_{c\text{MFP}}$
</LI><LI> the catalysis rate of the FP, or the rate with which it methylates umethylated plasmids
</LI></UL><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>Before starting the experiments, a lot of crucial parameters on the molecular system were still unknown.  Both the leaky expression rate of the operon and the activity of the fusion protein were hard to model because of this.  Sensible predictions on how the molecular system we were designing was going to function exactly therefore seemed impossible.  We already expected the promoter/sensor we were going to test, pLac, to show some leaky expression.  As to how much background activity this was going to cause, we couldn't predict and thus we proceeded to test out the construct in the lab.  The <A href="/wiki/index.php?title=Team:Amsterdam/Lab&amp;action=edit&amp;redlink=1" class="new" title="Team:Amsterdam/Lab (page does not exist)">laboratory results</A> seemed to suffer from two problems: 
</P><UL><LI> A basal methylation activity that was always recorded even in absence of the signal.This background noise reduces the dynamic range of our system; there is no way to tell whether observed band intensities are due to the signal having passed the <I>Cellular Logbook</I><A href="/wiki/index.php?title=Team:Amsterdam/modelling/timeinferrance&amp;action=edit&amp;redlink=1" class="new" title="Team:Amsterdam/modelling/timeinferrance (page does not exist)">some time ago</A> or simply because of the background noise. We conclude that storing a signal using the <I>Cellular Logbook</I> requires a transcriptional regulator that has a high repressing quality in order to diminish the background activity.
</LI><LI> A lack of a fully induced state for the pLac operon, but not for the later tested pBAD operon. We suspect this problem to be caused by a low uptake rate of the signalling compound (lac operon inducer IPTG) compared to the amount of repressor LacI present in the system controlling the operons. Especially for the part characterization experiments in which the pLac-MTase construct was cloned into a high copy number plasmid, this could be a problem. This problem is solved by adding the <I>LacY</I>-gene that codes for permease to the existing construct. Inclusion of this gene might also lead to higher background methylation percentages in the negative controls, however.
</LI></UL><H4><SPAN class="mw-headline" id="Stochastic_Differential_Equations"> Stochastic Differential Equations </SPAN></H4><P>Little amounts of certain key proteins can have large effects on the physiological behaviour of a cell. 
When trying to accurately model these small amounts of proteins the thermodynamic equilibrium assumption that characterizes ODE-models might not be valid;
ordinary differnential equations are unsuited towards modelling systems with small integer amounts of the constituting species.
In these cases a finer-grained modelling method is called for: the Stochastic Simulation Algorithm by Gillespie.
By modelling each individual molecular reaction separately, the discreteness of this small amounts system is accounted for. 
Despite the small amounts of species present in the system analyzed here, we found no qualitative differences between the ODE and stochastic versions of the model.
</P><H4><SPAN class="mw-headline" id="Comparison_of_SSA_implementations"> Comparison of SSA implementations </SPAN></H4><P>During the course of the summer we've investigated many stochastic simulation packages with implementation of the direct algorithms and some more coarse-grained optimizations thereof (e.g. the tau-leaping algorithm).
Most notable are <A href="http://www.xlr8r.info/SSA/" class="external text" rel="nofollow">xSSA</A> for Mathematica and <A href="http://www.stompy.sourceforge.net" class="external text" rel="nofollow">StochPy</A> for Python.
The former seems a great tool, but at the time of writing seems to lack the ability to process third order reactions (reactions with three reactants).
The latter has been developed at the Free University Amsterdam by Timo Maarleveld, currently PhD-student there.
In using this package, we've kept in close contact with Timo and submitted a lot of bug reports, helping the software to grow.
Additionally, we've also extended StochPy by writing a supplementary tool <A href="https://2012.igem.org/Team:Amsterdam/extra/software" class="external text" rel="nofollow">MDL2LaTeX</A>, that eases the publishing of models developed with StochPy.
</P><H1><SPAN class="mw-headline" id="Leakiness_and_background_noise">Leakiness and background noise</SPAN></H1><P>This model has as its main goal to identify how much the uninduced operon expression rate and the catalysis rate of the FP affect the methylation status of the plasmids.
Each plasmid is assumed to have one bit and one operon that includes the fusion protein.
Running the stochastic version of this model, we noticed that keeping the amount of equations to a minimum is paramount to success, as the computation time seems to increase exponentially with the amount of reactions in the system.
Focussing on the leakiness rate instead of the repressor-operon interactions that would require a lot of equations, we will therefore make one bold assumption:
all operons present in a single cell are permanently bound to a transcriptional repressor.
This frees us from having to code the signal entering the cell and binding the repressing TF, granting focus to the leaky expression rate and the catalysis rate of the fusion protein.  
For these models to have any forecastable value therefore, the amount of repressor molecules present in the system has to be much greater than the amount required to cover all operons.
</P><P>In order to not exclude the possible qualitative difference in model behaviour when all molecular reactions are computed individually, the system has also been modelled stochastically. To make the two systems perfectly comparable, the used rates are qualitatively identical to the ones used in the ODE system. 
Furthermore, the same parameter values have been used.
</P><H2><SPAN class="mw-headline" id="ODE_Model_definition">ODE Model definition</SPAN></H2><TABLE align="right"><TBODY><TR><TH>Parameter</TH><TH>Value</TH></TR><TR><TD>Ca</TD><TD>$200\ \text{or}\ 40$</TD></TR><TR><TD>ksFP</TD><TD>$30$</TD></TR><TR><TD>lMFP</TD><TD>$0.462$</TD></TR><TR><TD>lFP</TD><TD>$0.2$</TD></TR><TR><TD>kPlas</TD><TD>$0.00866434 \cdot Ca$</TD></TR><TR><TD> lPlas </TD><TD> $0.00866434$ </TD></TR><TR><TD colspan="2" align="center">Parameter values used to plot the ODE-system</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Time trajectories of ODE-model in which the construct has been inserted in a high copy number plasmid (top, $Ca = 200$) and a low copy number plasmid (bottom, $Ca = 40$). All initial species values are set to $0$, except for the intial value of unmethylated plasmids which is equal to $Ca$. With these parameters, the dynamic range of the system is completely taken up by the background noise. No qualitative difference is observed between the high and low copy number cases</DIV></DIV></DIV><P>FP-mRNA results only from leaky expression from the operons $O$ with leakiness rate $k_{\text{sMFP}}$ and is degraded with rate $\lambda_{\text{MFP}}$:
$$
\frac{d\text{mFP}}{dt} = k_{\text{sMFP}} \cdot \text{O} - \lambda_{\text{MFP}} \cdot \text{MFP}
$$
FP is created from mRNA and with rate $k_{\text{sFP}}$ and degraded with rate $\lambda_{\text{FP}}$:
$$
\frac{d\text{FP}}{dt} = k_{\text{sFP}} \cdot \text{MFP} - \lambda_{\text{FP}} \cdot \text{FP}
$$
Unmethylated plasmids are created from both unmethylated and methylated plasmids; methylation is not transferred to daughter plasmid during plasmid replication in prokaryotes. Plasmid growth continues until the plasmid copy number $Ca$ has been reached:
$$
\frac{d\text{PlasU}}{dt} = k_{\text{Plas}}  (1 - \frac{(\text{PlasU} + \text{PlasM})}{Ca}) - k_{cFP} \cdot \text{FP} \cdot \text{PlasU} - \lambda_{\text{Plas}} \cdot \text{PlasU},
$$
Methylated plasmids result from the fusion protein finding an umethylated plasmid and methylating it:
$$
\frac{d\text{PlasM}}{dt} = k_{c\text{FP}} \cdot \text{FP} \cdot \text{PlasU} - \lambda_{\text{PlasM}} \cdot \text{PlasM}
$$
The amount of (repressed) operons is identical to the amount of plasmids currently in the system:
$$
\text{O} = \text{PlasU} + \text{PlasM}
$$
</P><H2><SPAN class="mw-headline" id="Stochastic_model_definition"> Stochastic model definition </SPAN></H2><P>The equations in the stochastic model are qualitatively equal to the ones in the differential equation model.
The equations, parameter and initial species values can be viewed on <A href="/Team:Amsterdam/achievements/stochastic_model_definition" title="Team:Amsterdam/achievements/stochastic model definition">this page</A>.
Analyzing the behaviour of this model by looking at the time-lapse plots, we see a similar trend as in the ODE-model:
all plasmids are methylated within a short amount of time. The fusion protein amounts are shown to be widely varying, but this does not alter the fraction of methylated plasmids much.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Single trajectory of stochastic model. Just as in the deterministic model, all plasmids are methylated within a short amount of time due to the leaky expression with the used values for $ k_{cat} $ and $ k_{cFP} $</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Retrieving_sensible_parameter_values"> Retrieving sensible parameter values </SPAN></H2><H4><SPAN class="mw-headline" id="Leaky_expression_rate"> Leaky expression rate </SPAN></H4><P>There are discrepancies between the definitions used in the literature on what leaky expression exactly is:
</P><UL><LI> The most favoured definition seems to be the that leaky operons show low expression rates even when bound by a negative transcriptional regulator. In this perspective the transcriptional regulator is regarded to be unable to completely silence gene expression. In this case the only way to get a value for leaky expression is to measure it experimentally. In this process, care should be taken to be specific about what version of the operon exactly is being used. Some repressors are known to regulate transcription by binding to DNA regions that are very distant from the controlled operon. 'Plasmid versions' of these operons might not contain these distantly located binding sites and are therefore less likely to be optimally repressed.
</LI><LI> Another definition is that the TF <I>can</I> fully silence gene expression and that leaky expression is caused by seldomly occurring dissociation events between between the TF and the DNA operator sequence.
</LI></UL><P>During these short time windows of opportunity, the RNA polymerases would then be able to quickly transcribe a small amount of RNA.
Multiplying the maximal rate of transcription by the time fraction during which the polymerase is able to bind gives an indication of the leaky transcription rate defined this way.
</P><P>The following relationship must always be true: $[O_{\text{Total}}] = [O_{\text{Free}}] + [R_{2}O]$ with $R_{2}O$ denoting the operon bound by a dimerized repressor (as LacI, the repressor of the Lac operon, functions). $R_{2}O$ is defined as $\frac{[R_{2}][O]}{K_{d}}$. Solving for $O_\text{Free}$:
</P><P>$$
[O_{\text{Free}}] = [O_{\text{Total}}] (1 + \frac{[R_{2}]}{K_{d}})
$$
</P><P>We immediately see that the $[O_{\text{Free}}]$ depends on $[R_{2}]$ and $O_{\text{Total}}$.
The dissociation constant of the dimerized LacI-repressor for its operator sequence has been determined to be $10^{-12} M $ and the volume of <I>E. coli</I> is $0.7\ \mu m^{3}$.
The transcription rate in E.coli is 40-80 bp/s.
</P><P>More simply, the leaky expression rate can also be retrieved from this [<A href="http://oregonstate.edu/instruction/bb492/lectures/Regulation.html|website" class="external autonumber" rel="nofollow">[1]</A>], which states a 1000-fold decrease in expression of the repressor-bound operon compared to the free operon.
</P><H4><SPAN class="mw-headline" id="Fusion_protein_catalysis_rate"> Fusion protein catalysis rate </SPAN></H4><P>The catalysis rate $k_{cat}$ has not been determined for the MTase that we are using, M.ScaI. 
However, for another MTase with the same <A href="http://www.brenda-enzymes.info/php/result_flat.php4?ecno=2.1.1.113" class="external text" rel="nofollow">EC2.1.1.113 number</A> as M.ScaI, BamH1, the $k_{cat}$ has been determined to be $0.0175$ (<A href="#Cheng">1</A>).
</P><H4><SPAN class="mw-headline" id="Cell_growth_rate"> Cell growth rate </SPAN></H4><P>Our own experiments showed that the bacterial strain &lt;math&gt;DH5\alpha&lt;/math&gt; transformed with two of our preliminary constructs had growth rates (&lt;math&gt;\mu&lt;/math&gt;) between $80\ min^{-1}$ and $90\ min^{-1}$ ([Team:Amsterdam/Project/Growth_curves Experimental growth curves]).
In these simulations, cellular division was assumed to be either constant at 80 for single cells models or Gaussian distributed with $\mu = 80$ and $\sigma = 2$ in the cell division model.
</P><P>Looking at the plots for both the high and the low copy number plasmids, no qualitative difference is observable. All species in the system simply reach a steady state defined as the fraction between their production and degradation rates.
Most notably, in the steady state that this system reaches all plasmids are methylated.
This is very undesirable of course!
</P><H4><SPAN class="mw-headline" id="Other_parameters"> Other parameters </SPAN></H4><P>Values for the mRNA degradation rate and protein synthesis have been taking from (<A href="#Mantzaris">2</A>).
</P><H2><SPAN class="mw-headline" id="Steady_state_parameter_scanning"> Steady state parameter scanning </SPAN></H2><P>In order to deduce what ranges of operon leaky expression and FP catalysis rates would yield acceptable leaky expression, a steady state parameter scan has been performed.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Parameter scan to study the effects of the operon leaky expression rate $k_{s0MFP}$ and the FP catalysis rate $k_{c\text{FP}}$ on the steady state fraction of $ \frac{\text{Methylated plasmids}}{\text{Methylated} + \text{Unmethylated plasmids}}$ in the ODE-model</DIV></DIV></DIV><P>Our experimental results can thus be explained by any combination of the two parameters which which has an intermediate value in plot.
</P><P>A similar parameter scan has been painstakingly performed with the stochastic model. Simulations with 20 trajectories of 200-minute simulations for each parameter combination in a two-dimensional scan for both the $k_{cat}$ and $k_{cFP}$ in the same ranges as the ODE parm-scan. Unfortunately, even for these high amount of replications the variability in the resulting methylation fractions is very high, such that no trends are discernible at all in the results (data not shown). This variability can probably be reduced by performing even longer simulations and using more trajectories. A much more efficient implementation of the SSA will have to be used to reach this goal within reasonable time scales however. We hope to finish these simulations before the Jamboree. Because the deterministic simulations show similar qualitative behaviour as the more realistic stochastic simulations in the time-lapse plots, one could argue that we will obtain similar results here as the much less computationally expensive ODE-parameter scan.
</P><P>The steady state fraction of methylation shows almost no dependence upon the leaky transcription rate in the ODE model. This is likely caused by the steady state value that the mRNA reaches fairly quickly. The parameter scan density plots shows a very weak to no influence of the background noise. The fact that the degradation rate of the mRNA is much larger than the leaky transcription rate probably causes this. On the contrary, the steady state fraction of methylated plasmids is shown to be heavily dependent upon the catalysis rate of the fusion protein. This can steer future experiments to research the effects of lowered catalysis rates.
</P><H1><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H1><P>Unlike our initial expectations, the ODE-model analyzed here suggests that the catalysis rate of the FP is more important to the high background noise we have observed in the experiments than the leaky expression rate. 
This yields a new hypothesis to test in the lab: will lowering the catalysis rate significantly lower the background methylation rate?
This raises the question on how to lower the catalysis rate of the MTase. A few methods to do this come to mind:
</P><UL><LI> Lower the temperature during the experiments, which will slow down all reactions in the cell including the catalysis rate
</LI><LI> Mutate amino acid sequence of binding domain, making the affinity of the for the DNA smaller
</LI><LI> Mutate some other part of the protein
</LI><LI> Pick a different methyltransferase, one that has a lower catalysis rate
</LI><LI> Attach a fluorescent protein to the methyltransferase
</LI></UL><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P><SPAN id="Cheng"><SUP>
Cheng, X., &amp; Blumenthal, R. (1999). S-Adenosylmethionine-Dependent Methyltransferases: Structures and Functions (p. 400). World Scientific. Retrieved from <A href="http://books.google.com/books?id=oUCKHnsZuukC&amp;pgis=1" class="external free" rel="nofollow">http://books.google.com/books?id=oUCKHnsZuukC&amp;pgis=1</A></SUP></SPAN></P><P><SPAN id="Mantzaris"><SUP>
Stamatakis, M., &amp; Mantzaris, N. V. (2009). Comparison of deterministic and stochastic models of the lac operon genetic network. Biophysical journal, 96(3), 887–906. doi:10.1016/j.bpj.2008.10.028
</SUP></SPAN></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Amsterdam/achievements/stochastic_model">http://2012.igem.org/Team:Amsterdam/achievements/stochastic_model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Amsterdam/achievements/stochastic_model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Amsterdam/achievements/stochastic_model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Amsterdam/achievements/stochastic_model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Amsterdam/achievements/stochastic_model&amp;oldid=236460" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>