<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Amsterdam_data_experimental"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Amsterdam/data/experimental">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Amsterdam/data/experimental&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/data/experimental&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/data/experimental&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Amsterdam/data/experimental" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Amsterdam/data/experimental</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="main"><DIV id="main-area"><DIV id="sidebar-area"><DIV class="content-block"><DIV id="navmenu"><UL><LI><A href="/Team:Amsterdam">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Amsterdam/team/members">Members</A></LI><LI><A href="/Team:Amsterdam/team/advisors">Advisors</A></LI></UL><LI><A href="/Team:Amsterdam/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Amsterdam/project/molecular_design">Molecular Design</A></LI><LI><A href="/Team:Amsterdam/project/biobricks">BioBricks</A></LI><LI><A href="/Team:Amsterdam/project/features_and_applications">Features and Applications</A></LI><LI><A href="#">Data</A></LI><LI><A href="/Team:Amsterdam/data/experimental">Experimental Results</A></LI><LI><A href="/Team:Amsterdam/data/time_inference_model">Time Inference Model</A></LI><LI><A href="/Team:Amsterdam/data/background_activity">Background activity</A></LI><LI><A href="/Team:Amsterdam/project/diary/">Logbook Designer</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/webtool">Webtool</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/setup">Setup</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/manual">Manual</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/future_perspective">Future Perspective</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Amsterdam/practices/overview">Overview</A></LI><LI><A href="/Team:Amsterdam/practices/methods">Methods</A></LI><LI><A href="/Team:Amsterdam/practices/results">Results</A></LI><LI><A href="/Team:Amsterdam/practices/conclusion">Conclusion</A></LI><LI><A href="#">Safety</A></LI><LI><A href="/Team:Amsterdam/safety/questions">Questions</A></LI><LI><A href="#">Extra</A></LI><LI><A href="/Team:Amsterdam/extra/protocols">Protocols</A></LI><LI><A href="/Team:Amsterdam/extra/diary">Lab Diary</A></LI><LI><A href="/Team:Amsterdam/extra/faq">FAQ</A></LI><LI><A href="/Team:Amsterdam/extra/software">Software</A></LI></DIV></DIV></DIV><DIV id="content-area"><DIV id="sub-menu" class="content-block"><H1><SPAN class="mw-headline" id="Experimental_Results">Experimental Results</SPAN></H1><H2><SPAN class="mw-headline" id="Functionality_of_the_writer-reader_module_under_different_sensor_modules">Functionality of the writer-reader module under different sensor modules</SPAN></H2><P>We started measuring the first proof-of-concept of the Cellular Logbook by testing the functionality of a part of our writer-reader module in the context of various sensor modules. To this end, we tested the functionality of the synthesized Methyltransferase (MTase) (our writer) cloned under the control of the LacH promoter in the pSB1AT3 backbone and transformed in Library Efficient® DH5α™ competent cells (Invitrogen).
</P><H4><SPAN class="mw-headline" id="Set_up_of_the_writer-reader_module">Set up of the writer-reader module</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 1</DIV></DIV></DIV><P>As mentioned in Molecular design, the ScaI restriction enzyme is unable to cut methylated restriction sites. Therefore, we expect different possible restriction profiles through ScaI restriction digestion since there is one ScaI site residing in the pSB1AT3 backbone and we created one ScaI site via a scar inside our writer module. We expect to find either an off or intermediate methylation state knowing that the LacH promoter driving the MTase has some basal activity.
</P><P>Figure 1 shows the result of a ScaI restriction digestion of the pSB1AT3/LacH/MTase construct without IPTG induction. The pattern displayed corresponds to a combination of bands indicating an intermediate state between the ‘off’ and ‘on’ situation. Interpretation of the read-out: absence of MTase expression in <I>E. coli</I> leads to a complete digestion of our plasmid. Two bands of 2989 bp and 1621 bp are then observed (A). Incomplete methylation of the plasmid at only one of the two ScaI sites shows a linearized plasmid band 4610 bp. Complete methylation of our writer module prevents ScaI to cut and a typical uncut plasmid profile is observed (C).
</P><H4><SPAN class="mw-headline" id="Behavior_of_the_writer-reader_module_under_IPTG_induction">Behavior of the writer-reader module under IPTG induction</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 2</DIV></DIV></DIV><P>The next step was to see if IPTG-based induction of MTase expression would modify the methylation state of our writer-reader module thus changing the resulting restriction profile. In the presence of 1mM IPTG, the promoter should be activated providing MTase production inside the cells. Since our model for this experiment suggests that methylation occurs at a fast rate we expect that after 16h of IPTG induction, the read-out will dramatically shift to the uncut profile.
Figure 2
</P><P>The writer-reader module after 16h IPTG induction still shows a partial methylation profile, indicating that our writer-reader module is not fully methylated, we observe a gradual shift towards the uncut plasmid form. That means that IPTG induction leads to an increase of methylation of the plasmid and that our writer-reader module is able to store and read this information: Our writer-reader module is in principle working!!
</P><H4><SPAN class="mw-headline" id="Behavior_of_the_writer-reader_module_in_varying_growth_conditions">Behavior of the writer-reader module in varying growth conditions</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 3</DIV></DIV></DIV><P>We aimed to characterize the activity of our writer during bacterial growth.  We performed a growth curve experiment (figure 3) and collected samples at different time points to test the occurrence of methylation over different times of growth. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 4</DIV></DIV></DIV><P>Figure 4 shows ScaI digestion of plasmids extracted from samples activated or not by 10mM IPTG and collected between 0 and 12h. Both conditions reveal an intermediate restriction profile over. However, the two samples taken after 24 hours show a different result, i.e. almost only uncut plasmid is observed as a result of an significant increase of methylation of our reader.  
</P><P>We aimed to test the functionality of our system in the stationary state. It was previously described in the literature that the Lac promoter upon IPTG induction is performing better in the stationary phase because the lac-permease, encoded by the LacY gene and responsible for the correct IPTG uptake inside the cell, can be quantitatively integrated into the membrane. 
</P><P>The same bacterial batch was used in both experiments but for the stationary experiment first grown over a longer period of time (at least more than 24 hours), either with or without the inducer. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 5</DIV></DIV></DIV><P>Figure 5 shows the restriction profile taken from three different time points: 48, 72 and 96 hours. The first three restriction profiles correspond to growth without IPTG induction and the last three with 10 mM IPTG. During the culture no additional IPTG was added since we expect that IPTG is not degraded over time. We do not observe a change in the restriction profile, meaning that the time of culture does not influence our results. 
</P><P>
In conclusion, the presence of the LacH promoter showed to provide the activity of the Mtase but did not deliver a restriction profile fitting to the ‘off’ and ‘on’ states. Our observations are in line with our predictions as described in the Molecular Design. An alternative way to lower the basal activity of LacH is to perform our experiments at high LacI levels which is suggested to decrease the basal activity of LacH. In addition, LacH in high-copy plasmids could createsmuch more non-specific activity of the LacH than in a low-copy plasmid.
</P><H4><SPAN class="mw-headline" id="Behavior_of_the_writer-reader_module_under_reduced_basal_LacH_promoter_activity_using_LacIQ_E._Coli_strain">Behavior of the writer-reader module under reduced basal LacH promoter activity using LacIQ E. Coli strain</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:263px;"><DIV class="thumbcaption">Figure 6</DIV></DIV></DIV><P>Both modeling and experimental results show a strong basal activity of the LacH promoter leading to a substantial methylation of our memory module, even without IPTG present in the medium. We aimed to get rid of the basal activity of the LacH promoter without IPTG induction to create a better ‘off’ state and hence a better writer-reader design.
</P><P>Since the silencing of the LacH promoter depends on the concentration of the LacI repressor, we chose to replace our standard DH5α E. Coli strain by the LacIQ strain. Investigation done on the LacH promoter revealed that a higher expression of LacI will show a better suppression and decrease of basal activity of the LacH promoter.
</P><P>We transformed our pSB1AT3/LacH/Mtase vector in the LacIQ strain and performed a restriction digestion profile experiment after 24h of growth and under 10 mM IPTG induction.
</P><P>Figure 6 shows the pSB1AT3/LacH/Mtase cultured overnight in LacIQ E. Coli strain and digested with ScaI restriction enzyme.
</P><P>The results of the restriction profile for the LacIQ E. Coli strain are inconclusive (figure 6). No change in the ScaI restriction patterns are observed between plasmids isolated from both strains. Higher expression of LacI does not seem to reveal a restriction profile that confirms the ‘off’ state and it does not seem to creates a shift towards the ‘off’ state. 
</P><H4><SPAN class="mw-headline" id="Behavior_of_the_writer-reader_module_under_tight_control_of_and_arabinose-regulated_promoter">Behavior of the writer-reader module under tight control of and arabinose-regulated promoter</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 7</DIV></DIV></DIV><P>The Cellular Logbook was further characterized using the pBad promoter.  This promoter was chosen in order to overcome the leakiness observed with the LacH promoter. Expression of any gene cloned behind the pBad promoter is controlled by the AraC repressor and is considered to be fully suppressed in the absence of arabinose. However, in the presence of arabinose the promoter will generate a gradual response leading to gene expression. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 8</DIV></DIV></DIV><P>	The first characterization experiment involved induction of pSB1AT3/pBad/Mtase in Library Efficient® DH5α™ competent cells (Invitrogen) in stationary phase with 1 % arabinose. The construct was digested with ScaI restriction enzyme after 24 hours incubation at 37˚C. Surprisingly, the same intermediate digestion profile as the LacH was observed. A combination of different restriction profiles can be inferred from the results (figure 7). In the absence of arabinose, M.ScaI methyltransferase appears to be expressed and a significant number of plasmids present in the culture are fully methylated, accounting for the intense uncut DNA fragment observed. A similar pattern was observed in the presence of arabinose.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 9</DIV></DIV></DIV><P>The second characterisation experiment involved the induction of pSB1AT3/pBad/Mtase in Efficient® DH5α™ competent cells (Invitrogen) in stationary phase with 2 % arabinose and incubation at 37˚C for 48 hours. Subsequent digestion with ScaI restriction enzyme showed a shift towards the uncut restriction profile (figure 8). This result shows that under these conditions the writer-reader is optimally functional. Succes!
</P><P>
The third characterisation experiment concerns the stimulation of pSB1AT3/pBad/Mtase in Efficient® DH5α™ competent cells (Invitrogen) in stationary phase with addition of 2% arabinose daily. Samples were taken after 48, 72 and 96 hours, and digested with ScaI restriction enzyme (figure 9). These results are consistent with the previous results, showing a gradual shift towards the “on-state” with induction of the pBad promoter by addition of arabinose daily.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 10</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 11</DIV></DIV></DIV><P>The “off” state was not achieved in the absence of arabinose. All the experiments have been conducted into the high copy number plasmid pSB1AT3. It is known that leaky expression is relative to the copy number of the plasmid.1 Hence, the inability of the Cellular Logbook to show an “off” state in the absence of arabinose could be attributed to the high copy number plasmid used in these experiments as was discussed with the LacH.
</P><P>Similarly to the pSB1AT3/LacH/Mtase, a growth curve experiment was conducted to characterize the acquired construct of pSB1AT3/pBad/Mtase (figure 10).
</P><P>
Over the course of time the  methylation-dependent restriction profile observed in the gel showed a shift towards the ‘on’ digestion profile in the presence of arabinose (figure 11). This result shows that our Cellular Logbook is able to sense and write an arabinose signal present in the medium in a shorter period of time.</P><H2><SPAN class="mw-headline" id="Towards_the_Cellular_Logbook">Towards the Cellular Logbook</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 12</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 13</DIV></DIV></DIV><P>After months of cloning attempts, it appears that we succeeded in obtaining the final version of our Cellular Logbook: the pSB1AT3/LacH/PZF3838/Mtase/Mem2X (BBa_K874300). Digestion of extracted plasmid DNA with the restriction enzyme BamHI, present in the pSB1AT3 backbone vector and in the reader module (BBa_K874040), ensured that both the Polydactyl Zinc Finger PZF3838 (BBa_K874001) and the reader module (BBa_K874040) were successfully cloned in as shown in Figure 12 (colony 2, 2 bands expected: 3699 and 1769 bp). Preliminary characterization of BBa_K874300 was attempted only once due to time pressure and consisted of a ScaI digestion in the absence of IPTG (figure 13). Compared to the pSB1AT3/LacH/Mtase, BBa_K874300 shows a significant switch to the non-methylated profile, illustrated by the tremendous intensity of the lower bands (cut plasmid) compared to the first one (partially cut plasmid). These results show that the presence of the PZF3838 enhances the specificity of the writer.</P><H1><SPAN class="mw-headline" id="Reference_List">Reference List</SPAN></H1><P>	1.     	Bowers,L.M., Lapoint,K., Anthony,L., Pluciennik,A., &amp; Filutowicz,M. Bacterial expression system with tightly regulated gene expression and plasmid copy number. Gene 340, 11-18 (2004).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Amsterdam/data/experimental">http://2012.igem.org/Team:Amsterdam/data/experimental</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Amsterdam/data/experimental" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Amsterdam/data/experimental" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Amsterdam/data/experimental&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Amsterdam/data/experimental&amp;oldid=238142" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>