<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Amsterdam_modeling_generaldesign"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Amsterdam/modeling/generaldesign">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Amsterdam/modeling/generaldesign&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/modeling/generaldesign&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/modeling/generaldesign&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Amsterdam/modeling/generaldesign" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Amsterdam/modeling/generaldesign</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="main"><DIV id="main-area"><DIV id="sidebar-area"><DIV class="content-block"><DIV id="navmenu"><UL><LI><A href="/Team:Amsterdam">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Amsterdam/team/members">Members</A></LI><LI><A href="/Team:Amsterdam/team/advisors">Advisors</A></LI></UL><LI><A href="/Team:Amsterdam/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Amsterdam/project/molecular_design">Molecular Design</A></LI><LI><A href="/Team:Amsterdam/project/biobricks">BioBricks</A></LI><LI><A href="/Team:Amsterdam/project/features_and_applications">Features and Applications</A></LI><LI><A href="#">Data</A></LI><LI><A href="/Team:Amsterdam/data/experimental">Experimental Results</A></LI><LI><A href="/Team:Amsterdam/data/time_inference_model">Time Inference Model</A></LI><LI><A href="/Team:Amsterdam/data/background_activity">Background activity</A></LI><LI><A href="/Team:Amsterdam/project/diary/">Logbook Designer</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/webtool">Webtool</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/setup">Setup</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/manual">Manual</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/future_perspective">Future Perspective</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Amsterdam/practices/overview">Overview</A></LI><LI><A href="/Team:Amsterdam/practices/methods">Methods</A></LI><LI><A href="/Team:Amsterdam/practices/results">Results</A></LI><LI><A href="/Team:Amsterdam/practices/conclusion">Conclusion</A></LI><LI><A href="#">Safety</A></LI><LI><A href="/Team:Amsterdam/safety/questions">Questions</A></LI><LI><A href="#">Extra</A></LI><LI><A href="/Team:Amsterdam/extra/protocols">Protocols</A></LI><LI><A href="/Team:Amsterdam/extra/diary">Lab Diary</A></LI><LI><A href="/Team:Amsterdam/extra/faq">FAQ</A></LI><LI><A href="/Team:Amsterdam/extra/software">Software</A></LI></DIV></DIV></DIV><DIV id="content-area"><DIV id="sub-menu" class="content-block"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#The_molecular_design_in_a_nutshell"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The molecular design in a nutshell</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Choosing_a_methyltransferase"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Choosing a methyltransferase</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Extending_the_idea_to_multiple_bits"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Extending the idea to multiple bits</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Extending_from_multiple_to_.7E64_bits"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Extending from multiple to ~64 bits</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Signal_strength_measurement"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Signal strength measurement</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="The_molecular_design_in_a_nutshell"> The molecular design in a nutshell </SPAN></H2><P>We established in a very early stage of the design process to use DNA methylation as the molecular mechanism to create our storage mechanism. DNA methylation, which means enriching a specific nucleotide with a methyl group, is performed by a group of proteins called the methyltransferases. In making this design choice, we were heavily influenced by the DamID technology (<A href="#Steensel1">4</A>,<A href="#Steensel2">5</A>). Van Steensel was interested in the binding sites in drosophila genomes of various eukaryotic transcription factors. By fusing a bacterial methyltransferase (MTase) to these transcription factors, he was able to infer the transcription factor binding sites by reading out which genomic regions were methylated.
</P><P>We thought we could reverse this idea to create a memory unit, at the core constituted by an MTase that would methylate an especially designated genomic region but only in reaction to the sensing of a signal by the cell. The epigenetic status of the bit region could thus either be 1 (methylated or written) <I>or</I> 0 (ummethylated or unwritten), effectively forming a binary memory unit.
</P><P>Finally, the methylation status of the bit region can be assessed using digestion of the plasmid extracts with the MTase-coupled RE, followed by analyzing the band lengths intensities of a gel electrophoresis of the product.
</P><H2><SPAN class="mw-headline" id="Choosing_a_methyltransferase"> Choosing a methyltransferase </SPAN></H2><P>To create this system, we first required an MTase that: i) is not already present in the chassis organism <I>E.coli</I>, ii) has a binding motif that is not methylated by any other MTases in the system and iii) for which a restriction enzyme (RE) dependent upon the action of this methyltransferase has been identified. This last condition is always true for any methyltransferase which has been identified to be part of a bacterial <A href="http://en.wikipedia.org/wiki/Restriction_modification_system" class="external text" rel="nofollow">Restriction/Modification system</A>.
We wrote a python script to mine the <A href="http://rebase.neb.com/rebase/rebase.html" class="external text" rel="nofollow">REBASE</A> database of REs and MTases for a MTase that meets our needs. We found an ideal candidate: M.ScaI, originally from <I>Streptomyces caespitosus</I>.
The only remaining design question was how to control the activity of the fusion protein as we would want it to only be active in the presence of the generic signal. This design question will be more elaborately discussed in the next section.
</P><H2><SPAN class="mw-headline" id="Extending_the_idea_to_multiple_bits"> Extending the idea to multiple bits </SPAN></H2><P>Extending the design idea further, we realized that it should be possible to register and store the presence of multiple signals smultaneously in a single cell. This extension can be accomplished by fusing DNA binding proteins to the methyltransferase and adding their corresponding DNA binding motifs to the bit regions of the DNA. Either traditional bacterial transcriptional regulators could be used for this purpose or the Zinc-Finger Array (ZFA) technology (<A href="#Fu">1</A>). ZFA allows the construction of highly specific artificial protein-DNA interactions (<A href="#Kaseniit">2</A>). 
</P><P>By fusing the MTase to a Zinc Finger or traditional transcriptional regulator and extending the bit region with the transcriptional regulator’s binding motif, the binding affinity of the fusion protein for the DNA motif could increase about 18-fold (<A href="#McNamara">3</A>) (a C5-cytosine MTase was studied here, not in the class of N4C-MTases to which M.ScaI belongs). This will heavily diminish the possible aspecific cross-talk between FPs and bits assigned to other FPs. 
</P><P>The wide range of available Zinc Fingers could also allows for the usage of the same MTase for multiple signal. This assumes that the site specificity of the Zinc Finger Array is much higher than the site specifity of the MTase, which is very plausible (<A href="#McNamara">3</A>).
In this setup there would be a very low chance of cross-talk between FPs and other signals’ bits.
</P><P>The ZF-binding sites and methylation motif groups are distributed over the plasmid in a way that the distances between the TF-binding sites are unique: for all binding restriction sites &lt;math&gt;i&lt;/math&gt; the intermediate genomic distance with the next bit, $D(i,i+1)$, has to be: 
$$
D(i,i+1) = \frac{D(i-1,i)}{2}
$$
By designing the plasmid to adhere to this rule, a unique pattern of fragment lengths between each of the TF binding sites - or bits - will result for each combination of bit statuses. 
One plasmid will then be able to hold 
&lt;math&gt;
\log_2 (\text{Plasmid BP length}) - \log_2 (256)
&lt;/math&gt; bits
with the current achievable resolution in gel electrophoresis techniques (resolution down to 300 bp and &lt;math&gt;2^{7} = 256&lt;/math&gt;).
For the typical plasmid base pair length of &lt;math&gt;6 \cdot 10^{4}&lt;/math&gt; nucleotides this is &lt;math&gt;8\ \text{bits}&lt;/math&gt;.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Multiplebits.png" title="File:Multiplebits.png">File:Multiplebits.png</A><DIV class="thumbcaption">A plasmid storing three bits in which site 0 is always cleaved. Notice the unique distances between bits 1, 2 and 3. The value of each bit can be inferred from the unique band lengths pattern after digestion</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Extending_from_multiple_to_.7E64_bits"> Extending from multiple to ~64 bits </SPAN></H2><P>The amount of bits stored on a single plasmid could be enlarged even more. In the previous section all bits used the same MTase and are digested using the same restriction enzyme with the potential to store up to 8 bits. By finding a new MTase-RE combination and engineering the memory plasmid to have bits for this new combination slightly offset from the original MTase-RE binding sites, 8 additional bits will be storable on the plasmid. This requires that during the read-out step, digestions are performed for each RE individually. It is essential that only a single RE-type is for each individual plasmid extract to retain the ability to read out the bits from the unique band lengths. For each RE used, a separate band on the final gel must be used and determination of bit values will proceed in the same way as for single MTase memory modules. A plasmid bp-size of 60k will allow for about 8 different RE’s and thus:
</P><P>$$
8\ [\text{RE}] \cdot [\frac{8\ \text{bits}}{\text{RE}}] = 64\ \text{bits}
$$
</P><H2><SPAN class="mw-headline" id="Signal_strength_measurement"> Signal strength measurement </SPAN></H2><P>Binary encoding of a signal's presence is useful, but often in biology gaining insight in the quantity of a signal is even more relevant. By coupling different promoters with different strengths as responses to the same signal and assigning each promoter its own unique bit, the range in which the strength of the signal is located can be measured.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P><SPAN id="Fu"><SUP>Fu, Fengli, Jeffry D. Sander, Morgan Maeder, Stacey Thibodeau-Beganny, J. Keith Joung, Drena Dobbs, Leslie Miller, and Daniel F. Voytas. 2009. “Zinc Finger Database (ZiFDB): a repository for information on C2H2 zinc fingers and engineered zinc-finger arrays.” <I>Nucleic acids research</I> 37 (jan): 279. doi:10.1093/nar/gkn606. [<A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2686427\&amp;tool=pmcentrez\&amp;rendertype=abstract|http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2686427\&amp;tool=pmcentrez\&amp;rendertype=abstract" class="external autonumber" rel="nofollow">[1]</A>]</SUP></SPAN></P><P><SPAN id="Kaseniit"><SUP>
Kaseniit, K. E., Perli, S. D., &amp; Lu, T. K. (2011). Designing extensible protein-DNA interactions for synthetic biology. 2011 IEEE Biomedical Circuits and Systems Conference (BioCAS), 349–352. doi:10.1109/BioCAS.2011.6107799
</SUP></SPAN></P><P><SPAN id="McNamara"><SUP>McNamara, A. R., Hurd, P. J., Smith, A. E. F., &amp; Ford, K. G. (2002). Characterisation of site-biased DNA methyltransferases: specificity, affinity and subsite relationships. Nucleic acids research, 30(17), 3818–30. Retrieved from <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=137423&amp;tool=pmcentrez&amp;rendertype=abstract" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=137423&amp;tool=pmcentrez&amp;rendertype=abstract</A></SUP></SPAN></P><P><SPAN id="Steensel1"><SUP>
van Steensel, B., Delrow, J., &amp; Henikoff, S. (2001). Chromatin profiling using targeted DNA adenine methyltransferase. Nature genetics, 27(3), 304–8. doi:10.1038/85871
&lt;/span&gt;</SUP></SPAN></P><P><SPAN id="Steensel2"><SUP>
van Steensel, B., &amp; Henikoff, S. (2000). Identification of in vivo DNA targets of chromatin proteins using tethered dam methyltransferase. Nature biotechnology, 18(4), 424–8. doi:10.1038/74487
</SUP></SPAN></P></DIV><DIV id="sub-menu" class="content-block" align="center"><P><B>Click the arrows to go to the previous or next page</B></P></DIV><P>&lt;/div&gt;</P></DIV></DIV>&lt;/html&gt;</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Amsterdam/modeling/generaldesign">http://2012.igem.org/Team:Amsterdam/modeling/generaldesign</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Amsterdam/modeling/generaldesign" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Amsterdam/modeling/generaldesign" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Amsterdam/modeling/generaldesign&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Amsterdam/modeling/generaldesign&amp;oldid=177946" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>