<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Amsterdam_project_lab_setbacks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Amsterdam/project/lab_setbacks/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Amsterdam/project/lab_setbacks/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/project/lab_setbacks/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/project/lab_setbacks/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Amsterdam/project/lab_setbacks/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Amsterdam/project/lab setbacks/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="main"><DIV id="main-area"><DIV id="sidebar-area"><DIV class="content-block"><DIV id="navmenu"><UL><LI><A href="/Team:Amsterdam">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Amsterdam/team/members">Members</A></LI><LI><A href="/Team:Amsterdam/team/advisors">Advisors</A></LI></UL><LI><A href="/Team:Amsterdam/achievements">Achievements</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Amsterdam/project/molecular_design">Molecular Design</A></LI><LI><A href="/Team:Amsterdam/project/biobricks">BioBricks</A></LI><LI><A href="/Team:Amsterdam/project/features_and_applications">Features and Applications</A></LI><LI><A href="#">Data</A></LI><LI><A href="/Team:Amsterdam/data/experimental">Experimental Results</A></LI><LI><A href="/Team:Amsterdam/data/time_inference_model">Time Inference Model</A></LI><LI><A href="/Team:Amsterdam/data/background_activity">Background activity</A></LI><LI><A href="/Team:Amsterdam/project/diary/">Logbook Designer</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/webtool">Webtool</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/setup">Setup</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/manual">Manual</A></LI><LI><A href="/Team:Amsterdam/software/logbook_designer/future_perspective">Future Perspective</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Amsterdam/practices/overview">Overview</A></LI><LI><A href="/Team:Amsterdam/practices/methods">Methods</A></LI><LI><A href="/Team:Amsterdam/practices/results">Results</A></LI><LI><A href="/Team:Amsterdam/practices/conclusion">Conclusion</A></LI><LI><A href="#">Safety</A></LI><LI><A href="/Team:Amsterdam/safety/questions">Questions</A></LI><LI><A href="#">Extra</A></LI><LI><A href="/Team:Amsterdam/extra/protocols">Protocols</A></LI><LI><A href="/Team:Amsterdam/extra/diary">Lab Diary</A></LI><LI><A href="/Team:Amsterdam/extra/faq">FAQ</A></LI><LI><A href="/Team:Amsterdam/extra/software">Software</A></LI></DIV></DIV></DIV><DIV id="content-area"><DIV id="sub-menu" class="content-block" align="center"><H1><SPAN class="mw-headline" id="Lab_setbacks">Lab setbacks</SPAN></H1><P>The lab life of any researcher is a series of unavoidable experimental flaws, setbacks and incessant trials and errors. Truth is, even though one executes scrupulously experimental protocols, the outcome will occasionally be void results. Troubleshooting and optimisation are the keys to a successful project.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Lab_setbacks"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Lab setbacks</SPAN></A></LI><LI class="toclevel-2"><A href="#Primer_Design_.26_PCR"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Primer Design &amp; PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#To_silently_mutate_it_out.2C_or_not.3F"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">To silently mutate it out, or not?</SPAN></A></LI><LI class="toclevel-2"><A href="#Competency_lost"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Competency lost</SPAN></A></LI><LI class="toclevel-2"><A href="#Counteracting_the_leakiness"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Counteracting the leakiness</SPAN></A></LI><LI class="toclevel-2"><A href="#Reference_List"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Reference List</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Primer_Design_.26_PCR">Primer Design &amp; PCR</SPAN></H2><P>The Cellular Logbook project implied numerous experimental challenges. Brainstorming sessions led to the idea that several intermediary constructs should be designed to enable better characterisation of the biobrick. Hence, the project required a great deal of cloning. Since the classical digestion–gel extraction–ligation strategy is known for its inconsistency, we opted for the one step-isothermal Gibson assembly method (GM).1 A polymerase chain reaction (PCR) had to be performed prior to the GM, leading to the first setback; primer design. GM requires specifically optimised primers with overlapping sequences between the vector and the insert, therefore, adding constraints to the choice of primers. Great care was taken in designing highly specific primers with appropriate GC content and melting temperature. However, in combination with the Phusion® High-Fidelity polymerase, the PCR resulted in a heavy background noise, with the product of interest present in only small quantities. Optimisation of the PCR was attempted repeatedly using different PCR conditions, primer dilutions, addition of a PCR enhancer mix2, but we were eventually able to reduce the background only to some extent. Amplification of the polydactyl Zinc Finger (PZF) using overlapping-ends primers was the most difficult part of the project. Since the PZF contains a series of repeated motifs, the primers were designed in such a way that two-third of the primer would anneal outside of the PZF while the rest annealed inside the PZF sequence. This choice was made to reduce the probability of the primer annealing at different places inside the PZF sequence, but also to prevent insertion of too many nucleotides to the linker, thus hindering folding of the fusion protein. Finally, we opted for gel extraction of the appropriate-sized fragment, even though this approach should be avoided according to Daniel Gibson1. Any GM attempt with impure PCR products failed. It took about two months to obtain a pure PCR fragment for the PZF. Ultimately, we succeeded in cloning the PZF into order to make our final construct.
</P><H2><SPAN class="mw-headline" id="To_silently_mutate_it_out.2C_or_not.3F">To silently mutate it out, or not?</SPAN></H2><P>A ScaI restriction site was accidentally introduced during synthesis of the Myc linker, methyltransferase M.ScaI and double terminator (BBa_B0014). Since the number of ScaI restriction sites is vital for the development of the methylation-based system, it was decided to mutate this site out using site-directed mutagenesis (Agilent Technologies). However, due to time pressure, most of the experiments were conducted using the non-mutated constructs. For this reason, both the mutated and the non-mutated versions of our constructs were submitted to the iGEM headquarters.
</P><H2><SPAN class="mw-headline" id="Competency_lost">Competency lost</SPAN></H2><P>Another setback experienced during this project was the fact that the bacterial strain ordered for the transformation of our clones lost their competency during transportation. Considering the limited amount of time that we had for this project, a couple of weeks of waiting for the new bacterial strain to arrive can cause a consequent delay in the flow of experiments.
</P><H2><SPAN class="mw-headline" id="Counteracting_the_leakiness">Counteracting the leakiness</SPAN></H2><P>It was found during the course of the project that the promoter chosen, the lacI regulated, lambda pL hybrid (BBa_R0011), showed leakiness. When it comes to methylation, even low expression will cause the sites to be methylated. Therefore, it was imperative to optimise the system and reduce the leakiness of the promoter. As a result, the LacIQ bacterial strain was chosen to achieve this goal, however unsuccessfully. Switching to the pBAD weak promoter (BBa_K206001) was not helpful either. Consequently, an idea emerged that the copy number of the plasmids used might have a strong influence on the expression of the methyltransferase. This option involved more cloning steps which proved to be hassle but we finally managed to clone only the final construct involving the LacH promoter, PZF, methyltransferase, memory part into the pSB3C5 (low to medium copy number plasmid) and pSB4C5 (low copy number plasmid).
</P><P>“Success is stumbling from failure to failure with no loss of enthusiasm”
</P><PRE>– Winston S. Churchill
</PRE><P>Given the amount of setbacks encountered, the satisfaction of eventually succeeding most of the cloning steps and proving our concept was delightful. This experience enabled us to develop scientific creativity, troubleshooting skills and critical thinking.	
</P><H2><SPAN class="mw-headline" id="Reference_List">Reference List</SPAN></H2><P>	1.     	Gibson,D.G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nat. Methods 6, 343-345 (2009).
	2.     	Ralser,M. et al. An efficient and economic enhancer mix for PCR. Biochem. Biophys. Res. Commun. 347, 747-751 (2006).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Amsterdam/project/lab_setbacks/">http://2012.igem.org/Team:Amsterdam/project/lab_setbacks/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Amsterdam/project/lab_setbacks/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Amsterdam/project/lab_setbacks/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Amsterdam/project/lab_setbacks/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Amsterdam/project/lab_setbacks/&amp;oldid=157257" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>