"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Amsterdam/project/biobricks
From 2012.igem.org
Home
Team
Members
Advisors
Achievements
Project
Molecular Design
BioBricks
Features and Applications
Data
Experimental Results
Time Inference Model
Background activity
Logbook Designer
Webtool
Setup
Manual
Future Perspective
Human Practices
Overview
Methods
Results
Conclusion
Safety
Questions
Extra
Protocols
Lab Diary
FAQ
Software
BioBricks
<groupparts>iGEM12 Amsterdam</groupparts>
Favourites
M.ScaI Methyltransferase BBa_K874000
This part codes for the M.ScaI methyltransferase protein. M.ScaI is a type II methyltransferase (subtype beta) that recognizes site on the DNA of the following sequence 5..AGTACT..3. It methylates this site at the 5th (Cytosine) nucleotide leaving an N4-methylcytosine (m4). This methylation type (m4) is not found in native E. coli nor is the recognition site methylated by any of E. coli's native methylation systems (Dam, Dcm). Also this specific methylation inhibits restriction by M.ScaI's prototype restriction enzyme ( ScaI ).
Data-mining of the REBASE (m4) methyltransferase database revealed that M.ScaI was the best candidate based on the following parameters:
Methylation (m4) done by this M.ScaI inhibits its prototype (ScaI) restriction enzyme ability to restrict the site
E. coli's native methylation systems do not methylate the recognition site and thus can not interfere with systems using M.ScaI
E. coli's native restriction systems do not restrict the recognition site (in either methylated and unmethylated form) and thus can not interfere with systems using M.ScaI
Recognition site has high specificity (1 in 4048 random sequences)
Its prototype restriction enzyme is commercially available
Our experiments show that M.ScaI functions as expected in E. coli by showing that it is indeed able to methylate. However it might be worth noting that the protein is natively found in Streptomyces caespitosus which is a bacteria that has an optimal growth temperature of 26C and thus might not be expressed optimally in E. coli.
IPTG inducible expression of M.ScaI methyltransferase
(IPTG -> M.ScaI) BBa_K874100
This BioBrick contains the first proof of concept and contains the both the Reader and the Sensor described in our molecular design. Therefore this is also the most extensively studied BioBrick in our project, for an in dept view of the experiments we performed using this BioBrick you can look at the Experimental Setup section. We managed to insert this BioBrick in both the pSB1AT3 and pSB1C3 backbones but all testing was done in pSB1AT3.
Arabinose inducible expression of M.ScaI methyltransferase
(ARA -> M.ScaI) BBa_K874101
After the assessment of the first BioBrick we deemed it nessesary to change te promotor to the pBAD (Arabinose) promoter. This therefore is still the first proof of concept but contains a different Reader as described in our molecular design. This BioBrick was also extensively studied so for an in dept view of the experiments we performed using this BioBrick you can look at the Experimental Setup section. We managed to insert this BioBrick in both the pSB1AT3 and pSB1C3 backbones but all testing was done in pSB1AT3.
Important
Polydactyl Zinc Finger (PZF3838) BBa_K874001
This part codes for the 3838 Polydactyl Zinc Finger (PZF). It consists of 6 individual zinc fingers that together bind to a specific 18pb DNA sequence.
The actual binding sequence is 5..GGGGCCGGAGCCGCAGTG..3 and can be broken to the following codons per zinc finger:
Zinc Finger 1: 5..GGG..3
Zinc Finger 2: 5..GCC..3
Zinc Finger 3: 5..GGA..3
Zinc Finger 4: 5..GCC..3
Zinc Finger 5: 5..GCA..3
Zinc Finger 6: 5..GTG..3
This PZF was chosen because many of the PZF's we found where listed to be partial working or not verified to be working. This seems to be a common problem with PZF's which finnaly led us to enlist the help from Sylvia de Pater in obtaining a working PZF.
This PZF was designed in research done by Bert van der Zaal, Paul Hooykaas and Sylvia de Pater in order to recognise the E2C transcription factor binding site in Arabidopsis. [1] It has thus been tested and verify to fold and function in Arabidopsis.
A indication that the PZF functions properly in E. coli (LacIq) is also provided by the iGEM Amsterdam 2012 project. More information can be found on its verification on the BBa_K874200 part page.
On a final note! Since the individual Zinc Fingers in this PZF (or any PZF for that matter) are highly similar in sequence we strongly recommender that when doing PCR primers are used that ad-heal outside of the actual PZF..
Retrieved from " http://2012.igem.org/Team:Amsterdam/project/biobricks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
