<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFlorida_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFlorida/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UFlorida"><DIV class="menu_item direct_link">
				HOME
			</DIV><DIV class="menu_item">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Protocols"><DIV class="submenu_item">
					Protocols
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A></DIV><DIV class="menu_item">
			RESULTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="topnav" style="display: flex; align-items: flex-end; background-color: #333333; position: sticky; top: 18px;"><DIV class="dropdown"><A class="topnav" href="https://2018.igem.org/Team:UFlorida">Home</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Team">Members</A><A href="https://2018.igem.org/Team:UFlorida/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UFlorida/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Description">Description</A><A href="https://2018.igem.org/Team:UFlorida/Design">Design</A><A href="https://2018.igem.org/Team:UFlorida/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UFlorida/Protocols">Protocols</A><A href="https://2018.igem.org/Team:UFlorida/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UFlorida/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UFlorida/Model">Model</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UFlorida/Parts">Parts</A><A href="https://2018.igem.org/Team:UFlorida/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Engagement</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UFlorida/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV id="#main" class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Experiments</H1><DIV class="column full_size"><H4>HPLC Prep</H4><OL><LI> Transfer 1 mL of culture to 1.5 (or 2) mL microfuge tube.</LI><LI> Centrifuge at full speed for 90 seconds.</LI><LI> Pour culture supernatant into 3 mL syringe with 0.2 uM filter attached to bottom.</LI><LI> Collect the filtrate in a 2 mL HPLC-vial.</LI><LI> Add 10 uL of 1M H2SO4 to the filtered sample.</LI><LI> Seal vial, load auto-sampler for HPLC, then run HPLC.</LI></OL></DIV><DIV class="column full_size"><H4>Protocol for Rotating Samples(LB +  PO4 + Glucose)</H4><OL><LI>Add 0.15 mL of Glucose (30%) </LI><LI>Add 0.4 mL of PO4 (1M) </LI><LI>Add 3.5 mL of LB to 9 mL tube. </LI><LI>Inoculate with 10 uL of culture or from plated culture. </LI><LI>Rotate in 37°C environment for 24 hours. </LI></OL><DIV class="column full_size"><H4>Protocol for Standing Samples (LB+ Glucose) </H4><OL><LI>Add 0.3 mL of Glucose (30%)  </LI><LI>Fill rest of 9 mL tube with LB. </LI><LI>Inoculate with 10 uL of liquid culture or from plated culture. </LI><LI>Place in 37°C environment for 24 hours. </LI></OL></DIV><DIV class="column full_size"><H4>Protocol for Standing Samples (LB+PO4+ Arabinose or Glucose)</H4><P>*Arabinose was used when fermenting cells with Butyrate-producing parts*</P><P>*Glucose was used for fermenting cells with non-Butyrate-producing parts*</P><OL><LI>Add 0.3 mL of Arabinose (30%) or Glucose (30%)</LI><LI>Fill rest of 9 mL tube with LB</LI><LI>Add 0.9 mL of PO4 (1M)</LI><LI>Inoculate with 10 uL of liquid culture or from plated culture</LI><LI>Place in 37°C environment for 24 hours. </LI></OL></DIV><DIV class="column full_size"><H4>Protocol for Standing Samples (LB+PO4+ Arabinose or Glucose)</H4><OL><LI>Add 0.3 mL of Arabinose (30%) or Glucose (30%)</LI><LI>Fill rest of 9 mL tube with LB</LI><LI>Add 0.9 mL of PO4 (1M)</LI><LI>Inoculate with 10 uL of liquid culture or from plated culture</LI><LI>Place in 37°C environment for 24 hours. </LI></OL></DIV><DIV class="column full_size"><H4>Plating transformed cells on LB </H4><OL><LI>Add 100uL of LB broth in Eppendorf tube and add 25uL of the antibiotic into it. Mix with pipette. </LI><LI>Add mixing beads to the plate in the hood. Add the 125uL mixture of antibiotics and then shake. Dump the beads out. </LI><LI>Let it dry for about 10 minutes before plating bacteria. </LI><LI>Other antibiotics can be added similarly on the plate. </LI><LI>Bacteria can be plated in two ways: Streaking or shaking with beads: 
</LI><LI>If streaking (usually from glycerol stocks or making a pure culture from another plate, take a stick/inoculating loop and pick the colony. Streak onto the plate. </LI><LI>For plating from the electroporated culture prepared after transformation, add the beads onto the plate in the hood. Add the culture and spread by shaking the plate. Dump the beads out and put the plate in incubator. </LI><LI>Incubate overnight at 30°C for Nissle with Oligos
transformed or the pKD.
The temperature of incubation depends on the bacteria and the plasmid in it. Some can be kept at 37°C. Make sure
plates are kept in the correct incubator. </LI></OL></DIV><DIV class="column full_size"><H4>Fermentation Prep Protocol </H4><OL><LI>Measure OD of cultures (O/N) </LI><LI>Calculate amount of culture needed for inoculation (.05)*(250)=(culture cell density(O.D.))*(wanted volume of culture for inoculation) </LI><LI>In a 250 mL grad. cylinder: Add 25 mL Glucose, calculated amount of culture, and fill the rest with LB</LI></OL></DIV><DIV class="column full_size"><H4>No-SCAR </H4><OL><LI> Inoculate Cells and grow to an OD of 0.4-0.5</LI><LI> Cells were chilled in ice for at least 10 minutes. </LI><LI> Make competent cells</LI><LI> Load electroporation cuvette with 1 uL DNA (pCas9-CR4) and 50 uL of cells (Nissle), shake, place in electroporator. 5.5 milliseconds, 1.8kv. </LI><LI> Incubate at 37 C for 1 hour. </LI><LI> Plate onto chloramphenicol treated agar. </LI><LI> Incubate 37 C overnight. </LI><LI> Inoculate Cells grown overnight and  and grow to an OD of 0.4-0.5</LI><LI> Make competent cells</LI><LI> Load electroporation cuvette with 1 uL DNA (pKDsgRNA) and 50 uL of cells (Nissle_pCas9-Cr4), shake and place in electroporator</LI><LI> Incubate both at 30 C for 1 hour. </LI><LI> Plate onto chloramphenicol and spectinomycin treated agar. </LI><LI> Incubate both 30 C overnight. </LI><LI> Grow cells and induce with arabinose to activate lambda red system</LI><LI> Transform the overnight grown cells with desired oligos</LI><LI> Cure the plasmid of the pKDsgRNA</LI><LI> Transform with another desired pKDsgRNA and repeat steps 8-19 until desired deletions are attained</LI></OL></DIV><DIV class="column full_size"><H4>FimH Binding Assay</H4><OL><LI>Caco-2 cells were grown at 37oC, 5% CO2 in EMEM with 20% FBS as recommended by ATCC.</LI><LI>Nissle 1917 cells were induced by centrifuging and re-suspending in HBSS buffer conditioned in 0.01% arabinose and 0.5% rhamnose at 37C.</LI><LI> Caco-2 cells were infected with Nissle 1917 at a multiplicity of infection (MOI) of 5,000 (5,000 bacteria/Caco-2 cell)</LI><LI>These were then suspended in slides and visualized using an Olympus IX70 fluorescence microscope.</LI></OL></DIV><DIV class="column full_size"><H4>Miniprep (Qiagen Kit Protocol) </H4><OL><LI>Pellet 1–5 mL bacterial overnight culture by centrifugation at &gt;8000 rpm (6800 x g) for 3 min at room temperature (15–25°C). </LI><LI>Resuspend pelleted bacterial cells in 250μL Buffer P1 and transfer to a microcentrifuge tube. </LI><LI>Add 250μl Buffer P2 and mix thoroughly by inverting the tube 4–6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue. </LI><LI>Add 350μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, the solution will turn colorless. </LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. </LI><LI>Apply 800 μl supernatant from step 5 to the QIAprep 2.0 spin column by pipetting. For centrifuge processing, follow the instructions marked with a triangle (). For vacuum manifold processing, follow the instructions marked with a circle ().  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source. </LI><LI>Recommended: Wash the QIAprep 2.0 spin column by adding 0.5 ml Buffer PB.  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source. </LI><LI>Wash the QIAprep 2.0 spin column by adding 0.75 ml Buffer PE.  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source. Transfer the QIAprep 2.0 spin column to the collection tube. </LI><LI>Centrifuge for 1 min to remove residual wash buffer. </LI><LI>Place the QIAprep 2.0 column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAprep 2.0 spin column, let stand for 1 min, and centrifuge for 1 min.</LI></OL></DIV><DIV class="column full_size"><H4>Plating Cells </H4><OL><LI>Plating with beads</LI><LI> Perform under the hood or near a flame</LI><LI>Pipette ~200uL of liquid bacterial cell suspension onto the
desired plate</LI><LI>Pour a few sterilized glass beads onto the plate </LI><LI>Close the lid and shake the plate in the shape of a star in
the air</LI><LI>Open the plate and pour the beads into a waste bucket and
close the lid. </LI></OL></DIV><DIV class="column full_size"><H4>Making cells chemically competent</H4><OL><LI>Pellet the whole sample with cells until there is no more media. There should be a large pellet at the bottom of the Eppendorf tube. </LI><LI>Put 500 microliters of calcium chloride in the Eppendorf tube and spin down. </LI><LI>Decant the CaCl2 and resuspend in 100 microliters of CaCl2</LI><LI>Heat shock cells + plasmid at 42C for 55 seconds. (Put in heat block in cuvettes with water.) This should be a rapid transition. </LI><LI>Place back on ice for a few minutes</LI><LI>Add 1mL of SOC or SOB. Place Eppendorf tube inside long glass tube and incubate at 30 degrees C for 1 hour in spinning rack. </LI><LI>Rest of solutions kept on bench. </LI></OL></DIV><DIV class="column full_size"><H4>Transformation </H4><OL><LI>Load electroporation cuvette with 1 uL DNA  and 50 uL of cells to be transformed shake, place in electroporator.</LI><LI>Zap and check for 5.5 milliseconds and 1.8kv. </LI><LI>Immediately replenish the cells in 1uL of SOC and incubate
overnight in respective temperature. </LI></OL></DIV><DIV class="column full_size"><H4>Datsenko-Wanner Method of genome deletions </H4><OL><LI>PCR amplify the FRT flanked resistance gene using appropriate primers. Verify PCR cycle conditions</LI><LI>Transform strain expressing lambda red system</LI><LI>Select antibiotic resistant transformants that grow up on antibiotic plates overnight (screening)</LI><LI>Eliminate resistance cassette using an FLP flanked expression plasmid</LI></OL></DIV><DIV class="column full_size"><H4>3A Assembly </H4><OL><LI>Adapted the protocol from iGEM’s website</LI><LI>Miniprep the two part samples you desire to assemble together (in BioBrick RFC[10] plasmid backbones)</LI><LI>Digestion:
</LI><LI>Use ~ 100ng total for each reaction tube&gt;
</LI><LI>Add equal volume (4  uL) linearized plasmid backbone to Enzyme Master Mix. (5 ul NEB Buffer 2, 0.5 ul BSA, 0.5 ul EcoRI-HF, 0.5 ul PstI, 0.5 ul DpnI, 18 ul dH20) *linear plasmid backbone must have separate antibiotic resistance to plasmids with parts*</LI><LI>Add equal volume (4  uL) Part #1 to Enzyme Master Mix (5 ul NEB Buffer 2, 0.5 ul BSA,0.5 ul EcoRI-HF,0.5 ul SpeI,18.5 ul dH20</LI><LI>Add equal volume (4  uL) Part #2 to Enzyme Master Mix (5 ul NEB Buffer 2, 0.5 ul BSA, 0.5 ul XbaI, 0.5 ul PstI, 18.5 ul dH20</LI><LI>Digest all three reactions at 37C for 30 min, heat kill 80C for 20 min</LI><LI> Ligation:
</LI><LI>Combine 2 ul of digested plasmid backbone, 2 ul digested Part #1 digested, 2 ul digested part #2.</LI><LI>Add 1 ul T4 DNA ligase buffer.</LI><LI>Add 0.5 ul T4 DNA ligase</LI><LI>Add water to 10 ul</LI><LI>Ligate 16C for 30 min, heat kill 80C for 20 min</LI><LI>Transform with 1-2 uL of product following listed protocol</LI></OL></DIV><DIV class="column full_size"><H4>PCR Screening </H4><OL><LI>Pick a colony from plate with sterile techniques. Choose one colony per collection tube, being careful not to cross contaminate two colonies.</LI><LI> Resuspend each colony in 50 µL of sterile DI</LI><LI>Add 1 µL of suspension template to 49 µL of PCR Master Mix (depends on kit)</LI><LI>Vortex briefly, run PCR temperature program based off insert specific primer instructions</LI><LI>Run a gel of PCR products.</LI><LI>Choose a sample with the proper insert size and use the remaining sample to streak the bacteria onto a plate with the appropriate antibiotic resistance. Place the plate in the
37 C overnight.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>