Team:UFlorida/Protocols
HOME
TEAM
Team Members
Collaborations
Attributions
PROJECT
Description
Design
Experiments
Protocols
Notebook
InterLab
Model
Results
Improve
RESULTS
Parts Overview
Demonstrate
HUMAN PRACTICES
Human Practices
Education & Engagement
Home
Team
Members Collaborations Attributions
Project
Description Design Experiments Protocols Notebook InterLab Model
Results
Demonstrate Parts Results
Engagement
Human Practices Public Engagement Entrepreneurship
Experiments
HPLC Prep
Transfer 1 mL of culture to 1.5 (or 2) mL microfuge tube.
Centrifuge at full speed for 90 seconds.
Pour culture supernatant into 3 mL syringe with 0.2 uM filter attached to bottom.
Collect the filtrate in a 2 mL HPLC-vial.
Add 10 uL of 1M H2SO4 to the filtered sample.
Seal vial, load auto-sampler for HPLC, then run HPLC.
Protocol for Rotating Samples(LB +  PO4 + Glucose)
Add 0.15 mL of Glucose (30%)
Add 0.4 mL of PO4 (1M)
Add 3.5 mL of LB to 9 mL tube.
Inoculate with 10 uL of culture or from plated culture.
Rotate in 37°C environment for 24 hours.
Protocol for Standing Samples (LB+ Glucose)
Add 0.3 mL of Glucose (30%)
Fill rest of 9 mL tube with LB.
Inoculate with 10 uL of liquid culture or from plated culture.
Place in 37°C environment for 24 hours.
Protocol for Standing Samples (LB+PO4+ Arabinose or Glucose)
*Arabinose was used when fermenting cells with Butyrate-producing parts*
*Glucose was used for fermenting cells with non-Butyrate-producing parts*
Add 0.3 mL of Arabinose (30%) or Glucose (30%)
Fill rest of 9 mL tube with LB
Add 0.9 mL of PO4 (1M)
Inoculate with 10 uL of liquid culture or from plated culture
Place in 37°C environment for 24 hours.
Protocol for Standing Samples (LB+PO4+ Arabinose or Glucose)
Add 0.3 mL of Arabinose (30%) or Glucose (30%)
Fill rest of 9 mL tube with LB
Add 0.9 mL of PO4 (1M)
Inoculate with 10 uL of liquid culture or from plated culture
Place in 37°C environment for 24 hours.
Plating transformed cells on LB
Add 100uL of LB broth in Eppendorf tube and add 25uL of the antibiotic into it. Mix with pipette.
Add mixing beads to the plate in the hood. Add the 125uL mixture of antibiotics and then shake. Dump the beads out.
Let it dry for about 10 minutes before plating bacteria.
Other antibiotics can be added similarly on the plate.
Bacteria can be plated in two ways: Streaking or shaking with beads:
If streaking (usually from glycerol stocks or making a pure culture from another plate, take a stick/inoculating loop and pick the colony. Streak onto the plate.
For plating from the electroporated culture prepared after transformation, add the beads onto the plate in the hood. Add the culture and spread by shaking the plate. Dump the beads out and put the plate in incubator.
Incubate overnight at 30°C for Nissle with Oligos transformed or the pKD. The temperature of incubation depends on the bacteria and the plasmid in it. Some can be kept at 37°C. Make sure plates are kept in the correct incubator.
Fermentation Prep Protocol
Measure OD of cultures (O/N)
Calculate amount of culture needed for inoculation (.05)*(250)=(culture cell density(O.D.))*(wanted volume of culture for inoculation)
In a 250 mL grad. cylinder: Add 25 mL Glucose, calculated amount of culture, and fill the rest with LB
No-SCAR
Inoculate Cells and grow to an OD of 0.4-0.5
Cells were chilled in ice for at least 10 minutes.
Make competent cells
Load electroporation cuvette with 1 uL DNA (pCas9-CR4) and 50 uL of cells (Nissle), shake, place in electroporator. 5.5 milliseconds, 1.8kv.
Incubate at 37 C for 1 hour.
Plate onto chloramphenicol treated agar.
Incubate 37 C overnight.
Inoculate Cells grown overnight and  and grow to an OD of 0.4-0.5
Make competent cells
Load electroporation cuvette with 1 uL DNA (pKDsgRNA) and 50 uL of cells (Nissle_pCas9-Cr4), shake and place in electroporator
Incubate both at 30 C for 1 hour.
Plate onto chloramphenicol and spectinomycin treated agar.
Incubate both 30 C overnight.
Grow cells and induce with arabinose to activate lambda red system
Transform the overnight grown cells with desired oligos
Cure the plasmid of the pKDsgRNA
Transform with another desired pKDsgRNA and repeat steps 8-19 until desired deletions are attained
FimH Binding Assay
Caco-2 cells were grown at 37oC, 5% CO2 in EMEM with 20% FBS as recommended by ATCC.
Nissle 1917 cells were induced by centrifuging and re-suspending in HBSS buffer conditioned in 0.01% arabinose and 0.5% rhamnose at 37C.
Caco-2 cells were infected with Nissle 1917 at a multiplicity of infection (MOI) of 5,000 (5,000 bacteria/Caco-2 cell)
These were then suspended in slides and visualized using an Olympus IX70 fluorescence microscope.
Miniprep (Qiagen Kit Protocol)
Pellet 1–5 mL bacterial overnight culture by centrifugation at >8000 rpm (6800 x g) for 3 min at room temperature (15–25°C).
Resuspend pelleted bacterial cells in 250μL Buffer P1 and transfer to a microcentrifuge tube.
Add 250μl Buffer P2 and mix thoroughly by inverting the tube 4–6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.
Add 350μl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
Apply 800 μl supernatant from step 5 to the QIAprep 2.0 spin column by pipetting. For centrifuge processing, follow the instructions marked with a triangle (). For vacuum manifold processing, follow the instructions marked with a circle ().  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source.
Recommended: Wash the QIAprep 2.0 spin column by adding 0.5 ml Buffer PB.  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source.
Wash the QIAprep 2.0 spin column by adding 0.75 ml Buffer PE.  Centrifuge for 30–60 s and discard the flow-through, or  apply vacuum to the manifold to draw the solution through the QIAprep 2.0 spin column and switch off the vacuum source. Transfer the QIAprep 2.0 spin column to the collection tube.
Centrifuge for 1 min to remove residual wash buffer.
Place the QIAprep 2.0 column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM TrisCl, pH 8.5) or water to the center of the QIAprep 2.0 spin column, let stand for 1 min, and centrifuge for 1 min.
Plating Cells
Plating with beads
Perform under the hood or near a flame
Pipette ~200uL of liquid bacterial cell suspension onto the desired plate
Pour a few sterilized glass beads onto the plate
Close the lid and shake the plate in the shape of a star in the air
Open the plate and pour the beads into a waste bucket and close the lid.
Making cells chemically competent
Pellet the whole sample with cells until there is no more media. There should be a large pellet at the bottom of the Eppendorf tube.
Put 500 microliters of calcium chloride in the Eppendorf tube and spin down.
Decant the CaCl2 and resuspend in 100 microliters of CaCl2
Heat shock cells + plasmid at 42C for 55 seconds. (Put in heat block in cuvettes with water.) This should be a rapid transition.
Place back on ice for a few minutes
Add 1mL of SOC or SOB. Place Eppendorf tube inside long glass tube and incubate at 30 degrees C for 1 hour in spinning rack.
Rest of solutions kept on bench.
Transformation
Load electroporation cuvette with 1 uL DNA  and 50 uL of cells to be transformed shake, place in electroporator.
Zap and check for 5.5 milliseconds and 1.8kv.
Immediately replenish the cells in 1uL of SOC and incubate overnight in respective temperature.
Datsenko-Wanner Method of genome deletions
PCR amplify the FRT flanked resistance gene using appropriate primers. Verify PCR cycle conditions
Transform strain expressing lambda red system
Select antibiotic resistant transformants that grow up on antibiotic plates overnight (screening)
Eliminate resistance cassette using an FLP flanked expression plasmid
3A Assembly
Adapted the protocol from iGEM’s website
Miniprep the two part samples you desire to assemble together (in BioBrick RFC[10] plasmid backbones)
Digestion:
Use ~ 100ng total for each reaction tube>
Add equal volume (4  uL) linearized plasmid backbone to Enzyme Master Mix. (5 ul NEB Buffer 2, 0.5 ul BSA, 0.5 ul EcoRI-HF, 0.5 ul PstI, 0.5 ul DpnI, 18 ul dH20) *linear plasmid backbone must have separate antibiotic resistance to plasmids with parts*
Add equal volume (4  uL) Part #1 to Enzyme Master Mix (5 ul NEB Buffer 2, 0.5 ul BSA,0.5 ul EcoRI-HF,0.5 ul SpeI,18.5 ul dH20
Add equal volume (4  uL) Part #2 to Enzyme Master Mix (5 ul NEB Buffer 2, 0.5 ul BSA, 0.5 ul XbaI, 0.5 ul PstI, 18.5 ul dH20
Digest all three reactions at 37C for 30 min, heat kill 80C for 20 min
Ligation:
Combine 2 ul of digested plasmid backbone, 2 ul digested Part #1 digested, 2 ul digested part #2.
Add 1 ul T4 DNA ligase buffer.
Add 0.5 ul T4 DNA ligase
Add water to 10 ul
Ligate 16C for 30 min, heat kill 80C for 20 min
Transform with 1-2 uL of product following listed protocol
PCR Screening
Pick a colony from plate with sterile techniques. Choose one colony per collection tube, being careful not to cross contaminate two colonies.
Resuspend each colony in 50 µL of sterile DI
Add 1 µL of suspension template to 49 µL of PCR Master Mix (depends on kit)
Vortex briefly, run PCR temperature program based off insert specific primer instructions
Run a gel of PCR products.
Choose a sample with the proper insert size and use the remaining sample to streak the bacteria onto a plate with the appropriate antibiotic resistance. Place the plate in the 37 C overnight.
