<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFlorida_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFlorida/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UFlorida"><DIV class="menu_item direct_link">
				HOME
			</DIV><DIV class="menu_item">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Protocols"><DIV class="submenu_item">
					Protocols
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A></DIV><DIV class="menu_item">
			RESULTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="topnav" style="display: flex; align-items: flex-end; background-color: #333333; position: sticky; top: 18px;"><DIV class="dropdown"><A class="topnav" href="https://2018.igem.org/Team:UFlorida">Home</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Team">Members</A><A href="https://2018.igem.org/Team:UFlorida/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UFlorida/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Description">Description</A><A href="https://2018.igem.org/Team:UFlorida/Design">Design</A><A href="https://2018.igem.org/Team:UFlorida/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UFlorida/Protocols">Protocols</A><A href="https://2018.igem.org/Team:UFlorida/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UFlorida/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UFlorida/Model">Model</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UFlorida/Parts">Parts</A><A href="https://2018.igem.org/Team:UFlorida/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Engagement</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UFlorida/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV id="#main" class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Lab Notebook</H1></DIV><DIV class="column full_size"><H3>Week of 6/18/2018</H3><UL><LI>Isolated Plasmid pTRC99A</LI><LI>Isolate Plasmid pTrc99A from <I>E. coli</I> using via Qiagen miniprep protocol. </LI><LI>Streaked out Nissle 1917 on a plate</LI></UL></DIV><DIV class="column full_size"><H3>Week of 7/1/2018</H3><UL><LI>The two PCR's performed failed due to bad plasmid stock</LI><LI>Miniprepped fresh pKDsgRNA plasmid</LI><LI>PCR reaction</LI><LI>DpnI digestion</LI><LI>PCR purification</LI><LI>T4 Ligation* with 2 hour incubation at Room Temperature</LI><LI>Suspended plasmids from iGEM packaging</LI><LI>Transformed 4 biobricks into Dh5 alpha cells
</LI><LI>BBa_K1850010 (pSB1C3) fimH</LI><LI>BBa_K1850011 (pSB1C3) fimH</LI><LI>BBa_J04450 (pSB4K5) rfp</LI><LI>BBa_J04450 (pSB3T5) rfp</LI><LI>Transformation of the 4 pKDsgRNA plasmids into Dh5 alpha</LI></UL></DIV><DIV class="column full_size"><H3>Week of 7/9/2018</H3><UL><LI>Isolated 136, 49, 3TS, 4KS via Qiagen miniprep protocol
</LI><LI>Co-Electroporated biobricks in their respective plasmids (minipreppred from Dh5 alpha) into Nissle 1917 following electroporation protocol:
</LI><LI>Combination 1: BBa_K1850010 (pSB1C3) + BBa_J04450 (pSB3T5)</LI><LI>Combination 2: BBa_K1850011 (pSB1C3) + BBa_J04450 (pSB3T5)</LI></UL><LI>Transforming Competent Cells* (Nissle WT)</LI><LI>Cultured transformed Nissle-pCas9-CR4 cells in LB broth for 4 hours</LI><LI>Transforming Competent Cells (Nissle_pCas9-CR4)</LI><LI>Continuation of No-Scar</LI><LI>Make new Nissle and W3110 cultures: Repeat procedure from 7/11</LI></DIV><DIV class="column full_size"><H3>Week of 7/17/2018</H3><UL><LI>Transforming Competent Cells (Nissle with fimH 136KO or 49KO)</LI><LI>Transforming Competent Cells (Nissle_pCas9-CR4 transformed with pKDsgRNA-ldhA)</LI><LI>Transforming Competent Cells (Nissle-fimH 49KO transformed with RFP-pSB3T5; Nissle-fimH 136KO transformed with RFP-pSB3T5)</LI><LI>Transforming Competent Cells* (Nissle-pkDldha-pcas9cr4 transformed with ldha oligo and ldha deletion oligo and control oligo)</LI><LI>Assay for No-Scar and plating
</LI><LI>Transforming Competent Cells* (Nissle-fimH 49KO transformed with RFP-pSB3T5; Nissle-fimH 136KO transformed with RFP-pSB3T5)</LI><LI>Mutating ldhA gene</LI><LI>plasmid isolation of the 4K5 plasmid</LI><LI>Cultured Nissle in 9mL tubes for future fermentation (with and without phosphate buffer)
</LI><LI>+ Phosphate (0.9mL Buffer, 0.09mL Glucose (30%),8.01 mL LB)</LI><LI>-Phosphate (0.09mL Glucose (30%), 8.91 mL LB)</LI></UL><LI>Incubator for 4 days</LI><LI>Made Glycerol stock culture of Nissle 1917, stored at -80 C</LI><LI>Culture Nissle and W3110 cultures</LI><LI>Run HPLC: -Nissle, -Nissle, +P, -Nissle, -P, -LB, -W3110</LI><UL><LI>Look for presence of:Glucose, Succinate, Lactate, Formate, Acetate, Ethanol, Butyrate</LI><LI>Run HPLC and Fermentation on Nissle and W3110</LI><LI>HPLC Prep Protocol</LI><LI>Fermentation Prep Protocol</LI></UL></DIV><DIV class="column full_size"><H3>Week of 7/23/18</H3><UL><LI>Run HPLC for Nissle fermentation: t=0,t=~20,t=48</LI><LI>Re-ran HPLC after mixing samples.</LI><LI>Detection of LDHA-Deletion Colonies by PCR </LI><LI>Determine if culture genes have the deleted gene </LI><LI>Fermentation Protocol</LI><LI>Inoculated Nissle pCas9-cr4, Nissle 136 KO, fimH-pSB1C3, Nissle 49KO fimH-pSB1C3</LI><LI>Inoculated Nissle pCas9-cr4 pKDsgRNA-ldhA</LI><LI>PCR to detect the mutant strain
</LI><LI>Troubleshooting: Why aren't the oligos for LdhA deleting the gene based on PCR results???</LI><LI>Hypothesis: Oligos are supposed to replace Okazoki fragments on the lagging strand. If Ori Right Replication Fork replicates faster than the left the LdhA gene will replicate on the opposite side!</LI><LI>Quick way to test hypothesis: Use a gene upstream of LdhA, AdhE, with appropriate oligos.</LI></UL></DIV><DIV class="column full_size"><H3>Week of 7/30/2018</H3><UL><LI>Plate old iGEM parts in LB+chlor plates from DH5A.
</LI><LI>K158 7000</LI><LI>K158 7004</LI><LI>K243 2000</LI><LI>K205 2016</LI></UL><LI>Perform Transformation w/ Oligos for LdhA New (cDNA of Old oligo) and New+Old Oligos. </LI><LI>Plate iGEM written by RFP Nissile </LI><LI>Transform (pcas9cr4 nissle with pKD ldhA/adhe; wild type nissle with old parts (miniprepped, FimH 136 nissle with RFP)</LI><LI>Miniprep old parts (Shan's bacteria)</LI><LI>PCR to sequence our plasmids</LI><LI>Transforming pCas9cr4 with pKDsgRNA-AdhE and LdhA</LI><LI>Add the oligos (New &amp; Both) to:
</LI><LI> pCas9cr4 + pKDsgRNA-LdhA cells</LI><LI>Cas9 + pKD-AdhE, Cas9 + pKD-LdhA </LI><LI>ATC+ New</LI><LI>ATC+ Both</LI><LI>ATC- New</LI><LI>AtC- Both</LI><LI>Miniprep pKD46, pCP20, pKD4 from Shan's bacteria.</LI><LI> Miniprep old parts:K158 7000, K158 7004, K243 2000, K205 2016</LI><LI>Start of no-SCAR alternative pathway to delete genes. </LI><LI>Transformed by electroporation: K158 7005, K158 7004, PDK4, K205 2016, Nissle FimH KO 136 + rfp</LI><LI>miniprepped PKD46, PKD4, PCP20, K158 7005, K158 7004, K205 2016</LI><LI>streaked PCAS9 Cr4 + sgRNA_LDHA and plated; PKD46, K205 2016, K158 7004, K158 7005, FimH 136KO + 4K5  Nissle on KAN</LI><LI>PCR amplified FRT-flanked resistant Kan gene with appropriate H1P1 and H2P2 frdA primers. </LI><LI>Screened  colonies from the sgrna-ldha + pcas9cr4 (plated on chloramphenicol, induced with 188 nissle that had been &quot;no-scar&quot;ed with the new ldha oligo (lagging strand homology.)</LI><LI>screened 16 colonies that had been noscared with both the old and new ldha oligos (leading and lagging strand homologies)</LI><LI>Incubated pKDsgRNA-ldhA + pcas9cr4 to try no scar again</LI></DIV><DIV class="column full_size"><H3>Week of 8/6/2018</H3><UL><LI>Transformations, with 2 plasmids and 2 different strains: Nissle-pKD46, Nissle-pKDsgRNA, GM 272-pKD4, GM 272-pKDsgRNA</LI><LI>Stitch fragments 1 and 2 of the butyrate fragment together via overlap extension PCR. </LI><LI>Perform circular polymerase extension cloning into the amplified ptrc99A plasmid and the stitched together butyrate </LI><LI>fragment 1 and 2.</LI><LI>DNA Gel Extraction</LI><LI>Mini-prep</LI><LI>PCR on butryate fragment 2</LI><LI>Transformed WT Nissle with 46 pKD from miniprep</LI><LI>PCR of butyrate fragment 2 on gel</LI><LI>Transformations: Nissile with his pkD plasmid, Nissile with sgRNA</LI></UL></DIV><DIV class="column full_size"><H3>Week of 8/13/2018</H3><UL><LI>PCR amplified Kanamycin FRT flanked gene on pDK with H1P1 and H2P2 primers WITH Q5 polymerase</LI><LI>DNA purification</LI><LI>Transformation</LI><LI>Amplification of the butyrate downstream fragment (Nissle synthetic operon) with primers F1-But and R-CPEC, as well as the Q5 polymerase mastermix.</LI></UL></DIV><DIV class="column full_size"><H3>Week of 8/20/2018</H3><UL><LI>Transformed rfp into k205 Nissle </LI><LI>Transformed frdA oligos into Nissle pKDsgRNA-kan(frt) plated at 30 C on ATC to activate pCas9.</LI><LI>Overlap Extension PCR assembly of butyrate fragment</LI><LI>KOD Polymerase assembly of Butryate Fragment.</LI><LI>Patching of bacterial colonies</LI><LI>CPEC PCR Reaction</LI><LI>Dh5 alpha was transformed with the butyrate biobrick cloned into ptrc99a plasmid</LI><LI>PCR confirming the insertion of the butyrate biobrick into the dh5 alpha.  Screening 5 colonies with KOD polymerase.</LI><LI>Grew <I>E. coli Nissle</I> delta frdA with pkd46 plasmid</LI><LI>Dpn1 DNA digest of ldha/Kan gene</LI><LI>PCR Purified</LI><LI>Electroporated ldha/Kan DNA and <I>E. coli Nissle</I> delta frdA cells with pkd46 plasmid. </LI><LI>Transformation of pKDsgRNA and pKD46 with ldha oligos</LI><LI>PCR Purification oligos(ldha,frda,adhe)</LI><LI>PCR on Butyrate part 1 and Butyrate part 2</LI><LI>Ran resulting fragments on a gel alongside restriction digested pieces</LI><LI>Amplifying Butyrate Fragments</LI><LI>CPEC Reaction (12.5uL) KOD polymerase + ptrc99a plasmid + butyrate part</LI><LI>Incubate at 37 6 samples:(K205 2016, ldha- kan, Nissle frdA pkD46, K158 7005, pkD46  +ldha, K158 7004)</LI><LI>HPLC protocol</LI></UL></DIV><DIV class="column full_size"><H3>Week of 9/10/2018</H3><UL><LI>Screening butyrate CPEC transformed dh5-a colonies</LI><LI>Screening KOD colonies with Onetaq</LI><LI>Screened for butyrate insert with 3 different polymerases</LI><LI>Transformed <I>E. coli Nissle</I> delta frda with the ldha - Kan fragment</LI><LI>Crack and screen 100 Q5 colonies for a plasmid insertion into ptrc</LI><LI>Transformed ptrc99A plasmid into dh5 alpha</LI></UL></DIV><DIV class="column full_size"><H3>Week of 9/17/2018</H3><UL><LI>Screening</LI><LI>Confirmed gene deletion of ldhA by pcr and gel</LI></UL></DIV><DIV class="column full_size"><H3>Week of 9/24/2018</H3><UL><LI>Overlap PCR</LI><LI>Butyrate fragment 1 amplification</LI><LI>Nissle synthetic operon fragment amplification</LI><LI>Cloning PCR for  ter</LI><LI>Gel purification</LI><LI>VR screening of butyrate and fimH parts</LI><LI>HPLC confirmed for ldhA deletion: no lactate produced: compared to model</LI></UL></DIV><DIV class="column full_size"><H3>Week of 10/01/2018</H3><UL><LI>PCR screening for pcat ter</LI><LI>PCR Screen But1 ptrc99a plate (Extend for 3kb)</LI><LI>PCR Screen But2 ptrc99a colonies (Extend for 3kb)</LI><LI>PCR Screen Full But Ptrc99A (extend for 5.5 KB)</LI><LI>PCR Screen Full But - Pcat (extend for 5.5 KB)</LI><LI>Miniprep</LI><LI>Ter from Shan’s bacteria</LI><LI>Cloning pBAD from part</LI><LI>Fix “Ter from Shan’s bacteria” to:
</LI><LI>Overlap Extension PCR of the Ter Gene from Dr. Shanmugam’s gift plasmid with the pBAD promoter. Attempted assembly into BBa_K1587004</LI><LI>CPEC of the Ter gene with biobrick prefix and suffix into linearized psb1c3</LI><LI>Attempted insertion of the pBAD and pCAT promoters upstream of the Ter gene in psb1c3</LI><LI>PCR screen for Ter gene in pSB1C3, and for ter gene plus promoters in pSB1C3</LI></UL></DIV><DIV class="column full_size"><H3>Week of 10/08/2018</H3><UL><LI>Continued PCR screening of colonies for parts cloned in the correct orienations</LI><LI>HPLC analysis of transformed colonies for butyrate production.</LI><LI>Experiment 1: 
</LI><LI>The Small Scale Fermentation Prep Protocol for Rotating Samples was run for two trials one being Wild Type E. coli Nissle (WT Nissle) and the other was E. coli Nissle with an ldhA gene deletion. </LI><LI>The Small Scale Fermentation Prep Protocol for Standing Samples was run for two trials one being Wild Type E. coli Nissle and the other was E. coli Nissle with an ldhA gene deletion.  </LI><LI>Experiment 2: 
</LI><LI>The Small Scale Fermentation Prep Protocol for Standing Samples was run for ten total trials.  Five were Wild Type E. coli Nissle and the other five were E. coli Nissle with an ldhA gene deletion. </LI><LI>HPLC protocol was carried out for all samples mentioned</LI></UL></DIV><DIV class="column full_size"><H3>Week of 10/15/2018</H3><UL><LI>Experiment 1:
</LI><LI>The Small Scale Fermentation Prep Protocol for Standing Samples was run for eight different samples ( E. Coli Nissle -ldhA + Butyrate Producing Part(BPP), BEM3 +BPP, Nissle -ldhA -frdA, Nissle 7004, Nissle -ldhA +7004, WT Nissle, BEM3 7004, and BEM3).</LI><LI>HPLC protocol was carried out for six samples ( E. Coli Nissle -ldhA + BPP, BEM3 +part, Nissle -ldhA -frdA, Nissle 7004, Nissle -ldhA +7004, WT Nissle) and a control (LB+arabinose+PO4).</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>