<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFlorida_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFlorida/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UFlorida"><DIV class="menu_item direct_link">
				HOME
			</DIV><DIV class="menu_item">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Protocols"><DIV class="submenu_item">
					Protocols
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A></DIV><DIV class="menu_item">
			RESULTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="topnav" style="display: flex; align-items: flex-end; background-color: #333333; position: sticky; top: 18px;"><DIV class="dropdown"><A class="topnav" href="https://2018.igem.org/Team:UFlorida">Home</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Team">Members</A><A href="https://2018.igem.org/Team:UFlorida/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UFlorida/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Description">Description</A><A href="https://2018.igem.org/Team:UFlorida/Design">Design</A><A href="https://2018.igem.org/Team:UFlorida/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UFlorida/Protocols">Protocols</A><A href="https://2018.igem.org/Team:UFlorida/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UFlorida/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UFlorida/Model">Model</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UFlorida/Parts">Parts</A><A href="https://2018.igem.org/Team:UFlorida/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Engagement</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UFlorida/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV id="#main" class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Demonstrate</H1><P>
The first step of our system design was to delete naturally occurring genes in <I>E. coli Nissle 1917</I> that encode for interfering metabolic enzymes (<I>ldhA, adhE, frdA, pta </I> ). This would allow subsequent insertion of the butyrate gene and actually produce significant yield in a strain unable to do so before. Other iGEM teams have attempted to engineer butyrate production but were unsuccessful due to lacking key enzymes or accurate gene sequences. We were not able to assemble an RFC10-compliant butyrate pathway BioBrick. However, we have properly introduced butyrate production in two <I>E. coli</I> strains, proving our engineered system works. We also demonstrated that deleting the <I>ldhA</I> gene in <I>E. coli Nissle</I> did influence the carbon flux in a manner that increased yields of other by-products, such as ethanol, in lieu of lactate production.</P></DIV><DIV class="column full_size"><P>It follows that in this deletion strain and in subsequent deletion strains, a completely NADH-dependent plasmid-based butyrate production pathway could be functional and subjected to further metabolic engineering by directed evolution.  If an iGEM team were to create this pathway, either de novo, cloning, or by adding the <I>ter</I> gene to BBa_K1587004, they would be able to use our deletion strain as a chassis for their part.  Finally, metabolic engineering of <I>E. coli Nissle 1917</I> to produce other by-products could potentially make use of our deletion strain, which will be available to other iGEM teams by request.
</P></DIV><DIV class="column full_size"><P>As a last ditch attempt to produce a butyrate-producing part, we attempted to add the <I>Ter</I> gene, which was given to us as a gift from Dr. Shanmugam, to our system. This gene contains the same amino acid sequence (but different codon optimization) than our submitted <I>Ter</I> part (BBa_K2677001). Placed under control of the pBAD promoter (BBa_I0500) downstream of BBa_K1587004’s terminator, which contains 4 out of the 5 enzymes needed to produce butyrate in <I>E. coli</I>, would contain a complete butyrate pathway. Unfortunately, the AraC-pBAD-Ter fragment cloned in the wrong orientation into the XbaI and SpeI sites, and therefore it is not RFC10 compliant. Given more time and resources, we might have found a colony with the correct orientation; however, in an attempt to prove that our engineered system works, we will continue to show data for this plasmid, which will be made publicly available to any team that requests it. We will refer to this plasmid as the “butyrate pathway part” from here on.
</P></DIV><DIV class="column full_size"><H1> BEM3 </H1><P>BEM3 was a strain of <I>E. coli</I> used to test the function of our part, which is intended for use in our deletion strain of <I>E. coli Nissle 1917</I>. BEM3 has several pertinent genes deleted, including <I>ldhA</I> and <I>adhE</I>. It can grow anaerobically only if it has been transformed with a plasmid that contains a pathway for it to produce specific metabolites such as butyrate. We transformed BEM3 with this butyrate path part. We observed the following optical densities at the same time point after 24 hours of anaerobic incubation at 37 degrees:</P></DIV><DIV class="column full_size"><H1><CENTER><I>Anaerobic Incubation at 37 C</I></CENTER></H1></DIV><DIV class="column full_size"><P>From the measured OD’s, we can conclude that our part is functionally providing a metabolic pathway for BEM3 to survive anaerobically. BBa_K1587004 alone does not allow the BEM3 strain to grow anaerobically because it does not contain <I>Ter</I>, an NADH-dependent enzyme, and is therefore not redox balanced for butyrate production. <I>Ccr</I> is not sufficient for BBa_K1587004, as it is an NADPH-dependent enzyme. Our part increased the optical density to four times what was observed in the other groups, suggesting it is providing a pathway for BEM3 to produce butyrate and survive anaerobically, something that BBa_K1587004 alone did not do.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>