Team:UFlorida/Notebook
HOME
TEAM
Team Members
Collaborations
Attributions
PROJECT
Description
Design
Experiments
Protocols
Notebook
InterLab
Model
Results
Improve
RESULTS
Parts Overview
Demonstrate
HUMAN PRACTICES
Human Practices
Education & Engagement
Home
Team
Members Collaborations Attributions
Project
Description Design Experiments Protocols Notebook InterLab Model
Results
Demonstrate Parts Results
Engagement
Human Practices Public Engagement Entrepreneurship
Lab Notebook
Week of 6/18/2018
Isolated Plasmid pTRC99A
Isolate Plasmid pTrc99A from E. coli using via Qiagen miniprep protocol.
Streaked out Nissle 1917 on a plate
Week of 7/1/2018
The two PCR's performed failed due to bad plasmid stock
Miniprepped fresh pKDsgRNA plasmid
PCR reaction
DpnI digestion
PCR purification
T4 Ligation* with 2 hour incubation at Room Temperature
Suspended plasmids from iGEM packaging
Transformed 4 biobricks into Dh5 alpha cells
BBa_K1850010 (pSB1C3) fimH
BBa_K1850011 (pSB1C3) fimH
BBa_J04450 (pSB4K5) rfp
BBa_J04450 (pSB3T5) rfp
Transformation of the 4 pKDsgRNA plasmids into Dh5 alpha
Week of 7/9/2018
Isolated 136, 49, 3TS, 4KS via Qiagen miniprep protocol
Co-Electroporated biobricks in their respective plasmids (minipreppred from Dh5 alpha) into Nissle 1917 following electroporation protocol:
Combination 1: BBa_K1850010 (pSB1C3) + BBa_J04450 (pSB3T5)
Combination 2: BBa_K1850011 (pSB1C3) + BBa_J04450 (pSB3T5)
Transforming Competent Cells* (Nissle WT)
Cultured transformed Nissle-pCas9-CR4 cells in LB broth for 4 hours
Transforming Competent Cells (Nissle_pCas9-CR4)
Continuation of No-Scar
Make new Nissle and W3110 cultures: Repeat procedure from 7/11
Week of 7/17/2018
Transforming Competent Cells (Nissle with fimH 136KO or 49KO)
Transforming Competent Cells (Nissle_pCas9-CR4 transformed with pKDsgRNA-ldhA)
Transforming Competent Cells (Nissle-fimH 49KO transformed with RFP-pSB3T5; Nissle-fimH 136KO transformed with RFP-pSB3T5)
Transforming Competent Cells* (Nissle-pkDldha-pcas9cr4 transformed with ldha oligo and ldha deletion oligo and control oligo)
Assay for No-Scar and plating
Transforming Competent Cells* (Nissle-fimH 49KO transformed with RFP-pSB3T5; Nissle-fimH 136KO transformed with RFP-pSB3T5)
Mutating ldhA gene
plasmid isolation of the 4K5 plasmid
Cultured Nissle in 9mL tubes for future fermentation (with and without phosphate buffer)
+ Phosphate (0.9mL Buffer, 0.09mL Glucose (30%),8.01 mL LB)
-Phosphate (0.09mL Glucose (30%), 8.91 mL LB)
Incubator for 4 days
Made Glycerol stock culture of Nissle 1917, stored at -80 C
Culture Nissle and W3110 cultures
Run HPLC: -Nissle, -Nissle, +P, -Nissle, -P, -LB, -W3110
Look for presence of:Glucose, Succinate, Lactate, Formate, Acetate, Ethanol, Butyrate
Run HPLC and Fermentation on Nissle and W3110
HPLC Prep Protocol
Fermentation Prep Protocol
Week of 7/23/18
Run HPLC for Nissle fermentation: t=0,t=~20,t=48
Re-ran HPLC after mixing samples.
Detection of LDHA-Deletion Colonies by PCR
Determine if culture genes have the deleted gene
Fermentation Protocol
Inoculated Nissle pCas9-cr4, Nissle 136 KO, fimH-pSB1C3, Nissle 49KO fimH-pSB1C3
Inoculated Nissle pCas9-cr4 pKDsgRNA-ldhA
PCR to detect the mutant strain
Troubleshooting: Why aren't the oligos for LdhA deleting the gene based on PCR results???
Hypothesis: Oligos are supposed to replace Okazoki fragments on the lagging strand. If Ori Right Replication Fork replicates faster than the left the LdhA gene will replicate on the opposite side!
Quick way to test hypothesis: Use a gene upstream of LdhA, AdhE, with appropriate oligos.
Week of 7/30/2018
Plate old iGEM parts in LB+chlor plates from DH5A.
K158 7000
K158 7004
K243 2000
K205 2016
Perform Transformation w/ Oligos for LdhA New (cDNA of Old oligo) and New+Old Oligos.
Plate iGEM written by RFP Nissile
Transform (pcas9cr4 nissle with pKD ldhA/adhe; wild type nissle with old parts (miniprepped, FimH 136 nissle with RFP)
Miniprep old parts (Shan's bacteria)
PCR to sequence our plasmids
Transforming pCas9cr4 with pKDsgRNA-AdhE and LdhA
Add the oligos (New & Both) to:
pCas9cr4 + pKDsgRNA-LdhA cells
Cas9 + pKD-AdhE, Cas9 + pKD-LdhA
ATC+ New
ATC+ Both
ATC- New
AtC- Both
Miniprep pKD46, pCP20, pKD4 from Shan's bacteria.
Miniprep old parts:K158 7000, K158 7004, K243 2000, K205 2016
Start of no-SCAR alternative pathway to delete genes.
Transformed by electroporation: K158 7005, K158 7004, PDK4, K205 2016, Nissle FimH KO 136 + rfp
miniprepped PKD46, PKD4, PCP20, K158 7005, K158 7004, K205 2016
streaked PCAS9 Cr4 + sgRNA_LDHA and plated; PKD46, K205 2016, K158 7004, K158 7005, FimH 136KO + 4K5  Nissle on KAN
PCR amplified FRT-flanked resistant Kan gene with appropriate H1P1 and H2P2 frdA primers.
Screened  colonies from the sgrna-ldha + pcas9cr4 (plated on chloramphenicol, induced with 188 nissle that had been "no-scar"ed with the new ldha oligo (lagging strand homology.)
screened 16 colonies that had been noscared with both the old and new ldha oligos (leading and lagging strand homologies)
Incubated pKDsgRNA-ldhA + pcas9cr4 to try no scar again
Week of 8/6/2018
Transformations, with 2 plasmids and 2 different strains: Nissle-pKD46, Nissle-pKDsgRNA, GM 272-pKD4, GM 272-pKDsgRNA
Stitch fragments 1 and 2 of the butyrate fragment together via overlap extension PCR.
Perform circular polymerase extension cloning into the amplified ptrc99A plasmid and the stitched together butyrate
fragment 1 and 2.
DNA Gel Extraction
Mini-prep
PCR on butryate fragment 2
Transformed WT Nissle with 46 pKD from miniprep
PCR of butyrate fragment 2 on gel
Transformations: Nissile with his pkD plasmid, Nissile with sgRNA
Week of 8/13/2018
PCR amplified Kanamycin FRT flanked gene on pDK with H1P1 and H2P2 primers WITH Q5 polymerase
DNA purification
Transformation
Amplification of the butyrate downstream fragment (Nissle synthetic operon) with primers F1-But and R-CPEC, as well as the Q5 polymerase mastermix.
Week of 8/20/2018
Transformed rfp into k205 Nissle
Transformed frdA oligos into Nissle pKDsgRNA-kan(frt) plated at 30 C on ATC to activate pCas9.
Overlap Extension PCR assembly of butyrate fragment
KOD Polymerase assembly of Butryate Fragment.
Patching of bacterial colonies
CPEC PCR Reaction
Dh5 alpha was transformed with the butyrate biobrick cloned into ptrc99a plasmid
PCR confirming the insertion of the butyrate biobrick into the dh5 alpha.  Screening 5 colonies with KOD polymerase.
Grew E. coli Nissle delta frdA with pkd46 plasmid
Dpn1 DNA digest of ldha/Kan gene
PCR Purified
Electroporated ldha/Kan DNA and E. coli Nissle delta frdA cells with pkd46 plasmid.
Transformation of pKDsgRNA and pKD46 with ldha oligos
PCR Purification oligos(ldha,frda,adhe)
PCR on Butyrate part 1 and Butyrate part 2
Ran resulting fragments on a gel alongside restriction digested pieces
Amplifying Butyrate Fragments
CPEC Reaction (12.5uL) KOD polymerase + ptrc99a plasmid + butyrate part
Incubate at 37 6 samples:(K205 2016, ldha- kan, Nissle frdA pkD46, K158 7005, pkD46  +ldha, K158 7004)
HPLC protocol
Week of 9/10/2018
Screening butyrate CPEC transformed dh5-a colonies
Screening KOD colonies with Onetaq
Screened for butyrate insert with 3 different polymerases
Transformed E. coli Nissle delta frda with the ldha - Kan fragment
Crack and screen 100 Q5 colonies for a plasmid insertion into ptrc
Transformed ptrc99A plasmid into dh5 alpha
Week of 9/17/2018
Screening
Confirmed gene deletion of ldhA by pcr and gel
Week of 9/24/2018
Overlap PCR
Butyrate fragment 1 amplification
Nissle synthetic operon fragment amplification
Cloning PCR for  ter
Gel purification
VR screening of butyrate and fimH parts
HPLC confirmed for ldhA deletion: no lactate produced: compared to model
Week of 10/01/2018
PCR screening for pcat ter
PCR Screen But1 ptrc99a plate (Extend for 3kb)
PCR Screen But2 ptrc99a colonies (Extend for 3kb)
PCR Screen Full But Ptrc99A (extend for 5.5 KB)
PCR Screen Full But - Pcat (extend for 5.5 KB)
Miniprep
Ter from Shan’s bacteria
Cloning pBAD from part
Fix “Ter from Shan’s bacteria” to:
Overlap Extension PCR of the Ter Gene from Dr. Shanmugam’s gift plasmid with the pBAD promoter. Attempted assembly into BBa_K1587004
CPEC of the Ter gene with biobrick prefix and suffix into linearized psb1c3
Attempted insertion of the pBAD and pCAT promoters upstream of the Ter gene in psb1c3
PCR screen for Ter gene in pSB1C3, and for ter gene plus promoters in pSB1C3
Week of 10/08/2018
Continued PCR screening of colonies for parts cloned in the correct orienations
HPLC analysis of transformed colonies for butyrate production.
Experiment 1:
The Small Scale Fermentation Prep Protocol for Rotating Samples was run for two trials one being Wild Type E. coli Nissle (WT Nissle) and the other was E. coli Nissle with an ldhA gene deletion.
The Small Scale Fermentation Prep Protocol for Standing Samples was run for two trials one being Wild Type E. coli Nissle and the other was E. coli Nissle with an ldhA gene deletion.
Experiment 2:
The Small Scale Fermentation Prep Protocol for Standing Samples was run for ten total trials.  Five were Wild Type E. coli Nissle and the other five were E. coli Nissle with an ldhA gene deletion.
HPLC protocol was carried out for all samples mentioned
Week of 10/15/2018
Experiment 1:
The Small Scale Fermentation Prep Protocol for Standing Samples was run for eight different samples ( E. Coli Nissle -ldhA + Butyrate Producing Part(BPP), BEM3 +BPP, Nissle -ldhA -frdA, Nissle 7004, Nissle -ldhA +7004, WT Nissle, BEM3 7004, and BEM3).
HPLC protocol was carried out for six samples ( E. Coli Nissle -ldhA + BPP, BEM3 +part, Nissle -ldhA -frdA, Nissle 7004, Nissle -ldhA +7004, WT Nissle) and a control (LB+arabinose+PO4).
