<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFlorida_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFlorida/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UFlorida"><DIV class="menu_item direct_link">
				HOME
			</DIV><DIV class="menu_item">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Protocols"><DIV class="submenu_item">
					Protocols
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A></DIV><DIV class="menu_item">
			RESULTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="topnav" style="display: flex; align-items: flex-end; background-color: #333333; position: sticky; top: 18px;"><DIV class="dropdown"><A class="topnav" href="https://2018.igem.org/Team:UFlorida">Home</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Team">Members</A><A href="https://2018.igem.org/Team:UFlorida/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UFlorida/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Description">Description</A><A href="https://2018.igem.org/Team:UFlorida/Design">Design</A><A href="https://2018.igem.org/Team:UFlorida/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UFlorida/Protocols">Protocols</A><A href="https://2018.igem.org/Team:UFlorida/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UFlorida/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UFlorida/Model">Model</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UFlorida/Parts">Parts</A><A href="https://2018.igem.org/Team:UFlorida/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Engagement</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UFlorida/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV id="#main" class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H2>Gene Deletion</H2><P>We first attempted the no-SCAR (Scarless Cas9 Assisted Recombineering) method to remove ldhA (lactate dehydrogenase-A)  from the E. coli Nissle  genome.   No-SCAR is different from the traditional CRISPR system used in eukaryotes because it uses CRISPR to counterselect against the wild type DNA, theoretically leaving only mutant colonies on the plate. Cells are transformed with two plasmids carrying Cas9 and the guide RNA. Cells are then transformed again with DNA oligonucleotides that confer a mutation to the target site of the guide-RNA. Colonies are incubated on antibiotic plates and mutant bacteria survive while wild-type are killed off by the CRISPR-Cas9 complex. E. coli naturally lack an efficient double-strand-break-repair system and CRISPR-Cas9 engineering can kill off wild-type cells after genetic targeting by the specific guide-RNA. This method takes advantage of the lack of repair and targets colonies that have failed to mutate (i.e. no gene deletion) and induces cell death. </P></DIV><CENTER><DIV class="column full_size"><H3 style="black"> no-SCAR </H3></DIV></CENTER><CENTER><DIV class="column full_size"><SMALL>Diagram adapted from Reish 2015</SMALL></DIV></CENTER><DIV class="column full_size"><P>However no-SCAR proved very difficult to manage in our specific <I>E. coli Nissle</I> chassis. Over one hundred bacterial colonies were screened for the deletion before we moved on to a different approach. This approach was not sustainable to rid one gene let alone the five carbon-draining genes in the short time we had left so we moved on. </P></DIV><DIV class="column full_size"><DIV class="highlight decoration_A_full"><CENTER><P><B>Three groups: Control (C) Deletion (D) Mutation (M)</B>The Control group was transformed with an oligonucleotide homologous to a region of the genome outside of the guide RNAâ€™s recognition (rpoB). The Deletion group was transformed with an oligo that caused a deletion of the entire <I>ldhA</I> gene.  The Mutation group was transformed with an oligo that caused a stop mutation in the <I>ldhA</I> gene.  Cas9 was induced via addition of anhydrous tetracycline on the right plate but not on the left, and 10-fold serial dilutions were made from the bottom of the plate to the top. No group had any order of magnitude of more killing than any other, implying that either recombination was not efficiently occurring or that crispr-Cas9 was not selectively killing wild-type bacteria as we had hoped.</P></CENTER></DIV></DIV><DIV class="column full_size"><P>We switched to the Datsenko-Wanner method. This technique utilizes antibiotic resistance replacement of the desired gene deletion. To accomplish this we used primers to amplify out a kanamycin resistance gene. This sequence was flanked upstream and downstream with regions of homology to the specific gene to remove. This amplified oligo was then used to recombineer a deletion and select for by kanamycin resistance. The resistance was necessary to remove so we used a FLP recombinase to 'flip' out the resistance gene. PCR and patching was used to confirm deletion. Within a week of switching, the <I>frd </I> operon (fumarate reductase) was successfully deleted though dozens of colonies had to be screened in order to find the correct mutant. From then on this was our chosen deletion method.
</P></DIV><CENTER><DIV class="column full_size"><H3 style="black"> Datsenko-Wanner</H3></DIV></CENTER><CENTER><DIV class="column full_size"><SMALL>Diagram adapted from Datsenk and Wanner 2000</SMALL></DIV></CENTER><DIV class="column third_size"><DIV class="highlight decoration_A_full"><P><B>1. Integrate Resistance Cassette</B></P><P>#1. Ladder #2. Wild type (5kb)     #3. frd operon + frt-kanamycin-frt cassette (1.5kb) </P><P><B>2. Flip out Resistance Cassette</B></P><P>Frt-kanamycin cassette flipped out using the pcp20 plasmid encoded recombinase.  The sample to the right of those two bands represents the previously integrated 1.5kb fragment.</P></DIV></DIV><DIV class="column two_thirds_size"><P>The colonies with the confirmed removal of the Kan cassette were patched on kanamycin plates (pictured below). The plasmids we used, pCP20 and pKD46, had to be confirmed via patching that they're cured of the cell.  This is done because unless the untransformed cell is sensitive for the antibiotic you're selecting it with, you wont be able to select for properly transformed cells. By plating the same colony on multiple antibiotic resistant plates, you can confirm that the cell is sensitive and resistant to respective antibiotics.</P><DIV class="highlight decoration_A_full"><P><B>3. Patching</B></P></DIV><P>However, we discovered through HPLC that fumarate was still being produced at higher concentrations than what was predicted by our metabolic model. We went back to the sequence and discovered that frdA is present in two copies in the <I>E. coli Nissle</I> genome. Although the gel showed the gene to be deleted, we had not known that a duplication existed.The second operon existed internally to the primers we used to screen and the homologous regions we made to delete with, so is was neither deleted nor detected. Two deletions would have been necessary to channel the carbon flow away from fumarate.</P><P>We decided not to go forward with that option as our model showed that other genes would have more carbon flux impact if deleted.</P></DIV><DIV class="column two_thirds_size"><P>Next to be deleted was <I>ldhA</I> which fortunately does not have a gene copy.  This was validated by confirmation of fragment bands on a gel and lack of lactate production seen by HPLC measurements. 
</P><P>(Left) One colony had the 1.5kb kan-frt cassette integrated into its genome at the ldha location.  Out of the 58 colonies screened for ldhA deletions, we found one that had it. Subsequent flipping out and loss of kanamycin resistance was successful.  
</P></DIV><DIV class="column full_size"><H3>Butyrate Assembly</H3><P>Team members optimized the five gene sequences for <I>E. coli Nissle 1917</I> for butyrate upregulation. The gene pathway was too long (5 kb) to be synthesized in one string so it was ordered as a gblock composed as two fragments. Synthesis was chosen over ordering the parts in a plasmid for cost efficiency. Butyrate fragment (BF) #1 contained atoB, hbd, and crt and BF #2 had ter and tesB. The ordered sequences were delivered with unintentional alterations that had extra copies of the Xbol digestion sites. This made it impossible to properly digest and ligate our gene fragments together, which is the typical way to assemble. A variety of other molecular techniques were attempted: (1) Overlap extension (2) Blunt Ligation of fragments #1 and #2.</P><P>The two fragments had regions that were similar to each other (overlapping) of about 40 bases each.Theoretically, if run through enough cycles of PCR the two fragments would become one due to elongation over the overlapping region (overlap extension). Even though a 5.3 kb band was visible after 12 cycles of overlap extension PCR, it was not possible to amplify this fragment to concentrations high enough to successfully assemble the part via Circular Polymerization Extension Cloning or restriction digestion into psb1C3 or other plasmids such as ptrC99A.  This may have been due to poor primer binding, or due to the fact that our gBlocks were not ideal for PCR reactions.</P><P>We also attempted a blunt ligation of BF #1 and BF #2 into ptrc99A, to later be moved into psb1c3.  These attempts also proved unsuccessful, as there were no bands in PCR reactions that screened for inserts.</P></DIV><CENTER><DIV class="column full_size"><H3 style="black"> Overlap PCR</H3></DIV></CENTER><CENTER><DIV class="column full_size"><H3 style="black"> Circular Polymerase Extension Cloning</H3></DIV></CENTER><DIV class="column full_size"><P>After several months of work without any results, the Parts Registry and previous projects were searched for a solution. The Toulouse 2015 iGEM biobricks (BBa_K1587004 and BBa_K1587005) had four of the five enzymes contained in our gblock, but no ter. We attempted to improve the butyrate synthesis pathway BBa_K1587004.  A critical analysis of this pathway reveals that the <I>ccr</I> gene, which catalyzes the reduction of crotonyl-coA to butyryl-CoA, is NADPH-dependent.  Without an enzyme that couples NADH to NADPH, this pathway will not be functional for butyrate production.  Our initial rationale for gene deletions was to force <I>E. coli</I> to produce butyrate with our synthetic pathway in order to maintain redox balance, but this would be impossible without an enzyme that catalyzes the reduction using NADH instead of NADPH.  For this reason, we attempted to add the trans-enoyl reductase (<I>ter</I>) to their pathway via restriction digestion to form a complete redox balanced pathway that could be driven further via gene deletions.</P><P>
Thus, it was feasible to add our <I>ter</I> onto their assembly in order to complete the pathway. We successfully used PCR with primers containing the iGEM prefix and suffix to clone out <I>ter</I> from BF #2. This was followed by successful circular polymerase extension cloning into the psb1C3 backbone and transformation into <I>E. coli Nissle</I>, resulting in our first biobrick of <I>ter</I> without a promoter.  We attempted to add the pCAT and pBAD promoters to this via 3A assembly from the distribution kit, but such attempts did not work and there was not enough time to troubleshoot before the parts deadline.  We did confirm that our <I>ter</I>-psb1c3 part is RFC 10 compliant via restriction digestion, and have submitted it to the registry for future teams to use.</P><P> We did not stop trying. Below is our last attempt at assembling a functional butyrate pathway. This time.....we believe it worked!!! See Results/Demonstrate for more detail.
</P></DIV><CENTER><DIV class="column full_size"><H3>Final Butyrate Pathway Assembly Method</H3></DIV></CENTER><DIV class="column full_size"><H3>FimH Addition</H3></DIV><DIV class="column full_size"><P>Red fluorescent protein (RFP) (BBa_J04450) was inserted downstream of the engineered FimH via 3A assembly.  This part was then moved back into pSB1C3 using Ecor-I and Pst-I digestion. PCR and restriction digestion were used to confirm that this part was RFC 10 compliant. Visual confirmation of the RFP gene expression is seen in the pictures below. Red-pink colonies appeared on a culture plate after several days of growth.</P></DIV><DIV class="column third_size"><P> Tube #1 - LB only
</P><P> Tube #2 -wild-type <I> E. coli Nissle</I></P><P> Tube #3 - <I> E. coli Nissle</I> transformed with FimH-RFP (in one plasmid)</P><P> Tube #4 - <I> E. coli Nissle</I> transformed with RFP and FIMH (two separate plasmids)</P><P>Our FimH-RFP part BBa_K267000 is clearly less &quot;red&quot; than FimH and RFP cotransformed onto separate plasmids. However our submitted part is useful in assays as seen below. This occurrence is likely due to genetic context dependency.</P></DIV><DIV class="column full_size"><P>This test was performed on Caco-2 cells which are an immortal line of epithelial corectal cancer cells. Characterization assays showed that <I>E. coli Nissle</I> expressing our part were found closer to Caco-2 cells than colonies expressing RFP only, indicating that the Harvard Biodesign BBa_K1580011 is intact and functional. Box (A) shows Caco-2 cells alone, and Box (B) and (C) show Caco-2 cells infected with engineered Nissle 1917. In Box (C) the 137 isoform failed to result in binding to Caco-2 cells. Box (B) shows the 49 isoform successfully bound to Caco-2 cells in suspension, concluding that the expression of RFP and fimH 49 did not interfere with each other, and robustly allowed for the visualization of engineered Nissle 1917 using our method. As an extra control, Box (D) shows wild type Nissle 1917 and Caco-2 cells. Physical binding was not seen in this group either.</P><DIV class="column full_size"><P>Box (E) and Box (F) are the same cells but without prior addition of RFP by rhamnose. They show no binding because <I>fimH</I> is not being induced.This part was sequenced however the output was not added to the parts registry because we believe the data to be of poor quality due to improper primer binding. However the order of the composite part was confirmed by detection of FimH in many of the sequence reads near the beginning while RFP was not detected in the start region.</P></DIV><DIV class="column full_size"><H4>References</H4><P><SMALL> 1. Reisch CR, Prather KLJ. Scarless Cas9 Assisted Recombineering (no-SCAR) in Escherichia coli, an Easy-to-Use System for Genome Editing. Curr Protoc Mol Biol. 2017 Jan 5;117:31.8.1-31.8.20. doi: 10.1002/cpmb.29.</SMALL></P><P><SMALL>2. Datsenko KA, Wanner BL.One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.
Proc Natl Acad Sci U S A. 2000 Jun 6;97(12):6640-5</SMALL></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>