<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFlorida_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFlorida/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:UFlorida"><DIV class="menu_item direct_link">
				HOME
			</DIV><DIV class="menu_item">
			TEAM
		</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Protocols"><DIV class="submenu_item">
					Protocols
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A></DIV><DIV class="menu_item">
			RESULTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="topnav" style="display: flex; align-items: flex-end; background-color: #333333; position: sticky; top: 18px;"><DIV class="dropdown"><A class="topnav" href="https://2018.igem.org/Team:UFlorida">Home</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Team">Members</A><A href="https://2018.igem.org/Team:UFlorida/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UFlorida/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Description">Description</A><A href="https://2018.igem.org/Team:UFlorida/Design">Design</A><A href="https://2018.igem.org/Team:UFlorida/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UFlorida/Protocols">Protocols</A><A href="https://2018.igem.org/Team:UFlorida/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UFlorida/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UFlorida/Model">Model</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Results</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:UFlorida/Parts">Parts</A><A href="https://2018.igem.org/Team:UFlorida/Results">Results</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Engagement</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UFlorida/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:UFlorida/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:UFlorida/Entrepreneurship">Entrepreneurship</A></DIV></DIV></DIV><DIV id="#main" class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Results</H1><P>✔ Deleted one gene copy of <I>frdA</I> from <I>E. coli Nissle 1917</I> genome
✔ Deleted <I>ldhA</I> from <I>E. coli Nissle 1917</I> genome
✔ Combined RFC and fimH BioBricks to create a detectable system of functional binding to colon cancer cells 
✔ Submitted the <I>ter</I> gene in pSB1C3 to the registry
✔ Observed butyrate peak via HPLC after fermenting <I>LdhA/FrdA</I> deleted <I>E. coli Nissle 1917</I> strain transformed with a completed butyrate pathway.</P><P>Below is a graph of butyrate production by strains of Nissle with lactate dehydrogenase (<I>ldhA</I>) and one copy of fumarate reductase (<I>frdA</I>) deleted. We subtracted the amount of butyrate found in the media from these values. <I>Nissle – ldhA</I> + 7004 represents this same strain transformed with BBa_K1587004, which provides a theoretical butyrate pathway that is not entirely NADH dependent and was created by the 2015 Toulouse iGEM team. <I>Nissle – ldhA</I> part represents this same deletion strain transformed with the butyrate part.</P><P>Based on these results, it appears that transforming the butyrate part and to some extent transforming BBa_K1587004 into <I>E. coli Nissle 1917</I> with ldha deleted results in increased amounts of butyrate concentrations in the supernatant compared to wild type.</P><P>However, more samples need to be run in order to be statistically certain of this. Directed evolution can be employed to know for sure if the pathway has the potential to produce butyrate in greater titers. These will both be attempted by the jamboree. Deleting more genes, such as adhE, can increase the potential for butyrate production in this strain further and approach those that were observed in BEM3 plus the butyrate part.</P><P>Below represents the results of our engineered part transformed into BEM3, a strain given to us by Dr. Shanmugam to test the butyrate production by our part.  BEM3 had several metabolite producing genes deleted, including <I>ldhA, adhE, frdA,</I> and others, but we did not delete these genes ourselves as this strain was given to us.</P><P>Adjusted for the amount of butyrate naturally occurring in the media, it is apparent that our part is producing butyrate in this strain. More samples will be run to confirm the statistical significance of this finding, and directed evolution will be employed before the Jamboree to increase the titer of butyrate production. pH controlled fermentations will also be considered. BBa_K1587004 did not produce any butyrate in this context, probably because it did not permit the survival of the strain under anaerobic growth (see OD’s).</P><P>In conclusion, our part appears to make butyrate and appears to be working better than BBa_K1587004. BBa_K1587004 appears to have a modest effect on butyrate production in <I>Nissle</I> with <I>ldhA</I> deleted, but more samples need to be run to increase confidence in the <I>Nissle</I> results.</P><H1>Future Directions</H1><P><B>Gene Deletions</B></P><P>Our butyrate-producing system will have the most stoichiometric efficiency after all the genes we sought to delete have been deleted. After every deletion, we will apply the new information to our model to improve its efficiency, and with it, predict the effects of future gene deletions on our system.</P><P><B>Directed Evolution</B></P><P>In an attempt to maximize the butyrate production in our engineered system, directed evolution can be used to select for a more efficient pathway. Dr. Frances Arnold, who was awarded the Nobel Prize in chemistry this year, proved that you can co-evolve enzymes making up an entire biosynthetic pathway. Using a similar method, we could evolve our system to upregulate enzymes pertinent for butyrate biosynthesis. </P><P><B>Testing</B></P><P>With an engineered system that produces substantial amounts of butyrate, we can begin to start solving some of the problems which we sought to change. </P><P><I>Probiotic</I> - With a bacterium that produces enough butyrate, we can patent and market our probiotic for preventative care of the gut microbiome. </P><P><I>Therapeutic </I>- Our butyrate-producing probiotic will be co-transformed with our submitted part, RFP-FimH. This will allow us to explore the known apoptotic effects on colon cancer cells under a fluorescent microscope as previously shown in the project. From there, we can move on to in vivo experiments in mice to further the use of the bacteria. </P><P><I>Microbes in Space </I>- Due to the advantages of microbes in space, showing our bacteria can be cultured in reduced gravity would prove useful in the case of using Nissle in space for its probiotic and industrial benefits.</P><P><B>Part Submission</B></P><P>There was no part submitted for the working butyrate production. This butyrate-pathway part will be registered as soon as digestion and PCR confirm that it is in the correct orientation and RFC10 compliant.</P><P><B>Reflections</B></P><P>Looking at how we could improve the project, we have concurred that individual enzyme upregulation would have been a better alternative. This would entail ordering the parts as the individual gene sequences (with promoter, RBS, terminator) and piecing them together one by one using 3A assembly. Assembly would be much simpler because there would be no lengthy sequences to amplify via PCR. This would significantly correspond to the butyrate metabolic model because every step would produce by-product concentration data to feed back into the model. Our model currently represents how each gene deletion affects the carbon flux, but it would benefit from a project design that added butyrate pathway enzymes one at a time. A more costly alternative would be to have the entire pathway synthesized as an IDT gene in a sequence verified plasmid.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>