"
Page
Discussion
View source
History
teams
Log in
 
Team:Carnegie Mellon/Overview
From 2012.igem.org
▼ Table of Contents
1. FAQ
1.1 Question 1
1.2 Question 2
1.2 Question 3
An Introduction to Promoters
Promoters are upstream sequences that regulate transcription. Promoters are usually short sequences and act as binding sites for a variety of different RNA polymerases. Promoters have different binding affinities based on their sequence and can be characterized in a matter of different ways. Our project looks to measure some of these properties using fluorescence measurements. In our case, we are characterizing promoters that bind to RNA polymerase from the T7 phage. The T7 RNA polymerase binds to its promoter very tightly and produces a high amount of expression. The lac operator is a short sequence that binds to the LacI repressor, which prevents transcription. The LacI protein responds to lactose in the cell. Lactose analogs have been made which are not consumed by E. coli and "turn on" the gene of interest. Our promoters have different affinities to the T7 RNA polymerase and the LacI repressor and therefore have different measurable properties.
What is fluorescence, exactly?
Fluorescence is a property of some molecules, particularly aromatic organic dyes, that allows them to absorb photons at a certain wavelength and emit them at a longer, lower energy wavelength. Fluorescence is described using quantum mechanics principles and organic chemistry. Five and six-member rings tend to fluoresce brightly because of electron delocalization and the quantum properties that are associated with it. Fluorescent molecules come in a variety of flavors and uses based on their properties, and shape. Fluorescent molecules are known as fluorophores and can take the form of organic dyes or proteins. So far, many different types of fluorophores have been discovered, developed and studied in great detail. Typically, fluorescent proteins have a fluorophore that consists of a few side chains that react and form a complex similar to that of an organic dye. For example, GFP (the most common fluorescent protein) has an HBI fluorophore. Our Spinach construct binds to a dye that derives from this fluorophore. The fluorescence of a molecule can depend on conformation, in the case of our fluorogens, malachite green and DFHBI, which are conditional fluorophores, the molecule must be in a certain conformation to fluoresce. Otherwise, it will absorb photons, but it will emit them very inefficiently (extremely low quantum yield). Fluorescence is a widely studied phenomena and a lot of research is involved with improving current fluorescence technologies and its applications.
What is Spinach?
Spinach is an RNA sequence that can be expressed in cells (in this case, E. coli ) and fluoresces green when DFHBI (an organic dye) is bound to it that can be used to quantify RNA concentration in a cell. Spinach binds to an organic dye called DFHBI which doesn't fluoresce by itself but fluoresces very brightly when it is bound to Spinach. DFHBI is chemically derived from the chromophore in GFP but is altered to increase brightness when bound to RNA. Other fluorescent RNAs have been described but many are non-specific and have many unwanted functions like cytotoxicity. Other fluorescent RNAs also are difficult to quantify because the cells' RNases (RNA destroying enzymes) can cut out the fluorescent sequence at unpredictable times, making quantification impractical. Spinach utilizes a scaffold that derives from a tRNA sequence, which disguises the RNA so that some RNases leave it alone. RNase A has been shown to degrade Spinach, however.  Manipulations to the sequence that Spinach is attached to allows for a variety of analyses functions. RNA can be arranged to bind to just about any small molecule in the same way that Spinach was developed (using a SELEX method) to track cellular metabolites. This allows for quantification of another important system in cells. However, in our system, Spinach is incorporated in the mRNA (between the promoter and the RBS) so mRNA is quantified. Click for more information on Spinach . Spinach is the first published RNA sequence of its kind and more sequence/dye combinations are in development; as a result, in years to come, multiple genes (both RNA and protein) can be analyzed in great detail simultaneously.
What is a FAP?
A fluorogen activating protein is a small (26-35kD) protein that derives from the variable region in an antibody. FAPs are not fluorescent unless a fluorogen (also not normally fluorescent) is added, in which case the FAP changes the conformation of the fluorogen and the complex fluoresces brightly. FAPs are currently used to tag certain proteins like actin or tubulin in mammalian cells. FAPs are primarily expressed in S. cerevisiae or mammalian cells although some variants have been expressed in E. coli. The two main dyes that the current series of FAPs bind to are malachite green and thiazole orange; our construct uses a variant that binds to malachite green. These dyes are normally cell impermeable but can be designed to penetrate cell membranes. As a result, they were originally used to tag surface proteins. FAPs are excellent reporters because they are small proteins that are soluble and have virtually no maturation time and are highly photostable unlike traditional variants of GFP. FAP technology is widely unexplored but shows promise for new fluorescent technology. FAPs have been used to track individual molecules to 5nm definition as opposed to the typical 200nm . Engineered dyes, called dyedrons, have been developed that increase fluorescence intensity and can allow researchers to improve on live cell imaging techniques. Different FAPs are genetically unique and respond to different excitation wavelengths so researchers can image multiple proteins at the same time in order to understand complex biological processes.
Why is this project important?
The ability to monitor protein production with fluorescence is a growing field that promises advances in drug development and improving quality control in drug manufacturing.
Promoter strength directly affects a cell's ability to perform typical functions like divide or move. Designing a genetic circuit that will not overload the cells is key in synthetic biology.
Inducible promoters are widely used in synthetic biology but many are under-characterized.
Safety Information
1. Would any of your project ideas raise safety issues in terms of researcher safety, public safety or environmental safety: Our project ideas do not raise any researcher safety issues. One of the dyes that we used is malachite green, which has the ability to cause low concentrations of free radicals as described by J.C. Liao et al .  Even though there is a risk of a few free radicals being formed during measurements, all assay-ed micro-plates were properly disposed of and malachite green is relatively safe by itself. We only used non-pathogenic Escherichia. coli, which is a biosafety level 1 organism.. To facilitate the selection of E. coli, we transformed E. coli strains (DH5-alpha and BL21(DE3)) with ampicillin resistant genes. We also used BL21 (DE3) pLysS strain for gene expression, which contains a chloramphenicol resistance gene. While these antibiotic resistant strains may pose a threat to public safety if released from the lab environment, all safety protocols were followed and cells were disposed according to institutional requirements.
Other than the use of antibiotic resistant strains as mentioned above, our project did not incorporate any biological components, which pose a threat to environmental safety. Regarding possible toxic chemicals used such as ethidium bromide for running gels, these chemicals were all disposed of according to institutional requirement. The researchers using ethidium bromide were required to read all of the MSDS forms and participate in chemical lab safety before handling the equipment.
2. Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues? If yes, did you document these issues in the Registry? How did you manage to handle the safety issue? How could other teams learn from your experience? issues? If yes, No, our parts themselves do not pose any risks.
3. Is there a local biosafety group, committee, or review board at your institution? There is a biological safety committee here at Carnegie Mellon University that is part of the Environmental Health and Safety department. We are currently in the process of obtaining approval, and awaiting confirmation from the Institutional Biosafety Committee. We obeyed all of the federal, state and local laws pertaining to the disposal of hazardous waste and biohazard waste (including liquids, solids and sharps). Our project used only biosafety level 1 organisms. All of the chemicals we use in the lab have been cleared for laboratory use.
4. Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering? One possible idea is to create an inducible (light-based) “kill-switch” in E. coli and yeast similar to the apoptotic pathways in mammalian cells using LovTAP. This would allow for easy disposal of cells, by simply placing the cells in a dark environment when we do not need them.  A similar idea is to create a strain of E. coli with a knocked out metabolic pathway, causing it to depend on a laboratory supplied environment to survive (such as a chemical or light). This would be a more passive safeguard against an accidental release of the bacteria.
J.C Liao, J Roider, D.G Jay. “Chromophore-assisted laser inactivation of proteins is mediated by the photogeneration of free radicals”. Proc. Natl. Acad. Sci. USA, 91 (1994), pp. 2659–2663
Retrieved from " http://2012.igem.org/Team:Carnegie_Mellon/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
