"
Page
Discussion
View source
History
teams
Log in
 
Team:Carnegie Mellon/Bio-Overview
From 2012.igem.org
Home
Overview
Introduction
Team
Attributions
Acknowledgements
BioBricks
Overview
Submitted Parts
Methods and Results
Overview
Results
Protocols
Challenges
Notebook
Safety
Modeling
Overview
Derivations
Matlab
Expanded
Human Practices
Overview
Outreach
Circuit Kit
Software
Team Presentation
Teaching Presentation
Promoter Variants
We have created three new T7Lac hybrid promoters that are mutated based on both the T7 promoter and the lac operator. The hybrid promoters enable high gene expression levels by T7 RNA polymerase, while allowing control of the expression levels by IPTG. All three biobricks have been submitted to the Registry of Standard Biological Parts, together with detailed characterization of both RNA and protein expression levels of the promoters ( See our method of analysis ).
T7 Promoter
The T7 promoter comes from the phage T7 and is a strong promoter. T7 promoters can only be expressed in strains of bacteria that express T7 RNA polymerase (the unique polymerase that activates gene expression from the T7 promoter). T7 promoters can be classified into different classes, and each class corresponds to a different level of gene expression level. The commonly used T7 promoter is the class I variant because of its ability to produce a high amount of protein. The T7 promoter has three distinct regions: a recognition site, a melting box, and an initiation site. The recognition site is the specific region that the T7 promoter binds to. The melting box is a highly conserved sequence, which is also known as the TATA box. The melting box allows the T7 RNAP to melt the two strands of DNA and start adding NTPs to build mRNA. The initiation site is where the first nucleotide of the mRNA is added. T7 RNAP favors the addition of guanine. As a result, most T7 promoters have a poly-G region of 3-5 nucleotides to increase the chance of initiation.
TAATACGACTCACTATAGGG
T7 region:
TAATACGACTCAC - Recognition site
TATA - Melting box
GGGAGA - Initiation site
LacO Operator and LacI Repressor
The LacO operator is a DNA sequence that is recognized by the LacI repressor. The LacI repressor is found in the lac operon of E. coli. When a LacO operator is located in a promoter region, LacI repressor prevents transcription from the promoter by forming a "hairpin" like structure that prevents RNAPs from traveling along the DNA. The LacI repressor cannot bind to the LacO operator when lactose is bound to LacI. This property can be exploited to prevent the expression of a certain gene by modulating the level of lactose or a lactose analog. Since lactose is consumed by E. coli, researchers found an analog (called Isopropyl β-D-1-thiogalactopyranoside IPTG) that binds to the lacI repressor, but is not consumed by E. coli. The wild type lac operator is nearly-symmetrical and has its own binding properties. In our constructs, we characterized a symmetrical lac operator and measured its leaky expression levels (the amount of expression without the inducer present).
Retrieved from " http://2012.igem.org/Team:Carnegie_Mellon/Bio-Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
