<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UT-Knoxville_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UT-Knoxville/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Welcome to UTK iGEM 2017</TITLE></P><DIV><H1 id="back_text">Welcome to UTK iGEM 2018</H1><BUTTON id="back_button" type="button" onclick="window.location.href='https://igem.org/Main_Page'">Learn More!</BUTTON></DIV><NAV class="utkIgem" id="nav_bar"><A href="https://2018.igem.org/Team:UT-Knoxville/Home" id="nav_links" class="utkIgem">HOME</A><A href="https://2018.igem.org/Team:UT-Knoxville/Team" id="nav_links" class="utkIgem">MEET THE TEAM</A><A href="https://2018.igem.org/Team:UT-Knoxville/Demonstrate" id="nav_links" class="utkIgem">PROJECT</A><A href="https://2018.igem.org/Team:UT-Knoxville/Collaborations" id="nav_links" class="utkIgem">COLLABORATIONS</A><A href="https://2018.igem.org/Team:UT-Knoxville/Attributions" id="nav_links" class="utkIgem">ATTRIBUTIONS</A><A href="https://2018.igem.org/Team:UT-Knoxville/InterLab" id="nav_links" class="utkIgem">INTERLAB</A><A href="https://2018.igem.org/Team:UT-Knoxville/Human_Practices" id="nav_links" class="utkIgem">HUMAN PRACTICES</A><A href="https://2018.igem.org/Team:UT-Knoxville/Judging" id="nav_links" class="utkIgem">JUDGING</A><A href="https://2018.igem.org/Team:UT-Knoxville/Conference" id="nav_links" class="utkIgem">PAST CONFERENCES</A><A href="https://2018.igem.org/Team:UT-Knoxville/Pictures" id="nav_links" class="utkIgem">PICTURES</A></NAV><DIV id="utkIgemContent"><H1>Exploring the Bioremediation and Environmental Impact of Dehalogenating Microbes</H1><H2>Abstract</H2><P>Chemical pollution resulting from large-scale industrial practices can result in volatile organic compound (VOC) accumulation in water supplies. One VOC of interest, dichloroacetate (DCA), is a chlorinated carcinogenic contaminant at clinically high levels. Similarly, dichloromethane (DCM), is used for various industrial applications but its accumulation in water systems poses a threat to aquatic organisms and is considered a carcinogenic to humans. The goal of the UT Knoxville iGEM Team is to design biological systems in E. <I>coli</I> capable of degrading DCA and DCM in order to remove them from the water supply and metabolize them within the cell. Through the addition of Haloacid Dehalogenase (HADase) genes capable of breaking down DCA as well as the development of a DCM biosensor, we are generating biological organisms in order to facilitate our access to clean drinking water.</P><H1>HADase Homologs Project</H1><H2>Introduction</H2><P>Dichloroacetate (DCA) is a halogenated organic compound with profound implications towards human health. The chemical is considered a wasteful by-product from industrial applications, and is prevalent within surface water as an environmental contaminant; it is also developed as a result of natural geochemical processes (1). It is relatively difficult to understand the environmental toxicity of DCA because it is difficult to identify its environmental concentration. However, DCA is known to have a major impact on human health at clinically high levels of exposure (2).</P><P>Previous studies have identified the existence of microbes capable of degrading DCA by utilizing an enzyme known as haloacid dehalogenase (HADase) (3). HADases dehalogenate the chlorinated compound, converting DCA into glyoxylic acid; this product is subsequently metabolized by the cell.</P><P>Additionally, HADases are globally prevalent throughout the environment. Recent research has indicated the extensive prevalence of dehalogenases throughout the human gut microbiome, and further metagenomic analysis indicates the existence of dehalogenase-coding genes within a broad variety of different environments (4). The substantial existence and conservation of these HADase genes within the global environment indicates the impact of HADase proteins in response to chlorinated organic compounds such as DCA.</P><P>While the original intention of this project was to develop a metabolic system capable of degrading an environmental compound, the scope of this project has been impacted by the global prevalence of dehalogenating microbes. Metagenomic analysis indicates the conservation of dehalogenating systems through a broad variety of different environments; each HADase-containing organism potentially displays a unique story on the context of the protein itself. This year, the UTK iGEM team focused on developing a metabolic system capable of degrading dichloroacetate while studying the global significance of dehalogenating organisms.</P><H2>The Metabolic Approach: Designing a Dehalogenating Organism </H2><P> The primary objective for UTK iGEM is to create a biological system capable of breaking down chlorinated solvents. In order to genetically engineer E. <I>coli</I> capable of doing so, we must utilize identify and analyze potential HADase sequences for cloning. By utilizing the basic alignment search tool (BLAST) on NCBI, a broad variety of potential HADase genes can be identified. To optimize the BLAST, a characterized HADase gene was utilized to identify homologous HADase genes and protein sequences; the HADase sequence of the anaerobic organism Candidatus D. elyunquensis was used as our working sequence. With a protein sequence &gt;50% similar to the working sequence, we decided to identify and synthesize five HADase genes originating from different organisms on the NCBI database.</P><H2>The Metagenomic Approach: Exploring the Global Prevalence of HADase Proteins</H2><P>Another objective for UTK iGEM was to explore global HADase proteins and expand scientific knowledge by testing and integrating metagenomic HADase proteins into a biological system. With a help of an expert in the field, Dr. Robert Murdoch, we acquired a robust dataset of a variety of different metagenomic HADases; these HADases originate from metagenomic samples from environments around the world. In order to explore the global significance of HADase, a total of ten different HADase sequences were selected from Dr. Murdoch’s metagenomic dataset; these HADase sequences represent a robust set of different environments and allow us to expand our human practices by ensuring that the data collected for this work has both an engineering and environmental relevance based on our out of lab research. </P><H2>Cloning the HADase Sequences:</H2><P>Ten of these sequences were obtained from our metagenomic datasets, and come from a broad range of different environments. Five of these sequences were identified via BLAST; these HADase sequences have highly similar (&gt;50%) amino acid sequences with near identical active site residues. The metagenomic amino acid sequences are highly dissimilar giving us a broad variety of HADase sequences to work with.</P><P>Because of the robust variety of HADases we are working with, Gibson assembly cloning was utilized. This allows us to order HADase sequences with homologous ends corresponding to the vector, which allows us to use only one set of primers to amplify all HADase sequences via PCR (gel). The HADase sequences were incorporated into the NDEI/SalI site of the pET-28 a (+) vector. </P><P>Upon transformation of the HADase vector into E. <I>coli</I>, the HADase vector was verified via colony PCR; the results of HADase verification are shown in the gel below for HADase 1 and HADase 12. HADase 1 originates within the hydrothermal bacteria Kyrpidia sp. EA-1. HADase 12 is a metagenomic HADase environmentally originating in an active sludge microbial community from Klosterneuburg, Austria. Both of these sequences contain drastically different origin stories, carrying the same metabolic capability of degrading chlorinated organic compounds.</P><H2>Demonstration: Protein expression and verification of HADase 1  and HADase 12</H2><P>To verify protein expression, IPTG-induced protein activation and HPLC analysis were used. Firstly, we created a pre-batch of our K1 DNA vector and a BL21 DE3 control with an empty backbone. The medium we used for our pre-culture was LB broth with Kan-50. The pre-batch grew overnight at 37°C at a RPM 220. </P><P>Once the pre-batch was complete, we acquired two 250mL flasks that had been autoclaved and added 50mL of LB and Kan-50 to both flasks. We then inoculated with 1mL (1/50 Dilution)  of pre-batch culture. Our cultures incubated for 8 hours at 37°C at 220 RPM. IPTG was added, at a 1mM final concentration, to our culture and grown overnight. The next day we took the culture from the flask and added them into falcon tubes. We made this switch as we needed to centrifuge our cell culture so that we could remove the cells from the medium. After centrifuging, we removed the supernatant and resuspended the cells in Tris-Buffer. We then used a sonication machine to lyse our cells and release our HADase proteins into the supernatant. We then centrifuged cell materials could be removed leaving supernatant (crude cell extract) rich in HADases.</P><P>In order to test our first HADase, HADase1, we created three testing groups. For group 1 used 50uL of crude cell extracts to test, group 2 we used 20uL of crude cell extract, and lastly group 3 we added 50uL of control crude cell extract. We added 3.5mM of DCA to each group and quickly took a sample. The results of this test can be observed below in <B>Cell extracts of strain HAD1 + DCA</B>: 20 and 50 uL extracts. Each sample was added to a microcentrifuge tube with 1 uL of sulfuric acid to denature the HADase protein and stop its catalytic activity. For the HPLC, the absorbance peaks of DCA and Glyoxylate were pre-measured for reference before any samples were measured. We then had timed-interval samples, i.e. we took samples at 0, 10, 30, 45, 60, 120, 240, and 360 minutes. At time t=0, DCA was first added to the crude cell extract. We tested very frequently since the Km of our protein was unknown. With a better understanding of the HADase's substrate binding efficiency, a second test was run with a far higher crude cell volume of 500uL crude cell extract. Clearer data was obtained, demonstrating that our enzyme successfully functions by degrading dichloroacetate. Degradation of DCA by HADase1 overtime in the second trial is depicted below in <B>500 µl Cell Extracts of Strain HAD1 + DCA </B>. 
</P><P>
To further demonstrate the efficacy of our HADase1 enzyme and its potential as a more general tool for removing chlorinated solvents, we decide to test the enriched crude extract using monochloroacetate (MCA). As can be seen in <B>500 µl cell extracts of strain HAD1 + MCA</B> our extract was capable of rapidly breaking down MCA giving us evidence that HADase1 could be used as a general tool for the removal of chlorinated solvents. As the need to remove multiple pollutants is necessary in the real-world applications, this shows a significant step towards being able to develop organisms that can effectively remove pollutants from contaminated water sources. </P><P>Acquired from a metagenomic dataset of an Austrian wastewater sample, the HADase12 was prepared and tested in the same manner as HADase1. However, as opposed to HADase1, HADase12 was chosen as it was part of our approach towards understanding dehalogenation in the environment. Our goal was to examine the prevalence of functioning dehalogenases as it could provide clues and new avenues of research for the importance of dehalogenation in the environment. After testing HADase12 we found that similarly to HADase1 it could be used to breakdown DCA to completion as seen in <B>Cell extracts of strain HAD12 + DCA
</B>. Our findings that both of these enzymes are functional dehalogenases showcases their prevalence in varied environments and provides an impetus for our continued work examining their ecological importance. Our future work will build on the success of this project and examine both the remaining HADases and their ability to break down multiple substrates.</P><DIV class="center"><H1 style="overflow:visible;">Developing a Biosensor for Dichloromethane </H1></DIV><P>This project sought to create a biosensor that would detect dichloromethane in wastewater. Dichloromethane is commonly used in industrial practices for paint removal, metal cleaning, chemical processing, and pharmaceutical manufacturing. Its accumulation in water systems, however, poses a threat to the health of both humans and aquatic life due to its carcinogenic effects. By creating a biological device that will produce a measureable signal in the presence of DCM, we can contribute the its removal from the environment. The intensity of the signal will correspond to the concentration of DCM, thus giving a better idea of the severity of contamination</P><DIV><DIV class="center"><H2>Design Inspiration</H2></DIV><P>Using metagenomic data provided to the team by bioinformatician Dr. Robert Murdoch, a set of genes highly conserved across DCM degrading organisms was used to form the basis for the potential biosensor. The first four genes in this 11kb region are activated in the presence of DCM. They are known as the regulatory region. By putting them in our biosensor construct, the genes will activate when exposed to DCM and allow a downstream reporter protein to fluoresce, indicating the pollutant's presence. In order for the genes to activate, however, the promoter region must also be present in the construct. The first step in reaching the goal of DCM mitigation was thus designing an intermediate biosensor construct to test different segments of the conserved 11kb region to identify the promoter's location.</P></DIV><DIV><DIV class="center"><H2>Construct Design</H2></DIV><P>Two different testing constructs were designed for the potential biosensor. Both constructs were designed as fragments to be constructed using Gibson Assembly and utilized a pET-28A(+) backbone. For the first construct, the four-gene regulatory region is used as the first fragment. It was designed to be inserted as one piece along with all the innate intergenic regions. The next fragment contains a terminator as well as a multiple cloning site (MCS). The MCS is where the construct can be cut open to insert the promoter testing regions. Finally, the third fragment contains a ribosomal binding site and the green fluorescent protein (GFP) which will indicate the presence of DCM.</P></DIV><P style="font-size:20px;">The second testing construct is designed very similarly, but with one difference. The four-gene regulatory region is inserted as four separate fragments for each individual gene. E. <I>coli</I> specific ribosomal binding sites were also designed for each gene fragment rather than using those from the intergenic regions. Again, the last two fragments contain the same MCS and GFP sequences.</P><H2>Assembly and Testing</H2><P>The next step after the constructs were designed was to design primers for each fragment to be constructed through Gibson assembly. These primers were used to amplify each fragment through PCR. Once all fragments are successfully amplified, they should be assembled together using Gibson master mix and transformed into E. <I>coli</I>. With successful transformation of the testing constructs, different regions of the remaining 11kb conserved region can be inserted through the MCS and tested in the presence of known DCM concentrations until the correct promoter region is found.</P><H1>References</H1><OL><LI>&quot;Stacpoole PW, Henderson GN, Yan Z, Cornett R, James MO. Pharmacokinetics, metabolism and toxicology of dichloroacetate. Drug Metab Rev. 1998b;30:499–539. </LI><LI> Stacpoole PW. 2011. The Dichloroacetate Dilemma: Environmental Hazard versus Therapeutic Goldmine—Both or Neither? Environmental Health Perspectives. 119(2), 155-158. doi: 10.1289/ehp.1002554 </LI><LI>Löffler, F. E., J. R. Cole, K. M. Ritalahti, and J. M. Tiedje. 2003. Diversity of dechlorinating bacteria, p. 53-87. In M. M. Häggblom and I. D. Bossert (eds.), Dehalogenation: microbial processes and environmental applications. Kluwer Academic, New York.
  </LI><LI>Atashgahi, S., Shetty, S.A., Smidt, H., de Vos, W.M. 2018. Flux, Fate, and Impact of Halogenated Organic Compounds in the Gut. Frontiers in Physiology. 114(9). doi: 10.3389/fphys.2018.00888 </LI><LI>“Halogenated Solvents Industrial Alliance.” Halogenated Solvents Industry Alliance Inc : Uses, www.hsia.org/uses.asp.</LI><LI>Shestakova, M. and M. Sillanpää (2013). &quot;Removal of dichloromethane from ground and wastewater: A review.&quot; Chemosphere 93(7): 1258-1267.</LI><LI>Groundwater Monitoring, Philippe Quevauviller (Editor), Dr. Anne Marie Fouillac, Mr. Johannes Grath, Dr. Rob Ward ISBN: 978-0-470-77809-8</LI><LI>“Fact Sheet: Methylene Chloride or Dichloromethane (DCM).” EPA, Environmental Protection Agency, 10 May 2018, www.epa.gov/assessing-and-managing-chemicals-under-tsca/fact-sheet-methylene-chloride-or-dichloromethane-dcm-0#used.</LI><LI>Wright, J., Kirchner, V., Bernard, W., Ulrich, N., McLimans, C., Campa, M. F., … Lamendella, R. (2017). Bacterial Community Dynamics in Dichloromethane-Contaminated Groundwater Undergoing Natural Attenuation. Frontiers in Microbiology, 8, 2300. http://doi.org/10.3389/fmicb.2017.02300</LI><LI>Plekhanova, Y. V., et al. (2013). &quot;Aerobic methylobacteria as the basis for a biosensor for dichloromethane detection.&quot; Applied Biochemistry and Microbiology 49(2): 188-193.</LI></OL></DIV><DIV class="utkIgemFooter"><H1>CONNECT WITH iGEM ON SOCIAL MEDIA!</H1></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>