Team:Lambert GA/Attributions
Home
Team
Team Collaborations
Project
Description Design Experiments Notebook InterLab Model Results Demonstrate Attributions
Parts
Part Collection Updated Parts Composite Parts
Safety
Human Practices
Silver HP Integrated and Gold Public Engagement
Awards
Applied Design Hardware Model Software
Judging Form
Attributions and Acknowledgements
Attributions
Research
Constructs: Lauren Hong, Julia Leveille, Katie Barr, Gaurav Byagathavalli, Ellie Kim, Alyssa Franklin
Measurement Model: David Standeven, Emily Gibson, Nithik Balachandran, Natalie Shih
Software: Emily Gibson
Labwork
Labwork- Katie Barr, Gaurav Byagathavalli, Alyssa Franklin, Lauren Hong,  Ellie Kim, Julia Leveille, Christina Lee, Kevin Li,Natalie Shih, David Standeven
Proof of Concept- Chromoproteins: Natalie Shih, Alyssa Franklin
Chrome-Q
Hardware: David Standeven, Emily Gibson
Software: Emily Gibson, Michael Gibson
Wiki
Design: Nithik Balachandran, Kevin Li, Julia Leveille
Design Support: Ellie Kim, Christina Lee, Noora Chandasir
Information/Writing: Lambert's iGEM Team
Pictures/Diagrams: Lambert's iGEM Team
Design
Spiritwear, Banner, Animations: Ellie Kim
Poster: Noora Chandasir, Ellie Kim, Christina Lee
Handouts: Ellie Kim, Christina Lee
Ethics
Katie Barr, Kevin Li , David Standeven, Lauren Hong, Christina Lee, Ellie Kim, Gaurav Byagathavalli
Outreach
The Place:  Donated blankets and toiletries: Katie Barr, Julia Leveille
Human Practices: Julia Leveille, Katie Barr, Christina Lee, Ellie Kim, Megan Hong
Presentation
Presenters: Gaurav Byagathavalli, Emily Gibson, Lauren Hong
Script/Other Help: Natalie Shih, Julia Leveille
Acknowledgements
General Support
Dr. Mark Styczynski: Associate Professor at Georgia Institute of Technology
Monica McNerney: Doctoral Student at Georgia Institute of Technology
In addition to providing resources, Monica McNerney and Dr. Styczynski assisted Lambert iGEM through its project by acting as mentors and aiding in the problem-solving conversations. There have been multiple conference calls in which Dr. Styczynski and Monica McNerney helped the Lambert iGEM team troubleshoot and revise its project. Dr. Styczynski has also generously provided materials that a high school lab does not have access to. We would like to thank them for helping us complete our project.
Dr. Saad Bhamla generously provided the design files for 3-D printing our own paperfuge.  He also provided feedback on our designs for the improved version.
Dr. Anna Karls: Associate Professor at University of Georgia
Lambert iGEM received advice from  Dr. Karls, a microbiology professor at the University of Georgia. She gave feedback about how to assemble our construct more effectively.
Parts Support
Snapgene: Software for sequence analysis
IDT: Synthesis of g blocks
Funding/Materials Support
Lambert High School
National Science Foundation Grant #1254382
Gibson Ridge Software
Dr. Dana Barr Emory University
AGCO
Crystal
Gofundme
Johns Creek Chick fil A
Hazel Arnold
Wiki Support
Trevor Starling: Wiki Advice and Help
Outreach Support
Lambert Biotech Club
The Place
Atlanta Maker Faire
Sharon Elementary School
Sources
And, S. A. (2009, February 13). Sarita Ahlawat. ClpXP Degrades SsrA-tagged proteins in S.pneumoniae.Retrieved Summer, 2017, from http://jb.asm.org/content/191/8/2894.full
Andersen , J.B. , Sternberg , C. , Poulsen , L.K. , Bjorn , S.P. , Givskov , M. , and Molin , S. ( 1998 ) New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria . Appl Environ Microbiol 64 : 2240 – 2246 .
Baker, T. A., & Sauer, R. T. (2011, June 27). ClpXP, an ATP-powered unfolding and protein-degradation machine. Retrieved Summer, 2017, from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209554/
Bar-Nun, S., & Glickman, M. H. (2012). Proteasomal AAA-ATPases: Structure and function. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research, 1823(1), 67–82. doi:10.1016/j.bbamcr.2011.07.009. Retrieved Summer, 2017 from http://www.sciencedirect.com/science/article/pii/S0167488911001984
Bohn , C. , Binet , E. , and Bouloc , P. ( 2002 ) Screening for stabilization of proteins with a trans-translation signature in Escherichia coli selects for inactivation of the ClpXP protease . Mol Genet Genomics 266 : 827 –831 .
Burton , R.E. , Siddiqui , S.M. , Kim , Y.I. , Baker , T.A. , and Sauer , R.T. ( 2001 ) Effects of protein stability and structure on substrate processing by the ClpXP unfolding and degradation machine . EMBO J 20 : 3092 –3100 .
Ciechanover, A. (2005). Cell death and differentiation - abstract of article: Intracellular protein degradation: From a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting[ast]. Cell Death & Differentiation, 12(9), 1178–1190. doi:10.1038/sj.cdd.4401692
Cooper, G. M. (2000). Protein degradation. Retrieved Summer, 2017 from http://www.ncbi.nlm.nih.gov/books/NBK9957/
Dhakar, L. (n.d.). Image Color Picker (Z. A., Ed.). Retrieved October 10, 2017, from http://www.colorcodepicker.com/
Farrell, C., Grossman, A., & Sauer, R. (2005). Cytoplasmic degradation of ssrA-tagged proteins.Molecular microbiology., 57(6), 1750–61. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/16135238
Flynn , J.M. , Levchenko , I. , Seidel , M. , Wickner , S.H. , Sauer , R.T. , and Baker , T.A. ( 2001 ) Overlapping recognition determinants within the ssrA degradation tag allow modulation of proteolysis . Proc Natl Acad Sci USA 11 : 10584 – 10589.
Georgia Institute of Technology. (2015, September 1). “Bacterial litmus Test” provides inexpensive measurement of Micronutrients. Retrieved from GT News Center, http://www.news.gatech.edu/2015/09/01/bacterial-litmus-test-provides-inexpensive-measurement-micronutrients
Goldberg, A.L., A.S. Menon, S. Goff and D.T. Chin. 1987. The mechanism and regulation of the ATP-dependent protease La from Escherichia coli. Biochem. Soc. Trans. 15: 809-811. Retrieved October 1, 2017 from http://www.fao.org/wairdocs/ilri/x5550e/x5550e0d.htm
Hwang BJ, Woo KM, Goldberg AL, Chung CH. Protease Ti, a new ATP-dependent protease in Escherichia coli,contains protein-activated ATPase and proteolytic functions in distinct subunits. J Biol Chem. 1988;263:8727–8734.
Katayama-Fujimura Y, Gottesman S, Maurizi MR. A multiple-component, ATP-dependent protease from Escherichia coli. J Biol Chem. 1987;262:4477–4485.
Landry, B. P., & Stöckel, J. (2013). Use of degradation tags to control protein levels in the Cyanobacterium Synechocystis sp. Strain PCC 6803. Applied and Environmental Microbiology,79(8), 2833–2835. doi:10.1128/AEM.03741-12
Léa, D. (2017). Les meilleures banques en ligne - Banque en ligne de l'Université Cefi. Retrieved October 18, 2017, from http://univ-cefi.fr/les-meilleures-banques-en-ligne.html
Lee C, Schwartz MP, Prakash S, Iwakura M, Matouschek A. ATP-Dependent Proteases Degrade Their Substrates by Processively Unraveling Them from the Degradation Signal.
McNerney, M. P., Watstein, D. M., & Styczynski, M. P. (2015). Precision metabolic engineering: The design of responsive, selective, and controllable metabolic systems. Metabolic Engineering, 31, 123–131. doi:10.1016/j.ymben.2015.06.011
Minikel, E. V. (2013, June 11). Basics of protein degradation. Retrieved Summer, 2017, from http://www.cureffi.org/2013/07/11/basics-of-protein-degradation/
Mogk A, Schmidt R, Bukau B. The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies. Trends Cell Biol. 2007;17:165–172.
Purcell, O., Grierson, C. S., Bernardo, M. di, & Savery, N. J. (2012). Temperature dependence of ssrA-tag mediated protein degradation. Journal of Biological Engineering, 6(1), . doi:10.1186/1754-1611-6-10
Purple color codes. (n.d.). Retrieved October 10, 2017, from http://www.rapidtables.com/web/color/purple-color.htm
RGB Color Gradient Maker. (n.d.). Retrieved October 10, 2017, from http://www.perbang.dk/rgbgradient/
Schrader, E. K., Harstad, K. G., & Matouschek, A. (n.d.). Targeting proteins for degradation. , 5(11), . Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228941/
Snider, J., Thibault, G., & Houry, W. A. (2008). The AAA+ superfamily of functionally diverse proteins. , 9(4), . Retrieved Summer, 2017 from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2643927/
Tamura, K., Shimada, T., Ono, E., Tanaka, Y., Nagatani, A., Higashi, S., . . . Hara-Nishimura, I. (2003, September). Why green fluorescent fusion proteins have not been observed in the vacuoles of higher plants. The Plant Journal, 35(4), 545-555. doi:10.1046/j.1365-313X.2003.01822.x
Tanaka K. The proteasome: overview of structure and functions. Proc Jpn Acad Ser B Phys Biol Sci.2009;85:12–36.
Tao, L., & Biswas, I. (2015). Degradation of SsrA-tagged proteins in streptococci. , 161(Pt 4),. Retrieved September 9, 2017 from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857447/
Tu, D., Lee, J., Ozdere, T., Lee, T. J., & You, L. (2007, January ). Engineering Genetic Circuits: Foundations and Applications. Retrieved from http://people.duke.edu/~you/publications/Tu_etal_SyntheticBiology.pdf
Watstein, D. M., McNerney, M. P., & Styczynski, M. P. (2015). Precise metabolic engineering of carotenoid biosynthesis in Escherichia coli towards a low-cost biosensor. Metabolic Engineering,31, 171–180. doi:10.1016/j.ymben.2015.06.007
Sponsors
