Team:Lambert GA/Description
Home
Team
Team Collaborations
Project
Description Design Experiments Notebook InterLab Model Results Demonstrate Attributions
Parts
Part Collection Updated Parts Composite Parts
Safety
Human Practices
Silver HP Integrated and Gold Public Engagement
Awards
Applied Design Hardware Model Software
Judging Form
Abstract
Performing synthetic biology research requires the use of expensive equipment, which is problematic for underfunded laboratories in developing nations, new research facilities, and high school programs. There is a demand for low-cost, easily accessible laboratory equipment for manual centrifuging and plate reading. In order to address this need, an imaging device and an analyzing software app were designed to quantify the HSV color space from genetically recombined E.coli cells containing inducible chromoprotein constructs. The engineered chamber, Chrome-Q, captures controlled data by eliminating external light, which allows the Android compatible mobile app to calculate the HSV color space from predetermined RGB values. To inexpensively and efficiently pellet transformed cells, an existing 3-D-fuge design was optimized for integration with protocols developed for the Chrome-Q system. Proof of concept was achieved by the creation of an induction curve using three different chromoproteins from ATUM’s Protein Paintbox and the assembled genetic circuits: Promoter-R0040, Ribosomal Binding Site B0034, and Scrooge Orange/Virginia Violet/Tinsel Purple (TsPurple)-BBa_K1033906, using the Chrome-Q system. This project provides economically feasible devices and methods for underfunded research labs working with chromoprotein color expression.
Description
Purpose
When researching synthetic biology, underfunded labs often lack the monetary means to purchase expensive laboratory equipment; basic machinery, such as centrifuges, or more complex materials for analyzing data, such as plate readers, are commodities for many laboratories. The 2017 Lambert iGEM team decided to focus upon these issues, and, therefore, strives to make expensive equipment, like plate readers and centrifuges, more viable resources for underfunded labs.
Method
The approach Lambert iGEM used was multifaceted:
Design a chamber to optimize and standardize conditions for detecting color
Write a software application that can be used by anyone to quantify the color values using HSV
Test and improve on a low-cost centrifuge
Assemble and test these devices with chromoproteins under varying levels of IPTG induction
Using engineering design principles, the 2017 Lambert iGEM team aimed to develop an imaging device called the Chrome-Q to cost-effectively standardize conditions for quantifying chromoprotein data; additionally, the team improved upon the 3-D-fuge originally designed by the Prakash Lab to pellet cells at a low cost. Different chromoproteins - Tinsel Purple, Virginia Violet, and Scrooge Orange - were induced at varying levels of IPTG, causing the cells to express different intensities; pictures of the cells were taken using a Chrome-Q and were measured with a self-constructed software app that analyzes the pigments using HSV color space. Additionally, the team observed that a common issue encountered by synthetic biologists is the overlap of protein expression in the development of genetic circuits which negatively affects the quality of performance in a given cell. The 2017 Lambert iGEM Team worked to develop an inducible “switch” to further characterize ClpXP, a non-lysosomal proteolysis mechanism, with three Keio strains; the data showed a correlation between IPTG concentration and amount of protein degradation. Overall, Lambert iGEM desired to promote scientific research under any financial circumstance and to characterize non-lysosomal inducible protein degradation (CLiP’d).
Designed construct used to develop an inducible "switch"
Animation of ClpXp Protease
References
And, S. A. (2009, February 13). Sarita Ahlawat. ClpXP Degrades SsrA-tagged proteins in S.pneumoniae.Retrieved Summer, 2017, from http://jb.asm.org/content/191/8/2894.full
Andersen , J.B. , Sternberg , C. , Poulsen , L.K. , Bjorn , S.P. , Givskov , M. , and Molin , S. ( 1998 ) New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria . Appl Environ Microbiol 64 : 2240 – 2246 .
Baker, T. A., & Sauer, R. T. (2011, June 27). ClpXP, an ATP-powered unfolding and protein-degradation machine. Retrieved Summer, 2017, from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209554/
Bar-Nun, S., & Glickman, M. H. (2012). Proteasomal AAA-ATPases: Structure and function. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research, 1823(1), 67–82. doi:10.1016/j.bbamcr.2011.07.009. Retrieved Summer, 2017 from http://www.sciencedirect.com/science/article/pii/S0167488911001984
Bhamla, M. Saad, et al. “Hand-Powered Ultralow-Cost Paper Centrifuge.” Nature News, Nature Publishing Group, 10 Jan. 2017, www.nature.com/articles/s41551-016-0009.
Burton , R.E. , Siddiqui , S.M. , Kim , Y.I. , Baker , T.A. , and Sauer , R.T. ( 2001 ) Effects of protein stability and structure on substrate processing by the ClpXP unfolding and degradation machine . EMBO J 20 : 3092 –3100 .
Ciechanover, A. (2005). Cell death and differentiation - abstract of article: Intracellular protein degradation: From a vague idea thru the lysosome and the ubiquitin-proteasome system and onto human diseases and drug targeting[ast]. Cell Death & Differentiation, 12(9), 1178–1190. doi:10.1038/sj.cdd.4401692
Cooper, G. M. (2000). Protein degradation. Retrieved Summer, 2017 from http://www.ncbi.nlm.nih.gov/books/NBK9957/
Farrell, C., Grossman, A., & Sauer, R. (2005). Cytoplasmic degradation of ssrA-tagged proteins.Molecular microbiology., 57(6), 1750–61. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/16135238
Flynn , J.M. , Levchenko , I. , Seidel , M. , Wickner , S.H. , Sauer , R.T. , and Baker , T.A. ( 2001 ) Overlapping recognition determinants within the ssrA degradation tag allow modulation of proteolysis . Proc Natl Acad Sci USA 11 : 10584 – 10589.
Georgia Institute of Technology. (2015, September 1). “Bacterial litmus Test” provides inexpensive measurement of Micronutrients. Retrieved from GT News Center, http://www.news.gatech.edu/2015/09/01/bacterial-litmus-test-provides-inexpensive-measurement-micronutrients
Goldberg, A.L., A.S. Menon, S. Goff and D.T. Chin. 1987. The mechanism and regulation of the ATP-dependent protease La from Escherichia coli. Biochem. Soc. Trans. 15: 809-811. Retrieved October 1, 2017 from http://www.fao.org/wairdocs/ilri/x5550e/x5550e0d.htm
Hwang BJ, Woo KM, Goldberg AL, Chung CH. Protease Ti, a new ATP-dependent protease in Escherichia coli,contains protein-activated ATPase and proteolytic functions in distinct subunits. J Biol Chem. 1988;263:8727–8734.
Katayama-Fujimura Y, Gottesman S, Maurizi MR. A multiple-component, ATP-dependent protease from Escherichia coli. J Biol Chem. 1987;262:4477–4485.
Landry, B. P., & Stöckel, J. (2013). Use of degradation tags to control protein levels in the Cyanobacterium Synechocystis sp. Strain PCC 6803. Applied and Environmental Microbiology,79(8), 2833–2835. doi:10.1128/AEM.03741-12
Lee C, Schwartz MP, Prakash S, Iwakura M, Matouschek A. ATP-Dependent Proteases Degrade Their Substrates by Processively Unraveling Them from the Degradation Signal.
McNerney, M. P., Watstein, D. M., & Styczynski, M. P. (2015). Precision metabolic engineering: The design of responsive, selective, and controllable metabolic systems. Metabolic Engineering, 31, 123–131. doi:10.1016/j.ymben.2015.06.011
Minikel, E. V. (2013, June 11). Basics of protein degradation. Retrieved Summer, 2017, from http://www.cureffi.org/2013/07/11/basics-of-protein-degradation/
Mogk A, Schmidt R, Bukau B. The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies. Trends Cell Biol. 2007;17:165–172.
Purcell, O., Grierson, C. S., Bernardo, M. di, & Savery, N. J. (2012). Temperature dependence of ssrA-tag mediated protein degradation. Journal of Biological Engineering, 6(1), . doi:10.1186/1754-1611-6-10
Schrader, E. K., Harstad, K. G., & Matouschek, A. (n.d.). Targeting proteins for degradation. , 5(11), . Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228941/
Snider, J., Thibault, G., & Houry, W. A. (2008). The AAA+ superfamily of functionally diverse proteins. , 9(4), . Retrieved Summer, 2017 from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2643927/
Tanaka K. The proteasome: overview of structure and functions. Proc Jpn Acad Ser B Phys Biol Sci.2009;85:12–36.
Tao, L., & Biswas, I. (2015). Degradation of SsrA-tagged proteins in streptococci. , 161(Pt 4),. Retrieved September 9, 2017 from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857447/
Tu, D., Lee, J., Ozdere, T., Lee, T. J., & You, L. (2007, January ). Engineering Genetic Circuits: Foundations and Applications. Retrieved from http://people.duke.edu/~you/publications/Tu_etal_SyntheticBiology.pdf
Watstein, D. M., McNerney, M. P., & Styczynski, M. P. (2015). Precise metabolic engineering of carotenoid biosynthesis in Escherichia coli towards a low-cost biosensor. Metabolic Engineering,31, 171–180. doi:10.1016/j.ymben.2015.06.007
