"
Page
Discussion
View source
History
teams
Log in
 
Team:Newcastle University/Biological Brainstorming
From 2008.igem.org
Newcastle University
GOLD MEDAL WINNER 2008
Home
Team
Original Aims
Software
Modelling
Proof of Concept Brick
Wet Lab
Conclusions
Home >> Original Aims >> Biological Brainstorming
How polymorphic are the quorum sensing peptides? The literature suggest that their sequence varies so much that they can be used to type strains. Can we identify a profile that is unique to MRSA strains? What about Bacillus anthracis or other close relative such as Bacillus cereus and Bacillus thuringiensis ? Other Gram positives? We'll need to do a literature search to find all of the genes that contain the sequences of quorum peptides in these organisms to identify as their profiles.
Some simple aligments of protein sequences will then reveal just how variable the sequences are across species.
Ideas and comments on the Theme
Difficulties finding peptides that are secreted by the individual organisms that are unique to these organisms.
Concentrating on finding targets for:
Streptococcus pneumoniae
Bacillus anthracis
Clostridium dificile
These are all relevant target species as they are pathogenic to humans world wide and require detection systems.
Streptococcus pneumoniae uses a CSP quorum-sensing system. The gene encoding the CSP for S. pneumoniae is comC. However this may not be unique to S. pneumoniae. BLAST result for comC gives top two similarities to S.pneumoniae but then score of 69.7 for Streptococcus oralis. Need to look for other peptides, possibly just any secreted peptide rather than those involved in quorum sensing.
Looked at IgA1 protein of S.pneumoniae however this is homologous to that of Streptococcus sanguis. many other surface proteins are present however think that it needs to be one's that are actually secreted rather than just on the surface…………. S. pneumoniae does have hemolysins that are secreted however these bind cholesterol rather than actual receptors.
Signalling peptide pep27 is a possibility, it is unique to S.pneumoniae however it is not well characterised and little information is available via EMBL and UniProt, but I shall continue to look at information on this as it seems most viable peptide found so far.
Overall there are thirteen two-component systems identified in S.pneumoniae. Hopefullly at least one of these will be unique to the pathogen. comC etc. and also the YycFG are not unique so cannot be used. ciaR component system also used in other streptococcus species.
Gene pairs, 484hk, 481hk, 486hk, 478hk, 539hk, 480hk and ciaRH all have similarity to other species including Bacillus. It is possible that they will have unique receptors that are not found in other species.
Overall it appears that the best two-component system to use for detection of S.pneumoniae is using the vncS/R sensors that detect pep27.
Very difficult to find the repressor thatis supposed to exist for the vnc two-component system.
See Gilmore and Hoch 2007.
This sugggests to me that the response regulator is phosphorylated, but not by a repressor. So perhaps there is not a repressor for this system that represses vncR (unless it is the repressor that acts to phosphorylate the vncR).
It has been mentioned in 2000 that there is a RelA gene that mediates 'the' stringent response in the pathway. This remains to be uncharacterised, the only information I can find on this is an unreviewed Uniprot entry, it encodes the protein GTP pyrophosphokinase.
A the moment I would conclude that there is not a repressor of the vncR, and that it is purely the phosphorylation that causes repression.
Another diagram, Novak 1999
There is no evidence for a repressor for the vnc system so we will not be able to use it. Will try looking at other systems in S. pneumoniae but not very optimistic.
Of all of the other 2 component systems there are not any that are viable to use in our sensor. Other TCS include micABC, phoR/P, vicRK however these are all either homologous to other species, or do not have unique peptides or are not well enough characterised.
Change of plan again…We will be using the com C sensory system.
This shows the alignment of com C in S.pneumoniae with that of the S. oralis altough they are very similar there are differences. The com C is cleaved after the two glyceine residues, there are more difference after this part of the sequence.
Retrieved from " http://2008.igem.org/Team:Newcastle_University/Biological_Brainstorming "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
