<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Workbench"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Workbench">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Workbench&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Workbench&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Workbench&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Workbench" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Workbench</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A>&gt;&gt;<STRONG class="selflink">Workbench</STRONG></P><DIV id="maincol"><H2><SPAN class="mw-headline" id="Workbench">Workbench</SPAN></H2><H3><SPAN class="mw-headline" id="Aims_and_Objectives_:">Aims and Objectives :</SPAN></H3><P>The proposed Synthetic Biology Workbench (cool name needed) will incorporate the functionality of a biological network modeling and simulation program along with spiffy new algorithms that can be used in modern synthetic biology. This will enable users to finetune the expression of models suitable for their purposes, particularly for such fun activities like iGEM competitions.
</P><P>From a previously created repository of parts and devices, and using the (gentle) guidance of a set of assembly parameters, users will be able to create, edit and piece together biological networks at varying levels of complexity. Users will be able to piece together parts, adding ribosome binding sites, protein coding regions, terminators, etc. until they have exactly the device they desire.
</P><P>Once the biological circuit is ready in principle, it can be put into practice. The user will be able to request the optimal biological circuit, composed from BioBricks from the repository.
</P></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">DiS with a very subtle directed line, in a prototype version.</DIV></DIV></DIV><P>Our objectives are to:
</P><UL><LI> get a model
</LI><LI> load from SBML and/or CellML
</LI><LI> create using <A href="/Team:Newcastle_University/Parts_Repository" title="Team:Newcastle University/Parts Repository">parts</A> and <A href="/Team:Newcastle_University/Constraints_Repository" title="Team:Newcastle University/Constraints Repository">constraints</A></LI><LI> show the model
</LI><LI> network view
</LI><LI> basic parts view. Matt suggested being able to view both strands and flip them around.
</LI><LI> take in conditions for synthetic evolution and send models to the <A href="/Team:Newcastle_University/Evolutionary_Algorithm" title="Team:Newcastle University/Evolutionary Algorithm">evolutionary algorithm</A></LI><LI> output the sequence in a standard format
</LI></UL><H3><SPAN class="mw-headline" id="User_Stories">User Stories</SPAN></H3><H4><SPAN class="mw-headline" id="Parts_Repository">Parts Repository</SPAN></H4><OL><LI> The program must display a list of all available parts. The program requests a hierarchical list of all part names, part types, and their part ids from the Parts Repository (PR). Upon receiving the list, the program sorts the parts into different categories based on the part type, chooses a display icon for each type, and presents the list of part names with their part type icons to the user.
</LI><LI> The user wishes to change the type of chassis. The program retrieves a list of all chassis names and chassis ids from the PR and presents it to the user.
</LI><LI> The program must display a list of all types of parts. It requests a list of all part types and their part type ids from the PR. Then it displays the list of part names to the user.
</LI><LI> The program must display all of the information about a single part. It gives the part id to the PR and receives a full description of the part in return. It then displays the part information to the user.
</LI><LI> The program must save a CellML model of a part. It gives the part id to the PR and receives a segment of CellML in return. It then saves the CellML to file.
</LI></OL><H4><SPAN class="mw-headline" id="Constraints_Repository">Constraints Repository</SPAN></H4><OL><LI> The program must show all possible interactions to a particular part. It passes the part id to the Constraints Repository and receives a list of all possible interaction ids in return. It then retrieves the details of each interaction with the interaction id. It then shows the possible interactions to the user.
</LI></OL><H4><SPAN class="mw-headline" id="Evolutionary_Algorithm">Evolutionary Algorithm</SPAN></H4><OL><LI> The user finally wants to evolve her model in silico. The program collects the starting model, composed of a series of part ids, and the proposed behaviour in the form of a truth table, and gives it to the Evolutionary Algorithm (EA) for evolution. The program shows a “busy working” screen while the Evolutionary Algorithm computes the best model. When the EA finishes, the program receives the proposed best model(s) and displays them, along with their behaviour, to the user.
</LI></OL><H2><SPAN class="mw-headline" id="Outcome">Outcome</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">The architecture of DiS is based on certain software engineering principles. Certain design patterns, such as Singleton, have been used extensively</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure A: The opening screen of Darwin in Silico - the Synthetic Biology Workbench. Here, we see the first part of the workflow, the <I>Build</I>, where the initial structure of the proposed circuit is laid out.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure B: Parts can be dragged from the left pane onto the building area, and connected with different kinds of arrows to indicate different relationships.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure C: Before moving on to the <I>Behaviour</I> tab, the inputs to the system and the outputs from the circuit are specified by right clicking on the elements.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure D: In the <I>Behaviour</I> tab, the behaviour of the system is specified by a series of tables. In this image, when the input molecule &quot;CSP&quot; is present in high quantities, both GFP and OFP will also be expressed at high levels.</DIV></DIV></DIV><P>Darwin <I>in Silico</I> (DiS) is the Workbench: the graphical user interface (GUI) for the design of the genetic construct that was <A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">engineered</A> into <I>B. subtilis</I>.  The <A href="/Team:Newcastle_University/Evolutionary_Algorithm" title="Team:Newcastle University/Evolutionary Algorithm">evolutionary algorithm</A> (EA), <A href="/Team:Newcastle_University/Parts_Repository" title="Team:Newcastle University/Parts Repository">parts repository</A> (PR) and the <A href="/Team:Newcastle_University/Constraints_Repository" title="Team:Newcastle University/Constraints Repository">constraints repository</A> (CR) were intended to work behind the scenes of this interface in the design of the construct.  
</P><P>
The user can design a starting point for the genetic construct in the centre window of the GUI by dragging and dropping parts into place.  All of the two-component genes are available for this process, as are a large number of promoters, ribosome binding sites, transcription factors and sigma factors.  These parts are accessed from the parts repository database and built into a preliminary construct.  
</P><P>The constraints repository aids in the building of this construct by not allowing parts that are not compatible to be dropped in place next to each other.  Where parts are compatible with each other, constraint models are incorporated into the construct that define the interactions between these.  
</P><P>The user can also map system inputs to system outputs that can be used by the EA.  For example, the user could specify that in the presence of a certain set of quorum sensing peptides that a certain fluorescent protein would light up. The user can then run the EA through DiS, leading to the mutation of the hidden layer and the output of the hidden layer model that fulfils the output requirements the user has set.
</P><P>The combination of these three software components were supposed to communicate through DiS. Each team member was responsible for the construction of their component. Unfortunately, since development of the components proceeded in tandem and the lead developer had to leave the country before the PR, CR and EA were complete, the connections between the components were never incorporated into DiS. The outcome of this was a working build, where the evolutionary algorithm could communicate directly with the parts repository, but not with DiS. Once some more work has been carried out on the various components of the overall project, DiS will be functional.
</P><H4><SPAN class="mw-headline" id="DiS:_The_Ideal_and_the_Reality"> DiS: The Ideal and the Reality </SPAN></H4><P>It is possible to paint a very rosy-tinted picture of how the user might interact with DiS. <B>Bolded items</B> have been implemented in the current incarnation of DiS.
</P><OL><LI><B>The user fires up the workbench by double clicking an icon on the desktop etc.</B></LI><LI> A nice splash screen appears welcoming them to 'Darwin in Silico: Synthetic Biology Workbench'
</LI><LI><B>The main application window appears.</B> If this is the first time the program has run then a wizard may appear asking them to setup configuration details for the rest of the components of the system. If not, then it know where all of these are located by reference to its own internal persistence store of some kind.
</LI><LI><B>The application consists of a working area where a genetic circuit can be specified and a pallete of 'parts' read from the parts repository. These parts have pretty 'BioBrick' type icons</B> and can be expanded to give the actual instances of the various part types available in the database.
</LI><LI> The user selects 'New Project' and has to define some initial starting information. They have to enter their name etc and some pre-saved details about them pop up. They modify some information about the <I>in silico</I> experiment purpose etc.
</LI><LI><B>They are then taken to the blank working area where they can start to define details of the circuit to be evolved. See Figure A.</B></LI><LI> Somewhere is a menu relating to chassis details. They select <I>Bacillus subtilis 168</I> from a drop down list or the like.
</LI><LI> Immediately the parts list is constrained by the parts that are available for operation in <I>B. subtilis</I> 168.
</LI><LI><B>The user begins to add the initial parts to the circuit to be evolved. They know that they want the circuit to essentially respond to the concentration of three (of course may be more or less than this) peptides that are signaling molecules from outside. She select free peptides from the palette at the side and drags them on to the canvas on eat a time where they appear; AIP (a <I>Staph aureus</I> virulence regulating peptide), CSP (a competence stimulating peptide from <I>Streptococcus pneumoniae</I>) and papR (a virulence controlling peptide from <I>Bacillus anthracis</I>, the causative agent of anthrax). See Figure B.</B></LI><LI><B>She defines these quantitative levels of these peptides as inputs to the system somehow. </B></LI><LI><B>She then selects coding regions from the palette, and navigates to Green Fluorescent protein (gfp) coding region and drags the rectangular icon on to the work area. She repeats this exercise for orange and red fluorescent (ofp and rfp) protein. They automatically appear with a RBS that is optimal for the chosen chassis. She leaves them promoterless for now.</B></LI><LI><B>She highlights the fluorescent protein coding regions and marks them as system outputs in terms of the absolute levels of the proteins they encode. See Figure C.</B></LI><LI><B>Next she specifies the functions that specify the behaviour of the outputs for a specified input concentration, again specified by a truth table. In this case she specifies a high initial concentration for AIP and a corresponding high output for gfp and ofp. She assigns zero production to CSP, PapR and rfp.</B></LI><LI><B>She repeats this exercise for a new set of states. This time she chooses a high initial concentration for PapR and a corresponding high production to rfp and ofp. She assigns a zero production for CSP, AIP and gfp.</B></LI><LI><B>The process of 13 and 14 is repeated until the desired system behaviour has been fully specified. See Figure D.</B></LI><LI><B>Next she chooses the chassis location for the fluorescent protein coding regions. She right clicks on them and chooses high copy number plasmid pBD9 (she could have chosen chromosome instead or a range of other vectors)</B></LI><LI><B>She is almost ready to invoke the evolutionary algorithm. However, she has the option to narrow down the range of parts available to the evolutionary algorithm. In this case she restricts it to all transcription factors and their modifiers and all promoters that operate in the chosen chassis. See Figure E.</B></LI><LI> She then clicks the 'evolve this' button. Behind the scenes, the model representing this initial set of parts, together with metadata and data about the input/output behaviour is sent to the EA component.
</LI><LI> A dialog box showing the progress of the EA component pops up. 
</LI><LI> Once the EA has completed a window pops up showing the ID list of models that meet the fitness function specifications. The user can browse the selection graphically by selection of the relevant model. The graphical selection can be viewed as a conventional pictorial diagram of genetic regulation beloved of biologists or as a graph based view of the model. See Figure F.
</LI><LI> The user is able to view the levels of the different species in the model over time or with varying concentrations of the input signals as line type graphs.
</LI><LI> Ideally, she is able to pick one model tweak it and ask for it to be re-evolved.
</LI><LI> Finally once happy with the model, she clicks on 'model to sequence converter' and the chosen model is sent to the model to sequence converter. A GFF or EMBL formatted record is returned, for each of the genetic elements that require to be synthesised.
</LI></OL><TABLE border="1" width="203"><TBODY><TR><TD valign="top"><P>Figure E:In the <I>Evolve</I> tab, the user selects where the circuit should be placed - in a specified vector, or in the chromosome, for example. Then she chooses the parts that should be evolved, and clicks the 'Evolve!' button.
</P></TD><TD valign="top"><P>Figure F: Someday, when DiS is hooked up to the EA, the <I>Results</I> tab will be populated with a series of potential models that the user can save, edit and re-evolve, or export for synthesis and <I>in vitro</I> biological experiments.
</P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Further_Reading">Further Reading</SPAN></H3><H3><SPAN class="mw-headline" id="Contributors">Contributors</SPAN></H3><P>Lead: <A href="/Team:Newcastle_University/Morgan_Taschuk" title="Team:Newcastle University/Morgan Taschuk">Morgan Taschuk</A></P><DIV id="sidebar"><DIV class="post-it bluebox"><H3><SPAN class="mw-headline" id="Darwin_in_Silico:_Workbench"> Darwin <I>in Silico</I>: Workbench </SPAN></H3><UL><LI><A href="/Team:Newcastle_University/Workbench#aims" title="Team:Newcastle University/Workbench">Aims and Objectives</A></LI><LI><A href="/Team:Newcastle_University/Workbench#user-stories" title="Team:Newcastle University/Workbench">User Stories</A></LI><LI><A href="/Team:Newcastle_University/Workbench#outcomes" title="Team:Newcastle University/Workbench">Outcomes</A></LI><LI><A href="/Team:Newcastle_University/Workbench#progress" title="Team:Newcastle University/Workbench">DiS: the Ideal and the Reality</A></LI><LI><A href="/Team:Newcastle_University/Workbench#further-reading" title="Team:Newcastle University/Workbench">Further Reading</A></LI><LI><A href="/Team:Newcastle_University/Workbench#contributors" title="Team:Newcastle University/Workbench">Contributors</A></LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Workbench">http://2008.igem.org/Team:Newcastle_University/Workbench</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Workbench" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Workbench" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Workbench&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Workbench&amp;oldid=90467" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>