<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Constraints_Repository"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Constraints_Repository">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Constraints_Repository&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Constraints_Repository&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Constraints_Repository&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Constraints_Repository" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Constraints Repository</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A>&gt;&gt;<STRONG class="selflink">Constraints Repository</STRONG></P><DIV id="maincol"><H2><SPAN class="mw-headline" id="Constraints_Repository">Constraints Repository</SPAN></H2><H3><SPAN class="mw-headline" id="Aim:">Aim:</SPAN></H3><P>Develop a system that will be a repository of constraints that specify the way biological components can be assembled, and the parameters that describe their interactions.
</P><P>Components of a biological circuit can only be combined in ways which make biological sense. Much of this information is available in publications and public databases, but these are distributed worldwide and do not necessarily have compatible formats. The information therefore needs to be stored and described somewhere which is accessible to the rest of the projects, in a single, coherent form. 
</P><P>The objectives of the Constraints Repository (CR) were to:
</P><UL><LI>  To mine literature and public databases to determine all of the relevant constraints and associated parameters for the interactions between a defined set of biological components.
</LI><LI> To develop a database to capture the information gathered on constraints between two or more biological components. The database includes models based on the kinetic parameters between the biological components. The database must define the type of interactions involved. 
</LI><LI> To demonstrate that the database is capable of validating or rejecting an interaction between two biological components in a given context. This function contributes to the construction of genetic circuits.
</LI><LI> To develop an agreed interface to other system components. This project was part of the international genetically engineered machines competition (iGEM). There were three other computational projects responsible for different aspects of the software development. To facilitate communication between the projects, a standardized interface was produced.
</LI><LI> To explore the use of integrating data sources for populating data. The construction of this repository required the application of varied data sources. To effectively analyze the constraints, these sources were adapted into a consistent format to permit the integration and analysis of data in the repository.
</LI></UL></DIV><H3><SPAN class="mw-headline" id="Outcomes">Outcomes</SPAN></H3><P>A prototype datasheet has been designed for one of the well characterized BioBrick devices from the MIT registry of Standard Biological Parts. This feat has been the closest attempt to creating the said required tool. 
</P><P>Since the tool constructed in this project aims to describe the interactions between the components listed in the “BugBuster” and not MIT parts registry, only the interactions between these parts are considered. Although the parts themselves are not existing BioBrick parts, the Newcastle Team intend to make the necessary modifications to add the “BugBuster” parts to the MIT parts registry as novel BioBricks parts and devices. 
</P><P>Some examples of the questions asked to ascertain the compatibility of the biological compartments include;  
</P><UL><LI> How should two components interact to produce the desired effect within the system?
</LI><LI> What are the possible biological components, within the parts database, that will generate the correct interaction type?
</LI><LI> Are the kinetic parameters, associated with the interaction, likely to generate efficient transcription, gene expression or stable binding of the biological component?
</LI></UL><P>Each valid interaction is stored as a mathematical model using the <A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">CellML</A> format. This model is included in the database and is used by the circuit designer system, within the BugBuster Project, to produce a genetic circuit. 
</P><P>To populate the database, a thorough search of literature, public databases and existing tools was carried out. One biological component may interact with many different types of components and therefore possess many different interaction types. So initially, the type of interaction suitable within the context of the system must be identified. A list of potential parts which would induce a particular interaction type must be generated. This information is then coupled with a model to demonstrate the kinetic parameters.   
</P><P>In cases when literature did not contain information about an interaction, public databases and software tools were utilized. The Database of Interacting Proteins (DIP) stores experimentally determined interaction information about protein coding components. One can search for proteins in numerous ways including sequence similarity and homologous motifs. However DIP does not supply information regarding non-protein coding biological components. 
</P><P>Programmes, such as GeneDesigner and BioSpice are tools designed specifically for synthetic biology. They provide a means of computationally constructing genetic circuits. A repository of general constraints and models are included in the tools. These provided a useful source of information.  
</P><H4><SPAN class="mw-headline" id="The_matrix">The matrix</SPAN></H4><P>To populate the compatibility matrix I used three methods.
</P><OL><LI> literature searching (google the parts and look through journals for conformation that they can interact/control gene expression of the connecting part)
</LI><LI> Look at homologous structures of parts that have been confirmed to interact. If the structure is the same, they may also be compatible.
</LI><LI> use blast of the DNA sequence two parts together to see the sequence exists at all. This would confirm compatibility.
</LI></OL><TABLE border="1"><TBODY><TR><TD>LacI
</TD><TD>LacO
</TD><TD>XylR
</TD><TD>TetR
</TD><TD>ssrA
</TD><TD>ComK
</TD><TD>PcomG
</TD><TD>Pspank
</TD><TD>Phi29
</TD><TD>mCherry
</TD><TD>GFP
</TD></TR><TR><TD>LacI
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>LacO
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR><TR><TD>XylR
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>TetR
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD></TR><TR><TD>ssrA
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD></TR><TR><TD>ComK
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD></TR><TR><TD>PcomG
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>Pspank
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD></TR><TR><TD>phi29
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR><TR><TD>mcherry
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR><TR><TD>GFP
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>1
</TD><TD>1
</TD><TD>1
</TD><TD>0
</TD><TD>1
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Further_Reading">Further Reading</SPAN></H3><UL><LI><A href="http://dspace.mit.edu/handle/1721.1/30475" class="external text" rel="nofollow">BioJADE</A> - has some thoughts about databases
</LI><LI> Cai Y, Hartnett B, Gustafsson C, Peccoud J (2007) A syntactic model to design and verify synthetic genetic constructs derived from standard biological parts. Bioinformatics 23(20): 2760-2767. [PubMed ID: 17804435]
</LI></UL><H3><SPAN class="mw-headline" id="Contributors:">Contributors:</SPAN></H3><P>Lead: <A href="/Team:Newcastle_University/Nina_Nielsen-Dzumhur" title="Team:Newcastle University/Nina Nielsen-Dzumhur">Nina Nielsen-Dzumhur</A></P><DIV id="sidebar"><DIV class="post-it bluebox"><H3><SPAN class="mw-headline" id="Constraints_Repository_2"> Constraints Repository </SPAN></H3><UL><LI><A href="/Team:Newcastle_University/Constraints_Repository#aims" title="Team:Newcastle University/Constraints Repository">Aims and Objectives</A></LI><LI><A href="/Team:Newcastle_University/Constraints_Repository#outcomes" title="Team:Newcastle University/Constraints Repository">Outcomes</A></LI><LI><A href="/Team:Newcastle_University/Constraints_Repository#matrix" title="Team:Newcastle University/Constraints Repository">The Matrix</A></LI></UL><LI><A href="/Team:Newcastle_University/Constraints_Repository#further-reading" title="Team:Newcastle University/Constraints Repository">Further Reading</A></LI><LI><A href="/Team:Newcastle_University/Constraints_Repository#contributors" title="Team:Newcastle University/Constraints Repository">Contributors</A></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Constraints_Repository">http://2008.igem.org/Team:Newcastle_University/Constraints_Repository</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Constraints_Repository" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Constraints_Repository" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Constraints_Repository&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Constraints_Repository&amp;oldid=90470" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>