<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Nina_s_Lab_Journal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Nina%27s_Lab_Journal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Nina%27s_Lab_Journal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Nina%27s_Lab_Journal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Nina%27s_Lab_Journal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Nina%27s_Lab_Journal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Nina's Lab Journal</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Dry_Lab" title="Team:Newcastle University/Dry Lab" class="mw-redirect">Dry Lab</A>&gt;&gt;<STRONG class="selflink">Nina's Lab Journal</STRONG></P><H2><SPAN class="mw-headline" id="Nina.27s_Lab_Journal">Nina's Lab Journal</SPAN></H2><H3><SPAN class="mw-headline" id="17.2F05"><B>17/05</B></SPAN></H3><P>Yesterday I researched unique peptides in the Gram-positive anaerobic bacterium <I>Clostridium difficile</I>.
</P><P><B>Background</B></P><P>This pathogen is associated with antibiotic-related diarrhoea and life threatening pseudomembranous colitis so can be regarded as an important species to research. The current gold standard for diagnosis is toxigenic culture in which organisms are cultured on selected medium and tested for specific toxin production (A and B). This procedure is sensitive and specific but is slow and labour intensive. 
</P><P><B>Potential Peptides</B></P><P>I started by looking at quorum sensing peptides in C. difficile. Studies (Carter <I>et al</I>., 2005) have shown that at least one signalling molecule exists which can induce bioluminescence in a <I>Vibrio harveyi luxS</I> reporter strain. Carter then concluded that a <I>luxS</I> homologue (<I>luxScd</I>) quorum sensing system existed in <I>C. difficile</I>. The signalling molecule corresponded to autoinducer-2 (AI-2). However this diffusable molecule, regulated by the <I>rolA/B</I> two component system in <I>C. difficile</I>, is ubiquitous amongst many species of bacteria (Kleerebezem <I>et al.,</I> 1997), (Barth <I>et al.,</I> 2004), (Carter <I>et al.,</I> 2005), (Sebaihia <I>et al.,</I> 2006). It would therefore be foolish to consider a receptor of AI-2 as a potential diagnostic tool of <I>C. difficile</I> related diseases alone. In addition there is no direct evidence that <I>C. difficile</I> actually employs a quorum sensing system!! 
But I didn't want to give up here...
I decided to then explore the (unique to <I>C. difficile</I>) exotoxins A and B released. These toxins bind to receptors of target cells and are then endocytosed (Aktories <I>et al</I>., 2007). I had some difficulty identifying these receptors initially.
</P><P>I then found out why. The nature of toxin A (TcdA) and B (TcdB) receptors are not well defined, although carbohydrate structures may be involved (Aktories <I>et al.,</I> 2007). I tried to dig deeper and read through more literature for clues about these receptors. I wanted to know if they were specific to the exotoxins of <I>C. difficile</I> and exactly which cells of the host they were present on.
</P><P>I then found out that the receptor for TcdA was indeed only on the surface of intestinal epithelial cells and was indeed made of carbohydrates (Ho <I>et al.,</I> 2005). These cell surface carbohydrates included Gal-a1,3-Gal-b1,4-GlcNAc. However it also seems that they are not specific to the <I>C. difficile</I> toxins. I came to this conclusion since Ho <I>et al</I> mentioned that the binding site on TcdA for the receptor, consisting of a run of up to 50 amino acids, was similar to other extracellular bacterial carbohydrate-binding proteins.
</P><P><B>Conclusion</B></P><P>Further inspection of the relationship between the unique exotoxins of <I>C. difficile</I> and their relative receptors revealed a low binding affinity and attempts to crysallize the complex have thus so far failed (Ho <I>et al.,</I> 2005). In addition, the fact that the receptor is firstly carbohydrate and secondly, mammalian, contributes to my view that exploiting the exotoxins would not appropriate for our application. 
</P><P>Since a quorum sensing system has not been formally established in C. difficile I would also conclude that this species is not viable for our use. I would recommend concentrating on <I>Staphylococcus aureus, Bacillus anthracis</I> and <I>Streptococcus pneumoniae</I>. Having read quite a bit, I can see these have much better characterised QS systems.
</P><H3><SPAN class="mw-headline" id="19.2F05"><B>19/05</B></SPAN></H3><P>Today I will be working backwards to identify gram-positive bacteria that have well characterized quorum sensing systems (QSS). This should be a more efficient way of finding a suitable peptide. 
</P><P><B>List of Bacteria with well characterized QSS</B></P><P><I>Streptococcus pneumoniae</I> - currently being looked at by Megan
</P><P><I>Streptoccoccus mutans</I> and <I>gordonii</I> - cause dental plaque and sub-acute endocarditis. Possibly not severe enough for us to use?
</P><P><DEL><I>Bacillus subtillis</I></DEL> - not pathogenic 
</P><P><DEL><I>Lactococcus lactis</I></DEL> - not pathogenic. Used in dairy products.
</P><P><I>Staphylococcus aureus</I> - currently being looked at by Mark 
</P><P><DEL><I>Pseudomonas aeruginosa</I></DEL> -  Gram-negative
</P><P><DEL><I>Carnobacterium piscicola</I></DEL> - not pathogenic but releases a bacteriocin inhibiting QSS of <I>Listeria monocytogene</I> (approved and already looked at by Mark).
</P><P><DEL><I>Yersinia enterocoliticia</I></DEL> - Gram negative!! 
</P><P><DEL><I>Lactobacillus sake</I></DEL> - not pathogenic (used in fermented sausages) 
</P><P>Note to self... a lot of gram-negative well characterized QSS..annoying. What if I focus search to only G+ve disease causing bacteria? 
Another note to self... I cannot consider any of the bacteria which solely adopt the LuxS QSS (such as Clostridium difficile) since autoinducer emitted is the same for many bacteria and would not serve as a suitable protein to target. (Lerat <I>et al</I>., 2004).
</P><P><I>Streptococcus pyogenes</I>- myositis, toxic shock syndrome, puerperal fever, pharyngitis, cellulitis, rhematic fever etc..   
</P><P><I>Streptococcus agalactiae</I> - important human pathogen..meningistis, pneumonia, sepsis and major cause of infections in new borns. 
</P><P><I>Bacillus cereus</I> - Food pathogen particularly in rice. Has a Plc-PapR quorum sensing system.  
</P><P><B>Conclusion</B></P><P><I>Bacillus cereus</I>, <I>Streptococcus agalactiae</I> and <I>Streptococcus pyogenes</I> all serve as extra pathogenic bacteria which we can detect in addition to <I>Staph. aureus, Streptococcus pneumoniae</I> and <I>Listeria monocytogene</I>. They all have very well characterized quorum sensing systems and peptides which we can detect.
</P><H3><SPAN class="mw-headline" id="21.2F05"><B>21/05</B></SPAN></H3><P><B>The matrix.</B></P><P>To populate the compatibility matrix I used three methods. 
</P><P>Literature searching (google the parts and look through journals for conformation that they can interact/control gene expression of the connecting part)
</P><P>Look at homologous structures of parts that have been confirmed to interact. If the structure is the same, they may also be compatible.
</P><P>Use blast of the DNA sequence two parts together to see the sequence exists at all. This would confirm compatibility.
</P><H3><SPAN class="mw-headline" id="22.2F05"><B>22/05</B></SPAN></H3><P><B>Find out for sure whether B. subtilis has the PapR/PlcR system like B. cereus</B></P><P>The expression of various chromosomal genes encoding extracellular factors (i.e., phospholipases C, proteases, cell wall proteins, enterotoxins and hemolysins) is activated by a pleiotropic regulator, PlcR, that is specific to the B. cereus group. The activity of PlcR depends on the presence of PapR, a small signaling peptide that acts as a quorum-sensing effector. PapR is exported by the bacterial cell, processed, presumably as a pentapeptide, and then reimported into the cell, where it interacts with PlcR to facilitate its binding to its DNA targets. <B>This activating mechanism is strain specific</B>, and this specificity is determined by the first residue of the pentapeptide.
</P><P>(Slamti <I>et al.,</I> 2005)
</P><H3><SPAN class="mw-headline" id="25.2F05"><B>25/05</B></SPAN></H3><P><B>Compatibility Matrix</B></P><P>I completed a types of parts compatibility matrix to compare to the parts compatibility matrix.
This matrix allows one to see which repressors or promoters are compatible with which regulators etc. To work out compatibility, the following factors of each of the parts will be considered; 
</P><P>a) Synthetic biology, genetic engineering or naturally occurring literary evidence of the exact associate parts interacting, or having a desired effect/control of gene expression of one or either of each other.
</P><P>b) Already established compatibilities on the BioBricks parts registry <A href="http://partsregistry.org/Main_Page|parts_registry" class="external free" rel="nofollow">http://partsregistry.org/Main_Page|parts_registry</A></P><P>c) If no specific literature on a particular part and its compatibility with another part exists and it does not appear to have been constructed as a biobrick, I will look into structures similar to the part in question to deduce that the interactions may be similar to the relationship of parts that have already been established. 
This deduction can only be carried out if the two parts function an effect on each other by actually interacting.
</P><P>Once compatibility for each part and each target part (all other parts) has been established in each given  I should store the compatible groups of parts together in a database.
This database should also include models to the string of parts and should end up being &quot;BioBricks&quot; specifically designed to be inserted into our <I>B. subtilis</I> chassis.
The models should be designed in CellML. They should join together the individual part models designed by Megan.
The database should have unique Ids for the individual parts, the joined parts, the types of parts and the associated models (see ERD). This will allow Mark and the EA to sift through the models to identify the &quot;fittest&quot; order
</P><P><B>Pseudocode</B></P><P>I have also attempted to write a pseudocode for my constraints database.
I have started this off by drawing an ERD. The entities shall form the java classes that I shall use. 
I can use the structure of the ERD to establish how to structure the query and java that will obtain the list of parts from the parts repository and the database (Henkel <I>et al.,</I> 2007)
</P><H3><SPAN class="mw-headline" id="29.2F05"><B>29/05</B></SPAN></H3><P>Finished java practise set by morgan for a better understanding of the code I will write up.  
Designed 10 minute power point presentation of work done so far on individual project.
Read key biobrick assembly papers and did summaries. 
Used Bioinformatics to analyse the Plc/PapR two component system 
</P><H3><SPAN class="mw-headline" id="30.2F05"><B>30/05</B></SPAN></H3><P><B>Java</B></P><P>Did some more java practise with Morgan. Hello world again, for loops, if, else, etc.
Finished the other Java exercises. Made start on Individual Java exercise - made a constructor and defined the &quot;recipe&quot; for the Interactions class. 
</P><P>Put up final code on svn.
</P><H3><SPAN class="mw-headline" id="06.2F06"><B>06/06</B></SPAN></H3><P><B>CellML Practise</B></P><P>I attended the CellML tutorial meeting presented by Neil.
</P><P>Then after lunch did all three cellML tutorials. 
</P><P>Finally emailed Jan-Willem about sensor-FP BioBricks. The team and I are planning to make our first BioBrick using the two component system of B. cereus and fluorescent proteins.
</P><H3><SPAN class="mw-headline" id="07.2F06"><B>07/06</B></SPAN></H3><P><B>Bacillus anthracis</B></P><P>Today I will attempt to uncover a unique peptide associated with  a two component system in <I>Bacillus anthracis</I>.
</P><P>It has been suggested that <I>Bacillus anthracis</I> is a subspecies of <I>Bacillus cereus</I>. However one obvious difference between the two genomes is <I>anthracis</I> has truncated or degraded HKs and RRs and TCS related genes. This maybe due to the specific warmblooded environment that this pathogen requires. In contrast, <I>Bacillus cereus</I> and many other bacteria exploit the two component system to sense and respond to hostile and changing environments. Therefore in comparison other similar bacteria have a larger set of these particular proteins.
</P><P>However, I did come across evidence of the LuxS quorum sensing system. This produces the autoinducer A1-2.
</P><P><B>The problem with the luxS/AI-2 system</B> (similar problem to that of <I>Clostridium difficile</I>). 
</P><P>The LuxS/AI-2 (autoinducer-2) is a signalling molecule that functions in **interspecies** communication by regulating niche-specific genes with diverse functions in various bacteria, often in response to population density. LuxS (S-ribosylhomocysteinase; ) is an autoinducer-production protein that has a metabolic function as a component of the activated methyl cycle. LuxS converts S-ribosylhomocysteine to homocysteine and 4,5-dihydroxy-2,3-pentanedione (DPD); DPD can then spontaneously cyclise to active AI-2.
</P><P>Therefore as with <I>Clostridium difficile</I>, the luxS/AI-2 peptide will not be able to specifically diagnose anthrax. 
However to confirm this statement I performed a few bioinformatic analyses on the peptide.
</P><P>Used kegg to retrieve the luxS gene loci from the completed <I>Bacillus anthracis</I> genome (ames strain). 
</P><P>Added 1000 +/- to the nucleotide count to include all potential associated genes.
</P><P>Blasted the sequence in nBLAST.
</P><P>Retrieved several other <I>Bacilli</I> species as well as <I>anthracis</I> with 100% similarity.
</P><P>Conclusion: I would suspect using <I>Bacillus anthracis</I> would be futile. I cannot locate a unique quorum sensing peptide. In addition, this is a fairly badly annotated genome with degraded two component system related genes.
</P><P>Brilliant link to find out which bacteria use which two component systems;
</P><P><A href="http://www.genome.jp/kegg/pathway/ko/ko02020.html" class="external free" rel="nofollow">http://www.genome.jp/kegg/pathway/ko/ko02020.html</A></P><H3><SPAN class="mw-headline" id="08.2F06"><B>08/06</B></SPAN></H3><P><B>Parts for <I>B. cereus</I></B></P><P>Today I will draw up a detailed list of all the parts in <I>B. cereus</I> for strain ATCC 14579;
PlcR regulated protein PRP2; Transcriptional regulator PlcR; Transcriptional activator; PapR: PlcR accociated protein; transcriptional regulator PlcR, putative
</P><P><B>Looking at Parts on the operon</B></P><P>Must make sure the genesare  on the same strand as the two other proteins (in this case, complement strand).
Must make sure the genes are in the same direction.
Must make sure the genes are within 1000+\- bp of genes.
</P><P><B>List of parts on the operon:</B></P><P>ABC Transporter ATP binding protein, ABC permease protein, two component sensor kinase YocF, two component response regulator YocG, transcriptional activator PlcR, PapR protein, iron-sulphur-binding reductase, acetyl-CoA acetyltransferase, 3-hydroxybutyryl-CoA dehydrogenase, acyl-CoA dehydrogenase, short-chain specific, transcriptional regulator, TetR family, DNA-directed RNA polymerase subunit delta.
</P><H3><SPAN class="mw-headline" id="10.2F06"><B>10/06</B></SPAN></H3><P>Strains that we will be using:
</P><P><I>Streptocoocus pneumoniae</I> - R6 
</P><P><I>Staphylococcus aureus</I>- 8325
</P><P><I>Listeria monocytogene</I> - 10403S 
</P><P><I>Bacillus aureus</I> - ATCC 14579 
</P><P>Had a little play with the hibernate tutorial...bit stuck. Group decides Megan and Nina are better off extracting information out of their databases into Java using JDBC instead.
</P><P><A href="http://www.hibernate.org/hib_docs/reference/en/html/index.html" class="external free" rel="nofollow">http://www.hibernate.org/hib_docs/reference/en/html/index.html</A></P><H3><SPAN class="mw-headline" id="11.2F06"><B>11/06</B></SPAN></H3><P><B>Control BioBrick</B></P><P>Had a meeting with Jan about constructing simple BioBricks to test our two component systems. He suggested that for the control it might be wiser to integrate the <I>Bacillus subtilis</I> wild type with FPs into the <I>Bacillus subtilis</I>-168. The wildtype two component system is regulated by subtilin which is a peptide and acts as an antibiotic. However, 168 does not respond in the same way. (Bongers, Veening et al., 2005).
</P><P>Then I tried to start populating a matrix with parts we have so far.
</P><H3><SPAN class="mw-headline" id="13.2F06"><B>13/06</B></SPAN></H3><P>Attended a safety and ethics discussion with Prof. Colin Harwood. He showed us how and which forms we will have to fill in.   
</P><P>Wrote architecture of interface for constraints repository.
</P><PRE>- Get interactionsID for partID (partID): Set &lt;interactionID&gt;

- Get partners (interactionID): Set &lt;partID&gt;</PRE><PRE>- Get all interction typeIDs(): Set &lt;interactionTypeID&gt;</PRE><PRE>- Get interactionTypeID for interactionID (interactionID) &lt;interactionTypeID&gt;

- Get interactionIDs for interactionTypeID (interactionTypeID): Set &lt;interactionID&gt;</PRE><PRE>- Get model for interactionID (interactionID): &lt;model&gt;</PRE><PRE>- Get interactionTypeName for interactionTypeID (interactionTypeID): Set &lt;interactionTypeName&gt;</PRE><H3><SPAN class="mw-headline" id="15.2F06"><B>15/06</B></SPAN></H3><P>Started writing the interface Java doc. between the constraints repository and the EA. 
It was also decided that the constraints repository (CR) will not be communicating with the parts repository in the interface but the workbench instead. Each time a method was written, it was committed on svn so that Mark, Megan and Morgan can access everything simultaneously.
</P><H3><SPAN class="mw-headline" id="24.2F06"><B>24/06</B></SPAN></H3><P>Wrote an outline for my thesis (introduction etc..). The group will now also start writing up as much of the thesis as they can.
</P><H3><SPAN class="mw-headline" id="27.2F06"><B>27/06</B></SPAN></H3><P>Populate the database in Access and SQL. Wrote out all the SQL queries that would give the database functionality. Also wrote the introduction, aims and objectives for my thesis.
</P><H3><SPAN class="mw-headline" id="07.2F07"><B>07/07</B></SPAN></H3><P>Finished the introduction chapter of my thesisÂ :)
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Nina%27s_Lab_Journal">http://2008.igem.org/Team:Newcastle_University/Nina%27s_Lab_Journal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Nina%27s_Lab_Journal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Nina%27s_Lab_Journal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Nina%27s_Lab_Journal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Nina%27s_Lab_Journal&amp;oldid=39281" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>