<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Bacillus_subtilis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Bacillus_subtilis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Bacillus_subtilis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Bacillus_subtilis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Bacillus_subtilis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Bacillus_subtilis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Bacillus subtilis</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A>&gt;&gt;<STRONG class="selflink">Bacillus subtilis</STRONG></P><DIV id="sidebar"><DIV class="post-it bluewhitebox"><H3><SPAN class="mw-headline" id="Navigation"> Navigation </SPAN></H3><P><A href="/Team:Newcastle_University/Parallel_Evolution" title="Team:Newcastle University/Parallel Evolution" class="mw-redirect">Parallel Evolution</A><A href="/Team:Newcastle_University/Neural_Networks" title="Team:Newcastle University/Neural Networks">Neural Networks</A><STRONG class="selflink">About B. subtilis</STRONG><A href="/Team:Newcastle_University/Biological_Brainstorming" title="Team:Newcastle University/Biological Brainstorming">Biological Brainstorming</A></P></DIV></DIV><DIV id="maincol"><H2><SPAN class="mw-headline" id="Advantages_of_using_B._subtilis">Advantages of using <I>B. subtilis</I></SPAN></H2><P>There are several reasons why we chose to use <I>B. subtilis</I>, as opposed to <I>E. coli</I> as our chassis. 
</P><H3><SPAN class="mw-headline" id="Pathogenesis">Pathogenesis</SPAN></H3><P><I>B. subtilis</I> is categorised under Biohazard Level 1- i.e it is a minimum risk bacteria.  As a result of not being considered a human pathogen, the only necessary precautions to be implicated include using gloves, hand-washing, and autoclaving equipment. This saves time and money. 
</P><H3><SPAN class="mw-headline" id="Gram-Positive">Gram-Positive</SPAN></H3><P>The Gram stain technique is based on the ability of a bacterial cell wall to retain a crystal violet dye- allowing us to differentiate between two major cell wall types:
</P><UL><LI><B>Gram negative</B> cell walls have a lower percentage of peptidoglycan.  Gram-negative bacteria also have an additional outer membrane containing lipids, separated from the cell wall by the periplasmic space.
</LI><LI><B>Gram positive</B> cell walls contain a relatively large amount of peptidoglycan (up to 90%) but lower lipid content. This stains purple upon addition of crystal violet dye, whereas the wall of a gram negative bacteria will only stain pink. 
</LI></UL><P>After decolorisation, Gram-positive cells remain purple and Gram-negative cells lose their purple colour. When counterstain is applied, Gram-negative bacteria turn red.
</P><P>For a tutorial on Gram staining see:
<A href="http://www.ncl.ac.uk/dental/oralbiol/oralenv/tutorials/gramstain.htm" class="external free" rel="nofollow">http://www.ncl.ac.uk/dental/oralbiol/oralenv/tutorials/gramstain.htm</A></P><P><I>B. subtilis</I> is a gram-positive bacterium and therefore contains only a single membrane. Consequently, they absorb substances better than <I>E.coli</I>, and this is ideal for our project.  
</P><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P><I>B. subtilis</I> has the ability to take up exogenous DNA from the surrounding medium  and integrate it in their genome. This natural competency can be tightly regulated and induced in the lab by nutritional shortages using starvation medium.
</P><P>This makes transformation easier, as competence comes naturally to B.subtilis. 
</P><H3><SPAN class="mw-headline" id="Sporulation">Sporulation</SPAN></H3><P>During times of nutritional stress, <I>B.subtilis</I> cells produce an endospore which enables it to withstand extreme conditions such as high temperatures and salt concentrations. The purpose of this is to allow the bacterium to survive in the unfavourable environment until conditions return back to normal. Prior to the decision to produce the spore the bacterium may take up DNA from the surrounding medium.
</P><P>This means that <I>B.subtilis</I> can be easily stored and transported without damaging the cells and their ability to differentiate.
</P></DIV><H2><SPAN class="mw-headline" id="Using_a_B._subtilis_chassis_is_different_to_using_an_E._coli_chassis">Using a <I>B. subtilis</I> chassis is different to using an <I>E. coli</I> chassis </SPAN></H2><P>Although <I>B. subtilis</I> has many clear benefits in its usage, Gram-positive bacteria are generally relatively poorly characterised and consequently can be unpredictable to work with. Nevertheless, <I>B. subtilis</I> is one of the best known Gram-positives. This page intends to cover some issues about working with this bacterium.
</P><P>In this project, we used <I>B. subtilis</I> 168 (Bs168). This is a strain which is mutated in the <I>trp</I> operon and subsequently must have tryptophan present in its growth medium as it cannot synthesise this essential amino acid. The inability to synthesise tryptophan means that Bs168 is nutrient-inducible (a factor that can be exploited when introducing exogenous DNA).
</P><H3><SPAN class="mw-headline" id="Genetic_Transformation">Genetic Transformation</SPAN></H3><P>The transformation procedures in <I>E. coli</I> and <I>B. subtilis</I> are very different. The <I>E. coli</I> strains we were working with (TOP10 and DH5α) were already competent to take up exogenous DNA, or made competent chemically, and therefore require only basic preparation before transformation. This briefly consists of adding the DNA to the cells and incubating them on ice, then heat shocking and adding LB before incubating again at 37°C.
</P><P>In contrast, <I>B. subtilis</I> cells must be induced to take up foreign DNA.  Research has shown that when <I>B. subtilis</I> “encounters nutrient limitations and enters the stationary growth phase, it can engage in multiple adaptation strategies such as…uptake of exogenous DNA (competence)” <SUP><B>1</B></SUP>. It is therefore important to carefully control the growth phase and nutrient availability in the <I>B. subtilis</I> culture prior to transformation. We used the protocol below to achieve this.
</P><P>One of the main advantages of using <I>B. subtilis</I> is that it is naturally competent for transformation. When bacteria are in the stationary growth phase and are nutrient-starved, the cells begin to transcribe high levels of a tetramic protein called ComK, a transcription factor that activates over 100 genes (“including those essential for DNA binding and uptake” <SUP><B>2</B></SUP>. The competency of the cells to take up exogenous DNA is transient, and therefore observing correct timings in the transformation protocol is very important in increasing the population of competent cells.
</P><H4><SPAN class="mw-headline" id="Protocol_for_Genetic_Transformation_into_B._subtilis">Protocol for Genetic Transformation into <I>B. subtilis</I></SPAN></H4><UL><LI> Prepare two conical shake flasks of the following medium:
</LI><LI> 10mL SMM ('Spizizen salts')
</LI><LI> 125 µl sol E (40% glucose)
</LI><LI> 100 µl tryptophan
</LI><LI> 60 µl sol F (1M MgSO4)
</LI><LI> 10 µl casamino acid (CAA)
</LI><LI> 5µl ammonium iron
</LI></UL><LI> To one flask add 2mL liquid culture cells.
</LI><LI> Incubate both flasks at 37°C overnight. (<I>It is important that the flask without cells in is kept at 37°C also as this is the dilution medium and must be pre-warmed at the same temperature as the culture to avoid temperature stress of the cells.</I>)
</LI><LI> Add 500 µl overnight culture to the pre-warmed dilution medium and leave to incubate at 37 °C for 3 hours. (<I>This allows the cells to enter the stationary growth phase.</I>)
</LI><LI> Add 10mL SMM, 125 µl sol E and 60 µl sol F to the dilute culture and leave to incubate at 37°C for a further 2 hours. (<I>This starves the cells of amino acids such as tryptophan and CAA.</I>)
</LI><P>Transfer 10µl plasmid to be transformed into a 2mL tube. Add 400µl nutrient-starved cell culture and mix by inverting the tube five times.
</P><UL><LI> Incubate at 37°C whilst shaking for 1 hour. (<I>The tubes must be laid on their side to properly aerate the medium. To do this, tape the tubes to the rack and lay it on its side to prevent the tubes falling out whilst shaking.</I>)
</LI><LI> Plate cells on nutrient agar. 
</LI></UL><H3><SPAN class="mw-headline" id="Temperature_Sensitivity">Temperature Sensitivity</SPAN></H3><P>Unlike <I>E. coli</I>, <I>B. subtilis</I> is very temperature sensitive and sudden changes in temperature can be detrimental to it. When culturing, the cells should be kept at their 37°C optimum. We cultured all our colonies in a 37°C culture room. This way we could perform all pipetting and diluting of cells without changing their temperature.
</P><H3><SPAN class="mw-headline" id="Oxygen_Requirements">Oxygen Requirements</SPAN></H3><P><I>B. subtilis</I> is a strict aerobe and also requires much more oxygen than <I>E. coli</I>. If they become anoxic they will quickly die. Therefore it is important for culture media to always be kept shaking whilst the cells are growing to constantly provide them with oxygen. Allowing the broth to rest even for short periods of time can result in a lower transformation efficiency. All pipetting should be performed as quickly as possible to reduce the amount of time when the culture media are still.
</P><P>When making overnight cultures, conical shake flasks should preferably be used. The foam plug in the neck of the flasks allows exchange of air between the interior and exterior of the flask. Capped glass tubes can also be used as the caps are loose fitting and also allow air passage. Screw-cap plastic tubes should never be used as these are airtight. This is important for all overnight bacterial cultures but particularly for <I>B. subtilis</I> as it cannot survive anoxic conditions.
</P><H2><SPAN class="mw-headline" id="Problems_Encountered_when_Working_with_B._subtilis">Problems Encountered when Working with <I>B. subtilis</I></SPAN></H2><P>When we began work on <I>B.subtilis</I> we knew very little about the organism and therefore made some basic errors. For one transformation we used screw-cap plastic tubes, which significantly reduced both the number and size of the colonies on agar. Leaving them to grow for a further 6 hours allowed them to grow large enough for colony stabs to be taken.
</P><H2><SPAN class="mw-headline" id="Ribosome_Binding_Sites_.28RBS.29_for_B._subtilis">Ribosome Binding Sites (RBS) for <I>B. subtilis</I></SPAN></H2><P>'Perfect' <I>Bacillus</I> RBS' + atg<SUP><B>3, 4</B></SUP>:
</P><PRE>TAAGGAGGAACTACTATG</PRE><H2><SPAN class="mw-headline" id="Promoters">Promoters</SPAN></H2><P>All promoters below contain an RBS but the promoter sequence is not known for all of these.
</P><H3><SPAN class="mw-headline" id="Inducible_promoters_in_B._subtilis:">Inducible promoters in <I>B. subtilis</I>:</SPAN></H3><P>Phyper_spank:
</P><BLOCKQUOTE>Image from David Rudner, unpublished.</BLOCKQUOTE><P>Hy_spank
</P><UL><LI> IPTG inducible if LacI is present, otherwise very strong constitutive
</LI><LI> range between 0 and 1 mM IPTG<SUP><B>5</B></SUP>:
</LI></UL><PRE>GACTCTCTAGCTTGAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTC
GTTTTATCTGTTGTTTGTCGGTGAACGCTCTCCTGAGTAGGACAAATCCGCCGCTCTAGC
TAAGCAGAAGGCCATCCTGACGGATGGCCTTTTTGCGTTTCTACAAACTCTTGTTAACTC
TAGAGCTGCCTGCCGCGTTTCGGTGATGAAGATCTTCCCGATGATTAATTAATTCAGAAC
GCTCGGTTGCCGCCGGGCGTTTTTTATGCAGCAATGGCAAGAACGTTGCTCGAGGGTAAA
TGTGAGCACTCACAATTCATTTTGCAAAAGTTGTTGACTTTATCTACAAGGTGTGGCATA
ATGTGTGTAATTGTGAGCGGATAACAATTTAAGGAGGAACTACTATG</PRE><P>Pxyl
</P><UL><LI> xylose inducible
</LI><LI> range between 0 and 2 % xylose
</LI><LI> depends on relief from XylR repressor<SUP><B>6</B></SUP>:
</LI></UL><PRE>TCGGATCTTCATGAAAAACTAAAAAAAATATTGAAAATACTGATGAGGTTATTTAAGATT
AAAATAAGTTAGTTTGTTTGGGCAACAAACTAATGTGCAACTTACTTACAATATGACATA
AAATGCATCTAGAAAGGAGATTCCTAGGATGGGTACTAAGGAGGAACTACTATG</PRE><P>PspaS
</P><UL><LI> subtilin inducible
</LI><LI> range between 0 and 10%
</LI><LI> supernatant of a subtilin producing strain
</LI><LI> requires the SpaRK 2-component system<SUP><B>7</B></SUP>:
</LI></UL><PRE>GATCTTAAAAAAAGGAAAAAATTGATAAAATCTTGATATTTGTCTGTTACTATTTAGGTA
TTGAAAGGAGGTGACCACCATG</PRE><H3><SPAN class="mw-headline" id="Constitutive_promoters_in_B._subtilis:">Constitutive promoters in <I>B. subtilis</I>:</SPAN></H3><P><B>Low level:</B> PzapA:
</P><PRE>CAGCAGGCTGTACGGAATGGTTTTAATCAGGTTTCGGGCAATCTCGATGATTTGCGGATC
TTTTAAATCTTTAGAATCCTTAACACCGAGTTCCTTCATTAAAGGAAGCATGGTTTTGTC
GACGTACGCGCATACAACCGTCATTGGGCCAAAGTAATCTCCGGTTCCGACTTCGTCAGA
ACCGATAACAGACATTCCGGCGATATCTGCCGGAGGGGCATAGCGGGGATCAGCCGGCTT
TTTGGCCGTTTTCTTCTTTTCCTGAGGCTCTGCTGTTCCCCAGCGCGCGGATTCTGCCGC
AGCGTTTTTTCCTTGAAACAAGACCTTACCTGATTGATATGCTGTAATGGTACAGCCGGG
CGGTTTTGCCTGAAAAACGGCTCCCTGCGGAACAGAGGCTGTAAGTGAACCGCTGTACGT
CATTTTCATTTGGTCAATAGCAGACAACGATACTTTTATCACTGAATGGGACACGTAATA
ATCTCCTTTTTTTACACTTTTCGCTGTATATACCAGTGTATCATAACAGCGGGAGGCTCG
TCTTTCCATTCATTTAATAAACGTGTTATGATAAGAACTAGGATTCTCGCGGTAAGGAGG
AACTACTATG</PRE><P><B>Medium level:</B> PftsW:
</P><PRE>CTGCGAAAAGCGGCGTATCCGGCGGGATAATGGGCATTTCGCCTTACGATTTCGGGATCG
GCATATTCGGCCCCGCATTAGACGAAAAAGGGAACAGCATTGCAGGTGTAAAGCTTTTGG
AAATAATGTCTGAGATGTACAGGCTGAGTATTTTTTAATTTATGTCATATGCTTAAATCC
TCTTGCATTTTCTGTTGATACCCTTTATGATAAATAGAAGAATTAGGTACTCGCCTGGGG
AACGGAGGGATACTTTTGGCTTCAGAGATGATAGTTGACCATCGGCAAAAAGCTTTTGAA
CTCTTAAAAGTGGACGCTGAGAAGATTTTGAAGCTGATCCGAGTACAAATGGATAACTTA
ACGATGCCTCAATGTCCTCTATATGAAGAGGTTTTAGATACTCAAATGTTCGGGCTTTCG
AGGGAAATCGATTTTGCTGTCCGCCTGGGATTGGTTGATGAAAAAGACGGTAAAGACCTT
TTATACACATTGGAGCGCGAATTGTCTGCTTTGCATGACGCGTTTACAGCTAAATAAATG
ATAAAACTCAAACTTATTAACAGTTTGGGTTTTTTTATAACCGCTATTTTTCTCTCATCT
CATAAAAGACGGTCTTTTTTTACACATTCCTTCCGAATCGTATAGAGATTCTTCGTCTCG
TTTGATAAATTGTAGTAAAATAAAAAAGATAAATACATAAAAACCATAGATAATGGAAGT
TAGAAGCTAAGGAGGAACTACTATG</PRE><P><B>High level:</B> Pveg
</P><PRE>GCGTACAGACATTCTAAGCACTTTGTTGAAACAGTTCAAACAGCCCGAAAAAAGATCCCT
CACTTAGATCAGCTTGTTATTTTTGCGGGGGCCTGCCAATCCCATTTTGAATCACTCATC
AGAGCGGGTGCGAATTTTGCAAGTTCACCGTCAAGAGTCAATATTCATGCGCTTGATCCG
GTATATATCGTCGCGAAGATCAGCTTTACGCCGTTTATGGAACGGATTAATGTATGGGAA
GTGCTCCGTAATACGCTGACAAGAGAGAAAGGGCTTGGAGGTATTGAAACAAGAGGAGTT
CTGAGAATTGGTATGCCTTATAAGTCCAATTAACAGTTGAAAACCTGCATAGGAGAGCTA
TGCGGGTTTTTTATTTTACATAATGATACATAATTTACCGAAACTTGCGGAACATAATTG
AGGAATCATAGAATTTTGTCAAAATAATTTTATTGACAACGTCTTATTAACGTTGATATA
ATTTAAATTTTATTTGACAAAAATGGGCTCGTGTTGTACAATAAATGTTAAGAGGAGGAA
CTACTATG</PRE><H2><SPAN class="mw-headline" id="Bacillus_Terminators"><I>Bacillus</I> Terminators</SPAN></H2><P><I>rrnO</I>:
</P><PRE>TAGGACGCCGCCAAGCCAGCTTAAACCCAGCTCAATGAGCTGGGTTTTTTGTTTGTTAAA
AATGAAGAAGAAACTGTGAAGCGTATTTA</PRE><P><I>rrnA</I>:
</P><PRE>TAAAATCTAAGACATATCATGATTTATAAAAACAAAAAAACCTTGCAGGATATGCAAGGC
TTCTGGATGACCCGTACGGGATTCGAAC</PRE><H2><SPAN class="mw-headline" id="The_Centre_for_Bacterial_Cell_Biology">The Centre for Bacterial Cell Biology</SPAN></H2><P>Newcastle is lucky enough to have The Centre for Bacterial Cell Biology (CBCB), part of the Institute for Cell and Molecular Biosciences (ICaMB) at Newcastle University, on its doorstep. CBCB host the largest group of scientists in Europe working on the physiology and molecular biology of <I>Bacillus</I>.  As a result, we can make use of the expertise of the researchers based at the centre, which includes some of our advisors
</P><H2><SPAN class="mw-headline" id="Acknowledgements">Acknowledgements</SPAN></H2><P><A href="http://www.ncl.ac.uk/camb/cbcb/">The Centre for Bacterial Cell Biology</A></P><P><A href="http://www.ncl.ac.uk/camb/" class="external text" rel="nofollow">The Institute for Cell and Molecular Biosciences at Newcastle University</A></P><P><A href="http://www.cs.ncl.ac.uk/" class="external text" rel="nofollow">Newcastle University School of Computing Science</A></P><P><A href="http://www.ncl.ac.uk/camb/staff/profile/colin.harwood" class="external text" rel="nofollow">Colin Harwood</A></P><P><A href="http://www.ncl.ac.uk/camb/staff/profile/l.hamoen" class="external text" rel="nofollow">Leendert Hamoen</A></P><P><A href="http://www.ncl.ac.uk/camb/staff/profile/j.w.veening" class="external text" rel="nofollow">Jan-Willem Veening</A></P><P><A href="http://www.ncl.ac.uk/camb/staff/profile/jeff.errington" class="external text" rel="nofollow">Jeff Errington</A></P><P><A href="http://www.staff.ncl.ac.uk/j.s.hallinan/" class="external text" rel="nofollow">Jennifer Hallinan</A></P><P><A href="http://www.ncl.ac.uk/camb/staff/profile/anil.wipat" class="external text" rel="nofollow">Anil Wipat</A></P><P><A href="http://www.cs.ncl.ac.uk/people/matthew.pocock" class="external text" rel="nofollow">Matthew Pocock</A></P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI> Veening, J.-W., Igoshin, O. A., Eijlander, R. T., Nijland, R., Hamoen, L. W., Kuipers, O. P. (Apr 2008) “Transient Heterogeneity in Extracellular Protease Production by Bacillus subtilis” Molecular Systems Biology 4: 184
</LI><LI> Smits, W. K., Kuipers, O. P., Veening, J.-W. (2006) “Phenotypic Variation in Bacteria: the Role of Feedback Regulation” Nature 4: 259-271
</LI><LI> Vellanoweth and Rabinowitz (Mol Micro., 1992, 6 1105-1114)
</LI><LI> Ozbudak EM, Thattai M, Kurtser I, Grossman AD, van Oudenaarden A. Nat. Genet 2002 May;31(1):69-73
</LI><LI> Quisel, J., Burkholder, W.F., and Grossman, A.D. J Bacteriol. 2001 183 6573-8.
</LI><LI> Hastrup, S. (1988) Analysis of the Bacillus subtilis xylose region. In Genetics and Biotechnology of Bacilli. Ganesan, A.T., and Hoch, J.A. (eds). New York: Academic Press, pp. 79–83
</LI><LI> Bongers et al, 2005 Applied and Environmental Microbiology
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Bacillus_subtilis">http://2008.igem.org/Team:Newcastle_University/Bacillus_subtilis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Bacillus_subtilis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Bacillus_subtilis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Bacillus_subtilis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Bacillus_subtilis&amp;oldid=95954" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>