<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Biological_Brainstorming"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Biological_Brainstorming">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Biological_Brainstorming&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Biological_Brainstorming&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Biological_Brainstorming&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Biological_Brainstorming" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Biological Brainstorming</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A>&gt;&gt;<STRONG class="selflink">Biological Brainstorming</STRONG></P><P>How polymorphic are the quorum sensing peptides? The literature suggest that their sequence varies so much that they can be used to type strains. Can we identify a profile that is unique to MRSA strains? What about <A href="http://www.bacteriamuseum.org/species/anthrax.shtml" class="external text" rel="nofollow">Bacillus anthracis</A> or other close relative such as <A href="http://www.textbookofbacteriology.net/B.cereus.html" class="external text" rel="nofollow">Bacillus cereus</A> and <A href="http://www.ext.colostate.edu/pubs/Insect/05556.html" class="external text" rel="nofollow">Bacillus thuringiensis</A>? Other Gram positives? We'll need to do a literature search to find all of the genes that contain the sequences of quorum peptides in these organisms to identify as their profiles.
</P><P>Some simple aligments of protein sequences will then reveal just how variable the sequences are across species. 
</P><H2><SPAN class="mw-headline" id="Ideas_and_comments_on_the_Theme">Ideas and comments on the Theme</SPAN></H2><P>Difficulties finding peptides that are secreted by the individual organisms that are unique to these organisms.
</P><P>Concentrating on finding targets for:
</P><UL><LI><I>Streptococcus pneumoniae</I></LI><LI><I>Bacillus anthracis</I></LI><LI><I>Clostridium dificile</I></LI></UL><P>These are all relevant target species as they are pathogenic to humans world wide and require detection systems.
</P><P><I>Streptococcus pneumoniae</I> uses a CSP quorum-sensing system. The gene encoding the CSP for <I>S. pneumoniae</I> is comC. However this may not be unique to <I>S. pneumoniae</I>. BLAST result for comC gives top two similarities to <I>S.pneumoniae</I> but then score of 69.7 for <I>Streptococcus oralis</I>. Need to look for other peptides, possibly just any secreted peptide rather than those involved in quorum sensing.
</P><P>Looked at IgA1 protein of <I>S.pneumoniae</I> however this is homologous to that of <I>Streptococcus sanguis</I>. many other surface proteins are present however think that it needs to be one's that are actually secreted rather than just on the surface…………. <I>S. pneumoniae</I> does have hemolysins that are secreted however these bind cholesterol rather than actual receptors.
</P><P>Signalling peptide pep27 is a possibility, it is unique to <I>S.pneumoniae</I> however it is not well characterised and little information is available via EMBL and UniProt, but I shall continue to look at information on this as it seems most viable peptide found so far.
</P><P>Overall there are thirteen two-component systems identified in <I>S.pneumoniae</I>. Hopefullly at least one of these will be unique to the pathogen. comC etc. and also the YycFG are not unique so cannot be used. ciaR component system also used in other streptococcus species.
</P><P>Gene pairs, 484hk, 481hk, 486hk, 478hk, 539hk, 480hk and ciaRH all have similarity to other species including Bacillus. It is possible that they will have unique receptors that are not found in other species.
</P><P>Overall it appears that the best two-component system to use for detection of <I>S.pneumoniae</I> is using the vncS/R sensors that detect pep27.
</P><P>Very difficult to find the repressor thatis supposed to exist for the vnc two-component system.
</P><P>See Gilmore and Hoch 2007.
</P><P>This sugggests to me that the response regulator is phosphorylated, but not by a repressor. So perhaps there is not a repressor for this system that represses vncR (unless it is the repressor that acts to phosphorylate the vncR).
</P><P>It has been mentioned in 2000 that there is a RelA gene that mediates 'the' stringent response in the pathway. This remains to be uncharacterised, the only information I can find on this is an unreviewed Uniprot entry, it encodes the protein GTP pyrophosphokinase.
</P><P>A the moment I would conclude that there is not a repressor of the vncR, and that it is purely the phosphorylation that causes repression.
</P><P>Another diagram, Novak 1999
</P><P>There is no evidence for a repressor for the vnc system so we will not be able to use it. Will try looking at other systems in <I>S. pneumoniae</I> but not very optimistic.
</P><P>Of all of the other 2 component systems there are not any that are viable to use in our sensor. Other TCS include micABC, phoR/P, vicRK however these are all either homologous to other species, or do not have unique peptides or are not well enough characterised.
</P><P>Change of plan again…We will be using the com C sensory system.
</P><P>This shows the alignment of com C in S.pneumoniae with that of the <I>S. oralis</I> altough they are very similar there are differences. The com C is cleaved after the two glyceine residues, there are more difference after this part of the sequence.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Biological_Brainstorming">http://2008.igem.org/Team:Newcastle_University/Biological_Brainstorming</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Biological_Brainstorming" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Biological_Brainstorming" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Biological_Brainstorming&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Biological_Brainstorming&amp;oldid=49791" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>