<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_University_Evolutionary_Algorithm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Newcastle_University/Evolutionary_Algorithm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle_University/Evolutionary_Algorithm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Evolutionary_Algorithm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle_University/Evolutionary_Algorithm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle_University/Evolutionary_Algorithm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle University/Evolutionary Algorithm</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><H1><SPAN class="mw-headline" id="Newcastle_University"> Newcastle University </SPAN></H1><P><FONT color="gold" size="+2"><B>GOLD MEDAL WINNER 2008</B></FONT></P><TABLE class="wikitable"><TBODY><TR valign="bottom"><TH align="center"><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A></TH><TH align="center"><A href="/Team:Newcastle_University/Team" title="Team:Newcastle University/Team">Team</A></TH><TH align="center"><A href="/Team:Newcastle_University/Original_Aims" title="Team:Newcastle University/Original Aims">Original Aims</A></TH><TH align="center"><A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A></TH><TH align="center"><A href="/Team:Newcastle_University/Modelling" title="Team:Newcastle University/Modelling">Modelling</A></TH><TH align="center"><A href="/Team:Newcastle_University/ProofOfConceptBrick" title="Team:Newcastle University/ProofOfConceptBrick">Proof of Concept Brick</A></TH><TH align="center"><A href="/Team:Newcastle_University/Notebook" title="Team:Newcastle University/Notebook">Wet Lab</A></TH><TH align="center"><A href="/Team:Newcastle_University/Conclusions" title="Team:Newcastle University/Conclusions">Conclusions</A></TH></TR></TBODY></TABLE><P><A href="/Team:Newcastle_University" title="Team:Newcastle University">Home</A>&gt;&gt;<A href="/Team:Newcastle_University/Software" title="Team:Newcastle University/Software">Software</A>&gt;&gt;<STRONG class="selflink">Evolutionary Algorithm</STRONG></P><DIV id="maincol"><H2><SPAN class="mw-headline" id="Evolutionary_Algorithm">Evolutionary Algorithm</SPAN></H2><H3><SPAN class="mw-headline" id="Aims_and_Objectives:">Aims and Objectives:</SPAN></H3><P>To develop a system that will evolve genetic circuits represented as Artificial Neural Networks (ANNs)which meet the functional requirements specified by the team's target application.
</P><P>Traditional genetic engineering techniques have built small biological circuits by hand. However, this approach will not scale to whole-organism engineering. For synthetic biology at this scale computational design will be essential. “Soft” computing techniques such as evolutionary computation and computational intelligence were developed to handle exactly this sort of large, complex, hard-to-define problem.
</P><P>The EA should:
</P><UL><LI> Read parts list from <A href="/Team:Newcastle_University/Parts_Repository" title="Team:Newcastle University/Parts Repository">parts repository</A></LI><LI> Read constraints on parts assembly (from <A href="/Team:Newcastle_University/Constraints_Repository" title="Team:Newcastle University/Constraints Repository">Constraints Repository</A>)
</LI><LI> Assemble part models into a larger model
</LI><LI> Simulate the behaviour of the composite model
</LI><LI> Read desired 'input' behaviour from <A href="/Team:Newcastle_University/Workbench" title="Team:Newcastle University/Workbench">Workbench</A></LI><LI> Read desired 'output' behaviour from workbench
</LI><LI> Assess fitness
</LI><LI> Mutate the model
</LI><LI> Repeat the last two steps until a stopping criterion is encountered
</LI><LI> Output the fittest model as CellML (to workbench)
</LI></UL></DIV><H3><SPAN class="mw-headline" id="Interaction_Diagrams">Interaction Diagrams</SPAN></H3><P>The above is a schematic that shows the communication required between the workbench and the EA.  The workbench requests a new EAID which is created and provided by the EA.  The workbench then provides the EA with all the initial data from the user including initial parts, wiring (constraints) and population size.  
</P><P>The user interacting with the workbench then requests the EA to run..  The EA returns information to the user on request.  The current generation number the EA is working from can be requested by the user at any time.  The fitness value can be returned at a generation number specified by the user.  When the EA finishes evolving, or when the user at the workbench requests the EA to stop, the final composite model and fitness value are returned by the EA to the Workbench.
</P><H3><SPAN class="mw-headline" id="Outcomes">Outcomes</SPAN></H3><P>The work for the EA was concluded on 1 Sept 2008.
</P><P>Our aim was to develop a strain of <I>B. subtilis</I> which detects quorum-sensing peptides from other pathogenic gram-positive species and determines their presence by glowing a range of different colours. Our <I>B. subtilis</I> has a range of genes inserted that enable detection of these peptides. The issue that we had to overcome is the fact that we have more quorum-sensing peptides to detect than there are fluorescent proteins to show their presence. This makes planning the construct by hand much more difficult. Whilst there are at present only four different peptides to be detected, in the future this number could increase vastly and the <I>B. subtilis</I> should still be able to cope with an output of only three fluorescent proteins. 
</P><P>This is justification for the usage of artificial neural networks (ANNs) and evolutionary algorithms. Planning the construct by hand would be difficult, if not impossible. We had three layers of complexity that fit into the neural network structure, and we also wanted the potential user to be able to enter both the inputs and outputs to the network.  The mapping between the ANN and the genetic circuit can be summed up as follows:
</P><UL><LI> The input layer to the neural network is represented by the two-component genes, activated by the peptides from the four gram-positive pathogens.  Each receptor represents a node.  
</LI><LI> The output layer is represented by the three fluorescent proteins; GFP, YFP and mCherry.
</LI><LI> The hidden layer is an assortment of different transcription factors.  
</LI><LI> The user specifies the inputs and the outputs; i.e. in the presence of said peptide, said fluorescent protein will light up.
</LI></UL><P>This was the starting point from which the construct was designed.
</P><H4><SPAN class="mw-headline" id="Artificial_Neural_Networks">Artificial Neural Networks</SPAN></H4><P>Artificial Neural Networks (ANNs) are algorithms used for learning and optimisation (Engelbrecht, 1998).  As the name suggests, the idea is based on the organization of neurons in the brain.  An ANN is made up of three or more layers.  One of these layers is the input layer, another is the hidden layer and another is the output layer.  Each layer contains a number of nodes, points of contact for information flow mimicking what occurs in the nervous tissue.  Each node is normally a representation of some kind of variable object; for example, a promoter(Montana, 1989).  The network is usually represented as a directed graph; the links between the nodes make up what is called network topology.  Each node has both a weight and a threshold value associated with it(Montana, 1989).  The weight of an input to a node is the number which when multiplied with the input gives the weighted input.  There is a threshold value present on each of the nodes.  When the sum of the weights and inputs leading into that node are added together, if it exceeds the threshold value the node is activated.  Otherwise it stays in its inactivated state.  
</P><P>This ANN structure can be used as a base for the EA.  Each node in the network represents a different genetic part.  There are a number of different types of ANN including Radial Basis Function ANNs, feedforward ANNs and stochastic ANNs(Engelbrecht, 1998).  This project has been constructed using a feedforward ANN.
</P><P>A feedforward ANN differs from other ANNs as it has a closed topology.  The input layer is connected only to the hidden layer, and the output layer is too only connected to the hidden layer (Montana, 1989). The output layer values are calculated by the propagation of the output values of the nodes in the input and hidden layers.  The value of a node in the hidden or output layer is determined by the sum of each of the values of the nodes in the previous layer multiplied by the weights of each of the node connections between previous layer nodes and the current node(Montana, 1989). The ANN learns by adjusting the weights on the connections, which in the case of the network means replacing promoters with other promoters with a different binding affinity.
</P><H4><SPAN class="mw-headline" id="The_Genetic_Algorithm">The Genetic Algorithm</SPAN></H4><P>The Genetic algorithm is the most popular and widely used form of evolutionary algorithm.  Here a population of individual chromosomes are represented as a type of data structure.  This data structure can be characters, strings, bits, integers or any other appropriate type.  The genome is often a fixed length binary string.  There is a mutation operator, which alters a certain number of the chromosomes, and only those that satisfy the fitness criteria are allowed to recombine(Mitchell, 1996).  The population size is kept constant and individuals are evaluated at each generation.  Fitness is commonly evaluated as a probability proportional to fitness, or the error is measured between the observed and expected output (Peña-Reyes and Sipper, 2000). There is also a crossover operator between two selected parents, which exchange part of their genomes to form two offspring. The mutation and crossover operators preserve the population size (Peña-Reyes and Sipper, 2000). The user enters the expected output to the algorithm, the population will be evolved, and the process iterated until this solution is found (within a given error boundary)(Mitchell, 1996).
</P><H4><SPAN class="mw-headline" id="Mutation">Mutation</SPAN></H4><P>The mutate function was carefully planned.  There was no point in randomly changing parts, as it would be very computationally expensive.  The user sets the order of parts in the workbench, and only parts of a similar type can be swapped into the hidden layer.  For example, if a promoter region is mutated, only another promoter region can replace it.  Thhere is a repository of parts from which the mutate function can retrieve the new parts.  The order of the two-component genes is kept the same, as set by the user and regions such as ribosome-binding sites, promoters, transcription factors and sigma factors were swapped in and out of the hidden layer.  
</P><P>The Parts Repository is a SQL-accessible database full of these different parts that can be retrieved by the EA.  The aim of the Constraints Repository is to enable both the user controlling the workbench and the EA to place parts next to each other and to change parts in a sensible way.  Therefore if the user or mutate operator tried to design a construct that contained two unsuitable parts next to each other, it would not be allowed.  The mutate operator would not be allowed to replace parts with other unsuitable parts.  
</P><H4><SPAN class="mw-headline" id="Populations_and_Fitness_Functions">Populations and Fitness Functions</SPAN></H4><P>There were two different possible ways of setting up populations.  It was possible to create a large population of individuals each representing a different neural network structure (or construct).  It was also possible to take the evolutionary strategies approach and have a population size of just one.  It was decided initially to use a population size of one.
</P><P>The fitness function quantifies how optimal a solution is to a problem.  For EAs there are a wide variety of different genes that can be combined and mutated, and a variety of different conditions under which this can occur.  An EA often has to search across a ‘fitness landscape,’ involving many different gene combinations and conditions to fins the fittest solution.  The fitness function for this EA will have to work with error values.  The desired output that the user has input will be compared with the value that has fed through the network.  Only when this error reaches a low enough error threshold would the fittest solution have been found. 
</P><H4><SPAN class="mw-headline" id="Discussion_of_Evolutionary_Algorithm_Implementation">Discussion of Evolutionary Algorithm Implementation</SPAN></H4><P>The original aim of the EA was that it would input biological Part models from the Parts Repository and evolve them using a feedforward neural network structure.  Unfortunately, did not quite get this far before time ran out on the project.  The EA / neural network structure however, is in place.  This takes in a variety of double values as inputs and outputs and uses the complete mutate and fitness function operators to evolve these numbers to within an error threshold.  The EA also has the capacity to simulate composite CellML models through the run class using the program <A href="http://www.j-sim.org/" class="external text" rel="nofollow">JSim</A>.  
</P><P>The original EA was designed to:
</P><UL><LI> Have a feedforward neural network
</LI><LI> Integrate biological part and constraint models from the parts and constraints repositories
</LI><LI> Create composite models
</LI><LI> Mutate and evaluates fitness these part models based on numerical data held inside the models.
</LI><LI> Carry out hidden layer learning based on the user specifying inputs and outputs
</LI></UL><P>The actual EA carries out the following
</P><UL><LI> Has a feedforward neural network
</LI><LI> Carries out hidden layer learning based on the user entering numerical inputs and outputs
</LI><LI> Creates composite models – however this step is not incorporated into the neural network part of the EA yet.
</LI><LI> Simulates models – again this is not incorporated into the neural network part of the EA yet.
</LI></UL><P>Due to time constraints and unforeseen problems with software the EA does not do everything it should in the way we would like.  The software does however do everything we wanted it to.  It has a learning neural network algorithm, it produces composite part models and it simulates CellML models.  With some more work and more time spent on the algorithm, in conjunction with the parts and constraints repository the software will work to specifications.  All of the principles that were laid out at the start have been fulfilled; they just have not come together yet.  
</P><H3><SPAN class="mw-headline" id="Further_Reading">Further Reading</SPAN></H3><P>Adorf, H. (1990) A discrete stochastic neural network algorithm for constraint satisfaction problems, Neural Networks, 1990, 3, 917-924.
</P><P>Engelbrecht, A.P. (1998) Selectine learning using Sensitivity analysis, Neural Networks Proceedings, 1998, <I>IEEE World Congress on Computational Intelligence.</I>, <B>2</B>, 1150-1155.
</P><P>Estrada-Gil, J.K., Fernandez-Lopez, J.C., Hernandez-Lemus, E., Silva-Zolezzi, I., Hidalgo-Miranda, A., Jimenez-Sanchez, G. and Vallejo-Clemente, E.E. (2007) GPDTI: A Genetic Programming Decision Tree Induction method to find epistatic effects in common complex diseases, <I>Bioinformatics</I>, <B>23</B>, i167-174.
</P><P>Forrest, S. (1993) Genetic algorithms: principles of natural selection applied to computation, <I>Science</I>, <B>261</B>, 872-878.
</P><P>Hallinan, J., Wiles, <I>J  Evolutionary Algorithms</I>.
</P><P>Karayiannis, N.B. and Mi, G.W. (1997) Growing radial basis neural networks: merging supervised and unsupervised learning with network growth techniques, <I>Neural Networks, IEEE Transactions on</I>, <B>8</B>, 1492-1506.
</P><P>Lawrence S, G.L., Tsoi, AC (1997) Lessons in Neural Network Training: Overfitting May be Harder than Expected, <I>Proceedings of the Fourteenth National Conference on Artificial Intelligence</I>, 540-545.
</P><P>Mitchell, M. (1996) An Introduction to Genetic Algorithms. MiT Press, 205.
</P><P>Montana, D., Davis, L (1989) Training feedforward neural networks using genetic algorithms, <I>Proceedings of the Eleventh International Joint Conference</I>, <B>6</B>.
</P><P>Peña-Reyes, C.A. and Sipper, M. (2000) Evolutionary computation in medicine: an overview, <I>Artificial Intelligence in Medicine</I>, <B>19</B>, 1-23.
</P><P>Searle, W. (1996) Stopped Training and remedies for overfitting, <I>Proceedings of the 27th Symposium on the Interface</I>, 1995.
</P><H3><SPAN class="mw-headline" id="Contributors:">Contributors:</SPAN></H3><P>Lead: <A href="/Team:Newcastle_University/Mark_Wappett" title="Team:Newcastle University/Mark Wappett">Mark Wappett</A></P><DIV id="sidebar"><DIV class="post-it bluebox"><H3><SPAN class="mw-headline" id="Evolutionary_Algorithm_2"> Evolutionary Algorithm </SPAN></H3><UL><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#aims" title="Team:Newcastle University/Evolutionary Algorithm">Aims and Objectives</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#interaction-diagrams" title="Team:Newcastle University/Evolutionary Algorithm">Interaction Diagrams</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#outcomes" title="Team:Newcastle University/Evolutionary Algorithm">Outcomes</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#anns" title="Team:Newcastle University/Evolutionary Algorithm">Artificial Neural Networks</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#genetic-algorithm" title="Team:Newcastle University/Evolutionary Algorithm">Genetic Algorithm</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#mutation" title="Team:Newcastle University/Evolutionary Algorithm">Mutation</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#pops" title="Team:Newcastle University/Evolutionary Algorithm">Populations and Fitness Functions</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#implementation" title="Team:Newcastle University/Evolutionary Algorithm">Implementation</A></LI></UL><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#further-reading" title="Team:Newcastle University/Evolutionary Algorithm">Further Reading</A></LI><LI><A href="/Team:Newcastle_University/Evolutionary_Algorithm#contributors" title="Team:Newcastle University/Evolutionary Algorithm">Contributors</A></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Newcastle_University/Evolutionary_Algorithm">http://2008.igem.org/Team:Newcastle_University/Evolutionary_Algorithm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle_University/Evolutionary_Algorithm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle_University/Evolutionary_Algorithm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle_University/Evolutionary_Algorithm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle_University/Evolutionary_Algorithm&amp;oldid=90565" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>