<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ULB-Brussels_Project_Appendix"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ULB-Brussels/Project/Appendix">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ULB-Brussels/Project/Appendix&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/Project/Appendix&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/Project/Appendix&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ULB-Brussels/Project/Appendix" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ULB-Brussels/Project/Appendix</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>iGEM Team:ULB-Brussels Wiki</TITLE></P><DIV class="header"><DIV class="menu"><UL><LI><A class="mainlink" href="/Team:ULB-Brussels">Home</A></LI><LI><A class="mainlink" href="/Team:ULB-Brussels/Team">Team</A></LI><LI><A class="mainlink selection" href="/Team:ULB-Brussels/Project">Project</A></LI><LI><A class="mainlink" href="/Team:ULB-Brussels/Parts">Parts</A></LI><LI><A class="mainlink" href="/Team:ULB-Brussels/Safety">Safety</A></LI><LI><A class="mainlink" href="/Team:ULB-Brussels/Notebook">Notebook</A></LI><LI><A class="mainlink" href="/Team:ULB-Brussels/Sponsors">Sponsors</A></LI></UL></DIV><DIV class="submenu"><UL><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Introduction">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Biological">Biological model</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Properties">Properties</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Mathematical">Math modeling</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Perspectives">Perspectives</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Project/Appendix">Appendix</A></LI><LI><A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography">Bibliography</A></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Homolog_genes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Homolog genes</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Homolog_genes_similarities"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Homolog genes similarities</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Sequence_alignment"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Sequence alignment</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Sequence_optimization_and_Standard_compliance"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Sequence optimization and Standard compliance</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Bacterial_Chemotaxis"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Bacterial Chemotaxis</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Quorum_Sensing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Quorum Sensing</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Homolog_genes">Homolog genes</SPAN></H2><H3><SPAN class="mw-headline" id="Homolog_genes_similarities">Homolog genes similarities</SPAN></H3><P>Comparison of sequence similarities between genes implied in holdfast biosynthesis in <I>Caulobacter crescentus</I><A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(37)</A> and their homologs in <I>Escherichia coli</I><A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(36)</A> after sequence alignment <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(35)</A>.
</P><H3><SPAN class="mw-headline" id="Sequence_alignment">Sequence alignment</SPAN></H3><P>Sequence alignment between homolog genes in Caulobacter crescentus CB15 genome (GenBank id: 19172958) and <I>Escherichia coli K30</I> (GenBank id: 8469170). Stars indicate exact matches between sequences.
</P><CENTER><TABLE border="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsA-wzc
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsB-wzc
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsC-wzy
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsD-wza
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsE-wbaP
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 220px;"><DIV class="gallerytext"><P>hfsF-wzx
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Sequence_optimization_and_Standard_compliance">Sequence optimization and Standard compliance</SPAN></H2><CENTER><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Modifications applied to the sequences in order to be compliant with BioBrick prefix and suffix. Blue and green elements are added to the gene sequence, orange parts are removed</DIV></DIV></DIV></CENTER><CENTER><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Sequence comparison between wild-type hfsH from <I>Caulobacter crescentus</I><A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(37)</A> and our optimized gene for <I>Escherichia coli</I>. This sequence includes addition of prefix and suffix as well as removal of restriction sites recommended by BioBrick assembly standart 10.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Sequence comparison between wild-type hfsG from <I>Caulobacter crescentus</I><A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(37)</A> and our optimized gene for <I>Escherichia coli</I>. This sequence includes addition of prefix and suffix as well as removal of restriction sites recommended by BioBrick assembly standart 10.
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Bacterial_Chemotaxis"> Bacterial Chemotaxis </SPAN></H2><P>In order to respond to changing environment, bacteria developed sensory systems. One way to respond may be to move. And that is achieved by the action of small phosphorylated molcules binding to the flagellar rotor. There are many different sensory pathways in eukaryotes as well as in prokaryotes. Prokaryotes commonly use a histidine-aspartate phosphorelay (HAP) system consisting of a histidine protein kinase (HPK) and a response regulator (RR). The environmental signals can be concentrations of nutrients or toxis, oxygen levels, pH, osmolarity, light,...
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">he domain organization of selected histidine-aspartate-phosphorelay systems. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(30)</A></DIV></DIV></DIV><P>The HAP systems works as follows: the HPK (e.g. in <I>Escherichia coli</I>, CheA interacting with transmembrane receptors) is trans-autophosphorylated on a His residue. The phosphoryl group is then transferred to an Asp residue situated on a RR that becomes therefore activated and able to respond to the signal. In the chemosensory pathway of <I>E. coli</I>, there are two RRs competing for the phosphoryl group: CheY and CheB. The first one controls flagellar motor switching: by binding to the flagellar motor, it causes the switch to clockwise rotation, the cell tumbles and swims off in a new direction. The second one is a methylesterase that controls the adaptation of the methyl-accepting chemotaxis proteins or chemoreceptors (MCPs). <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(30)</A></P><P><I>E. coli</I> possesses at least four MCPs: Trg for ribose and galactose; Tar for aspartate; Tsr for serine; Tap for peptides. A fifth one may be a redox detector. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(31)</A></P><P>In the absence of gradients of chemoattractants, the cells walk randomly. They evaluate the changes in concentrations by comparing the number of bound receptors over the past 1s with the number during the past 3s. When this number increases they are expected to rotate counterclockwise and therefore be propelled forward. When they move away from the cells cluster, they tumble in order to come back. The tumble rate and the clusters boundaries are thus determinated by the sensory memory of cells. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(32)</A></P><H2><SPAN class="mw-headline" id="Quorum_Sensing"> Quorum Sensing </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Model of acyl-HSL quorum sensing in a single generalized bacterial cell. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(30)</A></DIV></DIV></DIV><P>Quorum sensing was first identified in <I>Vibrio harveyi</I> and <I>Vibrio fischeri</I>, two bioluminescent marine bacteria, and was then identified as a highly conserved regulatory system among the Proteobacteria.
</P><P>Quorum sensing is a community behaviour that enables microorganisms to communicate, perceive and monitor population density, and modulate gene expression by producing and responding to diffusible signal molecules. There are different signalling systems but the bestcharacterized mechanism is the one relying on the acylated homoserine lactones (acyl-HSL, e.g. 3OC<SUB>6</SUB>HSL).
</P><P>Acyl-HSLs are synthesized (at a low level) by acyl-HSL synthases like the LuxI protein. These signal molecules are then removed from the cell by diffusion. When the population increases, the signal concentration elevates and at a threshold level the acyl-HSLs come in interaction with transcription factors like the LuxR proteins. That multimer is now able to activate the quorum sensing-regulated genes by binding the Lux box. These target genes can include the LuxI coding region in order ton make a positive feedback loop. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(33)</A></P><P>Reaching the critical level of quorum sensing molecules enables therefore a coordinated	expression of specific genes, for example the luminescence genes in V. fischeri or the competence in S. pneumoniae. This cell-to-cell communication also plays a role in biofilm development. <A href="https://2009.igem.org/Team:ULB-Brussels/Projet/Bibliography" class="external text" rel="nofollow">(34)</A>
We implemented the LuxI-LuxR mechanism into our GluColi in order to monitor the glue production. So the target genes are in our case hfsG and hfsH coding for the Caulobacter enzymes needed to synthesize the glue.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:ULB-Brussels/Project/Appendix">http://2009.igem.org/Team:ULB-Brussels/Project/Appendix</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ULB-Brussels/Project/Appendix" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ULB-Brussels/Project/Appendix" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ULB-Brussels/Project/Appendix&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ULB-Brussels/Project/Appendix&amp;oldid=168829" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>