"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Project/Biological
From 2009.igem.org
iGEM Team:ULB-Brussels Wiki
Home
Team
Project
Parts
Safety
Notebook
Sponsors
Introduction
Biological model
Properties
Math modeling
Perspectives
Appendix
Bibliography
Contents
1Biological model
1.1Material and Methods
1.1.1Transformation
1.1.2Ligations
1.2GluColi, engineering a biological circuit
1.3Circuit validation
1.3.1Circuit I
1.3.2Circuit II
1.3.3Circuit III
Biological model
Material and Methods
On this figure we show the plasmid we created in order to insert the hfsG and hfsH genes.
pSB1AK3 final plasmid
The following bricks come from the BioBricks partsregistry . We chose a lactose inductible promoter so that we could easily control the expression of the genes of interest. This promoter is the BBa R0011 brick (located in the pSB1A2 ampicillin resistant plasmid).
The RBS (ribosome binding site) and the RFP bricks are located on the same plasmid (BBa K093005). RFP is a reporter gene. If the glue is not released (or we can not detect it), we will know the genes are expressed.
We chose a commonly used terminator, namely BBa B0015 (located on the pSB1AK3 kanamycin resistant plasmid). The replication origin is a pUC19-derived pMB1 (copy number of 100-300 per cell). The final plasmid pSB1AK3 is ampicillin and kanamycin resistant. Seeing the importance of hfsG ans hfsH in our project, they were optimized and synthesized MrGene ). Their sequences have been optimized for two reasons:
First they are compatible with all assembly standards
Secondly the use of codon is not the same in Caulobacter crescentus and in E. coli. This could limit the rate of translation of our genes.
The hfsG and hsfH sequences show the restriction sites and the optimized codons ( Appendix ).
Transformation
To increase the amount of plasmidic material, we performed transformations with DG1 E. coli competent cells from Delphi Genetics . Its genotype is the following one:
For the full protocol see the StabyExpressTM T7 Kit Manual, transformation using chemically competent cells .
Ligations
All the ligations were achieved in the Assembly standard 10 . We followed the assembly protocol recommended by New England BioLabs . The first ligation concerns promoter, RBS and RFP. We cut the promoter plasmid with the restriction enzymes EcoR1 and Spe1. We cut the RBS+RFP plasmid with EcoR1 and Xba1. This one was dephosphorylated to prevent it from self-ligating without any inserts. Seeing that both plasmids are ampicillin resistant, we cut the promoter plasmid with AflIII and ScaI. This will prevent any promoter plasmid from being  transformed. At this stage we have promoter, RBS and RFP in the RBS-RFP ampicillin resistant plasmid ( see figure below ).
RBS$-$RFP ampicillin resistant plasmid
We transformed DG1 E.coli competent cells with this construction. The culture was left overnight (37°C) on ampicillin resistant medium. We selected the appropriated colonies and put them back at 37°C in liquid medium.
We proceeded with the extraction of the plasmid using the GenElute Plasmid Miniprep Kit
(Sigma-Aldrich ref (PNL350))
. In order to verify our construction, we cut upstream (EcoR1) and downstream (Pst1) of the Promoter+RBS+RFP. Then we proceeded with the electrophoresis of the digested product. We tested the construction by adding IPTG to a sample and this one was analyzed by fluorescent microscopy.
The second ligation concerns hfsG, hfsH and the terminator. Seeing that both hfsG and hfsH
plasmids were ampicillin resistant and that we did not have appropriate enzymes to cut one of them, we chose to ligate hfsH and the terminator (which have different antibiotic resistances). The terminator plasmid is the destination plasmid and hfsH is the insert. The selection was made with kanamycine (following the above-cited protocol). After that, hfsG was added to this construction. At this stage, we have hfsG, hfsH and terminator in the terminator kanamycin resistant plasmid which is our final destination plasmid,  (see Figure below).  Finally we assembled the first and the second construction to obtain the final plasmid,  (see figure above).
hfsG and hfsH ampicillin and kanamycin resistant plasmid
GluColi, engineering a biological circuit
For our first participation in the iGEM competition, we aimed at producing a new biological material. Finding a new way to produce a biological and biodegradable glue became our objective. Once we found a way to produce this, it was interesting to think about how we could have this glue produced where it is needed.
We designed a biological circuit composed of three main parts:
Bacteria produciong glue are attracted to the target point to be repaired.
Glue production at the leaking point, the crack or the fracture.
Preventing GluColi proliferation.
Schema of the biological system
Escherichia coli has a natural system of chemotaxis ( Appendix ).
Thanks to its flagella, it can swim towards a local chemical gradient. In order to guide GluColi to its target, we thought that the leaking should use aspartate as we know it is a good chemoattractant for E. coli. As it is a natural system found in E. coli we did not intend to add bricks for chemotaxis.
After that, a certain bacterial density is needed to produce enough glue. We engineered a glue production system controlled by quorum sensing, ( Appendix ). As long as there are not enough bacteria at the target point, glue will not be produced. The quorum sensing mechanism senses the population density and coordinates gene expression. Glue production will be triggered only at high GluColi density.
The suggested circuit relies on a cascade of promoter repression/expression that will result in a positive feedback. Phage and quorum sensing promoter are used.
IPTG (Isopropyl β-D-1-thiogalactopyranoside) would permit the transcription of the downstream genes comprising cI434 which codes for a repressor issued from bacteriophage 434.
When  there is no leaking, thus no IPTG, the quorum sensing is OFF because of the promoter placed in front of the genes involved in it. This promoter is a hybrid promoter proposed by last yearís KULeuven team and works as follows : it is repressed by c2 P22, (from phage 22), and activated by the complex LuxRHSL.
If there is neither c2 P22 nor LuxR-HSL, then there will be a low background expression of the downstream genes. The activation by LuxR-HSL permits this device to be fully activated. The repression by c2 P22 is provided by the expression c2 P22 which is under control of the cI434 promoter.
This promoter is constitutive. So, as long as there will be no IPTG, c2 P22 will be produced and there will be no quorum sensing and no glue. As soon as IPTG is present, cI434 is produced in order to repress cI434 promoter, c2 P22 will not be expressed anymore and the quorum sensing can take place. Once LuxR and HSL dimerize, they can also fully activate the production of Caulobacter crescentus 's enzymes needed for the glue synthesis. We also added a lux box in front of the cI434 device to ensure that c2 P22 is well repressed when the glue needs to be produced.
Now that the glue is produced, as we are not sure it will kill our bacteria and with the intention of not letting the bacteria proliferate, we intended to introduce a toxin that would inhibit cell growth without really killing the bacteria. We thought about parE which target is E. coli gyrase (1) .  By lack of time, we could not test our circuit. Here under, we describe how we thought to implement it into E. coli.
We would use four different plasmids. The first plasmid would be the cI434/c2 P22 generator depending on the presence or absence of IPTG. The second one would be the LuxR generator, following by the LuxI generator plasmid. The last one would be responsible for the glue synthesis and the "death" of the bacteria.
The first plasmid includes a lac promoter, which is a strong promoter (BBa R0011) induced by IPTG and repressed by LacI. The lac promoter controls the expression of the repressor cI434 from the cI434 generator (BBa P0152) consisting of a RBS, the repressor itself and a terminator. As said above we also added a Lux box promoter (BBa R0062).
The second plasmid contains a very strong promoter controlled by the CI434 repressor (BBa R0052). This promoter controls the expression of the c2 P22 repressor gene included in the c2 P22 generator (BBa P0153 ). In that way, if there is IPTG, there will be no c2 P22.
The third plasmid includes a hybrid promoter (BBa K145150) consisting of the lux box and binding sites for c2 P22. Transcription from this promoter can be activated by a LuxR-HSL complex and repressed by c2 P22. The downstream gene codes for the LuxR protein (BBa I0462) which can sense 3OC6HSL ( Appendix ). When there is IPTG, as there is no c2 P22, the LuxR protein is therefore synthesized at a low rate, the quorum sensing takes place and enhances itself.
The fourth plasmid includes the same hybrid promoter (BBa K145150) and the LuxI generator (BBa K092400) that allows the synthesis of 3OC6. As well as the LuxR protein, this occurs after activation by the presence of IPTG.
The glue device itself is also under control of the hybrid promoter (BBa K145150) and allows the expression of hfsG (a glycosyltransferase) and hfsH (a polysaccharide deacetylase) genes and the toxin parE.
This circuit remains theoretical. The mathematical model is discussed in the section "Mathematical modeling".
Further, another aspect we should bear in mind is what would happen if the bacteria do not reach the target. A solution involving the synthesis of a toxin "after a while" should be considered.
Circuit validation
In order to validate experimentally our biological circuit and to characterize some of its parameters, some tests should be carried out. We divided the circuit in different parts.
Circuit I
This circuit (Experimental model:Construction : Bba R0011+BBa P0152 ; Part: BBa R10522 + Part: BBa E0240 ( see figure below) allows us to characterize and verify the performance of the C0056 inhibitor. For this purpose the C0053 gene needs to be substituted by a GFP.  Without IPTG no fluorescence should be detected. Chemical kinetics illustrating the fluorescence rate should be measured after adding IPTG to the substrate.
Experimental model
Theorical model
Circuit II
Another experiment should also be carried out to test the influence of the C0053  inhibitor on the K145150 promoter,  (see figure below)(Experimental circuit: Construction: BBa R0080 , BBa_C0053, BBa_K145150 , Part: BBa E0240).
Experimental circuit. Repression by c2p22
Theoretical quorum sensing circuit
The promoter upstream from the inhibitor is induced by arabinose.
Circuit III
The Quorum sensing should also be tested (see figure below): we use promoters whose  regulation can be controlled by LuxI and LuxR. The fluorescence rate would be measured according to time and population density. Then, we can determine the time necessary for the bacteria to reach a certain density allowing them to produce the LuxR-HSL complex.
Experimental quorum sensing circuit
Theoretical quorum sensing circuit
Retrieved from " http://2009.igem.org/Team:ULB-Brussels/Project/Biological "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
