"
Page
Discussion
View source
History
teams
Log in
Â 
Team:NYU Gallatin/Project/Cloning/Primers
From 2012.igem.org
Main menu
Home
Team
Project
Parts
Modeling
Notebook
Safety
Attributions
Profile
Aseatobacter
NYU Gallatin 2012 iGEM Team
Main menu
HomeHome of Aseatobacter.
Home
TeamThe brains of the operation.
Team
ProjectLearn more about our project.
Project
PartsOur work with the parts registry.
Parts
ModelingHow we put it all together.
Modeling
NotebookLab notebooks, news, and photos.
Notebook
SafetyOur commitment to safety.
Safety
AttributionsGive credit where credit is due.
Attributions
ProfileOfficial iGEM 2012 profile.
Profile
The Project
Project
Cloning
Growing
Designing
Technology
Socializing
Cloning: Designing Primers
Our plan is to generate 3 plasmids: one each with AGM1, NAG1, and UAP1. These we will Gibson assemble using the G-bits that Ellen ordered and a Gibson assembly kit sold by NEBiolabs. This part will require noprimers, so we can get started as soon as the DNA arrives.
The part that does require primers is after we have those three plasmids assembled (which will take, with luck, a week?). We want to use these plasmids to amplify the genes with overlapping sequences. These three genes (with overlapping sequences) we can then gibson assemble onto one larger plasmid. Don't know how Gibson assembly works? Refer to this website: https://2010.igem.org/Team:Cambridge/Gibson/Mechanism
How do we design primers with overlapping sequences? Recall that DNA open reading frames are read 5' --> 3'; in other words:
5' ATGnnnaaannncccnnnaaannncccnnnTAA 3'
and because DNA is double-stranded, the complementary sequence would be:
3' TACnnntttnnngggnnntttnnngggnnnATT 5'
When designing a primer, it is ABSOLUTELY VITAL that the 3' primer end has the identical sequence of the DNA you want to copy. This is the end the new nucleotides will be added to as the piece of DNA is synthesized. You need approximately 20 base pairs of your target sequence on the 3' END. Ideally, the very last nucleotide region is gc-rich, as these have 3 hydrogen bonds instead of the 2 that A-T pairs have, and this will decrease nonspecific amplification.
However, the 5' primer end does not have to match the target sequence, and this is what comes in handy when designing these gibson-assembly primers. The 5' sequence can be completely different, and in fact, what we WANT is for the homology to be to a DIFFERENT sequence - the sequence preceding the target sequence in the final plasmid product. This will facilitate the ends being "sticky" for each other when incubated with the exonuclease.
These extra ends can be upwards of 60-100 bases long - however, remember that each base costs money, so it's a balance between ensuring you will get the desired final product and cost/efficiency. We need 40 base pairs at the 5' end that will be homologous to the next sequence, in order to facilitate gibson assembly. Incorporating the ribosome binding sites may or may not count as homologous sequence, depending on how the DNA pieces are amplified. (This becomes simpler when drawn out, so try using different colors for gene1, gene2, the ribosome binding site, the plasmid backbone. Draw the primers and the amplified products using these colors. It really does help to clarify the cloning scheme.)
This can all be a bit confusing, so refer to the website I linked, and to the image I attached. The image I attached is the scheme for designing the primers for our plasmid (I'll detail this more below).
First up: how to design a primer
When designing a primer, you need to remember that sequence is added 5' --> 3'. Thus, the forward primer is easy to design; in the above sequence, it would be:
5' ATGnnnaaannnccc 3'
But the complementary sequence is also generated 5' --> 3'. And when we order primers, the company requires they be written in the direction of synthesis, so instead of writing:
3' tttnnngggnnnATT 5'
we have to write
5' TTAnnngggnnnttt 3'
This is the reverse of the complement sequence (in reference to the ORF of our genes). The sequence we've ordered (that has been shared with you) is only the forward sequence, which makes sense with regard to the NAG1, AGM1, and UAP1 open reading frames. However, this means we have to make sure that the reverseprimers are the REVERSE COMPLEMENT of the sequence we are dealing with. That is the direction the second strand is synthesized in. Another way to think of it is that the last nucleotide in the primer sequence is the one that will have nucleotides added to it by the DNA Polymerase - keeping this in mind with the 5' --> 3' synthesis, you should be set to design the primers.
Now that we all understand primer design, what primers do we need?
The primers between the genes are fairly simple. There are two gaps between the three genes. The primers will span both open reading frames, and incorporate the ribosome binding site in the middle. The 3' end will drive amplification of the desired gene, while the 5' end will facilitate gibson assembly (have I emphasized this point enough?). That means we need 4 primers between the 2 gaps (one going in each direction - don't forget REVERSE COMPLEMENT).
The primers on the very outside of the trio of genes is a special case (I will refer to these as the flankingprimers). We only need to generate primers in the direction of the gene (we don't have to amplify the plasmid), BUT. The ability to add different 5' ends means we can direct incorporation into different backbone plasmids. The bolded, black sequence in the shared IDT G-bits is homologous to the biobrick plasmid backbone. If we use this sequence, we can only gibson-assemble into the biobrick plasmid. HOWEVER, if we add sequence to the 5' end of the 5' flanking primer (draw it out if that is confusing) - we can incorporate additional features, such as the T7 promoter. ADDITIONALLY, if we use a different 5' sequence - say, one homologous to the pUC18 plasmid - we can gibson-assemble the entire trio of genes into the pUC18 plasmid (the one Acetobacter uses in just about every reference).
Whoa whoa whoa. Deciding which part of the pUC18 plasmid is a little tricky. If you've followed this email so far, the first thing to do is to get started on the biobrick-plasmid primers (since we've given those sequences to you). If anyone is physically at Genspace in the next few days, maybe they can design the pUC18-specificprimers with the help of an advisor. If you feel comfortable to give it a shot, by all means, please do.
The primers we need then:
3 primers that flank the 5' end of the gene trio:
- one that uses biobrick sequence
- one that uses biobrick sequence and incorporates the T7 promoter
- one that uses pUC18 sequence
2 primers that flank the 3' end of the gene trio:
- one that uses biobrick sequence
- one that uses pUC18 sequence
4 primers that bridge the 2 gaps between the three genes.
Thus, 9 total primers.
Now, the order of the genes probably doesn't matter much. To facilitate comparison, let's all use the order in which the genes are situated on the following sequence (I will share in a minute). Our cloning team's plan is for EACH MEMBER to take a stab at designing these primers. At the very least, design the forward primers if the whole "reverse complement" thing is confusing (which it is). The more members we have designing theprimers, the more likely our consensus sequence will be accurate. Also, the advisors shouldn't be generating all the DNA sequences - this is your opportunity to take the reins with cutting-edge molecular cloning!
Our goal is to have these sequences ordered by Monday. Please take an hour this weekend to sit down with the different sequences and design some primers. Send out the sequences to the group, and maybe add a small bit of rationale about why you chose the sequences you did (or we can discuss this at Genspace).
And, viola! Now we all can design these primer sequences! The sooner the better, so we can get them synthesized and sent out to us. I know I tend to send out long, wordy emails, so I apologize for the extreme length. There's a lot of information here; I hope you can understand and use it. If you don't understand, and have a question, by all means! Email me! I'm happy to clarify as best I can.
iGEM
The International Genetically Engineered Machine (iGEM) Foundation is dedicated to education and competition, advancement of synthetic biology, and the development of open community and collaboration.
Retrieved from " http://2012.igem.org/Team:NYU_Gallatin/Project/Cloning/Primers "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
