"
Page
Discussion
View source
History
teams
Log in
 
Team:NYU Gallatin/Notebook
From 2012.igem.org
Main menu
Home
Team
Project
Parts
Modeling
Notebook
Safety
Attributions
Profile
Aseatobacter
NYU Gallatin 2012 iGEM Team
Main menu
HomeHome of Aseatobacter.
Home
TeamThe brains of the operation.
Team
ProjectLearn more about our project.
Project
PartsOur work with the parts registry.
Parts
ModelingHow we put it all together.
Modeling
NotebookLab notebooks, news, and photos.
Notebook
SafetyOur commitment to safety.
Safety
AttributionsGive credit where credit is due.
Attributions
ProfileOfficial iGEM 2012 profile.
Profile
Take Notes
Notebook
Photos
Lab Notes
Team
Date
Note
Cloning
Sep/26/2012
Biobrick cloning:
I used the purified DNA pieces from 8.22.12 that Steven and Min purified (they’ve been at -20C so they should be okay).
Alkaline phosphatase the A-backbone:
10 uL DNA
4 uL sterile H2O
4 uL 10x buffer
2 ul alk phos enzyme
20 ul total
3’ overhangs require 15 min incubation at 37C (5’ overhangs require 60 min). SpeI leaves 3’ overhangs. Incubate 15 min at 37C...actually 45 min because the water bath isn’t up to inactivation temp.
Heat inactivate the enzyme 5 min at 65C.
Ligate:
2 uL A-pSC1C3
4 uL N
2 uL H2O
1 uL 10X ligase buffer
1 uL ligase
10 uL total
Also set up one ligation without insert (no N). Make up the volume with water so total is still 10 uL.
Incubate 1 h at RT.
Transform by heat shock transformation protocol into commercially competent DH5alpha cells. 2 transformation: one with 2 ul ligation, one with 8 ul ligation. Plate on LB-chlor plates. Incubate at 37C overnight.
During this time, we purified the DNA digested yesterday. We will ligate tomorrow using this DNA if today’s transformation is unsuccessful.
Cloning
Sep/25/2012
The following PCR conditions did not work:
Reducing the primer:template ratio 10x
Increasing the primer:template ratio 10x
Setting the annealing temp to 57C or 60C
Lowering the annealing temp to 45C for 2 cycles and 50C for the remaining 30 cycles.
The primers are probably not good.
More NAG1 and UAP1 DNA inserts were liberated with SpeI and PstI. Plasmid containing A was linearized with SpeI.
The correct pieces of DNA (circled) were cut out and put in the freezer, to be purified later
Growing
Sep/19/2012
We filled 2 more molds, 10R and unknown. Our biggest problem so far is cracks and holes; its unknown the best way to do it, as the tape might be the reason for contamination. Temp: 29.5
Cloning
Sep/19/2012
Restriction Enzyme Digest #2 (re-run):
Increasing amount of DNA from 10ul to 15ul
15ul DNA
2ul NEBuffer 2
1ul BSA
1ul EcoRI
1ul PstI
20ul Total
Reaction ran for 30+ minutes.
Gel Electrophoresis:
For loading dye, using only bromophenol blue (runs at 500bp)
Running on 7% gel: 1.05g agarose for 150ml volume TAE
2ul of 10x Bromophenol Blue loading dye
20ul of DNA
mix, total in each lane = 20ul
Set to run for 30 minutes.
Same results as yesterday.
Made 4ml cultures of new colonies: clones #16-22 are in the incubator at 11:40pm along with a neg. growth control tube..
PCR with A gene is running.
Cloning
Sep/18/2012
No growth in colonies 3, 4, 7, and negative control for growth.
Ellen did plasmid minipreps of colonies 1, 2, 5, 6, 8 - 15.
Restriction Digest (12 plasmid preps total) with EcoR1 and Pst1 was run at 37*C at 9:45pm for 30 minutes.
Master Cocktail (13):
52ul H2O
13ul PstI
13ul EcoRI-HF
26ul 10x NEBuffer 2
26ul 10x BSA
TOTAL = 130ul
10ul of cocktail mix and 10ul of DNA were combined.
RE digest products were run on a 1% gel for 40 minutes to see if an insert of 4kb comes out of the 2kb pSB1C3:
20ul DNA digest product
2ul of 10x bromophenol blue + xylene cyanol dye.
Mixed and loaded 20ul in each lane.
Lane 1: 10ul 1kb PLUS ladder
Lane 2: 20ul of colony 1
Lane 3: 20ul of colony 2
Lane 4: 20ul of colony 5
Lane 5: 20ul of colony 6
Lane 6: 20ul of colony 8
Lane 7: 20ul of colony 9
Lane 8: 20ul of colony 10
Lane 9: 20ul of colony 11
Lane 10: 20ul of colony 12
Lane 11: 20ul of colony 13
Lane 12: 20ul of colony 14
Lane 13: 20ul of colony 15
Results:
Cannot see 4kb insert in any lanes, and can see 2kb backbone in lanes 13, 14, 15.
Growing
Sep/17/2012
Two more molds were contaminated, but there are solid growths on both the tubes and most molds. We filled four but unfortunately, we ran out of substrates and the autoclave was malfunctioning. Temp in incubator lowered to 29C
Cloning
Sep/17/2012
Both plates showed colony growth, with Gibson Construct plate #1 (higher dilution) showing more growth than Gibson Construct plate #2 (lower dilution).
Negative control (E. coli no plasmid) showed no growth.
Made Chlor LB broth media:
250ml LB
184ul stock Chloramphenicol per 50ml
Picked 15 colonies and grew overnight in 4ml culture tubes
Used older LB broth in fridge. Poured one negative control for growth.
All in incubator at 37*C overnight.
Cloning
Sep/16/2012
Transformed E. coli with Gibson construct were plated onto two plates (see Ellen re: dilution)
Cloning
Sep/15/2012
PCR Purification (A, N, U)
(Invitrogen)
·    Add 4 volumes of PureLink Binding Buffer (B2) with isopropanol to 1 volume of the PCR product (50–100 μL). Mix well.
·    Add the sample with the appropriate Binding Buffer (from step 1 of this procedure) to the PureLink Spin Column.
·    Centrifuge the column at room temperature at 10,000 × g for 1 minute. (Discard the flow through)
·    Add 650 μL of Wash Buffer with ethanol to the column.
·    Centrifuge the column at room temperature at 10,000 × g for 1 minute. Discard the flow through from the collection tube and place the column into the tube.
·    Centrifuge the column at maximum speed at room temperature for 2–3 minutes to remove any residual Wash Buffer. Discard the collection tube.
·    Place the spin column in a clean 1.7-mL PureLink Elution Tube supplied with the kit.
·    Add 50 μL of Elution Buffer
·    Incubate the column at room temperature for 1 minute.
·    Centrifuge the column at maximum speed for 2 minutes.
·    The elution tube contains the purified PCR product. Remove and discard the column. The recovered elution volume is ~48 μL.
Made 1% Gel
·    Ran in a gel for ~15-20 min
N       U        A(in plasmid)    MWM
Cutting the gel and purifying
·    Equilibrate a water bath 50 C
·    Cut the gel
·    3 volumes (288ul)
·    Place it in water bath for 10min
·    Additional 5min
Purification procedure using centrifugation
·    Load the dissolved gel mixture with DNA onto the center of a pure link wash tube
·    Centrifuge the column at room temperature at 10,000 × g for 1 minute. (Discard the flow through)
·    Add 650 μL of Wash Buffer with ethanol to the column.
·    Centrifuge the column at room temperature at 10,000 × g for 1 minute. Discard the flow through from the collection tube and place the column into the tube.
·    Centrifuge the column at maximum speed at room temperature for 2–3 minutes to remove any residual Wash Buffer. Discard the collection tube.
·    Place the spin column in a clean 1.7-mL PureLink Elution Tube supplied with the kit.
·    Add 50 μL of Elution Buffer
·    Incubate the column at room temperature for 1 minute.
·    Centrifuge the column at maximum speed for 2 minutes.
·    The elution tube contains the purified PCR product. Remove and discard the column. The recovered elution volume is ~48 μL.
PCR
10ul plasmid prep a4
1ul 10X buffer
1ul Spe1
Made 1% Gel for PCR Purification for plasmid prep a4
(Also ran with Nag1 and Uap1)
First well DNA ladder (New England Bio quick-load), second well AGM1 plasmid,
Third well NAG1, and fourth well UAP1
Very little yield for linearized AGM1 plasmid but it is there. Std was 5uL, all others 10uL per lane. The smaller. The frag the more efficient the purification.
Transformation (Gibbson assembly)
Version 1                                                                             Version 2
7ul AGM1 plasmid                                                                9ul AGM1 plasmid
1ul NAG1                                                                              0.5 NAG1
2ul UAP1                                                                              0.5 UAP1
10ul                                                                                      10ul
+ 10ul master mix                                                                 + 10ul master mix
20ul total                                                                              20ul total
*PCR (version 1 and version 2) is in PCR machine
Growing
Sep/14/2012
I filled two molds and was unable to fill more since the autoclave was being used and it could not wait to autoclave more. Incubator temp 31
The lab strains look the same. Ph same as well as temp
Growing
Sep/12/2012
Lab strain looks much better and uniform from last time, a sheet is expected. Ph 4. 5
Wild strain is not growing quickly but its growth seems more concentrated. Ph 4
Temp. 23.5 degrees.
Growing
Sep/10/2012
The 10 beakers were moved to the incubator, temp. 30. Two more molds were. Contaminated and will be filled in. In all we filled 4 molds and 11 small tubes for test samples.
Kombucha-temp is 23. 5 in cabinet.
Assessment.
One of the lab strain, aceto food is growing weakly. There seems to be a particle floating inside.
Ph 4
Lab strain in blueberry juice is contaminated. Ph 2
Wild kombucha is growing best but still weakly. Ph 4. 5
Growing
Sep/07/2012
We moved the molds out of the incubator and into cabinets in the lab. Four from before were contaminated, they were cleaned and four more molds were made. 5 star shapes were also put together.
Ten smaller beakers were also inoculated with the mycelia and were originally in the fan space (fan off).
Growing
Sep/06/2012
Came into lab. Jimmy had already made the substrate, we filled out 5 of the 9 molds needed with the oak pellet substrate. To avoid contamination, we wrapped the finished molds in plastic wrap. We also used aluminum star shaped containers, in order to create smaller molds. Two and a half. All molds were made with h2o2 mycelium. No new contaminants, though one seems to be developed.
Kombucha. Made three with lab strain scoby, two with aceto food, one with blueberry juice
Growing
Sep/04/2012
Came into lab today, 7 out of 16 molds were contaminated. It seems like the tapes used to seal up cracks are the culprits as most of the mold growing seemed to originate at the tapes. There needs to be a better way to seal things up. what was done: we merely cleaned out the contaminated trays and are leaving the rest for tomorrow.
Cloning
Aug/24/2012
Ligation & Transformation
Two ligation reactions:
First rnx is Labeled “Lig 1 SH 8/24”
4 ul purified A + pSB1C3
4 ul purified N
1 ul 10x T4 ligase buffer
1 ul T4 ligase
10 ul total
The second one (that is a bit of a crapshoot but could potentially put all three pieces together):
Second rnx is Labeled “Lig 2 SH 8/24”
2 ul purified A + pSB1C3
3 ul purified N
3 ul purified U
1 ul 10x T4 ligase buffer
1 ul T4 ligase
10 ul total
Centrifuged for 20 seconds at 2.0rcf because T4 ligase was pipetted onto the side of the tube for Lig 2.
Began rnx at 7:58pm, running for 31 minutes on the timer.
Incubate at room temperature for 30 min to 1 h.
*** We should have used CIP/phosphatase so that the AGM1 gene with the pSB1C3 vector cannot religate... Since we did not, we are expecting to see more religated plasmids without the intended inserts for both Lig 1 and Lig 2 rnx’s.
Transformation set up for Ligation 1 and Ligation 2:
Pellet for 8.4rcf
Cloning
Aug/23/2012
PCR products of the N and U gene plasmid digests were retrieved and run on a 1% gel at 3:08pm for about 40 minutes.
5ul of 1KB Plus DNA ladder
2uL PCR product
2ul Xylene Cyanol loading dye
10ul H2O
14ul total
Lane 1: empty
Lane 2: 1KB PLUS DNA ladder
Lane 3: Tube A (NAG1)
Lane 4: Tube B (NAG1) + MgCl
Lane 5: Tube C (UAP1)
Lane 6: Tube D (UAP1) + MgCl
(Recall that we did not have the proper reverse primer for AGM1, so this was not included)
Predicted band sizes:
Lane 3: 747kb
Lane 4: 747kb
Lane 5: 1461kb
Lane 6: 1461kb
(with lanes 4 and 6 showing more distinct bands, if the PCR reaction was enhanced by the MgCl cofactor)
Also, running the gel for 40 minutes was way too long-- next time only 25-30 so our product doesn’t go off the gel!
Results:
PCR products (tubes A, B, C, D) are in the hot pink “Cloning Materials” box and labeled “Genes N/U RE & PCR Products 8/23”
Cloning
Aug/22/2012
We purified our PCR products via miniprep. Only then did we look more closely at the gel to see that our "products" were in the 250-400 bp range, when we expect 800-1600 bp range. Clearly the PCRs failed.
In examining the cause of PCR failure more closely, we realized that the reverse primer for gene 1 (AGM1) is incorrect. The reverse complement of the forward sequence was never generated (see the "primers" google document to confirm this). The good news is that it's easy to generate the correct sequence and order the primer. The bad news is that we can't do the final gibson assembly without it.
To troubleshoot the PCRs that should have worked (NAG1 and UAP1), we are running PCRs again overnight. We are using different concentrations of primer/template and testing different Mg2+ concentrations. This time our reactions looked like:
Forward primer (10 uM): 1 ul
Reverse primer (10 uM): 1 ul
template DNA: 1 ul
H2O: 22 ul
OR
Forward primer (10 uM): 1 ul
Reverse primer (10 uM): 1 ul
template DNA: 1 ul
100 mM MgCl: 1 ul
H2O: 21 ul
Our PCR cofactor (1% MgCl2) was:
900ul H2O
100ul 1M MgCl2
A Tube: NAG1
1ul of Forward Primer 1 (Gene 2F)
1ul of Reverse Primer 2 (Gene 2R)
1ul of Template (NAG1-5)
22ul of H2O
25ul Total
B Tube: NAG1 with MgCl cofactor
1ul of Forward Primer (Gene 2F)
1ul of Reverse Primer (Gene 2R)
1ul of Template (NAG1-5)
1ul MgCl solution
21ul of H2O
25ul Total
C Tube: UAP1
1ul of Forward Primer 1 (Gene 3F)
1ul of Reverse Primer 2 (Gene 3R)
1ul of Template (UAP1-3)
22ul of H2O
25ul Total
D Tube: UAP1 with MgCl cofactor
1ul of Forward Primer 1 (Gene 3F)
1ul of Reverse Primer 2 (Gene 3R)
1ul of Template (NAG1-5)
1ul MgCl solution
21ul of H2O
25ul Total
We also decided to start cloning the classic digest-and-ligate way using our genes which are already in the biobrick plasmids. We digested AGM1 plasmid with SpeI (thus linearizing it). We digested NAG1 and UAP1 plasmids with SpeI and XbaI, liberating the genes (and RBS) from their backbone plasmids.
AGM Plasmid:
3ul AGM1 Plasmid
2ul Buffer
1ul SpeI
0ul XbaI
14ul H2O
20ul Total
NAG Plasmid:
3ul NAG1 Plasmid
2ul Buffer
1ul SpeI
1ul XbaI
13ul H2O
20ul Total
UAP Plasmids:
3ul UAP Plasmid
2ul Buffer
1ul SpeI
1ul XbaI
13ul H2O
20ul Total
We ran a gel of our digests, and they look great - the exact sizes we expected (I will attach it later). Another confirmation that our minipreps contain the genes we expect them to (even if we don't have the full sequence). We cut the pieces out of the gel that were the correct sizes and gel purified them. They are in the fridge and ready to ligate together.
Cloning
Aug/21/2012
We ran a gel to confirm that we have PCR products of expected sizes. I glanced only quickly at the gel, saw that there were some bands (admittedly diffuse but the gel itself was a week old). The gel is attached (sad gel.png). Its name will become clear tomorrow.
Cloning
Aug/20/2012
We set up PCRs of the genes using primers containing overlapping sequences. For each PCR, we diluted the dessicated reagents (Taq Polymerase, dNTPs, MgCl, buffer) in the following:
Forward primer (10 uM): 2 ul
Reverse primer (10 uM): 2 ul
template DNA: 3 ul
H2O: 18 ul
(I may be off. I didn't take notes here. If I'm wrong and you remember, please let me know.)
Cloning
Aug/17/2012
The DNA sequences came back. None of the forward reactions worked (especially surprising as these primers were supplied by iGEM headquarters). Of the reverse reactions, some worked and some didn't. There was no read for the NAG1 gene, for example, but as we only had one digest that looked positive, we have to move forward with this clone (for now).
The best hits were:
AGM1 - clone 4
NAG1 - clone 5
UAP1 - clone 3
Cloning
Aug/16/2012
The cultures were well grown the next day (and the original colonies were still white, a promising sign). We miniprepped the plasmids from the cultures, and then used an XbaI/SpeI digest to identify plasmids with the correct-sized inserts. The expected sizes for the DNA fragments:
1.6 kB - AGM1
0.7 kB - NAG1
1.5 kB - UAP1
2.1 kB - biobrick plasmid
All of the positive hits had bands of these expected sizes. Some of them had larger bands that add to a singly-cut plasmid (insert + backbone, e.g., AGM1 + biobrick plasmid, i.e., 1.6 kB + 2.1 kB = 3.7 kB) but the ones we considered positive contained only predicted sized bands - no miscellaneous weirdos.
The digests were run on a 1% agarose gel. I've attached the picture (gel annotated.png) that shows the inserts with the correct sized bands (the samples with the asterisks). These were sent for sequencing using the VF and VR (verify forward and verify reverse) primers.
Cloning
Aug/15/2012
The next day we had lots of colonies (I don't have the pictures of the plates, but maybe someone else does). Many of these were red, indicating they were transformed due to residual RFP plasmid (as had happened previously). We circled several white colonies as prospective positive transformants. For each construct, we chose 5 clones and inoculated 4 ml of LB-chlor to grow these overnight at 37 deg C.
Cloning
Aug/14/2012
The second gibson assemblies that we did worked! We followed Ellen's advice to increase the G-bit DNA concentration. This means our final assembly reactions looked like this:
2.5 ul pSB1C3 (biobrick backbone plasmid DNA, linearized)
7.5 ul G-bit DNA*
10  ul master mix
20  ul total
* we divided the volume left by the number of G-bit parts. For NAG1, there were 3 parts, so each took up 2.5 ul. For UAP 1, there were 4 parts, so each took up 1.8 ul. For AGM1, there were 5 parts, so each took up 1.5 ul.
We incubated these for 1 hr at 50 deg C and then immediately transformed the commercial competent cells.
Cloning
Aug/12/2012
Gibson, Transformation:
So instead of 50 fentomoles of each, try 100 but 50 of the plasmid as before. I think we have room in the 10 microliters for that. And maybe close the lid of the PCR machine?
Gibson assembly protocol
-reaction is in 0.2mL PCR tube using program “Gibson” under user “ellen”
50C for 1h, hold at 4C
-total reaction volume is 20 microliters
-all G-Blocks were resuspended at a concentration of 50 fentomoles per microliter
-TOTAL amount of DNA in tube (sum of all parts) should be between 200-1000 fentomoles. NOTE: last time we were at the low end of this!
Recommended procedure:
For ‘U’
1.85 µL U-1
1.85 µL U-2
1.85 µL U-3
1.85 µL U-4
2.6 µL pSB1C3
10.0 µL Gibson Master Mix
20.0 µL
For ’N’
2.5 µL N-1
2.5 µL N-2
2.5 µL N-3
2.5 µL pSB1C3
10.0 µL Gibson Master Mix
20.0 µL
Transformation
Aug/10/2012
Cloning
Aug/10/2012
Results of Gibson Assembly:
4 big plates of the 3 chitin path genes A, U, and N, a + control
2 small plaets: neg control, baseline control without antibio
Results:
Lawn on antibiotic free control
Some contamination on A & U plates
No growth on N, + Control plates
[PASTE ALL PHOTOS FROM ALEX EMAIL AFTER RESIZING]
Pink colonies may be traces of RFP plasmids remaining from when biobrick plasmid part was PCR’ed.
Reasons for failure could be:
1) use the heated lid on the PCR machine - I did not shut it because I thought it might jack up the temp.
2) the manual says to use a 2-3x overage of inserts to backbone. The tech service guy said use equimolar so we did. maybe we need to rethink that.
3) overall total conc of DNA in the rxn mix could be upped.
Cloning
Aug/08/2012
Notes for  NAG1 Gibson
N-1     87163240
163696916       Date Ordered: 3rd Aug, 2012
200 ng = 1012.2 fmol
N-2     87163241
163696911       Date Ordered: 3rd Aug, 2012
200 ng - 1005.9 fmol
N-3      87163242
163696921       Date ordered: 3rd Aug, 2012
200 ng = 978.5 fmol
Gibson Assembly Master Mix
#E26115 Exp: 6/13
Lot: 0031206
PSBIC3
Biobrick Plasmid 2071 bases
650 * 2071 =
Mol. Weight: 1.346 x 106 fg/fmol
N1
10012.2 fmol / 20 μL
= 50.61 fmol/ μL
N2
1005.9 fmol / 20 μL
= 50.29 fmol/ μL
N3
978.5 fmol / 19 μL
= 51.50 fmol/ μL
4.3 μL water
+ 2.7 μL PSB1C3
+ 1.0 μL N1 (Concentration - 50.61 fmol/μL)
+ 1.0 μL N2 (Concentration - 50.29 fmol/μL)
+ 1.0 μL N3 (Concentration - 51.50 fmol/μL)
+ 10 μL Mastermix
= 20 μL
Aug/08/2012
Transformation
Aug/07/2012
Cloning
Aug/06/2012
G-blocks arrived
Transformation
Aug/04/2012
Transformation
Aug/03/2012
Transformation
Aug/02/2012
Transformation
Aug/01/2012
Transformation
Jul/30/2012
Transformation
Jul/20/2012
Growing
Jun/09/2012
Almost no growth, very thin film.
Cellulose very thin, seems to have grown on bottom (not top)
Thin growth, not measurable from sides, but def more significant than just scoby.
Super thick from bubbles, accidentally popped it.  A little thicker than #9.
Infested w/ flies.  Nice growth, not in a sheet, disposing.
Pretty thick, made a little pillow from the air bubbles.
Thin growth along top.
Thin growth along top.
Nice sheet, 2nd to 10+13.
Very thick cellulose, abt 1-2mm growth.
Thickest non-BB.  Weird flakes (brown) everywhere.
Thin growth, a little thicker than the other thins.
Almost entirely cellulose, very little liquid left.
iGEM
The International Genetically Engineered Machine (iGEM) Foundation is dedicated to education and competition, advancement of synthetic biology, and the development of open community and collaboration.
Retrieved from " http://2012.igem.org/Team:NYU_Gallatin/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
