"
Page
Discussion
View source
History
teams
Log in
 
Team:NYU Gallatin/Project/Cloning
From 2012.igem.org
Main menu
Home
Team
Project
Parts
Modeling
Notebook
Safety
Attributions
Profile
Aseatobacter
NYU Gallatin 2012 iGEM Team
Main menu
HomeHome of Aseatobacter.
Home
TeamThe brains of the operation.
Team
ProjectLearn more about our project.
Project
PartsOur work with the parts registry.
Parts
ModelingHow we put it all together.
Modeling
NotebookLab notebooks, news, and photos.
Notebook
SafetyOur commitment to safety.
Safety
AttributionsGive credit where credit is due.
Attributions
ProfileOfficial iGEM 2012 profile.
Profile
The Project
Project
Cloning
Transforming
Characterizing
Growing
Designing
Socializing
Cloning
Project
Designer Suzanne Lee showed that bacterial cellulose could be used in novel ways, including to make clothing using a more eco-friendly process. Our team set out to demonstrate that new characteristics could be added to the cellulose produced by Acetobacter xylinum using synthetic biology methods. Altering physical characteristics such as strength, color, odor, etc. would result in exciting new materials to create with.
Cellulose is a polymer made up of long Î²-1,4 glucan chains. In Acetobacter, the enzyme cellulose synthase catalyzes the biosynthesis of cellulose from UDP-glucose. We hypothesized that mixed polymers containing different sugars would have unique physical properties.  Cellulose synthase has been reported to utilize UDP-N-acetylglucosamine (NAG) molecules (components of chitin) as well, resulting in a polymer containing both glucose and NAG units- a cellulose-chitin hybrid. We wanted to test the properties of this new material, so we set out to engineer Acetobacter to produce this hybrid polymer.
Strategy
The yeast Candida albicans produces chitin and uses it in spore formation. To incorporate NAG into Acetobacter cellulose, we focused on three yeast genes (the pathway consisting of NAG5 (GlcNac kinase) catalyzes the conversion to GlcNac-6P, AGM1 (Phosphoacetyl-glucosamine mutase)  which converts GlcNac-6-P to GlcNac-1-P, and UAP1 (UDP-GlcNac pyrophosphorylase) which adds UDP.
The most efficient method was to use Gibson assembly to piece together each gene from small, ovelapping fragments of between 300 and 500bp ("G-blocks") and clone it into pSB1C3 separately. We could not construct the entire pathway with G-blocks because it was too large, and there is a limit of six fragments for efficient Gibson assembly. The coding sequences were modified to reflect codon usage in Acetobacter, and an Acetobacter RBS was added in front of each gene. The G-blocks that we had synthesized for each gene are shown below:
G-blocks for AGM1
GAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTGAGGAGGATGAACGACGCATGTCAATTGAACAAACATTATCACAATATTTACCATCACATCCAAAACCACAAGGTGTGACATTTACTTATGGGACAGCAGGATTCCGTATGAAAGCTGATAAATTAGATTATGTCACTTTTACCGTTGGGATCATTGCTTCATTAAGATCGAAATATTTACAAGGGAAAACCGTTGGTGTTATGATTACTGCTTCTCATAATCCCCCGGAAGATAATGGGGTTAAAGTTGTTGATCCATTAGGTAGTATGTTGGAAAGTTCATGGGAAAAATATGCTACTGATTTAGCCAATGCTTCTCCTTCTCCTTCTA   402
TATGCTACTGATTTAGCCAATGCTTCTCCTTCTCCTTCTAACGATTCAGAAGGGGAAAAAAATCTGTTGGTGGAAGTTATTAAAAATTTGGTTTCTGATTTGAAAATTGATTTATCTATTCCTGCTAATGTTGTTATTGCTAGGGATTCAAGAGAATCTAGTCCAGCATTATCAATGGCAACTATTGATGGATTTCAAAGTGTTCCCAACACTAAATATCAAGATTTTGGATTATTTACTACCCCAGAATTACATTATGTTACTAGAACATTAAACGATCCCGATTTTGGTAAACCAACTGAAGATGGTTATTATTCTAAATTAGCAAAATCTTTCCAAGAAATTTATACCATTTGTGAATCTAATAATGAAAAAATCGATATAACTATTGATGCTGCTAATGGTGTTGGAGCCCCCAAA  420
TATAACTATTGATGCTGCTAATGGTGTTGGAGCCCCCAAAATTCAAGAATTATTAGAAAAATATTTACATAAAGAAATCAGTTTTACCGTGGTTAACGGTGATTATAAACAACCAAATTTATTAAATTTTGATTGTGGAGCTGATTATGTCAAGACTAATCAAAAATTACCTAAAAATGTCAAACCAGTAAATAATAAATTATATGCTTCATTTGATGGCGATGCGGATAGATTAATATGTTATTATCAAAACAATGATAATAAATTCAAATTATTAGATGGTGATAAATTATCGACGTTATTTGCGTTATTTTTACAACAATTATTTAAACAAATTGACCCCACTAAGATTTCATTGAATATTGGTGTGGTTCAAACTGC  381
CCCACTAAGATTTCATTGAATATTGGTGTGGTTCAAACTGCTTATGCTAATGGATCTTCAACAAAATATGTTGAAGATGTTTTGAAAATTCCTGTTCGTTGTACTCCTACTGGTGTTAAACATTTACATCATGAAGCTGAAAATTTCGATATTGGTGTATATTTTGAAGCTAATGGGCATGGTACAGTTATTTTCAATCCTGAAGCCGAAAAGAAAATTTTCAATTATAAACCAAATAATGATAATGAAGCTAAAGCTATTAAAGTTTTACAAAATTTTAGTCAATTAATTAATCAAACTGTGGGTGATGCAATTTCCGATTTATTGGCCGTGTTAATTGTCGTTCATTATTTGAAATTATCACCAAGTGATTGGGATAATGAATATACTGA  392
TTGAAATTATCACCAAGTGATTGGGATAATGAATATACTGATTTACCTAATAAATTGGTTAAAGTGATTGTTCCTGATAGATCTATATTCAAAACTACAAATGCTGAAAGAACTTTGGTTGAACCTAAAGGTATGCAAGATGAAATTGATAAATTAGTTGCCCAATATCCAAATGGAAGATCTTTTGTAAGAGCTTCTGGTACTGAAGATGCTGTTAGAGTTTATGCTGAAGCTGATACGCAAAATAACGTTGAAGAATTATCTAAAGCAGTATCTGAATTAGTTAAATAGTACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCC 348
G-blocks for NAG5
GAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTGAGGAGGATGAACGACGCATGACTGAGA CTAGCATTAG TGGGTTGCGT GGCCCCAAGT CCATGTATTT TATGGAGATTGTTGATGTGT CGTCGCAAGA ATCCAGTGTG TTGTCGAGTA TTGTTGAATC GTTTACGTCTGCGGTATCTG CATCGAACTT GGGGGTATAC TCTGATGAAG TGCTTTGTGA TATCAAACTGTCGTTGAAAG AGAAATCCCC AATTACTATG TTACCTAACT ATAATGTCTC CCCTACAGGCGACGAGCATG GACAGTATTT GGTTATTGAT TTAGGAGGGT
CCCTACAGGCGACGAGCATG GACAGTATTT GGTTATTGAT TTAGGAGGGT CGACATTGAG AATAGCAGTGGTAGACATTT CCAAGCCACA CCCAAATTTG TCGAGAAGTG AGAGGATAAC TATAGTTGTGGAAAAGAGTT GGATTATTGG CAACGACTTT AAGAGGATTG ATGGTGAGTT TTTCAAGTATATTGGGTCCA AGATTAACGA GATATTGATG GGACAGAATG TTATTGATGT AAAGTCGGTT ATCAATACTG GTATAACCTG GTCGTTCCCG TTGGAAACAA CCGACTACAA TAGAGGCAAGATCAAGCATG TTTCCAAAGG GTATACTGTG GGAGAGG
CAA TAGAGGCAAGATCAAGCATG TTTCCAAAGG GTATACTGTG GGAGAGGATA TTTACGACAA GGATTTAAAGATGGTTTTGG AAGACACGTT GAGACAAGAG TACGGGTTGA CACTTGACGT GCAGTCTATATTGAACGACT CGTTGGCGGT GTATTCTGCT GGGTGCTTTA TTGATTCGAA GATGAAGTTGGCCATGGTGT TGGGGACAGG GATTAACATG TGCTGTTCCC TAAAAAGGTC AAGTGATATCCACCCCTCCA AGATGTTGGC TGATGCTACG TTGTTTAATT GTGAGCTCTC GTTGTTTGGCCAGAATCTAT GTAAAGACTT TGCTACGAAA TATGATATCA TTATAGACAA
CAGAATCTAT GTAAAGACTT TGCTACGAAA TATGATATCA TTATAGACAA GAGGTTTGCTGGTTTGCTGC ACCACTTTAA GACGTTTATG GAGCCTGACC CTATTACAAA AACGCTTTTCCAGCCGCACG AGTTGATGAC CAGTGGGAGG TACTTGCCAG AATTGACGAG TTGGTGGTGGTAGATTTGA TTGAGGCTGG CGAGATTTTT CAAAATGTTG ACCACCAACA GATGTACCAAGAGTATGGTG GGTTTAGTGG GGAGTTGATT TGTTTTGTGC ATGAGAATGA CGATTATGATGATATACATG ACAAGTTGTG CAAGGCCTAC GGCTGGACGA CGGTTGGGTTGAGTGACATTGTCTGCTTGA AAGAAGTTGT ATCGTGCATT ATCAAGCGGG
GAGTGACATTGTCTGCTTGA AAGAAGTTGT ATCGTGCATT ATCAAGCGGG CGGCGTTTAT TGTTGCCAATGCGATTATTG CGTTTTTCAA ATTGTTGGGC AGTGACGAGT TGGGTGGTGA TGTGACGATTGGGTATGTGG GGTCGGTCTT GAACTACTTT CACAAGTATA GACGGTTGAT TGTTGAGTATGTGAATAGCG CAGAGGAGGC CAAGGGGATA AAGGTTGACT TGAAGTTGAT TGAAAATAGCTCGATTATAG GTGCTGCCAT AGGTGCTGCC TATCATAAG TAGTACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACC
G-blocks for UAP1
GAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTGAGGAGGATGAACGACGCATGACAGTTAAATCACAACAACAAATTATTGATTCATTCAAACAAGCTAATCAAGATCAACTTTTCCAATATTATGATTCATTGACAATAAATCAACAACAAGAATTTATAGATCAATTGTCAACTATTGAAGAACCAGCTAAATTGATTTCTACTGTAGAACAAGCGATTCAATTTTCTCAAACCAATTCTACATCAAGAAATTTCACTCAATTACCTAATGAACAAACAGCATCAACTTTAGATTTATCAAAAGACATTTTACAAAATTGGACCGAATTAGGTTTAAAAGCCATTGGTAATGGAGAAGTTGCCGTTTTATTGATGGCAGGAGGTCAAGGAAC  433
GAGAAGTTGCCGTTTTATTGATGGCAGGAGGTCAAGGAACTAGATTAGGTTCAAGTGCTCCCAAAGGTTGTTTTAATATTGAATTACCATCACAAAAATCATTATTTCAAATTCAAGCTGAAAAAATTTTGAAAATTGAACAATTAGCTCAACAATATTTGAAATCGACTGAAAAACCAATTATTAATTGGTATATTATGACCAGTGGTCCTACTAGAAATGCTACTGAATCATTTTTCATTGAAAATAATTATTTTGGTTTAAATTCTCATCAAGTGATTTTTTTCAATCAAGGAACGTTGCCATGTTTTAATTTACAAGGCAATAAAATCTTATTAGAACTGAAAAATTCAATTTGTCAATCACCCGATGGTAATGGTGGATTATATAAGGC  384
TTTGTCAATCACCCGATGGTAATGGTGGATTATATAAGGCATTAAAAGATAATGGAATACTAGATGATTTCAATTCTAAGGGCATCAAACATATTCATATGTATTGTGTTGATAATTGTTTAGTTAAAGTTGCTGATCCAATTTTCATTGGATTTGCCATTGCCAAAAAATTTGATTTGGCAACAAAAGTGGTTAGAAAAAGAGACGCTAATGAAAGTGTTGGATTAATTGTTTTAGATCAAGATAATCAAAAACCTTGTGTTATTGAATATAGTGAAATTTCTCAAGAATTGGCTAACAAAAAAGACCCTCAAGATTCTTCTAAATTATTTTTAAGAGCTGCTAATATTGTTAATCATTATTATTCAGTGGAATTTTTAAATAAAATGATTCCTAAATGGATTTCATCTCAAAAATATTTACCATTCC 429
ATTCCTAAATGATTTCATCTCAAAAATATTTACCATTCCATATAGCTAAAAAGAAAATCCCAAGTTTGAATTTAGAAAATGGAGAATTTTATAAACCAACTGAACCAAATGGTATTAAATTAGAACAATTCATTTTCGATGTTTTCCCATCAGTCGAATTAAATAAATTTGGTTGTTTAGAAGTCGATCGTTTAGATGAATTTTCTCCATTGAAAAACGCCGATGGTGCTAAAAATGATACTCCAACAACTTGTAGAAATCATTACCTTGAAAGAAGTTCCAAATGGGTTATTCAAAATGGTGGAGTTATTGATAATCAAGGATTAGTTGAAGTTGATAGTAAAACCAGTTATGGTGGTGAAGGTTTAGAATTTGTTAATGGTAAACATTTCAAAAATGGCGATATTATTTAATACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCC  470
This worked very well, and resulted in the submission of three new parts to the BioBrick Library (Bba_K850000, Bba_K850001, Bba_K850002). We then set out to fuse these three genes into a single pSB1C3-based plasmid. PCR primers were synthesized to bracket each of the three genes and also add sequernce that would facilitate Gibson assembly of the entire pathway. Our reasoning was that we could then use Gibson assembly to piece together the three genes in the pathway together into pSB1C3. The sequences of these primers is shown below. We included two forward primers dfor the AGM1 gene- one that included the T7 promoter (known to work in Acetobacter, whose own promoters are not clearly understood yet), and one that did not.
Â 
Primers
Forward AGM1 (biobrick plasmid + promoter
CAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTAATACGACTCACTATAGGGAATACAAGCTACTTGTTCTTTTTGCATGAGGAGGATGAACGACGCATG
Reverse AGM1
GCAGTATCTGAATTAGTTAAATAGTGAGGAGGATGAACGACGCATGAGACAAGCTATATTTTCCAACCCTAACGATGCTGCGCAGCATCGTTAGGGTTGGAAAATATAGCTTGTCTCATGCGTCGTTCATCCTCCTCACTATTTAACTAATTCAGATACTGC
Forward NAG5
CGTTGAAGAATTATCTAAAGCAGTATCTGAATTAGTTAAATAGTGAGGAGGATGAACGACGCATGAGACAAGCTATATTTTCCAACCC
Reverse NAG5
GCTGGATTAAAGTCAAAGTTGTAGTGAGGAGGATGAACGACGCATGACAGTTAAATCACAACAACAAATTATTGATTCATTCATGAATGAATCAATAATTTGTTGTTGTGATTTAACTGTCATGCGTCGTTCATCCTCCTCACTACAACTTTGACTTTAATCCAGC
Forward UAP1
GTTTGCGATAACGCTGCCGCTGGATTAAAGTCAAAGTTGTAGTGAGGAGGATGAACGACGCATGACAGTTAAATCACAAC
Reverse UAP1 (biobrick plasmid)
CAAAAATGGCGATATTATTTAATACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGCGGCCGCTACTAGTATTAAATAATATCGCCATTTTTG
The primers for the NAG5 and UAP1 genes worked well, resulting in PCR products of the expected size. However, the AGM1 primers did not result in a PCR product even when used under a variety of conditions. We then altered our cloning strategy and attempeted to use traditional BioBrick assembly. The plasmid containing AGM1 (Bba_K850000) was digested with Spe1 and Pst1, and the product gel-purified and treated with Antarctic phosphatase. The NAG5 gene was cut out of the Bba_K850001 plasmid using Xba1 and Pst1 and gel-purified. These two pieces of DNA were ligated together to produce a plasmid containing the AGM1 and Xba1 parts of the pathway in a BioBrick format. This construct was then cut with Spe1 and Pst1, and the process repeated with the UDP1 gene cut from the Bba_K850002 BioBrick plasmid with Xba1 and Pst1.
This resulted in a plasmid containing the entire pathway (AGM1, NAG5 and UDP1) but no promoter. To test the pathway in Acetobacter, we cut it out of the BioBrick vector using EcoR1 and Pst1 and cloned it into the MCS of pUC18 which has a T7 promoter.
Learn how to design primers from Julie Wolf.
Protocols
Here are some protocols we used in our cloning process.
Transformation Protocol
Gibson Assembly
Chitin Synthase Mutagenesis
Amplification of CSH3 gene encoding chitin synthase from S. cerevisiae
We are also setting out to clone the chitin synthase of S. cereivisiae that was originally submitted as a part but is currently not BioBrick compatible (Part:BBa_K418007).
PCR primers were designed for amplifying CSH3 from colonies of S. cerevisiae. Primers had BioBrick prefix and suffix ends. The suffix end also had a terminal XhoI site for cloning into the pET28-b+ expression vector for His tagged.
Forward-CSH3 (BioBrick Prefix)
GTT TCT TCG AAT TCG CGG CCG CTT CTA GAT GAC CGG CTT GAA TGG AG
Reverse-CSH3 (BioBrick Suffix)
GTT TCT TCC TCG AGC TGC AGC GGC CGC TAC TAG TAT TAC TAT GCA ACG AAG GAG TCA C
As this gene has 2 illegal Xba1 sites and and illegal Pst1 site, 3 sets of primers will be designed to introduce point mutations in order to remove them, to ensure BioBrick compatibility. For this strategy we will use a PCR based mutagenesis system.
Illegal XbaI sites and PstI site in CSH3 as determined using NEBcutter:
Cut pos. MS Enzyme               Recognition sequence -------- -- -------------------- --------------------     1117    XbaI                  T^CTAG_A     1674    XbaI                 T^CTAG_A     2322    PstI                    C_TGCA^G
iGEM
The International Genetically Engineered Machine (iGEM) Foundation is dedicated to education and competition, advancement of synthetic biology, and the development of open community and collaboration.
Retrieved from " http://2012.igem.org/Team:NYU_Gallatin/Project/Cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
