<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Sheffield_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Sheffield/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Sheffield/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Sheffield/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Sheffield/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Sheffield/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Sheffield/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="buttons" align="center" style="border:solid #000"><TITLE>Template:Navigation</TITLE><P><STRONG>Project overview </STRONG></P><P><STRONG>iColi</STRONG>: </P><P>A multi-pathogen detecting biosensor. Quick, visible and simple. The water industry currently complies with statutory testing requirements, a random testing of water at a number of locations throughout the distribution network from water treatment plants to customers’ homes. This is a slow and intensive process requiring access to a laboratory. Even if a pathogen is detected this is usually too late to stop the pathogen reaching customers’ homes and causing potential infections. iColi will be a solution to this problem allowing water companies to make operation changes to prevent a pathogen reaching customers’ taps. By detecting a multitude of pathogens in a matter of minutes, the testing process becomes more streamlined and allows action to be taken following the discovery of a pathogen in the water. Currently this would best be deployed in service reservoirs where water is stored for approximately 48 hours before being piped to local homes. This allows ample time for samples to be taken, tested with iColi and appropriate action taken. To solve this problem we tried to establish a common link between pathogens which we could exploit to create a biosensor for a variety of waterborne pathogens. This led us to investigate the use of quorum sensing, the communication method of bacteria themselves. To test this principle we settled on a well characterised pathogenic bacteria Vibrio cholarae. Although not a problem in western water supplies, if trials are successful there may be a use for the detection in the developing world to detect cholera in water supplies before they are used. Two different methods were established to attempt to create a biosensor. These will be referred to as the chimeric protein method and the cholera system method. Both rely on the same principles: a quorum sensing molecule activating a pathway which in turn activates a promoter which will be linked to a reporter molecule initially GFP but in final iColi eChromi will be used to give a variety of visible results. Chimeric protein method The cholera quorum sensing molecule CAI-1 is detected by the receptor CqsS, which initiates a phosphorylation cascade through a number of other proteins. This protein has some homology with the E.coli stress response receptor BarA which also initiates a series of phosphorlyations, ultimately phosphorylating the response regulator UvrY. By combining the receptor domain of CqsS with the phosphorylation domain of BarA it was hoped that the E.coli’s natural stress response pathway could be hijacked. In addition if a reporter- GFP is placed downstream of the stress response pga promoter, it was hoped that GFP expression could be induced by the binding of the quorum sensing molecule CAi-1. Cholera system method As previously stated the cholera quorum sensing molecule CAI-1 is detected by CqsS initiating a phosphorylation cascade of proteins LuxO and Lux U which in turn activate the promoter sequences HapR and CsrB. Rather than hijack the E.coli stress response pathway, the whole cholera quorum detection pathway was inserted into E.coli on a plasmid in addition to the promoter sequence for HapR or CsrB with a downstream GFP reporter.      </P><P><STRONG>Promoters</STRONG></P><P>In the E.coli genome, promoters are sequences of DNA upstream of genes, required for the expression of those genes. RNA polymerase binds and initiates RNA synthesis at the promoters. By attatching a GFP downstream of a promoter, one can cause the expression of GFP in the place of the protein. This can be used as a reporter of gene activity. In E.coli, the promoter consist of two consensus sequences; one at -10 base pairs upstream of the transcription start site, consisting of a 5’-TATAAT-3’ sequence, and one at -35 base pairs consisting of a 5’-TTGACA-3’ sequence. The relative efficiency of promoters directly correlates to their similarity to the consensus sequence. A transcription initiation factor called a sigma factor aids the binding of RNA polymerase to the promoter in order to initiate transcription. The merits of various reporters were discussed for this project, for example lacZ, E.chromi and GFP. Ideally, the final product would have used E.chromi as a reporter as it is visible to the naked eye, a trait required for a good ‘in-field’ biosensor. Unfortunately, given the resources and time-frame of the project, it was decided that GFP would be an appropriate reporter to use. Promoters were designed for various components of the signalling pathways of both the barA and the cholera quorum sensing pathway. The promoters chosen and designed were for; -HapR- From the vibrio cholera system. It is a transcription factor that represses the virulence of vibrio cholera. When high cell density is detected it represses gene expression of AphA, a virulence gene activator, by binding to its promoter. It also displays negative feedback by binding to its own promoter to repress its own transcription. The promoter is located between +8 and +36 from the transcriptional start site. HapR is a downstream gene endigenous to vibrio cholera and can be used to report the activity of our ‘system design’ by attaching a GFP downstream of its promoter. -CsrB- Csr, a global regulatory system endigenous to E.coli, controls bacterial gene expression. CsrA is an RNA-binding protein that represses biofilm formation. CsrB is a 360 base non-coding RNA molecule that sequesters CsrA, activating the process of biofilm formation. -PgaA- This is another gene in the native barA pathway. The pgaABCD operon is needed for the synthesis of the adhesion PGA polysaccharide of E.coli required for biofilm formation. pgaA is downstream of CsrA/B in the barA pathway.</P><P><STRONG> Method </STRONG></P><P>These promoters were transformed in their plasmids into the DH5α strain of E.coli. The mini-prep technique was performed to extract the DNA. At the same time, a plasmid containing a GFP biobrick was also transformed and a mini-prep performed. A restriction digest was performed to excise the promoter genes from their plasmids and to cut the plasmid containing the GFP in a position just upstream of the GFP start site. Gel electrophoresis was used to ensure the digest was successful and then the fragments were retrieved by gel extraction. Each promoter was then ligated into a cut plasmid containing GFP, directly upstream of the GFP. These ligated plasmids could then be transformed into competent DH5α. The DNA was extracted by mini prep and another two restriction digests were performed; one to excise the promoter-GFP construct from its plasmid, and one to cut the iGEM, chloramphenicol-resistant, plasmid at the appropriate site. These were, again, run on a gel to check for accuracy, and then retrieved by gel extraction. The promoter-GFP construct was then ligated into the iGEM plasmid and transformed. The DNA for these biobricks was then obtained by mini-prep. Characterisation- 5ml cultures of LB with 5µl chloramphenicol were inoculated with DH5α containing the promoter-GFP constructs. Cultures were grown overnight at 37˚C, to saturation, then diluted 1:500 into 50ml LB/chloramphenicol at the beginning of the time course. Samples were taken from the overnight cultures (2ml), at the time of dilution (10ml), and every 45 minutes thereafter for 6hours. The samples were spun down and re-suspended in 1ml of PBS and stored on ice until all samples were collected. The samples were then put into the flow cytometer for analysis.      </P><P><STRONG>Results </STRONG></P><P> The image above shows a cell expressig GFP after induction with CAI-1, showing that our project may well be feasible. </P><STRONG>References </STRONG><P>CsrA post-translationally represses pgaABCD, responsible for synthesis of a biofilm polysaccharide adhesion of E.coli Xin Wang et al, Molecular Microbiology (2005) 56 (6), 1648-1663.      </P><P>The pgaABCD locus of E.coli promotes the synthesis of a polysaccharide adhesion required for biofilm formation’- Xin Wang et al, Journal of Bacteriology, May 2004, p2724-2734.      </P><P>Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB’- Tony Romeo, Molecular Microbiology (1998) 29 (6), 1321-1330. 
      
    www.conncoll.edu/ccacad/zimmer/GFP-ww/prasher.html </P><P><STRONG> Acknowledgements </STRONG></P>

The Sheffield iGEM team would like to thank all of their advisors for the advice given. The EPSRC for the funding and all work described above was performed by the team. 


</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Sheffield/Project">http://2010.igem.org/Team:Sheffield/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Sheffield/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Sheffield/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Sheffield/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Sheffield/Project&amp;oldid=201799" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>