Team:SCUT-FSE-CHINA/Protocol
Toggle navigation
HOME
PROJECT
Description
|  Background
|  Design
Construction
RESULTS
|  N&P Pathways
|  CRISPR/Cas9
|  Robust System
MODELING
|  Growth Model
|  Metabolic Model
|  Scale Up
HUMAN PRACTICE
Silver
Gold
Collaboration
INTERLAB
TEAM
Team
Attributions
PARTS
LAB
Lab Notes
Protocol
Safety
MEDAL
|  Bronze Criteria
|  Silver Criteria
|  Gold Criteria
Protocol
Part 1 N&P Pathway
Genome extraction
Materials:
Paenibacillus strain;
Klebsiella pneumonia strain;
E.Z.N.A. ®Bacterial DNA kit;
Methods:
Pick the strain from the slant medium using a vaccination loop to 20 ml of LB medium and shake for 1 day.
Harvest 1.5ml culture by centrifugation at 4000×g for 10 min at room temperature.
Remove medium completely and resuspend cells in 180 μl TE buffer. Add 20 μl of 50 mg/ml lysozyme solution. Incubate at 30℃ for 10 min.
Note: The amount of enzyme required and/or the incubation time may need to be modifies depending on the bacterial strain used. Complete digestion of the cell wall is essential for efficient lysis. Longer incubation time might yield better results.
Pellet digested cell by centrifugation at 5000×g for 5 min at room temperature. Discard the supernatant and add 200 μl Buffer BTL. Vortex to resuspend cells.
Optional: For complete digestion of bacterial cell wall especially of Gram-Positive bacteria, add 25-40 mg Glass Powder and vortex at maxi speed for 5 minutes. Let it stand to allow the beads to settle. Transfer supernatant to a new 1.5 ml centrifuge tube(nut supplied).
Add 25 μl Proteinase K solution and vortex to mix well. Incubate at 55℃ in a shaking water bath to effect complete lysis. Usually no more than 1 h is required for bacterial lysis. If no shaking water bath is available, incubate and shake or briefly vortex the samples every 20-30 minutes.
Add 5 μl RNase A to samples and invert tube several times to mix. Incubate at room temperature for 10-30 minutes.
(Optional) Centrifuge at 10000×g for 5 min to pellet insoluble debris. Carefully aspirate the supernatant and transfer to a sterile micro-centrifuge tube leaving behind any insoluble pellet.
Add 220 μl Buffer BDL and shake or briefly vortex to mix. Incubate at 65℃ for 10 minutes. A wispy precipitate may form upon addition of Buffer BDL; it does not interfere with DNA recovery.
Add 220 μl absolute ethanol (room temperature, 96-100%) and mix thoroughly by vortexing at maxi speed for 20 seconds. If any precipitation can be seen at this point, break the precipitation by pipetting up and down 10 times.
Assemble a HiBind DNA Mini column in a 2ml collection tube(provide). Transfer the entire sample from Step 10 into the column, including any precipitate that may have formed. Centrifuge at 10000×g for 1 min to bind DNA. Discard the collection tube and filtrate.
Place the column into a second 2 ml tube and wash by adding 500 μl Buffer HB. Centrifuge at 10000×g for 1 min. Discard flow-through and reuse the collection tube.
Place the column into the same collection tube and wash by adding 700 μl DNA Wash Buffer diluted with ethanol. Centrifuge at 10000×g for 1 min. Discard flow-through and reuse the collection tube.
Wash the column with a second 700 μl DNA Wash Buffer and centrifuge as above. Discard flow-through and reuse the collection tube.
Using the same 2 ml collection tube, centrifuge HiBind DNA Mini Column at maxi speed(≥10000×g) for 2 min to dry the column. This step is critical for removal of trace ethanol that might otherwise interfere with downstream applications.
Place the column into a nuclease-free 1.5 ml microfuge tube and add 50-100 μl of preheated (65℃) Elution Buffer to HiBind DNA Mini column matrix. Allow columns to incubate for 3 to 5 min at room temperature after addition of Elution Buffer.
Note: Incubating the HiBind DNA column at 65℃ rather than at room temperature prior to centrifugation will give a modest increase in DNA yield per elution.
To elute DNA from the column, centrifuge at 10000×g for 1 min.
Repeat the elution with a second 50-100 μl Elution Buffer, reserve at -20℃.
PCR from target gene
Materials:
2× PrimeSTAR GC Buffer(Mg2+ plus)
dNTP Mixture (2.5 mM each)
Primer 1
Primer 2
Template
Genome of Paenibacillus strain
Genome of Klebsiella pneumonia strain
PrimeSTAR HS DNA Polymerase(2.5U/μl)
Sterilized purified water
Methods:
Thermocycling:
Gel extraction of target gene
Materials:
PCR products: the genes of Formamidase and Phosphate dehydrogenase
E.Z.N.A.® Gel Extraction Kit
Methods:
Perform agarose gel/ethidium bromide electrophoresis to fractionate DNA fragments. Any type or grade of agarose may be used. It is strongly recommended however, that fresh TAE buffer, or TBE buffer be used as running buffer. Do not reuse running buffer as its pH will increase and reduce yields.
When adequate separation of bands has occurred, carefully excise the DNA fragment of interest using a wild, clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Determine the appropriate volume of the gel slice by weighing it in a clean 1.5 ml microcentrifuge tube. Assuming a density of 1g/ml of gel, the volume of gel is derived as follows: a gel slice of mass 0.3g will have a volume of 0.3 ml. Add an equal volume of Binding Buffer (XP2). Incubate the mixture at 55℃-60℃ for 7 min or until the gel has completely melted. Mix by shaking or vortexing the tube in increments of 2-3 minutes.
Important:Monitor the pH of the Gel/Binding Buffer mixture after the gel has completely dissolved. DNA yields will significantly decrease when pH＞8.0. If the color of the mixture becomes orange or red, add 5 μl of 5M Sodium Acetate, pH 5.2 to bring the pH down. After this adjustment, the color of the Gel/Binding Buffer mixture should be light yellow.
Place a HiBind® DNA column in a provided 2 ml collection tube.
Apply 700 μl of the DNA/agarose solution to the HiBind® DNA column, and centrifuge 10000×g for 1 min at room temperature.
Discard liquid and place the HiBind○R DNA column back into the same collection tube.For volumes greater than 700 μl, load the column and centrifuge successively, 700 μl at a time. Each HiBind® DNA column has a total capacity of 25 μg DNA. If the expected yield is larger, divide the sample into the appropriate number of columns.
Add 300 μl of Binding Buffer(XP2) into the HiBind® DNA column.Centrifuge at 10000×g for 1 min at room temperature to wash the column. Discard the flow-through and reuse the collection tube.
Wash the HiBind® DNA column by adding 700 μl of SPW Wash Buffer diluted with absolute ethanol. Centrifuge at 10000×g for 1 min at room temperature.
Note: SPW Wash Buffer Concentrate must be diluted with absolute ethanol before use. If refrigerated, SPW Wash Buffer must be brought to room temperature before use.
Optional: repeat step 8 with another 700 μl of SPW Wash Buffer diluted with absolute ethanol.
Note: Perform the second wash step for any salt sensitive downstream applications.
Discard liquid and centrifuge the empty HiBind® DNA column for 2 min at maximal speed(≥13000×g) to dry the column matrix. Do not skip this step, it is critical for the removal of ethanol from the HiBind® DNA column.
Place a HiBind○R DNA column into a clean 1.5 ml microcentrifuge tube. Add 30-50 μl (depending on desired concentration of final product) of Elution Buffer(10 mM Tris-HCl, pH 8.5) directly onto the column matrix and incubate at room temperature for 1 min.Centrifuge for 1 min at maximal speed(≥13000×g) to elute DNA. This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration. Store at -20℃.
Note: The efficiency of eluting DNA from the HiBind○R DNA column is dependent of pH. If eluting DNA with deionized water, make sure that the pH is around 8.0.
Preparation of competent cells
Materials:
LB broth
0.1M CaCl2
E. coli strain in -70℃
Methods:
Thaw the frozen E. coli stain on ice for 10 minutes;
Inoculate E. coli in a LB plate by scribing method, label well and incubate overnight at 37 ° C;
The next day, pick a single colony from the plate and inoculate it into a test tube containing 3 ml of LB broth and incubate overnight at 37 ° C;
The next day, add 1ml bacterial liquid from the last step into 100ml LB broth and incubate for 2-3hours at 37°C shaking at 200-300 rpm;
When OD600 of the bacterial liquid arrive 0.3-0.4, put the flask on ice for 10-15 minutes;
Pour the bacteria liquid into a 50ml centrifuge tube under aseptic conditions, 4 ℃, 4000 g centrifugation for 10 min;
Remove supernatant, place the tube on dry filter paper for 1 min to remove the remaining culture medium;
Add 10ml 0.1M CaCl2 into the tube, shake mix, suspend the bacteria, ice bath for 30 minutes;
4℃, 4000 g centrifugation for 10 min, remove supernatant, place the tube on dry filter paper for 1 min to remove the remaining culture medium;
Add 4ml 0.1M CaCl2 into the tube, suspend the bacteria;
Pack 0.2ml for each tube and store in 4 ℃;
Better use within 24-48 hours. If not temporarily, store competent cells in -30 ℃.
Plasmid miniprep
Materials:
Bacterial liquid
GENEray® Plasmid Miniprep Kit
Methods:
Add 200 μl Buffer CBS into the GenClean Column, 12000rpm centrifugation for 1 min, discard the waste in the collection tube, put the column back to the collection tube for the next step.
2-4 ml bacterial liquid cultured overnight in LB, 8000rpm centrifugation for 1 min, discard the supernatant.
Add 250 μl Solution Ⅰ, mix the cells thoroughly by flicking the tube and inverting.
Add 250 μl Solution Ⅱ, mix by shaking the tube up and down mildly 4-6 times until the solution becomes translucent. This step should be limited in 5 minutes. If the solution is still turbid after shaking, it is because of too much bacteria. We should decrease the amount of bacteria.
Add 350 μl Solution Ⅲ, mix by shaking the tube up and down mildly 8-10 times, store at room temperature for 2-5 minutes, then 12000rpm centrifugation for 10 min.
Transfer the supernatant in step 5 to the column in a 2ml collection tube, 6000rpm centrifugation at room temperature for 1 minutes, remove DNA column and discard the waste in the tube.
Put the column back to the tube and add 500 μl W1 Solution, 12000 rpm centrifugation at room temperature for 1 min., discard the waste in the tube.
Put the column back to the tube and add 500 μl Wash Solution, 12000 rpm centrifugation at room temperature for 1 min., discard the waste in the tube.
Repeat step 8.
Discard liquid and centrifuge the empty DNA column for 1 min at 12000 rpm to dry the column matrix.Then open the lid for minutes to dry the column matrix entirely.
Put GenClean Column to a clean 1.5 ml tube, add 50-100 μl Elution Buffer, store at 37℃ for 2 min. 12000 rpm centrifugation at room temperature for 1 min. The left liquid in tube is plasmid we need.
Store at -20℃.
Restriction Digestion
Materials:
Water, nuclease-free
10× FastDigest or 10× FastDigest Green Buffer
DNA
FastDigest enzyme
Methods:
DNA Insert Ligation into Vector DNA
1) Sticky-end ligation
Materials:
Linear vector DNA
Insert DNA
10×T4 DNA Ligase Buffer
T4 DNA Ligase
Water, nuclease-free
Methods:
2) Blunt-end ligation
Materials:
Linear vector DNA
10× T4 DNA Ligase Buffer
50% PEG 4000 Solution
T4 DNA Ligase
Water, nuclease-free
Total volume
Methods:
Transformation into E. coli
Materials:
Competent cells( E. coli BL21 and E. coli DH5α are what we use in all experiments)
Recombinant plasmid
Selected plates
LB broth
Ice
Methods:
Add 1-5 μl recombinant plasmid into competent cells.
Carefully flick the tube 4-5 times to mix cells and DNA.
Place the mixture on ice for 30 minutes.
Heat shock at exactly 42°C water bath for exactly 2 minutes.
Place on ice for 5 minutes.
Pipette 1 ml of room temperature LB media into the mixture.
Incubate at 37°C and 200-250 rpm for 60 minutes.
6000×g centrifugation at room temperature for 5 min.
Discard 1 ml supernatant, mix the cells thoroughly by flicking the tube and inverting.
Plate resuspended cells as above.
Incubate overnight at 37°C with plates upside down.
Preparation of MOPS Broth, Agar and Semi-solid LB medium
1) MOPS Broth (1 L)
Materials:
MOPS
Tricine
NaCl
MgCl2•6H2O
K2SO4
FeSO4•7H2O
CaCl2•2H2O
Micronutrients
Glucose
Formamide
Phosphite
Methods:
2) MOPS Agar (200 ml)
Materials:
2×MOPS Buffer
Agar
Glucose
Formamide
Phosphite
Methods:
Prepare 96ml 2×MOPS Buffer.
Solute 1g glucose and 2g agar with 100 ml sterilized purified water.
Autoclave at 121℃.
When the temperature of 2×MOPS Buffer and glucose solution is below 50℃, mix 2×MOPS concentrate(96ml), glucose solution(100ml), 200 mM formamide(2ml) and 1.32 mM phosphite solution(2ml), totally 200 ml.
3) Semi-solid LB medium
Materials:
NaCl
Peptone
Yeast powder
Agar powder
Methods:
Growth curve determination
Materials:
LB broth
MOPS medium
Engineering bacteria
Methods:
Purification of bacteria liquid:
Cultivate engineering bacteria in LB broth for 2-3 hours.
Add IPTG into the broth and shake at 37℃ for 1 day.
4℃, 7000rpm centrifugation for 5 min, discard the supernatant.
Wash and resuspend the bacteria with 0.9% NaCl.
Repeat step 4.
Resuspend the bacteria with 0.9% NaCl and store at 4℃.
Inoculation into MOPS medium
Determine the bacteria concentration at OD600 every 4 hour.
Note: the time we determine depends on the pre-test. If the bacteria grow quickly, we will do the determination more frequently, otherwise we will give a longer interval, e.g. 8 hours or 12 hours.
Verification of the resistance to bacterial contamination
Materials:
Engineering bacteria with EGFP
Other bacteria, e.g. Staphylococcus aureus
MOPS broth
LB plate
MOPS plate
Equipment:
Flow cytometry
Ultraviolet light
Methods:
Use Flow cytometry:
Inoculate engineering E. coli and other bacteria into MOPS broth.
Extract 2 ml sample every 4 hours.
Determine the amounts of engineering E. coli and other bacteria with Flow cytometry.
Note: Because of the GFP, we can use the green fluorescent channel to distinguish our engineering bacteria and other bacteria. Then we count the number of E. coli with GFP and the number of all the bacteria. Through calculation, we can compare the growth of our engineering E. coli and other bacteria.
Cultivation on selected plates
Inoculate engineering E. coli and other bacteria onto MOPS plate.
Observe the bacterial spot under white light and count the number of all bacterial spot.
Observe the bacterial spot under Ultraviolet light, and count the number of our engineering E. coli. (E. coli with GFP can show green bacterial spot under Ultraviolet light)
Subtract the number of engineering E. coli from the number of all bacterial spot. The left is the number of other bacteria. Then we can know the growth of our engineering E. coli and other bacteria.
Verification of the expression of EGFP
Materials:
Engineering E. coli with EGFP
MOPS medium
Equipment:
Fluorescence microscopy
UV spectrophotometer
Methods:
Cultivation of engineering bacteria with EGFP in MOPS medium.
Determination and photographing on Fluorescence microscopy.
Part 2 CRISPR/Cas9
Phage infection experiment
Materials:
Phage stock
Semi-solid LB medium
Solid LB medium
LB broth
MOPS broth
Engineering E. coli
pCas was a gift from Sheng Yang (Addgene plasmid # 62225).
pTargetF was a gift from Sheng Yang (Addgene plasmid # 62226).
Methods:
Two layer plating method
Add 100 μl phage dilution solution into 200 μl bacteria liquid in log phase, mix, room temperature for 5-10min, waiting for phage adsorption on the bacteria.
Add 3 to 5 ml of 55 ° C thawed semi-solid LB medium and mix.
Poured the mixture onto the LB plate, label well and incubate overnight at the appropriate temperature.
The next day phage infection can be observed by the number of plaques.
Note:Dilution concentration gradient of Phage solution is 10-1, 10-2, 10-3, 10-4.
Liquid infiltration method
Cultivate the bacteria to the logarithmic phase (OD600 0.4 ~ 0.6).
100ul of the phage stock dilution was added to 50ml of bacteria in logarithmic phase, store at room temperature for 5-10min, waiting for phage adsorption on the bacteria.
Place the bacteria solution in a shaker at 30 ° C and 160 rpm for cultivation.
Sample every two hours, measure the OD600 of the bacteria.
Lab of Applied Biocatalysis,
College of Food Sciences and Engineering,
South China University of Technology,
Guangzhou 510640,
Guangdong,
Peoples R China
PI. Dr.Lou wenyong
wylou@scut.edu.cn
Members. Email:
lihuixianscut@126.com
xjialiu5211@163.com
If you are interested in our project, welcome to contact us.
