<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Duke">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE class="links"><TBODY><TR><TD align="center"><A id="show-home">Home</A></TD><TD align="center"><A id="show-project">Project</A></TD><TD align="center"><A id="show-notebook">Notebook</A></TD><TD align="center"><A id="show-team">Team</A></TD></TR></TBODY></TABLE><DIV id="home-box"><P><CENTER><FONT size="+1"><FONT color="blue"><B> One-Step Construction of a Bioplastic Production
Pathway in E. <I>coli</I></B></FONT></FONT></CENTER><B>We, the determined and motivated <FONT size="+1"><FONT color="blue">Duke University iGEM 2009 team</FONT></FONT> of 6 undergraduate students and 3 graduate students, have developed a practical and powerful method to deliver on the promise of sustainable <FONT size="+1"><FONT color="green">green</FONT></FONT> synthetic biology!</B>
Due to the high costs and inefficiency of the gene and pathway cloning process, we invented a new procedure which <FONT size="+1"><FONT color="blue">lowers costs</FONT></FONT> and <FONT size="+1"><FONT color="blue">increases efficiency</FONT></FONT>. This method, Circular Polymerase Extension Cloning (CPEC), <FONT size="+1"><FONT color="blue">saves time</FONT></FONT> as well, since it does not involve ligation or restriction enzymes.  See publication of our method <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0006441">here.</A>
We have applied our method to the production of <FONT size="+1"><FONT color="blue">biologically derived plastics</FONT></FONT>. The high costs of the current method of producing biodegradable plastics in an environmentally sound manner has hindered its widespread use.  However, we have discovered a <FONT size="+1"><FONT color="blue">more efficient pathway</FONT></FONT> to produce these biodegradable plastics. </P><DIV style="border:1.5px dashed black"><H2>Why do we deserve <B>GOLD</B>?</H2><UL><LI> We had a <B>Great</B> summer working on our project</LI><LI> We submitted <B>2</B> biobricks: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K282001">BBa_K282001</A> and <A href="http://partsregistry.org/Part:BBa_K282000">BBa_K282000</A></LI><LI> We Created and implemented with great success a more accurate, flexible and efficient method of splicing multi-component biobricks systems into plasmid vectors. </LI><LI> and... </LI></UL><CENTER> *We were also proud to help Team Valencia with their survey.</CENTER></DIV></DIV><DIV id="project-box"><H1>Abstract</H1>
A convenient ligation-free, sequence-independent one-step plasmid assembly and cloning method is developed [Quan J, Tian J (2009) Circular Polymerase Extension Cloning of Complex Gene Libraries and Pathways. PLoS ONE 4(7): e6441]. This strategy, Circular Polymerase Assembly Cloning (CPEC), relies solely on polymerase extension to assemble and clone multiple fragments into any vector. Using this method, we are able to quickly assemble a metabolic pathway consisting of multiple enzymes and regulatory elements for the production of a biocompatible as well as biodegradable plastic polymer in E. <I>coli</I>. 

<H1> What is CPEC?</H1>

Circular Polymerase Extension Cloning (CPEC) is a much simplified 
sequence-independent cloning technology based entirely on the polymerase extension 
mechanism. This method extends overlapping regions between the insert and vector 
fragments to form a complete circular plasmid (Figure 1a). The basic principle of this strategy is that, after denaturation, both single-stranded vector and inserts overlap with each other by end sequences and extend by using the other as the primer in a typical PCR reaction. After complete circular plasmids with one nick in each strand are formed, the relaxed plasmids are transformed to competent cells and sealed as closed plasmids. CPEC strategy is not only suitable for the cloning of an individual gene but also for gene libraries, combinatorial libraries and multi-fragment plasmid assembling. With its simplicity, cost-effectiveness and high transformation efficiency, CPEC has become the most convenient, economical and accurate cloning method at the present day, especially for cloning applications with high complexity. 

<TABLE style="background:transparent; margin:auto"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1a. Biobrick spliced into vector using CPEC</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1b. Multicomponent Biobrick system spliced into vector simulatenously using CPEC</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H1>One-Step Construction of a Bioplastic Production Pathway using CPEC</H1>

We applied CPEC in the construction of a multi-component 
pathways in a single reaction. The proposed multi-way cloning mechanism is shown in Figure 1b. First, we constructed the biobrick parts containing genes involved in this pathway. Then we used multi-way CPEC for construction of 
a metabolic pathway for synthesizing a biodegradable plastic, poly(3HB-co-4HB) in E. <I>coli.</I><H3> Biodegradable Plastic Synthesis Pathway in E. <I>coli.</I></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:206px;"><DIV class="thumbcaption">Figure 2. PHA granule production in bacteria</DIV></DIV></DIV><P>Recently, polyhydroxyalkanoates (PHA) have received a lot of attention for their
potential use as bioplastic. They are naturally occurring biopolymers – under low nitrogen,
phosphorus, or oxygen conditions, many bacteria produce protein- and lipid-coated granules of
PHA for energy storage, and these granules can then in turn be harvested in large quantities as
bioplastics (Fig. 2). Bioplastics not only help reduce the reliance on petroleum-based plastics, but
also have a wide range of applications in the agricultural and medical fields because they are
biodegradable and biocompatible. The most common PHA is poly(3-hydroxybutyrate), or poly(3HB), which is produced by
the bacteria <I>Cupriavidus necator</I>. Three genes – phaA, phaB, and phaC – make up the
phaCAB operon that synthesizes the enzymes required for formation of poly(3HB), which is not
elastic enough for general plastic applications. But when phaCAB is expressed with cat2, a
succinate degradation gene from Clostridium kluyveri that produces the 4HB monomer, the
copolymer poly(3-hyroxybutyrate-co-4-hydroxybutyrate), or poly(3HB-co-4HB), is produced (Figure 3). Poly(3HB-co-4HB) is more elastic and thus has potential to be used in a variety of applications. 

</P><P>
However, bioplastic production is currently too costly to be used widely. For example,
poly(3HB-co-4HB) pathways that have been developed so far produce copolymers with very low
concentrations of 4HB, the monomer that gives the copolymer its elastic properties. This could
be because the enzyme that catalyzes PHA formation, PHA synthase, has higher affinity for the
3HB-CoA substrate (100%) than the 4HB-CoA substrate (0.03%). 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:669px;"><DIV class="thumbcaption">Figure 3. Pathway for poly(3HB-co-4HB) synthesis [Madison and Huisman, 1999]</DIV></DIV></DIV><P>In addition, the 3HB
pathway uses acetyl-CoA, an input to the citric acid cycle, as a precursor, but cat2 uses succinyl-
CoA, an intermediate in the cycle. Since more processing is required to obtain the 4HB cursor, the pathway may naturally favor production of the 3HB polymer, resulting in higher 3HB composition in the final copolymer. 


Thus, this project aims to develop a more efficient biopathway for poly(3HB-co-4HB) while increasing the 4HB monomer composition predictably. To achieve this goal, we designed vectors with various combinations of the phaCAB operon and the cat2 gene so that genetic manipulations can be made to the genes for PHA synthase and precursor processing to
increase bioplastic production and quality. 

</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:230px;"><DIV class="thumbcaption">Figure 4. The reddish tint present on some colonies indicates the presence of PHA granules in the bacteria</DIV></DIV></DIV><H3>3HB genetic pathway</H3>
The phaCAB operon genomic DNA from <I>Cupriavidus necator</I> was cloned into PCR Blunt II-TOPO vector as the backbone and the cloning product was transformed into bacteria cells which were then plated on kanamycin (50 μg/ml) plates. Successful insertion of phaCAB was shown by Nile Red fluorescence in colonies (Figure 4).

<H3>4HB genetic pathway</H3>
The cat2 gene was inserted into the pSOS vector to obtain the
pSOS-cat2 construct. To obtain 4HB production, though, the phaC gene must also be expressed.
The 4HB precursor γ-butyrolactone (GBL) is required for 4HB production in this pathway, as the
4HB precursor does not naturally exist in E. <I>coli.</I><H3>pASK vector constructs</H3>
The PCR-Blunt II-phaCAB and pSOS-cat2 plasmids described above were then used as
templates to create seven different vector constructs in pASK (Tables 1 and 2). The first
six vector constructs were created by PCR amplification of the insert, restriction digestion of the
vector and insert, ligation of the fragments, and finally transformation into E. <I>coli.</I> The seventh vector (pASKphaAB-term-pLZCat2phaC) was created using CPEC cloning and transformation into E. <I>coli</I> (Figure 5). All the constructs were sequenced to ensure that the constructs were correct. Aslo genotyping was done for the final construct 7 (Figure 6). 
<TABLE class="wikitable" border="0" align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 5. Construct 7.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Figure 6. (Upper left) Gel electrophoresis analysis of the final assembly product after a 20-cycle CPEC. 5 ml of the reaction was separated on a 0.8% agarose gel and visualized after ethidium bromide staining. The full-length plasmid was 8360 bp. (Upper right) Gel electrophoresis analysis of the multi-way CPEC reaction. 5 ml was taken out of the reaction after 2, 5, and 10 cycles and separated on a 0.8% agarose gel (lanes 1, 2 and 3). The starting lengths of the four fragments were 3280, 2959, 2040, and 171 bp, respectively. The 171-bp band was not visible. (Bottom) Restriction mapping of the isolated plasmids derived from the CPEC reaction. Plasmid DNA from five independent colonies (I-V) were digested with BamHI (lane 2, 8.4 kb), BamHI-XhoI (lane 3, 6.6 kb and 1.8 kb), and NdeI (lane 4, 5.4 kb and 3 kb). Purified plasmids not subjected to restriction digestion are shown in lane 1. The molecular weight marker used in this figure was NEB 1 kb DNA ladder.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3>Screening Experiments</H3>
Two different experiments were conducted to investigate (1) the optimal
anhydrotetracylcine level for inducing expression of bioplastic genes and (2) Nile Red as a
screening technique for polymer production. To determine the optimal anydrotetracycline levels
for phaCAB and cat2 gene expression, agar plates were prepared with 0, 5, 50, 150, and 200
ng/ml of anyhdrotetracyline, and Construct 7 E. coli were grown on them overnight at 37°C. The
number of colonies on each plate was then compared the following day. To test Nile Red as a
screening technique to detect PHA production, agar plates were prepared with various levels of
Nile Red (0, 0.2, 0.5,1.0 μg/ml). Bacteria that were known to produce PHA (positive control) and
negative control bacteria were then plated and grown overnight at 37°C. After sitting on the
benchtop for a few days, they were examined under equal levels of UV lighting.

<DIV class="thumb tright"><DIV class="thumbinner" style="width:192px;"><DIV class="thumbcaption">Figure.7. Precipitated product from pASK-phaC-no tag (2 tubes on left) and negative control pASK (2 tubes on right) harvesting.</DIV></DIV></DIV><H3>Polymer harvesting</H3>
To obtain in vivo PHA polymer, E. coli containing the desired construct were grown in 3
mL of LB broth overnight, of which 1 mL was then cultured in 50 mL of LB broth and grown at
30oC until the cells reached an optical density of .7 A (600 nm). Glucose was added to a final
concentration of 1%, and the cells were harvested after 48 hour of incubation at 37°C.
To harvest, the cells were centrifuged at 3500 rpm and suspended in saline (0.0067 M
phosphate buffer) solution. The cells were re-centrifuged to obtain a washed pellet, freezethawed
in dry ice, and then freeze-dried by lyophilization. The pellet was shaken overnight at 37°C in chloroform, after which bacterial cell waste was filtered out using Whatman paper and a vacuum flask. The polymer was then precipitated from the filtered solution by adding 10 volumes of ethanol, centrifuged, and then re-dissolved in chloroform. Finally, 10 volumes of methanol was used to re-precipitate the polymer before centrifuging to obtain a purified polymer pellet. A promising sign was that there was
much more white powder precipitate in the experimental samples than the negative control
samples after harvesting (Figure 7). 

<H3><SUP>1</SUP>H NMR Spectroscopy</H3>
To characterize harvested polymers, 300 MHz 1H NMR spectroscopy was used. The
NMR sample was prepared by re-dissolving 20 mg of each polymer sample in 600 μl of
deuterochloroform in an NMR tube. Attempts were made to harvest and identify the 3HB, 4HB, and poly(3HB-co-4HB)
polymers. The first harvest attempt was made on pUC19phaCAB and then on pASK-phaCAB-no
tag (Construct 1) to try and obtain 3HB polymer; the NMR results for the harvested product were
compared to that of a negative control (no insert) and commercial 3HB. Several harvest attempts
were also made on pSOS-cat2phaC bacteria (Construct 4) to try to confirm 4HB polymer
production. Finally, bacteria transformed with pASKphaAB-termpLZCat2phaC (Construct 7)
were harvested in an attempt to identify poly(3HB-co-4HB) by NMR. 

<TABLE class="wikitable" border="0" align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Figure 8a. 300 MHZ <SUP>1</SUP>H NMR for commercial 3HB</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Figure 8b. 300 MHZ <SUP>1</SUP>H NMR for experimental 3HB</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">Figure 8c. 300 MHZ <SUP>1</SUP>H NMR for negative control (pASK vector)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE class="wikitable" border="0" align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Figure 8d. Comparison of expansions of the methane proton region in the <SUP>1</SUP>H NMR spectra –commercial 3HB (left) and experimental (right).</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure 8e. Comparison of expansions of the CH2 regions in the <SUP>1</SUP>H NMR spectra – Commercial 3HB (left) and experimental (right)</DIV></DIV></DIV></TD></TR></TBODY></TABLE>
The <SUP>1</SUP>H NMR spectra confirmed production of 3HB in the experimental bacteria and
no 3HB production in the negative control bacteria (Fig. 8a-e). Commercial 3HB (from Sigma-
Aldrich) was used as a comparison control (Fig. 8a). The 3HB peaks in the experimental sample
matched the commercial 3HB sample, as well as 3HB spectra found in the literatures.
However, the NMR spectra for the experimental and control samples had extremely large peaks
from residual methanol and ethanol left over from the harvesting procedures, a problem
corrected in later NMR readings by freeze-drying at the end of the harvesting procedure.


Future directions 
would be to test the hypothesis to see if phaC can be manipulated to increase 
4HB-to-3HB composition in poly(3HB-co-4HB) and to increase efficient production 
of the bioplastic by engineering the FtsZ cell division protein to allow for cells 
to accumulate larger quantities of PHA granules before dividing. Ultimately, once 
an optimal biopathway is found, the goal would be to explore a model for mass 
production of PHA bioplastics so that novel applications of bioplastics can be <DIV class="thumb tright"><DIV class="thumbinner" style="width:608px;"><DIV class="thumbcaption">Table 1. Vector designs with various phaCAB and cat2 combinations</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:582px;"><DIV class="thumbcaption">Table 2. Restriction enzymes and details of the vector constructs</DIV></DIV></DIV><P>
feasible economically.</P><H1> Biobricks </H1>

In order to facilitate the completion of this project, several biobricks were created and used.  The two foremost (the ones submitted) were phaAB and a double terminator.

<H3> phaAB </H3>
Part: <B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K282000"> BBa_K282000 </A></B>
Used in the creation of bioplastics. Contains both the PhaA gene and the PhaB gene. Part of the pathway PhaAB - Terminator - Cat2PhaC .
<H3> Terminator </H3>
Part: <B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K282001"> BBa_K282001 </A></B>

Based off of Part:<B>BBa_B0014</B>.  Comprised of forward terminator Part:<B>BBa_B0010</B> and reverse terminator Part:<B>BBa_B0012</B>.

</DIV><DIV id="notebook-box"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Calendar"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Calendar</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Protocols"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#CPEC_Cloning"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">CPEC Cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#DNA_Purification"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">DNA Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#PCA_.28Polymerase_Cycle_Assembly.29"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">PCA (Polymerase Cycle Assembly)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#PCR_Product_Clean-up_for_DNA_Sequencing"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">PCR Product Clean-up for DNA Sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#PCR"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Single_Colony_PCR"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Single Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Transformation"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#NMR"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">NMR</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-11"><A href="#Lab_Safety"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Lab Safety</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Calendar">Calendar</SPAN></H1><H1><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H1><H2><SPAN class="mw-headline" id="CPEC_Cloning">CPEC Cloning</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials</B></P><UL><LI> Phusion™ High-Fidelity PCR Kit (FINNZYMES, Cat. No. F-553)
</LI><LI> Thermocycler
</LI></UL><P><B>Preparation</B></P><PRE>
5x Phusion HF Buffer    4 ul
10 mM dNTPs	        0.4 ul	
Vector		        50 ng/1kb	
Insert		        x ng*	
Phusion DNA Polymerase  0.2 ul
-------------------------------
H2O		        to 20 ul
</PRE><P>*The amount of insert is determined so that the molar ratio for vector and insert is 1 to 2.
</P><P><B>Procedures</B></P><PRE>
98°C                  30sec
  10X
     98°C             10 sec
     Annealing**      30 sec
     72°C             x sec***
72°C                  5min
4°C                   hold
</PRE><P>** Anneal at Tm + 3°C. The Tm should be calculated with the nearest-neighbor method.
***The extension time is usually calculated according to the shortest piece with 15 sec /kb if the cloning is not complicated. For example, if there is only one insert and is shorter than the vector, say, 600 bp, then I will use 15 sec for extension. Refer to the published paper for detailed information.
</P><H2><SPAN class="mw-headline" id="DNA_Purification">DNA Purification</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials :</B><UL><LI> E.Z.N.A Gel Purification Kit (Omega Bio-Tek, Cat No. D2500-02 )</LI><LI> Water bath equilibrated to 55-65C</LI><LI>Microcentrifuge capable of at least 10,000 x g</LI><LI>Nuclease-free 1.5 ml centrifuge bottles</LI><LI>Absolute (95%-100%) ethanol</LI><LI>Protective eye-wear</LI><LI>Isopropanol (for fragments &lt; 500 bp only)</LI></UL></P><P><B>Protocol:</B></P><OL><LI> Perform agarose gel electrophoresis to fractionate DNA fragments. Any type or grade of agarose may be used. It is strongly recommended, however, that fresh TAE buffer or TBE buffer be used as running buffer. Do not re-use running buffer as its pH will increase and reduce yields. 
</LI><LI> When adequate separation of bands has occurred, carefully excise the DNA fragment of interest using a wide, clean scalpel.
</LI><LI> Determine the approximate volume of the gel slice by weighing it in a clean 1.5 ml microfuge tube. Assuming a density of 1 g/ml of gel, the volume of gel is derived as follows: A gel slice of mass 0.3 g will have a volume of 0.3 ml. Add equal volume of Binding Buffer (XP2). Incubate the mixture at 55C-60C for 7 min or until the gel has completely melted. Mix by shaking or inverting the tube every 2-3 minutes. Centrifuge the tube briefly to collect all the liquid to the bottom of the tube.
</LI></OL><P>Note: For DNA fragment less than 500bp, add 1 sample volume of isopropanol after the addition of Binding Buffer (XP2).
</P><OL><LI> Apply up to 700 ul of the DNA/agarose solution to a HiBind® DNA spin column assembled in a clean 2 ml collection tube (provided) and centrifuge in a microcentrifuge at 8,000-10,000 x g for 1 min at room temperature. Discard the liquid. Re-use the collection tube in Steps 5-8. For volumes greater than 700 ul, load the column and centrifuge successively, 700 ul at a time. Each HiBind® spin-column has a total capacity of 25-30 ug DNA.
</LI><LI> Discard liquid and add 300ul Binding Buffer. Centrifuge at 10,000 x g for 1 minutes.
</LI><LI> Add 700 ul of SPW Buffer diluted with absolute ethanol into the column and wait 2-3 minutes. Centrifuge at 10,000 x g for 1 min at room temperature to wash the sample.
</LI><LI> Discard liquid and repeat step 6 with another 700 ul SPW Buffer.
</LI><LI> Discard liquid and, re-using the collection tube, centrifuge the empty column for 1 min at maxi speed (&gt;13,000 x g)  to dry the column matrix. This drying step is critical for good DNA yields.
</LI><LI> Place column into a clean 1.5 ml microcentrifuge tube (not provided). Add 30-50 ul depending on desired concentration of final product) Elution Buffer (or sterile deionized water) directly to the center of the column matrix, then incubate for 1 minute. Centrifuge 1 min at maxi speed (&gt;13,000 x g) to elute DNA. This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lo
</LI></OL><H2><SPAN class="mw-headline" id="PCA_.28Polymerase_Cycle_Assembly.29">PCA (Polymerase Cycle Assembly)</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials</B></P><UL><LI> Phusion™ High-Fidelity PCR Kit (FINNZYMES, Cat. No. F-553)
</LI><LI> Thermocycler
</LI></UL><P><B>Preparation</B></P><TABLE><TBODY><TR><TD>5x Phusion HF Buffer </TD><TD> 5 ul
</TD></TR><TR><TD>10 mM dNTPs </TD><TD> 0.5 ul
</TD></TR><TR><TD>Oligo mixture </TD><TD> 125 ng /250 ng /500 ng /
</TD></TR><TR><TD>Phusion DNA Polymerase	</TD><TD> 0.25 ul
</TD></TR><TR><TD> --------------------- </TD><TD> --------
</TD></TR><TR><TD>H2O </TD><TD> to 25 ul
</TD></TR></TBODY></TABLE><P><B>Procedures</B></P><PRE>
98°C              30sec 
  40X
      98°C        7 sec
      70-50°C     slow ramp, 0.1°C/sec
      50°C        30 sec 
      72°C        15 sec /kb
72°C              5 min
4°C               hold
</PRE><H2><SPAN class="mw-headline" id="PCR_Product_Clean-up_for_DNA_Sequencing">PCR Product Clean-up for DNA Sequencing</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials:</B></P><UL><LI> ExoSAP-IT® (usb, Cat. No. 78200)
</LI><LI>Thermocycler
</LI></UL><P><B>Protocol:</B></P><OL><LI> Remove ExoSAP-IT from -20°C freezer and keep on ice throughout this procedure.
</LI><LI> Mix 5 μl of a post-PCR reaction product with 2 μl of ExoSAP-IT for a combined 7 μl reaction volume.
</LI><LI> Incubate at 37°C for 15 min to degrade remaining primers and nucleotides.
</LI><LI> Incubate at 80°C for 15 min to inactivate ExoSAP-IT.
</LI><LI> The PCR product is now ready for use in DNA sequencing etc.
</LI></OL><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials</B></P><UL><LI> Phusion™ High-Fidelity PCR Kit (FINNZYMES, Cat. No. F-553)
</LI><LI> Thermocycler
</LI></UL><P><B>Preperation</B></P><TABLE><TBODY><TR><TD>5x Phusion HF Buffer </TD><TD> 10 ul
</TD></TR><TR><TD>10 mM dNTPs </TD><TD> 1 ul
</TD></TR><TR><TD>DNA template </TD><TD> 1 pg – 10 ng
</TD></TR><TR><TD>Forward primer (10 uM) </TD><TD> 2.5 ul
</TD></TR><TR><TD>Reverse primer (10 uM) </TD><TD> 2.5 ul
</TD></TR><TR><TD>Phusion DNA Polymerase </TD><TD> 0.5 ul
</TD></TR><TR><TD> -------------------- </TD><TD> -----
</TD></TR><TR><TD>H2O </TD><TD> to 50 ul
</TD></TR></TBODY></TABLE><P><B>Procedure</B></P><PRE>
98°C           30sec 
 30X
    98°C       10 sec
    Annealing* 30 sec
    72°C       15 sec per 1 kb
72°C           5min
4°C            hold
</PRE><P>
* Anneal at Tm + 3°C. The Tm should be calculated with the nearest-neighbor method.
</P><H2><SPAN class="mw-headline" id="Single_Colony_PCR">Single Colony PCR</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials</B></P><UL><LI> Taq DNA Polymerase with Standard Taq Buffer (NEB, Cat. No. M0273)
</LI><LI> 10 mM dNTP Mix (NEB, Cat. No. N0447)
</LI><LI> Thermocycler
</LI></UL><P><B>Preperation</B></P><TABLE><TBODY><TR><TD>10x Standard Taq Buffer </TD><TD> 2 ul
</TD></TR><TR><TD>10 mM dNTPs </TD><TD> 0.4 ul
</TD></TR><TR><TD>Bacteria culture* </TD><TD> 1 ul
</TD></TR><TR><TD>Forward primer (10 uM) </TD><TD> 1 ul
</TD></TR><TR><TD>Reverse primer (10 uM) </TD><TD> 1 ul
</TD></TR><TR><TD>Taq DNA Polymerase </TD><TD> 0.2 ul
</TD></TR><TR><TD> ----------------- </TD><TD> -------
</TD></TR><TR><TD>H2O </TD><TD> to 20 ul
</TD></TR></TBODY></TABLE><P>*Bacteria culture refers to E. coli cultured in LB solution overnight. 
</P><P><B>Procedures</B></P><PRE>
94°C              5 min 
 25X
     94°C         15 sec
     Annealing**  30 sec
     72°C         1 min per kb
72°C              5min
4°C               hold
</PRE><P>**Anneal at Tm which is calculated with salt-adjusted method.
</P><H2><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H2><P><A href="#Protocols"> Back to top</A></P><P><B>Materials:</B><UL><LI>GC5 Chemical Competent Cells (Genesee Scientific, Cat. No. 42-653)</LI><LI>SOC Medium (Sigma, Cat. No. S1797)</LI><LI>LB Agar (Sigma, Cat. No. L3027)</LI><LI>Petri Dishes (VWR, Cat. No. SC25373-187)</LI><LI>Cell Spreader (VWR, Cat. No. 89042-018)</LI><LI>37°C incubator</LI><LI>37°C shaker</LI><LI>water bath</LI></UL></P><P><B>Protocol:</B></P><P>1. Thaw 1 tube of competent cells on ice;
</P><P>2. Add 3 ul of cloning product or 1-50 ng of plasmid into competent cells while stirring gently;
</P><P>3. Keep the tube covered by ice for 30min;
</P><P>4. Heat-shock the competent cells in water bath for 45 sec at 42°C;
</P><P>5. Put the tube on ice for 2 minutes;
</P><P>6. Add 450 ul of SOC medium and then put it in a 37°C shaker for 1 hour;
</P><P>7. Dilute and spread an appropriate amount on an LB agar plate with the appropriate antibiotics;
</P><P>8. Place the plate up-side-down in 37°C incubator for 16-18 hours (overnight).
</P><H2><SPAN class="mw-headline" id="NMR">NMR</SPAN></H2><OL><LI> Login and start NMR program.
</LI><LI> Click Acqi. Load sample. Make sure spin is on and lock is off.
</LI><LI> Go to lock. Decrease number of sine waves to 1 to obtain step function. Make sure Zo is 1100, lockpower is below 30, lockgain is 36, lockphase is 352, and spin is 20.
</LI><LI> Go to shim. Increase lock level.
</LI><LI> Click main menu. Click set up. Click H1CDCl3. Type ‘nt=64’ (number of scans), ‘ss=2’ (dummy scans), ‘go’, ‘lb=0.2’ (line broadening).
</LI><LI> Viewing: Type ‘wft’, ‘dscale’, ‘dfp’ to display peak frequency.
</LI><LI> Zooming in: Click display. Go to interactive. Type ‘cr=8p, delta=8p’ to set width of zoom. Use cursor to set boundaries. Click to cut.
</LI><LI> Printing: Type “pL pscale(0) pltext ppf page’ to print spectra with text and peak frequencies.
</LI></OL><H1><SPAN class="mw-headline" id="Lab_Safety"> Lab Safety </SPAN></H1><P>None of our project ideas raised any safety issues whether it be researcher, environmental, or public safety.  Duke has an in-house review board for projects and they did not raise any flags regarding our project.   The biobrick parts created also do not pose any safety issues.
</P></DIV><DIV id="team-box"><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Not Including: Andrew Ang and Kevin Chien</DIV></DIV></DIV><H2>Students</H2><TABLE><TBODY><TR><TD><B>Andrew Ang</B> andrew.ang(at)duke.edu  Andrew Ang is a freshman at Duke, majoring in Biomedical Engineering. Apart from class and iGEM, he is involved in the Jazz Ensembles program and Asian Students Association at Duke. His hobbies include piano, saxophone, tennis, and squash. He is interested in molecular and synthetic biology, biomolecular engineering and medical research. He has previously worked as part of the MIT 2008 team, and he is excited to participate in iGEM again this year, and many more years to come.
</TD></TR><TR><TD><B>Kevin Chien</B> kevin.chien(at)duke.edu <P>Kevin Chien is a freshman at duke, majoring in Biomedical Engineering. He wants to explore his options in many different careers. He occasionally plays Club Frisbee and is interested in many different areas of knowledge, ranging from Astrophysics to Public Policy (and even sometimes History). He worked on this iGEM team last year as well and enjoyed his stay in Boston(hopes to have a good one this year too!).
</P></TD></TR><TR><TD><B>Yaoyao Fu</B> yf21(at)duke.edu <P>Yaoyao Fu is a master's student in Biomedical Engineering. She is interested in molecular synthetic biology. Her hobbies are traveling and sports. 
</P></TD></TR><TR><TD><B>Faith Kung</B> fk8(at)duke.edu <P>Faith Kung is a senior at Duke majoring in Biomedical Engineering with minors in Music and Biology. She enjoys working in a lab. Besides academics, her hobbies 
include arts and crafts, dance, and figure skating. Also, she is actively involved in the IV Christian Fellowship. Faith is applying to PhD programs 
in Biomedical Science and hopes to pursue a career in scientific research and 
education. She is excited about attending the iGEM competition this year.
</P></TD></TR><TR><TD><B>Sahil Prasada</B> sahil.prasada(at)duke.edu <P>Sahil Prasada is a freshman at Duke.  He plans to pursue medicine as a career. His interests lie in Detroit sports, tennis, and dancing. He is a member of the DBS Raas team on campus.  He is currently in the Trinity School of the Arts and Sciences but is considering transferring to the Pratt School of Engineering.  He hopes that the Detroit Lions may one day win the Superbowl. While waiting for this to occur, he will attend the iGEM competition and is looking forward to winning an award.
</P></TD></TR><TR><TD><B>Nicholas Tang</B> nicholas.tang(at)duke.edu <P>Nicholas Tang is a junior Biomedical Engineering/Electrical and Computer Engineering student. He plans to pursue his interests in computational biology and biological engineering. His hobbies include cycling, playing the violin and playing the piano. As a participant in the 2006, 2007, and 2008 Duke iGEM teams, he is excited to contribute to the progress of the synthetic biology community, and hopes to help in any way he can.
</P></TD></TR><TR><TD><B>Peter Zhu</B> peter.zhu(at)duke.edu  Peter Zhu Is a freshman at Duke University and a North Carolina local. Though he's not sure yet what to do with his life, he thinks Biomedical Engineering and pre-Law is looking pretty good. When he's not busy with the routines of life, he is listening to the Billboard Top 100, playing Chopin Preludes, searching for new places to eat, playing tennis, studying poker, and gaming Starcraft/DoTA. Peter is a regular at Bail Hai Mongolian Grill, Lime and Basil Vietnamese Pho, and Five Guy's Burgers---bacon cheeseburger with all toppings of course.
</TD></TR></TBODY></TABLE><H2>Faculty Advisors</H2><TABLE><TBODY><TR><TD><B>Dr. Jingdong Tian</B> jtian(at)duke.edu Department of Biomedical Engineering &amp; IGSP
</TD><TD><B>Dr. Lingchong You</B> you(at)duke.edu  Department of Biomedical Engineering &amp; IGSP
</TD><TD><B>Dr. Fan Yuan</B> fyuan(at)duke.edu  Department of Biomedical Engineering
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>Maggie Jiayuan Quan</B> jq7(at)duke.edu  Graduate Student
</TD><TD><B>Faisal Reza</B> faisal.reza(at)duke.edu  Graduate Student
</TD></TR></TBODY></TABLE></DIV><H3>Sponsors</H3><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Duke">http://2009.igem.org/Team:Duke</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke&amp;oldid=169204" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>