Team:ICT-Mumbai/Notebook
ICT-Mumbai 2017
Home
Project
Background Description Design Protocols Notebook Interlab Results
Team
Team Collaborations
Part
Safety Human Practices Attributions
ABOUT PORTFOLIO CONTACT SEARCH
Week 1: Ideation
During the first week, we tried to come up with ideas that we can work on. After long discussions and much debate, we finally decided to work on an issue that we think is quite relevant not only in our country but also in many developing and developed countries — that of stinking public toilets. For our inspiration and project summary, see the
project description
.
Week 2: Ground Reality
During this week, we decided to approach different NGOs and organizations that are involved in promoting and providing better sanitation. Sulabh International is an India-based NGO that promotes and provides better sanitation by building public pay and use toilets and community toilets. To understand and evaluate the feasibility of our idea and to understand the ground reality, we decided to meet with a representative of Sulabh International. Details of our interaction with this NGO can be found on our
human practices
page.
Week 3: We step into the lab!
Week 3, and we step into the lab!
During the first week in the lab, we learnt about the techniques and the principles behind them. It marked our first foray into the world of synthetic biology and it was great fun learning and performing these experiments.
1. Wet lab begins! We learn about different kinds of growth media, composition and their significance.
2. Introduction to agarose gel electrophoresis and DNA quantification methods.
Additionally, we learnt the precautions to follow while handling ethidium bromide containing gels and their disposal procedures. 3. We also kicked off our very first step towards establishing our proof of concept — extracting genomic DNA from Escherichia coli BW25113. We checked the size of the extracted product by agarose gel electrophoresis.
5ul and 10ul of the extracted products were loaded in lanes 1, 3 and 2, 4 respectively.
Week 4:The idea!
Glutamine synthetase (glnA), and the smaller and bigger subunits of glutamate synthetase (gltD and gltB, respectively) were our genes of primary interest. All the three genes of interest were amplified from E. coli BW25113 genomic DNA using polymerase chain reaction (PCR). PCR products were loaded and run on 1% agarose gel to check for the correct sizes of these three genes (glnA: 1640 bp, gltB: 4462 bp, gltD: 1420 bp). Among the three, we were successful in obtaining two genes — glnA and gltD. We repeated PCR for gltB for the second time by adding DMSO. This time too we could not see bands of correct size. We again carried out PCR using different annealing temperatures and the product obtained with 62°C as annealing temperature gave good bands. This was further gel purified and used as a template to carry out another PCR for the same gene. We could not see bands of correct size again.
Week 5: What next?
We anyway proceeded with cloning glnA in pSD113 (Ref. 1) and gltD in pUC19 cloning vector.
Step 1: RE digestion of glnA(insert) and pSD113(vector) with NdeI and HindIII              RE digestion of gltD(insert) and pUC19(vector) with SmaI and KpnI
Step 2: Digested vectors were gel purified and digested inserts were PCR purified. Further, glnA was ligated with pSD113 and gltD with pUC19 in 1:3 molar ratio.Each ligation mix was separately transformed in E. coli DH5α competent cells and plated separately on Luria Bertini (LB) medium + ampicillin agar plates. Few colonies from each plate were patched to create a masterplate and inoculated for plasmid preps and glycerol stocks.
Step 3:We carried out blue-white screening for gltD+pUC19, whereas we screened for glnA by carrying out a restriction digestion.
Week 6: We don't give up!
We repeated PCR for gltB using a different polymerase enzyme — Long PCR Enzyme Mix (Thermo Scientific #K0182).
We checked for the correct band (4462 bp) using agarose gel electrophoresis . Hurray! Positive results finally. gltB was further gel purified and RE digested with SalI and SmaI. pUC19 was digested with BamHI. Digested pUC19 was gel purified and digested gltB was PCR purified and a ligation reaction of both was was set up in 1:3 molar ratio. We sent out all three genes for sequencing.
Week 7: Interlab study begins!
While we were awaiting our sequencing results, we began our interlab study. All the test devices and controls were reconstituted and transformed from the kitplates. All of them were inoculated for making glycerol stocks the next day. Since our plate reader was under maintenance, we performed the measurements later.
Week 8: Making way for proteorhodopsin!
In order to make the GS-GOGAT cycle work, we needed proteorhodopsin (PR) that would produce ATP with the help of the proton gradient. However, this activity of PR is seen only in the presence of retinal. We found both, the gene for PR (BBa_K773002) and a device for retinal biosynthesis (BBa_K1604022) in the provided kitplate. For PR, we decided to use an araC regulated promoter (BBa_R0080) which was also present in the kitplate.
We reconstituted BBa_K773002 (Plate 2, well 1P), BBa_K1604022 (Plate 7, well 17K), BBa_R0080 (Plate 3, well 5G) and transformed in E. coli DH5α competent cells.
Colonies were found only for BBa_K773002. BBa_K1604022 and BBa_R0080 gave no colonies. We repeated the transformation for these two parts, this time we increased the incubation time for each. We were successful.
Week 9: Eureka moments!
We learned that farnesyl diphosphate (FPP) an intermediate in the carotene biosynthesis pathway is is also the starting molecule for a fragrant compound, limonene. Catalysed by limonene synthase (LIMS), FPP is converted to limonene. To our surprise we found the gene coding for LIMS+rbs (BBa_I74211) is present in the kitplate. We planned to clone this part downstream of BBa_K1604022 so as to obtain limonene.  Thus, we reconstituted and transformed this part from well 4I of plate 4 in E. coli DH5α competent cells. Colonies obtained were inoculated for making glycerol stocks and miniprepped the next day.
Week 10: Gene deletions!
We learned that ammonia assimilation in E. coli is regulated by PII proteins that adenylate glutamine synthetase (GS) under nitrogen rich conditions and render it inactive. It was found that the cell also contains another PII like protein called GlnK.(Ref. 2)
Since we wanted the cells to work under ammonia rich conditions, we had to delete glnB gene coding for PII, and, glnK gene coding for glnK. Gene deletions were carried out as described by Datsenko and Wanner (Ref. 3).
Week 11: Sequencing results arrive
Sequencing results were positive only for glnA. We anyway decided to get biobricked glnA and gltD synthesized from IDT. Since the PR in our kitplate is without a rbs, we decided to get that too synthesized from IDT. We sent out the sequences for the same. We decided to have glnA, gltB, gltD along with PR in one plasmid and BBa_K1604022 with BBa_I74211 in another plasmid containing a different ori in the same cell. We planned to carry out a 3A assembly using BBa_1604022, BBa_I74211 and BBa_JO4450.
Week 12: 3A assembly!
Sequencing results were positive only for glnA We carried out a variant of 3A assembly using BBa_1604022 (cut with EcoRI and SpeI), BBa_I74211(cut with XbaI and PstI) and BBa_JO4450 (cut with EcoRI and PstI). Unfortunately, it did not work.
Meanwhile, we received our gBlocks for glnA, gltDand rbs+PR from IDT. The concentrations of each of them were found to be extremely low that we reordered all the theree gblocks again.
From left to right: glnA fragment 1 (G1_F1), glnA fragment 2 (G2_F2), gltD fragment 1 (G3_F1), gltD fragment 2 (G3_F2), rbs+PR follwed by 1kb DNA ladder in the last lane.
Week 13: Discussions and decisions!
After a negative 3A result, we discussed and planned our next plan of action.
Week 14: Examination time!
We could not work this week as our university mid-semester examinations were scheduled.
Week 15: Eureka moment, again!
We learnt that indC from Photorhabdus luminescens encodes for indigoidene synthase which when acted on glutamine yields a blue coloured compound called indigoidine.We started looking for indC in the registry and found that the 2013 Heidelberg team had designed a range of parts with indC. We checked for their availability and ordered for these parts both from the registry as well as the Heidelberg team.
Week 16: Starting anew!
We cloned glnA in pET-43.1b vector between sites NdeI and HindII. While waiting for indC parts, we discussed about possible collaborations with other teams and started working on the team wiki>
Week 17: indC arrives!
We received indC (BBa_K1152008, BBa_K1152018 and BBa_K1152019) from the iGEM registry, whereas BBa_K1152013 and pRB5 were generously sent by the Heidelberg team. BBa_K1152008 was transformed, prepped, ligated with pET 43.1b+glnA and transformed in E. coli DH5α competent cells.
Four colonies of transformants (43+glnA+indC) were screened by performing RE digestion with NdeI
Results: Agarose gel electrophoresis revealed band of ~7000 bp in all colonies tested.
To reconfirm, colony 3 (C-3) was inoculated in LB+Amp. Plasmid prep of this colony was performed the next day and confirmed by double digestion with NotI and XhoI. Release of indC fragment (3855bp) was checked by running a 1% agarose gel. We could not see a band of the correct size.
Lane 2 contains 1kb DNA ladder whereas lane 3 contains RE digested pET 43.1b+glnA+indC.
Week 18: Getting glnA in place!
Since the concentration of G1_F1 obtained from IDT was very low, we PCR amplified it using Phusion polymerase.
From left to right: G1_F1, G1_F2 followed by 1kb DNA ladder
In the end, we determined the concentrations of G1_F1 and G2_F2 and carried out gene SOEing to join both the fragments. We did not get bands of the correct size (1640bp). We repeated gene SOEing with Pfu polymerase but to no avail.
Image of SOE PCR carried out usig Pfu
Meanwhile, we transformed BBa_K1152013 in E. coli DH5α which gave us blue colonies after 48 hours as reported by the Heidelberg team.
Week 19: What next?
We decided to co-transform both BBa_K1152013 and pET-43.1b containing glnA in E. coli BL21(DE3).However, both have pMB1 ori hence we decided to clone glnA+T7 promoter in a backbone with p15A ori.
Thus, we inoculated part BBa_J04450 with pSB3C5 backbone (p15A ori) and E. coli DH5α containing pET 43.1b with glnA. After miniprep the next day we carried out restriction digestion of BBa_J04450 (pSB3C5 backbone) with NotI and of pET43.1b containing glnA with Psp5II and XhoI to pull out glnA along with the T7 promoter. The digested products were further gel purified and blunted. Blunt products of the vector and the insert were ligated in 1:3 molar ratio.
Meanwhile, we decided to contribute a promoter designed by one of our PIs to the registry.
Week 20: On our toes!
We immediately designed and ordered primers to amplify the ychH promoter from E. coli MG1655 genomic DNA. The ychH promoter was inoculated in LB+Kanamycin and miniprepped the next day. We await our primers!
Meanwhile, ligated products (glnA + T7 + lacI with pSB3C5) were transformed in E. coli DH5α. Three colonies were picked up randomly and inoculated in LB+chloramphenicol. Along with that, E. coli DH5α transformed with BBa_K1152013 was also inoculated in LB+chloramphenicol. Both were miniprepped the next day.Both were double digested with EcoRI and PstI and a ligation reaction was setup in 1:3 molar ratio. This ligated product was transformed in E. coli DH5α. Many red colonies were seen the next day, which indicated self-ligation. Other colonies were screened and found to be negative.
Week 21: All about ychH promoter!
The primers for ychH promoter arrived. We carried out PCR using Pfu,  Phusion and Taq of which Taq gave the best results.(Correct sizes of the PCR products ~300 bp were confirmed using agarose gel electrophoresis). We set up more PCR reactions of the promoter using Taq. PCR products were gel purified and digested with EcoRI and PstI. The linearized plasmid backbone was also digested with EcoRI and PstI. Both pSB1C3 and ychH promoter were ligated in 1:3 molar ratio respectively. Ligated product was transformed the next day in E. coli DH5α. Eight colonies of transformants were randomly picked and inoculated in LB+Kanamycin. All were miniprepped the next day and screened for positives using PCR.
Week 22: Our part is ready!
Screening results for all eight colonies were positive!
From left to right: Gel1_Lanes 1-5: Colony number 1-5, lane 7: Positive control, lane 8: 1kb DNA ladder, Gel2_Lanes 1-3: Colony number 6-8, lane 7: Positive control, lane 8: 1 kb DNA ladder. We determined the concentration of each of these using Nanodrop spectrophotometer.
Submission: Based on the concentration obtained, we added 11 µl of miniprepped DNA prep (Colony 1: pSB1C3+ychH promoter) to A1 of the 96 well submission plate, covered it with the provided lid and kept it for overnight drying. Drying was carried out in the lab’s laminar flow hood which was surface sterilized with 90% ethanol. The miniprepped DNA had completely dried in the well the next day. We submitted our part as per the instructions given on the
iGEM parts submission page
.
Week 23: The last lap!
We decided to clone lacI + T7 + glnA (blunted insert) in another vector called pSEVA234 so that this can be co-transformed with BBa_K1152013 in E. coli BL21(DE3). Therefore, we inoculated pSEVA234 in LB+Kanamycin, miniprepped the next day and digested with SmaI. Both digested vector and insert were ligated in 1:3 molar ratio. Ligation product was transformed in E. coli DH5α and twelve colonies were randomly picked up and inoculated. All twelve were miniprepped and digested with XbaI to confirm the correct size (~1600 bp).
From left to right:Gel_1 Lane 1: 1kb DNA ladder, lane 2-8:Colony number 1-7;Gel_2 Lane 1-5: Colony number 8-12,lane 8: 1kb DNA ladder.
References:
1. Deb SS, Reshamwala SMS, Lali AM (2016). A series of template plasmids for Escherichia coli genome engineering. J Microbiol Methods 125:49-57. PMID: 27071533
2. van Heeswijk WC, Wen D, Clancy P, Jaggi R, Ollis DL, Westerhoff HV, Vasudevan SG (2000). The Escherichia coli signal transducers PII (GlnB) and GlnK form heterotrimers in vivo: fine tuning the nitrogen signal cascade. Proc Natl Acad Sci USA 97:3942-3947. PMID: 10760266
3. Datsenko KA, Wanner BL (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA 97:6640-6645. PMID: 10829079
