<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse_France_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse France/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Toulouse 2016</TITLE></P><DIV id="navi"><DIV id="topbar" class="hoc clear"><P>Follow us  @</P></DIV><HEADER id="header" class="hoc clear"><NAV id="mainav" style="max-width:auto; margin:0 auto;"><UL class="clear"><CENTER><LI><A href="https://2016.igem.org/Team:Toulouse_France">Home</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team#Presentation">Presentation</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Collaborations">Collaborations &amp; Meet Up</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team#Gallery">Gallery &amp; Fun Facts</A></LI></CENTER></UL><LI><A href="https://2016.igem.org/Team:Toulouse_France/Context">Project</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Predation">Predation</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Antifungals">Antifungals</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Confinement">Confinement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Confinement">Biological confinement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Device">Physical confinement : Device</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Description">Backbones description</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Experiments">Results</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Model">Modeling</A></LI><LI>Wet Lab
						</LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Composite_Parts">Parts</A></LI><UL style="text-align:left;"><LI><A href="https://2016.igem.org/Team:Toulouse_France/Composite_Parts">Composite Parts</A></LI></UL><LI><A href="https://2016.igem.org/Team:Toulouse_France/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Human_Practices">Ethics and Women in Science</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Attributions#Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Supports">Supports</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Medias">Medias</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Medal_Requirements">Achievements</A></LI></NAV></HEADER></DIV><DIV class="page" style="background-image:url('https://static.igem.org/mediawiki/2016/4/4b/Toulouse_France_LascauxFresqueTaureaux.png');"><DIV id="pageintro" class="hoc clear" style="padding:300px 0px;"><P class="sec_title" style="background-color:rgba(1,1,1,0.5);">Project Design</P></DIV></DIV><DIV class="column full_size" id="Predation" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Paleotilis project</P></U><P class="texteb"> 
		
		

					Developing a biological solution to protect one of the most invaluable humanity heritage is a beautiful challenge. But using a GMO in an unstable ecosystem is a very risky business. For these reasons, our project has been carefully designed from intense discussions with scientists, curators and the public. It was mandatory to ensure the best confinement strategy for our strain, to limit the material input in the cave and to ensure antifungal production only when required. These lead us to <B>divide our project in three modules</B> (Containment, Predation, Antifungal; Figure 1).
					<B style="font-size:12px;"> 
					Figure 1: <I>Bacillus subtilis</I> transformed with the two plasmids carrying the elements necessary to promote predation, to prevent the plasmid dissemination, and to produce antifungals molecules.
					</B></P></DIV></DIV></DIV><DIV class="column full_size" id="Predation" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Predation</P></U><P class="texteb"><B style="font-size:16px;">Context: </B>
					The Lascaux history has proved that any molecule addition could result in major and unexpected modifications of the cave microbiota. For example, the antifungal molecules abundantly used to fight against fungi infection have been degraded by <I>Pseudomonas fluorescens</I>, and this degraded molecules are used by fungi to proliferate… A major constraint we imposed to our project was thus to avoid adding nutriment in the cave. 
					<B style="font-size:16px;">Our solution: </B>
					We circumvented this difficulty by taking advantage of the predatory property of <I>Bacillus subtilis</I>: in starvation condition, one program of <I>Bacillus</I> is to produce toxins to kill other bacteria such as <I>Pseudomonas</I> species and to develop from their materials (Nandy et al, 2007). This has two valuable advantages: (i) no substrate are required when using the bacteria, and (ii) our <I>Bacillus</I> will reduce the deleterious population of <I>Pseudomonas fluorescens</I> (Martin-Sanchez et al, 2011). Besides, <I>Bacillus species</I> are inhabitants of the cave (Martin-Sanchez et al, 2014) and <I>Bacillus subtilis</I> is a model microorganisms with well-established genetic possibilities, i.e, the perfect chassis for our project (Figure 2).
					<CENTER><VIDEO width="600" height="auto;" controls=""><SOURCE src="https://static.igem.org/mediawiki/2016/c/c2/Toulouse_France_Designfigure.mp4" type="video/mp4">
					Your browser does not support the video tag.
					</SOURCE></VIDEO></CENTER><B style="font-size:12px;"> 
					Figure 2: In the cave, <I>Pseudomonas</I> species degrade the antifungal molecules previously used to prevent mould apparition. The degraded molecules are paradoxically used by fungal species to develop. Our <I>Bacillus</I> strain will develop from <I>Pseudomonas</I>, and in contact to fungi, will produce antifungal molecules to prevent mould development.
					</B><B style="font-size:16px;">Our constructions: </B>
					Two independent operons have been reported to allow for the predation property of <I>Bacillus subtilis</I>: the  sporulation killing factor (<I>skf</I>) and the sporulation delay protein (<I>sdp</I>) (Gonzalez-Pastor et al, 2003; Ellermeier et al, 2006; Gonzalez-Pastor, 2011). We tried both of them for this project. Since their expression is regulated at the transcriptional level, we decided to replace their promoter by pVeg for a constitutive expression (Figure 3). RFP expression was also added in prevision of a visualisation of the strain during test on stones.
					<B style="font-size:12px;"> 
					Figure 3: Predation constructions. SDP is composed of two operons (<I>sdpABC</I> and <I>sdpIR</I>) while SKF is a big operon (<I>skfABCEFGH</I>) of about 6 Kbp. These sequences encodes the necessary elements to produce, mature and secrete the killing factors, as well as the immunity agents to prevent lethality of the producing cells.
					</B></P></DIV></DIV></DIV><DIV class="column full_size" style="text-align:justify; font-size:14px; color:black; height:100%"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><DIV class="containerHP" style="margin:0px 0px; height:50px; height:100%"><DIV class="panel-group" id="accordion_tool" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="tool_heading1"><CENTER><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion_tool" href="#tool_collapse1" aria-expanded="false" aria-controls="tool_collapse1"><P style="text-align:center">REFERENCES</P></A></CENTER></DIV><DIV id="tool_collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="tool_heading1"><DIV class="panel-body"><P class="lead">	
										Ellermeier CD, Errett CH, Gonzalez-Pastor JE, and Losick R (2006) A Three-Protein Signaling Pathway Governing Immunity to a Bacterial Cannibalism Toxin. Cell. 124: 549–59. 
										González-Pastor JE. (2011). Cannibalism: A Social Behavior in Sporulating <I>Bacillus Subtilis</I>. FEMS Microbiology Reviews. 35: 415–24. 
										González-Pastor JE, Errett CH, and Losick R. (2003). Cannibalism by Sporulating Bacteria. Science. 301: 510–13.
										Martin-Sanchez PM, Jurado V, Porca E, Bastian F, Lacanette D, Alabouvette C, and Saiz-Jimenez C (2014). Airborne microorganisms in Lascaux Cave (France). International Journal of Speleology. 43:295-303
										Martin-Sanchez PM, Bastian F, Nováková A, Porca E, Jurado V, Sanchez-Cortes S, Lopez-and Tobar E (2011) Écologie Microbienne de La Grotte de Lascaux. https://www.researchgate.net/publication/257958803_Ecologie_Microbienne_de_la_Grotte_de_Lascaux. 
										Nandy SK, Prashant M, Bapat PM, and Venkatesh KV (2007). Sporulating Bacteria Prefers Predation to Cannibalism in Mixed Cultures. FEBS Letters. 581: 151–56.		

									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="column full_size" id="Antifungals" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Antifungals</P></U><P class="texteb"><B style="font-size:16px;">Context: </B>
					The Paleotilis project main idea is to contain the fungi patch progression in the Lascaux cave. Since treating with antifungal molecules has proved to be at the best a limited option, we wonder about how to produce antifungal in a targeted and efficient way.
					<B style="font-size:16px;">Our solution: </B>
					Since a variety of mould has been identified to cause problem in the cave, we wanted a wide spectrum solution and decided for an antifungal cocktail. 5 different molecules were selected:
					- D4E1, a 17 amino acids synthetic peptide analog to Cecropin B AMPs which has been shown to have antifungal activities by complexing with a sterol present in the conidia’s wall of numerous fungi (De Lucca et al, 1998). 

					- Dermaseptin-b1, a 78 amino acids peptide from <I>Phyllomedusa bicolor</I> which has antifungal activities against filamentus fungus. This peptide is membranotropic and depolarizes the fungi plasma membrane (Fleury et al, 1998).

					- GAFP-1 (<I>Gastrodia</I> Anti Fungal Protein 1), a mannose and chitin binding lectin originating from the Asiatic orchid <I>Gastrodia elata</I>, that inhibits the growth of ascomycete and basidiomycete fungal plant pathogens (Wang et al, 2001). 

					- Metchnikowin, a 26 residus prolin-rich peptid from <I>Drosophilia melanogaster</I> with a broad antifungal spectrum against Ascomycete and Basidiomycete. During its expression, it is cleaved in the endoplasmic reticulum. As we did not know if this cleavage is essential for the antifungal activity, we choose to use either the cut or entire form of this peptide (Levashina et al, 1995). 
					Besides, we wanted to express these peptides only in close vicinity of the fungi. We therefore investigated the possibility to use <I>Bacillus</I> promoters induced by N-Acetyl-Glucosamine (NAG). This molecule is the major component of chitin found at the surface of the fungi.

					<B style="font-size:16px;">Our constructions: </B>
					To simplify the cloning, we designed two genetics modules: Antifungal A and Antifungal B. Antifungal A expresses the cut Metchnikowin and D4E1, while Antifungal B produces entire Metchnikowin, GAFP-1 and Dermaseptin-b1 (figure 4). These two parts were designed to be easily unified as one 5 ORFs operon. For the secretion of these antifungal compounds, we added the AmyE signal peptide in N-terminal of each coding sequences. This peptide is cleaved during the secretion process. The pVeg promoter was used to express the construction during the validation assays. NheI, SacII and SalI restrictions sites were added to further facilitate the modules assembly.
					<CENTER><B style="font-size:12px;">Figure 4: Antifungal operons design and assembly
					</B></CENTER>To express the antifungal peptides only in presence of the fungi, we selected the <I>Bacillus</I> promoter of <I>nagA</I> and <I>nagP</I>, reported to be induced in presence of NAG (Bertram et al., 2011). To valid the expression and specificity of these promoters, the RFP reporter gene has been placed under their control (Figure 5). NheI restrictions sites were added to further facilitate the modules assembly.
					<CENTER><B>Figure 5: pNagA and pNagP constructions 
					</B></CENTER></P></DIV></DIV></DIV><DIV class="column full_size" style="text-align:justify; font-size:14px; color:black; height:100%"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><DIV class="containerHP" style="margin:0px 0px; height:50px; height:100%"><DIV class="panel-group" id="accordion_tool" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="tool_heading3"><CENTER><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion_tool" href="#tool_collapse3" aria-expanded="false" aria-controls="tool_collapse3"><P style="text-align:center">REFERENCES</P></A></CENTER></DIV><DIV id="tool_collapse3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="tool_heading3"><DIV class="panel-body"><P class="lead">	
										Bertram R, Rigali S, Wood N, Lulko AT, Kuipers OP &amp; Titgemeyer F (2011) Regulon of the N-acetylglucosamine utilization regulator NagR in <I>Bacillus subtilis</I>. J. Bacteriol. 193: 3525–3536 

										De Lucca AJ, Bland JM, Grimm C, Jacks TJ, Cary JW, Jaynes JM, Cleveland TE &amp; Walsh TJ (1998) Fungicidal properties, sterol binding, and proteolytic resistance of the synthetic peptide D4E1. Can. J. Microbiol. 44: 514–520 

										Fleury Y, Vouille V, Beven L, Amiche M, Wróblewski H, Delfour A &amp; Nicolas P (1998) Synthesis, antimicrobial activity and gene structure of a novel member of the dermaseptin B family. Biochim. Biophys. Acta 1396: 228–236 

										Levashina EA, Ohresser S, Bulet P, Reichhart J-M, Hetru C &amp; Hoffmann JA (1995) Metchnikowin, a Novel Immune-Inducible Proline-Rich Peptide from Drosophila with Antibacterial and Antifungal Properties. Eur. J. Biochem. 233: 694–700 

										Wang X, Bauw G, Van Damme EJ, Peumans WJ, Chen ZL, Van Montagu M, Angenon G &amp; Dillen W (2001) Gastrodianin-like mannose-binding proteins: a novel class of plant proteins with antifungal properties. Plant J. Cell Mol. Biol. 25: 651–661

									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="column full_size" id="Confinement" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Confinement</P></U><P class="texteb"><B style="font-size:16px;">Context: </B>
					Our project involved release of a genetically modified bacteria in the cave. Lascaux cave is not a completely closed system; there is interaction with external environment due to water infiltration. The risk to release a GMO in the cave is mainly in the horizontal gene transfer to native bacteria. 
					<B style="font-size:16px;">Our solution: </B>
					We decided for a double toxin-antitoxin system (Figure 6). The idea is to prevent plasmid transfer by having two plasmids that have to remain together or cause lethality.  One plasmid contains antitoxin 1 and toxin 2, the other contains antitoxin 2 and toxin 1.
					<CENTER><B style="font-size:12px;">
					Figure 6: concept of the double toxin/antitoxin system.
					</B></CENTER><B style="font-size:16px;">Our constructions: </B>
					The two pairs of Toxin-Antitoxin combinations selected for this project are MazF/MazE (Bravo et al., 1987, Zhang et al., 2005, Wang et al., 2013) and Zeta/Epsilon (Zielenkiewicz and Cegłowski, 2005 ; Mutschler et al., 2011). The toxin and antitoxin are under control of the pVeg constitutive promoter for <I>Bacillus subtilis</I> (BBa_K143012; Figure 5). 
					
					Since each fragment contains a toxin but not its corresponding antitoxin, we needed a strategy to avoid cell death during the cloning steps. We chose to use an unusual theophylline sensitive riboswitch to shutdown expression of the toxin when required:  In the absence of the ligand, the RBS sequence is available and RNA translation is possible. In ligand presence, RNA shape changes and RBS is no more accessible (Figure 7). It is unusual in the sense theophylline riboswitches usually work in the inverse way (Topp and Gallivan, 2008). The SacII and SalI restriction sites were added to simplify the modules assembly.
					<CENTER><B style="font-size:12px;">
					Figure 7: toxin/antitoxin constructions for each plasmid
					</B></CENTER></P></DIV></DIV></DIV><DIV class="column full_size" style="text-align:justify; font-size:14px; color:black; height:100%"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><DIV class="containerHP" style="margin:0px 0px; height:50px; height:100%"><DIV class="panel-group" id="accordion_tool" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading panel-title" role="tab" id="tool_heading2"><CENTER><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion_tool" href="#tool_collapse2" aria-expanded="false" aria-controls="tool_collapse2"><P style="text-align:center">REFERENCES</P></A></CENTER></DIV><DIV id="tool_collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="tool_heading2"><DIV class="panel-body"><P class="lead">	
										Bravo A, de Torrontegui G, and Díaz R (1987). Identification of components of a new stability system of plasmid R1, ParD, that is close to the origin of replication of this plasmid. Mol. Gen. Genet. 210: 101–110. 
										Mutschler H, Gebhardt M, Shoeman RL, and Meinhart A (2011). A novel mechanism of programmed cell death in bacteria by toxin-antitoxin systems corrupts peptidoglycan synthesis. PLoS Biol. 9, e1001033. 
										Topp S. and Gallivan JP (2008). Riboswitches in unexpected places—A synthetic riboswitch in a protein coding region. RNA 14: 2498–2503. 
										Wang X, Lord DM, Hong SH, Peti W, Benedik MJ, Page R, and Wood TK (2013). Type II Toxin/Antitoxin MqsR/MqsA Controls Type V Toxin/Antitoxin GhoT/GhoS. Environ. Microbiol. 15: 1734–1744. 
										Zhang Y, Zhang J, Hara H, Kato I, and Inouye M. (2005). Insights into the mRNA Cleavage Mechanism by MazF, an mRNA Interferase. J. Biol. Chem. 280: 3143–3150. 
										Zielenkiewicz U and Cegłowski P (2005). The Toxin-Antitoxin System of the Streptococcal Plasmid pSM19035. J. Bacteriol. 187: 6094–6105. 
	
									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="column full_size" id="Confinement" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Device</P></U><P class="texteb"> 	
				
					Since the bacteria was genetically modified, it was important to ensure its confinement. Indeed, at this step of the project, the risk about using this GMO in the cave is unknown. This is why we designed a device to physicaly confine our strain for testing and validation purposes.  

					The device has been designed to be stuck on the cave wall and to be easily manipulated.
                                        <CENTER><B style="font-size:12px;">
					Figure 8: Device global aspect
					</B></CENTER>
                                        The description of the device structure and use is available in the <A href="https://2016.igem.org/Team:Toulouse_France/Model">Modeling section.</A></P></DIV></DIV></DIV><DIV class="column full_size" id="Assembly" style="text-align:justify; font-size:14px; color:black;"><DIV style="padding-top:30px; padding-left: 10%; padding-right: 10%;"><DIV class="hoc container clear"><U><P class="title1" id="select1">Assembly</P></U><P class="texteb"> 	
					All the parts from the three modules were designed to be easily assembled (Figure 9). The EcoRI/NheI fragment containing a pNag promoter (figure 5) will have to be swapped with the EcorI/NheI fragment containing the pVeg promoter of the antifungal operon (figure 4). We also designed the toxin/antitoxin systems in a way that one of the toxin gene could be inserted in the middle of the antifungal operon (SacII/SalI insertion, figure 8). The rationale for this was to have the toxin as close as possible to the antifungal molecule to further reduce the risk of antifungal gene horizontal transfer. 

					<CENTER><B style="font-size:12px;"> 
					Figure 9: Final assembly of the parts
					</B></CENTER>Finally, if we decided to use the common <I>Bacillus subtilis</I> 168 strain for the modules validation, the final chassis was choosen to be a double mutant Spo0A- recA-. Indeed, lot of sequences are repeated on our plasmids (pVeg, RBS, terminators and backbone for instance). One possibility was to modify the sequences to reduce the homologies intra and between plasmids, or to find other elements and backbones, but it appears to be too complicated in a short delay. We therefore opted for a recA- strain to reduce the recombination capacity of <I>Bacillus</I>. Besides, we do not want our strain to be able of sporulation since this could jeopardize our capacity to neutralize the strain, hence the choice of a Spo0A mutation.
					And now, it is time for results!
				</P></DIV></DIV></DIV><DIV class="column full_size"><CENTER><A class="button-home" href="https://2016.igem.org/Team:Toulouse_France/Context" style="border: 1px solid #282828;-webkit-border-radius: 5px;-moz-border-radius: 5px;border-radius: 5px; 
			padding: 15px 15px; color: black; text-decoration: none; font-size: 18px; background: none; display: block; width: 250px; background-color:#008B8B">BACK: Context</A><A class="button-home" href="https://2016.igem.org/Team:Toulouse_France/Description" style="border: 1px solid #282828;-webkit-border-radius: 5px;-moz-border-radius: 5px;border-radius: 5px; 
			padding: 15px 15px; color: black; text-decoration: none; font-size: 18px; background: none; display: block; width: 250px; background-color:#3CB371">NEXT: Backbones description</A></CENTER></DIV><DIV class="bgded2" style="background-image:url('https://static.igem.org/mediawiki/2016/4/43/Toulouse_France_PanneauEmpreinte.jpg');"><FOOTER id="footer" class="hoc clear"><DIV style="margin: 20px 40%;  background-color:rgba(1,1,1,0.3);"><H6 class="title" style="color:rgb(255,255,255);">Contacts</H6><UL class="nospace linklist contact" style="color:rgb(255,255,255);"><ADDRESS>
									INSA Toulouse135, Avenue de Rangueil
									31077 TOULOUSE Cedex 4
									France</ADDRESS><LI>☎ +00 (33) 6 13 64 89 40</LI><LI>✉ igemtoulouse2016@gmail.com</LI></UL></DIV></FOOTER><DIV id="copyright" class="hoc clear"><CENTER><P style="text-align:center;">Website by <A href="https://2016.igem.org/Team:Toulouse_France">Team iGEM Toulouse 2016</A></P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>