<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse_France_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse France/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Toulouse 2016</TITLE></P><DIV id="navi"><DIV id="topbar" class="hoc clear"><P>Follow us  @</P></DIV><HEADER id="header" class="hoc clear"><NAV id="mainav" style="max-width:auto; margin:0 auto;"><UL class="clear"><CENTER><LI><A href="https://2016.igem.org/Team:Toulouse_France">Home</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team#Presentation">Presentation</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Collaborations">Collaborations &amp; Meet Up</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Team#Gallery">Gallery &amp; Fun Facts</A></LI></CENTER></UL><LI><A href="https://2016.igem.org/Team:Toulouse_France/Context">Project</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Predation">Predation</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Antifungals">Antifungals</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Confinement">Confinement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Confinement">Biological confinement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Design#Device">Physical confinement : Device</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Description">Backbones description</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Experiments">Results</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Model">Modeling</A></LI><LI>Wet Lab
						</LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Composite_Parts">Parts</A></LI><UL style="text-align:left;"><LI><A href="https://2016.igem.org/Team:Toulouse_France/Composite_Parts">Composite Parts</A></LI></UL><LI><A href="https://2016.igem.org/Team:Toulouse_France/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Human_Practices">Ethics and Women in Science</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Attributions#Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Supports">Supports</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Medias">Medias</A></LI><LI><A href="https://2016.igem.org/Team:Toulouse_France/Medal_Requirements">Achievements</A></LI></NAV></HEADER></DIV><DIV class="page" style="background-image:url('https://static.igem.org/mediawiki/2016/4/4b/Toulouse_France_LascauxFresqueTaureaux.png');"><DIV id="pageintro" class="hoc clear" style="padding:300px 0px;"><P class="sec_title" style="background-color:rgba(1,1,1,0.5);">Results</P></DIV></DIV><DIV class="column full_size" id="predation" style="background-color:#F5F5F5;  text-align:justify; padding:10px 10%;"><U><P class="title1" id="select1">Predation</P></U><P class="texteb"> 
			
			Our aim is to reinforce the natural predation capacity of <I>B. subtilis</I> and to ensure it is expressed independantly of the conditions. We

			first assessed that our wild type <I>Bacillus</I> chassis is not able of predation, then we built the operons

			allowing boosting the predation property.
			<U><B>Preliminary tests:</B></U>We tried different testing approaches to evaluate the predatory response of <I>B. subtilis</I> and eventually elaborate a protocol to do the preliminary tests. We tested the predation of <I>B. subtilis</I> Wild Type (strain 168) against <I>Pseudomonas fluorescens</I> (strain SBW25), a deleterious strain present in the cave. Briefly, the protocol consists in growing both strains in rich medium, mixing them in PBS and monitoring their growth (figure 1). 
		</P><P><CENTER><B>Figure 1: <I>Bacillus subtilis</I> WT 168 does not feed on <I>Pseudomonas fluorescens</I> SBW25. Both strains were grown overnight in rich medium and then mixed in PBS. The growths of the strains were then monitored during 8 hours by plate numeration. The graph represents the ratio between <I>B. subtilis</I> in PBS in presence of <I>P. fluorescens</I>  versus <I>B. subtilis</I> alone in PBS (data normalized to time 1H).				
			</B></CENTER></P><P class="texteb"> 	
			We observed no growth benefit when mixing <I>B. subtilis</I> and <I>P. fluorescens</I> compared to <I>B. subtilis</I> alone. We conclude that <I>B. subtilis</I> WT predation program is not the strain priority when facing starvation. Other surviving program as competence or sporulation are likely favoured by <I>B. subtilis</I> in such condition. This reinforces the need to prevent these programs by using a <I>spo0A</I> mutant and to promote the predation by overexpressing either the SKF or SDP operons.
			<U><B>SKF</B></U>
			This predation operon is composed of seven genes for a total of more than 6 kb. To get rid of restriction sites that could interfere with the cloning steps, we ordered the optimized sequences from IDT as four gblocks. From there, our strategy was to do Gibson cloning to obtain the full operon in pSB1C3 in <I>E. coli</I> and then to transfer it in <I>B. subtilis</I>. However, we did not manage to obtain the whole assembly (figure 2), neither partial ones, in spite of about 20 attempts…
		</P><P><CENTER><B>Figure 2 : Layout of SKF expected biobrick.	
			</B></CENTER></P><P class="texteb"><U><B>SDP</B></U>
			The SDP operon is smaller than the SKF one and it was possible to obtain the optimized sequences as two gblocks. Here again, we were unfortunate and did not get the expected clones in <I>E. coli</I> (figure 3). 
		</P><P><CENTER><B>Figure 3 : Layout of SDP expected biobrick.
			</B></CENTER>				

			To perform trouble shooting, we tried an assembly test with just the two gblocks and deposited the product on gel. We observed that the reaction seems to be effective with the presence of a new band corresponding to the combined size of the two gblocks (figure 4). 

			<CENTER><B>Figure 4: Gibson assemby of the two SDP Gblocks.
			</B></CENTER></P><P class="texteb"><U><B>Conclusions and perspectives</B></U>
			It seems our Gibson step is fine since we managed to obtain the SDP assembly, but we could not get <I>E. coli</I> transformants when performing the whole experience. The predation system is based on the production of toxins by <I>B. subtilis</I>, and these toxins were reported to be harmful to <I>E. coli</I> (Nandy et al., 2007, FEBS Letters. 581: 151–56). An explanation to our problems could be that SDP and SKF cloning in <I>E. coli</I> results in the bacterium death. We had thought about this problem, but we had believed the expression driven by the pVeg <I>Bacillus</I> promoter to be insufficient for such effect. Perspectives could be to use a tightly regulated promoter to prevent expression during the cloning step in <I>E. coli</I>, or to try a direct transformation of highly competent <I>Bacillus</I> strain.
		</P></DIV><DIV class="column full_size" id="antifongique" style="background-color:#F5F5F5; text-align:justify; padding: 10px 10%"><P class="title1" id="select1">Antifungals</P><P class="texteb">
			Here, we aimed to produce a cocktail of five antifungal peptides whose production in <I>Bacillus subtilis</I> will be triggered by presence of fungi.
			<U><B style="font-size:16px;">Operon constructions:</B></U>
			The whole antifungal operon was too big to be synthesized by IDT as one gblock. We therefore decided to divide it in two operons (figure 5), each of them with a promoter to be functional, with the possibility to eventually combine them. The sequence were optimized for the <I>Bacillus</I> codon usage and to remove inadequate restriction sites. Sub-cloning of the first operon (containing cut version of the Metchnikowin and D4E1) on the pSB1C3 backbone was rapidly performed, leading to the new composite part <A href="http://parts.igem.org/Part:BBa_K1937007">BBa_K1937007 (pSB1C3-AF_A)</A>. However, we did not manage to obtain the second operon in the pSB1C3 (encoding Dermaseptin B1, GAFP-1 and entire Metchnikowin antifungal peptides). We tried to directly sub-clone the gblock in the pSB1C3-AF_A but without success. We hypothesize that one of the peptide could be toxic for <I>E. coli</I>. This will have to be verified by sub-cloning the 3 peptides alone. The AF_A operon was subsequently cloned in the pSB<SUB>BS</SUB>0K-Mini plasmid to create biobrick <A href="http://parts.igem.org/Part:BBa_K1937008">BBA_K1937008</A>.
		</P><P><CENTER><B>Figure 5: Layout of antifungal operons and their assembly.
			</B></CENTER></P><P class="texteb">
			In order to express specifically the antifungal peptides in close vicinity to fungi, we choose the two N-acetyl-glucosamine (NAG) inducible promotors pNagA and pNagP. The constructions with the RFP reporter gene were ordered from IDT and successfully sub-cloned in the pSB1C3 (new parts <A href="http://parts.igem.org/Part:BBa_K1937003">BBa_K1937003</A> and <A href="http://parts.igem.org/Part:BBa_K1937005">BBa_K1937005</A> ; figure 6). They were subsequently cloned in the pSB<SUB>BS</SUB>0K-Mini plasmid to create biobricks <A href="http://parts.igem.org/Part:BBa_K1937004">BBA_K1937004</A> and <A href="http://parts.igem.org/Part:BBa_K1937006">BBa_K1937006</A>.
		</P><P><CENTER><B>Figure 6: Layout of the pNag-RFP constructions.
			</B></CENTER></P><P class="texteb"><U><B>pNag validation</B></U>
			We tested the expression and specificity of the RFP driven by pNagA and pNagP when growing in presence of glucose or NAG (figure 7). We observed a late and rather specific RFP expression on NAG. The late expression could mean that the formulation of our minimal medium is not optimal. The fact that the pNagA-RFP and pNagP-RFP strains seem able to slightly express the RFP on glucose (figure 7B, left panel close-up), albeit on weaker extend that on NAG (figure 7B, right panel close-up), could be due to the alleviating of the catabolic repression. 
			In conclusion, pNagA and pNagP appear as able to promote expression in response to NAG, even if the growth conditions could be improved to get higher and more homogeneous expression levels.

		</P><P><CENTER><B>Figure 7: NAG-driven expression of RFP. <I>B. subtilis</I> strains transformed with pSB<SUB>BS</SUB>0K-Mini (Control), pSB<SUB>BS</SUB>0K-Mini-NagA or pSB<SUB>BS</SUB>0K-Mini-NagP were spread on minimal medium with either glucose or NAG as carbon source. Red spots appeared only with pNagA or pNagP on NAG (close-ups on part 7B).
			</B></CENTER></P><P class="texteb"><U><B>Antifungal validation</B></U>
			We found out that the best culture conditions for the fungi that permits a slight growth of <I>Bacillus</I> were with ¼ PDA and 2% glucose. We tested different fungi (<I>Aspergillus niger, Talaromyces funiculosus</I> and <I>Chaetomium globosum</I>) but we eventually focussed on <I>Talaromyces funiculosus</I> that seems easier to manipulate to us.

			Our test consisted in adding, on fungi inoculated plates, paper patches soaked with either copper sulfate (positive control), LB medium (negative control), a suspension of <I>Bacillus subtilis</I> WT or <I>Bacillus subtilis</I> expressing the antifungal AF_A operon (figure 8). We observed that with our construction, a slight inhibition halo appeared around the patch. This effect is visible even after 8 days and was reproducible. These observations allow us to conclude that AF_A is functional.
		</P><P><CENTER><B>Figure 8: Antifungal tests (legend in the text).
			</B></CENTER></P><P class="texteb"><U><B>Test on the rock</B></U>
			As our therapeutic bacterium was supposed to treat fungi growing on the walls of a cave, we needed to test its activity in conditions that would mimic the cave’s environment. The experimental model we thought about was to test our modified bacteria on fungi artificially grown on rocks. The first step was therefore to be able to grow fungi on rocks.
			In order to do so, we have deposed on the surface of the rocks growth media with various nutriment compositions (see Table 1). The red color of the spots was due to ochre, whose purpose was to mimic the frescoes of the cave (See Figure below). The spots 1 to 3 contain various concentrations of glucose, the spots 4 to 6 various concentrations of tryptone and yeast extracts, whereas the spots 7-8 various concentrations of glucose, tryptone and yeast extract. 
			The fungi were then inoculated on each spot whereas our therapeutic agent only on Spot 8.
		</P><P class="texteb"><U><B>Results</B></U></P><P><CENTER><B>Figure 9: Test on the rock inoculated with <I>Talaromyces funiculosum</I> at T=0 (on the left) and T=3 weeks post infection (on the right).
			</B></CENTER></P><P class="texteb">					
			After 3 weeks, the growth of fungi was clearly visible on spots 1 to 6, with the most efficient growth on the spot 3 which had the following medium composition:
			<UL><LI>1g Glucose </LI><LI>1g Yeast Extract </LI><LI>1g Tryptone </LI></UL>
			Interestingly, no growth of the fungus was observed on spots 7 and 8. As the later one contained our therapeutic agent, this observation suggested that our modified bacterium might be active against the fungi. However, the absence of growth on Spot 7 would argue against this conclusion. It is therefore clear that this result needs to be confirmed by repeating the experiment several times. However, the ability to get the fungi grown on rock was already a good result, indicating that we might have a good model to test our bacteria.   
		</P><P class="texteb"><U><B>Conclusions and perspectives</B></U>Here, we showed that our pNagA and NagP parts are able to control gene expression in response to NAG and that the first part of our antifungal operon is functional. In both cases, the properties will have to be optimized, through a higher and more homogeneous expression from the NAG-driven promoters and through the completion of the antifungal operons to produce more than two antifungal peptides.
			We were able to set up a model which mimics the cave's environment. Thanks to it, we got encouraging results showing that the therapeutic agent might be functional.
			</P></DIV><DIV class="column full_size" id="confinement" style="background-color:#F5F5F5;  text-align:justify; padding: 0px 10%"><P class="title1" id="select1">Containement</P><P class="texteb">			

			Here, we fashioned a genetic system to prevent horizontal transfer of our synthetic constructions. 
			<U><B>Toxin/antitoxin systems constructions</B></U>
			The constructions were ordered as gblocks from IDT. The Epsilon/MazF construction was rapidly sub-cloned in the pSB1C3 backbone (new composite part <A href="http://parts.igem.org/Part:BBa_K1937009">BBa_K1937009</A>), and then in the pSB<SUB>BS</SUB>0K-Mini plasmid to create biobricks <A href="http://parts.igem.org/Part:BBa_K1937010">BBa_K1937010</A> (figure 10). However, we never managed to get the MazE/Zeta construction in the pSB1C3 backbone. Again, we can only speculate about the toxicity of the toxin.
		</P><P><CENTER><B>Figure 10: Layout of the toxin/antitoxin operons.
			</B></CENTER></P><P class="texteb"><U><B>Theophylline validation</B></U>
			To validate the theophylline riboswitch, we inferred that we should obtain clones of <I>Bacillus subtilis</I> transformed with the pSB<SUB>BS</SUB>0K-Mini –Epsilon/MazF only in presence of theophylline: the molecule should prevent the expression of the MazF toxin that is lethal since the antitoxin MazE is not present. Unfortunately, we did not get any clone, neither without nor with theophylline (figure 11). 				
		</P><P><CENTER><B>Figure 11: Result of the <I>Bacillus subtilis</I> transformation with pSB<SUB>BS</SUB>0K-Mini –Epsilon/MazF (we know this is not the most illustrative figure ever!).
			</B></CENTER></P><P class="texteb"><U><B>Conclusions and perspectives</B></U>
			At this step, we can only hypothesize that our system is leaking sufficient expression of the toxins for them to be lethal, either in <I>E. coli</I> or in <I>B. subtilis</I>. Further assays using inducible promoters will be necessary to set up the system without enduring these toxicity problems.
		</P></DIV><DIV class="column full_size" style="background-color:#F5F5F5;"><CENTER><A class="button-home" href="https://2016.igem.org/Team:Toulouse_France/Description" style="border: 1px solid #282828;-webkit-border-radius: 5px;-moz-border-radius: 5px;border-radius: 5px; 
			padding: 15px 15px; color: black; text-decoration: none; font-size: 18px; background: none; display: block; width: 250px; background-color:#3CB371">BACK: Backbones description</A><A class="button-home" href="https://2016.igem.org/Team:Toulouse_France/Model" style="border: 1px solid #282828;-webkit-border-radius: 5px;-moz-border-radius: 5px;border-radius: 5px; 
			padding: 15px 15px; color: black; text-decoration: none; font-size: 18px; background: none; display: block; width: 250px; background-color:#FF6347">NEXT: Modeling</A></CENTER></DIV><DIV class="bgded2" style="background-image:url('https://static.igem.org/mediawiki/2016/4/43/Toulouse_France_PanneauEmpreinte.jpg');"><FOOTER id="footer" class="hoc clear"><DIV style="margin: 20px 40%;  background-color:rgba(1,1,1,0.3);"><H6 class="title" style="color:rgb(255,255,255);">Contacts</H6><UL class="nospace linklist contact" style="color:rgb(255,255,255);"><ADDRESS>
									INSA Toulouse
									135, Avenue de Rangueil
									31077 TOULOUSE Cedex 4
									France</ADDRESS><LI>☎ +00 (33) 6 13 64 89 40</LI><LI>✉ igemtoulouse2016@gmail.com</LI></UL></DIV></FOOTER><DIV id="copyright" class="hoc clear"><CENTER><P style="text-align:center;">Website by <A href="https://2016.igem.org/Team:Toulouse_France">Team iGEM Toulouse 2016</A></P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>