"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo Tech/cph
From 2009.igem.org
Main
Team
Terraforming
Experiments
Parts
Safety
Contents
1 Achievement
2 Introduction ~Why do we need photo sensor~
3 Material and method
3.1 The mechanism of photo-sensor
3.2 Experimental overview
3.3 The protocol for the lacZ assay
4 Result
4.1 lacZ assay results for the constructed parts
4.2 measuring the absorbance spectra of PCB
4.3 Replication study for the previous iGEM part
5 Discussion
6 Reference
Achievement
We conducted the replication study for the previous iGEM part : BBa_K098010
Introduction ~Why do we need photo sensor~
Mars also has day and night like Earth. The strength of the sunlight reaching Mars varies from day to night. During the day, the Mars’ surface temperature is about at most -20 degrees Celsius and in the night it drops to -100 degrees Celsius, so it natural that E. coli should behave differently in the day and night. Photo-sensor can be one powerful tool to control or regulate gene expression that is suitable for the temperature. For example, we introduced anti-freeze protein into E. coli, and it should express an appropriate amount of the protein based on the surface temperature, which is in proportion to the strength of light. We believe that this sensor can also be applied and utilized for terra-forming in some other way.
Material and method
The mechanism of photo-sensor
Figure 1 : the mechanism of photo-sensor
Figure 1 shows the mechanism of photo-sensor. Cph8 consists of cph1 and envZ. Cph1 is an extracellular sensor and cph1 bound to PCB (converted from heam by ho1 and pcyA) responds to the light. EnvZ is an intracellular response regulator and has a histidine kinase domain that phosphorylates ompR protein so that it can regulate gene expression. When cph8 is not exposed to the light, the histidine kinase domain phosphorylates ompR, and then ompR-phosphate activates the PompC by binding to it. When cph8 is exposed to the light, the histidine kinase can’t phosphorylate ompR so PompC is inactive.
Experimental overview
We attempted to construct Biobrick coding cph8 and PCB-related enzymes, assay them, and characterize them. We used the strain, Ru1012 [MC4100 ara+ φ(OmpC-lacZ) 10-25 ΔenvZ::KanR], which chromosomally codes a ompC promoter fused to a lacZ reporter. That strain enables us to assay the activity of cph8 and PCB-related enzymes easily. Origin and resistance on the backbone is the most significant matter we have to consider, because we planned to transform Ru1012, putting two plasmids into it and the origins and resistances of the two plasmids should be different. Because Ru1012 codes KanR, we have to prepare two kinds of backbones. One codes AmpR, and the other codes Cm respectively.
The protocol for the lacZ assay
To quantitatively determine the performances of photo-receipting parts, we measured the amount of β-galactsidase expressed from lacZ reporter.
  1.  Diluted overnight cultures of photo-receipting strains grown at 37℃ into 1:1000 2mL LB medium containing appropriate antibiotics in        two 24-well plates
  2.  Incubated two plates at 37℃ for 5 hours, one was exposed with 660nm LED lamp  glowing at 7W, and the other was wrapped in        aluminum foil and kept in darkness
  3.  Diluted each strain into 1:100 in new wells
  4.  Incubated two plates at 37℃ for another 5 hours, one was exposed with LED lamp, and the other was wrapped in aluminum foil and kept        in darkness
  5.  Spun 2mL of the final culture at 6000 rpm for 10 minutes in a microcentrifuge and then resuspended in 2mL of Z buffer
  6.  Recorded the cell density by measuring the OD600 using spectrophotometer
  7.  Diluted 0.5mL of the resuspended cells into 1mL in Z buffer and permeabilized by the addition of 100µL of chloroform and 50µL of 0.1%        SDS
  8.  vortexed and allowed to equilibrate at 28℃ for 5 minutes
  9.  0.2mL of a solution containing 4.g/mL ONPG was added to the permeabilized cells
10.  Recorded the time when sufficient yellow color developed to be seen by eye, and stop the reaction by the adding 0.5mL of 1M Na2CO3         with vortexing
11.  Removed the chloroform by centrifuging 1mL of the quenched reaction at 13200 rpm for 5 minutes
12.  Measured OD420 and OD550
13.  Calculated using the formula :
v = the volume of culture used in the assay
 Z butter was prepared as follows :
H2O                           500[mL]
Na2HPO4・12H2O  10.75[g]
NaH2PO4・2H2O     3.06[g]
KCl                             0.375[g]
MgSO4・7H2O          0.123[g]
DTT                            1[mL]
Result
There are several solutions we came up with for this problem.
lacZ assay results for the constructed parts
First, we tried to change the backbone of cph8 into pSB6A and PCB-related into pSB3C. So we decided to put both plasmids into pSB6A and assay them respectively using the two given plasmids. One codes cph8 with ColE1 origin and CmR. The other codes PCB-related with p15A origin and AmpR. Also we used these plasmids, pBR322 with colE1 origin and AmpR and pACYA184 with p15A origin and CmR. We conducted assay in the following plasmids combination.
1. gcph(the backbone is changed into pACYC184)+PCB(we constructed)
2. cph(we constructed)+gPCB(the backbone is changed into pACYC184)
3. gcph(the backbone is changed into pACYC184)+gPCB(the backbone is changed into pBR322)
4. gcph(the backbone is changed into pBR322)+ gPCB(the backbone is changed into pACYC184) 1 and 2 were aimed at assaying if the     parts we made work or not. 3 and 4 were aimed at assaying if the parts we were given work or not.
The following gragh is the result.
Graph 1 : lacZ assay result of the constructed parts
Shown in this graph 1, the BioBrick parts we’ve made, in the 4 columns in the right, worked either the 660 nm LED light exposed or not. This result describes that the part we made didn’t work.
measuring the absorbance spectra of PCB
Next we tried to measure the absorbance spectra of PCB.
graph 2 : the absorbance spectra of PCB
Graph 2 describes that oth the sample(PCB) and positive control didn’t show high value near 660 nm. Maybe the consistency of PCB was too low to detect. However, we didn’t have time to purify it and assay it again.
Next we tried to measure the absorbance spectra of PCB.
Graph 2 describes that oth the sample(PCB) and positive control didn’t show high value near 660 nm. Maybe the consistency of PCB was too low to detect. However, we didn’t have time to purify it and assay it again.
Replication study for the previous iGEM part
We conducted assay for BBa_K098010, the iGEM part Harvard 2008 constructed. The following is the result.
Graph 3 : lacZ assay result for the replication study
Discussion
Reference
Synthetic biology: engineering Escherichia coli to see light, Levskaya A, nature, 2000
A short course in bacterial genetics : a laboratory manual and handbook for Escherichia coli and related bacteria, Jeffrey H. Miller, pp72-74
Retrieved from " http://2009.igem.org/Team:Tokyo_Tech/cph "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
