<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Electrochem"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Electrochem">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Electrochem&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Electrochem&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Electrochem&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Electrochem" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Electrochem</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Notebook</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Killswitch">Killswitch</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/PromoterScreen">Promoter Screen</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Electrochem">Electrochemistry</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Biosensor">Biosensor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/CatecholDegradation">Catechol</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/FluxAnalysis">Flux Analysis</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocol Manual</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Electrochemistry Notebook</H1><H2>Week 1 (May 1-4)</H2><P>This week involved planning various experiments and gathering chemicals to be tested. Unfortunately no lab work was conducted this week.</P><H2>Week 2 (May 7-11)</H2><P>Chlorophenol red (CPR) was oxidized in a sodium phosphate solution with zinc phthalocyanine modified carbon electrodes or just straight carbon electrodes. It was found that the modified electrodes had greater responses than their unmodified counterparts. All future work with CPR will be done with these electrodes. An IPTG induced <I>lacZ</I> was transformed for the registry and the <I>uidA</I> gene was amplified and biobricked under the control of the IPTG promoter. These genes will be used to cleave specific sugars from electrochemical analytes.</P><H2>Week 3 (May 14-18)</H2><P>A standard curve was generated using CPR as the analyte in a sodium phosphate solution. This was made with the modified electrodes. After this it was found that the reference electrode being used, a silver/ silver chloride electrode, had stopped functioning. Switching to a reduction of hydrogen electrode (RHE) reference fixed this problem. A gold working electrode was also briefly tested to reduce the background noise caused by the capacitance of carbon. Further testing will be needed to choose a final electrode.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 1: Standard curve for the oxidation of CPR at a carbon working electrode in 25mL pH7 0.1M PBS with Argon bubbling. The reference electrode was RHE and the counter electrode was platinum.</DIV></DIV></DIV></DIV><H2>Week 4 (May 21-25)</H2><P>This week involved testing the oxidation of CPR and para-aminophenol (PAP) in the same solution to determine if their oxidation potentials were unique. By showing that they are separate peaks it demonstrates that they can be used as two of the electrochemical components of a multiplexed biosensor. After a morning of failed tests on Thursday we finally managed to show that the peaks are unique (0.7V for PAP and 1.3V for CPR vs RHE), giving us the first step towards a final sensor.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 2: Electrochemical detection of CPR and PAP in the same cell. PAP has the reversible wave seen at 0.75V vs RHE, while CPR has the irreversible wave at 1.3V vs RHE. These values were recorded in 25mL pH7 0.1M PBS with Ar bubbling on a carbon working electrode with a platinum counter electrode and a RHE as the reference electrode.</DIV></DIV></DIV></DIV><H2>Week 5 (May 28- June 1)</H2><P>This week the data collected was inconsistent across multiple runs even though the electrodes used were from the same batch. It seems that each of these electrodes is unique, presenting a challenge for analyzing results when multiple electrode setups are used.</P><H2>Week 6 (June 4-8)</H2><P>This week marked the start of the construction of a mathematical model of the electrochemical system. The goal of this model is to identify the shortest time in which an electrochemical signal could be reached and what conditions would decrease this detection time. To do this the MATLAB toolbox SimBiology will be used as well as differential solvers and published rate constants. The model will include translation, transcription, RNA and protein degradation, and the actual conversion from the sugar conjugate into the electrochemical analyte.</P><H2>Week 7 (June 11-15)</H2><P>The list of final analytes was compiled this week, giving a goal of chemicals to test with and without their sugar conjugates. The genes necessary to break these compounds apart were also noted and primers were designed so that they could be amplified. More work was placed on the model as well, getting it to the point where it can go from a constant amount of plasmid DNA into the RNA transcript and then the protein product.</P><H2>Week 8 (June 18-22)</H2><P>The model has been updated to include the transport of PAPG, the sugar conjugated form of PAP, into the cell through the LacY transport protein. The diffusion of PAP across the membrane out into the solution was also included into the model. A diagramatic representation of this is shown below. The last of the chemicals needed for the electrochemical testing was ordered this week too, meaning that work can begin at an increased pace on the wetlab front when the shipment arrives.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 3: Diagramatic representation of the mathematical model of electrochemical detection. PAPGE and PAPE stand for PAPG and PAP external to the bacterial cell. Green circles are enzymatic reactions and red circles are degradation reactions.</DIV></DIV></DIV></DIV><H2>Week 9 (June 25-29)</H2><P>Results from the model have been obtained, hinting at a detection time of approximately 250 seconds, or just over 4 minutes. This is based on the detection of PAP at 0.06mM, which is equivalent to the 1500 μL of PAP sweep shown in Figure 2. The sequencing results from the <I>lacZ</I> and <I>uidA</I> genes have arrived and the <I>lacZ</I> construct is fine while the <I>uidA</I> construct has some mutations.</P><H2>Week 10 (July 3-6)</H2><P>A standard curve was attempted for PAP, however the graphs recorded resembled a solution of intense resistivity. A possible explanation for this is the clip connecting the potentiostat to the electrode having an incomplete connection. An electrode with a stronger connection was constructed and tested to remedy this, however it presented no detection of the PAP at any concentration. Switching to a platinum working electrode showed a strong response and testing will continue with this next week.</P><H2>Week 11 (July 9-13)</H2><P>Testing continued using platinum to detect the presence of PAP in a solution of PBS. Problems were encountered with the conductivity of the testing solution, but were remedied through the preparation of a new batch of PBS. One other concern was that the electrodes were becoming coated in various chemicals from the solution and thus degrading the strength of the signal obtained. To clean the electrodes they were submerged in a solution of concentrated H<SUB>2</SUB>SO<SUB>4</SUB> for two days. On the genetics side the <I>bglX</I> gene was biobricked and sent for sequencing this week, giving us that last brick for the electrochemistry project for now.</P><H2>Week 12 (July 16-20)</H2><P>Using the cleaned platinum electrode a standard curve of PAP was attempted however no results were obtain. After consulting with multiple graduate students and professors in electrochemistry it was discovered that the reason for this could be the solubility of PAP. With this in mind the PAP system was shelved for the time being until it can be brought into solution. Moving on to the chemical PDP (para-diphenol, commonly called hydroquinone) and PNP (para-nitrophenol), two other chemicals with specific sugar conjugates to be tested. These two compounds are more soluble than PAP and went into solution immediately. A quick run with both chemicals was conducted to determine if they were detectable, and it appeared that their oxidation potentials were around 1.0V and 1.6V vs RHE and that they were easily detectable.</P><H2>Week 13 (July 23-27)</H2><P>More characterization was done on PDP, including a potentiostatic run. This type of run is where the working electrode is held at a certain potential, in this case the oxidation potential of PDP, and as more of the analyte is added the current increases proportionally. The benefit of this kind of testing is the increased speed as there is no need to sweep over potentials where the compound would not react.</P><H2>Week 14 (July 30- Aug 3)</H2><P>Using the potentiostatic method PDP was consistently detected at 0.825V and 0.85V vs RHE with no detection of the sugar conjugate PDPG (para-diphenol-B-D-glucopyranoside) at this potential. As the graphs were cleaner at 0.825V vs RHE this potential will be used for future analysis of PDP and Beta-Glucosidase activity. A graph of potentiostatic testing at 0.825V vs RHE is shown below. Testing with PNP shows that the oxidation potential is between 1.5 and 1.7V.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 4: Potentiostatic run at 0.825V vs RHE with additions of 0.4μM every 60 seconds after a 5 minute current stabilization period at the beginning of the experiment. At the end of the graph PDPG was added twice and caused no increase in current.</DIV></DIV></DIV></DIV><H2>Week 15 (Aug 6-10)</H2><P>Potentiostatic runs of PDP were further characterized and runs on PNP at 1.5V and 1.6V vs RHE were started. PNPG (para-nitrophenol-B-D-glucorinide) was tested for any electrochemical reactions from 0-2V vs RHE but no reactions were observed. As this was the case no effect was observed during the potentiostatic runs with the addition of PNPG.</P><H2>Week 16 (Aug 13-17)</H2><P>This week some testing was conducted on the Biosensor prototype in parallel with the potentiostat usually used for the electrochemistry experiments. It turns out that the prototype only works with the blue carbon strip electrodes and cannot perform experiments with other electrodes. The oxidation potential for PDP was tested on these electrodes and determined to be -0.1V vs the pseudo-Ag/AgCl reference electrode on the strips. This was confirmed through potentiostatic detection at this voltage.</P><H2>Week 17 (Aug 20-24)</H2><P>PNP was tested on the blue strip electrodes and it's oxidation potential was determined to be at around +0.6V vs the pseudo-Ag/AgCl reference electrode. The potentiostat prototype was able to detect this chemical at the same oxidation potential when using the blue strip electrodes. Some software glitches were encountered but they were resolved by the end of the testing day.</P><H2>Week 18 (Aug 27-31)</H2><P>ONP (ortho-nitrophenol) was tested as a substitute for PAP. No discernible detection was noted during initial testing. More work was done to bring the prototype up to the same detection thresholds as that of the commercial potentiostat.</P><H2>Week 19 (Sept 3-7)</H2><P>The genetic circuit with <I>uidA</I> under the control of the IPTG-inducible <I>lacI</I> promoter was confirmed through sequencing this week. More work is still needed on the circuits for <I>lacZ</I> and <I>bglX</I>, as the Registry <I>lacZ</I> has a frameshift mutation, and the <I>bglX</I> gene has proven difficult to work with due to a recently discovered pstI cut site in the gene.</P><H2>Week 20 (Sept 10-14)</H2><P>Testing was done on the <I>uidA</I> circuit and a response was observed in the presence of IPTG. This is shown below in Figure 5. The experiment was repeated with and without IPTG to detect any leaky expression.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 5: Potentiostatic detection of PNP at 1.6V vs RHE by either an IPTG induced <I>uidA</I> gene or leaky expression of an uninduced <I>uidA</I> gene.</DIV></DIV></DIV></DIV><H2>Week 21 (Sept 17-21)</H2><P>This week work on a new <I>lacZ</I> substrate was halted and testing with CPR resumed. Due to limited time before the competition it was deemed best to use a working compound rather than to explore new options. Experiments mostly focused around confirming old data.</P><H2>Week 22 (Sept 24-28)</H2><P>A lot of coding was done on the wiki this week as the wiki freeze is just around the corner. Tests were performed on a suspected <I>bglX</I> culture but no response was observed. A new circuit is being constructed under the control of the constitutive <I>tetR</I> promoter to see if a response can be generated.</P><H2>Week 23 (Oct 1-23)</H2><P>WIKI FREEZE!!! Lots of coding was done to get the wiki to the point where it was presentable to the judges. Testing was also done with <I>lacZ</I> and CPRG to show that a response is possible. The constitutive <I>bglX</I> was also tested and you can find the results on our <A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">electroreporting page.</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Notebook/Electrochem">http://2012.igem.org/Team:Calgary/Notebook/Electrochem</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Electrochem" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Electrochem" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Electrochem&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Electrochem&amp;oldid=298224" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>