<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_OSCAR_Desulfurization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/OSCAR/Desulfurization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/OSCAR/Desulfurization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/OSCAR/Desulfurization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/OSCAR/Desulfurization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/OSCAR/Desulfurization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/OSCAR/Desulfurization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Project</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Jamboree</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Desulfurization</H1><H2>Why Remove Sulfur?</H2><P align="justify">
Sulfur is the third most abundant element in crude oil (Ma, 2010), and when sulfur containing hydrocarbons are burned they release S0<SUB>2</SUB> and S0<SUB>3</SUB> gasses into the atmosphere. Not only does this reduce the efficiency and value of our product, but it also contributes to global warming, acid rain, and various health issues due to the pollution (Reichmuth <I>et al</I>., 2000). Strict regulation on sulfur in fuels are now in place and low-sulfur gasoline is mandated across all of Canada (Source: Environment Canada). To upgrade the quality of our fuel we need to remove the sulfur but keep the hydrocarbon backbone.</P><H2>Our Vision</H2><P align="justify">
Though a few pathways for biodesulfurization exist in the microbial world, most involve the destruction of part of the carbon skeleton (an example would be the Kodama pathway)(Soleimani <I>et al</I>., 2007). This would effectively reduce the quality of our product. With this in mind the pathway we have chosen is the 4S pathway found in <I>Rhodococcus spp</I>. It has been characterized and shown to remove sulfur from the model substrate dibenzothiophene (DBT) and convert it to 2-hydroxybiphenyl (2-HBP) in a non-destructive manner. DBT and its derivatives make up 70% of the organic sulfur compounds found in crude oil (Ma 2010), and are also some of the most difficult to remove through chemical means. By using the 4S pathway we will be able to upgrade our fuel and remove recalcitrant compounds at the same time. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption">Figure 1: The 4S Desulfurization Pathway, showing the desulfurization of the model compound DBT by DszA, DszB, DszC, and DszD.</DIV></DIV></DIV></DIV><H2>4S pathway</H2><P align="justify">
Four enzymes are involved in the 4S pathway, 3 of which are directly involved in the conversion of DBT to 2-HBP. Dibenzothiophene monooxygenase (DszC) is responsible for the first two steps of the pathway, converting DBT to DBT-sulfoxide and finally to DBT-sulfone (DBTO<SUB>2</SUB>) through the addition of 2 oxygen atoms to the sulfur atom. DBT-sulfone monooxygenase (DszA) then carries out the next step in the pathway, producing 2-hydroxybiphenyl-2-sulfinic acid (HBPS) through addition of a final oxygen to the heteroatom. This causes cleavage of the chemical bonds at the sulfur, breaking the ring and converting the compound from a 3-ring structure to a 2-ring structure. HBPS is then converted to the final product of the 4S pathway by HBPS desulfinase (DszB), producing 2-HBP. At this point, the sulfur has been released from the hydrocarbon in the form of sulfite.</P><P align="justify">
The first three steps of the 4S pathway require FMNH<SUB>2</SUB> and subsequently reduces the reductive power of the cell. WIn order to regain this power an oxidoreductase (DszD) uses NADH to recycle the FMNH<SUB>2</SUB>, allowing the reaction to proceed. Without DszD the desulfurization pathway would grind to a halt.</P><P align="justify">
The <I>dszA</I>,<I>B</I>, and <I>C</I> genes form an operon on the pSOX plasmid of <I>R. erythropolis</I>, while <I>dszD</I> is found in the chromosome. Naturally this pathway is slow, however using synthetic biology approaches this process can be optimized.</P><H2>Our Approach</H2><H3>1) Find the genes!</H3><P align="justify">We isolated the plasmid containing the <I>dsz</I> genes from a desulfurising environmental isolate of <I>Rhodococcus</I> using a <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/plasmidminiprep">modified miniprep procedure</A>. As the native promoter has been shown to be repressed by various sulfur-containing compounds (Li <I>et al</I>., 1996), we designed primers for just the coding sequences of the <I>A, B, </I> and <I>C</I> genes. As these genes all have some illegal cutsites in them we constructed them into the PSB1C3 vector and started our <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/mutagenesis">mutagenesis protocol</A>.</P><P align="justify"> We performed an experiment to measure the desulfurization rate of select organosulfur compounds by our <I>Rhodococcus</I> strain (Figures 4-6 below). These experiments monitored the degradation of the compounds by our strain over time. We discovered that the <I>dsz</I> operon is capable of desulfurizing a wider range of compounds than just the commonly studied DBT. This shows that this pathway could be a promising solution for degradation of a wide variety of sulfur  containing toxins, including those that resemble naphthenic acids.  </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:722px;"><DIV class="thumbcaption">Figure 2:  <I>Rhodococcus</I> cells were grown in a modified M9 media containing 0.125mM DBT with no sulfur containing compounds (refer to desulfurization assay protocol for details). Samples were taken out at different time points and were run through the GC/MS to detect the amount of DBT. The control only contained modified M9 but no bacteria and it was run through the GC/MS after 6 days of incubation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:399px;"><DIV class="thumbcaption">Figure 3: The peak in this mass spectrum demonstrates presence of DBT based on its molecular weight of 184 g/mol. This peak is based on the average of our samples at retention time of 13.9 minute (refer to previous graph).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 19: <I>Rhodococcus</I> cells were grown in a modified M9 media containing 0.125mM of the indicated compound (<B>A:</B> dibenzothiophene, <B>B:</B> tetrahydro-4h-thiopyran-4-one, and <B>C:</B> benzo[b]thiophene-2-carboxyaldehyde)  with no other sulfur containing compounds present in the media (refer to desulfurization assay protocol for details). Samples were taken out at different time points and were run through GCMS to detect the amount of compound remaining. Samples were normalized to a control containing modified M9 but no bacteria, run through the GCMS at the last time point to account for abiotic breakdown. Degradation is seen for DBT as well as other sulfur-containing compounds resembling naphthenic acids, indicating that the pathway may have wider substrate specificity than previously thought.</DIV></DIV></DIV></DIV><H3>2) Mutagenesis: Biobrick Compatability and Increasing DszB Activity </H3><P align="justify">In total the <I>dszABC</I> genes had 7 PstI sites and 1 NotI site that needed to be mutated for the biobrick standard. The primers were designed such that the site was removed without the amino acid being changed. In addition, a point mutation of Y63F in DszB increased the activity of the protein (Oshiro <I>et al</I>., 2007), and was included in the mass mutagenesis we undertook. Mutagenesis was performed as described in <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/mutagenesis">this protocol.</A></P><H3>3) Replacing DszD with HpaC &amp; Introducing Catalase </H3><P align="justify">
As FMNH<SUB>2</SUB> is consumed in the first three steps of the pathway it needs to be regenerated or the process will grind to a halt. This usually falls to the <I>dszD</I> gene, however it has been shown that the <I>hpaC</I> gene from <I>E. coli</I> performs the same function more efficiently (Gala´n <I>et al</I>., 2000). One problem arises from this though, as high levels of FMNH<SUB>2</SUB> cause the production of toxic hydrogen peroxide inside the cell (Gala´n <I>et al</I>. 2000). To address this issue we have included a catalase gene (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902060"><I>P<SUB>lacI</SUB>-katG-LAA</I></A>) that will remove the peroxide that would be toxic to the cell.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Figure 7: Diagrammatic representation of the full &quot;optimization circuit&quot;, consisting of the oxidoreductase HpaC and a catalase (KatG).</DIV></DIV></DIV></DIV><H3>Results</H3><P align="justify">To show that catalase activity increased <I>E. coli</I> survivability in peroxide we cultured the inducible catalase against a catalase-free control with varying levels of peroxide. After growing overnight the negative didn't grow in any culture except in the absence of peroxide, while the catalase cultures could tolerate peroxide. This is shown below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 8: Catalase Assay. Overnight cultures of P<SUB>lacI</SUB> and P<SUB>lacI</SUB>-KatGLAA were innoculated into 0 mM, 1 mM, 5 mM, and 10 mM peroxide. Cultures were grown overnight and turbidity was observed. It was found that at 1 mM of peroxide, cultures with just the lacI promotor perished, however when KatG-LAA was expressed, the cells survived.</DIV></DIV></DIV></DIV><P align="justify">To test the action of HpaC to use NADH to recycle FMN into FMNH<SUB>2</SUB> cell lysates were exposed to NADH and it's absorbance at 340nm (Kamali <I>et al</I>., 2010) was measured over time. Both native HpaC expression and an induced <A href="http://partsregistry.org/Part:BBa_K902058"><I>P<SUB>lacI</SUB>-RBS-hpaC</I></A> system were tested as well as a negative control. The results are shown below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption">Figure 9: HpaC Assay with <B>A)</B> 2 mL cell lysate and <B>B)</B> 100 µL cell lysate. Cultures of P<SUB>lacI</SUB>-hpaC and P<SUB>lacI</SUB>-dszB were grown up overnight in LB with appropriate antibiotics. The following morning, cells were subcultured 1/4 into LB with 200 µM IPTG and allowed to grow for 2h in order to induce protein expression. 1 mL samples of cells were then transferred to 2 mL tubes, washed twice in 50 mM Tris-HCl (pH 7.5) and resuspended in this buffer. Samples were then subjected to 5 freeze-thaw cycles in order to lyse cells. After spinning down samples, various amounts of cell lysate were transferred to a cuvette, and a spectrophotometer was blanked at 340 nm with this sample. 140 µM NADH and 20 µM FMN was then added, the cuvette was quickly inverted, and readings were taken at 340 nm. P<SUB>lacI</SUB>-dszB was used as a control to measure native amounts of oxidoreductase activity, whereas the P<SUB>lacI</SUB>-hpaC cultures were used to measure activity when HpaC was expressed. The control was just Tris-HCl buffer with the NADH and FMN compounds added. Decrease in absorbance at 340 nm corresponds to the loss of NADH as it is converted to NAD+.</DIV></DIV></DIV></DIV><P align="justify">The assay showed that NADH does not abiotically convert into NAD+, however the native expression of HpaC did show a steady decrease in the levels of NADH. The induced overexpression of HpaC caused extremely rapid conversion into NAD+ as reflected by a sharp drop in the absorbance of NADH (see figure B). This drop was much sharper than what was seen when native levels of oxidoreductases were tested, showing that the <A href="http://partsregistry.org/Part:BBa_K902058"><I>P<SUB>lacI</SUB>-RBS-hpaC</I></A> was functional and that it would effectively recycle FMN.</P><H3>4) Optimizing Gene Order</H3><P align="justify">Further optimization of the system was achieved through reorganization of the reconstructed operon. Natively the genes are arranged ABC, however the catalytic efficiency of the protein products are 25:1:5 for A:B:C respectively (Li <I>et al</I>., 2008). By rearranging the genes into BCA there is stronger transcription of the weaker proteins, giving a more balanced system overall. These would all be constructed with the same strong ribosomal binding site, <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A>.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 10: Method of optimizing gene order. The top circuit represents that found natively in the organism, with the bottom circuit representing our modified version.</DIV></DIV></DIV></DIV><H2>Final Sulfur Constructs</H2><P align="justify">After all of the above considerations are met, four final constructs for our system will be made to allow us to test desulfurization under different conditions.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 11: First set of final constructs for the desulfurization operon, with constitutive Dsz expression and inducible expression of the optimization proteins; either HpaC on its own or coexpressed with KatG</DIV></DIV></DIV></DIV><P align="justify">
The first two constructs have the modified <I>dsz</I> operon (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902052"><I>dszB</I></A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K804005"><I>dszC</I></A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902050"><I>dszA</I></A>)  under the control of a constitutive TetR promotor (<A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>) This is to allow for the testing of the optimization circuit, which is under the control of a lacI promotor inducible by IPTG (<A href="http://partsregistry.org/Part:BBa_J04500">BBa_J04500</A>). The set-up of these two constructs will therefore allow for the expression of the <I>dsz</I> genes with the ability to test and compare their desulfurization rates  A) On their own  B) With the addition of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902057"><I>hpaC</I></A> C) With the addition of both <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902057"><I>hpaC</I></A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K137068"><I>katG-LAA</I></A></P><P align="justify">This will allow us to determine what the optimal construct and expression levels of the additional genes must be in order to have the most effective sulfur removal system.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 12: Second set of final constructs for the desulfurization operon, with all genes under an IPTG inducible promotor.</DIV></DIV></DIV></DIV><P align="justify">
Due to the large number of proteins being expressed in this system, the possibility of forming inclusion bodies is present. As such, a backup system was built where both the optimization circuit and the <I>dsz</I> operon were under the control of the inducible lacI promoter. This system would allow us to tune the expression of the genes, and determine which expression level is optimal for desulfurization in our bioreactor.</P><P align="justify">Currently the final steps of construction of these constructs is underway, following which functionality tests will begin.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">http://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/OSCAR/Desulfurization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/OSCAR/Desulfurization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/OSCAR/Desulfurization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/OSCAR/Desulfurization&amp;oldid=294208" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>