"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Decarboxylation
From 2012.igem.org
Home
Team
Members
Official Profile
The University
Outtakes
Contact Us
Project
Overview
Data Page
Accomplishments
Post-Regionals
Human Practices
Initiative
Interviews
Design
Killswitch
Regulation
Kill Genes
Safety
FRED
Toxin Sensing
Electroreporting
Modelling
Device Prototype
OSCAR
Decarboxylation
Decatecholization
Flux Analysis
Bioreactor
Upgrading
Desulfurization
Denitrogenation
Synergy
References
Attributions
Parts
Notebook
Journal
Protocols
Outreach
LAB ESCAPE!
Spark Science
Minds in Motion
Blog Posts
TEDxCalgary
Follow Us!
Sponsors
iGEM
Notebook
Journal
Killswitch
Toxin Sensing
Electrochemistry
Biosensor
Decarboxylation
Desulfurization
Denitrogenation
Catechol
Bioreactor
Flux Analysis
Protocol Manual
Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!
Decarboxylation Journal
Decarboxylation
Week 2 (May 7-11)
For the decarboxylation sub-project, the second week was entirely focused on literature research and the practice of basic laboratory techniques. Eight potential pathways were identified as potential candidates for naphthenic acid decarboxylation. The first of these would utilize only the University of Washington's "PetroBrick" (from iGEM 2011), consisting of the genes encoding the enzymes acyl-ACP reductase ( AAR ) and aldehyde decarbonylase ( ADC ). We have planned to verify the PetroBrick in the distribution plates and test its efficacy on naphthenic acids in the coming weeks. If this proves to be unsuccessful, we will begin investigating the alternative approaches, beginning with replacing AAR with carboxylic acid reductase (CAR) from Nocardia iowensis, a very unspecific reductase shown to work on structures resembling naphthenic acids. Failing this, the remaining pathways will be examined; however, the disadvantage in these pathways is their direct reliance on the success of the other steps, as the naphthenic acids must be degraded to the point of resembling branched-chain fatty acids (since all remaining pathways are related to fatty acid metabolism).
Week 3 (May 14-18)
The third week included some additional literature investigation in the first two days. The iGEM distributions arrived this week, and verification began on May 17th by transformation into E.coli, followed by colony PCR on May 18th (using standard protocols on the wiki). Additionally, primers were designed for CAR in Nocardia iowensis, along with primers for Nocardia posphopantetheine transferase (NPT), a second enzyme required for optimal function in the former, and a short list of contacts were acquired to request the donation of the required strain (called NRRL 5646) from researchers who have worked with it previously. A sort of form email was drafted for this purpose, and should this be unsuccessful, we will be purchasing the strain from DSMZ . In the following week, we will begin with development of overnight cultures and gel preparation.
Week 4 (May 22-25)
This week, a gel was prepared for the colony PCR prepared last week, and the gel was run, yielding results that were very difficult to see. The PCR and gel electrophoresis process were repeated, yielding the following gel:
Results indicate transformation with the <a href=" http://partsregistry.org/Part:BBa_K590025 ">PetroBrick</a>. Lane 1 contains the 1 kb plus ladder, while Lanes 2-11 contain the results of colony PCR. Each of these bands correspond to the <a href=" http://partsregistry.org/Part:BBa_K590025 ">PetroBrick</a> vector, which is 2070bp, indicating that the transformation was successful. Lane 12 contains a positive control, which varies distinctly from the colony PCR results. The negative control lane is empty as expected.
Overnight cultures were grown from the colonies were prepared this week. Sigma compounds - for the purpose of testing the PetroBrick on naphthenic acid analogues - were selected from the list provided. The compounds to be used include cyclohexanepentanoic acid, cycohexane-1,1-dicarboxylic acid, and benzo[b]thiophene-3-acetic acid. A mini-prep was completed on the overnight cultures prepared earlier in the week, according to protocol obtained from the wiki. The DNA concentration in the resulting samples was measured by nanodrop to confirm successful plasmid extraction, and the resulting DNA concentrations were as follows:
Tube 1: 44.9 ng/mL
Tube 2: 54.2 ng/mL
Tube 3: 54.4 ng/mL
Tube 4: 41.6 ng/mL
Tube 5: 34.3 ng/mL
Because tube 3 (prepared from Colony 3, Plate 3) had the highest concentration, it was to be used for the eventual sequencing of the plasmid. The restriction digest was completed also, but it was not run on a gel this week; this was left for Week 5.
Week 5 (May 28 - June 1)
On Tuesday (May 29), a gel was run on the restriction digests of the extracted plasmids from Week 4, which appeared to confirm the successful transformation of the PetroBrick , showing clear bands at the pertinent locations. The gel is as follows:
Lane 1 contained the 1 kb plus ladder, while lanes 2-6 contained the restriction digest results, which had used the five different miniprep tubes with plasmids isolated from five different colonies. As can be observed in the gel, the upper row of bands corresponds to 2392bp, indicating the PetroBrick part. The lower set of bands corresponds to the PetroBrick pSB1C3 vector at 2070 bp.
These results suggest that the PetroBrick plasmid had been successfully purified. The black circle indicates undigested plasmid in lane 3. There simply means that not all of the plasmid was digested into two separate DNA strands. It indicates the entire size of our PetroBrick plasmid.
Based on the apparent success of the gel, the products were sent away for sequencing. Long-term stocks were prepared for the alkane production medium outlined in University of Washington's PetroBrick protocols from 2011 (see https://2011.igem.org/Team:Washington/alkanebiosynthesis). These stocks are as follows:
1 L of 1M Tris (pH = 7.25)
10 mL of 1 mg/mL Thiamine
10 mL of 10% Triton x-100
1 M MgSO4
0.1 M FeCl3 (anhydrous)
The medium itself is to be prepared in Week 6 once the results of sequencing are (hopefully) acquired.
Week 6 (June 4 - June 8)
This week, we first made a streak plate of our confirmed PetroBrick E.coli Colony 3, Plate 2, as well as a long-term glycerol stock for storage. In accordance with the Washington team’s protocols, we prepared our M9 minGlucose Media (Production Media) with the following reagents:
75mL ddH2O
0.6g Na2HPO4
0.3g KH2PO4
0.05g NaCl (855.6mL 1M solution)
0.2g NH4Cl
20mL of 1M Tris (pH = 7.25)
1mL of 10% Triton
100mL of 1mg/mL thiamine hydrochloride
10mL of FeCl3 (anhydrous)
100mL of MgSO4
**3.0g glucose needed to be added later after the mixture was autoclaved, as glucose is known to caramelize under the conditions of the autoclave.
We also prepared an overnight stock containing 5mL of LB broth inoculated with E.coli from our verified colony. The following day we measured its optical density, demonstrating that our OD was 1.311, suitable for our purposes. We obtained our PetroBrick sequencing results with a complementarity of 87%, as compared to the theoretical PetroBrick in the PartsRegistry, which was deemed by iGEM team leaders to be high enough to continue in our protocol. We also performed glucose filtration, transferring 3g of glucose into our Production Media solution after it had been autoclaved in order to ensure sterile transfer. On the final day of the week, we were able to start our General Production Protocol that would be followed by a 48 hour incubation. It was in this time that hydrocarbon production was reportedly supposed to occur. Use of the GC would occur in the following week to test for alkane production.
Week 7 (June 11 - June 15)
After preparing our production media 48 hour incubation sample for gas chromatography and extracting with ethyl acetate, we left our sample for use in the GC. Our results indicated a small hydrocarbon peak, indicating a possibility that hydrocarbon production had been a success, but it was not distinct enough to make this conclusion. The procedure would be repeated, this time creating sigma compound solutions as well. These sigma compounds would resemble naphthenic acids. In order to ensure that hydrocarbon peaks would be produced from these compounds and not from glucose, we prepared a new production media lacking glucose, in hopes that incubated bacteria could survive for long enough without a food source to decarboxylate a sigma compound to some extent. Since the sigma compounds we initially selected for our purposes had not arrived as of yet, we opted for others: Cyclohexanepentanoic acid (CHPA), cyclohexanecarboxylic acid, and 1,4-cyclohexanedicarboxylic acid. We performed eight overnight cultures and prepared sigma compound solutions. Despite our many attempts, we were only able to acquire an OD of about 0.6 for these 8 tubes. We then performed our production protocol with the tubes, preparing them with the following reagents and inoculating them with the contents of our broth cultures:
Tube 1: Glucose
Tube 2: Glucose
Tube 3: Glucose and CHPA
Tube 4: Non-glucose and CHPA
Tube 5: Glucose and cyclohexanecarboxylic acid
Tube 6: Non-glucose and cyclohexanecarboxylic acid
Tube 7: Glucose and 1,4-cyclohexanedicarboxylic acid
Tube 8: Non-glucose and 1,4-cyclohexanedicarboxylic acid
In the above tubes, “glucose” indicated that glucose-containing production media was used, whereas “non-glucose” indicated that the production media lacking glucose was used. In each case where a sigma compound was added, it was present at a concentration of 25mg/L. Each production tube was then left for a 48 hour incubation over the weekend, to perform gas chromatography tests the following week.
Week 8 (June 18 - June 22)
This week, we received the results of our gas chromatography of the eight tubes created last week, and unfortunately no hydrocarbons had been produced in any of them. It was thus very important for us to spend time researching why this was unsuccessful. We have found a number of things to consider. Firstly, the Washington iGEM team (creator of the PetroBrick has worked on system optimization , and has come up with an number of ways to increase alkane production. Information regarding this can be found here: https://2011.igem.org/Team:Washington/Alkanes/Future/Vector. There are many other considerations as well. We have received feedback on our sequencing results, and have been told that 87% is not high enough; we should be obtaining results of 100%. This is a problem because it presents the possibility that there is a mutation somewhere in our PetroBrick that is preventing effective operation. We may need to design inner primers to sequence the entire PetroBrick , as it is 2392bp and this may have prevented 100% sequencing results. We have also been using plastic falcon tubes, while hydrocarbon production has been tested in glass tubes. Incubation time and initial amount of cells to be innoculated are also concerns, as we have been having many problems with cell growth. The Washington Team often had an OD600 of 10 in their production media, whereas we have been tracking the OD of our initial broth only, so it is possible that the OD600 of our broth has been very low. As demonstrated by iGEM Washington, the OD600 is of utmost importance when it comes to hydrocarbon production, and is something critical that we need to improve. We are also considering extracting alkanes from our cells using HCl rather than ethyl acetate, as we many not be removing hydrocarbons effectively from the cells. We will consider each of these things when we attempt hydrocarbon production again.
Week 9 (June 25 - June 29)
The week began initially with planning for continued attempts to produce hydrocarbons (Based on information that was learned using the University of Washington's " system optimization " techniques. We also backtracked to determine any additional sources of error within our procedures for the production media protocol. During the week, the sequencing results were examined in further detail, and it was determined that the entire gene sequence for aldehyde decarbonylase ( ADC ) was missing. This explained why our sequencing results for the PetroBrick yielded only 87% complementarity. Based on this, there were two possible justifications for only having the acyl-acp reductase ( AAR ) gene and no aldehyde decarbonylase ( ADC ). Either the parts registry did not have the proper PetroBrick (containing both AAR and ADC ) in the assigned well in the 2012 kit plate sent to us, or we had extracted from the wrong well in the kit plate, assuming that it was the well containing the PetroBrick . It was determined that the latter was true. As such, the transformation for the PetroBrick had to be redone, and if gene sequencing yielded positive results (high complementarity), we would try the production media protocol once more in an attempt to get hydrocarbon production.
Week 10 (July 2-July 6)
The main goal this week was to repeat the transformation and verification steps that were done previously with AAR for the PetroBrick (submitted by the University of Washington). This involved transforming the PetroBrick into E.coli, colony PCR for verification of a successful transformation, followed by mini-preparation and restriction digest. Results were submitted for gene sequencing. If the sequencing results come back as being successful (showing a high rate of complementarity), we will proceed forward with the production media protocol in an attempt to produce hydrocarbons.
Week 11 (July  9-July 13)
The sequencing results sent in for the PetroBrick did not yield a high complementarity rate, therefore were unsuccessful. As such, we were not yet able to proceed with the production media protocol to attempt to produce hydrocarbons because we are not certain as to whether the PetroBrick part has been isolated. It was then decided that the colony PCR would be redone with a new colony, followed again by a mini-preparation and restriction digest before gene sequencing occurs. If the sequencing results again yield a low complementarity rate, it can be deduced that the PetroBrick sent to us in the 2012 kit plate does not contain a proper functional PetroBrick .
Week 12 (July  16 -July 20)
It was decided that the decarboxylation team would be split up into three individual components, a) to strictly work with the PetroBrick , b) transformation and isolation of the aldehyde decarbonylate ( ADC ) and c) culturing the Nocardia strain NRRL 5646. With regards to the PetroBrick , the sequencing results were once again unsuccessful and yielded a low complementarity rate. As such, it was decided that we would contact the University of Washington, and ask them to send us the PetroBrick they composed and were working with in 2011. In addition, we also decided to produce our own biobrick identical to the PetroBrick by using the acyl-acp reductase (which we had successfully transformed and isolated previously) with aldehyde decarbonylase ( ADC ). The ADC was transformed into the E.coli,and colony PCR was done in order to determine whether successful this was successful. This was followed by mini-preparation and restriction digest before gene sequencing occurred.
Week 13 (July 23 - July 27)
The gene sequencing results for the aldehyde decarbonylase ADC ) were unsuccessful. It was presumed that this occurred because the mini-preparation procedure being used may potentially be leading to a high contamination rate. As such, a colony PCR was done with a new colony for ADC , followed my mini-preparation using a different procedure, and then a restriction digest. The sequencing results now yielded a high complementarity rate and were considered successful. We then proceeded with making the biobrick consisting of the acyl-acp reductase ( AAR ) being ligated with aldehyde decarbonylase ( ADC ). Upon successfully forming this biobrick (ideally the same as a functional PetroBrick ), we will be able to test for hydrocarbon production using the general production media protocol. The University of Washington also responded, and said that they will be able to send us their functional PetroBrick that was being used in 2011.  With regards to culturing Nocardia, a “brain and heart infusion (BHI)” broth was used to inoculate the Nocardia in an attempt to get growth. Six pairs of mutagenic primers were also composed for carboxylic acid reductase (CAR), which is found in the Nocardia, as well as primers for the OleT enzyme.
This week, we were pleased to find ample Nocardia iowensisgrowth on our BHI plate. We performed a PCR with our constructed CAR and NPT BioBrick primers to attempt to amplify these genes, so that we could insert them into BioBricks. This PCR utilized primers specific to our genes of interest. After running the gel, we found that we had ample amplification of both of these genes, as shown in the picture below:
In the above gel, lanes 4-5, 7, and 9-12 represent successfully amplifed NPT (669 bp). Lanes 14-16 and 18-20 show that CAR (4690 bp) has been successfully amplifed.
Following this, we performed a Cycle-Pure Spin Protocol to purify the amplified CAR and NPT DNA. The resulting concentration of CAR was 171.6 ng/ul, while NPT was 20.8 ng/ul. Using a restriction digest, the CAR gene was cut with AscI and HindIII, while NPT was cut with both XbaI and PstI, as well as with EcorI and PstI. Using these same enzymes, the PET vector was cut to be ligated with CAR, and the pSB1C3 vector was cut to be ligated with both NPT digests. Following incubation, the inserts were ligated together with their vectors. Transformations were then performed, with the CAR/PET ligation transformed on a Kan plate, and the NPT/ pSB1C3 E/P and X/P ligations transformed on chloramphenicol plates. Transformations were successful, but there were not a great deal of colonies on the CAR plate, and both NPT plates revealled a very high proportion of red colonies, indicating that our desired NPT gene had not been successfully ligated and transformed. Nonetheless, colony PCRs were run in order to test for successful transformations of the ligations the following week. Meanwhile, the results of the ADC gene sequencing came in, but unfortunately it was found that this gene did not correspond well to its theoretical sequence on PartsRegistry. It therefore could not be used with our AAR gene to produce a PetroBrick . The Washington iGEM team agreed to send us a plasmid stock of their PetroBrick and ADC .
Week 15 (August 7 - August 10)
We ran the gel on the CAR and NPT colony PCRs from last week, but unfortunately these gels were blank. Another colony PCR was performed, with twelve colonies from each of the following plates: CAR/PET, NPT(X/P), NPT(E/P). Both NPT plates did not look very promising, as they were completely covered by red colonies, with only a few white ones. Many of the white colonies from the previous week had become red. Eight overnight cultures were also performed so that a restriction digest could be done on the CAR colonies, to see if the part was there. These overnight cultures were prepared with the Kan antibiotic. Another ligation was done with the three combinations of cut inserts and vectors: CAR with PET, NPT (E/P) with pSB1C3 , and NPT (X/P) with pSB1C3 . Restriction digests were performed on the CAR overnight colonies with AscI and HindIII, and when the gel was run, a single lane was shown to contain both the CAR gene and PET vector, as shown below:
As shown in the gel above, lane 3 indicates 2 bands, the uppermost at about 5.4kb, identifying as PET. This single lane also contains a clear band of CAR, a bit under the 5000bp mark, at 4690bp. The ladder used was 1 kb plus.
This successful result allowed us to proceed with mutagenesizing the 6 illegal cut sites of CAR, including: 2 PstI sites, 1 EcorI site, and 3 NotI sites, with the primers that we had designed. Two additional colony PCRs of NPT were performed, but unfortunately a band indicating NPT was not visible in any of the traisl, even when an additional plate of both the NPT (E/P) and NPT (X/P) restriction digests were ligated and transformed. Very few white colonies could be seen on any of the transformations, but we were determined to continue trying. We then attempted to mutagenesize the first illegal cut site of CAR, which would be a PstI site. The mutagenesis primers needed to be diluted and stock solutions were made. Then, we attempted to mutagenize the PstI site of CAR with that site's specific primers, using various concentrations of template DNA (5ng, 20ng, 50ng). We used the KAPA system to perform this plasmid amplification, using the thermocycler settings of the Desulfurization team. The mutagenesis PCR would be run over the weekend to hopefully have successful results.
Week 16 (August 13 - August 17)
This week, the attempted PstI mutagenesis of CAR ws run on a gel. Unfortunately, amplification was unsuccessful, but we had an idea for why this might have occurred. It was discovered that our plasmid of 9kb would need a 7.5 minute extention time, rather than the 1 minute extention time that we had originally programmed into the thermocycler. The mutagenesis experiment was repeated, this time with a 7.5 minute extention time. When these results were analyzed with gel electrophoresis, we had bands appearing to indicate success with the 50ng samples of CAR that we had run, as shown below.
This gel shows 2 bands of CAR that indicate successful mutagenesis of the first PstI site in CAR. The CAR gene is contained in the PET vector. Lane 1 contains the 1 kb plus ladder, while lanes 2,3 contained samples in which 5ng of original CAR+PET plasmid were used, lanes 4,5 contained samples in which 20ng of original plasmid were used, and lanes 6,7 contained 50ng of plasmid sample and underwent mutagenesis by PCR amplification successfully. The bands are slightly over the 9kb band of the ladder, at about 10kb, which is roughly the size of the 10,090bp plasmid.
After combining these two PCR tubes and incubating with DpnI, we performed a transformation with this DNA and allowed the plate to incubate overnight. Also, we continued to try and verify NPT, but were once again unsuccessful. When we viewed the plate that we had transformed the following day, it was found that we did not have any colonies. We presumed this was because the 9kb plasmid was very difficult to uptake with its large size. Several more transformations were attempted. We also attempted a new procedure called site-directed mutagenesis, which would use a single mutagenesis primer per site, rather than one for each strand, and supposedly had the capacity to mutagenize multiple illegal cut sites at once. We would try to mutate 3 sites with this protocol: the 2 PstI sites, and 1 EcorI site. The PCR was run overnight, with eight different samples, at two different concentrations of DNA (50ng and 80ng), with four different annealing temperatures. The following day, each sample was incubated with DpnI for 6 hours, and then each was tranformed on a kan plate. A restriction digest of the NPT plasmid was once again performed, cutting with both (E/P) and (X/P) as before, this time using more DNA. An overnight culture of a single potentially successful NPT colony was made, to perform a restrictin digest to see if NPT was present - this would be done over the weekend. Finally, a transformation of the single mutagenesis KAPA PCR of PstI was performed, to hopefully yield more colonies.
Week 17 (August 20 - August 24)
This week, we carried forth with the second and third mutagenesis sites of the CAR gene. Our new plasmid with our first PstI site was PCR amplified with the Kapa system and our EcorI mutagenesis primers to carry out the second mutation, to remove the EcorI illegal cut site from our gene. After the PCR was complete, we ran our product on a gel to see if the amplification worked. Bands indicated success. Following this verification, we mixed our two seemingly successful PCR tubes with 1 uL of DpnI, and left them to incubate for one hour. The PCR product was then transformed into highly competent E.coli cells, and the plates were left overnight in the 37 degree incubator. Unfortunately a mistake prevented colonies from growing, but the process was repeated and our second attempt yielded many colonies. We made overnight cultures of 8 of these colonies, with 4 from each plate, with Kan as the antibiotic for selective growth. The next day, a miniprep was performed on each overnight culture to isolate the CAR+PET plasmids that had hopefully undergone their second mutation, of their single EcorI site. Once the plasmid isolation was complete, we performed a restriction digest using EcorI to cut, in order to verify that the illegal cut site had been removed and the plasmid could no longer be digested with this enzyme. We immediately proceeded with the third mutagenesis, which would this time be performed on the second (and last) PstI site. Using our new plasmid now with two illegal cut sites removed, we performed an amplification PCR, using our PstI(2) primers and the Kapa system. After running this on a gel, we determined bands of success, as shown below. We then once again treated our PCR amplified tubes with DpnI for one hour, and then transformed into highly competent E.coli. The plates were left overnight to allow colony growth. Our second accomplishment of the week was successfully amplifying the OleT gene from Jeotgalicoccus, by using the Kapa system. This was very successful, as shown in the gel below. We then performed the cycle-pure spin protocol to remove contamination from our PCR product, and then proceeded to perform a restriction digest on the product with HindIII and AscI so that we could ligate it into the PET vector. After this ligation was complete, it was transformed with E.coli on a Kan plate, and left overnight to incubate. Unfortunately we do not see colonies yet, but the incubation is not complete yet, and might still be successful. Thirdly, we obtained the PetroBrick stock from the Washington team on Tuesday, and immediately proceeded to transform it. We obtained a great deal of colonies, and then performed colony PCR on 12 of these colonies to ensure that our ~2400bp PetroBrick was present in these cells. Last but not least, we have still been working on a successful purification of NPT from a transformed plate. A colony PCR indicated that NPT was potentially in a few of these colonies, so we made overnight cultures of these four colonies, and performed a miniprep the next morning, and then a restriction digest to verify that the gene was present.
Week 18 (August 27 - August 31)
This week, the primary objectives from last week were continued. Another restriction digest of OleT and then a ligation was performed, in order to see if we could obtain more colonies following transformation. After preparing overnight cultures and minipreps of eight of these colonies, we attempted to cut the OleT/PET vector with EcorI to verify that the gene had successfully inserted into the vector. Unfortunately, we did not have any luck, even after several attempts. Our attempts to verify this will continue. We also performed ligations with NPT (cut with XbaI and PstI) with JI3002 (cut with SpeI and PstI), so that we could produce a biobrick with JI3002 before NPT to act as a promoter and ribosome binding site. We decided on how we will build our two primary constructions, and where each part will be. Also, we attempted to amplify the CAR gene in the PET vector (with 3 mutagenesis sites replaced) with biobrick primers, so that we could get it inserted into a biobrick plasmid. After several tries, this amplification was successful.
Week 19 (September 3 - September 7)
The team focused primarily on assays this week, including that of the PetroBrick as well as OleT. Procedures were also continued with CAR and NPT, and each were sent for sequencing, both shown to be successfully isolated when the results came in. It was decided that we would use CAR that had been not fully mutagenized, still with three NotI sites, as a piece in our construction with NPT so that we could perform assays. Mutagenesis would continue later, once we were able to obtain actual assay results. We continued to attempt to ligate JI3002 in front of NPT in a pSB1C3 vector. Following standard PetroBrick assay procedures (see earlier weeks), we were finally able to have successful assay results with proper graphical data. Instead of performing the assay with glucose, however, we used a standard mixture of naphthenic acids. Despite our earlier hypothesis that the PetroBrick would be unable to degrade naphthenic acids, we found that bacteria containing the PetroBrick were able to degrade these naphthenic acids into both alkanes and alkenes. These successful graphs are shown below.
Figure 1: The relative intensity of alkane production over a retention time in both E.coli that contain the PetroBrick, and in E.coli that are lacking the PetroBrick, as measured with GC-MS. Naphthenic acids were used as a substrate. A naphthenic acid standard was required to compare peaks.
Figure 2: The alkane and alkene mass spectrums generated by analysis of hydrocarbons produced from E.coli containing the PetroBrick as in Figure 1, using naphthenic acids as a substrate, as measured with GC-MS. Relative intensity to mass to charge ratio were compared.
Week 20 (September 10 - September 14)
We continued to attempt to ligate NPT and JI3002 together so that we could eventually perform assays with this gene. As well, we began trying to ligate the JI3002 promoter in front of CAR. After transforming both ligations and growing them on chlor plates, we noticed that for both attempted constructs, there were very few colonies. When we performed a PCR amplification and ran it on a gel, we did not find any significant bands; we were looking for 743 bp for NPT + J13002 , and 4764 bp for CAR + J13002 , and we did not see these bands. This process was repeated, but unfortunately it continued to be unsuccessful. Also, we once again attempted to clone OleT into a PET vector, but when we made overnight cultures of the few colonies we had, did minipreps, and performed our restriction digest with them using HindIII and AscI, we unfortunately did not see any bands. These processes will have to be repeated. Assays for OleT were continued, and results for these will hopefully be obtained next week so that we may input them before the Wiki freeze.
Week 21 (September 17 - September 21)
This week, we obtained successful results for our OleT assay! This is represented by the graphs below:
Figure 8.  Gas chromatograph demonstrating the production of olefins (alkenes) from fatty acids as shown from the increase in the peak with a retention time of 14.7 min.  The dramatic change in peak intensity at this point suggests that we are producing hydrocarbons.
Figure 9. Mass spectra of the peak in Figure 8 at retention time 14.7 min.  Demonstrating that this peak is an olefin, which is known to be produced in Micrococcus.  This verifies our proof-of-concept that the Micrococcus species can degrade fatty acids into olefins.
Now that our assay is complete, we need to successfully transform OleT into a PET vector so we can perform our assays with E.coli. Unfortunately, we still had no luck in ligating J13002 in front of CAR or NPT. Even when using super competent cells, we were only able to get a few colonies to grow after transformation, and these did not yield good results when we made PCR, overnight cultures, minipreparations, and restriction digests to test. Next week, we are going to attempt to ligate a promoter downstream of the PetroBrick as an alternative method to build our construction. If this is successful, we will ligate NPT and CAR into the construction as well, so we can perform decarboxylation assays with all three of the genes. We still had no success in cloning OleT into a PET vector.
Week 22 (September 24 - September 28)
This week, we were successful in building a construct with J13002 downstream of the PetroBrick genes. First a ligation was performed, which was then transformed into competent E.coli cells. After obtaining several colonies, we performed a PCR that had bands at 2466 bp, indicating the combined PetroBrick + J13002 part, and showing that J13002 had inserted successfully! We then made overnight colonies, minipreps, and performed a restriction digest with EcoRI. After running the gel, we saw that our bands indicated successful insertion of JI3002, with bands at 2466 bp and 2070 bp. Next week, we will try to ligate NPT downstream of this promotor, so that we can eventually ligate CAR in as well and perform assays. Sadly, we still did not have any sucess in inserting OleT into a BioBrick.
Week 23 (October 1 - October 5)
We did not manage to do any significant lab work this week, as it was the wiki freeze and we spent all of our time inputting data!
Week 24 (October 8 - October 12)
This week, we managed to ligate NPT downstream of our PetroBrick that had the J13002 part we inserted. This process was very similar to that of week 22. We cut our PetroBrick and NPT BioBrick with restriction enzymes so that we could ligate NPT downstream of the rest of our construction. We then transformed this on Chlor plates, performed PCR on our colonies, getting 3135 bp that indicated successful insertion, made overnight colonies, miniprepped these colonies, and cut with EcorI to see if our insert ligated properly (yielding two successful bands of 3135 bp and 2070bp). This did not succeed on the first try, but after a few attempts over the week, we were finally able to have success. OleT still has not been successfully inserted into a PET vector. However, we now have a BioBrick vector containing the PetroBrick genes, and J13002 with NPT downstream. If we can ligate CAR into this as well, then we can start to perform assays and see how effective our construction is a decarboxylating naphthenic acids.
Week 25 (October 15 - October 19)
Once again, we cut our PetroBrick / J13002 /NPT construction so that we could ligate CAR into the end of it. We tranformed this large construct into super competent cells on a chlor plate, and performed PCR, but this was not successful; we were looking for bands of 7973bp. Despite this, we made overnight cultures and minipreps to try and digest our construct with EcorI, but this did not yield any bands corresponding to our construct. We attempted this a few times, but had no success. Once again, we were not able to successfully ligate OleT into the PET vector.
Week 25 (October 22 - October 26)
We felt that we would try a different approach this week in building our construct. Instead of ligating CAR into the PetroBrick / J13002 /NPT BioBrick, we would try to ligate this construction into the PET vector containing CAR, with CAR downstream of NPT. After cutting and performing a ligation, we transformed into E.coli, but had very few cells. Nonetheless, we did a PCR to check if the size of our plasmid matched our ligated construct, but unfortunately it did not. Once again we were looking for 7973 bp, but did not obtain any of these bands. We did not have time to proceed further with this construction, as there was data to be inputted for the final Wiki Freeze.
Retrieved from " http://2012.igem.org/Team:Calgary/Notebook/Decarboxylation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
