"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Protocols/tnscreen
From 2012.igem.org
Home
Team
Members
Official Profile
The University
Outtakes
Contact Us
Project
Overview
Data Page
Accomplishments
Post-Regionals
Human Practices
Initiative
Interviews
Design
Killswitch
Regulation
Kill Genes
Safety
FRED
Toxin Sensing
Electroreporting
Modelling
Device Prototype
OSCAR
Decarboxylation
Decatecholization
Flux Analysis
Bioreactor
Upgrading
Desulfurization
Denitrogenation
Synergy
References
Attributions
Parts
Notebook
Journal
Protocols
Outreach
LAB ESCAPE!
Spark Science
Minds in Motion
Blog Posts
TEDxCalgary
Follow Us!
Sponsors
iGEM
Notebook
Journal
Killswitch
Promoter Screen
Electrochemistry
Biosensor
Decarboxylation
Desulfurization
Denitrogenation
Catechol
Bioreactor
Flux Analysis
Protocol Manual
Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!
Transposon-Mediated Mutant Library Generation
Materials
E.coli SM10 pOT182
Pseudomonas fluorescens PF-5
LB Broth
Pseudomonas isolation agar (PIA)
Mix naphthenic acid solution (Acros, 0.91g/mL) or any other toxin
Tetracycline (stock concentration at 5mg/mL)
X-Gal
Hydrogen peroxide
Decanoic acid
       Day 1
Around 3 or 4pm, prepare an overnight culture of donor E.coli SM10 pOT182 in 2mL of LB with 10μg/mL of tetracycline, and incubate overnight at 37°C.
Subculture the recipient Pseudomonas fluorescens PF-5 as well in 5mL LB+50mg/L NA media, and incubate at 30°C, shaking at 110rpm.
Note: this protocol is designed to isolate NA sensing promoters. To isolate promoters sensitive to other compounds, simply replace NA with the compound of interest.
Day 2
Make the selection media (Pseudomonas isolation agar/PIA) for the next day. The selection media contains PIA, tetracycline, NAs, and X-Gal.
To make the media, mix 45g of the PIA powder in 1L of milliQ or nanopure water containing 20mL of glycerol. Add 54.95µL of the Acros NAs. Label the bottle properly, mix well using a magnetic stir bar, and autoclave.
After autoclaving, allow agar to cool to about 55°C in a water bath. Add tetracycline to a final concentration of 50μg/mL, and the X-Gal to a final concentration of 20 μg/mL. Mix the media well by slowly stirring (magnetically) while adding X-Gal and tetracycline.
Pour the agar solution into labelled petri dishes, allow to solidify (in about 1hr), and store at 4°C in the dark to prevent antibiotic degradation.
At 2pm, re-subculture E. coli SM10 in 2mL of LB (no tetracycline) to remove the tetracyline. Incubate by shaking at 37°C for 2hr.
At 4pm, prepare the following in 2mL tubes:
Negative control: 500µL Pseudomonas fluorescens PF-5 (from the culture prepared in Day 1)
Negative control: 500µL of the E.coli SM10 (the re-subcultured sample)
Experimental sample: 500µL of the E.coli SM10 (the re-subcultured sample) and 500µL of of the Pseudomonas fluorescens PF-5 cultures
Mix all samples by vortexing and centrifuge all the tubes at max speed for 5min.
Discard the supernatant and resuspend the cell pellet in 50μL of LB.
Pipette all 50μL onto a LB (no additives) plate. A single LB plate can be divided in three sections (make sure to label all the sections). Draw on the back of the plate like so:
Figure 1: Mating spot setup on a a LB plate
Let the plate sit on the bench at room temp until all the spots have dried.
Incubate overnight at 37°C.
Day3
In the morning, scrape up the mating spots using a sterile pipette tip, and resuspend in 500μL sterile distilled water.
Mix thoroughly by vortexing or pipetting up and down. Do this for each spot from the LB plate.
For the mating spots (with both E.coli and Pseudomonas fluorescens), make serial dilutions from the resuspension (1/10, 1/100, and 1/1000).
Make spread plates using 100 µl of the undiluted mixture and each of the dilutions on PIA selection plates.
Plate 100μL of the resuspended Pseudomonas fluorescens PF-5 (alone) spot onto a PIA plate as well (no dilution required). Do the same for the E.coli SM10 spot.
Incubate overnight at 30°C. Check the next morning for growth.
Day4
Check the plates for growth. Do not allow the plates to overgrow in order to isolate single colonies.
Ensure no growth is present on the plates containing the
Pseudomonas fluorescens PF-5 (alone) and E.coli SM10 (alone) spots (At least at a reasonably low level indicating a small amount of spontaneous mutations).
On the experimental sample plates, isolate all blue colonies in overnight cultures (in LB+10μg/mL tetracycline+50mg/L NAs). Make glycerol stocks from the overnight cultures.
Day5
The strains isolated from the blue colonies are further screened in a 96 well plate. Each strain is inoculated into the following set of media (300μL) in the plate:
LB alone
LB with 200μg/mL X-Gal
LB with 50mg/L NAs and 200μg/mL X-Gal
Make sure the plate setup is recorded properly, and make an initial (0hr) reading using a spectrophotometer at 615nm to measure the X-Gal degradation product concentration.
Incubate the plate at 30°C in a shaker set at 140rpm for 16hr.
Day6
Measure the absorbance of the plate once again at 615nm at 16hr.
Based on the absorbance readings, record any colony that has a higher absorbance in the LB+NA+X-Gal sample than the LB+X-Gal sample. Preferably, the colony should show no growth in the LB+X-Gal sample.
Ensure the selected colonies are glycerol stocked , and subculture these colonies in LB+10μg/mL tetracycline+50mg/L NAs.
Day7
Using the selected strains from the previous day, perform a more detailed assay for each strain using undiluted, 1/10, 1/100, and 1/1000 of the culture in LB+50mg/L NAs (duplicates for each dilution).
Also test the colonies in LB medium with 10μM, 50μM, and 100μM hydrogen peroxide, as well a fatty acid (e.g. decanoic acid) to ensure the response is not stress-induced, and specific to NAs.
The assay can be set up again in a 96-well plate.
Incubate the plate at 30°C in a shaker set at 140rpm for 12h-24h.
Day8
Keep strains that respond specifically to NAs (or any other toxin), i.e. X-Gal degradation and growth in LB+NAs, but no/less X-gal production in LB+ varying amounts of hydrogen peroxide or LB+ decanoic acid.
For these strains, purify genomic DNA from overnight cultures. Measure the DNA concentration, and run on a gel to confirm that the DNA is not degraded.
Completely digest 600ng of the genomic DNA with BglII. (This restriction enzyme cuts frequently (6-cutter) in the Pseudomonas fluorescens PF-5 genome and not in the transposon). Run the reaction overnight at 37°C for 16hr.
Day 9
Inactivate the BglII enzyme by performing a enzymatic reaction cleanup (using a PCR purification kit, as the enzyme cannot be heat inactivated).
Using 50ng of DNA, perform a self-ligation to circularize all the genomic fragments.
Transform the ligation mixture (all 20µL) into E. coli DH5α or Top10. Plate on LB+10μg/mL tetracyline plates.
Day 10
Prepare overnight cultures of the transformed cells in 5mL of LB+10μg/mL tetracyline.
Day 11
Purify the plasmid from the overnight cultures by miniprep .
Run the isolated plasmids on a gel to ensure the right size (at least the size of the transposon).
Sequence the plasmids using sequencing primers targeted against the insertion sequences of the transposon.
When sequencing results are received, identify the flanking sequences to the transposon insertion. The sequence outside the transposon (in the plasmid) should have a BglII restriction site. This would allow us to determine where the sequence upstream to the insertion occurs, and where the sequence downstream of the insertion ends.
Blast the flanking sequences against the Pseudomonas fluorescens genome to determine the position of the genomic insertion.
Retrieved from " http://2012.igem.org/Team:Calgary/Notebook/Protocols/tnscreen "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
