<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_FRED_Reporting"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/FRED/Reporting">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/FRED/Reporting&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Reporting&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Reporting&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/FRED/Reporting" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/FRED/Reporting</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Project</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Jamboree</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">A Novel Electrochemical Reporting System</H1><P>For FRED to be able to tell us about the toxins he's sensing we needed a good reporter system that could function in a wide array of environments. Unfortunately the traditional fluorescent or luminescent reporters have significant drawbacks that prevent them from being useful in a tailings environment that is murky and potentially anaerobic. Due to these limitations we decided to improve upon <A href="https://2011.igem.org/Team:Calgary">last year's single output electrochemical sensor</A> using the <I>lacZ</I> gene to cleave a substrate into an easily detectable analyte. Our team has developed a novel system that utilizes <B>three separate reporter genes</B> to provide a triple-output electrochemical biosensor and can be used in a wide variety of applications. This system overcomes traditional reporters in that it is <B>fast</B>,<B> accurate</B>, and can <B>function in turbid environments</B> and even in the <B>absence of oxygen!</B></P><H2>Why Choose Hydrolases?</H2><P>To get our bacterial biosensors to report toxic compounds present in the tailings ponds, we needed a quick and reliable system that would function in a variety of aqueous environments. We turned to electrochemistry for this, as the turbidity of the solution doesn't affect the results and nanomolar levels of chemicals can consistently be detected. The idea behind electrochemistry is that the bacteria would either cleave a substrate to produce an oxidizable product (analyte), or transfer electrons directly into an electrode. The three most common methods through which bacteria produce an electrical response are the activities of phosphatases, hydrolases, and metal respiration. </P><P>The first system, that of the respiration of metals, involves using an organism that uses metal ions, such as Fe<SUP>3+</SUP>, as the terminal electron acceptors in the cellular respiration pathways. While this kind of a system has the potential to be useful in creating bioelectricity, its use as a biosensor is limited. This is because it requires putting one of the essential electron transport genes under an inducible promoter, such that when the promoter is activated, respiration is enabled causing a change in current. Although these bacteria can usually respire more than one type of metal, they bottleneck to a single pathway and output.</P><P>The second system relies on phosphatases: enzymes that remove a phosphate group from an electrochemical analyte. When the phosphate group is removed the resultant product could be oxidized or reduced at an electrode to produce a response that would be measured as a change in current. While this method solves the problem of reduced cell viability created in the first system, it also is limited to a single output, as the non-specific phosphatases would act on all substrates in a solution. The effectiveness of the system could be further reduced by background expression of phosphatases in the bacterium, as these enzymes are essential for processes such as signalling and metabolism. </P><P>With this in mind we favoured a hydrolase based system, which offers  the versatility and sensitivity of electrochemistry, without the pitfalls of disrupting metabolism or the limitations of a single channel output.</P><H2>How Does it Work?</H2><P>The enzymes encoded by our reporter genes are specific sugar hydrolases. This means that they target one kind of sugar and remove it from whatever compound they are attached to. We have chosen to use the sugars glucose, glucuronide, and galactose for our system. The genes responsible for their respective hydrolases are <I>bglX</I> (<A href="http://partsregistry.org/Part:BBa_K902004">BBa_K902004</A>), <I>uidA</I> (<A href="http://partsregistry.org/Part:BBa_K902000">BBa_K902000</A>), and <I>lacZ</I> (<A href="http://partsregistry.org/Part:BBa_I732005">BBa_I732005</A>). By having our electrochemical analyte conjugated to this sugar, when the hydrolase is expressed the sugar is cleaved from the analyte, allowing for its electrochemical detection. A diagrammatic representation of this system is shown below in Figure 1.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 1: Representation of cleavage of the sugar-analyte substrate by a hydrolase enzyme.</DIV></DIV></DIV></DIV><P>After the analyte is released we need to detect it. Electrochemistry is an excellent approach for this because of its fast and quantitative nature. A voltage is applied between two electrodes compared to a reference electrode and the resulting current is measured. By changing the applied voltage to that of the oxidation voltage of one of our analytes, the increase in current due to its oxidation when compared to an analyte free baseline is proportional to the amount of analyte present in the solution. This process happens so quickly that you can have an output value in a matter of seconds.</P><P>We used two different electrochemical techniques in our testing depending on what question the experiment was trying to answer. When we were characterizing the voltages at which our products oxidized we used <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/cvs">cyclic voltammetry</A>, which is where you apply a voltage and then slowly increase and decrease it over a designated sweep range. Any bumps in the graph are due to a reaction and can be standardized against baseline measurements. After the oxidation potential has been localized we can speed up our experiments by using <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/potstd">potentiostatic runs</A>. In this case, instead of sweeping the voltage we apply to the solution we hold it steady at the voltage that will oxidize our compound the moment it is released into the solution. Both of these techniques require the three electrodes in an electrolyte solution such as phosphate buffered saline and can routinely detect nanomolar concentrations of electrochemical analytes.</P><H2>Genes, Chemicals, and Circuits</H2><P>For our system to have a triple output we need three separate genetic circuits with three analytes possessing unique oxidation potentials. If one chemical overlaps with another we could get false-positives of one chemical due to oxidation of another. To this end we have chosen to use chlorophenol red (CPR), para-diphenol (PDP), and para-nitrophenol (PNP). These compounds are conjugated with their sugars to form CPR-Î²-D-galactopyranoside (CPRG), PDP-Î²-D-glucopyranoside (PDPG), and PNP-Î²-D-glucuronide (PNPG). An easy way to tell the analytes from their sugar conjugates is the addition of the letter G to the acronym. These chemicals are summarized below in Figure 2 along with the reporter genes used with each one.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 2: Analyte/sugar combinations as well as the reporter genes responsible for the detection of each compound.</DIV></DIV></DIV></DIV><P>Out of the three sugar conjugates the only one that exhibits any electrochemical activity is PDPG, with it's oxidation potential at 0.6V vs. the reduction of hydrogen reference electrode (RHE). The three analytes have potentials at 0.825V for PDP, 1.325V for CPR, and 1.6V for PNP vs RHE. As none of these peaks overlap and no sugar conjugates interfere with their signals the three chemicals can be detected in the same solution. Figure 3 shows sensitive simultaneous detection of our three analytes with no background interference.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 3: Cyclic voltammogram of the three electrochemical analytes vs RHE. PDP has a peak at 0.825V, while CPR is at 1.325V and PNP is at 1.6V. The concentration of all analytes was 40ÂµM</DIV></DIV></DIV></DIV><P>With the chemicals finalized we now needed to construct our circuits. 

There is a <I>lacZ</I> gene under the control of the <I>lacI</I> promoter in the registry already registry (<A class="green" href="http://partsregistry.org/Part:BBa_I732901">BBa_I732901</A>).  Upon testing this circuit however, it was determined to not be functional. In a simple assay where we added  CPRG to cells expressing this part and induced with 200 mM IPTG, we saw no color change, as we would have expected if the LacZ enzyme was being produced and cleaving the CPRG (yellow) into CPR (red).  This can be seen below</P>.

<P> Sequencing  of this part showed that it had a frameshift mutation.  Original validation of our system was thus done using one of the constitutive <I>lacZ</I> hits from the <A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">transposon screen</A>.  After Regionals, a new circuit was constructed and submitted as <A href="http://partsregistry.org/Part:BBa_K902090">BBa_K902090</A>.  A similar assay was performed and the results can be seen above as well.  On the right is the new circuit, while on the left, the old circuit.  The red color indicates that the new pat is indeed functional.  This circuit was used in later testing using the LacZ enzyme.  The <I>bglX</I> and <I>uidA</I> genes were amplified from the <I>E. coli</I> genome using PCR and biobricked as <A href="http://partsregistry.org/Part:BBa_K902004">BBa_K902004</A> and <A href="http://partsregistry.org/Part:BBa_K902000">BBa_K902000</A> respectively. These genes were then constructed under the <A href="http://partsregistry.org/Part:BBa_R0010"><I>lacI</I> promoter</A> to allow for comparison testing.</P><H2>Does it Work?</H2><P>Yes! We have been able to show that we can detect the action of our hydrolase enzymes acting on the sugar-conjugated compounds to give us an electrochemical signal (<B>Figure 4</B>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 4: A) Detection of <I>lacZ</I> activity on CPRG at 1.325V vs RHE through the production of CPR. B) Cleavage of PDPG into PDP by <I>bglX</I> being detected at 0.825V vs RHE. C) The action of <I>uidA</I> on PNPG at 1.6V vs RHE when under the control of the <A href="http://partsregistry.org/Part:BBa_R0010">R0010</A> promoter induced with IPTG or uninduced.</DIV></DIV></DIV></DIV><P>These graphs show two main points. The first being that we can successfully use hydrolase enzymes as reporters for gene expression with a sensitive output. This gives us the power to accurately watch bacteria respond to a stimuli in real time with the ability to differentiate between minute differences in expression strength. As these reporters do not rely on having a colour or fluorescence output they can be used in turbid solutions and even solutions free from oxygen. This removes two of the major limitations of current biosensors, allowing this branch of biotechnology to access a broad new market.</P><P>The second interesting conclusion that can be drawn for part C of Figure 4 is the leakiness of the <A href="http://partsregistry.org/Part:BBa_R0010">BBa_R0010</A> promoter. The bacteria were induced at time zero and a clear increase is seen almost immediately for the induced trial, but the current does still increase over time for the uninduced test. The leaky expression of the genes downstream of this promoter could be detrimental in situations such as toxic gene expression or time dependent events.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 5: Current production from the <I>lacZ</I> and <I>uidA</I> systems under the IPTG inducible promoter. Both samples were run with the same conditions and held at the oxidation potentials of their respective analytes.</DIV></DIV></DIV></DIV><P>We have also been able to directly compare the <I>lacZ</I> and <I>uidA</I> circuits under the same promoter, as shown in Figure 5. In doing this we see twice the current in the <I>uidA</I> system as opposed to the <I>lacZ</I> system with the same conditions. This is because the analyte produced through the action of <I>uidA</I> produces a product that oxidizes to release two electrons while the <I>lacZ</I> product only releases one electron when oxidized. As the current we measure is this release of electrons, a similar amount of the two enzymes would results in the doubling of current for <I>uidA</I> that we saw, showing that our systems are working as expected.</P><H2>What Next?</H2><P>With our electrochemical system functioning properly we can now hook up our reporter genes to promoters found in the <A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">transposon library</A> for a final detection system. We have also created a <A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">hardware and software platform</A> for a field-ready biosensor. Our system has also been <A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">mathematically modeled</A> in MATLAB to aid us in planning time courses for the experiments and the final prototype. When combined with the mechanical and biological containment mechanisms used in our system these genes create a novel and safe approach to biosensing in the oil sands and in many other potential applications.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Project/FRED/Reporting">http://2012.igem.org/Team:Calgary/Project/FRED/Reporting</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/FRED/Reporting" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/FRED/Reporting" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Reporting&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Reporting&amp;oldid=298507" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>