"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Desulfurization
From 2012.igem.org
Home
Team
Members
Official Profile
The University
Outtakes
Contact Us
Project
Overview
Data Page
Accomplishments
Post-Regionals
Human Practices
Initiative
Interviews
Design
Killswitch
Regulation
Kill Genes
Safety
FRED
Toxin Sensing
Electroreporting
Modelling
Device Prototype
OSCAR
Decarboxylation
Decatecholization
Flux Analysis
Bioreactor
Upgrading
Desulfurization
Denitrogenation
Synergy
References
Attributions
Parts
Notebook
Journal
Protocols
Outreach
LAB ESCAPE!
Spark Science
Minds in Motion
Blog Posts
TEDxCalgary
Follow Us!
Sponsors
iGEM
Notebook
Journal
Killswitch
Toxin Sensing
Electrochemistry
Biosensor
Decarboxylation
Desulfurization
Denitrogenation
Catechol
Bioreactor
Flux Analysis
Protocol Manual
Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!
Desulfurization Journal
Week 1 (May 1-4)
During this week, literature search was performed.
Week 2 (May 7-11)
Along with the rest of the team, this week was dedicated to familiarizing ourselves on the protocols that will be utilized during this years project; specifically the polymerase chain reaction, gel verification, preparation of overnight cultures, as well as developing a procedural flowchart to transform competent cells with registry biobricks. With regards to our sub-group specific goals, we reviewed the current available literature around various industrial and laboratory approaches to desulfurization of organic groups, especially in the petroleum industry. This included a comparison of non-biological processes such as conventional hydrodesulfurization, which is currently employed in petroleum product refinery stages, and how a biological approach would supplement and perhaps even offer several advantages over these methods. Current limitations to biological desulfurization, however, include such factors as biocatalyst stability, enzyme specificity and desulfurization rate. We also identified the enzyme desulfinase (DszB) as being one of the bottlenecks in the desulfurization 4S pathway. Overall, our goals moving forward involve determining the specific pathways involved in the desulfurization process as well as the reaction conditions we would want to employ, and identifying specific model compounds in addition to dibenzothiophene (DBT) that we could use to test the effectivity of our biosystem in order to determine its functionality in the conversion of naphthenic acids to economically valuable hydrocarbons.
Week 3 (May 14-18)
Building on the previous week's literature review, the 4S pathway was recognized as the preferred biological mechanism that we would explore in devising a desulfurization biosystem. Of specific interest is the dsz operon consisting of the genes for dszA, dszB, and dszC which selectively and non-destructively remove the sulfur from the hydrocarbon structure, and therefore preserves the carbon skeleton. In addition to these, another dsz gene exists. dszD, which codes for a FMN:NADH reductase, is an essential component of the pathway, but not part of the operon. Instead, it is on the genome. The enzyme produced by this gene is required to regenerate the FMNH2 consumed by the reactions carried out by DszA and DszC. Rhodococcus erythropolis IGTS8 is the most studied model organism in investigations of the 4S pathway, and has been shown in many different research endeavors to be capable of converting DBT to 2-HBP.
Figure 1: The 4S Desulfurization Pathway, showing the desulfurization of the model compound DBT by DszA, DszB, DszC, and DszD. Four enzymes are involved in the 4S pathway, 3 of which are directly involved in the conversion of DBT to 2-HBP. Dibenzothiophene monooxygenase (DszC) is responsible for the first two steps of the pathway, converting DBT to DBT-sulfoxide and finally to DBT-sulfone (DBTO2) through the addition of oxygen to the sulfur atom. DBT-sulfone monooxygenase (DszA) then carries out the next step in the pathway, producing 2-hydroxybiphenyl-2-sulfinic acid (HBPS) through addition of a final oxygen to the heteroatom. This causes cleavage of the chemical bonds at the heteroatom, breaking the ring and converting the compound from a 3-ring structure to a 2-ring structure. HBPS is then converted to the final product of the 4S pathway by HBPS desulfinase (DszB), producing 2-hydroxybiphenyl. At this point, the sulfur has been released from the hydrocarbon in the form of sulfite.
An alternative to the DszD gene is HpaC, an oxidoreductase encoded in the E. coli W genome. This enzyme has been shown to increase the rate of desulfurization. Following this, other protocols added to our growing lab methods 'toolkit' were a restriction digest protocol, PCR purification, and finally, DNA construction digest. Aims moving forward include obtaining strains of the R. erythropolis , while also executing a timeline devised to biobrick, test, and incorporate the genes necessary in the above processes in a biobrick circuit.
Week 4 (May 22-25)
This week was kicked off with a project development meeting with Emily and David, and we devised a protocol for biobricking the hpaC gene. Additionally, methods to place the genes coding for the 4 enzymes, DszA,B,C and HpaC into a single construct were explored. Within the lab, the PCR performed on the resuspended pUC18-hpaC was not successful initially.   Furthermore, we ordered the substrates/compounds that we intend to use for desulfurization tests. Once the substrates and the Rhodococcus strain arrive we are going to test how effectively the bacteria can desulfurize different sulphur-containing compounds that resemble naphthenic acids. Finally, we came across a paper where a team had developed an improved efficiency DszB through site-directed mutagenesis in 2007. This was through a point mutation to the gene, converting a tyrosine at position 63 to a phenylalanine residue. A member of this team was contacted to request the plasmid that contains the mutated gene. The conversion step carried out by DszB is the major bottleneck in the 4S pathway and if a strain or sample containing this mutation was obtained, it would significantly bolster our later testing efforts on DBT, as well as other compounds such as thiophane.
Week 5 (May 28 - June 1)
Since we wanted to make sure we would not run out of pUC18(plasmid containing the hpaC gene), we transformed some E.coli cells with it. We grew them on plates containing ampicillin (A), kanamycin (K), tetracycline (T) and chloramphenicol (C) antibiotics and they only grew on A. Therefore pUC18 has A resistance. We did a three sets of PCR with primers designed against hpaC, one using 1/10 dilution of pUC18, the other using 1/100 dilution of pUC18 and one with the colonies we had just obtained by transforming the E.coli cells.
The PCR worked and we saw bands of the same size for all three sets of PCR. (Unfortunately, the picture we saved is not a good one since some of the bands faded away under UV due to prolonged exposure. Following this, PCR purification was performed to obtain the pure hpaC with biobrick prefix and suffix attatched to gene, which would allow us to insert the sequence into a biobrick standard backbone. 3 sets of digestion, ligation, and transformation (using pairs of X&P enzymes, E&S enzymes and E&P enzymes) were carried out in order to insert the hpaC gene into the pSB1C3 vector. All the sets grew successfully. Following the above successes with hpaC, the arrival of our Rhodococcus strain afforded us the opportunity to begin investigation of the Dsz operon using the primers currently in our possession. This strain is an environmental isolate that has been shown to be an active desulfurizer. The gram-positive nature of the strain also dictated we explore various lysing strategies before the genes encoding the Dsz enzymes could be amplified for further purification and biobrick construction steps. PCR was carried out using dszA primers on three different treatments {microwave, lysate buffer, and a control} which yielded banding pattern around 1200 base pairs for the lysate treatment (2%SDS and 10% tritonX-100, plus heat for 5mins at 98C).
Week 6 (June 4 - June 8)
In order to confirm the hpaC biobrick construction, two sets of colony PCR were performed, choosing white colonies from the 3 plates we grew last week (white colonies indicate a loss of the RFP generator in the pSB1C3 backbone, and therefore allow for weeding out of the colonies which are simply the original plasmid vector). These reactions were carried out both with hpaC primers and with standard biobrick primers designed against the plasmid backbone. After running them on the gel we saw equal bands for the PCR reactions performed using hpaC primers (However, a PCR using biobrick primers was performed later and the same results were obtained). Colonies 1(-) and 5(-) were used to make overnight cultures, which were then miniprepped the following day to obtain the plasmid DNA of the putative hpaC biobrick. Digestions were performed on the miniprep products using EcoRI and PstI to look for part size as further verification for the genes presence in the plasmid. The results were good and two bands were observed on each column (one for vector and the other for hpaC)). hpaC was sent in for sequencing.
Figure 2: HpaC verification cPCR. HpaC gene was inserted into the pSB1C33 vector and E. coli cells were transformed. In order to confirm that pSB1C3 contains the hpaC gene, two sets of colony PCR's were conducted. One with biobrick primers, and the other with hpaC primers. Bands indicate successful amplification at the approximate size of the hpaC gene (517 bp)
Figure 3: HpaC confirmation digest. Plasmid was miniprepped and digested for 2h with EcoRI and SpeI before being run on a 1% gel for 1h and 120V. Bands between 500 and 700 bp indicate the hpaC part is present as an insert. Bands of about 2000bp show the size of the psb1c3 vector.
PCR reagents were prepared to re-test/confirm previous results of dszA amplification following two different lysing treatments (microwave + lysate buffer). This time, all three genes were amplified and gel verification showed clear banding patterns around 500bp range for all three genes for the microwave treatment. Remaining PCR products were run on a gel and extracted for further purification steps; however, presence of any genetic material were not confirmed through nanodropping which raised concerns about the composition of the purified products, the success of the initial amplification step, or perhaps even the lysis treatment. Further experimentation will have to be carried out to troubleshoot.
Week 7 (June 11 - June 15)
This week, we focused on amplifying dsz genes from our Rhodococcus strain for construction into biobricks. We also wanted to purify the BBa_K902057 hpaC and pUC18-hpaC plasmids to replenish our current stocks. For the dsz aspect, we were able to successfully grow extra plates of Rhodococcus strain which was used to inoculate PCR tubes. The PCR did not go well, with significant streaking and false positives with similar banding pattern to previous gels run in the previous week. A final gel verification of a random sample of a tube of PCR products from dszA,B,C respectively and two negative control treatments involving master mix only and the lysed cells only illustrated the lack of discrepancy between the supposed successful amplification and the lysed cells (with lysate buffer) alone. Because of this we decided to take a different approach involving plasmid isolation carried out before PCR, rather than applying the PCR reagents directly to a lysed culture sample.
hpaC verification through sequencing was successful, confirming the construction of our first biobrick. Subsequently, O/N cultures of the plasmid containing cultures were prepared and stored in glycerol at -80C. Furthermore, verification of catalase gene part ( katG-LAA ), which was sent as a culture stab from the parts registry was initiated. Our newly identified biobricked-hpaC was used as a positive control, but the banding pattern was not very conclusive.
Week 8 (June 18 - June 22)
PCR was reattempted on Rhodoccocus that was lysed using two different dilutions of the lysate buffer, but the gel verification confirmed the previous failure in using this approach. An alternative that involved preparation of an overnight culture of the Rhodococcus cells followed by a plasmid purification was followed. The plasmid purification eventually yielded plasmid samples with concentrations of 98.6ng/μL to 182.7ng/μL (4 samples obtained overall). Additionally, the catalase biobrick was used to transform some stock competent cells, and samples of some colonies were subsequently PCR'ed. Although, the gel verification showed some potential contamination, and the required banding patterns at around 2200bp was not obtained.
Week 9 (June 25 - June 29)
PCR was attempted to amplify the genes of the dsz operon utilising an adapted PCR protocol with purified Taq polymerase that had been isolated from the host organism. Eventually, some banding pattern was obtained between 1200 and 1500 base pairs when a gradient thermocycler was used with melting temperatures ranging betweeen 55°C to 65°C. This was assumed to be indicative of successful amplification of dszB; however, further purification and gel verification results were inconclusive and no yield was obtained when placed tested using a nanodrop machine.
Week 10 (July 2-July 6)
Top 10 E.coli cells were transformed with BBa_R0011 (IPTG inducible promoter in psb1C3 backbone), and resulting colonies were tested using cPCR.  Colony PCR was performed on cells containing the catalase biobrick. Catalase is 2217bp long but since biobrick primers add about 200bp, bands of 2400 bp were expected if the part was present in the biobrick. These bands were observed, indicating that the katG-LAA gene was most likely present.
Figure 4: Colony PCR on potential colonies containing katG-LAA. Biobrick primers were used. Bands at 2200 indicate successful amplification of the catalase part within the biobrick backbone. Smaller bands are indicative of primer degradation and are nonspecific amplification. +C indicates the positive control, and RFP generator, which should amplify at around 1000 bp.
PCR using Phusion high fidelity polymerase was carried out on dszA, dszB, and dszC in a gradient thermocycler. Amplification of non-specific bands was present for dszA and dszB, however strong banding for the desired size of the gene was observed for both (around 1500 for dszA, 1100 for dszB
Figure 5:DszA and DszB PCR from Rhodococcus. DszA amplicon is around 1300 bp, and is observed to run higher on the gel. DszB amplicon is expected to be 1098 bp, which is observed in addition to multiple nonspecific banding. +C indicates the positive control, and RFP generator, which when PCRed with biobrick primers should be around 1000 bp. As this control is running high as well, it is believed that both amplicons for DszA and DszB have been obtained. No contamination is observed in the NTC (no template control).
Figure 6:DszC PCR from Rhodococcus. The numbers above the figure show the colony number. DszC amplicon is around 1450 bp, and is observed to run at almost the right size on the gel. No contamination is observed in the NTC (no template control).
Examining the sequences of the dszABC genes led to the discovery that all 4 had multiple illegal enzyme cut-sites in them that we have to eliminate before biobrick composite part construction can occur. dszA has four PstI cut sites, dszB has a PstI and a NotI and dszC has a PstI cut site. The Stratagene QuickChange mutagenesis procedure is going to be used to eliminate illegal cut sites with the only alteration being that Kapa HiFi polymerase would be used during the process. Primers needed for the mutagenesis were designed based on the procedure mentioned above.
Week 11 (July  9-July 13)
Following successful amplification of the dsz operon genes in the previous week, the genes were constructed into the pSB1C3 vector. Colony PCR verifications were observed to be positive. Furthermore, the insertion of part BBa_J13002, (PtetR-RBS) in front of the previously biobricked hpaC was attempted. Overnight cultures were also prepared using two colonies each for PtetR-RBS and PlacI (an IPTG inducible promoter that we hope to build in front of
an RBS site, BBa_B0034). These cultures were then miniprepped to yield the respective parts.
Additionally, katG-LAA was built into a PSB1C3 backbone. The construction and availability of all these parts will be critical in the construction of our overall circuit for biodesulfurization. Colonies which looked good on cPCR were used to prepare overnight cultures, and were miniprepped and sent in for sequencing verification the following day. On the side, M9 minimal media was also prepared to carry out growth experimentation and overall desulfurization capability of Rhodococcus when exposed to DBT. The various growth treatments were M9 Media and glucose only, M9+glucose+DBT, M9+glucose+MgSO4+/-DBT, M9+glucose+MgCl2+/-DBT. 0.008g of FeCl2.4H2O was also added to each of the tubes. Samples were then inoculated with colonies of the Rhodococcus.
Week 12 (July  16 -July 20)
This week, while awaiting sequencing verification results which were required before we could begin the construction process, the desulfurization team initially aided in some of the tasks related to the other hydrocarbon groups. The success of the construction of PtetR-RBS with hpaC was also explored by using forward and reverse primers of BBa_R0040 (the promoter component of the composite part BBa_J13002). However, the eventual gel verification was inconclusive and sequencing results finally indicated an unsuccessful ligation. Additionally, the minimal media M9 preparation had been contaminated in the previous effort so this process was repeated to create tubes of each of the growth condition treatments detailed previously, and two repeats, one with an extra filtration step and one without was used to prepare the cultures.
Week 13 (July 23 - July 27)
Mutagenic primers were redesigned after the initial ones were found to have premature stop codons. As part of the redesign process in constructing our overall gene circuits for desulfurization, a backbone switch of BBa_R0011 into a chloramphenicol (Chlor) resistant vector was necessary. The subsequent transformed products were plated on a Chlor plate and selected colonies were used to prepare O/N cultures, then minipreped before finally being digested with enzymes EcoRI and PstI. The resulting gel verification images were inconclusive as they did not show the required banding pattern around 50bp. Meanwhile, colony PCR was run on colonies transformed with katG-LAA constructed into a PSB1C3 backbone, as well as the hpaC + BBa_J13002 construct. katG-LAA was shown to have been successfully amplified, so overnight cultures were prepared and subsequently miniprepped. On the other hand, the construct was not successful so a third attempt was carried out. Colony PCR treatments that used either BBa_R0011 forward primers or BBa_B0034 primers were used and the overall constructs were made either on a chlor-resistant, or ampicillin-resistant vectors. Preliminary images of the gel verification appeared to have confirmed the construct, although sequencing verification will be the final indicator of overall success.
Week 14 (July 30 - August 3)
Sequencing results from the previous week's constructs were available confirming that we constructed KatGLAA in a chlor-resistant backbone. However, switching the plasmid backbone of BBa_R0011 to PSB1C3 was not successful.  The construction of BBa_J13002 + hpaC was finally sent in for sequencing.  Site-directed mutagenesis of the dsz operon was also initiated: dszA has four PstI cut sites; dszB has a PstI and a NotI site; dszC has two PstI cut sites.  Site directed mutagenesis was started this week to change a single base pair in these genes in a way that eliminates the cut site but preserves the amino acid codons, so as to not mutate the protein coding sequence.  Ohshiro 2007 demonstrated that replacing the Tyr residue at position 63 of dszB gene with a Phe increases the activity of the enzyme.  Therefore we want to introduce the same mutation into our dszB.
For the first attempt at mutagenesis we chose to mutate the second PstI site in dszC (PstI2).  As a positive control for the procedure, we also performed the mutagenic PCR on a plasmid containing the β-galactosidase gene with a point mutation where the PCR would cause it to regain its function.  For both mutagenesis protocols we used the Kappa Hifi kit.  After confirming that the PCR worked by running some produce on a gel, the PCR products were DpnI digested, the purpose of which is to degrade the unmodified parental DNA (DpnI degrades methylated DNA only).  Control PCR products were plated on an ampicillin plate containing IPTG and X-gal. The colonies that grew on the control plates were blue indicating that the mutagenesis had worked for the β-galactosidase gene.  Minipreps of the O/N culture of dszC mutants were digested with PstI enzyme and the results indicated that the mutagenesis was successful.
Attempts to simultaneously perform all the mutations in dszC genes in one step using the Knight procedure failed ( Knight Multi-site Mutagenesis Procedure ). What enables simultaneous mutations is that Taq ligase closes the gaps in PCR products after each cycle. In the protocol it instructs to use Taq ligase buffer only for the PCR/ligation protocol. We suspected that the reason this procedure did not work might be that the Kappa polymerase is not functional in Taq ligase buffer. Therefore we did some experiments on the controls in Taq ligase kit and kappa polymerase kit to find out which buffer that Kappa polymerase and Taq ligase both work best in. The result was that both enzymes work best in a buffer made of half Taq ligase buffer and half Kappa polymerase buffer.
Figure 7: DszC PstI2 mutagenesis PCR with varying concentrations of template plasmid was performed. The gel shows the PCR products that were run on a gel. +C lanes show the PCR products of the control. The control was pWhitescript™ 4.5-kb control plasmid from Stratagene mutagenesis kit which was mutated with primers also from the kit. Bands indicate successful amplification of the plasmid.
Figure 8: The control and the mutated plasmids were digested with PstI restriction enzyme. The control is dszC biobrick. Other lanes show the plasmids purified from the cells transformed with PstI2 mutagenesis PCR products followed by digestion. The control shows three bands since it has three PstI cut sites. The other lanes show two bands which indicates one of the cut sites has been eliminated through mutagenesis.
Figure 9: The multisite mutagenesis using the Knight procedure was tried in different buffers to find out the buffer that Kappa works optimally at. All the PCRs were performed on the pWhitescript™ 4.5-kb control plasmid from Stratagene mutagenesis kit which contains the beta-galactosidase gene. Based on this gel, the optimal buffer is composed of 50% kappa buffer and 50% Taq ligase buffer.
Figure 10: Using the control provided in the NEB Taq ligase buffer (BsteII digested lambda DNA), we tried to find if the Taq ligase enzyme can function in combinations of Kappa Hifi buffer and Taq ligase buffer. Lane two is the control which is only the digested lambda DNA. Lanes 1 and 3 show that some of the bands compared to control have been ligated together. Therefore, Taq ligase functions just as good in the buffer composed of 50% kappa hifi buffer and 50% Taq ligase as it would in its own buffer.
Week 15 (August 6 - August 11)
Sequencing results for BBa_J13002 hpaC returned negative, so a 3-part ligation method was used to retry this construction. The following parts were ligated with the restriction enzymes indicated in brackets after each: BBa_J13002 (EcoRI/SpeI) + hpaC (XbaI/PstI) + BBa_PSB1K3 (EcoRI/PstI).  Also, the more conventional construction (only 1 insert) of BBa_J13002 (SpeI/PstI) + hpaC (XbaI/PstI) was reattempted. Furthermore, 3-way ligations were also attempted for BBa_B0034 + katG-LAA + BBa_PSB1K3 , and BBa_R0011 + BBa_B0034 + PSB1C3 , as well as the two-way contruction of just katG-LAA after the BBa_B0034 . After plating these transformations, colony PCRs were carried out and samples that gave an indication of being successful on the gels were used to prepare O/N cultures followed by miniprep.  With regards to the site-directed mutagenesis side of the experimentation, dszA-PstI1 (the first PstI cut site in dszA) , dszB-PstI and dszC(PstI2 mutated)-PstI1 mutagenesis were performed following the procedure explained in the previous week. The gel below shows the successful result of digest confirmation (Fig. 8).  Multisite mutagenesis (Knight method) was repeated using the modified buffer (half Taq ligase buffer and half Kappa buffer). However it was not successful again. We also tried doing multisite mutagenesis using Pfu Turbo polymerase and following the Knight procedure without any buffer modifications. No successful results were observed.
Figure 11: Confirmation of site-directed mutagensis to eliminate PstI cut sites in: dszAPstI1 cut site, dszBPstI cut site and dszCPstI1 (PstI2 site mutated).  Note that in each case, the unmutated controls for each gene are to the left of each set of plasmids derived from mutagenic PCRs.  Plasmids from cloned dsz genes are smaller than the bands to the right of them (which have undergone mutagenesis), indicating that in the plasmids on the right, the PstI cut site was lost.
Week 16 (August 12 - August 18)
The progress in mutagenesis of dsz genes was continued from the previous week: dszB(PstI mutated)-Y63F and dszA(PstI1 mutated)-PstI3 mutagenesis. The gel below shows the digest confirmation.
Figure 12: Digestion confirmation of biobricks of a) dszA (PstI1 mutated) PstI3 mutagenesis and b) dszB (PstI mutated) Y63F mutagenesis. Cutting dszA (PstI1 mutated) with PstI is expected to produce fragments of 2790, 333, 255, and 114bp. Cutting dszA (PstI1 and PstI3 mutated) is expected to produce fragments of 2730, 588, and 114bp. The primers for dszB-Y63F mutagenesis introduce an HpyAV cut site. pSB1C3 inteslf has two more HpyAV cut sites. Before the mutation bands of 2858 abd 310bp are expected, and after successful mutation bands of 1502, 1356, 310 are expected. Lane legend: 1- dszA (PstI1 mutated). 2-dszA (PstI1 mutated) digested with PstI. 3-dszA (PstI1 mutated) PstI3 mutagenesis c1 Digested with PstI. 4- dszA (PstI1 mutated) PstI3 mutagenesis c2 Digested with PstI. 5-dszA (PstI1 mutated) PstI3 mutagenesis c3 Digested with PstI. 6- dszA (PstI1 mutated) PstI3 mutagenesis c4 Digested with PstI. 7- Fermentas 1kb Plus Ladder. 8- Empty. 9- dszB (PstI mutated). 10-dszB (PstI mutated) digested with HpyAV. 11- dszB (PstI mutated) Y63F mutagenesis c1 digested with HpyAV. 12- dszB (PstI mutated) Y63F mutagenesis c2 digested with HpyAV. 13- dszB (PstI mutated) Y63F mutagenesis c3 digested with HpyAV. 14-dszB (PstI mutated) Y63F mutagenesis c4 digested with HpyAV.
We attempted a different approach to speed up the turnover time of the mutagenesis PCR. Briefly, after the PCR mutagenesis the PCR products were purified and then incubated with T4 polynucleotide kinase (PNK) and ligase.  After heat inactivating the ligase and T4 PNK, the products were DpnI digested. Subsequently another round of DNA purification was performed. However, the results were unsatisfactory after the digest confirmation.
Sequencing results came back. dszA (PstI1 and PstI3 mutated) and dszB(PstI and Y63F mutated) were good. However dszC (PstI1 and PstI2 mutated) had an insertion next to the PstI1 cut site. Mutagenesis was repeated on the dszC(PstI2 mutated). dszB(PstI and Y63F mutated)-NotI and dszA(PstI1 and PstI3 mutated)-PstI4 mutagenesis were also performed.
To investigate the desulfurisation capability of the Rhodococcus sp. from which we cloned the dsz operon, a desulfurization assay was prepared by inoculating different treatments of M9 media.  We also prepared some solutions that will be needed for analysis in the following week: a conditioning agent composed of 100ml of 95% ethanol, 50ml glycerol, 30ml of 12M HCl (aq) and 70g of NaCl(s) was prepared. The assay relies on the turbidity of a sample containing sulphate ions which are precipitated (hence the turbidometric nature of the assay) upon adding BaCl2(s), therefore if the dsz pathway is active, we expect a more turbid solution to form than in control samples.
Week 17 (August 19 - August 25)
This week, progress was made in determining the desulfurization activity of our Rhodococcus strain as measured by the sulfate release using a turbidometric assay. We encountered several challenges in our prescribed protocol as the concentrations that we used to prepare the standard curve may have been too dilute, or the composition of out conditioning agent may have been flawed. Additionally, steps were taken to determine the decomposition of DBT to 2-HBP through Gas Chromatograph-Mass Spectroscopy (GC-MS) analysis, but due to a preparation error, the DBT was added to a growth solution of M9 media prematurely and the autoclaving process decomposed the DBT releasing a yellow colouration into the solution. These two approaches in determining the desulfurization capability of the dsz operon will be further investigated.
Since the dszC second mutagenesis had proven to be unsuccessful last week, the dszC(PstI2 mutated)PstI1 mutagenesis was repeated. Also dszA(PstI1,3,4 mutated) PstI2 mutagenesis was performed. dszA and dszC were sent for sequencing on Wednesday. dszB was sent for sequencing on Friday. Sequencing results of dszA and dszC were back by Friday. dszC was successful. However, dszA contained an insertion next to the binding site of PstI4 cut sit, so the last two mutations must be redone. dszB(PstI and Y63F mutated)-NotI-mutagenesis was also repeated in case the result of the sequencing was not successful. These constructions were repeated. BBa_J13002 - dszB ,  and BBa_B0034 -dszC constructions were attempted, however they were not successful as indicated by colony PCR. Constructions of BBa_J13002 /hpaC were carried out and also came back negative in sequencing, however BBa_B0034 /katG-LAA (BBa_K902059) was sequence confirmed.
Figure 13: Another attempt to mutagenize dszC(PstI2 mutated)at PstI1 site. The gel shows the confirmation digestion. C is the control which is dszC(PstI2 mutated). Since the control has two PstI cut sites, two bands are observed on the gel. The mutagenesis has been successful since only one band is observed on the other lanes indicating that one of the cut sites has been eliminated during mutagenesis.
Figure 14: DszA (PstI1, PstI2, PstI3 mutated)PstI4 mutagenesis digestion confirmation gel. All the lanes show the results of the plasmids being cut with PstI enzyme. C indicates the lane containing the control which is dszA(PstI1, PstI3 and PstI4 mutated). The other lanes are the digestions of the plasmids that are mutagenized, from different colonies. Control plasmid has two PstI cut sites and therefore the two bands observed on the gel were expected. The other lanes show only one band of about 3500bp which is about the right size for dszA in a psb1c3 vector. Therefore one of the cutsites has been eliminated during mutagenesis.
Week 18 (August 26 - September 1)
dszB (BBa_K902052) sequencing results came back as successful. dszA(PstI1,3 mutated)-PstI2-mutagenesis was performed and sent for sequencing. Also dszA(PstI1,2,3 mutated)-PstI4-mutagenesis was performed, and this was also sent for sequencing.
Constructions of PtetR-RBS/dszB (BBa_K902053) and BBa_B0034 /dszC (BBa_K902056) were attempted, verification digested, and sent for sequencing. Sequencing results for these constructs came back as positive, along with successful mutagenesis of dszA (BBa_K902050) .
At this point, all of the dsz genes have been successfully made biobrick compatible, and hpaC has been biobricked. We have also successfully constructed BBa_B0034 with katG-LAA to be used in the optimization circuit, as well as PtetR-RBS/dszB (BBa_K902053) and BBa_B0034/dszC (BBa_K902056) .
Constructions of PlacI-RBS with hpaC , dszB , and katG-LAA were performed. As well, attempts to construct PtetR-dszB / RBS-dszC as well as PtetR-RBS / hpaC , BBa_B0034 / dszA , and BBa_J13002 / katG-LAA were also carried out. These parts are intended as construction intermediates towards building the final systems, as well as providing a way of testing the genes functionality (namely, to test HpaC for oxidoreductase activity and to test if over-expression of KatG in the cell will increase its ability to survive H2O2 stress). Transformations of all these constructions were carried out at the end of the week.
Week 19 (September 2- September 8)
Confirmation digests on colonies of the previous constructions that gave bands of the expected size with cPCR were performed. Positive results were found for colonies of PlacI-RBS / hpaC , PlacI-RBS / dszB , and BBa_B0034 / dszA . Sequencing was sent, and results indicated that the constructions of PlacI-RBS/hpaC (BBa_K902058) were successful, meaning that after many months of trying we FINALLY have a promoter in front of the hpaC gene and can proceed to test the parts functionality. Attempts to construct hpaC with the ( evil TetR promotor, BBa_J13002 ) were abandoned, as it was believed that this construction was failing due to toxicity of over-expressing the protein, and it was determined that this part was not necessary after all. PlacI-RBS/dszB, (BBa_K902054) also worked, though this was less exciting. BBa_B0034 /dszA came back as a bad read despite looking very good on the confirmation digest gel, so this part will be resent for sequencing. Constructions of PlacI-KatG were reattempted, and confirmation digests for this part looked good, and so samples were sent for sequencing.
Week 20 (September 9- September 15)
Construction attempts on PtetR-dszB / BBa_B0034-dszC , PlacI-dszB / BBa_B0034-dszC , and PlacI-hpaC / BBa_B0034-katG-LAA were performed. Colonies grew for the constructions, however further confirmation results were dissapointing (only 2 clones of PtetR-dszB / BBa_B0034-dszC appeared to have been successful). These clones were sent for sequencing, and constructions were reattempted.
However, when sequencing came back, somehow reads indicated that these clones were in fact a gene from the Denitrogenation project (which is 990bp and a completely different band then what we saw on the gel). We believe, somewhere, something has gone very wrong- further investigation into this will be carried out. In the meantime, the above constructions were reattempted, and BBa_B0034/dszA was re-prepped in case a contaminant in the plasmid stock was to blame for the bad reads found in this batch of sequencing as well as the last. In addition, plasmid switches of multiple sequence confirmed parts into a pSB1C3 backbone were carried out.
Week 21 (September 16- September 22)
Colonies for the transforms of PtetR-dszB / BBa_B0034-dszC , PlacI-dszB / BBa_B0034-dszC , and PlacI-hpaC / BBa_B0034-katG-LAA have been few and far between, and cPCR results are always discouraging. Sequencing results for other sections of the project have once again come back very confusing, and further research continues into the source of this madness.
The sulfur compound degradation assay was set up to test the desulfurization rate of the original Rhodococcus baikonurensis (refer to the protocol page).
Week 22 (September 23- September 29)
Attempts to construct PtetR-dszB / BBa_B0034-dszC , PlacI-dszB / BBa_B0034-dszC , , and PlacI-hpaC / BBa_B0034-katG-LAA continue. In the meantime, PlacI-KatG was tested for functionality. In order to do this, cultures of PlacI-KatG were grown up overnight in LB media. A strain carrying BBa_J04500 only was used as a negative control. The following morning, 20 µL of each culture was inoculated into 3 mL of LB with various concentrations of hydrogen peroxide; 0 mM, 1 mM, 5 mM, and 10 mM. These cultures were then allowed to grow overnight, and culture turbidity was observed. It was found that the negative control exhibited no growth after 12h at 1 mM peroxide, however cultures with induced expression of catalase were turbid after 12 h of growth at this concentration (Fig. 10). This demonstrated the ability of the catalase to protect the cells from excessive peroxide concentrations.
Figure 15: Catalase Assay. Overnight cultures of J04500 and J04500-KatGLAA were innoculated into 0 mM, 1 mM, 5 mM, and 10 mM peroxide. Cultures were grown overnight and turbitity was observed.
In addition to this, activity of PlacI-hpaC was tested. In order to do this, cultures of PlacI-hpaC and PlacI-dszB were grown up overnight in LB with appropriate antibiotics. Following this, protein expression was induced with IPTG, after which the assay was carried out as described in the following figure and on the protocols page.
Figure 16: HpaC Assay. Cultures of PlacI-hpaC and PlacI-dszB were grown up overnight in LB with appropriate antibiotics. The following morning, cells were subcultured 1/4 into LB with 200 µM IPTG and allowed to grow for 2h in order to induce protein expression. 2 mL samples of cells were then transferred to 2 mL tubes, washed twice in 50 mM Tris-HCl (pH 7.5) and resuspended in this buffer. Samples were then subjected to 5 freeze-thaw cycles in order to lyse cells. After spinning down samples, 1 mL of cell lysate was transferred to a cuvette, and a spectrophotometer was blanked at 340 nm (maximal absorbance of NADH) with this sample. 140 µM NADH and 20 µM FMN was then added, the cuvette was quickly inverted, and readings were taken at 340 nm. PlacI-dszB was used as a control to measure native amounts of oxidoreductase activity, whereas the PlacI-hpaC cultures were used to measure activity when HpaC was expressed. The control was just Tris-HCl buffer with the NADH and FMN compounds added. Decrease in absorbance at 340 nm corresponds to the loss of NADH as it is converted to NAD+.
When the assay was run, it was found that NADH does not convert readily to NAD+ on its own. When cell lysate containing the naturally expressed amounts of oxidoreductase was added, a decrease in absorbance could quickly be observed as the NADH was converted to NAD+. When cultures over-expressing HpaC were tested, the absorbance levels were found to start much lower than the control. We believe that this is because with the amount of cell lysate tested, when the HpaC protein is overexpressed the NADH is consumed almost immediately and therefore the data reflecting the drop in absorbance is missed. Further tests will use differing amounts of cell lysate in order to try to capture data that shows the drop in absorbance for HpaC cultures.
Week 23 (September 30-October 3): Wiki-Freeze PANIC!!
The GCMS results of the sulfur assay were received. DBT desulfurization was successful (figures below). Some compounds were not detected by GCMS since they were too polar. However, 2 additional compounds showed degradation in addition to DBT, indicating that the pathway has a wider substrate specificity!
Figure 17: Rhodococcus cells were grown in a modified M9 media containing 0.125mM DBT with no sulfur containing compounds (refer to desulfurization assay protocol in the protocols section for details). Samples were taken out at different time points and were run through GCMS to detect the amount of DBT. The control only contained modified M9 but no bacteria, run through the GCMS after 6 days of being in the incubator to account for abiotic degradation.
Figure 18: The peak in this mass spectra demonstrates presence of DBT based on its molecular weight of 184 g/mol. This peak is based on the average of our samples at retention time of 13.9 minute (refer to previous graph).
Figure 19: Rhodococcus cells were grown in a modified M9 media containing 0.125mM of the indicated compound (A: dibenzothiophene, B: tetrahydro-4h-thiopyran-4-one, and C: benzo[b]thiophene-2-carboxyaldehyde)  with no other sulfur containing compounds present in the media (refer to desulfurization assay protocol for details). Samples were taken out at different time points and were run through GCMS to detect the amount of compound remaining. Samples were normalized to a control containing modified M9 but no bacteria, run through the GCMS at the last time point to account for abiotic breakdown. Degradation is seen for DBT (the model studied compound) as well as other sulfur containing compounds resembling naphthenic acids, indicating that the pathway may have wider substrate specificity than previously thought.
The HpaC assay was repeated using 100 µL of cell lysate instead of 1 mL of 2x concentrated cell lysate, following the previously used protocol. With this smaller amount, a clear sharp decrease in the absorbance of NADH can be observed, indicating a very fast conversion by HpaC, and further confirming that the part was functional.
Figure 20: HpaC Assay. Cultures of PlacI-hpaC and PlacI-dszB were grown up overnight in LB with appropriate antibiotics. The following morning, cells were subcultured 1/4 into LB with 200 µM IPTG and allowed to grow for 2h in order to induce protein expression. 1 mL samples of cells were then transferred to 2 mL tubes, washed twice in 50 mM Tris-HCl (pH 7.5) and resuspended in this buffer. Samples were then subjected to 5 freeze-thaw cycles in order to lyse cells. After spinning down samples, 100 µL of cell lysate was transferred to a cuvette, and a spectrophotometer was blanked at 340 nm (maximal absorbance of NADH) with this sample. 140 µM NADH and 20 µM FMN was then added, the cuvette was quickly inverted, and readings were taken at 340 nm. PlacI-dszB was used as a control to measure native amounts of oxidoreductase activity, whereas the PlacI-hpaC cultures were used to measure activity when HpaC was expressed. The control was just Tris-HCl buffer with the NADH and FMN compounds added. Decrease in absorbance at 340 nm corresponds to the loss of NADH as it is converted to NAD+.
Week 24 (October 4- October 11)
This week, before the Americas West Regional Competition, attempts to construct the previously mentioned parts continued. Unfortunately, they were unsuccessful.
Week 25 (October 16- October 20)
Traditional attempts to build the sulfur operon and its sub-parts were repeated, however attempts failed once more. During this week, it was decided that due to time consstraints, alternative approaches to construct the operon would be undertaken. Due to this, splice-overlap-extention (SOE) PCR primers (containing scar-sites where appropriate) were designed according to protocol and ordered.
Primer: 1a (BBK)-J13/DszB F - 5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGTCCCTATCAGTGATAGAGATTGACATCCC-3'
Primer: 1b (BBK)-J04/DszB F - 5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGCAATACGCAAACCGCCTCTCCCC-3'
Primer: 2 Jxx/B-(RBS/C) R - 5'-GGTGACAGTGTCATCTAGTATTTCTCCTCTTTCTAGTACTATCGGTGGCGATTGAGGCTGTTG-3'
Primer: 3 (Jxx/B)-RBS/C F - 5'-CAACAGCCTCAATCGCCACCGATAGTACTAGAAAGAGGAGAAATACTAGATGACACTGTCACC-3'
Primer: 4 RBC/C-(RBS/A) R - 5'-GCCAGATGCATTTGTCGTTGTTGAGTCATCTAGTATTTCTCCTCTTTCTAGTATCAGGAGGTGAAGCCGGGAATCG-3'
Primer: 5 (RBS/C)-RBS/A F - 5'-CGATTCCCGGCTTCACCTCCTGATACTAGAAAGAGGAGAAATACTAGATGACTCAACAACGACAAATGCATCTGGC-3'
Primer: 6 RBS/A-(J04/HpaC) R - 5'-GGGGAGAGGCGGTTTGCGTATTGCTAGTATCATGAAGGTTGTCCTTGCAGTTGTGG-3'
Primer: 7 RBS/A-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCATGAAGGTTGTCCTTGCAGTTGTGG-3'
Primer: 8 (BBK)-J04/HpaC F - 5'-CGAATTCGCGGCCGCTTCTAGCAATACGCAAACCGCCTCTCCCC-3'
Primer: 9 (RBS/A)-J04/HpaC F - 5'-CCACAACTGCAAGGACAACCTTCATGATACTAGCAATACGCAAACCGCCTCTCCCC-3'
Primer: 10 J04/HpaC-(RBS/Kat) R - 5'-CTGACGTGCTCATCTAGTATTTCTCCTCTTTCTAGTATTAAATCGCAGCTTCCATTTCCAGCATCAC-3'
Primer: 11 J04/HpaC-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTAAATCGCAGCTTCCATTTCCAGCATCAC-3'
Primer: 12 (J04/HpaC)-RBS/Kat F - 5'-GTGATGCTGGAAATGGAAGCTGCGATTTAATACTAGAAAGAGGAGAAATACTAGATGAGCACGTCAG-3'
Primer: 13 RBS/Kat-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTATTAAGCAGCCAGAGCGTAGTTTTCGTC-3'
Colony PCR was done on potential colonies of PtetR-dszB / BBa_B0034-dszC , PlacI-dszB / BBa_B0034-dszC , , and PlacI-hpaC / BBa_B0034-katG-LAA . Faint bands around the expected size were seen, and these were stabbed with a pipette tip and stirred into water in order to be used as template in a further round of PCR to get amplification of the part. The plan was that these amplicons could then be used as an insert in further constructions, however no amplification was seen. Further optimization of this procedure could possibly result in a short-cut to construction, however the time needed to do this is a luxury that we do not have.
Additional attempts to create inserts by doing PCR on ligation reactions in addition to gel extraction of colony PCR bands were attempted as well, however these attempts were unsuccessful as well. SOE PCR and possible Gibson Assembly appear to be the Sulfur Teams' last hope at this point, and we are eagerly awaiting our primers.
Week 26 (October 21- October 26)
Much to our delight, SOE PCR primers were recieved this week. Therefore, alongside traditional construction of the aforementioned parts, SOE PCR was carried out according to protocol and these parameters:
Round 1:
Primers 1a and 2 with PtetR-dszB as a template = Product L1 (J13002/DszB with BBk prefix and B0034/DszC overlap)
Primers 1b and 2 with PlacI-dszB as a template = Product L2 (J04500/dszB with BBk prefix and B0034/dszC overlap)
Primers 3 and 4 with BBa_B0034-dszC as a template = Product L3 (B0034/dszC with dszB and B0034/dszA overlap)
Primers 5 and 6 with dszA as a template = Product L4 (B0034/dszA with dszC and J04500/hpaC overlap)
Primers 5 and 7 with dszA as a template = Product L5 (B0034/dszA with dszC overlap and BBk suffix)
Primers 8 and 10 with Plac-hpaC as a template = Product L6 (J04500/hpaC with BBk prefix and B0034/katG-LAA overlap)
Primers 9 and 10 with Plac-hpaC as a template = Product L7 (J04500/hpaC with dszA and B0034/katG-LAA overlap)
Primers 9 and 11 with Plac-hpaC as a template = Product L8 (J04500/hpaC with dszA overlap and BBk suffix)
Primers 12 and 13 with RBS-katG-LAA as a template = Product L9 (B0034/katG-LAA with J04500/hpaC overlap and BBk suffix)
Figure 21: Round 1 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with varying amounts of DNA template as indicated above the wells. Cycling conditions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 55%deg;C for 1 min, 72%deg;C for 2 min. 30 s.)x 31, Stage 3 (72%deg;C for 10 min.). Bands seen at ~1500 bp for L1 through L5, ~800 bp for L6-L8, and ~2200 bp for L9 indicate successful amplification of the desired product.
Round 2
Primers 1a and 4 with templates L1 and L3 = Product X1 (J13002/dszB/B0034/dszC with BBk prefix and B0034/dszA overlap)
Primers 1b and 4 with templates L2 and L3 = Product X2 (J04500/dszB/B0034/dszC with BBk prefix and B0034/dszA overlap)
Primers 3 and 6 with templates L3 and L4 = Product X3 (B0034/dszC/B0034/dszA with dszB and J04500/hpaC overlap)
Primers 3 and 7 with templates L3 and L5 = Product X4 (B0034/dszC/B0034/dszA with dszB overlap and BBk suffix)
Primers 5 and 10 with templates L4 and L7 = Product X5 (B0034/dszA/J04500/hpaC with dszC and B0034/katG-LAA overlap)
Primers 5 and 11 with templates L4 and L8 = Product X6 (B0034/dszA/J04500/hpaC with dszC overlap and BBk suffix)
Primers 8 and 13 with templates L6 and L9 = Product X7 (J04500/hpaC/B0034/katG-LAA with BBk prefix and suffix)
Primers 9 and 13 with templates L7 and L9 = Product X8 (J04500/hpaC/B0034/katG-LAA with dszA overlap and BBk suffix)
Figure 22: Round 2 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with varying amounts of gel-extracted DNA template from the previous PCR round as indicated above the wells. Bands were extracted using Qiagen QIAquick Gel Extraction Kit. Cycling conditions for the PCR reactions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 65%deg;C for 1 min, 72%deg;C for 3 min. 30 s.)x 31, Stage 3 (72%deg;C for 10 min.). Faint bands seen at ~3000bp  indicate amplification of the desired product.
Round 3
Primers 1a and 10 with templates X1 and X5 = Product Z1 (J13002/dszB/B0034/dszC/B0034/dszA/J04500/hpaC with B0034/katG-LAA overlap and BBk prefix)
Primers 1a and 11 with templates X1 and X6 = Product Z2 (J13002/dszB/B0034/dszC/B0034/dszA/J04500/hpaC with BBk prefix and suffix)
Primers 1a and 7 with templates X1 and L5 = Product Z3 (J13002/dszB/B0034/dszC/B0034/dszA with BBk prefix and suffix)
Primers 1b and 10 with templates X2 and X5 = Product Z4 (J04500/dszB/B0034/dszC/B0034/dszA/J04500/hpaC with B0034/katG-LAA overlap and BBk prefix)
Primers 1b and 7 with templates X2 and L5 = Product Z5 (J04500/dszB/B0034/dszC/B0034/dszA with BBk prefix and suffix)
Primers 1b and 11 with templates X2 and X6 = Product Z6 (J04500/dszB/B0034/dszC/B0034/dszA/J04500/hpaC with BBk prefix and suffix)
Primers 1b and 6 with templates L2 and X3 = Product Z7(J04500/dszB/B0034/dszC/B0034/dszA with J04500/hpaC overlap and BBk prefix)
Primers 1b and 7 with templates L2 and X4 = Product Z8 (J04500/dszB/B0034/dszC/B0034/dszA with BBk prefix and suffix)
Primers 1a and 6 with templates L1 and X3 = Product Z9 (J13002/dszB/B0034/dszC/B0034/dszA with J04500/hpaC overlap and BBk prefix)
Primers 1a and 7 with templates L1 and X4 = Product Z10 (J13002/dszB/B0034/dszC/B0034/dszA with BBk prefix and suffix)
Primers 5 and 13 with templates X5 and X8 = Product Z11 (B0034/dszA/J04500/hpaC/B0034/katG-LAA with dszC overlap and BBk suffix)
Figure 23: Round 3 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with 50 ng of gel-extracted DNA template from the previous PCR round. Bands were extracted using Qiagen QIAquick Gel Extraction Kit. Cycling conditions for the PCR reactions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 65%deg;C for 1 min, 72%deg;C for 6 min.)x 31, Stage 3 (72%deg;C for 10 min.). A faint band (~6000bp) can be seen in lane 4, indicating possible amplification of the desired product. Other reactions appear to have failed, as this bands are not present elsewhere.
Though it appears that S.O.E. PCR consists of crushed dreams and lies (or that extensive optimization would have to be performed in order to amplify the correct product), extraction was carried out on the single band seen, in the hopes that a subsequent round of PCR with the terminal primers for this product would possibly amplify it to a concentration that can be used in construction in the next week to assemble and test the sulfur operon constructs before competition. Conveniently, the first round of PCR creates Gibson Assembly compatible products. Because a kit is available in the lab, this will be attempted in the following days as well. Meanwhile, a round of traditional construction appears to have finally created the parts PtetR-dszB-B0034-dszC , PlacI-dszB-B0034-dszC , and PlacI-hpaC-B0034-katG . Though sequence confirmation could not be obtained before Wiki-Freeze, we are confident that these parts are correct due to the clear size difference seen between these parts and their sub-parts. A plasmid switch was attempted the night before DNA submission had to be sent, however due to technical issues the incubator stayed at a low temperature overnight, and cultures did not grow enough to obtain the first two parts in a pSB1C3 backbone. Because of the difficulty assembling these parts, the deadline for DNA submission, and the fact that having them in an alternate backbone in the registry would allow for easier and quicker assembly of the sulfur operon by others (no plasmid switches of large parts would have to be performed, as we have found that plasmid switching the larger constructs is quite difficult.) we decided to submit these parts in an ampicillin backbone. Further attempts to switch the backbone for these parts will continue, and hopefully we will be able to submit them in the standard backbone at a later date before the competition. In addition, a reconstruction of B0034-dszA was obtained and finally, at long last, sequence confirmed. Because of the verification of this part and the excellent sequence read when compared to those obtained for the previously submitted version, we decided to also resubmit this part to the registry.
Construction attempts of the operon will continue into the following week, as only 1 additional construction is needed to assemble a testable version of the operon. Hopefully, this data will be obtained before the competition, and documented on the respective parts pages.
This is the Desulfurization Team, signing off.
Retrieved from " http://2012.igem.org/Team:Calgary/Notebook/Desulfurization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
