<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_References"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/References">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/References&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/References&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/References&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/References" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/References</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Project</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Jamboree</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">References</H1><UL><LI>2009 iGEM Calgary. <A href="https://2009.igem.org/Team:Calgary/Notebook"> https://2009.igem.org/Team:Calgary/Notebook</A></LI><LI>Asenjo JA. (1949). Bioreactor system design. New York (NY): Marcel Dekker Inc.</LI><LI>Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner BL, and Mori H. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol 2006 Feb;1-11.</LI><LI>Becker SA, Feist AM, Mo ML, Hannum G, Palsson BØ, Herrgard MJ. Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox. Nature Protocols 2 2007 Mar; 727-723.</LI><LI>Becker SA, Feist AM, Mo ML, Hannum G, Palsson BØ, Herrgard MJ. Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0. Nature Protocols 6 2011 Aug; 1290-1307.</LI><LI>Behar FH, Albrecht P. Correlations between carboxylic acids and hydrocarbons in several crude oils alteration by biodegradation. Org Geochem 1984;6:597-604. </LI><LI>Bergman L. Growth and maintenance of yeast. Methods in Molecular Biology. 2001May; 177: 9-39.</LI><LI>Biran I, Levcov K, Hengge-Aronis R, Ron EZ, Rishpon J. On-line monitoring of gene expression. Microbiology.1999 April;145:2129-2133.</LI><LI>Canadian Centre for Occupational Health and Safety: Health Effects of Sulfur Dioxid. <A href="http://www.ccohs.ca/oshanswers/chemicals/chem_profiles/sulfurdi/health_sul.html">http://www.ccohs.ca/oshanswers/chemicals/chem_profiles/sulfurdi/health_sul.html</A> (Retrieved: 09/18/2012)</LI><LI>Clemente JS, Fedorak PM. A review of the occurrences, analyses, toxicity, and biodegradation of naphthenic acids. Chemosphere 2005 Feb 6;60(5):585-600.</LI><LI>Clemente JS, Prasad NGN, MacKinnon MD, Fedorak PM. A statistical comparison of naphthenic acids characterized by gas chromatography-mass spectrometry. Chemosphere 2003. 50:1265-1274.</LI><LI>Cromie MJ, Groisman EA. Promoter and riboswitch control of the Mg2+ transporter MgtA from Salmonella enterica. J Bacteriol 2010 Jan;192(2):604-607.</LI><LI>Dana G, Kuiken T, Rejeski D &amp; Snow A (2012) Synthetic biology: Four steps to avoid a synthetic-biology disaster. Nature 483:29.</LI><LI>Daus B, Mattusch J, Paschke A, Wennrich R, and Weiss H. Kinetics of the arsenite oxidation in seepage water from a tin mill tailings pond. Talanta 2000 May 5; 51(6) 1087-95. </LI><LI>Del Rio LF, Hadwin AK, Pinto LJ, MacKinnon MD, Moore MM. Degradation of naphthenic acids by sediment micro-organisms. J Appl Microbiol 2006 Nov;101(5):1049-1061. </LI><LI>Dingwall C, Lomonossoff GP, Laskey RA. High sequence specificity of micrococcal nuclease. Nucleic Acids Res 1981 Jun 25;9(12):2659-2673. </LI><LI>Diaz E, Garcia, JL. Genetics Engineering for Removal of Sulfur and Nitrogen from Fuel Heterocycles. Handbook of Hydrocarbon and Lipid Microbiology: 2144-2157. Springer, 2010. </LI><LI>Egan SM, Schleif RF, others. A regulatory cascade in the inductionof rhaBAD. J Mol Biol 1993;234(1):87-98.</LI><LI>Environmental Protection Agency, (1995), Nitrogen oxides (NOx) Why and how they are controlled. <A href="http://www.epa.gov/ttn/catc/dir1/fnoxdoc.pdf">http://www.epa.gov/ttn/catc/dir1/fnoxdoc.pdf</A></LI><LI>Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, Broadbelt LJ, Hatzimanikatis V, Palsson BØ. A genome-scale metabolic reconstruction for <I>Escherichia coli</I> K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information. Mol Syst Biol 3 2007 Jun; 121.</LI><LI>Frank RA, Fischer K, Kavanagh R, Burnison BK, Aresenault G, Headley J, Peru KM, VanDerKraak G, Solomon K. Effect of Carboxylic Acid Content on the Acute Toxicity of Oil Sands Naphthenic Acids. Environ Sci Technol 2009 Dec 11;43(2):266–271.</LI><LI>Galán B, Díaz E, García JL. Enhancing desulfurization by engineering a flavin reductase-encoding gene cassette in recombinant biocatalysts. Environ microbiol. 2000 Dec; 2(6): 687-94 </LI><LI>German Collection of Microorganisms and Cell Cultures (DSMZ): Nocardia iowensis. <A href="https://www.dsmz.de/catalogues/details/culture/DSM-45197.html">http://www.dsmz.de/catalogues/details/culture/DSM-45197.html</A></LI><LI>Groisman EA. The Pleiotropic Two-Component Regulatory System PhoP-PhoQ. J. Bacteriol. 2001 Mar; 183 (6): 1835-42</LI><LI>Gopinath AV, Russell D. An Inexpensive Field-Portable Programmable Potentiostat. Chem. Educator.2006 July;11:23-28.</LI><LI>Harayama S, Rekik M, Wasserfallen A and Bairoch A. Evolutionary relationships between catabolic pathways for aromatics: conservation of gene order and nucleotide sequences of catechol oxidation genes of pWW0 and NAH7 plasmids. MGG Mol Gen.Genet.1987; 210(2): 241–247.</LI><LI>He A, Li T, Daniels L, Fotheringham I, Rosazza J.P.N. Nocardia sp. Carboxylic Acid Reductase: Cloning, Expression, and Characterization of a New Aldehyde Oxidoreductase Family. Applied and Environmental Microbiology 2004 Mar;70(3):1874–1881.</LI><LI>Heckman KL, Pease LR. Gene splicing and mutagenesis by PCR-driven overlap extension. Nature Protocols 2007 2;4 924-932.</LI><LI>Helmann J. Measuring metals with RNA. J Mol Cell 2007 Sept 21; 27(6) 859-860. </LI><LI>Herman DC, Fedorak PM, MacKinnon MD, Costerton JW. Biodegradation of naphthenic acids by microbial populations indigenous to oil sands tailings. Can J Microbiol 1994 Jun;40(6):467-477.</LI><LI>Hirrlinger B, Stolz A, Knackmuss HJ. Purification and properties of an amidase from Rhodococcus erythropolis MP50 which enantioselectively hydrolyzes 2-arylpropionamides. J Bacteriology 1996; 178(12) 3501-3507.</LI><LI>Holowenko FM, Mackinnon MD, Fedorak PM. Naphthenic acids and surrogate naphthenic acids in methanogenic microcosms. Water Res 2001 Aug;35(11):2595-2606. </LI><LI>Jeske M, Altenbuchner J. The Escherichia coli rhamnose promoterrhaP BAD is in Pseudomonas putida KT2440 independent of Crp--cAMPactivation. Appl Microbiol Biotechnol 2010;85(6):1923-1933.</LI><LI>Kamali N, Tavallaie M, Bambai B, Karkhane AA, Miri M. Site-directed mutagenesis enhances the activity of NADH-FMN oxidoreductase (DszD) activity of <I>Rhodococcus erythropolis</I>. Biotechnol Lett. 2010; 32: 921-927</LI><LI>Kanehisa Laboratories. KEGG pathway: Benzoic acid family-reference pathway. 2010. http://www.genome.jp/kegg/pathway/map/map07110.html </LI><LI>Katzer JR, Sivasubramanian R. Process and Catalyst Needs for Hydrodenitrogenation. Catalysis Reviews: Science and Engineering 1979; 20(2):155-208. </LI><LI>Kayser KJ, Kilbane JJ. II Gas Technology Institute. Method for metabolizing carbazole in petroleum, US Patent No. 6,943,006 B2. Sep 13, 2005. </LI><LI>Kilbane JJ. Microbial biocatalyst developments to upgrade fossil fuels. Current Opinion in Biotechnology 2006; 17(3):305-314.</LI><LI>Knudsen S, Saadbye P, Hansen LH, Collier A, Jacobsen BL, Schlundt J, Karlstrom OH. Development and testing of improved suicide functions for biological containment of bacteria. Appl Environ Microbiol 1995 Mar; 61(3):985-991.</LI><LI>Lewenza S, Falsafi RK, Winsor G, Gooderham WJ, McPhee JB, Brinkman FS, et al. Construction of a mini-Tn5-luxCDABE mutant library in <I>Pseudomonas aeruginosa</I> PAO1: a tool for identifying differentially regulated genes. Genome Res 2005 Apr;15(4):583-589. </LI><LI>Li GQ, Li SS, Zhang ML, Wang J, Zhu L, Liang FL, Liu RL, Ma T. Genetic rearrangement strategy for optimizing the dibenzothiophene biodesulfurization pathway in <I>Rhodococcus erythropolis</I>. Appl Environ Microbiol. 2008 Feb; 74(4): 971-6 </LI><LI>Li MZ, Squires CH, Monticello DJ, Childs JD. Genetic analysis of the dsz promoter and associated regulatory regions of <I>Rhodococcus erythropolis</I> IGTS8. J Bacteriol. 1996 Nov; 178(22): 6409-18</LI><LI>Lim HN, Lee Y, Hussein R. Fundamental relationship between operon organization and gene expression. Proc Natl Acad Sci U S A. 2011 Jun 28;108(26):10626-31. </LI><LI>Ma T. The Desulfurization Pathway in <I>Rhodococcus</I>. Microbiology Monographs 2010; 16: 207-230.</LI><LI>Mayaux JF, Cerbelaud E, Soubrier F, Yeh P, Blanche F, Petre D. Purification, Cloning, and Primary Structure of a New Enantiomer-Selective Amidase from a Rhodococcus Strain: Structural Evidence for a Conserved Genetic Coupling with Nitrile Hydratase. J Bacteriology Nov. 1991; 173(21): 6694-6704. </LI><LI>Morales M, Le Borgne S. Microorganisms Utilizing Nitrogen-Containing Heterocyclic Hydrocarbons. Handbook of Hydrocarbon and Lipid Microbiology: 2144-2157. Springer, 2010.</LI><LI>Nakai C, Kagamiyama H, Nozaki M. Complete nucleotide sequence of the metapyrocatechase gene on the TOL plasmid of <I>Pseudomonas putida </I> mt-2. J Biol Chem. 1983 Mar; 258(5):2923-2928.</LI><LI>Oshiro T, Ohkita R, Takikawa T, Manabe M, Lee WC, Tanokura M, Izumi Y. Improvement of 2'-hydroxybiphenyl-2-sulfinate desulfinase, an enzyme involved in the dibenzothiophene desulfurization pathway, from <I>Rhodococcus erythropolis</I> KA2-5-1 by site-directed mutagenesis. Biosci Biotechnol Biochem. 2007 Nov.; 71(11):2815-21</LI><LI>Parts Registry: The PetroBrick – Strong Constitutive Expression of ADC and AAR in pSB1C3 <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K590025 (Retrieved 8/28/2012">http://partsregistry.org/wiki/index.php?title=Part:BBa_K590025 (Retrieved 8/28/2012</A></LI><LI>Paulsen IT, Press CM, Ravel J, Kobayashi DY, Myers GSA, Mavrodi DV, DeBoy RT, Seshadri R, Ren Q, Madupu R, Dodson RJ, Durkin AS, Brinkac LM, Daughtery SC, Sullivan SA, Rosovitz MJ, Gwinn ML, Zhou L, Schneider DJ, Cartinhour SW, Nelson WC, Weidman, Watkin K, Tran K, Khouri H, Pierson EA, Pierson III LS, Thomashow LS, Loper JE. Complete genome sequence of the plant commensal <I>Pseudomonas fluorescens</I> Pf-5. Nature Biotech. July 2003. 23(7):873-878.  </LI><LI>Phillips R, Kondev J, Theriot J. Physical Biology of the Cell. 1st ed. Garland Science. 2008. </LI><LI>Ramesh A, Wakeman CA, and Winkler WC. Insights into metalloregulation by M-box riboswitch RNAs via structural analysis of manganese-bound complexes. J Mol Biol 2011 Apr 8; 407(4) 556-70. </LI><LI>Ramos-Padron E, Bordenave S, Lin S, Bhaskar IM, Dong X, Sensen CW, et al. Carbon and sulfur cycling by microbial communities in a gypsum-treated oil sands tailings pond. Environ Sci Technol 2011 Jan 15;45(2):439-446.</LI><LI>Reit K., &amp; Tramper J. (1991). Basic bioreactor design. New York (NY):Marcel Dekker Inc.</LI><LI>Regulski EE, Moy RH, Weinberg Z, Barrick JE, Yao Z, Ruzzo WL, Breaker RR. A widespread riboswitch candidate that controls bacterial genes involved in molybdenum cofactor and tungsten cofactor metabolism.Mol Microbiol. 2008 May; 68(4): 918–932</LI><LI>Reznikoff WS. Transposon Tn5. Annu Rev Genet 2008;42:269-286. </LI><LI>Rude MA, Baron TS, Brubaker S, Alibhai M, Del Cardayre SB, and Schirmer A. Terminal olefin (1-alkene) biosynthesis by a novel p450 fatty acid decarboxylase from Jeotgalicoccus species. Applied and Environmental Microbiology 2011 Mar;77(5):1718–1727.</LI><LI>Schweigert N, Zehnder AJ, Eggen RI. Chemical properties of catechols and their molecular modes of toxic action in cells, from microorganisms to mammals. Environ Microbiol 2001 Feb; 3(2):81-91</LI><LI>Scott, A., Mackinnon, M.D., Fedorak, P.M. (2005). Naphthenic acids in tailings ponds are more difficult to degrade than commercial naphthenic acids. Environ Sci Tech, 39, 8388-8394.</LI><LI>Sheridan DL, Hughes TE. A faster way to make GFP-based biosensors: two new transposons for creating multicolored libraries of fluorescent fusion proteins. BMC Biotechnol 2004 Aug 18;4:17. </LI><LI>Shu L, Chiou Y, Orville AM, Miller MA, Lipscomb JD, Que L. X-ray absorption spectroscopic studies of the Fe(II) active site of catechol 2,3-dioxygenase. Implications for the extradiol cleavage mechanism. Biochem 1995; 34:6649-6659.</LI><LI>Slavcheva E, Shone B, Turnbull A. Review of napthenic acid corrosion in oilrefining. British Corrosion Journal 1999 Feb;34(2):125-131. </LI><LI>So J. Mini-transposon Tn5gfp constructs for differential tagging of microorganisms. Biotech and Bioprocess Eng 1999;4(2):154-156. </LI><LI>Soleimani M, Bassi A, Margaritis A. Biodesulfurization of refractory organic sulfur compounds in fossil fuels. Biotechnol Adv. 2007 Nov-Dec;25(6):570-96 </LI><LI>Sukovich DJ. Hydrocarbon Biosynthesis by Bacteria: Genes and Hydrocarbon Products. PhD Dissertation, Microbiology, Immunology, and Cancer Biology Graduate Program, University of Minnesota. 2010; 1 – 208.</LI><LI>Syncrude, (2012), Tailings Management. <A href="http://www.syncrude.ca/users/folder.asp?FolderID=5913 
">http://www.syncrude.ca/users/folder.asp?FolderID=5913 
</A></LI><LI>Timms-Wilson TM, Bailey MJ. Reliable use of green fluorescent protein in fluorescent pseudomonads. J Microbiol Methods 2001 Jul 30;46(1):77-80. </LI><LI>United Nations Environment Programme: Sulfur Air Pollution. <A href="http://www.unep.org/transport/pcfv/pdf/Ethiopia-AirPollutionsulphur.pdf">http://www.unep.org/transport/pcfv/pdf/Ethiopia-AirPollutionsulphur.pdf </A></LI><LI>United States Environmental Protection Agency: Sulfur Dioxide. <A href="http://www.epa.gov/air/sulfurdioxide/">http://www.epa.gov/air/sulfurdioxide/</A></LI><LI>UW iGEM: Diesel Production Background. <A href="https://2011.igem.org/Team:Washington/Alkanes/Background">https://2011.igem.org/Team:Washington/Alkanes/Background</A></LI><LI>Vaillancourt FH, Bolin JT, Eltis LD. The ins and outs of ring-cleaving dioxygenases. Crit Rev Biochem Mol. 2006; 41:241-267. </LI><LI>Venkitasubramanian P, Daniels L, Rosazza JPN. Reduction of Carboxylic Acids by Nocardia Aldehyde Oxidoreductase Requires a Phosphopantetheinylated Enzyme. Journal of Biological Chemistry 2007 Nov 13;282(1):478-485. </LI><LI>Vitreschak AG, Rodionov DA, Mironov AA, Gelfand MS. Riboswitches: the oldest mechanism for the regulation of gene expression? Trends Genet 2004 Jan;20(1):44-50.</LI><LI>Vogel U, Jensen KF. The RNA chain elongation rate in <I>Escherichia coli</I> depends on the growth rate. J Bacteriol. 1994 May;176(10):2807-13. </LI><LI>Waters LS, Sandoval M, and Storz G. The<I> Escherichia coli</I> MntR miniregulon includes genes encoding a small protein and an efflux pump required for manganese homeostasis. J Bacteriol 2011 Nov; 193(21) 5887-97.</LI><LI>Wright JK, Overath P. Purification of the lactose:H+ carrier of <I>Escherichia coli</I> and characterization of galactoside binding and transport. Eur J Biochem.1984 Feb 1;138(3):497-508.</LI><LI>Xiong AS, Peng RH, Cheng ZM, Li Y, Liu JG, Zhuang J, Gao F, Xu F, Qiao YS, Zhang Z, Chen JM, Yao QH. Concurrent mutations in six amino acids in beta-glucuronidase improve its thermostability. Protein Eng Des Sel. 2007 Jul;20(7):319-25. Epub 2007 Jun 8. </LI><LI>Xu P, Yu P, Li FP, Cai XF, Ma CQ. Microbial degradation of sulfur, nitrogen and oxygen heterocycles. Trends in Microbiology 2006; 14(9):398-405.</LI><LI>Yoshimura F, Nikaido H. Permeability of Pseudomonas aeruginosa outer membrane to hydrophilic solutes. J Bacteriol. 1982 Nov;152(2):636-42. </LI><LI>Young R, Bremer H. Polypeptide-chain-elongation rate in <I>Escherichia coli</I> B/r as a function of growth rate. Biochem J. 1976 Nov 15;160(2):185-94. </LI><LI>Zeuthen P, Knudsen KG, Whitehurst DD.Organic nitrogen compounds in gas oil blends, their hydrotreated products and the importance to hydrotreatment. Catalysis Today Feb 2001; 65(2-4):307-314.</LI><LI>Zhang X, Wiseman S, Yu H, Liu H, Giesy JP, Hecker M. Assessing the toxicity of naphthenic acids using a microbial genome wide live cell reporter array system. Environ Sci Technol 2011 Mar 1;45(5):1984-1991.</LI><LI>Zhang Y, Nelson M, Nietfeldt JW, Burbank DE, Van Etten JL.Characterization of Chlorella virus PBCV-1 CviAII restriction andmodification system. Nucleic Acids Res 1992;20(20):5351-5356.</LI><LI>Zhu G, Pang K, Parkin G. New Modes for Coordination of Aromatic Heterocyclic Nitrogen Compounds to Molybdenum:  Catalytic Hydrogenation of Quinoline, Isoquinoline, and Quinoxaline by Mo(PMe<SUB>3</SUB>)<SUB>4</SUB>H<SUB>4</SUB>. Journal of the American Chemical Society 2008; 130(5):1564-1565. </LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Project/References">http://2012.igem.org/Team:Calgary/Project/References</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/References" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/References" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/References&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/References&amp;oldid=299311" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>