<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Denitrogenation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Denitrogenation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Denitrogenation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Denitrogenation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Denitrogenation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Denitrogenation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Denitrogenation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Notebook</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Killswitch">Killswitch</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/PromoterScreen">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Electrochem">Electrochemistry</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Biosensor">Biosensor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/CatecholDegradation">Catechol</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/FluxAnalysis">Flux Analysis</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocol Manual</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Denitrogenation Journal</H1><H2>Week 1 (May 1-4)</H2><P>The following section covers wetlab aspect of our overall project focusing on microbial conversion of naphthenic acids into economically-valuable hydrocarbons. The approach taken in this endeavour will be from four strategic starting points - ring cleavage, decarboxylation, denitrogenation, and desulfurization. Overall, the 'hydrocarbons' aspect of the project is a critical one to our overall design and construction of a bioreactor capable of not only detecting but also converting naphthenic acids and other toxins to useful or at least innocuous compounds.   </P><H2>Week 2-3 (May 7-18)</H2><P>In the first two weeks of iGEM our group has focused on reviewing literature regarding the bioremediation of nitrogen groups attached to naphthenic acids. The most prevalent N heterocycle is carbazole, representing 75% of total nitrogen by mass. The upper pathway of carbazole biodegradation is catalyzed by the enzymes coded for by the <I>car</I> operon, <I>CarA</I> (<I>CarAaAcAd</I>), <I>CarB</I> (<I>CarBaBb</I>), and <I>CarC</I>. These enzymes convert carbazole to anthralinic acid. The lower pathway is catalyzed by the enzymes of the <I>ant</I> operon, <I>antA, B, and C</I>, yielding catechol while releasing CO2 and NH3. The <I>car</I> and <I>ant</I> operons are both regulated by the P<SUP>ant</SUP> promoter which is induced by the protein, antR. <I>carAa</I> also has its own promoter which is not induced by <I>antR</I> (Diaz &amp; Garcia, 2010). We have also investigated an alternative pathway using <I>CarA</I> combined with an amidase (<I>amdA</I>) that selectively cleaves NH2 from an intermediate of the car pathway (Xu et al, 2006). This could bypass much of the car/ant pathway and is possibly more efficient.</P><P>We have decided to use <I>Pseudomonas resinovorans</I> and <I>Rhodococcus erythropolis</I> to derive these genes from. <I>carABC</I> and <I>antABC</I> from<I> P. resinovorans</I> has been shown to have a wide range of nitrogen containing substrate specificity. <I>R. erythropolis</I> contains the <I>amdA</I> gene that we wish to use, and some evidence suggests that it may also be able to degrade sulfur rings through its <I>carABC</I> pathway.</P><H2>Week 4 (May 22-25)</H2><P>This week we reviewed the primers listed in the database and also designed some new ones. Primers for <I>carAa</I>,<I> carAc</I>, and <I>carAd</I> were designed individually, while primers for <I>CarBaBbC</I> and <I>antABC</I> were designed to encompass multiple genes in a sequence. These primers were designed to be used on <I>Pseudomonas putida</I> which we decided to use as our gene source since it was available to us as opposed to ordering <I>Pseudomonas resinovorans</I>. We also designed a primer for the<I> AmdA</I> gene from <I>Rhodococcus erthyroplois</I>. In addition to ordering these primers we also ordered the nitrogen containing compounds that we will need to test these enzymes on. We decided on using carbazole to make sure the enzymes can perform their natural function as well as pyrrolidine to test them on a similar ring structure. We also ordered cyclohexamine in order to independently test the function of the alternative <I>AmdA</I> pathway. Finally, we decided to eventually order 4-Piperidine butyric acid hydrochloride to test how the enzymes will work on nitrogen containing naphthenic acids. However, we decided since it is a very expensive compound we would wait to make sure the enzyme's work on more simple compounds before ordering it.</P><P>We also started our work in the wet lab by plating colonies and making an overnight culture of -80 freezer stock <I>Psuedomonas putida</I> on Thursday. We also resuspended the primers for <I>carAa</I>, <I>carAc</I>, <I>carAd</I>,<I> antA</I>, <I>antB</I>, and <I>ant C</I> that were already in the database. We were able to use the colonies that grew on the streak plates to start a colony PCR to attempt to isolate each of these genes on Friday.</P><H2>Week 5 (May 28 - June 1)</H2><P>On Monday and Tuesday we used the database primers to attempt to isolate <I>carAa</I>, <I>carAc</I>, <I>carAd</I>, <I>antA</I>, <I>antB</I>, and <I>antC</I>  from the <I>Psuedomonas putida</I>  we plated last week. <I>carAd</I>, <I>antB</I>, and <I>antC</I>  were all put in the gradient PCR machine to account for the wide range of their primers’ melting points, while the others were done via regular PCR. Unfortunately, no positive results were obtained from these reactions.  The PCR on <I>carAc</I>  and <I>antA</I>  was repeated on Wednesday, still not giving positive results. Finally, we attempted to use a salt concentration gradient on the PCR reaction for <I>carAc</I>  and <I>antA</I>, using concentrations that ranged from 1.0 microlitres/tube to 2.0 microlitres/tube in increments of 0.2 microlitres/tube. This helped as <I>carAc</I>  showed bands in samples that had concentrations of 1.2 and 1.4 microlitres/tube. <I> antA</I>  also showed bands, however they were not the correct size, indicating contamination and/or non-specific annealing of the primer.  The positive control also showed bands of the correct size. Earlier in the week we also made overnight cultures of 3 environmental strains (28, 29, 30) of <I>Psuedomonas putida</I> from -80 glycerol stock. On Friday we performed a genomic prep on these cultures, and plan on attempting to amplify genes from the isolated DNA next week. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:434px;"><DIV class="thumbcaption">Gel of PCR using database primers for <I>carAa</I> and <I>carAc</I> on <I>Pseudomonas putida</I></DIV></DIV></DIV></DIV><H2>Week 6 (June 4 - June 8)</H2><P> We started off this week by determining the DNA concentration of our genomic prep samples from last week using the nanodrop. DNA concentration for all three<I> P. putida</I> strains was at least 1000 ng/microlitre, well above what was needed for PCR. 1/2 and 1/3 dilutions were prepared for all three strains so as not to have an excess of template DNA in PCR reactions. PCR was performed on all 3 strains using primers for <I>carAc</I>, <I>carAd</I>, <I>antA</I>, <I> antB</I>, and <I>antC</I> using 6 replicates per gene. The only successful amplification appeared to be <I>antB</I> and <I>carAc</I>, both from strain 28 (with weaker bands in strain 29). We then performed another PCR, just on those two genes with an increased amount of Taq polymerase to hopefully get enough amplified DNA to move forward with. This resulted in strong bands for both at the expected size. We then performed PCR purification using the Qiaquick kit and obtained samples containing 33.5 ng/microlitre of <I>antB</I> DNA and 129 ng/microlitre of <I>carAc </I>DNA. These concentrations were both sufficient to begin a restriction digest and ligation of these parts into vector <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A>. Next week we hope to verify the results of the restriction digest, continue to amplify <I>carAc</I> and <I>antB</I> from strain 28, and hopefully submit a biobrick for sequencing.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:389px;"><DIV class="thumbcaption">Gel of PCR ran on genomic prep of <I>Pseudomonas putida</I> environmental strain 28 using primers for <I>carAc</I> and <I>antB</I>.</DIV></DIV></DIV></DIV><H2>Week 7 (June 11 - June 15)</H2><P>The purified PCR products for <I>carAc</I> and <I>antB</I>  were digested with restriction enzymes EcoRI+SpeI, EcoRI+PstI, and XbaI+PstI. However, only <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> plasmids that had been digested  with EcoRI+SpeI and EcoRI+PstI were available to attempt ligation. Gel results were inconclusive on the restriction digest product as the plasmid size appeared much larger than expected. However, transformation was still attempted on the ligation products for both genes (both using EcoRI+SpeI ligation into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> plasmid). The transformation products were plated onto chloroamphenicol (chlor) plates to select for colonies that had the chloro resistance gene on the <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> plasmid. Also this week another round of PCR was performed for the <I> car</I>  and <I> ant</I>  genes, however all, but <I>carAc</I>  showed bands in the negative control lane, possibly indicating contamination or the formation of primer dimers. The <I>carAc</I>  bands were fairly weak and a PCR purification resulted in very low DNA concentration, insufficient to move onto restriction digest. Next week's plan hinges heavily on the result of the transformation from this Friday. </P><H2>Week 8 (June 18 - June 22)</H2><P>The results of our transformation on <I>carAc</I> and <I>antB</I> from last week showed mostly red colonies indicating that the ligation was unsuccessful and that the PSB1C3 plasmid simply closed on itself with no insert. However, there were three colonies for <I>carAc</I> that were white (cut with both X+P and E+P). Colony PCR was performed on these three colonies using biobrick primer sets to verify that the part had been inserted in these colonies, but unfortunately all PCR results were negative indicating an unsuccessful ligation.</P><P>We also performed a miniprep on <I>Pseudomonas putida</I> strain #28 this week using a home-made protocol rather than the Qiagen kit. Miniprep A had a DNA concentration of 1539.8 ng/μL and Miniprep B had DNA concentration of 1001.2 ng/μL possibly indicating that there may have been a lot of genomic DNA contamination. However miniprep A product was still used as a PCR template for a reaction attempting to isolate <I>carAc</I>, <I>carAd</I>, <I>antA</I>, <I>antB</I>, and <I>antC</I>. Special conditions for this PCR included replacing 60 μL of water with 60 μL of betaine heated to 37 degrees Celsius and running a Mg gradient ranging from 0.5 μL - 2.5 μL per tube for each gene. Of these only <I>antB</I> had bands of the right size in 3 of the lanes (Mg concentrations of 1.5, 2, and 2.5 μL). However, all of the <I>antB</I> bands (including the negative control) contained a contamination band at around 100 bases, probably due to the formation of primer dimers. This forced us to do a gel extraction rather than a PCR purification. This gel extraction did not work as the nanodrop results indicated high amounts of agarose contamination. Another PCR was performed using the same conditions, but using miniprep B instead as there was no more miniprep A product remaining. Unfortunately there were no bands for <I>antB</I>, only the 100 base pair contamination bands. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:482px;"><DIV class="thumbcaption">Gel of PCR ran using miniprep product A of <I>Pseudomonas putida</I> as template DNA in an attempt to isolate <I>antA</I> and <I>antC</I> genes.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:468px;"><DIV class="thumbcaption">Gel of PCR ran using miniprep product A of <I>Pseudomonas putida</I> as template in an attempt to isolate <I>carAc</I> and <I>carAd</I> genes.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:419px;"><DIV class="thumbcaption">Gel of PCR ran using miniprep product A of <I>Pseudomonas putida</I> showing bands 3-5 of the correct size corresponding to antB genes and colony PCR ran using biobrick primers attempting to determine if <I>carAc</I> genes were successfully transformed into PSB1C3 plasmid.</DIV></DIV></DIV></DIV><H2>Week 9 (June 25 - June 29)</H2><P>This week we modified our PCR protocol to adjust for the homemade Taq polymerase we were using. Template DNA was boiled for 10 minutes on its own to replace the initialization step. This is because this polymerase could not tolerate the 95 degree heat during the regular initialization step. Another modification we made was to boil our primers for 5 minutes prior to adding them to the master mix. This was done to discourage the formation of primer-dimers. Finally we also attempted a touch-down PCR which started at an annealing temperature greater than the Tm of our primers and gradually worked its way down to below the Tm of the primers. This was done to increase the specificity of our reaction and also eliminate the formation of primer-dimers. Unfortunately these techniques still saw the formation of dimers, preventing us from performing a PCR purification. Again, the gel purification procedure did not lead to successful purification of our target gene. Due to the problems we have had so far our goal for next week is to perform an assay experiment to try and determine which of the <I>Pseudomonas putida</I> strains we have access to can actually degrade carbazole and therefore contain the pCAR1 plasmid. </P><H2>Week 10 (July 2-July 6)</H2><P>The goal for this week was to grow different strains of<I> Pseudomonas</I> in media containing B-N media + glucose with only carbazole as a source of nitrogen. In theory only those strains which contained the pCAR1 plasmid would be able to use carbazole as a nitrogen source and thrive in this media. The strains we tested from glycerol stock included <I>P. putida</I> LD1, <I>P.putida</I> environmental strains 28-30, and <I>P. fluorescens</I> PF5. Each of these strains had two cultures grown; one with carbazole and one without it. <I>E. coli</I> grown with carbazole was used as a negative control as it does not contain the pCAR plasmid. At timepoints of 22, 46, and 117 hours of growth samples were taken to do an OD600 reading to measure bacterial growth and had an ammonia assay performed on it. The detection of ammonia acted as a proxy for detecting carbazole degradation as it is an end-product of the pathway. With this in mind, an additional control strain of LD1 was grown in media containing glucose and ammonia. This is to detect the possibility that <I>Pseudomonas</I> can use ammonia as a source of nitrogen after it has degraded carbazole which would be problematic as far as using ammonia levels to determine carbazole degradation. So far we have been having difficulties reading the results of the ammonia assay with a plate reader and may resort to testing our samples using a different assay protocol next week. The OD600 readings were initially very high at 22 hours and then dropped at 46 hours.  </P><H2>Week 11 (July  9-July 13)</H2><P>This week we took our last readings from the experiment that we started last week. The OD600 readings taken at 117 hours showed minimal growth over the weekend for all strains except for environmental strain #29 (with carbazole) which increased from 0.086 to 0.313. No other strain showed an increase of 0.100 or greater. This shows that only strain #29 was able to thrive in the carbazole media, possibly indicating that it contains the pCAR plasmid enabling it to use carbazole as a nitrogen source. To verify this we used a new ammonia assay protocol to measure the production of ammonia in the different cultures and theoretically how much carbazole was degraded. Unlike the previous assay we were able to produce a standard curve with a consistent slope based on samples with known amounts of ammonia. However, the only samples that showed any elevation of ammonia levels above the blank were the LD1 samples that were cultured with ammonia chloride to control for the possibility of pseudomonas using ammonia as a nitrogen source. The ammonia levels in this sample were relatively constant which would indicate that the LD1 was not using ammonia as a nitrogen source. Unfortunately the fact that none of the test samples had elevated ammonia levels did not give us any evidence that these strains contain the pCAR plasmid. </P><H2>Week 12 (July  16 -July 20)</H2><P>We received a new strain of <I>Pseudomonas</I> LD2, this week that had previously been used to study the pCAR1 plasmid. It was grown up in PCA media and had its colonies used for colony PCR. We also received our new primers this week, further increasing the chances of successful PCR. Using the new strain and the new primers <I>carAa</I>, <I>carAc</I>, and <I>carAd</I> all showed at least some bands of close to the correct size. After PCR purification we obtained DNA concentrations of 21.3 ng/μL for <I>carAa</I>, 41.9 ng/ μL for <I>carAc</I>, and 32.8 ng/μL for<I> carAd</I>. These products were then restriction digested using EcoRI + PstI sites. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Gel of colony PCR from <I>Pseudomonas sp. LD2</I> for <I>carAa</I> and <I>carAc</I> genes. <I>carAa</I> expected size is 1155 bp and <I>carAc</I> is 323 bp.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Gel of colony PCR from <I>Pseudomonas sp. LD2</I> for <I>carAd</I> gene. <I>carAd</I> expected size is 989 bp.</DIV></DIV></DIV></DIV><H2>Week 13 (July 23 - July 27)</H2><P>The restriction digested <I>car</I> genes were ligated into vector <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> and transformed into competent <I>E. coli</I> cells. White colonies grew for<I> carAa</I> and <I>carAc</I> suggesting that ligation and transformation were successful, however no colonies grew for <I>carAd</I> suggesting that the transformation did not work. Colony PCR was performed to confirm the presence of these genes within the vectors, however only <I>carAc</I> was positive. The colonies containing the <I>carAc</I> plasmid were then mini prepped and restriction digested to isolate the plasmid and send for sequencing. Further attempts at transforming<I> carAa</I> and <I>carAd</I> were not successful. Another goal was to perform colony PCR on the <I>P. putida</I> to isolate the<I> antABC</I> genes using both the primers for the entire operon and those for the individual genes. No attempt showed any bands, although salt and temperature gradients were used as well as different types of polymerase. LD2 cultures were grown up to be plasmid prepped and genomic prepped in order to have a higher concentration of template DNA in the PCR master mix and more PCR will be attempted on these genes next week. Mutagenesis primers were also designed this week to remove the illegal restriction cut sites in the<I> carAd</I> and <I>antB</I> genes (NotI and 2xEcoRI respectively). Only 1 mutagenesis primer was designed for <I>antB</I> as one of its EcoRI sites was very close to the beginning of the gene and could be removed using a new primer. During this process it was also discovered that the original ant primers were designed for the wrong genes and therefore do not match the intended sequence. Although the new <I>antABC</I> primers were designed using the correct gene the PCR may not be working due to the length of the operon so the design of new individual primers may be necessary. Finally, we also aimed to grow up the freeze dried culture of <I>Rhodococcus erythropolis</I> (source of the<I> AmdA</I> gene) that we received from DSMZ. The reactivation media was produced and autoclaved to begin the process next week.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Gel of restriction digest of car genes. A refers to <I>carAa</I>, C to <I>carAc</I>, and D to <I>carAd</I>. Ladder shows that after purification and digestion DNA is still the correct size.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Gel of confirmation PCR done on competent Top 10 <I>E. coli</I> cells in which a PSB1C3 vector containing the <I>carAc</I> gene had been inserted. Red X marks the 500 bp mark on the weak ladder showing the product to be the expected size (323 bp gene + 2*100 bp biobrick sites on each end).</DIV></DIV></DIV></DIV><H2>Week 14 (July 30 - August 3)</H2><P>We received the sequencing results this week for <I>carAc</I> which came back as a 100% match to the expected nucleotide sequence. We also successfully ligated both<I> carAa</I> and<I> carAd</I> into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> and transformed into top 10 <I>E. coli</I> cells. Colony PCR verified the presence of the biobrick plasmid in these colonies. These were both miniprepped, digested and sent for sequencing by the end of the week. Using a salt gradient, addition of DMSO, and the KAPA buffer specifically designed for GC-rich amplicons we were able to amplify the ant operon from a genomic prep of LD2. However, all attempts at ligating and transforming this sequence were met with the unfortunate presence of red colonies (indicating failed insertion into the plasmid). Next week we will recut a <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> vector with Xba and Pst to attempt to mitigate this problem. We also reactivated the <I>R. erythropolis</I> following the DSMZ protocol for reviving freeze-dried cultures that can be found <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/dmszfreezedry">here</A> using DSMZ media #1 and grew it on plates made from DSMZ media #220 at 28 degrees. Colonies grew very quickly and were used for colony PCR and to make overnight cultures for downstream applications. The colony PCR using the amidase primers was unsuccessful, so a genomic prep was performed on the overnight culture. Unfortunately we lacked some of the required equipment and the genomic prep did not work, but with a few procedural modifications and the right equipment we are confident it will work next week. A glycerol stock of the<I> R. erythropoplis</I> was also made from another overnight culture. We also started construction on the <I>carAc </I>part this week by inserting a ribosome binding site (B0034) in front of the gene. Finally, since the<I> ant</I> operon was successfully amplified by PCR we designed mutagenesis primers for the 2nd EcoRI site in the operon rather than making new individual primers. We also designed a mutagenesis primer to remove the SpeI site in <I>AmdA</I>. Now that we have conclusively proven that the LD2 strain contains the pCAR1 plasmid we aim to demonstrate its ability to degrade carbazole and other nitrogen containing heterocycles using an assay experiment similar to the one performed in Week 10.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>carAa</I> and <I>carAd</I> amplified from genomic prep of <I>Pseudomonas sp. LD2</I>. Amplicons are larger than expected, potentially the gel ran diagonally. The 6th lane for each gene was a negative control.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>ant</I> operon amplified from genomic prep of <I>Pseudomonas sp. LD2</I> at the expected size of 3000 bp.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Lanes 2,4,5, and 11 contain <I>carAd</I> genes amplified from colony PCR (using biobrick primers) of the PCR product that had been ligated into PSB1C3 and transformed into competent top 10 <I>E.Coli</I>.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Lane 9 contains <I>carAa</I> genes amplified from colony PCR (using biobrick primers) of the PCR product that had been ligated into PSB1C3 and transformed into competent top 10 <I>E.Coli</I>. Although larger than expected, the positive control RFP generator in lane 10 is also larger than its expected size (~1200 bp including biobrick bookends) indicating that the gel may have a slant in it.</DIV></DIV></DIV></DIV><H2>Week 15 (August 7 - August 10)</H2><P>This week we received the sequencing results for <I>carAa</I> and <I>carAd</I>. <I>carAd</I> was a match, however <I>carAa</I> did not match at all and had misplaced biobrick restriction enzyme sites. Due to this we decided to restart the biobricking process for <I>carAa</I> all the way back to PCR from the plasmid. By the end of the week we had progressed to the point of transforming it into competent <I>E. coli</I> cells, however a thick lawn of bacteria had grown overnight making it impossible to pick colonies. A streak plate was made from this plate to attempt to get less dense growth. Since the sequencing came back positive for <I>carAd</I> the next step was to perform site directed mutagenesis on its NotI restriction enzyme site using the primers that were designed previously (for site-directed mutagenesis optimization, please refer to Desulfurization Journal Week 14). The PCR was successful, showing very bright bands at 3000 bp, and the PCR product was digested with DpnI to remove parental DNA. Transformation of this product will indicate whether the mutagenesis was successful. We were also able to successfully perform a genomic prep of <I>Rhodococcus erythropolis</I> and amplify the <I>AmdA</I> gene this week. It was ligated into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> and transformed into competent <I>E. coli</I> by Friday, and a confirmation colony PCR was run on Friday evening to verify that the gene was inserted. These results will be available by Monday. After many attempts the <I>ant</I> operon was finally ligated and transformed as well this week. Confirmation PCR showed bands of about 3000 bp which is consistent with the size of the operon. By Friday the successful colonies had been grown up and miniprepped, ready to send for sequencing early next week. The <I>carAc</I>/B0034 construct was also successfully transformed this week and verified by running a PCR using the B0034 forward primer. It was also miniprepped and is ready to send for sequencing next week. </P><P>In addition to this work constructing biobricks we also started an experiment to demonstrate the function of the pCAR plasmid in LD2. Strains of LD2 were grown up in LB and then washed and placed in B-N media + glucose and various experimental additions. These additions included carbazole, 4-PBAH (a nitrogen containing naphthenic acid), ammonia chloride, and a negative control with no nitrogen source. Strains of <I>E. coli</I> were also grown under the same conditions to compare growth and ammonia production of LD2 to a strain that does not contain the pCAR plasmid. These results should act as a preview of what we expect our final biobrick containing <I>E. col</I>i species to produce as well as allowing us to establish LD2 as a positive control for this experiment. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>AmdA</I> amplified from the genomic prep of <I>R. Erythropolis</I> shows bands just greater than 1500 base pairs in lanes 2,4-6,9, and 12-13. Lane 14 was a negative control. The expected size of <I>AmdA</I> is 1566 base pairs.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The two labelled lanes show amplification at 3000 base pairs from a confirmation colony PCR using the biobrick primers. The expected size of the <I>ant</I> operon with the biobrick bookends is about 3100 base pairs.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Strong amplification at just above 3000 base pairs from the <I>carAd</I> PCR using mutagenic primers. Since PSB1C3 is 2070 base pairs and <I>carAd</I> is 989 base pairs the amplification should run at about 3059 base pairs. 3 different amounts of template DNA were tested in lanes 2-7 as indicated and a positive control showed weaker amplification in lanes 8-9 at about 4000 base pairs.</DIV></DIV></DIV></DIV><H2>Week 16 (August 13 - August 17)</H2><P>The sequencing for the <I>carAc</I>/<A href="http://partsregistry.org/Part:BBa_B0034">B0034</A> construct came back with only <I>carAc</I> and no evidence of a <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A> site preceding it. The confirmation PCR that was run last week using the <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A> forward primer seemed to indicate that it was inserted, however since <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A> is only 12 base pairs the primer may not have been specific enough. The construction on this part was restarted this week. <I>carAa</I> colony PCR on last week's transformation showed two positive colonies, however upon culturing, miniprepping, and digesting these colonies the plasmid appeared to be too large (3000 base pairs as opposed to the expected 2000). This may be due to a problem of a mislabeled plasmid tube that has been experienced by other team members, causing the gene to be inserted into <A href="http://partsregistry.org/Part:pSB1AC3">pSB1AC3</A> rather than <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A>. To test this, a streak plate was made from these colonies and grown on an ampicillin resistant plate. Mutagenized <I>carAd</I> was successfully transformed into Top 10 <I>E. coli</I> cells, cultured, and miniprepped. The miniprep product was digested with NotI to test whether the NotI site was successfully removed. These results were good, as the plasmid ran at 2000 base pairs and the insert at just under 1000 base pairs in the mutagenized <I>carAd</I> whereas the non-mutagenized <I>carAd</I> had its insert cut into 2 bands. This will be sent for sequencing next week to confirm the mutation. Unfortunately the sequencing results came back negative for the <I>ant</I> operon meaning we will need to restart genomic PCR on it next week. Multiple transformations and confirmation PCRs were run on <I>AmdA</I> this week, however none displayed a band size of 1700-1800 base pairs as expected. We have continued to try different digested plasmids as other team members also had difficulties this week with ligation into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A>, possibly indicating a problem with our Antarctic Phosphatase stock.</P><P>The other news from this week was the final results from our carbazole degradation experiment attempting to demonstrate LD2's ability to degrade carbazole and hopefully other nitrogen containing compounds. Unfortunately, these results were very difficult to make sense of as the LD2 that were cultured with carbazole grew, but did not show elevated ammonia levels which would be indicative of carbazole degradation. One of our controls was LD2 cultured with ammonium chloride to test whether LD2 was capable of using ammonia as a nitrogen source. This would pose a problem for us as we were planning to use an increase in ammonia levels as a proxy for carbazole degradation rather than measuring carbazole degradation directly using GC-MS. However, the LD2 showed significantly less ammonia after 44 hours when compared to 26 hours indicating that it may be degraded, either by the LD2 or by an unaccounted for abiotic process. This has lead us to decide to use GC-MS and directly measure carbazole levels in our sample the next time we do this experiment.</P><H2>Week 17 (August 20 - August 24)</H2><P>The <I>carAa</I> was re-streaked onto ampicillin plates and grew, probably showing that it was ligated into <A href="http://partsregistry.org/Part:pSB1AC3">pSB1AC3</A>, not <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A>, which explained the irregularity of our confirmation digest last week. It was sent for sequencing, but in case it was incorrect we also started a new <I>carAa</I> pipeline by PCRing it from the <I>Pseudomonas</I> genome again and redigesting and ligating. It was transformed into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> over the weekend. <I>carAc</I> did not see a lot of progress this week as we spent much of the time attempting to construct a RBS (B0034) in front of it, but were unsuccessful. <I>carAd</I> was also subjected to this construction with B0034 and we were able to send it for sequencing by the end of the week. Several new constructs were also started for <I>carAd</I> in case the sequencing is negative. The <I>ant</I> operon was successfully transformed and confirmed by colony PCR this week, after which it was grown up and sent for sequencing. We also began to attempt mutagenesis on one of the EcoRI sites in the operon, but had no amplification of the plasmid. We also re-digested some PCR purified product to start another ligation in case sequencing failed. After many attempts we were also able to finally successfully transform an <I>AmdA</I>/PSB1C3 plasmid into Top 10 cells this week. After a miniprep they showed bands of about 1800 base pairs on the confirmation digest gel, so they were also sent for sequencing this week and had mutagenesis started on the illegal SpeI site. Unfortunately, the sequencing sent early in the week came back with ambiguous results. The forward reaction for<I> antABC</I> was a match, but its reverse reaction matched to <I>carAa</I>. Both <I>carAa</I> reactions matched nothing. Since these two genes are over 20 000 base pairs apart on the template plasmid it is virtually impossible that they were actually combined into one plasmid (PCR extension time would not have been long enough and even if it was the product would have run much above the 3000 base pair mark). This is possibly a mistake on Eurofins end, so we are not 100% sure if we have 1 good gene, both of them, or neither. Also the sequence for mutagenized <I>carAd</I> showed only about the first 100 base pairs, which were correct however not long enough meaning it was a poor reaction and inconclusive. Finally, we also re-started our <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/carbazole">carbazole degradation assay</A> using the GC-MS to quantitatively measure the loss of carbazole at biological sciences. This process will take 2 weeks to demonstrate LD2's ability to degrade carbazole. We are forcing it to use carbazole as a carbon and a nitrogen source in one sample and as only a nitrogen source in another sample (this sample has had glucose added to it).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:423px;"><DIV class="thumbcaption">Amplification is seen in lane 7 of this PCR of top 10 colonies into which AmdA/PSB1C3 had been transformed. This indicates that <I>AmdA</I> has been successfully inserted into a biobrick vector.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>carAa</I> amplified from pCAR1.2 plasmid of <I>Pseudomonas putida</I>.</DIV></DIV></DIV></DIV><H2>Week 18 (August 27 - August 31)</H2><P>The new stock of <I>carAa</I> that we had PCRed from LD2 was sent for sequencing this week and came back as a match, so we have begun constructing a RBS site and a TetR promoter (<A href="http://partsregistry.org/Part:BBa_J13002">J13002</A>) in front of it. At this point all the <I>car</I> genes have been successfully amplified, inserted into a biobrick plasmid, and sequence verified. We also continued to attempt to construct <I>carAc</I> with an RBS site (<A href="http://partsregistry.org/Part:BBa_B0034">B0034</A>) in front of it, but had no success. On a better note, we confirmed this week that<I> carAd</I> was mutated properly and we were also able to insert an RBS site in front of it. The next step for this gene will be to attempt a plasmid switch as it is currently in <A href="http://partsregistry.org/Part:pSB1A3">pSB1A3</A> and needs to be in <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A> for submission to the parts registry. After sending a new batch of the ant operon for sequencing we got the same confusing results as we did last week. We therefore hypothesized that our primers were annealing to a homologous region that is similar to the<I> ant</I> genes and very close to <I>carAa</I>. This means that we need to restart PCR on this gene as all of our current product is incorrect. The sequencing results for <I>AmdA</I> that came back this week were also somewhat confusing as they did not quite match the sequence we expected. However, they were a 100% match for the <I>AmdA</I> gene in a different strain of <I>Rhodococcus erythropolis</I>. This means that DSMZ probably sent us the wrong strain of bacteria and that we were very lucky that the primers we designed based off of the expected template actually worked to amplify a slightly different gene in another strain. The good news is that this new sequence contains no illegal cut sites, which explains why mutagenesis was not working (there was nothing to mutate) and allows us to immediately begin construction of <A href="http://partsregistry.org/Part:BBa_J13002">J13002</A> in front of <I>AmdA</I> this week. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:429px;"><DIV class="thumbcaption">Colony PCR amplification from top 10 <I>E. coli</I> cells with a PSB1C3/<I>carAa</I> construct transformed into them. Lanes 1-3, 5, and 7 were sent for sequencing with lane 5 coming back positive.</DIV></DIV></DIV></DIV><H2>Week 19 (September 3- September 7)</H2><P><I>carAd</I>/<A href="http://partsregistry.org/Part:BBa_B0034">B0034</A> was successfully plasmid switched into <A href="http://partsregistry.org/Part:pSB1C3">pSB1C3</A>, making it ready for submission. Construction of <A href="http://partsregistry.org/Part:BBa_J13002">J13002</A> and <I>carAa</I> may have been successful as the digest of the miniprep looked promising. It will be sent for sequencing next week. Unfortunately there was no progress made on the constructions of <I>AmdA</I> with <A href="http://partsregistry.org/Part:BBa_J13002">J13002</A> or <I>carAc</I> with <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A>. These will be redigested and religated for next week. We could not amplify the <I>ant</I> operon despite trying many variations of PCR using high fidelity KAPA polymerase. This is probably due to the multiple, large regions of homology on the operon that the reverse primer binds to making amplification difficult. Because the function of this gene can theoretically be replaced by <I>AmdA</I>, we may scale back efforts dedicated to it, as the chances of successful amplification appear slim. </P><H2>Weeks 20-22 (September 10 - September 30)</H2><P>Constructs that we continued to work on this week included <I>AmdA</I> with <A href="http://partsregistry.org/Part:BBa_J13002">J13002</A>, <I>carAa</I> with <A href="http://partsregistry.org/Part:BBa_J13002">J13002</A>, and <I>carAc</I> with <A href="http://partsregistry.org/Part:BBa_B0034">B0034</A>. Some sequencing was sent, but no positive results were obtained. A new approach to biobrick the <I>ant</I> operon was implemented this week. A new forward primer was designed that would anneal about 1500 base pairs upstream of the <I>ant</I> operon, avoiding the region of homology that is repeated at various points on the plasmid. Once this region is successfully amplified we could use this amplified DNA as a template for another PCR reaction which would use the normal <I>ant</I> primers. Since the template would no longer include regions of homology these primers should be free to work properly and amplify the <I>ant</I> operon. The PCR using the new primers was started this week, and although amplification at 4500 base pairs was observed as expected, there was also a lot of non-specific amplification. This means that to purify the amplified DNA that we want and avoid the other products a dreaded gel extraction had to be performed which, as usual, did not work. At this point it was decided that further attempts to amplify the <I>ant</I> operon would be futile and our efforts would be better served working on constructing the parts we do have. Unfortunately no progress was made in these weeks on construction. We have also obtained GC-MS readings from the carbazole degradation experiment that showed that <I>Pseudomonas</I> LD2 can indeed degrade carbazole, however only under low glucose conditions.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Carbazole loss over 13 days when cultured with <I>Pseudomonas sp. LD2</I> with and without the addition of glucose.</DIV></DIV></DIV></DIV><H2>Weeks 23-26 (October 1-October 24)</H2><P>Most of this time was spent attempting the same construction of <I>AmdA</I> with a TetR promoter as described above. After <A href="http://partsregistry.org/Part:BBa_K902041">this construct</A> was finished, we designed two experiments to characterize this biobrick. One was an in vivo experiment in which <I>E. coli</I> cells containing the biobrick were assayed with 3 compounds containing a primary amine or amide. Benzamide, which has similar structure to the naphthenic acids being targeted by other parts of the hydrocarbon project, and butyramide were previously shown to be good substrates for the AmdA enzyme (Hirrlinger et al, 1996). Benzamidine was also tested to explore the array of substrates that could possibly be degraded by the enzyme. The other approach we took was to create a cell lysate from <I>E. coli</I> that was grown up and resuspended it in a lysis buffer. This was assayed with the same 3 compounds to guard against the possibility that the in vivo system lacked the appropriate transfer proteins. Hopefully this stuff works.The data below from the GC shows that one compound gets completely degraded by the cells containing the <I>AmdA</I> gene as well as a new product being formed in this group. The MS data shows that the compound being obliterated by amidase is in fact benzamide and that the new product being formed is benzoic acid. Benzoic acid results from replacing the NH<SUB>2</SUB> with an OH group, which is the expected function of the AmdA enzyme. The protocol for this experiment can be found <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/AmidaseAssay">here</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">GC chromatograph demonstrating the degradation of benzamide by <I>AmdA</I> compared to an <I>E. coli</I> control.  Additionally, a smaller peak with an earlier retention time is produced solely in the sample run.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Mass Spectra for the peak at a retention time of 10.5 minutes from Figure 7 representing the substrate benzamine.  Additionally, the peak at retention time of 9 minutes represents benzoic acid based on its mass suggesting that their is a small accumulation of product.</DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">http://2012.igem.org/Team:Calgary/Notebook/Denitrogenation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Denitrogenation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Denitrogenation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Denitrogenation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Denitrogenation&amp;oldid=292998" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>