<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_FRED_Detecting"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/FRED/Detecting">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/FRED/Detecting&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Detecting&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Detecting&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/FRED/Detecting" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/FRED/Detecting</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Project</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Jamboree</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">A Transposon-Mediated Mutant Library for Toxin Detection</H1><P align="justify">
This year, our team wanted to identify a novel responsive element capable of detecting and quantifying different tailings ponds toxins (e.g. naphthenic acids, NAs) in solution. While numerous studies have begun to identify species of bacteria capable of surviving and sensing a variety of toxic compounds (e.g. NAs), the degradation pathways have not yet been fully characterized. Therefore, we needed to design and implement novel approaches to efficiently isolate the genetic elements that detect and potentially lead to the breakdown of these toxins.
</P><H2>Transposons: What, How, Why?</H2><P align="justify">
The transposable element (TE), Tn5, is a conservative transposon that can insert a segment of genes bordered by specific 19bp insertion sequences from one part of the genome (e.g. plasmid vector) randomly to another location like a chromosome (Reznikoff, 2008). The transposition event is catalyzed by a transposase enzyme encoded by <I>tnp</I> gene included in the TE. Using the appropriate selective pressure, the insertion can be maintained permanently in the genome.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 1: &quot;Transposition reaction from plasmid entry into the recipient cell to integration of the transposon into the genome. Modified from Transposons: Shifting Segments of the Genome&quot; by McGraw Hill</DIV></DIV></DIV></DIV><P align="justify">By inserting a vector construct containing the TE with selectable markers (such as tetracyclin resistance and lacZ) into an organism with a desirable phenotype, we can find out what genetic elements (e.g. genes and promoters) are responsible for that particular function. This can happen via a random insertion of a TE containing a promoterless reporter gene downstream of promoter elements that creates a transcriptional fusion, providing activity in response to specific environmental stimuli. Using a <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/tnscreen">bipartite-mating (conjugation) method</A> to transfer the TE vector into the organism of choice is an efficient method for creating the massive number of mutants required.</P><P align="justify">
Due to the complexity of biological systems, our team focused our efforts on utilizing a system for identification of promoter elements that respond specifically in the presence of environmental stimuli. Our hypothesis requires that the organisms we use respond specifically to particular toxins and result in upregulation of metabolic genes with little background effect in the cell. We recognize that any number of biological molecules may play a role in toxin sensing, such as enzymes, transcription factors, and even RNA elements (e.g. riboswitches). However, the identification of a promoter sequence takes us further in that we can better understand the degradation mechanism by elucidating the genes involved.
</P><H2>Toxin-Degrading Organism Used</H2><P align="justify"><I>Pseudomonas spp. </I>have been isolated from oil sands tailings ponds and shown to biodegrade model and tailings-associated NAs and nitrogen- and sulfur-containing heterocyclic aromatic compounds (Ramos-PadrÃ³n <I>et al</I>. 2010; Herman <I>et al</I>., 1994; Del Rio <I>et al</I>., 2006; Gieg &amp; Whitby, unpublished, 2012). This suggests that there exists systems that detect and up-regulate transcription specifically in response to these toxins.</P><P> We wanted to use a commercially available strain of <I>Pseudomonas fluorescens</I> characterized for a response to toxins found in tailings pond water (TPW). The <I>P. fluorescens </I>PF-5 strain (Paulsen <I>et al</I>., 2005) is reported to survive in and degrade a commercial mixture of naphthenic acids (Acros) (Gieg &amp; Whitby unpublished, 2012). Moreover, the genome sequence is available for this strain with annotations (Pseudomonas Genome Database V2, http://pseudomonas.com/). This allows us to use sequencing data from the mutants and identify where in the genome the TE insertion occurred, and what genes (if present) are located downstream of it.
</P><H2>Method Design</H2><H3>Mutant Library Generation</H3><P align="justify">
To construct the promoter library, a pOT182 vector construct (containing a IR-lacZ-Amp-pMB1ori-TetA-TetR-Tnp-IR transposable element)  is introduced into commercially purchased <I>E. coli SM10</I> donor strain.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption">Figure 2: The transposable Tn5 element used in the pOT182 plasmid, containing a lacZ reporter gene, ampicillin and tetracycline resistance, an <I> E. coli</I> origin of replication for use during downstream sequencing protocols, and transposase. The genes are flanked by the transposon insertion elements</DIV></DIV></DIV></DIV><P align="justify">The plasmid contains a RP4 mob conjugation region and a p15A origin of replication (ori), and is engineered to only replicate in <I>E. coli</I>. The TE construct is transferred from the <I>E. coli</I> donor strain to the recipient <I>P. fluorescens </I> PF-5 using bipartite mating via conjugation (enabled by the RP4 mob region). A random genomic library of transposon insertions is created in <I>P. fluorescens</I>, and selected by isolating the recipients that have a genomic TE insertion on Pseudomonas Isolation Agar/PIA with tetracycline. If a promoter element is fused upstream of the TE construct, then promoter activation will turn on the expression of lacZ, which can be detected by the degradation of a colorless compound, X-Gal, to an insoluble blue pigment product (an indoxyl compound) (Juers <I>et al</I>., 2012). If the fused promoter is activated in response to a stimulus, then the lacZ enzyme will be produced in response. Mutant strains sensitive to the particular toxic stimulus will appear as blue colonies on the selective plate.</P><H3>Mutant Strain Characterization</H3><P align="justify">Mutants generated are characterized for their roles in the response to toxins with dose response experiments, and compared to general stress-inducing agents (e.g. H<FONT style="text-transform: lowercase;">2</FONT>O<FONT style="text-transform: lowercase;">2</FONT>) and compounds such as fatty acids to ensure the specificity of the response. These measurements help to determine thresholds of detection, robustness of the signal, and specificity of response. The dose response curves will also assess the usefulness of correlating the concentration of NA to the level of response, and the possibility of measuring NA concentrations in a sample, rather than simply by presence/absence.</P><H3>Self-Cloning and Sequencing</H3><P align="justify">Last, self-cloning techniques are used to identify the upstream and downstream sequences from the TE insertion (Merriman and Lamont, 1993). The TE used is a self-cloning construct because it contains all the elements required for plasmid replication (i.e. origin of replication) and selection (Tet resistance). Genomic DNA from a desirable mutant is isolated, and restriction digested with BglII (a restriction enzyme that does not cut within the TE but numerous times within the genome). The resulting fragments may contain the TE construct with flanking sequences. The genomic fragments are circularized by self-ligation and transformed into <I>E. coli</I>. Plasmids from the transformed cells contain the TE construct with the upstream and downstream flanking sequencing connected by the BglII restriction site. Sequencing primers designed against the 19 bp recognition sequence in the TE to sequence the isolated plasmids.</P><P align="justify">For a detailed protocol, please consult our <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/tnscreen">methods section</A>.</P><H2>Results</H2><H3>Detection by Mutant <I>Pseudomonas fluorescens</I> PF-5</H3><P align="justify">After mating experiments and plating on selective media (Pseudomonas isolation agar, with tetracycline and naphthenic acids), 24 responsive (blue) colonies were found. Screens were conducted on these blue colonies found on selective plates comparing a response in LB and LB with 100mg/L naphthenic acids (both with X-Gal). When results were observed it was found that 4 mutant strains are differentially regulated in response to naphthenic acids: 66-1, 66-2, 170-1, and 199-1. These colonies were further screened to test the specificity of their responses.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 3: Transposons: Shifting Segments of the Genome: Initial Hit Screen Comparison Pictures. Colonies were inoculated in duplicate into both LB media, and LB media containing 100 mg/L ACROS commercial naphthenic acids. X-gal was added to the media at a final concentration of 200 Âµg/ml. Cells were allowed to grow at 30Â°C for 16hr. Blue coloration indicates levels of LacZ production. 4 colonies (66-1, 66-2, 170-1, and 190-1) showed differential regulation in naphthenic acids.</DIV></DIV></DIV></DIV><P align="justify">


Screens involving the use of different toxins at environmentally relevant concentrations were performed to determine if the sensing response was specific to naphthenic acids, or if a sensory response to general toxins had been found. In addition, hydrogen peroxide was used as one testing condition to determine if the response is simply stress-induced.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 4: Second Screen- 170-1. Cells were inoculated in duplicate at different dilutions into LB as a control, and LB containing different toxin compounds at environmental concentrations. Hydrogen peroxide was used to rule out a stress response. X-gal was added to the media. After 12h, deeper blue coloration was observed in the toxin wells compared to the LB control. The cells did not grow in the hydrogen peroxide due to an excessively high concentration.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure 5: Second Screen- 170-1. Cells were inoculated in duplicate at different dilutions into LB as a control, and LB containing different toxin compounds at environmental concentrations. Hydrogen peroxide was used to rule out a stress response. X-gal was added to the media. Absorbance was read at 615nm (maximal absorbance of X-gal) every hour. Higher absorbance was observed in the toxin wells compared to the LB control. The cells did not grow in the hydrogen peroxide due to an excessively high concentration.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 6: Second Screen- 66-1.  Second Screen- 170-1. Cells were inoculated in duplicate at different dilutions into LB as a control, and LB containing different toxin compounds at environmental concentrations. Hydrogen peroxide was used to rule out a stress response. X-gal was added to the media. After 24h, deeper blue coloration was observed in the toxin wells compared to the LB control. The cells did not grow in the hydrogen peroxide due to an excessively high concentration.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure 7: Second Screen- 66-1. Cells were inoculated in duplicate at different dilutions into LB as a control, and LB containing different toxin compounds at environmental concentrations. Hydrogen peroxide was used to rule out a stress response. X-gal was added to the media. Absorbance was read at 615nm (maximal absorbance of X-gal) every hour. Higher absorbance was observed in the toxin wells compared to the LB control. The cells did not grow in the hydrogen peroxide due to an excessively high concentration.</DIV></DIV></DIV></DIV><P align="justify">
From these screens, it was seen that both colony 66-1 and colony 170-1 appear to respond to toxins when compared to a response in LB media. In order to test the specificity of this response, an additional screen was performed using varying concentrations of hydrogen peroxide (to rule out activation by a general stress response in the cell) in addition to decanoic acid at a comparable concentration to that of the naphthenic acids used (to rule out activation due to sensing fatty acid compounds). The results of this can be seen below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 8: Stress response screen on <I>P. fluorescens</I> Pf5 transposon mutants. Cells were inoculated in duplicate at different dilutions (shown are <B>A:</B> 66-1 undiluted, <B>B:</B> 66-1 at 1/10 dilution, <B>C:</B> 170-1 undiluted, <B>D:</B> 170-1 at 1/10 dilution)  into LB as a control, LB containing varying concentrations of hydrogen peroxide, LB containing naphthenic acids at an environmental concentration, and LB containing decanoic acid at the same concentration as the naphthenic acids. 2 uL of 20mg/ml X-gal was added to the media and absorbance was read at 615nm (maximal absorbance of X-gal) every 4 hours for 12h. Higher absorbance was observed in the NA wells compared to the LB control, hydrogen peroxide, and decanoic acid for colony 66-1. Colony 170-1 showed a repressed response to naphthenic acids when compared to the LB control.</DIV></DIV></DIV></DIV><P>
These results show that colony 66-1 gives a response to naphthenic acids and other toxins that is not simply a response to fatty acids or a general stress response. Unfortunately, colony 170-1 does not show a useful reporter response.</P><H2>Promoter Constructs Isolated</H2><P align="justify">To determine the location of the transposon insertion, we utilized the self-cloning properties of the transposon. By digesting the genome, religating, and transforming the ligated genomic fragments into <I>E. coli</I>, plasmids containing the transposon and flanking gene sequences were isolated. These plasmids have been isolated and sent for sequencing. However, we are having difficulty with getting sequencing reactions to produce a read. The results so far are a promising step towards finding a sensory element for our reporter system that would allow for the detection of various toxins in tailings ponds. </P><P> 
Our next steps were to test these strains in conjunction with our electrochemical detector as well as see if they could detect tailings toxins. The results of this can be seen on the <A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy page</A>.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Project/FRED/Detecting">http://2012.igem.org/Team:Calgary/Project/FRED/Detecting</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/FRED/Detecting" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/FRED/Detecting" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Detecting&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/FRED/Detecting&amp;oldid=297028" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>