<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Protocols_tnscreen"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Protocols/tnscreen">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Protocols/tnscreen&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Protocols/tnscreen&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Protocols/tnscreen&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Protocols/tnscreen" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Protocols/tnscreen</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Notebook</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Killswitch">Killswitch</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/PromoterScreen">Promoter Screen</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Electrochem">Electrochemistry</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Biosensor">Biosensor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/CatecholDegradation">Catechol</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/FluxAnalysis">Flux Analysis</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocol Manual</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Transposon-Mediated Mutant Library Generation</H1><H2>Materials</H2><UL><LI><I>E.coli</I> SM10 pOT182</LI><LI><I>Pseudomonas fluorescens</I> PF-5</LI><LI>LB Broth </LI><LI>Pseudomonas isolation agar (PIA)</LI><LI>Mix naphthenic acid solution (Acros, 0.91g/mL) or any other toxin</LI><LI>Tetracycline (stock concentration at 5mg/mL)</LI><LI>X-Gal</LI><LI>Hydrogen peroxide</LI><LI>Decanoic acid</LI></UL><H2>       Day 1</H2><OL><LI>Around 3 or 4pm, prepare an <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/onculture">overnight culture</A> of donor <I>E.coli</I> SM10 pOT182 in 2mL of LB with 10μg/mL of tetracycline, and incubate overnight at 37°C. </LI><LI>Subculture the recipient <I>Pseudomonas fluorescens</I> PF-5 as well in 5mL LB+50mg/L NA media, and incubate at 30°C, shaking at 110rpm. </LI><UL><LI>Note: this protocol is designed to isolate NA sensing promoters. To isolate promoters sensitive to other compounds, simply replace NA with the compound of interest.</LI></UL><H2>Day 2</H2><LI>Make the selection media (Pseudomonas isolation agar/PIA) for the next day. The selection media contains PIA, tetracycline, NAs, and X-Gal.</LI><LI>To make the media, mix 45g of the PIA powder in 1L of milliQ or nanopure water containing 20mL of glycerol. Add 54.95µL of the Acros NAs. Label the bottle properly, mix well using a magnetic stir bar, and autoclave.</LI><LI>After autoclaving, allow agar to cool to about 55°C in a water bath. Add tetracycline to a final concentration of 50μg/mL, and the X-Gal to a final concentration of 20 μg/mL. Mix the media well by slowly stirring (magnetically) while adding X-Gal and tetracycline.</LI><LI>Pour the agar solution into labelled petri dishes, allow to solidify (in about 1hr), and store at 4°C in the dark to prevent antibiotic degradation.</LI><LI>At 2pm, re-subculture <I>E. coli</I> SM10 in 2mL of LB (no tetracycline) to remove the tetracyline. Incubate by shaking at 37°C for 2hr.</LI><LI>At 4pm, prepare the following in 2mL tubes:

</LI><LI>Negative control: 500µL <I>Pseudomonas fluorescens</I> PF-5 (from the culture prepared in Day 1)</LI><LI>Negative control: 500µL of the <I>E.coli</I> SM10 (the re-subcultured sample) </LI><LI>Experimental sample: 500µL of the <I>E.coli</I> SM10 (the re-subcultured sample) and 500µL of of the <I>Pseudomonas fluorescens</I> PF-5 cultures</LI><LI>Mix all samples by vortexing and centrifuge all the tubes at max speed for 5min.</LI><LI>Discard the supernatant and resuspend the cell pellet in 50μL of LB.</LI><LI>Pipette all 50μL onto a LB (no additives) plate. A single LB plate can be divided in three sections (make sure to label all the sections). Draw on the back of the plate like so:</LI><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:199px;"><DIV class="thumbcaption">Figure 1: Mating spot setup on a a LB plate</DIV></DIV></DIV></DIV><LI>Let the plate sit on the bench at room temp until all the spots have dried. </LI><LI>Incubate overnight at 37°C.</LI><H2>Day3</H2><LI>In the morning, scrape up the mating spots using a sterile pipette tip, and resuspend in 500μL sterile distilled water.</LI><LI>Mix thoroughly by vortexing or pipetting up and down. Do this for each spot from the LB plate.</LI><LI>For the mating spots (with both <I>E.coli</I> and <I>Pseudomonas fluorescens</I>), make serial dilutions from the resuspension (1/10, 1/100, and 1/1000).</LI><LI>Make spread plates using 100 µl of the undiluted mixture and each of the dilutions on PIA selection plates.</LI><LI>Plate 100μL of the resuspended <I>Pseudomonas fluorescens</I> PF-5 (alone) spot onto a PIA plate as well (no dilution required). Do the same for the <I>E.coli</I> SM10 spot.</LI><LI>Incubate overnight at 30°C. Check the next morning for growth.</LI><H2>Day4</H2><LI>Check the plates for growth. Do not allow the plates to overgrow in order to isolate single colonies.</LI><LI>Ensure no growth is present on the plates containing the </LI><LI>Pseudomonas fluorescens PF-5 (alone) and <I>E.coli</I> SM10 (alone) spots (At least at a reasonably low level indicating a small amount of spontaneous mutations).</LI><LI>On the experimental sample plates, isolate all blue colonies in <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/onculture">overnight cultures</A> (in LB+10μg/mL tetracycline+50mg/L NAs). Make <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/glycerolstock">glycerol stocks</A> from the overnight cultures.</LI><H2>Day5</H2><LI>The strains isolated from the blue colonies are further screened in a 96 well plate. Each strain is inoculated into the following set of media (300μL) in the plate:</LI><UL><LI>LB alone</LI><LI>LB with 200μg/mL X-Gal</LI><LI> LB with 50mg/L NAs and 200μg/mL X-Gal </LI></UL><LI>Make sure the plate setup is recorded properly, and make an initial (0hr) reading using a spectrophotometer at 615nm to measure the X-Gal degradation product concentration.</LI><LI>Incubate the plate at 30°C in a shaker set at 140rpm for 16hr.</LI><H2>Day6</H2><LI>Measure the absorbance of the plate once again at 615nm at 16hr.</LI><LI>Based on the absorbance readings, record any colony that has a higher absorbance in the LB+NA+X-Gal sample than the LB+X-Gal sample. Preferably, the colony should show no growth in the LB+X-Gal sample.</LI><LI>Ensure the selected colonies are <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/glycerolstock">glycerol stocked</A>, and subculture these colonies in LB+10μg/mL tetracycline+50mg/L NAs.</LI><H2>Day7</H2><LI>Using the selected strains from the previous day, perform a more detailed assay for each strain using undiluted, 1/10, 1/100, and 1/1000 of the culture in LB+50mg/L NAs (duplicates for each dilution). </LI><LI>Also test the colonies in LB medium with 10μM, 50μM, and 100μM hydrogen peroxide, as well a fatty acid (e.g. decanoic acid) to ensure the response is not stress-induced, and specific to NAs.</LI><LI>The assay can be set up again in a 96-well plate.</LI><LI>Incubate the plate at 30°C in a shaker set at 140rpm for 12h-24h.</LI><H2>Day8</H2><LI>Keep strains that respond specifically to NAs (or any other toxin), i.e. X-Gal degradation and growth in LB+NAs, but no/less X-gal production in LB+ varying amounts of hydrogen peroxide or LB+ decanoic acid.</LI><LI>For these strains, <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/gemomicprep">purify genomic DNA</A> from overnight cultures. Measure the DNA concentration, and run on a gel to confirm that the DNA is not degraded.</LI><LI>Completely <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/construction">digest</A> 600ng of the genomic DNA with BglII. (This restriction enzyme cuts frequently (6-cutter) in the <I>Pseudomonas fluorescens</I> PF-5 genome and not in the transposon). Run the reaction overnight at 37°C for 16hr.

<H2>Day 9</H2></LI><LI> Inactivate the BglII enzyme by performing a <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/pcrpurification">enzymatic reaction cleanup</A> (using a PCR purification kit, as the enzyme cannot be heat inactivated).</LI><LI>Using 50ng of DNA, perform a <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/construction">self-ligation</A> to circularize all the genomic fragments.</LI><LI>Transform the ligation mixture (all 20µL) into <I>E. coli</I> DH5α or Top10. Plate on LB+10μg/mL tetracyline plates.</LI><H2>Day 10</H2><LI>Prepare <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/onculture">overnight cultures</A> of the transformed cells in 5mL of LB+10μg/mL tetracyline.</LI><H2>Day 11</H2><LI>Purify the plasmid from the overnight cultures by <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/plasmidminiprep">miniprep</A>.</LI><LI>Run the isolated plasmids on a gel to ensure the right size (at least the size of the transposon).</LI><LI>Sequence the plasmids using sequencing primers targeted against the insertion sequences of the transposon.</LI><LI>When sequencing results are received, identify the flanking sequences to the transposon insertion. The sequence outside the transposon (in the plasmid) should have a BglII restriction site. This would allow us to determine where the sequence upstream to the insertion occurs, and where the sequence downstream of the insertion ends.</LI><LI>Blast the flanking sequences against the <I>Pseudomonas fluorescens</I> genome to determine the position of the genomic insertion.</LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Notebook/Protocols/tnscreen">http://2012.igem.org/Team:Calgary/Notebook/Protocols/tnscreen</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Protocols/tnscreen" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Protocols/tnscreen" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Protocols/tnscreen&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Protocols/tnscreen&amp;oldid=290568" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>