<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Killswitch"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Killswitch">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Killswitch&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Killswitch&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Killswitch&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Killswitch" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Killswitch</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Notebook</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Killswitch">Killswitch</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/PromoterScreen">Promoter Screen</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Electrochem">Electrochemistry</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Biosensor">Biosensor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/CatecholDegradation">Catechol</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/FluxAnalysis">Flux Analysis</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocol Manual</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Team Killswitch</H1><H2>Week 1 (May 1-4)</H2><P>This was the first week where we met with other team members and summarized the primary subprojects the team will be tackling this coming summer.</P><H2>Week 2 (May 7-11)</H2><P>Members were assigned to the killswitch team. We spent a majority of this week performing literature searches and narrowing the killswitch to a few ideas.</P><P>The mechanism of death is still settled upon the micrococcal nuclease, but regulation of the death genes will be difficult. Existing repressible promoters in the registry still tend to be leaky in their expression, but we need to test this out. We may look into the TetR-repressible promoter (<A href="http://partsregistry.org/Part:BBa_R0040">R0040</A>) and also the clλ regulated promoter (<A href="http://partsregistry.org/Part:BBa_R0051">R0051</A>).</P><P>We also found a few riboswitches responsible to various metal ions such as Mg<SUP>2+</SUP> and Mn<SUP>2+</SUP>. The <I>mgtA</I> riboswitch activates translation of the death gene when magnesium ions are not present in the solution. The <I>mntA</I> riboswitch deactivates translation of the death gene when manganese ions are not present. Considering this, we may be able to find a method of precipitating or otherwise sequestering Mn<SUP>2+</SUP> ions out of the tailings water prior to entering the bioreactor. This way, our bacteria would not die when they come into contact with the manganese in the tailings water. Possible additives include carbonate (CO<SUB>3</SUB><SUP>2-</SUP>) or hydroxide (OH<SUP>-</SUP>), though this may alter the pH greatly.</P><P>One other proposal would be to use promoters that activate when the bacteria detect the formation of a biofilm, for example, on glass beads. Engineering a monolayer of cells on a bead means that if a cell detaches, its death genes will activate. Further research into this is needed.</P><H2>Week 3 (May 14-18)</H2><P>We are continuing to look in to various other pathways. We discussed the possibility of a NOR-gated system to add a second layer of regulation to the kill gene. It would require the production of riboswitch ligands. Possible ligands include glucose, amino acids, and molybdenum cofactor.</P><P>We also looked into a co-dependent system where we would transform <I>E. coli</I> with two different plasmids and put them in the same bioreactor such that they produce ligands for each other that would repress the expression of the restriction enzyme and the nuclease. For this we explored riboswitches further and came up with the idea of using MOCO and SAM riboswitches as SAM is soluble in water and not found in the tailings ponds.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Proposed method of codependence. Two plasmids in two separate strains will produce ligands that repress the riboswitch controlling each other's death genes.</DIV></DIV></DIV></DIV><P> We also found a glucose repressible promoter that we could have potentially used upstream of the kill switch. This promoter is found in <I>F. tularensis</I>. However we also found that this promoter is not found in <I>E. coli</I>, and  <I>F. tularensis</I> has a unique polymerase which used this promoter. Hence using this promoter does not seem feasible. 

Next week we will be looking into further possibilities for the kill switch.

</P><H2>Week 4 (May 21-25)</H2><P>In terms of our literature search we decided against the glucose repressible promoter from F. tularensis because it was not native to E.coli and upon contact with the authors of the paper, we found that the promoter did not work in E.coli. We also came upon two systems that we are interested in further investigating. The first one was a glucose activated promoter that was found in the registry which we are thinking of coupling to an inverter and the second one is a rhamnose-based regulatory system.</P><P>On Tuesday, in addition to our literature research, we decided on some of the circuits that we will be building over the next couple of weeks and came up with a plan detailing what will be done each day. These are the circuits:
</P><P> On Wednesday we proceeded with the transformation of 3 parts from the iGEM kit into our E.coli: a tetR promoter, a GFP-double terminator, and a RBS-GFP-double terminator. 
On Thursday we verified our transformation process with a cPCR where we ran 10 colonies from each plate with our controls. We ran a gel of the PCR products and viewed it to verify the transformation and to decide which of the successful colonies to plate and grow overnight. </P><P> On Friday we isolated the plasmids through a mini prep. 
The plan for next week is to do a restriction digest and run it on a gel to confirm that the plasmids have the part. We also want to start more transformation of our parts into E.coli such as the riboswitches and the promoters. We want to verify the transformation, isolate the plasmids and verify, and start the DNA construction digest to start building our circuit.</P><H2>
Week 5 (May 28-June 1)
</H2><P>
During the week of May 28th, the killswitch team made progress toward transforming/verifying nine different parts in <I>E. coli</I>. Work began with continuation of the previous week's transformation of R0040 (TetR promoter), I13401 (GFP terminator), and I13504 (RBS GFP terminator). Note that these three parts will be necessary for characterization of the four different kill system triggers being explored by the killswitch team. Verification procedures (ie, colony PCR) showed successful transformation of I13401. Results for the other two parts were inconclusive and cloning of R0040 and I13504 will continue next week. 
</P><P>
The commercially synthesized mntA promoter, mgtA promoter, and mgtA riboswitch for the magnesium and manganese riboswitches were also transformed into <I>E. coli</I>. While colonies were successfully isolated, colony PCR was inconclusive because incorrect primers were used for the commercial plasmids. The team proceeded to isolate the plasmids from <I>E. coli</I>, perform a restriction digest of the parts, and ligation of these components into biobrick vectors. These products were transformed into other <I>E. coli</I> and the team is ready to verify presence of the cloned digest products.
</P><P>
In continuation with magnesium and manganese riboswitches, the team attempted to biobrick mntR (a repressor of a manganese transporter activated by manganese), PhoP (a transcriptional regulator of mgtA), and PhoQ (a signal transducer which activated PhoP in the presence of divalent cations). The below PCR indicates that while PhoQ and mntR were successfully isolated, PhoP was not. The former two were transformed into <I>E. coli</I>; the transformed samples grew into a lawn and colony PCR will thus be performed next with re-streaked plates. Finally, gradient PCR was used in an attempt to biobrick PhoP.
</P><P>Lanes 6-9: PhoQ (~671 bp)  Lanes 10-13: mntR (~500 bp)</P><P>
Primers for the MOCO and rhamnose killswitch actuators were also designed. With respect to the MOCO synthesis operon and MOCO riboswitch, primers were finalized and sent to senior members of the team. While primers for isolation of rhaSR and the rhaT promoter from <I>E. coli</I> were given in the paper which inspired the rhamnose promoter idea, the team was resistant to using the sequences since they did not seem to correlate with genes of interest in a BLAST search. Closer inspection revealed that the forward primer listed in the paper added a number of bases (&gt;50) which were not native to the <I>E. coli</I> genome. Only 18 base pairs of the primer were complementary to <I>E. coli</I>—this discrepancy was responsible for the negative results of the BLAST alignment. Work will continue next week to optimize the rhamnose primers. They hope to order both the MOCO and rhamnose primers next week so that they may proceed with construction of the kill switch. 
</P><H2> Week 6 (June 4th-8th)</H2><P>This week we continued our attempt to biobrick PhoP, PhoQ and mntR. A gradient PCR was run with temperature and salt gradient (0.5mM-3.0mM). however this PCR showed no success whatsoever. We discovered that the primers designed for these parts were faulty and were binding in several places in the gene and thereby favouring products that are smaller in size. </P><P>

phoP-S	GTTTCTTCGAATTCGCGGCCGCTTCTAGatgCGCGTACTGGTTGTTG</P><P>
phoP-AS	GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAtcaGCGCAATTCGAACAGATAG</P><P>
phoQ-S	GTTTCTTCGAATTCGCGGCCGCTTCTAGatgaaaaaattactgcgtc</P><P>
phoQ-AS	GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAttattcatctttcggcgcag</P><P>
mntR-S	GTTTCTTCGAATTCGCGGCCGCTTCTAGatgagtcgtcgcgcaggtac</P><P>
mntR-AS	GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAtcatttggcaccgtgtttc
 </P><P>Hence new primers were designed. </P><P> 	PhoPNEW-AS 	CTGCAGCGGCCGCTACTAGTAGCAGTAATTTTTTCATCAGCGCAATTCG</P><P> PhoQNEW-AS 	CTGCAGCGGCCGCTACTAGTAGCATATTTATTCATCTTTCGG </P><P>	mntRNEW-S 	gaattcgcggccgcttctagGAGGAAGCACAATGAGTC </P><P>	mntRNEW-AS 	CTGCAGCGGCCGCTACTAGTAGCGTGCGTAAAAAAGG</P><P>The killswitch team also focused on verification of both the magnesium/manganese (MgtA promoter, MgtA riboswitch and MntP promoter) and registry parts (R0040, I13401 and I13504).</P><P>For the magnesium/manganese parts this week we insert them in biobrick vectors in order to carry out colony PRR. Colony PCR of the magnesium/manganese parts could not have been done first since we did not have the right primer , therefore by doing a construction digest we could use the standard biobrick primers. From the gel, the results were inconclusive as the ladder was not clear and the parts were too small to tell the difference between them and primer dimers, therefore next week goal is to perform another CPCR of the parts. As well as doing colony PCR of the MgtA promoter, a restriction digest of this part was done. The gel showed good results as the bands were bright and the right size.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Restriction Digest of MgtA Promoter</DIV></DIV></DIV></DIV><P>For the registry parts R0040 no results were shown in the initial colony PCR, however another gel was done with R0040 from the 2010 and 2011 registry plate. The colony PCR of R0040 from 2011 showed the rights bands, however the results are inconclusive as the negative was contaminated, therefore most likely another colony PCR needs to be done.</P><P>The registry part I13401 had a successful week as both the colony PCR and the restriction digest verified our parts. Since the result was successful we performed glycerol stocks of the plasmid and prepared it for sequencing the following week.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Colony PCR of Registry Part I13401</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Restriction Digest of Registry Part I13401</DIV></DIV></DIV></DIV><P>The part I13504 showed no results this week therefore the goal next week is to find another part that is similar to I13504 and test its verification.</P><H2>Week 7 (June 11-14)</H2><P>

This week we continued the insertion of magnesium promoter (MgtA) into biobrick plasmid (PSB1C3) to carry out the colony PCR to confirm the presence of the Mgta part in the plasmid using Bbk_CP_F and Bbk_CP_R primers. We are currently in the process of further verification by using a mini prep.
We also transformed four registry parts (J06702 [RBS-mCherry-dTerm], I13507 [RBS-RFP-dTerm], E0840 [RBS-GFP-dTerm] and E0240 [RBS-GFP-dTerm]) directly from the kit plates and verified the parts using colony PCR the following day. The primers used to verify these parts were Bbk_CP_F and Bbk_CP_R. These parts were successfully transformed but we still saw some contaminant bands in the gels (see below).  
</P>]<P>

Transformation of MntP in PSB1C3 vector was verified using colony PCR. However, the results indicated that the presence of MntP as well as a contaminant band around 3000 bp.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Colony PCR of Registry Parts where J04650 was successful</DIV></DIV></DIV></DIV>]<P>
Transformation of registry parts I13521 [TetR-B0034-RFP-dTerm], I13507 and J04650 [RFP-dTerm] were performed and colony PCR was carried out to verify the presence of the parts in the cells. J04650 was the only part that was successfully transformed. Though the other parts were not successfully transformed, we can still see non-specific bands in the gel (see below). We are currently doing mini prep to additionally verify the success of transformation.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Colony PCR of Registry Parts where J04650 was successful</DIV></DIV></DIV></DIV><H2>Week 8 (June 18-June 22)</H2><P> This week we did a restriction digest of all the plasmids that were isolated. We confirmed that we had J04650, I13507, E0240 and mgtA riboswitch. Hence these were sent to sequencing. We got sequencing results back and confirmed that we have J04650, I13507, J06702, R0040, E0240 and mgtA riboswitch. Henceforth, we started constructing the mgtA riboswitch with J04650. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Restriction digest of mgtA riboswitch and J04650 construction</DIV></DIV></DIV></DIV><P> The mgtA riboswitch with J04650 was confirmed using PCR as well as restriction digest. We plan on sequencing it when we are able to construct it with a promoter. </P><H2>Week 9 (June 25-June 29)</H2><P> This week we attempted to construct the mgtA riboswith and J04650 construct with promoters such as MgtA and TetR. We also tried to plasmid switch I13507 such that we can put mgtA promoter on to that composite part. So far we have verified through colony PCR that it worked.  This was verified again with restriction digest however the gel did not look too good therefore we ran the inserts and vectors to see how they looked.  We will reattempt this again next week. </P><H2>Week 10 (July 3-July 6)</H2><P> This week, we attempted to religate our constructions from the previous week as they were not verified as positive in our colony PCR. We had run a gel to see if the inserts and vectors were digested properly and they were. Following transformation, we verified it through cPCR. The two gels show the results of construction involving two promoters (Mgta Promoter and a constitutive tetR promoter, R0040) and the mgta rbsw with different reporter gene constructs from the previous week. There are also some plasmid switches and constructions of the promoters and just the riboswitch. Based on these results, we isolated plasmids of R0040 + (Mgta RBSW and I13401) from L6 and L7 as well as I13507 from L8 from the first gel and R0040 + (Mgta RBSW and J04650) from the second gel (circled in red). We also transformed B0034, a ribosome binding site and constructed Mgta rbsw with a reporter construct with fast degrading GFP. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Gel 1: Colony PCR of Constructions</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Gel 2: Colony PCR of Constructions</DIV></DIV></DIV></DIV><P> We verified these constructions with a digest. The third gel shows the digest and the lanes circled are the ones that look good and were sent to sequencing. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Verification digest of constructions</DIV></DIV></DIV></DIV><H2>Week 11 (July 9-July 13)</H2><P> During the week of July 9th, construction was done on R0040 with J04650 and K082003 constructs as well as E0240 and B0034. Similarly the MgtA promoter was constructed with either J04650 construct, K082003 construct, E0240 or B0034. Furthermore a gel was run on the J04650 and K082003 constructs to check for genomic contamination which showed positive results for sequencing. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Gel of the Mini Prep to check for genomic contamination</DIV></DIV></DIV></DIV><P> From the restriction digest of the construction R0040 or MgtA promoter with the J04650 or K082003 constructs little results was shown, however some of the sequencing presented varying results. Sequencing of the R0040 with MgtA riboswitch and J04650 worked beside for weird sequencing around the scar site while both registry part E0240 and B0034 were good. The construct with MgtA promoter did not work as sequencing showed that the promoter was not part of the construct therefore the construction of these parts will have to continue another week.  </P><P> Results from colony PCR showed positive bands for R0040 (vector) with B0034 (insert), R0040 (vector) with E0240 (insert) and B0034 (vector) with R0040 (insert). </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Colony PCR of R0040 constructions</DIV></DIV></DIV></DIV><P> Construction of these constructs will be done another week as the restriction digest did not look good at the same time the ladder was not readable. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Unreadable gel due to the ladder smearing</DIV></DIV></DIV></DIV><H2>Week 12 (July 16 to July 20)</H2><P>During week twelve (July 16th to July 20th), the kill switch team worked on three different projects: first, transformation/verification of kill genes; second, transformation and PCR of parts for the rhamnose inducible kill system; and third, construction of parts for characterization of the magnesium riboswitch kill system.</P><P>The commercially synthesized kill genes K131009 (colicin E2 operon w/o SOS promoter) and S7 nuclease (micrococal nuclease gene) were transformed into E. coli. Colonies transformed with these genes, along with the previously transformed K117000 (for colicin activation), K112808 (holin-antiholin lysis system), and endolysin, were subjected to verification procedures including colony PCR and verification digests. Successful transformants were sent off for commercial sequencing.</P><P>The team also began work on the the rhamnose inducible kill system since primers for rhaR and rhaS genes were received along with the synthesized rhamnose promoter. Temperature gradient PCRs (54C-59C; also 54C-65C with phusion)for the rhaR and rhaS genes were performed with no success. The rhamnose  promoter was transformed into E. coli of which a plasmid stock was isolated by miniprep.</P><P>Finally, the team continued with constructions necessary for characterizing the magnesium riboswitch kill system. They continued with verification of the MgtA riboswithch with GFP and also GFP(+LVA tag) under a constitutive promoter.
</P><H2>Week 13 (July 23 to July 27)</H2><P>During the week thirteen, the killswitch team performed plasmid switches of S7, mntP riboswitch and pRha (rhamnose inducible promoter) from the commercial IDT synthesis vector into pSB1C3.</P><P>Additionally, the team attempted to remove RhaR and RhaS from the E. coli genome via PCR. Kappa polymerase was used with a 50-65 degrees Celsius temperature gradient with no success. Given this failure, touchdown PCR from 58 to 74 degrees Celsius was performed on the RhaR and RhaS genes. These strategies failed. The team re-analyzed the primers used in these protocols; the antisense primers were erroneous and had to be redesigned.</P><P>Additionally, we tried construction of mgtAp with mgtArbsw-K082003. However no positive clones appeared in the follow up verification.</P><P>We also worked on construction of R0040+B0034, R0040+E0240, R0040+MgtA Riboswitch, R0011+MgtA Riboswitch, MgtA Promoter+B0034, MgtA Promoter with MgtA Riboswich and GFP(+LVA tag), and B0034 with GFP(+LVA tag). These constructions are all in different phases of the verification process. The team also attempted to expedite construction of pRha (rhamnose promoter) and S7 (micrococcal nuclease) with BOO34 by building the directly from the commercial IDT synthesis vectors; however, this process failed due similar resistance genes on the IDT and BOO34 vectors.</P><H2>Week 14 (July 30 to August 3)</H2><P>This week we tried to construct R0040 and R0011 with mgtArb-K082003.  This construction was done both ways, i.e both R0040 and mgtArb-K082003 were used as both vectors and insert. Out of these 2 constructions, R0040-mgtArb-K082003 gave positive clones however R0011-mgtArb-K082003 did not. A sample from a single colony with R0040-mgtArb-K082003 in pSB1C3 was sent to sequencing; the results were postively verified.</P><P>Additional constructions that were started include the following:</P><UL><LI><P>	R0011-mgtArbs</P></LI><LI><P>	R0011-mgtArbs-K082003</P></LI><LI><P>	mgtArb-S7</P></LI><LI><P>	pRHA-B0034</P></LI><LI><P>	B0034-S7</P></LI><LI><P>	R0011-B0034</P></LI><LI><P>	mntPrbs-K082003</P></LI><LI><P>	mntPrbs-S7</P></LI><LI><P>	B0034-S7 (ie, S7 in pSB1C3)</P></LI></UL><P>Digest verification showed success with K082003+B0034, J13002+K082003, R0040+ MgtA Riboswitch, and R0040+E0240; henceforth these were submitted for sequencing.</P><H2>Week 15 (August 6 to August 10)</H2><P>Colony PCR was performed on constructions from the previous week. Positive results were obtained with the following constructions: R0011-MgtAriboswitch, MgtAriboswitch-S7, MntpRiboswithch-S7, R0011-B0034, and MntpRiboswich-K082003. Minipreps and verification digests were performed from the relevant colonies.</P><P>The commercially synthesized Mntp riboswitch and moco riboswitch were switched into PSB1C3 vectors. Colony PCR and restriction digest verification yielded positive results; these samples are destined for sequencing verification. Additionally, the team constructed tetR-RBS-GFP(+LVA. Verification procedures were performed. TetR-RBS-GFP(+LVA) is awaiting sequence verification.</P><P>Temperature gradient PCR and salt gradient PCR were used to optimize and amplify the moa operon from the E. coli genome. The rhaR gene was amplified from Top Ten E. coli colonies and and biobricked in pSB1C3 with Kapa polymerase. The rhaS gene was amplified from Top Ten E. coli on a 52 to 67 degree Celsius temperature with Kapa polymerase and DMSO.</P><P>
Continued working with MgtA Promoter+ B0034, R0011 +MgtA Riboswitch,B0034+K082003 and trying to get DNA of the MntP Promoter.</P><H2>Week 16 (August 13 to August 17)</H2><P>For the week of August 13th continuation of the construction of the MgtA Promoter +B0034, R0011+MgtA riboswitch and B0034+K082003 was conducted. Constructions of MgtA Promoter+B0034 and B0034+K082003 were tested with plasmid PCR several times however none of the plasmid PCR showed any bands on the gel. R0011+MgtA riboswitch restriction digest did not work therefore a reconstruction was done and will follow thought for the next week.  One of the colonies from MntP Promoter IDT plate digested well and was therefore plasmid switch into pSB1C3.</P><P> We started plasmid switching parts that were not in pSB1C3 into pSB1C3. Additionally, we also started to characterize the magnesium system using mgtAp-mgtArb-K082003, mgtAp-mgtArb, R0040-mgtArb-K082003. We also tried to construct R0011-mgtArb, mntPrb-S7, mntPrb-S7J13002-K082003, R0011-mgtArb-K082003, B0034-S7. At the end of the week we were able to sequence verify B0034-S7 and mntPrb-K082003. </P><P>Finally, the rhamnose promoter system has been reoriented:</P><P>In this finalized version of the construct, the RhaR gene has been left out from the system. Given that RhaR activates RhaS, and RhaS activates the pRha promoter, it should theoretically be sufficient to constituitively express the RhaS promoter to activate the system. Construct must continue so that it can be tested whether this new design will function and be repressible with glucose. Check out the <A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation#rhamnose">project page</A> for more details.
</P><H2>Week 17 (August 20 to August 24)</H2><P>For the week of August 20th most of the constructions were redone while we continue with any other constructions we have already started since we noticed that we haven’t fully heat kill the enzymes. MgtA Promoter + B0034 was constructed with another B0034 that was in a kan vector while the MntP Promoter was redone from IDT as the previous MntP Promoter was not working.</P><P> This week we tried to construct mgtAp-mgtArb-S7-mgtArb so that we can construct CviAII when it would arrive. </P><P> We also attempted redoing the nuclease assay with S7 and <I>E. coli</I> genomic prep.This time the three different amounts of S7 was used. We used 200U, 50U and 10U. </P><P> The moa operon was biobricked and sequence verified. We are trying to construct the moa operon with a costitutive promoter and a ribosome binding site. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">There is degredation with S7 and it looks like 10U is optimal for our purposes</DIV></DIV></DIV></DIV><P>Constructions of the rhamnose promoter (pRha) with GFP and S7 continued. These constructions have proven challenging and will have to be repeated next week.</P><H2>Week 18 (August 27 to August 31)</H2><P>For the week of August 27th constructions of MgtA Promoter +B0034, MntP Promoter +B0034 in Kan, R0010+ MgtA Riboswitch, R0010 +MntP Rbsw + S7, MntP promoter + MntP Riboswitch + S7 had to be redone as the ones done on last Friday did not grow or verification through PCR did not show positive results. After repeat trails of these construction CPCR showed that positive bands for MntP Promoter plasmid switch into kan vector, MgtA Promoter+B0034, MntP Promoter +B0034 (Kan), R0010 +MgtA Riboswitch and R0010 + MntP Riboswitch + S7. Since there was positive results was shown for these constructions, mini prep and digestion verification will be done the following week.</P><P> This week we kept reattempting our constructions however sequencing came back as things it should not have been randomly. Mostly Car genes from the denitrogenation team. Therefore we had to back to our glycerol stocks from the previously verified part and try to reconstruct our parts.</P><H2>Week 19 (September 3 to September 7)</H2><P>For the week of September 3rd constructions (MntP Promoter plasmid switch into kan vector, MgtA Promoter+B0034, MntP Promoter +B0034 (Kan), R0010 +MgtA Riboswitch and R0010 + MntP Riboswithc + S7), was verified with restriction digest and sent to sequencing. Also MntP Promoter+ MntP Riboswitch +S7 construction was reattempted after the enzyme incident where they were not fully heat killed.</P><H2>Week 20 (September 10 to September 14)</H2><P> The moco riboswitch was biobricked and sequence verified. Constructions of the moco riboswitch with different promoters (R0010, R0011, R0040) were attempted along with the moco riboswitch with the GFP LVA tag, CviaII and S7. The attempts of construction of the moa operon with the constitutive promoter and a ribosome binding site have not been successful thus far and is an ongoing project.</P><P> Some of the constructions of pRha and GFP looked promising in colony PCRs and verification digests. These samples were sent for sequencing. In the meantime, we attempted to induce the potential colonies with rhamnose to see if there was evidence of a successful construction prior to sequencing results. Results were negative compared to constituitively expressed GFP and a negative control. We awaiting sequencing results to confirm this failure.</P><H2>Week 21 (September 17 to September 21)</H2><P> The constructions with the moco riboswitch and the moa operon were not successful and were reattempted this week. </P><P> Rhamnose promoter constructions with GFP were negative as shown in sequencing results. We recommenced with this construction.</P><H2>Week 22 (September 24 to September 29)</H2><P> The constructions with the moco riboswitch and the kill genes and the GFP with LVA tag looked promising on the gels and were sent to sequencing. We tried to construct those constructs with a promoter. R0010 with the moco riboswitch and CviaII looked promising on the gels and was also sent to sequencing. </P><P> This week we reattempted the nuclease assay to compare BglII, BamHI, S7 and CviAII against each other. This time we used 10U of enzyme in each condition and tested degradation at every 45 minutes. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 1: This assay compares the enzymes present in the regitry i.e, BglII and BamHI to the enzymes added by us, S7 and CviAII. This shows that S7 and CviAII degrade the DNA much quicker than BglII and BamHI combined.</DIV></DIV></DIV></DIV><H2>Week 22 (October 1 to Oct 3)</H2><P> The molybdate assay was attempted this week. mgtAp-rb-S7 construct was tested with mutated S7 and the synthesized S7. OD readings and CFU measurements were taken. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption">Figure 2: This shows the OD600 values of mgtA circuits with S7 both mutated and unmutated. The negative control consists of <I>mgtAp-mgtArb</I>.</DIV></DIV></DIV></DIV><P> Figure 2 shows that the mgtAp-mgtArb-S7 (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K902018">BBa_K902018</A>) starts acting approximately 4 hours after induction. However,it also shows that 10mM MgCl<SUB>2</SUB> is not enough salt to inhibit the entire system because there is no difference in OD600 measurement at 4hr timepoint between 10mM and the 0mM concentrations. This test needs to be repeated with higher concentrations of Mg<SUP>2+</SUP> however this data suggests that the mutagenesis was successful and S7 is active and killing the cells at approximately 4hr which does not necessarily reflect upon the activity of S7 but also reflects upon the response time of the mgtA system.</P><H2>Week 23 (October 9 to October 12)</H2><P> The molybdate assay was not successful as the M9 media was contaminated. The assay was deemed not priority. Constructions of constitutive promoter and riboswitch were attempted with GFP (LVA tag) along with other constructions to complete the circuits shown in the regulation page. The constructs were transformed. 
</P><P>
With respect to the rhamnose expression system, our team repeated constructions of the following: Prha-B0034-GFP; and R0040-B0034-rhaS. However, given the trouble we had constructing these circuits, I decided to use gel extraction to isolate purified insert and vector following cutting with appropriate restriction enzymes. As well, I ensured that inserts were in an exact three to one ratio to their respective vectors. Finally, 1 microlitre of a 100mM solution of ATP were added to all ligation reactions since we suspected ATP degradation in our ligase buffer. The ligation reactions were carried out at room temperature overnight, and were frozen for the weekend of Regionals.
</P><H2> October 12 to October 14: iGEM Regionals!</H2><H2>Week 24 (October 15 to October 19)</H2><P> With MOCO, the transformants which grew were amplified using colony PCR. When the samples were run on a gel the bands did not line up properly. Sequencing results came back and TetR promoter with Moco riboswitch was sequenced verified as well as the moco riboswitch with kill gene S7. Further constructions to complete the circuit were attempted and we ended up with 2 sequenced verified constructs of TetR promoter with moco riboswitch and S7 and GFP respectively. 
</P><P>
With the rhamnose system, the ATP spiked ligation reactions of Prha—B0034--GFP and R0040—B0034--rhaS were transformed into E. coli—a number of colonies were produced which were subsequently analyzed with PCR and digestion of isolated plasmids. There were colonies in both construction that appeared to be of appropriate size.
</P><P>
We are hoping to complete the final rhamnose characterization circuit (Prha—B0034--GFP--R0040—B0034—rhaS) prior to finals; to this end, I designed primers for the <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/Gibson_Assembly">Gibson assembly</A> of this circuit from plasmid stocks of pRha, B0034—GFP, B0034—rhaS, and pSB1C3.
</P><H2>Week 25 (October 22 to October 26)</H2><P> The moco circuits were further constructed with a LacI promoter, a ribosome binding site, and the moaABCDE operon. This was sent to the registry. Characterization of these circuits are in progress. </P><P>
With the rhamnose system, sequencing results showed that we successfully constructed Prha—B0034—GFP. There was a mistake in the order so that R0040—B0034--rhaS was not send for sequencing. We will attempt to assemble the final kill switch construct using Biobrick Standard methods. Additionally, we will conduct an assay of Prha—B0034—GFP to see how rhamnose, glucose, and lack of these sugars might affect its fluorescence. The protocol can be seen <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/Prha_Characterization">here</A>. We successfully show induction of this promoter with rhamnose and repression with glucose. See the <A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">results</A> on our kill switch page.
</P><P>
Also with rhamnose, we attempted to use Pfu to PCR out four elements for the Gibson assembly of the final rhamnose construct. The reactions were conducted on a 50 degree Celsius to 64 degree Celsius temperature gradient. Two of the PCR components were successfully amplified (B0034—GFP and pSB1C3); the other two elements require further optimization (B0034-RhaS and Prha).
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Notebook/Killswitch">http://2012.igem.org/Team:Calgary/Notebook/Killswitch</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Killswitch" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Killswitch" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Killswitch&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Killswitch&amp;oldid=297385" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>