<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Desulfurization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Desulfurization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Desulfurization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Desulfurization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Desulfurization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Desulfurization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Desulfurization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2012.igem.org/Team:Calgary" id="homelink">Home</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Team" id="teamlink">Team</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Team">Members</A></LI><LI><A class="dropheader" href="http://ung.igem.org/Team.cgi?id=902">Official Profile</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/AbouttheUniversity">The University</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2012.igem.org/Team:Calgary/ContactUs">Contact Us</A></LI></UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project" id="projectlink">Project</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Accomplish">Accomplishments</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Post-Regionals">Post-Regionals</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Collaborations">Initiative</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Interviews">Interviews</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch">Killswitch</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/Regulation">Regulation</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/HumanPractices/Killswitch/KillGenes">Kill Genes</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Safety">Safety</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/FRED">FRED</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Detecting">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Reporting">Electroreporting</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/FRED/Prototype">Device Prototype</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Project/OSCAR">OSCAR</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/CatecholDegradation">Decatecholization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/FluxAnalysis">Flux Analysis</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Upgrading">Upgrading</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Desulfurization">Desulfurization</A></LI><LI><A style="margin:0 -1em 0 1em;" href="https://2012.igem.org/Team:Calgary/Project/OSCAR/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Synergy">Synergy</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/References">References</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Project/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Parts" id="partslink">Parts</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Outreach" id="outreachlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/VideoGame">LAB ESCAPE!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Spark">Spark Science</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/MindsInMotion">Minds in Motion</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/Blog">Blog Posts</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Outreach/TEDxCalgary">TEDxCalgary</A></LI><LI><A href="http://twitter.com/iGEMCalgary" target="_blank">Follow Us!</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Sponsors" id="sponsorslink">Sponsors</A></LI><LI><A href="https://2012.igem.org" target="_blank">iGEM</A></LI></DIV><DIV id="sidebar"><DIV id="list"><H2>Notebook</H2><UL><LI><A class="drop" href="https://2012.igem.org/Team:Calgary/Notebook">Journal</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Killswitch">Killswitch</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/PromoterScreen">Toxin Sensing</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Electrochem">Electrochemistry</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Biosensor">Biosensor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Decarboxylation">Decarboxylation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Desulfurization">Desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Denitrogenation">Denitrogenation</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/CatecholDegradation">Catechol</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Bioreactor">Bioreactor</A></LI><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/FluxAnalysis">Flux Analysis</A></LI></UL><LI><A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols">Protocol Manual</A></LI></DIV></DIV><DIV id="bodycontainer"><DIV id="jsnotice"><P>Hello! iGEM Calgary's wiki functions best with Javascript enabled, especially for mobile devices. We recommend that you enable Javascript on your device for the best wiki-viewing experience. Thanks!</P></DIV><H1 id="pagetitle">Desulfurization Journal</H1><H2>Week 1 (May 1-4)</H2><P>During this week, literature search was performed.</P><H2>Week 2 (May 7-11)</H2><P>Along with the rest of the team, this week was dedicated to familiarizing ourselves on the protocols that will be utilized during this years project; specifically the polymerase chain reaction, gel verification, preparation of overnight cultures, as well as developing a procedural flowchart to transform competent cells with registry biobricks. With regards to our sub-group specific goals, we reviewed the current available literature around various industrial and laboratory approaches to desulfurization of organic groups, especially in the petroleum industry. This included a comparison of non-biological processes such as conventional hydrodesulfurization, which is currently employed in petroleum product refinery stages, and how a biological approach would supplement and perhaps even offer several advantages over these methods. Current limitations to biological desulfurization, however, include such factors as biocatalyst stability, enzyme specificity and desulfurization rate. We also identified the enzyme desulfinase (DszB) as being one of the bottlenecks in the desulfurization 4S pathway. Overall, our goals moving forward involve determining the specific pathways involved in the desulfurization process as well as the reaction conditions we would want to employ, and identifying specific model compounds in addition to dibenzothiophene (DBT) that we could use to test the effectivity of our biosystem in order to determine its functionality in the conversion of naphthenic acids to economically valuable hydrocarbons.</P><H2>Week 3 (May 14-18)</H2><P> Building on the previous week's literature review, the 4S pathway was recognized as the preferred biological mechanism that we would explore in devising a desulfurization biosystem. Of specific interest is the dsz operon consisting of the genes for <I> dszA, dszB, </I>and<I> dszC</I> which selectively and non-destructively remove the sulfur from the hydrocarbon structure, and therefore preserves the carbon skeleton. In addition to these, another dsz gene exists.<I> dszD</I>, which codes for a FMN:NADH reductase, is an essential component of the pathway, but not part of the operon. Instead, it is on the genome. The enzyme produced by this gene is required to regenerate the FMNH2 consumed by the reactions carried out by DszA and DszC. <I>Rhodococcus erythropolis</I> IGTS8 is the most studied model organism in investigations of the 4S pathway, and has been shown in many different research endeavors to be capable of converting DBT to 2-HBP.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption">Figure 1: The 4S Desulfurization Pathway, showing the desulfurization of the model compound DBT by DszA, DszB, DszC, and DszD. Four enzymes are involved in the 4S pathway, 3 of which are directly involved in the conversion of DBT to 2-HBP. Dibenzothiophene monooxygenase (DszC) is responsible for the first two steps of the pathway, converting DBT to DBT-sulfoxide and finally to DBT-sulfone (DBTO2) through the addition of oxygen to the sulfur atom. DBT-sulfone monooxygenase (DszA) then carries out the next step in the pathway, producing 2-hydroxybiphenyl-2-sulfinic acid (HBPS) through addition of a final oxygen to the heteroatom. This causes cleavage of the chemical bonds at the heteroatom, breaking the ring and converting the compound from a 3-ring structure to a 2-ring structure. HBPS is then converted to the final product of the 4S pathway by HBPS desulfinase (DszB), producing 2-hydroxybiphenyl. At this point, the sulfur has been released from the hydrocarbon in the form of sulfite.</DIV></DIV></DIV></DIV><P>
 An alternative to the DszD gene is HpaC, an oxidoreductase encoded in the <I>E. coli</I> W genome. This enzyme has been shown to increase the rate of desulfurization. Following this, other protocols added to our growing lab methods 'toolkit' were a restriction digest protocol, PCR purification, and finally, DNA construction digest. Aims moving forward include obtaining strains of the <I> R. erythropolis </I>, while also executing a timeline devised to biobrick, test, and incorporate the genes necessary in the above processes in a biobrick circuit.</P><H2>Week 4 (May 22-25)</H2><P>This week was kicked off with a project development meeting with Emily and David, and we devised a protocol for biobricking the <I> hpaC </I> gene. Additionally, methods to place the genes coding for the 4 enzymes, DszA,B,C and HpaC into a single construct were explored. Within the lab, the PCR performed on the resuspended pUC18-hpaC was not successful initially.  
Furthermore, we ordered the substrates/compounds that we intend to use for desulfurization tests. Once the substrates and the <I>Rhodococcus</I> strain arrive we are going to test how effectively the bacteria can desulfurize different sulphur-containing compounds that resemble naphthenic acids. Finally, we came across a paper where a team had developed an improved efficiency DszB through site-directed mutagenesis in 2007. This was through a point mutation to the gene, converting a tyrosine at position 63 to a phenylalanine residue. A member of this team was contacted to request the plasmid that contains the mutated gene. The conversion step carried out by DszB is the major bottleneck in the 4S pathway and if a strain or sample containing this mutation was obtained, it would significantly bolster our later testing efforts on DBT, as well as other compounds such as thiophane.
</P><H2>Week 5 (May 28 - June 1)</H2><P>Since we wanted to make sure we would not run out of pUC18(plasmid containing the hpaC gene), we transformed some E.coli cells with it. We grew them on plates containing ampicillin (A), kanamycin (K), tetracycline (T) and chloramphenicol (C) antibiotics and they only grew on A. Therefore pUC18 has A resistance. We did a three sets of PCR with primers designed against <I>hpaC</I>, one using 1/10 dilution of pUC18, the other using 1/100 dilution of pUC18 and one with the colonies we had just obtained by transforming the <I>E.coli</I> cells. </P><P>The PCR worked and we saw bands of the same size for all three sets of PCR. (Unfortunately, the picture we saved is not a good one since some of the bands faded away under UV due to prolonged exposure. Following this, PCR purification was performed to obtain the pure <I>hpaC</I> with biobrick prefix and suffix attatched to gene, which would allow us to insert the sequence into a biobrick standard backbone. 3 sets of digestion, ligation, and transformation (using pairs of X&amp;P enzymes, E&amp;S enzymes and E&amp;P enzymes) were carried out in order to insert the <I>hpaC </I> gene into the <A href=" http://partsregistry.org/Part:pSB1C3">pSB1C3</A>
 vector. All the sets grew successfully.
Following the above successes with<I> hpaC</I>, the arrival of our <I>Rhodococcus</I> strain afforded us the opportunity to begin investigation of the Dsz operon using the primers currently in our possession. This strain is an environmental isolate that has been shown to be an active desulfurizer. The gram-positive nature of the strain also dictated we explore various lysing strategies before the genes encoding the Dsz enzymes could be amplified for further purification and biobrick construction steps. PCR was carried out using <I>dszA</I> primers on three different treatments {microwave, lysate buffer, and a control} which yielded banding pattern around 1200 base pairs for the lysate treatment (2%SDS and 10% tritonX-100, plus heat for 5mins at 98C).
</P><H2>Week 6 (June 4 - June 8)</H2><P> In order to confirm the <I>hpaC</I> biobrick construction, two sets of colony PCR were performed, choosing white colonies from the 3 plates we grew last week (white colonies indicate a loss of the RFP generator in the <A href=" http://partsregistry.org/Part:pSB1C3">pSB1C3</A>
 backbone, and therefore allow for weeding out of the colonies which are simply the original plasmid vector). These reactions were carried out both with <I>hpaC</I> primers and with standard biobrick primers designed against the plasmid backbone. After running them on the gel we saw equal bands for the PCR reactions performed using <I>hpaC</I> primers (However, a PCR using biobrick primers was performed later and the same results were obtained). Colonies 1(-) and 5(-) were used to make overnight cultures, which were then miniprepped the following day to obtain the plasmid DNA of the putative <I>hpaC</I> biobrick. Digestions were performed on the miniprep products using EcoRI and PstI to look for part size as further verification for the genes presence in the plasmid. The results were good and two bands were observed on each column (one for vector and the other for <I>hpaC</I>)). <I>hpaC</I> was sent in for sequencing. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 2: HpaC verification cPCR. HpaC gene was inserted into the pSB1C33 vector and E. coli cells were transformed. In order to confirm that pSB1C3 contains the hpaC gene, two sets of colony PCR's were conducted. One with biobrick primers, and the other with hpaC primers. Bands indicate successful amplification at the approximate size of the hpaC gene (517 bp)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:420px;"><DIV class="thumbcaption">Figure 3: HpaC confirmation digest. Plasmid was miniprepped and digested for 2h with EcoRI and SpeI before being run on a 1% gel for 1h and 120V. Bands between 500 and 700 bp indicate the hpaC part is present as an insert. Bands of about 2000bp show the size of the psb1c3 vector.</DIV></DIV></DIV></DIV><P> PCR reagents were prepared to re-test/confirm previous results of <I>dszA</I> amplification following two different lysing treatments (microwave + lysate buffer). This time, all three genes were amplified and gel verification showed clear banding patterns around 500bp range for all three genes for the microwave treatment. Remaining PCR products were run on a gel and extracted for further purification steps; however, presence of any genetic material were not confirmed through nanodropping which raised concerns about the composition of the purified products, the success of the initial amplification step, or perhaps even the lysis treatment. Further experimentation will have to be carried out to troubleshoot.</P><H2>Week 7 (June 11 - June 15)</H2><P> This week, we focused on amplifying <I>dsz</I> genes from our <I>Rhodococcus</I> strain for construction into biobricks. We also wanted to purify the <A href=" http://partsregistry.org/wiki/index.php?title=Part:BBa_K902057"> BBa_K902057 <I>hpaC</I></A> and pUC18-<I>hpaC</I> plasmids to replenish our current stocks. For the <I>dsz</I> aspect, we were able to successfully grow extra plates of <I>Rhodococcus</I> strain which was used to inoculate PCR tubes. The PCR did not go well, with significant streaking and false positives with similar banding pattern to previous gels run in the previous week. A final gel verification of a random sample of a tube of PCR products from dszA,B,C respectively and two negative control treatments involving master mix only and the lysed cells only illustrated the lack of discrepancy between the supposed successful amplification and the lysed cells (with lysate buffer) alone. Because of this we decided to take a different approach involving plasmid isolation carried out before PCR, rather than applying the PCR reagents directly to a lysed culture sample.</P><P><A href=" http://partsregistry.org/wiki/index.php?title=Part:BBa_K902057"><I>hpaC</I></A> verification through sequencing was successful, confirming the construction of our first biobrick. Subsequently, O/N cultures of the plasmid containing cultures were prepared and stored in glycerol at -80C. Furthermore, verification of catalase gene part (<A href=" http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A>), which was sent as a culture stab from the parts registry was initiated. Our newly identified biobricked-hpaC was used as a positive control, but the banding pattern was not very conclusive. </P><H2>Week 8 (June 18 - June 22)</H2><P>PCR was reattempted on <I>Rhodoccocus</I> that was lysed using two different dilutions of the lysate buffer, but the gel verification confirmed the previous failure in using this approach. An alternative that involved preparation of an overnight culture of the <I>Rhodococcus</I> cells followed by a plasmid purification was followed. The plasmid purification eventually yielded plasmid samples with concentrations of 98.6ng/μL to 182.7ng/μL (4 samples obtained overall). Additionally, the catalase biobrick was used to transform some stock competent cells, and samples of some colonies were subsequently PCR'ed. Although, the gel verification showed some potential contamination, and the required banding patterns at around 2200bp was not obtained. </P><H2>Week 9 (June 25 - June 29)</H2><P> PCR was attempted to amplify the genes of the <I>dsz</I> operon utilising an adapted PCR protocol with purified <I>Taq</I> polymerase that had been isolated from the host organism. Eventually, some banding pattern was obtained between 1200 and 1500 base pairs when a gradient thermocycler was used with melting temperatures ranging betweeen 55°C to 65°C. This was assumed to be indicative of successful amplification of <I>dszB</I>; however, further purification and gel verification results were inconclusive and no yield was obtained when placed tested using a nanodrop machine. </P><H2>Week 10 (July 2-July 6)</H2><P>Top 10 E.coli cells were transformed with <A href="http://partsregistry.org/Part:BBa_R0011">BBa_R0011</A>
 (IPTG inducible promoter in psb1C3 backbone), and resulting colonies were tested using cPCR. 
Colony PCR was performed on cells containing the catalase biobrick. Catalase is 2217bp long but since biobrick primers add about 200bp, bands of 2400 bp were expected if the part was present in the biobrick. These bands were observed, indicating that the <A href=" http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> gene was most likely present.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 4: Colony PCR on potential colonies containing katG-LAA. Biobrick primers were used. Bands at 2200 indicate successful amplification of the catalase part within the biobrick backbone. Smaller bands are indicative of primer degradation and are nonspecific amplification. +C indicates the positive control, and RFP generator, which should amplify at around 1000 bp.</DIV></DIV></DIV></DIV><P>

PCR using Phusion high fidelity polymerase was carried out on <I>dszA</I>, <I>dszB</I>, and <I>dszC</I> in a gradient thermocycler. Amplification of non-specific bands was present for <I>dszA</I> and <I>dszB</I>, however strong banding for the desired size of the gene was observed for both (around 1500 for <I>dszA</I>, 1100 for <I>dszB</I></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 5:DszA and DszB PCR from  <I>Rhodococcus</I>. DszA amplicon is around 1300 bp, and is observed to run higher on the gel. DszB amplicon is expected to be 1098 bp, which is observed in addition to multiple nonspecific banding. +C indicates the positive control, and RFP generator, which when PCRed with biobrick primers should be around 1000 bp. As this control is running high as well, it is believed that both amplicons for DszA and DszB have been obtained. No contamination is observed in the NTC (no template control).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 6:DszC PCR from  <I>Rhodococcus</I>. The numbers above the figure show the colony number. DszC amplicon is around 1450 bp, and is observed to run at almost the right size on the gel. No contamination is observed in the NTC (no template control).</DIV></DIV></DIV></DIV><P>
 Examining the sequences of the <I>dszABC</I> genes led to the discovery that all 4 had multiple illegal enzyme cut-sites in them that we have to eliminate before biobrick composite part construction can occur. <I>dszA</I> has four PstI cut sites, <I>dszB</I> has a PstI and a NotI and <I>dszC</I> has a PstI cut site. The Stratagene QuickChange mutagenesis procedure is going to be used to eliminate illegal cut sites with the only alteration being that Kapa HiFi polymerase would be used during the process. Primers needed for the mutagenesis were designed based on the procedure mentioned above. </P><H2>Week 11 (July  9-July 13)</H2><P> Following successful amplification of the <I>dsz</I> operon genes in the previous week, the genes were constructed into the <A href="http://partsregistry.org/Part:pSB1C3"> pSB1C3</A> vector. Colony PCR verifications were observed to be positive. Furthermore, the insertion of part <A href="http://partsregistry.org/Part:BBa_J13002"> BBa_J13002, (P<SUB>tetR</SUB>-RBS)</A>
  in front of the previously biobricked <I>hpaC</I> was attempted. Overnight cultures were also prepared using two colonies each for <A href="http://partsregistry.org/Part:BBa_J13002"><I>P<SUB>tetR</SUB>-RBS</I></A>  and <A href="http://partsregistry.org/Part:BBa_R0011"><I>P<SUB>lacI</SUB></I></A>  (an IPTG inducible promoter that we hope to build in front of <AHREF>an RBS site, BBa_B0034). These cultures were then miniprepped to yield the respective parts.</AHREF></P><P>Additionally, <A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> was built into a <A href="http://partsregistry.org/Part:PSB1C3"> PSB1C3</A> backbone. The construction and availability of all these parts will be critical in the construction of our overall circuit for biodesulfurization. Colonies which looked good on cPCR were used to prepare overnight cultures, and were miniprepped and sent in for sequencing verification the following day.
On the side, M9 minimal media was also prepared to carry out growth experimentation and overall desulfurization capability of <I>Rhodococcus</I> when exposed to DBT. The various growth treatments were M9 Media and glucose only, M9+glucose+DBT, M9+glucose+MgSO4+/-DBT, M9+glucose+MgCl2+/-DBT. 0.008g of FeCl2.4H2O was also added to each of the tubes. Samples were then inoculated with colonies of the <I>Rhodococcus</I>.</P><H2>Week 12 (July  16 -July 20)</H2><P>This week, while awaiting sequencing verification results which were required before we could begin the construction process, the desulfurization team initially aided in some of the tasks related to the other hydrocarbon groups. The success of the construction of <A href="http://partsregistry.org/Part:BBa_J13002"><I>P<SUB>tetR</SUB></I>-RBS</A>
with <A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> was also explored by using forward and reverse primers of <A href="http://partsregistry.org/Part:BBa_R0040">BBa_R0040</A> (the promoter component of the composite part BBa_J13002). However, the eventual gel verification was inconclusive and sequencing results finally indicated an unsuccessful ligation. Additionally, the minimal media M9 preparation had been contaminated in the previous effort so this process was repeated to create tubes of each of the growth condition treatments detailed previously, and two repeats, one with an extra filtration step and one without was used to prepare the cultures.</P><H2>Week 13 (July 23 - July 27)</H2><P> Mutagenic primers were redesigned after the initial ones were found to have premature stop codons. As part of the redesign process in constructing our overall gene circuits for desulfurization, a backbone switch of <A href="http://partsregistry.org/Part:BBa_R0011"> BBa_R0011</A> into a chloramphenicol (Chlor) resistant vector was necessary. The subsequent transformed products were plated on a Chlor plate and selected colonies were used to prepare O/N cultures, then minipreped before finally being digested with enzymes EcoRI and PstI. The resulting gel verification images were inconclusive as they did not show the required banding pattern around 50bp. Meanwhile, colony PCR was run on colonies transformed with <I>katG-LAA</I> constructed into a <A href="http://partsregistry.org/Part:PSB1C3"> PSB1C3</A> backbone, as well as the <A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> +<A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A> construct. <A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> was shown to have been successfully amplified, so overnight cultures were prepared and subsequently miniprepped. On the other hand, the construct was not successful so a third attempt was carried out. Colony PCR treatments that used either <A href="http://partsregistry.org/Part:BBa_R0011"> BBa_R0011</A> forward primers or <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
 primers were used and the overall constructs were made either on a chlor-resistant, or ampicillin-resistant vectors. Preliminary images of the gel verification appeared to have confirmed the construct, although sequencing verification will be the final indicator of overall success. </P><H2>Week 14 (July 30 - August 3)</H2><P>Sequencing results from the previous week's constructs were available confirming that we constructed KatGLAA in a chlor-resistant backbone. However, switching the plasmid backbone of <A href="http://partsregistry.org/Part:BBa_R0011"> BBa_R0011</A> to <A href="http://partsregistry.org/Part:PSB1C3"> PSB1C3</A> was not successful.  The construction of <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>+<A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> was finally sent in for sequencing.  Site-directed mutagenesis of the <I>dsz</I> operon was also initiated:  <I>dszA</I> has four PstI cut sites; <I>dszB</I> has a PstI and a NotI site; <I>dszC</I> has two PstI cut sites.  Site directed mutagenesis was started this week to change a single base pair in these genes in a way that eliminates the cut site but preserves the amino acid codons, so as to not mutate the protein coding sequence.  Ohshiro 2007 demonstrated that replacing the Tyr residue at position 63 of <I>dszB</I> gene with a Phe increases the activity of the enzyme.  Therefore we want to introduce the same mutation into our <I>dszB</I>.</P><P>
 For the first attempt at mutagenesis we chose to mutate the second PstI site in <I>dszC</I> (PstI2).  As a positive control for the procedure, we also performed the mutagenic PCR on a plasmid containing the β-galactosidase gene with a point mutation where the PCR would cause it to regain its function.  For both mutagenesis protocols we used the Kappa Hifi kit.  After confirming that the PCR worked by running some produce on a gel, the PCR products were DpnI digested, the purpose of which is to degrade the unmodified parental DNA (DpnI degrades methylated DNA only).  Control PCR products were plated on an ampicillin plate containing IPTG and X-gal. The colonies that grew on the control plates were blue indicating that the mutagenesis had worked for the β-galactosidase gene.  Minipreps of the O/N culture of <I>dszC</I> mutants were digested with PstI enzyme and the results indicated that the mutagenesis was successful.</P><P> 
Attempts to simultaneously perform all the mutations in <I>dszC</I> genes in one step using the Knight procedure failed (<A href="http://openwetware.org/wiki/Knight:Site-directed_mutagenesis/Multi_site">Knight Multi-site Mutagenesis Procedure</A>). What enables simultaneous mutations is that Taq ligase closes the gaps in PCR products after each cycle. In the protocol it instructs to use Taq ligase buffer only for the PCR/ligation protocol. We suspected that the reason this procedure did not work might be that the Kappa polymerase is not functional in Taq ligase buffer. Therefore we did some experiments on the controls in Taq ligase kit and kappa polymerase kit to find out which buffer that Kappa polymerase and Taq ligase both work best in. The result was that both enzymes work best in a buffer made of half Taq ligase buffer and half Kappa polymerase buffer. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 7: DszC PstI2 mutagenesis PCR with varying concentrations of template plasmid was performed. The gel shows the PCR products that were run on a gel. +C lanes show the PCR products of the control. The control was pWhitescript™ 4.5-kb control plasmid from Stratagene mutagenesis kit which was mutated with primers also from the kit. Bands indicate successful amplification of the plasmid.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 8: The control and the mutated plasmids were digested with PstI restriction enzyme. The control is dszC biobrick. Other lanes show the plasmids purified from the cells transformed with PstI2 mutagenesis PCR products followed by digestion. The control shows three bands since it has three PstI cut sites. The other lanes show two bands which indicates one of the cut sites has been eliminated through mutagenesis.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 9: The multisite mutagenesis using the Knight procedure was tried in different buffers to find out the buffer that Kappa works optimally at. All the PCRs were performed on the pWhitescript™ 4.5-kb control plasmid from Stratagene mutagenesis kit which contains the beta-galactosidase gene. Based on this gel, the optimal buffer is composed of 50% kappa buffer and 50% Taq ligase buffer.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 10: Using the control provided in the NEB Taq ligase buffer (BsteII digested lambda DNA), we tried to find if the Taq ligase enzyme can function in combinations of Kappa Hifi buffer and Taq ligase buffer. Lane two is the control which is only the digested lambda DNA. Lanes 1 and 3 show that some of the bands compared to control have been ligated together. Therefore, Taq ligase functions just as good in the buffer composed of 50% kappa hifi buffer and 50% Taq ligase as it would in its own buffer.</DIV></DIV></DIV></DIV><H2>Week 15 (August 6 - August 11)</H2><P>Sequencing results for <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A><A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> returned negative, so a 3-part ligation method was used to retry this construction. The following parts were ligated with the restriction enzymes indicated in brackets after each: <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>(EcoRI/SpeI) + <A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> (XbaI/PstI) + <A href=" http://partsregistry.org/Part:PSB1K3">BBa_PSB1K3</A> (EcoRI/PstI).  Also, the more conventional construction (only 1 insert) of <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>(SpeI/PstI) + <A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A> (XbaI/PstI) was reattempted. Furthermore, 3-way ligations were also attempted for <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
+<A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A>+<A href="http://partsregistry.org/Part:PSB1K3">BBa_PSB1K3</A>, and <A href="http://partsregistry.org/Part:BBa_R0011"> BBa_R0011</A>+<A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
+<A href="http://partsregistry.org/Part:PSB1C3"> PSB1C3</A>, as well as the two-way contruction of just <A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> after the <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>. After plating these transformations, colony PCRs were carried out and samples that gave an indication of being successful on the gels were used to prepare O/N cultures followed by miniprep.  With regards to the site-directed mutagenesis side of the experimentation, <I>dszA</I>-PstI1 (the first PstI cut site in <I>dszA</I>) ,<I> dszB</I>-PstI and <I>dszC</I>(PstI2 mutated)-PstI1 mutagenesis were performed following the procedure explained in the previous week. The gel below shows the successful result of digest confirmation (Fig. 8). 
Multisite mutagenesis (Knight method) was repeated using the modified buffer (half Taq ligase buffer and half Kappa buffer). However it was not successful again. We also tried doing multisite mutagenesis using Pfu Turbo polymerase and following the Knight procedure without any buffer modifications. No successful results were observed. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 11: Confirmation of site-directed mutagensis to eliminate PstI cut sites in: <I>dszA</I>PstI1 cut site, <I>dszB</I>PstI cut site and <I>dszC</I>PstI1 (PstI2 site mutated).  Note that in each case, the unmutated controls for each gene are to the left of each set of plasmids derived from mutagenic PCRs.  Plasmids from cloned <I>dsz</I> genes are smaller than the bands to the right of them (which have undergone mutagenesis), indicating that in the plasmids on the right, the PstI cut site was lost.</DIV></DIV></DIV></DIV><H2>Week 16 (August 12 - August 18)</H2><P>The progress in mutagenesis of <I>dsz</I> genes was continued from the previous week:  <I>dszB</I>(PstI mutated)-Y63F and <I>dszA</I>(PstI1 mutated)-PstI3 mutagenesis. The gel below shows the digest confirmation.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:710px;"><DIV class="thumbcaption">Figure 12: Digestion confirmation of biobricks of a) dszA (PstI1 mutated) PstI3 mutagenesis and b) dszB (PstI mutated) Y63F mutagenesis. Cutting dszA (PstI1 mutated) with PstI is expected to produce fragments of 2790, 333, 255, and 114bp. Cutting dszA (PstI1 and PstI3 mutated) is expected to produce fragments of 2730, 588, and 114bp. The primers for dszB-Y63F mutagenesis introduce an HpyAV cut site. pSB1C3 inteslf has two more HpyAV cut sites. Before the mutation bands of 2858 abd 310bp are expected, and after successful mutation bands of 1502, 1356, 310 are expected. Lane legend: 1- dszA (PstI1 mutated). 2-dszA (PstI1 mutated) digested with PstI. 3-dszA (PstI1 mutated) PstI3 mutagenesis c1 Digested with PstI. 4- dszA (PstI1 mutated) PstI3 mutagenesis c2 Digested with PstI. 5-dszA (PstI1 mutated) PstI3 mutagenesis c3 Digested with PstI. 6- dszA (PstI1 mutated) PstI3 mutagenesis c4 Digested with PstI. 7- Fermentas 1kb Plus Ladder. 8- Empty. 9- dszB (PstI mutated). 10-dszB (PstI mutated) digested with HpyAV. 11- dszB (PstI mutated) Y63F mutagenesis c1 digested with HpyAV. 12- dszB (PstI mutated) Y63F mutagenesis c2 digested with HpyAV. 13- dszB (PstI mutated) Y63F mutagenesis c3 digested with HpyAV. 14-dszB (PstI mutated) Y63F mutagenesis c4 digested with HpyAV.</DIV></DIV></DIV></DIV><P>We attempted a different approach to speed up the turnover time of the mutagenesis PCR. Briefly, after the PCR mutagenesis the PCR products were purified and then incubated with T4 polynucleotide kinase (PNK) and ligase.  After heat inactivating the ligase and T4 PNK, the products were DpnI digested. Subsequently another round of DNA purification was performed. However, the results were unsatisfactory after the digest confirmation.</P><P>
Sequencing results came back. <I>dszA</I> (PstI1 and PstI3 mutated) and <I>dszB</I>(PstI and Y63F mutated) were good. However <I>dszC</I> (PstI1 and PstI2 mutated) had an insertion next to the PstI1 cut site. Mutagenesis was repeated on the <I>dszC</I>(PstI2 mutated).
<I>dszB</I>(PstI and Y63F mutated)-NotI and <I>dszA</I>(PstI1 and PstI3 mutated)-PstI4 mutagenesis were also performed.</P><P> To investigate the desulfurisation capability of the <I>Rhodococcus</I> sp. from which we cloned the <I>dsz</I> operon, a desulfurization assay was prepared by inoculating different treatments of M9 media.  We also prepared some solutions that will be needed for analysis in the following week: a conditioning agent composed of 100ml of 95% ethanol, 50ml glycerol, 30ml of 12M HCl (aq) and 70g of NaCl(s) was prepared. The assay relies on the turbidity of a sample containing sulphate ions which are precipitated (hence the turbidometric nature of the assay) upon adding BaCl2(s), therefore if the <I>dsz</I> pathway is active, we expect a more turbid solution to form than in control samples. </P><H2>Week 17 (August 19 - August 25)</H2><P>This week, progress was made in determining the desulfurization activity of our <I>Rhodococcus</I> strain as measured by the sulfate release using a turbidometric assay. We encountered several challenges in our prescribed protocol as the concentrations that we used to prepare the standard curve may have been too dilute, or the composition of out conditioning agent may have been flawed. Additionally, steps were taken to determine the decomposition of DBT to 2-HBP through Gas Chromatograph-Mass Spectroscopy (GC-MS) analysis, but due to a preparation error, the DBT was added to a growth solution of M9 media prematurely and the autoclaving process decomposed the DBT releasing a yellow colouration into the solution. These two approaches in determining the desulfurization capability of the <I>dsz</I> operon will be further investigated. </P><P>Since the <I>dszC</I> second mutagenesis had proven to be unsuccessful last week, the <I>dszC</I>(PstI2 mutated)PstI1 mutagenesis was repeated. Also <I>dszA</I>(PstI1,3,4 mutated) PstI2 mutagenesis was performed. <I>dszA</I> and <I>dszC</I> were sent for sequencing on Wednesday. <I>dszB</I> was sent for sequencing on Friday. Sequencing results of <I>dszA</I> and <I>dszC</I> were back by Friday. <I>dszC</I> was successful. However, <I>dszA</I> contained an insertion next to the binding site of PstI4 cut sit, so the last two mutations must be redone. <I>dszB</I>(PstI and Y63F mutated)-NotI-mutagenesis was also repeated in case the result of the sequencing was not successful. These constructions were repeated. <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>-<A href="http://partsregistry.org/Part:BBa_K902052"><I>dszB</I></A>,  and <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
-<I>dszC</I> constructions were attempted, however they were not successful as indicated by colony PCR. Constructions of <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>/<I>hpaC</I> were carried out and also came back negative in sequencing, however <A href="http://partsregistry.org/Part:BBa_K902059">BBa_B0034
/<I>katG-LAA</I> (BBa_K902059)</A> was sequence confirmed.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 13: Another attempt to mutagenize dszC(PstI2 mutated)at PstI1 site. The gel shows the confirmation digestion. C is the control which is dszC(PstI2 mutated). Since the control has two PstI cut sites, two bands are observed on the gel. The mutagenesis has been successful since only one band is observed on the other lanes indicating that one of the cut sites has been eliminated during mutagenesis.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 14: DszA (PstI1, PstI2, PstI3 mutated)PstI4 mutagenesis digestion confirmation gel. All the lanes show the results of the plasmids being cut with PstI enzyme. C indicates the lane containing the control which is dszA(PstI1, PstI3 and PstI4 mutated). The other lanes are the digestions of the plasmids that are mutagenized, from different colonies. Control plasmid has two PstI cut sites and therefore the two bands observed on the gel were expected. The other lanes show only one band of about 3500bp which is about the right size for dszA in a psb1c3 vector. Therefore one of the cutsites has been eliminated during mutagenesis.</DIV></DIV></DIV></DIV><H2>Week 18 (August 26 - September 1)</H2><P><A href="http://partsregistry.org/Part:BBa_K902052"><I>dszB</I> (BBa_K902052)</A> sequencing results came back as successful. <I>dszA</I>(PstI1,3 mutated)-PstI2-mutagenesis was performed and sent for sequencing. Also <I>dszA</I>(PstI1,2,3 mutated)-PstI4-mutagenesis was performed, and this was also sent for sequencing. </P><P>Constructions of <A href="http://partsregistry.org/Part:BBa_K902053">P<SUB>tetR</SUB>-RBS/<I>dszB</I> (BBa_K902053)</A> and <A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034
/<I>dszC</I> (BBa_K902056)</A> were attempted, verification digested, and sent for sequencing. Sequencing results for these constructs came back as positive, along with successful mutagenesis of <A href="http://partsregistry.org/Part:BBa_K902050"><I>dszA</I> (BBa_K902050)</A>.</P><P>At this point, all of the <I>dsz</I> genes have been successfully made biobrick compatible, and <I>hpaC</I> has been biobricked. We have also successfully constructed <A href="http://partsregistry.org/Part:BBa_K902059">BBa_B0034 with <I>katG-LAA</I></A> to be used in the optimization circuit, as well as <A href="http://partsregistry.org/Part:BBa_K902053"><I>P<SUB>tetR</SUB>-RBS</I>/<I>dszB</I> (BBa_K902053) </A>and <A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034/<I>dszC</I> (BBa_K902056)</A>.
</P><P> Constructions of <A href="http://partsregistry.org/Part:BBa_J04500">P<SUB>lacI</SUB>-RBS</A>with <A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A>, <A href="http://partsregistry.org/Part:BBa_K902052"><I>dszB</I></A>, and <A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> were performed. As well, attempts to construct <A href="http://partsregistry.org/Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">RBS-<I>dszC</I></A> as well as <A href="http://partsregistry.org/Part:BBa_J13002">P<SUB>tetR</SUB>-RBS</A>/<A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A>, <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
/<A href="http://partsregistry.org/Part:BBa_K902050"><I>dszA</I></A>, and <A href="http://partsregistry.org/Part:BBa_J13002">BBa_J13002</A>/<A href="http://partsregistry.org/Part:BBa_K137068"><I>katG-LAA</I></A> were also carried out. These parts are intended as construction intermediates towards building the final systems, as well as providing a way of testing the genes functionality (namely, to test HpaC for oxidoreductase activity and to test if over-expression of KatG in the cell will increase its ability to survive H<SUB>2</SUB>O<SUB>2</SUB> stress). Transformations of all these constructions were carried out at the end of the week.</P><H2>Week 19 (September 2- September 8)</H2><P>Confirmation digests on colonies of the previous constructions that gave bands of the expected size with cPCR were performed. Positive results were found for colonies of <A href="http://partsregistry.org/Part:BBa_J04500"><I>P<SUB>lacI</SUB>-RBS</I></A>/<A href="http://partsregistry.org/Part:BBa_K902057"><I>hpaC</I></A>, <A href="http://partsregistry.org/Part:BBa_J04500"><I>P<SUB>lacI</SUB>-RBS</I></A>/<A href="http://partsregistry.org/Part:BBa_K902052"><I>dszB</I></A>, and <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>
/<A href="http://partsregistry.org/Part:BBa_K902050"><I>dszA</I></A>. Sequencing was sent, and results indicated that the constructions of <A href="http://partsregistry.org/Part:BBa_K902058"><I>P<SUB>lacI</SUB>-RBS/hpaC</I> (BBa_K902058)</A> were successful, meaning that after many months of trying we FINALLY have a promoter in front of the <I>hpaC</I> gene and can proceed to test the parts functionality. Attempts to construct <I>hpaC</I> with the (<A href="http://partsregistry.org/Part:BBa_J13002">evil TetR promotor, BBa_J13002</A>) were abandoned, as it was believed that this construction was failing due to toxicity of over-expressing the protein, and it was determined that this part was not necessary after all. <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB>-RBS/dszB</I>, (BBa_K902054)</A> also worked, though this was less exciting. <A href="http://partsregistry.org/Part:BBa_K902051">BBa_B0034
/<I>dszA</I></A> came back as a bad read despite looking very good on the confirmation digest gel, so this part will be resent for sequencing. Constructions of <A href="http://partsregistry.org/Part:BBa_K902060"><I>P<SUB>lacI</SUB></I>-<I>KatG</I></A> were reattempted, and confirmation digests for this part looked good, and so samples were sent for sequencing.</P><H2>Week 20 (September 9- September 15)</H2><P>Construction attempts on <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902058"><I>P<SUB>lacI</SUB></I>-<I>hpaC</I></A>/<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902059">BBa_B0034-<I>katG-LAA</I></A> were performed. Colonies grew for the constructions, however further confirmation results were dissapointing (only 2 clones of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A> appeared to have been successful). These clones were sent for sequencing, and constructions were reattempted.</P><P> However, when sequencing came back, somehow reads indicated that these clones were in fact a gene from the Denitrogenation project (which is 990bp and a completely different band then what we saw on the gel). We believe, somewhere, something has gone very wrong- further investigation into this will be carried out. In the meantime, the above constructions were reattempted, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902051">BBa_B0034/<I>dszA</I></A> was re-prepped in case a contaminant in the plasmid stock was to blame for the bad reads found in this batch of sequencing as well as the last. In addition, plasmid switches of multiple sequence confirmed parts into a <A href=" http://partsregistry.org/Part:pSB1C3"> pSB1C3</A> backbone were carried out. </P><H2>Week 21 (September 16- September 22)</H2><P>Colonies for the transforms of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902058"><I>P<SUB>lacI</SUB></I>-<I>hpaC</I></A>/<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902059">BBa_B0034-<I>katG-LAA</I></A> have been few and far between, and cPCR results are always discouraging. Sequencing results for other sections of the project have once again come back very confusing, and further research continues into the source of this madness. </P><P> The sulfur compound degradation assay was set up to test the desulfurization rate of the original <I>Rhodococcus baikonurensis </I> (refer to the protocol page).</P><H2>Week 22 (September 23- September 29)</H2><P>Attempts to construct <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, , and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902058"><I>P<SUB>lacI</SUB></I>-<I>hpaC</I></A>/<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902059">BBa_B0034-<I>katG-LAA</I></A> continue. In the meantime, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902060"><I>P<SUB>lacI</SUB></I>-<I>KatG</I></A> was tested for functionality. In order to do this, cultures of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902060"><I>P<SUB>lacI</SUB></I>-<I>KatG</I></A> were grown up overnight in LB media. A strain carrying <A href="http://partsregistry.org/Part:BBa_J04500">BBa_J04500</A> only was used as a negative control. The following morning, 20 µL of each culture was inoculated into 3 mL of LB with various concentrations of hydrogen peroxide; 0 mM, 1 mM, 5 mM, and 10 mM. These cultures were then allowed to grow overnight, and culture turbidity was observed. It was found that the negative control exhibited no growth after 12h at 1 mM peroxide, however cultures with induced expression of catalase were turbid after 12 h of growth at this concentration (Fig. 10). This demonstrated the ability of the catalase to protect the cells from excessive peroxide concentrations.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 15: Catalase Assay. Overnight cultures of J04500 and J04500-<I>KatGLAA</I> were innoculated into 0 mM, 1 mM, 5 mM, and 10 mM peroxide. Cultures were grown overnight and turbitity was observed.</DIV></DIV></DIV></DIV><P>In addition to this, activity of <A href="http://partsregistry.org/Part:BBa_K902058"><I>P<SUB>lacI</SUB>-hpaC</I></A> was tested. In order to do this, cultures of <A href="http://partsregistry.org/Part:BBa_K902058"><I>P<SUB>lacI</SUB>-hpaC</I></A> and <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB>-dszB</I></A> were grown up overnight in LB with appropriate antibiotics. Following this, protein expression was induced with IPTG, after which the assay was carried out as described in the following figure and on the protocols page.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 16: HpaC Assay. Cultures of P<SUB>lacI</SUB>-hpaC and P<SUB>lacI</SUB>-dszB were grown up overnight in LB with appropriate antibiotics. The following morning, cells were subcultured 1/4 into LB with 200 µM IPTG and allowed to grow for 2h in order to induce protein expression. 2 mL samples of cells were then transferred to 2 mL tubes, washed twice in 50 mM Tris-HCl (pH 7.5) and resuspended in this buffer. Samples were then subjected to 5 freeze-thaw cycles in order to lyse cells. After spinning down samples, 1 mL of cell lysate was transferred to a cuvette, and a spectrophotometer was blanked at 340 nm (maximal absorbance of NADH) with this sample. 140 µM NADH and 20 µM FMN was then added, the cuvette was quickly inverted, and readings were taken at 340 nm. P<SUB>lacI</SUB>-dszB was used as a control to measure native amounts of oxidoreductase activity, whereas the P<SUB>lacI</SUB>-hpaC cultures were used to measure activity when HpaC was expressed. The control was just Tris-HCl buffer with the NADH and FMN compounds added. Decrease in absorbance at 340 nm corresponds to the loss of NADH as it is converted to NAD+.</DIV></DIV></DIV></DIV><P>When the assay was run, it was found that NADH does not convert readily to NAD+ on its own. When cell lysate containing the naturally expressed amounts of oxidoreductase was added, a decrease in absorbance could quickly be observed as the NADH was converted to NAD+. When cultures over-expressing HpaC were tested, the absorbance levels were found to start much lower than the control. We believe that this is because with the amount of cell lysate tested, when the HpaC protein is overexpressed the NADH is consumed almost immediately and therefore the data reflecting the drop in absorbance is missed. Further tests will use differing amounts of cell lysate in order to try to capture data that shows the drop in absorbance for HpaC cultures.</P><H2>Week 23 (September 30-October 3): Wiki-Freeze PANIC!!</H2><P>The GCMS results of the sulfur assay were received. DBT desulfurization was successful (figures below). Some compounds were not detected by GCMS since they were too polar. However, 2 additional compounds showed degradation in addition to DBT, indicating that the pathway has a wider substrate specificity!</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:722px;"><DIV class="thumbcaption">Figure 17:  <I>Rhodococcus</I> cells were grown in a modified M9 media containing 0.125mM DBT with no sulfur containing compounds (refer to desulfurization assay protocol in the protocols section for details). Samples were taken out at different time points and were run through GCMS to detect the amount of DBT. The control only contained modified M9 but no bacteria, run through the GCMS after 6 days of being in the incubator to account for abiotic degradation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:399px;"><DIV class="thumbcaption">Figure 18: The peak in this mass spectra demonstrates presence of DBT based on its molecular weight of 184 g/mol. This peak is based on the average of our samples at retention time of 13.9 minute (refer to previous graph).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 19: <I>Rhodococcus</I> cells were grown in a modified M9 media containing 0.125mM of the indicated compound (<B>A:</B> dibenzothiophene, <B>B:</B> tetrahydro-4h-thiopyran-4-one, and <B>C</B>: benzo[b]thiophene-2-carboxyaldehyde)  with no other sulfur containing compounds present in the media (refer to desulfurization assay protocol for details). Samples were taken out at different time points and were run through GCMS to detect the amount of compound remaining. Samples were normalized to a control containing modified M9 but no bacteria, run through the GCMS at the last time point to account for abiotic breakdown. Degradation is seen for DBT (the model studied compound) as well as other sulfur containing compounds resembling naphthenic acids, indicating that the pathway may have wider substrate specificity than previously thought.</DIV></DIV></DIV></DIV><P>The HpaC assay was repeated using 100 µL of cell lysate instead of 1 mL of 2x concentrated cell lysate, following the previously used protocol. With this smaller amount, a clear sharp decrease in the absorbance of NADH can be observed, indicating a very fast conversion by HpaC, and further confirming that the part was functional.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Figure 20: HpaC Assay. Cultures of P<SUB>lacI</SUB>-hpaC and P<SUB>lacI</SUB>-dszB were grown up overnight in LB with appropriate antibiotics. The following morning, cells were subcultured 1/4 into LB with 200 µM IPTG and allowed to grow for 2h in order to induce protein expression. 1 mL samples of cells were then transferred to 2 mL tubes, washed twice in 50 mM Tris-HCl (pH 7.5) and resuspended in this buffer. Samples were then subjected to 5 freeze-thaw cycles in order to lyse cells. After spinning down samples, 100 µL of cell lysate was transferred to a cuvette, and a spectrophotometer was blanked at 340 nm (maximal absorbance of NADH) with this sample. 140 µM NADH and 20 µM FMN was then added, the cuvette was quickly inverted, and readings were taken at 340 nm. P<SUB>lacI</SUB>-dszB was used as a control to measure native amounts of oxidoreductase activity, whereas the P<SUB>lacI</SUB>-hpaC cultures were used to measure activity when HpaC was expressed. The control was just Tris-HCl buffer with the NADH and FMN compounds added. Decrease in absorbance at 340 nm corresponds to the loss of NADH as it is converted to NAD+.</DIV></DIV></DIV></DIV><H2>Week 24 (October 4- October 11)</H2><P>This week, before the Americas West Regional Competition, attempts to construct the previously mentioned parts continued. Unfortunately, they were unsuccessful.

</P><H2>Week 25 (October 16- October 20)</H2><P>Traditional attempts to build the sulfur operon and its sub-parts were repeated, however attempts failed once more. During this week, it was decided that due to time consstraints, alternative approaches to construct the operon would be undertaken. Due to this, splice-overlap-extention (SOE) PCR primers (containing scar-sites where appropriate) were designed according to <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/soe">protocol</A> and ordered.</P><P>


Primer: 1a (BBK)-J13/DszB F - 5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGTCCCTATCAGTGATAGAGATTGACATCCC-3'

Primer: 1b (BBK)-J04/DszB F - 5'-GTTTCTTCGAATTCGCGGCCGCTTCTAGCAATACGCAAACCGCCTCTCCCC-3'

Primer: 2 Jxx/B-(RBS/C) R - 5'-GGTGACAGTGTCATCTAGTATTTCTCCTCTTTCTAGTACTATCGGTGGCGATTGAGGCTGTTG-3'

Primer: 3 (Jxx/B)-RBS/C F - 5'-CAACAGCCTCAATCGCCACCGATAGTACTAGAAAGAGGAGAAATACTAGATGACACTGTCACC-3'

Primer: 4 RBC/C-(RBS/A) R - 5'-GCCAGATGCATTTGTCGTTGTTGAGTCATCTAGTATTTCTCCTCTTTCTAGTATCAGGAGGTGAAGCCGGGAATCG-3'

Primer: 5 (RBS/C)-RBS/A F - 5'-CGATTCCCGGCTTCACCTCCTGATACTAGAAAGAGGAGAAATACTAGATGACTCAACAACGACAAATGCATCTGGC-3'

Primer: 6 RBS/A-(J04/HpaC) R - 5'-GGGGAGAGGCGGTTTGCGTATTGCTAGTATCATGAAGGTTGTCCTTGCAGTTGTGG-3'

Primer: 7 RBS/A-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCATGAAGGTTGTCCTTGCAGTTGTGG-3'

Primer: 8 (BBK)-J04/HpaC F - 5'-CGAATTCGCGGCCGCTTCTAGCAATACGCAAACCGCCTCTCCCC-3'

Primer: 9 (RBS/A)-J04/HpaC F - 5'-CCACAACTGCAAGGACAACCTTCATGATACTAGCAATACGCAAACCGCCTCTCCCC-3'

Primer: 10 J04/HpaC-(RBS/Kat) R - 5'-CTGACGTGCTCATCTAGTATTTCTCCTCTTTCTAGTATTAAATCGCAGCTTCCATTTCCAGCATCAC-3'

Primer: 11 J04/HpaC-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTAAATCGCAGCTTCCATTTCCAGCATCAC-3'

Primer: 12 (J04/HpaC)-RBS/Kat F - 5'-GTGATGCTGGAAATGGAAGCTGCGATTTAATACTAGAAAGAGGAGAAATACTAGATGAGCACGTCAG-3'

Primer: 13 RBS/Kat-(BBK) R - 5'-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTATTAAGCAGCCAGAGCGTAGTTTTCGTC-3'
</P><P>
Colony PCR was done on potential colonies of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB></I>-<I>dszB</I></A>/<A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A>, , and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902058"><I>P<SUB>lacI</SUB></I>-<I>hpaC</I></A>/<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902059">BBa_B0034-<I>katG-LAA</I></A>. Faint bands around the expected size were seen, and these were stabbed with a pipette tip and stirred into water in order to be used as template in a further round of PCR to get amplification of the part. The plan was that these amplicons could then be used as an insert in further constructions, however no amplification was seen. Further optimization of this procedure could possibly result in a short-cut to construction, however the time needed to do this is a luxury that we do not have.</P><P>Additional attempts to create inserts by doing PCR on ligation reactions in addition to gel extraction of colony PCR bands were attempted as well, however these attempts were unsuccessful as well. <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/soe">SOE PCR</A> and possible <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/Gibson_Assembly">Gibson Assembly</A> appear to be the Sulfur Teams' last hope at this point, and we are eagerly awaiting our primers.
</P><H2>Week 26 (October 21- October 26)</H2><P>
Much to our delight, SOE PCR primers were recieved this week. Therefore, alongside traditional construction of the aforementioned parts, SOE PCR was carried out according to <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/soe">protocol</A> and these parameters:</P><H4>Round 1:</H4><P><UL><LI>Primers 1a and 2 with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902053"><I>P<SUB>tetR</SUB></I>-<I>dszB</I></A> as a template = Product L1 (J13002/DszB with BBk prefix and B0034/DszC overlap)
</LI><LI>
Primers 1b and 2 with <A href="http://partsregistry.org/Part:BBa_K902054"><I>P<SUB>lacI</SUB></I>-<I>dszB</I></A> as a template = Product L2 (J04500/<I>dszB</I> with BBk prefix and B0034/<I>dszC</I> overlap)
</LI><LI>
Primers 3 and 4 with <A href="http://partsregistry.org/Part:BBa_K902056">BBa_B0034-<I>dszC</I></A> as a template = Product L3 (B0034/<I>dszC</I> with <I>dszB</I> and B0034/<I>dszA</I> overlap)
</LI><LI>
Primers 5 and 6 with <A href="http://partsregistry.org/Part:BBa_K902050"><I>dszA</I></A> as a template = Product L4 (B0034/<I>dszA</I> with <I>dszC</I> and J04500/<I>hpaC</I> overlap)
</LI><LI>
Primers 5 and 7 with <A href="http://partsregistry.org/Part:BBa_K902050"><I>dszA</I></A> as a template = Product L5 (B0034/<I>dszA</I> with <I>dszC</I> overlap and BBk suffix)
</LI><LI>
Primers 8 and 10 with <A href="http://partsregistry.org/Part:BBa_K902058"><I>Plac-hpaC</I></A> as a template = Product L6 (J04500/<I>hpaC</I> with BBk prefix and B0034/<I>katG-LAA</I> overlap)
</LI><LI>
Primers 9 and 10 with <A href="http://partsregistry.org/Part:BBa_K902058"><I>Plac-hpaC</I></A> as a template = Product L7 (J04500/<I>hpaC</I> with <I>dszA</I> and B0034/<I>katG-LAA</I> overlap)
</LI><LI>
Primers 9 and 11 with <A href="http://partsregistry.org/Part:BBa_K902058"><I>Plac-hpaC</I></A>  as a template = Product L8 (J04500/<I>hpaC</I> with <I>dszA</I> overlap and BBk suffix)
</LI><LI>
Primers 12 and 13 with <A href="http://partsregistry.org/Part:BBa_K902059"><I>RBS-katG-LAA</I></A> as a template = Product L9 (B0034/<I>katG-LAA</I> with J04500/<I>hpaC</I> overlap and BBk suffix)
</LI></UL></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 21: Round 1 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with varying amounts of DNA template as indicated above the wells. Cycling conditions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 55%deg;C for 1 min, 72%deg;C for 2 min. 30 s.)x 31, Stage 3 (72%deg;C for 10 min.). Bands seen at ~1500 bp for L1 through L5, ~800 bp for L6-L8, and ~2200 bp for L9 indicate successful amplification of the desired product.</DIV></DIV></DIV></DIV><H4>Round 2</H4><P><UL><LI>Primers 1a and 4 with templates L1 and L3 = Product X1 (J13002/<I>dszB</I>/B0034/<I>dszC</I> with BBk prefix and B0034/<I>dszA</I> overlap)</LI><LI>Primers 1b and 4 with templates L2 and L3 = Product X2 (J04500/<I>dszB</I>/B0034/<I>dszC</I> with BBk prefix and B0034/<I>dszA</I> overlap)</LI><LI>Primers 3 and 6 with templates L3 and L4 = Product X3 (B0034/<I>dszC</I>/B0034/<I>dszA</I> with <I>dszB</I> and J04500/<I>hpaC</I> overlap)</LI><LI>Primers 3 and 7 with templates L3 and L5 = Product X4 (B0034/<I>dszC</I>/B0034/<I>dszA</I> with <I>dszB</I> overlap and BBk suffix)</LI><LI>Primers 5 and 10 with templates L4 and L7 = Product X5 (B0034/<I>dszA</I>/J04500/<I>hpaC</I> with <I>dszC</I> and B0034/<I>katG-LAA</I> overlap)</LI><LI>Primers 5 and 11 with templates L4 and L8 = Product X6 (B0034/<I>dszA</I>/J04500/<I>hpaC</I> with <I>dszC</I> overlap and BBk suffix)</LI><LI>Primers 8 and 13 with templates L6 and L9 = Product X7 (J04500/<I>hpaC</I>/B0034/<I>katG-LAA</I> with BBk prefix and suffix)</LI><LI>Primers 9 and 13 with templates L7 and L9 = Product X8 (J04500/<I>hpaC</I>/B0034/<I>katG-LAA</I> with <I>dszA</I> overlap and BBk suffix)</LI></UL></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 22: Round 2 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with varying amounts of gel-extracted DNA template from the previous PCR round as indicated above the wells. Bands were extracted using Qiagen QIAquick Gel Extraction Kit. Cycling conditions for the PCR reactions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 65%deg;C for 1 min, 72%deg;C for 3 min. 30 s.)x 31, Stage 3 (72%deg;C for 10 min.). Faint bands seen at ~3000bp  indicate amplification of the desired product.</DIV></DIV></DIV></DIV><H4>Round 3</H4><P><UL><LI>Primers 1a and 10 with templates X1 and X5 = Product Z1 (J13002/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I>/J04500/<I>hpaC</I> with B0034/<I>katG-LAA</I> overlap and BBk prefix)</LI><LI>Primers 1a and 11 with templates X1 and X6 = Product Z2 (J13002/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I>/J04500/<I>hpaC</I> with BBk prefix and suffix)</LI><LI>Primers 1a and 7 with templates X1 and L5 = Product Z3 (J13002/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with BBk prefix and suffix)</LI><LI>Primers 1b and 10 with templates X2 and X5 = Product Z4 (J04500/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I>/J04500/<I>hpaC</I> with B0034/<I>katG-LAA</I> overlap and BBk prefix)</LI><LI>Primers 1b and 7 with templates X2 and L5 = Product Z5 (J04500/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with BBk prefix and suffix)</LI><LI>Primers 1b and 11 with templates X2 and X6 = Product Z6 (J04500/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I>/J04500/<I>hpaC</I> with BBk prefix and suffix)</LI><LI>Primers 1b and 6 with templates L2 and X3 = Product Z7(J04500/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with J04500/<I>hpaC</I> overlap and BBk prefix)</LI><LI>Primers 1b and 7 with templates L2 and X4 = Product Z8 (J04500/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with BBk prefix and suffix)</LI><LI>Primers 1a and 6 with templates L1 and X3 = Product Z9 (J13002/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with J04500/<I>hpaC</I> overlap and BBk prefix)</LI><LI>Primers 1a and 7 with templates L1 and X4 = Product Z10 (J13002/<I>dszB</I>/B0034/<I>dszC</I>/B0034/<I>dszA</I> with BBk prefix and suffix)</LI><LI>Primers 5 and 13 with templates X5 and X8 = Product Z11 (B0034/<I>dszA</I>/J04500/<I>hpaC</I>/B0034/<I>katG-LAA</I> with <I>dszC</I> overlap and BBk suffix)</LI></UL></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Figure 23: Round 3 of Splice-Overlap-Extention PCR. Reactions were carried out as described above, with 50 ng of gel-extracted DNA template from the previous PCR round. Bands were extracted using Qiagen QIAquick Gel Extraction Kit. Cycling conditions for the PCR reactions were: Stage 1 (95%deg;C for 2 min.), Stage 2 (94%deg;C for 1 min., 65%deg;C for 1 min, 72%deg;C for 6 min.)x 31, Stage 3 (72%deg;C for 10 min.). A faint band (~6000bp) can be seen in lane 4, indicating possible amplification of the desired product. Other reactions appear to have failed, as this bands are not present elsewhere.</DIV></DIV></DIV></DIV><P>
Though it appears that S.O.E. PCR consists of crushed dreams and lies (or that extensive optimization would have to be performed in order to amplify the correct product), extraction was carried out on the single band seen, in the hopes that a subsequent round of PCR with the terminal primers for this product would possibly amplify it to a concentration that can be used in construction in the next week to assemble and test the sulfur operon constructs before competition. Conveniently, the first round of PCR creates <A href="https://2012.igem.org/Team:Calgary/Notebook/Protocols/Gibson_Assembly">Gibson Assembly</A> compatible products. Because a kit is available in the lab, this will be attempted in the following days as well. Meanwhile, a round of traditional construction appears to have finally created the parts <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902079"><I>P<SUB>tetR</SUB></I>-<I>dszB</I>-B0034-<I>dszC</I></A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902080"><I>P<SUB>lacI</SUB></I>-<I>dszB</I>-B0034-<I>dszC</I></A>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902081"><I>P<SUB>lacI</SUB></I>-<I>hpaC</I>-B0034-<I>katG</I></A>. Though sequence confirmation could not be obtained before Wiki-Freeze, we are confident that these parts are correct due to the clear size difference seen between these parts and their sub-parts. A plasmid switch was attempted the night before DNA submission had to be sent, however due to technical issues the incubator stayed at a low temperature overnight, and cultures did not grow enough to obtain the first two parts in a pSB1C3 backbone. Because of the difficulty assembling these parts, the deadline for DNA submission, and the fact that having them in an alternate backbone in the registry would allow for easier and quicker assembly of the sulfur operon by others (no plasmid switches of large parts would have to be performed, as we have found that plasmid switching the larger constructs is quite difficult.) we decided to submit these parts in an ampicillin backbone. Further attempts to switch the backbone for these parts will continue, and hopefully we will be able to submit them in the standard backbone at a later date before the competition. In addition, a reconstruction of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K902051">B0034-<I>dszA</I></A> was obtained and finally, at long last, sequence confirmed. Because of the verification of this part and the excellent sequence read when compared to those obtained for the previously submitted version, we decided to also resubmit this part to the registry.</P><P>Construction attempts of the operon will continue into the following week, as only 1 additional construction is needed to assemble a testable version of the operon. Hopefully, this data will be obtained before the competition, and documented on the respective parts pages.</P><P>This is the Desulfurization Team, signing off.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Calgary/Notebook/Desulfurization">http://2012.igem.org/Team:Calgary/Notebook/Desulfurization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Desulfurization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Desulfurization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Desulfurization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Desulfurization&amp;oldid=301307" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>