"
Page
Discussion
View source
History
teams
Log in
 
Team:Panama/26 June 2010
From 2010.igem.org
iGEM Panama
Home
Team
Students
Instructor
Project
Safety
Notebooks
Sponsors
Human Practice
Photo Gallery
Official Team Profile
Parts Submitted to the Registry
Meeting June 26
At this moment we have two ideas to develop.  We are debating which idea is more feasible.
Idea #1:
We want to produce Rhamnolipid in bacteria (E.coli) to use it as a biosurfactant.  We have to see how we can isolate the rhamnolipid and be sure that the translation is functional.
Steps to follow:
1.Amplify the Rhamnosyltransferase  1  complex from Pseudomona auruginosa and clone this fragment. Size aprox 2.2 Kb.
2.Ligation / Transformation
3.Expression with reporter gen.
4.Isolation
5.Is the protein functional?
6.Make sure that the rhamnolipid is produced in prescence of rhamnose and fatty acid.
Notes:
We can see the reaction (enzymatic measure) by spectrometry.  See kind of ligation.
But first, before the amplification we need to design our primers.
Primer design:
1.Check the gene sequence (In GenBank, FASTA format)
2.Design the primer sequence.
3.See kind of cloning.
4.M13 tail for cleavage site.
5.Look for cleavage site inside the rhamnosyltransferase 1 gene for our restriction enzymes.  We can´t cut our gene in the process.
In summary the steps that we need to follow are:
I. Primer design
II. PCR or amplification
III. Cloning
IV. Expression
V. Sequencing
The most important steps are I and II. We need a good primer design and PCR if we want to be successful.
Idea #2:
Production of Cecropin compound.  We want to produce the antibacterial cecropin compound. Naturally it is produced by insects and plants.
We want to use Anopheles gambiae, cecropin precursor. The gene is about 550 pb.
Ernesto's Idea
The steps to follow are the same in both groups.  Both groups have to design the primers, and have the experimental design for this week.
1.The sequence of the interested gene.
2.Design the primers. (50-100pb more at the beginning and end of the sequence).
3.Check for cloning sites. Which plasmid we are going to use, identify the restriction enzymes.
4.Assemble the blocks. In paper assemble all the system. Promoter + Ribosomal binding site + interest gene + reporter gene + translation end site.
All the design has to be on paper to analyze them and decided which idea is going to be the elected one.
May
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
Retrieved from " http://2010.igem.org/Team:Panama/26_June_2010 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
