Team:Edinburgh OG/Results/T4
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Back to results
Engineering T4 with KPC and VanA constructs
The method used to transform the T4 and T4-gene2 bacteriophage was based on the strategy used by Marinelli et al, called Bacteriophage Recombineering of Electroporated DNA (BRED) (Figure 1). In this technique a double stranded DNA construct containing the   desired inserted sequence is flanked with 100 bp sequence homology outside the region of the gene to be replaced. The host cell for recombineering of the phage and the DNA construct contains a plasmid conferring genes promoting recombination. The double-stranded   insert is electroporated into a cell which had previously been infected with the phage destined to be engineered. Recombination then takes place between the phage and the insert, resulting in recombinant phages which eventually lyse the host. A lysate   is produced containing a mix of transformed and non-transformed phages, which can be mixed with fresh cells. A plaque is formed when one cell is infected and lysed, releasing between 100-150 phages. These phages then infect neighbouring cells, also   resulting in lysis and continued infection until a circular clearing can be viewed on the plate – a ‘plaque’. As each fresh cell will typically be infected with multiple phages – some which will be recombinant, a primary screen   of the plaques produced from the lysate via PCR will show some plaques containing a mix of both types of phage. These plaques can then be re-plated and serially diluted, and subsequent plaques re-screened, one of which should contain a pure recombinant   phage. These are then harvested.
Figure 1: The steps of BRED, taken from Marinelli et al.
Producing VanA:KPC + 100 bp flanking homology construct
Firstly, the KPC chunks were amplified out of the DVK:KPC:GFP plasmid produced for Mock pathogens using PCR with primers adding fusion site E and BsaI to the 5’ end and fusion site F and BsaI to the 3’ end (Figure 2A). This   resulted in the expected 293bp product (Figure 2F). The KPC-EF construct then underwent MoClo with the VanA-AF construct which had also been produced for Mock pathogen creation and DVK-AF (Figure 2B). DVK:VanA:KPC was then transformed   into E. coli and produced 50 white colonies alongside 2 blue. Four white colonies were chosen and digested with AvrII, which had a recognition site in the VanA construct promoter, and shown to be the expected size (Figure 2G). PCR was   then conducted on each checked plasmid using primers to amplify out the fused VanA and KPC chunks and add 50 bp homology of the regions flanking SegC in the T4 genome (Figure 2C). The first three VanA:KPC+50   bp flanking homology products produced the correct band, with the fourth product showing a smaller size than expected (Figure 2H). All four however were amplified once more with primers which would add another 50 bp of homology to the area surrounding SegC, producing a double stranded construct VanA:KPC + 100 bp flanking homology (Figure 2D). Each construct was checked again, with the final product again producing a smaller band than expected (Figure 2I). Gel extraction was conducted   on the first three products which were at the correct size to separate them from a smaller band present at the bottom of the gel. This resulted in the final construct (Figure 2E), which would be electroporated into cells infected with a T4 bacteriophage.
Figure 2: Overview of production of VanA:KPC + 100 bp SegC flanking homology construct. A: Shows where the fusion site editor primers attached. B: Shows how product from the previous reaction could    be used in MoClo to produce C. C: Shows how the Homology +50 bp primers attached to produce D. D: Shows how Homology + 100 bp primers attached to produce the final VanA:KPC + 100 bp SegC flanking    homology construct to be used in BRED. E: The final construct. F: Results of PCR reaction show in A. G: Results of digestion of DVK:VanA:GFP produced by MoClo and shown in C, they were checked    with enzyme AvrII. H: Results of PCR product shown in D.
Results of BRED
BRED was first conducted with T4-gene2 in MG1655:RecD cells containing plasmid pDK46, which contains the lambda red recombinase genes exo, bet and gam. To screen for evidence of recombinant phages, PCR with primers   used to amplify the SegC region were used with primers complementary to the centre of the VanA:KPC construct (Figure 3A). During BRED, infected cells were electroporated with the VanA:KPC construct, after which recombination   between the replicating phage and the construct would occur within the cell - aided by the lambda red genes expressed on a pDK46. The host cell would then burst open to release a mix of recombinant and non-recombinant phages, producing a lysate containing   some recombinants. Four sets of electroporation were conducted, resulting in four T4-gene2 lysates. Lysate 2-4 were screened using PCR with the forward testing primer and reverse SegC primer (Figure 3B). This resulted in PCR products   of the expected size in lysates 3 and 4. Due to this result both Lysate 3 (Figure 3C) and Lysate 4 were mixed with fresh cells to produce plates containing multiple plaques. Individual plaques were then screened by PCR using the forward SegC primer   and the reverse testing primer. In total, 50 plaques from Lysate 4 and 100 plaques from Lysate 3 were screened. Plaque 32 from Lysate 3 resulted in a PCR product with the expected band size (Figure 3D).
Due to the low recombination frequency of T4-gene2 occurring within MG1655:RecD, BRED was repeated with wild type T4 and BL21:DE3:pDK46 cells. This time, five lysates (distinguished as from the T4 BRED procedure with a *) were produced   and screened using the forward testing primer and reverse SegC primer, and all five showed some evidence of a PCR product produced by recombinant phage amplification (Figure 3E). Lysate 1* and 2* and showed the highest PCR product, therefore Lysate   2* was chosen to be plated for plaque screening (Figure F). PCR was conducted on 25 plaques using the forward SegC primer and the reverse testing primer. Plaque 7 showed the expected band size of a recombinant (Figure 3G).
Unfortunately, both PCR products made from T4-gene2 plaque 32 and T4 plaque 7 were discovered on the last day in the laboratory so no more screening and verification of the findings was possible. If more time was available, PCR with the opposite   primers on the plaques would be conducted to see if the correct region was being amplified. As neither of the positive controls, lysate 3 and lysate 2* show a band of the same size, it could be argued this is a false positive result (Figure 3D). However,   during the PCR reaction the lids of both of these became loose and most of the reaction was lost. Therefore, lysate 3 and lysate 2* may not be showing a band due to a lack of product.
Figure 3: Results of primary BRED screen. A: The primers used to screen for recombinants, the flanking primers with T4 homology sequences would be used as a pair with the corresponding primer amplifying the KPC:VanA region. B: results of the T2-gene2 lysates which had undergone BRED, lysate 1 is not shown due to mistakes in the electroporation process, lysate 3 and 4 show the expected band size for amplifying a recombinant. C: Plaques produced from    lysate 3 which were picked for primary screening. D: Result showing correct amplification of plaque 32, although no positive control band shown for lysate 3. E: Lysates with a * indicate they were produced in the wild    type T4 BRED round, with lys3/4 from T4-gene2 acting as a positive control. All BRED T4 lysates show some evidence of band produced by recombinant phage amplification, although the most clear are from lysate 1 and 2. F: Plaques    produced from lysate 2* and picked for primary screen of recombinants. G: Plaque 7 of the BRED T4 produced the expected band size of a recombinant phage.
J. Marinelli et al., BRED: A Simple and Powerful Tool for Constructing Mutant and Recombinant Bacteriophage Genomes. Plos One 3, (2008).
Want more results?
