<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Model_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Model:Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><NAV class="collapse navbar-collapse navbar-right" role="navigation"><DIV class="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:Edinburgh_OG">Home</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Attributions">Attributions</A></LI></UL></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Applied_Design">Applied Design</A></LI></NAV></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Improve">Improve</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Gold_Integrated">Integrated Gold</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#call-to-action">Contact</A></LI><LI class="medals"><A href="https://2017.igem.org/Team:Edinburgh_OG/Medals">Medals</A></LI></DIV></DIV><DIV class="column full_size"><H1 class="subtitle text-center">Model: Background</H1><UL><LI><P>Simple growth models of bacterial populations are based on two interdependent factors: the concentration of bacteria (<EM>X</EM>) and concentration of a growth limiting substrate (<EM>S</EM>) (Monod, 1949)(Figure 1A). Bacteria-phage model can be considered as an extended version of a simple model that includes a third dynamic factor: the phage (<EM>P</EM>)(Figure 1B). The first model was presented as a set of ordinary differential equations (ODDs) by Campbell (1961). In this model, the change rate of the bacterial population is dependent on the concentration of bacteria and substrate, as well as the rate of phage-induced lysis, which in turn is also dependent on the bacterial concentration (Figure 1B).</P><DIV class="div-fig" style="width: 800px;"><P><STRONG>Figure 1.</STRONG> Schematic diagram showing interactions in (A) a two-factor model containing bacteria and substrate, and (B) an extended three-factor model also containing a lytic phage as proposed by Campbell (1961). Taken from Krysiak-Baltyn <EM>et al.</EM> (2016).</P></DIV><H2>Lytic and lysogenic cycles of phage replication</H2><P>In order to model phage-bacteria interactions it is important to understand a step-by-step process of phage infection. Figure 2 shows a lytic life cycle of a lytic phage in a simplified five-step process: from diffusion and attachment to the bacterial cell membrane to cell lysis. The lysogenic life cycle of a temperate phage starts in a similar way to the lytic phage, however following the infection the phage genome either integrates itself into the host genome (λ phage) or forms a circularized plasmid-like structure (P1 phage)(Howard-Varona <EM>et al.</EM>, 2017). The phage genome that has been integrated into the bacterial chromosome is also known as a prophage. Once integrated, temperate phage replicates together with the bacterial genome upon bacterial cell division. However, either spontaneously at low frequency (10-8–10-5) or due to unfavourable stress conditions, such as exposure to UV light, antibiotics, hydrogen peroxide and changes in temperature, nutrients or pH, can lead to a phage induction. Once induced, lytic-lysogenic switch gets activated leading to expression of viral genes, a switch to lytic life cycle and cell lysis.</P><DIV class="div-fig" style="width: 800px;"><P><STRONG>Figure 2.</STRONG> A simplified diagram of lytic and lysogenic life cycles of the phage. (1) diffusion of the phage particle to the bacterial cell; (2) phage attachment to the receptors along the bacterial membrane; (3) injection of phage genome into the cell; (3a) replication via cell division during lysogenic cycle; (3b) induction of lytic cycle; (4) replication of phage genome and assembly of capsid proteins; (5) cell lysis followed by the release of phage particles outside the cell. The burst size (<EM>b</EM>) shows the number of released phage particles. Adapted from Krysiak-Baltyn <EM>et al. </EM>(2016).</P></DIV><DIV class="div-fig" style="width: 800px;"><TABLE><TBODY><TR><TD><P><EM>b</EM></P></TD><TD><P>Burst size</P></TD><TD><P><EM>P</EM></P></TD><TD><P>Concentration of phage</P></TD></TR><TR><TD><P><EM>C</EM></P></TD><TD><P>Carrying capacity</P></TD><TD><P><EM>q</EM></P></TD><TD><P>Induction rate</P></TD></TR><TR><TD><P><EM>D</EM></P></TD><TD><P>Dilution rate </P></TD><TD><P><EM>S</EM></P></TD><TD><P>Concentration of substrate </P></TD></TR><TR><TD><P><EM>d</EM></P></TD><TD><P>Rate of decay of cells or phages </P></TD><TD><P><EM>T</EM></P></TD><TD><P>Latency time </P></TD></TR><TR><TD><P><EM>K</EM><SUB><EM>i</EM></SUB></P></TD><TD><P>Adsorption rate of phages to bacteria</P></TD><TD><P><EM>X</EM><SUB><EM>S</EM></SUB></P></TD><TD><P>Concentration of susceptible bacteria (here antibiotic resistant bacteria)</P></TD></TR><TR><TD><P><EM>K</EM><SUB><EM>m</EM></SUB></P></TD><TD><P>Half-saturation constant </P></TD><TD><P><EM>X</EM><SUB><EM>I</EM></SUB></P></TD><TD><P>Concentration of infected bacteria </P></TD></TR><TR><TD><P><EM>μ</EM><SUB><EM>max</EM></SUB></P></TD><TD><P>Maximum specific growth rate</P></TD><TD><P><EM>Y</EM></P></TD><TD><P>Bacterial yield, or bacterial cells per unit substrate</P></TD></TR></TBODY></TABLE><P><STRONG>Table 1.</STRONG> Parameters used in models of populations of phage and bacteria.</P></DIV><H2>Basic model</H2><P>Modelling of the processes described in Figure 2. involves developing and solving expressions for the concentrations of phages, infected and non-infected bacteria as well as the substrate. The key parameters involved into equations are listed in Table 1. Following the approach documented in Campbell (1961), the rate of infection is proportional to the concentration of bacteria, lytic phage and the absorption rate (<EM>K</EM><EM>i</EM>). The latter is the rate at which phage particles attach to the bacterial surface and it is dependent on (1) the concentrations of both bacteria and phage, (2) the time required for diffusion and attachment of the phage, and (3) the number of phage particles, which upon attachment to the bacterial cell, do not lead to infection.</P><P>Oppositely, the growth rate of phage population decreases due to attachment of phage particles onto bacterial cells:</P><P>And increases due to cell lysis and release of newly produced phage particles:</P><P>The <EM>latency time </EM>(<EM>T</EM>) is caused by the time delay between the infection and cell lysis (steps 3 and 5 in Figure 2).</P><P>The growth of bacterial population was represented as a logistic kinetic expression that is independent from the concentration of the growth limiting substrate, but dependent on the total <EM>carrying capacity</EM> (<EM>C</EM>) <EM>i.e. </EM>the maximum bacterial concentration after which bacteria cease cell division. Thus, the growth rate of bacteria slows down once it approaches the carrying capacity value.</P><P>Campbell’s model was describing continuous culture in a chemostat, which is a bioreactor with a continuous inflow of fresh medium and outflow of liquid culture with phage particles and bacteria. Therefore, a washout rate of both phages and bacteria was also taken into account as simple first-order kinetics, where <EM>D</EM> is the <EM>dilution rate</EM>:</P><P>In order to simulate real life environment, bacterial (<EM>d</EM><EM>X</EM>) and phage decay (<EM>d</EM><EM>P</EM>) rates were introduced. It has been shown that UV light can have a significant negative impact on phage infectivity by altering its DNA, which is does not necessarily destroy the virion particles (Suttle and Chen, 1992; Rastogi <EM>et al.</EM>, 2010). In nature phage decay is also associated with absorption by heat-labile particles or direct consumption by flagellates (Suttle and Chen, 1992; Deng <EM>et al.</EM>, 2014). Bacterial cells are more robust compared to the phages, but are subject to endogenous decay, which is caused by oxidation of internal cell components for production of energy and life maintenance whilst being limited by the availability of the growth substrate (Droste, 1998).</P><P>Arguably, one of the most important aspects of mathematical modelling is the initial set of assumptions that are used to build a model. The very first model of phage-bacterial interactions published by Campbell (1961) was based on the following assumptions:</P></LI><LI>1. A single bacterial cell can be infected only by one phage.</LI><LI>2. Both rates of phage attachment and infection are dependent on bacterial and phage concentrations.</LI><LI>3. The burst size does not change and is the same for each infection.</LI><LI>4. The latency time does not change and is the same for each infection.</LI><LI>5. The bacterial growth rate is expressed as a logistic function.</LI><P>Based on these assumptions, he came up with the following set of delayed differential equations (DDEs):</P><P>Notably, most of these assumptions are nowadays considered false by the community working on bacteria-phage interaction. Firstly, there are multiple binding sites to which phage particles can attach to on the bacterial membrane. Thus, many of them can be occupied at the same time, leading to multiple simultaneous infections (Figure 3). Secondly, the burst size is affected by a number of variables, including the cell size of the bacterium, its metabolic activity and particular phage-bacteria relationship (Parada, Herndl and Weinbauer, 2006). Similarly, the latency time depends on the cell density, selecting for a shorter latency time upon high cell density and opting for longer latency upon low density (Wang, Dykhuizen and Slobodkin, 1996). Lastly, the bacterial growth is naturally dependent on the availability of resources to feed on, therefore the logistic equation for the bacterial growth has been replaced for the Monod equation, which takes availability of the substrate into account.</P><DIV class="div-fig" style="width: 350px;"><P><STRONG>Figure 3.</STRONG> Multiple phages attached to the bacterial membrane via phage binding sites. Taken from Smith and Trevino (2009).</P></DIV><P>To tackle some of these limitations, Levin <EM>et al. </EM>(1977) proposed a new generalized model, that could include any number of phages and bacterial species, as well as substrates. The change rate of the concentration of substrate depends on its consumption by both non-infected and infected bacteria, multiplied by the inverse value of the bacterial yield (<EM>Y</EM>), and concentration of inflow and outflow medium:</P><P>The growth of non-infected bacteria is described by the Monod equation:</P><P>They also included a separate equation to track the change in population of infected bacteria as well as its removal from the chemostat, which had a negative impact on the number of phage particles generated from lysis and has not been included into Campbell’s (1961) model.</P><P>Although Levin’s model predicted the concentrations at steady-state within an order of magnitude from experimental values, it also predicted a total extinction of either phage or bacteria, or both, whereas this was not observed experimentally. Ultimately, the model was labelled not complex enough to accurately describe the interactions of phage and bacteria (Levin, Stewart and Chao, 1977). Since then, the model has been extended by others to include variable absorption rate and burst size (Smith and Thieme, 2012), multiple binding sites (Smith and Trevino, 2009), bacterial resistance to phages (Lenski and Levin, 1985; Cairns <EM>et al.</EM>, 2009) and spatial heterogeneity (Jones and Smith, 2011). Detailed description of these extensions is beyond the scope of this study.</P><P>In 2007 Qiu augmented Levin’s model by adding lysogenic cycle to it in order to study the dynamic mechanisms of the lytic-lysogeny decision. In contrast to Levin, <EM>lysogenic bacteria</EM>, that is bacteria infected with temperate phages, were able to grow by consuming substrate. He also introduced an <EM>induction rate</EM> (<EM>q</EM>) – that is the rate at which lysogenic bacteria enter lytic cycle either spontaneously or due to environmental factors discussed previously. The concentration of free phage particles (<EM>P</EM><EM>T</EM>) only increased due to induction of lysogenic bacteria (<EM>X</EM><EM>L</EM>).</P><P>It is worth noting that Qiu’s model did not account for time latency between infection and lysis, thus presenting a system of ODEs rather than DDEs (Qiu, 2007).</P><DIV class="btn-row"><DIV><A class="btn-result" style="width:90%;" data-wow-delay=".9s" href="https://2017.igem.org/Team:Edinburgh_OG/Model" style="visibility: visible; animation-delay: 0.9s;">Model Homepage</A></DIV><DIV style="clear:both;"><A class="btn-result btn-active" data-wow-delay=".9s" href="https://2017.igem.org/Team:Edinburgh_OG/Model:Background" style="visibility: visible; animation-delay: 0.9s;">Background</A><A class="btn-result" data-wow-delay=".9s" href="https://2017.igem.org/Team:Edinburgh_OG/Model:Methodology" style="visibility: visible; animation-delay: 0.9s;">Methodology</A></DIV><DIV style="clear:both;" class="btn-row"><A class="btn-result" data-wow-delay=".9s" href="https://2017.igem.org/Team:Edinburgh_OG/Model:Results" style="visibility: visible; animation-delay: 0.9s;">Results</A><A class="btn-result" data-wow-delay=".9s" href="https://2017.igem.org/Team:Edinburgh_OG/Model:References" style="visibility: visible; animation-delay: 0.9s;">References</A></DIV></DIV></UL></DIV></DIV></BODY></HTML>