Team:Edinburgh OG/Design
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Design
Design of our project summary
Our design is unique because it results in a population of bacteria to be re-sensitised to specific antibiotics. This allows previously redundant antibiotics to be re-used, through the exploitation of a two-phage system (Figure 1). We engineer the first     phage, lysogenic, to carry a CRISPR system which can cleave at the site of an antibiotic resistant gene, resulting in degradation of any plasmids containing that gene. We then introduce a second phage, lytic, to eliminate all the bacteria which have     not been re-sensitised by the lysogenic phage. This discriminative elimination is possible because all of the re-sensitised bacteria are immune to the lytic phage, which we have engineered to contain the targeted sequences cleaved by CRISPR. This     system firmly ensures all the treated population is re-sensitised to antibiotics.
In search of effective re-sensitisation of antibiotic resistant pathogens, our major aim was to give modularity to our system. We wanted to trial different CRISPR systems and associated proteins so we could optimise our system by choosing     ones which gave the better results. We also wanted to demonstrate that this system could work with any CRISPR system, as in principle it should be successful every time. We chose Staphylococcus aureus Cas9 (SaCas9), Francisella novicida Cpf1 (Cpf1) and Streptococcus pyogenes Cas9 (SpCas9) CRISPR systems. Due to IP issues with our collaborators, the data regarding SpCas9 are not shown. For the phages we chose lysogenic phages P1 and λ (lambda) and lytic phages T4 and T7.
To produce CRISPR systems which could be carried by both P1 and λ phages, we generated an SaCas9 cassette by improving BBa_K2019000, and a newly designed FnCpf1 cassette. These cassettes were compatible with both phages. We also produced a Spacer     cassette which allowed new spacers to be rapidly swapped in. Therefore, multiple spacers for different genes can be quickly tested, enlarging the range of potential antibiotic resistant genes. This also increases our project’s modularity. For     all the CRISPR systems, the backbone chosen was the P1 phagemid (J72114-J72100) (Kittleson, 2012), which was essential both to obtain P1 particles and to construct each complete CRISPR system.
In order to avoid using hazardous pathogens, we produced an E. coli testing platform. These cells contained plasmids which had four fragments of the antibiotic resistant genes blaKPC (hereinafter referred to as KPC) and vanA. These short regions contained protospacers for all three CRISPR systems, and were shuffled in order to ensure the gene was non-functional. These same gene regions were then placed into the T4 / T7 bacteriophage. This meant that upon infection, the     injected lytic DNA would be recognised by the CRISPR-system within the cell and be cleaved. The rest of the genome would therefore degrade and the cell would have immunity against lysis. But any bacteria which have avoided lysogenic infection by chance     or are resistant to infection (and have therefore retained their antibiotic resistant gene) will not have immunity to the lytic phage. This means all the antibiotic resistant cells as ‘mopped up’ and destroyed, allowing the surviving and     re-sensitised population of bacteria to be easily killed when an antibiotic is applied later if they cause an infection.
Benchling files:
https://benchling.com/s/seq-5DjFTFThzh0vg1rSa4fr/edit (SaCas9)
https://benchling.com/s/seq-a6aQ5LGHwBezYUdxmalK (FnCpf1)
Figure 1: Overall scheme of our project.  
Design of CRISPR systems
The FnCpf1, SaCas9 and SpCas9 CRISPR systems (the systems referred to as their affiliated protein name hereinafter) were codon-optimized for E. coli (https://www.idtdna.com/CodonOpt) and the BioBrick illegal sites removed.     The Cas systems cassettes were designed downstream of the Lac promoter and the medium Anderson RBS (iGEM registry part BBa_J61117).
Due to the large size of the CRISPR systems, the constructs were divided into 2 parts, FnCpf1/SaCas9/SpCas9 Part I and Part II (Figure 2). For all the CRISPR systems the backbone was the P1 phagemid. Therefore to place it within a λ genome, the     engineered phagemid would need to become a P1 particle and construct whichever CRISPR system we have inserted into it within an E. coli cell. Then this can be digested/ligated with the λ genome. Each Part I was firstly inserted in     the P1 phagemid using BsiWI and AvrII. Then, Part II was inserted into the corresponding Part I using HindIII and AvrII (for FnCpf1) or ApaI and AvrII (SaCas9/SpCas9).
Figure 2: The Cas constructs were divided into Part I and Part II. Once each Part I (top) was inserted in the P1 phagemid (not shown) with BsiWI and AvrII, each Part II (bottom) was introduced into Part I using HindIII/ApaI (for FnCpf1     and SaCas9/SpCas9 respectively) and AvrII as shown.
Inserting CRISPR systems into λ phage
The BsiWI and XbaI restriction enzymes were chosen to introduce each CRISPR construct into λcI857 (thermos inducible λ phage). Both enzymes are in the disposable region of λ, a region which, if eliminated, does not affect the replication     machinery. However, XbaI is an illegal site and the AvrII restriction enzyme (compatible with XbaI and present in the P1 phagemid) was adopted in the CRISPR systems design. A repeat cassette was designed downstream of each CRISPR system and under     its own Lac promoter.
Inserting CRISPR systems into P1 phage
The P1 Phagemid contains several P1 genes such as origin of replication and a pac site; recognised as the starting point of packaging (Kittleson 2012). However, for safety reasons, the P1 phagemid does not contain a functional replication machinery. Therefore, to obtain P1 phages, phagemid should be transformed into the host bacterium (E. coli C600, a lysogen of P1 phage, thus contains the whole P1 genome).
When the CRISPR cassettes were inserted into the phagemid, BsiWI and AvrII were used (Figure 2). The restriction sites were located in the lacZ locus in phagemid, allowing a rapid Blue-White screening.
Inserting spacers into CRISPR cassettes
The cassette contained two sub-repeats between which spacers could be inserted. The strategy to insert the spacers relied on the type IIS restriction enzyme BsaI. A BsaI-specific region was designed between the two repeats: the enzymes had its binding     site inside this region but cut to the right and left extremities of the repeats, producing two different but unique overhangs. In this way, it was possible to replace the BsaI-specific region with any spacer construct which had BsaI sites and the     compatible overhangs. Because the overhangs were unique, the correct directionality was assured. Finally, the T1 terminator of E. coli (iGEM registry part BBa_B0010) was added downstream of the repeat cassette.
Design of the spacer constructs
Each spacer (the target region for the Cas endonuclease) was divided into two cassettes (KPC Part I, KPC Part II and vanA Part I, vanA Part II). Spacer parts could be combined together through their BbsI sites, therefore when choosing spacer sequences     it was essential to avoid a Type IIS restriction sites. Part I + Part II could be inserted into the plasmid containing the Cas system with BsaI restriction/ligation (Figure 3). As a backup strategy, specific reverse primers for each Part I were designed.     The primers had overhangs to introduce the BsaI site. In this way, Part I of each spacer construct contained both BsaI sites, and therefore could be introduced directly into the FnCpf1 plasmid without being connected to Part II.
Figure 3: Design of the spacer constructs. Spacer Part I and Part II were ligated with BbsI. Then, BsaI sites directed the insertion of the complete spacer construct into FnCpf1 using the two different unique overhangs.
Experimental setting
To check our CRISPR systems we produced a E.coli testing platform (Mock pathogens hereinafter) by using Modular Cloning (or MoClo)(Iverson 2016). We produced a plasmid which contained fragments of either KPC or VanA fused     to GFP. This would provide us with a clear signal if the fragments were successfully cleaved by CRISPR, as the colonies would stop fluorescing. As an complementary screening method we also wanted the plasmid containing these sequences to     provide the cells with antibiotic resistance, as sudden susceptibility to the antibiotic after CRISPR exposure would also indicate cleavage of the plasmid. For safety reasons we could not fuse the entirety of KPC or VanA to GFP within E. coli. Instead, we chose four regions of KPC and VanA by identifying spacers which would work with all three CRISPR systems we were trialling, and interlinked them with repeats of glycine and serine. MoClo is a one-pot     digestion and ligation assembly method which utilizes the restriction enzyme BsaI (Figure 4).
Figure 4: Method of how our E. coli testing platform / ‘mock pathogens’ were produced. These plasmids were inserted into TOP10 cells and selected via blue/white screening. A: Displays construct assembly, BsaI:R     indicates the enzyme will cut to the right, BsaI:L indicates it will cut to the left. A, E and F = MoClo fusion sites. B: Overview of each construct and DVK-AF once it has been cut by BsaI. C: After ligation DVK will contain VanA fused to GFP.
The mock pathogens containing the plasmids with the spacers and kanamycin resistant gene would therefore be infected with an engineered P1 / λ phage. The CRISPR systems are then triggered quickly after infection, being translated from the phagemid     containing a chloramphenicol resistance gene (P1 phagemid). Therefore, if the cleavage by the CRISPR system occurs, it results in the loss of Kanamycin resistance in the Mock pathogen E. coli, and the acquisition of Chloramphenicol resistance.     On the other hand, if the CRISPR system doesn’t work, the host cells still displays Kanamycin resistance as the plasmid with the KPC / vanA regions have not been cut, as well as Chloramphenicol resistance - due to the presence of the     phagemid (Figure 5).
As we also fused the vanA and KPC gene regions to GFP, the cells would be fluorescing green before infection with the lysogenic phage. When the lysogenic phagemid is introduced, the CRISPR system should cut the plasmid. This would mean the cells would     stop fluorescing green as they no longer have the gene to make the KPC/vanA:GFP fused protein. Therefore, colonies which had stopped fluorescing and have newly acquired resistance to chloramphenicol will be proven to have successfully removed their     vanA/KPC plasmid due to the CRISPR system introduced by the lysogenic phage.
Figure 5: Strategy for checking the efficiency of SaCas9. After MoClo cells infected by phage with SaCas9 system are plated on Chloramphenicol plates (step 1), several colonies were inoculated into Kanamycin plate (step 2). Only ones     which have successfully been cleaved their target sequence cannot grow on Kanamycin plates.
Adding the lytic phage
After the cells have been exposed to the lysogenic phage, there will still be some bacteria which still contain the antibiotic resistant plasmid. This could be because the lysogenic phage did not reach them, the CRISPR system did not activate correctly     or the bacterium is resistant to lysogenic infection. We therefore designed a lytic phage to contain the same regions of KPC and vanA which were within our mock pathogens. Therefore, if the lytic DNA entered the cell with a working CRISPR system introduced     by the lysogenic phage it would be cleaved and degraded, allowing the cell re-sensitised to antibiotics to survive. However, cells without the CRISPR system would be susceptible to infection as they will have no defense against the lytic phage - resulting     in destruction.
T4 design
To first engineer T4 to contain the KPC and vanA regions, they were fused together using MoClo and PCR, and had their promoter removed (Figure 6). By using this technique it means the lytic T4 phage could be used against both KPC and VanA mock pathogens.     This could mean one lytic phage could work with several lysogenic phages which tackle different genes, which again demonstrates modularity in our design. These gene regions were placed into T4 using homologous recombination, and replaced an unessential     gene called SegC.
Figure 6: Overview of production of VanA:KPC + 100 bp SegC flanking homology construct. A: The plasmid used to make our mock pathogens, and shows where the fusion site editor primers attached. B: Shows how the product     from the previous reaction could be used in MoClo to produce C. C: Shows how the Homology +50 bp primers attached to produce D. D: Shows how Homology + 100 bp primers attached to produce E, E: the final VanA:KPC + 100 bp SegC flanking     homology construct to be used for homologous recombination into T4 bacteriophages.
The technique used to insert the gene region is called Bacteriophage Recombineering of Electroporated DNA (BRED), a method for engineering lytic bacteriophages by Marinelli et al. (2008). In this method, the lytic bacteriophage undergoes homologous recombination     with a double-stranded DNA construct within a bacterium, which is aided by homologous recombination proteins provided by a plasmid. We used the plasmid pDK46, with contains the lambda red recombineering genes. The DNA to be swapped in is flanked with     100 bp homology surrounding the gene to be replaced within T4, and is electroporated into the host bacteria just after infection with the lytic phage. The recombineered phage then replicates inside the cell and bursts out, to infect new cells and    produce more engineered lytic phages. Eventually, all the cells have been burst and the media is thick with a mix of engineered and non engineered phages (called a lysate). The lysate is then diluted and mixed with fresh cells and media, and plated.
The cells form a lawn over the plate, however small openings (plaques) appear on the plate where a T4 has initially infected and lysed a cell. As each lysed cell tends to release 50-150 new T4 phages, these then spread to neighbouring cells, eventually     producing a plaque in the lawn visible to the eye. Some plaques will contain a mix of engineered phages containing the gene regions of VanA and KPC, and the non-engineered wild-type. To check for this, plaques can be picked and undergo PCR with primers    which will amplify the region which was replaced (Figure 7A,E). To firstly show that homologous recombination successfully occurred, the lysate underwent PCR with the testing primers. Then, individual plaques on a plate (Figure 7C,F) also underwent     PCR, showing the band size expected for a recombinant T4 (7D,G). We conducted BRED on two T4 strains, a wild type strain and T4-gene2, which can only replicated in E.coli lacking the gene for RecD - making it safer to work with.
Figure 7: Results of primary BRED screen. A: The primers used to screen for recombinants, the flanking primers with T4 homology sequences would be used as a pair with the corresponding primer amplifying the KPC:VanA region. B: results of the T2-gene2 lysates which had undergone BRED, lysate 1 is not shown due to mistakes in the electroporation process, lysate 3 and 4 show the expected band size for amplifying a recombinant. C: Plaques produced from lysate 3     which were picked for primary screening. D: Result showing correct amplification of plaque 32, although no positive control band shown for lysate 3. E: Lysates with a * indicate they were produced in the wild type T4 BRED round, with lys3/4 from T4-gene2 acting as a positive control. All BRED T4 lysates show some evidence of band produced by recombinant phage amplification, although the most clear are from lysate 1 and 2. F: Plaques produced from lysate 2* and picked for primary recombinant screen. G: Plaque 7 of BRED with T4 produced the expected band size of a recombinant phage.
Once the engineered T4 phage is isolated it can be used to infect further cells and produce a lysate which contains pure engineered T4. This can then be diluted and applied to our mock pathogens after lysogenic phage infection.
T7 Design
The T7 phage was engineered slightly differently from lytic T4 to contain the same protospacer sequences as our E. coli testing platform. To do this we needed to engineer the phages and then select for those which had been successfully engineered.     The system for engineering and selection of the T7 lytic phages was based on a 2014 paper by Kiro et al. (doi: 10.4161/rna.27766).
The first step of this system involves engineering the T7 via homologous recombination between the T7 genome and a plasmid that has been engineered into E. coli (Figure 8).
Figure 8: When WT T7 infects the E. coli, homologous recombination occurs between the 1.7 flanking regions. This results in the replacement of 1.7 with our protospacer construct.
As in the work by Kiro et al., we chose to replace the non-essential 1.7 gene in T7 with our protospacer construct. Homology regions consisted of 100bp sequences that flank the 1.7 in wild-type (WT) T7. The sequences were taken from the full T7 genome     on the NCBI database. Flank constructs (Figure 9) were designed and synthesised to contain platform DNA, BsaI recognition sites and MoClo fusion sites – F & A for the 5’ flank, E & G for the 3’ flank – bordering the     homology region, resulting in a total of 134bp for each flank construct. MoClo was used to combine the flanks with the protospacer chunk construct (Figure 10) to create the homologous recombination plasmid (Figure 11).
Figure 9: Flank constructs (5’ and 3’) were designed to be combined with the protospacer constructs (Figure 3) via MoClo.
Figure 10: Protospacer construct design.
Figure 11: MoClo was used to combine the protospacer construct, flank constructs and the DVK_FG plasmid acquired from the CIDAR MoClo kit.
The second stage of this system is selection of engineered phages using CRISPR (Figure 12). A second strain of E. coli is engineered to contain CRISPR targeted against the 1.7 gene. When the T7 from the previous stage are allowed to infect     this CRISPR E. coli, any non-engineered phages will be destroyed by the CRISPR system as they will still contain 1.7. Successfully engineered phages will survive and be able to replicate and be purified.
Figure 12: Only successfully engineered T7 genomes will be immune to the CRISPR system. Any that still contain 1.7 will be cleaved.
Three pairs of spacer oligonucleotides were commercially synthesised as in Figure 13 (sequences in Table 1).
Figure 13: Pairs of oligonucleotides were synthesised to contain BsaI overhangs and a 20bp spacer sequence. Figure from Addgene.
Oligonucleotide name
Sequence
Spacer 1 Oligo 1
AAACTATCCACACACCTGTAACAAG
Spacer 1 Oligo 2
AAAACTTGTTACAGGTGTGTGGATA
Spacer 2 Oligo 1
AAACGGTCGCCCAGATGATTTGAAG
Spacer 2 Oligo 2
AAAACTTCAAATCATCTGGGCGACC
Spacer 3 Oligo 1
AAACCCTTCCGCTGAGACAATCGAG
Spacer 3 Oligo 2
AAAACTCGATTGTCTCAGCGGAAGG
Table 1: Spacer oligonucleotide sequences for T7 CRISPR selection system.
These oligonucleotides were designed such that, upon annealing to one another, they would have appropriate overhangs for ligation into BsaI-digested pCas9. Spacer oligonucleotides were designed to target the 1.7 gene in T7, and were based on     the T7 genome sequence from the NCBI database (accession number: V01146 J02518 X00411). Oligonucleotides for three spacers were produced, with the intent of repeating the CRISPR selection process three times to ensure complete removal of any non-engineered     phages.
The pCas9 plasmid used in CRISPR E. coli production was provided by Dr Russell Brown (Wang lab, University of Edinburgh) and was originally produced by Dr Luciano Maraffini (Addgene plasmid reference #42876).
What next?
The engineered T4 and T7 lytic phages would then be added to cells which had been pre-exposed to the lysogenic phages. The interval before delivery of the second lytic phage should be the length of time it takes for the lysogenic phage to reach every     cell and develop into a functioning phagemid.
Kittleson JT, DeLoache W, Cheng H-Y, Anderson JC. Scalable Plasmid Transfer using Engineered P1-based Phagemids. ACS synthetic biology. 2012;1(12):583-589. doi:10.1021/sb300054p
Kiro, R., Shitrit, D. & Qimron, U. Efficient engineering of a bacteriophage genome using the type I-E CRISPR Cas system. RNA Biol. 11, 42–44 (2014).
J. Marinelli et al., BRED: A Simple and Powerful Tool for Constructing Mutant and Recombinant Bacteriophage Genomes. Plos One 3, (2008).
V. Iverson, T. L. Haddock, J. Beal, D. M. Densmore, CIDAR MoClo: Improved MoClo Assembly Standard and New E-coli Part Library Enable Rapid Combinatorial Design for Synthetic and Traditional Biology. Acs Synthetic Biology 5,     99-103 (2016).
