Team:Edinburgh OG/Results/T7
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Back to results
Production of T7 phage
Homology E. coli Construction
E. coli TOP10 containing DVK_FG + T7 homology flanks + KPC were successfully constructed. The resulting plasmids were checked by digestion with AvrII and BbsI, which indicated that the construction was correct. Plasmid DNA was sequenced and no mismatches were seen when the insert sequence was aligned with the template, and chromatogram peaks were strong and unambiguous.
Production of E. coli TOP10 containing DVK_FG + T7 homology flanks + vanA was not successful. 4 white colonies were checked via digestion with EcoRI, BbsI and SpeI, and the expected band pattern was not seen. Colonies 1, 3 and 4 shared one banding pattern while colony 2 displayed another (Figure 1) so DNA from colonies 1 and 2 was sequenced.
Figure 1: 4 MoClo colonies were checked for vanA insertion via digestion with EcoRI (E) and SpeI (S). Uncut controls (U) were also run. Band sizes were not as predicted, indicating that the insert is incorrect.
In the colony 1 sequences, both the forward and reverse aligned well with the T7 flanks, and the first half of the protospacer construct. However, a large gap in alignment was seen spanning from the end of protospacer 2 to the end of protospacer 4 (Figure   2). It is unclear what could cause what seems like a precise excision of half of the vanA sequences. Chromatogram peaks were unambiguous so sequencing issues are unlikely.
Figure 2: Colony 1 sequences indicated a loss of two protospacers from the construct. Diagram created by modifying image produced by sequence analysis tool on Benchling.
The reverse colony 2 sequence showed no alignment to the insert or plasmid whatsoever (Figure 3). The forward sequence aligned to a short sequence prior to the insert, then contained a large gap in alignment followed by a very weak alignment to the 3’ end of the 3’ flank. It is unclear what could have caused such massive discrepancy between the expected and obtained sequences. Chromatogram peaks were significantly smaller and less clear than with colony 1, so it is possible that the discrepancy   is at least partially due to sequencing issues.
Figure 3: The colony 2 reverse sequence showed no alignment to the template. The forward sequence aligned to parts of the plasmid (with low sequence quality peaks) but had many mismatches with the insert. Diagram created by modifying   image produced by sequence analysis tool in Benchling.
CRISPR E. coli Construction
Construction of E. coli BL21 DE3 containing a CRISPR system and spacers against the T7 1.7 gene was unsuccessful. Several attempts were made to ligate the spacers into the pCas9 plasmid - using simultaneously phosphorylated, separately   phosphorylated and unphosphorylated spacer oligo pairs. In all cases, transformation into E. coli resulted in no colonies on chloramphenicol selection plates. The chloramphenicol plates, ligation protocol, and BL21 DE3 cells were successfully   used in other experiments by the team, so these are unlikely to be the source of the problem. The spacers were only 25bp in length, so it is possible that their small size prevented successful ligation.
Due to the lack of successful production of the CRISPR selection system, no work was done on the T7 phages themselves.
Want more results?
