Team:Edinburgh OG/Results/MoClo
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Back to results
MoClo: Construction of the testing platform
Figure 1: Overview of successful testing platform results. A: Results of blue–white screening with TOP10 cells containing DVK:KPC:GFP (white colonies).B: DVK:KPC:GFP checked using AvrII, with four colonies tested showing expected size band of ~3.5 kb. C: Results of VanA construct amplification via PCR using amplifying primers at four temperatures, 66°C shows the highest quantity of product. D: Result of VanA 66°C PCR cleanup, a secondary band is still present therefore gel extraction was conducted on the larger band at the correct size. E: Results of of blue – white screening with Top10 cells containing DVK:VanA:GFP (white colonies). F: DVK:VanA:GFP checked using EcoR1, four colonies tested and all show expected size of ~3.5 kb. G: Image of cells transformed with DVK:VanA:GFP and DVK:KPC:GFP against un-transformed TOP10 cells. There does not appear to be any difference in fluorescence. H: Cells previously containing DVK:VanA:GFP and infected with P1 phage containing CRISPR-cpf1 growing on LB media. I: Same cells as H grown on chloramphenicol, J: same cells as I / H grown on kanamycin.
KPC
Testing platform E. coli TOP10 containing KPC target sites were successfully created (Figure 1A). Four white colonies from the plate of transformed TOP10 were checked via digestion with AvrII, yielding bands of the correct sizes [3459bp] (Figure 1B, 2).
Figure 2: 4 white MoClo colonies were checked for correct insertion of the KPC construct via digestion with AvrII (A). All bands were of the expected size and uncut controls (U) indicated complete digestion had occurred.
DNA extracted from colony 4 was sequenced by Edinburgh Genomics to confirm that the insert was correct. Sequencing results confirmed that the insert was correct. While some mismatches were detected, they occurred near the beginning/end of the DNA where the chromatogram indicated sequence quality was low. No mismatches were shared between the forward and reverse sequences, and chromatogram peaks within the insert were clear and unambiguous.
vanA
Testing platform E. coli TOP10 containing vanA target sites were successfully created. Firstly the VanA gene region had to be amplified by PCR to produce a stock of it (Figure 1C), and then cleaned up to remove secondary primer dimer bands (Figure 1D). This then underwent MoClo with GFP, and 4 white colonies from the resulting plate of transformed TOP10 (Figure 1E) were checked via digestion with EcoRI, yielding bands of the correct sizes [3459bp] (Figure 1F).
DNA extracted from colony 1 cells was sequenced to check the insert. Both the forward and reverse query sequences show some mismatches in the first 100bp and 2 mismatched bases are seen late in the FWD product, but these are likely a result of low quality at the sequence ends rather than true errors. A gap is seen between the two query sequences (Figure 3) – the sequence reads were evidently not of sufficient length to result in overlap between the two products. Alternative sequencing primers could be used to confirm that no major issues are present in this gap.
Figure 3: A gap is seen between the forward and reverse vanA insert sequences (mismatches in red). Diagram created by modifying image produced by sequence analysis tool on Benchling.
ampC
Production of testing platform E. coli TOP10 containing ampC target sites was not successful. Digestion of the produced MoClo construct with AvrII, EcoRI and EcoRI+PstI did not produce bands of the expected sizes.The construct was also sequence to compare it to the predicted sequence.
The insert appeared to be almost entirely different from the template sequence, in both the ampC and GFP regions (Figure 4). Chromatogram peaks are clear and unambiguous, suggesting that the mismatches are real. It is possible that some sort of contamination occurred in the MoClo reaction mix, resulting in the production of an incorrect construct.
Figure 4: The ampC insertion appears to be completely incorrect, as both the forward and reverse sequences contained very high numbers of mismatches (indicated in red). Diagram created by modifying image produced by sequence analysis tool on Benchling.
GFP Expression
No colonies on plates of vanA+GFP or KPC+GFP cells appeared green either in natural light or under exposure to UV light in a UV spectrophotometer. Cell pellets from liquid cultures of colonies confirmed to contain the correct insert also did not appear green in
white or UV light. Fluorescence microscopy revealed no difference in fluorescence between transformed TOP10 and wild-type cells (Figure 1G).
The insert was initially under the control of a medium-strength Anderson promoter (BBa_J23111). It was hypothesised that this promoter was not adequate to produce detectable GFP expression, so PCR was used to replace it with a stronger equivalent (BBa_J23100).
MoClo was repeated with the new protospacer construct containing the strong promoter to create KPC_new_Anderson cells. The resulting plasmid was checked by sequencing and confirmed to contain the correct promoter.
KPC_new_Anderson cells were checked for GFP expression. Despite replacement of the medium-strength promoter, no change in GFP expression was seen upon examination with white/UV light or fluorescence microscopy. This suggests that, rather than being caused by weak expression, the lack of visible GFP is due to incorrect folding of the protein thanks to its attachment to the protospacer sequences.
While the lack of GFP expression is inconvenient in terms of simple visualisation of CRISPR activity, it doesn’t prohibit the detection of the system’s activity. The DVK_AF plasmid contains a kanamycin resistance gene, and the phagemid upon which the CRISPR is transferred contains a chloramphenicol resistance gene. Therefore, a functioning CRISPR system will cause the loss of kanamycin resistance and the gain of chloramphenicol resistance.
Antibiotic resistance expression
The mock pathogens were exposed to the engineered P1 phage containing CRISPR-Cpf1. We then grew the cells on plates containing either chloramphenicol, kanamycin or antibiotic free LB media. A number of the colonies which previously contained the VanA:GFP construct showed the expected result. They grew on pure LB plates (Figure 1H), showed resistance to chloramphenicol – implying the presence of the P1 phagemid (Figure 1I), and they no longer had resistance to kanamycin – indicating DVK:VanA:GFP had been cleaved by CRISPR-Cpf1 (Figure 1J). Although this system did not work with cells containing KPC chunks, a lysogenic phage containing CRISPR-spCas9 successfully demonstrated the same pattern in both cell types. Therefore it appears that the mock pathogens did contain the correct KPC and VanA chunks.
Want more results?
