<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><NAV class="collapse navbar-collapse navbar-right" role="navigation"><DIV class="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:Edinburgh_OG">Home</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Attributions">Attributions</A></LI></UL></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Applied_Design">Applied Design</A></LI></NAV></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Improve">Improve</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Gold_Integrated">Integrated Gold</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#call-to-action">Contact</A></LI><LI class="medals"><A href="https://2017.igem.org/Team:Edinburgh_OG/Medals">Medals</A></LI></DIV></DIV><DIV class="column full_size"><H1 class="subtitle text-center">Results</H1><H3> Click on each button on the image to be re-directed to a full results description for that section, or check out our results summary below!</H3><H1 class="subtitle text-center">Summary</H1><H2>Engineering P1 phage with SaCas9/Cpf1 system</H2><P>We engineered P1 phages with SaCs9 / Cpf1 systems, both of which target <EM>blaKPC</EM> and <EM>vanA</EM> genes. </P><H3>Key results:</H3><UL class="results"><LI>CRISPR cassettes (SaCas9/ FnCpf1) and spacer cassettes (one for <EM>bla</EM><EM>KPC</EM>, one for <EM>vanA</EM>) were successfully constructed within the P1 phagemid.</LI><LI>By using the P1 phagemid, we generated P1 phages with SaCas9 / FnCpf1 which target <EM>blaKPC</EM> and<EM> vanA</EM> genes. </LI><LI>We demonstrated that both P1 phage with SaCas9 and FnCpf1 system could cleave the target sequence. FnCpf1 with a very low efficiency whereas for SaCas9, the efficiency has to be further verified</LI></UL><DIV class="div-fig" style="width:400px;"><TABLE><TBODY><TR><TD><P>CRISPR system</P></TD><TD><P>Target gene</P></TD><TD><P>Ability to cleave the target sequence</P></TD></TR><TR><TD rowspan="2"><P>SaCas9 system</P></TD><TD><P><EM>bla</EM><EM>KPC</EM></P></TD><TD><P>✖</P></TD></TR><TR><TD><P><EM>vanA</EM></P></TD><TD><P>✔</P></TD></TR><TR><TD rowspan="2"><P>Cpf1 system</P></TD><TD><P><EM>bla</EM><EM>KPC</EM></P></TD><TD><P>✖</P></TD></TR><TR><TD><P><EM>vanA</EM></P></TD><TD><P>✔</P></TD></TR></TBODY></TABLE></DIV><H3>Future plan</H3><UL class="results"><LI>Repeat the experiments with more samples to confirm the efficiency of the SaCas9 system.</LI><LI>Add crRNA leader sequence upstream of the spacer cassette, to improve efficiency.</LI><LI>Test the complete two-phage system by infecting the re-sensitised bacteria with T7 and T4 separately.</LI></UL><H2>Engineering λ phage with SaCas9/Cpf1 system</H2><P>We attempted to engineer λ phages containing SaCas9 or Cpf1 systems, which target either<EM> blaKPC</EM> or<EM> vanA</EM></P><H3>Key results:</H3><UL class="results"><LI>No viable engineered λ were obtained for neither SaCas9 nor FnCpf1. This is likely due to the presence of active restriction systems in the <EM>E. coli </EM>strain used, or electroporation issues.</LI></UL><H3>Future plan:</H3><UL class="results"><LI>Optimise electroporation protocol, to introduce the the engineered λ into the testing <EM>E. coli.</EM></LI><LI>Use another <EM>E.coli </EM>strain (GM33) to electroporate engineered λ phage.</LI><LI>Test the complete two-phage system by infecting the re-sensitised bacteria with T7 and T4 separately.</LI></UL><H2>Engineering T4 phage</H2><P>Lytic wild-type T4 phage and T4-gene2 was engineered to contain the sequence which is targeted by CRISPR systems, based on the strategy called Bacteriophage Recombineering of Electroporated DNA (BRED) (Marinelli et al., 2008) (Figure 2). </P><H3>Key results:</H3><UL class="results"><LI>Both lysates show evidence of successful homologous recombination between KPC and VanA with T4 and T4-<EM>gene2</EM>. </LI><LI>Individual plaques for both strains were identified as having a mix of recombineered T4 and T4-<EM>gene2</EM> and non recombinants.</LI><LI>Due to time constraints, further purification of of T4 recombinants was not possible.</LI></UL><DIV class="div-fig" style="width:100%;"><P><STRONG>Figure 2: A: </STRONG>Shows the VanA and KPC region to be placed within T4, replacing the unessential gene SegC. To test for phages which had been successfully transformed the testing primers would be used with their corresponding flank primer.
			<STRONG>B: </STRONG>Shows correct size amplification within lysate 3 and 4 of the VanA/KPC region, indicating some recombinant T4-gene2 phages have been produced after BRED, T4=negative control, mm = master mix. <STRONG>C: </STRONG>Plaques produced from phages of Lysate 3, <STRONG>D:</STRONG> Plaque
			number 32 of the T4-gene2 plate has a mix of recombinant and non recombinant phages. <STRONG>E:</STRONG> An * stands for T4 lysates, lysate 1 and 2 show evidence of recombinants, <STRONG>F: </STRONG>Plaques produced from phages made in lysate 2, <STRONG>G:</STRONG> Plaque number 7 of the T4 plate has
			some recombinant phages containing VanA and KPC.</P></DIV><H3>Future plan</H3><UL class="results"><LI>Conduct PCR with the opposite primers to ensure the plaques identified as containing the recombineered T4 are showing the correct result.</LI><LI>Make a serial dilution with the recombinant plaques and re-plate. This should produce a plaque consisting of pure recombinant phages, which can be made into a lysate ready to be added to cells pre-infected with lysogenic phages.</LI><LI>Try out the system with the lysogenic bacteria, examining different parameters such as the period required before adding T4, the number of bacteria killed and methods to encapsulate it with a material that can degrade in water so it can be applied as
			a powder.</LI></UL><DIV class="div-ref"><P>J. Marinelli<EM> et al.</EM>, BRED: A Simple and Powerful Tool for Constructing Mutant and Recombinant Bacteriophage Genomes. <EM>Plos One</EM><STRONG>3</STRONG>, (2008).</P></DIV><H2>Engineering the T7 lytic phage</H2><P>We planned to engineer the T7 lytic phage to contain our protospacer chunks. We based this part of the work on research by Kiro <EM>et al. </EM>(2014, doi: 10.4161/rna.27766). This system requires the engineering of one strain of <EM>E. coli</EM> to
		contain a homologous recombination plasmid, and a second strain of <EM>E. coli</EM> to contain a CRISPR system.</P><H3>Key results:</H3><UL class="results"><LI><EM>E. coli </EM>TOP10 containing DVK_FG + T7 homology flanks + <EM>KPC</EM> were successfully constructed.</LI><LI>Production of <EM>E. coli </EM>TOP10 containing DVK_FG + T7 homology flanks + <EM>vanA</EM> was unsuccessful.</LI><LI>Construction of <EM>E. coli </EM>BL21 DE3 containing a CRISPR system and spacers against the T7 <EM>1.7</EM> gene was unsuccessful.</LI></UL><H3>Future plan:</H3><UL class="results"><LI>Alternative strategies for the creation of the CRISPR system should be tried - such as a different vector, longer spacers or different ligation conditions.</LI><LI>If these fail, an alternative selection system could used.</LI></UL><H2>Engineering E.coli testing platform</H2><P>To test our system, we created “mock pathogens” by engineering <EM>E. coli </EM>to contain the protospacers targeted by our CRISPR systems. We attempted to construct <EM>E. coli </EM>strains containing targets from <EM>KPC </EM>and<EM> vanA </EM>(our
		primary targets) as well as<EM> ampC </EM>(for possible future work). the parts were designed to create not-functional genes.</P><H3>Key results:</H3><UL class="results"><LI>Testing platform <EM>E. coli </EM>TOP10 containing <EM>KPC</EM> target sites were successfully created.</LI><LI>Testing platform <EM>E. coli </EM>TOP10 containing <EM>vanA</EM> target sites were successfully created.</LI><LI>Production of testing platform <EM>E. coli </EM>TOP10 containing <EM>ampC</EM> target sites was not successful.</LI><LI>The mock pathogen cells showed no increase in GFP expression when compared to a wild-type <EM>E. coli</EM> control.</LI></UL><H3>Future plan:</H3><UL class="results"><LI>More target genes can be engineered into the mock pathogens for additional proof-of-concept work, such as <EM>ampC</EM> and <EM>mecA.</EM></LI><LI>Analysis of protein production in the mock pathogens should be carried out to try to determine and overcome the lack of functional GFP.</LI></UL><H2>Modelling of two phage system</H2><P>To improve our understanding of bacteria-phage interactions, we developed a deterministic in silico model to show how populations of two phages with different life cycles interact when predating on bacteria in continuous and batch processes using a set
		of delayed differential equations. </P><H3>Key results:</H3><UL class="results"><LI>For the first time populations of lysogenic and temperate phages were modelled simultaneously in one system while predating on the same species of bacteria.</LI><LI>The entire model was written in Python,</LI><LI>Sensitivity analysis identified contribution of each parameter of the model towards removal of antibiotic resistant bacteria.</LI><LI>The time at which the second phage is added to the system had almost no effect on the total time it takes to remove all antibiotic resistant bacteria.</LI></UL><H3>Future plan:</H3><UL class="results"><LI>Development of a proper stochastic model, which then can be used to model interactions with small numbers of phage and bacteria particles involved.</LI><LI>Experimental identification of the initial parameters for the temperate phage.</LI><LI>Global sensitivity analysis to better investigate the interactions between different parameters of the model.</LI></UL></DIV></DIV></BODY></HTML>