<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><NAV class="collapse navbar-collapse navbar-right" role="navigation"><DIV class="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:Edinburgh_OG">Home</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Attributions">Attributions</A></LI></UL></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Applied_Design">Applied Design</A></LI></NAV></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Improve">Improve</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Gold_Integrated">Integrated Gold</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#call-to-action">Contact</A></LI><LI class="medals"><A href="https://2017.igem.org/Team:Edinburgh_OG/Medals">Medals</A></LI></DIV></DIV><DIV class="column full_size"><H1 class="subtitle text-center">Parts</H1><P>This year the iGEM Edinburgh_OG team focused on developing a modular toolkit using CRISPR systems and phages to re-sensitise antibiotic-resistant bacteria. As a BioBrick we submit the <EM>E. coli </EM>codon-optimised <EM>Staphylococcus aureus Cas9</EM>.</P><DIV class="highlight">&lt;groupparts&gt;iGEM17 Edinburgh_OG&lt;/groupparts&gt;</DIV><DIV class="div-fig" style="width:600px;"><P>http://www.nature.com/news/genome-editing-revolution-my-whirlwind-year-with-crispr-1.19063</P></DIV><H2>How does this part work?</H2><UL><LI>Our SaCas9 can be programmed to cleave specific target sequence followed by the PAM sequence (5’-NNGRRT-3’).</LI><LI>To express SaCas9, it requires suitable machinery such as promoter, RBS,and terminator.</LI><LI>To programme SaCas9, you need to design guide RNA (tracrRNA [2], 21 bp spacer flanked by direct repeats [2] ).</LI></UL><H3>Advantages of SaCas9 compared with the conventional <EM>Streptococcus pyogenes</EM> Cas9:</H3><UL><LI>Smaller size (1053 amino acids against 1368) resulting in an easier expression/delivery</LI><LI>Different PAM sequence recognised (5’-NNGRRT-3’ ) increasing the usability</LI><LI>Higher efficiency of SaCas9 over SpCas9 [2]</LI></UL><DIV class="div-ref"><P>[1] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., Zetsche, B., Shalem, O., Wu, X., Makarova, K. S., Koonin, E. V. Sharp, P.A., Zhang, F. 2015. In vivo genome editing using Staphylococcus aureus Cas9. Nature. 520 (7546).
    pp.186-191.</P><P>[2] Friedland AE, Baral R, Singhal P, et al. Characterization of Staphylococcus aureus Cas9: a smaller Cas9 for all-in-one adeno-associated virus delivery and paired nickase applications. Genome Biology. 2015;16:257. doi:10.1186/s13059-015-0817-8.</P></DIV></DIV></DIV></BODY></HTML>