Team:Edinburgh OG/Results
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Results
Click on each button on the image to be re-directed to a full results description for that section, or check out our results summary below!
Summary
Engineering P1 phage with SaCas9/Cpf1 system
We engineered P1 phages with SaCs9 / Cpf1 systems, both of which target blaKPC and vanA genes.
Key results:
CRISPR cassettes (SaCas9/ FnCpf1) and spacer cassettes (one for blaKPC, one for vanA) were successfully constructed within the P1 phagemid.
By using the P1 phagemid, we generated P1 phages with SaCas9 / FnCpf1 which target blaKPC and vanA genes.
We demonstrated that both P1 phage with SaCas9 and FnCpf1 system could cleave the target sequence. FnCpf1 with a very low efficiency whereas for SaCas9, the efficiency has to be further verified
CRISPR system
Target gene
Ability to cleave the target sequence
SaCas9 system
blaKPC
✖
vanA
✔
Cpf1 system
blaKPC
✖
vanA
✔
Future plan
Repeat the experiments with more samples to confirm the efficiency of the SaCas9 system.
Add crRNA leader sequence upstream of the spacer cassette, to improve efficiency.
Test the complete two-phage system by infecting the re-sensitised bacteria with T7 and T4 separately.
Engineering λ phage with SaCas9/Cpf1 system
We attempted to engineer λ phages containing SaCas9 or Cpf1 systems, which target either blaKPC or vanA
Key results:
No viable engineered λ were obtained for neither SaCas9 nor FnCpf1. This is likely due to the presence of active restriction systems in the E. coli strain used, or electroporation issues.
Future plan:
Optimise electroporation protocol, to introduce the the engineered λ into the testing E. coli.
Use another E.coli strain (GM33) to electroporate engineered λ phage.
Test the complete two-phage system by infecting the re-sensitised bacteria with T7 and T4 separately.
Engineering T4 phage
Lytic wild-type T4 phage and T4-gene2 was engineered to contain the sequence which is targeted by CRISPR systems, based on the strategy called Bacteriophage Recombineering of Electroporated DNA (BRED) (Marinelli et al., 2008) (Figure 2).
Key results:
Both lysates show evidence of successful homologous recombination between KPC and VanA with T4 and T4-gene2.
Individual plaques for both strains were identified as having a mix of recombineered T4 and T4-gene2 and non recombinants.
Due to time constraints, further purification of of T4 recombinants was not possible.
Figure 2: A: Shows the VanA and KPC region to be placed within T4, replacing the unessential gene SegC. To test for phages which had been successfully transformed the testing primers would be used with their corresponding flank primer. B: Shows correct size amplification within lysate 3 and 4 of the VanA/KPC region, indicating some recombinant T4-gene2 phages have been produced after BRED, T4=negative control, mm = master mix. C: Plaques produced from phages of Lysate 3, D: Plaque    number 32 of the T4-gene2 plate has a mix of recombinant and non recombinant phages. E: An * stands for T4 lysates, lysate 1 and 2 show evidence of recombinants, F: Plaques produced from phages made in lysate 2, G: Plaque number 7 of the T4 plate has    some recombinant phages containing VanA and KPC.
Future plan
Conduct PCR with the opposite primers to ensure the plaques identified as containing the recombineered T4 are showing the correct result.
Make a serial dilution with the recombinant plaques and re-plate. This should produce a plaque consisting of pure recombinant phages, which can be made into a lysate ready to be added to cells pre-infected with lysogenic phages.
Try out the system with the lysogenic bacteria, examining different parameters such as the period required before adding T4, the number of bacteria killed and methods to encapsulate it with a material that can degrade in water so it can be applied as    a powder.
J. Marinelli et al., BRED: A Simple and Powerful Tool for Constructing Mutant and Recombinant Bacteriophage Genomes. Plos One 3, (2008).
Engineering the T7 lytic phage
We planned to engineer the T7 lytic phage to contain our protospacer chunks. We based this part of the work on research by Kiro et al. (2014, doi: 10.4161/rna.27766). This system requires the engineering of one strain of E. coli to   contain a homologous recombination plasmid, and a second strain of E. coli to contain a CRISPR system.
Key results:
E. coli TOP10 containing DVK_FG + T7 homology flanks + KPC were successfully constructed.
Production of E. coli TOP10 containing DVK_FG + T7 homology flanks + vanA was unsuccessful.
Construction of E. coli BL21 DE3 containing a CRISPR system and spacers against the T7 1.7 gene was unsuccessful.
Future plan:
Alternative strategies for the creation of the CRISPR system should be tried - such as a different vector, longer spacers or different ligation conditions.
If these fail, an alternative selection system could used.
Engineering E.coli testing platform
To test our system, we created “mock pathogens” by engineering E. coli to contain the protospacers targeted by our CRISPR systems. We attempted to construct E. coli strains containing targets from KPC and vanA (our   primary targets) as well as ampC (for possible future work). the parts were designed to create not-functional genes.
Key results:
Testing platform E. coli TOP10 containing KPC target sites were successfully created.
Testing platform E. coli TOP10 containing vanA target sites were successfully created.
Production of testing platform E. coli TOP10 containing ampC target sites was not successful.
The mock pathogen cells showed no increase in GFP expression when compared to a wild-type E. coli control.
Future plan:
More target genes can be engineered into the mock pathogens for additional proof-of-concept work, such as ampC and mecA.
Analysis of protein production in the mock pathogens should be carried out to try to determine and overcome the lack of functional GFP.
Modelling of two phage system
To improve our understanding of bacteria-phage interactions, we developed a deterministic in silico model to show how populations of two phages with different life cycles interact when predating on bacteria in continuous and batch processes using a set   of delayed differential equations.
Key results:
For the first time populations of lysogenic and temperate phages were modelled simultaneously in one system while predating on the same species of bacteria.
The entire model was written in Python,
Sensitivity analysis identified contribution of each parameter of the model towards removal of antibiotic resistant bacteria.
The time at which the second phage is added to the system had almost no effect on the total time it takes to remove all antibiotic resistant bacteria.
Future plan:
Development of a proper stochastic model, which then can be used to model interactions with small numbers of phage and bacteria particles involved.
Experimental identification of the initial parameters for the temperate phage.
Global sensitivity analysis to better investigate the interactions between different parameters of the model.
