Team:Edinburgh OG/Parts
PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens
Toggle navigation
Home
Team
Project
Description
Results
Model
Experiments
Notebook
Collaborations
Interlab
Attributions
Design
Design
Applied Design
Parts
Basic Parts
Improve
Safety
Human Practices
Silver HP
Integrated Gold
Contact
Medals
Parts
This year the iGEM Edinburgh_OG team focused on developing a modular toolkit using CRISPR systems and phages to re-sensitise antibiotic-resistant bacteria. As a BioBrick we submit the E. coli codon-optimised Staphylococcus aureus Cas9.
<groupparts>iGEM17 Edinburgh_OG</groupparts>
http://www.nature.com/news/genome-editing-revolution-my-whirlwind-year-with-crispr-1.19063
How does this part work?
Our SaCas9 can be programmed to cleave specific target sequence followed by the PAM sequence (5’-NNGRRT-3’).
To express SaCas9, it requires suitable machinery such as promoter, RBS,and terminator.
To programme SaCas9, you need to design guide RNA (tracrRNA [2], 21 bp spacer flanked by direct repeats [2] ).
Advantages of SaCas9 compared with the conventional Streptococcus pyogenes Cas9:
Smaller size (1053 amino acids against 1368) resulting in an easier expression/delivery
Different PAM sequence recognised (5’-NNGRRT-3’ ) increasing the usability
Higher efficiency of SaCas9 over SpCas9 [2]
[1] Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., Zetsche, B., Shalem, O., Wu, X., Makarova, K. S., Koonin, E. V. Sharp, P.A., Zhang, F. 2015. In vivo genome editing using Staphylococcus aureus Cas9. Nature. 520 (7546).     pp.186-191.
[2] Friedland AE, Baral R, Singhal P, et al. Characterization of Staphylococcus aureus Cas9: a smaller Cas9 for all-in-one adeno-associated virus delivery and paired nickase applications. Genome Biology. 2015;16:257. doi:10.1186/s13059-015-0817-8.
