<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>PhagED: a molecular toolkit to re-sensitise ESKAPE pathogens</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><NAV class="collapse navbar-collapse navbar-right" role="navigation"><DIV class="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:Edinburgh_OG">Home</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Attributions">Attributions</A></LI></UL></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Applied_Design">Applied Design</A></LI></NAV></DIV><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Improve">Improve</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_OG/HP/Gold_Integrated">Integrated Gold</A></LI></DIV><LI><A href="https://2017.igem.org/Team:Edinburgh_OG#call-to-action">Contact</A></LI><LI class="medals"><A href="https://2017.igem.org/Team:Edinburgh_OG/Medals">Medals</A></LI></DIV></DIV><DIV class="column full_size"><H1 class="subtitle text-center">Description</H1><P>The threat posed by antibiotic resistant bacteria is a pressing issue which must be addressed. It is difficult and expensive to develop new antibiotics, so our project is designed to make currently available antibiotics useful again. Our aim is to create
    a toolkit to re-sensitise pathogens to antibiotics using CRISPR and a two-phage system, based on work by <STRONG>Yosef et al. (2015, doi:10.1073/pnas.1500107112)</STRONG>. An engineered lysogenic phage will transfer a CRISPR system to its host bacterium,
    designed to cleave resistance genes and also confer protection from an engineered lytic phage. When this lytic phage is added to the population, it kills any bacteria that have not been successfully re-sensitised. We chose to target genes found in
    the highly-resistant ESKAPE pathogens, and worked with 4 different phages - <STRONG>P1, λ (lambda), T4</STRONG> and <STRONG>T7.</STRONG> Our system was modelled in silico and tested empirically on a specially designed <EM>E. coli </EM>testing platform.</P><H2>Antibiotic Resistance</H2><P>Antibiotic-resistant bacteria pose an enormous threat to our current way of life. Antibiotic resistance already causes over <STRONG>700,000 deaths per year</STRONG>, and this number is expected to rise to a terrifying 10 million per year by 2050 (Figure 1) (O’Neill,
    2014). A solution to this problem is urgently needed and we believe that synthetic biology could be the key, which is why we chose to tackle antibiotic resistance with our iGEM project.</P><P>Antibiotic resistance can arise in a number of ways, such as spontaneous mutations or horizontal gene transfer. When a population is exposed to antibiotics, susceptible individuals are killed and any resistant bacteria survive to repopulate the niche.
    Every time a population is exposed to an antibiotic, a selection pressure is applied and resistant bacteria <STRONG>dominate</STRONG> the population (Levy, 1998).</P><P>Two of the major factors contributing to the rise of resistant bacteria are the overuse and misuse of antibiotics. Examples of overuse include the <STRONG>prescription</STRONG> of antibiotics in cases where the infection is minor and will clear up without medical intervention
    (Levy, 1998). Antibiotics are also heavily overused in <STRONG>agricultural</STRONG> settings. A staggering 80% of antibiotics purchased in America are used on livestock (Bartlett <EM>et al.</EM>, 2013), encouraging the growth of resistant bacteria and risking their
    spread through the food chain (Wright, 2010). Misuse includes the prescription of antibiotics for viral infections, and patients stopping their course of antibiotics before the recommended time (Levy, 1998).</P><P>Despite the ever-increasing number of antibiotics that are no longer effective against many pathogens, there is a <STRONG>lack of new antibiotics</STRONG> being developed. This is because development is incredibly costly (Sukkar, 2013) and there’s no guarantee
    that pharmaceutical companies will recoup these costs through profit, as resistance can arise in a matter of months (Bartlett <EM>et al., </EM>2013). Even if new antibiotics are developed, they can only stave off the antibiotic resistance crisis for
    so long. Instead,<STRONG> alternative strategies</STRONG> are required.</P><DIV class="div-fig" style="width:600px"><P><STRONG>Figure 1: </STRONG>The number of deaths currently caused by and predicted to be caused by AMR in the future, compared to other significant causes of death. Figure from O’Neill, 2014.</P></DIV><H2>Bacteriophages</H2><P>Bacteriophages (phages) are the most abundant life-form on the planet, with an estimated population of 10^32. Phages are viruses that infect bacteria, and they are incredibly <STRONG>specific</STRONG>– each species of phage only infects particular species or even
    strains of bacteria (Wittebole <EM>et al., </EM>2014).</P><P>Typical phage structure consists of an icosahedral “head” and a cylindrical “tail” (Figure 2). The phages DNA – which can be single or double stranded – is packaged within the head. Upon recognition of specific receptors
    on the host cell surface, the phage <STRONG>injects its DNA</STRONG> into the host bacterium via the tail structure (Montag <EM>et al., </EM>1987; Klug &amp; Huxley, 1971). Once inside the bacterium, the phage enters one of two possible lifecycles.</P><P>During the <STRONG>lysogenic cycle</STRONG>, the phage genome lies dormant and is replicated along with the host DNA and spreading to bacterial daughter cells. The phage genome may integrate into the host genome (as in λ phages) or form a separate plasmid (as
    in P1 phages) (Wittebole <EM>et al., </EM>2014).</P><P>The change from the lysogenic to the<STRONG> lytic cycle</STRONG> may be triggered by an external stimulus such as temperature or UV light, but some phages experience only the lytic cycle without the lysogenic. In the lytic stage, phages hijack the host’s cell
    machinery and resources and replicate themselves rapidly until they reach a critical population size – prompting the <STRONG>lysis of the host bacterium</STRONG> (Wittebole <EM>et al.</EM>, 2014).</P><DIV class="div-fig" style="width:400px"><P><STRONG>Figure 2: </STRONG>The structure of a T4 bacteriophage. Figure from Miller <EM>et al.</EM>, 2003.</P></DIV><H2>Bacteriophages &amp; Phage Therapy</H2><P>Lytic bacteriophages have been used as a treatment against bacterial infections since the early 20th century, but they fell out of fashion by the 1940s when antibiotics were invented and the comparable efficacy of phages was called into question (Sulakvelidze
    et al., 2001). So-called <STRONG>“phage therapy”</STRONG> involves the <EM>in vivo </EM>use of phages to kill pathogenic bacteria. Phage therapy is advantageous over broad spectrum antibiotics in several ways. Its specificity for particular bacteria reduces
    the negative impact of treatment on the patient microbiome (Drulis-Kawa <EM>et al.,</EM> 2012), and studies have shown that resistance to phages is less likely to develop than antibiotic resistance (Nobrega <EM>et al.</EM>, 2015).</P><P>However, there are also a number of difficulties with phage therapy. <STRONG>Current legislation</STRONG> is not set up to deal with administration of phages to patients – it is based upon antibiotics with detailed structures and known activities but many phages
    are poorly characterised and their behaviour hard to predict (Cooper, Mirzaei and Nilsson, 2016). The <EM>in vivo </EM>nature of phage therapy comes with the risk of the patient recognising the phages as foreign and removing them before they can kill the pathogen (Wittebole <EM>et al., </EM>2014), or the triggering of an aggressive
    immune response which can harm the patient (Pires <EM>et al., </EM>2016). To overcome these problems, an <STRONG><EM>ex vivo </EM></STRONG>approach should be considered.</P><H2>Bacteriophages &amp; CRISPR</H2><P><STRONG>CRISPR</STRONG> (Clustered Regularly Interspaced Short Palindromic Repeat) systems and <STRONG>CRISPR-associated (Cas) proteins </STRONG>are a component of the bacterial adaptive immune system, providing protection against foreign DNA such as that from bacteriophages (Qi <EM>et al., </EM>2014).</P><P>The CRISPR array contains palindromic DNA repeat sequences which flank <STRONG>‘spacers’–</STRONG> regions of DNA which have been associated with foreign bacteriophage DNA (Barrangou <EM>et al., </EM>2007). The array becomes transcribed into mRNA
    to produce <STRONG>‘guide RNA’ (gRNA),</STRONG> which is associated with an endonuclease such as Cas9 (Bikard <EM>et al., </EM>2012). If DNA containing the homologous sequence to the spacer (referred to as a <STRONG>‘protospacer’</STRONG>, normally 20-30 nt)
    appears in the cell, the gRNA will guide the Cas9 to the foreign DNA and bind to the complementary site. The Cas9 will then cleave and destroy the foreign DNA if there is a 2-6 bp <STRONG>protospacer adjacent motif (PAM)</STRONG> sequence next to the protospacer site (Figure 3)
    (Barrangou <EM>et al., </EM>2007). Spacers can be engineered to target almost any part of the genome - including antibiotic resistance genes.</P><P>Somewhat ironically, CRISPR systems targeting antibiotic resistance genes have been recently integrated into lysogenic bacteriophages. When the engineered phage genome is injected into the host, the CRISPR system becomes active and will target the desired
    genes. Studies by <STRONG>Citorik <EM>et al.</EM> (2014)</STRONG> and <STRONG>Bikard <EM>et al.</EM> (2014)</STRONG> successfully demonstrated the use of this system to re-sensitise pathogens to antibiotics by cleaving plasmid-mediated resistance genes. One weakness of this single-phage
    system is that any bacteria that are resistant to the lysogenic phage will have a selective advantage over their re-sensitised counterparts. Antibiotic treatment will result in the phage-resistant bacteria becoming dominant in the population, in the
    same manner as antibiotic-resistant bacteria.</P><P>To overcome this selection pressure for resistance,<STRONG> Yosef <EM>et al. </EM>(2015) </STRONG>developed a dual phage system. After the initial re-sensitisation by the lysogenic CRISPR-containing phage, the bacteria are then exposed to a lytic phage. This lytic phage
    is engineered to contain the same resistance gene protospacers that are targeted by the CRISPR system, meaning that any bacteria that have been successfully re-sensitised will also be <STRONG>immune to infection</STRONG> by the lytic phage. The system therefore provides
    a <STRONG>selective advantage to re-sensitised bacteria</STRONG>, reducing the likelihood that resistance to the lysogenic phage will develop.</P><DIV class="div-fig" style="width:600px"><P><STRONG>Figure 3: </STRONG>CRISPR-Cas results in  double stranded break at the site recognized by guide RNA. Figure taken from the abm website.</P></DIV><H1 class="subtitle text-center">Our Project</H1><H2>The Aim</H2><P>We based our project on the work of Yosef <EM>et al.</EM> (2015), and aimed to expand their ideas to create a<STRONG> toolkit for bacterial re-sensitisation. </STRONG>We planned to use two species of lysogenic phage, two species of lytic phage, three CRISPR systems
    and two target genes (Figure 4).</P><DIV class="div-fig" style="width:800px"><P><STRONG>Figure 4: A:</STRONG> We engineered lysogenic P1 /  λ (P1 in this example) to contain either a CRISPR-SaCas9, FnCpf1 or SpCas9 system containing
    spacers of <EM>VanA</EM> and <EM>KPC </EM>(<EM>VanA</EM> in this example), and a lytic T4 / T7 (T4 in this example) phage containing identical spacer sequences for targeting by the same CRISPR system. <STRONG>B:</STRONG> Cells containing  plasmids with gene regions of <EM>VanA</EM> and <EM>KPC</EM>  were mixed with engineered lysogenic P1 phages, triggering destruction of the  plasmid by CRISPR-SaCas9/Cpf1 (SaCas9/Cpf1 = blue, CRISPR = orange hairpins, black = <EM>VanA/KPC </EM>plasmid, cyan = lysogenic P1 phage DNA, purple = lytic T4 phage DNA). <STRONG>C:</STRONG> CRISPR-SaCas9/Cpf1 is continually produced, thus cells could destroy injected T4 DNA and survive infection. <STRONG>D:</STRONG> Cells resistant to lysogenic phages or avoided infection by chance are killed by lytic T4. Thus, only cells with no antibiotic
    resistance survive. Figure adapted from (Pires <EM>et al., </EM>2016)</P></DIV><H2>The Phages</H2><P>Because we planned to use an <EM>E. coli</EM> testing platform, we chose phages that were specific for <EM>E. coli</EM>. Our lysogenic phages were λ and P1, and our lytic phages were T4 and T7. We chose the λ and T7 phages as these were previously used successfully by Yosef <EM>et al.</EM> (2015), so we knew it was possible to create a working system with these phages.</P><P>P1 was chosen as it is a <STRONG>well-characterised phage</STRONG>, with a high efficiency of packaging and more than twice the head capacity of λ (Trempy <EM>et al.</EM>, 2004). Any cells lysogenised with P1 gain resistance to T7 (Goodgal, 1956). In an ideal
    scenario this would not be a problem, as all lysogenised cells should contain a CRISPR system and thus should be resistant to T7 anyway. However, in practice it is possible for a cell to be lysogenised but for the CRISPR system to be <STRONG>non-functional</STRONG>,
    so that the cell retains antibiotic resistance but cannot be killed by T7. For this reason, we chose an additional lytic phage – T4.</P><P>Like T7, T4 is a purely lytic phage with no lysogenic stage (Miller <EM>et al., </EM>2003). T7 and T4 are both well-characterised model species, though to our knowledge T4 has <STRONG>not previously been engineered using BRED</STRONG> (Bacteriophage Recombineering of
    Electroporated DNA).</P><H2>The Targets</H2><P>We decided to target two genes, which together allow us to target all members of the prominent <STRONG>ESKAPE group</STRONG> of resistant pathogens. This group has been singled out as the most important players in nosocomial infections, due to their ability to 'escape' the effects of antibiotics. The ESKAPE group consists of E<EM>nterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, </EM>and<EM> Enterobacter </EM>spp
    (Rice, 2008).</P><P>The <STRONG><EM>bla</EM><EM>KPC</EM> (<EM>KPC)</EM></STRONG> gene codes for <EM>Klebsiella pneumoniae </EM>carbapenemases, and has been found in a number of Gram negative pathogens including the 4 in the ESKAPE group (KAPE) (Arnold <EM>et al., </EM>2011; Robledo <EM>et al., </EM>2011;
    Marchaim <EM>et al., </EM>2008). <EM>KPC</EM> confers resistance to a number of β-lactam antibiotics including the carbapenems, which had previously been relied upon as a <STRONG>“last resort”</STRONG> antibiotic for use when others fail (Tzouvelekis,
    2012). Similarly to the other ESKAPE pathogens, <EM>KPC</EM> has a large global distribution (Figure 5). </P><P>The <STRONG><EM>vanA</EM></STRONG> gene confers resistance to glycopeptide drugs such as vancomycin, and is commonly associated with the Gram positive species vancomycin-resistant <EM>Staphylococcus aureus</EM> (VRSA) and vancomycin-resistant enterococci (VRE) –
    the ES of ESKAPE (Willems <EM>et al., </EM>2005; Haenni <EM>et al., </EM>2009).</P><P>We also identified<STRONG><EM>mecA</EM></STRONG> (associated with methicillin-resistant <EM>Staphylococcus aureus, </EM>MRSA) (Wielders <EM>et al., </EM>2002) and <STRONG><EM>ampC </EM></STRONG>(a β-lactamase gene found in Gram negative ESKAPE pathogens) (Jacoby, 2009) as potential
    targets for future expansion of the toolkit.</P><DIV class="div-fig" style="width:600px"><P><STRONG>Figure 5:</STRONG> Global spread of KPC, since publication it has also been discovered in Africa. Figure from (Nordmann 2011).</P></DIV><H2>The CRISPR</H2><P>We chose to use <STRONG>three CRISPR systems</STRONG> in our project: the cas9 systems from <EM>Staphylococcus aureus</EM> (SaCas9) and <EM>Streptococcus pyogenes </EM>(SpCas9) and the Cpf1 system from <EM>Francisella novicida </EM>(FnCpf1). Due to IP issues with our
    collaborators, the SpCas9 work will not be discussed.</P><P>The SaCas9 protein is only 1053 amino acids in length – much smaller than both Cpf1 and SpCas9 (SaCas9’s better-known <EM>Streptococcus pyogenes</EM> equivalent) (Friedland <EM>et al., 2015)</EM>. Its small size is advantageous to us, as
    bacteriophages are <STRONG>limited</STRONG> in terms of the amount of DNA that can be packaged in their heads. SaCas9 is a type II CRISPR system, and works as described above. Its PAM sequence is G-rich and located downstream to the protospacer recognition site (Fagerlund
    <EM>et al., </EM>2015).</P><P>FnCpf1 is a type V CRISPR system, so differs slightly from SaCas9 in a number of respects. It leaves a staggered cut rather than a blunt double-stranded break (Doudna, 2015) and has a T-rich PAM located upstream to the protospacer recognition site (Fagerlund
    <EM>et al., </EM>2015). The differing PAMs of our three CRISPR systems <STRONG>expand the range of potential target sequences</STRONG>. The FnCpf1 system does not require a tracrRNA and has a much shorter crRNA than its cas9 counterparts, making it cheaper and simpler
    to produce (Fagerlund <EM>et al.,</EM> 2015).</P><H2>References</H2><DIV class="div-ref"><P>abm, https://www.abmgood.com/CRISPR-Cas9-Nuclease-Nickase-Vectors-Viruses.html, Accessed 31/10/17.</P><P>Arnold, R.S. et al., Emergence of Klebsiella pneumoniae Carbapenemase-Producing Bacteria. Southern Medical Journal 104, 40-45 (2011).</P><P>Barrangou, R. <EM>et al.</EM>, CRISPR provides acquired resistance against viruses in prokaryotes. <EM>Science </EM><STRONG>315</STRONG>, 1709-1712 (2007).</P><P>Bartlett, J.G., Gilbert, D.N., Spellberg, B. Seven Ways to Preserve the Miracle of Antibiotics. <EM>Clinical Infectious Diseases </EM><STRONG>56</STRONG>, 1445-1450 (2013).</P><P>Bikard, D., Hatoum-Aslan, A., Mucida, D., Marraffini, L.A., CRISPR Interference Can Prevent Natural Transformation and Virulence Acquisition during In Vivo Bacterial Infection. Cell Host &amp; Microbe 12, 177-186 (2012).</P><P>Bikard, D. <EM>et al. </EM>Development of sequence-specific antimicrobials based on programmable CRISPR-Cas nucleases. <EM>Nat. Biotechnol. </EM><STRONG>32, </STRONG>1146–1150 (2014).</P><P>Citorik, R. J., Mimee, M. &amp; Lu, T. K. Sequence-specific antimicrobials using efficiently delivered RNA-guided nucleases. <EM>Nat. Biotechnol. </EM><STRONG>32, </STRONG>1141–1145 (2014).</P><P>Cooper, C. J., Mirzaei, M. K. and Nilsson, A. S. (2016) ‘Adapting drug approval pathways for bacteriophage-based therapeutics’, Frontiers in Microbiology. Frontiers Media SA, p. 1209. doi: 10.3389/fmicb.2016.01209.</P><P>Doudna, J. (2015). Genome-editing revolution: My whirlwind year with CRISPR. Nature, 528(7583), 469-71.</P><P>Drulis-Kawa, Z., Majkowska-Skrobek, G., Maciejewska, B., Delattre, A.S., Lavigne, R. Learning from Bacteriophages - Advantages and Limitations of Phage and Phage-Encoded Protein Applications. Current Protein &amp; Peptide Science 13, 699-722 (2012).</P><P>Fagerlund, R. D., Staals, R. H., Fineran, P. C. 2015. The Cpf1 CRISPR-Cas protein expands genome-editing tools. <EM>Genome biology</EM>. 16(1). pp. 251-253.</P><P>Friedland, A. E., Baral, R., Singhal, P., Loveluck, K., Shen, S., Sanchez, M., ... &amp; Kornepati, A. V. 2015. Characterization of Staphylococcus aureus Cas9: a smaller Cas9 for all-in-one adeno-associated virus delivery and paired nickase applications. <EM>Genome biology</EM>. 16(1). pp. 257- 268.</P><P>Goodgal, S. H. Incorporation of Phage DNA on Lysogenization of Shigella Dysenteriae. <EM>Biochim. Biophys. Acta </EM><STRONG>19, </STRONG>333–340 (1956).</P><P>Haenni, M., et al. vanA in Enterococcus faecium, Enterococcus faecalis, and Enterococcus casseliflavus Detected in French Cattle. Foodborne Pathogens and Disease 6, 1107-1111 (2009).</P><P>Jacoby, G. A. AmpC β-Lactamases. Clin. Microbiol. Rev. 22, 161–182 (2009).</P><P>Klug, H.E., Huxley, A. A Discussion on new developments in electron microscopy with special emphasis on their application in biology - Application of optical diffraction to helical structures in the bacteriophage tail. - Philosophical Transactions of the Royal Society of London. B, Biological Sciences, 181. (1971).</P><P>Levy, S. B. The Challenge of Antibiotic Resistance. <EM>Sci. Am. </EM>46–53 (1998).</P><P>Marchaim, D., Navon-Venezia, S., Schwaber, M.J., Carmeli, Y. Isolation of imipenem-resistant Enterobacter species: Emergence of KPC-2 carbapenemase, molecular characterization, epidemiology, and outcomes. Antimicrobial Agents and Chemotherapy 52, 1413-1418 (2008).</P><P>Miller, E.S. <EM>et al.</EM>, Bacteriophage T4 genome. <EM>Microbiology and Molecular Biology Reviews </EM>67, 86 (2003).</P><P>Montag, D., Riede, I., Eschbach, M.L., Degen, M., Henning, U. Receptor-Recognizing Proteins Of T-Even Type Bacteriophage - Constant And Hypervariable Regions And An Unusual Case Of Evolution. Journal of Molecular Biology 196, 165-174 (1987).</P><P>Nobrega, F.L., Costa, A.R., Kluskens, L.D., Azeredo, J. Revisiting phage therapy: new applications for old resources. Trends in Microbiology 23, 185-191 (2015).</P><P>Nordmann, P., Naas, t., Poirel, L., Global Spread of Carbapenemase-producing Enterobacteriaceae. Emerging Infectious Diseases 17, 1791-1798 (2011)</P><P>O’Neill, J. <EM>Review on Antimicrobial Resistance: Antimicrobial Resistance: Tackling a Crisis for the Health and Wealth of Nations</EM>. London: Review on Antimicrobial Resistance. (2014).</P><P>O’Neill, J. <EM>Tackling drug-resistant infections globally: final report and recommendations</EM>, <EM>The Review on Antimicrobial Resistance</EM>. London: Review on Antimicrobial Resistance. doi: 10.1016/j.jpha.2015.11.005. (2016).</P><P>Pires, P. B. et al. Genetically Engineered Phages: a Review of Advances over the Last Decade, American Society for Microbiology 80, 523-543 (2016).</P><P>Qi, L. S. et al. Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Cell 152, 1173–1183 (2013).</P><P>Rice, L.B. Federal funding for the study of antimicrobial resistance in nosocomial pathogens: No ESKAPE. Journal of Infectious Diseases 197, 1079-1081 (2008).</P><P>Robledo, I.E., Aquino, E.E., Vazquez, G.J. Detection of the KPC Gene in Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii during a PCR-Based Nosocomial Surveillance Study in Puerto Rico. Antimicrobial Agents and Chemotherapy 55, 2968-2970 (2011).</P><P>Sukkar, E. 2013. Why are there so few antibiotics in the research and development pipeline? <EM>The Pharmaceutical Journal</EM>. 291(13). Pp.520.</P><P>Sulakvelidze, A. et al. (2001) ‘Bacteriophage therapy.’, Antimicrobial agents and chemotherapy. American Society for Microbiology (ASM), 45(3), pp. 649–59. doi: 10.1128/AAC.45.3.649-659.2001.</P><P>Trempy, J, E., Trun, N. 2004. Chapter 7 Bacteriophage. In: Trempy, J, E., Trun, N<EM>. eds. Fundamental Bacterial Genetics</EM>. Blackwell Science Ltd. pp.105-125.</P><P>Tzouvelekis, L.S., Markogiannakis, A., Psichogiou, M., Tassios, P.T., Daikos, G.L. Carbapenemases in Klebsiella pneumoniae and Other Enterobacteriaceae: an Evolving Crisis of Global Dimensions. Clinical Microbiology Reviews 25, 682-+ (2012).</P><P>Wielders, C. L. C., Fluit, A. C., Brisse, S., Verhoef, J. &amp; Schmitz, F. J. mecA Gene Is Widely Disseminated in Staphylococcus aureus Population. J. Clin. Microbiol. 40, 3970–3975 (2002).</P><P>Willems, R.J.L., et al. Global spread of vancomycin-resistant Enterococcus faecium from distinct nosocomial genetic complex. Emerging Infectious Diseases 11, 821-828 (2005).</P><P>Wittebole, X., Roock, S. De &amp; Opal, S. M. A historical overview of bacteriophage therapy as an alternative to antibiotics for the treatment of bacterial pathogens. <EM>Virulence </EM><STRONG>5, </STRONG>226–235 (2014).</P><P>Wright, G.D. Antibiotic resistance in the environment: a link to the clinic? <EM>Current Opinion in Microbiology </EM><STRONG>13</STRONG>, 589-594 (2010).</P><P>Yosef, I., Manor, M., Kiro, R. &amp; Qimron, U. Temperate and lytic bacteriophages programmed to sensitize and kill antibiotic-resistant bacteria. <EM>Proc. Natl. Acad. Sci. </EM> 112, 7267–7272 (2015). </P></DIV></DIV></DIV></BODY></HTML>