"
Page
Discussion
View source
History
teams
Log in
 
Team:NCTU Formosa/WetLab/Results new
From 2009.igem.org
Home
Team
Team
Gallery
Project
Abstract
Introduction
Design
Parts
New Idea
Safety
Results
Lab works
Timer
Counter
Memory
New Idea
Modeling
Notebook
Calendar
Submitted Parts
Contact
A simple and rapid protocol to generate a promoter library with different transcriptional strength
protocol
This protocol contains four steps:
Degenerated primers design.
Using PCR to amplify the target plasmid.
PCR product transformation.
The measurement of promoter activity.
1. Degenerated primers design
At first, it is required to design a primer. It is no matter who has experiences about designing. Just have a check the points below when you want design your primer.
Normally, primer size is 25~45mer. Our design depends on the Tm value and GC ratio, so 30~40mer length is recommended.
The most important thing is that you have to check the Tm value, more than 78℃ or not. (At least more than 40％ of GC ratio.)
If the Tm value is under 78℃, it is necessary to change the primer length.
Design two strands, forward and reverse primers. In this step, locate the target nucleotide on the center of primer.
Avoid desalting grade. Must use over than minimum FPLC or OPC grade.
A pair of random mutation primers for the biobrick: BBa_K145279 is made as an example. In this case, we wanted to generate different GFP intensity of the promoter PtetR, so two point mutations were made in the -10 box (Table 1).
Table 1. The degenerated primers designed to amplify the biobrick plasmid: BBa_K145279. Yellow bars indicate the -10 boxes and N is a random mutation (A, T, C, G) in the primer.
2. Using PCR to amplify the target plasmid.
PCR Reaction Mixture Set Up
* It is important to use KOD-plus polymerase. Since DNA polymerase from Thermococcus kodakaraensis KOD is one of the most efficient thermostable PCR enzymes exhibiting higher accuracy and elongation velocity than any other commercially available DNA polymerase.
PCR conditions
The linear PCR product can join by self-ligation that can increase the transformation efficiency.
Ligation conditions
(1) Vortex the tube and spin down in a microcentrifuge for 3-5 seconds.
(2) Incubate the mixture for 8 hour at 16°C.
(3) Use the mixture for transformation.
3. PCR product transformation
ECOSTM competent cells are used for transformation.
Step:
Thaw competent cells in the room temperature water bath with circulating water or hold the tube under the running tap water for ~20 seconds until 1/3~1/2 volume is thawed.
Add DNA(pre-chilled on ice, volume should be ≦ 5% of competent cells) immediately. Vortex for 1 second or tap the tube with finger to mix well.
Heat-shock the cells in the pre-warmed 42℃ water bath for 30~45 seconds.
Plate the cells using plating beads onto a pre-chilled(4℃) and dried antibiotic-selected LB agar plate.
Incubate the plates at 37℃.
The measurement of promoter activity.
We assayed promoter activity using GFP reporter plasmid (BBa_K145279 ). Bacteria were grown to steady-stated, then 50 μM aTc was added to the media, and the cultures were grown with aTc-inducer. After 90 minutes treated, we started to measure the sample. All fluorescence measurements were performed on a flow cytometer equipped with a 488 nm argon excitation laser and a 515–545 nm emissions filter. For each sample, 100,000 events were collected. A small gate in the side-scatter and forward-scatter space was chosen to reduce the variations in cell size and help remove noise in fluorescence measurements. Fluorescence intensities were converted to molecules of equivalent fluoresces in based on daily measurements of SPHERO Rainbow Calibration Particles. First of all, we selected fourteen different colonies and one wild type colony as our samples and control. We loose one sample (Rm05), so the final data only fourteen different data as the result we showed below.
Fig. 2. Green fluorescence intensity of the mutated promoter PtetR. Mutated promoter activity from the low (A: Rm07) to the high (B:Rm14) state by aTc induction.
Fig. 3. Green fluorescence intensity was measured from the promoter PtetR with mutations. RmNo. indicates Random Mutation number of mutated promoter. Within this two point mutations group, we observed varied promoter strength about one decade.
Results and discussions
In our experiments, 14 colons were selected randomly, and more than 90% of the clones have different promoter activities. We assayed promoter activity by using a reporter gene, green fluorescent protein. Here, we reported all promoter activities in terms of MELF.
We measured the expressions of 14 transformants in each combination of the activator anhydrotetracycline (aTc). 14 colons are incubated in M9 media (Difco) and used glucose as the carbon source. After the broth is incubated at 37℃ overnight,  50 μM aTc is added to the media and measured the GFP after 90 minutes. The activator aTc regulates the construct mutant devices. It consists if a TetR generator and a GFP under the control of a TetR regulated promoter. Normally the production of GFP is switched off. It can be switched on by the addition of tetracycline (or aTc).
The result showed that there are low (Rm03, Rm11, Rm01, Rm06, Rm02, Rm04, Rm10, Rm09), medium-low (Rm13, Rm07) and high expressions (Rm08, Rm14, Rm12) in Green fluorescence intensity (Fig. 3). The low expressions are about 0.27 to 0.33 times than original promoter, and the medium-low expressions are about 0.47 to 0.52 times and high expressions are over 1.41 times.
To consider the combinations are drove in the assembly vectors, which are high copy number plasmids. If the device is high expression, there will harmful to the cultures growth. Actuality, we also observed the growth of high expression combinations, Rm 08 and Rm12, is much sluggish than original device. We chose the two medium-low combinations, Rm07 and Rm13 to measure the GFP expressions. We cultured the Rm07 and Rm13 in M9 media. After the optical density ( O.D.) of the media were above 0.1 then added 50 μM aTc to the media. The time added the aTc as the outset. Every 20 minutes collected the media once and measured the GFP. 100,000 events were collected for each measurement. Both Rm07 and Rm13 were detected the GFP. MEFL of Rm07 vibrated between 20,000 to 27,000. MEFL of Rm13 vibrated between 15,000 to 20,000. Though these two devices are categorized the group of medium-low expression, the GFP strength of Rm07 is slightly over Rm13(Fig. 4).
Fig.4. Green fluorescence intensity was measured in Rm07 and Rm 13, two medium-low expressions in green fluorescence.
Sequencing two devices Rm07 and Rm13, we obtain the diverseness at TATA box -10 sequences (Table. 2). Rm07 has one site distinct from promoter tetR sequences. The nucleotide t is substituted for c. Rm13 has both two sites distinct from promoter tetR sequences. The nucleotides aa are substituted for ct. These changes make the process of transcription become difference. The RNA polymerase recognizes the TATA box in particular sequences for starting transcription. When we change the TATA box at the particular sites, it will affect the efficiency of the RNA polymerase binding affinity, causing the different rates of downstream gene expression, making variations in GFP producing and expressing.
Table.2 The sequences of original promoter tetR and the mutants Rm07 and Rm13. The red ground color are the random mutant sites
Conclusions
One of the main issues encountered in designing genetic circuits is the ability to match the biological parts into combinations that result in the design specifications. How to get the biological parts to fit the design specifications?  In this project, we create the new protocol for constructing the variant promoter intensity. A library of promoters with different transcriptional strength can be built to tune the specific parameter values that model equations indicated. The multi-faced characterization of promoter strength enabled identification of optimal expression levels.
References
Cox, R. S., Surette, M. G., and Elowitz, M. B., Programming gene expression with combinatorial promoters. Molecular Systems Biology 3 (2007)
Hammer, K., Mijakovic, I., and Jensen, P. R., Synthetic promoter libraries - tuning of gene expression. Trends in Biotechnology 24 (2), 53 (2006)
Hooshangi, S., Thiberge, S., and Weiss, R., Ultrasensitivity and noise propagation in a synthetic transcriptional cascade. Proceedings of the National Academy of Sciences of the United States of America 102 (10), 3581 (2005).
Retrieved from " http://2009.igem.org/Team:NCTU_Formosa/WetLab/Results_new "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
