Team:Worldshaper-XSHS/Protocols-methods.html
iGEMwiki toolssearchteamslogin
Editing Team:Worldshaper-XSHS/Protocols-methods.html
Toggle navigation
Worldshaper- XSHS
Team
about us
Members
Collaborations
Project
Background
Design
Protocols and methods
Results
Future plans
Interlab
Model
Demonstrate
Parts
HP
Silver
Integrated and Gold
Engagement
Applied Design
Achievement
Medal requirements
Safety
Attributions
Protocols-methods
The main purpose of our experiment
Our main works
The Construction of Plasmids
Sensitivity detection
Appendix
Experimental system
TO Top
The main purpose of our experiment is as follows:
1 Screening of Nicotine Inductive Promoter and Construction of the First Version of Nicotine Monitoring System
①The construction of plasmids pSB1C3-Pnox + RBS + GFP + ter，pSB1C3-Pnox + RBS + BFP，pSB1C3-Pnox + RBS + RFP + ter，pSB1C3-Pnox.
②The construction of plasmids pMD-Pnica1 + RBS + GFP + ter，pMD-Pnica2 + RBS + GFP + ter.
2  The Construction of the Second Version of Nicotine Monitoring System                             The construction of plasmid pMD-Pnica2 + RBS + T7RNA poLymerase + PT7 + RBS + GFP + ter.
3  The Strengthening of the Promoter Pho that constructed last year                             The construction of plasmid pSB1C3-pho + RBS + T7RNA poLymerase + PT7 + RBS + RFP + ter.
In the construction of plasmid, we mainly did the following work:
Plasmid DNA extraction
①Transfer 1.5mL of the culture to a Eppendorf tube. Centrifuge at 8000rpm for 5 minutes. Discard the supernatant. Add 280μL of resuspension solution(P1 buffer). Completely resuspent cell pellet.
② Add 200μL of lysis solution(Buffer P2) and mix gently. Set aside for 1 minute.
③Add 200 μL of neutralizing solution(P3 buffer) and mix by inverting the tubes for 5-10 times.
④Centrifuge at 12000rpm for 10 min and carefully transfer the supernatant to a adsorption column. Centrifuge at 8000rpm for 30sec, and remove the supernatant.Add 500μL of DW1 Buffer and centrifuge at 8000rpm for 30sec.
⑤Add 500μL of Wash Solution and centrifuge at 9000rpm for 30sec. Repeat twice.
⑥Centrifuge for 1min and open the lid for 1min to make the alcohol evaporate completely. Move the adsorption column to a new Eppendorf tube and add 30 μL of TE solution to obtain the plasmid DNA.
Transformation
① Thaw 100µL competent E. coli DH5α cells on ice until it is no longer frozen.
②Add 10µL ligation product into 100µL competent E. coli DH5α cells suspension, and then carefully flick the tube 4-5 times to mix cells and DNA.
③Place the mixture on ice for 30 minutes.
④Heat shock at exactly 42°C for exactly 90 seconds. Then place the mixture on ice for 1 minutes.
⑤Pipette 1 ml of room temperature LB media into the mixture. Incubate the mixture at 37°C with vigorous shaking for 60 minutes.
⑥Plate the mixture onto LB agar media containing antibiotic (ampicillin/ chloramphenicol). Incubate overnight at 37°C with plates upside down.
Preparing electrocompetent bacteria
◆Before we start: preparing freshly grown E. coli DH5α cells
①Preparing the CaCl2 solution at a concentration of 0.1mol/L
②Incubate 50µL E. coli DH5α cell suspension into 4ml LB media at 37°C with shaking in centrifugal speed of 200rpm overnight (without antibiotic).
③The next day, incubate the 1mL Bacterial suspension at 37°C with shaking in centrifugal speed of 200rpm until the OD600 is approximately 0,5(+-0,1)
④Add 25mL above incubated suspension into centrifuge (with gauge of 50mL) tube and centrifuge for 10min at 4 ℃ in centrifugal speed of 4000rpm.
⑤Discard the filtrate.
⑥Add 21ml of the prepared CaCl2 solution into precipitated bacteria. Incubate the mixture on ice for 20min.
⑦Centrifuge the above suspension for 10min at 4 ℃ in centrifugal speed of 4000rpm.
⑧Discard the filtrate.
⑨Add 3.4ml of the prepared CaCl2 solution (precooled) and 0.6mL of pure glycerol into the suspension.
⑩Split charge the product into eppendorf tubes, each 100µL, and cryopreservation at -80℃.
The Construction of Plasmids
Construction of the Series Plasmids of Nox
(1)pSB1C3-Pnox
Figure 1. pSB1C3-Pnox
① Expande the newly synthesized pGH-nox plasmid transformed glycerol stocks in 3mL LB media.
②Extract the plasmid from 3mL of the above bacteria suspension and elute with 30µL Elution buffer.
③Do restriction enzyme digestion of pGH-nox with the restriction enzymes EcoRI and PstI in 50µL system.
④Run DNA gel electrophoresis of the production of pGH-nox digestion  in 3µL system. Confirm that the restriction enzymes digestion is successful. Get the product of enzyme digestion purified and then eluted by 30µL Elution buffer.
⑤Ligate the extractive together with new vector by ligase overnight in temperature of 16℃.
⑥Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
⑦After overnight incubation, inoculate five freshly grown single colony and incubate each with 1.5 mL LB medium overnight.
⑧Extract 5 tubes of plasmid from the above bacteria suspension and do restriction enzyme digestion of the plasmid with the restriction enzymes EcoRI and PstI in 10µL system. Then send them for sequencing.
⑧Extract 5 tubes of plasmid from the above bacteria suspension and do restriction enzyme digestion of the plasmid with the restriction enzymes EcoRI and PstI in 10µL system. Then send them for sequencing.
(2) pSB1C3-Pnox+RBS+BFP、pSB1C3-Pnox+RFP
①Do restriction enzyme digestion of existing plasmids pSB1C3- BFP, pSB1C3- RFP with the restriction enzymes EcoRI and XbaI.
②Do restriction enzyme digestion of the plasmid pGH-nox with the restriction enzymes EcoRI and SPeI.
③Ligate the extractive together with new vector (digested pSB1C3- BFP、pSB1C3- RFP) by ligase overnight.
④Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
⑤After overnight incubation, inoculate five freshly grown single colony and incubate each with 1.5 mL LB medium overnight.
⑥Extract 5 tubes of plasmid from the above bacteria suspension and do restriction enzyme digestion of the plasmid with the restriction enzymes EcoRI and PstI in 10µL system. Then send them for sequencing.
(3) The Monitoring of Fluorescence Protein
PSB1C3-nox is also induced into fluorescence without nicotine, indicating that nox was a non-inducible promoter.
Construction of Nica Series plasmids
(1)the Construction of the first version plasmids of nica（pMD-Pnica1 + RBS + GFP + ter，pMD/pSB1C3-Pnica2 + RBS + GFP + ter）
①Design the primer including predictive promoter and RBS, using pSB1C3-GFP plasmid as template, through the polymerase chain reaction amplifying the plasmids containing Pnica1+RBS+GFP+ter (69 ℃ annealing and 72 ℃ extending for 2 minutes) and Pnica2+RBS+GFP+ter (69 ℃ annealing and 72 ℃ extending for 2 minutes) .
②Purify the genetic segments of Pnica1+RBS+GFP+ter (whose length is about 1Kb) and Pnica2 + RBS + GFP + ter (whose length is about 1Kb).
③Ligate two genetic segments together with T-vector by T4 DNA ligase.
④Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
⑤After overnight incubation, inoculate three freshly grown single colony and incubate each with 3 mL LB medium overnight.
⑥Extract plasmids from the above bacteria suspension, each 3 tubes of 1.5mL,  and do restriction enzyme digestion of the plasmids with the restriction enzymes EcoRI and PstI in 10µL system. Then send them for sequencing.
pMD-Pnica1+RBS+GFP+terand pMD-Pnica2+RBS+GFP+ter Enzyme digestion map
Lane 1:pMD-Pnica1+RBS+GFP+ter-1; Lane 2:pMD-Pnica1+RBS+GFP+ter-2
Lane 3:pMD-Pnica1+RBS+GFP+ter-3; Lane4:pMD-Pnica2+RBS+GFP+ter-1
Lane 5: blank; Lane 6;pMD-Pnica2+RBS+GFP+ter-2; Lane7:pMD-Pnica2+RBS+GFP+ter-3
⑦.The result shows that pMD-Pnica1+RBS+GFP+ter-3 (nica1-3) and pMD-Pnica2+RBS+GFP+ter-1 (nica2-1) sequencing results matched the reference sequences (in which nica2-1 was reverse connected with T-vector).
⑧Using the above bacteria to detect the expression of green fluorescent protein of different colonies. The result as in figure 8, from which we indicated that Pnica1 was a nicotine non-inducible promoter. While the plasmid pMD-Pnica2+RBS+GFP+ter-1  transformed bacteria did not express GFP in the absence of nicotine, suggesting that Pnica2 might be a nicotine inducible promoter.
⑨ In order to further verify whether Pnica2 is a nicotine inducible promoter, we compounded a LB agar medium with nicotine concentration of 5g/L, and plated 10µL plasmid pMD-Pnica2-RBS-GFP-ter-1  transformed bacteria suspension (which is produce by bacterial glycerol stock overnight incubation) droplets on it, then incubate it overnight.
⑩Observe the medium under a fluorescence microscope and we found that there was weak green fluorescence. This result shows that nica2 is a nicotine inducible promoter as shown in the following picture.
⑪Do restriction enzyme digestion of the plasmid pMD-Pnica2 + RBS + GFP + ter with the restriction enzymes EcoRI and PstI in 50µL system.
⑫.Purify the genetic segment Pnica2 + RBS + GFP + ter (whose length is about 1kb) , and ligate the segments together with linear pSB1C3-vector by T4 DNA ligase.                            Successfully complete the construction of the plasmid pSB1C3-Pnica2 + RBS + GFP + ter.
(2) the construction of the second version of Nica series plasmid (pMD/pSB1C3-Pnica2+RBS+T7RNA poLymerase +PT7 +RBS +GFP+ter)
1.Design the primer including predictive promoter and RBS, using the plasmid pXS-Oxy4 (pSB1C3-Pvgb +RBS+T7RNA poLymerase+ PT7+RBS+GFP+ter) as template, through the polymerase chain reaction amplifying the genetic segment Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter.
2.Get the genetic segment Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter purified and store the product in single eppendorf tube.
3.Ligate the genetic segment Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter together with T-vector by T4 DNA ligase.
4.Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
⑤5.After overnight incubation, inoculate twenty four freshly grown single colony and incubate each with 3 mL LB medium overnight.
iv>
6.Using the primers T7poLy-F and the universal primer M13-F on the T vector, the PMD-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter plasmid was used as a template PCR to verify whether the reverse insertion was performed. The size of the clone insert 21 does not match and should be about 4 kb. Therefore, the second method (directed insertion) is used to insert Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter into the T vector.
7.Extract plasmid from the above bacteria suspension with gene forward inserted plasmid pMD-Pnica2 + RBS + T7RNA poLymerase + PT7 + RBS + GFP + ter, and do restriction enzyme digestion of the plasmid with the restriction enzymes EcoRI and PstI in 50µL system.
Bottom: Linear T carrier
10.Purify the genetic segments Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter and linear T-vector and ligate the two segments together by T4 DNA ligase overnight.
10.Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
10.After overnight incubation, the medium didn't form any bacteria strain. So we decide to take the third method into action:
ligate the genetic segment Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter together with pUC-19 vector (circular) by T4 DNA ligase.
11.Transform 1µL of the ligation product into competent E. coli DH5α cells. Plate 100µL of the transformation product onto a LB agar medium and incubate it at 37°C overnight.
12.After overnight incubation, inoculate four freshly grown single colony and incubate each with 2 mL LB medium overnight.
13.Extract plasmids from the above bacteria suspension, and do restriction enzyme digestion of the plasmid pUC-19 with the restriction enzymes EcoRI and PstI. Then send them for sequencing， confirm that the experimental result is correct.
14.Purify the genetic segments Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter and ligate the segment together with linear vector pUC-19 and pSB1C3 by T4 DNA ligase overnight at 16℃.
15.Transform the ligation product into competent E. coli DH5α cells. After centrifugation, discard the supernatant, get the product suspended and concentrated into 100µL, then plate it onto a LB agar medium and incubate at 37°C overnight.
16.After overnight incubation, inoculate five freshly grown single colony and incubate each with 2 mL LB medium overnight.
17.Extract plasmids from the above bacteria suspension, each 5 tubes of 1.5mL,  and do restriction enzyme digestion of the plasmids with the restriction enzymes EcoRI and PstI in 10µL system. Then send them for sequencing， confirm that the experimental result is correct.
1:pSB1C3-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter Enzyme digestion image
2 :pUC-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter Enzyme digestion image
18.The LB solid medium having a nicotine concentration of 5 g/L was placed, and the plate was inverted, and 10 μL of pUC-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter glycerol bacteria droplets were placed on the plate and cultured overnight.
⑲The plate was observed under a fluorescence microscope, and as a result, it was found that the expression level of GFP was significantly increased.
Droplet edge fluorescent protein map (a: under white light b: under blue light excitation)
Sensitivity detection
Experimental purpose
Experiment sensitivity of the designed monitoring system. In other words, we are going to find the minimum nicotine concentration that can induce the First Version of Monitoring System's (with plasmid pMD-Pnica2 + RBS + GFP + ter transformed) expression by experimenting OD600 and the fluorescence value of the bacteria suspension at the wavelength of 475-530nm in different times by using enzyme-labeled instrument, and compare with that of the Second Version of Monitoring System (with plasmid pMD-Pnica2 + RBS + T7RNA poLymerase + PT7 + RBS + GFP + ter transformed).
Experimental design
Activated E. coli
①Add 30µL plasmid pMD-Pnica2+RBS+GFP+ter transformed bacterial glycerol stock into a 10ml centrifuge tube containing 3mL LB media with 3µL Amp (in concentration of 1%), incubate at centrifugal speed of 200rpm at 37℃ overnight for 14h，measured that the final OD600=0.8.
② Add 90µL the above suspension into a 10ml centrifuge tube containing 3mL LB media with 3µL Amp (in concentration of 3%), incubate at centrifugal speed of 200rpm at 37℃ for 5h，measured that the final OD600=0.4.
③Add 30µL plasmids pMD-Pnica2+RBS+GFP+ter and pMD-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter transformed bacterial glycerol stocks into two 10ml centrifuge tubes containing 3mL LB media with 3µL Amp (in concentration of 1%) respectively, incubate at centrifugal speed of 200rpm at 37℃ overnight for 14h，measured that the final OD600=1.3 and 1.1 respectively.
④Add 90µL the above suspensions into two 10ml centrifuge tubes containing 3mL LB media with 3µL Amp (in concentration of 3%), incubate at centrifugal speed of 200rpm at 37℃ for approximately 5h，measured that the final OD600 are both approximately 0.4.
Prepare a nicotine gradient solution
Purchase the nicotine solution at a concentration of 1.0g/mL. Then dissolve the nicotine solution by LB media with Amp, add 100µL mixture into the 96 hole plate with flat black bottom per well.
Experiment 1: (of the first version)
①.Prepare the nicotine solution at different concentrations of 0 g/mL, 0.02 g/mL, 0.04 g/mL, 0.08 g/mL,0.16 g/mL, 0.32 g/mL, 0.64 g/mL, 1.28 g/mL, 2.56 g/mL, 5.12 g/mL, 10.24 g/mL.
②Set up three sets of experiments. Add the plasmid MD-Pnica2+RBS+GFP+ter transformed bacteria (whose OD600 is approximately 0.4) in concentration of approximately 3% (3µL) to the 96 hole plate with flat black bottom per well and incubated at centrifugal speed of 150rpm at 37 ℃. Measure the OD600 values and fluorescence values every two hours.
Experiment 2: (of the first version)
①Prepare the nicotine solution at different concentrations of 0 g/mL, 0.001 g/mL, 0.01 g/mL,0.1 g/mL,1 g/mL.
②Set up five sets of experiments. Add the plasmid MD-Pnica2+RBS+GFP+ter transformed bacteria (whose OD600 is approximately 0.4) in concentration of approximately 3% (3µL) to the 96 hole plate with flat black bottom per well and incubated at centrifugal speed of 150rpm at 37 ℃. Measure the OD600 values and fluorescence values each hour.
Experiment 3: (the Comparison of the First and the Second Version Monitoring System)
①.Prepare the nicotine solution at different concentrations of 0 g/mL, 0.0001 g/mL, 0.001 g/mL,0.01 g/mL.
②Set up five sets of experiments. Add the plasmid MD-Pnica2+RBS+GFP+ter and plasmid pMD-Pnica2+RBS+T7RNA poLymerase+PT7+RBS+GFP+ter transformed bacteria (whose OD600 is approximately 0.4) in concentration of approximately 3% (3µL) to the 96 hole plate with flat black bottom per well and incubated at centrifugal speed of 150rpm at 37 ℃. Measure the OD600 values and fluorescence values every two hours.
results analysis
①The result of the First Experiment shows that high concentration (above 1.28g/L) of nicotine had a strong inhibitory effect on the growth of Escherichia coli, which affected the expression of GFP in Escherichia coli, so we reduce the range of nicotine concentration and carry out the Second Experiment.
② The result of the Second Experiment shows that the fluorescence expression of nicotine when the concentration of nicotine is C=0.01g/L increased steadily within 4 hours, but that of C=0. 001g/L was unstable. In addition, when the concentration of nicotine is within C=0.1g/L and C=1g/L had obvious toxic effect on the bacteria strain.
③The result of the Third Experiment shows that when nicotine concentration was as low as 0.0001g/L, fluorescence expression appears in the second version monitoring system at 1h and steadily increased within 1-3h, while fluorescence expression in the first version monitoring system only appears at 4 h. The result shows that the second version monitoring system greatly improves the induction of nicotine.
Appendix
promoter source and sequence
①Promoter nox: gaining from the article 《Cloning of a Novel Nicotine Oxidase Gene from Pseudomonas sp.Strain HZN6 Whose Product NonenantioseLectively Degrades Nicotine to Pseudooxynicotine》
From website: https://www.ncbi.nlm.nih.gov/nuccore/JN391188.2
Length: about 400bp
Note: add the restriction enzymes EcoRI, XbaI restriction enzyme cutting sites at the left end and the SpeI, PstI restriction cutting enzyme sites at the right end.
Sequence(5'-3'):
gagcagcaatacggtttttcaattgcagtcaaggcgggcgggtttctttatatcggtggcgtaacagccgttgataaagagggaaatgaagtatacgccaatgatgctaacaagcaaatgcagcttatttatgagcgtctgggtgcgatcctggccgctcatgatgcagacttcagtaacgttgttagtgaaactatctattataccaccgataatgaaagttacattaaatctGtagacgtcagaactgctgcgtataagggcgtaatagcaccaagtgcatctggcgtcagggttgccgatttcttgagcgacaagacgcttatagaaataacagctgtagcttatctcggcgaataagtgggctgccagtttcatcggttctgtagattcgcaacgatactaaaggtgtcagaca
②Promoter nica1/nica2: gaining from the article 《A Novel Gene, Encoding 6-Hydroxy-3-SµccinoyLpyridine Hydroxylase,Involved in Nicotine Degradation by Pseudomonas putida Strain S16》 and with the help of NCBI to predict these two possible promoter
From website: https://www.ncbi.nlm.nih.gov/search/?term=+DQ988162
Length of nica1: 46bp
Length of nica2: 34bp
Note: The promoter is designed in the primer, and the ECORI, XbaI restriction enzyme cutting sites is added at the left end and the SpeI, PstI restriction enzyme cutting sites is added at the right end.
Sequence of nica1 (5'-3')：
ttcagatggcacaggtgcgaaaccctcgttataatccggccactta
Sequence of nica2 (5'-3'）：
Ttcgtagtacctgtttgtattgggccggtagcat
Experimental system
Worldshaper-XSHS, Xiaoshan High School
Adress: No.538,Gongxiu Road,Xiaoshan District,Hangzhou,Zhejiang Province,China
Summary:
This is a minor edit  Watch this page
Please note that all contributions to 2018.igem.org are considered to be released under the Creative Commons Attribution (see 2018.igem.org:Copyrights for details). If you do not want your writing to be edited mercilessly and redistributed at will, then do not submit it here. You are also promising us that you wrote this yourself, or copied it from a public domain or similar free resource. Do not submit copyrighted work without permission! Save page Show preview  Show changes Cancel | Editing help (opens in new window)
