<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Austin_Texas_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Austin_Texas/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Austin_Texas/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Austin_Texas/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Austin Texas/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL class="cssmenu"><LI class="home"><A href="/Team:Austin_Texas" title="home"><SPAN class="displace">Home</SPAN></A></LI><LI class="team"><A href="/Team:Austin_Texas/Team" title="team"><SPAN class="displace">Team</SPAN></A></LI><LI class="official_team_profile"><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Austin_Texas" title="official_team_profile"><SPAN class="displace">Official Team Profile</SPAN></A></LI><LI class="project"><A href="/Team:Austin_Texas/Project" class="selected" title="project"><SPAN class="displace">Project</SPAN></A></LI><LI class="parts_submitted"><A href="/Team:Austin_Texas/Parts" title="parts_submitted"><SPAN class="displace">Parts Submitted</SPAN></A></LI><LI class="modelling"><A href="/Team:Austin_Texas/Modeling" title="modeling"><SPAN class="displace">Modeling</SPAN></A></LI><LI class="notebook"><A href="/Team:Austin_Texas/Notebook" title="notebook"><SPAN class="displace">Notebook</SPAN></A></LI><LI class="safety"><A href="/Team:Austin_Texas/Safety" title="safety"><SPAN class="displace">Safety</SPAN></A></LI><LI class="attributions"><A href="/Team:Austin_Texas/Attributions" title="attributions"><SPAN class="displace">Attributions</SPAN></A></LI></UL></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_1:_ZombiEcoli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project 1: ZombiEcoli</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Distinguishing_Between_E.coli_populations"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Distinguishing Between E.coli populations</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mechanism_of_Infection"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mechanism of Infection</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Genetic_Switch"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Genetic Switch</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Project_2:_Caffeinated_Coli"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project 2: Caffeinated Coli</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Refactoring_Decaffeination_Operon"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Refactoring Decaffeination Operon</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Operon_Testing_and_Optimization"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Operon Testing and Optimization</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Characterizing_Inducible_Promoters"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Characterizing Inducible Promoters</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Project_3:_Spinach-mCherry_Dual_Fluoresence_Reporter"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Project 3: Spinach-mCherry Dual Fluoresence Reporter</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Project_4:_Adding_Zombie_Smell"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Project 4: Adding Zombie Smell</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_1:_ZombiEcoli"><B>Project 1: ZombiEcoli</B></SPAN></H2><H3><SPAN class="mw-headline" id="Distinguishing_Between_E.coli_populations">Distinguishing Between <I>E.coli</I> populations</SPAN></H3><P>Features of a circuit modeling transmissible disease must include certain E.coli behaviors,  one which allows for a zombie cell to infect normal cells, and one for a normal cell to irreversibly switch into a zombie cell. In order to easily distinguish between phenotypes of normal and zombie cells, different fluorescent proteins will be expressed in each cell population. A green fluorescent protein (GFP) will be expressed in normal cells, and a red fluorescent protein (RFP) expressed in zombie cells. The phenotypic switch will be controlled by an irreversible promoter flip accomplished through mutant CRE-Recombination LoxP sites. (Zhang 2002) This restructuring will result in a permanent change in genes being expressed upon infection.
</P><P>In the above figure, the left cell depicts a Z-Mode or zombie cell. The main features of this are its expression of RFP and the expression of LasI, an enzyme that produces quorum signaling molecule acyl-homoserine lactone (AHL). The quorum molecule diffuses out of the zombie cell and into the normal cell. Upon AHL reaching high enough concentration, LasR will activate the expression of CRE and the genetic restructuring event may occur.
</P><H3><SPAN class="mw-headline" id="Mechanism_of_Infection">Mechanism of Infection</SPAN></H3><P>The mechanism of infection that the zombie cells use is based on a quorum sensing (QS) molecule produced by the LasI protein encoded for by the gene LasI from <I>Pseudomonas aeruginosa</I>. The  QS molecule, acyl-homoserine lactone (AHL), permeates into the extracellular environment and is absorbed by the recipient normal cell. When AHL is in high enough concentration, the LasR enzyme from the gene LasR from Pseudomonas aeruginosa will convert AHL into a transcription factor to drive the LasR promoter, with the CRE-Recombinase protein under its control. CRE dictates the next behavior we want of our normal cells, to irreversibly switch into zombie cells, which can then infect other normal cells.
</P><P>Features of the converter plasmid include a scheme for detecting the AHL quorum sensing molecule (from the LasR enhancer protein) and transduce the quorum molecule into the production of CRE recombinase. The expression of the LasR protein is under control of the endogenous Lac operator by the LacIm1 protein. This feature will serve to limit the expression of this molecule during the cloning and testing of this parental plasmid sequence and allow swift induction of expression after adding IPTG. The LacIm1 is a codon isomer of the LacI gene, meaning that it has large codon deviation from the wild-type LacI gene. This will prevent recombination between the Converter and Reporter plasmids in the experiments (Basu 2005). Upon the expression of the LasR protein and in the presence of a sufficient local concentration of AHL the CRE expression will be turned on.
</P><H3><SPAN class="mw-headline" id="Genetic_Switch">Genetic Switch</SPAN></H3><P>Expression of CRE-Recombinase will then act as a genetic switch and trigger a restructuring in the Reporter plasmid. The genetic switch relies on a variant of the Cre/loxP recombination system. The protein CRE-Recombinase (causes recombination) catalyzes crossover events at specific DNA sequences termed loxP sites. Two variants, lox66 and lox71 can be combined to create the irreversible crossover event we want. CRE recombinase will trigger a one way recombination event of a LacUV5 promoter (Zhang 2002) causing it to switch and express the proteins it was previously facing away from. The LacUV5 flip is the essential difference between our normal and zombie cells. In a normal cell, IPTG induction will cause GFP expression, but if a cell has received the QS molecule AHL and expressed CRE-Recombinase, then the promoter will express the opposite set of genes. This results in a zombie cell expressing a RFP and the gene LasI, upon which it can now make the quorum molecule acyl-homoserine lactone, making it capable of further infecting proximal cells. 
</P><H2><SPAN class="mw-headline" id="Project_2:_Caffeinated_Coli"><B>Project 2: Caffeinated Coli</B></SPAN></H2><H3><SPAN class="mw-headline" id="Refactoring_Decaffeination_Operon"> Refactoring Decaffeination Operon </SPAN></H3><P>The first goal of this project involves refactoring the caffeine operon from the caffeine utilization pathway from <I>Psuedomonas putida</I> CBB5, first characterized by Summers et al. in early 2012. The operon, shown below, will be incorporated into the well characterized bacterium, <I>Escherichia coli</I> [3]. 
</P><P>Directly importing the operon into <I>E. coli</I> was determined impractical, as the strength and regulation of the ribosome binding sites (rbs) and operon-controlled promoters in the CBB5 operon may not be optimized for function in <I>E. coli</I>. Additionally, the preference in CBB5 for GTG start codons conflicts with E. coliâs preference for ATG â leading to problems in translation initiation.
</P><P>We therefore decided to separate out open reading frames for the genes of interest in the CBB5 operon and put them under controlled regulation in a refactored caffeine utilization operon for import into <I>E. coli</I>. The operon's design, listed below, aims to optimize its functionality in its new host. 
</P><P>This includes the N-demethylase proteins: <I>ndmA</I>, <I>ndmB</I>, <I>ndmC</I>, and the putative assisting protein <I>ndmD</I>. Additionally, two uncharacterized open reading frames <I>orf1</I> and <I>orf4</I> are included.
</P><H3><SPAN class="mw-headline" id="Operon_Testing_and_Optimization"> Operon Testing and Optimization </SPAN></H3><P>We will employ two different assays for operon functionality; growth on caffeine as a sole carbon source, and a genetic selection for caffeine demethylation to xanthine.  To evaluate the ability to use caffeine as a sole carbon source we will transform TOP 10 E.coli electrocompetent cells with the refactored caffeine utilization operon, grow transformed cells in rich media to saturation and then dilute 1:100 into M9 mineral media. Varying levels of caffeine concentrations will be used to determine the degree of caffeine utilization, and the optimal limit for growth.
</P><P>Since the cell has an extremely large requirement for carbon, the energy derived from demethylation may not be enough to support growth. For this reason a second assay for caffeine demethylation based on guanine auxotrophy has been devised. <I>E. coli</I> synthesizes the nucleotide guanine de novo via a pathway that involves Xanthosine-5â-phosphate (XMP) as an essential intermediate.  The enzyme responsible for the formation of XMP (from inosine-5â-phosphate[IMP]) is IMP dehydrogenase, which is encoded by the GuaB gene.  If GuaB is knocked out, the cell is unable to synthesize guanine and is therefore unable to grow on media lacking guanine.  We plan to take advantage of this engineered auxotrophy and use it as a way to select for cells that are able to demethylate caffeine to xanthine which can then be converted to XMP by xanthine-guanine phosphoribotransferase (gpt) and thereby relieve the metabolic block and restore guanine synthesis allowing for cell growth.
</P><P>Finally, after construction and preliminary testing of the caffeine degradation operon in<I>E. coli</I>, we will attempt to grow our cells in the presence of various commercial caffeinated beverages.
</P><H3><SPAN class="mw-headline" id="Characterizing_Inducible_Promoters"> Characterizing Inducible Promoters </SPAN></H3><P>In Summmers et al., the two open reading frames <I>orf1</I> and <I>orf4</I> are thought to be putative regulators of the caffeine degradation operon's N-demethylase proteins due to the proximity of their genes. They are hypothesized to bind to operator sequences in the intergenic regions between genes in the operon, which may serve as promoters for the various demethylases of the operon. 
</P><P>Analysis of the sizes of the intergenic regions of the CBB5 caffeine utilization operon shows that the regions upstream of the ndm genes are all greater than 150bp.  The large size of these intergenic regions and the fact that they precede the catabolic enzyme gene leads us to hypothesize that there are caffeine (or other methylxanthine) regulatory elements in these sequences.
</P><P>We will clone these open reading frames into the reporter plasmid pRA301. pRA301 contains a promoterless lacZ gene, preceded by a multiple cloning site (MCS). DNA fragments hypothesized to contain promoter elements can be cloned into the MCS and assayed for lacZ expression by Miller assay. Using this method, we can determine the regulatory functionality of each open reading frame by examining varying fluorescence levels.
</P><H2><SPAN class="mw-headline" id="Project_3:_Spinach-mCherry_Dual_Fluoresence_Reporter"><B>Project 3: Spinach-mCherry Dual Fluoresence Reporter </B></SPAN></H2><P>More info coming soon.
</P><H2><SPAN class="mw-headline" id="Project_4:_Adding_Zombie_Smell"><B>Project 4: Adding Zombie Smell</B></SPAN></H2><P>Background
</P><UL><LI><A href="http://en.wikipedia.org/wiki/Cadaverine" class="external text" rel="nofollow">Cadaverine</A></LI><LI><A href="http://partsregistry.org/Part:BBa_J45999:Design" class="external text" rel="nofollow">Odor free chassis</A> has a <I>tnaA</I> mutation that prevents degradation of Tryptophan to indole.
</LI><LI><I>cadA</I> and <I>ldcC</I> genes convert lysine to cadaverine. <A href="http://www.ncbi.nlm.nih.gov/pubmed/21552989" class="external text" rel="nofollow">Reference</A></LI></UL><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI>Zhang and Lutz, âCre recombinase-mediated inversion using lox66 and lox71: method to introduce conditional point mutations into the CREB-binding proteinâ Nucleic Acids Research, vol 30, issue 17, p. e90, 2002.
</LI><LI>Basu S, Gerchman Y, Collins CH, Arnold FH, Weiss R. A synthetic multicelluar system for programmed pattern formation. Nature, 2005 Apr 28;434(7037):1130-4.
</LI><LI>Summers RM, Louie TM, Yu CL, Gakhar L, Louie KC, Subramanian M, &quot;Novel, highly specific N-demethylases enable bacteria to live on caffeine and related purine alkaloids.&quot; Journal of Bacteriology, 2012, vol 194, no 8, pg 2041-2049.
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Austin_Texas/Project">http://2012.igem.org/Team:Austin_Texas/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Austin_Texas/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Austin_Texas/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Austin_Texas/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Austin_Texas/Project&amp;oldid=85696" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>