<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Austin_Texas_Caffeinated_coli"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Austin_Texas/Caffeinated_coli">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Austin_Texas/Caffeinated_coli&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/Caffeinated_coli&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Austin_Texas/Caffeinated_coli&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Austin_Texas/Caffeinated_coli" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Austin Texas/Caffeinated coli</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Austin_Texas/Decaffeinated_coli&amp;redirect=no" title="Team:Austin Texas/Decaffeinated coli">Team:Austin Texas/Decaffeinated coli</A>)</DIV><P><UL class="cssmenu" style="float:left;"><LI class="home"><A href="/Team:Austin_Texas" title="home"><SPAN class="displace">Home</SPAN></A></LI><LI class="team"><A href="/Team:Austin_Texas/Team" title="team"><SPAN class="displace">Team</SPAN></A></LI><LI class="official_team_profile"><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Austin_Texas" title="official_team_profile"><SPAN class="displace">Official Team Profile</SPAN></A></LI><LI class="human_practices"><A href="/Team:Austin_Texas/ZombiE_coli#Human_Practices" title="Human Practices"><SPAN class="displace">Human Practices</SPAN></A></LI><LI class="Caffeinated_coli"><A href="/Team:Austin_Texas/Caffeinated_coli" title="Caffeinated_coli" class="selected"><SPAN class="displace">Caffeinated coli</SPAN></A></LI><LI class="ZombiE_coli"><A href="/Team:Austin_Texas/ZombiE_coli" title="ZombiE_coli"><SPAN class="displace">ZombiE.coli</SPAN></A></LI><LI class="Spinach_reporter"><A href="/Team:Austin_Texas/Spinach_reporter" title="Spinach_reporter"><SPAN class="displace">Spinach reporter</SPAN></A></LI><LI class="notebook"><A href="/Team:Austin_Texas/Notebook" title="notebook"><SPAN class="displace">Notebook</SPAN></A></LI><LI class="parts_submitted"><A href="/Team:Austin_Texas/Parts" title="parts_submitted"><SPAN class="displace">Parts Submitted</SPAN></A></LI><LI class="safety"><A href="/Team:Austin_Texas/Safety" title="safety"><SPAN class="displace">Safety</SPAN></A></LI><LI class="attributions"><A href="/Team:Austin_Texas/Team#Attributions" title="attributions"><SPAN class="displace">Attributions</SPAN></A></LI></UL></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project:_Caffeinated_coli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project: Caffeinated coli</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Strategy"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Strategy</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Refactoring_Decaffeination_Operon"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Refactoring Decaffeination Operon</SPAN></A></LI><LI class="toclevel-4 tocsection-5"><A href="#Assembly"><SPAN class="tocnumber">1.2.1.1</SPAN><SPAN class="toctext">Assembly</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-6"><A href="#Operon_Testing_and_Optimization"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Operon Testing and Optimization</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Characterizing_Inducible_Promoters"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Characterizing Inducible Promoters</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#CBB5_Genome_sequencing"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">CBB5 Genome sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Decaffeination_operon_.28.2B_gst9.29_enables_growth_of_GuaB_knockout"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Decaffeination operon (+ gst9) enables growth of GuaB knockout</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Engineered_E._coli_stoichiometrically_convert_caffeine_to_guanine_for_cell_replication"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Engineered E. coli stoichiometrically convert caffeine to guanine for cell replication</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Growth_in_Caffeinated_Beverages"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Growth in Caffeinated Beverages</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Transcriptional_Regulators"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Transcriptional Regulators</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Inducible_Promoters"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Inducible Promoters</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Results_Summary"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">Results Summary</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Project:_Caffeinated_coli"><CENTER><B><DIV style="font-size:150%">Project: Caffeinated coli</DIV></B></CENTER></SPAN></H1><H2><SPAN class="mw-headline" id="Introduction"><DIV style="font-size:130%;text-align:center"><B>Introduction</B></DIV></SPAN></H2><P>Caffeine is commonly used in foods and beverages such as coffee and chocolate and in pharmaceuticals as a cardiac and respiratory stimulant. As a result of the wide use of caffeine, it has become widely present in human waste and as a pollutant in the environment. Bacteria capable of degrading caffeine have been found naturally and could be used for bioremediation.  We seek to port caffeine degradation functionality into <I>Escherichia coli</I> to produce strains that are better suited to degrade caffeine in an industrial setting. 
</P><P><I>Pseudomonas putida</I> CBB5, discovered by Ryan Summers and Mani Subramanian at the University of Iowa, can live on caffeine as the sole carbon and nitrogen source. CBB5 uses a nitrogen demethylation pathway to convert caffeine to xanthine and formaldehyde which are then used as the nitrogen and carbon sources respectively.
</P><P>The N-demethylation pathway consists of four demethylation genes: <I>ndmA</I>, <I>ndmB</I>, <I>ndmC</I>, and <I>ndmD</I>. NdmA, NdmB, and NdmC remove the methyl groups from the N-1, N-3, and N-7 respectively. This is done with the help of a reductase, NdmD.
</P><H2><SPAN class="mw-headline" id="Strategy"><DIV style="font-size:130%;text-align:center"><B>Strategy</B></DIV></SPAN></H2><H3><SPAN class="mw-headline" id="Refactoring_Decaffeination_Operon"> Refactoring Decaffeination Operon </SPAN></H3><P>The first goal of this project involves refactoring the caffeine operon from the caffeine utilization pathway from <I>Psuedomonas putida</I> CBB5, first characterized by Summers et al. in early 2012. The operon, shown below, will be incorporated into the well characterized bacterium, <I>Escherichia coli</I> [3]. 
</P><P>Directly importing the operon into <I>E. coli</I> was determined to be impractical, as the strength and regulation of the ribosome binding sites (RBSs) and operon-controlled promoters in the CBB5 operon may not be optimized for function in <I>E. coli</I>. Additionally, the use in CBB5 of GTG start codons conflicts with <I>E. coli’s</I> preference for ATG – leading to problems in translation initiation.
</P><P>We therefore decided to separate out open reading frames for the genes of interest in the CBB5 operon and put them under controlled regulation in a refactored caffeine utilization operon for import into <I>E. coli</I>. The operon's design, shown below and submitted as <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K734000" class="external text" rel="nofollow">BBa_K734000</A>, aims to optimize its functionality in its new host. 
</P><P>This includes the genes that code for the N-demethylase proteins NdmA, NdmB, and NdmC, and the gene that codes for the putative assisting protein NdmD. Also included is the glutathione S-transferase gene from <I>Janthinobacterium</I> sp. strain Marseille, necessary for functionality of NdmC. Constitutive expression occurs with a strong, well-characterized promoter (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A>) and a strong, well-characterized RBS (<A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A>). Finally, all GTG start codons have been replaced with ATG.
</P><P>Sources of parts used in our synthetic decaffeination circuit:
</P><H4><SPAN class="mw-headline" id="Assembly"> Assembly </SPAN></H4><P>Our operon was assembled via a one-step, six-piece Gibson assembly. Briefly, genes to be stitched together were PCR amplified with overhangs homologous to adjacent genes (or homologous to the vector backbone in the case of the 5' end of the promoter and the 3' end of <I>gst9</I>). The forward primers also contained our chosen RBS and ATG. In a one-pot reaction, a 5'-exonuclease chewed back on the homologous overhangs, allowing adjacent fragments to base pair, and a DNA ligase stitched them together. An overview is shown here.
</P><H3><SPAN class="mw-headline" id="Operon_Testing_and_Optimization"> Operon Testing and Optimization </SPAN></H3><P>We employed two different assays for operon functionality: growth on caffeine as a sole carbon source, and a genetic selection for caffeine demethylation to xanthine.  To evaluate the ability to use caffeine as a sole carbon source we will transform TOP 10 <I>E. coli</I> electrocompetent cells with the refactored caffeine utilization operon, grow transformed cells in rich media to saturation and then dilute 1:100 into M9 mineral media. Varying levels of caffeine concentrations will be used to determine the degree of caffeine utilization, and the optimal limit for growth.
</P><P>Since the cell has an extremely large requirement for carbon, the energy derived from demethylation may not be enough to support growth. For this reason a second assay for caffeine demethylation based on guanine auxotrophy has been devised. <I>E. coli</I> synthesizes the nucleotide guanine de novo via a pathway that involves xanthosine-5’-phosphate (XMP) as an essential intermediate.  The enzyme responsible for the formation of XMP (from inosine-5’-phosphate, IMP) is IMP dehydrogenase, which is encoded by the <I>guaB</I> gene.  If <I>guaB</I> is knocked out, the cell is unable to synthesize guanine and is therefore unable to grow on media lacking guanine.  We plan to take advantage of this engineered auxotrophy and use it as a way to select for cells that are able to demethylate caffeine to xanthine which can then be converted to XMP by xanthine-guanine phosphoribotransferase (gpt) and thereby relieve the metabolic block and restore guanine synthesis allowing for cell growth.
</P><P>Finally, after construction and preliminary testing of the caffeine degradation operon in<I>E. coli</I>, we will attempt to grow our cells in the presence of various commercial caffeinated beverages.
</P><H3><SPAN class="mw-headline" id="Characterizing_Inducible_Promoters"> Characterizing Inducible Promoters </SPAN></H3><P>In Summers et al (2012)., the two open reading frames <I>orf1</I> and <I>orf4</I> are thought to be putative regulators of the caffeine degradation operon's N-demethylase proteins due to sequence homology to other known protein regulators (AraC and gntR family). They are hypothesized to bind to operator sequences in the intergenic regions between genes in the operon, which may serve as promoters for the various demethylases of the operon. 
</P><P>Analysis of the sizes of the intergenic regions of the CBB5 caffeine utilization operon shows that the regions upstream of the <I>ndm</I> genes are all greater than 150bp.  The large size of these intergenic regions and the fact that they precede the catabolic enzyme gene leads us to hypothesize that there are caffeine (or other methylxanthine) regulatory elements in these sequences.
</P><P>We will clone these open reading frames into the reporter plasmid pRA301. pRA301 contains a promoterless <I>lacZ</I> gene, preceded by a multiple cloning site (MCS). DNA fragments hypothesized to contain promoter elements can be cloned into the MCS and assayed for <I>lacZ</I> expression by <A href="http://openwetware.org/wiki/Beta-Galactosidase_Assay_%28A_better_Miller%29" class="external text" rel="nofollow">Miller Assay</A>. Using this method, we can determine the regulatory functionality of each open reading frame by examining varying fluorescence levels.
</P><H3><SPAN class="mw-headline" id="CBB5_Genome_sequencing"> CBB5 Genome sequencing </SPAN></H3><P>Since only a partial sequence (13.1kb) of the <I>P. putida</I> CBB5 decaffeination operon is available (Summers 2012), we are submitting genomic DNA for whole genome sequencing.  The genome will be sequenced using Illumina next-gen sequencing by UT [<A href="https://wikis.utexas.edu/display/GSAF/Home+Page" class="external text" rel="nofollow">GSAF</A>].  The assembled genome will be deposited to Genbank.
</P><H2><SPAN class="mw-headline" id="Results"><DIV style="font-size:130%;text-align:center"><B>Results</B></DIV></SPAN></H2><H3><SPAN class="mw-headline" id="Decaffeination_operon_.28.2B_gst9.29_enables_growth_of_GuaB_knockout">Decaffeination operon (+ gst9) enables growth of GuaB knockout</SPAN></H3><P>Our initial refactored operon consisted of CBB5 genes <I>ndmABCD</I>.  We found that this operon was able to support growth of the <I>guaB</I>knockout on theophylline but not caffeine.  This indicated that the demethylase responsible for removing the 7-methyl group (NdmC) was not functional.  Of note, Summers et. al (2012) also could not detect NdmC activity when expressed in <I>E. coli</I>.  We reasoned that there could be a missing protein required for NdmC activity.  Summers et al. (2012) showed that an uncharacterized protein (coded by <I>orf8</I>) co-purified in CBB5 protein fractions assayed for NdmC activity.  We reasoned that this protein could be essential for NdmC function.  Unfortunately, the complete DNA sequence of <I>orf8</I> was not available; only a partial sequence of the ORF was contained in the known operon sequence.  
</P><P>
A protein homology search was performed using the available sequence to find potential homologs that might be able to substitute function of the missing ORF.  The search revealed that an uncharacterized gene, <I>gst9</I>, from <I>Janthinobacterium</I> sp. strain marseille shared a modest degree of sequence homology (66%).  We decided to synthesize the <I>gst9</I> gene from the available sequence and clone it into our decaffeination operon to see if it would enable NdmC activity and allow for complete demethylation of caffeine. 
</P><P>
Despite the relatively low level of homology, we found our hypothesis to be true: adding <I>gst9</I> to our refactored operon <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K734000" class="external text" rel="nofollow">(Bba_K734000)</A> did indeed enable growth of the <I>guaB</I> knockout on m9 mineral media:
</P><P>From this figure, we see that our decaffeination operon enables the GuaB knockout to grow in the absence of guanine or xanthine supplementation by instead demethylating the available caffeine to produce xanthine. This not only confirms the functionality of our refactored decaffeination operon, but also validates our genetic selection for caffeine demethylation through engineered guanine auxotrophy.
</P><P>After sequencing the <I>P. putida</I> CBB5 genome, an alignment of its putative GST (<I>orf8</I>) against the <I>gst9</I> of <I>Janthinobacterium</I> sp. (strain Marseille) showed less homology (63.5% homology at the nucleotide level) than that of the originally available partial sequence:
</P><P>
To more accurately determine the utilization of caffeine by our operon, we tested the growth of our <I>E. coli</I> cells containing refactored operon and the knocked out <I>guaB</I> gene under multiple caffeine concentration conditions. We found that our cells were able to grow at conditions as low as 10uM of caffeine, and peaked at a caffeine concentration between 200uM and 250uM. Cells were grown in concentrations as high as 5000uM, at which point cells began to die (presumably from caffeine toxicity). 
</P><P>From these growth conditions, we plated dilutions of up to 10<SUP>7</SUP> the concentration of original solution. This was used to determine individual cell growth based on caffeine. We discovered that approximately 7.6 +/- 0.8 pg of caffeine were utilized per cell. The graph above also shows a method by which we can convert directly from optical densities (an easily measured solution condition) to the caffeine concentration of the growth media. This is viable for cultures that contain cells with our refactored operon, and caffeine concentrations between 0 and approximately 250 uM.
</P><H3><SPAN class="mw-headline" id="Engineered_E._coli_stoichiometrically_convert_caffeine_to_guanine_for_cell_replication"> Engineered <I>E. coli</I> stoichiometrically convert caffeine to guanine for cell replication </SPAN></H3><P>As caffeine appears to limit cell growth under these conditions, we were curious about how much of the provided caffeine was being utilized to create DNA and RNA for cellular replication. The genome of <I>E. coli</I> is roughly 4.6 Mb. Since this is double-stranded DNA and approximately 50% consists of G-C base pairs, there are about 2.3x10<SUP>6</SUP> guanines needed per cell to replicate its DNA. The dry weight of a typical <I>E. coli</I> cell is approximately 280 femtograms and 20% of this is RNA (<A href="http://bionumbers.hms.harvard.edu" class="external text" rel="nofollow">Bionumbers</A>). Given the molecular weight of a typical RNA nucleotide is roughly 330 g/mol, and that 1/4 of these bases are guanine, this means that there are about 2.55x10<SUP>10</SUP> guanines needed per cell to replicate its RNA. So, overall the number of guanine molecules in RNA is roughly 10,000x the amount in DNA.
</P><P>We found that  7.6 +/- 0.8 pg of caffeine was utilized per cell. Given the molecular weight of 194.19 g/mol, this translates into 2.4x10<SUP>10</SUP> molecules of caffeine per cell. This is in amazing agreement with the number of molecules of guanine that we have calculated that is required to replicate a cell!
</P><DIV style="font-size:130%;text-align:center"><TABLE class="wikitable" style="text-align:center"><TBODY><TR><TH>Molecules per cell
</TH></TR><TR><TD> Caffeine molecules consumed (Calculated) </TD><TD> 2.40x10<SUP>10</SUP></TD></TR><TR><TD> Guanine molecules required (Theoretical) </TD><TD> 2.55x10<SUP>10</SUP></TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Growth_in_Caffeinated_Beverages">Growth in Caffeinated Beverages </SPAN></H3><P>In order to make our results more  palpable, we experimented with growth of our <I>E. coli</I> strain in various caffeinated beverages. Experiments were first performed on cells with only the <I>guaB</I> knockout modification, in order to prove that growth in these beverages was even possible. Initial results, shown below, indicate that growth is in fact possible for a wide variety of beverages, including Coca-Cola, 5-Hour Energy, Lipton Tea, and Starbucks Espresso.
</P><P>
The graph above shows that the auxotrophic selection of cells using xanthine derivatives again functions as expected – without xanthine, cells were unable to grow under most conditions. The noticeable optical densities for 50% Coca-Cola and 50% Espresso are due to the strong background color of the culture and not actual cell growth. 50% 5-Hour Energy and Espresso were both shown to be toxic to our cells. Finally, tea was shown to contain enough natural xanthines to allow for partial cell growth, and was excluded from further experiments.
</P><P>We then attempted to grow our cell strain containing both the <I>guaB</I> knockout gene and the refactored decaffeination operon. The results of this experiment are shown below.
</P><P>It is important to note that these cells were grown in m9 minimal media containing 0.2% casein as a carbon source. The cells were required to scavenge and convert the available caffeine in the beverage to xanthine in order to synthesize guanine and support cell growth. Our <I>E. coli</I> was able to utilize the caffeine inherent to each caffeinated beverage. However, without our refactored decaffeination operon, the cells were unable to grow under any conditions. The media contains the appropriate amount of each beverage to have about a 50 micromolar concentration of caffeine. 
</P><P>Utilizing the previous experiment in which we calculated the growth of our cells per caffeine molecule, we are able to estimate the total concentration of caffeine in each of the original caffeinated beverages. This is done by fitting the created equation that converts OD into predicted caffeine concentration. From caffeine concentration and the known dilution of caffeinated beverage added to each of the cultures, we can estimate the caffeine weight per beverage volume.
</P><P>These results are summarized below, and are very close to the values determined by the beverage manufacturers themselves. This serves as further reinforcement that our decaffeination operon works as intended, and even suggests a possible use in measuring the caffeine value of beverages. 
</P><P>
Finally, we took pictures of our cells grown in caffeinated beverages cultures, as shown below. Cells in the left tubes do not contain the decaffeination operon, while cells in the right tubes do contain the decaffeination operon. Both contain the <I>guaB</I> gene knocked out, and require xanthine or xanthine derived from caffeine in order to survive. As shown below, Coca-Cola, Diet Coke and Coffee, all of which contain caffeine, feature strong growth of the <I>guaB</I> knockouts with the decaffeination operon. However, Caffeine Free Coke shows no growth even with the operon, as there is no caffeine to be used as a xanthine source.
</P><H3><SPAN class="mw-headline" id="Transcriptional_Regulators">Transcriptional Regulators</SPAN></H3><P>As described earlier, we hypothesize that the large intergenic regions upstream of various genes in the CBB5 decaffeination operon, particularly upstream of <I>ndmA</I>, may contain methylxanthine regulated promoters.  We propose that <I>orf1</I> and <I>orf4</I>, which were annotated as putative regulators based on sequence homology, act as repressors for the promoters contained in the operon. 
</P><P>To test these hypotheses, we cloned the intergenic region upstream of the <I>ndmA</I> gene into a promoterless <I>lacZ</I> reporter plasmid, pRA301.  This vector enabled quantitative measurement of promoter strength using the β-galactosidase <A href="https://2010.igem.org/Team:BCCS-Bristol/Wetlab/BetaGalactosidaseAssays" class="external text" rel="nofollow">Miller assay</A>.  Additionally, we created compatible biobrick plasmids of each putative repressor, <I>orf1</I> and <I>orf4</I>, in order to assay their effect on <I>ndmA</I> promoter strength in the presence or absence of methylxanthine supplementation.
</P><P>We transformed these plasmids into the Top10 <I>E. coli</I> strain to create three strains: one containing only the <I>ndmA-lacZ</I> reporter plasmid, and co-transformants containing both the <I>ndmA-lacZ</I> plasmid and either <I>orf1</I> or <I>orf4</I> BioBrick plasmids.  All three <I>E. coli</I> strains were grown in m9 minimal media + .2% casein (+ appropriate supplement) at 30<SUP>o</SUP>C for 48 hrs, after which time Miller Assays were performed. The results, summarized in the figure below, provide strong evidence that expression of <I>orf4</I> leads to inhibited <I>ndmA</I> promoter functionality, while <I>orf1</I> expression provides negligible influence on the <I>ndmA</I> promoter. 
</P><P>A higher Miller Unit correlates to a higher level of <I>lacZ</I> production, effectively quantifying the degree of gene expression. As the Miller Unit of the strain co-expressed <I>ndmA</I> promoter with <I>orf4</I> is lower by a factor of roughly 4, it can be inferred that the protein encoded by <I>orf4</I> regulates the degree to which the <I>ndmA</I> promoter is expressed during transcription.
</P><P>It should be noted that while expression of <I>ndmA</I> drops significantly in this experiment, this does not imply that <I>orf4</I> only inhibits the expression of <I>ndmA</I>. This is discussed in the next section.
</P><H3><SPAN class="mw-headline" id="Inducible_Promoters">Inducible Promoters</SPAN></H3><P>As shown above, it appears that <I>orf4</I> acts as a transcriptional regulator of <I>ndmA</I> expression. One would expect a transcriptional regulator to regulate transcription in a way that is beneficial to the overall fitness of the organism. In certain situations in which a protein is necessary for survival, the transcriptional factor should up-regulate the production of said gene. Likewise, when a protein is not necessary, the transcriptional factor should down-regulate its production.
</P><P>We hypothesize that this is the way in which <I>orf4</I> regulates transcription of <I>ndmA</I>. In situations of high caffeine and related xanthine concentrations, the protein encoded by <I>orf4</I> up-regulates the expression of <I>ndmA</I>. In the experiment listed <A href="/Team:Austin_Texas/Caffeinated_coli#Transcriptional_Regulators" title="Team:Austin Texas/Caffeinated coli">above</A>, in which no xanthine derivative was added to the media, expression from the <I>ndmA</I> promoter is inhibited (as shown).
</P><P>In order to test this hypothesis, we subjected a strain of Top10 <I>E. coli</I> cells containing the orf4 gene and the <I>ndmA</I> promoter containing the <I>lacZ</I> gene to varying supplements of caffeine, theobromine, theophylline, and xanthine. Xanthine was shown to be insoluble at high concentrations, and was therefore removed from the experiment. The results of exposure to varying concentration levels is shown below.
</P><P>As predicted, the expression of <I>ndmA</I> appears to rise at higher caffeine concentrations. This implies that the protein encoded by orf4 relieves the repression of the ndmA promoter region as caffeine concentrations rise. This would allow for greater demethylation functionality of the decaffeination operon as more caffeine is provided to the cells, optimizing the operon's gene expression under varying substrate conditions. 
</P><P>From these experiments we conclude that the <I>ndmA</I> intergenic region contains a promoter that is negatively-regulated by the protein coded by <I>orf4</I> in the absence of methylxanthines.  Upon exposure to methylxanthines, it appears that <I>orf4</I>-mediated repression is relieved, leading to induction.
</P><H3><SPAN class="mw-headline" id="Results_Summary">Results Summary</SPAN></H3><P>The main accomplishments we achieved this summer include the following:
</P><UL><LI>Refactoring of decaffeination operon from <I>P. putida</I> into <I>E. coli</I></LI><LI>Construction of an auxotrophic selection method using our refactored operon, &quot;addicting&quot; <I>E. coli</I> to caffeine
</LI><LI>Growth of auxtrophic cells on common caffeinated beverages
</LI><LI>Discovery of a methyxanthine inducible promoter (<I>ndmA</I>) in the CBB5 operon
</LI><LI>Characterization of an open reading frame (<I>orf4</I>) in the CBB5 operon that acts as a transcription regulator
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Summers RM, Louie TM, Yu CL, Gakhar L, Louie KC, Subramanian M, &quot;Novel, highly specific N-demethylases enable bacteria to live on caffeine and related purine alkaloids.&quot; Journal of Bacteriology, 2012, vol 194, no 8, pg 2041-2049.
</P><P>Rodriguez del Rey Z, Granek EF, Sylvester S, &quot;Occurrence and concentration of caffeine in Oregon coastal waters.&quot; Marine Pollution Bulletin, 2012, vol 64, no 8, Pages 1417-1424.
</P><DIV id="footer"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Austin_Texas/Caffeinated_coli">http://2012.igem.org/Team:Austin_Texas/Caffeinated_coli</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Austin_Texas/Caffeinated_coli" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Austin_Texas/Caffeinated_coli" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Austin_Texas/Caffeinated_coli&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Austin_Texas/Caffeinated_coli&amp;oldid=301104" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>