<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2017.igem.org/Team:Austin_UTexas">HOME</A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Team">Meet the Team!</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Collaborations">  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Description">Description </A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/HP/Gold_Integrated">Integrated and Gold</A>&gt;</LI></NAV></DIV><DIV class="content_wrapper"><DIV class="column full_size"><DIV style="background-color: lightcyan;padding:14px;"><H2>Experiments</H2><P>Although bacteria can naturally synthesize GABA, we wanted to increase expression of the <I>gadB</I> gene and subsequently GABA production in order to imbue a more potent medicinal quality to our probiotic, with the idea that this GABA-overproducing probiotic can then be consumed by patients with bowel disorders or anxiety (1). Overexpression of the <I>gadB</I> gene will be accomplished by placing it under the control of either the P8 or P32 constitutive promoters from <I>Lactococcus lactis</I> (2).

To make our GABA-producing probiotic we first needed to assemble a GABA overexpression cassette plasmid using the Golden Gate assembly method. The intention here is that bacteria containing this GABA overexpression cassette plasmid should produce high levels of GABA. In short, Golden Gate Assembly is a new cloning method that allows for the creation of a multi-part DNA assembly (i.e. cassette plasmid) in a single reaction through the use of DNA parts containing specific, predefined suffixes and prefixes with recognition sites for Type IIs restriction enzymes (e.g. BsmBI and BsaI). The specificity of these suffixes and prefixes provides directionality of the desired DNA parts during the assembly process. For our purposes, we used the MoClo Yeast Tool Kit by John Dueber (3)



The first part of our Golden Gate assembly workflow was part assembly, in which the <I>gadB</I> gene and the P8/P32 promoters were individually cloned into the entry vector pYTK001 (Fig. 1). The gadB gene and P8/P32 promoter sequences contain flanking BsmBI sites that produce overhangs compatible with those cut by BsmBI in the entry vector pYTK001. Thus, BsmBI cloning should result in part plasmids containing the gadB gene and P8/P32 promoters set within the pYTK001 backbone. 
</P><FIGURE><FIGCAPTION>Figure 1. <I>gadB</I> gene and P8/P32 promoter part assembly process. Golden Gate compatible <I>gadB</I> and P8/P32 promoter sequences are cloned into the pYTK001 entry vector via BsmBI assembly.</FIGCAPTION></FIGURE><P>Additionally, the BsmBI sites and overhangs in pYTK001 are flanking a <I>gfp</I> reporter gene. During the part assembly process, our DNA sequences of interest should replace this <I>gfp</I> reporter gene. This provides a phenotypic screen that allows us to visually see which transformant colonies are negative and potentially positive. Under UV illumination, positive colonies containing our intended part plasmid assembly did not exhibit fluorescence under the UV illumination, while negative colonies did (Fig. 2.). The non-fluorescent colonies on the part plasmid transformation plates were miniprepped and subsequently sequence verified.</P><DIV class="caption">
Figure 2. <I>gadB gene</I> and P8/P32 promoter part plasmid <I>E. coli</I> transformations, compared to control transformations with pYTK001. Under UV illumination, transformants containing the correctly assembled part plasmids were non-fluorescent while negative transformants appeared fluorescent like colonies on the control plates.</DIV><P>After successfully creating the <I>gadB</I> gene and P8/P32 promoter part plasmids, the functionality of these part plasmids were then assessed by assembling them into cassette plasmids. 



To test if our <I>Lactococcus lactis</I> constitutive promoters function well within <I>E. coli</I>, we created test cassette plasmids containing the <I>E2-Crimson</I> reporter gene (which encodes a red fluorescent protein) inserted downstream of either the P8 or P32 promoters using BsaI Golden Gate assembly. To create these test cassette, we used the P8/P32 promoter part plasmids, <I>E2-Crimson</I> part plasmid, an M13 terminator part plasmid, connector part plasmids, and pYTK095 vector as the backbone (Fig. 3). If the P8 and P32 promoters are functional in <I>E. coli</I>, we should observe red fluorescence In colonies transformed with our test cassette plasmids.</P><FIGURE><FIGCAPTION>Figure 3. Golden Gate assembly process of the P8 and P32 test cassette plasmids.</FIGCAPTION></FIGURE><P>Upon first look, E. coli colonies transformed with these assemblies appeared purple-blue in color. This phenotype was due to the expression of the red fluorescent protein. Further, we noticed that E. coli colonies transformed with these assemblies fluoresced red under UV light, indicating that the P8 and P32 promoters are indeed expressing the E2-Crimson reporter gene and thus are functional in E. coli (Fig. 4).</P><DIV class="caption">
Figure 4. P8/P32 test cassette plasmid transformation plates, under normal and UV lights. Under normal lights, the colonies appeared purple-blue in color. Under UV, the colonies fluoresced red.</DIV><P>Using Golden Gate Assembly, we created cassette plasmids to test if the gadB gene could be overexpressed in E. coli via the P8 and P32 promoters. These cassette plasmids contained the gadB gene inserted downstream of either the P8 or P32 promoters. Part plasmids used for this cassette assembly included:</P><DIV class="caption">
Figure 5. Golden Gate assembly process of P8/<I>gadB</I> and P32/<I>gadB</I> test cassette plasmids.</DIV><P>Similar to the pYTK001 entry vector in part assembly, the pYTK095 backbone used for cassette assembly contained a <I>gfp</I> reporter gene that is replaced by sequences of interest. This allowed us to easily perform a phenotypic screen for positive colonies. Non-fluorescent colonies may potentially have had the correct cassette assembly, while fluorescent colonies did not (Fig. 6). The non-fluorescent colonies were then screened using colony PCR (Fig. 7). The positive colonies were then miniprepped and sequenced. The sequencing results indicated that in all of the samples there were several point mutations within the gadB gene. We hypothesized that overexpression of the <I>gadB</I> gene via the P8 and P32 constitutive promoters and the high-copy number ColE1 origin induced a high metabolic load on the cells, resulting in mutational degradation of <I>gadB</I>. To troubleshoot this problem, we decided to use a backbone containing the low-copy number p15A origin for the cassette assembly (Fig. 8)</P><DIV class="caption">
Figure 6. P8/gadB and P32/gadB cassette <I>E. coli</I> transformations alongside a pYTK095 control transformation under UV illumination. Potentially positive colonies containing the correct assemblies appeared non-fluorescent, while negative colonies appeared fluorescent.</DIV><DIV class="caption">
Figure 7. Gel results for colony PCR of P8/<I>gadB</I> and P32/<I>gadB</I> non-fluorescent transformants. The primer pair used for this PCR experiment targeted the region of the cassette plasmid containing the DNA inserts (i.e. connectors, P8/P32 promoters, <I>gadB</I> gene, and M13 terminator). The expected band length was 2 kb. Lane 1 contains a negative PCR control. Lane 2 contains a positive PCR control. Lanes 3-5 contain the PCR products from three p8/<I>gadB</I> colonies. Lanes 6-8 contain the PCR products from three p8/<I>gadB</I> colonies. Evidently, all of the picked colonies were positive for the desired amplified region.</DIV><DIV class="caption">
Figure 8. Modified Golden Gate assembly process of P8/<I>gadB</I> and P32/<I>gadB</I> test cassette plasmids using a backbone containing the low-copy number p15A origin.</DIV><P>After confirming <I>gadB</I> overexpression in <I>E. coli</I>, we want to assemble our final GABA overexpression cassette plasmid using the vector pMSP3535 as the backbone (Fig. 9). To do this, we first needed to make pMSP3535 Golden Gate compatible (i.e. free of BsaI restriction sites and containing correct overhangs for cassette assembly). We chose to work with pMSP3535 as it contains both a ColE1 origin for replication in<I>E. coli</I> and a pAMb1 origin for replication in Gram-positive bacteria including <I>Lactobacillus</I> species (4). Additionally, the pMSP3535 vector contains the resistance gene for erythromycin, which <I>Lactobacillus plantarum</I> is naturally susceptible to (5)

</P><DIV class="caption">
Figure 9. Golden Gate assembly of the GABA final overexpression cassette plasmid with the Golden Gate compatible pMSP3535 vector and the P8/P32 promoter, <I>gadB</I> gene, and M13 terminator part plasmids.</DIV><P>The process of making the pMSP3535 vector Golden Gate compatible involved two steps: 1) assembling the pMSP3535 backbone (pAMb1 origin and erythromycin resistance gene) with a new ColE1 origin; 2) assembling a <I>gfp</I> dropout part to the assembly of the pMSP3535 backbone and the new ColE1 origin (Fig. 10).</P><DIV class="caption">
Figure 10. Assembly workflow for creating the Golden Gate compatible pMSP3535 backbone. The first step involves replacing the ColE1 origin in the pMSP3535 backbone with a new ColE1 origin from pYTK001 to create a pMSP3535 + ColE1 assembly. The second step involves combining the pMSP3535 + ColE1 assembly with a <I>gfp</I> dropout to form the final Golden Gate compatible vector to be used to create our GABA overexpression plasmid. </DIV><P>Since the original pMSP3535 vector contains two illegal BsaI sites within the ColE1 origin, we sought to replace this ColE1 origin with a BsaI-free one isolated from pYTK001. This assembly process involved linearizing and adding BsmBI sites and compatible overhangs to the pMSP3535 backbone and the pYTK001 ColE1 origin via PCR. After the pMSP3535 backbone and ColE1 origin were successfully amplified by PCR (Fig. 11a), they were joined together using BsmBI assembly. Diagnostic PCR was performed on pMSP3535 + ColE1 minipreps from E. coli transformants to screen for positive samples containing both the pMSP3535 backbone and the ColE1 inserts (Fig. 11b). After confirming the presence of the pMSP3535 vector and ColE1 origin, we partially sequence-confirmed the two miniprep samples.</P><DIV class="caption">
Figure 11. (A) Agarose gel of successful PCR for the pMSP3535 backbone and the ColE1 origin. Lane 1 contains a negative PCR control. Lanes 2 and 3 contain the ColE1 origin (around 800 bp) and pMSP3535 backbone PCR products (around 4.3 kb). (B) Agarose gel of diagnostic PCR of miniprep samples from pMSP3535 + ColE1 transformants. Lane 1 contains a negative PCR control. Lane 2 contains a positive PCR control for the pMSP3535 backbone, while lanes 3 and 4 contain the pMSP3535 PCR from two miniprep samples. Lane 7 contains a positive PCR control for the ColE1 origin, while lanes 8 and 9 contain the ColE1 PCR from the same two miniprep samples. Evidently, the pMSP3535 backbone and ColE1 origin is present in both tested miniprep samples.</DIV><P>To this pMSP3535 + ColE1 assembly, we wanted to add a gfp dropout part containing internal BsaI sites that will generate overhangs compatible with those in the P8/P32 promoter and M13 terminator part plasmids. Additionally, the incorporation of this gfp dropout part will also allow us to visually screen for positive and negative transformants based on their fluorescence. BsmBI sites and compatible overhangs were added to the gfp dropout part by PCR amplifying it from pYTK047 (Fig. 12). We have been attempting to linearize and add BsmBI sites and overhangs to the positive pMSP3535 + ColE1 assemblies via PCR, with no success. However, results from diagnostic digests suggest that our assemblies may contain extra, undesired DNA such as IS elements (Fig. 13). Thus, as of right now, we are screening for more positive pMSP3535 + ColE1 transformants. Once we have troubleshooted this problem, the pMSP3535 + ColE1 and the <I>gfp</I> dropout PCR products will be joined through BsmBI assembly to form the final Golden Gate compatible pMSP3535 vector. 

</P><DIV class="caption">
Figure 12. Agarose gel of gfp dropout PCR from pYTK047. Lane 1 contains a negative PCR control. Lane 2 contains the <I>gfp</I> dropout PCR product with an expected band length of around 1 kb. </DIV><DIV class="caption">
Figure 13. Agarose gel of pMSP3535 + ColE1 assembly diagnostic digests. Lane 1 contains the undigested plasmid assembly. Lane 2 contains the ClaI-digested plasmid with expected 400 bp and 4.6 kb bands. The actual band generated is apparently above 10 kb. Lane 3 contains the XmnI-digested plasmid with expected 200 bp, 1.5 kb, and 3.3 kb bands. The generated band sizes were 2.5 kb and 4 kb. Lane 4 contains the KpnI-digested assembly with an expected 5.1 kb band. The generated band size was 6 kb. Lane 5 contains the Bg1II-digested assembly with an expected 5.1 kb band. The generated band size was 8 kb. </DIV><P>Since we are creating our Golden Gate compatible pMSP3535 vector in <I>E. coli</I>, we wanted to determine the natural susceptibility of <I>E. coli</I> to erythromycin as the minimum concentration to use has not been established clearly in the literature. Thus, we performed an erythromycin minimum inhibitory concentration test in liquid LB media (Fig. 14). After one-day incubation, we observed that <I>E. coli</I> was resistant up to around 150 µg/mL of erythromycin. From this experiment, we have determined that the optimal erythromycin concentration for selecting against E. coli in liquid culture is around 200-250 µg/mL.</P><DIV class="caption">
Figure 14. Erythromycin minimum inhibitory concentration tests for E. coli in liquid media. From left to right: 0 µg/mL, 50 µg/mL, 100 µg/mL, 150 µg/mL, 200 µg/mL, 250 µg/mL, 300 µg/mL, 350 µg/mL, and 400 µg/mL. </DIV><H2>References</H2><OL><LI>Yunes, R. A et al. GABA production and structure of <I>gadB</I>/<I>gadC</I> genes in <I>Lactobacillus</I> and <I>Bifidobacterium</I> strains from human microbiota. <I>Anaerobe</I>. 42: 197-204 (2016).</LI><LI>Zhu, D. et al. Isolation of strong constitutive promoters from <I>Lactococcus lactis</I> subsp. Lactis N8. <I>FEMS Microbiol Lett</I>. 363(16): pii: fnv107 (2015). </LI><LI>Lee, M. E. et al. A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. <I>ACS Synth. Biol</I>. 4(9): 975-86 (2015). </LI><LI>Pérez-Arellano, I. et al. Construction of Compatible Wide-Host-Range Shuttle Vectors for Lactic Acid Bacteria and <I>Escherichia coli</I>. Plasmid. 46(2): 106-16 (2001).</LI><LI>Flórez A. B. et al. Susceptibility of <I>Lactobacillus plantarum</I> Strains to Six Antibiotics and Definition of New Susceptibility–Resistance Cutoff Values. <I>Microbial Drug Resistance</I>. 12(4): 252-56 (2007).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>