<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2017.igem.org/Team:Austin_UTexas">HOME</A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Team">Meet the Team!</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Collaborations">  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Description">Description </A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Austin_UTexas/HP/Gold_Integrated">Integrated and Gold</A>&gt;</LI></NAV></DIV><DIV class="content_wrapper"><DIV class="column full_size"><DIV style="background-color: lightcyan;padding:14px; font-family: verdana"><DIV class="column_full_size_inner"><H2 style="font-family: verdana; font-size: 45px; text-align: center; color: rgb(0, 184, 230)">Results</H2><DIV class="column full_size"><P style="font-family: verdana">Although bacteria can naturally synthesize GABA, we wanted to <B>increase expression of the <I>gadB</I> gene and subsequently GABA production in order to give our intended probiotic, <I>Lactobacillus plantarum</I>, a more potent medicinal quality</B>, with the idea that this GABA-overproducing probiotic can then be consumed by patients with bowel disorders, hypertension or anxiety (1). Overexpression of the <I>gadB</I> gene will be accomplished by placing it under the control of either the P8 or P32 constitutive promoters from <I>Lactococcus lactis</I> (2).

</P><P style="font-family: verdana">To make our GABA-producing probiotic, we ultimately needed to assemble a GABA overexpression cassette plasmid. The intention is that bacteria containing this GABA overexpression cassette plasmid should produce high levels of GABA. In order to assemble this plasmid, we decided to utilize the Golden Gate Assembly method. In short, Golden Gate Assembly is a relatively new cloning method that allows for the creation of a multi-part DNA assembly (i.e. cassette plasmid) in a single reaction through the use of DNA parts containing specific, predefined suffixes and prefixes with recognition sites for Type IIs restriction enzymes (e.g. BsmBI and BsaI). The specificity of these suffixes and prefixes provides directionality of the desired DNA parts during the assembly process. For our purposes, we used the MoClo Yeast Tool Kit developed by John Dueber (3).</P><P style="font-family: verdana">We decided to first assemble and test our Golden Gate plasmids in <I>E. coli</I>, which was chosen due to the ease in which we could genetically manipulate it. We then wanted to use these Golden Gate plasmids to genetically manipulate <I>L. plantarum</I>. This part of the project required us to assemble a Golden Gate compatible shuttle vector (that is replicable in both <I>E. coli</I> and <I>L. plantarum </I>) and transform <I>L. plantarum</I>.  Our experimental results are detailed below. </P></DIV><DIV class="clear"><STRONG><H2 style="font-family: verdana; font-size: 17px; text-align: center; color:  rgb(0, 184, 230)">Click on one of the images below to learn more about our results!</H2></STRONG><DIV class="column sixth_size"><A href="#section1"><P>Part plasmid assembly </P></A></DIV><DIV class="column sixth_size"><A href="#section2"><P>Testing constitutive <I>Lactococcal</I> Promoters in <I>E. coli</I></P></A></DIV><DIV class="column fifth_size"><A href="#section3"><P> Testing <I>gadB</I> Overexpression in <I>E.
 coli</I></P></A></DIV><DIV class="column sixth_size"><A href="#section4"><P>Creating a Golden Gate compatible shuttle vector</P></A></DIV><DIV class="column sixth_size"><A href="#section5"><P><I>Lactobacillus plantarum</I> transformation</P></A></DIV><DIV class="naviSection" id="section1"><H2 style="font-family: verdana; font-size: 30px; text-align: center">Part plasmid assembly</H2><P style="font-family: verdana">The first part of our Golden Gate assembly workflow was part assembly, in which the <I>gadB</I> gene and the P8/P32 promoters were individually cloned into the entry vector pYTK001 <B>(Fig. 1)</B>. The <I>gadB</I> gene and P8/P32 promoter sequences contain flanking BsmBI sites that produce overhangs compatible with those cut by BsmBI in the entry vector pYTK001. Thus, BsmBI cloning should result in part plasmids containing the <I>gadB</I> gene and P8/P32 promoters set within the pYTK001 backbone. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 1.</B><I>gadB</I> gene and P8/P32 promoter part assembly process. Golden Gate compatible <I>gadB</I> and P8/P32 promoter sequences are cloned into the pYTK001 entry vector via BsmBI assembly.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Additionally, the BsmBI sites and overhangs in pYTK001 are flanking a <I>gfp</I> reporter gene. During the part assembly process, our DNA sequences of interest replaced this <I>gfp</I> reporter gene. This provided a phenotypic screen that allowed us to visually see which transformant colonies were negative and potentially positive. <B>Under UV illumination, positive colonies containing our intended part plasmid assembly did not exhibit fluorescence under the UV illumination, while negative colonies did (Fig. 2)</B>. The non-fluorescent colonies on the part plasmid transformation plates were miniprepped and subsequently sequence verified. Other part plasmids used in subsequent cassette assemblies were directly obtained from UT's Golden Gate Part Repository.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 2.</B><I>gadB gene</I> and P8/P32 promoter part plasmid <I>E. coli</I> transformations, compared to control transformations with pYTK001. Under UV illumination, transformants containing the correctly assembled part plasmids were non-fluorescent while negative transformants appeared fluorescent like colonies on the control plates.</DIV></DIV></DIV></DIV><P>Back to Top</P></DIV><DIV class="naviSection" id="section2"><H2 style="font-family: verdana; font-size: 30px; text-align: center">Testing constitutive <I>Lactococcal</I> promoters in <I>E. coli</I></H2><P style="font-family: verdana"><B>After successfully creating the <I>gadB</I> gene and P8/P32 promoter part plasmids, the functionality of these part plasmids were then assessed by assembling them into cassette plasmids.</B></P><P style="font-family: verdana">Although the functionality of the <I>Lactococcal lactis</I> P8 and P32 constitutive promoters has been well characterized in Gram-positive bacteria such as <I>Lactobacillus</I>, we wanted to test if these promoters could function well within <I>E. coli</I>. To do this, we created test cassette plasmids containing the <I>E2-Crimson</I> reporter gene (which encodes a red fluorescent protein) inserted downstream of either the P8 or P32 promoters using BsaI Golden Gate assembly. To create these test cassettes, we used the P8/P32 promoter part plasmids, the <I>E2-Crimson</I> part plasmid, an M13 terminator part plasmid, connector part plasmids, and the pYTK095 vector as the backbone <B>(Fig. 3).</B><B>If the P8 and P32 promoters are functional in <I>E. coli</I>, we expected to observe red fluorescence in colonies transformed with our test cassette plasmids.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 3.</B> Golden Gate assembly process of the P8 and P32 test cassette plasmids.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Upon first look, <I>E. coli</I> colonies transformed with these assemblies appeared purple-blue in color. This phenotype was due to the expression of the red fluorescent protein. Further, we noticed that <I>E. coli</I> colonies transformed with these assemblies fluoresced red under UV light, indicating that <B>the P8 and P32 promoters are indeed expressing the <I>E2-Crimson</I> reporter gene and thus are functional in <I>E. coli</I><B>(Fig. 4).</B></B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 4.</B> P8/P32 test cassette plasmid transformation plates, under normal (faced up) and UV lights (faced down). Under normal lights, the colonies appeared purple-blue in color. Under UV illumination, the colonies fluoresced red.</DIV></DIV></DIV></DIV><P>Back to Top</P></DIV><DIV class="naviSection" id="section3"><H2 style="font-family: verdana; font-size: 30px; text-align: center">Testing <I>gadB</I> overexpression in <I>E. coli</I></H2><P style="font-family: verdana"><B>Using Golden Gate Assembly, we created cassette plasmids to test if the <I>gadB</I> gene could be overexpressed in <I>E. coli</I> via the P8 and P32 promoters. </B> These cassette plasmids contained the <I>gadB</I> gene inserted downstream of either the P8 or P32 promoters. To make our <I>gadB</I> test cassette plasmids, we used pYTK095 as our backbone and part plasmids containing the P8/P32 promoters, <I>gadB</I> gene, M13 terminator, and connector sequences <B>(Fig. 5).</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 5.</B> Golden Gate assembly process of P8/<I>gadB</I> and P32/<I>gadB</I> test cassette plasmids.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Similar to the pYTK001 entry vector in part assembly, the pYTK095 backbone used for cassette assembly contained a <I>gfp</I> reporter gene that is replaced by sequences of interest. This allowed us to easily perform a phenotypic screen for positive colonies. <B>Non-fluorescent colonies may potentially have had the correct cassette assembly, while fluorescent colonies did not (Fig. 6).</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 6.</B> P8/<I>gadB</I> and P32/<I>gadB</I> cassette E. coli transformations alongside a pYTK095 control transformation under UV illumination. Potentially positive colonies containing the correct assemblies appeared non-fluorescent, while negative colonies appeared fluorescent.</DIV></DIV></DIV></DIV><P style="font-family: verdana">The non-fluorescent colonies were then screened using colony PCR, and positive colonies were then miniprepped and sequenced. <B>The sequencing results indicated that there were several mutations within the <I>gadB</I> gene in the samples.</B> These mutations are recorded in <B>Table 1.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Table 1.</B><I>gadB</I> mutations in sequenced P8/<I>gadB</I> and P32/<I>gadB</I> overexpression cassette plasmids.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Along with being one of the canonical amino acids utilized in protein synthesis, glutamate plays an important role as the main amino-group donor in the biosynthesis of nitrogen-containing compounds such as amino acids and nucleotides (4, 5). Thus, we hypothesized that <I>gadB</I> overexpression via the P8 and P32 constitutive promoters and the high-copy-number ColE1 origin induced a high metabolic load on the cells by shunting away glutamate from essential anabolic pathways. Additionally, having high <I>gadB</I> expression does not confer a selective advantage to the cells. We believed that transformants containing the mutationally inactivated <I>gadB</I> gene were favored in the population, as &quot;breaking&quot; the metabolically-taxing <I>gadB</I> gene gave these transformants a competitive advantage, allowing them to utilize glutamate sources towards growth. In contrast, transformants containing the functional <I>gadB</I> gene were selected against due to having a depletion of glutamate needed for important cellular processes. </P><P style="font-family: verdana"><B>To troubleshoot this problem, we decided to use a backbone containing the low-copy number p15A origin for the cassette assembly (Fig. 7).</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 7.</B> Golden Gate assembly process of P8/<I>gadB</I> and P32/<I>gadB</I> test cassette plasmids using a backbone containing the low-copy number p15A origin.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Copy number reduction of our <I>gadB</I> overexpression cassette plasmids (accomplished by replacing the ColE1 origin with the p15A origin) was expected to decrease the overall amount of <I>gadB</I> expression, thereby decreasing the metabolic load imposed on the cells and the mutation rates observed in the <I>gadB</I> gene. <B>However, we also observed several mutations within the <I>gadB</I> gene in the potentially positive cassette plasmids sequenced.</B> The observed mutations are detailed in Table 2.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Table 2.</B><I>gadB</I> mutations in sequenced P8/<I>gadB</I>/p15A and P32/<I>gadB</I>/p15A overexpression cassette plasmids.</DIV></DIV></DIV></DIV><P style="font-family: verdana"><B>Given our experimental results and the fact that inducibly expressed genetic devices are more evolutionarily stable than constitutively expressed ones (6), we attempted to inducibly express the <I>gadB</I> gene using the regulatory elements of the <I>lac</I> operon to see if expression and maintenance of a stable <I>gadB</I> gene was possible in <I>E. coli</I></B>. Our IPTG-inducible <I>gadB</I> expression cassette plasmid was assembled using a <I>lac</I> promoter and operator part plasmid, the <I>gadB</I> gene part plasmid, the M13 terminator part plasmid, connector part plasmids, a CP25 promoter part plasmid, a <I>lacI</I> part plasmid, and the SpecR and p15A origin part plasmid <B>(Fig. 8)</B>. Under this assembled regulatory system, in the absence of IPTG (an analog of the allolactose inducer) the LacI repressor will bind to the <I>lac</I> operator region to block transcription of the <I>gadB</I> gene. When present, IPTG will act as an inducer and bind to the LacI repressor to decrease its binding affinity for the <I>lac</I> operator, thereby allowing for <I>gadB</I> expression. This IPTG-inducible system provides us with a mechanism of controlling <I>gadB</I> expression. <B>Positive colonies have been identified and sequence verification is currently underway.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 8.</B> Golden Gate assembly process of IPTG-inducible <I>gadB</I> expression cassette plasmid.</DIV></DIV></DIV></DIV><P>Back to Top</P></DIV><DIV class="naviSection" id="section4"><H2 style="font-family: verdana; font-size: 28px; text-align: center">Creating a Golden Gate compatible shuttle vector</H2><P style="font-family: verdana">We wanted to assemble our final GABA overexpression cassette plasmid using the shuttle vector pMSP3535 as the backbone <B>(Fig. 9).</B> To do this, we first needed to make pMSP3535 Golden Gate compatible (i.e. free of BsaI restriction sites and containing correct overhangs for cassette assembly). We chose to work with pMSP3535 as it contains both a ColE1 origin for replication in <I>E. coli</I> and a pAMb1 origin for replication in Gram-positive bacteria including <I>Lactobacillus</I> species (7), allowing us to easily transplant our cassette plasmid from <I>E. coli</I> to <I>Lactobacillus plantarum</I> once assembled. Additionally, the <STRONG>pMSP3535 vector contains the resistance gene for erythromycin, of which <I>Lactobacillus plantarum</I> is naturally susceptible (8).</STRONG></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 9.</B> Golden Gate assembly of the GABA final overexpression cassette plasmid with the Golden Gate compatible pMSP3535 vector and the P8/P32 promoter, <I>gadB</I> gene, and M13 terminator part plasmids.</DIV></DIV></DIV></DIV><P style="font-family: verdana">The process of making the pMSP3535 vector Golden Gate compatible involved two steps: 1) assembling the pMSP3535 backbone (pAMb1 origin and erythromycin resistance gene) with a new ColE1 origin; 2) assembling a gfp dropout part to the assembly of the pMSP3535 backbone and the new ColE1 origin <B>(Fig. 10).</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 10.</B> Assembly workflow for creating the Golden Gate compatible pMSP3535 backbone. The first step involves replacing the ColE1 origin in the pMSP3535 backbone with a new ColE1 origin from pYTK001 to create a pMSP3535 + ColE1 assembly. The second step involves combining the pMSP3535 + ColE1 assembly with a <I>gfp</I> dropout to form the final Golden Gate compatible vector to be used to create our GABA overexpression plasmid.</DIV></DIV></DIV></DIV><P style="font-family: verdana"><B>Since the original pMSP3535 vector contained two illegal BsaI sites within the ColE1 origin, we sought to replace this ColE1 origin with a BsaI-free one isolated from pYTK001.</B> This assembly process involved linearizing and adding BsmBI sites and compatible overhangs to the pMSP3535 backbone and the pYTK001 ColE1 origin via PCR. After the pMSP3535 backbone and ColE1 origin were successfully amplified by PCR <B>(Fig. 11a)</B>, they were joined together using BsmBI assembly. Diagnostic PCR was performed on pMSP3535 + ColE1 minipreps from <I>E. coli</I> transformants to screen for positive samples containing both the pMSP3535 backbone and the ColE1 inserts (Fig. 11b). <B>After confirming the presence of the pMSP3535 vector and ColE1 origin, we partially sequence-confirmed the two miniprep samples.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Figure 11. (A) Agarose gel of successful PCR for the pMSP3535 backbone and the ColE1 origin. The ladder used is the 1KB NEB ladder. Lane 1 contains a negative PCR control. Lanes 2 and 3 contain the ColE1 origin (around 800 bp) and pMSP3535 backbone PCR products (around 4.3 kb). (B) Agarose gel of diagnostic PCR of miniprep samples from pMSP3535 + ColE1 transformants. Lane 1 contains a negative PCR control. Lane 2 contains a positive PCR control for the pMSP3535 backbone, while lanes 3 and 4 contain the pMSP3535 PCR from two miniprep samples. Lane 7 contains a positive PCR control for the ColE1 origin, while lanes 8 and 9 contain the ColE1 PCR from the same two miniprep samples. Evidently, the pMSP3535 backbone and ColE1 origin is present in both tested miniprep samples.</DIV></DIV></DIV></DIV><P style="font-family: verdana">To this pMSP3535 + ColE1 assembly, we wanted to add a <I>gfp</I> dropout part containing internal BsaI sites that will generate overhangs compatible with those in the P8/P32 promoter and M13 terminator part plasmids. Additionally, the incorporation of this <I>gfp</I> dropout part will also allow us to visually screen for positive and negative transformants based on their fluorescence. BsmBI sites and compatible overhangs were added to the <I>gfp</I> dropout part by PCR amplifying it from pYTK047. We have been attempting to linearize and add BsmBI sites and overhangs to the positive pMSP3535 + ColE1 assemblies via PCR, with no success. However, results from diagnostic digests suggested that our assemblies may have contained extra, undesired DNA such as IS elements <B>(Fig. 12).</B><B>Thus, as of right now, we are screening for more positive pMSP3535 + ColE1 transformants.</B> Once we have troubleshot this problem, the pMSP3535 + ColE1 and the <I>gfp</I> dropout PCR products will be joined through BsmBI assembly to form the final Golden Gate compatible pMSP3535 vector. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 12.</B> Agarose gel of pMSP3535 + ColE1 assembly diagnostic digests. The ladder used is the 1KB NEB ladder. Lane 1 contains the undigested plasmid assembly. Lane 2 contains the ClaI-digested plasmid with expected 400 bp and 4.6 kb bands. The actual band generated is apparently above 10 kb. Lane 3 contains the XmnI-digested plasmid with expected 200 bp, 1.5 kb, and 3.3 kb bands. The generated band sizes were 2.5 kb and 4 kb. Lane 4 contains the KpnI-digested assembly with an expected 5.1 kb band. The generated band size was 6 kb. Lane 5 contains the Bg1II-digested assembly with an expected 5.1 kb band. The generated band size was 8 kb.</DIV></DIV></DIV></DIV><H4 style="font-family: verdana">Assessing erythromycin susceptibility of <I>E. coli</I></H4><P style="font-family: verdana">Because we are creating our Golden Gate compatible pMSP3535 shuttle vector in <I>E. coli</I>, we wanted to determine the natural susceptibility of <I>E. coli</I> to erythromycin as the minimum concentration to use has not been established clearly in the literature. Thus, we performed an erythromycin minimum inhibitory concentration test in liquid LB media <B>(Fig. 13).</B> After one-day incubation, we observed that <I>E. coli</I> was resistant up to around 150 µg/mL of erythromycin. <B>From this experiment, we have determined that the optimal erythromycin concentration for selecting against <I>E. coli</I> in liquid culture is around 200-250 µg/mL.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:539px;"><DIV class="thumbcaption"><B>Figure 13.</B> Erythromycin minimum inhibitory concentration tests for <I>E. coli</I> in liquid media. From left to right: 0 µg/mL, 50 µg/mL, 100 µg/mL, 150 µg/mL, 200 µg/mL, 250 µg/mL, 300 µg/mL, 350 µg/mL, and 400 µg/mL.</DIV></DIV></DIV></DIV><P>Back to Top</P></DIV><DIV class="naviSection" id="section5"><DIV style="background-color: lightcyan;padding:14px;"><H2 style="font-family: verdana; font-size: 30px; text-align: center"><I>Lactobacillus plantarum</I> transformation</H2><P style="font-family: verdana"> In order to grow <I>Lactobacillus plantarum</I>, we used MRS broth in a CO<SUB>2</SUB> incubator without shaking. After characterizing the growth of the bacteria, we aimed to transform it with pMSP3535. We first attempted several protocols, including Landete 2014 (11) and Speer 2012 (12). However, we found success using a variation of the Welker protocol (13). After preparing the necessary solutions, we followed the Welker protocol with some minor differences, including inoculation of bacterial stocks in 10 mL of MRS broth rather than 25mL; utilizing a CO<SUB>2</SUB> incubator without shaking; subculturing cells in 0.9M NaCl rather than glycine, transforming with 100 ng of plasmid DNA  instead of the 200 ng that the protocol recommends, and electroporating with a resistance of 3kV and 600Ω. For more information about the protocols including broth recipe and transformation procedure, click <A href="https://2017.igem.org/Team:Austin_UTexas/Protocols">here!</A></P><P style="font-family: verdana"> Our first plasmid we chose to transform <I> Lactobacillus plantarum </I> with was pMSP3535. This plasmid expresses erythromycin resistance and contains the pAMB1 origin. 
After cells were electroporated, they recovered overnight in the appropriate recovery media. They were plated on MRS agar plates supplemented with 10 μg/mL erythromycin and left to grow for 2 days. The results of our transformation can be seen in <B>Figure 14</B>.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><STRONG>Figure 14.</STRONG> Transformations of <I>Lactobacillus plantarum</I> with pMSP3535. Plate A corresponds to the negative control: electroporated cells at 3kV and 600 Ω with no plasmid DNA. Plate B corresponds to cells with 100 ng of pMSP3535 plasmid DNA electroporated at 2kV and 400 Ω. Plate  cells were electroporated at 2.5kV and 400 Ω with 100 ng of pMSP3535 plasmid DNA. Plate D corresponds to the cells electroporated at 3kV and 600 Ω with 100 ng of pMSP3535.</DIV></DIV></DIV></DIV><P style="font-family: verdana">Colonies from each transformation plate were grown up in MRS broth supplemented with 10 μg/mL erythromycin. The next day, they were streaked on  10μg/mL erythromycin MRS agar plates to verify resistance to erythromycin. The overnight cultures and re-streaks are shown in <B>Figure 15</B> and <B>Figure 16</B> respectively.</P><STRONG><P style="font-family: verdana"> Genomic and plasmid sequence verification is underway.</P></STRONG><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:310px;"><DIV class="thumbcaption"><B><B>Figure 15.</B></B>Overnight cultures of transformed<I>Lactobacillus plantarum</I> were grown in erythromycin. Tube A is the negative control: untransformed <I> Lactobacillus plantarum</I> in MRS broth supplemented with 10μg/mL erythromycin. Tube B was our positive control: transformed <I> Lactobacillus plantarum</I> in MRS broth. Tube C is the experimental cells: transformed <I>Lactobacillus plantarum</I> grown in MRS broth supplemented in 10μg/mL erythromycin.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B><B>Figure 16.</B></B> This figure displays a transformed <I>Lactobacillus plantarum</I> re-streak plated on MRS agar supplemented with 10μg/mL erythromycin beside a negative control. Plate A corresponds to untransformed <I>L. plantarum</I> on the selective media and Plate B corresponds to the transformed bacteria on selective media.</DIV></DIV></DIV></DIV><H2 style="font-family: verdana; font-size: 30px; text-align: center">References</H2><OL style="font-size:13px; font-family: verdana"><LI>Yunes, R. A et al. GABA production and structure of <I>gadB</I>/<I>gadC</I> genes in <I>Lactobacillus</I> and <I>Bifidobacterium</I> strains from human microbiota. <I>Anaerobe</I>. 42: 197-204 (2016).</LI><LI>Zhu, D. et al. Isolation of strong constitutive promoters from <I>Lactococcus lactis</I> subsp. Lactis N8. <I>FEMS Microbiol Lett</I>. 363(16): pii: fnv107 (2015). </LI><LI>Lee, M. E. et al. A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. <I>ACS Synth. Biol</I>. 4(9): 975-86 (2015). </LI><LI>Commichau, F. M., et al. Glutamate metabolism in <I>Bacillus subtilis</I>: Gene Expression and Enzyme Activities Evolved To Avoid Futile Cycles and to allow Rapid Responses to Perturbations of the System. <I>AMS</I>. 190(10): 3557-64 (2008).</LI><LI>Sarkar, P.K. and Nout, M.J.R. (Eds.) 2014. Handbook of Indigenous Foods Involving Alkaline Fermentation. CRC Press/Taylor &amp; Francis Group, p. 629.</LI><LI>Sleight, S. C. et al. Designing and engineering evolutionary robust genetic circuits. Journal of Biological Engineering. (4):12 (2010).</LI><LI>Pérez-Arellano, I. et al. Construction of Compatible Wide-Host-Range Shuttle Vectors for Lactic Acid Bacteria and <I>Escherichia coli</I>. Plasmid. 46(2): 106-16 (2001).</LI><LI>Flórez A. B. et al. Susceptibility of <I>Lactobacillus plantarum</I> Strains to Six Antibiotics and Definition of New Susceptibility–Resistance Cutoff Values. <I>Microbial Drug Resistance</I>. 12(4): 252-56 (2007).</LI><LI> Cox, C. P., &amp; Briggs, M. (1954). Experiments On Growth Media For <I>Lactobacilli</I>. Journal of Applied Bacteriology, 17(1), 18-26. doi:10.1111/j.1365-2672.1954.tb02019.x</LI><LI>Man, J. C., Rogosa, M., &amp; Sharpe, M. E. (1960). A Medium For The Cultivation Of <I>Lactobacilli</I>. Journal of Applied Bacteriology, 23(1), 130-135. doi:10.1111/j.1365-2672.1960.tb00188.x</LI><LI>Speer, M., &amp; Richard, T. (n.d.).<I> Lactobacillus</I> transformation (Speer 2012). Retrieved May 15, 2017, from https://openwetware.org/wiki/Lactobacillus_transformation_(Speer_2012)</LI><LI>Landete, J. M., Arqués, J. L., Peirotén, Á, Langa, S., &amp; Medina, M. (2014). An improved method for the electrotransformation of lactic acid bacteria: A comparative survey. Journal of Microbiological Methods, 105, 130-133. doi:10.1016/j.mimet.2014.07.022</LI><LI>Dennis L. Welker, Joanne E. Hughes, James L. Steele, Jeff R. Broadbent; High-efficiency electrotransformation of <I>Lactobacillus casei</I>, FEMS Microbiology Letters, Volume 362, Issue 2, 1 January 2015, Pages 1–6, https://doi.org/10.1093/femsle/fnu033</LI><LI>Murphy, M. G., &amp; Condon, S. (1984). Comparison of aerobic and anaerobic growth of <I>Lactobacillus plantarum</I> in a glucose medium. Archives of Microbiology, 138(1), 49-53. </LI><LI>GABA 3D ball-and-stick model, viewed 1 November 2017, by Jynto, from <A href="https://commons.wikimedia.org/wiki/File:GABA_3D_ball.png#file">Wikimedia Commons.</A></LI></OL><A href="#navtop"><P>Back to Top</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>