<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HK_HCY_LFC_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HK HCY LFC/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="navigation-con"><DIV class="navigation-div"><DIV style="padding-left: 28%;"><A class="link" href="https://2018.igem.org/Team:HK_HCY_LFC">Home</A><DIV class="menu"><A class="link" style="width: 105px">Team</A><DIV class="menu-content" style="width: 137px"><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Team">Team Members</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Collaborations">Collaborations</A></DIV></DIV><DIV class="menu"><A class="link" style="width: 97px">Project</A><DIV class="menu-content" style="width: 129px"><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Description">Description</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Design">Design</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Experiments">Experiments</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Notebook">Notebook</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/InterLab">InterLab</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Model">Model</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Results">Results</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Demonstrate">Demonstration</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Improve">Improvement</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Attributions">Attributions</A></DIV></DIV><A class="link" style="width: 108px; top: -8px;" href="https://2018.igem.org/Team:HK_HCY_LFC/Parts">Parts Overview</A><A class="link" href="https://2018.igem.org/Team:HK_HCY_LFC/Safety">Safety</A><DIV class="menu"><A class="link" style="width: 108px; padding-top: 8px;">Human Practices</A><DIV class="menu-content" style="width: 142px"><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Human_Practices">Human Practices</A><A class="sublink" href="https://2018.igem.org/Team:HK_HCY_LFC/Public_Engagement">Education &amp; Engagement</A></DIV></DIV><A class="link" href="https://igem.org/2018_Judging_Form?team=HK_HCY_LFC">Judging Form</A></DIV></DIV></DIV><METACHARSET><TITLE>Team:HK HCY LFC - 20118.igem.org</TITLE><DIV class="description-div" style="left: 7%; height: 6500px; width: 85%;"><P class="section-title" style="top: 70px;">Description</P><DIV class="description-title" style="font-size: 20px; padding: 0px 15px; padding-top: 30px;"><A href="#SCA">Spinocerebella Ataxia</A></DIV><DIV class="description-title" style="font-size: 20px; padding: 0px 15px;"><A href="#GQUAD">G-quadruplex</A></DIV><DIV class="description-title" style="font-size: 20px; padding: 0px 15px;"><A href="#DNA">DNA Tweezer Nanomachine</A></DIV><P class="section-title" style="padding-bottom: 0px;">Spinocerebellar Ataxia</P><DIV class="description-title" style="padding-top: 15px;"><P class="description-text">
                Spinocerebellar ataxia (SCA) belongs to trinucleotide repeat expansion disorder, an incurable dominantly inherited neurodegenerative disease. The age of onset usually happens during adulthood (between 20-40 years old), [1] but it will become earlier and earlier when the expanded DNA is inherited by the offspring. Sadly, all types of SCA cannot be cured now and the degeneration only slows down through physical therapy and medicine. There are not more than 1000 SCA patients in Hong Kong. The symptoms vary with different types of SCA but there are some common symptoms like poor body coordination and speech disorder. [2] Doctors observe the patients and determine whether the patients suffer from SCA according to the International Cooperative Ataxia Rating Scale. Doctors also assess the family history of SCA. Genetic testing is used to identify the expansion in DNA. Genetic testing is available for SCAs1, 2, 3, 6, 7 and 8 in Hong Kong.
            </P><P class="description-text">
                Spinocerebellar ataxia type 3(SCA3), also known as Machado-Joseph disease, [2] is the most common type of SCA in countries such as United States, China and Germany. [3] It is caused by a repeat expansion of CAG in the ATXN3 gene of patients. 1 SCA3 is a condition characterized by progressive problems with movement. SCA3 can show a variety of symptoms including tremor, stiff muscles and neuropathy. Some individuals may have twitching movements of the face and tongue. There may be abnormal movements of the eyes. [3] Most of the patients cannot stand and walk well so a wheelchair is necessary. We can extract the serum to detect potential biomarkers (miR- 25, miR-125b, miR-29a and miR-34b) for SCA3 and determine whether the patients suffer from SCA3. [] In Hong Kong, two months are needed to obtain the result of genetic testing of SCA3. [4]
            </P><P class="description-text"><SPAN style="text-decoration: underline;">References</SPAN>
                [1] HKSCAA疾病簡介 From https://www.hkscaa.org/main_about_sca.php
                
                [2]Dominant Spinocerebellar Ataxias (SCA) http://www.ataxiacenter.umn.edu/aboutataxia/hereditary/sca/home.html
                
                [3]Tan EK and Ashizawa T (2001). Genetic Testing in Spinocerebellar Ataxias. Archives of neurology, 58(2):191-195.
                
                [4]Department of Health, Laboratory User Guides,CGS-LAB-MQS-MENU_01_2018
            </P></DIV><P class="section-title" style="padding-bottom: 0px;">G-quadruplex</P><DIV class="description-title" style="padding-top: 15px;"><P class="description-text">
                The DNA structure that has been used for DNA nanomachines known as the guanine quadruplex or G-quadruplex. A G-quadruplex (Figure 1A) is a DNA secondary structure that forms when two or more guanine tetrads stack in a planar fashion through π-stacking.[1] The guanine tetrads themselves are held together through a combination of Hoogstein and Watson–Crick bonding. The quadruplex can be contained to a single strand or can be formed by multiple strands and there are many potential orientations of G-quadruplexes based on factors such as loop length and quantity of tetrads.[2]
            </P></DIV><DIV class="description-title">Formation of G-quadruplex
            <P class="description-text">
                In double stranded DNA, the opportunity for forming G-quadruplexes arises during DNA replication, transcription and repair when DNA is rendered transiently single stranded through the breaking of Watson-Crick base pairing, which would permit the alternative Hoogsteen base pairing present in G-quadruplexes to take place.
                
                In addition, it can be envisaged that G-quadruplex formation could be favoured by superhelical stress, molecular crowding as well as specific G-quadruplex binding proteins.[3]
            </P></DIV><DIV class="description-title">What are aptamers?
            <P class="description-text">
                They are the DNA structures, the potential for responsiveness to pH,[4] light,[5-8] or small molecule targets[9-12] through incorporating DNA secondary structures such as DNA triplex structures[13,14] or the intercalated cytosine motif,[15-17] thus allowing us to create responsive nanostructures that can be classified as nanomachines. This responsiveness can also be derived from pre-existing artificially selected for motifs called “aptamers” which selectively bind proteins and small molecules.[18] Responsiveness to these cues can embed the ability to selectively deliver or trigger based on microenvironment cues, which enables us to use a DNA nanostructure to act as a biosensor which detects miRNA biomarkers.[1]
            </P></DIV><DIV class="description-title">What are the advantages of using aptamer-based sensors?
            <P class="description-text">
                Aptamer-based sensors are less expensive and more stable than antibodies, giving them distinct advantages in biosensing and diagnostics.[19] Similarly, aptamers also demonstrate high thermostability and have similar target-binding specificities to antibody–antigen interactions.[20]
                <SPAN style="text-decoration: underline;">Figure 1A[1]</SPAN></P></DIV><DIV class="description-title">The application of G-quadruplex in our project?
            <P class="description-text">
                In our project we will use G-quadruplex to detect our target miRNA. When the desired target hybridizes to the target recognition sequence, it closes the device and allows the split strand G-quadruplex to form its secondary structure, thus allowing it to function as an aptamer and bind to hemin. This allows 2,2′-azino-bis(3-ethylbenzothiazoline-6-sulfonic acid) (ABTS) to be oxidized in the presence of hydrogen peroxide and produces a colour change as a signal. We are using the device to detect whether a person suffers from Spinocerebellar ataxia.
            </P><P class="description-text"><SPAN style="text-decoration: underline;">References</SPAN>
                [1] S. M. Douglas, I. Bachelet, G. M. Church, Science 2012, 335, 831.
                
                [2] H.-Z. He, D. S.-H. Chan, C.-H. Leung, D.-L. Ma, Nucleic Acids Res.
                
                [3] Rhodes, D. &amp; Lipps, H. J. (2015). G-quadruplexes and their regulatory roles in biology . Retrieved from Nucleic Acids Research, Web site: http://academic.oup.com/nar/article/43/18/8627/2414447
                
                [4] S. Modi, S. M. G, D. Goswami, G. D. Gupta, S. Mayor, Y. Krishnan
                
                [5] Y. Kamiya, H. Asanuma, Acc. Chem. Res. 2014, 47, 1663.
                
                [6] R. E. Kohman, X. Han, Chem. Commun. 2015, 51, 5747.
                
                [7] X. Liang, H. Nishioka, N. Takenaka, H. Asanuma, ChemBioChem 2008, 9, 702.
                
                [8] M. Zhou, X. Liang, T. Mochizuki, H. Asanuma, Angew. Chem., Int. Ed. 2010, 49, 2167.
                
                [9] A. Shastri, L. M. McGregor, Y. Liu, V. Harris, H. Nan, M. Mujica, Y. Vasquez, A. Bhattacharya, Y. Ma, M. Aizenberg, O. Kuksenok, A. C. Balazs, J. Aizenberg, X. He, Nat. Chem. 2015, 7, 447.
                
                [10] M. Chang, C.-S. Yang, D.-M. Huang, ACS Nano 2011, 5, 6156.
                
                [11] S. C.-C. Shiu, Y.-W. Cheung, R. M. Dirkzwager, S. Liang, A. B. Kinghorn, L. A. Fraser, M. S. L. Tang, J. A. Tanner, Adv. Biosyst. 2017, 1, 1600006.
                
                [12] H. Zhang, Y. Ma, Y. Xie, Y. An, Y. Huang, Z. Zhu, C. J. Yang, Sci. Rep. 2015, 5, 10099.
                
                [13] X.-M. Li, J. Song, T. Cheng, P.-Y. Fu, Anal. Bioanal. Chem. 2013, 405, 5993. Nat. Nanotechnol. 2009, 4, 325.
                
                [14] Y. Chen, S.-H. Lee, C. Mao, Angew. Chem., Int. Ed. 2004, 43, 5335.
                
                [15] Y. Dong, Z. Yang, D. Liu, Acc. Chem. Res. 2014, 47, 1853.
                
                [16] J. L. Leroy, M. Guéron, J. L. Mergny, C. Hélène, Nucleic Acids Res. 1994, 22, 1600.
                
                [17] J. M. Majikes, L. C. C. Ferraz, T. H. LaBean, Bioconjugate Chem. 2017, 28, 1821.
                
                [18] T. Hermann, D. J. Patel, Science 2000, 287, 820. 2013, 41, 4345.
                
                [19] Y. Wu, L. Zou, S. Lei, Q. Yu, B. Ye, Biosens. Bioelectron. 2017, 97,
                
                [20] B. Wei, I. Cheng, K. Q. Luo, Y. Mi, Angew. Chem., Int. Ed. 2008, 47, 331.  2017, 1, 1600006.
            </P></DIV><P class="section-title" style="padding-bottom: 0px;">DNA Tweezer Nanomachine</P><DIV class="description-title">Introduction of DNA Tweezer Nanomachine
            <P class="description-text">
                A DNA tweezer nanomachine is a DNA nanostructure that is self-assembled, based on the base-pair formation in double helix. The tweezers in the structure act as the recognition site which can bind the targets toward them. The typical target for DNA tweezers is a strand of DNA or RNA which triggers structure switching from an open to closed state through complementary base pairing.[1] The nanomachine can be activated by target molecules. Therefore, it can be used for target detection in various fields such as pharmacology and medicine.
            </P></DIV><DIV class="description-title">Formation of a DNA tweezer nanomachine
            <P class="description-text">
                Through Watson–Crick base pairing of the strands, a DNA nanostructure is assembled. In the structure, two or more parts of strands act as tweezers which provide recognition sites for target molecules. The targets are bound to the tweezers facilitated by hydrogen bond. Due to the base-pairing formation, the DNA tweezers are closed by the presence of a target molecule which pulls the arms into close proximity generating a strain on the stem region and close the tweezers.[2] The closure of the tweezers is them observed through the signal produced by a reporting system. This enables the DNA tweezer nanomachine to detect the targets.
            </P></DIV><DIV class="description-title">Applications of DNA tweezer nanomachine
            <P class="description-text">
                One of the applications of the DNA tweezer nanomachines is their utilization as nanodrug delivery systems, which can be targeted to different tissues and can perform various tasks ranging from time-controlled drug release to gene therapy and cancer treatment.[3] Another application of the nanomachine is target detection. Only molecules that can pair up with the tweezers can turn the nanomachine from open to close state. Therefore, by observing using a reporting system, the concentration of the target is known.
            </P></DIV><DIV class="description-title">Advantages of DNA tweezer nanomachine
            <P class="description-text">
                Among various DNA machine mechanisms, DNA tweezers hold particular advantages for sensing applications as the mechanism does not involve strand displacement and the structure is formed from only a small number of single-stranded DNA (ssDNA) molecules. [4] Also, other conventional methods for the detection of nucleic acids are not really ideal, as they require skilled operators and complex equipment. Using a DNA tweezer nanomachine for specific nucleic acid detection, we can recognize the target nucleic acid molecules without competitive reaction in a biomolecular reaction.[5]
            </P></DIV><P class="section-title" style="padding-bottom: 0px;">Observation on the assembly of DNA tweezer nanomachine</P><DIV class="description-title">Polyacrylamide gel electrophoresis
            <P class="description-text">
                PAGE provides a versatile, gentle, high resolution method for fractionation and physical-chemical characterization of molecules on the basis of size, conformation, and net charge.[6] Due to the differences in size, length and structure of strands of DNA, they have different ability to run through the gel. For the DNA tweezer nanomachines that are successfully assembled, their sizes will be larger than that of DNA single strands. They will pass through the gel much more slowly. Therefore, they travel a shorter distance. The tweezers contain higher intensity of DNA strands in the similar area, hence they show brighter fringes than the single strand under PAGE.
                <SPAN style="text-decoration: underline;">Figure  2 Set up of Polyacrylamide gel electrophoresis[7]</SPAN></P></DIV><DIV class="description-title">Application of DNA tweezer nanomachine in our project
            <P class="description-text">
                In our project, we will use DNA tweezer nanomachine to detect target miRNA, i.e. a biomarker for SCA3. When the desired miRNA hybridized to the recognition site on the tweezers. The nanomachine will be turned from an open state to a closed state. The closure of the device allows the split strand G-quadruplex in the structure to form its secondary structure. The G-quadruplex acts as an aptamer and bind to hemin. This allows 2,2′-azino-bis(3-ethylbenzothiazoline-6-sulfonic acid) (ABTS) to be oxidized in the presence of hydrogen peroxide and produces a colour change as a signal. We detect the signal in terms of absorbance value at a particular wavelength of light.
            </P><P class="description-text"><SPAN style="text-decoration: underline;">References</SPAN>
                [1] Simon Chi-Chin Shiu, Yee-Wai Cheung, Roderick M. Dirkzwager, Shaolin Liang,
                
                Andrew B. Kinghorn, Lewis A. Fraser, Marco S. L. Tang, and Julian A. Tanner, Aptamer-Mediated Protein Molecular Recognition Driving a DNA Tweezer Nanomachine, Advanced Biosystem, 2017,1,1600006.
                
                [2] K. Nakatsuka, H. Shigeto, A. Kuroda, H. Funabashi, Biosens. Bioelectron. 2015, 74, 222.
                
                Nakatsuka K., Shigeto H., Kuroda A., Funabashi H., A split G-quadruplex-based DNA nano-tweezers structure as a signal-transducing molecule for the homogeneous detection of specific nucleic acids, PubMed, 2015 Dec 15;74:222-6.
                
                [3] Michael Berger, DNA nanomachines that can be turned on and off with a flip of a switch, Nanowerk, Dec 23, 2009
                
                [4] Simon Chi-Chin Shiu, Yee-Wai Cheung, Roderick M. Dirkzwager, Shaolin Liang,
                
                Andrew B. Kinghorn, Lewis A. Fraser, Marco S. L. Tang, and Julian A. Tanner, Aptamer-Mediated Protein Molecular Recognition Driving a DNA Tweezer Nanomachine, Advanced Biosystem, 2017,1,1600006.
                
                [5] K. Nakatsuka, H. Shigeto, A. Kuroda, H. Funabashi, Biosens. Bioelectron. 2015, 74, 222.
                
                Nakatsuka K., Shigeto H., Kuroda A., Funabashi H., A split G-quadruplex-based DNA nano-tweezers structure as a signal-transducing molecule for the homogeneous detection of specific nucleic acids, PubMed, 2015 Dec 15;74:222-6.
                
                [6] A. Chrambach, D. Rodbard, Polyacrylamide Gel Electrophoresis, PubMed, 1971 Apr 30;172(3982):440-51.
                
                [7] Protein Electrophoresis Methods, Bio-Rad
                
                Website:http://www.bio-rad.com/en-hk/applications-technologies/protein-electrophoresis-methods?ID=LUSOW4GRI
            </P></DIV></DIV><DIV class="footer" align="center"><P style="color: white; font-family: Segoe UI; font-size: 15px; font-weight: 300; text-align: center;">
© IGEM HCYxLFC 2018 - All Rights Reserved.</P></DIV></METACHARSET></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>