"
Page
Discussion
View source
History
teams
Log in
Â 
Team:MIT/Projects/Project2
From 2009.igem.org
Home | Projects | Protocols | Parts for Registry | References
Light-inducible protein localization in Yeast
The blue dots represent PhyB bound to a CFP, which diffuses throughout the cell. Meanwhile PIF3 tagged with YFP is constitutively localized to an organelle, (a mitochondria shown here). In the presence of red light, PhyB changes conformation so that it can bind to PIF3. As the PhyB travels throughout the cell it binds to PIF3, causing CFP to localize to the mitochondria. The introduction of far red light will reverse this process allowing PhyB-CFP fusion to diffuse throughout the cell again.
Objective
We set out to create a quick, reversible switch to control the localization to various targets within a yeast cell, such as the nuclear membrane, plasma membrane, mitochondrial membrane, and vacuole.
Motivation
To fully control cellular functioning, it would be useful to have a modular system that can provide both temporal and spatial control over any protein of interest within a cell. In other words, we should be able to control both when a gene is expressed and where the protein is functioning. The existing PhyB-PIF3 system provides temporal control over gene expression by acting as a transcriptional regulator. However, by adapting the PhyB-PIF3 system to control protein targeting, we can have both temporal and spatial control over protein function. Such a system could be used as a rapid on-off switch for localization-dependent processes. For example, because protein targeting is important in control of the cell cycle, the system could be used to control cell cycling and differentiation. Furthermore, reversible control of protein localization would be useful to study signal transduction and as a biophysical tool to study diffusion rates within a cell.
Background on PhyB-PIF3 System
The light switching mechanism that we use exploits the machinery by which plants and algae respond to varying light conditions. Phytochromes are light sensitive proteins that are composed of a polypeptide bound to a small pigmented molecule called a chromophore.  They have two conformations: the inactive Pr form and the active Pfr form. When red light is present, the phytochrome switches from the Pr to the Pfr form. When in the Pfr form, the phytochrome can bind to proteins called phytochrome interacting factors. In nature, the resulting complex can bind to DNA and acts as a transcriptional regulator. When far red light is present, the phytochrome dissociates from the phytochrome interacting factor, making the switching mechanism reversible. Peter H. Quail's lab at University of California Berkeley developed a synthetic biological switch using a phytochrome called PhyB, which is bound to a chromophore called phycocyanobilin (PCB). Red light causes PhyB to bind to a phytochrome interacting factor called PIF3, and far-red light causes them to unbind. Our system harnesses this switching mechanism to control post-translational processing instead of transcriptional regulation.
PhyB is a composed of a polypeptide covalently bound to a chromophore called PCB. Before exposure to light, PCB is in the inactive Pr state. When PCB is exposed to red light, it changes conformation to its active Pfr state, allowing PhyB to bind to PIF3. Far-red light returns PCB to its Pr form and causes PIF3 to dissociate from PhyB.
Mechanism
To localize our protein of interest, either PhyB or PIF3 is anchored to the target membrane, while the other travels through the cell by diffusion. The anchored protein contains a signal sequence, causing it to be constitutively localized to a particular membrane. The traveler, on the other hand, is fused to the protein of interest. When pulsed with red light, the PhyB will switch to the Pfr conformation. PhyB and PIF3 will bind, attaching the traveler, and its fused protein, to the anchor. Under far red light the anchor and the traveler dissociate, and the protein of interest again can diffuse throughout the cell. In our system, PIF3 is tagged with Yellow Fluorescent Protein (YFP) and PhyB is tagged with Cyan Fluorescent Protein (CFP) so that the localization and can be observed in real-time using fluorescence microscropy.
An overview of the mechanism for the localization of proteins using the PhyB-PIF3 system. In this diagram, PhyB is anchored to an organelle, while PIF3 is travels throughout the cell. The traveler is bound to the protein of interest. Under red light, the change in conformation of PhyB causes the traveler to bind to the anchor and thus the protein of interest to localize to the organelle. This process is reversed under far-red light.
Constructs
The diagram below shows the DNA constructs designed to reversibly control protein targeting in yeast. PhyB is fused with CFP, while PIF3 is fused with YFP. Plasmids with these two fusion constructs serve as our base constructs into which localization signal sequences can be added. Whether PIF3 or PhyB is used as the anchor depends on which signal sequence is used to direct the protein to the target. For signal sequences that function on the N-terminal of the protein, PhyB is used as the anchor and PIF3 as the traveler. For C-terminal signal sequences, PIF3 is used as the anchor and PhyB as the traveler.
Schematic of DNA for traveler and anchor constructs. For N-terminal signal sequences, PhyB is used as the anchor, while for C-terminal signal sequences, PIF3 is used as the anchor.
A table of the signaling sequences chosen.  These would serve to anchor one protein to a particular organelle.
Materials and Methods
Construct Construction
The PhyB-CFP and YFP-PIF3 constructs were assembled using two methods: sequential cloning and homologous recombination. PhyB-CFP was built into pRS313 with a Myo2 promoter, while YFP-PIF3 was built into pRS415 with an ADHI promoter. Using both methods, we were able to construct the base constructs, with PhyB-CFP and YFP-PIF3 without signal sequences inserted. More detail on the procedure and the results of the construction of these constructs can be found at http://openwetware.org/wiki/Constructing_the_Plasmids
Sequential Transformation Plasmid Construction
Construction of the PhyB-CFP and Pif3-YFP Plasmids began with the amplification of each of the individual sequences, PhyB, CFP, PIF3 and YFP, from the Berkley Lab Plasmids. These sequences were subsequently digested and ligated into the prS313 backbone, a plasmid that contains a constitutive Myo2 promoter and the His gene.  PIF3 and CFP were inserted using NotI and SacII, while PhyB, YFP used NotI and XmaI.  Once the plasmid was complete it was transformed into E. coli.  The PhyB and Pif3 plasmids transformed successfully, which was confirmed through a check digest and sequencing.  The corresponding fluorescent protein DNA was then incorporated into the plasmid through another round of digestion, ligation and transformation.  The sequencing results for these plasmids were positive for incorporation of both DNA pieces for both of the constructs.  Therefore, each of these plasmids were transformed into yeast and observed for fluorescence through microscopy.
The construct design with Restriction Enzymes for Sequential Transformation.
Alongside work in the base constructs, a variety of C and N terminus localization sequences were chosen to test the PhyB-PIF3 System. Each localization sequence, once amplified, would be cloned into the appropriate backbone using the restriction enzymes. N terminus tags were destined for the Pif3-YPF backbone and C terminus for the PhyB-CFP Backbone
Homologous Recombination
PhyB and CFP were PCR amplified using primers such that there was a region of homology of 40 base pairs between the PhyB product and the CFP product as well as 40 base pairs of homology with the pRS313 backbone on both the PhyB and CFP products. The pRS313 backbone was linearized using a double restriction enzyme digest. The high efficiency yeast transformation protocol was carried out such that the linearized backbone as well as the PhyB and CFP PCR products were added to the transformation mix. When crossing over between the regions of homology occured, the result is a circular plasmid with both PhyB and CFP, as shown the the diagram below. The yeast was then grown on selective media to select for strains in which the cross-over events that result in a circular plasmid occur. A similar procedure was carried out for PIF3 and YFP. PIF3 and YFP were PCR amplified so that they had a 40 base pair long region of homology with each other and with the pRS415 backbone. pRS415 was linearized and transformed into yeast with the PIF3 and YFP products. The yeast with the circular plasmid with YFP and PIF3 were then selective for using selective media.
A depiction of the plasmids into which the PhyB-CFP and PIF3-YFP were inserted.
The design of the constructs, including restriction sites, constructed by homologous recombination.
LED Arrays for Red and Far Red Light
For growing cells in red light and far red light, we designed these two LED arrays as an inexpensive alternative for adjusting light conditions.
Above are the LED arrays that we used create the red and far red light conditions under which the cells are incubated.
Plans
Because the plasmids with PhyB-CFP and YFP-PIF3 showed no fluorescence using microscopy, the next step would be to troubleshoot the plasmid construction. The lack of fluorescence could be caused by a lack of a Kozak consensus sequence, or perhaps due to point mutations that occurred during construction. These possibilities should be explored and the fusion constructs rebuilt. Then the signaling sequences should be cloned in, using either traditional cloning or homologous recombination, to create our anchor constructs.
Once all the constructs have been designed, the next step is to integrate Project 2 with Project 1. The plasmids are to be transformed into a yeast strain that has been engineered to produce PCB. What should result is a switch to control protein localization in yeast in which all necessary components are produced endogenously within the strain.
Retrieved from " http://2009.igem.org/Team:MIT/Projects/Project2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
