"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/Attributions
From 2012.igem.org
UBC iGEM »
Home
Team
Attributions
Project »
Microbial Consortia
Bio-desulfurization
Safety
Notebook
Protocols
Results »
Data
Parts
Key Results
Medal Criteria
Models »
Consortia
Pathway
Human Practice »
IP & iGEM
Patent Guide
Industrial Insight
British Columbia - 2012.igem.org
Special Thanks
Wet Laboratory Acknowledgements The Steven Hallam lab for hosting our iGEM team, providing us with laboratory space and reagents through the 2012 competition Dr. Bob Hancock of UBC for sending us a strain of Pseudomonas aeruginosa Dr. Jay Keasling of UC Berkeley for providing us the Dsz operon in E.coli Dr. Jonathan Van Hamme of Thompson Rivers University for sending us the Dsz operon VWR , Dr. John Church, the Annual Giving UBC Department of Microbiology & Immunology and BIOL Alumni Appeal Fund, Harry Lim Incorporated and the UBC Faculty of Applied Science for donations of reagents and resourcesHuman Practices Acknowledgements Ian Bell from UBC UILO and Emma Macfarlane from Gowlings for discussions regarding patents Peter Wynne from Chevron Canada , Alberta Innovates Technology Futures AITF representative, Karen Budwill, Oil Sands Leadership Initiative OSLI representatives, John Vidmar and Nicolas Choquette-Levy, and Dr. Dusko Posarac of UBC Chemical and Biological Engineering for discussions regarding industrial applications of microbial consortiaComputational Modeling Acknowledgements Dr. Susan Baldwin of UBC Chemical and Biological Engineering, Gurpal Bisra and Shing Hei Zhan (UBC iGEM alumni) for advice on the consortia model Niels Hanson from the Steven Hallam laboratory for advice on the pathway modelGeneral Acknowledgements Alberta Genetically Engineered Machines (aGEM) judges and Dr. Phil Hieter of UBC for their constructive advice on our project and presentation
Collaboration with University of Calgary iGEM
We thank Dr. Lisa Gieg and University of Calgary iGEM for collaborating and splitting the DSZ pathway for characterization: We wanted to collaborate with Calgary because of the friendship and hospitality they showed last year! Considering that we were both working with the same operon, we decided that Calgary would take DszA and DszB while we would take DszC and DszD. We also set up meetings to see what else we could do in collaboration. We initially wanted to see if we could do anything regarding modeling together, given that we were working with similar systems. However, Calgary was more interested in bioremediation, rather than fuel processing. We then struck on the idea of collaborating on kill switches. Calgary wanted to characterize a few new induction systems, including a rhamnose promoter and two new riboswitches for their kill switches. They have yet to have success with their rhamnose promoter. They sent us the sequence before they attempted anything with it, and we used that sequence to design primers to get the rhamnose promoter out of the genome. Noticing the paucity of standardization of kill switches on the registry, we took it upon ourselves to develop a plan to test the efficacy of any new toxic protein that might be wanted to be tested for kill switch capabilities. There was a fair amount of talk about how best to do a death assay. We suggested using Dendra2, a fluorescent protein that could be 'broken' at a specific non-cytotoxic wavelength of light, allowing us to quantify metabolic activity after supposed death after toxic protein induction by looking for further expression of the fluorescent protein. They argued that a dormant cell may not express this protein, and we wouldn't really be looking at death. We then decided that we ought to instead focus on plating assays, for this better tests the viability of cells. We both tried testing existing kill switches on the registry, but were unable to get any of them to work. Calgary postulated that this was due to the arabinose promoter, which they could never get it to work. In the mean time, we both sequenced some of the toxic genes used as kill switches in the registry, and found that everything we sequenced was indeed what it claimed to be. Calgary also sent us the gene for HPAC, an alternative NADH-FMN oxidoreductase, to be used in place of DszD. It is a native E coli enzyme, and apparently works better, though over-expression causes hydrogen peroxide formation. We had plans to see if we could get it expressed in a pET vector to look for peroxide generation, but we have not done this yet. Lastly, they sent us the yddg- strain from David's old lab at U of A, which could be used in complementation studies. Kerner et al. (2012) used the yddg part because it increases the rate of export of tyrosine by 3 times, and increases overall tyrosine production. We have created a constitutively expressed yddg part that we would ultimately like use to check for complementation (yddg also confers resistance to inhibiting concentrations of aromatic amino acids), but we have not made competent cells from this strain.
Collaboration with INSA-Lyon
To help team INSA-Lyon with their human practice project investigating the how intellectual property affects iGEM, we have provided them with the data from our survey results .  In return, INSA-Lyon assisted us by giving us feedback on drafts of our iGEM-specific intellectual property guide .
Our 2012 Sponsors
Retrieved from " http://2012.igem.org/Team:British_Columbia/Attributions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
