<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_BiobrickConstruction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/BiobrickConstruction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/BiobrickConstruction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/BiobrickConstruction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/BiobrickConstruction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/BiobrickConstruction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/BiobrickConstruction</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">UBC iGEM<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia">Home</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI></UL><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/ProjectConsortia">Microbial Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Desulfurization">Bio-desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A class="sf-with-ul" href="#">Results<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Data">Data</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Results">Key Results</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Medal">Medal Criteria</A></LI><LI><A class="sf-with-ul" href="#">Models<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Consortia">Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Pathway">Pathway</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP">IP &amp; iGEM</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP_FAQ">Patent Guide</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/Industry">Industrial Insight</A></LI></DIV><TITLE> British Columbia - 2012.igem.org </TITLE><DIV id="note"><P><B>June 15</B></P><P>1)Miniprepped the resistance cassette cultures. Previously, to get the part for the template for the PCR, we just picked a colony, but we thought that it would be beneficial to have a plasmid stock in case we needed to repeat the PCR. 
</P><P>2)PCR with new (biobrick) primers was attempted. We received primers for the Dsz genes and also for the amino acid genes. We did not, however, receive the DszA forward primer, and decided to hold off on working with the Dsz genes until later. We performed a cPCR for all the genes we had, and ran the results on the gel.
</P><P>3)Ran an important gel. Below are the results from yesterday's and today's PCR. Cameron made the gel and taped the edges of the gel box to prevent the gel from escaping. The gel itself was made to 1% agarose in 0.5% TBE, and run on a 50 well gel box. 3 uL of loading buffer were used for 8 uL (10 uL for yesterday's products) sample to increase contrast, and various ladders were tested. The negative control PCR tube for today's PCR popped open during the reaction, and evaporated.
</P><OL><LI> Broad Range Ladder
</LI><LI> TrpA (biobricks)+DMSO
</LI><LI> TrpA 
</LI><LI> TrpB +DMSO
</LI><LI> TrpB 
</LI><LI> ArgC +DMSO
</LI><LI> ArgC 
</LI><LI> ArgE +DMSO
</LI><LI> ArgE
</LI><LI> MetA +DMSO
</LI><LI> MetA
</LI><LI> TyrA +DMSO
</LI><LI> TyrA 
</LI><LI> empty
</LI><LI> 1kb ladder
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> empty
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> empty
</LI><LI> 1kb + Ladder
</LI><LI> TrpA tet 
</LI><LI> TrpA tet +DMSO
</LI><LI> TrpA tet +MgCl2
</LI><LI> TrpA tet +DMSO+MgCl2
</LI><LI> ArgE tet 
</LI><LI> ArgE tet +DMSO
</LI><LI> ArgE tet +MgCl2
</LI><LI> ArgE tet +DMSO+MgCl2
</LI><LI> TyrA tet 
</LI><LI> TyrA tet +DMSO
</LI><LI> TyrA tet +MgCl2
</LI><LI> TyrA tet +DMSO+MgCl2
</LI></OL><UL><LI> Figure 1. Myriad Gel 
</LI></UL><P>As you can see, we have successful bands for tyrA (from genome, biobrick primers) with DMSO, MetA with kanamycin resistance (for knockouts), trpA with tetracycline resistance, ArgE with tetracycline resistance, and tyrA with tetracyline resistance. The 1 kb+ ladder appears the most useful in this experiment. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P>The Amp plates made the day before worked.  However, the Kan plates (negative control, EPI300 pIJ790, and DH5a pIJ790) showed unexpected results.  The negative control, which only had untransformed K12 cells plated, had more colonies growing than those from the plates with resistance-transformed EPI300 pIJ790 and DH5a pIJ790.  Even then, there were only a few small colonies on the EPI300 pIJ790 plate and none at all on the DH5a pIJ790 plate.  It looks like the Kan plates may not be working, and it is possible that the EPI300 and DH5a cells with the pIJ790 plasmid are not competent.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><P><B>June 16</B></P><P>The time I came in Joe was making the PCR gel. 
</P><P>The cultures with 3I plasmid that left for growth in the 37 degree room actually did turn a bit cloudy (6:00 p.m.).
</P><P>Their ODs were then measured with a blank LB for reference, and the detailed info will be updated. 
</P><P>For the remaining cultures, 100µl of each culture was tested on Chlor plates with an control of K12 (20µl)  to see if they are actually the cell cultures that of our interest, and also testing the chlor plates that are newly made, and the remaining cultures were stored in the 4 degree room for potential future usage. 
</P><P>The plates were left in the 37 degree room, and will be collected by Joe and stored in the 4 degree room. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>June 17</B></P><P>As shown above in the gel image, there is still some PCR that needs to be done. It appears apparent that increased concentrations of DMSO and magnesium chloride help the reaction, so the appropriate amount of each was added to the mastermix. The reactions were as follows: TrpB with Kan cassette, TrpA, TrpB, ArgC, and MetA. 
</P><P>Since we were planning to put these into biobricks, it was imperative that we made sure that there were no restriction sites. To do this, we used the NCBI data for the gene sequence and ran it through Nebcutter, searching for all illegal sites. No illegal sites were found in TrpB, TyrA, TrpA, ArgC, or MetA. Lucky.
</P><P>As shown above, a reaction for the tyrA gene worked. Using this PCR product and the rfp plasmid, we digested with EcoRI and PstI and ligated with T4 DNA ligase. The resultant product was used to transform Epi300 cells. Tomorrow, we will check for white colonies among the red ones, which would suggest that the ligation was successful. A colony PCR will be done to confirm.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>June 18</B></P><P>1) A gel from yesterday's PCR was shown to have some products that worked, while others did not. The MetA, TrpA, TrpB, and ArgE biobrick PCR's appear to have product.
</P><P>2) The biobrick PCRs that did work were digested with EcoRI and PstI, as was the pSB1C3 linearized plasmid backbone. 
It was unknown whether or not the plasmid had any methylation, so DpnI was used only in the PCR product digestion. The procedure will be uploaded to the wiki in the near future. A gel was made showing the ligation products.
</P><P>3) The biobrick ligation products were used to transform K12 cells and were plated appropriately.
</P><P>It should be noted that several of the transformations sparked during the electroporation procedure. Some success has been reported with sparked cultures, so they were plated anyways. 
</P><P>As an update, here is a table showing what has been done so far. 
</P><TABLE class="wikitable"><TBODY><TR><TD> Gene</TD><TD>Type</TD><TD>PCR</TD><TD>Ligation</TD><TD>Transformation
</TD></TR><TR><TD>TrpA+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>TrpB+Kan resistance</TD><TD>Recombineering</TD><TD>Unsuccessful</TD><TD>NA</TD><TD>No
</TD></TR><TR><TD>ArgE+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>ArgC+Kan resistance</TD><TD>Recombineering</TD><TD>Unsuccessful</TD><TD>NA</TD><TD>No
</TD></TR><TR><TD>TyrA+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>MetA+Amp resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>MetA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>ArgE</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TrpA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TrpB</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TyrA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>ArgC</TD><TD>Biobrick</TD><TD>Unsuccessful</TD><TD>No</TD><TD>No
</TD></TR></TBODY></TABLE><P>It should be noted that when a ligation has been done, it does not mean that it was successful. We will not know until we see growth on the plates and a cPCR of the colonies has been done. Even then, it would be useful to have it sequenced. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P>Worked with Jacob on steps 3 and 4 listed above. I made the gel, practiced loading samples into wells (with Ruichen), and learned that the machine used to see the results is finnicky. Unfortunately, the 1 kb reference ladder did not work (unexpectedly and for unknown reasons).
</P><P>(Step 3) Transformation and plating of the biobrick PCR products were the last things we did today. Of the 5, only TrpB sparked during electroporation. 1 uL of PCR product was used to transform cells.
</P><P>- <A href="/wiki/index.php?title=User:Grace.yi&amp;action=edit&amp;redlink=1" class="new" title="User:Grace.yi (page does not exist)">Grace.yi</A></P><P><B>June 19</B></P><P>There were no colonies on the plates transformed with any putative biobricks, so the procedure was attempted again.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>June 20</B></P><P>Plated the cultures with 3I plasmid on Kan for control and Chlor for testing purpose. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>Still no biobrick colonies. Tried again with an altered ligation protocol. Also ordered a variety of kill switches from the registry, along with Joe's Panamanian Pseudomonas Rhamnosyltransferase. Repeated PCR on things that had previously failed, and got the final amino acid biobrick. It should be noted that I ran reactions with either PFU or phusion, and only the phusion worked. This may have been because of the low volume (0.5 uL) used.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>June 21</B></P><P>The culture has been proven to be transformed successfully with the 3I plasmid! They grew colonies on the Chlor plate and not the Kan plate. 
</P><P>Learned how to miniprep
</P><P>Checked out some cool posters for the microbiology Conference, and found that there is a assay of interest. The assay (2,6 DCPIP) helps organisms to grow on a diesel contaminated soil. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>All amino acids have been PCR'd out of the genome, and several colonies appeared on the ArgE ligation transformation plate. One colony was selected and grown in culture. The rest of the biobrick ligations were redone and allowed to ligate overnight. 
</P><P>The digests of the products were done with EcoRI and PstI in buffer 3, as opposed to the EcoRI-HF and PstI in buffer 2 we did previously. This was done because PstI has 100% activity in buffer 3, as opposed to 75% activity in buffer 2. Also, we learned that EcoRI-HF has 0% activity in buffer 3. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>June 22</B></P><P>Transformed with the overnight (Room temperature) ligation, using 1 uL ligation mix and 1800V. Joe then plated the cells after recovery. 
</P><P>Found a MetA biobrick colony on the appropriate plate from 2 days ago. Decided to grow it up in culture and run a PCR to confirm that it was successful. 
</P><P>Wanted to design mutagenesis primers, but couldn't remember what software to use. For future reference:
</P><P>NCBI for FASTA
</P><P>Nebcutter to find sites
</P><P>Virtual Ribosome for associated peptide
</P><P>Wikipedia for codon table
</P><P>PrimerX for basic design.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P>The IGTS8 and IGTS9 cell cultures in both terrific broth and LB grew dense and cloudy at 30C.  Glycerol stocks will be made out of these.
</P><P>We have been having issues with having no colonies grow on plates with cells transformed with PCR products ligated into vectors.  One possible cause is the transformation itself.  All of Joe's transformation resulted in a spark.  He had added 5 uL ligated to competent cells.  The high salt concentration may have heavily influenced the sparking.  So, I redid some of the transformations in a smaller volume of ligate.  I transformed Joe's ligated 3:3 dszB, 3:3 dszC, and 3:1 dszB plasmids into EPI300 cells.  Added 0.5 uL ligate to 40 uL competent cells and electroporated them.  The 3:1 dszB cells sparked during electroporation.  The cells were left to recover undisturbed at 37C for an hour before plating 50 uL transformants onto Chlor plates.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><P><B>June 23</B></P><P>Made competent <I>Pseudomonas putida</I> cells using the <A href="/Competent_Cell_Production" title="Competent Cell Production" class="mw-redirect">Competent Cell Production</A> protocol.
</P><P>We had trouble getting the cells transformed with the ligated amino acid pathway genes to grow when plated.  We're unsure if the problem lies in the digest, the ligation, or the transformation.  Thus, we decided to redo all these steps.
</P><P>Re-digested the &quot;M,&quot; &quot;TA,&quot; &quot;TB,&quot; &quot;TyrA,&quot; and &quot;ArgC&quot; PCR products provided by Jacob.  Made a master mix of 5 uL NEB buffer 2, 0.5 uL BSA, 0.5 uL EcoRI HF, 0.5 uL PstI, and 18.5 uL dH2O.  Incubated each of the mixtures of 4 uL master mix plus 4 uL PCR product in a Thermocycler for ____.
</P><P>Re-ligated digest of &quot;M,&quot; &quot;TA,&quot; &quot;TB,&quot; &quot;TyrA,&quot; and &quot;ArgC.&quot;  We suspect that it may be the ligase we used that was at fault for the undesired results seen in previous experiments.  So, we decided to add in more ligase and add in ligase from 2 different sources/tubes.  We added 1 uL pSBIC3 EP, 4.0 uL dH2O, 1 uL T4 ligase buffer, and 0.5 uL from each tube of T4 ligase were added to 3 uL of the respective digest products.  A control was also set up so that there was only plasmid DNA added.  The ligate mixtures were placed in the Thermocycler to incubate at 16C for 30 minutes and inactive at 65C for 20 minutes.
</P><P>John miniprepped MetA and ArgE, which are being prepared as potential biobricks.
</P><P>A new set of PCR reactions was set up for MetA (miniprepped potential biobrick), TyrA (colony PCR), ArgE (miniprepped potential biobrick), and yddG (colony PCR). Made a ligation master mix by combining 60 uL Phusion 5X buffer, 147 uL dH2O, 15 uL DMSO, 12 uL MgCl2, and 18 uL 10 mM dNTP.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><P><B>June 25</B></P><P>Designed primers for SDM of dszB gene
</P><P>PCRed yddG and Tyr A with Marianne 
</P><P>Had a Skype meeting with Calgary iGEM team
</P><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>June 26</B></P><P>Designed primers for SDM of dszC gene
</P><P>Joined Jacob and Joe for transformation of Heat(chemical) competent cells: 
</P><TABLE class="wikitable"><TBODY><TR><TD> Plasmid transformed</TD><TD>Volume of Ligation Mixture used (µL)
</TD></TR><TR><TD>dszB, dszC (3 to 3 ratio, Old and New)</TD><TD>3
</TD></TR><TR><TD>MetA, TrpA, TrpB, TryA, ArgC, positive and negative control</TD><TD>7
</TD></TR></TBODY></TABLE><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>June 27</B></P><P>Run the gel of Tyr A and yddg genes at 95 Volts for 1 hr and 30 min, with 1Kb ladder. 
</P><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>June 30</B>
Yesterday, we went to Dr. Ramey, who had taught students who made a successful knockout using the lambda red system. They used a different recombineering plasmid than we did, and we received this plasmid. It was, however, growing in a strain that we were unfamiliar with, and we decided to isolate the plasmid and transform our Epi300 cells with it. 
It is currently growing in a culture at 30°.
</P><P>We received only the TrpA gene from the registry, and successfully grew it on a plate. We are now growing a culture to isolate the plasmid and if Joe's knockout is indeed a knockout, we can characterize the existing biobrick part by complementation. 
</P><P>We ordered the Gibson assembly kit, and it ran upwards of 700 dollars for 50 reactions. 
</P><P>Joe successfully PCR'd out the DszD gene from Rhodococcus genomic DNA that he isolated.
</P><P>Our further attempts at PCR for the yddg and tyrA genes was unsuccessful.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 1</B></P><P>There are many (hundreds) of colonies on Joe's recombineering plates, and the concentration of colonies goes up as the concentration of antibiotic goes down. It appears that the addition of arabinose to the culture did nothing to increase recombineering efficiency. There are several large, well defined colonies on the full antibiotic plate. It appears that 2 days are required for sufficient growth at 30°.
</P><P>With this success in mind, I PCR purified a couple of other knockouts (ArgE Tet and TrpB Kan) and transformed them, reusing the electroshock cuvettes we borrowed from Rafael. A pack of 50 was ordered so we could do more than 4 transformations per day. 
</P><P>We only had one chlor plate left, and 4 potential biobricks which were to be put on the chloramphenicol resistant pSB1C3 plasmid. Since there was only one plate (and one electroshock cuvette), only the MetA biobrick ligation product was transformed. 
</P><P>The pKD46 recombineering vector that we received from Dr. Ramey grew well overnight at 30°C, and was miniprepped, and then transformed into the Epi300 strain. The recovery period was done at 30°C as well. If the transformation is successful, Ting will make electrocompetent cells out of the Epi300+pKD46. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 7</B></P><P>Colonies with potential biobricks of trpA, tyrA, argE, metA, and dszD were successfully grown. This time, chemically competent cells were used, and they were plated on 1/2 concentration chlor plates. We also transformed and grew out the previously characterized minC biobrick.
</P><P>PCR of the yddg gene failed yet again. It is possible that the primers are not correct. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 9</B></P><P>The potential biobricks were successfully miniprepped, and a PCR was performed to isolate what was actually between the VF2 and the VR primers in the plasmid. The product will then be run on a gel, and if it appears the correct length, will be sent for sequencing.
</P><P>Our Pseudomonas competent cells appear to work (conferring antibiotic resistance), but they are not expressing rfp. 
</P><P>We made spec plates, but a lawn grew with our transformed cells. Interestingly, on top of this lawn grew several well defined colonies. We attempted to grow these colonies (and the proper negative controls) in varying amounts of spectinomycin. It is possible the plates had insufficient spec, or that the strain we are working with is naturally resistant (there was a lawn, but no large colonies, on the negative control).
</P><P>Several more biobricks from the registry were transformed.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 15</B></P><P>Sent the potential biobricks for sequencing last Friday, ordered primers for mutagenesis on DszC, ordered primers for attachment of fluorescent genes to amino acid genes, received all but yddg knockouts from Yale, found that the Arg knockouts work as described. 
</P><P>An attempt at doing an experiment to calibrate the fluorescent proteins for the plate reader (looking for quenching, linear relationship between OD and fluorescence, etc) was done on Sunday, but as it turns out, we had the wrong promoter for a fluorescent protein, and the wrong antibiotic resistance for one of the other proteins. 
</P><P>Recently, there was a meeting with the kill switch team from Calgary. Primers to create a biobrick with an intein in it for temperature specific activity were ordered. 
</P><P>Competent cells were made from each of the knockouts, and they worked well. The yddg strain was ordered through collaboration with Calgary as well as the keio collection proper in Japan. 
</P><P>Our Gibson assembly kit arrived.
</P><P>The sequencing results from our amino acid biobricks came back, and it looks like only MetA was successful. It should be noted that the melting temperatures of the Vf2 and the VR primers are 60°, for cPCR validation of biobricks.
</P><P>Several new combinations of fluorescent genes, antibiotic resistances, and Keio strains were made. We used the wrong ones last weekend, but now, we are going to test TyrA- +GFP, TrpA- +RFP, MetA- +YFP, and also wildtype with the fluorescent genes. Recently, we switched backbones for the 13M RFP to Psb1C3, for we do not know the history of our current psb1C3 with rfp (we suspect it might be on the lac promoter, as per the registry guidelines), and we also put the GFP on the psb1C3 instead of a kanamycin resistant plasmid, because we were transforming into kanamycin resistant auxotrophs. 
</P><P>We are trying to grow the ArgC- and the ArgE- together in M9 to see if anything results, with negative controls for both. We also trying to grow the wildtype with the TrpB auxotroph with rfp to look for qualitative red fluorescence. 
</P><P>We miniprepped 4/6 of the biobricks that we ordered from the registry. We intend on sequencing them in the near future. We, lacking LB amp plates at the time, plated one on an M9 plate, but we now think that the strain the registry sent us is auxotrophic for something. We replated on LB amp after making new plates. The last one that we ordered did not arrive. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 17</B></P><P>Today, we made dh5a and k12 competent cells. We also ran a cPCR on our biobrick colonies, and the results showed that they did not have an insert of the proper length.
</P><P>As said yesterday, we grew the two arg auxotrophs together, and they grew substantially more (~0.15 OD600) than the negative control. The wt grew to an OD of about 1. We are growing them for another night to see if the OD of the mix increases at all. 
</P><P>Since sequencing confirmed a MetA biobrick, we transformed K12 cells with the remaining stock of our plasmid, so we could amplify it. 
</P><P>All relevant transformed strains from yesterday grew well on the plates.
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 19</B>
We miniprepped our metA biobrick (so we have enough to send to the registry and use for experiments), as well as new potential candidates for other biobricks (tyrA and ArgC).
</P><P>We have, in the past few days, designed a plasmid that will aid in the testing of biobricks.  
</P><P>Concerning the kill switch plasmid parts, we want to take the psb1C3 plasmid, add a constitutively expressed LacI gene, and a lac promoter and rbs immediately before the cut sites. This way, a toxic protein can be well controlled. In fact, while we are designing this for toxic proteins, it could have uses in general protein production. Previous attempts for using the lac promoter on a high copy plasmid have failed leaky expression) due to the lack of the LacI protein produced in the genome. This new plasmid design remedies this, and will be accomplished by Gibson assembly. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>July 23</B></P><P>Analyzed the data, and found out that the growth rate and the amino acid (Tyr) concentration has a correlation as follows: 
</P><UL><LI>Figure 4. cell growth-rate at different Tyr concentrations
</LI></UL><UL><LI>Figure 5. Log scale of cell growth-rate at different Tyr concentrations 
</LI></UL><P>It was found that the cells depleted the nutrient quickly at low Tyr concentration, and for future experiments it is suggested to use lower initial cell concentrations to extend the time of its exponential growth phase. Also, the wild type has almost the exact growth rate as the autotroph when the the Try concentration is exceptionally high (0.01 M). 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>July 26</B></P><P>The growth rate and the amino acids (Met and Trp) concentration has a correlation as follows: 
</P><UL><LI>Figure 6. cell growth-rate at different Met concentrations
</LI></UL><UL><LI>Figure 7. Log scale of cell growth-rate at different Met concentrations 
</LI></UL><UL><LI>Figure 8. cell growth-rate at different Trp concentrations
</LI></UL><UL><LI>Figure 9. Log scale of cell growth-rate at different Trp concentrations 
</LI></UL><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>August 3rd</B></P><P>Started to work on the data obtained on July 28th plate readings. 
</P><P>Eventually arrived at the following graphs of growth rates observations, when n=3
</P><P>From these graphs, the maximum growth rate, and Ks values that are used in Monod Kinetics can be determined. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P><B>August 12</B></P><P>A large set of primers arrived, which gave us a lot to do.
</P><P>With a set of the primers that arrived, we were able to PCR amplify certain parts that will be able to be Gibson assembled to create new biobricks. On the plasmid that has RFP (a modified BBa_K093012, placed on the Psb1C3), we wanted to put arabinose inducible MetA or arabinose inducible TyrA. However, one of the primers necessary to amplify the plasmid itself, which would be necessary to incorporate the proper homologous sequence for Gibson assembly, did not arrive. Additionally, the PCR to amplify the MetA gene did not work the first time, but has been repeated. The arabinose promoter was successfully PCR'd out of the E. coli genome for both cases. 
For the plasmid with YFP, the BBa_I13973, we wanted to put rhamnose inducible TrpA or TyrA after the YFP gene. We received all the primers necessary for this, but the PCR for the plasmid backbone gave several bands and was repeated at a higher annealing temperature. The results are not yet known. The rhamnose promoter was successfully PCR'd out of the genome, as was the TrpA gene. However, we lacked one of the promoters for the TyrA gene. Since there were several bands for the YFP plasmid, we did not do a Gibson assembly. Additionally, it is possible that the primers were designed for the pSB1C3 rather than the pSB1A2, but that should have little effect due to the similarities on the ends of the two plasmids. A simple digest and ligation should be able to move the part from the one plasmid to the other.
We also wanted to put a IPTG inducible (BBa_K091111, LacIQ) metA gene after a GFP. The PCR for the MetA gene and the GFP plasmid were successful (although the GFP plasmid appeared as a fairly weak band on the gel), but the PCR for the LacIQ promoter failed. The initial DNA for this PCR was added directly from the parts distribution kit, and as such the composition and concentration of exactly what was added was unknown. The PCR was repeated as a cPCR on the genome, so as to at least get a usable part with the right overhang. 
</P><P>The final primers for the gibson assembly of the kill switch primer also arrived. There are four parts to this plasmid: the plasmid backbone, the lacI promoter, the constitutively expressed lacI gene, and the constitutively expressed RFP for easy selection. The plasmid backbone, the promoter, and the constitutively expressed RFP PCR products has been ready for several weeks, but we lacked the primers for the constituvely expressed lacI gene until now. To generate the DNA for the constitutively expressed lacI gene, we took BBa_K081005 (const. promoter+rbs) cut with only SpeI and 	BBa_I732100 (lacI) cut with only XbaI, and ligated the two together, and then did a PCR on the ligation. It worked beautifully. 
</P><P>The primers also arrived to isolate the VMA Sce-I intein, and Alina provided yeast genomic DNA. The PCR was done successfully. 
</P><P>A previous potential part, ArgE, was restriction confirmed, and will be sent for sequencing at the soonest convenience.
</P><P>The PCR for yddg was finally a success. The part has been restriction digested, along with the pSB1C3 plasmid, and a ligation will follow. 
</P><P>We finally received the DszA forward primer, and did a restriction digest on it as well. It will be ligated as well in the near future. The DszA gene  has several Pst1 sites, which limits some of the reactions we can do with it, but we can still place things before it by cutting with EcoRI and/or XbaI. 
</P><P>Joe has found inconsistent results with his fluorescent proteins on the various promoters, and we have plans to standardize the promoter and rbs for the fluorescent proteins that will be used. Additionally, there is some overlap between YFP and GFP that is proving difficult to account for, so we are going to try to switch one to either BFP or CFP. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>August 13</B></P><P>There were colonies on the MinC Intein plate, the ArgE BB plate, and the GFP construct LB plate, but nothing on the GFP construct M9 plate. This may have been because there was no lactose or IPTG to induce the metA gene. Interestingly, there was a green fluorescent patch, but it was rather weak and had no distinct colonies. We plan to pick several colonies from the LB plate and try growing them in M9 culture with and without IPTG overnight. We also plan to do a cPCR on the MinC plate using the intein primers. There were no colonies on the DszA or yddg biobrick plate. We have had no luck with electroporating cells with ligation mixtures, although we have had some success with heat shock competent cells. We think there may be something in the ligation mixture that is interfering with the electroporation, and could probably be fixed by purifying the ligation mix. 
</P><P>We also did a PCR for the TyrA BB product, the previous failed YFP plasmid, and several other previously failed PCRs. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>August 15</B></P><P>After two days of shaking at 37° in 1 mM IPTG, the supposed IPTG inducible metA constructs in the metA auxotrophs were unable to grow. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>August 16</B></P><P>The supposed metA constructs still appears unable to complement a metA knockout. It has been suggested that the concentration of metA may have been too high, and this would have led to a severe detriment to the cell, and it has also been suggested that the IPTG, being a rather old stock, was no longer good. Another step that we are going to take is to see if the expression of the construct can be seen on an sds page under induction.
</P><P>The YFP PCR for gibson assembly has yet to give us a solid band, so we are trying to move it into the psb1c3 plasmid in case there are some plasmid specific sequences that were giving us poor results. The ligation and transformation has been done, and we are waiting for colonies. The PCR products necessary for the gibson assembly of the GFP-IPTG-TrpA gene were also all available, and that gibson assembly has been done. However, due to the failure of several gels, there was very little product left, and PCR purification would likely result in unusuable quantities, so we tried a Gibson assembly with the straight PCR product, and we will see if it works. 
</P><P>All of the parts necessary for the assembly of the kill switch plasmid have been successfully PCR'd.
</P><P>Another PCR was done to generate some of the other parts required for assembly of other parts. 
</P><P>A cPCR of the colonies with the MinC protein with the Sce VME intein showed that the gene for the intein was present in at least 2 colonies.
</P><P>There are two distinct bands on the RFP PCR, one at the correct length, the other much lower. 
</P><P>The cPCR on the DszA colonies gave no result. This may have been due to an unoptimized Taq protocol,and might be worth repeating. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P><B>August 17</B></P><P>The plans for the metA BB construct induction were further discussed. Overnight colonies will be made of the construct and a negative control, then they will be diluted in 1 in 100 LB, allowed to grow to 0.4 OD, induced with 0.1 mM new IPTG, and harvested at a certain OD following this, then run on an SDS PAGE. This will test to see if the IPTG really would stimulate the production of the metA
The cultures for the MinteinC (As it shall henceforth be known) grew well, but were not miniprepped. Once they are miniprepped, then site directed mutagenesis must be done. The cultures of the potential yddg biobricks grew well, and will be confirmed either by PCR of the plasmid using VR and VF2, or by restriction digest. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><DIV style="text-align: center;"><H2><SPAN class="mw-headline" id="Back_to_top"><A href="https://2012.igem.org/Team:British_Columbia/BiobrickConstruction" class="external text" rel="nofollow">Back to top</A></SPAN></H2></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:British_Columbia/BiobrickConstruction">http://2012.igem.org/Team:British_Columbia/BiobrickConstruction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/BiobrickConstruction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/BiobrickConstruction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/BiobrickConstruction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/BiobrickConstruction&amp;oldid=267176" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>