<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_ConsortiaNotebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/ConsortiaNotebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/ConsortiaNotebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/ConsortiaNotebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/ConsortiaNotebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/ConsortiaNotebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/ConsortiaNotebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">UBC iGEM<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia">Home</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI></UL><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/ProjectConsortia">Microbial Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Desulfurization">Bio-desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A class="sf-with-ul" href="#">Results<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Data">Data</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Results">Key Results</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Medal">Medal Criteria</A></LI><LI><A class="sf-with-ul" href="#">Models<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Consortia">Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Pathway">Pathway</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP">IP &amp; iGEM</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP_FAQ">Patent Guide</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/Industry">Industrial Insight</A></LI></DIV><TITLE> British Columbia - 2012.igem.org </TITLE><DIV id="note"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#June_8"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">June 8</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#June_12"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">June 12</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#June_14"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">June 14</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#June_15"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">June 15</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#June_16"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">June 16</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#June_17"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">June 17</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#June_18"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">June 18</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#June_19"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">June 19</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#June_20"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">June 20</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#June_21"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">June 21</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#June_22"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">June 22</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#June_23"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">June 23</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#June_25"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">June 25</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#June_26"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">June 26</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#June_27"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">June 27</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#June_30"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">June 30</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#July_1"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">July 1</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#July_7"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">July 7</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#July_9"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">July 9</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#July_10"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">July 10</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#July_15"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">July 15</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#July_17"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">July 17</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#July_19"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">July 19</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#July_20"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">July 20</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#July_22"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">July 22</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#July_23"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">July 23</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#July_26"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">July 26</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#August_2"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">August 2</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#August_3rd"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">August 3rd</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#August_9"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">August 9</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#August_12"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">August 12</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#August_13"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">August 13</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#August_15"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">August 15</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#August_16"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">August 16</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#August_17"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">August 17</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#September_15"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">September 15</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="June_8">June 8</SPAN></H2><P>Started kill switch assays on colicin E2 and BamHI. Colicin E2 is on the SOS promoter, which is best switched on by mitomycin. However, we currently do not have access to mitomycin or any of its closely related derivatives, so, judging by the previous;y mentioned paper, it might be able to be induced by b-lactam antibiotics in low concentrations. 
</P><P>Procedure outline:
Dilute 1:100 22A, 3C and wt strains in 25 mL LB. Measure OD every hour. At start of exponential phase, add arabinose to 0.01% or amp to 4 ug/mL to strain+control. Plate 25 uL 2 hours later, mid exponential phase. Details to follow. 
</P><P>The plate reader is still broken, with no estimated repair date. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="June_12">June 12</SPAN></H2><P>Checked plates from Jacob's June 11 kill switch assay.  Every single plate (3C+, 3C-, 22A+, and 22A- in both 5 uL and 10 uL plating quantities) grew a lawn.  Only 3C+ and 3C- plates had a few distinguishable colonies.
</P><P>Checked plates of the cells transformed on June 11 by Ting-Chia and started 3 mL cultures for miniprepping.  Only the DH5a pIJ790 and EPI300 pIJ790 transformed with the 14C and 7C biobricks, respectively, did not seem show any growth upon the first check.  It was unclear as to whether there were actually any 7C (EPI300 pIJ790) colonies growing, so both plates were returned to the 37ºC room to incubate.  For the other plates that did show growth, single colonies were picked and inoculated in 3 mL starter cultures for miniprep.
</P><P>QIAprep Miniprep kit from Qiagen was used to miniprep 4-3I, 2-3C, and 3-22A.  The cells used were previously inoculated cultures that were stored at 4ºC.  DNA product was stored in sterile dH2O and frozen in the -20ºC freezer.
</P><P>We received plates of <I>Rhodococcus</I> sp. strain JVH1, <I>Rhodococcus rhodochrous</I> IGTS8 (ATCC 53968), and <I>Baillus sphaericus</I> IGTS9 (ATCC 53969).  Single colonies were picked and inoculated in 3 mL LB at 37ºC at first.  Plates were then parafilm-sealed and stored at 4ºC.
</P><P>After 20 hours at 37ºC, the plates with 14C and 7C (EPI300 pIJ790) had colonies.  14C only had one colony, while 7C had a couple of small colonies.  The origins of the colonies were uncertain, so the plates were discarded and the cells will be transformed in K12 the next day.  Also, 13M biobrick (RFP gene), which has been verified, will be transformed into EPI300 pIJ790 and DH5a IJ790 to check if the competent cells are working properly or if they are to blame for the failed 14C and 7C plates.
</P><P>Acquired trp, tyr, met, and arg for use in our co-culture experiments.
</P><P>- Tingchiawong
</P><H2><SPAN class="mw-headline" id="June_14">June 14</SPAN></H2><P>Transformed the remaining interesting kill switches without promoters into K12.  Also transformed 17F into EPI300 pIJ790 and DH5a pIJ790 cells to determine if previously failed plates were due to ineffective competent cells.  All transformed cultures were then plated onto appropriate antibiotic plates (+Amp or +Kan), along with a control for each.  An extra control plate was spread onto a Chlor plate from a batch that previously produced unclear results.  The plates were placed at 37ºC at noon and left them to grow overnight.
</P><P>Made fresh batch of LB + Amp plates, which were used for half the Amp resistant samples from above.  Check control plate tomorrow for colonies to ensure plates work.
</P><P>Grew two 250 mL cultures of EPI300 to make into competent cells.  Iced them when OD<SUB>600</SUB> reached 0.34 and 0.30.  Finished glycerol stocks were stored in new freezer box &quot;iGEM 2012 Comp Cells.&quot;
</P><P>- Tingchiawong
</P><P>
PCR Party held in the evening to start PCR experiments. PCR programmed to run overnight at keep products chilled at 4ºC until tomorrow, when gels will be ran.
- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><P>Ruichen's Protocols learned for PCR:
</P><P>1. get PCR tubes
</P><P>2.get all reagents (except for pfu)
</P><P>3. make master mix, in centrifuge tube
</P><P>4. aliquot into PCR tubes (Master Mix)
</P><P>5. add into the remaining reagents, except the pfu (normally jus the enzyme, and when colony PCR, pick a colony in stead of adding DNA). 
</P><P>6. keep sterile, pick colony (using tips, 10µl), add into tubes
</P><P>7. add pfu
</P><P>8. set up reaction conditions
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><H2><SPAN class="mw-headline" id="June_15">June 15</SPAN></H2><P>A very busy day!
</P><P>1)Made 1 liter's worth of Kan plates, and about 3/4 liter's worth of chlor plates. We are now out of empty plates, and they have been placed on the list of things to buy. The Hallam lab is also out of empty plates. This is not good, for many of our knockouts are replaced by tetracycline resistance, and we have no tetracycline plates. We may have to scrounge up some from other labs. Noting that the kan plates that we used previously often grew colonies for negative controls, we increased the amount of kan stock we added from 1 mL to 1.5 mL. We have yet to test whether that is too great for the cells with the appropriate plasmid, but we have confirmed that it is lethal to normal cells by the fact that nothing grew on the negative test for the epi300 cells that Ting recently made. 
</P><P>2)Tested epi300 cells by transforming with the rfp cassette for easy identification of whether or not the transformation was actually successful, or if other factors were in play regarding resistance. Negative controls were also employed for the newly made plates.
</P><UL><LI>Update: many colonies on the rfp plate, but none on the negative controls. The transformation efficiency appears to be slightly lower than previous competent cells, but is still fairly high.
</LI></UL><P>3)Miniprepped the resistance cassette colonies. Previously, to do the PCR, we just picked a colony, but we thought that it would be beneficial to have a plasmid stock in case we needed to repeat the PCR. 
</P><P>4)PCR with new (biobrick) primers was attempted. We received primers for the Dsz genes and also for the amino acid genes. We did not, however, receive the DszA forward primer, and decided to hold off on working with the Dsz genes until later. We performed a cPCR for all the genes we had, and ran the results on the gel.
</P><P>5) We attempted to do a solvent tolerance assay, but were thwarted. As it turns out, gasoline has some rather unpleasant volatile compounds that were interpreted by other members of the LSC as dangerous. Also, the cells that we wanted to grow showed no growth for the first 5 hours. Personally, I think that while it would be nice to have the relevant data, it seems that the hurdles to test to see if the addition of the prefoldin chaperone helps our cultures grow in organic solvents to remove the sulfur compounds are too great. We may revisit the project, but right now, it seems that we should put the idea behind us. As of 2:18, June 16th, nothing is growing in John's other experiment, which may or not be documented on the wiki.
</P><P>6)Ran the gel. Below are the results from yesterday's and today's PCR. Cameron made the gel and taped the edges of the gel box to prevent the gel from escaping. The gel itself was made to 1% agarose in 0.5% TBE, and run on a 50 well gel box. 3 uL of loading buffer were used for 8 uL (10 uL for yesterday's products) sample to increase contrast, and various ladders were tested. The negative control for today's PCR popped open during the reaction, and evaporated. Despite this, I feel fairly confident that the data is accurate. 
</P><P>Running a gel
</P><OL><LI> Broad Range Ladder
</LI><LI> TrpA (biobricks)+DMSO
</LI><LI> TrpA 
</LI><LI> TrpB +DMSO
</LI><LI> TrpB 
</LI><LI> ArgC +DMSO
</LI><LI> ArgC 
</LI><LI> ArgE +DMSO
</LI><LI> ArgE
</LI><LI> MetA +DMSO
</LI><LI> MetA
</LI><LI> TyrA +DMSO
</LI><LI> TyrA 
</LI><LI> empty
</LI><LI> 1kb ladder
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> Mehul's Stuff metA p1002 Kan
</LI><LI> empty
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> Ruichen's Stuff
</LI><LI> empty
</LI><LI> 1kb + Ladder
</LI><LI> TrpA tet 
</LI><LI> TrpA tet +DMSO
</LI><LI> TrpA tet +MgCl2
</LI><LI> TrpA tet +DMSO+MgCl2
</LI><LI> ArgE tet 
</LI><LI> ArgE tet +DMSO
</LI><LI> ArgE tet +MgCl2
</LI><LI> ArgE tet +DMSO+MgCl2
</LI><LI> TyrA tet 
</LI><LI> TyrA tet +DMSO
</LI><LI> TyrA tet +MgCl2
</LI><LI> TyrA tet +DMSO+MgCl2
</LI></OL><UL><LI> Figure 1. Myriad Gel 
</LI></UL><P>As you can see, we have successful bands for tyrA (from genome, biobrick primers) with DMSO, MetA with kanamycin resistance (for knockouts), trpA with tetracycline resistance, ArgE with tetracycline resistance, and tyrA with tetracyline resistance. The 1 kb+ ladder appears the most useful in this experiment. 
</P><P>- <A href="/wiki/index.php?title=User:Jacobtoth&amp;action=edit&amp;redlink=1" class="new" title="User:Jacobtoth (page does not exist)">Jacob Toth</A></P><P>Jacob summarized well, and I am just going to add more info based on his summary. 
</P><P>As for the perish dish, we used up all ours and also used the remaining a bag and half of Hallam Lab's. (which we need to reimburse back to them). 
</P><P>I (Ruichen) added perish dish (10 bags) to the Hallam's Lab ordering list, using iGEM's name. 
</P><P>As for the organic solvent tolerance assay, indeed there has been no growing sign in the transformed culture (with 3I plasmid) for the first few hours, and it was left in the 37 degree room over night. 
</P><P>and for the controlled cultures, they were growing as expected when the organic solvents (Pentane and Octane) were not introduced, the experiment was continued (with the disruption of Jacob mentioned before). The organic solvent were added into controlled cultures at different concentrations (0%, 5%, 10%, and 15%), and their growth were monitored roughly each hour for the first few hours, and when the disruption occurred the cultures were taken out of the 37 degree room and stored in the 4 degree room at 5 p.m.
</P><P>I (Ruichen) returned to the Lab after dinner at 8:30 p.m., and each culture were examined for OD readings again, and the cultures with Pentane and Octane were incubated in the incubator with vacuums at 37 degree for 1 hour and 30 minutes respectively. Then the OD were read again at 9:30 p.m.. 
</P><P><B>The detailed OD reading data will be posted later with analyzed results. </B></P><UL><LI>Figure 2. Octane Tolerance Control Test
</LI></UL><UL><LI>Figure 3. Pentane Tolerance Control Test
</LI></UL><P>
-  <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>The Amp plates made the day before worked.  However, the Kan plates (negative control, EPI300 pIJ790, and DH5a pIJ790) showed unexpected results.  The negative control, which only had untransformed K12 cells plated, had more colonies growing than those from the plates with resistance-transformed EPI300 pIJ790 and DH5a pIJ790.  Even then, there were only a few small colonies on the EPI300 pIJ790 plate and none at all on the DH5a pIJ790 plate.  It looks like the Kan plates may not be working, and it is possible that the EPI300 and DH5a cells with the pIJ790 plasmid are not competent.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><H2><SPAN class="mw-headline" id="June_16">June 16</SPAN></H2><P>The time I came in Joe was making the PCR gel. 
</P><P>The cultures with 3I plasmid that left for growth in the 37 degree room actually did turn a bit cloudy (6:00 p.m.).
</P><P>Their ODs were then measured with a blank LB for reference, and the detailed info will be updated. 
</P><P>For the remaining cultures, 100µl of each culture was tested on Chlo plates with an control of K12 (20µl)  to see if they are actually the cell cultures that of our interest, and also testing the chlo plates that are newly made, and the remaining cultures were stored in the 4 degree room for potential future usage. 
</P><P>The plates were left in the 37 degree room, and will be collected by Joe and stored in the 4 degree room. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><H2><SPAN class="mw-headline" id="June_17">June 17</SPAN></H2><P>As shown above in the gel image, there is still some PCR that needs to be done. Additionally, it appears that Joe's knockout PCR was not working either, so that PCR was to be repeated as well. It appears apparent that increased concentrations of DMSO and magnesium chloride help the reaction, so the appropriate amount of each was added to the mastermix. The reactions were as follows: TrpB with Kan cassette, TrpA, TrpB, ArgC, and MetA. 
</P><P>Since we were planning to put these into biobricks, it was imperative that we made sure that there were no restriction sites. To do this, we used the NCBI data for the gene sequence and ran it through Nebcutter, searching for all illegal sites. No illegal sites were found in TrpB, TyrA, TrpA, ArgC, or MetA. Lucky.
</P><P>As shown above, a reaction for the tyrA gene worked. Using this PCR product and the rfp plasmid, we digested with EcoRI and PstI and ligated with T4 DNA ligase. The resultant product was used to transform Epi300 cells. Tomorrow, we will check for white colonies among the red ones, which would suggest that the ligation was successful. A colony PCR will be done to confirm.
</P><P>-Jacob
</P><P>
Made more agar plates (Tet) (15 plates) (with an concentration of 1/1000 dilution rate) (and this has been proven to be too high of a concentration later). 
</P><P>-Ruichen
</P><H2><SPAN class="mw-headline" id="June_18">June 18</SPAN></H2><P>Yesterday's plates did not grow, but the experiment was repeated today.
</P><P>1) Made tetracycline plates using LB agar and Hallam lab tetracycline, diluted 400 uL in 400 mL. Previous kan and amp plates were shown to not have LB in them, and were also discovered not to support growth. However, a working kan plate was gifted by Cameron and used for a knockout experiment. 
</P><P>2) A gel from yesterday's PCR was shown to have some products that worked, while others did not. The MetA, TrpA, TrpB, and ArgE biobrick PCR's worked fine, but the ArgE-Kan cassette and the ArgC biobrick PCRs did not, and were not repeated today. From personal correspondence with Joe, his PCR was not working either, despite trying different cycles and conditions. He suspects that there may be something wrong with the template, given the inconsistent performance of the Kan plates. 
</P><P>3) The biobrick PCRs that did work were digested with EcoRI and PstI, as was the psb1C3 linearized plasmid backbone. 
it was unknown whether or not the plasmid had any methylation, so DpnI was used only in the PCR product digestion. The procedure will be uploaded to the wiki in the near future. A gel was made showing the ligation products.
</P><P>4) The biobrick ligation products were used to transform K12 cells and were plated appropriately. The successful amino acid-antibiotic cassettes were also transformed into EPI300, DH5a, and BL21 cells with the appropriate recombineering plasmid, and plated. To date, all the tet resistance strains have been plated, as well as Mehul's Amp resistant strain. The kanamycin resistant strains remain recalcitrant, perhaps due to, again, the inconsistent performance of the kanamycin plates. 
</P><P>5) Joe, looking into rhodococcus growth, decided to make some TB.
</P><P>It should be noted that several of the transformations sparked during the electroporation procedure. Some success has been reported with sparked cultures, so they were plated anyways. 
</P><P>As an update, here is a table showing what has been done so far. 
</P><TABLE class="wikitable"><TBODY><TR><TD> Gene</TD><TD>Type</TD><TD>PCR</TD><TD>Ligation</TD><TD>Transformation
</TD></TR><TR><TD>TrpA+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>TrpB+Kan resistance</TD><TD>Recombineering</TD><TD>Unsuccessful</TD><TD>NA</TD><TD>No
</TD></TR><TR><TD>ArgE+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>ArgC+Kan resistance</TD><TD>Recombineering</TD><TD>Unsuccessful</TD><TD>NA</TD><TD>No
</TD></TR><TR><TD>TyrA+Tet resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>MetA+Amp resistance</TD><TD>Recombineering</TD><TD>Successful</TD><TD>NA</TD><TD>?
</TD></TR><TR><TD>MetA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>ArgE</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TrpA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TrpB</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>TyrA</TD><TD>Biobrick</TD><TD>Successful</TD><TD>Done</TD><TD>?
</TD></TR><TR><TD>ArgC</TD><TD>Biobrick</TD><TD>Unsuccessful</TD><TD>No</TD><TD>No
</TD></TR></TBODY></TABLE><P>It should be noted that when a ligation has been done, it does not mean that it was successful. We will not know until we see growth on the plates and a cPCR of the colonies has been done. Even then, it would be useful to have it sequenced. 
</P><P>-Jacob
</P><P>
Worked with Jacob on steps 3 and 4 listed above. I made the gel, practised loading samples into wells (with Ruichen), and learned that the machine used to see the results is finnicky. Unfortunately, the 1 kb reference ladder did not work (unexpectedly and for unknown reasons).
</P><P>(Step 3) Transformation and plating of the biobrick PCR products were the last things we did today. Of the 5, only TrpB sparked during electroporation. 1 uL of PCR product was used to transform cells.
</P><P>(Step 4) Of the successful amino acid/antibiotic cassettes, Jacob and I transformed genes &quot;4&quot;, &quot;7&quot;, &quot;12&quot; (Tet resistant) and &quot;M3&quot; (Amp resistant)- names of the genes to follow - into EPI300, DH5a, and BL21 all with the recombineering plasmid pIJ790. The ones that sparked during electroporation are: 4 (EPI300), 4 (DH5a), 7 (BL21), and M3 (BL21). Quite a mix, but the addition of DMSO and MgCl<SUB>2</SUB> to boost PCR efficiency increased the salt concentration of the solution and may have caused sparking. Gene 4 had both compounds added and 7 had DMSO (unsure about M3 - that was Mehul). Only 1 uL of PCR product was used to transform cells to reduce chances of sparking, which may have helped.
</P><P>- <A href="/wiki/index.php?title=User:Grace.yi&amp;action=edit&amp;redlink=1" class="new" title="User:Grace.yi (page does not exist)">Grace.yi</A></P><H2><SPAN class="mw-headline" id="June_19"> June 19</SPAN></H2><P>Designed primers with Jacob and Mehul
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>Designed primers with Ruichen and Mehul.
</P><P>More specifically, the primers were for constitutively expressed GFP, RFP, and YFP to be put in the PCC1FOS vector. An EcoRI site was added just before the start of the promoter, and a BamHI site was added to the 5' end of the reverse complement. The melting temperature was normalized to about 59°C, and the primers were checked for any secondary structure. 
</P><P>There were no colonies on the plates transformed with any putative biobricks, so the procedure was attempted again.
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="June_20"> June 20</SPAN></H2><P>Plated the cultures with 3I plasmid on Kan for control and Chlor for testing purpose. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>Still no biobrick colonies. Tried again with an altered ligation protocol. Also ordered a variety of kill switches from the registry, along with Joe's Panamanian Pseudomonas Rhamnosyltransferase. Repeated PCR on things that had previously failed, and got the final amino acid biobrick and Ruichen's knockout resistance construct. Joe's knockout resistance cassette remains recalcitrant. It should be noted that I ran reactions with either PFU or phusion, and only the phusion worked. This may have been because of the low volume (0.5 uL) used.
</P><P>At the meeting, the gasoline debacle was discussed, as was the general progress of the project, and potential fixes for some of the problems that we have been having. 
</P><P>I suggested to Joe that he use the 2-17F biobrick for his forward kanamycin resistance cassette, and other ideas were thrown around as well. 
</P><P>-Jacob
</P><P>After consultation, we moved the IGTS8, IGTS9, and Rhodococcus JHV1 strains that were growing on terrific broth and also ones growing on LB from the 37C shaker to one at 30C.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><H2><SPAN class="mw-headline" id="June_21">June 21</SPAN></H2><P>The culture has been proven to be transformed successfully with the 3I plasmid! They grew colonies on the Chlor plate and not the Kan plate. 
</P><P>Learned how to miniprep
</P><P>Checked out some cool posters for the microbiology Conference, and found that there is a assay of interest. The assay (2,6 DCPIP) helps organisms to grow on a diesel contaminated soil. 
</P><P>- <A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><P>Today was relatively fruitful. All amino acids have been PCR'd out of the genome, all the resistance cassettes are ready for the knockouts, and several colonies appeared on the ArgE plate. One colony was selected and grown in culture. The rest of the biobrick ligations were redone and allowed to ligate overnight. 
</P><P>The digests of the products were done with EcoRI and PstI in buffer 3, as opposed to the EcoRI-HF and PstI in buffer 2 we did previously. This was done because PstI has 100% activity in buffer 3, as opposed to 75% activity in buffer 2. Also, we learned that EcoRI-HF has 0% activity in buffer 3. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="June_22">June 22</SPAN></H2><P>Transformed with the overnight (Room temperature) ligation, using 1 uL ligation mix and 1800V. Joe then plated the cells after recovery. 
</P><P>Cameron reported knockouts!
</P><P>Found a MetA biobrick colony on the appropriate plate from 2 days ago. Decided to grow it up in culture and run a PCR to confirm that it was successful. 
</P><P>Had previously grown up 10 mL of each fluorescent strain (GFP, RFP, YFP), but the plate reader became unavailable. 
</P><P>Wanted to design mutagenesis primers, but couldn't remember what software to use. For future reference:
</P><P>NCBI for FASTA
</P><P>Nebcutter to find sites
</P><P>Virtual Ribosome for associated peptide
</P><P>Wikipedia for codon table
</P><P>PrimerX for basic design.
</P><P>To whom it may concern, I think doing 4 sequential site directed mutagenesis reactions on dszA is going to take far more time and nearly as much money as simply synthesizing it. 
</P><P>Joe appears to be successfully growing IGTS8 (Rhodococcus) and IGTS9 (Bacillus) in TB at 30°C, and Pseudomonas putida at 37°. He also reports being unsuccessful in doing a cPCR for the dszD gene.
</P><P>-Jacob
</P><P>The IGTS8 and IGTS9 cell cultures in both terrific broth and LB grew dense and cloudy at 30C.  Glycerol stocks will be made out of these.
</P><P>We have been having issues with having no colonies grow on plates with cells transformed with PCR products ligated into vectors.  One possible cause is the transformation itself.  All of Joe's transformation resulted in a spark.  He had added 5 uL ligated to competent cells.  The high salt concentration may have heavily influenced the sparking.  So, I redid some of the transformations in a smaller volume of ligate.  I transformed Joe's ligated 3:3 dszB, 3:3 dszC, and 3:1 dszB plasmids into EPI300 cells.  Added 0.5 uL ligate to 40 uL competent cells and electroporated them.  The 3:1 dszB cells sparked during electroporation.  The cells were left to recover undisturbed at 37C for an hour before plating 50 uL transformants onto Chlor plates.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><H2><SPAN class="mw-headline" id="June_23">June 23</SPAN></H2><P>Made competent <I>Pseudomonas putida</I> cells using the <A href="/Competent_Cell_Production" title="Competent Cell Production" class="mw-redirect">Competent Cell Production</A> protocol.
</P><P>We had trouble getting the cells transformed with the ligated amino acid pathway genes to grow when plated.  We're unsure if the problem lies in the digest, the ligation, or the transformation.  Thus, we decided to redo all these steps.
</P><P>Re-digested the &quot;M,&quot; &quot;TA,&quot; &quot;TB,&quot; &quot;TyrA,&quot; and &quot;ArgC&quot; PCR products provided by Jacob.  Made a master mix of 5 uL NEB buffer 2, 0.5 uL BSA, 0.5 uL EcoRI HF, 0.5 uL PstI, and 18.5 uL dH2O.  Incubated each of the mixtures of 4 uL master mix plus 4 uL PCR product in a Thermocycler for ____.
</P><P>Re-ligated digest of &quot;M,&quot; &quot;TA,&quot; &quot;TB,&quot; &quot;TyrA,&quot; and &quot;ArgC.&quot;  We suspect that it may be the ligase we used that was at fault for the undesired results seen in previous experiments.  So, we decided to add in more ligase and add in ligase from 2 different sources/tubes.  We added 1 uL pSBIC3 EP, 4.0 uL dH2O, 1 uL T4 ligase buffer, and 0.5 uL from each tube of T4 ligase were added to 3 uL of the respective digest products.  A control was also set up so that there was only plasmid DNA added.  The ligate mixtures were placed in the Thermocycler to incubate at 16C for 30 minutes and inactive at 65C for 20 minutes.
</P><P>John miniprepped MetA and ArgE, which are being prepared as potential biobricks.
</P><P>A new set of PCR reactions was set up for MetA (miniprepped potential biobrick), TyrA (colony PCR), ArgE (miniprepped potential biobrick), and yddG (colony PCR). Made a ligation master mix by combining 60 uL Phusion 5X buffer, 147 uL dH2O, 15 uL DMSO, 12 uL MgCl2, and 18 uL 10 mM dNTP.
</P><P>- <A href="/wiki/index.php?title=User:Tingchiawong&amp;action=edit&amp;redlink=1" class="new" title="User:Tingchiawong (page does not exist)">Tingchiawong</A></P><H2><SPAN class="mw-headline" id="June_25">June 25</SPAN></H2><P>Designed primers for SDM of dszB gene
</P><P>PCRed yddG and Try A with Marianne 
</P><P>Had a Skype meeting with Calgary iGEM team (details can be found in <B>Team Collaboration</B> section
</P><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><H2><SPAN class="mw-headline" id="June_26">June 26</SPAN></H2><P>Designed primers for SDM of dszC gene
</P><P>Joined Jacob and Joe for transformation of Heat(chemical) competent cells: 
</P><TABLE class="wikitable"><TBODY><TR><TD> Plasmid transformed</TD><TD>Volume of Ligation Mixture used (µL)
</TD></TR><TR><TD>dszB, dszC (3 to 3 ratio, Old and New)</TD><TD>3
</TD></TR><TR><TD>MetA, TrpA, TrpB, TryA, ArgC, positive and negative control</TD><TD>7
</TD></TR></TBODY></TABLE><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><H2><SPAN class="mw-headline" id="June_27">June 27</SPAN></H2><P>Run the gel of Try A and yddg genes at 95 Volts for 1 hr and 30 min, with 1Kb ladder. 
</P><P>-<A href="/User:Ruichen" title="User:Ruichen">Ruichen</A></P><H2><SPAN class="mw-headline" id="June_30">June 30</SPAN></H2><P>In the last few days, we have confirmed that our restriction enzymes work, that our electroshock cuvettes do not due to corrosion from bleach, and still have had no confirmed biobricks. We picked another candidate colony from the ArgE plate because the PCR of the plasmid between VF2 and VR (standard biobrick sequencing primers) was only about 200 bp, while we were expecting about 1.2 kb. We got the same result from our potential MetA colony. We received new ligase, and if that doesn't work, we are going to try the gibson assembly method. Using a previous restriction digest, we attempted another ligation of our new potential biobricks.
</P><P>The first set of chemically competent cells did not grow anything. The procedure was repeated and completed today. The cells were transformed with a known plasmid and appropriately plated. 
</P><P>Rafael gave us 4 new electroshock cuvettes to temporarily replace the ones that we had destroyed. Joe used 3 of them to test his knockout, and as of today, there is one candidate colony growing on a 1/4 antibiotic concentration plate. A battery of tests have yet to be run to confirm that it is a knockout. 
</P><P>Yesterday, we went to Dr. Ramey, who had taught students who made a successful knockout using the lambda red system. They used a different recombineering plasmid than we did, and we received this plasmid. It was, however, growing in a strain that we were unfamiliar with, and we decided to isolate the plasmid and transform our Epi300 cells with it. 
It is currently growing in a culture at 30°.
</P><P>We received only the TrpB gene from the registry, and successfully grew it on a plate. We are now growing a culture to isolate the plasmid and if Joe's knockout is indeed a knockout, we can characterize the existing biobrick part by complementation. 
</P><P>We ordered the Gibson assembly kit, and it ran upwards of 700 dollars for 50 reactions. 
</P><P>Joe successfully PCR'd out the DszD gene from Rhodococcus genomic DNA that he isolated.
</P><P>We made Tet, Kan, and Amp plates with 1/2 and 1/4 antibiotic concentration to try growing our knockouts. There was no growth from a resistance-less K12 cell on any of the plates tested. 
</P><P>Our PCR for the yddg and tyrA genes was unsuccessful.
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_1">July 1</SPAN></H2><P>There are many (hundreds) of colonies on Joe's recombineering plates, and the concentration of colonies goes up as the concentration of antibiotic goes down. It appears that the addition of arabinose to the culture did nothing to increase recombineering efficiency. There are several large, well defined colonies on the full antibiotic plate. It appears that 2 days are required for sufficient growth at 30°.
</P><P>With this success in mind, I PCR purified a couple of other knockouts (ArgE Tet and TrpB Kan) and transformed them, reusing the electroshock cuvettes we borrowed from Rafael. A pack of 50 was ordered so we could do more than 4 transformations per day. They are currently growing in the 30°C room on the second floor.
</P><P>We only had one chlor plate left, and 4 potential biobricks which were to be put on the chloramphenicol resistant pSB1C3 plasmid. Since there was only one plate (and one electroshock cuvette), only the MetA biobrick ligation product was transformed. 
</P><P>The pKD46 recombineering vector that we received from Dr. Ramey grew well overnight at 30°C, and was miniprepped, and then transformed into the Epi300 strain. The recovery period was done at 30°C as well. If the transformation is successful, Ting will make electrocompetent cells out of the Epi300+pKD46. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_7">July 7</SPAN></H2><P>In the past couple of days, we have found candidate knockouts for all relevant genes but trpA, and attempted to confirm that they were all actual knockouts by PCR. Unfortunately, none of the PCR's worked. We suspect that it was because of a bad enzyme, so some of the reactions were repeated. Currently, the results are inconclusive as to whether or not the knockouts are actually knockouts. We also ordered the relevant knockouts from the Keio collection.
</P><P>Colonies with potential biobricks of trpA, tyrA, argE, metA, and dszD were successfully grown. This time, chemically competent cells were used, and they were plated on 1/2 concentration chlor plates. We also transformed and grew out the previously characterized minC biobrick.
</P><P>To physiologically characterize the knockouts, we made M9 plates, and plan to set up an amino acid gradient. Sadly, we ran out of M9 salts before we could make tet plates, so some knockouts can't be tested in this manner yet. We also made spec plates to grow some things from the registry, but a lawn grew on the negative control. Looking into the literature, it looks like the concentrated antibiotic we were using was not the correct strength. 
</P><P>PCR of the yddg gene failed yet again. It is possible that the primers are not correct. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_9">July 9</SPAN></H2><P>Does anybody else update the wiki these days?
</P><P>We have been in contact with Calgary regarding kill switches. More details to follow later.
</P><P>One of our potential knockouts and a positive control were grown on the M9 plates. There was growth on both plates, although the growth on the positive control was much faster and greater than the potential knockout. Since the potential knockout was able to grow on the amino acid-less M9 media, it appears to be not auxotrophic. Characterization with PCR continues to be difficult, and it was suggested that we simply sequence the strains. 
</P><P>The plates with the potential knockouts were placed in the 37° incubator overnight to cure the plasmid.
</P><P>The potential biobricks were successfully miniprepped, and a PCR was performed to isolate what was actually between the VF2 and the VR primers in the plasmid. The product will then be run on a gel, and if it appears the correct length, will be sent for sequencing.
</P><P>Our Pseudomonas competent cells appear to work (conferring antibiotic resistance), but they are not expressing rfp. 
</P><P>We made spec plates, but a lawn grew with our transformed cells. Interestingly, on top of this lawn grew several well defined colonies. We attempted to grow these colonies (and the proper negative controls) in varying amounts of spectinomycin. It is possible the plates had insufficient spec, or that the strain we are working with is naturally resistant (there was a lawn, but no large colonies, on the negative control).
</P><P>Several more biobricks from the registry were transformed.
</P><P>Joe made M9 plates with an amino acid, to further test his knockout. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_10">July 10</SPAN></H2><P>The PCR to isolate and amplify the insert from the potential biobrick showed many bands, characteristic of non-specific primer binding. As another test to see if the insert was correct, we cut with EcoRI and PstI to look for the appropriate insert size, and indeed found that there was an insert at about the right size. With this in mind, simply sequencing from the plasmid is the next step. However, our sequencing company only accepts orders until 1:30 PM, and it was 4:30 before everything that needed to be done was done. 
</P><P>The transformations from yesterday's transformations appeared to work. Also, interestingly, the colony from on top of the spec plate lawn, but not the spec plate lawn itself, was able to grow in 40 and 80 ug/mL spec. 
</P><P>Cameron reports knockouts from Yale.
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_15">July 15</SPAN></H2><P>Sent the potential biobricks for sequencing last Friday, ordered primers for mutagenesis on DszC, ordered primers for attachment of fluorescent genes to amino acid genes, received all but yddg knockouts from Yale, found that the Arg knockouts work as described. Several other primers were also ordered, but I am not privy to their nature. 
</P><P>An attempt at doing an experiment to calibrate the fluorescent proteins for the plate reader (looking for quenching, linear relationship between OD and fluorescence, etc) was done on Sunday, but as it turns out, we had the wrong promoter for a fluorescent protein, and the wrong antibiotic resistance for one of the other proteins. Given this, we gave up, and decided to do it later with the proper strains. 
</P><P>Recently, there was a meeting with the kill switch team from Calgary. They suggested doing more sequencing, and that is indeed a good idea if we want to pursue this project. I think that the intein primers were ordered, but again, I am not privy to exactly what was ordered.
</P><P>Competent cells were made from each of the knockouts, and they worked well. The yddg strain was ordered through collaboration with Calgary as well as the keio collection proper in Japan. 
</P><P>Today, I watched two pairs of eyes. One was black, and the other grey. And while the owners thereof, for the space of five seconds, walked past each other, the grey shot at the black, and the black fiddled the grey. Bonus points for identifying the poem without using a search engine. 
</P><P>Our Gibson assembly kit came! Doing the math, it turns out that it is about 30 times more expensive than gold, so we will use it wisely. Personally, I would have liked to just order the enzymes and buffer reagents, but c'est la vie. Our biobrick assembly kit also came, but that is rather less exciting.
</P><P>The sequencing results from our amino acid biobricks came back, and it looks like only MetA was successful. It should be noted that the melting temperatures of the Vf2 and the VR primers are 60°, for cPCR validation of biobricks.
</P><P>Several new combinations of fluorescent genes, antibiotic resistances, and Keio strains were made. We used the wrong ones last weekend, but now, we are going to test TyrA- +GFP, TrpA- +RFP, MetA- +YFP, and also wildtype with the fluorescent genes. Recently, we switched backbones for the 13M RFP to Psb1C3, for we do not know the provenance of our current psb1C3 with rfp (we suspect it might be on the lac operon, as per the registry guidelines), and we also put the GFP on the psb1C3 instead of a kanamycin resistant plasmid, because we were transforming into kanamycin resistant auxotrophs. 
</P><P>Just for qualitative fun, we are trying to grow the ArgC- and the ArgE- together in M9 to see if anything results, with negative controls for both. We also tried growing a wildtype with a TrpB auxotroph with rfp to look for qualitative red fluorescence. The fluorescence calibration experiment must be done before we can do these sorts of things quantitatively. 
</P><P>We miniprepped 4/6 of the biobricks that we ordered from the registry. We intend on sequencing them in the near future. We, lacking LB amp plates at the time, plated one on an M9 plate, but we now think that the strain the registry sent us is auxotrophic for something. We replated on LB amp after making new plates. The last one that we ordered did not arrive. 
</P><P>Joe made M9 plates with and without 1 mM arginine, and it was found that the two arginine auxotrophs grew as expected. 
</P><P>We bought a fridge for our new office, but have yet to fill it. If anyone with extra snacks reads this and wants to help, our fridge is currently underfilled. 
</P><P>Overnight cultures have been set up to make competent cells out of DH5a, K12, and IGTS8.
</P><P>Some new polymerase system has found its way into our freezer, and nobody seems to know its provenance. Since we do not share the freezer, we assume some kindly benefactor gifted it to us. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_17">July 17</SPAN></H2><P>Today, we made dh5a and k12 competent cells. We also ran a cPCR on our biobrick colonies that turned out to not be biobricks.
</P><P>As said yesterday, we grew the two arg auxotrophs together, and they grew substantially more (~0.15 OD600) than the negative control. The wt grew to an OD of about 1. We are growing them for another night to see if the OD of the mix increases at all. 
</P><P>Since sequencing confirmed a MetA biobrick, we transformed K12 cells with the remaining stock of our plasmid, so we could amplify it. 
</P><P>All relevant transformed strains from yesterday grew well on the plates. We are now ready to properly do the fluorescence calibration experiment, but due to time constraints, we are going to do it on Thursday.
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_19">July 19</SPAN></H2><P>We tested the growth rate of our 3 main auxotrophs in various (log scale from 100 pM to 10 mM) concentrations of the amino acids to do monod kinetics. We also miniprepped our metA biobrick (so we have enough to send to the registry and use for experiments), new potential candidates for other biobricks (tyrA and ArgC), our construct which is identical to 13M on plate 3, except on a different plasmid, and a biobrick that we received from the registry. Sequencing to follow. 
</P><P>Joe has, to the best of my knowledge, identified potential biobrick colonies for the DszB,C, and D. Again, sequencing to follow. 
</P><P>We have, in the past few days, designed a plasmid that will aid in the testing of biobricks. The primers are to be designed and ordered today, if all goes according to plan. The primers that we tried to order through the lab a while ago spent some time in bureaucratic limbo, but I think they have been ordered now. Again, no one appears completely sure of what has been ordered, and one of our chief contacts on the subject has apparently gone on vacation. 
</P><P>It appears that Joe and Grace were attempting the fluorescence calibration experiment, but things apparently were going wrong, and we have not yet analyzed the data completely. 
</P><P>Concerning the kill switch plasmid parts, we want to take the psb1C3 plasmid, add a constitutively expressed LacI gene, and a lac promoter and rbs immediately before the cut sites. This way, a toxic protein can be well controlled. In fact, while we are designing this for toxic proteins, it could have uses in general protein production. Previous attempts for using the lac promoter on a high copy plasmid have failed leaky expression) due to the lack of the LacI protein produced in the genome. This new plasmid design remedies this, and will be accomplished by Gibson assembly. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="July_20">July 20</SPAN></H2><P>Obtained the data from the 3 main auxotrophs growth rate dependences of amino acid concentration. 
</P><P>Obtained the reading of fluorescence-population correlation plate. 
</P><P>-Ruichen
</P><H2><SPAN class="mw-headline" id="July_22">July 22</SPAN></H2><P>Joe did 5 more fluorescence-population correlation plate, though found out that the yfp is not expressing correctly. 
</P><P>-Ruichen
</P><H2><SPAN class="mw-headline" id="July_23"> July 23</SPAN></H2><P>Analyzed the data, and found out that the growth rate and the amino acid (Tyr) concentration has a correlation as follows: 
</P><UL><LI>Figure 4. cell growth-rate at different Tyr concentrations
</LI></UL><UL><LI>Figure 5. Log scale of cell growth-rate at different Tyr concentrations 
</LI></UL><P>It was found that the cells depleted the nutrient quickly at low Tyr concentration, and for future experiments it is suggested to use lower initial cell concentrations to extend the time of its exponential growth phase. Also, the wild type has almost the exact growth rate as the autotroph when the the Try concentration is exceptionally high (0.01 M). 
</P><P>
-Ruichen
</P><H2><SPAN class="mw-headline" id="July_26">July 26</SPAN></H2><P>The growth rate and the amino acids (Met and Trp) concentration has a correlation as follows: 
</P><UL><LI>Figure 6. cell growth-rate at different Met concentrations
</LI></UL><UL><LI>Figure 7. Log scale of cell growth-rate at different Met concentrations 
</LI></UL><UL><LI>Figure 8. cell growth-rate at different Trp concentrations
</LI></UL><UL><LI>Figure 9. Log scale of cell growth-rate at different Trp concentrations 
</LI></UL><P>
-Ruichen
</P><H2><SPAN class="mw-headline" id="August_2">August 2</SPAN></H2><UL><LI>Field trip to Chevron Burnaby site!
</LI></UL><P>
Ruichen
</P><H2><SPAN class="mw-headline" id="August_3rd">August 3rd</SPAN></H2><P>Started to work on the data obtained on July 28th plate readings. 
</P><P>Eventually arrived at the following graphs of growth rates observations, when n=3
</P><P>From these graphs, the maximum growth rate, and Ks values that are used in Monod Kinetics can be determined. 
</P><P>
Ruichen
</P><H2><SPAN class="mw-headline" id="August_9">August 9</SPAN></H2><P>Having trouble uploading the refinery trip pictures because the files exceed the 2MB limit! Any tips on shrinking file sizes would be much appreciated!
</P><P>- <A href="/wiki/index.php?title=User:Grace.yi&amp;action=edit&amp;redlink=1" class="new" title="User:Grace.yi (page does not exist)">grace.yi</A></P><H2><SPAN class="mw-headline" id="August_12">August 12</SPAN></H2><P>A large set of primers arrived, which gave us a lot to do.
</P><P>With a set of the primers that arrived, we were able to PCR amplify certain parts that will be able to be Gibson assembled. On the plasmid that has RFP (a modified BBa_K093012, placed on the Psb1C3), we wanted to put arabinose inducible MetA or arabinose inducible TyrA. However, one of the primers necessary to amplify the plasmid itself, which would be necessary to incorporate the proper homologous sequence for Gibson assembly, did not arrive. Additionally, the PCR to amplify the MetA gene did not work the first time, but has been repeated. The arabinose promoter was successfully PCR'd out of the E. coli genome for both cases. 
For the plasmid with YFP, the BBa_I13973, we wanted to put rhamnose inducible TrpA or TyrA after the YFP gene. We received all the primers necessary for this, but the PCR for the plasmid backbone gave several bands and was repeated at a higher annealing temperature. The results are not yet known. The rhamnose promoter was successfully PCR'd out of the genome, as was the TrpA gene. However, we lacked one of the promoters for the TyrA gene. Since there were several bands for the YFP plasmid, we did not do a Gibson assembly. Additionally, it is possible that the primers were designed for the pSB1C3 rather than the pSB1A2, but that should have little effect due to the similarities on the ends of the two plasmids. A simple digest and ligation should be able to move the part from the one plasmid to the other.
We also wanted to put a IPTG inducible (BBa_K091111, LacIQ) metA gene after a GFP. The PCR for the MetA gene and the GFP plasmid were successful (although the GFP plasmid appeared as a fairly weak band on the gel), but the PCR for the LacIQ promoter failed. The initial DNA for this PCR was added directly from the parts distribution kit, and as such the composition and concentration of exactly what was added was unknown. The PCR was repeated as a cPCR on the genome, so as to at least get a usable part with the right overhang. 
</P><P>The final primers for the gibson assembly of the kill switch primer also arrived. There are four parts to this plasmid: the plasmid backbone, the lacI promoter, the constitutively expressed lacI gene, and the constitutively expressed RFP for easy selection. The plasmid backbone, the promoter, and the constitutively expressed RFP PCR products has been ready for several weeks, but we lacked the primers for the constituvely expressed lacI gene until now. To generate the DNA for the constitutively expressed lacI gene, we took BBa_K081005 (const. promoter+rbs) cut with only SpeI and 	BBa_I732100 (lacI) cut with only XbaI, and ligated the two together, and then did a PCR on the ligation. It worked beautifully. 
</P><P>The primers also arrived to isolate the VMA Sce-I intein, and Alina provided yeast genomic DNA. The PCR was done successfully. However, upon further reflection, it appears that we forgot to DpnI digest the original template for the plasmid backbone on which we were going to add this part, and we actually did not have any product for it. Another PCR needs to be done, and DpnI digests will be done before running it on a gel. 
</P><P>A previous potential part, ArgE, was restriction confirmed, and will be sent for sequencing at the soonest convenience.
</P><P>The PCR for yddg was finally a success. The part has been restriction digested, along with the pSB1C3 plasmid, and a ligation will follow. 
</P><P>We finally received the DszA forward primer, and did a restriction digest on it as well. It will be ligated as well in the near future. It has several Pst1 sites, which limits some of the reactions we can do with it, but we can still place things before it by cutting with EcoRI and/or XbaI. 
</P><P>In an effort to see how much amino acid each type of cell in the co-culture will produce, we have been harvesting supernatants of our normal cultures as per the protocol in Kerner et al. Once we have supernatants of the cultures at various ODs, we are going to grow the auxotrophs in this supernatant, and then look at the growth rate and final OD. The list of which supernatants we have can be found in the lab. 
</P><P>Joe has found inconsistent results with his fluorescent proteins on the various promoters, and we have plans to standardize the promoter and rbs for the fluorescent proteins that will be used. Additionally, there is some overlap between YFP and GFP that is proving difficult to account for, so we are going to try to switch one to either BFP or CFP. 
</P><P>We also wanted to grow three of the consortia together to see what sort of final OD they reach, and have been monitoring it sporadically for the past 24 hours. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="August_13">August 13</SPAN></H2><P>Having harvested the supernatant of the various cultures at various ODs, we filter sterilized it with a 0.2 uM filter to remove any cells that may have still remained. Now that we have the sterilized supernatant, we are going to follow the protocol in Kerner et al 2012, and add 1/5 5X M9 media, and grow different auxotrophic cultures, noting the OD and growth rate. From this, we will try to calculate, or at least approximate, the amino acid export rate of the cells. 
</P><P>There were colonies on the MinC Intein plate, the ArgE BB plate, and the GFP construct LB plate, but nothing on the GFP construct M9 plate. This may have been because there was no lactose or IPTG to induce the metA gene. Interestingly, there was a green fluorescent patch, but it was rather weak and had no distinct colonies. We plan to pick several colonies from the LB plate and try growing them in M9 culture with and without IPTG overnight. We also plan to do a cPCR on the MinC plate using the intein primers. There were no colonies on the DszA or yddg plate. We have had no luck with electroporating cells with ligation mixtures, although we have had some success with heat shock competent cells. We think there may be something in the ligation mixture that is interfering with the electroporation, and could probably be fixed by purifying the ligation mix. 
</P><P>The data that we collected regarding the growth curves of the auxotrophs and the mixture are very different from what the plate reader previously showed. We think this may be because of a lack of correction for path length by the plate reader, but aren't entirely sure. 
</P><P>We also did a PCR on the TyrA BB product, the previous failed YFP plasmid, and several other previously failed PCRs. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="August_15">August 15</SPAN></H2><P>After two days of shaking at 37° in 1 mM IPTG, the supposed IPTG inducible metA constructs in the metA auxotrophs were unable to grow. 
</P><P>The gels of the PCR products that we have been getting lately have not been working properly. We think that there is a problem with the buffer due to how many times it has been reusued, or possibly by the new TBE that was recently made. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="August_16">August 16</SPAN></H2><P>The supposed metA constructs still appears unable to complement a metA knockout. It has been suggested that the concentration of metA may have been too high, and this would have led to a severe detriment to the cell, and it has also been suggested that the IPTG, being a rather old stock, was no longer good. Another step that we are going to take is to see if the expression of the construct can be seen on an sds page under induction.
</P><P>Despite, or perhaps because of this failure, we are moving on, trying to make new constructs that will be used for tunability. The YFP has yet to give us a solid band, so we are trying to move it into the psb1c3 plasmid in case there are some plasmid specific sequences that were giving us poor results. The ligation and transformation has been done, and we are waiting for colonies. The PCR products necessary for the gibson assembly of the GFP-IPTG-TrpA gene were also all available, and that gibson assembly has been done. However, due to the failure of several gels, there was very little product left, and PCR purification would likely result in unusuable quantities, so we tried a Gibson assembly with the straight PCR product, and we will see if it works. 
</P><P>All of the parts necessary for the assembly of the kill switch plasmid have been successfully PCR'd, but several of the parts fall victim to the same caveat as that of the previously discussed products. That being said, we tried it anyways,and will check for colonies tomorrow. 
</P><P>The TBE that we were using for the gels has not been working for anyone in the lab, although it has been suggested that that may simply because people don't change their buffers. The TAE gels work fine.
</P><P>Another PCR was done to generate some of the other parts required for assembly of other parts. The new forward primer for the arabinose promoter does not seem to be working, although the old one worked well. We recently ran out of phusion polymerase, and have resorted to using an eclectic mix of collected polymerases in our PCR box. 
</P><P>A cPCR of the colonies with the MinC protein with the Sce VME intein showed that the gene for the intein was present in the cells that were supposed to carry it! A small victory, perhaps, but it shows that the Gibson assembly works, contrary to out experiences with the positive control. A site directed mutagenesis must be performed on the gene to convert an internal lysine into a phenylalanine, which has been shown to make the intein temperature sensitive. The primers for this reaction have been eluted, but we are still waiting for a plasmid stock of the minC-intein plasmid. By the way, the MinC protein was chosen because it can be used as a kill switch and has the necessary sequence (G-C) that the intein uses for self excision. 
</P><P>There are two distinct bands on the RFP PCR, one at the correct length, the other much lower. Seems like a perfect chance for gel extraction. Several other products were also generated today, but there are still two essential parts, that of YFP and that of the new arabinose promoter, which have not yet been successfully PCR'd. 
</P><P>The cPCR on the DszA colonies gave no result. This may have been due to an unoptimized Taq protocol,and might be worth repeating. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="August_17">August 17</SPAN></H2><P>The Gibson assembled parts using unpurified PCR products did not work. I think it was because of the extra salts, etc that would be present in the PCR mix. As a note, make sure to somehow purify your PCR product before doing a Gibson assembly!
</P><P>There were many colonies on the 6I ligation heat shock (but not electroshock) plate. However, there were of insufficient size to work with, so they were left to grow for another day (or 2, considering how the power will be out on the 18th). 
</P><P>The plans for the metA construct induction were further discussed. Overnight colonies will be made of the construct and a negative control, then they will be diluted in 1 in 100 LB, allowed to grow to 0.4 OD, induced with 0.1 mM new IPTG, and harvested at a certain OD following this, then run on an SDS PAGE. This will test to see if the IPTG really would stimulate the production of the metA gene. The negative controls were not in place yet. Four potential constructs have been grown up overnight, and are labeled 1-4 on tape in the right section of the left shaker. Is there some label on the metA protein that we could use to do a Western? 
</P><P>The data from the supernatant growth experiment was partially analyzed. It appears that 14 hours was not sufficient to reach a final OD in some cases, but there was growth in many of the wells. It was also suggested that we could just test the relevant amino acid concentration using HPLC. One thing that I was wary about concerning this is that the HPLC would only pick up pure amino acid, and if there was, for instance, a short protein fragment, it could be used for growth, but not show up on the HPLC. Any thoughts?
</P><P>The cultures for the MinteinC (As it shall henceforth be known) grew well, but were not miniprepped. Once they are miniprepped, then site directed mutagenesis must be done. The cultures of the potential yddg biobricks grew well, and will be confirmed either by PCR of the plasmid using VR and VF2, or by restriction digest. 
</P><P>From our previous experiments concerning the consortia syntrophy, we saw that three cultures inoculated straight from LB into minimal M9 led to a mixed culture OD that reached about 1.3, similar to 0.1 mM (saturating) Met for metA-, and 0.1 mM Trp for TrpA-. We suspect that this is due to syntrophy, but have yet to do the relevant tests to see if 1 in 100 LB actually does supply significant amounts of amino acids. 
</P><P>I will be going on vacation for the next couple of weeks, and will return in September. 
</P><P>-Jacob
</P><H2><SPAN class="mw-headline" id="September_15">September 15</SPAN></H2><P><B>Traveling to Edmonton for aGEM!! </B></P><P><B>Team Dinner -at Vancouver International Airport(YVR)</B></P><P>
- Ruichen
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:British_Columbia/ConsortiaNotebook">http://2012.igem.org/Team:British_Columbia/ConsortiaNotebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/ConsortiaNotebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/ConsortiaNotebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/ConsortiaNotebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/ConsortiaNotebook&amp;oldid=239378" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>