<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_Attributions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/Attributions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/Attributions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Attributions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Attributions&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/Attributions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/Attributions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">UBC iGEM<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia">Home</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI></UL><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/ProjectConsortia">Microbial Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Desulfurization">Bio-desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A class="sf-with-ul" href="#">Results<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Data">Data</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Results">Key Results</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Medal">Medal Criteria</A></LI><LI><A class="sf-with-ul" href="#">Models<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Consortia">Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Pathway">Pathway</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP">IP &amp; iGEM</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP_FAQ">Patent Guide</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/Industry">Industrial Insight</A></LI></DIV><TITLE> British Columbia - 2012.igem.org </TITLE><P align="center"><FONT face="arial" narrow=""><FONT size="4"><B>Special Thanks</B></FONT></FONT></P><FONT face="arial" narrow=""><B>Wet Laboratory Acknowledgements</B>
The <A href="http://www.cmde.science.ubc.ca/hallam/index.php">Steven Hallam lab</A> for hosting our iGEM team, providing us with laboratory space and reagents through the 2012 competition
Dr. Bob Hancock of UBC for sending us a strain of Pseudomonas aeruginosa
Dr. Jay Keasling of UC Berkeley for providing us the Dsz operon in E.coli
Dr. Jonathan Van Hamme of Thompson Rivers University for sending us the Dsz operon<A href="http://www.vwr.com/">VWR</A>, Dr. John Church, the Annual Giving UBC Department of Microbiology &amp; Immunology and BIOL Alumni Appeal Fund, Harry Lim Incorporated and the UBC Faculty of Applied Science for donations of reagents and resources<B>Human Practices Acknowledgements</B>
Ian Bell from <A href="http://www.uilo.ubc.ca/welcome">UBC UILO</A> and Emma Macfarlane from <A href="http://www.gowlings.com/">Gowlings</A> for discussions regarding patents
Peter Wynne from <A href="http://www.chevron.ca/">Chevron Canada</A>, <A href="http://www.albertatechfutures.ca/">Alberta Innovates Technology Futures AITF</A> representative, Karen Budwill, <A href="http://www.osli.ca/">Oil Sands Leadership Initiative OSLI</A> representatives, John Vidmar and Nicolas Choquette-Levy, and Dr. Dusko Posarac of UBC Chemical and Biological Engineering for discussions regarding industrial applications of microbial consortia<B>Computational Modeling Acknowledgements</B>
Dr. Susan Baldwin of UBC Chemical and Biological Engineering, Gurpal Bisra and Shing Hei Zhan (UBC iGEM alumni) for advice on the consortia model
Niels Hanson from the Steven Hallam laboratory for advice on the pathway model<B>General Acknowledgements</B><A href="http://agemjamboree.eventbrite.com/?ebtv=C">Alberta Genetically Engineered Machines (aGEM)</A> judges and Dr. Phil Hieter of UBC for their constructive advice on our project and presentation<P align="center"><FONT face="arial" narrow="" size="4"><B>Collaboration with University of Calgary iGEM</B></FONT></P></FONT><FONT face="arial" narrow="">
We thank Dr. Lisa Gieg and University of Calgary iGEM for collaborating and splitting the DSZ pathway for characterization:

We wanted to collaborate with Calgary because of the friendship and hospitality they showed last year! Considering that we were both working with the same operon, we decided that Calgary would take DszA and DszB while we would take DszC and DszD. We also set up meetings to see what else we could do in collaboration. We initially wanted to see if we could do anything regarding modeling together, given that we were working with similar systems. However, Calgary was more interested in bioremediation, rather than fuel processing. We then struck on the idea of collaborating on kill switches.

Calgary wanted to characterize a few new induction systems, including a rhamnose promoter and two new riboswitches for their kill switches. They have yet to have success with their rhamnose promoter. They sent us the sequence before they attempted anything with it, and we used that sequence to design primers to get the rhamnose promoter out of the genome. 

Noticing the paucity of standardization of kill switches on the registry, we took it upon ourselves to develop a plan to test the efficacy of any new toxic protein that might be wanted to be tested for kill switch capabilities. There was a fair amount of talk about how best to do a death assay. We suggested using Dendra2, a fluorescent protein that could be 'broken' at a specific non-cytotoxic wavelength of light, allowing us to quantify metabolic activity after supposed death after toxic protein induction by looking for further expression of the fluorescent protein. They argued that a dormant cell may not express this protein, and we wouldn't really be looking at death. We then decided that we ought to instead focus on plating assays, for this better tests the viability of cells. We both tried testing existing kill switches on the registry, but were unable to get any of them to work. Calgary postulated that this was due to the arabinose promoter, which they could never get it to work. In the mean time, we both sequenced some of the toxic genes used as kill switches in the registry, and found that everything we sequenced was indeed what it claimed to be. 

Calgary also sent us the gene for HPAC, an alternative NADH-FMN oxidoreductase, to be used in place of DszD. It is a native E coli enzyme, and apparently works better, though over-expression causes hydrogen peroxide formation. We had plans to see if we could get it expressed in a pET vector to look for peroxide generation, but we have not done this yet.

Lastly, they sent us the yddg- strain from David's old lab at U of A, which could be used in complementation studies. Kerner et al. (2012) used the yddg part because it increases the rate of export of tyrosine by 3 times, and increases overall tyrosine production. We have created a constitutively expressed yddg part that we would ultimately like use to check for complementation (yddg also confers resistance to inhibiting concentrations of aromatic amino acids), but we have not made competent cells from this strain.<P align="center"><FONT face="arial" narrow="" size="4"><B>Collaboration with INSA-Lyon</B></FONT></P></FONT><FONT face="arial" narrow="">
To help team INSA-Lyon with their <A href="https://2012.igem.org/Team:Lyon-INSA/humanPractice">human practice project</A> investigating the how intellectual property affects iGEM, we have provided them with the data from our <A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP">survey results</A>.  In return, INSA-Lyon assisted us by giving us feedback on drafts of our <A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP_FAQ">iGEM-specific intellectual property guide</A>.
<FONT face="arial" narrow="" size="4"><P align="center"><B>Our 2012 Sponsors</B></P></FONT><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:British_Columbia/Attributions">http://2012.igem.org/Team:British_Columbia/Attributions</A>&quot;</DIV></FONT></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/Attributions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/Attributions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/Attributions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/Attributions&amp;oldid=267104" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>