<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_Pathway"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/Pathway">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/Pathway&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Pathway&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Pathway&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/Pathway" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/Pathway</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="#">UBC iGEM<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia">Home</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Attributions">Attributions</A></LI></UL><LI><A class="sf-with-ul" href="#">Project<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/ProjectConsortia">Microbial Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Desulfurization">Bio-desulfurization</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Protocols">Protocols</A></LI><LI><A class="sf-with-ul" href="#">Results<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Data">Data</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Results">Key Results</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Medal">Medal Criteria</A></LI><LI><A class="sf-with-ul" href="#">Models<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Consortia">Consortia</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Pathway">Pathway</A></LI><LI><A class="sf-with-ul" href="#">Human Practice<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP">IP &amp; iGEM</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/IP_FAQ">Patent Guide</A></LI><LI><A href="https://2012.igem.org/Team:British_Columbia/Human_Practices/Industry">Industrial Insight</A></LI></DIV><TITLE> British Columbia - 2012.igem.org </TITLE><P align="center"><FONT face="arial" narrow="" size="5"><B>Our Pathway Model
</B></FONT></P><FONT face="arial" narrow="">

The study of environmental genomics attempts to capture the taxonomic and functional diversity of natural microbial communities. Our host at UBC, the Hallam lab, designs novel tools for analyzing the gene content in the context of distributed metabolism. Recently, a pipeline has been developed for the automated construction and visualizing of metabolic pathways from genomic data by integrating software such as Pathway Tools, Pathologic and Metacyc [1]. This provided us an opportunity to model pathway compartmentalization and distribution amongst microbes in the natural environment as it applies to our project.  <H2><B>Summary of the pipeline for community-level metabolic analysis (Figure 1):</B></H2> First, open reading frames are predicted from sequence data with Prodigal, are then annotated by protein BLAST, and later summarized in a GenBank file. Pathway/genome databases (PGDBs) are generated from sequence data in a manner which does not constrain predictions within the scope of model organisms. This produces a community-based analysis that can be visualized using Pathway Tools. 

<H2><B>Modeling Metabolism:</B></H2><I>Rhodococcus erythropolis</I> and <I>Pseudomnas fluorescens</I> are often both prevalent in similar niches. With a high likelihood that these organisms encounter each other, studies have been conducted to assess their metabolic properties in co-culture.  In a study by Kayser et al, it was shown that the 4S biodesulferization pathway in <I>R. erythropolis</I> demonstrated higher activity the presence of <I>P. fluorescens</I> [2]. As <I>P. fluorescens</I> does not biodesulferize DBT and sulfate is known to repress the 4S pathway, we looked to analyze the genomes of the two organisms in the context of sulfur metabolism. 

The 4S pathway releases sulfite, which is toxic to the cell, and therefore genomes were analyzed with the aforementioned pipeline for pathways involved in metabolizing sulfite. It was found that both organisms have annotated genes which convert sulfite into sulfate via a reductase, however the organisms differ in downstream metabolism of sulfate. While both organisms contain a pathway for assimilatory sulfate metabolism, only <I>P. fluorescens</I> has the capability for dissimilatory sulfate metabolism (Figure 2, 3). Based on these findings, we can hypothesize that <I>R. erythropolis</I> and <I>P. fluorescens</I> likely excrete and catabolize any excess sulfate. This provides an explanation for the improved desulfurization found in co-culture conditions. <I>P. fluorescens</I> potentially removes sulfate from the environment, allowing increased secretion by <I>R. erythropolis</I>, and thereby derepressing 4S pathway. The prediction of distributed sulfite metabolism to improve biodesulferization in the environment provides both a testable hypothesis, as well as a grounds for improving biodesulferization through synthetic pathway distribution. 


We then scanned the literature for more evidence of shared metabolism between both <I>R. erythropolis</I> and <I>P. fluorescens</I> in order to further analyze gene content in the context of co-culture experiments.  In a study by Goswami et al., the metabolism of chlorinated aromatic compounds and phenol was compared in monoculture versus co-culture using <I>R. erythropolis</I> and <I>P. fluorescens</I> [3]. This study showed that the growth rate of pure cultures of <I>R. erythropolis</I> was higher than <I>P. fluorescens</I> on chlorinated aromatics, however in mixed culture, <I>P. fluorescens</I> showed a higher growth rate. For the degradation of phenol, <I>R. erythropolis</I> showed higher growth rates in both pure and mixed culture. The authors of this study suggested that these results were likely a product of substrate competition. We attempted to analyze the genomes of both <I>R. erythropolis</I> and <I>P. fluorescens</I>, separately and together in an attempt to offer an alternate interpretation of the co-culture results. The first pathways assessed were those involved in chlorinated aromatic degradation. It was found that <I>P. fluorescens</I> contains a higher diversity of genes involved in catabolizing chlorinated aromatics; however, only <I>R. erythropolis</I> seems to be able to degrade phenol (Figure 4, 5). This suggests the possibility of the compartmentalization of different components of these metabolic processes, leading to the different growth kinetics observed in co-culture. For example, while <I>R. erythropolis</I> may be more efficient at degrading certain chlorinated aromatics, in co-culture, the diversity of catabolism of chlorinated aromatics allows <I>P. fluorescens</I> to grow more rapidly. Chlorinated aromatic degradation by <I>P. fluorescens</I>, however, would result in the accumulation of downstream products, such as phenol, that only <I>R. erythropolis</I> can catabolism. This provides a metabolic network which could both select and sustain both microbes in the presence of diverse chlorinated aromatics. 




The sums of general aromatic degradation pathways were compared for the organisms genomes separately and together (Figure 6). This resulted in emergent predicted pathways in combination as well as combinatorial increases araomatic degradation potential. 
Ultimately, gene annotation-based models for distributed metabolism in the environment may help to engineer and optimize complex metabolism through synthetic consortia. <H2><B>References</B></H2>
[1] Hanson, N.W; Page, A.P; Konwar, K.M; Howes, C.G; Hallam, S.J. Metabolic interaction networks for the whole community, 2012. Unpublished.

[2] Kayser, K.J; Biolaga-Jones, B.A.; Jackowski, K; Odusan, O; Kildane, J.J. Utilzation of organosulfur compounds by anexic and mixed culture of <I>Rhodococcus rhodochrous</I> IGTS8, 1993. Journal of General Microbioology, 139: 3123-3129.

[3] Goswami, M; Shivaraman, N; Singh, R.P. Microbial metabolism of 2-chlorophenol, phenol and p-cresol by <I>Rhodococcus erythropolis</I> M1 and co-culture with <I>Pseudomonas fluorescens</I> P1, 2005. Microbiological Research, 160: 101-109.<DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:British_Columbia/Pathway">http://2012.igem.org/Team:British_Columbia/Pathway</A>&quot;</DIV></FONT></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/Pathway" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/Pathway" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/Pathway&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/Pathway&amp;oldid=266922" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>