<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-Denmark skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-Denmark</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="navbarPosition"><UL class="nav nav-tabs"><LI id="navbarIntroduction" class="active"><A data-toggle="tab" href="#menu1">Introduction</A></LI><LI id="navbarProject"><A data-toggle="tab" href="#menu2">Project &amp; Results</A></LI><LI id="navbarProcedures"><A data-toggle="tab" href="#menu3">Parts &amp; Procedures</A></LI><LI id="navbarPractices"><A data-toggle="tab" href="#menu4">Practices</A></LI><LI id="navbarProspects"><A data-toggle="tab" href="#menu5">Prospects</A></LI><LI id="navbarCredits"><A data-toggle="tab" href="#menu6">Credits</A></LI></UL><DIV class="tab-content"><DIV id="menu1" class="tab-pane fade in active"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#home"><H5>Home</H5></A><A class="nav-item nav-link" href="#introduction"><H5>Opening</H5></A><A class="nav-item nav-link" href="#achievements"><H5>Achievements</H5></A><A class="nav-item nav-link" href="#world-situation"><H5>World Situation</H5></A><A class="nav-item nav-link" href="#our-solution"><H5>Our Solution</H5></A></NAV></DIV><DIV id="menu2" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#project-and-results"><H5>Opening</H5></A><A class="nav-item nav-link" href="#project-design"><H5>Project Design</H5></A><A class="nav-item nav-link" href="#demonstration-and-results"><H5>Demonstration &amp; Results</H5></A></NAV></DIV><DIV id="menu3" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#parts-and-procedures"><H5>Opening</H5></A><A class="nav-item nav-link" href="#parts"><H5>Parts</H5></A><A class="nav-item nav-link" href="#notebook"><H5>Notebook</H5></A><A class="nav-item nav-link" href="#sops-and-protocols"><H5>SOPs &amp; Protocols</H5></A><A class="nav-item nav-link" href="#safety"><H5>Safety</H5></A></NAV></DIV><DIV id="menu4" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#practices"><H5>Opening</H5></A><A class="nav-item nav-link" href="#bioethics"><H5>Bioethics</H5></A><A class="nav-item nav-link" href="#integrated-practices"><H5>Integrated Practices</H5></A><A class="nav-item nav-link" href="#education-and-public-engagement"><H5>Education &amp; Public Engagement</H5></A></NAV></DIV><DIV id="menu5" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#prospects"><H5>Opening</H5></A><A class="nav-item nav-link" href="#perspectives"><H5>Perspectives</H5></A><A class="nav-item nav-link" href="#future-igem-teams"><H5>To Future iGEM Teams</H5></A></NAV></DIV><DIV id="menu6" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#credits"><H5>Opening</H5></A><A class="nav-item nav-link" href="#team"><H5>Team</H5></A><A class="nav-item nav-link" href="#collaborations"><H5>Collaborations</H5></A><A class="nav-item nav-link" href="#attributions"><H5>Attributions</H5></A><A class="nav-item nav-link" href="#final-words"><H5>Final Words</H5></A></NAV></DIV></DIV></DIV><DIV id="highlightToggle"><DIV id="editorsHighlights"><P><I>editors <SPAN class="highlighted">highlights</SPAN></I></P></DIV></DIV><DIV class="container-fluid" style="margin-bottom:-50px;"><DIV id="introductionContent"><DIV id="home"><DIV id="homePage"><DIV id="homePageText"><H1 class="hr"><SPAN id="PowerLeafH1">PowerLeaf</SPAN> - A Bacterial Solar Battery</H1><P>ENERGY MADE BEAUTIFUL</P></DIV></DIV><DIV class="row" id="abstract"><DIV class="col-md-5 col-sm-6 verticalAlignColumnsAbstract"><H3><SPAN class="highlighted">Abstract</SPAN></H3><P><SPAN class="highlighted">With the PowerLeaf, iGEM SDU is introducing a novel solution for long-term storage of solar energy, thus becoming an alternative to solar cells</SPAN>, without using environmentally harmful resources. We aim to accomplish this through the creation of a device visually shaped to resemble a plant leaf, thereby providing a nature-in-city ambience. The team invested heavily in public engagement and collaborations to investigate how the device hypothetically could be implemented into an urban environment. From a technical perspective, <SPAN class="highlighted">the bacterial solar battery is composed of an energy storing unit and an energy converting unit</SPAN>. The energy storing unit is defined by a genetically engineered <I>Escherichia coli</I>, that fixates carbon dioxide into the chemically stable polymer cellulose, while the energy converting unit uses genetically engineered <I> Escherichia coli</I> to consume the stored cellulose. Electrons retrieved from this process, are transferred to an anode by optimised nanowires, thereby creating an electrical current. Last but not least, the energy storing unit has a light-dependent system, which activates dormancy during nighttime to reduce energy lost by metabolism.
</P></DIV></DIV><DIV class="row" style="padding-top: 180px; margin-top:-110px; margin-bottom:110px;" id="igem-goes-green"><DIV class="col-xs-6 verticalAlignColumnsAbstract"><DIV style="text-align:center;"><P>A green project, <SPAN class="highlighted">a <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-our-green-wiki">green wiki</SPAN></SPAN>, and a great performance in the <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-igem-goes-green">iGEM Goes Green</SPAN> initiative! Green just got greener.</P></DIV></DIV></DIV><DIV class="modal fade" id="about-our-green-wiki" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">About Our Green Wiki</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>Our project is all about ensuring a greener and more sustainable future for ourselves and the coming generations. This of course meant, that our wiki had to to follow this purpose as well. CO<SUB>2</SUB> Neutral Website sponsored our wiki with a <A href="https://www.ingenco2.dk/crt/dispcust/c/4676/l/1
" target="_blank">CO<SUB>2</SUB> offset</A> equal to the amount of CO<SUB>2</SUB> produced by having the wiki running until October 31 2018. This does not mean the wiki is CO<SUB>2</SUB> neutral in itself, but that the offset, equal to its pollution, is compensated. Compensating an offset could for example be global initiatives like replacing stoves in Africa with energy efficient stoves and building new, renewable energy sources. Also companies will be instructed in reducing their CO<SUB>2</SUB> offset.</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="about-igem-goes-green" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">iGEM Goes Green</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><DIV style="text-align:center;"><P class="raleway P-Larger"><I>“It’s not easy being green”</I></P><P class="raleway"><I>Kermit the Frog</I></P></DIV><P class="P-Larger"><B>The Collaboration</B></P><P>As part of a Human Practices collaboration project the iGEM team of TU Dresden invited us to participate in iGEM Goes Green. The main idea of the collaboration is to calculate and consider the emission of CO<SUB>2</SUB> related to iGEM.

We are grateful for the opportunity to be a part in a collaboration which focuses on the green aspects of GMO, as it corresponds with a main focus of our project.</P><P class="P-Larger"><B>Our Carbon Footprint</B></P><P>Much like the coastline paradox presented by Mandelbrot, our total carbon footprint is hard to measure since we cannot take the following into account: Every single kilometer we have driven, the exact size of our trees and so forth.

The estimated carbon footprint of a flight from Copenhagen, Denmark, to Boston, Massachusetts, USA and back again is 3.430 kg/person 
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.atmosfair.de/en/kompensieren/flug?locale=en&amp;departure=CPH&amp;arrival=BOS ">Flight emission calculator </A></SPAN></SPAN>. With a team of 14 individuals, this culminates in a total of 48.020 kg or roughly 48 tonnes of carbon dioxide for our team - just from flying intercontinental. Added to this is the carbon footprint of the Danish trains at 30 g/km/person <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.dsb.dk/globalassets/arsrapport/2016/miljorapport-2016.pdf">DSB environmental report</A></SPAN></SPAN>.
 With the same 14 individuals and 175 km to Copenhagen from Odense this yields a total of 147 kg.
We have been on a total of three trips, one to Langeland, Denmark, two to Copenhagen, Denmark, and one to Delft, Holland. We have travelled to these destinations in two Volkswagen Transporter vans. The carbon footprint of these is 188 kg/km/car  <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://viewer.zmags.com/publication/4b44c1a8#/4b44c1a8/4">Volkswagen automobile details</A></SPAN></SPAN>.
  With a total of 2320 km travelled our trips yield a total of 844.5 kg of CO<SUB>2</SUB>.

Thus our total carbon footprint of travelling is roughly 49 tonnes of CO<SUB>2</SUB>.

Regarding our wiki, we have received help from our sponsors CO<SUB>2</SUB> Neutral Website in calculating the carbon footprint. They estimated roughly 10,000 views during the next year, but sponsored carbon elimination equivalent to 120,000 views. All in all the sponsorship from CO<SUB>2</SUB> Neutral Website has made our wiki net CO<SUB>2</SUB> negative.

We received a spreadsheet from TU-Dresden to estimate our carbon footprint from our lab <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://github.com/igem-dresden/GoGreenGuide/releases/tag/v1.0">Go Green Guide</A></SPAN></SPAN>. The carbon footprint calculated from our lab is estimated at a total of 1 tonne of CO<SUB>2</SUB>.

Thus our total carbon footprint is roughly 50 tonnes CO<SUB>2</SUB></P><P class="P-Larger"><B>Our Solution</B></P><P>To reduce our net carbon footprint we have made some local and one global initiative.
The local initiatives are planting trees, buying green and travelling green.
After thorough systematic research we could conclude that in general planting grass and most trees would increase our net carbon footprint, so we planted a few apple trees <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www3.epa.gov/climatechange/Downloads/method-calculating-carbon-sequestration-trees-urban-and-suburban-settings.pdf">Carbon sequestration of trees</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=The+effects+of+household+management+practices+on+the+global+warming+potential+of+urban+lawns">Global warming and lawns</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.researchgate.net/publication/237613871_Assessing_Soil_Carbon_Sequestration_in_Turfgrass_Systems_Using_Long-Term_Soil_Testing_Data">Carbon sequestration in turfgrass</A></SPAN></SPAN>.

Acknowledging this is not even cents on the dollar, we had to do more. Therefore some of the team changed their eating habits into a more local, organic and sustainable diet. The main focus of this diet was that a majority of the vegetables had to be local and should be picked up on nearby farms. For this reason one of the teammates signed up for “Odense Food Community”
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://translate.google.com/translate?hl=da&amp;sl=da&amp;tl=en&amp;u=http%3A%2F%2Fwww.odenseff.dk%2F">Odense Food Community (translated) </A></SPAN></SPAN>.

The last element of the local initiatives was to travel green. In general the team has been travelling by bike or public transport when going to and from the university. For the purpose of travelling to every meetup, we have been driving in vans, leaving very little carbon footprint.
Had we been able to go to Boston in a more eco-friendly way, we of course would.

Regarding global initiatives we started a partnership with CO<SUB>2</SUB> Neutral Website, which ended up in a sponsorship from them. What they have done in our name is</P><UL class="list"><LI>Build energy efficient stoves for families in Africa, reducing CO<SUB>2</SUB> emissions as less trees are logged. It also creates local jobs and creates a healthier life. Certified and audited through the Gold Standard.</LI><LI>Build boreholes for clean water instead of families boiling water. This also improves quality of life and reduces CO<SUB>2</SUB> emission. Certified and audited through the Gold Standard.</LI><LI>Build new, renewable energy sources. So far we are involved in several windmill parks.</LI><LI>Help protect rainforest from devastating deforestation. Certified and audited through the Redd+.</LI><LI>Help companies reduce their climate impact through energy efficiency.</LI></UL><P>Thus we have had a global impact on climate, which cannot be estimated precisely.</P><P class="P-Larger"><B>Conclusion</B></P><P>In conclusion we would like to highlight the following</P><UL class="list"><LI>Due to flying being a necessity for us when going to Boston we have a massive carbon footprint.</LI><LI>We acknowledge that any local initiative we have made are pennies on the dollar in regards to CO<SUB>2</SUB> emission and reduction.</LI><LI>The main initiative from us is the partnership with CO<SUB>2</SUB> Neutral Website, which has had a global impact.</LI><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV><DIV id="introduction"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlineIntroduction" class="headline"><SPAN class="highlighted">Introduction</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><P><SPAN class="largeFirstLetter">W</SPAN>elcome to our wiki! <SPAN class="highlighted">We are the University of Southern Denmark's iGEM team</SPAN> and we have been waiting in great anticipation for the chance to tell you our story. 

Our adventure began with a meeting between strangers from eight different studies. Despite our different backgrounds, we had one thing in common; a shared interest in synthetic biology. Soon after this first meeting, we were herded off to a weekend in a cottage - far away from our regular lives. The cottage was a place to bond and discuss project ideas. It immediately became apparent that <SPAN class="highlighted">being an interdisciplinary team was going to be our strength</SPAN> as each member had unique qualities that enabled them to efficiently tackle different aspects of the iGEM competition. So, we made it our goal to take advantage of these qualities.

We decided to make a proof-of-concept project. Specifically, we wanted to use <SPAN class="highlighted">bacteria as a novel and greener solution for solar energy storage</SPAN>. This project was later dubbed the PowerLeaf – A Bacterial Solar Battery. 

Since it is a one-page wiki, you can <SPAN class="highlighted">just keep on scrolling, and you will be taken on a journey through our iGEM experience</SPAN>.
</P></DIV></DIV></DIV><DIV class="topShadow" id="achievements"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-top-50"><DIV class="row border-left-achievements"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Achievements</SPAN></H2><P>At the Giant Jamboree 2017 we succeeded in getting a Gold Medal and were nominated 'Best Energy Project'.</P></DIV><DIV class="row margin-top-200"><DIV class="col-xs-8 verticalAlignColumns padding0 left right"><P class="P-Larger"><B><SPAN class="highlighted">Bronze Medal Requirements</SPAN></B></P><P><B>Register and Attend</B> – Our team applied on the 30<SUP>th</SUP> of March 2017 and got accepted the 4<SUP>th</SUP> of May 2017. We had an amazing summer and are looking forward to attending the Giant Jamboree!<B>Meet all the Deliverables Requirements</B> –  You are reading the team wiki now, so that is one cat in the bag. You can find all attributions made to the project in the <A href="https://2017.igem.org/Team:SDU-Denmark#attributions" target="_blank">Credits section</A> of the wiki. The team poster and team presentation are ready to be presented at the Giant Jamboree. We also filled the <A href="https://2017.igem.org/Safety/Final_Safety_Form?team_id=2449" target="_blank">safety form</A>, the <A href="https://igem.org/2017_Judging_Form?id=2449" target="_blank">judging form</A> and all our <A href="http://parts.igem.org/cgi/dna_transfer/batch_list.cgi?group_id=2951" target="_blank">parts</A> were registered and submitted.<B>Clearly State the Attributions</B> – All attributions made to our project have been clearly credited in the <A href="https://2017.igem.org/Team:SDU-Denmark#attributions" target="_blank">Credits section</A>.<B>Improve and/or Characterise an Existing Biobrick Part or Device</B> – The characterisation of the OmpR-regulated promoter <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A> was improved, as the level of noise was studied on different vectors. 
Induction and inhibition of the pBAD promoter, <A href="http://parts.igem.org/Part:BBa_I0500" target="_blank">BBa_I0500</A>, were studied, whereby the characterisation of this part was improved. 
Furthermore, we characterized if the periplasmic beta-glucosidase could make <I>E. coli</I> live on cellobiose in fluid medium <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank">BBa_K523014</A>, submitted by the <A href="https://2011.igem.org/Team:Edinburgh" target="_blank">2011 iGEM Edinburgh Team</A>. 
The data obtained in these experiments are presented in the <A href="https://2017.igem.org/Team:SDU-Denmark#demonstration-and-results" target="_blank">demonstration and results section</A>.</P></DIV></DIV></DIV><DIV class="row border-left-invsisble"><DIV class="col-xs-8 verticalAlignColumns padding0 left"><P class="P-Larger"><B><SPAN class="highlighted">Silver Medal Requirements</SPAN></B></P><P><B>Validated Part/Contribution</B> –  We created the part <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank">BBa_K2449004</A>, containing a cellobiose phosphorylase, Cep94A. This enzyme enables <I>Escherichia coli</I> to survive on cellobiose, which we validated by growth experiments. The data obtained in these experiments are presented in the <A href="https://2017.igem.org/Team:SDU-Denmark#demonstration-and-results" target="_blank">Demonstration &amp; Results section</A>.<B>Collaboration</B> – We have collaborated with several teams throughout our project by taking part in discussions, meetups, and answering questionnaires - we even hosted our first meetup for our fellow Danish iGEM teams. You can read all about this in the <A href="https://2017.igem.org/Team:SDU-Denmark#collaborations" target="_blank">Credits section</A>.<B>Human Practices</B> – Our philosopher, historian, and biologist have discussed the <A href="https://2017.igem.org/Team:SDU-Denmark#bioethics" target="_blank">ethical and educational aspects</A> of our project in great detail. In extension to their work, we have been working extensively with <A href="https://2017.igem.org/Team:SDU-Denmark#education-and-public-engagement" target="_blank">education and public engagement </A>.</P></DIV></DIV><DIV class="row border-left-invsisble margin-bottom-200"><DIV class="col-xs-8 verticalAlignColumns padding0 left right"><P class="P-Larger"><B><SPAN class="highlighted">Gold Medal Requirements</SPAN></B></P><P><B>Integrated Human Practices</B> – For the <A href="https://2017.igem.org/Team:SDU-Denmark#integrated-practices" target="_blank">development and implementation</A> of the device, we reached out to and remained in contact with city planners from our hometown throughout our project. This regarded advice and conversations on anything from the possible design, value, safety, use, placement, and plastic type of our device. The ideas generated from these conversations, were integrated in our overall project. Last but not least, we focused on demonstrating this process on our wiki in order to inspire future iGEM teams. <B>Model Your Project</B> –  Through extensive <A href="https://2017.igem.org/Team:SDU-Denmark#modelling" target="_blank">modelling</A>, we have learned that it is possible to regulate bacterial dormancy. However, the modelling showed that it would be inadequate to only regulate the toxin RelE, as this would make the bacteria unable to exit dormancy. To regulate dormancy properly, would also require tight regulation of the antitoxin RelB. This information was used to shape the entire approach of the light-dependent dormancy system.</P></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="world-situation"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-introduction"><DIV class="col-xs-12"><H2><SPAN class="highlighted">World Situation</SPAN></H2></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>A Global Challenge</B></SPAN></P><P>In the world of today, <SPAN class="highlighted">it is becoming increasingly important to ensure a sustainable future</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="hhttp://wwwoecdorg/greengrowth/MATERIAL%20RESOURCES,%20PRODUCTIVITY%20AND%20THE%20ENVIRONMENT_key%20findingspdf"> Green Growth Papers (Myriam Linster). Material Resources, Productivity and the Environment. 2013.</A></SPAN></SPAN>. Not just for our generation, but especially for the generations to come, as their possibilities should not be limited by our choices. 
Our solution, is the development of a green and renewable technology, which offers new advantages to the field of sustainable energy. <SPAN class="highlighted">There are currently certain limitations to the existing options for renewable energy</SPAN>, namely the intermittency and the diluteness problem <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.researchgate.net/publication/279212503_Global_Lithium_Resources_and_Sustainability_Issues"> Alexandre Chagnes JS. Global Lithium Resources and Sustainability Issues.  Lithium Process Chemistry: Elsevier; June 2015. p. pp.1-40.</A></SPAN></SPAN>. The intermittency problem describes the discontinuous energy production, along with inefficient storage. On the other hand, the diluteness problem is characterised as the resource-demanding production of technical devices, such as solar cells and batteries. This means that a lack of resources eventually could eliminate some of the  current forms of green technology. As such, we need to <SPAN class="highlighted">introduce a new and sustainable approach to green energy</SPAN> to ensure the continuation of our beautiful world for the coming generations.
    </P><P class="P-Larger"><SPAN class="highlighted"><B>In a Local Environment</B></SPAN></P><P>We are a team of young adults raised to be aware of climate changes and the potential limitations to our ways of life. As a generation that appreciates open source and shared information, we have been encouraged to constantly challenge the ideas of yesterday. With this in mind, <SPAN class="highlighted">we decided the best solution to the eventual energy crisis would be to seek out experts and the general public, even children, in order to rethink the current notion</SPAN> that the only way to save our planet, is to compromise our living standards.

Fortunately, we learned through interaction with local agents that a great deal of people share our belief: that <SPAN class="highlighted">we ought to pursue the development of low energy cities with a high quality of life</SPAN>. In fact, we even discovered that our own hometown Odense wants to be the greenest, most renewable city in Denmark by 2050 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.odense.dk/borger/miljoe-og-affald/klima">Odense Municipality’s website, regarding their politics on  the current climate changes.</A></SPAN></SPAN>. 

In the pursuit of this goal <SPAN class="highlighted">we rose to the challenge of creating a truly green solution</SPAN>, which would provide an environmental friendly source of energy.
<SPAN class="highlighted">Please keep scrolling if you wish to read more about our solution</SPAN>, or go straight to <A href="https://2017.igem.org/Team:SDU-Denmark#bioethics" target="_blank">bioethics</A> if you are curious why we not only <I>could</I>, but <I>ought</I> to do something about the current and forthcoming energy crisis.
    </P><DIV id="inspiration-from-children"><P class="P-Larger"><SPAN class="highlighted"><B>Inspiration</B></SPAN></P><P>Our early ideas were reviewed after attending the Danish Science Festival, where we met several young minds with creative and inspiring ideas. The children came to our workshop with their parents to learn about bacteria, the history of GMO, ethics, and iGEM. They attended our “Draw-a-Bacteria”-competition, where they designed their own bacteria, some even with detailed stories. From this <SPAN class="highlighted">the children taught us a thing or two about the endless possibilities of GMO</SPAN>.</P><P style="font-size:12px;"><I>See a selection of their amazing drawings <A href="https://static.igem.org/mediawiki/2017/3/30/T--SDU-Denmark--danish-science-festival-drawings.pdf" target="_blank">here</A>.</I></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="our-solution"><DIV class="row anchorMargin presentationBackground"><DIV class="row"><DIV class="col-lg-8 col-md-10"><DIV class="row border-left-introduction-our-solution"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Our Solution</SPAN></H2><DIV style="text-align:center;"><P class="raleway P-Larger"><I>“Well, if it can be thought, it can be done, a problem can be overcome”</I></P><P class="raleway" style="text-align:center;"><I>E.A. Bucchianeri, Brushstrokes of a Gadfly</I></P></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P><SPAN class="highlighted">The vision for our bacterial solar battery is to combine two aspects, energy storage and energy conversion, by which we will produce a new and improved type of solar battery. We have named this vision The PowerLeaf</SPAN>. The PowerLeaf consist of two chambers that will be referred to as <I>the outer chamber or energy storing unit</I> and <I>the inner chamber or energy converting unit</I>.</P><UL class="list" style="margin-top:15px;"><LI><SPAN class="highlighted">The energy storing unit is comprised of genetically engineered <I>Escherichia coli</I> (<I>E. coli</I>), which  uses solar energy for ATP production to fixate carbon dioxide into the chemically stable polymer cellulose. <SPAN class="highlighted"> The cellulose works as the battery</SPAN> in the PowerLeaf, storing the chemical energy. A light sensing system activates dormancy during nighttime, leading to a reduced loss of energy through metabolism.</SPAN></LI><LI><SPAN class="highlighted">The energy converting unit uses genetically engineered <I>E. coli</I> to consume the stored cellulose by using an inducible switch. Retrieved electrons are transferred by extracellular electron carriers to an anode, resulting in an electrical current.</SPAN></LI></UL><P><SPAN class="highlighted">The complete system will be combined into a single device containing a compartment for each of the two units</SPAN>. Details about the construction and device will be discussed in the <A href="https://2017.igem.org/Team:SDU-Denmark#integrated-practices" target="_blank">Integrated Practices section</A>.
<SPAN class="highlighted">The device was originally designed to resemble a plant leaf aimed to provide a nature-in-city ambience</SPAN>. This hypothetical implementation of the PowerLeaf in an urban environment was developed through careful consideration, public engagement, and collaborations. We worked with local city planners from our hometown Odense, as well as with a plastic specialist from SP Moulding, the purpose of which was to advance our pre-established design, as well as attaining other changeable designs.

Our vision was clear and ambitions were high. As it turned out though, we had aimed too high, considering the limited timeframe, so at an early stage, <SPAN class="highlighted">we decided to focus on the following features:</SPAN></P><OL class="list"><LI><SPAN class="highlighted">Light-dependent dormancy system</SPAN></LI><LI><SPAN class="highlighted">Converting CO<SUB>2</SUB> into glucose</SPAN></LI><LI><SPAN class="highlighted">Biosynthesis and secretion of cellulose produced from glucose</SPAN></LI><LI><SPAN class="highlighted">Converting cellulose to glucose</SPAN></LI><LI><SPAN class="highlighted">Extracellular electron transfer</SPAN></LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><P>
It will then be up to future iGEM teams to continue on the development of the PowerLeaf. We would love to see our project become a reality one day, and so we have <SPAN class="highlighted">created a special page <A href="#future-igem-teams" target="_blank">for future iGEM teams</A>, which includes suggestions for a further development of the project</SPAN>.
</P></DIV></DIV></DIV></DIV><DIV id="projectContent"><DIV id="project-and-results"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlineProject" class="headline"><SPAN class="highlighted">Project &amp; Results</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><P><SPAN class="highlighted">Our device is composed of two units, an energy storing unit and an energy converting unit</SPAN>, each divided into systems, all of which have been given a symbol to help you navigate throughout the wiki.</P><P style="margin-bottom:0;"><SPAN class="highlighted"><B>Energy Storing Unit</B></SPAN></P><DIV><DIV class="verticalAlignColumnsAbstract"><P><SPAN class="highlighted">Dormancy System</SPAN></P></DIV></DIV><DIV><DIV class="verticalAlignColumnsAbstract"><P><SPAN class="highlighted">Carbon Fixation</SPAN></P></DIV></DIV><DIV><DIV class="verticalAlignColumnsAbstract"><P><SPAN class="highlighted">Cellulose Biosynthesis</SPAN></P></DIV></DIV><P style="margin-bottom:0;"><SPAN class="highlighted"><B>Energy Converting Unit</B></SPAN></P><DIV><DIV class="verticalAlignColumnsAbstract"><P><SPAN class="highlighted">Breakdown of Cellulose</SPAN></P></DIV></DIV><DIV><DIV class="verticalAlignColumnsAbstract"><P><SPAN class="highlighted">Extracellular Electron Transfer</SPAN></P></DIV></DIV><P>In the <A href="https://2017.igem.org/Team:SDU-Denmark#project-design" target="_blank">Project Design section</A>, you will first be given a short introduction to the background, followed by the approach of that system, before you move on to the next system. Once you reach the next section of the wiki, <A href="https://2017.igem.org/Team:SDU-Denmark#demonstration-and-results" target="_blank">Demonstration &amp; Results</A>, you will be guided through the performed experiments and the derived conclusions. To make things easier for you, we have continued to use the above symbols throughout our wiki.
</P></DIV></DIV></DIV><DIV class="topShadow" id="project-design"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-project"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Project Design</SPAN></H2></DIV></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-design-dormancy-system" style="margin-top:-125px;"><DIV class="col-xs-12"><DIV class=""><DIV class="project-design-headline"><H2>Dormancy System</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B>Introduction</B></P><P>Cyanobacteria contain signal transduction systems, thereby making them capable of <SPAN class="highlighted">sensing and responding to light</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3732953/">Bussell AN, Kehoe DM. Control of a four-color sensing photoreceptor by a two-color sensing photoreceptor reveals complex light regulation in cyanobacteria. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(31):12834-9.</A></SPAN></SPAN>. This ability gives the organisms the opportunity to <SPAN class="highlighted">adapt and optimize their metabolism to a circadian rhythm</SPAN>. Photoreceptors in the plasma membrane, of which phytochromes are especially abundant and well described, are responsible for this property <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11145881">Vierstra RD, Davis SJ. Bacteriophytochromes: new tools for understanding phytochrome signal transduction. Seminars in cell &amp; developmental biology. 2000;11(6):511-21.</A></SPAN></SPAN>. In 2004, the <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM team</A> made a light response system consisting of a photoreceptor combined with an intracellular indigenous regulator system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16306980">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. EnvZ and OmpR make up the two-component system naturally found in <I>E. coli</I>. The photoreceptor known as Cph1 was isolated from the cyanobacteria <I>Synechocytis</I> PCC6803. Cph1 has functional combination sites, which combined with the kinase EnvZ form a two-domain receptor, known as Cph8. Activation of Cph8 is mediated by the chromophore phycocyanobilin, PCB, that is <SPAN class="highlighted">sensitive to red light</SPAN> with maximal absorbance at 662 nm <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11532008">Lamparter T, Esteban B, Hughes J. Phytochrome Cph1 from the cyanobacterium Synechocystis PCC6803. Purification, assembly, and quaternary structure. European journal of biochemistry. 2001;268(17):4720-30.</A></SPAN></SPAN>.
<SPAN class="highlighted">When not exposed to light</SPAN>, PCB activates the phytochrome Cph1, thus promoting kinase activity through the EnvZ kinase. When the transcription factor OmpR is phosphorylated by EnvZ, <SPAN class="highlighted">expression of genes regulated by the OmpR-regulated promoter is initiated. Excitation of PCB by red light</SPAN> results in a situation where the transcription factor OmpR is not regulated. The absence of phosphorylated OmpR leads to no activation of the OmpR-regulated promoter, thereby <SPAN class="highlighted">preventing gene expression</SPAN>.
</P><DIV class="figure-text"><P><B>Figure 1.</B> Left: Red light activates PCB, which in turn inactivates the photoreceptor complex Cph8, preventing gene expression from the OmpR-regulated promoter. Right: In absence of light, PCB is inactive, which enables the Cph8 to phosphorylate the transcription factor OmpR. This promotes gene expression from the OmpR-regulated promoter.</P></DIV><P>
The <SPAN class="highlighted">photocontrol device can be used to regulate a toxin-antitoxin system</SPAN>, enabling the implementation of a light-dependent dormancy system. A toxin-antitoxin system is composed of two gene products, a cytotoxin and an antitoxin, the latter which neutralises the the toxic effect caused by the toxin.
 In <I>E. coli</I> K-12 the cytotoxin RelE and antitoxin RelB comprise such a system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/9767574">Gotfredsen M, Gerdes K. The Escherichia coli relBE genes belong to a new toxin-antitoxin gene family. Molecular microbiology. 1998;29(4):1065-76.</A></SPAN></SPAN>. Expression of the <SPAN class="highlighted">cytotoxin RelE inhibits translation in the cells</SPAN>, due to its ability to cleave mRNA found in the A-site of the ribosome. <SPAN class="highlighted">RelB neutralises the toxic effect of RelE</SPAN> through interaction between the two proteins. Whether the cell lies dormant in response to expression of RelE depends on the ratio of antitoxin RelB and RelE present in the cell. Several studies have shown that RelB and RelE form a complex with RelB:RelE stoichiometry of 2:1 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. When the RelB:RelE stoichiometric-ratio is lowered to 1:1, studies show that RelB is not able to protect the cells against the RelE-caused translational inhibition <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. For further information about the light-dependent dormancy system, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#light-sensing-system-theory">read here</SPAN>.
</P><DIV class="modal fade" id="light-sensing-system-theory" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Light-Dependent Dormancy System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><B>Dormancy Optimises the Efficiency of the Bacterial Solar Battery</B>
Photosynthetic bacteria draw energy from sunlight to drive the fixation of carbon, implying that the bacteria are not able to carry out carbon fixation in absence of light. However, the bacteria constantly metabolise and the carbon fixed during time of exposure to light, will be used as an energy source for the cells. To circumvent this, a photocontrol system created by the <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM 2004 team</A> was used in combination with the RelE-RelB toxin-antitoxin system native to <I>E. coli</I>. In this way, a light-dependent dormancy system was implemented in <I>E. coli</I>.<B>The Photocontrol Device Mediates Light-Dependent Gene Expression</B>
Plants and several photosynthetic microorganisms, such as cyanobacteria, contain signal transduction systems, which makes them capable of reacting to light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3732953/">Bussell AN, Kehoe DM. Control of a four-color sensing photoreceptor by a two-color sensing photoreceptor reveals complex light regulation in cyanobacteria. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(31):12834-9.</A></SPAN></SPAN>. This ability gives the organisms the opportunity to adapt and optimise the regulation of their metabolic rate in response to sunlight. This property is achieved by photoreceptors incorporated in their plasma membrane, of which phytochromes are the most abundant and well described <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11145881">Vierstra RD, Davis SJ. Bacteriophytochromes: new tools for understanding phytochrome signal transduction. Seminars in cell &amp; developmental biology. 2000;11(6):511-21.</A></SPAN></SPAN>.

Several two-component signal transduction systems evolved in <I>E. coli</I> enables it to respond to various external conditions, such as osmotic stress, lack of metabolites and other external stress factors. Nothing indicates that light initiates such a two-component signal transduction pathway in wild type <I>E. coli</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.microbiologyresearch.org/docserver/fulltext/micro/22/1/mic-22-1-113.pdf?expires=1507966841&amp;id=id&amp;accname=guest&amp;checksum=574A5913441399B962AA6A4F887C733E">Alper T, Gillies NE. The relationship between growth and survival after irradiation of Escherichia coli strain B and two resistant mutants. Journal of general microbiology. 1960;22:113-28.</A></SPAN></SPAN>. The <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM 2004 team</A> applied the light sensing property of phototrophs to an <I>E. coli</I>. By aligning different phytochromes with the intrinsic kinase EnvZ from <I>E. coli</I> they revealed a way to create a two-component system consisting of a photoreceptor with an intracellular indigenous regulator system found in <I>E. coli</I>. By establishing this system the bacteria acquired the ability to respond to red light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16306980">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. The photoreceptor from phytochrome known as Cph1 was isolated from the cyanobacteria <I>Synechocytis</I> PCC6803. Cph1 has a fusion site, which can be used to combine it with the kinase EnvZ, from the EnvZ-OmpR kinase-regulator system, to form a two-domain receptor known as Cph8. The chromophore phycocyanobilin (PCB) absorbs light in the red region with maximal absorbance at 662 nm <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11532008">Lamparter T, Esteban B, Hughes J. Phytochrome Cph1 from the cyanobacterium Synechocystis PCC6803. Purification, assembly, and quaternary structure. European journal of biochemistry. 2001;268(17):4720-30.</A></SPAN></SPAN>. When heterogeneously expressed in <I>E. coli</I>, it can, in combination with the light receptor Cph8, be used to form a light-sensitive circuit, making <I>E. coli</I> able to respond to red light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053042/">abor JJ, Levskaya A, Voigt CA. Multichromatic control of gene expression in Escherichia coli. Journal of molecular biology. 2011;405(2):315-324. </A></SPAN></SPAN>. 

In situations where no red light is present, the photoreceptor PCB activates the phytochrome Cph1, thereby promoting kinase activity through the EnvZ kinase, illustrated in figure 1. When the transcription factor OmpR is phosphorylated by EnvZ, expression of genes controlled by the OmpR-regulated promoter is initiated. Excitation of the PCB by red light, results in a situation, where EnvZ will not be able to phosphorylate the transcription factor OmpR. The lack of phosphorylated OmpR leads to no activation of the OmpR-regulated promoter, thereby preventing gene expression by this promoter.<B>RelE and RelB Comprise a Toxin-Antitoxin System in <I>E. coli</I></B>
A toxin-antitoxin system is a system composed of two gene products, of which one specifies a cell toxin and the other an antitoxin, which neutralises the toxic effect caused by the toxin. In <I>E. coli</I> K-12 the cytotoxin RelE and antitoxin RelB comprise such a system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/9767574">Gotfredsen M, Gerdes K. The Escherichia coli relBE genes belong to a new toxin-antitoxin gene family. Molecular microbiology. 1998;29(4):1065-76.</A></SPAN></SPAN>. Expression of the cytotoxin RelE inhibits translation in the cells, due to its ability to cleave mRNA found in the A-site of the ribosome <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12526800">Pedersen K, Zavialov AV, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112(1):131-40.</A></SPAN></SPAN>. RelB neutralise the toxic effect of RelE through interaction between the two proteins. In situations of amino acid starvation, it is appropriate for the bacteria to halt the translation in order to avoid errors owing to absent amino acids. Consequently, one of the exciting factors for the expression of RelE is conditioned by amino acid starvation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12526800">Pedersen K, Zavialov AV, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112(1):131-40.</A></SPAN></SPAN>. 

Whether the cell lie dormant in response to expression of RelE depends on the ratio of RelB and RelE present in the cell. Several studies have shown that RelB RelE form a complex with RelB:RelE stoichiometry of 2:1 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>, When the RelB:RelE stoichiometric-ratio is lowered to 1:1, studies show that RelB is not able to protect the cells against the RelE-caused translational inhibition <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. To prevent free RelE circulating and discharging toxic effects in the cells under favorable conditions, studies in vivo have shown that RelB is present in 10x higher concentrations than RelE <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN>. The heterologous induction of RelE could cause dissonance in the RelB:RelE ratio leading to serious consequences for the cells <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11717402">Christensen SK, Mikkelsen M, Pedersen K, Gerdes K. RelE, a global inhibitor of translation, is activated during nutritional stress. Proceedings of the National Academy of Sciences of the United States of America. 2001;98(25):14328-33.</A></SPAN></SPAN>. The bacteria are not killed when RelE is present in abundance, but high expression of the RelE gene makes awakening of the bacterial cells a challenge <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294780/">Tashiro Y, Kawata K, Taniuchi A, Kakinuma K, May T, Okabe S. RelE-Mediated Dormancy Is Enhanced at High Cell Density in Escherichia coli. J Bacteriol. 2012;194(5):1169-76.</A></SPAN></SPAN>. Hence, introducing a toxin to cells in a successful manner constitutes a challenge. 

</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV id="modelling"><P class="P-Larger"><B>Modelling</B></P><P><B>Modelling of the RelE-RelB System is Essential to Avoid Irrevocable Dormancy</B></P><P>Controllable dormancy is a feature that holds the potential to be applied in many different situations. However, inducing dormancy and bringing the bacteria back to a metabolic active state is like balancing on a tightrope, and to establish the basis of future implementations, the properties of this system would have to be investigated further. In an endeavour to provide this basic knowledge, <SPAN class="highlighted">stochastic modelling utilising the <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#gillespie-algorithm"> Gillespie algorithm </SPAN> was performed in an attempt to prognosticate the system</SPAN> and simulate the interactions between the toxin and antitoxin. 
<SPAN class="highlighted">The toxin RelE is inhibited by the antitoxin RelB through complex formation</SPAN>, and both proteins interact with their promoter in a feedback mechanism. 
To consolidate the model, the capacity of the toxin-antitoxin system was assessed in an experiment, as the controllability of the dormancy system was studied through manual  regulation of RelE and RelB expression.
You can read more about the modelling <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#toxin-antitoxin-system">here</SPAN>. </P><DIV class="figure-text"><P><B>Figure 2.</B><B>Left</B>: The time required for the bacteria to enter dormancy varies with the expression level of RelB. The percentage of dormant bacteria, defined as containing RelE amounts above 40 molecules per cell as a function of time in minutes. <B>Right</B>: Only one of the tested configurations, RelB<SUB>2</SUB>:50-RelE:35, causes the bacteria to regain their activity within the modelled time. The percentage of dormant bacteria, defined as containing RelE amounts above 15 molecules per cell as a function of time in minutes. The data is based on the simulation of 1000 independent bacteria.</P></DIV><P>
The simulated data revealed, that when enhanced RelE production is implemented, in order to induce dormancy in <I>E. coli</I>, the effect come easily. However, <SPAN class="highlighted"> implementation of RelB expression is also found necessary</SPAN> to ensure that the bacteria are able to enter an active state again. 

The model showed that <SPAN class="highlighted">the system is sensitive to the RelE:RelB ratio</SPAN>, as well as the total amount of produced toxin. As seen in Figure 2, implementation with production rates in the vicinity of <SPAN class="highlighted">50 and 35 molecules per minute for RelB and RelE respectively, was found to be suitable for balancing our system</SPAN>; the bacteria lay dormant within the computed time and re-enter an active state within minutes. 

The simulated data made it evident that <SPAN class="highlighted">implementing an optimised dormancy system comprises a challenge</SPAN>, as the individual expression levels of RelE and RelB, as well as their interaction, has a crucial impact on the regulation of dormancy. Thus, controlled gene expression of both RelE and RelB is required to implement a controllable dormancy system in the PowerLeaf. 

If you want to dig deeper into this crucial modelling of the dormancy system, read the full results <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#model-results">here</SPAN>. </P></DIV><DIV class="modal fade" id="gillespie-algorithm" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Gillespie Algorithm</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>
The Gillespie algorithm is a way to calculate the evolution of stochastic functions; in this case cell concentrations. To use the algorithm, two things are required:</P><OL class="list"><LI>Reaction rates of the system at a given configuration.</LI><LI>A random number generator.</LI></OL><P>
For each time step, two things are calculated using the random number generator: </P><OL class="list"><LI>The time before next reaction.</LI><LI>Which reaction occurs.</LI></OL><P>
The time before next step is given by:
</P><DIV style="text-align:center;"><P>Δt=S<SUP>-1</SUP>log(r<SUB>1</SUB><SUP>-1</SUP>)</P></DIV><P>
Where S is the sum of the reaction rates and r<SUB>1</SUB> is a random number between 0 and 1. This gives the time, as if the system was one reaction with reaction rate S, using the random number to give an exponential distribution. 

The reaction is chosen proportionally to each individual reaction rate using another random number, where reactions with high rates will occur most frequently. 

As each reaction is carried out, the new time is the sum of the previous and added reaction times. 

These calculations are carried out until the time reaches the wanted limit, or a specific number of reactions has occurred. It is necessary to have a limit on the number of reactions, as it elsewise is possible for the calculations to continue indefinitely.

</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="toxin-antitoxin-system" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Stochastic Modelling of the RelE-RelB Dormancy System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>RelE and RelB Regulate Dormancy and Influence Their Own Expression</B></P><P>
Modelling of the effects of different RelE and RelB expression levels were performed as an important aspect in the implementation of the RelE-RelB toxin-antitoxin system. The toxin RelE constrains bacterial growth by mRNA degradation, thereby inhibiting translation, whereas the antitoxin RelB inhibits this toxic effect by forming complexes with RelE. As seen in Figure 1, three different protein complexes are formed, namely RelB<SUB>2</SUB>, RelB<SUB>2</SUB>RelE, and RelB<SUB>2</SUB>RelE<SUB>2</SUB>, containing zero, one, and two RelE molecules respectively <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://doi.org/10.1016/j.jmb.2008.04.039.">Guang-Yao Li, Yonglong Zhang, Masayori Inouye, Mitsuhiko Ikura, Structural Mechanism of Transcriptional Autorepression of the Escherichia coli RelB/RelE Antitoxin/Toxin Module, In Journal of Molecular Biology, Volume 380, Issue 1, 2008, Pages 107-119, ISSN 0022-2836</A></SPAN></SPAN>.
</P><DIV class="figure-text"><P><B>Figure 1.</B> The three toxin-antitoxin complexes RelB<SUB>2</SUB>, RelB<SUB>2</SUB>RelE, and RelB<SUB>2</SUB>RelE<SUB>2</SUB>.</P></DIV><P>The expression of both RelE and RelB is regulated by the <I>relBE</I> promoter, which is influenced differently by each of the complexes, as seen in Figure 2. When small amounts of RelE is present, RelB<SUB>2</SUB> and RelB<SUB>2</SUB>RelE repress transcription through <I>relBE</I> by binding to the operator sequence. However, when high amounts of RelE are present, the toxin mitigates this repression by reacting with complexes bound to the operator sequence <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>.
</P><DIV class="figure-text"><P><B>Figure 2.</B> The interactions between the toxin-antitoxin complexes and the relBE promoter controlling the expression of RelE and RelB. RelE mediates the degradation of mRNA, thereby inhibiting translation.</P></DIV><P>During starvation, the half-life of RelB decreases significantly due to a Lon-protease<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>, causing a shift in the equilibrium of RelB and RelE to a higher level of RelE. In a non-starvation situation, the interactions with the operator sequence keeps the amount of free RelE at a low level, thereby stabilising the system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. In our simulation, the shift in equilibrium is made by introducing additional expression of RelE. 
Two different models were used with two different approaches:
</P><OL class="list"><LI>How a given configuration of RelB and RelE production increases the RelE concentration and whether it could induce dormancy within 2 hours.</LI><LI>The time required for the bacteria to exit dormancy for each of the configurations, that is, how long it takes the levels of free RelE to decrease again. </LI></OL><P class="P-Larger"><B>Rates and Reactions</B></P><P>The skeleton of toxin-antitoxin system inherent to <I>E. coli</I> in the model was based on the study by Cataudella et al. 2012 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. In an <I>E. coli</I> cell with a size of 1-2 μm, each nM of concentration can be approximated to 1 molecule. Thus all units are converted to molecules<SUP>-1</SUP>, as this fits the premises of the Gillespie algorithm. To simplify the model, the high affinity of RelE and RelB was used to ignore single RelB and only consider the dimer, RelB<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.uniprot.org/uniprot/P0C079">UniProtKB - P0C079 (RELB_ECOLI)</A></SPAN></SPAN>. Thus, all mentions of RelB in the model refers to its dimer. 

Whilst RelB has a relatively low half-life at about 3-5 minutes<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>, RelE is rather stable and its half-life, here set to 43 min, is primarily an effect from dilution caused by the bacterial growth<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. However, when growth is restricted during dormancy, the half-life of RelE is increased to 2000 min, corresponding to approximately one day, as the dilution effect is no longer applicable. As the protein complexes are relatively stable, their half-life was set to the same as RelE. However, for RelE to become active in the inherent system under starvation, RelB in complexes must decay<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>, for which reason the rate was set to a fourth of free RelB.


The transcription rates of RelE and RelB are based on the concentration of RelE and RelB under stable conditions where RelB is ten times more prevalent than RelE<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN>. Consequently, RelB has been given a transcription rate 100 times higher than RelE to make up for the higher half-life of RelE.

For the full set of reactions, <A href="https://static.igem.org/mediawiki/2017/1/1a/T--SDU-Denmark--Equations-for-model.pdf" target="_blank">read here</A>
Each bacterial chromosome has two operators, each of which can bind one RelB<SUB>2</SUB> dimer (O(RelB<SUB>2</SUB>)) or either one or two RelB<SUB>2</SUB>RelE complexes (O(RelB<SUB>2</SUB>RelE) and O(RelB<SUB>2</SUB>RelE)<SUB>2</SUB>)<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>. 
Each cell is assumed to have four chromosomes with one relBE promoter each, as this is an average number of chromosomes for an exponentially growing <I>E. coli</I> cell<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>. This was found to stabilise the inherent system considerably compared to a system containing one chromosome per cell, as the systems exhibited similar behaviour but with different amounts of noise<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>.

The initial values in the model are listed below.
</P><TABLE class="table"><TBODY><TR><TH> Molecule </TH><TH> Initial number of copies </TH></TR><TR><TD> mRNA </TD><TD> 7 </TD></TR><TR><TD> RelB<SUB>2</SUB></TD><TD> 410 </TD></TR><TR><TD> RelE </TD><TD> 0 </TD></TR><TR><TD> RelB<SUB>2</SUB>RelE </TD><TD> 65 </TD></TR><TR><TD> RelB<SUB>2</SUB>RelE<SUB>2</SUB></TD><TD> 11 </TD></TR><TR><TD> Free operator sites </TD><TD> 0 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>) </TD><TD> 2 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>RelE) </TD><TD> 0 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>RelE)<SUB>2</SUB></TD><TD> 2 </TD></TR></TBODY></TABLE><P>All values are integers as the Gillespie algorithm works with discrete numbers of molecules. The values were chosen based on a stable equilibrium found by letting the model run a simulation of the inherent system over 450 minutes with different starting values. 

Ranging from 1-350 molecules per minute, the implemented expression rates of RelE and RelB in the model might seem too high, as the rates in the inherit system is effectively around 80-100 for RelB and 2-5 for RelE. However, the possibility of placing the <I>relE</I> and <I>relB</I> genes under regulation of controllable promoters makes the high total production values reasonable.

When the inherent toxin-antitoxin system is activated under starvation, 40-70 molecules of free RelE are found in each cell, making it reasonable to believe that the cells enter dormancy when a few tens of free RelE copies are present. This result obtained from the model is in agreement with literature<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>
For the full list of constants, see the attached Table 1 at the bottom of this page. 
</P><P class="P-Larger"><B>Running the Model</B></P><P>To give a stochastic view of the system, the Gillespie algorithm was run in the computer programming language MATLAB, utilising an <A href="https://se.mathworks.com/matlabcentral/fileexchange/34707-gillespie-stochastic-simulation-algorithm" target="_blank">implementation</A> made by MATLAB user Nezar. 

For the runs simulating dormancy, deterministic initial values were used and the system was run for 30 minutes without activation of the inserted toxin promoter. This resulted in a stochastic distribution of initial values mimicking variations between cells. Analysis showed, that 30 minutes was enough for the model to find a stable distribution, which is realistic considering the growth cycle of an <I>E. coli</I> cell. 

For the runs simulating activation, the data generated at the end of a dormancy run was used as initial value and deactivated expression of RelE. When the concentration of free RelE decreases to below 15 copies, a cell is considered active. This value was probably set too low, but tests displayed marginal difference between 15 and 45 copies, where the lower limit was chosen to decrease uncertainty of the cell state. 

All runs simulated 1000 cells, which should be sufficient to get stable averages and the model assumed well-mixed conditions in every cell and considered each cell independently. Furthermore, the model has no cut-off for maximum values of RelE, as the exact relation between RelE concentration and dormancy state is unknown, yet a functional cut-off was found through activation times.
<A href="https://static.igem.org/mediawiki/2017/1/1e/Matlab-scripts_SDU.zip" target="_blank">Download of Matlab-scripts</A></P><P class="P-Larger"><B>Table 1</B></P><TABLE class="table"><TBODY><TR><TH>Constant</TH><TH>Identifier</TH><TH>Units</TH><TH>Value</TH></TR><TR><TD>mRNA transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>α<SUB>0</SUB></TD><TD>1/min</TD><TD>154.665</TD></TR><TR><TD>mRNA half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>m</SUB></TD><TD>min</TD><TD>7.2</TD></TR><TR><TD>RelB half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>B</SUB></TD><TD>min</TD><TD>4.3</TD></TR><TR><TD>RelE half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>E</SUB></TD><TD>min</TD><TD>43      (growing)  2000 (dormant)</TD></TR><TR><TD>Bound RelB half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>c</SUB></TD><TD>min</TD><TD>17</TD></TR><TR><TD>RelB transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>B</SUB></TD><TD>1/min</TD><TD>15</TD></TR><TR><TD>RelE transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>E</SUB></TD><TD>1/min</TD><TD>0.3</TD></TR><TR><TD>Binding rate<SPAN class="reference"><SPAN class="referencetext">Sneppen K, Zocchi G. Physics in Molecular Biology. Cambridge, UK: Cambridge University Press; 2005.</SPAN></SPAN></TD><TD>k <SUB>b</SUB></TD><TD>1/min</TD><TD>3.8</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD>K<SUB>D</SUB> (B<SUB>2</SUB>E)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD> K<SUB>D</SUB>(B<SUB>2</SUB>E<SUB>2</SUB>)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate O.B<SUB>f</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>K<SUB>D1</SUB></TD><TD>molecules</TD><TD>10</TD></TR><TR><TD>Dissociation rate O.B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD> K<SUB>D2</SUB></TD><TD>molecules</TD><TD>0.04</TD></TR><TR><TD>Dissociation rate O.(B<SUB>2</SUB>E)<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  K<SUB>D3</SUB></TD><TD>molecules</TD><TD>30</TD></TR><TR><TD>Cleavage rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.sciencedirect.com/science/article/pii/S0092867402012485?">Pedersen K, et al. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112:131–140. doi: 10.1016/S0092-8674(02)01248-5</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  k<SUB>c</SUB></TD><TD>1/min  1/molecules</TD><TD>2.0</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="model-results" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Modelling Results of the RelE-RelB System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Results</B></P><P>As the bacteria only require a few tens of RelE molecules to enter dormancy, the threshold was placed at 40 copies to allow for lag in activation. This is of course an oversimplification, but it is not a problem that activation and dormancy are defined at different levels, since the exact number of RelE molecules required to induce dormancy is unknown.<B>The Impact of Promoters with Different Production Rates on RelE Expression</B>
During the implementation of RelE, gene expression was simulated for promoters with different strengths, which were chosen through an iterative process. Promoters with production rates of 3.5 and 10.5 molecules per minute, both induce dormancy rather slowly, with the latter inducing dormancy in approximately 50 minutes. In cells cloned with a promoter producing 35 molecules per minute, the cells will enter dormancy in about 10 minutes, while promoters producing 105 and 350 molecules per minute both have a negligible timeframe. In the simulated dormancy system, the three strongest promoters exhibited similar results, indicating that a certain threshold value had been transcended. This means, that not only was the gene expression disproportional to the promoter strength, but the risk of overshooting was also increased tremendously.</P><DIV class="figure-text"><P><B>Figure 1.</B> The increase of free RelE molecules in <I>E. coli</I> cells, after activation of the artificial RelE production, shown logarithmically. The condition for induced dormancy is an amount of free RelE molecules around tens of copies. The three highest levels of RelE production, correlating with the highest promoter strengths, show little difference in the time at which dormancy occurs. When the RelE production is set to 10.5 molecules per minute, dormancy is induced more slowly and stabilises at lower concentrations. The lowest RelE production value does not trigger dormancy, and has only little effect on the system.</P></DIV><P><B>RelB is Required for Activation of Bacteria after Dormant State</B>
If RelB was not expressed, the bacteria remained dormant for hours after RelE production had ceased, making it clear, that production of the antitoxin RelB was necessary for activation. Considering the stability of RelE in non-growing conditions, it was not surprising to find that RelB production would be the primary element in sequestering free RelE.</P><DIV class="figure-text"><P><B>Figure 2.</B> The logarithmic plots shows that the decrease in free RelE in dormant bacteria is low without artificial expression of RelB. None of the simulated bacteria reentered an active state within the modelled time.</P></DIV><P><B>Appropriate Ratio of RelE and RelB Expression is Essential</B>
Different expression rates of RelB were combined with production rates of RelE at 35 and 100 molecules per minute, corresponding to relatively medium and strong expression levels respectively. This revealed that variation in RelB had a higher impact on the time required before the dormant state was reached when the RelE production is lowered. Out of the established configurations, the RelB<SUB>2</SUB>:50-RelE:35 configuration showed promising results. Compared to the RelB<SUB>2</SUB>:35-RelE:35 configuration, where only few bacteria reenter an active state within the modelled time set to 2.5 hours, the RelB<SUB>2</SUB>:50-RelE:35 configuration revealed a high sensitivity to the expression level of RelB. This indicated a need for the expression of RelB to be higher than RelE, but as the best results were achieved at low production rates of RelE, it is important to stringently control the expression of RelB to ensure that the bacteria are able to enter dormancy.</P><DIV class="figure-text"><P><B>Figure 3.</B> The variation in time required for the bacteria to enter an active state for different expression levels of RelB is dependent on the level of RelE expression. All configurations achieve dormancy within the modelled time.</P></DIV><DIV class="figure-text"><P><B>Figure 4.</B> RelB<SUB>2</SUB>:50-RelE:35 induces an active state within minutes, whereas RelB:35-RelE:35 only causes few of the bacteria to enter an active state. In the remaining configurations all bacteria remained dormant.</P></DIV><P class="P-Larger"><B>Discussion of Model Regarding the Artificial Dormancy System</B></P><P><B>Model Limitations</B>
As the model only simulates the dormancy for 2 hours, not all configurations have reached equilibrium, therefore these configurations might attain a higher concentration of free RelE than modelled. This could result in a prolonged phase reentering an active state.
The activation model has a rather weak predictability, as the half-life of RelE is quite high. This essentially means, that the model is unable to reduce the total amount of RelE, given by RelE<SUB>tot</SUB>=RelE<SUB>free</SUB>+RelE<SUB>bound</SUB>, because of the short simulated timespan. Thus, in activation runs, where the number of free RelE molecules reaches a high level, the amount of RelE<SUB>bound</SUB> is equally high. The problem arises, since the model works under the assumption that bound RelB is not completely stable, causing RelE-RelB complexes to dissociate, whereby RelE is freed. This causes the induced RelB expression to approach equilibrium, implying that the decrease in free RelE is rather slow. It is therefore not only the amount of free RelE that determines the activation time, but also whether the amount of dissociating complexes is high enough to counter the RelB production. Hence, it has no relevance to further test high RelE production in this model, as a high number of complexes will easily be achieved. <B>Light Sensitivity</B>
One thing the model does not include, is the actual sensitivity to light. Out in the open, the amount of light is rarely an on/off switch, which means there will be periods with varying degrees of activation. Since the bacteria should be active during overcast days, the system requires a threshold, both in the sensitivity of the light-regulated promoter, but also in the activation of the toxin-antitoxin system. Variation of light is implicitly modelled through variation in promoter strength, for instance a half probability of activation translates roughly to a half production rate in the individual bacteria. Because of this, it is important not only to find functioning configurations, but also to investigate the closely related configurations, so that the bacteria neither lay dormant at overcast days, nor make the dormancy system obsolete in moonlight. 
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><P class="P-Larger"><B>Approach</B></P><P>
In 2004 the <A href="https://2004.igem.org/austin.cgi" target="_blank">Austen and UCSF iGEM team</A> created a <SPAN class="highlighted">device sensitive to light,</SPAN> laying the foundation for the <A href="http://parts.igem.org/Coliroid" target="_blank">Coliroid project</A>. In this project, the <SPAN class="highlighted">system is combined with the RelE-RelB toxin-antitoxin system</SPAN> in the endeavour to mediate <SPAN class="highlighted">light-dependent dormancy in bacteria</SPAN>. As tight regulation is required for the RelE-RelB system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294780/&quot;">Tashiro Y, Kawata K, Taniuchi A, Kakinuma K, May T, Okabe S. RelE-Mediated Dormancy Is Enhanced at High Cell Density in Escherichia coli. J Bacteriol. 2012;194(5):1169-76.</A></SPAN></SPAN>, <SPAN class="highlighted">modelling of the toxin-antitoxin system</SPAN> is essential. The impact of different RelE-RelB expression levels was simulated by modelling. Using the results obtained by this modelling, a hypothetical working system-design was devised. 

On basis of the modulated system, the potential of different vectors and promoters in various combinations was tested. This constitutes the foundation for how the design of the light-dependent dormancy system in <I>E. coli</I> has been optimised, and the final approach shaped. Ultimately, the light-dependent dormancy system, which is illustrated in Figure 3, was composed of the following parts: 
</P><UL class="list"><LI><SPAN class="highlighted">The photocontrol device controlled by the PenI-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0074" target="_blank">BBa_R0074</A>, on a high copy vector.</SPAN></LI><LI><SPAN class="highlighted">The antitoxin RelB controlled by pBAD, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449031" target="_blank">BBa_K2449031</A>, on a low copy vector.</SPAN></LI><LI><SPAN class="highlighted">The toxin RelE controlled by the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>, on either a low copy vector or the chromosome. </SPAN></LI></UL><P>For further information about our approach, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#light-sensing-system-approach">read here</SPAN>.</P><DIV class="figure-text"><P><B>Figure 3.</B> The final design of the light-dependent dormancy system. RelE under control of the OmpR-regulated promoter on the chromosome or a low copy vector, RelB under the control of the pBAD promoter on a low copy vector, and the photocontrol device controlled by the PenI-regulated promoter on a high copy vector.</P></DIV><DIV class="modal fade" id="light-sensing-system-approach" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Light Sensing System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P><B>Balancing Bacterial Dormancy Requires Accurate Regulation of the System</B>
The genes needed for inducing dormancy when the bacteria are not exposed to light, are found in the photocontrol device part, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>. This part is composed of three genes named <I>ho1</I>, <I>pcyA</I>, and <I>cph8</I>, all of which are essential to ensure the cells ability to respond to red light. When the photocontrol device is exposed to light, a phosphorylation cascade activates the transcription factor OmpR, which in turn induces transcription through the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>. This system is combined with the RelE-RelB toxin-antitoxin system in the endeavour to mediate light-dependent dormancy in bacteria. In the first considered design of the light-dependent dormancy system, the aim was to clone the photocontrol device, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>, under control of a constitutive promoter, RelE under control of the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>, and RelB under control of a constitutive promoter, all into one high copy BioBrick assembly plasmid pSB1C3, as seen on Figure 1.</P><DIV class="figure-text"><P><B>Figure 1.</B> All three components of the light-dependent dormancy system cloned into one high copy plasmid.</P></DIV><P><B>The Photocontrol Device was Placed under Control of a Constitutive Promoter</B>
From the <A href="http://parts.igem.org/Promoters/Catalog/Anderson" target="_blank">constitutive promoter family</A> the weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, the two medium promoters, <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, and <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, and the strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, were tested by fluorescence microscopy to determine which one to use for expression of the photocontrol device and RelB. Overnight cultures of the submitted parts expressed in <I>E. coli</I> TOP10 illustrated a clear gradient of increasing red fluorescent protein (RFP) expression correlated with the strength of the promoter, as seen on Figure 2.</P><DIV class="figure-text"><P><B>Figure 2.</B> Cultures of <I>E. coli</I> TOP10. From left to right: WT, the weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, the two medium promoters, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, and <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, and the strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>.</P></DIV><P>
On the basis of the data obtained by fluorescence microscopy, the strong constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, was chosen to control the photocontrol device. For still inexplicable reasons the photocontrol device emerged difficult to clone with the strong constitutive promoter. Although the molecular cloning of these parts was optimised several times very few successful clonings were accomplished, and the few times correct assembly was obtained, the BioBrick was not reproducibly purifiable. By sequencing, it was deduced that a region of the plasmid containing both the promoter and the BioBrick prefix had vanished. To circumvent this inconvenient cloning two other promoters, that are constitutive in <I>E. coli</I>, were examined, namely the PenI-regulated, <A href="http://parts.igem.org/Part:BBa_R0074" target="_blank">BBa_R0074</A>, and the Mnt-regulated, <A href="http://parts.igem.org/Part:BBa_R0073" target="_blank">BBa_R0073</A>, promoters. By performing fluorescence microscopy on composite parts of the promoters controlling yellow fluorescent protein (YFP), <A href="http://parts.igem.org/Part:BBa_I6102" target="_blank">BBa_I6102</A>, and <A href="http://parts.igem.org/Part:BBa_I6103" target="_blank">BBa_I6103</A>, respectively, the expression levels were assessed. The obtained results revealed that the PenI-regulated promoter facilitated very strong expression of the marker gene, whereas the expression controlled by the Mnt-regulated promoter was noticeably lower. Based on these findings, the photocontrol device was placed under the control of the PenI-regulated promoter instead of the strong constitutive promoter from the constitutive promoter family. As it turned out, this cloning likewise emerged difficult. After several attempts, it was decided to focus on the other aspects of the dormancy system.
<B>Regulation of the OmpR-dependent Promoter Required a Low Copy Vector </B>
The first construct containing the genes required for the light-induced dormancy was designed as shown in Figure 1. As the conducted modelling clarified, the necessity for stringent regulation of the RelE and RelB expression, the properties of the OmpR-regulated promoter were studied thoroughly. To assess the functionality of the OmpR-regulated promoter in practice, a reporter system containing the OmpR-regulated promoter controlling RFP was cloned into the <I>E. coli</I> strain MG1655 Δ<I>ompR</I>. The phenotype of the resulting cultures revealed a dysregulation of the OmpR-regulated promoter. Thorough research lead to the finding that the OmpR-dependent promoter is not controllable when cloned on a high copy vector. As the modelling revealed, and which is evident from <SPAN class="reference-2">Figure 2-Main-Page</SPAN>, a relatively low expression of RelE is required to induce dormancy, whereas high expression levels quickly result in overshooting. Since the OmpR-regulated promoter is an integrated part of the light sensing system, replacement is not an option. Therefore, the variability of the <I>relE</I> gene copy number was studied, and it was found that the OmpR-regulated promoter should be cloned into the bacterial chromosome or a low copy vector to obtain proper regulation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/16306980&quot;">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. This intriguing finding let to the aspiration to investigate the controllability of the OmpR-dependent promoter on vectors with different copy numbers compared to the chromosome, thereby improving the characterisation of the promoter for the benefit to future iGEM teams. 

To incorporate DNA onto the bacterial chromosome, homologous recombination with the red λ recombinase is a suitable approach <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/2958633&quot;">Thompson JF, de Vargas LM, Skinner SE, Landy A. Protein-protein interactions in a higher-order structure direct lambda site-specific recombination. Journal of molecular biology. 1987;195(3):481-93.</A></SPAN></SPAN>. Using this technique, a short fragment of chromosomal DNA at the bacterial attachment site attB <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/14687564&quot;">Groth AC, Calos MP. Phage integrases: biology and applications. Journal of molecular biology. 2004;335(3):667-78.</A></SPAN></SPAN> can be replaced with a linear DNA fragment encoding the OmpR-dependent promoter, RelE, and an chloramphenicol resistance cassette. Using polymerase chain reaction (PCR), the linear DNA sequence was flanked by sequences, which are homologous to part of the chromosome. The linear DNA fragment was electroporated into bacteria containing the pKD46 plasmid, encoding the red λ recombinase <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/2958633&quot;">Thompson JF, de Vargas LM, Skinner SE, Landy A. Protein-protein interactions in a higher-order structure direct lambda site-specific recombination. Journal of molecular biology. 1987;195(3):481-93.</A></SPAN></SPAN>, which mediated the recombination. The fundamental concept of this approach is illustrated in Figure 3.
</P><DIV class="figure-text"><P><B>Figure 3.</B> The principle behind the recombination. By PCR, the two flanking sequences are assembled with the fragment containing the chloramphenicol resistance gene (camR), the OmpR-regulated promoter, and the relE gene. The flanking sequences are homologous to part of the chromosome around the bacterial attachment site (attB), enabling the homologous recombination.</P></DIV><P>
Using this approach, it was attempted to assemble the OmpR-regulated promoter controlling RelE with the flanking regions by PCR. The assembly of the fragment derived from the plasmid with either one of the flanking fragments was achieved. However, the assembly of the complete chromosomal insertion fragment containing all three segments emerged problematic. Several attempts with numerous process optimisations were performed, but unfortunately without success. To circumvent this, another attachment site was chosen. The assembly of the complete fragment by PCR was achieved and chromosomal insertion by electroporation was attempted several times, but in vain. In consideration of the fact that RelE is a bacterial toxin, the cloning of this gene was inconvenient. The model indicated, that overexpression of the antitoxin RelB would bypass this difficulty. As the homologous recombination of this fragment emerged such a challenging task, it was decided to focus on the conduct of the OmpR-regulated promoter on different plasmids, as this was evidently the more convenient approach of the general implementation of the OmpR-regulated promoter. 
<B>An Inducible Promoter was Chosen to Regulate the Gene Expression of the Antitoxin RelB</B>
Originally, it was thought to place the antitoxin RelB under a constitutive promoter of appropriate strength. However, as the modelling revealed, strict regulation of RelB is essential to counteract the toxic effect of RelE and enable a functional dormancy system. Thus, it was deduced that it would be more suitable to utilise an inducible promoter for this purpose and it was decided to put RelB under control of the LacI-regulated, lambda pL hybrid promoter, <A href="http://parts.igem.org/Part:BBa_R0011" target="_blank">BBa_R0011</A>. The <A href="https://2015.igem.org/Team:William_and_Mary" target="_blank">William and Mary team</A> found that the LacI-regulated, lambda pL hybrid promoter had a low level of noise,  a measure of the variability in gene expression between cells in the population, when cloned into a low copy vector. Therefore, this regulated promoter was chosen. 

To mimic the expression of  RelB, a reporter system composed of the LacI-regulated, lambda pL hybrid promoter and GFP was assembled. Hereby, the appropriate concentration of IPTG required to induce the promoter on a low copy vector, was identified. During the cloning, it was discovered that the site between the LacI-regulated, lambda hybrid promoter and this particular construct under the tested conditions had formed a hotspot for transposons. In the light of this finding, another inducible promoter was chosen. The <A href="https://2015.igem.org/Team:HKUST-Rice" target="_blank">HKUST-Rice iGEM 2015 team</A> demonstrated that induction of the AraC-regulated promoter, pBAD, caused a gradual increase in gene expression when cloned into the low to medium copy plasmid, pSB3K3, in contrast to an all-or-none behavior when cloned into the high copy vector, pSB1K3. Taking these results into consideration, the pBAD promoter was used in combination with the pSB3K3 vector to regulate the expression of the antitoxin RelB. Again, a reporter system was used to mimic the expression of RelB, as the part, <A href="http://parts.igem.org/Part:BBa_I6058" target="_blank">BBa_I6058</A>, containing YFP controlled by pBAD, was assessed on different vectors. 
<B>The Final Approach for the Three Components Comprised Three Different Vectors</B>
Based on the modelling the system approaches reviewed in the preceding part, the final design, which is illustrated on Figure 4, was established. Ultimately, the dormancy system was composed of the photocontrol device controlled by the PenI-regulated promoter on a high copy vector, RelB controlled by pBAD, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449031" target="_blank">BBa_K2449031</A>, on a low copy vector  and RelE controlled by the OmpR-regulated promoter on either a low copy vector or the chromosome.

</P><DIV class="figure-text"><P><B>Figure 4.</B> The final design of the light-dependent dormancy system. RelE under control of the OmpR-regulated promoter on the chromosome or a low copy vector, RelB under the control of the pBAD promoter on a low copy vector, and the photocontrol device controlled by the PenI-regulated promoter on a high copy plasmid.</P></DIV></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-results-dormancy-system">Click here if you wish to go directly to the Project Demonstration &amp; Results section of the Dormancy System.</A></I></P></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-design">Click here to return to the Project Design overview.</A></I></P></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-design-carbon-fixation"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Carbon Fixation</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B>Introduction</B></P><P>
Carbon fixation in autotrophic organisms is responsible for the net fixation of 7×10<SUP>16</SUP> g carbon annually <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Berg+(2011)+Ecological+Aspects+of+the+Distribution+of+Different+Autotrophic+CO2+Fixation+Pathways">Berg IA. Ecological aspects of the distribution of different autotrophic CO2 fixation pathways. Applied and Environmental Microbiology. 2011;77(6):1925-36.</A></SPAN></SPAN>. Six different pathways related to carbon fixation have been discovered, but the most widespread of these, is the Calvin-Benson-Bassham (CBB) cycle found in photosynthetic eukaryotes, e.g. plants and algae, as well as in photo- and chemosynthetic bacteria <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://books.google.dk/books?id=puEsBAAAQBAJ&amp;pg=PA21&amp;lpg=PA21&amp;dq=calvin+cycle+most+widespread&amp;source=bl&amp;ots=8QGIRwvzDj&amp;sig=7jfO_H3MSc67XxB8xRM3nVdavdA&amp;hl=en&amp;sa=X&amp;ved=0ahUKEwj64OL0-pXVAhXrbZoKHbEWCzcQ6AEINjAD#v=onepage&amp;q=cyano&amp;f=false">B. Bowien MG, R. Klintworth, U. Windhövel. Metabolic and Molecular Regulation of the CO2-assimilating Enzyme System in Aerobic Chemoautotrophs.  Microbial Growth on C1 Compounds: Proceedings of the 5th International Symposion. 1st ed. Institute for Microbiology, Georg-August-University Göttingen, Federal Republic of Germany: Martinus Nijhoff Publishers; 1987.</A></SPAN></SPAN>. <SPAN class="highlighted">Out of the eleven enzymes needed for the Calvin cycle, only three are heterologous to <I>E. coli</I></SPAN>, namely: ribulose-1,5-bisphosphate carboxylase/oxygenase (<SPAN class="highlighted">RuBisCo</SPAN>), sedoheptulose-1,7-bisphosphatase (<SPAN class="highlighted">SBPase</SPAN>) and phosphoribulokinase (<SPAN class="highlighted">PRK</SPAN>). By the concurrent heterologous expression of the three genes encoding these enzymes, <I>E. coli</I> can be engineered to perform the full Calvin cycle.</P><DIV class="figure-text"><P><B>Figure 4.</B> A simplified illustration of the Calvin cycle, with the enzymes heterologous to <I>E. coli</I> and their respective substrates and products shown.</P></DIV><P>The <SPAN class="highlighted">carboxysome is a microcompartment</SPAN> utilised by many chemoautotrophic bacteria, including cyanobacteria, as a CO<SUB>2</SUB> accumulating mechanism to <SPAN class="highlighted">increase carbon fixation efficiency</SPAN>. This organelle-like polyhedral body is able to increase the internal concentrations of inorganic carbon by 4000-fold compared to the external concentration <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. One type of carboxysome is the ɑ-carboxysome, which consists of a proteinaceous outer shell composed of <SPAN class="highlighted">six different shell proteins designated CsoS1ABCD and CsoS4AB. This shell encloses RuBisCo, the shell associated protein (CsoS2), and the enzyme carbonic anhydrase (CsoS3)</SPAN>. In the proteobacteria <I>Halothiobacillus neapolitanus</I>, these genes are clustered into the <SPAN class="highlighted"><I>cso</I> operon</SPAN>. The carbonic anhydrase converts HCO<SUB>3</SUB><SUP>-</SUP>, which diffuses passively into the carboxysome, to CO<SUB>2</SUB>, thereby driving the continued diffusion of HCO<SUB>3</SUB><SUP>-</SUP> into the microcompartment <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. The increased CO<SUB>2</SUB> concentration in the vicinity of RuBisCo increases the rate of carbon fixation by saturating the RuBisCo enzyme and increasing the CO<SUB>2</SUB> to O<SUB>2</SUB> ratio, enabling carboxylation to dominate over oxygenation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. The shell associated protein is essential for the biogenesis of the ɑ-carboxysome <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25826651">Cai F, Dou Z, Bernstein SL, Leverenz R, Williams EB, Heinhorst S, et al. Advances in Understanding Carboxysome Assembly in Prochlorococcus and Synechococcus Implicate CsoS2 as a Critical Component. Life (Basel, Switzerland). 2015;5(2):1141-71.</A></SPAN></SPAN>.</P><DIV class="figure-text"><P><B>Figure 5.</B> An illustration of the ɑ-carboxysome. The shell proteins CsoS1ABC and CsoS4AB enclose the enzymes RuBisCo and carbonic anhydrase.</P></DIV><P><SPAN class="highlighted">For the Calvin cycle to proceed, energy in the form of ATP and electrons carried by NADPH are required</SPAN>. The photosystems are complexes in photosynthesising organisms that can supply this by photophosphorylation. To engineer <I>E. coli</I> to do photosynthesis, 13 genes is needed for the assembly of chlorophyll a and 17 genes for the assembly of photosystem II, which needs to be heterogeneously expressed. An alternative process in which a diverse array of phototrophic bacteria and archaea harvest energy from light, is through a retinal-containing protein called proteorhodopsin that catalyses the light-activated proton efflux across the cell membrane, and thereby drive ATP synthesis. Opposed to the photosystems, the proteorhodopsin is anoxygenic and generates no NADPH, which is crucial for the Calvin cycle to proceed <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892948/">Walter JM, Greenfield D, Bustamante C, Liphardt J. Light-powering Escherichia coli with proteorhodopsin. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(7):2408-12.</A></SPAN></SPAN>. For further information about the carbon fixation, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#co2-fixation-theory">read here</SPAN>.
</P><DIV class="modal fade" id="co2-fixation-theory" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Carbon Fixation</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><B>Carbon Fixation through the Calvin Cycle</B>
Carbon fixation in autotrophic organisms is responsible for the net fixation of 7×10<SUP>16</SUP> g carbon annually, thereby being the most imperative biosynthetic process in nature <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Berg+(2011)+Ecological+Aspects+of+the+Distribution+of+Different+Autotrophic+CO2+Fixation+Pathways">Berg IA. Ecological aspects of the distribution of different autotrophic CO2 fixation pathways. Applied and Environmental Microbiology. 2011;77(6):1925-36.</A></SPAN></SPAN>. Six different autotrophic pathways for carbon fixation have been discovered in a variety of organisms <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3424341/">Ducat DC, Silver PA. Improving Carbon Fixation Pathways. Current opinion in chemical biology. 2012;16(3-4):337-44.</A></SPAN></SPAN>. The most widespread of these, is the Calvin-Benson-Bassham (CBB) cycle, found in photosynthetic eukaryotes, e.g. plants and algae, as well as in photo- and chemosynthetic bacteria <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://books.google.dk/books?id=puEsBAAAQBAJ&amp;pg=PA21&amp;lpg=PA21&amp;dq=calvin+cycle+most+widespread&amp;source=bl&amp;ots=8QGIRwvzDj&amp;sig=7jfO_H3MSc67XxB8xRM3nVdavdA&amp;hl=en&amp;sa=X&amp;ved=0ahUKEwj64OL0-pXVAhXrbZoKHbEWCzcQ6AEINjAD#v=onepage&amp;q=cyano&amp;f=false">B. Bowien MG, R. Klintworth, U. Windhövel. Metabolic and Molecular Regulation of the CO2-assimilating Enzyme System in Aerobic Chemoautotrophs.  Microbial Growth on C1 Compounds: Proceedings of the 5th International Symposion. 1st ed. Institute for Microbiology, Georg-August-University Göttingen, Federal Republic of Germany: Martinus Nijhoff Publishers; 1987.</A></SPAN></SPAN>. The Calvin cycle, as this pathway is also called, can proceed under aerobic conditions, and only three enzymes and one microcompartment involved are heterologous to the gram-negative bacteria <I>E. coli</I>, making this the most obvious choice for the implementation of a carbon fixation pathway. In contrast, the 3-hydroxypropionate pathway for carbon fixation would require the transfer of ten heterologous genes <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23376595">1.	Mattozzi M, Ziesack M, Voges MJ, Silver PA, Way JC. Expression of the sub-pathways of the Chloroflexus aurantiacus 3-hydroxypropionate carbon fixation bicycle in E. coli: Toward horizontal transfer of autotrophic growth. Metabolic engineering. 2013;16:130-9.</A></SPAN></SPAN>. Furthermore, the reductive carboxylic acid cycle found in phylogenetically diverse autotrophic bacteria and archaea <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Evidence+for+Autotrophic+CO2+Fixation+via+the+Reductive+Tricarboxylic+Acid+Cycle+by+Members+of+the+%CE%B5+Subdivision+of+Proteobacteria%E2%80%A0">Hugler M, Wirsen CO, Fuchs G, Taylor CD, Sievert SM. Evidence for autotrophic CO2 fixation via the reductive tricarboxylic acid cycle by members of the epsilon subdivision of proteobacteria. J Bacteriol. 2005;187(9):3020-7.</A></SPAN></SPAN> and the noncyclic reductive acetyl-CoA or Wood-Ljungdahl pathway <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2646786/">Ragsdale SW, Pierce E. Acetogenesis and the Wood-Ljungdahl Pathway of CO(2) Fixation. Biochimica et biophysica acta. 2008;1784(12):1873-98.</A></SPAN></SPAN> require strict anaerobic conditions. 

The Calvin cycle involves eleven enzymes, of which eight are intrinsic to <I>E. coli</I>. The three heterologous enzymes are RuBisCo, SBPase and PRK. The latter phosphorylates ribulose-5-phosphate to ribulose-1,5-bisphosphate. This is the substrate for RuBisCo, which catalyses the carboxylation, whereby glycerate-3-phosphate is produced. Later in the cycle, SBPase catalyses the dephosphorylation of sedoheptulose-1,7-bisphosphate to sedoheptulose-7-phosphate, which is later converted to ribulose-5-phosphate, completing the circle. The net effect of three full cycles is the conversion of three CO<SUB>2</SUB> molecules into one molecule glyceraldehyde-3-phosphate, which can be used for energy production via glycolysis or polysaccharide biosynthesis. Separately, these enzymes have previously been heterogeneously expressed in <I>E. coli</I> using various donor species, such as wheat <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/9758762">Dunford RP, Catley MA, Raines CA, Lloyd JC, Dyer TA. Purification of active chloroplast sedoheptulose-1,7-bisphosphatase expressed in Escherichia coli. Protein expression and purification. 1998;14(1):139-45.</A></SPAN></SPAN>, the algae <I>Chlamydomonas sp.</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/15849430">Tamoi M, Nagaoka M, Shigeoka S. Immunological properties of sedoheptulose-1,7-bisphosphatase from Chlamydomonas sp. W80. Bioscience, biotechnology, and biochemistry. 2005;69(4):848-51.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26828117">Vira C, Prakash G, Rathod JP, Lali AM. Cloning, expression, and purification of Chlamydomonas reinhardtii CC-503 sedoheptulose 1,7-bisphosphatase in Escherichia coli. Preparative biochemistry &amp; biotechnology. 2016;46(8):810-4.</A></SPAN></SPAN>, and the cyanobacteria <I>Synechococcus</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16423843">Parikh MR, Greene DN, Woods KK, Matsumura I. Directed evolution of RuBisCO hypermorphs through genetic selection in engineered E.coli. Protein engineering, design &amp; selection : PEDS. 2006;19(3):113-9.</A></SPAN></SPAN>. <B>The Carboxysome Increases the Efficiency of the Carboxylation by RuBisCo</B>
Many photosynthesising bacteria have developed CO<SUB>2</SUB> concentrating mechanisms to increase the efficiency of the carbon fixation process. Cyanobacteria and many chemoautotrophic bacteria utilise organelle-like polyhedral bodies, that increase the internal concentrations of inorganic carbon 4000-fold compared to external levels  <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. These microcompartments, called carboxysomes, appear to have arisen twice during evolution and have undergone a process of convergent evolution <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3811607/">Rae BD, Long BM, Badger MR, Price GD. Functions, Compositions, and Evolution of the Two Types of Carboxysomes: Polyhedral Microcompartments That Facilitate CO(2) Fixation in Cyanobacteria and Some Proteobacteria. Microbiology and Molecular Biology Reviews : MMBR. 2013;77(3):357-79.</A></SPAN></SPAN>. The two types, designated ɑ and β, share main structural and functional features <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3811607/">Rae BD, Long BM, Badger MR, Price GD. Functions, Compositions, and Evolution of the Two Types of Carboxysomes: Polyhedral Microcompartments That Facilitate CO(2) Fixation in Cyanobacteria and Some Proteobacteria. Microbiology and Molecular Biology Reviews : MMBR. 2013;77(3):357-79.</A></SPAN></SPAN>. The ɑ-carboxysome consists of a proteinaceous outer shell composed of six different shell proteins designated CsoS1ABCD and CsoS4AB, and encloses RuBisCo, the shell associated protein CsoS2, and the enzyme carbonic anhydrase CsoS3. On average, ~250 RuBisCo molecules are localised within each carboxysome, and these are organised into three to four concentric layers <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17669419">Iancu CV, Ding HJ, Morris DM, Dias DP, Gonzales AD, Martino A, et al. The structure of isolated Synechococcus strain WH8102 carboxysomes as revealed by electron cryotomography. Journal of molecular biology. 2007;372(3):764-73.</A></SPAN></SPAN>. The carbonic anhydrase converts HCO<SUB>3</SUB><SUP>-</SUP>, which diffuses passively into the carboxysome, to CO<SUB>2</SUB>, thereby driving the continued diffusion of HCO<SUB>3</SUB><SUP>-</SUP> into the microcompartment <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. The increased CO<SUB>2</SUB> concentration in the vicinity of RuBisCo increases the rate of carbon fixation by saturating the RuBisCo enzyme and increasing the CO<SUB>2</SUB> to O<SUB>2</SUB> ratio, enabling carboxylation to prevail over oxygenation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027813/">Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</A></SPAN></SPAN>. The shell associated protein is essential for the biogenesis of the ɑ-carboxysome <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/25826651">Cai F, Dou Z, Bernstein SL, Leverenz R, Williams EB, Heinhorst S, et al. Advances in Understanding Carboxysome Assembly in Prochlorococcus and Synechococcus Implicate CsoS2 as a Critical Component. Life (Basel, Switzerland). 2015;5(2):1141-71.</A></SPAN></SPAN>. The genes encoding the enzymes and shell proteins forming the ɑ-carboxysome from <I>Halothiobacillus neapolitanus</I> are clustered into the <I>cso</I> operon. This operon has been heterogeneously expressed in <I>E. coli</I> and its transcriptional regulation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17899012">Cai F, Heinhorst S, Shively JM, Cannon GC. Transcript analysis of the Halothiobacillus neapolitanus cso operon. Archives of microbiology. 2008;189(2):141-50.</A></SPAN></SPAN> and functionality <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22184212">Bonacci W, Teng PK, Afonso B, Niederholtmeyer H, Grob P, Silver PA, et al. Modularity of a carbon-fixing protein organelle. Proceedings of the National Academy of Sciences of the United States of America. 2012;109(2):478-83.</A></SPAN></SPAN> has been studied.
<B>Energy and Electrons are Required for Carbon Fixation</B>
For the Calvin cycle to proceed, energy in the form of ATP and electrons carried by NADPH are required. In photosynthesising organisms, such as plants and cyanobacteria, these constituents are provided by photophosphorylation performed by the photosystem complex. When engineering <I>E. coli</I> to perform photophosphorylation, 13 genes for the biosynthesis of chlorophyll a and 17 genes for the biosynthesis of photosystem II need to be heterogeneously expressed. Several attempts have been made at expressing part of it, such as the <I>psbA</I> gene <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/8053946">Efimov VA, Reverdatto SV, Beilinson BA, Fradkov AF, Chakhmakhcheva OG. [Expression of the gene coding for the D1-protein of barley photosystem II in Escherichia coli]. Bioorganicheskaia khimiia. 1994;20(5):524-35.</A></SPAN></SPAN> and the 18-kDa protein of photosystem II <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/7757336">Kuwabara T, Takeuchi M, Honda S, Nakajima N, Watanabe A, Kondo N. Expression in Escherichia coli of the extrinsic 18-kDa protein of photosystem II of spinach. Plant &amp; cell physiology. 1995;36(3):435-9.</A></SPAN></SPAN>, both of which was successful. 

An alternative process, in which a diverse array of phototrophic bacteria and archaea harvest energy from light, is through a retinal-containing protein, called proteorhodopsin, that catalyse light-activated proton efflux across the cell membrane and thereby drive ATP synthesis. In contrast to photosystems, the process involving proteorhodopsin is anoxygenic and generates no NADPH vital for the Calvin cycle to proceed <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892948/">Walter JM, Greenfield D, Bustamante C, Liphardt J. Light-powering Escherichia coli with proteorhodopsin. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(7):2408-12.</A></SPAN></SPAN>. The heterologous expression of this light-powered proton pump in <I>E. coli</I> enabled photophosphorylation when the bacteria were exposed to light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/">Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(13):5590-5.
</A></SPAN></SPAN>
, and even generated a proton motive force, which turned the flagellar motor, yielding light-dependent motility <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892948/">Walter JM, Greenfield D, Bustamante C, Liphardt J. Light-powering Escherichia coli with proteorhodopsin. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(7):2408-12.</A></SPAN></SPAN>.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><P class="P-Larger"><B>Approach</B></P><P>In order to engineer <I>E. coli</I> in the outer chamber to <SPAN class="highlighted">turn atmospheric CO<SUB>2</SUB> into cellulose</SPAN>, the carbon first needs to be fixated by the bacteria. This requires the heterologous expression of the genes encoding the three enzymes <SPAN class="highlighted">RuBisCo, SBPase, and PRK</SPAN>. Furthermore, the implementation of the carboxysome from the <I>cso</I> operon can increase the levels of carbon fixation. The <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec" target="_blank">2014 Bielefeld iGEM team</A> had worked with a similar approach in their project. In an endeavour to optimise the carbon fixation process, our project build upon their experiences. The assembly of the individual parts into a composite part, <A href="http://parts.igem.org/Part:BBa_K2449030" target="_blank">BBa_K2449030</A>, was achieved, however, the cloning of these parts with a promoter proved problematic. Consequently, it was decided to prioritise other aspects of the project and therefore <SPAN class="highlighted">keep this part theoretical henceforth</SPAN>. For further information about our approach, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#co2-fixation-approach">read here</SPAN>.
</P><DIV class="modal fade" id="co2-fixation-approach" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Carbon Fixation</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P><B>Engineering <I>E. coli</I> to Perform the Calvin Cycle</B>
In order to engineer <I>E. coli</I> in the outer chamber to turn atmospheric CO<SUB>2</SUB> into cellulose, the carbon first needs to be fixated by the bacteria. This requires the heterologous expression of the genes encoding the three enzymes RuBisCo, SBPase, and PRK. In correspondence with the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec" target="_blank">2014 Bielefeld iGEM team</A>, who worked with a similar subpart of their project, we discussed the cloning of these genes and received two crucial parts. The first of this was <A href="http://parts.igem.org/Part:BBa_K1465214" target="_blank">BBa_K1465214</A>, containing RubisCO from <I>Halothiobacillus neapolitanus</I> and PRK from <I>Synechococcus elongatus</I> under the control of a composite promoter controllable by IPTG. The second was <A href="http://parts.igem.org/Part:BBa_K1465228" target="_blank">BBa_K1465228</A>, which contained SBPase from <I>Bacillus methanolicus</I>. Through the concurrent expression of these parts in <I>E. coli</I>, all enzymes required to fixate CO<SUB>2</SUB> through the Calvin cycle were present in the cells.

The first approach was to assemble both parts on one plasmid by molecular cloning, as shown on Figure 1. In doing this, it was discovered that the <A href="http://parts.igem.org/Part:BBa_K1465214" target="_blank">BBa_K1465214</A> part was missing roughly 1 kbp when digested with standard restriction enzymes with recognition sites within the BioBrick prefix and suffix. Sequencing of the part revealed that the part was missing the entire promoter sequence of 1239 bp. Consequently, the assembly of the parts additionally required the cloning of a promoter. For this purpose, the Tac-promoter (Ptac) from the part <A href="http://parts.igem.org/Part:BBa_K864400" target="_blank">BBa_K864400</A> was chosen, as this is commonly used to overexpress genes in <I>E. coli</I>. With the objective to place the two Calvin cycle parts on different vectors, it was attempted to clone Ptac in front of both parts. However, this did not succeed and it was decided to prioritise other aspects of the project and therefore keep this part theoretical henceforth.
<B>Implementing the Carboxysome in <I>E. coli</I> to Increase Carbon Fixation Efficiency</B>
The implementation of the microcompartment carboxysome in the test organism can increase the efficiency of the carbon fixation process substantially. As for the Calvin cycle parts, we corresponded with the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec" target="_blank">2014 Bielefeld iGEM team</A> on their experience of the implementation of the carboxysome, and received the two parts that together contained the <I>cso</I> operon from <I>Halothiobacillus neapolitanus.</I> These parts were  <A href="http://parts.igem.org/Part:BBa_K1465204" target="_blank">BBa_K1465204</A>, containing <I>csoS2</I>, and  <A href="http://parts.igem.org/Part:BBa_K1465209" target="_blank">BBa_K1465209</A>, containing <I>csoS3</I>, <I>csoS14</I>, and <I>csoS1D</I>. As part of the optimisation, we aimed to combine these parts into one part containing the entire <I>cso</I> operon, as shown on Figure 1. Furthermore, a promoter was required to drive the gene expression. For this purpose, Ptac was chosen, as for the genes encoding the Calvin cycle enzymes. </P><DIV class="figure-text"><P><B>Figure 1.</B> The envisaged design of the carboxysomal genes, encoded in the composite part, <A href="http://parts.igem.org/Part:BBa_K2449030" target="_blank">BBa_K2449030</A>, under control of the Tac-promoter.</P></DIV><P>We succeeded in assembling both carboxysome parts in the composite part, <A href="http://parts.igem.org/Part:BBa_K2449030" target="_blank">BBa_K2449030</A>, but for some inexplicable reason, this composite part likewise emerged difficult to clone with the Tac-promoter. Therefore, it was decided to focus at other aspects of the project and, as for the Calvin cycle part, keep this part theoretical henceforth. 

A fitting approach to verify the expression of the genes involved in carbon fixation, could consist of matrix-assisted laser desorption/ionization mass spectrometry (MALDI MS) and quantitative PCR (qPCR). For this, one could build upon the experience obtained by the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec" target="_blank">2014 Bielefeld iGEM team</A>. 

In order to test the functionality of the parts we had hoped to assemble, we had reached out to <A href="http://www.nordcee.dk/" target="_blank">Nordcee</A>, which is a major research group within the Department of Biology at SDU. Our hope was to compare the carbon fixation rates for cultures expressing the Calvin cycle enzymes alone and in combination with the carboxysome, whereby the effectivity of the microcompartment also would have been assessed. In cooperation with Nordcee, measurements of CO<SUB>2</SUB>  decrease in an airtight culture would have been carried out with an infrared CO<SUB>2</SUB>analyser. Another test approach would have been using <SUP>14</SUP>CO<SUB>2</SUB>, and then measure the amount our organism had taken in. 
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-design">Click here to return to the Project Design overview.</A></I></P></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-design-cellulose-biosynthesis"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Cellulose Biosynthesis</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B>Introduction</B></P><P><SPAN class="highlighted">Bacterial cellulose is one of the most abundant biopolymers produced by different species of gram-negative bacteria</SPAN>, especially by <I>Acetobactors</I>.  <I>Glucoacetobacter xylinus</I> is a bacterial species that produces cellulose of high quality in large quantities<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://doi.org/10.1007/s10570-013-9994-3">Lin, SP., Loira Calvar, I., Catchmark, J.M. et al. Cellulose (2013) 20: 2191.</A></SPAN></SPAN>. Cellulose is produced from the resource glucose-6-phosphate. This phosphorylated glucose is a key intermediate in the core carbon metabolism of bacteria given its importance in glycolysis, gluconeogenesis and the pentose phosphate pathway <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.amazon.com/Prescotts-Microbiology-Joanne-Willey/dp/0073402400">Joanne Willey LS, Christopher J. Woolverton. Prescott’s Microbiology. 9th edition 2014.</A></SPAN></SPAN>. Even though the pathway, where glucose and glucose-6-phosphate is converted into cellulose, only includes few steps, it requires a great amount of energy. Not only does the cell spend energy on forming UDP-glucose for cellulose biosynthesis, it also uses glucose, which otherwise would have contributed to generation of ATP <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27247386">Florea M, Hagemann H, Santosa G, Abbott J, Micklem CN, Spencer-Milnes X, et al. Engineering control of bacterial cellulose production using a genetic toolkit and a new cellulose-producing strain. Proceedings of the National Academy of Sciences of the United States of America. 2016;113(24):E3431-40.</A></SPAN></SPAN>.
 <SPAN class="highlighted">The ability for <I>G. xylinus</I> to produce cellulose nanofibers from UDP-glucose, crystallize, and secrete it, is controlled by genes in the Acetobacter cellulose synthase (acs) operon <I>acsABCD</I></SPAN>. This operon encodes four different proteins: AcsA, AcsB, AcsC and AcsD. A dimer, known as AcsAB, is formed by a catalytic domain, AcsA, and a regulatory domain, AcsB. This dimer is responsible for synthesising the cellulose nanofibers from UDP-glucose, whereas AcsC and AcsD secrete cellulose and form an interconnected cellulose pellicle around the cells <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://link-springer-com.proxy1-bib.sdu.dk/article/10.1007%2Fs10570-014-0521-y">Mehta K, et al. Characterization of an acsD disruption mutant provides additional evidence for the hierarchical cell-directed self-assembly of cellulose in Gluconacetobacter xylinus. Cellulose. 2014;22:119–137.</A></SPAN></SPAN>, as illustrated in Figure 6.
</P><DIV class="figure-text"><P><B>Figure 6.</B> The AcsAB dimer synthesises cellulose nanofibers. AcsC and AcsD mediate the secretion and formation of an interconnected cellulose pellicle.</P></DIV><P>Other genera, including some <I>E. coli</I> strains, secrete cellulose as a component of their biofilm. Even though cellulose biosynthesis is intrinsic to <I>E. coli</I>, the quantity of the production is incomparable to cellulose biosynthesis in <I>G. xylinus</I>. Indigenously, <I>E. coli</I> is not capable of degrading cellulose into a metabolisable energy source <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26452465">Gao D, Luan Y, Wang Q, Liang Q, Qi Q. Construction of cellulose-utilizing Escherichia coli based on a secretable cellulase. Microbial Cell Factories. 2015;14:159.</A></SPAN></SPAN>. However, if this structural and water-holding polymer is enzymatically degraded, first into cellobiose and then to glucose residues, the cellulose polymer is a potent source of energy <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17244702"> Arai T, Matsuoka S, Cho HY,
Yukawa H, Inui M, Wong SL, et al. Synthesis of Clostridium cellulovorans
minicellulosomes by intercellular complementation. Proceedings of the National
Academy of Sciences of the United States of America. 2007;104(5):1456-60.</A></SPAN></SPAN>.
</P><P class="P-Larger"><B>Approach</B></P><P style="width:100%;">
To <SPAN class="highlighted">link the two bacterial compartments of the PowerLeaf</SPAN>, an efficient way to <SPAN class="highlighted">store the harvested energy</SPAN> was required. Through research, we found that storing the chemical energy in <SPAN class="highlighted">cellulose would be a suitable approach</SPAN>, since this is a polysaccharide that bacteria normally are unable to degrade <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26452465">Gao D, Luan Y, Wang Q, Liang Q, Qi Q. Construction of cellulose-utilizing Escherichia coli based on a secretable cellulase. Microbial Cell Factories. 2015;14:159.</A></SPAN></SPAN>. After looking into earlier iGEM projects it was found that the 
<A href="https://2014.igem.org/Team:Imperial" target="_blank">2014 project Aqualose from Imperial College London</A>, had worked with optimisation of cellulose biosynthesis in <I>E. coli</I>. Our aim was to <SPAN class="highlighted">enhance cellulose biosynthesis in <I>E. coli</I> MG1655</SPAN>, which naturally secretes small amounts of cellulose as a part of its biofilm <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18599830">Gualdi L, Tagliabue L, Bertagnoli S, Ierano T, De Castro C, Landini P. Cellulose modulates biofilm formation by counteracting curli-mediated colonization of solid surfaces in Escherichia coli. Microbiology (Reading, England). 2008;154(Pt 7):2017-24.</A></SPAN></SPAN>. This would be achieved by the cloning of plasmids containing the <SPAN class="highlighted">cellulose synthase operon <I>acsABCD</I></SPAN>, utilising the two parts <A href="http://parts.igem.org/Part:BBa_K1321334" target="_blank">BBa_K1321334</A> and <A href="http://parts.igem.org/Part:BBa_K1321335" target="_blank">BBa_K1321335</A>, constructed by Imperial College London 2014. This would enhance the cellulose biosynthesis and thereby optimise the energy outcome of the entire system in our project.
Due to cloning difficulties, it was decided to prioritise other aspects of the project and therefore <SPAN class="highlighted">keep this part theoretical henceforth</SPAN>. For further information about the cellulose biosynthesis approach, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#cellulose-production-approach">read here</SPAN>.
</P></DIV></DIV><DIV class="modal fade" id="cellulose-production-approach" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Cellulose Biosynthesis</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P><B>Optimised Cellulose Biosynthesis in <I>E. coli</I></B>
To enhance the cellulose biosynthesis, parts containing the coding sequence of the Cellulose Synthase enzyme were cloned into the <I>E. coli</I> strain MG1655. The original idea was to clone the entire cellulose synthase operon <I>acsABCD</I> into one vector under control of the Ptac, as seen in Figure 1. With a plasmid with a total length over 9000 bp, the cloning emerged difficult. After several unsuccessful attempts, a different approach was sought.
</P><DIV class="figure-text"><P><B>Figure 1.</B> The <I>acsABCD</I> operon controlled by Ptac cloned into one high copy vector.</P></DIV><P>In this design, it was attempted to implement the <I>acsABCD</I> operon into <I>E. coli</I> MG1655 on two separate vectors, with both parts controlled by Ptac, <A href="http://parts.igem.org/Part:BBa_K864400" target="_blank">BBa_K864400</A>, ensuring equal expression levels of the parts. The AcsAB dimer, encoded in the part <A href="http://parts.igem.org/Part:BBa_K1321334" target="_blank">BBa_K1321334</A>, was attempted to be inserted into the vector pSB1C3. The part <A href="http://parts.igem.org/Part:BBa_K1321335" target="_blank">BBa_K1321335</A>, containing AcsC and AcsD, was inserted into the vector pSB1A3. Several combinations of the two parts and different vectors carrying different resistance cassettes were attempted, but unfortunately without success. Correspondence with a supervisor from the <A href="https://2014.igem.org/Team:Imperial" target="_blank">Imperial College London team</A>, revealed that cloning with these parts had emerged difficult for them as well. Due to time constraints, it was decided to prioritise other aspects of the project and therefore keep this part theoretical henceforth. 

If the cloning of the <I>acsABCD</I> operon had been successful, the cellulose biosynthesis would have been tested using fluorescent brightener 28. This is a colourless organic compound that fluoresces with a bright blue color under ultraviolet radiation, and it is used as a fluorescent brightening agent for polyamide and cellulose fabrics. Fluorescent brightener 28 binds non-specifically to polysaccharides with β-1,3 and β-1,4-linkages, of which the latter is present in cellulose <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.pern-brio.eu/protocols/protocol-calcofluor-mut.pdf">Staining of fungal hyphae and propagules with fluorescent brightener</A></SPAN></SPAN>.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-design">Click here to return to the Project Design overview.</A></I></P></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-design-cellulose-breakdown"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Breakdown of Cellulose</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B><SPAN class="highlighted">Introduction</SPAN></B></P><P class="">
Cellulose is a natural biopolymer used for a vast variety of biological purposes and it is most commonly found in plants, where it serves as the main structural component. Since plants are primary producers, many organisms of the Earth’s ecosystems have adapted accordingly<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://mmbr.asm.org/content/66/3/506.long"> Lynd
LR, Weimer PJ, van Zyl WH, Pretorius IS. Microbial Cellulose Utilization:
Fundamentals and Biotechnology. Microbiology and Molecular Biology Reviews.
2002;66(3):506-77.</A></SPAN></SPAN>. One of the key evolutionary features for the primary consumers was the development of the ability to <SPAN class="highlighted">degrade cellulose into glucose, which could then be used as a cellular fuel. </SPAN>A simple organism, able to efficiently do so, is the <I>Cellulomonas fimi</I>, which converts cellulose to glucose in a two-step process, with cellobiose as the intermediate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3403577/"> Jung
SK, Parisutham V, Jeong SH, Lee SK. Heterologous Expression of Plant Cell Wall
Degrading Enzymes for Effective Production of Cellulosic Biofuels. Journal of Biomedicine and Biotechnology. 2012;2012.</A></SPAN></SPAN>.
</P><P class=""><B>Breakdown of Cellulose to Cellobiose</B>
Cellulose is a long polysaccharide consisting of β-1,4-linked <SMALL>D</SMALL>-glucose units and many organisms, including <SPAN class="highlighted"><I>E. coli</I>, lack the enzymes able to degrade these strong β-linkages. To overcome this, the <I>C. fimi</I> has developed two cellulases, namely the endo-β-1,4-glucanase and exo-β-1,4-glucanase, respectively encoded by the <I>cenA</I> and <I>cex</I> genes <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3403577/"> Jung
SK, Parisutham V, Jeong SH, Lee SK. Heterologous Expression of Plant Cell Wall
Degrading Enzymes for Effective Production of Cellulosic Biofuels. Journal of Biomedicine and Biotechnology. 2012;2012.</A></SPAN></SPAN>.</SPAN> The endoglucanase is able to randomly degrade the amorphous structure of cellulose, thereby allowing the exoglucanase to cleave the β-1,4-linkages at every other <SMALL>D</SMALL>-glucose unit. Thus, disaccharides are released in the form of cellobiose <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/9134758/"> Lam TL, Wong RS, Wong WK.
Enhancement of extracellular production of a Cellulomonas fimi exoglucanase in
Escherichia coli by the reduction of promoter strength. Enzyme and microbial
technology. 1997;20(7):482-8.</A></SPAN></SPAN>, as illustrated in Figure 7. <SPAN class="highlighted">Cellulose itself is too large to be transported across the bacterial cell membrane</SPAN>, and therefore, the breakdown of cellulose into cellobiose must take place in the extracellular fluid.
</P><DIV class="figure-text"><P><B>Figure 7.</B> Degradation of the β-1,4-linkages in cellulose mediated by the enzymes endo-β-1,4-glucanase and exo-β-1,4-glucanase, thereby creating cellobiose.
</P></DIV><P class=""><B>The α-Hemolysin Transport System</B>
The ɑ-hemolysin transport system is an <SPAN class="highlighted">ABC transporter complex consisting of three proteins, namely the outer membrane protein TolC, hemolysin B (HlyB), and hemolysin D (HlyD)</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11755084"> Gentschev I, Dietrich G, Goebel W. The E. coli
alpha-hemolysin secretion system and its use in vaccine development. Trends in
microbiology. 2002;10(1):39-45.</A></SPAN></SPAN>, which can effectively transport intracellular hemolysin A (HlyA) to the extracellular fluid. Utilising a linker peptide, the protein of interest can be fused with HlyA. <SPAN class="highlighted">Once a protein is HlyA-tagged, it can be recognised by the ATP-binding cassette HlyB, which will initiate transportation of the HlyA-tagged protein to the extracellular fluid, as seen in Figure 8<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11755084"> Gentschev I, Dietrich G, Goebel W. The E. coli
alpha-hemolysin secretion system and its use in vaccine development. Trends in
microbiology. 2002;10(1):39-45.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22239833"> Su
L, Chen S, Yi L, Woodard RW, Chen J, Wu J. Extracellular overexpression of
recombinant Thermobifida fusca cutinase by alpha-hemolysin secretion system in
E. coli BL21(DE3). Microbial Cell Factories. 2012;11:8.</A></SPAN></SPAN></SPAN>.
</P><DIV class="figure-text"><P><B>Figure 8.</B> The enzymes encoded by the <I>cenA</I> and <I>cex</I> genes are linked to HlyA. HlyB recognises HlyA and initiates transportation of the HlyA-tagged protein from the cytosol to the extracellular fluid.
</P></DIV><P class=""><B>Uptake of Cellobiose</B>
While cellulose is too large to be pass the cell membrane, <SPAN class="highlighted">transportation of cellobiose is a common feature found in many organisms. An example is <I>E. coli</I></SPAN> that utilises the membrane protein lactose permease (LacY)<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294459/"> Sekar R, Shin HD, Chen R. Engineering Escherichia coli Cells for Cellobiose Assimilation through a
Phosphorolytic Mechanism. Applied and Environmental Microbiology.
2012;78(5):1611-4.</A></SPAN></SPAN>, whereby the cellobiose is enzymatically catabolised in the cytosol.
</P><P class=""><B>Degradation of Cellobiose to Glucose</B>
Through evolutionary events, many organisms have developed the ability to express enzymes, capable of breaking the β-linkage in cellobiose. <SPAN class="highlighted"><I>E. coli</I> expresses the periplasmic β-glucosidase encoded by the <I>bglX</I> gene, which is known to have said feature, hydrolysing the cellobiose β-linkage<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" http://www.uniprot.org/uniprot/P33363">UniProt entry for <I>bglX</I></A></SPAN></SPAN>.</SPAN><I>Saccharophagus degradans</I> expresses <SPAN class="highlighted">a different enzyme that efficiently cleaves the β-linkage in cellobiose, namely cellobiose phosphorylase encoded by the <I>cep94A</I> gene.</SPAN> This enzyme phosphorylates the cellobiose at its β-linkage, resulting in the degradation of cellobiose to <SMALL>D</SMALL>-glucose and α-<SMALL>D</SMALL>-glucose-1-phosphate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294459/"> Sekar R, Shin HD, Chen R. Engineering Escherichia coli Cells for Cellobiose Assimilation through a
Phosphorolytic Mechanism. Applied and Environmental Microbiology.
2012;78(5):1611-4.</A></SPAN></SPAN>, as seen in Figure 9. 
</P><DIV class="figure-text"><P><B>Figure 9. </B>Phosphorylation of the β-1,4-linkages in cellobiose by the enzyme cellobiose phosphorylase, thereby producing <SMALL>D</SMALL>-glucose and α-<SMALL>D</SMALL>-glucose-1-phosphate.
</P></DIV><P class="P-Larger"><B>Approach</B></P><P class=""><B>Cellulose to Cellobiose</B>
In the endeavour to engineer <I>E. coli</I> to utilise cellulose as its only carbon source, inspiration was drawn from the <A href="https://2008.igem.org/Team:Edinburgh" target="_blank">Edinburgh 2008 iGEM team</A> project, who developed two BioBricks containing the <I>cenA</I> and <I>cex</I> genes. In this project, the α-hemolysin transport system was utilised by creating HlyA-tagged endo- and exo-β-1,4-glucanases, using a peptide linker. To implement this system in <I>E. coli</I>, heterogeneous expression of <I>hlyB</I>, <I>hlyD</I>, <I>cenA-hlyA</I> and <I>cex-hlyA</I> was required.

To achieve  this, DNA synthesis of <I>cenA</I> and <I>cex</I> was ordered, each tagged with HlyA. The genes encoding HlyB and HlyD were retrieved from the part <A href="http://parts.igem.org/Part:BBa_K1166002" target="_blank">BBa_K1166002</A> by phusion PCR. Using the resulting PCR product, the following construct was composed for the degradation of cellulose into cellobiose, as illustrated in Figure 10.
</P><DIV class="figure-text"><P><B>Figure 10. </B><A href="http://parts.igem.org/Part:BBa_K2449026" target="_blank">BioBrick</A>, containing the genes <I>cenA</I>, <I>cex</I>, <I>hlyB</I>, and <I>hlyD</I> controlled by PenI-regulated promoters.
</P></DIV><P class=""><B>Cellobiose to Glucose</B>
The <A href="https://2011.igem.org/Team:Edinburgh" target="_blank">Edinburgh 2011 iGEM team</A> created a BioBrick with the <I>bglX</I> gene, which is endogenous to <I>E. coli</I>, in the endeavour to increase the efficiency of the degradation of cellobiose to glucose. However, it seems that the enzymatic activity of the periplasmic β-glucosidase has faded as a result of evolution, rendering <I>E. coli</I> incapable of surviving solely on cellobiose. Thus, even though <I>E. coli</I> can absorp cellobiose, it is not able to survive with this as its only carbon source.

As a solution to this, a part containing the <I>cep94A</I> gene was synthesised, with the intend to enable <I>E. coli</I> to survive solely on cellobiose. Thus, a construct containing <I>cep94A</I> controlled by a LacI-regulated promoter was composed, as illustrated in Figure 11. 

</P><DIV class="figure-text"><P><B>Figure 11.</B><A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank">BioBrick</A> comprising <I>cep94A</I> controlled by a LacI-regulated promoter. This part was cloned into both a high and low copy vector.
</P></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-results-cellulose-breakdown">Click here if you wish to go directly to the Project Demonstration &amp; Results section of the Breakdown of Cellulose.</A></I></P></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-design">Click here to return to the Project Design overview.</A></I></P></DIV><DIV class="row padding-top-125" id="project-design-extracellular-electron-transfer"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Extracellular Electron Transfer</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B><SPAN class="highlighted">Introduction</SPAN></B></P><P class=""><B>Microbial Fuel Cell</B>
Electrochemical devices, such as batteries and fuel cells, are broadly used in electronics to convert chemical energy into electrical energy. <SPAN class="highlighted">A Microbial Fuel Cell (MFC) is an open system electrochemical device</SPAN>, consisting of two chambers; an anode chamber and a cathode chamber, which are separated by a proton exchange membrane, as illustrated in Figure 12. Both the anode and the cathode in an MFC can use various forms of graphite as base material and in the anode chamber, <SPAN class="highlighted">microbes are utilised as catalysts to convert organic matter into metabolic products, protons, and electrons <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.wiley.com//legacy/wileychi/li1/">Khanal YLaSK. Microbial Fuel Cells, Capter 19. In: Khanal. EbYLaSK, editor. Bioenergy: Principles and Applications. First Edition ed: Published 2017 by John Wiley &amp; Sons, Inc.; 2016.</A></SPAN></SPAN></SPAN>. This is carried out through metabolic pathways such as glycolysis, thereby generating ATP needed to maintain cellular life. This metabolic pathway also releases electrons, which are carried by NAD<SUP>+</SUP> in its reduced form, NADH. 
</P><DIV class="figure-text"><P><B>Figure 12.</B> A microbial fuel cell utilising glucose as substrate. The glucose is consumed to protons, electrons, and CO<SUB>2</SUB>. The electrons are transferred to the anode while the protons diffuse over the proton exchange membrane. A gradient causes the electrons to flow through an external load to the cathode, which generates an electrical current.
</P></DIV><P>
Under aerobic conditions, the generated NADH will deliver its electron as part of the electron transfer chain, thereby returning to its oxidised form NAD<SUP>+</SUP>. Under <SPAN class="highlighted">anaerobic conditions</SPAN> the electron transport chain will be unable to continue, which will cause the generated NADH to accumulate, and as a consequence, the concentration of available NAD<SUP>+</SUP> for glycolysis will decrease. This will drive the cell to carry out other metabolic pathways, such as fermentation, in order to maintain its ATP levels. Instead, the accumulating NADH generated <SPAN class="highlighted">under anaerobic conditions, can be utilised to drive an electrical current by depositing the retrieved electrons to an anode coupled with an appropriate cathode</SPAN>. The cathode catalyst in an MFC will usually catalyse the reaction of 2 H<SUP>+</SUP> + ½ O<SUB>2</SUB> per H<SUB>2</SUB>O. <SPAN class="highlighted">The transfer of electrons from NADH to the anode can be executed in three different ways, as shown in Figure 13: redox shuttles, direct contact electron transfer, and bacterial nanowires</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16999087"> Logan BE, Hamelers B, Rozendal R, Schroder U, Keller J, Freguia S, et al. Microbial fuel cells: methodology and technology. Environmental science &amp; technology. 2006;40(17):5181-92.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.wiley.com//legacy/wileychi/li1/">Khanal YLaSK. Microbial Fuel Cells, Capter 19. In: Khanal. EbYLaSK, editor. Bioenergy: Principles and Applications. First Edition ed: Published 2017 by John Wiley &amp; Sons, Inc.; 2016.</A></SPAN></SPAN>.</P><DIV class="figure-text"><P><B>Figure 13.</B> Three different ways to transfer electrons from microorganisms to an anode. a) Transfer of electrons to the anode using a redox shuttle (Med). Two different types of redox shuttles exit: One going through the membrane and another receiving electrons from membrane proteins. b) Transfer of electrons to the anode by direct contact. c) Electrons are carried from the inside of the cell, directly to the anode through nanowires. 
</P></DIV><P>
The redox shuttles use extracellular electron mediators, which hold the advantage of not being limited by the surface area of the anode, although it is restricted by the slow diffusion of the extracellular mediators. The direct contact electron transfer is, in contrast to the redox shuttles, strongly limited by the surface area of the anode, but the membrane bound cytochromes that are in direct contact with the anode, rapidly deliver the electrons. <SPAN class="highlighted">Bacterial nanowires are known to efficiently transfer electrons</SPAN>, as for the direct contact electron transfer. However, bacterial nanowires are not as strictly limited by the surface area of the anode as the direct contact electron transfer. This is due to the ability of bacterial nanowires to form complex networks of interacting nanowires in biofilm, thereby efficiently transferring electrons from distant microbes to the anode<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.wiley.com//legacy/wileychi/li1/">Khanal YLaSK. Microbial Fuel Cells, Capter 19. In: Khanal. EbYLaSK, editor. Bioenergy: Principles and Applications. First Edition ed: Published 2017 by John Wiley &amp; Sons, Inc.; 2016.</A></SPAN></SPAN>. 
</P><P><B>Bacterial Nanowires</B>
Nanowires are long electrically conductive pili found on the surface of various microorganisms, such as the metal reducing <I>Geobacter sulfurreducens</I>. <SPAN class="highlighted"><I>G. sulfurreducens</I> utilises nanowires to transfer accumulating electrons retrieved from metabolism, to metals in the nearby environment</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1392927/"> Mahadevan R, Bond DR, Butler JE, Esteve-Nuñez A, Coppi MV, Palsson BO, et al. Characterization of Metabolism in the Fe(III)-Reducing Organism Geobacter sulfurreducens by Constraint-Based Modeling. Applied and Environmental Microbiology. 2006;72(2):1558-68.</A></SPAN></SPAN>. This gram-negative bacteria is strictly anaerobic, as it is unable to transfer its electrons to the environment in the presence of the highly reducing oxygen. Nanowires found in <I>G. sulfurreducens</I> are type IV pili polymer chains composed of pilA monomers, and they can reach a length of nearly 10 mm<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://jb.asm.org/content/194/10/2551.full"> Richter LV, Sandler SJ, Weis RM. Two Isoforms of Geobacter sulfurreducens pilA Have Distinct Roles in Pilus Biogenesis, Cytochrome Localization, Extracellular Electron Transfer, and Biofilm Formation. Journal of Bacteriology. 2012;194(10):2551-63.</A></SPAN></SPAN>. The proteins required for the effective transfer of electrons by nanowires is a complex and poorly understood system, which includes an extensive series of c-type cytochromes, as shown in Figure 14<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.biochemsoctrans.org/content/40/6/1295"> Morgado L, Fernandes AP, Dantas JM, Silva MA, Salgueiro CA. On the road to improve the bioremediation and electricity-harvesting skills of Geobacter sulfurreducens: functional and structural characterization of multihaem cytochromes. Biochemical Society transactions. 2012;40(6):1295-301.</A></SPAN></SPAN>.</P><DIV class="figure-text"><P><B>Figure 14.</B> The electrons from NADH are transferred to menaquinone (MQ), reducing it to menaquinol (MQH<SUB>2</SUB>), the inner membrane-associated MaCA cytochrome receives the electrons and reduces the periplasmic triheme cytochromes (PpcA-PpcE). The electrons are mediated to the outer membrane-associated cytochromes, OmcB and OmcE, and further transferred to cytochromes on the pili <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.biochemsoctrans.org/content/40/6/1295"> Morgado L, Fernandes AP, Dantas JM, Silva MA, Salgueiro CA. On the road to improve the bioremediation and electricity-harvesting skills of Geobacter sulfurreducens: functional and structural characterization of multihaem cytochromes. Biochemical Society transactions. 2012;40(6):1295-301.</A></SPAN></SPAN>.
</P></DIV><P><SPAN class="highlighted">The electrical conductivity of the nanowires in <I>G. sulfurreducens</I> can be optimised by exchanging endogenous <I>pilA</I> with heterologous <I>pilA</I></SPAN> rich in aromatic amino acids. Tan Yang et. al  2017<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://mbio.asm.org/content/8/1/e02203-16.full"> Tan Y, Adhikari RY, Malvankar NS, Ward JE, Woodard TL, Nevin KP, et al. Expressing the Geobacter metallireducens pilA in Geobacter sulfurreducens Yields Pili with Exceptional Conductivity. mBio. 2017;8(1).</A></SPAN></SPAN><SPAN class="highlighted">heterogeneously expressed <I>pilA</I> from <I>G. metallireducens</I> in <I>G. sulfurreducens</I>, which increased the electrical conductivity of the recombinant bacteria 5000-fold</SPAN>. This optimisation holds great potential in the development of highly efficient bacterial strains for MFCs. With the intention of optimising an MFC, <I>G. sulfurreducens</I> is a lot easier to work with than <I>G. metallireducens</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://mbio.asm.org/content/8/1/e02203-16.full"> Tan Y, Adhikari RY, Malvankar NS, Ward JE, Woodard TL, Nevin KP, et al. Expressing the Geobacter metallireducens pilA in Geobacter sulfurreducens Yields Pili with Exceptional Conductivity. mBio. 2017;8(1).</A></SPAN></SPAN>, since <I>G. metallireducens</I> has a longer generation time.
</P><P class="P-Larger"><B>Approach</B></P><P style="width:100%;"><SPAN class="highlighted">Originally, it was intended to implement bacterial nanowires from <I>G. sulfurreducens</I> into <I>E. coli</I></SPAN>. Through research, it was found that the <A href="https://2013.igem.org/Team:Bielefeld-Germany" target="_blank">Bielefeld iGEM team from 2013</A> had come to the conclusion, that <SPAN class="highlighted">this task was too comprehensive to undertake in the limited time of an iGEM project</SPAN>. However, a different approach was deviced, as postdoc Oona Snoeyenbos-West suggested us to use <I>G. sulfurreducens</I> as the model organism for our MFC. 
<SPAN class="highlighted">It was then decided to work on optimisation of the <I>G. sulfurreducens</I> by increasing the electrical conductivity of its endogenous nanowires. To achieve this, synthesis of the <I>pilA</I> genes from <I>G. metallireducens</I></SPAN> was ordered, which was used to create a BioBrick. Using the same approach for homologous recombination as in the dormancy system, a DNA fragment containing the chloramphenicol resistance cassette of the pSB1C3 vector, was made for later selection of recombinant <I>G. sulfurreducens</I>. The PCR product was ligated with fragments retrieved from the 500 bp upstream and downstream regions of the chromosomal <I>pilA</I> genes of the <I>G. sulfurreducens</I> PCA strain, creating the fragment seen in Figure 15. 
</P><DIV class="figure-text"><P><B>Figure 15.</B> The linear DNA fragment intended for homologous recombination into <I>G. sulfurreducens</I>. 
</P></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-results-extracellular-electron-transfer">Click here if you wish to go directly to the Project Demonstration &amp; Results section of the Extracellular Electron Transfer.</A></I></P></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#project-design">Click here to return to the Project Design overview.</A></I></P></DIV></DIV></DIV></DIV><DIV class="topShadow" id="demonstration-and-results"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-project"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Demonstration &amp; Results</SPAN></H2></DIV></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-results-dormancy-system" style="margin-top:-125px;"><DIV class="col-xs-12"><DIV class=""><DIV class="project-design-headline"><H2>Dormancy System</H2></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P class="P-Larger"><B><SPAN class="highlighted">Determination of Noise Levels in Constitutive Promoter Family Members</SPAN></B></P><P class=""><SPAN class="highlighted"><B>Fluorescence microscopy and flow cytometry revealed that a strong constitutive promoter was suitable for the expression of the photocontrol device</B></SPAN>. 
To assess which of the constitutive promoters would be suitable for a uniform expression of the photocontrol device, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>, and the antitoxin RelB, <A href="http://parts.igem.org/Part:BBa_K2449028" target="_blank">BBa_K2449028</A>, <SPAN class="highlighted">the expression levels and the noise of four different members of the <A href="http://parts.igem.org/Promoters/Catalog/Anderson" target="_blank">Anderson promoter collection</A> and their RFP reporter systems, were studied by fluorescence microscopy</SPAN>. These were, in increasing promoter strength, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, and <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>. 
Additionally, the change in RFP expression levels and noise during growth were tested for the promoters with the highest and lowest relative promoter strength by flow cytometry and qualitative analysis by fluorescence microscopy. Combining these two techniques, the expression and noise levels for the promoters were determined as follows:</P><UL class="list"><LI>The weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, exhibited a relatively low expression of RFP, indicating low gene expression and an increasing high level of noise throughout growth.</LI><LI>Both medium strength promoters, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A> and <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, displayed a moderate level of both noise and protein expression of the RFP reporter.</LI><LI>The strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, exhibited a comparatively high expression of the reporter RFP and an increasing high level of noise throughout growth. </LI></UL><P>For further information about the experiments and a proposed cause for the increasing noise level, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-noise-levels">read here.</SPAN><SPAN class="highlighted">As the strong constitutive promoter exhibited the most uniform expression, this was chosen to regulate the expression of the photocontrol device genes</SPAN>. With the <A href="https://2017.igem.org/Team:SDU-Denmark#modelling" target="_blank">modelling results</A> in mind, it was decided that the <I>relB</I> gene should be regulated by a tightly controllable uniform promoter, thereby ruling out the constitutive promoter family members as a possibility.
</P><P class="P-Larger"><B><SPAN class="highlighted">Expression Level of the Mnt- and Penl-Regulated Promoters</SPAN></B></P><P class=""><B><SPAN class="highlighted">The PenI-regulated promoter was chosen to control the expression of the photocontrol device, as it mediated a stronger expression than the Mnt-regulated promoter</SPAN>.</B>
The cloning of the strong constitutive promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, and the photocontrol device, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>, emerged problematic. For further information about these difficulties, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-mnt-penl">read here.</SPAN>
Consequently, the applicability of two different promoters was studied. These were the PenI-regulated, <A href="http://parts.igem.org/Part:BBa_R0074" target="_blank">BBa_R0074</A>, and the Mnt-regulated promoters, <A href="http://parts.igem.org/Part:BBa_R0073" target="_blank">BBa_R0073</A>, whose repressors are not found in <I>E. coli</I>, making the gene expression constitutive in this organism.
<SPAN class="highlighted">The relative expression and noise levels were quantitatively assessed using fluorescence microscopy of YFP reporter systems, <A href="http://parts.igem.org/Part:BBa_I6103" target="_blank">BBa_I6103</A></SPAN> and <A href="http://parts.igem.org/Part:BBa_I6104" target="_blank">BBa_I6104</A>, expressed on pSB1C3 in MG1655 at OD<SUB>600</SUB>=0.3-0.5. For this purpose, an Olympus IX83 with a photometrics prime camera was used, set to an excitation at 470 nm, emission at 515-560 nm, and exposure time at 20 ms.</P><DIV class="figure-text"><P><B>Figure 16.</B> Fluorescence microscopy of YFP under the control of PenI-regulated and Mnt-regulated promoters on pSB1C3 in <I>E. coli</I> MG1655 at OD<SUB>600</SUB>=0.3-0.5. </P></DIV><P>From the data obtained, it was evident that the PenI-regulated promoter mediated a substantially higher level of YFP expression than the Mnt-regulated promoter, as seen in Figure 16. Furthermore, the PenI-regulated promoter displayed a notably lower level of noise. <SPAN class="highlighted">On the basis of these results, the PenI-regulated promoter was chosen to control the expression of the photocontrol device</SPAN>.
</P><P>From the data obtained, it was evident that the PenI-regulated promoter mediated a substantially higher level of YFP expression than the Mnt-regulated promoter, as seen in Figure 16. Furthermore, the PenI-regulated promoter displayed a notably lower level of noise. <SPAN class="highlighted">On the basis of these results, the PenI-regulated promoter was chosen to control the expression of the photocontrol device</SPAN>.
</P><P class="P-Larger"><B><SPAN class="highlighted">Leaky Expression by the OmpR-Regulated Promoter on Different Vectors</SPAN></B></P><P><B><SPAN class="highlighted">Leaky expression by the OmpR-regulated promoter is reduced when cloned into a low copy vector compared to a high copy vector</SPAN>.</B>
Proper regulation of the OmpR-dependent promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>, is necessary for the implementation of a functional dormancy system, as the balance between RelE and RelB is imperative. To verify that the OmpR-regulated promoter is up to the task, a reporter system containing RFP under control of the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_M30011" target="_blank">BBa_M30011</A>, was cloned into <I>E. coli</I> strain SØ928 Δ<I>ompR</I>, lacking the OmpR transcription factor, on a high copy vector. By using a Δ<I>ompR</I> strain, the background generated by stimulation of the intrinsic OmpR system is removed, and the strain functions as a negative control. 
RFP expression was assessed by fluorescence microscopy using an Olympus IX83 with a photometrics prime camera, set to an excitation at 550 nm, emission at 573-613 nm, and exposure time at 500 ms. <SPAN class="highlighted">Assessing the RFP expression by fluorescence microscopy, it was discovered that the OmpR-regulated promoter mediated gene expression even in the absence of its transcription factor</SPAN>, see Figure 17. This observation was confirmed by going through the literature<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/16306980&quot;">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. </P><DIV class="figure-text"><P><B>Figure 17.</B> Fluorescence microscopy of RFP controlled by the OmpR-regulated promoter on a high copy vector in <I>E. coli</I> strain SØ928 Δ<I>ompR</I>. </P></DIV><P><SPAN class="highlighted">On the basis of this finding, the <I>relE</I> gene controlled by the OmpR-regulated promoter required a low copy plasmid or insertion into the chromosome. Protein expression of RFP in pSB1C3 with a copy number of 100-300 plasmids per cell, and pSB3K3 with a copy number of 10-12 plasmids per cell, was studied by flow cytometry</SPAN>. As for the determination of noise levels in the weak, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, and strong <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A> constitutive promoters, the experiment was carried out in both LB medium and M9 minimal medium, the latter supplemented with 0.2% glycerol. In the LB medium, selection was carried out by the addition of 30 µg/mL chloramphenicol, 30 µg/mL kanamycin, or 50 µg/mL ampicillin, depending on the resistance, and for M9 minimal medium, the concentrations used were 60 µg/mL chloramphenicol, 60 µg/mL kanamycin, and 100 µg/mL ampicillin. Excitation of RFP was at 561 nm, and emission was measured around 580 nm. Expression levels in both <I>E. coli</I> MG1655 and <I>E. coli</I> MG1655 Δ<I>ompR</I> were studied to determine the baseline of the leaky expression not influenced by intrinsic pathways including the OmpR transcription factor. </P><DIV class="figure-text"><P><B>Figure 18.</B>  Flow cytometric fluorescence measurements in arbitrary units as a function of time. <B>Left</B>: Cultures were grown in LB medium. <B>Right</B>: Cultures were grown in M9 minimal medium supplemented with 0.2% glycerol. Fluorescence of RFP expressed by the the OmpR-regulated promoter on the high copy vector, pSB1C3, and the low copy vector, pSB3K3, in MG1655 WT and Δ<I>ompR</I> MG1655 strain. All fluorescence levels were measured relative to the negative control WT <I>E. coli</I> MG1655, and the weak and strong constitutive promoters are included as references. Standard error of mean is shown, but are in several cases indistinguishable from the graph. </P></DIV><P>Fluorescence levels in the two different media display similar behavior, as seen in Figure 18. The main difference observed, was that the decrease in fluorescence over time was faster in LB medium than in M9 minimal medium, in concordance with the observations made in previous experiments. On a general level, the data revealed, that MG1655 cloned with the POmpR-RFP reporter system on the high copy vector exhibited a fluorescence level, equivalent to that mediated by the strong constitutive promoter. On the low copy vector, the POmpR-RFP reporter system yielded a fluorescence level comparable to the gene expression mediated by the weak constitutive promoter. On the other hand, expression levels in the MG1655 Δ<I>ompR</I> strain were markedly reduced compared to MG1655, indicating that pathways including the transcription factor OmpR interfere with RFP expression under these conditions. Again, the fluorescence levels observed for the POmpR-RFP reporter system on the low copy vector were distinctly lower than for the high copy vector. 
All things considered, the OmpR-regulated promoter was found to exhibit leaky expression comparable to the expression levels mediated by the constitutive promoters. When cloned into a low copy vector, the leaky expression was reduced prominently. Thus, <SPAN class="highlighted">to obtain proper regulation of RelE expression by the OmpR-dependent promoter, a low copy vector is required</SPAN>.
</P><P class="P-Larger"><B><SPAN class="highlighted">Transposon Hotspot Formation in LacI-Regulated lambda pL Hybrid Promoter Reporter System</SPAN></B></P><P class=""><B><SPAN class="highlighted">During cloning, formation of a transposon hotspot between the LacI-regulated lambda pL hybrid promoter and a reporter system was observed, making this promoter inapt for the dormancy system</SPAN></B>
To control the gene expression of RelB, the LacI-regulated lambda pL hybrid promoter, <A href="http://parts.igem.org/Part:BBa_R0011" target="_blank">BBa_R0011</A>, was chosen. Due to a putative formation of a transposon hotspot between the promoter sequence and the GFP reporter, which is described further <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-transposon-hotspot"> here</SPAN>, another inducible promoter was chosen to regulate the expression of RelB.</P><P class="P-Larger"><B><SPAN class="highlighted">Induction and Subsequent Inhibition of the pBAD Promoter</SPAN></B></P><P class=""><B><SPAN class="highlighted">Expression by the pBAD promoter can be regulated tightly by induction and subsequent inhibition</SPAN>.</B>
The pBAD promoter holds great potential to regulate the expression of the <I>relB</I> gene in our system, as it is capable of both an induction and repression. The <A href="https://2015.igem.org/Team:HKUST-Rice" target="_blank">HKUST-Rice iGEM team</A> from 2015 found that the pBAD promoter exhibits an almost all-or-none behaviour upon induction with arabinose when located on a high copy vector, but allows for gradual induction when cloned into a low copy vector. Thus, it was evident that this promoter on a high copy vector would be inappropriate for the regulation of RelB expression. Based on these findings, <SPAN class="highlighted">a low copy vector was used to investigate the ability to inhibit gene expression subsequent to induction of pBAD</SPAN>. 

RelB expression was simulated by fluorescence microscopy using a pBAD-YFP reporter system, <A href="http://parts.igem.org/Part:BBa_I6058" target="_blank">BBa_I6058</A>.
For this purpose, an Olympus IX83 with a photometrics prime camera was used, set to an excitation at 470 nm, emission at 515-560 nm, and exposure time at 150 ms. Transformed <I>E. coli</I> MG1655 cells were cultured in M9 minimal medium supplemented with 0.2% glycerol and 30 µg/mL chloramphenicol, to avoid catabolite repression from glucose residues present in LB medium. Two cultures were incubated, of which one was induced with 0.2 % arabinose from the beginning. At OD<SUB>600</SUB>=0.1, designated time 0, the cultures were split in two and 0.2 % glucose was added to one of each pair. Samples were obtained at time 0, before division of the cultures, and at 30 min, 60 min, and 120 min.  
The resulting images revealed, that the inducer arabinose was required to stimulate expression of YFP, and that the addition of the repressor glucose to a uninduced culture had no effect. Furthermore, <SPAN class="highlighted">it was evident that addition of arabinose induced expression of YFP, and that subsequent addition of glucose terminated the pBAD regulated gene expression on a low copy vector, resulting in a reduced fluorescence level</SPAN>. 30 minutes after inhibition this reduction was already evident, and after 120 minutes the gene expression controlled by pBAD was even further decreased, as seen in Figure 19.</P><DIV class="figure-text"><P><B>Figure 19.</B> YFP fluorescence levels in <I>E. coli</I> MG1655 transformed with the pBAD-YFP reporter system on pSB3K3. <B>Left</B>: Cultures with the inducer arabinose added. <B>Right</B>: Cultures not induced with arabinose. Both cultures were split up at OD<SUB>600</SUB>=0.1, designated time 0, and the inhibitor glucose was added to one half of each culture. Images were obtained at 0, 30, 60, and 120 minutes. </P></DIV><P><SPAN class="highlighted">This experiment made it clear, that gene expression controlled by the pBAD promoter is both inducible and repressible as required when cloned into the low copy vector pSB3K3</SPAN>. Consequently, the pBAD promoter was found to be suitable for controlling gene expression of the antitoxin RelB in the implemented dormancy system.</P></DIV></DIV><DIV class="modal fade" id="about-noise-levels" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Determination of Noise Levels</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>For these experiments, the promoters controlling an RFP reporter system were cloned into <I>E. coli</I> strain MG1655 on pSB1A2. The cultures were grown in LB medium containing 50 µg/mL ampicillin and examined by fluorescence microscopy. For this purpose an Olympus IX83 with a photometrics prime camera was used, set to an excitation at 550 nm, emission at 573-613 nm, and exposure time at 200 ms, and cultures were taken at OD<SUB>600</SUB>=0.3-0.5. For flow cytometry, both LB medium with 50 µg/mL ampicillin and M9 medium with 100 µg/mL ampicillin were used. Excitation of RFP was at 561 nm, and emission was measured around 580 nm. 
The images obtained by fluorescence microscopy, which are given in Figure 1, revealed that the weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, presented high variability in the expression of RFP between cells, indicating a high level of noise. Contrarily, RFP expression was more uniform in the bacterial cells containing the medium promoter <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">Bba_J23106</A>. Similarly, another medium strength promoter <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, showed uniform level of RFP expression, although less consistent than the observed for <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>. Expectedly, the strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A> exhibited high RFP expression in cells and a low level of noise was observed. </P><DIV class="figure-text"><P><B>Figure 1.</B> Fluorescence microscopy of RFP under the regulation of four members of the Anderson promoter collection, (<A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A> and <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>) on pSB1A2 in <I>E. coli</I> MG1655 at OD<SUB>600</SUB>=0.3-0.5. <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A> displayed a low and noisy expression of RFP, whereas both BBa_J23106 and BBa_J23110 showed a more uniform expression of medium intensity. The strongest promoter, BBa_J23102, exhibited a high level of consistent RFP expression</P></DIV><P>The noise levels observed by fluorescence microscopy led to the design of an experiment, by which the cell population could be quantitatively studied. The change in expression levels and noise during growth of cell populations were tested for the weak and strong constitutive promoters, for which flow cytometry was used to assess the expression of the RFP reporter system. First, the cell populations were studied in LB media with 50 µg/mL ampicillin, thereby maintaining similar conditions as for the fluorescence microscopy experiment. All cultures were prepared from overnight cultures, obtaining a starting OD<SUB>600</SUB>=0.005. 
</P><DIV class="figure-text"><P><B>Figure 2.</B> Flow cytometric fluorescence measurements in arbitrary units as a function of time. Left: Cultures were grown in LB medium. Right: Cultures were grown in M9 minimal medium supplemented with 0.2% glycerol. Fluorescence of RFP expressed by the weak and strong constitutive promoters were measured relative to the negative control WT <I>E. coli</I> MG1655. Samples where measured in technical replicates and standard error of mean is shown, but are in several cases indistinguishable from the graph.</P></DIV><P>
The data obtained in this experiment revealed that the strong promoter displayed a higher basal level of RFP expression than the weak promoter. Moreover, both promoters exhibited similar decrease in fluorescence over time, as seen in Figure 2, indicating that a considerable portion of the cell populations for both promoters lost their ability to fluoresce during growth, with one population containing around 60% non-fluorescent bacteria, clearly divided from the fluorescent portion. As expected, the strong promoter displayed a high level of RFP expression, roughly 500-fold of the non-fluorescent MG1655 control at 1 hour. A 10-fold decrease in fluorescence was observed from 1 hour to 4 hours. The RFP expression mediated by the weak promoter exhibited a similar decrease in fluorescence over time, though with measured fluorescence levels 5 times lower than for the strong promoter. 
 
Comparing these findings to the fluorescence microscopy data, it was hypothesised, that the increasing noise could be caused by bacterial plasmid loss. This could be ascribed to degradation of ampicillin in the medium, since the mechanism behind ampicillin resistance relies on the β-lactamase mediated cleavage of the antibiotic, thereby relieving the selective pressure. Consequently, the bacteria, which have lost their plasmid could be able to compete with the bacteria still containing their plasmid. 
The fluorescence microscopy for the strong promoter did not exhibit a considerable level of noise, however, this does not reject the hypothesis. The increase in noise was revealed by flow cytometry, as measurements were carried out at several times throughout growth. However, the fluorescence microscopy measurements were only carried out once in the exponential phase, where the noise was not prevalent yet. 
To substantiate the hypothesis, the experiment was performed anew using M9 minimal medium supplemented with 0.2 % glycerol and 100 µg/mL ampicillin, in an attempt to decrease the plasmid loss rate and optimise the selection.
The resulting data revealed, that the decrease in fluorescence levels had a lower rate for both promoters when the bacteria were cultured in M9 minimal medium, seen in the right graph in Figure 2, than in LB medium, seen in the left graph in Figure 2. Furthermore, when the bacteria were grown in M9 minimal medium with double amount of antibiotics compared to the LB medium, they displayed a higher level of fluorescence. Some of these observations could potentially be ascribed to the increased selection, that was strived after in this experiment. Moreover, the difference between the measured fluorescence levels for the two reporter systems was notably lower. This could indicate, that the reason behind the difference in expression levels between the weak and strong constitutive promoters, is due to the fact that bacteria cloned with the weak constitutive promoter are more prone to plasmid loss. However, it was observed that the bacteria expressing RFP under control of the weak promoter, did in fact fluoresce with a lower intensity.
 
To verify the hypothesis regarding the plasmid loss, the cultures were plated on 50 µg/mL ampicillin LA selection plates as well as LA plates without antibiotic, 12 hours after incubation start. For comparison, cultures containing either chloramphenicol and kanamycin resistance cassettes, were likewise plated out after similar treatment. The average plasmid loss measured as non-resistant CFU as a percentage of the total population is seen Figure 3.
</P><DIV class="figure-text"><P><B>Figure 3.</B> The average plasmid loss measured as non-resistant CFU as a percentage counts for the ampicillin resistance of the weak and strong promoter, as well as for chloramphenicol and kanamycin resistance controls. Samples were obtained for spread-plating at 12 hours after incubation start.</P></DIV><P>
From this test, it is evident that antibiotic resistance is reduced for the ampicillin-resistance carrying bacteria, compared to chloramphenicol and kanamycin resistant cultures.
This consolidates the hypothesis, that the bacteria transformed with the weak and strong promoter controlling RFP expression lose their plasmids throughout growth. However, it was further observed, that the plasmid loss in bacteria carrying the strong constitutive promoter was markedly higher than for bacteria carrying the weak constitutive promoter. This could be due to the fact that RFP in high concentrations is toxic for the cells, resulting in an increased pressure to lose the plasmid.
 
The photocontrol device genes were, on the basis of the above mentioned promoter experiments, placed under control of the strong promoter <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>. However, the observed plasmid loss should be taken into account, and compensated for, e.g. by continuous supply of antibiotic or substitution of the ampicillin resistance cassette.
Furthermore, as a result of the irregular expression levels controlled by the constitutive promoters, it was decided to place the <I>relB</I> gene under control of a tightly regulated uniform promoter, as the <A href="https://2017.igem.org/Team:SDU-Denmark#modelling" target="_blank">modelling</A> had established the necessity of this. </P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="about-mnt-penl" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Expression by the Mnt- and Penl-Regulated Promoters</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>
To verify the correct assembly of the strong constitutive promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, the part was sequenced using the <A href="http://parts.igem.org/Primers/Catalog" target="_blank">iGEM primer VF2</A> and a <A href="https://2017.igem.org/File:T--SDU-Denmark--Primer-fil.zip" target="_blank">self designed reverse primer</A> , located in the <I>ho1</I> gene of the photocontrol device. Through this, it was discovered that a plasmid fragment coinciding with the length of the promoter and the BioBrick prefix, had been removed from the plasmid, as illustrated on Figure 1.</P><DIV class="figure-text"><P><B>Figure 1.</B> A fragment containing the strong constitutive promoter and the BioBrick prefix with the EcoRI (E) and XbaI (X) restriction sites had been removed from the plasmid containing the photocontrol device. </P></DIV><P>The original sequencing data can be found<A href="https://2017.igem.org/File:T--SDU-Denmark--BBa-J23102-BBa-K519030_Sequencing.zip" target="_blank"> here</A>
In the light of this finding, it was investigated whether the Mnt-regulated or PenI-regulated promoter was suitable to control the expression of the photocontrol device.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="about-transposon-hotspot" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Transposon Hotspot</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>
During the cloning of GFP, <A href="http://parts.igem.org/Part:BBa_I13504" target="_blank">BBa_I13504</A> under control of the LacI-regulated lambda pL hybrid promoter, <A href="http://parts.igem.org/Part:BBa_R0011" target="_blank">BBa_R0011</A>, it was observed that the part yielded three bands when digested with EcoRI and PstI, see Figure 1. One band was observed at 2000 bp, corresponding to the pSB1C3 backbone, one band at around 950 bp, correlating with the expected length for the part, and a third band with a length around 1200 bp.</P><DIV class="figure-text"><P><B>Figure 1.</B> Agarose gel showing the digestion of the composite part by EcoRI and PstI. Three bands where observed. The bands at 2000 bp and 950 bp corresponded to the backbone and inserted part respectively. The third band at 1200 bp was unexpected.</P></DIV><P>
To ascertain the reason for the band with the unaccountable length, the part was sequenced using the <A href="http://parts.igem.org/Primers/Catalog" target="_blank">iGEM primers</A> VF2 and VR. The sequencing revealed a repeated sequence inserted between the LacI-regulated, lambda pL hybrid promoter and the GFP reporter, encoding a transposable element. The cloning was repeated, with a similar result. Given that a <A href="http://parts.igem.org/Part:BBa_K358000" target="_blank">similar part </A>, differing only in the RBS sequence, had been successfully assembled and tested before by the <A href=" https://2010.igem.org/Team:Kyoto/Project" target="_blank">Kyoto iGEM team from 2010</A>. It was hypothesised, that this particular assembly could cause the formation of a transposon hotspot.</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#demonstration-and-results">Click here to return to the Project Demonstration &amp; Results overview.</A></I></P></DIV><DIV class="row margin-bottom-75 padding-top-125" id="project-results-cellulose-breakdown"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Breakdown of Cellulose</H2></DIV></DIV><DIV style="text-align:center; margin-bottom:20px;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P>Several experiments were conducted to showcase our bacterium’s ability to grow on cellulose as the only carbon source, including Congo red screening assays and growth experiments. In addition, we performed Coomassie stained SDS-PAGE to visualise the expression of the cellulose degrading enzymes <I>cep94A</I>, <I>cex</I> and <I>cenA</I>. 
The results showed that <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A>, which expresses cellobiose phosphorylase, was functioning as expected. However, we were unable to detect any traces of the desired <A href="http://parts.igem.org/Part:BBa_K2449026" target="_blank">endoglucanase (<I>cenA</I>) and exoglucanase (<I>cex</I>)</A> secreted by the type I secretion system. </P><P class="P-Larger"><B><SPAN class="highlighted">Detection of Cellulase Secreted by Type I Secretion System was Inconclusive</SPAN></B></P><P><SPAN class="highlighted">To assess our cellulose degrading strains of <I>E. coli</I>, we performed a screening with carboxymethyl cellulose (CMC) Congo red agar plates</SPAN>. Congo red binds to cellulose molecules, thus making it possible to visualise the breakdown of the Congo red bound cellulose. Consequently, if the cellulose molecules are degraded the red color will fade. As seen in Figure 20 the experiment did not show any visible difference between the bacteria containing cellulase secreting BioBricks and the controls, <SPAN class="highlighted">suggesting that the type I secretion system is not working as intended</SPAN>.</P><DIV class="figure-text"><P><B>Figure 20.</B><I>E. coli</I> cultures containing seven different BioBricks plated on a Congo red plate containing the water soluble cellulose derivative CMC. Cellobiose phosphorylase on pSB1C3 with LacI promoter <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank">(<I>cep94A</I>)</A>, exoglucanase on pSB1C3 with PenI promoter <A href="http://parts.igem.org/Part:BBa_K2449017" target="_blank">(<I>cex</I>)</A>, endoglucanase on pSB1C3 with PenI promoter <A href="http://parts.igem.org/Part:BBa_K2449027" target="_blank">(<I>cenA</I>)</A>, endoglucanase + exoglucanase on pSB1C3 with PenI
promoter <A href="http://parts.igem.org/Part:BBa_K2449024" target="_blank">(<I>cenA</I>+<I>cex</I>)</A>, endoglucanase + secretion system on pSB1C3 with
PenI promoter <A href="http://parts.igem.org/Part:BBa_K2449025" target="_blank">(<I>cenA</I>+SS)</A>, endoglucanase + exoglucanase + secretion system on pSB1C3
with PenI promoter <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank">(<I>cenA</I> and <I>cex</I> + SS)</A> and LacI promoter + RBS on pSB1C3 <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">(negative control)</A>.</P></DIV><P class="P-Larger"><SPAN class="highlighted"><B>Measurable Expression of Cellobiose Phosphorylase</B></SPAN></P><P><SPAN class="highlighted">Figure 21-Right shows that <I>E. coli</I> containing <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> produces a protein with a molecular weight of approximately 90kDa</SPAN>, which is not produced by a negative control that only contains a PenI regulated promoter. This is in accordance with the <SPAN class="highlighted">expected weight of the protein expressed by the BioBrick containing the <I>cep94A</I> at 92.7 kDA</SPAN>. The protein can be found in the cell lysate regardless of whether the plasmid has a medium or high copy backbone, which is shown in Figure 21-Left.</P><DIV class="figure-text"><P><B>Figure 21.</B><B>Left</B>: Coomassie stained SDS-PAGE showing cell lysate and media from <I>E. coli</I> containing the cellobiose phosphorylase <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank">(<I>cep94A</I>)</A>, with Lacl promoter on the medium copy  plasmid pSB3K3 and the high copy plasmid pSB1C3.The LacI promoter was induced with IPTG. <B>Right</B>: Coomassie stained SDS-PAGE of whole cell lysates of <I>E.coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> with a Lacl promoter compared to a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A> containing LacI + RBS. Approximate reference molecular weights are indicated to the left.</P></DIV><P>Secretion of the cellulases, endoglucanase and exoglucanase, was examined on a SDS-PAGE, with samples taken from the media isolated from an overnight culture. From the SDS-PAGE at <SPAN class="highlighted">Figure 22 it was not possible to identify any proteins at the expected mass of endoglucanase and exoglucanase (47 kDa and 51 kDA, respectively)</SPAN>, that were differentially expressed compared to control strains. However, a positive control should have been included for more conclusive results.</P><DIV class="figure-text"><P><B>Figure 22.</B> Coomassie stained SDS-PAGE wíth Media
isolated from five <I>E.coli</I> strains, from left to right: Endoglucanase
and exoglucanase + secretion system with PenI promoter  <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449026" target="_blank">(<I>cenA</I>&amp;<I>cex</I> + SS)</A>,
endoglucanase + secretion system with PenI promoter <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449025" target="_blank">(<I>cenA</I> + SS)</A>, endoglucanase + exoglucanase with PenI promoter <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449024" target="_blank">(<I>cenA</I> and <I>cex</I>)</A>, endoglucanase with PenI promoter <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449011" target="_blank">(<I>cenA</I>)</A>, exoglucanase with PenI promoter <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449012" target="_blank"><I>cex</I></A>. All plasmids were on pSB1C3 plasmid.
Approximate molecular weights are indicated to the left.
</P></DIV><P class="P-Larger"><SPAN class="highlighted"><B>Cellobiose Phosphorylase Makes <I>E. coli</I> Able to Grow on Cellobiose</B></SPAN></P><P>The first growth experiments that were conducted included aeration of the bacteria at 200 rpm. In these experiments shown in Figure 23, <SPAN class="highlighted">we do not observe any major differences in growth between the strains containing the <I>cep94A</I> gene, the <I>bglX</I> gene and the control</SPAN>.</P><DIV class="figure-text"><P><B>Figure 23.</B> Growth experiment with the aeration at 200 rpm. On the x-axis time in hours and OD<SUB>600</SUB> on a logarithmic y-axis with standard deviation. All the experiments were conducted at once in the same water bath at 37°C and the data shown is an average of technical triplicates. Measurements were performed on <I>E. coli</I> MG1655 with inserted <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> encoding cellobiose phosphorylase on pSB1C3 backbone, <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank"><I>bglX</I></A> encoding
beta-glucosidase and a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A> with LacI promoter+RBS. All the strains were growing on cellobiose+casamino acids referred as Cellobiose and just casamino acids referred as none. The Lacl promoters were induced with IPTG.</P></DIV><P>The cultures were left at 37°C overnight without aeration. Increased growth was measured on the spectrophotometer at OD<SUB>600</SUB> in the flasks with <I>E. coli</I> expressing the <I>cep94A</I> gene, indicating that cep94A coding cellobiose phosphorylase gives <I>E. coli</I> the ability to live on cellobiose. To be certain that it was the <I>cep94A</I> that ensured <I>E. coli</I>  was living on cellobiose and not a contamination, we conducted a test with only a control (LacI and RBS) and <I>cep94A</I>, with measurements at 48 and 72 hours. <SPAN class="highlighted">This experiment revealed that cultures containing the <I>cep94A</I> are able to grow without aerating to an OD<SUB>600</SUB> around 0.9, as shown in Figure 24</SPAN>. The control living only on casamino acid only grew to an OD<SUB>600</SUB> around 0.2 and did not change between the two measurements.</P><DIV class="figure-text"><P><B>Figure 24.</B> Column diagram with standard deviation visualising the difference in OD<SUB>600</SUB> between a negative
control which is <I>E. coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">Lac-promoter + RBS
on pSB1C3</A>(green) and <I>E.coli</I>
transformed with <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> which encodes the cellobiose phosphorylase(yellow) after respectively 48 and 72 hours without aeration. The Lacl promoters was induced with IPTG.</P></DIV><P>As we intend to break down cellulose to energy, we need to combine the BioBricks, meaning a strain containing <I>cep94A</I> on the pSB3K3 backbone and the￼ secretion system with the endo- and exoglucanase on pSB1C3. This led us to repeating the first growth experiment with a slow aeration (40 rpm), just to make sure the media was mixed. <SPAN class="highlighted">The results are shown in Figure 25, which shows that <I>E. coli</I> is able to grow on cellobiose when transformed with the <I>cep94A</I>/cellobiose phosphorylase on the high copy backbone pSB1C3 and unable to grow when transformed with <I>bglX</I> or with <I>cep94A</I> on the medium copy backbone pSB3K3. Due to no growth on <I>cep94A</I> on the medium copy backbone pSB3K3, in the strain living on cellulose did not grow either</SPAN>. After 72 hours it was visually displayed that only <I>cep94A</I> transformed <I>E. coli</I> grew, which is illustrated in Figure 26.</P><DIV class="figure-text"><P><B>Figure 25.</B> The four figures all have time in hours on the x-axis and OD<SUB>600</SUB> on a logarithmic y-axis with standard deviation. All the experiments were conducted at once in the same water bath at 37°C and the data shown is an average of
technical triplicates, taken every eight hours. Measurements was performed on
<I>E. coli</I> MG1655 with inserted <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> encoding cellobiose phosphorylase on pSB1C3 backbone and PenI promoter, <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> encoding cellobiose phophorylase on pSB3K3 backbone and PenI promoter, <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank"><I>bglX</I></A> encoding beta-glucosidase with a Lacl promoter and a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A> with LacI promoter+RBS. Measurements was done on all strains growing on cellobiose+casamino acids referred as Cellobiose and just casamino acids referred as none. An addition parallel growth experiment was done on <I>E. coli</I> MG1655 with transformation of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449026" target="_blank">secretion system and cellulases</A> on pSB1C3
backbone and <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> encoding cellobiose phosphorylase on a pSB3K3 backbone, these measurements were done on strains growing on cellulose+casamino acids (Cellulose) and just casamino acids (none). The Lacl promoters was induced with IPTG. (A) The eight curves shown in this figure signifies all the different BioBricks supposed to degrade cellobiose, as well as a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A>. (B) The <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449026" target="_blank">type I secretion system with cellulases</A> ability to grow with cellulose as sole carbon
source. (C) <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> on the pSB1C3 plasmid compared to the pSB3K3 plasmid. (D) <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> compared to <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank"><I>bglX</I></A>.</P></DIV><DIV class="figure-text"><P><B>Figure 26.</B> Cuvettes containing 2 mL of six different samples from the growth experiment from Figure 25. The cuvettes contain from left to right, <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> in cellobiose media, <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> with no carbon source, a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A> in cellobiose media, a <A href="http://parts.igem.org/Part:BBa_J04500" target="_blank">negative control</A> with no carbon source, <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank"><I>bglX</I></A> in cellobiose media and <A href="http://parts.igem.org/Part:BBa_K523014" target="_blank"><I>bglX</I></A> with no carbon source. All plasmids was on the pSB1C3 backbone.</P></DIV><P><SPAN class="highlighted">In conclusion, we created a BioBrick containing <A href="http://parts.igem.org/Part:BBa_K2449004" target="_blank"><I>cep94A</I></A> coding for the cellobiose phosphorylase, which made it possible for the <I>E. coli</I> to live on cellobiose. It was not possible to show or detect whether the bacteria could live on cellulose as intended, and we detected no expression of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449026" target="_blank">the endoglucanase and exoglucanase</A>.</SPAN></P></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#demonstration-and-results">Click here to return to the Project Demonstration &amp; Results overview.</A></I></P></DIV><DIV class="row padding-top-125" id="project-results-extracellular-electron-transfer"><DIV class="col-xs-12"><DIV class="row"><DIV class="project-design-headline"><H2>Extracellular Electron Transfer</H2></DIV></DIV><DIV style="text-align:center; margin-bottom:20px;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV><P>As shown in the design of our device, the nanowires transfer electrons from the bacteria to the anode. Optimisation of the nanowires was examined using homologous recombination to exchange the endogeneous <I>G. sulfurreducens </I><A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449002 " target="_blank">PilA gene </A> with the higher conducting <I>G. metallireducens</I> pilA gene, rich in aromatic amino acids. The recombination was more difficult than anticipated, as we were unable to properly isolate the strain with successful homologous recombination. Instead the bacteria transformed with the <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank"><I>oprF</I></A>, which encodes a membrane protein that leads to improved extracellular shuttle-mediated electron transfer from the <A href="https://2013.igem.org/Team:Bielefeld-Germany" target="_blank">Bielefeld 2013 iGEM team</A>, was tested against the <I>G. sulfurreducens</I> PCA wild type. The results showed that the nanowires in <I>G. sulfurreducens</I> were more durable and conductive than <I>E. coli</I> with <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank"><I>OprF</I></A>, thus making nanowires a better alternative for use in MFCs.</P><P class="P-Larger"><SPAN class="highlighted"><B>Insertion of <I>G. metallireducens pilA</I> Into <I>G. sulfurreducens</I> Proved Difficult</B></SPAN></P><P>A fragment consisting of <I>pilA</I> from <I>G. metallireducens</I>, <I>camR</I>, and the 500 up- and downstream regions of the endogenous <I>pilA</I> genes of <I>G. sulfurreducens</I> were successfully constructed by PCR. The fragment was validated on gel electrophoresis, as seen in Figure 27-Left, with the band just above 3000 bp. The first electroporation was performed, and afterwards chloramphenicol was added for a concentration of 10µg/ml, this was added to isolate  the <I>G. sulfurreducens</I> recombinants, as done by Coppi MV et al. 2001<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC92998/"> Coppi MV, Leang C, Sandler SJ, Lovley DR. Development of a Genetic System for Geobacter sulfurreducens.
Applied and Environmental Microbiology. 2001;67(7):3180-7.</A></SPAN></SPAN>. After the selection of the <I>G. Sulfurreducens</I> recombinants, a PCR of its genomic DNA was carried out to identify the expected 3000 bp insertion. <SPAN class="highlighted">However, we were only able to retrieve a 1700 bp fragment</SPAN>, as seen in Figure 27-Right, and not the expected 3000 bp fragment. For this reason, a new experiment using the same 3000 bp fragment was performed, this time increasing the chloramphenicol concentration to 30 µg/ml for the selection of recombinants. <SPAN class="highlighted">This, however, resulted in no bacterial growth after electroporation. Due to a lack of time, we eventually decided to put this part of our project on hold</SPAN>. </P><DIV class="figure-text"><P><B>Figure 27.</B><B>Left</B>: DNA fragment consisting of <A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449002 " target="_blank">PilA gene with LacI promoter and RBS</A> from G. metallireducens, CamR and the 500 bp up- and downstream regions to pilA on <I>G. sulfurreducens</I>. <B>Right</B>:The PCR product from <I>G. sulfurreducens’</I> genome, which grew on a chloramphenicol concentration at 10µg/ml with primers 500 bp up- and downstream from <I>pilA</I>.</P></DIV><P class="P-Larger"><SPAN class="highlighted"><B>Nanowires From <I>G. sulfurreducens</I> Have Higher Electrical Conductivity Than <I>E. coli</I></B></SPAN></P><P>Even though we could not accomplish implementation of nanowires in <I>E. coli</I>, nor optimise the nanowires in <I>G. sulfurreducens</I>, we still found it relevant to test the <I>G. sulfurreducens</I> wild type nanowire efficiency, against previously used methods for extracellular electron transfer in MFCs in the iGEM competition. <SPAN class="highlighted">For that reason, we tested the electrical conductivity of <I>G. sulfurreducens</I> PCA wild type against <I>E. coli</I> ER25663127 containing <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A></SPAN>, a membrane protein, which helps carrying electrons to an extracellular electron shuttle in form of methylene blue.</P><DIV class="figure-text"><P><B>Figure 28.</B> Single chamber, three electrode systems were used for measuring the electrochemical current generation. The measurements were done with an electrode potential at -0.2V(vs. Ag/AgCl KCl sat.). Starting from the left: <I>E.coli</I> with <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A>, <I>E.coli</I> wild type, and then <I>G. sulfurreducens</I> wild type.</P></DIV><P>The results achieved from this experiment is shown in Figure 29, which shows that by using nanowires as the electron carrier, it is possible to tremendously increase the electrical current compared to the extracellular electron shuttle used previously. As seen for the <I>E. coli</I> wild type, the current decreases over time, which indicates no activity. <I>E. coli</I> containing the membrane porin <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A> shows a rapid increase, followed by a gradual decrease over 6-8 hours. <SPAN class="highlighted">The current of <I>E. coli</I> remains low, while <I>G. sulfurreducens'</I> current starts increasing after 20 hours</SPAN>, unfortunately the experiment had to stop due to time constraints.</P><DIV class="figure-text"><P><B>Figure 29.</B> Illustration of the microbial current generated from <I>G. sulfurreducens</I> wild type, <I>E. coli</I> ER25663127 wild type, and <I>E. coli</I> ER25663127 containing <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A>. The measurements was done with an electrode potential of -0.2V (vs. Ag/AgCl KCl sat.). Ten minutes before the measurements, IPTG and methylene blue were added to the <I>E. coli</I> containing chambers to achieve concentrations at respectively 1mM and 0.1mM. Acetate was added as electron donor to achieve a concentration of 10 mM to the <I>G. sulfurreducens</I> ten minutes before the measurements.</P></DIV><P>This shows that the electrical current is limited by time for the <I>E.coli</I> with <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A>, which Bielefeld also concluded. They measured the voltage over time, and their results showed that it reached the maximum after 30 mins. <SPAN class="highlighted">Highly interesting, the current from <I>G.sulfurreducens</I> starts to increase after 20 hours. This makes <I>G.sulfurreducens</I> a much more viable and stable electron transporter than the electron shuttle, <A href="http://parts.igem.org/Part:BBa_K1172502" target="_blank">OprF</A></SPAN>.</P></DIV></DIV><DIV style="text-align:center;"><P style="font-size:14px;"><I><A href="#demonstration-and-results">Click here to return to the Project Demonstration &amp; Results overview.</A></I></P></DIV></DIV></DIV></DIV></DIV><DIV id="proceduresContent"><DIV id="parts-and-procedures"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlineProcedures" class="headline"><SPAN class="highlighted">Parts &amp; Procedures</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><P><SPAN class="largeFirstLetter">I</SPAN>n this section, you will find all the needed information to replicate our approach and experiments. <SPAN class="highlighted">The constructed parts, notebook, SOPs and protocols will show in a pop-up window</SPAN> from which you can obtain all the necessary knowledge, should you be interested. An essential part of going to the lab is risk and safety assessments, which you will find at the end of the section. 
</P></DIV></DIV></DIV><DIV class="topShadow" id="parts"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-procedures"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Parts</SPAN></H2></DIV></DIV></DIV><DIV class="modal fade" id="BasicParts" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Basic Parts</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10">&lt;groupparts&gt;iGEM17 SDU-Denmark&lt;/groupparts&gt;</DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="notebook"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-procedures"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Notebook</SPAN></H2></DIV></DIV><DIV class="modal fade" id="modalNotebook" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Notebook</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><B>Week 8</B>
                                                    As 12 strangers we met for the first time on the 22<SUP>nd</SUP> of February. We cooked and ate a lovely meal together, and hereby had our first team building experience.
                                                    <B>Week 10</B>
                                                    We went to the DTU-Denmark BioBrick tutorial event, where we had an unforgettable experience. Here, we met some of the other Scandinavian teams, with whom we had a wonderful time.<B>Week 11</B>
                                                    A <A class="modal-link" href="https://static.igem.org/mediawiki/2017/d/d1/T--SDU-Denmark--Cooperation_agreement.pdf" target="_blank">group contract</A> was discussed and the first version was written. As part of the promotion of the team and iGEM, a banner was created for the information screens at our university, the University of Southern Denmark.
                                                    <B>Week 12</B>Excursion to a cabin on Langeland. At this idyllic spot, different ideas for our project were discussed and narrowed down to only a handful of ideas. It was a fun weekend and friendships among the team members began to sprout.<B>Week 14</B>
                                                    Through an iterative process, the list of project ideas was cut down. Ultimately, the PowerLeaf - A Bacterial Solar Battery was chosen as our project. 
                                                    <B>Week 16</B>
                                                    The team had a weekend of lab training with the supervisor Patrick, where we learned about all the basic applications for cloning. The work on the dormancy system had begun! 
                                                    <B>Week 17</B>
                                                    We had a lab safety course with Lab Technician Simon Rose. Thereby, we became qualified for unsupervised laboratory work. Now we were properly equipped to start the work!

During the Danish Science Festival, we presented our iGEM team. Here, we talked to a lot of people about GMO, especially with children. We had prepared several activities for the children to participate in, one being a drawing competition. In this competition, the most innovative and creative idea of a bacteria won. From this activity, the team received a lot of fantastic drawings, with endless creative ideas, that the team took into consideration and used to further shape our project.<B>Week 18</B>
                                                    The team had a course in innovation with Ann Zahle Andersen, from the patent office of the University of Southern Denmark. Here, we gained a better understanding of our project from a business point of view.

Bastian and Magnus, 6. grade students from a local public elementary school, interviewed us on the topic of GMO. After the interview, the boys received a guided tour around the iGEM lab. The boys used the obtained information about GMO to write an assignment at their school.<B>Week 19</B>
                                                    We got the first look at promoter reporter systems in the fluorescence microscope.  
                                                    <B>Week 21</B>
                                                    The team started their collaboration with Odense Municipality. The main focus in this collaboration was to integrate our project into an urban setting.                                                    <B>Week 23</B>
                                                    Putting our final touches on the PowerPoint presentation for the Nordic Meetup, held by the KU iGEM team. This was the first time, our project was presented for other teams. At the meetup, the team members met a lot of wonderful people, some for the first time, others were enjoyable reunions.<B>Week 24</B>
                                                    We received parts from the Bielefeld team for the carbon fixation system. There was a breakthrough regarding the secretion system, as we succeeded in the creation of the <I>cex</I> part.  
                                                    <B>Week 26</B>
                                                    The team filled out the first three parts of the safety form. We had a talk with postdoc Oona Snoeyenbos-West, who suggested that we should work with <I>Geobacter sulfurreducens</I> to test the bacterial nanowires. 
                                                    <B>Week 27</B>
                                                    The entire team went to the Netherlands, where we joined the European Meetup at TU Delft. At this meetup, we had the pleasure of meeting new people from several European iGEM teams, alongside meeting old friends from the Nordic teams. The team extended the trip while in the Netherlands, to spend a day in beautiful Amsterdam together.<B>Week 28</B>
                                                    The genes encoding RelE and RelB, which comprise a toxin-antitoxin system, were purified from <I>E. coli</I>. 
                                                    <B>Week 29</B>
                                                    We cracked the code of modelling and our modelling system stabilised. Furthermore, the secretion system was created.
                                                    <B>Week 30</B>
                                                    A presentation was held at the <A href="https://unf.dk/" target="_blank">UNF</A> (Danish Youth Association of Science) high school camp, where we taught them about synthetic biology and how to work with genes. 

In the lab, we received parts from the Imperial College team for cellulose biosynthesis. We also began creating the BioBrick for our nanowires. The <I>cenA</I> part was created, and the assembly of the entire secretion system was begun.
                                                    <B>Week 31</B>
                                                    Our lab workers analysed the properties of the LacI-regulated, lambda pL hybrid promoter for the energy storing unit.
                                                    <B>Week 33</B>
                                                    The team celebrated an early christmas. Yes, in July! Crazy, we know! There was a christmas tree, a cosy atmosphere and more food, especially gravy, than should be possible to consume. The team had a jolly good time.

We finally made our PCR fragment for <I>Geobacter</I>, which would be used for homologous recombination. Testing on our cellobiose phosphorylase, that we received from IDT, was begun.
                                                    <B>Week 34</B>
                                                    We held our own SDU meetup with focus on wiki and ethics for KU and DTU. The reunion with the other Danish iGEM teams was a welcoming experience.<B>Week 35</B>
                                                    The abstract for the iGEM Jamboree was finished. In this week we also decided that we should enter the energy track.
                                                    <B>Week 36</B>
                                                    The team members working on human practices had a really informative meeting with Kristina Dienhart from Smart City Odense. 
We held a presentation at the local high school <A href="http://www.otg.dk/sde/site.aspx?p=10807" target="_blank">OTG</A> (The Technical High School of Odense). SDU communication helped us produce a video about our project, which reached more than 27000 people on Facebook. Furthermore, an article about our project and team was published in the local newspaper Fyens Stiftstidende.

The genes encoding the carboxysome were assembled. Furthermore, the expression levels of the Mnt- and PenI-regulated promoters were compared in the fluorescence microscope.
                                                    <B>Week 37</B>
                                                    We hosted an event for <A href="https://atusyd.dk/" target="_blank">ATU</A> (Academy for Talented Youths) with transformation and synthetic biology as the main focus. This began with a general presentation, after which we went to our lab, where the aspiring scientists from ATU performed plasmid purification and gel electrophoresis. The team had a marvelous day as it was wonderful to see so many young individuals being engaged in science. 
                                                    <B>Week 38</B>
                                                    We had a human practises meeting with Rikke Falgreen Mortensen about integrating our project in MyBolbro. 
Furthermore, we held a presentation about GMO and our project for the local public school, Odense Friskole.

We were surprised to discover that a transposon hotspot had formed between the LacI-regulated, lambda pL hybrid promoter and the GFP reporter.
                         <B>Week 39</B>
                                                    We held a presentation at the local high school, Mulernes Gymnasium, where synthetic biology was the main focus. 

The lab workers discovered that a region containing the strong constitutive promoter and the BioBrick prefix had vanished from the photocontrol device plasmid. Furthermore, it was decided to keep the cellulose biosynthesis subproject theoretical henceforth. 
                                                    <B>Week 40</B>
                                                    Materials for our prototype were set after a talk with the plastic expert Flemming Christiansen.
Second part of the course in innovation and patents with Ann Zhale Andersen was completed.
The last two part of the safety form part were completed and submitted. Additionally, it was decided to keep the carbon fixation subproject theoretical henceforth. 
                                                    <B>Week 41</B>
                                                    The final promoter for the RelB expression, pBAD, was chosen and cloned.

We ceased trying to perform the recombination in <I>Geobacter</I> after several difficulties.
                                                    <B>Week 42</B>
                                                    We got together with the old iGEM teams from SDU-Denmark at the Old-iGEM Meetup. Both the project in general and our wiki were presented to them, and we got a lot of useful feedback to improve our project, presentation, and wiki. We shared a lovely meal and had a great night. 
We managed to create our secretion system part after several tries and problems.
                                                    <B>Week 43</B>
                                                    Parts were sent to HQ and the Judging form was filled out. 
In the lab, we obtained promising results, as the features of the pBAD and OmpR-regulated promoters were studied in the fluorescence microscope and flow cytometer. 
Besides working in the lab to get the last results ready for the wiki freeze, the team spend the weekend going through all of the text.
                                                    <B>Week 44</B>
                                                    With the wiki freeze around the corner, we worked day and night, caffeine concentrations in our blood running high. We shall make it!
                                                    </P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="sops-and-protocols"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-procedures"><DIV class="col-xs-12"><H2><SPAN class="highlighted">SOPs and Protocols</SPAN></H2></DIV></DIV><DIV class="modal fade" id="SOPs" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">SOPs</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/8/83/T--SDU-Denmark--SOP-1.pdf" target="_blank"><B>SOP01</B></A> - LA plates with antibiotic</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/d/db/T--SDU-Denmark--SOP-2.pdf" target="_blank"><B>SOP02</B></A> - ONC <I>E. Coli</I></P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/9/9e/T--SDU-Denmark--SOP-3.pdf" target="_blank"><B>SOP03</B></A> - Gel purification</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/f2/T--SDU-Denmark--SOP04.pdf" target="_blank"><B>SOP04</B></A> - Colony PCR with MyTaq</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/1/13/T--SDU-Denmark--SOP05.pdf" target="_blank"><B>SOP05</B></A> - Plasmid MiniPrep</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/f9/T--SDU-Denmark--SOP06.pdf" target="_blank"><B>SOP06</B></A> - TSB transformation</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/5/5f/T--SDU-Denmark--SOP07.pdf" target="_blank"><B>SOP07</B></A> - Fast digest</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/9/97/T--SDU-Denmark--SOP-8.pdf" target="_blank"><B>SOP08</B></A> - M9 minimal medium</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/4/4e/T--SDU-Denmark--SOP09.pdf" target="_blank"><B>SOP09</B></A> - Ligation</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/fa/T--SDU-Denmark--SOP10.pdf" target="_blank"><B>SOP10</B></A> - Phusion PCR</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/b/b2/T--SDU-Denmark--SOP11.pdf" target="_blank"><B>SOP11</B></A> - Bacterial freeze stock</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/8/8a/T--SDU-Denmark--SOP12.pdf" target="_blank"><B>SOP12</B></A> - Making LB and LA media</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/5/52/T--SDU-Denmark--SOP13.pdf" target="_blank"><B>SOP13</B></A> - Agarose gel DNA</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/8/83/T--SDU-Denmark--SOP14.pdf" target="_blank"><B>SOP14</B></A> - Table Autoclave</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/3/36/T--SDU-Denmark--SOP15.pdf" target="_blank"><B>SOP15</B></A> - Preparing Primers</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/6/68/T--SDU-Denmark--SOP16.pdf" target="_blank"><B>SOP16</B></A> - PCR Protocol USER Cloning</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/e/e4/T--SDU-Denmark--SOP17.pdf" target="_blank"><B>SOP17</B></A> - Excision and Ligation of PCR Product in USER Cloning</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/f8/T--SDU-Denmark--SOP18.pdf" target="_blank"><B>SOP18</B></A> - Speedy Vac</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/3/3a/T--SDU-Denmark--SOP19.pdf" target="_blank"><B>SOP19</B></A> - Preparing Eurofins sequencing samples</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/d/df/T--SDU-Denmark--SOP20.pdf" target="_blank"><B>SOP20</B></A> - Antibiotic stock production</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/a/ab/T--SDU-Denmark--SOP-21.pdf" target="_blank"><B>SOP21</B></A> - Electroporation</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/6/63/T--SDU-Denmark--SOP22.pdf" target="_blank"><B>SOP22</B></A> - P1 phage transduction</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/2/2b/T--SDU-Denmark--SOP23.pdf" target="_blank"><B>SOP23</B></A> - Genome extraction</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/1/1d/T--SDU-Denmark--SOP24.pdf" target="_blank"><B>SOP24</B></A> - Plasmid miniprep without kit</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/c/c8/T--SDU-Denmark--SOP25.pdf" target="_blank"><B>SOP25</B></A> - NBAFYE <I>Geobacter</I></P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/3/38/T--SDU-Denmark--SOP26.pdf" target="_blank"><B>SOP26</B></A> - Electroporation <I>Geobacter</I></P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/0/01/T--SDU-Denmark--SOP27.pdf" target="_blank"><B>SOP27</B></A> - SDS-PAGE</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="Protocols" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Protocols</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><B><A class="modal-link" href="https://static.igem.org/mediawiki/2017/a/ac/T--SDU-Denmark--Protocol01.pdf" target="_blank">Protocol 1</A></B> - Biobrick assembly - Light Sensing System</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/c/c7/T--SDU-Denmark--Protocol02.pdf" target="_blank"><B> Protocol 2</B></A> - Biobrick analysis - Light Sensing System </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/7/7e/T--SDU-Denmark--Protocol03.pdf" target="_blank"><B> Protocol 3</B></A> - Biobrick assembly - RelE-RelB system </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/6/67/T--SDU-Denmark--Protocol04.pdf" target="_blank"><B> Protocol 4</B></A> - Biobrick analysis - RelE-RelB </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/1/1c/T--SDU-Denmark--Protocol05.pdf" target="_blank"><B> Protocol 5</B></A> - Biobrick assembly - Dormancy System </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/f3/T--SDU-Denmark--Protocol06.pdf" target="_blank"><B> Protocol 6</B></A> - Biobrick assembly - Dormancy System Analysis </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/f/f1/T--SDU-Denmark--Protocol07.pdf" target="_blank"><B> Protocol 7</B></A> - BioBrick assembly - pOmpR characterisation </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/6/6b/T--SDU-Denmark--Protocol-8.pdf" target="_blank"><B> Protocol 8</B></A> - Biobrick assembly - Carbon fixation </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/a/ae/T--SDU-Denmark--Protocol-9.pdf" target="_blank"><B> Protocol 9</B></A> - Biobrick assembly - Cellulose biosynthesis </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/c/cc/T--SDU-Denmark--Protocol-10.pdf" target="_blank"><B> Protocol 10</B></A> - Biobrick analysis - Growth experiment</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/d/d2/T--SDU-Denmark--protocol11.pdf" target="_blank"><B> Protocol 11</B></A> - Biobrick assembly - Cellulose secretion </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/8/86/T--SDU-Denmark--Protocol12.pdf" target="_blank"><B> Protocol 12</B></A> - Biobrick assembly - Cellulose consumption </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/a/a5/T--SDU-Denmark--Protocol13.pdf" target="_blank"><B> Protocol 13</B></A> - Biobrick assembly - Metallireducens PilA </P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/3/30/T--SDU-Denmark--Protocol14.pdf" target="_blank"><B> Protocol 14</B></A> - Cellulose degradation analysis - SDS-page</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/5/56/T--SDU-Denmark--Protocol15.pdf" target="_blank"><B> Protocol 15</B></A> - Cellulose degradation - Cellulase screening</P><P><A class="modal-link" href="https://static.igem.org/mediawiki/2017/4/4b/T--SDU-Denmark--Protocol16.pdf" target="_blank"><B> Protocol 16</B></A> - Electrical analysis - Conductivity test</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="safety"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-procedures"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Safety</SPAN></H2></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>Proper Risk Management</B></SPAN></P><P>
Biosafety and proper risk assessment are important aspects to consider before any handling of genetically modified organisms (GMOs). There are several concerns that must be addressed properly. The safety of the public as well as of the environment is of the utmost importance, but the safety of the person in direct contact with the GMOs should not be compromised either. The risk associated with laboratorial work can be evaluated using the statement <SPAN class="highlighted">“Risk = Hazard ✕ Probability”. To responsibly address this inquiry, the entire team was given a mandatory lab safety course held by Lab Technician Simon Rose</SPAN>. In addition, we received a detailed handbook regarding lab safety. This ensured that all our team members were well equipped to work safely in the lab. Throughout the project we have continuously been evaluating the safety of our work. These assessments can be found in the <A href="https://2017.igem.org/Safety/Final_Safety_Form?team_id=2449" target="_blank">safety form</A>. Furthermore, our team participated in the 5<SUP>th</SUP> annual BioBrick workshop hosted by DTU BioBuilders. Here we participated in a lab safety course before entering their laboratories. Both of these lab safety courses gave us the necessary knowledge to work safely with GMO and handle waste appropriate, as well as the according procedures in case of an emergency.
 
In the lab, we worked with several potentially harmful chemical agents such as dimethylsulfoxide <SPAN class="highlighted">(DMSO), ethidium bromide, chloroform, phenol, Congo red, antibiotics, and autoclaved glycerol. These chemical agents were handled using gloves at all times, and whenever deemed necessary, handled in a fume hood</SPAN>. Gloves were worn when necessary, and clean lab coats were worn restrictedly in the laboratorial areas. To visualise bands in agarose gels, we used an UV board. UV rays are carcinogenic when exposure is frequent and prolonged. To reduce the amount of exposure, several precautions were made. Gloves, long sleeves and a facial screen were worn at all times, and the time spend at the UV board was kept at a minimum. 
</P><P class="P-Larger"><SPAN class="highlighted"><B>Public and Environmental Risk Assessment</B></SPAN></P><P>
The chassis organisms containing the system is meant to be contained in a device, which is incorporated into an urban environment. While this device would be a safely enclosed container, it still possess the risk of physical breakage from violent acts or environmental disturbances. For this reason, we consulted a plastics expert, who advised us to use the plastic known as Polycarbonate. <SPAN class="highlighted">This plastic is remarkably durable, with the ability to ward off most physical traumas. The plastics expert has estimated that such a container would last in an urban environment for at least 20 years</SPAN>, and most likely more than that. To illustrate the durability of the plastics, he notified us of several devices from the 1980s made of the same plastic, which still stand strong today.
<SPAN class="highlighted">One of the biggest concerns would be the release of GMOs into nature</SPAN>. While the GMOs used are not pathogenetic, they would be able to share the plasmids containing antibiotic resistance selectors to other bacteria, that might be pathogenic. Antibiotic resistance in pathogenic bacteria complicates the treatment of an infected individual and could, in tragic cases, be the difference between life and death. However small the risk of this scenario might be, it should be addressed properly. Furthermore, antibiotic resistant <I>E. Coli</I> strains could outmatch some of their fellow <I>E. Coli</I> strains through natural selection. This could negatively affect the balance of nature, that we are aiming to restore with the development of the PowerLeaf.
 
To avoid these risks, several kill switch mechanisms should be implemented into the final device. This could be performed by <SPAN class="highlighted">implementation of a kill switch activated by exposure to light in the energy converting unit</SPAN>. This would of course mean, that the energy converting unit’s container would need to block all sunlight. A task that could easily be carried out by adding Carbon Black to the required areas of the container. <SPAN class="highlighted">The energy storing unit, which requires light to function, could then have a kill switch which makes it dependent on the presence of the energy converting unit</SPAN>. This could be accomplished by having harmless molecules, not naturally found in nature but required for the survival of the energy converting unit, circulating in the system. A similar effect could be accomplished by making the bacteria in the energy converting and storing units codependent on each other for their survival. The implementation of such kill switch mechanisms would tremendously improve the biosafety of the device by opposing hazards related to any kind of physical breakage.
</P><P class="P-Larger"><B>Biodiversity</B></P><P>
Biological diversity is defined as: <I>The variability of living organism from all sources. Terrestrial and marine, and of cause the ecological complex they are a part of. This also include diversity within species, and between species and ecosystems<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.cbd.int/">Convention on Biological Diversity</A></SPAN></SPAN>.</I>

From a biologist point of view, this include all genes, species and ecosystems of a region.

Biodiversity is threatened by climate change. The climate change have not caused extinctions, but have lead to a decrease of fitness in a number of species. By reinventing how to use transgenic organisms, the aspiration is to improve on the degrading biodiversity that can already be seen. If you want to learn more about the impact caused by GMO and climate change on biodiversity <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-bio-diversity">you can read more here</SPAN>.</P><DIV class="modal fade" id="about-bio-diversity" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Bio Diversity</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Climate change's’ effect on biodiversity</B></P><P> 
Although climate change in itself does not threaten biodiversity, the aftermath of climate change have severe consequences, such as habitat destruction or loss, which is the greatest threat to biodiversity within a few decades<SPAN class="reference"><SPAN class="referencetext">Leadley, P., Pereira, H.M., Alkemade, R., Fernandez-Manjarres, J.F., Proenca, V., Scharlemann, J.P.W. et al. (2010). Biodiversity scenarios: projections of 21st century change in biodiversity and associated ecosystem services. In: Secretariat of the Convention on Biological Diversity (ed. Diversity SotCoB). Published by the Secretariat of the Convention on Biological Diversity, Montreal, p. 1–132. Technical Series no. 50.</SPAN></SPAN>. It could lead to migration and/or directional selection, which would affect ecosystems and their function. Should a species disappear from an ecosystem, it would have a negative effect on the fitness of any specie that depends on that species<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://science.sciencemag.org/content/305/5690/1632">Koh, Lian Pin, et al. &quot;Species coextinctions and the biodiversity crisis.&quot; science 305.5690 (2004): 1632-1634. </A></SPAN></SPAN>. This can already be seen in flowers that have specific pollinators, where the petals have changed phenotype as a result of climate change<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1111/j.1461-0248.2010.01538.x/full">Toby Kiers, E., et al. &quot;Mutualisms in a changing world: an evolutionary perspective.&quot; Ecology letters 13.12 (2010): 1459-1474. </A></SPAN></SPAN>. This results in a mismatch between pollinators and flowers, leading to one or both having a decrease in fitness.

The consequences is not limited to local ecosystems, whole biomes can be affected. Large portions of Amazonian rainforest could be replaced by tropical savannah, and boreal forests can be expected to expand at the expense of tundra biomes<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1029/2008GB003357/full">Lapola, David M., Marcos D. Oyama, and Carlos A. Nobre. &quot;Exploring the range of climate biome projections for tropical South America: the role of CO2 fertilization and seasonality.&quot; Global Biogeochemical Cycles 23.3 (2009). </A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1029/2007JG000528/full">Alo, Clement Aga, and Guiling Wang. &quot;Potential future changes of the terrestrial ecosystem based on climate projections by eight general circulation models.&quot; Journal of Geophysical Research: Biogeosciences 113.G1 (2008). </A></SPAN></SPAN>.
Coral reef degradation has already been observed, as the ocean temperature rises and it becomes more acidic<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://science.sciencemag.org/content/318/5857/1737">Hoegh-Guldberg, Ove, et al. &quot;Coral reefs under rapid climate change and ocean acidification.&quot; science 318.5857 (2007): 1737-1742. </A></SPAN></SPAN>.
The changes to biomes will have an effect on the ecosystems found within, and hereby also the biodiversity, which can have an overall negative effect in the fitness of species. The most extreme case of fitness decrease is extinction, and by that also a loss in biodiversity.</P><P class="P-Larger"><B>GMO’s effect on biodiversity</B></P><P>
The largest impact on biodiversity from GMO is seen in agriculture. In agriculture an increase in GM plants can be seen, as crops have been modified to be herbicide tolerant (HT). This is primarily seen in soy, maize and cotton<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://cmsdata.iucn.org/downloads/ip_gmo_09_2007_1_.pdf ">UCN (2007) ”Current knowledge of the impacts of genetically modified organisms on biodiversity and human health”. </A></SPAN></SPAN>. There are also plants modified to be arthropod tolerant (Bt), and is toxic to specific groups of arthropods.
 
Crops who have been modified with both HT and Bt, are so called stacked genes crops. HT crops allow farmers to more effectively manage pest and weeds in their fields, where Bt crops kills of pests as they feed on the plant<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://cmsdata.iucn.org/downloads/ip_gmo_09_2007_1_.pdf ">UCN (2007) ”Current knowledge of the impacts of genetically modified organisms on biodiversity and human health”. </A></SPAN></SPAN>.

The overall effect of HT crops could end with a local fall in biodiversity, as weeds and other plants, together with arthropods, are simply removed with herbicides. Another threat would be the effect of horizontal gene-transfer between related species, where wild type plants get the HT gene from domesticated plants. This is seen in the golf field grass <I>Agrostis stolonifera</I> from a farm and the wild type <I>Agrostis gigantea</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.pnas.org/content/101/40/14533.full
 ">Watrud, Lidia S., et al. &quot;Evidence for landscape-level, pollen-mediated gene flow from genetically modified creeping bentgrass with CP4 EPSPS as a marker.&quot; Proceedings of the National Academy of Sciences of the United States of America 101.40 (2004): 14533-14538”. </A></SPAN></SPAN>.
The HT gene was found in one-third of <I>A. gigantea seeds</I>, and up to 9 miles from the farm. This horizontal gene-transfer could lead to “superweeds”, needing more and stronger herbicide to get rid of<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.researchgate.net/publication/293647479_IS_GMO_SUSTAINABLE_A_REVIEW_OF_THE_ENVIRONMENTAL_RISKS_OF_GM_PLANTS_IN_COMPARISON_WITH_CONVENTIONAL_AND_ORGANIC_CROPS">Prusak, Anna, Gene Rowe, and Jacek Strojny. &quot;Is GMO&quot; Sustainable&quot;? A Review of the Environmental Risks of GM Plants in Comparison with Conventional and Organic Crops.&quot; Modern Management Review 19.21 (4) (2014): 187-200.</A></SPAN></SPAN>.

Bt crops is toxic to arthropods, as seen with the Monarch butterfly in North America. The Monarch butterfly larvae would die from eating milkweed covered in pollen from BT crops, but thanks to Bt cotton in Mexico, the number of hibernating butterflies in cotton farms increased four times as less pesticide where needed in those areas<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.researchgate.net/publication/293647479_IS_GMO_SUSTAINABLE_A_REVIEW_OF_THE_ENVIRONMENTAL_RISKS_OF_GM_PLANTS_IN_COMPARISON_WITH_CONVENTIONAL_AND_ORGANIC_CROPS">Prusak, Anna, Gene Rowe, and Jacek Strojny. &quot;Is GMO&quot; Sustainable&quot;? A Review of the Environmental Risks of GM Plants in Comparison with Conventional and Organic Crops.&quot; Modern Management Review 19.21 (4) (2014): 187-200.</A></SPAN></SPAN>.

Horizontal gene-transfer of the HT gene will not harm the biodiversity “in the wild”. As the gene does not increase the fitness of the domestic plant, unless there is a caretaker to spray for weeds and pests. Gene-transfer of the Bt gene could hurt the wild biodiversity. The plants overall fitness would increase, due to the ability to fight off arthropods who feed on it. This would lead to an overall fitness decrease in the feeding arthropods, as feeding animals dies from the toxin. This might lead to birds who feed on said arthropods to migrate from the area in search of food, leading to another loss in biodiversity<SPAN class="reference"><SPAN class="referencetext">Berthold, Peter. Bird migration: a general survey. Oxford University Press on Demand, 2001.</SPAN></SPAN>.
</P><P class="P-Larger"><B>Future impact on biodiversity</B></P><P>
As stated, climate change does have a negative impact on biodiversity. One way for GMOs to help improve biodiversity is to decrease the output of greenhouse gasses, or decrease the amount already found in our atmosphere. HT and Bt crops already help with that, lowering the amount of herbicides and pesticides used in agriculture fields. Optimizing photosynthesis for agriculture plants<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.plantphysiol.org/content/155/1/79.short">Ort, Donald R., Xinguang Zhu, and Anastasios Melis. &quot;Optimizing antenna size to maximize photosynthetic efficiency.&quot; Plant physiology 155.1 (2011): 79-85.</A></SPAN></SPAN>, would not only reduce CO<SUB>2</SUB>, but could also increase growth in the plants. Incorporating photosynthesis into daily life usage, like our PowerLeaf, is one such possible and responsible solution.

GMO alone will not be able to negate the negative impact on biodiversity from climate change, and some effects we are already experiencing. GMO is able to help us on a direction to reduce the impacts on biodiversity. If used in conservation by reintroducing newly extinct species<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.sciencedirect.com/science/article/pii/S0093691X08007784">Folch, J., et al. &quot;First birth of an animal from an extinct subspecies (Capra pyrenaica pyrenaica) by cloning.&quot; Theriogenology 71.6 (2009): 1026-1034.</A></SPAN></SPAN>, possible rebuilding lost ecosystems.

Lastly GMO can also help ecosystems by engineering flora and fauna, so they can resist changes caused by climate change, or promote a specific phenotype. When talking about releasing transgenic organisms into the natural environment, it leads to a whole other discussion of GMO.</P><P class="P-Larger"><B>The PowerLeaf</B></P><P>
To avoid the discussion of GMO in the environment, and not at least to mitigate any effect, positive and negative, on the environment we designed our PowerLeaf to be isolated in a material called polycarbonate. This, alongside with a kill switch, should ensure that no contamination of local fauna and flora would arise. Therefore our product will not be able to influence wild types, neither by outcompeting nor horizontal gene-transfer. With these considerations in mind, we believe that our project is not only responsible but also environmentally safe.</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><P class="P-Larger"><B>List of Assessed Items</B></P><P><B>Chassis Organisms</B><I>Escherichia coli</I> strains: K12, TOP10, MG1655, KG22, BW25113, DF25663127, SØ928<I>Geobacter Sulfurreducens</I> strain: PCA<B>Vectors</B><A href="http://parts.igem.org/Part:pSB1A2" target="_blank">pSB1A2</A>: An iGEM plasmid backbone carrying a ampicillin resistance gene<A href="http://parts.igem.org/Part:pSB1A3" target="_blank">pSB1A3</A>: An iGEM plasmid backbone carrying an ampicillin resistance gene<A href="http://parts.igem.org/Part:pSB1C3" target="_blank">pSB1C3</A>: An iGEM plasmid backbone carrying a chloramphenicol resistance gene<A href="http://parts.igem.org/Part:pSB3C5" target="_blank">pSB3C5</A>: An iGEM plasmid backbone carrying a chloramphenicol resistance gene<A href="http://parts.igem.org/Part:pSB1K3" target="_blank">pSB1K3</A>: An iGEM plasmid backbone carrying a kanamycin resistance gene<A href="http://parts.igem.org/Part:pSB4K5" target="_blank">pSB4K5</A>: An iGEM plasmid backbone carrying a kanamycin resistance gene<A href="http://parts.igem.org/Part:pSB3K3" target="_blank">pSB3K3</A>: An iGEM plasmid backbone carrying a kanamycin resistance gene<B>Bacteriophages</B>
P1 phage, using its site-specific recombinase for transduction of <I>E. Coli</I></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="practicesContent"><DIV id="practices"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlinePractices" class="headline"><SPAN class="highlighted">Practices</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><DIV style="text-align:center;"><P class="raleway P-Larger" style="text-align:center;"><I>“Change is the law of life; and those who look only to the past or present are certain to miss the future”</I></P><P class="raleway"><I>John F. Kennedy</I></P></DIV><P><SPAN class="highlighted"><SPAN class="largeFirstLetter">W</SPAN>elcome to our Human Practices!</SPAN> Now, when it comes to the particulars of our Human Practices, you will find that it has been separated into three main parts. This is all strictly for narrative purposes, as every single aspect of our project and Human Practices are deeply intertwined through a shared philosophy: If you want change, look to the future! 
</P><P class="P-Larger"><B>1. A Philosopher’s Guide to the Future</B></P><P>To ensure an ethically sound iGEM product and experience, we have discussed the ethical considerations that ought to be taken into account. As luck would have it, one of our team members is a philosopher with an interest in bioethics. Thus a guidebook was created, a guide <SPAN class="highlighted">that amongst other things includes an overview of some of the bioethical arguments iGEM participants are likely to encounter, when discussing synthetic biology</SPAN>. You can find these considerations in our section on <A href="#bioethics" target="_blank">Bioethics</A>.</P><P class="P-Larger"><B>2. An Implementation of the Future</B></P><P><SPAN class="highlighted">We reached out to our local Municipality of Odense along with various experts, for their advice on the development and implementation of our device in an urban environment</SPAN>. You can read more about this in our section on <A href="#integrated-practices" target="_blank">Integrated Human Practice</A>.</P><P class="P-Larger"><B>3. A Trip to the Future and Beyond!</B></P><P>Considering how our main philosophy was: If you want change, look to the future! <SPAN class="highlighted">It seemed prudent to bring our message on sustainability to the next generation</SPAN>. You can read more about our efforts in the section on <A href="#education-and-public-engagement" target="_blank">Education and Public Engagement</A>.</P></DIV></DIV></DIV><DIV class="topShadow" id="bioethics"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-practices"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Bioethics</SPAN></H2><H4>- <I>A Philosopher’s Guide to the Future</I></H4><DIV style="text-align:center;"><P class="raleway P-Larger" style="text-align:center;"><I>“The facts of life... to make an alteration in the evolvement of an organic life system is fatal. A coding sequence cannot be revised once it's been established.”</I></P><P class="raleway"><I>Tyrell, Bladerunner</I></P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P><SPAN class="highlighted">Synthetic biology and the iGEM competition are aimed to help solve societal issues</SPAN>, issues such as agriculture, medical research, and environmental resource management, the last of which has been our motivation throughout our project. However, while <SPAN class="highlighted">synthetic biology offers many new exciting possibilities</SPAN>, several concerns have to be met when dealing with living organisms. Against, the excitement of the iGEM community, <SPAN class="highlighted">skeptics have pointed to the uncertainty and potentiality for unwanted consequences</SPAN> that might arise from working with synthetic biology.  <SPAN class="highlighted">We as a team decided to give these concerns some serious thought</SPAN>! We have had several conversations on topics such as meta ethics, applied ethics, sustainability, GMO, and so on. Furthermore, we established an open and honest mode of discourse to use when engaging with other teams and the public. <SPAN class="highlighted">We held a workshop in bioethics in relation to our Danish Meetup</SPAN>. This workshop consisted of a historical insight into the history of GMO provided by our very own historian, as well as a hefty, kind-hearted debate, supervised by our own philosopher. 
 
All of these considerations, <SPAN class="highlighted">debates, and results led our philosopher Lene to write a longer guidebook entailing an overview of the various bioethical arguments often made for and against the use of synthetic biology</SPAN>. The guidebook is mainly an insight into the most important ethical considerations made by our team, e.g. how they shaped our product, but it also offers a personal and clear overview of arguments and principles meant to help future teams to get a conversation on ethics started! 
<SPAN class="highlighted">Make sure to give this <A href="https://static.igem.org/mediawiki/2017/7/75/T--SDU-Denmark--guidebook.pdf" target="_blank">guidebook</A> a read</SPAN>, it is definitely worth it!</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="integrated-practices"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-practices"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Integrated Practices</SPAN></H2><H4>- <I>An Implementation of the Future</I></H4><DIV style="text-align:center; margin:35px 0 25px 0;"><P class="raleway P-Larger" style="text-align:center;"><I>“The best way to predict your future, is to create it”</I></P><P class="raleway">Abraham Lincoln - (former) president of the United States of America</P></DIV><P>Not that we can claim to be anything like Abraham Lincoln, or even vampire hunters. <SPAN class="highlighted">Nonetheless, we do agree that to create the future we all hope for, we must contribute to find a sustainable solution for a greener future</SPAN>. Before we can tackle the task of providing a sustainable future for the entire world, we must first look to our own local environment. <SPAN class="highlighted">We believe that the best way to gain a better understanding of a global dilemma, is to examine how a local environment is affected by it. </SPAN> Hopefully, this approach will help future iGEM teams find a connection between global issues and local ones. This approach has helped us elucidate specific issues and find sustainable solutions, which can be implemented into our society with the help and endorsement of local agents. </P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><B>A Statement from the Mayor of Odense</B></P><P><SPAN class="highlighted">We first decided to reach out to the mayor of Odense, to investigate the possibilities for iGEM to help in the government's endeavours to make Odense a CO<SUB>2</SUB> neutral city.</SPAN></P><P class="raleway citation"><I>“We face a series of challenges that we have to recognise, in the chase of the good and sustainable life in the city of Odense. Some of these concern local circumstances, while others contain national and even global issues. We as the municipality can only go so far on our own. So, we are entirely dependent on the help of local agents. <SPAN class="highlighted">It makes me so happy, when the students of the city, have taken on the mantle of developing new green technologies that solves global issues</SPAN>, while contributing to local city growth”</I></P><P class="raleway citation-by"><A href="https://static.igem.org/mediawiki/2017/7/7e/T--SDU-Denmark--mail-mayor.pdf" target="_blank">Peter Rahbæk Juel</A> - Mayor of Odense</P><P><SPAN class="highlighted">The core philosophy of our Integrated Human Practices has been to incorporate local experts in the development of our project.</SPAN> We wanted to examine how results produced in the laboratory, could be used to shape a product that corresponds with the green values of Odense.  
We sought the advice of experts in other fields, e.g. expert in plastics to design the best possible exterior of our device. Furthermore, we met with business developer Ann Zahle Andersen to investigate the core value of our product from a business perspective. <SPAN class="highlighted">We believe that Human Practices have played an essential role in our iGEM project. </SPAN> Everything from the design of our prototype to ethical considerations have been influenced by the people we engaged with.
</P><P class="P-Larger"><B>Interviewing Smart City Odense</B></P><DIV><P style="display:inline;"><SPAN class="highlighted">For the possible implementation of the PowerLeaf in the local environment of Odense, we decided to reach out to Kristina Dienhart, project manager of Smart City Odense.</SPAN> Smart City Odense is a project within Odense Municipality, that seeks to combine urban planning with new technologies and open-data toward creating a smarter, more sustainable city. 
This made us aware of necessities essential to Odense and its citizens. She gave us feedback that we integrated into numerous areas of our overall project.
</P></DIV><UL class="list" style="margin-top:15px;"><LI><B>Changeability </B>- <SPAN class="highlighted">From Mrs. Dienhart’s point of view, one of the most advantageous attributes of our device is the potential for changeability</SPAN> in the size and shape of the design. We had yet to consider the PowerLeaf as a device not limited by physical dimensions. This has been the most significant element we took with us from the interview. Changeability is a necessity to a city planner, as various laws and aesthetic considerations need to be taken into account, when altering or creating an urban environment. </LI><LI><B>Accessibility </B>- She also discussed accessibility with us. <SPAN class="highlighted">The citizen will not use our device unless it is easily accessible.</SPAN> This means that the overall design of the PowerLeaf, regardless of its aesthetics, always needs to be designed with a user in mind. Reflecting on the advice of Mrs. Dienhart, <SPAN class="highlighted">we decided to reevaluate the means of implementation of the PowerLeaf to ensure that the need for accessibility and user-comfort is met.</SPAN></LI><LI><B>Essentiality </B>- She supported our notion if the needs for <I>accessibility</I> and <I>changeability</I> are met, the PowerLeaf could help ensure that citizens of Odense use and remain in the public space for a longer amount of time. Something that is valuable, not only to the individual citizen, but also to the community as a whole, as it creates a sense of city cohesion and hence a high quality of city life. </LI></UL><P><SPAN class="highlighted">Mrs. Dienhart introduced us to several considerations that shaped large parts of our project.</SPAN> We do not know the needs of every urban area in Odense and consequently, we have aimed to create a device that is changeable to a city in movement such as Odense. 

Furthermore, <SPAN class="highlighted">this interview was also a source of inspiration for our ethical and safety thoughts.</SPAN> While we <I>ought</I> to strive for a sustainable tomorrow, we do not necessarily have to provide an exhaustive description of what the future should look like.
</P><P class="P-Larger"><B>Interviewing the City Renewal Project My Bolbro</B></P><DIV style="margin-top:-25px;"><P style="display:inline;"><SPAN class="highlighted">Rikke Falgreen Mortensen is the manager of the Bolbro’s city renewal project called Mit Bolbro i.e. My Bolbro.</SPAN> We arranged a meeting with her with the intent of further investigating how the PowerLeaf <I>could</I> and <I>should</I> be integrated in an urban area of Odense, in this case the neighbourhood of Bolbro.</P></DIV><P>
Bolbro is an old neighbourhood in Odense historically known to be the home of the working class. While Bolbro provides a homely atmosphere known to the locals, it has had a hard time attracting new residents. However, <SPAN class="highlighted">this is subject to change as the neighbourhood in 2016 received approximately 1.6 million US dollars to renew its city space</SPAN> and to create an even more appealing, and vibrant neighbourhood. This will be achieved by including the locals, as Bolbro is characterized by having a strong, engaging civil society. Mrs. Mortensen is not only an expert in urban renewal, but also in how to include local citizens in reshaping the public space in which they reside. <SPAN class="highlighted">Mrs. Mortensen also argued that a changeable design would be the optimal solution to fit the challenges one faces in creating a vibrant, green city ambience.</SPAN> Such a task depends on different preferences, laws and needs. A technology needs to be both <I>flexible</I> and <I>accessible</I> to successfully contribute to the process of creating an engaging city environment. She showed great interest in our device and even offered to implement it in the parks of Bolbro, should the product become a reality.</P><P> We had a discussion with Mrs. Mortensen about the creation of a prototype based on the wishes of Bolbro’s local citizens. Following this conversation, she provided us with a <A href="https://static.igem.org/mediawiki/2017/4/4b/T--SDU-Denmark--mail-rikke.pdf" target="_blank">pitch</A> that aimed to help us develop this prototype.</P><P class="raleway citation"><I>“Hauge’s square is a spot in Bolbro, which we aim to make a central place in Bolbro; a place that invites the citizen to meet and dwell. <SPAN class="highlighted">Your solution should be able to contribute to help citizens recharge their phones, e.g. a solution could be implanting the PowerLeaf into a interactive furniture, but where the demand an aesthetic pleasing design still remains.</SPAN>” - Translated from Danish</I></P><P class="raleway citation"><I>“A part of the vision of this project is the concept of making a pop-up park with differently designed multi-furniture, preferably in wood and organic design, which are removable to the various areas where we are going to develop in the district. It is furniture that should be able to be used to relax in and at the same time also motivates children to move. <SPAN class="highlighted">There is also a need for charging devices and it therefore demands that your solution is an integrated</SPAN>, but still mobile solution, as the park will move physically over time. Finally, the playground is to be developed especially for the young audience, which is a major consumer of power for phones. <SPAN class="highlighted">The playground must be a place where youngsters hang out after school, while maintaining its status as a green space</SPAN>. - Translated from Danish”</I></P><P>The making of the furniture as a prototype called for a revisit of our safety concerns. We now knew that children would be climbing and playing on the furniture, making it crucial that the material of the PowerLeaf will not break. This is a concern we discussed with Flemming Christiansen, which you can read all about next.</P><DIV id="plastic-expert" style="margin-top:-70px; padding-top:70px;"><P class="P-Larger"><B>Finding the Proper Material</B></P><P><B>Criteria to the Prototype</B>
The system itself will consist of two different compartments, an outer and an inner chamber. The first will be facing the sun, while the other will be facing the building or furniture. Since one culture of the bacteria depends on solar energy to produce its product, the outer compartment must allow for sunlight to pass. <SPAN class="highlighted">It should here be noted that the prototype is purely hypothetical, as the membrane, between the two compartments, should exclusively be permeable to cellulose</SPAN>. For that reason, we wanted to find a material, that fulfilled our established criteria, so that we could illustrate the technology.  
The device itself will be made entirely from plastic, a material that is thought to be undesirable due to the difficulties in its disposal. This is due to plastic being of a xenobiotic nature, making it generally recalcitrant to microbial degradation<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://ac.els-cdn.com/0167779988900844/1-s2.0-0167779988900844-main.pdf?_tid=f4188d40-be6e-11e7-916f-00000aacb35e&amp;acdnat=1509477121_f4df2995d544bf9afdfe37f0e99577e2"> Fewson CA. Biodegradation of xenobiotic and other persistent compounds: the causes of recalcitrance. Cell. 1988.</A></SPAN></SPAN>. Following these concepts, we can identify the following set of criteria for the desired material:</P><UL class="list"><LI><B>Solar exposure.</B> The material covering the solar cell, must allow sunlight to pass through to reach the bacteria.</LI><LI><B>UV resistance.</B><SPAN class="highlighted">As the material will be exposed to the sun, it must be resistant to the UV radiation</SPAN>.</LI><LI><B>Bacterial growth</B> The material must neither be growth inhibitory, nor toxic to the bacteria.</LI><LI><B>Easy to mold.</B> The outside of the device could be molded depending on the circumstances, as the device only relies on the bacterial technology.</LI><LI><B>Durability.</B> The device will be located outside, <SPAN class="highlighted">meaning that the material must be able to withstand hard conditions and heavy weight</SPAN>. </LI><LI><B>Temperature.</B> The material must allow for an appropriate constant temperature for the bacteria, despite the variations in sun exposure.</LI><LI><B>Longevity.</B> We would like for the material to have as long a durability as possible, since replacing the device could prove cumbersome. <SPAN class="highlighted">We are aiming for at least twenty years of durability</SPAN>. </LI><LI><B>Price.</B> We are looking for a material that is as cheap as possible, without sacrificing the necessary criteria.</LI><LI><B>Environmentally friendly.</B><SPAN class="highlighted">Considering the goal of this project being the creation of an environmentally friendly energy source, the ideal material would be as green as possible</SPAN>. </LI></UL><P><B>Interview with Flemming Christiansen</B></P><DIV><P style="display:inline;">For the purpose of finding the necessary materials for our prototype, <SPAN class="highlighted">we contacted one of the leading plastic experts in Denmark, Flemming Christiansen</SPAN> , who acts as the market development manager of SP Moulding. He has been acting as a plastics consultant since his graduation as a master of science in Engineering, with a speciality in plastics. A meeting was quickly arranged for the purpose of confirming our criteria, the technical design, the material, and the possible price of creating the PowerLeaf. </P></DIV><P>In accordance with our established criteria, <SPAN class="highlighted">Mr. Christiansen suggested that we use the plastic known as Polycarbonate, specifically Lexon 103R-III</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://plastics.ulprospector.com/datasheet/e17532/lexan-103r-resin"> Polycarbonate</A></SPAN></SPAN>. Unfortunately, the material cannot fulfil the criteria on its own. He therefore suggested that we take a few liberties with it. In order to prevent UV degradation to the exposed parts, we will be adding certain additives to the surface. This increases the UV resistance of the device, without hindering the sunlight from reaching the bacteria.

During our conversations with Mr. Christiansen, we reached the topic of what to do in case of a breach. Should the container against all expectations be damaged, the environment will be exposed to the GMO inside. The solution we came up with was the possible implementation of a kill-switch in the inner compartment, making it vulnerable to sunlight. Should the bacteria of said unit be exposed to sunlight, they would perish. <SPAN class="highlighted">As the outer compartment would be dependent on the continued coexistence of the two units, the entire GMO system would be purged in case of a breach</SPAN> . To implement this feature, the inner chamber would be covered with Carbon Black, which has the ability to absorb sunlight, thus leaving the compartment itself in darkness. 
The process of constructing our device would be through an extensive use of Injection Moulding, which is considered pricey equipment. The material is expensive at 4-5.5 USD per kg at orders above 1 metric ton, according to Mr. Christiansen, but its longevity and durability means it would not need to be replaced for a long time. Lastly, we discussed the reusability of Polycarbonate, which he assured us was of no concern, as the material could be <I>reused</I> and <I>recycled</I> with ease.
</P><DIV style="text-align:center;"><P><SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#detailed-prototype"><I>Click here to see a detailed version of the prototype.</I></SPAN></P></DIV><DIV class="modal fade" id="detailed-prototype" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Detailed Prototype</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>This prototype was designed with Flemming Christiansens advice in mind. 
The first compartment will have a dome shaped top, thus allowing sunlight to enter, while eliminating as many refractions as possible. A nourishment and wastes system will be implemented along the sides of the leafs as well as through the bottom.</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV><P class="P-Larger"><B>Workshop with Business Developer Ann Zahle Andersen</B></P><DIV><P style="display:inline;">For the purpose of getting a business perspective on our project, we met with Business Developer Ann Zahle Andersen. She arranged two workshops for us based on a business model canvas. She encouraged us to view our project, as if we meant to make startup business. <SPAN class="highlighted">These workshops gave us a better comprehension of society’s pull and pushes on a project like ours</SPAN>. This forced us as a team to get to the bottom of what we found important about our project.</P></DIV><P class="P-Larger"><B>Upcoming Meeting with Borgernes Hus</B></P><P>Borgernes Hus i.e. House of the Citizens is a new initiative offered by the City Central Library. The initiative aims to offer guidance and advice to projects such as ours. It is meant to aid Odense in its journey towards the status of becoming a modern Danish city. Unfortunately, the building remains under construction until after our trip to Boston, meaning that they have been busy with the construction while our project was underway. <SPAN class="highlighted">It is for this reason that we, along with director Jens Winther Bang Petersen, agreed that a future collaboration would be the most suitable solution.</SPAN>
It is our hope, that a collaboration with Borgernes Hus will be of assistance to future iGEM teams from SDU as well as other students from Odense.
</P></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="education-and-public-engagement"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-practices"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Education &amp; Public Engagement</SPAN></H2><H4>- <I>A Trip to the Future and Beyond!</I></H4><P><I>If you want change, look to the future!</I> Such was the wording of our core philosophy. A philosophy that was carried out, by <SPAN class="highlighted">reaching out to the people of our society to ensure the engagement of the next generation, within the world of synthetic biology.</SPAN>
Ever since World War II, the West has seen an expansion and intensification of anti-scientific sentiment, which today primarily concern Genetically Modified Organisms (GMO). We will for that reason explore GMO’s role in history, to see if a historical perspective will allow us reach a new understanding of these sentiments. You can read all about it <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#historical-perspective">here</SPAN>.</P><DIV class="modal fade" id="historical-perspective" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">A Historical Perspective on GMO</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>From Food Concerns to Sustainable Energy</B></P><P>When talking about the history of GMO, we have the rare privilege of choice, as  depending on how you define GMO, One could potentially choose any historical period as their starting point. For instance, we could be examining the breeding of wolves to dogs, or simple agriculture, where the bad seeds are destroyed in favour of higher quality crops<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" http://sitn.hms.harvard.edu/flash/2015/from-corgis-to-corn-a-brief-look-at-the-long-history-of-gmo-technology/">Rangel G. From Corgis to Corn: A Brief Look at the Long History of GMO Technology. Genetically Modified Organisms and Our Food. August 2015.</A></SPAN></SPAN>
For the purpose of narrative, we will be starting our examination with the dawn of the 20<SUP>th</SUP> century. A time in which stale and rotten food lead to health concerns. Through several years, the public was continuously made aware of health problems regarding food in the United States. This culminated in the 1906 statute:, which allowed the inspection and ban of products<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.revolvy.com/main/index.php?s=Federal%20Food%20and%20Drugs%20Act%20of%201906&amp;item_type=topic"> JP S. The 1906 Food and Drugs Act and Its Enforcement. FDA History - Part I US Food and Drug Administration. 2013.</A></SPAN></SPAN>. During the time leading up to this act, the public grew to appreciate items labeled as pure. A trend we can observe in most journalistic material from the time period<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://chroniclingamerica.loc.gov/lccn/2010270501/"> O’Keefe TJ. The Alliance herald. : (Alliance, Box Butte County, Neb.) 1902-1922. T.J. O'Keefe; 1902.</A></SPAN></SPAN>
This need for pure food naturally originated partly from several years of unsatisfactory food and health concerns<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.revolvy.com/main/index.php?s=Federal%20Food%20and%20Drugs%20Act%20of%201906&amp;item_type=topic"> JP S. The 1906 Food and Drugs Act and Its Enforcement. FDA History - Part I US Food and Drug Administration. 2013.</A></SPAN></SPAN>, and partly from a political fiction: The Jungle, by Upton Sinclair<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.abebooks.com/Jungle-Sinclair-Upton-Grosset-Dunlap-New/1291165375/bd"> Sinclair U. The Jungle. New York: Grosset &amp; Dunlap,; 1906 1906.</A></SPAN></SPAN>. 
The readers became aware of unsanitary practices in the American food industry. Sinclair was considered a whistleblower, having exposed the practices of the food industry. After the act of 1906, we can observe a rise in concerns over the purity of food, which in turn lead to a focus on maintaining a certain standard in foods. Over time, this concern became the driving force behind the development of pesticides, and to a certain extent, GMO. With the introduction of these solutions to food purity, a new and opposite response could quickly be observed in the public. With pesticides and GMO, the public quickly grew concerned that food would be unnatural, and thus we can observe what is known as a reverse halo effect<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.newyorker.com/tech/elements/the-psychology-of-distrusting-g-m-o-s"> Konnikova M. The Psychology of Distrusting G.M.O.s. THE NEW YORKER. 2013.</A></SPAN></SPAN>.  A reverse halo effect is defined by the unconscious creation of a negative assumption based on a positive evaluation. In this case, the positive evaluation, would be the ensured purity and health benefits of GMO foods, where the negative assumption would be the concerns for <I>unnaturalness</I>.
So we detect a tendency in the public to follow a reverse halo effect, which is most often associated with GMO foods. The problem we face today, is the continuation of this reverse halo effect, where the public would receive any GMO device negatively, simply by association to the word GMO. So how do we change this dynamic between GMO and the public? 
The conclusion we reached was that in order to change a tendency in public opinion, it was important to educate the coming generations on the possibilities of GMO. Another important aspect seemed to be a general perception of GMO as an unnatural element that would harm both humans and nature alike. As such, we wanted to illustrate how GMO could be used, not to harm nature, but to help lead it towards a more sustainable future. So, we set out to teach children of all ages about GMO in general, as well as about our own PowerLeaf, to show them that this concept, which is considered to be unnatural, could actually be used to help nature.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><B><SPAN class="”highlighted”">Danish Science Festival</SPAN></B></P><P><SPAN class="highlighted">At the Danish Science Festival we hosted a workshop for kindergarteners</SPAN>, during which we taught them about <SPAN class="highlighted">synthetic biology, sustainability, the history of GMO, and bioethics.</SPAN> The children would in turn teach us as well, as they showed us the endless possibilities for bacteria designs,  through <SPAN class="highlighted">the “Draw-a-Bacteria”-contest.</SPAN> This <A href="#inspiration-from-children" target="_blank">inspired</A> us to reevaluate our initial idea.</P><P class="P-Larger"><B><SPAN class="”highlighted”">School Project Interview with 6<SUP>th</SUP> Graders</SPAN></B></P><P>Following the Danish Science Festival, we were contacted by two enthusiastic 6<SUP>th</SUP> graders, Bastian and Magnus. The two boys wanted to learn more about iGEM and GMO, which they intended to write about in a school project. They were curious to what range GMO could be used, and how we utilised it in our project, the PowerLeaf.</P><P class="P-Larger"><B><SPAN class="”highlighted”">UNF Summer Camp</SPAN></B></P><P>The UNF Summer Camp is an opportunity for high school students to show extra dedication towards science. We talked to some of the brightest young minds imaginable, all of whom aim to work in different fields of science in the future. <SPAN class="highlighted">At the summer camp, we held a presentation about our project, the iGEM competition, as well as how to handle and work with genes. We taught them how to assemble BioBricks and provided them with BioBricks for DNA assembly experiments, creating a ‘hands-on’ experience for these enthusiastic teenagers.</SPAN>
One of the high school students suggested that the Powerleaf should be able to rotate according to the sun, to ensure maximum exposure and outcome. We took this brilliant advice into consideration and contacted Robot Systems Engineer student, Oliver Klinggaard, who helped us with the potential implementation of a pan/tilt system. He provided us with his recent project report on the subject, as well as a description of the adjustments required for the implementation in our system, which you can find <A href="https://static.igem.org/mediawiki/2017/9/99/T--SDU-Denmark--PanTilt-system.pdf" target="_blank">here</A>.

Two students from the UNF Summer Camp thought the PowerLeaf was an interesting approach to sustainable energy, and they wanted to hear even more! So, they contacted us in late October, as they were interested to work on a project about green technology.</P><P class="P-Larger"><B><SPAN class="”highlighted”">The Academy for Talented Youth</SPAN></B></P><P><SPAN class="highlighted">We hosted a workshop for the Academy for Talented Youth, an association for some of the most talented high school students in Denmark.</SPAN> During the workshop we invited the students into our laboratories, where they conducted a miniprep and a gel electrophoresis on bacteria containing our BioBricks. Additionally, we held a presentation and discussion about our project, with the dedicated students. We strongly believe in mutual communication and made sure to compile feedback, all of which was positive! </P><P class="P-Larger"><B><SPAN class="”highlighted”">Presentations for the Local Schools</SPAN></B></P><P><SPAN class="highlighted">The local high schools, Mulernes Legatskole and Odense Tekniske Gymnasium, invited us to present our project, in addition to starting discussions with the students about GMO.</SPAN><SPAN class="highlighted">An 8<SUP>th</SUP> grade class from the local public school, Odense Friskole, were invited to see our laboratory workspace.</SPAN> It was a challenge to successfully convey our project and the concept of synthetic biology in a way that would be easily understandable by 8<SUP>th</SUP> graders, who have only recently been introduced to science. A challenge that we accepted and solved, by relaying the fundamentals in synthetic biology, e.g. the basics of a cell, DNA, and GMO.
<SPAN class="highlighted">From all of these presentations and interactions with younger individuals, we had a strong intuition that it had made an influence on their awareness of synthetic biology. This intuition was supported by the positive feedback provided by teachers and students.</SPAN> An awareness of how new scientific technologies can be a feasible solution to a possible energy crisis. Technologies such as synthetic biology, with endless capabilities to achieve efficacy, since no one knows what tomorrow brings. For more information about this read <A href="#future-igem-teams" target="_blank">To Future iGEM Teams</A></P><P class="P-Larger"><B><SPAN class="”highlighted”">Final Presentation at SDU-Denmark</SPAN></B></P><P><SPAN class="highlighted">The day before we travelled to Boston, we booked one of the big auditoriums at the University of Southern Denmark, for the final rehearsal of our jamboree presentation.</SPAN> We made sure to take note of all the feedback and tips we received, while also implementing these into our final presentation. This event was promoted on all the information screens at our university in order to attract a broad audience and increase the interest for iGEM. Thus, making it possible to reach a substantial amount of future applications for the SDU-iGEM team and <SPAN class="highlighted">ensure that the iGEM spirit will continue to prosper in the future!</SPAN></P><P class="P-Larger"><B><SPAN class="”highlighted”">Social Media</SPAN></B></P><P>Social media is an easy way to impact a high number of people, so a strategy was concocted with the intention of reaching as many people as possible with our outreach.
<SPAN class="highlighted">Our strategy yielded marvellous results, amongst which was a video on our project, that reached viewers equal to 16% of our hometown’s population, along with becoming the second most seen bulletin of the year from University of Southern Denmark.</SPAN> They have also asked us to film our experiences at the Jamboree, which will feature on the homepage of the student’ initiative BetonTV. 
<SPAN class="highlighted">Several articles were written about our project in local newspapers, one was even featured in the saturday special.You can read all about our social media strategy and results </SPAN><SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-social-media">here</SPAN>. The commercial can be seen right here:</P><DIV class="modal fade" id="about-social-media" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Social Media Report</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><DIV style="text-align:center;"><P class="raleway P-Larger"><I>&quot;I hate social media. I have Twitter, just so I can tell people what to buy.”</I></P><P class="raleway"><I>Louis C. K. </I></P></DIV><P class="P-Larger"><B>Introduction</B></P><P> 
As part of our iGEM project we wanted to raise awareness on synthetic biology. As such it seemed beneficial to have a successful social media strategy. The motivation for such a strategy came from the knowledge that social media in general has a large influence on people, especially the younger generation.  
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.emeraldinsight.com/doi/abs/10.1108/02634500810902839">Opportunities for green marketing: young
consumers</A></SPAN></SPAN>.
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.businessnewsdaily.com/4373-young-better-earth-day-targets.html">Social Media Influencing Green
Buying</A></SPAN></SPAN></P><P class="P-Larger"><B> Our Strategy</B></P><P>
The design of our logo consisting of a light font and a picture of a leaf combined with an electrical plug, was determined in the start of April. We sought to make a simple logo that showed the essence of our project, The PowerLeaf, not only in name but in brand as well.

During spring we narrowed down the relevant hashtags of our Twitter and Instagram to #science, #igem, #sdu, #gmo, and #forsksdu. These hashtags were used at relevant occasions and generally helped expanding our reach on social media. A few times we even used a couple of silly hashtags like #adventuretime and #lifechoices when we found it appropriate.

A complete relaunch of our social media platforms was made in the start of April. One of the changes we made was the setup of the Facebook page and Twitter account, that we changed to resemble an enterprise.
</P><P class="P-Larger"><B> Results</B></P><DIV class="row"><DIV class="col-md-6"><P> Looking at Figure 1 we see the number of followers on our Facebook page throughout the year. There is a sharp increase around the start of April, this correlate nicely with the relaunch of our social media. Since we started using the Facebook page on March 1<SUP>st</SUP>, we have seen an increase of followers on 44%.

Figure 2 shows the number of unique interactions with our Facebook posts each day. Some posts gained around 200 interactions per day, while the highest amount of interactions per day was seen at end of April. This high amount of interactions correlates with us being at the Danish Science Festival, where we inoculated bacteria from the fingers of the attendants on agar plates. The photo gallery of these plates was widely shared on our Facebook page, and it probably was the reason for the rise in amounts of interactions.

During the summer, we realised that an increasing fraction of our followers on Facebook were non-Danish speakers. Therefore, we set out to analyse the distribution of first languages among our followers seen in Figure 3, which showed that one in three of our followers were not speaking Danish as their first language. This data led to us switching the language used in our Facebook posts from Danish to English. </P></DIV><DIV class="col-md-6"><DIV class="figure-text"><P><B>Figure 1.</B> Number of followers on our Facebook page throughout the year.</P></DIV><DIV class="figure-text"><P><B>Figure 2. </B> Unique interactions per day on our Facebook page. </P></DIV><DIV class="figure-text"><P><B>Figure 3.</B> Distribution of first language among our followers</P></DIV></DIV></DIV><P>On the 8<SUP>th</SUP> of September a commercial about our project aired on the website and Facebook page of the University of Southern Denmark. We made this commercial in collaboration with the communications department of the University of Southern Denmark, specifically a producer named Anders Boe. It was seen 27,526 times, which is equivalent to 16% of our hometowns population. The click-through rate was around 10%, which is much greater than the average rate of videos on their Facebook page. The commercial led to random people, including doctors on the hospital of Odense, asking team members about the project and about GMO in general. With all this in mind it seems fair to state that the commercial was a smash hit of our human outreach and our engagement in the social media. In combination with our easily recognisable logo and our human outreach initiatives, we have been using our social media as a platform to show interest in synthetic biology and for sharing our work.

After seeing the commercial a local newspaper reached out to us to bring an interview in their Saturday special <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.fyens.dk/indland/Forsoemt-sommer-Studerende-bruger-ferien-paa-solceller-af-bakterier/artikel/3180933">Newspaper article from Fyens.dk (Danish)</A></SPAN></SPAN>. In collaboration with the student media RUST and Beton TV we have been featured in their magazine and a video of our attendance in Boston will be shown on their Facebook page briefly after our return. This is done to ensure an awareness of the iGEM competition among the students on the University of Southern Denmark. We hope this will inspire a lot of students to apply for the iGEM team next year, since it is a hallmark of the SDU-Denmark team to be interdisciplinary.

The effort put into our social media strategy and our goal of following it has paid off. Social media has shaped our outreach and made it possible for us to talk about iGEM and our project to people we did not know. Several times our outreach has led us to public discussion of GMO and solar panels due to persons contacting us, wanting to know more. 
 </P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="prospectsContent"><DIV id="prospects"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlineProspects" class="headline"><SPAN class="highlighted">Prospects</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><P><SPAN class="largeFirstLetter">T</SPAN>he aim of our prospect section is to expand on the vision of the PowerLeaf; a vision we would love to see realised. <SPAN class="highlighted">An overview of the project has been created, in the hope that it will benefit future iGEM teams</SPAN>. Additionally, it is aimed to assist iGEM teams-to-be, should they wish to take the PowerLeaf to the next level.  
</P></DIV></DIV></DIV><DIV class="topShadow" id="perspectives"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-prospects"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Perspectives</SPAN></H2></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>Building a Product for a Better Future</B></SPAN></P><P>The purpose of the PowerLeaf is to provide a greener alternative to the currently available sources of energy. An important aspect of such an undertaking is to <SPAN class="highlighted">limit the use of depleting resources</SPAN> in the construction of the device itself. This is accomplished through the use of the most common resources available, namely recyclable plastics and bacteria. In turn, this will contribute to our dream of building a better future. A future where fear of reaching a critical shortage of natural resources has been eliminated. Another benefit of the PowerLeaf is the bacteria's ability to self-replicate, if provided with the essential nutrients.

As tools for genomic editing improves, the advancement of biological devices will conceivably become even more complex and independent. They will do so by <SPAN class="highlighted">introducing new metabolic pathways originated from other organisms using genetic engineering. This could potentially allow the PowerLeaf to become completely independent</SPAN> through its self-replication. Independency would occur, when the bacteria are modified to produce their own essential nutrients directly from unwanted pollution in the environment. This trait would lead to cleaner cities, along with providing a natural solution of sustainable energy.
    </P><P class="P-Larger"><SPAN class="highlighted"><B>Genetic Code Expansions for Biological Engineering</B></SPAN></P><P>In an effort to advance the current technologies used in synthetic biology, research groups are working on genetic code expansion. We had an interesting talk from postdoc and former iGEM participant, Julius Fredens, about his work on genetic code expansion. Once a technology like this expands, it will completely revolutionise biological engineering, including that of the PowerLeaf. <SPAN class="highlighted">Genetic code expansion could be used for optimisation of the systems in the PowerLeaf, such as optimisation of nanowires, improvement of the light-sensing system, and making the breakdown of cellulose inducible</SPAN>.
    </P></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="future-igem-teams"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-prospects"><DIV class="col-xs-12"><H2><SPAN class="highlighted">To Future iGEM Teams</SPAN></H2><P>Hello future iGEM’er and congratulations on starting your iGEM journey! You are going to have an amazing time with plenty of wonderful experiences, new friendships, and an extensive amounts of obtained knowledge. In this section some ideas for <SPAN class="highlighted">improvement and further development of our project, the PowerLeaf</SPAN>, will be presented.</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>Further Development of Our Project</B></SPAN></P><P>For those of you that are interested in our project and would like to improve upon it, this is the section you have been looking for. We have listed the systems and the related information needed for the fulfillment of the device we envisioned. Nonetheless, you should not feel restricted by these suggestions. You shall be more than welcome to contact any of us regarding questions about the project. You can find each of our team members contact informations in the <A href="#team" target="_blank">Team section</A> in the credits.</P><P style="font-size:20px;"><SPAN class="highlighted"><B>Investigated and Researched Systems</B></SPAN></P><UL class="list"><LI><SPAN class="highlighted"><B>Dormancy System</B></SPAN>
Utilised by the energy storing unit, the dormancy system can reduce metabolism during times of the day with no solar energy available, e.g. at nighttime. We encountered a few challenges during the assembly and optimisation of the dormancy system. However, through modeling we obtained vital knowledge on how to regulate the system. If you are interested, you can read more about the work performed on the light-dependent <A href="#project-design-dormancy-system">dormancy system</A>.</LI><LI><SPAN class="highlighted"><B>Carbon Fixation</B></SPAN>
We received the required parts from the <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec" target="_blank">Bielefeld 2014 iGEM team</A> and began assembling the parts into one fully functional BioBrick. 
However, we encountered some trouble cloning these parts. You can give it a go anyways, or maybe even try to redo the carbon fixation by using a system from a different organism. Regrettably, we eventually had to let go of this system, so we could focus on other components of the PowerLeaf. If you are interested, you can read more about the work we did regarding the <A href="#project-design-carbon-fixation">carbon fixation</A>.</LI><LI><SPAN class="highlighted"><B>Cellulose Biosynthesis and Secretion</B></SPAN>
These parts were retrieved from the <A href="https://2014.igem.org/Team:Imperial" target="_blank">2014 Imperial College iGEM team</A>. Assembly of these long BioBricks emerged to be troublesome, which regularly occurs when cloning long sequences of DNA. Thus, this part could be the very thing to improve, as it is a rather central part of the system. If you are interested, you can read more about our work on the <A href="#project-design-cellulose-biosynthesis">cellulose biosynthesis and secretion</A>.</LI><LI><SPAN class="highlighted"><B>Cellulose Breakdown</B></SPAN>
Employed by the energy converting unit, this system degrades cellulose to glucose, from which electrons could be retrieved. After extensive work the system showed promising results as our bacteria were able to grow on cellobiose. If you are interested, you can read more about the work we did regarding the <A href="#project-design-cellulose-breakdown">cellulases</A>.</LI><LI><SPAN class="highlighted"><B>Extracellular Electron Transfer</B></SPAN>
Inspired by a previous study <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://mbio.asm.org/content/8/1/e02203-16.full"> Tan Y, Adhikari RY, Malvankar NS, Ward JE, Woodard TL, Nevin KP, et al. Expressing the Geobacter metallireducens pilA in Geobacter sulfurreducens Yields Pili with Exceptional Conductivity. mBio. 2017;8(1).</A></SPAN></SPAN>, we set out to create the required BioBricks for the system. However, to successfully implement optimized nanowire in <I>G. sulfurreducens</I>, further work would have to be performed. If you are interested, you can find additional information about the work we did regarding the <A href="#project-design-extracellular-electron-transfer">nanowires</A>.</LI></UL><P style="font-size: 20px;"><SPAN class="highlighted"><B>Subjects that Ought to be Implemented in the Device</B></SPAN></P><UL class="list"><LI><SPAN class="highlighted"><B>ATP Production from Solar Energy</B></SPAN>
Harvest of solar energy comes to mind as one of the most essential systems needed for the PowerLeaf to <I>actually</I> work. We had to select parts of the PowerLeaf to work on, and at the end of the day, we decided to cut this subpart. Instead, we had a great Skype call early on in the process with the <A href="https://2017.igem.org/Team:Macquarie_Australia" target="_blank">Australian Macquarie iGEM team</A>, who has been working for several years with the implementation of the photosynthetic systems in <I>E. coli</I>.</LI><LI><SPAN class="highlighted"><B>Cellulose and the Cellulases forming an On/Off Switch</B></SPAN>
This is also a very crucial part of the PowerLeaf, since it would otherwise be generating an electrical current non-stop, even when not needed. Thereby overthrowing the potential for long term-storage of solar energy. We believe this can be solved either through precise gene circuit regulation or by physical compartmentalisation, but there might be even more elegant ways in which to handle this challenge.</LI><LI><SPAN class="highlighted"><B>Physical Engineering of the Hardware</B></SPAN>
It should be possible for the energy storing unit to convert CO<SUB>2</SUB> into cellulose, thereby producing O<SUB>2</SUB> making its chamber aerobic. For the energy converting unit to effectively transfer retrieved electrons to an anode, it requires anaerobic conditions. Thus the removal of O<SUB>2</SUB> from the device is an obstacle to overcome. It will require some out-of-the-box thinking to come up with a novel idea without having to require more energy than produced.To make a workable prototype of the PowerLeaf, proper engineering of the hardware is necessary. This includes elements such as anodes, chambers, circulation of important nutrients, removal of wastes, and the use of appropriate materials. We worked out the optimal type of plastic for the system with the help of a local expert. You can read about our work regarding the plastic <A href="#plastic-expert" target="_blank">here</A>.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="creditsContent"><DIV id="credits"><DIV class="contentHeadline"><DIV class="centerContentHeadline"><P id="headlineCredits" class="headline"><SPAN class="highlighted">Credits</SPAN></P></DIV></DIV><DIV class="row presentationBackground "><DIV class="col-md-8 col-xs-10 margin-bottom-200 margin-top-50"><DIV style="text-align:center;"><P class="raleway P-Larger"><I>“You'll stay with me?”</I> - <I>“Until the very end”</I></P><P class="raleway" style="text-align:center;"><I>J.K. Rowling, Harry Potter and the Deathly Hallows</I></P></DIV><P><SPAN class="largeFirstLetter">J</SPAN>ust like in the movies, you only get to meet the brilliant minds behind the project in the closing credits. Some might leave the cinema without sitting through the credits, but we hope you will sit through ours, as you will get to know us on a more personal level. We probably have more in common than you think. And do not forget that <SPAN class="highlighted">behind every great team is an equally great amount of external <A href="#attributions" target="_blank">attributions</A></SPAN>. The contributors have supported and inspired us, especially when things have been rough and deadlines closeby. <SPAN class="highlighted">When you finish this section, we kindly ask that you turn your attention to our various <A href="#collaborations" target="_blank">collaborations</A></SPAN>, all of which have been amazing experiences. They truly show of that wonderful iGEM spirit!

Finally, <SPAN class="highlighted"> we do not want you to miss out on the ‘after-the-credits-clip’ that summarises the fun we had during this fantastic iGEM adventure.</SPAN>  This is will be the moment you will get that long-awaited ‘thank you for listening, we hope you enjoyed our wiki and project’. 
</P></DIV></DIV></DIV><DIV class="topShadow" id="team"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10"><DIV class="row border-left-credits"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Team</SPAN></H2><P>Welcome to the team page! Here you will get to know us on a more personal level. Our team is made up by <SPAN class="highlighted"> 12 students from 8 very different majors. </SPAN> As friends, we experienced the most amazing summer together, filled with various fun activities, both in- and outside the lab. We would like to mention a few. We had road trips, dinners, Game of Thrones nights, heck, we even celebrated Christmas in July! All of this was shared with our amazing supervisors, to whom we are truly grateful.
    </P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Ellen Gammelmark</B></SPAN></P><P class="raleway">Study: Biochemistry and Molecular Biology
E-mail: elgam15@student.sdu.dk
Why, hello there! My name is Ellen and I have spend most of my waking hours either in the lab with a pipette in hand, or just outside the lab with a computer on my lap. You know - learn iGEM, live iGEM, love iGEM!</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Emil Bøgh Hansen</B></SPAN></P><P class="raleway">Study: Biology
E-mail: ehans15@student.sdu.dk
Howdy! I’m the first of many Emil’s and the team's only biologist! I am a huge wolf enthusiast! This summer I left my boots in the closet, in order to put on a proper lab coat and suit up for iGEM. In addition to my time in the lab, I have also looked into how GMO can influence the environment and what we need to do to ensure a safe iGEM project.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Emil Søndergaard</B></SPAN></P><P class="raleway">Study: History
E-mail: emsoe09@student.sdu.dk
Ahoy thar! My name is Emil, and I want to be the next Indiana Jones. But before I can raid any tombs, I’ve decided to raid iGEM trophies. When I’m not cooking or travelling, I’m drawing on my background in history for communications and human practices.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Emil Vyff Jørgensen</B></SPAN></P><P class="raleway">Study: Physics
E-mail: ejoer15@student.sdu.dk
Mojn! I am yet another Emil. I might not be a model biochemist, so instead I am modelling biochemistry! My iGEM existence is a stochastic binary function between naps and extreme bursts of energy.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Felix Boel Pedersen</B></SPAN></P><P class="raleway">Study: Biochemistry and Molecular Biology
E-mail: feped15@student.sdu.dk
Aloha. My name is Felix and I bring joy to others by eating my daily ryebread with paté and wearing my magical red racer rain coat. Speaking of magic, I’m the team’s wiki lizard (get it?). I also do dry-lab, and whenever I miss the “sunlight”, I kindly join the others in the wet lab.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Frederik Bartholdy Flensmark Neergaard</B></SPAN></P><P class="raleway">Study: Biochemistry and Molecular Biology
E-mail: frnee15@student.sdu.dk
Hey yo! I’m Frederik and I have worked day and night on iGEM, mostly drinking beers at night time, but that should count as well. When I’m not working in lab or on the PC, I have fun with my teammates and tell bad dad jokes. I also make crazy ideas come true - like celebrating Christmas in July.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Frederik Damsgaard Højsager</B></SPAN></P><P class="raleway">Study: Medicine
E-mail: frhoe14@student.sdu.dk
Heyah! I’m the other Frederik. I’m a green, lean, coffee-machine. I’ve been the steady supplier and consumer of coffee on the team. My main focus has been on how to build a sustainable iGEM project. I’ve been planting trees, eating green and lowering our team's carbon footprint. Oh, and did I also mention that I starred in our commercial? You can get autographs later.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Jonas Borregaard Eriksen</B></SPAN></P><P class="raleway">Study: Pharmacy
E-mail: jerik15@student.sdu.dk
Hey sup? I’m Jonas and used to like sports, partying, eating cake, hanging out with friends and other such things most people like to do. During iGEM these interests have changed… As I have been enslaved in the lab, I’ve come to realise that the only purpose of my life is to be in the lab.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Lene Vest Munk Thomsen</B></SPAN></P><P class="raleway">Study: Philosophy
E-mail: letho11@student.sdu.dk
Hey, is it solipsistic in here, or is it just me? When not wondering whether or not there is an external world, I’ve been busy working out how to implement our solar battery into our local community and what to gain from doing so. Oh, and imposing metaethics on my team members, but I Kant go into detail with this here.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Malte Skovsager Andersen</B></SPAN></P><P class="raleway">Study: Biochemistry and Molecular Biology
E-mail: malta14@student.sdu.dk
Eyy, I’m Malte. I’ve mostly been working in the lab wrapped in the dankest of lab coats, doing the most exciting of experiments. All in the name of why the heck not. In the lab the utmost highest level of patience is needed, especially when tasked with testing if BioBricks function as intended. This has, as seen in the image, sadly caused me to pull out most of my hair.</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left right"><P class="P-Larger"><SPAN class="highlighted"><B>Sarah Hyllekvist Jørgensen</B></SPAN></P><P class="raleway">Study: Biochemistry and Molecular Biology
E-mail: sajo415@student.sdu.dk
Despite my favorite occupation is digging into literature, my main attribution to our project has been to run around in the lab. Luckily, there is a clear link between wet- and dry-lab. Even though I am the smallest member of the SDU iGEM team, I have definitely risen to the occasion. iGEM has been an amazing period of my life!</P></DIV></DIV><DIV class="row team-rows"><DIV class="col-xs-8 memberText verticalAlignColumns padding0 left"><P class="P-Larger"><SPAN class="highlighted"><B>Sofie Mozart Mortensen</B></SPAN></P><P class="raleway">Study: Biomedicine
E-mail: sofmo15@student.sdu.dk
Hi there! My name is Sofie, and I am the team mama! I am the one who makes sure everyone gets their fair share of cake and baked goodies. When I’m not in the kitchen busy making cakes for my teammates, you can find me in the lab with a pipette in my hand.</P></DIV></DIV><P class="P-Larger"><SPAN class="highlighted"><B>Project Synergy</B></SPAN></P><P>We have all been working together in every aspect of our project. Nevertheless, <SPAN class="highlighted">some people have had to focus on some areas more than others.</SPAN>  The main groups are listed as follows;</P><UL class="list"><LI>The group <SPAN class="highlighted">focusing on fixation of CO<SUB>2</SUB>, production of cellulose and light-dependent dormancy system</SPAN>  consisted of Sarah Hyllekvist Jørgensen, Ellen Gammelmark, Sofie Mozart Mortensen and Emil Bøgh Hansen.</LI><LI>The group <SPAN class="highlighted">focusing on the breakdown of cellulose to create an electrical current and the optimisation of nanowires</SPAN>  consisted of Felix Boel Pedersen, Frederik Bartholdy Flensmark Neergaard, Jonas Borregaard Eriksen and Malte Skovsager Andersen.</LI><LI>The group <SPAN class="highlighted">focusing on the implementation of the device in an urban environment as well as our outreach</SPAN>  consisted of Emil Søndergaard, Frederik Damsgaard Højsager and Lene Vest Munk Thomsen.</LI><LI>The <SPAN class="highlighted">mathematical modelling of our project</SPAN>  was single-handedly performed by Emil Vyff Jørgensen.</LI><LI><SPAN class="highlighted">Coding and design of the wiki</SPAN> was performed by Felix Boel Pedersen and Frederik Damsgaard Højsager.</LI></UL><P>To ensure a good team spirit and dynamic we formulated a <A href="https://static.igem.org/mediawiki/2017/d/d1/T--SDU-Denmark--Cooperation_agreement.pdf" target="_blank">cooperation agreement</A>.
 
Nothing can be done alone, so please scroll further down, to read about all the people who contributed to making our project successful. We are so grateful for all the help and support we have been offered throughout our iGEM experience.</P></DIV></DIV></DIV><DIV class="topShadow" id="collaborations"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-credits"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Collaboration</SPAN></H2><DIV style="text-align:center;"><P class="raleway P-Larger"><I>&quot;Alone we can do so little; together we can do so much&quot;</I></P><P class="raleway"><I>Helen Keller </I></P></DIV><P>The American author Helen Keller had it right! <SPAN class="highlighted">As an iGEM team, you can reach many goals, but as an entire community, we can aspire to achieve so much more</SPAN>. We would like to thank all the people that made our iGEM experience so memorable, we truly enjoyed your companionship!</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>Danish Ethics and Wiki Workshop at SDU</B></SPAN></P><P>In the spirit of the iGEM community, <SPAN class="highlighted">we hosted a meetup in August for our fellow Danish iGEM teams: InCell from the University of Copenhagen (KU), and the Snakebite Detectives from the Technical University of Denmark (DTU)</SPAN>.  A total of seven members from these two teams joined us for breakfast and attended our meetup. This was the first ever iGEM meetup hosted by our university, so we decided to make it memorable. We took advantage of our interdisciplinary team roster, and thus designed <SPAN class="highlighted">a wiki and ethics workshop to aid our fellow Danish teams</SPAN>. 

We utilised the broad interdisciplinary profile of our team, to have Emil S. and Lene <SPAN class="highlighted">present the perception of science throughout the history and the meta-bioethical aspects of GMO</SPAN>, respectively. Emil S. has a Bachelor of Arts in History, and Lene has a Bachelor of Arts in Philosophy. The ethical presentation was purposely turned into an ethical debate, where viewpoints of ethical conduct were exchanged and discussed. After the presentations and discussions on metaethics, it was time for the wiki workshop.

The SDU-Denmark iGEM teams have won the Best Wiki prize several times in the past. As such, we wanted to share the knowledge gained from our university's experience. To facilitate this exchange of knowledge on wiki development, we recruited our current supervisor Thøger Jensen Krogh, to facilitate presentations on how to design a good wiki page. Thøger was qualified for this task through his role as the designer of the SDU iGEM 2013 and 2014 team wikis, both of which won the special prize on both occasions. During the presentation, Thøger had arranged several exercises where <SPAN class="highlighted">the attendees got to mingle, discuss and evaluate their wikis. This resulted in a steady flow of information and constructive feedback between all three teams</SPAN>. 

After a long day of learning and discussing, we went for a tour around campus under the summer sun. This concluded in a visit to the roof terrace of the campus dormitory, followed by dinner. It was requested, by our fellow Danish teams, to make the SDU meetup an iGEM tradition. 
    </P><P class="P-Larger"><SPAN class="highlighted"><B>Attending Meetups</B></SPAN></P><P>Besides hosting our own meetup, <SPAN class="highlighted">we also attended several meetups during our iGEM experience</SPAN>. The first of which, was the 5th Annual Biobrick Workshop in March, hosted by the Technical University of Denmark, DTU-Denmark. This meetup not only gave us our first experience with Biobricks, but also worked as a foundation for friendships across different teams.

Our second meetup, the Nordic iGEM Conference was hosted by the University of Copenhagen, UCopenhagen, and took place in June. The main focus of this meetup was the traditional mini Jamboree. Participating in this  meetup gave us useful feedback from both the judges as well as from our fellow iGEM teams. This helped us greatly to shape and develop our project for the better.

To celebrate the beginning of our iGEM summer, we went on a road trip to attend the European Meetup, hosted by the Delft University of Technology in the Netherlands. Here we discussed ideas regarding our project at a poster session, and learned about all the other great iGEM projects. We also made new friends from all over Europe.
</P><P class="P-Larger"><SPAN class="highlighted"><B>Further Collaboration</B></SPAN></P><P>In regard to our project, we have <SPAN class="highlighted">been in contact with the iGEM teams from Bielefeld and Imperial College, who helped us by sending crucial parts, relevant to the execution of our project</SPAN>. 

As our project revolves around global warming and green sustainable energy, we were thrilled to hear about <SPAN class="highlighted">the iGEM Goes Green initiative made by the TU Dresden iGEM team</SPAN>. Following their guidelines, we have calculated the carbon footprint of our laboratory work and travelling. We have, in part, tried to make up for our carbon footprint, by changing our travelling and eating habits in our everyday lives. Furthermore, <SPAN class="highlighted">we have reduced our daily electricity consumption, our wiki became CO<SUB>2</SUB> neutral, and we made an effort to sort our waste</SPAN>. The full report can be scrutinised <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#about-igem-goes-green">here</SPAN>. Due to our team being the most green <I>dream</I> team, TU-Dresden asked us to lead the iGEM Goes Green project in year 2018. 
<SPAN class="highlighted">We sought expertise from the Macquarie Australia iGEM team</SPAN>, who has worked with the implementation of photosynthesis in <I>E. coli</I> since 2013. We had an interesting Skype call with their team, where we discussed the particular challenges their previous teams had experienced throughout their projects. 

We were also <SPAN class="highlighted">able to help the Stony Brook iGEM team by facilitating communication with members of the SDU iGEM team from 2016</SPAN>. 

During our project we received several questionnaires from fellow iGEM teams. We were <SPAN class="highlighted">delighted to help the teams by answering their questionnaires</SPAN>. These included from:</P><UL class="list"><LI>Waterloo - regarding 3D printing of lab equipment</LI><LI>Dalhousie - regarding the common conception of science literature</LI><LI>University of Washington - regarding communication platforms used by teams</LI><LI>Vilnius-Lithuania - regarding cotransformation</LI><LI>Nanjing-China - regarding a whole-cell sensor for formaldehyde</LI><LI>University of Sydney - regarding the use and accessibility of insulin</LI><LI>Georgia State - regarding disabilities</LI><LI>Greece - regarding modular RNAi-based logic circuits</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="attributions"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-credits"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Attributions</SPAN></H2><P>Nothing can be done alone, so please scroll further to read about the contributors, who helped make this project a reality.</P></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><SPAN class="highlighted"><B>Laboratory, Technical and General support</B></SPAN></P><P>We would like to give <SPAN class="highlighted">a special thanks to our supervisors:</SPAN></P><UL class="list mini-list"><LI><SPAN class="highlighted">Assistant professor, <STRONG>Mikkel Girke Jørgensen</STRONG></SPAN>, for his general support and advice on the project, the laboratory, the fundraising and our team synergy.</LI><LI><SPAN class="highlighted">Ph.D. student and former iGEM participant, <STRONG>Patrick Rosendahl Andreassen</STRONG></SPAN>, for his guidance and technical assistance in the laboratory.</LI><LI><SPAN class="highlighted">Ph.D student and former iGEM participant,  <STRONG>Thøger Jensen Krogh</STRONG></SPAN>, for his help in developing the wiki, as well as his laboratory guidance.</LI><LI><SPAN class="highlighted">Cand.phil student and former iGEM participant, <STRONG>Tim Munk</STRONG></SPAN>, for his focus on team dynamics and advice for our human practices.</LI></UL><P><SPAN class="highlighted">We would also like to thank:</SPAN></P><UL class="list mini-list"><LI>Academic assistant, <SPAN class="highlighted"><STRONG>Tina Kronborg</STRONG></SPAN>, for her guidance in the lab, as well as for providing us with laboratory equipment.</LI><LI>Medical Laboratory Technician, <SPAN class="highlighted"><STRONG>Simon Rose</STRONG></SPAN>, for giving us a course in lab safety, risk assessment and general guidance in the lab.</LI><LI>Postdoc, <SPAN class="highlighted"><STRONG>Oona Sneoyenbos-West</STRONG></SPAN>, for providing us with <I>Geobacter Sulfurreducens</I> PCA and the necessary knowledge on how to grow this particular bacterial strain. Furthermore, she helped us greatly with helpful discussions regarding the advancement of our project. We would also like to thank her for lending us her laboratory, for the cultivation of <I>G. Sulfurreducens</I> PCA.</LI><LI>Postdoc, <SPAN class="highlighted"><STRONG>Satoshi Kawaichi</STRONG></SPAN>, for his assistance in measuring the electrical conductivity of our nanowires, as well as providing us with knowledge on the <I>G. Sulfurreducens</I>.</LI><LI>Business scout and PhD, <SPAN class="highlighted"><STRONG>Ann Zahle Andersen</STRONG></SPAN>, for presenting us with the necessary tools  for the development of innovative business ideas.</LI><LI>Stud.scient, <SPAN class="highlighted"><STRONG>Kristian Severin Rasmussen</STRONG></SPAN>, for helping us use the oCelleScope for testing.</LI><LI>Stud.scient, <SPAN class="highlighted"><STRONG>Brian Baltzar</STRONG></SPAN>, for hosting a workshop regarding the use of Adobe Illustrator, which has been a great help to the development of graphics for our wiki.</LI><LI>Ph.D student, <SPAN class="highlighted"><STRONG>Richard Xavier Etienne Valli</STRONG></SPAN>, for helpful discussions in the lab.</LI><LI>Software Developer, <SPAN class="highlighted"><STRONG>Jonas Hartwig</STRONG></SPAN>, for his help on some JQuery functionality on the wiki.</LI><LI>Stud.scient, <SPAN class="highlighted"><STRONG>Birka Jensen</STRONG></SPAN>, for general advice and suggestion on how to build an iGEM wiki.</LI><LI>Stud.med, <SPAN class="highlighted"><STRONG>Ida Charlotte Hvam</STRONG></SPAN>, for helpful discussions on the development of our wiki, helping with last minute figures to the wiki, as well as proofreading its content.</LI><LI>Stud.med, <SPAN class="highlighted"><STRONG>Maria Victoria Mikkelsen</STRONG></SPAN>, for helpful guidance regarding the composition of experiment pages and proofreading.</LI><LI>Stud.med, <SPAN class="highlighted"><STRONG>Liza Gaardsted Hansen</STRONG></SPAN>, for proofreading our wiki in the 11<SUP>th</SUP> hour.</LI><LI>Ph.D student and current iGEM advisor for the Bielefeld team, <SPAN class="highlighted"><STRONG>Boas Pucker</STRONG></SPAN>, for providing us with BioBricks created by former iGEM teams from Bielefeld.</LI><LI>Our iGEM HQ Representative, <SPAN class="highlighted"><STRONG>Traci Haddock-Angelli</STRONG></SPAN>, for her general guidance and assistance in registering our Danish Ethics and Wiki Workshop to the official iGEM meetup page.</LI><LI>iGEM HQ Representative and Lab Technician, <SPAN class="highlighted"><STRONG>Abigail Sison</STRONG></SPAN>, for her help in registering our Danish Ethics and Wiki Workshop to the official iGEM meetup page.</LI><LI>Stud.polyt, <SPAN class="highlighted"><STRONG>Oliver Klinggaard</STRONG></SPAN>, for helpful discussions on the implementation of a pan-tilt system and for providing us with his project report on the subject.</LI><LI><SPAN class="highlighted"><STRONG>DTU BioBuilders</STRONG></SPAN>, for hosting their 5<SUP>th</SUP> Annual Biobrick Workshop and for attending our Danish Ethics and Wiki Workshop.</LI><LI>The <SPAN class="highlighted"><STRONG>UNIK Copenhagen iGEM team</STRONG></SPAN>, for hosting the Nordic Meetup and for attending our Danish Ethics and Wiki Workshop.</LI><LI>The <SPAN class="highlighted"><STRONG>TU-Delft iGEM team</STRONG></SPAN>, for hosting the European Meetup.</LI><LI><SPAN class="highlighted"><STRONG>Mimo Antabi</STRONG></SPAN>, for adding our adverts to the University of Southern Denmark’s info screens preceding the Danish Science Festival.</LI><LI><SPAN class="highlighted"><STRONG>Anders Boe</STRONG></SPAN>, for putting an enormous amount of work into the postproduction of our commercial.</LI><LI><SPAN class="highlighted"><STRONG>Allan Haurballe Madsen</STRONG></SPAN>, for helping us with our appearance at the Danish Science Festival.</LI><LI>Outreach Coordinator and PhD, <SPAN class="highlighted"><STRONG>Lise Junker Nielsen</STRONG></SPAN>, for for helping us with the Danish Science Festival as well as with the visit from the Academy for Talented Youth. We would also like to thank her for providing us with iPads for laboratory use.</LI><LI><SPAN class="highlighted"><STRONG>The Danish Science Festival</STRONG></SPAN>, for having us at their annual event. We would also like to thank all the visitors who attended our booth, especially the children who participated in our ‘Draw-a-Bacteria’-contest.</LI><LI>The high schools <SPAN class="highlighted"><STRONG>Odense Technical Gymnasium</STRONG></SPAN>, <SPAN class="highlighted"><STRONG>Mulernes Legatskole</STRONG></SPAN>  and <SPAN class="highlighted"><STRONG>Academy for Talented Youth</STRONG></SPAN>, for letting us present our project and to engage in rewarding discussions about synthetic biology.</LI><LI>The <SPAN class="highlighted"><STRONG>UNF Biotech Camp</STRONG></SPAN>, for having us present our project to the attending students.</LI><LI>The elementary school, <SPAN class="highlighted"><STRONG>Odense Friskole</STRONG></SPAN>, for letting us present our project for their 8<SUP>th</SUP> grade students, thereby forcing us to contemplate how to convey a subject depending on the audience.</LI><LI><SPAN class="highlighted"><STRONG>All former iGEM participants from SDU</STRONG></SPAN>, for attending our preliminary presentation and giving us feedback before the Giant Jamboree.</LI><LI>The following groups and associations, for helping us develop our human practices: <SPAN class="highlighted"><STRONG>SP-Moulding</STRONG></SPAN>, <SPAN class="highlighted"><STRONG>Borgernes Hus</STRONG></SPAN>, <SPAN class="highlighted"><STRONG>Kommunens bygninger</STRONG></SPAN>, <SPAN class="highlighted"><STRONG>Bolbro - områdefornyelse</STRONG></SPAN>, <SPAN class="highlighted"><STRONG>Odense Byudvikling</STRONG></SPAN>.</LI><LI>Project manager (Smart City Odense), <SPAN class="highlighted"><STRONG>Kristina Dienhart</STRONG></SPAN>, for wonderful advice on how to involve Odense Municipality in our project. We want to thank her for our conversation on the implementation of The PowerLeaf in Odense. We would also like to thank her for leading us to the ongoing project of Bolbro’s city renewal.</LI><LI>Project Manager Bolbro City Renewal (DitBolbro), <SPAN class="highlighted"><STRONG>Rikke Flagreen Mortensen</STRONG></SPAN>, for meeting with us to discuss how The PowerLeaf could be implemented in the neighbourhood of Bolbro. We would also like to thank her for providing us with a pitch and reading-materials on Odense’s green goals.</LI><LI>Director of audience - and Central Library Odense (Borgernes Hus), <SPAN class="highlighted"><STRONG> Jens Winther Bang Petersen</STRONG></SPAN>, for our email correspondence and upcoming meeting with members of our iGEM team.</LI><LI>MATLAB user <SPAN class="highlighted"><STRONG><A href="https://se.mathworks.com/matlabcentral/profile/authors/1417891-nezar" target="_blank" style="padding:0;"><I>Nezar</I></A></STRONG></SPAN>, for an easy implementation of the gillespie algorithm into MATLAB.</LI><LI><SPAN class="highlighted"><STRONG>Flemming Christiansen</STRONG></SPAN>, for his guidance and expertise in choices regarding materials and prototype.</LI></UL><P class="P-Larger"><SPAN class="highlighted"><B>Sponsors</B></SPAN></P><P>Thanks to:</P><UL class="list mini-list"><LI><SPAN class="highlighted"><STRONG>The Faculty of Science at University of Southern Denmark</STRONG></SPAN>, for providing us with the fundamental funds required for our participation in the iGEM competition, and for providing us with laboratory benches and essential equipment.</LI><LI><SPAN class="highlighted"><STRONG>The Faculty of Health Sciences at University of Southern Denmark</STRONG></SPAN>, for their much needed funding of our project.</LI><LI><SPAN class="highlighted"><STRONG>Integrated DNA Technologies</STRONG></SPAN>, for providing us with 20 kilobases of gBlock gene fragments.</LI><LI><SPAN class="highlighted"><STRONG>SnapGene</STRONG></SPAN>, for providing our team with memberships to their software during the duration of the competition.</LI><LI><SPAN class="highlighted"><STRONG>PentaBase</STRONG></SPAN>, for sponsoring us with 10,000 DKK worth of oligos and a further 10% discount.</LI><LI><SPAN class="highlighted"><STRONG>New England Biolabs</STRONG></SPAN>, for providing our team with a BioBrick® Assembly Kit, a Q5® High-Fidelity 2X Master Mix and a Quick-Load® Purple 2-Log DNA Ladder.</LI><LI><SPAN class="highlighted"><STRONG>CO<SUB>2</SUB> Neutral Website</STRONG></SPAN>, for attributing to green energy in our name, and thereby eliminating the carbon footprint our wiki makes.</LI><LI><SPAN class="highlighted"><STRONG>Piktochart</STRONG></SPAN>, for extending their student offer to our mail, and for providing us with easy access to great graphics.</LI></UL><P class="P-Larger"><SPAN class="highlighted"><B>Litterature</B></SPAN></P><DIV class="modal fade" id="articles" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Articles</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>


Administration USDoEEI. Method for Calculating Carbon Sequestration by Trees in Urban and Suburban Settings. 1988.</P><P>
Alper T, Gillies NE. The relationship between growth and survival after irradiation of Escherichia coli strain B and two resistant mutants. Journal of general microbiology. 1960;22:113-28.</P><P>
Anna Prusak GR, Jacek Strojny. IS GMO “SUSTAINABLE”? A REVIEW OF THE ENVIRONMENTAL RISKS OF GM PLANTS IN COMPARISON WITH CONVENTIONAL AND ORGANIC CROPS. MODERN MANAGEMENT REVIEW. 2004.</P><P>
Arai TM, S. Cho, H. Y. Yukawa, H. Inui, M. Wong, S. L. Doi, R. Synthesis of Clostridium cellulovorans minicellulosomes by intercellular complementation. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(5):1456-60.</P><P>
Arras J. Theory and Bioethics. Stanford Encyclopedia of Philosophy. 2010.
Berg IA. Ecological aspects of the distribution of different autotrophic CO2 fixation pathways. Applied and Environmental Microbiology. 2011;77(6):1925-36.</P><P>
Bonacci W, Teng PK, Afonso B, Niederholtmeyer H, Grob P, Silver PA, et al. Modularity of a carbon-fixing protein organelle. Proceedings of the National Academy of Sciences of the United States of America. 2012;109(2):478-83.</P><P>
Brink D. Mill's Moral and Political Philosophy. Stanford Encyclopedia of Philosophy. 2007.</P><P>
Bussell AN, Kehoe DM. Control of a four-color sensing photoreceptor by a two-color sensing photoreceptor reveals complex light regulation in cyanobacteria. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(31):12834-9.</P><P>
Cai F, Dou Z, Bernstein SL, Leverenz R, Williams EB, Heinhorst S, et al. Advances in Understanding Carboxysome Assembly in Prochlorococcus and Synechococcus Implicate CsoS2 as a Critical Component. Life (Basel, Switzerland). 2015;5(2):1141-71.</P><P>
Cai F, Heinhorst S, Shively JM, Cannon GC. Transcript analysis of the Halothiobacillus neapolitanus cso operon. Archives of microbiology. 2008;189(2):141-50.</P><P>
Cataudella I. TA, Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012.</P><P>
Cholby M. The murderer at The Door: what Kant should have said. Philosophy and Phenomenological Research vol 71. 2009.</P><P>
Christensen SK, Mikkelsen M, Pedersen K, Gerdes K. RelE, a global inhibitor of translation, is activated during nutritional stress. Proceedings of the National Academy of Sciences of the United States of America. 2001;98(25):14328-33.</P><P>
Coppi MV, Leang C, Sandler SJ, Lovley DR. Development of a Genetic System for Geobacter sulfurreducens. Applied and Environmental Microbiology. 2001;67(7):3180-7.</P><P>
Datsenko KA, Wanner BL. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences of the United States of America. 2000;97(12):6640-5.</P><P>
David M. Lapola MDO, Carlos A. Nobre. Exploring the range of climate biome projections for tropical South America: The role of CO2 fertilization and seasonality. Global Biogeochemical Cycles. 2009.</P><P>
Donald R. Ort XZ朱, Anastasios Melis. Optimizing Antenna Size to Maximize Photosynthetic Efficiency. American Society of Plant Biologists. 2011.</P><P>
Dryden J. Autonomy. Encyclopedia of Philosophy.</P><P>
DSB. Miljø-Rapport 2016. 2016.</P><P>
Ducat DC, Silver PA. Improving Carbon Fixation Pathways. Current opinion in chemical biology. 2012;16(3-4):337-44.</P><P>
Dunford RP, Catley MA, Raines CA, Lloyd JC, Dyer TA. Purification of active chloroplast sedoheptulose-1,7-bisphosphatase expressed in Escherichia coli. Protein expression and purification. 1998;14(1):139-45.</P><P>
E. Toby Kiers TMP, Anthony R. Ives, John F. Bruno, Judith L. Bronstein. Mutualisms in a changing world: an evolutionary perspective. Ecology Letters. 2010.</P><P>
Efimov VA, Reverdatto SV, Beilinson BA, Fradkov AF, Chakhmakhcheva OG. [Expression of the gene coding for the D1-protein of barley photosystem II in Escherichia coli]. Bioorganicheskaia khimiia. 1994;20(5):524-35.</P><P>
Fewson CA. Biodegradation of xenobiotic and other persistent compounds: the causes of recalcitrance. Cell. 1988.</P><P>
Florea M, Hagemann H, Santosa G, Abbott J, Micklem CN, Spencer-Milnes X, et al. Engineering control of bacterial cellulose production using a genetic toolkit and a new cellulose-producing strain. Proceedings of the National Academy of Sciences of the United States of America. 2016;113(24):E3431-40.</P><P>
Gao D, Luan Y, Wang Q, Liang Q, Qi Q. Construction of cellulose-utilizing Escherichia coli based on a secretable cellulase. Microbial Cell Factories. 2015;14:159.</P><P>
Gentschev I, Dietrich G, Goebel W. The E. coli alpha-hemolysin secretion system and its use in vaccine development. Trends in microbiology. 2002;10(1):39-45.</P><P>
Gotfredsen M, Gerdes K. The Escherichia coli relBE genes belong to a new toxin-antitoxin gene family. Molecular microbiology. 1998;29(4):1065-76.</P><P>
Gotfredsen MaG, K. The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular microbiology. 1998.</P><P>
Groth AC, Calos MP. Phage integrases: biology and applications. Journal of molecular biology. 2004;335(3):667-78.</P><P>
Gu C CJn, Hornberger G, Carrico A. The effects of household management practices on the global warming potential of urban lawns. J Environ Manage. 2015.</P><P>
Gualdi L, Tagliabue L, Bertagnoli S, Ierano T, De Castro C, Landini P. Cellulose modulates biofilm formation by counteracting curli-mediated colonization of solid surfaces in Escherichia coli. Microbiology (Reading, England). 2008;154(Pt 7):2017-24.</P><P>
Guang-Yao Li YZ, Masayori Inouye, Mitsuhiko Ikura. Structural Mechanism of Transcriptional Autorepression of the Escherichia coli RelB/RelE Antitoxin/Toxin Module. In Journal of Molecular Biology. 2008.</P><P>
Heather JM, Chain B. The sequence of sequencers: The history of sequencing DNA. Genomics. 2016;107(1):1-8.</P><P>
Hugler M, Wirsen CO, Fuchs G, Taylor CD, Sievert SM. Evidence for autotrophic CO2 fixation via the reductive tricarboxylic acid cycle by members of the epsilon subdivision of proteobacteria. J Bacteriol. 2005;187(9):3020-7.</P><P>
Iancu CV, Ding HJ, Morris DM, Dias DP, Gonzales AD, Martino A, et al. The structure of isolated Synechococcus strain WH8102 carboxysomes as revealed by electron cryotomography. Journal of molecular biology. 2007;372(3):764-73.</P><P>
J.Folch MJC, P.Chesné, J.L.Alabart, V.Domínguez, Y.Cognié, A.Roche, A.Fernández-Árias, J.I.Martí, P.Sánchez, E.Echegoyen, J.F.Beckers, A. SánchezBonastre, X.Vignon First birth of an animal from an extinct subspecies (Capra pyrenaica pyrenaica) by cloning. Theriogenology. 2009.</P><P>
JP S. The 1906 Food and Drugs Act and Its Enforcement. FDA History - Part I US Food and Drug Administration. 2013.</P><P>
Jung SK, Parisutham V, Jeong SH, Lee SK. Heterologous Expression of Plant Cell Wall Degrading Enzymes for Effective Production of Cellulosic Biofuels. Journal of Biomedicine and Biotechnology. 2012;2012.</P><P>
Kasana RC, Salwan R, Dhar H, Dutt S, Gulati A. A rapid and easy method for the detection of microbial cellulases on agar plates using gram's iodine. Current microbiology. 2008;57(5):503-7.</P><P>
Konnikova M. The Psychology of Distrusting G.M.O.s. THE NEW YORKER. 2013.</P><P>
Kuwabara T, Takeuchi M, Honda S, Nakajima N, Watanabe A, Kondo N. Expression in Escherichia coli of the extrinsic 18-kDa protein of photosystem II of spinach. Plant &amp; cell physiology. 1995;36(3):435-9.</P><P>
Lam TL, Wong RS, Wong WK. Enhancement of extracellular production of a Cellulomonas fimi exoglucanase in Escherichia coli by the reduction of promoter strength. Enzyme and microbial technology. 1997;20(7):482-8.</P><P>
Lamparter T, Esteban B, Hughes J. Phytochrome Cph1 from the cyanobacterium Synechocystis PCC6803. Purification, assembly, and quaternary structure. European journal of biochemistry. 2001;268(17):4720-30.</P><P>
Lee K. Opportunities for green marketing: young consumers. Emerald Group Publishing Limited. 2008.</P><P>
Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</P><P>
Lian Pin Koh RRD, Navjot S. Sodhi, Robert K. Colwell, Heather C. Proctor, Vincent S. Smith. Species Coextinctions and the Biodiversity Crisis. Science (New York, NY). 2004.</P><P>
Lidia S. Watrud EHL, Anne Fairbrother, Connie Burdick, Jay R. Reichman, Mike Bollman, Marjorie Storm, George King, and Peter K. Van de Water. Evidence for landscape-level, pollen-mediated gene flow from genetically modified creeping bentgrass with CP4 EPSPS as a marker. PNAS. 2004.</P><P>
Linster) GGPM. Material Resources, Productivity and the Environment. http://wwwoecdorg/greengrowth/MATERIAL%20RESOURCES,%20PRODUCTIVITY%20AND%20THE%20ENVIRONMENT_key%20findingspdf. 2013.</P><P>
Lo ABY-S. Environmental Ethics. Stanford Encyclopedia of Philosophy. 2002.</P><P>
Logan BE, Hamelers B, Rozendal R, Schroder U, Keller J, Freguia S, et al. Microbial fuel cells: methodology and technology. Environmental science &amp; technology. 2006;40(17):5181-92.</P><P>
Lynd LR, Weimer PJ, van Zyl WH, Pretorius IS. Microbial Cellulose Utilization: Fundamentals and Biotechnology. Microbiology and Molecular Biology Reviews. 2002;66(3):506-77.</P><P>
Mahadevan R, Bond DR, Butler JE, Esteve-Nuñez A, Coppi MV, Palsson BO, et al. Characterization of Metabolism in the Fe(III)-Reducing Organism Geobacter sulfurreducens by Constraint-Based Modeling. Applied and Environmental Microbiology. 2006;72(2):1558-68.</P><P>
Malvankar NS, Vargas M, Nevin KP, Franks AE, Leang C, Kim BC, et al. Tunable metallic-like conductivity in microbial nanowire networks. Nature nanotechnology. 2011;6(9):573-9.</P><P>
Mangan NM, Brenner MP. Systems analysis of the CO(2) concentrating mechanism in cyanobacteria. eLife. 2014;3.</P><P>
Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(13):5590-5.</P><P>
Mattozzi M, Ziesack M, Voges MJ, Silver PA, Way JC. Expression of the sub-pathways of the Chloroflexus aurantiacus 3-hydroxypropionate carbon fixation bicycle in E. coli: Toward horizontal transfer of autotrophic growth. Metabolic engineering. 2013;16:130-9.</P><P>
Moore LAM. Deontological Ethics. Stanford Encyclopedia of Philosophy. 2007.</P><P>
Morgado L, Fernandes AP, Dantas JM, Silva MA, Salgueiro CA. On the road to improve the bioremediation and electricity-harvesting skills of Geobacter sulfurreducens: functional and structural characterization of multihaem cytochromes. Biochemical Society transactions. 2012;40(6):1295-301.</P><P>
O. Hoegh-Guldberg PJM, A. J. Hooten, R. S. Steneck, P. Greenfield, E. Gomez, C. D. Harvell, P. F. Sale, A. J. Edwards, K. Caldeira, N. Knowlton, C. M. Eakin, R. Iglesias-Prieto, N. Muthiga, R. H. Bradbury, A. Dubi, M. E. Hatziolos. Coral Reefs Under Rapid Climate Change and Ocean Acidification. Science (New York, NY). 2007.</P><P>
Overgaard M, Borch, J., Jørgensen, M. G. and Gerdes, K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008.</P><P>
Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</P><P>
Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</P><P>
Overgaard M. BJ, Gerdes K. . RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J Mol Biol 2009.</P><P>
Parikh MR, Greene DN, Woods KK, Matsumura I. Directed evolution of RuBisCO hypermorphs through genetic selection in engineered E.coli. Protein engineering, design &amp; selection : PEDS. 2006;19(3):113-9.</P><P>
Pedersen K1 ZA, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003.</P><P>
Pedersen K, Zavialov AV, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112(1):131-40.</P><P>
Rae BD, Long BM, Badger MR, Price GD. Functions, Compositions, and Evolution of the Two Types of Carboxysomes: Polyhedral Microcompartments That Facilitate CO(2) Fixation in Cyanobacteria and Some Proteobacteria. Microbiology and Molecular Biology Reviews : MMBR. 2013;77(3):357-79.</P><P>
Ragsdale SW, Pierce E. Acetogenesis and the Wood-Ljungdahl Pathway of CO(2) Fixation. Biochimica et biophysica acta. 2008;1784(12):1873-98.</P><P>
Rangel G. From Corgis to Corn: A Brief Look at the Long History of GMO Technology. Genetically Modified Organisms and Our Food. August 2015.</P><P>
Raser JM, O'Shea EK. Noise in gene expression: origins, consequences, and control. Science (New York, NY). 2005;309(5743):2010-3.</P><P>
Richter LV, Sandler SJ, Weis RM. Two Isoforms of Geobacter sulfurreducens PilA Have Distinct Roles in Pilus Biogenesis, Cytochrome Localization, Extracellular Electron Transfer, and Biofilm Formation. Journal of Bacteriology. 2012;194(10):2551-63.</P><P>
Sekar R, Shin HD, Chen R. Engineering Escherichia coli Cells for Cellobiose Assimilation through a Phosphorolytic Mechanism. Applied and Environmental Microbiology. 2012;78(5):1611-4.</P><P>
Sinnott-Armstrong W. Consequentialism. Stanford Encyclopedia of Philosophy. 2003.</P><P>
Sneppen K ZG. Physics in Molecular Biology. . Cambridge, UK: Cambridge University Press. 2005.</P><P>
Su L, Chen S, Yi L, Woodard RW, Chen J, Wu J. Extracellular overexpression of recombinant Thermobifida fusca cutinase by alpha-hemolysin secretion system in E. coli BL21(DE3). Microbial Cell Factories. 2012;11:8.</P><P>
Tabor JJ. Multichromatic control of gene expression in Escherichia coli. 2011;405(2):315-24.</P><P>
Tamoi M, Nagaoka M, Shigeoka S. Immunological properties of sedoheptulose-1,7-bisphosphatase from Chlamydomonas sp. W80. Bioscience, biotechnology, and biochemistry. 2005;69(4):848-51.</P><P>
Tan Y, Adhikari RY, Malvankar NS, Ward JE, Woodard TL, Nevin KP, et al. Expressing the Geobacter metallireducens PilA in Geobacter sulfurreducens Yields Pili with Exceptional Conductivity. mBio. 2017;8(1).</P><P>
Tashiro Y, Kawata K, Taniuchi A, Kakinuma K, May T, Okabe S. RelE-Mediated Dormancy Is Enhanced at High Cell Density in Escherichia coli. J Bacteriol. 2012;194(5):1169-76.</P><P>
Thompson JF, de Vargas LM, Skinner SE, Landy A. Protein-protein interactions in a higher-order structure direct lambda site-specific recombination. Journal of molecular biology. 1987;195(3):481-93.</P><P>
Union ITWC. CURRENT KNOWLEDGE OF THE IMPACTS OF GENETICALLY MODIFIED ORGANISMS ON BIODIVERSITY AND HUMAN HEALTH 2007.</P><P>
Vierstra RD, Davis SJ. Bacteriophytochromes: new tools for understanding phytochrome signal transduction. Seminars in cell &amp; developmental biology. 2000;11(6):511-21.</P><P>
Vira C, Prakash G, Rathod JP, Lali AM. Cloning, expression, and purification of Chlamydomonas reinhardtii CC-503 sedoheptulose 1,7-bisphosphatase in Escherichia coli. Preparative biochemistry &amp; biotechnology. 2016;46(8):810-4.</P><P>
W. Glynn Mangold DJF. Social media: The new hybrid element of the promotion mix. Business Horizons. 2009.</P><P>
Walter JM, Greenfield D, Bustamante C, Liphardt J. Light-powering Escherichia coli with proteorhodopsin. Proceedings of the National Academy of Sciences of the United States of America. 2007;104(7):2408-12.</P><P>
Yaling Qian RFF, John M. Kimble. Assessing Soil Carbon Sequestration in Turfgrass Systems Using Long-Term Soil Testing Data. 2002.




</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="websites" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Websites</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>Antitoxin RelB [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: http://www.uniprot.org/uniprot/P0C079.]

Calculate Flight Emissions [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: https://www.atmosfair.de/en/kompensieren/flug?locale=en&amp;departure=CPH&amp;arrival=BOS.]

Convention on Biological Diversity [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: https://www.cbd.int/.]

iGEM-dresden. The GoGreenGuide [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: https://github.com/igem-dresden/GoGreenGuide/releases/tag/v1.0.]

LEXAN™ 103R resin Datasheet Polycarbonate [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: https://plastics.ulprospector.com/datasheet/e17532/lexan-103r-resin.]

Odense Fødevare Fællesskab [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: http://www.odenseff.dk/.

Volswagen Erhversbiler [Accessed: 1<SUP>st</SUP> of Nov. 2017] [Available from: http://viewer.zmags.com/publication/4b44c1a8#/4b44c1a8/4.]
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="books" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Books</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>Adler F. Miljøetik. In: Christensen A-MS, editor. Filosofisk etik: Aarhus Universitetsforlag 2011; 2011. p. s. 197-214.

Alexandre Chagnes JS. Global Lithium Resources and Sustainability Issues.  Lithium Process Chemistry: Elsevier; 2015. p. pp.1-40.

Joanne Willey LS, Christopher J. Woolverton. Prescott’s Microbiology. 9th edition ed2014.

Kappel K. Bioetik. In: Christensen A-MS, editor. Filosofisk etik: Aarhus Universitetsforlag 2011.; 2011. p. 183-96.

Khanal YLaSK. Microbial Fuel Cells, Capter 19. In: Khanal. EbYLaSK, editor. Bioenergy: Principles and Applications. First Edition ed: Published 2017 by John Wiley &amp; Sons, Inc.; 2016.

O’Keefe TJ. The Alliance herald. : (Alliance, Box Butte County, Neb.) 1902-1922. T.J. O'Keefe; 1902.

Sinclair U. The Jungle. New York: Grosset &amp; Dunlap,; 1906 1906.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="topShadow" id="final-words"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-credits"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Final Words</SPAN></H2><P><SPAN class="highlighted">Thank you for your time! We hope you enjoyed our wiki and getting to know our project</SPAN>.

Now you can sit back, relax, and be proud of your hard work. While you do so, <SPAN class="highlighted">feel free to enjoy some of the less serious pictures and snippets from our amazing iGEM adventure</SPAN>.  
            </P></DIV></DIV></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>