<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-Denmark_testvyff skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-Denmark/testvyff</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="navbarPosition"><UL class="nav nav-tabs"><LI id="navbarIntroduction" class="active"><A data-toggle="tab" href="#menu1">Introduction</A></LI><LI id="navbarProject"><A data-toggle="tab" href="#menu2">Project &amp; Results</A></LI><LI id="navbarProcedures"><A data-toggle="tab" href="#menu3">Parts &amp; Procedures</A></LI><LI id="navbarPractices"><A data-toggle="tab" href="#menu4">Practices</A></LI><LI id="navbarProspects"><A data-toggle="tab" href="#menu5">Prospects</A></LI><LI id="navbarCredits"><A data-toggle="tab" href="#menu6">Credits</A></LI></UL><DIV class="tab-content"><DIV id="menu1" class="tab-pane fade in active"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#home"><H5>Home</H5></A><A class="nav-item nav-link" href="#introduction"><H5>Opening</H5></A><A class="nav-item nav-link" href="#achievements"><H5>Achievements</H5></A><A class="nav-item nav-link" href="#world-situation"><H5>World Situation</H5></A><A class="nav-item nav-link" href="#our-solution"><H5>Our Solution</H5></A></NAV></DIV><DIV id="menu2" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#project-and-results"><H5>Opening</H5></A><A class="nav-item nav-link" href="#project-design"><H5>Project Design</H5></A><A class="nav-item nav-link" href="#demonstration-and-results"><H5>Demonstration &amp; Results</H5></A></NAV></DIV><DIV id="menu3" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#parts-and-procedures"><H5>Opening</H5></A><A class="nav-item nav-link" href="#parts"><H5>Parts</H5></A><A class="nav-item nav-link" href="#notebook"><H5>Notebook</H5></A><A class="nav-item nav-link" href="#sops-and-protocols"><H5>SOPs &amp; Protocols</H5></A><A class="nav-item nav-link" href="#safety"><H5>Safety</H5></A></NAV></DIV><DIV id="menu4" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#practices"><H5>Opening</H5></A><A class="nav-item nav-link" href="#bioethics"><H5>Bioethics</H5></A><A class="nav-item nav-link" href="#integrated-practices"><H5>Integrated Practices</H5></A><A class="nav-item nav-link" href="#education-and-public-engagement"><H5>Education &amp; Public Engagement</H5></A></NAV></DIV><DIV id="menu5" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#prospects"><H5>Opening</H5></A><A class="nav-item nav-link" href="#perspectives"><H5>Perspectives</H5></A><A class="nav-item nav-link" href="#future-igem-teams"><H5>To Future iGEM Teams</H5></A></NAV></DIV><DIV id="menu6" class="tab-pane fade"><NAV class="navbar-nav"><A class="nav-item nav-link" href="#credits"><H5>Opening</H5></A><A class="nav-item nav-link" href="#team"><H5>Team</H5></A><A class="nav-item nav-link" href="#collaborations"><H5>Collaborations</H5></A><A class="nav-item nav-link" href="#attributions"><H5>Attributions</H5></A><A class="nav-item nav-link" href="#final-words"><H5>Final Words</H5></A></NAV></DIV></DIV></DIV><DIV id="highlightToggle"><DIV id="editorsHighlights"><P><I>editors <SPAN class="highlighted">highlights</SPAN></I></P></DIV></DIV><DIV class="topShadow" id="modelling"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-project"><DIV class="col-xs-12"><H2><SPAN class="highlighted">Modelling</SPAN></H2></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P><TABLE><TBODY><TR><TH>Constant</TH><TH>Identifier</TH><TH>Units</TH><TH>Value</TH></TR><TR><TD>mrna transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>α<SUB>0</SUB></TD><TD>1/min</TD><TD>154.665</TD></TR><TR><TD>mrna half life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>m</SUB></TD><TD>min</TD><TD>7.2</TD></TR><TR><TD>RelB half life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>B</SUB></TD><TD>min</TD><TD>4.3</TD></TR><TR><TD>RelE half life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>E</SUB></TD><TD>min</TD><TD>43      (growing)  2000 (dormant)</TD></TR><TR><TD>Bound RelB half life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>c</SUB></TD><TD>min</TD><TD>17</TD></TR><TR><TD>RelB transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>B</SUB></TD><TD>1/min</TD><TD>15</TD></TR><TR><TD>RelE transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>E</SUB></TD><TD>1/min</TD><TD>0.3</TD></TR><TR><TD>Binding rate<SPAN class="reference"><SPAN class="referencetext">Sneppen K, Zocchi G. Physics in Molecular Biology. Cambridge, UK: Cambridge University Press; 2005.</SPAN></SPAN></TD><TD>k <SUB>b</SUB></TD><TD>1/min</TD><TD>3.8</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD>K<SUB>D</SUB> (B<SUB>2</SUB>E)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD> K<SUB>D</SUB>(B<SUB>2</SUB>E<SUB>2</SUB>)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate O.B<SUB>f</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>K<SUB>D1</SUB></TD><TD>molecules</TD><TD>10</TD></TR><TR><TD>Dissociation rate O.B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD> K<SUB>D2</SUB></TD><TD>molecules</TD><TD>0.04</TD></TR><TR><TD>Dissociation rate O.(B<SUB>2</SUB>E)<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  K<SUB>D3</SUB></TD><TD>molecules</TD><TD>30</TD></TR><TR><TD>Cleavage rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.sciencedirect.com/science/article/pii/S0092867402012485?">Pedersen K, et al. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112:131–140. doi: 10.1016/S0092-8674(02)01248-5</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  k<SUB>c</SUB></TD><TD>1/min  1/molecules</TD><TD>2.0</TD></TR></TBODY></TABLE></P><P> In order to find the best way to implement the toxin-antitoxin system, we resort to modelling. We use the <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#gillespie-algorithm">gillespie algorithm</SPAN> to model the interactions of the <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#toxin-antitoxin-system">toxin antitoxin system</SPAN>.<SPAN class="btn-link btn-lg" data-target="#https://static.igem.org/mediawiki/2017/1/1e/Matlab-scripts_SDU.zip">toxin antitoxin system</SPAN>.
We find that when we implement enhanced relE production as a tool to make the bacteria dormant, an additional implementation of relB to ensure don’t stay dormant when in light again. 

The model found that the system is sensitive to the relE:relB ratio as well as the total production, and that an implementation with production rates in the vicinity of 50 and 35  molecules pr. min for relB and relE respectively yields close to the wished for effect: THe bacteria goes dormant in an hour and wakes up quickly.
</P><DIV class="modal fade" id="gillespie-algorithm" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Gillespie Algorithm</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P>
The Gillespie algorithm is a way to get calculate the evolution of stochastic functions; in this case cell concentrations. To use the algorithm, two things are needed:</P><OL class="list"><LI>reaction rates of the system (a1, a2, a3) at a given configuration.</LI><LI>a random number generator.</LI></OL><P>For each time step two things are calculated, using the random number generator: The time before next reaction and which reaction occurs. 
The time before next step is given by </P><DIV style="text-align:center;"><P> Hello Emil</P></DIV><P>
Where S is the sum of the reaction rates and r1 is a random number between 0 and 1. This gives the time as if the system was one reaction with reaction rate S, using the random number to give a statistic distribution. 
The reaction is chosen proportionally to each individual reaction rate, using another random number. This way, reaction with high rates compared to other reactions will happen the most. 

The reaction is carried out, the time is moved the calculated amount, and new reaction rates can now be calculated, to repeat the whole process. This continues until the time reaches the wanted limit or a specific number of reactions have occurred. It is necessary to have a limit on the number of reactions as it is possible for the time steps to grow smaller and smaller, in which case the calculation time quickly become either immense or impossible. 
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="toxin-antitoxin-system" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Toxin/Antitoxin System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P>
We are modelling the RelE RelB toxin-antitoxin system. RelE is a toxin restricting growth, by inducing a dormant state. This is inhibited by RelB, which forms complexes with RelE. Two different complexes are made: RelB<SUB>2</SUB>RelE and RelB<SUB>2</SUB>RelE<SUB>2</SUB>, containing 1 and 2 RelE molecules respectively <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://doi.org/10.1016/j.jmb.2008.04.039.">Guang-Yao Li, Yonglong Zhang, Masayori Inouye, Mitsuhiko Ikura, Structural Mechanism of Transcriptional Autorepression of the Escherichia coli RelB/RelE Antitoxin/Toxin Module, In Journal of Molecular Biology, Volume 380, Issue 1, 2008, Pages 107-119, ISSN 0022-2836</A></SPAN></SPAN>. 

Both RelE and RelB are expressed from the same promoter, RelBE. When only small amounts of RelE is present, RelB and RelB<SUB>2</SUB>RelE represses transcription of RelBE, by binding to the operator. 

At higher concentrations of RelE, the toxin mitigates this repression, by reacting with bound complexes
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>.
</P><P>
For natural purposes the half life of RelB decreases significantly under starvation due to lon-protease, with shifts the equilibrium of RelB and RelE to a high state of RelE. The interactions with the promoter, keeps the amount of free RelE at a very low value outside starvation and stabilises the system
 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>.
In our simulation the shift in equilibrium is made by introducing additional translation of RelE. 

We used two model in two ways. First we saw how a given configuration of relB and RelE production increased the relE concentration and if it could cause dormancy within 2 hours. Second we investigated for how long each configuration</P><P class="P-Larger"><B>Rates and reactions</B></P><P>In the units off all reaction rates we use the approximation that in an E.coli. with a size of 1-2 μm, 1 molecule in the cell = 1nm. Thus we convert all units to be measured in molecules, as this fits the premises of the gillespie algorithm. 

To simplify the model all rates for relB are for relB2, that is we use the high affinity of relE and relB, to look at all relB as alreadry in dimers
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.uniprot.org/uniprot/P0C079">UniProtKB - P0C079 (RELB_ECOLI)</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>.

RelB has a relatively low half-life at about 3-5 minutes <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>, while RelE is stable and it’s half life is an effect from dilution  due to growing bacteria (we use 43 min)
<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>
. During dormancy, growth is restricted and we increase RelE half life to 2000 min (around a day) as the dilution.

The transcription rates of RelE and RelB is based on the concentration of RelE and RelB under stable conditions. Here RelB is 10 times more prevalent than RelE <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN>
so to make up for the higher half life of RelE, RelB has been given a much higher transcription rate than RelE (100 times)

The complexes are close to stable and given the same half life as RelE. However, to get free RelE to work RelB in complexes need to decay as well.

<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>


 The rate is set to a fourth of free RelB.</P><P>For the promoter bindings we let the operator be inhibited by binding with either RelB or 1-2 RelB<SUB>2</SUB>RelE, given that the operator has to binding sites <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>. We consider the cell to have four chromosomes with one promoter on each (less chromosomes would let the system work, but with more noise).

The initial values in the model are given by:</P><P><I>insert table with numbers and constants</I></P><P>Notice that all values are integers as the gillespie algorithm works with discrete numbers of molecules. The values were chosen based on a stable equilibrium found by letting the model run a simulation of the inherent system over 450 minutes, with different starting values. 

The implemented total production rates shown in the model might seem too high as they range from 1-350 molecules pr. min, while the rates in the inherit system is effectively around 80-100 for relB and 2-5 for relE. The possibility of placing the system on high-copy plasmids, however, makes the high total production values reasonable, as the individual relE promoter only need a production of 0.01-2 (assuming a few hundred copies).

Considering the inherent toxin-antoxin system activating under starvation, we see that the the magnitude of relE copies around 50-80 molecules pr. cell. This makes it reasonable to believe that the cell enters dormancy when a few tens of free relE copies.</P><P class="P-Larger"><B>Running the model</B></P><P>The model is using the gillespie algorithm to give a stochastic view of the system and is run on matlab. The code uses an implementation by matlab user Nezar (https://se.mathworks.com/matlabcentral/fileexchange/34707-gillespie-stochastic-simulation-algorithm). 

For the dormancy runs we input deterministic initial values, and then let the system run 30 min without activating the inserted toxin promoter. This results in a stochastic distribution of initial values mimicking variations between cells. Analysis show that 30 minutes is enough for the model to find a stable distribution, which is realistic considering the growth cycle of an e.coli.  

For wakeup runs, we use the data generated at the end of a sleep run as initial value and remove the production of relE. We consider a cell to be woken up when the concentration of free relE decreases below 15 copies. This is perhaps a low value, but tests show marginal difference between 15 and 45 copies, where the low bound is chosen to decrease uncertainty of the cells state. 

All runs simulate 1000 cells, which should be sufficient to get stable averages. The model assumes well-mixed conditions in each cell, but considers each cell independently. 
The model has no cut off for maximum values of relE, as we don’t know the exact relation between relE concentration and hibernation state, yet as a functional cutoff is found through wake-up times, it is not necessary.
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>