<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-Denmark_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-Denmark/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid" style="margin-bottom:-50px;"><DIV class="topShadow" id="project-design"><DIV class="row anchorMargin presentationBackground"><DIV class="col-lg-8 col-md-10 margin-bottom-200"><DIV class="row border-left-project"><DIV class="col-xs-12"><DIV class="row margin-bottom-75 padding-top-125" id="project-design-dormancy-system" style="margin-top:-125px;"><DIV class="col-xs-12"><DIV class=""><DIV class="project-design-headline"><H2>Dormancy System</H2></DIV></DIV></DIV><DIV style="text-align:center;"><P><SPAN class="reference-2">Project Overview</SPAN></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-xs-12"><P class="P-Larger"><B>Introduction</B></P><P>Cyanobacteria contain signal transduction systems, thereby making them capable of <SPAN class="highlighted">sensing and responding to light</SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3732953/">Bussell AN, Kehoe DM. Control of a four-color sensing photoreceptor by a two-color sensing photoreceptor reveals complex light regulation in cyanobacteria. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(31):12834-9.</A></SPAN></SPAN>. This ability gives the organisms the opportunity to <SPAN class="highlighted">adapt and optimize their metabolism to a circadian rhythm</SPAN>. Photoreceptors in the plasma membrane, of which phytochromes are especially abundant and well described, are responsible for this property <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11145881">Vierstra RD, Davis SJ. Bacteriophytochromes: new tools for understanding phytochrome signal transduction. Seminars in cell &amp; developmental biology. 2000;11(6):511-21.</A></SPAN></SPAN>. In 2004, the <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM team</A> made a light response system consisting of a photoreceptor combined with an intracellular indigenous regulator system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16306980">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. EnvZ and OmpR make up the two-component system naturally found in <I>E. coli</I>. The photoreceptor known as Cph1 was isolated from the cyanobacteria <I>Synechocytis</I> PCC6803. Cph1 has functional combination sites, which combined with the kinase EnvZ form a two-domain receptor, known as Cph8. Activation of Cph8 is mediated by the chromophore phycocyanobilin, PCB, that is <SPAN class="highlighted">sensitive to red light</SPAN> with maximal absorbance at 662 nm <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11532008">Lamparter T, Esteban B, Hughes J. Phytochrome Cph1 from the cyanobacterium Synechocystis PCC6803. Purification, assembly, and quaternary structure. European journal of biochemistry. 2001;268(17):4720-30.</A></SPAN></SPAN>.
<SPAN class="highlighted">When not exposed to light</SPAN>, PCB activates the phytochrome Cph1, thus promoting kinase activity through the EnvZ kinase. When the transcription factor OmpR is phosphorylated by EnvZ, <SPAN class="highlighted">expression of genes regulated by the OmpR-regulated promoter is initiated. Excitation of PCB by red light</SPAN> results in a situation where the transcription factor OmpR is not regulated. The absence of phosphorylated OmpR leads to no activation of the OmpR-regulated promoter, thereby <SPAN class="highlighted">preventing gene expression</SPAN>.
</P><DIV class="figure-text"><P><B>Figure 1.</B> Left: Red light activates PCB, which in turn inactivates the photoreceptor complex Cph8, preventing gene expression from the OmpR-regulated promoter. Right: In absence of light, PCB is inactive, which enables the Cph8 to phosphorylate the transcription factor OmpR. This promotes gene expression from the OmpR-regulated promoter.</P></DIV><P>
The <SPAN class="highlighted">photocontrol device can be used to regulate a toxin-antitoxin system</SPAN>, enabling the implementation of a light-dependent dormancy system. A toxin-antitoxin system is composed of two gene products, a cytotoxin and an antitoxin, the latter which neutralises the the toxic effect caused by the toxin.
 In <I>E. coli</I> K-12 the cytotoxin RelE and antitoxin RelB comprise such a system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/9767574">Gotfredsen M, Gerdes K. The Escherichia coli relBE genes belong to a new toxin-antitoxin gene family. Molecular microbiology. 1998;29(4):1065-76.</A></SPAN></SPAN>. Expression of the <SPAN class="highlighted">cytotoxin RelE inhibits translation in the cells</SPAN>, due to its ability to cleave mRNA found in the A-site of the ribosome. <SPAN class="highlighted">RelB neutralises the toxic effect of RelE</SPAN> through interaction between the two proteins. Whether the cell lies dormant in response to expression of RelE depends on the ratio of antitoxin RelB and RelE present in the cell. Several studies have shown that RelB and RelE form a complex with RelB:RelE stoichiometry of 2:1 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. When the RelB:RelE stoichiometric-ratio is lowered to 1:1, studies show that RelB is not able to protect the cells against the RelE-caused translational inhibition <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. For further information about the light-dependent dormancy system, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#light-sensing-system-theory">read here</SPAN>.
</P><DIV class="modal fade" id="light-sensing-system-theory" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Light-Dependent Dormancy System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P><B>Dormancy Optimises the Efficiency of the Bacterial Solar Battery</B>
Photosynthetic bacteria draw energy from sunlight to drive the fixation of carbon, implying that the bacteria are not able to carry out carbon fixation in absence of light. However, the bacteria constantly metabolise and the carbon fixed during time of exposure to light, will be used as an energy source for the cells. To circumvent this, a photocontrol system created by the <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM 2004 team</A> was used in combination with the RelE-RelB toxin-antitoxin system native to <I>E. coli</I>. In this way, a light-dependent dormancy system was implemented in <I>E. coli</I>.<B>The Photocontrol Device Mediates Light-Dependent Gene Expression</B>
Plants and several photosynthetic microorganisms, such as cyanobacteria, contain signal transduction systems, which makes them capable of reacting to light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3732953/">Bussell AN, Kehoe DM. Control of a four-color sensing photoreceptor by a two-color sensing photoreceptor reveals complex light regulation in cyanobacteria. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(31):12834-9.</A></SPAN></SPAN>. This ability gives the organisms the opportunity to adapt and optimise the regulation of their metabolic rate in response to sunlight. This property is achieved by photoreceptors incorporated in their plasma membrane, of which phytochromes are the most abundant and well described <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11145881">Vierstra RD, Davis SJ. Bacteriophytochromes: new tools for understanding phytochrome signal transduction. Seminars in cell &amp; developmental biology. 2000;11(6):511-21.</A></SPAN></SPAN>.

Several two-component signal transduction systems evolved in <I>E. coli</I> enables it to respond to various external conditions, such as osmotic stress, lack of metabolites and other external stress factors. Nothing indicates that light initiates such a two-component signal transduction pathway in wild type <I>E. coli</I><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.microbiologyresearch.org/docserver/fulltext/micro/22/1/mic-22-1-113.pdf?expires=1507966841&amp;id=id&amp;accname=guest&amp;checksum=574A5913441399B962AA6A4F887C733E">Alper T, Gillies NE. The relationship between growth and survival after irradiation of Escherichia coli strain B and two resistant mutants. Journal of general microbiology. 1960;22:113-28.</A></SPAN></SPAN>. The <A href="https://2004.igem.org/austin.cgi" target="_blank">UT Austin iGEM 2004 team</A> applied the light sensing property of phototrophs to an <I>E. coli</I>. By aligning different phytochromes with the intrinsic kinase EnvZ from <I>E. coli</I> they revealed a way to create a two-component system consisting of a photoreceptor with an intracellular indigenous regulator system found in <I>E. coli</I>. By establishing this system the bacteria acquired the ability to respond to red light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16306980">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. The photoreceptor from phytochrome known as Cph1 was isolated from the cyanobacteria <I>Synechocytis</I> PCC6803. Cph1 has a fusion site, which can be used to combine it with the kinase EnvZ, from the EnvZ-OmpR kinase-regulator system, to form a two-domain receptor known as Cph8. The chromophore phycocyanobilin (PCB) absorbs light in the red region with maximal absorbance at 662 nm <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11532008">Lamparter T, Esteban B, Hughes J. Phytochrome Cph1 from the cyanobacterium Synechocystis PCC6803. Purification, assembly, and quaternary structure. European journal of biochemistry. 2001;268(17):4720-30.</A></SPAN></SPAN>. When heterogeneously expressed in <I>E. coli</I>, it can, in combination with the light receptor Cph8, be used to form a light-sensitive circuit, making <I>E. coli</I> able to respond to red light <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053042/">abor JJ, Levskaya A, Voigt CA. Multichromatic control of gene expression in Escherichia coli. Journal of molecular biology. 2011;405(2):315-324. </A></SPAN></SPAN>. 

In situations where no red light is present, the photoreceptor PCB activates the phytochrome Cph1, thereby promoting kinase activity through the EnvZ kinase, illustrated in figure 1. When the transcription factor OmpR is phosphorylated by EnvZ, expression of genes controlled by the OmpR-regulated promoter is initiated. Excitation of the PCB by red light, results in a situation, where EnvZ will not be able to phosphorylate the transcription factor OmpR. The lack of phosphorylated OmpR leads to no activation of the OmpR-regulated promoter, thereby preventing gene expression by this promoter.<B>RelE and RelB Comprise a Toxin-Antitoxin System in <I>E. coli</I></B>
A toxin-antitoxin system is a system composed of two gene products, of which one specifies a cell toxin and the other an antitoxin, which neutralises the toxic effect caused by the toxin. In <I>E. coli</I> K-12 the cytotoxin RelE and antitoxin RelB comprise such a system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/9767574">Gotfredsen M, Gerdes K. The Escherichia coli relBE genes belong to a new toxin-antitoxin gene family. Molecular microbiology. 1998;29(4):1065-76.</A></SPAN></SPAN>. Expression of the cytotoxin RelE inhibits translation in the cells, due to its ability to cleave mRNA found in the A-site of the ribosome <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12526800">Pedersen K, Zavialov AV, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112(1):131-40.</A></SPAN></SPAN>. RelB neutralise the toxic effect of RelE through interaction between the two proteins. In situations of amino acid starvation, it is appropriate for the bacteria to halt the translation in order to avoid errors owing to absent amino acids. Consequently, one of the exciting factors for the expression of RelE is conditioned by amino acid starvation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12526800">Pedersen K, Zavialov AV, Pavlov MY, Elf J, Gerdes K, Ehrenberg M. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112(1):131-40.</A></SPAN></SPAN>. 

Whether the cell lie dormant in response to expression of RelE depends on the ratio of RelB and RelE present in the cell. Several studies have shown that RelB RelE form a complex with RelB:RelE stoichiometry of 2:1 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>, When the RelB:RelE stoichiometric-ratio is lowered to 1:1, studies show that RelB is not able to protect the cells against the RelE-caused translational inhibition <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard M, Borch J, Jorgensen MG, Gerdes K. Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular microbiology. 2008;69(4):841-57.</A></SPAN></SPAN>. To prevent free RelE circulating and discharging toxic effects in the cells under favorable conditions, studies in vivo have shown that RelB is present in 10x higher concentrations than RelE <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/19747491">Overgaard M, Borch J, Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. Journal of molecular biology. 2009;394(2):183-96.</A></SPAN></SPAN>. The heterologous induction of RelE could cause dissonance in the RelB:RelE ratio leading to serious consequences for the cells <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/11717402">Christensen SK, Mikkelsen M, Pedersen K, Gerdes K. RelE, a global inhibitor of translation, is activated during nutritional stress. Proceedings of the National Academy of Sciences of the United States of America. 2001;98(25):14328-33.</A></SPAN></SPAN>. The bacteria are not killed when RelE is present in abundance, but high expression of the RelE gene makes awakening of the bacterial cells a challenge <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294780/">Tashiro Y, Kawata K, Taniuchi A, Kakinuma K, May T, Okabe S. RelE-Mediated Dormancy Is Enhanced at High Cell Density in Escherichia coli. J Bacteriol. 2012;194(5):1169-76.</A></SPAN></SPAN>. Hence, introducing a toxin to cells in a successful manner constitutes a challenge. 

</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV id="modelling"><P class="P-Larger"><B>Modelling</B></P><P><B>Modelling of the RelE-RelB System is Essential to Avoid Irrevocable Dormancy</B></P><P>Controllable dormancy is a feature that holds the potential to be applied in many different situations. However, inducing dormancy and bringing the bacteria back to a metabolic active state is like balancing on a tightrope, and to establish the basis of future implementations, the properties of this system would have to be investigated further. In an endeavour to provide this basic knowledge, <SPAN class="highlighted">stochastic modelling utilising the <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#gillespie-algorithm"> Gillespie algorithm </SPAN> was performed in an attempt to prognosticate the system</SPAN> and simulate the interactions between the toxin and antitoxin. 
<SPAN class="highlighted">The toxin RelE is inhibited by the antitoxin RelB through complex formation</SPAN>, and both proteins interact with their promoter in a feedback mechanism. 
To consolidate the model, the capacity of the toxin-antitoxin system was assessed in an experiment, as the controllability of the dormancy system was studied through manual  regulation of RelE and RelB expression.
You can read more about the modelling <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#toxin-antitoxin-system">here</SPAN>. </P><DIV class="figure-text"><P><B>Figure 2.</B> Left: The time required for the bacteria to enter dormancy varies with the expression level of RelB. The percentage of dormant bacteria, defined as containing RelE amounts above 40 molecules per cell as a function of time in minutes. Right: Only one of the tested configurations, RelB<SUB>2</SUB>:50-RelE:35, causes the bacteria to regain their activity within the modelled time. The percentage of dormant bacteria, defined as containing RelE amounts above 15 molecules per cell as a function of time in minutes. The data is based on the simulation of 1000 independent bacteria.</P></DIV><P>
The simulated data revealed, that when enhanced RelE production is implemented, in order to induce dormancy in <I>E. coli</I>, the effect come easily. However, <SPAN class="highlighted"> implementation of RelB expression is also found necessary</SPAN> to ensure that the bacteria are able to enter an active state again. 

The model showed that <SPAN class="highlighted">the system is sensitive to the RelE:RelB ratio</SPAN>, as well as the total amount of produced toxin. As seen in Figure 2, implementation with production rates in the vicinity of <SPAN class="highlighted">50 and 35 molecules per minute for RelB and RelE respectively, was found to be suitable for balancing our system</SPAN>; the bacteria lay dormant within the computed time and re-enter an active state within minutes. 

The simulated data made it evident that <SPAN class="highlighted">implementing an optimised dormancy system comprises a challenge</SPAN>, as the individual expression levels of RelE and RelB, as well as their interaction, has a crucial impact on the regulation of dormancy. Thus, controlled gene expression of both RelE and RelB is required to implement a controllable dormancy system in the PowerLeaf. 

If you want to dig deeper into this crucial modelling of the dormancy system, read the full results <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#model-results">here</SPAN>. </P></DIV><DIV class="modal fade" id="gillespie-algorithm" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Gillespie Algorithm</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P>
The Gillespie algorithm is a way to calculate the evolution of stochastic functions; in this case cell concentrations. To use the algorithm, two things are required:</P><OL class="list"><LI>Reaction rates of the system at a given configuration.</LI><LI>A random number generator.</LI></OL><P>
For each time step, two things are calculated using the random number generator: </P><OL class="list"><LI>The time before next reaction.</LI><LI>Which reaction occurs.</LI></OL><P>
The time before next step is given by:
</P><DIV style="text-align:center;"><P>Δt=S<SUP>-1</SUP>log(r<SUB>1</SUB><SUP>-1</SUP>)</P></DIV><P>
Where S is the sum of the reaction rates and r<SUB>1</SUB> is a random number between 0 and 1. This gives the time, as if the system was one reaction with reaction rate S, using the random number to give an exponential distribution. 

The reaction is chosen proportionally to each individual reaction rate using another random number, where reactions with high rates will occur most frequently. 

As each reaction is carried out, the new time is the sum of the previous and added reaction times. 

These calculations are carried out until the time reaches the wanted limit, or a specific number of reactions has occurred. It is necessary to have a limit on the number of reactions, as it elsewise is possible for the calculations to continue indefinitely.

</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="toxin-antitoxin-system" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Stochastic Modelling of the RelE-RelB Dormancy System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>RelE and RelB Regulate Dormancy and Influence Their Own Expression</B></P><P>
Modelling of the effects of different RelE and RelB expression levels were performed as an important aspect in the implementation of the RelE-RelB toxin-antitoxin system. The toxin RelE constrains bacterial growth by mRNA degradation, thereby inhibiting translation, whereas the antitoxin RelB inhibits this toxic effect by forming complexes with RelE. As seen in Figure 1, three different protein complexes are formed, namely RelB<SUB>2</SUB>, RelB<SUB>2</SUB>RelE, and RelB<SUB>2</SUB>RelE<SUB>2</SUB>, containing zero, one, and two RelE molecules respectively <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://doi.org/10.1016/j.jmb.2008.04.039.">Guang-Yao Li, Yonglong Zhang, Masayori Inouye, Mitsuhiko Ikura, Structural Mechanism of Transcriptional Autorepression of the Escherichia coli RelB/RelE Antitoxin/Toxin Module, In Journal of Molecular Biology, Volume 380, Issue 1, 2008, Pages 107-119, ISSN 0022-2836</A></SPAN></SPAN>.
</P><DIV class="figure-text"><P><B>Figure 1.</B> The three toxin-antitoxin complexes RelB<SUB>2</SUB>, RelB<SUB>2</SUB>RelE, and RelB<SUB>2</SUB>RelE<SUB>2</SUB>.</P></DIV><P>The expression of both RelE and RelB is regulated by the <I>relBE</I> promoter, which is influenced differently by each of the complexes, as seen in Figure 2. When small amounts of RelE is present, RelB<SUB>2</SUB> and RelB<SUB>2</SUB>RelE repress transcription through <I>relBE</I> by binding to the operator sequence. However, when high amounts of RelE are present, the toxin mitigates this repression by reacting with complexes bound to the operator sequence <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>.
</P><DIV class="figure-text"><P><B>Figure 2.</B> The interactions between the toxin-antitoxin complexes and the relBE promoter controlling the expression of RelE and RelB. RelE mediates the degradation of mRNA, thereby inhibiting translation.</P></DIV><P>During starvation, the half-life of RelB decreases significantly due to a Lon-protease<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>, causing a shift in the equilibrium of RelB and RelE to a higher level of RelE. In a non-starvation situation, the interactions with the operator sequence keeps the amount of free RelE at a low level, thereby stabilising the system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. In our simulation, the shift in equilibrium is made by introducing additional expression of RelE. 
Two different models were used with two different approaches:
</P><OL class="list"><LI>How a given configuration of RelB and RelE production increases the RelE concentration and whether it could induce dormancy within 2 hours.</LI><LI>The time required for the bacteria to exit dormancy for each of the configurations, that is, how long it takes the levels of free RelE to decrease again. </LI></OL><P class="P-Larger"><B>Rates and Reactions</B></P><P>The skeleton of toxin-antitoxin system inherent to <I>E. coli</I> in the model was based on the study by Cataudella et al. 2012 <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. In an <I>E. coli</I> cell with a size of 1-2 μm, each nM of concentration can be approximated to 1 molecule. Thus all units are converted to molecules<SUP>-1</SUP>, as this fits the premises of the Gillespie algorithm. To simplify the model, the high affinity of RelE and RelB was used to ignore single RelB and only consider the dimer, RelB<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.uniprot.org/uniprot/P0C079">UniProtKB - P0C079 (RELB_ECOLI)</A></SPAN></SPAN>. Thus, all mentions of RelB in the model refers to its dimer. 

Whilst RelB has a relatively low half-life at about 3-5 minutes<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>, RelE is rather stable and its half-life, here set to 43 min, is primarily an effect from dilution caused by the bacterial growth<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>. However, when growth is restricted during dormancy, the half-life of RelE is increased to 2000 min, corresponding to approximately one day, as the dilution effect is no longer applicable. As the protein complexes are relatively stable, their half-life was set to the same as RelE. However, for RelE to become active in the inherent system under starvation, RelB in complexes must decay<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>, for which reason the rate was set to a fourth of free RelB.


The transcription rates of RelE and RelB are based on the concentration of RelE and RelB under stable conditions where RelB is ten times more prevalent than RelE<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN>. Consequently, RelB has been given a transcription rate 100 times higher than RelE to make up for the higher half-life of RelE.

For the full set of reactions, <A href="https://static.igem.org/mediawiki/2017/1/1a/T--SDU-Denmark--Equations-for-model.pdf" target="_blank">read here</A>
Each bacterial chromosome has two operators, each of which can bind one RelB<SUB>2</SUB> dimer (O(RelB<SUB>2</SUB>)) or either one or two RelB<SUB>2</SUB>RelE complexes (O(RelB<SUB>2</SUB>RelE) and O(RelB<SUB>2</SUB>RelE)<SUB>2</SUB>)<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>. 
Each cell is assumed to have four chromosomes with one relBE promoter each, as this is an average number of chromosomes for an exponentially growing <I>E. coli</I> cell<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>. This was found to stabilise the inherent system considerably compared to a system containing one chromosome per cell, as the systems exhibited similar behaviour but with different amounts of noise<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN>.

The initial values in the model are listed below.
</P><TABLE class="table"><TBODY><TR><TH> Molecule </TH><TH> Initial number of copies </TH></TR><TR><TD> mRNA </TD><TD> 7 </TD></TR><TR><TD> RelB<SUB>2</SUB></TD><TD> 410 </TD></TR><TR><TD> RelE </TD><TD> 0 </TD></TR><TR><TD> RelB<SUB>2</SUB>RelE </TD><TD> 65 </TD></TR><TR><TD> RelB<SUB>2</SUB>RelE<SUB>2</SUB></TD><TD> 11 </TD></TR><TR><TD> Free operator sites </TD><TD> 0 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>) </TD><TD> 2 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>RelE) </TD><TD> 0 </TD></TR><TR><TD> O(RelB<SUB>2</SUB>RelE)<SUB>2</SUB></TD><TD> 2 </TD></TR></TBODY></TABLE><P>All values are integers as the Gillespie algorithm works with discrete numbers of molecules. The values were chosen based on a stable equilibrium found by letting the model run a simulation of the inherent system over 450 minutes with different starting values. 

Ranging from 1-350 molecules per minute, the implemented expression rates of RelE and RelB in the model might seem too high, as the rates in the inherit system is effectively around 80-100 for RelB and 2-5 for RelE. However, the possibility of placing the <I>relE</I> and <I>relB</I> genes under regulation of controllable promoters makes the high total production values reasonable.

When the inherent toxin-antitoxin system is activated under starvation, 40-70 molecules of free RelE are found in each cell, making it reasonable to believe that the cells enter dormancy when a few tens of free RelE copies are present. This result obtained from the model is in agreement with literature<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN>
For the full list of constants, see the attached Table 1 at the bottom of this page. 
</P><P class="P-Larger"><B>Running the Model</B></P><P>To give a stochastic view of the system, the Gillespie algorithm was run in the computer programming language MATLAB, utilising an <A href="https://se.mathworks.com/matlabcentral/fileexchange/34707-gillespie-stochastic-simulation-algorithm" target="_blank">implementation</A> made by MATLAB user Nezar. 

For the runs simulating dormancy, deterministic initial values were used and the system was run for 30 minutes without activation of the inserted toxin promoter. This resulted in a stochastic distribution of initial values mimicking variations between cells. Analysis showed, that 30 minutes was enough for the model to find a stable distribution, which is realistic considering the growth cycle of an <I>E. coli</I> cell. 

For the runs simulating activation, the data generated at the end of a dormancy run was used as initial value and deactivated expression of RelE. When the concentration of free RelE decreases to below 15 copies, a cell is considered active. This value was probably set too low, but tests displayed marginal difference between 15 and 45 copies, where the lower limit was chosen to decrease uncertainty of the cell state. 

All runs simulated 1000 cells, which should be sufficient to get stable averages and the model assumed well-mixed conditions in every cell and considered each cell independently. Furthermore, the model has no cut-off for maximum values of RelE, as the exact relation between RelE concentration and dormancy state is unknown, yet a functional cut-off was found through activation times.
<A href="https://static.igem.org/mediawiki/2017/1/1e/Matlab-scripts_SDU.zip" target="_blank">Download of Matlab-scripts</A></P><P class="P-Larger"><B>Table 1</B></P><TABLE class="table"><TBODY><TR><TH>Constant</TH><TH>Identifier</TH><TH>Units</TH><TH>Value</TH></TR><TR><TD>mRNA transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>α<SUB>0</SUB></TD><TD>1/min</TD><TD>154.665</TD></TR><TR><TD>mRNA half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>m</SUB></TD><TD>min</TD><TD>7.2</TD></TR><TR><TD>RelB half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>B</SUB></TD><TD>min</TD><TD>4.3</TD></TR><TR><TD>RelE half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>τ<SUB>E</SUB></TD><TD>min</TD><TD>43      (growing)  2000 (dormant)</TD></TR><TR><TD>Bound RelB half-life<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812701/">Overgaard M., Borch J., Gerdes K. RelB and RelE of Escherichia coli form a tight complex that represses transcription via the ribbon-helix-helix motif in RelB. J. Mol. Biol. 2009;394:183–196. doi: 10.1016/j.jmb.2009.09.006</A></SPAN></SPAN></TD><TD>τ<SUB>c</SUB></TD><TD>min</TD><TD>17</TD></TR><TR><TD>RelB transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>B</SUB></TD><TD>1/min</TD><TD>15</TD></TR><TR><TD>RelE transcription rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD><I>trans</I><SUB>E</SUB></TD><TD>1/min</TD><TD>0.3</TD></TR><TR><TD>Binding rate<SPAN class="reference"><SPAN class="referencetext">Sneppen K, Zocchi G. Physics in Molecular Biology. Cambridge, UK: Cambridge University Press; 2005.</SPAN></SPAN></TD><TD>k <SUB>b</SUB></TD><TD>1/min</TD><TD>3.8</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD>K<SUB>D</SUB> (B<SUB>2</SUB>E)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate B<SUB>2</SUB>E<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18532983">Overgaard, M., Borch, J., Jørgensen, M. G. and Gerdes, K. (2008), Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity. Molecular Microbiology, 69: 841–857. doi:10.1111/j.1365-2958.2008.06313.x</A></SPAN></SPAN></TD><TD> K<SUB>D</SUB>(B<SUB>2</SUB>E<SUB>2</SUB>)</TD><TD>molecules</TD><TD>0.3</TD></TR><TR><TD>Dissociation rate O.B<SUB>f</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>K<SUB>D1</SUB></TD><TD>molecules</TD><TD>10</TD></TR><TR><TD>Dissociation rate O.B<SUB>2</SUB>E<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD> K<SUB>D2</SUB></TD><TD>molecules</TD><TD>0.04</TD></TR><TR><TD>Dissociation rate O.(B<SUB>2</SUB>E)<SUB>2</SUB><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1998.00993.x/abstract">Gotfredsen, M. and Gerdes, K. (1998), The Escherichia coli relBE genes belong to a new toxin–antitoxin gene family. Molecular Microbiology, 29: 1065–1076. doi:10.1046/j.1365-2958.1998.00993.x</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  K<SUB>D3</SUB></TD><TD>molecules</TD><TD>30</TD></TR><TR><TD>Cleavage rate<SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="http://www.sciencedirect.com/science/article/pii/S0092867402012485?">Pedersen K, et al. The bacterial toxin RelE displays codon-specific cleavage of mRNAs in the ribosomal A site. Cell. 2003;112:131–140. doi: 10.1016/S0092-8674(02)01248-5</A></SPAN></SPAN><SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413109/">Cataudella I., Trusina A., Sneppen K., Gerdes K., Mitarai N. Conditional cooperativity in toxin-antitoxin regulation prevents random toxin activation and promotes fast translational recovery. Nucleic Acids Res. 2012;40:6424–6434. doi: 10.1093/nar/gks297</A></SPAN></SPAN></TD><TD>  k<SUB>c</SUB></TD><TD>1/min  1/molecules</TD><TD>2.0</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><DIV class="modal fade" id="model-results" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Modelling Results of the RelE-RelB System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Results</B></P><P>As the bacteria only require a few tens of RelE molecules to enter dormancy, the threshold was placed at 40 copies to allow for lag in activation. This is of course an oversimplification, but it is not a problem that activation and dormancy are defined at different levels, since the exact number of RelE molecules required to induce dormancy is unknown.<B>The Impact of Promoters with Different Production Rates on RelE Expression</B>
During the implementation of RelE, gene expression was simulated for promoters with different strengths, which were chosen through an iterative process. Promoters with production rates of 3.5 and 10.5 molecules per minute, both induce dormancy rather slowly, with the latter inducing dormancy in approximately 50 minutes. In cells cloned with a promoter producing 35 molecules per minute, the cells will enter dormancy in about 10 minutes, while promoters producing 105 and 350 molecules per minute both have a negligible timeframe. In the simulated dormancy system, the three strongest promoters exhibited similar results, indicating that a certain threshold value had been transcended. This means, that not only was the gene expression disproportional to the promoter strength, but the risk of overshooting was also increased tremendously.</P><DIV class="figure-text"><P><B>Figure 1.</B> The increase of free RelE molecules in <I>E. coli</I> cells, after activation of the artificial RelE production, shown logarithmically. The condition for induced dormancy is an amount of free RelE molecules around tens of copies. The three highest levels of RelE production, correlating with the highest promoter strengths, show little difference in the time at which dormancy occurs. When the RelE production is set to 10.5 molecules per minute, dormancy is induced more slowly and stabilises at lower concentrations. The lowest RelE production value does not trigger dormancy, and has only little effect on the system.</P></DIV><P><B>RelB is Required for Activation of Bacteria after Dormant State</B>
If RelB was not expressed, the bacteria remained dormant for hours after RelE production had ceased, making it clear, that production of the antitoxin RelB was necessary for activation. Considering the stability of RelE in non-growing conditions, it was not surprising to find that RelB production would be the primary element in sequestering free RelE.</P><DIV class="figure-text"><P><B>Figure 2.</B> The logarithmic plots shows that the decrease in free RelE in dormant bacteria is low without artificial expression of RelB. None of the simulated bacteria reentered an active state within the modelled time.</P></DIV><P><B>Appropriate Ratio of RelE and RelB Expression is Essential</B>
Different expression rates of RelB were combined with production rates of RelE at 35 and 100 molecules per minute, corresponding to relatively medium and strong expression levels respectively. This revealed that variation in RelB had a higher impact on the time required before the dormant state was reached when the RelE production is lowered. Out of the established configurations, the RelB<SUB>2</SUB>:50-RelE:35 configuration showed promising results. Compared to the RelB<SUB>2</SUB>:35-RelE:35 configuration, where only few bacteria reenter an active state within the modelled time set to 2.5 hours, the RelB<SUB>2</SUB>:50-RelE:35 configuration revealed a high sensitivity to the expression level of RelB. This indicated a need for the expression of RelB to be higher than RelE, but as the best results were achieved at low production rates of RelE, it is important to stringently control the expression of RelB to ensure that the bacteria are able to enter dormancy.</P><DIV class="figure-text"><P><B>Figure 3.</B> The variation in time required for the bacteria to enter an active state for different expression levels of RelB is dependent on the level of RelE expression. All configurations achieve dormancy within the modelled time.</P></DIV><DIV class="figure-text"><P><B>Figure 4.</B> RelB<SUB>2</SUB>:50-RelE:35 induces an active state within minutes, whereas RelB:35-RelE:35 only causes few of the bacteria to enter an active state. In the remaining configurations all bacteria remained dormant.</P></DIV><P class="P-Larger"><B>Discussion of Model Regarding the Artificial Dormancy System</B></P><P><B>Model Limitations</B>
As the model only simulates the dormancy for 2 hours, not all configurations have reached equilibrium, therefore these configurations might attain a higher concentration of free RelE than modelled. This could result in a prolonged phase reentering an active state.
The activation model has a rather weak predictability, as the half-life of RelE is quite high. This essentially means, that the model is unable to reduce the total amount of RelE, given by RelE<SUB>tot</SUB>=RelE<SUB>free</SUB>+RelE<SUB>bound</SUB>, because of the short simulated timespan. Thus, in activation runs, where the number of free RelE molecules reaches a high level, the amount of RelE<SUB>bound</SUB> is equally high. The problem arises, since the model works under the assumption that bound RelB is not completely stable, causing RelE-RelB complexes to dissociate, whereby RelE is freed. This causes the induced RelB expression to approach equilibrium, implying that the decrease in free RelE is rather slow. It is therefore not only the amount of free RelE that determines the activation time, but also whether the amount of dissociating complexes is high enough to counter the RelB production. Hence, it has no relevance to further test high RelE production in this model, as a high number of complexes will easily be achieved. <B>Light Sensitivity</B>
One thing the model does not include, is the actual sensitivity to light. Out in the open, the amount of light is rarely an on/off switch, which means there will be periods with varying degrees of activation. Since the bacteria should be active during overcast days, the system requires a threshold, both in the sensitivity of the light-regulated promoter, but also in the activation of the toxin-antitoxin system. Variation of light is implicitly modelled through variation in promoter strength, for instance a half probability of activation translates roughly to a half production rate in the individual bacteria. Because of this, it is important not only to find functioning configurations, but also to investigate the closely related configurations, so that the bacteria neither lay dormant at overcast days, nor make the dormancy system obsolete in moonlight. 
</P></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV><P class="P-Larger"><B>Approach</B></P><P>
In 2004 the <A href="https://2004.igem.org/austin.cgi" target="_blank">Austen and UCSF iGEM team</A> created a <SPAN class="highlighted">device sensitive to light,</SPAN> laying the foundation for the <A href="http://parts.igem.org/Coliroid" target="_blank">Coliroid project</A>. In this project, the <SPAN class="highlighted">system is combined with the RelE-RelB toxin-antitoxin system</SPAN> in the endeavour to mediate <SPAN class="highlighted">light-dependent dormancy in bacteria</SPAN>. As tight regulation is required for the RelE-RelB system <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294780/&quot;">Tashiro Y, Kawata K, Taniuchi A, Kakinuma K, May T, Okabe S. RelE-Mediated Dormancy Is Enhanced at High Cell Density in Escherichia coli. J Bacteriol. 2012;194(5):1169-76.</A></SPAN></SPAN>, <SPAN class="highlighted">modelling of the toxin-antitoxin system</SPAN> is essential. The impact of different RelE-RelB expression levels was simulated by modelling. Using the results obtained by this modelling, a hypothetical working system-design was devised. 

On basis of the modulated system, the potential of different vectors and promoters in various combinations was tested. This constitutes the foundation for how the design of the light-dependent dormancy system in <I>E. coli</I> has been optimised, and the final approach shaped. Ultimately, the light-dependent dormancy system, which is illustrated in Figure 3, was composed of the following parts: 
</P><UL class="list"><LI><SPAN class="highlighted">The photocontrol device controlled by the PenI-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0074" target="_blank">BBa_R0074</A>, on a high copy vector.</SPAN></LI><LI><SPAN class="highlighted">The antitoxin RelB controlled by pBAD, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449031" target="_blank">BBa_K2449031</A>, on a low copy vector.</SPAN></LI><LI><SPAN class="highlighted">The toxin RelE controlled by the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>, on either a low copy vector or the chromosome. </SPAN></LI></UL><P>For further information about our approach, <SPAN class="btn-link btn-lg" data-toggle="modal" data-target="#light-sensing-system-approach">read here</SPAN>.</P><DIV class="figure-text"><P><B>Figure 3.</B> The final design of the light-dependent dormancy system. RelE under control of the OmpR-regulated promoter on the chromosome or a low copy vector, RelB under the control of the pBAD promoter on a low copy vector, and the photocontrol device controlled by the PenI-regulated promoter on a high copy vector.</P></DIV><DIV class="modal fade" id="light-sensing-system-approach" tabindex="-1" data-backdrop="false" style="background-color:rgba(0,0,0,0.6);"><DIV class="modal-dialog modal-lg"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal">×</BUTTON><H2 class="modal-title">Light Sensing System</H2></DIV><DIV class="modal-body" margin-right="10%"><DIV class="row"><DIV class="col-md-10"><P class="P-Larger"><B>Approach</B></P><P><B>Balancing Bacterial Dormancy Requires Accurate Regulation of the System</B>
The genes needed for inducing dormancy when the bacteria are not exposed to light, are found in the photocontrol device part, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>. This part is composed of three genes named <I>ho1</I>, <I>pcyA</I>, and <I>cph8</I>, all of which are essential to ensure the cells ability to respond to red light. When the photocontrol device is exposed to light, a phosphorylation cascade activates the transcription factor OmpR, which in turn induces transcription through the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>. This system is combined with the RelE-RelB toxin-antitoxin system in the endeavour to mediate light-dependent dormancy in bacteria. In the first considered design of the light-dependent dormancy system, the aim was to clone the photocontrol device, <A href="http://parts.igem.org/Part:BBa_K519030" target="_blank">BBa_K519030</A>, under control of a constitutive promoter, RelE under control of the OmpR-regulated promoter, <A href="http://parts.igem.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>, and RelB under control of a constitutive promoter, all into one high copy BioBrick assembly plasmid pSB1C3, as seen on Figure 1.</P><DIV class="figure-text"><P><B>Figure 1.</B> All three components of the light-dependent dormancy system cloned into one high copy plasmid.</P></DIV><P><B>The Photocontrol Device was Placed under Control of a Constitutive Promoter</B>
From the <A href="http://parts.igem.org/Promoters/Catalog/Anderson" target="_blank">constitutive promoter family</A> the weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, the two medium promoters, <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, and <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, and the strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, were tested by fluorescence microscopy to determine which one to use for expression of the photocontrol device and RelB. Overnight cultures of the submitted parts expressed in <I>E. coli</I> TOP10 illustrated a clear gradient of increasing red fluorescent protein (RFP) expression correlated with the strength of the promoter, as seen on Figure 2.</P><DIV class="figure-text"><P><B>Figure 2.</B> Cultures of <I>E. coli</I> TOP10. From left to right: WT, the weak promoter, <A href="http://parts.igem.org/Part:BBa_J23114" target="_blank">BBa_J23114</A>, the two medium promoters, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>, and <A href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>, and the strong promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>.</P></DIV><P>
On the basis of the data obtained by fluorescence microscopy, the strong constitutive promoter, <A href="http://parts.igem.org/Part:BBa_J23102" target="_blank">BBa_J23102</A>, was chosen to control the photocontrol device. For still inexplicable reasons the photocontrol device emerged difficult to clone with the strong constitutive promoter. Although the molecular cloning of these parts was optimised several times very few successful clonings were accomplished, and the few times correct assembly was obtained, the BioBrick was not reproducibly purifiable. By sequencing, it was deduced that a region of the plasmid containing both the promoter and the BioBrick prefix had vanished. To circumvent this inconvenient cloning two other promoters, that are constitutive in <I>E. coli</I>, were examined, namely the PenI-regulated, <A href="http://parts.igem.org/Part:BBa_R0074" target="_blank">BBa_R0074</A>, and the Mnt-regulated, <A href="http://parts.igem.org/Part:BBa_R0073" target="_blank">BBa_R0073</A>, promoters. By performing fluorescence microscopy on composite parts of the promoters controlling yellow fluorescent protein (YFP), <A href="http://parts.igem.org/Part:BBa_I6102" target="_blank">BBa_I6102</A>, and <A href="http://parts.igem.org/Part:BBa_I6103" target="_blank">BBa_I6103</A>, respectively, the expression levels were assessed. The obtained results revealed that the PenI-regulated promoter facilitated very strong expression of the marker gene, whereas the expression controlled by the Mnt-regulated promoter was noticeably lower. Based on these findings, the photocontrol device was placed under the control of the PenI-regulated promoter instead of the strong constitutive promoter from the constitutive promoter family. As it turned out, this cloning likewise emerged difficult. After several attempts, it was decided to focus on the other aspects of the dormancy system.
<B>Regulation of the OmpR-dependent Promoter Required a Low Copy Vector </B>
The first construct containing the genes required for the light-induced dormancy was designed as shown in Figure 1. As the conducted modelling clarified, the necessity for stringent regulation of the RelE and RelB expression, the properties of the OmpR-regulated promoter were studied thoroughly. To assess the functionality of the OmpR-regulated promoter in practice, a reporter system containing the OmpR-regulated promoter controlling RFP was cloned into the <I>E. coli</I> strain MG1655 ΔOmpR. The phenotype of the resulting cultures revealed a dysregulation of the OmpR-regulated promoter. Thorough research lead to the finding that the OmpR-dependent promoter is not controllable when cloned on a high copy vector. As the modelling revealed, and which is evident from <SPAN class="reference-2">Figure 2-Main-Page</SPAN>, a relatively low expression of RelE is required to induce dormancy, whereas high expression levels quickly result in overshooting. Since the OmpR-regulated promoter is an integrated part of the light sensing system, replacement is not an option. Therefore, the variability of the <I>relE</I> gene copy number was studied, and it was found that the OmpR-regulated promoter should be cloned into the bacterial chromosome or a low copy vector to obtain proper regulation <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/16306980&quot;">Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, et al. Synthetic biology: engineering Escherichia coli to see light. Nature. 2005;438(7067):441-2.</A></SPAN></SPAN>. This intriguing finding let to the aspiration to investigate the controllability of the OmpR-dependent promoter on vectors with different copy numbers compared to the chromosome, thereby improving the characterisation of the promoter for the benefit to future iGEM teams. 

To incorporate DNA onto the bacterial chromosome, homologous recombination with the red λ recombinase is a suitable approach <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/2958633&quot;">Thompson JF, de Vargas LM, Skinner SE, Landy A. Protein-protein interactions in a higher-order structure direct lambda site-specific recombination. Journal of molecular biology. 1987;195(3):481-93.</A></SPAN></SPAN>. Using this technique, a short fragment of chromosomal DNA at the bacterial attachment site attB <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/14687564&quot;">Groth AC, Calos MP. Phage integrases: biology and applications. Journal of molecular biology. 2004;335(3):667-78.</A></SPAN></SPAN> can be replaced with a linear DNA fragment encoding the OmpR-dependent promoter, RelE, and an chloramphenicol resistance cassette. Using polymerase chain reaction (PCR), the linear DNA sequence was flanked by sequences, which are homologous to part of the chromosome. The linear DNA fragment was electroporated into bacteria containing the pKD46 plasmid, encoding the red λ recombinase <SPAN class="reference"><SPAN class="referencetext"><A target="blank" href="”https://www.ncbi.nlm.nih.gov/pubmed/2958633&quot;">Thompson JF, de Vargas LM, Skinner SE, Landy A. Protein-protein interactions in a higher-order structure direct lambda site-specific recombination. Journal of molecular biology. 1987;195(3):481-93.</A></SPAN></SPAN>, which mediated the recombination. The fundamental concept of this approach is illustrated in Figure 3.
</P><DIV class="figure-text"><P><B>Figure 3.</B> The principle behind the recombination. By PCR, the two flanking sequences are assembled with the fragment containing the chloramphenicol resistance gene (camR), the OmpR-regulated promoter, and the relE gene. The flanking sequences are homologous to part of the chromosome around the bacterial attachment site (attB), enabling the homologous recombination.</P></DIV><P>
Using this approach, it was attempted to assemble the OmpR-regulated promoter controlling RelE with the flanking regions by PCR. The assembly of the fragment derived from the plasmid with either one of the flanking fragments was achieved. However, the assembly of the complete chromosomal insertion fragment containing all three segments emerged problematic. Several attempts with numerous process optimisations were performed, but unfortunately without success. To circumvent this, another attachment site was chosen. The assembly of the complete fragment by PCR was achieved and chromosomal insertion by electroporation was attempted several times, but in vain. In consideration of the fact that RelE is a bacterial toxin, the cloning of this gene was inconvenient. The model indicated, that overexpression of the antitoxin RelB would bypass this difficulty. As the homologous recombination of this fragment emerged such a challenging task, it was decided to focus on the conduct of the OmpR-regulated promoter on different plasmids, as this was evidently the more convenient approach of the general implementation of the OmpR-regulated promoter. 
<B>An Inducible Promoter was Chosen to Regulate the Gene Expression of the Antitoxin RelB</B>
Originally, it was thought to place the antitoxin RelB under a constitutive promoter of appropriate strength. However, as the modelling revealed, strict regulation of RelB is essential to counteract the toxic effect of RelE and enable a functional dormancy system. Thus, it was deduced that it would be more suitable to utilise an inducible promoter for this purpose and it was decided to put RelB under control of the LacI-regulated, lambda pL hybrid promoter, <A href="http://parts.igem.org/Part:BBa_R0011" target="_blank">BBa_R0011</A>. The <A href="https://2015.igem.org/Team:William_and_Mary" target="_blank">William and Mary team</A> found that the LacI-regulated, lambda pL hybrid promoter had a low level of noise,  a measure of the variability in gene expression between cells in the population, when cloned into a low copy vector. Therefore, this regulated promoter was chosen. 

To mimic the expression of  RelB, a reporter system composed of the LacI-regulated, lambda pL hybrid promoter and GFP was assembled. Hereby, the appropriate concentration of IPTG required to induce the promoter on a low copy vector, was identified. During the cloning, it was discovered that the site between the LacI-regulated, lambda hybrid promoter and this particular construct under the tested conditions had formed a hotspot for transposons. In the light of this finding, another inducible promoter was chosen. The <A href="https://2015.igem.org/Team:HKUST-Rice" target="_blank">HKUST-Rice iGEM 2015 team</A> demonstrated that induction of the AraC-regulated promoter, pBAD, caused a gradual increase in gene expression when cloned into the low to medium copy plasmid, pSB3K3, in contrast to an all-or-none behavior when cloned into the high copy vector, pSB1K3. Taking these results into consideration, the pBAD promoter was used in combination with the pSB3K3 vector to regulate the expression of the antitoxin RelB. Again, a reporter system was used to mimic the expression of RelB, as the part, <A href="http://parts.igem.org/Part:BBa_I6058" target="_blank">BBa_I6058</A>, containing YFP controlled by pBAD, was assessed on different vectors. 
<B>The Final Approach for the Three Components Comprised Three Different Vectors</B>
Based on the modelling the system approaches reviewed in the preceding part, the final design, which is illustrated on Figure 4, was established. Ultimately, the dormancy system was composed of the photocontrol device controlled by the PenI-regulated promoter on a high copy vector, RelB controlled by pBAD, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2449031" target="_blank">BBa_K2449031</A>, on a low copy vector  and RelE controlled by the OmpR-regulated promoter on either a low copy vector or the chromosome.

</P><DIV class="figure-text"><P><B>Figure 4.</B> The final design of the light-dependent dormancy system. RelE under control of the OmpR-regulated promoter on the chromosome or a low copy vector, RelB under the control of the pBAD promoter on a low copy vector, and the photocontrol device controlled by the PenI-regulated promoter on a high copy plasmid.</P></DIV></DIV></DIV></DIV><DIV class="modal-footer"><A href="" class="btn btn-default" data-dismiss="modal">Close</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>