<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UB_Indonesia_backup"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UB_Indonesia/backup">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UB_Indonesia/backup&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UB_Indonesia/backup&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UB_Indonesia/backup&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UB_Indonesia/backup" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UB Indonesia/backup</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>iGEM2014 | UB INDONESIA</TITLE></P><DIV class="navbar navbar-fixed-top" data-activeslide="1"><DIV class="container"><DIV class="nav-collapse collapse navbar-responsive-collapse"><UL class="nav row"><LI data-slide="1" class="col-12 col-sm-2"><A id="menu-link-1" href="#slide-1" title="Next Section"><SPAN class="text">HOME</SPAN></A></LI><LI data-slide="2" class="col-12 col-sm-2"><A id="menu-link-2" href="#slide-2" title="Next Section"><SPAN class="text">PROJECT</SPAN></A></LI><LI data-slide="3" class="col-12 col-sm-2"><A id="menu-link-3" href="#slide-3" title="Next Section"><SPAN class="text">NOTEBOOK</SPAN></A></LI><LI data-slide="4" class="col-12 col-sm-2"><A id="menu-link-4" href="#slide-4" title="Next Section"><SPAN class="text">HUMAN PRACTICE</SPAN></A></LI><LI data-slide="5" class="col-12 col-sm-2"><A id="menu-link-5" href="#slide-5" title="Next Section"><SPAN class="text">COLLABORATIONS</SPAN></A></LI><LI data-slide="6" class="col-12 col-sm-2"><A id="menu-link-6" href="#slide-6" title="Next Section"><SPAN class="text">PARTS</SPAN></A></LI></UL></DIV><DIV class="nav-collapse collapse navbar-responsive-collapse"><UL class="nav row"><LI data-slide="7" class="col-12 col-sm-4"><A id="menu-link-7" href="#slide-7" title="Next Section"><SPAN class="text">GALERY</SPAN></A></LI><LI data-slide="8" class="col-12 col-sm-4"><A id="menu-link-8" href="#slide-8" title="Next Section"><SPAN class="text">ABOUT US</SPAN></A></LI><LI data-slide="9" class="col-12 col-sm-4"><A id="menu-link-9" href="#slide-9" title="Next Section"><SPAN class="text">CONTACT</SPAN></A></LI></UL></DIV></DIV></DIV><DIV class="slide story" id="slide-1" data-slide="1"><DIV class="container"><DIV id="home-row-1" class="row clearfix"><DIV class="col-12"><H1 class="font-semibold"><FONT color="white">BRAWIJAYA UNIVERSITY <SPAN class="font-thin">, INDONESIA</SPAN></FONT></H1><H4 class="font-thin"><FONT color="white">This is team wiki to share our iGEM experience</FONT></H4></DIV></DIV><DIV id="home-row-2" class="row clearfix"><DIV class="col-12 col-sm-4"><SPAN>iGEM UB</SPAN></DIV><DIV class="col-12 col-sm-4"><SPAN>BRAWIJAYA UNIVERSITY</SPAN></DIV><DIV class="col-12 col-sm-4"><SPAN>iGEM2014</SPAN></DIV></DIV></DIV></DIV><DIV class="slide story" id="slide-2" data-slide="2"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-bold"> PROJECT</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-12 font-thin">Abstract <SPAN class="font-semibold">...</SPAN></DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="background" style="display:none;width:700px;"><H2><CENTER>“A BETTER FUTURE WITH SYNTHETIC BIOLOGY”</CENTER></H2><P align="justify">Study of synthetic biology emerged as a multidisciplinary science that help people in solving various problems. Among the general public, synthetic biology is famous as bioengineering and extreme genetic engineering which have an advantage in redesigning a life through genetical circuit, thus allows people to manipulate a new organism for use in treatment and diagnosis of disease, provision of quality seeds, remediation of environmental pollution caused by industrial and mining as well as the mitigation of global warming. 

Indonesia as one of the currently high development countries in science is allowing for synthetic biology to grow rapidly. Prospects for the synthetic biology's development in Indonesia, among others, is the independence of technology in the health sector include the development of a specific diagnosis, the provision of pharmaceutical raw materials, and technology screening for Indonesia natural resources which have potency to treat various types of diseases.

Brawijaya University iGEM team will develop synthetic biology in order to solve health sector problems, focus in Cervical Cancer. We have three sub-projects, they are: plant engineering for cervical cancer prevention, developing Cervical Cancer Care Program application for smartphone as helpdesk, and scanning technology for natural materials treatment (SCT Kit).</P></DIV><H2 class="font-thin"><FONT color="#fff">Background</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="overview" style="display:none;width:700px;"><H2><CENTER>“UB_INDONESIA BRINGS C3P (CERVICAL CANCER PROGRAM)”</CENTER></H2><P align="justify">
Cervical cancer is the second most common cancer in women worldwide. Yet, because of poor access to screening and treatment services, the number of deaths occur in women increase highly, especially in our country, Indonesia. Effective methods for early detection of precancerous lesions using cytology (Pap smear) exist, but not effective enough for detection in all age. However, new technological developments, using synthetic biology, offer the potential to tackle cervical cancer in a more comprehensive way and build a healthier future for girls and women. Here we go with our great idea!
UB IGEM TEAM is on progress in developing C3P (Cervical Cancer Care Program) as the theme to give solutions for cervical cancer cases. Starting from creating a rapid test tool using HPV DNA which is integrated with the software, creating a high antioxidant tea for preventing efforts, and creating a kit for selecting potential anti-cancer herbs.
We try to make an idea to build a new prototype for early detection to cervical cancer, especially caused by HPV 18 and HPV 16. We construct a new specific TALEs that can bind to our target sequences and we called it TALE 1 and TALE 2. In order to make sure the presence of DNA  from HPV 16 and 18, we need reporter which connect to detector.  Fluorescent protein usually used as reporter gene to study about gene expression. It will be connected to detector using linker. When the HPV DNA is bind to detector, blue colour will appear in the second strip. The Intensity of colour will be detected in our software and we can determined the risk level of cervical cancer disease. Dark blue colour show that the amount of  DNA  from HPV 16 and 18 is high and indicate the patient get severe cervical cancer.
Beside that, we try to take the advantage of tea plant. Tea plant (Camelia sinensis) has catechin compounds that contains antioxidants.  Catechin family that most effective used as antioxidant is Epigallocathecin gallate (EGCG).  Based on research in vitro or in silico, EGCG is able to inhibit the proliferation of cancer cells because stop over-expression between L1 HPV 16- EGFR (Epithel Growth Factor Receptor) bond but there are few of EGCG content in tea plant. Based on these problems, we would to over- expression EGCG by knockdown non-compound EGCG gene, so hopefully the content of EGCG on tea are more prominent.
SCT (screening Cancer Therapy) is a kit which contains of hela cells as cervical cancer cells and it was designed to screening the plants that have compund potentially for cervical cancer therapy. Herbs that potentially as a therapy of cervical cancer in the kit are marked by the absence of green or red colors, while the negative results are marked by the emergence of green or red in the kit. The kit is expected to provide convenience to the researchers in the field of cervical cancer and cervical cancer effects can be mitigated with natural compound that not a lot of side effects. 
From all of above, we bring C3P (Cervical Cancer Program) for iGEM 2014. We named our each project with preventing, screening and therapy. beside, we also do social action in other to socialize our project and be closer to society to inform them how to avoid cervical cancer. 
The responsible of the scientist is not only the result in the lab, but also share “the right thing” for society. So, thanks to iGEM that gives us an opportunity for doing both, research and social action. <B>“Share your knowledge. It’s a way to achieve immortality.” - Dalai Lama</B></P></DIV><H2 class="font-thin"><FONT color="#fff">Overview</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="preventing" style="display:none;width:700px;"><H2><CENTER>“Prevent is Better than Cure”</CENTER></H2><P align="justify">Indonesia is an agricultural country that most of their society works as farmer. One of commodity in Indonesia is tea plantation which is located in Malang (Brawijaya University takes place). Malang is resited at 440-667 (mdpl) altitude, one of the tourist destinations in East Java because of the potential of natural and climatic owned. Malang climate conditions during 2008 temperatures recorded ranging from 22.7 ° C to 25.1 ° C. While the maximum temperature reached 32.7 ° C and minimum temperature of 18.4 ° C. The average air humidity range 79% - 86%. With a maximum moisture content of 99% and a minimum at 40%. These conditions favor the development of the tea plant (Camellia sinensis). 

Tea extract contain EGCG used as the most effective antioxidant to against cervical cancer. But, levels of EGCG (Epigallocathecin gallate) in tea is very slightly. So, we will increase the level of this EGCG compound using siRNA to knockdown gene non-EGCG with expectation it will produce high yield of antioxidant. The applications are for supplement or feminine hygiene.

Tea plant (Camelia sinensis) has catechin compounds that contains antioxidants.  Catechin family that most effective used as antioxidant is Epigallocathecin gallate (EGCG).  Based on research in vitro or in silico, EGCG is able to inhibit the proliferation of cancer cells because stop over-expression between L1 HPV 16- EGFR (Epithel Growth Factor Receptor) bond but there are few of EGCG content in tea plant. Based on these problems, we would to over- expression EGCG by knockdown non-compound EGCG gene, so hopefully the content of EGCG on tea are more prominent.  We design siRNA sequence from LAR gene (to inhibit non-EGCG compound) using bioinformatics tool and use P97 promoter to initiate expression in mammalian cell (using HeLa cell). That part will be insert into the psB1C3 linear backbone. To checking performance of this part, we insert sequence target of siRNA into the plasmid Bba_E0240 and get the promoter CMV from Bba_K747096 part. We also prepare for positive control and negative control for this experiment. Positive control part containing siRNA for GFP, insert into Bba_K747096 part that contain CMV promoter, and negative control containing CMV promoter and insert to the Bba_E0240 part containing GFP. Each part will be growth in HeLa cell culture and observed the succeding by fluorescent microscope.
</P></DIV><H2 class="font-thin"><FONT color="#fff">Preventing</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="screening" style="display:none;width:700px;"><H2><CENTER>“Cercival Cancer Screening Kit”</CENTER></H2><P align="justify"><B>
A. DETECTOR
1. Background</B>
Cervical cancer is one of the most common cancers in womenworldwide,with an estimated global incidence of 470,000 newcases and approximately 233,000 deaths per year. Then, we try to make an idea to build a new prototype for early detection to cervical cancer, especially caused by HPV 18 and HPV 16. From the previous results of the two teams, Calgary Team 2013 and Freiburg Team 2012, we build a new prototype with some combination from those teams. We learn about the prototype sensor from Calgary Team 2012 that consist of detector, linker, and reporter that combined into one small test pack. Then, to make the detector, we learn from Freiburg Team 2012 to constructing a new specific TALEs that can bind to our target sequences and we called it TALE 1 and TALE 2. 
TALEs or Transcription Activator-Like Effectors comes from natural bacterial effector proteins used by Xanthomonas sp. to modulate gene transcription in host plants to facilitate bacterial colonization (Boch J and Bogdanove, 2010).The central region of theprotein contains tandem repeats of 34 amino acids sequences (termed monomers) that arerequired for DNA recognition and binding (Kay, 2009). Here is the natural structure of TALE from Xanthomonas sp. :</P><CENTER>
Figure 1. Structure of TALEs derived from Xanthomonas sp. (Sanjana et al, 2013)</CENTER><P align="justify"><B>2. What did We Do ?</B>
As explained before in the Background, we try to make a prototype based on the concept of <A href="https://2013.igem.org/Team:Calgary">Calgary Team 2013</A>. So we construct a new TALEs that can bind with our target sequences, especially in HPV 18 and HPV 16. For building the detector, we received help from <A href="https://2012.igem.org/Team:Freiburg/Project/Overview">Freiburg Team 2012</A> to construct two new TALEs. First, we search the target sequences with the alignment of whole genome of HPV 18 and HPV 16 via NCBI and MEGA5. We search two target sequences in left side and right side. Here are some works that we’ve done:</P><CENTER>
Left side: site 7106 until 7119

Right side: site 7851 until 7864
</CENTER><P align="justify"><B>3. CONSTRUCTING TALEs</B>
Based on <A href="https://2012.igem.org/Team:Freiburg/Project/Tal">Freiburg Team 2012 method</A>, we designed our TALE into 2, TALE 1 and TALE 2. 
For TALE 1, we use: <A href="http://parts.igem.org/Part:BBa_K747012">(1) BBa_K747012,</A><A href="http://parts.igem.org/Part:BBa_K747016">(2) BBa_K747016,</A><A href="http://parts.igem.org/Part:BBa_K747042">(3) BBa_K747042,</A><A href="http://parts.igem.org/Part:BBa_K747048">(4) BBa_K747048,</A><A href="http://parts.igem.org/Part:BBa_K747064">(5) BBa_K747064,</A><A href="http://parts.igem.org/Part:BBa_K747091">(6) BBa_K747091</A>
For TALE 2, we use: <A href="http://parts.igem.org/Part:BBa_K747012">(1) BBa_K747012,</A><A href="http://parts.igem.org/Part:BBa_K747026">(2) BBa_K747026,</A><A href="http://parts.igem.org/Part:BBa_K747036">(3) BBa_K747036,</A><A href="http://parts.igem.org/Part:BBa_K747052">(4) BBa_K747052,</A><A href="http://parts.igem.org/Part:BBa_K747076">(5) BBa_K747076,</A><A href="http://parts.igem.org/Part:BBa_K747095">(6) BBa_K747095</A></P><P align="justify"><B>4. VECTOR</B>
For the vector for our TALEs, we received help from Lucas Schneider (member of Freiburg Team 2012) and Nicole Densch (Labmanager from <A href="http://www.bioss.uni-freiburg.de/cms/2766.html">BIOSS</A>, Freiburg). We request the pTALEN and pTAL – TF from BIOSS and sent it to us on Whatman paper. Based on Sanjana et al. (2013), TALE-TFs are constructed by replacing the natural activation domain within the TALE C-term with the synthetic transcription activation domain VP64.</P><CENTER>
Figure 2. TALEs can be used to generate custom transcription factors (TALE-TFs) and modulate the transcription of endogenous genes from the genome (Sanjana et al., 2013)
</CENTER><P align="justify">
TALENs are constructed by fusing a C-term truncation (+63aa) of the TALE DNA binding domain with the non-specific FokI endonuclease catalytic domain. TALENs form dimers through binding to two target sequences separated by ~17 bases (Sanjana et al., 2013).
</P><CENTER>
Figure 3. TALE nucleases (TALENs) can be used to generate site-specific double strand breaks to facilitate genome editing through non-homologous repair or homology-directed repair (Sanjana et al., 2013)
</CENTER><B>B. LINKER</B><P align="justify">
As we’ve said before that we inspired by the <A href="https://2013.igem.org/Team:Calgary">Calgary Team 2013</A>’s project, so we decided to learn the linker from previous work by them. The E/K coil is a heterodimeric coiled-coil that often used for biosensors and as an expression and affinity purification tag. The E coil contains all glutamic acid residues and the K coil contains all lysine residues. This coiled-coil can be found in a wide variety of proteins and contain a single type of secondary structure or the α-helix. There are two ways to increasing the stability of this sequence: the hydrophobic core of the coiled-coil (positions at a and d) and the α-helical propensity of surface esposed positions at b, c, e, f, and g (Litowski, 2002).
</P><CENTER>
Figure 4. Helical wheel of the E4/K4 heterodimer (Litowski, 2002)
</CENTER><P align="justify">
We used linker to connect  Tale and reporter. We used E and K coil since they can interact each other with high specificity. E-coil was fused to amilCP blue chromoprotein (reporter), while K-coil bound to Tale. This fusion make the reporter can be linked to Tale (detector) through K coil-. This fusion involving PCR and these part was digested and ligated as Igem protocol. 
</P><CENTER> </CENTER><B>C. REPORTER</B><P align="justify">
In order to make sure the presence of DNA  from HPV 16 and 18, we need reporter which connect to detector.  Fluorescent protein usually used as reporter gene to study about gene expression. Gene encoding fluorescent protein was synthesized in four stages (Laufer et al, 2013). Chromoprotein can be used as good reporter. Another useful feature of the chromoproteins is the “kindling” behavior, which makes them prospective photoactivatable markers. In this project, we used amilCP blue chromoprotein. This chromoprotein was extracted from Acropora millepora and  display purple-blue colour (Alieva et al, 2008).
This is one of chromoprotein collection from <A href="https://2012.igem.org/Team:Uppsala_University">Uppsala Team 2012</A>. This chromoprotein will be connected to detector using linker. amilCP bind to E coil and interact with Tale-K coil. This reporter show the presence of Human Papilloma Virus by colour change. When the HPV DNA is bind to detector, spesific sequence of HPV will be recognized and blue colour appears  in the second strip. The Colour intensity will be detected in our software and we can determined the risk level of cervical cancer disease. Dark blue colour show the amount of  DNA  from HPV 16 and 18 is high and indicate the patient get severe cervical cancer.
</P><B>D. PROTOTYPE</B><P align="justify">
With those concept as explained before, we’re trying to make a prototype for early detection to cervical cancer that has some common with test pack. These are our model for the prototype:
</P><P align="justify">
We test the sample from menstrual blood that we hypothesized if she got cervical cancer, the menstrual blood will go through the endometrium and then to cervix which is can bring some of the infected cells. Then the give a little bit of menstrual blood to the applicator. Inside the test pack we’ll use nitrocellulose membrane to get the result. Next steps is the binding from TALE 1 that already bind with the linker and reporter to the HPV DNA that can be found in menstrual blood. We can’t be sure if the TALE 1 and HPV DNA is binding without the TALE 2, so the sample will be catch by TALE 2 that stands for positive line. If that two TALEs can’t bind to the DNA it means negative result. Meanwhile we make control line too to compare the positive line and the control line. 
</P><B>E. SOFTWARE</B><P align="justify">
This software help user to early detect her cervical cancer with capture image from c3+ kit. This software also inform all about cervical cancer, especially preventing, screening, and therapy. 
</P><B>REFERENCES:</B><UL><LI>Alieva, N.et al. 2008. Diversity and Evolution of Coral Fluorescent Proteins. (http://plosone.org)</LI><LI>Kay S, Hahn S, Marois E, Wieduwild R, Bonas U. 2009. Detailed analysis of the DNA recognition motifs of the Xanthomonas type III effectors AvrBs3 and AvrBs3Deltarep16. Plant J. 59:859–871.</LI><LI>Boch J, Bonas U. 2010. Xanthomonas AvrBs3 family-type III effectors: discovery and function. Annu Rev Phytopathol; 48:419–436.</LI><LI>Litowski, Jennifer R. And Robert S. 2002. Hodges. Stability, and specificity propensity on protein folding, -helical α hydrophobicity and -helical coiled-coils: effects of α designing eterodimeric two-stranded. J. Biol. Chem, 277:37272-37279.</LI><LI>Sanjana et al. A Transcription Activator-Like Effector (TALE) Toolbox Genome Engineering. Nat Protoc.; 7(1): 171-192.</LI></UL></DIV><H2 class="font-thin"><FONT color="#fff">Screening</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="therapy" style="display:none;width:700px;"><H2><CENTER>“SCT - Screening Cancer Therapy”</CENTER></H2><P align="justify">
SCT (screening Cancer Therapy) is a kit which contains of hela cells as cervical cancer cells and it was designed to screening the plants that have compund potentially for cervical cancer therapy. The kit designed with plasmid that contain with HPV 16 or HPV 18 promoter. Each promoter to be accompained by two types of fluorescence reporter such as green (GFP) and red (RFP). The reporter compound obtained from biobrick. GFP obtained from Bba_E0240 (Plate 4, well 11N), then RFP obtained from Bba_K516132 (Plate 1, well 11A). Plasmid also designed with another compund such as RBS and double terminator. GFP act as a marker for the detection viruses whether plasmids have been inserted in hela cells and RFP as a marker for checking the ability of herbal compound used in the treatment of cervical cancer. 
Herbs that potentially as a therapy of cervical cancer in the kit are marked by the absence of green or red colors, while the negative results are marked by the emergence of green or red in the kit. The construct in the kit design is herbs can inhibit HPV promoter so that promoters cannot work anymore. The kit is expected to provide convenience to the researchers in the field of cervical cancer and cervical cancer effects can be mitigated with natural compound that not a lot of side effects. 
</P></DIV><H2 class="font-thin"><FONT color="#fff">Therapy</FONT></H2></DIV></DIV></DIV></DIV><DIV class="slide story" id="slide-3" data-slide="3"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-semibold">NOTEBOOK</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">Description</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><H2 class="font-thin"><FONT color="#fff">Timeline</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="lab-journal" style="display:none;width:700px;"><H2><CENTER>“LAB JOURNAL”</CENTER></H2><P align="justify"><UL type="circle"><LI><B>PREVENTING</B></LI>
A. MAY <UL><LI>Week 3</LI> 
Discussion. Discussion about biobrick and what we can do by used this biobrick.
<LI>Week 4</LI>
Discussion. Discussion about project - fix about cervical cancer.
</UL><B>B. JUNE</B><UL><LI>Week 1</LI>
Discussion. Discussion about project - fix that we will do research in “preventing, screening and therapy” for cervical cancer. And our sub-team focus in preventing cervical cancer.
<LI>Week 2</LI>
Bioinformatic. Study bioinformatic and protocol for each project.
<LI>Week 3</LI>
Bioinformatic. Study bioinformatic for each project and discussion about human practice.
<LI>Week 4</LI>
Laboratory. Study in laboratory - learn how to work with gene (competent cell, transformation, etc.)
</UL><B>C. AUGUST</B><UL><LI>Week 1</LI>
Laboratory. Having traine about working in molecular biology, especially in using lab tools (PCR, Electrophoresis, etc.)
<LI>Week 2</LI>
Discussion. Discussion about each project development, human practice, and future plan 
<LI>Week 3</LI>
Preparation. Competent cell preparation
<LI>Week 4</LI>
Preparation. Competent cell preparation
</UL><B>D. September</B><UL><LI>Week 1</LI>
Discussion &amp; Laboratory. Take hela cells from Yogyakarta and did transformation. Cultured hela cell and discuss with Prof Nellen from Kassel university about funding. Prof Nellen also works at molecular biology for school and the public in Kassel University, it is a student club. He told us how to make an event for funding our project, beside he suggess us to have a good relation and collaboration with other university.
<LI>Week 2</LI>
Discussion. Discussion about protocol and wet lab work timeline. We face a problem to choose what is the best promotor for the siRNA. We confused in using CAMP or U6/H1 promotor. Then we discuss it with our beloved supervisor, Mr. Widodo, he give us other promotor named pUR 6 and ask to check it in NetCutter. 
<LI>Week 3</LI>
Laboratory.  Annealing oligonucleotide of siRNA
<LI>Week 4</LI>
Laboratory. Transformation to DH5α
</UL><B>E. OCTOBER</B><UL><LI>Week 1</LI>
Laboratory. Cloning and PCR oligonucleotide.
<LI>Week 2</LI>
Laboratory. Electrophoresis. 
</UL><LI><B>THERAPY</B></LI><B>A. AUGUST</B><UL><LI>Week 3 </LI>
Lab Preparation. We learn more about protocol that we need. Restriction and Ligation protocol from iGEM protocol and adjust it.
<LI>Week 4 </LI>
Growing HeLa cells. Thawing HeLa cells from freezing cells. Growing into two flask and incubate in 37ᵒC.
</UL><B>B. SEPTEMBER</B><UL><LI>Week 1</LI>
Splitting HeLa cells. The confluence HeLa cells (80 – 90%) splitting into another flask then incubate in 37ᵒC.
<LI>Week 2</LI>
Cell observation. The splitting HeLa cells observed to control the growth.
<LI>Week 3</LI>
Order Synthetic Gene. Order P97 and P105 promoter sequence to IDT (Integrated DNA technology).
<LI>Week 4</LI>
Cloning. Insert the P97 and P105 promoter sequence into pJET vector (helped by ITB iGEM Group) then cloned.
</UL><B>C. OCTOBER</B><UL><LI>Week 1</LI>
Cloning. Restriction the P97 and P105 promoter sequence from pJET vector using EcoRI and Xbal. Beside that, also digest the pSB1C3 backbone using same restriction enzyme. Then, electrophoresis the digested sequence and ligase with pSB1C3 plasmid backbone.
<LI>Week 2</LI>
Cloning. Electrophoresis again and then using gel extraction to dissolve the gel. After that, ligase the P97 or P105 promoter sequence into pSB1C3 plasmid backbone.
<LI>Week 3</LI>
Submit part to iGEM
</UL><LI><B>SCREENING</B></LI></UL></P><P align="justify"><B>April – May</B>
We found some literatures for our Screening Project and combine some projects from previous team. We decided to use Golden Gate Assembly from <A href="https://2012.igem.org/Team:Freiburg">Freiburg team 2012</A> to synthesis our TALE. Our screening kit is related to <A href="https://2013.igem.org/Team:Calgary">Calgary team 2013</A> model but we use AmylCP reporter from <A href="https://2011.igem.org/Team:Uppsala-Sweden">UPPSALA team 2011</A>. <B>June-July</B>
The TALE and AmylCP will be linked by E coil and K coil that created by  <A href="https://2013.igem.org/Team:Calgary">Calgary team 2013</A>, so we ordered them from IGEM. The bio bricks stocks is limited so we have to clone them, we transformed them to E. coli competent cell. In the beginning we use local strain E. coli but the cells has low efficiency.</P><CENTER>
Figure 1. <A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A> not inserted to local E. coli competent cell bacteria.</CENTER><P align="center"><B>August</B>
We made DH5-α competent cell bacteria and test the efficiency with tansform our parts. First trial we succeded to transform our TALE direpeat parts from kit plate 2. We run the isolated plasmid into agarose gel before and after PCR.</P><CENTER>
Figure 2. The bio bricks were transformed into the new competent cell (DH5-α). Size of the parts are about 2200bp shown in the yellow box. 
Figure 3. PCR products after we inserted the parts into DH5-α competent cell. The control band are shown in the K1 and K2 line that related to there is no problem with PCR technique. Only 4I part that successfully amplified.</CENTER><P align="justify"><B>September - October</B>
These months we tried to synthesize our TALEs after clone the parts, but still there are 2 parts that not amplified yet (6A and 12E). We don’t have BsmBI restriction enzyme to Golden Gate Assembly but we found that EcoRI has the same restriction site as BsmBI. We substituted the BsmBI with EcoRI and  the parts successfully inserted into pSB1C3 (3200 bp).</P><CENTER>
Figure 4. There are two TALES we tried to synthesize, TALE 1 and TALE 2. Every TALE has triplo but only T1-2, T2-1, and T2-2 succesfully assembled.</CENTER><P align="justify">
We collected as many as possible references for the application content. Our application is android mobile based and we started to encode it. The application consist of preventing and therapy suggestion also screening method information. Special feature from our application is snap tool that can detect the kit result.
</P></DIV><H2 class="font-thin"><FONT color="#fff">Lab Journal</FONT></H2><H4 class="font-thin"><FONT color="#fff">Detail.</FONT></H4></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="protocols" style="display:none;width:700px;"><H2><CENTER>“PROTOCOLS”</CENTER></H2><P align="justify"><B>P97 and P105 digestion. </B><UL type="disc"><LI>Add 15 ul of DNA (p97 and p105).</LI><LI>Add dH20 for a total volume of 3,5 ul.</LI><LI>Add 2,5ul of buffer O </LI><LI>Add 0.5ul of EcoRI </LI><LI>Add 0.5ul of  Xbal</LI><LI>Add 0.5ul of Pstl</LI><LI>There should be a total volume of 25 ul.</LI><LI>Mix well and spin down.</LI><LI>Incubate the restriction digest at 37˚C for 30minutes, and then 80˚C for 20minutes.</LI></UL><B>SiRNADigestion</B>
Make the composisition of the mixture, include:
<UL type="circle"><LI>SiRNA 2 μl (100 nmol)</LI><LI>Add 10x buffer 2 μl </LI><LI>Add EcoR1 1 μl (20 u/ μl)</LI><LI>Add Xbal 1 μl (20 u/ μl)</LI><LI>Add ddH2O 14 μl and mix gentle</LI></UL><B>PSB1C3Digestion</B>
Make the composisition of the mixture, include:
<UL type="circle"><LI>ddH2O 10 μl</LI><LI>Add plasmid 4 μl </LI><LI>Add 10x buffer 2 μl </LI><LI>Add EcoR1 2 μl</LI><LI>Add Xbal 2 μl and then mix gentle</LI></UL><B>Ligation of sample and PSB1C3</B>
Make the composisition of the mixture, include:
<UL type="circle"><LI>10x KAPA buffer 2 μl</LI><LI>Add (5 u/ μl) 1 μl </LI><LI>Add Psb1c3 (25 ng/ μl) 4 μl </LI><LI>Add sample 4 μl</LI><LI>Add ddH2O and then mix gentle</LI><LI>Ligate at 16˚C for 30 minutes, then heat kill 80˚C for 20 minutes</LI><LI>Transform with 2 ul of product</LI></UL><B>Transformation</B><UL type="circle"><LI>Thaw competent cells on ice.</LI><LI>Place 25 µL of cells in a pre-chilled 2ml tube </LI><LI>Place 25µL in 2ml tube and label it for another. Use it for control. </LI><LI>Add 150 µL of resuspended DNA to 2ml tube. </LI><LI>Pipet up and down gently for a few times. keep the competent cells still on ice.</LI><LI>Add 1 µL of the RFP Control/li&gt;
</LI><LI>Incubate the cells on ice for30 minutes.</LI><LI>Heat shock at 42ºC for 60 seconds.</LI><LI>Return to ice for 5 minutes. </LI><LI>Add 250 μl of SOC media to each transformation </LI><LI>Incubate the cells at 37ºC for 2 hours while the tubes are rotating or shaking. Note:2 hour recovery time helps in transformation efficiency </LI><LI>Do not forget to label two petri dishes with selective LB agar and kanamycin antibiotic with antibiotic resistance, part number and plasmid backbone. </LI><LI>Place 20 µl and 200 µl of the transformation to dishes. Spread it. This step help ensure that you will be able to pick out a single colony.</LI><LI>Place 20 µl and 200 µl of the transformation to the dishes. Spread it.</LI><LI>Incubate the plates at 37ºC for 12-14 hours </LI><LI>Count the colonies on the 20 μl control plate</LI><LI>Calculated the competent cell.</LI></UL><B>Polymerase Chain Reaction</B><UL><LI>Make the composisition of the mixture, include:</LI>
Add 23 μl of ddH20 
Add 0.5  μl of Primer Forward 
Add 0.5 μl of Primer Reverse 
Add 1 μl of DNA samples. There should be a volume of 25 ul <LI>Running on PCR machinewith 35x PCR cycle, include :</LI>
Hot start	: 5 minutes on 94oC
Denaturation	: 30 second on 94oC
Annealing	: 30 second on 50oC
Extention 	: 60 second on 72oC
Post extention : 10 minutes on 72oC</UL><B>Annealing Oligonucleotide</B><UL><LI>siRNA forward (10 μl) and siRNA reverse (10 μl) mix gentle</LI><LI>Add TE buffer 25 μl, ddH2O 25 μl</LI><LI>Heat to 95oC for 2 minutes</LI><LI>Ramp cool to 25oC over period of 45 minutes</LI></UL><B>Agarose Gel Electrophoresis</B><UL><LI>Dissolve the 2% agar, 0,3 gr agarose on 15 ml TBE + 1 μl EtBr.</LI><LI>Heat the mixture until the agar is dissolve.</LI><LI>Cool the mixture and the comb on the plate.</LI><LI>Pour the cooled agar in to the plate.</LI><LI>When the agar has solidified, removed the come carefully.</LI><LI>Places the gel in to electrophoresis chamber.</LI><LI>Pour 1X TBE over gel so that gel is covered by a 3-5mm buffer.</LI><LI>Load samples in the wells in the gels : 1 μl DNA and 3 μl loading dye.</LI><LI>Load DNA ladder 2 μl to the well.</LI><LI>Place the lid on the chamber and connect the electrode leads to the power supply.</LI><LI>Turn on the power supply and adjust the voltage 50-100 volts.</LI><LI>Run the gel for 5-10 minutes.</LI><LI>Load samples into well.</LI><LI>Hook electrodes to gel apparatus.</LI><LI>Run the apparatus at 100V for 20 minutes.</LI><LI>Visualize the gel and record the results by UV-transilluminator</LI></UL><B>Isolation Plasmid used Kit-free Alkaline Lysis Plasmid Miniprep</B><UL><LI>Prepare the following solutions:</LI><PRE>a. Solution I (Resuspension buffer) 
     i. 25 mM Tris-HCl (pH 8) 
     ii. 50 mM glucose 
     iii. 10 mM EDTA
b. Solution II (Denaturing Solution)  
     i. 0.2 N NaOH  
     ii. 1.0% SDS  
c. Solution III (Renaturing Solution: Potassium Acetate)  
     i. 120 mL 5M Potassium acetate  
     ii. 23 mL glacial acetic acid  
     iii. 57 mL of de-ionized water 
Store Solution I at 4°C  
Store Solution II at room temperature  
Store Solution III at 4°C</PRE><LI>Grow 2mL overnight cultures from single colonies of bacteria containing your plasmid of interest.</LI><LI> Add 1.5mL of the stock culture to a 1.75mL microfuge tube.</LI><LI>Centrifuge in microfuge tube at 10,000g for 30sec.</LI><LI>Pour off the supernatant, being careful not to disturb the bacterial pellet.</LI><LI>Resuspend the pellet in 100ul of cold Solution I.</LI><LI>Vortex the solution for 2 minutes or until all bacteria are fully resuspended.</LI><LI>Add 200μl of Solution II and invert the tube carefully 5 times to mix the contents. The contents will become clear and thicker as the proteins and DNA are denatured.</LI><LI>Incubate solution on ice for 5 minutes.</LI><LI>Add 150μl of cold Solution III to each tube.</LI><LI>Mix by inverting several times. A white precipitate will be formed which contains the bacterial proteins and genomic DNA.</LI><LI>Incubate tube on ice for 5 minutes.</LI><LI>Centrifuge the tube for 5 minutes at 12,000g</LI><LI>Add either 700μL of cold 100% ethanol or 350uL room temperature isopropanol to the solution to precipitate the plasmid DNA.</LI><LI>Pour out the supernatant</LI><LI>Air dry the pellet (can be done by inverting the tube at an angle over kimwipe) for 20-30 minutes.</LI><LI>Resuspend pellet with 25-50μl of TE.</LI></UL><B>GATE kit For Easy TAL Effector Assembly</B><UL><LI>Choose your DNA target sequence. </LI></UL>
This sequence should be 14 bp long and they start and end with a thymine base.
1.	Find the right di-repeats: 
2.	Remove the flanking thymine bases from your target sequence, split it into six pairs of nucleotides and number the repeats from one to six.To get the needed concentration of 60ng/μl please resuspend the DNA in 10μl H2O</P><CENTER>TGCAAGTTCGAGCT
GC-AA-GT-TC-GA-GC
1. GC 2. AA 3. GT 4. TC 5. GA 6.GC</CENTER><P align="justify">
3.	Insert the di-repeats into the expression vector. 
Mix the following components in a PCR tube and  prepare a negative control lacking the di-repeats.</P><CENTER><TABLE border="1"><TBODY><TR><TH>Component</TH><TH>Amount (μl)</TH></TR><TR><TD>6 di-repeat plasmids (60ng)
Expression vector (170ng)
(pTALEN/pTAL-TF/pTAL-KRAB)
T4 Ligase Buffer (10x)
T4 Ligase (60 U)
BsmBI (15 U)
dH2O</TD><TD>
1 each
1
2
1
1,5
8,5</TD></TR><TR><TD>Total</TD><TD>20</TD></TR></TBODY></TABLE><B>PCR Cycle</B><TABLE border="1"><TBODY><TR><TH>Cycle</TH><TH>Temperature and time</TH></TR><TR><TD>1-13</TD><TD>37 °C, 5 minutes </TD></TR><TR><TD>20 °C, 5 minutes </TD></TR><TR><TD>14</TD><TD>50 °C, 10 minutes </TD></TR><TR><TD>15</TD><TD>80 °C, 10 minutes </TD></TR></TBODY></TABLE></CENTER>
4.	Transform the products into a competent E. coli, such as DH10B and Top10. Note that selection of the right clones will not be successful with ccdB-resistant E.coli strains. 
Add 5 μl of the ligation product to 50 μl of competent bacteria, incubate on ice for 30 min, perform heat shock at 42 °C for 1 minute, then incubate on ice for 5 minutes. Add 500 μl of pre-warmed LB medium, incubate at 37 °C for 1 h on a shaking thermo block, plate on an agar plate containing chlorampenicol and incubate at 37°C overnight.
5.	Put a single colony on a new LB agar plate with chlorampenicol and Incubate for 16 hours.
6.	If the repeat array already in the expression plasmid, then transfecting into cells.
7.	Perform electrophoresis in order to check the successfull of TALE assembly.</DIV><H2 class="font-thin"><FONT color="#fff">Protocols</FONT></H2><H4 class="font-thin"><FONT color="#fff">Detail.</FONT></H4></DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV></DIV></DIV><DIV class="slide story" id="slide-4" data-slide="4"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-semibold">HUMAN PRACTICE</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">Teks description</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="workshop" style="display:none;width:700px;"><H2><CENTER>“Early Step for Synthetic Biology Development in Indonesia”</CENTER></H2><P align="justify">
iGEM is not only a competition which provides a stage for us to perform our work done at the laboratory but also an effort to popularize synthetic biology to the public. Synthetic biology is not popular yet in Indonesia and there are many people who do not know about this field. Therefore we decided to hold a workshop entitled “Potential and applications of synthetic biology in Indonesia” which was held on September, 26th - 27th 2014.
ITB iGEM team and UI iGEM team also took part in our activities and make it more fun. The participants not just come from undergraduate student but also from postgraduate student from various university in Indonesia. We got that audience are so enthusiastic through our lecture. The lecture has five parts, including introduction of synthetic biology and its potential for immunology therapy, potential of biology synthetic for health and medicine field, introduction about iGEM, presentation material from ITB iGEM team, project of syntetic biology community of UGM, and presentation material from iGEM-UB.
Synthetic biology is a newly-developing subject. We hope through our event there will be a better understanding about what synthetic biology is, its potential and encouraged people to develop synthetic biology in Indonesia. Beside, we held a workshop about bioinformatics. Why we choose bioinformatics? Because it is needed for molecule prediction before we do some molecular work at laboratory. It also useful and be a basic for developing synthetic biology. Thanks to Synbio community from Gadjah Mada University and student from University State of Malang who took part on our workshop.
Well, hopefully this small step can be the beginning of a big step in the world of synthetic biology in Indonesia. Such as a cell that will develop into tissues and has a function. Go synthetic biology!
</P></DIV><H2 class="font-thin"><FONT color="#fff">Workshop</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="education" style="display:none;width:700px;"><H2><CENTER>“Say Hi to Young Scientist Wannabe!”</CENTER></H2><P align="justify">
Synthetic Biology (Synbio) can not stop on our generation, but must be passed on the generations below us. Therefore, we hold socialization Synbio to students high school in Malang, East Java, Indonesia. Our focus is opening their minds to the real world’s problem and things we can do through science and engineering. We presented it by video animation, power point and brochure. 
We hold imaginationclasses, in which students were divided into groups and given the opportunity toimagine bio-machine that can solve the problems in the world. Wow! The results were shocking! They have wild imagination in implantingsynbio concept clearly and precisely. Well, although some ideas seem impossible, but hey, there is nothing impossible in science, isn’t it?</P></DIV><H2 class="font-thin"><FONT color="#fff">Education</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="society" style="display:none;width:700px;"><H2><CENTER>“From Lab For Society”</CENTER></H2><P align="justify">
Doing research and explore a new thing which will be useful for society is really important, but, the other thing that also important is the socialization to society. So, we bring synthetic biology to solve society’s problem.
We are visiting Indonesia Child Cancer Care Foundation in Surabaya, East Java, Indonesia. The foundation is established on the basis of concern for the rise of cancer that occurs in children. By the time we tell the management of the foundation that our focus was cervical cancer, surprisingly the foundation said that there are some children infected with cervical cancer, myom and a cyst ( to fyl further kept secret for the sake of patients ). The foundation also suggested students to socialize more often to society from all age  so cancer prevention can be done.
According to Ms. Novi, as Volunteer Coordinator, many cancer patients due to lack of knowledge about cancer in society. It causes people less careful and many new cancer cases have been detected at late stage. Therefore she really supports our research on early detection of cervical cancer.
We made several visits to the foundation. There, we aided by Gencorp organization from Surabaya in giving encouragement to cancer patients. We also discussed about things that we can do for cancer patients. One thing that we will never forget is that we were invited to visit hospitals and meet cancer patient directly. We are so far, only serves in the scale of laboratory in developing the treatment of cancer, a direct view of how the treatment of cancer research is needed. It gives us spirit in resolving this research. Be closer to patients aforetime and feel how this research has a lot of meaning.
Besides being active in activities at the foundation, we also give the material to friends in the traveling community GAMANANTA, which is a large community of traveler Indonesia, about how to maintain the health of vital organs during travel. This was greeted with great enthusiasm, we even double enthusiastic. We believe that, by sharing, the knowledge that we have will provide a double benefit to the community. Go Synthetic Biology for Indonesia!
</P></DIV><H2 class="font-thin"><FONT color="#fff">Society</FONT></H2></DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV></DIV></DIV><DIV class="slide story" id="slide-5" data-slide="5"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-semibold">COLLABORATIONS</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">Description</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="collaboration" style="display:none;width:700px;"><H2><CENTER>“Let’s Meet Up!”</CENTER></H2><P align="justify">UB iGEM Team had collaboration with UI iGEM Team and ITB iGEM team in holding Synthetic Biology for Indonesia Community. The aim of this community is to form a synthetic biology  association in Indonesia.Programs that have been done previously is an online course in facebook group named “Synthetic Biology for Indonesia” and already got approximately 121 members from all over Indonesia.
For developing this community, we conducted a forum for us to meet up with others synbio community all over Indonesia. This forum (meet up) was initiated by UB iGEM Team, ITB iGEM Team and UI iGEM Team. For this year, 2014, the meeting was held in Brawijaya University, Malang, East Java, Indonesia on September, 27th 2014. The event combined with seminar and workshop in synthetic biology. We also invited synthetic biology community of Gadjah Mada University, synthetic biology community of Sumbawa Technology University, Biology department of Airlangga University and Biology department of State of Malang University.
We tend to do annual meet up for discussing anything about synthetic biology, start from research until social action. In the other hand, each of the university member will link together to be strengthen community and cooperate with government, practitioners and scientist in Indonesia.</P></DIV><H2 class="font-thin"><FONT color="#fff">SYNBIO FOR INDONESIA</FONT></H2></DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV></DIV></DIV><DIV class="slide story" id="slide-6" data-slide="6"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-semibold">PARTS</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">Description</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="sub-parts" style="display:none;width:700px;"><H2><CENTER>“Submited Parts”</CENTER></H2><UL type="circle"><LI>Therapy</LI><P align="justify"><B>E0240_P105</B>
We upgraded the standard biobrick backbone pSB1C3 with added by promotor sequences of  p105 and GFP (E0240). This pSB1C3 has contained  chloramphenicol resistance. Our major achievements is to confirm that herbal compound can  react with promotor of HPV 16 and 18 in HeLa cells.  GFP give green colour to confirm whether herbal compound can attach and stop the HeLa cell activity transformed with p105 promotor sequence
</P><P align="justify"><B>E0240_P97</B>
Standard biobrick backbone pSB1C3 containing chloramphenicol resistance.  The backbone added  by promotor sequences of  p97 and GFP (E0240).  GFP act as a marker that give green colour to confirm  whether herbal compound can attach  and stop the HeLa cell activity transformed with p97 promotor sequence.
</P><B>Preventing and Therapy Team Safety</B><CENTER><TABLE border="1"><TBODY><TR><TH>Part Number</TH><TH>DNA Source</TH><TH>Species Origin</TH><TH>Species Risk Group</TH><TH>Part Function</TH></TR><TR><TD>Bba_K747096</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>CMV Promoter</TD></TR><TR><TD>Bba_E0240</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>GFP Generator</TD></TR><TR><TD>Bba_K516132</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Constitutive promoter with mRFP, RBS B0032</TD></TR><TR><TD>pTALEN</TD><TD>From BIOSS in University of Freiburg, Germany</TD><TD>Xanthomonas spp.</TD><TD>2</TD><TD>Vector</TD></TR><TR><TD>pTAL-TF</TD><TD>From BIOSS in University of Freiburg, Germany</TD><TD>Xanthomonas spp.</TD><TD>2</TD><TD>Vector</TD></TR><TR><TD>p97 promoter</TD><TD>Synthesized by IDT</TD><TD>HPV type 16</TD><TD>2</TD><TD>Cell Line</TD></TR><TR><TD>p105 promoter</TD><TD>Synthesized by IDT</TD><TD>HPV type 16</TD><TD>2</TD><TD>Cell Line</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>Part Number</TH><TH>DNA Source</TH><TH>Species Origin</TH><TH>Species Risk Group</TH><TH>Part Function</TH></TR><TR><TD>Bba_K747012</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747016</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747042</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747048</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747064</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747091</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747012</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747026</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747036</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747052</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747076</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747095</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Protein Domain </TD></TR><TR><TD>Bba_K747101</TD><TD>Part registry</TD><TD>E. coli DH5α</TD><TD>1</TD><TD>Plasmid </TD></TR></TBODY></TABLE></CENTER><H2 class="font-thin"><FONT color="#fff">Submitted Parts</FONT></H2><DIV class="col-12 col-lg-4 col-sm-6"><DIV id="fav-parts" style="display:none;width:700px;"><H2><CENTER>“”</CENTER></H2></DIV><H2 class="font-thin"><FONT color="#fff">Favorite Parts</FONT></H2></DIV><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="slide story" id="slide-8" data-slide="8"><DIV class="row title-row"><DIV class="col-12 font-semibold">ABOUT US</DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">Description</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV class="row content-row"><DIV class="col-12 col-lg-4 col-sm-6"><H2 class="font-thin"><FONT color="#fff">Members</FONT></H2></DIV><DIV class="col-12 col-lg-4 col-sm-6"><H2 class="font-thin"><FONT color="#fff">Sponsors</FONT></H2></DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV></DIV><DIV class="slide story" id="slide-9" data-slide="9"><DIV class="container"><DIV class="row title-row"><DIV class="col-12 font-light">Leave us a <SPAN class="font-semibold">message</SPAN></DIV></DIV><DIV class="row line-row"><DIV class="hr"> </DIV></DIV><DIV class="row subtitle-row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-10 font-light">You can find us literally anywhere, just push a button and we’re there</DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV><DIV id="contact-row-4" class="row"><DIV class="col-sm-1 hidden-sm"> </DIV><DIV class="col-12 col-sm-2 with-hover-text"><SPAN class="hover-text font-light ">in/igemub</SPAN></DIV><DIV class="col-12 col-sm-2 with-hover-text"><SPAN class="hover-text font-light ">igemub@gmail.com</SPAN></DIV><DIV class="col-12 col-sm-2 with-hover-text"><SPAN class="hover-text font-light ">Malang, Indonesiazip code 65145</SPAN></DIV><DIV class="col-12 col-sm-2 with-hover-text"><SPAN class="hover-text font-light ">facebook/igembrawijaya</SPAN></DIV><DIV class="col-12 col-sm-2 with-hover-text"><SPAN class="hover-text font-light ">@igem_ub</SPAN></DIV><DIV class="col-sm-1 hidden-sm"> </DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UB_Indonesia/backup">http://2014.igem.org/Team:UB_Indonesia/backup</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UB_Indonesia/backup" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UB_Indonesia/backup" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UB_Indonesia/backup&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UB_Indonesia/backup&amp;oldid=284943" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>