"
Page
Discussion
View source
History
teams
Log in
 
Team:Northwestern/Project/Background
From 2011.igem.org
PROJECT
Introduction
Background
Project Description
Modelling
RESULTS
Summary
Data
Biobricks
CONSIDERATIONS
Application
Human Practices
Safety
ABOUT US
Students
Advisers
Photo Gallery
NOTEBOOK
Brainstorming
Protocols
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
ATTRIBUTIONS
University
Sponsors
Project
  Background
Pseudomonas aeruginosa 101
Pseudomonas aeruginosa colonizes immunocompromised patients, often affecting patients in intensive care units [1] . P. aeruginosa is responsible for 12% of hospital-acquired urinary tract infections, 10% of bloodstream infections, and 8% of surgical wound infections [1] .  Once a patient is infected with this pathogen, treatment is difficult because this microbe is often resistant to commonly-used antimicrobial agents [1] .  Quick, cheap, and frequent screenings for P. aeruginosa in hospital rooms, particularly after rooms are cleaned between patients, could help prevent the spread of this hospital-borne infection. However, there is currently no efficient way to detect the presence of P. aeruginosa. We aim to develop a quick, cheap, and sensitive detector of P. aeruginosa so that contaminated rooms and equipment can be properly cleaned before they come in contact with a patient.
Pseudomonas aeruginosa is armed with an arsenal of cell density detection systems that directly govern gene expression [2] . The importance of the quorum sensing system is illustrated by the number of genes regulated by this system, which controls virulence, protein secretion, toxins and other factors that mediate immunomodulatory activity [3] . The resulting complexity of P. aeruginosa demonstrates its potential to survive in wildly disparate environments [4] .
The process of quorum sensing is a complex summation of individual systems involving multiple levels of transcriptional regulation and competitive inhibition [2] . Over 50 P. aeruginosa genes directly participate in quorum systems and, conservatively speaking, 616 more show a statistically significant difference in expression in the presence of autoinducers [2] . Two separate quorum sensing systems, Las and Rhl, have been characterized [2] . These two systems bear a striking resemblance to the lux gene system discovered initially in Vibrio fischeri tying quorum sensing capabilities to bioluminescence [5] . An overview of the quorum sensing signaling model in P. aeruginosa is illustrated in Figure 1.
Figure 1: Cell signaling model in P. aeruginosa. [1]
Quorum Sensing
As in many gram-negative pathogens, the cell-to-cell signaling of Pseudomonas aeruginosa is facilitated by small homoserine lactone-based molecules, called autoinducers [5] .Under non-quorum conditions, the pathogenic cells produce the autoinducers at a basal rate. Over a period of time, the concentration of the autoinducers increases significantly as demonstrated in Figure 2. As the concentration of these molecules increases past a threshold, they bind to specific protein transcriptional factors [6] .There are two crucial mechanisms governing the cell signaling cascades that take place within P. aeruginosa, namely, the Las and the Rhl signaling pathways [6] .
Figure 2: Accumulation of autoinducers released over time by P. aeruginosa
Type 1 Quorum Sensing
The las signaling system defines Type 1 quorum. The las system is affected by the autoinducer 3-oxo-C12-HSL (N-[3-oxododecanoyl]-L-homoserine lactone) or PAI-1. Arguably, the two most important components of the las signaling system are the lasR gene which produces the transcriptional regulator protein LasR, and the lasI gene which facilitates the production of the autoinducer PAI-1 [7] . When the intracellular concentration of PAI-1 rises, it binds to its cognate R protein, LasR and forms a LasR/PAI-1 dimer [7] . The LasR/PAI-1 dimer initiates Type 1 quorum signaling cascade acting as the principle transcription regulating protein complex [8] . Additionally, the LasR/PAI-1 protein/dimer complex can transcriptionally regulate the LasR protein production [8] . Moreover, the LasR/PAI-1 dimer facilitates the expression of virulence genes such as LasB along with virulence factors such as LasA protease and endotoxin A in addition to several other pathways [9] .
Type 2 Quorum Sensing
The Type 2 quorum induced cell signaling machinery is predominately defined by the rhl system [9, 10] . Transcriptional activation of the rhl gene is facilitated by GacA and the LasR/PAI-1 protein complex, thereby making the Type 2 a secondary cell signaling apparatus, dependent on the Type 1 quorum system [9] . The primary components of the rhl system are the rhll and the rhlR gene sequences [6] . The rhll gene is primarily responsible for the production of a homoserine lactone, C4-HSL (N-butyrylhomoserine lactone) or PAI-2 [6] . On the other hand, the rhlR gene is responsible for the production of the PAI-2’s cognate R-protein, RhlR [6] . Contrary to the las system, RhlR can be deactivated by the competitive binding of the autoinducer PAI-1 [12] . Therefore, Type 2 cell signaling machinery is controlled at two levels - transcriptionally and post-transcriptionally - by the Type 1 quorum sensing machinery [12] . However, the binding of PAI-2 to RhlR forms the RhlR/PAI-2 protein/dimer complex, which augments the production of PAI-2 via positive transcriptional regulation of the rhll gene [12] . Upon formation, the RhlR/PAI-2 protein/dimer complex is vital for the optimal activation of the LasB gene and the production of LasA protease, among other genes involved with protein secretion and toxins [3] . For the purposes of our project, only the rhlR gene and rhlP promoter for the Type 2 quorum sensing will be incorporated into our E. coli strain.  This circumvents the complex regulatory systems in P. aeruginosa, but allows us to clearly detect PAI-2.
[1] Van Delden C, Iglewski BH. Cell-to-Cell Signaling and Pseudomonas aeruginosa Infections. Emerging Infectious Diseases. 1998;Vol.4,No.4:551-560.
[2] Wagner VE, Bushnell D, Passador L, Brooks AI, Iglewski BH. Microarray analysis of Pseudomonas aeruginosa quorum-sensing regulons: Effects of growth phase and environment. J Bacteriol. 2003;185:2080–2095.
[3] Latifi A, Winson MK, Foglino M, Bycroft Bw, Stewart GS, Lazdunski A, et al. Multiple homologues of LuxR and LuxI control expression of virulence determinants and secondary metabolites through quorum sensing in Pseudomonas aeruginosa PAO1. Mol Microbiol 1995;17:333-43.
[4] Stover CK, Pham XQ, Erwin AL, Mizoguchi SD, Warrener P, et al. Complete genome sequence ofPseudomonas aeruginosa PA01, an opportunistic pathogen. Nature. 2000;406:959–964.
[5] Whitehead NA, Barnard AML, Slater H, Simpson NJL, Salmond GPC. Quorum-sensing in gram-negative bacteria. Fems Microbiol Rev. 2001;25:365–404. doi: 10.1111/j.1574-6976.2001.tb00583.x.
[6] Fuqua C, Winans SC, Greenberg EP. Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators [review]. Annu Rev Microbiol 1996;50:727-51.
[7] Passador L, Cook JM, Gambello MJ, Rust L, Iglewski BH. Expression of Pseudomonas aeruginosa virulence genes requires cell-to-cell communication. Science 1993;260:1127-30.
[8] Albus, A. M., E. C. Pesci, L. J. Runyen-Janecky, S. E. West, and B. H. Iglewski. 1997. Vfr controls quorum sensing in Pseudomonas aeruginosa. J. Bacteriol. 179:3928-3935.
[9] Toder DS, Ferrell SJ, Nezezon JL, Rust L, Iglewski BH (1994) lasA and lasB genes of Pseudomonas aeruginosa: analysis of transcription and gene product activity. Infect Immun 62: 1320–1327.
[10] Pearson JP, Passador L, Iglewski BH, Greenberg EP. A second N-acylhomoserine lactone signal produced by Pseudomonas aeruginosa. Proc Natl Acad Sci U S A 1995;92:1490-4.
[11] Ochsner UA, Koch AK, Fiechter A, Reiser J. Isolation and characterization of a regulatory gene affecting rhamnolipid biosurfactant synthesis in Pseudomonas aeruginosa. J Bacteriol 1994;176:2044-54.
[12] Pesci EC, Pearson JP, Seed PC, Iglewski BH. Regulation of Las and Rhl quorum sensing inPseudomonas aeruginosa. J Bacteriol 1997;179:3127-32.
Retrieved from " http://2011.igem.org/Team:Northwestern/Project/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
