"
Page
Discussion
View source
History
teams
Log in
 
Team:Northwestern/Notebook/Protocols/3A Assembly
From 2011.igem.org
PROJECT
Introduction
Background
Project Description
Modelling
RESULTS
Summary
Data
Biobricks
CONSIDERATIONS
Application
Human Practices
Safety
ABOUT US
Students
Advisers
Photo Gallery
NOTEBOOK
Brainstorming
Protocols
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
ATTRIBUTIONS
University
Sponsors
3A Assembly
Introduction
3A assembly (which stands for three antibiotic assembly) is a method for assembling two BioBrick parts and selecting for correct assemblies with antibiotics. 3A assembly is now the preferred assembly method at iGEM HQ and the Knight lab at MIT.
It differs from Standard Assembly in that it relies on three way ligation (between the two parts and the backbone vector) for assembly rather than two way ligation between a part and a part + vector.  It uses effective antibiotic selection to eliminate unwanted background colonies and eliminates the need for gel purification and colony PCR of the resulting colonies. In theory, about 97% of the colonies should be the desired assembly.
Motivation
3A assembly was designed such that gel purification of the digested parts is unnecessary.  Incorrect assemblies are selected against via either positive or negative selection.  The motivations for eliminating the gel purification step are as follows:
Gel purification is slow and not easily susceptible to automation and scaling.  As the Registry moves to larger scale assembly of parts, gel purification can be a limiting step in terms of how many assemblies can be done in parallel.
Gel purification tends to be inefficient especially on small pieces of DNA.  It can be difficult to isolate small pieces of DNA (< 200 bp) from a gel.  Thus, when assembling two small pieces of DNA, 3A assembly might prove advantageous.
Description
Construction Plasmid
Right Part
Left Part
The left part is in a plasmid with an antibiotic resistance cassette for antibiotic A.
The right part is in a plasmid with an antibiotic resistance cassette for antibiotic B.
A construction plasmid is selected with neither resistance A nor B. It has antibiotic resistance C.
The left part's plasmid is cut with EcoRI and SpeI.
The right part's plasmid is cut with XbaI and PstI.
The construction plasmid backbone is a linear piece of DNA. It has a few bases beyond the EcoRI and PstI restriction sites. It is cut with EcoRI and PstI.
All 3 restriction digests are heated to heat kill all of the restriction enzymes.
An equimolar quantity of all 3 restriction digest products are combined in a ligation reaction.
The desired result is the left part's SpeI overhang ligated with the right part's XbaI overhang resulting in a mixed site that cannot be cut with any of our enzymes.
The new composite part is ligated into the construction plasmid backbone at the EcoRI and PstI sites.
When the ligation result is transformed into cells and grown on plates with antibiotic C, only colonies with the correct construction(*) survive.
Protocol Variation
Our team did not order primers for the plasmid backbone when we used 3A assembly, and instead amplified circular backbones from the iGEM kits. This was problematic because during ligation, the DNA still contains pieces of the excised part from the backbone. This piece sometimes reinserted itself instead of giving the desired construct, meaning that some of the colonies that grew when the ligations were transformed had only the backbone. The excised insert from the backbone contains a gene that transcribes RFP, so after transformation, some of the colonies were red and some were clear. The clear colonies were what we wanted, whereas the red colonies were just the original backbone with the reinserted piece. We also did not PCR purify the restriction digests. These alterations still worked to give us the colonies that we needed but was somewhat inefficient, and we recommend that next year's team use the edited 3A protocol below, using PCR'd linearized backbones.
There were also some variations that we believed increased our construction efficiency, namely that we increased digestion time to 1.5 hours and ligation time to 1 hour. Ligation can also take place overnight at 16 degrees C. If the following protocol is not working as well as desired, consider implementing these changes.
Materials
Two parts in BioBricks format to be assembled.  The parts are in a plasmid or they can be generated via PCR.
A construction vector that has a different resistance marker from either of two parts.
These construction backbones are available from the Registry.
<partinfo>pSB1A3</partinfo>
<partinfo>pSB1C3</partinfo>
<partinfo>pSB1K3</partinfo>
<partinfo>pSB1T3</partinfo>
Procedure
Miniprep your two parts.
Digest your two parts and construction plasmid backbone destination vector with the following enzymes
Upstream part with EcoRI and SpeI
Upstream Part Plasmid
500 ng
EcoRI-HF
1 μL
SpeI
1 μL
10X NEBuffer 2
5 μL
100X BSA
0.5 μL
H2O
to 50 μL
Downstream part with XbaI and PstI
Downstream Part Plasmid
500 ng
XbaI
1 μL
PstI
1 μL
10X NEBuffer 2
5 μL
100X BSA
0.5 μL
H2O
to 50 μL
Destination plasmid backbone with EcoRI and PstI. Also digest the construction plasmid backbone with DpnI if possible to eliminate any plasmid remaining from the PCR.
Destination Plasmid Backbone
500 ng
EcoRI-HF
1 μL
SpeI
1 μL
DpnI
1 μL
10X NEBuffer 2
5 μL
100X BSA
0.5 μL
H2O
to 50 μL
Purify the restriction digest.
For parts < 200 bp in length, do a Knight:Micropure EZ and Microcon purification (or alternatively an Ethanol precipitation of small DNA fragments ).  All other parts and the construction vector can be purified using a QIAquick PCR purification kit or Qiagen Minelute PCR Purification Kit (elutes in a smaller volume giving greater concentration of the DNA).
Incubate all three restriction digest reactions at 37ºC for 10 minutes and then heat inactivate at 80ºC for 20 minutes.
Ligation
Upstream Part digestion
2 μL
Downstream Part digestion
2 μL
Destination Backbone digestion
2 μL
10X T4 DNA Ligase Buffer
2 μL
T4 DNA Ligase
0.5 μL
H2O
11 μL
Incubate at room temperature for 10 minutes and then heat inactivate at 80ºC for 20 minutes.
Transform 2 μL of the ligation product into 50 μL of competent E. coli cells. Select using the antibiotic corresponding to the destination plasmid backbone.
If the input parts are good, almost all colonies will be correct.
Miniprep clones that generated a band of the appropriate size and sequence.
Troubleshooting
In our experience, most problems with 3A Assembly come from the transformation process. Because 3A Assembly uses a triple ligation, the concentration of assembled DNA is about 10% of that produced by a double ligation process. Particularly, if your lab makes its own competent cells (and particularly chemically competent cells) it is important to test the competency. If you are having trouble obtaining colonies, we suggest that you try commercial competent cells. Be sure to follow the instructions carefully. Do not add more ligation product than specified and be careful when resuspending the competent cells.
The second most common problems are caused by defects in the DNA entering the ligation. We find ligation to be a very reliable process. However, the sticky ends of the cut parts can be destroyed by enzymes. Always ligate immediately after digestion (do not store digestion products in the freezer and then ligate later). Also, the original DNA may be incorrect or defective.
Another common mechanism for failure involves a piece of genomic DNA being cloned into the construction plasmid.  This result typically occurs when the assembly is somehow unfavorable to the cell.  This issue can be partially checked via the colony PCR step in the protocol though if the genomic DNA is of the same size as your assembly, then the single colony PCR screen is ineffective.  A careful purification of the construction plasmid can help to avoid this result as can phosphatase treatment of the linear construction plasmid .
Analysis
File:Ccdb Mixture A.gif
Since the transformed cells are plated on plates supplemented with the antibiotic corresponding to the construction plasmid, only cells containing the construction plasmid backbone should survive.
Since the construction plasmid itself has been reduced in quantity by PCR and digestion by DpnI, few colonies will contain the original construction plasmid.
Adapted from OpenWetWare
Retrieved from " http://2011.igem.org/Team:Northwestern/Notebook/Protocols/3A_Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
