"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Northwestern/Notebook/Protocols/Primer Design
From 2011.igem.org
PROJECT
Introduction
Background
Project Description
Modelling
RESULTS
Summary
Data
Biobricks
CONSIDERATIONS
Application
Human Practices
Safety
ABOUT US
Students
Advisers
Photo Gallery
NOTEBOOK
Brainstorming
Protocols
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
ATTRIBUTIONS
University
Sponsors
Primer Design
The following protocol is for creating a new biobricks part from a protein coding sequence.
Design a forward primer to your new BioBrick part comprised of the BioBrick protein coding prefix sequence GTT TCT TCG AAT TCG CGG CCG CTT CTA G followed by the first 20-30 nucleotides of the protein coding region, beginning with the ATG
Design a reverse primer to your new BioBrick part comprised of the last 20-30 or so nucleotides of the protein coding sequence, excluding the stop codon, followed by the BioBrick protein coding suffix sequence 5'-TAA TAA TAC TAG TAG CGG CCG CTG CAG GAA GAA AC-3'. Note that the BioBrick protein coding suffix sequence includes the double stop codon TAA TAA. (Note: In some cases, for example if you wish use your part directly for making protein fusions using BioScaffold parts [1] or make your part more versatile by giving other users the capacity to do so in the future, then you should only include one stop codon TAA or make a version of your part that has only one stop codon. In this case your sequence should be followed by the BioScaffold and BioBrick compatible protein suffix 5'-TAA TAC TAG TAG CGG CCG CTG CAG GAA GAA AC-3' instead of the BioBrick protein coding suffix sequnce. Since the scar region that is created by the composition of parts, also introduces a stop codon after protein parts, including only one stop codon within the frame of the part may be sufficient (especially if your part will be followed by a subsequent part.))
Be sure and follow the guidelines above for design of the primer sequence that is complementary to the template.
Then take the reverse complement of your reverse primer.
Once you have obtained your primers, you can follow the usual PCR protocol using a High Fidelity polymerase. Digest and ligate as usual.
Retrieved from " http://2011.igem.org/Team:Northwestern/Notebook/Protocols/Primer_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
