<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wisconsin-Madison_notebook_Mary"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wisconsin-Madison/notebook/Mary">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wisconsin-Madison/notebook/Mary&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/notebook/Mary&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/notebook/Mary&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wisconsin-Madison/notebook/Mary" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wisconsin-Madison/notebook/Mary</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav_content" style="font-size: 80%"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#6-13"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">6-13</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#6-14"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">6-14</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#6-15"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">6-15</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#6-16"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">6-16</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#6-17"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">6-17</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#6-21"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">6-21</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#6-22"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">6-22</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#6-23"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">6-23</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#6-24"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">6-24</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#6-25"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">6-25</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#6-28"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">6-28</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#6-29"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">6-29</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#6-30"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">6-30</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#7-1"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">7-1</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#7-2"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">7-2</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#7-3"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">7-3</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#7-5"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">7-5</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#7-6"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">7-6</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#7-7"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">7-7</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#7-8"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">7-8</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#7-26"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">7-26</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#7-28"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">7-28</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#8-6"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">8-6</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></DIV><DIV id="page_content"><PRE>5-24 to 5-29 Summary
</PRE><P>During this week it was mainly my goal to make good competent cells.  I went through the process twice. The first time, I got 12 aliquots of 40uL because I miss read the final OD measurement, which made the cells concentrated. The second time, I got 40 aliquots of 40 uL and the final OD measurement within the 0.08-0.12 range.
</P><P>To test the transformation efficiency, I used pBAD18 ligation product to transform for both series of competent cells (5-26 and 5-27). However, after poor growth results on the plates with the 5-27 culture, I questioned if the ligation product was correct to use.  To test for the transformation efficiency I was supposed to use an intact plasmid, such as miniprep product. So I re-did the efficiency test with 5-27 cells (5-26 cells were used) and used the pBAD34 miniprep product. The 10^-2 dilution plate to have 463 colonies, with the efficiency to be 2.74 x10^7 CFUs/ug
</P><P>Also made point mutation primers for pBAD18 for the PstI second cut site in bla operon. I also made primers for the bile-induction promoter found in <I>Lactobacillus acidophilus</I> NCFM to use in its neotype strain, which is currently being sent to Prof. Jim Steele.  
</P><PRE> 6-01 to 6-04 Summary
</PRE><P>During this week, the team goal is to clone the encapsulation parts separately and together in one plasmid.  Also, to test these parts in an acid test separately and together. 
</P><P>For research: Find out about how to make our project safer, how to incorporate the encryption project into the lactose project, probiotics background information, how to improve the encapsulation process from Imperial College London.
</P><PRE> 6-7 to 6-11 Summary
</PRE><P>During this week:
</P><UL><LI>The Lactobacillus neotype strain has not arrived yet in Steele's lab. Wes has some MRS broth we can use to culture the strains.
</LI><LI>Contacted Steele for lactobacillus plasmids and expression strain to express the plasmid iGEM constructs
</LI><LI>Received primers on 6-7 and I helped elute the primers. We followed the formula: [(Amount of nm) x 1000pm/1nm] / 60pm total dilution. This gave us how much RNase free water to use for elution. After elution then we made 3x 30uL aliquots of the primers, except for the two sequencing primers.
</LI></UL><UL><LI> 6-08: Miniprep 2 pellets of pBAD18BB, 2 pellets of pBAD33BB, 2 pellets of YgiV, and 2 pellets of RcsB. Used the new Miniprep kit (6-08) and documented concentrations from the Nanodrop. Got the MRS premix broth from Wes. 
</LI></UL><UL><LI> 6-09: The RcsB transformations Sarah and Hannah plated had about 10 colonies with the control plate with one colony. The YgiV transformation had no colonies on it. So Yue Wu and I re-did the ligation and transformation. We also ran a gel of the ligation Sarah did.
</LI></UL><P>Bile promoter research: 
</P><UL><LI>The  <I>Lactobacillus acidophilus</I> neotype arrived today and will pick up 6-10. Need to make plates and broth to make freezer stocks of the culture.
</LI><LI>Lactobacillus bile promoter primers?
</LI><LI>Found an article about a bile-inducible efflux transporter in <I>Bifidobacterium longum</I> that is  induced 23 to 40 fold induction. <A href="/File:Bile-Inducible_Efflux_Trans_in_B._longum_Gueimonde_2009.pdf" title="File:Bile-Inducible Efflux Trans in B. longum Gueimonde 2009.pdf">File:Bile-Inducible Efflux Trans in B. longum Gueimonde 2009.pdf</A></LI></UL><PRE> 6-14 to 6-18
</PRE><H4><SPAN class="mw-headline" id="6-13">6-13</SPAN></H4><UL><LI> Plated more acidophilus plates with and without parafilm. and tubes with and with out parafilm. One was in shaker and the other was in incubator.
</LI></UL><H4><SPAN class="mw-headline" id="6-14">6-14</SPAN></H4><DL><DD><TABLE style="border: 1px solid green"><TBODY><TR><TD><I>L. acidophilus</I></TD></TR></TBODY></TABLE></DD></DL><DL><DD><UL><LI><I>L. acidophilus</I> plates with parafilm grew much quicker than the plate without parafilm. Most likely due to a more anaerobic environment, which Lb. a. likes better.
</LI><LI> No growth in the tubes yet.
</LI></UL></DD></DL><DL><DD><TABLE style="border: 1px solid blue"><TBODY><TR><TD>Ligation
</TD></TR></TBODY></TABLE></DD></DL><DL><DD><UL><LI> Ligation of YgiV with pBAD33BB. One ligation was a two hour benchtop and the other was overnight at 15C to see which one would be better.
</LI></UL></DD><DD><A href="/File:IGEM_two_hr_ligation_gel_3_6-14.jpg" title="File:IGEM two hr ligation gel 3 6-14.jpg">File:IGEM two hr ligation gel 3 6-14.jpg</A></DD></DL><OL><LI> Ladder
</LI><LI> Control (pBAD33BB vector only)
</LI><LI> YgiV:pBAD33BB ligation
</LI></OL><DL><DD><TABLE style="border: 1px solid purple"><TBODY><TR><TD>Electroporation/Transformation
</TD></TR></TBODY></TABLE></DD></DL><DL><DD><UL><LI> YgiV:pBAD33BB had a time constant of 4.40ms
</LI><LI> Control (pBAD33BB vector only) had a time constant of 5.40ms
</LI><LI> Incubated in a shaker for 1 hour and then plated pelleted cells on chlorophenicol plates.
</LI></UL></DD></DL><DL><DD><TABLE style="border: 1px solid orange"><TBODY><TR><TD>Competent cells
</TD></TR></TBODY></TABLE></DD></DL><DL><DD><UL><LI> Prep: UV centrifuge bottles and put in freezer
</LI><LI> Made 4 x 250ml and 2x 100ml salt free lb
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="6-15">6-15</SPAN></H4><DL><DD><TABLE style="border: 1px solid pink"><TBODY><TR><TD>Screening
</TD></TR></TBODY></TABLE></DD></DL><DL><DD><UL><LI>Screened two clones that grew from the transformation from 6-14. Followed the colony PCR. Did a 1:10 dilution of a 0.5 uL aliquot of pBAD seq primers, because the individual amount was too small to accurately pipette. After colony PCR, Yue Wu ran a gel of the colony PCR product (see below).
</LI></UL></DD></DL><DL><DD><A href="/File:IGEM_2010-06-14_Colony_PCR_YgiV_pBAD33BB.jpg" title="File:IGEM 2010-06-14 Colony PCR YgiV pBAD33BB.jpg">File:IGEM 2010-06-14 Colony PCR YgiV pBAD33BB.jpg</A></DD></DL><H4><SPAN class="mw-headline" id="6-16">6-16</SPAN></H4><P>Peter and I made competent cells 50 aliquots of 40 uL. Follow protocol on page 1 of notebook.
</P><P>Tranformed competet cells with pBAD33BB miniprep (6-14) to test the efficency.
</P><H4><SPAN class="mw-headline" id="6-17">6-17</SPAN></H4><P>Transformation efficency: 7 CFUs were on the 10^-5 plate, 87 CFUs were on the 10^-4 plate, ~640 CFUs were on the 10^-3 plate, 10^-2 had ~ 2,000, 10^-1 had a lawn. I used the 10^-4 for the effciency.  The efficiency was 1.41E10 CFUs/ug
</P><P>Nate did Alkyl lysis prep on the two potienal colonies used for colony PCR.
</P><P>Then I digested with PstI-HF only and PstI-HF, XbaI double digest. After digest, I ran a gel (see below).
</P><DL><DD>Lanes
</DD><DD>  2-log Ladder
<OL><LI> Colony 1 Undigested Alkyl Lysis product 
</LI><LI> Colony 1 Digested (PstI-HF)
</LI><LI> Colony 1 Digested (PstI-HF and XbaI)
</LI><LI> Colony 2 Undigested Alkyl Lysis product
</LI><LI> Colony 2 Digested (PstI-HF)
</LI><LI> Colony 2 Digested (PstI-HF and XbaI)
</LI><LI> YgiV gel extration purified cut product
</LI><LI> pBAD33BB miniprep
</LI></OL></DD></DL><P><A href="/File:IGEM_2010-06-17_post_screen_alkyl_lysis_digestion2.jpg" title="File:IGEM 2010-06-17 post screen alkyl lysis digestion2.jpg">File:IGEM 2010-06-17 post screen alkyl lysis digestion2.jpg</A></P><PRE> 6-21 to 6-22
</PRE><H4><SPAN class="mw-headline" id="6-21">6-21</SPAN></H4><P>On Friday, 6-19, I did a colony PCR of RfaI:pBAD35BB from Richard's transformation colonies. I came in on Saturday morning to run a gel and didn't find anymore Loading Dye, so I put the colony PCR product into the freezer. Monday (6-21) I got a 1ml of loading dye from the bottle in Brian's lab and ran a gel of the colony PCR product.
</P><P>The gel didn't show any bands in the 700 region. The positive control (pBAD35BB [60.1 ng/ul]) had a faint band in the 3kB region and a fuzzy streak at the end (could be primer dimer) didn't not have a band in the 700 region. The negative control was negative of any bands. Since there was no bands in any of the colonies screened, I will not be doing Alkly lysis and digestion.
</P><P>Peter and Sarah looked at the pBAD35BB sequence from the Biotech center and found that the PstI cut site had a point mutation that prohibited the PstI enzyme to cut. This could be one of the problems that lead to unsuccessful clones with the pBAD35BB vector.
</P><H4><SPAN class="mw-headline" id="6-22">6-22</SPAN></H4><P>Today, I did a miniprep of YgiV and RfaI encapsulation parts with Hannah. Since pBAD35BB PstI cut site is incorrect. We decided to clone both parts into pBAD33BB.  I ran a gel of the miniprep product to double check the nanodrop concentration.  From the nanodrop, the concentration YgiV [238.6 ng/uL] and RfaI [139.2 ng/uL]
</P><P>Richard and I did a overnight benchtop digestion of both parts and pBAD33BB vector. The parts were ones that he earlier minipreped, and not the ones Hannah and I did today.
</P><H4><SPAN class="mw-headline" id="6-23">6-23</SPAN></H4><P>Today, I did research for the pH testing of L. acidophilus strain and MG1655 without any of the colanic acid genes to see how well they grow without the colanic acid genes.  In order to use the plate reader, we need to transform a RFP gene into L. acidophilus and MC1655. That is simple for MG1655, but for L. acidophilus conjugation might be nessarary, according to Prof. Steele.  The Registry of Parts have two plasmids that can mobilize between E. coli and Lactobacillus, but in the notes, they warn that certain strains are picky. So I am currently researching articles for acidophilus plasmids.
</P><P>Another option is to use a chemical like Resazurin, which when the cells metabolize the chemical it becomes florescent. 
</P><P>Screened 14 colonies from pBAD35BB? transformation plate. 
</P><H4><SPAN class="mw-headline" id="6-24">6-24</SPAN></H4><P>Prepped the 14 overnight cultures of the screens by Alkyl lysis miniprep. 
Made solution for competent cells.
</P><H4><SPAN class="mw-headline" id="6-25">6-25</SPAN></H4><UL><LI>Making competent cells today!
</LI><LI>More iGEM parts came in and making Streptomycin plates for one of the parts.
</LI></UL><UL><LI>Here is the file on a different mechanism of induction by bile. <A href="/File:AcrAB_efflux_pump_by_RamA_Nlkaldo_2008.pdf" title="File:AcrAB efflux pump by RamA Nlkaldo 2008.pdf">File:AcrAB efflux pump by RamA Nlkaldo 2008.pdf</A></LI></UL><P>It is found in <I>Salmonella enterica</I> serovar Typhimurium. The protien, RamA binds to the acrAB promoter in the presence of bile. The transcription of RamA is induced by indole. Indole is found in the intestines due to bacterial metabolism of the amino acid tryptophan.  The paper suggests that RamA is a major regulator of the operon acrAB and that when bile is present the bile some how changes the RamA from an inactive state to an active state thus increasing the expression. On page 24252, they have a diagram of how they think the indole and bile coordinate the induction. 
</P><P>The paper also mentions that <I>E. coli</I> does not have RamA to control acrAB expression and instead uses multiple proteins to control acrAB expression.  If you put the Salmonella acrAB gene into E. coli will the native proteins also induce expression or will the Salmonella acrAB gene be different enough that the native proteins won't recognize and induce expression? 
</P><P>To help with this question, I plan on doing a BLAST of the Salmonella acrAB operon and the ramA gene to see how similar it is with <I>E. coli</I>. Also I plan on doing the same BLAST but for Lactobacillus. If Lactobacillus doesn't have a acrAB gene, this might prove beneficial because we won't have to worry about other native proteins also transcribing the engineered gene.
</P><PRE> 6-28 to 7-2
</PRE><H4><SPAN class="mw-headline" id="6-28">6-28</SPAN></H4><P>In the group meeting with Wes, we discussed about the pBAD35BB mutated PstI site. need to cut with SpeI and PstI. PstI is on both parent and Biobricked clone (this would be to check if the PstI site worked from the PCR amplification on new pBAD35 miniprep. 
</P><P>Also we talked about the logistics of the pH acid test on <I>L. acidophilus</I> and MG1655 <I>E. coli</I>. If we are using a GFP/RFP protien, we need to transform it into the two hosts. Easy for <I>E. coli</I>, but for <I>L. acidophilus</I> better to use conjugation.  There is a plasmid in the Registry of Parts that the MIT 2008 team used to transform into a Lactobacillus spieces from <I>E. coli</I>. So we plan on trying to use this plasmid.  Also Wes suggested chemical transformation.
</P><P>In lab today, I did a transformation of pBAD33BB (miniprepped 6-21-10) to test the competent cell transformation efficiency. The time constant was 4.6 ms
</P><P>Also the Lactose-Drug team decided to use the plasmids the parts came in to clone (ie pSB1AK3). Richard still had no clonig success with pBAD33BB and Nate has had greater success using the plamids the parts come in. So we started overnight cultures to beginning cloning in these vectors.
</P><H4><SPAN class="mw-headline" id="6-29">6-29</SPAN></H4><P>Competent cell transformation plates: 
</P><UL><LI>10^-1 &gt;~2,000 CFUs
</LI><LI>10^-2 ~2,000 CFUs
</LI><LI>10^-3 ~356 CFUs
</LI><LI>10^-4 22 CFUs
</LI><LI>10^-5 2 CFUs
</LI></UL><P>To calculate the transformation efficiency, I used 10^-3, 356 CFUs
</P><P>
Miniprep
</P><P>Hannah and I miniprepped 2 tubes of K200021 (LacI promoter), which we are going to use as our vector, and one tube of K137113 (RcsA, colanic acid encapsulation gene).  
</P><DL><DD>Concentrations from nanodrop:
</DD><DD>K200021 A [138.6 ng/uL]
</DD><DD>K137113   [207.2 ng/uL]
</DD><DD>K200021 B [91.4 ng/uL]
</DD></DL><P>Digest of LacI A and RscA parts above:  Want 1.5ng for 50uL Rxn, for 2 hours at 37C in water bath. 
</P><P>For LacI A vector, I digested with SpeI and PstI. For RscA, I digested with XbaI and PstI. This will make a scar with SpeI and XbaI and the two PstI will create a PstI site. This will keep the four biobrick site intact to clone in the next part.  The vector length should be 3.2kb and the insert length should be 625 bp.
</P><P>Gel Extraction of the two parts:
</P><P>Ran 2 cut products (digest product) and 2 uncut products (miniprep product). The gel was run at 75V for 45 min. After a couple of minutes I notice it was going towards the negative electrodes (it was about 2mm past the wells). I had accidentally switched the electrodes in the machine, even though the chamber direction was correct, but it seemed it didn't effect any of the bands. 
See gel picture below.
<A href="/File:IGEM_2010-06-29_K200021_RscA_gel_extraction.jpg" title="File:IGEM 2010-06-29 K200021 RscA gel extraction.jpg">File:IGEM 2010-06-29 K200021 RscA gel extraction.jpg</A></P><P>
Ligation
</P><P>Concentration from gel purification were K200021 [12.9ng/uL] and RcsA [5.5ng/uL].  I used the Clontech website to calculate the ng ratio. Vector was 100ng and insert 58.8ng. I divided these by the respective concentrations, giving 7.7uL and 10.7uL respectively. However, I want to do a 10uL reaction so I divided 7.7uL and 10.7uL by 2 to keep the ratio. This gave me 3uL for the vector and 5 uL for the insert. I ran the ligation in the PCR machine at 16C overnight.
</P><H4><SPAN class="mw-headline" id="6-30">6-30</SPAN></H4><P>Gel of ligation: 
1uL of product was used for all wells. For the ladder, 6uL was used of the premix ladder. There is a light smear in the LacI + RcsA ligation, so it possible it worked.
</P><P><A href="/File:IGEM_2010-06-30_LacI_RscA_ligation_product.jpg" title="File:IGEM 2010-06-30 LacI RscA ligation product.jpg">File:IGEM 2010-06-30 LacI RscA ligation product.jpg</A></P><P>Transformation:
</P><P>For LacI + RcsA, the time constant was 3.4ms. For the vector only control, the time constant was 3.8ms.
</P><P>Plating:  I plated 100uL and the pellet of both the ligation and control
</P><H4><SPAN class="mw-headline" id="7-1">7-1</SPAN></H4><P>Transformation plates:
</P><DL><DD>100uL aliquot
</DD><DD>Control 3CFUs
</DD><DD>LacI + RcsA 25CFUs
</DD></DL><DL><DD>Pellet
</DD><DD>Control 15 CFUs
</DD><DD>LacI + RcsA 221 CFUs
</DD></DL><P>Since the transformation results had a significant different between the control and variable, I did a colony PCR on 10 random colonies from the 100uL plate. After PCR, I ran a gel which had 8 successful clones. 1,3-5, 7-10. The RcsA insert was 624bp and the LacI promoter is 75bp, which is a total of 699bp. The successful clones hand one band around between the 600 and 700 region on the ladder. So it is most likely a clone with the RcsA insert and sequencing needs to be done to confirm it.  See gel below.
<A href="/File:IGEM_2010-07-01_LacI%2BRcsA_Colony_PCR_gel.jpg" title="File:IGEM 2010-07-01 LacI+RcsA Colony PCR gel.jpg">File:IGEM 2010-07-01 LacI+RcsA Colony PCR gel.jpg</A></P><P>I inoculated 5 and 7 for overnight cultures.
</P><H4><SPAN class="mw-headline" id="7-2">7-2</SPAN></H4><P>Peter and I made 52 aliquots of 40uL competent cells. Sarah also minipreped the two overnight cultures along with her overnight cultures.  The next step to do is to insert LacI and RcsA construct in front of two terminators (B0015) to complete the construct.
To do this I want to clone the previous construct in front of the B0015 part in the pSB1AK3 plasmid. So I digested the LacI + RcsA construct with EcoRI and SpeI. The pSB1AK3:B0015 was digested with EcoRI and XbaI.  DNA digested was 1.5 ug each as a 50uL reaction for 2 hours in a 37C water bath.
</P><P>After digestion, I ran a gel to extract the desired DNA fragments. For pSB1AK3:B0015, I wanted the 3318bp length and for LacI + RcsA construct I wanted the fragment at 700bp. See below.
</P><P><A href="/File:IGEM_2010-07-02_LacI%2BRcsA_pSB1AK3.jpg" title="File:IGEM 2010-07-02 LacI+RcsA pSB1AK3.jpg">File:IGEM 2010-07-02 LacI+RcsA pSB1AK3.jpg</A></P><P>
After gel extraction, I did a gel purification and a 10uL overnight ligation reaction, using 3uL of vector (pSB1AK3:B0015) and 5uL of insert (LacI+RcsA). 
</P><P>I also received and plated part BBa_I742123 (pTG262) plasmid part that is a shuttle vector between <I>E.coli</I> and <I>Lactobacillus</I> species.
</P><H4><SPAN class="mw-headline" id="7-3">7-3</SPAN></H4><P>I froze my ligation for a transformation on Monday.  The pTG262 didn't grow.
</P><H4><SPAN class="mw-headline" id="7-5">7-5</SPAN></H4><P>Transformation
</P><P>Transformed the ligation done 7-2/7-3. 
</P><DL><DD>Time constants
</DD><DD>Control 4.20ms
</DD><DD>LacI+RcsA+B0015 4.10ms
</DD></DL><P>Transformation of ygiV:pSB1AK3 for tranformation efficiency of compet. cells.
</P><DL><DD>Time constant 5.10
</DD></DL><P>Also ran a ligation gel. See below.  There is a little smear, which means it might have worked.
</P><DL><DD>Lane 1  Ligation product
</DD><DD>Lane 2  Control (vector B0015:pSB1AK3 only)
</DD><DD>Lane 3  B0015:pSB1AK3 miniprep product
</DD><DD>Lane 4  B0015:pSB1AK3 digested (gel extraction product)
</DD><DD>Lane 5  LacI+RcsA digested (gel extraction product)
</DD></DL><P><A href="/File:IGEM_2010-07-05_Ligation_(21_%2B_13)_%2B_B0015.jpg" title="File:IGEM 2010-07-05 Ligation (21 + 13) + B0015.jpg">File:IGEM 2010-07-05 Ligation (21 + 13) + B0015.jpg</A></P><H4><SPAN class="mw-headline" id="7-6">7-6</SPAN></H4><P>Transformation plates for Trans. efficiency
</P><DL><DD>10^-1 solid lawn
</DD><DD>10^-2 speckled lawn
</DD><DD>10^-3 &gt;~2,000
</DD><DD>10^-4 ~2,000
</DD><DD>10^-5 ~380 CFUs
</DD></DL><P>Transformation plates for ligation
</P><P>Control 8 CFUs
Ligation ~800 CFUs
</P><P>Colony PCR
</P><P>I picked 10 random colonies from the ligation plate and used them for colony PCR. I used the pSB1AK3:B0015 miniprepped product as the positive control. I also had a negative control of just water in place of the DNA.
</P><P>Then I ran a gel of the colony PCR. The successful clones showed a band around 1 kb, which should have LacI (75bp) + RcsA (625bp) + B0015 (129bp)= 825bp + the length of the primers. See picture below. 
</P><P><A href="/File:IGEM_2010-07-06_(21%2B13)_%2BB0015_colony_PCR.jpg" title="File:IGEM 2010-07-06 (21+13) +B0015 colony PCR.jpg">File:IGEM 2010-07-06 (21+13) +B0015 colony PCR.jpg</A></P><P>I chose 4 and 8 for overnight cultures.
</P><H4><SPAN class="mw-headline" id="7-7">7-7</SPAN></H4><P>Miniprepped the two overnight cultures. Then digested one of them (#8) and the LacI+ RcsA construct in a 10uL reaction for 2 hours at 37C as a second check to see if the the right parts ligated together.  
</P><P>Then I ran 5uL of product on a 3% gel to see the two diffent lengths. The difference should be ~129bp. See below. 
</P><DL><DD>Lane 1 LacI+RcsA digest 
</DD><DD>Lane 2 LacI+RcsA+B0015(TT) 
</DD><DD>Lane 3 LacI+RcsA uncut 
</DD><DD>Lane 4 LacI+RcsA+B0015(TT)uncut
</DD></DL><P><A href="/File:IGEM_2010-07-07_Digest_for_sequencing.jpg" title="File:IGEM 2010-07-07 Digest for sequencing.jpg">File:IGEM 2010-07-07 Digest for sequencing.jpg</A></P><H4><SPAN class="mw-headline" id="7-8">7-8</SPAN></H4><P>Hannah and I did sequencing PCR reaction for sending to the Biotech center.
</P><PRE> 7-19 to 7-23 Summary
</PRE><P>Making primers for isolating the ramA gene, RamA binding site,arcR?,pacrAB? from Salmonella enterica LT2. 
</P><P>Trying to decide whether to use the acrR:acrAB repression system or the cI phage repressor system. I know I don't want to use the pLacI promoter because the system will be in the system will be in the presence of lactose and will prevent the repressor from binding. If I do use the acrR:acrAB system I will probably need to knock out MarA because its a transcriptional activator for the acrAB promoter.
</P><P>Making acid dunking and acid growth curve assays. Also making pH 2-7 solutions of LB/MRS broth to use next week.
</P><H4><SPAN class="mw-headline" id="7-26">7-26</SPAN></H4><P>I decided to use the acrR:acrAB repression system because it will be easier to use since it already has the ramA binding site and encodes for a repressor. making primers ro amplify the ramA and acrRAB region.
</P><H4><SPAN class="mw-headline" id="7-28">7-28</SPAN></H4><P>Made competent cells and ordered primers
</P><H4><SPAN class="mw-headline" id="8-6">8-6</SPAN></H4><UL><LI>Made pH-media gradient with Peter(has data)
</LI><LI>Made PCR reactions for temperture gradients for amplify ramA and acrRAB. Made two master mixes for 8,10ul reactions. Range for ramA was 72C to 80C and the range for acrRAB was 71C to 81C.
</LI><LI>Ran gel of PCR reactions. See below.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Wisconsin-Madison/notebook/Mary">http://2010.igem.org/Team:Wisconsin-Madison/notebook/Mary</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wisconsin-Madison/notebook/Mary" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wisconsin-Madison/notebook/Mary" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/notebook/Mary&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/notebook/Mary&amp;oldid=137995" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>