<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wisconsin-Madison_delivery"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wisconsin-Madison/delivery">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wisconsin-Madison/delivery&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/delivery&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wisconsin-Madison/delivery&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wisconsin-Madison/delivery" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wisconsin-Madison/delivery</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav_content" style="font-size: 80%"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Physiology"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Physiology</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Ingestion"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Ingestion</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Stomach"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Stomach</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Small_Intestine"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Small Intestine</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Large_Intestine"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Large Intestine</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Disorders"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Disorders</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Gluten_Intolerance.2FCeliac_Disease"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Gluten Intolerance/Celiac Disease</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#High_Cholesterol"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">High Cholesterol</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Lactose_Intolerance"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Lactose Intolerance</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Project"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Project</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Overview"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Growth"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Growth</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Enzyme_Production"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Enzyme Production</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Capsule_Formation"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Capsule Formation</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Lysis_Systems"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Lysis Systems</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Inducible-Repressible_System"><SPAN class="tocnumber">3.5.1</SPAN><SPAN class="toctext">Inducible-Repressible System</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Bile_Induced_System"><SPAN class="tocnumber">3.5.2</SPAN><SPAN class="toctext">Bile Induced System</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Encryption_System"><SPAN class="tocnumber">3.5.3</SPAN><SPAN class="toctext">Encryption System</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Industrial_Scaling"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Industrial Scaling</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV><DIV id="page_content"><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P>Lactose and gluten intolerance involve enzyme production deficiency in the digestive system.  These disorders and high cholesterol could be treated by supplementation of the correct enzyme at the correct place in the digestive system.  We have designed a universal platform for polypeptide release within the small intestine of the human gastrointestinal tract.  Our automated, model system is designed to treat lactose intolerance where patients are unable to break down the lactose disaccharide into monomeric sugars due to deficiency of lactase.  The system produces the enzyme beta-galactosidase, a functional homologue of human lactase, and releases it to break down lactose. Colonic acid production occurs prior to ingestion to shield the bacteria from the high acidity of the stomach.  Lysis occurs in the duodenum of the small intestine through one of three proposed systems: a timed inducible/repressible system, a bile induced system, or an encryption system.  Delivery of different enzymes can be accomplished by simple exchange of one part.
</P><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><H3><SPAN class="mw-headline" id="Physiology">Physiology</SPAN></H3><H4><SPAN class="mw-headline" id="Ingestion">Ingestion</SPAN></H4><P>Mechanical digestion of food begins in the mouth.  From the mouth, food is passed through the esophagus.  Total time for both processes is usually less than a minute.  
</P><H4><SPAN class="mw-headline" id="Stomach">Stomach</SPAN></H4><P>Mechanical digestion continues in the stomach, as well as chemical process such as low pH and enzymes that catalyze breakdown of biological molecules.  These chemical aspects of digestion also serve to protect the body from infection by pathogens, destroying most microrganisms.  The pH for this section of digestion is around 2-the lowest in any area of the digestive system.  Food remains in the stomach for an average of 4 hours.  
</P><H4><SPAN class="mw-headline" id="Small_Intestine">Small Intestine</SPAN></H4><P>The small intestine is the primary site of nutrient absorption and a major site of enzymatic breakdown of food.  The liver creates bile from cholesterol which is stored in the gall bladder and released into the upper part of the small intestine as bile salts.  These salts help emulsify ingested fats.  The pancreas also secretes enzymes into this section.  The pH of this region ranges from 6-6.5, and transit takes an average of 3 hours.
</P><H4><SPAN class="mw-headline" id="Large_Intestine">Large Intestine</SPAN></H4><P>The large intestine reabsorbs water before the waste is excreted.  The pH in this region ranges from 5.5-7 and transit time is usually over a day.  It is heavily colonized by a wide variety of bacteria.
</P><H3><SPAN class="mw-headline" id="Disorders">Disorders</SPAN></H3><H4><SPAN class="mw-headline" id="Gluten_Intolerance.2FCeliac_Disease">Gluten Intolerance/Celiac Disease</SPAN></H4><P>Gluten is a complex protein found in wheat.  Gluten Intolerance is the name given to people who have adverse reactions to gluten.  In a specific type of genetically linked gluten intolerance, Celiac disease, gluten causes production of antibodies and an allergic reaction which damages the small intestine over time.  This leads to poor nutrient absorption and over time, malnutrition.  Currently there is no enzyme that remains stable through the digestive tract to effectively treat this condition enzymatically, and most suffers avoid gluten consumption.
</P><H4><SPAN class="mw-headline" id="High_Cholesterol">High Cholesterol</SPAN></H4><P>High cholesterol over time can lead to heart disease, and atherosclerosis or hardening of the arterial walls, which will eventually lead to heart attacks (NIH, 2008). 
</P><P>Bile salts are produced from cholesterol in the liver and stored in the gall bladder.  Certain enteric organisms have the ability to modify bile salts with an enzyme called bile salt hydrolase, which deconjugates the bile salts into a form that is less harmful to the organism (Liong and Shah, 2005).  This deconjugation changes the bile salt conformation, which allows it to be reabsorbed by the epithelial lining and to be excreted from the body through the blood stream (Liong and Shah, 2005).  By delivering this enzyme to the small intestine, bile salts would be modified and excreted from the body as waste, forcing the liver to produce more bile salts from serum cholesterol, thereby lowering it and preventing the patient from the harmful effects of high cholesterol.  
</P><H4><SPAN class="mw-headline" id="Lactose_Intolerance">Lactose Intolerance</SPAN></H4><P>Lactose intolerance is a digestive disorder resulting from an inability to digest lactose.  Symptoms include abdominal pain, bloating, diarrhea and nausea (NIH, 2006).  The disorder is widespread, particularly among certain racial demographics.  For example, 95% of Asians, 60-80% of African Americans and Ashenkazi Jews, 80-100% of American Indian, and 50-80% of Hispanics are lactose intolerant (NIH, 2006). In America, it is estimated there are 30 million to 50 million Americans who are lactose intolerant (NIH, 2006).  The disease is caused by deficiency in the enzyme lactase from primary genetic causes or secondary causes such as injury to the small intestine or chemotherapy.  Administration of enzymes has been shown to effectively treat the disorder (NIH, 2009).  Enzymes stable in the digestive system have been purified and are currently used as an effective treatment.
</P><H2><SPAN class="mw-headline" id="Project">Project</SPAN></H2><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>This project seeks to deliver functional polypeptides to the the small intestine.  Bacteria are used to produce the enzyme, act as shield for the enzyme through the stomach, then release the enzyme through properly timed lysis.  The specific enzyme discussed here is Beta-galactosidase, the enzyme that breaks lactose into glucose and galactose.
</P><H3><SPAN class="mw-headline" id="Growth">Growth</SPAN></H3><P>The final system will be engineered into <I>Lactobacillus acidophilus</I>, a strain used in active yogurt cultures.  This approach has several advantages.  First ingestion of the culture is more pleasant when it is consumed with a tasty treat.  Yogurt is also known to buffer for the stomach making it less acidic and promoting survival of the bacterial shuttle to the small intestine(Martini et. al., 1987).  Finally there are fewer concerns about safety when using a commonly consumed microbe found naturally in the human gastrointestinal tract. Here <I>Escherischia coli</I> is used to test and develop the system due to the well established molecular biology techniques used to manipulate it genetically.
</P><H3><SPAN class="mw-headline" id="Enzyme_Production">Enzyme Production</SPAN></H3><P>This model system will address lactose intolerance. Producing Human Lactase in our bacterial cells would be difficult so we have chosen an enzyme that acts as functional homologue of human lactase called Beta-Galactacidase (B-Gal). 
</P><CENTER><P><B>Left</B> Lactase,  <B>Right</B> Beta-Galactacide
</P></CENTER><P>We have taken advantage of the wealth of devices that can be found in the iGEM Registry of Standard Parts. In 2008, the Cal Tech team had created a series of devices containing the gene for Beta-Galactacidase production. The difference in each device is the activity of each of the constitutive promoters. Thanks to their work, we can fine tune the amount of Beta-Galactacidase produces within our cells.
</P><P>It is important to remember that this is the exchangeable construct of our project. Because 1) testing for B-Gal activity is easy and 2) the gene <I>B-Gal</I> is easily obtained and very well characterized, we have chosen to create a Lactose Intolerance specific treatment. 
</P><P>This system can act as a platform for the treatment of other disorders using the following steps:
</P><OL><LI> Find a disorder that can be solved by polypeptide delivery to the small intestine
</LI><LI> Place the gene for this enzyme into the expression vector
</LI><LI> Add to iDIET system
</LI></OL><H3><SPAN class="mw-headline" id="Capsule_Formation">Capsule Formation</SPAN></H3><P>Our goals is to have each cell be surrounded by a protective 'capsule' to allow them to safely travel through the harsh acidic environment of the stomach to arrive in the small intestine for their main purpose. Imperial 2008 used Transcription Factor RcsB to stimulate a the production of Colonic Acid from the capsule synthesis pathway of <I>E.coli</I>. Colonic Acid s a polysaccharide containing a repeat unit with D-glucose, L-fucose, D-galactose, and D-glucuronate. Colonic Acid has been shown to increase cell survivability in acidic conditions. We investigated the pathway for capsular polysaccharides synthesis and found more transcription factors that could give a similar or even better results!
</P><P>RcsA and RcsB are transcription factors that are know to be positive regulators of capsular polysaccharides synthesis. We placed RcsA, RcsB, and a combination of the two under a IPTG inducible promoter to test both quantity of colonic acid produced and cell survivability. These two transcription factors form a heterodimer that is know to activate around 19 genes related to colonic acid synthesis.  RcsB is also know to form a homodimmer and positively regulate cell division. RcsA and RcsB belong to the multicomponent RcsF/RcsC/RcsD/RcsA-RcsB phosphorelay system.
</P><P>More information on these transcription factors and their usage in Biology can be found here: [<A href="http://biocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10820" class="external text" rel="nofollow">Ecocyc-RcsA</A>] &amp; [<A href="http://biocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10821" class="external text" rel="nofollow">Ecocyc-RcsB</A>]
</P><P>We wanted to test what IPTG induction levels were appropriate to protect our cells from an acidic environment of pH=2 before placing our devices under constitute control with appropriate ribosome binding sites.
</P><P>Our final system will utilize a probiotic stain called <I>Lactobacillus acidophilus</I> that is native to the human gut. The pathways in this bacterium are different than that of <I>E.coli</I>. Our next stage of research will involve finding a pathway, gene, or transcription factor that upregulates capsule synthesis and protocols to engineer this strain.
</P><P>
One caution we had to consider in up-regulating this pathway is biofilm formation. Colonic Acid synthesis is one of the first steps of the biofilm formation pathway. To ensure a biofilm is not produced we have taken advantage of an inhibitory transcription factor YgiV that blocks the activation of further steps in this pathway. For now, we are content to test our constructs in an <I>E.Coli</I> chassis.
</P><H3><SPAN class="mw-headline" id="Lysis_Systems">Lysis Systems</SPAN></H3><P>To deliver the dose of enzymes to the small intestine, the bacteria will lyse.  Lysis must occur in the small intestine, too early and the stomach could harm enzyme activity, too late and the enzyme isn't able to act on its target.
</P><H4><SPAN class="mw-headline" id="Inducible-Repressible_System">Inducible-Repressible System</SPAN></H4><P><B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K318512" class="external text" rel="nofollow">gadAp</A>: A pH Sensitive Promoter</B>. One of the most apparent environmental changes which occurs upon chyme movement from the stomach to the small intestine is a sharp (approximately 2-10)  pH gradient. However, in order to utilize this as a signaling mechanism for cell lysis, we required a pH sensitive element. Results of northern blot analysis (Tucker <I>et al.</I>, 2002; Waterman <I>et al.</I>, 2003) indicate that a particular glutamate decarboxylase (GadA) is up-regulated, most likely as a part of a glutamate to gamma-aminobutyrate reaction which exports intercelluar protons which is required for <I>E. coli</I> stationary phase cell survival of highly acidic (pH 2) conditions (Castanie-Cornet <I>et al.</I>, 1999). Based on a high ratio of low pH:neutral pH expression and a wealth of available information, we chose to use GadA's promoter, gadAp, as a native-<I>E. coli</I> sensor for acidic conditions. In order to maximize pH-mediated response of the gadA promoter and minimize induction caused by other known responses (such as the stationary phase), we plan to run error-prone PCR followed by screening of clones for maximal, dedicated acid response.
</P><P>General information about this promoter and other related acid-sensitive promoters in this system can be found here: [<A href="http://biocyc.org/ECOLI/NEW-IMAGE?type=OPERON-IN-CHROM-BROWSER&amp;object=TU586" class="external text" rel="nofollow">Ecocyc-gadAp</A>].
</P><P>A previously characterized inducible-repressible promoter (<A href="http://partsregistry.org/Part:BBa_R0065" class="external text" rel="nofollow">BBa_R0065</A>) will be used to accomplish our goal of delivering <I>E. coli</I> lactase to the human intestine. This system will be used in combination with the previously described gadA promoter, LuxR to yield expression after the active cell has seen the acidic conditions of the stomach. In order to better describe the order of events, refer to the following diagrams and descriptions:
</P><P><B>Before Ingestion</B></P><P>The gadA promoter sees marginally acidic or neutral conditions and therefore yields expression of insignificant levels of both LuxR and cI-LVA. Since expression of both LuxR and cI-LVA are at low levels, the level of induction from inducible-repressible promoter is limited and cell lysis is not induced.
</P><P><B>In Stomach</B></P><P>Upon arrival to the stomach, the acidic pH conditions experienced by the cells induces expression from the gadA promoter. LuxR and cI-LVA are both, therefore, expressed. In the presence of both the activator and repressor of the inducible-repressible promoter BBa_R0065, there is only very limited expression of the downstream genes. We believe that any &quot;leaky&quot; expression will be further reduced by use of a very weak activator (LuxR without the presence of molecules of AHL). A future step would be to remove the N-terminal end of the protein in order to mitigate potential complications due to accidental presence of AHL.
</P><P><B>Immediately After Stomach</B></P><P>After leaving the stomach, the repressor cI is degraded at a faster rate as compared to LuxR due to an attached LVA tag. This causes an increasing cell lysis signal.
</P><P><B>In Small Intestine</B></P><P>Eventually, the LuxR:cI ratio is sufficiently high to cause the cell lysis signal via the inducible-repressible promoter.
</P><H4><SPAN class="mw-headline" id="Bile_Induced_System">Bile Induced System</SPAN></H4><P>The bile inducible system consists of the ramA gene and the acrRA operon that were PCR amplified from <I>Salmonella enterica</I> strain LT2, locus tag STM0581. The <I>Salmonella enterica</I> strain LT2 chromosomal DNA came from Diana Downs' Lab at the University of Wisconsin-Madison. RamA is a regulatory protien that binds to bile and then binds to a specific sequence upstream of the acrAB operon (Nikaido <I>et.al</I>, 2008). This regulatory binding causes an increase in transcription of the acrAB gene (Nikaido <I>et.al</I>, 2008).  To apply this concept to the timed lysis mechanism, the <I>acrRA</I> operon was cloned in front of a GFP reporter part, E0240, from the registry.  Since <I>E. coli</I> DH10B cells do not endogenously produce ramA protein, the ramA gene was also cloned behind a constitutive promoter, J23100, so the protein is constantly present in the cell to bind with bile.  Ideally, these two constructs would then be cloned into plasmids with two different origins of replication for compatibility inside the cell. 
</P><P>In theory, when the cells come into contact with bile, the ramA will bind bile and then bind to the DNA, thus causing transcription of the acrA and the lysis gene, which will cause the cells to lyse and release beta-galactosidase. See images below. 
</P><P>Bioinformatics
</P><P>Conserved domains  <A href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi?RID=BKT8A21D014&amp;mode=all" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi?RID=BKT8A21D014&amp;mode=all</A> 
Some of the conserved domains within the protein are HTHAraC and PRK 10219.  HTHAraC is a bacterial regulatory helix-turn-helix protein.  The Blast score was 78 and the location of the domain is 15-56 amino acids. The other conserved domain is PRK 10219, which is a DNA-binding transcriptional regulator SoxS.  The blast score was 244 with the location to be 6-105 amino acids.
</P><P>Homologues 
</P><P>From the <I>Salmonella enterica</I> LT2 sequence at NCBI, the RamA attachment site is located within the acrR gene.  Based from Nikaido et.al., and from the <I>Salmonella enterica</I> LT2 sequence, the proposed attachment site is 5â-atggcac gaaaaa ccaaaca-3â. The Blast of the RamA protein sequence (NP_459573.1) resulted in similar proteins found in other <I>Salmonella enterica</I> species and subspecies, Citrobacter species, <I>Klebsiella pneumonia</I>, and Enterobacter species <A href="http://www.ncbi.nlm.nih.gov//blast/Blast.cgi" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov//blast/Blast.cgi</A> . 
The blast of the <I>ramA</I> gene sequence (NC_003197.1)  resulted in similar sequences found in <I>Salmonella enterica</I> subspecies enterica with different serovars, <I>Citrobacter koseri</I> ATCC BAA-895, and <I>Citrobacter rodentium</I> ICC168 (<A href="http://www.ncbi.nlm.nih.gov//blast/Blast.cgi" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov//blast/Blast.cgi</A>). 
</P><P><B>1) Before Small Intestine</B></P><P><B>2) In Small Intestine</B></P><H4><SPAN class="mw-headline" id="Encryption_System">Encryption System</SPAN></H4><P>The encryption system that we developed could also be used for precise, timed lysis in the small intestine.  By using a pH-sensitive promoter and a bile-salt-sensitive promoter, we could initiate recombinase production at the right times, triggering lysis.  The part <A href="http://partsregistry.org/Part:BBa_K318521" class="external text" rel="nofollow">K318521</A> is a device capable of initating cellular lysis after low pH and bile salt input respectively.  Otherwise the construct disables.  See the page about encryption for more information.
</P><H3><SPAN class="mw-headline" id="Industrial_Scaling">Industrial Scaling</SPAN></H3><P>We have considered the possibility of starting an iDIET company, pending ethical and safety issues.  We created a business plan for implementation of our iDIET concept and designed a process for production of an iDIET yogurt.
</P><P>With in 5 years we could have a fully operational production facility in southern Wisconsin, utilizing about 1000 US gallons per week of milk to produce approximately 400000 gallons of yogurt per year for a single strain of L. acidophilus.  Our estimated production facility cost is roughly $750000 plus or minus 10%.  Our operating gross margin is roughly 30-40% profit ($1.4-1.9M), but after estimated operational costs, this margin decreases to 15-20% ($700-950K). Based upon a production rate of 400000 gallons per year, we would recoup the costs of the facility in approximately one year.  After the first year, the profit could be used to develop the business by increasing production scale or to pay dividends to shareholders, if the business was public.
</P><P>A preliminary production process design:
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Wisconsin-Madison/delivery">http://2010.igem.org/Team:Wisconsin-Madison/delivery</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wisconsin-Madison/delivery" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wisconsin-Madison/delivery" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/delivery&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wisconsin-Madison/delivery&amp;oldid=206800" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>