"
Page
Discussion
View source
History
teams
Log in
 
Team:Wisconsin-Madison/notebook/Peter
From 2010.igem.org
Contents
1 Week of 7/12 Update
2 Week of 7/5 Update
3 Week of 6/25 Update
4 Week of 6/18 Update
5 Week of 6/10 Update
6 Thursday 4/8 Notebook
7 Friday 4/9 Notebook
Week of 7/12 Update
Major projects for this week: cloning of gadAp into pSB1A3, planning for next clones, planning for error-prone PCR
I used colony PCR product from MG1655 E. coli colonies to prepare double stranded template for my cloning reactions. So as not to waste more goTaq than necessary, I've also designed a PCR procedure to produce gadAp product from colony PCR product itself. Basically, all I had to do was alter the annealing temperature to take into account the fact that the entire primer binds. I believe the problems I had the last few weeks with gadAp were due to the lack of overhang on the primers, but I believe by increasing the digestion time to 4 hours for the PCR product has fixed this problem. I did not attempt a double insertion this time and will do sequential steps to add in RFP, cI and LuxR behind it.
File:IGEM 2010-07-12 high exposure.jpg
The colony PCR results from the experimental plate. I picked colonies 1 and 4 for further analysis by restriction mapping. Colony one appears at the proper location ~360bp but I wanted to be certain I could get a positive result in one attempt so I also picked 4 in order to see if perhaps it was just running oddly on the gel or in case my primers were in an odd location.
File:Capture.JPG
I digested each of the two small growths (from colonies 1 and 4) using EcoRI and PstI, EcoRI, and SspI. The banding pattern I would expect compared to what I got is shown below:
File:Table.JPG
Cutting with SspI had the added benefit of showing that the insertion is likely gadAp as one of the two cut sites was within the backbone and the other was within the insert itself.
The next step for working with this clone is to add an RFP with no RBS behind it. This part has been miniprepped today and I will proceed with the cloning this weekend.
Week of 7/5 Update
Major project for this week: cloning of LuxRΔ2-163(no stop) and gadAp into pSB1A2
Cloning plan/Digestions:
I used part #BBa_K142003 (pSB1A2 2079bp + LacI 1128bp) as the source of my vector for this cloning. I did 5 separate digestions to prepare: A) LuxRΔ2-163 PCR product cut with X and P, B) gadAp PCR product cut with E and S, K142003 cut with (C) X and P, (D) E and S, and (E) P and E. All digestions were run for 2 hours at 37C and placed directly into ligations.
Ligations:
I combined the above digestions as follows. A with C, B with D, and A/B with E. I did not expect the final ligation to work as it would require 2 digestion products to properly ligate to the vector backbone. Ligations were left overnight at 16C and transformed the following morning. A vector-only control was run for all samples.
Colony PCR:
Colonies were picked from across many different plates. On gel 1, the first 9 lanes following the ladder come from the LuxRΔ2-163 insertion. The following 3 are the vector-only control and the last two are the negative and positive controls for colony PCR. On the second gel, the colonies correspond to the remaining plates---none of these showed any difference from the uncut K142003, leading me to believe that the colonies simply correspond to the left-overs of this part.
File:IGEM 2010-06-09 LuxR and GadA col PCR 1.jpg
File:IGEM 2010-06-09 LuxR and GadA col PCR 2.jpg
I believe that these gels show me that my LuxR cloning may be successful, although it is suspicious that some of the negative controls for the insertion show up in the same location as the experimental samples. I don't know why this would be, considering that a ligated vector backbone should have a smaller number of base pairs between the primers than the 400-500 we see on the PCR gel.
Restriction Mapping:
File:Digestion.jpg
Lane 1: uncut pSB1A2 LuxRΔ2-163 Lane 2: cut with EcoRI Lane 3: cut with PstI and EcoRI
In lane 3, you can clearly see in the high exposure sample a band corresponding to the approximate size of LuxRΔ2-163 (264 bp) + the bio brick prefix and suffix. I believe this clone was successful.
New Developments:
I've since restarted my cloning to insert the gadAp from the PCR product. I am currently at the colony PCR stage of this cloning. If this cloning ends up being unsuccessful (given the colony numbers of the plates it is likely it was unsuccessful), I will reattempt the PCR reaction using new primers I have designed which now include additional bases beyond the cut sites. I expect this cloning to be more successful.
Week of 6/25 Update
I started the week by attempting a new site directed mutagenesis of pBAD18BB and pBAD33BB. The new protocol included altering the PCR times to fit better with the protocol for Quikchange Site Directed Mutagenesis:
98C - 30s
98C - 10s    -|
55C - 1min    | 25x
72C - 2.5min -|
72C - 5min
4C  - indef.
2.5 hours of digestion with DpnI at 37C
The results of this mutagenesis were less surprising than those from the previous week, but still were not successful. The gel (not shown as it was on the now-fried computer), showed banding around 100-200 bp on the experimental samples and no banding on the polymerase(-) controls. The plates were similar to what was seen with the previous mutagenesis reactions; there were many colonies on both the pBAD18BB plate and its negative control while none were present on the pBAD33BB plate. It was decided that the colonies on the pBAD18 plate were probably not worth screening due to the large number on the negative control.
I've seen this sort of problem before in my unsuccessful mutagenesis attempts in my other lab---either no colonies or extensive negative control colonies in combination with low MW bands on the gel. I've never found a good solution to the problem--perhaps the primers are simply not ideal? Regardless, Sarah and I have discussed a simple solution to our problems:
1. Using pBAD34 instead of pBAD18. This does not contain an additional cut site and is also a high copy number plasmid with an Amp resistance. Sarah executed an around the world PCR to biobrick the plasmid and I'm working on alkaline lysis to screen colonies for restriction mapping at the time of writing.
2. pBAD33, for whatever reason, does not appear to be cut an additional time by EcoRI. We're not sure if the site is no longer present or if it simply does not cut at that location. This indicates that we probably do not have to do site directed mutagenesis on this plasmid.
Ligations from last week's attempts to put RFP into pBAD35BB were cancelled as the reason why they probably were not working came out from Sarah's attempts to cut pBAD35BB and further examination of the sequencing data---it was missing a complete PstI cut site. A single bp mutation seems to have caused our copy of pBAD35BB to be unable to be cut by PstI. Therefore, I threw away my ligation materials for this purpose and started working towards putting RFP into pBAD33BB instead.
In addition to starting the digestion of RFP and pBAD 33, the ligation and screening were also performed this week. The colony PCR results were a little unclear.
File:Colony PCR 624.jpg
I believe some of the original bands got bleached out---there were additional bands for some of the colonies at ~600bp and near 1000bp. Regardless, it appeared that the colony PCR did not work properly on these samples (negative and positive controls were lanes 10 and 11 and showed no banding at all).
We therefore took the colonies and prepared small cultures for alkaline lysis. These are currently being digested.
My final major accomplishment of the week was preparing the primers for our activator in the inducible/repressible system. After a bit of research, I think I've determined that removing the first 163 amino acids of LuxR should completely remove the N-terminal signal binding domain and cause it to have a higher level of activation in the absence of AHL than the wild type.
Next week we should have pBAD33, 34, and 35 prepared and I will begin putting RFP into all of them as well as inserting the gadA (pH) promoter. In addition, Sarah and I have worked on drawing out a plan for which constructs need to be acid tested for the encapsulation system.
Week of 6/18 Update
I transformed the ligations that were attempted last week and found that we had a couple colonies from them---these were later confirmed by Sarah and colony PCR to not contain RFP and the pH promoter.
I started the week by producing additional PCR products of the pH promoter and rfp, after subsequent digestion and gel extraction of both the pBAD35BB and the rfp product. However, after the extraction, my concentrations were both less than 10ng/uL---and the samples looked very dirty on the nanodrop. Therefore, I ran a gel, and found that their concentrations were in fact far less than the small amount of ladder I added-inadequate for a ligation.
As such, Sarah and I re-ran the PCR reaction for RFP, this time after PCR cleanup, digestion and gel extraction we had a clean sample with a high concentration: 70ng/uL. In addition, Sarah started an overnight digestion of pBAD35BB, after PCR cleanup, this again showed a concentration less than 10ng/uL. Therefore, today (6/18) I started a 2 hour digestion and the gel extraction is running at the time of writing.
In parallel with the work on the pH promoter, I've also been working on preparing pBAD18 BB and pBAD33 BB for cloning by removing extraneous cut sites by site directed mutagenesis. After running two reactions (both unsuccessful) and seeing the gel of the products of the reactions (which contains >10kb size fragments and a smear of DNA all the way down to about 1kb), I think we may need to reorder primers. This is because we originally designed the primers for site-directed mutagenesis by the quikchange protocol, but we ended up using Phusion polymerase. I am worried that perhaps this polymerase does not work for this type of priming protocol---the protocol suggested by the Phusion company for one of their other polymerases suggests something more similar to around-the-world PCR primers and not the self-annealing primers that are used for the quikchange protocol.
It would probably be best to discuss this part of the project in the meeting.
Week of 6/10 Update
After we received the plasmids on Monday, I began by attempting PCR reactions to pull the pH promoter (gadAp) and the RFP off of MG1655 genomic DNA and an RFP containing plasmid respectively.
My first reaction following the protocol of colony PCR provided to us (6/7) was unsuccessful at showing bands of PCR product at ~200, no bands were present for this sample (lane 2 below).
However, the reaction to pull off rfp was successful, and showed a band at about 700bps (lane 3 below). File:Rfp and pH prom 6-7.jpg
I reattempted the pH promoter PCR optimizing the temperature with 5 small-scale reactions in combination of increasing the length of the initial cell lysis step to 5 minutes and doubling the concentration of template and primers. This PCR was successful, and the high temperature bands showed the highest yield.
Therefore, I ran large scale reactions of both the pH promoter PCR and RFP PCR to get alot of product to make sure that I could gain a high enough concentration for easy ligation. After PCR cleanup and enzymatic digestion for ligation, I ran the results on a gel (lane 1: pH promoter, lane 2: rfp, lane 3: pBAD 35BB).
File:IGEM 2010-06-09 Peter ph rfp Mary ligation prod Ygi Rscb.jpg
Due to the additional band in the RFP lane, I was forced to do a gel extraction of the digested RFP product. The final concentrations of each of the products was as followed before ligation: gadAp 44.3 ng/uL (184bp) rfp 50.4 ng/uL (756bp) pBAD35 BB 13.3 ng/uL (4184bp)
An overnight ligation was attempted with 90 ng of vector and a 3x molar ratio of insert:vector.
Thursday 4/8 Notebook
Attempted two sets double digestion with ClaI, EcoRI, and SpeI of pET-28b-bb (bri b). Treatments were as follows:
1. ClaI + EcoRI in buffer 4
2. ClaI + SpeI in buffer 4
3. ClaI in buffer 4
Lanes on .7% agarose gel were the same with additional lane 4 of 2-log ladder.
File:IGEM 2010-04-08 21hr 49min.jpg
No fragments appear and bands all ran to same (about 6kbp). It appears that this plasmid does not contain the biobrick sequence.
Friday 4/9 Notebook
Wanted to see if any of the pET28b-bb plasmids we have in the freezer have a successful insertion of the biobrick section of the plasmid. As such, ran ClaI + EcoRI digestions of all of the plasmids. Treatments were as follows (1hr, 37C with 1uL of each enzyme in 30uL reactions with 1xBSA):
1. plasmid bri A ClaI + EcoRI in buffer 4
2. plasmid bri B ClaI + EcoRI in buffer 4
3. plasmid bri C ClaI + EcoRI in buffer 4
4. plasmid bri D ClaI + EcoRI in buffer 4
5. plasmid bri E ClaI + EcoRI in buffer 4
6. plasmid ours A ClaI + EcoRI in buffer 4
File:IGEM 2010-04-09 19hr 47min.jpg
Lane 4 is a ladder, the final lane containing no DNA is the 6th treatment above. Again, none of the plasmids appear to have been cut at all. This strikes me as odd considering that the wt pET28b should contain an EcoRI cut site as well. Maybe my digestions have been unsuccessful for some reason?
If not, it appears redoing the PCR to insert the biobrick insert into the pET28b vector will be necessary.
Retrieved from " http://2010.igem.org/Team:Wisconsin-Madison/notebook/Peter "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
