"
Page
Discussion
View source
History
teams
Log in
 
Team:UANL Mty-Mexico/Safety/Organisms
From 2014.igem.org
Home
Project
Wetlab
Math Model
Human Practices
Safety
Team
Sponsors
Overview
Organisms
Risks
Safety Proposal
Safety
ORGANISMS
BACTERIOPHAGE P1
Organism description
Bacteriophage P1 has been widely used to construct new bacterial strains and was used extensively to map the Escherichia coli chromosome. P1 has served as a model organism for different aspects of phage and biology such as DNA restriction modification, site-specific recombination and plasmid replication (Łobocka MB, et al 2004).
Understanding phage genetics allowed the develop- ment of major basic tools still in use in recombinant DNA technologies. Phage P1 exhibits the classical bacteriophage morphology with an icosahedral head, 220nm long inflexible tail with a complete tube surrounded by a contractile sheath, baseplate and six kinked tail fibers. The icosahedral head contains the phage genome. A variable part(encoded by an invertible segment of P1 DNA) of the tail fibers(1 to 2nm thick) determines the specificity of P1 adsorption on different hosts.
P1, like lambda, made its mark early in molecular biology. The significant capacity of P1 for mediating generalized transduction led promptly to P1 becoming a workhorse of genetic exchange among strains of E. coli, a role it is still playing today. Moreover, because P1 can package slightly more than twice as much DNA as can, and packaging can be efficiently carried out in vitro, P1-based vectors are now in common use for cloning and in vitro packaging of eukaryotic DNA. The recognition that P1 is maintained as a plasmid prophage led to the identification of its plasmid maintenance functions (Łobocka MB, et al 2004).
Properties
Plasmid 40784: BBa_J72113-BBa_J72152
Gene/insert name: Phagemid + GFP
Vector backbone: p15a, CamR
Vector type: Bacterial Expression, SynBio
Bacterial resistance(s): Chloramphenicol
Growth strain(s): DH10B
Growth temperature (℃): 37℃
High or low copy: High Copy
Terms and Licenses: UBMTA Ancillary Agreement for Plasmids Containing FP Materials. 
Comments: For detailed composition information, look up the backbone (BBa_J72113) and insert (BBa_J72152) on the Registry of Standard Biological Parts. There is a G insert at 5860 of depositor's seq, in between the "phagemid" and GFP, T301C, T459C, these do not effect function. Addgene has sequenced a portion of this plasmid for verification. Full plasmid sequence is available only if provided by the depositing laboratory.
Plasmid 40780: BBa_J72110-BBa_J72103
Gene/insert name: AraC-pBad + coi + cin + repL + pacA
Insert size: 4722
Vector backbone: p15a, AmpR, CamR
Vector type: Bacterial Expression, SynBio; Phagemid
Bacterial resistance(s): Ampicillin
Growth strain(s): DH10B
Growth temperature (℃): 37℃
High or low copy: High Copy
Terms and Licenses: UBMTA Ancillary Agreement for Plasmids Containing FP Materials.
Comments: For detailed composition information, look up the backbone (BBa_J72110) and insert (BBa_J72103) on the Registry of Standard Biological Parts. T459C and T301C in AraC, C4096T, deletion T4313, C4425T, G insert at 4912, do not effect function. Addgene has sequenced a portion of this plasmid for verification. Full plasmid sequence is available only if provided by the depositing laboratory.
Potential risks
Although most bacteriophages do not represent a threat to human health (unless they are carrying virulence factors), the use of recombinant viral particles in some instances might raise some biosafety concerns by bringing and potentially disseminating new genetic traits among bacterial populations. Aspects that should be considered are the expected survival of the phage, its multiplication and dissemination in the identified ecosystems, and the anticipated interaction among the phage and the organisms likely to be present in the exposed ecosystems.
The genetic information encoded by the bacteriophage, its ability to transfer its genome among bacterial hosts, its host spectrum as well as its persistence in the environment are properties that should be assessed. From a biosafety point of view, the introduction of new genetic material into the bacterial gene pool may have positive, negative, or neutral outcomes depending on the genetic marker introduced; thus, the nature of this genetic marker is of great importance.
The survival of a bacteriophage outside a host is extremely variable and depends on the nature of the phage itself. It is also highly influenced by surrounding environmental conditions, such as pH, ionic strength, temperature, soil structure, adsorption property, or sunlight.  For the bacteriophage P1 to survive in the environment, they need to infect susceptible bacteria hosts to replicate and propagate themselves. The host range of a bacteriophage, defined by which bacterial strains can be infected, depends on the host cell surface receptor (proteins, lipopolysaccharide, or other cell surface components) recognized by this phage. Many phages are known to be highly specific for their receptors and are therefore characterized by a narrow host range, limiting their infectivity to a single species or to specific bacterial strains within a species.
Phages are obligate parasites, and their use will inevitably involve the manipulation of their bacterial hosts. Consequently, assessing the risk of the activity necessitates the evaluation of the susceptible bacteria involved. Risks associated with their manipulation depend mainly on the nature of the inserted genetic material and the bacterial hosts used to propagate these vectors and should be assessed on a case-by-case basis. In this laboratory where we use non-pathogenic E. coli laboratory strains, in this case, both phage P1 and the bacterial strains do not represent any risk to human health with the correct manipulation techniques, and therefore the risk of such an activity could be considered negligible, also because the genetic material used in this project is not pathogenic, thus is not considered hazardous.
E. coli - m1061 (DH10B derivative)
Organism description
Escherichia coli is a member of the family Enterobacteriaceae and is a Gram-negative rod which can be motile by peritrichous flagella or nonmotile. Escherichia is also a facultative anaerobe which has both a respiratory and a fermentative type of metabolism, and commonly occurs in the intestinal tract of humans and other animals.
E. coli belongs to the taxonomic family known as Enterobacteriaceae, which is one of the best-defined groups of bacteria. The strain E. coli K-12 is a debilitated strain which does not normally colonize the human intestine. It has also been shown to survive poorly in the environment, has a history of safe commercial use, and is not known to have adverse effects on microorganisms or plants. Because of its wide use as a model organism in research in microbial genetics and physiology, and its use in industrial applications, E. coli K-12 is one of the most extensively studied microorganisms.
Properties
Blue/white selection
While DH10B has been classically reported to be galU galK, the preliminary genome sequence for DH10B indicates that DH10B (and by their lineage also TOP10 and any other MC1061 derivatives)
Genome sequence indicates that DH10B is actually deoR+
Streptomycin resistant
Leucine auxotroph
Potential risks
E. coli K-12 is considered an exempt host system under the NIH Guidelines. This microorganism also falls under the Class 1 Containment under the European Federation of Biotechnology guidelines.
No data were available for assessing the release and survival specifically for fermentation facilities using E. coli. Therefore, the potential worker exposures and routine releases to the environment from large-scale, conventional fermentation processes were estimated on information available from eight premanufacture notices submitted to EPA under TSCA Section 5 and from published information collected from non-engineered microorganisms.
These values are based on reasonable worst-case scenarios and typical ranges or values are given for comparison. During fermentation processes, worker exposure is possible during laboratory pipetting, inoculation, sampling, harvesting, extraction, processing and decontamination procedures.
The natural habitat of E. coli is the large bowel of mammals. However, E. coli K-12 has lost the ability to colonize the gut and cannot survive in the bowel for very long. The ability of E. coli to survive under environmental conditions is also limited.
E. coli K-12 is a well-studied bacterium which has been the subject of extensive research in microbial physiology and genetics and commercially exploited for a variety of industrial uses. The natural habitat of the parent species, E. coli, is the large bowel of mammals. E. coli K-12 has a history of safe use. Its derivatives are currently used in a large number of industrial applications, including the production of specialty chemicals.E. coli K12 strains are not likely to pose a risk to human or animal health, to plants, or to other microorganisms.
Health: May cause eye irritation with susceptible persons. May cause skin irritation in susceptible persons. May be harmful by inhalation.May be harmful if swallowed.
In case of contact: Rinse with plenty of water. If symptoms arise, call a physician.
Other organisms
Thank you for visiting our wiki. Follow us in:
Retrieved from " http://2014.igem.org/Team:UANL_Mty-Mexico/Safety/Organisms "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
