<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UANL_Mty-Mexico_project_DNA-Program-Supression"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UANL_Mty-Mexico/project/DNA-Program-Supression">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UANL_Mty-Mexico/project/DNA-Program-Supression&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UANL_Mty-Mexico/project/DNA-Program-Supression&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UANL_Mty-Mexico/project/DNA-Program-Supression&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UANL_Mty-Mexico/project/DNA-Program-Supression" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UANL Mty-Mexico/project/DNA-Program-Supression</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="width:100%; background-color: #FFFF;"><TABLE align="center"><TBODY><TR><TD bgcolor="#000000" align="center" class="Estilo2"><UL class="nav"><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico">Home</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project">Project</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/wetlab">Wetlab</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/MathModel">Math Model</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/Human_Pratices">Human Practices</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/Safety/Overview">Safety</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/team">Team</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/Sponsors">Sponsors</A></LI></UL></TD></TR></TBODY></TABLE></DIV><DIV style="width:100%; background-color: #FFFF;"><TABLE align="center"><TBODY><TR><TD bgcolor="#000000" align="center" class="Estilo2"><UL class="nav"><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project">Overview</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project/abstract">Abstract</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project/DNA-Program-Delivery">DNA/Program Delivery</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project/DNA-Program-Supression">DNA/Program Suppression</A></LI><LI align="center"><A href="https://2014.igem.org/Team:UANL_Mty-Mexico/project/The_Whole_Scheme">The Whole scheme</A></LI></UL></TD></TR></TBODY></TABLE></DIV><TABLE id="title" width="100%"><TBODY><TR><TD><CENTER><DIV class="Estilo6">
Project</DIV></CENTER></TD></TR></TBODY></TABLE><TABLE id="subtitle" width="100%"><TBODY><TR><TD><DIV class="Estilo6"><CENTER>DNA / Program Supression</CENTER></DIV></TD></TR></TBODY></TABLE><TABLE style="background-color: #FFFFFF;" width="100%" id="texto"><TBODY><TR><TD><P align="justify"><B><FONT color="black" size="5px">DNA Specific Deletion </FONT></B></P><P>The DNA, just like a lot of other molecules, suffers from deletion; be it to repair, insert a fragment, recombine or as a defense strategy. There are multiple causes for this, but the one that interest us is due to enzymatic action, among which we find exonucleases, restriction enzymes, and other molecules that will be revised shortly.
</P><P align="justify"><B><FONT color="black" size="5px"> Endonucleases </FONT></B></P><P> Nucleases are enzymes with the ability to fragment DNA through phosphodiester bond ruptures. When the cutting site is in the 5’ or 3' end, it’s called an exonuclease; on the other hand, if it's inside the DNA strand, it´s called an endonuclease. Among endonucleases, restriction enzymes, which can recognize specific DNA sequences, have been of utmost interest in the manipulation of DNA, from polymorphism identification (molecular diagnosis) to the construction of new DNA sequences (genetic engineering). 
</P><P align="justify">
According to their characteristics, they are divided into four types, from I to IV. 
Moreover, each of them has their own specific applications. The most studied and 
used are type II restriction enzymes, because they recognize a specific palindromic 
sequence and they generally inside this sequence. On the other hand, type I 
enzymes cut approximately at a 1000 bp distance, while type III cuts at a 24-26 bp 
distance. Finally, type IV has low specificity and only cuts methylated DNA 
(13). 
</P><P align="justify">
Among type 2 restriction enzymes, there are those who can cut just one strand, 
which is called nicking. They are generally named with an N prefix, for example, 
N.bstSEI (13). 
</P><P align="justify">
Restriction enzymes can recognize symmetric and asymmetric sequences 
(10). One way of classifying those that recognize asymmetric 
sequences is in 5 classes according to their characteristics, 
which are shown in the table.
</P><P align="justify">
Type II restriction enzymes have come to be used so much in the molecular 
biology field, that commercially they are the most exploited. But even among them, 
there can be advantages and disadvantages. For example, if an enzyme has a 
recognition site of a few nucleotides, it is better suited for diagnostic trials than for 
genetic engineering. For this reason the use and research of new enzymes has 
begun, along with the de novo design of others. 
</P><P align="justify">
The mode of action of type II restrictions enzymes can be imitated with artificially 
designed enzymes. For example, the union of proteins and/or peptides that 
recognize certain sequences with others that have the capacity of excising DNA. 
</P><P align="justify">
In our project we plan to use endonucleases, together with polypeptides that 
recognize longer sequences than restriction enzymes, in order to increase the 
specificity. 
</P><P align="justify"><B><FONT color="black" size="5px"> Zinc- Finger Nucleases (ZFN)  </FONT></B></P><P align="justify">
Zinc-finger nucleases are agents that have been used for DNA modification by 
means of the fusion of a zinc finger, designed or preexistent, with the active 
domain of the Fokl enzyme; this Phusion molecule is called “Zinc Finger Nuclease” 
or ZFN (4). They have been used in different organisms, from 
animals to plants (8) with the purpose of modifying them, for 
example, through an integration of complete genes (9). 
</P><P align="justify">
An important aspect to be considered before using this technology, is that in order 
to cut it needs to dimerize with the functional domain of another Fokl (2). That’s why it´s required to design adjacent ZFNs whose Fokl domains 
interact in an intermediate site. There are also other advantages, like its high 
specificity due to the ability to design binding sites of over 18 bp (17) 
and because they normally only cut once they have bound to the specific site 
(18). On the other hand, when the ZFP has not joined its specific 
site, Fokl remains as a monomer even to the 15 µM (3), 
making the appearance of cuts in non-specific sites more difficult. 
</P><P align="justify">
However, it is necessary to mention that some possible disadvantages exist, ones 
that may surface in spite of good planning. One of them is that the effectiveness of 
a ZFN in one species doesn’t it will function in others (--). Another downside may 
be that the designed ZFNs work, but once they homodimerize they cut sites they 
weren’t designed for, becoming toxic for the cell (1). Due to this, 
the designing of new ZFN requires experimentation to assure its correct 
functioning. 
</P><P align="justify">
In our project ZFNs will be used, and to avoid the problems mentioned above, we 
will use ZFNs that have been previously tested in E. coli. This is because our aim 
is to prove the utility of said protein in our project. 
</P><P align="justify"><B><FONT color="black" size="5px"> TALEN (Transcription Activator-Like Effector Nucleases): </FONT></B></P><P align="justify">
There are a number of ways to generate ruptures in double stranded 
DNA, one of them is the technology known as TALEN. They are phusion 
proteins composed of a DNA binding domain called TALE (Transcription 
activator-like effector), that is naturally found in the pathogenic plant 
bacteria Xanthomonas, and the Fok l nuclease domain (1)(2)(3). 
</P><P align="justify">
The TALE domain can be modified in order to bind to a specific site in 
the DNA strand. This modification is based in the change of two amino 
acids in the RVD or repetitive variable Di-residue region (1) (2) inside of 
each TALEN. Two amino acids have been determined to correspond to 
each of the 4 nitrogen bases that make up the DNA strand, this means 
the TALE domain can be directed to practically any sequence (2)(3). 
Each TALE domain consists of approximately 33-35 repeated amino 
acids in the RVD region (1), so each TALEN is able to recognize a site 
specific region of up to 35 base pairs, which highlights its high 
specificity. If a single TALEN were directed to a DNA strand, this would 
just cause a nick that would in no way affect the organism, making our 
whole experimental goal void. In order to solve this, 2 TALEN are going 
to be directed to both strands in opposing sites (Figure 1), separated by 
a 5 base pair spacer, to rupture the DNA molecule completely. The Talen 
technology is classified in the new era genome edition, along with the 
Zinc Finger Nucleases (ZFN) and the CRISP/Cas system (1). 
</P><P align="justify">
The two criteria that led us to select this tool as the primary factor in 
our project were the high specificity of the TALE domain, and the lack of 
literature reporting the use of TALEN technology to generate double 
ruptures in a bacterial model. 
</P><P align="justify"><B><FONT color="black" size="5px"> Intracellular degradation of lineal DNA </FONT></B></P><P align="justify">
When endonucleases break double stranded DNA in the middle of its sequence, 
lineal DNA fragments remain, which are degraded by their previously mentioned 
counterparts, the exonucleases, which are enzymes with nuclease activity that 
fragment DNA by their ends. 
 </P><P align="justify">
In vivo, this mechanism is used to fragment free DNA remaining after phenomena 
like recombination, which consists in the excision of a sequence from the 
chromosome when it is damaged and its substitution with a newly synthesized 
chain. 
</P><P align="justify">
However, the exonuclease system is also used for other purposes in the cell, like 
the degradation of exogeneous DNA, the destruction of DNA after the digestion of 
bacteriophages, virus, lineal plasmids, or RNA degradation. Therefore, the 
presence of exonucleases, its activity, quantity and class will determine the amount 
of time lineal DNA will last and thus, the velocity in which it will be inactivated. 
</P><P align="justify">
The role of these exonucleases in the project is essential, because even if the 
endonucleases can fragment DNA, the coding DNA will remain and the 
reprogramation scheme will fail. 
</P><P align="justify">
In bacterias, this mechanism is well studied, because the group of rec proteins is in 
charge of the exnonuclease activity, associated in Exo complexes. In particular, the 
ExoV nuclease, which is the most powerful known exonuclease, with helicase 
activity and single stranded DNA endonuclease activity, is capable of degrading up 
to 1kb per second and of separating up to 30 kbp per union event. So, even if the 
ATP and calcium concentrations aren´t very high, the system will ensure that lineal 
DNA will be quickly degraded. 
 </P><P align="justify">
On the other hand, we have expression systems or transgenic organisms, like 
yeasts or cultures (especially cultures). And having repairing DNA systems, they 
also require exonucleases. 
</P><P align="justify"><I>PLANTS</I>
Plants in general lack a variety of double stranded DNA exonucleases, but they 
have different helicases (at least 5 homologous of RecQ) and single stranded DNA 
endonucleases. 
</P><P align="justify">
However, a similar protein to hWRN has been described in Arabidopsis thaliana, 
an exonuclease capable of digesting the delayed strand of the double stranded DNA. But it hardly digests single stranded or blunt ended DNA, so specific 
endonucleases capable of generating sticky ends must be made to make this DNA 
degradation in plants successful. Also, if further investigation concludes that plants 
lack an efficient double stranded DNA degradation system, it must be included in 
the RSP system. 
</P><P align="justify">
Other nucleases in plants exist, but they are not well studied, or they´re not present 
in the whole organism. 
</P><P align="justify"><I>YEAST AND OTHER EUKARYOTES</I>
In yeasts and other eukaryotes, the Exol protein functions as part of the 
recombining system, but in plants and bacteria, this protein has a single and 
double stranded desoxyribonuclease activity from 5’ to 3’, as well as a 
desoxyribonuclease activity from 3’ to 5’. Also, several single stranded 
exonucleases as well as endogenous endonucleases have been described. 
</P><P align="justify">
From these information, we can infer that the endogenous DNA degradation 
systems, of the organisms that are going to be used, are capable of digesting the 
single stranded DNA that remains after the endonuclease digestion of the system. 
And even if this is not the case, known exonucleases can be added to the RSP 
systems to make this degradation possible. 
</P><P align="justify"><B><FONT color="black" size="5px"> Bibliography </FONT></B></P><OL><LI>
[1] Beumer, K., Bhattacharyya, G., Bibikova, M., Trautman, J.K. &amp; Carroll, D (2006) 
Efficient gene targeting in Drosophila with zinc-finger nucleases. Genetics, 172, 2391-2403.
</LI><LI>
[2] Bitinaite J, Wah DA, Aggarwal, AK, Schildkraut I (1998) FokI dimerization is required for DNA cleavage. Proc. Natl. Acad. Sci., 95: 10570-10575.
</LI><LI>
[3] Kaczorowski, T, Skowron, P &amp; Podhajska, AJ (1989) Purification and characterization of the FokI restriction endonuclease. Gene, 80: 209-216.
</LI><LI>
[4] Kim YG, Cha J, Chandrasegaran S. (1996) Hybrid restriction enzymes: zinc finger fusions to Fok I cleavage domain. Proc. Natl. Acad. Sci. 93: 1156-1160.
</LI><LI>
[5] Kuzminov A, Stahl FW. (1997) “Stability of linear DNA in recA mutant Escherichia coli cells reflects ongoing chromosomal DNA degradation”. Journal of Bacteriology 179(3): 880-888
</LI><LI>
[6] Li X, Hejna J, Moses RE. (2005) “The yeast Snm1 protein is a DNA 5’-exonuclease”. DNA Repair, 4: 163-170
</LI><LI>
[7] Matsushima R, Tang LY, Zhang Lingang, Yamada H, Twell D, Sakamoto W. 
(2011) “A conserved Mg2+
-dependent exonuclease degrades organelle DNA 
during Arabidopsis pollen development” . The Plant Cell 23:1608-1624.
</LI><LI>
[8] Miller JC, Holmer MC, Wang J, Guschin DY, Lee YL, Rupniewski I, 
Beasejour CM, Waite AJ, Wang NS, Kim KA, Gregory PD, Pabo CO, Rebar 
EJ (2007) An improved zinc fingernuclease architecture for highly specific 
genome editing. Nature Biotechnology, 25(7): 778 785. 
</LI><LI>
[9] Moehle EA, Rock JM, Lee YL, Jouvenot Y, DeKelver RC, Grgory PD, Urnov 
FD, Holmes MC (2007) Targeted gene addition into a specified location in 
the human genome using designed zinc finger nucleases. Proc. Natl. Acad. 
Sci. 104: 3055-3060.
</LI><LI>
[10] Pingoud A, and A. Jeltsch (2001) Structure and function of type II restriction 
endonucleases. Nucleic Acid Research, 29(18): 3705-3727. 
</LI><LI>
[11]Plchova H, Hartung F, Puchta H (2003) “Biochemical characterization of an 
exonuclease from Arabidopsis thaliana reveals similarities to the DNA 
exonuclease of the human Werner syndrome protein”. The Journal of 
Biological Chemistry 278(45): 44128-44138
</LI><LI>
[12]Qiu J, Qian Y, Chen V, Guan MX, Shen B. (1999) “Human exonuclease 1 
functially complements its yeast homologues in DNA recombination, RNA 
primer removal and mutation avoidance”. The Journal of Biological 
Chemistry, 274(25):17893-17900
</LI><LI>
[13]Roberts RJ, M Belfort, T Bestor, AS Bhagwat , TA Bickle , J Bitinaite, RM 
Blumenthal, S Degtyarev , DT Dryden and K Dybvig (2003) A nomenclature for 
restriction enzymes, DNA methyltransferases, homing endonucleases and their 
genes. Nucleic Acid Research, 21: 1805-1815.
</LI><LI>
[14]Roman LJ, Kowalczykowski SC. (1989) “Characterization of the helicase 
activity of the Escherichia coli RecBCD enzyme using a novel helicase 
assay.” Biochemistry, 28: 2863-2873
</LI><LI>
[15]Simmon VF, Lederberg. (1972) “Degradation of bacteriophage lambda 
deoxyribonucleic acid after restriction by Escherichia coli K-12”. Journal of 
Bacteriology, 112(1): 161-169
</LI><LI>
[16]Skarstad K, Boye E. (1993) “Degradation of individual chromosomes in recA 
mutants of Escherichia coli. “ Journal of Bacteriology, 175(17): 5505-5509
</LI><LI>
[17] Urnov F.D., Miller JC, Lee YL, Beansejour CM, Rock JM, Augustus S, Jamieson 
AC, Porteus MH, Gregory PD, Holmes MC (2005) Highly efficient endogenous 
human gene correction using designed zinc-finger nucleases. Nature, 435: 646–
651.
</LI><LI>
[18] Vanamee ES, Santagata S, Aggarwal AK (2001) FokI requires two specific DNA 
sites for
cleavage. J. Mol. Biol., 309, 69–78.
</LI></OL></TD></TR></TBODY></TABLE><DIV><TABLE align="center" style="background-color: #000000" width="100%" height="39"><TBODY><TR><TD><DIV align="center" width="400" class="Estilo5">Thank you for visiting our wiki. Follow us in:</DIV></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UANL_Mty-Mexico/project/DNA-Program-Supression">http://2014.igem.org/Team:UANL_Mty-Mexico/project/DNA-Program-Supression</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UANL_Mty-Mexico/project/DNA-Program-Supression" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UANL_Mty-Mexico/project/DNA-Program-Supression" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UANL_Mty-Mexico/project/DNA-Program-Supression&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UANL_Mty-Mexico/project/DNA-Program-Supression&amp;oldid=396743" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>