<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vanderbilt_Project_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vanderbilt/Project/Background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Vanderbilt iGEM 2015</TITLE></P><DIV class="container"><DIV class="intro-text"><DIV class="intro-heading" style="font-style:normal; font-family:Uncial Antiqua; font-size: 2em;"><FONT id="header_text" color="#000000">Demons in the Code</FONT></DIV></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top" class="index container" style="margin: 0; padding: 0; width: 100%; background-color: #3e0812"><NAV class="navbar navbar-fixed-top navbar-default" role="navigation" style="background-color: #3e0812; padding-top: 14px;"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bottom_menu"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bottom_menu"><UL class="nav navbar-nav"><LI class="dropdown-header no-space">Team</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/About"><FONT color="#7e1d2e">About</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Members"><FONT color="#7e1d2e">Members</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Attributions"><FONT color="#7e1d2e">Attributions</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Sponsors"><FONT color="#7e1d2e">Sponsors</FONT></A></LI></UL><LI class="dropdown-header no-space">Project</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background"><FONT color="#7e1d2e">Background</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence"><FONT color="#7e1d2e">Sequence</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit"><FONT color="#7e1d2e">Circuit</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism"><FONT color="#7e1d2e">Organism</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore"><FONT color="#7e1d2e">Nanopore</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements"><FONT color="#7e1d2e">Achievements</FONT></A></LI><LI class="dropdown-header no-space">Parts</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Part_Collection"><FONT color="#7e1d2e">Part Collection</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Optimized_RFPs"><FONT color="#7e1d2e">Optimized RFPs</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Stable_Circuits"><FONT color="#7e1d2e">Stable Circuits</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Repair Enzymes"><FONT color="#7e1d2e">Repair Enzymes</FONT></A></LI><LI class="dropdown-header no-space">Notebook</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/May"><FONT color="#7e1d2e">May</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/June"><FONT color="#7e1d2e">June</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/July"><FONT color="#7e1d2e">July</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/August"><FONT color="#7e1d2e">August</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/September"><FONT color="#970821">September</FONT></A></LI><LI class="dropdown-header no-space">Practices</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Bioethics"><FONT color="#7e1d2e">Bioethics</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Collaborations"><FONT color="#7e1d2e">Collaborations</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Applications"><FONT color="#7e1d2e">Applications</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Interlab"><FONT color="#7e1d2e">Inter-Lab Study</FONT></A></LI><LI class="dropdown-header no-space">Software</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/Development"><FONT color="#7e1d2e">Development</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/User_Guide"><FONT color="#7e1d2e">User Guide</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/Stats"><FONT color="#7e1d2e">Statistics</FONT></A></LI><LI class="dropdown-header no-space">Modeling</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Sequences"><FONT color="#7e1d2e">Sequences</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Circuits"><FONT color="#7e1d2e">Circuits</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Populations"><FONT color="#7e1d2e">Populations</FONT></A></LI></DIV></DIV></NAV><DIV id="breadcrumb_container" class="container" style="position:fixed; top:88px; z-index:2;"><DIV class="row"><DIV class="btn-group btn-breadcrumb"><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background" class="btn btn-default chosen">Background</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence" class="btn btn-default">Sequence</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit" class="btn btn-default">Circuit</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism" class="btn btn-default">Organism</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore" class="btn btn-default">Nanopore</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements" class="btn btn-default">Achievements</A></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1" style="background-color: #44111a; height: 100%;"><H2> Abstract </H2><DIV class="row"><DIV class="col-md-7"><P>
      Every system that is genetically engineered harbors a potentially fatal vulnerability. The source of life's great diversity - spontaneous mutation - is for the synthetic biologist the source of constant apprehension and risk. The relentlessness of genetic mutation has discouraged previous attempts at ever overcoming this perpetually looming menace. Whether in a multifaceted genetic circuit or a simple protein expression platform, mutation is inevitable, and once it disrupts function, the organism will no longer experience the burden of transgene expression, causing the mutant to outcompete whatever has the intended sequence. Evolution and mutation work hand in hand to select against the maintenance of synthetic DNA sequences. Indeed, the mantra has been that time in the form of mutation and evolution will always find a way to erode and ultimately destroy everything that an engineer builds, no matter how ingeniously it may designed. 
      </P><P>
      This year, here at Vanderbilt iGEM, we are fighting back. We are proposing a novel approach based on rationally designed genomic architectures that promises to offer synthetic biologists unprecedented control over the evolutionary stability of their creations. At the heart of our strategy is an advanced computational algorithm that integrates decades worth of scientific data identify and correct the highly-mutation prone 'hotspots' that lurk in every gene. Our strategy has a strong foundation in a rich literature from the fields of cancer biology and others that have annotated and characterized mutation hotspots for almost every conceivable source of mutagen, from ultraviolet radiation to recombination to polymerase errors. 
      </P><P>
      When combined with synthetic DNA technology, our process becomes a simple and reliable optimization that is universally applicable to any coding gene being expressed in any organism. Our project first demonstrates the power of this rational synthetic gene design strategy by employing several canonical as well as highly original protocols for assaying DNA damage and its effects on the stability of artificial genetic elements. From these techniques, we can quantify everything from the selective subset of mutation types occurring on an <I>in vitro</I> level, up to how mutational loss of function translates at the scale of populations of genetically modified organisms. 
      </P><P>
      To complement our work, we have harnessed our algorithm for use in what is becoming one of the most important tools for engineering biological molecules: directed evolution experiments. Not only can our engineered changes increase evolutionary stability in applications such as transgenic bioreactors, but it can also construct gene sequences that are more prone to mutate, thus accelerating studies into how to use evolutionary selection to produce tailored functional modifications to proteins. 
      </P><P>
      Finally, we have investigated ways to build new genetically modified organisms that exhibit greatly increased resistance to mutation. Combining our sequence-based strategies with the introduction of exogenous genes and removal of endogenous genes has enabled us to produce an expression platform for synthetic genes that not only has enhanced DNA repair mechanisms, but also has an entire artificial pathway introduced for the elimination of mutant strains from a population. 
      </P><P>
      While any single engineered change to reduce mutation may still fail, when our innovative approaches to modulating evolutionary stability are taken in combination, they offer an unprecedented hope for taming evolutionary entropy. More than a victory for synthetic biology, we prove that through rational design principles- exactly what mutation most virulently tries to uproot- and with enough clever innovations, it is possible to defend against what seemed like an inevitability of nature. Score one for engineering.
      </P></DIV></DIV><H2>Importance of Genetic Stability</H2><P>
  The implications of reducing evolutionary potential are far-reaching. All of synthetic biology is impacted by the instability of genes, but there are several areas where it is of particular concern, namely manufacturing, medicine, and biosafety.
  </P><DIV class="row"><DIV class="col-md-7"><P>
    In terms of production, bioreactors are especially vulnerable to gene mutation due to the metabolic stress exhibited by the cells in such an environment. The constant push for maximum production gives an evolutionary advantage to any cells that spontaneously mutated to produce less or no product. An entire incubator could be overtaken by these mutated cells leading to a loss in money, time, and resources. In a medical context where pharmaceutical agents are produced, a mutation can have even worse consequences of creating a deleterious drug that would be harmful, such as antibody-producing strain of cells that undergoes a spontaneous mutation leading to unintended interactions.
    </P><P>
    The advent of gene therapy which allows insertion of genes directly into living cells poses parallel difficulties in terms of ensuring continued function. The cell will not only try to get rid of the foreign DNA because of its associated metabolic load, the sequence will be prone to mutations that either cause a nonfunctional or dysfunctional protein product.
    </P></DIV></DIV><P>
  Similarly, genetically modified organisms that are introduced to the environment have the possibility to escape their intended area as well as transfer their genes to other organisms. To combat this issue, many have developed sophisticated killswitch circuits, but what has not been addressed is the degradation of this safety mechanism which is subject to more evolutionary pressure than other components due to its lethality. Both the circuit and the genes composing it are subject to mutation resulting in inactivation and the spread of GMO [<A href="#orgtarget6">3</A>]. 
  </P><DIV class="row"><DIV class="col-md-7"><P>
    All of these issues are driven by the effect of metabolic load and selective pressures on genes which cause them to either be inactivated or deleted. It has been demonstrated that increasing the metabolic load of the cell decreases its rate of replication which is directly correlated to its Darwinian fitness. Cells that have to produce are less adapted to their environment and are more likely to be outcompeted in a heterogeneous population where other cells are not under the same load []. Moreover, the load of translating active proteins that create their own secondary metabolites compounds the stress on the cell leading to further decreased fitness.
    </P><P>
    The metabolic load itself can eliminate the cells ability to survive, but there is an even greater danger posed by increasing cellular load which is stress-induced mutagenesis. The stress of augmented rate of protein production strains the cell into initiating a global response system that changes gene expression and thereby cell metabolism. This includes pathways that increase mutation potential by up-regulating error prone DNA polymerases and down-regulating error correcting enzymes as well as encouraging the movement of insertion sequences. This mutator state under stressful conditions from increased metabolic load is important for adaptive evolution but is counterproductive to the goals of synthetic biologists attempting for stable expression [].
    </P></DIV></DIV><H2> Strategies to Combat Mutation </H2><P>We took a three-pronged approach to minimize mutation by both repairing mutations that occur and preventing mutagenesis from happening. Our strategies target the three levels at which genetic constructs are implemented in engineered biological systems- the physical DNA sequence, its genomic and circuit context, and lastly the entire organism which is host to the synthetic DNA. </P><H2> Sequence </H2><DIV class="row"><DIV class="col-md-7"><P>
      At the heart of our strategy is an advanced computational algorithm that integrates decade’s worth of scientific data in order to identify and correct the highly-mutation prone ‘hotspots’ that lurk in every gene. Our strategy has a strong foundation in a rich literature from the fields of cancer biology and others that have annotated and characterized mutation hotspots for almost every conceivable source of mutagen, from ultraviolet radiation to recombination to polymerase errors.
      </P><P>
      Our project first demonstrates the power of this rational synthetic gene design strategy by employing several canonical as well as highly original protocols for assaying DNA damage and its effects on the stability of artificial genetic elements. From these techniques, we can quantify everything from the selective subset of mutation types occurring on an <I>in vitro</I> level, up to how mutational loss of function translates at the scale of populations of genetically modified organisms.
      </P><P>
      To complement our work, we have harnessed our algorithm for use in what is becoming one of the most important tools for engineering biological molecules: directed evolution experiments. Not only can our engineered changes increase evolutionary stability in applications such as transgenic bioreactors, but it can also construct gene sequences that are more prone to mutate, thus accelerating studies into how to use evolutionary selection to produce tailored functional modifications to proteins.
      </P></DIV></DIV><H2>Circuit</H2><P>
  Genetic circuits have become the mainstays of synbio as the machinery through which biological processes are executed and controlled. They are crucial in many regulatory networks such as feedback loops and killswitches and are used as simulated logic gates to control cell actions. To this end, it is of great importance that these circuits remain stable however current design guidelines used by the synthetic biology community are especially prone to mutation. There are four major factors of gene circuits which can increase or decrease their evolutionary potential.
  </P><H4>Polycistronic Units</H4><P>
  One method of improving evolutionary stability is the inclusion of a bidirectional promoter (BDP) which concurrently transcribes two monocistronic genes. Currently, prokaryotic promoters are particularly susceptible to mutation however a BDP has the advantage of self-protection by overlapping essential promoter sequences on both strands of the DNA. Not only is the BDP a more robust promoter, having the protein of interest expressed at the same time as antibiotic resistance or an essential endogenous gene guarantees that any mutation in the promoter would eliminate the cell from the population. Initial research indicates that the inclusion of a BDP doubles the evolutionary half-life [<A href="#orgtarget7">7</A>].
  </P><H4>Metabolic Load</H4><DIV class="row"><DIV class="col-md-8"><P>
      Another way of decreasing the rate of mutation is to lessen the metabolic load on the cell. This can be easily done by incorporating weak promoters and ribosome binding sites throughout the circuit lessening the rate of protein expression. Ensuring rapid degradation of protein product also reduces metabolic load although the mechanism by which this is done (degradation tag) actually increases the load by using up cellular proteases. Additionally, plasmid copy-number can have a large influence on metabolic load meaning medium- to low-copy number plasmids are preferable for circuit stability. Studies show that reducing expression 4-fold can increase the evolutionary half-life of the circuit 17-fold [<A href="#orgtarget6">6</A>]. 
      </P><P>
      Following the same reasoning as above, replacing constitutive promoters with inducible ones substantially decreases the stress on the cell. A constitutive promoter constantly engages the cell transcriptional and translational machinery leading to unsustainable protein production. And inducible promoter has the advantage of only increasing the metabolic load for a limited time that is tightly controlled. Empirical research demonstrated that after 60 generations, a circuit under the control of a constitutive promoter completely lost its function but the same circuit under an inducible promoter still had 25% of its original expression [<A href="#orgtarget6">6</A>]. 
      </P></DIV></DIV><H4>Homology</H4><P>
  One of the most crucial things that can easily debilitate a circuit is sequence homology. Homology with endogenous genomic areas, intra-circuit homology of various components, and even among BioBrick scar sites left over from assembly can all have a substantial harmful impact on the gene network [<A href="#orgtarget5">5</A>]. A common practice is to include a double terminator (B0015) at the end of each gene in a circuit, however this creates a lot of homology. So much so that in fact the evolutionary half-life of such circuits is around 7 generations, which is less than 24 hours of growth. Complete elimination of terminator homology gave the identical circuit of an evolutionary half-life of 125 generations, an almost 18-fold change [<A href="#orgtarget6">6</A>].
  </P><DIV class="row"><DIV class="col-md-8"><H4>Visualizing Evolution in Real Time</H4><P>
    Engineering organisms with recombinant DNA puts strain on the metabolism of the organism which can reduce the cell’s fitness. This can be quantified by using VERT (Visualizing Evolution in Real Time), a technique that uses three different fluorescent proteins to investigate the stability dynamics created by the transcriptional interactions of these three genes [<A href="#orgtarget6">6</A>]. In order to create a competition differential, randomized promoters, ribosome binding sites (RBS), and terminators were included to create several unique circuits which would lead to certain proteins having transcriptional and translational advantages. Over time, each circuit has the potential to reach a particular equilibrium point of protein expression giving the cell population a unique color. See image below.
    </P></DIV></DIV><P>
  We harnessed this technique to empirically demonstrate that our circuit optimization algorithms increase circuit stability. In the context of VERT, this means that the cell population would maintain the same rate of expression for all three fluorescent proteins rather than have them vary with respect to one another over time. This was accomplished by optimizing the circuit three different ways. We avoided using strong RBS, replacing them with medium-strength RBS since decreased expression levels have been shown to have significantly higher evolutionary stability. The homology of the circuit was also reduced to minimize the similarities among the promoters, RBS, terminators as well as the fluorescent protein coding regions. Finally, the polycistronic potential of the genetic network was increased by including a bidirectional promoter between the first and second protein ensuring synchronized transcription. Additionally a P2A peptide, an autocatalyzing self-splicing linker, was included between the second and third protein guaranteeing synchronized translation.
  </P><H2>Organism</H2><P>
  We have investigated ways to build new genetically modified strains that exhibit greatly increased resistance to mutation. Combining our sequence-based strategies with the introduction of exogenous genes and removal of endogenous genes has enabled us to produce an expression platform for synthetic genes that not only has enhanced DNA repair mechanisms, but also has an entire artificial pathway introduced for the elimination of mutant strains from a population.
  </P><H4>Repair Enzymes</H4><P>
  The pKIKO (Knock-In-Knock-Out) system allows for the simultaneous replacement of genomic DNA and integration of foreign DNA. This is being used by us to integrate several repair enzymes that will maintain the fidelity of genomic DNA while removing error-prone polymerases endogenous to the cell. Here are a few enzyme that are included for the mutation optimization of the cell:
  </P><DIV class="row"><DIV class="col-md-8"><UL class="org-ul"><LI>T4 PDG (T4 Endonuclease V) is an enzyme which specifically recognizes UV induced cyclobutane pyrimidine photodimers and creates a nick in the DNA strand through its glycosylase and APlyase domains. This enzyme would be able to nick DNA after UV exposure and direct intrinsic cellular machinery to repair it by eliminating the lesion. Introduction of T4 PDG to the cell would increase the cell’s resistance to UV irradiation and lessen the rate of mutation. </LI><LI>FPG is an enzyme which finds 8-oxoguanine base pairs, which are the result of oxidative damage, and excises them. Unlike the previous enzyme, FPG completely removes the site by cutting 5’ and 3’ to the base pair. This generates an apurinic site that can then be easily filled through Watson-Crick base pairing by DNA polymerases. Introduction of FPG to the cell would allow the cell to effectively repair oxidative damage and reduce the rate of mutation due to replication fork stalling. </LI><LI>T7 Endonuclease I is an enzyme which cleaves DNA creating a double-stranded break within 3 basepairs 5’ of the mismatched DNA. In addition to heteroduplexes, the enzyme also cuts cruciform structures, Holliday junctions, and to a lesser extent, nicked dsDNA. Introduction of this enzyme would largely prevent DNA replication errors and increase the sequence of fidelity making the cell more resistant to spontaneous mutations.</LI><LI>Nuclease S1 is an enzyme that it primarily cleaves nicked dsDNA as well as DNA that has gaps, mismatches, or loops, similarly to T7. This enzyme could be introduced to the cell and in conjunction with the previous three which generate nicks, would ensure a complete double stranded break at every damaged location.</LI><LI>Other enzymes such as AlkB for removal of methyl groups and MGMT for repairing alkylated basepairs are just a few examples of other Base Excision Repair enzymes. The possibilities are endless for the combination of these enzymes in a cell to minimize evolutionary potential.</LI></UL><P>
  In addition to simply adding in the enzymes by themselves or in some sort of combination, we have devised a system for a more precise repair mechanism. In the case of T4 PDG, the enzyme could be put under the control of an umuDC promoter which is UV dependent. This would allow the cell to have a specific response to irradiation which is better than the constitutively expressing the protein for reasons aforementioned, mainly the increase in metabolic load. A similar paradigm could be developed for other error correction enzymes and promoters.
  </P></DIV></DIV><H4>Incorruptible Cell</H4><P>
  Finally, a prokaryote that employs all these enzymes, in addition to the mechanism of on-homologous end joining knocked out, has the opportunity to be the ultimate killswitch. This “Incorruptible Cell” would employ enzymes to identify mutated base pairs and generate lethal double-stranded breaks . Because the cell is engineered to be incapable of repairing double-strand breaks, if any the bases in its genome are ever mutagenized in a way that these enzymes can recognize, the cell would effectively commit suicide instead of becoming a mutant.
  </P><H2>References</H2><OL class="org-ol"><LI><A id="orgtarget1" href="http://www.ncbi.nlm.nih.gov/pubmed/25849635">Ceroni F, Algar R, Stan GB, Ellis T. Quantifying cellular capacity identifies gene expression designs with reduced burden. Nat Methods. 2015;12(5):415-8.</A></LI><LI><A id="orgtarget2" href="http://www.ncbi.nlm.nih.gov/pubmed/17917873">
      Foster PL. Stress-induced mutagenesis in bacteria. Crit Rev Biochem Mol Biol. 2007;42(5):373-97..</A></LI><LI><A id="orgtarget3" href="http://www.ncbi.nlm.nih.gov/pubmed/23355834">Moe-behrens GH, Davis R, Haynes KA. Preparing synthetic biology for the world. Front Microbiol. 2013;4:5.</A></LI><LI><A id="orgtarget4" href="http://www.ncbi.nlm.nih.gov/pubmed/24556867">Renda BA, Hammerling MJ, Barrick JE. Engineering reduced evolutionary potential for synthetic biology. Mol Biosyst. 2014;10(7):1668-78.</A></LI><LI><A id="orgtarget5" href="http://www.ncbi.nlm.nih.gov/pubmed/21040586">Sleight SC, Bartley BA, Lieviant JA, Sauro HM. Designing and engineering evolutionary robust genetic circuits. J Biol. Engineering 2010;4:12.</A></LI><LI><A id="orgtarget6" href="http://www.ncbi.nlm.nih.gov/pubmed/24004180">Sleight SC, Sauro HM. Visualization of evolutionary stability dynamics and competitive fitness of Escherichia coli engineered with randomized multigene circuits. ACS Synth Biol. 2013;2(9):519-28.</A></LI><LI><A id="orgtarget7" href="http://www.ncbi.nlm.nih.gov/pubmed/23093602">Yang S, Sleight SC, Sauro HM. Rationally designed bidirectional promoter improves the evolutionary stability of synthetic genetic circuits. Nucleic Acids Res. 2013;41(1):e33.</A></LI></OL></DIV></DIV><DIV class="row seven-cols" style="margin: 0; border: 0; width: 100%; background-color: #3e0812; font-size: 0.8em;"><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Team</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Team/About"><FONT color="#9e515e">About</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Members"><FONT color="#9e515e">Members</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Attributes"><FONT color="#9e515e">Attributes</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Sponsors"><FONT color="#9e515e">Sponsors</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Project</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background"><FONT color="#9e515e">Background</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence"><FONT color="#9e515e">Sequence</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit"><FONT color="#9e515e">Circuit</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism"><FONT color="#9e515e">Organism</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore"><FONT color="#9e515e">Nanopore</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements"><FONT color="#9e515e">Achievements</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Parts</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Part_Collection"><FONT color="#9e515e">Part Collection</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Optimized_RFPs"><FONT color="#9e515e">Optimized RFPs</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Visualizing_Evolution"><FONT color="#9e515e">Visualizing Evolution</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Repair Enzymes"><FONT color="#9e515e">Repair Enzymes</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Notebook</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/May"><FONT color="#9e515e">May</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/June"><FONT color="#9e515e">June</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/July"><FONT color="#9e515e">July</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/August"><FONT color="#9e515e">August</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/September"><FONT color="#9e515e">September</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Practices</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Safety"><FONT color="#9e515e">Safety</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Collaborations"><FONT color="#9e515e">Collaborations</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Real_World_Applications"><FONT color="#9e515e">Applications</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Interlab"><FONT color="#9e515e">Interlab</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Software</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Software/Development"><FONT color="#9e515e">Development</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Software/User_Guide"><FONT color="#9e515e">User Guide</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Software/Stats"><FONT color="#9e515e">Statistics</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Modeling</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Sequences"><FONT color="#9e515e">Sequences</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Circuits"><FONT color="#9e515e">Circuits</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Populations"><FONT color="#9e515e">Populations</FONT></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>