Team:Vanderbilt/Project/Organism
Vanderbilt iGEM 2015
Demons in the Code
Toggle navigation
Team
About
Members
Attributions
Sponsors
Project
Background
Sequence
Circuit
Organism
Nanopore
Achievements
Parts
Part Collection
Optimized RFPs
Stable Circuits
Repair Enzymes
Notebook
May
June
July
August
September
Practices
Bioethics
Collaborations
Applications
Inter-Lab Study
Software
Development
User Guide
Statistics
Modeling
Sequences
Circuits
Populations
Background Sequence Circuit Organism Nanopore Achievements
Long-Term in vivo Gene Stability
The first question that our team addressed was a very basic one, testing the basic assumptions behind our project. We hypothesized that mutant cells would no longer be subject to the metabolic load of recombinant protein production or the toxicity of its activity. To validate this hypothesis, we conducted evolutionary competition assays. The set up was purposefully simple, with the intent of re-creating realistic competition conditions. To identical starting populations of cells producing RFP on a high-copy plasmid, as is common for recombinant protein production, we added a set amount of cells with plasmids that were almost identical, except that they lacked a functional promoter. Initially we examined a range of starting values for mutant cells- with cultures ranging from having a majority of mutants, to only 1% of the population being mutant.
As indicated on the graph to the right , there is a steep decrease in RFP fluorescence in all of the cultures with mutant cells present. Most strikingly, after only three days, all of the populations with mutant cells- even the ones that began with only 1% mutant cells- had their RFP intensity reduced to effectively zero. At the rate of growth of our experiments, this means that within approximately 30 generations, the selective pressure favoring these mutants is intense enough to cause them to completely overtake the original population.
Based on our initial results with mathematical modeling, we were curious whether or not one of the core assumptions behind most models of population evolution was correct- namely, that the selective advantage of a competitor cell is constant. We put this assumption to experimental rigor by looking at the rate of fluorescence decrease (which corresponds to the spread of "mutant" cells throughout the population) after 24 hours of the competition experiment indicated earlier. Populations that began with the fewest number of mutants experience the greatest proliferation of mutants, whereas populations already saturated with mutants show slower mutant proliferation. In response to this finding, we invented a revision to the established equations of population growth that factor in this density-dependence.
After demonstrating the principle that mutant cells will rapidly overtake the population, we next moved in to mutagenizing cell cultures with UV irradiation. We exposed identical cultures to varying amounts of ultraviolet radiation, from 1 J/M2 to 200 J/M2. These values were chosen based on published literature values for irradiation experiments, the maximum and minimum of which are around 1 and 200 J/ M2 respectively. After several days of re-starting and measuring the cultures, we found a general shift toward lower fluorescence values for highly irradiated cultures, and less change for less irradiated samples. However, the values are fairly variable, and one unirradiated control had a noticeable decrease in fluorescence (although still less than any irradiated sample).
We modified our approach, by using a low-copy number plasmid and higher UV dosages (500 J/M2 ). The dose was determined by seeing the maximum dose at which E. coli would no longer survive, which was around 10 J/M2 , and dividing it in half. The low-copy plasmid was chosen based on the concern that the evolutionary selective pressure for a cell that has one out of a hundred high-copy plasmids mutated may be too low, whereas if it were one out of ten low-copy plasmids the net effect could be greater. Despite these changes, we had similar results, as shown.
DNA Repair Enzymes
While our sequence-optimization strategy may reduce the frequency of mutation, it will never be able to eliminate mutations entirely. Instead, we needed modifications at the organismal level that could correct any of the mutations that a gene may incur. We selected four DNA repair enzymes, taken from across multiple species, to express in E. coli . These enzymes and their functions are listed on our Background page.
To protect the repair enzymes themselves against disruptive mutation, we processed their amino acid sequences through our mutation-optimization software. We synthesized these genes and used BioBrick assembly to turn them into parts for the Registry. We confirmed all our assemblies first by diagnostic digests, then by DNA sequencing.
To begin testing what effect these enzymes have on incidences of mutation when expressed, we added a strong IPTG-inducible promoter to each by BioBrick assembly. These assemblies we confirmed in the same way, with diagnostic digests followed by sequencing. To confirm that protein was being expressed, we grew up cultures of transformed bacteria and lysed them after induction with IPTG. We used SDS-PAGE to run samples of lysate from transformed cells with the expression plasmids and cells with a plasmid containing no promoter. The SDS-PAGE gels were stained by coomassie stain and imaged.
Our coomassie did not reveal any readily apparent bands that appeared in the protein-expressing cells but not in the controls without a promoter. However, the overall amount of protein loaded was very high and caused significant smearing instead of distinct protein bands. This was especially pronounced in the regions of each protein lane that were approximately at the expected molecular weight for our enzymes (as determined by a standard marker). We are planning to repeat this experiment using diluted samples of lysate to better resolve bands that may correspond to our repair enzymes.
Team
About
Members
Attributes
Sponsors
Project
Background
Sequence
Circuit
Organism
Nanopore
Achievements
Parts
Part Collection
Optimized RFPs
Visualizing Evolution
Repair Enzymes
Notebook
May
June
July
August
September
Practices
Safety
Collaborations
Applications
Interlab
Software
Development
User Guide
Statistics
Modeling
Sequences
Circuits
Populations
