Team:Vanderbilt/Project/Circuit
Vanderbilt iGEM 2015
Demons in the Code
Toggle navigation
Team
About
Members
Attributions
Sponsors
Project
Background
Sequence
Circuit
Organism
Nanopore
Achievements
Parts
Part Collection
Optimized RFPs
Stable Circuits
Repair Enzymes
Notebook
May
June
July
August
September
Practices
Bioethics
Collaborations
Applications
Inter-Lab Study
Software
Development
User Guide
Statistics
Modeling
Sequences
Circuits
Populations
Background Sequence Circuit Organism Nanopore Achievements
Homology and Recombination
Homologous elements in a genetic circuit pose high risks for destabilizing genomic  rearrangements. Unlike sources of genomic instability that primarily generate point mutations, which in many circumstances may not produce meaningful functional changes, these rearrangements will typically generate considerable functional consequences. This issue of recombination has, among other challenges, posed a barrier to implementing redundancy as a means of decreasing the probability of loss of function. Were it possible to stably introduce two copies of a given gene, the probability for that gene to be rendered nonfunctional would be greatly reduced, by a factor of that probability squared.
One function of our circuit optimization program is to eliminate the amount of homology between two sequences. We were interested in the possibility of using this optimization strategy to minimize homology to the point where it may become possible to integrate multiple copies of certain genes as a viable strategy to increase their evolutionary longevity. Using a set of repair endonucleases (T7 Endonuclease, for use with our "Incorruptible Cell") as our model, we sought to develop a methodology for quantifying how well our homology optimization reduces rates of recombination.
Using our algorithm, we were able to generate a second copy of the T7 repair enzyme that has the number homologies greater than five bases reduced by 78.8% compared to the alternative approach of repeating the same sequence twice. To quantify the effect this optimization has on recombination, we placed our two T7 sequences flanking a portion of the yeast genome including the ura3 gene. One version would have the same native T7 sequence repeated at both ends, while the other would have the native sequence with the homology-optimized sequence. Once integrated into the genome of uracil auxotrophic yeast, if a recombination event occurred between the two T7 sequences, it would excise ura3 as a byproduct. Selection on 5-FOA media, which is lethal to yeast expressing uracil, would then produce a colony for every cell in which recombination occurred.
The sequences with ura3 and T7 were first extracted and modified by extension PCR. Following extraction, the PCR fragments were again amplified by extension PCR, in the process introducing homology regions for Gibson and assembly and gal1 homology arms for integration into the yeast genome. Following Gibson assembly, the three fragments (two T7 and one ura3) were placed into a shuttle vector. Transformed cells with the correct ampicillin resistance were found for both the assembly of the control cassette and the homology optimized cassette. Next, these assembled vectors will be purified and introduced into yeast for experiments.
Bidirectional Promoters
In parallel, we have been developing a bidirectional promoter system. Although our methods are effective at lowering the mutation rates of genes, the promoters of these genes remain vulnerable targets for mutations. We paired our bidirectional promoter, based on a modified IPTG-inducible promoter developed by Yang et al 2012, with an ampicillin gene transcribed in the reverse direction. This pairing allows us to select against mutations in the promoter sequence, by making any promoter mutation lethal when the cells are in the presence of ampicillin.
We cloned a BioBrick version of this bidirectional promoter, and confirmed our assembly by sequencing. We conducted a second round of assembly to insert a reporter gene with an RBS on one end of the bidirectional promoter to confirm that the promoter was causing expression. However, we were unable to insert the RBS when we attempted our assembly. We are in the process of extracting another reporter gene from the Registry that includes an RBS in the part .
We also constructed and submitted a BioBrick that contains a bidirectional promoter and an ampicillin resistance gene. After confirming our part by sequencing, we conducted a test to determine if the plasmid conferred antibiotic resistance. Cells transformed with the plasmid containing the bidirectional promoter to express the beta lactamase antibiotic-resistance gene were able to grow in media containing ampicillin, while control cells that had only the intermediate plasmids we used for construction were unable to grow. This both validated our BioBrick part for selecting against promoter mutations, and demonstrated that the bidirectional promoter we synthesized is able to induce transcription, at least in the reverse direction.
VERT Circuit
To integrate our circuit optimization strategies, we used Visualizing Evolution in Real Time (VERT). We incorporated the various stability optimization strategies that our team has developed into a single circuit with three fluorescent proteins. It has bidirectional promoters and 3A peptides to reduce the number of sites for promoter mutations, and all of its gene sequences have been analyzed by our software to minimize the amount of sequence homology between each of the genes. Comparison to a control circuit, which represents the most intuitive, and likely most common, way of assembling three genes then informs us of how much our circuit-stabilizing strategies have reduced the circuit's rate of evolutionary change.
We synthesized the components of the optimized circuit into three parts, which were then assembled into pSB1C3 using Gibson assembly. A colony containing a correctly-sized construct was found by screening and purified. For the control circuit, based on traditional circuit design principles, we extracted genes from parts in the Registry using extension PCR, in preparation for Gibson assembly. For our control circuit, we have begun extracting parts of the registry to combine the control circuit's three reporter genes into a single vector, making it as similar as possible to the circuit-optimized version except from the experimental modifications that we are testing.
Team
About
Members
Attributes
Sponsors
Project
Background
Sequence
Circuit
Organism
Nanopore
Achievements
Parts
Part Collection
Optimized RFPs
Visualizing Evolution
Repair Enzymes
Notebook
May
June
July
August
September
Practices
Safety
Collaborations
Applications
Interlab
Software
Development
User Guide
Statistics
Modeling
Sequences
Circuits
Populations
