<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vanderbilt_Project_Sequence skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vanderbilt/Project/Sequence</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Vanderbilt iGEM 2015</TITLE></P><DIV class="container"><DIV class="intro-text"><DIV class="intro-heading" style="font-style:normal; font-family:Uncial Antiqua; font-size: 2em;"><FONT id="header_text" color="#000000">Demons in the Code</FONT></DIV></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top" class="index container" style="margin: 0; padding: 0; width: 100%; background-color: #3e0812"><NAV class="navbar navbar-fixed-top navbar-default" role="navigation" style="background-color: #3e0812; padding-top: 14px;"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bottom_menu"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bottom_menu"><UL class="nav navbar-nav"><LI class="dropdown-header no-space">Team</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/About"><FONT color="#7e1d2e">About</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Members"><FONT color="#7e1d2e">Members</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Attributions"><FONT color="#7e1d2e">Attributions</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Team/Sponsors"><FONT color="#7e1d2e">Sponsors</FONT></A></LI></UL><LI class="dropdown-header no-space">Project</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background"><FONT color="#7e1d2e">Background</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence"><FONT color="#7e1d2e">Sequence</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit"><FONT color="#7e1d2e">Circuit</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism"><FONT color="#7e1d2e">Organism</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore"><FONT color="#7e1d2e">Nanopore</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements"><FONT color="#7e1d2e">Achievements</FONT></A></LI><LI class="dropdown-header no-space">Parts</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Part_Collection"><FONT color="#7e1d2e">Part Collection</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Optimized_RFPs"><FONT color="#7e1d2e">Optimized RFPs</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Stable_Circuits"><FONT color="#7e1d2e">Stable Circuits</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Repair Enzymes"><FONT color="#7e1d2e">Repair Enzymes</FONT></A></LI><LI class="dropdown-header no-space">Notebook</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/May"><FONT color="#7e1d2e">May</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/June"><FONT color="#7e1d2e">June</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/July"><FONT color="#7e1d2e">July</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/August"><FONT color="#7e1d2e">August</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/September"><FONT color="#970821">September</FONT></A></LI><LI class="dropdown-header no-space">Practices</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Bioethics"><FONT color="#7e1d2e">Bioethics</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Collaborations"><FONT color="#7e1d2e">Collaborations</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Applications"><FONT color="#7e1d2e">Applications</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Interlab"><FONT color="#7e1d2e">Inter-Lab Study</FONT></A></LI><LI class="dropdown-header no-space">Software</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/Development"><FONT color="#7e1d2e">Development</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/User_Guide"><FONT color="#7e1d2e">User Guide</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Software/Stats"><FONT color="#7e1d2e">Statistics</FONT></A></LI><LI class="dropdown-header no-space">Modeling</LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Sequences"><FONT color="#7e1d2e">Sequences</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Circuits"><FONT color="#7e1d2e">Circuits</FONT></A></LI><LI><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Populations"><FONT color="#7e1d2e">Populations</FONT></A></LI></DIV></DIV></NAV><DIV id="breadcrumb_container" class="container" style="position:fixed; top:88px; z-index:2;"><DIV class="row"><DIV class="btn-group btn-breadcrumb"><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background" class="btn btn-default">Background</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence" class="btn btn-default chosen">Sequence</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit" class="btn btn-default">Circuit</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism" class="btn btn-default">Organism</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore" class="btn btn-default">Nanopore</A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements" class="btn btn-default">Achievements</A></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1" style="background-color: #44111a; height: 100%;"><H2><I>In vitro</I> Assay Methodology</H2><DIV class="row"><DIV class="col-md-8"><P>To truly demonstrate that our mutation optimization procedure reduces the physical occurrence of DNA mutations, we had to take an in vitro approach. We needed methods for visualizing the incidence of mutagenic DNA lesions on purified samples of DNA. Ideally, these methods would be compatible with a wide variety of DNA lesions, and would have specificity to the particular lesion that we are trying to quantify. 
</P><P>After searching through the published literature, we were unable to find any single protocol that met all of these criteria. However, we did identify several that each had components of our ideal assay. Therefore, we decided to adapt protocols from several sources in order to develop a reliable, high-resolution method for quantifying virtually any form of DNA damage.</P><P>That starting point for our approach was a class of DNA repair enzymes. What these enzymes have in common is that they recognize certain forms of mutagenic lesions on a DNA helix, where they then hydrolyze the DNA strand's sugar-phosphate backbone (in addition to other potential activities, such as excising entire bases). These enzymes were excellently suited to give us the lesion specificity and targeting diversity that we were looking for. By selecting different repair enzymes, we could tailor the range of substrates that are targeted: for example, NEIL1 recognizes most forms of oxidized DNA bases, while FPG is restricted to 8-oxoguanine. There are enzymes for oxidation, ultraviolet radiation, alkylation, and more, which spans effectively all of the hotspots that we were interested in studying. </P><P>To visualize and quantify the enzymatically-processed DNA samples, we investigated multiple options. One of these was looking at the conversion of plasmids from their usual supercoiled confirmation to a nicked confirmation. Another utilizes alkaline denaturing electrophoresis to separate the DNA into single strands, which then can be assayed for fragmentation, which appears as long smears on a gel. We have also considered approaches that utilize S1 Nuclease to convert single strand breaks into double strand breaks. </P></DIV></DIV><H2>Irradiated DNA Lesion Experiments</H2><P>Our first set of experiments involved pairing alkaline denaturing electrophoresis with the repair enzyme T4 Pyrimidine Dimer Glycosylase (T4 PDG) to reveal whether or sequences with UV mutation sites removed actually exhibit greater resistance to damage by UV irradiation. Because our protocol was highly adapted from a number of sources, we first had to demonstrate the assay produced consistent results under controlled conditions. A large amount of troubleshooting was necessary to optimize both the conditions of enzymatic processing, alkaline denaturation of the DNA samples, running conditions for the alkaline gel, and neutralization and staining conditions for generating images.</P><P>After considerable modifications, we managed to develop a protocol that consistantly
1. Shows little or no fragmented DNA with samples that are not irradiated and not processed with enzyme
2. Has no significant difference between unirradiated and unprocessed control samples and samples that are either irradiated without enzymatic processing or enzymatically processed without irradiation
3. Is responsive to dosage, meaning samples with greater irradiation show greater fragmentation
</P><P>
As the images and quantification above demonstrate, our final protocol satisfies all of these conditions, thus establishing it as a effective method for quantifying UV-induced DNA damage. The linearity seen in response to increasing UV dosage further allows us to state that differences in measured fragmentations between control and mutation-optimized RFP reflect reductions in mutagenic lesions that are equivalently linear. By that logic, an optimized sequence that produces a ten percent reduction in fragmentation intensity relative to control should have ten percent fewer mutagenic lesions than the control. </P><P>
During our early experiments we used a UV-mutation optimized sequence, labeled RFPyy, with 12% of its UV mutation sites eliminated. Quantification of RFPyy's rate of fragmentation relative to the E1010 RFP control sequence revealed a trending to significant decrease in UV-induced fragmentation on the mutation optimized sequence (two-sample t-test: n=7, t*two tailed=2.57, p=0.13). The results are depicted graphically as bar graphs with standard error bars, alongside dot plots of raw data. For each data point, the &quot;background&quot; seen in the no enzyme negative control lane was subtracted from the fragmentation smear intensity to correct for differences in amounts loaded, fragmentation of the stock sample, and other factors. Two representative alkaline gel images are also shown, in greyscale and inverted. At 1000 mJ/cm2, it was found that non-specific DNA shearing occurs at such high magnitudes that there is no longer appreciable differences between enzyme treated and untreated samples. The graph shown is for assays conducted at the 500 mJ/cm2 UV dosage.
</P><P>
Although statistical analysis of RFPyy failed to produce statistical significance according to the conventional definitions of alpha levels, the modest 12% reduction in mutation-prone sites may not be sufficient to produce significant results because of the intrinsic variability of the technique, which combines variation from irradiation, enzymatic treatment, denaturation, electrophoresis, and staining. Despite all of these factors and the variability they introduce, we nevertheless observe a strong trend that may be capable of significance with further replication.</P><P>
While these assays were ongoing, the mutation optimization algorithm was being constantly improved. One large improvement that was made was incorporating more precise values for the relate mutagenicity of each type of UV hotspot relative to each other. Our updated algorithm is now capable of optimizing RFP twice as well as had been done with RFPyy. Our improved optimized RFP has a 35% reduction in UV-prone sites, which, based on how strong our results were with RFPyy, would almost certainly produce a result that is both statistically significant and of considerable magnitude. </P><H2>Oxidized DNA Lesion Experiments</H2><P>
To evaluate our technique's performance with respect to oxidation sites, we primarily used the conversion of supercoiled plasmid to nicked plasmid as our form of visualization. An alkaline gel assay was run, and produced results roughly in accordance with the data found by the plasmid confirmation method. However, the alkaline gel assay requires a minimum of ten hours to complete, and requires specialized equipment and extra care to set up. </P><DIV class="row"><DIV class="col-md-8"><P>
Our optimized RFP for these experiments had a 49.5% reduction in oxidation hotspots. Our repair enzyme was Formamidopyrimidine DNA glycosylase (FPG), which is specific in its activity for the 8-oxoguanine DNA lesions that those hotspots are correlated to. For oxidizing our DNA in vitro, we initially tried methylene blue. The advantage of methylene blue is that it releases oxidizing radicals in response to exposure to light, which would allow precisely timed oxidation exposures. Unfortunately, we found the presence of methylene blue interfered with electrophoresis in our samples, so we transitioned to a protocol based on the Fenton reaction, which utilizes trace amounts of iron (II) particles to catalyze the conversion of hydrogen peroxide (H2O2) into oxidizing radical species. </P><P>
Just as with our highly-modified assay for detecting UV DNA damage, we had to demonstrate the same three conditions were true for the oxidation assay we developed. At a dose of 50 μM 
H2O2, we had very good results, showing the conversion of supercoiled plasmid to a nicked form. As with extreme amounts of UV irradiation, we found 500 μM H2O2 will produce damage severe enough to convert plasmids to their nicked form without FPG incubation.  </P></DIV></DIV><P>
After validating our protocol, we began experimentation on our mutation-optimized sequences. With our RFP sequences, both control and optimized, we saw a third band present that was not visible in the original validation of the assay. Based on established information on the relative motilities of relaxed, nicked, and supercoiled DNA, we theorized that the newly-appeared top band represented plasmid in a relaxed conformation. Since the RFP plasmids were extracted in parallel, but the plasmid for the initial validation was not, variation in the negative control case of no mutagen and no enzyme would be expected. In line with this assessment, the relaxed plasmids behave as expected: they exactly parallel the intensity of the supercoiled plasmid, being converted like the supercoiled plasmid is upon oxidation and enzymatic treatment. </P><P>
For quantification, we quantified the ratio of the intensity of the nicked band to the supercoiled band. The relaxed band was not included for the data reported below, although because it parallels the supercoiled bands, factoring it in does not greatly alter our results. Samples with less oxidation would have a lower ratio, since they would retain more supercoiled plasmid while having less plasmid converted to nicked. The same trends are seen when quantification is instead done looking only at decreases in the intensity of the supercoiled conformation.</P><P>
Due predominantly to a low sample size, the reduction found in the RFP with minimized oxidation hotspots was not significant (two-sample t-test: n=4, t*two tailed=4.30, p=0.29). Nevertheless, there is replication of a trend, which could yield more conclusive results with more testing. As the promising results with the UV-optimized sequence demonstrates the principle behind reducing mutation with rational, algorithmic strategies, there is reason to be hopefully the same principles will apply to oxidation hotspots as well, as the previously noted trend suggests.</P><P>
Given the large decrease in oxidation sites in the optimized RFP, it is still necessary to conclude that the observed improvement in sequence stability with oxidation was lower than seen in the ultraviolet experiments. One probable explanation for this is that different hotspots contribute differently to overall mutagenic potential. For oxidation hotspots, each has a certain elevated probability of undergoing mutation, but those probabilities are likely lower than the corresponding probabilities for UV dimer hotspots. The different technique used also would explain part of the discrepancy. </P><P>
In total, these experimental results strongly suggest that the optimizations that we make at the sequence level to reduce the number of mutation hotspots do indeed translate to lowered rates of mutation at a biochemical level. Despite the intrinsic variation of these techniques, they were still able to consistently detect  the occurrence of DNA damage, and were at least sensitive enough to produce apparent trends in mutation reduction following optimization. As these protocols were only just developed in their current form by our team, there is undoubtedly opportunities to optimize these protocols further. Given the advantages of these techniques to detect DNA damage, as previously outlined, there is both the means and the incentive to develop them further as research tools for both testing our own hypothesis about various forms of mutation-optimization as well as for other researchers more generally. </P><H2>Optimized Sequence Expression</H2><P>
Because the current paradigm for optimizing synthetic gene sequences is codon adaptation index (CAI), which is purported to correlate to expression levels, one of our team's main priorities was demonstrating that our novel optimization method based on lowering mutation did not interfere with gene expression. While the initial form of RFP that we generated to have resistance to UV mutation was confirmed to still produce RFP, there was a several-fold decrease in expression levels relative to the CAI-optimized control. This was despite the fact that the sequence was optimized to select the codon with a higher CAI associated with it when neither of two or more codons had fewer hotspots than the other. In response to this setback, we made substantial revisions to the algorithm, this time factoring in &quot;limiting codons&quot; instead of standard CAI values. These limiting codons are from tRNAs of exceptionally low abundance in an organism, although varying definitions exist for how low the abundance must be for this classification. </P><DIV class="row"><DIV class="col-md-8"><P>
Using this new approach based on limiting codons instead of CAI, we were able to increase the expression of our UV-optimized by 200%. Upon further research and refinement of our definition of what constitutes a limiting codon, in addition to re-introducing a CAI-like metric to the mutation optimization algorithm's decision making process, we were able to further significantly increase expression, reaching levels close to but slightly below the RFP E1010 sequence on the Registry. Through our collaboration with the iGEM team at the University of Washington St. Louis, we have achieved an independent replication of our results, under a range of IPTG inducer concentrations. </P><P>
These findings have vastly improved the expression of our optimized genes, and in the process has revealed some insights into the correlation of metrics like CAI and limiting codons to actual gene expression. With further refinement of our algorithm's criteria, it is reasonable to expect that we will be able to continue this ever-improving trend, and constantly produce genes with comparable expression to ones optimized by traditional methods. </P><P>
Interestingly, the differences in expression between our most recent iterations of mutation-optimized genes and their traditionally optimized versions are not directly attributable to CAI. When CAI scores are calculated for mutation-optimized and traditionally optimized sequences, they rarely differ. As an example, the figure below shows a graphical representation of the CAI for comparison with our mutation-optimized sequences. 
</P></DIV></DIV></DIV></DIV><DIV class="row seven-cols" style="margin: 0; border: 0; width: 100%; background-color: #3e0812; font-size: 0.8em;"><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Team</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Team/About"><FONT color="#9e515e">About</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Members"><FONT color="#9e515e">Members</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Attributes"><FONT color="#9e515e">Attributes</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Team/Sponsors"><FONT color="#9e515e">Sponsors</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Project</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Project/Background"><FONT color="#9e515e">Background</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Sequence"><FONT color="#9e515e">Sequence</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Circuit"><FONT color="#9e515e">Circuit</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Organism"><FONT color="#9e515e">Organism</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Nanopore"><FONT color="#9e515e">Nanopore</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Project/Achievements"><FONT color="#9e515e">Achievements</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Parts</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Part_Collection"><FONT color="#9e515e">Part Collection</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Optimized_RFPs"><FONT color="#9e515e">Optimized RFPs</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Visualizing_Evolution"><FONT color="#9e515e">Visualizing Evolution</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Parts/Repair Enzymes"><FONT color="#9e515e">Repair Enzymes</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Notebook</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/May"><FONT color="#9e515e">May</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/June"><FONT color="#9e515e">June</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/July"><FONT color="#9e515e">July</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/August"><FONT color="#9e515e">August</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Notebook/September"><FONT color="#9e515e">September</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Practices</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Safety"><FONT color="#9e515e">Safety</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Collaborations"><FONT color="#9e515e">Collaborations</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Real_World_Applications"><FONT color="#9e515e">Applications</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Practices/Interlab"><FONT color="#9e515e">Interlab</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Software</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Software/Development"><FONT color="#9e515e">Development</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Software/User_Guide"><FONT color="#9e515e">User Guide</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Software/Stats"><FONT color="#9e515e">Statistics</FONT></A></DIV><DIV class="col-md-1" style="height:100%; text-align: center;"><H4><FONT color="#7e1d2e">Modeling</FONT></H4><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Sequences"><FONT color="#9e515e">Sequences</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Circuits"><FONT color="#9e515e">Circuits</FONT></A><A href="https://2015.igem.org/Team:Vanderbilt/Modeling/Populations"><FONT color="#9e515e">Populations</FONT></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>