Team:CLSB-UK/Results
CLSB-UK
Project
Design
Parts
Demonstrate
Judging
Contribution
Lab work
Results
Experiments
Safety
Notebook
InterLab
Human Practices
Silver
Engagement
High School Guide
Gold Integrated
Safety
Cost model
Modeling
NUPACK model
Mass action kinetics
Stochastic model
Cost model
Software
Overview
Toehold tools
SynBioBot
Hardware
Overview
Interpreter
Measurement
People
Team
Collaborations
Attributions
Sponsors
We strongly recommend using Google Chrome or Mozilla Firefox to view our wiki.
Contents
1 Results
1.1 Amplification of toehold switches
1.2 Switch for 15b-5p
1.2.1 Leakage
1.2.2 Specificity
1.2.3 Error analysis
1.2.4 Modeling
1.3 Applications
1.3.1 NSCLC
1.3.2 Wider Uses
1.4 Future work
1.5 Achievements
1.6 Conclusion
1.7 References
Results
We have designed a cell free biosensor that can be used to quantify miRNA levels, allowing for cheap and early diagnosis of non-small cell lung cancer (NSCLC).
Our biosensors consist of toehold switches that are activated in the presence of the miRNAs hsa-miR-15b-5p (15b-5p) and hsa-miR-27b-3p (27b-3p). Abnormal levels of these miRNAs are indicative of NSCLC. [1]
The toehold switches are responsible for regulating the translation of a reporter protein (GFP or luciferase). The toehold switch is typically in the off state, resulting in minimal production of the reporter protein. However, upon the addition of a trigger miRNA, the toehold switch is activated and translation of the reporter protein occurs. The number of toehold switches activated and thus the miRNA levels can be quantified based on the fluorescence or luminescence intensity from the sensors.
Here we demonstrate the ability of toehold switches to be used in miRNA quantification and discuss the role that these sensors could play in the future.
Amplification of toehold switches
We inserted our toehold switch constructs into the pSB1C3 plasmid backbone and transformed E. coli with these plasmids. We used E. coli to amplify the plasmids before miniprepping them for a use in a cell free system.
These constructs contained the relatively strong promoter BBa_J23111. After amplification of our constructs in E. coli, we picked 5 transformed colonies and sequenced the plasmids. However, the sequences did not correspond to what we were expecting.  Using BLAST sequence analysis to identify the sequences, we found parts of GFP and luciferase, but it appeared that at least some of the inserts had been excised from the plasmid. Additionally, we found mutations around the promoter regions, leading us to believe that the transcripts produced by our constructs are toxic to E. coli.
To resolve this issue, we replaced BBa_J23111 with the arabinose inducible promoter BBa_K808000. This promoter was reported to have low levels of leakage and should therefore be suitable for regulating the expression of lethal parts in E. coli. After inserting our new constructs into E. coli and amplifying them, we sequenced the plasmids.
We found that the toehold switch for 15b-5p had been successfully amplified. Unfortunately, we still found some mutations around the promoter region for 27b-3p, indicating it might be more toxic than 15b-5p. This meant even low concentrations of this part produced due to the leaky promoter were toxic to E. coli. Consequently, we were only able to characterize our toehold switch 15b-5p due to time constraints.
Using BBa_K808000 we were unable to insert our constructs containing luciferase into the pSB1C3 plasmid backbone in one ligation step as the construct was too large. This meant that we were unable to characterize the 15b-5p switch with luciferase as we did not have enough time to ligate the individual components of the constructs into the pSB1C3 backbone.
Switch for 15b-5p
When testing the switch with 90nM of 15b-5p we found that one of our replicates was invalid (repeat 3 on graph) and as such we omitted that set of results from our analysis.
We found our toehold switch required a miRNA concentration of 90nM to be significantly activated, but the minimum concentration required for activation lies between 9nM and 90nM.
Set time to:
6,000s
12,000s
18,000s
24,000s
30,000s
36,000s
We also found that peak fluorescence increased with miRNA concentration. However, unlike with Pardee et al, we did not see a linear response in fluorescence intensity with increasing trigger RNA. [2]
We believe this is because the promoter regulating our toehold switches was only activated when the arabinose was added - which was at the same time as the miRNA was added. This may have resulted in excess miRNAs relative to the number of toehold switches, particularly when 900nM of miRNA were added. Consequently, many miRNAs were not bound to the toehold switches and as such were degraded, reducing fluorescence output. To resolve this, the arabinose should be added first and the system left so that the switches could be produced before adding miRNA.
-ve control is just the cell free extract  and the -ve mix is the cell free extract + arabinose + DNA without miRNA
We found our sensors were activated in under an hour and that fold-changes (the fluorescence observed with miRNA compared to the fluorescence observed without miRNA) were greater than 20 fold after 4 hours. The maximum fold-change achieved was 25.5 fold at 10 hours with 900nM of 15b-5p. We also found that the fold-change was lower when 90nM of 15b-5p was added, with a maximum fold-change achieved being 19 fold at 10 hours. Therefore, our sensors are capable of discriminating between two miRNA concentrations. As such, it could be used to discriminate between people with and without NSCLC.
Additionally, the rate of increase in fold-change was negligible after ~7 hours with 900nM of 15b-5p, whereas fold-change was still increasing after 10 hours with 90nM of 15b-5p. The difference in fluorescence is therefore at its greatest before 7 hours.
The large fold-changes demonstrate that our toehold switches work well, they have very low levels of leakage and they allow for easy distinction between the on and the off state. Additionally, the large fold-changes allows for discrimination of small differences in miRNA concentration, thus increasing sensitivity.
Moderately expressed miRNAs exhibit copy numbers of up to 133,970/μl in plasma. [3] Standard miRNA extraction kits concentrate miRNA levels in body fluids by ~20 fold during extraction. [4] This results in a concentration of 4.5 pM for moderately expressed miRNAs in plasma after extraction. In comparison to other body fluids, especially breast milk (~153 times more concentrated) and seminal fluid (~58 time more concentrated), total RNA concentration in plasma is low. [5] This would suggest that the miRNA concentration in plasma is also low relative to other body fluids. Therefore, although our 15b-5p GFP sensors are not suitable for quantification of miRNAs that are not highly expressed in plasma, they may be sensitive enough for use with other body fluids.
The use of luciferase may allow for more sensitive measurements. Luciferase is far more sensitive than GFP and it is possible to detect as little as 10-19 moles of firefly luciferases with commercial instruments. [5] We believe that if our toehold switches regulated the production of luciferase, it could allow for moderate-low abundance miRNAs in plasma and other body fluids to be quantified.
Furthermore, we speculate that the novel riboregulators we proposed could have higher fold-changes. This would amplify the signal of miRNAs, permitting quantification of even lower miRNA concentrations.
Leakage
Our experiments showed that our switches had very low levels of leakiness as our set of tests containing no arabinose had lower fluorescence than the negative control (just the cell free extract with nothing in it). Nevertheless, we anticipate that there was some system leakage.
We propose a method for increasing fold-change for lower miRNA concentrations by decreasing promoter induction. During our experiments, we added 0.5% arabinose to our cell free systems, as our characterization of BBa_K808000 revealed this would maximize the number of toehold switches produced. However, our mass action kinetics model showed that for low concentrations of miRNAs, the ratio of open toehold switches (OTS) to closed toehold switches (CTS) at 0.5% arabinose will be extremely low. Since the closed toehold switches are slightly leaky, large induction of the promoter leads to a significant increase in fluorescence due to the leakiness alone, thus resulting in a very low signal to noise ratio. Therefore, we postulate that reducing the concentration of the inducer for low concentrations of miRNAs will help to increase fold-changes, making the activation at lower concentrations more noticable.
Specificity
In addition to testing our 15b-5p toehold switch with 15-5p, we tested it with 900nM of 27b-3p. This was to demonstrate that our switches were activated only in the presence of 15b-5p. We observed a mean fold-change of 1.25 throughout the 10 hours that the experiment was left to run, with the fold-change never surpassing 1.33. This data indicates that our toehold switches are only activated in the presence of the correct target miRNA.
Error analysis
Small variation in fluorescence is crucial for accurately distinguishing between different miRNA concentrations. Our results show that we could distinguish between less than an order of magnitude difference in miRNA concentration as the standard deviation bars do not overlap.
Since we did triplicates and therefore only had three results (two in the case of the 90 nM test), the standard deviation could be reduced by increasing the number of replicates.
Furthermore, we only added 1μl of miRNAs into the cell free system. Small volumes such as this are prone to variation due to pipetting errors. These errors could be reduced by automating the process.
Finally, we do not believe that we obtained complete activation of the switches with 900nM of 15b-5p caused by simultaneously adding the arabinose and the miRNAs, explained previously. Therefore, we would expect a greater difference between the results, further aiding in discrimination between the two concentrations.
Modeling
We fit our mass action kinetics model to the experimental data. We were happy to see our model fit well, and the experimental data allowed us to come up with parameters other teams can use in future models of cell free systems.
View the modeling page for more information.
Applications
NSCLC
For NSCLC, there is a difference of ~4 fold in miRNA concentration between individuals with and without cancer. [5] We are confident that we would effectively be able to differentiate between these levels based off our data. Our sensors using GFP are unsuitable for quantifying serum or plasma miRNA concentrations but we expect that by using the reporter protein luciferase, we would be able to quantify 15b-5p and 27b-3p levels.
Wider Uses
Up to 700μl of body fluid can be eluted into 50μl of nuclease free water during miRNA extraction. [4] This could then divided up into 50 wells, allowing for 50 miRNAs to be quantified. If the process were automated and therefore volumes smaller than 1μl could be accurately pippeted, more miRNAs could be quantified from the same initial volume of body fluid. Since accurate screening can require 2-3 miRNAs, [1] [6] we could detect over 20 different diseases from a 700μl sample.
To further increase the number of miRNAs that can quantified, it is possible to create a multiplexing assay using multiple toehold switches in one well. Each switch would regulate the production of a different reporter protein with a distinct emission, thereby increasing the number of miRNAs that could be quantified several fold.
Future work
Testing the switch for 27b-3p
Characterization of the toehold switches with luciferase
Finding the minimum concentration of miRNA to significantly activate our switches with both GFP and luciferase
Testing the switches with homologous miRNAs to determine how specific the switches are
Testing the second series of switches and novel riboregulators
Testing the switches with samples obtained from patients
Achievements
Demonstrated our parts require the use of a non-leaky promoter due to the production of toxic transcripts
Demonstrated the toehold switch constructs containing GFP work in a cell free system
Demonstrated the 15b-5p switch is significantly activated with 90nM 15-5p miRNA
Demonstrated that fluorescence output increases with miRNA concentration
Demonstrated the 15b-5p toehold switch is only activated in the presence of specific miRNAs
Conclusion
Our current sensors may be capable of detecting diseases using highly expressed miRNAs from plasma or serum. However, we believe that these sensors are more suited for other body fluids with higher miRNA concentrations such as breast milk and seminal fluid.
The use of luciferase would allow for more sensitive tests, however a key advantage of using fluorescent proteins is that they would allows for multiplexing due to their spectral diversity. Multiplexing would increase the number of miRNAs that can be detected from one sample by several fold.
We also recommend testing our second series of switches and novel riboregulators for their potentially increased specificity and fold-change.
References
↑ 1.0 1.1 Hennessey, P. T., Sanford, T., Choudhary, A., Mydlarz, W. W., Brown, D., Adai, A. T., ... & Califano, J. A. (2012). Serum microRNA biomarkers for detection of non-small cell lung cancer. PloS one, 7(2), e32307.
↑ Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., ... & Daringer, N. M. (2016). Rapid, low-cost detection of Zika virus using programmable biomolecular components. Cell, 165(5), 1255-1266.
↑ Mitchell, P. S., Parkin, R. K., Kroh, E. M., Fritz, B. R., Wyman, S. K., Pogosova-Agadjanyan, E. L., ... & Lin, D. W. (2008). Circulating microRNAs as stable blood-based markers for cancer detection. Proceedings of the National Academy of Sciences, 105(30), 10513-10518.
↑ 4.0 4.1 QIAGEN (n.d.). RNA Isolation Kit: RNeasy Mini Kit. Retrieved October 8: 2017: fromhttps://www.qiagen.com/gb/shop/sample-technologies/rna/total-rna/rneasy-mini-kit/
↑ 5.0 5.1 5.2 Zhang, Y., Phillips, G. J., & Yeung, E. S. (2008). Quantitative imaging of gene expression in individual bacterial cells by chemiluminescence. Analytical chemistry, 80(3), 597-605.
↑ Slater, E. P., Strauch, K., Rospleszcz, S., Ramaswamy, A., Esposito, I., Klöppel, G., ... & Bartsch, D. K. (2014). MicroRNA-196a and-196b as potential biomarkers for the early detection of familial pancreatic cancer. Translational oncology, 7(4), 464-471.
Major sponsors
Mr John
Mendez
Contact us
