<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="jumbotron full_width" style="background-image: url('https://static.igem.org/mediawiki/2017/5/5d/T--CLSB-UK--goldhp_cover.jpg');"><DIV><H1 style="font-size: 89px">Integrated Human Practices</H1><H2 style="font-size: 24px">To the disappointment of our modeling team, this page does not involve any calculus</H2></DIV></DIV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Integrated_Human_Practices"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Integrated Human Practices</SPAN></A></LI><LI class="toclevel-1"><A href="#Design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-1"><A href="#Modeling"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-1"><A href="#Applications"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Applications</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Sensitivity_and_Specificity"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Sensitivity and Specificity</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#NICE_Criteria_for_screening_tools"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">NICE Criteria for screening tools</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Education"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Education</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#References"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Integrated_Human_Practices">Integrated Human Practices</SPAN></H1><P>From our initial idea, the design of our toehold switches and our model to the future applications of our project, the opinions of specialists have been vital in influencing our work.
</P><P>Here, we have broken our project down into its different parts. For each, we explain the role feedback played in improving the aspect of the project, and why we changed what we did.
</P><H1 id="Design" style="color: white; background: black; padding: 30px 0; margin: 50px 0 20px 0"><SPAN class="mw-headline" id="Design">Design</SPAN></H1><DIV class="image_container"><DIV class="image_holder" style="margin: 36px 2% 0 0; width: 28%"><P style="font-size: 12px; color: #888;">Attribution: Image provided by Dr Kirsten Jensen</P></DIV><DIV><H2 id="Dr Kirsten Jensen">Dr Kirsten Jensen</H2><P>Our initial constructs used a constitutive promoter. After cloning our construct in pSB1C3 plasmid, we attempted to amplify them in <I>E. coli</I>. However upon sequencing the amplified inserts, none of the sequences were what we expected to find.
</P><P>We asked Dr Kirsten Jensen, an expert in synthetic biology at Imperial College London for advice. She inspected our sequences and suggested that the transcripts from our constructs were toxic to <I>E. coli</I>. She therefore suggested that we switch our promoter to an inducible promoter to avoid synthesis of the transcript during amplification in <I>E. coli</I>.
</P><P>Following her advice, we used an arabinose inducible promoter, which resolved the issue for the 15b-5p switches and allowed us to continue with our experiments. However, the 27b-3p switches were still toxic.
</P><P>See the <A href="/Team:CLSB-UK/Results" title="Team:CLSB-UK/Results">results page</A> for more details.
</P></DIV></DIV><DIV class="image_container"><DIV><H2 id="Dr Alex Green">Dr Alex Green</H2><P>Dr Alex Green is an expert in the world of synthetic biology – and in particular toehold switches. He confirmed that our existing design should work and suggested looking into the design of his B-series of Zika virus toehold switches to improve our existing switches. We did so, and modified our design to base them on these switches which have reduced system leakage. They have shorter loops and no-refolding domain which helped increase sensitivity. Based on his feedback, we also increased the loop size for the 15b-5p switch design. This compensates for its long stem (which reduces on state activity), increasing dynamic range. With his help, our switches were optimized for their application as a diagnostic tool. Read more about our first series switches on our <A href="/Team:CLSB-UK/Design" title="Team:CLSB-UK/Design">design page</A>.
</P></DIV><DIV class="image_holder" style="margin: 36px 0 0 2%; width: 28%"><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Image provided by Dr Alex Green</P></DIV></DIV><DIV class="image_container"><DIV class="image_holder" style="margin: 36px 2% 0 0; width: 28%"><P style="font-size: 12px; color: #888;">Attribution: Logo provided by iGEM EPFL</P></DIV><DIV><H2 id="iGEM EPFL">iGEM EPFL</H2><P>After speaking to several experts who highlighted the importance of reducing false positives and false negatives (see <A href="#Applications">applications section</A> below), we began to explore ways to improve the design of our toehold switches. We spoke to iGEM EPFL - another team working with toehold switches. We explained the issues we had with homologous miRNAs that reduce the specificity of our sensors. iGEM EPFL suggested that we design our switches to utilize the mechanism employed in “Ribocomputing devices for sophisticated in vivo logic computation”.<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP></P><P>Based on this advice we designed a second series of toehold switches that can distinguish between miRNA sequences with one nucleotide differences. This improved <A href="/Team:CLSB-UK/Design#Second_series_of_toehold_switches" title="Team:CLSB-UK/Design">our design</A>, effectively reducing false positives, a crucial part of any diagnostic method.
</P><P>With reduced cross-hybridisation, we could create an effective multiplexing assay: multiple switches that each regulate the production of a different reporter protein with a distinct emission peak. This would drastically increase the efficiency and scope of the test. As the second generation of switches can detect single base mismatch differences the sensitivity and specificity of the entire system would be improved.
</P></DIV></DIV><DIV class="image_container"><DIV><H2 id="Professor David Rubinsztein">Professor David Rubinsztein</H2><P>Professor David Rubinsztein, Professor of Molecular Neurogenetics at Cambridge University, suggested that we use luciferase instead of GFP as our reporter protein. He explained that it may be able to provide more sensitive readings. We produced 4 composite parts as a result - with 2 being toehold switches containing luciferase. However, the constructs with luciferase were so long that, in combination with our inducible promoter, we were unable to insert them into the pSB1C3 plasmid backbone in one ligation step. Unfortunately, we did not have time to ligate the individual components of the constructs into the pSB1C3 backbone so we were unable to test the switches with luciferase. If we had managed to test our switches with luciferase, we anticipate that our sensitivity would be significantly higher.
</P></DIV><DIV class="image_holder" style="margin: 36px 0 0 2%; width: 28%"><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Image provided by Professor David Rubinsztein</P></DIV></DIV><H1 id="Modeling" style="color: white; background: black; padding: 30px 0; margin: 50px 0 20px 0"><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H1><DIV class="image_container"><DIV class="image_holder" style="margin: 36px 2% 0 0; width: 28%"><P style="font-size: 12px; color: #888;">Us visiting him at Imperial</P></DIV><DIV><H2 id="Dr Thomas Ouldridge">Dr Thomas Ouldridge</H2><P>Early on in our project we met Dr Thomas Ouldridge, a Royal Society University Research Fellow at the Department of Bioengineering at Imperial College London. He strengthened our modeling foundations and told us what assumptions were sensible for our system. He explained energy landscapes of nucleic strand displacements, which helped us increase the specificity of our first series of switches despite us not modeling the toehold switch binding base-by-base.
</P><P>We went back to him a couple of months later with our mass action kinetics model. He located where we could remove unneeded complexity from our model. After changing the model accordingly, we were able to to narrow our model to have just 4 main parameters. This allowed us to vary them to test the effects of changing the parts of our toehold switch, for example the RBS and promoter strength to optimize the dynamic range. This made our model orders of magnitude more useful to both our design and wetlab teams. See our <A href="/Team:CLSB-UK/Model#MAK" title="Team:CLSB-UK/Model">modeling page</A> to learn more about why fewer parameters is better and how our model influenced our design and implementation of our project.
</P></DIV></DIV><H1 id="Applications" style="color: white; background: black; padding: 30px 0; margin: 50px 0 20px 0"><SPAN class="mw-headline" id="Applications">Applications</SPAN></H1><DIV class="image_container"><DIV><H2 id="Dr Daniel Pregibon">Dr Daniel Pregibon</H2><P>We spoke to Dr Daniel Pregibon, who works for Abcam, and developed a new method of quantifying miRNAs. He advised that our method could prove especially  effective in low resource settings and in situations where results are needed urgently. He also advised us on the clinical use of miRNAs as biomarkers.
</P><P>We have looked further into all of this on our <A href="/Team:CLSB-UK/Silver" title="Team:CLSB-UK/Silver" class="mw-redirect">silver human practices page</A>.
</P></DIV><DIV class="image_holder" style="margin: 36px 0 0 2%; width: 28%"><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Image provided by Dr Daniel Pregibon</P></DIV></DIV><DIV class="image_container"><DIV class="image_holder" style="margin: 36px 2% 0 0; width: 28%"><P style="font-size: 12px; color: #888;">Attribution: Image provided by Dr Justin Daniels</P></DIV><DIV><H2 id="Dr Justin Daniels">Dr Justin Daniels</H2><P>We were unsure as to the clinical implementation of our project, so we spoke with Dr Justin Daniels, a pediatrician and clinical director, who has served on NICE (National Institute for Health and Care Excellence, a UK public body that provides national guidance to improve healthcare) appraisal committees. He suggested its main value could be as a screening tool for high risk individuals. We knew previously about how beneficial early detection of NSCLC is, but we had not explored the clinical implementation of our sensor before talking to him. He suggested offering the test to high risk individuals for whom the pre-symptomatic diagnosis of their cancer could lead to an increase in the rate of survival: in the case of NSCLC, this could be carried out by offering the test to those with a certain smoking history. To determine which individuals had a sufficiently high risk of developing NSCLC, for our sensor to be cost effective, we have designed a cost model - take a look below.
</P><P>He also proposed that if the concept of our test were to be proven and a variety of miRNAs could be detected as biomarkers for a range of cancers, then our test could be used as a screening tool for anyone over a certain age, due to the increase in cancer prevalence with age. His comments led us to research the <A href="#NICE_Criteria_for_screening_tools">required criteria below</A> for our sensor to be used as a screening tool. His advice on the high sensitivity and specificity required fed into our need to create a second series of switches.
</P></DIV></DIV><DIV class="image_container"><DIV><H2 id="Dr Sujal Desai">Dr Sujal Desai</H2><P>We discussed the use of our sensor for high risk population screening with Dr Desai, a consultant chest radiologist. He informed us that lung cancer rarely presents before age 40, and to maximize the test’s value it should be offered to those aged 55 and above, with a smoking history - so that we would not waste money offering the test to too many people who were very unlikely to have cancer.
</P><P>To integrate their advice, we worked with our our modeling team to work out who would we should screen. Tammegami et al.’s probability score is the probability of an individual getting lung cancer in the next 6 years.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP></P><P>Using our model which combined this with the data for our project and standard NSCLC treatment, we worked out for what probabilities it would be cost effective for us to screen. The model showed that our tool could be cost effective at cost per QALY - saving lives and saving the NHS money. It shows that even if our sensitivity was lower than we expect, our sensor would be extremely cost effective.
</P><P>Visit the <A href="/Team:CLSB-UK/Model#Cost_model" title="Team:CLSB-UK/Model">cost model page</A> for more information.
</P></DIV><DIV class="image_holder" style="margin: 36px 0 0 2%; width: 28%"><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Image provided by Dr Sujal Desai</P></DIV></DIV><DIV class="image_container"><DIV class="image_holder" style="margin: 36px 2% 0 0; width: 28%"><P style="font-size: 12px; color: #888;">Attribution: Image provided by Professor Laurence Lovat</P></DIV><DIV><H2 id="Professor Laurence Lovat">Professor Laurence Lovat</H2><P>We spoke to Professor Laurence Lovat in order to further our understanding of the criteria our system needs to meet in order for it to be clinically viable. He suggested we could use a tube with a preservative in its lid that is only released upon closing the tube. The tube could then be sent out via post to an individual to provide a saliva or urine sample, and could be posted back to a centralized lab for processing. For blood, there would be localized centers for tests (or just GP centers) and one main processing lab.
</P><P>He also recommended using a multiplexing assay to detect multiple different miRNAs simultaneously from the same sample. However, at the time we had only designed one series of switches which would be activated by homologs. For a multiplexing assay, cross-hybridization levels need to be minimal - so we knew we had to think about new ways to design our switches. Our second series of switches, designed after talking with iGEM EPFL, were designed to solve this problem.
</P><P>Professor Lovat’s team at UCL also helped us choose the best pieces of equipment for miRNA extraction which helped us with our <A href="/Team:CLSB-UK/Model#Cost_model" title="Team:CLSB-UK/Model">cost analysis</A>. We also discussed carrying out miRNA extractions in the field with them, as well as automation of the miRNA extraction protocol.
</P></DIV></DIV><H2><SPAN class="mw-headline" id="Sensitivity_and_Specificity">Sensitivity and Specificity</SPAN></H2><P>Professor Lovat and Dr Desai were concerned about the specificity and sensitivity of our test, and how it would be used for screening.
</P><P>There are two types of sensitivity and specificity:
</P><P>In terms of the toehold switch, sensitivity is the lowest number of miRNAs that can be detected and specificity is whether only the target miRNA can activate the switch. The feedback of doctors prompted us to seek out the advice of iGEM EPFL, to create the <A href="/Team:CLSB-UK/Design#Second_series_of_toehold_switches" title="Team:CLSB-UK/Design">second generation switches</A> with massively superior specificity.
</P><P>In terms of a screening tool, <B>sensitivity</B> relates to the false negative rate, while <B>specificity</B> relates to the false positive rate.
</P><UL><LI> A 100% sensitive test will always pick up the disease but may also pick up patients that do not have the disease (no false negatives).</LI><LI> A 100% specific test will never pick up healthy patients but may also miss patients that do have the disease (no false positives).</LI></UL><DIV id="ssdiagram_1" style="margin-top: 40px; text-align: center"><SVG style="max-width: 600px" xmlns="http://www.w3.org/2000/svg" viewbox="0 15 265.2 200"><G transform="translate(-16.68 -75.657)"><TEXT x="82.3" y="104.1" stroke-width=".8" font-size="12.1" word-spacing="0" font-weight="800" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.3" y="104.1">High Sensitivity</TSPAN></TEXT><TEXT x="82.5" y="114.6" stroke-width=".5" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="300" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.5" y="114.6">Few false negatives</TSPAN></TEXT><TEXT x="82.5" y="241.6" stroke-width=".5" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.5" y="241.6">NEGATIVE RESULT</TSPAN></TEXT><TEXT x="217.4" y="104.1" stroke-width=".8" font-size="12.1" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="800" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.4" y="104.1">Low Specificity</TSPAN></TEXT><TEXT x="217.6" y="114.6" stroke-width=".5" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.6" y="114.6">Lots of false positives</TSPAN></TEXT><TEXT x="217.6" y="241.6" stroke-width=".5" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="300" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.6" y="241.6">POSITIVE RESULT</TSPAN></TEXT><TEXT x="132.8" y="263.9" stroke-width=".5" font-weight="300" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="132.8" y="263.9">Healthy</TSPAN></TEXT><TEXT x="132.8" y="276.8" stroke-width=".5" letter-spacing="0" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="132.8" y="276.8">Has disease</TSPAN></TEXT></G></SVG><P>A highly sensitive test will have few false negatives but lots of false positives. Switch to <A onclick="document.getElementById('ssdiagram_1').style.display = 'none'; document.getElementById('ssdiagram_2').style.display = 'block';" style="cursor:pointer">high specificity</A>.</P></DIV><DIV id="ssdiagram_2" style="margin-top: 40px; text-align: center; display: none"><SVG style="max-width: 600px" xmlns="http://www.w3.org/2000/svg" viewbox="0 15 265.2 200"><G transform="translate(-16.684 -75.657)"><TEXT x="82.3" y="104.1" stroke-width=".8" font-size="12.1" word-spacing="0" font-weight="800" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.3" y="104.1">Low Sensitivity</TSPAN></TEXT><TEXT x="82.5" y="114.6" stroke-width=".5" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="300" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.5" y="114.6">Lots of false negatives</TSPAN></TEXT><TEXT x="82.5" y="241.6" stroke-width=".5" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="82.5" y="241.6">NEGATIVE RESULT</TSPAN></TEXT><TEXT x="217.4" y="104.1" stroke-width=".8" font-size="12.1" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="800" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.4" y="104.1">High Specificity</TSPAN></TEXT><TEXT x="217.6" y="114.6" stroke-width=".5" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" text-anchor="middle" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.6" y="114.6">Few false positives</TSPAN></TEXT><TEXT x="217.6" y="241.6" stroke-width=".5" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" text-anchor="middle" font-weight="300" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="217.6" y="241.6">POSITIVE RESULT</TSPAN></TEXT><TEXT x="132.8" y="263.9" stroke-width=".5" font-weight="300" font-size="7.8" letter-spacing="0" word-spacing="0" font-family="Montserrat" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="132.8" y="263.9">Healthy</TSPAN></TEXT><TEXT x="132.8" y="276.8" stroke-width=".5" letter-spacing="0" font-size="7.8" word-spacing="0" font-weight="300" font-family="Montserrat" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="132.8" y="276.8">Has disease</TSPAN></TEXT></G></SVG><P>A highly specific test will have lots of false negatives but few false positives. Switch to <A onclick="document.getElementById('ssdiagram_2').style.display = 'none'; document.getElementById('ssdiagram_1').style.display = 'block';" style="cursor:pointer">high sensitivity</A>.</P></DIV><P>The sensitivity and specificity of a screening tool depend on the cutoffs used on a Reciever Operating Characteristic (ROC) curve, and as one increases the other is likely to decrease. Depending on the type of screening we want to do, we can alter the cutoffs on the ROC curve.
</P><P>For a high-risk screening (e.g. heavy smokers for NSCLC), we would set the cut-offs at 100% sensitivity so it would always pick up NSCLC, and compromise on the specificity which would be at 84%. For mass population screening, we would use a 100% specificity cut-off which would cause the sensitivity to be 56%. The cutoffs here are based on Hennessy et al.'s ROC curve for miRs-15b-5p and 27b-3p in diagnosing NSCLC.<SUP id="cite_ref-hennessey_3-0" class="reference"><A href="#cite_note-hennessey-3">[3]</A></SUP></P><P>The ROC curve for the two miRNAs we are using is shown here. The area under curve (AUC) of 98% represents the accuracy of the test.
</P><P><SVG style="max-width: 600px;margin: 0 auto;display: block;overflow: visible;" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 207.4 202.2"><TEXT x="188.1" y="161.2" stroke-width=".7" font-family="Montserrat" font-weight="300" font-size="10.9" word-spacing="0" text-anchor="end" white-space="normal" letter-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="188.1" y="161.2">AUC = 0.98</TSPAN></TEXT><G word-spacing="0" letter-spacing="0" font-family="Montserrat" white-space="normal" stroke-linecap="square" stroke-linejoin="bevel"><TEXT x="38.9" y="185.9" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="38.9" y="185.9">0.0</TSPAN></TEXT><TEXT x="70.2" y="185.9" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="70.2" y="185.9">0.2</TSPAN></TEXT><TEXT x="102" y="185.9" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="102" y="185.9">0.4</TSPAN></TEXT><TEXT x="134.1" y="185.9" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="134.1" y="185.9">0.6</TSPAN></TEXT><TEXT x="165.6" y="185.9" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="165.6" y="185.9">0.8</TSPAN></TEXT><TEXT x="199.5" y="185.9" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="199.5" y="185.9">1.0</TSPAN></TEXT><TEXT x="20.6" y="174.8" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="20.6" y="174.8">0.0</TSPAN></TEXT><TEXT x="21" y="145.8" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="21" y="145.8">0.2</TSPAN></TEXT><TEXT x="20.9" y="116.9" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="20.9" y="116.9">0.4</TSPAN></TEXT><TEXT x="21" y="88" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="21" y="88">0.6</TSPAN></TEXT><TEXT x="20.9" y="59" fill="#666" stroke-width=".6" font-size="8.7" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="20.9" y="59">0.8</TSPAN></TEXT><TEXT x="22" y="30.1" fill="#666" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="22" y="30.1">1.0</TSPAN></TEXT><TEXT x="116" y="200.4" stroke-width=".6" font-weight="300" font-size="8.7" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="116" y="200.4" font-weight="600">1 - Specificity</TSPAN></TEXT><TEXT x="-77.2" y="6.8" stroke-width=".6" font-weight="600" font-size="8.7" transform="rotate(-90 .01 .04)" text-anchor="end"><TSPAN x="-77.2" y="6.8">Sensitivity</TSPAN></TEXT><TEXT x="105.2" y="8.8" stroke-width=".7" font-size="11.4" font-weight="300" text-anchor="middle" transform="translate(-.03 .048)"><TSPAN x="105.2" y="8.8" font-weight="600">ROC curve for miR-15b and miR-27b</TSPAN></TEXT></G><TEXT x="187.8" y="134.4" stroke-width=".4" font-size="6.4" word-spacing="0" font-weight="300" stroke-linejoin="bevel" font-family="Montserrat" text-anchor="end" white-space="normal" letter-spacing="0" stroke-linecap="square" style="-inkscape-font-specification:'Montserrat, Light';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:end"><TSPAN x="187.8" y="134.4">At specificity = 1, sensitivity = 0.56</TSPAN></TEXT><TEXT x="187.8" y="125" stroke-width=".4" font-size="6.4" letter-spacing="0" style="-inkscape-font-specification:'Montserrat, Light';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:end" font-weight="300" font-family="Montserrat" text-anchor="end" white-space="normal" word-spacing="0" stroke-linecap="square" stroke-linejoin="bevel"><TSPAN x="187.8" y="125">At sensitivity = 1, specificity = 0.84</TSPAN></TEXT></SVG><SPAN style="font-size: 12px; color: #888; text-align: center; display: block">Graph reproduced from data provided in Hennessey et al.'s paper<SUP id="cite_ref-hennessey_3-1" class="reference"><A href="#cite_note-hennessey-3">[3]</A></SUP></SPAN></P><P>
The choice to use high specificity for mass population ensures would minimize false positive rates for cost efficiency necessary for population wide screening. However, the low sensitivity means that some people with cancer may not be referred for further testing, and are thus left untreated. Through using many miRNA signatures in tandem, we would try to reduce the rate of false negatives.  The goal of mass population screening is also to pick up as many people with a cancer, as early as possible, not everyone, meaning the lower sensitivity would be acceptable. 
</P><P>In the case of high risk screening, the low specificity (and thus higher false positive rate) is less problematic than it may seem, as the ratio of positives to negatives in the high risk group is greater than the general population. Therefore as more true positives would be picked up, the cost of unnecessary CT scans would be offset by the additional correct diagnoses. Everyone with a positive test result would have a CT scan anyway to confirm the presence of their cancer.
</P><P>The major drawback to a low specificity is the psychological impact of a patient being told they have cancer when in fact they do not. Again, through using multiple miRNA signatures in tandem - i.e. the levels of different miRNAs related to a single disease, and calculating a weighted score involving all of them, so that an abnormal level of just one miRNA does not necessarily lead to a false positive.
</P><P>We can also alleviate the problem of false positives by ensuring doctors tell patients that the results our sensor provides just indicate the need for further testing, and are not a diagnosis.
</P><P>Dr Desai also advised that the result from our test could be combined with a score from patient history to quantify their risk (e.g. Tammegami et al.’s probability score). This total score would then be used to increase accuracy of the test.
</P><H2><SPAN class="mw-headline" id="NICE_Criteria_for_screening_tools">NICE Criteria for screening tools</SPAN></H2><P>To incorporate the feedback from doctors into the applications of our project, we have explained how our sensor meets (or would be applied so that it meets) the NICE criteria for a screening tool.
</P><TABLE><TBODY><TR><TH> NICE Criteria </TH><TH> Our Sensor
</TH></TR><TR><TD> The condition being screened should be a serious health problem in terms of frequency and/or severity </TD><TD style="background: #e8f5e9;"> Lung cancer is the most common cause of cancer-related mortality, with 1.6 million deaths worldwide in 2012. In the UK in 2014, there were 46,403 new cases and 35,895 deaths.<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP> 80-85% of all lung cancers are NSCLC.<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP></TD></TR><TR><TD> The epidemiology, natural history and development of the condition from latent to declared states should be understood and/or there should be robust evidence about the association between the marker and the disease </TD><TD style="background: #fff9c4;"> Much is known about NSCLC, and although there are still a lot of research going into understanding cancer characteristics. The same is true for most other cancers. Although miRNAs are still being characterized, Hennessey et al. demonstrated a potential link between miR-15b and miR-27b to NSCLC. However, validation is still required with a large patient cohort for clinical use. Expert Dan Pregibon anticipates that the use of miRNAs as biomarkers is not far off.
</TD></TR><TR><TD> All the cost-effective primary prevention should have been implemented as far as practicable </TD><TD style="background: #e8f5e9;"> Smoking cessation is the most effective primary prevention for NSCLC<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>, as smoking is responsible for 90% of cases.<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP> In the UK, the government has implemented numerous policies to reduce rates of smoking. These include heavily taxing tobacco, banning TV advertising and requiring cigarette packets to have “smoking kills” branding.Rates of smoking are declining<SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP>, although they are still sufficiently high for lung cancer to remain as prevalent it is. Primary prevention to secondhand smoke has also been implemented as far as impossible - with smoking banned in enclosed workplaces, public buildings and on public transport.<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP> However, no further measures can be taken to avoid exposure by going outside.The other risk factors for cancer are less well known - as there are a myriad of carcinogens in the world we live in - the food standards agency works to minimize carcinogens in food, drugs and cosmetics,<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP> while NICE ensures that medical radiation exposure is minimized.
</TD></TR><TR><TD> The screening test should be simple, safe, precise and validated </TD><TD style="background: #fff9c4;"> As we are using a cell free system, our test is safe. Our test would comprise a blood test, followed by miRNA purification and fluorescence measurements which are all relatively simple. These steps do not require the expertise of radiologists that are needed for CT scans. However, for our test to be shown to be precise and validated, further characterisation is required with the miRNA biomarkers and validation with a large data set of patients.
</TD></TR><TR><TD> There should be evidence that intervention at a pre-symptomatic stage leads to better outcomes </TD><TD style="background: #e8f5e9;"> NSCLC is far easier to treat in its earlier stages. The same is true for almost all cancers, as once a cancer has metastasized it can only be treated palliatively. Early stage tumors can be surgically excised and treated adjuvantly with curative intent.
</TD></TR><TR><TD> The test should be acceptable to the target population </TD><TD style="background: #e8f5e9;"> From a patient’s perspective, our sensor would just be a blood test. As such there should not be any issues with it. We are using a cell free systems which is sterile and abiotic - there are no bacteria involved which could potentially give rise to objections.
</TD></TR><TR><TD> There should be a policy on further diagnostic investigation of those with a positive test result. There should be agreed policies covering which individuals should be offered interventions and what the the appropriate intervention offered is assurance standards </TD><TD style="background: #e8f5e9;"> We have investigated this in <A href="/Team:CLSB-UK/Silver" title="Team:CLSB-UK/Silver" class="mw-redirect">silver human practices</A> - those with a positive result would go onto have a CT scan and move onto the standard NICE diagnostic and treatment pathway (in the case of NSCLC).<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP></TD></TR><TR><TD> The distribution of test values in the target population should be known and a suitable cut-off level defined and agreed. There should be evidence from high quality randomized controlled trials that the screening programme is effective in reducing mortality or morbidity or evidence from high quality trials that the test accurately measures risk when helping a patient make an “informed choice” </TD><TD style="background: #fff9c4;"> Hennessey et al.’s cohort was not large enough for this to be conclusively known.<SUP id="cite_ref-hennessey_3-2" class="reference"><A href="#cite_note-hennessey-3">[3]</A></SUP> Further validation with a larger data set is required. The same is true for the links between miRNAs and disease more widely. However, preliminary results are promising. Our test would also need to undergo clinical trials - to test the efficacy of the toehold switches in measuring miRNA levels.
</TD></TR><TR><TD> There should be evidence that the complete screening programme (test, diagnostic procedures, treatment/ intervention) is clinically, socially and ethically acceptable to health professionals and the public </TD><TD style="background: #fff9c4;"> To be clinically acceptable our test would need to to undergo large scale clinical trials. This would validate our research showing that it would be specific and cost effective. Socially and ethically, there should be no issues, for the same reasons as above.
</TD></TR><TR><TD> The benefit gained should outweigh any harms from screening such as overdiagnosis, overtreatment, false positives, false reassurance, uncertain findings and complications. </TD><TD style="background: #e8f5e9;"> The NLST showed 20% reduction in mortality with high risk screening for NSCLC using LDCT. Screening is therefore recommended in the US to high risk population groups. However, our test could prevent unnecessary further testing in many - and overdiagnosis could be reduced by examining the miRNA signature of the tumor. Furthermore, as mentioned in <A href="/Team:CLSB-UK/HP/Silver#usage" title="Team:CLSB-UK/HP/Silver">silver human practices</A>, our test would not be offered as a conclusive diagnosis - only to rule out cancer in some and refer the others for further testing. There would also be psychological benefit using our test compared to existing methods in that a possible cancer could be ruled out far more quickly.
</TD></TR><TR><TD> The screening programme should be economically balanced in relation to expenditure on medical care with regard to evidence from cost benefit and/or cost effectiveness analyses </TD><TD style="background: #e8f5e9;"> We have created a <A href="/Team:CLSB-UK/Model#Cost_model" title="Team:CLSB-UK/Model">cost effectiveness model</A> - that works out who we should screen for the sensor to be cost effective and result in the cost per QALY being within NICE’s guidelines.
</TD></TR><TR><TD> Clinical management of the condition should be optimized in health-care providers prior to participation in the screening programme. There should be a plan for managing and monitoring the screening programme and a set of quality. Adequate staffing and facilities for testing, diagnosis, treatment and programme management should be available prior to the commencement of the screening programme </TD><TD style="background: #e8f5e9;"> Clinical management of NSCLC is optimized by NICE. NICE would be in charge of creating management and monitoring plan for our test, were it to be used clinically. Our test would actually free up healthcare staff and facilities. Early detection would reduce the number of cancers diagnosed at later stages where the only option is palliative care, which is more expensive than earlier stage treatment.This in turn would free up healthcare staff and facilities.<SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP></TD></TR><TR><TD> Information about the test and its outcomes must be understood by the individual being screened.Potential participants should be provided with evidence-based information on the purposes and potential consequences of the programme so they can make an informed choice.Decisions about the screening interval and sensitivity of the process should be scientifically justifiable to the public </TD><TD style="background: #fff9c4;"> When the individual is being screened, information, similar to that on our wiki, could be presented to them. Evidence for the efficacy of screening could only be given once clinical trials have been carried out. The screening interval could also be optimized from these trials.
</TD></TR><TR><TD> All other options for managing the condition with cost effective intervention should have been considered </TD><TD style="background: #e1f5fe;"> This is not applicable to NSCLC - early detection is required to reduce mortality rates - when detected late, it can only be treated palliatively.
</TD></TR><TR><TD> If people carrying a mutation are identified through screening, their natural history should be understood, which includes psychological implications.If the test is for a mutation or a genetic variants, the method and means for findings to be kept under review, should be clearly set out </TD><TD style="background: #e1f5fe;"> N/A
</TD></TR></TBODY></TABLE><H1 id="Education" style="color: white; background: black; padding: 30px 0; margin: 50px 0 20px 0"><SPAN class="mw-headline" id="Education">Education</SPAN></H1><DIV class="image_container"><DIV><P>After running a couple of workshops relating to synthetic biology, it was clear to us that genetic modification was seen as dangerous, or unsafe by most younger students. Bacteria were generally seen as risky to work with as many people did not realize there were non-pathogenic strains. Often synthetic biology jargon would cause people to turn off. This was not only the case with younger students, but with adults as well.
</P><P>We therefore changed our approach to engagement - focusing on making our project more accessible and therefore more exciting. We came up with real world scenarios to pique interest in synthetic biology - from diagnosing disease to reducing the impacts of climate change. We emphasized the real world problems synthetic biology can solve rather than the theoretical biology.
</P></DIV></DIV><P>We found a successful strategy was to pose questions, designed to help rectify misconceptions. We tried to ask students and parents questions like - “do you think it is dangerous to work with bacteria” or “is genetic modification safe?” Initial responses were generally negative, but the questions gave us a platform to address the issues.
</P><P>The concept of toehold switches had also proved extremely challenging to explain. We subsequently developed the analogy of a knot to explain them. The knot prevents a protein making factory (ribosome) from reading the instructions (mRNA) to make a protein that glows (GFP). Through diagrams, and use of animations, we were able to explain the concept of our project. We delivered an assembly where we used this analogy to explain our project and sent out a quiz. To our delight people understood most of our project and a few understood nearly everything. 109 completed the quiz and the majority scored over 75%. Find out more on our <A href="/Team:CLSB-UK/Engagement" title="Team:CLSB-UK/Engagement">engagement page</A>.
</P><P>We thereby vastly improved our education and public engagement programmes, learning from how people to develop an approach for explaining difficult concepts in an accessible and interesting way. We improved our presentation and communication skills throughout the project. We were delighted that rest of the open days, as well our assemblies and workshops went so well and got people interested in synthetic biology. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Green, A. A., Kim, J., Ma, D., Silver, P. A., Collins, J. J., &amp; Yin, P. (2016, September). Ribocomputing devices for sophisticated in vivo logic computation. In Proceedings of the 3rd ACM International Conference on Nanoscale Computing and Communication (p. 11). ACM.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Tammemägi, M. C., Katki, H. A., Hocking, W. G., Church, T. R., Caporaso, N., Kvale, P. A., ... &amp; Berg, C. D. (2013). Selection criteria for lung-cancer screening. New England Journal of Medicine, 368(8), 728-736.</SPAN></LI><LI id="cite_note-hennessey-3"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-hennessey_3-0">3.0</A></SUP><SUP><A href="#cite_ref-hennessey_3-1">3.1</A></SUP><SUP><A href="#cite_ref-hennessey_3-2">3.2</A></SUP></SPAN><SPAN class="reference-text">Hennessey, P. T., Sanford, T., Choudhary, A., Mydlarz, W. W., Brown, D., Adai, A. T., ... &amp; Califano, J. A. (2012). Serum microRNA biomarkers for detection of non-small cell lung cancer. PloS one, 7(2), e32307.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">(n.d.). Lung cancer statistics, Cancer Research UK. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="http://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/lung-cancer">http://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/lung-cancer</A></SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">(2016, May 16). What Is Non-Small Cell Lung Cancer? American Cancer Society. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="https://www.cancer.org/cancer/non-small-cell-lung-cancer/about/what-is-non-small-cell-lung-cancer.html">https://www.cancer.org/cancer/non-small-cell-lung-cancer/about/what-is-non-small-cell-lung-cancer.html</A></SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">(n.d.). Lung cancer - Prevention - NHS Choices. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="http://www.nhs.uk/Conditions/Cancer-of-the-lung/Pages/Prevention.aspx">http://www.nhs.uk/Conditions/Cancer-of-the-lung/Pages/Prevention.aspx</A></SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">(n.d.). Lung cancer: diagnosis and management - NICE. Retrieved October 7, 2017, from <A rel="nofollow" class="external free" href="https://www.nice.org.uk/guidance/cg121/chapter/introduction">https://www.nice.org.uk/guidance/cg121/chapter/introduction</A></SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Peto, R., Darby, S., Deo, H., Silcocks, P., Whitley, E., &amp; Doll, R. (2000). Smoking, smoking cessation, and lung cancer in the UK since 1950: combination of national statistics with two case-control studies. Bmj, 321(7257), 323-329.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">(n.d.). Smoking at work: the law - GOV.UK. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="https://www.gov.uk/smoking-at-work-the-law">https://www.gov.uk/smoking-at-work-the-law</A></SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">(n.d.). Committee on Carcinogenicity (COC) | Food Standards Agency. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="https://www.food.gov.uk/science/ouradvisors/carcinogenicity">https://www.food.gov.uk/science/ouradvisors/carcinogenicity</A></SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">(2011, April 1). Lung cancer: diagnosis and management | Guidance and ... - NICE. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="https://www.nice.org.uk/guidance/cg121/chapter/1-guidance">https://www.nice.org.uk/guidance/cg121/chapter/1-guidance</A></SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">(n.d.). Saving lives, averting costs, Cancer Research UK. Retrieved October 27, 2017, from <A rel="nofollow" class="external free" href="http://www.cancerresearchuk.org/sites/default/files/saving_lives_averting_costs.pdf">http://www.cancerresearchuk.org/sites/default/files/saving_lives_averting_costs.pdf</A></SPAN></LI></OL></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>