<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#First_series_toehold_switches"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">First series toehold switches</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Toehold_length"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Toehold length</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Base_of_the_stem"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Base of the stem</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Mini-loop"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Mini-loop</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Top_of_the_stem"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Top of the stem</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#System_leakage"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">System leakage</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Loop_size"><SPAN class="tocnumber">1.1.5.1</SPAN><SPAN class="toctext">Loop size</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Eliminating_downstream_refolding"><SPAN class="tocnumber">1.1.5.2</SPAN><SPAN class="toctext">Eliminating downstream refolding</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#Homologs"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Homologs</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Validating_our_NUPACK_modeling"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Validating our NUPACK modeling</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Expressing_our_toehold_switch_constructs"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Expressing our toehold switch constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Design_summary"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Design summary</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Anti-miRNA_design"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Anti-miRNA design</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Design_summary_2"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Design summary</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Our_changes"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Our changes</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Implementation_of_the_sensors"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Implementation of the sensors</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Design">Design</SPAN></H1><P>Toehold switches are programmable synthetic riboregulators that regulate translation in response to a specific trigger RNA.
</P><DIV id="toehold_switch" class="start"><DIV class="hairpin" style="position: absolute;left: 35%; width: 500px; top: 15px;"><P>Initially the start codon is surrounded by a strong secondary structure - <SPAN style="color:#0288d1;font-weight:800">the hairpin stem</SPAN>. As the ribosome cannot melt the stem, translation of a fluorescent reporter protein coding sequence is repressed.</P><A onclick="document.getElementById('mi_up').beginElement(); document.getElementById('toehold_switch').className = 'miRNA_bound'">Play animation</A></DIV><DIV class="rbs" style="position: absolute;left: 45%; width:500px; top: 15px;"><P>When a <SPAN style="color:#ffcc80;font-weight:800">trigger RNA</SPAN> binds to the toehold region (a linear sequence at the 5’ end of the switch) it undergoes a branch migration with the lower part of the stem. This relieves the secondary structure around the start codon. The ribosome bound to the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> can then melt the remaining stem</P><A onclick="document.getElementById('rb_ribosome').beginElement(); document.getElementById('toehold_switch').className = 'ribosome_bound'">Next</A></DIV><DIV class="open" style="position: absolute; top: 15px; left: 15px; width: 500px"><P>This activates translation and fluorescence from the <SPAN style="color:#76ff03;font-weight:800">reporter protein</SPAN> can be observed.</P><A id="reset_btn" onclick="document.getElementById('th_reset').beginElement(); document.getElementById('mi_down').beginElement(); document.getElementById('toehold_switch').className = 'closed'">Reset</A></DIV></DIV><P>We have designed genetic constructs which contain toehold switch sensors that are activated in the presence of specific microRNAs (miRNAs). The toehold switches regulate translation of the reporter protein GFP, allowing for fluorometric analysis to obtain the level of a target miRNA in a body fluid.
</P><P>Our toehold switches are based off the design parameters delineated by Green et al.<SUP id="cite_ref-green_1-0" class="reference"><A href="#cite_note-green-1">[1]</A></SUP> and Pardee et al.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> We designed two series of sequence specific toehold switches to sense our target miRNAs – hsa-miR-15b-5p and hsa-miR-27b-3p, and propose novel riboregulators with improved specificity and reaction kinetics.
</P><P>We designed our first series of toehold switches to have very low leakage. However, these switches were incapable of discriminating between miRNA sequences with close sequence homology. We therefore designed a second series of toehold switches, which our <A href="/Team:CLSB-UK/Model" title="Team:CLSB-UK/Model">modeling</A> showed are capable of achieving single base mismatch specificity.
</P><P>The software package NUPACK is capable of predicting the secondary structure of one or more interacting nucleic acid strands.<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> We used NUPACK to model our toehold switch prototypes, enabling us to obtain toehold switches that fulfil the desired parameters discussed below.
</P><H2><SPAN class="mw-headline" id="First_series_toehold_switches">First series toehold switches</SPAN></H2><P><SVG style="max-height: 300px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 208.47 146.32"><G fill="none" stroke-width="2.9" stroke-linejoin="round"><TEXT font-size="9.4" transform="translate(-238.532 -.152)" font-family="Montserrat"><TSPAN x="380.5" y="28" fill="#29b6f6">Ribosome</TSPAN><TSPAN x="382.5" y="38" fill="#29b6f6">Binding</TSPAN><TSPAN x="380.5" y="48" fill="#29b6f6">Site</TSPAN><TSPAN x="353.4" y="99.1" fill="#651fff">Start Codon</TSPAN><TSPAN x="368" y="141" fill="#f4511e" text-anchor="middle">Linker</TSPAN><TSPAN x="288" y="141" fill="#ffcc80" text-anchor="middle">Trigger Binding Site</TSPAN><TSPAN x="416.9" y="123.8" fill="#64dd17" text-anchor="middle">Protein</TSPAN><TSPAN x="417.2" y="131.8" fill="#64dd17" text-anchor="middle">coding</TSPAN><TSPAN x="417" y="139.8" fill="#64dd17" text-anchor="middle">sequence</TSPAN></TEXT></G></SVG></P><P>Toehold switches contain several predetermined regions: the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> (ribosome binding site), the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>, the 21 nucleotide (nt) <SPAN style="color:#f4511e;font-weight:800">linker sequence</SPAN> between the hairpin structure and the <SPAN style="color:#64dd17;font-weight:800">protein coding sequence</SPAN>. We developed a very simple <A href="/Team:CLSB-UK/Software/TST" title="Team:CLSB-UK/Software/TST" class="mw-redirect">software tool</A> that sped up the design process by automating these regular structures.
</P><P>We designed two sets of toehold switches for characterisation of two different reporter proteins, GFP and luciferase, to establish the most suitable reporter protein for our system. We tested GFP to determine if fluorescent proteins would be suitable for our sensor. Fluorescent proteins have a wide spectral variety and strong signal output,<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP> making them suitable for a multiplexing assay. On the other hand, luciferases are much more sensitive and have faster maturation times.<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> Although we also wanted to test luciferase <A href="/Team:CLSB-UK/Results" title="Team:CLSB-UK/Results">we found experimentally</A> that it couldn’t be inserted into our plasmid.
</P><P>We used the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> (AGAGGAGA) and <SPAN style="color:#f4511e;font-weight:800">linker sequence</SPAN> (AACCUGGCGGCAGCGCAAAAG) used by Green et al (2014). The <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> is strong and lacks a secondary structure. The <SPAN style="color:#f4511e;font-weight:800">linker sequence</SPAN> ensures correct formation of the toehold switch in both the off and on state. In the off state, this <SPAN style="color:#f4511e;font-weight:800">linker sequence</SPAN> does not bind to the toehold region, and in the on state it does not bind with the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> and <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>. Consequently, we used this linker sequence to allow for better comparison of dynamic ranges achieved with previously designed toehold switches.
</P><P><SVG style="margin: 30px auto;" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 541.08 209.2"><TEXT y=".05" font-size="9.4" font-family="Montserrat"><TSPAN x="3.9" y="156.25" fill="#ffcc80" class="tbs">Trigger binding site</TSPAN><TSPAN x="1.5" y="139.85" fill="#ffcc80" class="tbs">5'</TSPAN><TSPAN x="198.1" y="139.85" fill="#b3b3b3" class="tbs">3'</TSPAN><TSPAN x="40.5" y="114.35" fill="#66bb6a" class="ntbs">Lower stem</TSPAN><TSPAN x="12.8" y="139.45" fill="#d500f9" class="ntbs">Toehold region</TSPAN><TSPAN x="66.3" y="179.85" fill="#b3b3b3">Binds to</TSPAN><TSPAN x="1.5" y="207.35" fill="#ffcc80">Target / Trigger RNA (~22nt)</TSPAN><TSPAN x="1.5" y="191.05" fill="#ffcc80" class="tbs">3'</TSPAN><TSPAN x="130" y="191.05" fill="#ffcc80" class="tbs">5'</TSPAN></TEXT><TEXT x="280.74" y=".05" font-size="9.4" font-family="Montserrat"><TSPAN x="282.24" y="139.85" fill="#ffcc80" class="tbs">5'</TSPAN><TSPAN x="531" y="139.85" fill="#b3b3b3" class="tbs">3'</TSPAN><TSPAN x="372" y="139.45" fill="#66bb6a" class="ntbs">Lower stem</TSPAN><TSPAN x="293.54" y="139.45" fill="#d500f9" class="ntbs">Toehold region</TSPAN><TSPAN x="305" y="139.45" fill="#ffcc80" class="tbs">Trigger binding site</TSPAN><TSPAN x="303" y="163" fill="#ffcc80">Target / Trigger RNA</TSPAN><TSPAN x="282" y="163" fill="#ffcc80" class="tbs">3'</TSPAN><TSPAN x="410.74" y="163" fill="#ffcc80" class="tbs">5'</TSPAN></TEXT></SVG></P><P>The <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN> that acts as the <SPAN style="color:#ffcc80;font-weight:800">trigger RNA</SPAN> binds to the <SPAN style="color:#ffcc80;font-weight:800">trigger binding site</SPAN>. The trigger binding site is the sequence found at the 5’ end of the toehold switch and is the reverse complement of the target miRNA sequence. The trigger binding site includes the <SPAN style="color:#d500f9;font-weight:800">toehold region</SPAN> and <SPAN style="color:#66bb6a;font-weight:800">lower stem</SPAN>. Because mature miRNAs are only ~22nt in length, the entire miRNA sequence binds to the trigger binding site. Therefore, the trigger binding site and the region complementary to the trigger binding site in the stem are pre-determined sequences.
</P><P>The binding of the trigger miRNA is initiated in the toehold region. The toehold region is a linear region found at the 5’ end of the trigger binding site that is complementary to the 3’ end of the trigger miRNA (miRNAs bind to the 5’ end of the toehold switch in a 3’-5’ direction).
</P><H3><SPAN class="mw-headline" id="Toehold_length">Toehold length</SPAN></H3><P>Dynamic range in previous toehold switches increased with an increasing toehold region length, up to 12nt.<SUP id="cite_ref-green_1-1" class="reference"><A href="#cite_note-green-1">[1]</A></SUP> Longer toehold regions result in more stable miRNA-toehold duplexes, significantly reducing the risk of spontaneous dissociation of the miRNA from the toehold region.<SUP id="cite_ref-ouldridge_6-0" class="reference"><A href="#cite_note-ouldridge-6">[6]</A></SUP></P><P>We designed our toehold switches to contain longer toehold regions, preferably 12nt in length. However, this was not possible to achieve with one of the switches, for reasons discussed in the <A href="#Homologs">Homologs section</A>.
</P><H3><SPAN class="mw-headline" id="Base_of_the_stem">Base of the stem</SPAN></H3><P>The miRNA completes a branch migration with the base of the stem, resulting in strand displacement. Our switches’ toehold regions are 9 and 12nt long and the miRNAs are 22 and 21nt long respectively. The miRNAs can therefore complete a branch migration with a stem of 13 and 9nt.
</P><P>The longer the stem, the more thermodynamically stable it is in the off state. Therefore more energy is required to unravel the stem, reducing leakage in the off state. The branch migration is completed just before the mini-loop which contains the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>. This results in a smaller secondary structure around the RBS in the on state, that can easily be melted by the ribosome.
</P><H3><SPAN class="mw-headline" id="Mini-loop">Mini-loop</SPAN></H3><P><SVG style="max-height: 400px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 108.3 74.4"><G font-weight="600" text-anchor="middle" font-family="Montserrat" font-size="6.1"><TEXT font-size="6.3"><TSPAN x="76" y="4.8">Start Codon</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(59.92)"><TSPAN x="71.3" y="-57.5">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(90)"><TSPAN x="40.8" y="-88.6">U</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(130.135)"><TSPAN x="-19.1" y="-97.4">G</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(-90)"><TSPAN x="-41.4" y="63.5">G</TSPAN></TEXT><TEXT font-size="2.9" font-weight="300"><TSPAN x="75.7" y="39.4">Wobble pair</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(-57.035)"><TSPAN x="8.2" y="73.8">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".3" transform="rotate(-127.595)"><TSPAN x="-78.7" y="22.3">G</TSPAN></TEXT></G></SVG></P><P>The <SPAN style="color:#651fff;font-weight:800">start codon</SPAN> is left largely unpaired. This ensures that there is no pairing with the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN> in the on state. This is important as the miRNA is too short to be able to displace a sequence bound to the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>. However, we placed a weak wobble-pair in the mini-loop to increase stem stability in the off state. This wobble-pair breaks apart in the on state as the surrounding secondary structure is unraveled during the branch migration, so the on-state activity is not affected.
</P><H3><SPAN class="mw-headline" id="Top_of_the_stem">Top of the stem</SPAN></H3><P>Reduced G-C content in the top of the stem increases dynamic range. This is due to the decreased free energy required for the ribosome to melt the stem in the on state, as fewer bonds per nucleotide need to be broken. Therefore, we designed our switch to have only one G-C base pair at the top of the stem to ensure correct loop formation, whilst allowing for a high dynamic range to be achieved.
</P><H3><SPAN class="mw-headline" id="System_leakage">System leakage</SPAN></H3><P>Our models demonstrated that toehold switches needed to have very low levels of leakage to reduce background levels of fluorescence as we are working with small quantities of miRNAs. For more information, see our <A href="/Team:CLSB-UK/Model#MAK" title="Team:CLSB-UK/Model">mass action kinetics model</A>.
</P><P>To reduce leakage we decreased the size of the loop containing the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> and eliminated the downstream refolding domain present in the original toehold switches by Green et al.
</P><H4><SPAN class="mw-headline" id="Loop_size">Loop size</SPAN></H4><P>Decreasing loop size encourages stem formation and thus toehold switch formation.
</P><P>Decreasing loop size also decreases leakage in the off state. This is due to the reduced accessibility of the RBS to ribosomes. However, on state activity is also decreased due to reduction in loop mediated docking.
</P><P>Therefore, we designed our switches to contain a loop region of just 12-15nt long. Furthermore, we had an adenine rich pre-RBS sequence in the loop, which increases translational efficiency.<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP> This also ensured no base-pairing occurred within the loop and aided in correct formation of the smaller secondary structure in the on state.
</P><H4><SPAN class="mw-headline" id="Eliminating_downstream_refolding">Eliminating downstream refolding</SPAN></H4><P>As bases are exposed during branch migration, RNA refolding can occur. Smaller secondary structures that are able to form as a result decrease the energetic barrier to further strand displacement. This increases the rate of stem unraveling and thus switch activation. However, it reduces the free energy required to unravel the stem, increasing leakage in the off state. We therefore ensured that our sensors lack the downstream refolding domain.
</P><H2><SPAN class="mw-headline" id="Homologs">Homologs</SPAN></H2><P>We used miRBase to find mature miRNAs with similar sequences to our target miRNAs - hsa-miR-15b-5p and hsa-miR-27b-3p.<SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP><SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP> For brevity we’ve removed the hsa-miR prefix and the 3p and 5p suffix from miRNAs discussed here onwards.
</P><P>We found that both miRNAs each had one homolog that demonstrated significant sequence similarity. 15a-5p is similar to 15b-5p and 27a-3p is similar to 27b-3p.
</P><P>In our NUPACK simulations, the first series of toehold switches for 27b were unable to prevent the homolog 27a binding as 27a only differs from 27b by one nucleotide. Conversely, we were able to dramatically reduce the effects of the homolog 15a for the sensor for 15b by altering the kinetic barriers of strand displacement.
</P><P>Proximal mismatches are mismatches adjacent to the toehold region. They markedly reduce the probability of successful strand displacement. Proximal mismatches cause a homologous miRNA to immediately enter a significantly energetically less-favourable state to begin branch migration as it must enclose a mismatch.
</P><P><SVG style="max-height: 400px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 182.6 93.55"><G transform="translate(24.983 -10.85)" font-family="Montserrat"><TEXT stroke-width=".1" font-weight="300" font-size="4.23" text-anchor="middle"><TSPAN x="51.18" y="92.98"><TSPAN fill="#ffcc80" font-weight="600">A</TSPAN> can bind to <TSPAN fill="#0288d1" font-weight="600">U</TSPAN> after</TSPAN><TSPAN x="51.18" y="98.27">breaking just the <TSPAN fill="#0288d1" font-weight="600">U-A</TSPAN></TSPAN><TSPAN x="51.18" y="103.56">bond, releasing energy</TSPAN></TEXT><TEXT stroke-width=".18" font-weight="600" font-size="6.35" text-anchor="middle"><TSPAN x="50.99" y="15.78">Target miRNA</TSPAN></TEXT><TEXT stroke-width=".18" font-weight="600" font-size="6.35" text-anchor="middle"><TSPAN x="124.83" y="15.78">Homologous miRNA</TSPAN></TEXT><TEXT stroke-width=".1" font-weight="300" font-size="4.23" text-anchor="middle"><TSPAN x="125.06" y="92.98"><TSPAN fill="#ffcc80" font-weight="600">T</TSPAN> can only bind to <TSPAN fill="#0288d1" font-weight="600">A</TSPAN> after</TSPAN><TSPAN x="125.06" y="98.27">breaking both the <TSPAN fill="#0288d1" font-weight="600">U-A</TSPAN></TSPAN><TSPAN x="125.06" y="103.56">and <TSPAN fill="#0288d1" font-weight="600">A-U</TSPAN> bonds</TSPAN></TEXT><G clip-path="url(#hm_a)"><TEXT fill="#fff" stroke-width=".14" font-weight="600" font-size="8.06" transform="rotate(13.374 47.358 1.51) scale(1.15894)" text-anchor="middle"><TSPAN x="55.22" y="59.77" style="line-height:0;text-align:center"><TSPAN>A</TSPAN></TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -135.365 -40.515) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47"><TSPAN>U</TSPAN></TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -139.96 -45.11) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -126.902 -48.978) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -131.498 -53.574) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">U</TSPAN></TEXT></G><G transform="translate(74.072)" clip-path="url(#hm_a)"><TEXT fill="#fff" stroke-width=".14" font-weight="600" font-size="8.06" transform="rotate(13.374 47.358 1.51) scale(1.15894)" text-anchor="middle"><TSPAN x="55.22" y="59.77" style="line-height:0;text-align:center">C</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".14" font-weight="600" font-size="8.06" text-anchor="middle" transform="rotate(13.374 47.358 1.51) scale(1.15894)"><TSPAN x="63.17" y="59.76" style="line-height:0;text-align:center">T</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -135.365 -40.515) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47"><TSPAN>U</TSPAN></TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -139.96 -45.11) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -126.902 -48.978) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">A</TSPAN></TEXT><TEXT fill="#fff" stroke-width=".09" font-weight="600" transform="rotate(-90 -131.498 -53.574) scale(1.47068)" text-anchor="middle" font-size="6.35"><TSPAN x="-149.09" y="95.47">U</TSPAN></TEXT></G></G></SVG></P><P>This reduces the rate of transition of the homologous miRNA from being bound only to the toehold region to also being partially bound to the stem. This causes the homologous miRNA to spend more time bound only to the toehold region. This increases the probability of the homologous miRNA detaching from the toehold region. Short toehold regions are particularly prone to this as they form fewer bonds resulting in a more transient toehold binding. Additionally, even if the homologous miRNA does partially bind to the stem, the branch point is more likely to move back down the stem due the unstable mismatch in the newly-formed duplex.<SUP id="cite_ref-ouldridge_6-1" class="reference"><A href="#cite_note-ouldridge-6">[6]</A></SUP> We used these principles to significantly decrease toehold switch activation by 15a.
</P><P>The homolog 15a contains three consecutive mismatches at its 3’ end and one mismatch centrally located in the miRNA. To position the central mismatch at the base of the stem (a proximal mismatch), the stem length was increased to 13nt. As a result, the first codon after the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN> was a stop codon. Therefore, we removed the first nucleotide of the stop codon, resulting in a stem length of 12nt. Consequently, the miRNA has an overhang of one nucleotide. This left a toehold region 9nt long.
</P><P>Since 15a has three mismatches at its 3’ end, it can only bind to 6nt of the toehold region. The short toehold region results in a less stable miRNA-toehold duplex. Additionally, 15a is forced to spend a considerable amount of time weakly bound to only the toehold region due to the proximal mismatch. The combination of these factors drastically increases the probability of 15a detaching from the toehold. The number of toehold switches activated by 15a is therefore made negligible. 
</P><P>The increased stem length also makes the toehold switch more stable, which reduces leakage. However, it also makes it harder to complete strand displacement, thus reducing on state activity. Therefore we used a slightly larger loop size of 15nt to counteract the increased stem stability.
</P><P>A further strategy to increase the specificity of the first series of toehold switches involves using CRISPR C2c2. C2c2 is a CRISPR effector protein that can be programmed to cleave single stranded RNA with single base mismatch specificity.<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP><SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP> It may therefore be possible to use CRISPR C2c2 to cleave homologous miRNAs to prevent them from activating the toehold switch. However, miRNAs may be too short to be targeted by C2c2 and the additional use of CRISPR C2c2 increases the cost of the system.
</P><H3><SPAN class="mw-headline" id="Validating_our_NUPACK_modeling">Validating our NUPACK modeling</SPAN></H3><P>Green et al. found that the term ∆GRBS-Linker is the best indicator of switch performance.<SUP id="cite_ref-green_1-2" class="reference"><A href="#cite_note-green-1">[1]</A></SUP> The ∆GRBS-Linker is the free energy of the structure formed by the sequence immediately after the trigger binding site through to the end of the 21nt linker. Typically, the smaller the |∆GRBS-Linker| is, the higher the dynamic range. We found that our |∆GRBS-Linker| values were quite small, so we therefore expected to achieve quite large dynamic ranges.
</P><P>Our experiments back this up. See our <A href="/Team:CLSB-UK/Results" title="Team:CLSB-UK/Results">results page</A> for more information.
</P><H3><SPAN class="mw-headline" id="Expressing_our_toehold_switch_constructs">Expressing our toehold switch constructs</SPAN></H3><P>To express our toehold switch constructs, we used the inducible promoter BBa_K808000. This was necessary as we found that the transcripts produced from our constructs were toxic to <I>E. coli</I>, making successful amplification impossible. Therefore, we chose BBa_K808000, a promoter with low levels of leakage, to express our parts, allowing for amplification in <I>E. coli</I>, although we still had some complications. In future designs, we recommend using a promoter that has even lower leakage than BBa_K808000 (e.g. BBa_K1067007) to allow for successful amplification of the toehold switch constructs before use in the cell free system.
</P><H3><SPAN class="mw-headline" id="Design_summary">Design summary</SPAN></H3><DIV class="image_container"><SVG style="width: 27%; margin-right: 3%; margin-top: 20px" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 206.9 146.35"><G fill="none" stroke-width="2.9" stroke-linejoin="round"><TEXT y=".05" font-size="9.4" font-family="Montserrat" transform="translate(-240.003 -.002)"><TSPAN x="380.5" y="28.05" fill="#29b6f6">Ribosome</TSPAN><TSPAN x="382.5" y="38.05" fill="#29b6f6">Binding</TSPAN><TSPAN x="380.5" y="48.05" fill="#29b6f6">Site</TSPAN><TSPAN x="353.4" y="99.15" fill="#651fff">Start Codon</TSPAN></TEXT></G></SVG><DIV><P>Both switches featured the same <SPAN style="color:#0288d1;font-weight:800">conserved region</SPAN> at the top of the hairpin module (GGAUUUACAAAA<SPAN style="color:#29b6f6;font-weight:800">AGAGGAGA</SPAN>GUAAA<SPAN style="color:#651fff;font-weight:800">AUG</SPAN>). This sequence contains the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> and <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>. This region of the toehold switch is not affected by the invading miRNA strand and forms the low secondary structure in the on state. The other parts of the switch were designed for a specific miRNA and to have low system leakage.
</P><P>In future designs, we recommend adjusting the linker sequence in the toehold switch sequence to minimize secondary structure formation in the on state. This helps increase on-state activity and dynamic range. 
</P></DIV></DIV><DIV class="image_container"><DIV><P>We also recommend adding a base-pair or two after the trigger binding site in the stem. By increasing stem length, lower leakage can be achieved and on state activity is not reduced. This is explained by the fact that when the branch migration point has progressed most of the way up the base of the stem, spontaneous detachment of the last few base-pairs is likely to occur.<SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP> This is because the strand being displaced is so weakly attached, that it is unable to stably interact with the region it was bound to. This feature provides a ‘shortcut’ for strand displacement and means that the trigger RNA does not need to bind to all of base of the stem to completely unravel it. The extra one or two base pairs will help to reduce leakage in the off state. Since these last base pairs are disrupted during the branch migration and don’t stably reform in the on state, they do not affect the |∆GRBS-Linker| and on state activity, thus increasing dynamic range.
</P></DIV><SVG style="width: 27%; margin-left: 3%; margin-top: 40px" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 126.9 77.11"><G fill="none" stroke-width="2.9" stroke-linejoin="round"><TEXT x="53.318" y="39.244" fill="#000" font-size="9.4" font-family="Montserrat" text-anchor="end"><TSPAN x="53.318" y="39.244">Extra pairs</TSPAN></TEXT></G></SVG></DIV><H2 style="clear: both;"><SPAN class="mw-headline" id="Second_series_of_toehold_switches">Second series  toehold switches</SPAN></H2><P>Our sensors must be able to distinguish between the target miRNAs and homologs. Therefore, we designed a second series of switches that can distinguish between two strands which differ by just one nucleotide. Our second series of toehold switches utilize a mechanism that colocalizes several RNA molecules to activate a single toehold switch<SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP>.
</P><P>Toehold switches are typically activated by a single trigger RNA. However, it is possible to divide the trigger RNA sequence into separate RNA molecules. The RNA molecules contain hybridization domains that can be used to join them together to form the complete trigger RNA sequence. As a result, the toehold switch sequence will only be activated when all the separate RNA molecules combine.
</P><DIV class="image_container"><SVG style="width: 29%; margin-right: 1%; margin-bottom: 5px" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 167 178"><TEXT fill="url(#ss_f)" font-size="9.4" font-family="Montserrat" text-anchor="middle"><TSPAN x="47.8" y="70.4">Trigger Binding Site</TSPAN></TEXT><TEXT font-size="9.4" font-family="Montserrat"><TSPAN x="69.6" y="96.6" fill="#b3b3b3">Binds to</TSPAN></TEXT><TEXT fill="#c2185b" font-size="9.4" font-family="Montserrat" text-anchor="middle"><TSPAN x="36.3" y="124.6">anti-miRNA</TSPAN></TEXT><TEXT fill="#ffcc80" font-size="9.4" font-family="Montserrat"><TSPAN x="93.2" y="124.6">Target</TSPAN></TEXT><TEXT fill="#ffcc80" font-size="9.4" font-family="Montserrat"><TSPAN x="93.1" y="135">miRNA</TSPAN></TEXT><TEXT fill="#ba68c8" font-size="9.4" font-family="Montserrat" text-anchor="end"><TSPAN x="68.7" y="145.6">hybridization</TSPAN></TEXT><TEXT fill="#ba68c8" font-size="9.4" font-family="Montserrat" text-anchor="end"><TSPAN x="68.7" y="154">domain</TSPAN></TEXT></SVG><DIV><P>In our second series of switches, we use an <SPAN style="color:#c2185b;font-weight:800">anti-miRNA</SPAN> molecule that binds to a <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN> to activate our toehold switches. The target miRNA contains one half of the trigger sequence and the anti-miRNA contains the other half of the trigger sequence. The miRNA and anti-miRNA strands <SPAN style="color:#ba68c8;font-weight:800">hybridize</SPAN> to form the complete trigger sequence, allowing for translational activation.
</P><P>This new series of switches are much more specific due to the hybridization step of the separate RNA molecules. We modeled the second series of switches using NUPACK and found that for a miRNA to bind to the anti-miRNA and form a stable duplex at around 35°C (varies based on G-C content and position of mismatch), the hybridization domain needs to be at least 12 base pairs long. Hence, we designed our anti-miRNAs to contain a 12 base pair hybridization domain that is specific to the target miRNA. Consequently, homologous miRNAs will not bind to the anti-miRNA as mismatches prevent the formation of the 12 base pair duplex. This results in single-base mismatch specificity, thus removing the issue of homologous miRNAs that bind to the toehold switch and produce false positives.
</P></DIV></DIV><P>The anti-miRNAs could be produced in the cell free system alongside the toehold switches. To conserve resources in the cell free system, an inducible promoter should be used for expressing the anti-miRNA sequence. We recommend using the same inducible promoter for both the anti-miRNA and toehold switch for ease. Since, the toehold switch transcripts are toxic in <I>E. coli</I>, this promoter must have very low leakage. For reference, we found BBa_K808000 to be too leaky when expressing our part BBa_K2206007. 
</P><H3><SPAN class="mw-headline" id="Anti-miRNA_design">Anti-miRNA design</SPAN></H3><P>The anti-miRNA can be designed to bind to either the 5’ or 3’ end of the target miRNA. The end chosen depends on the nucleotides that differ between the target miRNA and its homologs. If the differing nucleotides are in the 5’ end of the miRNAs, then the anti-miRNA binds to the 5’ end and vice versa. This is because the miRNA to anti-miRNA hybridization step is the source of the single base mismatch specificity of the new toehold switches.
</P><P><SVG style="max-width: 500px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 148.1 68.2"><TEXT fill="#c2185b" font-size="9.4" font-family="Montserrat" text-anchor="middle"><TSPAN x="36.3" y="14.8">anti-miRNA</TSPAN></TEXT><TEXT fill="#ffcc80" font-size="9.4" font-family="Montserrat"><TSPAN x="93.2" y="14.8">Target</TSPAN></TEXT><TEXT fill="#ffcc80" font-size="9.4" font-family="Montserrat"><TSPAN x="93.1" y="25.2">miRNA</TSPAN></TEXT><TEXT fill="#ba68c8" font-size="9.4" font-family="Montserrat" text-anchor="end"><TSPAN x="68.7" y="35.8">hybridization</TSPAN></TEXT><TEXT fill="#ba68c8" font-size="9.4" font-family="Montserrat" text-anchor="end"><TSPAN x="68.7" y="44.2">domain</TSPAN></TEXT></SVG></P><P>The only sequence constraint imposed upon the <SPAN style="color:#c2185b;font-weight:800">anti-miRNA</SPAN> is the <SPAN style="color:#ba68c8;font-weight:800">hybridization domain</SPAN>, which must be perfectly complementary to the 12nt of the <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN> that it binds to. The rest of the anti-miRNA can be programmed to contain any sequence of any length.
</P><P>Toehold switches are designed based on a specific trigger RNA sequence. The non-hybridization region of the anti-miRNA has no sequence constraints, so the region of the toehold switch complementary to it is fully programmable.
</P><P><SVG style="max-height: 400px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 612.9399 312.3361"><TEXT fill="#c2185b" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" text-anchor="middle" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="84.5404" y="342.2657">anti-miRNA</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="197.2734" y="342.2657">Target</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="197.0752" y="362.8707">miRNA</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" text-anchor="middle" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="596.9011" y="342.2657">anti-miRNA</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" text-anchor="end" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="484.5799" y="342.2657">Target</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.9812" font-size="18.5902" font-family="Montserrat" text-anchor="end" transform="translate(-11.18 -84.515) scale(.8857)"><TSPAN x="484.3817" y="362.8707">miRNA</TSPAN></TEXT><TEXT fill="url(#ssd_f)" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="middle"><TSPAN x="83.8132" y="123.2408">Trigger Binding Site</TSPAN></TEXT><TEXT stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="122.0003" y="169.1353" fill="#b3b3b3">Binds to</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="163.3404" y="310.1472">3'</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="end"><TSPAN x="133.2112" y="310.1472">5'</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="2.3592" y="188.5792">3'</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="end"><TSPAN x="258.4577" y="188.5792">5'</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="2.3592" y="150.0418">5'</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="end"><TSPAN x="164.0411" y="72.7918">3'</TSPAN></TEXT><TEXT fill="url(#ssd_i)" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="middle"><TSPAN x="404.8998" y="123.2408">Trigger Binding Site</TSPAN></TEXT><TEXT stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="422.0665" y="169.1353" fill="#b3b3b3">Binds to</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="566.0563" y="188.5792">5'</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="end"><TSPAN x="335.3573" y="188.5792">3'</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat"><TSPAN x="323.2706" y="150.0418">5'</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="end"><TSPAN x="484.9525" y="72.7918">3'</TSPAN></TEXT><TEXT fill="#fff" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="middle"><TSPAN x="18.2996" y="24.6201">1</TSPAN></TEXT><TEXT fill="#fff" stroke-width="1.7517" font-size="16.466" transform="rotate(.027)" font-family="Montserrat" text-anchor="middle"><TSPAN x="346.393" y="24.7952">2</TSPAN></TEXT><TEXT fill="#ffcc80" stroke-width="1.7517" transform="rotate(.027)" font-size="16.466" font-family="Montserrat"><TSPAN x="406.7736" y="310.0324">5'</TSPAN></TEXT><TEXT fill="#c2185b" stroke-width="1.7517" transform="rotate(.027)" font-size="16.466" font-family="Montserrat" text-anchor="end"><TSPAN x="462.3574" y="309.9921">3'</TSPAN></TEXT></SVG></P><P>The <SPAN style="color:#c2185b;font-weight:800">anti-miRNA</SPAN> can either bind to the 3’ or 5’ end of the <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN>. This determines the exposed sequence and thus how the toehold switch can be designed.
</P><OL><LI> If the <SPAN style="color:#c2185b;font-weight:800">anti-miRNA</SPAN> binds to the toehold region, it allows for a toehold region of any sequence and length. A programmable toehold region can be designed such that it does not form a secondary structure, and the linker does not bind to it. It also allows for high G-C content, ensuring highly stable binding of the anti-miRNA, reducing the probability of the anti-miRNA detaching.</LI><LI> If the <SPAN style="color:#c2185b;font-weight:800">anti-miRNA</SPAN> binds to the base of the stem, it allows for a stem of any sequence and length.  The length and G-C content of the stem could then be chosen to reduce leakage. The length of stem unraveled could also be changed to allow for a larger part of the teohold region to be unraveled, providing a lower  |∆GRBS-Linker|.</LI></OL><P>For both the second series switches the anti-miRNA is exposed at its 3’ end. The exposed anti-miRNA sequence therefore binds to the toehold region. This exposed sequence is 12nt long and contains all guanines to ensure stable binding to the toehold region. This also prevents secondary structure formation within the toehold region and with the linker sequence.
</P><H3><SPAN class="mw-headline" id="Design_summary_2">Design summary</SPAN></H3><P>The second series of toehold switches were forward engineered using similar design principles employed in the first series of toehold switches.
</P><DIV class="image_container"><SVG style="width: 28%; margin-right: 2%; margin-top: 20px" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 206.9 146.35"><G fill="none" stroke-width="2.9" stroke-linejoin="round"><TEXT y=".05" font-size="9.4" font-family="Montserrat" transform="translate(-240.003 -.002)"><TSPAN x="380.5" y="28.05" fill="#29b6f6">Ribosome</TSPAN><TSPAN x="382.5" y="38.05" fill="#29b6f6">Binding</TSPAN><TSPAN x="380.5" y="48.05" fill="#29b6f6">Site</TSPAN><TSPAN x="353.4" y="99.15" fill="#651fff">Start Codon</TSPAN></TEXT></G></SVG><DIV><P>The conserved region (GGAUUUACAAAA<SPAN style="color:#29b6f6;font-weight:800">AGAGGAGA</SPAN>GUAAA<SPAN style="color:#651fff;font-weight:800">AUG</SPAN>) used in the first series of toehold switches was also used in the second series. However, the 21nt linker sequence was changed to reduce secondary structure formation around the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN> to increase translation in the on state.
</P></DIV></DIV><DIV class="image_container"><DIV><P>A mismatch was introduced into the base of the stem to increase the probability of successful branch migration. However, the number of base pairs in the stem was kept the same by adding a base pair after the trigger binding site.
</P><P>For both the switches in the second series, the anti-miRNA is exposed at its 3’ end, resulting in it binding to the toehold region. Therefore, the switch was designed to contain a 12 cytosine long toehold region. This allows for highly-stable interactions between the toehold region and the anti-miRNA to prevent anti-miRNA dissociation.
</P><P>The |∆GRBS-Linker| values of the second series of switches are lower than the first series due to changes in the linker sequence, so we would expect to achieve a higher dynamic range. However, the mismatch in the base of the stem may result in slightly more system leakage.
</P></DIV><SVG style="width: 28%; margin-left: 2%; margin-top: 30px; margin-bottom: 10px;" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 126.9 77.11"><G fill="none" stroke-width="2.9" stroke-linejoin="round"><TEXT fill="#000" font-size="9.4" font-family="Montserrat" transform="translate(-280.003 -69.245)"><TSPAN x="349.86" y="135.32">Mismatch</TSPAN></TEXT></G></SVG></DIV><DIV class="image_container"><DIV><H2 id="Novel Riboregulators">Novel Riboregulators</H2><P>Molecular beacons have previously been used to efficiently and effectively quantify miRNA levels<SUP id="cite_ref-14" class="reference"><A href="#cite_note-14">[14]</A></SUP>. The molecular beacons achieved high specificity, however, they suffered from low signal-background ratios and were only capable of sensing concentrations of miRNA greater than 5nM. We propose a new form of riboregulators that combine mechanisms used in molecular beacons and toehold switches for miRNA quantification.
</P><P>Molecular beacons are single-stranded nucleic acid probes that have a hairpin structure. The <SPAN style="color:#ffcc80;font-weight:800">probe sequence</SPAN> is complementary to a target <SPAN style="color:#ffcc80;font-weight:800">nucleic acid sequence</SPAN>. Attached to the 5’ end of the hairpin loop is a <SPAN style="color:#00c853;font-weight:800">fluorophore</SPAN> and attached to the 3’ end is a <SPAN style="color:#434343;font-weight:800">quencher</SPAN>. Since the fluorophore and quencher are in close proximity of each other, the fluorophore is unable to fluoresce.
</P><P>Binding of the target nucleic acid to the loop causes the stem to break apart<SUP id="cite_ref-15" class="reference"><A href="#cite_note-15">[15]</A></SUP> due to the rigid nature of nucleic acid double helices<SUP id="cite_ref-16" class="reference"><A href="#cite_note-16">[16]</A></SUP>. As a result, the fluorophore and quencher are moved apart, allowing the fluorophore to fluoresce.
</P></DIV><SVG style="margin: 40px 0 40px 3%; width: 27%" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 139 163.7"><G fill="none" stroke-width="2.9" transform="translate(-273 29.047)" stroke-linejoin="round"><TEXT font-size="9.4" font-family="Montserrat"><TSPAN x="342.1" y="37" fill="#ffcc80" text-anchor="middle">Probe</TSPAN><TSPAN x="349.1" y="3.9" fill="#b3b3b3">Binds to</TSPAN><TSPAN x="342.3" y="-22.1" fill="#ffcc80" text-anchor="middle">Nucleic acid sequence</TSPAN><TSPAN x="330" y="128" fill="#00c853" text-anchor="end">Fluorophore</TSPAN><TSPAN x="359.7" y="128" fill="#333">Quencher</TSPAN><TSPAN x="342.4" y="46.1" fill="#ffcc80" text-anchor="middle">sequence</TSPAN></TEXT></G></SVG></DIV><P>Molecular beacons are able to distinguish between their targets more easily than linear probes (such as anti-miRNA) as there is a wider range of temperatures where the fluorescence from the target RNA (solid) is at a maximum whilst for the homologous RNAs (dashed) it is at a minimum.<SUP id="cite_ref-17" class="reference"><A href="#cite_note-17">[17]</A></SUP></P><P><SVG style="max-width: 600px; margin: 20px auto; display: block" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 130 79.5"><G font-size="4.9" font-family="Montserrat"><TEXT x="126" y="72.5" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="126" y="72.5">Temperature</TSPAN></TEXT><TEXT x="-41.6" y="98.2" transform="rotate(-90 -50.892 43.706)" text-anchor="middle"><TSPAN x="-41.6" y="98.2">Fluorescence</TSPAN></TEXT><TEXT x="141.4" y="83.7" stroke-width=".4" font-size="3.1" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="141.4" y="83.7">Target miRNA</TSPAN></TEXT><TEXT x="179.1" y="83.8" stroke-width=".4" font-size="3.1" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="179.1" y="83.8">Homologous miRNA</TSPAN></TEXT><TEXT x="193.5" y="72.5" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="193.5" y="72.5">Temperature</TSPAN></TEXT><TEXT x="-41.6" y="165.7" transform="rotate(-90 -50.892 43.706)" text-anchor="middle"><TSPAN x="-41.6" y="165.7">Fluorescence</TSPAN></TEXT><TEXT x="193.1" y="12.4" font-weight="600" font-size="8.3" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="191" y="12.4" font-size="6.7">Molecular Beacons</TSPAN></TEXT><TEXT x="125.6" y="12.4" font-size="8.3" font-weight="600" text-anchor="middle" transform="translate(-94.598 -7.187)"><TSPAN x="125.6" y="12.4" font-size="6.7">Linear probes</TSPAN></TEXT></G></SVG></P><H3><SPAN class="mw-headline" id="Our_changes">Our changes</SPAN></H3><P>A strong secondary structure in the 5’ region directly adjacent to the RBS would significantly decrease translation by preventing contacts between the ribosome’s 30S subunit and the pre-RBS region.<SUP id="cite_ref-18" class="reference"><A href="#cite_note-18">[18]</A></SUP></P><P><SVG style="max-width: 900px; margin: 30px auto; display: block" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 478.7 273.9"><G stroke-width=".7" font-size="9.4" font-family="Montserrat"><TEXT fill="#b3b3b3" transform="translate(0 -.025)"><TSPAN x="170" y="31.5">Binds to</TSPAN></TEXT><TEXT fill="#ffcc80" transform="translate(0 -.025)"><TSPAN x="163.3" y="69.7" text-anchor="middle">Probe</TSPAN><TSPAN x="163.3" y="81.4" text-anchor="middle">sequence</TSPAN></TEXT><TEXT fill="#ffcc80" transform="translate(0 -.025)"><TSPAN x="162.9" y="7" text-anchor="middle">Nucleic acid sequence</TSPAN></TEXT><TEXT fill="#29b6f6" transform="translate(0 -.025)"><TSPAN x="202.6" y="127.9" text-anchor="middle">Ribosome</TSPAN><TSPAN x="202.6" y="139.7" text-anchor="middle">binding site</TSPAN></TEXT><TEXT fill="#651fff" transform="translate(0 -.025)"><TSPAN x="273.8" y="116.1" text-anchor="middle">Start</TSPAN><TSPAN x="273.8" y="127.9" text-anchor="middle">codon</TSPAN></TEXT><TEXT fill="#f4511e" transform="translate(0 -.025)"><TSPAN x="298.5" y="139.7" text-anchor="middle">Linker</TSPAN></TEXT><TEXT fill="#64dd17" transform="translate(0 -.025)"><TSPAN x="350.8" y="127.9" text-anchor="middle">Protein</TSPAN><TSPAN x="350.8" y="139.7" text-anchor="middle">coding region</TSPAN></TEXT><TEXT fill="#ffcc80" transform="translate(0 -.025)"><TSPAN x="115.6" y="272.1" text-anchor="middle">Probe sequence</TSPAN></TEXT><TEXT fill="#ffcc80" transform="translate(0 -.025)"><TSPAN x="115.6" y="250.9" text-anchor="middle">Nucleic acid sequence</TSPAN></TEXT><TEXT fill="#29b6f6" transform="translate(0 -.025)"><TSPAN x="293.4" y="244.4" text-anchor="middle">Ribosome</TSPAN><TSPAN x="293.4" y="256.2" text-anchor="middle">binding site</TSPAN></TEXT><TEXT fill="#651fff" transform="translate(0 -.025)"><TSPAN x="364.5" y="232.6" text-anchor="middle">Start</TSPAN><TSPAN x="364.5" y="244.4" text-anchor="middle">codon</TSPAN></TEXT><TEXT fill="#f4511e" transform="translate(0 -.025)"><TSPAN x="389.2" y="256.2" text-anchor="middle">Linker</TSPAN></TEXT><TEXT fill="#64dd17" transform="translate(0 -.025)"><TSPAN x="441.5" y="244.4" text-anchor="middle">Protein</TSPAN><TSPAN x="441.5" y="256.2" text-anchor="middle">coding region</TSPAN></TEXT></G></SVG></P><P>Therefore, a programmable hairpin loop immediately upstream of the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN>, that changes conformation in response to an <SPAN style="color:#ffcc80;font-weight:800">arbitrary RNA sequence</SPAN>, could be used to regulate translation of a fluorescent or luminescent reporter protein.
</P><P>The loop of the hairpin structure should be complementary to part or all of the <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN>. When the <SPAN style="color:#ffcc80;font-weight:800">target miRNA</SPAN> binds to the loop, the arms of the hairpin structure are pried apart. This relieves the secondary structure immediately upstream of the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN>, allowing for translation of a reporter protein. Consequently fluorescence or luminescence intensity is indicative of miRNA concentration.
</P><P>Due to the complete lack of sequence constraints, it is possible to have no secondary structure around the <SPAN style="color:#651fff;font-weight:800">start codon</SPAN>, increasing on state activity. Furthermore, the lack of branch migration results in more favourable reaction kinetics. Finally, the high specificity allows for distinction between miRNAs with close sequence homology.
</P><P>We believe that we could achieve substantially larger fold changes with these new riboregulators in comparison to our first series of toehold switches. The larger fold change and use of luciferase could allow for much miRNA lower concentrations of miRNAs to be measured. Therefore, these new riboregulators would be well suited for sensitive and specific miRNA quantification.
</P><H2><SPAN class="mw-headline" id="Implementation_of_the_sensors">Implementation of the sensors</SPAN></H2><P>Constructs containing a toehold switch and reporter protein can be embedded into paper along with transcription and translation machinery from <I>E. coli</I> by freeze-drying, resulting in a cheap and safe test that can even be used in LEDCs. A cell free sensor also allows for easy addition of miRNAs. Each strip of paper could contain multiple wells, each containing a toehold switch for a specific miRNA. However, toehold switches have very low levels of crosstalk<SUP id="cite_ref-green_1-3" class="reference"><A href="#cite_note-green-1">[1]</A></SUP>, which make it possible to create a multiplexing assay. By having multiple switches that each regulate the production of a different reporter protein with a distinct emission peak in one well, the efficiency of the test would increase drastically.
</P><P>See our <A href="/Team:CLSB-UK/HP/Silver#Clinical_Implementation" title="Team:CLSB-UK/HP/Silver">silver human practices page</A> to learn more about the clinical implementation of our project.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL class="references"><LI id="cite_note-green-1"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-green_1-0">1.0</A></SUP><SUP><A href="#cite_ref-green_1-1">1.1</A></SUP><SUP><A href="#cite_ref-green_1-2">1.2</A></SUP><SUP><A href="#cite_ref-green_1-3">1.3</A></SUP></SPAN><SPAN class="reference-text">Green, A. A., Silver, P. A., Collins, J. J., &amp; Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell, 159(4), 925-939.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., ... &amp; Daringer, N. M. (2016). Rapid, low-cost detection of Zika virus using programmable biomolecular components. Cell, 165(5), 1255-1266.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Zadeh, J. N., Steenberg, C. D., Bois, J. S., Wolfe, B. R., Pierce, M. B., Khan, A. R., ... &amp; Pierce, N. A. (2011). NUPACK: analysis and design of nucleic acid systems. Journal of computational chemistry, 32(1), 170-173.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">(n.d.). The Advantages of Fluorescent Proteins over Luciferase for In Vivo Imaging. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="https://www.uvp.com/pdf/FP129.pdf">https://www.uvp.com/pdf/FP129.pdf</A></SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Zhang, Y., Phillips, G. J., &amp; Yeung, E. S. (2008). Quantitative imaging of gene expression in individual bacterial cells by chemiluminescence. Analytical chemistry, 80(3), 597-605.</SPAN></LI><LI id="cite_note-ouldridge-6"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-ouldridge_6-0">6.0</A></SUP><SUP><A href="#cite_ref-ouldridge_6-1">6.1</A></SUP></SPAN><SPAN class="reference-text">Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., &amp; Turberfield, A. J. (2013). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324-5324.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Vimberg, V., Tats, A., Remm, M., &amp; Tenson, T. (2007). Translation initiation region sequence preferences in Escherichia coli. BMC molecular biology, 8(1), 100.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Kozomara, A., &amp; Griffiths-Jones, S. (2013). miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic acids research, 42(D1), D68-D73.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Griffiths-Jones, S., Grocock, R. J., Van Dongen, S., Bateman, A., &amp; Enright, A. J. (2006). miRBase: microRNA sequences, targets and gene nomenclature. Nucleic acids research, 34(suppl_1), D140-D144.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Abudayyeh, O. O., Gootenberg, J. S., Konermann, S., Joung, J., Slaymaker, I. M., Cox, D. B., ... &amp; Severinov, K. (2016). C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science, 353(6299), aaf5573.</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Gootenberg, J. S., Abudayyeh, O. O., Lee, J. W., Essletzbichler, P., Dy, A. J., Joung, J., ... &amp; Myhrvold, C. (2017). Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, eaam9321.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., &amp; Turberfield, A. J. (2013). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324-5324.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">Green, A. A., Kim, J., Ma, D., Silver, P. A., Collins, J. J., &amp; Yin, P. (2016, September). Ribocomputing devices for sophisticated in vivo logic computation. In Proceedings of the 3rd ACM International Conference on Nanoscale Computing and Communication (p. 11). ACM.</SPAN></LI><LI id="cite_note-14"><SPAN class="mw-cite-backlink"><A href="#cite_ref-14">↑</A></SPAN><SPAN class="reference-text">Baker, M. B., Bao, G., &amp; Searles, C. D. (2011). In vitro quantification of specific microRNA using molecular beacons. Nucleic acids research, 40(2), e13-e13.</SPAN></LI><LI id="cite_note-15"><SPAN class="mw-cite-backlink"><A href="#cite_ref-15">↑</A></SPAN><SPAN class="reference-text">Tyagi, S., &amp; Kramer, F. R. (1996). Molecular beacons: probes that fluoresce upon hybridization. Nature biotechnology, 14(3), 303-308.</SPAN></LI><LI id="cite_note-16"><SPAN class="mw-cite-backlink"><A href="#cite_ref-16">↑</A></SPAN><SPAN class="reference-text"> Shore, D., Langowski, J., &amp; Baldwin, R. L. (1981). DNA flexibility studied by covalent closure of short fragments into circles. Proceedings of the National Academy of Sciences, 78(8), 4833-4837.</SPAN></LI><LI id="cite_note-17"><SPAN class="mw-cite-backlink"><A href="#cite_ref-17">↑</A></SPAN><SPAN class="reference-text">Bonnet, G., Tyagi, S., Libchaber, A., &amp; Kramer, F. R. (1999). Thermodynamic basis of the enhanced specificity of structured DNA probes. Proceedings of the National Academy of Sciences, 96(11), 6171-6176.</SPAN></LI><LI id="cite_note-18"><SPAN class="mw-cite-backlink"><A href="#cite_ref-18">↑</A></SPAN><SPAN class="reference-text">Malmgren, C., Engdahl, H. M., Romby, P., &amp; Wagner, E. G. (1996). An antisense/target RNA duplex or a strong intramolecular RNA structure 5'of a translation initiation signal blocks ribosome binding: the case of plasmid R1. Rna, 2(10), 1022-1032.</SPAN></LI></OL></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>