<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="jumbotron full_width" style="background-image: url('https://static.igem.org/mediawiki/2017/5/57/T--CLSB-UK--notebook_cover.jpg');"><DIV><H1 style="font-size: 89px">Notebook</H1><H2 style="font-size: 24px">Let it glow! (well, technically fluoresce)</H2></DIV></DIV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Notebook"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Notebook</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#23_March"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">23 March</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#30_March"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">30 March</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#22_June"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">22 June</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#26_June"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">26 June</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#27_June"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">27 June</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#29_June"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">29 June</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#30_June"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">30 June</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#03_June"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">03 June</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#04_July"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">04 July</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#05_July"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">05 July</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#06_July"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">06 July</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#10_July"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">10 July</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#11_July"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">11 July</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#12_July"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">12 July</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#13_July"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">13 July</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#14_July"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">14 July</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#15_July"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">15 July</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#17_July"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">17 July</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#18_July"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">18 July</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#19_July"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">19 July</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#20_July"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">20 July</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#24_July"><SPAN class="tocnumber">1.22</SPAN><SPAN class="toctext">24 July</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#25_July"><SPAN class="tocnumber">1.23</SPAN><SPAN class="toctext">25 July</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#26_July"><SPAN class="tocnumber">1.24</SPAN><SPAN class="toctext">26 July</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#11_September"><SPAN class="tocnumber">1.25</SPAN><SPAN class="toctext">11 September</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#20_September"><SPAN class="tocnumber">1.26</SPAN><SPAN class="toctext">20 September</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#21_September"><SPAN class="tocnumber">1.27</SPAN><SPAN class="toctext">21 September</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#22_September"><SPAN class="tocnumber">1.28</SPAN><SPAN class="toctext">22 September</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#25_September"><SPAN class="tocnumber">1.29</SPAN><SPAN class="toctext">25 September</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#26_September"><SPAN class="tocnumber">1.30</SPAN><SPAN class="toctext">26 September</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#02_October"><SPAN class="tocnumber">1.31</SPAN><SPAN class="toctext">02 October</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#03_October"><SPAN class="tocnumber">1.32</SPAN><SPAN class="toctext">03 October</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#04_October"><SPAN class="tocnumber">1.33</SPAN><SPAN class="toctext">04 October</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#05_October"><SPAN class="tocnumber">1.34</SPAN><SPAN class="toctext">05 October</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#06_October"><SPAN class="tocnumber">1.35</SPAN><SPAN class="toctext">06 October</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#12_October"><SPAN class="tocnumber">1.36</SPAN><SPAN class="toctext">12 October</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#13_October"><SPAN class="tocnumber">1.37</SPAN><SPAN class="toctext">13 October</SPAN></A></LI></UL></DIV><H1><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H1><H2><SPAN class="mw-headline" id="23_March">23 March</SPAN></H2><P>First lab preparation session. Practiced pipetting different volumes into eppendorfs using micropipettes.
</P><H2><SPAN class="mw-headline" id="30_March">30 March</SPAN></H2><P>Second lab preparation session. Made bacterial streaks and lawns and learnt about sterile technique.
</P><H2><SPAN class="mw-headline" id="22_June">22 June</SPAN></H2><P>Starting the real lab work! Amplifying the parts from IDT with PCR and running the gels to check if we had amplified a sufficient amount, unfortunately, this PCR did not go to well, so we will try again. These were the conditions used for the first round of PCR reactions:
</P><TABLE><TBODY><TR><TH> Component </TH><TH colspan="2"> Volume for the 25Î¼L reaction (Î¼L)
</TH></TR><TR><TD> GFP/ Luciferase </TD><TD> Basic blocks
</TD></TR><TR><TD> Q5 High-Fidelity 2X Master Mix </TD><TD> 12.5 </TD><TD> 12.5
</TD></TR><TR><TD> 10uM forward primer </TD><TD> 1.25 </TD><TD> 1.25
</TD></TR><TR><TD> 10uM reverse primer </TD><TD> 1.25 </TD><TD> 1.25
</TD></TR><TR><TD> Template DNA (1ng) </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> Nuclease-free water </TD><TD> 9 </TD><TD> 9
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH> Step </TH><TH> Temperature (s) </TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturing </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD> 25 cycles </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 62 </TD><TD> 20
</TD></TR><TR><TD> 72 </TD><TD> 30
</TD></TR><TR><TD> Final extension </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> Hold </TD><TD> 4 </TD><TD> Indefinite
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="26_June">26 June</SPAN></H2><P>Second round of PCR, this time we were met with more success. We slightly modified the conditions the second time in order to increase the yield: 
</P><TABLE><TBODY><TR><TH> Component </TH><TH> Volume for the 25Î¼L reaction (Î¼L) </TH><TH> Volume for the 50Î¼L reaction (Î¼L)
</TH></TR><TR><TD> Q5 High-Fidelity 2X Master Mix </TD><TD> 12.5 </TD><TD> 25
</TD></TR><TR><TD> 10uM forward primer </TD><TD> 1.25 </TD><TD> 2.5
</TD></TR><TR><TD> 10uM reverse primer </TD><TD> 1.25 </TD><TD> 2.5
</TD></TR><TR><TD> Template DNA (1ng) </TD><TD> 2.5 </TD><TD> 5
</TD></TR><TR><TD> Nuclease-free water </TD><TD> 7.5 </TD><TD> 15
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH> Step </TH><TH> Temperature (s) </TH><TH> Time (s)
</TH></TR><TR><TD> Initial denaturing </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD> 25 cycles </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 64 </TD><TD> 25
</TD></TR><TR><TD> 72 </TD><TD> 30
</TD></TR><TR><TD> Final extension </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> Hold </TD><TD> 4 </TD><TD> Indefinite
</TD></TR></TBODY></TABLE><P>These were the results of the second round of PCR:
</P><P><SVG style="max-height: 600px;" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 175.9 214.8"><G transform="translate(-25.003 -15.71)"><G fill="#fff"><TEXT x="34.5" y="57" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="34.5" y="57" font-family="Montserrat">Ladder</TSPAN></TEXT><TEXT x="68.9" y="92.2" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="68.9" y="92.2" font-family="Montserrat">15GFP</TSPAN></TEXT><TEXT x="81.2" y="66.8" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="81.2" y="66.8" font-family="Montserrat">27GFP</TSPAN></TEXT><TEXT x="101.5" y="41.1" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="101.5" y="41.1" font-family="Montserrat">15Luc</TSPAN></TEXT><TEXT x="155.5" y="88.4" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="155.5" y="88.4" font-family="Montserrat">27Basic</TSPAN></TEXT><TEXT x="140" y="70.8" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="140" y="70.8" font-family="Montserrat">15Basic</TSPAN></TEXT><TEXT x="126.1" y="55.3" stroke-width=".3" font-size="10.6" font-family="sans-serif" letter-spacing="0" word-spacing="0"><TSPAN x="126.1" y="55.3" font-family="Montserrat">27Luc</TSPAN></TEXT></G></G></SVG></P><H2><SPAN class="mw-headline" id="27_June">27 June</SPAN></H2><P>PCR cleanup and double digests were a great success, our banding patterns were very good when we ran the gels, but the lane containing the digest from the linearized plasmid backbone was empty. These were the reagents for the digest of the gBlocks: 
</P><TABLE><TBODY><TR><TH> Reagent </TH><TH> Volume for the 25Î¼L reaction (Î¼L) Reagent Volumes (Âµl)
</TH></TR><TR><TD> Buffer (10x) </TD><TD> 5
</TD></TR><TR><TD> DNA </TD><TD> 43
</TD></TR><TR><TD> Restriction Enzyme EcoRI </TD><TD> 1
</TD></TR><TR><TD> Restriction Enzyme SpeI </TD><TD> 1
</TD></TR><TR><TD> Nuclease Free Water (Î¼l) </TD><TD> 0
</TD></TR><TR><TD> Total Volume (Î¼l) </TD><TD> 50
</TD></TR></TBODY></TABLE><P>
Recommended maximum of 1 Âµg of DNA per 10 units of enzyme.
</P><P>These were the reagents for the digest of the plasmid, BBa_J04450: 
</P><TABLE><TBODY><TR><TH> Reagent </TH><TH> Reagent Volumes (Âµl)
</TH></TR><TR><TD> Buffer (10x) </TD><TD> 3
</TD></TR><TR><TD> DNA * </TD><TD> 4
</TD></TR><TR><TD> Restriction Enzyme EcoRI </TD><TD> 1.1
</TD></TR><TR><TD> Restriction Enzyme PstI </TD><TD> 1.1
</TD></TR><TR><TD> Deionized Water (Î¼l) </TD><TD> 19.8
</TD></TR><TR><TD> Total Volume (Î¼l) </TD><TD> 30
</TD></TR></TBODY></TABLE><UL><LI> Recommended maximum of 1 Âµg of substrate per 10 units of enzyme.</LI></UL><P>28 June - Repeated digests for plasmids and this time we could easily identify the bands which corresponded to the plasmid backbone. We carried out the ligation protocol as shown below: 
</P><TABLE><TBODY><TR><TH> Component </TH><TH> Volume (Î¼l) </TH><TH> Volume (Î¼l) </TH><TH> Volume (Î¼l) </TH><TH> Volume (Î¼l) </TH><TH> Volume (Î¼l) </TH><TH> Volume (Î¼l)
</TH></TR><TR><TH> GFP15b </TH><TH> GFP27b </TH><TH> Luc15b </TH><TH> Luc27b </TH><TH> 15b </TH><TH> 27b
</TH></TR><TR><TD> 10X T4 DNA Ligase Buffer </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> Vector DNA: 40 ng (0.040 pmol) </TD><TD> 9 </TD><TD> 9 </TD><TD> 9 </TD><TD> 9 </TD><TD> 9 </TD><TD> 9
</TD></TR><TR><TD> Insert DNA mass (ng) </TD><TD> 52 </TD><TD> 52 </TD><TD> 106 </TD><TD> 106 </TD><TD> 8 </TD><TD> 8
</TD></TR><TR><TD> DNA concentration (ng/ul) </TD><TD> 9.7 </TD><TD> 88.6 </TD><TD> 42 </TD><TD> 18 </TD><TD> 16 </TD><TD> 14
</TD></TR><TR><TD> Insert DNA volume </TD><TD> 6 </TD><TD> 3 </TD><TD> 5 </TD><TD> 7 </TD><TD> 4 </TD><TD> 4
</TD></TR><TR><TD> Nuclease-free water </TD><TD> 2 </TD><TD> 5 </TD><TD> 3 </TD><TD> 1 </TD><TD> 4 </TD><TD> 4
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> Total </TD><TD> 20 </TD><TD> 20 </TD><TD> 20 </TD><TD> 20 </TD><TD> 20 </TD><TD> 20
</TD></TR></TBODY></TABLE><P>Now we are ready to transform our competent cells. 
</P><H2><SPAN class="mw-headline" id="29_June">29 June</SPAN></H2><P>A small subteam traveled to Imperial university in order to transform <I>E. coli</I> with our plasmids.
</P><H2><SPAN class="mw-headline" id="30_June">30 June</SPAN></H2><P>Transformations seemed to have gone well as you can see in the image below and we have lots of colonies growing. Letâs hope the plasmids contain our inserts!
</P><H2><SPAN class="mw-headline" id="03_June">03 June</SPAN></H2><P>Successfully transformed colonies of <I>E. coli</I> from the plates were re-streaked and set up in LB broths in preparation for minipreps. These were the materials and respective quantities which made up the LB medium:
</P><TABLE><TBODY><TR><TH> Material </TH><TH> Quantity
</TH></TR><TR><TD> NaCl </TD><TD> 1g
</TD></TR><TR><TD> Bacto Tryptone </TD><TD> 6g
</TD></TR><TR><TD> Yeast Extract </TD><TD> 3g
</TD></TR><TR><TD> ddHâO </TD><TD> 600ml
</TD></TR><TR><TD> 5M NaOH </TD><TD> 100Î¼l
</TD></TR><TR><TD> 1000x Stock of Chloramphenicol (25mg/ml) </TD><TD> 600Î¼l
</TD></TR><TR><TD> Agar </TD><TD> 9g
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="04_July">04 July</SPAN></H2><P>Spinning down our broths and doing our first set of minipreps. Disaster struck at the elution step of the minipreps so we will have to set up more broths tomorrow.
</P><H2><SPAN class="mw-headline" id="05_July">05 July</SPAN></H2><P>Second round of making broths and re-streaking our bacteria, tonight we will pray that we have learned from our mistakes and hope that our minipreps will go well tomorrow!
</P><H2><SPAN class="mw-headline" id="06_July">06 July</SPAN></H2><P>The centrifuge broke so there will be no lab work for today as the minipreps heavily rely on the centrifuge.
</P><H2><SPAN class="mw-headline" id="10_July">10 July</SPAN></H2><P>We bought a new centrifuge in order to avoid missing out on more crucial lab work sessions. Next round of minipreps, we will send off some of our plasmids to be sequenced to confirm the composition of the plasmids. Instead of using elution buffer for the last step of the minipreps we used nuclease-free water because it allows us to insert the miniprepped DNA directly into the cell-free extracts.
</P><H2><SPAN class="mw-headline" id="11_July">11 July</SPAN></H2><P>Not a high enough concentration of our plasmids to be sequenced so we need to carry out more minipreps.
</P><H2><SPAN class="mw-headline" id="12_July">12 July</SPAN></H2><P>Made more broths and re-streaked our transformed <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="13_July">13 July</SPAN></H2><P>Minipreps and sending the plasmids off for sequencing.
</P><H2><SPAN class="mw-headline" id="14_July">14 July</SPAN></H2><P>Received the results from sequencing - unfortunately, our plasmids were contaminated so we have to start from the beginning. The sequencing data indicated that there might have been some contamination of the plasmid backbone used for ligations. 
</P><H2><SPAN class="mw-headline" id="15_July">15 July</SPAN></H2><P>We tested the plasmid backbone by running it on a gel and came to the conclusion that it must be contaminated with nucleases.
</P><H2><SPAN class="mw-headline" id="17_July">17 July</SPAN></H2><P>PCR Amplification, PCR Cleanup and Fluorometer testing. 
</P><H2><SPAN class="mw-headline" id="18_July">18 July</SPAN></H2><P>Digests and ligations.
</P><H2><SPAN class="mw-headline" id="19_July">19 July</SPAN></H2><P>Prepared the selection plates for transformation	
</P><H2><SPAN class="mw-headline" id="20_July">20 July</SPAN></H2><P>Transformed competent <I>E. coli</I> with constructs. However, there were not many white colonies therefore we had to restreak these colonies using âdilution streakingâ to isolate the cells that had been transformed with our recombinant plasmid.
</P><P>This was the first selection plate and as you can see there arenât any obvious white colonies:
</P><P>We utilized âdilution streakingâ on the second selection plate and were able to isolate some white colonies:
</P><H2><SPAN class="mw-headline" id="24_July">24 July</SPAN></H2><P>Restreaks and broths
</P><H2><SPAN class="mw-headline" id="25_July">25 July</SPAN></H2><P>Minipreps and sent plasmids off for sequencing.
</P><H2><SPAN class="mw-headline" id="26_July">26 July</SPAN></H2><P>The sequencing data came back and we worked out that the construct was heavily mutated by the <I>E. coli</I>. This meant that the constructs must have been toxic to them. During the next few weeks we would be redesigning the gBlocks with an inducible promoter rather than an constitutive one. We would also have to wait for the gBlocks to be delivered by IDT. We decided to discontinue the use of Luciferase gBlocks because we were running out of time and measuring the activity of luciferase is more difficult than GFP. We also tried to clone these parts into pSB1C3 but the ligation/transformation was unsuccessful so we decided not to persist with  luciferase.
</P><H2><SPAN class="mw-headline" id="11_September">11 September</SPAN></H2><P>The new gBlocks arrived and we immediately amplified them so that we had more DNA to work with. A biology field trip meant that we were unable to do lab work for over a week.
</P><H2><SPAN class="mw-headline" id="20_September">20 September</SPAN></H2><P>We carried out ligations. The protocol was slightly modified from before and did not contain Luciferase gBlocks due to the reasons stated above:
</P><TABLE><TBODY><TR><TH> 15bÂ ! 27b </TH><TH> GFP15b </TH><TH> GFP27b
</TH></TR><TR><TD> Component </TD><TD> Volume (Î¼l) </TD><TD> Volume (Î¼l) </TD><TD> Volume (Î¼l) </TD><TD> Volume (Î¼l)
</TD></TR><TR><TD> 10X T4 DNA Ligase Buffer </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> Vector DNA: 30 ng (0.030 pmol) </TD><TD> 2.1 </TD><TD> 2.1 </TD><TD> 2.1 </TD><TD> 2.1
</TD></TR><TR><TD> Insert DNA mass (ng) </TD><TD> 6 </TD><TD> 6 </TD><TD> 90 </TD><TD> 90
</TD></TR><TR><TD> DNA concentration (ng/ul) </TD><TD> 15 </TD><TD> 15 </TD><TD> 18 </TD><TD> 18
</TD></TR><TR><TD> Insert DNA volume </TD><TD> 1 </TD><TD> 1 </TD><TD> 6 </TD><TD> 6
</TD></TR><TR><TD> Nuclease-free water </TD><TD> 4.9 </TD><TD> 4.9 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> Total </TD><TD> 10 </TD><TD> 10 </TD><TD> 10.1 </TD><TD> 10.1
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="21_September">21 September</SPAN></H2><P>Restreaks and LB Broths
</P><H2><SPAN class="mw-headline" id="22_September">22 September</SPAN></H2><P>Spinning down broths and pelleting the <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="25_September">25 September</SPAN></H2><P>We resuspended the <I>E. coli</I> pellets (first step of miniprep protocol) in the morning and then finished the procedure later. We were able to isolate plasmid DNA for all four of our constructs.
</P><H2><SPAN class="mw-headline" id="26_September">26 September</SPAN></H2><P>More minipreps and we sent off the plasmids for sequencing. Two of the constructs came back as correct. For the other two we needed to pick up a few more colonies, perform test digests and send the most promising plasmids for sequencing.
</P><H2><SPAN class="mw-headline" id="02_October">02 October</SPAN></H2><P>We setup test digests in order to work out our plasmid concentrations. This was also to see which construct needed more minipreps.
</P><H2><SPAN class="mw-headline" id="03_October">03 October</SPAN></H2><P>Setup LB Broths.
</P><H2><SPAN class="mw-headline" id="04_October">04 October</SPAN></H2><P>LB Broths were spun down. <I>E. coli</I> pellets were resuspended.
</P><H2><SPAN class="mw-headline" id="05_October">05 October</SPAN></H2><P>Minipreps
</P><H2><SPAN class="mw-headline" id="06_October">06 October</SPAN></H2><P>More minipreps
</P><H2><SPAN class="mw-headline" id="12_October">12 October</SPAN></H2><P>A small subteam traveled to Imperial University in order to conduct the penultimate protocol. This specifically involved the characterisation of our araC-Pbad promoter where we would determine the optimum L-Arabinose concentration for GFP expression. Although this promoter has been previously characterized, it had not be done using toehold switches and in a cell-free system so we thought it would be beneficial for future iGEM teams. 
Our experiment not only involved different concentrations of L-Arabinose but also positive and negative controls as shown below:
</P><TABLE><TBODY><TR><TH> Well </TH><TH> Test </TH><TH> Volume of master mix (Î¼L) </TH><TH> Substitute Nuclease Free Water (Î¼L) </TH><TH> Arabinose (Î¼L) </TH><TH> DNA Template (Î¼L)
</TH></TR><TR><TD> 1 AB </TD><TD> -ve control </TD><TD> 27 </TD><TD> 6 </TD></TR><TR><TD> 2 AB </TD><TD> -ve DNA </TD><TD> 27 </TD><TD> 3 </TD><TD> 3 AraC
</TD></TR><TR><TD> 3 AB </TD><TD> 2% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 4 AB </TD><TD> 1% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 5 AB </TD><TD> 0.50% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 6 AB </TD><TD> 0.10% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 7 AB </TD><TD> 0.05% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 8 AB </TD><TD> =+ve control </TD><TD> 27 </TD><TD> 3 </TD><TD> 3 constitutive
</TD></TR></TBODY></TABLE><P>We were able to establish that 0.5% of L-Arabinose was the optimum concentration for GFP expression.
</P><H2><SPAN class="mw-headline" id="13_October">13 October</SPAN></H2><P>Today the aim was to characterize our 15GFP toehold switch and this would also be carried out in Imperial Universityâs labs. We wanted to validate that an increase in miRNA concentration would result in an increase of GFP expression. In order to create a range of miRNA concentrations we set up a serial dilution:
</P><TABLE><TBODY><TR><TH> Tube Name </TH><TH> Stock Concentration </TH><TH> Final Concentration (/11) </TH><TH> Volume of Previous Concentration (Î¼L) </TH><TH> Volume of Nuclease Free Water (Î¼L)
</TH></TR><TR><TD> A </TD><TD> 10 Î¼M </TD><TD> 9x10â¸ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> B </TD><TD> 1 Î¼M </TD><TD> 9x10â· fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> C </TD><TD> 100 nM </TD><TD> 9x10â¶ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> D </TD><TD> 10 nM </TD><TD> 9x10âµ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> E </TD><TD> 1 nM </TD><TD> 9x10â´ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> F </TD><TD> 100 pM </TD><TD> 9000 fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> G </TD><TD> 10:00 PM </TD><TD> 900 fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> H </TD><TD> 1:00 PM </TD><TD> 90 fM </TD><TD> 10 </TD><TD> 90
</TD></TR></TBODY></TABLE><P>Now we added the miRNA to the cell-free system along with other components such as arabinose and amino acids. It was essential that we also carried out controls in order to determine factors such as background fluorescence:
</P><TABLE><TBODY><TR><TH> Wells </TH><TH> Test </TH><TH> Volume of master mix (Î¼L) </TH><TH> Substitute Nuclease Free Water (Î¼L) </TH><TH> Arabinose (Î¼L) </TH><TH> miR (Î¼L) </TH><TH> DNA Template (Î¼L)
</TH></TR><TR><TD> 1 CDE </TD><TD> -ve control </TD><TD> 24 </TD><TD> 9 </TD></TR><TR><TD> 2 CDE </TD><TD> -ve miR </TD><TD> 24 </TD><TD> 6 </TD><TD> 3 </TD></TR><TR><TD> 3 CDE </TD><TD> -ve DNA </TD><TD> 24 </TD><TD> 6 </TD><TD> 3
</TD></TR><TR><TD> 4 CDE </TD><TD> -ve mix </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 5 CDE </TD><TD> A </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 6 CDE </TD><TD> B </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 7 CDE </TD><TD> C </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 8 CDE </TD><TD> D </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 9 CDE </TD><TD> E </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 10 CDE </TD><TD> F </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 11 CDE </TD><TD> G </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 12 CDE </TD><TD> H </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 13 CDE </TD><TD> 27 </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 (miRNA27) </TD><TD> 3
</TD></TR><TR><TD> 14 CDE </TD><TD> Positive Control </TD><TD> 24 </TD><TD> 6 </TD><TD> 3
</TD></TR></TBODY></TABLE></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>