<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Amplification_of_toehold_switches"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Amplification of toehold switches</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Switch_for_15b-5p"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Switch for 15b-5p</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Leakage"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Leakage</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Specificity"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Specificity</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Error_analysis"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Error analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Modeling"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Applications"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Applications</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#NSCLC"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">NSCLC</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Wider_Uses"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Wider Uses</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Future_work"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Future work</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Achievements"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Achievements</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Conclusion"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P>We have designed a cell free biosensor that can be used to quantify miRNA levels, allowing for cheap and early diagnosis of non-small cell lung cancer (NSCLC).
</P><P>Our biosensors consist of toehold switches that are activated in the presence of the miRNAs hsa-miR-15b-5p (15b-5p) and hsa-miR-27b-3p (27b-3p). Abnormal levels of these miRNAs are indicative of NSCLC.<SUP id="cite_ref-hennessey_1-0" class="reference"><A href="#cite_note-hennessey-1">[1]</A></SUP></P><P>The toehold switches are responsible for regulating the translation of a reporter protein (GFP or luciferase). The toehold switch is typically in the off state, resulting in minimal production of the reporter protein. However, upon the addition of a trigger miRNA, the toehold switch is activated and translation of the reporter protein occurs. The number of toehold switches activated and thus the miRNA levels can be quantified based on the fluorescence or luminescence intensity from the sensors.
</P><P>Here we demonstrate the ability of toehold switches to be used in miRNA quantification and discuss the role that these sensors could play in the future.
</P><H2><SPAN class="mw-headline" id="Amplification_of_toehold_switches">Amplification of toehold switches</SPAN></H2><P>We inserted our toehold switch constructs into the pSB1C3 plasmid backbone and transformed <I>E. coli</I> with these plasmids. We used <I>E. coli</I> to amplify the plasmids before miniprepping them for a use in a cell free system.
</P><P>These constructs contained the relatively strong promoter BBa_J23111. After amplification of our constructs in <I>E. coli</I>, we picked 5 transformed colonies and sequenced the plasmids. However, the sequences did not correspond to what we were expecting.  Using BLAST sequence analysis to identify the sequences, we found parts of GFP and luciferase, but it appeared that at least some of the inserts had been excised from the plasmid. Additionally, we found mutations around the promoter regions, leading us to believe that the transcripts produced by our constructs are toxic to <I>E. coli</I>.
</P><P>To resolve this issue, we replaced BBa_J23111 with the arabinose inducible promoter BBa_K808000. This promoter was reported to have low levels of leakage and should therefore be suitable for regulating the expression of lethal parts in <I>E. coli</I>. After inserting our new constructs into <I>E. coli</I> and amplifying them, we sequenced the plasmids.
</P><P>We found that the toehold switch for 15b-5p had been successfully amplified. Unfortunately, we still found some mutations around the promoter region for 27b-3p, indicating it might be more toxic than 15b-5p. This meant even low concentrations of this part produced due to the leaky promoter were toxic to <I>E. coli</I>. Consequently, we were only able to characterize our toehold switch 15b-5p due to time constraints.
</P><P>Using BBa_K808000 we were unable to insert our constructs containing luciferase into the pSB1C3 plasmid backbone in one ligation step as the construct was too large. This meant that we were unable to characterize the 15b-5p switch with luciferase as we did not have enough time to ligate the individual components of the constructs into the pSB1C3 backbone.
</P><H2><SPAN class="mw-headline" id="Switch_for_15b-5p">Switch for 15b-5p</SPAN></H2><P>When testing the switch with 90nM of 15b-5p we found that one of our replicates was invalid (repeat 3 on graph) and as such we omitted that set of results from our analysis.
</P><P>
We found our toehold switch required a miRNA concentration of 90nM to be significantly activated, but the minimum concentration required for activation lies between 9nM and 90nM.
</P><DIV style="text-align: center;"><P style="display: inline">Set time to: </P><UL id="fluorescenceMiRNA_chart_buttons"><LI><BUTTON onclick="setFluorescenceMiRNATime(6000); return false">6,000s</BUTTON></LI><LI><BUTTON onclick="setFluorescenceMiRNATime(12000); return false">12,000s</BUTTON></LI><LI><BUTTON onclick="setFluorescenceMiRNATime(18000); return false">18,000s</BUTTON></LI><LI><BUTTON onclick="setFluorescenceMiRNATime(24000); return false">24,000s</BUTTON></LI><LI><BUTTON onclick="setFluorescenceMiRNATime(30000); return false">30,000s</BUTTON></LI><LI><BUTTON onclick="setFluorescenceMiRNATime(36000); return false">36,000s</BUTTON></LI></UL></DIV><P>
We also found that peak fluorescence increased with miRNA concentration. However, unlike with Pardee et al, we did not see a linear response in fluorescence intensity with increasing trigger RNA.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP></P><P>We believe this is because the promoter regulating our toehold switches was only activated when the arabinose was added - which was at the same time as the miRNA was added. This may have resulted in excess miRNAs relative to the number of toehold switches, particularly when 900nM of miRNA were added. Consequently, many miRNAs were not bound to the toehold switches and as such were degraded, reducing fluorescence output. To resolve this, the arabinose should be added first and the system left so that the switches could be produced before adding miRNA.
</P><P style="font-size: 12px; color: #888; text-align: center">-ve control is just the cell free extract 
and the -ve mix is the cell free extract + arabinose + DNA without miRNA</P><P>
We found our sensors were activated in under an hour and that fold-changes (the fluorescence observed with miRNA compared to the fluorescence observed without miRNA) were greater than 20 fold after 4 hours. The maximum fold-change achieved was 25.5 fold at 10 hours with 900nM of 15b-5p. We also found that the fold-change was lower when 90nM of 15b-5p was added, with a maximum fold-change achieved being 19 fold at 10 hours. Therefore, our sensors are capable of discriminating between two miRNA concentrations. As such, it could be used to discriminate between people with and without NSCLC.
</P><P>Additionally, the rate of increase in fold-change was negligible after ~7 hours with 900nM of 15b-5p, whereas fold-change was still increasing after 10 hours with 90nM of 15b-5p. The difference in fluorescence is therefore at its greatest before 7 hours.
</P><P>The large fold-changes demonstrate that our toehold switches work well, they have very low levels of leakage and they allow for easy distinction between the on and the off state. Additionally, the large fold-changes allows for discrimination of small differences in miRNA concentration, thus increasing sensitivity. 
</P><P>Moderately expressed miRNAs exhibit copy numbers of up to 133,970/μl in plasma.<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> Standard miRNA extraction kits concentrate miRNA levels in body fluids by ~20 fold during extraction.<SUP id="cite_ref-qiagen_4-0" class="reference"><A href="#cite_note-qiagen-4">[4]</A></SUP> This results in a concentration of 4.5 pM for moderately expressed miRNAs in plasma after extraction. In comparison to other body fluids, especially breast milk (~153 times more concentrated) and seminal fluid (~58 time more concentrated), total RNA concentration in plasma is low.<SUP id="cite_ref-zhang_5-0" class="reference"><A href="#cite_note-zhang-5">[5]</A></SUP> This would suggest that the miRNA concentration in plasma is also low relative to other body fluids. Therefore, although our 15b-5p GFP sensors are not suitable for quantification of miRNAs that are not highly expressed in plasma, they may be sensitive enough for use with other body fluids.
</P><P>The use of luciferase may allow for more sensitive measurements. Luciferase is far more sensitive than GFP and it is possible to detect as little as 10<SUP>-19</SUP> moles of firefly luciferases with commercial instruments.<SUP id="cite_ref-zhang_5-1" class="reference"><A href="#cite_note-zhang-5">[5]</A></SUP> We believe that if our toehold switches regulated the production of luciferase, it could allow for moderate-low abundance miRNAs in plasma and other body fluids to be quantified. 
</P><P>Furthermore, we speculate that the <A href="/Team:CLSB-UK/Design#Novel_Riboregulators" title="Team:CLSB-UK/Design">novel riboregulators</A> we proposed could have higher fold-changes. This would amplify the signal of miRNAs, permitting quantification of even lower miRNA concentrations. 
</P><H3><SPAN class="mw-headline" id="Leakage">Leakage</SPAN></H3><P>Our experiments showed that our switches had very low levels of leakiness as our set of tests containing no arabinose had lower fluorescence than the negative control (just the cell free extract with nothing in it). Nevertheless, we anticipate that there was some system leakage.
</P><P>We propose a method for increasing fold-change for lower miRNA concentrations by decreasing promoter induction. During our experiments, we added 0.5% arabinose to our cell free systems, as <A href="/Team:CLSB-UK/Contribution" title="Team:CLSB-UK/Contribution">our characterization</A> of BBa_K808000 revealed this would maximize the number of toehold switches produced. However, our <A href="/Team:CLSB-UK/Model#MAK" title="Team:CLSB-UK/Model">mass action kinetics model</A> showed that for low concentrations of miRNAs, the ratio of open toehold switches (OTS) to closed toehold switches (CTS) at 0.5% arabinose will be extremely low. Since the closed toehold switches are slightly leaky, large induction of the promoter leads to a significant increase in fluorescence due to the leakiness alone, thus resulting in a very low signal to noise ratio. Therefore, we postulate that reducing the concentration of the inducer for low concentrations of miRNAs will help to increase fold-changes, making the activation at lower concentrations more noticable.
</P><H3><SPAN class="mw-headline" id="Specificity">Specificity</SPAN></H3><P>In addition to testing our 15b-5p toehold switch with 15-5p, we tested it with 900nM of 27b-3p. This was to demonstrate that our switches were activated only in the presence of 15b-5p. We observed a mean fold-change of 1.25 throughout the 10 hours that the experiment was left to run, with the fold-change never surpassing 1.33. This data indicates that our toehold switches are only activated in the presence of the correct target miRNA. 
</P><H3><SPAN class="mw-headline" id="Error_analysis">Error analysis</SPAN></H3><P>Small variation in fluorescence is crucial for accurately distinguishing between different miRNA concentrations. Our results show that we could distinguish between less than an order of magnitude difference in miRNA concentration as the standard deviation bars do not overlap.
</P><P>Since we did triplicates and therefore only had three results (two in the case of the 90 nM test), the standard deviation could be reduced by increasing the number of replicates.
</P><P>Furthermore, we only added 1μl of miRNAs into the cell free system. Small volumes such as this are prone to variation due to pipetting errors. These errors could be reduced by automating the process.
</P><P>Finally, we do not believe that we obtained complete activation of the switches with 900nM of 15b-5p caused by simultaneously adding the arabinose and the miRNAs, explained previously. Therefore, we would expect a greater difference between the results, further aiding in discrimination between the two concentrations.
</P><H3><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H3><P>We fit our <A href="/Team:CLSB-UK/Model#Results_fitting_and_analysis" title="Team:CLSB-UK/Model">mass action kinetics model</A> to the experimental data. We were happy to see our model fit well, and the experimental data allowed us to come up with parameters other teams can use in future models of cell free systems.
</P><P>View the <A href="/Team:CLSB-UK/Model#Results_fitting_and_analysis" title="Team:CLSB-UK/Model">modeling page</A> for more information.
</P><H2><SPAN class="mw-headline" id="Applications">Applications</SPAN></H2><H3><SPAN class="mw-headline" id="NSCLC">NSCLC</SPAN></H3><P>For NSCLC, there is a difference of ~4 fold in miRNA concentration between individuals with and without cancer.<SUP id="cite_ref-zhang_5-2" class="reference"><A href="#cite_note-zhang-5">[5]</A></SUP> We are confident that we would effectively be able to differentiate between these levels based off our data. Our sensors using GFP are unsuitable for quantifying serum or plasma miRNA concentrations but we expect that by using the reporter protein luciferase, we would be able to quantify 15b-5p and 27b-3p levels.
</P><H3><SPAN class="mw-headline" id="Wider_Uses">Wider Uses</SPAN></H3><P>Up to 700μl of body fluid can be eluted into 50μl of nuclease free water during miRNA extraction.<SUP id="cite_ref-qiagen_4-1" class="reference"><A href="#cite_note-qiagen-4">[4]</A></SUP> This could then divided up into 50 wells, allowing for 50 miRNAs to be quantified. If the process were automated and therefore volumes smaller than 1μl could be accurately pippeted, more miRNAs could be quantified from the same initial volume of body fluid. Since accurate screening can require 2-3 miRNAs,<SUP id="cite_ref-hennessey_1-1" class="reference"><A href="#cite_note-hennessey-1">[1]</A></SUP><SUP id="cite_ref-pcslater_6-0" class="reference"><A href="#cite_note-pcslater-6">[6]</A></SUP> we could detect over 20 different diseases from a 700μl sample.
</P><P>To further increase the number of miRNAs that can quantified, it is possible to create a multiplexing assay using multiple toehold switches in one well. Each switch would regulate the production of a different reporter protein with a distinct emission, thereby increasing the number of miRNAs that could be quantified several fold.
</P><H2><SPAN class="mw-headline" id="Future_work">Future work</SPAN></H2><UL><LI>Testing the switch for 27b-3p</LI><LI>Characterization of the toehold switches with luciferase</LI><LI>Finding the minimum concentration of miRNA to significantly activate our switches with both GFP and luciferase</LI><LI>Testing the switches with homologous miRNAs to determine how specific the switches are</LI><LI>Testing the <A href="/Team:CLSB-UK/Design#Second_series_of_toehold_switches" title="Team:CLSB-UK/Design">second series of switches</A> and <A href="/Team:CLSB-UK/Design#Novel_Riboregulators" title="Team:CLSB-UK/Design">novel riboregulators</A></LI><LI>Testing the switches with samples obtained from patients</LI></UL><H2><SPAN class="mw-headline" id="Achievements">Achievements</SPAN></H2><UL><LI>Demonstrated our parts require the use of a non-leaky promoter due to the production of toxic transcripts</LI><LI>Demonstrated the toehold switch constructs containing GFP work in a cell free system</LI><LI>Demonstrated the 15b-5p switch is significantly activated with 90nM 15-5p miRNA</LI><LI>Demonstrated that fluorescence output increases with miRNA concentration</LI><LI>Demonstrated the 15b-5p toehold switch is only activated in the presence of specific miRNAs</LI></UL><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>Our current sensors may be capable of detecting diseases using highly expressed miRNAs from plasma or serum. However, we believe that these sensors are more suited for other body fluids with higher miRNA concentrations such as breast milk and seminal fluid.
</P><P>The use of luciferase would allow for more sensitive tests, however a key advantage of using fluorescent proteins is that they would allows for multiplexing due to their spectral diversity. Multiplexing would increase the number of miRNAs that can be detected from one sample by several fold.
</P><P>We also recommend testing our <A href="/Team:CLSB-UK/Design#Second_series_of_toehold_switches" title="Team:CLSB-UK/Design">second series of switches</A> and <A href="/Team:CLSB-UK/Design#Novel_Riboregulators" title="Team:CLSB-UK/Design">novel riboregulators</A> for their potentially increased specificity and fold-change. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL class="references"><LI id="cite_note-hennessey-1"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-hennessey_1-0">1.0</A></SUP><SUP><A href="#cite_ref-hennessey_1-1">1.1</A></SUP></SPAN><SPAN class="reference-text">Hennessey, P. T., Sanford, T., Choudhary, A., Mydlarz, W. W., Brown, D., Adai, A. T., ... &amp; Califano, J. A. (2012). Serum microRNA biomarkers for detection of non-small cell lung cancer. PloS one, 7(2), e32307.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., ... &amp; Daringer, N. M. (2016). Rapid, low-cost detection of Zika virus using programmable biomolecular components. Cell, 165(5), 1255-1266.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Mitchell, P. S., Parkin, R. K., Kroh, E. M., Fritz, B. R., Wyman, S. K., Pogosova-Agadjanyan, E. L., ... &amp; Lin, D. W. (2008). Circulating microRNAs as stable blood-based markers for cancer detection. Proceedings of the National Academy of Sciences, 105(30), 10513-10518.</SPAN></LI><LI id="cite_note-qiagen-4"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-qiagen_4-0">4.0</A></SUP><SUP><A href="#cite_ref-qiagen_4-1">4.1</A></SUP></SPAN><SPAN class="reference-text">QIAGEN (n.d.). RNA Isolation Kit: RNeasy Mini Kit. Retrieved October 8: 2017: fromhttps://www.qiagen.com/gb/shop/sample-technologies/rna/total-rna/rneasy-mini-kit/</SPAN></LI><LI id="cite_note-zhang-5"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-zhang_5-0">5.0</A></SUP><SUP><A href="#cite_ref-zhang_5-1">5.1</A></SUP><SUP><A href="#cite_ref-zhang_5-2">5.2</A></SUP></SPAN><SPAN class="reference-text">Zhang, Y., Phillips, G. J., &amp; Yeung, E. S. (2008). Quantitative imaging of gene expression in individual bacterial cells by chemiluminescence. Analytical chemistry, 80(3), 597-605.</SPAN></LI><LI id="cite_note-pcslater-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-pcslater_6-0">↑</A></SPAN><SPAN class="reference-text">Slater, E. P., Strauch, K., Rospleszcz, S., Ramaswamy, A., Esposito, I., Klöppel, G., ... &amp; Bartsch, D. K. (2014). MicroRNA-196a and-196b as potential biomarkers for the early detection of familial pancreatic cancer. Translational oncology, 7(4), 464-471.</SPAN></LI></OL></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>