Team:CLSB-UK/Design
CLSB-UK
Project
Design
Parts
Demonstrate
Judging
Contribution
Lab work
Results
Experiments
Safety
Notebook
InterLab
Human Practices
Silver
Engagement
High School Guide
Gold Integrated
Safety
Cost model
Modeling
NUPACK model
Mass action kinetics
Stochastic model
Cost model
Software
Overview
Toehold tools
SynBioBot
Hardware
Overview
Interpreter
Measurement
People
Team
Collaborations
Attributions
Sponsors
We strongly recommend using Google Chrome or Mozilla Firefox to view our wiki.
Contents
1 Design
1.1 First series toehold switches
1.1.1 Toehold length
1.1.2 Base of the stem
1.1.3 Mini-loop
1.1.4 Top of the stem
1.1.5 System leakage
1.1.5.1 Loop size
1.1.5.2 Eliminating downstream refolding
1.2 Homologs
1.2.1 Validating our NUPACK modeling
1.2.2 Expressing our toehold switch constructs
1.2.3 Design summary
1.2.4 Anti-miRNA design
1.2.5 Design summary
1.2.6 Our changes
1.3 Implementation of the sensors
1.4 References
Design
Toehold switches are programmable synthetic riboregulators that regulate translation in response to a specific trigger RNA.
Initially the start codon is surrounded by a strong secondary structure - the hairpin stem. As the ribosome cannot melt the stem, translation of a fluorescent reporter protein coding sequence is repressed.
Play animation
When a trigger RNA binds to the toehold region (a linear sequence at the 5’ end of the switch) it undergoes a branch migration with the lower part of the stem. This relieves the secondary structure around the start codon. The ribosome bound to the RBS can then melt the remaining stem
Next
This activates translation and fluorescence from the reporter protein can be observed.
Reset
We have designed genetic constructs which contain toehold switch sensors that are activated in the presence of specific microRNAs (miRNAs). The toehold switches regulate translation of the reporter protein GFP, allowing for fluorometric analysis to obtain the level of a target miRNA in a body fluid.
Our toehold switches are based off the design parameters delineated by Green et al. [1] and Pardee et al. [2] We designed two series of sequence specific toehold switches to sense our target miRNAs – hsa-miR-15b-5p and hsa-miR-27b-3p, and propose novel riboregulators with improved specificity and reaction kinetics.
We designed our first series of toehold switches to have very low leakage. However, these switches were incapable of discriminating between miRNA sequences with close sequence homology. We therefore designed a second series of toehold switches, which our modeling showed are capable of achieving single base mismatch specificity.
The software package NUPACK is capable of predicting the secondary structure of one or more interacting nucleic acid strands. [3] We used NUPACK to model our toehold switch prototypes, enabling us to obtain toehold switches that fulfil the desired parameters discussed below.
First series toehold switches
Ribosome
Binding
Site
Start Codon
Linker
Trigger Binding Site
Protein
coding
sequence
Toehold switches contain several predetermined regions: the RBS (ribosome binding site), the start codon, the 21 nucleotide (nt) linker sequence between the hairpin structure and the protein coding sequence. We developed a very simple software tool that sped up the design process by automating these regular structures.
We designed two sets of toehold switches for characterisation of two different reporter proteins, GFP and luciferase, to establish the most suitable reporter protein for our system. We tested GFP to determine if fluorescent proteins would be suitable for our sensor. Fluorescent proteins have a wide spectral variety and strong signal output, [4] making them suitable for a multiplexing assay. On the other hand, luciferases are much more sensitive and have faster maturation times. [5] Although we also wanted to test luciferase we found experimentally that it couldn’t be inserted into our plasmid.
We used the RBS (AGAGGAGA) and linker sequence (AACCUGGCGGCAGCGCAAAAG) used by Green et al (2014). The RBS is strong and lacks a secondary structure. The linker sequence ensures correct formation of the toehold switch in both the off and on state. In the off state, this linker sequence does not bind to the toehold region, and in the on state it does not bind with the RBS and start codon. Consequently, we used this linker sequence to allow for better comparison of dynamic ranges achieved with previously designed toehold switches.
Trigger binding site
5'
3'
Lower stem
Toehold region
Binds to
Target / Trigger RNA (~22nt)
3'
5'
5'
3'
Lower stem
Toehold region
Trigger binding site
Target / Trigger RNA
3'
5'
The target miRNA that acts as the trigger RNA binds to the trigger binding site. The trigger binding site is the sequence found at the 5’ end of the toehold switch and is the reverse complement of the target miRNA sequence. The trigger binding site includes the toehold region and lower stem. Because mature miRNAs are only ~22nt in length, the entire miRNA sequence binds to the trigger binding site. Therefore, the trigger binding site and the region complementary to the trigger binding site in the stem are pre-determined sequences.
The binding of the trigger miRNA is initiated in the toehold region. The toehold region is a linear region found at the 5’ end of the trigger binding site that is complementary to the 3’ end of the trigger miRNA (miRNAs bind to the 5’ end of the toehold switch in a 3’-5’ direction).
Toehold length
Dynamic range in previous toehold switches increased with an increasing toehold region length, up to 12nt. [1] Longer toehold regions result in more stable miRNA-toehold duplexes, significantly reducing the risk of spontaneous dissociation of the miRNA from the toehold region. [6]
We designed our toehold switches to contain longer toehold regions, preferably 12nt in length. However, this was not possible to achieve with one of the switches, for reasons discussed in the Homologs section .
Base of the stem
The miRNA completes a branch migration with the base of the stem, resulting in strand displacement. Our switches’ toehold regions are 9 and 12nt long and the miRNAs are 22 and 21nt long respectively. The miRNAs can therefore complete a branch migration with a stem of 13 and 9nt.
The longer the stem, the more thermodynamically stable it is in the off state. Therefore more energy is required to unravel the stem, reducing leakage in the off state. The branch migration is completed just before the mini-loop which contains the start codon. This results in a smaller secondary structure around the RBS in the on state, that can easily be melted by the ribosome.
Mini-loop
Start Codon
A
U
G
G
Wobble pair
A
G
The start codon is left largely unpaired. This ensures that there is no pairing with the start codon in the on state. This is important as the miRNA is too short to be able to displace a sequence bound to the start codon. However, we placed a weak wobble-pair in the mini-loop to increase stem stability in the off state. This wobble-pair breaks apart in the on state as the surrounding secondary structure is unraveled during the branch migration, so the on-state activity is not affected.
Top of the stem
Reduced G-C content in the top of the stem increases dynamic range. This is due to the decreased free energy required for the ribosome to melt the stem in the on state, as fewer bonds per nucleotide need to be broken. Therefore, we designed our switch to have only one G-C base pair at the top of the stem to ensure correct loop formation, whilst allowing for a high dynamic range to be achieved.
System leakage
Our models demonstrated that toehold switches needed to have very low levels of leakage to reduce background levels of fluorescence as we are working with small quantities of miRNAs. For more information, see our mass action kinetics model .
To reduce leakage we decreased the size of the loop containing the RBS and eliminated the downstream refolding domain present in the original toehold switches by Green et al.
Loop size
Decreasing loop size encourages stem formation and thus toehold switch formation.
Decreasing loop size also decreases leakage in the off state. This is due to the reduced accessibility of the RBS to ribosomes. However, on state activity is also decreased due to reduction in loop mediated docking.
Therefore, we designed our switches to contain a loop region of just 12-15nt long. Furthermore, we had an adenine rich pre-RBS sequence in the loop, which increases translational efficiency. [7] This also ensured no base-pairing occurred within the loop and aided in correct formation of the smaller secondary structure in the on state.
Eliminating downstream refolding
As bases are exposed during branch migration, RNA refolding can occur. Smaller secondary structures that are able to form as a result decrease the energetic barrier to further strand displacement. This increases the rate of stem unraveling and thus switch activation. However, it reduces the free energy required to unravel the stem, increasing leakage in the off state. We therefore ensured that our sensors lack the downstream refolding domain.
Homologs
We used miRBase to find mature miRNAs with similar sequences to our target miRNAs - hsa-miR-15b-5p and hsa-miR-27b-3p. [8] [9] For brevity we’ve removed the hsa-miR prefix and the 3p and 5p suffix from miRNAs discussed here onwards.
We found that both miRNAs each had one homolog that demonstrated significant sequence similarity. 15a-5p is similar to 15b-5p and 27a-3p is similar to 27b-3p.
In our NUPACK simulations, the first series of toehold switches for 27b were unable to prevent the homolog 27a binding as 27a only differs from 27b by one nucleotide. Conversely, we were able to dramatically reduce the effects of the homolog 15a for the sensor for 15b by altering the kinetic barriers of strand displacement.
Proximal mismatches are mismatches adjacent to the toehold region. They markedly reduce the probability of successful strand displacement. Proximal mismatches cause a homologous miRNA to immediately enter a significantly energetically less-favourable state to begin branch migration as it must enclose a mismatch.
A
can bind to
U
after
breaking just the
U-A
bond, releasing energy
Target miRNA
Homologous miRNA
T
can only bind to
A
after
breaking both the
U-A
and
A-U
bonds
A
U
A
A
U
C
T
U
A
A
U
This reduces the rate of transition of the homologous miRNA from being bound only to the toehold region to also being partially bound to the stem. This causes the homologous miRNA to spend more time bound only to the toehold region. This increases the probability of the homologous miRNA detaching from the toehold region. Short toehold regions are particularly prone to this as they form fewer bonds resulting in a more transient toehold binding. Additionally, even if the homologous miRNA does partially bind to the stem, the branch point is more likely to move back down the stem due the unstable mismatch in the newly-formed duplex. [6] We used these principles to significantly decrease toehold switch activation by 15a.
The homolog 15a contains three consecutive mismatches at its 3’ end and one mismatch centrally located in the miRNA. To position the central mismatch at the base of the stem (a proximal mismatch), the stem length was increased to 13nt. As a result, the first codon after the start codon was a stop codon. Therefore, we removed the first nucleotide of the stop codon, resulting in a stem length of 12nt. Consequently, the miRNA has an overhang of one nucleotide. This left a toehold region 9nt long.
Since 15a has three mismatches at its 3’ end, it can only bind to 6nt of the toehold region. The short toehold region results in a less stable miRNA-toehold duplex. Additionally, 15a is forced to spend a considerable amount of time weakly bound to only the toehold region due to the proximal mismatch. The combination of these factors drastically increases the probability of 15a detaching from the toehold. The number of toehold switches activated by 15a is therefore made negligible.
The increased stem length also makes the toehold switch more stable, which reduces leakage. However, it also makes it harder to complete strand displacement, thus reducing on state activity. Therefore we used a slightly larger loop size of 15nt to counteract the increased stem stability.
A further strategy to increase the specificity of the first series of toehold switches involves using CRISPR C2c2. C2c2 is a CRISPR effector protein that can be programmed to cleave single stranded RNA with single base mismatch specificity. [10] [11] It may therefore be possible to use CRISPR C2c2 to cleave homologous miRNAs to prevent them from activating the toehold switch. However, miRNAs may be too short to be targeted by C2c2 and the additional use of CRISPR C2c2 increases the cost of the system.
Validating our NUPACK modeling
Green et al. found that the term ∆GRBS-Linker is the best indicator of switch performance. [1] The ∆GRBS-Linker is the free energy of the structure formed by the sequence immediately after the trigger binding site through to the end of the 21nt linker. Typically, the smaller the |∆GRBS-Linker| is, the higher the dynamic range. We found that our |∆GRBS-Linker| values were quite small, so we therefore expected to achieve quite large dynamic ranges.
Our experiments back this up. See our results page for more information.
Expressing our toehold switch constructs
To express our toehold switch constructs, we used the inducible promoter BBa_K808000. This was necessary as we found that the transcripts produced from our constructs were toxic to E. coli, making successful amplification impossible. Therefore, we chose BBa_K808000, a promoter with low levels of leakage, to express our parts, allowing for amplification in E. coli, although we still had some complications. In future designs, we recommend using a promoter that has even lower leakage than BBa_K808000 (e.g. BBa_K1067007) to allow for successful amplification of the toehold switch constructs before use in the cell free system.
Design summary
Ribosome
Binding
Site
Start Codon
Both switches featured the same conserved region at the top of the hairpin module (GGAUUUACAAAAAGAGGAGAGUAAAAUG). This sequence contains the RBS and start codon. This region of the toehold switch is not affected by the invading miRNA strand and forms the low secondary structure in the on state. The other parts of the switch were designed for a specific miRNA and to have low system leakage.
In future designs, we recommend adjusting the linker sequence in the toehold switch sequence to minimize secondary structure formation in the on state. This helps increase on-state activity and dynamic range.
We also recommend adding a base-pair or two after the trigger binding site in the stem. By increasing stem length, lower leakage can be achieved and on state activity is not reduced. This is explained by the fact that when the branch migration point has progressed most of the way up the base of the stem, spontaneous detachment of the last few base-pairs is likely to occur. [12] This is because the strand being displaced is so weakly attached, that it is unable to stably interact with the region it was bound to. This feature provides a ‘shortcut’ for strand displacement and means that the trigger RNA does not need to bind to all of base of the stem to completely unravel it. The extra one or two base pairs will help to reduce leakage in the off state. Since these last base pairs are disrupted during the branch migration and don’t stably reform in the on state, they do not affect the |∆GRBS-Linker| and on state activity, thus increasing dynamic range.
Extra pairs
Second series  toehold switches
Our sensors must be able to distinguish between the target miRNAs and homologs. Therefore, we designed a second series of switches that can distinguish between two strands which differ by just one nucleotide. Our second series of toehold switches utilize a mechanism that colocalizes several RNA molecules to activate a single toehold switch [13] .
Toehold switches are typically activated by a single trigger RNA. However, it is possible to divide the trigger RNA sequence into separate RNA molecules. The RNA molecules contain hybridization domains that can be used to join them together to form the complete trigger RNA sequence. As a result, the toehold switch sequence will only be activated when all the separate RNA molecules combine.
Trigger Binding Site
Binds to
anti-miRNA
Target
miRNA
hybridization
domain
In our second series of switches, we use an anti-miRNA molecule that binds to a target miRNA to activate our toehold switches. The target miRNA contains one half of the trigger sequence and the anti-miRNA contains the other half of the trigger sequence. The miRNA and anti-miRNA strands hybridize to form the complete trigger sequence, allowing for translational activation.
This new series of switches are much more specific due to the hybridization step of the separate RNA molecules. We modeled the second series of switches using NUPACK and found that for a miRNA to bind to the anti-miRNA and form a stable duplex at around 35°C (varies based on G-C content and position of mismatch), the hybridization domain needs to be at least 12 base pairs long. Hence, we designed our anti-miRNAs to contain a 12 base pair hybridization domain that is specific to the target miRNA. Consequently, homologous miRNAs will not bind to the anti-miRNA as mismatches prevent the formation of the 12 base pair duplex. This results in single-base mismatch specificity, thus removing the issue of homologous miRNAs that bind to the toehold switch and produce false positives.
The anti-miRNAs could be produced in the cell free system alongside the toehold switches. To conserve resources in the cell free system, an inducible promoter should be used for expressing the anti-miRNA sequence. We recommend using the same inducible promoter for both the anti-miRNA and toehold switch for ease. Since, the toehold switch transcripts are toxic in E. coli, this promoter must have very low leakage. For reference, we found BBa_K808000 to be too leaky when expressing our part BBa_K2206007.
Anti-miRNA design
The anti-miRNA can be designed to bind to either the 5’ or 3’ end of the target miRNA. The end chosen depends on the nucleotides that differ between the target miRNA and its homologs. If the differing nucleotides are in the 5’ end of the miRNAs, then the anti-miRNA binds to the 5’ end and vice versa. This is because the miRNA to anti-miRNA hybridization step is the source of the single base mismatch specificity of the new toehold switches.
anti-miRNA
Target
miRNA
hybridization
domain
The only sequence constraint imposed upon the anti-miRNA is the hybridization domain, which must be perfectly complementary to the 12nt of the target miRNA that it binds to. The rest of the anti-miRNA can be programmed to contain any sequence of any length.
Toehold switches are designed based on a specific trigger RNA sequence. The non-hybridization region of the anti-miRNA has no sequence constraints, so the region of the toehold switch complementary to it is fully programmable.
anti-miRNA
Target
miRNA
anti-miRNA
Target
miRNA
Trigger Binding Site
Binds to
3'
5'
3'
5'
5'
3'
Trigger Binding Site
Binds to
5'
3'
5'
3'
1
2
5'
3'
The anti-miRNA can either bind to the 3’ or 5’ end of the target miRNA. This determines the exposed sequence and thus how the toehold switch can be designed.
If the anti-miRNA binds to the toehold region, it allows for a toehold region of any sequence and length. A programmable toehold region can be designed such that it does not form a secondary structure, and the linker does not bind to it. It also allows for high G-C content, ensuring highly stable binding of the anti-miRNA, reducing the probability of the anti-miRNA detaching.
If the anti-miRNA binds to the base of the stem, it allows for a stem of any sequence and length.  The length and G-C content of the stem could then be chosen to reduce leakage. The length of stem unraveled could also be changed to allow for a larger part of the teohold region to be unraveled, providing a lower  |∆GRBS-Linker|.
For both the second series switches the anti-miRNA is exposed at its 3’ end. The exposed anti-miRNA sequence therefore binds to the toehold region. This exposed sequence is 12nt long and contains all guanines to ensure stable binding to the toehold region. This also prevents secondary structure formation within the toehold region and with the linker sequence.
Design summary
The second series of toehold switches were forward engineered using similar design principles employed in the first series of toehold switches.
Ribosome
Binding
Site
Start Codon
The conserved region (GGAUUUACAAAAAGAGGAGAGUAAAAUG) used in the first series of toehold switches was also used in the second series. However, the 21nt linker sequence was changed to reduce secondary structure formation around the start codon to increase translation in the on state.
A mismatch was introduced into the base of the stem to increase the probability of successful branch migration. However, the number of base pairs in the stem was kept the same by adding a base pair after the trigger binding site.
For both the switches in the second series, the anti-miRNA is exposed at its 3’ end, resulting in it binding to the toehold region. Therefore, the switch was designed to contain a 12 cytosine long toehold region. This allows for highly-stable interactions between the toehold region and the anti-miRNA to prevent anti-miRNA dissociation.
The |∆GRBS-Linker| values of the second series of switches are lower than the first series due to changes in the linker sequence, so we would expect to achieve a higher dynamic range. However, the mismatch in the base of the stem may result in slightly more system leakage.
Mismatch
Novel Riboregulators
Molecular beacons have previously been used to efficiently and effectively quantify miRNA levels [14] . The molecular beacons achieved high specificity, however, they suffered from low signal-background ratios and were only capable of sensing concentrations of miRNA greater than 5nM. We propose a new form of riboregulators that combine mechanisms used in molecular beacons and toehold switches for miRNA quantification.
Molecular beacons are single-stranded nucleic acid probes that have a hairpin structure. The probe sequence is complementary to a target nucleic acid sequence. Attached to the 5’ end of the hairpin loop is a fluorophore and attached to the 3’ end is a quencher. Since the fluorophore and quencher are in close proximity of each other, the fluorophore is unable to fluoresce.
Binding of the target nucleic acid to the loop causes the stem to break apart [15] due to the rigid nature of nucleic acid double helices [16] . As a result, the fluorophore and quencher are moved apart, allowing the fluorophore to fluoresce.
Probe
Binds to
Nucleic acid sequence
Fluorophore
Quencher
sequence
Molecular beacons are able to distinguish between their targets more easily than linear probes (such as anti-miRNA) as there is a wider range of temperatures where the fluorescence from the target RNA (solid) is at a maximum whilst for the homologous RNAs (dashed) it is at a minimum. [17]
Temperature
Fluorescence
Target miRNA
Homologous miRNA
Temperature
Fluorescence
Molecular Beacons
Linear probes
Our changes
A strong secondary structure in the 5’ region directly adjacent to the RBS would significantly decrease translation by preventing contacts between the ribosome’s 30S subunit and the pre-RBS region. [18]
Binds to
Probe
sequence
Nucleic acid sequence
Ribosome
binding site
Start
codon
Linker
Protein
coding region
Probe sequence
Nucleic acid sequence
Ribosome
binding site
Start
codon
Linker
Protein
coding region
Therefore, a programmable hairpin loop immediately upstream of the RBS, that changes conformation in response to an arbitrary RNA sequence, could be used to regulate translation of a fluorescent or luminescent reporter protein.
The loop of the hairpin structure should be complementary to part or all of the target miRNA. When the target miRNA binds to the loop, the arms of the hairpin structure are pried apart. This relieves the secondary structure immediately upstream of the RBS, allowing for translation of a reporter protein. Consequently fluorescence or luminescence intensity is indicative of miRNA concentration.
Due to the complete lack of sequence constraints, it is possible to have no secondary structure around the start codon, increasing on state activity. Furthermore, the lack of branch migration results in more favourable reaction kinetics. Finally, the high specificity allows for distinction between miRNAs with close sequence homology.
We believe that we could achieve substantially larger fold changes with these new riboregulators in comparison to our first series of toehold switches. The larger fold change and use of luciferase could allow for much miRNA lower concentrations of miRNAs to be measured. Therefore, these new riboregulators would be well suited for sensitive and specific miRNA quantification.
Implementation of the sensors
Constructs containing a toehold switch and reporter protein can be embedded into paper along with transcription and translation machinery from E. coli by freeze-drying, resulting in a cheap and safe test that can even be used in LEDCs. A cell free sensor also allows for easy addition of miRNAs. Each strip of paper could contain multiple wells, each containing a toehold switch for a specific miRNA. However, toehold switches have very low levels of crosstalk [1] , which make it possible to create a multiplexing assay. By having multiple switches that each regulate the production of a different reporter protein with a distinct emission peak in one well, the efficiency of the test would increase drastically.
See our silver human practices page to learn more about the clinical implementation of our project.
References
↑ 1.0 1.1 1.2 1.3 Green, A. A., Silver, P. A., Collins, J. J., & Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell, 159(4), 925-939.
↑ Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., ... & Daringer, N. M. (2016). Rapid, low-cost detection of Zika virus using programmable biomolecular components. Cell, 165(5), 1255-1266.
↑ Zadeh, J. N., Steenberg, C. D., Bois, J. S., Wolfe, B. R., Pierce, M. B., Khan, A. R., ... & Pierce, N. A. (2011). NUPACK: analysis and design of nucleic acid systems. Journal of computational chemistry, 32(1), 170-173.
↑ (n.d.). The Advantages of Fluorescent Proteins over Luciferase for In Vivo Imaging. Retrieved October 25, 2017, from https://www.uvp.com/pdf/FP129.pdf
↑ Zhang, Y., Phillips, G. J., & Yeung, E. S. (2008). Quantitative imaging of gene expression in individual bacterial cells by chemiluminescence. Analytical chemistry, 80(3), 597-605.
↑ 6.0 6.1 Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., & Turberfield, A. J. (2013). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324-5324.
↑ Vimberg, V., Tats, A., Remm, M., & Tenson, T. (2007). Translation initiation region sequence preferences in Escherichia coli. BMC molecular biology, 8(1), 100.
↑ Kozomara, A., & Griffiths-Jones, S. (2013). miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic acids research, 42(D1), D68-D73.
↑ Griffiths-Jones, S., Grocock, R. J., Van Dongen, S., Bateman, A., & Enright, A. J. (2006). miRBase: microRNA sequences, targets and gene nomenclature. Nucleic acids research, 34(suppl_1), D140-D144.
↑ Abudayyeh, O. O., Gootenberg, J. S., Konermann, S., Joung, J., Slaymaker, I. M., Cox, D. B., ... & Severinov, K. (2016). C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science, 353(6299), aaf5573.
↑ Gootenberg, J. S., Abudayyeh, O. O., Lee, J. W., Essletzbichler, P., Dy, A. J., Joung, J., ... & Myhrvold, C. (2017). Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, eaam9321.
↑ Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., & Turberfield, A. J. (2013). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324-5324.
↑ Green, A. A., Kim, J., Ma, D., Silver, P. A., Collins, J. J., & Yin, P. (2016, September). Ribocomputing devices for sophisticated in vivo logic computation. In Proceedings of the 3rd ACM International Conference on Nanoscale Computing and Communication (p. 11). ACM.
↑ Baker, M. B., Bao, G., & Searles, C. D. (2011). In vitro quantification of specific microRNA using molecular beacons. Nucleic acids research, 40(2), e13-e13.
↑ Tyagi, S., & Kramer, F. R. (1996). Molecular beacons: probes that fluoresce upon hybridization. Nature biotechnology, 14(3), 303-308.
↑ Shore, D., Langowski, J., & Baldwin, R. L. (1981). DNA flexibility studied by covalent closure of short fragments into circles. Proceedings of the National Academy of Sciences, 78(8), 4833-4837.
↑ Bonnet, G., Tyagi, S., Libchaber, A., & Kramer, F. R. (1999). Thermodynamic basis of the enhanced specificity of structured DNA probes. Proceedings of the National Academy of Sciences, 96(11), 6171-6176.
↑ Malmgren, C., Engdahl, H. M., Romby, P., & Wagner, E. G. (1996). An antisense/target RNA duplex or a strong intramolecular RNA structure 5'of a translation initiation signal blocks ribosome binding: the case of plasmid R1. Rna, 2(10), 1022-1032.
Major sponsors
Mr John
Mendez
Contact us
