<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Experiments"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Limitations"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Limitations</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methodology"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Methodology</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Protocols"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Resuspening_gBlocks"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Resuspening gBlocks</SPAN></A></LI><LI class="toclevel-3"><A href="#Resuspending_the_primers_for_PCR"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Resuspending the primers for PCR</SPAN></A></LI><LI class="toclevel-3"><A href="#Amplifying_gBlocks"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Amplifying gBlocks</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Materials_2"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_2"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Preparing_the_Gel"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">Preparing the Gel</SPAN></A></LI><LI class="toclevel-3"><A href="#Preparing_the_material_for_loading"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">Preparing the material for loading</SPAN></A></LI><LI class="toclevel-3"><A href="#Running_the_gel"><SPAN class="tocnumber">1.7.3</SPAN><SPAN class="toctext">Running the gel</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_3"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_3"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Cleanup_Procedure"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">Cleanup Procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_4"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_4"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_5"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_5"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Single-temperature_double_digest_reaction"><SPAN class="tocnumber">1.13.1</SPAN><SPAN class="toctext">Single-temperature double digest reaction</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_6"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_6"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Set-up_the_T4_DNA_Ligase_Reaction"><SPAN class="tocnumber">1.15.1</SPAN><SPAN class="toctext">Set-up the T4 DNA Ligase Reaction</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_7"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_7"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Set-up_the_T4_DNA_Ligase_Reaction_2"><SPAN class="tocnumber">1.17.1</SPAN><SPAN class="toctext">Set-up the T4 DNA Ligase Reaction</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials:"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">Materials:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure:"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">Procedure:</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials:_2"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">Materials:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure:_2"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">Procedure:</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials:_3"><SPAN class="tocnumber">1.22</SPAN><SPAN class="toctext">Materials:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure_8"><SPAN class="tocnumber">1.23</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials:_4"><SPAN class="tocnumber">1.24</SPAN><SPAN class="toctext">Materials:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure:_3"><SPAN class="tocnumber">1.25</SPAN><SPAN class="toctext">Procedure:</SPAN></A></LI><LI class="toclevel-2"><A href="#Preliminaries:"><SPAN class="tocnumber">1.26</SPAN><SPAN class="toctext">Preliminaries:</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials:_5"><SPAN class="tocnumber">1.27</SPAN><SPAN class="toctext">Materials:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure:_4"><SPAN class="tocnumber">1.28</SPAN><SPAN class="toctext">Procedure:</SPAN></A></LI><LI class="toclevel-3"><A href="#Single_Temperature_DD_reaction:"><SPAN class="tocnumber">1.28.1</SPAN><SPAN class="toctext">Single Temperature DD reaction:</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_8"><SPAN class="tocnumber">1.29</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_the_DNA_Template"><SPAN class="tocnumber">1.30</SPAN><SPAN class="toctext">Preparing the DNA Template</SPAN></A></LI><LI class="toclevel-2"><A href="#Cell-Free_Extract_Standard_Protocol"><SPAN class="tocnumber">1.31</SPAN><SPAN class="toctext">Cell-Free Extract Standard Protocol</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_9"><SPAN class="tocnumber">1.32</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_the_DNA_Template_2"><SPAN class="tocnumber">1.33</SPAN><SPAN class="toctext">Preparing the DNA Template</SPAN></A></LI><LI class="toclevel-2"><A href="#Resuspending_the_miRNA_mimicks"><SPAN class="tocnumber">1.34</SPAN><SPAN class="toctext">Resuspending the miRNA mimicks</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H1><H2><SPAN class="mw-headline" id="Limitations">Limitations</SPAN></H2><P>Initially, a constitutive promoter was used, but as our constructs were produced in toxic levels to <I>E. coli</I>, we designed new constructs with an inducible promoter.
</P><P>Our 27b-3p switch was still produced in toxic concentrations in <I>E. coli</I> under the inducible promoter, meaning we were unable to characterize it.
</P><P>We wanted to test both luciferase and GFP as reporter proteins, but our constructs containing both luciferase and the inducible promoter were too large to be inserted into the plasmid backbone in one ligation step. We did not have time to ligate the individual components into the plasmid backbone and so we were also unable to characterize the toehold switches with luciferase.
</P><H2><SPAN class="mw-headline" id="Methodology">Methodology</SPAN></H2><OL><LI>Amplified constructs containing toehold switch and reporter protein using PCR.</LI><LI>Inserted constructs into pSB1C3 plasmid backbone.</LI><LI>Transformed <I>E. coli</I> with plasmids </LI><LI>Cultured <I>E. coli</I></LI><LI>Mini-Prepped plasmids.</LI><LI>Inserted plasmids into cell free extract.</LI><LI>Measured fluorescence intensity in response to varying miRNA concentrations.</LI></OL><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Amplifying gBlocks using PCR</H1><DIV class="protocol_content"><P>gBlock Gene Fragments are normalized to 250 or 1000 ng, depending on length, which is a suitable quantity for many applications. However, we want to amplify gBlocks Gene Fragments to get additional starting material.
</P><UL><LI>GFP/ Luciferase 15b/27b - 1000ng</LI><LI>15b/27b - 250ng</LI></UL><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><UL><LI>gBlock Gene Fragments</LI><LI>Amplification primers</LI><LI>Phusion DNA Polymerase (www.NEB.com/ Phusion)</LI><LI>PCR or gel purification kit (depending on application)</LI></UL><H2><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Resuspening_gBlocks">Resuspening gBlocks</SPAN></H3><OL><LI>The dried down gBlocks Gene Fragment pellet can become displaced from the bottom of the tube during shipping. Centrifuge the tube for 3−5 sec at a minimum of 3000 x g to pellet the material to the bottom of the tube.</LI><LI>Add TE to the tube for your desired final concentration</LI><LI>Briefly vortex and centrifuge</LI><LI>Store at -20°C for up to 24 months</LI></OL><TABLE><TBODY><TR><TH colspan="2"> Resuspension volume of TE buffer (μL)
</TH></TR><TR><TD> gBlock gene fragment final concentration </TD><TD> 15b/27b toehold switch (250ng) </TD><TD> GFP switch (1000ng)
</TD></TR><TR><TD> 10 ngμL </TD><TD> 25 </TD><TD> n/a
</TD></TR><TR><TD> 20 ngμL </TD><TD> n/a </TD><TD> 50
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Resuspending_the_primers_for_PCR">Resuspending the primers for PCR</SPAN></H3><OL><LI>Centrifuge the tube for 3−5 seconds at a minimum of 3000 x g to pellet the material to the bottom of the tube.</LI></OL><OL><LI>To make a 100 μM concentration: Take the number of nmoles of oligo in the tube and multiply that by 10. This number will be the volume (μL) of buffer to add to get a 100 μM solution.</LI></OL><OL><LI>Store resuspended oligonucleotides as several small aliquots at –20°C.</LI></OL><H3><SPAN class="mw-headline" id="Amplifying_gBlocks">Amplifying gBlocks</SPAN></H3><OL><LI>A precipitate (most noticeable after the first 1–2 freeze/thaw cycles) is not uncommon. To ensure optimal performance, the master mix should be thawed and resuspended prior to use. Stability testing using up to 15 freeze/thaw cycles has shown no negative effect on master mix performance.</LI><LI>Dilute the gBlocks 15b/27b by using 1uL of the stock and 9uL of the TEB buffer. For other blocks use 1μL of the block and 19uL of the TEB buffer. This will give us both in concentrations of 1 ng/μL. Dilute the primers 1:10 to get 10μM concentration.</LI><LI>Assemble all reaction components on ice and quickly transfer the reactions to a thermocycler preheated to the denaturation temperature (98°C). All components should be mixed prior to use.</LI></OL><TABLE><TBODY><TR><TH> Component </TH><TH> Volume for the 25μL reaction (μL)
</TH></TR><TR><TD>Q5 High-Fidelity 2X Master Mix </TD><TD> 12.5
</TD></TR><TR><TD>10uM forward primer </TD><TD> 1.25
</TD></TR><TR><TD>10uM reverse primer </TD><TD> 1.25
</TD></TR><TR><TD>Template DNA (1ng) </TD><TD> 1
</TD></TR><TR><TD>Nuclease-free water </TD><TD> 9
</TD></TR></TBODY></TABLE><OL><LI>Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary.</LI><LI>Transfer PCR tubes to a thermocycler and begin thermocycling (Programme 226).</LI></OL><TABLE><TBODY><TR><TH> Step </TH><TH> Temperature (°C) </TH><TH> Time (s)
</TH></TR><TR><TD>Initial denaturing </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD>25 cycles </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 62 </TD><TD> 20
</TD></TR><TR><TD> 72 </TD><TD> 30
</TD></TR><TR><TD>Final extension </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD>Hold </TD><TD> 4 </TD><TD> Indefinite
</TD></TR></TBODY></TABLE><P>The temperatures and times which the IDT document suggested for the thermocycling procedure were not successful at first. Our yield was lower than expected and this was due to “unclean” amplification where there was lots of non-specific primer binding. In an attempt to overcome this, we tried many different temperatures and timings. The table above illustrates the combination which produced the best results and gave us the highest yield.  
</P><P>The primers we used for our PCR reactions were designed using a primer tool on the Benchling website.
</P><P>Source: <A rel="nofollow" class="external text" href="https://www.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-amplification.pdf?sfvrsn=12">IDT</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Gel Electrophoresis</H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Materials_2">Materials</SPAN></H2><UL><LI>Gel Box</LI><LI>Gel Tray</LI><LI>Comb</LI><LI>1% Agarose in 1X TAE</LI><LI>SYBR Safe</LI><LI>100ml beaker</LI><LI>Pipettes and pipette tips</LI></UL><H2><SPAN class="mw-headline" id="Procedure_2">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Preparing_the_Gel">Preparing the Gel</SPAN></H3><OL><LI>Prepare 300ml of the TAE buffer by dissolving 6ml of the 50X concentrated buffer in 294ml of distilled water.</LI><LI>Add 0.5g of agarose powder into 5ml of the diluted buffer and heat whilst stirring continually until clear.</LI><LI>Pour into the gel mould and add the comb with a required number of wells.</LI><LI>Pipette in 3uL of SYBR Safe dye and use the pipette tip to mix it around.</LI></OL><H3><SPAN class="mw-headline" id="Preparing_the_material_for_loading">Preparing the material for loading</SPAN></H3><OL><LI>Mix 3uL of the DNA sample with 2uL of the loading dye.</LI></OL><H3><SPAN class="mw-headline" id="Running_the_gel">Running the gel</SPAN></H3><OL><LI>Add enough 1X TAE to fill the reservoirs at both ends of the gel box and cover the surface of the gel - the gel should be immersed. You should use the remaining TAE buffer from 300ml.</LI><LI>Load 1μL prepared ladder (+dye +loading buffer). The concentration is 100μg/ml, i.e. there will be a total of 0.1ug = 100ng of ladder in the well. The mass of ladder is important to know if you need to quantify your bands by comparison with the ladder bands.</LI></OL><P><I>Load ladder in left-most lane.</I></P><OL><LI>Use 2 μL loading dye per 3 μL of sample.</LI><LI>Load samples left to right.</LI></OL><P><I>The capacity of the 8 well, 1.5mm wide well is approximately 45 μL. The capacity of the 15 well, 1.5mm wide well is approximately 15 μL.</I></P><OL><LI>Place cover on gel box such that your samples will run towards the positive, red electrode. Make sure that the cables from the cover are connected to the power supply correctly.</LI><LI>Turn on the power supply and run your gel at ~85 V for 1 hr 20 mins (voltage and time values can vary). Check regularly to see how far the dye has moved and stop the gel when the dye has moved approximately 80% of the distance.</LI><LI>Verify that bubbles are rising from the electrodes once you start your gel to ensure your gel is running properly.</LI></OL><P>SOURCE: <A rel="nofollow" class="external text" href="http://www.openwetware.org/wiki/Agarose_gel_electrophoresis">OpenWetWare</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">PCR Cleanup</H1><DIV class="protocol_content"><P>Input amount of DNA to be purified should not exceed the binding capacity of the column (5 μg). A starting sample volume of 20–100 μl is recommended. For smaller samples, TE can be used to adjust the volume to the recommended volume range. Centrifugation should be carried out at 16,000 x g in a standard laboratory microcentrifuge at room temperature.
</P><H2><SPAN class="mw-headline" id="Materials_3">Materials</SPAN></H2><UL><LI>Monarch® PCR &amp; DNA Cleanup Kit</LI><LI>DNA samples from PCR reaction</LI></UL><H2><SPAN class="mw-headline" id="Procedure_3">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Cleanup_Procedure">Cleanup Procedure</SPAN></H3><OL><LI>Add ethanol to Monarch DNA Wash Buffer prior to use (4 volumes of ≥ 95% ethanol per volume of Monarch DNA Wash Buffer).</LI><LI>Dilute sample with DNA Cleanup Binding Buffer according to the table below. Mix well by pipetting up and down or flicking the tube. Do not vortex. A starting sample should include combined products of PCR amplification, i.e. 72uL for GFP15b/GFP27b/15b/27b</LI></OL><TABLE><TBODY><TR><TH>  Sample type </TH><TH>  Ratio of binding buffer to sample
</TH></TR><TR><TD> dsDNA &lt; 2 kb </TD><TD>  5:1
</TD></TR></TBODY></TABLE><OL><LI>Insert column into collection tube and load sample onto column and close the cap. Spin for 1 minute, then discard flow-through. Make sure that the sample is inserted into the center of the column.</LI><LI>Re-insert column into collection tube. Add 200 μl DNA Wash Buffer and spin for 1 minute. Discarding flow-through is optional.</LI><LI>Repeat wash (Step 4).</LI><LI>Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If in doubt, re-spin for 1 minute to ensure traces of salt and ethanol are not carried over to next step.</LI><LI>Add 20μl of DNA Elution Buffer to the center of the matrix. Wait for 1 minute, then spin for 1 minute to elute DNA.</LI></OL><P>
SOURCE: <A rel="nofollow" class="external text" href="https://www.neb.com/protocols/2015/11/23/monarch-pcr-and-dna-cleanup-kit-protocol">New England Biolabs</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Single-temperature double digests for gBlocks</H1><DIV class="protocol_content"><P>This protocol was set up whilst waiting to see if we managed to successfully isolate the white colonies from the previous ligations. As PstI could have been the problem, we decided to try cloning the gBlocks into the plasmid using the EcoRI and SpeI instead and this combination of enzymes proved more successful that EcoRI and PstI.  
</P><P>
This is the Double Digest Protocol with EcoRI-HF and SpeI, using a common reaction and same incubation temperature for both enzymes. 
</P><P>Double Digest Recommendations for EcoRI-HF® + SpeI:
Digest in NEBuffer 2.1 at 37°C.
<I>At least one enzyme has &lt; 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.</I></P><TABLE><TBODY><TR><TH> Enzyme </TH><TH> Temp. (°C) </TH><TH> % Activity in NEBuffer 1.1 </TH><TH> % Activity in NEBuffer 2.1 </TH><TH> % Activity in NEBuffer 3.1
</TH></TR><TR><TD> EcoRI-HF® </TD><TD> 37 </TD><TD> 10 </TD><TD> 100 </TD><TD> 10
</TD></TR><TR><TD> SpeI </TD><TD> 37 </TD><TD> 75 </TD><TD> 100 </TD><TD> 25
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Materials_4">Materials</SPAN></H2><UL><LI>Synthesized gBlocks</LI><LI>NEBuffer</LI><LI>NEB Restriction Enzymes</LI><LI>Nuclease Free Water</LI></UL><H2><SPAN class="mw-headline" id="Procedure_4">Procedure</SPAN></H2><P>Alternatively to the first 2 steps, use the gBlocks from the PCR amplification reaction.  
</P><OL><LI>The dried down gBlocks Gene Fragment pellet can become displaced from the bottom of the tube during shipping. Centrifuge the tube for 3−5 seconds at a minimum of 3000 x g to pellet the material to the bottom of the tube.</LI><LI>Add 20µL TE buffer to the tube for your 50ng/µL final concentration (for the basic blocks dissolve in 12µL of TE buffer to get the same concentration)</LI><LI>Briefly vortex and centrifuge</LI><LI>Set up the following reaction (total reaction volume 50 µl). Restriction enzymes should be added last. </LI></OL><TABLE><TBODY><TR><TH> Reagent Volumes (µl)
</TH></TR><TR><TD>Buffer (10x) </TD><TD> 5
</TD></TR><TR><TD>DNA * </TD><TD> 43
</TD></TR><TR><TD>Restriction Enzyme EcoRI </TD><TD> 1
</TD></TR><TR><TD> Restriction Enzyme SpeI </TD><TD> 1
</TD></TR><TR><TD>Nuclease Free Water (μl) </TD><TD> 0
</TD></TR><TR><TD>Total Volume (μl) </TD><TD> 50
</TD></TR></TBODY></TABLE><UL><LI>Recommended maximum of 1 µg of substrate per 10 units of enzyme.</LI></UL><OL><LI> Mix components by pipetting the reaction mixture up and down, or by &quot;flicking&quot; the reaction tube.</LI><LI> Quick (&quot;touch&quot;) spin-down in a microcentrifuge. Do not vortex the reaction.</LI><LI> Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C.</LI><LI> PCR cleanup at the end to remove the short fragments.</LI></OL><P>SOURCE: <A rel="nofollow" class="external text" href="https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5">protocols.io</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Single-temperature double digest of BBa_J04450</H1><DIV class="protocol_content"><P>We previously tried using a linearized plasmid backbone but were not successful at identifying correctly ligated constructs. This prompted us to use BBa_J04450. 
</P><P>We need to digest enough of the BBa_J04450 plasmid for 4 reactions as we have 4 parts. 
</P><P>This is the Double Digest Protocol with EcoRI-HF and PstI, using a common reaction and same incubation temperature for both enzymes. 
</P><P>Double Digest Recommendations for EcoRI-HF® + PstI:
Digest in NEBuffer 2.1 at 37°C.
<I>At least one enzyme has &lt; 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.</I></P><TABLE><TBODY><TR><TH> Enzyme </TH><TH> Temp. (°C) </TH><TH> % Activity in NEBuffer 1.1 </TH><TH> % Activity in NEBuffer 2.1 </TH><TH> % Activity in NEBuffer 3.1
</TH></TR><TR><TD> EcoRI-HF® </TD><TD> 37 </TD><TD> 10 </TD><TD> 100 </TD><TD> 10
</TD></TR><TR><TD> Pstl </TD><TD> 37 </TD><TD> 75 </TD><TD> 75 </TD><TD> 100
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Materials_5">Materials</SPAN></H2><UL><LI>DNA</LI><LI>2.1 NEBuffer</LI><LI>NEB Restriction Enzymes</LI><LI>Deionized Water</LI></UL><P>Before the procedure aliquot all the enzymes and buffer first to prevent freeze-thaw in the future. 
</P><UL><LI>Red - EcoRI-HF</LI><LI>Yellow - PstI</LI><LI>Green - DpnI</LI><LI>Blue - 2.1 NEBuffer</LI></UL><H2><SPAN class="mw-headline" id="Procedure_5">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Single-temperature_double_digest_reaction">Single-temperature double digest reaction</SPAN></H3><OL><LI>Set up the following reaction (total reaction volume 30 µl). Restriction enzymes should be added last.</LI></OL><TABLE><TBODY><TR><TD>Reagent Volumes (µl)
</TD></TR><TR><TD>Buffer (10x) </TD><TD> 3
</TD></TR><TR><TD>DNA * </TD><TD> 4
</TD></TR><TR><TD>Restriction Enzyme EcoRI </TD><TD> 1.1
</TD></TR><TR><TD>Restriction Enzyme PstI </TD><TD> 1.1
</TD></TR><TR><TD>Deionized Water (μl) </TD><TD> 19.8
</TD></TR><TR><TD>Total Volume (μl) </TD><TD> 30
</TD></TR></TBODY></TABLE><UL><LI>Recommended maximum of 1 µg of substrate per 10 units of enzyme.</LI></UL><OL><LI>Mix components by pipetting the reaction mixture up and down, or by &quot;flicking&quot; the reaction tube.</LI><LI>Quick (&quot;touch&quot;) spin-down in a microcentrifuge. Do not vortex the reaction.</LI><LI>Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C.</LI><LI>PCR cleanup the product to remove the short fragments.</LI></OL><P>
SOURCE: <A rel="nofollow" class="external text" href="https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5">protocols.io</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Ligation with T4 DNA Ligase</H1><DIV class="protocol_content"><P>This was the protocol we used for the first set of gBlocks with constitutive promoters. 
</P><H2><SPAN class="mw-headline" id="Materials_6">Materials</SPAN></H2><UL><LI>10X T4 DNA Ligase Reaction Buffer</LI><LI>T4 DNA Ligase</LI><LI>Vector DNA - from linearized plasmid digest</LI><LI>Insert DNA - from gBlock digests</LI><LI>Nuclease-free water</LI></UL><H2><SPAN class="mw-headline" id="Procedure_6">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Set-up_the_T4_DNA_Ligase_Reaction">Set-up the T4 DNA Ligase Reaction</SPAN></H3><P><I>Note:</I> T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.
</P><P><I>Tip: Aliquot the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.</I></P><OL><LI>Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.</LI><LI>Set up the following reaction in a microcentrifuge tube on ice:</LI></OL><TABLE><TBODY><TR><TH>! ! GFP15b </TH><TH> GFP27b </TH><TH> Luc15b </TH><TH> Luc27b </TH><TH> 15b </TH><TH> 27b
</TH></TR><TR><TD> Component </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl)
</TD></TR><TR><TD>  10X T4 DNA Ligase Buffer </TD><TD>  2 </TD><TD>  2 </TD><TD>  2 </TD><TD>  2 </TD><TD>  2 </TD><TD>  2
</TD></TR><TR><TD>  Vector DNA: 40 ng (0.040 pmol) </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9
</TD></TR><TR><TD>  Vector DNA: 40 ng (0.040 pmol) </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9 </TD><TD>  9
</TD></TR><TR><TD>  Insert DNA mass (ng) </TD><TD>  52 </TD><TD>  52 </TD><TD>  106 </TD><TD>  106 </TD><TD>  8 </TD><TD>  8
</TD></TR><TR><TD>  DNA concentration (ng/ul) </TD><TD>  9.7 </TD><TD>  88.6 </TD><TD>  42 </TD><TD>  18 </TD><TD>  16 </TD><TD>  14
</TD></TR><TR><TD>  Insert DNA volume </TD><TD>  6 </TD><TD>  3 </TD><TD>  5 </TD><TD>  7 </TD><TD>  4 </TD><TD>  4
</TD></TR><TR><TD>  Nuclease-free water </TD><TD>  2 </TD><TD>  5 </TD><TD>  3 </TD><TD>  1 </TD><TD>  4 </TD><TD>  4
</TD></TR><TR><TD>  T4 DNA Ligase </TD><TD>  1 </TD><TD>  1 </TD><TD>  1 </TD><TD>  1 </TD><TD>  1 </TD><TD>  1
</TD></TR><TR><TH> Total </TH><TH>  20 </TH><TH>  20 </TH><TH>  20 </TH><TH>  20 </TH><TH>  20 </TH><TH>  20
</TH></TR></TBODY></TABLE><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. Use 25 μL DH5α cells, and add 2 μL of reaction mixture.</LI></OL></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Ligation with T4 DNA Ligase and ara-gBlocks</H1><DIV class="protocol_content"><P>This was the protocol we used for the new set of gBlocks which had arabinose promoters. We decided to discontinue the use of Luciferase gBlocks at this point in time. 
</P><H2><SPAN class="mw-headline" id="Materials_7">Materials</SPAN></H2><UL><LI>10X T4 DNA Ligase Reaction Buffer</LI><LI>T4 DNA Ligase</LI><LI>Vector DNA - from linearized plasmid digest</LI><LI>Insert DNA - from gBlock digests</LI><LI>Nuclease-free water</LI></UL><H2><SPAN class="mw-headline" id="Procedure_7">Procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Set-up_the_T4_DNA_Ligase_Reaction_2">Set-up the T4 DNA Ligase Reaction</SPAN></H3><P>Vector concentration - 231ng from the digestion reaction which underwent PCR cleanup into 16uL of nuclease free water, making the concentration 14.4ng/uL. To get 30ng we need 2.1 uL of this stock.
</P><P>15b/27b concentration - from the gel 15ng/ul. Use 1uL per reaction.
</P><P>GFP15/GFP27 concentration - we digested 300ng of gBlocks and this underwent PCR cleanup into 16uL giving approximate concentration of 18ng/uL. To get 90 ng, we need 6uL of the block.
</P><P><I>Note:</I> T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.
</P><P><I>Tip: Aliquot the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.</I></P><OL><LI>Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.</LI><LI>Set up the following reaction in a microcentrifuge tube on ice:</LI></OL><TABLE><TBODY><TR><TH> 15b </TH><TH> 27b </TH><TH> GFP15b </TH><TH> GFP27b
</TH></TR><TR><TD> Component </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl) </TD><TD> Volume (μl)
</TD></TR><TR><TD>  10X T4 DNA Ligase Buffer </TD><TD>  1 </TD><TD>  1 </TD><TD>  1 </TD><TD>  1
</TD></TR><TR><TD>  Vector DNA: 30 ng (0.030 pmol) </TD><TD>  2.1 </TD><TD>  2.1 </TD><TD>  2.1 </TD><TD>  2.1
</TD></TR><TR><TD>  Insert DNA mass (ng) </TD><TD>  6 </TD><TD>  6 </TD><TD>  90 </TD><TD>  90
</TD></TR><TR><TD>  DNA concentration (ng/ul) </TD><TD>  15 </TD><TD>  15 </TD><TD>  18 </TD><TD>  18
</TD></TR><TR><TD>  Insert DNA volume </TD><TD>  1 </TD><TD>  1 </TD><TD>  6 </TD><TD>  6
</TD></TR><TR><TD>  Nuclease-free water </TD><TD>  4.9 </TD><TD>  4.9 </TD><TD>  0 </TD><TD>  0
</TD></TR><TR><TD>  T4 DNA Ligase </TD><TD>  1 </TD><TD>  1 </TD><TD>  1 </TD><TD>  1
</TD></TR><TR><TH>  Total </TH><TH>  10 </TH><TH>  10 </TH><TH>  10.1 </TH><TH>  10.1
</TH></TR></TBODY></TABLE><OL><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. Use 25 μL DH5α cells, and add 2 μL of reaction mixture.</LI></OL><P>SOURCE: <A rel="nofollow" class="external text" href="https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202">New England Biolabs</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Preparing selection plates for transformations</H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Materials:">Materials:</SPAN></H2><UL><LI>NaCl				</LI><LI>Bacto tryptone						</LI><LI>yeast extract							</LI><LI>agar							</LI><LI>ddH2O 			</LI><LI>5M NaOH											</LI><LI>1000x stock of chloramphenicol (25mg/ml)							 								</LI></UL><H2><SPAN class="mw-headline" id="Procedure:">Procedure:</SPAN></H2><OL><LI>1g NaCl						</LI><LI>6g Bacto tryptone						</LI><LI>3g yeast extract						</LI><LI>ddH2O to 600ml						</LI><LI>100μl of 5M NaOH (adjust the pH to approx. 7)						</LI><LI>9g agar						</LI><LI>Shake. Autoclave for 20 min				</LI><LI>Let it cool to 40-50°C							</LI><LI>Add 600μl of 1000X stock solution of chloramphenicol (stock solution should have a concentration of 25mg/ml) and gently swirl the bottle to mix. Final conc. of chloramphenicol should be 25μg/ml			</LI><LI>Store the plates at 4°C for no longer than 2 weeks									</LI></OL><P>Source: Liljeruhm, J., Gullberg, E., &amp; Forster, A. C. (2014). Synthetic Biology: A Lab Manual. World Scientific.
</P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Transforming competent <I>E. coli</I> with constructs</H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Materials:_2">Materials:</SPAN></H2><UL><LI>NEB 5-alpha Competent <I>E. coli</I> cells	</LI><LI>Plasmid DNA		</LI><LI>Plasmid with no insert			</LI><LI>SOC recovery broth	</LI><LI>Plates with Chloramphenicol </LI></UL><H2><SPAN class="mw-headline" id="Procedure:_2">Procedure:</SPAN></H2><OL><LI>Thaw a tube of NEB 5-alpha Competent <I>E. coli</I> cells on ice for 10 minutes.	</LI><LI>Add 1-5 μl containing 1 pg-100 ng of plasmid DNA to the cell mixture.			</LI><LI>Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.		</LI><LI>Place the mixture on ice for 30 minutes. Do not mix.		</LI><LI>Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.		</LI><LI>Place on ice for 5 minutes. Do not mix.	</LI><LI>Pipette 950 μl of room temperature SOC into the mixture.		</LI><LI>Place at 37°C for 60 minutes., shaking vigorously (250 rpm) or rotating.		</LI><LI>Warm selection plates to 37°C.	</LI><LI>Mix the cells thoroughly by flicking the tube and inverting.</LI><LI>Perform several 10-fold serial dilutions in SOC.</LI><LI>Spread 50-100 μl of each dilution onto a selection plate.</LI><LI>Incubate overnight at 37°C.					</LI></OL><P>Source: <A rel="nofollow" class="external text" href="https://www.neb.com/protocols/1/01/01/high-efficiency-transformation-protocol-c2987">New England Biolabs</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Preparing LB medium for growing <I>E. coli</I></H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Materials:_3">Materials:</SPAN></H2><UL><LI>NaCl </LI><LI>Bacto tryptone</LI><LI>yeast extract </LI><LI>ddH2O </LI><LI>5M NaOH </LI><LI>1000x stock chloramphenicol (25mg/ml)</LI><LI>Agar</LI></UL><H2><SPAN class="mw-headline" id="Procedure_8">Procedure</SPAN></H2><OL><LI>1g NaCl			</LI><LI>6g Bacto tryptone		</LI><LI>3g yeast extract		</LI><LI>ddH2O to 600ml	</LI><LI>100μl of 5M NaOH (adjust the pH to approx. 7)		</LI><LI>9g agar</LI><LI>Shake. Autoclave for 20 min.</LI><LI>Let it cool to 40-50°C.</LI><LI>Add 600μl of 1000X stock solution of chloramphenicol (stock solution should have a concentration of 25mg/ml) and gently				</LI><LI>swirl the bottle to mix. Final conc. of chloramphenicol should be 25μg/ml.</LI><LI>Store at 4°C for no longer than 2 weeks. 	</LI></OL><P>Source: Liljeruhm, J., Gullberg, E., &amp; Forster, A. C. (2014). Synthetic Biology: A Lab Manual. World Scientific.
</P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Miniprep of plasmids from transformed bacteria</H1><DIV class="protocol_content"><UL><LI>For 250-prep kit add 144 ml of ethanol to 36 ml of Monarch Plasmid Wash Buffer 2</LI><LI>All centrifugation steps should be carried out at 16,000 x g (~13,000 RPM).</LI><LI>If precipitate has formed in Lysis Buffer (B2), incubate at 30–37°C, inverting periodically to dissolve. Store Plasmid Neutralisation Buffer (B3) at 4°C after opening. </LI></UL><H2><SPAN class="mw-headline" id="Materials:_4">Materials:</SPAN></H2><UL><LI>NaCl </LI><LI>Bacto tryptone </LI><LI>yeast extract </LI><LI>ddH2O </LI><LI>5M NaOH </LI><LI>1000x stock chloramphenicol (25mg/ml)</LI></UL><H2><SPAN class="mw-headline" id="Procedure:_3">Procedure:</SPAN></H2><OL><LI>Pellet 1–5 ml bacterial culture by centrifugation for 30 seconds. Discard supernatant. </LI><LI>Re-suspend pellet in 200 μl Plasmid Resuspension Buffer (B1). Vortex or pipet to ensure cells are completely resuspended. There should be no visible clumps. </LI><LI>Add 200 μl Plasmid Lysis Buffer (B2), gently invert tube 5–6 times, and incubate at room temperature for 1 minute. Color should change to dark pink, and solution will become transparent and viscous. Do not vortex.</LI><LI>Add 400 μl of Plasmid Neutralisation Buffer (B3), gently invert tube until neutralized, and incubate at room temperature for 2 minutes. Sample is neutralized when color is uniformly yellow and precipitate forms. Do not vortex. </LI><LI>Centrifuge lysate for 2–5 minutes. For culture volumes &gt;1 ml, we recommend a 5 minute spin to ensure efficient RNA removal by RNase A. Pellet should be compact; spin longer if needed. </LI><LI>Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through. </LI><LI>Re-insert column in the collection tube and add 200 μl of Plasmid Wash Buffer 1. Centrifuge for 1 minute. Discarding the flow-through is optional. </LI><LI>Add 400 μl of Plasmid Wash Buffer 2 and centrifuge for 1 minute. </LI><LI>Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If there is any doubt, re-spin the column for 1 minute.</LI><LI>Add ≥ 30 μl DNA Elution Buffer to the center of the matrix. Wait for 1 minute, then spin for 1 minute at 16,000 x g to elute the DNA. Nuclease-free water (pH 7–8.5) can also be used to elute the DNA.</LI></OL><P>SOURCE: <A rel="nofollow" class="external text" href="https://international.neb.com/protocols/2015/11/20/monarch-plasmid-dna-miniprep-kit-protocol-t1010">New England Biolabs</A></P></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Single-temperature Double Digest for gBlocks</H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Preliminaries:">Preliminaries:</SPAN></H2><UL><LI>This is the Double Digest Protocol with EcoRI-HF and PstI, using a common reaction and same incubation temperature for both enzymes.</LI><LI>Double Digest Recommendations for EcoRI-HF® + PstI:</LI><LI>Digest in NEBuffer 2.1 at 37°C.</LI></UL><P><I>At least one enzyme has &lt; 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.</I></P><TABLE><TBODY><TR><TH> Enzyme:   </TH><TH>       Temp: (*C)   </TH><TH> %Activity in NEBuffer
</TH></TR><TR><TD>       1.1  </TD><TD>      2.1   </TD><TD>    3.1
</TD></TR><TR><TD> EcoRI-HF  </TD><TD>      37          </TD><TD>            10      </TD><TD>   100    </TD><TD>  10
</TD></TR><TR><TD> Pstl	  </TD><TD> 37              </TD><TD>        75    </TD><TD>     75   </TD><TD>     100
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Materials:_5">Materials:</SPAN></H2><UL><LI>Synthesized gBlocks › NEBuffer</LI><LI>NEBuffer</LI><LI>1X</LI><LI>NEB Restriction Enzymes</LI><LI>Deionized Water </LI></UL><H2><SPAN class="mw-headline" id="Procedure:_4">Procedure:</SPAN></H2><H3><SPAN class="mw-headline" id="Single_Temperature_DD_reaction:">Single Temperature DD reaction:</SPAN></H3><OL><LI> Use the plasmids recovered from minipreps to test that the vector and insert are the right size. </LI><LI> Set up the following reaction (total reaction volume 20 μl).</LI></OL><TABLE><TBODY><TR><TH> A </TH><TH> B
</TH></TR><TR><TD>Material </TD><TD>Reagent volumes (ul)
</TD></TR><TR><TD> Buffer (10x) </TD><TD> 2
</TD></TR><TR><TD> DNA * </TD><TD> 10
</TD></TR><TR><TD> Restriction enzyme EcoR1 </TD><TD> 1
</TD></TR><TR><TD> Restriction enzyme Pstl </TD><TD> 1
</TD></TR><TR><TD> Nuclease free water </TD><TD> 6
</TD></TR><TR><TH> Total Volume (ul) </TH><TH> 20
</TH></TR></TBODY></TABLE><P><I>Recommended maximum of 1 μg of substrate per 10 units of enzyme. Restriction enzymes should be added to the mixture last.'</I></P><OL><LI> Mix components by pipetting the reaction mixture up and down, or by &quot;flicking&quot; the reaction tube. </LI><LI> Quick (&quot;touch&quot;) spin-down in a microcentrifuge. Do not vortex the reaction.</LI><LI> Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C. </LI><LI> PCR purify at the end to remove the short fragments.</LI></OL></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Characterizing araC-Pbad Promoter</H1><DIV class="protocol_content"><P>Promoter page (on the parts registry website) suggests that concentrations of arabinose between 0.001 - 0.02% are used to overexpress this promoter and we have modeled it essentially as a constitutive promoter so we should test this construct with the cell-free extract to confirm this.
</P><H2><SPAN class="mw-headline" id="Materials_8">Materials</SPAN></H2><UL><LI>DNA Template (BBa_K808000)</LI><LI>Total Amino Acid Mixture</LI><LI>S30 Premix Without Amino Acids</LI><LI>S30 Extract</LI><LI>Nuclease Free Water</LI><LI>L-Arabinose</LI></UL><H2><SPAN class="mw-headline" id="Preparing_the_DNA_Template">Preparing the DNA Template</SPAN></H2><P>Miniprepped plasmids have been test digested and all the insert sizes look OK. Final concentration was measured to be 410ng/μL.
</P><H2><SPAN class="mw-headline" id="Cell-Free_Extract_Standard_Protocol">Cell-Free Extract Standard Protocol</SPAN></H2><P>We will setup the following tests:
+ve control = GFP from a constitutive promoter (to show that cell free extract works)
-ve control = just cell free extract
-ve Ara = cell free extract with arabinose but no DNA
-ve DNA = cell free extract with DNA but no arabinose
5 tests with 2%, 1%, 0.5%, 0.1% and 0.05% of arabinose (final concentration) respectively
</P><P>Prepare arabinose stock. As the final volume will be 11μL and we want to pipette in 1μL of arabinose stock conc. will need to be 11 times that of the final, i.e. 22%, 11%, 5.5%, 1.1% and 0.55%. Dissolve 1.1 g in 5 ml of water to get 22% and make a dilution series.
</P><TABLE><TBODY><TR><TD> Component of the System </TD><TD> Volume (μL) </TD><TD> Volume for 3 Replicates (μL) </TD><TD> Volume for 30 Reactions (μL) </TD></TR><TR><TD> DNA template </TD><TD> 1 </TD><TD> 3 </TD></TR><TR><TD> Total amino acid mixture </TD><TD> 1 </TD><TD> 3 </TD><TD> 30 (15+15) </TD></TR><TR><TD> S30 premix without amino acids </TD><TD> 4 </TD><TD> 12 </TD><TD> 120 </TD><TD> Master mix
</TD></TR><TR><TD> S30 extract </TD><TD> 3 </TD><TD> 9 </TD><TD> 90 </TD></TR><TR><TD> Nuclease free water instead of miR </TD><TD> 1 </TD><TD> 3 </TD><TD> 30 </TD></TR><TR><TD> Nuclease free water/ arabinose </TD><TD> 1 </TD><TD> 3 </TD></TR></TBODY></TABLE><P>
2. As we will be setting up 9 reactions with 3 replicates each we will need to mix enough cell free extract for 30 reactions.
</P><P>3. Prepare master mix
</P><P>4. Pipette up and down to mix the components
</P><P>5. Incubate the reaction mixture at 30°C for 10 hours in the plate reader, and measure every 10 minutes.
</P><TABLE><TBODY><TR><TH> Well </TH><TH> Test </TH><TH> Volume of master mix (μL) </TH><TH> Substitute Nuclease Free Water (μL) </TH><TH> Arabinose (μL) </TH><TH> DNA Template (μL)
</TH></TR><TR><TD> 1 AB </TD><TD> -ve control </TD><TD> 27 </TD><TD> 6 </TD></TR><TR><TD> 2 AB </TD><TD> -ve DNA </TD><TD> 27 </TD><TD> 3 </TD><TD> 3 AraC
</TD></TR><TR><TD> 3 AB </TD><TD> 2% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 4 AB </TD><TD> 1% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 5 AB </TD><TD> 0.50% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 6 AB </TD><TD> 0.10% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 7 AB </TD><TD> 0.05% </TD><TD> 27 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 8 AB </TD><TD> +ve control </TD><TD> 27 </TD><TD> 3 </TD><TD> 3 (constitutive promoter)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="protocol"><H1 onclick="$(this.parentNode).toggleClass('shown')">Characterizing 15-5p Toehold Switch</H1><DIV class="protocol_content"><H2><SPAN class="mw-headline" id="Materials_9">Materials</SPAN></H2><UL><LI>Nuclease Free Water</LI><LI>miRs</LI><LI>DNA Template</LI><LI>Total Amino Acid Mixture</LI><LI>S30 Premix Without Amino Acids</LI><LI>S30 Extract</LI><LI>L-Arabinose</LI></UL><H2><SPAN class="mw-headline" id="Preparing_the_DNA_Template_2">Preparing the DNA Template</SPAN></H2><P>Miniprepped plasmids have been sequenced and the sequence of our toehold switch for 15b-5p has been confirmed. Concentration of this plasmid has been measured to be 95ng/μL so will need to be dried down to the conc. of approx. 500ng/μL so that only 1 μL has to be used per reaction. Final concentration has been measured to be 460ng/μL.
</P><H2><SPAN class="mw-headline" id="Resuspending_the_miRNA_mimicks">Resuspending the miRNA mimicks</SPAN></H2><OL><LI>Centrifuge the tube for 3−5 sec at a minimum of 3000 x g to pellet the material to the bottom of the tube.</LI><LI>To make a 10 μM concentration: Take the number of nmoles of miRNA mimick in the tube (5) and multiply that by 100 (500). This number will be the number of μL of nuclease free water to add to get a 100 μM solution.</LI><LI> Store resuspended oligonucleotides as several small aliquots at –20°C.</LI></OL><P>Serial Dilutions of miRs:
</P><TABLE><TBODY><TR><TH> Tube Name </TH><TH> Stock Concentration </TH><TH> Final Concentration (/11) </TH><TH> Volume of Previous Concentration (μL) </TH><TH> Volume of Nuclease Free Water (μL)
</TH></TR><TR><TD> A </TD><TD> 10 μM </TD><TD> 9x10⁸ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> B </TD><TD> 1 μM </TD><TD> 9x10⁷ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> C </TD><TD> 100 nM </TD><TD> 9x10⁶ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> D </TD><TD> 10 nM </TD><TD> 9x10⁵ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> E </TD><TD> 1 nM </TD><TD> 9x10⁴ fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> F </TD><TD> 100 pM </TD><TD> 9000 fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> G </TD><TD> 10:00 PM </TD><TD> 900 fM </TD><TD> 10 </TD><TD> 90
</TD></TR><TR><TD> H </TD><TD> 1:00 PM </TD><TD> 90 fM </TD><TD> 10 </TD><TD> 90
</TD></TR></TBODY></TABLE><P>
There will be 13 tests including controls with 3 repeats each so we need enough master mix for 45 reactions.
</P><TABLE><TBODY><TR><TH> Component of the System </TH><TH> Volume (μL) </TH><TH> Volume for 3 Replicates (μL) </TH><TH> Volume for 45 Reactions (μL) </TH></TR><TR><TD> DNA template </TD><TD> 1 </TD><TD> 3 </TD></TR><TR><TD> Total amino acid mixture </TD><TD> 1 </TD><TD> 3 </TD><TD> 60 (30 + 30) </TD><TD> Master mix
</TD></TR><TR><TD> S30 premix without amino acids </TD><TD> 4 </TD><TD> 12 </TD><TD> 240 </TD></TR><TR><TD> S30 extract </TD><TD> 3 </TD><TD> 9 </TD><TD> 180 </TD></TR><TR><TD> L-Arabinose </TD><TD> 1 </TD><TD> 3 </TD></TR></TBODY></TABLE><P>
+ve control = GFP from a constitutive promoter (to show that cell free extract works)
-ve control = just cell free extract
-ve ara = cell free extract with arabinose but no DNA
-ve DNA = cell free extract with DNA but no arabinose
-ve mix = cell free extract with arabinose and DNA but no miR
</P><TABLE><TBODY><TR><TH> Wells </TH><TH> Test </TH><TH> Volume of master mix (μL) </TH><TH> Substitute Nuclease Free Water (μL) </TH><TH> Arabinose (μL) </TH><TH> miR (μL) </TH><TH> DNA Template (μL)
</TH></TR><TR><TD> 1 CDE </TD><TD> -ve control </TD><TD> 24 </TD><TD> 9 </TD></TR><TR><TD> 2 CDE </TD><TD> -ve miR </TD><TD> 24 </TD><TD> 6 </TD><TD> 3 </TD></TR><TR><TD> 3 CDE </TD><TD> -ve DNA </TD><TD> 24 </TD><TD> 6 </TD><TD> 3
</TD></TR><TR><TD> 4 CDE </TD><TD> -ve mix </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 5 CDE </TD><TD> A </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 6 CDE </TD><TD> B </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 7 CDE </TD><TD> C </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 8 CDE </TD><TD> D </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 9 CDE </TD><TD> E </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 10 CDE </TD><TD> F </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 11 CDE </TD><TD> G </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 12 CDE </TD><TD> H </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> 13 CDE </TD><TD> 27 </TD><TD> 24 </TD><TD> 3 </TD><TD> 3 (miRNA27) </TD><TD> 3
</TD></TR><TR><TD> 14 CDE </TD><TD> +ve control </TD><TD> 24 </TD><TD> 6 </TD><TD> 3
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>