Team:CLSB-UK/Experiments
CLSB-UK
Project
Design
Parts
Demonstrate
Judging
Contribution
Lab work
Results
Experiments
Safety
Notebook
InterLab
Human Practices
Silver
Engagement
High School Guide
Gold Integrated
Safety
Cost model
Modeling
NUPACK model
Mass action kinetics
Stochastic model
Cost model
Software
Overview
Toehold tools
SynBioBot
Hardware
Overview
Interpreter
Measurement
People
Team
Collaborations
Attributions
Sponsors
We strongly recommend using Google Chrome or Mozilla Firefox to view our wiki.
Contents
1 Experiments
1.1 Limitations
1.2 Methodology
1.3 Protocols
1.4 Materials
1.5 Procedure
1.5.1 Resuspening gBlocks
1.5.2 Resuspending the primers for PCR
1.5.3 Amplifying gBlocks
1.6 Materials
1.7 Procedure
1.7.1 Preparing the Gel
1.7.2 Preparing the material for loading
1.7.3 Running the gel
1.8 Materials
1.9 Procedure
1.9.1 Cleanup Procedure
1.10 Materials
1.11 Procedure
1.12 Materials
1.13 Procedure
1.13.1 Single-temperature double digest reaction
1.14 Materials
1.15 Procedure
1.15.1 Set-up the T4 DNA Ligase Reaction
1.16 Materials
1.17 Procedure
1.17.1 Set-up the T4 DNA Ligase Reaction
1.18 Materials:
1.19 Procedure:
1.20 Materials:
1.21 Procedure:
1.22 Materials:
1.23 Procedure
1.24 Materials:
1.25 Procedure:
1.26 Preliminaries:
1.27 Materials:
1.28 Procedure:
1.28.1 Single Temperature DD reaction:
1.29 Materials
1.30 Preparing the DNA Template
1.31 Cell-Free Extract Standard Protocol
1.32 Materials
1.33 Preparing the DNA Template
1.34 Resuspending the miRNA mimicks
Experiments
Limitations
Initially, a constitutive promoter was used, but as our constructs were produced in toxic levels to E. coli, we designed new constructs with an inducible promoter.
Our 27b-3p switch was still produced in toxic concentrations in E. coli under the inducible promoter, meaning we were unable to characterize it.
We wanted to test both luciferase and GFP as reporter proteins, but our constructs containing both luciferase and the inducible promoter were too large to be inserted into the plasmid backbone in one ligation step. We did not have time to ligate the individual components into the plasmid backbone and so we were also unable to characterize the toehold switches with luciferase.
Methodology
Amplified constructs containing toehold switch and reporter protein using PCR.
Inserted constructs into pSB1C3 plasmid backbone.
Transformed E. coli with plasmids
Cultured E. coli
Mini-Prepped plasmids.
Inserted plasmids into cell free extract.
Measured fluorescence intensity in response to varying miRNA concentrations.
Protocols
Amplifying gBlocks using PCR
gBlock Gene Fragments are normalized to 250 or 1000 ng, depending on length, which is a suitable quantity for many applications. However, we want to amplify gBlocks Gene Fragments to get additional starting material.
GFP/ Luciferase 15b/27b - 1000ng
15b/27b - 250ng
Materials
gBlock Gene Fragments
Amplification primers
Phusion DNA Polymerase (www.NEB.com/ Phusion)
PCR or gel purification kit (depending on application)
Procedure
Resuspening gBlocks
The dried down gBlocks Gene Fragment pellet can become displaced from the bottom of the tube during shipping. Centrifuge the tube for 3−5 sec at a minimum of 3000 x g to pellet the material to the bottom of the tube.
Add TE to the tube for your desired final concentration
Briefly vortex and centrifuge
Store at -20°C for up to 24 months
Resuspension volume of TE buffer (μL)
gBlock gene fragment final concentration
15b/27b toehold switch (250ng)
GFP switch (1000ng)
10 ngμL
25
n/a
20 ngμL
n/a
50
Resuspending the primers for PCR
Centrifuge the tube for 3−5 seconds at a minimum of 3000 x g to pellet the material to the bottom of the tube.
To make a 100 μM concentration: Take the number of nmoles of oligo in the tube and multiply that by 10. This number will be the volume (μL) of buffer to add to get a 100 μM solution.
Store resuspended oligonucleotides as several small aliquots at –20°C.
Amplifying gBlocks
A precipitate (most noticeable after the first 1–2 freeze/thaw cycles) is not uncommon. To ensure optimal performance, the master mix should be thawed and resuspended prior to use. Stability testing using up to 15 freeze/thaw cycles has shown no negative effect on master mix performance.
Dilute the gBlocks 15b/27b by using 1uL of the stock and 9uL of the TEB buffer. For other blocks use 1μL of the block and 19uL of the TEB buffer. This will give us both in concentrations of 1 ng/μL. Dilute the primers 1:10 to get 10μM concentration.
Assemble all reaction components on ice and quickly transfer the reactions to a thermocycler preheated to the denaturation temperature (98°C). All components should be mixed prior to use.
Component
Volume for the 25μL reaction (μL)
Q5 High-Fidelity 2X Master Mix
12.5
10uM forward primer
1.25
10uM reverse primer
1.25
Template DNA (1ng)
1
Nuclease-free water
9
Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary.
Transfer PCR tubes to a thermocycler and begin thermocycling (Programme 226).
Step
Temperature (°C)
Time (s)
Initial denaturing
98
30
25 cycles
98
10
62
20
72
30
Final extension
72
300
Hold
4
Indefinite
The temperatures and times which the IDT document suggested for the thermocycling procedure were not successful at first. Our yield was lower than expected and this was due to “unclean” amplification where there was lots of non-specific primer binding. In an attempt to overcome this, we tried many different temperatures and timings. The table above illustrates the combination which produced the best results and gave us the highest yield.
The primers we used for our PCR reactions were designed using a primer tool on the Benchling website.
Source: IDT
Gel Electrophoresis
Materials
Gel Box
Gel Tray
Comb
1% Agarose in 1X TAE
SYBR Safe
100ml beaker
Pipettes and pipette tips
Procedure
Preparing the Gel
Prepare 300ml of the TAE buffer by dissolving 6ml of the 50X concentrated buffer in 294ml of distilled water.
Add 0.5g of agarose powder into 5ml of the diluted buffer and heat whilst stirring continually until clear.
Pour into the gel mould and add the comb with a required number of wells.
Pipette in 3uL of SYBR Safe dye and use the pipette tip to mix it around.
Preparing the material for loading
Mix 3uL of the DNA sample with 2uL of the loading dye.
Running the gel
Add enough 1X TAE to fill the reservoirs at both ends of the gel box and cover the surface of the gel - the gel should be immersed. You should use the remaining TAE buffer from 300ml.
Load 1μL prepared ladder (+dye +loading buffer). The concentration is 100μg/ml, i.e. there will be a total of 0.1ug = 100ng of ladder in the well. The mass of ladder is important to know if you need to quantify your bands by comparison with the ladder bands.
Load ladder in left-most lane.
Use 2 μL loading dye per 3 μL of sample.
Load samples left to right.
The capacity of the 8 well, 1.5mm wide well is approximately 45 μL. The capacity of the 15 well, 1.5mm wide well is approximately 15 μL.
Place cover on gel box such that your samples will run towards the positive, red electrode. Make sure that the cables from the cover are connected to the power supply correctly.
Turn on the power supply and run your gel at ~85 V for 1 hr 20 mins (voltage and time values can vary). Check regularly to see how far the dye has moved and stop the gel when the dye has moved approximately 80% of the distance.
Verify that bubbles are rising from the electrodes once you start your gel to ensure your gel is running properly.
SOURCE: OpenWetWare
PCR Cleanup
Input amount of DNA to be purified should not exceed the binding capacity of the column (5 μg). A starting sample volume of 20–100 μl is recommended. For smaller samples, TE can be used to adjust the volume to the recommended volume range. Centrifugation should be carried out at 16,000 x g in a standard laboratory microcentrifuge at room temperature.
Materials
Monarch® PCR & DNA Cleanup Kit
DNA samples from PCR reaction
Procedure
Cleanup Procedure
Add ethanol to Monarch DNA Wash Buffer prior to use (4 volumes of ≥ 95% ethanol per volume of Monarch DNA Wash Buffer).
Dilute sample with DNA Cleanup Binding Buffer according to the table below. Mix well by pipetting up and down or flicking the tube. Do not vortex. A starting sample should include combined products of PCR amplification, i.e. 72uL for GFP15b/GFP27b/15b/27b
Sample type
Ratio of binding buffer to sample
dsDNA < 2 kb
5:1
Insert column into collection tube and load sample onto column and close the cap. Spin for 1 minute, then discard flow-through. Make sure that the sample is inserted into the center of the column.
Re-insert column into collection tube. Add 200 μl DNA Wash Buffer and spin for 1 minute. Discarding flow-through is optional.
Repeat wash (Step 4).
Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If in doubt, re-spin for 1 minute to ensure traces of salt and ethanol are not carried over to next step.
Add 20μl of DNA Elution Buffer to the center of the matrix. Wait for 1 minute, then spin for 1 minute to elute DNA.
SOURCE: New England Biolabs
Single-temperature double digests for gBlocks
This protocol was set up whilst waiting to see if we managed to successfully isolate the white colonies from the previous ligations. As PstI could have been the problem, we decided to try cloning the gBlocks into the plasmid using the EcoRI and SpeI instead and this combination of enzymes proved more successful that EcoRI and PstI.
This is the Double Digest Protocol with EcoRI-HF and SpeI, using a common reaction and same incubation temperature for both enzymes.
Double Digest Recommendations for EcoRI-HF® + SpeI: Digest in NEBuffer 2.1 at 37°C. At least one enzyme has < 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.
Enzyme
Temp. (°C)
 % Activity in NEBuffer 1.1
 % Activity in NEBuffer 2.1
 % Activity in NEBuffer 3.1
EcoRI-HF®
37
10
100
10
SpeI
37
75
100
25
Materials
Synthesized gBlocks
NEBuffer
NEB Restriction Enzymes
Nuclease Free Water
Procedure
Alternatively to the first 2 steps, use the gBlocks from the PCR amplification reaction.
The dried down gBlocks Gene Fragment pellet can become displaced from the bottom of the tube during shipping. Centrifuge the tube for 3−5 seconds at a minimum of 3000 x g to pellet the material to the bottom of the tube.
Add 20µL TE buffer to the tube for your 50ng/µL final concentration (for the basic blocks dissolve in 12µL of TE buffer to get the same concentration)
Briefly vortex and centrifuge
Set up the following reaction (total reaction volume 50 µl). Restriction enzymes should be added last.
Reagent Volumes (µl)
Buffer (10x)
5
DNA *
43
Restriction Enzyme EcoRI
1
Restriction Enzyme SpeI
1
Nuclease Free Water (μl)
0
Total Volume (μl)
50
Recommended maximum of 1 µg of substrate per 10 units of enzyme.
Mix components by pipetting the reaction mixture up and down, or by "flicking" the reaction tube.
Quick ("touch") spin-down in a microcentrifuge. Do not vortex the reaction.
Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C.
PCR cleanup at the end to remove the short fragments.
SOURCE: protocols.io
Single-temperature double digest of BBa_J04450
We previously tried using a linearized plasmid backbone but were not successful at identifying correctly ligated constructs. This prompted us to use BBa_J04450.
We need to digest enough of the BBa_J04450 plasmid for 4 reactions as we have 4 parts.
This is the Double Digest Protocol with EcoRI-HF and PstI, using a common reaction and same incubation temperature for both enzymes.
Double Digest Recommendations for EcoRI-HF® + PstI: Digest in NEBuffer 2.1 at 37°C. At least one enzyme has < 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.
Enzyme
Temp. (°C)
 % Activity in NEBuffer 1.1
 % Activity in NEBuffer 2.1
 % Activity in NEBuffer 3.1
EcoRI-HF®
37
10
100
10
Pstl
37
75
75
100
Materials
DNA
2.1 NEBuffer
NEB Restriction Enzymes
Deionized Water
Before the procedure aliquot all the enzymes and buffer first to prevent freeze-thaw in the future.
Red - EcoRI-HF
Yellow - PstI
Green - DpnI
Blue - 2.1 NEBuffer
Procedure
Single-temperature double digest reaction
Set up the following reaction (total reaction volume 30 µl). Restriction enzymes should be added last.
Reagent Volumes (µl)
Buffer (10x)
3
DNA *
4
Restriction Enzyme EcoRI
1.1
Restriction Enzyme PstI
1.1
Deionized Water (μl)
19.8
Total Volume (μl)
30
Recommended maximum of 1 µg of substrate per 10 units of enzyme.
Mix components by pipetting the reaction mixture up and down, or by "flicking" the reaction tube.
Quick ("touch") spin-down in a microcentrifuge. Do not vortex the reaction.
Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C.
PCR cleanup the product to remove the short fragments.
SOURCE: protocols.io
Ligation with T4 DNA Ligase
This was the protocol we used for the first set of gBlocks with constitutive promoters.
Materials
10X T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Vector DNA - from linearized plasmid digest
Insert DNA - from gBlock digests
Nuclease-free water
Procedure
Set-up the T4 DNA Ligase Reaction
Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.
Tip: Aliquot the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.
Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.
Set up the following reaction in a microcentrifuge tube on ice:
! ! GFP15b
GFP27b
Luc15b
Luc27b
15b
27b
Component
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
10X T4 DNA Ligase Buffer
2
2
2
2
2
2
Vector DNA: 40 ng (0.040 pmol)
9
9
9
9
9
9
Vector DNA: 40 ng (0.040 pmol)
9
9
9
9
9
9
Insert DNA mass (ng)
52
52
106
106
8
8
DNA concentration (ng/ul)
9.7
88.6
42
18
16
14
Insert DNA volume
6
3
5
7
4
4
Nuclease-free water
2
5
3
1
4
4
T4 DNA Ligase
1
1
1
1
1
1
Total
20
20
20
20
20
20
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
Heat inactivate at 65°C for 10 minutes.
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. Use 25 μL DH5α cells, and add 2 μL of reaction mixture.
Ligation with T4 DNA Ligase and ara-gBlocks
This was the protocol we used for the new set of gBlocks which had arabinose promoters. We decided to discontinue the use of Luciferase gBlocks at this point in time.
Materials
10X T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Vector DNA - from linearized plasmid digest
Insert DNA - from gBlock digests
Nuclease-free water
Procedure
Set-up the T4 DNA Ligase Reaction
Vector concentration - 231ng from the digestion reaction which underwent PCR cleanup into 16uL of nuclease free water, making the concentration 14.4ng/uL. To get 30ng we need 2.1 uL of this stock.
15b/27b concentration - from the gel 15ng/ul. Use 1uL per reaction.
GFP15/GFP27 concentration - we digested 300ng of gBlocks and this underwent PCR cleanup into 16uL giving approximate concentration of 18ng/uL. To get 90 ng, we need 6uL of the block.
Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.
Tip: Aliquot the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.
Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.
Set up the following reaction in a microcentrifuge tube on ice:
15b
27b
GFP15b
GFP27b
Component
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
10X T4 DNA Ligase Buffer
1
1
1
1
Vector DNA: 30 ng (0.030 pmol)
2.1
2.1
2.1
2.1
Insert DNA mass (ng)
6
6
90
90
DNA concentration (ng/ul)
15
15
18
18
Insert DNA volume
1
1
6
6
Nuclease-free water
4.9
4.9
0
0
T4 DNA Ligase
1
1
1
1
Total
10
10
10.1
10.1
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
Heat inactivate at 65°C for 10 minutes.
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. Use 25 μL DH5α cells, and add 2 μL of reaction mixture.
SOURCE: New England Biolabs
Preparing selection plates for transformations
Materials:
NaCl
Bacto tryptone
yeast extract
agar
ddH2O
5M NaOH
1000x stock of chloramphenicol (25mg/ml)
Procedure:
1g NaCl
6g Bacto tryptone
3g yeast extract
ddH2O to 600ml
100μl of 5M NaOH (adjust the pH to approx. 7)
9g agar
Shake. Autoclave for 20 min
Let it cool to 40-50°C
Add 600μl of 1000X stock solution of chloramphenicol (stock solution should have a concentration of 25mg/ml) and gently swirl the bottle to mix. Final conc. of chloramphenicol should be 25μg/ml
Store the plates at 4°C for no longer than 2 weeks
Source: Liljeruhm, J., Gullberg, E., & Forster, A. C. (2014). Synthetic Biology: A Lab Manual. World Scientific.
Transforming competent E. coli with constructs
Materials:
NEB 5-alpha Competent E. coli cells
Plasmid DNA
Plasmid with no insert
SOC recovery broth
Plates with Chloramphenicol
Procedure:
Thaw a tube of NEB 5-alpha Competent E. coli cells on ice for 10 minutes.
Add 1-5 μl containing 1 pg-100 ng of plasmid DNA to the cell mixture.
Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
Place on ice for 5 minutes. Do not mix.
Pipette 950 μl of room temperature SOC into the mixture.
Place at 37°C for 60 minutes., shaking vigorously (250 rpm) or rotating.
Warm selection plates to 37°C.
Mix the cells thoroughly by flicking the tube and inverting.
Perform several 10-fold serial dilutions in SOC.
Spread 50-100 μl of each dilution onto a selection plate.
Incubate overnight at 37°C.
Source: New England Biolabs
Preparing LB medium for growing E. coli
Materials:
NaCl
Bacto tryptone
yeast extract
ddH2O
5M NaOH
1000x stock chloramphenicol (25mg/ml)
Agar
Procedure
1g NaCl
6g Bacto tryptone
3g yeast extract
ddH2O to 600ml
100μl of 5M NaOH (adjust the pH to approx. 7)
9g agar
Shake. Autoclave for 20 min.
Let it cool to 40-50°C.
Add 600μl of 1000X stock solution of chloramphenicol (stock solution should have a concentration of 25mg/ml) and gently
swirl the bottle to mix. Final conc. of chloramphenicol should be 25μg/ml.
Store at 4°C for no longer than 2 weeks.
Source: Liljeruhm, J., Gullberg, E., & Forster, A. C. (2014). Synthetic Biology: A Lab Manual. World Scientific.
Miniprep of plasmids from transformed bacteria
For 250-prep kit add 144 ml of ethanol to 36 ml of Monarch Plasmid Wash Buffer 2
All centrifugation steps should be carried out at 16,000 x g (~13,000 RPM).
If precipitate has formed in Lysis Buffer (B2), incubate at 30–37°C, inverting periodically to dissolve. Store Plasmid Neutralisation Buffer (B3) at 4°C after opening.
Materials:
NaCl
Bacto tryptone
yeast extract
ddH2O
5M NaOH
1000x stock chloramphenicol (25mg/ml)
Procedure:
Pellet 1–5 ml bacterial culture by centrifugation for 30 seconds. Discard supernatant.
Re-suspend pellet in 200 μl Plasmid Resuspension Buffer (B1). Vortex or pipet to ensure cells are completely resuspended. There should be no visible clumps.
Add 200 μl Plasmid Lysis Buffer (B2), gently invert tube 5–6 times, and incubate at room temperature for 1 minute. Color should change to dark pink, and solution will become transparent and viscous. Do not vortex.
Add 400 μl of Plasmid Neutralisation Buffer (B3), gently invert tube until neutralized, and incubate at room temperature for 2 minutes. Sample is neutralized when color is uniformly yellow and precipitate forms. Do not vortex.
Centrifuge lysate for 2–5 minutes. For culture volumes >1 ml, we recommend a 5 minute spin to ensure efficient RNA removal by RNase A. Pellet should be compact; spin longer if needed.
Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through.
Re-insert column in the collection tube and add 200 μl of Plasmid Wash Buffer 1. Centrifuge for 1 minute. Discarding the flow-through is optional.
Add 400 μl of Plasmid Wash Buffer 2 and centrifuge for 1 minute.
Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If there is any doubt, re-spin the column for 1 minute.
Add ≥ 30 μl DNA Elution Buffer to the center of the matrix. Wait for 1 minute, then spin for 1 minute at 16,000 x g to elute the DNA. Nuclease-free water (pH 7–8.5) can also be used to elute the DNA.
SOURCE: New England Biolabs
Single-temperature Double Digest for gBlocks
Preliminaries:
This is the Double Digest Protocol with EcoRI-HF and PstI, using a common reaction and same incubation temperature for both enzymes.
Double Digest Recommendations for EcoRI-HF® + PstI:
Digest in NEBuffer 2.1 at 37°C.
At least one enzyme has < 100% activity in this buffer, so additional units of enzyme and/or longer incubation time may be necessary.
Enzyme:
Temp: (*C)
 %Activity in NEBuffer
1.1
2.1
3.1
EcoRI-HF
37
10
100
10
Pstl
37
75
75
100
Materials:
Synthesized gBlocks › NEBuffer
NEBuffer
1X
NEB Restriction Enzymes
Deionized Water
Procedure:
Single Temperature DD reaction:
Use the plasmids recovered from minipreps to test that the vector and insert are the right size.
Set up the following reaction (total reaction volume 20 μl).
A
B
Material
Reagent volumes (ul)
Buffer (10x)
2
DNA *
10
Restriction enzyme EcoR1
1
Restriction enzyme Pstl
1
Nuclease free water
6
Total Volume (ul)
20
Recommended maximum of 1 μg of substrate per 10 units of enzyme. Restriction enzymes should be added to the mixture last.'
Mix components by pipetting the reaction mixture up and down, or by "flicking" the reaction tube.
Quick ("touch") spin-down in a microcentrifuge. Do not vortex the reaction.
Incubate for 1 hour at the enzyme-specific appropriate temperature. Then heat inactivate at 80°C.
PCR purify at the end to remove the short fragments.
Characterizing araC-Pbad Promoter
Promoter page (on the parts registry website) suggests that concentrations of arabinose between 0.001 - 0.02% are used to overexpress this promoter and we have modeled it essentially as a constitutive promoter so we should test this construct with the cell-free extract to confirm this.
Materials
DNA Template (BBa_K808000)
Total Amino Acid Mixture
S30 Premix Without Amino Acids
S30 Extract
Nuclease Free Water
L-Arabinose
Preparing the DNA Template
Miniprepped plasmids have been test digested and all the insert sizes look OK. Final concentration was measured to be 410ng/μL.
Cell-Free Extract Standard Protocol
We will setup the following tests: +ve control = GFP from a constitutive promoter (to show that cell free extract works) -ve control = just cell free extract -ve Ara = cell free extract with arabinose but no DNA -ve DNA = cell free extract with DNA but no arabinose 5 tests with 2%, 1%, 0.5%, 0.1% and 0.05% of arabinose (final concentration) respectively
Prepare arabinose stock. As the final volume will be 11μL and we want to pipette in 1μL of arabinose stock conc. will need to be 11 times that of the final, i.e. 22%, 11%, 5.5%, 1.1% and 0.55%. Dissolve 1.1 g in 5 ml of water to get 22% and make a dilution series.
Component of the System
Volume (μL)
Volume for 3 Replicates (μL)
Volume for 30 Reactions (μL)
DNA template
1
3
Total amino acid mixture
1
3
30 (15+15)
S30 premix without amino acids
4
12
120
Master mix
S30 extract
3
9
90
Nuclease free water instead of miR
1
3
30
Nuclease free water/ arabinose
1
3
2. As we will be setting up 9 reactions with 3 replicates each we will need to mix enough cell free extract for 30 reactions.
3. Prepare master mix
4. Pipette up and down to mix the components
5. Incubate the reaction mixture at 30°C for 10 hours in the plate reader, and measure every 10 minutes.
Well
Test
Volume of master mix (μL)
Substitute Nuclease Free Water (μL)
Arabinose (μL)
DNA Template (μL)
1 AB
-ve control
27
6
2 AB
-ve DNA
27
3
3 AraC
3 AB
2%
27
3
3
4 AB
1%
27
3
3
5 AB
0.50%
27
3
3
6 AB
0.10%
27
3
3
7 AB
0.05%
27
3
3
8 AB
+ve control
27
3
3 (constitutive promoter)
Characterizing 15-5p Toehold Switch
Materials
Nuclease Free Water
miRs
DNA Template
Total Amino Acid Mixture
S30 Premix Without Amino Acids
S30 Extract
L-Arabinose
Preparing the DNA Template
Miniprepped plasmids have been sequenced and the sequence of our toehold switch for 15b-5p has been confirmed. Concentration of this plasmid has been measured to be 95ng/μL so will need to be dried down to the conc. of approx. 500ng/μL so that only 1 μL has to be used per reaction. Final concentration has been measured to be 460ng/μL.
Resuspending the miRNA mimicks
Centrifuge the tube for 3−5 sec at a minimum of 3000 x g to pellet the material to the bottom of the tube.
To make a 10 μM concentration: Take the number of nmoles of miRNA mimick in the tube (5) and multiply that by 100 (500). This number will be the number of μL of nuclease free water to add to get a 100 μM solution.
Store resuspended oligonucleotides as several small aliquots at –20°C.
Serial Dilutions of miRs:
Tube Name
Stock Concentration
Final Concentration (/11)
Volume of Previous Concentration (μL)
Volume of Nuclease Free Water (μL)
A
10 μM
9x10⁸ fM
10
90
B
1 μM
9x10⁷ fM
10
90
C
100 nM
9x10⁶ fM
10
90
D
10 nM
9x10⁵ fM
10
90
E
1 nM
9x10⁴ fM
10
90
F
100 pM
9000 fM
10
90
G
10:00 PM
900 fM
10
90
H
1:00 PM
90 fM
10
90
There will be 13 tests including controls with 3 repeats each so we need enough master mix for 45 reactions.
Component of the System
Volume (μL)
Volume for 3 Replicates (μL)
Volume for 45 Reactions (μL)
DNA template
1
3
Total amino acid mixture
1
3
60 (30 + 30)
Master mix
S30 premix without amino acids
4
12
240
S30 extract
3
9
180
L-Arabinose
1
3
+ve control = GFP from a constitutive promoter (to show that cell free extract works) -ve control = just cell free extract -ve ara = cell free extract with arabinose but no DNA -ve DNA = cell free extract with DNA but no arabinose -ve mix = cell free extract with arabinose and DNA but no miR
Wells
Test
Volume of master mix (μL)
Substitute Nuclease Free Water (μL)
Arabinose (μL)
miR (μL)
DNA Template (μL)
1 CDE
-ve control
24
9
2 CDE
-ve miR
24
6
3
3 CDE
-ve DNA
24
6
3
4 CDE
-ve mix
24
3
3
3
5 CDE
A
24
3
3
3
6 CDE
B
24
3
3
3
7 CDE
C
24
3
3
3
8 CDE
D
24
3
3
3
9 CDE
E
24
3
3
3
10 CDE
F
24
3
3
3
11 CDE
G
24
3
3
3
12 CDE
H
24
3
3
3
13 CDE
27
24
3
3 (miRNA27)
3
14 CDE
+ve control
24
6
3
Major sponsors
Mr John
Mendez
Contact us
