Team:CLSB-UK/Notebook
CLSB-UK
Project
Design
Parts
Demonstrate
Judging
Contribution
Lab work
Results
Experiments
Safety
Notebook
InterLab
Human Practices
Silver
Engagement
High School Guide
Gold Integrated
Safety
Cost model
Modeling
NUPACK model
Mass action kinetics
Stochastic model
Cost model
Software
Overview
Toehold tools
SynBioBot
Hardware
Overview
Interpreter
Measurement
People
Team
Collaborations
Attributions
Sponsors
We strongly recommend using Google Chrome or Mozilla Firefox to view our wiki.
Notebook
Let it glow! (well, technically fluoresce)
Contents
1 Notebook
1.1 23 March
1.2 30 March
1.3 22 June
1.4 26 June
1.5 27 June
1.6 29 June
1.7 30 June
1.8 03 June
1.9 04 July
1.10 05 July
1.11 06 July
1.12 10 July
1.13 11 July
1.14 12 July
1.15 13 July
1.16 14 July
1.17 15 July
1.18 17 July
1.19 18 July
1.20 19 July
1.21 20 July
1.22 24 July
1.23 25 July
1.24 26 July
1.25 11 September
1.26 20 September
1.27 21 September
1.28 22 September
1.29 25 September
1.30 26 September
1.31 02 October
1.32 03 October
1.33 04 October
1.34 05 October
1.35 06 October
1.36 12 October
1.37 13 October
Notebook
23 March
First lab preparation session. Practiced pipetting different volumes into eppendorfs using micropipettes.
30 March
Second lab preparation session. Made bacterial streaks and lawns and learnt about sterile technique.
22 June
Starting the real lab work! Amplifying the parts from IDT with PCR and running the gels to check if we had amplified a sufficient amount, unfortunately, this PCR did not go to well, so we will try again. These were the conditions used for the first round of PCR reactions:
Component
Volume for the 25μL reaction (μL)
GFP/ Luciferase
Basic blocks
Q5 High-Fidelity 2X Master Mix
12.5
12.5
10uM forward primer
1.25
1.25
10uM reverse primer
1.25
1.25
Template DNA (1ng)
1
1
Nuclease-free water
9
9
Step
Temperature (s)
Time (s)
Initial denaturing
98
30
25 cycles
98
10
62
20
72
30
Final extension
72
300
Hold
4
Indefinite
26 June
Second round of PCR, this time we were met with more success. We slightly modified the conditions the second time in order to increase the yield:
Component
Volume for the 25μL reaction (μL)
Volume for the 50μL reaction (μL)
Q5 High-Fidelity 2X Master Mix
12.5
25
10uM forward primer
1.25
2.5
10uM reverse primer
1.25
2.5
Template DNA (1ng)
2.5
5
Nuclease-free water
7.5
15
Step
Temperature (s)
Time (s)
Initial denaturing
98
30
25 cycles
98
10
64
25
72
30
Final extension
72
300
Hold
4
Indefinite
These were the results of the second round of PCR:
Ladder
15GFP
27GFP
15Luc
27Basic
15Basic
27Luc
27 June
PCR cleanup and double digests were a great success, our banding patterns were very good when we ran the gels, but the lane containing the digest from the linearized plasmid backbone was empty. These were the reagents for the digest of the gBlocks:
Reagent
Volume for the 25μL reaction (μL) Reagent Volumes (µl)
Buffer (10x)
5
DNA
43
Restriction Enzyme EcoRI
1
Restriction Enzyme SpeI
1
Nuclease Free Water (μl)
0
Total Volume (μl)
50
Recommended maximum of 1 µg of DNA per 10 units of enzyme.
These were the reagents for the digest of the plasmid, BBa_J04450:
Reagent
Reagent Volumes (µl)
Buffer (10x)
3
DNA *
4
Restriction Enzyme EcoRI
1.1
Restriction Enzyme PstI
1.1
Deionized Water (μl)
19.8
Total Volume (μl)
30
Recommended maximum of 1 µg of substrate per 10 units of enzyme.
28 June - Repeated digests for plasmids and this time we could easily identify the bands which corresponded to the plasmid backbone. We carried out the ligation protocol as shown below:
Component
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
GFP15b
GFP27b
Luc15b
Luc27b
15b
27b
10X T4 DNA Ligase Buffer
2
2
2
2
2
2
Vector DNA: 40 ng (0.040 pmol)
9
9
9
9
9
9
Insert DNA mass (ng)
52
52
106
106
8
8
DNA concentration (ng/ul)
9.7
88.6
42
18
16
14
Insert DNA volume
6
3
5
7
4
4
Nuclease-free water
2
5
3
1
4
4
T4 DNA Ligase
1
1
1
1
1
1
Total
20
20
20
20
20
20
Now we are ready to transform our competent cells.
29 June
A small subteam traveled to Imperial university in order to transform E. coli with our plasmids.
30 June
Transformations seemed to have gone well as you can see in the image below and we have lots of colonies growing. Let’s hope the plasmids contain our inserts!
03 June
Successfully transformed colonies of E. coli from the plates were re-streaked and set up in LB broths in preparation for minipreps. These were the materials and respective quantities which made up the LB medium:
Material
Quantity
NaCl
1g
Bacto Tryptone
6g
Yeast Extract
3g
ddH₂O
600ml
5M NaOH
100μl
1000x Stock of Chloramphenicol (25mg/ml)
600μl
Agar
9g
04 July
Spinning down our broths and doing our first set of minipreps. Disaster struck at the elution step of the minipreps so we will have to set up more broths tomorrow.
05 July
Second round of making broths and re-streaking our bacteria, tonight we will pray that we have learned from our mistakes and hope that our minipreps will go well tomorrow!
06 July
The centrifuge broke so there will be no lab work for today as the minipreps heavily rely on the centrifuge.
10 July
We bought a new centrifuge in order to avoid missing out on more crucial lab work sessions. Next round of minipreps, we will send off some of our plasmids to be sequenced to confirm the composition of the plasmids. Instead of using elution buffer for the last step of the minipreps we used nuclease-free water because it allows us to insert the miniprepped DNA directly into the cell-free extracts.
11 July
Not a high enough concentration of our plasmids to be sequenced so we need to carry out more minipreps.
12 July
Made more broths and re-streaked our transformed E. coli.
13 July
Minipreps and sending the plasmids off for sequencing.
14 July
Received the results from sequencing - unfortunately, our plasmids were contaminated so we have to start from the beginning. The sequencing data indicated that there might have been some contamination of the plasmid backbone used for ligations.
15 July
We tested the plasmid backbone by running it on a gel and came to the conclusion that it must be contaminated with nucleases.
17 July
PCR Amplification, PCR Cleanup and Fluorometer testing.
18 July
Digests and ligations.
19 July
Prepared the selection plates for transformation
20 July
Transformed competent E. coli with constructs. However, there were not many white colonies therefore we had to restreak these colonies using “dilution streaking” to isolate the cells that had been transformed with our recombinant plasmid.
This was the first selection plate and as you can see there aren’t any obvious white colonies:
We utilized “dilution streaking” on the second selection plate and were able to isolate some white colonies:
24 July
Restreaks and broths
25 July
Minipreps and sent plasmids off for sequencing.
26 July
The sequencing data came back and we worked out that the construct was heavily mutated by the E. coli. This meant that the constructs must have been toxic to them. During the next few weeks we would be redesigning the gBlocks with an inducible promoter rather than an constitutive one. We would also have to wait for the gBlocks to be delivered by IDT. We decided to discontinue the use of Luciferase gBlocks because we were running out of time and measuring the activity of luciferase is more difficult than GFP. We also tried to clone these parts into pSB1C3 but the ligation/transformation was unsuccessful so we decided not to persist with  luciferase.
11 September
The new gBlocks arrived and we immediately amplified them so that we had more DNA to work with. A biology field trip meant that we were unable to do lab work for over a week.
20 September
We carried out ligations. The protocol was slightly modified from before and did not contain Luciferase gBlocks due to the reasons stated above:
15b ! 27b
GFP15b
GFP27b
Component
Volume (μl)
Volume (μl)
Volume (μl)
Volume (μl)
10X T4 DNA Ligase Buffer
1
1
1
1
Vector DNA: 30 ng (0.030 pmol)
2.1
2.1
2.1
2.1
Insert DNA mass (ng)
6
6
90
90
DNA concentration (ng/ul)
15
15
18
18
Insert DNA volume
1
1
6
6
Nuclease-free water
4.9
4.9
0
0
T4 DNA Ligase
1
1
1
1
Total
10
10
10.1
10.1
21 September
Restreaks and LB Broths
22 September
Spinning down broths and pelleting the E. coli.
25 September
We resuspended the E. coli pellets (first step of miniprep protocol) in the morning and then finished the procedure later. We were able to isolate plasmid DNA for all four of our constructs.
26 September
More minipreps and we sent off the plasmids for sequencing. Two of the constructs came back as correct. For the other two we needed to pick up a few more colonies, perform test digests and send the most promising plasmids for sequencing.
02 October
We setup test digests in order to work out our plasmid concentrations. This was also to see which construct needed more minipreps.
03 October
Setup LB Broths.
04 October
LB Broths were spun down. E. coli pellets were resuspended.
05 October
Minipreps
06 October
More minipreps
12 October
A small subteam traveled to Imperial University in order to conduct the penultimate protocol. This specifically involved the characterisation of our araC-Pbad promoter where we would determine the optimum L-Arabinose concentration for GFP expression. Although this promoter has been previously characterized, it had not be done using toehold switches and in a cell-free system so we thought it would be beneficial for future iGEM teams.  Our experiment not only involved different concentrations of L-Arabinose but also positive and negative controls as shown below:
Well
Test
Volume of master mix (μL)
Substitute Nuclease Free Water (μL)
Arabinose (μL)
DNA Template (μL)
1 AB
-ve control
27
6
2 AB
-ve DNA
27
3
3 AraC
3 AB
2%
27
3
3
4 AB
1%
27
3
3
5 AB
0.50%
27
3
3
6 AB
0.10%
27
3
3
7 AB
0.05%
27
3
3
8 AB
=+ve control
27
3
3 constitutive
We were able to establish that 0.5% of L-Arabinose was the optimum concentration for GFP expression.
13 October
Today the aim was to characterize our 15GFP toehold switch and this would also be carried out in Imperial University’s labs. We wanted to validate that an increase in miRNA concentration would result in an increase of GFP expression. In order to create a range of miRNA concentrations we set up a serial dilution:
Tube Name
Stock Concentration
Final Concentration (/11)
Volume of Previous Concentration (μL)
Volume of Nuclease Free Water (μL)
A
10 μM
9x10⁸ fM
10
90
B
1 μM
9x10⁷ fM
10
90
C
100 nM
9x10⁶ fM
10
90
D
10 nM
9x10⁵ fM
10
90
E
1 nM
9x10⁴ fM
10
90
F
100 pM
9000 fM
10
90
G
10:00 PM
900 fM
10
90
H
1:00 PM
90 fM
10
90
Now we added the miRNA to the cell-free system along with other components such as arabinose and amino acids. It was essential that we also carried out controls in order to determine factors such as background fluorescence:
Wells
Test
Volume of master mix (μL)
Substitute Nuclease Free Water (μL)
Arabinose (μL)
miR (μL)
DNA Template (μL)
1 CDE
-ve control
24
9
2 CDE
-ve miR
24
6
3
3 CDE
-ve DNA
24
6
3
4 CDE
-ve mix
24
3
3
3
5 CDE
A
24
3
3
3
6 CDE
B
24
3
3
3
7 CDE
C
24
3
3
3
8 CDE
D
24
3
3
3
9 CDE
E
24
3
3
3
10 CDE
F
24
3
3
3
11 CDE
G
24
3
3
3
12 CDE
H
24
3
3
3
13 CDE
27
24
3
3 (miRNA27)
3
14 CDE
Positive Control
24
6
3
Major sponsors
Mr John
Mendez
Contact us
