<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK_HP_Silver skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK/HP/Silver</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="jumbotron full_width" style="background-image: url('https://static.igem.org/mediawiki/2017/6/6b/T--CLSB-UK--silver_cover.jpg');"><DIV><H1 style="font-size: 89px">Silver Human Practices</H1><H2 style="font-size: 24px">Sorry to whoever's head got chopped off in this photo</H2></DIV></DIV><DIV class="content_wrapper"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Silver_Human_Practices"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Silver Human Practices</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Screening"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Screening</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Other_methods_of_quantifying_miRNAs"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Other methods of quantifying miRNAs</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#qPCR"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">qPCR</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Microarrays"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Microarrays</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Applications_to_other_disease"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Applications to other disease</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Safety"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Safety</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Silver_Human_Practices">Silver Human Practices</SPAN></H1><P>Our sensor has the potential to reduce the mortality rates of all kinds of diseases through early diagnosis. We investigated how it would be used effectively in a clinical environment.
</P><P>In addition, we discussed our design with as many experts and members of the general public as possible - so we could get their opinions and, at the same time, improve public understanding about synthetic biology and its possibilities in the field of diagnostics.
</P><DIV class="image_container"><DIV><H2 id="Engagement">Engagement</H2><P>Synthetic biology is incredible. We took every opportunity to teach others about it and our project. We have hosted regular events to engage the public in our work, iGEM and synthetic biology. This year we had a stand at a number of open days, gave several talks, held an iGEM open evening and also held numerous workshops for younger boys at our school. Additionally, we created a guide for high schools entering iGEM based off our own experiences and those of other teams who completed our survey.
</P><P>See our <A href="/Team:CLSB-UK/Engagement" title="Team:CLSB-UK/Engagement">engagement</A> and <A href="/Team:CLSB-UK/HP/HS_Guide" title="Team:CLSB-UK/HP/HS Guide">high schools guide</A> for more details.
</P></DIV></DIV><DIV class="image_container"><DIV><H2 id="Clinical Implementation">Clinical Implementation</H2><P>Our project is a diagnostic tool. We looked into how our sensor could improve current cancer diagnostics and be applied as a screening tool.
</P><P>We produced a survey which was filled out by 16 doctors, including oncologists, gastroenterologists and respiratory consultants. You can view a more precise breakdown of the responses <A href="/Team:CLSB-UK/HP/Silver/Survey_Responses" title="Team:CLSB-UK/HP/Silver/Survey Responses">here</A>.
</P><P>Several problems with cancer diagnostics methods were raised by these doctors. In particular, NSCLC was described as difficult to detect. Late presentation of symptoms was identified as the biggest challenge in its diagnosis. This played a role in our decision to choose NSCLC as our tool could be used to screen patients, and achieve the early diagnosis that would reduce mortality rates.
</P><P>Tests are usually only done once symptoms appear - as this is when the patient goes to their doctor. Unfortunately, for cancers like NSCLC, symptoms usually only appear once the cancer has metastasized and can only be treated palliatively. Early detection is vital as survival rates are much higher in earlier stages - stage one NSCLC tumors can easily be excised surgically.
</P></DIV><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Philippe Verdy, in public domain</P></DIV><H3><SPAN class="mw-headline" id="Screening">Screening</SPAN></H3><P>As the primary application of our sensor would be in screening, we carried out research into previous attempts to screen for NSCLC. This allowed us to gain an understanding of the problems with existing technologies and how our tool could be an improvement.
</P><P>Screening for NSCLC is attractive due to the asymptomatic nature of the disease in its early stages, and relatively good prognosis when detected early. Numerous trials have been carried out to investigate the efficacy of low dose computerized tomography (LDCT) in improving mortality rates for NSCLC. Previous trials have shown other screening methods such as sputum cytology or chest radiography to produce no reduction in mortality rates. The national screening trial in America showed with LDCT, a 20% reduction in mortality rates from lung cancer in high risk groups compared to X-ray.<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> However, the benefits of screening remains highly controversial due to high rates of false positives leading to unnecessary further testing, thoracotomies for benign lesions and associated cost.<SUP id="cite_ref-gopal_2-0" class="reference"><A href="#cite_note-gopal-2">[2]</A></SUP></P><P><SVG style="max-height: 500px;" xmlns="http://www.w3.org/2000/svg" stroke-width="28.2" viewbox="0 0 16452.9 9111.1" stroke-linejoin="round"><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="2279.9" y="8345.7"><TSPAN fill="#595959">IA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="4437.9" y="8345.7"><TSPAN fill="#595959">IB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="6551.9" y="8345.7"><TSPAN fill="#595959">IIA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="8709.9" y="8345.7"><TSPAN fill="#595959">IIB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="10825.9" y="8345.7"><TSPAN fill="#595959">IIIA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="12982.9" y="8345.7"><TSPAN fill="#595959">IIIB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="15225.9" y="8345.7"><TSPAN fill="#595959">IV</TSPAN></TSPAN></TSPAN></TEXT><G fill="#a4a4a4" font-family="Montserrat" font-size="317.5"><TEXT x="-599.1" y="-439.3"><TSPAN><TSPAN x="1036.9" y="8077.7"><TSPAN>0</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-506.3" y="-439.3"><TSPAN><TSPAN x="919.7" y="6935.7"><TSPAN>10</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-572.6" y="-439.3"><TSPAN><TSPAN x="853.4" y="5794.7"><TSPAN>20</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-575.1" y="-439.3"><TSPAN><TSPAN x="850.9" y="4653.7"><TSPAN>30</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-577" y="-439.3"><TSPAN><TSPAN x="849" y="3511.7"><TSPAN>40</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-584.4" y="-439.3"><TSPAN><TSPAN x="841.6" y="2370.7"><TSPAN>50</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-591.2" y="-439.3"><TSPAN><TSPAN x="834.8" y="1229.7"><TSPAN>60</TSPAN></TSPAN></TSPAN></TEXT></G><TEXT x="-171216.7" y="-39739.3" fill="#040404" stroke-width="1203.8" font-weight="600" font-size="511.8" font-family="Montserrat" text-anchor="middle"><TSPAN><TSPAN x="8099.1" y="397.7"><TSPAN>5 year NSCLC survival rates by stage</TSPAN></TSPAN></TSPAN></TEXT><G fill="#040404" text-anchor="middle" font-family="Montserrat" font-size="388.1"><TEXT x="396.9" y="-439.3"><TSPAN><TSPAN x="8113.9" y="9034.7"><TSPAN>Stage Diagnosed</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-5495.3" y="-5793.1" transform="rotate(-90)"><TSPAN><TSPAN x="-4608.3" y="287.9"><TSPAN>Survival rate (%)</TSPAN></TSPAN></TSPAN></TEXT></G></SVG></P><P><SPAN style="font-size: 12px; color: #888; text-align: center; display: block">Graph data from the American Cancer Society.<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP></SPAN></P><P>
The meta-analysis performed by Gopal et al. showed that using LDCT to screen high risk individuals 70% of lung cancers were diagnosed at stage one compared to the 16% detected at stage one during routine care. However, the benefits in early detection may be outweighed by the limitations of the screening method. The screening arm produced a massive number of false positives - for every 9 stage one tumors detected, there were 235 false positive nodules and 4 thoracotomies for benign lesions performed. Compared to the control, the false positive rate was a 3.1 times larger and patients were 4 times as likely to have an unnecessary thoracotomy.<SUP id="cite_ref-gopal_2-1" class="reference"><A href="#cite_note-gopal-2">[2]</A></SUP> With the risks of complications from diagnostic procedures, radiation exposure and effect on quality of life, there are major downsides to such a screening method. <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></P><P>There is a lack of compelling evidence to support LDCT screening for lung cancer - however screening represents one of the most realistic prospects to reduce NSCLC mortality rates aside from reducing the numbers of smokers. With frequency and length of screening optimized to reduce false positives, LDCT may yet prove effective.<SUP id="cite_ref-dajac_5-0" class="reference"><A href="#cite_note-dajac-5">[5]</A></SUP></P><P>Screening is also effective in many other cancers. However, the biology of the cancer has a strong effect on whether screening is likely to be useful.
Screening would be redundant for indolent cancers - and treatment-related morbidities may mean it has a net negative impact. Screening is also ineffective for cancers which develop very rapidly, as they they are likely to present between screens. Therefore, very frequent screening is needed for high risk population groups but preventative action is usually more beneficial.<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP></P><P>Screening has a bias to detect indolent cancers, leading to overdiagnosis - often cancers detected will never become symptomatic or their progression would be so slow that the patient will die of other causes first.<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP> Depending on organ type, indolent cancers account for 15-75% of those detected. Therefore, it is vital that a new method of screening can differentiate between indolent and progressive cancers.
</P><P>MiRNAs have the potential to achieve this - it has been shown that a combination of miRNAs can identify NSCLC in asymptomatic individuals with 80% accuracy and  differentiate benign nodules from NSCLC tumors.<SUP id="cite_ref-bianchi_8-0" class="reference"><A href="#cite_note-bianchi-8">[8]</A></SUP> This would avoid the problems that accompany unnecessary further testing. Even if diagnosis still took place using LDCT screening, an effective way of measuring the relevant miRNAs (that our sensor could potentially provide) would be vital to classify a tumor and reducing overdiagnosis.
</P><P>Our screening method could produce massive increases in the detection of stage I cancers with far lower cost and greater objectivity than current methods. It also could reduce the rates of overdiagnosis. Furthermore, the results of our test could conclusively rule out the possibility of cancer in certain individuals preventing the need for further investigation. 
</P><DIV class="image_container"><DIV><H2 id="Benefits of our system">Benefits of our system</H2><P>The problems with current diagnostic methods were also identified by doctors who completed our survey. Many patients have to undergo these tests - as there is no other way to rule out NSCLC. Although biopsy and CT scan would still be required to stage a tumor, our sensor could rule out cancer in many patients. Below are some of the problems with current diagnostic methods, and how our sensor be beneficial.
</P><P>In our survey, all doctors agreed that new diagnostic methods were needed and that liquid biopsies could be more effective than current methods. We carried out research into current methods based off their concerns:
</P></DIV><P style="text-align: right; font-size: 12px; color: #888;">Attribution: NithinRao, in public domain</P></DIV><P>CT scanning is the best way of making an initial diagnosis of NSCLC.<SUP id="cite_ref-lcnice_9-0" class="reference"><A href="#cite_note-lcnice-9">[9]</A></SUP> It is more accurate than an X-ray and sputum cytology (examining coughed up mucus) - these have been shown to provide no benefit in screening and can increase false positive rates.<SUP id="cite_ref-dajac_5-1" class="reference"><A href="#cite_note-dajac-5">[5]</A></SUP> Sputum cytology is rarely used in the UK as it has low sensitivity with inconsistent results (although there is promise for the future <SUP id="cite_ref-sputum_10-0" class="reference"><A href="#cite_note-sputum-10">[10]</A></SUP>). Further tests, such as ultrasound may be used to confirm the results of a CT, and repeat CT scanning is used to assess the growth of a tumor. PET-CT, ultrasound, MRI, bronchoscopy (with possible biopsy) and X-ray are performed for specific cases.<SUP id="cite_ref-lcnice_9-1" class="reference"><A href="#cite_note-lcnice-9">[9]</A></SUP></P><P>However, CT scanning has its limitations as the initial diagnostic method. It is cumbersome as it requires specialist centers with immobile equipment.<SUP id="cite_ref-bianchi_8-1" class="reference"><A href="#cite_note-bianchi-8">[8]</A></SUP> The results of a CT scan are difficult to interpret; they are so sensitive that benign lesions are frequently picked up as CT scanning also cannot distinguish between benign nodules and those that will progress to be cancerous.<SUP id="cite_ref-sputum_10-1" class="reference"><A href="#cite_note-sputum-10">[10]</A></SUP> Repeat scans are then required to observe whether the lesion grows (and is therefore malignant) or is benign.
</P><P>When low dose CT was trialed as a lung cancer screening program it caused high rates of false positives (see the <A href="#Screening">screening section</A>).<SUP id="cite_ref-gopal_2-2" class="reference"><A href="#cite_note-gopal-2">[2]</A></SUP>. False positive CT scans then lead to unnecessary invasive procedures - like bronchoscopy or even surgery.<SUP id="cite_ref-dajac_5-2" class="reference"><A href="#cite_note-dajac-5">[5]</A></SUP> Furthermore, it takes up to two weeks to schedule a CT and the results from current tests scans can take a further week to come back in the UK.<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP> CT scans for NSCLC are usually done with contrast - which needs to be injected - meaning the procedure is somewhat invasive. Furthermore, CT scans result in exposure to radiation which may increase the risk of cancer in the future<SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP>.
</P><P>As a result, there is still a need for away to accurately screen high risk population groups  for NSCLC and minimize unnecessary procedures. We believe that our system could be that solution.
</P><DL><DT> Late presentation of symptoms</DT><DD> Our system can be used in screening by setting cut-offs appropriately. Abnormal miRNA levels can show cancers in their early stages <SUP id="cite_ref-etheridge_13-0" class="reference"><A href="#cite_note-etheridge-13">[13]</A></SUP>. Our test could rule out cancer in an individual, and refer those with positive results for further testing. </DD></DL><DL><DT> Equivocal test results</DT><DD> The results are easy to interpret as miRNA levels can be several times higher or lower in patients with cancer - through adjusting the cut off values, and using multiple miRNAs in tandem, cancer can be ruled out in many patients. This has the potential to be more efficient than radiologists interpreting a CT scan.Furthermore, our test is highly specific but would have lower sensitivity (see gold HP) whereas CT scans are highly sensitive but have lower specificity. By using the results of both in combination, rates of false positives could be massively reduced. </DD></DL><DL><DT> Ease of use</DT><DD> Blood sample can be taken by someone with minimal training without a highly trained radiologist required. The processing of the miRNA would be easiest and most efficient in a specialist lab. However, there are kits for miRNA extraction<SUP id="cite_ref-14" class="reference"><A href="#cite_note-14">[14]</A></SUP> that could be used in rural areas, and combined with our cheap £4 <A href="/Team:CLSB-UK/Hardware" title="Team:CLSB-UK/Hardware">fluorometer</A>, it could allow screening for cancer in developing countries.</DD></DL><DL><DT> Cost</DT><DD> Our system is cheap, less than £17/test ($23). Most doctors said that liquid biopsies could be more cost-effective. See our <A href="/Team:CLSB-UK/Model#Cost_Analysis" title="Team:CLSB-UK/Model">cost analysis</A> on our modeling page for an exact breakdown.</DD></DL><DL><DT> Invasiveness</DT><DD> Our system for NSCLC simply involves taking a blood sample, making it minimally-invasive. However, miRNAs found in other body fluids (e.g. urine, saliva) could allow for completely non-invasive diagnosis. For example using miRNAs in urine to diagnose prostate cancer (see <A href="#Applications_to_other_disease">below</A>)</DD></DL><DL><DT> Timeframe</DT><DD> Results from our system are available within hours. The processing of blood to purify miRNA would take ~2 hours according to expert Dr Dan Pregibon currently. However, there is scope for this to be automated. After adding the purified miRNA to our sensor, and quantifying fluorescence, different miRNA levels could be distinguished within hours (peak fluorescence occurs within 10 hours). See our <A href="/Team:CLSB-UK/Results" title="Team:CLSB-UK/Results">results page</A> for more details.Dr Desai, a consultant radiologist, said that the results of a blood test using our sensor could help to relieve a patient’s anxiety far sooner than is currently possible. Our test, which can be carried out at a GP surgery, would comprise sending off the sample and processing the results with our sensor. This could be achieved in a few days rather than the 3 weeks required for a CT scan. In a hospital, this process could be even faster.</DD></DL><DL><DT> Portability</DT><DD> Through using the flourometer, and miRNA extraction kits, our sensor could be highly portable - unlike a CT scanner. This would be highly important for use in developing countries - bringing the test to individuals, when transport infrastructure for them to get to an urban hospital may not be in place.In developed countries, blood tests could be carried out locally and the samples would be sent to a national lab for processing and testing. For example, nurses could be sent to higher-risk population groups e.g. nursing homes, allowing more people to be screened as there would be fewer barriers to screening (e.g. mobility issues).</DD></DL><P>Dr Desai, a respiratory consultant, stressed that another benefit of our test would be in reducing the number of CT scans required to rule out NSCLC. Given the shortage of radiologists, this could prove extremely beneficial to the NHS. This shortage would also make screening with CT impractical. 
</P><DIV id="usage" class="image_container"><DIV><P>It is important to note that our project would not replace current diagnostic methods; CT and PET scans, along with biopsy would still be crucial for staging tumors and informing a therapeutic decision. Our sensor would be used to:</P><UL><LI>Screen high risk population groups for cancer and other diseases, and refer people with positive test results to further testing. </LI><LI>Rule out cancer or other diseases in certain high risk individuals - by setting cut-offs appropriately, and using several miRNA signatures in tandem - preventing unnecessary further testing could be prevented (this is further explained in <A href="/Team:CLSB-UK/HP/Gold_Integrated" title="Team:CLSB-UK/HP/Gold Integrated">gold human practices</A>)</LI><LI>Be used in tandem with a CT scan to reduce false positives</LI><LI>Mass-population screening. For example, testing for a variety of diseases every year for those over a certain age</LI><LI>Provide point of care information to doctors. With a multiplexing assay of toehold switches, a myriad of diseases could be screened for, and ruled out, simultaneously assisting doctors in making a diagnosis.</LI></UL></DIV></DIV><P>We believe our tool could save millions of lives through early detection of various diseases. Non-small cell lung cancer was chosen as a proof of concept as NSCLC is often treatable when detected early (as cited by doctors), and had 2 well characterized miRNA biomarkers. However, due to the programmable nature of toehold switches, our concept could be applied any miRNA sequence.
</P><P>Using multiple switches that regulate the production of reporter proteins with distinct emission peaks, with numerous wells in tandem, hundreds of cancers and other diseases could be screened for simultaneously, all using a single blood sample.
</P><P>We interviewed several doctors to gain further insight into how our design could be optimized for clincial use. These feature on our <A href="/Team:CLSB-UK/HP/Gold_Integrated" title="Team:CLSB-UK/HP/Gold Integrated">integrated human practices page</A>.
</P><H2><SPAN class="mw-headline" id="Other_methods_of_quantifying_miRNAs">Other methods of quantifying miRNAs</SPAN></H2><P>To further consider the applications of our project, we also looked at current methods of quantifying miRNAs in qPCR and microarrays. We saw that both had their limitations, and a novel method was required if miRNAs were to be used clinically as a biomarker for cancer.
</P><H3><SPAN class="mw-headline" id="qPCR">qPCR</SPAN></H3><P>qPCR is an accurate and sensitive method of quantifying miRNAs. To provide a common sequence feature for amplification of miRNAs, a polyadenylation step is required. This adds an oligonucleotide tail to the end of the miRNA to which the primer binds to. After this, a reverse transcription transcription step is carried out to produce cDNA. Standard qPCR can then be used.<SUP id="cite_ref-15" class="reference"><A href="#cite_note-15">[15]</A></SUP></P><P>This multi-step process is labour-intensive and as such, it is not suitable for scaling and would therefore be unsuitable for mass screening. Additionally, the enzymes and reagents are costly when attempting to quantify numerous different miRNAs.
</P><H3><SPAN class="mw-headline" id="Microarrays">Microarrays</SPAN></H3><P>It can be challenging to develop probes and hybridization conditions that work well to detect numerous miRNAs at once. Consequently, microarrays typically have issues with cross-hybridization <SUP id="cite_ref-16" class="reference"><A href="#cite_note-16">[16]</A></SUP>  leading to false positives. Furthermore, results are inconsistent even from the same platform. <SUP id="cite_ref-17" class="reference"><A href="#cite_note-17">[17]</A></SUP></P><P>We also spoke to Dan Pregibon, an expert who developed a new way of detecting miRNAs. He was intrigued by our idea to use toehold switches to sense these miRNAs. He said that our method could be used in less developed countries and to provide point of care information, in a way his would not. However, he explained that while his sensor has undergone significant testing, it is not ready for clinical use until miRNAs have been more robustly characterized. The same would apply to our sensor, however, research into miRNA diagnostics is very promising. It is likely to only be a matter of time until miRNAs have been sufficiently characterized to be used in clinical diagnosis.
</P><H2><SPAN class="mw-headline" id="Applications_to_other_disease">Applications to other disease</SPAN></H2><P>There many other diseases where screening has the potential to be effective - and miRNAs can be used as biomarkers to achieve this. MiRNAs, and their links to diseases, are still being researched. However, new relationships are constantly being discovered, and they have the potential to be applied to a myriad of cancers - and potentially other diseases in a point of care setting.
</P><DIV class="image_container"><DIV><H2 id="Pancreatic Cancer">Pancreatic Cancer</H2><P>Elevated levels of miR-196a and 196b can act as biomarkers for pancreatic cancer. These could be used for the screening of high risk individuals, for example those with familial pancreatic cancer. Current imaging methods used for screening are often unable to detect early cancers and high grade precursor lesions in high risk individuals <SUP id="cite_ref-pcslater_18-0" class="reference"><A href="#cite_note-pcslater-18">[18]</A></SUP>. The main reason mass population screening cannot be done is that there is no suitably accurate test.<SUP id="cite_ref-19" class="reference"><A href="#cite_note-19">[19]</A></SUP> Early detection could increase survival rates, as it allows curative pancreatic resection surgery.<SUP id="cite_ref-pcslater_18-1" class="reference"><A href="#cite_note-pcslater-18">[18]</A></SUP> Similar to NSCLC, late detection is a primary reason why mortality rates are so low - pancreatic cancer is asymptomatic in early stages, and symptoms usually only present themeselves in later stages where it is far more difficult to treat.<SUP id="cite_ref-20" class="reference"><A href="#cite_note-20">[20]</A></SUP> At stage 3 there is a median survival rate of 6-11 months and at stage 4 it is 2-6 months,<SUP id="cite_ref-21" class="reference"><A href="#cite_note-21">[21]</A></SUP> whereas surgically resectable pancreatic cancer has survival rates of 11-20 months<SUP id="cite_ref-22" class="reference"><A href="#cite_note-22">[22]</A></SUP>. Currently only 8% of people diagnosed are able to have surgery due to late diagnosis.<SUP id="cite_ref-23" class="reference"><A href="#cite_note-23">[23]</A></SUP> MiRNA as a target for screening could allow early detection and thus reduce mortality rates.
</P></DIV><SVG style="margin-top: 45px; margin-bottom: 20px;" xmlns="http://www.w3.org/2000/svg" version="1" viewbox="0 0 337.56 255.08"><TEXT x="127.11" y="9.46" style="-inkscape-font-specification:'Montserrat, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center" font-size="13.4" font-family="Montserrat" text-anchor="middle">Cancer</TEXT><TEXT x="67.65" y="254.88" fill="#333" font-size="13.4" style="-inkscape-font-specification:'Montserrat, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:end" font-family="Montserrat" text-anchor="end">Bowel</TEXT><TEXT x="96.82" y="255.02" fill="#333" font-size="13.4" style="-inkscape-font-specification:'Montserrat, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start" font-family="Montserrat">Pancreas</TEXT></SVG><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Cancer Research UK, <A style="color: #888" href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank" rel="noopener">© CC BY-SA 4.0</A></P></DIV><DIV class="image_container"><DIV style="margin: 36px 2% 0 0; width: 28%"><SVG style="width: 100%;" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 342.47 339.5"><TEXT x="-.55" y="97.44"><TSPAN x="-.55" y="97.44" fill="#333" font-size="14.1" font-family="Montserrat">Prostate</TSPAN><TSPAN x="-.55" y="111.44" fill="#333" font-size="14.1" font-family="Montserrat">gland</TSPAN></TEXT><TEXT x="1.41" y="146.93" fill="#333" font-size="14.1" font-family="Montserrat">T1</TEXT><TEXT x="1.41" y="180.94" fill="#333" font-size="14.1" font-family="Montserrat">T2</TEXT><TEXT x="326.88" y="207.46" fill="#333" font-size="14.1" font-family="Montserrat">T3</TEXT><TEXT x="287.45" y="83.46" fill="#333" font-size="14.1" font-family="Montserrat">Bladder</TEXT></SVG><P style="font-size: 12px; color: #888;">Attribution: Cancer Research UK, <A style="color: #888" href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank" rel="noopener">© CC BY-SA 4.0</A></P></DIV><DIV><H2 id="Prostate Cancer">Prostate Cancer</H2><P>
It has been shown that levels of 3 miRNAs found in urine are indicative of prostate cancer with high accuracy - giving the potential for a non-invasive test. <SUP id="cite_ref-pcfred_24-0" class="reference"><A href="#cite_note-pcfred-24">[24]</A></SUP> Furthermore, prostate cancer is another good candidate for screening - there are 100% 5 year survival rates when its detected in stages 1 and 2, 95% in stage 3 but this plummets to 30% in stage 4.<SUP id="cite_ref-25" class="reference"><A href="#cite_note-25">[25]</A></SUP> Early diagnosis allows curative treatment via radical prostatectomy or radiation, whereas metastatic cancer is incurable.<SUP id="cite_ref-pcfred_24-1" class="reference"><A href="#cite_note-pcfred-24">[24]</A></SUP></P><P>There is currently no screening program for prostate cancer as a reliable enough test does not exist.<SUP id="cite_ref-26" class="reference"><A href="#cite_note-26">[26]</A></SUP> Some trials report no benefit for prostate cancer screening using a protein biomarker PSA.<SUP id="cite_ref-27" class="reference"><A href="#cite_note-27">[27]</A></SUP> However, there are factors to explain this; the Sandblom et al. study used a small patient cohort, there were improvements in treatment that occurred during the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial and a different cutoff value was used in these trials to the value used in the European Randomized study of Screening for Prostate Cancer where benefits were seen. Despite the large European study showing increased survival rates, overdiagnosis and false positives remained problematic.<SUP id="cite_ref-28" class="reference"><A href="#cite_note-28">[28]</A></SUP> Furthermore, PSA has low specificity for prostate cancer as elevated levels can be caused by benign prostatic hyperplasia, prostatitis and diet alterations.<SUP id="cite_ref-29" class="reference"><A href="#cite_note-29">[29]</A></SUP> The 3-miRNA method could prevent unnecessary biopsies and aid the detection of early stage, aggressive prostate cancer.<SUP id="cite_ref-pcfred_24-2" class="reference"><A href="#cite_note-pcfred-24">[24]</A></SUP> The same study also identified a separate urine based 3-miRNA model that could determine the likelihood of prostate cancer recurrence after surgery, which could help inform a therapeutic decision.
</P></DIV></DIV><P>Other cancers with miRNA biomarkers include: oral cancer (saliva) <SUP id="cite_ref-30" class="reference"><A href="#cite_note-30">[30]</A></SUP>, while bladder cancer (urine)<SUP id="cite_ref-31" class="reference"><A href="#cite_note-31">[31]</A></SUP> and cervical cancer (serum).<SUP id="cite_ref-32" class="reference"><A href="#cite_note-32">[32]</A></SUP> The possibilities continue to grow as research continues.
</P><P>MiRNAs have also shown to be potential biomarkers for the diagnosis of other diseases. For instance they have been shown to be indicative of drug induced liver injury (DILI). It is serious clinical problem as liver failure  can present suddenly, without prior symptoms. More sensitive and reliable blood-based biomarkers are needed for rapid detection (i.e. before liver failure).<SUP id="cite_ref-dilimirna_33-0" class="reference"><A href="#cite_note-dilimirna-33">[33]</A></SUP> Acetaminophen (paracetamol) is the most common cause of DILI. <SUP id="cite_ref-34" class="reference"><A href="#cite_note-34">[34]</A></SUP> There is a highly effective antidote, but there can be difficulties in achieving the diagnosis that warrants it. For instance those who have DILI despite having taken the recommended dose of paracetamol would not normally be considered for the antidote. Prescribing the antidote in more potential cases of DILI results in significant costs and unnecessary unpleasant side-effects. Effective biomarkers are needed to assess whether treatment is required.<SUP id="cite_ref-35" class="reference"><A href="#cite_note-35">[35]</A></SUP> MiRNAs miR-122 and 192 were shown to be a more sensitive measure of DILI than levels of ALT, the enzyme currently used. Elevated ALT levels may be due to other factors (such as muscle inflammation or myopathies), whereas elevated levels of miRNAs are specific to acetaminophen overdose induced liver injury.<SUP id="cite_ref-dilimirna_33-1" class="reference"><A href="#cite_note-dilimirna-33">[33]</A></SUP> Therefore miRNAs again show great promise as future biomarkers to aid in diagnosis of DILI.
</P><P>MiRNAs as biomarkers are also applicable to other conditions from myocardial infarction<SUP id="cite_ref-36" class="reference"><A href="#cite_note-36">[36]</A></SUP> to pregnancy<SUP id="cite_ref-37" class="reference"><A href="#cite_note-37">[37]</A></SUP> and associated complications like pre-eclampsia.<SUP id="cite_ref-38" class="reference"><A href="#cite_note-38">[38]</A></SUP></P><P>The specificity of miRNAs is key - their restricted expression profiles gives them great potential to be used as biomarkers for specific tissue or stage specific diseases.<SUP id="cite_ref-etheridge_13-1" class="reference"><A href="#cite_note-etheridge-13">[13]</A></SUP> However, it is important to note further studies with larger patient cohorts are needed for clinical tests to be developed for the examples mentioned here.
</P><P>Our method of quantifying miRNAs cheaply and effectively has the potential to diagnose and screen for a myriad of diseases. Our sensor could save millions of lives through the early detection of cancer and helping diagnose other diseases.
</P><DIV class="image_container"><DIV><H2 id="Applications in developing countries">Applications in developing countries</H2><P>Professor Sam Janes, Professor of Respiratory Medicine at UCL, said that for less economically developed countries, our test could be the ‘the biggest life saving intervention on the field.’ Dr Daniel Pregibon also emphasized this could be one of the main applications of our project.
</P><P>Aapproximately 58% of lung cancer cases in 2012 occurred in less developed countries.<SUP id="cite_ref-39" class="reference"><A href="#cite_note-39">[39]</A></SUP> Solid household fuels for cooking and heating are prevalent in one third of South African households and a significant proportion of (8.7% to 34.6%) males in Benin, Malawi, Mozambique, Niger, Sierra Leone and Swaziland are moderate smokers.<SUP id="cite_ref-40" class="reference"><A href="#cite_note-40">[40]</A></SUP></P></DIV><DIV class="image_holder" style="margin: 36px 0 0 2%; width: 28%"><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Lindsay Fox, <A style="color: #888" href="https://creativecommons.org/licenses/by/2.0/" target="_blank" rel="noopener">© CC BY 2.0</A></P></DIV></DIV><P>The low cost of our sensor, combined with its relative simplicity (specialists are not required to interpret the results) means it could complement emerging healthcare systems in these less developed countries. Cervical cancer screening programs have been beneficial in developed countries. The World Health Organization, state the main reasons why screening tests aren’t implemented in less developed countries are (the same reasons would be true for NSCLC):<SUP id="cite_ref-ccaner_41-0" class="reference"><A href="#cite_note-ccaner-41">[41]</A></SUP></P><UL><LI> The various competing health needs such as HIV/Aids in Sub-Saharan Africa, communicable diseases and maternal or perinatal complications</LI><LI> A lack of developed infrastructure for traditional diagnosis after screening such as high-quality cytology labs, transportation of samples to be observed by a specialist or methods of processing results</LI><LI> A large urban-rural development divide means that although treatment options could be available in cities for the majority of the population it is too far away to be accessed easily</LI></UL><P>Our tool overcomes these problems.
</P><P>Our test is cheap and it doesn’t require high-quality cytology labs or complex infrastructure, or trained personnel to interpret - currently interpreting cervical smears is extremely difficult and several years of expertise and education which is not always available in developing countries. This would allow our test to be used by governments with various competing health needs.
</P><P>It’s portability means that healthcare workers can reach out to rural communities with kits to carry out the screening and deliver results within hours. This means the large administrative task of carrying out any medical screening is streamlined for the less developed country. The early diagnosis provided by our test has the potential to reduce high cancer mortality rates.
</P><P>Furthermore, developing countries are likely to improve their healthcare provision and a cheap screening test for cancer would become increasingly important. As the population ages, cancers will be responsible for more deaths.
</P><P>Pre-invasive cervical lesions can currently be excised by the loop excision electrosurgical procedure, which can be carried out in an outpatient-setting, under local anesthetic. It has a cure rate of 80-95% with very minimal specialized equipment.
</P><P>Cervical cancer also has miRNA biomarkers.<SUP id="cite_ref-42" class="reference"><A href="#cite_note-42">[42]</A></SUP> For cervical cancer, the lack of a viable screening program, rather than a lack of treatment options or accessibility is a key to the high cancer mortality rates in these countries compared to more developed countries.<SUP id="cite_ref-ccaner_41-1" class="reference"><A href="#cite_note-ccaner-41">[41]</A></SUP></P><P>In the case of NSCLC treatment is generally palliative since it is rarely detected early enough for curative treatment. In less developed countries, palliative care is expensive and inaccessible since healthcare systems prioritise chemotherapeutic drugs to patients with a chance of survival.<SUP id="cite_ref-43" class="reference"><A href="#cite_note-43">[43]</A></SUP></P><P>Although there is no low cost, quick and relatively easily treatment to excise NSCLC tumors (patients generally have to undergo a lobectomy to remove part of the affected lung), the development of video-assisted thoracic surgery (VATS) for lobectomies could cut costs for tumor excision in the long-term.<SUP id="cite_ref-44" class="reference"><A href="#cite_note-44">[44]</A></SUP> This form of lobectomy uses cameras and surgical instruments used through very small incisions. Whilst it involves equipment that is more expensive to purchase initially and highly trained surgeons, the benefits of a lower chance of infection and a shorter hospital stay eventually outweigh this.
</P><P>Newly industrialising countries with widespread emerging healthcare systems like China would also benefit from our tool. Lung cancer, in large part down to high rates of smoking, caused 529,153 deaths in China in 2011.<SUP id="cite_ref-45" class="reference"><A href="#cite_note-45">[45]</A></SUP> In conjunction with anti-smoking policies governments could introduce, our screening test could be very effective.
</P><P>Dr Desai cited the epidemiological evidence that strongly suggests the smoking rate increases in these countries will increase the prevalence of NSCLC.
</P><P>Both China and India have fairly large isolated rural populations,<SUP id="cite_ref-46" class="reference"><A href="#cite_note-46">[46]</A></SUP> so again the portability of our test has benefits, and the low cost of our system is also obviously attractive to governments. These countries already have various treatment options and the capability to roll out administrative networks for processing screening results. The problem again lies with the fact that cancers like NSCLC are detected too late, which is why screening is so essential.
</P><DIV class="image_container"><DIV><H2 id="False positives">False positives</H2><P>High false positive rates and unnecessary further testing are major downsides to current screening methods. It can be debilitating to be given a diagnosis of cancer and it can have a major psychological impact. It is paramount that when the diagnosis is given, its is accurate - so false positive rates need to be minimized.
</P><P>Some of the common physiological impacts of a cancer diagnosis include anxiety, emotional upheaval, and depression. Between 20 and 50% of patients with a solid tumor are reported to have linked depression. <SUP id="cite_ref-47" class="reference"><A href="#cite_note-47">[47]</A></SUP></P><P>Misdiagnosing cancer also has negative economic impacts. It costs the NHS money to carry out all the scans, tests and procedures needed to treat someone diagnosed as positive. The average cost of the unnecessary tests for a false positive in a prostate, lung, colorectal and ovarian screening trial was over $1000 per patient.<SUP id="cite_ref-48" class="reference"><A href="#cite_note-48">[48]</A></SUP></P><P>MiRNA levels have the potential to produce false positives - even when several are quantified and the results combined. As such, it is unlikely that our tool would ever give a conclusive diagnosis of cancer. However, depending on the cutoff value chosen it could definitely rule out cancer in individuals. In others, the results could also strongly indicate the patient has a disease, but further testing would be needed to make the final diagnosis, stage a tumor, and inform therapeutic decisions.
</P></DIV><P style="text-align: right; font-size: 12px; color: #888;">Attribution: Graham Beards, <A style="color: #888" href="https://creativecommons.org/licenses/by-sa/3.0/" target="_blank" rel="noopener">© CC BY-SA 3.0</A></P></DIV><P>There are several issues associated with current miRNA measurement techniques that make these methods unsuitable for mass screening. The quantity and quality of miRNAs isolated from a biological sample is also variable and miRNA levels differ in serum and plasma (and may be affected by anticoagulants). Implementing a standard process for sample preparation is key to solving these problems as well as creating better reagent sets and investigating the differences in miRNA levels from different sources.<SUP id="cite_ref-etheridge_13-2" class="reference"><A href="#cite_note-etheridge-13">[13]</A></SUP></P><H2><SPAN class="mw-headline" id="Safety">Safety</SPAN></H2><P>We also considered the safety of all the different parts of our project. As a high school in the UK, making sure all aspects of our work were completely safe was central to our project. See our <A href="/Team:CLSB-UK/Safety" title="Team:CLSB-UK/Safety">safety page</A> for details.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">National Lung Screening Trial Research Team. (2011). Reduced lung-cancer mortality with low-dose computed tomographic screening. N Engl J Med, 2011(365), 395-409.</SPAN></LI><LI id="cite_note-gopal-2"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-gopal_2-0">2.0</A></SUP><SUP><A href="#cite_ref-gopal_2-1">2.1</A></SUP><SUP><A href="#cite_ref-gopal_2-2">2.2</A></SUP></SPAN><SPAN class="reference-text">Gopal, M., Abdullah, S. E., Grady, J. J., &amp; Goodwin, J. S. (2010). Screening for lung cancer with low-dose computed tomography: a systematic review and meta-analysis of the baseline findings of randomized controlled trials. Journal of thoracic oncology, 5(8), 1233-1239.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">(2015) Cancer Treatment &amp; Survivorship Facts &amp; Figures 2014-2015, American Cancer Society</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Esserman, L. J., Thompson, I. M., Reid, B., Nelson, P., Ransohoff, D. F., Welch, H. G., ... &amp; Bissell, M. (2014). Addressing overdiagnosis and overtreatment in cancer: a prescription for change. The lancet oncology, 15(6), e234-e242.</SPAN></LI><LI id="cite_note-dajac-5"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-dajac_5-0">5.0</A></SUP><SUP><A href="#cite_ref-dajac_5-1">5.1</A></SUP><SUP><A href="#cite_ref-dajac_5-2">5.2</A></SUP></SPAN><SPAN class="reference-text">Dajac, J., Kamdar, J., Moats, A., &amp; Nguyen, B. (2016). To Screen or not to Screen: Low Dose Computed Tomography in Comparison to Chest Radiography or Usual Care in Reducing Morbidity and Mortality from Lung Cancer. Cureus, 8(4).</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Esserman, L. J., Thompson, I. M., Reid, B., Nelson, P., Ransohoff, D. F., Welch, H. G., ... &amp; Bissell, M. (2014). Addressing overdiagnosis and overtreatment in cancer: a prescription for change. The lancet oncology, 15(6), e234-e242.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Welch, H. G., &amp; Black, W. C. (2010). Overdiagnosis in cancer. Journal of the National Cancer Institute, 102(9), 605-613.</SPAN></LI><LI id="cite_note-bianchi-8"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-bianchi_8-0">8.0</A></SUP><SUP><A href="#cite_ref-bianchi_8-1">8.1</A></SUP></SPAN><SPAN class="reference-text">Bianchi, F., Nicassio, F., Marzi, M., Belloni, E., Dall'Olio, V., Bernard, L., ... &amp; Di Fiore, P. P. (2011). A serum circulating miRNA diagnostic test to identify asymptomatic high‐risk individuals with early stage lung cancer. EMBO molecular medicine, 3(8), 495-503.</SPAN></LI><LI id="cite_note-lcnice-9"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-lcnice_9-0">9.0</A></SUP><SUP><A href="#cite_ref-lcnice_9-1">9.1</A></SUP></SPAN><SPAN class="reference-text">(2011, April 1). Lung cancer: diagnosis and management | Guidance and ... - NICE. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="https://www.nice.org.uk/guidance/cg121/chapter/1-guidance">https://www.nice.org.uk/guidance/cg121/chapter/1-guidance</A></SPAN></LI><LI id="cite_note-sputum-10"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-sputum_10-0">10.0</A></SUP><SUP><A href="#cite_ref-sputum_10-1">10.1</A></SUP></SPAN><SPAN class="reference-text">Kim, C. E., Tchou-Wong, K. M., &amp; Rom, W. N. (2011). Sputum-based molecular biomarkers for the early detection of lung cancer: limitations and promise. Cancers, 3(3), 2975-2989.</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">(n.d.). CT scan - NHS Choices. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="http://www.nhs.uk/conditions/CT-scan/Pages/Introduction.aspx">http://www.nhs.uk/conditions/CT-scan/Pages/Introduction.aspx</A></SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">(2008, September 4). Patient dose information: guidance - GOV.UK. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="https://www.gov.uk/government/publications/medical-radiation-patient-doses/patient-dose-information-guidance">https://www.gov.uk/government/publications/medical-radiation-patient-doses/patient-dose-information-guidance</A></SPAN></LI><LI id="cite_note-etheridge-13"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-etheridge_13-0">13.0</A></SUP><SUP><A href="#cite_ref-etheridge_13-1">13.1</A></SUP><SUP><A href="#cite_ref-etheridge_13-2">13.2</A></SUP></SPAN><SPAN class="reference-text">Etheridge, A., Lee, I., Hood, L., Galas, D., &amp; Wang, K. (2011). Extracellular microRNA: a new source of biomarkers. Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis, 717(1), 85-90.</SPAN></LI><LI id="cite_note-14"><SPAN class="mw-cite-backlink"><A href="#cite_ref-14">↑</A></SPAN><SPAN class="reference-text">(n.d.). miRNA Isolation Kit: miRNeasy Mini Kit - QIAGEN Online Shop. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="https://www.qiagen.com/gb/shop/sample-technologies/rna/mirna/mirneasy-mini-kit/">https://www.qiagen.com/gb/shop/sample-technologies/rna/mirna/mirneasy-mini-kit/</A></SPAN></LI><LI id="cite_note-15"><SPAN class="mw-cite-backlink"><A href="#cite_ref-15">↑</A></SPAN><SPAN class="reference-text">Benes, V., &amp; Castoldi, M. (2010). Expression profiling of microRNA using real-time quantitative PCR, how to use it and what is available. Methods, 50(4), 244-249.</SPAN></LI><LI id="cite_note-16"><SPAN class="mw-cite-backlink"><A href="#cite_ref-16">↑</A></SPAN><SPAN class="reference-text">Lu, J., Getz, G., Miska, E. A., Alvarez-Saavedra, E., Lamb, J., Peck, D., ... &amp; Downing, J. R. (2005). MicroRNA expression profiles classify human cancers. nature, 435(7043), 834-838.</SPAN></LI><LI id="cite_note-17"><SPAN class="mw-cite-backlink"><A href="#cite_ref-17">↑</A></SPAN><SPAN class="reference-text">Sato, F., Tsuchiya, S., Terasawa, K., &amp; Tsujimoto, G. (2009). Intra-platform repeatability and inter-platform comparability of microRNA microarray technology. PloS one, 4(5), e5540.</SPAN></LI><LI id="cite_note-pcslater-18"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-pcslater_18-0">18.0</A></SUP><SUP><A href="#cite_ref-pcslater_18-1">18.1</A></SUP></SPAN><SPAN class="reference-text">Slater, E. P., Strauch, K., Rospleszcz, S., Ramaswamy, A., Esposito, I., Klöppel, G., ... &amp; Bartsch, D. K. (2014). MicroRNA-196a and-196b as potential biomarkers for the early detection of familial pancreatic cancer. Translational oncology, 7(4), 464-471.</SPAN></LI><LI id="cite_note-19"><SPAN class="mw-cite-backlink"><A href="#cite_ref-19">↑</A></SPAN><SPAN class="reference-text">(n.d.). Pancreatic cancer screening | Pancreatic Cancer UK. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="https://www.pancreaticcancer.org.uk/information-and-support/facts-about-pancreatic-cancer/screening-for-pancreatic-cancer/">https://www.pancreaticcancer.org.uk/information-and-support/facts-about-pancreatic-cancer/screening-for-pancreatic-cancer/</A></SPAN></LI><LI id="cite_note-20"><SPAN class="mw-cite-backlink"><A href="#cite_ref-20">↑</A></SPAN><SPAN class="reference-text">(2016, June 24). Pancreatic cancer - NHS Choices. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="http://www.nhs.uk/conditions/Cancer-of-the-pancreas/Pages/Introduction.aspx">http://www.nhs.uk/conditions/Cancer-of-the-pancreas/Pages/Introduction.aspx</A></SPAN></LI><LI id="cite_note-21"><SPAN class="mw-cite-backlink"><A href="#cite_ref-21">↑</A></SPAN><SPAN class="reference-text">Amikura, K., Kobari, M., &amp; Matsuno, S. (1995). The time of occurrence of liver metastasis in carcinoma of the pancreas. International Journal of Gastrointestinal Cancer, 17(2), 139-146.</SPAN></LI><LI id="cite_note-22"><SPAN class="mw-cite-backlink"><A href="#cite_ref-22">↑</A></SPAN><SPAN class="reference-text">Kayahara, M., Nagakawa, T., Ueno, K., Ohta, T., Takeda, T., &amp; Miyazaki, I. (1993). An evaluation of radical resection for pancreatic cancer based on the mode of recurrence as determined by autopsy and diagnostic imaging. Cancer, 72(7), 2118-2123.</SPAN></LI><LI id="cite_note-23"><SPAN class="mw-cite-backlink"><A href="#cite_ref-23">↑</A></SPAN><SPAN class="reference-text">Richter, A., Niedergethmann, M., Sturm, J. W., Lorenz, D., Post, S., &amp; Trede, M. (2003). Long-term results of partial pancreaticoduodenectomy for ductal adenocarcinoma of the pancreatic head: 25-year experience. World journal of surgery, 27(3), 324-329.</SPAN></LI><LI id="cite_note-pcfred-24"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-pcfred_24-0">24.0</A></SUP><SUP><A href="#cite_ref-pcfred_24-1">24.1</A></SUP><SUP><A href="#cite_ref-pcfred_24-2">24.2</A></SUP></SPAN><SPAN class="reference-text">Fredsøe, J., Rasmussen, A. K., Thomsen, A. R., Mouritzen, P., Høyer, S., Borre, M., ... &amp; Sørensen, K. D. (2017). Diagnostic and Prognostic MicroRNA Biomarkers for Prostate Cancer in Cell-free Urine. European Urology Focus.</SPAN></LI><LI id="cite_note-25"><SPAN class="mw-cite-backlink"><A href="#cite_ref-25">↑</A></SPAN><SPAN class="reference-text">Bannister, N., &amp; Broggio, J. (2016). Cancer survival by stage at diagnosis for England (experimental statistics): Adults diagnosed 2012, 2013 and 2014 and followed up to 2015.</SPAN></LI><LI id="cite_note-26"><SPAN class="mw-cite-backlink"><A href="#cite_ref-26">↑</A></SPAN><SPAN class="reference-text">(n.d.). Screening | Prostate cancer | Cancer Research UK. Retrieved October 25, 2017, from <A rel="nofollow" class="external free" href="http://www.cancerresearchuk.org/about-cancer/prostate-cancer/getting-diagnosed/screening">http://www.cancerresearchuk.org/about-cancer/prostate-cancer/getting-diagnosed/screening</A></SPAN></LI><LI id="cite_note-27"><SPAN class="mw-cite-backlink"><A href="#cite_ref-27">↑</A></SPAN><SPAN class="reference-text">Sandblom, G., Varenhorst, E., Rosell, J., Löfman, O., &amp; Carlsson, P. (2011). Randomised prostate cancer screening trial: 20 year follow-up. Bmj, 342, d1539.</SPAN></LI><LI id="cite_note-28"><SPAN class="mw-cite-backlink"><A href="#cite_ref-28">↑</A></SPAN><SPAN class="reference-text">Schröder, F. H., Hugosson, J., Roobol, M. J., Tammela, T. L., Zappa, M., Nelen, V., ... &amp; Denis, L. J. (2014). Screening and prostate cancer mortality: results of the European Randomised Study of Screening for Prostate Cancer (ERSPC) at 13 years of follow-up. The Lancet, 384(9959), 2027-2035.</SPAN></LI><LI id="cite_note-29"><SPAN class="mw-cite-backlink"><A href="#cite_ref-29">↑</A></SPAN><SPAN class="reference-text">Velonas, V. M., Woo, H. H., Remedios, C. G. D., &amp; Assinder, S. J. (2013). Current status of biomarkers for prostate cancer. International journal of molecular sciences, 14(6), 11034-11060.</SPAN></LI><LI id="cite_note-30"><SPAN class="mw-cite-backlink"><A href="#cite_ref-30">↑</A></SPAN><SPAN class="reference-text"> Park, N. J., Zhou, H., Elashoff, D., Henson, B. S., Kastratovic, D. A., Abemayor, E., &amp; Wong, D. T. (2009). Salivary microRNA: discovery, characterization, and clinical utility for oral cancer detection. Clinical Cancer Research, 15(17), 5473-5477.</SPAN></LI><LI id="cite_note-31"><SPAN class="mw-cite-backlink"><A href="#cite_ref-31">↑</A></SPAN><SPAN class="reference-text">Hanke, M., Hoefig, K., Merz, H., Feller, A. C., Kausch, I., Jocham, D., ... &amp; Sczakiel, G. (2010, December). A robust methodology to study urine microRNA as tumor marker: microRNA-126 and microRNA-182 are related to urinary bladder cancer. In Urologic Oncology: Seminars and Original Investigations (Vol. 28, No. 6, pp. 655-661). Elsevier.</SPAN></LI><LI id="cite_note-32"><SPAN class="mw-cite-backlink"><A href="#cite_ref-32">↑</A></SPAN><SPAN class="reference-text">Ma, Q., Wan, G., Wang, S., Yang, W., Zhang, J., &amp; Yao, X. (2014). Serum microRNA-205 as a novel biomarker for cervical cancer patients. Cancer cell international, 14(1), 81.</SPAN></LI><LI id="cite_note-dilimirna-33"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-dilimirna_33-0">33.0</A></SUP><SUP><A href="#cite_ref-dilimirna_33-1">33.1</A></SUP></SPAN><SPAN class="reference-text">Wang, K., Zhang, S., Marzolf, B., Troisch, P., Brightman, A., Hu, Z., ... &amp; Galas, D. J. (2009). Circulating microRNAs, potential biomarkers for drug-induced liver injury. Proceedings of the National Academy of Sciences, 106(11), 4402-4407.</SPAN></LI><LI id="cite_note-34"><SPAN class="mw-cite-backlink"><A href="#cite_ref-34">↑</A></SPAN><SPAN class="reference-text"> Leise, M. D., Poterucha, J. J., &amp; Talwalkar, J. A. (2014, January). Drug-induced liver injury. In Mayo clinic proceedings (Vol. 89, No. 1, pp. 95-106). Elsevier.</SPAN></LI><LI id="cite_note-35"><SPAN class="mw-cite-backlink"><A href="#cite_ref-35">↑</A></SPAN><SPAN class="reference-text">Bateman, D. N., Carroll, R., Pettie, J., Yamamoto, T., Elamin, M. E., Peart, L., ... &amp; Sandilands, E. A. (2014). Effect of the UK's revised paracetamol poisoning management guidelines on admissions, adverse reactions and costs of treatment. British journal of clinical pharmacology, 78(3), 610-618.</SPAN></LI><LI id="cite_note-36"><SPAN class="mw-cite-backlink"><A href="#cite_ref-36">↑</A></SPAN><SPAN class="reference-text">Adachi, T., Nakanishi, M., Otsuka, Y., Nishimura, K., Hirokawa, G., Goto, Y., ... &amp; Iwai, N. (2010). Plasma microRNA 499 as a biomarker of acute myocardial infarction. Clinical chemistry, 56(7), 1183-1185.</SPAN></LI><LI id="cite_note-37"><SPAN class="mw-cite-backlink"><A href="#cite_ref-37">↑</A></SPAN><SPAN class="reference-text">Gilad, S., Meiri, E., Yogev, Y., Benjamin, S., Lebanony, D., Yerushalmi, N., ... &amp; Bentwich, Z. (2008). Serum microRNAs are promising novel biomarkers. PloS one, 3(9), e3148.</SPAN></LI><LI id="cite_note-38"><SPAN class="mw-cite-backlink"><A href="#cite_ref-38">↑</A></SPAN><SPAN class="reference-text">Pineles, B. L., Romero, R., Montenegro, D., Tarca, A. L., Han, Y. M., Kim, Y. M., ... &amp; Hassan, S. S. (2007). Distinct subsets of microRNAs are expressed differentially in the human placentas of patients with preeclampsia. American journal of obstetrics and gynecology, 196(3), 261-e1.</SPAN></LI><LI id="cite_note-39"><SPAN class="mw-cite-backlink"><A href="#cite_ref-39">↑</A></SPAN><SPAN class="reference-text">(n.d.). Lung cancer statistics | World Cancer Research Fund International. Retrieved October 6, 2017, from <A rel="nofollow" class="external free" href="http://www.wcrf.org/int/cancer-facts-figures/data-specific-cancers/lung-cancer-statistics">http://www.wcrf.org/int/cancer-facts-figures/data-specific-cancers/lung-cancer-statistics</A></SPAN></LI><LI id="cite_note-40"><SPAN class="mw-cite-backlink"><A href="#cite_ref-40">↑</A></SPAN><SPAN class="reference-text">Urman, A., Josyula, S., Rosenberg, A., Lounsbury, D., &amp; Rohan, T. (2016). Burden of lung cancer and associated risk factors in Africa by region. J Pulm Respir Med, 6(340), 2.</SPAN></LI><LI id="cite_note-ccaner-41"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-ccaner_41-0">41.0</A></SUP><SUP><A href="#cite_ref-ccaner_41-1">41.1</A></SUP></SPAN><SPAN class="reference-text">Denny, L., Quinn, M., &amp; Sankaranarayanan, R. (2006). Screening for cervical cancer in developing countries. Vaccine, 24, S71-S77.</SPAN></LI><LI id="cite_note-42"><SPAN class="mw-cite-backlink"><A href="#cite_ref-42">↑</A></SPAN><SPAN class="reference-text">Ma, Q., Wan, G., Wang, S., Yang, W., Zhang, J., &amp; Yao, X. (2014). Serum microRNA-205 as a novel biomarker for cervical cancer patients. Cancer cell international, 14(1), 81.</SPAN></LI><LI id="cite_note-43"><SPAN class="mw-cite-backlink"><A href="#cite_ref-43">↑</A></SPAN><SPAN class="reference-text">Ezemba, N., Ekpe, E. E., &amp; Eze, J. C. (2012). Challenges of lung cancer management in a developing country. Nigerian journal of medicine: journal of the National Association of Resident Doctors of Nigeria, 21(2), 214-217.</SPAN></LI><LI id="cite_note-44"><SPAN class="mw-cite-backlink"><A href="#cite_ref-44">↑</A></SPAN><SPAN class="reference-text">Lacin, T., &amp; Swanson, S. (2013). Current costs of video-assisted thoracic surgery (VATS) lobectomy. Journal of thoracic disease, 5(Suppl 3), S190.</SPAN></LI><LI id="cite_note-45"><SPAN class="mw-cite-backlink"><A href="#cite_ref-45">↑</A></SPAN><SPAN class="reference-text">Zheng, R., Zeng, H., Zuo, T., Zhang, S., Qiao, Y., Zhou, Q., &amp; Chen, W. (2016). Lung cancer incidence and mortality in China, 2011. Thoracic cancer, 7(1), 94-99.</SPAN></LI><LI id="cite_note-46"><SPAN class="mw-cite-backlink"><A href="#cite_ref-46">↑</A></SPAN><SPAN class="reference-text">The World Bank (2012) World Development Indicators. Retrieved October 6, 2017, from <A rel="nofollow" class="external free" href="https://data.worldbank.org/indicator/SP.RUR.TOTL.ZS?locations=IN-CN">https://data.worldbank.org/indicator/SP.RUR.TOTL.ZS?locations=IN-CN</A></SPAN></LI><LI id="cite_note-47"><SPAN class="mw-cite-backlink"><A href="#cite_ref-47">↑</A></SPAN><SPAN class="reference-text">Pasquini, M., &amp; Biondi, M. (2007). Depression in cancer patients: a critical review. Clinical Practice and epidemiology in mental health, 3(1), 2.</SPAN></LI><LI id="cite_note-48"><SPAN class="mw-cite-backlink"><A href="#cite_ref-48">↑</A></SPAN><SPAN class="reference-text">Lafata, J. E., Simpkins, J., Lamerato, L., Poisson, L., Divine, G., &amp; Johnson, C. C. (2004). The economic impact of false-positive cancer screens. Cancer Epidemiology and Prevention Biomarkers, 13(12), 2126-2132.</SPAN></LI></OL></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>