Team:CLSB-UK
CLSB-UK
Project
Design
Parts
Demonstrate
Judging
Contribution
Lab work
Results
Experiments
Safety
Notebook
InterLab
Human Practices
Silver
Engagement
High School Guide
Gold Integrated
Safety
Cost model
Modeling
NUPACK model
Mass action kinetics
Stochastic model
Cost model
Software
Overview
Toehold tools
SynBioBot
Hardware
Overview
Interpreter
Measurement
People
Team
Collaborations
Attributions
Sponsors
We strongly recommend using Google Chrome or Mozilla Firefox to view our wiki.
Project BATMAN
a new way to detect cancer using toehold switches
Late presentation and non-specific symptoms are the main
reasons 1.6 million people worldwide die from lung cancer every year
Dr Sujal Desai, Consultant Chest Radiologist
We have developed a new way to detect cancer at an early stage by measuring micro-RNAs (miRNAs), biomarkers found in blood. We use toehold switches to regulate expression of GFP in response to specific miRNAs. This method could be applied to a myriad of diseases, but we have chosen to use non-small cell lung cancer (NSCLC) as a proof of concept. We hope our work in NSCLC detection demonstrates the potential that toehold switches have to offer as a cheap and effective diagnostic tool.
Biomarkers in the blood
Abnormal levels of miRNAs miR-15b-5p and miR-27b-3p in blood serum are indicative of NSCLC. [1]
We designed two sequence-specific sensors that use synthetic riboregulators called toehold switches to detect miR-15b-5p and miR-27b-3p. They work by regulating the expression of fluorescent reporter proteins in the presence of these specific miRNAs. Our sensors work in a cell-free system, allowing them to be used safely and in a low-tech environment.
Non-small cell lung cancer
Lung cancer is the most common cause of cancer-related mortality, with 1.6 million deaths in 2012. That’s 20% of all reported deaths due to cancer. Non-small cell lung cancer (NSCLC) makes up ~80% of all incidences of lung cancer. [2] 58% of all cases in 2012 were reported in less developed countries. [3]
NSCLC is aggressive and pathologically diverse. [4] NSCLC commonly presents with a cough, haemoptysis and the symptoms of metastatic disease such as bone pain or jaundice. Only a minority of patients present with operable tumors; in most cases patients have advanced stage cancer and treatment is with palliative intent. [5] Treatment centers around chemotherapy and radiotherapy, but new treatments show promise including immunotherapies. [6] [2]
About 90% of lung cancers are caused by smoking and as smoking rates have declined, there has been a corresponding reduction in the incidence of lung cancers. [7] However, nearly 30% of the global population are still estimated to smoke. [8]
Contents
1 Late diagnosis kills
2 Toehold switches
2.1 Our project
2.2 References
Late diagnosis kills
IA
IB
IIA
IIB
IIIA
IIIB
IV
0
10
20
30
40
50
60
5 year NSCLC survival rates by stage
Stage Diagnosed
Survival rate (%)
Graph data from the American Cancer Society. [9]
NSCLC’s high mortality rate is largely down to the late stage at which it’s normally diagnosed. [10] After metastasis, any treatment therefore is with palliative intent. However, surgery can be curative in early stages. [11] [12]
MicroRNA (miRNA)
MicroRNAs (miRNAs) can act as potent biomarkers for a myriad of diseases.
MiRNAs are short non-coding RNAs of 19-24 nucleotides in length. They are involved in regulating post-transcriptional gene expression in eukaryotes. [13] [14] [15] MiRNAs can be found extracellularly in various body fluids, including serum [16] [1] , plasma [17] , saliva [18] , urine [19] and breast milk [20] .
It has been hypothesized that cells actively secrete miRNAs in microvesicles or bound to RNA binding proteins. [21] These shield the circulating miRNAs from ribonucleases [21] and are capable of delivering miRNAs to recipient cells. [22] [23] [20] These miRNAs can then trigger  signaling events, indicating that circulating miRNAs play a role in cell-to-cell communication. [23] [20] [24]
Recent studies have shown that differential levels of miRNAs in body fluids are indicative of specific diseases, including many forms of cancer. [1] [17] [19] [25] MiRNA stability, accessibility by non-invasive methods (liquid biopsy) and their ability to diagnose diseases in early stages give circulating miRNAs the potential to be effective biomarkers. [26]
Currently, microarrays and qPCR are typically used to quantify circulating miRNAs. However, qPCR is labour intensive, whilst microarrays suffer from cross-hybridisation and inconsistent results. [27] [28]
Most current biomarkers are proteins, however there are numerous challenges to identifying new protein markers. These include the difficulties in developing suitably specific detection agents, the variety of post-translational modifications and the lack of proteins of interest in the blood.
MiRNAs are stable, tissue or stage specific biomarkers which have great potential for effective clinical use. [26]
Toehold switches
Toehold switches are synthetic riboregulators that control gene expression post transcriptionally. They are hairpin loops found at the 5’ end of messenger RNA that can switch between two configurations - one which promotes and one which represses translation based on the presence or absence of a trigger RNA. Toehold switches have much greater possible dynamic ranges than previous riboregulators.
Initially the start codon is surrounded by a strong secondary structure - the hairpin stem. As the ribosome cannot melt the stem, translation of a fluorescent reporter protein coding sequence is repressed.
Play animation
When a trigger RNA binds to the toehold region (a linear sequence at the 5’ end of the switch) it undergoes a branch migration with the lower part of the stem. This relieves the secondary structure around the start codon. The ribosome bound to the RBS can then melt the remaining stem
Next
This activates translation and fluorescence from the reporter protein can be observed.
Reset
We used toehold switches to detect miRNAs, as they have very few sequence constraints so an arbitrary RNA sequence can act as the trigger RNA. As such, toehold switch mediated miRNA detection could act as a ubiquitous mechanism, capable of detecting any miRNA sequence for any disease.
Previous riboregulators have relied on base pairing to the RBS to prevent ribosome binding. [29] [30] [31] For these, the trigger RNA must contain the RBS sequence to displace the sequence bound to the RBS, dramatically reducing programmability of the trigger RNA. However, the trigger RNA for a toehold switch has very few sequence constraints due to the lack of base pairing to the RBS and start codon (as strand displacement does not occur in the regions which have fixed sequences).
The large dynamic range offered by toehold switches allows for low miRNAs concentrations to be detected and for discrimination of small differences in miRNA concentration.
In previous riboregulators trigger RNA binding is initiated in the short loop domain of the hairpin module. [29] [30] [32] Subsequently, the binding thermodynamics in previous riboregulators are not as favourable as the toehold switch’s because toehold switches have longer linear trigger binding regions. This increases the probability of spontaneous trigger RNA dissociation and reduces the likelihood of successful strand displacement of the RBS from the incumbent sequence bound to it in previous riboregulators. [33] The dynamic range of previous riboregulators has therefore been limited to a maximum of 55 fold [29] , whereas toehold switches display a mean dynamic range of over 400 fold. [34]
Clinical implementation
For NSCLC, miRNAs extracted from blood serum would be added to a cell free system containing an excess of toehold switches. These toehold switches detect a specific miRNA and regulate translation of GFP in response. The intensity of fluorescence produced by our circuits is therefore indicative of the miRNA levels in the body fluid.
We’ve looked in great detail into how our sensor would be used in a clinical setting as part of our human practices work . We spoke to several doctors as part of our integrated human practices and developed a strategy for how our test could be used in screening programmes for NSCLC. Additionally, we have improved the specificity of our sensor on their advice. We have also identified other diseases which our sensor could potentially detect and researched how our method would be beneficial.
Our cost analysis shows our system would cost less than £20 per test. To prove screening with our test would be cost-effective, we created a cost model . The cost model takes into account the probabilities of different risk groups having cancer, the test’s false positive and negative rates, and the cost of treatment.
To help reduce costs, we designed and built a £4 combined fluorometer and densitometer to cheaply quantify the fluorescence from our circuit without a plate reader. This allows our test to be used in the field and in less developed countries. To complement the hardware we developed an application to interpret the readings, so the device can be used on most laptops, tablets and even phones.
Our project
Oh, and for those curious, Project BATMAN stands for Biosynthetic Applications of Toehold switches, MiRNA And NSCLC. And yes, we know the ‘A’ stands for ‘and’, while the ‘N’ stands for ‘non-small cell lung cancer’, but we were really determined to shoehorn in an acronym.
To learn more about the different aspects of our project check out the links below.
Design
Results
Human Practices
Modeling
Software
Hardware
References
To judges: we strongly recommend you read the key papers on our judging page first to save time.
↑ 1.0 1.1 1.2 Hennessey, P. T., Sanford, T., Choudhary, A., Mydlarz, W. W., Brown, D., Adai, A. T., ... & Califano, J. A. (2012). Serum microRNA biomarkers for detection of non-small cell lung cancer. PloS one, 7(2), e32307.
↑ 2.0 2.1 Chan, B. A., & Hughes, B. G. (2015). Targeted therapy for non-small cell lung cancer: current standards and the promise of the future. Translational lung cancer research, 4(1), 36.
↑ Ferlay, J., Soerjomataram, I., & Ervik, M. (2012). GLOBOCAN, cancer incidence and mortality worldwide: IARC cancer base no. 11 [Internet]. Lyon, France: International Agency for Research on Cancer; 2013.
↑ Board,  P. A. T. E. (2017). Non-Small Cell Lung Cancer Treatment (PDQ®).
↑ Bower, M., & Waxman, J. (2010). Oncology: lecture notes. Wiley-Blackwell.
↑ Chen, Z., Fillmore, C. M., Hammerman, P. S., Kim, C. F., & Wong, K. K. (2014). Non-small-cell lung cancers: a heterogeneous set of diseases. Nature reviews. Cancer, 14(8), 535.
↑ (n.d.). Lung cancer: diagnosis and management - NICE. Retrieved October 7, 2017, from https://www.nice.org.uk/guidance/cg121/chapter/introduction
↑ Gopal, M., Abdullah, S. E., Grady, J. J., & Goodwin, J. S. (2010). Screening for lung cancer with low-dose computed tomography: a systematic review and meta-analysis of the baseline findings of randomized controlled trials. Journal of thoracic oncology, 5(8), 1233-1239.
↑ (2015) Cancer Treatment & Survivorship Facts & Figures 2014-2015, American Cancer Society
↑ Dajac, J., Kamdar, J., Moats, A., & Nguyen, B. (2016). To Screen or not to Screen: Low Dose Computed Tomography in Comparison to Chest Radiography or Usual Care in Reducing Morbidity and Mortality from Lung Cancer. Cureus, 8(4).
↑ Uramoto, H., & Tanaka, F. (2014). Recurrence after surgery in patients with NSCLC. Translational lung cancer research, 3(4), 242.
↑ Molina, J. R., Yang, P., Cassivi, S. D., Schild, S. E., & Adjei, A. A. (2008, May). Non-small cell lung cancer: epidemiology, risk factors, treatment, and survivorship. In Mayo Clinic Proceedings(Vol. 83, No. 5, pp. 584-594). Elsevier.
↑ Bartel, D. P. (2004). MicroRNAs: genomics, biogenesis, mechanism, and function. cell, 116(2), 281-297.
↑ Ambros, V. (2004). The functions of animal microRNAs Nature 431: 350–355. Proceedings of the National Academy of Sciences of the United States of America, 103, 3687-3692.
↑ Kim, V. N., Han, J., & Siomi, M. C. (2009). Biogenesis of small RNAs in animals. Nature reviews Molecular cell biology, 10(2), 126-139.
↑ Zen, K., & Zhang, C. Y. (2012). Circulating microRNAs: a novel class of biomarkers to diagnose and monitor human cancers. Medicinal research reviews, 32(2), 326-348.
↑ 17.0 17.1 Mitchell, P. S., Parkin, R. K., Kroh, E. M., Fritz, B. R., Wyman, S. K., Pogosova-Agadjanyan, E. L., ... & Lin, D. W. (2008). Circulating microRNAs as stable blood-based markers for cancer detection. Proceedings of the National Academy of Sciences, 105(30), 10513-10518.
↑ Park, N. J., Zhou, H., Elashoff, D., Henson, B. S., Kastratovic, D. A., Abemayor, E., & Wong, D. T. (2009). Salivary microRNA: discovery, characterization, and clinical utility for oral cancer detection. Clinical Cancer Research, 15(17), 5473-5477.
↑ 19.0 19.1 Hanke, M., Hoefig, K., Merz, H., Feller, A. C., Kausch, I., Jocham, D., ... & Sczakiel, G. (2010, December). A robust methodology to study urine microRNA as tumor marker: microRNA-126 and microRNA-182 are related to urinary bladder cancer. In Urologic Oncology: Seminars and Original Investigations (Vol. 28, No. 6, pp. 655-661). Elsevier.
↑ 20.0 20.1 20.2 Kosaka, N., Izumi, H., Sekine, K., & Ochiya, T. (2010). microRNA as a new immune-regulatory agent in breast milk. Silence, 1(1), 7.
↑ 21.0 21.1 Chen, X., Liang, H., Zhang, J., Zen, K., & Zhang, C. Y. (2012). Secreted microRNAs: a new form of intercellular communication. Trends in cell biology, 22(3), 125-132.
↑ Vickers, K. C., Palmisano, B. T., Shoucri, B. M., Shamburek, R. D., & Remaley, A. T. (2011). MicroRNAs are transported in plasma and delivered to recipient cells by high-density lipoproteins. Nature cell biology, 13(4), 423.
↑ 23.0 23.1 Valadi, H., Ekström, K., Bossios, A., Sjöstrand, M., Lee, J. J., & Lötvall, J. O. (2007). Exosome-mediated transfer of mRNAs and microRNAs is a novel mechanism of genetic exchange between cells. Nature cell biology, 9(6), 654.
↑ Turchinovich, A., Samatov, T. R., Tonevitsky, A. G., & Burwinkel, B. (2013). Circulating miRNAs: cell–cell communication function?. Frontiers in genetics, 4.
↑ Slater, E. P., Strauch, K., Rospleszcz, S., Ramaswamy, A., Esposito, I., Klöppel, G., ... & Bartsch, D. K. (2014). MicroRNA-196a and-196b as potential biomarkers for the early detection of familial pancreatic cancer. Translational oncology, 7(4), 464-471.
↑ 26.0 26.1 Etheridge, A., Lee, I., Hood, L., Galas, D., & Wang, K. (2011). Extracellular microRNA: a new source of biomarkers. Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis, 717(1), 85-90.
↑ Lu, J., Getz, G., Miska, E. A., Alvarez-Saavedra, E., Lamb, J., Peck, D., ... & Downing, J. R. (2005). MicroRNA expression profiles classify human cancers. nature, 435(7043), 834-838.
↑ Sato, F., Tsuchiya, S., Terasawa, K., & Tsujimoto, G. (2009). Intra-platform repeatability and inter-platform comparability of microRNA microarray technology. PloS one, 4(5), e5540.
↑ 29.0 29.1 29.2 Callura, J. M., Cantor, C. R., & Collins, J. J. (2012). Genetic switchboard for synthetic biology applications. Proceedings of the National Academy of Sciences, 109(15), 5850-5855.
↑ 30.0 30.1 Isaacs, F. J., Dwyer, D. J., Ding, C., Pervouchine, D. D., Cantor, C. R., & Collins, J. J. (2004). Engineered riboregulators enable post-transcriptional control of gene expression. Nature biotechnology, 22(7), 841.
↑ Rodrigo, G., Landrain, T. E., & Jaramillo, A. (2012). De novo automated design of small RNA circuits for engineering synthetic riboregulation in living cells. Proceedings of the National Academy of Sciences, 109(38), 15271-15276.
↑ Takahashi, M. K., & Lucks, J. B. (2013). A modular strategy for engineering orthogonal chimeric RNA transcription regulators. Nucleic acids research, 41(15), 7577-7588.
↑ Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., & Turberfield, A. J. (2014). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324.
↑ Green, A. A., Silver, P. A., Collins, J. J., & Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell, 159(4), 925-939.
Major sponsors
Mr John
Mendez
Contact us
