<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CLSB-UK skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CLSB-UK</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><A href="/Team:CLSB-UK"><H1>CLSB-UK</H1></A><UL class="nav"><LI class="dropdown"><A href="/Team:CLSB-UK">Project</A></LI><LI><A href="/Team:CLSB-UK/Design">Design</A></LI><LI><A href="/Team:CLSB-UK/Parts">Parts</A></LI><LI><A href="/Team:CLSB-UK/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:CLSB-UK/Judging">Judging</A></LI><LI><A href="/Team:CLSB-UK/Contribution">Contribution</A></LI></UL><LI class="dropdown"><A href="/Team:CLSB-UK/Notebook">Lab work</A></LI><LI><A href="/Team:CLSB-UK/Results">Results</A></LI><LI><A href="/Team:CLSB-UK/Experiments">Experiments</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Notebook">Notebook</A></LI><LI><A href="/Team:CLSB-UK/InterLab">InterLab</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/HP">Human Practices</A></LI><LI><A href="/Team:CLSB-UK/HP/Silver">Silver</A></LI><LI><A href="/Team:CLSB-UK/Engagement">Engagement</A></LI><LI><A href="/Team:CLSB-UK/HP/HS_Guide">High School Guide</A></LI><LI><A href="/Team:CLSB-UK/HP/Gold_Integrated">Gold Integrated</A></LI><LI><A href="/Team:CLSB-UK/Safety">Safety</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Model#">Modeling</A></LI><LI><A href="/Team:CLSB-UK/Model#NUPACK">NUPACK model</A></LI><LI><A href="/Team:CLSB-UK/Model#MAK">Mass action kinetics</A></LI><LI><A href="/Team:CLSB-UK/Model#Stochastic">Stochastic model</A></LI><LI><A href="/Team:CLSB-UK/Model#Cost_model">Cost model</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Software">Software</A></LI><LI><A href="/Team:CLSB-UK/Software">Overview</A></LI><LI><A href="/Team:CLSB-UK/Software/ToeholdSwitchTools">Toehold tools</A></LI><LI><A href="/Team:CLSB-UK/Software/SynBioBot">SynBioBot</A></LI><LI class="dropdown"><A href="/Team:CLSB-UK/Hardware">Hardware</A></LI><LI><A href="/Team:CLSB-UK/Hardware">Overview</A></LI><LI><A href="/Team:CLSB-UK/Hardware/Interpreter">Interpreter</A></LI><LI><A href="/Team:CLSB-UK/Measurement">Measurement</A></LI><LI class="dropdown" style="position: relative"><A href="/Team:CLSB-UK/Team">People</A></LI><LI><A href="/Team:CLSB-UK/Team">Team</A></LI><LI><A href="/Team:CLSB-UK/Collaborations">Collaborations</A></LI><LI><A href="/Team:CLSB-UK/Attributions">Attributions</A></LI><LI><A href="/Team:CLSB-UK/Sponsors">Sponsors</A></LI></DIV><NAV id="useBetterBrowser" style="top: 66px;background-color: #ffda47;z-index: 1;display: none;height: 32px;"><DIV class="content_wrapper"><P style="font-size: 18px;margin: 7px 0;line-height: 1;text-align: center;">We strongly recommend using <A style="font-weight: 800; color: #07e;" href="https://www.google.com/chrome/" target="_blank" rel="noopener">Google Chrome</A> or <A style="font-weight: 800; color: #07e;" href="https://www.mozilla.org/en-US/firefox/new/" target="_blank" rel="noopener">Mozilla Firefox</A> to view our wiki.</P></DIV></NAV><DIV class="stripe_content content_wrapper" id="homepage_banner"><H1>Project BATMAN</H1><P>a new way to detect cancer using toehold switches</P></DIV><DIV class="content_wrapper" style="margin-top: 30px;"><P class="mobile_hide" style="text-align: right; font-weight: 800; font-size: 26px; margin-bottom: -8px">Late presentation and non-specific symptoms are the mainreasons 1.6 million people worldwide die from lung cancer every year</P><P class="mobile_hide" style="text-align: right; color: #999">Dr Sujal Desai, Consultant Chest Radiologist</P><P style="font-size: 16px; margin: 2em 0">We have developed a new way to detect cancer at an early stage by measuring micro-RNAs (miRNAs), biomarkers found in blood. We use toehold switches to regulate expression of GFP in response to specific miRNAs. This method could be applied to a myriad of diseases, but we have chosen to use non-small cell lung cancer (NSCLC) as a proof of concept. We hope our work in NSCLC detection demonstrates the potential that toehold switches have to offer as a cheap and effective diagnostic tool.</P><DIV class="image_container"><DIV><H2 id="Biomarkers in the blood">Biomarkers in the blood</H2><P>Abnormal levels of miRNAs miR-15b-5p and miR-27b-3p in blood serum are indicative of NSCLC.<SUP id="cite_ref-Hennessey_1-0" class="reference"><A href="#cite_note-Hennessey-1">[1]</A></SUP></P><P>We designed two sequence-specific sensors that use synthetic riboregulators called toehold switches to detect miR-15b-5p and miR-27b-3p. They work by regulating the expression of fluorescent reporter proteins in the presence of these specific miRNAs. Our sensors work in a cell-free system, allowing them to be used safely and in a low-tech environment.
</P></DIV></DIV></DIV><DIV class="stripe_content content_wrapper" style="clear: left; margin-bottom: 20px;"><DIV class="image_container"><DIV><H2 id="Non-small cell lung cancer">Non-small cell lung cancer</H2><P>Lung cancer is the most common cause of cancer-related mortality, with 1.6 million deaths in 2012. That’s 20% of all reported deaths due to cancer. Non-small cell lung cancer (NSCLC) makes up ~80% of all incidences of lung cancer.<SUP id="cite_ref-chan_2-0" class="reference"><A href="#cite_note-chan-2">[2]</A></SUP> 58% of all cases in 2012 were reported in less developed countries.<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP></P><P>NSCLC is aggressive and pathologically diverse.<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP> NSCLC commonly presents with a cough, haemoptysis and the symptoms of metastatic disease such as bone pain or jaundice. Only a minority of patients present with operable tumors; in most cases patients have advanced stage cancer and treatment is with palliative intent.<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> Treatment centers around chemotherapy and radiotherapy, but new treatments show promise including immunotherapies.<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP><SUP id="cite_ref-chan_2-1" class="reference"><A href="#cite_note-chan-2">[2]</A></SUP></P><P>About 90% of lung cancers are caused by smoking and as smoking rates have declined, there has been a corresponding reduction in the incidence of lung cancers.<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP> However, nearly 30% of the global population are still estimated to smoke.<SUP id="cite_ref-gopal_8-0" class="reference"><A href="#cite_note-gopal-8">[8]</A></SUP></P></DIV></DIV><DIV style="text-align: center; clear:both;"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Late_diagnosis_kills"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Late diagnosis kills</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Toehold_switches"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Toehold switches</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Our_project"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Our project</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#References"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV><H2><SPAN class="mw-headline" id="Late_diagnosis_kills">Late diagnosis kills</SPAN></H2></DIV><P><SVG id="latediagnosis_graph" xmlns="http://www.w3.org/2000/svg" stroke-width="28.2" viewbox="0 0 16452.9 9111.1" stroke-linejoin="round"><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="2279.9" y="8345.7"><TSPAN fill="#eee">IA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="4437.9" y="8345.7"><TSPAN fill="#eee">IB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="6551.9" y="8345.7"><TSPAN fill="#eee">IIA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="8709.9" y="8345.7"><TSPAN fill="#eee">IIB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="10825.9" y="8345.7"><TSPAN fill="#eee">IIIA</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="12982.9" y="8345.7"><TSPAN fill="#eee">IIIB</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-599.1" y="-439.3"><TSPAN font-size="317" font-weight="400" font-family="Calibri, sans-serif"><TSPAN x="15225.9" y="8345.7"><TSPAN fill="#eee">IV</TSPAN></TSPAN></TSPAN></TEXT><G fill="#ccc" font-family="Montserrat" font-size="317.5"><TEXT x="-599.1" y="-439.3"><TSPAN><TSPAN x="1036.9" y="8077.7"><TSPAN>0</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-506.3" y="-439.3"><TSPAN><TSPAN x="919.7" y="6935.7"><TSPAN>10</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-572.6" y="-439.3"><TSPAN><TSPAN x="853.4" y="5794.7"><TSPAN>20</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-575.1" y="-439.3"><TSPAN><TSPAN x="850.9" y="4653.7"><TSPAN>30</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-577" y="-439.3"><TSPAN><TSPAN x="849" y="3511.7"><TSPAN>40</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-584.4" y="-439.3"><TSPAN><TSPAN x="841.6" y="2370.7"><TSPAN>50</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-591.2" y="-439.3"><TSPAN><TSPAN x="834.8" y="1229.7"><TSPAN>60</TSPAN></TSPAN></TSPAN></TEXT></G><TEXT x="-171216.7" y="-39739.3" fill="#fff" stroke-width="1203.8" font-weight="600" font-size="511.8" font-family="Montserrat" text-anchor="middle"><TSPAN><TSPAN x="8099.1" y="397.7"><TSPAN>5 year NSCLC survival rates by stage</TSPAN></TSPAN></TSPAN></TEXT><G fill="#fff" text-anchor="middle" font-family="Montserrat" font-size="388.1"><TEXT x="396.9" y="-439.3"><TSPAN><TSPAN x="8113.9" y="9034.7"><TSPAN>Stage Diagnosed</TSPAN></TSPAN></TSPAN></TEXT><TEXT x="-5495.3" y="-5793.1" transform="rotate(-90)"><TSPAN><TSPAN x="-4608.3" y="287.9"><TSPAN>Survival rate (%)</TSPAN></TSPAN></TSPAN></TEXT></G></SVG><SPAN style="font-size: 12px; color: #888; text-align: center; display: block">Graph data from the American Cancer Society.<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP></SPAN></P><DIV style="text-align: center"><P>NSCLC’s high mortality rate is largely down to the late stage at which it’s normally diagnosed.<SUP id="cite_ref-dajac_10-0" class="reference"><A href="#cite_note-dajac-10">[10]</A></SUP> After metastasis, any treatment therefore is with palliative intent. However, surgery can be curative in early stages. <SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP><SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP></P></DIV></DIV><DIV class="content_wrapper"><DIV class="image_container"><DIV><H2 id="MicroRNA (miRNA)">MicroRNA (miRNA)</H2><P>MicroRNAs (miRNAs) can act as potent biomarkers for a myriad of diseases.
</P><P>MiRNAs are short non-coding RNAs of 19-24 nucleotides in length. They are involved in regulating post-transcriptional gene expression in eukaryotes.<SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP><SUP id="cite_ref-14" class="reference"><A href="#cite_note-14">[14]</A></SUP><SUP id="cite_ref-15" class="reference"><A href="#cite_note-15">[15]</A></SUP> MiRNAs can be found extracellularly in various body fluids, including serum<SUP id="cite_ref-16" class="reference"><A href="#cite_note-16">[16]</A></SUP><SUP id="cite_ref-Hennessey_1-1" class="reference"><A href="#cite_note-Hennessey-1">[1]</A></SUP>, plasma<SUP id="cite_ref-mitchell_17-0" class="reference"><A href="#cite_note-mitchell-17">[17]</A></SUP>, saliva<SUP id="cite_ref-18" class="reference"><A href="#cite_note-18">[18]</A></SUP>, urine<SUP id="cite_ref-hanke_19-0" class="reference"><A href="#cite_note-hanke-19">[19]</A></SUP> and breast milk<SUP id="cite_ref-kosaka_20-0" class="reference"><A href="#cite_note-kosaka-20">[20]</A></SUP>.
</P><P>It has been hypothesized that cells actively secrete miRNAs in microvesicles or bound to RNA binding proteins.<SUP id="cite_ref-chen_21-0" class="reference"><A href="#cite_note-chen-21">[21]</A></SUP> These shield the circulating miRNAs from ribonucleases<SUP id="cite_ref-chen_21-1" class="reference"><A href="#cite_note-chen-21">[21]</A></SUP> and are capable of delivering miRNAs to recipient cells.<SUP id="cite_ref-22" class="reference"><A href="#cite_note-22">[22]</A></SUP><SUP id="cite_ref-valadi_23-0" class="reference"><A href="#cite_note-valadi-23">[23]</A></SUP><SUP id="cite_ref-kosaka_20-1" class="reference"><A href="#cite_note-kosaka-20">[20]</A></SUP> These miRNAs can then trigger  signaling events, indicating that circulating miRNAs play a role in cell-to-cell communication.<SUP id="cite_ref-valadi_23-1" class="reference"><A href="#cite_note-valadi-23">[23]</A></SUP><SUP id="cite_ref-kosaka_20-2" class="reference"><A href="#cite_note-kosaka-20">[20]</A></SUP><SUP id="cite_ref-24" class="reference"><A href="#cite_note-24">[24]</A></SUP></P><P>Recent studies have shown that differential levels of miRNAs in body fluids are indicative of specific diseases, including many forms of cancer.<SUP id="cite_ref-Hennessey_1-2" class="reference"><A href="#cite_note-Hennessey-1">[1]</A></SUP><SUP id="cite_ref-mitchell_17-1" class="reference"><A href="#cite_note-mitchell-17">[17]</A></SUP><SUP id="cite_ref-hanke_19-1" class="reference"><A href="#cite_note-hanke-19">[19]</A></SUP><SUP id="cite_ref-25" class="reference"><A href="#cite_note-25">[25]</A></SUP> MiRNA stability, accessibility by non-invasive methods (liquid biopsy) and their ability to diagnose diseases in early stages give circulating miRNAs the potential to be effective biomarkers.<SUP id="cite_ref-etheridge_26-0" class="reference"><A href="#cite_note-etheridge-26">[26]</A></SUP></P><P>Currently, microarrays and qPCR are typically used to quantify circulating miRNAs. However, qPCR is labour intensive, whilst microarrays suffer from cross-hybridisation and inconsistent results.<SUP id="cite_ref-27" class="reference"><A href="#cite_note-27">[27]</A></SUP><SUP id="cite_ref-28" class="reference"><A href="#cite_note-28">[28]</A></SUP></P><P>Most current biomarkers are proteins, however there are numerous challenges to identifying new protein markers. These include the difficulties in developing suitably specific detection agents, the variety of post-translational modifications and the lack of proteins of interest in the blood.
</P><P>MiRNAs are stable, tissue or stage specific biomarkers which have great potential for effective clinical use.<SUP id="cite_ref-etheridge_26-1" class="reference"><A href="#cite_note-etheridge-26">[26]</A></SUP></P></DIV></DIV><H1><SPAN class="mw-headline" id="Toehold_switches">Toehold switches</SPAN></H1><P>Toehold switches are synthetic riboregulators that control gene expression post transcriptionally. They are hairpin loops found at the 5’ end of messenger RNA that can switch between two configurations - one which promotes and one which represses translation based on the presence or absence of a trigger RNA. Toehold switches have much greater possible dynamic ranges than previous riboregulators.
</P><DIV id="toehold_switch" class="start"><DIV class="hairpin" style="position: absolute;left: 35%; width: 500px; top: 15px;"><P>Initially the start codon is surrounded by a strong secondary structure - <SPAN style="color:#0288d1;font-weight:800">the hairpin stem</SPAN>. As the ribosome cannot melt the stem, translation of a fluorescent reporter protein coding sequence is repressed.</P><A onclick="document.getElementById('mi_up').beginElement(); document.getElementById('toehold_switch').className = 'miRNA_bound'">Play animation</A></DIV><DIV class="rbs" style="position: absolute;left: 45%; width:500px; top: 15px;"><P>When a <SPAN style="color:#ffcc80;font-weight:800">trigger RNA</SPAN> binds to the toehold region (a linear sequence at the 5’ end of the switch) it undergoes a branch migration with the lower part of the stem. This relieves the secondary structure around the start codon. The ribosome bound to the <SPAN style="color:#29b6f6;font-weight:800">RBS</SPAN> can then melt the remaining stem</P><A onclick="document.getElementById('rb_ribosome').beginElement(); document.getElementById('toehold_switch').className = 'ribosome_bound'">Next</A></DIV><DIV class="open" style="position: absolute; top: 15px; left: 15px; width: 500px"><P>This activates translation and fluorescence from the <SPAN style="color:#76ff03;font-weight:800">reporter protein</SPAN> can be observed.</P><A id="reset_btn" onclick="document.getElementById('th_reset').beginElement(); document.getElementById('mi_down').beginElement(); document.getElementById('toehold_switch').className = 'closed'">Reset</A></DIV></DIV><P>We used toehold switches to detect miRNAs, as they have very few sequence constraints so an arbitrary RNA sequence can act as the trigger RNA. As such, toehold switch mediated miRNA detection could act as a ubiquitous mechanism, capable of detecting any miRNA sequence for any disease.
</P><P>Previous riboregulators have relied on base pairing to the RBS to prevent ribosome binding.<SUP id="cite_ref-callura_29-0" class="reference"><A href="#cite_note-callura-29">[29]</A></SUP><SUP id="cite_ref-isaacs_30-0" class="reference"><A href="#cite_note-isaacs-30">[30]</A></SUP><SUP id="cite_ref-31" class="reference"><A href="#cite_note-31">[31]</A></SUP> For these, the trigger RNA must contain the RBS sequence to displace the sequence bound to the RBS, dramatically reducing programmability of the trigger RNA. However, the trigger RNA for a toehold switch has very few sequence constraints due to the lack of base pairing to the RBS and start codon (as strand displacement does not occur in the regions which have fixed sequences).
</P><P>The large dynamic range offered by toehold switches allows for low miRNAs concentrations to be detected and for discrimination of small differences in miRNA concentration.
</P><P>In previous riboregulators trigger RNA binding is initiated in the short loop domain of the hairpin module.<SUP id="cite_ref-callura_29-1" class="reference"><A href="#cite_note-callura-29">[29]</A></SUP><SUP id="cite_ref-isaacs_30-1" class="reference"><A href="#cite_note-isaacs-30">[30]</A></SUP><SUP id="cite_ref-32" class="reference"><A href="#cite_note-32">[32]</A></SUP> Subsequently, the binding thermodynamics in previous riboregulators are not as favourable as the toehold switch’s because toehold switches have longer linear trigger binding regions. This increases the probability of spontaneous trigger RNA dissociation and reduces the likelihood of successful strand displacement of the RBS from the incumbent sequence bound to it in previous riboregulators.<SUP id="cite_ref-33" class="reference"><A href="#cite_note-33">[33]</A></SUP> The dynamic range of previous riboregulators has therefore been limited to a maximum of 55 fold<SUP id="cite_ref-callura_29-2" class="reference"><A href="#cite_note-callura-29">[29]</A></SUP>, whereas toehold switches display a mean dynamic range of over 400 fold.<SUP id="cite_ref-34" class="reference"><A href="#cite_note-34">[34]</A></SUP></P><DIV class="image_container"><DIV><H2 id="Clinical implementation">Clinical implementation</H2><P>For NSCLC, miRNAs extracted from blood serum would be added to a cell free system containing an excess of toehold switches. These toehold switches detect a specific miRNA and regulate translation of GFP in response. The intensity of fluorescence produced by our circuits is therefore indicative of the miRNA levels in the body fluid.
</P><P>We’ve looked in great detail into how our sensor would be used in a clinical setting as part of <A href="/Team:CLSB-UK/HP/Silver" title="Team:CLSB-UK/HP/Silver">our human practices work</A>. We spoke to several doctors as part of our <A href="/Team:CLSB-UK/HP/Gold_Integrated" title="Team:CLSB-UK/HP/Gold Integrated">integrated human practices</A> and developed a strategy for how our test could be used in screening programmes for NSCLC. Additionally, we have improved the specificity of our sensor on their advice. We have also identified other diseases which our sensor could potentially detect and researched how our method would be beneficial.
</P><P>Our cost analysis shows our system would cost less than £20 per test. To prove screening with our test would be cost-effective, we created a <A href="/Team:CLSB-UK/Model#Cost_model" title="Team:CLSB-UK/Model">cost model</A>. The cost model takes into account the probabilities of different risk groups having cancer, the test’s false positive and negative rates, and the cost of treatment.
</P><P>To help reduce costs, we designed and built a £4 <A href="/Team:CLSB-UK/Hardware" title="Team:CLSB-UK/Hardware">combined fluorometer and densitometer</A> to cheaply quantify the fluorescence from our circuit without a plate reader. This allows our test to be used in the field and in less developed countries. To complement the hardware we developed an application to interpret the readings, so the device can be used on most laptops, tablets and even phones.
</P></DIV></DIV><DIV style="text-align: center; clear:both;"><H2><SPAN class="mw-headline" id="Our_project">Our project</SPAN></H2><P>Oh, and for those curious, Project BATMAN stands for <B>B</B>iosynthetic <B>A</B>pplications of <B>T</B>oehold switches, <B>M</B>iRNA <B>A</B>nd <B>N</B>SCLC. And yes, we know the ‘A’ stands for ‘and’, while the ‘N’ stands for ‘non-small cell lung cancer’, but we were really determined to shoehorn in an acronym.
</P><P>To learn more about the different aspects of our project check out the links below.
</P></DIV><DIV id="cta_links"><A href="/Team:CLSB-UK/Design"><H3>Design</H3></A><A href="/Team:CLSB-UK/Results"><H3>Results</H3></A><A href="/Team:CLSB-UK/HP"><H3>Human Practices</H3></A><A href="/Team:CLSB-UK/Model"><H3>Modeling</H3></A><A href="/Team:CLSB-UK/Software"><H3>Software</H3></A><A href="/Team:CLSB-UK/Hardware"><H3>Hardware</H3></A></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>To judges: we strongly recommend you read the <A href="/Team:CLSB-UK/Judging#Key_Papers" title="Team:CLSB-UK/Judging">key papers on our judging page</A> first to save time.
</P><OL class="references"><LI id="cite_note-Hennessey-1"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-Hennessey_1-0">1.0</A></SUP><SUP><A href="#cite_ref-Hennessey_1-1">1.1</A></SUP><SUP><A href="#cite_ref-Hennessey_1-2">1.2</A></SUP></SPAN><SPAN class="reference-text">Hennessey, P. T., Sanford, T., Choudhary, A., Mydlarz, W. W., Brown, D., Adai, A. T., ... &amp; Califano, J. A. (2012). Serum microRNA biomarkers for detection of non-small cell lung cancer. PloS one, 7(2), e32307.</SPAN></LI><LI id="cite_note-chan-2"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-chan_2-0">2.0</A></SUP><SUP><A href="#cite_ref-chan_2-1">2.1</A></SUP></SPAN><SPAN class="reference-text">Chan, B. A., &amp; Hughes, B. G. (2015). Targeted therapy for non-small cell lung cancer: current standards and the promise of the future. Translational lung cancer research, 4(1), 36.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Ferlay, J., Soerjomataram, I., &amp; Ervik, M. (2012). GLOBOCAN, cancer incidence and mortality worldwide: IARC cancer base no. 11 [Internet]. Lyon, France: International Agency for Research on Cancer; 2013.</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Board,  P. A. T. E. (2017). Non-Small Cell Lung Cancer Treatment (PDQ®).</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Bower, M., &amp; Waxman, J. (2010). Oncology: lecture notes. Wiley-Blackwell.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Chen, Z., Fillmore, C. M., Hammerman, P. S., Kim, C. F., &amp; Wong, K. K. (2014). Non-small-cell lung cancers: a heterogeneous set of diseases. Nature reviews. Cancer, 14(8), 535.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text"> (n.d.). Lung cancer: diagnosis and management - NICE. Retrieved October 7, 2017, from <A rel="nofollow" class="external free" href="https://www.nice.org.uk/guidance/cg121/chapter/introduction">https://www.nice.org.uk/guidance/cg121/chapter/introduction</A></SPAN></LI><LI id="cite_note-gopal-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-gopal_8-0">↑</A></SPAN><SPAN class="reference-text">Gopal, M., Abdullah, S. E., Grady, J. J., &amp; Goodwin, J. S. (2010). Screening for lung cancer with low-dose computed tomography: a systematic review and meta-analysis of the baseline findings of randomized controlled trials. Journal of thoracic oncology, 5(8), 1233-1239.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">(2015) Cancer Treatment &amp; Survivorship Facts &amp; Figures 2014-2015, American Cancer Society</SPAN></LI><LI id="cite_note-dajac-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-dajac_10-0">↑</A></SPAN><SPAN class="reference-text">Dajac, J., Kamdar, J., Moats, A., &amp; Nguyen, B. (2016). To Screen or not to Screen: Low Dose Computed Tomography in Comparison to Chest Radiography or Usual Care in Reducing Morbidity and Mortality from Lung Cancer. Cureus, 8(4).</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Uramoto, H., &amp; Tanaka, F. (2014). Recurrence after surgery in patients with NSCLC. Translational lung cancer research, 3(4), 242.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Molina, J. R., Yang, P., Cassivi, S. D., Schild, S. E., &amp; Adjei, A. A. (2008, May). Non-small cell lung cancer: epidemiology, risk factors, treatment, and survivorship. In Mayo Clinic Proceedings(Vol. 83, No. 5, pp. 584-594). Elsevier.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">Bartel, D. P. (2004). MicroRNAs: genomics, biogenesis, mechanism, and function. cell, 116(2), 281-297.</SPAN></LI><LI id="cite_note-14"><SPAN class="mw-cite-backlink"><A href="#cite_ref-14">↑</A></SPAN><SPAN class="reference-text">Ambros, V. (2004). The functions of animal microRNAs Nature 431: 350–355. Proceedings of the National Academy of Sciences of the United States of America, 103, 3687-3692.</SPAN></LI><LI id="cite_note-15"><SPAN class="mw-cite-backlink"><A href="#cite_ref-15">↑</A></SPAN><SPAN class="reference-text">Kim, V. N., Han, J., &amp; Siomi, M. C. (2009). Biogenesis of small RNAs in animals. Nature reviews Molecular cell biology, 10(2), 126-139.</SPAN></LI><LI id="cite_note-16"><SPAN class="mw-cite-backlink"><A href="#cite_ref-16">↑</A></SPAN><SPAN class="reference-text">Zen, K., &amp; Zhang, C. Y. (2012). Circulating microRNAs: a novel class of biomarkers to diagnose and monitor human cancers. Medicinal research reviews, 32(2), 326-348.</SPAN></LI><LI id="cite_note-mitchell-17"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-mitchell_17-0">17.0</A></SUP><SUP><A href="#cite_ref-mitchell_17-1">17.1</A></SUP></SPAN><SPAN class="reference-text">Mitchell, P. S., Parkin, R. K., Kroh, E. M., Fritz, B. R., Wyman, S. K., Pogosova-Agadjanyan, E. L., ... &amp; Lin, D. W. (2008). Circulating microRNAs as stable blood-based markers for cancer detection. Proceedings of the National Academy of Sciences, 105(30), 10513-10518.</SPAN></LI><LI id="cite_note-18"><SPAN class="mw-cite-backlink"><A href="#cite_ref-18">↑</A></SPAN><SPAN class="reference-text">Park, N. J., Zhou, H., Elashoff, D., Henson, B. S., Kastratovic, D. A., Abemayor, E., &amp; Wong, D. T. (2009). Salivary microRNA: discovery, characterization, and clinical utility for oral cancer detection. Clinical Cancer Research, 15(17), 5473-5477.</SPAN></LI><LI id="cite_note-hanke-19"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-hanke_19-0">19.0</A></SUP><SUP><A href="#cite_ref-hanke_19-1">19.1</A></SUP></SPAN><SPAN class="reference-text">Hanke, M., Hoefig, K., Merz, H., Feller, A. C., Kausch, I., Jocham, D., ... &amp; Sczakiel, G. (2010, December). A robust methodology to study urine microRNA as tumor marker: microRNA-126 and microRNA-182 are related to urinary bladder cancer. In Urologic Oncology: Seminars and Original Investigations (Vol. 28, No. 6, pp. 655-661). Elsevier.</SPAN></LI><LI id="cite_note-kosaka-20"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-kosaka_20-0">20.0</A></SUP><SUP><A href="#cite_ref-kosaka_20-1">20.1</A></SUP><SUP><A href="#cite_ref-kosaka_20-2">20.2</A></SUP></SPAN><SPAN class="reference-text">Kosaka, N., Izumi, H., Sekine, K., &amp; Ochiya, T. (2010). microRNA as a new immune-regulatory agent in breast milk. Silence, 1(1), 7.</SPAN></LI><LI id="cite_note-chen-21"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-chen_21-0">21.0</A></SUP><SUP><A href="#cite_ref-chen_21-1">21.1</A></SUP></SPAN><SPAN class="reference-text">Chen, X., Liang, H., Zhang, J., Zen, K., &amp; Zhang, C. Y. (2012). Secreted microRNAs: a new form of intercellular communication. Trends in cell biology, 22(3), 125-132.</SPAN></LI><LI id="cite_note-22"><SPAN class="mw-cite-backlink"><A href="#cite_ref-22">↑</A></SPAN><SPAN class="reference-text">Vickers, K. C., Palmisano, B. T., Shoucri, B. M., Shamburek, R. D., &amp; Remaley, A. T. (2011). MicroRNAs are transported in plasma and delivered to recipient cells by high-density lipoproteins. Nature cell biology, 13(4), 423.</SPAN></LI><LI id="cite_note-valadi-23"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-valadi_23-0">23.0</A></SUP><SUP><A href="#cite_ref-valadi_23-1">23.1</A></SUP></SPAN><SPAN class="reference-text">Valadi, H., Ekström, K., Bossios, A., Sjöstrand, M., Lee, J. J., &amp; Lötvall, J. O. (2007). Exosome-mediated transfer of mRNAs and microRNAs is a novel mechanism of genetic exchange between cells. Nature cell biology, 9(6), 654.</SPAN></LI><LI id="cite_note-24"><SPAN class="mw-cite-backlink"><A href="#cite_ref-24">↑</A></SPAN><SPAN class="reference-text">Turchinovich, A., Samatov, T. R., Tonevitsky, A. G., &amp; Burwinkel, B. (2013). Circulating miRNAs: cell–cell communication function?. Frontiers in genetics, 4.</SPAN></LI><LI id="cite_note-25"><SPAN class="mw-cite-backlink"><A href="#cite_ref-25">↑</A></SPAN><SPAN class="reference-text">Slater, E. P., Strauch, K., Rospleszcz, S., Ramaswamy, A., Esposito, I., Klöppel, G., ... &amp; Bartsch, D. K. (2014). MicroRNA-196a and-196b as potential biomarkers for the early detection of familial pancreatic cancer. Translational oncology, 7(4), 464-471.</SPAN></LI><LI id="cite_note-etheridge-26"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-etheridge_26-0">26.0</A></SUP><SUP><A href="#cite_ref-etheridge_26-1">26.1</A></SUP></SPAN><SPAN class="reference-text">Etheridge, A., Lee, I., Hood, L., Galas, D., &amp; Wang, K. (2011). Extracellular microRNA: a new source of biomarkers. Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis, 717(1), 85-90.</SPAN></LI><LI id="cite_note-27"><SPAN class="mw-cite-backlink"><A href="#cite_ref-27">↑</A></SPAN><SPAN class="reference-text">Lu, J., Getz, G., Miska, E. A., Alvarez-Saavedra, E., Lamb, J., Peck, D., ... &amp; Downing, J. R. (2005). MicroRNA expression profiles classify human cancers. nature, 435(7043), 834-838.</SPAN></LI><LI id="cite_note-28"><SPAN class="mw-cite-backlink"><A href="#cite_ref-28">↑</A></SPAN><SPAN class="reference-text">Sato, F., Tsuchiya, S., Terasawa, K., &amp; Tsujimoto, G. (2009). Intra-platform repeatability and inter-platform comparability of microRNA microarray technology. PloS one, 4(5), e5540.</SPAN></LI><LI id="cite_note-callura-29"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-callura_29-0">29.0</A></SUP><SUP><A href="#cite_ref-callura_29-1">29.1</A></SUP><SUP><A href="#cite_ref-callura_29-2">29.2</A></SUP></SPAN><SPAN class="reference-text">Callura, J. M., Cantor, C. R., &amp; Collins, J. J. (2012). Genetic switchboard for synthetic biology applications. Proceedings of the National Academy of Sciences, 109(15), 5850-5855.</SPAN></LI><LI id="cite_note-isaacs-30"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-isaacs_30-0">30.0</A></SUP><SUP><A href="#cite_ref-isaacs_30-1">30.1</A></SUP></SPAN><SPAN class="reference-text">Isaacs, F. J., Dwyer, D. J., Ding, C., Pervouchine, D. D., Cantor, C. R., &amp; Collins, J. J. (2004). Engineered riboregulators enable post-transcriptional control of gene expression. Nature biotechnology, 22(7), 841.</SPAN></LI><LI id="cite_note-31"><SPAN class="mw-cite-backlink"><A href="#cite_ref-31">↑</A></SPAN><SPAN class="reference-text">Rodrigo, G., Landrain, T. E., &amp; Jaramillo, A. (2012). De novo automated design of small RNA circuits for engineering synthetic riboregulation in living cells. Proceedings of the National Academy of Sciences, 109(38), 15271-15276.</SPAN></LI><LI id="cite_note-32"><SPAN class="mw-cite-backlink"><A href="#cite_ref-32">↑</A></SPAN><SPAN class="reference-text">Takahashi, M. K., &amp; Lucks, J. B. (2013). A modular strategy for engineering orthogonal chimeric RNA transcription regulators. Nucleic acids research, 41(15), 7577-7588.</SPAN></LI><LI id="cite_note-33"><SPAN class="mw-cite-backlink"><A href="#cite_ref-33">↑</A></SPAN><SPAN class="reference-text">Machinek, R. R., Ouldridge, T. E., Haley, N. E., Bath, J., &amp; Turberfield, A. J. (2014). Programmable energy landscapes for kinetic control of DNA strand displacement. Nature communications, 5, 5324.</SPAN></LI><LI id="cite_note-34"><SPAN class="mw-cite-backlink"><A href="#cite_ref-34">↑</A></SPAN><SPAN class="reference-text">Green, A. A., Silver, P. A., Collins, J. J., &amp; Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell, 159(4), 925-939.</SPAN></LI></OL></DIV><FOOTER><DIV class="content_wrapper"><DIV class="sponsors"><H3><A href="/Team:CLSB-UK/Sponsors">Major sponsors</A></H3><UL><LI><SVG xmlns="http://www.w3.org/2000/svg" viewbox="0 0 39.7 31.8"><TEXT fill="#fff" font-size="8" font-family="Lato, Arial, Helvetica, sans-serif"><TSPAN x="4.1" y="13.9">Mr John</TSPAN><TSPAN x="4.1" y="23.9">Mendez</TSPAN></TEXT></SVG></LI></UL></DIV><DIV class="contact"><H3>Contact us</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>