<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NJU_NJUT_China_oldpage"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NJU_NJUT_China/oldpage">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NJU_NJUT_China/oldpage&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NJU_NJUT_China/oldpage&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NJU_NJUT_China/oldpage&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NJU_NJUT_China/oldpage" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NJU NJUT China/oldpage</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="wrapper"><UL id="navigation"><LI class="home" style="width: 100px;"><A href=""><SPAN>Home</SPAN></A></LI><LI class="about" style="width: 115px;"><A href="https://2013.igem.org/Team:NJU_NJUT_China/team" style="width:115px;"><SPAN>TEAM</SPAN></A></LI><LI class="search" style="width: 135px;"><A href="https://2013.igem.org/Team:NJU_NJUT_China/project" style="width:135px;"><SPAN>PROJECT</SPAN></A></LI><LI class="photos" style="width: 150px;"><A href="" style="width:150px;"><SPAN>MODELING</SPAN></A></LI><LI class="rssfeed" style="width: 115px;"><A href="" style="width:115px;"><SPAN>DATA</SPAN></A></LI><LI class="podcasts" style="width: 125px;"><A href="" style="width:125px;"><SPAN>SAFETY</SPAN></A></LI><LI class="contact" style="width: 205px;"><A href="" style="width:205px;"><SPAN>HUMANPRACTICE</SPAN></A></LI></UL><DIV id="pxs_container" class="pxs_container"><DIV class="pxs_loading">Loading images...</DIV></DIV><DIV style="clear: both;"> </DIV><DIV id="page"><DIV id="page-bgtop"><DIV id="page-bgbtm"><DIV id="content"><DIV class="post"><H2 class="title"><A href="https://2013.igem.org/Team:NJU_NJUT_China/project">Project   </A></H2><P class="meta">Basic Information</P><DIV class="entry"><P>    Most of the bacteria and archaea acquire virus resistance by integrating short viral nucleotide acid fragments into the clusters of regularly interspaced short palindromic repeats (CRISPRs).			  And CRISPR-based defense system can also protects them against the invading DNA and/or RNA elements. It is believed that the integrated CRISPR sequences have the ability to form a genetic memory which prevents the host from being infected. The CRISPRs and Cas (CRISPR-associated) interact and form this prokaryotic adaptive immune system....</P><P> </P></DIV></DIV><DIV class="post"><H2 class="title"><A href="#">Introduction to our experiment</A></H2><P class="meta">information</P><DIV class="entry"><DIV class="aa" id="change"><P>Based on the fundamental synthetic biology methods, include the molecular cloning, long PCR, recombinant construction, restriction digestion and yeast transformation, our project aims to enable  Saccharomyces cerevisiae to resist viruses utilizing the CRISPR system found in most bacteria and archaea lately. </P><P>To ensure the accurate output, we divide the original linear gene into several elements , and also purify and propagate the complete target sequence to make the both two models work in the host cell. To present the actual effect of CRISPR in yeast, we design our own mimic virus construction based on the homologous sequences shown in the specific viral genome database, which can be recognized, and control the baseline both for anti-virus ability and CRISPR's effect in its original form.</P><P>More specifically, we first obtain our CRISPR system by genome-extraction from E.coli str W3110, and clone the respective target genes as modules. The construction, as well as the mimic virus construction, IS made by restriction-ligase way. We transform the mimic virus plasmid first by lithium acetate method and verify the availability by several antibiotics. We then import CRISPR system in the same way, utilizing the Ochre mutation and GFP switch-off to evaluate the anti-virus feasibility.</P></DIV><P class="links" onclick="unfold('change')">Read More </P></DIV></DIV></DIV></DIV><DIV id="sidebar"><DIV class="sp"><DIV id="col"><DL class="sliderbox" id="slider2"><DT class="open"><SPAN class="date">
		            Epub 2012 Dec 30		            </SPAN><SPAN class="title">Folia microbiologica </SPAN></DT><DD><DIV class="text"><P style="font-size:15px"> Molecular identification and characterization of clustered regularly interspaced short palindromic repeat (CRISPR) gene cluster in Taylorella equigenitalis.</P>
			            Clustered regularly interspaced short palindromic repeats (CRISPRs), of approximately 10,000base pairs (bp) in length, were shown to occur in the Japanese Taylorella equigenitalis strain, EQ59. The locus was composed of the putative CRISPRs-associated with 5 (cas5), RAMP csd1, csd2, recB, cas1, a leader region, 13 CRISPR consensus sequence repeats (each 32bp; 5'-TCAGCCACGTTCGCGTGGCTGTGTGTTTAAAG-3'). These were in turn separated by 12 non repetitive unique spacer regions of similar length. In addition, a leader region, a transposase/IS protein, a leader region, and cas3 were also seen. All seven putative open reading frames carry their ribosome binding sites. Promoter consensus sequences at the -35 and -10 regions and putative intrinsic rho-independent transcription terminator regions also occurred. A possible long overlap of 170bp in length occurred between the recB and cas1 loci. Positive reverse transcription PCR signals of cas5, RAMP csd1, csd2-recB/cas1, and cas3 were generated. A putative secondary structure of the CRISPR consensus repeats was constructed. Following this, CRISPR results of the T. equigenitalis EQ59 isolate were subsequently compared with those from the Taylorella asinigenitalis MCE3 isolate.		                </DIV></DD><DT><SPAN class="date">
		            Epub 2013 May 24		            </SPAN><SPAN class="title">Genetics</SPAN></DT><DD><DIV class="text"><P style="font-size:16px"> Genome Engineering of Drosophila with the CRISPR RNA-Guided Cas9 Nuclease.</P>
			            We have adapted a bacterial CRISPR RNA/Cas9 system to precisely engineer the Drosophila genome and report that Cas9-mediated genomic modifications are efficiently transmitted through the germline. This RNA-guided Cas9 system can be rapidly programmed to generate targeted alleles for probing gene function in Drosophila.		              </DIV></DD><DT><SPAN class="date">
		            Epub 2013 Jun 30		            </SPAN><SPAN class="title">Nature methods </SPAN></DT><DD><DIV class="text"><P style="font-size:16px">Heritable genome editing in C. elegans via a CRISPR-Cas9 system.</P>
			            We report the use of clustered, regularly interspaced, short palindromic repeats (CRISPR)-associated endonuclease Cas9 to target genomic sequences in the Caenorhabditis elegans germ line using single-guide RNAs that are expressed from a U6 small nuclear RNA promoter. Our results demonstrate that targeted, heritable genetic alterations can be achieved in C. elegans, providing a convenient and effective approach for generating loss-of-function mutants.		              </DIV></DD><DT><SPAN class="date">
		            JUL 18 2013		            </SPAN><SPAN class="title">CELL</SPAN></DT><DD><DIV class="text"><P style="font-size:16px">CRISPR-Mediated Modular RNA-Guided Regulation of Transcription in Eukaryotes</P>
			            The genetic interrogation and reprogramming of cells requires methods for robust and precise targeting of genes for expression or repression. The CRISPR-associated catalytically inactive dCas9 protein offers a general platform for RNA-guided DNA targeting. Here, we show that fusion of dCas9 to effector domains with distinct regulatory functions enables stable and efficient transcriptional repression or activation in human and yeast cells, with the site of delivery determined solely by a coexpressed short guide (sg)RNA. Coupling of dCas9 to a transcriptional repressor domain can robustly silence expression of multiple endogenous genes. RNA-seq analysis indicates that CRISPR interference (CRISPRi)-mediated transcriptional repression is highly specific. Our results establish that the CRISPR system can be used as a modular and flexible DNA-binding platform for the recruitment of proteins to a target DNA sequence, revealing the potential of CRISPRi as a general tool for the precise regulation of gene expression in eukaryotic cells.		              </DIV></DD><DT><SPAN class="date">
		            JUL 15 2013		            </SPAN><SPAN class="title">BIOCHEMICAL JOURNAL</SPAN></DT><DD><DIV class="text"><P style="font-size:16px">CRISPR interference: a structural perspective</P>
			            CRISPR (cluster of regularly interspaced palindromic repeats) is a prokaryotic adaptive defence system, providing immunity against mobile genetic elements such as viruses. Genomically encoded crRNA (CRISPR RNA) is used by Cas (CRISPR-associated) proteins to target and subsequently degrade nucleic acids of invading entities in a sequence-dependent manner. The process is known as 'interference'. In the present review we cover recent progress on the structural biology of the CRISPR/Cas system, focusing on the Cas proteins and complexes that catalyse crRNA biogenesis and interference. Structural studies have helped in the elucidation of key mechanisms, including the recognition and cleavage of crRNA by the Cas6 and Cas5 proteins, where remarkable diversity at the level of both substrate recognition and catalysis has become apparent. The RNA-binding RAMP (repeat-associated mysterious protein) domain is present in the Cas5, Cas6, Cas7 and Cmr3 protein families and RAMP-like domains are found in Cas2 and Cas10. Structural analysis has also revealed an evolutionary link between the small subunits of the type I and type III-B interference complexes. Future studies of the interference complexes and their constituent components will transform our understanding of the system.		              </DIV></DD><DT><SPAN class="date">
		            JUL 2013		            </SPAN><SPAN class="title">CELL REPORTS  </SPAN></DT><DD><DIV class="text"><P style="font-size:16px"> Highly Efficient Targeted Mutagenesis of Drosophila with the CRISPR/Cas9 System</P>
			            Here, we present a simple and highly efficient method for generating and detecting mutations of any gene in Drosophila melanogaster through the use of the CRISPR/Cas9 system (clustered regularly interspaced palindromic repeats/CRISPR-associated). We show that injection of RNA into the Drosophila embryo can induce highly efficient mutagenesis of desired target genes in up to 88% of injected flies. These mutations can be transmitted through the germline to make stable lines. Our system provides at least a 10-fold improvement in efficiency over previously published reports, enabling wider application of this technique. We also describe a simple and highly sensitive method of detecting mutations in the target gene by high-resolution melt analysis and discuss how the new technology enables the study of gene function.		              </DIV></DD></DL></DIV></DIV></DIV></DIV><DIV id="myfooter"><P><A href="#">IGEM_NJU_NJUT</A>.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NJU_NJUT_China/oldpage">http://2013.igem.org/Team:NJU_NJUT_China/oldpage</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NJU_NJUT_China/oldpage" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NJU_NJUT_China/oldpage" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NJU_NJUT_China/oldpage&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NJU_NJUT_China/oldpage&amp;oldid=202820" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>