"
Page
Discussion
View source
History
teams
Log in
 
Team:NJU NJUT China/project
From 2013.igem.org
main
Poject Introduction
basic information
    All cellular systems evolve ways to combat predators and genomic parasites. In bacteria and archaea, numerous resistance mechanisms have been developed against phage. Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated (Cas) adaptive immune systems are found in bacteria and archaea to protect the hosts against the invasion of viruses and plasmids. Most of the bacteria and archaea acquire virus resistance by integrating short viral nucleotide acid fragments into the CRISPR sequences. It is believed that the integrated CRISPR sequences have the ability to form a genetic memory which prevents the host from being infected. The CRISPRs and Cas (CRISPR-associated) interact and form this prokaryotic adaptive immune system. A CRISPR array consists of the palindromic repeating sequences of typically 30bp that are interspaced by similar-sized acquired spacer sequences. The CRISPR loci are generally flanked by an AT-rich leader sequence which contains promoter and the binding sites for regulatory proteins. Highly-variable CRISPR loci provide insights into the phage and host population dynamics, and new avenues for enhanced phage resistance and genetic typing and tagging of industrial strains.
    The CRISPR/Cas immune system encompasses a multistep process by which DNA-encoded spacers lead to sequence-specific interference of invasive nucleic acids, mediated by a ribonucleoproteic complex. There are two main processes characteristic of CRISPR-based immunity, namely spacer acquisition (immunization) and crRNA-mediated interference (immunity). Thus far, most of the molecular basis for the CRISPR/Cas mechanism of action has focused on the interference step, with notable focus on the biochemical details of crRNA biogenesis and target nucleic acid interaction.
    There are three highly diverse CRISPR/Cas types exist, and major structural and functional differences are displayed in their mode of generating resistance against the invading DNA and/or RNA elements. The CRISPR-associated protein Cas9 that belongs to the type II CRISPR/Cas system has attracted much attention due to its potential use in genomic engineering. The cas9 signature gene encodes a large multifunctional protein with the ability to generate crRNA and targets phage and plasmid DNA for degradation. Cas9 contains a RuvC-like nuclease domain and a McrA-like HNH nuclease domain. The functional S. thermophilus CRISPR model is a Type II system that has been shown to provide defense against bacteriophage and plasmid DNA.
    In the spacer acquisition process, small fragments of invasive nucleic acid are incorporated into the host genome between CRISPR repeats at the leader end of the locus. The polarized integration of novel repeat-spacer units at the leader end provides a chronological time line of spacer acquisition events. The sequence on the viral genome that corresponds to a spacer is termed proto-spacer. In most cases, a very short stretch of nucleotides is conserved in the immediate vicinity of the proto-spacer, the CRISPR motif, or the proto-spacer-associated motif (PAM). The ability to acquire novel spacers in vivo has been experimentally documented in S. thermophilus and Streptococcus mutans, in which trans-encoded small CRISPR RNA (tracrRNA) is involved in the processing of pre-CRISPR RNA (pre-crRNA) into crRNA. Subsequent recruitment of a housekeeping RNase III catalyzes, together with Cas9, cleavage of the pre-crRNA in the repeat; this yields 66-nt crRNA molecules that are further trimmed at the 5’end to produce 39–42-nt mature crRNA containing a 20-nt spacer sequence. During Type II CRISPR interference, target DNA is cleaved in the proto-spacer sequence by the CRISPR/Cas complex.
    All organisms need to continuously adapt to changes in their environment. Horizontal gene transfer (HGT), a dominant evolutionary process, at least, in prokaryotes, appears to be a form of Lamarckian inheritance. Obviously, the CRISPR system is a much more impressive example. Now, genome-editing by Cas9 is so efficient that it creates a popular method to knock out one gene from the host genome. However, people met lots of problems during attempts to specifically-targeting and cleavage-facilitating. As the best result of natural selection, we are sure that the CRISPR/Cas systems have their specification and high-efficiency when choosing the spacer. Based on the CRISPR library of S. thermophiles, E. Coli, Lactobacillus and Salmonella, we design one model to simulate the selection of spacer with the help of PAM (proto-spacer associated motifs), and then develop a theory to guide the choosing target-sites.
    As the imaginary result of our model, we choose the CRISPR sequence from E.coli K12, which is the most usual bacteria with the CRISPR/Cas system, to be our genetic memory loaded into the vector plasmid1 has been well designed. This plasmid is taken to generate the guiding-RNA (crRNA & tracrRNA) and interact with the Cas9 protein produced by the Cas9 plasmid2. We also design a set of target plasmids3 which have the linear structure of eGFP-target sequence. The target sequence is one existing spacer in the CRISPR of E.coli K12, which can pair with crRNA produced by the plasmid1. This linear structure will be destroyed, theoretically, when the Cas9/gRNA complex recognize the target-site, which is a 30nt sequence, and then cleavage the GFP gene, which is adjacent to the 30-nt target site.
    In some sense, our experiments is significant. Firstly, our model give us an efficient and novel method to design the DNA sequence which can be transcribed into the crRNA, which works with the core protein Cas9 to recognize the target site. With the help of our model, we can import this system into the superbugs, which can produce the well-selected guiding-RNA specifically paired with the resistance genes. Meanwhile, it is believed that the chance that this kind of crRNAs pair with other sites is very low. Subsequently, Cas9 interacts with gRNA and then forms the protein-nucleic acid complex. This complex can theoretically destroy any chosen resistance genes, and its carrier, or rather the common plasmids, can exist in the bacteria steady and continuously. Secondly, we can use this system into several fields, such as fermentation, antiviral field, improving the plants and any other bioindustrial works. At the same time, the mechanism of heredity and genome evolution embodied in the CRISPR-Cas system seems to be bona fide Lamarckian we can also learn the co-evolution of bacteria and virus from these systems.
Reference.
1. Zhang J, Rouillon C, Kerou M, et al. Structure and mechanism of the CMR complex for CRISPR-mediated antiviral immunity[J]. Molecular cell, 2012, 45(3): 303-313.
2. Brouns S J J, Jore M M, Lundgren M, et al. Small CRISPR RNAs guide antiviral defense in prokaryotes[J]. Science, 2008, 321(5891): 960-964.
3. Díez-Villaseñor C, Almendros C, García-Martínez J, et al. Diversity of CRISPR loci in Escherichia coli[J]. Microbiology, 2010, 156(5): 1351-1361.
4. Marraffini L A, Sontheimer E J. CRISPR interference limits horizontal gene transfer in staphylococci by targeting DNA[J]. science, 2008, 322(5909): 1843-1845.
5. Shen B, Zhang J, Wu H, et al. Generation of gene-modified mice via Cas9/RNA-mediated gene targeting[J]. Cell research, 2013.
6. Westra E R, Swarts D C, Staals R H J, et al. The CRISPRs, they are a-Changin': how prokaryotes generate adaptive immunity[J]. Annual review of genetics, 2012, 46: 311-339.
7. Cong L, Ran F A, Cox D, et al. Multiplex genome engineering using CRISPR/Cas systems[J]. Science, 2013, 339(6121): 819-823.
8. Karginov F V, Hannon G J. The CRISPR system: small RNA-guided defense in bacteria and archaea[J]. Molecular cell, 2010, 37(1): 7-19.
9. Mali P, Yang L, Esvelt K M, et al. RNA-guided human genome engineering via Cas9[J]. Science, 2013, 339(6121): 823-826.
10. Barrangou R, Horvath P. CRISPR: new horizons in phage resistance and strain identification[J]. Annual review of food science and technology, 2012, 3: 143-162.
11. Hale C R, Majumdar S, Elmore J, et al. Essential features and rational design of CRISPR RNAs that function with the Cas RAMP module complex to cleave RNAs[J]. Molecular cell, 2012, 45(3): 292-302.
12. Semenova E, Jore M M, Datsenko K A, et al. Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence[J]. Proceedings of the National Academy of Sciences, 2011, 108(25): 10098-10103.
Retrieved from " http://2013.igem.org/Team:NJU_NJUT_China/project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
