<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Queens_Canada_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Queens Canada/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><NAV id="qgemnav"><UL><LI><A href="https://2015.igem.org/Team:Queens_Canada/Sponsorship">Sponsorship</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/Academics">Academics</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/Outreach">Community Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/IP">Intellectual Property</A></LI></UL><LI><A href="https://2015.igem.org/Team:Queens_Canada/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/AFP_Scaffold"> The Ice Queen</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Circ_AFP"> Icefinity</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada">Home</A></LI></NAV></DIV><DIV class="intro"><H1>LAB NOTEBOOK</H1><P>On this page we have documented our lab work and have divided it all into three components, by the construct being generated: </P><P align="center"><A href="#AFPEcoil">AFP-E coil</A></P><P align="center"><A href="#circAFP">Circularized Antifreeze Protein</A></P><P align="center"><A href="#Scaffold">Scaffold-K coil</A></P></DIV><DIV id="AFPEcoil" style="padding: 30px 200px 20px 200px;"><H2>Antifreeze Protein - E coil</H2><TABLE border="2px"><TBODY><TR><TH>DAY</TH><TH>TASK</TH><TH>OUTCOME</TH></TR><TR><TD>June 1st</TD><TD>Plated E. coli with Type III AFP donated by Dr. Davies and incubated overnight</TD><TD>Very few Type 3 AFP colonies</TD></TR><TR><TD>June 2nd</TD><TD>Prepared O/N cultures of Type 3 AFP</TD></TR><TR><TD>June 3rd</TD><TD>Mini-prepped wtAFP - Concentration of DNA measured with NanoDrop  - Annealed E-coil oligonucleotides
</TD></TR><TR><TD>June 4th</TD><TD>Digested wtAFP backbone treated with SAP - wtAFP run on agarose to verify linearization - Performed XhoI digest
</TD><TD>Unsuccessful: SAP likely degraded backbone too much/reaction allowed to proceed for too long</TD></TR><TR><TD>June 5th</TD><TD>Digested wtAFP backbone, treated with SAP</TD></TR><TR><TD>June 8th</TD><TD>Ligated annealed E-coil oligos into digested wtAFP plasmid with vector: oligo ratios 2:1, 1:1, and 2:3 - Plated wtAFP cells.</TD></TR><TR><TD>June 9th</TD><TD>(-) Control plate had highest colony count; shouldn’t have any - (+) Control had a few colonies - Experimental plates had a few colonies
</TD></TR><TR><TD>June 10th</TD><TD>Inoculated 10 colonies from the 2:1 plate to ON cultures with Amp</TD></TR><TR><TD>June 11th</TD><TD>Colony PCR'd liquid cultures of AFP-Ecoil colonies, and ran 2% agarose gels with PCR products and 100bp DNA ladder</TD><TD>Unsuccessful insertion of oligos</TD></TR><TR><TD>June 15th</TD><TD>Digests were attempted yet again, then SAP treatment -&gt; transformation</TD><TD>Successful: colonies present</TD></TR><TR><TD>June 16th</TD><TD>Colony screening was done through colony PCR</TD><TD>unsuccessful: colony PCR failed</TD></TR><TR><TD>June 22nd</TD><TD>Attempted insertion of oligos once more, this time with gel extraction of bands after XhoI digestion</TD></TR><TR><TD>June 26th</TD><TD>Ligation attempt - Ligated cells plated on Amp</TD><TD>Only successful plate was wtAFP without E-coil - Seems to be nuclease contamination degrading plasmid
</TD></TR><TR><TD>July 2nd</TD><TD>Again, digestion-extraction-ligation protocol performed (this time treated vectors qithout SAP). </TD><TD>Successful: colonies present,picked MANY to find right one</TD></TR><TR><TD>July 3rd</TD><TD>XhoI digest for wtAFP-E coil plate for colony screening</TD><TD>Successful: certain colonies appear to have an insert - sent for sequencing to confirm the coil is in the right orientation; successful!</TD></TR><TR><TD>August 4th</TD><TD>MTransformed correct AFP-E coil into BL21 cells for protein expression</TD><TD>Successful: colonies present</TD></TR><TR><TD>August 11th</TD><TD>Made 2L of AFP E-coil cells</TD></TR><TR><TD>August 12th </TD><TD>Lysed AFP E-coil cells and kept the sup overnight in the cold room (for ice-affinity purification tomorrow) - ran SDS-PAGE of the TCL/sup/pellet to confirm overexpression/solubility of the AFP-Ecoil protein 
</TD></TR><TR><TD>August 13th</TD><TD>Did one round of ice-affinity purification to confirm that the AFP-E.coil fusion can still bind to ice - ran SDS-PAGE of sup, liquid fraction 1, ice fraction 1</TD><TD>The AFP appeared to bind to ice, but we let the crystal grow too large and a bunch of other proteins were incorporated into the ice fraction too - will do another round of ice-affinity tomorrow to further purify
</TD></TR><TR><TD>August 14th</TD><TD>Ran a second round of ice-affinity purification, starting with the ice fraction from the first round - Ran an SDS-PAGE of the sup, ice/liquid fractions 1, ice/liquid fractions 2
</TD><TD>Successful! It appears that the AFP-E coil bound to ice well in the second ice-affinity purification; much better separation!</TD></TR></TBODY></TABLE></DIV><DIV id="circAFP" style="padding: 30px 200px 20px 200px;"><H2>Circularized AFP (Icefinity)</H2><TABLE border="2px"><TBODY><TR><TH>DAY</TH><TH>TASK</TH><TH>OUTCOME</TH></TR><TR><TD>June 1st</TD><TD>Re-suspended part BBa_K1362000 and transformed into electro- competent E. coli cells plated on Cam resistance plates</TD><TD>BBa_K1362000 red colonies visible</TD></TR><TR><TD>June 2nd</TD><TD>Grew 1L culture of BBa_K1362000 in LB for imaging, and prepared ON culture of BBa_K1362000 with Cam
</TD></TR><TR><TD>June 3rd</TD><TD>Mini-prepped BBa_K1362000 liquid culture - Concentration of DNA measured with NanoDrop
</TD></TR><TR><TD>June 9th</TD><TD>AFP-extein gene arrived</TD></TR><TR><TD>June 10th</TD><TD>Re-suspended DNA order 
- Performed Golden Gate Assembly using BBa_K1362000 from June 3rd and AFP-extein DNA 
- Transformed reaction products into TopTen electrocompetent cells
 - Plated on Cam plates
</TD><TD>Successful: Plates had more white than red colonies.</TD></TR><TR><TD>June 11th</TD><TD>O/N culture prepared from white colonies on plate</TD></TR><TR><TD>June 12th</TD><TD>Colony screening of white colonies; digested with NotI to determine if insert was present</TD><TD>Successful: Insert DNA appears correct size</TD></TR><TR><TD>June 17th</TD><TD>Primers resuspended in dH2O - Prepared samples for sequencing
</TD></TR><TR><TD>July 2nd</TD><TD>Transformed 2 samples in BL21 electrocompetent cells and plated</TD><TD>Succesful: 3 colonies on one of the plates, zero on the other</TD></TR><TR><TD>July 6th</TD><TD>Prepared ON culture with one transformed colony
</TD></TR><TR><TD>July 7th</TD><TD>Prepared liquid culture, grew to OD 0.85 - Induced protein expression  at 23 degrees C with 1mM IPTG overnight.
</TD></TR><TR><TD>July 8th</TD><TD>Protein expression;  then attempted ice affinity purification - Ran on SDS-PAGE
</TD><TD>Unsuccessful: appears to be no protein expressed</TD></TR><TR><TD>July 10th</TD><TD>Careful troubleshooting and review of plasmid sequence</TD><TD>Identified problem pSB1C3 has no T7 promoter sequence… unable to express protein. </TD></TR><TR><TD>July 14th - 17th</TD><TD>Insert containing AFP-linker-extein was PCR amplified</TD><TD>PCR successful: amplicon band of correct size is visible</TD></TR><TR><TD>July 22nd</TD><TD>Digested both the vector and insert with Not1 and Xba1</TD></TR><TR><TD>July 23rd</TD><TD>Digests were then ligated then transformed in electrocompetent Topten E.coli K12 cells.</TD><TD>Successful insertion of insert into our promoter-containing vector. </TD></TR><TR><TD>July 28th</TD><TD>colonies were chosen, DNA purified, then digested with Not1 and Xba1 to confirm correct insertion size</TD><TD>Successful: band of insert is of the correct size.</TD></TR><TR><TD>July 30th - August 3rd</TD><TD>miniprepped DNA was containing our insert and T7 promoter was transformed into BL21 cells for expression</TD><TD>Successful</TD></TR><TR><TD>August 10th</TD><TD>O/N cultures of circAFP samples and AFP wild type</TD></TR><TR><TD>August 11th</TD><TD>Grew up 2L cultures and induced protein expression</TD><TD>AFP circ sequences confirmed with T7 promoter :D</TD></TR><TR><TD>August 12th</TD><TD>Isolated cells from cultures - Lysed small fractions of frozen pellet to test intein reaction at various temperatures </TD><TD>Gels show a band below the Type III wild type</TD></TR><TR><TD>August 13th</TD><TD>Ran gels for test conditions to determine optimal splicing conditions</TD></TR><TR><TD>August 14th</TD><TD>Ran TH assay on small fraction crude lysate supernatant to see if protein was active - Lysed frozen pellets - Also performed ice-affinity purification for the circAFP
</TD><TD>TH gap of 0.3oC → protein is active!!! - Purification will be determined Monday
</TD></TR><TR><TD>August 19th - 20th</TD><TD>Purified circAFP using size exclusion - Ran SDS-PAGE</TD><TD>Successful: single band present on gel</TD></TR><TR><TD>August 24th-28th</TD><TD>Subjected pure circAFP to various temperatures - Ran TH assays on the various circAFP to determine activity and thermostability - Analyzed results, put together bar graph. </TD><TD>Successful: circAFP appears to retain about 80% of its activity after exposure to 90 degrees C.</TD></TR><TR><TD>September 15th</TD><TD>ice-affinity purification was attempted with circAFP - SDS-PAGE was run</TD><TD>successful: gel indicated a completely pure circAFP</TD></TR></TBODY></TABLE></DIV><DIV id="Scaffold" style="padding: 30px 200px 20px 200px;"><H2>Scaffold - K coil</H2><TABLE border="2px"><TBODY><TR><TH>DAY</TH><TH>TASK</TH><TH>OUTCOME</TH></TR><TR><TD>June 18th</TD><TD>T3-10 in pET29b arrived from Baker lab</TD></TR><TR><TD>June 22nd</TD><TD>Transformed BL21 cells (electrocompetent) with T3-10 scaffold plasmid on Kan plates</TD></TR><TR><TD>June 23rd</TD><TD>Selected colony to make liquid culture</TD></TR><TR><TD>June 24th</TD><TD>Prepared 1L LB culture with T3-10, grew to OD ~ 2.0 - Protein expression induced (25 deg, 1mM IPTG)
</TD></TR><TR><TD>June 25th</TD><TD>Cells lysed and centrifuged to remove pellet - Supernatant run through Ni-column and fractions run on SDS-gel to purify
</TD><TD>Good expression of scaffold protein</TD></TR><TR><TD>June 29th</TD><TD>Ran purified T3-10 on S200 column - Re-suspended protein in new buffer
</TD><TD>Major peak indicative of trimer - May be some 12-mer (small peak)
</TD></TR><TR><TD>June 30th</TD><TD>Re-ran S200 column - Incubated sample at RT for 48 hours in new buffer</TD><TD>No change in chromatograth</TD></TR><TR><TD>July 13th</TD><TD>SDS-gel to verify T33-21 unit sizes from Ni2+ purification</TD><TD>Units correct size</TD></TR><TR><TD>July 15th-16th</TD><TD>Prepared ON cultures of T3-10 scaffold protein- mini-prepped T3-10</TD></TR><TR><TD>July 17th</TD><TD>K coil oligonucleotides arrived</TD></TR><TR><TD>July 20th</TD><TD>Re-suspended k-coils in dH2O - XhoI digested plasmids, SAP treated - Agarose gel to check before ligation
 - Plated T3-10 on Kan plate
</TD><TD>Unsuccessful: gel inconclusive</TD></TR><TR><TD>July 21st</TD><TD>Ran T3-10 mini preps on agarose gel - Prepared ON cultures of T3-10
</TD><TD>Plasmid confirmed</TD></TR><TR><TD>July 22nd</TD><TD>XhoI digested plasmids, SAP treated - Agarose gel to check before ligation - set up overnight ligations with T3-10 and K coil oligos
</TD><TD>The gels of the XhoI digest looked better - 2 of the samples were used in parallel with hopes one will ligate correctly
</TD></TR><TR><TD>July 23rd</TD><TD>O/N ligations were transformed into Top10 cells and plated on Kan plates overnight - 2 negative controls (to check digest/SAP) and 1 positive control (test ligase) were used - insert:vector ratios of 1:1 and 1:3 were used
</TD><TD>Successful: transformations look good, screening next week!</TD></TR><TR><TD>July 27th-28th</TD><TD>Grew O/N cultures of potential T3-10-K coil, miniprepped, and screened with digests with BamHI</TD><TD>Successful: some appear to have a BamHI site, some samples sent for sequencing</TD></TR><TR><TD>August 10th</TD><TD>Sequences indicate cloning was unsuccessful</TD></TR><TR><TD>August 28th</TD><TD>T3-10-K coil sequence arrived</TD></TR><TR><TD>August 2nd-4th</TD><TD>Prepared to transform T3-10-K coil into iGEM BioBrick vector pSB1C3</TD><TD>ligated DNA was prepped for shipping</TD></TR></TBODY></TABLE><H1>NOTEBOOK</H1><P align="center"><A href="#AFPEcoil">AFP-E coil</A></P><P align="center"><A href="#circAFP">Circularized Antifreeze Protein</A></P><P align="center"><A href="#Scaffold">Scaffold-K coil</A></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>