<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Queens_Canada_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Queens Canada/Background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar" style="margin-top: -50px;"><NAV id="qgemnav"><UL><LI><A href="https://2015.igem.org/Team:Queens_Canada/Sponsorship">Sponsorship</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/Academics">Academics</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/Outreach">Community Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Practices/IP">Intellectual Property</A></LI></UL><LI><A href="https://2015.igem.org/Team:Queens_Canada/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/AFP_Scaffold"> The Ice Queen</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Circ_AFP"> Icefinity</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Queens_Canada">Home</A></LI></NAV></DIV><DIV id="backgroundintro"><H1>BACKGROUND INFORMATION</H1><P align="center">QGEM this year centered around the topic of protein engineering. We wanted to give some background information on the proteins and complexes we worked on. </P></DIV><DIV id="AFPinfo"><H1>ANTIFREEZE PROTEINS</H1><P>Have you ever wondered how fish and other organisms can survive in sub-zero Arctic Oceans without freezing (Figure 1)? Or why some plants can recover from a frost more easily than others? Some organisms use glycerol or other solutes to tolerate the below-freezing temperatures of extreme environments. However, a large number of diverse species have been found to use a special class of proteins termed antifreeze proteins that inhibit ice growth enabling survival in sub-zero climates.</P><FIGURE style="float: left; width: 400px;"><FIGCAPTION>Figure 1. <STRONG>Image of an ocean pout found in the Northwest Atlantic Ocean <SUP>1</SUP>.</STRONG>Ocean pout harbour the Type III Antifreeze Protein in their blood which enables them to survive in cold water.</FIGCAPTION></FIGURE><H2>AFP Activity</H2><P>Antifreeze proteins, or AFPs, are naturally occurring, highly diverse proteins with the unique ability to bind to an ice surface. There is great diversity in the structure of AFPs, a consequence of AFPs evolving independently in multiple different species and geographic locations. Despite this vast diversity of structures a common feature of most AFPs is the ability of one face of the protein, termed the ice-binding surface (IBS), to non-covalently bind to ice and inhibit ice growth. The IBS composition and the mechanism of ice inhibition varies between AFPs, however the IBS is generally comprised of small, hydrophobic amino acids that order water molecules in an ice-like array on their ice-binding surface. This array of ordered-waters then hydrogen binds to the ice crystal, anchoring the AFP to the ice surface and inhibiting expansion of the ice (Figure 2).</P><FIGURE><FIGCAPTION>Figure 2. <STRONG>Diagram describing the possible mechanisms of AFP ice binding.</STRONG> The first column <STRONG>A,</STRONG> shows the Hydrogen Bonding Hypothesis: threonines at the IBS hydrogen-bond with the upper ice surface and move deeper into the ice. <STRONG>B</STRONG> describes the Hydrophobic effect in AFP binding, where methyl groups displace ice-like waters on the ice surface. <STRONG> C </STRONG> is the Anchored Clathrate Hypothesis. Figure taken from Davies (2014), Figure 6<SUP>2</SUP>.</FIGCAPTION></FIGURE><H2>AFP Function</H2><P>AFPs are found in organisms such as the ocean pout, where they act to inhibit growth of ice crystals below a solution's freezing point. Ice inhibition occurs when multiple AFPs bind to the same ice crystal, and small curvatures are created along the ice surface. It is then energetically unfavorable for more water molecules to bind to the ice, thereby inhibiting growth of the ice crystal.Therefore the presence of AFPs requires the temperature of a solution to be below the original melting-freezing point for further ice growth. Because AFPs lower only the freezing point, a gap between the melting and freezing points is created, which is known as the thermal hysteresis (TH) gap. A TH gap can thus be used to identify novel anti-freeze proteins, and the size of the TH gap can provide quantitative assessment of known AFP activity. </P><P>Like their structures the TH activity, or functionality, of AFPs varies greatly. For both components of our project, QGEM chose to study a moderately active Type III AFP from the ocean pout fish. A Type III AFP was selected because of its globular structure, ideal for circularization, and its well-characterized ice-binding surface. The IBS of the ocean pout AFP spans two faces of the protein and is composed of mainly alanine and threonine amino acids. Additionally, the Type III AFP was chosen because of the close proximity of the N and C termini of the protein, which made it an ideal candidate for protein circularization. </P><H2>Applications</H2><P>Ice growth inhibition by AFPs is already being applied commercially and experimentally in various industries.  Commercially, AFPs are used in frozen foods such as ice creams to maintain a smooth creamy texture.  However, scientists have focused on optimizing AFP function through synthetic biology and protein engineering for a variety of applications.  The oil and gas industry is investigating the use of AFPs, some of which have been found to inhibit the formation of gas hydrates which cause safety and operational challenges.  There are also attempts to genetically alter frost sensitive crops to produce AFPs.  Numerous experiments are also applying AFPs for cryopreservation of cells and organs.  Our project focuses on improving these current experiments using AFPs in cryopreservation.</P><H2>References</H2><P>1. image from <A href="http://brasdorpreservation.ca/bras-dor-lakes/featured-page-2/">http://brasdorpreservation.ca/bras-dor-lakes/featured-page-2/</A></P><P>2. Davies, P.L. (2014). &quot;Ice-binding proteins: a remarkable diversity of structures for stopping and starting ice growth&quot;. Trends in Biochemicala Sciences. 39(11):548-555.</P></DIV><DIV id="EKcoilinfo"><H1>THE E/K COIL SYSTEM</H1><P align="center"><EM>The soul mate story of heterodimer coils.</EM></P><H2>Coiled-Coil Motifs</H2><P>Coiled-coils are a naturally occurring phenomenon. Consisting of multiple alpha-helices, these protein structures are identified as left-handed helices that interact non-covalently. First identified in 1972 by Sodex et al. as a series of hydrophobic repeats within tropomyosin<SUP>1</SUP>. Similar patterns were also located in fibrous proteins<SUP>2</SUP> and intermediate filaments<SUP>3</SUP>. </P><P>Interactions can be engineered to introduce highly specific connections between helical sequences. While many naturally occurring coiled-coils allow non-specific interactions between coils, engineered constructs improve the affinity and specificity of these interactions, creating motifs with strengths nearing that of covalent contacts. </P><H2>Engineered Interactions</H2><P>One such system is the E/K coil motif characterized by Lindhout et al in 2004<SUP>4</SUP>. This motif is composed of a seven amino acid repeat in the pattern of:</P><UL><LI>a-b-c-d-e-f-g</LI></UL><P>Here, 'a' and 'd' are hydrophobic amino acids and 'e' and 'g' are charged residues. The remaining amino acids can be modified to alter the stability, solubility and external interactions of the system<SUP>5</SUP>. The charged residues control the specificity of the interaction between the coils, stabilizing the closely packed hydrophobic core that forms between them <SUP>6, 7</SUP>(Crick 1953, McLachlan and Stewart 1975). </P><P>The E and K coils contain negatively charged glutamic acid and positively residues charged lysine at the 'e' and 'g' positions respectively. The electrostatic interactions promote specificity between the coils and lead to the formation of heterodimers. These systems introduce a wide range of potential applications in biotechnology.</P><H2>References</H2><P>1. Sodek J, Hodges R, Smillie L, and Jurasek L. (1972). &quot;Amino-acid sequence of rabbit skeletal tropomyosin and its coiled-coil structure&quot;. Proceedings of the National Academy of Sciences of the United States of America. 69(12):3800-3804.</P><P>2. Titus M. (1993). &quot;Myosins&quot;. Current Opinion in Cell Biology. 5(1):77-78</P><P>3. Stewart M. (1993). &quot;Intermediate filament structure and assembly&quot;. Current Opinion in Cell Biology. 5(1):3-4.</P><P>4. Lindhout et al. (2004).&quot;NMR solution structure of a highly stable de novo heterodimeric coild-coil.&quot; Biopolymers. 75:367-375.</P><P>5. Dutta K, Alexandrov A, Huang H, and Pascal S. (2001). &quot;pH-induced folding of an apoptotic coiled coil&quot;. Protein Science. 10(12):2533.</P><P>6. Crick F. (1953). &quot;The packing of alpha-helices: Simple coiled-coils&quot;. Acta Crystallographica. 6(8-9):690-691,695.</P><P>7. McLachlan A and Stewart M. (1975). &quot;Tropomyosin coiled-coil interactions: Evidence for an unstaggered structure&quot;. Journal of Molecular Biology. 98(2):295.</P><H1>SELF-ASSEMBLING PROTEIN SCAFFOLDS</H1><P>The formation of multi-protein units in nature have long been studied and attributed to specific and entropically favourable interactions that occur at protein-protein interfaces. Such units serve as a basis for new waves of protein engineering and the production of self-assembling multimers of designed size and conformation.</P><P>Protein scaffold design involves the computational re-engineering of the interface interactions of naturally occurring trimer subunits. These can be created to form larger congregates whose strength compares to the natural units. Recent work in this area has helped bridge the gap between computational design of proteins and production of synthetic products. The structures created by the Baker and Yeates lab groups<SUP>1</SUP> are among the most accurate examples of theoretical design and actual production and assembly. These units serve as a critical component of this year's project as we try to strategically attach proteins to the scaffold to increase local concentration and alignment of active ice binding surfaces</P><H2>References</H2><P>1. King, NP et. al. (2012). &quot;Computational Design of Self-Assembling Protein Nanomaterials with Atomic Level Accuracy&quot;. Science. 336:1171-1174.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>