<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SSTi-SZGD_Degradation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SSTi-SZGD/Degradation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>SSTi-SZGD---Degradation</TITLE></P><DIV class="SSTi-SZGD" onselectstart="return false;" unselectable="on" style=" -moz-user-select: none;"><DIV class="header a-fadeinT"><DIV class="logo"><SPAN>SSTi-SZGD</SPAN></DIV><NAV class="nav"><UL><LI class="Home mainnav"><A href="https://2017.igem.org/Team:SSTi-SZGD"><SPAN>Home</SPAN></A></LI><LI class="Project mainnav hvr-overline-from-center"><SPAN>Project</SPAN></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Expression">Expression</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Degradation">Degradation</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Experiments">Experiments</A></LI></UL><LI class="Application mainnav hvr-overline-from-center"><SPAN>Application</SPAN></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Business_Plan">Business Plan</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Hardware">Hardware</A></LI><LI class="Human_Practices mainnav hvr-overline-from-center"><SPAN>Human Practices</SPAN></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/HP/Silver">Summary</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/HP/Gold_Integrated">Intergrated HP</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/HP/Outreach">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Collaborations">Collaboration</A></LI><LI class="Achievement mainnav hvr-overline-from-center"><SPAN>Achievement</SPAN></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Medal">Medal</A></LI><LI class="Team mainnav hvr-overline-from-center"><SPAN>Team</SPAN></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:SSTi-SZGD/Attributions">Attributions</A></LI></NAV></DIV></DIV><DIV class="Content"><DIV class="Pesticide"><DIV class="left"><P class="Title">1 Carbendazim Pesticide Esterase</P></DIV><DIV class="right"><P class="Title">2 Organophosphate Pesticide Hydrolase</P></DIV><P class="section">
				We wanted to make a good use of the LightOFF system to solve the environmental problem regards to pesticide residue. Since pesticide degradation enzymes are produced naturally by microorganisms, we proposed to produce hydrolases degrade specific pesticide residues from genes heterologously expressed in E. coli, using LightOFF system. These engineered bacterial cells thus serve as an enzyme-production ‘super factory’.
				</P></DIV><DIV class="Mhel"><P class="Title">Mhel-- Carbendazim Pesticide Esterase</P><DIV class="left"><P class="section">
					Carbendazim (methyl-1H-benzimidazol-2-ylcarbamate, or MBC) is a systemic benzimidazole fungicide that is used in many countries around the world to control a broad range of fungal diseases in agricultural crops. MBC also is the hydrolytic product and active component of some other widely used systemic fungicides, such as benomyl and thiophanate methyl (1,2). MBC is quite stable in soil and water, which in turn can lead to the contamination of foodstuffs. MBC is a suspected mutagen, carcinogen, and endocrine disruptor, and its use is tightly controlled by regulatory bodies in many countries (3).
					</P></DIV><DIV class="right"><P class="section">
					     Although MBC itself in soil and water can be chemically and physically degraded slowly, and researchers have used microorganisms, such as Mycobacterium sp., Nocardioides sp. or Rhodococcus erythropolis to degrade MBC in soil (4). In recent years, there have been increasing reports on using recombinant Escherichia coli to express MBC degrading enzyme as E. coli is capable of growing to a much higher density than those native microorganisms, and has advantage over the development of large-scale detoxification processes.
					</P></DIV><P class="section">
				A gene called mheI (for MBC-hydrolyzing enzyme), isolated from Mycobacterium sp. SD-4., encodes MBC-hydrolyzing esterase (MHE) that degrades MBC pesticide (5). Degradation of MBC is achieved by hydrolyzing MBC to 2-aminobenzimidazole (2-AB), which is then converted to benzimidazole or 2-hydroxybenzimidazole (2-HB). The conversion of 2-AB was inhibited by NH4NO3. The benzene ring of 2-HB was further opened through meta catechol cleavage. MHE is responsible for carrying out the first step detoxification (MBC to 2-AB), without the need of any co-factor. 2-AB is a substantially less toxic substance compare to MBC. MHE is a serine hydrolase of 242 amino acid residues and has a molecular size of 25-27 KDa (6). In nature, MHE is extracellular and could diffuse away from the cells. Contaminated sites often contain insoluble crystals of MBC because of its low aqueous solubility (8 ppm). MHE hydrolyzes the insoluble MBC into the much more soluble 2-AB, which could again diffuse back to the cells and be utilized as a carbon and energy source (7).
				</P><DIV class="right"><P class="Headline">Figure 1:</P><P class="section">
					    Illustration of carbendazim degradation pathway: first degraded to 2-aminobenzimidazole by the hydrolytic enzyme, and then the benzimidazole ring was opened to produce monomethyl permanganic (benzimidazole cyanate) and then converted to 2-hydroxybenzimidazole. This enzymatic pathway substantially reduces the toxicity of MBC. Adapted from (2)
					</P></DIV></DIV><DIV class="Results"><P class="Title">Design＆Results</P><P class="section">
					 We intend to test whether mheI can be efficiently expressed using lightOFF system. Verification of lightOFF vector with reporter gene mCherry has proven to be workable. Instead of replacing mCherry with mheI, we made a fusion protein of them to indicate the take-up of mheI gene.
				</P><DIV class="left"><P class="Headline">1. Construction of pLEV1(408)-mheI-mcherry vector</P><P class="section">
					MheI (GenBank no. KX698097.2) was commercially synthesized and subcloned into LightOFF vector pLEV1(408) at the Psil /BglII restriction sites. Pink colonies indicated the expression of mCherry. Colony PCR and Sanger Sequencing further confirmed that mheI was successfully cloned in the vector and transformed in E. coli (Figure 2).
					</P></DIV><DIV class="right"><P class="Headline">Figure 2:</P><P class="section">
     				Illustration of recombinant mheI plasmid, pLEV1(408)-mheI-mCherry (Left), and agarose gel electrophoresis of colony PCR amplifying DNA section harboring ColE promoter and mheI gene in the plasmid using primer pair ColE-F and mheI-R (right). The expected product size is 868 bp.
					</P></DIV><P class="Headline">2. MheI protein expression--SDS-PAGE &amp; ELISA test</P><P class="section">
				MheI gene expression was induced in darkness by wrapping in aluminum foil and grew at 37℃ for up to 18 h until pink cell culture obtained. Total protein was extracted by ultrasonic extraction using Beibo E.coli Total Protein Extraction Kit. Bradford protein assay suggested that the concentration of total protein was 229.92 μg/ ml. Total cytoplasmic protein fractions from induced cells and non-induced cells harboring pLEV1(408)-mheI-mCherry were analyzed by SDS-Polyacrylamide gel electrophoresis (PAGE) and ELISA.
				</P><P class="section">
				SDS-PAGE analysis showed that MHE was expressed in total protein fraction, and absence in cell lysates and control strain (Figure 3). Furthermore, ELISA studies, with anti-MHE serum, showed a positive reaction (yellow color) with total protein fraction, and negative reactions with cell lysate and control strain (Figure 4). Together these results suggested that MHE was successfully expressed using lightOFF system.
				</P><DIV class="left"><DIV class="left"><P class="Headline">Figure 3:</P><P class="section">
						SDS-PAGE analyses of MHE proteins in total cytoplasmic protein fractions from induced cells and non-induced cells harboring pLEV1(408)-mheI-mCherry. A 12% SDS-PAGE gel was used. 
						</P></DIV></DIV><DIV class="right"><P class="Headline">Figure 4:</P><P class="section">
     					ELISA studies of MHE protein expression in total cytoplasmic protein fractions from induced cells (MheI dark) and non-induced cells (mhei light) harboring pLEV1(408)-mheI-mCherry. The absorbance (OD) was measured at 450nm using a microplate reader. Data are mean values+/-standard derivations from three replicates. 
					</P></DIV><DIV class="left"><P class="Headline"> 3. Enzyme activity test of MHE</P><P class="section">
					Enzymatic activity of MHE was examined at 30℃for 1 h in 5ml of 20mM Tris-HCl buffer solution (pH 7.4) containing 40 μM of MBC as the substrate. The reaction was initiated by the addition of MHE crude protein solution to a final concentration of 0.1 μg/ml, and was stopped by equal volume of ethyl acetate. The organic phase was collected and MHE activity was measured by examining the absorbance of MBC at 287 nm on UV spectrophotometer (Hitachi, Japan). One unit of enzyme activity was defined as the amount of enzyme required to hydrolyze 1 μmol of MBC to 2-AB at 30℃ within 1 min.
					These preliminary results showed that total protein expressed from pLEV1(408)-mheI-mCherry was able to hydrolyze MBC substrate, while total protein expressed from pLEV1(408) vector did not process MBC degradation ability. Therefore it is valid to suggest that MHE expressed using LightOFF system retains the enzyme function.
					</P></DIV><DIV class="right Adj1"><P class="Headline">Figure 5:</P><P class="section">
 					Specific MHE enzyme activities of total protein fraction (mheI dark) and control protein fractions (inactive mheI, mheI light, LightOFF). The activity was assayed with carbendazim (MBC) as substrate. Data are mean values+/-standard derivations from three replicates.
					</P></DIV><P class="Headline">OPH--Organophosphate Pesticide Hydrolase</P><P class="section">
				Synthetic organophosphate compounds are a group of highly toxic chemicals widely used in many pesticides (Paraoxon, Parathion, Coumaphos, and Diazinon) and chemical nerve agents (Sarin and Soman), accounting for 38% of total pesticides used globally [7]. OP pesticides are acetylcholinesterase (AChE) inhibitors, and poisoning in humans have been reported extensively [8]. The increasing concern about their safety and the widespread use of OPs in agriculture has promoted the development of effective and safe methods for OP detoxification (Figure 6), among them, enzymatic degradation has attracted many attentions due to its specificity, convenient and safe application compared to chemical and physical methods.
				</P><P class="Headline">Figure 6:</P><P class="section">
				Degradation pathway for OP pesticides parathion, methyl parathion and paraoxon. Primary metabolites include dimethylthiophosphate(methyl parathion), diethylthiophosphate (parathion) or diethylphosphate (paraoxon), and p-nitrophenol. Further degradation may be possible through other metabolic pathways dependent on the strain(s) involved. Bd-opd, B. diminuta GM opd gene; Sf-opd, S. fuliginis opd gene. Adapted from (8)
				</P><CENTER><P>Table 1: Summary of characteristics for representative OP degradation genes, adapted from (10)</P></CENTER><P class="section Adj2">
				Organophosphorus hydrolase (OPH), secreted from various Agrobacterium, Pseudomonas and Flavobacterium sp. strains, is a homodimeric phosphotriesterase that hydrolyzes a wide range of OPs with high specificity (9). Hydrolysis reduces OPs toxicity by many folds of magnitude. In recent years, using recombinant Escherichia coli to express OPH has attracted increasing attention as E. coli can grow to much higher densities, and recombinant gene expression can be induced externally. However, enzyme secretion efficiency is reduced by cell membrane diffusion barrier of E. coli, and practical application of enzymatic degradation is hampered by high cost of enzyme purification. One feasible solution is to secrete target protein in the periplasmic space of the cells to reduce the substrate diffusion barrier so that these cells can be used as live biocatalysis (8).
				</P><DIV class="left"><P class="section">
					There are a number protein export pathways available in E. coli, such as Ser translocase, Thy translocase, etc. Among them, the twin-arginine translocation (Tat) system has an intrinsic advantage of being able to transport folded proteins across the cytoplasmic membrane and without the requirement ATP hydrolysis. This helps retaining protein function in periplasmic space (9). In addition, the Tat pathway has been successfully used for periplasmic secretion of several foreign proteins such as cofactor-containing, multimeric, and disulfide-containing proteins (9, 10).
					</P></DIV><DIV class="right"><P class="section">
					Proteins are targeted to a membrane-embedded Tat translocase by specialized N-terminal twin-arginine signal peptides bearing a consensus motif of SRRxFLK (11,12). In E. coli, the Tat translocase consists of the TatA, TatB, and TatC proteins. These three classes of membrane proteins form two types of high-molecular-weight complexes, including the TatBC signal recognition complex and TatA transport channel complex (13-15). The TatBC complex binds a Tat substrate by the specific recognition for the twin-arginine motif, while TatA complex forms a channel through which substrates are translocated across the cytoplasmic membrane.
					</P><P class="Headline">Figure 7:</P><P class="section">
					Schematic diagram of TorA-OPH fusion protein. The TorA signal peptide is followed by four amino acids residues of the mature TorA protein fused directly to the OPH domain. The twin-arginine motif of ‘SRRRFLA’ is underlined.
					</P></DIV><P class="section">
				In the current study, opdA gene (<A href="http://parts.igem.org/Part:BBa_K215090" target="_blank">BBa_K215090</A>), derived from an organophosphorus-degrading bacterium a strain of Agrobacterium, was expressed in recombinant E. coli DH5a. In order to develop efficient whole cell biocatalysts, twin-arginine signal peptide (TorA) of E.coli trimethylamine N-oxide (TMAO) reductase was synthesized to the N-terminus of opdA gene (Figure 7). TMAO reductase is a periplasmic enzyme that catalyzes reduction of TMAO to trimethylamine, and functions as a component of the anaerobic respiratory chain which provides energy for bacterial cell growth (15). Yang, et al (2009) (16) demonstrated the effectiveness of OPH exporting to periplasm using T7-driven pET vector. With IPTG induction, however, may limit the feasibility of live cells biocatalysts in the field. In our project, we cloned TorA-opdA conjugate into LightOFF vector thus the expression of fusion gene can be induced by darkness in a controlled manner. Therefore ‘live biocatalysts’ can be used directly for large-scale detoxification of OPs in the field, purely by visible light regulation.  
				</P><P class="SubTitle">Design&amp;Results</P><P class="Headline">1. Construction of pLEV1(408)-TorA-opdA vector</P><P class="section">
				We constructed recombinant plasmid pLEVI(408)-TorA-opdA, size of 6227bp, by commercial synthesis provided by Genscript, and sub cloned in pLEV1(408) vector at Psil/BglII restriction sites. pLEVI(408)-TorA-opdA vector was transformed into E. coil DH5a cells, and were selected by Streptomycin resistance (80 μg/ml). Colony PCR and Sanger Sequencing of the extracted plasmids were performed to confirm the uptake of TorA-opdA gene (Figures 8).
				</P><P class="Headline">Figure 8:</P><P class="section">
				Illustration of Recombinant opdA plasmid, pLEV1(408)-TorA-opdA (Left). The agarose gel electrophoresis of colony PCR amplifying DNA section harboring ColE promoter and opdA gene in pLEV1(408)-TorA-opdA using primer pair ColE-F and OpdA-R (Right, Lane 1), the expected product size is 455 bp.
				</P><P class="Headline">2. OPH expression in periplasmic----SDS-PAGE &amp; ELISA test</P><DIV class="left"><P class="section">
					For successful use of Tat translocase for periplasmic secretion of OPH protein, we verified the location of OPH by isolating periplasmic fraction and cytoplasmic fraction from cell expressing periplasmic OPH. opdA gene expression was induced in darkness by wrapping in aluminum foil and grew at 37℃ for up to 18 h. Total periplasmic protein was extracted using Arginine Extraction method (17) (refer to Experiments).
					</P></DIV><DIV class="right"><P class="section">
					Cell culture was harvest and 1 gram of cells were added 10 ml arginine buffer at 0.4mol/L, PH8.0. After extracting on ice for 45 min, cells were centrifuged to obtain supernatant as periplasmic fraction. Cytoplasmic protein fraction was extracted by ultrasonic extraction using Beibo E.coli Total Protein Extraction Kit. Bradford protein assay suggested that the concentration of periplasmic fraction was 224.165 μg/mL, and cytoplasmic fraction was 341.890 μg/mL. Periplamic fraction, cytoplasmic fraction, whole cell, and periplasmic fraction of pLEV1 (408)-mCherry vector (LightOFF) were analyzed by SDS-PAGE and ELISA.
					</P></DIV><P class="section">
				SDS-PAGE results showed that mature OPH (size of 33 KDa) was found in periplasmic fraction, and was absent in cytoplasmic fraction and control strain, indicating a correct localization of OPH (Figure10). furthermore, OPH expression was also detected in whole cell sample, indicating a possible usage as whole cell biocatalyst (Figure 9).
				</P><DIV class="left"><P class="Headline">Figure 9:</P><P class="section">
					SDS-PAGE analyses of OPH proteins in periplasmic fraction, periplasmic fraction of control strain, cytoplasmic fraction, and whole cell. A 12% SDS-PAGE gel was used. Abbreviations: P, periplasmic fraction; L, periplasmic fraction of pLEV1(408); C, cytoplasmic fraction; W, whole cell; M, protein molecular weight marker.
					</P></DIV><P class="section">
				We also conducted ELISA studies, using anti-OPH serum, which showed a positive reaction with periplasmic fraction, and almost negative reactions with cytoplasmic fraction and control strain. There was also a positive reaction with whole cell sample, although the concentration dropped almost 50% compared to periplasmic fraction (Figure 10). Altogether these results suggested that OPH was successfully expressed using light-repressed system and correctly exported to the periplasm space.
				</P><P class="Headline">Figure 10:</P><P class="section">
				ELISA studies of OPH protein expression in periplasmic fraction, cytoplasmic fraction, whole cell, and periplasmic fraction of control strain. The absorbance (OD) was measured at nm using a microplate reader. Abbreviations: LF, light off periplasmic fraction; periplasmic fraction, whole cell and cytoplasmic fraction. The absorbance (OD) was measured at 450nm using a microplate reader. Data are mean values+/-standard derivations from three replicates.
				</P><P class="Headline">3. Enzyme activity studies of OPH</P><DIV class="left"><P class="section">
					We compared OPH activities for periplasmic fraction, cytoplasmic fraction, whole cell, and periplasmic fraction of pLEV1(408)-mCherry. OPH activity assay mixtures (1 mL, 3% methanol) contained 5 μl paraoxon (added from a 10 mg/mL methanol stock solution), 870 μL of 50 mM Tris-HCl buffer (pH 7.4), and 100 μL of cells (17,18). The enzyme activity was measured using a UV/VIS spectrophotometer at 37℃ by monitoring the increases of linear optical density over time at 405 nm as parathion was hydrolyzed to p-nitrophenol (ε405=17,700 M-1 cm-1). Activities were expressed as units (1 μmol of p-nitrophenol formed per minute) per ml (total reaction volume). 
					</P></DIV><DIV class="right"><P class="section">
					Results showed that periplasmic fraction contained much higher OPH activity than the other samples, whole cell had relatively higher OPH activity, but was lower than that of periplasmic fraction by around 30%. In comparison, cytoplasmic fraction and periplasmic fraction of LightOFF control strain had much lower, almost basal level, OPH activities (Figure 11). 
					</P></DIV><P class="Headline">Figure 11:</P><P class="section Adj2">
				Specific OPH activities of whole cell, periplasmic fraction, cytoplasmic fraction and control periplasmic fraction (lightOFF). The activity was assayed with paraoxon as substrate. Data are mean values+/-standard derivations from three replicates.In addition, increased concentration of periplasmic fraction showed increased OPH activities. As increased enzyme solution was added, more paraoxon was hydrolyzed and resulted to increasing formation of p-nitrophenols (Figure 12).
				</P><P class="section">
				In addition, increased concentration of periplasmic fraction showed increased OPH activities. As increased enzyme solution was added, more paraoxon was hydrolyzed and resulted to increasing formation of p-nitrophenols (Figure 13). 
				</P><P class="Headline">Figure 12:</P><P class="section Adj2">
				Illustration of OPH activity Vs various concentrations of periplasmic fractions from OPH-expressed cell strain (black) and control strain (Red). The activity was assayed with paraoxon as substrate. Data are mean values+/-standard derivations from three replicates.
				</P><P class="Headline">4.Whole cell OPH activity---Stability study</P><DIV class="left"><P class="section">
					We also assayed the remaining whole cell OPH activities under resting cell conditions to investigate OPH stability. E. coli harboring pLEV1(408)-TorA-opdA was cultured in 50 ml of LB medium supplemented with 80 μg/ml Streptomycin, and induced in darkness at 37℃ for 16-20 hr. Cells were harvested and resuspended in equal volume of 1x phosphate-buffered saline (PBS) and incubated in a shaker at 30℃ . Over the six-day period, 1 ml of sample was removed everyday to analyze OPH activity as described above.
					</P></DIV><DIV class="right"><P class="section">
					The results showed that whole cell activity could be retained for several days in periplasmic expressing OPH cells (Figure 13). Interestingly, whole cell OPH activity increased on the 2nd day of incubation, which might be from time gap for periplasmic translocation of cytoplasmic premature OPH. This further translocated-functional OPH might lead to increased apparent whole cell activity during early resting cell incubation. Previous studies also showed a longer time (up to 14 days) of stable whole cell OPH activity (8). We could optimize the experimental conditions to increase the stability period of OPH activity. 
					</P></DIV><P class="Headline">Figure 13:</P><P class="section Adj2">
				Whole cell OPH activity in suspended E. coli culture expressing periplasmic OPH. The activity was assayed with paraoxon as substrate. Data are mean values+/-standard derivations from three replicates.
				</P><DIV class="left"><P class="Headline Adj3">Conclusion: </P><P class="section">
					These results suggest that :
					1) lightOFF system is efficient and effective in producing functional degrading hydrolases, and it is a valuable candidate for industrial production of enzymes, 
					2) periplasmic expressing OPH strain can potentially be used as ‘live biocatalysts’ due to its effect in OP compound detoxificiation as well as long-stand stability, 
					3) Biosatety concern associated with live biocatalysts can be monitored by promoting cell death via light-inducible ROS release in supernova system.
					</P></DIV><DIV class="right"><P class="Headline Adj3">Future studies:</P><P class="section">
					Given more time, we will first test whether co-expression of target gene and supernova does not interfere with normal cell growth. Then we will study enzymatic activity in soil sample as well as perform field tests on the degradation efficiency of our hydrolases over a wider range of temperature, humidity and pH conditions. Furthermore, in order to increase degradation efficiency, we would optimize the promoter and regulatory sequence of lightOFF vector, as well as introducing mutants to target genes. 
					</P></DIV><P class="Headline">Reference:</P><P class="section">
				[1]	Mazellier, P., Leroy, E., De Laat, J. &amp; Legube, B. 2003.Degradation of carbendazim by UV/H2O2 investigated by kinetic modelling. Environ. Chem. Lett. 168–72.
				[2]	Zhang, X.J., Huang, Y. J., Harvey, P.R., Li, H.M. Ren, Y., Li, J.S., Wang, J.N. &amp; Yang, H.T.  2013. Isolation and characterization of carbendazim-degrading rhodococcus erythropolis djl-11. PLoS One 8.
				[3]	Kim, O.S. Cho, Y. J. Lee, K., Yoon, S.H., Kim M, Na, H., Park, S.C., Jeon, Y.S., Lee, J.H., Yi, H. Won, S. &amp; Chun, J. 2012. Introducing EzTaxon-e: a prokaryotic 16S rRNA gene sequence database with phylotypes that represent uncultured species. Int. J.Syst. Evolut. Microbiol. 62,716–721. 
				[4]	Aharonson, N. and J. Katan. 1993. Delayed and enhanced biodegradation of soil-applied diphenamid, carbendazim, and aldicarb. Arch. Insect Biochem Physiol. 22,451–466.
				[5]	Alcalde, M., M. Ferrer, F. J. Plou, and A. Ballesteros. 2006. Environmental biocatalysis: from remediation with enzymes to novel green processes. Trends Biotechnol. 24,281–287.
				[6]	Arora, S., I. Mukherjee, and T. P. Trivedi. 2008. Determination of pesticide residue in soil, water and grain from IPM and non-IPM field trials of rice. Bull. Environ. Contam. Toxicol. 81,373–376.  
				[7] Xu, J.L., Gu, X.Y., Shen, B., Wang, Z.C., Wang, K., Li, S.P. 2006. Isolation and characterization of a carbendazim-degrading Rhodococcus sp. djl-6. Curr. Microbiol. 53 (1), 72–76.
				[8] Kang, D. G., Lim, G-B. &amp; Cha, H. J, 2005. Functional periplasmic secretion of organophosphorous hydrolase using the twin-arginine translocation pathway in Escherichia coli. Journal of Biotechnology, 118, 379-385.
				[9] Sogorb, M. A.; Vilanova, E. &amp; Carrera, V. 2004. Future application of phosphotriesterases in the prophylaxis and treatment of organophosphorus insecticide and nerve agent poisoning. Toxicol. Lett.151, 219–233.
				[10] Mulbry, W.W., Karns, J.S., 1989. Parathion hydrolase specified by the Flavobacterium opd gene: relationship between the gene andprotein. J. Bacteriol. 171, 6740–6746.
				[11] Lee, P. A.; Tullman-Ercek, D. &amp; Georgiou, G. 2006. The bacterial twin-arginine translocation pathway. Annu. Rev. Microbiol. 60, 373–395.
				[12] Berks, B. C. 1996. A common export pathway for proteins binding complex redox cofactors. Mol. Microbiol. 22, 393–404.
				[13] Chaddock, A. M.; Mant, A.; Karnauchov, I.; Brink, S.; Herrmann, R. G.; Klosgen, R. B.; Robinson, C. 1995. A new type of signal peptide: ::central role of a twin-arginine motif in transfer signals for theΔpH-dependent thylakoidal protein translocase. EMBO J. 14,2715–2722.
				[14] Gohlke, U.; Pullan, L.; McDevitt, C. A.; Porcelli, I.; de Leeuw, E.; Palmer, T.; Saibil, H. R.; Berks, B. C. 2005. The TatA component of the twinarginine protein transport system forms channel complexes of variable diameter. Proc. Natl. Acad. Sci. U.S.A. 02, 10482–10486.
				[15] Alami, M.; Luke, I.; Deitermann, S.; Eisner, G.; Koch, H. G.; Brunner, J. &amp; Muller, M. 2003. Differential interactions between a twin-arginine signal peptide and its translocase in Escherichia coli. Mol. Cell, 12, 937–946.
				[16] Yang, C., Fredual, F. &amp; Qiao, C. 2009. Export of Methyl Parathion Hydrolase to the Periplasm by the Twin-Arginine Translocation Pathway in Escherichia coli. J. Agric. Food Chem. 57, 8901–8905 8901.
				[17] LI, Zhen-guo, Xu, Ming-bo, NIU, Gang, YAO, Wen-bing. 2011. Study on extraction of periplasmic proteins in E. coil. Journal of Biology. 28,95-97.
				[18] Bolhuis, A.; Mathers, J. E.; Thomas, J. D.; Barrett, C. M.; Robinson,C. 2001. TatB and TatC form a functional and structural unit of the twinarginine translocase from Escherichia coli. J. Biol. Chem. 276,20213–20219.</P></DIV></DIV><FOOTER class="footer_box"><DIV class="footer"><DIV class="copyright"><P><SPAN class="left">
							A product for the degradation of soil pesticide residues
						</SPAN><SPAN class="right">
							Copyright © 2017 Lucky      power by  iGEM Team:SSTi-SZGD
						</SPAN></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>