Team:SSTi-SZGD/Expression
SSTi-SZGD---Expression
SSTi-SZGD
Home
Project
Description
Expression
Degradation
Applied Design
InterLab
Safety
Notebook
Experiments
Application
Business Plan
Software
Hardware
Human Practices
Summary
Intergrated HP
Outreach
Collaboration
Achievement
Parts
Medal
Team
Team
Attributions
Single-component light off system
Light-regulated gene expression systems provide transient, non-invasive and reversible means to control biological processes, and often serve as an alternative to chemical-induced systems, i.e. IPTG in T7 system (1). In bacterial cells, many photoreceptor-based light sensitive system are two-component regulatory systems that are respond to blue, green or red light. These two-component systems, however, have drawbacks such as complex systems, co-factors dependent, low induction rate, etc. (1,2).
In order to efficiently produce enzymes for pesticide residue degradation, and further move onto generate ‘live cell biocatalysts’, we wished to use a system with non-invasive means of recombinant gene expression, as well as being simple, easy to manipulate and with high induction efficiency. During the process of screening previous iGEM works, we came across the Luminesensor ( BBa_K819007 ), a single component light-regulated system. By making improvement of Luminesensor with technical guidance from Prof. Yang Yi of Eastern China University (ECU) (3), we obtained a light-repressed system containing a single light repressor LexA-VVD (Figure 1), and a ColE promoter containing a SOS operator sequence for LexA binding, regulates the expression of target genes (Figure 2). This system is referred to LightOFF system, and it was shown to have comparable induction efficiency to chemical-regulated systems, i.e. T7-driven pET system (Figure 3) (3).
Figure 1:
Schematic diagram of LightOFF system. A blue light sensor VIVID (VVD), which is a small light-oxygen-voltage (LOV) domain-containing protein from Neurospora crassa, was fused to the C-terminus of DNA-binding domain of LexA repressor of the Escherichia coli SOS regulon to create a light switchable system. Adapted from Chen et al. (2016).
Figure 2:
Schematic diagram of LightOFF system. When the system is exposed to blue light, a cysteine-flavin adduct is formed in the VVD domain, causing conformational changes of the domain and the subsequent dimerization of the fusion protein. The activated protein dimer binds its cognate operator sequence of ColE promoter and represses the promoter activity. Whereas in dark, dimerization does not occur, so the promoter is not repressed and gene expression proceeds.Adapted from Chen et al. (2016)
Figure 3:
Comparison of gene expression regulation by Lightoff or chemical-induced systems (adapted from 3). LightOFF system in JM109 (DE3, ΔsulA, ΔLexA) cells, or pET system in BL21 (DE3) pLysS (pET system1) or BL21(DE3) (pET system2) cells and expression were analyzed both non-induced and induced. adapted from Chen et al. (2016)
1.Experimental Aim:
In order to apply the LightOFF system, we firstly test whether it could work in a wild-type E. coli strain by studying the expression of a reporter gene.   Experimental Aim: Verify the function and sensitivity of LightOFF system in wild-type E. coli DH5a using a reporter gene mCherry.
2.Progress and Results:
(1)  Construction of LightOFF vector
LexA and VVD genes were amplified from JM109 cell genome and pGAVPO respectively, and then fused to the constitutive promoter ( BBa_J23116 ) and rrnB T1 terminator ( BBa_B0010 ). The construct was then cloned into pCDFDuet1 vector with the promoter being removed. The resultant vector is named pLEV1. A ColE promoter, derived from E. coli ColE1 plasmid, was synthesized by Generay Biotech Co. and fused to a reporter gene mCherry and rrnB T1 terminator. This construct was cloned into pLEV1 vector to result to pLEV1-mCherry vector. Site-directed mutagenesis, using Takara MutanBEST kit, on VVD protein with 174V and M165I double mutations to generate a pLEV1(408) mutant (1).
Figure 4:
The agarose gel electrophoresis of enzyme digestion of pLEV1(408)-mCherry using EcoR1 and XbaI. The digested fragments are 4450bp and 1480bp respectively (Lane 1). uncut plasmid has a size of 5800bp (Lane 2).
Result :
Plasmid construction was successful and verified by restriction digestion, colony PCR with ColE-F (5’-CCGCTGCATTTTCCCTGTC-3’) and LEV1-R (5’-GCGCAGCGTTTGGGGTTC-3’) and Sanger Sequencing. We also cloned LEV1 fusion protein and ColE promoter into the assembly standard BioBrick pSB1C3 for parts submission verifie.
Figure 5:
The agarose gel electrophoresis of colony PCR amplifying DNA section harboring ColE promoter and LEV1 gene in pLEV1(408)-mcherry using primer pair ColE-F and LEV1-R, and amplying ColE promoter using primer pair ColE-F and ColE-R. The expected PCR product sizes are 1908 bp and 180 bp respectively.
(2) Reporter gene expression
pLEV1(408)-mCherry were transformed into E. coli DH5a and grew in LB medium+Streptomycin (80 μg/ml). mCherry expression was induced in darkness by wrapping the cultural flask in aluminum foil at 37℃ for 14 hours. Red colonies appeared on the plate Indicated that mCherry was expressed. A cell growth study showed that induced cells had increased level fluorescence intensity, while non-induced cells had basal level of fluorescence (Figure 6), suggesting that pLEV1(408)-mCherry has high induction efficiency and low leakage rate, as well as being able to express in wild-type E. coli strain. Together pLEV1(408) light-repressed system was proven to be in good working condition. It offers all the required features, i.e. simple, easy to manipulate, non-invasive, for the built-up of ‘live cell biocatalysts’.
Figure 6:
Cell growth and mCherry expression study in lightOFF system. Wild-type DH5a cells transformed with pLEV1(408) vector. The cells were illuminated with blue light or wrapped in aluminum foil to induce mCherry expression. Fluorescence and OD600 were measured over a 20-h period of time. a.u. arbitrary units.
Supernova suicide system
For “live cell biocatalysts”, due to the involvement of GMOs, it is essential to add a precaution step to prevent GMO leakage to the natural environment. We selected a suicide gene that encodes Supernova, which is a mutant form of well-known KillerRed, the first engineered genetically-encoded photosensitizer (4). Photosensitizer contains chromphores that generate reactive oxygen species (ROS) upon illumination, and it is commonly used for illumination-induced destruction of targeted cells (5, 6).
Supernova contains three amino acid mutations compare to KillRed, and shows proper localization when fused with target protein (7). Furthermore, unlike KillerRed, Supernova expression along does not interfere with normal cell growth prior to light irradiation (7). Supernova retains the ability to generate ROS, hence promote chromophore-assisted light inactivation (CALI)-based functional analysis of target proteins, overcoming the major drawbacks of KillerRed.
1.Experimental Aim:
To study whether supernova expression can be induced in the LightOFF system and causes cell death upon light illumination by using intensive white light (wavelength 550-600nm).
2.Progress and results:
(1) Construction of pLEV1(408)-supernova vector
Supernova ( BBa_K1491017 ) was cloned into pLEV1 (408) vector at Psil/BglII restriction site and transformed in E. coli DH5a cells. Induction of supernova expression was performed similar to mCherry expression. In brief, grew in LB agar plate+Streptomycin (40 μg/ml) and wrapped in aluminum foil for 24-36 hr at 28 C until red colony appeared. Colony PCR (Figure 7) and restriction enzyme digestion (Figure 8) of extracted plasmid were performed to verify the uptake of supernova gene.
Figure 7:
The agarose gel electrophoresis of colony PCR amplifying a section of ColE promoter and supernova gene in pLEV1(408)-supernova plasmid using primer pair ColE-F and supernova-R. the expected PCR product size is 609bp.
Figure 8:
The agarose gel electrophoresis of enzyme digestion of pLEV1(408)-supernova using EcoR1 and Nde1. The digested fragments are 4500bp and 1309bp respectively.
(2)  Supernova expression its effect on cell apoptosis by illumination
DH5a Cells contained pLEVI (408)-supernova vector were inoculated into LB medium with Streptomycin (40 μg/ml) and grown in light irradiance at 37°C until OD600 reached ∼0.3. Then transferred to darkness for 14-20 hr to induce supernova expression. Pink color pelleted cells indicated supernova expression (Figure 9). 100 μl of culture was diluted in 1ml of PBS buffer and divided into two equal volumes. One was irradiated with intensive orange-yellow light (arac mercury light bulb 160-W with an orange-yellow filter, light intensity 1 W/cm2) for 1 h to induce the release of ROS (reactive oxygen species), while the other kept in darkness.
Supernova contains three amino acid mutations compare to KillRed, and shows proper localization when fused with target protein (7). Furthermore, unlike KillerRed, Supernova expression along does not interfere with normal cell growth prior to light irradiation (7). Supernova retains the ability to generate ROS, hence promote chromophore-assisted light inactivation (CALI)-based functional analysis of target proteins, overcoming the major drawbacks of KillerRed.
The results showed that 1) supernova was successfully expressed after darkness induction, as indicated by red color cells visible to naked eyes and increased fluorescence (Figure 9), and 2) subsequent intensive light irradiation caused apoptosis of bacterial cells, as diluted irradiated culture failed to grow, and non-irradiated showed normal cell growth (Figure 9). Therefore, light-regulated expression system combined with CALI photosensitizer is feasible and efficient in monitoring cell death.
Figure 9:
Cell growth and supernova expression studies. Wild-type DH5a cells transformed with pLEV1(408)-supernova or pLEV1(408)-TorA-opdA plasmid. Cultures were induced in darkness for 12 h. Fluorescence and OD600 were measured over a 14-h period of time. Inert: supernova expression indicated by pink cells.
Figure 10:
The apoptosis effect of supernova to E. coli cells. Light exposure greatly reduced the CFU number of bacteria cells, while darkness exposure allowed normal growth during a 48-h period of time. LB plates were grown for 24 h after light irradiation/kept in darkness.
References:
1) Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, et al. (2005) Synthetic biology: Engineering Escherichia coli to see light. Nature 438: 441–442.
2) Tabor, J. J., Levskaya, A. & Voigt, C. A, 2011. Multi-chromatic Control of Gene Expression in Escherichia coli. J. Mol.Biol. 405:315–324.
3) Chen, X., Liu, R., Ma, Z., Xu, X, Zhang, H., Xu, J. & Yang, 2016. An extraordinary stringent and sensitive light-switchable gene expression system for bacterial cells. Cell Research, 26 (7): 854-7.
4) Bulina, M. E., Chudakov, D. M., Britanova, O. V. & Lukyanov. K.. 2003. A genetically encoded photosensitizer. Nat. Biotechnol. 24, 95-99.
5) Tour, O., Meijer, R. M., Zacharias, D. A., Adams, S. R. & Tsien, R. Y, 2003. Genetically targeted chromophore-assisted light inactivation. Nat. Biotechnol. 21, 1505–1508.
6) Wong, E. V., David, S., Jacob, M. H. & Jay, D. G, 2003. Inactivation of myelin-associated glycoprotein enhances optic nerve regeneration. J. Neurosci. 23, 3112–3117.
7) Takemoto, K., Matsuda, T., Sakai, N., Fu, D., Noda, M., Uchiyama, S., Kotera, I., Arai, Y., Horiuchi, M., Fukui, K. and Ayabe, T., 2013. SuperNova, a monomeric photosensitizing fluorescent protein for chromophore-assisted light inactivation. Scientific reports.
A product for the degradation of soil pesticide residues Copyright © 2017 Lucky      power by  iGEM Team:SSTi-SZGD
