Team:TJU/Future Work
Home
Project
Overview
Background
Design
Results
Future Work
Modeling
Achievement
Judging
Parts
Outreach
Collaborations
Human Practices
Safety
Team
Notebook
Data
Protocol
Attributions
Overview
Background
Microbial fuel cells (MFCs)
Typical exoelectrogens of MFCs
Microbial consortia
Intracellular protein degradation
Design
Lactate producing system
Flavins producing system
Co-culture MFC
References
Results
Lactate producing system
Flavins producing system
Co-culture MFC
Future Work
Proteolysis system
References
Future work
Although we made several innovative research in our project, there are still many other aspects to develop and optimize in the future. 1. We plan to find out biofilm parts of E.coli and modify several key point to reduce adhesion effect. 2. We need to further adjust ratio of our three-strain system to bring a quicker speed toward maximum. 3. Keeping power generation unchanged, we can reduce the volume of our device so that it will increase current density. 4. Making improvements in Shewanella for more electricity output. We intend to weaken any other unrelated functions except for power production in Shewanella and using co-culture system to produce necessary nutrition for its survival.
Proteolysis system
During the experiment, our MFC in anaerobic condition has been faced with unexpected lower pH harmful to microbe consortia. We aim to construct a more stable and robust MFC system, which will achieve the self-regulating and more sustainable output without excessive manipulation. So we have done some research on the acid-sensing and proteolysis system in bacteria. Based on the research, we attempt to build up “Sensing-Regulating” system. Our research and preliminary are as follows.
1 Sensing system—sensing the fluctuation of key metabolites
Every genus of cell has an optimum range of pH to live and Shewanella along with E. coliis no exception. Previous research has identified that Shewanella can maintain a maximal population density when pH keeps above 6.2 while E. colican survive in 5.6~8.1.[1] As an attempt to develop the superior sensing mechanism, we screen out promoter 170 (P170) and regulatory protein RcfB and introduce them into E. colifor adaption.
P170, a strongly acid-inducible promoter from Lactococcus lactis, is up-regulated at low pH during the transition to stationary phase.[2] The trans-acting protein RcfB, however, is involved in basal activity of P170 and is also essential for pH induction. The protein RcfB, upon activation by lactate, bind a DNA recognition motif within a promoter region and activates transcription.[3] When the cells are exposed to acid environments, the RcfB activation by an ‘acid’ signal allows its binding to the ACiD-box, resulting in transcription activation.[3]
Figure 1. The principle of P170 and RcfB
So we construct a part as follows. With the constitutive expression of RcfB, once the pH reaches the threshold that usually ranges from pH 6.0 to pH 6.5, P170 will be induced so that RFP can reflect red fluorescence under UV detection. In this way, we can sense the acidity in our system. Furthermore, other functional genes will replace RFP gene as a strategy for application extension.
Figure 2. The parts we designed for acid sensing
2 Regulating system—an effective tool in regulating metabolic pathway
Here comes our regulating system for acid recovery.
The ssrA tag sequence (mf-ssrA tag) and the Lon protease (mf-Lon) from Mesoplasma florum are made up of the crucial elements of a protein degradation system. Experiments has been identified that mf-ssrA tag is efficiently recognized by mf-Lon [4] Besides, degradation system based on the Gram-positive M. florum tmRNA system lies in proteomic level and does not rely on host degradation systems, which makes it an adaptive method in variety of bacteria. Here, we use this proteolysis system for lactate regulation and keep it in an oscillatory value of acidity.
So we design a part like follows. Once the T7 promoter is activated, with the reflection of red fluorescence, we are able to detect mf-Lon activity on various tags. Henceforth, we can substitute the RFP with functional genes. We believe this toolbox is intended to be a fast and convenient method for prospective proteolysis option and degradation research.
Figure 3. The parts we designed for inducible proteolysis
Notably, the proteolysis system has potential to apply in many fields. As a strategy to make it a toolbox for further research, we use different tags to test the strength of its proteolysis. With the reflection of red fluorescence, we are able to detect mf-Lon activity on various tags based on the distinguish value of UV detection. This toolbox is intended to be a fast and convenient method for prospective proteolysis option and degradation research.
References
[1] Min C, Chung H, Choi W, et al. Linear correlation between inactivation of E. coli and OH radical concentration in TiO2 photocatalytic disinfection.[J]. Water Research, 2004, 38(4):1069–1077. [2] Madsen S M, Arnau J, Vrang A, et al. Molecular characterization of the pH-inducible and growth phase-dependent promoter P170 of Lactococcus lactis.[J]. Molecular Microbiology, 1999, 32(1):75-87. [3] Madsen, Søren M, Hindré, Thomas, Le Pennec, Jean‐Paul, et al. Two acid‐inducible promoters from Lactococcus lactis require the cis‐acting ACiD‐box and the transcription regulator RcfB[J]. Molecular Microbiology, 2005, 56(3):735-746. [4] Eyal G, Sauer R T. Evolution of the ssrA degradation tag in Mycoplasma: specificity switch to a different protease.[J]. Proceedings of the National Academy of Sciences, 2008, 105(42):16113-16118.
E-mail: ggjyliuyue@gmail.com |Address: Building No.20, No.92 Weijin road, Tianjin University, China | Zip-cod: 300072
Copyright 2015@TJU iGEM Team
