<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_Future_Work skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU/Future Work</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-1"><DIV class="menu"><UL><LI><A href="https://2015.igem.org/Team:TJU">Home</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background">Background </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future_Work">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:TJU/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Achievement</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Parts">Parts </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Practices">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Data">Data</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Attributions">Attributions</A></LI></DIV></DIV><DIV id="contain"><DIV id="body-PART"><DIV id="body-PART-l"><DIV class="fixed"><DIV class="cont"><UL><LI class="hmain" style="margin-top:20px;"><A href="https://2015.igem.org/Team:TJU/Overview">Overview</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Microbial">Microbial fuel cells (MFCs)</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Typical">Typical exoelectrogens of MFCs</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#consortia">Microbial consortia</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Intracellular">Intracellular protein degradation</A></LI></UL><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Lactate producing system">Lactate producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Flavins producing system">Flavins producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Co-culture MFC">Co-culture MFC</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#References">References</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Lactate producing system">Lactate producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Flavins producing system">Flavins producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Co-culture MFC">Co-culture MFC</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Future Work">Future Work</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future Work#Proteolysis system">Proteolysis system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future Work#References2">References</A></LI></DIV></DIV></DIV><DIV id="body-PART-r"><DIV align="center"><H2>Future work</H2></DIV><P>Although we made several innovative research in our project, there are still many other aspects to develop and optimize in the future. 
1.	We plan to find out biofilm parts of <SPAN style="font-style: italic">E.coli</SPAN> and modify several key point to reduce adhesion effect. 
2.	We need to further adjust ratio of our three-strain system to bring a quicker speed toward maximum. 
3.	Keeping power generation unchanged, we can reduce the volume of our device so that it will increase current density. 
4.	Making improvements in <SPAN style="font-style: italic">Shewanella</SPAN> for more electricity output. We intend to weaken any other unrelated functions except for power production in <SPAN style="font-style: italic">Shewanella</SPAN> and using co-culture system to produce necessary nutrition for its survival. </P><H3> Proteolysis system </H3><P>During the experiment, our MFC in anaerobic condition has been faced with unexpected lower pH harmful to microbe consortia. We aim to construct a <SPAN style="font-style: normal;font-weight: bold; color:#ddbf73;">more</SPAN> stable and robust MFC system, which will achieve the self-regulating and more sustainable output without excessive manipulation. So we have done some research on the acid-sensing and proteolysis system in bacteria. Based on the research, we attempt to build up “Sensing-Regulating” system. Our research and preliminary are as follows.</P><P style="font-weight: bolder">1 Sensing system—sensing the fluctuation of key metabolites</P><P>Every genus of cell has an optimum range of pH to live and <SPAN style="font-style: italic">Shewanella</SPAN> along with <SPAN style="font-style: italic">E. coli</SPAN>is no exception. Previous research has identified that <SPAN style="font-style: italic">Shewanella</SPAN> can maintain a maximal population density when pH keeps above 6.2 while <SPAN style="font-style: italic">E. coli</SPAN>can survive in 5.6~8.1.[1] As an attempt to develop the superior sensing mechanism, we screen out promoter 170 (P170) and regulatory protein RcfB and introduce them into <SPAN style="font-style: italic">E. coli</SPAN>for adaption.</P><P>P170, a strongly acid-inducible promoter from Lactococcus lactis, is up-regulated at low pH during the transition to stationary phase.[2] The trans-acting protein RcfB, however, is involved in basal activity of P170 and is also essential for pH induction. The protein RcfB, upon activation by lactate, bind a DNA recognition motif within a promoter region and activates transcription.[3] When the cells are exposed to acid environments, the RcfB activation by an ‘acid’ signal allows its binding to the ACiD-box, resulting in transcription activation.[3]</P><DIV class="kuang" style="width:370px"><DIV id="Enlarge"><P><B>Figure 1.</B><SPAN style="font-size: 14px"> The principle of P170 and RcfB</SPAN></P></DIV></DIV><P>So we construct a part as follows. With the constitutive expression of RcfB, once the pH reaches the threshold that usually ranges from pH 6.0 to pH 6.5, P170 will be induced so that RFP can reflect red fluorescence under UV detection. In this way, we can sense the acidity in our system. Furthermore, other functional genes will replace RFP gene as a strategy for application extension.</P><DIV class="kuang" style="width:520px"><DIV id="Enlarge"><P><B>Figure 2.</B><SPAN style="font-size: 14px"> The parts we designed for acid sensing</SPAN></P></DIV></DIV><P style="font-weight: bolder">2 Regulating system—an effective tool in regulating metabolic pathway </P><P>Here comes our regulating system for acid recovery.</P><P>The ssrA tag sequence (mf-ssrA tag) and the Lon protease (mf-Lon) from <SPAN style="font-style: italic">Mesoplasma florum</SPAN> are made up of the crucial elements of a protein degradation system. Experiments has been identified that mf-ssrA tag is efficiently recognized by mf-Lon [4] Besides, degradation system based on the Gram-positive <SPAN style="font-style: italic">M. florum</SPAN> tmRNA system lies in proteomic level and does not rely on host degradation systems, which makes it an adaptive method in variety of bacteria. Here, we use this proteolysis system for lactate regulation and keep it in an oscillatory value of acidity.</P><P>So we design a part like follows. Once the T7 promoter is activated, with the reflection of red fluorescence, we are able to detect mf-Lon activity on various tags. Henceforth, we can substitute the RFP with functional genes. We believe this toolbox is intended to be a fast and convenient method for prospective proteolysis option and degradation research.</P><DIV class="kuang" style="width:470px"><DIV id="Enlarge"><P><B>Figure 3.</B><SPAN style="font-size: 14px"> The parts we designed for inducible proteolysis</SPAN></P></DIV></DIV><P>Notably, the proteolysis system has potential to apply in many fields. As a strategy to make it a toolbox for further research, we use different tags to test the strength of its proteolysis. With the reflection of red fluorescence, we are able to detect mf-Lon activity on various tags based on the distinguish value of UV detection. This toolbox is intended to be a fast and convenient method for prospective proteolysis option and degradation research. </P><H3>References</H3><P><SPAN style="font-weight: lighter ;font-size:14px;"> 
 [1] Min C, Chung H, Choi W, et al. Linear correlation between inactivation of <SPAN style="font-style: italic">E. coli</SPAN> and OH radical concentration in TiO<SUB>2</SUB> photocatalytic disinfection.[J]. Water Research, 2004, 38(4):1069–1077. 
[2] Madsen S M, Arnau J, Vrang A, et al. Molecular characterization of the pH-inducible and growth phase-dependent promoter P170 of <SPAN style="font-style: italic">Lactococcus lactis</SPAN>.[J]. Molecular Microbiology, 1999, 32(1):75-87.
[3] Madsen, Søren M, Hindré, Thomas, Le Pennec, Jean‐Paul, et al. Two acid‐inducible promoters from <SPAN style="font-style: italic">Lactococcus lactis</SPAN> require the cis‐acting ACiD‐box and the transcription regulator RcfB[J]. Molecular Microbiology, 2005, 56(3):735-746.
[4] Eyal G, Sauer R T. Evolution of the ssrA degradation tag in <SPAN style="font-style: italic">Mycoplasma</SPAN>: specificity switch to a different protease.[J]. Proceedings of the National Academy of Sciences, 2008, 105(42):16113-16118.</SPAN></P></DIV><DIV id="foot"><P>E-mail: ggjyliuyue@gmail.com |Address: Building No.20, No.92 Weijin road, Tianjin University, China | Zip-cod: 300072</P><P>Copyright 2015@TJU iGEM Team</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>