<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-1"><DIV class="menu"><UL><LI><A href="https://2015.igem.org/Team:TJU">Home</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background">Background </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future_Work">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:TJU/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Achievement</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Parts">Parts </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Practices">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Data">Data</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Attributions">Attributions</A></LI></DIV></DIV><DIV id="contain"><DIV id="body-PART"><DIV id="body-PART-l"><DIV class="fixed"><DIV class="cont"><UL><LI class="hmain"><A href="#Gene Expression Model for Lactate production">Gene Expression Model for Lactate production</A></LI><LI><A href="#Simplification and Assumption1">Simplification and Assumption</A></LI><LI><A href="#Equation Establishment1">Equation Establishment</A></LI><LI><A href="#Simulation Results1">Simulation Results</A></LI></UL><LI class="hmain"><A href="#Models for monocultures of Shewanella in MFC">Models for monocultures of <EM>Shewanella</EM> in MFC</A></LI><LI><A href="#Shewanella Growth Model">Simplification and Assumption</A></LI><LI><A href="#Extracellular Electron Transfer Model">Equation Establishment</A></LI><LI><A href="#Simulation Results2">Simulation Results</A></LI><LI><A href="#Parameters and variables2">Parameters and variables</A></LI><LI class="hmain"><A href="#Models for monocultures of Shewanella in MFC">Model for co-culture MFC</A></LI><LI><A href="#Growth Model of E.coli">Growth Model of <EM>E.coli</EM></A></LI><LI><A href="#Extracellular Electron Transfer Model">Equation Establishment</A></LI><LI><A href="#Simulation Results3">Simulation Results</A></LI><LI><A href="#Parameters and variables3">Parameters and variables</A></LI><LI class="hmain"><A href="#Model for P170 promoter">Model for P170 promoter</A></LI><LI><A href="#Introduction">Introduction</A></LI><LI><A href="#Assumptions">Assumptions</A></LI><LI><A href="#Equation Establishment4">Equation Establishment</A></LI><LI><A href="#Simulation Results4">Simulation Results</A></LI><LI><A href="#Parameters and variables4">Parameters and variables</A></LI></DIV></DIV></DIV><DIV id="body-PART-r"><H3>1 Gene Expression Model for Lactate production </H3><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Simplification and Assumption</SPAN></P><P align="justify">① Simplifications and assumptions for metabolic pathway and gene insertion:supposing that we have adequate glucose and constant pyruvate, the latter used to product lactate can be replenished by glucose in <EM>E.coli</EM>. Naturally, <EM>E.coli</EM> can itself
produce lactate dehydrogenase (Ldh) through LdhA gene expression. Later, we attempt to introduce plasmids with Ldh gene, which can produce the extra Ldh as an approach to generate even more lactate. However, the excessive Ldh, to a certain degree, will also inhibit Ldh gene expression.</P><P align="justify">② Simplifications and assumptions for Michaelis-Menten equation: we want to figure out the relations between the speed of lactate production and the amount of Ldh expression. But, there is a precondition in Michaelis-Menten equation, namely, the total amount of enzyme in solution keeps unchanged.</P><P align="left">So, we need to modify the existing Michaelis-Menten equation:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"> </TD></TR></TBODY></TABLE><P align="justify">The above equation reflects a general enzyme catalytic reaction process. With the help of steady state approximation, the equation can transform from first-order reaction to zero-order reaction. Because of the constant substrate concentration as we assumed, the variable in the equation is the variation of ldh amount, which means that the independent variable in the Michaelis-Menten equation turns into the enzyme amount. (variable change from [S] into [E])</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"> </TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"><P>in which :</P></TD><TD align="right" width="126"><P>(1)</P></TD></TR></TBODY></TABLE><P>③ Simplification of transmembrane transfer: the original pH in the medium is slightly greater than 7 and the pH can keep constant with the buffer function of KHPO4 together with KH2PO4. Suppose that <EM><EM>Shewanella</EM></EM> can absorb lactate brought by <EM>E.coli</EM> timely, thus, pH can remain stable, which is around 7.2 based on our hypothesis. Further, we assume that transmembrane transfer belongs to codiffusion where the concentration difference of H+ and acid groups can drive the whole process. Therefore, the transmembrane transfer is only related to the intra membrane concentration of H+ and acid groups.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Equation Establishment</SPAN></P><P align="justify">Firstly, we establish the equation of ldh amount changing with time. The following equations represent the ldh production rate in <EM>E.coli</EM> without the plasmid insertion and with the plasmid introduction, respectively.</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(2)</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(3)</P></TD></TR></TBODY></TABLE><P align="left">Next, we construct the lactate production rate equation:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(4)</TD></TR></TBODY></TABLE><P align="justify">Finally, the lactate transmembrane transfer equations has been figured out. It shows that lactate will be transferred to the outside at the time it produced. We represent the intra membrane with c and extra membrane with e.</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"> </TD></TR></TBODY></TABLE><P align="left">The transmembrane transfer rate:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(5)</P></TD></TR></TBODY></TABLE><P align="left">When lactate production is combined with its transfer, we can get the following equation:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(6)</P></TD></TR></TBODY></TABLE><A name="Simulation Results" id="Simulation Results1"><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Simulation Results</SPAN></P></A><DIV id="Enlarge"><P><B>Figure 1.</B><SPAN style="font-size: 14px"> The relations between amount of ldh and lactate with time passed, respectively.</SPAN></P></DIV></DIV><P align="justify">From the simulation results, we can clearly see that the <EM>E. coli</EM> with Ldh have a significantly increased in number and lactic acid production of lactate dehydrogenase, which is consist with our experimental results.</P><H3>2  Models for monocultures of <EM><EM>Shewanella</EM></EM> in MFC </H3><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;"><EM>Shewanella</EM> Growth Model Based on Carbon Balance</SPAN></P><P align="justify">At the very beginning, we construct a fermentation kinetic model based on the Monod Equation to draw a picture of carbon balance between biomass increase and base consumption during <EM>S.oneidensis</EM> MR-1 growth. Before generation our modeling, we simplify the center process of carbon metabolism in <EM>Shewanella</EM> via making assumptions as follows:</P><P align="justify">① Lactate is the preferable carbon source for <EM>Shewanella</EM>. Lactate can transform into pyruvate by dehydrogenation, which then converts mostly into acetyl coenzyme. Subsequently, acetyl coenzyme can either be oxidized into CO2 and release energy by TCA Circle or transform into acetate.</P><P align="justify">② When there are too much lactate, the producing rate of acetyl coenzyme will be greater than the consuming rate of that in TCA circle. The excessive acetyl coenzyme will convert into acetate and release to the outside. Inversely, the shortage of lactate leads the utilization of acetate which then maintains the metabolism by acetyl coenzyme formation.</P><P align="justify">③ Although both acetate and lactate can be used as carbon sources for <EM>Shewanella</EM>, acetate can restrain its growth. On the one hand, acetate will drop the acidity inside the cell, which, a step forward, can affect the activity of enzyme in vivo resulted in uncompetitive inhibition to cell growth. On the other hand, acetate can itself limit the growth by uncompetitive inhibition. In particular, the process of acetate metabolism and lactate metabolism are competing for acetyl coenzyme, which reduces the lactate utilization and influences cell growth. (Tang &amp; Meadows, 2006)</P><P align="justify">④ The redox of mediator in the outer membrane is the rate-limiting step of electron transfer chain in <EM>Shewanella</EM>. TCA circle, an important pathway for the synthesis of energy and biomass , largely depends on the concentration of electron receptor at the end of electron transfer chain. When there is a deficiency of oxidized mediator, TCA circle will be inhibited, thereby, affect the whole process of metabolism as well as biomass increase.</P><P align="justify">According to the four assumptions above along with acetate uncompetitive inhibition and competitive inhibition, we decide to use multi-substrates Monod equation(Blanch &amp; Clark, 1997) with two kinds of inhibition to demonstrate the relations between restrictive substrates and biomass increase rate. Considering that the accumulation of NADH will remarkably affect metabolic process under anaerobic conditions, we add a product factor in the Monod equation reflecting metabolites inhibition function.  μ<SUBS>L</SUBS>  and μ<SUBS>A</SUBS> refers to the lactate and acetate biomass specific rate, respectively. We quantify the relations between specific rate of lactate/acetate and restrictive base concentration as following:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(7)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(8)</TD></TR></TBODY></TABLE><P align="justify">Biomass, lactate and acetate concentration can vary with time and its construction depends on kinetic model which is based on the general kinetics of the fermentation process model. Consumption of lactate and acetate can be both used for biomass increase and endogenous metabolism maintenance. However, considering the rarely energy losses caused by metabolism in vivo, we ignore the carbon consumption brought by metabolism. In this way, lactate can only be used for biomass increase and acetate production while acetate is merely used for biomass increase.</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(9)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(10)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(11)</P></TD></TR></TBODY></TABLE><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Extracellular Electron Transfer Model</SPAN></P><P align="justify">Extracellular Electron Transfer (EET) process can deliver the electrons between electrode and cell membrane with mediating functions of redox active molecules including FMN and riboflavin. It can be subdivided into the oxidation of reduced mediator and reduction of oxidized mediator in electrode.</P><P align="justify">We establish a model kinetics of electrode process on the basis of Bulter-Volmer equation to illustrate the oxidation of reduced mediator in electrode. Firstly, we make several assumptions just like the following tips:</P><P align="justify">① Although both of FMN and riboflavin can serve as mediator in electrode redox, we only take riboflavin into consideration for simplification in the model.</P><P align="justify">② There are two kinds of EET pathway model for riboflavin mediating. One of these called single-electron transfer in which riboflavin acts as cofactor of MtrC protein and can be reduced into semiquinone (Sq) with the standard electrode potential of -260v. Another one is two-electron transfer where the dissociative riboflavin binds to outer membrane  c-Cyts as a cofactor in the hydroquinone (Hq) form with shifted potential of -145v. Under conditions of the same electrode and concentration, the exchange current density of single-electron transfer is much high than that of two-electron transfer since enzymatic reaction can greatly decrease its activation energy. Therefore, we assumed that current generation reaction in anode mainly belongs to the single-electron transfer.</P><P align="justify">③ Oxidation of Sq in electrode is the reversible first order reaction. The net rate for this kind of reaction is made up by the difference between oxidized rate of Sq and reduced rate of riboflavin in electrode.</P><P align="left">Accordingly, current density in anode can be calculated by Bulter-Vlomer equation:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(12)</P></TD></TR></TBODY></TABLE><P align="left">Reduction and Oxidation reaction rate constant can be calculated by Arrhenius equation:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(13)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(14)</P></TD></TR></TBODY></TABLE><P align="justify">As for the reduction of oxidized mediator in outer membrane for <EM>Shewanella</EM>, we establish a kinetics of enzyme catalyzed reaction on the basis of Michaelis-Menten equation, accordingly.</P><P align="left">① Similarly, we also consider the single-electron transfer as the only pathway in vivo.</P><P align="justify">② In such single-electron transfer process, riboflavin reduction kinetics characteristics follows the general rules of enzymatic reaction.</P><P align="justify">③ The reduction of riboflavin is related to the reducing state of OM c-Cyts heme center which is also the terminal of EET pathway. Due to the electrons brought by NADH or NADPH from respiration, we argue that the process of riboflavin reduction into Sq is coupled with NADH from respiration process.</P><P align="justify">Based on the assumptions above, we believe the respiration of OM riboflavin has something to do with its concentration and NADH production rate. According to the kinetics of enzyme catalyzed reaction, we use multi-substrates Michaelis-Menten equation to determine the whole process:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(15)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(16)</P></TD></TR></TBODY></TABLE><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"><P>(17)</P></TD></TR></TBODY></TABLE><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Simulation Results</SPAN></P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 2.</B><SPAN style="font-size: 14px"> The growth curve of <EM>S.oneidensis</EM> in MFC.</SPAN></P></DIV></DIV><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 3.</B><SPAN style="font-size: 14px"> The conc. of lactate and acetate varies with time.</SPAN></P></DIV></DIV><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 4.</B><SPAN style="font-size: 14px"> The conc. of Sq and Hq varies with time.</SPAN></P></DIV></DIV><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 5.</B><SPAN style="font-size: 14px"> The Voltage varies with time.</SPAN></P></DIV></DIV><P align="justify">As can be seen from the simulation results, when the initial inoculation amount Shewanella is 0.1 g/L, it's growth time in anaerobic  MFC is approximately 10 hours. The output voltage of the battery is gradually stabilizing after 20 hours, and the voltage drops after 70 hours due to the mass death of Shewanella.Although specific values are different, the simulation results are consistent with the experimental results in the trends of voltage.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Parameters and variables</SPAN></P><TABLE width="840" border="2" cellspacing="0" cellpadding="5"><TBODY><TR><TD align="left" width="168">Parameter</TD><TD align="left" width="336">Discribtion</TD><TD align="left" width="168">Number</TD><TD align="left" width="168">[Unit]</TD></TR><TR><TD align="left" width="168">v</TD><TD align="left" width="336">volume of reactor</TD><TD align="left" width="168">140E-3</TD><TD align="left" width="168">[L]</TD></TR><TR><TD align="left" width="168">μ<SUB>max,L</SUB></TD><TD align="left" width="336">the maximal specific growth rate taking lactate as substrate</TD><TD align="left" width="168">0.52</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">μ<SUB>max,A</SUB></TD><TD align="left" width="336">the maximal specific growth rate taking acetate as substrate</TD><TD align="left" width="168">0.28</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>e</SUB></TD><TD align="left" width="336">endogenous metabolism rate constant</TD><TD align="left" width="168">0.08</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>c</SUB></TD><TD align="left" width="336">acetate competitive inhibition constant</TD><TD align="left" width="168">1.3</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>u</SUB></TD><TD align="left" width="336">acetate uncompetitive inhibition constant</TD><TD align="left" width="168">8.3E-3</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>R</SUB></TD><TD align="left" width="336">acetate productive inhibition constant</TD><TD align="left" width="168">12.6E-3</TD><TD align="left" width="168">[mol∙L<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>s,L</SUB></TD><TD align="left" width="336">half-saturation constant of Monod equation taking lactate as substrate</TD><TD align="left" width="168">12E-3</TD><TD align="left" width="168">[mol∙L<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>s,A</SUB></TD><TD align="left" width="336">half-saturation constant of Monod equation taking aactate as substrate</TD><TD align="left" width="168">12.3E-3</TD><TD align="left" width="168">[mol∙L<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>M<SUB>ox</SUB></SUB></TD><TD align="left" width="336">half-constant of Monod equation taking oxidized mediator as substrate</TD><TD align="left" width="168">6E-6</TD><TD align="left" width="168">[mol∙L<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">Y<SUB>X/L</SUB></TD><TD align="left" width="336">apparent biomass coefficient of lactate</TD><TD align="left" width="168">19.1</TD><TD align="left" width="168">[g dry cell/ mol lactate]</TD></TR><TR><TD align="left" width="168">Y<SUB>X/A</SUB></TD><TD align="left" width="336">apparent biomass coefficient of acetate</TD><TD align="left" width="168">16.8</TD><TD align="left" width="168">[g dry cell/ mol acetate]</TD></TR><TR><TD align="left" width="168">n</TD><TD align="left" width="336">the numbers of electron transfer</TD><TD align="left" width="168">1</TD><TD align="left" width="168">[mol e<SUP>-</SUP>/mol reaction]</TD></TR><TR><TD align="left" width="168">k<SUB>f</SUB></TD><TD align="left" width="336">oxidation rate constant</TD><TD align="left" width="168">5.2504E-5</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>b</SUB></TD><TD align="left" width="336">reduction rate constant</TD><TD align="left" width="168">5.2574E-5</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">α</TD><TD align="left" width="336">oxidization transfer coefficient in anode</TD><TD align="left" width="168">0.5</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">F</TD><TD align="left" width="336">faraday's constant</TD><TD align="left" width="168">96485</TD><TD align="left" width="168">[C∙mol<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">R</TD><TD align="left" width="336">ideal gas constant</TD><TD align="left" width="168">8.314</TD><TD align="left" width="168">[J∙mol<SUP>-1</SUP>∙K<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">T</TD><TD align="left" width="336">thermodynamic temperature</TD><TD align="left" width="168">303.15</TD><TD align="left" width="168">[K]</TD></TR><TR><TD align="left" width="168">ΔE</TD><TD align="left" width="336">The potential between riboflavin and the heme left of OM c-Cyts</TD><TD align="left" width="168">0.395</TD><TD align="left" width="168">[v]</TD></TR><TR><TD align="left" width="168">k<SUB>0</SUB></TD><TD align="left" width="336">standard rate constant</TD><TD align="left" width="168">0.10</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">v<SUB>m</SUB><SUP>f</SUP></TD><TD align="left" width="336">the maximum reaction rate constant</TD><TD align="left" width="168">6E-5</TD><TD align="left" width="168">[h<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">k<SUB>iA</SUB></TD><TD align="left" width="336">the dissociation constant of NADH</TD><TD align="left" width="168">1E-2</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>mA</SUB></TD><TD align="left" width="336">Michaelis constant when NADH saturation</TD><TD align="left" width="168">1E-2</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>mB</SUB></TD><TD align="left" width="336">Michaelis constant when riboflavin saturation</TD><TD align="left" width="168">1E-2</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>1</SUB></TD><TD align="left" width="336">the inhibition constants of riboflavin</TD><TD align="left" width="168">5E-4</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>2</SUB></TD><TD align="left" width="336">the inhibition constants of NADH</TD><TD align="left" width="168">10</TD><TD align="left" width="168">[dimensionless]</TD></TR></TBODY></TABLE><H3>3 Model for co-culture MFC</H3><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Growth Model of <EM>E.coli</EM></SPAN></P><P align="justify">In order to figure out the kinetics feature of growth, substrate consumption and accumulation of metabolite in lactate producing <EM>E.coli</EM>, we construct kinetics model of fermentation process based on carbon balance. First of all, the fermentation process of lactate in <EM>E.coli</EM> under anaerobic conditions has been simplified and we make several assumptions as following:</P><P align="justify">① Except for carbon sources which serves as restrictive substrates, the growth of <EM>E.coli</EM> will not be affected by other nutrition in the medium.</P><P align="justify">② <EM>E.coli</EM> will use carbon sources to maintain its survival, division and proliferation while the accumulation of metabolites is also counted in.</P><P align="justify">③ The growth model of <EM>E.coli</EM> is equilibrium increase rather than structural increase, which means the ascending rate of biomass in <EM>E.coli</EM> increases as the same as the rate for cell weight accumulation.</P><P align="justify">④ We take lactate as the only one metabolites for accumulation and lactate increase is positively related to its growth rate. Besides, overmuch lactate will inhibit the growth of <EM>E.coli</EM>.</P><P align="justify">We further construct the carbon equilibrium calculations for anaerobic lactate fermentation in batches. Therefore, we figure out the following differential equations:</P><TABLE width="840" border="0" cellspacing="0"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(18)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(19)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(20)</TD></TR></TBODY></TABLE><P align="justify">Owing to the assumption of equilibrium increase in <EM>E.coli</EM>, the relations between specific growth rate of biomass and concentration of restrictive substrates can reflect in the Monod equations that we modeled. However, considering that the accumulation of lactate may inhibit bacteria growth, we introduce an extra inhibition factor for general Monod equation. Due to the positive relations between lactate accumulation and cell growth, the production of metabolites also share the same relation with biomass increase. In this way, Q<SUB>p</SUB> can be represented by using the function of μ<SUB>E</SUB>.</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(21)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(22)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(23)</TD></TR></TBODY></TABLE><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Simulation Results</SPAN></P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 6.</B><SPAN style="font-size: 14px"> The growth curve of <EM><EM>E.coli</EM></EM>.</SPAN></P></DIV></DIV><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 7.</B><SPAN style="font-size: 14px"> The conc. of lactate and acetate varies with time.</SPAN></P></DIV></DIV><P align="justify">As can be seen from the above figures, experimental results are very close to the simulation results of Ordinary Differential Equations, thus it is considered a successful model. Based on the growth model of <EM>E.coli</EM>, We constructed the co-culture MFC model.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Model For Co-culture MFC</SPAN></P><P align="justify">We constructed the <EM>E.coli</EM> growth model and the monocultures <EM>Shewanella</EM> MFC model into a co-culture MFC model.  In this model, <EM>E. coli</EM> is responsible for the decomposition of glucose to lactic acid, to provide a carbon source for the <EM>S.onedensis</EM>, <EM>S.onedensis</EM> responsible for lactate decomposition of carbon dioxide, transporting the electrons to the anode by  the redox reaction of riboflavin. The simulation Results is shown as follow: </P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 8.</B><SPAN style="font-size: 14px"> The Biomass of <EM>E.coli</EM> and <EM>S.onedensis</EM> in MFC varies with time.</SPAN></P></DIV></DIV><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 9.</B><SPAN style="font-size: 14px"> The Voltage varies with time.</SPAN></P></DIV></DIV><P align="justify">As can be seen in figure 8, when the amount of  <EM>E. coli</EM> is small, due to lack of carbon supply <EM>S.onedensis</EM> begin to die, the situation changed after the rapid growth of <EM>E. coli</EM>. According to this result, we reached the following conclusions: In order to maintain the activity of Shewanella at the begining, The primary objective is to ensure the lactate production of E. coli. This conclusion has played an important role in the construction of three-bacterium MFC.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Parameters and variables</SPAN></P><TABLE width="840" border="2" cellspacing="0" cellpadding="5"><TBODY><TR><TD align="left" width="168">Parameter</TD><TD align="left" width="336">Discribtion</TD><TD align="left" width="168">Number</TD><TD align="left" width="168">[Unit]</TD></TR><TR><TD align="left" width="168">k</TD><TD align="left" width="336">lactate inhibition constant</TD><TD align="left" width="168">0.085</TD><TD align="left" width="168">[L/mol]</TD></TR><TR><TD align="left" width="168">k<SUB>S,G</SUB></TD><TD align="left" width="336">Half-saturation constant taking glucose as substrate</TD><TD align="left" width="168">14E-3</TD><TD align="left" width="168">[L/mol]</TD></TR><TR><TD align="left" width="168">k<SUB>S,L</SUB></TD><TD align="left" width="336">Half-saturation constant taking lactate as substrate</TD><TD align="left" width="168">20E-3</TD><TD align="left" width="168">[L/mol]</TD></TR><TR><TD align="left" width="168">μ<SUB>EG,max</SUB></TD><TD align="left" width="336">E.coli's maximal specific growth rate taking glucose as substrate</TD><TD align="left" width="168">1.3</TD><TD align="left" width="168">[h<SUP>-1<SUP>]</SUP></SUP></TD></TR><TR><TD align="left" width="168">μ<SUB>E/L,max</SUB></TD><TD align="left" width="336">E.coli's maximal specific growth rate taking lactate as substrate</TD><TD align="left" width="168">0.4</TD><TD align="left" width="168">[h<SUP>-1<SUP>]</SUP></SUP></TD></TR><TR><TD align="left" width="168">k<SUB>L,C</SUB></TD><TD align="left" width="336">lactate competitive inhibition constant</TD><TD align="left" width="168">1</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">k<SUB>L,U</SUB></TD><TD align="left" width="336">lactate uncompetitive inhibition constant</TD><TD align="left" width="168">200E-3</TD><TD align="left" width="168">[dimensionless]</TD></TR><TR><TD align="left" width="168">m<SUB>E</SUB></TD><TD align="left" width="336">Endogenous metabolism rate constant of <EM>E.coli</EM></TD><TD align="left" width="168">0.006</TD><TD align="left" width="168">[h<SUP>-1<SUP>]</SUP></SUP></TD></TR><TR><TD align="left" width="168">k<SUB>L,R</SUB></TD><TD align="left" width="336">Lactate productive inhibition constant</TD><TD align="left" width="168">12.6E-3</TD><TD align="left" width="168">[mol∙L<SUP>-1</SUP>]</TD></TR><TR><TD align="left" width="168">v</TD><TD align="left" width="336">volume of reactor</TD><TD align="left" width="168">140E-3</TD><TD align="left" width="168">[L]</TD></TR><TR><TD align="left" width="168">Y<SUB>XE/G</SUB></TD><TD align="left" width="336">apparent biomass coefficient of lactate</TD><TD align="left" width="168">8.1</TD><TD align="left" width="168">[g dry cell/ mol lactate]</TD></TR><TR><TD align="left" width="168">Y<SUB>XE/L</SUB></TD><TD align="left" width="336">apparent biomass coefficient of lactate</TD><TD align="left" width="168">5.8</TD><TD align="left" width="168">[g dry cell/ mol lactate]</TD></TR><TR><TD align="left" width="168">Y<SUB>L/G</SUB></TD><TD align="left" width="336">apparent biomass coefficient of lactate</TD><TD align="left" width="168">1.5</TD><TD align="left" width="168">[g dry cell/ mol lactate]</TD></TR></TBODY></TABLE><H3>4 Model for P170 promoter</H3><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Introduction</SPAN></P><P align="justify">P170 is an acid-induced promoter. When rcfB protein is activated by lactic acid, it can bind to P170 and make it sensitive to RNA polymerase. In order to detect and control the concentration of lactic acid, we use P170 and RFP gene, which shows the relationship between lactic acid and fluorescence.</P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 10.</B><SPAN style="font-size: 14px">Reaction network of genetic regulation circuit</SPAN></P></DIV></DIV><P align="left">From the reaction network We can extract follow equation sets From the reaction network:</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"> </TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126"> </TD></TR></TBODY></TABLE><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Assumptions</SPAN></P><P align="justify">1. When genes transcript and translate downward, the mRNA can bind with ribosome and then start translation subsequently, which can insure a constant amount during the whole process. Therefore, we use steady-state approximation on mRNA and mRNA-rib.</P><P align="justify">2. Because there is some acid buffer in the cell to transfer the produced lactate out of cell, pH will have a rather small influence on enzyme activity. Therefore, we assume that all enzymes activity are not affected by acid concentration, the rate constant keeps unchanged.</P><P align="justify">3. Concentration of protein will eventually keep constant, which is because expressed protein can have a negative effect on gene expression in return. Thus we assume that the concentration of rcfB protein is constant, and its magnitude is determined by the strength of the promoter upstream.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Equation Establishment</SPAN></P><P align="justify">Genetic regulation circuit is shown in Figure 10. When rcfB protein is activated by lactic acid, it can bind to P170 and make it sensitive to RNA polymerase.</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(24)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD width="126"> </TD><TD align="right" width="126">(25)</TD></TR></TBODY></TABLE><P align="justify">where K<SUB>La</SUB>  and K are constants and n is Hill coefficient The acP170 is the activated state, the P170 can be activated in two ways. One is to bind with the lactic acid dimerization and RcfB which can make it easy to be activated. Another one, we assume that a   of P170 can be activated directly. Both the binding and unbinding P170 can be promoted while they have different rate constant. 
</P><P align="left">We conclude that:</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(26)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(27)</TD></TR></TBODY></TABLE><P align="left">According to the equation above, we can consolidate that:</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(28)</TD></TR></TBODY></TABLE><P align="left">Where km1 and km2 are integration parameters.</P><P align="left">Next, by using law of mass action and steady state solution, we deduce the equation below. </P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(29)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(30)</TD></TR></TBODY></TABLE><P align="left">Where the concentration of mRNA and mRNA-rib satisfies:</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(31)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(32)</TD></TR></TBODY></TABLE><P align="left">In the end, the concentration of RFP and the fluorescence can be written as:</P><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(33)</TD></TR></TBODY></TABLE><TABLE width="840"><TBODY><TR><TD align="left" width="126"> </TD><TD align="right" width="126">(34)</TD></TR></TBODY></TABLE><P align="justify">Thus we construct the connection between fluorescence and lactic acid, from which we can know the concentration of lactic acid through the system.</P><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Simulation Results</SPAN></P><P align="left">We set k<SUB>m1</SUB> =10 and use promoter J23100, then we get the following result </P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 11.</B><SPAN style="font-size: 14px">The result shows how P170,mRNA,mRNA-rib,fluorescence change with the lactic acid.</SPAN></P></DIV></DIV><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Fine tuning</SPAN></P><P align="left">From the equation, we can predict that:</P><P align="left">1. the integration parameter k<SUB>m1</SUB> can characterize the relative level of leakage.</P><P align="left">2. the larger the RcfB concentration is, the quicker the fluorescence will be.</P><P align="justify">At first we have the constitutive promoter fine-tuning. The BIT-China suggests us using different promoters in our construction. The performance of promoter P170 and rCfB play an important role. We choose to use different constitutive promoter J23103 J23109 J23107 and J23100 for the result (Figure 11). According to the figures, we can get the highest rate of change in fluorescence when we use J23100 promoter which is also the most sensitive one among all the promoter testing. Therefore, we choose J23100 promoter since it is convenient to represent lactate concentration by using fluorescence as the measurement. Particularly, when we have a stronger promoter, the amount of rcfB will be more during stable state. As a result gained from equation, the P170 can be activated in a faster speed and consequently, the changing rate of fluorescence will be greater as well.</P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 12.</B><SPAN style="font-size: 14px">Fluorescence changes with time when rcfB is constitutively activated by J23100, J23103, J23107 and J23109,respectively.</SPAN></P></DIV></DIV><P align="justify">Next, we will explore the effect brought by leakage which means that P170 can activate and lead to fluorescence expression when lactate doesn’t exist. We have predicted that k<SUB>m1</SUB> is the index of leakage. k<SUB>m1</SUB> is higher, then we get more leakage shown up. Here we make k_m1 equals to 1, 10, 50 respectively and the result is shown in Figure 12. As predicted, the result is well suited to our calculation, because k<SUB>m1</SUB> is a constant unrelated to lactate concentration. It has a remarkable  influence on the result when lactate amount is relatively small.
Then we will analyze the constants related to k<SUB>m1</SUB>, which mainly contains rate k<SUB>3</SUB> that is directly activated by P170 as well as proportion , which can both affect leakage. Our target is reducing the leakage expression, thus, we need to lower those constants as much as possible.</P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Figure 13.</B><SPAN style="font-size: 14px">Fluorescence changes with time when k_m1 equals to 1, 10, 50. Difference has been show to illustrate the situation of different k<SUB>m1</SUB> under limited lactate concentration.</SPAN></P></DIV></DIV><P><SPAN style="font-style: normal; font-weight: bold; color:#ddbf73;">Parameters and variables</SPAN></P></DIV><DIV id="foot"><P>E-mail: ggjyliuyue@gmail.com |Address: Building No.20, No.92 Weijin road, Tianjin University, China | Zip-cod: 300072</P><P>Copyright 2015@TJU iGEM Team</P></DIV></DIV>&lt;/div&gt;</DIV></DIV></DIV></DIV></BODY></HTML>