<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU/Background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-1"><DIV class="menu"><UL><LI><A href="https://2015.igem.org/Team:TJU">Home</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background">Background </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future_Work">Future Work</A></LI></UL><LI><A href="https://2015.igem.org/Team:TJU/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Achievement</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Parts">Parts </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Practices">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:TJU/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Data">Data</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Protocol">Protocol</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Attributions">Attributions</A></LI></DIV></DIV><DIV id="contain"><DIV id="body-PART"><DIV id="body-PART-l"><DIV class="fixed"><DIV class="cont"><UL><LI class="hmain" style="margin-top:20px;"><A href="https://2015.igem.org/Team:TJU/Overview">Overview</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Microbial">Microbial fuel cells (MFCs)</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Typical">Typical exoelectrogens of MFCs</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#consortia">Microbial consortia</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Background#Intracellular">Intracellular protein degradation</A></LI></UL><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Lactate producing system">Lactate producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Flavins producing system">Flavins producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#Co-culture MFC">Co-culture MFC</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Design#References">References</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Lactate producing system">Lactate producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Flavins producing system">Flavins producing system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Results#Co-culture MFC">Co-culture MFC</A></LI><LI class="hmain"><A href="https://2015.igem.org/Team:TJU/Future Work">Future Work</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future Work#Proteolysis system">Proteolysis system</A></LI><LI><A href="https://2015.igem.org/Team:TJU/Future Work#References2">References</A></LI></DIV></DIV></DIV><DIV id="body-PART-r"><DIV align="center"><H2>Background</H2></DIV><H3>1 Microbial fuel cells (MFCs)</H3><P>A microbial fuel cell(MFC) is an electrochemical device which converts the chemical energy of fuel to electrical energy by the catalytic actions of microorganisms.[1]</P><P>Compared with energy from fossil fuel, such as oil and coal, MFC is of plethora advantages over other kinds of energy generators, e.g. wide sources of reactant, no emissions of environmental polluting gases (such as SO<SUB>x</SUB>, NO<SUB>x</SUB>, CO<SUB>2</SUB> and CO), high conversion in theory, mild reaction condition, as a result, lack of sonic pollution, and so forth.[2]</P><P>Benefited from those merits, MFCs have a wide range of potential applications. For example, MFCs are suitable for powering electrochemical sensors and small telemetry systems to transmit obtained signals to remote receivers while traditional batteries cannot because of restricted lifetime and frequent changes and recharges. Additionally, realistic energetically autonomous would probably be equipped with MFCs that utilize different fuels like sugar, fruit, dead insects, grass and weed. Such systems can solely power itself by MFCs to perform some behavior including motion, sensing, computing and communication. [3] </P><DIV class="kuang" style="width:470px"><DIV id="Enlarge"><P><B>Figure 1.</B><SPAN style="font-size: 14px">Typical microbial fuel cell device</SPAN></P></DIV></DIV><H3>2 Typical exoelectrogens of MFCs</H3><P>Several types of biofuel cells including microbial fuel cell and enzymatic biofuel cell have been well documented in the literature. Various microbial or biochemical fuel cells have been developed using <SPAN style="font-style: italic">Desulfovibrio desulfuricans</SPAN>, <SPAN style="font-style: italic">Proteous vulgaris</SPAN>, <SPAN style="font-style: italic">Escherichia coli</SPAN>, <SPAN style="font-style: italic">Pseudomonas</SPAN> species and redox enzymes as biocatalysts.[4] </P><P>Of all the MFCs above, <SPAN style="font-style: italic">Shewanella</SPAN> species are best known for extracellular electron transfer(EET). <SPAN style="font-style: italic">Shewanella</SPAN>, a facultative anaerobe, has been widely used as a model anode biocatalyst in MFCs due to its easiness of cultivation, adaptability to aerobic and anaerobic environment and both respiratory and electron transfer versatility. They are able to exist in various kinds of environments and couple the oxidation of organic substrates such as riboflavins to the reduction of a wide variety of electron acceptors. When the electron acceptor are electrodes, their characteristic of transferring electrons from the cell surface to conductive materials shows great potential to improve the efficiency of bioelectrochemical systems. As a result, <SPAN style="font-style: italic">Shewanella</SPAN> has become the most widely used bacteria in MFCs.[5] </P><H3>3 Microbial consortia</H3><P>Microorganisms rarely live in isolated niches. More than 99% of microorganisms in environments cannot be successfully cultured by traditional cultivation technologies; one reason is that the maintenance of viability of these microbes may need supplementary metabolites or other signaling chemicals provided by other microbes in the ecosystems and communities. [6] </P><P style="font-weight: bolder">3.1 Interaction modes</P><P>Engineering cell-cell interactions and communications is at the central point of engineering synthetic communities. The interactions between microorganisms are ubiquitous and play a central role in determining the fate and evolutionary dynamics of individual organisms in microbial consortia, as well as system properties such as stability and dynamics of the entire communities.[6] </P><P>Also, delineation of interaction modes between microorganisms in natural niches could facilitate engineering novel microbial consortia and their traits. The interactions within the microbial ecological communities may have a null (0, or neutral), positive (+, beneficial or win), or negative (-, or detrimental or loss) impact on the partner microorganisms involved. Thus, the binary interaction outcomes could be classified into six different categories of pairwise interaction modes , i.e., neutralism(0/0), commensalism (+/0), amensalism (-/0), mutualism (+/+), competition (-/-), and parasitism or predation (+/-). [6] </P><P style="font-weight: bolder">3.2 Advantageous relationships of microbial consortia</P><P>The win-win relationship of symbiotic association is mutualism (+/+), in which both partner microorganisms derive benefits from one another. e.g., Stable co-culture in the same bioreactor was achieved by designing a mutualistic relationship between the two species in which a metabolic intermediate produced by <SPAN style="font-style: italic">E. coli</SPAN> was used and functionalized by yeast. This synthetic consortium produced 33 mg/L oxygenated taxanes, including a monoacetylated dioxygenated taxane.[7] </P><P>Commensalism (+/0) is a relationship in which one partner derives benefits from the other, while the other partner is not affected (neither harmful nor beneficial) by the association, e.g., an artificial microbial consortium consisting of <SPAN style="font-style: italic">Ketogulonicigenium vulgare</SPAN> and <SPAN style="font-style: italic">Bacillus megaterium</SPAN> for vitamin C production.[6] </P><P>Inspired by the concept of co-culture system, MFCs populated by mixed microbial communities have garnered much attention owing to their stability, robustness due to nutrient adaptability, stress resistance and general tendency to produce higher current density than those obtained using pure cultures. e.g., a mutualistic system established by the fermentative bacterium <SPAN style="font-style: italic">Escherichia coli</SPAN> and the dissimilatory metal-reducing bacterium <SPAN style="font-style: italic">Shewanella oneidensis</SPAN> in a bioelectrochemical device is reported with formate as the key metabolism.[8] </P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P align="center"><B>Figure 2.</B><SPAN style="font-size: 14px">Typical interaction modes in microbial consortia</SPAN></P></DIV></DIV><H3>4 Intracellular protein degradation</H3><P style="font-weight: bolder">4.1 Theory of protein degradation in bacteria</P><P>Modifications that target proteins for proteolysis, and thereby affect protein stability, have been extensively studied. These modifications can serve as quality control features that regulate protein synthesis, remove defective proteins, recycle amino acids and inactivate proteins. </P><P>Bacteria co-translationally tag stalled or interrupted translational products with the ssrA peptide. This 11-aminoacid signal, targets the defective translation product to cellular proteases, including ClpA/XP, Lon and etc. The ssrA signal is encoded by a small, stable RNA containing an alanyl tRNA domain and mRNA domain encoding the open reading frame of the tag (tmRNA). Upon encountering stalled or incomplete translation, the tmRNA and accessory factors are recruited to the ribosome to rescue the idle translation complex by replacing it with the tmRNA encoding the ssrA tag. The ssrA mRNA serves as the template for the addition of the ssrA peptide. The ssrA tag is added co-translationally to the carboxy-terminus (C-terminus) of incomplete proteins to target them for.[10] </P><P style="font-weight: bolder">4.2 Mf-lon</P><P>Interestingly, the ssrA tags encoded by the tmRNA molecules in most <SPAN style="font-style: italic">Mycoplasma</SPAN> are very different from those found in other bacteria. Most <SPAN style="font-style: italic">Mycoplasma</SPAN> genomes encode only two AAA+ proteases, Lon and FtsH, and have lost the genes for ClpXP, ClpAP. In the absence of ClpXP, the unusual <SPAN style="font-style: italic">Mycoplasma</SPAN> ssrA tag could serve as a degradation signal for the endogenous Lon or FtsH proteases.[11] </P><P>It has been shown that the ssrA tag sequence of <SPAN style="font-style: italic">M. florum</SPAN> (mf-ssrA) is efficiently recognized by the <SPAN style="font-style: italic">M. florum</SPAN> Lon protease (mf-Lon). Appending this tag to the C terminus of native or denatured proteins resulted in their rapid proteolysis by mf-Lon. Furthermore, mf-Lon did not degrade proteins bearing the <SPAN style="font-style: italic">E. coli</SPAN> ssrA tag (ec-ssrA), and <SPAN style="font-style: italic">E. coli</SPAN> Lon
(ec-Lon) did not efficiently degrade proteins bearing the mf-ssrA tag. [11] </P><DIV class="kuang" style="width:370px"><DIV id="Enlarge"><P align="center"><B>Figure 3.</B><SPAN style="font-size: 14px">The degradation mechanism of Mf-lon.</SPAN></P></DIV></DIV><H3>References</H3><P><SPAN style="font-weight: lighter ;font-size:14px;"> [1]Guerrero-Rangel N, Garza R D L, Garza-Garcia Y, et al. Comparative Study of Three Cathodic Electron Acceptors on the Performance of Medatiorless Microbial Fuel Cell[J]. International Journal of Electrical &amp; Power Engineering, 2010, (1):27-31 
[2]Rahimnejad M, Adhami A, Darvari S, et al. Microbial fuel cell as new technology for bioelectricity generation: A review[J]. AEJ - Alexandria Engineering Journal, 2015, 88.
[3]Du Z, Li H, Gu T. A state of the art review on microbial fuel cells: A promising technology for wastewater treatment and bioenergy[J]. Biotechnology Advances, 2007, 25(5):464–482.
[4]Kim H J, Park H S, Hyun M S, et al. A mediator-less microbial fuel cell using a metal reducing bacterium, <SPAN style="font-style: italic">Shewanella putrefaciens</SPAN>[J]. Enzyme and Microbial Technology, 2002, 30(2): 145-152
[5]Saffarini D, Brockman K, Beliaev A, et al. <SPAN style="font-style: italic">Shewanella oneidensis</SPAN> and Extracellular Electron Transfer to Metal Oxides[M] Bacteria-Metal Interactions. Springer International Publishing, 2015: 21-40.
[6]Song H, Ding M Z, Jia X Q, et al. Synthetic microbial consortia: from systematic analysis to construction and applications[J]. Chemical Society Reviews, 2014, 43(20): 6954-6981.
[7] Zhou K, Qiao K, Edgar S, et al. Distributing a metabolic pathway among a microbial consortium enhances production of natural products.[J]. Nature Biotechnology, 2015, 33:377-383. 
[8] Wang VB, Sivakumar K, Yang L, et al. Metabolite-enabled mutualistic interaction between <SPAN style="font-style: italic">Shewanella oneidensis</SPAN> and <SPAN style="font-style: italic">Escherichia coli</SPAN> in a co-culture using an electrode as electron acceptor.[J]. Scientific Reports, 2015, 5.
[9] Sauer R T, Baker T A. AAA+ proteases: ATP-fueled machines of protein destruction[J]. Annual review of biochemistry, 2011, 80: 587-612.
[10]Burns K E, Darwin K H. Pupylation versus ubiquitylation: tagging for proteasome‐dependent degradation[J]. Cellular microbiology, 2010, 12(4): 424-431.
[11]Gur E, Sauer R T. Evolution of the ssrA degradation tag in <SPAN style="font-style: italic">Mycoplasma</SPAN>: specificity switch to a different protease[J]. Proceedings of the National Academy of Sciences, 2008, 105(42): 16113-16118.</SPAN></P></DIV><DIV id="foot"><P>E-mail: ggjyliuyue@gmail.com |Address: Building No.20, No.92 Weijin road, Tianjin University, China | Zip-cod: 300072</P><P>Copyright 2015@TJU iGEM Team</P></DIV></DIV>&lt;/div&gt;</DIV></DIV></DIV></DIV></DIV></BODY></HTML>