"
Page
Discussion
View source
History
teams
Log in
 
Team:Berlin/Project/Journal
From 2014.igem.org
Home
Project
Team
Safety
Workshop
Blog
iGEM Berlin
Home
Project
Team
Safety
Workshop
Blog
Explore our Project:
1. What is it all about?
2. Project-related Activities
3. Detailed Description
4. Our Results
5. Lab Summary
6. Lab Journal
7. Intellectual Property
6
Lab Journal
04/02 Wednesday - Cultivating E. coli Nissle 1917
LB plates were produced without antibiotic. Therefore 10 ml MQ-H2O were sterile filtered in 15 ml falcon tube. 1 capsule Mutaflor 331800 was solved in the sterile water.                  The E. coli Nissle solution was incubated at 37°C and 200 rpm for 1 h and crossed out at 4 plates to get different dilution of cells.
dilution
E. coli Nissle solution
H2O
dilution factor
I
2 µl
1998 µl
1:103
II
100 µl from dilution I
900 µl
1:104
III
10 µl from dilution II
990 µl
1:105
04/03 Thursday - Genomic DNA extraction of ECN - first step
For extraction of the genomic DNA of E. coli Nissle, 2 E. coli colonies were picked from LB plate (dilution 1:103) and used for inoculation of 6 ml LB precultures. These were grown over night at 37°C and 200 rpm.
04/04 Friday - Genomic DNA extraction of ECN - second step
For preparation of the genomic DNA 1 ml of each preculture was taken and a genomic extraction performed using the Wizard Genomic DNA purification Kit. See instructions of the purification Kit by Promega. Genomic DNA was stored at 12°C in fridge.              As E. coli Nissle is a natural organism without resistence genes both precultures were streaked out on LB, LB+KAN and LB+AMP. All plates were incubated over the weekend at 37°C.              DNA concentration was measured by 260 nm using Goldi. DNA was diluted 1:40 in a UV cuvette and DNA absorption measured at 260/28 nm in goldi.
'''For genomic DNA:'''
1:
1390 ng/µl
2:
988 ng/µl
04/05 Saturday - Genomic DNA extraction of ECN - results
Bacteria growth only on LB plate indicates that there was no contamination with AMP or KAN resistant E. coli. This doesn´t mean that there is no contamination at all. See for sure on PCR.
LB
LB+KAN
LB+AMP
+
-
-
04/06 Sunday
Production of BfR, FTNA1 and FTNA2 Restriction digest After 2 purifications of plasmids:
p1
84ng/µl
10µl
p2
72,2ng/µl
10µl
p3
54,8ng/µl
23µl
Restriction program
1x
4x
plasmid P2
3µl
12µl
BamHI (FD)
1µl
4µl
HindIII (FD)
1µl
4µl
Buffer (FD)
2µl
8µl
H2O
13µl
52µl
--> 37°C for 1h (restriction)            --> 80°C for 10min (deactivation)
PCR
6,38µl
4,56µl
2,71µl
BamHI (FD)
1µl
1µl
1µl
HindIII (FD)
1µl
1µl
1µl
Buffer (FD)
2µl
2µl
2µl
H2O
19,62µl
21,44µl
23,29µl
--> 37°C for 1h (restriction)            --> 80°C for 10min (deactivation)
Plasmid
1
2
3
P
10µl
10µl
18µl
BamHI (FD)
1µl
1µl
1µl
HindIII (FD)
1µl
1µl
1µl
Buffer (FD)
2µl
2µl
3µl
nuc.free H2O
6µl
6µl
7µl
--> 37°C for 1h (restriction)            --> 80°C for 10min (deactivation)
PCR Purification (without isopropanol)
Elution buffer 20µl; 1min incubated
plasmid P1-3
45,3ng/µl
BfR
12,9ng/µl
FTNA1
34,4 ng/µl
FTNA2
29,9ng/µl
Ligation Assay
-
Control
BFR
BFR
BFR
FTNA1
FTNA1
FTNA1
FTNA2
FTNA2
FTNA2
BFR_E
BFR_E
BFR_E
Dilution
-
1x
3x
5x
1x
3x
5x
1x
3x
5x
1x
3x
5x
Vector DNA
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
1,1
Insert DNA
0
0,4
1,6
2,0
0,10
0,47
0,79
0,18
0,52
0,90
0,40
1,16
2,0
Buffer
1
1
1
1
1
1
1
1
1
1
1
1
1
T4 DNA-Ligase
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
0,25
H2O
7,65
7,25
6,5
5,65
7,5
7,20
6,85
7,50
7,20
6,85
7,25
6,50
5,65
Transformation --> Transformation of 5µl Sample into DH5α-Cells            --> Incubation o/n at 37°C after streaking out on LB+amp plates using sterile beads
BFR
FTNA1
FTNA2
5,2ng
5,51ng
5,43ng
15,6ng
16,5ng
16,27ng
26,0ng
27,4ng
27,13ng
04/07 Monday - Cloning Results
Sample
CFU
Control
13
tBFR 1
121
tBFR 3
38
tBFR 5
664
BFR 1
87
BFR 3
151
BFR 5
150
FTNA1 1
65
FTNA1 3
131
FTNA1 6
191
FTNA2 1
84
FTNA2 3
114
FTNA2 5
151
--> picked 5 colos per construct for coloPCR (50µl TE-buffer; 96°C; 10min)            --> rescue plate
colPCR Programm
-
1x
Mastermix 23x
nuc.free H2O
13,7µl
315,1µl
10x Dream Taq Buffer
2µl
46µl
25mM MgCl2
1,2µl
27,6µl
10mM dNTPs
0,5µl
11,5µl
Primer T5 prom (PB16)
0,5µl
11,5µl
Primer T5 term (PB17)
0,5µl
11,5µl
boiled colos
1µl + 18,4ml Mastermix
Taq Polymerase
0,5µl
Temperature
Time
95°C
3'
30 cycles
95°C
30
55°C
30
72°C
1'
72°C
10'
12°C
hold
04/08 Tuesday - Amplification of ferritin genes
Primer designed for amplification:
primer Bfr:
FW: 5'   GC   G| G A T C C AAAGGTGATACTAAAGTTATAAATTATCTC(GC 23,33% Tm = 57,5)  3'  OK
RW: 5'   GC   A| A G C T T TCAACCTTCTTCGCG(GC 50% Tm = 58,5)  3'                    OK
primer FtnA1:
FW: 5'   GC   G| G A T C C GCAACCGCTGGAATG(GC 60%  Tm = 60,5) 3'                    OK
RW: 5'   GC   A| A G C T T TCAATGCAGCTGATGC(GC 50,0% Tm = 58,5)  3'                 OK
primer FtnA2:
FW: 5'   GC   G| G A T C C CTGAAACCAGAAATGATTG(GC 36,8%  Tm = 65,1)  3'             OK
RW: 5'   GC   A| A G C T T TTAGTTTTGTGTGTCGAGG(GC 42,1%  Tm = 56,8)  3'             OK
Using the Thermoscientific  PCR mastermix phusion polymerase, 2x 50 µl reactions per amplification:
Nuc-free H2O
39,5 µl
2x mastermix
50,0 µl
FW primer
5,0 µl
RW primer
5,0 µl
template DNA
0,5 µl
PCR programs:
98°C
10s
1x
98°C
5s
30x
primer
Bfr/ FtnA1/ FtnA2
60,5°C/ 58,9°C/ 56,1°C
30x
72°C
10s
30x
72°C
60s
Beispiel
1x
Transformation of pQE_80L into DH5alpha
Plasmid from plasmid database of TU workgroup by Prof. Budisa was taken:
plasmid ID: 4; concentration = 308 ng/µl
Unthaw 50 µl aliquot of DH5alpha or BL21 DE3 gold on ice for 10 min.Than Use 308 ng and 616 ng of plasmids to bind on Ca
2+
-surface of the cell membranes. Heat shock was performed for 30-90 s. The cells were incubated on ice for 2 min and then there was added 950 µl LB media. The cells were incubated at 37°C for 60 min. After this 70 µl of transformed E. coli suspension was streaked out onto a plate with the appropriate selection marker. The plate was incubated over night at 37°C. To prepare preculture 2 colonies were picked and added into 5 ml LB media + 5 µl AMP. The precultures were incubated over night at 37°C and 200 rpm.
04/10 Thursday - Expression of Ferritin in LB
→Inoculate 20ml LB + amp with 1ml of a preculture
Title
No.
Wavelength
Absorbance
Volume
BfR
1
600nm
0,529A
1,51ml
FTNA1
2
600nm
0,307A
3,25ml
FTNA2
3
600nm
0,605A
1,32ml
→Incubation for 2h until OD600=0,6-08
Title
No.
Wavelength
Absorbance
BfR
1
600nm
0,825A
FTNA1
2
600nm
0,883A
FTNA2
3
600nm
0,859A
→ Take SDS Sample non-induced
→Induce with 20µl IPTG
→Add Fe2+
→Expression: 4h at 22°C
Cultivation was aborted because of heat development
.
04/14 Monday - Miniprep
Miniprep of the cell-separation streak out of the previous cloning procedure
Miniprep-Kit of ThermoScientific
-
Title
Concentration in ng/µl
A4
BfR
66
B3
FTNA1
138
C5
FTNA2
88
D2
BfR_Electro
144
DNA Concentration Determination
Dilution Factor
20
Integration Time
1s
Factor
50
Units
µg/ml
-
Title
No.
260nm
280nm
320nm
Ratio
Conc.
A4
BfR
1
0,056
-0,022
-0,010
2,05
66
B3
FTNA1
2
0,135
0,067
-0,003
1,96
138
C5
FTNA2
3
0,078
0,035
-0,008
2,00
86
D2
BfR-Electro
4
0,150
0,078
0,006
2,02
144
04/16 Wednesday - Sequencing
Preparation of the sequencing for the Samples from 16.01.2013
T5 prom. Primer
3µl
Plasmid
12µl
A4
6µl
B3
6µl
C5
6µl
D2
6µl
04/17 Thursday - Expression of BfR and FTNA1/2
4x sterile 250ml Erlenmeyer
+50ml LB
+50µl amp
+2,5ml preculture
Incubation: 1h at 30°C
No.
Title
Wavelength
Absorbance
1
BfR
600nm
0,909A
2
FTNA1
600nm
0,980A
3
FTNA2
600nm
0,930A
4
BfR+20??
600nm
0,970A
→Take SDS-Sample
→Induction with 50µl IPTG
→Addition of 0,0525g Fe-citrate (20mM dissolved in 1ml sterile Water)
→Incubation for 3h at 30°C
→continue incubation o/n
→Centrifuge with 6800g for 5min
04/18 Friday - Production of LB-Plates
Production of a Mutaflor-Supression-culture →10ml sterile LB
→Add the content of a Mutaflor capsula (white powder)
Incubation: 37°C at 220rpm
Production of dilutions of the Suppression-culture and out streaking →1µl of the suppression-culture to 999µl LB (1:1000)
→100µl of that dilution to 900µl LB (1:104)
→10µl of that dilution to 990µl LB (1:105)
→Streak out 50µl of each culture on LB with sterile beads
Incubation: 37°C
04/24 Thursday - Mini-Prep
1
PSB
D40
9fP
CR/LB
2
PSB
D20
9fP
CR/LB
3
PSB
D40
fs
LB/CR
4
PSB
D40
LB/CR
5
PSB
AB
Turyu
CR/LB
6
PSB
D48
9fP
CB/CR
04/28 Monday -PCR Plasmid/ Primer and Parameter Check
(Salah and Christina)
Check of:
-Plasmids/Primer
-Annealing temperature
-Phusion and Q5 Polymerase Plasmids
iGEM Plasmid pkD4 (Stock 200ng/µl) =>1:200 diluted = 1ng/µl
WH (Willi Hauck) Plasmid pkD4 (0,72ng/µl)
Primer
P1 FieF (iGEM)                                   fwd tnaA KO (WH)
P2 FieF (iGEM)                                   rev  tnaA KO (WH)
Combinations
(4 combinations with Phusion; 4 combinations with Q5 )
Plasmid
Primer
Primer
1.
iGEM Plasmid
P1 (iGEM)
P2 (iGEM)
2.
WH Plasmid
fwd tnaA KO
rev  tnaA KO POSITIVCONTROL
3.
iGEM Plasmid
fwd tnaA KO
rev  tnaA KO
4.
WH Plasmid
P1 (iGEM)
P2 (iGEM)
Each of the four combinations was prepared 3 times => for Gradient PCR
Phusion Polymerase PCR Assay
20µl Reaction
Plasmid
Primer
Primer
1.
iGEM Plasmid
P1 (iGEM)
P2 (iGEM)
2.
WH Plasmid
fwd tnaA KO
rev  tnaA KO            POSITIVCONTROL
3.
iGEM Plasmid
fwd tnaA KO
rev  tnaA KO
4.
WH Plasmid
P1 (iGEM)
P2 (iGEM)
Q5 Polymerase PCR Assay
20µl Reaction
5xQ5 Reaction Puffer (G2 round container)
4µl
1x
10mM dNTPs
0,4µl
200µM
10µM forward Primer
1µl
0,5µM
10µM reverse Primer
1µl
0,5µM
Template DNA
1µl
1pg-1ng sein
Q5 High Fidelity DNA Polymerase
0,2µl
0,02U/µl
GC enhancer
4 µl
Nuclease-Free Water
to 20µl
8,4µl
FUR PCR Assay
(Along with the above mentioned PCR Assays another Gradient PCR with FUR-Primer is performed)
Aliquot of the FUR Primer (100µM) 1:10 diluted =>10µM
20µl Reaction with Phusion-Polymerase (see above)
with WH Plasmid and P1 FUR & P2 FUR Primer
Gradient PCR Program
Thermocycling Conditions for the Gradient PCR
Programm-NAME: Q5 grad trp KO
Step
TEMP
TIME
Initial Denaturation
98°C
30 sec
5 Cycles
98°C
8 sec
64±5°C
25 sec    GRADIENT
72°C
Trp ko: 1528bpà45s
25 Cycles
98°C
8 sec
72°C
70 sec
Final Extension
72°C
2 min
Hold
8°C
Gel-Electrophoresis
1% Agarose Gel
5µl Sample +1µl loading dye (1:6)
8µl diluted GeneRuler Mix ladder
MODE: 90V 25min
Sample label:
P = with Phusion Polymerase
Q = with Q5 Polymerase
1 = 1. Combination
2 = 2. Combination
3 = 3. Combination
4 = 4. Combination
Gradient
1 = 59°C
4 = 63°C
8 = 69°C
Gel 1 Phusion
'
'
'
'
'
'
'
'
'
'
'
'
'
P FUR
P1
P1
P1
P2
P2
P2
Marker
P3
P3
P3
P4
P4
P4
8
1
4
8
1
4
8
1
4
8
1
4
8
Gel 2 Q5
Q1
Q1
Q1
Q2
Q2
Q2
Marker
Q3
Q3
Q3
Q4
Q4
Q4
1
4
8
1
4
8
1
4
8
1
4
8
Results
Gel 1 Phusion Result:
For FUR-Primer no amplification
For iGEM-Plasmid no amplification
Combination 2 (WH Plasmid & Primer)= only non-specific amplification
Combination 4 (WH Plasmid & iGEM Primer)=Bands for 59°C and 63°C + non-specific amplification
Gel 2 Q5 Result:
For iGEM-Plasmid no amplifikation
Combination 2  (WH Plasmid & Primer) = bands for all three annealing temperature observed+non-specific amplification
Combination 4 (WH Plasmid & iGEM Primer)= Bands at all three annealing temperature+non-specific amplification
iGEM Plasmid not to be used anymore.             FUR need to be checked again.
04/29 Tuesday - Mini-Prep
(Aritra)
[pKD46 + DH5α]→ 155ng/µl
[pKD4 + DH5α]→ 302ng/µl
05/02 Friday - Genomic DNA extraction from Pseudomonas putida
(Johann)
-->see Protocol Wizard ® Genomic DNA Purification Kit                  - Strain from VLB Martin Senz PHO Psedomonas putida KT2242  B_0712  from 29.04.2014 culture
- 2 days in 30°C  Shaking Incubator.
- Mikro 22R Centrifuge Hettich (16000g)
Final DNA Concentration measured after purification = 95ng/µl
02.05.2014 - DNA Digestion to test Plasmids
Samples
Conc. DNA (ng/µl)
Enzymes used
Expected band length
Evaluation
pKD4
302
Xbal
1874,1393
Possible plasmid, failed
pKD46
40
EcoRI
4820,1509
Failed
pKD46
39
EcoRI
4820,1509
Failed
pKD4
60
Xbal
1874,1393
Possible(passed); whole plasmid
pSBAC3
88
SpeI/EcoRI
2047,22
Possible (passed); smallpattern
pSBAC3-D20-GFP
115
SpeI/EcoRI
2047,957
whole plasmid; failed
pSBAC3-D40-Bfr
86
SpeI/EcoRI
2047, 774
Whole plasmid; failed
pSBAC3-D40-GFP
80
SpeI/EcoRI
2047,1014
Whole plasmid; failed
pSBAC3-AB-Tyrosin
110
SpeI/EcoRI
2047,3319
Failed
pSBAC3-D40
164
SpeI/EcoRI
2047,267
Failed
pSBAC3-D40-GFP-ssr
106
SpeI/EcoRI
2047,1032
Passed; whole plasmid
Gel Electrophoresis Key:
-band1: Ladder
-band5:Ladder
-residual bands, see table above
Results pKD4 with 60(ng/µl) best candidate for PCR.
05/05 Monday - Preculture of strains from Budisa strain database in LB Medium
1. BU31 (pKD4) in 50 ml LB+Kanamycin+Ampicillin  37°C 220 rpm
2. BU32 (pKD46) in 50 ml LB+Ampicillin 30°C 240rpm
3. BU31 (pKD46) in LB plate+Ampicillin  30°C
Midiprep Midiprep of BU31 (pKD4) and BU32 (pKD46) from strain Database.
Concentration pKD4 = 104 ng/µl
Concentration pKD46 = 161 ng/µl
05/06 Tuesday - Gel-Electrophoresis
Kanamycin Cassette PCR (pKD4 from MidiPrep 05.05.2014 104ng/µl Christina)
Key:
- 1st: Ladder
- 2nd and 3rd: Fief
- 4th: Negative control
- 7th and 8th: FUR
Results FieF worked well; FUR no bands Restriction digest of Plasmid pKD4 and pKD46
'
'
'
pKD4
pKD46
Plamid
19.2µl
12.5µl
Digestion Enzym
Xbal 1µl
Xmal 1µl
nuclease free H2O
24.8µl
31.5µl
10xFO green Buffer
5µl
5µl
Total
50µl
50µl
- Both the reaction mix incubated at 37°C for 1 hr.               - After that another 1 hr at RT.
Gel-Electrophoresis - 20µl of both the samples together with whole plasmids loaded on 1% Agarose Gel
Results Key:
-1st: pKD4 (XbaI digested)
-2nd: pKD4 (undigested)
- 3rd: Ladder
- 4th: pKD46 (undigested)
-5th: pKD46 (XmaI digested)
Gel Extraction -Used GeneJet GelExtraction Kit by Thermo Scientific Results DNA Concentration: 16ng/µl
05/16 Friday - Biotransformation of Ferritin and pKD46 in Nissle and DH5-α strain
concentration Ferritin 120ng/µl
concentration pKD 46 95 ng/µl
Protocol            *electro competent cells of DH5α and Nissle were thawed on ice            *0.4µl of Ferritin and 0.53 µl of pKD46 was added to the cells            *cells with DNA were transferred to cuvettes for electroporation, then 950ml (LB-medium) was added to the cells immediately            *incubation at 37°C for ferritin, 30°C for pKd46 for 1 h with shaking            *the cells were plated on Amp plates for pKD46 and CM for Ferritin            *Transformation did not work.
05/21 Wednesday - PCR Gel-Extraction of FieF
1% Agarosegel + 0,5µl EtBr
45µl Sample+9µl Loading Dye
8µl Ladder
--> Gel-Extraction Kit used with 50µl nuclease-free water for elution
Gene Knockout of FieF (Salah) Preparation of the preculture Used Precultures:
- RV +pKD46
- WM10 +pKD46
The precultures was diluted to OD600=0,1 and then incubated for 2h at 30°C until they reach OD600=0,6
Title
Wavelength
Absorbance
Reference
600nm
0,000A
RV+pKD46
600nm
0,066A
WM10+pKD46
600nm
0,050A
(The culture was diluted 1/10 for the OD measurement)
→The precultures are ready for the recombinase induction with Arabinose Induction with Arabinose The cultures have been induced with 500µl arabinose (1M)            →After the induction, the cultures have to be incubated for further 30min at 30°C
Title
Wavelength
Absorbance
Reference
600nm
0,000A
RV+pKD46
600nm
0,084A
WM10+pKD46
600nm
0,067A
→From now on the cultures need to be cooled on ice Preparation for the Electroporation
Electroporation Electroporation-Program: Ec1
-Amount of DNA sample: 75-150ng
Sample
Conc
Used Volume
Strain
Electroporation Time
FieF1
42ng/µl
2µl (84ng)
RV
4,4ms
FieF2
9ng/µl
10µl (90ng)
RV
4,1ms
FieF1
42ng/µl
2µl (84ng)
WM10
4,3ms
FieF2
9ng/µl
10µl (90ng)
WM10
4,8ms
After the transformation, the cultures was immediately prepared with 950µl SOC and incubated for 60min at 30°C without antibiotics --> recovery Selection-cultures The selection of positive clones is done by using LB plates with kan+amp
Incubation: 30°C o/n
05/22 Thursday - Transformation of pKD46
Preparation of the preculture Used Precultures:
- MG1655
- Nissle
The precultures was diluted to OD600=0,1 and then incubated for 2h at 30°C until they reach OD600=0,6
Title
Wavelength
Absorbance
Reference
600nm
0,000A
MG1655
600nm
0,062A
NIssle
600nm
0,054A
(The culture was diluted 1/10 for the OD measurement)
→From now on the cultures need to be cooled on ice Preparation for the Electroporation
Electroporation Electroporation-Program: Ec1
-Amount of DNA sample: 75-150ng
Sample
Conc
Used Volume
Strain
Electroporation Time
MG1655 1
42ng/µl
2µl (84ng)
RV
5,4ms
MG1655 2
9ng/µl
10µl (90ng)
RV
5,8ms
Nissle 1
42ng/µl
2µl (84ng)
WM10
0,7ms (thrown away)
Nissle 2
9ng/µl
10µl (90ng)
WM10
5,8ms
After the transformation, the cultures was immediately prepared with 950µl SOC and incubated for 60min at 30°C without antibiotics --> recovery Selection-cultures The selection of positive clones is done by using LB plates with amp
Incubation: 30°C o/n
PCR of ATPCS and PPMT
Mastermix
ATPCS
PPMT
nuc.free water
18.9
17.5
2x Fusion Mastermix
25
25
Fw Primar
2.5
2.5
Rw Primar
2.5
2.5
template DNA
1.1
2.5
ATPCS
Program
98°C
10
98°C
5 30 cycles
62°C
5  30 cycles
72°C
5   30 cycles
72°C
60 secs
8°C
store
PPMT
Program
98°C
10
98°C
5 30 cycles
63,1°C
5  30 cycles
72°C
21  30 cycles
72°C
60 secs
8°C
store
ATPCS 1469 bp; PPMT 234 bp            --> wrong Program
05/23 Friday - Digestion of Vector for Cloning
Plasmid: pQE80L
Old Plasmid of Johann:
-P1≈1000ng
-P3≈400ng
P3
P1
SacI
1µl
SacI
1µl
HindIII
1µl
BamHI
1µl
Buffer
2µl
Buffer
2µl
Plasmid 1
4µl
Plasmid 3
16µl
nuc.water
12µl
nuc. water
0µl
--> Incubation: 1,5h at 37°C
--> Deactivation: 20min at 80°C
05/27 Tuesday - Production of LB Plates
LB Agar Plates prepared:
4x with Ampicillin   (25µl in 25ml)
4x with Ampicillin+ Kanamycin   (25µl+25µl in 25ml)
4x Cm (25µl in 25ml)
05/28 Wednesday
Picked up from MPI: AMB-1 and Microfluidic Chip. (From  the two AMB-1 cultures, one is in the freeze and the other one in the RT-Incubator)
Digestion of pQE80L:
Marker: 2 log DNA ladder 5µl
Sample 1: P1 Sacl-BamHI 10µl
Sample 2: P3 Sacl-HindIII 10µl
06/02 Monday - Colony PCR and analytical gel electrophoresis for identifying the right clones
Primer: C1  C2
C2   C5
Components
20 µL Reaktion
Nuc. Free water
13.7
10x Dream Taq Polymerase Puffer
2
25mM MgCl2
1.2
10mM dNTPs
0.5
10mM FW Primer
0.5
10mM RW Primer
0.5
Colony
1
Taq DNA Pol
0.6
Denaturation
95°C
10min
'
Denaturation
95°C
30sec
Annealing
55°C
30 sec
Extension
72°C
X min
30x
Extension
72°C
10 min
End
12
hold
Backup plates done. Cells given direct to PCR. 11 colonies selected.
Colony PCR  with some of the clones performed. (All negative)
06/03 Tuesday - Strategy 2: PCR of ATPCS 2PPMT
PCR Assay
-
Sample1(PPMT)
Sample2(ATPCS)
nuc. free water
17,5µl
18,9µl
FW Primer
2,5µl
2,5µl
RW Primer
2,5µl
2,5µl
2x Phusion Mastermix
25µl
25µl
Template DNA
2,5µl (250ng)
1,1µl (250ng)
PCR Program
PPMT (234bp)
ATPCS (1469bp)
98°C
10'
98°C
10'
30cycles start
98°C
5"
98°C
5"
62°C
5"
63°C
5"
72°C
10"
72°C
45"
30cycles stop
72°C
1'
72°C
1'
8°C
hold
8°C
hold
Gel-Electrophoresis Key:
2nd band= Ladder
3rd band= PPMT
5th band= ATPCS
06/04 Wednesday - PCR amplification of ATPCS and PPMT
(see under protocol "Amplification BamHI_PPMT_Sacl and Sacl_ATPCS_HindIII"                After Gel: 1 Band visible for ATPCS around 1500bp (expected); no band for PPMT                Purification of the ATPCS-PCR product from gel.                conc. of ATPCS-fragment 6ng/microL                freezed at -20°C on 1st floor
04.06.2014 - Colony PCR for Knockout
6 clones were picked from 2 Agar plates; Agar plates labeled with numbers with RED
3 PCR:  a) Primers C1, C3
b) Primers C2, C4
c) Primers C1, C5
BandNr:
Clone Nr:
PCR type
Primers
1
6
C
C1,C5
2
1
A
C1,C3
3
2
A
C1, C3
4
3
A
C1, C3
5
4
A
C1, C3
6
5
A
C1, C3
7
6
A
C1, C3
8
1
B
C2,C4
9
2
B
C2,C4
10
Marker
11
3
B
C2,C4
12
4
B
C2,C4
13
5
B
C2,C4
14
6
B
C2,C4
15
1
C
C1,C5
16
2
C
C2,C4
17
3
C
C2,C4
18
4
C
C2,C4
19
5
C
C2,C4
Results: all samples with same primers are identical                PCR A: one band an dimer (900-800bp)                PCR B: one band (1200bp)                PCR C: two bands (800 and 400 bp)                               - 2 of the colonies are quite luckily mix colonies that contains our knockout, bands are shown for wild type and 2 show amplification for kanR cassette               - they should be transferred to a new plate for single colony test
06/05 Thursday - New PCR protocol for ATPCS and PPMT with Q5 High Fidelity MasterMix
PPMT PCR Reaction:
Component
25µl
Final Conc.
2x Q5 MM
12. Mai
1x
10µM FW Primer
02. Mai
0,5µM
10µM RW Primer
2,5
0,5µM
95ng/µL Putida genomic DNA
2.63 (250ng)
9.99 ng/µL
Nuc. Free water
Apr 87
ATPCS PCR Reaction
result
Component
25µl
Final Conc.
2x Q5 MM
1.,5
1x
10µM FW Primer
Jan 25
0,5µM
10µM RW Primer
Jan 25
0,5µM
230ng/µL Arabidopsis  cDNA
1.1 (250ng)
10 ng/µL
Nuc. Free water
08. Sep
PCR Program PPMT 234 bp result
denaturing
98°C
180sec
Denaturing
98°C
10 sec
Annealing            30x
64°C
30 sec
Elongation
72°C
20 sec
Final Elongation
72°C
2min
Store
8°C
ATPCS 1469bp result
denaturing
98°C
30 sec
Denaturing
98°C
10 sec
Annealing               30x
62°C
30 sec
Elongation
72°C
60 sec
Final Elongation
72°C
2min
Store
8°C
05.06.2014
Cryostocks: RV 308 ferritin (Cm)
Nissle ferritin (Cm)
500 µl 7%DMSO+ 500 µl Cell Culture medium => -80°C
06/06 Friday - PCR for ATPCS and PPMT with Q5 High Fidelity MasterMix
PPMT PCR Reaction:
Component
25µl
25µl
Final Conc.
2x Q5 MM
12. Mai
12,5
1x
10µM FW Primer
02. Mai
Jan 25
0,5µM
10µM RW Primer
2,5
Jan 25
0,5µM
95ng/µL Putida genomic DNA
5,26 (500ng)
5,26 (500 ng)
Nuc. Free water
Apr 87
4,74
ATPCS PCR Reaction
Component
25µl
Final Conc.
2x Q5 MM
12. Mai
1x
10µM FW Primer
Jan 25
0,5µM
10µM RW Primer
Jan 25
0,5µM
230ng/µL Arabidopsis cDNA
2.2 (500ng)
Nuc. Free water
08. Sep
PCR program
PPMT 234 bp
denaturing
98°C
120sec
Denaturing
98°C
10 sec
Annealing 30x
64°C
30 sec
Elongation
72°C
20 sec
Final Elongation
72°C
2min
Store
8°C
ATPCS 1469bp
denaturing
98°C
30 sec
Denaturing
98°C
10 sec
Annealing 30x
62°C
30 sec
Elongation
72°C
60 sec
Final Elongation
72°C
2min
Store
8°C
10µl of PCR reaction + 2µl 6XDNA loading dye was loaded onto a 1 % agarose gel
ATPCS (one clear thick expected band at Test of purity and quality of genomic DNA by running it through a 1% agarose gel1400 bp)
PPMT1 (no band)
PPMT2 (no band)
>
Test of genomic DNA of P.Putida for purity and quality by running it on a 1% agarose gel
- looks fine on a gel. One clear band over 10 000 bp
!!! Improve PPMT !!!:
Try using DMSO (NEB PCR grad) and set annealing temperature down to 55°C
Try different DMSO conc. : 0; 1; 5; 8  %
06/11 Wednesday - Continuation of work from 06.06.2014: PCR for (ATPCS and) PPMT with Q5
PPMT PCR without success last weak. Therefore the temperature of the annealing step should reduce from 64°C up to 55°C in the pcr program.
1) corrected pcr programm for PPMT
'
T [°C]
t [s]
Denaturating
98
120
Denaturating
98
10
Annealing
55
30
Elongation
72
60
Final Elongation
72
120
Store
8
2) PPMT pcr reaction
Different samples with concentrations on DMSO 0%, 1%, 5%, 8% were prepared. Use 99,97% DMSO the samples were prepared with the following pattern:              The final volume is 25μl.
'
#1 [μl]
#2 [μl]
Q5 MM
12,5
12,5
10 μl FW Primer
2,5
1,25
10 μl RW Primer
2,5
1,25
Putida genomic DNA 95 ng/μl
5,26
5,26
dazu 1.: DMSO,
0
0
nuclease free H2O
2,24
4,74
dazu 2.: DMSO,
0,25
0,25
nuclease free H2O
1,99
4,49
dazu 3.: DMSO,
1,25
1,25
nuclease free H2O
0,99
3,49
dazu 4.: DMSO,
2
2
nuclease free H2O
0,24
2,74
the 7 samples were labeled by the following schema:
PPMT.1.1, PPMT.1.2, PPMT.1.3, PPMT.1.4 (Primer je 2,5μl, DMSO Konz. 0,1,5,8 %)
PPMT.2.2, PPMT.2.3., PPMT.2.4 (Primer je 1,25μl, DMSO Konz. 1,5,8 %)
06/12 Thursday - PCR of PPMT
Amplification of PPMT with DMSO
''(Continuation of work from 06.06.2014: PCR for PPMT with Q5''                For Protocol please see lab journal from 11.06.2014
following labeling for 8 samples was used:
PPMT.1.1, PPMT.1.2, PPMT.1.3, PPMT.1.4 (Primer je 2,5μl, DMSO Konz. 0,1,5,8 %)
PPMT 2.1, PPMT.2.2, PPMT.2.3., PPMT.2.4 (Primer je 1,25μl, DMSO Konz. 0,1,5,8 %)
Gel Electrophoresis
1µl loading buffer+5 µl sample  loaded on 1% agarose gel; 90V
(rest of sample in iGEM freezer)
Results
contamination from water (band 3000bp for every sample)
with higher DMSO conc. (5% and 8%) bands from visible
06/17 Tuesday - PCR Amplification of ATPCS from cDNA
PCR-pipetting scheme
Volume [µl]
Chemicals
12,5
2x Q5 Mastermix
1,25
10µM fwd Primer
1,25
10µM rev Primer
2,2
Arabinopsis Thaliana cDNA
8,9
nucl.free H2O
26,1µl total volume
PCR-program
PCR-step
Temperature [°C]
Time [sec]
Denaturation
98
30
"30 cycles start"
Denaturation
98
10
Annealing
62
30
Elongation
72
60
"30 cycles stop"
Final Elongation
72
120
Store
8
PCR Purification of ATPCS - Kit Purification
- Elutionbuffer: 30µl
- Final concentration: 69 µg/ml
06/18 Wednesday - PCR of drag outs to separate cells for FieF Knockout
Pick up 7 single colonies + 20µl d. H2O each (on ice)
--> Labeling: K1, K2, K3, K4, K5, K6, K7
PCR for each colony with Primer combinations: C1-C3, C2-C4, C1-C5 --> 7x3= 21 Samples
PCR Assay
Amount [µl]
Chemical
1
Sample (Colony)
13,5
d. H2O
2
10x Dream Taq Polymerasebuffer
1,2
25mM MgCl2
0,5
10µM fwd Primer
0,5
10µM rev Primer
0,5
10mM dNTPs
0,6
Taq DNA Polymerase
PCR Program Colony-PCR: Elongation Time : 1,2 Min, Annealing Temp.: 55°C
Gel Electrophoresis
Result RV-strain is not appropriate for Fief-Knockout. Unknown genome leads to unspecific binding of designed primers, too many bonds in gel electrophoresis picture.
06/19 Thursday
Continue working with colonies: K1, K5, K6, K7 from 18.06.2014
PCR-programme:
Amount [µl]
Chemical
1
Sample (Colony)
13,5
d. H2O
2
10x Dream Taq Polymerasebuffer
1,2
25mM MgCl2
0,5
10µM fwd Primer C1
0,5
10µM rev Primer C4
0,5
10mM dNTPs
0,6
Taq DNA Polymerase
PCR-program:
Elongation time: 2Min, Annealing Temperature: 56°C
Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 1 of 4
Aim:               Test the possibility of Magnetization of E.coli cells by the expression of human Ferritin:
Procedure:
To evaluate the effect of Ferritin  on the Magnetization different cultures were compared:
For Both RV308 and Nissle
Culture
1
2
3
4
Plasmid
+
-
+
+
iron
+
+
-
+
induction
+
+
+
-
- Transform the Human Ferritin gen carried on the Plasmid PC514 provided from (iGEM Team of Calgary university, Canada) to E. coli strains              (Nissle and  RV308) via electroporation, plate on (Amp) plates for selection.
06/20 Friday Restriction digest of pQE_80L and ATPCS-PCR fragment
1x
5x Mastermix
pQE_80L
H2O nucfree
23,1µl
115,5µl
2µl
10x green FD buffer
2µl
10µl
2µl
DNA
2,89µl
14,5µl
14µl
BamHI
1µl
0µl
1µl
SacI
1µl
0µl
1µl
5x Mastermix = Mastermix for ATPCS
Enzymes were added to each single aliquot and not into the Mastermix
Incubation: for 1,5 h at 37 °C
Deactivation: 80°C for 5 min
I used the wrong restriction enzymes as I got confused with the PCR fragment. Digestion with SacI and HindIII is needed not  BamHI. Repeat PCR for ATPCS as well as restriction digest!
Test expression of Ferritin in RV308 (pSB1C3_Ferrtin) and Nissle (pSB1C3_Ferrtin)
5ml overnight culture used as an inoculum.
4 ml culture were diluted to 40 ml with LB and grown for 1 h at 37°C until an OD600 of 0.7 and 0.8 was reached.
Non-induced SDS sample was taken and induced with 1 mM IPTG
After expression for 3 h induced SDS sample was taken. SDS PAGE Ferritin band expected at 21kDa
Key:
- 1st band: Nissle (non induced)
- 2nd band: RV308 (non induced)
- 3rd band: Ladder
- 4th band: Nissle (induced)
- 5th band: RV308 (induced)
06/25 Wednesday - Amplification of ATPCS (cDNA)
PCR-pipetting scheme
Volume [µl]
Chemicals
12,5
2x Q5 Mastermix
1,25
10µM fwd Primer
1,25
10µM rev Primer
1,5
Arabinopsis Thaliana cDNA
8,9
nucl.free H2O
25,4µl total volume
PCR-program
PCR-step
Temperature [°C]
Time [sec]
Denaturation
98
30
"30 cycles start"
Denaturation
98
10
Annealing
62
30
Elongation
72
60
"30 cycles stop"
Final Elongation
72
120
Store
8
PCR Purification -Purification Kit
→DNA concentration measurement showed, that the Absorbance ratio of 260nm:280nm is quite low, which means that targeted DNA got contaminated.
Preparation of pre-culture Strains:
- e.coli nissle
- e.coli nissle + ferritin
- RV308
- RV308 + ferritin
Assay - 5ml LB media + 1 picked clone of the pre-day
- Addition of 5µl CN37 to the strains with ferritin
Incubation: 37°C; 170rpm; o/n
06/27 Friday
Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 2 of 4 - Preculture of 1 positive clone over night ( preculture: 7 ml LB+ 7µL Amp + 1 Clone)
06/28 Saturday
Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 3 of 4
- Transfer the Precultures (OD= 0.4-0.5) to 25, 50 mL flasks (half filled with LB) and cultivate them at 37°C for 3 hours              RV308 (+,+,+) & (-,+,+) And Nissle (+,+,+) were cultivated in 50 mL flasks
Strain:  OD
RV308    0,78
Nissle   0,86
- Induction with (20µl (50 ML flasks)/10µl (25 ML flasks)) IPTG (concentration 1 M).
- Add FeCl2 and MnCl2 (100µL in 50ml flasks/ 50µl in 25ml flasks) Incubation over night at 37C°
Strain:  OD
RV308    2.56
Nissle   2.8
06/29 Sunday
Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 4 of 4 -Sampling for testing the Ferritin content via SDS-Page, different ODs were taken in consideration!
-Prepare the sample for SDS-gel by centrifugation and discard the supernatant then add (60µl Dis.water + 15 SDS+ MH-EtoH), boil for 10 min at 94°C.
- Test the Magnetization using an permanent magnet under light microscope.
Evaluation:
SDS Page:  was not clear!
Magnetization :
RV308
Culture
1
2
3
4
Plasimd
+
-
+
+
iron
+
+
-
+
induction
+
+
+
-
Culture
1
2
3
4
Plasimd
+
-
+
+
iron
+
+
-
+
induction
+
+
+
-
Nissle In samples with Iron aggregates were observed with strong Mag.effect.
06/30 Monday - Colony PCR and Ligation ATPCS
PCR pipetting scheme
Volume [µl]
Chemicals
13,7
nucl.free H2O
2
10x Dream Taq Buffer
1,2
25mM MgCl2
0,5
10mM dNTPs
0,5
10µM fwd Primer
0,5
10µM rev Primer
1
Sample (Colony)
0,6
Dream Taq DNA Polymerase
20µl Total Volume
PCR program
PCR-step
Temperature [°C]
Time [sec]
Denaturating
95
180
30 cycles start
Denaturating
95
30
Annealing
55
30
Elongation
72
120
30cycles stop
Final Elongation
72
600
Store
12
Gel Electrophoresis 1% Agarose gel with EtBr; 8µl sample; 5µl Ladder
"Key:"
First band= Ladder
Each other band is a different picked clone (10 clones were picked)
07/01 Tuesday - ATPCS Ligation
The ligation of ATPCS was checked with 1% Agarose gel (stained with ethidiumbromid) after colony pcr.
07/02 Wednesday - Knockout of Fief and Fur in RV308 and Nissle
To prepare the knockout 5 µl CM solution (pFerritin with canamycin resistance cassette)and 500 µl culture solution were added to 5 ml LB media. For each stain two samples were determined (RV1/2 and N1/2). The samples were incubated at 37°C up to an OD600 of 0.53-0.55.                After incubation 1 ml of each culture was transferred in sterile eppi and washed for 7 min at 7000 rpm and 4°C. The supernatant was discarded, the pellet was resuspended in 1 ml sterile H2O and washed for 7 min at 7000 rpm and 4°C and discarded the supernatant.                 The pellet of N1 was resuspended in 25 µl pRFP (c = 41 ng/µl) and 25 µl sterile H2O. The pellets of N2 and RV1/2 were resuspended in 2 µl pRFP and 50 µl sterile H2O. 50 µl of each suspension were transferred in electroporation cuvette and electroporated for a few sec (table below). Directly after this, 1 ml fresh LB media (37°C) was added, the suspension was transferred in a sterile eppi and incubated at 37°C at 600 rpm. The cultures were recovered, deleted on CM+Amp plates and incubated at 37°C ove night.
Überschrift
[kV]
[ms]
N1
1,8
2,3
N2
1,2
5,4
RV1
1,2
5,3
RV2
1,5
5,3
Transformation of stains RV, Nissle, WM
The stains RV308[ferritin], Nissle[ferritin], RV308, Nissle and WM110 were transformed with RFP. RV308[ferritin] and Nissle[ferritin] were deleted on CM+Amp plates and 5 ml culture was added with CM/Amp and incubated at 37°C. The transformed strains RV308, Nissle and WM110 were deleted on Amp plates and 5 ml culture was added with Amp and incubated at 37°C.
Results of transformation
stain
colonies on plate
cells in culture
Nissle1 + RFP
3
Yes
Nissle2 + RFP
19
Yes
WM110 1 + RFP
0
Yes
WM110 2 + RFP
n.d.
Yes
RV308 1 + RFP
0
Yes
RV308 2 + RFP
87
Yes
Nissle[ferritin]1 + RFP
108
No
Nissle[ferritin]2 + RFP
0
No
RV308[ferritin] 1 + RFP
330
Yes
RV308[ferritin] 2 + RFP
300
Yes
The cultures of WM110 1 + RFP and WM110 2 + RFP were deleted on Amp plates and incubated at 37°C. Moreover 5 ml precultures from RV308 + RFP + Amp and RV308[ferritin] + RFP + CM+Amp were prepared.
07/03 Thursday - PCR amplification of knockout cassette FUR/FieF
The pcr preparation was performed at the pattern in the table below.
pcr preparation
FUR (25µl)
FieF (50µl)
Os High Fedility 2x Mastermix
12,5 µl
25 µl
forward primer 10mM
1,25 µl
2 µl
reverse primer 10mM
1,25 µl
2,5 µl
pkD4 plasmid DNA 1 ng/µl
1 µl
1 µl
nuclease-free H2O
to 25 µl
to 50 µl
The amplification was performed with the program in the table below.
step
T [°C]
t [sec]
initial denaturation
98
30
5 cycles
98/ 63/ 72
8/ 25/ 45
25 cycles
98/ 72
8/ 70
final elongation
72
120
--> hold at 8°C
The amplification results were analyzed in agarose gelelectrophoresis (1% agarose; 0,5µl Gel Red; 10x diluted ladder; 6x loading dye).
07/04 Friday - Expression von RV308 (RFP) und RV308 (RFP + Ferritin)
50 ml LB + 50 µl Amp inoculated with 5 ml overnight culture of RV308 (RFP)
50 ml LB + 50 µl Amp/Cm inoculated with 5 ml overnight culture of RV308 (RFP + Ferritin)
After a OD of 0.6-0.7 was reached 50 ml cultures were portioned into 4 sterile 100 ml flasks so that there was a final volume of 10 ml in each flask.
The cultures were then cultivated the following:
RV308 (RFP)
RV308 (RFP + Ferritin)
Induced 1 mM IPTG
+
-
+
-
+
-
+
-
Fe/Mn each 1 mM*
+
+
-
-
+
+
-
-
*Fe/Mn were solubilized by autoclaving and 1M stock solution
Although not planned IPTG and metal ion solutions were added simultaneously due to time constraints. It was planned to add metal ions 2h after induction
07/07 Monday - Preculture prepared of:
1. ATPCS Klon 1 - 10 (Amp)  --> check if anything grew and contact Johann
2. WM110 (RFP), RV308 (RFP), Nissle (RFP), RV308 (Ferritin + RFP) --> make Cryostock for each
Overgrown WM110 (RFP) plate streaked out again, please take out and put in cool room
Rune:
PCR of PPMT and ATPCS
07/09 Wednesday - PCR for  PPMT with goTaq (Promega)
25 mikrol reaction:
12,5 mikrol goTaqMastermix
1,25 mikrol fw Primer 10mikrom
1,25 mikrol b Primer 10mikrom
1,0 mikrol DNA Template PPMT
9,0 mikrol nuclease free Wasser
'''with following PCR program parameters:'''                Denaturation  98°C  10 min
Denaturation loop 98°C  1 min
Annealing Temp loop 56°C  1 min
Elongation Time loop  72°C  1 min
Final Elongation  72°C  5 min
Storage 8°C infinite
Miniprep Miniprep of ATPCS clone number 3 = 94ng/µl and sent for Sequencing
07/17 Thursday - Midiprep
#pQE_80L = 183 ng/µl
#pMA-T_PPMT =321ng/µl
#pKD46 = 218ng/µl
Digestion
Digestion pQE_80L with HindIII and SacI
deactivated at 80°C for 10 mins
07/24 Thursday - miniPrep + restriction PPMT und iGEM plasmids
Plasmide:                1. pBADex-mYFP Venus (Amp)
2. pEx-HISII (Amp)
3. pJS418_phagemid(dummy) (Cm)
Restriction-enzymes: XbaI | PstI (used SpeI instead :X)
kit-miniprep
1. 220ng/µl   2. 130ng/µl   3. 430ng/µl
restriction:
500 ng DNA + 1µl XbaI + 1µl PstI + 5µl 10x Buffer + ->50µl Wasser (x3) [ppmt x4]
1. 2,3µl 2. 3,8µl 3. 1,2 µl PPMT 1,6µl
2nd try w/ right enzyme:
1,5 µg DNA + 0.5 µl enzymes + 3µl BUffer + -> 30 µl Wasser
1. 6,5µl 2. 11 µl 3. 3,5 µl PPMT 4,5 µl (x3)
Fragments: 1. 4043/700 2. 4450/55 3. 3800/861 PPMT 2400/251
Gel1 L|3| 1 |1|2|2|2|P|3|3   Gel2 L|P|P|P
Neu: Gel 1|2|3|L|P|P|P
GelEx QiagenKit -> extrakte in grüner box im freezer
Ligation 2 + P
10µl Ansatz: 1µl BUffer + 0.5 µl Ligase + 1.4 µl PPMT + 7.14 µl pEX-HisII [78ng]
07/25 Friday
1. Ordered Primer for cloning hu_Ferritin into pQE80L
2.) Ligation of PPMT Fragment from pMAC_PPMT into pEX_HisII
pMAC_PPMT and pEX_HisII were digested with HF PstI and HF XbaI (NEB)
PPMT (Insert)  and cut pEX_HisII (Vector DNA) were purified via an Gel extraction
Ligation was performed using NEB T4 DNA Ligase in molar ratios of (Insert:Vector) and 40 ng Vector were used.
0:1    2:1      3:1    5:1
Ratio
02:01
03:01
05:01
Amount Insert
4,04 ng
6,06 ng
10,1 ng
Ligase was deactivated and ligation transformed after weekend
07/28 Monday
1. Degradation test of prepped TB-Expression Plasmids by running samples on agarose gel
pBADex_MYFP_Venus  (no observable degradation)
pEX_HisII (no observable degradation)
pJS418_Phagemid (Dummy) (no observable degradation)
Data:
Fabian : 28072014_1
2.Transformation of 0.2 µl pEX_His_PPMT ligation into DH5alpha, DH10b, and MG1655 via electroporation
07/29 Tuesday
1.) Transformation worked and we got colonies for pEX_HisII_PPMT
Colony PCR was performed (Sascha)
See Standard Protocol
2.) Ligation of ATPCS PCR Fragment into pQE_80L
ATPCS PCR fragment and PQE80L Vector were digested with FD HindIII and FD SacI (ThermoScientific)
ATPCS was purified via PCR purification and Vector DNA was purified via an Gel extraction
Vector DNA was dephosphorylized using Fast-AP and deactivated
Ligation was performed using NEB T4 DNA Ligase in molar ratios of (Insert:Vector) and 40 ng Vector were used.
0:1    2:1      3:1    5:1
Ratio
02:01
03:01
05:01
Amount Insert
24,96 ng
37,41 ng
62,36 ng
Ligase was deactivated and ligation transformed next day
<img src="
" alt="" />
#pMAC_PPMT*              #pQE_80L*              #pSB1c3              #pKD46*
(*)Bands at 20 kbp for Promega prepped Plasmids.
07/30 Wednesday
Results Colony PCR for pEX-HisII-PPMT clones preculture 4 and 5 in incubator for Miniprep and Sequencing
Transformation Chemical competent cells XL Blue * 3 µl for each Ligation Assay              * 10 min incubation on ice              *60 s Heat schock              * 2 min incubation on ice              *950µl LB added              *1h --> 37 °C for Recovery              *platted on Amp-plates Eporation *0.2 µl Ligation assay in competent DH5α [iGEM]              *1.7 kV              * 1h; 37°C              * platted on Amp plates
07/31 Thursday
Miniprep for sequencing of pEx_His_ppmt in DH10B? 90-100 ng/µl
Cryostock & miniprep of pjs418 in DH5a 518ng/µl
PCR of 7 clones with pQe80l_ATPCS
16 µl water
2 green dreamtaq buffer
0.5 dNTP
0.5 primer (each)
colony
0.6 dream taq
95° 3min
95° 30s x30
55° 30s x30
72° 2min x30
72° 10min
Colony PCR did not work--> no positive clones              concentration after MiniPrep
* pJS418 = 518 ng/µl              * Clones 4 = 90 ng/µl              * Clones 5= 105 ng/µl
08/06 Wednesday
PCR
'
BamHI_PPMT_GS
GS_ATPCS_HindIII
BamHI_HuFerritin_HindIII
Q5 2x Mastermix
12,5
12,5
25
10  µM Primer fw
1,25
1,25
1,25
10  µM Primer rev
1,25
1,25
1,25
template (1ng)
0,30
0,5
1
nucfree H2O
9,7
9,5
19
Program:
BamHI_PPMT_GS
'
'
GS_ATPCS_HindIII
'
'
BamHI_HuFerritin_HindIII
'
98
30
98
30
98
30
98
10
98
10
98
10
58
30
56,3
30
58,6
30
72
12
72
60
72
45
72
2'
72
2'
72
2'
8
end
8
end
8
end
Analytical Gel               Expected Bands clearly seen for PPMT and Ferritin. Only little ATPCS band shows Primer Dimer inhibit PCR. Anyhow we will try a Gel extraction and after a Assembly PCR Generation of Heme-free BFR by Site-directed Mutagenesis, part 1 of 5
Site-directed mutagenesis was performed by the QuickChange method
Primer design with http://www.bioinformatics.org/primerx/                methionine (on position 52) was substituted by histidine                BFR M52H
08/07 Thursday
Analyzing 2 colonies of pQE80L_ATPCS whether ATPCS was inserted by sequencing revealed no proper results.
Tested by colony PCR (see protocols) using 55°C Annealing temp and 2' Elongation time (expected bands should be 2000bps).
Gel showed PCR was about 500 bp big -> ATPCS NOT INSERTED.
Outcome:BamHI_PPMT_GS    super
GS_ATPCS_HindIII   bad
BamHI_Ferritin    super
08/08 Friday - Gradienten PCR GS_ATPCS_HindIII
PCR
'
3x GS_ATPCS_HindIII
Q5 2x Mastermix
12,5
10  µM Primer fw
1,25
10  µM Primer rev
1,25
template (1ng)
0,5
nucfree H2O
9,5
Program:
GS_ATPCS_HindIII
'
'
98°C
30
98°C
10
56,3-62°C
30
34
72°C
60
72°C
2'
8°C
end
- preparative agarose gel 1% and gel ex of GS_ATPCS_HindIII bands at ca. 1490 bp
- accidentally also loaded BamHI_PPMT_GS onto Gel and was also  extracted (elution with 25 µl elution buffer).
- BamHI_Ferritin_HindIII was PCR purified
- DNA concentration measurement for  purified pcr fragments:
GS_ATPCS_HindIII = 33 ng/µl
BamHI_PPMT_GS = 48 ng/µl
BamHI_Ferritin_HindII = 125 ng/µl
Assembly PCR
An Assembly PCR was used to align BamHI_PPMT_GS with GS_ATPCS_HindIII and amplify the complementary structure in order to form a fusion protein coding sequence.
Therefore, both fragments were added a templated equal molar amounts.
Calcultation:
bpATPCS = 1493
bpPPMT = 255
mATPCS/bpATPCS = mPPMT/bpPPMT
mATPCS = (1ng * 1493) / 255
mATPCS= 58 ng
'
BamHI_PPMT_GS_ATPCS_HindIII
Q5 2x Mastermix
25
100  µM Primer fw
0.25 (not added till later to second PCR run)
100  µM Primer rev
0.25 (not added till later to second PCR run)
template (1ng)
1,750 µl ATPCS + 0.208 µl PPMT
nucfree H2O
18.042
Program:
1. PCR BamHI_PPMT_GS_ATPCS_HindIII
'
'
98
30
98
10
68
30
5
72
55
72
1'
8
end
2. PCR BamHI_PPMT_GS_ATPCS_HindIII
'
'
98
30
98
10
68
30
30
72
55
72
2'
8
end
Transformation into DH10B #BFR M52H              # BFR M52H              *Recover in LB--> 1h at 37°C
08/11 Monday - Plan-prepare fragments for digest
Constructs aimed for *In pQE_80L (Amp)     BamHI_PPMT_GS_ATPCS_Hind III  (PCR Construct available)              Construct Size 1724 of 4731
*In pQE_80L (Amp)    BamHI_FTH_FTL_Hind III   (PCR Construct available)              Construct Size 1077 of 4731
*In pQE_80L (Amp)   Sacl_ATPCS_Hind III    (PCR Construct not available)              Construct Size 1400 of 4731
*In pJS418 (CM)    XbaI_PPMT_PstI       (PCR Construct available)              Construct Size 255 of 3800
Q5 MasterMix
25μl
ATPCS SacI
2.5
ATPCS Hind III
2.5
cDNA At.
0.5
H2O
19.5
PCR Program
98°C
30 sec
98°C
10 sec
62°C 34cycles
30 sec
72°C
60 sec
72°C
2 mins
4°C
Store
PCR Fragmente PPMT_GS_ATPCS+ATPCS+Purification (Gelex)
After Gelex:
PPMT  48ng/μl
Ferritin 125ng/μl (Pur from PCR)
ATPCS 33ng/μl
08/12 Tuesday - Preparing chemically competent cells
V=100ml, 37°C
BU36=DH10B                OD at 600nm=0.461A
DH10B competent cells
Digestion
Inserts PCR Pur
JBH1  PPMT_GS_ATPCS 1
JBH2  PPMT_GS_ATPCS 2
JBH3 Ferritin
JSH  ATPCS
PCR Fragment
20
20
8
20
Buffer
3
3
3
3
FD BamHI /FD SacI
1
1
1
1
FD Hind III
1
1
1
1
nuclease free H2O
5
5
17
5
Vector
VBH pQE_80L
VSH pQE_80L SacI
2x PJS_418
2x P_Mac_PPMT
Vector
5.5
5.5
1.9
3.2
Enzyme 1
1 BamHI
1 SacI
1 FDxBa
1 FDxBa
Enzyme 2
1 HindIII
1 HindIII
1 FD PST
1 FD PST1
Buffer
3
3
2
2
Fast AP
1
1
0
0
nuclease free H2O
18.5
18.5
* 35 ng/L  PPMT ATPCS 1              * 15ng/L  PPMT ATPCS 2              * 25ng/L  ATPCS              * 13ng/L  P_Mac_PPMT              * 31ng/L  PJS_418
Ligation 5:1
JBH1 in VBH
JBH2 in VBH
JBH3 in VBH
JSH in VSH
PPMT in PJS_418
10x T4 DNA ligase Buffer
1μl
1μl
1μl
1μl
1μl
Vector
1.42
1.42
1.42
1.33
1.29
Insert
3.29
7.08
1.36
7.17
1.03
Nuclease free H2O
3.79
0
5.72
0
6.18
T4 DNA ligase (NEB)
0.5
0.5
0.5
0.5
0.5
Ampicillin
JBH1 in VBH
JBH2 in VBH
JSH in VSH
Chloramphenicol
PPMT in PJS_418
Transformation in DH10b (cc.c)
106 Colony for every approach
08/13 Wednesday - pQE 80L Colony PCR
Mastermix
1x
35x
10x
nuclease free H2O
14
490
140
10 dreamtaq
1
70
20
25µM MgCl2
1.2
42
12
10mM dNTPs
0.5
17.5
5
10mM Forward Primer
0.5
17.5
5
10mM Reverse Primer
0.5
17.5
5
Taq
0.3
10.5
3
PCR Programm
95°C
3 mins
95°C
30 secs
55°C
30 secs
72°C
60 secs
72°C
10 min
8°C
Store
Samples :
# PPMT GS ATPCS   size 1912              # PPMT GS ATPCS   size 1912              # ATPCS in pQE80L   size 1770              # Ferritin in pQE80L   size 1376              # PPMT in PJS   size 420
pQE80L=300bp
pJS = 170bp
Sequence clones A - H + J
Generation of Heme-free BFR by Site-directed Mutagenesis, part 2 of 5
QuickChange Site-Directed Mutagenesis
20µL approach:
Header
component
2 µL
10x Pfu Buffer (MgSO4)
0,6 µL
10µM forward Primer
0,6 µL
10µM reverse Primer
x µL
30ng Template
0,4 µL
dNTPs
0,6 µL
Pfu Polymerase
to 20 µL
PCR water
Template BFR:
0,5 µL              A4 (66ng/µL)
0,2 µL    D2 (144ng/µL)
labeling
Template
Annealing Temp.
17
D2 (144ng/µL)
71°C
15
D2 (144ng/µL)
55°C
67
A4 (66ng/µL)
71°C
65
A4 (66ng/µL)
55°C
PCR Program
Temp
time
95°C
30sec
________
______
16 Cyclen
95°C
30sec
55/71°C
1min
68°C
5min
________
______
68°C
10sec
4°C
finale
Σ20µL -8µL fürs Gel= 12µL +1µL DpnI
37°C 90min
chemical Transformation (Fabian)
in DH10b
plating out on Amp plates
08/15 Friday - Clone Sequencing
Clone
Name
Construct present
concentration after isolation
A
pQE_80L_PPMT GS ATPCS
yes
388ng/µL
B
pQE_80L_PPMTGSATPCS
yes
373ng/µL
C
pQE_80L_PPMTGSATPCS
yes
442ng/µL
D
pQE_80L_PPMTGSATPCS
no
389 ng/µL
E
pQE_80L_huFerritin
yes
490ng/µL
F
pQE_80L_huFerritin
no
402ng/µL
G
pQE_80L_ATPCS
yes
497ng/µL
H
pQE_80L_ATPCS
yes
443 ng/µL
I
pJS418_PPMT
yes
596 ng/µL
08/19 Tuesday
Generation of Heme-free BFR by Site-directed Mutagenesis, part 3 of 5 ÜN culture
5ml LB + 5µL Amp →shakingincubator
08/20 Wednesday
Generation of Heme-free BFR by Site-directed Mutagenesis, part 4 of 5 MiniPrep nach Protocol
Photometer
4µL Probe + 76µL dilution (water)
08/21 Thursday
Generation of Heme-free BFR by Site-directed Mutagenesis, part 5 of 5 Sequencing:
Primer PB 16
Crystocks:
ÜN culture
500µL culture + 500µL DMSO
2xclon  17
2xclon 67
08/22 Friday - SDS-PAGE with Coomassie staining
Samples
# Marker              # Pellet ATPCS/PPMT              #15:00 after induction ATPCS/PPMT              #overnight ATPCS/PPMT              #GS Pellet              #PGSA after induction 15:00              #GS P/A overnight              #human Ferritin Pellet              #human Ferritin after induction 15:00              #human Ferritin overnight
Analysis of Expression
Human Ferritin = 42 kDa (Potparam  Expasy)
ATPCS = 56.3 kDA (Potparam  Expasy)
PPMT = 7.9 kDA (Potparam  Expasy)
ATPCSGSPPMT = 64.3 kDa (Potparam Expasy)
08/29 Friday - Calibration curve for Iron concentration measurement
Header
Header
0.017A
10 µg/ml
0.105A
50 µg/ml
1.076A
100 µg/ml
2.095A
150 µg/ml
2.747A
200 µg/ml
09/03 Wednesday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 1 of 8 Aim:               Test the possibility of Magnetization of E.coli cells by the expression of human Ferritin:
Procedure: ...
-Precultures of Nissle and RV308 (wild types)
09/04 Thursday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 2 of 8 -Prepare chemical competent cells ( Nissle and RV308) . see Protocol
09/05 Friday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 3 of 8 -Chemical biotransformation (double Trafo, 200ng of each) of, red fluorescence protein carried on plasmid (kan) Human-Ferritin carried on PQE-80L (Amp) [was prepared by iGEM-Berlin using the '''Biobrick''' from Calgary, unlike does not contains polypeptide at the beginning].
09/06 Saturday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 4 of 8 -Positive colones for both RV308 and Nissle
09/11 Thursday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 5 of 8 - Precultures of the transformed colones
09/12 Friday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 6 of 8 - Transfer the precultures into 250ml filled until 100ml with LB and incubate them at 37°C                  Strain    OD
RV308     0.71
Nissle    0.6
- Induction with 100µl IPTG (1M), incubation over night at '''30°C''' and 200rpm
09/13 Saturday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 7 of 8 -samples were taken for protein analysis SDS-Page                 Strain    OD
RV308     6.2
Nissle    5.2
- adding  7ml of 1M Mn-citrate (wrongly) and incubated at 4°C over weekend
09/15 Monday
Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 8 of 8
-Test the Magnetization using a permanent magnet under Fluorescent microscope.
Evaluation
-Cells grew              -no red fluorescence observed              -no Magnetization observed
Possible Explanation:              - Adding Mn ions instead of Fe could be effected the Magnetization               - The red Fluorescence gen contains a cadaverin chain, this could explain the non-fluorescence.
09/16 Tuesday
Preparing Cultures for fluorescence microscopy
E. coli Nissle 1917 (pQE_80L Hu_Ferritin + RFP)
E. coli RV308 (pQE_80L_HuFerritin + RFP)
Nissle OD600 = 5.08
RV 308 OD600 = 5.81
take OD=1
wash in 1000 µl PBS three times ( centrifuge at 6000g for 2min, discard supernatent and resuspend pellet in PBS)                finally resuspend pellet in 500 µl                Put it into a cool box and take it over to the fluorescence microscope facility. Constructing PQE_80L_T5_ATPCS_lac_PPMT
Plasmid for Co-Expression of PPMT and ATPCS on one plasmid.                Source:               pJS418_PPMT (amplifying insert)               pQE_80L_ATPCS (as target vector) PCR of HindIII_lac_PPMT_HindIII Construction of the HindIII_lac_PPMT_HindIII in pQE80L_Hind III
3x HindIII_lac_PPMT_HindIII
Q5 2x Mastermix
12,5
10 µM Primer fw
1,25
10 µM Primer rev
1,25
template (1ng)
0,5
nucfree H2O
9,5
Program
Temperature
Duration in seconds
Cycles
98
30
1
98
5
loop start
62-65
15
loop 32
72
10
loop end
72
2'
1
8
infinite
1
09/17 Wednesday - Digest and Dephosphorylation of vector pQE_80L_ATPCS
pQE80L_ATPCS digest
1x
3x
pQE_80L_ATPCS
2 µl
7 µl
FD HindIII
1 µl
3 µl
10x FD Buffer
2µl
7 µl
FastAP (Thermo)
1 µl
3 µl
nucfree H2O
14 µl
51 µl
Gelextraction resulted in HindIII cut pQE80L_ATPCS 20 µl with 12 ng/µl
In parallel the PCR fragment HindIII_lac_PPMT_HindIII was purified and is ready for digest.
09/18 Thursday
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 1 of 7
'Aim
-Double Trafo of the Human-Ferritin and fluorescence marker.
Procedure:…
Stains: RV308, Nissle and DH0B (control)
-chemical Double-biotransformation of:
-0.5 µl of Human-ferrtin on PQE-80L(490 ng/µl) with (Amp)
-3 µl green fluorescence protein Nerm ASGP-R-GFP (83 ng/µl) with (Kan)
for control:
- 3 µl green fluorescence protein Nerm ASGP-R-GFP (83 ng/µl) with (Kan) is transformed into DH05
- Double Trafo of (pJS418_PPMT and pQE_80L_ATPCS)
09/19 Friday
PCR of BB0 - BB3 parts (Saba)
compound
V/µl (50µl total)
Q5 2xMM
25.0
10µM P_fv
2.5
10µM P_rev
2.5
DNA (template 1ng)
2.0 (of diluted plasmid)
nuc.-free H2O
28.0
Primers
Reaction
fw Primer
rev. Primer
TmTm/°C
Template
bp
BB0
BBa_K1438000_fw
BBa_K1438000_rev
62
pQE80L_BFR
507
BB1
BBa_K1438000_fw
BBa_K1438000_rev
62
pQE80L_BFR M52H
507
BB2
BBa_K1438002_fw
BBa_K1438002_rev
68
pQE80L_Hu-Fer
1000
PCR Programms BB0 & BB1
T/°C
t/min
cycles
98
0:30
30
98
0:10
30
62
0:30
30
72
0:15
30
72
2:00
30
8
infinity
30
BB2
T/°C
t/min
cycles
98
0:30
30
98
0:10
30
68
0:30
30
72
0:40
30 (probably too long since ~1000bp and not 1408bp however worked well as seen on gel)
72
2:00
30
8
infinity
30
Biobricks derived from pQE80L constructs
BB1  BFR  191µg/ml
BB2  BFRM  173µg/ml
BB2  HuFerritin  166 µg/ml Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 2of 7 -No positive colones were detected for double-trafo expect these for control (transformation was successful in DH05).
09/20 Saturday - Digest of PSB1C3-Ferritin
Plasmid:  PSB1C3 - ferritin with Peptide   120ng/µl
component
µL
NEB Xba1
2 µL
NEB Pst1
2 µL
NEB Buffer 3.1
20 µL
nucleasefree H2O
151 µL
PSB1C3 Feritin
25 µL
total volume
200µL
37°C for 1,5h              (inactivate for 20min at 80°C)
Gel electrophoresis 1% Agarose-Gel
+ 3 µL EthBr
200µL Probe + 40 µL loading dye (6x)
5 µL GenRuler-Mix
no Gel-extraction, because nothing was seen on the gel, except the marker (GenRuler)
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 3 of 7 -Transformation of RF (red fluorescence protein) on with (Kan) into DH05
09/21 Sunday
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 4 of 7 -Transformation was successful
09/22 Monday - Precultures
Precultures of                PSB1K3_RFP in DH10b (Kan)
pQE80L_ATPCS & pJS418_PPMT in Dh10b (Amp & Cm)
Clones with pQE80L_ATPCS'n'PPMT in DH10b (Amp)
PSB1C3_Ferritin in DH10b (Cm) Checking Insertion of gene by colony PCR
Performing PCR for clones of ATPCS-PPMT construct
1x in µl
12x in µl
10x in µl
nuclease free H2O
14
168
140
10x DreamTaq buffer
2
24
20
25mM MgCl2
1.2
14.4
12
10mM dNTPs
0.5
6
5
10mM FW Primer
HindIII-lac-prom-fw
0.5
6
5
10mM RW Primer
PB17 (T7 term_rev)
0.5
6
5
Taq Polymerase
0.3
3.6
3
Programm colony PCR
Temp
Time
Denaturing
95°C
7 min
Denaturing
95°C
30
Annealing
69°C
30
Elongation
72°C
1 min
Final Elongation
72°C
10 min
Store
8°C
Dissolved colonies 4 and 6 which showed band at 400 bp were grown each in a cultivation tube with LB+Amp100 at 37°C
09/23 Tuesday - Mutagenesis of ATPCS
The elimination of the XbaI restriction site of all our ATPCS containing constructs was achieved by quick change mutagenesis.
... Details Saba....
After digest with DpnI the reaction was transformed into competent DH10b E. coli cells via heat shock transformation.
After streaking out all aliquots on LB-Amp Agar plates, they were incubated at 37 °C over night.
Digest of miniprepped PSB1C3_Ferritin (Calgary) for extraction of vector for BioBrick preparation
Components
Volume in µl
NEB XbaI
2
NEB PstI
1,5
NEB Buffer 3.1
5
nucfree H2O
26,5
plasmid PSB1C3_ferritin (Calgary)
15
50 µl reaction
The reaction was incubated at 37 °C for 1,5 h.
The digested vector was purified using a 1% agarose gel and the Roth GelNebulizer purification kit. The vector band was expected at 2000 bp and the insert band at about 1000 bp.
both colonies from Saba were grown --> Mini-prep (Johann) to harvest plasmid DNA
Mutagenesis of ATPCS              in                  pQE80L_ATPCS-GS-PPMT
pQE80L_ATPCS_PPMT
pQE80L_ATPCS
3x PCR-Reactions
Q5 MM   12.0µl
PB_for SN_ATPCS_xbaI_Mut_for   1.25µl
PB_rev SN_ATPCS_xbaI_Mut_rev   1.25µl
nuc.-free H2O   10.5µl
template (1ng)   ~ 1.0µl (dilations of original plasmids)
Total 26µl
PCR Program
T/°C
t/min
98
0:30
98
0:10
69
0:30
72
 :
72
2:00
Trafo of 5µl PCR-Product in DH10b cells --> incubation at 37 °C
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 5 of 7
- Preculture of DH05 with RF plasmid
09/24 Wednesday
Berlin Vector - pQE-80L-JBFS-huFerritin Assembly PCR Running Gel Electrophoresis
* 1% Agarose, Ethidium bromide
* 50µL Assembly PCR product + 10µL 6*DNA dye              * 10µL GeneRuler DNA Ladder Mix (ThermoScientific)              * Gel Electrophoresis conditions 100V, 30min
Gel Extraction Gel Extraction using GeneJET Gel Extraction Kit (ThermoScientific)                 * Determination of Concentration by Ultraviolet Spectrophotometry
 ?
Double Digest
* Digestion of 1μg of purified Insert DNA using FastDigest BamHI / HindIII              * 1h at 37°C  ?
* Heat inactivation of FastDigest BamHI / HindIII, 15min at 80°C
PCR purification
* PCR purification of BamHI / HindIII digested insert using GeneJET PCR Purification Kit (ThermoScientific)              * Determination of Concentration by Ultraviolet Spectrophotometry
 ?
Ligation
* Using the Molar Ligation Ratio 1:5 (1 times vector and 5 times insert)
* 20min at RT
Chemical Transformation
* 100µL Aliquots of Chemically Competent Cells - DH10B E.coli + 10µL Ligation approach chilled on ice for 30 min              * Heat shock for 1min at 42°C              * Add 900µL preheated (37°C) LB              * Incubate the cells for 1h at 37°C with shaking approx. 200rpm              * Plate 200µL and spin down the rest and plate on LB Ampicillin, incubate overnight at 37°C
Saba o/n culture of clones from successful pQE80L_ATPCS-mut_GS_PPMT and pQE80L_ATPCS-mut_PPMT transformation              --> 37°C, 200pm, o/n 5-6 nl LB+AB (Amp)
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 6 of 7
- Miniprep of the Preculture, DNA concentration 71 ng/µl
Chemical Biotransformation of: -3 µl of RFP 71 ng/µl -2 µl Ferritin jbfs_m (different ferrtin)(120 ng/µl) with  (Amp)'
in RV308 and Nissle (2 plates each)
09/25 Thursday - PCR of Biobrick Parts
Isolation of plasmid-DNA with Mini-Prep Kit of pQE80L_ATPCSMut_GS_PPMT and pQE80L_ATPCS_PPMT PCR of Biobrick-Parts 03,04,06 & 10-15
BB
PB_fw
PB_rew
Tm/°C
template
bp
part
03
BBa-K1438003_fw
BBa-K1438003_rew
67
pQE80L_PPMT_GS_ATPCS
1715
PPMT GS ATPCS
04
BBa-K1438004_fw
BBa-K1438003_rew
65
pQE80L_PPMT_ATPCS
1408
ATPCS
10
XbaI_T7_prom
for Lambda_Term_suffix_rew
66 -->67
pQE80L_BFR
690 (+377bp-->cp)
BFR_cp
11
"
"
66 -->67
pQE80L_BFRM52H
690 (+377bp)
BFRM52H_cp
12
"
"
66 -->65
pQE80L_FTH_FLH(HuFer)
1337(+377bp)
FTH_FLH_cp
13
"
"
66 -->67
pQE80L_PPMT_GS_ATPCS
1968(+377bp)
PPMT_GSATPCS_cp
14
"
"
66 -->67
pQE80L_PPMT_ATPCS
2091(+377bp)
ATPCS_cp
06
"
"
66
pQE80L_fbfs_mil_Ferritin
1403(+377bp)
fbfs_mil_Ferritin_cp
PCR-Ansätze V/µl Q5 2xMM       25.0                 10µM PB_fw    2.5                 10µM PB_rew   2.5                 template      2.0 (~1ng)                 nuc.-free H2O 18.0 PCR-Programm
T/°C
t/min
cycles
98
30"
30
98
10"
"
66/67
30"
"
72
40"/12"/1'
"
72
2'
"
8
infinity
"
PCR purification of 1,2, 3 (Jbfs_mil_Ferritin_cp),BF-cp and BFM-cp biobrick parts
Concentration µg/ml
fbfs_mit_Ferritin1
42
fbfs_mit_Ferritin2
35
fbfs_mit_Ferritin3
42
ATPCS and PPMT
-12 too low
Bacterioferritin CP
56
BFM52H CP
57
Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 7 of 7
- Double-trafo was successful in all RV308 plates  but not in Nissle
09/30 Tuesday
Ligation of Biobrick parts with PSBIC3 backbone (digested on 23.09.2014 by Johann Transformation into DH10b c.c.cells by chemical transformation
5µl of each ligation was transformed to 100µl aligusted cells each
--> after recovery at 37°C, 30min cells were plated on Cm/LB-Agar plates after centrifuging-->pelleting and resuspending in 100µl LB to plate all cells.                --> incubation at 37°C, o/n Mini Prep von pSB 1U 3-RFP concentration 112 ng/µl
Preculture
pQE-80L-jbfs-Ferritin 5ml LB+5µl Amp at 37°C overnight
10/01 Wednesday - Plasmid Digestion & PCR Probe
Plasmid Digestion
Plasmid Digestion
NEB Buffer 3.1
5µl
NEB XbaI
0.5µl
NEB PstI
0.5µl
P_MA_T_PPMT 312ng/µl
3.2µl
nucl. free water
40.5µl
PCR Probe Ligation plates evaluated:
All plates had clones except for BFR_cp from each plate transformation one clone was cultivated with LB/Cm (~5ml) o/n at 37°C
10/02 Thursday - Cultures were Mini-preped
--> elution with 40µl EB                --> DNA concentration measurement
10/06 Monday- Colony PCR
Biobrick
Sequecing
Sequenz (?)
(???)
BB0
BFR
100Y BFR M52H
-A ws ATGf (??)
BB1
BFR M52H
100% BFR
-A wr
BB2
HFTN_LFFN (?)
96Y HuFerritin (Rw Sep nötig) ??
AnP
ATPCS
f 70% ok, but fragment f
redo PCR ... Gelex (?)
BFM1cp
BFR1cp
 ?
BFM2cp
BFR2cp
 ?
HuFecp
HuFerritin cp
 ?
Colony-PCR of other clones
PCR-Programm
T/°C
t/min
cycles
95
7'
30
95
30"
30
52
30"
30
72
1'
30
72
10"
30
8
infinity
30
--> gel (1% Agarose)
Clones 3, 5, 6, 7 and 13 were grown o/n in LB/Cm for Mini-prep on the following day. therefore 5µl of the dissolved clones were used for each inoculation
--> 37°C, 200rpm
2x M9 Media Production
(Christina)
2X M9 mineral medium
For 500 ml M9 mineral medium add to XXX ml sterile water:
100ml
M9 salt solution (10X)
Na2HPO4
KH2PO4
NaCl
NH4Cl || 67.4mM
44.0mM
17.10mM
18.70mM
10/07 Tuesday
Mini-Prep of cultures (Ben) and preparation of sequencing samples (Saba): 9µl nuc.-free H2O + 3µl DNA + 3µl PB794
Preculture of Knockouts * λΔ FieF +PCP 20 : 5ml LB+tet/amp (1:1000)               * λΔ FUR  : 5ml  LB+tet/kan (1:1000)               * MG1655 ΔFieF: 5ml LB + amp/kan (1:1000)
incubation at 30°C
10/08 Wednesday - PCR of PSB1C3 Backbone
Q5 Mastermix 2x
Component
Volume in µl
Q5 Mastermix 2x
25
Primer fw (PSB1C3 Suffix fw)
2,5
Primer rev (PS1C3 Prefix rev
2,5
template (PSB1C3_BBa_K1189018)
2
nuc free H2O
18
Temperature in °C
Time
Nr. of cyles
98
30
98
5
loop start
70
15
30
72
1'
loop end
72
2'
8
infinit
Extraction of PCR Product (PSB1C3 linearized)
After extracting the expected band at 2000 bp it was purified using the EMDMillipore Montage Gel Extraction Device.
Resulting in about 120 µl DNA extract in TAE buffer with an concentration of 9ng/µl
Digestion of PCR Product
NEB Buffer 3.1
15µl
NEB PstI
1.5µl
NEB XbaI
1.5µl
nuc. free water
83µl
PCR Product
100µl
* 1.5 hours at 37°C              * PCR Purification              * DNA concentration measurement = 35ng/µl
Ligation Calculation              *BFM52H = 3.22µl              *BF_cp= 6.7µl              *jbfs_m = 8.06 µl              *BB_AnP = 10.64 µl              * HuFer = 9.45 µl Iron Concentration measurement 278 mg FeSO4.7H2O dissolved in 10 ml d.H2O
λ FieF PCP2 Amp  1:100 ---> 3.0 OD
iGEM BV125 tet 1:100 --> 3.0 OD
λ FieF PCP1 Amp 1:20 -->3.26 OD              iGEM BV125 tet 1:20---> 2.44
too high OD!
Both samples diluted 1:2              Samples              #  FieF + 5µl 0.1 M Fe-stock solution              # FieF + 15 µl 0.1 M Fe-Stock solution              # BV + 5 µl 0.1M Fe-stock solution              # BV + 15 µl 0.1 M Fe-stock solution
Overnight incubation at 30°C on a shaker
10/09 Thursday
Mini Prep (using: AccuPrep Plasmid Mini Extraction Kit Ver. 2.0, Company: BIONEER), using 100µl Elution Buffer
and dsDNA Measurement: 4 µl Sample + 76µl MilliQ, blanked with MilliQ
Samplenumber
Samplename
dsDNA conc [ng/µl]
Sample [µl] used for Sequencing
nucl.free Water [µl] used for Sequencing
#1
cp HuF
111
9
3
#2
AGSP
107
9,4
2,6
#3
AGSP
59
12
0
#4
AGSP
64
12
0
#5
jbfs_Fer
54
12
0
#6
A'n'P
67
12
0
For Sequencing:              *total volume: 15µl              *Primer used: PB795 3µl each Sequencing approach              *Sample [µl] see table              *nucl.free Water [µl] see table
evaluation of Trafos ---> no growth --->Repeat Trafo in other cells
10/10 Friday - STRATEGY 1 - Knockout of FieF and FUR Genes
Detection PCR kan-deletion-cassette for FieF and FUR Genes was transformed into 2 different strains (MG1655 and λ MAGE). Additionally pCP20 was already transformed into one of the λ MAGE Clones (has to be re-validated).
The positive clones have to be validated.
Detection-PCR of one clone of Plates 1-3:
Plate 1: MG1655 ΔFieF
Plate 2: λ ΔFieF + pCP20
Plate 3: λ ΔFUR
Primer-Combinations FieF
-
Combination
WT length
Clone length
Annealing Temperature range
A
C1+C2
2060bp
2654bp
58,4-60,1°C
B
C1+C3
0
1046bp
59,5-60,1°C
C
C4+C2
0
1181bp
57,5-58,4°C
D
C1+C5
1134bp
0
60,1-60,5°C
--> Annealing Temperature: 57°C
FUR
-
Combination
WT length
Clone length
Annealing Temperature range
A
C1+C2
1130bp
2219bp
59,5°C
B
C1+C3
0
823bp
59,5°C
C
C4+C2
0
969bp
57,5-59,5°C
D
C1+C5
500bp
0
59,5°C
--> Annealing Temperature: 57°C PCR-Assay
-
1x
Mastermix 15x
nuc.free H2O
14,4µl
216µl
10x Dream Taq Buffer
2µl
30µl
50mM MgCl2
0,6µl
9µl
10mM dNTPs
0,5µl
7,5µl
10mM Primer Combination
1µl
10µl nuc.free H2O+ picked colony
1µl + 18,4ml Mastermix
Taq Polymerase
0,5µl
PCR-Program
Temperature
Time
95°C
7'
30 cycles
95°C
30"
57°C
30"
72°C
2'40"
72°C
5'
12°C
hold
Gel-Electrophoresis
1% Agarose-Gel + 1µl GelRed; 90V; 30min
10/14 Tuesday - Iron-Assay
1. Day: inoculation of 5ml LB Media (+ Antibiotics)
* 37°C or 30°C/ON
2.Day:inoculation of 10ml LB (1:10)
* 30°C or 37°C
* grow until OD600 0,6
* take 1ml
* centrifuge 2min at max. speed
* discard the supernatant
* wash with 1X M9 Media
* 1XM9:
:500µL 2X M9+
:500µL sterile water
* wash 2x and recover 1mL 1XM9 Media
* diluted 1:1000 with 1X M9
* plate 20µL per well                  24well plate without Antibiotics
*A:MG1655 wt               *B:Nissel 1917 wt               *C:RV308 wt               *D:DH10b
concentration of Fe-citrat
0    0,5    1    2    5    10mM
Sponsors
Müller-Breslau-Str. 10
10623 Berlin
Phone: +49 1778800244
Mail: mail@igem.berlin
Home |
Project |
Team |
Safety |
Workshop |
Blog
©2014 iGem Berlin, Layout/Umsetzung: Valerian Blos, Marius Wessel
Retrieved from " http://2014.igem.org/Team:Berlin/Project/Journal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
