<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Berlin_Project_Journal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Berlin/Project/Journal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Berlin/Project/Journal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Berlin/Project/Journal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Berlin/Project/Journal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Berlin/Project/Journal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Berlin/Project/Journal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="site-wrapper-sub-project"><DIV class="main-menue clearfix navbar-fixed-top hidden-xs hidden-sm hidden-md" style="margin-top:20px;"><UL class="main-menue-ul"><LI><A href="https://2014.igem.org/Team:Berlin" class="main-menue-links">Home</A></LI><LI class="active"><A href="https://2014.igem.org/Team:Berlin/Project" class="main-menue-links">Project</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Team" class="main-menue-links">Team</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Safety" class="main-menue-links">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Workshop" class="main-menue-links">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Blog" class="main-menue-links">Blog</A></LI></UL></DIV><DIV class="navmenu navmenu-default navmenu-fixed-left offcanvas hidden-lg"><A class="navmenu-brand" href="https://2014.igem.org/Team:Berlin"><H1>iGEM Berlin</H1></A><UL class="nav navmenu-nav"><LI><A href="https://2014.igem.org/Team:Berlin" class="active">Home</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Blog">Blog</A></LI></UL></DIV><SECTION data-speed="200" data-type="background" id="section-sub"><DIV class="container-fluid container-green"><DIV class="row featurette" id="main-featurette-row"><DIV class="start-text-container-project"><H2>Explore our Project:</H2></DIV></DIV></DIV><DIV id="subside-content"><DIV class="container"><DIV class="row"><DIV class="col-xs-3 submenue-project" style="text-align:left;"><A href="https://2014.igem.org/Team:Berlin/Project" class="sub-link-project"> 1. What is it all about?</A><A href="https://2014.igem.org/Team:Berlin/Project/Activities" class="sub-link-project"> 2. Project-related Activities</A><A href="https://2014.igem.org/Team:Berlin/Project/Detailed-Description" class="sub-link-project">3. Detailed Description</A><A href="https://2014.igem.org/Team:Berlin/Project/Results" class="sub-link-project">4. Our Results</A><A href="https://2014.igem.org/Team:Berlin/Project/Summary" class="sub-link-project">5. Lab Summary</A><A href="https://2014.igem.org/Team:Berlin/Project/Journal" class="sub-link-project">6. Lab Journal</A><A href="https://2014.igem.org/Team:Berlin/Project/Property" class="sub-link-project">7. Intellectual Property</A></DIV><DIV class="col-xs-11 col-sm-9 blog-text" style="margin-bottom:40px;"><DIV class="project-number">6</DIV><DIV class="project-headline-float"><H2 class="green-text project-headline">Lab Journal</H2></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline">04/02 Wednesday - Cultivating E. coli Nissle 1917</H2>
                LB plates were produced without antibiotic. Therefore 10 ml MQ-H<SUB>2</SUB>O were sterile filtered in 15 ml falcon tube. 1 capsule Mutaflor 331800 was solved in the sterile water. 
                The E. coli Nissle solution was incubated at 37°C and 200 rpm for 1 h and crossed out at 4 plates to get different dilution of cells.
              
<TABLE class="wikitable"><TBODY><TR><TH> dilution </TH><TH> E. coli Nissle solution </TH><TH> H<SUB>2</SUB>O </TH><TH> dilution factor
</TH></TR><TR><TD> I </TD><TD> 2 µl </TD><TD> 1998 µl </TD><TD> 1:10<SUP>3</SUP></TD></TR><TR><TD> II </TD><TD> 100 µl from dilution I </TD><TD> 900 µl </TD><TD>1:10<SUP>4</SUP></TD></TR><TR><TD> III </TD><TD> 10 µl from dilution II </TD><TD> 990 µl </TD><TD> 1:10<SUP>5</SUP></TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/03 Thursday - Genomic DNA extraction of ECN - first step </H2>

            For extraction of the genomic DNA of E. coli Nissle, 2 E. coli colonies were picked from LB plate (dilution 1:10<SUP>3</SUP>) and used for inoculation of 6 ml LB precultures. These were grown over night at 37°C and 200 rpm. 
            </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/04 Friday - Genomic DNA extraction of ECN - second step</H2>

            For preparation of the genomic DNA 1 ml of each preculture was taken and a genomic extraction performed using the Wizard Genomic DNA purification Kit. See instructions of the purification Kit by Promega. Genomic DNA was stored at 12°C in fridge.

            As E. coli Nissle is a natural organism without resistence genes both precultures were streaked out on LB, LB+KAN and LB+AMP. All plates were incubated over the weekend at 37°C.

            DNA concentration was measured by 260 nm using Goldi. DNA was diluted 1:40 in a UV cuvette and DNA absorption measured at 260/28 nm in goldi.
'''For genomic DNA:'''
            
<TABLE class="wikitable"><TBODY><TR><TD> 1: </TD><TD> 1390 ng/µl
</TD></TR><TR><TD> 2: </TD><TD> 988 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/05 Saturday - Genomic DNA extraction of ECN - results </H2>

            Bacteria growth only on LB plate indicates that there was no contamination with AMP or KAN resistant E. coli. This doesn´t mean that there is no contamination at all. See for sure on PCR. 
<TABLE class="wikitable"><TBODY><TR><TH> LB </TH><TH> LB+KAN </TH><TH> LB+AMP
</TH></TR><TR><TD> + </TD><TD> - </TD><TD> -
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/06 Sunday </H2><PRE><STRONG>Production of BfR, FTNA1 and FTNA2</STRONG><EM>Restriction digest</EM>
           After 2 purifications of plasmids:
</PRE><TABLE class="wikitable"><TBODY><TR><TD> p1 </TD><TD> 84ng/µl </TD><TD> 10µl
</TD></TR><TR><TD> p2 </TD><TD> 72,2ng/µl </TD><TD> 10µl
</TD></TR><TR><TD> p3 </TD><TD>   54,8ng/µl </TD><TD> 23µl
</TD></TR></TBODY></TABLE><PRE><STRONG>Restriction program</STRONG></PRE><TABLE class="wikitable"><TBODY><TR><TD> 1x </TD><TD> 4x
</TD></TR><TR><TD> plasmid P2 </TD><TD> 3µl </TD><TD> 12µl
</TD></TR><TR><TD> BamHI (FD) </TD><TD>   1µl </TD><TD> 4µl
</TD></TR><TR><TD> HindIII (FD) </TD><TD>   1µl </TD><TD> 4µl
</TD></TR><TR><TD> Buffer (FD) </TD><TD>   2µl </TD><TD> 8µl
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD>   13µl </TD><TD> 52µl
</TD></TR></TBODY></TABLE><PRE>           --&gt; 37°C for 1h (restriction)
           --&gt; 80°C for 10min (deactivation)
</PRE><TABLE class="wikitable"><TBODY><TR><TD> PCR </TD><TD> 6,38µl </TD><TD> 4,56µl </TD><TD> 2,71µl
</TD></TR><TR><TD> BamHI (FD) </TD><TD>   1µl </TD><TD> 1µl </TD><TD> 1µl
</TD></TR><TR><TD> HindIII (FD) </TD><TD>   1µl </TD><TD> 1µl </TD><TD> 1µl
</TD></TR><TR><TD> Buffer (FD) </TD><TD>   2µl </TD><TD> 2µl </TD><TD> 2µl
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD>   19,62µl </TD><TD> 21,44µl </TD><TD> 23,29µl
</TD></TR></TBODY></TABLE><PRE>           --&gt; 37°C for 1h (restriction)
           --&gt; 80°C for 10min (deactivation)
</PRE><TABLE class="wikitable"><TBODY><TR><TD> Plasmid </TD><TD> 1 </TD><TD> 2 </TD><TD> 3
</TD></TR><TR><TD> P </TD><TD>   10µl </TD><TD> 10µl </TD><TD> 18µl
</TD></TR><TR><TD> BamHI (FD) </TD><TD>   1µl </TD><TD> 1µl </TD><TD> 1µl
</TD></TR><TR><TD> HindIII (FD) </TD><TD>   1µl </TD><TD> 1µl </TD><TD> 1µl
</TD></TR><TR><TD> Buffer (FD) </TD><TD>   2µl </TD><TD> 2µl </TD><TD> 3µl
</TD></TR><TR><TD> nuc.free H<SUB>2</SUB>O </TD><TD>   6µl </TD><TD> 6µl </TD><TD> 7µl
</TD></TR></TBODY></TABLE><PRE>           --&gt; 37°C for 1h (restriction)
           --&gt; 80°C for 10min (deactivation)
</PRE><PRE><EM>PCR Purification (without isopropanol)</EM></PRE><PRE>           Elution buffer 20µl; 1min incubated
</PRE><TABLE class="wikitable"><TBODY><TR><TD> plasmid P1-3 </TD><TD> 45,3ng/µl
</TD></TR><TR><TD> BfR </TD><TD> 12,9ng/µl
</TD></TR><TR><TD> FTNA1 </TD><TD> 34,4 ng/µl
</TD></TR><TR><TD> FTNA2 </TD><TD> 29,9ng/µl
</TD></TR></TBODY></TABLE><PRE><EM>Ligation Assay</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> Control </TH><TH> BFR </TH><TH> BFR </TH><TH> BFR </TH><TH> FTNA1 </TH><TH> FTNA1 </TH><TH> FTNA1 </TH><TH> FTNA2 </TH><TH> FTNA2 </TH><TH> FTNA2 </TH><TH> BFR_E </TH><TH> BFR_E </TH><TH> BFR_E
</TH></TR><TR><TD> Dilution </TD><TD> - </TD><TD> 1x </TD><TD> 3x </TD><TD> 5x</TD><TD>1x</TD><TD> 3x </TD><TD> 5x</TD><TD>1x </TD><TD> 3x </TD><TD> 5x </TD><TD>1x </TD><TD> 3x </TD><TD> 5x
</TD></TR><TR><TD> Vector DNA </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1 </TD><TD> 1,1
</TD></TR><TR><TD> Insert DNA </TD><TD> 0 </TD><TD> 0,4 </TD><TD> 1,6 </TD><TD> 2,0 </TD><TD> 0,10 </TD><TD> 0,47 </TD><TD> 0,79 </TD><TD> 0,18 </TD><TD> 0,52 </TD><TD> 0,90 </TD><TD> 0,40 </TD><TD> 1,16 </TD><TD> 2,0
</TD></TR><TR><TD> Buffer </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> T4 DNA-Ligase </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25 </TD><TD> 0,25
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 7,65 </TD><TD> 7,25 </TD><TD> 6,5 </TD><TD> 5,65 </TD><TD> 7,5 </TD><TD> 7,20 </TD><TD> 6,85 </TD><TD> 7,50 </TD><TD> 7,20 </TD><TD> 6,85 </TD><TD> 7,25 </TD><TD> 6,50 </TD><TD> 5,65
</TD></TR></TBODY></TABLE><PRE><STRONG>Transformation</STRONG>
           --&gt; Transformation of 5µl Sample into DH5α-Cells
           --&gt; Incubation o/n at 37°C after streaking out on LB+amp plates using sterile beads
</PRE><TABLE class="wikitable"><TBODY><TR><TH> BFR </TH><TH> FTNA1 </TH><TH> FTNA2
</TH></TR><TR><TD> 5,2ng </TD><TD> 5,51ng </TD><TD> 5,43ng
</TD></TR><TR><TD> 15,6ng </TD><TD> 16,5ng </TD><TD> 16,27ng
</TD></TR><TR><TD> 26,0ng </TD><TD> 27,4ng </TD><TD> 27,13ng
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/07 Monday - Cloning Results </H2><TABLE class="wikitable"><TBODY><TR><TH> Sample </TH><TH> CFU
</TH></TR><TR><TD> Control </TD><TD> 13
</TD></TR><TR><TD> tBFR 1 </TD><TD> 121
</TD></TR><TR><TD> tBFR 3 </TD><TD> 38
</TD></TR><TR><TD> tBFR 5 </TD><TD> 664
</TD></TR><TR><TD> BFR 1 </TD><TD> 87
</TD></TR><TR><TD> BFR 3 </TD><TD> 151
</TD></TR><TR><TD> BFR 5 </TD><TD> 150
</TD></TR><TR><TD> FTNA1 1 </TD><TD> 65
</TD></TR><TR><TD> FTNA1 3 </TD><TD> 131
</TD></TR><TR><TD> FTNA1 6 </TD><TD> 191
</TD></TR><TR><TD> FTNA2 1 </TD><TD> 84
</TD></TR><TR><TD> FTNA2 3 </TD><TD> 114
</TD></TR><TR><TD> FTNA2 5 </TD><TD> 151
</TD></TR></TBODY></TABLE><PRE>           --&gt; picked 5 colos per construct for coloPCR (50µl TE-buffer; 96°C; 10min)
           --&gt; rescue plate
</PRE><PRE><B>colPCR Programm</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> 1x </TH><TH> Mastermix 23x
</TH></TR><TR><TD> nuc.free H<SUB>2</SUB>O </TD><TD> 13,7µl </TD><TD> 315,1µl
</TD></TR><TR><TD> 10x Dream Taq Buffer </TD><TD> 2µl </TD><TD> 46µl
</TD></TR><TR><TD> 25mM MgCl<SUB>2</SUB></TD><TD> 1,2µl </TD><TD> 27,6µl
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 0,5µl </TD><TD> 11,5µl
</TD></TR><TR><TD> Primer T5 prom (PB16) </TD><TD> 0,5µl </TD><TD> 11,5µl
</TD></TR><TR><TD> Primer T5 term (PB17) </TD><TD> 0,5µl </TD><TD> 11,5µl
</TD></TR><TR><TD> boiled colos </TD><TD> 1µl + 18,4ml Mastermix
</TD></TR><TR><TD> Taq Polymerase </TD><TD> 0,5µl
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 95°C </TD><TD> 3'
</TD></TR><TR><TD>30 cycles
</TD></TR><TR><TD> 95°C </TD><TD> 30
</TD></TR><TR><TD> 55°C </TD><TD> 30
</TD></TR><TR><TD> 72°C </TD><TD> 1'
</TD></TR><TR><TD> 72°C </TD><TD> 10'
</TD></TR><TR><TD> 12°C </TD><TD> hold
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/08 Tuesday - Amplification of ferritin genes </H2>

            Primer designed for amplification:
<PRE><B>primer Bfr:</B></PRE><PRE>           FW: 5'   GC   G| G A T C C AAAGGTGATACTAAAGTTATAAATTATCTC(GC 23,33% Tm = 57,5)  3'  OK
</PRE><PRE>           RW: 5'   GC   A| A G C T T TCAACCTTCTTCGCG(GC 50% Tm = 58,5)  3'                    OK
</PRE><PRE><B>primer FtnA1:</B></PRE><PRE>           FW: 5'   GC   G| G A T C C GCAACCGCTGGAATG(GC 60%  Tm = 60,5) 3'                    OK
</PRE><PRE>           RW: 5'   GC   A| A G C T T TCAATGCAGCTGATGC(GC 50,0% Tm = 58,5)  3'                 OK
</PRE><PRE><B>primer FtnA2:</B></PRE><PRE>           FW: 5'   GC   G| G A T C C CTGAAACCAGAAATGATTG(GC 36,8%  Tm = 65,1)  3'             OK
</PRE><PRE>           RW: 5'   GC   A| A G C T T TTAGTTTTGTGTGTCGAGG(GC 42,1%  Tm = 56,8)  3'             OK
</PRE><PRE>           Using the Thermoscientific  PCR mastermix phusion polymerase, 2x 50 µl reactions per amplification:
</PRE><TABLE class="wikitable"><TBODY><TR><TD> Nuc-free H<SUB>2</SUB>O </TD><TD> 39,5 µl
</TD></TR><TR><TD> 2x mastermix </TD><TD> 50,0 µl
</TD></TR><TR><TD> FW primer </TD><TD>   5,0 µl
</TD></TR><TR><TD> RW primer </TD><TD>   5,0 µl
</TD></TR><TR><TD> template DNA </TD><TD>   0,5 µl
</TD></TR></TBODY></TABLE><PRE><B>PCR programs:</B></PRE><TABLE class="wikitable"><TBODY><TR><TD> 98°C </TD><TD> 10s </TD><TD> 1x
</TD></TR><TR><TD> 98°C </TD><TD> 5s </TD><TD> 30x
</TD></TR><TR><TD>primer</TD><TD> Bfr/ FtnA1/ FtnA2 </TD><TD> 60,5°C/ 58,9°C/ 56,1°C </TD><TD> 30x
</TD></TR><TR><TD> 72°C </TD><TD> 10s </TD><TD> 30x
</TD></TR><TR><TD> 72°C </TD><TD> 60s </TD><TD> Beispiel </TD><TD> 1x
</TD></TR></TBODY></TABLE><PRE><STRONG>Transformation of pQE_80L into DH5alpha</STRONG></PRE><PRE>           Plasmid from plasmid database of TU workgroup by Prof. Budisa was taken: 
           plasmid ID: 4; concentration = 308 ng/µl</PRE><PRE>           Unthaw 50 µl aliquot of DH5alpha or BL21 DE3 gold on ice for 10 min.Than Use 308 ng and 616 ng of plasmids to bind on Ca<BIG>2+</BIG>-surface of the cell membranes. Heat shock was performed for 30-90 s. The cells were incubated on ice for 2 min and then there was added 950 µl LB media. The cells were incubated at 37°C for 60 min. After this 70 µl of transformed E. coli suspension was streaked out onto a plate with the appropriate selection marker. The plate was incubated over night at 37°C. To prepare preculture 2 colonies were picked and added into 5 ml LB media + 5 µl AMP. The precultures were incubated over night at 37°C and 200 rpm.
 
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/10 Thursday - Expression of Ferritin in LB </H2>

           →Inoculate 20ml LB + amp with 1ml of a preculture<TABLE><TBODY><TR><TH> Title </TH><TH> No. </TH><TH> Wavelength </TH><TH> Absorbance </TH><TH> Volume
</TH></TR><TR><TD> BfR </TD><TD> 1 </TD><TD> 600nm </TD><TD> 0,529A </TD><TD> 1,51ml
</TD></TR><TR><TD> FTNA1</TD><TD> 2 </TD><TD> 600nm </TD><TD> 0,307A </TD><TD> 3,25ml
</TD></TR><TR><TD> FTNA2 </TD><TD> 3 </TD><TD> 600nm </TD><TD> 0,605A </TD><TD> 1,32ml
</TD></TR></TBODY></TABLE><PRE>           →Incubation for 2h until OD<SUB>600</SUB>=0,6-08</PRE><TABLE><TBODY><TR><TH> Title </TH><TH> No. </TH><TH> Wavelength </TH><TH> Absorbance
</TH></TR><TR><TD> BfR </TD><TD> 1 </TD><TD> 600nm </TD><TD> 0,825A
</TD></TR><TR><TD> FTNA1</TD><TD> 2 </TD><TD> 600nm </TD><TD> 0,883A
</TD></TR><TR><TD> FTNA2 </TD><TD> 3 </TD><TD> 600nm </TD><TD> 0,859A
</TD></TR></TBODY></TABLE><PRE>           → Take SDS Sample non-induced
           →Induce with 20µl IPTG
           →Add Fe<SUP>2+</SUP>
           →Expression: 4h at 22°C
           
            Cultivation was aborted because of heat development
            </PRE></DIV><DIV class="sub-content-project">                                              .                                    
            <H2 class="sub-content-project-headline"> 04/14 Monday - Miniprep </H2>
            Miniprep of the cell-separation streak out of the previous cloning procedure <B>Miniprep-Kit of ThermoScientific</B><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> Title </TH><TH> Concentration in ng/µl
</TH></TR><TR><TD> A4 </TD><TD> BfR </TD><TD> 66
</TD></TR><TR><TD> B3 </TD><TD> FTNA1 </TD><TD> 138
</TD></TR><TR><TD> C5 </TD><TD> FTNA2 </TD><TD> 88
</TD></TR><TR><TD> D2 </TD><TD> BfR_Electro </TD><TD> 144
</TD></TR></TBODY></TABLE><PRE><STRONG><BIG><B>DNA Concentration Determination</B></BIG></STRONG></PRE><TABLE><TBODY><TR><TD> Dilution Factor </TD><TD> 20
</TD></TR><TR><TD> Integration Time </TD><TD> 1s
</TD></TR><TR><TD> Factor </TD><TD> 50
</TD></TR><TR><TD> Units </TD><TD> µg/ml
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> Title </TH><TH> No. </TH><TH> 260nm </TH><TH> 280nm </TH><TH> 320nm </TH><TH> Ratio </TH><TH> Conc.
</TH></TR><TR><TD> A4 </TD><TD> BfR </TD><TD> 1 </TD><TD> 0,056 </TD><TD> -0,022</TD><TD> -0,010 </TD><TD> 2,05 </TD><TD> 66
</TD></TR><TR><TD> B3 </TD><TD> FTNA1 </TD><TD> 2 </TD><TD> 0,135 </TD><TD> 0,067 </TD><TD> -0,003 </TD><TD> 1,96 </TD><TD> 138
</TD></TR><TR><TD> C5 </TD><TD> FTNA2 </TD><TD> 3 </TD><TD> 0,078 </TD><TD> 0,035</TD><TD> -0,008 </TD><TD> 2,00 </TD><TD> 86
</TD></TR><TR><TD> D2 </TD><TD> BfR-Electro </TD><TD> 4 </TD><TD> 0,150 </TD><TD> 0,078 </TD><TD> 0,006 </TD><TD> 2,02 </TD><TD> 144
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/16 Wednesday - Sequencing </H2>
            Preparation of the sequencing for the Samples from 16.01.2013<TABLE><TBODY><TR><TD> T5 prom. Primer </TD><TD> 3µl
</TD></TR><TR><TD> Plasmid </TD><TD> 12µl
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD> A4 </TD><TD> 6µl
</TD></TR><TR><TD> B3 </TD><TD> 6µl
</TD></TR><TR><TD> C5 </TD><TD> 6µl
</TD></TR><TR><TD> D2 </TD><TD> 6µl
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/17 Thursday - Expression of BfR and FTNA1/2 </H2>
            4x sterile 250ml Erlenmeyer
            +50ml LB
            +50µl amp
            +2,5ml preculture
            Incubation: 1h at 30°C

<TABLE class="wikitable"><TBODY><TR><TH> No. </TH><TH> Title </TH><TH> Wavelength </TH><TH> Absorbance
</TH></TR><TR><TD> 1 </TD><TD> BfR </TD><TD> 600nm </TD><TD> 0,909A
</TD></TR><TR><TD> 2 </TD><TD> FTNA1 </TD><TD> 600nm </TD><TD> 0,980A
</TD></TR><TR><TD> 3 </TD><TD> FTNA2 </TD><TD> 600nm </TD><TD> 0,930A
</TD></TR><TR><TD> 4 </TD><TD> BfR+20?? </TD><TD> 600nm </TD><TD> 0,970A
</TD></TR></TBODY></TABLE><P>            
            →Take SDS-Sample
            →Induction with 50µl IPTG
            →Addition of 0,0525g Fe-citrate (20mM dissolved in 1ml sterile Water)
            →Incubation for 3h at 30°C
            →continue incubation o/n
            →Centrifuge with 6800g for 5min</P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/18 Friday - Production of LB-Plates </H2><EM>Production of a Mutaflor-Supression-culture</EM>
            →10ml sterile LB
            →Add the content of a Mutaflor capsula (white powder)
            Incubation: 37°C at 220rpm<EM>Production of dilutions of the Suppression-culture and out streaking</EM>
            →1µl of the suppression-culture to 999µl LB (1:1000)
            →100µl of that dilution to 900µl LB (1:10<SUP>4</SUP>)
            →10µl of that dilution to 990µl LB (1:10<SUP>5</SUP>)
            →Streak out 50µl of each culture on LB with sterile beads
            Incubation: 37°C
            </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/24 Thursday - Mini-Prep </H2><TABLE class="wikitable"><TBODY><TR><TD> 1 </TD><TD> PSB </TD><TD> D40 </TD><TD> 9fP </TD><TD> CR/LB
</TD></TR><TR><TD> 2 </TD><TD> PSB </TD><TD> D20 </TD><TD> 9fP </TD><TD> CR/LB
</TD></TR><TR><TD> 3 </TD><TD> PSB </TD><TD> D40 </TD><TD> fs </TD><TD> LB/CR
</TD></TR><TR><TD> 4 </TD><TD> PSB </TD><TD> D40 </TD><TD> LB/CR
</TD></TR><TR><TD> 5 </TD><TD> PSB </TD><TD> AB </TD><TD> Turyu </TD><TD> CR/LB
</TD></TR><TR><TD> 6 </TD><TD> PSB </TD><TD> D48 </TD><TD> 9fP </TD><TD> CB/CR
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/28 Monday -PCR Plasmid/ Primer and Parameter Check </H2>
            (Salah and Christina)

            Check of:
            -Plasmids/Primer
            -Annealing temperature
            -Phusion and Q5 Polymerase

           <EM>Plasmids</EM><PRE>           iGEM Plasmid pkD4 (Stock 200ng/µl) =&gt;1:200 diluted = 1ng/µl
</PRE><PRE>           WH (Willi Hauck) Plasmid pkD4 (0,72ng/µl)
</PRE><PRE><EM>Primer</EM></PRE><PRE>           P1 FieF (iGEM)                                   fwd tnaA KO (WH)
</PRE><PRE>           P2 FieF (iGEM)                                   rev  tnaA KO (WH)
</PRE><PRE><EM>Combinations</EM></PRE><PRE>           (4 combinations with Phusion; 4 combinations with Q5 )
</PRE><TABLE border="2"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Primer</B></TD><TD><B>Primer</B></TD></TR><TR><TD><B>1. </B></TD><TD> iGEM Plasmid
</TD><TD> P1 (iGEM)
</TD><TD> P2 (iGEM)
</TD></TR><TR><TD><B>2. </B></TD><TD> WH Plasmid
</TD><TD> fwd tnaA KO
</TD><TD> rev  tnaA KO            <B>POSITIVCONTROL</B></TD></TR><TR><TD><B>3. </B></TD><TD> iGEM Plasmid
</TD><TD> fwd tnaA KO
</TD><TD> rev  tnaA KO
</TD></TR><TR><TD><B>4. </B></TD><TD> WH Plasmid
</TD><TD> P1 (iGEM)
</TD><TD> P2 (iGEM)
</TD></TR></TBODY></TABLE><PRE>           Each of the four combinations was prepared 3 times =&gt; for Gradient PCR
</PRE><PRE><EM>Phusion Polymerase PCR Assay</EM></PRE><PRE>           20µl Reaction
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Plasmid </TH><TH> Primer </TH><TH> Primer
</TH></TR><TR><TD> 1. </TD><TD> iGEM Plasmid </TD><TD> P1 (iGEM) </TD><TD> P2 (iGEM)
</TD></TR><TR><TD> 2. </TD><TD> WH Plasmid </TD><TD> fwd tnaA KO </TD><TD> rev  tnaA KO            POSITIVCONTROL
</TD></TR><TR><TD> 3. </TD><TD> iGEM Plasmid </TD><TD> fwd tnaA KO </TD><TD> rev  tnaA KO
</TD></TR><TR><TD> 4. </TD><TD> WH Plasmid </TD><TD> P1 (iGEM) </TD><TD> P2 (iGEM)
</TD></TR></TBODY></TABLE><PRE><EM>Q5 Polymerase PCR Assay</EM></PRE><PRE>           20µl Reaction
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>5xQ5 Reaction Puffer (G2 round container)</B></TD><TD align="center" style="background:#f0f0f0;"><B>4µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>1x</B></TD></TR><TR><TD> 10mM dNTPs </TD><TD>0,4µl</TD><TD>200µM
</TD></TR><TR><TD> 10µM forward Primer</TD><TD>1µl</TD><TD>0,5µM
</TD></TR><TR><TD> 10µM reverse Primer</TD><TD>1µl</TD><TD>0,5µM
</TD></TR><TR><TD> Template DNA</TD><TD>1µl</TD><TD>1pg-1ng sein
</TD></TR><TR><TD> Q5 High Fidelity DNA Polymerase</TD><TD>0,2µl</TD><TD>0,02U/µl
</TD></TR><TR><TD> GC enhancer</TD><TD>4 µl</TD></TR><TR><TD> Nuclease-Free Water </TD><TD>to 20µl</TD></TR><TR><TD>8,4µl</TD></TR></TBODY></TABLE><PRE><EM>FUR PCR Assay</EM></PRE><PRE>           (Along with the above mentioned PCR Assays another Gradient PCR with FUR-Primer is performed)</PRE><PRE>           Aliquot of the FUR Primer (100µM) 1:10 diluted =&gt;10µM</PRE><PRE>           20µl Reaction with Phusion-Polymerase (see above)</PRE><PRE>           with WH Plasmid and P1 FUR &amp; P2 FUR Primer</PRE><PRE><EM>Gradient PCR Program</EM></PRE><PRE>           Thermocycling Conditions for the Gradient PCR </PRE><PRE>           Programm-NAME: Q5 grad trp KO</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Step</B></TD><TD align="center" style="background:#f0f0f0;"><B>TEMP</B></TD><TD align="center" style="background:#f0f0f0;"><B>TIME</B></TD></TR><TR><TD> Initial Denaturation</TD><TD>98°C</TD><TD>30 sec
</TD></TR><TR><TD> 5 Cycles</TD><TD>98°C</TD><TD>8 sec
</TD></TR><TR><TD>64±5°C</TD><TD>25 sec    GRADIENT
</TD></TR><TR><TD>72°C</TD><TD>Trp ko: 1528bpà45s
</TD></TR><TR><TD> 25 Cycles</TD><TD>98°C</TD><TD>8 sec
</TD></TR><TR><TD>72°C</TD><TD>70 sec
</TD></TR><TR><TD> Final Extension</TD><TD>72°C</TD><TD>2 min
</TD></TR><TR><TD> Hold</TD><TD>8°C</TD></TR></TBODY></TABLE><PRE><EM>Gel-Electrophoresis</EM></PRE><PRE>           1% Agarose Gel</PRE><PRE>           5µl Sample +1µl loading dye (1:6)</PRE><PRE>           8µl diluted GeneRuler Mix ladder</PRE><PRE>           MODE: 90V 25min</PRE><PRE><B>Sample label:</B></PRE><PRE>           P = with Phusion Polymerase</PRE><PRE>           Q = with Q5 Polymerase</PRE><PRE>           1 = 1. Combination</PRE><PRE>           2 = 2. Combination</PRE><PRE>           3 = 3. Combination</PRE><PRE>           4 = 4. Combination</PRE><PRE><B>Gradient</B></PRE><PRE>           1 = 59°C
</PRE><PRE>           4 = 63°C
</PRE><PRE>           8 = 69°C
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Gel 1 Phusion</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> P FUR</TD><TD>P1</TD><TD>P1</TD><TD>P1</TD><TD>P2</TD><TD>P2</TD><TD>P2</TD><TD>Marker</TD><TD>P3</TD><TD>P3</TD><TD>P3</TD><TD>P4</TD><TD>P4</TD><TD>P4
</TD></TR><TR><TD> 8</TD><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8
</TD></TR><TR><TD> Gel 2 Q5</TD></TR><TR><TD>Q1</TD><TD>Q1</TD><TD>Q1</TD><TD>Q2</TD><TD>Q2</TD><TD>Q2</TD><TD>Marker</TD><TD>Q3</TD><TD>Q3</TD><TD>Q3</TD><TD>Q4</TD><TD>Q4</TD><TD>Q4
</TD></TR><TR><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8</TD><TD>1</TD><TD>4</TD><TD>8
</TD></TR></TBODY></TABLE><PRE><STRONG>Results</STRONG></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Gel 1 Phusion Result:</B></TD><TD align="center" style="background:#f0f0f0;"><B>For FUR-Primer no amplification</B></TD></TR><TR><TD>For iGEM-Plasmid no amplification
</TD></TR><TR><TD>Combination 2 (WH Plasmid &amp; Primer)= only non-specific amplification
</TD></TR><TR><TD>Combination 4 (WH Plasmid &amp; iGEM Primer)=Bands for 59°C and 63°C + non-specific amplification
</TD></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Gel 2 Q5 Result:</B></TD><TD>For iGEM-Plasmid no amplifikation
</TD></TR><TR><TD>Combination 2  (WH Plasmid &amp; Primer) = bands for all three annealing temperature observed+non-specific amplification
</TD></TR><TR><TD>Combination 4 (WH Plasmid &amp; iGEM Primer)= Bands at all three annealing temperature+non-specific amplification
</TD></TR></TBODY></TABLE><P>
            iGEM Plasmid not to be used anymore.
            FUR need to be checked again.
            </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04/29 Tuesday - Mini-Prep </H2>
            (Aritra)
            [pKD46 + DH5α]→ 155ng/µl
            [pKD4 + DH5α]→ 302ng/µl</DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/02 Friday - Genomic DNA extraction from Pseudomonas putida </H2>
            (Johann)

            --&gt;see Protocol Wizard ® Genomic DNA Purification Kit

                - Strain from VLB Martin Senz PHO Psedomonas putida KT2242  B_0712  from 29.04.2014 culture
                - 2 days in 30°C  Shaking Incubator.
                - Mikro 22R Centrifuge Hettich (16000g)

            Final DNA Concentration measured after purification = 95ng/µl
            </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 02.05.2014 - DNA Digestion to test Plasmids </H2><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Samples</B></TD><TD align="center" style="background:#f0f0f0;"><B>Conc. DNA (ng/µl)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Enzymes used</B></TD><TD align="center" style="background:#f0f0f0;"><B>Expected band length</B></TD><TD align="center" style="background:#f0f0f0;"><B>Evaluation</B></TD></TR><TR><TD> pKD4</TD><TD>302</TD><TD>Xbal</TD><TD>1874,1393</TD><TD>Possible plasmid, failed
</TD></TR><TR><TD> pKD46</TD><TD>40</TD><TD>EcoRI</TD><TD>4820,1509</TD><TD>Failed
</TD></TR><TR><TD> pKD46</TD><TD>39</TD><TD>EcoRI</TD><TD>4820,1509</TD><TD>Failed
</TD></TR><TR><TD> pKD4</TD><TD>60</TD><TD>Xbal</TD><TD>1874,1393</TD><TD>Possible(passed); whole plasmid
</TD></TR><TR><TD> pSBAC3</TD><TD>88</TD><TD>SpeI/EcoRI</TD><TD>2047,22</TD><TD>Possible (passed); smallpattern
</TD></TR><TR><TD> pSBAC3-D20-GFP</TD><TD>115</TD><TD>SpeI/EcoRI</TD><TD>2047,957</TD><TD>whole plasmid; failed
</TD></TR><TR><TD> pSBAC3-D40-Bfr</TD><TD>86</TD><TD>SpeI/EcoRI</TD><TD>2047, 774</TD><TD>Whole plasmid; failed
</TD></TR><TR><TD> pSBAC3-D40-GFP</TD><TD>80</TD><TD>SpeI/EcoRI</TD><TD>2047,1014</TD><TD>Whole plasmid; failed
</TD></TR><TR><TD> pSBAC3-AB-Tyrosin</TD><TD>110</TD><TD>SpeI/EcoRI</TD><TD>2047,3319</TD><TD>Failed
</TD></TR><TR><TD> pSBAC3-D40</TD><TD>164</TD><TD>SpeI/EcoRI</TD><TD>2047,267</TD><TD>Failed
</TD></TR><TR><TD> pSBAC3-D40-GFP-ssr</TD><TD>106</TD><TD>SpeI/EcoRI</TD><TD>2047,1032</TD><TD>Passed; whole plasmid
</TD></TR></TBODY></TABLE><PRE><STRONG>Gel Electrophoresis</STRONG><B>Key:</B>
           -band1: Ladder
           -band5:Ladder
           -residual bands, see table above
           
<EM>Results</EM>

            pKD4 with 60(ng/µl) best candidate for PCR.
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/05 Monday - Preculture of strains from Budisa strain database in LB Medium </H2>
          

             1. BU31 (pKD4) in 50 ml LB+Kanamycin+Ampicillin  37°C 220 rpm
             2. BU32 (pKD46) in 50 ml LB+Ampicillin 30°C 240rpm
             3. BU31 (pKD46) in LB plate+Ampicillin  30°C <STRONG>Midiprep</STRONG>
            

            Midiprep of BU31 (pKD4) and BU32 (pKD46) from strain Database.

            Concentration pKD4 = 104 ng/µl

            Concentration pKD46 = 161 ng/µl</DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/06 Tuesday - Gel-Electrophoresis </H2>
        
            Kanamycin Cassette PCR (pKD4 from MidiPrep 05.05.2014 104ng/µl Christina)<B>Key:</B>
           - 1st: Ladder<PRE>           - 2nd and 3rd: Fief</PRE><PRE>           - 4th: Negative control
           - 7th and 8th: FUR<EM>Results</EM>
            FieF worked well; FUR no bands

             
           <STRONG>Restriction digest of Plasmid pKD4 and pKD46</STRONG></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD>pKD4</TD><TD>pKD46
</TD></TR><TR><TD> Plamid </TD><TD>19.2µl</TD><TD>12.5µl
</TD></TR><TR><TD> Digestion Enzym</TD><TD>Xbal 1µl</TD><TD>Xmal 1µl
</TD></TR><TR><TD> nuclease free H2O</TD><TD>24.8µl</TD><TD>31.5µl
</TD></TR><TR><TD> 10xFO green Buffer</TD><TD>5µl</TD><TD>5µl
</TD></TR><TR><TD> Total</TD><TD>50µl</TD><TD>50µl
</TD></TR></TBODY></TABLE><PRE>              - Both the reaction mix incubated at 37°C for 1 hr.
              - After that another 1 hr at RT.
</PRE><PRE><EM>Gel-Electrophoresis</EM>
           - 20µl of both the samples together with whole plasmids loaded on 1% Agarose Gel<STRONG>Results</STRONG><B>Key:</B>
           -1st: pKD4 (XbaI digested)
           -2nd: pKD4 (undigested)
           - 3rd: Ladder
           - 4th: pKD46 (undigested)
           -5th: pKD46 (XmaI digested)</PRE><PRE><EM>Gel Extraction</EM>
           -Used GeneJet GelExtraction Kit by Thermo Scientific
           <STRONG>Results</STRONG>
           DNA Concentration: 16ng/µl
</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/16 Friday - Biotransformation of Ferritin and pKD46 in Nissle and DH5-α strain </H2>
            concentration Ferritin 120ng/µl
            concentration pKD 46 95 ng/µl<PRE>           Protocol
           *electro competent cells of DH5α and Nissle were thawed on ice
           *0.4µl of Ferritin and 0.53 µl of pKD46 was added to the cells
           *cells with DNA were transferred to cuvettes for electroporation, then 950ml (LB-medium) was added to the cells immediately
           *incubation at 37°C for ferritin, 30°C for pKd46 for 1 h with shaking
           *the cells were plated on Amp plates for pKD46 and CM for Ferritin
           *Transformation did not work.
           
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/21 Wednesday - PCR Gel-Extraction of FieF </H2>


            1% Agarosegel + 0,5µl EtBr
            45µl Sample+9µl Loading Dye
            8µl Ladder
   
           --&gt; Gel-Extraction Kit used with 50µl nuclease-free water for elution<STRONG>Gene Knockout of FieF</STRONG>
           (Salah)
           <EM>Preparation of the preculture</EM><B>Used Precultures:</B>
           - RV +pKD46
           - WM10 +pKD46
           The precultures was diluted to OD<SUB>600</SUB>=0,1 and then incubated for 2h at 30°C until they reach OD<SUB>600</SUB>=0,6<TABLE class="wikitable"><TBODY><TR><TH> Title </TH><TH> Wavelength </TH><TH> Absorbance
</TH></TR><TR><TD> Reference </TD><TD> 600nm </TD><TD> 0,000A
</TD></TR><TR><TD> RV+pKD46 </TD><TD> 600nm </TD><TD> 0,066A
</TD></TR><TR><TD> WM10+pKD46 </TD><TD> 600nm </TD><TD> 0,050A
</TD></TR></TBODY></TABLE><PRE>           (The culture was diluted 1/10 for the OD measurement)
           →The precultures are ready for the recombinase induction with Arabinose
           <EM>Induction with Arabinose</EM>
           The cultures have been induced with 500µl arabinose (1M)
           →After the induction, the cultures have to be incubated for further 30min at 30°C
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Title </TH><TH> Wavelength </TH><TH> Absorbance
</TH></TR><TR><TD> Reference </TD><TD> 600nm </TD><TD> 0,000A
</TD></TR><TR><TD> RV+pKD46 </TD><TD> 600nm </TD><TD> 0,084A
</TD></TR><TR><TD> WM10+pKD46 </TD><TD> 600nm </TD><TD> 0,067A
</TD></TR></TBODY></TABLE><PRE>           →From now on the cultures need to be cooled on ice
           <EM>Preparation for the Electroporation</EM></PRE><PRE><EM>Electroporation</EM><B>Electroporation-Program: Ec1</B>
           -Amount of DNA sample: 75-150ng</PRE><TABLE class="wikitable"><TBODY><TR><TH> Sample </TH><TH> Conc </TH><TH> Used Volume </TH><TH> Strain </TH><TH> Electroporation Time
</TH></TR><TR><TD> FieF1 </TD><TD> 42ng/µl </TD><TD> 2µl (84ng) </TD><TD> RV </TD><TD> 4,4ms
</TD></TR><TR><TD> FieF2 </TD><TD> 9ng/µl </TD><TD> 10µl (90ng) </TD><TD> RV </TD><TD> 4,1ms
</TD></TR><TR><TD> FieF1 </TD><TD> 42ng/µl </TD><TD> 2µl (84ng) </TD><TD> WM10 </TD><TD> 4,3ms
</TD></TR><TR><TD> FieF2 </TD><TD> 9ng/µl </TD><TD> 10µl (90ng) </TD><TD> WM10 </TD><TD> 4,8ms
</TD></TR></TBODY></TABLE><PRE>           After the transformation, the cultures was immediately prepared with 950µl SOC and incubated for 60min at 30°C without antibiotics --&gt; recovery
           <EM>Selection-cultures</EM>
           The selection of positive clones is done by using LB plates with kan+amp
           Incubation: 30°C o/n
           
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/22 Thursday - Transformation of pKD46 </H2><EM>Preparation of the preculture</EM><B>Used Precultures:</B>
           - MG1655
           - Nissle
           The precultures was diluted to OD<SUB>600</SUB>=0,1 and then incubated for 2h at 30°C until they reach OD<SUB>600</SUB>=0,6<TABLE class="wikitable"><TBODY><TR><TH> Title </TH><TH> Wavelength </TH><TH> Absorbance
</TH></TR><TR><TD> Reference </TD><TD> 600nm </TD><TD> 0,000A
</TD></TR><TR><TD> MG1655 </TD><TD> 600nm </TD><TD> 0,062A
</TD></TR><TR><TD> NIssle </TD><TD> 600nm </TD><TD> 0,054A
</TD></TR></TBODY></TABLE><PRE>           (The culture was diluted 1/10 for the OD measurement)</PRE><PRE>           →From now on the cultures need to be cooled on ice
           <EM>Preparation for the Electroporation</EM></PRE><PRE><EM>Electroporation</EM><B>Electroporation-Program: Ec1</B>
           -Amount of DNA sample: 75-150ng</PRE><TABLE class="wikitable"><TBODY><TR><TH> Sample </TH><TH> Conc </TH><TH> Used Volume </TH><TH> Strain </TH><TH> Electroporation Time
</TH></TR><TR><TD> MG1655 1 </TD><TD> 42ng/µl </TD><TD> 2µl (84ng) </TD><TD> RV </TD><TD> 5,4ms
</TD></TR><TR><TD> MG1655 2 </TD><TD> 9ng/µl </TD><TD> 10µl (90ng) </TD><TD> RV </TD><TD> 5,8ms
</TD></TR><TR><TD> Nissle 1 </TD><TD> 42ng/µl </TD><TD> 2µl (84ng) </TD><TD> WM10 </TD><TD>0,7ms (thrown away)
</TD></TR><TR><TD> Nissle 2 </TD><TD> 9ng/µl </TD><TD> 10µl (90ng) </TD><TD> WM10 </TD><TD> 5,8ms
</TD></TR></TBODY></TABLE><PRE>           After the transformation, the cultures was immediately prepared with 950µl SOC and incubated for 60min at 30°C without antibiotics --&gt; recovery
           <EM>Selection-cultures</EM>
           The selection of positive clones is done by using LB plates with amp
           Incubation: 30°C o/n
</PRE><PRE><STRONG> PCR of ATPCS and PPMT </STRONG></PRE><TABLE class="wikitable"><TBODY><TR><TH> Mastermix </TH><TH> ATPCS </TH><TH> PPMT
</TH></TR><TR><TD> nuc.free water</TD><TD> 18.9 </TD><TD> 17.5
</TD></TR><TR><TD> 2x Fusion Mastermix </TD><TD> 25 </TD><TD> 25
</TD></TR><TR><TD> Fw Primar </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> Rw Primar </TD><TD> 2.5</TD><TD> 2.5
</TD></TR><TR><TD> template DNA </TD><TD> 1.1 </TD><TD> 2.5
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> ATPCS </TH><TH> Program
</TH></TR><TR><TD> 98°C </TD><TD> 10
</TD></TR><TR><TD> 98°C</TD><TD> 5 30 cycles
</TD></TR><TR><TD> 62°C</TD><TD> 5  30 cycles
</TD></TR><TR><TD> 72°C </TD><TD> 5   30 cycles
</TD></TR><TR><TD> 72°C</TD><TD>  60 secs
</TD></TR><TR><TD> 8°C</TD><TD> store
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> PPMT</TH><TH> Program
</TH></TR><TR><TD> 98°C </TD><TD> 10
</TD></TR><TR><TD> 98°C</TD><TD> 5 30 cycles
</TD></TR><TR><TD> 63,1°C</TD><TD> 5  30 cycles
</TD></TR><TR><TD> 72°C </TD><TD> 21  30 cycles
</TD></TR><TR><TD> 72°C</TD><TD>  60 secs
</TD></TR><TR><TD> 8°C</TD><TD> store
</TD></TR></TBODY></TABLE><PRE>           ATPCS 1469 bp; PPMT 234 bp
           --&gt; wrong Program
           
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/23 Friday - Digestion of Vector for Cloning </H2>
            Plasmid: pQE80L

            Old Plasmid of Johann: 
            -P1≈1000ng
            -P3≈400ng<TABLE class="wikitable"><TBODY><TR><TH> P3 </TH><TH> P1 </TH></TR><TR><TD> SacI </TD><TD> 1µl </TD><TD> SacI </TD><TD> 1µl
</TD></TR><TR><TD> HindIII </TD><TD> 1µl </TD><TD> BamHI </TD><TD> 1µl
</TD></TR><TR><TD> Buffer </TD><TD> 2µl </TD><TD> Buffer </TD><TD> 2µl
</TD></TR><TR><TD> Plasmid 1 </TD><TD> 4µl </TD><TD> Plasmid 3 </TD><TD> 16µl
</TD></TR><TR><TD> nuc.water </TD><TD> 12µl </TD><TD> nuc. water </TD><TD> 0µl
</TD></TR></TBODY></TABLE><PRE>           --&gt; Incubation: 1,5h at 37°C
           --&gt; Deactivation: 20min at 80°C
           
            </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/27 Tuesday - Production of LB Plates </H2>
        

            LB Agar Plates prepared:

            4x with Ampicillin   (25µl in 25ml)

            4x with Ampicillin+ Kanamycin   (25µl+25µl in 25ml)

            4x Cm (25µl in 25ml)
            </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05/28 Wednesday </H2>
              Picked up from MPI: AMB-1 and Microfluidic Chip. (From  the two AMB-1 cultures, one is in the freeze and the other one in the RT-Incubator)

              Digestion of pQE80L: 

              Marker: 2 log DNA ladder 5µl

              Sample 1: P1 Sacl-BamHI 10µl

              Sample 2: P3 Sacl-HindIII 10µl</DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/02 Monday - Colony PCR and analytical gel electrophoresis for identifying the right clones </H2>

              Primer: C1  C2<PRE>                      C2   C5</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Components</B></TD><TD align="center" style="background:#f0f0f0;"><B>20 µL Reaktion</B></TD></TR><TR><TD> Nuc. Free water</TD><TD>13.7
</TD></TR><TR><TD> 10x Dream Taq Polymerase Puffer</TD><TD>2
</TD></TR><TR><TD> 25mM MgCl2</TD><TD>1.2
</TD></TR><TR><TD> 10mM dNTPs</TD><TD>0.5
</TD></TR><TR><TD> 10mM FW Primer</TD><TD>0.5
</TD></TR><TR><TD> 10mM RW Primer</TD><TD>0.5
</TD></TR><TR><TD> Colony</TD><TD>1
</TD></TR><TR><TD> Taq DNA Pol</TD><TD>0.6
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Denaturation</B></TD><TD align="center" style="background:#f0f0f0;"><B>95°C</B></TD><TD align="center" style="background:#f0f0f0;"><B>10min</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> Denaturation</TD><TD>95°C</TD><TD>30sec</TD></TR><TR><TD> Annealing</TD><TD>55°C</TD><TD>30 sec</TD></TR><TR><TD> Extension</TD><TD>72°C</TD><TD>X min</TD><TD>30x
</TD></TR><TR><TD> Extension</TD><TD>72°C</TD><TD>10 min</TD></TR><TR><TD> End</TD><TD>12</TD><TD>hold</TD></TR></TBODY></TABLE><P>
              Backup plates done. Cells given direct to PCR. 11 colonies selected.

              Colony PCR  with some of the clones performed. (All negative)
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/03 Tuesday - Strategy 2: PCR of ATPCS 2PPMT </H2><PRE><EM>PCR Assay</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> Sample1(PPMT) </TH><TH> Sample2(ATPCS)
</TH></TR><TR><TD> nuc. free water </TD><TD> 17,5µl </TD><TD> 18,9µl
</TD></TR><TR><TD> FW Primer </TD><TD> 2,5µl </TD><TD> 2,5µl
</TD></TR><TR><TD> RW Primer </TD><TD> 2,5µl </TD><TD> 2,5µl
</TD></TR><TR><TD> 2x Phusion Mastermix </TD><TD> 25µl </TD><TD> 25µl
</TD></TR><TR><TD> Template DNA </TD><TD> 2,5µl (250ng) </TD><TD> 1,1µl (250ng)
</TD></TR></TBODY></TABLE><PRE><EM>PCR Program</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> PPMT (234bp) </TH><TH> ATPCS (1469bp) </TH></TR><TR><TD> 98°C </TD><TD> 10' </TD><TD> 98°C </TD><TD> 10'
</TD></TR><TR><TD>30cycles start
</TD></TR><TR><TD> 98°C </TD><TD> 5&quot; </TD><TD> 98°C </TD><TD> 5&quot;
</TD></TR><TR><TD> 62°C </TD><TD> 5&quot; </TD><TD> 63°C </TD><TD> 5&quot;
</TD></TR><TR><TD> 72°C </TD><TD> 10&quot; </TD><TD> 72°C </TD><TD> 45&quot;
</TD></TR><TR><TD>30cycles stop
</TD></TR><TR><TD> 72°C </TD><TD> 1' </TD><TD> 72°C </TD><TD> 1'
</TD></TR><TR><TD> 8°C </TD><TD> hold </TD><TD> 8°C </TD><TD> hold
</TD></TR></TBODY></TABLE><PRE><EM>Gel-Electrophoresis</EM><B>Key:</B>
             2nd band= Ladder
             3rd band= PPMT
             5th band= ATPCS</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/04 Wednesday - PCR amplification of ATPCS and PPMT </H2>

              (see under protocol &quot;Amplification BamHI_PPMT_Sacl and Sacl_ATPCS_HindIII&quot;

              After Gel: 1 Band visible for ATPCS around 1500bp (expected); no band for PPMT

              Purification of the ATPCS-PCR product from gel.

              conc. of ATPCS-fragment 6ng/microL

              freezed at -20°C on 1st floor
</DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 04.06.2014 - Colony PCR for Knockout </H2>

              6 clones were picked from 2 Agar plates; Agar plates labeled with numbers with RED

<PRE>             3 PCR:  a) Primers C1, C3
</PRE><PRE>                     b) Primers C2, C4
</PRE><PRE>                     c) Primers C1, C5
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>BandNr:</B></TD><TD align="center" style="background:#f0f0f0;"><B>Clone Nr:</B></TD><TD align="center" style="background:#f0f0f0;"><B>PCR type</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primers</B></TD></TR><TR><TD> 1</TD><TD>6</TD><TD>C</TD><TD>C1,C5
</TD></TR><TR><TD> 2</TD><TD>1</TD><TD>A</TD><TD>C1,C3
</TD></TR><TR><TD> 3</TD><TD>2</TD><TD>A</TD><TD>C1, C3
</TD></TR><TR><TD> 4</TD><TD>3</TD><TD>A</TD><TD>C1, C3
</TD></TR><TR><TD> 5</TD><TD>4</TD><TD>A</TD><TD>C1, C3
</TD></TR><TR><TD> 6</TD><TD>5</TD><TD>A</TD><TD>C1, C3
</TD></TR><TR><TD> 7</TD><TD>6</TD><TD>A</TD><TD>C1, C3
</TD></TR><TR><TD> 8</TD><TD>1</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 9</TD><TD>2</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 10</TD><TD>Marker</TD></TR><TR><TD> 11</TD><TD>3</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 12</TD><TD>4</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 13</TD><TD>5</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 14</TD><TD>6</TD><TD>B</TD><TD>C2,C4
</TD></TR><TR><TD> 15</TD><TD>1</TD><TD>C</TD><TD>C1,C5
</TD></TR><TR><TD> 16</TD><TD>2</TD><TD>C</TD><TD>C2,C4
</TD></TR><TR><TD> 17</TD><TD>3</TD><TD>C</TD><TD>C2,C4
</TD></TR><TR><TD> 18</TD><TD>4</TD><TD>C</TD><TD>C2,C4
</TD></TR><TR><TD> 19</TD><TD>5</TD><TD>C</TD><TD>C2,C4
</TD></TR></TBODY></TABLE><P>
              Results: all samples with same primers are identical

              PCR A: one band an dimer (900-800bp)

              PCR B: one band (1200bp)

              PCR C: two bands (800 and 400 bp)
               
              - 2 of the colonies are quite luckily mix colonies that contains our knockout, bands are shown for wild type and 2 show amplification for kanR cassette
              - they should be transferred to a new plate for single colony test
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/05 Thursday - New PCR protocol for ATPCS and PPMT with Q5 High Fidelity MasterMix </H2><PRE><B>PPMT PCR Reaction:</B></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Component</B></TD><TD align="center" style="background:#f0f0f0;"><B>25µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>Final Conc.</B></TD></TR><TR><TD> 2x Q5 MM</TD><TD>12. Mai</TD><TD>1x
</TD></TR><TR><TD> 10µM FW Primer</TD><TD>02. Mai</TD><TD>0,5µM
</TD></TR><TR><TD> 10µM RW Primer</TD><TD>2,5</TD><TD>0,5µM
</TD></TR><TR><TD> 95ng/µL Putida genomic DNA</TD><TD>2.63 (250ng)</TD><TD>9.99 ng/µL
</TD></TR><TR><TD> Nuc. Free water</TD><TD>Apr 87</TD></TR></TBODY></TABLE><PRE><B>ATPCS PCR Reaction</B></PRE><PRE>             result
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Component</B></TD><TD align="center" style="background:#f0f0f0;"><B>25µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>Final Conc.</B></TD></TR><TR><TD> 2x Q5 MM</TD><TD>1.,5</TD><TD>1x
</TD></TR><TR><TD> 10µM FW Primer</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 10µM RW Primer</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 230ng/µL Arabidopsis  cDNA</TD><TD>1.1 (250ng)</TD><TD>10 ng/µL
</TD></TR><TR><TD> Nuc. Free water</TD><TD>08. Sep</TD></TR></TBODY></TABLE><PRE>             PCR Program
             <B>PPMT 234 bp   </B>
             result
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>denaturing</B></TD><TD align="center" style="background:#f0f0f0;"><B>98°C</B></TD><TD align="center" style="background:#f0f0f0;"><B>180sec</B></TD></TR><TR><TD> Denaturing</TD><TD>98°C</TD><TD>10 sec
</TD></TR><TR><TD> Annealing            30x</TD></TR><TR><TD>64°C</TD><TD>30 sec
</TD></TR><TR><TD> Elongation</TD><TD>72°C</TD><TD>20 sec
</TD></TR><TR><TD> Final Elongation</TD><TD>72°C</TD><TD>2min
</TD></TR><TR><TD> Store</TD><TD>8°C</TD></TR></TBODY></TABLE><PRE><B>ATPCS 1469bp </B>
             result
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>denaturing</B></TD><TD align="center" style="background:#f0f0f0;"><B>98°C</B></TD><TD align="center" style="background:#f0f0f0;"><B>30 sec</B></TD></TR><TR><TD> Denaturing</TD><TD>98°C</TD><TD>10 sec
</TD></TR><TR><TD> Annealing               30x</TD></TR><TR><TD>62°C</TD><TD>30 sec
</TD></TR><TR><TD> Elongation</TD><TD>72°C</TD><TD>60 sec
</TD></TR><TR><TD> Final Elongation</TD><TD>72°C</TD><TD>2min
</TD></TR><TR><TD> Store</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 05.06.2014 </H2>

              Cryostocks: RV 308 ferritin (Cm)

                                 Nissle ferritin (Cm)

              500 µl 7%DMSO+ 500 µl Cell Culture medium =&gt; -80°C </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/06 Friday - PCR for ATPCS and PPMT with Q5 High Fidelity MasterMix </H2>

              PPMT PCR Reaction:<TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Component</B></TD><TD align="center" style="background:#f0f0f0;"><B>25µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>25µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>Final Conc.</B></TD></TR><TR><TD> 2x Q5 MM</TD><TD>12. Mai</TD><TD>12,5</TD><TD>1x
</TD></TR><TR><TD> 10µM FW Primer</TD><TD>02. Mai</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 10µM RW Primer</TD><TD>2,5</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 95ng/µL Putida genomic DNA</TD><TD>5,26 (500ng)</TD><TD>5,26 (500 ng)</TD></TR><TR><TD> Nuc. Free water</TD><TD>Apr 87</TD><TD>4,74</TD></TR></TBODY></TABLE><PRE>             ATPCS PCR Reaction</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Component</B></TD><TD align="center" style="background:#f0f0f0;"><B>25µl</B></TD><TD align="center" style="background:#f0f0f0;"><B>Final Conc.</B></TD></TR><TR><TD> 2x Q5 MM</TD><TD>12. Mai</TD><TD>1x
</TD></TR><TR><TD> 10µM FW Primer</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 10µM RW Primer</TD><TD>Jan 25</TD><TD>0,5µM
</TD></TR><TR><TD> 230ng/µL Arabidopsis cDNA</TD><TD>2.2 (500ng)</TD></TR><TR><TD> Nuc. Free water</TD><TD>08. Sep</TD></TR></TBODY></TABLE><PRE>             PCR program</PRE><PRE>             PPMT 234 bp</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>denaturing</B></TD><TD align="center" style="background:#f0f0f0;"><B>98°C</B></TD><TD align="center" style="background:#f0f0f0;"><B>120sec</B></TD></TR><TR><TD> Denaturing</TD><TD>98°C</TD><TD>10 sec
</TD></TR><TR><TD> Annealing 30x</TD></TR><TR><TD>64°C</TD><TD>30 sec
</TD></TR><TR><TD> Elongation</TD><TD>72°C</TD><TD>20 sec
</TD></TR><TR><TD> Final Elongation</TD><TD>72°C</TD><TD>2min
</TD></TR><TR><TD> Store</TD><TD>8°C</TD></TR></TBODY></TABLE><PRE>             ATPCS 1469bp
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>denaturing</B></TD><TD align="center" style="background:#f0f0f0;"><B>98°C</B></TD><TD align="center" style="background:#f0f0f0;"><B>30 sec</B></TD></TR><TR><TD> Denaturing</TD><TD>98°C</TD><TD>10 sec
</TD></TR><TR><TD> Annealing 30x</TD></TR><TR><TD>62°C</TD><TD>30 sec
</TD></TR><TR><TD> Elongation</TD><TD>72°C</TD><TD>60 sec
</TD></TR><TR><TD> Final Elongation</TD><TD>72°C</TD><TD>2min
</TD></TR><TR><TD> Store</TD><TD>8°C</TD></TR></TBODY></TABLE><P>
              10µl of PCR reaction + 2µl 6XDNA loading dye was loaded onto a 1 % agarose gel

              ATPCS (one clear thick expected band at Test of purity and quality of genomic DNA by running it through a 1% agarose gel1400 bp)

              PPMT1 (no band)

              PPMT2 (no band)&gt;
               
  
</P><PRE><B>Test of genomic DNA of P.Putida for purity and quality by running it on a 1% agarose gel</B></PRE><PRE>             - looks fine on a gel. One clear band over 10 000 bp</PRE><PRE><B>!!! Improve PPMT !!!:</B></PRE><PRE>             Try using DMSO (NEB PCR grad) and set annealing temperature down to 55°C</PRE><PRE>             Try different DMSO conc. : 0; 1; 5; 8  %
             

              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/11 Wednesday - Continuation of work from 06.06.2014: PCR for (ATPCS and) PPMT with Q5 </H2>

               

              PPMT PCR without success last weak. Therefore the temperature of the annealing step should reduce from 64°C up to 55°C in the pcr program.<B>1) corrected pcr programm for PPMT</B><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>T [°C]</B></TD><TD align="center" style="background:#f0f0f0;"><B>t [s]</B></TD></TR><TR><TD> Denaturating</TD><TD>98</TD><TD>120
</TD></TR><TR><TD> Denaturating</TD><TD>98</TD><TD>10
</TD></TR><TR><TD> Annealing</TD><TD>55</TD><TD>30
</TD></TR><TR><TD> Elongation</TD><TD>72</TD><TD>60
</TD></TR><TR><TD> Final Elongation</TD><TD>72</TD><TD>120
</TD></TR><TR><TD> Store</TD><TD>8</TD></TR></TBODY></TABLE><PRE><B>2) PPMT pcr reaction</B></PRE><PRE>             Different samples with concentrations on DMSO 0%, 1%, 5%, 8% were prepared. Use 99,97% DMSO the samples were prepared with the following pattern:
             The final volume is 25μl.</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>#1 [μl]</B></TD><TD align="center" style="background:#f0f0f0;"><B>#2 [μl]</B></TD></TR><TR><TD> Q5 MM</TD><TD>12,5</TD><TD>12,5
</TD></TR><TR><TD> 10 μl FW Primer</TD><TD>2,5</TD><TD>1,25
</TD></TR><TR><TD> 10 μl RW Primer</TD><TD>2,5</TD><TD>1,25
</TD></TR><TR><TD> Putida genomic DNA 95 ng/μl</TD><TD> 5,26</TD><TD> 5,26
</TD></TR><TR><TD> dazu 1.: DMSO,</TD><TD>0</TD><TD>0
</TD></TR><TR><TD>             nuclease free H<SUB>2</SUB>O</TD><TD>2,24</TD><TD>4,74
</TD></TR><TR><TD> dazu 2.: DMSO,</TD><TD>0,25</TD><TD>0,25
</TD></TR><TR><TD>             nuclease free H<SUB>2</SUB>O</TD><TD>1,99</TD><TD>4,49
</TD></TR><TR><TD> dazu 3.: DMSO,</TD><TD>1,25</TD><TD>1,25
</TD></TR><TR><TD>             nuclease free H<SUB>2</SUB>O</TD><TD>0,99</TD><TD>3,49
</TD></TR><TR><TD> dazu 4.: DMSO,</TD><TD>2</TD><TD>2
</TD></TR><TR><TD>             nuclease free H<SUB>2</SUB>O</TD><TD>0,24</TD><TD>2,74
</TD></TR></TBODY></TABLE><PRE><B>the 7 samples were labeled by the following schema:</B></PRE><PRE>             PPMT.1.1, PPMT.1.2, PPMT.1.3, PPMT.1.4 (Primer je 2,5μl, DMSO Konz. 0,1,5,8 %)
</PRE><PRE>             PPMT.2.2, PPMT.2.3., PPMT.2.4 (Primer je 1,25μl, DMSO Konz. 1,5,8 %) 
               
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/12 Thursday - PCR of PPMT </H2>
              Amplification of PPMT with DMSO
              ''(Continuation of work from 06.06.2014: PCR for PPMT with Q5''

              For Protocol please see lab journal from 11.06.2014<PRE><B>following labeling for 8 samples was used:</B></PRE><PRE>             PPMT.1.1, PPMT.1.2, PPMT.1.3, PPMT.1.4 (Primer je 2,5μl, DMSO Konz. 0,1,5,8 %)
</PRE><PRE>             PPMT 2.1, PPMT.2.2, PPMT.2.3., PPMT.2.4 (Primer je 1,25μl, DMSO Konz. 0,1,5,8 %)
</PRE><PRE><EM>Gel Electrophoresis</EM></PRE><PRE>             1µl loading buffer+5 µl sample  loaded on 1% agarose gel; 90V</PRE><PRE>             (rest of sample in iGEM freezer)
</PRE><PRE><STRONG>Results</STRONG></PRE><PRE>             contamination from water (band 3000bp for every sample)</PRE><PRE>             with higher DMSO conc. (5% and 8%) bands from visible
 
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/17 Tuesday - PCR Amplification of ATPCS from cDNA </H2><EM>PCR-pipetting scheme</EM><TABLE class="wikitable"><TBODY><TR><TH> Volume [µl] </TH><TH> Chemicals
</TH></TR><TR><TD> 12,5 </TD><TD> 2x Q5 Mastermix
</TD></TR><TR><TD> 1,25 </TD><TD> 10µM fwd Primer
</TD></TR><TR><TD> 1,25 </TD><TD> 10µM rev Primer
</TD></TR><TR><TD> 2,2 </TD><TD> Arabinopsis Thaliana cDNA
</TD></TR><TR><TD> 8,9 </TD><TD> nucl.free H<SUB>2</SUB>O
</TD></TR><TR><TD> 26,1µl total volume
</TD></TR></TBODY></TABLE><PRE><EM>PCR-program</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> PCR-step </TH><TH> Temperature [°C] </TH><TH> Time [sec]
</TH></TR><TR><TD> Denaturation </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD>&quot;30 cycles start&quot;
</TD></TR><TR><TD> Denaturation </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> Annealing </TD><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> Elongation </TD><TD> 72 </TD><TD> 60
</TD></TR><TR><TD>&quot;30 cycles stop&quot;
</TD></TR><TR><TD> Final Elongation </TD><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> Store </TD><TD> 8 </TD></TR></TBODY></TABLE><PRE><STRONG>PCR Purification of ATPCS</STRONG>
              - Kit Purification
              - Elutionbuffer: 30µl

              - Final concentration: 69 µg/ml
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/18 Wednesday - PCR of drag outs to separate cells for FieF Knockout  </H2>
  
             Pick up 7 single colonies + 20µl d. H<SUB>2</SUB>O each (on ice) 
             --&gt; Labeling: K1, K2, K3, K4, K5, K6, K7 
             PCR for each colony with Primer combinations: C1-C3, C2-C4, C1-C5 --&gt; 7x3= 21 Samples<PRE><EM>PCR Assay</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> Amount [µl] </TH><TH> Chemical
</TH></TR><TR><TD> 1 </TD><TD> Sample (Colony)
</TD></TR><TR><TD> 13,5 </TD><TD> d. H<SUB>2</SUB>O
</TD></TR><TR><TD> 2 </TD><TD> 10x Dream Taq Polymerasebuffer
</TD></TR><TR><TD> 1,2 </TD><TD> 25mM MgCl<SUB>2</SUB></TD></TR><TR><TD> 0,5 </TD><TD> 10µM fwd Primer
</TD></TR><TR><TD> 0,5 </TD><TD> 10µM rev Primer
</TD></TR><TR><TD> 0,5 </TD><TD> 10mM dNTPs
</TD></TR><TR><TD> 0,6 </TD><TD> Taq DNA Polymerase
</TD></TR></TBODY></TABLE><PRE><EM>PCR Program</EM>
             Colony-PCR: Elongation Time : 1,2 Min, Annealing Temp.: 55°C <EM>Gel Electrophoresis<EM><STRONG>Result</STRONG>
              RV-strain is not appropriate for Fief-Knockout. Unknown genome leads to unspecific binding of designed primers, too many bonds in gel electrophoresis picture.
              </EM></EM></PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/19 Thursday </H2>

              Continue working with colonies: K1, K5, K6, K7 from 18.06.2014
  
             PCR-programme:
<TABLE class="wikitable"><TBODY><TR><TH> Amount [µl] </TH><TH> Chemical
</TH></TR><TR><TD> 1 </TD><TD> Sample (Colony)
</TD></TR><TR><TD> 13,5 </TD><TD> d. H<SUB>2</SUB>O
</TD></TR><TR><TD> 2 </TD><TD> 10x Dream Taq Polymerasebuffer
</TD></TR><TR><TD> 1,2 </TD><TD> 25mM MgCl<SUB>2</SUB></TD></TR><TR><TD> 0,5 </TD><TD> 10µM fwd Primer C1
</TD></TR><TR><TD> 0,5 </TD><TD> 10µM rev Primer C4
</TD></TR><TR><TD> 0,5 </TD><TD> 10mM dNTPs
</TD></TR><TR><TD> 0,6 </TD><TD> Taq DNA Polymerase
</TD></TR></TBODY></TABLE><PRE>             PCR-program: 
             Elongation time: 2Min, Annealing Temperature: 56°C</PRE><PRE><STRONG> Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 1 of 4 </STRONG></PRE><PRE><B>Aim</B>: 
             Test the possibility of Magnetization of E.coli cells by the expression of human Ferritin:
</PRE><PRE><B>Procedure</B>:
</PRE><PRE>             To evaluate the effect of Ferritin  on the Magnetization different cultures were compared:</PRE><PRE>             For Both RV308 and Nissle
</PRE><TABLE class="wikitable"><TBODY><TR><TH>Culture </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4
</TH></TR><TR><TD>Plasmid </TD><TD> +</TD><TD> -</TD><TD> +</TD><TD> +
</TD></TR><TR><TD>iron </TD><TD> +</TD><TD> +</TD><TD> -</TD><TD> +
</TD></TR><TR><TD>induction </TD><TD> +</TD><TD> +</TD><TD> +</TD><TD> -
</TD></TR></TBODY></TABLE><PRE>             - Transform the Human Ferritin gen carried on the Plasmid PC514 provided from (iGEM Team of Calgary university, Canada) to E. coli strains
             (Nissle and  RV308) via electroporation, plate on (Amp) plates for selection.
 
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/20 Friday Restriction digest of pQE_80L and ATPCS-PCR fragment </H2><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>1x</B></TD><TD align="center" style="background:#f0f0f0;"><B>5x Mastermix</B></TD><TD align="center" style="background:#f0f0f0;"><B>pQE_80L</B></TD></TR><TR><TD> H2O nucfree</TD><TD>23,1µl</TD><TD>115,5µl</TD><TD>2µl
</TD></TR><TR><TD> 10x green FD buffer</TD><TD>2µl</TD><TD>10µl</TD><TD>2µl
</TD></TR><TR><TD> DNA</TD><TD>2,89µl</TD><TD>14,5µl</TD><TD>14µl
</TD></TR><TR><TD> BamHI</TD><TD>1µl</TD><TD>0µl</TD><TD>1µl
</TD></TR><TR><TD> SacI</TD><TD>1µl</TD><TD>0µl</TD><TD>1µl
</TD></TR></TBODY></TABLE><PRE>             5x Mastermix = Mastermix for ATPCS </PRE><PRE>             Enzymes were added to each single aliquot and not into the Mastermix</PRE><PRE><B>Incubation:</B> for 1,5 h at 37 °C</PRE><PRE><B>Deactivation:</B> 80°C for 5 min</PRE><PRE><I>I used the wrong restriction enzymes as I got confused with the PCR fragment. Digestion with SacI and HindIII is needed not  BamHI.</I><I>Repeat PCR for ATPCS as well as restriction digest!</I></PRE><PRE><STRONG>Test expression of Ferritin in RV308 (pSB1C3_Ferrtin) and Nissle (pSB1C3_Ferrtin)</STRONG></PRE><PRE>             5ml overnight culture used as an inoculum.</PRE><PRE>             4 ml culture were diluted to 40 ml with LB and grown for 1 h at 37°C until an OD600 of 0.7 and 0.8 was reached.</PRE><PRE>             Non-induced SDS sample was taken and induced with 1 mM IPTG</PRE><PRE>             After expression for 3 h induced SDS sample was taken.
             <EM>SDS PAGE</EM>
             Ferritin band expected at 21kDa<B>Key:</B>
             - 1st band: Nissle (non induced)
             - 2nd band: RV308 (non induced)
             - 3rd band: Ladder
             - 4th band: Nissle (induced)
             - 5th band: RV308 (induced)
</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/25 Wednesday - Amplification of ATPCS (cDNA) </H2><EM>PCR-pipetting scheme</EM><TABLE class="wikitable"><TBODY><TR><TH> Volume [µl] </TH><TH> Chemicals
</TH></TR><TR><TD> 12,5 </TD><TD> 2x Q5 Mastermix
</TD></TR><TR><TD> 1,25 </TD><TD> 10µM fwd Primer
</TD></TR><TR><TD> 1,25 </TD><TD> 10µM rev Primer
</TD></TR><TR><TD> 1,5 </TD><TD> Arabinopsis Thaliana cDNA
</TD></TR><TR><TD> 8,9 </TD><TD> nucl.free H<SUB>2</SUB>O
</TD></TR><TR><TD> 25,4µl total volume
</TD></TR></TBODY></TABLE><PRE><EM>PCR-program</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> PCR-step </TH><TH> Temperature [°C] </TH><TH> Time [sec]
</TH></TR><TR><TD> Denaturation </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD>&quot;30 cycles start&quot;
</TD></TR><TR><TD> Denaturation </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> Annealing </TD><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> Elongation </TD><TD> 72 </TD><TD> 60
</TD></TR><TR><TD>&quot;30 cycles stop&quot;
</TD></TR><TR><TD> Final Elongation </TD><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> Store </TD><TD> 8 </TD></TR></TBODY></TABLE><PRE><EM>PCR Purification</EM>
             -Purification Kit </PRE><PRE>             →DNA concentration measurement showed, that the Absorbance ratio of 260nm:280nm is quite low, which means that targeted DNA got contaminated.
</PRE><PRE><STRONG><B>Preparation of pre-culture</B></STRONG><B>Strains:</B>
             - e.coli nissle
             - e.coli nissle + ferritin
             - RV308
             - RV308 + ferritin</PRE><PRE><EM>Assay</EM>
             - 5ml LB media + 1 picked clone of the pre-day
             - Addition of 5µl CN37 to the strains with ferritin</PRE><PRE><B>Incubation:</B> 37°C; 170rpm; o/n
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/27 Friday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 2 of 4 </STRONG>

              - Preculture of 1 positive clone over night ( preculture: 7 ml LB+ 7µL Amp + 1 Clone)
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/28 Saturday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 3 of 4 </STRONG><PRE>             - Transfer the Precultures (OD= 0.4-0.5) to 25, 50 mL flasks (half filled with LB) and cultivate them at 37°C for 3 hours
             RV308 (+,+,+) &amp; (-,+,+) And Nissle (+,+,+) were cultivated in 50 mL flasks</PRE><PRE>              Strain:  OD
              RV308    0,78
              Nissle   0,86</PRE><PRE>             - Induction with (20µl (50 ML flasks)/10µl (25 ML flasks)) IPTG (concentration 1 M).
             - Add FeCl2 and MnCl2 (100µL in 50ml flasks/ 50µl in 25ml flasks) Incubation over night at 37C°</PRE><PRE>              Strain:  OD
              RV308    2.56
              Nissle   2.8</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/29 Sunday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 1st Experiment part 4 of 4 </STRONG>

              -Sampling for testing the Ferritin content via SDS-Page, different ODs were taken in consideration! 

              -Prepare the sample for SDS-gel by centrifugation and discard the supernatant then add (60µl Dis.water + 15 SDS+ MH-EtoH), boil for 10 min at 94°C.
               
              - Test the Magnetization using an permanent magnet under light microscope. <PRE><B>Evaluation</B>:
</PRE><PRE><B>SDS Page</B>:  was not clear! 
</PRE><PRE><B>Magnetization :</B></PRE><PRE><B>RV308</B></PRE><TABLE class="wikitable"><TBODY><TR><TH>Culture </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4
</TH></TR><TR><TD>Plasimd   </TD><TD> +</TD><TD>-</TD><TD> +</TD><TD> +
</TD></TR><TR><TD>iron      </TD><TD> +</TD><TD> +</TD><TD> -</TD><TD> +
</TD></TR><TR><TD>induction </TD><TD> +</TD><TD> +</TD><TD> +</TD><TD> -
</TD></TR><TABLE class="wikitable"><TBODY><TR><TH>Culture </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4
</TH></TR><TR><TD>Plasimd   </TD><TD> +</TD><TD> -</TD><TD> +</TD><TD> +
</TD></TR><TR><TD>iron      </TD><TD> +</TD><TD> +</TD><TD> -</TD><TD> +
</TD></TR><TR><TD>induction </TD><TD> +</TD><TD> +</TD><TD> +</TD><TD> -
</TD></TR><B>Nissle</B>
               
              In samples with Iron aggregates were observed with strong Mag.effect.
              <DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 06/30 Monday - Colony PCR and Ligation ATPCS </H2><EM>PCR pipetting scheme</EM><TABLE class="wikitable"><TBODY><TR><TH> Volume [µl] </TH><TH> Chemicals
</TH></TR><TR><TD> 13,7 </TD><TD> nucl.free H<SUB>2</SUB>O
</TD></TR><TR><TD> 2 </TD><TD> 10x Dream Taq Buffer
</TD></TR><TR><TD> 1,2 </TD><TD> 25mM MgCl<SUB>2</SUB></TD></TR><TR><TD> 0,5 </TD><TD> 10mM dNTPs
</TD></TR><TR><TD> 0,5 </TD><TD> 10µM fwd Primer
</TD></TR><TR><TD> 0,5 </TD><TD> 10µM rev Primer
</TD></TR><TR><TD> 1 </TD><TD> Sample (Colony)
</TD></TR><TR><TD> 0,6 </TD><TD> Dream Taq DNA Polymerase
</TD></TR><TR><TD>20µl Total Volume
</TD></TR></TBODY></TABLE><PRE><EM>PCR program</EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> PCR-step </TH><TH> Temperature [°C] </TH><TH> Time [sec]
</TH></TR><TR><TD> Denaturating </TD><TD> 95 </TD><TD> 180
</TD></TR><TR><TD><I>30 cycles start</I></TD></TR><TR><TD> Denaturating </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD>Annealing </TD><TD> 55 </TD><TD> 30
</TD></TR><TR><TD>Elongation</TD><TD> 72 </TD><TD> 120
</TD></TR><TR><TD><I> 30cycles stop</I></TD></TR><TR><TD>Final Elongation </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> Store </TD><TD> 12 </TD></TR></TBODY></TABLE><PRE><EM>Gel Electrophoresis</EM>
             1% Agarose gel with EtBr; 8µl sample; 5µl Ladder
             &quot;Key:&quot;
             First band= Ladder
              
              Each other band is a different picked clone (10 clones were picked)</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/01 Tuesday - ATPCS Ligation </H2>

              The ligation of ATPCS was checked with 1% Agarose gel (stained with ethidiumbromid) after colony pcr.

              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/02 Wednesday - Knockout of Fief and Fur in RV308 and Nissle </H2>

              To prepare the knockout 5 µl CM solution (pFerritin with canamycin resistance cassette)and 500 µl culture solution were added to 5 ml LB media. For each stain two samples were determined (RV1/2 and N1/2). The samples were incubated at 37°C up to an OD<SUB>600</SUB> of 0.53-0.55. 
              After incubation 1 ml of each culture was transferred in sterile eppi and washed for 7 min at 7000 rpm and 4°C. The supernatant was discarded, the pellet was resuspended in 1 ml sterile H<SUB>2</SUB>O and washed for 7 min at 7000 rpm and 4°C and discarded the supernatant. 

              The pellet of N1 was resuspended in 25 µl pRFP (c = 41 ng/µl) and 25 µl sterile H<SUB>2</SUB>O. The pellets of N2 and RV1/2 were resuspended in 2 µl pRFP and 50 µl sterile H<SUB>2</SUB>O. 50 µl of each suspension were transferred in electroporation cuvette and electroporated for a few sec (table below). Directly after this, 1 ml fresh LB media (37°C) was added, the suspension was transferred in a sterile eppi and incubated at 37°C at 600 rpm. The cultures were recovered, deleted on CM+Amp plates and incubated at 37°C ove night.
 
<TABLE class="wikitable"><TBODY><TR><TH> Überschrift </TH><TH> [kV] </TH><TH> [ms]
</TH></TR><TR><TD> N1 </TD><TD> 1,8 </TD><TD> 2,3
</TD></TR><TR><TD> N2 </TD><TD> 1,2 </TD><TD> 5,4
</TD></TR><TR><TD> RV1 </TD><TD> 1,2 </TD><TD> 5,3
</TD></TR><TR><TD> RV2 </TD><TD> 1,5 </TD><TD> 5,3
</TD></TR></TBODY></TABLE><PRE><STRONG>Transformation of stains RV, Nissle, WM</STRONG></PRE><PRE>             The stains RV308[ferritin], Nissle[ferritin], RV308, Nissle and WM110 were transformed with RFP. RV308[ferritin] and Nissle[ferritin] were deleted on CM+Amp plates and 5 ml culture was added with CM/Amp and incubated at 37°C. The transformed strains RV308, Nissle and WM110 were deleted on Amp plates and 5 ml culture was added with Amp and incubated at 37°C.
</PRE><PRE><STRONG>Results of transformation</STRONG></PRE><TABLE class="wikitable"><TBODY><TR><TH> stain </TH><TH> colonies on plate </TH><TH> cells in culture
</TH></TR><TR><TD> Nissle1 + RFP </TD><TD> 3 </TD><TD> Yes
</TD></TR><TR><TD> Nissle2 + RFP </TD><TD> 19 </TD><TD> Yes
</TD></TR><TR><TD> WM110 1 + RFP </TD><TD> 0 </TD><TD> Yes
</TD></TR><TR><TD> WM110 2 + RFP </TD><TD> n.d. </TD><TD> Yes
</TD></TR><TR><TD> RV308 1 + RFP </TD><TD> 0 </TD><TD> Yes
</TD></TR><TR><TD> RV308 2 + RFP </TD><TD> 87 </TD><TD> Yes
</TD></TR><TR><TD> Nissle[ferritin]1 + RFP </TD><TD> 10<SUP>8</SUP></TD><TD> No
</TD></TR><TR><TD> Nissle[ferritin]2 + RFP </TD><TD> 0 </TD><TD> No
</TD></TR><TR><TD> RV308[ferritin] 1 + RFP </TD><TD> 330 </TD><TD> Yes
</TD></TR><TR><TD> RV308[ferritin] 2 + RFP </TD><TD> 300 </TD><TD> Yes
</TD></TR></TBODY></TABLE><P> 
              The cultures of WM110 1 + RFP and WM110 2 + RFP were deleted on Amp plates and incubated at 37°C. Moreover 5 ml precultures from RV308 + RFP + Amp and RV308[ferritin] + RFP + CM+Amp were prepared.
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/03 Thursday - PCR amplification of knockout cassette FUR/FieF </H2>             The pcr preparation was performed at the pattern in the table below.<TABLE class="wikitable"><TBODY><TR><TH> pcr preparation </TH><TH> FUR (25µl) </TH><TH> FieF (50µl)
</TH></TR><TR><TD> Os High Fedility 2x Mastermix </TD><TD> 12,5 µl </TD><TD> 25 µl
</TD></TR><TR><TD> forward primer 10mM </TD><TD> 1,25 µl </TD><TD> 2 µl
</TD></TR><TR><TD> reverse primer 10mM </TD><TD> 1,25 µl </TD><TD> 2,5 µl
</TD></TR><TR><TD> pkD4 plasmid DNA 1 ng/µl </TD><TD> 1 µl </TD><TD> 1 µl
</TD></TR><TR><TD> nuclease-free H<SUB>2</SUB>O </TD><TD> to 25 µl </TD><TD> to 50 µl
</TD></TR></TBODY></TABLE><PRE>             The amplification was performed with the program in the table below.</PRE><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> T [°C] </TH><TH> t [sec]
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD> 5 cycles </TD><TD> 98/ 63/ 72 </TD><TD> 8/ 25/ 45
</TD></TR><TR><TD> 25 cycles </TD><TD> 98/ 72 </TD><TD> 8/ 70
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 120
</TD></TR></TBODY></TABLE><PRE>             --&gt; hold at 8°C
</PRE><PRE>             The amplification results were analyzed in agarose gelelectrophoresis (1% agarose; 0,5µl Gel Red; 10x diluted ladder; 6x loading dye).
</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/04 Friday - Expression von RV308 (RFP) und RV308 (RFP + Ferritin) </H2>

               

              50 ml LB + 50 µl Amp inoculated with 5 ml overnight culture of RV308 (RFP)

               

              50 ml LB + 50 µl Amp/Cm inoculated with 5 ml overnight culture of RV308 (RFP + Ferritin)

               

              After a OD of 0.6-0.7 was reached 50 ml cultures were portioned into 4 sterile 100 ml flasks so that there was a final volume of 10 ml in each flask.

               

              The cultures were then cultivated the following:<TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>RV308 (RFP)</B></TD><TD align="center" style="background:#f0f0f0;"><B>RV308 (RFP + Ferritin)</B></TD></TR><TR><TD> Induced 1 mM IPTG</TD><TD>+</TD><TD>-</TD><TD>+</TD><TD>-</TD><TD>+</TD><TD>-</TD><TD>+</TD><TD>-
</TD></TR><TR><TD> Fe/Mn each 1 mM*</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-</TD><TD>+</TD><TD>+</TD><TD>-</TD><TD>-
</TD></TR></TBODY></TABLE><PRE>             *Fe/Mn were solubilized by autoclaving and 1M stock solution</PRE><P> 
               

              Although not planned IPTG and metal ion solutions were added simultaneously due to time constraints. It was planned to add metal ions 2h after induction
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/07 Monday - Preculture prepared of: </H2>
               

              1. ATPCS Klon 1 - 10 (Amp)  --&gt; check if anything grew and contact Johann

              2. WM110 (RFP), RV308 (RFP), Nissle (RFP), RV308 (Ferritin + RFP) --&gt; make Cryostock for each

               

              Overgrown WM110 (RFP) plate streaked out again, please take out and put in cool room

               

              Rune:

              PCR of PPMT and ATPCS

      
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/09 Wednesday - PCR for  PPMT with goTaq (Promega) </H2>


              25 mikrol reaction:

              12,5 mikrol goTaqMastermix

              1,25 mikrol fw Primer 10mikrom

              1,25 mikrol b Primer 10mikrom

              1,0 mikrol DNA Template PPMT 

              9,0 mikrol nuclease free Wasser

              '''with following PCR program parameters:'''

              Denaturation  98°C  10 min
              Denaturation loop 98°C  1 min
              Annealing Temp loop 56°C  1 min
              Elongation Time loop  72°C  1 min
              Final Elongation  72°C  5 min
              Storage 8°C infinite<STRONG> Miniprep </STRONG>
              Miniprep of ATPCS clone number 3 = 94ng/µl and sent for Sequencing
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/17 Thursday - Midiprep </H2>
              #pQE_80L = 183 ng/µl
              #pMA-T_PPMT =321ng/µl
              #pKD46 = 218ng/µl<STRONG>Digestion</STRONG>
              Digestion pQE_80L with HindIII and SacI
              deactivated at 80°C for 10 mins
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/24 Thursday - miniPrep + restriction PPMT und iGEM plasmids </H2>

              Plasmide:

              1. pBADex-mYFP Venus (Amp)

              2. pEx-HISII (Amp)

              3. pJS418_phagemid(dummy) (Cm)

               

              Restriction-enzymes: XbaI | PstI (used SpeI instead :X)

              kit-miniprep

              1. 220ng/µl   2. 130ng/µl   3. 430ng/µl

              restriction:

              500 ng DNA + 1µl XbaI + 1µl PstI + 5µl 10x Buffer + -&gt;50µl Wasser (x3) [ppmt x4]

              1. 2,3µl 2. 3,8µl 3. 1,2 µl PPMT 1,6µl

               

              2nd try w/ right enzyme:

              1,5 µg DNA + 0.5 µl enzymes + 3µl BUffer + -&gt; 30 µl Wasser

              1. 6,5µl 2. 11 µl 3. 3,5 µl PPMT 4,5 µl (x3)

               

              Fragments: 1. 4043/700 2. 4450/55 3. 3800/861 PPMT 2400/251

              Gel1 L|3| 1 |1|2|2|2|P|3|3   Gel2 L|P|P|P

               
              Neu: Gel 1|2|3|L|P|P|P

              GelEx QiagenKit -&gt; extrakte in grüner box im freezer

              Ligation 2 + P

              10µl Ansatz: 1µl BUffer + 0.5 µl Ligase + 1.4 µl PPMT + 7.14 µl pEX-HisII [78ng]
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/25 Friday </H2>
              1. Ordered Primer for cloning hu_Ferritin into pQE80L

              2.) Ligation of PPMT Fragment from pMAC_PPMT into pEX_HisII

               

              pMAC_PPMT and pEX_HisII were digested with HF PstI and HF XbaI (NEB)

              PPMT (Insert)  and cut pEX_HisII (Vector DNA) were purified via an Gel extraction

               

              Ligation was performed using NEB T4 DNA Ligase in molar ratios of (Insert:Vector) and 40 ng Vector were used.

              0:1    2:1      3:1    5:1<TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Ratio</B></TD><TD align="center" style="background:#f0f0f0;"><B>02:01</B></TD><TD align="center" style="background:#f0f0f0;"><B>03:01</B></TD><TD align="center" style="background:#f0f0f0;"><B>05:01</B></TD></TR><TR><TD> Amount Insert</TD><TD>4,04 ng</TD><TD>6,06 ng</TD><TD>10,1 ng
</TD></TR></TBODY></TABLE><P> 
              Ligase was deactivated and ligation transformed after weekend
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/28 Monday </H2>

              1. Degradation test of prepped TB-Expression Plasmids by running samples on agarose gel   pBADex_MYFP_Venus  (no observable degradation)

              pEX_HisII (no observable degradation)

              pJS418_Phagemid (Dummy) (no observable degradation)

              Data:

              Fabian : 28072014_1

              2.Transformation of 0.2 µl pEX_His_PPMT ligation into DH5alpha, DH10b, and MG1655 via electroporation

              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/29 Tuesday </H2>

              1.) Transformation worked and we got colonies for pEX_HisII_PPMT

              Colony PCR was performed (Sascha)

              See Standard Protocol

               

              2.) Ligation of ATPCS PCR Fragment into pQE_80L

               

              ATPCS PCR fragment and PQE80L Vector were digested with FD HindIII and FD SacI (ThermoScientific)

              ATPCS was purified via PCR purification and Vector DNA was purified via an Gel extraction

              Vector DNA was dephosphorylized using Fast-AP and deactivated

              Ligation was performed using NEB T4 DNA Ligase in molar ratios of (Insert:Vector) and 40 ng Vector were used.

              0:1    2:1      3:1    5:1
 
<TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Ratio</B></TD><TD align="center" style="background:#f0f0f0;"><B>02:01</B></TD><TD align="center" style="background:#f0f0f0;"><B>03:01</B></TD><TD align="center" style="background:#f0f0f0;"><B>05:01</B></TD></TR><TR><TD> Amount Insert</TD><TD>24,96 ng</TD><TD>37,41 ng</TD><TD>62,36 ng
</TD></TR></TBODY></TABLE><PRE>             Ligase was deactivated and ligation transformed next day
</PRE><PRE>&lt;img src=&quot;&quot; alt=&quot;&quot; /&gt;</PRE><PRE>             #pMAC_PPMT*
             #pQE_80L*
             #pSB1c3
             #pKD46*
</PRE><P> 
              (*)Bands at 20 kbp for Promega prepped Plasmids.
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/30 Wednesday</H2><STRONG> Results Colony PCR for pEX-HisII-PPMT clones </STRONG>
       

              preculture 4 and 5 in incubator for Miniprep and Sequencing<PRE><STRONG> Transformation </STRONG><EM>Chemical competent cells XL Blue </EM>
             * 3 µl for each Ligation Assay
             * 10 min incubation on ice
             *60 s Heat schock
             * 2 min incubation on ice
             *950µl LB added
             *1h --&gt; 37 °C for Recovery
             *platted on Amp-plates
             <EM> Eporation </EM>
             *0.2 µl Ligation assay in competent DH5α [iGEM]
             *1.7 kV
             * 1h; 37°C
             * platted on Amp plates
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 07/31 Thursday </H2>
              Miniprep for sequencing of pEx_His_ppmt in DH10B? 90-100 ng/µl

              Cryostock &amp; miniprep of pjs418 in DH5a 518ng/µl

              PCR of 7 clones with pQe80l_ATPCS

              16 µl water

              2 green dreamtaq buffer

              0.5 dNTP

              0.5 primer (each)

              colony

              0.6 dream taq

               

              95° 3min

              95° 30s x30

              55° 30s x30

              72° 2min x30

              72° 10min
 
    
              Colony PCR did not work--&gt; no positive clones
             concentration after MiniPrep
             * pJS418 = 518 ng/µl
             * Clones 4 = 90 ng/µl
             * Clones 5= 105 ng/µl
              
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/06 Wednesday </H2>

              PCR
 
<TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>BamHI_PPMT_GS</B></TD><TD align="center" style="background:#f0f0f0;"><B>GS_ATPCS_HindIII</B></TD><TD align="center" style="background:#f0f0f0;"><B>BamHI_HuFerritin_HindIII</B></TD></TR><TR><TD> Q5 2x Mastermix</TD><TD>12,5</TD><TD>12,5</TD><TD>25
</TD></TR><TR><TD> 10  µM Primer fw</TD><TD>1,25</TD><TD>1,25</TD><TD>1,25
</TD></TR><TR><TD> 10  µM Primer rev</TD><TD>1,25</TD><TD>1,25</TD><TD>1,25
</TD></TR><TR><TD> template (1ng)</TD><TD> 0,30</TD><TD> 0,5</TD><TD> 1
</TD></TR><TR><TD> nucfree H2O</TD><TD> 9,7</TD><TD> 9,5</TD><TD> 19
</TD></TR></TBODY></TABLE><PRE>             Program:
</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>BamHI_PPMT_GS</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>GS_ATPCS_HindIII</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>BamHI_HuFerritin_HindIII</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> 98</TD><TD>30</TD><TD>98</TD><TD>30</TD><TD>98</TD><TD>30
</TD></TR><TR><TD> 98</TD><TD>10</TD><TD>98</TD><TD>10</TD><TD>98</TD><TD>10
</TD></TR><TR><TD> 58</TD><TD>30</TD><TD>56,3</TD><TD>30</TD><TD>58,6</TD><TD>30
</TD></TR><TR><TD> 72</TD><TD>12</TD><TD>72</TD><TD>60</TD><TD>72</TD><TD>45
</TD></TR><TR><TD> 72</TD><TD>2'</TD><TD>72</TD><TD>2'</TD><TD>72</TD><TD>2'
</TD></TR><TR><TD> 8</TD><TD>end</TD><TD>8</TD><TD>end</TD><TD>8</TD><TD>end
</TD></TR></TBODY></TABLE><P> 
               Analytical Gel
              Expected Bands clearly seen for PPMT and Ferritin. Only little ATPCS band shows Primer Dimer inhibit PCR. Anyhow we will try a Gel extraction and after a Assembly PCR


              <STRONG> Generation of Heme-free BFR by Site-directed Mutagenesis, part 1 of 5 </STRONG><STRONG>Site-directed mutagenesis was performed by the QuickChange method </STRONG>

              Primer design with http://www.bioinformatics.org/primerx/

              methionine (on position 52) was substituted by histidine

              BFR M52H
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/07 Thursday</H2>

              Analyzing 2 colonies of pQE80L_ATPCS whether ATPCS was inserted by sequencing revealed no proper results.

              Tested by colony PCR (see protocols) using 55°C Annealing temp and 2' Elongation time (expected bands should be 2000bps).

              Gel showed PCR was about 500 bp big -&gt; ATPCS NOT INSERTED.

              Outcome:BamHI_PPMT_GS    super
              GS_ATPCS_HindIII   bad
              BamHI_Ferritin    super

              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/08 Friday - Gradienten PCR GS_ATPCS_HindIII </H2><PRE><B>PCR</B></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>3x GS_ATPCS_HindIII</B></TD></TR><TR><TD> Q5 2x Mastermix</TD><TD>12,5
</TD></TR><TR><TD> 10  µM Primer fw</TD><TD>1,25
</TD></TR><TR><TD> 10  µM Primer rev</TD><TD>1,25
</TD></TR><TR><TD> template (1ng)</TD><TD> 0,5
</TD></TR><TR><TD> nucfree H2O</TD><TD> 9,5
</TD></TR></TBODY></TABLE><PRE><B>Program:</B></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>GS_ATPCS_HindIII</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> 98°C</TD><TD>30</TD></TR><TR><TD> 98°C</TD><TD>10</TD></TR><TR><TD> 56,3-62°C</TD><TD>30</TD><TD>34</TD></TR><TR><TD> 72°C</TD><TD>60</TD></TR><TR><TD> 72°C</TD><TD>2'</TD></TR><TR><TD> 8°C</TD><TD>end</TD></TR></TBODY></TABLE><PRE>             - preparative agarose gel 1% and gel ex of GS_ATPCS_HindIII bands at ca. 1490 bp</PRE><PRE>             - accidentally also loaded BamHI_PPMT_GS onto Gel and was also  extracted (elution with 25 µl elution buffer).</PRE><PRE>             - BamHI_Ferritin_HindIII was PCR purified</PRE><PRE>             - DNA concentration measurement for  purified pcr fragments:</PRE><PRE>             GS_ATPCS_HindIII = 33 ng/µl</PRE><PRE>             BamHI_PPMT_GS = 48 ng/µl</PRE><PRE>             BamHI_Ferritin_HindII = 125 ng/µl</PRE><PRE><B>Assembly PCR</B></PRE><PRE>             An Assembly PCR was used to align BamHI_PPMT_GS with GS_ATPCS_HindIII and amplify the complementary structure in order to form a fusion protein coding sequence.</PRE><PRE>             Therefore, both fragments were added a templated equal molar amounts.</PRE><PRE>             Calcultation:</PRE><PRE>             bpATPCS = 1493</PRE><PRE>             bpPPMT = 255</PRE><PRE>             mATPCS/bpATPCS = mPPMT/bpPPMT</PRE><PRE>             mATPCS = (1ng * 1493) / 255</PRE><PRE>             mATPCS= 58 ng</PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;"><B>BamHI_PPMT_GS_ATPCS_HindIII</B></TD></TR><TR><TD> Q5 2x Mastermix</TD><TD>25
</TD></TR><TR><TD> 100  µM Primer fw</TD><TD>0.25 (not added till later to second PCR run)
</TD></TR><TR><TD> 100  µM Primer rev</TD><TD>0.25 (not added till later to second PCR run)
</TD></TR><TR><TD> template (1ng)</TD><TD> 1,750 µl ATPCS + 0.208 µl PPMT
</TD></TR><TR><TD> nucfree H2O</TD><TD> 18.042
</TD></TR></TBODY></TABLE><PRE><B>Program:</B></PRE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>1. PCR BamHI_PPMT_GS_ATPCS_HindIII</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> 98</TD><TD>30</TD></TR><TR><TD> 98</TD><TD>10</TD></TR><TR><TD> 68</TD><TD>30</TD><TD> 5</TD></TR><TR><TD> 72</TD><TD>55</TD></TR><TR><TD> 72</TD><TD>1'</TD></TR><TR><TD> 8</TD><TD>end</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>2. PCR BamHI_PPMT_GS_ATPCS_HindIII</B></TD><TD align="center" style="background:#f0f0f0;">'
</TD><TD align="center" style="background:#f0f0f0;">'
</TD></TR><TR><TD> 98</TD><TD>30</TD></TR><TR><TD> 98</TD><TD>10</TD></TR><TR><TD> 68</TD><TD>30</TD><TD> 30</TD></TR><TR><TD> 72</TD><TD>55</TD></TR><TR><TD> 72</TD><TD>2'</TD></TR><TR><TD> 8</TD><TD>end</TD></TR></TBODY></TABLE><PRE><EM> Transformation into DH10B </EM>
             #BFR M52H
             # BFR M52H
             *Recover in LB--&gt; 1h at 37°C
             
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/11 Monday - Plan-prepare fragments for digest </H2>
              Constructs aimed for
              
             *In pQE_80L (Amp)     BamHI_PPMT_GS_ATPCS_Hind III  (PCR Construct available)
             Construct Size 1724 of 4731
<PRE>             *In pQE_80L (Amp)    BamHI_FTH_FTL_Hind III   (PCR Construct available)
             Construct Size 1077 of 4731
</PRE><PRE>             *In pQE_80L (Amp)   Sacl_ATPCS_Hind III    (PCR Construct not available)
             Construct Size 1400 of 4731
</PRE><PRE>             *In pJS418 (CM)    XbaI_PPMT_PstI       (PCR Construct available)
             Construct Size 255 of 3800
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Q5 MasterMix </TH><TH> 25μl
</TH></TR><TR><TD> ATPCS SacI </TD><TD> 2.5
</TD></TR><TR><TD> ATPCS Hind III</TD><TD> 2.5
</TD></TR><TR><TD> cDNA At. </TD><TD> 0.5
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 19.5
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> PCR Program </TH></TR><TR><TD> 98°C</TD><TD> 30 sec
</TD></TR><TR><TD> 98°C </TD><TD> 10 sec
</TD></TR><TR><TD> 62°C 34cycles </TD><TD> 30 sec
</TD></TR><TR><TD> 72°C </TD><TD> 60 sec
</TD></TR><TR><TD> 72°C</TD><TD> 2 mins
</TD></TR><TR><TD> 4°C</TD><TD> Store
</TD></TR></TBODY></TABLE><PRE>             PCR Fragmente PPMT_GS_ATPCS+ATPCS+Purification (Gelex)</PRE><P>
              After Gelex:
              PPMT  48ng/μl

              Ferritin 125ng/μl (Pur from PCR)

              ATPCS 33ng/μl </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/12 Tuesday - Preparing chemically competent cells </H2>
              V=100ml, 37°C
              BU36=DH10B

              OD at 600nm=0.461A 
              DH10B competent cells<PRE><I><B>Digestion</B></I></PRE><TABLE class="wikitable"><TBODY><TR><TH> Inserts PCR Pur </TH><TH> JBH1  PPMT_GS_ATPCS 1 </TH><TH> JBH2  PPMT_GS_ATPCS 2 </TH><TH> JBH3 Ferritin </TH><TH> JSH  ATPCS
</TH></TR><TR><TD> PCR Fragment  </TD><TD> 20</TD><TD> 20 </TD><TD>8</TD><TD> 20
</TD></TR><TR><TD> Buffer </TD><TD> 3 </TD><TD> 3</TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> FD BamHI /FD SacI</TD><TD> 1</TD><TD> 1</TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> FD Hind III </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> nuclease free H<SUB>2</SUB>O </TD><TD> 5 </TD><TD> 5 </TD><TD> 17</TD><TD> 5
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Vector </TH><TH> VBH pQE_80L </TH><TH> VSH pQE_80L SacI </TH><TH> 2x PJS_418 </TH><TH> 2x P_Mac_PPMT
</TH></TR><TR><TD> Vector </TD><TD> 5.5 </TD><TD> 5.5</TD><TD> 1.9</TD><TD> 3.2
</TD></TR><TR><TD> Enzyme 1</TD><TD> 1 BamHI </TD><TD> 1 SacI </TD><TD> 1 FDxBa</TD><TD> 1 FDxBa
</TD></TR><TR><TD> Enzyme 2 </TD><TD> 1 HindIII </TD><TD> 1 HindIII </TD><TD> 1 FD PST </TD><TD> 1 FD PST1
</TD></TR><TR><TD> Buffer </TD><TD> 3</TD><TD> 3 </TD><TD> 2</TD><TD> 2
</TD></TR><TR><TD> Fast AP </TD><TD> 1</TD><TD> 1 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> nuclease free H<SUB>2</SUB>O</TD><TD> 18.5</TD><TD> 18.5 </TD></TR></TBODY></TABLE><PRE>             * 35 ng/L  PPMT ATPCS 1
             * 15ng/L  PPMT ATPCS 2
             * 25ng/L  ATPCS
             * 13ng/L  P_Mac_PPMT
             * 31ng/L  PJS_418
              
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Ligation 5:1 </TH><TH> JBH1 in VBH </TH><TH> JBH2 in VBH </TH><TH> JBH3 in VBH</TH><TH> JSH in VSH </TH><TH> PPMT in PJS_418
</TH></TR><TR><TD> 10x T4 DNA ligase Buffer</TD><TD> 1μl</TD><TD> 1μl </TD><TD> 1μl </TD><TD> 1μl </TD><TD> 1μl
</TD></TR><TR><TD> Vector </TD><TD> 1.42 </TD><TD> 1.42 </TD><TD> 1.42</TD><TD> 1.33</TD><TD> 1.29
</TD></TR><TR><TD> Insert </TD><TD> 3.29 </TD><TD> 7.08 </TD><TD> 1.36 </TD><TD> 7.17 </TD><TD> 1.03
</TD></TR><TR><TD> Nuclease free H<SUB>2</SUB>O </TD><TD> 3.79 </TD><TD> 0 </TD><TD> 5.72 </TD><TD> 0 </TD><TD> 6.18
</TD></TR><TR><TD> T4 DNA ligase (NEB)</TD><TD> 0.5 </TD><TD> 0.5 </TD><TD> 0.5 </TD><TD> 0.5 </TD><TD> 0.5
</TD></TR></TBODY></TABLE><PRE>             Ampicillin  
             JBH1 in VBH
             JBH2 in VBH 
             JSH in VSH </PRE><PRE>             Chloramphenicol 
             PPMT in PJS_418
</PRE><PRE>             Transformation in DH10b (cc.c)
             10<SUP>6</SUP> Colony for every approach  </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/13 Wednesday - pQE 80L Colony PCR </H2><TABLE class="wikitable"><TBODY><TR><TH> Mastermix </TH><TH> 1x</TH><TH> 35x </TH><TH> 10x
</TH></TR><TR><TD> nuclease free H<SUB>2</SUB>O</TD><TD> 14 </TD><TD> 490 </TD><TD> 140
</TD></TR><TR><TD> 10 dreamtaq</TD><TD> 1 </TD><TD> 70</TD><TD> 20
</TD></TR><TR><TD> 25µM MgCl<SUB>2</SUB></TD><TD> 1.2 </TD><TD> 42</TD><TD> 12
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 0.5</TD><TD> 17.5 </TD><TD> 5
</TD></TR><TR><TD> 10mM Forward Primer </TD><TD> 0.5</TD><TD> 17.5</TD><TD> 5
</TD></TR><TR><TD> 10mM Reverse Primer </TD><TD> 0.5 </TD><TD> 17.5 </TD><TD> 5
</TD></TR><TR><TD> Taq</TD><TD> 0.3 </TD><TD> 10.5 </TD><TD> 3
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> PCR Programm </TH></TR><TR><TD> 95°C</TD><TD> 3 mins
</TD></TR><TR><TD> 95°C </TD><TD> 30 secs
</TD></TR><TR><TD>55°C</TD><TD> 30 secs
</TD></TR><TR><TD> 72°C</TD><TD> 60 secs
</TD></TR><TR><TD> 72°C </TD><TD> 10 min
</TD></TR><TR><TD> 8°C</TD><TD> Store
</TD></TR></TBODY></TABLE><PRE>             Samples : 
             # PPMT GS ATPCS   size 1912
             # PPMT GS ATPCS   size 1912
             # ATPCS in pQE80L   size 1770
             # Ferritin in pQE80L   size 1376
             # PPMT in PJS   size 420
</PRE><PRE>             pQE80L=300bp       
             pJS = 170bp</PRE><PRE>             Sequence clones A - H + J</PRE><PRE><STRONG> Generation of Heme-free BFR by Site-directed Mutagenesis, part 2 of 5 </STRONG></PRE><PRE>             QuickChange Site-Directed Mutagenesis</PRE><PRE>             20µL approach:
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Header </TH><TH> component
</TH></TR><TR><TD> 2 µL </TD><TD> 10x Pfu Buffer (MgSO4)
</TD></TR><TR><TD> 0,6 µL </TD><TD> 10µM forward Primer
</TD></TR><TR><TD> 0,6 µL </TD><TD> 10µM reverse Primer
</TD></TR><TR><TD> x µL </TD><TD> 30ng Template
</TD></TR><TR><TD> 0,4 µL </TD><TD> dNTPs
</TD></TR><TR><TD> 0,6 µL </TD><TD> Pfu Polymerase
</TD></TR><TR><TD> to 20 µL </TD><TD> PCR water
</TD></TR></TBODY></TABLE><PRE>             Template BFR: </PRE><PRE>             0,5 µL              A4 (66ng/µL)</PRE><PRE>             0,2 µL    D2 (144ng/µL)</PRE><TABLE class="wikitable"><TBODY><TR><TH> labeling </TH><TH> Template </TH><TH> Annealing Temp.
</TH></TR><TR><TD> 17 </TD><TD> D2 (144ng/µL) </TD><TD> 71°C
</TD></TR><TR><TD> 15 </TD><TD> D2 (144ng/µL) </TD><TD> 55°C
</TD></TR><TR><TD> 67 </TD><TD> A4 (66ng/µL) </TD><TD> 71°C
</TD></TR><TR><TD> 65 </TD><TD> A4 (66ng/µL) </TD><TD> 55°C
</TD></TR></TBODY></TABLE><PRE>             PCR Program</PRE><TABLE><TBODY><TR><TH> Temp </TH><TH> time
</TH></TR><TR><TD> 95°C   </TD><TD> 30sec
</TD></TR><TR><TD> ________ </TD><TD> ______
</TD></TR><TR><TD> 16 Cyclen </TD></TR><TR><TD> 95°C   </TD><TD>  30sec
</TD></TR><TR><TD> 55/71°C  </TD><TD> 1min
</TD></TR><TR><TD> 68°C   </TD><TD> 5min
</TD></TR><TR><TD> ________ </TD><TD> ______
</TD></TR><TR><TD> 68°C </TD><TD> 10sec
</TD></TR><TR><TD> 4°C  </TD><TD> finale
</TD></TR></TBODY></TABLE><PRE>             Σ20µL -8µL fürs Gel= 12µL +1µL DpnI 
</PRE><PRE>             37°C 90min</PRE><PRE><B>chemical Transformation (Fabian)</B></PRE><PRE>             in DH10b</PRE><P> 

              plating out on Amp plates 
            </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/15 Friday - Clone Sequencing </H2><TABLE class="wikitable"><TBODY><TR><TH> Clone </TH><TH> Name </TH><TH> Construct present </TH><TH> concentration after isolation
</TH></TR><TR><TD> A</TD><TD> pQE_80L_PPMT GS ATPCS </TD><TD> yes </TD><TD> 388ng/µL
</TD></TR><TR><TD> B</TD><TD> pQE_80L_PPMTGSATPCS </TD><TD> yes </TD><TD> 373ng/µL
</TD></TR><TR><TD> C </TD><TD> pQE_80L_PPMTGSATPCS </TD><TD> yes</TD><TD> 442ng/µL
</TD></TR><TR><TD> D </TD><TD> pQE_80L_PPMTGSATPCS </TD><TD> no</TD><TD> 389 ng/µL
</TD></TR><TR><TD> E </TD><TD> pQE_80L_huFerritin </TD><TD> yes</TD><TD> 490ng/µL
</TD></TR><TR><TD> F</TD><TD> pQE_80L_huFerritin </TD><TD> no</TD><TD> 402ng/µL
</TD></TR><TR><TD> G </TD><TD> pQE_80L_ATPCS</TD><TD> yes </TD><TD> 497ng/µL
</TD></TR><TR><TD> H </TD><TD> pQE_80L_ATPCS </TD><TD> yes  </TD><TD> 443 ng/µL
</TD></TR><TR><TD> I </TD><TD> pJS418_PPMT</TD><TD> yes </TD><TD> 596 ng/µL
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/19 Tuesday </H2><STRONG> Generation of Heme-free BFR by Site-directed Mutagenesis, part 3 of 5 </STRONG>

              ÜN culture

              5ml LB + 5µL Amp →shakingincubator
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/20 Wednesday </H2><STRONG> Generation of Heme-free BFR by Site-directed Mutagenesis, part 4 of 5 </STRONG>


              MiniPrep nach Protocol
              Photometer
              4µL Probe + 76µL dilution (water)

              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/21 Thursday </H2><STRONG> Generation of Heme-free BFR by Site-directed Mutagenesis, part 5 of 5 </STRONG>


              Sequencing:
              
              Primer PB 16 
              
              Crystocks:
              
              ÜN culture
              
              500µL culture + 500µL DMSO
              2xclon  17
              2xclon 67
</DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/22 Friday - SDS-PAGE with Coomassie staining </H2>
              Samples<PRE>             # Marker
             # Pellet ATPCS/PPMT
             #15:00 after induction ATPCS/PPMT
             #overnight ATPCS/PPMT
             #GS Pellet
             #PGSA after induction 15:00
             #GS P/A overnight
             #human Ferritin Pellet
             #human Ferritin after induction 15:00
             #human Ferritin overnight
</PRE><PRE><I><B>Analysis of Expression</B></I>
             Human Ferritin = 42 kDa (Potparam  Expasy)
             ATPCS = 56.3 kDA (Potparam  Expasy)
             PPMT = 7.9 kDA (Potparam  Expasy)
             ATPCSGSPPMT = 64.3 kDa (Potparam Expasy)
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 08/29 Friday - Calibration curve for Iron concentration measurement </H2><TABLE class="wikitable"><TBODY><TR><TH> Header </TH><TH>  Header
</TH></TR><TR><TD> 0.017A </TD><TD> 10 µg/ml
</TD></TR><TR><TD> 0.105A </TD><TD> 50 µg/ml
</TD></TR><TR><TD> 1.076A </TD><TD> 100 µg/ml
</TD></TR><TR><TD> 2.095A </TD><TD> 150 µg/ml
</TD></TR><TR><TD> 2.747A </TD><TD> 200 µg/ml
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/03 Wednesday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 1 of 8 </STRONG><B>Aim</B>: 
             Test the possibility of Magnetization of E.coli cells by the expression of human Ferritin:
<PRE><B>Procedure</B>: ...
</PRE><P> 

              -Precultures of Nissle and RV308 (wild types)
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/04 Thursday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 2 of 8 </STRONG>


              -Prepare chemical competent cells ( Nissle and RV308) . see Protocol
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/05 Friday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 3 of 8 </STRONG>

              -Chemical biotransformation (double Trafo, 200ng of each) of, red fluorescence protein carried on plasmid (kan) Human-Ferritin carried on PQE-80L (Amp) [was prepared by iGEM-Berlin using the '''Biobrick''' from Calgary, unlike does not contains polypeptide at the beginning].
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/06 Saturday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 4 of 8 </STRONG>

              -Positive colones for both RV308 and Nissle
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/11 Thursday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 5 of 8 </STRONG>
              - Precultures of the transformed colones
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/12 Friday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 6 of 8 </STRONG>

              - Transfer the precultures into 250ml filled until 100ml with LB and incubate them at 37°C 

               Strain    OD
               RV308     0.71
               Nissle    0.6


              - Induction with 100µl IPTG (1M), incubation over night at '''30°C''' and 200rpm
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/13 Saturday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 7 of 8 </STRONG>

              -samples were taken for protein analysis SDS-Page

               Strain    OD
               RV308     6.2
               Nissle    5.2

              - adding  7ml of 1M Mn-citrate (wrongly) and incubated at 4°C over weekend
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/15 Monday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, 2nd Experiment part 8 of 8 </STRONG><PRE>             -Test the Magnetization using a permanent magnet under <B>Fluorescent microscope.</B></PRE><PRE><B>Evaluation</B></PRE><PRE>             -Cells grew
             -no red fluorescence observed
             -no Magnetization observed
</PRE><PRE>             Possible Explanation:
             - Adding Mn ions instead of Fe could be effected the Magnetization 
             - The red Fluorescence gen contains a cadaverin chain, this could explain the non-fluorescence.
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/16 Tuesday </H2><STRONG> Preparing Cultures for fluorescence microscopy </STRONG>

              E. coli Nissle 1917 (pQE_80L Hu_Ferritin + RFP) 
              E. coli RV308 (pQE_80L_HuFerritin + RFP)

              Nissle OD600 = 5.08  
              RV 308 OD600 = 5.81

              take OD=1

              wash in 1000 µl PBS three times ( centrifuge at 6000g for 2min, discard supernatent and resuspend pellet in PBS)

              finally resuspend pellet in 500 µl

              Put it into a cool box and take it over to the fluorescence microscope facility.

              <STRONG> Constructing PQE_80L_T5_ATPCS_lac_PPMT </STRONG>

              Plasmid for Co-Expression of PPMT and ATPCS on one plasmid.

              Source:
              pJS418_PPMT (amplifying insert)
              pQE_80L_ATPCS (as target vector)

              <EM> PCR of HindIII_lac_PPMT_HindIII </EM>
 
             Construction of the HindIII_lac_PPMT_HindIII in pQE80L_Hind III
               
<TABLE class="wikitable"><TBODY><TR><TH> 3x HindIII_lac_PPMT_HindIII
</TH></TR><TR><TD> Q5 2x Mastermix</TD><TD> 12,5
</TD></TR><TR><TD> 10 µM Primer fw</TD><TD> 1,25
</TD></TR><TR><TD> 10 µM Primer rev</TD><TD> 1,25
</TD></TR><TR><TD> template (1ng)</TD><TD> 0,5
</TD></TR><TR><TD> nucfree H2O</TD><TD> 9,5
</TD></TR></TBODY></TABLE><PRE><EM> Program </EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> Temperature</TH><TH> Duration in seconds</TH><TH> Cycles
</TH></TR><TR><TD> 98</TD><TD> 30</TD><TD>  1</TD></TR><TR><TD> 98</TD><TD> 5</TD><TD>  loop start</TD></TR><TR><TD> 62-65</TD><TD> 15</TD><TD> loop 32</TD></TR><TR><TD> 72</TD><TD> 10</TD><TD> loop end</TD></TR><TR><TD> 72</TD><TD> 2'</TD><TD> 1
</TD></TR><TR><TD> 8</TD><TD> infinite</TD><TD> 1
</TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/17 Wednesday - Digest and Dephosphorylation of vector pQE_80L_ATPCS </H2><TABLE class="wikitable"><TBODY><TR><TH> pQE80L_ATPCS digest</TH><TH> 1x</TH><TH> 3x
</TH></TR><TR><TD> pQE_80L_ATPCS</TD><TD> 2 µl </TD><TD> 7 µl
</TD></TR><TR><TD> FD HindIII</TD><TD> 1 µl </TD><TD> 3 µl
</TD></TR><TR><TD> 10x FD Buffer</TD><TD> 2µl </TD><TD> 7 µl
</TD></TR><TR><TD> FastAP (Thermo)</TD><TD>  1 µl</TD><TD> 3 µl
</TD></TR><TR><TD> nucfree H2O</TD><TD> 14 µl </TD><TD> 51 µl
</TD></TR></TBODY></TABLE><P> 
              Gelextraction resulted in HindIII cut pQE80L_ATPCS 20 µl with 12 ng/µl
              

              In parallel the PCR fragment HindIII_lac_PPMT_HindIII was purified and is ready for digest.

              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/18 Thursday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 1 of 7 </STRONG><PRE>             '<I><B>Aim</B></I></PRE><PRE>             -Double Trafo of the Human-Ferritin and fluorescence marker.</PRE><PRE><B>Procedure:</B>…
</PRE><PRE>             Stains: RV308, Nissle and DH0B (control)
             -chemical Double-biotransformation of:
              -0.5 µl of Human-ferrtin on PQE-80L(490 ng/µl) with (Amp)
              -3 µl green fluorescence protein Nerm ASGP-R-GFP (83 ng/µl) with (Kan)</PRE><PRE>             for control:
             - 3 µl green fluorescence protein Nerm ASGP-R-GFP (83 ng/µl) with (Kan) is transformed into DH05
             - Double Trafo of (<B>pJS418_PPMT and pQE_80L_ATPCS)</B></PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/19 Friday </H2><B>PCR of BB0 - BB3 parts (Saba)</B><TABLE class="wikitable sortable"><TBODY><TR><TH> compound </TH><TH> V/µl (50µl total)
</TH></TR><TR><TD> Q5 2xMM </TD><TD> 25.0
</TD></TR><TR><TD> 10µM P_fv </TD><TD> 2.5
</TD></TR><TR><TD> 10µM P_rev </TD><TD> 2.5
</TD></TR><TR><TD> DNA (template 1ng)</TD><TD> 2.0 (of diluted plasmid)
</TD></TR><TR><TD> nuc.-free H2O </TD><TD> 28.0
</TD></TR></TBODY></TABLE><PRE><B>Primers</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> Reaction </TH><TH> fw Primer </TH><TH> rev. Primer </TH><TH> TmTm/°C </TH><TH> Template </TH><TH> bp
</TH></TR><TR><TD> BB0 </TD><TD> BBa_K1438000_fw </TD><TD> BBa_K1438000_rev </TD><TD> 62 </TD><TD> pQE80L_BFR </TD><TD> 507
</TD></TR><TR><TD> BB1 </TD><TD> BBa_K1438000_fw </TD><TD> BBa_K1438000_rev </TD><TD> 62 </TD><TD> pQE80L_BFR M52H </TD><TD> 507
</TD></TR><TR><TD> BB2 </TD><TD> BBa_K1438002_fw </TD><TD> BBa_K1438002_rev </TD><TD> 68 </TD><TD> pQE80L_Hu-Fer </TD><TD> 1000
</TD></TR></TBODY></TABLE><PRE><B>PCR Programms</B><B>BB0 &amp; BB1</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> T/°C </TH><TH> t/min </TH><TH> cycles
</TH></TR><TR><TD> 98 </TD><TD> 0:30 </TD><TD> 30
</TD></TR><TR><TD> 98 </TD><TD> 0:10 </TD><TD> 30
</TD></TR><TR><TD> 62 </TD><TD> 0:30 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 0:15 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 2:00 </TD><TD> 30
</TD></TR><TR><TD> 8 </TD><TD> infinity </TD><TD> 30
</TD></TR></TBODY></TABLE><PRE><B>BB2</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> T/°C </TH><TH> t/min </TH><TH> cycles
</TH></TR><TR><TD> 98 </TD><TD> 0:30 </TD><TD> 30
</TD></TR><TR><TD> 98 </TD><TD> 0:10 </TD><TD> 30
</TD></TR><TR><TD> 68 </TD><TD> 0:30 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 0:40 </TD><TD> 30 (probably too long since ~1000bp and not 1408bp however worked well as seen on gel)
</TD></TR><TR><TD> 72 </TD><TD> 2:00 </TD><TD> 30
</TD></TR><TR><TD> 8 </TD><TD> infinity </TD><TD> 30
</TD></TR></TBODY></TABLE><PRE>

              Biobricks derived from pQE80L constructs
              BB1  BFR  191µg/ml
              BB2  BFRM  173µg/ml 
              BB2  HuFerritin  166 µg/ml

              <STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 2of 7 </STRONG>
              -No positive colones were detected for double-trafo expect these for control (transformation was successful in DH05).
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/20 Saturday - Digest of PSB1C3-Ferritin </H2><STRONG>Plasmid:  PSB1C3 - ferritin with Peptide   120ng/µl </STRONG><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> µL
</TH></TR><TR><TD> NEB Xba1 </TD><TD> 2 µL
</TD></TR><TR><TD> NEB Pst1 </TD><TD> 2 µL
</TD></TR><TR><TD> NEB Buffer 3.1 </TD><TD> 20 µL
</TD></TR><TR><TD> nucleasefree H<SUB>2</SUB>O </TD><TD> 151 µL
</TD></TR><TR><TD> PSB1C3 Feritin </TD><TD> 25 µL
</TD></TR><TR><TD> total volume </TD><TD> 200µL
</TD></TR></TBODY></TABLE><PRE>             37°C for 1,5h
             (inactivate for 20min at 80°C)
</PRE><PRE><STRONG>Gel electrophoresis </STRONG>
             1% Agarose-Gel 
             + 3 µL EthBr</PRE><PRE>             200µL Probe + 40 µL loading dye (6x)</PRE><PRE>             5 µL GenRuler-Mix</PRE><PRE>             no Gel-extraction, because nothing was seen on the gel, except the marker (GenRuler) </PRE><PRE><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 3 of 7 </STRONG>
             -Transformation of RF <B>(red fluorescence protein)</B> on with (Kan) into DH05
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/21 Sunday </H2><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 4 of 7 </STRONG>

              -Transformation was successful
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/22 Monday - Precultures </H2>

              Precultures of

              PSB1K3_RFP in DH10b (Kan)
              pQE80L_ATPCS &amp; pJS418_PPMT in Dh10b (Amp &amp; Cm)
              Clones with pQE80L_ATPCS'n'PPMT in DH10b (Amp)
              PSB1C3_Ferritin in DH10b (Cm)

              <STRONG> Checking Insertion of gene by colony PCR </STRONG>
              Performing PCR for clones of ATPCS-PPMT construct<TABLE class="wikitable"><TBODY><TR><TH> 1x in µl </TH><TH> 12x in µl  </TH><TH> 10x in µl
</TH></TR><TR><TD> nuclease free H<SUB>2</SUB>O</TD><TD> 14</TD><TD> 168 </TD><TD> 140
</TD></TR><TR><TD> 10x DreamTaq buffer</TD><TD> 2 </TD><TD> 24 </TD><TD> 20
</TD></TR><TR><TD> 25mM MgCl<SUB>2</SUB></TD><TD> 1.2</TD><TD> 14.4</TD><TD> 12
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 0.5</TD><TD> 6 </TD><TD> 5
</TD></TR><TR><TD> 10mM FW PrimerHindIII-lac-prom-fw </TD><TD> 0.5 </TD><TD> 6 </TD><TD> 5
</TD></TR><TR><TD> 10mM RW Primer PB17 (T7 term_rev) </TD><TD> 0.5 </TD><TD> 6 </TD><TD> 5
</TD></TR><TR><TD> Taq Polymerase </TD><TD> 0.3 </TD><TD> 3.6 </TD><TD> 3
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Programm colony PCR </TH><TH> Temp </TH><TH> Time
</TH></TR><TR><TD> Denaturing</TD><TD> 95°C</TD><TD> 7 min
</TD></TR><TR><TD> Denaturing</TD><TD> 95°C</TD><TD> 30
</TD></TR><TR><TD> Annealing </TD><TD> 69°C </TD><TD> 30
</TD></TR><TR><TD> Elongation </TD><TD> 72°C </TD><TD> 1 min
</TD></TR><TR><TD> Final Elongation </TD><TD> 72°C </TD><TD> 10 min
</TD></TR><TR><TD> Store</TD><TD> 8°C </TD></TR></TBODY></TABLE><P> 
               Dissolved colonies 4 and 6 which showed band at 400 bp were grown each in a cultivation tube with LB+Amp100 at 37°C
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/23 Tuesday - Mutagenesis of ATPCS </H2>

              The elimination of the XbaI restriction site of all our ATPCS containing constructs was achieved by quick change mutagenesis.<PRE>             ... Details Saba....
</PRE><PRE>             After digest with DpnI the reaction was transformed into competent DH10b E. coli cells via heat shock transformation.
             After streaking out all aliquots on LB-Amp Agar plates, they were incubated at 37 °C over night.</PRE><PRE><STRONG> Digest of miniprepped PSB1C3_Ferritin (Calgary) for extraction of vector for BioBrick preparation </STRONG></PRE><TABLE class="wikitable"><TBODY><TR><TH> Components </TH><TH> Volume in µl
</TH></TR><TR><TD> NEB XbaI</TD><TD> 2
</TD></TR><TR><TD> NEB PstI </TD><TD> 1,5
</TD></TR><TR><TD> NEB Buffer 3.1 </TD><TD> 5
</TD></TR><TR><TD> nucfree H2O </TD><TD> 26,5
</TD></TR><TR><TD> plasmid PSB1C3_ferritin (Calgary) </TD><TD> 15
</TD></TR><TR><TD> 50 µl reaction
</TD></TR></TBODY></TABLE><PRE>
             The reaction was incubated at 37 °C for 1,5 h.</PRE><PRE>             The digested vector was purified using a 1% agarose gel and the Roth GelNebulizer purification kit. The vector band was expected at 2000 bp and the insert band at about 1000 bp.</PRE><PRE><B>both colonies from Saba were grown --&gt; Mini-prep (Johann) to harvest plasmid DNA</B></PRE><PRE>             Mutagenesis of ATPCS
             in 
                pQE80L_ATPCS-GS-PPMT
                pQE80L_ATPCS_PPMT
                pQE80L_ATPCS
             3x PCR-Reactions
                Q5 MM   12.0µl
                PB_for SN_ATPCS_xbaI_Mut_for   1.25µl
                PB_rev SN_ATPCS_xbaI_Mut_rev   1.25µl
                nuc.-free H2O   10.5µl
                template (1ng)   ~ 1.0µl (dilations of original plasmids)
             Total 26µl</PRE><PRE><B>PCR Program</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> T/°C </TH><TH> t/min
</TH></TR><TR><TD> 98 </TD><TD> 0:30
</TD></TR><TR><TD>98 </TD><TD> 0:10
</TD></TR><TR><TD> 69 </TD><TD> 0:30
</TD></TR><TR><TD> 72 </TD><TD> :
</TD></TR><TR><TD> 72 </TD><TD> 2:00
</TD></TR></TBODY></TABLE><PRE>             Trafo of 5µl PCR-Product in DH10b cells --&gt; incubation at 37 °C
</PRE><PRE><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 5 of 7 </STRONG></PRE><PRE>             - Preculture of DH05 with RF plasmid
</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/24 Wednesday </H2><STRONG> Berlin Vector - pQE-80L-JBFS-huFerritin Assembly PCR </STRONG><STRONG> Running Gel Electrophoresis </STRONG><PRE>             * 1% Agarose, Ethidium bromide 
</PRE><PRE>             * 50µL Assembly PCR product + 10µL 6*DNA dye
             * 10µL GeneRuler DNA Ladder Mix (ThermoScientific)
             * Gel Electrophoresis conditions 100V, 30min
              
<STRONG>Gel Extraction </STRONG>

              Gel Extraction using GeneJET Gel Extraction Kit (ThermoScientific) 
 
             * Determination of Concentration by Ultraviolet Spectrophotometry
</PRE><PRE> ?
</PRE><PRE><STRONG>Double Digest </STRONG></PRE><PRE>             * Digestion of 1μg of purified Insert DNA using FastDigest BamHI / HindIII
             * 1h at 37°C
             ?
</PRE><PRE>             * Heat inactivation of FastDigest BamHI / HindIII, 15min at 80°C
</PRE><PRE><STRONG>PCR purification </STRONG></PRE><PRE>             * PCR purification of BamHI / HindIII digested insert using GeneJET PCR Purification Kit (ThermoScientific)
             * Determination of Concentration by Ultraviolet Spectrophotometry
</PRE><PRE> ?
</PRE><PRE><STRONG>Ligation </STRONG></PRE><PRE>       * Using the Molar Ligation Ratio 1:5 (1 times vector and 5 times insert)
</PRE><PRE>             * 20min at RT
</PRE><PRE><STRONG>Chemical Transformation </STRONG></PRE><PRE>             * 100µL Aliquots of Chemically Competent Cells - DH10B <I>E.coli</I> + 10µL Ligation approach chilled on ice for 30 min
             * Heat shock for 1min at 42°C
             * Add 900µL preheated (37°C) LB
             * Incubate the cells for 1h at 37°C with shaking approx. 200rpm
             * Plate 200µL and spin down the rest and plate on LB Ampicillin, incubate overnight at 37°C
</PRE><PRE><STRONG>Saba</STRONG>
             o/n culture of clones from successful pQE80L_ATPCS-mut_GS_PPMT and pQE80L_ATPCS-mut_PPMT transformation
             --&gt; 37°C, 200pm, o/n 5-6 nl LB+AB (Amp)
</PRE><PRE><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 6 of 7 </STRONG></PRE><PRE>             - Miniprep of the Preculture, DNA concentration 71 ng/µl
</PRE><PRE><B>Chemical Biotransformation of:</B>
              -<B>3 µl of RFP 71 ng/µl</B>
              -2 µl <B>Ferritin jbfs_m (different ferrtin)(120 ng/µl) with  (Amp)'</B></PRE><PRE>             in RV308 and Nissle (2 plates each)
              
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/25 Thursday - PCR of Biobrick Parts </H2><STRONG>Isolation of plasmid-DNA with Mini-Prep Kit of pQE80L_ATPCSMut_GS_PPMT and pQE80L_ATPCS_PPMT</STRONG><B>PCR of Biobrick-Parts 03,04,06 &amp; 10-15</B><TABLE class="wikitable"><TBODY><TR><TH> BB </TH><TH> PB_fw </TH><TH> PB_rew </TH><TH> Tm/°C </TH><TH> template </TH><TH> bp </TH><TH> part
</TH></TR><TR><TD> 03 </TD><TD> BBa-K1438003_fw </TD><TD> BBa-K1438003_rew </TD><TD> 67 </TD><TD> pQE80L_PPMT_GS_ATPCS </TD><TD> 1715 </TD><TD> PPMT GS ATPCS
</TD></TR><TR><TD> 04 </TD><TD> BBa-K1438004_fw </TD><TD> BBa-K1438003_rew </TD><TD> 65 </TD><TD> pQE80L_PPMT_ATPCS </TD><TD> 1408 </TD><TD> ATPCS
</TD></TR><TR><TD> 10 </TD><TD> XbaI_T7_prom </TD><TD> for Lambda_Term_suffix_rew </TD><TD> 66 --&gt;67 </TD><TD> pQE80L_BFR </TD><TD> 690 (+377bp--&gt;cp) </TD><TD> BFR_cp
</TD></TR><TR><TD> 11 </TD><TD> &quot; </TD><TD> &quot; </TD><TD> 66 --&gt;67 </TD><TD> pQE80L_BFRM52H </TD><TD> 690 (+377bp) </TD><TD> BFRM52H_cp
</TD></TR><TR><TD> 12 </TD><TD> &quot; </TD><TD> &quot; </TD><TD> 66 --&gt;65 </TD><TD> pQE80L_FTH_FLH(HuFer) </TD><TD> 1337(+377bp) </TD><TD> FTH_FLH_cp
</TD></TR><TR><TD> 13 </TD><TD> &quot; </TD><TD> &quot; </TD><TD> 66 --&gt;67 </TD><TD> pQE80L_PPMT_GS_ATPCS </TD><TD> 1968(+377bp) </TD><TD> PPMT_GSATPCS_cp
</TD></TR><TR><TD> 14 </TD><TD> &quot; </TD><TD> &quot; </TD><TD> 66 --&gt;67 </TD><TD> pQE80L_PPMT_ATPCS </TD><TD> 2091(+377bp) </TD><TD> ATPCS_cp
</TD></TR><TR><TD> 06 </TD><TD> &quot; </TD><TD> &quot; </TD><TD> 66 </TD><TD> pQE80L_fbfs_mil_Ferritin </TD><TD> 1403(+377bp) </TD><TD> fbfs_mil_Ferritin_cp
</TD></TR></TBODY></TABLE><PRE><B>PCR-Ansätze V/µl</B>
                Q5 2xMM       25.0
                10µM PB_fw    2.5
                10µM PB_rew   2.5
                template      2.0 (~1ng)
                nuc.-free H2O 18.0
             <B>PCR-Programm</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> T/°C </TH><TH> t/min </TH><TH> cycles
</TH></TR><TR><TD> 98 </TD><TD> 30&quot; </TD><TD> 30
</TD></TR><TR><TD> 98 </TD><TD> 10&quot; </TD><TD> &quot;
</TD></TR><TR><TD> 66/67 </TD><TD> 30&quot; </TD><TD> &quot;
</TD></TR><TR><TD> 72 </TD><TD> 40&quot;/12&quot;/1' </TD><TD> &quot;
</TD></TR><TR><TD> 72 </TD><TD> 2' </TD><TD> &quot;
</TD></TR><TR><TD> 8 </TD><TD> infinity </TD><TD> &quot;
</TD></TR></TBODY></TABLE><PRE>             PCR purification of 1,2, 3 (Jbfs_mil_Ferritin_cp),BF-cp and BFM-cp biobrick parts
</PRE><TABLE class="wikitable"><TBODY><TR><TH> Concentration µg/ml
</TH></TR><TR><TD> fbfs_mit_Ferritin1 </TD><TD> 42
</TD></TR><TR><TD> fbfs_mit_Ferritin2 </TD><TD> 35
</TD></TR><TR><TD> fbfs_mit_Ferritin3 </TD><TD> 42
</TD></TR><TR><TD> ATPCS and PPMT </TD><TD> -12 too low
</TD></TR><TR><TD> Bacterioferritin CP</TD><TD> 56
</TD></TR><TR><TD> BFM52H CP </TD><TD> 57
</TD></TR></TBODY></TABLE><P><STRONG> Magnetization of E. coli by expression of human Ferritin, Bio-transformations, 3rd Experiment part 7 of 7 </STRONG>
              - Double-trafo was successful in all RV308 plates  but not in Nissle
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 09/30 Tuesday </H2><STRONG>Ligation of Biobrick parts with PSBIC3 backbone (digested on 23.09.2014 by Johann</STRONG>


              Transformation into DH10b c.c.cells by chemical transformation

              5µl of each ligation was transformed to 100µl aligusted cells each

              --&gt; after recovery at 37°C, 30min cells were plated on Cm/LB-Agar plates after centrifuging--&gt;pelleting and resuspending in 100µl LB to plate all cells.

              --&gt; incubation at 37°C, o/n

              <STRONG> Mini Prep von pSB 1U 3-RFP </STRONG>
              concentration 112 ng/µl

              Preculture 
              pQE-80L-jbfs-Ferritin 5ml LB+5µl Amp at 37°C overnight 
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/01 Wednesday - Plasmid Digestion &amp; PCR Probe </H2><STRONG>Plasmid Digestion </STRONG><TABLE class="wikitable"><TBODY><TR><TH> Plasmid Digestion </TH></TR><TR><TD> NEB Buffer 3.1 </TD><TD> 5µl
</TD></TR><TR><TD> NEB XbaI </TD><TD> 0.5µl
</TD></TR><TR><TD> NEB PstI </TD><TD> 0.5µl
</TD></TR><TR><TD> P_MA_T_PPMT 312ng/µl </TD><TD> 3.2µl
</TD></TR><TR><TD> nucl. free water </TD><TD> 40.5µl
</TD></TR></TBODY></TABLE><P><STRONG>PCR Probe </STRONG><EM>Ligation plates evaluated:</EM>

              All plates had clones except for BFR_cp from each plate transformation one clone was cultivated with LB/Cm (~5ml) o/n at 37°C
              </P></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/02 Thursday - Cultures were Mini-preped </H2>

              --&gt; elution with 40µl EB

              --&gt; DNA concentration measurement
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/06 Monday- Colony PCR </H2><TABLE class="wikitable"><TBODY><TR><TH> Biobrick </TH><TH> Sequecing </TH><TH> Sequenz (?) </TH><TH> (???)
</TH></TR><TR><TD> BB0 </TD><TD> BFR </TD><TD> 100Y BFR M52H </TD><TD> -A ws ATGf (??)
</TD></TR><TR><TD> BB1 </TD><TD> BFR M52H </TD><TD> 100% BFR </TD><TD> -A wr
</TD></TR><TR><TD> BB2 </TD><TD> HFTN_LFFN (?) </TD><TD> 96Y HuFerritin (Rw Sep nötig) ?? </TD></TR><TR><TD> AnP </TD><TD> ATPCS </TD><TD> f 70% ok, but fragment f </TD><TD> redo PCR ... Gelex (?)
</TD></TR><TR><TD> BFM1cp </TD><TD> BFR1cp </TD><TD> ? </TD></TR><TR><TD> BFM2cp </TD><TD> BFR2cp </TD><TD> ? </TD></TR><TR><TD> HuFecp </TD><TD> HuFerritin cp </TD><TD> ? </TD></TR></TBODY></TABLE><PRE><EM>Colony-PCR of other clones</EM></PRE><PRE><B>PCR-Programm</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> T/°C </TH><TH> t/min </TH><TH> cycles
</TH></TR><TR><TD> 95 </TD><TD> 7' </TD><TD> 30
</TD></TR><TR><TD> 95 </TD><TD> 30&quot; </TD><TD> 30
</TD></TR><TR><TD> 52 </TD><TD> 30&quot; </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 1' </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 10&quot; </TD><TD> 30
</TD></TR><TR><TD> 8 </TD><TD> infinity </TD><TD> 30
</TD></TR></TBODY></TABLE><PRE>             --&gt; gel (1% Agarose)
</PRE><PRE>             Clones 3, 5, 6, 7 and 13 were grown o/n in LB/Cm for Mini-prep on the following day. therefore 5µl of the dissolved clones were used for each inoculation
</PRE><PRE>             --&gt; 37°C, 200rpm
</PRE><P><STRONG>2x M9 Media Production</STRONG>
              (Christina)

              2X M9 mineral medium

              For 500 ml M9 mineral medium add to XXX ml sterile water:</P><TABLE class="wikitable"><TBODY><TR><TD> 100ml </TD><TD> M9 salt solution (10X) </TD><TD> Na<SUB>2</SUB>HPO<SUB>4</SUB><PRE>             KH<SUB>2</SUB>PO<SUB>4</SUB>
             NaCl
             NH<SUB>4</SUB>Cl || 67.4mM
             44.0mM
             17.10mM
             18.70mM
</PRE></TD></TR></TBODY></TABLE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/07 Tuesday </H2><EM>Mini-Prep of cultures (Ben) and preparation of sequencing samples (Saba):</EM>

              9µl nuc.-free H2O + 3µl DNA + 3µl PB794<STRONG> Preculture of Knockouts </STRONG>
              * λΔ FieF +PCP 20 : 5ml LB+tet/amp (1:1000)
              * λΔ FUR  : 5ml  LB+tet/kan (1:1000)
              * MG1655 ΔFieF: 5ml LB + amp/kan (1:1000)
              incubation at 30°C
              </DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/08 Wednesday - PCR of PSB1C3 Backbone </H2>

              Q5 Mastermix 2x 

<TABLE class="wikitable"><TBODY><TR><TH> Component </TH><TH> Volume in µl
</TH></TR><TR><TD> Q5 Mastermix 2x </TD><TD> 25
</TD></TR><TR><TD> Primer fw (PSB1C3 Suffix fw)</TD><TD> 2,5
</TD></TR><TR><TD> Primer rev (PS1C3 Prefix rev</TD><TD> 2,5
</TD></TR><TR><TD> template (PSB1C3_BBa_K1189018)</TD><TD> 2
</TD></TR><TR><TD> nuc free H2O</TD><TD> 18
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Temperature in °C</TH><TH> Time</TH><TH> Nr. of cyles
</TH></TR><TR><TD> 98</TD><TD> 30</TD></TR><TR><TD> 98</TD><TD> 5</TD><TD> loop start</TD></TR><TR><TD> 70</TD><TD> 15</TD><TD> 30</TD></TR><TR><TD> 72</TD><TD> 1'</TD><TD> loop end
</TD></TR><TR><TD> 72</TD><TD> 2'</TD></TR><TR><TD> 8</TD><TD> infinit</TD></TR></TBODY></TABLE><PRE><STRONG> Extraction of PCR Product (PSB1C3 linearized) </STRONG></PRE><PRE>             After extracting the expected band at 2000 bp it was purified using the EMDMillipore Montage Gel Extraction Device.
             Resulting in about 120 µl DNA extract in TAE buffer with an concentration of 9ng/µl</PRE><TABLE class="wikitable"><TBODY><TR><TH> Digestion of PCR Product </TH></TR><TR><TD> NEB Buffer 3.1 </TD><TD> 15µl
</TD></TR><TR><TD> NEB PstI</TD><TD> 1.5µl
</TD></TR><TR><TD> NEB XbaI </TD><TD> 1.5µl
</TD></TR><TR><TD> nuc. free water </TD><TD> 83µl
</TD></TR><TR><TD> PCR Product </TD><TD> 100µl
</TD></TR></TBODY></TABLE><PRE>             * 1.5 hours at 37°C
             * PCR Purification
             * DNA concentration measurement = 35ng/µl
</PRE><PRE><STRONG> Ligation </STRONG>
             Calculation
             *BFM52H = 3.22µl
             *BF_cp= 6.7µl
             *jbfs_m = 8.06 µl
             *BB_AnP = 10.64 µl
             * HuFer = 9.45 µl
             <STRONG> Iron Concentration measurement </STRONG>
             278 mg FeSO<SUB>4</SUB>.7H<SUB>2</SUB>O dissolved in 10 ml d.H<SUB>2</SUB>O 
             λ FieF PCP<SUB>2</SUB> Amp  1:100 ---&gt; 3.0 OD
             iGEM BV125 tet 1:100 --&gt; 3.0 OD 
             λ FieF PCP<SUB>1</SUB> Amp 1:20 --&gt;3.26 OD
             iGEM BV125 tet 1:20---&gt; 2.44
             too high OD!
</PRE><PRE>             Both samples diluted 1:2
             Samples
             #  FieF + 5µl 0.1 M Fe-stock solution
             # FieF + 15 µl 0.1 M Fe-Stock solution
             # BV + 5 µl 0.1M Fe-stock solution
             # BV + 15 µl 0.1 M Fe-stock solution
</PRE><PRE>             Overnight incubation at 30°C on a shaker 
     
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/09 Thursday </H2> 
              Mini Prep (using: AccuPrep Plasmid Mini Extraction Kit Ver. 2.0, Company: BIONEER), using 100µl Elution Buffer 
              and dsDNA Measurement: 4 µl Sample + 76µl MilliQ, blanked with MilliQ

<TABLE class="wikitable"><TBODY><TR><TH> Samplenumber </TH><TH> Samplename </TH><TH> dsDNA conc [ng/µl] </TH><TH> Sample [µl] used for Sequencing </TH><TH> nucl.free Water [µl] used for Sequencing
</TH></TR><TR><TD> #1 </TD><TD> cp HuF </TD><TD> 111 </TD><TD> 9 </TD><TD> 3
</TD></TR><TR><TD> #2 </TD><TD> AGSP </TD><TD> 107 </TD><TD> 9,4 </TD><TD> 2,6
</TD></TR><TR><TD> #3 </TD><TD> AGSP </TD><TD> 59 </TD><TD> 12 </TD><TD> 0
</TD></TR><TR><TD> #4 </TD><TD> AGSP </TD><TD> 64 </TD><TD> 12 </TD><TD> 0
</TD></TR><TR><TD> #5 </TD><TD> jbfs_Fer </TD><TD> 54 </TD><TD> 12 </TD><TD> 0
</TD></TR><TR><TD> #6 </TD><TD> A'n'P </TD><TD> 67 </TD><TD> 12 </TD><TD> 0
</TD></TR></TBODY></TABLE><PRE>             For Sequencing:
             *total volume: 15µl
             *Primer used: PB795 3µl each Sequencing approach
             *Sample [µl] see table
             *nucl.free Water [µl] see table
</PRE><PRE>             evaluation of Trafos ---&gt; no growth ---&gt;Repeat Trafo in other cells
 
              </PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/10 Friday - STRATEGY 1 - Knockout of FieF and FUR Genes </H2><EM> Detection PCR </EM>
              kan-deletion-cassette for FieF and FUR Genes was transformed into 2 different strains (MG1655 and λ MAGE). Additionally pCP20 was already transformed into one of the λ MAGE Clones (has to be re-validated).
              The positive clones have to be validated.

              Detection-PCR of one clone of Plates 1-3:

              Plate 1: MG1655 ΔFieF Plate 2: λ ΔFieF + pCP20
              Plate 3: λ ΔFUR<EM> Primer-Combinations </EM><B>FieF</B><TABLE class="wikitable"><TBODY><TR><TH>- </TH><TH> Combination </TH><TH> WT length </TH><TH> Clone length </TH><TH> Annealing Temperature range
</TH></TR><TR><TD> A </TD><TD>C1+C2 </TD><TD> 2060bp </TD><TD> 2654bp </TD><TD> 58,4-60,1°C
</TD></TR><TR><TD> B </TD><TD>C1+C3 </TD><TD> 0 </TD><TD> 1046bp </TD><TD> 59,5-60,1°C
</TD></TR><TR><TD> C </TD><TD>C4+C2 </TD><TD> 0 </TD><TD> 1181bp </TD><TD> 57,5-58,4°C
</TD></TR><TR><TD> D </TD><TD>C1+C5 </TD><TD> 1134bp </TD><TD> 0 </TD><TD> 60,1-60,5°C
</TD></TR></TBODY></TABLE><PRE>             --&gt; Annealing Temperature: 57°C</PRE><PRE><B>FUR</B></PRE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH>Combination </TH><TH> WT length </TH><TH> Clone length </TH><TH> Annealing Temperature range
</TH></TR><TR><TD> A </TD><TD>C1+C2 </TD><TD> 1130bp </TD><TD> 2219bp </TD><TD> 59,5°C
</TD></TR><TR><TD> B </TD><TD>C1+C3 </TD><TD> 0 </TD><TD> 823bp </TD><TD> 59,5°C
</TD></TR><TR><TD> C </TD><TD>C4+C2 </TD><TD> 0 </TD><TD> 969bp </TD><TD> 57,5-59,5°C
</TD></TR><TR><TD> D </TD><TD>C1+C5 </TD><TD> 500bp </TD><TD> 0 </TD><TD> 59,5°C
</TD></TR></TBODY></TABLE><PRE>             --&gt; Annealing Temperature: 57°C
             <EM> PCR-Assay </EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> - </TH><TH> 1x </TH><TH> Mastermix 15x
</TH></TR><TR><TD> nuc.free H<SUB>2</SUB>O </TD><TD> 14,4µl </TD><TD> 216µl
</TD></TR><TR><TD> 10x Dream Taq Buffer </TD><TD> 2µl </TD><TD> 30µl
</TD></TR><TR><TD> 50mM MgCl<SUB>2</SUB></TD><TD> 0,6µl </TD><TD> 9µl
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 0,5µl </TD><TD> 7,5µl
</TD></TR><TR><TD> 10mM Primer Combination </TD><TD> 1µl
</TD></TR><TR><TD> 10µl nuc.free H<SUB>2</SUB>O+ picked colony</TD><TD> 1µl + 18,4ml Mastermix
</TD></TR><TR><TD> Taq Polymerase </TD><TD> 0,5µl
</TD></TR></TBODY></TABLE><PRE><EM> PCR-Program </EM></PRE><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 95°C </TD><TD> 7'
</TD></TR><TR><TD>30 cycles
</TD></TR><TR><TD> 95°C </TD><TD> 30&quot;
</TD></TR><TR><TD> 57°C </TD><TD> 30&quot;
</TD></TR><TR><TD> 72°C </TD><TD> 2'40&quot;
</TD></TR><TR><TD> 72°C </TD><TD> 5'
</TD></TR><TR><TD> 12°C </TD><TD> hold
</TD></TR></TBODY></TABLE><PRE><EM> Gel-Electrophoresis </EM>
              1% Agarose-Gel + 1µl GelRed; 90V; 30min</PRE></DIV><DIV class="sub-content-project"><H2 class="sub-content-project-headline"> 10/14 Tuesday - Iron-Assay </H2>
              1. Day: inoculation of 5ml LB Media (+ Antibiotics)
                
              * 37°C or 30°C/ON
              2.Day:inoculation of 10ml LB (1:10)
              * 30°C or 37°C
              * grow until OD<SUB>600</SUB> 0,6
              * take 1ml
              * centrifuge 2min at max. speed
              * discard the supernatant 
              * wash with 1X M9 Media
              * 1XM9:
              :500µL 2X M9+
              :500µL sterile water
              * wash 2x and recover 1mL 1XM9 Media
              * diluted 1:1000 with 1X M9
              * plate 20µL per well



              24well plate without Antibiotics
              
              *A:MG1655 wt
              *B:Nissel 1917 wt
              *C:RV308 wt
              *D:DH10b
              
              concentration of Fe-citrat
              
              0    0,5    1    2    5    10mM
              </DIV></TBODY></TABLE></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></SECTION></DIV><SECTION data-speed="1" data-type="background"><DIV class="container-fluid sponsor-wrapper"><DIV class="row sponsor-heading-row"><DIV class="col-xs-12"><H3>Sponsors</H3></DIV></DIV></DIV></SECTION><DIV class="footer"><DIV class="row"><DIV class="col-sm-2">Müller-Breslau-Str. 10 10623 Berlin</DIV><DIV class="col-sm-2">Phone: +49 1778800244  Mail: <A href="mailto:mail@igem.berlin">mail@igem.berlin</A></DIV><DIV class="col-sm-6"><DIV class="footer-sitemap"><UL><LI><A href="https://2014.igem.org/Team:Berlin" class="active">Home |</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Project">Project |</A></LI><LI><A href="http://igem.berlin/team.php">Team |</A></LI><A href="http://igem.berlin/safety.php">Safety |</A><LI><A href="http://igem.berlin/workshop2014_neu.php">Workshop |</A></LI><LI><A href="http://igem.berlin/blog.php">Blog</A></LI></UL></DIV><DIV class="footer-imprint">
        ©2014 iGem Berlin, Layout/Umsetzung: Valerian Blos, Marius Wessel
      </DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Berlin/Project/Journal">http://2014.igem.org/Team:Berlin/Project/Journal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Berlin/Project/Journal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Berlin/Project/Journal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Berlin/Project/Journal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Berlin/Project/Journal&amp;oldid=396435" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>