<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Berlin_Project_Detailed-Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Berlin/Project/Detailed-Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Berlin/Project/Detailed-Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Berlin/Project/Detailed-Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Berlin/Project/Detailed-Description&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Berlin/Project/Detailed-Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Berlin/Project/Detailed-Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="site-wrapper-sub-project"><DIV class="main-menue clearfix navbar-fixed-top hidden-xs hidden-sm hidden-md" style="margin-top:20px;"><UL class="main-menue-ul"><LI><A href="https://2014.igem.org/Team:Berlin" class="main-menue-links">Home</A></LI><LI class="active"><A href="https://2014.igem.org/Team:Berlin/Project" class="main-menue-links">Project</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Team" class="main-menue-links">Team</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Safety" class="main-menue-links">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Workshop" class="main-menue-links">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Blog" class="main-menue-links">Blog</A></LI></UL></DIV><DIV class="navmenu navmenu-default navmenu-fixed-left offcanvas hidden-lg"><A class="navmenu-brand" href="https://2014.igem.org/Team:Berlin"><H1>iGEM Berlin</H1></A><UL class="nav navmenu-nav"><LI><A href="https://2014.igem.org/Team:Berlin" class="active">Home</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Workshop">Workshop</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Blog">Blog</A></LI></UL></DIV><SECTION data-speed="200" data-type="background" id="section-sub"><DIV class="container-fluid container-green"><DIV class="row featurette" id="main-featurette-row"><DIV class="start-text-container-project"><H2>Explore our Project:</H2></DIV></DIV></DIV><DIV id="subside-content"><DIV class="container"><DIV class="row"><DIV class="col-xs-3 submenue-project" style="text-align:left;"><A href="https://2014.igem.org/Team:Berlin/Project" class="sub-link-project"> 1. What is it all about?</A><A href="https://2014.igem.org/Team:Berlin/Project/Activities" class="sub-link-project"> 2. Project-related Activities</A><A href="https://2014.igem.org/Team:Berlin/Project/Detailed-Description" class="sub-link-project">3. Detailed Description</A><A href="https://2014.igem.org/Team:Berlin/Project/Results" class="sub-link-project">4. Our Results</A><A href="https://2014.igem.org/Team:Berlin/Project/Summary" class="sub-link-project">5. Lab Summary</A><A href="https://2014.igem.org/Team:Berlin/Project/Journal" class="sub-link-project">6. Lab Journal</A><A href="https://2014.igem.org/Team:Berlin/Project/Property" class="sub-link-project">7. Intellectual Property</A></DIV><DIV class="col-xs-11 col-sm-9 blog-text" style="margin-bottom:40px;"><DIV class="project-number">3</DIV><DIV class="project-headline-float"><H2 class="green-text project-headline">Detailed Description</H2></DIV><P><STRONG> 3.1 Summary:  iGEM Berlin 2014 - A remote control for E. coli</STRONG>
As the first iGEM team from Berlin to take part in the competition, we decided to construct a simple BioBrick that enables synthetic biologists to remotely control the movement of the laboratory workhorse Escherichia coli. A seemingly simple and non-invasive mechanism for this remote control is the use of magnetic fields, which enable a vast variety of applications. In nature, these are already used by several mammals, as well as bacteria for orientation.
Remote-controlled E. coli cells could be used as “bacteria-based nanorobots” in order to maneuver them live in the intestine - targeting diseased tissue or even tumors. Combined with BioBricks from other iGEM teams a local, site-directed treatment of intestine cancer is envisioned[1]. Therefore, we work with the probiotic E. coli strain Nissle 1917, which already has an 80 year history as a treatment for chronically inflamed intestine tissue and is sold commonly as “Mutaflor”[2].
Our experiments should enable us to construct simple BioBricks capable to produce a diverse set of nanoparticles, including quantum dots, magnetic nanoparticles, semiconductor nanoparticles, noble metal nanoparticles and many more[3]. For example, cheap and efficient production of magnetic nanoparticles could be used in hyperthermal cancer therapy. Subsequently, we would like to investigate if our method can also be adapted to the synthesis of rare earth metals, which despite their abundance are not mineable. This would offer an alternative to the environmental hazardous mining operations that are common today.
A remote control for E. coli will enable a whole variety of application relevant for bioprocess engineering. Therefore one can imagine neglecting centrifugation for cell separation - simply use a magnet. Also, it should be possible to use high frequent oscillating magnetic fields for cell lysis or even use rotating magnetic fields for self-stirring cultures in bioreactors solving scale up limiting issues like tip speed.
Nature is the world’s most skilled engineer and has naturally occurring magnetotactic bacteria such as Magnetospirrilium magneticum. Under certain conditions, these organisms form magnetosomes, which are chains of magnetite nanoparticles. These function as an intracellular compass for the cell and even allow them to orientate along earth’s weak magnetic field.
As previous iGEM teams have shown, synthesizing fully functional magnetosomes in E. coli is highly difficult as more than 60 highly regulated genes are involved[4]. The Berlin iGEM team came up with an alternative strategy that does not rely on the formation of magnetosomes.
By knocking out the iron efflux transporter gene FieF and the iron uptake suppressor Fur, we want to increase the total iron level of the cytosol. By sequestering iron in a ferritin protein, iron crystals are formed and the cell is detoxified. In order to create ferromagnetic crystals, we will use intensive high-throughput growth medium optimization to discover the best conditions for the formation of magnetic nanoparticles in E. coli. 
We will work with other metal binding proteins such as metallothioneins and phytochelatin synthases in order to achieve nanoparticle synthesis. Once we have discovered the best way to magnetize E. coli bacteria, we will build and characterize suitable BioBricks that can be used by any research lab or iGEM team in the world in order to remote control the cellular movement.
References:
[1]	https://2012.igem.org/Team:Penn/ProjectResults (11.07.2014)
[2]	http://www.mutaflor.de/cms/ (11.07.2014)
[3]	Park, T. J., Lee, S. Y., Heo, N. S. and Seo, T. S. (2010), In Vivo Synthesis of 
Diverse Metal Nanoparticles by Recombinant Escherichia coli. Angew. Chem.
Int. Ed., 49: 7019–7024.
[4]	https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Toolkit (11.07.14)
<STRONG>3.2 Basics of biomagnetism</STRONG>
      In order to control E. coli remotely, we decided to use magnetic fields because of a few key reasons.
      First, magnetic fields allow to control the cells by an external force field which means that cells do not have to be directly treated or media conditions have to be changed as it is the case in many chemotaxis assays.
      Second, magnets are widely spread and electro magnets easily built as you can see in this video:
      <A href="https://www.youtube.com/watch?v=wzXRFp0DDrU">How to build your own DIY super magnet</A>
      The accessibility and safety of magnetic fields opens up the field for new innovation and ideas using remote controlled bacteria as you can see in this <A href="https://www.youtube.com/watch?v=fCSOdQK5PIY">video in which magnetotactic bacteria are used to rearrange building blocks to a pyramid</A>.
       Third, magnetic fields do have a high energy density and are way more viable to transfer energy than other energy fields like electrical fields. However, the magnetic moment of an atom is the product of the atoms orbital angular momentum and its electron spins. Atomic magnetism is based on unfilled electron orbits.
      Nobel gases and alkyl halogenide are non-magnetic or “diamagnetic”. Other elements like metals may have an increased magnetic moment as long as their electron orbit are unfilled, meaning they are unbound. Mn2+, Fe3+ have a magnetic moment of about 5 µB, Cr2+, Mn3+, Fe2+ and Co3+ have a  <A href="ttp://www.diss.fu-berlin.de/diss/servlets/MCRFileNodeServlet/FUDISS_derivate_000000000626/2_f-Kapitel1.pdf?hosts=">magnetic moment of 3 µB.</A>
      Because of the variety of atomic magnetic properties, magnetism can be divided into different forms by looking at apparent forces and effects. 
      There is diamagnetism, antiferromagnetism, paramagnetism, ferrimagnetism, superparamagnetism and Ferromagnetism. Ferromagnetic forced are about 1000 times stronger than super paramagnetic forces. However, it turns out that creating ferromagnetic particles or residues is very difficult (see magnetite formation in magnetospirrilium bacteria). Also note that in a lot of biological papers, people tend to mix e.g. super paramagnetism with ferromagnetism [1].
      
      Magnetic properties are depended on the composition, structure and the size of the metallic particles. The magnetic moment is proportional to the alignment of the electron spins within a component. If electron spins are aligned magnetism can be observed. However on a nanoscale real ferromagnetism can only be noted after a critical particle size of 128 nm.[2]
      This is due to thermal fluctuations that prevent the alignment of electron spins and therefore prevent the development of sufficient magnetic moments. Because of this phenomena all measurements for magnetism are usually conducted at very low temperatures.
      
      All magnetic effects that are observed for particles between 1 nm to 100 nm are considered superparamagnetic. Superparamagnetic nanoparticles only show magnetic effects when an external magentic field is applied. This field aligns the electron spins in the particle resulting in a magnetic moment mediated by an external magnetic field.
      
References:
[1] Park, T. J., Lee, S. Y., Heo, N. S. and Seo, T. S. (2010), In Vivo Synthesis of Diverse Metal Nanoparticles by Recombinant Escherichia coli. Angew. Chem. Int. Ed., 49: 7019–7024. doi: 10.1002/anie.201001524
[2]An-Hui Lu, An-Hui; E. L. Salabas; Ferdi Schüth (2007). &quot;Magnetic Nanoparticles: Synthesis, Protection, Functionalization, and Application&quot;. Angew. Chem. Int. Ed. 46 (8): 1222–1244<STRONG>3.3 Iron Homeostasis, E. coli Nissle and Knockouts</STRONG>
      As an essential element for almost all life iron is often necessary for the activity of certain proteins but can also be problematic because of its toxicity and poor solubility. Organisms have evolved to regulate their iron effectively and as iron in an organism is usually supplied in a limited condition pathogens evolved iron acquisition systems to outcompete other microorganism.[1]
E. coli Nissle 1917 is a probiotic strain which is used for medical treatment of diseases (infections) of the gastrointestinal tract. This probiotic strain is available at pharmacies for the public. Since EcN is classified as a non-pathogenic strain in category S0 one might never assume that this strain could ever become pathogenic or be a possible danger to seriously ill patients consuming the probiotic. However, genomic analysis of EcN showed that the genome of tested EcN cultures possess genes from horizontal transformations such as adherence factors or iron uptake systems. Do these findings pose the possible transfer of virulence factors as well? Different studies showed that depending on environmental conditions EcN developed pathogenecity (Dr. Schulze). Depending on the location E. coli manifests it is differenced between intestinal pathogenic E. coli (IPEC) and extraintestinal pathogenic E. coli (ExPEC). IPEC causes severe diarrhea, while ExPEC are often constituents of human microbiome. They behave unobtrusive until they leave their familiar habitat and become pathogens of urinary tract infections, Sepsis and natal meningitis. Although these findings are concerning horizontally transferred genes are mostly acquired for the purpose of surviving and fitness. For instance the expression of the aerobactin iron uptake system is important for the fitness of strain Nissle 1917, as the availability of free iron in the human body is limited. Thus, together with the expression of other systems involved in iron acquisition the aerobactin system contributes to competitiveness, survival, and colonization. From the authors point view, the presence of six different determinants involved in iron uptake is noteworthy and seems to contribute to the great adaptability and fitness of E. coli strain Nissle 1917.  A very efficient iron acquisition system, which bacteria including E. coli use are siderophore mediated transport pathways.  
              We tested the probiotic E. coli Nissle 1917 strain because of its additional iron acquisition systems. Since this E. coli strain is not a laboratory strain we addressed its safety in our safety section.

Additionally, we attempted to knockout two genes that regulate the iron uptake systems in E. coli. First, the ferrous iron efflux transporter (FieF) which belongs to the cation diffusion facilitator family and is responsible for transporting excessive iron out of the cytosol into the cell environment. Another, relevant regulator for the iron homeostasis is the ferric uptake regulator (fur) -which was previously topic of another <A href="https://2013.igem.org/Team:Evry" class="link">iGEM project.</A> However, by knocking out both of these genes we hope to increase the iron content inside of the cell in order to load our ferritin proteins more efficiently.[2]
      
       [1]  Simon C. Andrews, Andrea K. Robinson, Francisco Rodriguez-Quinones; School of Animal and Microbial Sciences, University of Reading, Reading RG6 6AJ, UK; 2003
[2] Grass G, Otto M, Fricke B, Haney CJ, Rensing C, Nies DH, Munkelt D. FieF
(YiiP) from Escherichia coli mediates decreased cellular accumulation of iron and
relieves iron stress. Arch Microbiol. 2005 Jan;183(1):9-18. Epub 2004 Nov 11.
PubMed PMID: 15549269.<STRONG>3.4 Ferritin Strategy</STRONG>
      Ferritins as scaffolds for magnetic nanoparticle synthesis
      

      In order to protect themselves against radical stress as well as the lack of co-factors, organisms evolved a high regulated and stable iron acquisition system, also known as iron homeostasis. [Iron Homeostasis of E. coli] To put it in a nutshell iron is taken up by iron transporters or siderophore mediated mechanism and is transporters through the outer membrane into the periplasm, in the periplasm it “changes” transporters and is transported through the inner membrane into the cytoplasm. It always gets released as Fe2+ into the cytoplasm.
      In order to protect themselves against superoxide formation by the Fenton-Reaction a lot of organisms evolved iron storage proteins. One of the superfamily of these proteins is called ferritin. These ferritins are highly symmetrical protein nanocages synthesizing iron concentrates required for cells to make cofactors of iron proteins. Through their ability to cage in biominerals they were the first and most obvious scaffolds for the synthesis of magnetic nanoparticles. These natural metal storage homomers form solid particles inside of their protein shell. Ferritins are ubiquitous in nature and protect the cell from redox stress through iron overload and from iron deficiency. They consist of 24 protein subunits which can consists of a heavy (catalytically active) and light chain (catalytically inactive but stabilizing). Caged ferritin minerals can  have diameters as large as 8-12 nm with thousands of iron and oxygen atoms. Between species ferritins have different affinity for phosphate. Phosphate is low in animal ferritin iron minerals (Fe:P = 8:1) whereas in bacterial and plant ferritins iron minerals are usually occurring in higher relations (Fe:P = 1:1).[1]
      Mössbauer studies on the superparamagnetic character of bacterioferritins (bfr) revealed that the phosphate concentration in a ferritin iron mineral reduces superparamegntic effects heavily due to replacement of the iron bridges between the iron atoms with phosphate. As these bridges have a lower exchange constant the order temperature is reduced further. [2]
      The hollow ferritin nanocages are used in the chemical industry as scaffolds for synthesis of magnetite particles as well as for delivery of magnetic resonance imaging (MRI) contrast agents, drug delivery and catalysis.
      
      [1] Ferritins for Chemistry and for Life.
      Elizabeth C Theil, Rabindra K Behera, Takehiko Tosha
      Children's Hospital Oakland Research Institute, University of California, Berkeley ; Department of Nutritional Science and Toxicology, University of California, Berkeley.
      Coordination Chemistry Reviews (Impact Factor: 11.02). 01/2013; 257(2):579-586. DOI: 10.1016/j.ccr.2012.05.013
      
      [2] Mössbauer studies of superparamagnetism in E. coli; Hawkins, C.; Williams, J. M.
      Journal of Magnetism and Magnetic Materials, Volume 104, p. 1549-1550<STRONG>Constructing an optimized human ferritin BioBrick</STRONG>
In general ferritins are found in all kingdoms of life and in many different cells of multicellular organisms. They fulfill manifold tasks synthesize iron concentrates required for cells to make cofactors of iron proteins (heme, FeS, mono and diiron) as well as caged ferritin Fe2O3*H2O is acting as an antioxidant crucial for bio metabolism of proteins [1]. Human Ferritin (huferritin) consits of two different domains, light and heavy chain.
The presented plasmid DNA construct JBFS_Mil_Ferritin &lt;partinfo&gt;BBa_K1438025&lt;/partinfo&gt; from our iGEM Team Berlin 2014 is a standardized pQE80L plasmid backbone containing human ferritin. This part was built from a iGEM Calgary 2013 part that they provided us. We were able to improve this part further, please see the part in the registry for informartion
We created this construct according to a paper published 2010 in Journal of Biological Chemistry (JBC) [3], they presented different compositions of Ferritin made of light (L) and heavy (H) chain, linked with an Glycine-Serine - construct (GS-Linker,*). 
“We found that the L*H chimera exhibits significantly enhanced iron-loading ability [...] compared to wild-type ferritin [3].“
Therefore it was necessary to combine different methods used in standard molecular biology, polymerase chain reaction (PCR) for amplification of single light and heavy chain.
Assembly PCR for combination of amplified light and heavy chain containing GS-Linker sequence and furthermore ligation and transformation [2,3].
Currently our Team is characterizing the optimized ferritin construct against different terms and conditions e.g. supplementation of iron in media, to proof a possible magnetization of Escherichia coli.


References:
[1] Ferritins for Chemistry and for Life; Coord Chem Rev. 2013 Jan 15;257(2):579-586. Epub 2012 May 18., Theil EC1, Behera RK, Tosha T.

[2] Improved Coexpression and Multiassembly Properties of Recombinant Human Ferritin Subunits in Escherichia coli; J. Microbiol. Biotechnol. (2008), 18(5), 926–932; 
Lee, Jung-Lim, Robert E. Levin, and Hae-Yeong Kim

[3] Design and characterization of a chimeric ferritin with enhanced iron loading and transverse NMR relaxation rate;
J Biol Inorg Chem. 2010 Aug;15(6):957-65. doi: 10.1007/s00775-010-0657-7. Epub 2010 Apr 17.; Iordanova B1, Robison CS, Ahrens ET.<STRONG>iGEM Berlin Ferritin Library</STRONG>
      During our summer we collected a variety of ferritin-coding sequences from bacterial and mammalian sources. As ferritins are common among all organisms we categorized our ferritins in three major groups.
      <A href="https://static.igem.org/mediawiki/2014/9/98/IGEM_Berlin_Ferritin_Library.pdf">Click here to see the PDF!</A><STRONG>3.5 Alternative magnetization strategy using metal binding peptides</STRONG>
      By talking to the Fussenegger group from the ETH Zurich, who published the superparamagnetism paper about ferritins we got the tip to look for another strategy as they experienced the limitations of ferritins. [1]
      For this reason, we came up with a different strategy. Park et al came up with a different strategy for the synthesis of biogenic nanoparticles in E. coli. [2] A strategy where they produced impressive results showing one strategy to synthesize a whole array of diverse nanoparticles with E. coli just by expressing a combination of two metal binding proteins as you can see in the following figure.Freeze-dried cell pellets consisting of various biogenic synthesised nanoparticles
      Two heavy metal binging proteins were combined and co-expressed on one plasmid. Peptides called phytochelatins are produced in fungus and plants to detoxify the cell from harmful heavy metals. Structurally phytochelatins are (gamma-Glu-Cys)n-Gly (n=2-7) peptides and function as metal ion accumulators through formation of peptide-metal conjugates. In this study the phytochelatin synthase from Arabidopsis thaliana (Columbia leave) was used (ATPCS). The other peptide that was used in combination with ATPCS was a metallothionein from Pseudomonas Putida KT2240 strain (PPMT). Metallothioneins are low-molecular proteins with a high content of cysteine and bind well cadmium, zinc, nickel and copper.

      For expression of ATPCS a trc promoter was used while PPMT was expressed using a T5 promoter. After co expressing both proteins in an standard E. coli strain(DH5alpha) for 4 h the culture broth was centrifuged and fresh metal rich LB media added. (1 – 5 mM of corresponding final metal concentration.
      After further incubation at 37°C for about 6 - 12 hours the cultures and the biogenic synthesized nanoparticles can be harvested.
      Park et al reported further that by incubating these ATPCS and PPMT co-expressing in 1.0 mM FeSO4 and MnCl2 magnetic nanoparticles where obtained and cell moved by high magnetic fields.

      [1] Kim T, Moore D, Fussenegger M. Genetically programmed superparamagnetic
      behavior of mammalian cells. J Biotechnol. 2012 Dec 31;162(2-3):237-45. doi:
      10.1016/j.jbiotec.2012.09.019. Epub 2012 Oct 2. PubMed PMID: 23036923.
      
      [2] Park, T. J., Lee, S. Y., Heo, N. S. and Seo, T. S. (2010), In Vivo Synthesis of Diverse Metal Nanoparticles by Recombinant Escherichia coli. Angew. Chem. Int. Ed., 49: 7019–7024. doi: 10.1002/anie.201001524
      
      [3] Wu CM, Lin LY. Immobilization of metallothionein as a sensitive biosensor chip
      for the detection of metal ions by surface plasmon resonance. Biosens
      Bioelectron. 2004 Nov 1;20(4):864-71. PubMed PMID: 15522603.
      <STRONG>3.6 Magnetic modelling</STRONG>
To test the supermagnetic behavior of our E.coli containing iron-loaded ferritin, we placed strong permanent neodym-magnets N45 (1.32-1.37 T) under the petri dish and observed under light microscope the movement of the cells towards the magnets. After excluding contamination of magnetized particles outside the cell, we could conclude that our E.Coli. were successfully magnetized. 
In order to achieve the maneuverability of the cells, by controlling the magnetic fields, different geometries of electromagnets were tried. 
These consisted in the iron core of a microwave, metal rods and nails with different permeability with a corresponding coil. Pure iron is difficult to find. 
The magnetic force acting on the cells is proportional to its magnetic moment μ and the gradient of the magnetic field ∇B: 

Already with a magnetic field of 0.5T, the native ferritin is saturated (Gossuin et., 2009) and its magnetic moment is reported to be between 250 and 400 μ_B (Brem et al., 2006). 
Doing a Finite Element Analysis with the software Ansoft Maxwell for the gradient of the magnetic field of a simple electromagnet, rectangle with coil, we made the qualitative and already intuitive observation that the gradient was stronger at the edges. This was confirmed by the paper (Hoke, C. Dahmani et al., 2008) in which multiple possible magnet forms were simulated in order to maximize a high field Gradient. The simulation was performed with COMSOL. It was found that the best configuration for a higher gradient was a loop with a tip at one end and a flat surface at the other end. 
So the best and easiest option for our application should be a curved nail. 
The relevant equations for the simulation were ∇xH = J and ∇B = 0 with the relation B = μ_0 μ_r H. 
The magnetic vector potential A produces the governing equation of the magnetostatics mode: 

It follows that the basic input parameters are the relative permeability of the magnet and the external current density. For the Comsol simulation in this paper an external current density of 1.9e6 A/m^2 and a relative permeability of 4*10^3 (iron) were chosen. As a result, a B-field of 1.43T and a magnetic flux density directly under the magnet tip of 588mT could be reached. But this flux density drops rapidly with the distance. The field gradient for a distance of 1mm amounts 27.08T/m and for a distance of 2cm, only 10.37T/m. 
Because of this result we came to the idea of having many electromagnets under the petri glass in order to have a high gradient across the petri glass and consequently stronger magnetic forces over the whole area. 
In order to have an idea of the speed the cells can reach, we follow the derivation of the paper of Martin Fusseneger but use as upper boundary, the field gradient simulated in the paper above for 1mm, meaning 27.08T/m. 

The net force F_net acting on the the cells is given by the magnetic force F_mag caused by the magnetic field gradient ∇B minus the drag the force F_drag (Fussenegger): 

Where ν stands fort the viscosity of the media (0.948mPas), R the Radius of the cell (9x10^-6m) and v the velocity of the cell. 
Equating the magnetic and the dragging force, we obtain: 

For a permenant magnet of 1.40 T and a magnetic field gradient of 150T/m, a minimum speed of 15*10^{-6}m/s is calculated. 
For an electromagnet like the one on the paper of Dahmani with a magnetic field gradient of 10.37T/m at 2cm, which is much stronger than ours, we calculate a maximal speed of 6.48*10^{-6}m/s, less than half than with a permanent magnet. This is the maximum limit of speed we can expect with our home made electromagnets.


 [1] Kim T, Moore D, Fussenegger M. Genetically programmed superparamagnetic behavior of mammalian cells. J Biotechnol. 2012 Dec 31;162(2-3):237-45. doi: 10.1016/j.jbiotec.2012.09.019. Epub 2012 Oct 2. PubMed PMID: 23036923.

[2] (Hoke, C. Dahmani et al., 2008) Design of a High Field Gradient Electromagnet for Magnetic Drug Delivery to a Mouse Brain)
</P></DIV></DIV></DIV></DIV></SECTION></DIV><SECTION data-speed="1" data-type="background"><DIV class="container-fluid sponsor-wrapper"><DIV class="row sponsor-heading-row"><DIV class="col-xs-12"><H3>Sponsors</H3></DIV></DIV></DIV></SECTION><DIV class="footer"><DIV class="row"><DIV class="col-sm-2">Müller-Breslau-Str. 10 10623 Berlin</DIV><DIV class="col-sm-2">Phone: +49 1778800244  Mail: <A href="mailto:mail@igem.berlin">mail@igem.berlin</A></DIV><DIV class="col-sm-6"><DIV class="footer-sitemap"><UL><LI><A href="https://2014.igem.org/Team:Berlin" class="active">Home |</A></LI><LI><A href="https://2014.igem.org/Team:Berlin/Project">Project |</A></LI><LI><A href="http://igem.berlin/team.php">Team |</A></LI><A href="http://igem.berlin/safety.php">Safety |</A><LI><A href="http://igem.berlin/workshop2014_neu.php">Workshop |</A></LI><LI><A href="http://igem.berlin/blog.php">Blog</A></LI></UL></DIV><DIV class="footer-imprint">
        ©2014 iGem Berlin, Layout/Umsetzung: Valerian Blos, Marius Wessel
      </DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Berlin/Project/Detailed-Description">http://2014.igem.org/Team:Berlin/Project/Detailed-Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Berlin/Project/Detailed-Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Berlin/Project/Detailed-Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Berlin/Project/Detailed-Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Berlin/Project/Detailed-Description&amp;oldid=385123" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>