<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tec-Monterrey_module2_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tec-Monterrey/module2.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tec-Monterrey/module2.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/module2.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/module2.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tec-Monterrey/module2.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tec-Monterrey/module2.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wiki-body"><P>&lt;body&gt;</P><PRE>&lt;header id=&quot;wiki-header&quot; class=&quot;initial-nav-scroll-up&quot;&gt;&lt;/header&gt;</PRE><DIV id="wiki-content" class="main-content"><DIV class="well"><DIV class="container"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Aim"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Aim</SPAN></A></LI><LI class="toclevel-1"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2"><A href="#About_M13_Bacteriophage"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">About M13 Bacteriophage</SPAN></A></LI><LI class="toclevel-2"><A href="#Phage_display_and_phage_engineering"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Phage display and phage engineering</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Strategy:_in_vitro_directed_mutagenesis_by_PCR"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Strategy: in vitro directed mutagenesis by PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#Phage_genome_VCSM13"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Phage genome VCSM13</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_peptide_PIII"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">PCR of peptide PIII</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_PLux"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">PCR of PLux</SPAN></A></LI><LI class="toclevel-2"><A href="#pET28a_.28.2B.29_Phagemid"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">pET28a (+) Phagemid</SPAN></A></LI><LI class="toclevel-2"><A href="#Graphical_summary_of_the_strategy"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Graphical summary of the strategy</SPAN></A></LI><LI class="toclevel-1"><A href="#Biobricks"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Biobricks</SPAN></A></LI><LI class="toclevel-2"><A href="#p42_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366109.22.3EBBa_K1366109.3C.2Fa.3E.29"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">p42 (&lt;a href=&quot; http://parts.igem.org/Part:BBa_K1366109&quot;&gt;BBa_K1366109&lt;/a&gt;)</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Aim">Aim</SPAN></H1>&lt;section id=&quot;column&quot;&gt;<P> 
	  This module consists of two modifications made to the bacteriophage genome using PCR. The first change is the addition of a peptide to the PIII protein to modify the tropism of the phage, so it will be able to introduce itself into the cancerous cells. The other modification consisted of adding a pLux promoter before pII, as this will allow us to express the phage only when bacteria reach a certain cell density.
      </P>&lt;/section&gt;<H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><H2><SPAN class="mw-headline" id="About_M13_Bacteriophage"> About M13 Bacteriophage</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				Bacteriophages are a class of extremely specific viruses that infect bacteria and lack the ability to infect more complex organisms because of major differences in cell-surface molecules and in the intracellular machinery that is key for their replication [3]. To introduce themselves into the bacteria they attach to specific receptors on the surface of the host cell; the specificity of these interactions influences the bacterial host range [5].
				</P><P>
				M13 is an <EM>Escherichia coli</EM>-specific filamentous bacteriophage that consists of a single-stranded DNA core surrounded by a proteinaceous coat [6]. The genome of this virus contains 11 genes, five of which encode the coat proteins (PIII, PVI, PVII, PVIII and PIX), and the remaining are necessary for viral replication and assembly [3].The infection is initiated when PIII binds to the F pilus on the surface of an <EM>E. coli</EM> cell which facilitates the translocation of the viral DNA into the cytoplasm. Inside the host cell, the single stranded DNA is converted to a double-stranded form by PII. This form is further duplicated and used as the template for the production of all the phage proteins. Also, it is used as the template for the production of new single-stranded viral DNA through a “rolling circle” replication mode. Phage assembly is initiated by the incorporation of PIV and PIX at one end of the particle. The elongation process follows by the assembly of thousands of PVII molecules along the length of the particle. The process is terminated by the addition of PVI and PIII at the other end of the particle [3].
				</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Module2&quot; data-title=&quot;Fig. 2.1: Structure of M13 bacteriophage. pIII, the protein that determines the virus tropism is shown in purple. The blue protein shows the pep42 that is used by the phage to recognize the GRP78 receptor of the cancerous cells.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2.1: Structure of M13 bacteriophage. pIII, the protein that determines the virus tropism is shown in purple. The blue protein shows the pep42 that is used by the phage to recognize the GRP78 receptor of the cancerous cells. &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>&lt;/br&gt;
				The potential use of bacteriophages includes therapy for bacterial infections and anticancer treatments by modifying the microenvironment of tumors or serving as a platform for foreign peptides that may induce anticancer effects [3]. One of the first recorded uses of bacteriophages in therapy was in 1919 with the successful treatment of dysentery  in which four different patients were treated with a phage preparation. Likewise, in 1921, the use of these phages was found to be effective in the treatment of staphylococcal skin disease [1]. Another use of bacteriophages is the treatment of tumors. It was already reported by mid-1800s that tumor regression can occur during natural viral infections. For example, some viruses are known to be oncolytic; this means that their propagation in cancer cells is much more efficient than in normal cells. Others, like adenoviruses, can replicate effectively if the p53 gene is inactivated, which happens in at least 50% of human cancers [6].
				</P><P>
				However, the use of oncolytic viruses comes with its own risks. One of the methods to circumvent these dangers is the use of bacteriophages, which have been reported since 1940 to have antitumor activity in mice and rabbits [3]. By using these viruses, the excretion of cytokines can be expressed, which in turn can alter the immunosuppressive tumor microenvironment in order to promote tumor toxicity. At the same time, phage display can be used to increase specificity between the tumors and the bacteriophages [6].
				</P><H2><SPAN class="mw-headline" id="Phage_display_and_phage_engineering"> Phage display and phage engineering </SPAN></H2><P>
				This module’s focus is to use this method to modify the PIII coat protein so that the peptide pep42 is expressed. Pep42 is a cell-penetrating peptide that allows the use of phages as a vehicle for site-directed cancer therapy. In detail, it is a cyclic 13-mer oligopeptide (CTVALPGGYVRVC) that specifically binds to glucose-regulated protein 78 (GRP78) [1]. GRP78 is an intracellular chaperone and a member of the heat shock protein 70 (HSP70) family that provides a protective cellular response against stress conditions. Normal GRP78 expression is maintained at low levels but is upregulated in stress environments and induced in tumor environments. GRP78 overexpression has been reported in a variety of tumors, such as prostate, colon, skin, and breast cancers. This GRP78 overexpression represents a potentially attractive target as it results in the presence of GRP78 molecules on the cell surface of these cancer cells [1].
				</P><P>
				Another modification will be made by adding a Lux promoter upstream the PII protein to control the phage production under the quorum sensing system. This will allow us to have a control in the production of the phages. Our hypothesis is that if the bacteria can only reach high densities in the tumor area, the density needed to activate the Lux system will be reached only when the bacteria is in the tumor.
				</P><P>
				Further modification in the phage will be made by disrupting the f1 origin of replication. The f1 origin is the site in which the PII protein cleaves the DNA and allows to start the “rolling circle” replication mode that leads to the production of single stranded DNA. So by this disruption, the genome of the M13 will not be transformed into its single stranded form, which also means that will not be packaged by the coat proteins. A co-transformation of the bacteria with both the phage and a phagemid will cause the preferential packaging of the phagemid into the phage coat proteins. The phagemid will have the DNA of our effectors, the apoptin and the survivin siRNA (described in &lt;a href=&quot;module4.html&quot;&gt;Module IV&lt;/a&gt;), which give the phage the ability to become an anti-cancer vector. 
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Strategy:_in_vitro_directed_mutagenesis_by_PCR"> Strategy: in vitro directed mutagenesis by PCR </SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><H2><SPAN class="mw-headline" id="Phage_genome_VCSM13"> Phage genome VCSM13 </SPAN></H2><P>
				It’s genome has 8,669 bp which is longer than regular phages. It is noteworthy that this phage has a mutation in the oriF1, which is the site where the PII protein makes a cut and starts the rolling circle replication and the generation of single-stranded DNA (ssDNA), a process that will be interrupted by the mutation. However, the phage is still able to replicate because it has a bacterial origin of replication (P15A). As the phage cannot generate ssDNA, it will allow us to use with a phagemid, whose DNA will be packed in the phage.				
				</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;VCSM13&quot; data-title=&quot;Fig. 2.2: M13 bacteriophage genome&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2.2: M13 bacteriophage genome. &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><H2><SPAN class="mw-headline" id="PCR_of_peptide_PIII">PCR of peptide PIII </SPAN></H2><P>
				In order to change the tropism of the virus, we performed a PCR to insert the pep42 peptide to the phage pIII. This peptide will allow the phage to internalize into cancerous cells when it binds to the GRP78 receptor. Two pairs of primers were designed to contain the sequence encoding the peptide sequence; also, the restriction site KpnI was added in order to be subsequently re-ligated.
				</P><P>
				These are the primers that were used in this section:
				</P><P><UL id="primers"><LI><STRONG>F1:</STRONG> GTG GTA CCT GTA CCG TTG CGC TGC CGG GCG GTT ATG TGC GTG TTT GTG CTG AAA CTG TTG AAA GTT G </LI><LI><STRONG>R1:</STRONG> GTG GTA CCG GAG TGA GAA TAG AAA GGA AC </LI><LI><STRONG>F2:</STRONG> GTG GTA CCG CTG AAA CTG TTG AAA GTT G </LI><LI><STRONG>R2:</STRONG> GTG GTA CCA CAA ACA CGC ACA TAA CCG CCC GGC AGC GCA ACG GTA CAG GAG TGA GAA TAG AAA GGA AC </LI></UL></P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;VCSM13 pep42 circle&quot; data-title=&quot;Fig. 2.3: M13 bacteriophage genome with the addition of pep42&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2.3: M13 bacteriophage genome with the addition of pep42. &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><H2><SPAN class="mw-headline" id="PCR_of_PLux"> PCR of PLux </SPAN></H2><P>		PCR will be used as well to insert will promoter for the quorum sensing system (pLux) before the gene coding for PII. As bacteria are only able to colonize the tumour area, the cell density required to activate the pLux will be achieved when the bacteria are concentrated there. The way quorum sensing works is detailed in &lt;a href=&quot;module3.html&quot;&gt;Module III&lt;/a&gt;.				
				</P><P>
				The PCR employed two pairs of primers containing the sequence encoding the promoter, as well as the sequence of restriction sites XbaI and SpeI to subsequently re-link and be put on the pSB1C3 backbone to the match the iGEM BioBrick system.
				</P><P>
				These are the primers that were used in this section:
				</P><P><UL id="primers"><LI><STRONG>F1:</STRONG> GTT CTA GAA CCT GTA GGA TCG TAC AGG TTT ACG CAA GAA AAT GGT TTG TTA TAG TCG AAT AAA CAA TCT TCC TGT TTT TGG</LI><LI><STRONG>R1:</STRONG> GTA CTA GTG CCG AAA TCG GCA AAA TC </LI><LI><STRONG>F2:</STRONG> GTT CTA GAA CCT GTA GGA TCG TAC AGG TTT ACG CAA GAA AAT GGT TTG TTA TAG TCG AAT AAA GCT TTT CTG ATT ATC AAC CGG </LI><LI><STRONG>R2:</STRONG> GTA CTA GTC GAA ATC GGC AAA ATC CCT</LI></UL></P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;VCSM13 pep42 pLux FIN&quot; data-title=&quot;Fig. 2.4: M13 bacteriophage genome with the addition of pep42 and Lux promoter.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2.4: M13 bacteriophage genome with the addition of pep42 and Lux promoter. &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><H2><SPAN class="mw-headline" id="pET28a_.28.2B.29_Phagemid"> pET28a (+) Phagemid  </SPAN></H2><P>
				The phagemid has 5,369 bp and has an origin of replication F1, so it is able to generate ssDNA. In addition, it has the lacZ gene in the MCS, which uses to select the colonies that were transformed with the phagemid linked to the effector. The description of the effectors is detailed in &lt;a href=&quot;module4.html&quot;&gt;Module IV&lt;/a&gt;. 
				</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;pET-28a+&quot; data-title=&quot;Fig. 2.5: M13 bacteriophage genome with the addition of pep42 and Lux promoter.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block&quot; src=&quot;images/module2/pET_28a_Map2.png&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2.5: M13 bacteriophage genome with the addition of pep42 and Lux promoter. &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;&lt;/section&gt;</P><H2><SPAN class="mw-headline" id="Graphical_summary_of_the_strategy">Graphical summary of the strategy </SPAN></H2><P> To see the graphical summary of the strategy &lt;a href=&quot;&quot; data-lightbox=&quot;Graphical Summary&quot; data-title=&quot;Fig. 2.6: Graphical summary of the strategy.&quot;&gt;Click here&lt;/a&gt;.
	</P><H1><SPAN class="mw-headline" id="Biobricks">Biobricks</SPAN></H1><H2><SPAN class="mw-headline" id="p42_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366109.22.3EBBa_K1366109.3C.2Fa.3E.29">p42 (&lt;a href=&quot; <A href="http://parts.igem.org/Part:BBa_K1366109" class="external free" rel="nofollow">http://parts.igem.org/Part:BBa_K1366109</A>&quot;&gt;BBa_K1366109&lt;/a&gt;)</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;&lt;p&gt;
		This part allows the production of a recombinant pIII protein linked with a pep42 peptide. This peptide change the tropism of the virus M13 and enables it to bind to a receptor called GRP78 that it is highly present in cancer cells. The binding between pep42 and GRP78 receptor allows the bacteriophage to penetrate the cancer cells and this initiates the transfection of DNA mediated with bacteriophages.
</P>&lt;/section&gt;<H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P>
	  *
      </P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><OL><LI><P>
		Arap, M. A., Lahdenranta, J., Mintz, P. J., Hajitou, A., Sarkis, Á. S., Arap, W., &amp; Pasqualini, R. (2004). Cell surface expression of the stress response chaperone GRP78 enables tumor targeting by circulating ligands. <EM>Cancer cell,6</EM>(3), 275-284.
		</P></LI><LI><P>
		Bar, H., Yacoby, I., &amp; Benhar, I. (2008). Killing cancer cells by targeted drug-carrying phage nanomedicines. <EM>BMC biotechnology, 8</EM>(1), 37.
		</P></LI><LI><P>
		Bloch, H. (1940). Experimental investigation on the relationships between bacteriophages and malignant tumors. <EM>Arch Virol, 1,</EM> 481-496.
		</P></LI><LI><P>
		Budynek, P., Dąbrowska, K., Skaradziński, G., &amp; Górski, A. (2010). Bacteriophages and cancer. <EM>Archives of microbiology, 192(5)</EM>, 315-320.
		</P></LI><LI><P>
		Drulis-Kawa, Z., Majkowska-Skrobek, G., Maciejewska, B., Delattre, A. S., &amp; Lavigne, R. (2012). Learning from bacteriophages-advantages and limitations of phage and phage-encoded protein applications. <EM>Current protein &amp; peptide science, 13</EM>(8), 699.
		</P></LI><LI><P>
		Eriksson, F., Tsagozis, P., Lundberg, K., Parsa, R., Mangsbo, S. M., Persson, M. A., ... &amp; Pisa, P. (2009). Tumor-specific bacteriophages induce tumor destruction through activation of tumor-associated macrophages. <EM>The Journal of Immunology, 182</EM>(5), 3105-3111.
		</P></LI><LI><P>
		Frost, L. S. (1993). Conjugative pili and pilus-specific phages. <EM>In Bacterial conjugation</EM> (pp. 189-221). Springer US.
		</P></LI><LI><P>
		Marvin, D. A. (1998). Filamentous phage structure, infection and assembly.<EM>Current opinion in structural biology, 8</EM>(2), 150-158.
		</P></LI><LI><P>
		Sidhu, S. S. (2001). Engineering M13 for phage display. <EM>Biomolecular engineering, 18</EM>(2), 57-63.
		</P></LI><LI><P>
		Sinkovics, J. G., &amp; Horvath, J. C. (2008). Natural and genetically engineered viral agents for oncolysis and gene therapy of human cancers. <EM>Archivum immunologiae et therapiae experimentalis, 56</EM>(1), 1-59.
		</P></LI><LI><P>
		Smith, G. P., &amp; Petrenko, V. A. (1997). Phage display. <EM>Chemical reviews, 97</EM>(2), 391-410.
		</P></LI><LI><P>
		Sulakvelidze, A., Alavidze, Z., &amp; Morris, J. G. (2001). Bacteriophage therapy.<EM>Antimicrobial agents and chemotherapy, 45</EM>(3), 649-659.
		</P></LI></OL><P>&lt;/section&gt;</P></DIV></DIV></DIV><P>&lt;/body&gt;</P><P>&lt;footer id=&quot;wiki-footer&quot; &gt;&lt;/footer&gt;</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Tec-Monterrey/module2.html">http://2014.igem.org/Team:Tec-Monterrey/module2.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tec-Monterrey/module2.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tec-Monterrey/module2.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tec-Monterrey/module2.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tec-Monterrey/module2.html&amp;oldid=385644" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>