<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tec-Monterrey_ITESM14_module1_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tec-Monterrey/ITESM14_module1.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tec-Monterrey/ITESM14_module1.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_module1.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_module1.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tec-Monterrey/ITESM14_module1.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tec-Monterrey/ITESM14 module1.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wiki-body"><P>&lt;body&gt;</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Aim"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Aim</SPAN></A></LI><LI class="toclevel-1"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2"><A href="#Escherichia_coli_and_cancer"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Escherichia coli and cancer</SPAN></A></LI><LI class="toclevel-2"><A href="#Endotoxins"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Endotoxins</SPAN></A></LI><LI class="toclevel-2"><A href="#Membrane_engineering"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Membrane engineering</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Biobricks"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Biobricks</SPAN></A></LI><LI class="toclevel-2"><A href="#C1_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366101.22_target.3D.22_blank.22.3E_BBa_K1366101.3C.2Fa.3E.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">C1 (&lt;a href=&quot; http://parts.igem.org/Part:BBa_K1366101&quot; target=&quot;_blank&quot;&gt; BBa_K1366101&lt;/a&gt;)</SPAN></A></LI><LI class="toclevel-2"><A href="#C2_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366102.22_target.3D.22_blank.22.3E_BBa_K1366102.3C.2Fa.3E.29"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">C2 (&lt;a href=&quot; http://parts.igem.org/Part:BBa_K1366102&quot; target=&quot;_blank&quot;&gt; BBa_K1366102&lt;/a&gt;)</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Aim">Aim</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><P>
		   The project this year is a continuation of last year’s project &lt;a href=&quot;<A href="https://2013.igem.org/Team:TecMonterrey" class="external free" rel="nofollow">https://2013.igem.org/Team:TecMonterrey</A>&quot; target=&quot;_blank&quot;&gt;iGem Tecnológico de Monterrey 2013&lt;/a&gt; in which the team developed a modified <EM>Escherichia coli</EM> that was able to deliver a pro-apoptotic protein (Apoptin) to kill cancer cells. The aim of this module was to address one of the most relevant issues of our past project: the immune response caused by the introduction of bacteria to the blood. Thus, we attempted to remove a significant amount of endotoxins found on the membrane of <EM>E. coli</EM> in order to prevent and/or reduce the impact of the possible immunological response caused by the bacterial therapy.  &lt;/b&gt;</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><H2><SPAN class="mw-headline" id="Escherichia_coli_and_cancer"><EM>Escherichia coli</EM> and cancer</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				Last year our team developing a bacterial therapy, based on the fact that <EM>E. coli</EM> it’s able to colonize hypoxic environments, such as tumors.Nevertheless, the risk of causing septic shock was a significant issue. Thus, in this project we aimed to find a solution for this problem.
				</P><P>&lt;/section&gt;</P><H2><SPAN class="mw-headline" id="Endotoxins"> Endotoxins</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				Endotoxins are outer membrane components native to gram negative bacteria, specifically, for project matters,<EM>Escherichia coli</EM>. Endotoxins are a primary concern in project design due to adverse reactions that these can trigger from the human system, in particular, septic shock [1].This module is focused on two different endotoxins: lipopolysaccharides and murein lipoprotein.  
				</P><P>
				Lipopolysaccharide (LPS) are composed of a membrane side in lipid, (Lipid A) and an outer polysaccharide (0-Antigen), these are joined by a core polysaccharide region [6]. LPS rests in the outer bacterial cell membrane and is known to cause most of the endotoxin-related inflammations in humans. 
				</P><P>
				Murein lipoprotein (Braun’s lipoprotein) endotoxin consists of peptidoglycan and a lipid subsection, bound together by lysine. It serves a structural purpose within the cell, holding the rigid membrane together due to its strongly interacting bonds [1]. Furthermore, it induces a LPS-like reaction, due to toll-like receptor 2 recognition of Braun’s Lipoprotein [5]. Additionally, it works in synergy with LPS in septic shock rising events [10].			
				</P><P>
				The immune system is able to detect Endotoxins through specific toll-like receptors [5]. LPS interacts with TLR4 and its coreceptor, myeloid differentiation protein 2. Braun’s lipoprotein binds with TLR2. Both Lpp and LPS induce toxic and biological responses within hosts through the receptor binding and consequent induction of innate immune system production of cytokines such as tumor necrosis factor alpha, gamma interferon, interleukin-1-beta, and IL-6 [9], these of which are prone to trigger inflammation when produced in great amounts.
				</P><P>&lt;/section&gt;</P><H2><SPAN class="mw-headline" id="Membrane_engineering">Membrane engineering</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;&lt;/br&gt;&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Modulo1blanco&quot; data-title=&quot;Fig. 1.2: Membrane engineering: Gram negative bacterial membrane. The LPS is the outermost layer of the structure while the Braun lipoprotein binds the LPS with the cell wall (r). msbB and lpp genes will be deleted using the Red Lambda system.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 1.2: Membrane engineering: Gram negative bacterial membrane. The LPS is the outermost layer of the structure while the Braun lipoprotein <U>binds</U> the LPS with the cell wall (r). <EM>msbB</EM> and <EM>lpp</EM> genes will be deleted using the Red Lambda system.    &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>&lt;/br&gt;
				Engineering <EM>E. coli’s</EM> membrane to remove LPS and Braun’s lipoprotein reduces the chance of toll-like receptors 4 and 2 to react abruptly. Decreasing the concentration of these endotoxins will inhibit the triggering of their corresponding toll-like receptors, subsequently reducing inflammation. This will reduce further impairment due to auto-immune complications in the individual receiving bacterial therapy. [2]. 
				</P><P>
				For the endotoxin removal, both genes, msbb and lpp will be deleted from <EM>E.coli’s</EM> genome. These genes are, naturally, the most relevant to the production of inflammation factors msbB for the impairment of meristic acid transporter formation crucial in LPS toxicity and lpp Braun’s Lipoprotein. The phage lambda-derived Red recombination system was chosen for the engineering of <EM>E. coli’s</EM> genome due to its highly specific sequence targeting and simplicity. For a detailed procedure of the experimental protocol of how the red lambda system works &lt;a href=&quot;<A href="http://cdn.b-bridge.com/products/K003_Protocol.pdf" class="external free" rel="nofollow">http://cdn.b-bridge.com/products/K003_Protocol.pdf</A>&quot; target=&quot;_blank&quot;&gt; click here&lt;/a&gt;.
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Biobricks">Biobricks</SPAN></H1><P>
		Two constructs, C1 and C2, which are required by the red lambda system were designed for the gene knockout. These contain antibiotic resistance genes, and the LuxR or LuxI sequences flanked by 50 bp homologous to the genes <EM>msBb</EM> and <EM>lpp</EM> to be deleted.  The purpose of these sequences is detailed in &lt;a href=&quot;#tab_module3&quot; target=&quot;_blank&quot;&gt; Module 3 &lt;/a&gt;.
		</P><H2><SPAN class="mw-headline" id="C1_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366101.22_target.3D.22_blank.22.3E_BBa_K1366101.3C.2Fa.3E.29"><STRONG> C1 </STRONG>(&lt;a href=&quot; <A href="http://parts.igem.org/Part:BBa_K1366101" class="external free" rel="nofollow">http://parts.igem.org/Part:BBa_K1366101</A>&quot; target=&quot;_blank&quot;&gt; BBa_K1366101&lt;/a&gt;)</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;</P><P>
		This construct contains the sequence homologies of 50 bp to delete the lpp gene (Braun’s lipoprotein). It includes an ampicillin resistance gene and the sequences for the LuxR synthesis (when this protein is coupled with the homoserine lactone (AHL), it activates the transcription of genes controlled by the lux box). The flippase recognition target (FRT) sequences included are used to remove the antibiotic resistance with a specific recombinase. For more detail on the role of AHL and FRT, refer to &lt;a href=&quot;#tab_module3&quot; target=&quot;_blank&quot;&gt; Module 3 &lt;/a&gt;.
		</P><P>
		Its general structure is as follows:
		</P><P>
		
		lpp // FRT // Amp Promoter // RBS // AmpR // Terminator // FRT // Promoter J23100// RBS// LuxR // Terminator // lpp
		&lt;/br&gt;&lt;/br&gt;</P><P>&lt;figure&gt;</P><P><SMALL>(a)</SMALL></P><P>&lt;a href=&quot;&quot; data-lightbox=&quot;C1_Map&quot; data-title=&quot;Fig. 1.3: (a) Structure of construct 2, that contains an ampicilin resistance gene and a LuxR protein.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><P><SMALL>(b)</SMALL></P><P>&lt;a href=&quot;&quot; data-lightbox=&quot;Lpp&quot; data-title=&quot;Fig. 1.3: (b) Segment of the E. coli genome where the insertion will be made; the homologous bases are colored pink.&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Figure 1.3: (a) Structure of construct 2, that contains an ampicilin resistance gene and a LuxR protein. (b) Segment of the E. coli genome where the insertion will be made; the homologous bases are colored pink. &lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>&lt;/section&gt;</P><H2><SPAN class="mw-headline" id="C2_.28.3Ca_href.3D.22_http:.2F.2Fparts.igem.org.2FPart:BBa_K1366102.22_target.3D.22_blank.22.3E_BBa_K1366102.3C.2Fa.3E.29"><STRONG> C2 </STRONG>(&lt;a href=&quot; <A href="http://parts.igem.org/Part:BBa_K1366102" class="external free" rel="nofollow">http://parts.igem.org/Part:BBa_K1366102</A>&quot; target=&quot;_blank&quot;&gt; BBa_K1366102&lt;/a&gt;)</SPAN></H2><P>&lt;section id=&quot;column&quot;&gt;</P><P>
		  This construct contains the sequence homologies to delete the msbB gen (LPS myristic acid moiety carrier), a kanamycin resistance gene and the sequences for the Lux I production (AHL synthase). 		  </P><P>
		  Its general structure is as follows:
		  </P><P>
		  msBb // FRT // Kan Promoter// RBS // KanR // Terminator // FRT // Cat Promoter // RBS// LuxI // Terminator // msBb
		  &lt;/br&gt;&lt;/br&gt;&lt;/br&gt;&lt;/br&gt;</P><P>&lt;figure&gt;</P><P><SMALL>(a)</SMALL></P><P>&lt;a href=&quot;&quot; data-lightbox=&quot;C2_Map&quot; data-title=&quot;Fig. 1.4: (a) Structure of construct 2, that contains an ampicilin resistance gene and a LuxR protein.&quot;&gt;&lt;img class=&quot;img img-responsive&quot;style=&quot;margin:0px auto;display:block &quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><P><SMALL>(b)</SMALL></P><P>&lt;a href=&quot;&quot; data-lightbox=&quot;msbB&quot; data-title=&quot;Fig. 1.4: (b) Segment of the E. coli genome where the insertion will be made; the homologous bases are colored pink.&quot;&gt;&lt;img class=&quot;img img-responsive&quot;style=&quot;margin:0px auto;display:block &quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Figure 1.4: (a) Structure of construct 2, that contains an ampicilin resistance gene and a LuxR protein. (b) Segment of the E. coli genome where the insertion will be made; the homologous bases are colored pink.&lt;/Figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><OL><LI><P>
			Braun, V., &amp; Rehn, K. (1969). Chemical characterization, spatial distribution and function of a lipoprotein (murein-lipoprotein) of the E. coli cell wall.<EM> European Journal of Biochemistry, 10(3)</EM>, 426-438.
			</P></LI><LI><P>
			Fadl, A. A., Sha, J., Klimpel, G. R., Olano, J. P., Niesel, D. W., &amp; Chopra, A. K. (2005). Murein lipoprotein is a critical outer membrane component involved in Salmonella enterica serovar Typhimurium systemic infection.<EM> Infection and immunity, 73(2)</EM>, 1081-1096.		
			</P></LI><LI><P>
			Morrison, D. C., &amp; Ulevitch, R. J. (1978). The effects of bacterial endotoxins on host mediation systems. A review.<EM>The American journal of pathology, 93(2)</EM> , 526.
			</P></LI><LI><P>
			Mosberg, J. A., Lajoie, M. J., &amp; Church, G. M. (2010). Lambda red recombineering in Escherichia coli occurs through a fully single-stranded intermediate.<EM>Genetics, 186(3)</EM>, 791-799. 
			</P></LI><LI><P>
			Neilsen, P. O., Zimmerman, G. A., &amp; McIntyre, T. M. (2001). Escherichia coli Braun lipoprotein induces a lipopolysaccharide-like endotoxic response from primary human endothelial cells.<EM>The Journal of Immunology, 167(9),</EM>5231-5239. 
			</P></LI><LI><P>
			Rietschel, E. T., Kirikae, T., Schade, F. U., Mamat, U., Schmidt, G., Loppnow, H., ... &amp; Di Padova, F. (1994). Bacterial endotoxin: molecular relationships of structure to activity and function.<EM>The FASEB Journal, 8(2)</EM>, 217-225.
			</P></LI><LI><P>
			Sha, J., Fadl, A. A., Klimpel, G. R., Niesel, D. W., Popov, V. L., &amp; Chopra, A. K. (2004). The two murein lipoproteins of Salmonella enterica serovar Typhimurium contribute to the virulence of the organism.<EM> Infection and immunity, 72(7)</EM>, 3987-4003. 
			</P></LI><LI><P>
			Stritzker, J., Hill, P. J., Gentsche, I., &amp; Szalay, A. A. (2010). Myristoylation negative msbB-mutants of probiotic E. coli Nissle 1917 retain tumor specific colonization properties but show less side effects in immunocompetent mice.<EM>Bioengineered, 1(2)</EM>, 139-145. 
			</P></LI><LI><P>
			Ulevitch, R. J., &amp; Tobias, P. S. (1995). Receptor-dependent mechanisms of cell stimulation by bacterial endotoxin.<EM> Annual review of immunology, 13(1)</EM>, 437-457.  
			</P></LI><LI><P>
			Zhang, H., Peterson, J. W., Niesel, D. W., &amp; Klimpel, G. R. (1997). Bacterial lipoprotein and lipopolysaccharide act synergistically to induce lethal shock and proinflammatory cytokine production. <EM> The Journal of Immunology, 159(10)</EM>, 4868-4878. 
			</P></LI></OL><P>&lt;/section&gt;</P></DIV><PRE>&lt;/div&gt;</PRE><P>&lt;/div&gt;</P><P>&lt;/body&gt;</P><P>&lt;/div&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Tec-Monterrey/ITESM14_module1.html">http://2014.igem.org/Team:Tec-Monterrey/ITESM14_module1.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tec-Monterrey/ITESM14_module1.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tec-Monterrey/ITESM14_module1.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_module1.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_module1.html&amp;oldid=393817" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>