<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tec-Monterrey_ITESM14_protocols_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tec-Monterrey/ITESM14_protocols.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tec-Monterrey/ITESM14_protocols.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_protocols.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_protocols.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tec-Monterrey/ITESM14_protocols.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tec-Monterrey/ITESM14 protocols.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wiki-body"><DIV id="wiki-content" class="main-content"><P style="margin-top:1%; font-size:1.5vw;">
  Here you can find a detailed description of the different experimental protocols that were used for the making of our project.
</P><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1">
          Protocol 1-Calcium competent cells
        </A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">Around 5 hours (depends in the growth rate of the strain) </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">100 mM CaCl<SUB>2</SUB></TD></TR><TR><TD colspan="3">200mL LB </TD></TR><TR><TD colspan="3">4 sterile centrifuge tubes</TD></TR><TR><TD colspan="3">Overnight <EM>E. coli</EM> culture</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Centrifuge (Thermo)</TD></TR><TR><TD colspan="3">Spectrometer (Thermo) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TD colspan="3">Incubator with shaker (TalBoys)</TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Autoclave.</U> 200mL of LB media placed in a 1L flask, Glycerol, 4 centrifuge tubes, microcentrifuge tubes, 1L of 100 mM CaCl<SUB>2</SUB>.</TD></TR><TR><TD colspan="3"><U>Chill overnight at 4°C.</U> 100 mM CaCl<SUB>2</SUB>.</TD></TR><TR><TD colspan="3"><U>Prepare starter culture cells.</U> Inoculate a 25mL starter culture of LB (no antibiotics). Grow culture at 37°C in shaker overnight.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Inoculate 200mL of LB media with approximately 20mL starter culture. Measure the OD600 to make sure it is around 0.1</TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Grow in 37°C shaker.</TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Measure the OD<SUB>600</SUB> every 30 minutes. When the OD<SUB>600</SUB> reaches 0.4-0.6, immediately put the cells on ice. Chill the culture for 30 minutes.</TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> Harvest the cells by centrifugation at ~4000 rpm for 15 minutes at 4°C.</TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Decant the supernatant and gently resuspend each pellet in about 30mL of ice cold MgCl<SUB>2</SUB>.</TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Put the cells on ice and chill the culture for 30 minutes.</TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Repeat two times the steps 4-6 varying the incubation time by 20 and 15 minutes respectively.</TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Harvest the cells by centrifugation at ~4000 rpm for 15 minutes at 4°C.</TD></TR><TR><TD align="right">9.</TD><TD colspan="2">Decant the supernatant and gently resuspend the pellet of one of the tubes in 500μL of ice cold MgCl<SUB>2</SUB> and 15% of glycerol. </TD></TR><TR><TD align="right">10.</TD><TD colspan="2">Once the pellet is completely resuspended, transfer the solution into the next tube and resuspend the pellet. Repeat this step for all the pellets. </TD></TR><TR><TD align="right">11.</TD><TD colspan="2">Aliquot 50μL into sterile 1.5 mL microfuge tubes and store at -80°C freezer.</TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB medium</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>CaCl solution</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Tips</CENTER></TD><TD><CENTER>Place in a container and with chlorine solution then sterilize</CENTER></TD><TD><CENTER>Be ordered and use the container all the time</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesSeidman, C. E., Struhl, K., Sheen, J., &amp; Jessen, T. (2005). Introduction of plasmid DNA into cells. <EM>Current protocols in molecular biology</EM>, 1-8. Edited byMercedes VázquezDateJuly 1st, 2013RevisionSeptember 10th, 2014</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2">
          Protocol 2-Transformation of Calcium Competent Cells
        </A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Approximate time</TH></TR><TR><TD colspan="3"> 90 min </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Liquid LB media </TD></TR><TR><TD colspan="3">Petri dishes </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TD colspan="3">Water bath (Thermo) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Autoclave.</U> LB liquid media, LB-agar media. 
              </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Mix 1 to 5μl of DNA (usually 10pg to 100ng) into 50μL of competent cells in a microcentrifuge tube. </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Place the competent cell/DNA mixture on ice for 20min.</TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Heat shock each transformation tube by placing the bottom of the tube into a 42°C water bath for 45 seconds (time varies depending on the competent cells you are using). </TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> Put the tubes back on ice for 2 min. </TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Add 500μl LB media (without antibiotic) and grow in 37°C shaking incubator for 1 hour. </TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Plate 100 μl of the transformation onto a 10cm LB agar plate containing the appropriate antibiotic. </TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Incubate plates at 37°C overnight. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Petri dishes with culture</CENTER></TD><TD><CENTER>Place in a bag to sterilize </CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Liquid LB with culture</CENTER></TD><TD><CENTER>Place in a container and with chlorine solution then sterilize</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesSeidman, C. E., Struhl, K., Sheen, J., &amp; Jessen, T. (2005). “Basic Protocol 1: Transformation Using Calcium Chloride.” Introduction of plasmid DNA into cells. <EM>Current protocols in molecular biology</EM>, 1-8.  Edited byMercedes VázquezDateJuly 1st, 2013RevisionSeptember 10th, 2014</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3">
          Protocol 3-Miniprep
        </A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3"> To isolate a plasmid from an <EM>E. coli</EM> culture </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">60-180 min (depends in the incubation time) </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Solution I. (50 mM TRIS pH 8.0, 10 mM EDTA) </TD></TR><TR><TD colspan="3">Solution II (200 mM NaOH, 1% w/v SDS) </TD></TR><TR><TD colspan="3">Solution III (3 M potassium acetate, pH 5.5) </TD></TR><TR><TD colspan="3">EtOH 100% </TD></TR><TR><TD colspan="3">EtOH 70% </TD></TR><TR><TD colspan="3">ddH<SUB>2</SUB>O</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Nanodrop (Thermo) </TD></TR><TR><TD colspan="3">Centrifuge (BioRad) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Prepare starter culture cells. </U>  Inoculate a 15mL starter culture of LB with the cells that have the desired plasmid (with the appropriate antibiotic). Grow culture at 37°C in shaker overnight. 
                </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Centrifuge the starter culture. </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Decant the supernatant and resuspend the pellet in about 1mL of LB media </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Transfer the cells into a Eppendorf tube. </TD></TR><TR><TD align="right">4.</TD><TD colspan="2">Centrifuge 14,000 rpm / 1 minute. </TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Decant the supernatant.</TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Add 200uL of solution I.</TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Use a pipette to resuspend the pellet.</TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Add 2uL of RNAse.</TD></TR><TR><TD align="right">9.</TD><TD colspan="2">Incubate 5min at room temperature.</TD></TR><TR><TD align="right">10.</TD><TD colspan="2">Add 200uL of Solution II.</TD></TR><TR><TD align="right">11.</TD><TD colspan="2">Mix gently by inversion 6 times. </TD></TR><TR><TD align="right">12.</TD><TD colspan="2">Incubate 5 min at room temperature. </TD></TR><TR><TD align="right">13.</TD><TD colspan="2">Add 200uL of Solution III. </TD></TR><TR><TD align="right">14.</TD><TD colspan="2">Mix gently by inversion 6 times. </TD></TR><TR><TD align="right">15.</TD><TD colspan="2">Incubate 5 min on ice. </TD></TR><TR><TD align="right">16.</TD><TD colspan="2">Centrifuge 14,000 rpm for 10 min. </TD></TR><TR><TD align="right">17.</TD><TD colspan="2">Transfer the supernatant to a new tube (around 500uL). </TD></TR><TR><TD align="right">18.</TD><TD colspan="2">Add 2 volumes cool of 100% EtOH (around 1mL). </TD></TR><TR><TD align="right">19.</TD><TD colspan="2">Incubate at -20°C for 10 min (can be from 10 min to 2 hours). </TD></TR><TR><TD align="right">20.</TD><TD colspan="2">Centrifuge 14,000 rpm for 10 min. </TD></TR><TR><TD align="right">21.</TD><TD colspan="2">Decant the supernatant. </TD></TR><TR><TD align="right">22.</TD><TD colspan="2">Add 200uL of cool 70% EtOH. </TD></TR><TR><TD align="right">23.</TD><TD colspan="2">Use a pipette to resuspend the pellet. </TD></TR><TR><TD align="right">24.</TD><TD colspan="2">Centrifuge 14,000 rpm for 5 min. </TD></TR><TR><TD align="right">25.</TD><TD colspan="2">Decant the supernatant. </TD></TR><TR><TD align="right">26.</TD><TD colspan="2">Dry pellet at 37 ° C for 5 min. </TD></TR><TR><TD align="right">27.</TD><TD colspan="2">Add 30uL of ddH<SUB>2</SUB>O and resuspend pellet. </TD></TR><TR><TD align="right">28.</TD><TD colspan="2">Measure DNA concentration in nanodrop. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Tube with cell pellets</CENTER></TD><TD><CENTER>Pour chlorine solution and clean the tube</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Eppendorf with protein waste</CENTER></TD><TD><CENTER>Dispose the tube in normal container</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Gloves must be worn at all times</LI><LI>Can be done safely outside the hood</LI><LI>Try to keep the tubes closed to avoid contamination</LI></OL>ReferencesQiagen. (2009, November 20). Miniprep/Qiagen kit protocol. Retrieved June 2, 2014, from http://openwetware.org/wiki/Miniprep/Qiagen_kit_protocol Edited byMercedes VázquezDateJuly 1st, 2013RevisionSeptember 10th, 2014</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse4">
          Protocol 4-Electro-competent cells
        </A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">Around 5 hours (depends in the growth rate of the strain)  </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Sterile dH<SUB>2</SUB>O</TD></TR><TR><TD colspan="3">200mL LB </TD></TR><TR><TD colspan="3">4 sterile centrifuge tubes</TD></TR><TR><TD colspan="3">Overnight <EM>E. coli</EM> culture</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Centrifuge (Thermo)</TD></TR><TR><TD colspan="3">Spectrometer (Thermo) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TD colspan="3">Incubator with shaker (TalBoys)</TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Autoclave.</U> 200mL of LB media placed in a 1L flask, Glycerol, 4 centrifuge tubes, microcentrifuge tubes, 1L of sterile dH<SUB>2</SUB>O. </TD></TR><TR><TD colspan="3"><U>Chill overnight at 4°C.</U> 100 mM CaCl<SUB>2</SUB>.</TD></TR><TR><TD colspan="3"><U>Prepare starter culture cells.</U> Inoculate a 25mL starter culture of LB (no antibiotics). Grow culture at 37°C in shaker overnight.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Inoculate 200mL of LB media with approximately 20mL starter culture. Measure the OD<SUB>600</SUB> to make sure it is around 0.1 </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Grow in 37°C shaker.</TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Measure the OD<SUB>600</SUB> every 30 minutes. When the OD<SUB>600</SUB> reaches 0.4-0.6, immediately put the cells on ice. Chill the culture for 30 minutes.</TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> Harvest the cells by centrifugation at ~4000 rpm for 15 minutes at 4°C.</TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Decant the supernatant and gently resuspend each pellet in about 30mL of ice cold dH<SUB>2</SUB>O.</TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Put the cells on ice and chill the culture for 30 minutes.</TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Repeat two times the steps 4-6 varying the incubation time by 20 and 15 minutes respectively. </TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Harvest the cells by centrifugation at ~4000 rpm for 15 minutes at 4°C. </TD></TR><TR><TD align="right">9.</TD><TD colspan="2">Decant the supernatant and gently resuspend the pellet of one of the tubes in 500μL of ice cold dH<SUB>2</SUB>O and 15% of glycerol.  </TD></TR><TR><TD align="right">10.</TD><TD colspan="2">Once the pellet is completely resuspended, transfer the solution into the next tube and resuspend the pellet. Repeat this step for all the pellets. </TD></TR><TR><TD align="right">11.</TD><TD colspan="2">Aliquot 50μL into sterile 1.5 mL microfuge tubes and store at -80°C freezer. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB medium</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>dH<SUB>2</SUB>O </CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Tips</CENTER></TD><TD><CENTER>Place in a container and with chlorine solution then sterilize</CENTER></TD><TD><CENTER>Be ordered and use the container all the time</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesSeidman, C. E., Struhl, K., Sheen, J., &amp; Jessen, T. (2005). Introduction of plasmid DNA into cells. <EM>Current protocols in molecular biology</EM>, 1-8. Edited byMercedes VázquezDateJuly 1st, 2013RevisionSeptember 10th, 2014</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse5">
          Protocol 5-Transformation of electro-competent cells
        </A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">Around 2 hours </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">1 mL, 100µL and 10µL Micropipettes </TD></TR><TR><TD colspan="3">Liquid LB media  </TD></TR><TR><TD colspan="3">Petri dishes </TD></TR><TR><TD colspan="3">Electroporation cells </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions)  </TD></TR><TR><TD colspan="3">Electroporator (BioRad)  </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Autoclave.</U> LB liquid media (500 µL per transformation), LB-agar media (one control per antibiotic). 
              </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">In the flow hood, place 1 µL of DNA in one side of the cell.</TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Using the pipette pour 50 µL of the competent cells in the same side of the electroporation cell, so the DNA and cells will mix and reach the bottom </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Electroporate the cell outside the hood using a potential of 1.8 mV </TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> If the transformation is successful, a curve will appear on the screen of the electroporator otherwise an “arc” sign will appear.</TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Using the micropipette, place 500 µL of LB media in the cell. </TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Add 500μl LB media (without antibiotic) and grow in 37°C shaking incubator for 1 hour. </TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Plate 100 μl of the transformation onto a 10cm LB agar plate containing the appropriate antibiotic. </TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Incubate plates at 37°C overnight. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Petri dishes with culture</CENTER></TD><TD><CENTER>Place in a bag to sterilize</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Liquid LB with culture</CENTER></TD><TD><CENTER>Place in a container and with chlorine solution then sterilize</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Tips</CENTER></TD><TD><CENTER>Place in a container and with chlorine solution then sterilize</CENTER></TD><TD><CENTER>Be ordered and use the container all the time</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesSeidman, C. E., Struhl, K., Sheen, J., &amp; Jessen, T. (2005). Introduction of plasmid DNA into cells. <EM>Current protocols in molecular biology</EM>, 1-8. Edited byMercedes VázquezDateJuly 1st, 2013RevisionSeptember 10th, 2014</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse6">
          Protocol 6-Digestion with restriction enzymes
        </A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">Analyze DNA constructs and assemblies using restriction enzymes.</TD></TR><TR><TD colspan="3">Cut DNA fragments in order to ligate them.  </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">2 hrs </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">PCR tubes </TD></TR><TR><TD colspan="3">DNA sample  </TD></TR><TR><TD colspan="3">Restryction Enzymes (EcoRI, SpeI, PstI, XbaI) </TD></TR><TR><TD colspan="3">NEB Enzyme buffer </TD></TR><TR><TD colspan="3">Nuclease free dH<SUB>2</SUB>O  </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Thermocycler (BioRad) </TD></TR><TR><TD colspan="3">Nanodrop (Thermo)  </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3">Measure the DNA concentration of the sample using the Nanodrop. </TD></TR><TR><TD colspan="3">Calculate the required concentrations of the components depending on the DNA of the sample, and the reaction volume. </TD></TR><TR><TD colspan="3">Ideally you must put 1 enzyme unit per each µg of DNA, yet the provider recommends using 10 times that quantity and adjust the volumes. <STRONG>Our most typical reaction was set up to be 20 µL. </STRONG></TD></TR><TR><TH>Reaction volume</TH><TH>Restriction Enzyme</TH><TH>DNA</TH><TH>10X BEBuffer</TH></TR><TR><TD><CENTER>10 µl rxn </CENTER></TD><TD><CENTER>1 unit</CENTER></TD><TD><CENTER>0.1 µg</CENTER></TD><TD><CENTER>1 µl</CENTER></TD></TR><TR><TD><CENTER>25 µl rxn</CENTER></TD><TD><CENTER>5 units</CENTER></TD><TD><CENTER>0.5 µg</CENTER></TD><TD><CENTER>2.5 µl</CENTER></TD></TR><TR><TD><CENTER>50 µl rxn</CENTER></TD><TD><CENTER>10 units</CENTER></TD><TD><CENTER>1 µg</CENTER></TD><TD><CENTER>5 µl</CENTER></TD></TR></TBODY></TABLE></CENTER>Steps1.Set and label your PCR tubes. 2.With the micropipette first pour the nuclease free dH<SUB>2</SUB>O. 3.Pour the NEB buffer 10X to get the 1X concentration.4. Put the DNA sample next, this sample must be as clean as possible. 5.<STRONG>The enzymes must be put lastly</STRONG>, be careful to handle them because they are very temperature sensitive. <STRONG>(Handle with ice and return them to the fridge as soon as possible)</STRONG>.6.Set up the Thermocycler at 37º for 1h and inactivate the enzyme at 80º for 20 mins.7.Analyze using agarose electrophoresis.Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Digestion waste in PCR tube </CENTER></TD><TD><CENTER>Place in trash container.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Gloves must be worn at all times. </LI><LI>Handle the enzymes carefully. </LI><LI> Typically the digestion will be used to confirm or to use the digestion to ligate. The first could generate some waste, but the second usually won't.  </LI></OL>ReferencesDouble Digest Protocol with Standard Restriction Enzymes <A href="https://www.neb.com/protocols/2014/05/07/double-digest-protocol-with-standard-restriction-enzymes" target="_blank"> https://www.neb.com/protocols/2014/05/07/double-digest-protocol-with-standard-restriction-enzymes</A>Optimizing Restriction Endonuclease Reactions <A href="https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions" target="_blank"> https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions</A>Digestion Protocol for BioBrick Assembly Kit (E0546)<A href="https://www.neb.com/protocols/1/01/01/digestion-protocol-e0546" target="_blank"> https://www.neb.com/protocols/1/01/01/digestion-protocol-e0546</A> Edited byEduardo A. Ramírez Rodríguez DateSeptember 9th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse7">
          Protocol 7-Agarose based electrophoresis
        </A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">Analyze DNA fragments based on their movement in a uniform electric field. </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">120 min</TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Agarose (BioRad) </TD></TR><TR><TD colspan="3">TAE Buffer (Promega) </TD></TR><TR><TD colspan="3">Loading dye (Promega) </TD></TR><TR><TD colspan="3">10 kb and 1 kb weight stair (Promega) </TD></TR><TR><TD colspan="3">Ethidium bromide (BioRad)  </TD></TR><TR><TD colspan="3">Pipettes and tips  </TD></TR><TR><TD colspan="3">DNA sample  </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Electrophoresis chamber (Fisher Scientific) </TD></TR><TR><TD colspan="3">Voltage source (BioRad)  </TD></TR><TR><TD colspan="3">Transiluminator (Thermo)  </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Digestion.</U>  Usually this protocol precedes the electrophoresis.</TD></TR><TR><TD colspan="3"><U>Check if there is TAE available. </U> A stock 50X can be made from: 
                <LI> Tris-base: 242 g </LI><LI> Acetate (100% acetic acid): 57.1 ml  </LI><LI> EDTA: 100 ml 0.5M sodium EDTA </LI><LI> Add dH<SUB>2</SUB>O up to one litre.  </LI></TD></TR><TR><TH colspan="3">Related protocols  </TH></TR><TR><TD colspan="3">Digestion, enzyme restriction analysis.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD colspan="3"><U>Prepare the gel </U></TD></TR><TR><TD align="right">1.</TD><TD colspan="2">
                  The amount of agarose depends on the size of the DNA. (A guide can be seen in the table below) </TD></TR><TR><TH>Agarose Concentration (g/100mL)</TH><TH>Optimal DNA Resolution (kb)</TH></TR><TR><TD><CENTER>0.5</CENTER></TD><TD><CENTER>1 - 30 </CENTER></TD></TR><TR><TD><CENTER>0.7</CENTER></TD><TD><CENTER>0.8 - 12</CENTER></TD></TR><TR><TD><CENTER>1.0</CENTER></TD><TD><CENTER>0.5 - 10</CENTER></TD></TR><TR><TD><CENTER>1.2</CENTER></TD><TD><CENTER>0.4 - 7</CENTER></TD></TR><TR><TD><CENTER>1.5</CENTER></TD><TD><CENTER>0.2 - 3</CENTER></TD></TR></TBODY></TABLE></CENTER>2.Measure out the appropriate mass of agarose into a beaker with the appropriate volume of buffer. The volume in the lab gelbox is 50mL. 3.Microwave until the agarose is fully melted and cool down a few minutes.4. Pour the agarose solution into the taped gelbox. Carefuly pop or shove to the side any bubbles, put in the comb, and let it cool for about 30 minutes, until the gel is solid. <U>Load the gel </U>5.Mix the DNA samples with 5X loading dye solution to make it reach 1X. (Except for 1 µL of sample you can use 1µL of Loading dye). 6.Pour the mixture in the wells.<STRONG> Be careful with the bubbles and breaking the gel with the pipette. </STRONG>7.Fill the chamber with TAE and run the gel 70 min at 90 V and ~180 mA <U>Stain and visualize the gel  </U>8.Stain the gel soaking it in EtBr, be careful not to break it. 9.Reveal usin the transiluminator, first focus using the white light lamp and use the UV to make the bands visible. Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>TAE</CENTER></TD><TD><CENTER>Throw through the sink, the school has a water treatment plant for all the facilities. </CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Gel</CENTER></TD><TD><CENTER>The gel has EtBr, which is very mutagenic, gloves must be used to handle the gel when you soak it in the solution. The gel must be disposed in the EtBr container</CENTER></TD><TD><CENTER>Do not touch the gel or the EtBr solution with your bare hands</CENTER></TD></TR><TR><TD><CENTER>EtBr solution</CENTER></TD><TD><CENTER>Every once in a while the solution must be changed, and the discarded solution must be</CENTER></TD><TD><CENTER>See above</CENTER></TD></TR><TR><TD><CENTER>Gloves </CENTER></TD><TD><CENTER>You must discard the glove you use when you handle the gel and the computer. They must be thrown in the EtBr labeled container.</CENTER></TD><TD><CENTER>Make sure to dispose your gloves</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Remember to use the EtBr at the delimited area in the lab.</LI><LI>Dispose your gloves after handling the gel. </LI><LI>If you accidentally touch a surface of EtBr designed area, remember to soak yourself with ethanol 70% to inactivate the EtBr. Seriously, this <STRONG>shouldn’t happen.</STRONG></LI></OL>ReferencesAgarose Gel Electrophoresis <A href="http://www.addgene.org/plasmid-protocols/gel-electrophoresis/">http://www.addgene.org/plasmid-protocols/gel-electrophoresis/</A> Edited byEduardo A. Ramírez Rodríguez DateSeptember 28th, 2014 Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse8">
          Protocol 8-Cell Lysis and Protein Purification
        </A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">Extract the protein produced by <EM>E. coli</EM> and purification of polyhistidine-tagged proteins from a soluble protein extract by metal affinity chromatography. </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">500mL solution Tween 20 5% v/v in K<SUB>2</SUB>HPO<SUB>4</SUB> 25mM pH 7.5.  </TD></TR><TR><TD colspan="3">100mL Biuret reagent   </TD></TR><TR><TD colspan="3">Equilibration Buffer: 10mM Imidazole </TD></TR><TR><TD colspan="3">Wash Buffer: 25mM Imidazole  </TD></TR><TR><TD colspan="3">Elution Buffer: 250mM Imidazole </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys) </TD></TR><TR><TD colspan="3">Sonicator </TD></TR><TR><TD colspan="3">Spectrometer (Thermo) </TD></TR><TR><TD colspan="3">50 and 15mL plastic tubes  </TD></TR><TR><TD colspan="3">Centrifuge (Thermo)  </TD></TR><TR><TD colspan="3">HisPur Ni-NTA Spin Columns  </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Grow E. coli culture to 0.8OD.</U>Centrifuge and keep the pellet (wet biomass) for lysis. Supernatant can be kept to compare the concentration of protein with the lysate. </TD></TR><TR><TD colspan="3">Prepare Purification Buffers under this specifications: </TD></TR><TR><TD><STRONG>Imidazole Finale Conc. (mM) </STRONG></TD><TD><STRONG>10X PBS (mL)</STRONG></TD><TD><STRONG>2M Imidazole (µL) </STRONG></TD><TD><STRONG>Water (mL)  </STRONG></TD></TR><TR><TD>10 </TD><TD>1</TD><TD>50</TD><TD>8.95</TD></TR><TR><TD>25 </TD><TD>1</TD><TD>125</TD><TD>8.875</TD></TR><TR><TD>40 </TD><TD>1</TD><TD>200</TD><TD>8.8</TD></TR><TR><TD>60</TD><TD>1</TD><TD>300</TD><TD>8.7</TD></TR><TR><TD>75 </TD><TD>1</TD><TD>375</TD><TD>8.625</TD></TR><TR><TD>150 </TD><TD>1</TD><TD>750</TD><TD>8.25</TD></TR><TR><TD>200</TD><TD>1</TD><TD>1000</TD><TD>8</TD></TR><TR><TD>250</TD><TD>1</TD><TD>1250</TD><TD>7.75</TD></TR><TR><TD>500</TD><TD>1</TD><TD>2250</TD><TD>6.5</TD></TR></TBODY></TABLE></CENTER>Steps1.Take 2.0g of wet biomass and place it in a plastic test tube up to 50ml. 2.Resuspend the solution in 28ml of Tween 20 5% v/v in K<SUB>2</SUB>HPO<SUB>4</SUB> 25mM pH 7.5.  3.Introduce the sonication probe in the suspension of cells. Sonicate the sample 10 seconds and turn of another 10 seconds for 3 minutes.<STRONG> The tube should be put in ice during the procedure.</STRONG>4. Centrifuge samples (4,000rpm) for 2 min. 5.Quantify total protein in the lysate and supernatant using Biuret method. Use LB medium as blank with Biuret reagent.Spectrometer reading at 540nm should be compared with standard curve previously made.<U>Purification</U>6.Prepare sample by mixing protein extract with Equilibration Buffer so the total volume equals two resin-bed volumes. 7.Remove the bottom tab from the HisPur Ni-NTA Spin Column by gently twisting. Place column into a centrifuge tube. <STRONG>Use 1.5, 15 or 50mL centrifuge tubes for the 0.2, 1 and 3mL spin columns, respectively.</STRONG>8.Centrifuge column at 700 × g for 2 minutes to remove storage buffer. 9.Equilibrate column with two resin-bed volumes of Equilibration Buffer. Allow buffer to enter the resin bed. 10.Centrifuge column at 700 × g for 2 minutes to remove buffer. 11.Add the prepared protein extract to the column and allow it to enter the resin bed. <STRONG>For maximal binding, the sample can be incubated for 30 minutes at room temperature or 4°C on an end-over-end rocking platform.</STRONG>12.Centrifuge column at 700 × g for 2 minutes and collect the flow-through in a centrifuge tube.13.Wash resin with two resin-bed volumes of Wash Buffer. Centrifuge at 700 × g for 2 minutes and collect fraction in a centrifuge tube. Repeat this step two more times collecting each fraction in a separate centrifuge tube.14.Elute His-tagged proteins from the resin by adding one resin-bed volume of Elution Buffer. Centrifuge at 700 × g for 2 minutes. Repeat this step two more times, collecting each fraction in a separate tube.Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB Medium </CENTER></TD><TD><CENTER>Place in container with chlorine solution.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Supernatant Waste </CENTER></TD><TD><CENTER>Sterilize. </CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Wash Buffer</CENTER></TD><TD><CENTER>Place in container with chlorine solution and sterilize.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Gloves must be worn at all times. </LI></OL>ReferencesThermo Scientific HisPur<SUP>TM</SUP> Ni-NTA Purification Kit<A target="_blank" href="https://www.piercenet.com/instructions/2162206.pdf "> https://www.piercenet.com/instructions/2162206.pdf </A> Edited byEduardo Serna Morales DateSeptember 28th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse9">
          Protocol 9-Red Lambda Protocol
        </A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">To make target genetic changes in <EM>E. coli</EM>. </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">3.5h  </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">25 mL LB </TD></TR><TR><TD colspan="3">1.1mL glycerol  </TD></TR><TR><TD colspan="3">IPTG </TD></TR><TR><TD colspan="3">MOPS 1mM </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TD colspan="3">Centrifuge (Thermo)  </TD></TR><TR><TD colspan="3">Electroporation System (Gene Pulser MXcell) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Digestion.</U> Electrocompetent cells/ Calcium competent cells.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Select a colony and place it on 20mL LB + 100 µg/ml Amp 30°C in a 125mL flask.</TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Grow up to ~10<SUP>7</SUP> cel/mL. </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Add IPTG to final concentration of 1mM.</TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> After reaching the density between 0.5-1x10<SUP>8</SUP> cel/mL apply heatshock for 15 min at 42°C swirling.</TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Transfer to ice for 10min (swirling).</TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Centrifuge (Cell recovery).</TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Resuspend in 1mL of glycerol 20% - 1mM MOPS (unbuffered) <STRONG>frozen</STRONG>. </TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Transfer to 1.5mL eppendorf. </TD></TR><TR><TD align="right">9.</TD><TD colspan="2">Centrifuge 30seg at moderate speed. </TD></TR><TR><TD align="right">10.</TD><TD colspan="2">Pull supernatant. </TD></TR><TR><TD align="right">11.</TD><TD colspan="2">Resuspend in the same buffer. </TD></TR><TR><TD align="right">12.</TD><TD colspan="2">Centrifuge 30seg at moderate speed. </TD></TR><TR><TD align="right">13.</TD><TD colspan="2">Repeat steps 11 and 12.  </TD></TR><TR><TD align="right">14.</TD><TD colspan="2">Resuspend in 90-100µL 1mM MOPS 20% <STRONG>frozen</STRONG> glycerol.  </TD></TR><TR><TD align="right">15.</TD><TD colspan="2">Cool electroporation cells on ice for 10 mins.  </TD></TR><TR><TD align="right">16.</TD><TD colspan="2">Add 50µL of cells to the electroporation cell, with 1-5 µL of DNA and incubate for 1min on ice.   </TD></TR><TR><TD align="right">17.</TD><TD colspan="2">Dry out the electroporation cell fast and efficiently.  </TD></TR><TR><TD align="right">18.</TD><TD colspan="2">Apply shock. Peak discharge 20kv/cm. RC time 4.2mS. Resistor of 200Ω. </TD></TR><TR><TD align="right">19.</TD><TD colspan="2">Suspend the cell with 0.3 mL of LB and dilute in 2.7mL of LB. </TD></TR><TR><TD align="right">20.</TD><TD colspan="2">Grow for 1.5h at 37°C. </TD></TR><TR><TD align="right">21.</TD><TD colspan="2">Plating with <STRONG>resistance at half.</STRONG></TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB medium</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Petri dishes with culture </CENTER></TD><TD><CENTER>Place in a bag to sterilize.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesMurphy et al. Lambda Red-mediated recombinogenic engineering of enterohemorrhagic and enteropathogenic E. coli. BMC Mol Biol. 2003 Dec 13. 4():11.  Edited byMercedes VázquezDateSeptember 25th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse10">
          Protocol 10-Quorum Sensing Assay
        </A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">Characterize the operation of quorum sensing by fluorescence.</TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">Around 5 hours (depends in the growth rate of the strain) </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">100 mL LB </TD></TR><TR><TD colspan="3">5mM potassium phosphate buffer </TD></TR><TR><TD colspan="3">Bioassay medium (0.05% w/v tryptone, 0.03% v/v glycerol, 100mM sodium chloride, 50mM magnesium sulfate and 5mM potassium phosphate buffer, pH 6.5, containing antibiotic) </TD></TR><TR><TD colspan="3">Overnight <EM>E. coli</EM> culture</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys)</TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions)  </TD></TR><TR><TD colspan="3">Spectrometer (Thermo)  </TD></TR><TR><TD colspan="3">96-well microplate with clean background </TD></TR><TR><TD colspan="3">Centrifuge (Thermo) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Autoclave.</U> 100mL of LB , 2 centrifuge tubes. </TD></TR><TR><TD colspan="3"><U>Prepare starter culture cells.</U> Inoculate a 25mL starter culture of LB (no antibiotics). Grow culture at 37°C in shaker overnight.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Add overnight to LB medium until OD600nm reaches 0.1 </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Incubate cell in incubator with shaker (250 rpm and 37°C) </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">After 2 hours, centrifuge 3mL of sample at 14,000rpm for one minute </TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> Wash the pellet twice with 600µL of PBS 1X.</TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Load 200µL of the suspension to a 96 wells Costar plate. (Load the suspension in triplicate)
              <LI style="list-style-type:none"> Our cells are able to produce acyl-HSL (it is not necessary to add more).</LI></TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Measure fluorescence of GFP (395nm excitation, 509nm emission, 495nm cutoff). </TD></TR><TR><TD align="right">7.</TD><TD colspan="2">Measure cell density at 600nm. </TD></TR><TR><TD align="right">8.</TD><TD colspan="2">Repeat steps 3 to 7. (For better characterization it is recommended to repeat this steps for 16-20 hours) </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB medium</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Microplate </CENTER></TD><TD><CENTER>Place in a bag to sterilize. </CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>References Edited byMercedes VázquezDateSeptember 25th,2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse11">
          Protocol 11-Phage Purification by Isoelectric Precipitation 
        </A></H4></DIV><DIV id="collapse11" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">To achieve phage recuperation (98.4%) from culture media.  </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">Around 3 hours</TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">250mL of LB medium </TD></TR><TR><TD colspan="3">5M HCl  </TD></TR><TR><TD colspan="3">50mL of mili-Q water </TD></TR><TR><TD colspan="3">Overnight <EM>E. coli</EM> culture</TD></TR><TR><TD colspan="3">5M NaOH  </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Incubator with shaker (TalBoys) </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions)  </TD></TR><TR><TD colspan="3">Centrifuge (Thermo)  </TD></TR><TR><TD colspan="3">Oven for calculations of dry weight </TD></TR><TR><TD colspan="3">4 tubes for centrifugation </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Virus Culture.</U> Add 200µL of <EM>E. coli ER2738</EM> and 10µL of virus suspension (approx. 1x1010 PFUml-1) to a suspension of 200mL of LB in an Erlenmeyer flask of 500mL.
            <LI>LB Medium: 
              </LI><LI>10gl<SUP>-1</SUP> of tryptone. </LI><LI>5gl<SUP>-1</SUP> NaCl (pH 7.0-7.5).</LI><LI>5gl<SUP>-1</SUP> of yeast extract. </LI><LI>Culture conditions: 
              </LI><LI>37°C  </LI><LI>220 rpm </LI><LI>16 h</LI></TD></TR><TR><TD colspan="3">Bacterial cells are removed by centrifugation (5,000 rpm) for 10 min and a second centrifugation (12,000 rpm) for 10 min. </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Adjust he pH of the supernatant to pH 4.2 with 5M HCl and is mixed briefly. </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Concentrate the virus precipitated of the 200mL culture in a 50mL tube by centrifugation (13,000 rpm) for 10min at 20°C. <STRONG>The supernatant is discarded</STRONG>.</TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Remove the residual LB, suspend pellets in 40mL of Mili-Q water by vortexing and centrifuge (13,000 rpm) for 10min at 20°C. </TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> Dry washed pellets at 70°C in an oven for calculations of dry weight (covered with a jacket layer Bio-Shield Dupont) or suspend in 10mL Mili-Q water. </TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Adjust the respuspended pellets pH to 7 with 5M NaOH and keep samples at 4°C. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LB medium</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesDong, D., Sutaria, S., Hwangbo, J. Y., &amp; Chen, P. (2013). A simple and rapid method to isolate purer M13 phage by isoelectric precipitation. Applied Microbiology and Biotechnology, 97(18), 8023– 9. doi:10.1007/s00253-013-5049-9 Edited byMercedes VázquezDateSeptember 25th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse12">
          Protocol 12-Phage Quantification by Spectrophotometry 
        </A></H4></DIV><DIV id="collapse12" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">To quantify phage concentration in virion/mL. </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">30 minutes</TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Milipore DI water</TD></TR><TR><TD colspan="3">TBS </TD></TR><TR><TD colspan="3">4 sterile centrifuge tubes</TD></TR><TR><TD colspan="3">Overnight <EM>E. coli</EM> culture</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Nanodrop</TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Phage purification.</U></TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Start nanodrop with 2µL Milipore DI water.</TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Use 2µL of 1X TBS as blank for each phage solution at 290nm and 320nm.</TD></TR><TR><TD align="right">3.</TD><TD colspan="2">To calculate the phage concentration this equation is needed:
              </TD></TR><TR><TD colspan="2">If the concentration is above 2OD at 269nm, the preparation is too concentrated; use a bigger dilution factor.
              </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Phage Waste</CENTER></TD><TD><CENTER>Wash with detergent and/or hypochlorite solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>TBS solution</CENTER></TD><TD><CENTER>Place in container with chlorine solution.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesYin, H. (2013) Searching for M13 Bacteriophage with affinity for nanodiamond particles. Department of materials science and engineering.  Edited byMercedes VázquezDateSeptember 25th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse13">
          Protocol 13-Tricine-SDS-PAGE
        </A></H4></DIV><DIV id="collapse13" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">Separate proteins according to their electrophoretic mobility.</TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Glycerol (Serva) </TD></TR><TR><TD colspan="3">Tetramethylethylenediamine (TEMED;Sigma) </TD></TR><TR><TD colspan="3">Mercaptoethanol (Sigma) </TD></TR><TR><TD colspan="3">Ammonium persulfate (Sigma) </TD></TR><TR><TD colspan="3">Urea (Serva) </TD></TR><TR><TD colspan="3">
				Reducing sample buffers: <UL><LI> 
				Buffer A: 12% SDS (wt/vol), 6% mercaptoethanol (vol/vol), 30% glycerol(wt/vol), 0.05% Coomassie blue G-250 (Serva), 150 mM Tris/HCl (pH 7.0)
				</LI><LI> 
				Buffer A/4: buffer A diluted with 3 volumes of water
				</LI><LI> 
				Buffer C: buffer A without glycerol
				</LI></UL></TD></TR><TR><TD colspan="3">
				Nonreducing sample buffers: <UL><LI> 
				Buffer B: 12% SDS (wt/vol), 30% glycerol (wt/vol), 0.05% Coomassie blue G-250 (Serva), 150 mM Tris/HCl (pH 7.0)
				</LI><LI> 
				Buffer B/4: buffer B diluted with 3 volumes of water
				</LI><LI> 
				Buffer D: buffer B without glycerol
				</LI></UL></TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Vertical electrophoresis apparatus (Bio Rad) </TD></TR><TR><TD colspan="3">Power Supply (600V, 500mA) for SDS-PAGE and electroblotting</TD></TR><TR><TD colspan="3">Polyvinylidene fluoride (PVDF) membranes or nitrocellulose membranes</TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"> Prepare Reducing and nonreducing sample buffers.
			  </TD></TR><TR><TD colspan="3"> Prepare the Electrode and gel buffers for Tricine–SDS-PAGE as follows: 
			  </TD></TR><TR><TH>Anode Buffer(10x)</TH><TH>Cathode Buffer(10x)</TH><TH>Gel Buffer(3x)</TH></TR><TR><TH>Tris (M)</TH><TD><CENTER>1.0</CENTER></TD><TD><CENTER>1.0</CENTER></TD><TD><CENTER>3.0</CENTER></TD></TR><TR><TH>Tricine (M)</TH><TD><CENTER>-</CENTER></TD><TD><CENTER>1.0</CENTER></TD><TD><CENTER>-</CENTER></TD></TR><TR><TH>HCl (M)</TH><TD><CENTER>0.225</CENTER></TD><TD><CENTER>-</CENTER></TD><TD><CENTER>1.0</CENTER></TD></TR><TR><TH>SDS (%)</TH><TD><CENTER>-</CENTER></TD><TD><CENTER>1.0</CENTER></TD><TD><CENTER>0.3</CENTER></TD></TR><TR><TH>pH</TH><TD><CENTER>8.9</CENTER></TD><TD><CENTER>-8.25</CENTER></TD><TD><CENTER>8.45</CENTER></TD></TR></TBODY></TABLE></CENTER>Steps1.
					Select and cast the appropriate base (separating) gel. The amounts of reagents required for two gels with dimensions 0.07 × 14 × 14 cm and containing 10% and 16% acrylamide are given in the table that follows. Do not degas the gel mixtures, because the gel buffer contains SDS. Overlay the poured gels with several drops of water. Leave the gels for about 30 min to polymerize.<STRONG> The freshly prepared ammonium persulfate (APS) solution and TEMED should be added last, immediately before pouring the gels, because these polymerize the gels.</STRONG><CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>4% Sample Gel</TH><TH>10% gel</TH><TH>16% gel</TH><TH>16%/6 M urea</TH></TR><TR><TH>AB-3 (ml)</TH><TD><CENTER>1</CENTER></TD><TD><CENTER>6</CENTER></TD><TD><CENTER>10</CENTER></TD><TD><CENTER>10</CENTER></TD></TR><TR><TH>Gel Buffer (3x)</TH><TD><CENTER>3</CENTER></TD><TD><CENTER>10</CENTER></TD><TD><CENTER>10</CENTER></TD><TD><CENTER>10</CENTER></TD></TR><TR><TH>Glycerol (g)</TH><TD><CENTER>-</CENTER></TD><TD><CENTER>3</CENTER></TD><TD><CENTER>3</CENTER></TD><TD><CENTER>-</CENTER></TD></TR><TR><TH>Urea (g)</TH><TD><CENTER>-</CENTER></TD><TD><CENTER>-</CENTER></TD><TD><CENTER>-</CENTER></TD><TD><CENTER>10.8</CENTER></TD></TR><TR><TH>Add water to final volume (ml)</TH><TD><CENTER>12</CENTER></TD><TD><CENTER>30</CENTER></TD><TD><CENTER>30</CENTER></TD><TD><CENTER>30</CENTER></TD></TR><TR><TH>Polymerize by adding:APS (10%) (µl)</TH><TD><CENTER>90</CENTER></TD><TD><CENTER>150</CENTER></TD><TD><CENTER>100</CENTER></TD><TD><CENTER>100</CENTER></TD></TR><TR><TH>  TEMED (µl)</TH><TD><CENTER>9</CENTER></TD><TD><CENTER>15</CENTER></TD><TD><CENTER>10</CENTER></TD><TD><CENTER>10</CENTER></TD></TR></TBODY></TABLE></CENTER>2.Overlay the polymerized separating gel (10% or 16%, or 16%/6 M urea) directly with a 4% sample (stacking) gel, except if resolution of proteins &lt;5 kDa is desired. If resolution of proteins &lt;5 kDa is desired, then use AB-6 instead of AB-3 for the separating gel and overlay the separating gel with a 1-cm 10% gel. The 16% separating gel and the overlaid 10% ‘spacer gel’ can be polymerized together if no glycerol is added to the 10% acrylamide gel mixture (the common role of glycerol in SDS gels is to increase the density of solutions and to facilitate gel casting; it has no obvious effect on protein separation). Introducing a 10% ‘spacer gel’ between 4% stacking and 16% separating gels considerably sharpens the bands for proteins and peptides of 1–5 kDa.3.Adjust protein concentrations so that a suitable amount of protein can be loaded onto the gel. <STRONG>Concentrate samples, preferentially by techniques that do not increase the salt concentration (such as ultrafiltration). </STRONG>Roughly 0.2–1 µg of protein for each protein band (in 0.7 × 5 mm gel strips) is sufficient for Coomassie staining. Accordingly, the desired protein concentration in the sample is 0.1 mg ml–1 for each protein band. 4. Mix samples with SDS-containing sample buffers. The volume and buffer to be used depend on the origins of the samples. For low-density samples such as elution fractions from chromatographic columns, add 5 µl of reducing or nonreducing sample incubation buffer A or B to 15 µl of sample. For high-density samples such as fractions from sucrose density gradients, add 5 µl of sample buffer C or D to 15 µl of sample. For pellet samples, resuspend the pellet in 15–20 µl of buffer A/4 or B/4. 5.Incubate samples at 37 °C for 15 min or for up to 60 min for samples that were in pellet form. <STRONG>Avoid boiling samples, because membrane proteins can irreversibly aggregate in SDS at temperatures &gt;50 °C.</STRONG>6.Mount the gels in the vertical electrophoresis apparatus, and add anode buffer as the lower electrode buffer and cathode buffer as the upper electrode buffer.7.Load samples under the cathode buffer. Apply 10-µl sample volumes to 0.7 × 5 mm sample wells.8.Set running conditions appropriate to your type of gel; guidance on appropriate running conditions are given in the table below.<CENTER><TABLE id="other-table" width="60%"><TBODY><TR><TH>10% (0.7mm)</TH><TH>16% (0.7mm)</TH><TH>16% (1.6mm)</TH></TR><TR><TH>Initial voltage</TH><TD><CENTER>30V</CENTER></TD><TD><CENTER>30V</CENTER></TD><TD><CENTER>30V</CENTER></TD></TR><TR><TH>Next voltage step</TH><TD><CENTER>190V</CENTER></TD><TD><CENTER>200V</CENTER></TD><TD><CENTER>90V constant</CENTER></TD></TR><TR><TH>Voltage at end of run</TH><TD><CENTER>270V</CENTER></TD><TD><CENTER>300V</CENTER></TD><TD><CENTER>-</CENTER></TD></TR><TR><TH>Time</TH><TD><CENTER>3-4h</CENTER></TD><TD><CENTER>5-6h</CENTER></TD><TD><CENTER>Overnight</CENTER></TD></TR></TBODY></TABLE></CENTER>9.Protein can be visualized directly in the gel by Coomassie staining or silver staining. Both Coomassie staining and silver staining can be done providing Coomassie staining is done first. Both procedures are compatible with subsequent mass spectrometric analysis. Alternatively, the proteins can be transferred to a PVDF membrane by electroblotting. Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD>Liquid and gel material</TD><TD>Discarded through hazardous waste management system. </TD><TD>Avoid having direct contact with it. </TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Gloves must be worn at all times. </LI><LI>All the procedure must be done in a restricted area. </LI></OL>ReferencesSchägger, H. (2006). tricine–sds-page. <EM>Nature protocols, 1</EM>(1), 16-22.  Edited byMercedes Vázquez DateOctober 15th, 2014 Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse14">
          Protocol 14-Chromogenic Lal Endotoxin Assay
        </A></H4></DIV><DIV id="collapse14" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">To quantifie endotoxins by a modified Limulus Amebocyte Lysate and a synthetic color producing substrate. Detection limit of 0.005 EU/ml, concentration range of 0.005 to 1 EU/ml.  </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Limulus Amebocyte Lysate (LAL) </TD></TR><TR><TD colspan="3">Stop Solution </TD></TR><TR><TD colspan="3"><EM>E. coli</EM> endotoxin standard </TD></TR><TR><TD colspan="3">Color standards </TD></TR><TR><TD colspan="3">0.1N sodium hydroxide  </TD></TR><TR><TD colspan="3">0.1N hydrochloric acid  </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Endotoxin free tubes and tips </TD></TR><TR><TD colspan="3">Spectrometer (Thermo)  </TD></TR><TR><TD colspan="3">Water Bath (Thermo)  </TD></TR><TR><TD colspan="3">Incubator (Tal Boys) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3">Dilute test specimen using LAL Reagent Water, adjust pH to 6-8 with 0.1N sodium hydroxide or 0.1N hydrochloric acid. </TD></TR><TR><TD colspan="3">Prepare 1EU/ml endotoxin solution before making standard serial dilutions. In each assay, at least four endotoxin standard solutions covering desired concentration range should be prepared to generate a standard curve. If the endotoxin concentration for the test sample is expected to be in the range of 0.01 - 0.1 EU/ml, the serial endotoxin standard solutions could be 0.1, 0.05, 0.025 and 0.01 EU/ml, respectively. If the endotoxin concentration in sample is expected in the range of 0.1 - 1 EU/ml, the serial endotoxin standard solutions could be 1, 0.5, 0.25 and 0.1 EU/ml, respectively. </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Dispense 100µL of standard or test sample into endotoxin-free vials. Mix samples for 30 minutes with a vortexer. Each test must include a blank as well as at least four endotoxin standards in duplicate. The blank sample vial contains 100 μl of LAL Reagent Water instead of test sample. </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Add 100 µl of reconstituted LAL to each vial. Cap the vials and mix well by swirling gently. </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">If the endotoxin concentration in sample is expected in the range of 0.01 - 0.1 EU/ml, incubate the rack with all vials at 37°C±1°C for <STRONG>T1</STRONG> using a water bath or heating block. If the endotoxin concentration is expected in the range of 0.1 - 1 EU/ml, incubate at 37°C±1°C for <STRONG>T2. T1 Time ranges from 40 to 60 minutes, and T2 Time ranges from 8 to 16 minutes.</STRONG></TD></TR><TR><TD align="right">4.</TD><TD colspan="2"> After proper incubation, add 100 µl of reconstituted chromogenic substrate solution to each vial. Cap the vials and swirl gently to mix well.<STRONG> Do not shake or vortex to avoid foaming</STRONG>. Incubate at 37°C±1°C for 6 minutes. </TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Add 500 µl of reconstituted Color-stabilizer #1 (Stop Solution) to each vial and swirl gently to mix well. Do not shake or vortex to avoid foaming. Add 500 µl of reconstituted Color-stabilizer #2 to each vial and mix well. Finally add 500 µl of reconstituted Color-stabilizer #3 to each vial. Gently swirl each vial to mix well. <STRONG>Bubbles must be avoided. </STRONG></TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Read the absorbance of each reaction vial at 545 nm using distilled water as blank to adjust the photometer to zero absorbance. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>LAL vials</CENTER></TD><TD><CENTER>Place in a container with chlorine solution</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR><TR><TD><CENTER>Endotoxin residues</CENTER></TD><TD><CENTER>Sterilize.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Every step must be done in the hood to avoid contamination</LI><LI>Gloves must be worn at all times</LI></OL>ReferencesToxinSensorTM Chromogenic LAL Endotoxin Assay Kit Genscript <A href="http://www.genscript.com/document/scm_files/prod"> http://www.genscript.com/document/scm_files/prod</A> Edited byEduardo Serna Morales DateSeptember 28th, 2013Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse15">
          Protocol 15-Polymerase Chain Reaction (PCR)
        </A></H4></DIV><DIV id="collapse15" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH colspan="3">Objective</TH></TR><TR><TD colspan="3">DNA amplification using Taq DNA polymerase. </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Nuclease free water </TD></TR><TR><TD colspan="3">dNTPs </TD></TR><TR><TD colspan="3">10X ThermoPol or Standard Taq Reaction Buffer</TD></TR><TR><TD colspan="3">Taq DNA Polymerase</TD></TR><TR><TD colspan="3">Primers </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Thermocycler </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TD colspan="3">PCR Tubes </TD></TR><TR><TD colspan="3">Microcentrifuge (Thermo) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3"><U>Design Primers.</U> Serves as a starting point for DNA synthesis. Oligonucleotide primers are generally 20–40 nucleotides in length and ideally have a GC content of 40–60%.</TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Prepare the following 50 µl reaction in a 0.5 ml PCR tube on ice:</TD></TR><TR><TH>Concentration </TH><TH>25µL reaction </TH><TH>50 µL reaction </TH><TH>Final Concentration </TH></TR><TR><TD><CENTER>10X ThermoPol or Standard Taq Reaction Buffer</CENTER></TD><TD><CENTER>2.5 µL </CENTER></TD><TD><CENTER>5 µL </CENTER></TD><TD><CENTER>1X</CENTER></TD></TR><TR><TD><CENTER>10 mM dNTPs </CENTER></TD><TD><CENTER>0.5 µL </CENTER></TD><TD><CENTER>1 µL </CENTER></TD><TD><CENTER>200 µM</CENTER></TD></TR><TR><TD><CENTER>10 µM Forward Primer </CENTER></TD><TD><CENTER>0.5 µL  </CENTER></TD><TD><CENTER>1 µL </CENTER></TD><TD><CENTER>0.2 µM(0.05-1mM) </CENTER></TD></TR><TR><TD><CENTER>10 µM Reverse Primer</CENTER></TD><TD><CENTER>0.5 µL</CENTER></TD><TD><CENTER>1 µL</CENTER></TD><TD><CENTER>0.2 µM(0.05-1mM)</CENTER></TD></TR><TR><TD><CENTER>Template DNA </CENTER></TD><TD><CENTER>Variable </CENTER></TD><TD><CENTER>Variable </CENTER></TD><TD><CENTER>&lt;1000ng</CENTER></TD></TR><TR><TD><CENTER><EM>Taq</EM> DNA Polymerase </CENTER></TD><TD><CENTER>0.125 µL  </CENTER></TD><TD><CENTER>0.25 µL  </CENTER></TD><TD><CENTER>1.25units/50 µL PCR </CENTER></TD></TR><TR><TD><CENTER>Nuclease-free water </CENTER></TD><TD><CENTER>To 25 µL  </CENTER></TD><TD><CENTER>To 50 µL  </CENTER></TD></TR></TBODY></TABLE></CENTER>Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary. 2.Transfer PCR tubes from ice to a PCR machine with the block preheated to 95°C and begin thermocycling. 3.Thermocycling conditions for a routine PCR: <CENTER><TABLE id="other-table" width="70%"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD><CENTER>Initial Denaturation </CENTER></TD><TD><CENTER>95°C</CENTER></TD><TD><CENTER>30sec </CENTER></TD></TR><TR><TD><CENTER>30 cycles </CENTER></TD><TD><UL id="list"><LI>95°C </LI><LI>45-68°C </LI><LI>68°C </LI></UL></TD><TD><UL id="list"><LI>15-30sec  </LI><LI>15-60sec <STRONG>(30 cycles)</STRONG></LI><LI>1min/kb </LI></UL></TD></TR><TR><TD><CENTER>Final Extension </CENTER></TD><TD><CENTER>72°C </CENTER></TD><TD><CENTER>5 minutes</CENTER></TD></TR><TR><TD><CENTER>Hold </CENTER></TD><TD><CENTER>4-10°C </CENTER></TD></TR></TBODY></TABLE></CENTER>Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Waste</CENTER></TD><TD><CENTER>Place in container with chlorine solution. </CENTER></TD><TD><CENTER>N/A </CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Gloves must be worn at all times.</LI><LI>Must be done in clean areas to avoid contamination. </LI></OL>ReferencesNEB PCR Protocol for Taq DNA Polymerase with Standard Taq Buffer (M0273) <A href="https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273" target="_blank"> https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</A> Edited byEduardo Serna Morales DateSeptember 28th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse16">
          Protocol 16-Apoptosis Detection
        </A></H4></DIV><DIV id="collapse16" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH colspan="3">Objective</TH></TR><TR><TD colspan="3">Detect apoptosis in mammalian cells through the use of FITC Annexin V and 7-AAD </TD></TR><TR><TH colspan="3">Approximate time</TH></TR><TR><TD colspan="3">60 min </TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">FITC Annexin V (Biolegend) </TD></TR><TR><TD colspan="3">7-AAD Viability Staining Solution (Biolegend) </TD></TR><TR><TD colspan="3">Annexin V Binding Buffer (Biolegend) </TD></TR><TR><TD colspan="3">Cell Staining Buffer (Biolegend) </TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Flow cytometer </TD></TR><TR><TD colspan="3">Laminar flow hood (Scientific Visions) </TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3">The treatment with the effectors should be done during different time lapses before the apoptosis staining. </TD></TR><TR><TH colspan="3">Steps</TH></TR><TR><TD align="right">1.</TD><TD colspan="2">Wash cells twice with cold BioLegend's Cell StainingBuffer, and then resuspend cells in Annexin V Binding Buffer at a concentration of 0.25-1.0 x 10<SUP>7</SUP> cells/ml. </TD></TR><TR><TD align="right">2.</TD><TD colspan="2">Transfer 100 µl of cell suspension in a 5 ml test tube.  </TD></TR><TR><TD align="right">3.</TD><TD colspan="2">Add 5 µl of FITC Annexin V. </TD></TR><TR><TD align="right">4.</TD><TD colspan="2">Add 5 µl of 7-AAD Viability Staining Solution.  </TD></TR><TR><TD align="right">5.</TD><TD colspan="2">Gently vortex the cells and incubate for 15 min at room temperature (25°C) in the dark. </TD></TR><TR><TD align="right">6.</TD><TD colspan="2">Add 400 µl of Annexin V Binding Buffer to each tube. Analyze by flow cytometry with proper machine settings. </TD></TR><TR><TH colspan="3">Waste disposal</TH></TR><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Cell Staining Buffer</CENTER></TD><TD><CENTER>Collect in a flask and dispose with the help of a specialized company</CENTER></TD><TD><CENTER>N/A </CENTER></TD></TR><TR><TD><CENTER>Cytometry Solution with cells </CENTER></TD><TD><CENTER>Collect in a flask and dispose with the help of a specialized company</CENTER></TD><TD><CENTER>Avoid contact with 7-AAD </CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>7-AAD is a potential carcinogen. It is recommended that the user wear protective clothing, gloves, and eye/face protection in order to avoid contact with skin and eyes. </LI></OL>ReferencesFITC Annexin V Apoptosis Detection Kit with 7-AAD (Biolegend) http://www.biolegend.com/fitc-annexin-v-apoptosis-detection-kit-with-7-aad-8264.html Edited byClaudia Nallely Alonso Cantú DateOctober 9th, 2014Revision</DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title text-center"><A data-toggle="collapse" data-parent="#accordion" href="#collapse17">
          Protocol 17-Western Blot
        </A></H4></DIV><DIV id="collapse17" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><TABLE width="100%"><TBODY><TR><TH>Objective</TH></TR><TR><TD colspan="3">To detect specific proteins in a sample.</TD></TR><TR><TH colspan="3"> Material</TH></TR><TR><TD colspan="3">Antibody Diluent</TD></TR><TR><TD colspan="3">10X Wash Buffer</TD></TR><TR><TD colspan="3">HRP Reagent</TD></TR><TR><TD colspan="3">Detection Reagent 1</TD></TR><TR><TD colspan="3">Primary Antibody</TD></TR><TR><TH colspan="3"> Equipment &amp; Apparatus</TH></TR><TR><TD colspan="3">Rotary platform shaker for agitation of membrane during incubations</TD></TR><TR><TD colspan="3">Blot Gel Tank (Bio-rad)</TD></TR><TR><TD colspan="3">Power Supply</TD></TR><TR><TD colspan="3">Membrane to transfer the protein</TD></TR><TR><TH colspan="3">Previous steps</TH></TR><TR><TD colspan="3">Prepare reagents:</TD></TR><TR><TH>Reaction volume</TH><TH>Restriction Enzyme</TH></TR><TR><TD><CENTER>Fast Western 1X Wash Buffer</CENTER></TD><TD><CENTER>Mix 1 part of the Fast Western 10X Wash Buffer with 9 parts of water.</CENTER></TD></TR><TR><TD><CENTER>Primary Antibody Working Solution</CENTER></TD><TD><CENTER>Shake the Fast Western Antibody Diluent well before use. Dilute the primary antibody with diluent to a concentration ranging from 0.2 to10 μg/ml. Use ~0.125 ml of antibody per cm<SUP>2</SUP> of membrane (e.g., 10 ml per 8 × 10 cm blot). The stability of diluted primary antibodies varies depending on the antibody. For best results prepare working dilution immediately before use.</CENTER></TD></TR><TR><TD><CENTER>Fast Western Optimized HRP Reagent Working Dilution</CENTER></TD><TD><CENTER>Mix 1 part of Fast Western Optimized HRP Reagent with 9 parts of Fast Western Antibody Diluent. Use 0.125 ml per cm<SUP>2</SUP> of membrane (e.g., 10 ml per 8 × 10 cm blot). For best results, use this solution within 1 hour.</CENTER></TD></TR><TR><TD><CENTER>Detection Reagent Working Solution</CENTER></TD><TD><CENTER>Mix Detection Reagents 1 and 2 at 1:1. Use 0.125 ml Working Solution per cm<SUP>2</SUP> of membrane (e.g., 10 ml per 8 × 10 cm blot). For best results prepare working solution immediately before use (Step 9). The working solution is stable for up to 1 hour at room temperature.</CENTER></TD></TR></TBODY></TABLE></CENTER><U>Transfer of proteins: </U>Detailed instructions for the transfer process can be found on the websites of the manufacturers of transfer apparatus, and will vary depending on the system. The principle is the same in each case though. In wet transfer, the gel and membrane are sandwiched between sponge and paper (sponge/paper/gel/membrane/paper/sponge) and all are clamped tightly together after ensuring no air bubbles have formed between the gel and membrane. The sandwich is submerged in transfer buffer to which an electrical field is applied. The negatively-charged proteins travel towards the positively-charged electrode, but the membrane stops them, binds them, and prevents them from continuing on. A standard buffer for wet transfer is the same as the 1X Tris-glycine buffer used for the migration/running buffer without SDS but with the addition of methanol to a final concentration of 20%. For proteins larger than 80 kD, it is recommended that SDS is included at a final concentration of 0.1%.Steps1.Remove blot from the transfer apparatus and place in a clean incubation tray.2.Briefly wash blot in Fast Western 1X Wash Buffer to remove transfer buffer. 3.Add the Primary Antibody Working Dilution to the blot and incubate for 30 minutes at room temperature (RT) with shaking. 4. Remove blot from the primary antibody solution and place it in a clean incubation tray.5.Add the Fast Western Optimized HRP Reagent Working Dilution and incubate for 10 minutes at RT with shaking. Higher sensitivity can be obtained by increasing the incubation time to 15 minutes; however, one or more additional washes will be required to reduce background.6.Remove blot from the HRP solution and place it in a clean incubation tray.7.Wash membrane by suspending it in approximately 20 ml of Fast Western 1X Wash Buffer and agitating for 5 minutes. Repeat this wash twice.8.Remove blot and place it in a clean incubation tray. Add the Detection Reagent Working Solution and incubate for 1-5 minutes at RT.9.Remove blot from Detection Reagent Working Solution and place it in a plastic sheet protector or clear plastic wrap. Use an absorbent tissue to remove excess liquid and to carefully press out any bubbles from between the blot and the membrane protector.10.Expose the blot to film or use your preferred imaging method.Waste disposal<CENTER><TABLE id="other-table" width="80%"><TBODY><TR><TH>Waste</TH><TH>Place to dispose</TH><TH>What to avoid</TH></TR><TR><TD><CENTER>Unpolymerized solutions</CENTER></TD><TD><CENTER>Has to be disposed as hazardous waste.</CENTER></TD><TD><CENTER>Avoid any contact</CENTER></TD></TR><TR><TD><CENTER>Polimerized Gels</CENTER></TD><TD><CENTER>Can be disposed in the trash.</CENTER></TD><TD><CENTER>N/A</CENTER></TD></TR></TBODY></TABLE></CENTER>Safety Notes<OL><LI>Acrylamide is a neurotoxin. Gloves and face mask must be worn at all times.  </LI><LI>Wash hands thoroughly after use.</LI></OL>ReferencesPierce Fast Western Blot Kit  <A href="https://www.piercenet.com/instructions/2162133.pdf" target="_blank"> https://www.piercenet.com/instructions/2162133.pdf</A>Definitive guide to Western Blot  <A href="http://www.abcam.com/ps/pdf/protocols/WB-beginner.pdf" target="_blank"> http://www.abcam.com/ps/pdf/protocols/WB-beginner.pdf</A> Edited byMercedes Vázquez DateOctober 15th, 2014Revision</DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Tec-Monterrey/ITESM14_protocols.html">http://2014.igem.org/Team:Tec-Monterrey/ITESM14_protocols.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tec-Monterrey/ITESM14_protocols.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tec-Monterrey/ITESM14_protocols.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_protocols.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_protocols.html&amp;oldid=348442" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>