<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tec-Monterrey_ITESM14_results_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tec-Monterrey/ITESM14_results.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tec-Monterrey/ITESM14_results.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_results.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_results.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tec-Monterrey/ITESM14_results.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tec-Monterrey/ITESM14 results.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wiki-body"><P>&lt;body&gt;</P><P>&lt;header id=&quot;wiki-header&quot; class=&quot;initial-nav-scroll-up&quot;&gt;&lt;/header&gt;</P><DIV id="wiki-content" class="main-content"><DIV class="well"><DIV class="container"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Module_1_Results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Module 1 Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Expected_results_from_Module_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Expected results from Module 2</SPAN></A></LI><LI class="toclevel-1"><A href="#Module_2_Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Module 2 Results</SPAN></A></LI><LI class="toclevel-2"><A href="#Bacteriophage_M13_Western_Analysis"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Bacteriophage M13 Western Analysis</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Module_3_Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Module 3 Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Module_4_Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Module 4 Results</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Module_1_Results">Module 1 Results</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><P>
			After the lambda-red protocol was performed and the cells underwent an electroporation with the digestions carrying BBa_K1366101 and BBa_K1366102 (See protocol 9); the cells were submitted to antibiotic selection and only four plates presented E. coli colonies. Those were BBa_K1366101+ BBa_K1366102 with the LuxR and LuxI parts of the Quorum sensing of Vibrio fischeri respectively (Fig. 1), BBa_K1366101 with the LuxR (Fig. 2), BBa_K1366102 with the LuxI (Fig. 3), and BBa_K1366102 without any additional part inserted (Fig. 4). 
			</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Result 1&quot; data-title=&quot;Fig. 1&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 1 &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Result 2&quot; data-title=&quot;Fig. 2&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 2 &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Result 3&quot; data-title=&quot;Fig. 3&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 3 &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Result 4&quot; data-title=&quot;Fig. 4&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 4 &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>
			In order to corroborate these results, the circled colonies in the images were submitted to a PCR analysis. For BBa_K1366101 recombination, the pair of primers used were:
			</P><P>
			Forward: ACTCAGGGCGGTAAATCTGC
			</P><P>
			Reverse: ATGGTGAACCAGAGCAAGGG
			</P><P>
			For this reaction the expected amplicons were 936 bp for the wild type, 1875 bp for the inserted biobrick in the genome without any additional part, and 2765 bp for the inserted biobrick with the LuxR part of the Quorum Sensing.
			</P><P>
			For BBa_K1366101 recombination, the pair of primers used were:
			</P><P>
			Forward: GGGTAAAGGTGAAGGCGACA
			</P><P>
			Reverse: TTGCACCACACAGAGGTGTT
			</P><P>
			And the expected amplicons were, for the wild type <EM>E. coli</EM>  2757 pb, 2881 bp for the inserted biobrick in the genome without any additional part and 3569 bp for the inserted biobrick with the LuxI part of the Quorum Sensing. 
			</P><P>
			All PCR parameters used are based in the protocol for Q5 High-Fidelity 2X Master Mix by New England BioLabs.
			</P><P>
			The results of the first round of PCR are presented in the next image:
			</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Result 4&quot; data-title=&quot;Fig. 5&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;</P><CENTER>&lt;figcaption&gt; Fig. 5 &lt;/figcaption&gt;</CENTER><P>&lt;/figure&gt;</P><P>
			According to this electrophoresis gel, BBa_K1366101 was not inserted in E. coli genome. However it can be seen that BBa_K1366102 with the LuxI sequence from Quorum sensing was incorporated to the bacteria’s genome. For this reason we conclude that BBa_K1366102 combined with the lambda-red system is functional and can be used to delete <EM>msbB</EM> gen and incorporate any other sequence in <EM>E. coli’s</EM> DNA.
			</P><P>
			During the following weeks, prior Giant Jamboree, we will be trying to delete <EM>lpp</EM> gen with BBa_K1366101, and implement the inflammation test (Alridge, et al. 2008), to prove that both deletions minimize the immune system response, thus demonstrating that an attenuated <EM>E. coli</EM> can be a useful vector for localized and specific cancer therapies, making it safer to use bacteria as a delivery system. 
			</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Expected_results_from_Module_2">Expected results from Module 2</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				The activity of the pep42 peptide expressed on the pIII of bacteriophage M13, shall be verified by tests on cell lines. We will work with the NIH 3T3*, PC3* and MCF7* lines. The bacteriophage peptide must be internalized in MCF7 and PC3 cells, which express GRP78 on their cell membrane protein, a heat shock protein. This membrane overexpression is characteristic of cancer cells due to tumor environments to which they are exposed. The GRP78 protein is the peptide ligand and its interaction phage promotes endocytosis.
				</P><P>
				To check the activity of our phage, we will make the following tests by fluorescence microscopy and western blot analysis.
				</P><H2>Phage Internalization &lt;h2&gt;<P>
				Expected results in phage internalization by fluorescence microscopy.				</P><P>
				It needs a cellular permeabilization in the procedure.<EM> See protocol: fluorescence microscopy</EM></P><CENTER><TABLE id="other-table" width="100%">&lt;col style=&quot;width:25%&quot;&gt;&lt;col style=&quot;width:25%&quot;&gt;&lt;col style=&quot;width:25%&quot;&gt;&lt;col style=&quot;width:25%&quot;&gt;<TBODY><TR><TH>Cell lines</TH><TH>Standard phage</TH><TH>Phage with peptide</TH><TH>No phage</TH></TR><TR><TD><CENTER>NIH</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal</CENTER></TD></TR><TR><TD><CENTER>PC3</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal</CENTER></TD></TR><TR><TD><CENTER>MCF7</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal inside the cell</CENTER></TD><TD><CENTER>No signal</CENTER></TD></TR></TBODY></TABLE></CENTER><P>
				We also will make a western blot analysis using cellular lysis sample to evaluate the presence of the phage-peptide. We will use as control the standard phage that we produced and confirmed by western blot. 
				</P><P>
				See protocol:&lt;a href=&quot;<A href="https://2014.igem.org/Team:Tec-Monterrey/ITESM14_project.html#tab_protocols" class="external free" rel="nofollow">https://2014.igem.org/Team:Tec-Monterrey/ITESM14_project.html#tab_protocols</A>&quot;&gt; See protocol: western blot &lt;/a&gt;</P>&lt;h2&gt;GRP78 presence of membrane </H2><P>
				It doesn’t need cellular permeabilization in the procedure. 
				</P><CENTER><TABLE id="other-table" width="100%">&lt;col style=&quot;width:50%&quot;&gt;&lt;col style=&quot;width:50%&quot;&gt;<TBODY><TR><TH>Cell lines</TH><TH>Expected result</TH></TR><TR><TD><CENTER>NIH</CENTER></TD><TD><CENTER>No signal</CENTER></TD></TR><TR><TD><CENTER>PC3</CENTER></TD><TD><CENTER>Signal in membrane</CENTER></TD></TR><TR><TD><CENTER>MCF7</CENTER></TD><TD><CENTER>Signal in membrane</CENTER></TD></TR></TBODY></TABLE></CENTER><P>
				In these tests, we will determine the phage-peptide interaction only with the cells containing the GRP78 in the membrane and thus allow for its internalization and check the activity of this peptide as a peptide of unique internalization for cancer cells.				
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Module_2_Results">Module 2 Results</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><H2><SPAN class="mw-headline" id="Bacteriophage_M13_Western_Analysis">Bacteriophage M13 Western Analysis </SPAN></H2><P>
				Phages were amplified by infection to E. coli XL1-Blue, then recovered and purified for further testing. Checking conducted phage DNA by electrophoresis, and activity analysis reinfection of their capsid proteins by western blot the pVIII. 
				</P><P>
				For the western blot different dilutions were loaded on <U>SDS-PAGE </U>(Tris-Tricine) to standardize to detect viable concentrations. 
				</P><P>
				The concentration of the purified sample was 4x10<SUP>13</SUP> virions per milliliter. The phage samples were prepared in different concentrations with dilutions in a total volume of 32 microliters. The western blot was revealed by chemiluminiscence.				</P><P>&lt;figure&gt;&lt;a href=&quot;&quot; data-lightbox=&quot;Image 6&quot; data-title=&quot;Fig. 6&quot;&gt;&lt;img class=&quot;img img-responsive&quot; style=&quot;margin:0px auto;display:block width:20%;&quot; src=&quot;&quot;&gt;&lt;/a&gt;&lt;/figure&gt;&lt;p&gt;
				For detection, two antibodies were used, a primary antibody; anti-M13 antibody Bacteriophage Coat Protein g8p (abcam) and a secondary antibody; Rabbit Anti-Mouse IgG (HRP).
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Module_3_Results">Module 3 Results</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				When the results of Module I are obtained we will characterize the proper expression and synergy of both genes inserted in E. coli’s genome. For this, we will transform our new E.coli Dh5- (Δmsbb, lpp) with the BBa_J37032. This biobrick encodes a green fluorescent protein regulated by the plux promoter. Then, through the use of the Quorum Sensing Assay we are going to compare the fluorescence activity of this new transformed strain with that of a wild type Dh5-Alpha. As cellular density increases, we expect to find significant difference in the measurement, due to the correct synthesis of AHL by LuxI and the consequent activation of LuxR, which will induce GFP expression. From this proof-of-concept experiment, we will be able to assure the quorum sensing system works adequately in our strain, and therefore will function optimally for the production of the Trojan horse (Bacteriophage) at high cellular densities in a tumorous environment.  
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="Module_4_Results">Module 4 Results</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><P>
				For us to evaluate this module, phagemids must be produced with the mutations in pIII and have the effectors included in its sequence. The effectors will be added by ligation; C4 (BBa_K1366106) with the phagemid will be used as proof of concept to demonstrate that the phage is capable of transfecting DNA to cancerous cells and that these cells are able to express the phagemid by producing GFP and therefore shining under UV light. C5 (BBa_K1366104), C6 (BBa_K1366105), C7 (BBa_K1366106), C8 (BBa_K1366107), and C9 (BBa_K1366108) will be ligated next, as they are the effectors. After this is done, the effectors should be added in solution to the breast and prostate cancerous tissue and left to incubate. Samples must be taken at different periods of time and each sample should be analyzed by flow cytometry with FITC-Annexin V and 7-AAD, as stated in protocol 16: Apoptosis Detection. These different periods of time will be set to evaluate the effect of our three different systems (Survivin siRNA, Apoptin, and Survivin siRNA+Apoptin) in causing apoptosis. As the length of the period of time increases, we expect that more cells will be found undergoing apoptosis.
				</P><P>&lt;/section&gt;</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>&lt;section id=&quot;column&quot;&gt;</P><OL><LI><P><EM>Protocol for Q5® High-Fidelity 2X Master Mix.</EM> (2014). Obtenido de New England Biolabs: <A href="https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492" class="external free" rel="nofollow">https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492</A></P></LI><LI><P><EM>Aldridge, C., Razzak, A., Babcock, T. A., Helton, W. S., &amp; Espat, N. J. (2008). Lipopolysaccharide-stimulated RAW 264.7 macrophage inducible nitric oxide synthase and nitric oxide production is decreased by an omega-3 fatty acid lipid emulsion. Journal of Surgical Research, 149(2), 296-302.
		&lt;/p&gt;&lt;/li&gt;&lt;/ol&gt;&lt;/section&gt;</EM></P><P>&lt;/div&gt;&lt;/div&gt;</P><P>&lt;/div&gt;</P><P>&lt;/body&gt;</P><P>&lt;/div&gt;</P></LI></OL></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Tec-Monterrey/ITESM14_results.html">http://2014.igem.org/Team:Tec-Monterrey/ITESM14_results.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tec-Monterrey/ITESM14_results.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tec-Monterrey/ITESM14_results.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_results.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tec-Monterrey/ITESM14_results.html&amp;oldid=399181" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>