"
Page
Discussion
View source
History
teams
Log in
 
Team:METU Turkey/characterization.html
From 2013.igem.org
Characterization
Home
Team
Project
Modeling
Circuit
Coumeric Acid
Kill Switch
Human Practice
HP in METU
HP WITH OTHER UNIVERSITIES
HP WITH PEOPLE
WORKS IN PROGRESS
Extras
Characterization
Notebook
Protocols
Safety
Attributions
Contact
KILL SWITCH MECHANISM:
 
For our kill switch system, we used a sense-antisense RNA mechanism under an IPTG inducible circuit. MazF which is a non-specific ribonuclease coding gene is used as killing agent against Bacillus subtilis. In our system, MazF is produced constitutively under Pveg promoter. Also the same Pveg promoter is responsible for constitutive LacI production. Anti MazF is an antisense RNA producing sequence which is reverse complement to first bases of MazF and its RBS sequence. Its production is regulated by pHyperspank promoter which is inhibited by LacI and reactivated by IPTG.
According to this kill switch mechanism, in a medium where there is no IPTG, LacI will bind to pHyperspank promoter and inhibit its work. By this way, Anti MazF antisense RNA cannot be produced. Therefore, MazF production will not be inhibited and it will kill bacteria. On the other hand, if there is IPTG in medium, it will bind to LacI and prevent inhibition of pHyperspank by LacI. As a result, Anti MazF system will work and an antisense RNA will be produced. Since this antisense RNA sequence will be reverse complement of mRNA produced from MazF gene, it will bind to this mRNA and prevent the protein expression. In this situation, though both Anti MazF and MazF mRNA’s will be produced, there will be no MazF protein expression and so bacteria will not die.
 
Characterization of kill switch mechanism:
 
To see if this kill switch system, and so Anti MazF, is working or not we performed transformation experiments with whole kill switch construct and spreaded on media which contain IPTG and without IPTG. Briefly, MazF construct (K1197006) and Anti MazF construct (K1197001) were assembled in pSB1C3 in E.coli. Then, this construct were amplified and inserted into B.subtilis vector, K823023. Then, this vector was linearized and transformed by electroporation. And it was plated on agar with or without IPTG. Also, overnight cultures of transformants in both LB liquid media w/IPTG and w/o IPTG was left overnight. And as theoretically expected, we saw that bacteria lived and reproduced in media which contain IPTG, while they cannot reproduce in media which do not contain IPTG in both liquid and solid media.
 
 
B.subtilis transformation results of Kill Switch construct in solid and liquid media with or without IPTG:
CHARACTERIZATION OF PROMOTERS
We have done the characterization of three promoters as models. RFP genes were transcribed using these promoters and the amount of RFP molecules were modeled by considering their brightness. The results are as follows:
1- K606040: Promoter Hyperspank B. subtilis & E. coli (iGEM11_Paris_Bettencourt)
We simulated the production of RFP under two different IPTG concentrations to observe its basal and maximum transcription rates.
As seen in the graph, when there is no IPTG with the increasing LacI expression, the RFP amount becomes constant and this means there is no more production of RFP.
 
 
 
 
 
As seen in this graph, with 5000 IPTG molecules, LacIs form complexes with IPTG. Therefore there is decreased inhibition on K606040 and RFP production continues and the amount keeps getting higher.
2- K823003 Pveg (iGEM12_LMU-Munich)
 
We did alignment by using CLUSTAL W between two Pveg promoters; K143012 and K823003. The result is as follows:
K823003 sequence has 237 bp and K143012 sequence has 97 bp as seen in the figure. We can conclude that K823003 includes K143012. After BLASTing the remaining part of the K823003 which is not same as K143012 there were no results for specific sequences for polimerase, activator and inhibator binding. This shows K823003 has alike promoter strength as K143012. Only the primers were desinged further from the main promoter region. So for this promoter we can use the promoter strength of the K143012.
3- K143012 Promoter veg: Constitutive Promoter for B. subtilis (iGEM08_Imperial_College)
We used known parameters (parameter strength=0.79 rpu) of K143012 to simulate RFP production by using this promoter. Even in maximum IPTG concentration,the efficiency of K606040 is 120 times lower than K143012. It confirms that ,this promoter ,Pveg, is one of the strongest promoters of B. subtilis.
This graph shows constitutive production of RFP by using K143012.
Retrieved from " http://2013.igem.org/Team:METU_Turkey/characterization.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
