"
Page
Discussion
View source
History
teams
Log in
 
Team:UMaryland/notebook
From 2014.igem.org
Wetlab Notebook
Homepage
Photos
Team
Meet Our Team Members
iGEM Page
Attributions
Sponsors
Project
Overview
Background
parts (BioBricks)
Results
future applications
Safety
Notebook
Wetlab Notebook
Human Practices
Overview | Mission Statement
Public School Outreach
HighSchool
Community Interaction
Public Awareness
Notebook
February to May
Faculty and students interested in participating in UMaryland iGEM began meeting in February. Smaller interest meetings had occurred before to spread the word, as this is our first year! In these meetings, ideas were thrown around and brainstormed, and the logistics of participating in the iGEM competition were discussed. Every week for the first few months, a different student would give a presentation on a project and wiki from a previous year of iGEM in order to give all members a feel for what sort of projects have been done and what to expect throughout the process. In the later months, students presented then their own ideas for what UMaryland iGEM should pursue in 2014. Near the end of the semester, the group finally decided which of the proposed projects we would take – the development of a biosensor which would detect for the eastern oyster pathogen, P. marinus, in real-time using CvGal1, a galactoside-binding protein (galectin) found in the hemocytes of oysters, which has a strong affinity for binding to P. marinus.
 
May
UMaryland iGEM began working in the lab at the end of May. Having chosen the outer membrane protein A (OmpA) to express the galectin on the surface of bacteria, the team ordered the part (BBa_K103006) and transformed it into DHα E. coli to create a stock of this plasmid. Team members also learned and refreshed on standard lab practices and protocols which would carry throughout most of the project: chemical transformation and bacterial culture, aseptic technique, minipreparation of a plasmid, and gel electrophoresis. From gel electrophoresis of the miniprep samples, cut at the unique BioBrick prefix and suffix with restriction enzymes EcoRI and PstI, as well as sequencing done by GENEWIZ, Inc., we could confirm that the OmpA plasmid was successfully cloned. This procedure would become our standard for cloning and confirming the identity of all subsequent plasmids.
 
June
The team also began mapping out the BioBrick constructs intended for this year.
The two constructs presented above differ only by the promoter sequence: the pBAD promoter is activated by arabinose, while the pLAC promoter is activated by IPTG. For this reason, the team also prepared a stock of pSB1C3-pBAD-RBS-GFP (BBa_K750000) and pSB1C3-pLAC-RBS-RFP (BBa_J04450) plasmids. By removing the fluorescent protein genes and linearizing the DNA, we would use these vectors as the basis for all the clones we intended to express, as the promoter, ribosomal binding site, and terminator are already part of the BioBrick pSB1C3 backbone.
Appropriately-designed primers were ordered from Integrated DNA Technologies (IDT) for preparation of Gibson assembly. With the primers, we PCR amplified the OmpA gene from its plasmid and the two vectors mentioned previously. Gels confirmed the unique sizes of the PCR products, and a NanoDrop spectrophotometer analyzed the DNA concentrations. The use of a NanoDrop machine would also become a standard in our DNA analysis, particularly for Gibson assembly, where concentrations are vital.
Due to issues with acquiring the CvGal1 gene, the team began a side project in the meantime. β-galactosidase, readily available as a BioBrick, was considered in place of CvGal1. Instead of binding a galactose sugar, as the galectin would, β-galactosidase would instead cleave it into monosaccharides. The LacZα (BBa_K329005) and full-length LacZ (BBa_K1155007) genes were prepared similar to OmpA, with the end result being PCR products of the isolated, desired LacZ gene. Since we were working with mainly DH5α E. coli, which have the LacZΩ fragment already present, the LacZα fragment expressed in DH5α is expected to combine and express the whole β-galactosidase. This mechanism was the preferred method, as the LacZα fragment is only around 240 bp in size. The full-length LacZ gene (approx. 3 kbp) was also used, however, in the event that the two-fragment method did not work.
The team also moved the OmpA gene from its original backone, the old pSB1A2 (ampicillin resistance), to the current BioBrick standard, pSB1C3 (chloramphenicol resistance). Using standard restriction enzyme cloning techniques and the pSB1C3 linearized backbone supplied by iGEM, the team was able to “cut-and-paste” the OmpA gene into its new vector. The process was fairly simple, due to the different antibiotic resistances acting as a general screen between the old and new plasmids. This was our first BioBrick made (BBa_K1489002), classifying as an improvement to an existing part.
 
July
Going into July, the team continued this LacZ side project. We encountered many roadblocks when attempting to clone the LacZ gene into the pSB1C3-pBAD-RBS vector (pBAD was eventually chosen as the promoter of choice for the rest of our project) via Gibson reactions. The only viable plasmid produced, screened through blue colonies on X-gal-coated plate, was an attempt at cloning OmpA-LacZα. However, both the gel and sequencing results revealed a serious error within the OmpA gene.
Around the same time, at the end of July, we had received both CvGal1 and bovine galectin-1 gene fragments, ordered from IDT’s gBlock system. Both gBlocks were based on the wildtype genes, but modified to ensure a proper codon reading frame and to remove potential problems with the expressed protein, such as disulfide bonds forming due to the oxidative nature of the extracellular matrix. Immediately, the team PCR amplified all the fragments for Gibson assembly, where we faced an unresponsive CvGal1 gene. Every other fragment looked correct on the gel, however, so the team proceeded with focusing solely on bovine galectin-1 while troubleshooting CvGal1.
 
August
August was filled with many trials, but two main goals: 1) Gibson cloning of the bovine galectin gene into the pSB1C3-pBAD-RBS backbone, one with fusion to the OmpA gene and one without, and 2) PCR amplification of the CvGal1 gBlock.
For the first goal, the team faced many issues with residual plasmid transforming, making it unclear whether the Gibson reaction was assembling the gene fragments properly. Finally, on August 15, it seemed we had at last cloned the bovine galectin into the vector (without OmpA). (Though done in parallel, the OmpA-Bovine galectin construct did not clone properly.) The gel revealed the plasmid’s identity clearly, but for the wrong reason – the bovine galectin gene had an EcoRI site in the middle of its sequence which had gone unnoticed, until this product was cloned and analyzed.
As for the CvGal1 gBlock, after various PCR attempts in which the team tested different Mg2+ concentrations, DMSO concentrations, cycle time and temperature conditions, and DNA polymerases, the team contacted IDT about the inability to amplify the gene fragment. To the time of this writing, they are still in the process of remaking the CvGal1 gBlock, as the persistent issue appeared to be on their end.
 
September
With the school semester starting again, our team members had much less time to spend in the lab. However, we continued to make progress, and toward the end of September, we had successfully cloned the OmpA-Bovine galectin construct as well. Persistence, fine-tuned procedures, and a bit of luck paid dividends as the sequencing results came back as a clear match.
The case of the unwanted EcoRI site remained within the bovine galectin genes, but the team was ready with a site-directed mutagenesis kit and primers. Mutagenesis on the EcoRI site to create a silent mutation was successful on its first try, much to our relief, and a second round of sequencing revealed that the EcoRI site in the bovine galectin gene in both constructs had been removed (but the BioBrick prefix’s EcoRI site remained intact).
 
These are our two main project BioBricks: the Bovine galectin alone in pSB1C3-pBAD-RBS (BBa_K1489000) and the Bovine galectin gene fused to the end of the OmpA gene in the same vector (BBa_K1489004). Additionally, since the pSB1C3 backbone has a SacI site (which pSB1A2 did not) and OmpA has a SacI site (as designed by the creators of that BioBrick), the pSB1C3-OmpA which we created back in June had two unusable SacI sites. In order to maintain the integrity of that BioBrick with its unique restriction sites (along with the standard EcoRI and PstI sites), we also performed SDM on the pSB1C3-OmpA (BBa_K1489002) to replace the now-useless SacI site at the end of the OmpA gene with a KasI site, creating our fourth BioBrick (BBa_K1489003).
October
With the BioBricks made, the team began preliminary protein expression tests. Multiple cultures of DH5α E. coli, transformed with either the bovine galectin or OmpA-BvGal1 BioBricks, were grown up to a noticeable OD. They were then induced with arabinose at increasing concentrations, from no induction to 0.2% m/v. After further incubation of the bacteria to allow for protein production, a 1 mL sample of each culture was pelleted. The cells were then lysed by resuspension in 20 μL SDS and boiling the solution, yielding a soluble protein fraction in the supernatant after centrifugation. The remaining pellet was boiled in 100 μL SDS with harsher conditions in order to extract as much insoluble protein as possible. These protein samples (soluble and insoluble fractions) were run through SDS-PAGE, followed by western blotting (taking advantage of the His6 tag placed in the synthesized bovine galectin-1). Anti-mouse was used for the secondary antibody. Expression of the bovine galectin BioBrick (BBa_K1489000) appeared strong and at the correct size, but expression of OmpA-BvGal1 (BBa_K1489004) featured weak bands in both the induced and uninduced samples, though of correct size.
Wetlab Notebook
Retrieved from " http://2014.igem.org/Team:UMaryland/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
