<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Dundee 2016</TITLE></P><DIV style="padding-right:5%; padding-bottom:0px;"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:Dundee" class="navbar-brand">UoD iGEM 2016</A></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Design">Design</A></LI><LI class="active"><A href="https://2016.igem.org/Team:Dundee/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Hardware">Hardware</A></LI></UL><LI><A href="https://2016.igem.org/Team:Dundee/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Proof">Proof of Concept</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Human_Practices">Human Practices Overview</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/HP/Silver">HP/silver</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/HP/Gold">HP/gold</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Engagement">Public Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Part_Collection">Parts Collection</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Experiments">Awards</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Team">Meet The Agents</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Dundee/Collaborations">Collaboration</A></LI></DIV></DIV><DIV id="BackImg"><H1><SPAN>Results</SPAN></H1></DIV><DIV class="container"><DIV id="SplashTitle"><P>Promoters</P></DIV></DIV><DIV id="container"><DIV id="SplashTitleUnder" style="padding-top:1%;"><P>pH Sensing Device</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Introduction</H3><P>Bacteria have the ability to sense their surroundings and respond to them by turning on genetic regulatory systems. One of the environmental stimuli to which bacteria may respond is pH<SUP>1</SUP>. Bacteria respond to environmental stimuli for a variety of reasons for example to trigger pathogenesis or survival genes, which help the bacterium, survive, thrive and compete in a new environment<SUP>1</SUP>.</P><P>We found several promoters that were submitted from previous iGEM teams, this included P<I>asr</I> (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1231000">BBa_K1231000</A>) and P<I>gadA</I> (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K1231001">BBa_K1231001</A>) both submitted by the <A href="https://2013.igem.org/Team:Northwestern">2013 Northwestern iGEM team</A>. These have been reported to be activated at low pH. So we started by characterising these two promoters and finding the best one suitable for our project. We cloned <I>gfp</I> (<A href="http://parts.igem.org/Part:BBa_E0840">BBa_E0840</A>) downstream of both of these promoters in order to measure expression levels (P<I>asr-gfp</I> (<A href="http://parts.igem.org/Part:BBa_K1962014
 ">BBa_K1962014</A>) and P<I>gadA-gfp</I> (<A href="http://parts.igem.org/Part:BBa_K1962013
 ">BBa_K1962013</A>). For our bile salt responsive promoter we used P<I>acrRA</I> (<A href="http://parts.igem.org/Part:BBa_K318514">BBa_K318514</A>), submitted by the <A href="https://2012.igem.org/Team:Wisconsin-Madison
 ">2012 Wisconsin –Madison
 </A> iGEM team, which consists of the <I>acrRA</I> operon found in <I>Salmonella enterica</I> strain. This operon contains the RamA binding sequence, the <I>acrR</I> gene and part of the <I>acrA</I> gene. We then cloned a <I>gfp</I> biobrick (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_E0840">BBa_E0840</A>) downstream of P<I>acrRA</I> (<A href="http://parts.igem.org/Part:BBa_K1962010
 ">BBa_K1962010</A>). Due to <I>E. coli</I> not containing its own RamA transcription factor we codon optimised the <I>ramA</I> gene sequence for expression in <I>E. coli</I> and got it synthesised by IDT as a gBlock gene fragment. RamA was also submitted by the Wisconsin-Maddison 2012 iGEM team (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K318516">BBa_K318516</A>) but was out of stock in the iGEM registry so we also submitted this as a biobrick (<A href="http://parts.igem.org/Part:BBa_K1962009
 ">BBa_K1962009</A>). </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>P<I>asr</I></H3><P>Enterobacteria can respond to low pH by de novo synthesis of specific proteins and altered levels of gene expression. The response to environmental stresses, such as pH, is often a complex mechanism and also depends on other environmental factors such as nutrition, the presence or absence of oxygen or starvation<SUP>2</SUP>. The <I>E. coli asr</I> (acid shock RNA) gene encodes small RNA, of about 450 nucleotides in length<SUP>2</SUP>. This gene is inducible by low external pH and contributes to the organism’s survival2. It has been suggested that <I>asr</I> genes may be regulated by the two component system PhoBR<SUP>2</SUP>.</P><P>In this two-component system the protons from the environment (H+) activate the sensory part (PhoR- in the periplasm) of the two-component system, which then transduces the signal to the activator protein- PhoB, which can bind promoter DNA of <I>asr</I><SUP>2</SUP>. The promoter region of <I>asr</I> was analysed and showed to contain a sequence similar to the <I>pho</I>-box; this is a consensus sequence able to bind to the PhoB protein<SUP>2</SUP>. These interactions of H+ from the external environment with this two-component system are thought to lead to <I>asr</I> transcription<SUP>2</SUP>. Interestingly the PhoBR system also controls the <I>pho</I> regulon which is induced by phosphate starvation, the link between these two factors is not fully understood however, it has been suggested that the level of Asr expressed in minimal media is higher than in enriched media<SUP>2</SUP>. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3><I>PgadA</I></H3><P>Bacteria have a variety of environmental response mechanisms; the GAD (glutamate decarboxylase) system in <I>E. coli</I> has been suggested to be the most effective response to environmental acidic conditions<SUP>3</SUP>. This system uses two main isoforms – <I>gadA</I> and <I>gadB</I> and a putative glutamate/Q-amino butyric acid antiporter encoded by <I>gadC</I><SUP>3</SUP>. By decarboxylation of glutamate the protons that leak into the cell can be consumed<SUP>4</SUP>. The end product, γ-aminobutyric acid (GABA), is then transported out of the cell by GadC4. The control of this system is very complex involving two repressors (H-NS and cyclic AMP receptor protein), one activator (GadX), one repressor activator (GadW) and two sigma factors (σS and σ70) <SUP>3</SUP>. 

</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Design</H3><P>The device was designed to be active upon ingestion at a pH of approximately 6 or below in order to release our synthetic colicins. The promoter is not constitutive and activates transcription of genes at a low pH. The pH range of the promoter was investigated in this project. </P><P>In 2013 the Northwestern iGEM team submitted two pH sensitive promoter P<I>asr</I> (<A href="http://parts.igem.org/Part:BBa_K1231000
 ">BBa_K1231000</A>) and P<I>gadA</I> (<A href="http://parts.igem.org/Part:BBa_K1231001
 ">BBa_K1231001</A>). In order to further characterise both of these promoters we cloned <I>gfp</I> (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_E0840">BBa_E0840</A>) downstream of both promoters (Fig 1). We then used this construct to measure and compare the GFP expression levels in response to different pH conditions. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 1:  Schematic representation of pH sensitive promoters-<I>gfp</I> constructs in pSB1C3 plasmid.</P></DIV><P>We wanted to test both our pH sensitive promoters, P<I>asr</I> and P<I>gadA</I> by monitoring for the production of GFP by western blotting. The western blots showed levels of GFP protein production by the cells at a range of pH values. In Fig 2A the expression of GFP at pH 5 is much higher than the other pH readings. This would indicate that at this pH the largest amount of GFP was being produced under the induction of the <I>gadA</I> promoter. The promoter is slightly leaky as there is also expression of protein within cells at other pH values. In Fig 2B the GFP protein expression under the regulation of P<I>asr</I> can be seen, the range of activity for this promoter is much wider indicating that this promoter is leakier than P<I>gadA</I>. The level of expression of GFP under the regulation of Pasr appears to be uniform with only a slight increase in GFP at pH5. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 2: GFP production under the control of pH sensitive promoters P<I>gadA</I> and P<I>asr</I>. Promoters with <I>gfp</I> cloned downstream of them were transformed into MG1655 <I>E. coli</I> cells, 5 ml of these cells were incubated for 16h at 37<SUP>o</SUP>C. After 16h, 200 uL of the overnight cultures were introduced into 1.8 ml of MOPS pH adjusted LB for the respective pH values indicated above. After 20 min at ambient conditions a 1ml aliquot was pelleted. The pellets were re-suspended in 100 ul=L Laemmli buffer and 15uL samples were then separated by SDS PAGE (12% acrylamide) and transferred to PVDF membrane followed by probing with anti-GFP antibody. Fig 1A - Shows the GFP production under the control of the P<I>gadA</I> and Fig 2B shows the GFP production under the control of the P<I>asr</I> promoter both in varying pH conditions.  </P></DIV><P>We then compared the results from the western blots to their respective coomassie-stained protein gels. Fig 3 shows the amount of total protein produced by the cells at the respective pH values. The low levels of general protein expression at pH5 (Fig 3A), emphasise the GFP expression observed in Fig 3B at pH5. Likewise, the high general protein expression observed in cells over pH 6 and 7 in Fig 3B lessen the value of the GFP expression observed at this respective pH in Fig 2B.

</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 3: Coomassie stained protein gels for each pH sensitive promoter-GFP construct.  SDS-PAGE gels were loaded with the same samples as shown in Fig 2, and stained with coomassie dye. Fig 3A shows total protein expression in cells containing P<I>gadA-gfp</I> on a pSB1C3 plasmid. Fig 3B shows total protein expression within cells containing the P<I>asr-gfp</I> construct on a pSB1C3 plasmid. Level of background protein expression gives a clearer overview of the protein expression observed in Fig2A and 2B. </P></DIV><P>The data gathered from SDS PAGE experiments gave us an overview of the pH sensitive promoter’s optimal pH range, we now wanted to be able to use a more quantitative approach to assign optimal promoter function conditions. To do this we used a 96 well plate reader to monitor GFP fluorescence and cell growth over the space of 20 hours. This experiment took obtained GFP fluorescence (excitation wavelength = 395nm and excitation wavelength = 509nm) and OD<SUB>600nm</SUB> values every 20 minutes yielding a large volume of data and more quantitative values for promoter function over 20 hours. To normalise flouresence measurements we used the following formula: </P><DIV id="CytoGifHeader"><P>GFP / OD<SUB>600nm</SUB> = Fluorescence per  unit absorbance </P></DIV><P>The data was then plotted to observe the trend over time as well as taking a snap shot at 16h which is the length of time that overnight samples were left before running SDS PAGE experiments as seen in Fig 2 and 3. The data collected from the plate reader experiments showed a slight difference in optimal pH for the pH sensitive promoter <I>asr</I>. As can be seen in Fig 4A and 4B the maximal fluorescence observed from GFP fluorescence per unit of absorbance over time was at a pH range of 8 which is significantly higher than what we were expecting from the results observed in Fig 3. This may have been due to the cell growth time. For P<I>gadA</I> we observed a coherent trend of optimal promoter function around pH5/pH6 as can be seen in Fig 4A and 4B. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 4: 96 well plate reader experiments. Full time frame showing trends in GFP per unit of absorbance over 20h. The two pH sensitive promoters cloned with downstream<I>gfp</I> were transformed into MG1655 <I>E. coli</I> cells, 5 ml of these cells were incubated for 16h at 37<SUP>o</SUP>C. After 16h, 200 ul of the overnight cultures were introduced into 1.8ml of MOPS pH adjusted LB for the respective pH values.  200 ul of each pH-cell buffered LB was transferred into each well. A negative control containing no cells and a negative control containing non-buffered fresh LB were also transferred in 200ul samples into the plate. Data represents the mean value of 3 repeat samples for each construct. Fig 4A - shows GFP fluorescence for P<I>gadA-gfp</I> and Fig 4B - shows GFP fluroescence for Pasr-gfp</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 5: 96 well plate reader experiments. 16h time snap-shot out of the total 20h which can be seen in Fig 4. 16h was the average time of overnight cultures thus this would have been enough time for the cell growth to stabilize and adapt to the conditions. Fig 5A - shows GFP fluorescence for P<I>gadA-gfp</I> and Fig 5B - shows GFP fluorescence for P<I>asr-gfp</I></P></DIV></DIV></DIV><DIV id="container"><DIV id="SplashTitleUnder" style="padding-top:1%;"><P>Bile Salts Sensing Device</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Introduction</H3><P>Bacteria that live in the intestine must be able to withstand the antimicrobial properties of bile salts<SUP>1</SUP>. Two mechanisms that bacteria have developed to withstand bile salts are modified membrane structures and efflux pumps to transport bile out of the cell<SUP>1</SUP>. </P><P>This part of the construct consists of biobrick <A href="">BBa_K318514</A> which consists of the <I>acrRA</I> operon containing a RamA binding site, <I>acrR</I> and <I>acrA</I> genes.</P><P><I>S. enterica</I> has multiple efflux pumps, including AcrAB. The transcription of proteins required to make this pump can be induced with the help of a transcription factor called RamA. This transcription factor recognises and is bound to by bile salts and then binds to the RamA binding site on the <I>acrRA</I> operon<SUP>2</SUP>. This regulatory binding causes increased transcription of the <I>acrAB</I> gene<SUP>2</SUP>. The efflux pump can then begin to remove bile salts out of the cell and thus protect the bacterium from it’s environment<SUP>1</SUP>.  To apply this concept to the timed lysis mechanism, a GFP reporter (part <A href="http://parts.igem.org/wiki/index.php/Part:BBa_E0840">BBa_E0840</A>) was cloned downstream of it. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3><I>acrRA</I>:</H3><P>The device was then further designed to lyse the cells upon entering the intestine thus to release the colicins produced within the cell during the time spent in the stomach. This was also a design to prevent the release of live GMO into the environment. First we wanted to characterise the bile salts sensitive promoter <I>acrRA</I> (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K318514">BBa_K318514</A>). This part was submitted by the <A href="https://2012.igem.org/Team:Wisconsin-Madison">2012 Wisconsin – Madison team</A>, which consists of the <I>acrRA</I> operon found in <I>Salmonella enterica</I>. This system requires a transcription factor known as RamA which binds to the <I>acrRA</I> operon and activated downstream genes. <I>ramA</I> (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K318516">BBa_K318516</A>) was previously submitted by the Wisconsin – Madison iGEM team in 2012, however, we were unable to obtain this part and so codon optimised the <I>ramA</I> gene for <I>E. coli</I> and had this synthesised by IDT as a gBlock gene fragment and submitted it as a biobrick. </P><P>We cloned gfp (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_E0840">BBa_E0840</A>) downstream of the <I>acrRA</I> promoter in order to detect changes in gene expression. RamA was cloned into the pUniprom vector downstream of the constitutive tat promoter. RamA was amplified with a C-terminal HA tag in order to detect expression of the protein.</P><DIV id="CytoGifHeader"><P> Schematic of pSB1C3-P<I>acrRA-gfp</I> construct and pUniprom - <I>ramA-HA</I>.  </P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Results</H3><P>To test this part, <I>ramA</I> (<A href="http://parts.igem.org/Part:BBa_K318516
 ">BBa_K318516</A>), which had been cloned into a pUniprom backbone. this vector was supplied by Professor Tracy Palmer and it contains a constitutive <I>tat</I> promoter for expression. The pSB1C3- P<I>acrRA-gfp</I> and pUniprom-<I>ramA-HA</I> were transformed into <I>E. coli</I> MG1655 cells, and plated onto cml/amp selective media. Colonies from the Lysogeny Broth (LB) transformation agar plate were streaked on MacConkey agar plates and left overnight at 37<SUP>o</SUP>C. They were then imaged with a fluorescence microscope to check for GFP expression (Fig 6). The LB agar plates were also viewed under a microscope with a GFP visualisation software to see whether the different nutrient agar plates affected promoter function (Fig 7). </P><DIV id="CytoGifHeader"><P>Figure 6:  Microscopy fluorescence imaging of acrRA-gfp with and without RamA transcription factor on LB nutrient agar plates.</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 7: Microscopy fluorescence imaging for P<I>acrRA-gfp</I> with and without RamA transcription factor on MacConkey agar plates. </P></DIV><P>The results from the GFP fluorescent images in Figures 6A-D was unexpected, as when only P<I>acrRa-gfp</I> is present there are large amounts of GFP fluorescence (Fig 6B). And there doesn't seem to much difference in the presence of  pUniprom-<I>ramA-HA</I> (Fig 6B). However, when the cells containing P<I>acrRA-gfp</I> and pUniprom-<I>ramA-HA</I> were plated onto MacConkey (Fig 7A) there was a larger signal of GFP fluorescence compared to when the RamA transcription factor was not present (Fig 7D). Similar results were seen when cells were grown in LB and an anti-GFP western blot was carried out (Fig 8). GFP was observed even in the absence of RamA, it is difficult to tell if there is any difference in the amount of GFP being detected.  
</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 8: MG1655 cells harbouring pSB1C3-P<I>acrRA-gfp</I> and pUniprom-<I>ramA</I> wre grown overnight at 37<SUP>o</SUP>C. 1 ml of the overnight culture was pelleted and re-suspended in 100 ul Laemmli buffer and 15 ul samples were then separated by SDS PAGE (12% acrylamide) and transferred to PVDF membrane followed by probing with anti-GFP antibody.   

</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>In order to further understand the role of RamA we first wanted to determine that we were able to detect the expression of RamA-HA. In Fig 9 the RamA transcription factor containing its HA tag was not observed, this may be due to the amount of RamA being produced by the cells being rather low. So we repeated the experiment using the Rosetta <I>E. coli</I> strain which contains a viral T7 polymerase, activated by IPTG. In Fig 10 you can see successful blotting for RamA-HA.</P><DIV id="CytoGifHeader"><P>Figure 9: Anti-HA tag western blot for pUniprom-<I>ramA</I> and pSB1C3 PacrRA-gfp MG1655 cells. </P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 10: pUniprom-<I>ramA</I> was transformed into Rosetta cells containing a viral T7 polymerase which is activated by IPTG. Anti-HA tag western blot for RamA in pUniprom within the Rosetta cells (DE3). Cells were subcultured from an overnight at 37<SUP>O</SUP>C. Once an OD of 0.4 was reached the samples were added to a final concentration of 1mM IPTG. 1ml of this was pelleted and Laemlli buffer added before loading 15 µl samples onto the protein gel. SDS PAGE (12% acrylamide) was run and transferred to PVDF membrane followed by probing with anti-HA antibody.</P></DIV><P>We then conducted a plate reader experiment to better determine whether the presence or absence of bile salts (in this case Sodium Cholate, which is the salt of cholic acids) would make a significant difference to the activation of the promoter and to test whether the promoter would still be active in minimal media. From Fig 11 we can see that the promoter construct (P<I>acrRA-gfp</I> + <I>ramA</I>) is active with and without the addition of sodium cholate however, the activity in its presence is higher. Sodium cholate appears to have an effect but it’s significance is debatable. Figure 12 shows the activity of all constructs in the presence of Sodium cholate and in the absence of the transcription factor (RamA) the promoter is being activated. In conclusion we have discovered that this part is active in nutrient broth, MacConkey agar, sodium cholate-minimal media broth and minimal media without sodium cholate, we do note that in the presence of bile-like salts the activity of the promoter is increased. The mechanisms of how this works on a molecular level are still unknown. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Figure 11: 96 well plate reader experiment, measuring OD<SUB>600</SUB>nm and GFP fluorescence over 20h. pSB1C3-<I>acrRA-gfp</I> transformed with or without pUniprom- <I>ramA</I>. Control consists of both empty pUniprom and empty pSB1C3. 16h overnights were grown at 37<SUP>o</SUP>C and then normalized to an OD<SUB>600nm</SUB> of 1 with minimal media. Stock of Sodium cholate (in sterile water) was diluted with minimal media to make up to a concentration of 10µg/ml. The control contains no Sodium cholate. 6µl of normalized sample was added to 194µl of either Sodium Cholate minimal media or Sodium cholate free minimal media (control)
Fig 10 showing the GFP fluorescence per unit of absorbance in the presence of Sodium Cholate in the different constructs. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P> Figure 12: 96 well plate reader experiment, measuring OD<SUB>600</SUB>nm and GFP fluorescence over 20h. pSB1C3-<I>acrRA-gfp</I> transformed with or without pUniprom- <I>ramA</I>. Control consists of both empty pUniprom and empty pSB1C3. 16h overnights were grown at 37<SUP>o</SUP>C and then normalized to an OD<SUB>600nm</SUB> of 1 with minimal media. Stock of Sodium cholate (in sterile water) was diluted with minimal media to make up to a concentration of 10µg/ml. The control contains no Sodium cholate. 6µl of normalized sample was added to 194µl of either Sodium Cholate minimal media or Sodium cholate free minimal media (control). Fig 11 showing the difference in GFP fluorescence per unit absorbance when pSB1C3-P<I>acrRA-gfp</I> is grown in the presence or absence of Sodium cholate (10µg/ml)</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>The device was then further designed to lyse the cells upon entering the intestine thus to release the synthetic colicins produced within the cell during the time spent in the stomach. This was also a design to prevent the release of live GMO into the environment. The cloning of pSB1C3 -<I>chiWX</I> was in progress during the wiki freeze.</P><DIV id="CytoGifHeader"><P>Schematic to show construct design, P<I>acrRA</I> bile salt promoter followed by genes from a lysis cassette taken from <I>Serratia marcescens</I> consisting of chiW and chiX taken from Prof. Sargent’s Lab at Dundee University.   </P></DIV></DIV></DIV><DIV id="container"><DIV id="SplashTitleUnder" style="padding-top:1%;"><P>Bacteriocins</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Introduction</H3><P>Bacteriocins are proteins secreted by bacteria to kill other similar species of bacteria in order to gain a competitive advantage within a niche. They have 3 common domains; a receptor binding domain, which binds to specific receptors on the surface of the target cell; a translocation domain, responsible for getting the bacteriocin into the target cell; and a cytotoxic domain, responsible for target cell damage. The cytotoxic domain can take many forms, it can be a DNase or RNase, or can have pore forming or peptidoglycan endopeptidase activity to name a few.</P><P>Bacteriocins have corresponding immunity proteins which are expressed by their producing cell. This protein serves the function of inhibiting the toxic nature of its cognate toxin. This protein works by taking part in a protein-protein interaction with the previously mentioned toxin and dissociates from this complex when the bacteriocin is released from the cell resulting in the toxin being activated and able to attack target cells.</P><DIV id="CytoGifHeader"><P>Bacteriocins and immunities: The producing cell contains an immunity protein which inhibits the bacteriocins that it produces. When the bacteriocins leave this cell, the immunity protein remains in the producing cell, making it immune to the toxin as it leaves the cell and attacks the target cells. </P></DIV><P>Since bacteria are constantly evolving resistance to antibacterials, we needed to come up with a way to prevent them from becoming resistant to our bacteriocins. We decided to truncate the bacteriocin to remove the cytotoxic domain and incorporate a multiple cloning site onto the receptor binding domain of our bacteriocins, this enables variation of our bacteriocins cytotoxic domain, ensuring that our target bacteria will not be able to recognise the bacteriocin.</P><P>Colicins are the name given to the bacteriocins which are secreted by <I>Escherichia coli</I> to target other strains of <I>E. coli</I> and similar bacteria such as <I>Salmonella enterica</I>, which are common causes of infection in livestock, therefore these are the bacteriocins we decided to work with.</P><P>First we wanted to test the toxicity of full length, unmodified colicins, so we can compare this to the toxicity of our modified colicins. We started working with colicin Ia, E3 and E9.</P></DIV></DIV><DIV id="container"><DIV id="SplashTitleUnder" style="padding-top:1%;"><P>Colicin Ia/E3/E9</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>Colicin Ia is a protein that is used by species of <I>E. coli</I> in order to kill closely related species of <I>E. coli</I> in certain circumstances e.g. competition. The producer cell normally contains a protein which confers immunity to this toxin, which in this case is <A href="http://parts.igem.org/Part:BBa_K1962001">BBa_K1962001</A>. And this protein is believed to form a complex with the cytotoxic domain of the bacteriocin rendering it inactive. When this bacteriocin is secreted the immunity protein dissociates from the complex leaving the cytotoxic domain active resulting in the pore forming activity of this bacteriocin returning and having effect on the target cells. The pore forming activity of this bacteriocin results in the depolarisation of the bacterial inner membrane resulting in the collapse of the proton motive force.</P><DIV id="CytoGifHeader"><P>Structure of colicin Ia: The three domains of colicin Ia, the cytotoxic domain: blue, the Translocation domain: red, Channel forming domain: green. </P></DIV><P>It is also ribosomally synthesised and is present in the cytoplasm when this process is complete. When this protein is present in the cytoplasm of the host cell, it always exists in a high affinity complex with its immunity protein, (this interaction has been observed to be in the femtomolar range, which makes this one of the associations observed in protein complexes). The function of this is to prevent the DNase activity of this colicin effecting the host cell.  This method of translocation occurs by the Tol system of membrane proteins as this belongs to group A. The receptor on target cells that this colicin usually makes use of in order to gain entry is the same receptor used by vitamin B12, BtuB. It is thought that when the colicin binds this receptor a change in conformation occurs resulting in the dissociation of the immunity-colicin complex. </P><DIV id="CytoGifHeader"><P>Structure of E9: The three domains of colicin E9, the cytotoxic domain: blue, the Translocation domain: red, Channel forming domain: green. </P></DIV><P>This protein achieves its aim of killing target cells through its DNase activity. This comes from the 15kDa C-terminal domain where the DNA hydrolysis occurs. This colicin also has the ability to form ion channels in target cells lipid bilayers, similar to a pore forming colicin, however these channels do not result in death. It is believed that these ion channels have a function in allowing the DNase C-terminal domain to translocate across the membrane in order to kill the target. The operon encoding this colicin contains a lysis gene as the last gene of the operon. The function of the <I>cxl</I> gene is to produce a lysis protein in order to release the colicin into the extracellular medium in order to carry out its function. The lysis protein is always produced at the same time as the colicin however, in lower amounts than the colicin. </P><H3>Colicin E3 </H3><P>The sequence of this colicin was obtained online from the Uniprot database (P00646) and a synthetic gene obtained from IDT. </P><P>This colicin is also ribosomally synthesised and is present in the cytoplasm when this process is complete. When this protein is present in the host cells cytoplasm, it always exists in a high affinity complex with its immunity protein. As with other colicins the complex that is formed between colicin E3 and E3 immunity remains present until the colicin reaches its target cell, this means that the immunity protein is able to leave the host cell along with the colicin. The mechanism of uptake used by this colicin is very similar to that of colicin Ia and colicin E9 as it makes use of the vitamin B12 receptor, BtuB. It is at this point that the colicin-immunity protein complex dissociates and a new complex forms between the receptor and the protein. It is the formation of this complex that recruits the Tol membrane proteins along with OmpF, an outer membrane protein involved in the translocation of this colicin.  </P><P>This colicin achieves its goal of killing target cells by making use of its 16s rRNase activity. This activity comes from the proteins cytotoxic domain and its effect in the cell is to destroy the 16s rRNA subunit of the cells ribosome. The effect of this process is to prevent cell growth by making the target cell unable to produce any proteins thus causing cell death.</P><DIV id="CytoGifHeader"><P>Structure of E3: The three domains of colicin E3, the cytotoxic domain: blue, the Translocation domain: red, Channel forming domain: green. </P></DIV></DIV></DIV><DIV id="container"><DIV id="SplashTitleUnder" style="padding-top:1%;"><P>Modified Bacteriocins</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>As well as cloning the full length colicin our idea was to create a truncated version of colicin Ia and colicin E9. <A href="http://parts.igem.org/Part:BBa_K1962002">BBa_K1962002</A> and <A href="http://parts.igem.org/Part:BBa_K1962006">BBa_K1962006</A>
 encode a truncated version of colicin Ia and E9 that lacks the C-terminal bacteriocin domain. These parts have been engineered to contain a multiple cloning site at the 3' end, preceding the RFC [10] suffix and regulation double stop codons, containing the following in-frame restriction sites: BamHI, KpnI, SalI, BglII and NheI. The presence of this multiple cloning site will allow for different toxic domains to be fused at the C-terminus of the truncated colicin and thereby generating a synthetic colicin in a rapid and facile manner. </P><DIV id="CytoGifHeader"><P>Colicin Ia with MCS: The truncated colicin Ia contains a multiple cloning site containing the restriction sites BamHI, KpnI, SalI, BglIII and NheI.</P></DIV><DIV id="CytoGifHeader"><P>Colicin E9 with MCS: The truncated colicin E9 contains a multiple cloning site containing the restriction sites BamHI, KpnI, SalI, BglIII and NheI. </P></DIV><P style="text-align:center;">We decided to select a number of warheads. </P><DIV id="CytoGifHeader"><P>Truncated bacteriocins with varying warheads: Our modified bacteriocins do not have cytotoxic domain, so that different cytotoxic domains, or warheads, can be readily added. </P></DIV><P>The warheads we have selected to add to the MCS of our truncated colicins are toxins Ssp1 and Ssp2 secreted from <I>Serratia marcescens</I>.</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>	Design</H3></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>Our idea was to design the constructs with our colicins under the control of our characterised pH sensitive promoters (<A href="http://parts.igem.org/Part:BBa_K1962013">BBa_K1962013</A> and <A href="http://parts.igem.org/Part:BBa_K1962014">BBa_K1962014</A>) and bile salts sensitive promoter (<A href="http://parts.igem.org/Part:BBa_K1962010">BBa_K1962010</A>). Therefore, in the presence of the chicken GI tract the colicins will be released in the stomach and upon interaction with bile salts in the intestine further colicins would be released. In order to do this we would first need to clone the colicin Ia/E3/E9 immunity protein with an RBS downstream of the pH and bile salt promoters to protect our producing cell However, we only successfully managed to clone the colicin Ia and E3 immunities (<A href="http://parts.igem.org/Part:BBa_K1962015">BBa_K1962015</A>, <A href="http://parts.igem.org/Part:BBa_K1962016">BBa_K1962016</A>, <A href="http://parts.igem.org/Part:BBa_K1962012">BBa_K1962012</A>, <A href="http://parts.igem.org/Part:BBa_K1962011">BBa_K1962011</A>). These composite parts were submitted to the igem registry. 

	         
			 </P><DIV id="CytoGifHeader"><P>Image A:  P<I>asr-col Ia-im</I>. The gene sequence for the colicin Ia Immunity protein was cloned downstream of the P<I>asr</I> and RBS (BBa_B0030).Image B: P<I>asr-col E3-im</I>. The gene sequence for the colicin E3 Immunity protein was cloned downstream of the P<I>asr</I> and RBS (BBa_B0030).
</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Results</H3></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>The immunity proteins for colicin Ia and E3 were cloned in front of the <I>asr</I> pH sensitive promoter and, on a separate plasmid, the <I>acrRA</I> bile salt promoter containing a pSB1C3 backbone. A HA tag was fused onto the C-terminal of the immunity proteins to allow visualisation of expression via western blotting.</P><DIV id="CytoGifHeader"><P>Western blot of colicin Ia and E3 immunities with p<I>asr</I>: The col Ia and E3 immunities were amplified with a HA tag to detect expression via a western blot. The <I>asr</I> promoter was blotted alongside as a control. No expression of the immunities in front of the <I>asr</I> promoter was detected.
 </P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>The immunity proteins for colicin Ia and E3 were cloned in front of the <I>acrRA</I> bile salt promoter containing a pSB1C3 backbone. A HA tag was fused onto the C-terminal of the immunity proteins to allow visualisation of expression via western blotting.</P><DIV id="CytoGifHeader"><P>Western blot of colicin Ia and E3 immunities with bile salt sensitive p<I>acrRA</I>: The col Ia and E3 immunities were amplified with a HA tag to detect expression via a western blot. Empty pSB1C3 and <I>acrRA</I> promoter was blotted alongside as a control. The constructs with ramA were also blotted to see if this has any effect on <I>acrRA</I> function. No expression of <I>acrRA</I> col Ia immunity was detected however we were able to see expression of the E3 immunity in the <I>acrRA</I> col E3 immunity with and without ramA.  </P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>We then attempted to clone in the full length colicin Ia and E3 in front of its immunity in the <I>asr</I> and <I>acrRA</I> immunity, but we were unsuccessful. This could be because the immunity was not being expressed, since we never managed to show its expression via a western blot. </P><P>In the future, we would like to clone the colicin Ia immunity in front of the <I>gadA</I> pH promoter, followed by the full length colicin Ia. We then moved onto working with our colicin chimeras. </P></DIV></DIV><DIV class="container"><DIV id="SplashTitleUnder"><P>Our Warheads</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>Ssp1 and Ssp2 is an antibacterial toxin secreted from the Type VI secretion system (T6SS) in <I>Serratia marcescens</I>. Bacteria contain numerous independent systems to facilitate protein transport across the cell membrane. Gram negative bacteria have evolved six general secretion systems, Type I-VI.  Type VI secretion systems span both bacterial membranes, and deliver secreted proteins directly from the cytoplasm into the target cell. T6SS in <I>Serratia</I> secretes Ssp1 and Ssp2 which is an antibacterial toxin which kills closely related cells and has peptidoglycan endopeptidase activity, degrading peptidoglycan in the membrane of the target cell.</P><P>Ssp1 and Ssp2 from <I>S .marcescens</I> were cloned onto multiple cloning sites on the C-terminal of the receptor binding domain of truncated colicins Ia and E9, resulting in modified <I>col Ia-ssp2</I>, <I>col E9-ssp1</I> and <I>col E9-ssp2</I>. These were fused with a HA-tag then cloned into pBAD18 vector with pBAD promoter of the araBAD operon. pBAD can be repressed with glucose and induced with arabinose. The modified colicins were ligated with pBAD18 and grown on LB plates with kanamycin resistance and 0.5% glucose, to repress expression of the toxin while it is taken up by cells. 
</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>p<I>col Ia-ssp2</I> fusion in pBAD: p<I>col Ia-ssp2</I> fusion was amplified with a HA tag for detection by western blot and cloned into a pBAD18 vector with an inducible pBAD promoter so that expression of the toxin can be controlled. 
</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>Western blots were carried out to determine if our fusion proteins were being expressed:
</P><DIV id="CytoGifHeader"><P>Detection of Ha-tagged p<I>col Ia-ssp2</I>: Single colonies of p<I>col Ia-MCS</I> and p<I>col Ia-ssp2</I> 2:1 and 3:1 were grown overnight in 5ml LB with 0.5% glucose at 37°C. 250μl of cells were then grown in 25ml of LB at 37°C until they reached an OD of 0.5A. They were then represssed with 0.2% glucose and induced with 0.2% arabinose and 0.5% arabinose and incubated at 37°C for 5 hours. 1ml sample of each was taken, pelleted, then 100μl of laemmli sample buffer and B-mercaptoethanol was added to each and they were boiled for 10 mins. 5μl of pre-induced and induced samples were separated by SDS-PAGE (10% acrylamide) transferred to membrane and probed with HA-antibody. Expression of p<I>col Ia-ssp2</I> was detected when induced by both concentrations of arabinose. 
</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>p<I>col Ia-ssp2</I> fusion in pBAD: p<I>col Ia-ssp2</I> fusion was amplified with a HA tag for detection by western blot and cloned into a pBAD18 vector with an inducible pBAD promoter so that expression of the toxin can be controlled. 
</P></DIV><DIV id="CytoGifHeader"><P>p<I>col Ia-ssp2</I> fusion in pBAD: p<I>col Ia-ssp2</I> fusion was amplified with a HA tag for detection by western blot and cloned into a pBAD18 vector with an inducible pBAD promoter so that expression of the toxin can be controlled. 
</P></DIV><DIV id="CytoGifHeader"><P>Detection of Ha-tagged p<I>col E9-ssp1</I> and <I>ssp2</I>: Single colonies of p<I>col E9-ssp1</I> and p<I>col E9-ssp2</I> were grown overnight in 5ml LB with 0.5% glucose at 37°C. 250μl of cells were then grown in 25ml of LB at 37°C until they reached an OD of 0.5A. They were then represssed with 0.2% glucose and induced with 0.2% arabinose and 0.5% arabinose and incubated at 37°C for 5 hours. 2ml sample of each was taken, pelleted, then 100μl of laemmli sample buffer and B-mercaptoethanol was added to each and they were boiled for 10 mins. 20μl of samples were separated by SDS-PAGE (10% acrylamide) transferred to membrane and probed with HA-antibody. p<I>col E9-ssp1</I> and p<I>col E9-ssp2</I> were expressed when induced with arabinose at both concentrations.
</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>Once we had confirmed that our fusion proteins were being expressed, we needed to see if they were toxic.
</P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Testing toxicity of modified colicins - Plate Toxicity</H3><DIV id="CytoGifHeader"><P>Testing toxicity of p<I>col Ia-ssp2</I>: A group of colonies of p<I>col Ia-MCS</I> and p<I>col Ia-ssp2</I> were added to 1ml LB and OD adjusted to 1.0A, then serial diluted 1/10 6 times. These were then spotted onto 0.2% glucose, 0.01% arabinose and 0.1% arabinose plates. No toxicity was observed.</P></DIV></DIV></DIV><DIV id="CytoGifHeader"><P>Testing toxicity of p<I>col Ia-ssp2</I> and p<I>col E9-ssp1</I>and p<I>col E9-ssp2</I>: A group of colonies of p<I>col Ia-MCS</I> and p<I>col Ia-ssp2</I>, p<I>col E9-ssp1</I> and p<I>col E9-ssp2</I> were added to 1ml LB and OD adjusted to 1.0A, then serial diluted 1/10 6 times. These were then spotted onto 0.2% glucose, 0.2% arabinose and 0.4% arabinose plates. No toxicity was observed.</P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P>From the plate toxicity tests we would expect to see inhibition of cell growth in the areas spotted with p<I>col Ia-ssp2</I>, p<I>col E9-ssp1</I> and <I>ssp2</I>. However, the cells grow normally in these areas and do not differ significantly from the p<I>col Ia-MCS</I> control.
We concluded that although our toxins were being expressed, they were not being secreted by the cell. Therefore we would need to find a way of enabling our toxins to exit their producing cell and reach their target cell. So we decided to design a lysis cassette to include in our construct. 
While our lysis construct was being put together we did some lysis experiments in the lab to test this concept. </P></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><H3>Lysis Experiment</H3><P>When the plate toxicity tests presented a negative result, we concluded that the modified colicins may not be being secreted by the cell, therefore we lysed the cells and spotted them on MG1655 <I>E. coli</I> to test their toxicity.When the plate toxicity tests presented a negative result, we concluded that the modified colicins may not be being secreted by the cell, therefore we lysed the cells and spotted them on MG1655 <I>E. coli</I> to test their toxicity.</P><DIV id="CytoGifHeader"><P>Diagram showing the cell lysis protocol. </P></DIV></DIV><DIV id="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>Testing toxicity of p<I>col Ia-ssp2</I> after lysing cells: Cells containing p<I>col Ia-ssp2</I> were lysed for 30 seconds using a sonicator. 5μl of the lysate was then spotted onto agar plated with MG1655 cells. p<I>col Ia-MCS</I> with glucose and arabinose, p<I>col Ia-ssp2</I> glucose and PBS were spotted as negative controls, and ampicillin was plated as a positive control. The plates were then incubated at 37°C overnight.</P></DIV><P style="text-align:center;">The experiment was repeated, but the cells were sonicated for longer.</P><DIV id="CytoGifHeader"><P>Testing toxicity of p<I>col Ia-ssp2</I> after lysing cells: Cells containing p<I>col Ia-ssp2</I> were lysed for 1.5 minutes using a sonicator. 20μl of the lysate was then spotted onto agar plated with MG1655 cells. p<I>col Ia-MCS</I> with glucose and arabinose, p<I>col Ia-ssp2</I> glucose and PBS were spotted as negative controls, and ampicillin was plated as a positive control. The plates were then incubated at 37°C overnight. </P></DIV></DIV></DIV><DIV class="container"><DIV id="ProjectDesCon"><P style="text-align:center;">We then tested the toxicity of our p<I>col E9-ssp1</I> and <I>ssp2</I> fusions:</P><DIV id="CytoGifHeader"><P>Testing toxicity of p<I>col E9-ssp1</I>and p<I>col E9-ssp2</I> after lysing cells: Cells containing p<I>col E9-ssp1</I> and cells containing p<I>col E9-ssp2</I> were lysed for 1.5 minutes using a sonicator. 20μl of the lysate was then spotted onto agar plated with MG1655 cells. Empty pBAD vector and PBS were spotted as negative controls, and ampicillin was plated as a positive control. The plates were then incubated at 37°C overnight. </P></DIV><P style="text-align:center;">
We also subcultured our cells and monitored their growth with and without our modified toxins.
</P><DIV id="CytoGifHeader"><P>Growth curve comparing the growth of p<I>col Ia-ssp2</I> cells when the toxin is repressed and induced: Overnights of p<I>col Ia-ssp2</I> and p<I>col Ia-MCS</I> (control) were subcultured in 25ml LB until they reached an OD of 0.67A. 0.2% glucose was added to one culture and 0.5% arabinose was added to the other. These were then incubated for a total of 5.5 hrs, with the OD measured at regular intervals. No significant difference in the growth of those induced compared to those repressed was observed.</P></DIV><DIV id="CytoGifHeader"><P>Growth curve comparing the growth of p<I>col E9-ssp1</I> and p<I>col E9-ssp2</I> cells when the toxin is repressed and induced: Overnights of p<I>col E9-ssp1</I> and p<I>col E9-ssp2</I> were subcultured in 25ml LB until they reached an OD of 0.7A. 0.2% glucose was added to one culture, 0.2% arabinose and 0.5% arabinose was added to the others. These were then incubated for a total of 5 hrs, with the OD measured at regular intervals. No significant difference in the growth of those induced compared to those repressed was observed.  </P></DIV><P style="text-align:center;">Our modified toxins did not display the desired effect in the lysis experiment, or when the growth curves where analysed. However, we believe that this may be due to the fact that the Ssp1 and Ssp2 warheads may not be compatible with our construct. Ssp1 and Ssp2 are very small proteins and therefore may not work well when attached to our large truncated colicins. However, the beauty of our concept is that warheads can be readily added to the colicins, therefore different combinations can easily be assembled in order to achieve the desired result. In future we would plan to characterise the toxicity of a purified version of our colicin. As a next step we would like to clone a predicted pore forming warhead (POW) onto colicin Ia. POW is believed to have homology with colicin Ia and therefore may be more likely to work.

</P></DIV></DIV><DIV id="SplashTitleUnder"><P>References</P></DIV><DIV class="container"><DIV id="ProjectDesCon"><DIV id="CytoGifHeader"><P>1: Terry A. Krulwich, George Sachs and Etana Padan,‘Molecular aspects of bacterial pH sensing and homeostasis’, Nature Reviews Microbiology 2011; (9); 330-343.</P><P>2. Edita Sužiedėlienė, Kęstutis Sužiedėlis, Vaida Garbenčiūtė, and Staffan Normark1,‘The Acid-Inducible asr Gene in <I>Escherichia coli</I>: Transcriptional Control by the phoBR Operon’, J Bacteriol. 1999; 181(7): 2084–2093.</P><P>3. Scott R. Waterman, P.L.C. Small, ‘Identi¢cation of the promoter regions and cs -dependent regulation of the gadA and gadBC genes associated with glutamate-dependent acid resistance in Shigella flexneri’, FEMS Microbiology Letters, 2003; 225: 155-160</P><P>4. Marie-Pierre Castanie-Cornet, Thomas A. Penfound, Dean Smith, John F. Elliott and John W. Foster, ‘Control of Acid Resistance in <I>Escherichia coli</I>’, J Bacteriol. 1999 Jun; 181(11): 3525–3535. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>