<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Schools_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee Schools/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Dundee_Schools">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Dundee_Schools/Team"> Meet the Agents  </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Collaborations"> Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Description">   Description </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Design"> Design </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Basic_Part"> Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Composite_Part"> Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Part_Collection"> Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Dundee_Schools/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Dundee_Schools/Attributions"> ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/HP/Silver"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/HP/Gold">Gold </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Integrated_Practices"> Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Engagement"> Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Model">Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Dundee_Schools/Medals">Medals</A></LI></DIV><DIV class="content_wrapper"><H1 id="team_name"> Dundee Schools </H1><H4 id="page_name"> Experiments </H4><TABLE><TBODY><TR><TH colspan="2">Standard PCR reaction (final volume = 50 μl)
 </TH></TR><TR><TH>Component</TH><TH>Volume (μl)</TH></TR><TR><TD>5x Herculase buffer</TD><TD>10</TD></TR><TR><TD>100 μM forward primer</TD><TD>0.5</TD></TR><TR><TD>100 μM reverse primer</TD><TD>0.5</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.5</TD></TR><TR><TD>DMSO</TD><TD>2.5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>34.5</TD></TR><TR><TD>gDNA</TD><TD>1</TD></TR><TR><TD>Herculase II</TD><TD>0.5</TD></TR></TBODY></TABLE><H4>Agarose Gel Electrophoresis</H4><P>DNA products were run on a 1% (w/v) agarose gel prepared in 1 x tris-acetate-EDTA (TAE) buffer containing GelRed (in a ratio of 1:20,000) to stain the nucleic acids. The sample was mixed with 10 x DNA loading dye and loaded on the gel with the DNA marker (Roche). Gels were then run in 1 x TAE buffer for 30 min at 100 V. After sufficient separation the gel was visualised under UV light on the BioRad GelDoc.

TAE Buffer (pH 7.6): 40mM Tris, 20mM acetic acid, 1mM EDTA</P><H4>Gel Extraction of DNA products</H4><P>DNA fragments were excised from agarose gels and the QIAquick Gel Extraction Kit protocol was followed (Qiagen) according to manufacturer’s instructions. DNA eluted in 50 μl of ddH<SUB>2</SUB>O.</P><H4>Double Restriction Digests</H4><P>Restriction digests were performed to prepare DNA for ligation.
60 μl reactions following clean up after PCR were set up:</P><OL><LI>50 μl PCR product</LI><LI>1.5 μl restriction enzyme 1</LI><LI>1.5 μl restriction enzyme 2</LI><LI>6 μl cognate digestion buffer for enzyme(s)</LI><LI>1 μl ddH<SUB>2</SUB>O</LI></OL><P>Digests were carried out using enzymes supplied by Roche and New England BioLabs and incubated at 37°C for 3 hours. A 2.5 μl aliquot of alkaline phosphatase was added to restriction digests of vectors at 2 hours and a second aliquot at 2.5 hours.</P><H4>PCR purification</H4><P>Following restriction digest, DNA products were then cleaned using the QIAquick PCR purification kit (Qiagen) according to the manufacturer’s instructions and eluted in 50 μl ddH<SUB>2</SUB>O.</P><H4>Transformation and Competent Cells</H4><OL><LI>Add 100μl of thawed (defrosted) competent cells into a chilled clean Eppendorf with up to 10 uL of the DNA you wish to transform</LI><LI>Rest on ice for 5 minutes</LI><LI>Heat shock at 42<SUP>°</SUP>C for 90 seconds</LI><LI>Rest on ice for 5 minutes</LI><LI>Add 1μl liquid broth and incubate at 37<SUP>°</SUP>C for 1 hour</LI><LI>Spin down at full speed for 10 minutes and remove supernatant</LI><LI>Add 50μl LB and plate</LI><LI>Leave in hot room over night</LI></OL><H4>Overnight Culture</H4><OL><LI>Add 10μl liquid broth in tube</LI><LI>Add 10μl antibiotic</LI><LI>Scrape colonies and put clean pipette tip in solution</LI><LI>Store in hot room overnight</LI></OL><H4>Mini Prep</H4><P>For isolation of plasmid DNA, the QIAprep Spin Miniprep Kit (Qiagen) was followed according to the manufacturer’s instructions and DNA eluted in 50 μl of ddH<SUB>2</SUB>O.</P><H4>Colony PCR</H4><OL><LI>Take a clean petri dish and spot 12 μL H<SUB>2</SUB>O on to the plastic for every colony you wish to test.</LI><LI>Pick up your colony and resuspend it in the spot of water you just made</LI><LI>Take 3 μL of this and spot it on to a fresh LB plate (or other appropriate)</LI><LI>Take another 3 μL and add to PCR reaction mix</LI><LI>In the PCR program, add an extra 15 min lysis step at 95<SUP>o</SUP>C</LI></OL><H4>Making spiRNA</H4><OL><LI>Design primers to amplify the entire spiRNA plasmid (BBa_K1963003 or BBa_K1963004) that add a 24nt sequence of perfect homology to target gene</LI><LI>PCR amplify the plasmid using these primers and a DNA polymerase that gives a product with blunt ends</LI><LI>PCR purify this and ligate 5 μL of the purified sample in a 10 μL reaction</LI><LI>Transform in to JM110 competent cells</LI><LI>Miniprep and sequence the resulting plasmid</LI></OL><H3>SDS</H3><H4>Resolving gel</H4><P>Recipe for 12% acrylamide gel:</P><UL><LI>Water - 2.1 ml</LI><LI>Acrylamide (30% stock) - 4 ml</LI><LI>Tris HCl, 1M pH 8.8 - 3.75 ml</LI><LI>SDS 10% w/v - 100 μl</LI><LI>APS 10% w/v - 50 μl</LI><LI>TEMED - 5 μl</LI></UL><P>After casting the resolving gel, you should add a layer of isopropanol on top, the overlay, to get rid of any bubbles and keep a straight line on the top. Once the resolving gel has set you can pour it off by tipping the cassette or by using some blotting paper to sook it out. Then add the stacking gel right to the top and <B>insert the comb immediately</B>.</P><H4>Stacking gel</H4><UL><LI>Water - 2.16 ml</LI><LI>Acrylamide (30% stock) - 400 μl</LI><LI>Tris HCl, 1M pH 6.8 - 375 μl</LI><LI>SDS 10% w/v - 30 μl</LI><LI>APS 10% w/v - 30 μl</LI><LI>TEMED 3 μl</LI></UL><H4>Running gel</H4><P>Run the gel in SDS running buffer for an hour at 200V.
Check regularly for bubbles in the wells that could affect the mobility of samples in these lanes.</P><H4>Western blot</H4><OL><LI>Take overnight cultures and dilute 1/50 with LB</LI><LI>Grow to an OD600 of 0.4</LI><LI>Induce the samples with different concentrations of rhamnose: 0.1, 0.2, 0.4 and 0.5% (v/v)</LI><LI>Incubate the samples and take out 1ml of the sample each hour: 0hrs, 1hr, 2hrs, 3hrs, 4hrs and 16hrs</LI><LI>Take each sample and spin down in centrifuge for 10 mins to separate cell and supernatant</LI><LI>Run samples on SDS gel</LI><LI>Place gel between 2x filter paper, 1 PVDF membrane, 2x filter paper (bottom to top) all soaked in transfer buffer (0.6g tris, 2.9 glycine, 40 ml methanol, make up to 200 ml with water)</LI><LI>Assemble this in semi-dry blot apparatus and transfer at 20V for 20 mins</LI><LI>Take membrane which should now have protein transferred to it and block in 5% skim milk (made up in PBS). Shake RT at least an hour.</LI><LI>Wash 3x in TBS tween (0.1% tween) 10 minutes each</LI><LI>Add antibody – 3μL in 30ml TBStween, incubate with shaking 1 hour RT</LI><LI>Wash 3x ten minutes in TBST</LI><LI>Take 1ml BioRad HRP detection solution and pipette on to membrane, incubate 2 min RT and develop with GeneGnome.</LI></OL><H4>Swimming motility assay</H4><OL><LI>Make 0.25% agar LB plates</LI><LI>Place a single colony from a streak plate of your desired strain on the surface of the plate</LI><LI>Incubate plate face up at 37<SUP>o</SUP>C for approximately 36 hours</LI><LI>Image plates</LI><LI>Using ImageJ, measure the colony area with the Wand tool set to Legacy mode.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>