<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_CFPS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/CFPS</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><P><NAV id="pagenav" class="navbar navbar-fixed-bottom"><UL class="nav nav-tabs" role="tablist" style="margin-left: 0px"><LI><A href="#Preliminary">Preliminary Experiments</A></LI><LI><A href="#RraA">Optimization: Positive effector RraA</A></LI><LI><A href="#paperCFPS">CFPS on Paper</A></LI><LI><A href="#robustness">Robustness</A></LI><LI><A href="#lowcost">Low Cost Energy Solution</A></LI><LI><A href="#Resultsummary">Result Summary</A></LI></UL></NAV></P><DIV class="container"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2015/2/2b/Bielefeld-CeBiTec_CFPS_image_paper_discs_general.png)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px">CFPS</H1><P>Molecular machinery at work</P></DIV></DIV><H2> Motivation </H2><P><I>Biosensor</I> and <I>cell-free</I> are <B>not</B> mutually exclusive. Cell-free biosensors have been described in the literature (<A href="#Pellinen2004">Pellinen et al. 2004</A>) and it was demonstrated that they have quite many advantages (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#advantages">see background page</A>) compared to cell-based approaches. The output of a biosensor is commonly realised via proteins. For easy evaluation, the output should be in a form that can be adressed directly, for example color or fluorescence. The challenge for us who were eager to stay cell-free therefore was to build functional proteins <I>without the help of whole cells</I>. Fortunately, the components needed to faciliate <I>in vitro</I> protein production from a DNA template are well defined. All components together can be summarized to the term <B>molecular machinery</B>. Parts of this machinery are for example RNA polymerase, ribosomes, amino acids... for a detailed summary please be referred to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">our background page</A>.</P><P> With the molecular machinery, the socalled <B>Cell-Free Protein Synthesis (CFPS)</B> is possible. It is astonishing that CFPS rarely appeared in iGEM in the recent years despite its advantages. We set out to establish <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#CFPSprotocols">CFPS protocols</A> and setups that can easily be performed by us and future iGEM teams. Still, our main goal was the <I>in vitro</I> production of a reporter protein as necessary condition to create a cell-free biosensor. To be up to date in a research area that evolves at a fast pace, we gathered literature and information. Furthermore, we asked two experts in the field of <I>in vitro</I> transcription and translation, who gave us valuable hints and comments for our preliminary experiments: Mr. Zachary Z. Sun, PhD candidate at California Institute of Technology, and Michael Jewett, PhD from Northwestern University.   
         </P><DIV id="Preliminary"><H2> Preliminary Experiments</H2><H3> What we aimed for </H3><P> With only theoretical knowledge from literature and hints from experts, our first ambitious goal was to be able to produce the reporter protein sfGFP completely <I>in vitro</I>. The crude extract from <I>E. coli</I> cells should provide us with the molecular machinery, and we should add other compounds like energy source, amino acids and DNA template. </P><H3> What we used to reach the aim </H3><P> To check if protein production occured we used a plate reader to measure fluorescence signals. To prevent protein degradation in our reaction, we tried extract from two <I>E. coli</I> strains designed to express proteins: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms#Ecoli">ER2566 and KRX</A>. Both have been genetically engineered for very low endogenous protease activity and possess genomic coded polymerase of bacteriophage T7. T7 polymerase is a monomer, very specific to its promoter and more stable than bacterial polymerases (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sousa2003">Sousa and Mukherjee, 2003</A>), which makes it perfectly suited for <I>in vitro</I> transcription. </P><H3> How we started</H3><P> To extract the molecular machinery needed for <I>in vitro</I> transcription and translation from <I>E. coli</I> cells, they have to be cultivated and harvested first. For optimal yield of ribosomes, cell harvest should in general be performed at mid- to late exponential growth phase (<A href="#Smith2014b">Smith et al. 2014</A>). At this stage of growth, the cells would have a highly active translation machinery. We therefore started to measure growth curves for the strains <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms#Ecoli">ER2566 and KRX</A> at 100 mL scale in shaking flasks.
    
</P><DIV><FIGURE style="margin:auto; width:500px"><FIGCAPTION> Growth curves of the two <I>E. coli</I> strains ER2566 and KRX. Induction of T7-polymerase was facilitated by adding IPTG to the ER2566 culture and rhamnose to the KRX culture, respectively. For details see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook/CFPS">notebook</A>.</FIGCAPTION></FIGURE></DIV><DIV><P> We observed that induction of T7-polymerase expression at OD<SUB>600</SUB> = 0.8-1.4 was not critical for growth behaviour. Our conclusion was that a cell harvest at an OD<SUB>600</SUB> = 3-4 would be optimal. At this time of cultivation there would be enough cell mass for the following purification steps but no risk of being in stationary growth phase. Cell harvest at this OD<SUB>600</SUB> would furthermore be consistent with published data (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>). </P></DIV><H3> How to create highly active cell extract </H3><P> To generate crude cell extract, the bacteria have to be disrupted. This can be achieved by different methods, for example bead beating (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al. 2013</A>) or homogenizing with high pressure (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Yang2012">Yang et al. 2012</A>). We decided for sonication as a recent publication from Kwon and Jewett showed that it is fast, cheap, reliable, easy to perform and that it works for small volumes (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>). We share the opinion that sonication can be recommended to iGEM teams in particular because equipment is relatively cheap compared to other disruption devices (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Shrestha2012">Shrestha et al. 2012</A>).
    
    
    </P><P> The parameter with the greatest impact on crude cell extract quality is the sonifiers energy output (expressed in Joule per second) (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>). Insufficient energy does not lyse cells efficiently, whereas too much energy inactivates proteins. Although we had access to a Branson sonifier, we faced a problem: The device did not display its energy output and its technical specifications also shed no light on the issue. After consulting the technical support, we were able to tackle this problem: We measured how the temperature of a water sample changed as a function of sonication time. With an equation that takes all important parameters into account we were able to correlate the sonication time needed for a desired energy output (for details please be referred to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#Sonication">protocol section</A>). </P><H3> Cell disruption - outcome </H3><P> We wondered how effective our sonifier was when it comes to the disruption of <I>E. coli</I>. For this we cultivated, harvested and washed cells of our working strain ER2566. Then we sonicated them in 24 cycles of 10 seconds because this was the optimal sonication time we determined for our specific sonifier setup. We took aliquots and plated them onto LB plates to get a first idea about the efficiency. As expected, we observed a decrease in cell viability with increasing sonication time.  </P><FIGURE style="margin:auto; width:600px"><FIGCAPTION> Sonication efficiency test. 10 µL aliquots were taken out of the 1.5 mL vessel containing the resuspended cell pellet. Aliquots were diluted in 990 µL of water, then 20 µL of these samples were plated on LB. For details see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook/CFPS">notebook</A>.  </FIGCAPTION></FIGURE><DIV><P> After the subsequent centrifugation steps according to the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#sonication">protocol</A>, we took aliquots in the same manner. Viable cell numbers on the plates were even lower, touching one-digit levels. </P></DIV><H3> Cell disruption efficiency summarized </H3><P> During our project, we continued to investigate how many cells survived the process of sonication. Our goal was to create highly active but completely cell-free extract. In our fully optimized extract, a 100 µL aliquot of flash-freezed crude cell extract contained only 8 colony forming units (CFU). Although 8 colony forming units in crude cell extract are still too many in purposes of biosafety issues, we conclude that our sonication is extremely efficient in decreasing viable cell numbers. Strikingly, our final biosensor device based on freeze-dried extract did not contain any living <I>E. coli</I> at all, like you can see  here. We can therefore proudly say that our methods are excellently suited to construct a complete cell-free environment coming from liquid cultures. Keep on reading to learn about how our extract performed in producing reporter proteins!</P><DIV class="row"><DIV class="col-md-12"><H3> Our reporter construct</H3><P> Most CFPS approaches are based on T7 polymerase and its corresponding promoter (see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#transcription">background page</A>). We therefore used T7 transcription machinery in our cell-free expression experiments, which turned out to be a good idea: In tests with endogenous σ<SUB>70</SUB>-polymerase and non-T7 promoters we observed very low protein production (for details, see heavy metal result page). </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P> An ideal reporter protein for <I>in vitro</I> protein synthesis is superfolder GFP, abbreviated sfGFP (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al. 2013</A>). In our first experiments with self-made <I>E. coli</I>cell extract, we used sfGFP under control of the T7 promoter (<A href="http://parts.igem.org/Part:BBa_I746909" target="_blank">P<SUB>T7</SUB>-sfGFP</A>) from the parts registry. </P><H3> Experimental setup </H3><P> To perform <I>in vitro</I> expression of sfGFP from a DNA template, first we had to set up the reaction. To do so, we combined the three necessary components which are: Our previously described cell extract, the DNA template and eventually the reaction buffer with energy source, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#aminoacids#AApreparation" target="_blank">amino acids</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#aminoacids#cofactorpremix" target="_blank">cofactor mixture</A>. Final reaction volume was 20 µL, and reactions were performed in a 364 black microwell plate at 37 °C. In these inital experiments, we used a FLUOstar microplate reader to measure fluorescence at 515 nm, exciting at 480 nm. Unfortunately the device had seen better days and could not heat nor shake, therefore reactions had to be interrupted each time fluorescence was measured. </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><H3> Expressing sfGFP <I>in vitro</I></H3><P> We had to test various reaction compositions to find a functional formula that enabled production of sfGFP. The extract purification and glutamate salt concentrations were particulary important. Finally, a 10 fold increased fluorescence signal was observed when compared to the negative control. This was the first time we realized that we made it; <I>in vitro</I> sfGFP transcription and translation with our extract was possible! </P></DIV><DIV class="col-md-6"><FIGURE style="width: 600px; margin-left: 20px; float: right"><FIGCAPTION> Proving that sfGFP synthesis is possible with our self-made extract. The mean value of two technical replicates was used for evaluation. Fluorescence signal was in this case normalized on fluorescence signal noise of negative control. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3> Template optimization</H3><H4> Experimental setup </H4><P> We were sure that a further optimization of sfGFP production was possible. Based on literature screening (for details see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">background page</A>, we designed a <B>translation enhancing sequence (5'-untranslated region, 5'-UTR)</B> and inserted it into P<SUB>T7</SUB>-sfGFP, thereby creating <A href="http://parts.igem.org/Part:Part:BBa_K1758102" target="_blank"> P<SUB>T7</SUB>-UTR-sfGFP, BBa_K1758102</A>. Our assumption was that if translation was a bottleneck in our extract, this sequence would improve sfGFP production. As verification for the beneficial effect of the 5'-UTR, we performed <I>in vivo</I> experiments first, normalizing the fluorescence signal to culture OD<SUB>600</SUB>. We employed another reporter protein, mutated red fluorescent protein (mRFP), as well. This was done to check if the 5'-UTR was useful for translation in general. Similar to above mentioned cloning strategy, we constructed P<SUB>T7</SUB>-UTR-mRFP (<A href="http://parts.igem.org/Part:Part:BBa_K1758106" target="_blank">BBa_K1758106</A>) after assembling P<SUB>T7</SUB>-mRFP (<A href="http://parts.igem.org/Part:Part:BBa_K1758105" target="_blank">BBa_K1758105</A>). For mRFP fluorescence signal normalization on OD<SUB>600</SUB> we faced the problem that mRFP emits fluorescence at 607 nm (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al. 2013</A>). We also tried to express mRFP <I>in vitro</I> to check if it might be a reporter protein equally or better suited for our purposes. mRFP was excited at 580 nm, its emission was measured at 610 nm. </P><H4> UTR performance </H4><P> This was the case <I>in vivo</I>: We observed a faster production of sfGFP and mRFP when the plasmid DNA contained 5'-UTR in front of sfGFP coding sequence (see nearby photo of shaking flasks). When fluorescence was quantified in the microplate reader and signals were normalized on OD<SUB>600</SUB>, the results for mRFP seemed less convincing. This might be due to normalization strategy as discussed previously. Both experiments however equally demonstrate the usefulness of 5'-UTR for protein production in general. </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><FIGURE style="width: 550px; margin-right: 20px; float: left"><FIGCAPTION><I>In vivo</I> characterization of 5'-UTR with sfGFP. Relative fluorescence units were normalized on OD<SUB>600</SUB>. Error bars represent standard deviation of triplicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><FIGURE style="width: 550px; margin-left: 20px; float: right"><FIGCAPTION><I>In vivo</I> characterization of 5'-UTR with mRFP. Relative fluorescence units were normalized on OD<SUB>600</SUB>. One might consider biases due to mRFP fluorescence emission at 607 nm like discussed previously. Error bars represent standard deviation of triplicates. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><FIGURE style="margin:auto; width:600px"><FIGCAPTION> Cultures expressing sfGFP and mRFP, resprectively. From left to right: <A href="http://parts.igem.org/Part:Part:BBa_K1758102" target="_blank"> P<SUB>T7</SUB>-UTR-sfGFP (BBa_K1758102)</A>, <A href="http://parts.igem.org/Part:BBa_I746909" target="_blank">P<SUB>T7</SUB>-sfGFP (BBa_I746909)</A>, <A href="http://parts.igem.org/Part:Part:BBa_K1758106" target="_blank">P<SUB>T7</SUB>-UTR-mRFP (BBa_K1758106)</A> and <A href="http://parts.igem.org/Part:Part:BBa_K1758105" target="_blank">P<SUB>T7</SUB>-mRFP (BBa_K1758105)</A>.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-5"><P> This was the case when 5'-UTR was employed in <I>in vitro</I> experiments: We observed a more than 3 fold increase in fluorescence. This clearly showed the importance of this enhancing element in CFPS, and further demonstrated that translation efficiency <I>in vitro</I> is a major issue for protein synthesis.</P></DIV><DIV class="col-md-7"><FIGURE style="float:right; width:580px"><FIGCAPTION> Importance of 5'-untranslated region (UTR) for <I>in vitro</I> protein synthesis. T7 refers to T7 promoter. sfGFP lysate refers to cell lysate won by sonication; cells were induced to produce sfGFP <I>in vivo</I>. Decreasing sfGFP lysate signal probably relied on evaporation. Mean values from two technical replicates are shown. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P><I>In vitro</I> mRFP expression with construct P<SUB>T7</SUB>-UTR-mRFP (<A href="http://parts.igem.org/Part:Part:BBa_K1758106" target="_blank">BBa_K1758106</A>) showed us that mRFP was not nearly as well performing as sfGFP. Although we observed a signal differing from the negative control, this signal was very low and raising very slowly, as depicted in the nearby figure. Thus sfGFP remained our reporter protein and we used it in all following CFPS reactions. </P></DIV><DIV class="col-md-6"><FIGURE style="width: 500px; margin-left: 20px; float: right"><FIGCAPTION> Performance of mRFP in our CFPS reaction. mRFP was produced but the fluorescence signal was very low, even after several hours. Error bars represent standard deviation of triplicates. </FIGCAPTION></FIGURE></DIV></DIV><H3> Standardisation of experimental setup </H3><DIV class="row"><DIV class="col-md-6"><P> With these results, we had a good positive control plasmid for our following reactions. The reaction buffer for our CFPS reaction was now defined. The composition of the reaction buffer is depicted in the nearby table. </P><P> Nevertheless, we observed batch-to-batch variation in activity of our extracts, a phenomenon also described in the literature (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2015">Takahashi et al. 2015</A>). To exclude any batch-to-batch variation in following reactions, we set up a 5 L fermentation and harvested cell pellets to produce by far enough cell extract for the summer (details in the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook/CFPS">notebook</A>). </P></DIV><DIV class="col-md-6"><DIV style="float:right; margin-left:20px; margin-right: 20px"><TABLE class="scientific"><TBODY><TR><TH>component</TH><TH>final concentration in mM</TH></TR><TR><TD>ATP</TD><TD>1.5</TD></TR><TR><TD>GTP</TD><TD>1.5</TD></TR><TR><TD>UTP</TD><TD>0.9</TD></TR><TR><TD>CTP</TD><TD>0.9</TD></TR><TR><TD>Amino acids</TD><TD>2</TD></TR><TR><TD>Coenzyme A</TD><TD>0.27</TD></TR><TR><TD><I>E. coli</I> tRNA </TD><TD>0.2 *</TD></TR><TR><TD>Folinic acid</TD><TD>0.07</TD></TR><TR><TD>HEPES</TD><TD>50</TD></TR><TR><TD>K-Glutamate</TD><TD>4 *</TD></TR><TR><TD>Mg-Glutamate</TD><TD>80 *</TD></TR><TR><TD>NAD</TD><TD>0.33</TD></TR><TR><TD>Na-Oxalate</TD><TD>4</TD></TR><TR><TD>PEP</TD><TD>33</TD></TR><TR><TD>Putrescine</TD><TD>1</TD></TR><TR><TD>Spermidine</TD><TD>1.5</TD></TR></TBODY></TABLE><P style="font-size:70%"> *: tRNA value is given in mg/mL. For optimal performance, K-glutamate and Mg-glutamate salt concentrations may have to be adjusted depending on extract and batch.</P></DIV></DIV></DIV><P> Several other issues were tackled: </P><UL><LI> To improve measurement of fluorescence, we started using a Tecan plate reader. The device was able to shake microplates and constantly heat to 37 °C, therefore we were able to measure kinetics of sfGFP production without interrupting the reaction. </LI><LI> To get reliable data, we used at least triplicates for every following CFPS reaction. We normalized on accompanied cell lysate from cells that had produced sfGFP. </LI><LI> Depending on <I>E. coli</I> strain and cell harvest, it can be necessary to perform a so called <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#costs" title="details about run-off reaction">run-off reaction</A>. This means heating purified extract to 37 °C before flash-freezing it to enable degradation of endogenous DNA. We determined that 30 min of run-off reaction resulted in best performance for our extract. </LI></UL><H3> Exemplifying standardized CFPS setup </H3><P> Measuring nearly every minute, we traced production of sfGFP in real time (see nearby figure). The experiment once again showed the beneficial effect of 5'-UTR previously discussed. With our construct, we were able to produce nearly as much sfGFP as was present in disrupted cells that had produced sfGFP <I>in vivo</I>. </P><FIGURE style="margin:auto; width:600px"><FIGCAPTION> Tracing CFPS kinetics in Tecan platereader. RFU signals were normalized to first signal of sfGFP lysate. Purified GFP refers to the signal of a His-tagged GFP solution at a concentration of about 320 µg/mL we kindly received from our advisor Lukas. Error bars represent standard deviation of triplicates. </FIGCAPTION></FIGURE></DIV><DIV id="RraA"><H2> Optimization: Positive effector RraA </H2><H3> General optimization </H3><DIV class="row"><DIV class="col-md-6"><P> During our project, we constantly optimized our extract. Our success is demonstrated by the following bar graph which shows how the fluorescence signal of our positive control in CFPS reactions increased during summer. In the end we were able to see the greenish color of a reaction at 30 µL scale <B>with the naked eye</B>, and when the tube was pictured with our measurement device, sfGFP fluorescence became even more observable like you can see in the nearby pictures. Time issues held us back from further optimization, although we are confident that further optimization is still possible.  </P><DIV><FIGURE style="width: 220px; margin-left: 20px; float: left"><FIGCAPTION><I>In vitro</I> produced sfGFP pictured without filters. Also it might not be obvious, the greenish color can be seen with the naked eye </FIGCAPTION></FIGURE><FIGURE style="width: 220px; margin-right: 20px; float: right"><FIGCAPTION><I>In vitro</I> produced sfGFP pictured with our simple fluorescence measurement device </FIGCAPTION></FIGURE></DIV></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION> Evolution of reaction performance. Fluorescence signal of positive controls accompanied in the experiments raised constantly during our project. The first two bars shown were single performances whereas the others represent the mean of three replicates with standard deviation as error bars.</FIGCAPTION></FIGURE></DIV></DIV><P> One optimization approach based on the addition of a protein to the cell-free reaction is depicted in the following section.</P><H3> Positive Effector RraA – Background</H3><P> The product of the <I>rraA</I> gene (Regulator of ribonuclease activity A, former menG) has been reported to interact with RNase E from <I>E. coli</I> and to alternate its activity (<A href="#Lee2003">Lee et al. 2003</A>,<A href="#Yeom2008">Yeom et al. 2008</A>, <A href="#Gorna2010">Gorna et al. 2010</A>). RNase E is coded by the <I>rne</I> gene and is essential for <I>E. coli</I>. The protein takes a dual role in the bacterium as it enables processing of important RNAs but also participates in nonspecific degradation of RNA (<A href="#Mackie2013">Mackie 2013</A>). </P><P> Airen showed that if RraA-protein is added to a cell-free protein synthesis reaction, the productivity raises about 30% in his cell-free system (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>). He postulated that activity of RNase E in the reaction is lowered due to the interaction with RraA-protein, therefore mRNA-levels are stabilized.</P><H3> RraA – Experimental Setup </H3><P> To characterize RraA <I>in vivo</I>, we created two <I>E. coli</I> strains. One carried the <I>rne</I> gene that codes for RNase E under control of the inducible T7-promoter (P<SUB>T7</SUB>-<I>rne</I>-plasmid), whereas a second strain carried an additional second plasmid with the RraA coding sequence (P<SUB>T7</SUB>-<I>rraA</I>-plasmid; <A href="http://parts.igem.org/Part:BBa_K1758122" target="_blank">BBa_K1758122</A>). For <I>in vitro</I> characterization, purified RraA that contained an N-terminal 6xHis-Tag was added to our CFPS reaction to see if RraA had the same positive effect in our cell-free setup. </P><H3> RraA – Results </H3><DIV class="row"><DIV class="col-md-6"><P> In the <I>in vivo</I> experiment there was no observable growth drop when T7 polymerase was induced in the strain carrying both plasmids, P<SUB>T7</SUB>-<I>rne</I> and P<SUB>T7</SUB>-<I>rraA</I>. However, induction of T7 polymerase in the strain carrying P<SUB>T7</SUB>-<I>rne</I>-plasmid only lead to a clear growth inhibition. This difference was apparent although the strain carrying <I>rne</I>-plasmid altogether grew slower than the double transformed strain. Therefore we conclude RraA overexpression rescues <I>E. coli</I> by decreasing activity of RNase E.</P></DIV><DIV class="col-md-6"><FIGURE style="margin:auto; width:600px"><FIGCAPTION> Growth curves of <I>E. coli</I> expressing RNase E (<I>rne</I>) and RraA. Two cultures were induced to express T7 polymerase at 1.25 h. </FIGCAPTION></FIGURE></DIV></DIV><P> To verify the effect <I>in vitro</I>, RraA in 50 mM HEPES buffer at pH = 7.2 was added to CFPS reactions to a final concentration of 0.3 mg/mL. To exclude that an observed effect resulted from the buffer alone, two control reactions were additionally performed. In these reactions RraA was omitted and the missing volume was filled up with water and buffer only, respectively. </P><DIV class="row"><DIV class="col-md-8"><FIGURE style="width: 700px; margin-right: 20px; float: left"><FIGCAPTION> RraA improves final fluorescence signal when added to the reaction. Final concentration of RraA in the reaction was 0.3 mg/mL.</FIGCAPTION></FIGURE></DIV><DIV class="col-md-4"><FIGURE style="width: 200px; margin-left: 20px; margin-right: 20px; float: right"><FIGCAPTION> Purified RraA after SDS-PAGE. Protein identity was proven via mass spectrometry.</FIGCAPTION></FIGURE></DIV></DIV><P> We could verify Airens observation that RraA is a positive effector in cell-free protein synthesis. More precisely, the results of Airens and our experiment are similar: In our reaction the final signal raised about 33.8 ± 4.5 %, and in Airens experiment the signal raised 28.6 ± 3.1 % respectively when compared to a reaction were RraA is not present (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>). </P><P> The reason for this effect is investigated in detail in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>. The shape of the fluorescence signal curve when RraA is present (green dots) indicates that the protein acts as stabilisator in our reaction. It is likely that by reducing RNase E activity, the rate of mRNA and rRNA degradation is slower. </P><P> If one is interested in obtaining high amounts of protein via CFPS, the addition of RraA – or similar positive effectors – is definitely recommended. </P><P> This experiment demonstrates how versatile CFPS can be. You can simply add an external molecule to the reaction and analyze its impact on protein yield. Although the reason why a substance is beneficial or disadvantageous may remain unclear, the experimenter has free access to the reaction. Less time is needed in this case when compared to <I>in vivo</I> optimization (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al. 2013</A>, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2015">Takahashi et al. 2015</A>).  </P><H3> Positive Effector RraA – at a glance </H3><P> We optimized our extract by adding RraA to the cell-free reaction. The protein is useful in CFPS because it can inhibit the RNase E activity in the extract, which thus leads to a final output signal that is about 33% higher. The coding sequence of RraA and a protein generator to produce His-tagged RraA has been send to the parts registry (Biobricks <A href="http://parts.igem.org/Part:BBa_K1758120" target="_blank">BBa_K1758120</A> and <A href="http://parts.igem.org/Part:BBa_K1758122" target="_blank">BBa_K1758122</A>, respectively). iGEM teams as well as labs that deal with <I>E. coli</I> cell-free protein synthesis have now the chance to use these biobricks for protein production optimization. </P></DIV><DIV id="paperCFPS"><DIV class="row"><DIV class="col-md-12"><H2> CFPS on Paper</H2><P> As depicted above, we established a robust cell-free protein synthesis system for the production of sfGFP in our lab. The previous reactions were mostly performed in 364 well plates, so to test if our cell extract worked on <B>paper</B> we proceeded with using different papers as base for our reactions.</P><H3> CFPS on Paper – Experimental Setup</H3></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P> We prepared various paper discs with a diameter of about 6 mm using a punch. The CFPS reactions were prepared according to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#CFPSprotocols">protocols</A> at 15 µL scale, and were then applied onto the paper. To keep track of the reaction, fluorescence signals were measured with Tecan plate reader as previously described. </P><P> We compared our CFPS setup with commercially available S30 T7 cell-free expression system. To do this, the manufactors protocol was scaled down from a 50 µL to a 15 µL scale. </P></DIV><DIV class="col-md-3"><FIGURE style="width: 300px;margin:auto; float: right"><FIGCAPTION> Paper discs </FIGCAPTION></FIGURE></DIV><DIV class="col-md-3"><FIGURE style="width: 250px; margin:auto; float: right"><FIGCAPTION> Paper discs in a 96-well microplate used for measurements in plate reader </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H3> CFPS on Paper – Results </H3><P> At first, we tried to conduct the reactions without any further processing of the paper discs. No signal was seen. We supposed papers were full of RNases which would have adverse effects on the RNA in the reaction. </P><P> By autoclaving the paper discs three times, we hoped to reduce RNase contaminations. When we repeated the experiment with autoclaved paper, results were far better. Although the standard deviation was quite high, probably because of little differences in the shape of the paper discs, we demonstrated that it is possible to conduct a CFPS reaction on simple autoclaved paper. </P><P> The fluorescence signal depended on paper type. In total, we tested five different paper types on suitability. CFPS worked on every paper tested, however, for subsequent experiments we decided to use the two paper that worked best: 827B from Macherey &amp; Nagel (M&amp;N) and C350L from Munktell. Interestingly, when we used the commercially available S30 T7 cell-free expression system at the same scale, the resulting fluorescence signal was far lower. This may be attributed to the fact that the commercial system is not designed to work under these conditions. </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><FIGURE style="width: 550px; margin-right: 20px; float: left"><FIGCAPTION> CFPS reaction with self-made <I>E. coli</I> extract on different types of autoclaved paper. Basal fluorescence of different papers were subtracted. Fluorescence signals of CFPS reactions were then normalized on sfGFP containing lysate that was applied on the respective paper. M&amp;N: Macherey &amp; Nagel</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><FIGURE style="width: 550px; margin-left: 20px; float: right"><FIGCAPTION> CFPS reaction with commercial <I>E. coli</I> extract on different types of autoclaved paper. Basal fluorescence of different papers were subtracted. Fluorescence signals of CFPS reactions were then normalized on sfGFP containing lysate that was applied on the respective paper. M&amp;N: Macherey &amp; Nagel </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P> We wondered if it was possible to lyophilize the paper discs <B>together with the CFPS reaction</B>, like <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pardee2014">Pardee et al.</A> did. The 15 µL reactions were applied on paper discs, put into microcentrifuge tubes  and flash-freezed in liquid nitrogen. Lyophilization was carried out for 24 h. Afterwards, the paper discs were rehydrated with <B>tap water</B> instead of RNase free water. </P><P> The results were once again beyond our best expectations. <B>CFPS worked well on paper after lyophilization even though tap water was used for rehydration</B>. Commercial cell extract showed a signal, too, but our crude cell extract performed better on both paper types. </P></DIV><DIV class="col-md-6"><FIGURE style="width: 600px; margin-left: 20px; float: right"><FIGCAPTION> CFPS on paper after 24 h of lyophilization. Protein synthesis was initiated by rehydration with 15 µL of <B>tap water</B>. Fluorescence signals were normalized to values of freshly added sfGFP lysate applied on the respective paper. M&amp;N: Macherey &amp; Nagel </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-6"><FIGURE style="width: 600px; margin-right: 20px; float: left"><FIGCAPTION> CFPS on paper after lyophilization and 6 days storage at 8 °C. M&amp;N: Macherey &amp; Nagel</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P> In the preceding experiment, rehydration was performed right after lyophilization. As our aim was the creation of a cell-free biosensor for the detection of variuos substances outside of the lab, the shelf life of the lyophilized paper discs was of outstanding interest. We repeated the reaction und the same conditions, except that after lyophiliztation, the microcentrifuge tubes were stored at 8°C and room temperature. 6 days later, reactions were rehydrated with 15 µL of tap water and fluorescence signals were measured in Tecan plate reader. The reaction worked and fluorescence signals were still reasonable high, although not as high the signals from non-stored paper-based reactions. </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P> Additionally, 6 day storage was performed at room temperature and 60 °C. Reaction started after rehydration when stored at room temperature, which was remarkably. When the tubes containing the paper discs with the reaction components were provisional sealed with adhesive film, a higher fluorescence signal was observed (see nearby graph). Therefore we propose that a professional sealing would be better for storage and prevent the paper from moistening. </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><FIGURE style="width: 600px; margin-right: 20px; float: left"><FIGCAPTION> CFPS after storage at room temperature for 6 days. Tubes were stored with adhesive film (parafilm) wrapped round. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><FIGURE style="width: 600px; margin-left: 20px; float: right"><FIGCAPTION> CFPS on paper after lyophilization and 6 days storage at room temperature. Tubes were not sealed with adhesive film. M&amp;N: Macherey &amp; Nagel </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P> Storage at 60 °C was not beneficial or more precisely fatal for extract activity. No fluorescence signal raised after rehydration. This was not unexpected, and would have been rather incredible though. </P><H3> CFPS on Paper – Conclusions and Outlook </H3><P> We showed that our cell-free expression system based on crude cell extract works very well on paper. Furthermore, it is very amenable to these conditions as well as to other environments (see <A href="#robustness">robustness section</A>). </P><P> We demonstrated that <B>conduction of a CFPS on paper is still possible after 6 days of storage at 8 °C</B>, a fact that is of outstanding interst for our final application. The fluorescence signal was lowered by about 50% compared to the directly started reaction, however, storage conditions can easily be adapted to maintain activity, as has been proposed for example by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pardee2014">Pardee et al. 2014</A>. </P></DIV></DIV><DIV class="row"><DIV class="col-md-10" style="float:left"><P> With a smartphone and our <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">measurement device</A>, we took pictures of the fluorescent paper discs and evaluated the results with our <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Software">app</A>. Click on the nearby picture to take a look on these pages for more images, details and lots of interesting stuff! </P></DIV></DIV><DIV id="biosafetyCFPS"><DIV class="row"><DIV class="col-md-12"><FIGURE style="width: 260px; margin-left: 20px; float: right"><FIGCAPTION>No colonies grew after streaking our <B>lyophilised paper discs (round dots)</B> onto LB plates, even after two days at 37 °C. Little stains resulted from paper disc streaking. </FIGCAPTION></FIGURE><H4>Biosafety issues</H4><P> For our final application, we applied cell extract and all compounds needed for protein synthesis on paper and lyophilized it. We were able to prove that absolutely no <I>E. coli</I> culture survives this process by placing the paper onto an LB plate and putting it into a 37 °C incubator for 2 days. We could not observe any colony forming units, not after any approach of extract lyophilization. This is consistent with recent findings from <A href="#Smith2015">Smith et al. 2015</A> who showed that sterile filtration and lyophilization are methods to free cell extracts from any cells. Lyophilisation was preferred due to stable expression thereafter, which fit to our results. 
      </P></DIV></DIV></DIV></DIV><DIV id="robustness"><H2> Robustness </H2><P> Our CFPS system occured to be very robust towards different substances. In particular, 5% of EtOH and 9% lake water instead of RNase free water had little and no effect, respectively, on final fluorescence signal, as was the case for tap water when rehydrating lyophilized extracts (see section <A href="#paperCFPS">CFPS on Paper</A>). </P><DIV class="row"><DIV class="col-md-6"><FIGURE style="width: 600px; margin-right: 20px; float: left"><FIGCAPTION> CFPS sufficiently resists to 5% (v/v) Ethanol. At 15% (v/v) Ethanol nearly no fluorescence signal arises. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><FIGURE style="width: 600px; margin-left: 20px; float: right"><FIGCAPTION> CFPS sufficiently resists to 9% (v/v) lake water. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P> As our cell-free system is designed to work outside the lab, we dealt with the question if it is possible to conduct sfGFP synthesis at temperatures other than 37°C. The optimal temperature for cell-free protein synthesis depends on the protein and the system itself (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>). </P><P> Our CFPS still worked well at 29 °C and 25 °C, although still 37 °C yielded the highest fluorescence. As we can synthesize sfGFP at temperatures around room temperature, a performance in the field is quite conceivable. </P><FIGURE style="margin:auto; width:500px"><FIGCAPTION> Relative fluorescence units in positive control setups at different temperatures. Tecan device measurement specifications were identical in every run. Maximum fluorescence signal during measurement is shown. Experiments were performed in triplicates at least. </FIGCAPTION></FIGURE><P> We further investigated the effects of different heavy metals on our CFPS system, results hereof can be found at the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metal result page</A>. </P></DIV></DIV></DIV><DIV id="lowcost"><H2> Low Cost Energy Solution</H2><FIGURE style="width: 150px; margin-left: 20px; float: right"><FIGCAPTION> Click on the image to open a pdf of our cost estimation for performance of CFPS following our protocols. </FIGCAPTION></FIGURE><H3> Cost Reduction – Considerations </H3><P> Laboratory reagents are never for free. But you will always try to get the most out of your money. With this in mind, you may be surprised that the reagents for a small scale CFPS reactions cost less than the reagents needed for a PCR (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pardee2014">Pardee et al. 2014</A>). We calculated the cost to perform a 15 µL CFPS reaction one has to spend when following our protocols. To keep it simple, we only took into account all the chemicals needed to perform CFPS. Click on the dollar image to open a .pdf of our cost estimation. Based on our calculations, one CFPS reaction is as cheap as 16 ¢!</P><P> We analyzed how it would be possible to even further reduce the costs of CFPS reactions. One major cost aspect relies on the energy source which enables ATP regeneration. Phosphoenolpyruvate (PEP) is already cheaper than the often used 3-Phosphoglyerate (3-PGA). However, we tested two easily affordable substances to fuel our reactions: The common polysaccharide maltodextrin and the polyphosphate molecule Hexametaphosphate, which have been shown to work in cell-free protein synthesis (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015a">Caschera and Noireaux, 2015a</A>). </P><DIV class="row"><DIV class="col-md-12"><H3> Testing Low Cost Energy Solution </H3><FIGURE style="width: 600px; margin-left: 20px; float: right"><FIGCAPTION> CFPS with low cost energy solution compared to our fully optimized, PEP-based system. </FIGCAPTION></FIGURE><P> We showed that the previously mentioned energy sources can substitute PEP when Na-oxalate is removed from the reaction. The observed fluorescence signal was similar in multiple performances. The low cost energy solution is a good alternative for CFPS. A cost estimation revealed: When Hexametaphosphate and maltodextrin substitute for PEP and Na-oxalate, the costs for a 15 µL reaction (reagents only) are reduced by about 20%.  </P><P> However, the output signal obtained with the low cost energy solution can not compete with the signal from our fully optimized PEP-based system, as you can see in the nearby graph. The advantage is that one surely can optimize the low cost energy solution system as well (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015a">Caschera and Noireaux, 2015a</A>), albeit this was not the aim of our project. </P></DIV></DIV></DIV><DIV id="Resultsummary" class="conclusion" style="padding: 0px 10px 5px 10px; margin-top: 60px"><H2> Result Summary</H2><H3> Our <A data-toggle="tooltip" title="Cell-free protein synthesis">CFPS</A> Achievements – At a Glance </H3><TABLE style="background-color:transparent; cellspacing=3;"><TBODY><TR><TD> We established CFPS methods and protocols for the iGEM community. </TD></TR><TR><TD> We managed to produce high amounts of superfolder GFP in self-made <I>E. coli</I> extract.  </TD></TR><TR><TD> We optimized our extract in terms of performance, that is: It is robust, fast and even works on paper. </TD></TR><TR><TD> We successfully <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">modelled</A> the cell extract in simbiology. </TD></TR><TR><TD> Our extract contains vanishingly low amounts of living cells. <I>However</I>, after a final treatment of the extract, we did not observe even one living cell.  </TD></TR><TR><TD> We tested the effect of various <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">heavy metals</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">date rape drugs</A> on cell-free protein synthesis. </TD></TR><TR><TD> CFPS builds the basis for our functional and extensible cell-free biosensor. </TD></TR></TBODY></TABLE><H3> Our CFPS Achievements - More Detailed</H3><DIV class="row"><DIV class="col-md-9"><P> Based on literature screening and hints from experts, we combined cell-free protein synthesis methods and protocols to a functional setup. Cell extract of different <I>E. coli</I> strains can hereby be easily tested for CFPS suitability in high-throughput manner, like pioneered by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett</A>. The methods and protocols are available for everyone in our <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#CFPSprotocols">protocols section</A>.</P><P> A translation enhancing 5'-untranslated region (5'-UTR) was designed based on literature. With 5'-UTR employed, reporter expression increased drastically, as well <I>in vivo</I> as <I>in vitro</I>.  </P><P> At a scale as little as 15 µL, we managed to express sfGFP <I>in vitro</I> in no time at high efficiency. The accessibility of this cell-free system towards optimization was exemplified by addition of the protein RraA to the reaction. RraA addition lead to an increase in final fluorescence signal of 33.8 ± 4.5 %, what was consistent with the literature (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>). Our CFPS setup builds an open environment that easily can further be optimized depending on purpose. </P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P> When we used our optimized self-made cell extract, <I>in vitro</I> sfGFP production was superior to commercially available cell-free expression kits. </P><P> We demonstrated that our CFPS reaction has the potential to work under field circumstances. It is robust against various substances and works near room temperature. After lyophilization, we could not observe any living cell in several tests, so we can say that we created a cell-free environment. </P><P> We verified the observations from <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pardee2014">Pardee et al.</A> that CFPS works on different types of paper. With freeze-drying or lyophilization, storage was possible, and reaction start could be triggered by addition of water. </P><P> Paper-based CFPS was a key feature in our biosensor project and thus CFPS was integrated into an application-oriented context. Paper-based CFPS paves the way to a new age of biosensors with great potential for field usage. This new generation biosensors are storable, extensible, safe, reliable and can be used by everybody. Of course professional biosafety tests have to be performed prior to a field usage of such a biosensor.  </P></DIV></DIV></DIV><H2> References </H2><DIV class="references"><P id="Gorna2010"> 
Górna, Maria W.; Pietras, Zbigniew; Tsai, Yi-Chun; Callaghan, Anastasia J.; Hernández, Helena; Robinson, Carol V.; Luisi, Ben F. (2010): The regulatory protein RraA modulates RNA-binding and helicase activities of the E. coli RNA degradosome. In RNA (New York, N.Y.) 16 (3), pp. 553–562. DOI: 10.1261/rna.1858010.
</P><P id="Lee2003">
Lee, Kangseok; Zhan, Xiaoming; Gao, Junjun; Qiu, Ji; Feng, Yanan; Meganathan, R. et al. (2003): RraA: a Protein Inhibitor of RNase E Activity that Globally Modulates RNA Abundance in E. coli. In Cell 114 (5), pp. 623–634. DOI: 10.1016/j.cell.2003.08.003.
</P><P id="Mackie2013">
Mackie, George A. (2013): RNase E: at the interface of bacterial RNA processing and decay. In: Nature reviews. Microbiology 11 (1), S. 45–57. DOI: 10.1038/nrmicro2930.


</P><P id="Pellinen2004">

Pellinen, Teijo; Huovinen, Tuomas; Karp, Matti (2004): A cell-free biosensor for the detection of transcriptional inducers using firefly luciferase as a reporter. In: Analytical biochemistry 330 (1), S. 52–57. DOI: 10.1016/j.ab.2004.03.064.

</P><P id="Smith2014b">

Smith, Mark Thomas; Hawes, Anna Katz; Shrestha, Prashanta; Rainsdon, Jay Marvin; Wu, Jeffrey C.; Bundy, Bradley Charles (2014): Alternative fermentation conditions for improved Escherichia coli-based cell-free protein synthesis for proteins requiring supplemental components for proper synthesis. In: Process Biochemistry 49 (2), S. 217–222. DOI: 10.1016/j.procbio.2013.10.012.
</P><P id="Smith2015">
Smith, Mark Thomas; Bennett, Anthony M.; Hunt, Jeremy M.; Bundy, Bradley C. (2015): Creating a completely &quot;cell-free&quot; system for protein synthesis. In Biotechnology progress. DOI: 10.1002/btpr.2157.

</P><P id="Yeom2008">
Yeom, Ji-Hyun; Go, Hayoung; Shin, Eunkyoung; Kim, Hyun-Lee; Han, Seung Hyun; Moore, Christopher J. et al. (2008): Inhibitory effects of RraA and RraB on RNAse E-related enzymes imply conserved functions in the regulated enzymatic cleavage of RNA. In FEMS microbiology letters 285 (1), pp. 10–15. DOI: 10.1111/j.1574-6968.2008.01205.x.
</P><P> Find further references <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CFPSreferences">here</A> at the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">CFPS background page</A>! 
</P></DIV><DIV class="row footer"><DIV class="col-md-2 col-md-offset-10 text-center"><A type="button" class="btn btn-default btn-next" href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA"><P>So what about this PRIA?</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>