<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Project_CFPS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Project/CFPS</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2015/0/0d/Bielefeld-CeBiTec_2015_UTR_characterization_6%2C25h_filter_sfGFP.jpg)"><H1>CFPS</H1><P>Cell-Free Protein Synthesis with the molecular machinery</P></DIV><NAV id="pagenav" class="navbar navbar-fixed-bottom"><UL class="nav nav-tabs" role="tablist" style="margin-left: 0px"><LI><A href="#OverviewCFPS">Motivation and Overview</A></LI><LI><A href="#globally">Globally Affecting Factors</A></LI><LI><A href="#transcription">Transcription</A></LI><LI><A href="#translation">Translation</A></LI><LI><A href="#costs">Cost Considerations</A></LI><LI><A href="#advantages">Advantages and Possible Uses</A></LI></UL></NAV><DIV id="OverviewCFPS"><H1>Motivation and Overview</H1><P> We aim to design cell-free biosensors for an application in the open field, e.g. to detect <A href="https://2015.igem.org/team:Bielefeld-CeBiTec/Project/HeavyMetals" target="_blank">heavy metals</A> in water or <A href="https://2015.igem.org/team:Bielefeld-CeBiTec/Project/DateRapeDrugs" target="_blank">date rape drugs</A> in beverages. A promising approach for the development of these biosensors is Cell-Free Protein Synthesis, CFPS. </P><P> You may ask: <B>Cell-free</B> protein synthesis? Transcription and translation without any organism? How is that possible? </P><P><B> By applying one of iGEMs central ideas: Combination! For Cell Free Protein Synthesis (CFPS) you need: </B></P><UL><LI><B> A DNA template which carries all information about your protein of interest</B></LI><LI><B> The protein synthesis machinery composed of proteins and ribosomes </B></LI><LI><B> Amino acids, energy resources and cofactors to fuel the machinery.</B></LI></UL><P> This simple principle is summarized in the following illustration:</P></DIV><H1> Theoretical Background </H1><P> &quot;Cell-free protein synthesis&quot; (CFPS) is the production of proteins from nucleic acid templates without living cells, but with the use of the transcription and translational &quot;hardware&quot; that originally is built up by cells. This is possible because this &quot;hardware&quot; does not necessarily need a whole cell, but only some well characterized components to function properly. Among these components, the most obvious are RNA-Polymerase and Nucleoside-Triphosphates (NTPs) for transcription, as well as ribosomes and amino acids for the translation. However, the production of proteins  relies on many other molecules and the environment where the production takes place.  In the following we present a short overview on all parts (and aspects) that are necessary and essential for in vitro transcription and translation. When using crude cell extracts of <I>E. coli</I>, most compounds are brought to the reaction via the extract itself. 
</P><DIV id="globally"><H2> Globally Affecting Factors </H2><P><B>strain used</B></P><UL><LI> The early experiments on CFPS relayed on RNase- and protease-deficient strains <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Zubay1973">(Zubay, 1973</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pratt1984">Pratt, 1984</A>). Since then, many different strains were tested and genetically improved to support the system.   </LI><LI> Especially, strains that have less RNase- and protease activity, produce rare tRNAs, and carry mutations that counteract the depletion of amino acids during reaction build the best basis for an efficient in vitro protein synthesis. You can find detailed overviews in following publications:
</LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#JewettSwartz2004">Jewett and Swartz, 2004</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2006">Calhoun and Swartz, 2006</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Hong2015">Hong et al., 2015 </A></LI></UL><P><B>pH value</B></P><UL><LI>For efficient protein synthesis, a pH value above 6.5 is indispensable (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2008">Kim et al., 2008</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#JewettSwartz2004">Jewett and Swartz, 2004</A>). This is the main reason for the use of pH-adjusted washing buffers and phosphate buffer in the media, when <I>E. coli</I> extracts are prepared in-house. The optimal pH  for the reaction itself varies depending on the protein to be produced. To prevent precipitation of the protein, the pH should generally be between 6.5 and 8.5 (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Tokmakov2014">Tokmakov et al., 2014</A>). 
            </LI><LI>Different approaches for stabilizing the pH value have been proposed, ranging from the use of buffers  (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>) to the use of engineered enzymes (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2015">Kim et al., 2015</A>). </LI></UL><P><B>Energy</B></P><UL><LI> Life depends upon ATP as it is the main energy source in cellular organisms. To provide ATP throughout the CFPS-reaction, ATP has to be permanently regenerated. The translation machinery also needs two GTP for every amino acid that is incorporated into the growing peptide chain (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Mavelli2015">Mavelli et al., 2015</A>). Therefore, molecules with high energy content are needed that fuel phosphorylation of ADP (for example via creatin phosphate, abbreviated CP) or that fuel glycolysis (for example phosphoenolpyruvate, abbreviated PEP). In general, the use of the high energetic CP in combination with creatin kinase gives the best results (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Shimizu2010">Shimizu and Ueda, 2010</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kazuta2014">Kazuta et al., 2014</A>). However, CP and creatin kinase are very expensive compared to other reagents. In recent years many different energy sources have been established which give good results at less costs. You can find detailed information in reviews and particulary in these publications:  </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KimSwartz2001">Kim and Swartz, 2001</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#JewettSwartz2004">Jewett and Swartz, 2004</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Ma2010">Ma et al., 2010</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015a">Caschera and Noireaux, 2015a</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Cai2015">Cai et al., 2015</A></LI></UL><P><B>Medium</B></P><UL><LI> 2xYTP and 2xYTPG medium are most frequently  employed for production of homemade <I>E. coli</I> cell extract (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015a">Caschera and Noireaux, 2015a</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>). The phosphate buffer helps to maintain a stable pH during cultivation and cell harvest. 
    </LI></UL></DIV><DIV id="transcription"><H2> Transcription </H2><P><B>Template</B></P><UL><LI> The use of DNA as template has some advantages over the use of an mRNA template. For example, one needs only small amounts of DNA compared to mRNA to obtain the same output signal as long as transcription is not limiting (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Rosenblum2014">Rosenblum and Cooperman, 2014</A>). Compared to mRNA, DNA, especially plasmid DNA, is more stable during the reaction (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>). The use of linear DNA as a template can support protein production if the PURE system is used (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Chizzolini2014">Chizzolini et al., 2014</A>). Higher amounts of DNA template result in higher amounts of mRNA produced, but this is only to a certain level advantageous for protein production (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Chizzolini2014">Chizzolini et al., 2014</A>). 
    </LI><LI> To improve the synthesis of your protein of interest in <I>E. coli</I> cell extracts, the adjustment of the DNA-sequence takes less work but is of outstanding importance (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>). The resulting mRNA has to be highly expressible: RBS, distance and identity of the sequence between the RBS and the start codon, an 5'-upstream enhancer sequence and the first codons of the CDS greatly influence the &quot;expressibility&quot; of the mRNA (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al., 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Karig2012">Karig et al., 2012</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2013">Takahashi et al., 2013</A>). According to Spirin and Swartz, other sequence elements have minor effects, if any at all (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>). With an optimized DNA-sequence, the signal output observed can be increased drastically (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al., 2013</A>). Therefore, optimization of DNA should be carried out before the optimization of the other, more expensive compounds in the cell-free reaction.  </LI><UL><LI> Apart from the later translated region, the 5'-untranslated region (5'-UTR), is of major importance for an efficient reaction. </LI><FIGURE style="margin:auto; width:700px"><FIGCAPTION><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1758100" target="_blank">BBa_K1758100</A></FIGCAPTION></FIGURE><LI> These are the features of this sequence: </LI><UL><LI> AATAATTTTGTTTTAACTTTAA</LI><UL><LI> To increase the efficiency of translation initiation, the T7 <I>g10</I> leader sequence can be employed. This sequence contains the socalled epsilon motif TTAACTTTA (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>), and was originally described by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Olins1988 ">Olins et al. 1988</A>. It enhances the binding of the mRNA to the 16 S rRNA (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Olins1988">Olins et al. 1988</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Olins1989">Olins, Rangwala 1989</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2013">Takahashi et al. 2013</A>), and Olins et al. showed that this sequence can massively improve heterologous gene expression in <I>E. coli</I> (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Olins1988">Olins et al. 1988</A>).          </LI></UL><LI> poly-A-spacer </LI><UL><LI> With kinetic studies, Takahashi et al. showed that a spacer between the epsilon motive and the RBS improves the translation rate <I>in vitro</I>. This works when the spacer does not interact with the 30S subunit of the ribosome, which is the case for example for an all-adenine spacer (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2013">Takahashi et al. 2013</A>). They further determined a 10-A-spacer as suitable when using <I>E. coli</I> S30 extracts instead of the PURE system.</LI></UL><LI> GAAGGAG </LI><UL><LI> The effect of the ribosome binding site (RBS) on <I>in vitro</I> translation has been investigated extensively (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Karig2012">Karig et al. 2012</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al. 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2013">Takahashi et al. 2013</A>). Most important for an efficient reaction is a defined distance (4-9 bases) of the RBS to the following start codon (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Karig2012">Karig et al. 2012</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al. 2013</A>). The strength of the RBS has a minor effect as long as it is not too weak (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Karig2012">Karig et al. 2012</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2013">Takahashi et al. 2013</A>).
          </LI></UL><LI> AATAATCT</LI><UL><LI> According to Lentini et al. 2013, the sequence composition between RBS and start codon affects the expression level of the following gene. An AT-rich region gives the best results, whereas expression is lower with the biobrick scar TACTAGAG for example (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al. 2013</A>) </LI></UL></UL></UL><LI>Quality and purity of the DNA template is an important aspect. When using plasmid purification kits one has to keep in mind that RNase is part of the resuspension buffer. In the literature, opinions vary regarding which is the best method for purification (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kigawa2004">Kigawa et al., 2004</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2005">Calhoun and Swartz, 2005</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al., 2013</A>).  </LI></UL><P><B>RNA-Polymerase</B></P><UL><LI> The most widely used RNA-Polymerase for cell-free protein production is the T7-Polymerase from bacteriophage T7 which is highly efficient and provides fast transcription (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#ShinNoireaux2010">Shin and Noireaux, 2010</A>). The T7-Polymerase is highly specific to its T7-Promoter, and is commonly used to express proteins <I>in vivo</I>(<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sousa2003">Sousa and Mukherjee, 2003</A>). Simple in-house production of T7-Polymerase has been described in many publications, so even <I>E. coli</I> strains without endogenous T7-Polymerase can be utilized (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Yang2012a">Yang et al., 2012a</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Roos2014">Roos et al., 2014</A>). </LI><LI> To overcome the limits of T7-Polymerase, Shin and Noireaux established CFPS based on endogenous polymerases and sigma factors from <I>E. coli</I> (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#ShinNoireaux2010">Shin and Noireaux, 2010</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#ShinNoireaux2012">Shin and Noireaux, 2012</A>). With this work the possibilities of <I>in vitro</I> transcription-translation were  extended; it became possible to characterize other than bacteriophage promotors <I>in vitro</I> (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#ChappellFreemont2013">Chappell and Freemont, 2013</A>). This lead to the analysis of genetic circuits outside of living cells  (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Chappell2013">Chappell et al., 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al., 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Takahashi2015">Takahashi et al., 2015</A>).</LI></UL><P><B>NTPs</B></P><UL><LI>The four nucleoside triphosphates (NTPs) ATP, CTP, GTP and UTP are the building blocks of RNA and therefore essential for transcription. They are one of the main cost factors, and it was shown that it is possible to use nucleoside monophosphates (NMPs) instead, although CFPS is less effective in this case (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2005">Calhoun and Swartz, 2005</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2008">Kim et al., 2008</A>).  </LI></UL></DIV><DIV id="translation"><H2> Translation </H2><P><B>Amino acids</B></P><UL><LI> Nearly all proteins are built up by L-amino acids. Neglecting selenomethionine and pyrrolysine, there are 20 proteinogenic amino acids that all have to be brought to the cell-free reaction. This is a difficult task for the experimenter, because these molecules have different solubilities and pKa-values. Recently, an easy procedure for the production of an amino acid mastermix has been presented (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015b">Caschera and Noireaux, 2015b</A>).</LI><LI> Some amino acids – especially cysteine, tryptophan, arginine and serine – are depleted much faster in the cell-free reaction than others, for example due to enzymatic reactions (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2006">Calhoun and Swartz, 2006</A>). This is one of the biggest problems that occur during batch CFPS and leads to fast stop of protein production, unless appropriate knockout strains are used or amino acid concentrations are increased (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#JewettSwartz2004">Jewett and Swartz, 2004</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2006">Calhoun and Swartz, 2006</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pedersen2011">Pedersen et al., 2011</A>).</LI></UL><P><B>Other components involved in translation</B></P><UL><LI> Amino acids are fused to corresponding tRNAs and thus activated for translation through the action of aminoacyl-tRNA-synthetases, a process that is called aminoacylation. Additional <I>E. coli</I> tRNA supplied to the CFPS-reaction is advantageous (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2005">Calhoun and Swartz, 2005</A>), though it was observed that under certain conditions an addition has no effect (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#ShinNoireaux2010">Shin and Noireaux, 2010</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Cai2015">Cai et al., 2015</A>)  </LI><LI> Elongation and releasing factors coordinate translation. For example, the GTP-binding protein EF-Tu is the major elongation factor in <I>E. coli</I> (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>). It has been shown that the supplementation of EF-Tu improves translation efficiency (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Underwood2005">Underwood et al., 2005</A>), and that EF-Tu could mediate the incorpotation of non-natural aminoacids into proteins, for example selenocysteine (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#MIller2015">Miller et al., 2015</A>)</LI><LI> The importance of the mRNA sequence is described in the previous section </LI></UL><P><B>The protein of interest</B></P><UL><LI> Proteins are a very wide group of molecules, different in amino acid composition, 3D-structure and physicochemical properties. Because of the individuality of proteins, it is necessary to optimize the cell-free synthesis reaction in every case (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A>) A recent approach based on bioinformatics was made to better estimate how one can improve CFPS for a certain protein (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Tokmakov">Tokmakov et al., 2014</A>). Generally, the production is more successful when the transcript is short and the protein folds fast and is soluble (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al., 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Chizzolini2014">Chizzolini et al., 2014</A>). Aspects like the formation of disulfide bonds or a chromophore have to be considered as well as intrinsic disorder of the protein (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Tokmakov2015">Tokmakov et al., 2015</A>).  </LI><LI> CFPS can be used to produce proteins whose production is limited <I>in vivo</I> due to toxicity or membrane association, and also to incorporate unnatural or labeled amino acids (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kigawa2004">Kigawa et al., 2004</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Xu2005">Xu et al., 2005</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#BundySwartz2010">Bundy and Swartz, 2010</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Quast2015b">Quast et al., 2015b</A>). It has to be considered that production of complex proteins that are further processed after translation, for example membrane-bound proteins, is in general more feasible in eukaryotic cell extracts (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Quast2015a">Quast et al., 2015a</A>). </LI></UL></DIV><DIV id="additives"><H2> Additives</H2><P> There are lots of possible additives that are known to contribute to a more efficient <I>in vitro</I> protein production, as well small substances like DTT as enzymes (a good overview in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#SpirinSwartz2008">Spirin and Swartz, 2008</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Airen2011">Airen, 2011</A>). Although some cofactors, for example coenzyme A, NAD and folinic acid, seem more important than others, the effects often differ depending on conditions like cell extract, scale, protein of interest and so on.</P></DIV><DIV id="costs"><H2> Cost considerations</H2><P><I>In vitro</I> transcription and translation has been an expensive issue. Commercial systems composed of purified single components, like the PURE system (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Shimizu2010">Shimizu and Ueda, 2010</A>) or <I>E. coli</I> (T7) S30 Extract, work efficiently, but are often more than ten times more expensive than in-house generated reactions. A milestone for lowering the costs of selfmade extracts was the finding of Kim and coworkers that after lysis, highly active cell extracts can be produced with a single centrifugation step and a subsequent run-off reaction at 37 °C (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2006">Kim et al., 2006</A>). The run-off reaction facilitates degeneration of endogenous DNA, whereby the background expression is reduced (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Pratt1984">Pratt, 1984</A>). A final centrifugation step after the run-off reaction can be helpful (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>).  </P><P>Indeed, many researchers have proven that it is possible to further minimize the costs per reaction down to a fraction of cents (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CalhounSwartz2005">Calhoun and Swartz, 2005</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2011">Kim et al., 2011</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al., 2013</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015">Caschera and Noireaux, 2015</A>). Protocols for preparation of cell extracts, especially for efficient lysis of harvested cells, have been described in recent years (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Shrestha2012">Shrestha et al., 2012</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015">Kwon and Jewett, 2015</A>). With the Multiplex Automated Genome Engineering (MAGE) method, it became possible to  tag and purify all necessary proteins for translation (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Wang2012">Wang et al., 2012</A>). This approach has the potential to simplify and reduce the costs for cell-free <I>in vitro</I> reactions based on purified components (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kelwick2014">Kelwick et al., 2014</A>) .</P><P>These papers inspired us to experiment with selfmade <I>E. coli</I> cell extracts, aiming to develop protocols for the production of cheap cell extracts and to open cell-free protein synthesis to the iGEM community. </P></DIV><DIV id="advantages"><H2> Advantages and possible uses </H2><P>In general, the developments in CFPS proceed into two major directions: These are on the one hand the production of large quantities of proteins (Scale-up direction) without needing recombinant hosts. On the other hand the aim is to investigate distinct proteins or protein synthesis procedures at a small scale for a minimum of costs, and to enrich a protein of interest in a minimal volume (miniaturization direction) (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Alexandrov">Alexandrov and Johnston</A>). </P><P>With <I>in vitro</I> Transcription-Translation, it long was difficult to produce identical or higher amounts of active proteins when compared to <I>in vivo</I>. Recent publications proof that this is now possible and that the scale-up of CFPS becomes more and more efficient (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2014">Caschera and Noireaux, 2014</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kazuta2014">Kazuta et al., 2014</A>). </P><P>CFPS offers great advantages over classical overexpression in living cells: The reaction is not restricted to the experimenter by a cell wall, so one can add exactly the components to the reaction that are needed, for example additional co-factors, inducers or catalysts. One recent example is the use of an engineered enzyme to maintain a stable pH without the need of a chemical buffer (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kim2015">Kim et al., 2015</A>). Depending on the system, it is possible to use linear DNA or RNA as a template, skipping ligation or reverse transcription (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Rosenblum2014">Rosenblum and Cooperman, 2014</A>)</P><FIGURE style="float:right; width:120px; margin-left:20px"><FIGCAPTION> Click here to see how we modelled our self-made cell extract!</FIGCAPTION></FIGURE><P> One advantage of the clearly defined environment the cell-free reaction takes place in, is that &quot;[...] it should be possible to build a complete mathematical model describing the cellular mimic that could aid in designing new features.&quot; (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2013">Lentini et al., 2013</A>). Recent approaches have been made to mathematically model and characterize the PURE system (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Karzbrun2011">Karzbrun et al., 2011</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Stögbauer2012">Stögbauer et al., 2012</A>;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Mavelli2015"> Mavelli et al., 2015</A>). </P><P>Regarding Synthetic biology, CFPS emerged as a valuable tool, with the long-term potential to not only be integrated into, but create whole new fields of research. With CFPS, the synthesis of proteins that would kill the cells used for production became possible, just to mention for example antibiotic-like proteins (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Xu2005">Xu et al., 2005</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Yang2012b">Yang et al., 2012b</A>). The incorporation of non-natural amino acids into proteins via in vitro translation with appropriate Aminoacyl-tRNA-synthetases enables construction of proteins with completely new functionalities (for a review, see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Quast2015b">Quast et al., 2015b</A>). </P><P>The advantages of bacterial extracts are fast translation rate and flexibility. For example it is possible to synthesize complex molecules like proteins that contain disulfide-bonds and even membrane proteins with the help of <I>E. coli</I> extract (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Billerbeck2013">Billerbeck et al., 2013</A>;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Kimura-Soyema2014"> Kimura-Soyema et al., 2014</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Roos2014">Roos et al., 2014</A>). CFPS with eukaryotic cells is another option. Recently, synthesis of 80 µg/mL eYFP was reported with the use of BY-2 cell lysates  (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Buntru">Buntru et al., 2015</A>). Compared to other eukaryotic lysates, Buntru et el. showed that BY-2-lysates are cost effective (3$ for a 15 µl reaction). It is interesting that it is possible to combine extracts from <I>E. coli</I> with eukaryotic extracts, and that this combination can help to produce active, properly folded protein (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Zarate2010">Zárate et al., 2010</A>). </P><P>The level of miniaturization is becoming more and more “minimal”, as CFPS was reported to work in 300 nl-lipid droplets (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Taylor2015">Taylor and Sarles, 2015</A>). This points out that CFPS-systems are a valuable part  for the construction of artificial cells (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Lentini2014">Lentini et al., 2014</A>). Other exciting new developments, for example DNA hydrogels based on X-DNA (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Park2009">Park et al., 2009</A>; <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Zheng2012">Zheng et al., 2012</A>), reveal that CFPS is going to revolutionize research and applications around the world.</P></DIV><H2 id="CFPSreferences">References</H2><DIV class="references"><P id="Airen2011">
    Airen, I. O. (2011): Genome-wide Functional Genomic Analysis for Physiological Investigation and Improvement of Cell-Free Protein Synthesis. PhD thesis.
</P><P id="Alexandrov">
    Alexandrov, Kirill; Johnston, Wayne A.: Cell-free protein synthesis. Methods and protocols (Methods in Molecular Biology, Methods and Protocols, 1118).
</P><P id="Billerbeck2013">
    Billerbeck, Sonja; Härle, Johannes; Panke, Sven (2013): The good of two worlds: increasing complexity in cell-free systems. In Current opinion in
    biotechnology 24 (6), pp. 1037–1043. DOI: 10.1016/j.copbio.2013.03.007    
    
        
</P><P id="BundySwartz2010">
    Bundy, Bradley C.; Swartz, James R. (2010): Site-Specific Incorporation of p -Propargyloxyphenylalanine in a Cell-Free Environment for Direct
    Protein−Protein Click Conjugation. In Bioconjugate Chem. 21 (2), pp. 255–263. DOI: 10.1021/bc9002844  
    
        

</P><P id="Buntru 2015">
    Buntru, Matthias; Vogel, Simon; Stoff, Katrin; Spiegel, Holger; Schillberg, Stefan (2015): A versatile coupled cell-free transcription-translation system
    based on tobacco BY-2 cell lysates. In Biotechnology and bioengineering 112 (5), pp. 867–878. DOI: 10.1002/bit.25502    
    
        

</P><P id="Cai2015">
    Cai, Qi; Hanson, Jeffrey A.; Steiner, Alexander R.; Tran, Cuong; Masikat, Mary Rose; Chen, Rishard et al. (2015): A simplified and robust protocol for
    immunoglobulin expression in Escherichia coli cell-free protein synthesis systems. In Biotechnology progress. DOI: 10.1002/btpr.2082    
    

</P><P id="CalhounSwartz2005">
    Calhoun, Kara A.; Swartz, James R. (2005): An economical method for cell-free protein synthesis using glucose and nucleoside monophosphates. In
    Biotechnology progress 21 (4), pp. 1146–1153. DOI: 10.1021/bp050052y    
    
        

</P><P id="CalhounSwartz2006">
    Calhoun, Kara A.; Swartz, James R. (2006): Total amino acid stabilization during cell-free protein synthesis reactions. In Journal of biotechnology 123
    (2), pp. 193–203. DOI: 10.1016/j.jbiotec.2005.11.011    
    

</P><P id="CascheraNoireaux2014">
    Caschera, Filippo; Noireaux, Vincent (2014): Synthesis of 2.3 mg/ml of protein with an all Escherichia coli cell-free transcription-translation system. In
    Biochimie 99, pp. 162–168. DOI: 10.1016/j.biochi.2013.11.025    

</P><P id="CascheraNoireaux2015a">
    Caschera, Filippo; Noireaux, Vincent (2015a): A cost-effective polyphosphate-based metabolism fuels an all E. coli cell-free expression system. In
    Metabolic engineering 27, pp. 29–37. DOI: 10.1016/j.ymben.2014.10.007    
    
        
   
</P><P id="CascheraNoireaux2015b">
    Caschera, Filippo; Noireaux, Vincent (2015b): Preparation of amino acid mixtures for cell-free expression systems. In BioTechniques 58 (1), pp. 40–43. DOI:
    10.2144/000114249    
    

</P><P id="ChappellFreemont2013">
    Chappell, James; Freemont, Paul S. (2013): In vivo and in vitro characterization of σ70 constitutive promoters by real-time PCR and fluorescent
    measurements. In Methods in molecular biology (Clifton, N.J.) 1073, pp. 61–74. DOI: 10.1007/978-1-62703-625-2_7    
    

</P><P id="Chappell2013">
    Chappell, James; Jensen, Kirsten; Freemont, Paul S. (2013): Validation of an entirely in vitro approach for rapid prototyping of DNA regulatory elements
    for synthetic biology. In Nucleic acids research 41 (5), pp. 3471–3481. DOI: 10.1093/nar/gkt052    
    
        

</P><P id="Chizzolini2014">
    Chizzolini, Fabio; Forlin, Michele; Cecchi, Dario; Mansy, Sheref S. (2014): Gene position more strongly influences cell-free protein expression from
    operons than T7 transcriptional promoter strength. In ACS synthetic biology 3 (6), pp. 363–371. DOI: 10.1021/sb4000977    
    

</P><P id="Hong2015">
    Hong, Seok Hoon; Kwon, Yong-Chan; Martin, Rey W.; Des Soye, Benjamin J; de Paz, Alexandra M; Swonger, Kirsten N. et al. (2015): Improving Cell-Free Protein
    Synthesis through Genome Engineering of Escherichia coli Lacking Release Factor 1. In Chembiochem : a European journal of chemical biology 16 (5), pp.
    844–853. DOI: 10.1002/cbic.201402708    
    
        

</P><P id="JewettSwartz2004">
    Jewett, Michael C.; Swartz, James R. (2004): Mimicking the Escherichia coli cytoplasmic environment activates long-lived and efficient cell-free protein
    synthesis. In Biotechnology and bioengineering 86 (1), pp. 19–26. DOI: 10.1002/bit.20026    
    
        
</P><P id="Karig2012">
    Karig, David K.; Iyer, Sukanya; Simpson, Michael L.; Doktycz, Mitchel J. (2012): Expression optimization and synthetic gene networks in cell-free systems.
    In Nucleic acids research 40 (8), pp. 3763–3774. DOI: 10.1093/nar/gkr1191    
    
 </P><P id="Karzbrun2011">
    Karzbrun, Eyal; Shin, Jonghyeon; Bar-Ziv, Roy H.; Noireaux, Vincent (2011): Coarse-Grained Dynamics of Protein Synthesis in a Cell-Free System. In Phys.
    Rev. Lett. 106 (4). DOI: 10.1103/PhysRevLett.106.048104    
    

</P><P id="Kazuta2014">
    Kazuta, Yasuaki; Matsuura, Tomoaki; Ichihashi, Norikazu; Yomo, Tetsuya (2014): Synthesis of milligram quantities of proteins using a reconstituted in vitro
    protein synthesis system. In Journal of bioscience and bioengineering 118 (5), pp. 554–557. DOI: 10.1016/j.jbiosc.2014.04.019    
    
        
</P><P id="Kelwick2014">
    Kelwick, Richard; MacDonald, James T.; Webb, Alexander J.; Freemont, Paul (2014): Developments in the tools and methodologies of synthetic biology. In
    Frontiers in bioengineering and biotechnology 2, p. 60. DOI: 10.3389/fbioe.2014.00060    
    

</P><P id="Kigawa2004">
    Kigawa, Takanori; Yabuki, Takashi; Matsuda, Natsuko; Matsuda, Takayoshi; Nakajima, Rie; Tanaka, Akiko; Yokoyama, Shigeyuki (2004): Preparation of
    Escherichia coli cell extract for highly productive cell-free protein expression. In Journal of structural and functional genomics 5 (1-2), pp. 63–68. DOI:
    10.1023/B:JSFG.0000029204.57846.7d    
    

</P><P id="KimSwartz2001">
    Kim, Dong-Myung; Swartz, James R. (2001): Regeneration of adenosine triphosphate from glycolytic intermediates for cell-free protein synthesis. In Biotech.
</P><P id="Kim2015">
    Kim, Ho-Cheol; Kim, Kwang-Soo; Kang, Taek-Jin; Choi, Jong-Hyun; Song, Jaejoon; Choi, Yun Hee et al. (2015): Implementing bacterial acid resistance into
    cell-free protein synthesis for buffer-free expression and screening of enzymes. In Biotechnology and bioengineering. DOI: 10.1002/bit.25671    
    
        
</P><P id="Kim2011">
    Kim, Ho-Cheol; Kim, Tae-Wan; Kim, Dong-Myung (2011): Prolonged production of proteins in a cell-free protein synthesis system using polymeric carbohydrates
    as an energy source. In Process Biochemistry 46 (6), pp. 1366–1369. DOI: 10.1016/j.procbio.2011.03.008    
    
        
</P><P id="Kim2006">
    Kim, Tae-Wan; Keum, Jung-Won; Oh, In-Seok; Choi, Cha-Yong; Park, Chang-Gil; Kim, Dong-Myung (2006): Simple procedures for the construction of a robust and
    cost-effective cell-free protein synthesis system. In Journal of biotechnology 126 (4), pp. 554–561. DOI: 10.1016/j.jbiotec.2006.05.014    
    
        
</P><P id="Kim2008">
    Kim, Tae-Wan; Kim, Ho-Cheol; Kim, Dong-Myung (2008): A Highly Efficient and Economical Cell-free Protein Synthesis System Using the S12 Extract of
    Escherichia Coli.
</P><P id="Kimura-Soyema2014">
    Kimura-Soyema, Tomomi; Shirouzu, Mikako; Yokoyama, Shigeyuki (2014): Cell-free membrane protein expression. In Methods in molecular biology (Clifton, N.J.)
    1118, pp. 267–273. DOI: 10.1007/978-1-62703-782-2_18    
    
    
</P><P id="KwonJewett2015">
    Kwon, Yong-Chan; Jewett, Michael C. (2015): High-throughput preparation methods of crude extract for robust cell-free protein synthesis. In Scientific
    reports 5, p. 8663. DOI: 10.1038/srep08663    
    
        
</P><P id="Lentini2013">
    Lentini, Roberta; Forlin, Michele; Martini, Laura; Del Bianco, Cristina; Spencer, Amy C.; Torino, Domenica; Mansy, Sheref S. (2013): Fluorescent proteins
    and in vitro genetic organization for cell-free synthetic biology. In ACS synthetic biology 2 (9), pp. 482–489. DOI: 10.1021/sb400003y    
    
        
</P><P id="Lentini2014">
    Lentini, Roberta; Santero, Silvia Perez; Chizzolini, Fabio; Cecchi, Dario; Fontana, Jason; Marchioretto, Marta et al. (2014): Integrating artificial with
    natural cells to translate chemical messages that direct E. coli behaviour. In Nature communications 5, p. 4012. DOI: 10.1038/ncomms5012    
    
        

</P><P id="Ma2010">
    Ma, Rongrong; Yang, Zhong; Huang, Lei; Zhu, Xiangcheng; Kai, Lei; Cai, Jin et al. (2010): Construction of an efficient Escherichia coli cell-free system
    for in vitro expression of several kinds of proteins. In Eng. Life Sci. 10 (4), pp. 333–338. DOI: 10.1002/elsc.201000052    
    
        
    
</P><P id="Mavelli2015">
    Mavelli, Fabio; Marangoni, Roberto; Stano, Pasquale (2015): A Simple Protein Synthesis Model for the PURE System Operation. In Bulletin of mathematical
    biology. DOI: 10.1007/s11538-015-0082-8    
    
        
   
</P><P id="Miller2015">
    Miller, Corwin; Bröcker, Markus J.; Prat, Laure; Ip, Kevan; Chirathivat, Napon; Feiock, Alexander et al. (2015): A synthetic tRNA for EF-Tu mediated
    selenocysteine incorporation in vivo and in vitro. In FEBS letters. DOI: 10.1016/j.febslet.2015.06.039    
    
        
   
</P><P id="Olins1988">
    Olins, Peter O.; Devine, Catherine S.; Rangwala, Shaukat H.; Kavka, Kamilla S. (1988): The T7 phage gene 10 leader RNA, a ribosome-binding site that
    dramatically enhances the expression of foreign genes in Escherichia coli. In Gene 73 (1), pp. 227–235.
</P><P id="Olins1989"> 
    Olins, Peter O.; Rangwala, Shaukat H. (1989): A novel sequence element derived from bacteriophage T7 mRNA acts as an enhancer of translation of the lacZ
    gene in Escherichia coli. In J Biol Chem.
    
</P><P id="Pardee2014">    
    Pardee, Keith; Green, Alexander A.; Ferrante, Tom; Cameron, D. Ewen; DaleyKeyser, Ajay; Yin, Peng; Collins, James J. (2014): Paper-based synthetic gene networks. In Cell 159 (4), pp. 940–954. DOI: 10.1016/j.cell.2014.10.004.
    
</P><P id="Park2009">
    Park, Nokyoung; Kahn, Jason S.; Rice, Edward J.; Hartman, Mark R.; Funabashi, Hisakage; Xu, Jianfeng et al. (2009): High-yield cell-free protein production
    from P-gel. In Nature protocols 4 (12), pp. 1759–1770. DOI: 10.1038/nprot.2009.174    
    

</P><P id="Pedersen2011">
    Pedersen, Anders; Hellberg, Kristofer; Enberg, Johan; Karlsson, B. Göran (2011): Rational improvement of cell-free protein synthesis. In New biotechnology
    28 (3), pp. 218–224. DOI: 10.1016/j.nbt.2010.06.015    
    
</P><P id="Pratt1984">
    Pratt, J. M. (1984): Coupled Transcription-Translation in Prokaryotic Cell-Free Systems. In B. D. Hames, S. J. Higgins (Eds.): Transcription and
    translation. A practical approach. Oxford, Washington, D.C.: IRL Press (Practical approach series).
</P><P id="Quast2015a">
    Quast, Robert B.; Kortt, Oliver; Henkel, Jörg; Dondapati, Srujan K.; Wüstenhagen, Doreen A.; Stech, Marlitt; Kubick, Stefan (2015a): Automated production
    of functional membrane proteins using eukaryotic cell-free translation systems. In Journal of biotechnology 203, pp. 45–53. DOI:
    10.1016/j.jbiotec.2015.03.015    
    
</P><P id="Quast2015b">
    Quast, Robert B.; Mrusek, Devid; Hoffmeister, Christian; Sonnabend, Andrei; Kubick, Stefan (2015b): Cotranslational incorporation of non-standard amino
    acids using cell-free protein synthesis. In FEBS letters. DOI: 10.1016/j.febslet.2015.04.041    
    
    
</P><P id="Roos2014">
    Roos, Christian; Kai, Lei; Haberstock, Stefan; Proverbio, Davide; Ghoshdastider, Umesh; Ma, Yi et al. (2014): High-level cell-free production of membrane
    proteins with nanodiscs. In Methods in molecular biology (Clifton, N.J.) 1118, pp. 109–130. DOI: 10.1007/978-1-62703-782-2_7    
    
        
</P><P id="Rosenblum2014">
    Rosenblum, Gabriel; Cooperman, Barry S. (2014): Engine out of the chassis: cell-free protein synthesis and its uses. In FEBS letters 588 (2), pp. 261–268.
    DOI: 10.1016/j.febslet.2013.10.016    
    
        
</P><P id="Shimizu2010">
    Shimizu, Yoshihiro; Ueda, Takuya (2010): PURE technology. In Methods in molecular biology (Clifton, N.J.) 607, pp. 11–21. DOI: 10.1007/978-1-60327-331-2_2    
    
    
</P><P id="ShinNoireaux2010">
    Shin, Jonghyeon; Noireaux, Vincent (2010): Efficient cell-free expression with the endogenous E. Coli RNA polymerase and sigma factor 70. In Journal of
    biological engineering 4, p. 8. DOI: 10.1186/1754-1611-4-8    
    
        
</P><P id="ShinNoireaux2012">
    Shin, Jonghyeon; Noireaux, Vincent (2012): An E. coli cell-free expression toolbox: application to synthetic gene circuits and artificial cells. In ACS
    synthetic biology 1 (1), pp. 29–41. DOI: 10.1021/sb200016s    
    
        

</P><P id="Shrestha2012">
    Shrestha, Prashanta; Holland, Troy Michael; Bundy, Bradley Charles (2012): Streamlined extract preparation for Escherichia coli-based cell-free protein
    synthesis by sonication or bead vortex mixing. In BioTechniques 53 (3), pp. 163–174. DOI: 10.2144/0000113924    
    
        
</P><P id="Smith2014"> 
     Smith, Mark Thomas; Wilding, Kristen M.; Hunt, Jeremy M.; Bennett, Anthony M.; Bundy, Bradley C. (2014): The emerging age of cell-free synthetic biology. In FEBS letters  588 (17), pp. 2755–2761. DOI: 10.1016/j.febslet.2014.05.062. 
        
</P><P id="Sousa2003">
    Sousa, Rui; Mukherjee, Srabani (2003): T7 RNA Polymerase. In, vol. 73: Elsevier (Progress in Nucleic Acid Research and Molecular Biology), pp. 1–41.
</P><P id="SpirinSwartz2008">
    Spirin, Alexander S.; Swartz, James R. (2008): Cell-Free Protein Synthesis Systems: Historical Landmarks, Classification, and General Methods.
</P><P id="Stögbauer2012">
    Stögbauer, Tobias; Windhager, Lukas; Zimmer, Ralf; Rädler, Joachim O. (2012): Experiment and mathematical modeling of gene expression dynamics in a
    cell-free system. In Integrative biology : quantitative biosciences from nano to macro 4 (5), pp. 494–501. DOI: 10.1039/c2ib00102k    
    
    
</P><P id="Sun2013">
    Sun, Zachary Z.; Hayes, Clarmyra A.; Shin, Jonghyeon; Caschera, Filippo; Murray, Richard M.; Noireaux, Vincent (2013): Protocols for implementing an
    Escherichia coli based TX-TL cell-free expression system for synthetic biology. In Journal of visualized experiments : JoVE (79), pp. e50762. DOI:
    10.3791/50762    
    
        
</P><P id="Takahashi2015">
    Takahashi, Melissa K.; Hayes, Clarmyra A.; Chappell, James; Sun, Zachary Z.; Murray, Richard M.; Noireaux, Vincent; Lucks, Julius B. (2015): Characterizing
    and prototyping genetic networks with cell-free transcription-translation reactions. In Methods (San Diego, Calif.). DOI: 10.1016/j.ymeth.2015.05.020    
 </P><P id="Takahashi2013">
    Takahashi, Shuntaro; Furusawa, Hiroyuki; Ueda, Takuya; Okahata, Yoshio (2013): Translation enhancer improves the ribosome liberation from translation
    initiation. In Journal of the American Chemical Society 135 (35), pp. 13096–13106. DOI: 10.1021/ja405967h    
    
        

</P><P id="Taylor2015">
    Taylor, Graham J.; Sarles, Stephen A. (2015): Using Cell-Free Expression to Create Light-Activated Proteins In Situ in Droplet Interface Bilayer Networks.
    In MRS Proc. 1720. DOI: 10.1557/opl.2015.63    
    
        

</P><P id="Tokmakov2014">
    Tokmakov, Alexander A.; Kurotani, Atsushi; Shirouzu, Mikako; Fukami, Yasuo; Yokoyama, Shigeyuki (2014): Bioinformatics analysis and optimization of
    cell-free protein synthesis. In Methods in molecular biology (Clifton, N.J.) 1118, pp. 17–33. DOI: 10.1007/978-1-62703-782-2_2    
    
    
</P><P id="Tokmakov2015">
        Tokmakov, Alexander A.; Kurotani, Atsushi; Ikeda, Mariko; Terazawa, Yumiko; Shirouzu, Mikako; Stefanov, Vasily et al. (2015): Content of intrinsic disorder influences the outcome of cell-free protein synthesis. In: Scientific reports 5, S. 14079. DOI: 10.1038/srep14079.      
        
        </P><P id="Underwood2005">
    Underwood, Kelly A.; Swartz, James R.; Puglisi, Joseph D. (2005): Quantitative polysome analysis identifies limitations in bacterial cell-free protein
    synthesis. In Biotechnology and bioengineering 91 (4), pp. 425–435. DOI: 10.1002/bit.20529    
    
        

</P><P id="Wang2012">
    Wang, Harris H.; Huang, Po-Yi; Xu, George; Haas, Wilhelm; Marblestone, Adam; Li, Jun et al. (2012): Multiplexed in vivo His-tagging of enzyme pathways for
    in vitro single-pot multienzyme catalysis. In ACS synthetic biology 1 (2), pp. 43–52. DOI: 10.1021/sb3000029    
    
        

</P><P id="Xu2005">
    Xu, Zhinan; Chen, Haiqin; Yin, Xiufei; Xu, Naizheng; Cen, Peilin (2005): High-Level Expression of Soluble Human b-Defensin-2 Fused With Green Fluorescent
    Protein in Escherichia coli Cell-Free System. In ABAB 127 (1), pp. 053–062. DOI: 10.1385/ABAB:127:1:053    
    
        

</P><P id="Yang2012a">
    Yang, William C.; Patel, Kedar G.; Wong, H. Edward; Swartz, James R. (2012a): Simplifying and streamlining Escherichia coli-based cell-free protein
    synthesis. In Biotechnology progress 28 (2), pp. 413–420. DOI: 10.1002/btpr.1509    
    
        

</P><P id="Yang2012b">
    Yang, Zhong; Ma, Rongrong; Huang, Lei; Zhu, Xiangcheng; Sheng, Jiayuan; Cai, Jin et al. (2012b): High-level production of soluble adenine nucleotide
    translocator from Schistosoma japonicum in E. coli cell-free system. In Process Biochemistry 47 (3), pp. 395–400. DOI: 10.1016/j.procbio.2011.11.010    
    
        

</P><P id="Zarate2010">
    Zárate, Xristo; Henderson, David C.; Phillips, Keenan C.; Lake, April D.; Galbraith, David W. (2010): Development of high-yield autofluorescent protein
    microarrays using hybrid cell-free expression with combined Escherichia coli S30 and wheat germ extracts. In Proteome science 8, p. 32. DOI:
    10.1186/1477-5956-8-32    
    
        

</P><P id="Zheng2012">
    Zheng, Qiang; Shi, Rui; Zhu, Xiangcheng; Huang, Lei; Cai, Jin; Han, Wenbiao; Xu, Zhinan (2012): Functional expression of Bacillus subtilis xylanase A in an
    Escherichia coli derived cell-free protein synthesis system and subsequent expression improvement via DNA gel technique. In Process Biochemistry 47 (7),
    pp. 1186–1191. DOI: 10.1016/j.procbio.2012.02.002    
    
        

</P><P id="Zubay1973">
    Zubay, G. (1973): In vitro synthesis of protein in microbial systems. In Annual review of genetics 7, pp. 267–287. DOI: 10.1146/annurev.ge.07.120173.001411    
</P></DIV><DIV class="row footer"><DIV class="col-md-2 col-md-offset-5 text-center"><A type="button" class="btn btn-default btn-next" href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS"><P>Results</P></A></DIV><DIV class="col-md-2 col-md-offset-3 text-center"><A type="button" class="btn btn-default btn-next" href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA"><P>Our second biosensor design: PRIA</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>