Team:Bielefeld-CeBiTec/Modeling/Biosensor
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Biosensor Model
Adding repression, induction and resource sharing.
Model system
TX/TL
Competition
Repression and induction
Final model
Subsequent to developing a model of cell-free protein synthesis, we expanded this model in order to describe a repressor-based biosensor.
A model system for our model
The most difficult part of modeling a biological system is finding parameters which accurately describe the processes of interest. We were aware that we would not be able to obtain exact parameters for the repressor and activator proteins we wanted to work with. For this reason, we decided to turn to the best characterized example of gene regulation out there: the lac operon.
Several models of the lac operon with focus on various aspects have been published. We came across a paper by Stamatakis and Mantzaris , who investigated the influence of stochasticity on the system, with special focus on the positive feedback that is created by the LacY permease. Although this effect was not of interest to us, we figured that their deterministic model of the lac operon might serve as a basis for modeling our biosensors. Consequently, we implemented this model in SimBiology® and and performed a system check by reproducing the simulation data presented in the publication. The model is based on mass action kinetics and includes central steps such as the transcription and translation of lacI and lacY, repression by LacI and derepression by IPTG, as well as the transport of IPTG into the cell either by diffusion or LacY-mediated active transport. The model assumes a dimeric version of LacI and only one operator site. While these assumptions are simplifications with regard to the lac operon, they are in line with many systems we worked with.
In the next step, we combined the relevant reactions of the lac operon model with our model of cell-free protein synthesis. More precisely, we adopted the dimerization, repression and derepression reactions and parameters. Transcription and translation reactions were described in accordance with our CFPS model, while other reactions of the lac operon model were omitted. This includes the transport of the analyte, which is not necessary in an open system like a CFPS reaction. Furthermore, we assumed the degradation of the repressor to be negligible, as one reaction only spans a few hours and because we used the E. coli strain ER2566 for the extract preparation, which contains fewer proteases than others.
In the following, the important elements of the model are described in more detail.
Transcription and translation
The transcription and translation of the repressor are described with the differential equations and parameters that were determined for sfGFP in our CFPS model. This is an approximation, which requires the genetic build-up of the plasmids and the translation and folding efficiency of the proteins to be similar. For a more accurate description, it would be necessary to measure the expression of the repressor and determine the specific parameters.
Competition for resources
Competition for resources as predicted by the model (solid lines) and as measured for two plasmids encoding sfGFP and mRFP1, respectively. The model predictions were normalized to the sfGFP fluorescence after 160 min when using only one plasmid in order to account for day-to-day variations in CFPS performance.
As we found out during the experiments for our CFPS model, the protein synthesis capacity of our CFPS is limited, which means that increasing the plasmid concentration does not result in a linear increase in sfGFP amount above approximately 7 nM plasmid. The extended model can describe the expression of two genes, so it needs to be taken into account that they compete for the limited resources. We modeled the competition of reporter and repressor mRNA for translation resources as a competitive inhibition. In this special case, this is possible by adding the concentration of the "inhibiting" mRNA to the denominator of the Hill equation for translation ( Cortés et al. 2001 ). The result is that the maximal velocity of both translation reactions remains unchanged - when one mRNA species is present in abundance, it outcompetes the other. However, when both mRNA species are present in high equimolar amounts, both can only be translated at half maximal speed.
We validated this approach by comparing the model predictions to an experiment in which we used two plasmids which encoded sfGFP and mRFP1, respectively. We measured the sfGFP fluorescence without a second plasmid (green data points in the figure) and with an equimolar amount of mRFP1 plasmid (red data points). As predicted by the model, the addition of a second plasmid resulted in a decrease in sfGFP production. This decrease was slightly lower than predicted, which might indicate that mRFP1 was not expressed as well as sfGFP.
Repression and induction
After its translation, the repressor dimerizes and can then bind to the operator and prevent the expression of the reporter, except for a slight leak expression. Leak expression plays an important role in gene regulation ( Huang et al. 2015 ), however, as we estimated it to be approximately 2 % of the unrepressed transcription (in accordance with ( Stamatakis, Mantzaris 2009 )), it is of little relevance in our model. Nevertheless, we decided to include it in our model, as it might be important when the model is to be adapted to systems which display strong leak transcription.
The repressor can be inactivated by binding of two analyte molecules. In this context it is important that both the operator-bound repressor and the free repressor dimer can be inactivated. Importantly, all these reactions are reversible.
The final model
Our final model consists of twelve species and eleven differential equations, which describe a repressor-based biosensor in a cell-free protein synthesis reaction. The following diagram illustrates the species and reactions.
The following abbreviations were used for the species:
Symbol
Species
GR
repressor gene
mR
repressor mRNA
R
repressor monomer
R2
repressor dimer
O
free operator of reporter gene
R2O
repressor-operator complex
A
analyte
R2A2
inactive repressor
mF
reporter mRNA
Fin
inactive reporter
F
reporter
TLR
TL resources
The differential equations are the following:
d
[
m
R
]
dt
=
v
TX
2
·
[
G
R
]
2
K
TX
2
2
+
[
G
R
]
2
−
λ
m
2
·
[
m
R
]
d
[
R
]
dt
=
k
TL
2
·
[
TLR
]
·
[
m
R
]
3
K
TL
2
3
+
[
m
R
]
3
+
[
m
F
]
3
−
2
·
k
2
R
·
[
R
]
2
+
2
·
k
−
2
R
·
[
R
2
]
d
[
R
2
]
dt
=
k
2
R
·
[
R
]
2
−
k
−
2
R
·
[
R
2
]
−
k
r
·
[
R
2
]
·
[
O
]
+
k
−
r
·
[
R
2
O
]
−
k
dr
1
·
[
A
]
2
·
[
R
2
]
+
k
−
dr
1
·
[
R
2
A
2
]
d
[
R
2
O
]
dt
=
k
r
·
[
R
2
]
·
[
O
]
−
k
−
r
·
[
R
2
O
]
−
k
dr
2
·
[
A
]
2
·
[
R
2
O
]
+
k
−
dr
2
·
[
R
2
A
2
]
·
[
O
]
d
[
A
]
dt
=
−
2
·
k
dr
1
·
[
A
]
2
·
[
R
2
]
+
2
·
k
−
dr
1
·
[
R
2
A
2
]
−
2
·
k
dr
2
·
[
A
]
2
·
[
R
2
O
]
+
k
−
dr
2
·
[
R
2
A
2
]
·
[
O
]
d
[
R
2
A
2
]
dt
=
k
dr
1
·
[
A
]
2
·
[
R
2
]
−
k
−
dr
1
·
[
R
2
A
2
]
+
k
dr
2
·
[
A
]
2
·
[
R
2
O
]
−
k
−
dr
2
·
[
R
2
A
2
]
·
[
O
]
d
[
O
]
dt
=
−
k
r
·
[
R
2
]
·
[
O
]
+
k
−
r
·
[
R
2
O
]
+
k
dr
2
·
[
A
]
2
·
[
R
2
O
]
−
k
−
dr
2
·
[
R
2
A
2
]
·
[
O
]
d
[
m
F
]
dt
=
k
TX
1
·
[
O
]
2
K
TX
1
2
+
[
O
]
2
−
λ
m
1
·
[
m
F
]
+
k
leak
·
[
R
2
O
]
d
[
F
in
]
dt
=
k
TL
1
·
[
TLR
]
·
[
m
F
]
3
K
TL
1
3
+
[
m
F
]
3
+
[
m
R
]
3
–
k
mat·
[
F
in
]
d
[
F
]
dt
=
k
mat
·
[
F
in
]
d
[
TLR
]
dt
=
−
v
λ
TLR
·
[
TLR
]
K
λ
TLR
+
[
TLR
]
All parameters and their sources are summarized in the following table:
Parameter
Description
Value
Reference
vTX1
reporter transcription rate constant
18.2 nM min-1
( Stögbauer et al. 2012 )
vTX2
repressor transcription rate constant
18.2 nM min-1
( Stögbauer et al. 2012 )
KTX1
Michaelis-Menten constant for reporter transcription
8.5 nM
( Stögbauer et al. 2012 )
KTX2
Michaelis-Menten constant for repressor transcription
8.5 nM
( Stögbauer et al. 2012 )
λm1
reporter mRNA degradation rate constant
0.08 min-1
(Karzbrun et al. 2011)
λm2
repressor mRNA degradation rate constant
0.08 min-1
(Karzbrun et al. 2011)
kTL1
reporter translation rate constant
0.0076 min-1
( Stögbauer et al. 2012 )
kTL2
repressor translation rate constant
0.0076 min-1
( Stögbauer et al. 2012 )
KTL1
Michaelis-Menten constant for translation of reporter
29.9 nM
Fitting
KTL2
Michaelis-Menten constant for translation of repressor
29.9 nM
Fitting
vλTLR
TL resources degradation rate constant
13.5 nM min-1
Fitting
KλTLR
Michaelis-Menten constant for degradation of TL resources
53.2 nM
Fitting
kmat
reporter maturation rate constant
0.2 min-1
( Stögbauer et al. 2012 )
k2R
repressor dimerization rate constant
50 nM-1 min-1
( Stamatakis, Mantzaris 2009 )
k-2R
repressor dimer dissociation rate constant
0.001 min-1
( Stamatakis, Mantzaris 2009 )
kr
association rate constant for repression
960 nM-1 min-1
( Stamatakis, Mantzaris 2009 )
k-r
dissociation rate constant for repression
2.4 min-1
( Stamatakis, Mantzaris 2009 )
kdr1
association rate constant for first derepression mechanism
3.0E7 nM-2 min-1
( Stamatakis, Mantzaris 2009 )
k-dr1
dissociation rate constant for first derepression mechanism
12 min-1
( Stamatakis, Mantzaris 2009 )
kdr2
association rate constant for second derepression mechanism
3.0E7 nM-2 min-1
( Stamatakis, Mantzaris 2009 )
k-dr2
dissociation rate constant for second derepression mechanism
4800 nM-1 min-1
( Stamatakis, Mantzaris 2009 )
kleak
leak reporter transcription rate constant
0.0033 min-1
Estimation in accordance with ( Stamatakis, Mantzaris 2009 ) and ( Stögbauer et al. 2012 )
References
Cortés, Antoni; Cascante, Marta; Cárdenas, María Luz; Cornish-Bowden, Athel (2001): Relationships between inhibition constants, inhibitor concentrations for 50% inhibition and types of inhibition: new ways of analysing data. In Biochemical Journal (357), pp. 263–268.
Huang, Lifang; Yuan, Zhanjiang; Liu, Peijiang; Zhou, Tianshou (2015): Effects of promoter leakage on dynamics of gene expression. In BMC systems biology 9, p. 16. DOI: 10.1186/s12918-015-0157-z.
Karzbrun, Eyal; Shin, Jonghyeon; Bar-Ziv, Roy H.; Noireaux, Vincent (2011): Coarse-Grained Dynamics of Protein Synthesis in a Cell-Free System. In Phys. Rev. Lett. 106 (4). DOI: 10.1103/PhysRevLett.106.048104.
Stamatakis, Michail; Mantzaris, Nikos V. (2009): Comparison of deterministic and stochastic models of the lac operon genetic network. In Biophysical journal 96 (3), pp. 887–906. DOI: 10.1016/j.bpj.2008.10.028.
Stögbauer, Tobias; Windhager, Lukas; Zimmer, Ralf; Rädler, Joachim O. (2012): Experiment and mathematical modeling of gene expression dynamics in a cell-free system. In Integrative biology : quantitative biosciences from nano to macro 4 (5), pp. 494–501. DOI: 10.1039/c2ib00102k.
Time to apply the model!
