<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Notebook_CFPS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Notebook/CFPS</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="text-center"><H1>Cell-free Protein Synthesis</H1><BUTTON id="button1" class="btn btn-success">Expand all</BUTTON><BUTTON id="button2" class="btn btn-danger">Hide all</BUTTON></DIV><DIV class="panel-group notebook" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseOne">Week 1 (4/20/2015-4/26/2015)</A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 04/22: </LI><UL><LI> Transformation of electrocompetent <I>E. coli</I> KRX with BBa_K1365020 (sfGFP (Bs))
  </LI></UL></UL><UL><LI> 04/23: </LI><UL><LI> preparing glycerol stock of KRX with BBa_K1365020 </LI></UL></UL><UL><LI> 04/24-04/25: </LI><UL><LI> Mini-prep of BBa_K1365020 </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwo">Week 2 (4/27/2015-5/3/2015)</A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>04/28: </LI><UL><LI> Transformation of  electrocompetent <I>E. coli</I> KRX with BBa_I746909 (T7-promoter, RBS, sfGFP, double terminator). Before plating on LB(Cm), 50 µL of 200 g∙L<SUP>-1</SUP> rhamnose-solution is added to the plate to induce T7-Polymerase. Inoculation of 10 mL preculture of simple KRX from plate. Overnight culturing, shaking vigorously at 220 rpm.</LI></UL></UL><UL><LI>04/29: </LI><UL><LI>colonies of KRX with BBa_I746909 look greenish! 2.2 mL Overnight cultures from KRX are used to inoculate 100 mL for start OD<SUB>600</SUB>=0.144. Measuring of growth curves for a first estimation of appropriate moment for cell harvest (see image below) </LI></UL><P style="font-size:70%"><B> Growth curves (duplicate) from <I> E. coli </I> KRX at 37 °C in 100 mL 2xTY+P-medium. 1 L shaking flasks without baffles were used. Shaking at 200 rpm. </B></P></UL><UL><LI>04/30: </LI><UL><LI>An aliquot of ER2566 also known as &quot;T7 Express Competent E. coli (High Efficiency)&quot; from NEB was kindly provided by Fabian. Cells were plated on LB.  </LI><LI> Plasmid-mini-prep from 3 mL of overnight culture (<I>E. coli</I> KRX with BBa_I746909), two glycerol stocks prepared. </LI><LI> Preparation of amino acid stock solutions for CFPS acccording to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#AApreparation" target="_blank">protocol</A> from <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015b" target="_blank">Caschera and Noireaux 2015b</A>. First, all aminoacids were seperatly weighed into microcentrifuge tubes, which took a while. Then, 500 µL of 5 M KOH  was added to each amino acid. Solubilization was achieved via multiple inverting and, if necessary, vortexing. Especially tyrosine takes a while, and is a suspension rather than a solution. Stock solutions are afterwards stored at -20 °C. Note: According to Caschera and Noireaux, these stock solutions can only be stored a few weeks. </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseThree">Week 3 (5/4/2015-5/10/2015)</A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>05/04: </LI><UL><LI> Preparation of two equimolar amino acid mixtures à 381.6 µL according to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015b" target="_blank">Caschera and Noireaux 2015b</A> (see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#CFPSprotocols" target="_blank">protocols</A>). Water is added to a final volume of 4 mL each.

pH adjusted with 110 µL glacial acetic acid to pH = 6.5 and aliquoting in 0.5 mL Eppendorf-tubes à 410 µL and 205 µL, respectively.  Aliquots were flash-freezed in liquid nitrogen and stored at -80 °C. </LI><LI> EcoR1 and Pst1 restriction analysis of isolated plasmid with BBa_I746909, bands as expected.
</LI><LI> 2x 10 mL 2xYT+P medium are inoculated with ER2566 from plate. 1x 10 mL is inoculated with KRX from glycerol stock. All cultures grow over night in 100 mL shake flasks at 37 °C and 220 rpm. </LI><LI> 05/05: Inoculation of 3x 100 mL 2xYT+P, each with one of the overnight cultures. Start OD<SUB>600</SUB>: </LI><UL><LI> KRX ⇒ 0.139 </LI><LI> ER2566 No. 1 ⇒ 0.146 </LI><LI> ER2566 No. 2 ⇒ 0.140 </LI></UL><LI>⇒ Plotting of growth curves. After 1.75 h, KRX and ER2566 No.2 are induced with 0.5 mL of 200 g∙L<SUP>-1</SUP> rhamnose  and 100 µL of 0.5 M IPTG to observe if production of T7-Polymerase has an impact on growth behavior. 
</LI></UL><P style="font-size:70%"><B> Growth curves of ER2566 and KRX</B></P><UL><LI> Also, Primerdesign for RraA and RraB. </LI></UL></UL><UL><LI>05/06: Preparation of ON cultures: </LI><UL><LI> T7p-RBS-sfGFP in 10 mL LB with Cm and 50 µl of 20% (w/w) rhamnose </LI><LI> T7p-RBS-sfGFP in 25 mL LB with Cm for midi-prep </LI><LI> Rosetta-gami2 in 10 mL 2xYT+P for growth curve </LI></UL></UL><UL><LI>05/07: </LI><UL><LI> Proving expression of sfGFP when T7-Polymerase is expressed in KRX through addition of rhamnose </LI><LI> midi-prep of 25 mL T7p-RBS-sfGFP ON culture </LI><LI> measuring of growth curve of Rosetta-gami2 (conditions like 05/05)</LI></UL><P style="font-size:70%"><B> Growth curves of Rosetta-gami2 in 100 mL 2xYT+P. Similar to 04/29, 1 L shaking flasks without baffles were used. Shaking at 200 rpm.  </B></P></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFour">Week 4 (5/11/2015-5/17/2015)</A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>05/11: </LI><UL><LI> Preparation of S30-Buffer (10 mM TRIS, 14 mM Mg-Glutamate, 60 mM K-Glutamate, to pH = 8.2 with acetic acid), autoclave </LI><LI> Inoculation of 10 mL 2xYT+P with ER2566 from glycerol stock, ON culture </LI></UL></UL><UL><LI>05/12: </LI><UL><LI> Inoculation of 2x 200 mL 2xYT+P (in 1 L shaking flasks without baffles) with ON culture of ER2566 to start OD<SUB>600</SUB> = 0.137, 200 rpm, 37 °C </LI><LI> Cultures induced at t = 1.5 h with 200 µL of 0.5 M IPTG each for a final IPTG concentration of 0.5 mM. For OD<SUB>600</SUB> progression see the following table. </LI></UL><TABLE><TBODY><TR><TH>time</TH><TH>OD<SUB>600</SUB> flask 1</TH><TH>OD<SUB>600</SUB> flask 2</TH></TR><TR><TD>0 h</TD><TD>0,137</TD><TD>0,137</TD></TR><TR><TD>1 h</TD><TD>0,379</TD><TD>0,397</TD></TR><TR><TD>1.5 h ⇒  induction</TD><TD>0,866</TD><TD>0,838</TD></TR><TR><TD>3.5 h</TD><TD>2,405</TD><TD>2,446</TD></TR><TR><TD>4 h </TD><TD>2,657</TD><TD>2,585</TD></TR></TBODY></TABLE><UL><LI> Cell harvest (see protocols)</LI></UL></UL><UL><LI> 05/13: </LI><UL><LI> Cell pellets thaw on ice for 1.5 h. Pellet weight: 0.677 g and 0.655 g respectively </LI><LI> Cells are resuspended in 1.5 mL icecold S30-buffer (with 3 µL 1 M DTT added each) with 10s vortex - 30 s ice resting cycles until pellets are fully resuspended. Suspensions are each transferred to a 2 mL microcentrifuge tube. </LI><LI> Cell disruption via sonification. Our power calibration (see protocols) revealed that the energy input of this specific Sonificator (Bandelin HD2070, MS73 tip) at this specific setup (70% power, cycle = constant) was about 3,69 J∙s<SUP>-1</SUP>. Therefore, for a total energy input of about 900 J, 24x 10s sonification - 10s cooling cycles were performed. The tip was dipped in about 2/3 of the cell suspension. During sonification, the microcentrifuge tube rested in ice rather than in an ice bath. As we observed that the tube becomes slightly warmer in the first case, all later sonifications were conducted with an ice bath. </LI><LI> After 10 cycles of sonification / cooling, we took 10 µL of the first tube after each further cycle and diluted them in 990 µL each. These samples were later used to conduct our first test about how effective our sonification kills <I>E. coli</I>: 20 µL of the diluted samples were plated on LB</LI><LI> Last steps of cell extract harvest were carried out as described in the protocol section with following variation: 
</LI><LI> cell extract in tube 2 has to be centrifuged a second time as the unsolvable fraction did not precipitate completely after the first  centrifugation </LI><LI> 10 µL of cell extract (supernatant after centrifugation) of tube 1 are separately flash-freezed for subsequent Bradford-Assay </LI></UL><LI> Cell extract from tube 2 is subjected to a run-off reaction for 50 min at 37 °C at 300 rpm in a thermomixer. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFive">Week 5 (5/18/2015-5/24/2015)</A></H4></DIV><DIV id="collapseFive" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>05/20: cell disruption</LI><UL><LI type="circle"> weight of the Pellet: 486 mg </LI><LI type="circle"> unfreeze the pellet on ice for two hours </LI><LI type="circle"> resuspend the pellet in 1 mL S30 buffer (with 2 mM DTT)
                    </LI><LI type="disk"> 10 sec vortex </LI><LI type="disk"> 30 sec on ice </LI><LI type="disk"> repeat these steps until die pellet is resuspended (in our case we repeated these steps 5 times) </LI></UL><LI type="circle"> The resuspended pellet had a volume of 1300 µL, therefore we added 200 µL S30 buffer (supplemented with 0.4 µL 1M DTT) to get a Volume of 1500 µL for the sonification step </LI><LI type="circle"> incubation the suspension for 10 min on ice </LI><LI type="circle"> sonification with Bandelin HD 2070 and the MS 73 cone end, we used 70% amplitude and no cycles 
                    </LI><LI> 10 sec sonifications </LI><LI> 10 sec incubation on ice </LI><LI> repeat step 1 and 2 (29 times, which equals an energy of ~ 900 J/sec) </LI><LI type="circle"> centrifuge for 10 min, 12000 rcf </LI><LI type="circle"> split the supernatant into two tubes </LI><LI type="circle"> incubate tube 2 (650 µL) for 50 min at 37°C and 300 rpm and centrifuge again for 10 min, 12000 rcf </LI><LI type="circle"> aliquot the supernatant and the cell extract from tube 1 (100 µL) </LI><LI type="circle"> freeze in liquid nitrogen and store at - 80°C </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSix">Week 6 (5/25/2015-5/31/2015)</A></H4></DIV><DIV id="collapseSix" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 05/27: </LI><UL><LI> Next CFPS test with plasmid T7p-RBS-sfGFP (BBa_I746909). Beside the negative control without DNA-template, BBa_K1365020 (sfGFP (Bs)), a plasmid with pSB1C3-backbone containing an sfGFP without promoter, is employed as additional negative control. Reactions with two different extracts (see 05/13: one with, one without run-off treatment) are performed. Furthermore, commercial RNase inhibitor is tested as it seems to be a useful supplement. A final RNase-inhibitor concentration of 1 U/µL is used in the respective reaction mixtures. CFPS is performed over night.
</LI></UL></UL><UL><LI> 05/28: </LI><UL><LI> PCR with Phusion Polymerase, primers: pSB1C3_RraA_suf/pSB1C3_RraA_pre and pSB1C3_RraB_suf/pSB1C3_RraB_pre, template: plasmid T7p-RBS-sfGFP (BBa_I746909). No bands in agarose gel.</LI><LI> Fluorescence measurement of CFPS-reaction from 05/27 with NanoDrop 3300</LI><LI> Next CFPS test with plasmid T7p-RBS-sfGFP (BBa_I746909). The reactions tested are the same as on 05/27, without testing the RNase Inhibitor ⇒ glycerol may inhibit protein syntesis <I>in vitro</I> (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Underwood2005" target="_blank">Underwood et al. 2005</A>). As a positive control, a sample from an overnight culture (BBa_I746909) induced to express sfGFP is employed. Reactions are performed at two different temperatures – 30 °C and 37 °C – in incubators over night without shaking. </LI></UL></UL><UL><LI> 05/29: </LI><UL><LI> No evaporation can be seen in the tubes! After 16 h incubation, the fluorescence is measured with NanoDrop 3300. </LI><LI> PCR problems continue</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSeven">Week 7 (6/1/2015-6/7/2015)</A></H4></DIV><DIV id="collapseSeven" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 06/01: </LI><UL><LI> Small amounts of amplified PCR product detectable, positive control works, but still far away from optimum.</LI></UL></UL><UL><LI> 06/03: </LI><UL><LI> ON culture of ER2566 (in 2xYT+P medium) is used for inoculation of 3x 100 mL 2xYT+P to OD<SUB>600</SUB> = 0.13. Cultures are induced with IPTG to 1 mM final concentration after 1 h 20 min at OD<SUB>600</SUB> about 0.7. </LI><LI> Cell harvest at t = 3.5 h at OD<SUB>600</SUB> = 3.3 </LI><LI> Subsequent to harvest and washing procedures, pellets are directly resuspended and lysed via sonification as before (29x for about 900 J). Half of the suspension is subjected to a run-off reaction at 37 °C for 50 min with a following . All extract is then flash-freezed and stored at -80 °C.
</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEight">Week 8 (6/8/2015-6/14/2015)</A></H4></DIV><DIV id="collapseEight" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 06/08: </LI><UL><LI> Primerdesign </LI><LI> Preparations for next CFPS reaction </LI></UL></UL><UL><LI> 06/09: </LI><UL><LI> CFPS with the same templates as on 05/28 </LI><LI> After assembling of CFPS compounds, 10 µL of each reaction were directly pipetted into a 364 black well plate, sealed with transparent film and incubated at 37 °C. Fluorescence was measured without the film after 3, 6.25 and 24 h respectively. Fluorescence did not exceed the negative control distinctly in any case. </LI></UL></UL><UL><LI> 06/10: </LI><UL><LI> CFPS with plasmid T7p-RBS-sfGFP (BBa_I746909) in 21 µL volume with “normal” and run-off extracts from 05/13, 05/20 and 06/03 respectively. For each reaction a negative control with water instead of DNA is employed </LI><LI> For the first time, Mg-glutamate and K-glutamate were additionally supplemented to the reactions as they were no part of the “reaction buffer”. 3 M K-glutamate and 100 mM Mg-glutamate stock solutions were prepared and directly used to supply the reaction with a final concentration of 4 mM Mg-glutamate and 80 mM K-glutamate. </LI><LI> To each mix of DNA, glutamate salts and water, a master mix consisiting of “reaction buffer” and one of the above mentioned extracts was added. Reactions were performed in microcentrifuge tubes for 3 h and then 20 µL were transferred to a 364 well plate which was used for fluorescence measurements and further reaction performance. Relative fluorescence units were normalized to corresponding negative controls. The maximum of fluorescence was observed after 7.75 h for 06/03 run-off extract, which fluorescence was 10 times higher when compared to its negative control. First promising results that showed we can really produce sfGFP <I>in vitro</I>!
                  </LI></UL></UL><UL><LI> 06/11: </LI><UL><LI> PCR with Q5-Polymerase for amplification of 5’-UTR-flanked pSB1C3-backbone. As PCR worked with Q5, most of subsequent PCRs related to CFPS were performed with Q5. </LI><LI> PCR with Q5 to amplify RraA and RraB out of <I>E. coli</I> K12 genomic DNA and to amplify pSB1C3 backbone with RraA and RraB overhangs for Gibson assembly</LI><LI> Bands as expected, gel purification of all four products. PCR for backbone amplification only worked at an annealing temperature of 71.6 °C.</LI></UL></UL><UL><LI> 06/12: </LI><UL><LI> Gibson assembly to insert RraA and RraB in pSB1C3 and to add UTR to T7p-RBS-sfGFP (BBa_I746909) plasmid. Subsequent transformation and plating on LB with Cm. </LI><LI> Transformation of electrocompetent <I>E. coli</I> KRX with BBa_K864100 (SYFP2) </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseNine">Week 9 (6/15/2015-6/21/2015)</A></H4></DIV><DIV id="collapseNine" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 06/15: </LI><UL><LI> Glycerol stocks prepared (RraA in pSB1C3 and RraB in pSB1C3, SYFP2, T7p-UTR-RBS-sfGFP)</LI><LI> Plasmid isolation of RraA in pSB1C3 and RraB in pSB1C3. Restriction analysis with Not1-HF. Bands as expected!</LI><LI> Plasmid isolation of T7p-UTR-RBS-sfGFP with columns of Promega kit and elution buffer of Thermo kit (10 mM TRIS-Cl, pH = 8.5) as material ran out. Restriction analysis with Not1-HF. Bands as expected!  </LI><LI> Chosen of the above plasmids were send to sequencing core facility </LI></UL></UL><UL><LI> 06/16: </LI><UL><LI> Preparation of a new <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#cofactorpremix" target="_blank">cofactor premix</A> (see following table)</LI></UL><TABLE><TBODY><TR><TH>Compound</TH><TH> Stock in mM</TH><TH>volume stock solution to add in µl</TH><TH>concentration in cofactor premix in mM</TH></TR><TR><TD>NAD</TD><TD>174</TD><TD>52.16</TD><TD>6.6</TD></TR><TR><TD>folinic acid</TD><TD>33,9</TD><TD>56.78</TD><TD>1.4</TD></TR><TR><TD><I>E. coli</I> tRNA </TD><TD>50*</TD><TD>110.00</TD><TD>4*</TD></TR><TR><TD>CoA</TD><TD>65</TD><TD>114.23</TD><TD>5.4</TD></TR><TR><TD>Putrescine</TD><TD>200</TD><TD>137.50</TD><TD>20</TD></TR><TR><TD>Hepes</TD><TD>2000</TD><TD>687.50</TD><TD>1000</TD></TR><TR><TD>Spermidine</TD><TD>1500</TD><TD>27.50</TD><TD>30</TD></TR><TR><TD>RNase-free water</TD><TD> </TD><TD>to 1375 µL</TD></TR></TBODY></TABLE><P style="font-size:70%"> HEPES was added first so that risks due to pH changes could be averted. </P><P style="font-size:70%"> * tRNA values given are in mg/mL </P><UL><LI> New reaction buffer aliquots were generated in which the above mentioned cofactor premix was used. For calculation we used the excel template from <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013" target="_blank">
Sun et al. 2013</A>, which takes into account a reserve for pipetting.</LI></UL><TABLE><TBODY><TR><TH>Compound</TH><TH>Stock </TH><TH>Final </TH><TH>amount of stock for 1 buffer in µl</TH><TH>amount of stock for 4 buffers  in µL</TH></TR><TR><TD>Amino Acids </TD><TD>12 mM</TD><TD>2 mM</TD><TD>14.96</TD><TD>62.84</TD></TR><TR><TD>Cofactor premix </TD><TD>20x</TD><TD>1x</TD><TD>4.49</TD><TD>18.85</TD></TR><TR><TD>NTPs </TD><TD>20x </TD><TD>1x</TD><TD>4.49</TD><TD>18.85</TD></TR><TR><TD>Na-Oxalat </TD><TD>100 mM</TD><TD>4 mM</TD><TD>3.59</TD><TD>15.08</TD></TR><TR><TD>PEP</TD><TD>330 mM</TD><TD>33 mM</TD><TD>8.98</TD><TD>37.71</TD></TR><TR><TD>Water</TD><TD>na</TD><TD>na</TD><TD>1.20</TD><TD>5.03</TD></TR></TBODY></TABLE><UL><LI> In subsequent CFPS-test, run-off cell extract from 06/03 was used. We tested the impact of RNase inhibitor, high amount of Mg-glutamate (12 mM) and high amount of K-glutamate (140 mM) on the reaction with T7p-RBS-sfGFP (BBa_I746909). Additionally, a T7p-UTR-RBS-sfGFP plasmid from a mini-prep plasmid isolation (06/15) was tested without any change to other components when compared to 06/10. All reactions were performed in duplicate.</LI><LI> Reaction conditions and volumes were the same as on 06/10. One negative control without DNA-template was used with 80 mM K-glutamate and 4 mM Mg-glutamate added. To compare relative fluorescent units, an sfGFP-containing cell lysate and a purified GFP with 6xHis-tag (kindly provided by Lukas) was employed. </LI><LI>For results, see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS" target="_blank">here</A></LI></UL></UL><UL><LI> 06/17: </LI><UL><LI> Plasmid isolation of SYFP2 (BBa_K864100). Restriction analysis with Not1-HF. Bands as expected!</LI><LI> Sequencing results: RraA in pSB1C3 and RraB in pSB1C3 look good, but T7p-UTR-RBS-sfGFP plasmid has only 9 A instead of the desired 10 A in spacer region. As CFPS worked well with this plasmid, it is nevertheless used in further experiments. </LI><LI> Transformation of chemocompetent ER2566 with pRARE plasmid kindly provided by Julian. </LI></UL></UL><UL><LI> 06/20: </LI><UL><LI> Generating new cell extrakt. We used <I> e. coli </I> KRX and BL(21)DE </LI><LI> 3 x100 mL were cultivated </LI><LI> after sonication we splited the extract. One half for the run off reaction ( 60 min 37°C) and the other half got directly flash freezed </LI><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTen">Week 10 (6/22/2015-6/28/2015)</A></H4></DIV><DIV id="collapseTen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 06/22: </LI><UL><LI> 3A-assembly of T7+RBS (BBa_K525998, upstream part), SYFP2 (BBa_K864100) and pSB1K3.m1 (destination vector) with NEB BioBrick Assembly Kit. </LI><LI> PCR problems with amplification of pSB1C3 backbone with His-TEV-EF-Tu  
</LI></UL></UL><UL><LI> 06/23: </LI><UL><LI> Transformation of chemocompetent <I>E. coli</I> KRX with ligation mix from 06/22. </LI><LI> glycerol stock prepared from ER2566 + pRARE. </LI><LI> PCR problem from 06/22 solved via restriction analysis: Wrong template was used. Therefore new try with correct template. Though it seemed that some false priming occured, relevant band is purified out of agarose gel.</LI></UL></UL><UL><LI> 06/24: </LI><UL><LI> colonies on plates with ligated T7-RBS-SYFP2 </LI><LI> Gibson assembly with His-TEV-EF-Tu from idt and pSB1C3 backbone with corresponding overhang, followed by transformation and plating.</LI></UL></UL><UL><LI> 06/25: </LI><UL><LI> Glycerol stock prepared from ON culture of T7-RBS-SYFP2 </LI><LI> Mini-prep from same culture, restriction analysis with Not1-HF. In 3% agarose gel, the difference between SYFP2 only and with preceded promoter was visible. Plasmids were send to sequencing core facility  </LI><LI> colonies on plates from Gibson assembly 06/24, Colony PCR, bands as expected, preparation of ON culture </LI></UL></UL><UL><LI> 06/26: </LI><UL><LI> CFPS reaction to compare extracts from different <I>E. coli</I> strains. To find out how fast the fluorescence signal is generated, the relative fluorescent units were measured every 30 min. Unfortunately, the delay between pipetting mastermix to DNA and first measurement was 15 min due to high number of samples. Results were inconsistent. </LI><LI> Glycerol stock prepared and plasmid mini-prep from His-TEV-EF-Tu in pSB1C3. Plasmid was send to sequencing core facility</LI><LI> PCR on His-TEV-EF-Tu in pSB1C3 to provide RraA and RraB with His-TEV-tag. </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEleven">Week 11 (6/29/2015-7/5/2015)</A></H4></DIV><DIV id="collapseEleven" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> 06/30: </LI><UL><LI> CFPS reaction </LI><LI> Sequencing results, as hoped no mutation was observed in idt sequence and T7-RBS-SYFP2 conctruct. </LI><LI> 50 mL LB + Cm were inoculated with His-TEV-EF-Tu in pSB1C3 (construct has T7p from former pT7-RBS-sfGFP) </LI><LI> Start of 5 L fermentation project: ER2566 from glycerol stock was plated on LB and cultivated in 37 °C incubator for about 16 h. </LI></UL></UL><UL><LI> 07/01: </LI><UL><LI> 4x 200 mL LB+Cm in 1 L shaking flasks were inoculated with His-TEV-EF-Tu in pSB1C3 ON culture, induction after 2 h with 20% (w/v) Rhamnose. Harvest of the cells after about 4 more hours via centrifugation at 4 °C for 30 min, flash freezing </LI><LI> Gel extraction of pSB1C3 backbone to provide RraA with His-TEV-Tag (RraA-purif). Afterwards, Gibson assembly and  transformation via heat shock to get RraA-purif and RraB-purif. </LI><LI> Inoculation of 2x 4 mL 2xYT+P with single colony from ER2566 plate. Cultivation at 37 °C for about 10 h. Afterwards, inoculation of 100 mL preculture and cultivation at 37 °C for 10 h. </LI></UL></UL><UL><LI> 07/02: </LI><UL><LI> 5 L fermentation in 7 L stirred reactor </LI><UL><LI> Biostat NLF 7 L reactor is filled with 5 L of 2xYT+P medium and steam-sterilized with the kind help of Thomas. During fermentation, software Bioscala is used, and pH is permanently and automatically adjusted to 7 </LI><LI> 50 mL of preculture are used for inoculation to start OD<SUB>600</SUB>=0.17 </LI><LI> Induction with IPTG after 1.5 h at OD<SUB>600</SUB> of about 0.88</LI><P style="font-size:70%"><B>Growth curve of ER2566 during 5 L fermentation </B></P><LI> Cell harvest after 3.2 h of cultivation at a final OD<SUB>600</SUB> of about 4.6. The culture passed through a self-built cooling system made of a Liebig condenser. Cooling is enabled by julabo thermostat. Half of the culture was directly poured into prechilled 50 mL tubes and centrifugated, while the other half was first kept in a separate 5 L glass bottle at 4 °C due to missing space in the centrifuge. </LI><P style="font-size:70%"><B> Cooling apparatus for harvesting <I>E. coli</I> culture </B><LI> All pellets were twice washed with 10 mL icecold S30A buffer per g of wet cell mass according to our protocol. S30A contained 50 mM instead of 10 mM TRIS. The third and final washing step was then conducted with 10 mL icecoled S30 buffer per g wet cells. </LI><LI> All 50 mL tubes with washed cell pellets were flash freezed in liquid nitrogen and stored at -80 °C</LI></P></UL></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwelve">Week 12 (7/6/2015-7/12/2015)</A></H4></DIV><DIV id="collapseTwelve" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 07/06: </LI><UL><LI> RraA-purif and RraB-purif were send to sequencing. </LI></UL></UL><UL><LI> 07/08: </LI><UL><LI> Glycerol stocks for RraA-purif and RraB-purif. Overexpression of both proteins similar to overexpression of His-TEV-EF-Tu on 07/01 except that after induction, cells were only cultivated for subsequent 3 h according to Gorna et al. 2010. </LI></UL><UL><LI> cell disruption via sonication</LI><LI> cells from fermentation were used </LI><LI> after sonication different run off reactions were made (20, 30, 40,50, 60, 70 and 80 min at 37°C and 140 rpm) </LI></UL><UL><LI> 07/09: </LI><UL><LI> To test self-made NTP solutions in comparison to purchased solutions, solid NTPs were weighted, dissolved in RNase-free water and 15% KOH was added. The solution was prepared according to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013" target="_blank">Sun et al., 2013</A>, but scaled down to 65%. Also, different salts of NTPs with different molecular weights were used, therefore the calculation had to be adjusted. NTP mixes were aliquoted and flash freezed. </LI></UL></UL><UL><LI> 07/10: </LI><UL><LI> CFPS reaction with self-made NTP-solution.
    </LI></UL></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseThirteen">Week 13 (7/13/2015-7/19/2015)</A></H4></DIV><DIV id="collapseThirteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 07/13: </LI><UL><LI> CFPS reaction, fluorescence signal did not rise to values comparable to 06/16.
    </LI></UL></UL><UL><LI> 07/14: </LI><UL><LI> PCR for adjusting the gap between RBS and first codon
    </LI><LI> PCRs to bring 5'-UTR in front of RFP and YFP </LI></UL></UL><UL><LI> 07/16: </LI><UL><LI> CFPS reaction, different approaches to determine the error source, but no success.
    </LI></UL><UL><LI> cell disruption from the remains of the fermentation via sonication </LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFourteen">Week 14 (7/20/2015-7/26/2015)</A></H4></DIV><DIV id="collapseFourteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> preparation of a new cofactor premix
    </LI><TABLE><TBODY><TR><TH>Compound</TH><TH> Stock in mM</TH><TH>volume stock solution to add in µL</TH><TH>concentration in cofactor premix in mM</TH></TR><TR><TD>NAD</TD><TD>174</TD><TD>26.08</TD><TD>6.6</TD></TR><TR><TD>folinic acid</TD><TD>33,9</TD><TD>28.39</TD><TD>1.4</TD></TR><TR><TD><I>E. coli</I> tRNA </TD><TD>50*</TD><TD>55.00</TD><TD>4*</TD></TR><TR><TD>CoA</TD><TD>65</TD><TD>57.12</TD><TD>5.4</TD></TR><TR><TD>Putrescine</TD><TD>200</TD><TD>68.75</TD><TD>20</TD></TR><TR><TD>Hepes</TD><TD>2000</TD><TD>343.75</TD><TD>1000</TD></TR><TR><TD>Spermidine</TD><TD>1500</TD><TD>13.75</TD><TD>30</TD></TR><TR><TD>RNase-free water</TD><TD> </TD><TD>to 687.5 µL </TD></TR></TBODY></TABLE><LI> Flash freezing of aliquots, stored at -80 °C, except for one aliquot that is directly used for a CFPS reaction. </LI><LI> Fluorescence signal did not rise to values comparable to 06/16.</LI></UL><UL><LI> 07/24: </LI><UL><LI> Primerdesign for RraA and RraB characterisation. </LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFifteen">Week 15 (7/27/2015-8/2/2015)</A></H4></DIV><DIV id="collapseFifteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> CFPS finally works! A mix of commercial NTP solutions gives great results compared to self-made mix from 07/09. The DNA template was purified via mini-prep kit, the washing steps were repeated once before elution. </LI></UL><UL><LI> 07/28: </LI><UL><LI> CFPS reaction with 3 replicates to test different ratios of extract/reaction buffer </LI><LI> PCR to adjust the gap between RBS and first codon finally works, subsequent gel extraction. </LI></UL></UL><UL><LI> 07/29: </LI><UL><LI>
    Promega 30S extract arrived!
    </LI><LI> Phosphorylation and ligation of UTR-mut-gap with T4 PNK and T4 DNA ligase followed by transformation. </LI><LI> CFPS reaction to test influence of arsenic on <I>in vitro</I> reaction. </LI></UL></UL><UL><LI> 07/30: </LI><UL><LI> Plasmid mini-prep of P<SUB>T7</SUB>-K516030 (RFP-generator). Restriction analysis via NotH1, size as expected.</LI><LI> PCR on P<SUB>T7</SUB>-K516030 with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#UTR_RFP_fwd">UTR_RFP_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#UTR_rev">UTR_rev</A> to get UTR in front of RFP</LI><LI> Colony-PCR on colonies from UTR-mut-gap. Bands as expected, however this gave no information about the sequence</LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSixteen">Week 16 (8/3/2015-8/9/2015)</A></H4></DIV><DIV id="collapseSixteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> Gibson assembly with P<SUB>T7</SUB>-UTR-SYFP</LI><LI> PCR on <I>E. coli</I> genomic DNA with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#rne_to_pZ9_fwd">rne_to_pZ9_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#rne_to_pZ9_rev">rne_to_pZ9_rev</A> to get RNase E DNA and PCR on pZ9 backbone with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pZ9_pT7_rne_fwd">pZ9_pT7_rne_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pZ9_pT7_rne_rev">pZ9_pT7_rne_rev</A></LI></UL><UL><LI> 08/04: </LI><UL><LI> Plasmid mini-prep of UTR-mut-gap from 07/30.
        </LI><LI> Preparation of Maltodextrin and Hexametaphosphate for test of low-cost energy solution in CFPS</LI><LI> Gibson assembly of RNase E and pZ9 backbone (hereafter named pZ9-rne), transformation and plating on LB+Kan</LI></UL></UL><UL><LI> 08/05: </LI><UL><LI> Sequencing of UTR-mut-gap was successful in 6 of 11 cases! ⇒ Glycerol stocks
    </LI></UL></UL><UL><LI> 08/07: </LI><UL><LI> Gibson assembly of P<SUB>T7</SUB>-UTR-RFP  
        </LI></UL></UL><UL><LI> 08/08: </LI><UL><LI> CFPS with T7-S30 extract from Promega to compare it with our extract.</LI><UL><LI> Two amino acid mixtures (-Leu and -Cys) were thawed, 50 µL of each combined, mixed, aliquoted and flash-freezed, except of one aliquot that was directly used for the reaction. </LI><LI> S30 premix without amino acids was thawed, aliquoted and flash-freezed, except of one aliquot that was directly used for the reaction.</LI><LI> &quot;T7 S30 circular&quot; extract was carefully thawed on ice, aliquoted and flash-freezed except of one aliquot that was directly used for the reaction.</LI></UL><LI> Glycerol stocks prepared and plasmid mini-prep of P<SUB>lac</SUB>-UTR-sfGFP, pZ9-rne and P<SUB>T7</SUB>-UTR-RFP. Subsequent restriction analysis. Bands as expected. </LI><LI> CFPS with <I>E. coli</I> KRX extract </LI></UL></UL><UL><LI> 08/09: </LI><UL><LI> CFPS with potential P<SUB>T7</SUB>-UTR-RFP plasmid to test wether we can express RFP in our extract. Additionally, low-cost reaction buffer is tested.  
    
    </LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSeventeen">Week 17 (8/10/2015-8/16/2015)</A></H4></DIV><DIV id="collapseSeventeen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> Sequencing results of P<SUB>lac</SUB>-UTR-sfGFP, pZ9-rne and P<SUB>T7</SUB>-UTR-RFP looked good, although exactly the plasmid P<SUB>T7</SUB>-UTR-RFP that was used on 08/09 had a mutation right after RBS.     
    </LI><LI> pZ9+rne is plated on LB+Kan to generate fast competent cells on 08/12 </LI></UL><UL><LI> 08/10: </LI><UL><LI>production of new cell extract </LI><LI>we used <I> e.coli </I> with the heavy metal activator or repressor plasmid to generate cell extraxt with the appropiate activator or repressor  </LI><LI>same procedure like <A data-toggle="collapse" href="#cellharvest">Cell harvest</A> and <A data-toggle="collapse" href="#Sonication">Sonication</A>. </LI></UL></UL><UL><LI> 08/11: </LI><UL><LI> Paper pieces provided with a hole-punch were autoclaved 3 times, on this paper pieces we run an CFPS </LI></UL></UL><UL><LI> 08/12: </LI><UL><LI> Planning of CFPS to help at modelling of <I>in vitro</I> reaction. </LI><LI> Preparation of electrocompetent pZ9-rne cells, followed by transformation with RraA-purif and RraB-purif, respectively. Plating on LB+Kan+Cm. </LI><LI> CFPS with the self made extract from 8/10. We tested to induce ArsO with different Arsenic concentrations </LI></UL></UL><UL><LI> 08/13: </LI><UL><LI> CFPS: Spiking experiments to test if some compounds are limiting after 2 h. </LI><LI> CFPS: Test to repress of ArsO. Therefore purified Arsenic Repressor were added</LI></UL></UL><UL><LI> 08/14: </LI><UL><LI> Preparation of glycerol stocks of pZ9-rne + RraA-purif and pZ9-rne + RraB-purif. 10 mL LB+Kan+Cm are inoculated with 100 µL of each culture </LI><LI> CFPS with addition of RraA from elution 1 of 07/11 and test of P<SUB>lac</SUB>-UTR-sfGFP in Promega S30 extract in 15 µL reactions. </LI></UL></UL><UL><LI> 08/15: </LI><UL><LI> Characterization of double transformants (pZ9-rne + RraA and pZ9-rne + RraB)</LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEighteen">Week 18 (8/17/2015-8/23/2015)</A></H4></DIV><DIV id="collapseEighteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 08/18: </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#Sonification">Sonification</A> of pellets form 07/02 and further processing creates enough ER2566 cell extract for the rest of the project </LI><LI> CFPS: with different plasmidcontrentrations of ArsO </LI><LI> CFPS: with 3 times autoclaved Paper pieces (Munktell C350L, Munktell FN3 and Filter paper from Merck)
    
    </LI></UL></UL><UL><LI> 08/19: </LI><UL><UL> Characterization of UTR <I>in vivo</I> by comparison of P<SUB>T7</SUB>-UTR-sfGFP and P<SUB>T7</SUB>-sfGFP.
        <LI> 2x 50 mL of LB+Cm are inoculated to a start OD<SUB>600</SUB> of 0.1 each. </LI><LI> Probes are taken after 3.25 h and 6.5 h and fluorescence is analyzed with Tecan platereader. </LI><LI> Normalized fluorescence signal of P<SUB>T7</SUB>-UTR-sfGFP culture after 3.25 h is even higher than fluorescence signal of P<SUB>T7</SUB>-sfGFP after 6.5 h! </LI></UL><UL> CFPS: on Paper and lyophilized<LI> CFPS: after the reactions are pipetted, the reaction ist pipetted on a paper disc put in a reaction tube (the reaction tubes were preparated with holes in the lid)  and flash freezed. </LI><LI> out of the liquid nitrogen the tubes were placed into the lyophilizer </LI><LI> the reactions were lyophilized for 3 or 24 hours </LI><LI> on the next day the reactions were rehydrated with 15 µL RNase free water to start the reaction </LI></UL></UL><UL><LI> 08/20: </LI><UL><LI> New PEP aliquots are prepared </LI><LI> Buffer exchange of RraA to 50 mM HEPES à pH 7.2 via Vivaspin columns.  Concentration of RraA was 4.5 mg/mL afterwards. </LI></UL></UL><UL><UL> CFPS: <LI> test of different Plasmidkonzentrations (15 nm, 10 nm, 7 nm, ,5 nm, 3 nm and 1 nm)</LI><LI> Influence of Alkohol on the reaction (5 % and 15 %) </LI></UL></UL><UL><LI> 08/21: </LI><UL><LI> CFPS reaction </LI><LI> Plating of 100 µL of cell extract aliquot on LB to see how many cells are still living </LI><LI> Same experiment to characterize UTR with RFP. Results confirm the use of UTR for fast expression, but experiment will be repeated with triplicates. </LI></UL></UL><UL><LI> 08/22: </LI><UL><LI> 8 cells living in a 100 µL cell extract aliquot. </LI><LI> preparation of new amino acid stocks, see <A data-toggle="collapse" href="#AApreparation">Amino acid preparation</A></LI><LI> CFPS: with the new Aminoacids stocks in comparison to the old one </LI></UL></UL><UL><LI> 08/23: </LI><UL><LI> CFPS reaction at 25 °C, and another CFPS reaction to see what happens when the DNA template is added delayed in time.   </LI></UL></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseNineteen">Week 19 (8/24/2015-8/30/2015)</A></H4></DIV><DIV id="collapseNineteen" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 08/24: </LI><UL><LI> PCR to add UTR to SYFP-template, fails. </LI></UL></UL><UL><LI> 08/25: </LI><UL><LI> CFPS reaction to compare the influence of 50 mM HEPES buffer at pH = 7.2 to the influence of purified His-TEV-RraA in 50 mM HEPES buffer at pH = 7.2. </LI></UL></UL><UL><LI> 08/26: </LI><UL><LI> Re-performance of UTR-characterization with triplicates. </LI><LI> CFPS: test influence of Chromium on the CFPS, and test of the heavy metal promoters </LI></UL></UL><UL><LI> 08/27: </LI><UL><LI> Further RraA characterization: 25 mL cultivation of pZ9-rne and pZ9-rne + RraA-purif (triplicates each). Rhamnose was added from the start to induce T7-Polymerase and expression of RNase E and RraA, respectively. After 2.5 h at OD<SUB>600</SUB> of 1.22, 2x 1 mL samples were quickly transferred into microcentrigfuge tubes, centrifuged for 15 s, and the pellet was flash-freezed. </LI><LI> RNA-isolation with Macherey-Nagel kit. Subsequent PCR with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#RraA_fwd">RraA_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#RraA_rev">RraA_rev</A> showed DNA contamination in the samples. </LI><LI> CFPS: test of the heavy metal promoters at 15 and 29 °C. </LI></UL></UL><UL><LI> 08/28: </LI><UL><LI> Digestion of RNA probes from 08/27 with Qiagen DNase (off-column digestion). Afterwards, samples were loaded onto RNA columns, washed and eluted. Nanodrop showed samples were contaminated with solvents. RNA was stored at -80°C. </LI><LI> PCR with split primers to create biobrick UTR in pSB1C3, successful only for one part. </LI></UL></UL><UL><LI> 08/29: </LI><UL><LI> Gradient PCR for amplification of missing fragment from 08/28. Bands showed up at 66 °C and were subsequently purified. </LI><LI> Gibson assembly, heat transformation and plating on LB+Cm to bring UTR to pSB1C3. </LI><LI> Restriction of P<SUB>T7</SUB>-mRFP generator, P<SUB>T7</SUB>-UTR-mRFP generator and P<SUB>T7</SUB>-SYFP with EcoR1HF and Pst1 to bring them from pSB1K3 to shipping vector. Afterwards, ligation ON.</LI><LI> CFPS: influence of Copper </LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapsetwenty">Week 20 (8/31/2015-9/6/2015)</A></H4></DIV><DIV id="collapsetwenty" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 9/3: </LI><UL><LI> CFPS: optimizing the different cell extract (extracts for the heavy metal reactions, with activator and repressor inside </LI></UL><UL><LI> 9/4: </LI><UL><LI> CFPS: generating new cell extract. Same procedure like 8/10, but this time the culture were induced with rhamnose to induce T7-Polymerase production </LI></UL><UL><LI> 9/5: </LI><UL><LI> CFPS: test of the new generated cell extract from the day before </LI></UL><UL><LI> 9/6: </LI><UL><LI> CFPS: comparison of Induktion of copAP and pT7-copAP with different copper concentrations </LI></UL><UL><LI> 9/6: </LI><UL><LI> CFPS: Induktion of pT7-prcnA, pT7-pbrAP and test of the influence of lead to the CFPS</LI><LI> CFPS: Induktion of pT7-chrP in the extract with our generated Repressor and in the extract with the repressor, generated from the igem Team Dundee 2015 </LI><LI> CFPS: lyophilization for 24 hours, rehydration  direktly after the lyophilization. 1 Part were stored ad 8 °C, another at 60°C and another part at room temperature. The storage occur for 6 days. Afterwards they were rehydrated with 15 µL table water </LI></UL></UL></UL></UL></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapsetwentyone">Week 21 (9/7/2015-9/13/2015)</A></H4></DIV><DIV id="collapsetwentyone" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI> 9/9: </LI><UL><LI> CFPS: induction of pT7-blcP with GBL (0,3%; 1%; 3%), GHB (0,3%; 1%; 3%) and Succinat </LI></UL></UL><UL><LI> 9/13: </LI><UL><LI> CFPS: try to induce the reaction after lyophilization with contaminated water </LI><UL><LI> preparation of different reactions (quadruple) </LI><UL><LI> pT7-blcP in extract with the corresponding repressor </LI><LI> pT7-pmerT in extract with the corresponding activator </LI><LI> pT7-copAP in extract with the corresponding activator </LI><LI> and our positiv control in our normal extract without overexpression of any repressor or activator </LI></UL><LI> The reaction was scaled up to 30 µL. The hole 30 µL were placed on a paper disc. </LI><LI> The paper disc were placed in a reaction tube with holes in it </LI><LI> Flash freeze </LI><LI> lyophilization for 3 hours </LI><LI> rehydration with different water (with and without contaminations) </LI><UL><LI> every reaction was rehydrated with 15 µL table water for comparison </LI><LI> every reaction was rehydrated with the corresponding inducer </LI><LI> pT7-copAP and pT7-pmerT were induced with a mixture of copper and mercury in table water </LI></UL></UL></UL></UL></P></DIV></DIV></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>