Team:Bielefeld-CeBiTec/Modeling/CFPS
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
CFPS Model
Or: Why does protein synthesis stop?
Introduction
Cessation of protein synthesis
TX/TL
Maturation
Fitting
Validation
Conclusion
Our model of cell-free protein synthesis (CFPS) describes the expression of sfGFP as observed in our experiments. The components of the model are illustrated in the diagram below. The rounded rectangles represent species such as DNA and proteins and the circles represent reactions. The major steps are transcription and translation, the maturation of sfGFP and the degradation of translation resources.
Cessation of protein synthesis
Protein synthesis is a common element of biomathematical models, but to describe cell-free protein synthesis, the special conditions have to be taken into account. Due to the dilution of the cell extract and a partial loss of activity, transcription and translation rates in vitro are typically one to two orders of magnitude lower than in vivo ( Karzbrun et al. 2011 ). The most important difference, however, are the limited resources in a batch-mode CFPS. Our experimental data show that the fluorescence signal reaches a plateau after 1-3 hours. This can have various reasons, for example depletion of energy resources or amino acids ( Jewett, Swartz 2004 ), degradation of ribosomes ( Stögbauer et al. 2012 ), accumulation of inhibitory byproducts ( Kim, Swartz 1999 ) or an unfavorable pH shift ( Kim et al. 2008 ).
Delayed plasmid addition to CFPS. CFPS reactions were incubated at 37 °C and plasmids were added at the given time points. Gaps in the data were caused by removing the plate for addition of plasmids.
Although the reasons can be manifold, several publications agree that translation is the limiting step, whereas transcription proceeds when protein synthesis has long stopped ( Iskakova et al. 2006 ; Stögbauer et al. 2012 ). However, it is an important question whether translation stops because resources have been consumed by the translation reaction or due to a process that is independent of translation, such as the degradation of ribosomes. We performed several wet lab experiments to investigate this issue. The first was a spiking experiment to find out if sfGFP production can be reconstituted by adding fresh components once the plateau has been reached, but neither amino acids, nor cell extract, nor energy source (phosphoenolpyruvate) had a significant effect. Next, we tested various plasmid concentrations in the CFPS reaction. If the reason for the termination of translation was the consumption of resources, one would expect the final protein amount to be the same regardless of the plasmid concentration. However, we observed a dependence of the protein amount on the plasmid concentration. Furthermore, the plateau was reached at approximately the same time, which indicated that the termination of protein synthesis is caused by a process that is not directly driven by translation. These results were underpinned by an experiment in which we added the plasmid after the other reaction components had already been incubated at 37 °C for a certain period of time. The longer the delay was, the less fluorescence was observed. Therefore, a process that proceeds independently of translation must be responsible for the stop of protein synthesis.
Consequently, we decided to include a species named "TL resources" that catalyzes the translation reaction and degrades over time ( Stögbauer et al. 2012 ). This virtual species comprises all components that are necessary for translation. Its degradation could be caused by several processes, such as enzymatic activity or a pH shift. We found that the degradation can be best described by Michaelis-Menten kinetics. The differential equation for the degradation of TL resources (TLR) is as follows. For an explanation of the parameters, refer to the table below.
d
[
TLR
]
dt
=
−
v
λ
TLR
·
[
TLR
]
K
λ
TLR
+
[
TLR
]
Transcription and translation
When fitting parameters to our data, we noticed that it was not possible to obtain good fits for both high and low plasmid concentrations using mass action or Michaelis-Menten kinetics for transcription and translation. However, we obtained good results when using Hill kinetics for both reactions. The Hill coefficients were empirically determined to be 2 for transcription and 3 for translation.  Hill coefficients greater than one are generally interpreted as indicators of positive cooperativity. Although this might not be as intuitive as in the case of allosteric enzymes, it is reasonable to think of protein synthesis as a cooperative process. For example, when one ribosome is bound to an mRNA, it prevents the formation of secondary structures and thus facilitates the binding of a second ribosome to the same transcript ( Iskakova et al. 2006 ; Underwood et al. 2005 ).
For the mRNA, a degradation constant of 0.08 min-1 was chosen in accordance with an mRNA lifetime of 12 min, which has been measured in an E. coli cell extract ( Karzbrun et al. 2011 ). We found that it yielded better results to include an mRNA degradation term, even though our optimized CFPS contains an RNase E inhibitor. Furthermore, the degradation term prevents an unrealistic, unlimited increase of the mRNA concentration.
Transcription of the sfGFP gene (O) and translation of the mRNA (mR) to the inactive fluorescent protein (Fin) are described by the following differential equations.
d
[
m
F
]
dt
=
k
TX
1
·
[
O
]
2
K
TX
1
2
+
[
O
]
2
−
λ
m
1
·
[
m
F
]
d
[
F
in
]
dt
=
k
TL
1
·
[
TLR
]
·
[
m
F
]
3
K
TL
1
3
+
[
m
F
]
3
–
k
mat
·
[
F
in
]
Maturation of sfGFP
After its translation, sfGFP requires a maturation step in which the chromophore is converted into its active form. This leads to a delay until fluorescence can be measured and was thus incorporated into our model. According to the literature, this maturation takes  approximately five minutes ( Stögbauer et al. 2012 ). GFP is a very stable protein and even in very long measurements, we observed only a very slight decrease of fluorescence intensity. Therefore, we did not include degradation terms for the reporter protein.
The maturation from the inactive (Fin) to the active (F) state is described as a first-order reaction:
d
[
F
]
dt
=
k
mat
·
[
F
in
]
Fitting
Experimental data and fitted curves. Several concentrations of a plasmid encoding sfGFP were tested in a CFPS reaction. Subsequently, three model parameters were fitted to this data set. The fitted curves are shown as solid lines.
Since "TL resources" is a virtual species, we could only determine its initial concentration and its degradation rate by fitting to our data. We also optimized the Michaelis-Menten constant of the translation to describe our data more accurately. To do this, we used the data sets which we had obtained by testing various plasmid concentrations and a delayed plasmid addition. In the first step, we estimated the sfGFP concentrations by comparing the fluorescence intensity to isolated GFP of known concentration. We then fitted three parameters and one concentration to the delayed plasmid addition data set using Matlab´s nlinfit (nonlinear least-squares problems) algorithm. The obtained Michaelis-Menten constant of the TL resources´ degradation was used for the model, while the other three values were fitted again to a portion of the data for various plasmid concentrations. We used this sequential fitting approach due to the day-to-day variations between experiments, which would otherwise compromise the goodness of fit. Furthermore, it seemed more important for the final biosensor model that alterations of the plasmid concentration are described accurately, because a repressor effectively reduces the concentration of plasmids that can be transcribed.
The final parameters of our CFPS model are summarized in the following table:
Parameter
Description
Value
Reference
vTX1
reporter transcription
18.2 nM min-1
( Stögbauer et al. 2012 )
KTX1
Michaelis-Menten constant for reporter transcription
8.5 nM
( Stögbauer et al. 2012 )
λm1
reporter mRNA degradation constant
0.08 min-1
( Karzbrun et al. 2011 )
kTL1
reporter translation rate constant
0.0076 min-1
( Stögbauer et al. 2012 )
KTL1
Michaelis-Menten constant for translation of reporter
29.9 nM
Fitting
[TLR]0
initial concentration of TL resources
1520 nM
Fitting
vλTLR
TL resources degradation rate constant
13.5 nM min-1
Fitting
KλTLR
Michaelis-Menten constant for degradation of TL resources
53.2 nM
Fitting
kmat
reporter maturation rate constant
0.2 min-1
(Stögbauer et al. 2012)
Validation
Hold-out validation. The model predictions for two plasmid concentrations which had not been part of the training data were compared to the experimental results. The fitted curves are shown as solid lines.
To check if the model performs equally well on unseen data as on the training data set, we compared the prediction of the model for two plasmid concentrations which had not been part of the fitting data set to the experimental data. As can be seen from the figure, the model predicts the sfGFP expression for 7 nM plasmid very well. For 1 nM, the model predicts a lower expression than we observed, but the difference appeared to be acceptable to us.
Conclusion
Our model of cell-free protein synthesis is able to predict the expression of sfGFP as a function of gene dosage and time of plasmid addition. It predicts the concentration of sfGFP, however, it should be noted that this is based on estimations and that the unavoidable batch-to-batch variations would require a frequent optimization of the parameters in order to predict exact concentrations. Nevertheless, even without such an optimization, the model is a valuable tool for predicting the qualitative behavior of CFPS reactions.
By investigating the cause of the cessation of protein synthesis, we also developed a better understanding of the processes in a CFPS reaction. This is important, as this knowledge can be used in order to improve the output signal. The experiments we performed for our modeling showed us that the amount of sfGFP can be raised by increasing the plasmid concentration to more than the 10 nM we had used up to that point. Consequently, we worked with 15 nM in further experiments.
References
Iskakova, Madina B.; Szaflarski, Witold; Dreyfus, Marc; Remme, Jaanus; Nierhaus, Knud H. (2006): Troubleshooting coupled in vitro transcription-translation system derived from Escherichia coli cells: synthesis of high-yield fully active proteins. In Nucleic acids research 34 (19), pp. e135. DOI: 10.1093/nar/gkl462.
Jewett, Michael C.; Swartz, James R. (2004): Substrate replenishment extends protein synthesis with an in vitro translation system designed to mimic the cytoplasm. In Biotechnology and bioengineering 87 (4), pp. 465–472. DOI: 10.1002/bit.20139.
Karzbrun, Eyal; Shin, Jonghyeon; Bar-Ziv, Roy H.; Noireaux, Vincent (2011): Coarse-Grained Dynamics of Protein Synthesis in a Cell-Free System. In Phys. Rev. Lett. 106 (4). DOI: 10.1103/PhysRevLett.106.048104.
Kim, D. M.; Swartz, J. R. (1999): Prolonging cell-free protein synthesis with a novel ATP regeneration system. In Biotechnology and bioengineering 66 (3), pp. 180–188.
Kim, Tae-Wan; Kim, Ho-Cheol; Oh, In-Seok; Kim, Dong-Myung (2008): Kim Highly Efficient Cell-Free Protein Synthesis. In Biotechnology and Bioprocess Engineering 13 (4), pp. 464–469.
Stögbauer, Tobias; Windhager, Lukas; Zimmer, Ralf; Rädler, Joachim O. (2012): Experiment and mathematical modeling of gene expression dynamics in a cell-free system. In Integrative biology : quantitative biosciences from nano to macro 4 (5), pp. 494–501. DOI: 10.1039/c2ib00102k.
Underwood, Kelly A.; Swartz, James R.; Puglisi, Joseph D. (2005): Quantitative polysome analysis identifies limitations in bacterial cell-free protein synthesis. In Biotechnol. Bioeng. 91 (4), pp. 425–435. DOI: 10.1002/bit.20529.
Continue with the complete biosensor model.
