Team:Bielefeld-CeBiTec/Notebook/HeavyMetals
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Heavy Metals
Expand all
Hide all
Week 1 (4/20/2015-4/26/2015)
We transformed BioBricks from the distribution:
BBa_J33201 (arsR)
BBa_K516030 (mRFP1 with RBS and terminator)
Colonies were used to inoculate overnight cultures.
Plasmids were isolated using a miniprep kit.
Week 2 (4/27/2015-5/3/2015)
We transformed BioBricks from the distribution:
BBa_K346001 (MerR) without success
BBa_K346002 (PmerT) without success
BBa_K608002 (konstitutiv Promoter+RBS) without success
Week 3 (5/4/2015-5/10/2015)
ArsR ( BBa_J33201 ) and RFP ( BBa_K516030 ) were combined using 3A assembly:
Restriction digest of arsR, RFP and pSB1K3.m1
PCR Clean-up
Ligation: 2 µL per fragment, 30 min
Transformation by electroporation and heat shock (for comparison)
No colonies, therefore the ligation was repeated with significantly more DNA. Incubation overnight.
Electroporation resulted in colonies, which were screened with a colony PCR.
A product length of 1705 bp was expected. The gel shows that most plasmids seem to be correct.
This result was confirmed by sequencing.
We repeated the Transformation of BioBricks from the distribution:
BBa_K346001 (MerR) without success
BBa_K346002 (PmerT)
BBa_K608002 (konstitutiv Promoter+RBS)
Gibson-Primerdesign
Week 4 (5/11/2015-5/17/2015)
...
Week 5 (5/18/2015-5/24/2015)
Amplification of rcnR and cueR for Gibson Assembly with Taq-Polymerase used E. coli K12 as template
rcnR amplification at 77.0°C and 78.9°C with rcnR.rev/fwd primes
PCR worked, positive control worked, no amplification of rcnR.
cueR amplification at 71.9°C and 74.6°C with cueR.rev/fwd primers
PCR worked, positive control worked, no amplification of cueR.
Amplification of rcnR and cueR for Gibson Assembly with Phusion Polymerase  used E. coli K12 as template
rcnR amplification at 54.8 °C and 56.1°C with rcnR.rev/fwd primers
PCR worked, positive control worked, no amplification of rcnR.
cueR amplification at 61.7°C and 62.8°C with cueR.rev/fwd primers
PCR worked, positive control worked, no amplification of rcnR.
Week 6 (5/25/2015-5/31/2015)
...
Week 7 (6/1/2015-6/7/2015)
rcnR amplification out of E. coli K12
phuison PCR with temerature gradient between 55-63°C
with pSB1C3_rcnR_fwd/rev (calulated annealing temperatures 66°C and 71°C
with pSB1C3_copAP_fwd/rev (calulated annealing temperatures 53°C and 73°C
PCR worked, positive control worked, no amplification of copAP and rcnR.
CueR amplification out of E. coli K12
phuison PCR with temerature gradient between 55.6-67.6°C
with pSB1C3_rcnR_fwd/rev (calulated annealing temperatures 66°C and 71°C
with pSB1C3_copAP_fwd/rev (calulated annealing temperatures 53°C and 73°C
PCR worked, positive control worked, no amplification of copAP and rcnR.
Week 8 (6/8/2015-6/14/2015)
Insertion of a second repressor binding site into BBa_J33201
Two primer pairs ( Cm_fwd , Ars_1_rev , Ars_2_fwd , Cm_rev ) were used to amplify the plasmid and introduce a second repressor binding site after arsR
A PCR with Phusion Polymerase failed. A gradient PCR did not show any improvement.
A comparison of Phusion and Q5 showed that Q5 works very well.
However, an upscalling of the reaction to produce more product did not work.
Therefore, two 20 µL reactions were used per fragment. This PCR worked well and the products were extracted from the gel.
We performed a Gibson assembly reaction with the two fragments and transformed the mix via heat shock.
A colony PCR was performed with five colonies, but only one reaction yielded product.
Five colonies were used to inoculate overnight cultures.
Plasmids were isolated from the cultures and a restriction digest was performed to check them.
The bands were as expected for all colonies.
One plasmid was handed in for sequencing, which confirmed its correctness.
Gerneration of Gibson fragments
phuison PCR with temerature gradient between 55.6-67.6°C
with pSB1C3_rcnR_fwd/rev (calulated annealing temperatures 66°C and 71°C
with pSB1C3_rcnA_fwd/rev (calulated annealing temperatures 49°C and 66°C
with pSB1C3_cueR_fwd/rev (calulated annealing temperatures 73°C and 74°C
with pSB1C3_copAP_fwd/rev (calulated annealing temperatures 53°C and 73°C
PCR worked, positive control worked, no amplification of cueR, copAP, rcnA and rcnR.
CueR amplification out of E. coli K12
KOD Polymerase PCR of CueR at 60.4°C, 61°C, 61.7°C and 62.3°C
PCR worked, positive control worked, no amplification of cueR.
rcnR amplification out of E. coli K12
KOD Polymerase PCR of rcnR at 60.4°C, 61°C, 61.7°C and 62.3°C
PCR worked, positive control worked, no amplification of rcnR.
Gerneration of Gibson fragments
Q5 PCR amplification with temperature gradient between 54.8°C and 68°C of the pSB1C3 backbone with Gibson overlaps:
pSB1C3_rcnR_fwd/rev (calculated temperatures: 66°C+71°C)
pSB1C3_rcnA_fwd/rev (calculated temperatures: 49°C+66°C)
pSB1C3_cueR_fwd/rev (calculated temperatures: 73°C+74°C)
pSB1C3_copAP_fwd/rev (calculated temperatures: 53°C+73°C)
no desired fragments just backbone residues
Week 9 (6/15/2015-6/21/2015)
3A assembly of the modified arsR and RFP ( BBa_K516030 )
Restriction digest of the modified arsR with EcoRI-HF and SpeI-HF
Heat inactivation: 20 min at 80 °C
Ligation with RFP and pSB1K3 (from week 3)
Electroporation with 2 µL ligation mix
Only one colony grew on the plates, which according to a colony PCR was not correct.
5 and 20 µL of the ligation  mix were transformed again into chemocompetent cells.
This time there were plenty of colonies. 10 were screened with a colony PCR and most appeared to be correct.
Phusion PCR amplification at 68°C of the pSB1C3 backbone with Gibson overlaps, using pSB1C3 sfGFP as template and split primer:
pSB1C3_rcnR_fwd + Cm_rev
pSB1C3_rcnR_rev + Cm_fwd
pSB1C3_cueR_suf + Cm_fwd
pSB1C3-cueR_pre + Cm_rev
PCR worked, positive control worked, no amplification of cueR and rcnR.
Week 10 (6/22/2015-6/28/2015)
First test with arsenic sensor
5 ml cultures (LB + kan) were prepared from glycerol stocks of E. coli harboring pSB1K3-arsR-RFP and pSB1K3-arsR-arsO-RFP and cultured at 37 °C.
After 16 h, cultures were diluted 1:100 with fresh LB + kan.
After 1 h at 37 °C, arsenic at 1000 µg/L was added.
After 24 h at 37 °C, the cultures were centrifuged (5 min, 5000xg).
The cell pellets were clearly red, indicating that the arsenic sensors are functional.
Phusion PCR amplification of the repressors/ activators with Gibson overlaps at 68°C , using pSB1C3 sfGFP as template and split primer:
kPrm_rev + Cm_fwd
cueR_rev + Cm_rev
rcnR_fwd + Cm_rev
chrB_fwd + Cm_rev
merR_fwd + Cm_rev
pbrR_fwd + Cm_rev
PCR worked, positive control worked, no amplification of cueR, copAP, rcnA and rcnR.
Phusion PCR amplification of the specific promoters with Gibson overlaps at 68°C , using pSB1C3 sfGFP as template and split primer:
Cm_fwd + rcnA_rev
Cm_fwd + copAP_rev
Cm_fwd + Cr_rev
Cm_fwd + pbrA_rev
Cm_rev + RFP_fwd
PCR worked, positive control worked, no amplification of specific promoters.
Week 11 (6/29/2015-7/5/2015)
Arsenic sensor test in 96 well plate
Dilutions of a 50 mM sodium arsenite solution were prepared
As for the first test, 5 mL cultures of the two strains were inoculated from glycerol stocks.
After 18 h, the cultures were diluted to reach an OD600 of 0.4.
After 30 min at 37 °C, 100 µL culture and 100 µL sodium arsenite solution were combined in a black 96 well plate.
The final arsenic concentrations ranged from 0 to 1000 µg/L.
Every combination of strain and concentration was measured in triplicate.
The plate was incubated at 37 °C for 4 h.
RFP fluorescence was measured in a plate reader.
No increase of fluorescence with the arsenic concentration was observable.
Measuring different dilutions of an RFP culture supernatant showed that the linear range was very small. Therefore, this instrument does not seem to be suitable for RFP measurements.
Repitition of the Phusion PCR amplification for Gibson assembly from week 10 using Primermixes
Cm_fwd + rcnA_rev (expected Primerlength:838 bp)
Cm_fwd + copAP_rev (expected Primerlength:837 bp)
Cm_fwd + Cr_rev (expected Primerlength:834 bp)
Cm_fwd + pbrA_rev (expected Primerlength:832 bp)
Amplification of the fragments was successfull.
Following gelextraction of the samples Cm_fwd + rcnA_rev, Cm_fwd + copAP_rev, Cm_fwd + Cr_rev, Cm_fwd + pbrA_rev
Q5 PCR amplification of pSB1C3 backbone for Gibson Assembly with cueR and rcnR overlaps used split primers
Cm_rev + pSB1C3_cueR_suf (expected Primerlength:1320 bp)
Cm_rev + pSB1C3_rcnR_fwd (expected Primerlength:1320 bp)
Cm_fwd + pSB1C3_cueR_pre (expected Primerlength:835 bp)
Cm_fwd + pSB1C3_rcnR_pre (expected Primerlength:835 bp)
Amplification of the Cm_rev + pSB1C3_cueR_suf and m_fwd + pSB1C3_cueR_pre fragments was successfull.
Following gelextraction of the samples Cm_rev + pSB1C3_Cuer_suf and m_fwd + pSB1C3_cueR_pre
Week 12 (7/6/2015-7/12/2015)
Optimization of arsenic sensor for CFPS
We decided to modify the arsenic sensor so that it is likely to work better in the CFPS.
The devices we decided to clone are:
An sfGFP under the control of the T7 promoter and the arsenic operator including the UTR which, as we showed, improves the sfGFP expression in the CFPS.
An sfGFP with the UTR behind arsR and its natural promoter.
A his-tagged arsR for purification and controlled addition to the CFPS reaction.
In the first step, the T7 promoter needed to be inserterd in front of arsR. Therefore, BBa_I719005 (T7 promoter) from the distribution was transformed by heat shock.
A colony PCR showed bands which matched the expected length.
Overnight cultures were inoculated with the colonies and plasmids were isolated.
Restriction digest:
pSB1A2-pT7 with EcoRI-HF and SpeI
pSB1K3-arsR-RFP with XbaI and PstI
pSB1C3 (linearized) with EcoRI-HF, PstI and DpnI (+ FastAP)
Ligation of the tree fragments and heat shock transformation resulted in small colonies, which were not correct according to a Colony PCR.
Week 13 (7/13/2015-7/19/2015)
Optimization of arsenic sensor for CFPS
The ligation of T7 promoter, arsR and pSB1C3 was repeated, and this time there were several colonies which showed the expected band after a colony PCR.
Several PCRs were performed to create fragments for Gibson Assembly:
Ars_3_fwd and Cm_rev on BBa_J33201
Cm_fwd and Ars_3_rev on BBa_J33201
Cm_fwd and UTR_noT7_rev on BBa_I746909
UTR_fwd and Cm_rev on BBa_I746909
All reactions worked very well and the products were extracted from the gel.
The products from PCR 1+2 and 3+4 were combined by Gibson Assembly.
Following heat shock transformation, plenty of colonies grew on the plates.
Some were checked by Colony PCR and all seemed to be correct.
3 overnight cultures per construct were inoculated and plasmids were isolated.
The isolated plasmids were used for further cloning steps:
A restriction digest was performed with the isolated pSB1C3-UTR-sfGFP (XbaI and PstI) and pSB1K3 (EcoRI-HF, PstI, DpnI, FastAP)
Ligation:
arsR-arsO2 + RFP + pSB1K3
arsR + UTR-sfGFP + pSB1K3
PCR:
UTR_arsO_rev and Cm_fwd on pSB1C3-pT7-arsR
UTR_fwd and Cm_rev on BBa_I746909
The products were extracted from the gel and combined via Gibson Assembly.
The Gibson Assembly and ligation mixes were transformed via heat shock.
Q5 PCR amplification of pSB1C3 backbone with cueR overlaps using split primers at 63°C
Cm_rev + pSB1C3_cueR_suf (expected Primerlength:1320 bp)
Cm_rev + RFP_fwd (expected Primerlength:1320 bp)
Cm_fwd + rcnR_fwd (expected Primerlength:835 bp)
Cm_fwd + merR_fwd (expected Primerlength:835 bp)
Cm_fwd + chrB_fwd (expected Primerlength:835 bp)
Amplification of the Cm_rev + pSB1C3_cueR_suf, pSB1C3_cueR_suf fragment was successfull.
Following gelextraction of the samples Cm_rev + pSB1C3_Cuer_suf and m_fwd + pSB1C3_cueR_pre
Phusion PCR amplification of pSB1C3 backbone with konstitutive promoter and pSB1C3 backbone with pbrR overlaps using split primers
Cm_rev + kProm_fwd (expected Primerlength:1320 bp) at 63°C
Amplification of the Cm_rev + kProm_fwd was not successfull.
Cm_fwd + pbrR_rev (expected Primerlength:835 bp) 68°C
Amplification of the Cm_fwd + pbrR_rev was successfull.
Following gelextraction of the samples Cm_rev + pSB1C3_Cuer_suf and m_fwd + pSB1C3_cueR_pre
Phusion PCR amplification with temperature gradient between 62°C-70°C:
Cm_rev + RFP_fwd at 63°C
Cm_fwd + pSB1C3_cueR_suf at 68°C
Cm_rev + kProm_fwd at 63°C
PCR worked, positive control worked, successfull amplification of pSB1C3-suf_RFP, pSB1C3-suf_CueR and pSB1C3-suf_kProm.
Following gelextraction of the samples Cm_rev + pSB1C3_Cuer_suf and m_fwd + pSB1C3_cueR_pre
Gibson Assembly with former generated fragments:
for assembly used both split primer backbone fragments with overlap and the synthesized promoter + RFP or constitutive  promoter + specific activator/ repressor
transformation into E. coli via heatshock
Guideline for Induction levels Trinkwasserschutzverordnung and WHO guidelines
no growth inhibition
Week 14 (7/20/2015-7/26/2015)
Optimization of arsenic sensor for CFPS
The colonies which had grown after Gibson assembly and transformation were checked with a colony PCR. The bands were as expected.
Overnight cultures were inoculted and the plasmids were isolated and handed in for sequencing.
The sequencing results for arsR-UTR-sfGFP were slighly unclear at one position, but likely the device is correct.
All other devices were clearly correct.
Colony PCR with Taq Polymerase of Gibson clones(Gibson Assembly from week 13) using VF and VR Primers:
Template: some bacteria resuspended in 50µL steril water and lysed by heat at 90°C for 10min
potential right clones were transferred to new LB agar plates with Chlorampohenicol
CrProm
copAP
CueR
rcnA
pbrA
pbrR
Plasmid DNA isolation using thermo fisher miniprep kit and analytic Jena kit
The colonies which had grown after Gibson assembly and transformation were checked with a colony PCR with Taq-Polymerase:
rcnR
rcnA
merR
chrB
pbrA
potential right clones (rcnA, merR, pbrA, rcnR) were transferred to new LB agar plates with Chlorampohenicol, Plasmid DNA was isolated with thermo fisher miniprep kit and handed in for sequencing
Repetition of growth inhibition test with higher heavy metal concentrations
lead: 2 g/L and 20g/L
mercury: 6 g/L and 60 g/L
chromium: 6 g/L and 60 g/L
nickel: 8 g/L and 80 g/L
copper: 20 g/L and 40 g/L
KOD Amplification of Gibsonfragments for merR and rcnR with temperature gradient between 64°C-68°C, sfGFP and RFP used as template
no desired fragments
KOD Amplification of Gibsonfragments for merR and rcnR with temperature gradient between 52°C-54°C, sfGFP and RFP used as template
The bands were as expected.
PCR clean-up of rcnR using M&N PCR clean-up and gel extraction kit
Gel extraction of merR using M&N PCR clean-up and gel extraction kit
Repetition of PCR for kProm fragment and RFP backbone fragment
afterwards PCR cleanup
Gibson Assamby for MerR and RcnR
afterwards heat shock transformation
The colonies which had grown after Gibson assembly and transformation were checked with a colony PCR. The bands of  one merR was as expected.
Week 15 (7/27/2015-8/2/2015)
Cloning of a his-tagged arsenic repressor
To get an arsenic repressor which can be purified and added to the CFPS, a his-tagged version under the control of the T7 promoter was cloned.
The first step were the following PCRs:
arsRhis_arsR_fwd and arsRhis_arsR2_r on pSB1C3-arsR ( BBa_J33201 )
arsRhis_Trp_fwd and arsRhis_Trp_rev on pSB1C3-tryptophanase
The products were extracted from the gel.
Gibson Assembly and heat shock transformation were performed.
A colony PCR indicated that the cloning was successful, which was confirmed by sequencing.
Colony PCR of RcnR clones
colony PCR indicated that the Gibson assembly was not successful
Biobrick assembly to generate a plasmid out of cueR and copAP
Restriction of konst. Prom. + cueR BBa_K1758320 with EcoRI and SpeI
Restriction of  CopAP_RFP BBa_K1758321 with EcoRI and XbaI
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
Biobrick assembly to generate a plasmid out of pbrA and pbrR
Restriction of konst. Prom. + pbrR BBa_K1758330 with EcoRI and SpeI
Restriction of pbrA + mRFP BBa_K1758331 with EcoRI and XbaI
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
Generation of kProm + rcnR BBa_K1758350 and rcnA +mRFP mRFP BBa_K1758351
Gibson assembly of rcnR- and rcnA-fragments
heat shock transformation
colony PCR of rcnR and rcnA clones
colony PCR indicated that the Gibson assembly of rcnR was not successful and rcnA was successful
KOD amplification of rcnR backbone with overlaps at 54°C
gel extraction of amplified rcnR backbone
3A assembly to generate a plasmid out of pbrA and pbrR
Restriction of konst. Prom. + pbrR BBa_K1758330 with EcoRI and SpeI
Restriction of pbrA + mRFP BBa_K1758331 with PstI and XbaI
ligation of former restrictions
colony PCR of ligation
colony PCR indicated that the ligation was not successful
3A assembly to generate a plasmid out of cueR and copAP
Restriction of konst. Prom. + cueR BBa_K1758320 with EcoRI and SpeI
Restriction of CopAP_RFP BBa_K1758321 with PstI and XbaI
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
3A assembly to generate a plasmid out of chrB and crProm
Restriction of konst. Prom. + chrB BBa_K1758310 with EcoRI and SpeI
Restriction of Chromium-Prom. + RFP BBa_K1758340 with PstI and XbaI
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
Week 16 (8/3/2015-8/9/2015)
Expression of arsenic repressor
An overnight culture was inoculated from the glycerol stock which had been prepared the previous week.
Two 200 mL cultures in 1 L shaking flasks were inoculated to an OD600 of 0.1. and cultivated at 37 °C and 180 rpm.
At an OD of 0.6, expression of the T7 polymerase was induced by adding rhamnose to a final concentration of 0.1 %.
The cultures were grown overnight at 18 °C and 130 rpm.
After 24 h, the bacteria were harvested by centrifugation (10 min, 5500xg).
The cells were lysed by sonification.
Purifiaction was carried out with the Macherey-Nagel Protino Ni-TED 1000 Packed Columns kit.
An SDS-PAGE was done to analyze the purification.
Measurement of the arsenic sensors´ background signal
To reduce the background signal of the arsenic sensor, a second operator site had been introduced downstream of arsR. Due to different information concerning the exact position of the operator, two versions had been prepared.
To find out if this had an effect, the following devices were tested:
pSB1C3-arsR
pSB1K3-arsR-RFP
pSB1K3-arsR-arsO1-RFP (first version of the second operator)
pSB1K3-arsR-arsO2-RFP (second version of the second operator)
Single colonies were obtained from the glycerol stocks by plating.
Five cultures per device were inoculated with single colonies.
After 15 h, the RFP fluorescence and the OD600 were measured with a Tecan Infinite M200.
The RFU was normalized by diving it by the OD and subtracting the singal of plain LB medium.
The results showed that the modifications had not reduced the background signal. Instead, it was even higher than that of the unmodified device.
3A assembly to generate a plasmid out of pbrA and pbrR
Restriction of konst. Prom. + pbrR BBa_K1758330
gel extraction with M&N gel extraction kit
Restriction of pbrA + mRFP BBa_K1758331 with PstI and XbaI
gel extraction with M&N gel extraction kit
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
3A assembly to generate a plasmid out of cueR and copAP
Restriction of konst. Prom. + cueR BBa_K1758320 with EcoRI and SpeI
gel extraction with M&N gel extraction kit
Restriction of CopAP_RFP BBa_K1758321 with PstI and XbaI
gel extraction with M&N gel extraction kit
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was successful, generation of CueR + CopAP + mRFP BBa_K1758322
3A assembly to generate a plasmid out of chrB and crProm
Restriction of konst. Prom. + chrB BBa_K1758310 with EcoRI and SpeI
Restriction of Chromium-Prom. + RFP BBa_K1758340 with PstI and XbaI
ligation of former restrictions
heat shock transformation
colony PCR of ligation
colony PCR indicated that the ligation was not successful
PCR amplification of pSB1K3 with KOD
gel extraction of the fragment for 3 A assembly
Gibson assembly of rcnR (out of E.coli K12) rcnR and rcnA with syntesised rcnA
colony PCR indicated that the Gibson assembly of rcnR was successful
colony PCR indicated that the Gibson assembly of rcnA was successful
This result was confirmed by sequencing.
Gibson assembly of konst. Prom. + chrB BBa_K1758310 ,  with syntesised chrB
heat shock transformation
colony PCR indicated that the Gibson assembly of chrB was not successful
Week 17 (8/10/2015-8/16/2015)
Colony PCR of multiple Transformations of Lead Ligation and konst. Prom. + chrB BBa_K1758310 ,  Gibson Assembly
Gibson-Assembly of konst. Prom. + chrB BBa_K1758310 (with synthesized chrB):
Amplifikation of ChrB overlap part for Gibson-Assembly.
RcnR Amplifikation for removing the PstI restriktionsite with Mutagenesis primer, used E.coli K12 as template.
Week 18 (8/17/2015-8/23/2015)
Optimization of arsenic repressor for CFPS
In order to thest whether it is possible to build a CFPS biosensor by co-expressing the repressor protein in the reaction, the his-tagged arsenic repressor plasmid was optimized.
A double terminator was inserted by 3A assembly. However, a test digestion showed unexpected bands.
Sequencing confirmed that the double terminator was in fact a different BioBrick.
Assembly of sfGFP with promoter (change RFP with UTR+sfGFP)
PCR-amplification of  copAP, CrProm, pbrA and  rcnA with KOD
Gel extraction of generated DNA-fragments
Gibson assembly of sfGFP with promoters
heat shock Transformation of Gibson-Assembly of copAP, CrProm, pbrA, rcnA
colony PCR
colony PCR indicated that all gibsons (copAP, CrProm, pbrA and  rcnA) were  successful
Sequencing confirmed that the UTR and sfGFP was inserted in all constructs.
Construction of konst. Prom. + chrB BBa_K1758310
Amplifikation of ChrB overlap part for Gibson-Assembly used chrB_fwd + Cm_rev
Gibson Assembly of ChrB
heat shock transformation
colony PCR
colony PCR indicated that all gibson chrB was successful
Sequencing confirmed the result.
Repetition of rcnR Gibson Assembly
Amplification of RcnR with KOD polymerase
PCR cleanup of KOD PCR of 17.8 using M&N cleanup kit
Gibson Assembly
heat shock transformation
colony PCR
colony PCR indicated that all gibson rcnR was  successful
Sequencing confirmed the result.
RcnR Amplifikation for removing the PstI restriktionsite with Mutagenesis primer
PCR clean up with M&N PCR clean up kit
Restruction of rcnR to check the success of the PCR
Creation of the merT-5-UTR-sfGFP BBa_K1758342 construct:
Amplification of sfGFP with merT overlap used  primermix of UTR-merT_fwd and Cm_rev (calculated annealing temperature 56°C and 59°C)
Amplification of sfGFP with merT overlap was successful
Amplification of pSB1C3 backbone with merT overlap used  primermix of pSB1C3-merT_rev and Cm_fwd (calculated annealing temperature 67°C and 63°C)
Amplification pSB1C3 backbone with merT overlap was successful
PCR cleanup of KOD PCR of 17.8 using M&N cleanup kit
Gibson Assembly
Heat shock transformation
Colony PCR of merT clones
Colony PCR indicated that the Gibson assembly of merT was successful
Week 19 (8/24/2015-8/30/2015)
Optimization of arsenic repressor for CFPS
The UTR was inserted between the T7 promoter and arsR-his for expression in a CFPS reaction.
A PCR was performed with BBa_K1758301 as template and the following primer combinations:
Cm_fwd and UTR_arsR_rev
UTR_arsR_fwd and < Cm_rev
The PCR did not work with Q5 and Phusion in HF buffer. However, it worked with Phusion in GC Buffer.
The fragments were extracted from the gel, combined by Gibson Assembly and transformed via heat shock.
Creation of the biosensor constructs ( kProm + ChrB+ChrP+UTR-sfGFP BBa_K1758313 , konst.Prom + CueR+CopAP-UTR-sfGFP BBa_K1758324 , kProm+PbrR+PbrA-UTR-sfGFP BBa_K1758333 , k Prom + merR+merT-UTR-sfGFP BBa_K1758343 and   kProm + rcnR+rcnA-UTR-sfGFP BBa_K1758353 )
Restriction of specific promoter –UTR-sfGFP (ChrP-5-UTR-sfGFP BBa_K1758312 , CopAP-5-UTR-sfGFP BBa_K1758323 ,PbrA-5-UTR-sfGFP BBa_K1758332 , merT-5-UTR-sfGFP BBa_K1758342 and rcnA-5-UTR-sfGFP BBa_K1758352 ) with EcoRI and XbaI
Restriction of specific constitutive promoter activator/ repressor (konst. Prom. + chrB BBa_K1758310 , konst. Prom. + cueR BBa_K1758320 , konst. Prom. + pbrR BBa_K1758330 , konst. Prom. + merR BBa_K1758340 and kProm + rcnR BBa_K1758350 ) with EcoRI and SpeI
Ligation of associated specific Promoters and activator/ repressor
Heat shock transformation of the ligations
Colony PCR of ligations
Colony PCR indicated that the ligation of konst.Prom + CueR+CopAP-UTR-sfGFP BBa_K1758324 , kProm+PbrR+PbrA-UTR-sfGFP BBa_K1758333 , k Prom + merR+merT-UTR-sfGFP BBa_K1758343 were successful
This result was confirmed by sequencing.
Colony PCR indicated that the ligation of ChrP-5-UTR-sfGFP BBa_K1758312 and rcnA-5-UTR-sfGFP BBa_K1758352 were not successful
Creation of an T7-promoter localized ahead the specic promoter
Restriction of specific promoter –UTR-sfGFP (pbrA-, copAP-, rcnA-, crProm-, merT- UTR-sfGFP) with PstI and XbaI
Restriction of T7-Plasmid (pSB1A3) with PstI and SpeI
Ligation of specific promoter and T7
Heat shock transformation of the ligations
Colony PCR of ligations
Colony PCR indicated that the ligation of merT+T7 and crProm+T7 were successful
This result was confirmed by sequencing.
Colony PCR indicated that the ligation of copAP+T7, rcnA+T7 and pbrA+T7  were not successful
Characterisation of kProm + merR+merT-UTR-sfGFP BBa_K1758343
Cultivation: 50 mL LB-media in 250 mL Erlenmeyer flask at 37°C and 180 rpm (over night culture were inoculated to an o.D. of 0,1)
Induction with different Lead-concentrations  at o.D. 0,6-0,8
Periodic measurement of o.D. and fluorescence (after induction)
Characterisation of kProm + pbrR+pbrA-UTR-sfGFP BBa_K1758333
Cultivation: 50 mL LB-media in 250 mL Erlenmeyer flask at 37°C and 180 rpm (over night culture were inoculated to an o.D. of 0,1)
Induction with different Lead-concentrations  at o.D. 0,6-0,8
Periodic  measurement of o.D. and fluorescence (after induction)
Characterisation of kProm + cueR+copAP-UTR-sfGFP BBa_K1758324
Cultivation: 50 mL LB-media in 250 mL Erlenmeyer flask at 37°C and 180 rpm (over night culture were inoculated to an o.D. of 0,1)
Induction with different Lead-concentrations  at o.D. 0,6-0,8
Periodic  measurement of o.D. and fluorescence (after induction)
Week 20 (8/31/2015-9/6/2015)
Characterization of arsenic sensor in vivo
Glycerol stocks of pSB1C3-arsR and pSB1K3-arsR-RFP were streaked on plates in order to obtain single colonies.
Three overnight cultures per construct were inoculated with single colonies.
On the following day, 50 mL cultures in LB with the correct antibiotic were inoculated to an OD600 of 0.1.
When the cultures had reached an OD of 0.6-0.7, arsenic solutions were added to the arsR-RFP cultures to final concentrations of 0 µg/L, 10 µg/L, 50 µg/L, 100 µg/L, 500 µg/L and 1000 µg/L.
Every 1.5-2 h, the OD600 and the RFP fluorescence were measured in a Tecan Reader.
The OD600 of LB medium was subtracted from the other OD values. The RFU was then divided by the corresponding OD and the average of the normalized RFU of the arsR cultures was subtracted from each value.
Creation of the biosensor constructs (constitutive promoter repressor/ activator + specific promoter UTR sfGFP)
Restriction of specific promoter –UTR-sfGFP (rcnA ( BBa_K1758352 ), crProm ( BBa_K175832 )) with EcoRI and XbaI
Restriction of specific constitutive promoter activator/ repressor (merR ( BBa_K1758340 ), rcnR ( BBa_K1758350 )) with EcoRI and SpeI
Ligation of associated specific Promoters and activator/ repressor
Heat shock transformation of the ligations
Colony PCR of ligations
Colony PCR indicated that the ligation of rcnA + rcnR, and crProm + ChrB  were successful
This result was confirmed by sequencing.
Creation of an T7-promoter localized ahead the specic promoter
Restriction of specific promoter –UTR-sfGFP (pbrA-, copAP-, rcnA- UTR-sfGFP) with PstI and XbaI
Restriction of T7-Plasmid (pSB1A3) with PstI and SpeI
Ligation of specific promoter and T7
Heat shock transformation of the ligations
Colony PCR of ligations
Colony PCR indicated that the ligation of copAP+T7, rcnA+T7 and pbrA+T7  (in pSB1A3) were successful
This result was confirmed by sequencing.
Characterisation of const.Prom + CueR+CopAP-UTR-sfGFP BBa_K1758324 ,   kProm+PbrR+PbrA-UTR-sfGFP BBa_K1758333 ,    kProm + rcnR+rcnA-UTR-sfGFP BBa_K1758353 and  kProm + ChrB+ChrP+UTR-sfGFP BBa_K1758313
Measurment of o.D. and fluorescence in BioLector of constructs induced with different heavy metal concentrations
Week 21 (9/7/2015-9/13/2015)
Backbone exchange ot specific promoters+T7
Restriction of T7-specific promoter –UTR-sfGFP (T7-rcnA, T7-crProm,T7-merT, T7-copAP,T7-pbrA) with EcoRI and PstI
Restriction pSB1C3-Vector with EcoRI and PstI
Ligation of T7-Promoter-construct + pSB1C3-vector
Heat shock transformation of the ligations
Colony PCR of ligations
Colony PCR indicated that the ligation of  T7-crProm BBa_K1758314 ,T7-merT BBa_K1758344 and  T7-copAP BBa_K1758325 were successful
This result was confirmed by sequencing.
