<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2015/b/b8/Bielefeld-CeBiTec_protocols_jumbotron.jpg)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px">Protocols</H1><P>(Never) change a running system!</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="text-center"><BUTTON id="button1" class="btn btn-success">Expand all</BUTTON><BUTTON id="button2" class="btn btn-danger">Hide all</BUTTON></DIV><H1>Standard protocols</H1><DIV class="panel-group scientific" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BluntEndLigation">Blunt end ligation</A></H4></DIV><DIV id="BluntEndLigation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> For &quot;intramolecular&quot; ligation of a PCR product the 5'-ends are phosphorylated with T4 Polynucleotidekinase (PNK)</LI><LI> Set up the following reaction:</LI><TABLE><TBODY><TR><TH>reagent</TH><TH>volume [ µL]</TH></TR><TR><TD>10x T4 DNA ligase buffer</TD><TD>2.5</TD></TR><TR><TD>T4 PNK </TD><TD>1</TD></TR><TR><TD>PEG 4000 50% (w/v)</TD><TD>2.5</TD></TR><TR><TD>DNA (100 to 600 ng)</TD><TD>x </TD></TR><TR><TD>ddH<SUB>2</SUB>O </TD><TD>to 25 </TD></TR></TBODY></TABLE><LI> Incubate at 37 °C, 30 min</LI><LI> Heat inactivate at 65 °C, 20 min</LI><LI> Add 1 µL T4 DNA ligase after reaction has cooled down to room temperature </LI><LI> Incubate at room temperature for at least 2 h, overnight also works. </LI><LI> Next step: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock">Transformation via heat shock</A></LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#ColonyPCR">Colony PCR</A></H4></DIV><DIV id="ColonyPCR" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>One Reaction mix contains</LI><UL><LI>5 µL 5x GoTaq buffer (Promega)</LI><LI>1 µL MgCl<SUB>2</SUB> (25 mM stock)</LI><LI>0.5 µL dNTPs (10 mM each) </LI><LI>0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM</LI><LI>17.625 µL ddH<SUB>2</SUB>O </LI><LI>0.125 µL GoTaq polymerase (Promega)</LI><LI>0.5 µL template </LI></UL><LI>PCR program </LI><UL><LI>Cell lysis and denaturation: 5 min, 95 °C</LI><LI>30 cycles</LI><UL><LI>Denaturation: 10 s, 95 °C </LI><LI>Hybridisation: 30 s, annealing temperature </LI><LI>Elongation: 60 s/kb of product, 72 °C </LI></UL><LI>Final elongation: 5 min, 72 °C </LI></UL><LI>Template alternatives </LI><UL><LI>Pick a colony with sterile tip, elute in 100 µL ddH<SUB>2</SUB>O or buffer, store at 4 °C during PCR, plate if insert is of correct size</LI><LI>Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size </LI></UL><LI>Gel electrophoresis for control of fragment size </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Dephosphorylation">Dephosphorylation</A></H4></DIV><DIV id="Dephosphorylation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Reaction mix</LI><UL><LI>30 µL purified vector DNA</LI><LI>4 µL 10x SAP buffer </LI><LI>2 µL SAP </LI><LI>4 µL H<SUB>2</SUB>O</LI></UL><LI>Protocol</LI><UL><LI>Incubate for 30 min at 37 °C </LI><LI>Inactivate for 15 min at 65 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#DNAPurification">DNA Purification</A></H4></DIV><DIV id="DNAPurification" class="panel-collapse collapse"><DIV class="panel-body">
              
We used the following kits for DNA purification. For each kit the specific protocol is described seperately:  <DIV class="panel-group inneraccordion" id="accordion2"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationA"><U>Thermo Scientific GeneJET Gel Extraction Kit   	</U></A></H4><DIV id="collapseDNAPurificationA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Note: all centrifugations should be carried out in a table-top microcentrifuge at &gt; 12000 x g/10.000-14.000 rpm</LI><LI>Preparation</LI><UL><LI>Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA band as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL tube. Record the weight of the gel slice. Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination. </LI><LI>Add 1:1 volume of Binding Buffer to the gel slice (volume:weigt)</LI><LI>Incubate the gel mixture at 50-60 °C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel slice is completely dissolved. Vortex the gel buffer mixture briefly before loading on the column. </LI><LI>Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow. </LI><LI><I>Optional</I>: Use this step only when DNA fragment is &lt;500 bp or &gt;10 kb long. 
</LI><LI>If the DNA fragment is &lt;500 bp, add 1:2 volume of 100 % isopropanol to the solubilized gel solution. Mix thoroughly.</LI><LI>If the DNA fragment is &gt;10 kb, add 1:2 volume of water to the solubilized gel solution. Mix thoroughly. </LI></UL></UL><LI>Binding of DNA</LI><UL><LI>Transfer up to 800 µL of the solubilized gel solution (from step 3/4) to the GeneJET purification column. Centrifuge 1 min at 11,000 x g. Discard the flow-through and place the column back into the same collection tube. Note: If the total volume exceeds 800 µL, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow through after each spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel slice per column. </LI><LI>Add 100 µL of Binding Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.</LI></UL><LI>Washing</LI><UL><LI>Add 700 µL of Wash Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. (We altered the protocol to to wash steps a 350 µL optional)</LI><LI>Centrifuge the empty GeneJet purification column for an additional 1 min to completely remove residual wash buffer </LI></UL><LI>Elution</LI><UL><LI>Transfer the GeneJET purification column into a clean 1.5 mL microcentrifuge tube. Add 50 µL of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 minute. Note: for low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended. If DNA fragment is &gt;10 kb, preheat Elution Buffer to 65 °C before applying to column. If the elution volume is 10 µL and DNA amount is &lt;5 µg, incubate column for 1 min at room temperature before centrifugation.(We altered the protocol to 30 µL Elution Buffer preheated to 60 °C and 1 min incubation before centrifuging for all sizes.) </LI><LI>Discard the GeneJet purification column and store the purified DNA at -20 °C. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationB"><U>Qiagen QIAquick Gel Extraction Kit (Centrifugation)   	</U></A></H4><DIV id="collapseDNAPurificationB" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Note: All centrifugation steps are carried out at 17.900 x g (13.000 rpm) in a conventional table-top microcentrifuge </LI><LI>Preparation</LI><UL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel </LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume gel (100 mg gel - 100 µL). The maximum amount of gel per spin column is 400 mg. For &gt;2 % agarose gels, add 6 volumes Buffer QG</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). Vortex the tube every 2-3 minutes to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow(similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.0, ad mix. The mixture turns yellow. </LI></UL><LI>Binding of DNA</LI><UL><LI>Add 1 volume isopropanol to the sample and mix.</LI><LI>Place a QIAquick spin column in a provided 2 mL collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min until all the samples have passed through the column. Discard the flow-through and place the QIAquick column back into the same tube. For sample volumes &gt;800 µL, load and spin again. </LI></UL><LI>Washing</LI><UL><LI>If DNA will subsequently be used for sequencing, </LI><LI>in vitro</LI> transcription or microinjection, add 500 µL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube <LI>To wash, add 750 µL Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. If the DNA will be used for salt-sensitive applications (sequencing, blut-ended-ligation) let the column stand 2-5 minutes after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 mL collection tube for 1 min to remove residual wash buffer </LI><LI>Place QIAquick column into a clean 1.5 mL microcentrifuge tube </LI></UL><LI>Elution</LI><UL><LI>To elute DNA, add 50 µL Buffer EB (10 mM TRIS-HCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µL Buffer EB to the center of the QIAquick column, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.</LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume of loading dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel. </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationC"><U>Analytik Jena InnuPREP Gel Extraction Kit   	</U></A></H4><DIV id="collapseDNAPurificationC" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Excise the DNA fragment from the agarose gel with a sharp scalpel. Note: minimize the agarose gel slice. Check the weight. Do not use more than 300 mg gel slice for one spin filter.</LI><LI>Transfer the gel slice into a 1.5 mL or 2.0 mL reaction tube and add the 650 µL Gel Solubilizer. </LI><LI>Incubate for 10 min at 50°C until the agarose gel slice is completely dissolved. </LI></UL><LI>Binding of DNA </LI><UL><LI>Add 50 µL Binding Optimizer and mix the suspension by vortexing or piepetting sometimes up and down. </LI><LI>Apply the sample onto the spin filter (green) located in a 2.0 mL Receiver Tube. Close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube.</LI></UL><LI>Washing</LI><UL><LI>Open the Spin Filter and add 700 µL Washing Solution LS, close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube. Repeat the entire step. </LI><LI>Centrifuge at max speed for 2 min to remove all traces of ethanol. Discard the 2.0 mL Receiver Tube.</LI></UL><LI>Elution</LI><UL><LI>Place the Spin Filter into a 1.5 mL elution Tube. Carefully open the cap of the Spin Filter and add 30-50 µL Elution Buffer (optionally preheated to 50°C). Incubate at room temperature for 1 min. Centrifuge at 6.000 x g (8.000 rpm) for 1 min. A second elution step will increase the yield of extracted DNA. Note: The DNA can be eluted with a lower or a higher volume of Elution Buffer. (...) Store the extracted DNA at +4 °C. For long time storage placing at -20°C us recommended </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#collapseDNAPurificationD"><U>Promega Wizard SV Gel and PCR CleanUp System   	</U></A></H4><DIV id="collapseDNAPurificationD" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Gel Slice/PCR product preparation</LI><UL><LI>Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 mL microcentrifuge tube.</LI><LI>Add 10 µL of Membrane Binding solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is copletely dissolved.</LI><LI>Alternative for PCR amplifications: Add an equal volume of Membran Binding Solution to the PCR amplification </LI></UL><LI>Binding of DNA </LI><UL><LI>Insert SV Minicolumn into Collection Tube.</LI><LI>Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 min. </LI><LI>Centrifuge at 16.000 x g for 1 minute. Discard flow through and reinsert Minicolumn into Collection Tube.</LI></UL><LI>Washing </LI><UL><LI>Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16.000 x g for 1 minute. Discard flow-through and reinsert Minicolumn into Collection Tube.</LI><LI>Repeat previous Step with 500 µL Membrane Washing Solution. Centrifuge at 16.000 x g for 5 minutes.</LI><LI>Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol. </LI></UL><LI>Elution</LI><UL><LI>Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 50 µL of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 minute. Centrifuge at 16.000 x g for 1 minute. </LI><LI>Discard Minicolumn and store DNA at 4 °C or -20°C. </LI></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#GibsonAssembly">Gibson Assembly</A></H4></DIV><DIV id="GibsonAssembly" class="panel-collapse collapse"><DIV class="panel-body"><LI>Modified from Gibson et al. (2009) </LI><LI>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5' exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.</LI><LI>Preparation of DNA molecules for <I>in vitro </I> recombination </LI><UL><LI> Generate the complenentary sequence overlaps by PCR using the Phusion DNA-polymerase. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products. </LI><LI>Seperate the PCR product of interest by gel electrophoresis, extract the products via <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#dnapurification" target="blank">Gel Extraction</A>. </LI></UL><LI><I>In vitro</I> recombination </LI><UL><LI>Preparing assembly mixture </LI><UL><LI>320 µL 5x isothermal reaction buffer </LI><LI>0.64 µL of 10 U ml<SUP>-1</SUP> T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U mL<SUP>-1</SUP> T5 exonuclease) </LI><LI>20 µL of 2 U mL<SUP>-1</SUP> Phusion DNA polymerase </LI><LI>160 µL of 40 U mL<SUP>-1</SUP> taq DNA ligase </LI><LI>add ddH<SUB>2</SUB>O up to a final volume of 1.2 mL </LI></UL><LI>Aliquote 15 µL of the assembly mixture and store at -20 °C </LI><LI>That 15 µL assembly mixture on ice until use </LI><LI>Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each fragment).</LI><LI>Incubate the mixture at 50 °C for 15 to 60 min with 60 min being optimal. </LI></UL><LI>Transformation via <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#heatshock" target="blank">heat shock</A> or <A href="https://2015.igem.org/team:bielefeld-cebitec/protocols#electrotransformation" target="blank">electroporation</A> without cleaning up the assembly product.</LI></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#PCR">PCR</A></H4></DIV><DIV id="PCR" class="panel-collapse collapse"><DIV class="panel-body">
We used the following polymerases. The master mix ingredients and standard protocols are described for each specific enzyme separately:
<DIV class="panel-group inneraccordion" id="accordion4"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion4" href="#collapsePCRA"><U>Phusion 	</U></A></H4><DIV id="collapsePCRA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Phusion</LI><UL><LI>Phusion master mix (25 µL)</LI><UL><LI>5 µL 5 x HF-buffer </LI><LI>0.5 µL 10 mM dNTPs </LI><LI>Up to 20 ng*µL <SUP>-1</SUP>template </LI><LI>0.25 µL Phusion DNA polymerase </LI><LI>0.25-0.5 µL primer 10 µM each</LI><LI>0.75 µL DMSO optional </LI><LI>Ad 25 µL dH<SUB>2</SUB>O </LI></UL><LI>Phusion standard protocol </LI><UL><LI>Initial denaturation: 98°C - 60 s </LI><LI>30-35 cycles of</LI><UL><LI>Denaturation: 98 °C - 20 s </LI><LI>Annealing: 50-70 °C - 20 s </LI><LI>Elongation: 72 °C - 15 s/kb </LI></UL><LI>Final elongation: 72 °C - 4 min </LI><LI>Storage: 6 °C </LI></UL></UL><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion4" href="#collapsePCRB"><U>Q5 	</U></A></H4><DIV id="collapsePCRB" class="panel-collapse collapse in"><DIV class="panel-body"><LI>Q5 </LI><UL><LI>Q5 master mix (25 µL)</LI><UL><LI>5 µL 5 x Q5 buffer </LI><LI>0.5 µL 10 mM dNTPs </LI><LI>Up to 50 ng*µL<SUP>-1</SUP>template </LI><LI>0.1 µL Q5 DNA polymerase </LI><LI>0.5-1 µL primer 10 µM each </LI><LI>Ad 25 µL dH<SUB>2</SUB>O </LI></UL><LI>Q5 standard protocol </LI><UL><LI>Initial denaturation: 98°C - 30 s </LI><LI>30-35 cycles of</LI><UL><LI>Denaturation: 98 °C - 20 s </LI><LI>Annealing: 50-75 °C - 30 s </LI><LI>Elongation: 72 °C - 20 s/kb </LI></UL><LI>Final elongation: 72 °C - 5 min </LI><LI>Storage: 4 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion4" href="#collapsePCRC"><U>KOD	</U></A></H4><DIV id="collapsePCRC" class="panel-collapse collapse in"><DIV class="panel-body"><LI>KOD</LI><UL><LI>KOD master mix (50 µL)</LI><UL><LI>5 µL 10 x KOD Hot Start DNA polymerase buffer </LI><LI>3 µL 25 mM MgSo<SUB>4</SUB></LI><LI>1 µL 10 mM dNTPs </LI><LI>1 µL KOD hot start DNA polymerase </LI><LI>1.5 µL primer 10 µM each </LI><LI>10 ng*µL<SUP>-1 template </SUP></LI><LI>Ad 50 µL dH<SUB>2</SUB>O</LI></UL><LI>KOD standard protocol </LI><UL><LI>Initial denaturation: 95°C - 120 s </LI><LI>30-35 cycles of</LI><UL><LI>Denaturation: 95 °C - 20 s </LI><LI>Annealing: 50-70 °C - 20 s </LI><LI>Elongation: 70 °C - individual elongation time </LI><LI>Calculation elongation time</LI><UL><LI>&lt;0.5 kb: 10 s/kb </LI><LI>0.5-1 kb: 15 s/kb </LI><LI>1-3 kb: 20 s/kb </LI><LI>&gt;3kb: 25 s/kb </LI></UL></UL><LI>Final elongation: 70 °C - 4 min </LI><LI>Storage: 6 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#GeneratingCompetentCells">Generating competent cells</A></H4></DIV><DIV id="GeneratingCompetentCells" class="panel-collapse collapse"><DIV class="panel-body">
We generated chemocompetent, ultra-chemocompetent and electrocompetent <I>E. coli</I> using the following protocols:
<DIV class="panel-group inneraccordion" id="accordion5"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisA"><U>Chemo competent cells 	</U></A></H4><DIV id="collapseCompisA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Cool 5 50 mL Falcons </LI><LI>Cool TSS buffer </LI><LI>Cool centrifuge </LI></UL><LI>Protocol</LI><UL><LI>Inoculate 2 x 3 mL LB with bacterial stock over night at 37 °C and 200 rpm </LI><LI>Inoculate 1 x 250 mL LB with the over night cultures in 1 L flask at 37 °C and 140 rpm </LI><LI>Incubate to OD<SUB>600</SUB> 0.4-0.6 </LI><LI>Devide the culture into 5 cooled 50 mL falcons and incubate on ice for 10 min </LI><LI>Onwards all steps at 4 °C </LI><LI>Centrifuge at 3000 rpm, 4 °C, 10 min </LI><LI>Discard supernatant </LI><LI>Pipette remaining media out carefully </LI><LI>Add 10% volume cold TSS buffer of original cultur, about 1-2 mL</LI><LI>Resuspend pellet</LI><LI>Aliquot 100 µL and drop in liquid nitrogen immediatly </LI><LI>Store at -80 °C</LI></UL><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisB"><U>Electro competent cells 	</U></A></H4><DIV id="collapseCompisB" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Autoclave 550 mL LB-Medium </LI><LI>Cool 1 L bidest. H<SUB>2</SUB>O </LI><LI>Cool 50 mL 10% glycerine </LI><LI>Cool 10 50 mL Falcons </LI></UL><LI>Protocol</LI><UL><LI>Inoculate 2 x 3 mL LB with bacterial stock, incubate over night at 37 °C and 200 rpm </LI><LI>Inoculate 2 x 250 mL LB with the over night culture in 1 L flask at 37 °C and 140 rpm </LI><LI>Cultivate until OD<SUB>600</SUB>0.4-0.6 </LI><LI>Store on ice for 15-30 min </LI><LI>Onwards all steps at 4 °C </LI><LI>Divide the cultures into the pre-cooled 50 mL Falcons and centrifuge at 4000 rpm, 4 °C, 15 min with slow acceleration and decceleration </LI><LI>Discard supernatant </LI><LI>Resuspend in 5 mL cooled bidest H<SUB>2</SUB>O shaking gently </LI><LI>Pool two suspensions, add bidest. H<SUB>2</SUB> to 50 mL and centrifuge again at 4000 rpm, 4°C, 15 min </LI><LI>Discard supernatant </LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O </LI><LI>Add bidest H<SUB>2</SUB>O up to 50 mL and centrifuge again at 4000 rpm, 4 °C, 15 min </LI><LI>Discard supernatant </LI><LI>Resuspend in 5 mL cooled 10% glycerine </LI><LI>Transfer suspensions in two 50 mL falcons and centrifuge again at 4000 rpm, 4 °C, 15 min </LI><LI>Discard supernatant </LI><LI>Add 1 volume similar to the pellet of 10% glycerine and resuspend </LI><LI>Aliquot in 50 µL and freeze in liquid nitrogen immediately </LI><LI>Store at -80 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion5" href="#collapseCompisC"><U>Ultra chemo competent cells 	</U></A></H4><DIV id="collapseCompisC" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Preparation</LI><UL><LI>Inoculate 10 mL LB medium in a 100 mL flask with 50 µL cells from glycerol culture, e.g. <I>E. coli </I> DH5a MCR </LI><LI>Incubate over night at 37 °C and 180 rpm </LI></UL><LI>Protocol</LI><UL><LI>Precool centrifuge and media </LI><LI>Inoculate 250 mL SOB-Medium in a 1 L flask with 2.5 mL of the preculture </LI><LI>Incubate at 19 °C  and 180 rpm up to OD<SUB>600</SUB> 0.5 (about 24-36 hours) </LI><LI>Cool flask 10 min in ice water </LI><LI>Onwards all steps at 4 °C </LI><LI>Devide the culture in 5 50 mL cooled falcons </LI><LI>Centrifuge 5 min at 4 °C and 5000 x g </LI><LI>Discard supernatant </LI><LI>Resuspend in each 6 mL icecold TB Buffer </LI><LI>Incubate 10 min on ice </LI><LI>Centrifuge 5 min at 4 °C and 5000 x g </LI><LI>Discard supernatant </LI><LI>Resuspend in each 6 mL icecold TB Buffer </LI><LI>Add 420 µL DMSO and vortex immediately </LI><LI>Aliquot 150 µL and freeze in liquid nitrogen immediately </LI><LI>Store at - 80 °C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#PlasmidIsolation">Plasmid Isolation</A></H4></DIV><DIV id="PlasmidIsolation" class="panel-collapse collapse"><DIV class="panel-body">
 For Plasmid Isolation we used the following kits. The methods are described seperately: 
<DIV class="panel-group inneraccordion" id="accordion3"><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationA"><U>PureYield<SUP>TM</SUP> Plasmid Miniprep System (Promega)   	</U></A></H4><DIV id="PlasmidIsolationA" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Prepare Lysate</LI><UL><LI>Add 600 µL of bacterial culture to a 1.5 mL microcentrifuge tube </LI><LI>Alternative for larger cultures: Centrifuge 1.5 mL bacterial culture for 30 s at maximum speed in a microcentrifuge. Discard the supernatant. Add an additional 1.5 mL bacterial culture to the same tube and repeat previous step. Add 600 µL of TE buffer or water to the pellet, and resuspend completely. </LI><LI>Add 100 µL of Cell Lysis Buffer (blue), and mix by inverting the tube 6 times.</LI><LI>Add 350 µL of cold (4-8 °C) Neutralization Solution and mix thoroughly by inverting.</LI><LI>Centrifuge at maximum speed in a microcentrifuge for 3 minutes. </LI><LI>Transfer the supernatant (about 900 µL) to a PureYield<SUP>TM</SUP> Minicolumn without disturbing the cell debris pellet </LI><LI>Place the minicolumn into a Collection Tube and centrifuge at maximum speed in a microcentrifuge for 15 s </LI><LI>Discard the flowthrough, and place the minicolumn into the same Collection Tube.</LI></UL><LI>Wash</LI><UL><LI>Add 200 µL of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 s </LI><LI>Add 400 µL of Column Wash Solution (CWC)to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds </LI></UL><LI>Elute</LI><UL><LI>Transfer the microcolumn to a clean 1.5 mL microcentrifuge tube, then add 30 µL of Elution Buffer or nuclease-free water directly to the minicolumn matrix. Let stand for 1 minute at room temperature </LI><LI>Centrifuge for 15 seconds to elute the plasmid DNA. Cap the microcentrifuge tube, and store eluted plasmid DNA at -20°C </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationB"><U>Thermo Scientific GeneJET Plasmid Miniprep Kit (Centrifugation)   	</U></A></H4><DIV id="PlasmidIsolationB" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>All purification steps should be carried out at room temperature.</LI><LI>All centrifugations should be carried out in a table-top microcentrifuge at &gt;12.000 x g (10.000-14.000 rpm)</LI><LI>Use 1-5 mL of <I>E. coli</I> culture in LB media for purification of high-copy plasmids.</LI><LI>Use up to 10 mL culture for low-copy plasmids</LI><LI>Protocol</LI><UL><LI>Resuspend the pelleted cells in 250 µL of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cells clumps remain.Note: Ensure RNase A has been added to the Resuspension Solution.</LI><LI>Add 250 µL of the Lysis Solution and mix throughly by inverting the tube 4-6 tines until the solution becomes viscous and slightly clear. Note: Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 4 min to avoid denaturation of supercoiled DNA.</LI><LI>Add 350 µL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. Note:It is important to mix throughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.</LI><LI>Centrifuge for 5 min to pellet cell debris and chromosomal DNA </LI><LI>Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate.</LI><LI>Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.</LI><LI>Add 500 µL of the Wash Solution (diluted with ethanol prior to use) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.</LI><LI>Repeat the wash procedure using 500 µL of the Wash Solution </LI><LI>Discard the flow-through and centrifuge an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.</LI><LI>Transfer the GeneJET spin column into a fresh 1.5 mL microcentrifuge tube. Add 50 µL of the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. incubate for 2 min at room temperature and centrifuge for 2 min. </LI><LI>Discard the column and store the purified plasmid DNA at -20°C.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationC"><U>Analytik Jena innuPREP Plasmid Mini Kit (Centrifugation)   	</U></A></H4><DIV id="PlasmidIsolationC" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>Starting material: 0.5-5.0 mL</LI><LI>Pellet cells </LI><UL><LI>Centrifuge: max speed, 1 min </LI><LI>Discard supernatant completely </LI></UL><LI>Resuspend cells </LI><UL><LI>Add 250 µL Resuspension Buffer</LI><LI>Vortex</LI></UL><LI>Lysis (do not vortex) </LI><UL><LI>Add 250 µL Lysis Buffer </LI><LI>Mix: invert tube 6-8 times</LI><LI>Lysis time: &lt;5 min</LI></UL><LI>Neutralization</LI><UL><LI>Add 350 µL Neutralization Buffer </LI><LI>Mix: invert tube 6-8 times</LI><LI>Centrifuge max speed, 8 min </LI></UL><LI>Binding of DNA </LI><UL><LI>Add Spin Filter to Receiver Tube </LI><LI>Add clarified sample Spin Filter </LI><LI>Centrifuge at 11.000 x g (12.000 rpm) 1 min </LI></UL><LI>Washing</LI><UL><LI>Add 500 µL Washing Solution A </LI><LI>Centrifuge 1 min at 11.000 g (12.000 rpm) </LI><LI>Add 700 µL Washing Solution B </LI><LI>Centrifuge 11.000 x g (12.000 rpm) 1 min </LI></UL><LI>Remove ethanol</LI><UL><LI>Discard filtrate </LI><LI>Add Spin Filter to Receiver Tube </LI><LI>Centrifuge Max speed, 2 min </LI></UL><LI>Elution</LI><UL><LI>Add Spin Filter to a 1.5 mL reaction tube </LI><LI>Add 50-100 µL Elution Buffer P </LI><LI>Incubation 1 min at room temperature </LI><LI>Centrifuge at 11.000 x g (12.000 rpm) 1 min </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion2" href="#PlasmidIsolationD"><U>QIAprep Spin Miniprep Kit (Qiagen)   	</U></A></H4><DIV id="PlasmidIsolationD" class="panel-collapse collapse in"><DIV class="panel-body"><UL><LI>All centrifugation steps are carried out at 13.000 rpm (17.900m x g) in a conventional table-top microcentrifuge </LI><LI>Pellet 1-5 mL bacterial overnight culture by centrifugation at &gt;8.000 rpm (6.800 x g) for 3 min at room temperature (15-25 °C) </LI><LI>Resuspend pelleted bacterial cells in 250 µL Buffer P1 and transfer to a microcentrifuge tube. </LI><LI>Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min . If using LyseBlue reagent, the solution will turn blue. </LI><LI>Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.</LI><LI>Centrifuge for 10 minutes at 13.000 rpm (17.900 x g) in a table-top microcentrifuge.</LI><LI>Apply the supernatant to the QIAprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through.</LI><LI>Wash the QIAprep spin column by adding 500 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through </LI><LI>Wash the QIAprep spin column by adding 750 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through. Transfer the QIAprep spin column to the collection tube.</LI><LI>Centrifuge for 1 min to remove residual wash buffer.</LI><LI>Place the QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 µL Buffer EB(10mM Tris-Cl, pH8.5) or water to the center of the QIAprep spin column, let it stand for 1 min, and centrifuge for 1 min.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#HisTagPurification">His tag purification</A></H4></DIV><DIV id="HisTagPurification" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Protocol (Macherey-Nagel Purification of His-tag proteins Kit) Protino® Ni-TED 1000 Packed Columns Kit </LI><LI>Preparation of cleared lysates under native conditions </LI><UL><LI>That the cell pellet from an <I>E. coli</I> expression culture on ice (if frozen). Resuspend 1 g of pelleted, wet cells in 2-5 mL LEW buffer, pipette up and down, or use stirring until complete resuspension without visible cell aggregates. Perform this step on ice </LI><LI>Add lysozyme to a final concentration of 1 mg/mL. Stir the solution on ice for 30 min </LI><LI>Sonicate the suspension on ice according to the instructions provided by the manufacturer (e.g. use 10 x 15 s bursts with a 15 s cooling period between each burst). Carefully check samples´ appearance after sonication. If the lysate is still viscous from incomplete fragmentation of DNA, add 5 µg/mL DNase I and stir on ice for 15 min.</LI><LI>Centrifuge the crude lysate at 10.000 x g for 30 min at 4 °C to remove cellular debris. Carefully transfer the supernatant  to a clean tube without disturbing the pellet. If the supernatant is not clear, centrifuge a second time or filter through a 0.45 µm membrane (e.g. cellulose acetate) to avoid clogging of the IMAC column with insoluble material. Store supernatant on ice.</LI></UL><LI>Purification protocol </LI><UL><LI>Column equilibration: Equilibrate Protino Ni-TED/IDA packed columns with 1 x LEW Buffer. Allow the column to drain by gravity (2.0 mL). Protino Ni-TED/IDA1000 packed columns are designed to fit into most 15 mL conical centrifuge tubes for convenient fraction collection</LI><LI>Binding: Add the cleared lysate to the pre-equilibrated column and allow the column to drain by gravity </LI><LI>Washing: Wash the column with 1 x LEW Buffer. Allow the column to drain by gravity (2 x 2.0 mL)</LI><LI>Elution: Wlute the polyhistidine-tagged protein in a new collection tube by adding 1 x Elution Buffer. Allow the column to drain by gravity (3 x 1.5 mL). Use protein assay and/or SDS-PAGE analysis to determine which fraction(s) contain(s) the majority of the polyhistidine-tagged protein. </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BioBrickAssembly">Standard BioBrick Assembly</A></H4></DIV><DIV id="BioBrickAssembly" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Modified from Silver Lab</LI><LI>Minimum size of the BioBricks for this methods is 150 bp, whereas it should be at least 500 bp bigger or smaller than the used backbone. The BioBricks, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other BioBrick called vector. The methods differ in prefix and suffix insertion. </LI><LI>Suffix insertion</LI><UL><LI>Digestion of inserted Biobrick </LI><UL><LI> Acquire &gt; 700 ng DNA (10 µL volume) </LI><LI> Add 1 µL 10 x NEBuffer </LI><LI> Add 0.5 µL XbaI </LI><LI> Add 0.5 µL PstI </LI><LI> Digest 1 h at 37 °C </LI><LI> Inactivate 20 min at 80 °C </LI><LI> Clean up via gel electrophoresis, cut and extract from gel </LI></UL><LI> Digestion of the vector </LI><UL><LI> Acquire about 700 ng DNA (10 µL volume) </LI><LI> Add 1 µL 10 x NEBuffer 2.1 </LI><LI> Add 0.5 µL SpeI </LI><LI> Add 0.5 µL  PstI </LI><LI> Digest 1 h at 37 °C </LI><LI> Inactivate 20 min at 80 °C</LI><LI> Add 1 µL Antarctic Phosphatase (AP) and 1.2 µL 10 x AP reaction buffer </LI><LI> Incubate 1 h at 37°C </LI><LI> Clean up with PCR clean up kit </LI></UL><LI> Ligation </LI><UL><LI> Mix 50-200 ng vector with 3-10 fold molar access of insert</LI><LI> Add 2 µL T4-Ligase-Buffer, 1 µL T4-DNA-Ligase </LI><LI> Incubate for a minimum 20-30 min at room temperature (we usually ligate over night) </LI><LI> Inactivate 5 min at 70 °C </LI><LI> Store at -20 °C or transform </LI></UL></UL><LI> Prefix insertion </LI><UL><LI> Follow protocol of Suffix insertion, replace the restriction enzymes as following: </LI><UL><LI> Digest insert with EcoRI and SpeI </LI><LI> Digest vector with EcoRI and XbaI </LI></UL></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#ElectroTransformation">Transformation via Electroporation</A></H4></DIV><DIV id="ElectroTransformation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Thaw 50 µL electrocompetent <I>E. coli</I> cells on ice, dilute with icecold 50 µL glycerine (10%) if necessary</LI><LI>Add 0.5-5 µL plasmid to 50 µL electrocompetent cells</LI><LI>Store cells on ice for 1 minute</LI><LI>Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω</LI><LI>Transfer transformation reaction to 450 µL SOC-Medium and incubate 1 h at 37 °C</LI><LI>Plate on selective LB-Medium</LI><LI>Incubate over night at 37 °C</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#HeatShock">Transformation via Heat Shock</A></H4></DIV><DIV id="HeatShock" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Thaw 100 µL chemo competent <I>E. coli</I> cells on ice </LI><LI>Add 0.5-5 µL plasmid to 100 µl chemocompetent cells </LI><LI>Store cells on ice for 10-30 min on ice </LI><LI>Heat shock for 90 seconds at 42 °C </LI><LI>Store reaction on ice for 60 seconds </LI><LI>Optional: Preheat SOC medium to 37 °C</LI><LI>Transfer reaction to 1 ml SOC medium and incubate at 37 °C for at least 1 hour</LI><LI>Centrifuge 3 minutes at 12000 rpm and plate on selective LB medium </LI><LI>Incubate at 37 °C over night </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#SDS-PAGE">SDS-PAGE</A></H4></DIV><DIV id="SDS-PAGE" class="panel-collapse collapse"><DIV class="panel-body"><LI><U> Pouring the polyacrylamide gel: </U></LI><UL><LI> For 2 <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#resolvingGel" target="_blank">resolving gel (14 %)</A>: </LI><UL><LI>4.7 mL  Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)</LI><LI>2.7 mL H<SUB>2</SUB>O</LI><LI>2.5 mL 0.5 M TRIS-HCl, pH 6.8</LI><LI>0.1 mL 10% SDS</LI></UL></UL><LI> Add 50 µL 10 % ammonium persulfate and 10 µL TEMED and mix. </LI><LI> Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel. </LI><LI> Layer isopropanol on top of the gel. </LI><LI> Leave the resolving gel at room temperature for 30-45 minutes to polymerize. </LI><LI> Remove isopropanol and wash the surface of the separating gel with H<SUB>2</SUB>O. Wait until the surface is dry.</LI><LI> For 2 <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media##stackingGel" target="_blank">stacking gel (4 %)</A>: </LI><UL><LI>1.3 mL Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)</LI><LI>6.1 mL H<SUB>2</SUB>O</LI><LI>2.5 mL 0.5 M TRIS-HCl, pH 6.8</LI><LI>0.1 mL 10% SDS</LI><LI>50 µL 10 % ammonium persulfate</LI><LI>5 µL TEMED</LI></UL><LI> Add 50 µL 10 % ammonium persulfate and 5 µL TEMED to each aliquote and mix. </LI><LI> Insert comb without getting bubbles stuck underneath.</LI><LI> Leave the gel at room temperature for 30-45 minutes to polymerize.</LI><LI> For storage:</LI><UL><LI> Remove sealing and store the gel wrapped in moistened paper towel at 4 °C. </LI></UL><LI><U> Preparing the sample: </U></LI><UL><LI> Mix your protein mixture 3:1 with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#PBJR" target="_blank">PBJR buffer</A> (15 µL protein solution + 5 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#PBJR" target="_blank">PBJR buffer</A>) </LI><LI> Heat for 5 minutes at 95 °C</LI></UL><LI><U> Running the gel: </U></LI><UL><LI> Remove sealing, put the polymerized gel into gel box and pour 1x <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#SDSrunning" target="_blank">SDS-PAGE running buffer</A> into the negative and positive electrode chamber. </LI><LI> Remove comp without destroying the gel pockets. </LI><LI> Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets.  </LI><LI> Pipet slowly 20 µL of the sample into the gel pockets. </LI><LI> Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample. </LI><LI> Connect the power lead and run the gel with 180 V until the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm. </LI></UL><LI><U> Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining): </U></LI><UL><LI> After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box. </LI><LI> Add 100 mL of the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#stainSDS" target="_blank">Colloidal Coomaassie Brilliant Blue staining solution</A> to your polyacrylamid gel. </LI><LI> Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation. </LI><LI> Remove the staining solution. </LI><LI> Wash the gel with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#collapseTen" target="_blank">destaining buffer</A>.</LI><LI> Incubate the gel in H<SUB>2</SUB>O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one.
       </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#MALDI-TOF-sampleprep">MALDI-TOF/TOF sample preparation</A></H4></DIV><DIV id="MALDI-TOF-sampleprep" class="panel-collapse collapse"><DIV class="panel-body"><LI><U> Tryptic digest of gel lanes for analysis with MALDI-TOF: </U></LI><UL><LI>Be careful that skin scales or hair do not contaminate your sample. So wear hand gloves and tie up your hair.</LI><LI>Reaction tubes have to be cleaned with 60 % (v/v) CH<SUB>3</SUB>CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively, microtiter plates from Greiner® (REF 650161) can be used without washing. If you work with reaction tubes from Eppendorf, you do not need the wash step either. </LI><LI>Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary, cut the parts to smaller slices</LI><LI> Gel slices should be washed two times. Therefore add 200 µL 30 % (v/v) acetonitrile in 0.1 M ammonium hydrogen carbonate each time and shake lightly for 20 minutes till the gel slices are destained. Remove supernatant and discard to special waste </LI><LI> Dry gel slices at least 30 minutes in a Speedvac. </LI><LI> Rehydrate gel slices in 15 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#trypsinsolution" target="_blank">trypsin solution</A> followed by short centrifugation. </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#trypsinsolution" target="_blank">Trypsin-solution</A>: 1 µL trypsin + 14 µL 10 mM NH<SUB>4</SUB>HCO<SUB>3</SUB></LI><UL><LI> For this solution solubilize lyophilized trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C. </LI></UL></UL><LI> Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight </LI><LI> Dry gel slices at least 60 minutes in a Speedvac. </LI><LI> According to the size of the gel slice, add 5 - 20 µL 50 % (v/v) ACN / 0.1 % (v/v) TFA </LI><LI> Samples can be used for MALDI measurement or stored at -20 °C </LI></UL><LI><U> Preparation and Spotting for analysis of peptides on Bruker AnchorChips: </U></LI><UL><LI> Spot 0.5 - 1 µL of sample aliquot </LI><LI> Add 1 µL <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#hccamatrix" target="_blank">HCCA matrix solution</A> to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area. </LI><LI> Dry the prepared spots at room temperature </LI><LI> Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#MALDIstandard" target="_blank">Peptide Calibration Standard II</A>”, Part number #222570), add 125 µL of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial.</LI><LI> Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots. </LI><LI> Analyze samples in ultrafleXtreme by Bruker Daltonics. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#BioLector ">Protocol used for BioLector measurment</A></H4></DIV><DIV id="BioLector" class="panel-collapse collapse"><DIV class="panel-body"><UL><H3>Preparation</H3><LI>Prepare overnight cultures of the strains you want to test with the BioLector.</LI><H3>Materials</H3><UL><LI>Growth media (LB)</LI><LI>Overnight cultures of tested strains</LI><LI>Gas Permeable Seal</LI><LI>48-well plates</LI></UL><H3>Procedure</H3><LI>Measure OD of the overnight cultures.</LI><LI>Centrifuge the overnight cultures and discard the supernatant.</LI><LI>Add LB-medium till an o.D. of 0,1.</LI><LI>Add 1 mL bacteria suspension in every well of the 48-wellplate.</LI><LI>Stick the Adhesive Gas Permeable Seal to the plate and make sure it is stuck on tightly.</LI><LI>Apply the Adhesive Seal for Evaporation Reduction on top of the Gas Permeable Seal.</LI><LI>Put the BioLector plate into the BioLector in the specified slot.</LI><LI>Set the temperature and moisture to be held during the measurement using the Start Assistant in the BioLector.determine the used filters for biomass measurement 620nm and for fluorescence measurement 486nm (ex) and 510 (em). 
</LI><LI>Run the BioLector for circa 41 hours.</LI><LI>Use the following settings:
<DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#C-Lysis">C-Lysis</A></H4></DIV></DIV></LI><LI>Preparation of lysis buffer </LI><UL><LI>2 volumes 98 % glycerol (v/v) </LI><LI>1 volume 0.44 M NaOH </LI><LI>1 volume 5 % SDS (w/v) </LI></UL><LI>Protocol</LI><UL><LI>Resuspend cells in 20 µL H<SUB>2</SUB>O </LI><LI>Add 5 µL Loading buffer </LI><LI>Seperate in agarose gelelectrophoresis with supercoiled marker </LI><LI>Elute from gel</LI></UL></UL></DIV></DIV></DIV><H1 id="CFPSprotocols">CFPS protocols</H1><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#AApreparation">Amino acid preparation</A></H4></DIV><DIV id="AApreparation" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> According to protocol from <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015b" target="_blank">Caschera and Noireaux 2015b</A>. Weigh all aminoacids seperatly into microcentrifuge tubes. Add 500 µl of 5 M KOH to each amino acid. Solubilization is achieved via multiple inverting and, if necessary, vortexing. Especially tyrosine takes a while, and is a suspension rather than a solution. Stock solutions are afterwards stored at -20 °C. Note: Caschera and Noireaux say these stock solutions can only be stored a few weeks, however, we did not see a loss in performance after more than 3 months (16 weeks). </LI><TABLE><TBODY><TR><TH>Molecular weight</TH><TH>mass to weigh (in mg) to obtain desired stock-solution </TH><TH>concentration in stock solution in mM</TH></TR><TR><TD>Alanine </TD><TD>89.09</TD><TD>182</TD><TD>4089</TD></TR><TR><TD>Arginine </TD><TD>174.20</TD><TD>202</TD><TD>2314</TD></TR><TR><TD>Asparagine-Monohydrate </TD><TD>150.14</TD><TD>282</TD><TD>3759</TD></TR><TR><TD>Aspartic acid </TD><TD>133.10</TD><TD>250</TD><TD>3752</TD></TR><TR><TD>Cysteine </TD><TD>121.16</TD><TD>149</TD><TD>2465</TD></TR><TR><TD>Glutamic acid </TD><TD>147.13</TD><TD>269</TD><TD>3655</TD></TR><TR><TD>Glutamine </TD><TD>146.15</TD><TD>183</TD><TD>2501</TD></TR><TR><TD>Glycine </TD><TD>75.07</TD><TD>158</TD><TD>4210</TD></TR><TR><TD>Histidine </TD><TD>155.15</TD><TD>255</TD><TD>3281</TD></TR><TR><TD>Isoleucine </TD><TD>131.18</TD><TD>247</TD><TD>3765</TD></TR><TR><TD>Leucine </TD><TD>131.18</TD><TD>167</TD><TD>2549</TD></TR><TR><TD>Lysine </TD><TD>146.19</TD><TD>175</TD><TD>2392</TD></TR><TR><TD>Methionine </TD><TD>149.21</TD><TD>186</TD><TD>2491</TD></TR><TR><TD>Phenylalanine </TD><TD>165.19</TD><TD>142</TD><TD>1716</TD></TR><TR><TD>Proline </TD><TD>115.13</TD><TD>224</TD><TD>3883</TD></TR><TR><TD>Serine </TD><TD>105.90</TD><TD>209</TD><TD>3953</TD></TR><TR><TD>Threonine </TD><TD>119.12</TD><TD>229</TD><TD>3853</TD></TR><TR><TD>Tryptophan </TD><TD>204.23</TD><TD>170</TD><TD>1661</TD></TR><TR><TD>Tyrosine </TD><TD>181.19</TD><TD>217</TD><TD>2396</TD></TR><TR><TD>Valine </TD><TD>117.15</TD><TD>152</TD><TD>2595</TD></TR></TBODY></TABLE><LI> Combine stock solutions to an amino acid mixture like depicted below. Add water to 4 mL and 110 µL of glacial acetic acid to adjust pH to about 6.5. You can add less glacial acetic acid and check the pH by pipetting some µL of the solution on pH paper. We observed slighty better performance for our reactions at pH = 7.9. Aliquot (do not forget to mix properly!) and flash-freeze in liquid nitrogen, store at -80 °C.</LI><TABLE><TBODY><TR><TH>Volume (in µL) for mixture</TH><TH>Concentration in mixture (without water and glacial acetic acid added) </TH><TH>concentration in final mixture</TH></TR><TR><TD>Alanine </TD><TD>13.6</TD><TD>146</TD><TD>13.56</TD></TR><TR><TD>Arginine </TD><TD>22.2</TD><TD>135</TD><TD>12.53</TD></TR><TR><TD>Asparagine-Monohydrate </TD><TD>13.6</TD><TD>134</TD><TD>12.44</TD></TR><TR><TD>Aspartic acid </TD><TD>13.6</TD><TD>134</TD><TD>12.44</TD></TR><TR><TD>Cysteine </TD><TD>22.2</TD><TD>143</TD><TD>13.28</TD></TR><TR><TD>Glutamic acid </TD><TD>13.6</TD><TD>130</TD><TD>12.07</TD></TR><TR><TD>Glutamine </TD><TD>22.2</TD><TD>145</TD><TD>13.46</TD></TR><TR><TD>Glycine </TD><TD>13.6</TD><TD>150</TD><TD>13.93</TD></TR><TR><TD>Histidine </TD><TD>13.6</TD><TD>117</TD><TD>10.86</TD></TR><TR><TD>Isoleucine </TD><TD>13.6</TD><TD>134</TD><TD>12.44</TD></TR><TR><TD>Leucine </TD><TD>22.2</TD><TD>148</TD><TD>13.74</TD></TR><TR><TD>Lysine </TD><TD>22.2</TD><TD>139</TD><TD>12.91</TD></TR><TR><TD>Methionine </TD><TD>22.2</TD><TD>145</TD><TD>13.46</TD></TR><TR><TD>Phenylalanine </TD><TD>34</TD><TD>153</TD><TD>14.21</TD></TR><TR><TD>Proline </TD><TD>13.6</TD><TD>138</TD><TD>12.81</TD></TR><TR><TD>Serine </TD><TD>13.6</TD><TD>141</TD><TD>13.09</TD></TR><TR><TD>Threonine </TD><TD>13.6</TD><TD>137</TD><TD>12.72</TD></TR><TR><TD>Tryptophan </TD><TD>34</TD><TD>148</TD><TD>13.74</TD></TR><TR><TD>Tyrosine </TD><TD>22.2</TD><TD>139</TD><TD>12.91</TD></TR><TR><TD>Valine </TD><TD>22.2</TD><TD>151</TD><TD>14.02</TD></TR><TR><TD><B>sum</B></TD><TD><B>381.6</B></TD><TD><B>2807</B></TD><TD><B>260.62</B></TD></TR></TBODY></TABLE></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#cofactorpremix">Cofactor premix</A></H4></DIV><DIV id="cofactorpremix" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media" target="_blank">Stock solutions</A> needed: </LI><UL><LI> 2 M HEPES</LI><LI> 174 mM NAD</LI><LI> 33.9 mM folinic acid</LI><LI> 65 mM coenzyme A (CoA)</LI><LI> 50 mg/mL <I>E. coli</I> tRNA (Roche)</LI><LI> 200 mM putrescine </LI><LI> 1.5 M spermidine </LI></UL><LI> Combine stock solutions to a create cofactor premix that is 20x final reaction concentration, depicted in the following list  </LI><UL><LI> 1 M HEPES</LI><LI> 6.6 mM NAD</LI><LI> 1.4 mM folinic acid</LI><LI> 5.4 mM coenzyme A (CoA)</LI><LI> 4 mg/mL <I>E. coli</I> tRNA (Roche)</LI><LI> 20 mM putrescine </LI><LI> 30 mM spermidine </LI></UL><LI> Aliquot and flash-freeze. During summer we prepared this premix more than one time, and we observed that the age of the cofactor premix does not affect the cell free reaction <B>as long as aliquots are stored at -80 °C</B>. </LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#cellharvest">Cell harvest</A></H4></DIV><DIV id="cellharvest" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> The following harvest protocol mainly orientates to the procedures in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013" target="_blank">Sun et al. 2013</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015" target="_blank">Kwon and Jewett 2015</A>.</LI><LI> Gather all materials needed: Harvest tubes and falcon tubes, washing buffer, DTT aliquot, liquid nitrogen, and a lot of ice. Start the harvest when <I>E. coli</I> culture reaches mid- to late exponential growth phase. For the <I>E. coli</I> we used in our experiments and cultivations, the mid- to late exponential growth phase was reached at an OD<SUB>600</SUB> of 3-4 (see growth curves in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook/CFPS" target="_blank">notebook section</A>).</LI><LI> harvest protocol - <B>keep everything on ice between the steps!</B></LI><OL><LI> Transfer culture into prechilled and weighted harvest tubes or falcons</LI><LI> Centrifugate: 5000x g, 4 °C, 15 min. While centrifugating, add DTT to S30 buffer to a final concentration of 2 mM</LI><LI> Discard supernatant and weigh pellets </LI><LI> Add about 10 mL of S30 washing buffer per 1 g of wet cells and resuspend cells by vortexing and vigourous shaking. For us, cycles of 15 s vortexing/shaking and 30 s resting on ice worked well.  </LI><LI> Centrifugate: 5000x g, 4 °C, 12 min</LI><LI> Discard supernatant</LI><LI> Repeat steps 4 to 6 two times </LI><LI> Centrifuge a last time at 5000x g, 4 °C, for 5 min and remove residual washing buffer by pipetting.</LI><LI> Flash freeze pellets in liquid nitrogen and store at -80 °C, unless you want to proceed with sonication directly. </LI></OL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Sonication">Sonication</A></H4></DIV><DIV id="Sonication" class="panel-collapse collapse"><DIV class="panel-body"><P> The following strategy can be pursued if there is no access to a sonifier which can display its energy output in Joule per second or alike. </P><P> First, prepare multiple reference samples which are absolutely identical (see below for further requirements). Right before sonication, ensure the temperature of the liquid. Sonicate for a defined time interval. When sonication has ended, measure the temperature of the liquid again <B>without any delay</B>. </P><DIV><FIGURE style="float: right; margin-left: 20px; width: 200px "><FIGCAPTION> Equation to calculate the necessary sonication time for a given setup. P: Power in W or J/s, respectively; V: sonicated volume in L; c: specific heat capacity of water (at room temperature); Δϑ: temperature difference between starting temperature and temperature after heating in K; Δt: time difference between both temperature measurements. </FIGCAPTION></FIGURE><P> Perform at least triplicates for every interval. To calculate the power P of your sonifier in Joule per seconds and thereby calculate the necessary sonication time for a given setting, use the nearby equation. You need to consider the following:</P><UL><LI>
The Amplitude of the sonifier must be kept constant during experiment to eliminate the influence of viscosity or heating on the result
</LI><LI> The starting temperature must remain the same in all reference measurements </LI><LI> The liquid as well as the vessel used in the reference experiment must be similar to the ones in the intended experiments </LI></UL><P> For our setting (see below), a sonication time of 290 s was appropriate for a total energy input of 800-900 J.  </P><P> Our lysis procedure is described in the following section </P></DIV><P><B> Sonication of the pellet from <A href="#cellharvest">Cell harvest</A> with the HF-Generator GM 2070 in combination with UW 2070, SH 70G and MS73; setted at 70% amplitude and no cycles </B></P><UL><LI> clean the sonifier tip with 70% (v/v) EtOH prior to usage  </LI><LI> resuspend the pellet in 1 mL S30 buffer per gramm pellet </LI><LI> aliquot the cell suspension à 1500 µL in 2 mL reaction tubes</LI><LI> for each sonication procedure, place one tube in an ice bath and place the sonifier tip in the suspension, so that it is briefly at 1/3 height of the tube</LI><OL><LI> sonicate for 10 s  </LI><LI> wait 10 seconds </LI><LI> repeat the previous two steps 28 times </LI></OL><LI> Centrifuge for 10 min at 12000x g and 4 °C. </LI><LI><B>Carefully</B> transfer the supernatant (cell extract) into a new tube. <B>Take care to not disturb the pellet</B>. </LI><LI> Consider to conduct a run-off reaction at 37 °C, slightly shaking at 250-300 rpm, in order to digest endogenous DNA and mRNA. When using the extract for the first time, perform various run-off reactions, e.g. 20 min to 80 min in 10 min intervals. For us, 30 min gave the best results. </LI><LI> Centrifuge a second time at 10000x g, 4 °C, 10 min after performing the run-off reaction.  </LI><LI> Transfer the supernatant (cell extract) into new tubes (we used 50-100 µL aliquots). Flash-freeze in liquid nitrogen and store at -80 °C. 
</LI></UL><P> An extensive analysis on sonication and run-off reaction parameters can be found in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#KwonJewett2015" target="_blank">Kwon and Jewett 2015</A>.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#CFPSreaction">Conduction of a CFPS reaction</A></H4></DIV><DIV id="CFPSreaction" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> To prevent RNase contamination, work with gloves all the time. It is strongly recommended to use filter tips or low-stick tips, respectively, and recently calibrated pipettes in order to work precisely with very low volumes. Do triplicates or quadrupates for a reliable data evaluation. </LI></UL><H3> Materials needed</H3><UL><LI> reagents</LI><UL><LI> cell extract </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#cofactorpremix" target="_blank">cofactor premix</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#AApreparation" target="_blank">amino acids</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#collapseNine" target="_blank">NTPs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#collapseNine" target="_blank">energy source and corresponding cofactors</A></LI><LI> DNA template </LI><LI> Mg- and K-glutamate solutions </LI><LI> RNase free water </LI><LI><I>any other additives</I></LI></UL><LI> other materials </LI><UL><LI> Depending on what protein is to be produced, a device capable of measuring fluorescence and/or luminescence. In our case this was a TECAN infinite M200 plate reader. </LI><LI> Depending on what device you use (for example plate reader), a 386 black well plate and transparent sealing foil </LI><LI> RNase free microcentrifuge tubes </LI><LI> 70% EtOH for RNase decontamination </LI></UL></UL><H3> Procedure </H3><OL><LI> Prepare a sheet <I>in silico</I> that contains all information you need for pipetting. For our purpose we created a suitable excel spreadsheet based on the one from <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al., 2013</A></LI><LI> Clean your working place thorough with 70% EtOH. Prepare RNase free micocentrifuge tubes. Let all aliquots thaw on ice. </LI><LI> Start to prepare your DNA samples by adding water, DNA, glutamate salts and any other item of choice to microcentrifuge tubes at room temperature. </LI><LI> Combine aliquots from 2. to a reaction buffer in a microentrifuge tube. Shortly vortex after the addition of each reagent and <B>keep on ice</B>. </LI><LI> At the latest after your reaction buffer is ready, prepare your measurement device (software program, temperature and so on). </LI><LI> Prepare a master mix made of reaction buffer and <I>E. coli</I> cell extract according to your spreadsheet, shortly vortex and keep on ice for now. <B>From now on, do not waste excessive time with dreaming but work eagerly so that, as long as your template is missing, no energy in the buffer is wasted by cell extract proteins!</B></LI><LI> Add the calculated master mix amount to each of your DNA sample after carefully mixing the master mix. <B>Attention: The master mix is viscous</B>, avoid bubbles by pipetting the last drop to the tube wall. </LI><LI> Shortly vortex each sample and centrifuge for 30 s at 10000 <I>g</I> at room temperature. Thereby any residual sample will be brought down, and bubbles can be reduced.</LI><LI> Run the reaction </LI><UL><LI> in the microcentrifuge tubes or</LI><LI> in a multi-well plate by transferring the samples into wells, sealing with appropriate foil to prevent evaporation. Optional: If you have a plate centrifuge, you can minimize bubbles in the plate with a short centrifugation step prior to measurement (4000 <I>g</I>, 30 s according to <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#Sun2013">Sun et al., 2013</A>)     </LI></UL></OL><UL><LI><B>Attention: Consider a volume loss during pipetting.</B> We routinely performed reactions at a final scale of 15 µL, but in our spreadsheet we calculated with 15.75 µL.</LI><LI> Run the reaction at the desired temperature for a desired time. In most of our experiments we measured for 3 h and saw no further raise in fluorescence signl. In most cases this is 37 °C but optimal temperature varies depending on protein product. Some proteins are well produced over a wide range of temperatures, which is the case for sfGFP. </LI><LI> Other considerations</LI><UL><LI> For best performance, a DNA plasmid of high purity and water of high quality are recommended. For sfGFP expression plasmids we found that plasmid mini-preparation kits, especially from Promega and Qiagen, provide sufficient DNA quality.  
                </LI><LI> Shaking can be helpful, especially when oxygen is needed for the reaction (see <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS#CascheraNoireaux2015a">Caschera and Noireaux 2015</A>). In our Tecan measurements we shaked 5 s with 1 mm amplitude every time before fluorescence was measured. </LI></UL></UL></DIV></DIV></DIV><H1 id="PRIAmain">PRIA protocols</H1><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#PRIA">Plasmid Repressor Interaction Assay (PRIA)</A></H4></DIV><DIV id="PRIA" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>After every step the reaction tube is centrifuged with 1000 <I>g</I> for 1 minute. </LI><LI>The supernatant after centrifugation and every wash step (starting from the step when the protein is added to the agarose) is stored for analyzing the samples for protein and DNA amounts. </LI><LI>As negative controls no protein to the agarose were added. </LI><LI>Steps: </LI><UL><LI>25 µL Ni-NTA agarose is put in a reaction tube. Then the sample is centrifuged. </LI><LI>The agarose is washed three times with Kpi buffer (Volume: 50 µL). </LI><LI>250 pmol protein in 20 µL Kpi buffer is added and incubated for 30 min. Then the sample is centrifuged.  </LI><LI>The agarose is incubated three times in Kpi Buffer (Volume: 50 µL).Then the sample is centrifuged.</LI><LI>0.75 pmol plasmid (lacO) is mixed with 20 µL binding buffer (Incubation time: 15 min). </LI><LI>Unbound DNA is washed away three times with binding buffer (Volume: 50 µL, Incubation time: 15 min). Then the sample is centrifuged.</LI><LI>3x elution with analytes in binding buffer (Volume: 50 µL, Incubation time: 15 min) </LI><LI>Imidazol solution (50 µl) is used to release proteins from the agarose. </LI><LI>Water (volume: 10 µL) is mixed with agarose.</LI><LI>The DNA amount of the supernatant after centrifugation is analyzed via gel electrophoresis.</LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#EMSA">Electrophoretic Mobility Shift Assay (EMSA)</A></H4></DIV><DIV id="EMSA" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>Required solutions:</LI><UL><LI>5x EMSA buffer (100 mM Na<SUB>2</SUB>HPO<SUB>4</SUB>, 375 mM KCl, 25 % Glycerol, adjust ph to 8.0)</LI><LI>EMSA running buffer (20 mM Na<SUB>2</SUB>HPO<SUB>4</SUB>, adjust ph to 8.0)</LI><LI>3 % agarose gel prepared with EMSA running buffer</LI><LI>MgCl<SUB>2</SUB>: 25 mM</LI><LI>EDTA: 2,5 mM</LI><LI>BSA: 10 mg/ml</LI><LI>Dithiotreitol</LI><LI>salmon sperm</LI><LI>Cy3 labeled DNA</LI><LI>protein</LI></UL><LI>Reaction:</LI><UL><LI>0.05 pmol Cy3 labeled DNA</LI><LI>5 pmol purified protein of interest</LI><LI>4 µl 5x EMSA buffer</LI><LI>2 µl MgCl<SUB>2</SUB> final concentration: 2.5 mM</LI><LI>0.8 µl EDTA final concentration: 0.1 mM</LI><LI>1 µl BSA final amount: 10 µg</LI><LI>1 µl salmon sperm final amount: 1 µg</LI><LI>optional: 5 mM DTT</LI></UL><LI>mix all components of the reaction carefully and add up with water to 20 µl H<SUB>2</SUB>O. </LI><LI>Incubate at room temperature in the dark for 15 min.</LI><LI>Keep in mind that Cy3 is light sensitive, always keep your samples in the dark after addition of the DNA fragment.</LI><LI>Apply reaction to 3 % agarose gel. </LI><LI>Run gel at 80 V for approximately 80 min at 4 °C. (Adjust voltage and duration to size of your DNA fragement.) </LI><LI>Detect Cy3 with a fluorescence scanner. In our case it was the Typhoon 8600 variable Mode Imager.</LI></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Immob">Immobilization of dsDNA on paper</A></H4></DIV><DIV id="Immob" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>Dissolve 10 mg <I>p</I>-Phenylenediisothiocyanate either in 30 ml pure ethanol or in 500 µL dried DMSO and add up to 30 ml with water. (the drying is essential, perform it with 4 Angström molecular sieves at least 24 h)</LI><LI> Whatman filter paper is activated for 1-2 days by orbital shaking in the PDITC solution.</LI><LI>Wash paper twice with DMSO or water respectively. Dry it for approximately 5 min</LI><LI>Generate an aminolabeled DNA fragement</LI><UL><LI>The optimal concentration required for the immobilization is 20 µM. You can achieve this through hybridization of 2 Primers in 50 mM sodium phosphate buffer, pH 8.5. Boil the primers at 98 °C for a few minutes and afterwrds slowly decrease the temperature.</LI><LI>Alternatively, you can perform an conventional PCR with one of the primers aminolabeled. We recommend labeling of the other oligo with a fluorescent dye. This enables detection of the immobbilized DNA on Scanners like the Typhoon or Ettan Dige.</LI><LI>Pipett 1-0.1 µL onto the paper and incubate it over night in an humidification chamber (avoid direct contact of the liquid with the paper)</LI><LI>Block the paper with 50 °C warm blocking solution (0.1 M Tris, 50 mM ethanolamine, pH 9.0)</LI><LI>wash the paper three times for 2 min with water and dry it by an airstream. (If you wish to immobilze ssDNA, add an washing step with 50°C 4xSSC buffer, 30 min.)</LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#PICO">PicoGreen Assay</A></H4></DIV><DIV id="PICO" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>PicoGreen Assay</LI><UL><LI>99 µl TE-buffer (10 mM Tris-HCl, 1mM EDTA, pH 7.5) are mixed with 1 µL sample solution in a well of a black 96 well plate</LI><LI>For the determination of the concentration, a standard row of the concentrations 1, 0.5, 0.25, 0.125 and 0.0625 and  0 µg/mL of lambda DNA is done.</LI><LI>PicoGreen solutions are diluted in a ratio 1/200 in TE-buffer. </LI><LI>100 µl diluted PicoGreen solution is mixed with the diluted sample.</LI><LI>The samples are measured in a Tecan Reader. They are excited at 480 nm and the fluorescence emission intensity is measured at 520 nm. </LI></UL></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#Tecan">Measurement of Fluorescence in plate reader</A></H4></DIV><DIV id="Tecan" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>You can detect immobilized DNA and your fusion protein by measuring the fluorescence. </LI><LI>You need Cy3- and amino-labeled DNA with an operator site for immobilization, a repressor fused to a fluorescence protein and cellulose on a black 96-well plate.</LI><UL><LI>Excitation for Cy3: 545 nm, Emission for Cy3: 590 nm</LI><LI>For example: Excitation wavelength for sfGFP: 480 nm, Emission wavelength for sfGFP: 515 nm</LI><LI>The gain needs to be adapted.</LI></UL><LI>After every step the supernatant is transferred into another 96-well plate for measurement. </LI><LI>The plate with the cellulose is dried after having taken out the supernatant and measured again. </LI></UL><UL><LI>Cellulose on the plate is prepared in a black 96-well plate.</LI><LI>DNA is immobilized on cellulose.</LI><LI>Protein in Kpi buffer (Concentration: 20 µg/mL, Volume: 25 µL) is added to the wells with immobilized DNA (Incubation time: 15 min).</LI><LI>The wells are washed with binding buffer (Volume: 200 µL, incubation time: 10 min).</LI><LI>100 µl analyte solution (e.g. 0.5 mM IPTG in binding buffer) is put in the well and shaken for 30 min. </LI></UL></P></DIV></DIV></DIV><H1>2015 InterLab protocols</H1><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#InterlabProtocol">Growing Cells for Measurement</A></H4></DIV><DIV id="InterlabProtocol" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>streak out 1 LB plate per device and control (for each biological replicate 1 plate)</LI><LI>incubate LB plates over night at 37°C (19 hours)</LI><LI>inoculate liquid cultures (100 mL flasks with 10 mL LB-medium)</LI><LI>incubate at 37 °C, 300 rpm, 17 hours</LI></UL></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#InterlabProtocol2">Plate Reader Measurement</A></H4></DIV><DIV id="InterlabProtocol2" class="panel-collapse collapse"><DIV class="panel-body"><P><UL><LI>measure the OD600 of the overnight cultures</LI><LI>dilute all samples to an OD600 of 0.5</LI><LI>re-measure the OD600, if it is within 5 % of 0.5, proceed</LI><LI>measure the samples with a plate reader (150 µl/ well, 96-well-plate, Excitation Wavelength
: 475 nm, Emission Wavelength: 509 nm)</LI></UL></P></DIV></DIV></DIV><DIV class="row footer"><DIV class="col-md-2 col-md-offset-10 text-center"><A type="button" class="btn btn-default btn-next" href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media"><P>Media &amp; Buffers</P></A></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>