<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Notebook_DRdrugs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Notebook/DRdrugs</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
         </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-8"><DIV class="text-center"><H1>Date Rape Drugs</H1><BUTTON id="button1" class="btn btn-success">Expand all</BUTTON><BUTTON id="button2" class="btn btn-danger">Hide all</BUTTON></DIV><DIV class="panel-group notebook" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseOne">Week 1 (4/20/2015-4/26/2015)</A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design</LI><UL><LI>Literature was screened for substances that are used as date rape drugs</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwo">Week 2 (4/27/2015-5/3/2015)</A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design</LI><UL><LI>GHB was identified as potential agent </LI><LI>Experts were contacted because of legal restriction of the handling of GHB </LI><LI>Further research showed potential identified GABA as analogon that is legal to handle and easy to work with </LI><LI>Research then suggested the use of the GABA inducible operon in <I>Bacillus subtilis </I></LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseThree">Week 3 (5/4/2015-5/10/2015)</A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design </LI><UL><LI>Local experts were contacted regarding the cultivation and induction of <I>B. subtilis </I></LI><LI>Primers were designed for several targets within the <I>gab</I> operon </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFour">Week 4 (5/11/2015-5/17/2015)</A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design</LI><UL><LI>Further screening of literature and genebanks showed another GABA inducable operon in <I>Rhizobium leguminasorum</I></LI><LI>For practical reasons regarding cultivation <I>B. subtilis </I> inducable operon <I>gab</I> was chosen to be obtained via pcr for cloning while <I>Rh. leguminasorum </I> was not </LI><LI>Parts from both operons were ordered as synthesized fragments </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFive">Week 5 (5/18/2015-5/24/2015)</A></H4></DIV><DIV id="collapseFive" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design</LI><UL><LI>Primers were designed for various experiments regarding both operons and parts of them </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSix">Week 6 (5/25/2015-5/31/2015)</A></H4></DIV><DIV id="collapseSix" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Pre-cultures of <I>B. subtilis </I></LI><UL><LI><I>Bacillus subtilis </I> substrain 168 contains the GABA inducable operon <I>gab</I> and is in stock in our laboratory. qRT-PCR is planned to determine a cross-reaction of this operon on date rape drugs. The operon is synthesized and therefor not obtained from these cultures </LI><LI><I>Bacillus subtilis </I> substrain 168 was obtained from glycerin culture stored at - 80°C in our lab</LI><LI>Cultivation on LB plates at 30 °C and 37 °C </LI><LI>12 hours growth showed more and bigger colonies on the plate incubated at 37 °C </LI><LI>Cultivation in 15 mL LB as well as 15 mL TY liquid medium at 37 °C / 250 rpm each in 100 mL flasks </LI><LI>OD<SUB>600</SUB> after 24 hours was significantly higher in LB which was therefor chosen for further cultivation </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSeven">Week 7 (6/1/2015-6/7/2015)</A></H4></DIV><DIV id="collapseSeven" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Cultivation of <I>B. subtilis </I></LI><UL><LI>Medium was composed of CgXII-N, salt solution and 30% glucose </LI><LI>Ammonium sulfate or GABA were added in various concentrations to determine the use as nitrogen source </LI><LI>Unfortunately, cultures did not or almost not grow </LI><LI>3 cultures grew after 72 hours incubation </LI><LI>Further experiment design to improve cultivation </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEight">Week 8 (6/8/2015-6/14/2015)</A></H4></DIV><DIV id="collapseEight" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Experiment design</LI><UL><LI>As important analyte GBL (y-Butyrolactone)was identified because of the lack of legal restrictions and therefor increasing use as date rape drugs </LI><LI>Investigations show inducable operon in <I>Agrobacterium tumefaciens</I></LI></UL><LI>Growth inhibition test: GBL - <I>B. subtilis</I> and <I>E. coli</I></LI><UL><LI>To determine growth inhibition by GBL growth in presence of several concentrations of GBL solution was tested </LI><LI>Both <I>B. subtilis</I> and <I>E.coli</I> were cultivated on LB plates </LI><LI>Filter paper circles were soaked with 3 microliters 1,3,5,10,50 and 100% GBL solutions and incubated </LI><LI>Both cultures grew almost uninfluenced. GBL concentrations of 50 and 100% show minor inhibition </LI></UL><LI>Cultivation of <I>B. subtilis</I> for qRT-PCR</LI><UL><LI>Because of earlier lack of sufficient growth, an optimal medium for this particular substrang was determined </LI><LI>Pre-cultures were grown in LB low salt, c-medium and CgXII</LI><LI>Tryptophane, ferric ammonium citrate as well as III´salts were filtrated sterile and freshly added </LI><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseNine">Week 9 (6/15/2015-6/21/2015)</A></H4></DIV><DIV id="collapseNine" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Insertion of GABA gBlocks into pSB1C3</LI><UL><LI>Two GABA inducible systems from <I>Bacillus subtilis</I> and <I>Rhizobium leguminosarum</I> were obtained as gBlocks from IDT.</LI><LI>To enable cloning into pSB1C3, we wanted to add suitable overhangs to the vector.</LI><LI>However, the first PCR did not yield product.</LI></UL><LI> Cultivation of <I>Agrobacterium tumefaciens</I> substrain C58.1 for determination of optimal growth conditions </LI><UL><LI> Retrieved from glycerin stock held at -80 °C </LI><LI> Cultivated on antibiotic free LB plates </LI><LI> After 72 h growth cultivated in liquid LB w/o MgSO<SUB>4</SUB> and with 160 mg/l MgSO<SUB>4</SUB> at 28 °C and 180 rpm </LI><LI> After 22 h growth higher OD<SUB>600</SUB> with MgSO<SUB>4</SUB> (12.9) than without (5.8) </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTen">Week 10 (6/22/2015-6/28/2015)</A></H4></DIV><DIV id="collapseTen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Insertion of GABA gBlocks into pSB1C3</LI><UL><LI>To determine the optimal conditions for the amplification of pSB1C3, we tested Q5, Phusion, GC enhancer and three annealing temperatures.</LI><LI>Q5 with GC enhancer and annealing at 69 °C worked best.</LI><LI>Using these conditions we were able to amplify the vectors for both gBlocks.</LI><LI>Following gel extraction, the vectors and the gBlocks were combined via Gibson assembly.</LI><LI>2 µL of each reaction were transformed into NEB 5-alpha Competent <I>E. coli</I> via heat shock.</LI><LI>On the following day, the colonies were screened with a colony PCR and three colonies per construct were choosen for overnight cultures.</LI><LI>Following plasmid isolation, sequencing samples were prepared.</LI></UL><LI>PCR on various elements of the operon and repressor in <I> Agrobacterium tumefaciens </I></LI><UL><LI>To test the primers for the sequences containing the repressor (BlcR), the promoter and operator (P<SUB>blc</SUB>), the operon without repressor (P<SUB>blc</SUB>-<I>BlcABC</I>) and for two vectors (pSB1C3, K516030), we performed Q5 <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A>  on <I>A. tu.</I> obtained from LB plate after heat destruction (95 °C, 10 min) as well as on <I>E.coli</I> containing the backbones </LI><LI>Primers were used as following:
</LI><LI>BlcR:<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR_fwd" target="_blank">BlcR_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR_rev" target="_blank">BlcR_rev</A></LI><LI>P<SUB>blc</SUB>:<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="_blank">pro_Blc_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_rev" target="_blank">pro_Blc_rev</A></LI><LI>pSB1C3 Vector with overhang for BlcR: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pSB1C3_BlcR_fwd" target="_blank">pSB1C3_BlcR_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pSB1C3_BlcR_rev" target="_blank">pSB1C3_BlcR_rev</A></LI><LI>K516030 Vector with overhang for P<SUB>blc</SUB>: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_fwd" target="_blank">K516030proB_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_rev" target="_blank">K516030proB_rev</A></LI><LI>P<SUB>blc</SUB>-BlcABC: <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="_blank">pro_Blc_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcC_rev" target="_blank">BlcC_rev</A></LI></UL><LI>For BlcR and P<SUB>blc</SUB> 25 s extention period and 62 °C annealing temperature worked well </LI><LI>For P<SUB>blc</SUB>-BlcABC 90 s extention period and 68 °C annealing temperature worked well </LI><LI>Gel electrophoresis on 1 % Agarose gel showed weak bands for P<SUB>blc</SUB> and BlcR so PCR was repeated the next day and a 2 % Agarose gel was chosen for analysis of BlcP template</LI><LI>Gel electrophoresis showed clear bands and BlcR PCR product was cut out from gel using a non exposure technique to prevent mutation caused by ethidium bromide or UV radiation and stored at 4 °C</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEleven">Week 11 (6/29/2015-7/5/2015)</A></H4></DIV><DIV id="collapseEleven" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>BlcR: Transformation into <I>Escherichia coli</I></LI><UL><LI>To obtain the sequence coding for the repressor we performed PCR, ligated framents into vectors via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank"> Gibson Assembly</A>  and transformed into <I>E. coli</I> via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> and verificated the insertion with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A></LI><LI>PCR on vector pSB1C3 was repeated as described before with primers containing overhangs for blcR</LI><LI>PCR product was <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">isolated</A>  (Analytik Jena: Innu Prep gel extraction kit) </LI><LI>BlcR fragment was inserted into pSB1C3 vector using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A>  and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> transformed into chemo competent <I> Escherichia coli </I></LI><LI>Cells were cultivated on chloramphenicol containing LB plates </LI><LI>5 colonies were tested for insert using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A> with verification primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#VF" target="blank">VF</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#VR" target="_blank">VR</A> with annealing temperature of 60 °C and 45 s denaturation and annealing period </LI><LI>4 colonies were found positive for insert of the vector </LI><LI>The colonies were grown in LB containing chloramphenicol for following <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmid isolation</A></LI><LI>Plasmids were purified to a concentration of 105-181 ng/microliter and sequenced </LI></UL><LI>Amplification and extraction of promoter and operator region of <I>Agrobacterium tumefaciens</I></LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">Q5 PCR</A>  using Primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="_blank">pro_Blc_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_rev" target="_blank">pro_Blc_rev</A> was performed on <I>A.th.</I>obtained from LB plate after heat destruction (95 °C, 10 min) with parameters as described before </LI><LI>3 % Agarose gel showed sharp bands that were cut out </LI><LI>K516030 backbone for insertion of P<SUB>blc</SUB> (promoter and operator sequence) was amplified via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A>  using the primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_fwd" target="_blank">K516030proB_fwd</A>&amp;<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_rev" target="_blank">K516030proB_rev</A> on K516030 carrying <I>E. coli</I> obtained from LB plate </LI><LI>negative control showed contamination, but sharp bands were obtained </LI><LI>PCR was repeated using the same primers but purified plasmid from stock as template </LI><LI>Bands were cut out and purified using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">gel extraction</A>  kit (Analytik Jena: innuPrep gel extraction kit)</LI><LI>P<SUB>blc</SUB> was inserted in K516030 vector using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A>  and transformed into chemo competent <I>E.coli</I> via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> , that were grown on LB plates containing chloramphenicol at 37 °C </LI><LI>Unfortunately, no growing colonies were obtained </LI></UL><LI>3A assembly of GABA sensitive elements and RFP</LI><UL><LI>Restriction digest of GABA gBlocks in pSB1C3 with EcoRI and SpeI and RFP with XbaI and PstI</LI><LI>Ligation of the fragments and pSB1K3 for 2 h</LI><LI>Heat shock transformation</LI><LI>Large colonies grew only on the plates with the GABA sensor from Rhizobium</LI><LI>A colony PCR showed that these large colonies were positive, while smaller colonies were contaminations</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwelve">Week 12 (7/6/2015-7/13/2015)</A></H4></DIV><DIV id="collapseTwelve" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Transformation of BlcR in <I>E. coli</I> and sequencing </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">ColonyPCR</A> was performed on colonies grown on LB plates containing canamycin for selection </LI><LI>To detect inserts verification primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#VR" target="_blank">VR</A>  and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#VF" target="_blank">VF</A> were used </LI><LI>Three positive colonies were cultivated, plasmids were <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">isolated</A>,purified for sequencing and glycerol stocks were prepared </LI><LI>Sequences of BlcR in pSB1C3 Vector did not show mutations, but an illegal restriction site. Primers for mutation were designed </LI></UL><LI>Repeat of amplification on P<SUB>blc</SUB> and assembly </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">ColonyPCR</A> was performed using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="blank">pro_Blc_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_rev" target="blank">pro_Blc_rev</A> Primer and K516030 containing <I>E. coli </I> from LB plate cultivation</LI><LI>Analysis in 3% agarose gel electrophoresis showed 5 fragments of expected size that were cut out </LI><LI>DNA was <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">eluted </A>from gel using Analytik Jena innuPrep gel extraction kit and stored</LI><LI>K516030 backbone was amplificated via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> using Primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_rev" target="blank">K516030proB_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_fwd" target="blank">K516030proB_fwd</A> with overhang for P<SUB>blc</SUB></LI><LI>Sharp bands of approximately 3500 bp were obtained while expected fragments should be of 2964 bp size </LI><LI>Repeat showed same results, probably because of false priming </LI><LI>Repeat of the PCR with purified plasmid as template showed bands of expected size</LI><LI>DNA was <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">purified</A> using Thermo Scientific GeneJET gel extraction kit </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed with P<SUB>blc</SUB> and K516030 backbone, purified for sequencing and transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> in chemo competent <I>E. coli</I></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A> identified 3 positive insertions, these clones were cultivated </LI></UL><LI>Assembly of P<SUB>blc</SUB> with constitutive promoter </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#BioBrickAssembly" target="blank">3 A BioBrick Assembly</A> was performed using the constitutive promoter K608002 upstream, P<SUB>blc</SUB> clone 4 downstream and pSB1K3 as destination plasmid to construct a constitutive expression of RFP with the operator site in P<SUB>blc</SUB> as target for the GBL responsive BlcR </LI><LI>The product was transformed into chemo competent <I>E. coli</I> via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A></LI><LI>Transformed cells were cultivated in liquid LB and analysed for fluoreszence. As control a P<SUB>blc</SUB> (clone 7) was analyzed. The construct with K608002 in front of P<SUB>blc</SUB> showed weaker fluoreszence </LI><LI>Unfortunately, sequencing for clone 4, that was used in the assembly, came back negative for insertion from sequencing </LI></UL><LI>Amplification of BlcABC operon and its vector</LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed on <I>A. tumifaciens</I> cultivated on LB plates using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcC_rev" target="blank">BlcC_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="blank">pro_Blc_fwd</A> primers </LI><LI>Results of gel electrophoresis showed expexted size of approximately 3566 kb </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed on pSB1C3 purified linear vector with primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K516030proB_rev" target="blank">K516030proB_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pSB1C3blcOnewfw" target="blank">pSB1C3blcOnewfw</A>, but was not successful. </LI></UL><LI>3A assembly of GABA sensitive elements and RFP</LI><UL><LI>The ligation was repeated over night</LI><LI>Following heat shock transformation, several  colonies grew on the plastes.</LI><LI>A colony PCR showed that most of them were correct.</LI><LI>Overnight cultures were inoculated and plasmids were isolated the following day.</LI><LI>Sequencing showed that both fragments were present in the vector, but did not cover the complete sequence.</LI></UL><LI>First test with GABA sensors</LI><UL><LI>5 mL cultures were inoculated with <I>E. coli</I> harboring the GABA sensor plasmids (1x <I>Rhizobium</I>, 2x <I>B. subtilis</I>).</LI><LI>For each clone, there was one negative control and one culture with 10 mg GABA.</LI><LI>After growth over night at 37 °C, the cultures were transfered into tubes and centrifuged for 5 min.</LI><LI>The pelletes of all negative controls did not appear red.</LI><LI>The pellets of the cultures with the <I>B. subtilis</I> sensor and GABA were clearly red.</LI><LI>The sensor from <I>Rhizobium</I> did not seem to work.</LI><LI>A test with filters in front of a smartphone camera showed a signal from all pellets. Either the filters were not effective enough or the sesnsors produce a significant background signal.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseThirteen">Week 13 (7/13/2015-7/19/2015)</A></H4></DIV><DIV id="collapseThirteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Amplification of BlcABC operon and its vector</LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> to obtain the vector with overhang for BlcABC was repeated as before using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcC_rev" target="blank">BlcC_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pro_Blc_fwd" target="blank">pro_Blc_fwd</A> primers</LI><LI>Bands were cut out and stored</LI></UL><LI>Mutagenesis of BlcR</LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed for the vector with overhang for the mutation of the illegal restriction site in the sequence coding for the repressor BlcR using primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_2_fwd" target="blank">Ars_2_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR_rev" target="blank">BlcR_rev</A> as well as <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_1_rev" target="blank">Ars_1_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#pSB_BlcRfwd_new" target="blank">pSB_BlcRfwd_new</A></LI><LI>Electrophoresis showed bands in expected sizes of 1689 bp and 1318 bp that were cut </LI><LI>DNA was purified using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">PCR CleanUp</A></LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFourteen">Week 14 (7/20/2015-7/26/2015)</A></H4></DIV><DIV id="collapseFourteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of mutated BlcR</LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed on the fragments of the split primer PCR last week </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">Heat shock transformation</A> into chemo competent <I>E. coli</I> was performed and insertion was verified by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A></LI><LI>Positive clones were cultivated and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated </A> and sequenced </LI></UL><LI>Assembly of BlcABC </LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed with BlcABC and pSB1C3 with overhang </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">Heat shock transformation</A> into chemo competent <I>E. coli</I> was performed and insertion was verified with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A></LI></UL><LI>Assembly of P<SUB>blc</SUB> clones 7 &amp; 9</LI><UL><LI>Since sequencing of clone 4 was negative for insertion, experiments are repeated with the positive clones 7&amp;9</LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed with both clones each with vector pSB1K3, products were transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A>, cultivated on LB plate containing kanamycin </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">ColonyPCR</A> identified no positive clones, repeat next week </LI></UL><LI>Measurement of the induction of the GABA sensor with GABA and GBL</LI><UL><LI>A 5 mL overnight culture of <I>E. coli</I> KRX harboring the GABA sensor out of <I>B. subtilis</I> was prepared in LB with kanamycin.</LI><LI>5 % GBL were added to a second culture to test if this has any effect on <I>E. coli</I></LI><LI>GABA solutions with different concentrations were prepared.</LI><LI>On the following day it was observed that the culture with GBL had not grown.</LI><LI>Several 5 mL cultures were inoculated from the overnight culture without GBL to an OD<SUB>600</SUB> of 0.1.</LI><LI>At OD<SUB>600</SUB> 0.4-0.5 100 µL of the GABA solutions were added. The final concentrations were 10 mg/L, 1 mg/L, 100 µg/L, 10 µg/L and 1 µg/L.</LI><LI>Culturing at 37 °C.</LI><LI>After 4.5 h, 50 µL per culture were transferred into a black 384 well plate and measured with a Tecan reader.</LI><LI>A culture with the plasmid encoding the tryptophanase instead of any fluorescent protein was measured as negative control.</LI><LI>There was no noteworthy difference between the RFP fluorescence of the GABA cultures. However, their signal was stronger than that of the tryptophanase.</LI><LI>The fluorescence was measured again after 24 h. This time, the cultures were diluted to an OD<SUB>600</SUB> of 2 beforehand to account for the differences in growth.</LI><LI>A slight induction seemed to be observable at the two highest tested GABA concentrations.</LI><LI>The background signal was very high. Apart from leakiness of the system, we reasoned that this might also be due to traces of GABA in the LB medium. Therefore, we decided to repeat the measurement with M9.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseFifteen">Week 15 (7/27/2015-8/2/2015)</A></H4></DIV><DIV id="collapseFifteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of P<SUB>blc</SUB></LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">Heat shock transformation</A> in chemo competent <I>E. coli</I> was repeated as last week: P<SUB>blc</SUB> in pSB1K3 vector</LI><LI>Lacking any success, we switched strategy to obtain K608002-P<SUB>blc</SUB>-RFP construct and performed an assembly of K608002 in pSB1C3, cut with PstI and SpeI, P<SUB>blc</SUB> cut with XbaI and PstI</LI><LI>The construct was transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> and cultivated on LB-kanamycin plate </LI><LI>4 clones were cultivated in liquid LB, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmid isolation</A> followed</LI><LI>Verification via restriction control with NotHI showed fragments of expected size, while sequencing revealed no insertion of K608002 in tested clones 1 and 4 </LI></UL><LI>Isolation of BlcR_new (without illegal restriction site)</LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">Plasmid Isolation</A> was performed on 3 positive clones with Analytik Jena Innu Prep Plasmid Mini Kit </LI></UL><LI>Control and repeat of assembly of BlcABC operon </LI><UL><LI>Performed <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A> revealed no insertion </LI><LI>Repeat of <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> with a molar ratio of 1:1 </LI><LI>Following <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock transformation</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">ColonyPCR</A> was negative for insertion again </LI><LI>Third repeat showed insertion in several colonies, 3 of which were cultivated, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated</A> and sequenced. BlcR was shown to have been succesfully mutated. No illegal restriction site was found </LI></UL><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSixteen">Week 16 (8/3/2015-8/9/2015)</A></H4></DIV><DIV id="collapseSixteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of BlcABC </LI><UL><LI>Repeat of <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">GibsonAssembly</A> with both parts of BlcABC and the vector with fitting overhang, followed by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock transformation</A> into chemo competent <I>E. coli</I></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">ColonyPCR</A> did not show the insert, though it might not have been visible in the agarose gel electrophoresis </LI><LI>Alternatively a <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">C-lysis</A> was performed and identified 6 possibly positive clones. three of which were cultivated, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated</A> and restricted with EcoRI. Bands obtained were of the expexted size. Plasmids probably containing BlcABC in pSB1C3 were sequenced </LI></UL><LI>Assembly of P<SUB>blc</SUB></LI><UL><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#BioBrickAssembly" target="blank">3 A Assembly</A> was performed using purified K608002 as upstream part, P<SUB>blc</SUB> clone 9 in front of RFP from plasmid isolation (week 14) as downstream part and pSB1K3 as destination plasmid </LI><LI>Following <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> no colonies grew on selection plate with LB and kanamycin. </LI><LI>Regarding these failing attempts of assembly, P<SUB>blc</SUB> clone 7 and two pSB1K3 plasmids were restricted with EcoRI and PstI, purified with PCR Purification Kit (Analytik Jena) and vectors were <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#Dephosphorylation" target="blank">dephosphorylated</A>. P<SUB>blc</SUB> was extracted from gel (InnuPrep geneJET gel extraction kit) </LI><LI>Following ligation, the product was transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank"> heat shock</A>  into chemo competent <I>E. coli</I> and cultivated on LB-kanamycin selection plate </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">Plasmid isolation</A> was performed after cultivation of 3 clones in liquid LB + kanamycin. Plasmids were purified and sequenced </LI></UL><LI>Measurement of the GABA sensor induction in M9</LI><UL><LI>Single colonies of the GABA sensor <I>E. coli</I> were obtained from the glycerol stock by plating.</LI><LI>Three colonies were used to inoculate 5 mL cultures in M9 with kanamycin.</LI><LI>The precultures had to be cultured for 48 h at 37 °C to reach a suitable OD.</LI><LI>The cultures were diluted to an OD<SUB>600</SUB> of 0.8. 300 µL were then added to 4.6 mL M9 with kanamycin and 100 µL GABA or GBL solution.</LI><LI>The measurement was carried out on the following day. RFP fluorescence of 50 µL culture was measured at 570 nm and 607 nm in a black 384 well plate. The OD<SUB>600</SUB> was measured with 200 µL in a transparent 96 well plate.</LI><LI>The values were normalized as follows: The OD<SUB>600</SUB> of M9 medium was subtracted from the OD<SUB>600</SUB> of the cultures. The relative fluorescence was divided by the optical density of the culture. Finally, the normalized fluorescence of a culture without plasmid-encoded fluorescence proteins (instead it harboured the his-tagged arsR plasmid) was subtracted from the other values. The mean of two cultures was used for this autofluorescence correction, because one culture did not grow. All other values were measured in triplicate. The error bars represent the standard deviation.</LI><LI>As can be seen, the background signal in M9 was much lower than in LB. Furthermore, a strong induction by GABA can be observed. The sensor did not respond to GBL.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseSeventeen">Week 17 (8/10/2015-8/16/2015)</A></H4></DIV><DIV id="collapseSeventeen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of P<SUB>blc</SUB></LI><UL><LI>Meanwhile as second strategy SpeI &amp; PstI cut pSB1C3 containing K608002 constitutive promoter was <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">dephosphorylated</A> and P<SUB>blc</SUB> followed by RFP was restricted with XbaI and PstI. Ligation and transformation via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> were performed </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#ColonyPCR" target="blank">Colony PCR</A> using <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K60_BlcPshort_f" target="blank">K60_BlcP_short_f</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#VR" target="blank">VR</A> identified 14 of 15 clones as insert positive. To differ between correct insert of pSB1C3-K608002-P<SUB>blc</SUB> construct and remaining P<SUB>blc</SUB>-pSB1C3 remains after gel extraction of P<SUB>blc</SUB> liquid cultures in LB + chloramphenicol were prepared for PCR with primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K60_BlcPshort_f" target="blank">K60_BlcPshort_f</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K60_BlcPshort_r" target="blank">K60_BlcPshort_r</A>, that binds on K608002 </LI><LI>Insert of P<SUB>blc</SUB> could be varified in all clones, but fragments seemed of slightly wrong size. Four of the clones were prepared for sequencing using Fermentas Kit for Plasmid Isolation </LI><LI>Unfortunately, sequencing revealed missing constitutive promoter K608002 in all clones </LI><LI>As new strategy, split primers were used to obtain a fragment containing P<SUB>blc</SUB> with part of pSB1C3 from these plasmids to perform Gibson Assembly with another fragment containing K608002 and the other half of pSB1C3 vector. <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">Q5 PCR</A> was therefor performed with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K60_BlcPshort_f" target="blank">K60_BlcPshort_f</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_1_rev" target="blank">Ars_1_rev</A>. The fragments were eluted from gel using Fermentas gel extraction kit</LI><LI>Second fragment is amplified next week </LI></UL><LI>Subcloning of GABA sensor BioBrick</LI><UL><LI>The GABA sensor with RFP was subcloned into pSB1C3.</LI><LI>Both the device and pSB1C3 were cut with EcoRI and PstI. FastAP was added to the digest of the GABA sensor.</LI><LI>After heat inactivation, 3 µL of each digest were used for ligation.</LI><LI>The ligation reaction was transformed via heat shock.</LI><LI>A test digest and sequencing confirmed that the subloning was successful.</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseEightteen">Week 18 (8/17/2015-8/23/2015)</A></H4></DIV><DIV id="collapseEightteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of P<SUB>blc</SUB></LI><UL><LI>The second fragment containing K608002 in pSB1C3 were amplified in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">Q5 PCR</A> with cells with plasmids containing the constitutive promoter in the vector that were used for the heavy-metal detectors as well as plasmids containing K608002, BlcR and pSB1C3 that were used for the synthesis of the repressor as templates. Primers were <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#K60_BlcPshort_r" target="blank">K60_BlcPshort_r</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_2_fwd" target="blank">Ars_2_fwd</A>. Plasmids were therefore isolated from K608002-pSB1C3 containing cells grown on LB-Cm plate using Fermentas Kit in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">Plasmid Isolation </A>. The other templates are already in stock as purified plasmids from previous experiments </LI><LI>2% Agerose Gel Electrophporesis showed no fragments of expected size</LI><LI>Several PCRs with various enzymes did not work, so we chose to do a <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">Phusion PCR</A> with a gradient from 55°C to 70°C and 4 different templates, namely a K608002 in pSB1C3 vector and 3 of our BlcR repressor sequence in each pSB1C3 vector behind the constitutive promoter K608002. This reaction -finally- the correct sized fragment at annealing temperature of 65°C. The remaining PCR product each of that temperatur was again seperated in gel electrophoresis and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNApurification" target="blank">extracted</A> from gel using Fermentas gel extraction kit </LI><LI>The amplifications were repeated to obtain more template, procedure for purification was repeated as well </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly </A> was performed with the obtained parts from the previous PCRs to obtain a complete K608002-P<SUB>blc</SUB>-pSB1C3 construct, the product was <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock transformed </A> and cultivated on Lb Canamycin plates</LI><LI>Grown clones were cultivated, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated</A> and prepared for sequencing </LI><LI>Sequencing revealed again missing insertion of the constitutive promoter K608002</LI><LI>Culture from the distribution was therefor cultivated on LB plate again for next weeks experiments, in addition primers were designed to test the promoter in vitro in our cell extract </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseNineteen">Week 19 (8/24/2015-8/31/2015)</A></H4></DIV><DIV id="collapseNineteen" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Assembly of P<SUB>blc</SUB></LI><UL><LI>Plasmids were <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">isolated</A> from the cultivated K608002 plasmid containing cells </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#BioBrickAssembly" target="blank">3 A Assembly</A> was performed with the constitutive promoter K608002, P<SUB>blc</SUB> and a pSB1T3 vector from the distribution </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">Heat shock</A> transformation was performed with ultra competent <I>E. coli </I></LI><LI>Unfortunately, no colonies grew </LI><LI>As new strategy, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed on T7-UTR-sfGFP plasmid in stock of our CFPS team with Primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Pblc_UTR_fwd" target="blank">Pblc_UTR_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Pblc_UTR_rev" target="blank">Pblc_UTR_rev</A> in combination with split primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_1_rev" target="blank">Ars_1-rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_2_fwd" target="blank">Ars_2_fwd</A>, which contain the operator site of P<SUB>blc</SUB> in their gibson-overhang to obtain a construct with T7 promoter, P<SUB>blc</SUB> operator site and UTR plus sfGFP </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed, the product was transfromed into ultra competent <I>E. coli</I> via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> and cultivated on LB-Cm plates</LI><LI>Clones were cultivated and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated </A>using PureYield Plasmid Miniprep System (promega) </LI><LI>Plasmids were sequenced and one clone was found positive for yield of T7 promoter as well as P<SUB>blc</SUB>, UTR and sfGFP. This clone was cultivated, plasmids again isolated and a glycerol stock was prepared </LI><LI>Culture with added rhamnose revealed strong increase in fluorescence compared to a not induced culture, which shows a functional promoter construct and functional T7</LI></UL><LI>Promoter-Repressor-Device construction</LI><UL><LI>The plasmid was used for <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#BioBrickAssembly" target="blank">3 A Assembly</A> and was cut with EcoRI and SpeI to reveal T7-P<SUB>blc</SUB>-UTR-sfGFP as upstream part. A former constructed plasmid yielding K608002 promoter followed by BlcR (repressor) was cut with XbaI and PstI. Destination vector pSB1T3 was cut EcoRI and PstI. Ligation was performed overnight, followed by <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock transformation</A> into ultracompetent cells, that were cultivated on Lb-tetracycline plate  </LI></UL><LI>His-Tagging of BlcR </LI><UL><LI>To further analyze the Repressor BlcR in vitro, we designed primers for addition of a His-tag sequence </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR-His_fwd" target="blank">BlcR-His_fwd</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR-His_rev" target="blank">BlcR-His_rev</A> with split primers as above </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#DNAPurification" target="blank">Gel extraction</A> was performed with Analytik Jena Innu Prep Gel Extraction kit, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed and the product was transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> into chemo competent <I> E. coli</I></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">Plasmid isolation</A> was performed using Promega PureYield Plasmid Mini Prep System. Plasmids were sequenced </LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwenty">Week 20 (9/01/2015-9/08/2015)</A></H4></DIV><DIV id="collapseTwenty" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI> His-Tagging of BlcR </LI><UL><LI>Unfortunately, sequening revealed a mutation in the His-Tag and therefore a frameshift</LI><LI>Starting over, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PCR" target="blank">PCR</A> was performed with split primers <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_1_rev" target="blank">Ars_1_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#Ars_2_fwd" target="blank">Ars_2_fwd</A> in combination with <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR-His_rev" target="blank">BlcR-His_rev</A> and <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers#BlcR-His_fwd" target="blank">BlcR-His_fwd</A>. </LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GibsonAssembly" target="blank">Gibson Assembly</A> was performed and the product was transformed via <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HeatShock" target="blank">heat shock</A> into <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#GeneratingCompetentCells" target="blank">chemocompetent cells</A></LI><LI>Following cultivation, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids were isolated</A> using PureYield Plasmid Mini Prep System (Promega) and were prepared for sequencing</LI><LI>In addition, restriction control was performed with EheI, that cuts following the his-tag. In case of successful adding of the his-tag sequence, two bands are obtained, while without two only one cut is performed resulting in one band. Restriction control revealed 4 positive clones. </LI><FIGURE><FIGCAPTION> Restriction analysis of potential His-BlcR plasmid. 2 log ladder used as marker, 2 bands of expected size reveal 4 positive clones. </FIGCAPTION></FIGURE><LI>Proteins were purified using Protino Ni-TED 1000 packed columns kit for  <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#HisTagPurification" target="blank"> his tag purification</A> and were analyzed in further tests with EMSA and PRIA</LI><LI>Sequencing revealed one positive clone missing two histidines, but remaining functional</LI><LI>Concentrating the protein and storing in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media#hepes">HEPES buffer</A></LI></UL><LI>Promoter-Repressor-Device construction</LI><UL><LI>Several clones were cultivated, <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols#PlasmidIsolation" target="blank">plasmids isolated</A> and prepared for sequencing</LI><LI>Sequencing revealed the sequence as expected</LI><LI>Meanwhile 1 clone was cultivated and compared to a clone containing only the promoter, not the repressor. The cultures were added rhamnose for the induction of the T7 promoter</LI><LI>Both constructs expressed fluorescent protein, which was appearing stronger when no repressor was present. For further characterization, cultures were prepared for fluorescence detection with and without possible analytes</LI></UL></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#collapseTwentyOne">Week 21 (9/09/2015-9/16/2015)</A></H4></DIV><DIV id="collapseTwentyOne" class="panel-collapse collapse"><DIV class="panel-body"><UL><LI>Backbone change of the device pSB1T3-T7-P<SUB>blc</SUB>-UTR-sfGFP-K608002-BlcR </LI><UL><LI>Restriction of pSB1C3-T7-UTR-sfGFP (vector) and pSB1T3-T7-P<SUB>blc</SUB>-UTR-sfGFP-K608002-blcR(insert) with EcoRI and PstI</LI><LI>Dephosphorylation of the vector</LI><LI>Ligation of vector and insert</LI><LI>Transformation of the ligated DNA in <I>E. coli</I> KRX</LI><LI>We confirmed the right sequence with colony PCR.</LI></UL><LI>Interaction study of BlcR-<I>P<SUB>blc</SUB></I>via EMSA</LI><UL><LI>We could confirm that BlcR formed a complex with P<SUB>blc</SUB>.</LI><LI>With this proof of functionality, we set out to investigate how the two analytes GBL and GHB can influence the interaction.</LI><FIGURE style="margin:auto; width:300px"><FIGCAPTION> EMSA shifts caused by addition of BlcR protein (see <A href="http://parts.igem.org/Part:BBa_K1758370" target="_blank">BBa_K1758370</A>) and BlcR-sfGFP fusion protein (see <A href="http://parts.igem.org/Part:BBa_K1758204" target="_blank">BBa_K1758204</A>), respectively, to Cy3-labeled blc-operator site. 5 pmol of following proteins were applied: 1: BlcR-sfGFP, 2: ArsR-sfGFP (see <A href="http://parts.igem.org/Part:BBa_K1758203" target="_blank">BBa_K1758203</A>), 3: BlcR, 4: none. </FIGCAPTION></FIGURE></UL><LI>Characterization of pSB1T3-T7-P<SUB>blc</SUB> -UTR-sfGFP-K608002-blcR</LI><UL><LI>Cultivation of <I>E. coli</I> KRX containing this plasmid in test tubes</LI><LI>We tested the function of the device with <I>gamma</I>-butyrolactone (0.2% and 1%) and <I>gamma</I>-Hydroxybutyric acid (0.2% and 1%). </LI><LI>We did triplicates of each test condition. For comparison, we did not add analyte solution for 3 cultures. We did not induce 3 other cultures with rhamnose. </LI><LI>Inoculation of cultures with OD<SUB>600</SUB> 0.1 in 6 mL LB medium</LI><LI>Simultaneous addition of 0.1 % rhamnose and analyte solution in the cultures at OD<SUB>600</SUB> 1.0</LI><LI>Results: </LI><FIGURE style="width: 400px; margin-right: 20px; float: left"><FIGCAPTION><I>In vivo</I> characterization of GBL / GHB sensor with strain containing <A href="http://parts.igem.org/Part:BBa_K1758377" target="_blank">BBa_K1758377</A>. All experiments were perfomed as triplicates. All samples except &quot;control, not induced&quot; were induced to express T7 polymerase at OD<SUB>600</SUB> = 0.7-0.8 </FIGCAPTION></FIGURE><LI>At 1 % GHB the culture did not grow as GHB had a toxic effect on <I>E. coli</I>.  </LI><LI>As you see in the figure, the difference between induced control and the cultures induced with 0.2 % GHB, 0.2 % and 1 % GBL was not significant so that we could not say for sure that more fluorescence was generated after the release of blc repressor through binding the analyte. </LI></UL></UL></DIV></DIV></DIV></UL></DIV></DIV></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>