<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bielefeld-CeBiTec_Results_HeavyMetals skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bielefeld-CeBiTec/Results/HeavyMetals</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Bielefeld 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-ex1-collapse"><UL class="nav navbar-nav" style="margin-left: -0.5em"><LI class="menu-item dropdown"><A href="#" class="dropdown-toggle glow" data-toggle="dropdown" id="project">Project </A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Overview">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Design</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorDesignMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Biosafety">Biosafety</A></LI></UL><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Targets</A></LI><LI class="menu-item "><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BiosensorFunctionMotivation">Motivation</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/DateRapeDrugs">Date Rape Drugs</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Description">Foundations</A></LI><LI><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/Detection">Output signal</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="results">Results
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results">Overview</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Design Evolution</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/CFPS">Cell Free Protein Synthesis</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/PRIA">Plasmid Repressor Interaction Assay</A></LI><LI class="menu-item dropdown dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biosensor Characterization</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals">Heavy Metals</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs">Date Rape Drugs</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Design">Output Signal Processing</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/it-really-works">ALL-IN-ONE Performance</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Parts">Parts</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/InterlabStudy">Interlab Study</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Achievements">Achievements</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="modeling">Modeling
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/CFPS">CFPS Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Biosensor">Biosensor Model</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Model Applications</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="team">Team
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Team">Team Members</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Attributions">Attributions</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/BielefeldUniversity">Bielefeld &amp; University</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Contact">Contact</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="pandp">Human Practices
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices">Overview</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/DualUse">Dual Use</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/Scenarios">Future Scenarios</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StreetScience">Street Science</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/StudentsAcademy">Students Academy</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Practices/PressReview">Press Review</A></LI><LI class="menu-item dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="notebook">Notebook
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Notebook">Lab Notebook</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Protocols">Protocols</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Media">Media &amp; Buffers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Primers">Primers</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Organisms">Organisms</A></LI><LI class="dropdown"><A class="dropdown-toggle glow" data-toggle="dropdown" href="#" id="partners">Partners
          </A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Sponsors">Sponsors</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Partners">Partners</A></LI><LI class="menu-item"><A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><P><NAV id="pagenav" class="navbar navbar-fixed-bottom"><UL class="nav nav-tabs" role="tablist" style="margin-left: 0px"><LI><A href="#arsenic" id="arsenicnav">Arsenic</A></LI><LI><A href="#chromium" id="chromiumnav">Chromium</A></LI><LI><A href="#copper" id="coppernav">Copper</A></LI><LI><A href="#lead" id="leadnav">Lead</A></LI><LI><A href="#mercury" id="mercurynav">Mercury</A></LI><LI><A href="#nickel" id="nickelnav">Nickel</A></LI></UL></NAV></P><DIV class="container"><DIV class="row"><DIV class="col-md-12"><DIV class="jumbotron" style="background-image: url(https://static.igem.org/mediawiki/2015/d/d1/Bielefeld-CebBiTec_Jumbotron_heavy_metals_results_2.jpeg)"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px">Heavy Metals</H1><P>Results</P></DIV></DIV><FIGURE style="width: 600px; float: right; margin-left: 20px"><FIGCAPTION>Figure 1: Influence of heavy metals on the growth of <I>E.coli</I> KRX. The tested concentrations were
20 µg/L lead, 60 µg/L mercury, 60 µg/L chromium, 80 µg/L nickel, 40 mg/L copper, which represent ten times the WHO guideline. The influence of arsenic was not tested as <I>E. coli</I> is known to be resistant to arsenic.
</FIGCAPTION></FIGURE><P>We tested our heavy metal biosensors in <I>Escherichia coli</I> as well as in our cell-free protein synthesis.</P><P>Prior to the <I>in vivo</I> characterization, we tested whether the heavy metals have a negative effect on the growth of <I>E. coli</I>.</P><P>As can be seen from the figure, we observed no significant difference between the growth in the presence of heavy metals and the controls. This first experiment showed us that <I>in vivo</I> characterization of these sensors is possible. Most cultivations for <I>in vivo</I> characterization were performed in the BioLector. Due to the accuracy of this device, we could measure our samples in duplicates. Subsequently, all functional biosensors were tested <I>in vitro</I>.</P></DIV></DIV><DIV class="row" style="margin-top: 40px"><DIV class="col-md-12"><DIV id="Teststrip"><P><B>Click on the test strip for the results of our biosensor tests in <I>E. coli</I> and in our CFPS:</B></P></DIV></DIV></DIV><DIV id="arsenic" style="display: none"><H1>Arsenic</H1><FIGURE style="width: 400px; float: right; margin-left: 20px"><FIGCAPTION>Figure 2: Genetic build-up of the arsenic biosensor we used for <I>in vivo</I> characterization. Both the repressor (arsR) and the reporter (mRFP1) are under the control of the same promoter, which is controlled by ArsR.</FIGCAPTION></FIGURE><P>We choose to work with the chromosomal arsenic operon of <I>E. coli</I>, which was used by the team from Edinburgh in 2006. This operon encodes an efflux pump which confers resistance against arsenic. The expression is controlled by the repressor ArsR, which negatively autoregulates its own expression. As<SUP>III</SUP> can bind to three cysteine residues in ArsR. The resulting conformational change deactivates the repressor (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals#Chen2014">Chen, Rosen 2014</A>).</P><P>By placing a reporter gene downstream of <I>arsR</I>, an arsenic biosensor can be constructed. In this case, both the repressor and the reporter are under the control of the same promoter (Figure 2). In this respect, the arsenic sensor is different from the other heavy metal biosensors we worked with, as their repressors or activators are expressed constitutively. However, the genetic build-up of the arsenic sensor is well-established. Consequently, we decided to keep this deviating design.</P><H2><I>in vivo</I></H2><P>We tested an arsenic sensor with mRFP1 as reporter gene <I>in vivo</I> to confirm that the sensor is functional. Furthermore, we examined whether it is possible to detect the safety limit as defined by the WHO. The results are shown in figure 3. We observed a reaction approximately five hours after addition of arsenic. The safety limit of 10 µg/L could clearly be distinguished from the negative control and the fluorescence signal increased up to a concentration of 500 µg/L. The signal in the presence of 1000 µg/L was slightly lower than in the presence of 500 µg/L.</P><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: Time course of the induction of an arsenic biosensor with RFP for different arsenic concentrations in vivo. Error bars represent the standard deviation of three biological replicates.</FIGCAPTION></FIGURE><H2><I>in vitro</I></H2><FIGURE style="width: 400px; float: right; margin-left: 20px"><FIGCAPTION>Figure 4: Genetic structure of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1758300" taget="_blank">BBa_K1758300</A>, which was used for characterization of the arsenic sensor <I>in vitro</I>. The arsenic operator was placed downstream of the T7 promoter in order to control the expression of sfGFP in the presence of the arsenic repressor, ArsR.</FIGCAPTION></FIGURE><P><I>E. coli</I> is resistant to arsenic because it possesses an efflux pump. The cell extract is not protected by such mechanisms, therefore we tested the effect of arsenic on the synthesis of sfGFP. We observed no significant effect for the relevant safety limits of 10 µg/L and 50 µg/L.</P><FIGURE style="width: 600px"><FIGCAPTION>Figure 5: Influence of arsenic on cell-free protein synthesis. We measured the expression of sfGFP (<A href="http://parts.igem.org/Part:BBa_K1758102" target="_blank">BBa_K1758102</A>) in the presence of the two internationally relevant safety limits of arsenic. The relative fluorescence was normalized to the fluorescence of a reference sample (culture supernatant of cells expressing sfGFP). Error bars represent the standard deviation of three biological replicates.</FIGCAPTION></FIGURE><P>In order to test the arsenic sensor in our cell-free protein synthesis, we cloned a device containing sfGFP under control of the T7 promoter and the arsenic operator combined with our optimized untranslated region (UTR). We tested this device in a cell extract that had been generated from cells expressing the arsenic repressor. As our <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">modeling</A> had shown us, this approach is superior to co-expressing the repressor in the reaction. We observed an induction when adding arsenic up to a concentration of 1.87 mg/L. As high arsenic concentrations inhibit the performance of the CFPS, we normalized the results to this effect. In the final application, this task is performed by our app.</P><DIV class="row" style="margin-bottom: 20px"><DIV class="col-md-6"><FIGURE style="width: 600px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 6: Induction of arsenic sensor in vitro. For this experiment, a cell extract that already contained the arsenic repressor was used in combination with <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1758300">BBa_K1758300</A>. Error bars represent the standard deviation of three biological replicates.</FIGCAPTION></FIGURE></FIGURE></DIV><DIV class="col-md-6"><FIGCAPTION>Figure 7: Normalized induction of arsenic sensor <I>in vitro</I>. For this experiment, a cell extract that already contained the arsenic repressor was used in combination with <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1758300">BBa_K1758300</A>. The data were normalized to account for the negative effect of arsenic on cell extract performance. Error bars represent the standard deviation of three biological replicates.</FIGCAPTION></DIV></DIV><DIV class="row"><DIV class="col-md-12"><P>Compared to the <I>in vivo</I> results, the response to arsenic was relatively small and we measured a high background signal. We assume that this is due to the different construct we used <I>in vitro</I>. This construct had been optimized for our CFPS by exchanging the natural promoter for the T7 promoter and exchanging mRFP1 for our optimized sfGFP. However, we assume that the repression in the presence of ArsR was not effective enough to observe a clear induction. The reason is most likely that the distance between the T7 promoter and the arsenic operator was too large. The distance was a result of our cloning strategy and would likely be suitable for <I>E. coli</I> promoters. However, the T7 promoter requires the operator to be very close for an efficient repression (<A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/HeavyMetals#Karig2012">Karig et al. 2012</A>). </P><P>In addition, we performed an experiment in which the arsenic repressor was not present in the reaction from the beginning, but was encoded on a second plasmid. The plasmid concentrations we used had been predicted by our model. In accordance with the aforementioned results, we observed no clear repression and addition of arsenic showed no effect. This experiment is discussed on the <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Modeling/Application">Modeling pages</A>.</P><H2>To summarize</H2><P>The arsenic sensor is the most prominent example of heavy metal biosensors. Our results confirm that it is possible to detect the safety limit of 10 µg/L arsenic with a cell-based arsenic senosor. The results of the tests in our CFPS indicate that this approach is suitable for the detection of arsenic as well. However, we believe that the genetic construct requires further optimization.</P><H3>References</H3><P id="Chen2014">Chen, Jian; Rosen, Barry P. (2014): Biosensors for inorganic and organic arsenicals. In Biosensors 4 (4), pp. 494–512. DOI: 10.3390/bios4040494.</P><P id="Karig2012">Karig, David K.; Iyer, Sukanya; Simpson, Michael L.; Doktycz, Mitchel J. (2012): Expression optimization and synthetic gene networks in cell-free systems. In Nucleic acids research 40 (8), pp. 3763–3774. DOI: 10.1093/nar/gkr1191.</P></DIV></DIV></DIV><DIV id="chromium" style="display: none"><H1>Chromium</H1><P>Chromium is an essential part of the earth´s crust (Mitchell D. Cohen et al.), but most of it is produced trough industrial uses (Paustenbach et al. 2003). We built a biosensor for the detection of hexavalent (Cr<SUP>VI</SUP>), because it is toxic and has cancerogenic effects on the human body. An intoxication of chromium can lead to damages of the nervous system. The World Health Organization recommends a limit of 50 µg/L in drinking water (Guidelines for drinking-water quality 2011, WHO 2003).</P><H2><I>in vivo</I></H2><P>The chromium sensor (<A href="http://parts.igem.org/Part:BBa_K1758313" target="_blank">BBa_K1758313</A>) was constructed by using the basic construction we showed in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals" target="_blank">Our biosensors</A>. We work with the chromate inducible operon of <I>Ochrobactrum tritici</I> 5bvl1 which enables a resistance for chromium VI and superoxide. For our sensor we used the Cr<SUP>VI</SUP> dependent repressor <I>chrB</I> which was introduced by team BIT 2013 (<A href="http://parts.igem.org/Part:BBa_K1058007" target="_blank">BBa_K1058007</A>),  and optimized this sequence for the use in <EM> E. coli </EM>. The repressor protein becomes deactivated by the binding of Cr<SUP>6+</SUP>-ions The associated chromium responsive promoter is <I>chrP</I> (introduced by BIT 2013) (<A href="http://parts.igem.org/Part:BBa_K1058007" target="_blank">BBa_K1058007</A>). For output we used sfGFP and a 5’UTR untranslated region in front of sfGFP to optimize the expression of the reporter protein and increase its fluorescence (see figure 2).</P><P>Our sensor for chromium detection consists of ChrB the repressor and the chromate specific promoter ChrP. The promoter is regulated by ChrB, which binds Cr<SUP>6+</SUP>-ions. Behind the promoter is a sfGFP for detection of a fluorescence signal.</P><FIGURE style="width: 600px"><FIGCAPTION>Figure 2: The concept of our <I>in vivo</I> chromium sensor (<A href="http://parts.igem.org/Part:BBa_K1758313" target="_blank">BBa_K1758313</A>), which consists of the repressor under the control of a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_K1758310" target="_blank">BBa_K1758310</A>)and the operator and promoter sequence of the chromium inducible promoter. An untranslated region in front of the sfGFP, which is used for detection, enhances its expression (<A href="http://parts.igem.org/Part:BBa_K1758312" target="_blank">BBa_K1758312</A>)</FIGCAPTION></FIGURE><I>In vivo</I> we could show that the addition of different concentrations of chromium has different effects to transcription of sfGFP.<DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: Time course of the induction of a chromium biosensor with sfGFP for different chromium concentrations in vivo. The data are measured with BioLector and normalized on OD600. Error bars represent the standard deviation of two biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 4: Fluorescence levels at three different stages of cultivation. Shown are levels after 60 minutes, 150 minutes and 650 minutes. Error bars represent the standard deviation of two biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>We tested our <I>in vivo</I> chromium sensor with sfGFP as reporter gene, to test the functionality of the system. Moreover we tested different chromium concentrations. The kinetic of our sensors response to different chromium concentrations is shown in figure 3. The first 30 hours show a strong increase in fluorescence. After that the increase in fluorescence is slower. For better visualization the kinetics of figure 3 are represented as bars in figure 4. A fluorescence level difference for 60 min, 150 min and 650 min is represented.</P><P>Our data lead to the conclusion that in a cell-based system it is possible to detect chromium. In contrast to our expectations with higher chromium concentrations we got lower fluorescence levels. These observations needed further investigation. Additionally the bar chart showed that the chromium sensor needs a long time to get different fluorescence levels at different chromium concentrations in <I>in vivo </I> experiments. The bar chart showed significant differences between the chromium concentrations after 650 minutes. </P><H2><I>in vitro</I></H2><P>For the characterization of the chromium sensor with CFPS we used parts differing from that we used in vivo characterization. For the in vitro characterization we used a cell extract produced from cells which contain the plasmid (<A href="http://parts.igem.org/Part:BBa_K1758310" target="_blank">BBa_K1758310</A>). The plasmid contains the gene <I>chrB</I> under the control of a constitutive promoter, so that the cell extract is enriched with repressor molecules. In addition to that we added plasmid-DNA of the chromium specific promoter <I>chrP</I> with 5’UTR-sfGFP under the control of T7-promoter (<A href="http://parts.igem.org/Part:BBa_K1758314" target="_blank">BBa_K1758314</A> (figure 6))to the cell extract. The T7-promoter is needed to get a better fluorescence expression. </P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 1000px"><FIGCAPTION> Figure 5: To produce the cell extract for <I>in vitro</I> characterization a construct (<A href="http://parts.igem.org/Part:BBa_K1758310" target="_blank">BBa_K1758310</A> ) with chromium repressor under the control of a constitutive promoter and strong RBS (BBa_K608002)  is needed.</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION> T7-chrP-UTR-sfGFP <A href="http://parts.igem.org/Part:BBa_K1758314" target="_blank">BBa_K1758314</A> used for<I>in vitro</I> characterization.</FIGCAPTION></FIGURE></DIV></DIV><FIGURE style="width: 600px"><FIGCAPTION>Figure 7: Influence of different chromium concentrations on our crude cell extract. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE><P>Chromium’s influence on the cell extract as shown in figure 7 is minimal for low concentrations. Higher chromium concentrations have a measurable impact on the viability of the cell extract, which is visible at concentrations of 120 µg/L and obvious at concentrations of 240 µg/L chromium.</P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 520px"><FIGCAPTION>Figure 8: Chromium specific cell extract made from <I>E. coli</I> cells which already expressed the repressor before cell extract production. Induction with different chromium concentrations. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE><P>The decrease of fluorescence for higher chromium concentrations in chromium specific cell extract is shown in figure 8. An increase of fluorescence at higher chromium concentrations would have been expected resulting out of the induction of the chromium sensor. 

A factor which should be considered is the influence of high chromium concentrations to the cell extract. The test for influence of chromium on the specific cell extract, illustrated in figure 7 showed that the influence of chromium at low concentrations is not significant. But the graphic shows that high concentrations of chromium induce fatal damages to the cell extract. </P></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 520px"><FIGCAPTION>Figure 9: Chromium specific cell extract made from <I>E. coli</I> cells which already expressed the repressor before cell extract production. Induction with different chromium concentrations. The data are normalized on chromium’s influence to the cell extract. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>Taking the influence of different chromium concentrations under consideration measured fluorescence can be normalized on chromium’s influence on the cell extract (figure 9). Normalized data suggest, that higher concentrations of chromium induce fluorescence in relevance to chromium’s influence on the cell extract. </P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 500px"><FIGCAPTION>Figure 10: Chromium sensor with alternative repressor build by team Dundee 2015, which has only the first 15 codons optimized in chromium specific cell extract under the induction with different chromium concentrations. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 540px"><FIGCAPTION>Figure 11: Chromium sensor with alternative repressor build by team Dundee 2015, which has only the first 15 codons optimized in chromium specific cell extract under the induction with different chromium concentrations. Data are normalized on chromium’s influence to the specific cell extract Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>In addition to the measurements of our chromium sensor in CFPS we measured our chromium inducible promoter with the repressor of team Dundee (figure 10, 11), which works similar to ours. In contrast to our repressor only first 15 codons of their repressor are codon-optimized. Measurements with their repressor showed tendencies similar to our measured repressor. After normalization induction with higher chromium concentrations showed a detectable fluorescence response for both measured datasets. </P><H2>To summarize</H2><H2><P>Our chromium sensor detects the presence of chromium <I>in vivo</I>, but the outcome differed from our expectations. We would have expected an increase in fluorescence by increasing chromium concentrations. Our <I>in vitro</I> data suggest that these decrease in fluorescence could be explained by chromium’s influence on <I>E. coli</I> which is not reflected in growth but shown by chromium´s influence on the cell extract. Before normalizing the <I>in vitro</I> data the same pattern as <I>in vivo</I> could be observed. After normalization an increase in signal is noticeable. Therefore with optimization our chromium sensor would be compatible to our cell free sensor system.</P></H2><H3>References</H3><DIV class="references"><P> Guidelines for drinking-water quality (2011). 4th ed. Geneva: World Health Organization, zuletzt geprüft am 20.08.2015.</P><P> Mitchell D. Cohen; Biserka Kargacin; Catherine B. Klein; and Max Costa: Mechanisms of Chromium Carcinogenicity and Toxicity, zuletzt geprüft am 19.08.2015.</P><P> Paustenbach, Dennis J.; Finley, Brent L.; Mowat, Fionna S.; Kerger, Brent D. (2003): Human health risk and exposure assessment of chromium (VI) in tap water. In: Journal of toxicology and environmental health. Part A 66 (14), S. 1295–1339. DOI: 10.1080/15287390306388.</P><P> WHO (2003): Mercury in Drinking-water Background document for development of WHO Guidelines for Drinking-water Quality, checked 15.08.15
</P></DIV></DIV><DIV id="copper" style="display: none"><H1>Copper</H1><P>
There are some possibilities for the contamination of drinking water with copper, for the production of pipes, valves and fittings copper is used (Guidelines for Drinking-water Quality, Fourth Edition). Copper is an essential trace element for humans, animals and plants, but an overdose can lead to anemia, liver and brain damages (US EPA ORD NCEA Integrated Risk Information System (IRIS) 2014). Additionally, high input of copper is associated with aging diseases as Atherosclerosis and Alzheimer’s disease (Brewer 2012). These damages can finally cause death. The World Health Organization recommends a limit of 2 mg/L in drinking water (Guidelines for Drinking-water Quality, Fourth Edition).
 </P><H2><I>in vivo</I></H2><P>Our sensor for copper detection consists of CueR a MerR like activator and the copper specific promoter <I>copAP</I>. The promoter is regulated by CueR, which binds Cu <SUP>2+</SUP> ions. We also used a <I>sfGFP</I> downstream the promoter for detection through a fluorescence signal.</P><P>For our copper sensor we used the native operator of cooper homeostasis from <I>E.coli</I> K12. We constructed a part (<A href="http://parts.igem.org/Part:BBa_K1758324" target="_blank">BBa_K1758324</A>) using the basic genetic structur shown in <A href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Project/HeavyMetals" target="_blank">our biosensors</A>.The operator sequence, which includes the promoter (<I>copAP</I>),  is regulated by the activator CueR. In BBa_K1758324 we combined a codon optimized version of <I>cueR</I> (<A href="http://parts.igem.org/Part:BBa_K1758320" target="_blank">BBa_K1758320</A>) under the control of a constitutive promoter with <I>sfGFP</I> under the control of the corresponding promoter <I>copAP</I>  (<A href="http://parts.igem.org/Part:BBa_K1758321" target="_blank">BBa_K1758321</A>)(figure 2). Through the addition of a 5’ UTR upstream of the <I>sfGFP</I> we optimized the expression of <I>sfGFP</I> and increased fluorescence. </P><FIGURE style="width: 600px"><FIGCAPTION>Figure 2: The concept of our <I>in vivo</I> copper sensor (<A href="http://parts.igem.org/Part:BBa_K1758324" target="_blank">BBa_K1758324</A>), which consists of the activator under the control of a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_K1758320" target="_blank">BBa_K1758320</A>) and the operator and promoter sequence of the copper inducible promoter. An untranslated region in front of the sfGFP, which is used for detection, enhances its expression (<A href="http://parts.igem.org/Part:BBa_K1758323" target="_blank">BBa_K1758323</A>). </FIGCAPTION></FIGURE><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: Time course of the induction of a copper biosensor with sfGFP for different copper concentrations <I>in vivo</I>. The data are measured with BioLector and normalized on OD<SUB>600</SUB>. Error bars represent the standard deviation of two biological replicates.</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 4: Fluorescence levels at three different stages of cultivation. Shown are levels after 60 minutes, 150 minutes and 650 minutes. </FIGCAPTION></FIGURE></DIV></DIV><P>We tested our <I>in vivo</I> copper sensor with <I>sfGFP</I> as reporter gene, to test the functionality of the system. Moreover, we tested different copper concentrations. The kinetic of our sensors response to different copper concentrations is shown in figure 3. The first ten hours show a strong increase in fluorescence. After that the increase in fluorescence is slower. For better visualization the kinetics of figure 3 are represented as bars in figure 4. A fluorescence level difference for 60 min, 150 min and 650 min is represented.</P><P><I>In vivo</I> we could show that the adding different concentrations of copper has effects on the transcription levels of <I>sfGFP</I>.</P><P>The shown data suggest that sensing copper with our device is possible even if the detectable concentrations are higher than the desireble sensitivity limits. Therfore we tested the copper sensor in our <I>in vitro</I> transcription translation approach.</P><H2><I>in vitro</I></H2><P>For the characterization of the copper sensor with CFPS we used parts differing from that we used in vivo characterization. For the <I>in vitro</I> characterization we used a cell extract out of cells which contain the plasmid  (<A href="http://parts.igem.org/Part:BBa_K1758320" target="_blank">BBa_K1758320</A>) (figure 5), so that the resulting extract is enriched with the activator CueR. To this extract we added plasmid-DNA of the copper specific promoter <I>copAP</I> with 5’-UTR-<I>sfGFP</I> under the control of T7-promoter (<A href="http://parts.igem.org/Part:BBa_K1758325" target="_blank">BBa_K1758325</A>) to the cell extract. The T7-promoter is needed to get a better fluorescence expression. </P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 400px"><FIGCAPTION> Figure 5: To produce the cell extract for <I>in vitro</I> characterization a construct (<A href="http://parts.igem.org/Part:BBa_K1758320" target="_blank">BBa_K1758320</A>) with copper activator under the control of a constitutive promoter and strong RBS (BBa_K608002) is needed. 
</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 400px"><FIGCAPTION>Figure 6: T7-copAP-UTR-sfGFP <A href="http://parts.igem.org/Part:BBa_K1758325" target="_blank">BBa_K1758325</A> used for <I>in vitro</I> characterization. </FIGCAPTION></FIGURE></DIV></DIV><P>The results presented in figure 7 illustrate the influences of different copper concentrations on the cell extract. </P><FIGURE style="width: 600px"><FIGCAPTION>Figure 7: Influence of different copper concentrations on our crude cell extract. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE><P>As shown in figure 7 copper has no negative influence on the functionality of our cell extract. Therefore, a relatively stable system for copper sensing is provided.

First tests with specific cell extract and different copper concentrations lead to further tests and normalizations, illustrated in figure 8.</P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 8: Copper specific cell extract made from <I>E. coli</I> cells which have already expressed the activator before cell extract production. Induction of copper inducible promoter without T7 upstream of the operator site with different copper concentrations. Error bars represent the standard deviation of three biological replicates. 
</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 9: Copper specific cell extract made from <I>E. coli</I> cells which have already expressed the activator before cell extract production. Induction of copper inducible promoter without T7 in front of the operator site with different copper concentrations. Error bars represent the standard deviation of three biological replicates. Data are normalized on coppers influence to the cell extract.</FIGCAPTION></FIGURE></DIV></DIV><P>In addition,we measured the operator device under the control of T7 promoter as described before.</P><P>Fluorescence was normalized to influence of copper on the the cell extract (figure 10 and figure 11).</P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 10: Copper specific cell extract made from <I>E. coli</I> cells which have already expressed the activator before cell extract production. Induction with different copper concentrations. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 11: Copper specific cell extract made from <I>E. coli</I> cells which have already expressed the activator before cell extract production. Induction of copper inducible promoter with different copper concentrations. Error bars represent the standard deviation of three biological replicates. Data are normalized on coppers influence to the cell extract. </FIGCAPTION></FIGURE></DIV></DIV><P>Compared to the former fluorescence levels the T7 reporter device showed higher levels. Therefore, a reporter device under the control of T7 promoter is more suitable for our CFPS.</P><P>After normalizing on coppers influence to the cell extract these differences were even more obvious.</P><H2>To summarize</H2><P>Our copper sensors <I>in vivo</I> data show that detection of different copper concentrations is possible. The fluorescence levels differ clearly between different induction concentrations. As shown above higher copper concentration, higher the fluorescence signal. Therefore the concept of our sensor is functional even if the concentration needed for induction are to high to reach sensitively concerning the WHO guidelines for copper. Our sensor has been tested <I>in vitro</I> as well. For copper we tested our original CopAP construct without a T7 promoter in front of the inducible at first.  After realizing that the sensor shows the right tendencies but the general fluorescence is quite low we created an inducible promoter under the control of a T7 promoter to use in CFPS. Fluorescence levels of this device showed the same tendencies as the one without but were higher fluorescence’s, which helps detecting it.</P><H3>References</H3><DIV class="references"><P id="Guidelines for Drinking-water Quality, Fourth Edition "> Background document for development of WHO Guidelines for Drinking-water Quality, checked on 9/9/2015. Copper excess, zinc deficiency, and cognition loss in Alzheimer's disease - Brewer - 2012 - BioFactors - Wiley Online Library. Available online at http://onlinelibrary.wiley.com/doi/10.1002/biof.1005/abstract, checked on 8/28/2015. </P><P id=" US EPA ORD NCEA Integrated Risk Information System (IRIS) 2014"> US EPA ORD NCEA Integrated Risk Information System (IRIS) (2014): Copper (CASRN 7440-50-8) | IRIS | US EPA. Available online at http://www.epa.gov/iris/subst/0368.htm, updated on 10/31/2014, checked on 9/2/2015.</P></DIV></DIV><DIV id="lead" style="display: none"><H1>Lead</H1> 

Lead is one of the most frequently used metals. Therefore, lead is found in many different parts of the environment (World Health Organization (WHO): Fact sheet number 379, Lead poisoning and health). The contamination of drinking water is often caused by obstructed pipes. Long time absorption leads to adverse health effects in most organs in the body (EPA Health Effects: How Lead Affects the body). The main targets are the nervous system, brain and liver. These damages can finally cause death. The World Health Organization recommends a limit of 10 µg of lead in 1 litre drinking water (WHO: Guidelines for Drinking-water Quality, fourth edition (2015)).  

<H2><I><I>in vivo</I> characterization </I></H2><P>In addition to the other heavy metal sensors, we constructed a sensor for lead detection. It consists of the repressor PbrR which binds at the operator box downstream of the <I>pbrAP </I>promoter. The binding of the repressor is reversible in the presence of Pb<SUP>2+</SUP> Ions. Those ions can weakened the repressors binding and hence, all genes downstream of the <I>pbrAP</I> promoter can be expressed. Like the former sensors this one encloses a sfGFP for detection via fluorescence. So if no lead is present in the media, the repressor binds to the operator box and the <I>pbrAP</I> promoter is blocked meaning that the transcription of <I>sfGFP</I> is prevented. No fluorescence signal is the results. By supplementation of lead, the repressor is separated from the operator box and the genes downstream of the promoter can be expressed.  </P><P>The <I>pbrAP </I>promoter, the operator box and the PbrR repressor are parts of the chromosomal lead operon of Cupriavidus metallidurans (figure 2). This was cloned and transformed into <I>E.coli </I>KRX. This operon includes now the promoter <I>pbrAP </I>(<A href="http://parts.igem.org/Part:BBa_K1758332" target="_blank">BBa_K1758332</A>), which is regulated by the repressor PbrR. The PbrR belongs to the MerR family, of metal-sensing regulatory proteins, and is Pb<SUP>2+</SUP>-inducible. Our sensor system comprises <I>pbrR</I> (<A href="http://parts.igem.org/Part:BBa_K1758330" target="_blank">BBa_K1758330</A>), which is under the control of a constitutive Promoter and <I>pbrAP</I> and a 5’ untranslated region, which controls the transcription of a sfGFP and increases the fluorescence. Fluorescence implemented by sfGFP protein is the measured output signal (figure 3 and figure 4).   </P><FIGURE style="width: 600px"><FIGCAPTION>Figure 2: The concept of our in vivo lead sensor (<A href="http://parts.igem.org/Part:BBa_K1758332" target="_blank">BBa_K1758332</A>) which consists of the repressor under the control of a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_K1758330" target="_blank">BBa_K17583230</A>) and the operator and promoter sequence of the lead inducible promoter. An untranslated region in front of the sfGFP, which is used for detection, enhances its expression (<A href="http://parts.igem.org/Part:BBa_K1758332" target="_blank">BBa_K1758332</A>).</FIGCAPTION></FIGURE><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: Time course of the induction of a lead biosensor with sfGFP for different lead concentrations <I>in vivo</I>. The data are measured with BioLector and normalized to the OD<SUB>600</SUB>. Error bars represent the standard deviation of two biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 4: Fluorescence levels at three different stages of cultivation. Shown are levels after 60 minutes, 150 minutes and 650 minutes. Error bars represent the standard deviation of two biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>We tested our lead sensor with sfGFP as reporter gene to verify the functionality of the system. Subsequently, we tested different lead concentrations. The kinetic of our sensors response to different lead concentrations is shown in figure 3. The first 40 hours show a strong increase in fluorescence. After that the increase in fluorescence reaches a plateau. For better visualization the kinetics of figure 3 are represented as bars in figure 4. A fluorescence level difference for 60 min, 150 min and 650 min is represented.</P><P> The results of the lead sensor show no significant differences between the different concentrations (figure 3). This might be due to the <I>pbrAP’s</I> weak promoter strength in <I>E. coli.</I> Further reasons are most likely in the weak repressor binding to its operator.   So, we suggest for the usage of this sensor, it has to be optimized. Moreover we were lacking time for further in vivo characterizations and different experimental setups. Hence, we did not use this sensor in further experiments regarding Cell-free-Protein-synthesis (CFPS). . In the future a characterization in the CFPS systems would be desirable.   </P><H2>To summarize</H2><P>Our lead sensor was characterized <I>in vivo</I> only. The differences between inductions with various lead concentrations are really slight therefore this sensor needs further optimization which was not possible in this limited time. But as there is a fluorescence response to lead this sensor has the potential work as expected. In the future a characterization in CFPS systems would be interesting.</P><H3>References</H3><DIV class="references"><P id="EPA"> EPA Health Effects: How Lead Affects the body, checked on 2015-09-17. </P><P id="WHO Guidelines for Drinking water"> WHO Guidelines for Drinking-water Quality fourth edition, checked on 2015-09-09.  </P><P id="WHO"> WHO lead poisoning and health, fact sheet number 379, reviewed August 2015, checked on 2015-09-17. 
 </P></DIV></DIV><DIV id="mercury" style="display: none"><H1>Mercury</H1><P>The main sources of mercury exposure are generated through humans. For example mercury contamination can be caused by medical waste (damaged measurement instruments), Fluorescent-lamps, Chloralkali plants and thermal power plants. In the environment, mercury is one of the most toxic elements . Acute effects of a mercury intoxication can range from diseases of the liver, kidney, gastrointestinal tract, to neuromuscular and neurological problems. A chronic intoxication of mercury results in kidney changes, changes in the central nervous system and other effects like cancer. The World Health Organization recommends a limit of 6 µg/L in drinking water. </P><H2><I>in vivo</I></H2><P>One of the already existing sensors we used for our system is the mercury sensor consisting of MerR the activator and the mercury specific promoter <I>pmerT</I>. The promoter is regulated by the MerR, which binds Hg<SUP>2+</SUP>-ions. Similar to the former sensors we added a sfGFP for detection via fluorescence. </P><P>For our mercury sensor we used parts of the mercury sensor constructed by iGEM team Peking 2010. These parts consist of the mercury dependent <I>mer</I> operon from <I>Shigella flexneri</I> R100 plasmid <I>Tn21</I>. The expression of the genes in the <I>mer</I> operon depends on the regulation by MerR its activator and promoter <I>PmerT</I>. For our sensor we used the codon optimized activator (<A href="http://parts.igem.org/Part:BBa_K1758340" target="_blank">BBa_K1758340</A>), under control of a constitutive promoter,(<A href="http://parts.igem.org/Part:BBa_K346001" target="_blank">BBa_K346001</A>). Additionally to this activator we designed and constructed the specific promoter <I>PmerT</I>(<A href="http://parts.igem.org/Part:BBa_K346002" target="_blank">BBa_K346002</A>)(figure 2). For our sensor we added a 5’-UTR downstreamd of this promoter, which increased the fluorscence of the used reporter protein sfGFP.</P><FIGURE style="width: 600px"><FIGCAPTION>Figure 2:  The concept of our <I>in vivo</I> mercury sensor (<A href="http://parts.igem.org/Part:BBa_K1758343" target="_blank"> BBa_K1758343</A>), which consists of the activator under the control of a constitutive promoter  <A href="http://parts.igem.org/Part:BBa_K1758340" target="_blank"> BBa_K1758340</A>)and the operator and promoter sequence of the mercury inducible promoter. An untranslated region in front of the sfGFP, which is used for detection, enhances its expression (<A href="http://parts.igem.org/Part:BBa_K1758342" target="_blank"> BBa_K1758342</A>).</FIGCAPTION></FIGURE><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: During  cultivation the sfGFP signal in reaction to different mercury concentrations was measured. The induction with mercury happened after 165 minutes. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 4: Fluorescence levels at two different stages of cultivation. Shown are levels after 120 minutes and 190 minutes. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>We tested our mercury sensor with sfGFP as reporter gene, to test the functionality of the system. Moreover we tested different concentrations. The kinetic of our sensors response to different mercury concentrations is shown in figure 3. A strong increase in fluorescence levels is notecible after induction with mercury after 120 min. For better visualization the kinetics of figure 3 are represented as bars in figure 4. A fluorescence level difference for 120 min and 190 min is represented.</P><P><I>In vivo</I> data show a highly significant, well working sensor, which even reacts to concentrations below the threshold of the water guidelines by the WHO (Figure 3 and 4).  </P><P>The mercury detection was measured during the cultivation of <I>E. coli</I> KRX at 37 °C (Figure 3 and 4). The strain contained the plasmid with the activator <I>merR</I>  under the control of a constitutive promoter and the specific promoter with an operator binding site, which reacts to the activator with bound Hg <SUP>2+</SUP>-ions. The specific promoter is located upstream of the sfGFP CDS. Therefore, the mercury in the medium is detected via formation of sfGFP. <I>In vivo</I>this sensor devise shows a fast answer to occurrence of his heavy metal contrary to the other sensor systems <I>In vivo</I>.</P><P> Therefore we tested our sensor <I>in vitro</I> to check if an already functioning highly optimized sensor provides required data for guideline detections </P><H2><I>in vitro</I></H2><P>For the characterization of the mercury sensor with CFPS we used parts differing from that we used in the <I>in vivo</I> characterization. For the <I>in vitro</I> characterization we used a cell extract out of cells, which contained the plasmid (<A href="http://parts.igem.org/Part:BBa_K1758340" target="_blank"> BBa_K1758340</A>)(figure 5). In addition, we added plasmid DNA to the cell extract. This plasmid consisted of the mercury specific promoter <I>pmerT</I> with 5’-UTR-sfGFP. The entire sequence was placed under the control of of T7-promoter (<A href="http://parts.igem.org/Part:BBa_K1758344" target="_blank"> BBa_K1758344</A>)(figure 6). The T7-promoter is needed to get a better fluorescence expression.</P><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 400px"><FIGCAPTION>Figure 5: To produce the cell extract for <I>in vitro</I> characterization a construct (<A href="http://parts.igem.org/Part:BBa_K175840" target="_blank">BBa_K175840</A>) with chromium repressor under the control of a constitutive promoter and strong RBS (BBa_K608002) is needed. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 400px"><FIGCAPTION>T7-<I>PmerT</I>-UTR-sfGFP <A href="http://parts.igem.org/Part:BBa_K1758344" target="_blank"> BBa_K175844</A> used for<I>in vitro</I> characterization.</FIGCAPTION></FIGURE></DIV></DIV><FIGURE style="width: 600px"><FIGCAPTION>Figure 7: Influence of different mercury concentrations on our crude cell extract. Error bars represent the standard deviation of three biological replicates.</FIGCAPTION></FIGURE><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 8: Mercury specific cell extract made from E. coli cells, which have already expressed the activator before cell extract production. Induction of mercury inducible promoter without T7 in front of the operator site with different mercury concentrations. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 9: Mercury specific cell extract made from <I>E. coli</I> cells, which have already expressed the activator before cell extract production. Induction of mercury inducible promoter without T7 in front of the operator site with different mercury concentrations. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P><I>In vitro</I> this sensor showed good results. The fluorescence level was high at low concentrations. Additionally, it showed that the expression level at 6 µg/L (Guideline of WHO for Mercury) reached the maximal signal. This result indicated the potential for measurement of concentrations under 6 µg/L.To confirm this hypothesis, it takes more experiments and tests with lower concentrations. Due to the high expression of sfGFP at low concentrations and the same expression level at different concentrations, it is not possible to quantify mercury with CFPS analyses . , Our model predicted this observation. During the measurement we noticed that the heavy metals have negative influences on the cell extract. Because of this fact, we used a correction factor, which resulted from the heavy metals influence on the CFPS system. This already optimized sensor showed the high potential of optimized sensors in CFPS.</P><H2>To summarize</H2><P>We demonstrated, that our mercury sensor works well <I>in vivo</I> (figure 4). There is a clearly noticeable increase in fluorescence after induction with mercury. Even the threshold concentration, which is given in the WHO guideline, can be measured. This well working sensor was tested in <I>in vitro</I> as well. The presented data suggested, that maximal output is reached at concentrations of 6 µg/ L,which represent the value of the former mentioned WHO guideline. Further optimization could lead to a decreasing <I>in vitro</I>  detection threshold.</P></DIV><DIV id="nickel" style="display: none"><H1>Nickel</H1><P> Naturally nickel occurrences are quite low and rare. In Germany most drinking water pollutions by nickel take place in the last meters of the plumbing system. The reason for these pollutions are wrong tap ware. There may be higher concentrations in drinking-water in special cases of release from natural or industrial nickel deposits. Higher nickel concentrations in water can lead to dermatitis as itching of fingers and other parts of the body through long term skin contact. The World Health Organization (WHO) recommends a limit of 70 µg/L in drinking water. </P><H2><I>in vivo</I></H2><P>We aimed to construct a sensor for nickel detection. It consists of <I>rcnR</I> the repressor and the nickel specific promoter <I>prcnA</I>. The promoter is regulated by the RcnR, which binds Ni<SUP>2+</SUP>-ions. As the former sensors this one encloses a sfGFP for detection via fluorescence.</P><P> Our nickel biosensor consists of parts of the <I>rcn</I>-operon from E. coli, which encodes a nickel- and cobalt-efflux system. This system is highly sensitive to nickel. In absence of nickel or cobalt RcnR binds to the operator and inhibits the nickel responsive promoter. With Ni<SUP>2+</SUP>-ions present the repression of the promoter <I>prcnA</I> will be reversed, because the repressor RcnR binds Ni<SUP>2+</SUP>-ions and cannot attach to the DNA. For our biosensor we construct the part <A href="http://parts.igem.org/Part:BBa_K1758353" target="_blank"> BBa_K1758353 </A> by using the basic construction shown in figure 2. For this part we used the repressor RcnR under control of a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_K1758350" target="_blank"> BBa_K1758350 </A>) and the nickel specific promoter PrcnA with a 5’UTR  in front of sfGFP (<A href="http://parts.igem.org/Part:BBa_K1758352" target="_blank"> BBa_K1758352 </A>) as reporter protein. </P><FIGURE style="width: 600px"><FIGCAPTION>Figure 2: The concept of our <I>in vivo</I> nickel sensor (<A href="http://parts.igem.org/Part:BBa_K1758354" target="_blank"> BBa_K1758354</A>), which consists of the activator under the control of a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_K1758350" target="_blank"> BBa_K1758350</A>)and the operator and promoter sequence of the nickel inducible promoter. An untranslated region in front of the sfGFP, which is used for detection, enhances its expression (<A href="http://parts.igem.org/Part:BBa_K1758352" target="_blank"> BBa_K1758352</A>) </FIGCAPTION></FIGURE><DIV class="row"><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 3: Time course of the induction of a nickel biosensor with sfGFP for different nickel concentrations in vivo. The data are measured with BioLector and normalized on OD<SUB>600</SUB>. Error bars represent the standard deviation of two biological replicates. </FIGCAPTION></FIGURE></DIV><DIV class="col-md-6 text-center" style="margin-bottom: 50px"><FIGURE style="width: 600px"><FIGCAPTION>Figure 4: Fluorescence levels at three different stages of cultivation. Shown are levels after 60 minutes, 150 minutes and 650 minutes. Error bars represent the standard deviation of three biological replicates. </FIGCAPTION></FIGURE></DIV></DIV><P>We tested our nickel sensor with sfGFP as reporter gene, to test the functionality of the system. Moreover we tested different concentrations. The kinetic of our sensors response to different nickel concentrations is shown in figure 3. The first five hours show a strong decrease in fluorescence. After that there is a slight increase in fluorescence. Starting levels of fluorescence are not reached. For better visualization the kinetics of figure 3 are represented as bars in figure 4. A fluorescence level difference for 60 min, 150 min and 650 min is represented.</P><P> The data for our nickel sensor show a trend that differs for that of the other sensors. There is no indication for a working sensor <I>in vivo</I> (Figure 3 and 4). There is a fluorescence signal, but it decreases in the first five hours. After reaching a minimum the fluorescence increases slowly. Additionally, there is no difference in fluorescence as response to various nickel concentration. Nickel could influence the cells and thereby caused a precipitation, which could result in decrease of fluorescence. 
With this sensor no production of sfGFP via fluorescence level change could be detected. Therefore, this sensor is not suitable for our approach. Due to this observation, no <I>in vitro</I> data using CFPS were taken.</P><H2>To summarize</H2><P>With this sensor no production of sfGFP via fluorescence level change could be detected. Therefore, this sensor is not suitable for our approach. Consequently, no <I>in vitro</I> tests were performed.
To create a working sensor based on this concept further optimization is needed.</P></DIV><DIV class="conclusion" style="padding: 0px 10px 5px 10px; margin-top: 60px"><H2>To summarize all </H2><P>We have characterized heavy metal sensors for arsenic, chromium, copper, lead, mercury and nickel. The results for our nickel characterization indicated that the constructed nickel sensor is not suitable for our test strip. The sensors for lead and chromium showed great potential, as they showed responses to chromium or lead, but require further optimization. Copper, our new heavy metal sensor, worked as expected and was able to detect different copper concentrations. The already well-characterized sensors for arsenic and mercury were tested as well. While the arsenic sensor worked well <I>in vivo</I>, it requires some omptimization for the use <I>in vitro</I>. Mercury showed that a fully optimized sensor works very well in our <I>in vitro</I> system and has the potential to detect even lower concentrations than <I>in vivo</I>.</P></DIV><DIV class="row footer"><DIV class="col-md-2 col-md-offset-10 text-center"><A type="button" class="btn btn-default btn-next" href="https://2015.igem.org/Team:Bielefeld-CeBiTec/Results/DateRapeDrugs"><P>And what about the date rape drugs?</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>