Team:Bielefeld-CeBiTec/Notebook/PlasmidDisplay
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Plasmid Repressor Interaction Assay
Expand all
Hide all
Week 1 (4/20/2015-4/26/2015)
Use of BioBricks
Heat shock transformation of BioBricks needed for further assays was performed. Glycerol stocks were generated and the plasmids isolated according to the kits manual. The relevant BioBricks were:
BBa_K525998 : T7 promotor and RBS
BBa_C0012 : LacI repressor with LVA Tag
BBa_R0010 : lac Promotor
BBa_K592008 : lac Operator
Tranformation of the given lacI plasmid
A plasmid with a gene coding for a His-tagged LacI Protein was transformed, which was kindly provided by Stefan Hoffmann, University of Potsdam.
Coding sequence was equal to BBa_C0012 except for the His-tag instead of the LVA-tag.
Week 2 (4/27/2015-5/3/2015)
Purification of His-tagged LacI
was performed according to the manual of the Protino® Ni-TED 1000 Packed Columns Kit from Macherey Nagel
Test digest of transformed BioBricks
All fragment sizes corresponded to the expected ones.
Plasmid isolation of BBa_R0010 and BBa_K592008
Week 3 (5/4/2015-5/10/2015)
Purification of His-tagged LacI
Purification was repeated to a higher purity
Use of the FPLC from BioRad for purification.
Concentration of pure LacI: 5.59 mg/mL
Plasmid isolation of BBa_R0010 and BBa_K592008
needed to be redone in order to generate enough material for our interaction assays.
Week 4 (5/11/2015-5/17/2015)
Verification of LacI-His
SDS-PAGE for analyzing the purity of purified LacI-His
LacI-His was expected in the 38 kDa bands
Bands (size 38 kDa) were cut out and destained.
SDS-PAGE for the identification of LacI.
Week 5 (5/18/2015-5/24/2015)
MS-Analysis of SDS-PAGE bands
Tryptic digestion of the proteins from the bands.
Measurement of the digested proteins via MALDI-TOF/TOF.
The measured peptide spectre was compared with spectre of other proteins with an E. coli database.
LacI identified (Mascot score: 765).
Week 6 (5/25/2015-5/31/2015)
Development of the Plasmid Repressor Interaction Assay (PRIA)
After every step the reaction tubes were centrifuged with 1000 g for 1 minute.
The supernatant after centrifugation was stored at -20 °C for subsequent analysis of protein and DNA amounts.
The DNA amount of the supernatant after centrifugation was analyzed via nanodrop and the protein amount analyzed via SDS-PAGE.
As negative controls we did not add protein to the Ni-NTA agarose in one sample and no plasmid in another sample.
We tested the interaction of LacI with the lac promoter and lac operator.
We also performed the complex formation at 0 °C and 37 °C
Steps:
25 µL Ni-NTA agarose was put in a reaction tube. Then the tube was centrifuged.
The Ni-NTA agarose was washed three times with 50 µL 200 mM potassium phosphate (Kpi) buffer .
10 µg protein (we used Lac Repressor (LacI) with a His-Tag as model protein) in 20 µL Kpi buffer was added and incubated for 30 min.  Then the sample was centrifuged.
The column with the protein was washed three times in Kpi Buffer (Volume: 30 µL). Then the sample was centrifuged.
1.5 µg plasmid (lacO) was mixed with 20 µL binding buffer (with 250 mM KCl) and incubated with the column for 15 minutes. Then the sample was centrifuged.
Unbound DNA was washed out three times with 50 µL binding buffer.
3x elution with 50 µL binding buffer and analyte (for this experiment: 0.5 mM IPTG).
50 µL imidazole was used to release proteins from the Ni-NTA agarose.
The DNA concentration could not be measured by the nanodrop.
Week 7 (6/1/2015-6/7/2015)
Plasmid isolation of E. coli KRX with pSB1C3-lacO
Suffix insertion: LacI-LVA as insert and pSB1C3-T7-RBS as backbone
The Standard protocol for suffix insertion was followed. Unfortunately unsuccessful
Optimizing PRIA
The DNA amount of the supernatant after centrifugation was analyzed via gel electrophoresis.
Repetition of the assay performed before.
Negative control in which no proteins are bound to the Ni-NTA agarose. Plasmid DNA with the operator site cannot bind and is washed out.
Plasmid Repressor Interaction Assay with LacI and lacO. lacO can be released after the addition of IPTG.
We could detect DNA after staining in ethidium bromide.
A high DNA amount was unbound.
Elution with IPTG is possible. DNA could be detected in the samples after adding the analyte solution to the Ni-NTA agarose.
We did not see any difference between the temperatures at which the formation of the DNA-protein complex was performed.
Not adding protein to the Ni-NTA agarose is a good negative control, to see that there is neither unspecific release of DNA in the elution steps nor unspecific interaction of the plasmid with the column.
The rest of the Ni-NTA agarose after the final step was dissolved in 10 µL water and also used for the analysis via gel electrophoresis.
Determination of protein amount in the supernatant of every step in the assay.
Detection via SDS-PAGE
No protein detected, except in the step where it was eluted from the Ni-NTA agarose with imidazole. This means that the signal we detect is due to release of the plasmid, not due to loss of protein.
Week 8 (6/8/2015-6/14/2015)
PRIA: Testing analyte solutions with different concentrations
We wanted to test 0.05 mM, 0.5 mM and 5 mM IPTG in binding buffer as elution buffer to determine the sensitivity of our procedure.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in water to find out whether salt is necessary for disrupting the unspecific linkage between the Ni-NTA agarose and unbound DNA.
Plasmid Repressor Interaction Assay with LacI and lacO. We release DNA from the DNA-protein complex by adding 50 µM IPTG in the binding buffer.
Plasmid Repressor Interaction Assay with LacI and lacO. We release DNA from the DNA-protein complex by adding 500 µM IPTG in the binding buffer.
Plasmid Repressor Interaction Assay with LacI and lacO. We release DNA from the DNA-protein complex by adding 5 mM IPTG in the binding buffer.
As you see in the first figure, only water as wash buffer is not suitable for washing out unspecific binding of the plasmid to the Ni-NTA agarose. You need salt for washing out.
We standardized our assay. Every step took 15 minutes and the volume of the added solutions is 50 µL.
Cloning of His-tagged LacI in pSB1C3 backbone
Restriction of pSB1A3, pSB1C3-lacI-LVA and pSB1C3-T7-RBS for a 3A Assembly
Repetition of the suffix insertion, this time with success
Transformation in E. coli and subsequent plasmid isolation
PCRs with split primers was performed to add the His-tag. Used primers:
Cm_fwd
Cm_rev
lacIhis0418_rv
lacIhis0418_fw
Testing PRIA on an ELISA plate
Coating of an ELISA plate to test if the protein can still perform its function if it is unspecifically immobilized
Further evidence of LacI-lacO interaction
Preparation of an electrophoretic mobility shift assay (EMSA)
PCR for the creation of Cy3-labeled DNA fragments containing the lac operator, lac promoter, just T7 and RBS as a negative control and the ars operator. Used Primers:
VR
VF-Cy3
Week 9 (6/15/2015-6/21/2015)
Interaction study of LacI-lacO via EMSA
We performed an electrophoretic mobility shift assay with lacO and LacI
A shift was visible, but no clear band could be seen.
Week 10 (6/22/2015-6/28/2015)
Repetition of EMSA for lacO-LacI interaction
We added different amounts of the LacI, to achieve a clear shift of DNA band.
The gel melted.
Optimizing PRIA with different salt concentrations
Concentrations of 100 mM, 250 mM and 500 mM KCl in the binding buffer were tested in PRIA.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 100 mM KCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 250 mM KCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 500 mM KCl.
Using binding buffer with 500 mM KCl was best for PRIA, because no plasmid remained on the column and a strong signal was visible in the first elution step.
Test of PRIA in a 96 well plate for ELISA
96 well plate as new system for PRIA
Unspecific adsorption of LacI.
No elution of lacO. DNA was unbound.
The plate was treated with the same procedure as the Ni-NTA agarose.
Analysis by SDS-PAGE showed different bands, probably the BSA used for blocking of the ELISA plate was seen as well.
Week 11 (6/29/2015-7/5/2015)
PRIA with different salt concentrations and complex formation prior to adding the protein to the Ni-NTA agarose.
We tried to create the complex of lacO and LacI first and then bind it on the Ni-NTA agarose.
We tested higher salt concentrations (0.5 M, 0.75 M, 1 M, 2 M KCl) in the binding buffer. Moreover, we examined whether we could also utilize sodium chloride instead of potassium chloride.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 500 mM NaCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 500 mM KCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 750 mM KCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 1 M KCl.
Plasmid Repressor Interaction Assay with LacI and lacO. We washed the DNA-protein complex in binding buffer with 2 M KCl.
Sodium chloride could also be used instead of potassium chloride, which could be useful in terms of usability.
Cloning of pSB1C3-T7-RBS-lacI-His
Transformation of pSB1C3-T7-RBS-lacI-HisTag in E. coli KRX was successful.  Colonies were used to inoculate overnight cultures.
Week 12 (7/6/2015-7/12/2015)
Repetition of EMSA with different protein concentrations
EMSA shift caused by given amount of LacI added to 0.05 pmol Cy3-labeled lacO fragment.
Test to bind DNA in filter paper
The test was done to examine whether DNA would be washed out of paper.
500 ng DNA was applied on a filter paper strip. Then the filter paper was stained with ethidium bromide (EtBr).
For the negative control, a filter paper strip without DNA was stained with EtBr.
No detection of DNA possible due to the high intrinsic signal of the paper strip.
We transformed a BioBrick for a cellulose binding domain (CBD) which is reported to bind strongly to cellulose.
Transformation of Biobrick BBa_K1321340 in E. coli KRX
Colony PCR of 5 clones on the agar plate after transformation.
5 colonies were used to inoculate overnight cultures.
Plasmid isolation
Digestion with EcoRI and PstI for screening
We wanted to develop two new assays.
One assay is based on GFP-labeled protein and immobilized DNA (with Cy3- and amino-label) on filter paper. The other one is based on immobilized protein (contains a cellulose binding domain) and Cy3-labeled DNA.
Primer design for cloning CBD and sfGFP to the devices ArsR, BlcR and LacI.
These are the devices we wanted to clone via Gibson Assembly:
pSB1C3-T7-RBS-LacI-sfGFP-His
PCR 1.1
Template 1: T7-RBS-lacI-His-tag
Primer 1.1: C3-lacI-His-fw
Primer 1.2: T7-RBS-lacI-rv
PCR 1.2
Template 2: pSB1C3-T7-RBS-sfGFP
Primer 2.1: lacI-sfGFP-fw
Primer 2.2:lacI-sfGFP-His-rv
pSB1C3-T7-ArsR-sfGFP
PCR 2.1
Template 1: pSB1C3-BBaJ33201
Primer 1.1: arsR-L-sfGFP-rv
Primer 1.2:T7-RBS-arsR-fw
PCR 2.2
Template 2: pSB1C3-T7-RBS-sfGFP
Primer 2.1: T7-RBS-arsR-rv
Primer 2.2: L-sfGFP-fw
pSB1C3-T7-RBS-ArsR-sfGFP-His
PCR 3.1
Template 1: pSB1C3-T7-RBS-arsR-sfGFP
Primer 1.1: Cm_rev
Primer 1.2: UberHis-arsR-fw
PCR 3.2
Template 2: pSB1C3-T7-RBS-arsR-sfGFP
Primer 2.1: Cm_fwd
Primer 2.2: arsR-UberHis_rv
pSB1C3-T7-RBS-blcR-sfGFP
PCR 4.1
Template 1: pSB1C3-blcR
Primer 1.1: BlcR-T7-RBS-fw
Primer 1.2:BlcR-L-sfGFP-rv
PCR 4.2
Template 2: pSB1C3-T7-RBS-sfGFP
Primer 2.1: T7-RBS-arsR-rv
Primer 2.2: L-sfGFP-fw
pSB1C3-T7-RBS-blcR-sfGFP-His
PCR 5.1
Template 1: pSB1C3-blcR
Primer 1.1: Cm_rev
Primer 1.2: UberHis-arsR-fw
PCR 5.2
Template 2: pSB1C3-T7-RBS-blcR-sfGFP
Primer 2.1: Cm_fwd
Primer 2.2: arsR-UberHis_rv
pSB1C3-T7-RBS-arsR-CBD
PCR 6.1
Template 1: pSB1C3-K1321340
Primer 1.1: arsR-dCBD-fw
Primer 1.2: CBD-eHisTerm-rv
PCR 6.2
Template 2: pSB1C3-T7-RBS-arsR-sfGFP
Primer 2.1: arsR-rv
Primer 2.2: eHis-dTerm-fw
pSB1C3-T7-RBS-blcR-CBD
PCR 7.1
Template 1: pSB1C3-K1321340
Primer 1.1: bclR-CBD-fw
Primer 1.2: CBD-eGFPTerm-rv
PCR 7.2
Template 2: pSB1C3-T7-RBS-blcR-sfGFP
Primer 2.1: eGFP-dTerm-fw
Primer 2.2: bclR-rv
pSB1C3-T7-RBS-lacI-CBD
PCR 8.1
Template 1: pSB1C3-K1321340
Primer 1.1: lacI-CBD-fw
Primer 1.2: lacI-CBD-Vektor-rv
PCR 8.2
Template 2: pSB1C3-T7-RBS-lacI-HisTag
Primer 2.1: EndHis-Vektor-lacI-fw
Primer 2.2: lacI-rv
Cy3- and amino-labeled lacO operator
No Gibson Assembly.
Template: pSB1C3-lacO
Primer 1: VF-Cy3
Primer 2: SuffixR-fw (amino-labeled)
Cy3- and amino-labeled arsenic operator
No Gibson Assembly.
Primer 1: arsO-rv (Cy3-labeled)
Primer 2: arsO-fw (amino-labeled)
Primers were annealed, no template necessary
Cy3- and amino-labeled blc promoter
No Gibson Assembly.
Template: pSB1C3-Pblc-RFP
Primer 1: VF-Cy3
Primer 2: SuffixR-fw (amino-labeled)
Week 13 (7/13/2015-7/19/2015)
Coating of a 96-well plate with microcrystalline cellulose.
The instruction is from the iGEM team 2014 at Imperial university: 40 g cellulose is dissolved in 250 mL water. 200 µL cellulose solution is put in a well and incubated in an incubator (37 °C) overnight. We did a serial dilution to determine the optimal amount of cellulose per well. The optimal amount was 100 µL of a solution with 16 g/L Avicell cellulose.
Cloning the devices for our new assays.
We ran several PCRs for cloning the devices mentioned before and extracted DNA out of gel.
We assembled the DNA through Gibson Assembly and transformed the assembled plasmids in E. coli KRX.
Clones were evaluated through colony PCR. We cultivated several clones which contained the plasmid.
Week 14 (7/20/2015-7/26/2015)
Plasmid isolation of the overnight cultures
All devices were sent to sequencing.
The sequencing results confirmed that at least one sequence of every device was as expected.
Cultivation
E. coli KRX with the plasmid pSB1C3-T7-RBS-arsR-sfGFP, pSB1C3-T7-RBS-lacO-sfGFP-His and pSB1C3-T7-RBS-blcR-sfGFP were cultivated and harvested to check whether the fusion proteins could be functionally expressed. Fluorescence of the pellets was detected via our special form of photography.
Immobilization of DNA
Preparation of necessary buffers for the activation of paper.
Week 15 (7/27/2015-8/2/2015)
Immobilization of DNA on paper
Filter paper was activated by PDITC dissolved in DMSO. The DMSO was not dried, so the PDITC did not dissolve properly.
Cy3- and amino-labeled DNA was immobilized on the activated paper and could be detected with the Typhoon scanner. It was created by hybridization of an amino-labeled oligonucleotide and a Cy3-labeled oligonucleotide. After washing with water the DNA signals were weaker. After washing with SSC buffer the signal vanished completely due to the denaturating effects of SSC.
Cultivation
E. coli KRX with the plasmid pSB1C3-T7-RBS-arsR-sfGFP, pSB1C3-T7-RBS-lacO-sfGFP-His and  pSB1C3-T7-RBS-blcR-sfGFP were cultivated and were disrupted via sonification.
Quantification of protein
Concentrations of the cell lysates with the fusion proteins were determined with Rotiquant.
A SDS PAGE was performed to check if the fusion proteins with a CBD were overexpressed. Unfortunately, an overexpression could not been seen clearly.
Week 16 (8/3/2015-8/9/2015)
Immobilization of DNA
PDITC was dissolved in ethanol and acetic acid, and used for activation of Paper. No washing with SSC buffer
The oligonucleotides that were to be immobilized were in PCR buffer or 50 mM sodium phosphate buffer, pH 8.0. The oligonucleotides in the 50 mM sodium phosphate buffer performed better and were more resistant to washing.
We blocked the membranes with the immobilized DNA with milk powder or BSA
Immobilization of protein
We planned to add the CBD fusion proteins to paper and wash the unbound protein off
Test for the visualization of protein on paper: Coomassie staining solutions and destaining solutions normally used for SDS PAGEs worked just fine and stained the protein spots on paper specifically.
The proteins bound strongly and unspecific to paper. We tested various buffers to remove them and washed overnight. Nevertheless, no diminishing of the coomassie signal was detected.
EMSA to check ArsR for binding to DNA
successful clear band shift.
Cloning of an T7-RBS-UTR-sfGFP-His device
Week 17 (8/10/2015-8/16/2015)
Purification of His-tagged ArsR and analysis via SDS-Page
Immobilized DNA in 96 well plate
DNA was immobilized on the cellulose coated 96 well plate following the same protocol as for normal paper
100 µL of the protein solution with 20 µg/mL were added to each well. The proteins were the GFP-tagged fusion proteins.
The wells were washed with 200 µL of KBT buffer
90 µL of sample solution with a final concentration of 100 µg/mL was added.
Immobilized DNA on Paper
We were able to detect the immobilized Cy3-labeled DNA and the GFP-tagged Fusion proteins with an Ettan DIGE fluorescence scanner.
PRIA and EMSA for GFP-tagged proteins were performed
The formation of the protein DNA complex was proven by both methods, nevertheless no release of DNA could be measured.
Week 18 (8/17/2015-8/23/2015)
Measurement in Tecan reader
We immobilized amino- and Cy3-labeled DNA on PDITC-activated paper in the 96 well plate, added ArsR in every well and put analytes in the well.
As we wanted to test the operator-repressor interaction and the effect of the analytes on the interaction we tested ArsR-arsO, BlcR-Pblc and LacI-lacO.
Our samples were:
3x operator DNA with repressor plus analytes
3x operator DNA with repressor without analytes
1x immobilized operator DNA without repressor proteins
3x blank
For more information on the procedure, you could read the protocol "Measurement of fluorescence in plate reader"
Results
The measurement on filter paper in a 96 well plate did not deliver the expected results. Reasons were:
The measured data within a triplicate fluctuated. The differences of the GFP fluorescence of the sample were too low.
The release of the fusion protein after addition of the analytes was unspecific. So it did not matter if analytes were in the sample or not.
The binding buffer was not suitable for washing out unspecific binding. A high amount of fusion protein could be measured in the blank samples.
The amount of DNA was too low. After every wash step a little amount of immobilized DNA was released.
For a better and functional procedure, we had to test different buffer which could prevent unspecific interaction and stabilize the complex formation between the operator DNA and the repressor protein.
The difficulty is that you need different binding and elution conditions for interaction between repressor protein and operator DNA.
PRIA on Ni-NTA agarose for the interaction between LacI-sfGFP and lacO, arsR-sfGFP and lacO, BlcR-sfGFP and Pblc and ArsR and arsO
We wanted to test the interaction between LacI-sfGFP and lacO, ArsR-sfGFP and lacO, BlcR-sfGFP and Pblc and ArsR and arsO.
We tested the same conditions like in the protocol "PRIA protocols"
As a positive control we immobilized LacI.
DNA was detected in the imidazole step and on the Ni-NTA agarose, indicating that the DNA-protein complex was formed, but the binding could not be disrupted by the analyte.
The reason could be that the analyte binding site was complicated by the fusion of sfGFP.
Another reason is that the conditions were not appropriate for the interaction.
We presumed that DTT needed to be added in every buffer to guarantee the reducing form of the protein.
Repetition of the PRIA on Ni-NTA agarose for the interaction between LacI-sfGFP and lacO, arsR-sfGFP and lacO, blc-sfGFP and Pblc and arsR and arsO
The test was negative again. So our presumption was wrong.
Week 19 (8/24/2015-8/30/2015)
PRIA was repeated with conditions similar to those of EMSA
Plasmid Repressor Interaction Assay with arsR-sfGFP and arsO. We used 1x EMSA buffer as wash and binding buffer.
Plasmid Repressor Interaction Assay with LacI-sfGFP and lacO. We used 1x EMSA buffer as wash and binding buffer.
Plasmid Repressor Interaction Assay with blcRsfGFP and Pblc. We used 1x EMSA buffer as wash and binding buffer.
EMSA was performed to check whether a reaction to the analytes GBL, GHB or SSA could be seen for blcR
this was not the case
Week 20 (9/1/2015-9/6/2015)
Interaction of fusion proteins with analytes
We tested if our sfGFP arsR fusion protein would show FRET when interacting with the Cy3-labeled arsO. This was not the case. 1 µM solutions were measured in a nanofluorimeter and the heights of the peaks at 565 and 510 nm were compared.
The same procedure was performed for BlcR and PBlc. No FRET could be measured. Assumingly SSA quenched the fluorescence of GFP.
Immobilized Protein and DNA
The detection limit for GFP-tagged proteins and Cy3-labeled DNA was determined by scanning of an serial dilution on paper in the Ettan DIGE
By washing we determined the durability of the immobilized DNA on paper. Their was no difference between paper activated with PDITC dissolved in EtOH and PDITC dissolved in DMSO dried with molecular sieves. Therefore we recommend the activation of paper with PDITC dissolved in EtOH, which is easily available. Furthermore, the activated paper can be used up to two weeks after activation for the immobilization of DNA.
Week 21 (9/7/2015-9/13/2015)
Reproduction of PRIA on Ni-NTA agarose for lacO-LacI interaction
PRIA with LacI and lacO.
We tested the interaction in triplicates and stored the samples for the quantification of DNA in the samples through PicoGreen Assay.
PicoGreen Assay for the quantification of DNA for PRIA
We wanted to quantify DNA in every sample to know how much DNA is unbound and eluted.
We measured the triplicates we made before and the elution samples.
Figure A: PicoGreen Assay for the quantification of the samples of this week's PRIA on Ni-NTA agarose for lacO-LacI interaction.
Figure B: PicoGreen Assay for the quantification of the samples of PRIA on Ni-NTA agarose for lacO-LacI interaction in week 11.
As you can see in figure A, most of the DNA were unbound and washing out unbound DNA is necessary to exclude falsification of the elution step. All four wash steps were needed.
DNA could be eluted by adding the analytes. Most of the bound DNA were released after the first elution step.
You could also see in the imidazole step that DNA was barely bound to the protein. So almost every bound DNA plasmid was eluted.
In figure B you can see that both binding buffers are equal. In the previous PRIA experiments we used the buffer with 500 mM.
All in all, the PicoGreen Assay shows us that our developed Plasmid Repressor Interaction Assay works and that we can wash unbound DNA and elute bound DNA efficiently.
Week 22 (9/14/2015-9/18/2015)
Winter is coming, so we are preparing for Wiki-FREEZE!!
