Team:Bielefeld-CeBiTec/Notebook/DRdrugs
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Date Rape Drugs
Expand all
Hide all
Week 1 (4/20/2015-4/26/2015)
Experiment design
Literature was screened for substances that are used as date rape drugs
Week 2 (4/27/2015-5/3/2015)
Experiment design
GHB was identified as potential agent
Experts were contacted because of legal restriction of the handling of GHB
Further research showed potential identified GABA as analogon that is legal to handle and easy to work with
Research then suggested the use of the GABA inducible operon in Bacillus subtilis
Week 3 (5/4/2015-5/10/2015)
Experiment design
Local experts were contacted regarding the cultivation and induction of B. subtilis
Primers were designed for several targets within the gab operon
Week 4 (5/11/2015-5/17/2015)
Experiment design
Further screening of literature and genebanks showed another GABA inducable operon in Rhizobium leguminasorum
For practical reasons regarding cultivation B. subtilis inducable operon gab was chosen to be obtained via pcr for cloning while Rh. leguminasorum was not
Parts from both operons were ordered as synthesized fragments
Week 5 (5/18/2015-5/24/2015)
Experiment design
Primers were designed for various experiments regarding both operons and parts of them
Week 6 (5/25/2015-5/31/2015)
Pre-cultures of B. subtilis
Bacillus subtilis substrain 168 contains the GABA inducable operon gab and is in stock in our laboratory. qRT-PCR is planned to determine a cross-reaction of this operon on date rape drugs. The operon is synthesized and therefor not obtained from these cultures
Bacillus subtilis substrain 168 was obtained from glycerin culture stored at - 80°C in our lab
Cultivation on LB plates at 30 °C and 37 °C
12 hours growth showed more and bigger colonies on the plate incubated at 37 °C
Cultivation in 15 mL LB as well as 15 mL TY liquid medium at 37 °C / 250 rpm each in 100 mL flasks
OD600 after 24 hours was significantly higher in LB which was therefor chosen for further cultivation
Week 7 (6/1/2015-6/7/2015)
Cultivation of B. subtilis
Medium was composed of CgXII-N, salt solution and 30% glucose
Ammonium sulfate or GABA were added in various concentrations to determine the use as nitrogen source
Unfortunately, cultures did not or almost not grow
3 cultures grew after 72 hours incubation
Further experiment design to improve cultivation
Week 8 (6/8/2015-6/14/2015)
Experiment design
As important analyte GBL (y-Butyrolactone)was identified because of the lack of legal restrictions and therefor increasing use as date rape drugs
Investigations show inducable operon in Agrobacterium tumefaciens
Growth inhibition test: GBL - B. subtilis and E. coli
To determine growth inhibition by GBL growth in presence of several concentrations of GBL solution was tested
Both B. subtilis and E.coli were cultivated on LB plates
Filter paper circles were soaked with 3 microliters 1,3,5,10,50 and 100% GBL solutions and incubated
Both cultures grew almost uninfluenced. GBL concentrations of 50 and 100% show minor inhibition
Cultivation of B. subtilis for qRT-PCR
Because of earlier lack of sufficient growth, an optimal medium for this particular substrang was determined
Pre-cultures were grown in LB low salt, c-medium and CgXII
Tryptophane, ferric ammonium citrate as well as III´salts were filtrated sterile and freshly added
Week 9 (6/15/2015-6/21/2015)
Insertion of GABA gBlocks into pSB1C3
Two GABA inducible systems from Bacillus subtilis and Rhizobium leguminosarum were obtained as gBlocks from IDT.
To enable cloning into pSB1C3, we wanted to add suitable overhangs to the vector.
However, the first PCR did not yield product.
Cultivation of Agrobacterium tumefaciens substrain C58.1 for determination of optimal growth conditions
Retrieved from glycerin stock held at -80 °C
Cultivated on antibiotic free LB plates
After 72 h growth cultivated in liquid LB w/o MgSO4 and with 160 mg/l MgSO4 at 28 °C and 180 rpm
After 22 h growth higher OD600 with MgSO4 (12.9) than without (5.8)
Week 10 (6/22/2015-6/28/2015)
Insertion of GABA gBlocks into pSB1C3
To determine the optimal conditions for the amplification of pSB1C3, we tested Q5, Phusion, GC enhancer and three annealing temperatures.
Q5 with GC enhancer and annealing at 69 °C worked best.
Using these conditions we were able to amplify the vectors for both gBlocks.
Following gel extraction, the vectors and the gBlocks were combined via Gibson assembly.
2 µL of each reaction were transformed into NEB 5-alpha Competent E. coli via heat shock.
On the following day, the colonies were screened with a colony PCR and three colonies per construct were choosen for overnight cultures.
Following plasmid isolation, sequencing samples were prepared.
PCR on various elements of the operon and repressor in Agrobacterium tumefaciens
To test the primers for the sequences containing the repressor (BlcR), the promoter and operator (Pblc), the operon without repressor (Pblc-BlcABC) and for two vectors (pSB1C3, K516030), we performed Q5 PCR on A. tu. obtained from LB plate after heat destruction (95 °C, 10 min) as well as on E.coli containing the backbones
Primers were used as following:
BlcR: BlcR_fwd & BlcR_rev
Pblc: pro_Blc_fwd & pro_Blc_rev
pSB1C3 Vector with overhang for BlcR: pSB1C3_BlcR_fwd & pSB1C3_BlcR_rev
K516030 Vector with overhang for Pblc: K516030proB_fwd & K516030proB_rev
Pblc-BlcABC: pro_Blc_fwd & BlcC_rev
For BlcR and Pblc 25 s extention period and 62 °C annealing temperature worked well
For Pblc-BlcABC 90 s extention period and 68 °C annealing temperature worked well
Gel electrophoresis on 1 % Agarose gel showed weak bands for Pblc and BlcR so PCR was repeated the next day and a 2 % Agarose gel was chosen for analysis of BlcP template
Gel electrophoresis showed clear bands and BlcR PCR product was cut out from gel using a non exposure technique to prevent mutation caused by ethidium bromide or UV radiation and stored at 4 °C
Week 11 (6/29/2015-7/5/2015)
BlcR: Transformation into Escherichia coli
To obtain the sequence coding for the repressor we performed PCR, ligated framents into vectors via Gibson Assembly and transformed into E. coli via heat shock and verificated the insertion with Colony PCR
PCR on vector pSB1C3 was repeated as described before with primers containing overhangs for blcR
PCR product was isolated (Analytik Jena: Innu Prep gel extraction kit)
BlcR fragment was inserted into pSB1C3 vector using Gibson Assembly and heat shock transformed into chemo competent Escherichia coli
Cells were cultivated on chloramphenicol containing LB plates
5 colonies were tested for insert using Colony PCR with verification primers VF and VR with annealing temperature of 60 °C and 45 s denaturation and annealing period
4 colonies were found positive for insert of the vector
The colonies were grown in LB containing chloramphenicol for following plasmid isolation
Plasmids were purified to a concentration of 105-181 ng/microliter and sequenced
Amplification and extraction of promoter and operator region of Agrobacterium tumefaciens
Q5 PCR using Primers pro_Blc_fwd & pro_Blc_rev was performed on A.th.obtained from LB plate after heat destruction (95 °C, 10 min) with parameters as described before
3 % Agarose gel showed sharp bands that were cut out
K516030 backbone for insertion of Pblc (promoter and operator sequence) was amplified via PCR using the primers K516030proB_fwd & K516030proB_rev on K516030 carrying E. coli obtained from LB plate
negative control showed contamination, but sharp bands were obtained
PCR was repeated using the same primers but purified plasmid from stock as template
Bands were cut out and purified using gel extraction kit (Analytik Jena: innuPrep gel extraction kit)
Pblc was inserted in K516030 vector using Gibson Assembly and transformed into chemo competent E.coli via heat shock , that were grown on LB plates containing chloramphenicol at 37 °C
Unfortunately, no growing colonies were obtained
3A assembly of GABA sensitive elements and RFP
Restriction digest of GABA gBlocks in pSB1C3 with EcoRI and SpeI and RFP with XbaI and PstI
Ligation of the fragments and pSB1K3 for 2 h
Heat shock transformation
Large colonies grew only on the plates with the GABA sensor from Rhizobium
A colony PCR showed that these large colonies were positive, while smaller colonies were contaminations
Week 12 (7/6/2015-7/13/2015)
Transformation of BlcR in E. coli and sequencing
ColonyPCR was performed on colonies grown on LB plates containing canamycin for selection
To detect inserts verification primers VR and VF were used
Three positive colonies were cultivated, plasmids were isolated ,purified for sequencing and glycerol stocks were prepared
Sequences of BlcR in pSB1C3 Vector did not show mutations, but an illegal restriction site. Primers for mutation were designed
Repeat of amplification on Pblc and assembly
ColonyPCR was performed using pro_Blc_fwd and pro_Blc_rev Primer and K516030 containing E. coli from LB plate cultivation
Analysis in 3% agarose gel electrophoresis showed 5 fragments of expected size that were cut out
DNA was eluted from gel using Analytik Jena innuPrep gel extraction kit and stored
K516030 backbone was amplificated via PCR using Primers K516030proB_rev and K516030proB_fwd with overhang for Pblc
Sharp bands of approximately 3500 bp were obtained while expected fragments should be of 2964 bp size
Repeat showed same results, probably because of false priming
Repeat of the PCR with purified plasmid as template showed bands of expected size
DNA was purified using Thermo Scientific GeneJET gel extraction kit
Gibson Assembly was performed with Pblc and K516030 backbone, purified for sequencing and transformed via heat shock in chemo competent E. coli
Colony PCR identified 3 positive insertions, these clones were cultivated
Assembly of Pblc with constitutive promoter
3 A BioBrick Assembly was performed using the constitutive promoter K608002 upstream, Pblc clone 4 downstream and pSB1K3 as destination plasmid to construct a constitutive expression of RFP with the operator site in Pblc as target for the GBL responsive BlcR
The product was transformed into chemo competent E. coli via heat shock
Transformed cells were cultivated in liquid LB and analysed for fluoreszence. As control a Pblc (clone 7) was analyzed. The construct with K608002 in front of Pblc showed weaker fluoreszence
Unfortunately, sequencing for clone 4, that was used in the assembly, came back negative for insertion from sequencing
Amplification of BlcABC operon and its vector
PCR was performed on A. tumifaciens cultivated on LB plates using BlcC_rev and pro_Blc_fwd primers
Results of gel electrophoresis showed expexted size of approximately 3566 kb
PCR was performed on pSB1C3 purified linear vector with primers K516030proB_rev and pSB1C3blcOnewfw , but was not successful.
3A assembly of GABA sensitive elements and RFP
The ligation was repeated over night
Following heat shock transformation, several  colonies grew on the plastes.
A colony PCR showed that most of them were correct.
Overnight cultures were inoculated and plasmids were isolated the following day.
Sequencing showed that both fragments were present in the vector, but did not cover the complete sequence.
First test with GABA sensors
5 mL cultures were inoculated with E. coli harboring the GABA sensor plasmids (1x Rhizobium, 2x B. subtilis).
For each clone, there was one negative control and one culture with 10 mg GABA.
After growth over night at 37 °C, the cultures were transfered into tubes and centrifuged for 5 min.
The pelletes of all negative controls did not appear red.
The pellets of the cultures with the B. subtilis sensor and GABA were clearly red.
The sensor from Rhizobium did not seem to work.
A test with filters in front of a smartphone camera showed a signal from all pellets. Either the filters were not effective enough or the sesnsors produce a significant background signal.
Week 13 (7/13/2015-7/19/2015)
Amplification of BlcABC operon and its vector
PCR to obtain the vector with overhang for BlcABC was repeated as before using BlcC_rev and pro_Blc_fwd primers
Bands were cut out and stored
Mutagenesis of BlcR
PCR was performed for the vector with overhang for the mutation of the illegal restriction site in the sequence coding for the repressor BlcR using primers Ars_2_fwd and BlcR_rev as well as Ars_1_rev and pSB_BlcRfwd_new
Electrophoresis showed bands in expected sizes of 1689 bp and 1318 bp that were cut
DNA was purified using PCR CleanUp
Week 14 (7/20/2015-7/26/2015)
Assembly of mutated BlcR
Gibson Assembly was performed on the fragments of the split primer PCR last week
Heat shock transformation into chemo competent E. coli was performed and insertion was verified by Colony PCR
Positive clones were cultivated and plasmids isolated and sequenced
Assembly of BlcABC
Gibson Assembly was performed with BlcABC and pSB1C3 with overhang
Heat shock transformation into chemo competent E. coli was performed and insertion was verified with Colony PCR
Assembly of Pblc clones 7 & 9
Since sequencing of clone 4 was negative for insertion, experiments are repeated with the positive clones 7&9
Gibson Assembly was performed with both clones each with vector pSB1K3, products were transformed via heat shock , cultivated on LB plate containing kanamycin
ColonyPCR identified no positive clones, repeat next week
Measurement of the induction of the GABA sensor with GABA and GBL
A 5 mL overnight culture of E. coli KRX harboring the GABA sensor out of B. subtilis was prepared in LB with kanamycin.
5 % GBL were added to a second culture to test if this has any effect on E. coli
GABA solutions with different concentrations were prepared.
On the following day it was observed that the culture with GBL had not grown.
Several 5 mL cultures were inoculated from the overnight culture without GBL to an OD600 of 0.1.
At OD600 0.4-0.5 100 µL of the GABA solutions were added. The final concentrations were 10 mg/L, 1 mg/L, 100 µg/L, 10 µg/L and 1 µg/L.
Culturing at 37 °C.
After 4.5 h, 50 µL per culture were transferred into a black 384 well plate and measured with a Tecan reader.
A culture with the plasmid encoding the tryptophanase instead of any fluorescent protein was measured as negative control.
There was no noteworthy difference between the RFP fluorescence of the GABA cultures. However, their signal was stronger than that of the tryptophanase.
The fluorescence was measured again after 24 h. This time, the cultures were diluted to an OD600 of 2 beforehand to account for the differences in growth.
A slight induction seemed to be observable at the two highest tested GABA concentrations.
The background signal was very high. Apart from leakiness of the system, we reasoned that this might also be due to traces of GABA in the LB medium. Therefore, we decided to repeat the measurement with M9.
Week 15 (7/27/2015-8/2/2015)
Assembly of Pblc
Heat shock transformation in chemo competent E. coli was repeated as last week: Pblc in pSB1K3 vector
Lacking any success, we switched strategy to obtain K608002-Pblc-RFP construct and performed an assembly of K608002 in pSB1C3, cut with PstI and SpeI, Pblc cut with XbaI and PstI
The construct was transformed via heat shock and cultivated on LB-kanamycin plate
4 clones were cultivated in liquid LB, plasmid isolation followed
Verification via restriction control with NotHI showed fragments of expected size, while sequencing revealed no insertion of K608002 in tested clones 1 and 4
Isolation of BlcR_new (without illegal restriction site)
Plasmid Isolation was performed on 3 positive clones with Analytik Jena Innu Prep Plasmid Mini Kit
Control and repeat of assembly of BlcABC operon
Performed Colony PCR revealed no insertion
Repeat of Gibson Assembly with a molar ratio of 1:1
Following heat shock transformation and ColonyPCR was negative for insertion again
Third repeat showed insertion in several colonies, 3 of which were cultivated, plasmids isolated and sequenced. BlcR was shown to have been succesfully mutated. No illegal restriction site was found
Week 16 (8/3/2015-8/9/2015)
Assembly of BlcABC
Repeat of GibsonAssembly with both parts of BlcABC and the vector with fitting overhang, followed by heat shock transformation into chemo competent E. coli
ColonyPCR did not show the insert, though it might not have been visible in the agarose gel electrophoresis
Alternatively a C-lysis was performed and identified 6 possibly positive clones. three of which were cultivated, plasmids isolated and restricted with EcoRI. Bands obtained were of the expexted size. Plasmids probably containing BlcABC in pSB1C3 were sequenced
Assembly of Pblc
3 A Assembly was performed using purified K608002 as upstream part, Pblc clone 9 in front of RFP from plasmid isolation (week 14) as downstream part and pSB1K3 as destination plasmid
Following heat shock no colonies grew on selection plate with LB and kanamycin.
Regarding these failing attempts of assembly, Pblc clone 7 and two pSB1K3 plasmids were restricted with EcoRI and PstI, purified with PCR Purification Kit (Analytik Jena) and vectors were dephosphorylated . Pblc was extracted from gel (InnuPrep geneJET gel extraction kit)
Following ligation, the product was transformed via heat shock into chemo competent E. coli and cultivated on LB-kanamycin selection plate
Plasmid isolation was performed after cultivation of 3 clones in liquid LB + kanamycin. Plasmids were purified and sequenced
Measurement of the GABA sensor induction in M9
Single colonies of the GABA sensor E. coli were obtained from the glycerol stock by plating.
Three colonies were used to inoculate 5 mL cultures in M9 with kanamycin.
The precultures had to be cultured for 48 h at 37 °C to reach a suitable OD.
The cultures were diluted to an OD600 of 0.8. 300 µL were then added to 4.6 mL M9 with kanamycin and 100 µL GABA or GBL solution.
The measurement was carried out on the following day. RFP fluorescence of 50 µL culture was measured at 570 nm and 607 nm in a black 384 well plate. The OD600 was measured with 200 µL in a transparent 96 well plate.
The values were normalized as follows: The OD600 of M9 medium was subtracted from the OD600 of the cultures. The relative fluorescence was divided by the optical density of the culture. Finally, the normalized fluorescence of a culture without plasmid-encoded fluorescence proteins (instead it harboured the his-tagged arsR plasmid) was subtracted from the other values. The mean of two cultures was used for this autofluorescence correction, because one culture did not grow. All other values were measured in triplicate. The error bars represent the standard deviation.
As can be seen, the background signal in M9 was much lower than in LB. Furthermore, a strong induction by GABA can be observed. The sensor did not respond to GBL.
Week 17 (8/10/2015-8/16/2015)
Assembly of Pblc
Meanwhile as second strategy SpeI & PstI cut pSB1C3 containing K608002 constitutive promoter was dephosphorylated and Pblc followed by RFP was restricted with XbaI and PstI. Ligation and transformation via heat shock were performed
Colony PCR using K60_BlcP_short_f and VR identified 14 of 15 clones as insert positive. To differ between correct insert of pSB1C3-K608002-Pblc construct and remaining Pblc-pSB1C3 remains after gel extraction of Pblc liquid cultures in LB + chloramphenicol were prepared for PCR with primers K60_BlcPshort_f and K60_BlcPshort_r , that binds on K608002
Insert of Pblc could be varified in all clones, but fragments seemed of slightly wrong size. Four of the clones were prepared for sequencing using Fermentas Kit for Plasmid Isolation
Unfortunately, sequencing revealed missing constitutive promoter K608002 in all clones
As new strategy, split primers were used to obtain a fragment containing Pblc with part of pSB1C3 from these plasmids to perform Gibson Assembly with another fragment containing K608002 and the other half of pSB1C3 vector. Q5 PCR was therefor performed with K60_BlcPshort_f and Ars_1_rev . The fragments were eluted from gel using Fermentas gel extraction kit
Second fragment is amplified next week
Subcloning of GABA sensor BioBrick
The GABA sensor with RFP was subcloned into pSB1C3.
Both the device and pSB1C3 were cut with EcoRI and PstI. FastAP was added to the digest of the GABA sensor.
After heat inactivation, 3 µL of each digest were used for ligation.
The ligation reaction was transformed via heat shock.
A test digest and sequencing confirmed that the subloning was successful.
Week 18 (8/17/2015-8/23/2015)
Assembly of Pblc
The second fragment containing K608002 in pSB1C3 were amplified in Q5 PCR with cells with plasmids containing the constitutive promoter in the vector that were used for the heavy-metal detectors as well as plasmids containing K608002, BlcR and pSB1C3 that were used for the synthesis of the repressor as templates. Primers were K60_BlcPshort_r and Ars_2_fwd . Plasmids were therefore isolated from K608002-pSB1C3 containing cells grown on LB-Cm plate using Fermentas Kit in Plasmid Isolation . The other templates are already in stock as purified plasmids from previous experiments
2% Agerose Gel Electrophporesis showed no fragments of expected size
Several PCRs with various enzymes did not work, so we chose to do a Phusion PCR with a gradient from 55°C to 70°C and 4 different templates, namely a K608002 in pSB1C3 vector and 3 of our BlcR repressor sequence in each pSB1C3 vector behind the constitutive promoter K608002. This reaction -finally- the correct sized fragment at annealing temperature of 65°C. The remaining PCR product each of that temperatur was again seperated in gel electrophoresis and extracted from gel using Fermentas gel extraction kit
The amplifications were repeated to obtain more template, procedure for purification was repeated as well
Gibson Assembly was performed with the obtained parts from the previous PCRs to obtain a complete K608002-Pblc-pSB1C3 construct, the product was heat shock transformed and cultivated on Lb Canamycin plates
Grown clones were cultivated, plasmids isolated and prepared for sequencing
Sequencing revealed again missing insertion of the constitutive promoter K608002
Culture from the distribution was therefor cultivated on LB plate again for next weeks experiments, in addition primers were designed to test the promoter in vitro in our cell extract
Week 19 (8/24/2015-8/31/2015)
Assembly of Pblc
Plasmids were isolated from the cultivated K608002 plasmid containing cells
3 A Assembly was performed with the constitutive promoter K608002, Pblc and a pSB1T3 vector from the distribution
Heat shock transformation was performed with ultra competent E. coli
Unfortunately, no colonies grew
As new strategy, PCR was performed on T7-UTR-sfGFP plasmid in stock of our CFPS team with Primers Pblc_UTR_fwd and Pblc_UTR_rev in combination with split primers Ars_1-rev and Ars_2_fwd , which contain the operator site of Pblc in their gibson-overhang to obtain a construct with T7 promoter, Pblc operator site and UTR plus sfGFP
Gibson Assembly was performed, the product was transfromed into ultra competent E. coli via heat shock and cultivated on LB-Cm plates
Clones were cultivated and plasmids isolated using PureYield Plasmid Miniprep System (promega)
Plasmids were sequenced and one clone was found positive for yield of T7 promoter as well as Pblc, UTR and sfGFP. This clone was cultivated, plasmids again isolated and a glycerol stock was prepared
Culture with added rhamnose revealed strong increase in fluorescence compared to a not induced culture, which shows a functional promoter construct and functional T7
Promoter-Repressor-Device construction
The plasmid was used for 3 A Assembly and was cut with EcoRI and SpeI to reveal T7-Pblc-UTR-sfGFP as upstream part. A former constructed plasmid yielding K608002 promoter followed by BlcR (repressor) was cut with XbaI and PstI. Destination vector pSB1T3 was cut EcoRI and PstI. Ligation was performed overnight, followed by heat shock transformation into ultracompetent cells, that were cultivated on Lb-tetracycline plate
His-Tagging of BlcR
To further analyze the Repressor BlcR in vitro, we designed primers for addition of a His-tag sequence
PCR was performed with BlcR-His_fwd and BlcR-His_rev with split primers as above
Gel extraction was performed with Analytik Jena Innu Prep Gel Extraction kit, Gibson Assembly was performed and the product was transformed via heat shock into chemo competent E. coli
Plasmid isolation was performed using Promega PureYield Plasmid Mini Prep System. Plasmids were sequenced
Week 20 (9/01/2015-9/08/2015)
His-Tagging of BlcR
Unfortunately, sequening revealed a mutation in the His-Tag and therefore a frameshift
Starting over, PCR was performed with split primers Ars_1_rev and Ars_2_fwd in combination with BlcR-His_rev and BlcR-His_fwd .
Gibson Assembly was performed and the product was transformed via heat shock into chemocompetent cells
Following cultivation, plasmids were isolated using PureYield Plasmid Mini Prep System (Promega) and were prepared for sequencing
In addition, restriction control was performed with EheI, that cuts following the his-tag. In case of successful adding of the his-tag sequence, two bands are obtained, while without two only one cut is performed resulting in one band. Restriction control revealed 4 positive clones.
Restriction analysis of potential His-BlcR plasmid. 2 log ladder used as marker, 2 bands of expected size reveal 4 positive clones.
Proteins were purified using Protino Ni-TED 1000 packed columns kit for his tag purification and were analyzed in further tests with EMSA and PRIA
Sequencing revealed one positive clone missing two histidines, but remaining functional
Concentrating the protein and storing in HEPES buffer
Promoter-Repressor-Device construction
Several clones were cultivated, plasmids isolated and prepared for sequencing
Sequencing revealed the sequence as expected
Meanwhile 1 clone was cultivated and compared to a clone containing only the promoter, not the repressor. The cultures were added rhamnose for the induction of the T7 promoter
Both constructs expressed fluorescent protein, which was appearing stronger when no repressor was present. For further characterization, cultures were prepared for fluorescence detection with and without possible analytes
Week 21 (9/09/2015-9/16/2015)
Backbone change of the device pSB1T3-T7-Pblc-UTR-sfGFP-K608002-BlcR
Restriction of pSB1C3-T7-UTR-sfGFP (vector) and pSB1T3-T7-Pblc-UTR-sfGFP-K608002-blcR(insert) with EcoRI and PstI
Dephosphorylation of the vector
Ligation of vector and insert
Transformation of the ligated DNA in E. coli KRX
We confirmed the right sequence with colony PCR.
Interaction study of BlcR-Pblcvia EMSA
We could confirm that BlcR formed a complex with Pblc.
With this proof of functionality, we set out to investigate how the two analytes GBL and GHB can influence the interaction.
EMSA shifts caused by addition of BlcR protein (see BBa_K1758370 ) and BlcR-sfGFP fusion protein (see BBa_K1758204 ), respectively, to Cy3-labeled blc-operator site. 5 pmol of following proteins were applied: 1: BlcR-sfGFP, 2: ArsR-sfGFP (see BBa_K1758203 ), 3: BlcR, 4: none.
Characterization of pSB1T3-T7-Pblc -UTR-sfGFP-K608002-blcR
Cultivation of E. coli KRX containing this plasmid in test tubes
We tested the function of the device with gamma-butyrolactone (0.2% and 1%) and gamma-Hydroxybutyric acid (0.2% and 1%).
We did triplicates of each test condition. For comparison, we did not add analyte solution for 3 cultures. We did not induce 3 other cultures with rhamnose.
Inoculation of cultures with OD600 0.1 in 6 mL LB medium
Simultaneous addition of 0.1 % rhamnose and analyte solution in the cultures at OD600 1.0
Results:
In vivo characterization of GBL / GHB sensor with strain containing BBa_K1758377 . All experiments were perfomed as triplicates. All samples except "control, not induced" were induced to express T7 polymerase at OD600 = 0.7-0.8
At 1 % GHB the culture did not grow as GHB had a toxic effect on E. coli.
As you see in the figure, the difference between induced control and the cultures induced with 0.2 % GHB, 0.2 % and 1 % GBL was not significant so that we could not say for sure that more fluorescence was generated after the release of blc repressor through binding the analyte.
