Team:Bielefeld-CeBiTec/Protocols
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Protocols
(Never) change a running system!
Expand all
Hide all
Standard protocols
Blunt end ligation
For "intramolecular" ligation of a PCR product the 5'-ends are phosphorylated with T4 Polynucleotidekinase (PNK)
Set up the following reaction:
reagent
volume [ µL]
10x T4 DNA ligase buffer
2.5
T4 PNK
1
PEG 4000 50% (w/v)
2.5
DNA (100 to 600 ng)
x
ddH2O
to 25
Incubate at 37 °C, 30 min
Heat inactivate at 65 °C, 20 min
Add 1 µL T4 DNA ligase after reaction has cooled down to room temperature
Incubate at room temperature for at least 2 h, overnight also works.
Next step: Transformation via heat shock
Colony PCR
One Reaction mix contains
5 µL 5x GoTaq buffer (Promega)
1 µL MgCl2 (25 mM stock)
0.5 µL dNTPs (10 mM each)
0.25 µL primer mix (prefix/suffix primers or sequencing primers) 100 mM
17.625 µL ddH2O
0.125 µL GoTaq polymerase (Promega)
0.5 µL template
PCR program
Cell lysis and denaturation: 5 min, 95 °C
30 cycles
Denaturation: 10 s, 95 °C
Hybridisation: 30 s, annealing temperature
Elongation: 60 s/kb of product, 72 °C
Final elongation: 5 min, 72 °C
Template alternatives
Pick a colony with sterile tip, elute in 100 µL ddH2O or buffer, store at 4 °C during PCR, plate if insert is of correct size
Pick colony, streak at marked position on a new plate and solute remaining cells on the tip in the PCR tube with reaction mixture, cultivate if insert is of correct size
Gel electrophoresis for control of fragment size
Dephosphorylation
Reaction mix
30 µL purified vector DNA
4 µL 10x SAP buffer
2 µL SAP
4 µL H2O
Protocol
Incubate for 30 min at 37 °C
Inactivate for 15 min at 65 °C
DNA Purification
We used the following kits for DNA purification. For each kit the specific protocol is described seperately:
Thermo Scientific GeneJET Gel Extraction Kit
Note: all centrifugations should be carried out in a table-top microcentrifuge at > 12000 x g/10.000-14.000 rpm
Preparation
Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA band as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL tube. Record the weight of the gel slice. Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
Add 1:1 volume of Binding Buffer to the gel slice (volume:weigt)
Incubate the gel mixture at 50-60 °C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel slice is completely dissolved. Vortex the gel buffer mixture briefly before loading on the column.
Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
Optional: Use this step only when DNA fragment is <500 bp or >10 kb long.
If the DNA fragment is <500 bp, add 1:2 volume of 100 % isopropanol to the solubilized gel solution. Mix thoroughly.
If the DNA fragment is >10 kb, add 1:2 volume of water to the solubilized gel solution. Mix thoroughly.
Binding of DNA
Transfer up to 800 µL of the solubilized gel solution (from step 3/4) to the GeneJET purification column. Centrifuge 1 min at 11,000 x g. Discard the flow-through and place the column back into the same collection tube. Note: If the total volume exceeds 800 µL, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow through after each spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel slice per column.
Add 100 µL of Binding Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Washing
Add 700 µL of Wash Buffer to the GeneJet purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. (We altered the protocol to to wash steps a 350 µL optional)
Centrifuge the empty GeneJet purification column for an additional 1 min to completely remove residual wash buffer
Elution
Transfer the GeneJET purification column into a clean 1.5 mL microcentrifuge tube. Add 50 µL of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 minute. Note: for low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended. If DNA fragment is >10 kb, preheat Elution Buffer to 65 °C before applying to column. If the elution volume is 10 µL and DNA amount is <5 µg, incubate column for 1 min at room temperature before centrifugation.(We altered the protocol to 30 µL Elution Buffer preheated to 60 °C and 1 min incubation before centrifuging for all sizes.)
Discard the GeneJet purification column and store the purified DNA at -20 °C.
Qiagen QIAquick Gel Extraction Kit (Centrifugation)
Note: All centrifugation steps are carried out at 17.900 x g (13.000 rpm) in a conventional table-top microcentrifuge
Preparation
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume gel (100 mg gel - 100 µL). The maximum amount of gel per spin column is 400 mg. For >2 % agarose gels, add 6 volumes Buffer QG
Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). Vortex the tube every 2-3 minutes to help dissolve the gel. After the gel slice has dissolved completely, check that the color of the mixture is yellow(similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.0, ad mix. The mixture turns yellow.
Binding of DNA
Add 1 volume isopropanol to the sample and mix.
Place a QIAquick spin column in a provided 2 mL collection tube. To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min until all the samples have passed through the column. Discard the flow-through and place the QIAquick column back into the same tube. For sample volumes >800 µL, load and spin again.
Washing
If DNA will subsequently be used for sequencing,
in vitro
transcription or microinjection, add 500 µL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube
To wash, add 750 µL Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. If the DNA will be used for salt-sensitive applications (sequencing, blut-ended-ligation) let the column stand 2-5 minutes after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 mL collection tube for 1 min to remove residual wash buffer
Place QIAquick column into a clean 1.5 mL microcentrifuge tube
Elution
To elute DNA, add 50 µL Buffer EB (10 mM TRIS-HCl, pH 8.5) or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µL Buffer EB to the center of the QIAquick column, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time up to 4 min can increase the yield of purified DNA.
If the purified DNA is to be analyzed on a gel, add 1 volume of loading dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.
Analytik Jena InnuPREP Gel Extraction Kit
Preparation
Excise the DNA fragment from the agarose gel with a sharp scalpel. Note: minimize the agarose gel slice. Check the weight. Do not use more than 300 mg gel slice for one spin filter.
Transfer the gel slice into a 1.5 mL or 2.0 mL reaction tube and add the 650 µL Gel Solubilizer.
Incubate for 10 min at 50°C until the agarose gel slice is completely dissolved.
Binding of DNA
Add 50 µL Binding Optimizer and mix the suspension by vortexing or piepetting sometimes up and down.
Apply the sample onto the spin filter (green) located in a 2.0 mL Receiver Tube. Close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube.
Washing
Open the Spin Filter and add 700 µL Washing Solution LS, close the cap and centrifuge at 10.000 x g (12.000 rpm) for 1 min. Discard the filtrate and re-use the Receiver Tube. Place the Spin Filter back into the 2.0 mL Receiver Tube. Repeat the entire step.
Centrifuge at max speed for 2 min to remove all traces of ethanol. Discard the 2.0 mL Receiver Tube.
Elution
Place the Spin Filter into a 1.5 mL elution Tube. Carefully open the cap of the Spin Filter and add 30-50 µL Elution Buffer (optionally preheated to 50°C). Incubate at room temperature for 1 min. Centrifuge at 6.000 x g (8.000 rpm) for 1 min. A second elution step will increase the yield of extracted DNA. Note: The DNA can be eluted with a lower or a higher volume of Elution Buffer. (...) Store the extracted DNA at +4 °C. For long time storage placing at -20°C us recommended
Promega Wizard SV Gel and PCR CleanUp System
Gel Slice/PCR product preparation
Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 mL microcentrifuge tube.
Add 10 µL of Membrane Binding solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is copletely dissolved.
Alternative for PCR amplifications: Add an equal volume of Membran Binding Solution to the PCR amplification
Binding of DNA
Insert SV Minicolumn into Collection Tube.
Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 min.
Centrifuge at 16.000 x g for 1 minute. Discard flow through and reinsert Minicolumn into Collection Tube.
Washing
Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16.000 x g for 1 minute. Discard flow-through and reinsert Minicolumn into Collection Tube.
Repeat previous Step with 500 µL Membrane Washing Solution. Centrifuge at 16.000 x g for 5 minutes.
Empty the Collection Tube and recentrifuge the column assembly for 1 minute with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.
Elution
Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge tube.
Add 50 µL of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 minute. Centrifuge at 16.000 x g for 1 minute.
Discard Minicolumn and store DNA at 4 °C or -20°C.
Gibson Assembly
Modified from Gibson et al. (2009)
This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5' exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
Preparation of DNA molecules for in vitro recombination
Generate the complenentary sequence overlaps by PCR using the Phusion DNA-polymerase. If necessary add 5 M Betain in the reaction mix by reducing the amount of H2O to decrease the number of false PCR products.
Seperate the PCR product of interest by gel electrophoresis, extract the products via Gel Extraction .
In vitro recombination
Preparing assembly mixture
320 µL 5x isothermal reaction buffer
0.64 µL of 10 U ml-1 T5 exonuclease (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U mL-1 T5 exonuclease)
20 µL of 2 U mL-1 Phusion DNA polymerase
160 µL of 40 U mL-1 taq DNA ligase
add ddH2O up to a final volume of 1.2 mL
Aliquote 15 µL of the assembly mixture and store at -20 °C
That 15 µL assembly mixture on ice until use
Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each fragment).
Incubate the mixture at 50 °C for 15 to 60 min with 60 min being optimal.
Transformation via heat shock or electroporation without cleaning up the assembly product.
PCR
We used the following polymerases. The master mix ingredients and standard protocols are described for each specific enzyme separately:
Phusion
Phusion
Phusion master mix (25 µL)
5 µL 5 x HF-buffer
0.5 µL 10 mM dNTPs
Up to 20 ng*µL -1template
0.25 µL Phusion DNA polymerase
0.25-0.5 µL primer 10 µM each
0.75 µL DMSO optional
Ad 25 µL dH2O
Phusion standard protocol
Initial denaturation: 98°C - 60 s
30-35 cycles of
Denaturation: 98 °C - 20 s
Annealing: 50-70 °C - 20 s
Elongation: 72 °C - 15 s/kb
Final elongation: 72 °C - 4 min
Storage: 6 °C
Q5
Q5
Q5 master mix (25 µL)
5 µL 5 x Q5 buffer
0.5 µL 10 mM dNTPs
Up to 50 ng*µL-1template
0.1 µL Q5 DNA polymerase
0.5-1 µL primer 10 µM each
Ad 25 µL dH2O
Q5 standard protocol
Initial denaturation: 98°C - 30 s
30-35 cycles of
Denaturation: 98 °C - 20 s
Annealing: 50-75 °C - 30 s
Elongation: 72 °C - 20 s/kb
Final elongation: 72 °C - 5 min
Storage: 4 °C
KOD
KOD
KOD master mix (50 µL)
5 µL 10 x KOD Hot Start DNA polymerase buffer
3 µL 25 mM MgSo4
1 µL 10 mM dNTPs
1 µL KOD hot start DNA polymerase
1.5 µL primer 10 µM each
10 ng*µL-1 template
Ad 50 µL dH2O
KOD standard protocol
Initial denaturation: 95°C - 120 s
30-35 cycles of
Denaturation: 95 °C - 20 s
Annealing: 50-70 °C - 20 s
Elongation: 70 °C - individual elongation time
Calculation elongation time
<0.5 kb: 10 s/kb
0.5-1 kb: 15 s/kb
1-3 kb: 20 s/kb
>3kb: 25 s/kb
Final elongation: 70 °C - 4 min
Storage: 6 °C
Generating competent cells
We generated chemocompetent, ultra-chemocompetent and electrocompetent E. coli using the following protocols:
Chemo competent cells
Preparation
Cool 5 50 mL Falcons
Cool TSS buffer
Cool centrifuge
Protocol
Inoculate 2 x 3 mL LB with bacterial stock over night at 37 °C and 200 rpm
Inoculate 1 x 250 mL LB with the over night cultures in 1 L flask at 37 °C and 140 rpm
Incubate to OD600 0.4-0.6
Devide the culture into 5 cooled 50 mL falcons and incubate on ice for 10 min
Onwards all steps at 4 °C
Centrifuge at 3000 rpm, 4 °C, 10 min
Discard supernatant
Pipette remaining media out carefully
Add 10% volume cold TSS buffer of original cultur, about 1-2 mL
Resuspend pellet
Aliquot 100 µL and drop in liquid nitrogen immediatly
Store at -80 °C
Electro competent cells
Preparation
Autoclave 550 mL LB-Medium
Cool 1 L bidest. H2O
Cool 50 mL 10% glycerine
Cool 10 50 mL Falcons
Protocol
Inoculate 2 x 3 mL LB with bacterial stock, incubate over night at 37 °C and 200 rpm
Inoculate 2 x 250 mL LB with the over night culture in 1 L flask at 37 °C and 140 rpm
Cultivate until OD6000.4-0.6
Store on ice for 15-30 min
Onwards all steps at 4 °C
Divide the cultures into the pre-cooled 50 mL Falcons and centrifuge at 4000 rpm, 4 °C, 15 min with slow acceleration and decceleration
Discard supernatant
Resuspend in 5 mL cooled bidest H2O shaking gently
Pool two suspensions, add bidest. H2 to 50 mL and centrifuge again at 4000 rpm, 4°C, 15 min
Discard supernatant
Resuspend pellet in 5 mL cooled bidest H2O
Add bidest H2O up to 50 mL and centrifuge again at 4000 rpm, 4 °C, 15 min
Discard supernatant
Resuspend in 5 mL cooled 10% glycerine
Transfer suspensions in two 50 mL falcons and centrifuge again at 4000 rpm, 4 °C, 15 min
Discard supernatant
Add 1 volume similar to the pellet of 10% glycerine and resuspend
Aliquot in 50 µL and freeze in liquid nitrogen immediately
Store at -80 °C
Ultra chemo competent cells
Preparation
Inoculate 10 mL LB medium in a 100 mL flask with 50 µL cells from glycerol culture, e.g. E. coli DH5a MCR
Incubate over night at 37 °C and 180 rpm
Protocol
Precool centrifuge and media
Inoculate 250 mL SOB-Medium in a 1 L flask with 2.5 mL of the preculture
Incubate at 19 °C  and 180 rpm up to OD600 0.5 (about 24-36 hours)
Cool flask 10 min in ice water
Onwards all steps at 4 °C
Devide the culture in 5 50 mL cooled falcons
Centrifuge 5 min at 4 °C and 5000 x g
Discard supernatant
Resuspend in each 6 mL icecold TB Buffer
Incubate 10 min on ice
Centrifuge 5 min at 4 °C and 5000 x g
Discard supernatant
Resuspend in each 6 mL icecold TB Buffer
Add 420 µL DMSO and vortex immediately
Aliquot 150 µL and freeze in liquid nitrogen immediately
Store at - 80 °C
Plasmid Isolation
For Plasmid Isolation we used the following kits. The methods are described seperately:
PureYieldTM Plasmid Miniprep System (Promega)
Prepare Lysate
Add 600 µL of bacterial culture to a 1.5 mL microcentrifuge tube
Alternative for larger cultures: Centrifuge 1.5 mL bacterial culture for 30 s at maximum speed in a microcentrifuge. Discard the supernatant. Add an additional 1.5 mL bacterial culture to the same tube and repeat previous step. Add 600 µL of TE buffer or water to the pellet, and resuspend completely.
Add 100 µL of Cell Lysis Buffer (blue), and mix by inverting the tube 6 times.
Add 350 µL of cold (4-8 °C) Neutralization Solution and mix thoroughly by inverting.
Centrifuge at maximum speed in a microcentrifuge for 3 minutes.
Transfer the supernatant (about 900 µL) to a PureYieldTM Minicolumn without disturbing the cell debris pellet
Place the minicolumn into a Collection Tube and centrifuge at maximum speed in a microcentrifuge for 15 s
Discard the flowthrough, and place the minicolumn into the same Collection Tube.
Wash
Add 200 µL of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 s
Add 400 µL of Column Wash Solution (CWC)to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds
Elute
Transfer the microcolumn to a clean 1.5 mL microcentrifuge tube, then add 30 µL of Elution Buffer or nuclease-free water directly to the minicolumn matrix. Let stand for 1 minute at room temperature
Centrifuge for 15 seconds to elute the plasmid DNA. Cap the microcentrifuge tube, and store eluted plasmid DNA at -20°C
Thermo Scientific GeneJET Plasmid Miniprep Kit (Centrifugation)
All purification steps should be carried out at room temperature.
All centrifugations should be carried out in a table-top microcentrifuge at >12.000 x g (10.000-14.000 rpm)
Use 1-5 mL of E. coli culture in LB media for purification of high-copy plasmids.
Use up to 10 mL culture for low-copy plasmids
Protocol
Resuspend the pelleted cells in 250 µL of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cells clumps remain.Note: Ensure RNase A has been added to the Resuspension Solution.
Add 250 µL of the Lysis Solution and mix throughly by inverting the tube 4-6 tines until the solution becomes viscous and slightly clear. Note: Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 4 min to avoid denaturation of supercoiled DNA.
Add 350 µL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. Note:It is important to mix throughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.
Centrifuge for 5 min to pellet cell debris and chromosomal DNA
Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate.
Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Add 500 µL of the Wash Solution (diluted with ethanol prior to use) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube.
Repeat the wash procedure using 500 µL of the Wash Solution
Discard the flow-through and centrifuge an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.
Transfer the GeneJET spin column into a fresh 1.5 mL microcentrifuge tube. Add 50 µL of the Elution Buffer to the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. incubate for 2 min at room temperature and centrifuge for 2 min.
Discard the column and store the purified plasmid DNA at -20°C.
Analytik Jena innuPREP Plasmid Mini Kit (Centrifugation)
Starting material: 0.5-5.0 mL
Pellet cells
Centrifuge: max speed, 1 min
Discard supernatant completely
Resuspend cells
Add 250 µL Resuspension Buffer
Vortex
Lysis (do not vortex)
Add 250 µL Lysis Buffer
Mix: invert tube 6-8 times
Lysis time: <5 min
Neutralization
Add 350 µL Neutralization Buffer
Mix: invert tube 6-8 times
Centrifuge max speed, 8 min
Binding of DNA
Add Spin Filter to Receiver Tube
Add clarified sample Spin Filter
Centrifuge at 11.000 x g (12.000 rpm) 1 min
Washing
Add 500 µL Washing Solution A
Centrifuge 1 min at 11.000 g (12.000 rpm)
Add 700 µL Washing Solution B
Centrifuge 11.000 x g (12.000 rpm) 1 min
Remove ethanol
Discard filtrate
Add Spin Filter to Receiver Tube
Centrifuge Max speed, 2 min
Elution
Add Spin Filter to a 1.5 mL reaction tube
Add 50-100 µL Elution Buffer P
Incubation 1 min at room temperature
Centrifuge at 11.000 x g (12.000 rpm) 1 min
QIAprep Spin Miniprep Kit (Qiagen)
All centrifugation steps are carried out at 13.000 rpm (17.900m x g) in a conventional table-top microcentrifuge
Pellet 1-5 mL bacterial overnight culture by centrifugation at >8.000 rpm (6.800 x g) for 3 min at room temperature (15-25 °C)
Resuspend pelleted bacterial cells in 250 µL Buffer P1 and transfer to a microcentrifuge tube.
Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for more than 5 min . If using LyseBlue reagent, the solution will turn blue.
Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 minutes at 13.000 rpm (17.900 x g) in a table-top microcentrifuge.
Apply the supernatant to the QIAprep spin column by decanting or pipetting. Centrifuge for 30-60 s and discard the flow-through.
Wash the QIAprep spin column by adding 500 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through
Wash the QIAprep spin column by adding 750 µL Buffer PB. Centrifuge for 30-60 s and discard the flow-through. Transfer the QIAprep spin column to the collection tube.
Centrifuge for 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 µL Buffer EB(10mM Tris-Cl, pH8.5) or water to the center of the QIAprep spin column, let it stand for 1 min, and centrifuge for 1 min.
His tag purification
Protocol (Macherey-Nagel Purification of His-tag proteins Kit) Protino® Ni-TED 1000 Packed Columns Kit
Preparation of cleared lysates under native conditions
That the cell pellet from an E. coli expression culture on ice (if frozen). Resuspend 1 g of pelleted, wet cells in 2-5 mL LEW buffer, pipette up and down, or use stirring until complete resuspension without visible cell aggregates. Perform this step on ice
Add lysozyme to a final concentration of 1 mg/mL. Stir the solution on ice for 30 min
Sonicate the suspension on ice according to the instructions provided by the manufacturer (e.g. use 10 x 15 s bursts with a 15 s cooling period between each burst). Carefully check samples´ appearance after sonication. If the lysate is still viscous from incomplete fragmentation of DNA, add 5 µg/mL DNase I and stir on ice for 15 min.
Centrifuge the crude lysate at 10.000 x g for 30 min at 4 °C to remove cellular debris. Carefully transfer the supernatant  to a clean tube without disturbing the pellet. If the supernatant is not clear, centrifuge a second time or filter through a 0.45 µm membrane (e.g. cellulose acetate) to avoid clogging of the IMAC column with insoluble material. Store supernatant on ice.
Purification protocol
Column equilibration: Equilibrate Protino Ni-TED/IDA packed columns with 1 x LEW Buffer. Allow the column to drain by gravity (2.0 mL). Protino Ni-TED/IDA1000 packed columns are designed to fit into most 15 mL conical centrifuge tubes for convenient fraction collection
Binding: Add the cleared lysate to the pre-equilibrated column and allow the column to drain by gravity
Washing: Wash the column with 1 x LEW Buffer. Allow the column to drain by gravity (2 x 2.0 mL)
Elution: Wlute the polyhistidine-tagged protein in a new collection tube by adding 1 x Elution Buffer. Allow the column to drain by gravity (3 x 1.5 mL). Use protein assay and/or SDS-PAGE analysis to determine which fraction(s) contain(s) the majority of the polyhistidine-tagged protein.
Standard BioBrick Assembly
Modified from Silver Lab
Minimum size of the BioBricks for this methods is 150 bp, whereas it should be at least 500 bp bigger or smaller than the used backbone. The BioBricks, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other BioBrick called vector. The methods differ in prefix and suffix insertion.
Suffix insertion
Digestion of inserted Biobrick
Acquire > 700 ng DNA (10 µL volume)
Add 1 µL 10 x NEBuffer
Add 0.5 µL XbaI
Add 0.5 µL PstI
Digest 1 h at 37 °C
Inactivate 20 min at 80 °C
Clean up via gel electrophoresis, cut and extract from gel
Digestion of the vector
Acquire about 700 ng DNA (10 µL volume)
Add 1 µL 10 x NEBuffer 2.1
Add 0.5 µL SpeI
Add 0.5 µL  PstI
Digest 1 h at 37 °C
Inactivate 20 min at 80 °C
Add 1 µL Antarctic Phosphatase (AP) and 1.2 µL 10 x AP reaction buffer
Incubate 1 h at 37°C
Clean up with PCR clean up kit
Ligation
Mix 50-200 ng vector with 3-10 fold molar access of insert
Add 2 µL T4-Ligase-Buffer, 1 µL T4-DNA-Ligase
Incubate for a minimum 20-30 min at room temperature (we usually ligate over night)
Inactivate 5 min at 70 °C
Store at -20 °C or transform
Prefix insertion
Follow protocol of Suffix insertion, replace the restriction enzymes as following:
Digest insert with EcoRI and SpeI
Digest vector with EcoRI and XbaI
Transformation via Electroporation
Thaw 50 µL electrocompetent E. coli cells on ice, dilute with icecold 50 µL glycerine (10%) if necessary
Add 0.5-5 µL plasmid to 50 µL electrocompetent cells
Store cells on ice for 1 minute
Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ω
Transfer transformation reaction to 450 µL SOC-Medium and incubate 1 h at 37 °C
Plate on selective LB-Medium
Incubate over night at 37 °C
Transformation via Heat Shock
Thaw 100 µL chemo competent E. coli cells on ice
Add 0.5-5 µL plasmid to 100 µl chemocompetent cells
Store cells on ice for 10-30 min on ice
Heat shock for 90 seconds at 42 °C
Store reaction on ice for 60 seconds
Optional: Preheat SOC medium to 37 °C
Transfer reaction to 1 ml SOC medium and incubate at 37 °C for at least 1 hour
Centrifuge 3 minutes at 12000 rpm and plate on selective LB medium
Incubate at 37 °C over night
SDS-PAGE
Pouring the polyacrylamide gel:
For 2 resolving gel (14 %) :
4.7 mL  Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)
2.7 mL H2O
2.5 mL 0.5 M TRIS-HCl, pH 6.8
0.1 mL 10% SDS
Add 50 µL 10 % ammonium persulfate and 10 µL TEMED and mix.
Pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel.
Layer isopropanol on top of the gel.
Leave the resolving gel at room temperature for 30-45 minutes to polymerize.
Remove isopropanol and wash the surface of the separating gel with H2O. Wait until the surface is dry.
For 2 stacking gel (4 %) :
1.3 mL Bisacrylamid/Acrylamid (0.8 %; , 30 %;, at the ratio of 37.5:1)
6.1 mL H2O
2.5 mL 0.5 M TRIS-HCl, pH 6.8
0.1 mL 10% SDS
50 µL 10 % ammonium persulfate
5 µL TEMED
Add 50 µL 10 % ammonium persulfate and 5 µL TEMED to each aliquote and mix.
Insert comb without getting bubbles stuck underneath.
Leave the gel at room temperature for 30-45 minutes to polymerize.
For storage:
Remove sealing and store the gel wrapped in moistened paper towel at 4 °C.
Preparing the sample:
Mix your protein mixture 3:1 with PBJR buffer (15 µL protein solution + 5 µL PBJR buffer )
Heat for 5 minutes at 95 °C
Running the gel:
Remove sealing, put the polymerized gel into gel box and pour 1x SDS-PAGE running buffer into the negative and positive electrode chamber.
Remove comp without destroying the gel pockets.
Pipet the SDS running buffer in the gel pockets up and down for flushing the gel pockets.
Pipet slowly 20 µL of the sample into the gel pockets.
Make sure to include at least one lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample.
Connect the power lead and run the gel with 180 V until the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm.
Staining the polyacrylamide gel (Colloidal Coomassie Brilliant Blue staining):
After finishing the SDS-PAGE remove gel from gel casting form and transfer it into a box.
Add 100 mL of the Colloidal Coomaassie Brilliant Blue staining solution to your polyacrylamid gel.
Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation.
Remove the staining solution.
Wash the gel with destaining buffer .
Incubate the gel in H2O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one.
MALDI-TOF/TOF sample preparation
Tryptic digest of gel lanes for analysis with MALDI-TOF:
Be careful that skin scales or hair do not contaminate your sample. So wear hand gloves and tie up your hair.
Reaction tubes have to be cleaned with 60 % (v/v) CH3CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively, microtiter plates from Greiner® (REF 650161) can be used without washing. If you work with reaction tubes from Eppendorf, you do not need the wash step either.
Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes. If necessary, cut the parts to smaller slices
Gel slices should be washed two times. Therefore add 200 µL 30 % (v/v) acetonitrile in 0.1 M ammonium hydrogen carbonate each time and shake lightly for 20 minutes till the gel slices are destained. Remove supernatant and discard to special waste
Dry gel slices at least 30 minutes in a Speedvac.
Rehydrate gel slices in 15 µL trypsin solution followed by short centrifugation.
Trypsin-solution : 1 µL trypsin + 14 µL 10 mM NH4HCO3
For this solution solubilize lyophilized trypsin in 200 µl of provided buffer and activate Trypsin for 15 minutes at 30 °C. For further use it can be stored at -20 °C.
Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C overnight
Dry gel slices at least 60 minutes in a Speedvac.
According to the size of the gel slice, add 5 - 20 µL 50 % (v/v) ACN / 0.1 % (v/v) TFA
Samples can be used for MALDI measurement or stored at -20 °C
Preparation and Spotting for analysis of peptides on Bruker AnchorChips:
Spot 0.5 - 1 µL of sample aliquot
Add 1 µL HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip. Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area.
Dry the prepared spots at room temperature
Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “ Peptide Calibration Standard II ”, Part number #222570), add 125 µL of 0.1 % TFA (v/v) in 30 % ACN to the vial. Vortex and sonicate the vial.
Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots.
Analyze samples in ultrafleXtreme by Bruker Daltonics.
Protocol used for BioLector measurment
Preparation
Prepare overnight cultures of the strains you want to test with the BioLector.
Materials
Growth media (LB)
Overnight cultures of tested strains
Gas Permeable Seal
48-well plates
Procedure
Measure OD of the overnight cultures.
Centrifuge the overnight cultures and discard the supernatant.
Add LB-medium till an o.D. of 0,1.
Add 1 mL bacteria suspension in every well of the 48-wellplate.
Stick the Adhesive Gas Permeable Seal to the plate and make sure it is stuck on tightly.
Apply the Adhesive Seal for Evaporation Reduction on top of the Gas Permeable Seal.
Put the BioLector plate into the BioLector in the specified slot.
Set the temperature and moisture to be held during the measurement using the Start Assistant in the BioLector.determine the used filters for biomass measurement 620nm and for fluorescence measurement 486nm (ex) and 510 (em).
Run the BioLector for circa 41 hours.
Use the following settings:
C-Lysis
Preparation of lysis buffer
2 volumes 98 % glycerol (v/v)
1 volume 0.44 M NaOH
1 volume 5 % SDS (w/v)
Protocol
Resuspend cells in 20 µL H2O
Add 5 µL Loading buffer
Seperate in agarose gelelectrophoresis with supercoiled marker
Elute from gel
CFPS protocols
Amino acid preparation
According to protocol from Caschera and Noireaux 2015b . Weigh all aminoacids seperatly into microcentrifuge tubes. Add 500 µl of 5 M KOH to each amino acid. Solubilization is achieved via multiple inverting and, if necessary, vortexing. Especially tyrosine takes a while, and is a suspension rather than a solution. Stock solutions are afterwards stored at -20 °C. Note: Caschera and Noireaux say these stock solutions can only be stored a few weeks, however, we did not see a loss in performance after more than 3 months (16 weeks).
Molecular weight
mass to weigh (in mg) to obtain desired stock-solution
concentration in stock solution in mM
Alanine
89.09
182
4089
Arginine
174.20
202
2314
Asparagine-Monohydrate
150.14
282
3759
Aspartic acid
133.10
250
3752
Cysteine
121.16
149
2465
Glutamic acid
147.13
269
3655
Glutamine
146.15
183
2501
Glycine
75.07
158
4210
Histidine
155.15
255
3281
Isoleucine
131.18
247
3765
Leucine
131.18
167
2549
Lysine
146.19
175
2392
Methionine
149.21
186
2491
Phenylalanine
165.19
142
1716
Proline
115.13
224
3883
Serine
105.90
209
3953
Threonine
119.12
229
3853
Tryptophan
204.23
170
1661
Tyrosine
181.19
217
2396
Valine
117.15
152
2595
Combine stock solutions to an amino acid mixture like depicted below. Add water to 4 mL and 110 µL of glacial acetic acid to adjust pH to about 6.5. You can add less glacial acetic acid and check the pH by pipetting some µL of the solution on pH paper. We observed slighty better performance for our reactions at pH = 7.9. Aliquot (do not forget to mix properly!) and flash-freeze in liquid nitrogen, store at -80 °C.
Volume (in µL) for mixture
Concentration in mixture (without water and glacial acetic acid added)
concentration in final mixture
Alanine
13.6
146
13.56
Arginine
22.2
135
12.53
Asparagine-Monohydrate
13.6
134
12.44
Aspartic acid
13.6
134
12.44
Cysteine
22.2
143
13.28
Glutamic acid
13.6
130
12.07
Glutamine
22.2
145
13.46
Glycine
13.6
150
13.93
Histidine
13.6
117
10.86
Isoleucine
13.6
134
12.44
Leucine
22.2
148
13.74
Lysine
22.2
139
12.91
Methionine
22.2
145
13.46
Phenylalanine
34
153
14.21
Proline
13.6
138
12.81
Serine
13.6
141
13.09
Threonine
13.6
137
12.72
Tryptophan
34
148
13.74
Tyrosine
22.2
139
12.91
Valine
22.2
151
14.02
sum
381.6
2807
260.62
Cofactor premix
Stock solutions needed:
2 M HEPES
174 mM NAD
33.9 mM folinic acid
65 mM coenzyme A (CoA)
50 mg/mL E. coli tRNA (Roche)
200 mM putrescine
1.5 M spermidine
Combine stock solutions to a create cofactor premix that is 20x final reaction concentration, depicted in the following list
1 M HEPES
6.6 mM NAD
1.4 mM folinic acid
5.4 mM coenzyme A (CoA)
4 mg/mL E. coli tRNA (Roche)
20 mM putrescine
30 mM spermidine
Aliquot and flash-freeze. During summer we prepared this premix more than one time, and we observed that the age of the cofactor premix does not affect the cell free reaction as long as aliquots are stored at -80 °C.
Cell harvest
The following harvest protocol mainly orientates to the procedures in Sun et al. 2013 and Kwon and Jewett 2015 .
Gather all materials needed: Harvest tubes and falcon tubes, washing buffer, DTT aliquot, liquid nitrogen, and a lot of ice. Start the harvest when E. coli culture reaches mid- to late exponential growth phase. For the E. coli we used in our experiments and cultivations, the mid- to late exponential growth phase was reached at an OD600 of 3-4 (see growth curves in notebook section ).
harvest protocol - keep everything on ice between the steps!
Transfer culture into prechilled and weighted harvest tubes or falcons
Centrifugate: 5000x g, 4 °C, 15 min. While centrifugating, add DTT to S30 buffer to a final concentration of 2 mM
Discard supernatant and weigh pellets
Add about 10 mL of S30 washing buffer per 1 g of wet cells and resuspend cells by vortexing and vigourous shaking. For us, cycles of 15 s vortexing/shaking and 30 s resting on ice worked well.
Centrifugate: 5000x g, 4 °C, 12 min
Discard supernatant
Repeat steps 4 to 6 two times
Centrifuge a last time at 5000x g, 4 °C, for 5 min and remove residual washing buffer by pipetting.
Flash freeze pellets in liquid nitrogen and store at -80 °C, unless you want to proceed with sonication directly.
Sonication
The following strategy can be pursued if there is no access to a sonifier which can display its energy output in Joule per second or alike.
First, prepare multiple reference samples which are absolutely identical (see below for further requirements). Right before sonication, ensure the temperature of the liquid. Sonicate for a defined time interval. When sonication has ended, measure the temperature of the liquid again without any delay.
Equation to calculate the necessary sonication time for a given setup. P: Power in W or J/s, respectively; V: sonicated volume in L; c: specific heat capacity of water (at room temperature); Δϑ: temperature difference between starting temperature and temperature after heating in K; Δt: time difference between both temperature measurements.
Perform at least triplicates for every interval. To calculate the power P of your sonifier in Joule per seconds and thereby calculate the necessary sonication time for a given setting, use the nearby equation. You need to consider the following:
The Amplitude of the sonifier must be kept constant during experiment to eliminate the influence of viscosity or heating on the result
The starting temperature must remain the same in all reference measurements
The liquid as well as the vessel used in the reference experiment must be similar to the ones in the intended experiments
For our setting (see below), a sonication time of 290 s was appropriate for a total energy input of 800-900 J.
Our lysis procedure is described in the following section
Sonication of the pellet from Cell harvest with the HF-Generator GM 2070 in combination with UW 2070, SH 70G and MS73; setted at 70% amplitude and no cycles
clean the sonifier tip with 70% (v/v) EtOH prior to usage
resuspend the pellet in 1 mL S30 buffer per gramm pellet
aliquot the cell suspension à 1500 µL in 2 mL reaction tubes
for each sonication procedure, place one tube in an ice bath and place the sonifier tip in the suspension, so that it is briefly at 1/3 height of the tube
sonicate for 10 s
wait 10 seconds
repeat the previous two steps 28 times
Centrifuge for 10 min at 12000x g and 4 °C.
Carefully transfer the supernatant (cell extract) into a new tube. Take care to not disturb the pellet.
Consider to conduct a run-off reaction at 37 °C, slightly shaking at 250-300 rpm, in order to digest endogenous DNA and mRNA. When using the extract for the first time, perform various run-off reactions, e.g. 20 min to 80 min in 10 min intervals. For us, 30 min gave the best results.
Centrifuge a second time at 10000x g, 4 °C, 10 min after performing the run-off reaction.
Transfer the supernatant (cell extract) into new tubes (we used 50-100 µL aliquots). Flash-freeze in liquid nitrogen and store at -80 °C.
An extensive analysis on sonication and run-off reaction parameters can be found in Kwon and Jewett 2015 .
Conduction of a CFPS reaction
To prevent RNase contamination, work with gloves all the time. It is strongly recommended to use filter tips or low-stick tips, respectively, and recently calibrated pipettes in order to work precisely with very low volumes. Do triplicates or quadrupates for a reliable data evaluation.
Materials needed
reagents
cell extract
cofactor premix
amino acids
NTPs
energy source and corresponding cofactors
DNA template
Mg- and K-glutamate solutions
RNase free water
any other additives
other materials
Depending on what protein is to be produced, a device capable of measuring fluorescence and/or luminescence. In our case this was a TECAN infinite M200 plate reader.
Depending on what device you use (for example plate reader), a 386 black well plate and transparent sealing foil
RNase free microcentrifuge tubes
70% EtOH for RNase decontamination
Procedure
Prepare a sheet in silico that contains all information you need for pipetting. For our purpose we created a suitable excel spreadsheet based on the one from Sun et al., 2013
Clean your working place thorough with 70% EtOH. Prepare RNase free micocentrifuge tubes. Let all aliquots thaw on ice.
Start to prepare your DNA samples by adding water, DNA, glutamate salts and any other item of choice to microcentrifuge tubes at room temperature.
Combine aliquots from 2. to a reaction buffer in a microentrifuge tube. Shortly vortex after the addition of each reagent and keep on ice.
At the latest after your reaction buffer is ready, prepare your measurement device (software program, temperature and so on).
Prepare a master mix made of reaction buffer and E. coli cell extract according to your spreadsheet, shortly vortex and keep on ice for now. From now on, do not waste excessive time with dreaming but work eagerly so that, as long as your template is missing, no energy in the buffer is wasted by cell extract proteins!
Add the calculated master mix amount to each of your DNA sample after carefully mixing the master mix. Attention: The master mix is viscous, avoid bubbles by pipetting the last drop to the tube wall.
Shortly vortex each sample and centrifuge for 30 s at 10000 g at room temperature. Thereby any residual sample will be brought down, and bubbles can be reduced.
Run the reaction
in the microcentrifuge tubes or
in a multi-well plate by transferring the samples into wells, sealing with appropriate foil to prevent evaporation. Optional: If you have a plate centrifuge, you can minimize bubbles in the plate with a short centrifugation step prior to measurement (4000 g, 30 s according to Sun et al., 2013 )
Attention: Consider a volume loss during pipetting. We routinely performed reactions at a final scale of 15 µL, but in our spreadsheet we calculated with 15.75 µL.
Run the reaction at the desired temperature for a desired time. In most of our experiments we measured for 3 h and saw no further raise in fluorescence signl. In most cases this is 37 °C but optimal temperature varies depending on protein product. Some proteins are well produced over a wide range of temperatures, which is the case for sfGFP.
Other considerations
For best performance, a DNA plasmid of high purity and water of high quality are recommended. For sfGFP expression plasmids we found that plasmid mini-preparation kits, especially from Promega and Qiagen, provide sufficient DNA quality.
Shaking can be helpful, especially when oxygen is needed for the reaction (see Caschera and Noireaux 2015 ). In our Tecan measurements we shaked 5 s with 1 mm amplitude every time before fluorescence was measured.
PRIA protocols
Plasmid Repressor Interaction Assay (PRIA)
After every step the reaction tube is centrifuged with 1000 g for 1 minute.
The supernatant after centrifugation and every wash step (starting from the step when the protein is added to the agarose) is stored for analyzing the samples for protein and DNA amounts.
As negative controls no protein to the agarose were added.
Steps:
25 µL Ni-NTA agarose is put in a reaction tube. Then the sample is centrifuged.
The agarose is washed three times with Kpi buffer (Volume: 50 µL).
250 pmol protein in 20 µL Kpi buffer is added and incubated for 30 min. Then the sample is centrifuged.
The agarose is incubated three times in Kpi Buffer (Volume: 50 µL).Then the sample is centrifuged.
0.75 pmol plasmid (lacO) is mixed with 20 µL binding buffer (Incubation time: 15 min).
Unbound DNA is washed away three times with binding buffer (Volume: 50 µL, Incubation time: 15 min). Then the sample is centrifuged.
3x elution with analytes in binding buffer (Volume: 50 µL, Incubation time: 15 min)
Imidazol solution (50 µl) is used to release proteins from the agarose.
Water (volume: 10 µL) is mixed with agarose.
The DNA amount of the supernatant after centrifugation is analyzed via gel electrophoresis.
Electrophoretic Mobility Shift Assay (EMSA)
Required solutions:
5x EMSA buffer (100 mM Na2HPO4, 375 mM KCl, 25 % Glycerol, adjust ph to 8.0)
EMSA running buffer (20 mM Na2HPO4, adjust ph to 8.0)
3 % agarose gel prepared with EMSA running buffer
MgCl2: 25 mM
EDTA: 2,5 mM
BSA: 10 mg/ml
Dithiotreitol
salmon sperm
Cy3 labeled DNA
protein
Reaction:
0.05 pmol Cy3 labeled DNA
5 pmol purified protein of interest
4 µl 5x EMSA buffer
2 µl MgCl2 final concentration: 2.5 mM
0.8 µl EDTA final concentration: 0.1 mM
1 µl BSA final amount: 10 µg
1 µl salmon sperm final amount: 1 µg
optional: 5 mM DTT
mix all components of the reaction carefully and add up with water to 20 µl H2O.
Incubate at room temperature in the dark for 15 min.
Keep in mind that Cy3 is light sensitive, always keep your samples in the dark after addition of the DNA fragment.
Apply reaction to 3 % agarose gel.
Run gel at 80 V for approximately 80 min at 4 °C. (Adjust voltage and duration to size of your DNA fragement.)
Detect Cy3 with a fluorescence scanner. In our case it was the Typhoon 8600 variable Mode Imager.
Immobilization of dsDNA on paper
Dissolve 10 mg p-Phenylenediisothiocyanate either in 30 ml pure ethanol or in 500 µL dried DMSO and add up to 30 ml with water. (the drying is essential, perform it with 4 Angström molecular sieves at least 24 h)
Whatman filter paper is activated for 1-2 days by orbital shaking in the PDITC solution.
Wash paper twice with DMSO or water respectively. Dry it for approximately 5 min
Generate an aminolabeled DNA fragement
The optimal concentration required for the immobilization is 20 µM. You can achieve this through hybridization of 2 Primers in 50 mM sodium phosphate buffer, pH 8.5. Boil the primers at 98 °C for a few minutes and afterwrds slowly decrease the temperature.
Alternatively, you can perform an conventional PCR with one of the primers aminolabeled. We recommend labeling of the other oligo with a fluorescent dye. This enables detection of the immobbilized DNA on Scanners like the Typhoon or Ettan Dige.
Pipett 1-0.1 µL onto the paper and incubate it over night in an humidification chamber (avoid direct contact of the liquid with the paper)
Block the paper with 50 °C warm blocking solution (0.1 M Tris, 50 mM ethanolamine, pH 9.0)
wash the paper three times for 2 min with water and dry it by an airstream. (If you wish to immobilze ssDNA, add an washing step with 50°C 4xSSC buffer, 30 min.)
PicoGreen Assay
PicoGreen Assay
99 µl TE-buffer (10 mM Tris-HCl, 1mM EDTA, pH 7.5) are mixed with 1 µL sample solution in a well of a black 96 well plate
For the determination of the concentration, a standard row of the concentrations 1, 0.5, 0.25, 0.125 and 0.0625 and  0 µg/mL of lambda DNA is done.
PicoGreen solutions are diluted in a ratio 1/200 in TE-buffer.
100 µl diluted PicoGreen solution is mixed with the diluted sample.
The samples are measured in a Tecan Reader. They are excited at 480 nm and the fluorescence emission intensity is measured at 520 nm.
Measurement of Fluorescence in plate reader
You can detect immobilized DNA and your fusion protein by measuring the fluorescence.
You need Cy3- and amino-labeled DNA with an operator site for immobilization, a repressor fused to a fluorescence protein and cellulose on a black 96-well plate.
Excitation for Cy3: 545 nm, Emission for Cy3: 590 nm
For example: Excitation wavelength for sfGFP: 480 nm, Emission wavelength for sfGFP: 515 nm
The gain needs to be adapted.
After every step the supernatant is transferred into another 96-well plate for measurement.
The plate with the cellulose is dried after having taken out the supernatant and measured again.
Cellulose on the plate is prepared in a black 96-well plate.
DNA is immobilized on cellulose.
Protein in Kpi buffer (Concentration: 20 µg/mL, Volume: 25 µL) is added to the wells with immobilized DNA (Incubation time: 15 min).
The wells are washed with binding buffer (Volume: 200 µL, incubation time: 10 min).
100 µl analyte solution (e.g. 0.5 mM IPTG in binding buffer) is put in the well and shaken for 30 min.
2015 InterLab protocols
Growing Cells for Measurement
streak out 1 LB plate per device and control (for each biological replicate 1 plate)
incubate LB plates over night at 37°C (19 hours)
inoculate liquid cultures (100 mL flasks with 10 mL LB-medium)
incubate at 37 °C, 300 rpm, 17 hours
Plate Reader Measurement
measure the OD600 of the overnight cultures
dilute all samples to an OD600 of 0.5
re-measure the OD600, if it is within 5 % of 0.5, proceed
measure the samples with a plate reader (150 µl/ well, 96-well-plate, Excitation Wavelength : 475 nm, Emission Wavelength: 509 nm)
Media & Buffers
