Team:Bielefeld-CeBiTec/BiosensorDesignMotivation
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Biosensor Design
A handy tool for everyone
The term "biosensor" is widely used for sensors that are based on biological sensing elements. With these, the targeted detection of specific substances (referred to as analytes) is possible. Biosensors have the potential to be used for a wide range of analytical purposes ( Turner 2013 ). Furthermore, they can be cheaper, more sensitive and more specific than conventional detection methods like chemical sensors ( Kaur et al. 2015 ). Consequently, biosensors are a field of active research and especially popular in the iGEM competition ( Choffnes et al. 2011 ). Teams tried to use various different approach for implementing biosensors against toxic agents, hazardous chemicals, drugs, diseases and environmentally dangerous chemicals.
However, some issues remain: As great as the potentials of biosensors are, the hurdle of field applicability has in most cases not been overcome. This is because most biosensors make use of living, genetically modified microorganisms (GMOs) ( Daunert et al. 2000 , Lei et al. 2006 ).  The application of these whole-cell biosensors is often not possible due to legal issues and safety concerns, as it must be prevented that genetically modified organisms are released into the environment. Moreover, these sensors are often not very user-friendly, as they have a limited shelf life and their application outside the laboratory is complicated and requires special equipment. Another disadvantage of using living cells as a biosensor is that the diffusion of the analyte across the cell membrane is a limiting step, which results in a slower response time of the biosensor ( Strosnider 2003 , Yagi 2007 ).
Nevertheless, there is a great need for handy and cheap detection tools for toxic substances, which became clear to us when we talked to experts and did a survey in Bielefeld. In our eyes, biosensors are very promising candidates for solving severe problems all over the world. Therefore, we decided to develop biosensors that overcome the aforementioned shortcomings and are easily applicable in everyday life.
After intensive research, we decided that such a biosensor needs to be cell-free. This has a number of advantages, especially in terms of biosafety and the legal situation. To make the biosensor easily usable for everyone, we aimed at developing a simple test strip. Paper has emerged as a cheap and reliable material for biosensors. By freeze-drying biological materials on paper, their shelf life can be extended significantly ( Jokerst et al. 2012 , Pardee et al. 2014 ). For this reason, it is the ideal material for our biosensor.
Another challenge in the field of biosensors is multiplexing ( Chen and Rosen 2014 ). Often, it is not sufficient to test for one toxic substance to decide whether drinking water is safe or not, because most of the time several pollutants are present. Consequently, we wanted to build a modular biosensor that is easily extensible.
The basis of our biosensors is the interaction between repressor or activator protein and DNA. The principles are described in the following sections.
The repressor protein is a transcription factor that regulates the gene expression. It inhibits transcription upon binding the operator site, a DNA sequence for transcription regulation. The complex formation can only be broken if the target substances (referred to as “analyte”) is available. They can bind the repressor, which changes its conformation. The repressor protein is released and the transcription can start as the RNA polymerase is not hindered anymore.
The activator protein works on the opposite principle. It is also a transcription regulator and does not inhibit the transcription, but enables it. So by binding the analyte, the activator binds the operator site. This time, the RNA polymerase can start to transcribe the genes as the affinity of the RNA polymerase to the promoter is enhanced.
In short: We decided to design a cell-free, paper-based test strip in order to make biosensors applicable in everyday life.
References
Chen, Jian; Rosen, Barry P. (2014): Biosensors for inorganic and organic arsenicals. In: Biosensors 4 (4), S. 494–512. DOI: 10.3390/bios4040494.
Choffnes, Eileen R.; Pray, Leslie A.; Relman, David A. (2011): The science and applications of synthetic and systems biology. Workshop summary. Washington, D.C.: National Academies Press.
Daunert, Sylvia; Barrett, Gary; Feliciano, Jessika S.; Shetty, Ranjit S.; Shrestha, Suresh; Smith-Spencer, Wendy (2000): Genetically Engineered Whole-Cell Sensing Systems: Coupling Biological Recognition with Reporter Genes. In Chem. Rev. 100 (7), pp. 2705–2738. DOI: 10.1021/cr990115p.
Jokerst, Jana C.; Adkins, Jaclyn A.; Bisha, Bledar; Mentele, Mallory M.; Goodridge, Lawrence D.; Henry, Charles S. (2012): Development of a paper-based analytical device for colorimetric detection of select foodborne pathogens. In: Analytical chemistry 84 (6), S. 2900–2907. DOI: 10.1021/ac203466y.
Kaur, Hardeep; Kumar, Rabindra; Babu, J. Nagendra; Mittal, Sunil (2015): Advances in arsenic biosensor development--a comprehensive review. In Biosensors & bioelectronics 63, pp. 533–545. DOI: 10.1016/j.bios.2014.08.003.
Lei, Yu; Chen, Wilfred; Mulchandani, Ashok (2006): Microbial biosensors. In: Analytica chimica acta 568 (1-2), S. 200–210. DOI: 10.1016/j.aca.2005.11.065.
Pardee, Keith; Green, Alexander A.; Ferrante, Tom; Cameron, D. Ewen; DaleyKeyser, Ajay; Yin, Peng; Collins, James J. (2014): Paper-based synthetic gene networks. In Cell 159 (4), pp. 940–954. DOI: 10.1016/j.cell.2014.10.004.
Turner, Anthony P F (2013): Biosensors: sense and sensibility. In: Chemical Society reviews 42 (8), S. 3184–3196. DOI: 10.1039/c3cs35528d.
Strosnider, Heather (2003): Whole-Cell Bacterial Biosensors and the Detection of Bioavailable Arsenic.
Yagi, Kiyohito (2007): Applications of whole-cell bacterial sensors in biotechnology and environmental science. In Applied microbiology and biotechnology 73 (6), pp. 1251–1258. DOI: 10.1007/s00253-006-0718-6.
Our first biosensor design: CFPS
