Team:Bielefeld-CeBiTec/Notebook/CFPS
iGEM Bielefeld 2015
Toggle navigation
Project
Overview
Biosensor Design
Motivation
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosafety
Biosensor Targets
Motivation
Heavy Metals
Date Rape Drugs
Foundations
Output signal
Results
Overview
Design Evolution
Cell Free Protein Synthesis
Plasmid Repressor Interaction Assay
Biosensor Characterization
Heavy Metals
Date Rape Drugs
Output Signal Processing
ALL-IN-ONE Performance
Parts
Interlab Study
Achievements
Modeling
Overview
CFPS Model
Biosensor Model
Model Applications
Team
Team Members
Attributions
Bielefeld & University
Contact
Human Practices
Overview
Dual Use
Future Scenarios
Street Science
Students Academy
Press Review
Notebook
Lab Notebook
Protocols
Media & Buffers
Primers
Organisms
Partners
Sponsors
Partners
Collaborations
Cell-free Protein Synthesis
Expand all
Hide all
Week 1 (4/20/2015-4/26/2015)
04/22:
Transformation of electrocompetent E. coli KRX with BBa_K1365020 (sfGFP (Bs))
04/23:
preparing glycerol stock of KRX with BBa_K1365020
04/24-04/25:
Mini-prep of BBa_K1365020
Week 2 (4/27/2015-5/3/2015)
04/28:
Transformation of  electrocompetent E. coli KRX with BBa_I746909 (T7-promoter, RBS, sfGFP, double terminator). Before plating on LB(Cm), 50 µL of 200 g∙L-1 rhamnose-solution is added to the plate to induce T7-Polymerase. Inoculation of 10 mL preculture of simple KRX from plate. Overnight culturing, shaking vigorously at 220 rpm.
04/29:
colonies of KRX with BBa_I746909 look greenish! 2.2 mL Overnight cultures from KRX are used to inoculate 100 mL for start OD600=0.144. Measuring of growth curves for a first estimation of appropriate moment for cell harvest (see image below)
Growth curves (duplicate) from E. coli KRX at 37 °C in 100 mL 2xTY+P-medium. 1 L shaking flasks without baffles were used. Shaking at 200 rpm.
04/30:
An aliquot of ER2566 also known as "T7 Express Competent E. coli (High Efficiency)" from NEB was kindly provided by Fabian. Cells were plated on LB.
Plasmid-mini-prep from 3 mL of overnight culture (E. coli KRX with BBa_I746909), two glycerol stocks prepared.
Preparation of amino acid stock solutions for CFPS acccording to protocol from Caschera and Noireaux 2015b . First, all aminoacids were seperatly weighed into microcentrifuge tubes, which took a while. Then, 500 µL of 5 M KOH was added to each amino acid. Solubilization was achieved via multiple inverting and, if necessary, vortexing. Especially tyrosine takes a while, and is a suspension rather than a solution. Stock solutions are afterwards stored at -20 °C. Note: According to Caschera and Noireaux, these stock solutions can only be stored a few weeks.
Week 3 (5/4/2015-5/10/2015)
05/04:
Preparation of two equimolar amino acid mixtures à 381.6 µL according to Caschera and Noireaux 2015b (see protocols ). Water is added to a final volume of 4 mL each.  pH adjusted with 110 µL glacial acetic acid to pH = 6.5 and aliquoting in 0.5 mL Eppendorf-tubes à 410 µL and 205 µL, respectively.  Aliquots were flash-freezed in liquid nitrogen and stored at -80 °C.
EcoR1 and Pst1 restriction analysis of isolated plasmid with BBa_I746909, bands as expected.
2x 10 mL 2xYT+P medium are inoculated with ER2566 from plate. 1x 10 mL is inoculated with KRX from glycerol stock. All cultures grow over night in 100 mL shake flasks at 37 °C and 220 rpm.
05/05: Inoculation of 3x 100 mL 2xYT+P, each with one of the overnight cultures. Start OD600:
KRX ⇒ 0.139
ER2566 No. 1 ⇒ 0.146
ER2566 No. 2 ⇒ 0.140
⇒ Plotting of growth curves. After 1.75 h, KRX and ER2566 No.2 are induced with 0.5 mL of 200 g∙L-1 rhamnose  and 100 µL of 0.5 M IPTG to observe if production of T7-Polymerase has an impact on growth behavior.
Growth curves of ER2566 and KRX
Also, Primerdesign for RraA and RraB.
05/06: Preparation of ON cultures:
T7p-RBS-sfGFP in 10 mL LB with Cm and 50 µl of 20% (w/w) rhamnose
T7p-RBS-sfGFP in 25 mL LB with Cm for midi-prep
Rosetta-gami2 in 10 mL 2xYT+P for growth curve
05/07:
Proving expression of sfGFP when T7-Polymerase is expressed in KRX through addition of rhamnose
midi-prep of 25 mL T7p-RBS-sfGFP ON culture
measuring of growth curve of Rosetta-gami2 (conditions like 05/05)
Growth curves of Rosetta-gami2 in 100 mL 2xYT+P. Similar to 04/29, 1 L shaking flasks without baffles were used. Shaking at 200 rpm.
Week 4 (5/11/2015-5/17/2015)
05/11:
Preparation of S30-Buffer (10 mM TRIS, 14 mM Mg-Glutamate, 60 mM K-Glutamate, to pH = 8.2 with acetic acid), autoclave
Inoculation of 10 mL 2xYT+P with ER2566 from glycerol stock, ON culture
05/12:
Inoculation of 2x 200 mL 2xYT+P (in 1 L shaking flasks without baffles) with ON culture of ER2566 to start OD600 = 0.137, 200 rpm, 37 °C
Cultures induced at t = 1.5 h with 200 µL of 0.5 M IPTG each for a final IPTG concentration of 0.5 mM. For OD600 progression see the following table.
time
OD600 flask 1
OD600 flask 2
0 h
0,137
0,137
1 h
0,379
0,397
1.5 h ⇒ induction
0,866
0,838
3.5 h
2,405
2,446
4 h
2,657
2,585
Cell harvest (see protocols)
05/13:
Cell pellets thaw on ice for 1.5 h. Pellet weight: 0.677 g and 0.655 g respectively
Cells are resuspended in 1.5 mL icecold S30-buffer (with 3 µL 1 M DTT added each) with 10s vortex - 30 s ice resting cycles until pellets are fully resuspended. Suspensions are each transferred to a 2 mL microcentrifuge tube.
Cell disruption via sonification. Our power calibration (see protocols) revealed that the energy input of this specific Sonificator (Bandelin HD2070, MS73 tip) at this specific setup (70% power, cycle = constant) was about 3,69 J∙s-1. Therefore, for a total energy input of about 900 J, 24x 10s sonification - 10s cooling cycles were performed. The tip was dipped in about 2/3 of the cell suspension. During sonification, the microcentrifuge tube rested in ice rather than in an ice bath. As we observed that the tube becomes slightly warmer in the first case, all later sonifications were conducted with an ice bath.
After 10 cycles of sonification / cooling, we took 10 µL of the first tube after each further cycle and diluted them in 990 µL each. These samples were later used to conduct our first test about how effective our sonification kills E. coli: 20 µL of the diluted samples were plated on LB
Last steps of cell extract harvest were carried out as described in the protocol section with following variation:
cell extract in tube 2 has to be centrifuged a second time as the unsolvable fraction did not precipitate completely after the first  centrifugation
10 µL of cell extract (supernatant after centrifugation) of tube 1 are separately flash-freezed for subsequent Bradford-Assay
Cell extract from tube 2 is subjected to a run-off reaction for 50 min at 37 °C at 300 rpm in a thermomixer.
Week 5 (5/18/2015-5/24/2015)
05/20: cell disruption
weight of the Pellet: 486 mg
unfreeze the pellet on ice for two hours
resuspend the pellet in 1 mL S30 buffer (with 2 mM DTT)
10 sec vortex
30 sec on ice
repeat these steps until die pellet is resuspended (in our case we repeated these steps 5 times)
The resuspended pellet had a volume of 1300 µL, therefore we added 200 µL S30 buffer (supplemented with 0.4 µL 1M DTT) to get a Volume of 1500 µL for the sonification step
incubation the suspension for 10 min on ice
sonification with Bandelin HD 2070 and the MS 73 cone end, we used 70% amplitude and no cycles
10 sec sonifications
10 sec incubation on ice
repeat step 1 and 2 (29 times, which equals an energy of ~ 900 J/sec)
centrifuge for 10 min, 12000 rcf
split the supernatant into two tubes
incubate tube 2 (650 µL) for 50 min at 37°C and 300 rpm and centrifuge again for 10 min, 12000 rcf
aliquot the supernatant and the cell extract from tube 1 (100 µL)
freeze in liquid nitrogen and store at - 80°C
Week 6 (5/25/2015-5/31/2015)
05/27:
Next CFPS test with plasmid T7p-RBS-sfGFP (BBa_I746909). Beside the negative control without DNA-template, BBa_K1365020 (sfGFP (Bs)), a plasmid with pSB1C3-backbone containing an sfGFP without promoter, is employed as additional negative control. Reactions with two different extracts (see 05/13: one with, one without run-off treatment) are performed. Furthermore, commercial RNase inhibitor is tested as it seems to be a useful supplement. A final RNase-inhibitor concentration of 1 U/µL is used in the respective reaction mixtures. CFPS is performed over night.
05/28:
PCR with Phusion Polymerase, primers: pSB1C3_RraA_suf/pSB1C3_RraA_pre and pSB1C3_RraB_suf/pSB1C3_RraB_pre, template: plasmid T7p-RBS-sfGFP (BBa_I746909). No bands in agarose gel.
Fluorescence measurement of CFPS-reaction from 05/27 with NanoDrop 3300
Next CFPS test with plasmid T7p-RBS-sfGFP (BBa_I746909). The reactions tested are the same as on 05/27, without testing the RNase Inhibitor ⇒ glycerol may inhibit protein syntesis in vitro ( Underwood et al. 2005 ). As a positive control, a sample from an overnight culture (BBa_I746909) induced to express sfGFP is employed. Reactions are performed at two different temperatures – 30 °C and 37 °C – in incubators over night without shaking.
05/29:
No evaporation can be seen in the tubes! After 16 h incubation, the fluorescence is measured with NanoDrop 3300.
PCR problems continue
Week 7 (6/1/2015-6/7/2015)
06/01:
Small amounts of amplified PCR product detectable, positive control works, but still far away from optimum.
06/03:
ON culture of ER2566 (in 2xYT+P medium) is used for inoculation of 3x 100 mL 2xYT+P to OD600 = 0.13. Cultures are induced with IPTG to 1 mM final concentration after 1 h 20 min at OD600 about 0.7.
Cell harvest at t = 3.5 h at OD600 = 3.3
Subsequent to harvest and washing procedures, pellets are directly resuspended and lysed via sonification as before (29x for about 900 J). Half of the suspension is subjected to a run-off reaction at 37 °C for 50 min with a following . All extract is then flash-freezed and stored at -80 °C.
Week 8 (6/8/2015-6/14/2015)
06/08:
Primerdesign
Preparations for next CFPS reaction
06/09:
CFPS with the same templates as on 05/28
After assembling of CFPS compounds, 10 µL of each reaction were directly pipetted into a 364 black well plate, sealed with transparent film and incubated at 37 °C. Fluorescence was measured without the film after 3, 6.25 and 24 h respectively. Fluorescence did not exceed the negative control distinctly in any case.
06/10:
CFPS with plasmid T7p-RBS-sfGFP (BBa_I746909) in 21 µL volume with “normal” and run-off extracts from 05/13, 05/20 and 06/03 respectively. For each reaction a negative control with water instead of DNA is employed
For the first time, Mg-glutamate and K-glutamate were additionally supplemented to the reactions as they were no part of the “reaction buffer”. 3 M K-glutamate and 100 mM Mg-glutamate stock solutions were prepared and directly used to supply the reaction with a final concentration of 4 mM Mg-glutamate and 80 mM K-glutamate.
To each mix of DNA, glutamate salts and water, a master mix consisiting of “reaction buffer” and one of the above mentioned extracts was added. Reactions were performed in microcentrifuge tubes for 3 h and then 20 µL were transferred to a 364 well plate which was used for fluorescence measurements and further reaction performance. Relative fluorescence units were normalized to corresponding negative controls. The maximum of fluorescence was observed after 7.75 h for 06/03 run-off extract, which fluorescence was 10 times higher when compared to its negative control. First promising results that showed we can really produce sfGFP in vitro!
06/11:
PCR with Q5-Polymerase for amplification of 5’-UTR-flanked pSB1C3-backbone. As PCR worked with Q5, most of subsequent PCRs related to CFPS were performed with Q5.
PCR with Q5 to amplify RraA and RraB out of E. coli K12 genomic DNA and to amplify pSB1C3 backbone with RraA and RraB overhangs for Gibson assembly
Bands as expected, gel purification of all four products. PCR for backbone amplification only worked at an annealing temperature of 71.6 °C.
06/12:
Gibson assembly to insert RraA and RraB in pSB1C3 and to add UTR to T7p-RBS-sfGFP (BBa_I746909) plasmid. Subsequent transformation and plating on LB with Cm.
Transformation of electrocompetent E. coli KRX with BBa_K864100 (SYFP2)
Week 9 (6/15/2015-6/21/2015)
06/15:
Glycerol stocks prepared (RraA in pSB1C3 and RraB in pSB1C3, SYFP2, T7p-UTR-RBS-sfGFP)
Plasmid isolation of RraA in pSB1C3 and RraB in pSB1C3. Restriction analysis with Not1-HF. Bands as expected!
Plasmid isolation of T7p-UTR-RBS-sfGFP with columns of Promega kit and elution buffer of Thermo kit (10 mM TRIS-Cl, pH = 8.5) as material ran out. Restriction analysis with Not1-HF. Bands as expected!
Chosen of the above plasmids were send to sequencing core facility
06/16:
Preparation of a new cofactor premix (see following table)
Compound
Stock in mM
volume stock solution to add in µl
concentration in cofactor premix in mM
NAD
174
52.16
6.6
folinic acid
33,9
56.78
1.4
E. coli tRNA
50*
110.00
4*
CoA
65
114.23
5.4
Putrescine
200
137.50
20
Hepes
2000
687.50
1000
Spermidine
1500
27.50
30
RNase-free water
 
to 1375 µL
HEPES was added first so that risks due to pH changes could be averted.
* tRNA values given are in mg/mL
New reaction buffer aliquots were generated in which the above mentioned cofactor premix was used. For calculation we used the excel template from Sun et al. 2013 , which takes into account a reserve for pipetting.
Compound
Stock
Final
amount of stock for 1 buffer in µl
amount of stock for 4 buffers  in µL
Amino Acids
12 mM
2 mM
14.96
62.84
Cofactor premix
20x
1x
4.49
18.85
NTPs
20x
1x
4.49
18.85
Na-Oxalat
100 mM
4 mM
3.59
15.08
PEP
330 mM
33 mM
8.98
37.71
Water
na
na
1.20
5.03
In subsequent CFPS-test, run-off cell extract from 06/03 was used. We tested the impact of RNase inhibitor, high amount of Mg-glutamate (12 mM) and high amount of K-glutamate (140 mM) on the reaction with T7p-RBS-sfGFP (BBa_I746909). Additionally, a T7p-UTR-RBS-sfGFP plasmid from a mini-prep plasmid isolation (06/15) was tested without any change to other components when compared to 06/10. All reactions were performed in duplicate.
Reaction conditions and volumes were the same as on 06/10. One negative control without DNA-template was used with 80 mM K-glutamate and 4 mM Mg-glutamate added. To compare relative fluorescent units, an sfGFP-containing cell lysate and a purified GFP with 6xHis-tag (kindly provided by Lukas) was employed.
For results, see here
06/17:
Plasmid isolation of SYFP2 (BBa_K864100). Restriction analysis with Not1-HF. Bands as expected!
Sequencing results: RraA in pSB1C3 and RraB in pSB1C3 look good, but T7p-UTR-RBS-sfGFP plasmid has only 9 A instead of the desired 10 A in spacer region. As CFPS worked well with this plasmid, it is nevertheless used in further experiments.
Transformation of chemocompetent ER2566 with pRARE plasmid kindly provided by Julian.
06/20:
Generating new cell extrakt. We used e. coli KRX and BL(21)DE
3 x100 mL were cultivated
after sonication we splited the extract. One half for the run off reaction ( 60 min 37°C) and the other half got directly flash freezed
Week 10 (6/22/2015-6/28/2015)
06/22:
3A-assembly of T7+RBS (BBa_K525998, upstream part), SYFP2 (BBa_K864100) and pSB1K3.m1 (destination vector) with NEB BioBrick Assembly Kit.
PCR problems with amplification of pSB1C3 backbone with His-TEV-EF-Tu
06/23:
Transformation of chemocompetent E. coli KRX with ligation mix from 06/22.
glycerol stock prepared from ER2566 + pRARE.
PCR problem from 06/22 solved via restriction analysis: Wrong template was used. Therefore new try with correct template. Though it seemed that some false priming occured, relevant band is purified out of agarose gel.
06/24:
colonies on plates with ligated T7-RBS-SYFP2
Gibson assembly with His-TEV-EF-Tu from idt and pSB1C3 backbone with corresponding overhang, followed by transformation and plating.
06/25:
Glycerol stock prepared from ON culture of T7-RBS-SYFP2
Mini-prep from same culture, restriction analysis with Not1-HF. In 3% agarose gel, the difference between SYFP2 only and with preceded promoter was visible. Plasmids were send to sequencing core facility
colonies on plates from Gibson assembly 06/24, Colony PCR, bands as expected, preparation of ON culture
06/26:
CFPS reaction to compare extracts from different E. coli strains. To find out how fast the fluorescence signal is generated, the relative fluorescent units were measured every 30 min. Unfortunately, the delay between pipetting mastermix to DNA and first measurement was 15 min due to high number of samples. Results were inconsistent.
Glycerol stock prepared and plasmid mini-prep from His-TEV-EF-Tu in pSB1C3. Plasmid was send to sequencing core facility
PCR on His-TEV-EF-Tu in pSB1C3 to provide RraA and RraB with His-TEV-tag.
Week 11 (6/29/2015-7/5/2015)
06/30:
CFPS reaction
Sequencing results, as hoped no mutation was observed in idt sequence and T7-RBS-SYFP2 conctruct.
50 mL LB + Cm were inoculated with His-TEV-EF-Tu in pSB1C3 (construct has T7p from former pT7-RBS-sfGFP)
Start of 5 L fermentation project: ER2566 from glycerol stock was plated on LB and cultivated in 37 °C incubator for about 16 h.
07/01:
4x 200 mL LB+Cm in 1 L shaking flasks were inoculated with His-TEV-EF-Tu in pSB1C3 ON culture, induction after 2 h with 20% (w/v) Rhamnose. Harvest of the cells after about 4 more hours via centrifugation at 4 °C for 30 min, flash freezing
Gel extraction of pSB1C3 backbone to provide RraA with His-TEV-Tag (RraA-purif). Afterwards, Gibson assembly and  transformation via heat shock to get RraA-purif and RraB-purif.
Inoculation of 2x 4 mL 2xYT+P with single colony from ER2566 plate. Cultivation at 37 °C for about 10 h. Afterwards, inoculation of 100 mL preculture and cultivation at 37 °C for 10 h.
07/02:
5 L fermentation in 7 L stirred reactor
Biostat NLF 7 L reactor is filled with 5 L of 2xYT+P medium and steam-sterilized with the kind help of Thomas. During fermentation, software Bioscala is used, and pH is permanently and automatically adjusted to 7
50 mL of preculture are used for inoculation to start OD600=0.17
Induction with IPTG after 1.5 h at OD600 of about 0.88
Growth curve of ER2566 during 5 L fermentation
Cell harvest after 3.2 h of cultivation at a final OD600 of about 4.6. The culture passed through a self-built cooling system made of a Liebig condenser. Cooling is enabled by julabo thermostat. Half of the culture was directly poured into prechilled 50 mL tubes and centrifugated, while the other half was first kept in a separate 5 L glass bottle at 4 °C due to missing space in the centrifuge.
Cooling apparatus for harvesting E. coli culture
All pellets were twice washed with 10 mL icecold S30A buffer per g of wet cell mass according to our protocol. S30A contained 50 mM instead of 10 mM TRIS. The third and final washing step was then conducted with 10 mL icecoled S30 buffer per g wet cells.
All 50 mL tubes with washed cell pellets were flash freezed in liquid nitrogen and stored at -80 °C
Week 12 (7/6/2015-7/12/2015)
07/06:
RraA-purif and RraB-purif were send to sequencing.
07/08:
Glycerol stocks for RraA-purif and RraB-purif. Overexpression of both proteins similar to overexpression of His-TEV-EF-Tu on 07/01 except that after induction, cells were only cultivated for subsequent 3 h according to Gorna et al. 2010.
cell disruption via sonication
cells from fermentation were used
after sonication different run off reactions were made (20, 30, 40,50, 60, 70 and 80 min at 37°C and 140 rpm)
07/09:
To test self-made NTP solutions in comparison to purchased solutions, solid NTPs were weighted, dissolved in RNase-free water and 15% KOH was added. The solution was prepared according to Sun et al., 2013 , but scaled down to 65%. Also, different salts of NTPs with different molecular weights were used, therefore the calculation had to be adjusted. NTP mixes were aliquoted and flash freezed.
07/10:
CFPS reaction with self-made NTP-solution.
Week 13 (7/13/2015-7/19/2015)
07/13:
CFPS reaction, fluorescence signal did not rise to values comparable to 06/16.
07/14:
PCR for adjusting the gap between RBS and first codon
PCRs to bring 5'-UTR in front of RFP and YFP
07/16:
CFPS reaction, different approaches to determine the error source, but no success.
cell disruption from the remains of the fermentation via sonication
Week 14 (7/20/2015-7/26/2015)
preparation of a new cofactor premix
Compound
Stock in mM
volume stock solution to add in µL
concentration in cofactor premix in mM
NAD
174
26.08
6.6
folinic acid
33,9
28.39
1.4
E. coli tRNA
50*
55.00
4*
CoA
65
57.12
5.4
Putrescine
200
68.75
20
Hepes
2000
343.75
1000
Spermidine
1500
13.75
30
RNase-free water
 
to 687.5 µL
Flash freezing of aliquots, stored at -80 °C, except for one aliquot that is directly used for a CFPS reaction.
Fluorescence signal did not rise to values comparable to 06/16.
07/24:
Primerdesign for RraA and RraB characterisation.
Week 15 (7/27/2015-8/2/2015)
CFPS finally works! A mix of commercial NTP solutions gives great results compared to self-made mix from 07/09. The DNA template was purified via mini-prep kit, the washing steps were repeated once before elution.
07/28:
CFPS reaction with 3 replicates to test different ratios of extract/reaction buffer
PCR to adjust the gap between RBS and first codon finally works, subsequent gel extraction.
07/29:
Promega 30S extract arrived!
Phosphorylation and ligation of UTR-mut-gap with T4 PNK and T4 DNA ligase followed by transformation.
CFPS reaction to test influence of arsenic on in vitro reaction.
07/30:
Plasmid mini-prep of PT7-K516030 (RFP-generator). Restriction analysis via NotH1, size as expected.
PCR on PT7-K516030 with UTR_RFP_fwd and UTR_rev to get UTR in front of RFP
Colony-PCR on colonies from UTR-mut-gap. Bands as expected, however this gave no information about the sequence
Week 16 (8/3/2015-8/9/2015)
Gibson assembly with PT7-UTR-SYFP
PCR on E. coli genomic DNA with rne_to_pZ9_fwd and rne_to_pZ9_rev to get RNase E DNA and PCR on pZ9 backbone with pZ9_pT7_rne_fwd and pZ9_pT7_rne_rev
08/04:
Plasmid mini-prep of UTR-mut-gap from 07/30.
Preparation of Maltodextrin and Hexametaphosphate for test of low-cost energy solution in CFPS
Gibson assembly of RNase E and pZ9 backbone (hereafter named pZ9-rne), transformation and plating on LB+Kan
08/05:
Sequencing of UTR-mut-gap was successful in 6 of 11 cases! ⇒ Glycerol stocks
08/07:
Gibson assembly of PT7-UTR-RFP
08/08:
CFPS with T7-S30 extract from Promega to compare it with our extract.
Two amino acid mixtures (-Leu and -Cys) were thawed, 50 µL of each combined, mixed, aliquoted and flash-freezed, except of one aliquot that was directly used for the reaction.
S30 premix without amino acids was thawed, aliquoted and flash-freezed, except of one aliquot that was directly used for the reaction.
"T7 S30 circular" extract was carefully thawed on ice, aliquoted and flash-freezed except of one aliquot that was directly used for the reaction.
Glycerol stocks prepared and plasmid mini-prep of Plac-UTR-sfGFP, pZ9-rne and PT7-UTR-RFP. Subsequent restriction analysis. Bands as expected.
CFPS with E. coli KRX extract
08/09:
CFPS with potential PT7-UTR-RFP plasmid to test wether we can express RFP in our extract. Additionally, low-cost reaction buffer is tested.
Week 17 (8/10/2015-8/16/2015)
Sequencing results of Plac-UTR-sfGFP, pZ9-rne and PT7-UTR-RFP looked good, although exactly the plasmid PT7-UTR-RFP that was used on 08/09 had a mutation right after RBS.
pZ9+rne is plated on LB+Kan to generate fast competent cells on 08/12
08/10:
production of new cell extract
we used e.coli with the heavy metal activator or repressor plasmid to generate cell extraxt with the appropiate activator or repressor
same procedure like Cell harvest and Sonication .
08/11:
Paper pieces provided with a hole-punch were autoclaved 3 times, on this paper pieces we run an CFPS
08/12:
Planning of CFPS to help at modelling of in vitro reaction.
Preparation of electrocompetent pZ9-rne cells, followed by transformation with RraA-purif and RraB-purif, respectively. Plating on LB+Kan+Cm.
CFPS with the self made extract from 8/10. We tested to induce ArsO with different Arsenic concentrations
08/13:
CFPS: Spiking experiments to test if some compounds are limiting after 2 h.
CFPS: Test to repress of ArsO. Therefore purified Arsenic Repressor were added
08/14:
Preparation of glycerol stocks of pZ9-rne + RraA-purif and pZ9-rne + RraB-purif. 10 mL LB+Kan+Cm are inoculated with 100 µL of each culture
CFPS with addition of RraA from elution 1 of 07/11 and test of Plac-UTR-sfGFP in Promega S30 extract in 15 µL reactions.
08/15:
Characterization of double transformants (pZ9-rne + RraA and pZ9-rne + RraB)
Week 18 (8/17/2015-8/23/2015)
08/18:
Sonification of pellets form 07/02 and further processing creates enough ER2566 cell extract for the rest of the project
CFPS: with different plasmidcontrentrations of ArsO
CFPS: with 3 times autoclaved Paper pieces (Munktell C350L, Munktell FN3 and Filter paper from Merck)
08/19:
Characterization of UTR in vivo by comparison of PT7-UTR-sfGFP and PT7-sfGFP.
2x 50 mL of LB+Cm are inoculated to a start OD600 of 0.1 each.
Probes are taken after 3.25 h and 6.5 h and fluorescence is analyzed with Tecan platereader.
Normalized fluorescence signal of PT7-UTR-sfGFP culture after 3.25 h is even higher than fluorescence signal of PT7-sfGFP after 6.5 h!
CFPS: on Paper and lyophilized
CFPS: after the reactions are pipetted, the reaction ist pipetted on a paper disc put in a reaction tube (the reaction tubes were preparated with holes in the lid)  and flash freezed.
out of the liquid nitrogen the tubes were placed into the lyophilizer
the reactions were lyophilized for 3 or 24 hours
on the next day the reactions were rehydrated with 15 µL RNase free water to start the reaction
08/20:
New PEP aliquots are prepared
Buffer exchange of RraA to 50 mM HEPES à pH 7.2 via Vivaspin columns.  Concentration of RraA was 4.5 mg/mL afterwards.
CFPS:
test of different Plasmidkonzentrations (15 nm, 10 nm, 7 nm, ,5 nm, 3 nm and 1 nm)
Influence of Alkohol on the reaction (5 % and 15 %)
08/21:
CFPS reaction
Plating of 100 µL of cell extract aliquot on LB to see how many cells are still living
Same experiment to characterize UTR with RFP. Results confirm the use of UTR for fast expression, but experiment will be repeated with triplicates.
08/22:
8 cells living in a 100 µL cell extract aliquot.
preparation of new amino acid stocks, see Amino acid preparation
CFPS: with the new Aminoacids stocks in comparison to the old one
08/23:
CFPS reaction at 25 °C, and another CFPS reaction to see what happens when the DNA template is added delayed in time.
Week 19 (8/24/2015-8/30/2015)
08/24:
PCR to add UTR to SYFP-template, fails.
08/25:
CFPS reaction to compare the influence of 50 mM HEPES buffer at pH = 7.2 to the influence of purified His-TEV-RraA in 50 mM HEPES buffer at pH = 7.2.
08/26:
Re-performance of UTR-characterization with triplicates.
CFPS: test influence of Chromium on the CFPS, and test of the heavy metal promoters
08/27:
Further RraA characterization: 25 mL cultivation of pZ9-rne and pZ9-rne + RraA-purif (triplicates each). Rhamnose was added from the start to induce T7-Polymerase and expression of RNase E and RraA, respectively. After 2.5 h at OD600 of 1.22, 2x 1 mL samples were quickly transferred into microcentrigfuge tubes, centrifuged for 15 s, and the pellet was flash-freezed.
RNA-isolation with Macherey-Nagel kit. Subsequent PCR with RraA_fwd and RraA_rev showed DNA contamination in the samples.
CFPS: test of the heavy metal promoters at 15 and 29 °C.
08/28:
Digestion of RNA probes from 08/27 with Qiagen DNase (off-column digestion). Afterwards, samples were loaded onto RNA columns, washed and eluted. Nanodrop showed samples were contaminated with solvents. RNA was stored at -80°C.
PCR with split primers to create biobrick UTR in pSB1C3, successful only for one part.
08/29:
Gradient PCR for amplification of missing fragment from 08/28. Bands showed up at 66 °C and were subsequently purified.
Gibson assembly, heat transformation and plating on LB+Cm to bring UTR to pSB1C3.
Restriction of PT7-mRFP generator, PT7-UTR-mRFP generator and PT7-SYFP with EcoR1HF and Pst1 to bring them from pSB1K3 to shipping vector. Afterwards, ligation ON.
CFPS: influence of Copper
Week 20 (8/31/2015-9/6/2015)
9/3:
CFPS: optimizing the different cell extract (extracts for the heavy metal reactions, with activator and repressor inside
9/4:
CFPS: generating new cell extract. Same procedure like 8/10, but this time the culture were induced with rhamnose to induce T7-Polymerase production
9/5:
CFPS: test of the new generated cell extract from the day before
9/6:
CFPS: comparison of Induktion of copAP and pT7-copAP with different copper concentrations
9/6:
CFPS: Induktion of pT7-prcnA, pT7-pbrAP and test of the influence of lead to the CFPS
CFPS: Induktion of pT7-chrP in the extract with our generated Repressor and in the extract with the repressor, generated from the igem Team Dundee 2015
CFPS: lyophilization for 24 hours, rehydration  direktly after the lyophilization. 1 Part were stored ad 8 °C, another at 60°C and another part at room temperature. The storage occur for 6 days. Afterwards they were rehydrated with 15 µL table water
Week 21 (9/7/2015-9/13/2015)
9/9:
CFPS: induction of pT7-blcP with GBL (0,3%; 1%; 3%), GHB (0,3%; 1%; 3%) and Succinat
9/13:
CFPS: try to induce the reaction after lyophilization with contaminated water
preparation of different reactions (quadruple)
pT7-blcP in extract with the corresponding repressor
pT7-pmerT in extract with the corresponding activator
pT7-copAP in extract with the corresponding activator
and our positiv control in our normal extract without overexpression of any repressor or activator
The reaction was scaled up to 30 µL. The hole 30 µL were placed on a paper disc.
The paper disc were placed in a reaction tube with holes in it
Flash freeze
lyophilization for 3 hours
rehydration with different water (with and without contaminations)
every reaction was rehydrated with 15 µL table water for comparison
every reaction was rehydrated with the corresponding inducer
pT7-copAP and pT7-pmerT were induced with a mixture of copper and mercury in table water
