<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_State_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn_State/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn_State/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Protocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn_State/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn State/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1><FONT color="white"> WELCOME TO PENN STATE iGEM 2014! </FONT></H1><P style="color:#E7E7E7"><A href="https://2014.igem.org/wiki/index.php?title=Team:Penn_State/Protocol&amp;action=edit" style="color:#00008B"> Click here  to edit this page!</A></P><TABLE width="100%"><TBODY><TR height="50px"><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>HOME</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/2014_Judging_Form?id=1506" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>JUDGING FORM</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>OFFICIAL PROFILE</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Team" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>TEAM</B></FONT></FONT></A></TD><TD style="border:2px solid navy" align="center"><A href="https://2014.igem.org/Team:Penn_State/Project" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> PROJECTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Parts" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>PARTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Notebook" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> WETLAB</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Safety" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B> SAFETY</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/HumanPractices2" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B>HUMAN PRACTICES</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Attributions" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>ATTRIBUTIONS</B></FONT></FONT></A></TD></TR></TBODY></TABLE><TABLE align="center" width="1000px"><TBODY><TR><TD colspan="6"><CENTER><H1>Wetlab Protocols</H1></CENTER></TD></TR><TR><TD><P>Click <A href="https://2014.igem.org/Team:Penn_State/Notebook">HERE</A> to return to the weekly summaries</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Minipreparation of plasmids</H2><P><B>Purpose:</B>  Minipreparation (miniprep) is done in a laboratory setting in order to extract a plasmid from bacteria. There are several different scales of executing this task, but miniprep is the smallest. The steps of the miniprep are to weaken the cell wall, lyse the cell, precipitate out the lipids and proteins, remove of any chromosomal DNA or RNA that is still present, and then wash the plasmid to make sure that all salts are removed so that only the plasmid remains. This process uses a binding column that is comprised of a silica matrix in order to bind the plasmid DNA so that any other remaining cell fragments can be washed away. This procedure follows the E.Z.N.A. plasmid DNA Mini Kit 1 from the Omega Bio-Tek company.</P><P><B>Vacuum Procedure:</B></P><P>1. A culture was inoculated and grown overnight in a shaker at 37C at 300 rpm.</P><P>2.  The culture was centrifuged at 10,000 rpm for 1 minute at room temperature</P><P>3. The culture median was decanted being careful not to discard any of the cell pellet.</P><P>4.  250 µL of Solution I was added and mixed thoroughly and resuspend the pellet.  <I>*Note: Solution I is used in order to weaken the cell wall.</I></P><P>5. After the pellet has been resuspended, the contents were added to a microcentrifuge tube.</P><P>6. 250 µL of Solution II was added and the microcentrifuge was gently inverted until a clear lysate formed.  <I>*Note: Solution II is used in order to lyse the cell wall. Vigorous mixing was avoided to prevent DNA shearing.</I></P><P>7. 350 µL of Solution III was added to the microcentrifuge tube and then it was inverted several times until a white precipitate forms.  <I>*Note: Solution III is used to bind all of the proteins and lipids from the cell so that they can be removed.</I></P><P>8. The solution was centrifuged at maximum speed for 10 minutes.</P><P>9. The cleared supernatant was transferred to a HiBind DNA Mini Column that was attached to the vacuum.</P><P>10. The vacuum was turned on and the liquid was drawn through the mini column.</P><P>11. 500 µL of HBC Buffer was added to the column and vacuumed through.</P><P>12. 700 µL of DNA wash buffer was added to the column and vacuumed through</P><P>13. Step 12 was repeated.</P><P>14. The column was added to a collection tube and then centrifuged for 2 minutes on maximum speed to dry the column.</P><P>15. The column was transferred to a clean 1.5 mL microcentrifuge tube.</P><P>16. 30 µL of sterile deionized water was added to the column and allowed to sit for 1 minute.</P><P>17. The microcentrifuge tube with the column inside of it was centrifuged for 1 minute at maximum speed.</P><P>18. The column was discarded and the microcentrifuge tube was stored at -20C until the plasmid was needed.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Preparation of Electrocompetent Cells</H2><P><B>Purpose:</B> One of the techniques used to incorporate new genetic material into a cell is called electroporation. In this process, electrical pulses create holes in the cell membrane so that genetic material, such as plasmids, is able to permeate the plasma membrane. It is most commonly used in bacterial cells because electroporation is highly efficient at incorporating the new DNA into the bacterial cells that will then be used for cloning. There are special cuvettes that are used for this process that enable the cells to be evenly distributed without air bubbles and for media to be directly added immediately after electroporation occurs so that the cells can recover. Electrocompetent cells are different than normal bacterial cells because they have a weaker cell wall so that electroporation will have a high efficiency.</P><P><B>Procedure:</B></P><P>Part 1: Overnight cell growth</P><P>1.  5 mL of LB media were added to a test tube.</P><P>2. 5 µL of Streptomycin were added to the same test tube.</P><P>3. E. coli cells of the desired strand were used to inoculate the media.</P><P>4. The cells were allowed to grow overnight in a shaker at 37C and 300 rpm.</P><P>Part 2: Growing Electrocompetent cells</P><P>1. 100 mL LB media was added to 2 Erlenmeyer flasks (total 200 mL of media)</P><P>2. 100 µL of Streptomycin was added to each of the Erlenmeyer flasks.</P><P>3. The optical density (OD) of cells grown overnight culture was measured.</P><P>4. The Erlenmeyer flasks were inoculated with enough of the overnight media so that the OD in each Erlenmeyer flask was 0.01.</P><P>5. The two flasks were then put in a shaker at 30 C with 300 rpm. <I>*Note: Growing the cells at 30C instead of 37C will alter the growth of the cell membrane</I></P><P>6. Once an OD of 0.5 was reached in the flasks, they were removed from the shaker.</P><P>Part 3: Harvesting Electrocompetent cells.</P><P>1. While the cells are growing in Part 2, 50 micrcentrifuge tubes were labeled and then set in a box in the -80C freezer.</P><P>2. The centrifuge was turned on an cooled to 4C.</P><P>3. The contents of the Erlenmeyers flasks were transferred to 4x50mL Falcon Tubes, while making sure the tubes remained chilled.</P><P>4. The tubes were put in the pre-chilled centrifuge and spun down at 4500 rpm for 10 minutes.</P><P>5. The supernatant was poured out, being careful not to disturb the pellets at the bottom.</P><P>6. 25 mL of 10% glycerol was added to each of the tubes and used the gently resuspend the pellet.</P><P>7. The contents of the 4 x 25mL Falcon tubes were transferred so that there were 2 X 50 mL Falcon tubes.</P><P>8. These 2 tubes were then spun at 4C and 4500 rpm for 10 minutes.</P><P>9. Again, the supernatant was poured out and the pellets were resuspended in 25 mL of 10% glycerol.</P><P>10. The contents of the 2 x 25mL Falcon tubes were transferred to 1 x 50mL Falcon tube.</P><P>11. The tube was spun down again (making sure to balance the centrifuge) at 4C and 4500 rpm for 10 minutes.</P><P>12. The supernatant was discarded.</P><P>13. This time, the cells were resuspended with 2 mL of 20% glycerol.</P><P>14. Quickly, the solution was aliquoted in 55 µL of solution into the pre-chilled centrifuge tubes.</P><P>15. The cells were stored in the -80C fridge until they were needed to be used.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Making Agar Plates</H2><P><B>Purpose:</B> In order to make antibiotic resistant plates in order to test and make sure that cells have incorporated the desired plasmid.</P><P><B>Procedure:</B></P><P>1. The autoclave was turned on to warm up.</P><P>2. 300 mL of distilled water was measured and put in an appropriately sized beaker.</P><P>3. 7.5 g of LB miller and 4.5 g of Agar powder were measured and added to the distilled water in the beaker.</P><P>4a. The bottle was autoclaved for sterilization purposes.</P><P>4b. While the autoclave was running, ~16 Petri dishes were labeled with the date and the type of antibiotic that would be added to the plates.</P><P>5. The bottle was removed from the autoclave and then allowed to cool to about 60 C.</P><P>6. When the bottle was cooled, 300 µL of the desired antibiotic was added to the solution.</P><P>7. The bottle was swirled to ensure that the antibiotic was mixed equally in the solution.</P><P>8. Approximately 50 mL of the solution was distributed to each of the Petri dishes, under sterile conditions, and allowed to cool.</P><P>9. Once the agar had solidified, they were placed lid side down and stored in the refrigerator. <I>*Note: Lid side down is important so that any condensation does not collect on the agar. This could affect the quality of the plates.</I></P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Transformation</H2><P><B>Purpose:</B> This procedure is done in order to incorporate new genetic material into a cell. This lab uses transformation on bacterial cells to insert plasmids. Once the desired plasmids have been constructed and isolated, they are inserted in to the cells by electroporation or heat shock. Once these cells have the new genetic material introduced, they are allowed to grow and recover in a shaker and then are plated so that cells without the incorporated antibiotic resistance and the new gene will die.</P><P><B>Procedure:</B></P><P>1.A 50 µL aliquot of electrocompetent cells were allowed to thaw on ice.</P><P>2. An electroporation cuvette was put on ice to chill.</P><P>3. 2-3µL of the desired plasmid was added to the electrocompetent cells and mixed gently by pipetting.</P><P>4. The cuvette was wiped and then placed in the electroporator. <I>*Note: For E. Coli, the electroporator should be set for 2500 V</I></P><P>5. The time constant was recorded (should be between 4.6 and 5.6)</P><P>6. 600 µL of SOC was added directly to the cuvette and mixed with the cells.</P><P>7. The cuvette was incubated at 37 C at various times depending on the antibiotic resistance marker:</P><P><I>        Ampicillin – 15 min</I></P><P><I>        Chloramphenicol – 30 min</I></P><P><I>        Kanamycin/Streptomycin – 45 minutes</I></P><P>8. After the incubation period, the cells were added to the pre-made agar plate and spread using a sterilized spreader.</P><P>9. The plate was put in the 37 C oven for the cells to grow overnight.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Polymerase Chain Reaction</H2><P><B>Purpose:</B> Polymerase Chain Reaction (PCR) is a laboratory technique that is used to help amplify specific sections of DNA. These strands of DNA are then able to be analyzed and put into new cells to see how their behavior is affected. This technique is mostly used when a cell is being designed that will use genes from multiple sources. DNA is able to replicate on its own, but it takes several hours to occur. PCR is able to optimize the temperature and process that is needed for replication to happen so that instead of taking several hours in order for one copy to be made, thousands can be made in the same amount of time. Each reaction follows the same three steps; denaturation, annealing, and extension. This procedure is the suggested protocol by New England BioLabs Inc.</P><P><B>Procedure:</B></P><P>1. A bucket of ice was gathered to ensure that all materials remained cold.</P><P>2. The following components (total 50µL) were added to a PCR tube, from largest quantity to smallest:</P><P><I>Nuclease-free water - 37 µL</I></P><P><I>5x Phusion HF Buffer - 10 µL</I></P><P><I>10 mM dNTPs - 1 µL</I></P><P><I>10 µM Forward Primer - 0.25µL</I></P><P><I>10 µM Reverse Primer - 0.25 µL</I></P><P><I>Template DNA - 1 µL</I></P><P><I>Phusion DNA Polymerase - 0.5 µL</I></P><P>3. The thermocycler was programmed for the reaction using the following conditions:</P><P><I>Initial Denaturation: 98 C for 30 seconds</I></P><P><I>Denature: 98 C for 5-10 seconds</I></P><P><I>Annealing: 45-72 C for 10-30 seconds</I></P><P><I>Extension: 72C for 15-30 seconds per kb</I></P><P><I>Repeat Denature, Annealing, and Extension Cycle 25-35 times</I></P><P><I>Final Extension: 72 C for 5-10 minutes</I></P><P><I>Hold: 4 C for forever</I></P><P>4. Products were removed from the thermocycler and stored at -20C until they were needed.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Gel Electrophoresis</H2><P><B>Purpose:</B>

Gel Electrophoresis is a laboratory technique that can be used to separate DNA fragments of different lengths. This is done by applying an electric field to DNA fragments. Since DNA is negatively charged, all of the DNA will flow toward the positive end of the electric field; the gel allows smaller strands of DNA to move faster than larger strands, so that is how the separation occurs. After a gel has run for 1-1.5 hours, there should be a visual difference between different strands of DNA present in the sample. In our lab, it is most commonly used to separate PCR products, but this has also been used as a technique to sequence and compare different types of DNA in forensics.
</P><P><B>Procedure:</B></P><P>
1.  An appropriate sized well and comb for the amount of samples was chosen. Typically, a smaller well is chosen with a comb that either has 6 or 10 slots.</P><P>
2.  The comb and well was rinsed to make sure that any of the residue from the last gel that was run was washed away.</P><P>
3. 50µL of a 0.8% agarose solution was mixed with 0.5µL GelStar solution from New England BioLabs Inc.</P><P>
4. The gel was allowed to sit until it solidified and then the comb was removed.</P><P>
5. The well was filled with 1X TAE until it covered the gel.</P><P>
6. An appropriate kilobase pair ladder was chosen based on the prediction of the desired results of the separation.</P><P>
7. 20 µL of the ladder was added to the first well.</P><P>
8. 10µL of loading dye was added to each sample being run and then mixed thoroughly, without creating bubbles.</P><P>
9. The entire contents of the PCR tube was added to the designated well for that sample, being careful to gently load the sample so that the DNA does not spread out of the well.  </P><P><I>Note: Each sample was documented as to which well it was placed in. This was useful information for comparing conditions and reactants of the PCR so that if an experiment was repeated, it could be improved for better results.</I></P><P>
8. The lid was put on the well and set so that the samples are starting closest to the negative (black) end so that the samples would run toward the positive (red) end.</P><P>
9. The voltage machine was set so that it read between 100-120 V so that the separation could occur in a timely fashion.</P><P>
10. The gel was run until the dyed bands were approximately halfway through the gel.
</P><P>
11. The gel was removed from the well and then ultraviolet light was used to compare the ladder to the samples that were run.</P><P>
12. If the band was in the correct region for the desired product, the experiment proceeded to gel extraction. If not, PCR was repeated altering one or two of the reactants in hopes to improve the end result.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Gel Extraction</H2><P><B>Purpose:</B>
Gel extraction is the next step after the desired results from gel electrophoresis have been obtained. For this procedure, the band from the gel is cut out and then the gel is dissolved and filtered so that only the DNA from that segment of the gel remains. This DNA is then used for subsequent experiments and can be sequenced when it is inserted in a plasmid.

</P><P><B>Procedure:</B></P><P>
1. Microcentrifuge tubes, equal to the number of samples that were extracted, were weighed and recorded.</P><P>
2. An extraction knife and tweezers were wiped with ethanol in order to remove any contaminants.</P><P>
3. The desired bands of the gel were cut out and put in to the pre-weighed microcentrifuge tubes.</P><P>
4. The tubes were weighed again, with the gel inside of them.</P><P>
5. Three times the weight OF THE GEL in µL of Agarose Dissolving Buffer (ADB) was added to the tube.
        	Example: If the microcentrifuge tube was 0.98g and the tube with the gel was added weight 1.28g, then 90 µL of ADB should be added to the tube.</P><P>
6. The gel was left to dissolve in an oven at 60 C until the agarose was completely dissolved. It was periodically taken out of the oven and vortexed to assure that all of the gel was adequately dissolved.</P><P>
7. After the gel was completely dissolved. The contents of the centrifuge tube were added to a Hi-Bind column and then vacuumed through the column.</P><P>
8. 700 µL of DNA Wash Buffer was added to each Hi-Bind column and the vacuumed through the column.</P><P>
9. Step 8 was repeated.</P><P>
10. The Hi-Bind columns were then put in the centrifuge with a collection tube and allowed to dry spin in order to remove any residual Wash Buffer.</P><P>
11. The Hi-Bind column was transferred to a microcentrifuge tube.</P><P>
12. 20 µL of autoclaved water was added to the center of the Hi-Bind column and allowed to sit for one.</P><P>
13.  The tube was centrifuged at high speed for 1 minute.</P><P>
14. The DNA wash through was then nano-dropped for concentration and left for storage at -20 C.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Restriction Digest</H2><P><B>Purpose:</B>
Digestion takes advantage of naturally occurring sequences in order to cleave DNA at specific sections. This is often utilized to isolate specific fragments of a plasmid so that new genetic material can be introduced.  Restriction sites are usually a six nucleotide sequence and it is important to choose restriction sites that are unique to the vector if cloning is being attempted; otherwise, it is possible that the plasmid will cut in multiple locations. Restriction enzymes can also be used to check and see if plasmid has the desired genes in it by choosing a enzyme that will produce multiple DNA fragments and then running it through gel electrophoresis to see if the experimental bands match the desired lengths of DNA.

</P><P><B>Procedure:</B></P><P>
1. The necessary restriction enzymes were identified by using ApE.</P><P>
2. The following solutions were added to a PCR tube, adding the restriction enzymes last.</P><P><I>10x Appropriate Buffer - 5 µL</I></P><P><I>PCR Product - 2 µg</I></P><P><I>Restriction Enzyme 1 - 1µL</I></P><P><I>Restriction Enzyme 2 - 1 µL</I></P><P><I>ddH2O - to 50 µL</I></P><P>
*The buffer may change based on the restriction enzymes that were being used. The Double Digest Finder was used from the New England BioLabs website. https://www.neb.com/tools-and-resources/interactive-tools/double-digest-finder.</P><P>
3. The Double Digest Finder was used to determine the temperature at which the digestion would take place.</P><P>
4. The reaction was allowed to run for 6-9 hours at the temperature the Double Digest Finder provided.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>PCR Clean Up</H2><P><B>Purpose:</B>
PCR clean-up is done after digestion and ligation in order to remove salts and to concentrate the product of these reactions. By using a small amount of dd H20 at the end of the clean-up, it is possible to use all of product at a higher concentration since most of the reactions in PCR reaction tubes are volume limited. This will give more colonies with the desired plasmid inserted compared to using the same volume of a less concentrated solution.

</P><P><B>Procedure:</B></P><P>
Clean-up for Digestion</P><P>
1. 250 µL of DNA Binding Buffer was added to the PCR tube with the Digestion products and was mixed via pipette.</P><P>
2. This solution was transferred to a Zymo spin column and the column was placed in a collection tube.</P><P>
3. The tube was centrifuged at 14,000 rpm for 1.5 minutes.</P><P>
4. The flow-through was discarded and 600µL of wash buffer was added to the center of the column.</P><P>
5. The tube was spun again at 14,000 rpm for 1.5 minutes.</P><P>
6. Steps 4 and 5 were repeated.</P><P>
7. The flow through was discarded and then the column was dry spun at 14,000 rpm for 2 minutes.
        	Note: Dry spinning is important to remove all of the ethanol from the wash buffer because this can interfere with transformation.</P><P>
8. The column was transferred to a new microcentrifuge tube and 10 µL of dd H20 was added to the column.</P><P>
9. The column was allowed to sit for 1 minute.</P><P>
10. The tube was centrifuged at 14,000 rpm for 2 minutes and then the concentration of the DNA was measured via nanodrop.</P><P>
Clean-up for Ligation</P><P>
1. 200 µL of DNA Binding Buffer was added to the PCR tube with the Digestion products and was mixed via pipette.</P><P>
2. This solution was transferred to a Zymo spin column and the column was placed in a collection tube.</P><P>
3. The tube was centrifuged at 14,000 rpm for 1.5 minutes.</P><P>
4. The flow-through was discarded and 600µL of wash buffer was added to the center of the column.</P><P>
5. The tube was spun again at 14,000 rpm for 1.5 minutes.</P><P>
6. Steps 4 and 5 were repeated.</P><P>
7. The flow through was discarded and then the column was dry spun at 14,000 rpm for 2 minutes.
        	Note: Dry spinning is important to remove all of the ethanol from the wash buffer because this can interfere with transformation.</P><P>
8. The column was transferred to a new microcentrifuge tube and 4 µL of dd H20 was added to the column.</P><P>
9. The column was allowed to sit for 1 minute.</P><P>
10. The tube was centrifuged at 14,000 rpm for 2 minutes and then the concentration of the DNA was measured via nanodrop.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Ligation Reaction</H2><P><B>Purpose:</B>
This procedure is done to add a segment of DNA into a vector backbone that has compatible digested restriction enzyme sites. When digestion occurs, a “sticky end” of DNA is left over. This section of DNA is complementary to the other digested restriction site of the backbone or insert. This makes it possible for these nucleotides to connect and form a new plasmid that incorporates the desired insert. The plasmid then can undergo PCR clean-up and it is then ready to be transformed in to a cell.

</P><P><B>Procedure:</B></P><P>1. The following solutions were added to a PCR tube</P><P><I>1000 fmole insert - varies</I></P><P><I>10 fmole cut plasmid - varies</I></P><P><I>T4 Ligase Buffer - 2µL</I></P><P><I>T4 Ligase- 1 µL</I></P><P><I>ddH2O - to 20 µL</I></P><P>** In order to calculate fmoles, DNA(fmoles) = DNA (ng/µL) * 1515/ ( # bp in DNA)</P><P>
2. The reaction was allowed to run for 5-10 minutes at room temperature.</P><P>
3. The ligated product immediately was used for PCR clean-up.</P>
        	Note: This is so that the ligation does not start to generate non-specific ligation products. 
</TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Resuspending PCR Primers Protocol</H2><P><B>Purpose:</B>
In order to proceed to PCR, primers shipped as lyophilized DNA must be resuspended and diluted to a known concentration. This allows consistent and correct amounts of primers to be added to PCR reactions. It also ensures that stock solutions are protected for future use.Modified from: http://fg.cns.utexas.edu/fg/protocol__resuspending_PCR_primers.html


</P><P><B>Procedure:</B></P><P>
1. Spin down tubes of dried primers (ensures the pellet is in the bottom of the tube)</P><P>
2. Create a 100 uM solution of primers by resuspending with 10 ul water for each ng DNA in the tube. For example, if there are 
38.2 nmol of primer then by adding 382 µl of H2O, a 100 µM primer stock is created.</P><P>
3. Incubate master stock primers newly suspended in water for 10 minutes.  Mix well before making working stock dilutions.</P><P>
4. Create a 10 uM solution of primers as a “working solution.” This reduces the number of freeze/thaw cycles that the master primer stock goes through and reduces the chances of contaminating the primary source for the primer. Dilute the primer master stock in a sterile microcentrifuge tube 1:10 with water.</P><P>
5. Store stock solutions and working solutions at -20 degrees C.</P></TD></TR></TBODY></TABLE><TABLE align="center" width="900px"><TBODY><TR><TD><H2>Steady State Growth and Fluorescence Measurement</H2><P><B>Purpose:</B>
One of the ways that gene expression can be measured is by fluorescence. By measuring the intensity of a fluorescent protein, it is possible to predict the expression of a gene that is kept on the same plasmid. One of the ways that measuring fluorescence is useful in the codon optimization project is that the gene that is trying to be optimized is a green fluorescent protein (GFP). Using a TECAN, it is possible to see which variations of GFP have the most expression while keeping the concentration of cells the same for all of the variations.

</P><P><B>Procedure:</B></P><P>
1.  A standard 96-well microplate containing 200 µL of LB media and the desired antibiotic was inoculated from single colonies. A non-fluorescent cell culture was also inoculated. The cultures were allowed to grow overnight at 37 C at 250 rpm until an OD of 2.0 was reached.</P><P>Note: Only the inner 60 wells are used for measurement because the outer wells can have media evaporate and this would affect the results. The outer wells were filled with blank media to prevent this from happening to inner wells.</P><P>
2.  A fresh 96-well transparent, flat bottom microplate wwas filled with 198 µL M9 minimal media. Microplate wells were inoculated by overnight cultures using a 1:100 dilution.</P><P>
3. The new microplate was placed inside the spectrophotometer and incubated at 37 C at 250 rpm.</P><P>
4. When a culture reached an OD between 0.15 and 0.20, 10-20µL samples of each culture were transferred to a fresh, transparent, flat bottom microplate containing 180-190µL of pre-warmed M9 mediate and the selective antibiotic.</P><P>
5. This microplate was placed inside the TECAN and incubated at 37 C at 250 rpm.</P><P>
6. An additional 10 µL of the media of each culture in the old TECAN plate was transferred to a round-bottom microplate that contained 190 µL PBS and 2 mg/mL kanamycin for flow cytometry measurements.</P><P>
7. Steps 4-6 were repeated until a sufficient amount of data was collected about the protein expression level. This was typically done 3-4 times which results in a total time of 24-36 hours for this experiment.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Penn_State/Protocol">http://2014.igem.org/Team:Penn_State/Protocol</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn_State/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn_State/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn_State/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn_State/Protocol&amp;oldid=364813" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TD></TR></TBODY></TABLE></DIV></DIV></DIV></BODY></HTML>