<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NJAU_China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NJAU China/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav" id="main" role="main"><UL class="menu"><LI><A href="https://2015.igem.org/Team:NJAU_China">HOME</A></LI><LI><A href="#">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Attributions">Attribution</A></LI></UL><LI><A>PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Experiments">Experiments&amp;Protocol</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Parts ">Team Parts</A></LI><LI class="active"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Practices">Human Practice</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Modeling ">MODEL</A></LI><LI><A href="https://2015.igem.org/Team:NJAU_China/Measurement">MEASUREMENT</A></LI></DIV><TITLE>Team:NJAU_CHINA/Results</TITLE><DIV class="main_content"><DIV class="section"><H1>preliminary experiment</H1><P>In order to realize the function of memory in our experiment, we must prove two things: the first is that the function of oriT is sufficient and necessary. In another word, the only function of the sequence of oriT is that it can completely be the starting site of the transfer. The second is that we can control the process of conjugation, control the expression of one key protein for example. In our experiments we chose to control traI, one kind of DNA helicase which works in the process of conjugation.</P><H2>1.The function of oriT</H2><UL><LI><SPAN>Experimental plan</SPAN><P>Using the PCR method, get the target gene fragment, oriT. Connect oriT with the plasmid vector, which contains ampicillin resistance gene. The new plasmid was transformed into E.coli K-12 (HfrH). Get the recon. And co- cultivate it with E.coli BL21 (DE3), which contains the chloramphenicol resistance gene. Dilution coating on solid culture medium, observation of whether there is colony growths.</P></LI><LI><SPAN>Experiment expectation</SPAN><P>We used a double resistance medium. In theory, only the E.coli which contains two kinds of resistance genes has the ability to grow on the medium. If the BL21 (DE3) received the plasmid by the conjugation, then it can grow on the medium. Otherwise, any of these two species of bacteria cannot grow on the medium. Therefore, if there are colonies growing on the culture medium, the conjugation process is bound to occur. The function of oriT is verified.</P></LI><LI><SPAN>Experiment results</SPAN><P id="“figure”">Figure 1</P><P id="“figure”">Figure 2</P><P id="“figure”">Figure 3</P><P>Figure 1-3: the E.coli K-12(HfrH) contains the new plasmid can grow on the ampicillin medium but can’t grow on the chloramphenicol medium.</P><P id="“figure”">Figure 4</P><P id="“figure”">Figure 5</P><P id="“figure”">Figure 6</P><P>Figure 4-6: the E.coli BL21 (DE3) can grow on the chloramphenicol medium but can’t grow on the ampicillin medium.</P><P id="“figure”">Figure 7 </P><P id="“figure”">Figure 8</P><P id="“figure”">Figure 9</P><P>Figure 7-9: after the conjugation, co- cultivate the mixed E.coli, there are some colonies growing on the Double resistance plate.</P></LI></UL><H2 style="width">2.The gene knockout of <I>traI</I> and functional deletion verification</H2><UL><LI><SPAN>The verification of gene knockout</SPAN><P>The wo rk of the knockout was done by Genscript Company. In order to verify whether the gene has been completely knocked out, we performed a molecular level validation.</P><P>traIdelseqF/R and traI-tF/R primers were used to test whether traI gene in HfrΔtraI strain is deleted. (Figure9, Figure10)The genomic DNA of the wild strain Hfr was amplified by traIdelseqF/R, which was 6065 bp (because of the length of the strip is too long to amplify), HfrΔtraI strain’s corresponds to 794 bp. Using primers traI-tF/R amplification Hfr wild strain genome obtained 412bp entries, HfrΔtraI strains and the negative control showed no amplification. These results indicate that, traI gene has been knocked out in HfrΔtraI.</P><P>
			TraIdelseqF/R and traI-tF/R sequences are as follows:
traI-tF: TGATGCACATAATCAGGCCGTG
traI-tR: AGTCTGTGAGCGTCCGGAAAA
traIDelseq-F: CACTGGCGGGTTTGTTCTTT
traIDelseq-R: CACTGCGCGTTTCACACCAT</P><P id="“figure”">Figure 10</P><P id="“figure”">Figure 11</P></LI><LI><SPAN>The verification of HfrΔtraI</SPAN><P>Because of the lack of traI in the strain HfrΔtraI, There will be a totally different result when the functional verification of oriT is repeated.</P><P id="“figure”">Figure 12</P><P id="“figure”">Figure 13</P><P id="“figure”">Figure 14</P><P>Figure 11-13: the HfrΔtraI contains the new plasmid can grow on the ampicillin medium but can’t grow on the chloramphenicol medium.</P><P id="“figure”">Figure 15</P><P id="“figure”">Figure 16</P><P id="“figure”">Figure 17</P><P>Figure 14-16: after the conjugation, co- cultivate the mixed E.coli, there are still no colonies growing on the Double resistance plate.</P><P>If we compare these two texts (figure 17 &amp; figure 18), we can find when traI was expressed and shew biological activity, the conjugation was on the rails. If not, the conjugation will not be normal.</P><P>Therefore, the traI gene is proved to be a key gene in the conjugation process, without which the conjugation cannot be normal. The conclusion of this step is the base of controlling the conjugation process.</P><P id="“figure”">Figure 18</P><P id="“figure”">Figure 19</P></LI></UL><H1>Formal experiment</H1><H2>1.Protocols of testing shuttle</H2><UL><LI><SPAN>shuttle vector design</SPAN><P>The shuttle plasmid will be shuttled from donor cell to recipient cell by bacterial conjugation. And It will express Green Fluorescent Protein (GFP) while in E.coli BL21（recipient cell）.</P></LI><LI><SPAN>Plasmid Construction.(gene circuit)</SPAN></LI><LI><SPAN>Strains.</SPAN><P>Hfr strains are the donor cell of bacterial conjugation and they are used to harbor shuttle plasmid.</P><P>Hfr(△traI) strains harbored shuttle plasmid too , but it can’t conjugate bacterially.</P><P>BL21 strains are the recipient cell of bacterial conjugation. </P></LI><LI><SPAN>Culture</SPAN><P>Hfr strains which harbored shuttle plasmid and Hfr(△traI) which harbored shuttle plasmid and BL21 strains were grown overnight (~12 hours) at 37℃ and shaken at 200 rpm in 100 ml LB not containing antibiotic . Then put 0.3ml Hfr strains which harbored shuttle plasmid or Hfr(△traI) which harbored shuttle plasmid into 2ml clean tubes together with 1ml BL21 . All tubes were maintained at 37℃. Taking 200μl into a new, identical set of 96-well plates from tubes and then measured OD600 and fluorescence at intervals. </P></LI><LI><SPAN>Result</SPAN><P>We have transformed the date into fpu (fluorescence per OD600 units), while the result was not up to our expectations. For more detail, we did not test obvious fluorescence differences in the mixed strains. As we have did the preliminary experiment which confirms the function of oriT is verified, we discussed the reason probably is that the efficiency of conjugation was not with a high performance so that it is not easy to test fluorescence. In the future, we will try to find an appropriate proportion between these strains.</P></LI></UL><H2>2.The test of blue light sensor--- BBa_K1649003</H2><UL><LI><SPAN>Design</SPAN><P>Blue light sensor plasmid constitutively expressing YF1 with its regulated promoter FixK2 following GFP reporter. The sensor is inactive under blue light and active in dark. So this part can be used to test whether the blue light sensor works.</P></LI><LI><SPAN>Plasmid Construction.(gene circuit)</SPAN><P>BBa_K1649003 was cloned into pET28a using the NotI and XbaI sites. This plasmid contains a Kanamycin resistance gene and a pBR322 origin.</P></LI><LI><SPAN>Strains.</SPAN><P>BL21 strains are used to harbor blue light sensor plasmid. The control is BL21 strain.</P></LI><LI><SPAN>Culture</SPAN><P>BL21 strains transformed with blue light sensor plasmid (marked as pCNGG) and BL21 strains ( negative control) were each picked 3 colonies and grown overnight (~12 hours) at 37℃ and shaken at 200 rpm in 3 ml LB containing antibiotic. The overnights are then diluted 1:200 into 2 mL fresh media containing antibiotics in 2 Costar 24-well plates. One plate was exposed with blue LED which we call p1 and the other were wrapped in aluminum foil and kept in darkness which we call p2. All plates were maintained at 37℃. Each strain was taken 100μl into a new, identical set of 96-well plates and then measured OD600 and fluorescence at intervals. </P></LI><LI><SPAN>Result </SPAN><P>fpu: fluorescence per OD600 units</P><P>
			t-test:
			Assumption: 
			There is no significant difference between the pCNGG-dark and the pCNGG-light data.
			Test calculation:
			The average of pCNGG-dark = 20971.72; the average of pCNGG-light = 8431.88
			So the average of defference = 12539.84
			Based on the comparison method based on the t- test, because of the free degree = 7
			So the standard error = 959.58
			And the t = 13.0680198
			Because in the one-tailed test, t0.05 = 4.029 &lt; t,
			So negative the assumption, and there is an extremely significant difference between the pCNGG-dark and the pCNGG-light data.
			And we can consider that the blue light sensor truly works.</P></LI></UL><H2>3.Red light sensor BBa_K1649001</H2><UL><LI><SPAN>Design</SPAN><P>This part is a red light sensor combined with a tetR not logical gate followed by a GFP reporter. GFP can be used to evaluate the efficiency of red light sensor when this part is regulated by different promoters by measuring the florescence. In our experiment, the promoter is T7 promoter which is originally on pET28a and can be regulated by IPTG. In the preliminary experiment, we have searched out the suitable concentration of TPTG which are 0.6 mM and 0.8 mM. The sensor is inactive in dark and active under the red light.</P></LI><LI><SPAN>Plasmid Construction.(gene circuit)</SPAN><P>BBa_K1649001 was cloned into pET28a using the NotI and XbaI sites. This plasmid contains a Kanamycin resistance gene and a pBR322 origin.</P></LI><LI><SPAN>Strains.</SPAN><P>BL21 strains are used to harbor red light sensor plasmid. The control is BL21 strains.</P></LI><LI><SPAN>Culture</SPAN><P>BL21 strains transformed with red light sensor plasmid and BL21 strains ( negative control) were each picked 5 colonies and grown overnight (~12 hours) at 37℃ and shaken at 200 rpm in 3 ml LB containing antibiotic Kanamycin. The overnights were then diluted 1:500 into 100 mL fresh media in flasks and shaken at 200 rpm for 2 hours at 37℃. Group 1: 5 flasks containing BL21 strains that harbor red light sensor plasmid and adding antibiotic Kanamycin and 0.6 mM IPTG. Group 2: 5 flasks containing BL21 strains that harbor red light sensor plasmid and adding antibiotic Kanamycin and 0.8 mM IPTG. Group 3: 5 flasks containing BL21 strains and adding antibiotic Kanamycin and 0.8 mM IPTG. Each strain were then taken 200μl into 2 new, identical set of 96-well plates. One plate was exposed to red LED which we call p1 and the other were wrapped in aluminum foil and kept in darkness which we call p2. All plates were maintained at 37℃ and measured OD600 and fluorescence at intervals. </P></LI><LI><SPAN>Result</SPAN><P>From the statistics, we have found that 0.6 mM IPTG has a better induction, so the dates shown below are from 0.6 mM IPTG induced experiment. </P><P>fpu: fluorescence per OD600 units</P><P>Figure. The fluorescence curve of part with and without the presence of red light.</P><P>The raw data is above. To figure out whether this part works with the presence of light, we need to do paired two-sample t-test.</P><P>
			We made an assumption that there’s no difference between two samples. This made μd=0
We set the significance level as 5%.
We use the test function in Microsoft Office Excel to figure out the consequence of paired two-sample t-test (one tail).
P (&gt;t) =0.0477&lt;0.05
Result: The assumption is false. The red light do have positive effect on our part.</P></LI></UL><H2>4.CRISPER-Cas9 system</H2><UL><LI><SPAN>Design:</SPAN><P>CRISPER-Cas9 system with special sgRNA is one kind of mechanism to clean foreign invasive nucleic acid effectively. In our design, we chose this system to be the main part in Erase-plasmid. However, because of the difficulty in ligating this system with red light receptor, we improve the way to test the function of it. We construct two kind if plasmids and use two-plasmid system to make it come true. The first plasmid is pet28a with cas9 under control of T7 promoter. Another is pSB1C3 with the the sequence of GFP sgRNA and GFP under control of two different constitute promoters. When these two plasmids exist in the same cell, GFP and GFP sgRNA will express continuously. When induced by IPTG or other inducers, T7 promoter will transfer cas9 protein. With the guide of sgRNA, cas9 protein could target to bind on the GFP and cut this sequence in order to accomplish the process of “erase”.</P></LI><LI><SPAN>Strain: DH-5α, BL21</SPAN></LI><LI><SPAN>Plasmid diagram:</SPAN><SPAN>pSB1C3:</SPAN><SPAN>pet28a:</SPAN></LI><LI><SPAN>Protocol:</SPAN><P>
			Construct two kind of plasmids. One is pet28a with cas9. Another is pSB1C3 with the sequence of GFP sgRNA and GFP under control of two different constitute promoters.
Cotransform these plasmids into BL21. Culture the cells overnight in 3mL LB.
Dilute the cultures for 50 times with fresh LB and culture it in 37℃ until OD600 0.5-0.6.
Divide the cultures into each tube for 1mL. Induce the cells with different concentration IPTG (0mM, 0.1mM, 0.3mM, 0.5mM, 0.7mM, 1.0mM) and continue to shake culture.
Add 150ul in each hole of the 96-well plate and use microplate reader to test the fluorescence change in 2h, 4h, 6h, 8h, 10h.
			</P></LI><LI><SPAN>Result:</SPAN><P>Unfortunately, because of time limitation, we could not finish this experiment before the deadline of wiki. But we still try to accomplish it and hope to achieve nice result. Thus, if you are interested in this experiment, please pay attention to our presentation, we will show you the latest data of the experiment. </P></LI></UL><P>References: 
Levskaya, A., Chevalier, A. A., Tabor, J. J., Simpson, 673Z. B., Lavery, L. A., Levy, M. et al. (2005). Synthetic biology: engineering Escherichia coli to see light. Nature, 438, 441–442.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>