<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Berkeley_Assembly"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UC_Berkeley/Assembly">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Berkeley/Assembly&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/Assembly&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/Assembly&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Berkeley/Assembly" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Berkeley/Assembly</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="menuminwidth0"><DIV class="menuminwidth1"><DIV class="menuminwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2008.igem.org/Team:UC_Berkeley" class="menu">Home</A></LI></UL><UL><LI><A class="menu">Project Overview</A></LI><LI><A title="Layered Assembly" href="https://2008.igem.org/Team:UC_Berkeley/LayeredAssembly"> Project Motivation</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Gateway</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Gateway Overview" href="https://2008.igem.org/Team:UC_Berkeley/GatewayOverview">Gateway Overview</A></LI><LI><A title="Plasmid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPlasmid">Plasmid Based Gateway</A></LI><LI><A title="Genomic Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayGenomic">Genomic Based Gateway</A></LI><LI><A title="Phagemid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPhagemid">Phagemid Based Gateway</A></LI></UL><LI><A><SPAN class="drop"><SPAN>Self Lysis</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Self Lysis Device" href="https://2008.igem.org/Team:UC_Berkeley/LysisDevice">Experimental Lysis Device</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Modeling" title="Modeling of the lysis device">Modeling</A></LI><LI><A title="Assembly" href="https://2008.igem.org/Team:UC_Berkeley/Assembly"> Assembly</A></LI><LI><A title="Protein Purification" href="https://2008.igem.org/Team:UC_Berkeley/ProteinPurification"> Protein Purification</A></LI><UL><LI><A class="menu">Human Practices</A></LI><LI style="width:15em;"><A title="Summary" href="https://2008.igem.org/Template:Team:UC_Berkeley/Notebook/MT_anthropological_narrative" style="width:15em;"> Summary and Narrative</A></LI><LI style="width:15em;"><A title="Ars Synthetica" href="http://www.ars-synthetica.net" style="width:15em;"> Ars Synthetica</A></LI><LI style="width:15em;"><A style="width:15em;"><SPAN class="drop"><SPAN>Picture Slideshows</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage/detail/08a9f807d">A Day in the Life of an iGem-er</A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage_express/detail/0a23ffbf4">How to Become a Synthetic Biologist</A></LI></UL><UL><LI><A class="menu">Team</A></LI><LI><A title="Our Team" href="https://2008.igem.org/Team:UC_Berkeley/Team">Who We Are</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Notebooks</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Jin_Huh">Jin's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Molly_Allen">Molly's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Chris_Brown">Christie's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Aron_Lau">Aron's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Marlee_Tichenor">Marlee's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Madhvi_Venkatesh">Madhvi's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Bing_Xia">Bing's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Dirk_VandePol">Dirk's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Cici_Chen">Cici's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Sherine_Cheung">Sherine's Notebook</A></LI></UL><UL><LI><A class="menu">Resources</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Protocols" title="Protocols">Protocols</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9Tu4QZmdP-lKw&amp;hl=en" title="Our Parts">Our Parts</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9S0htjL2N6qGA&amp;hl=en" title="Our Oligos">Our Oligos</A></LI><LI><A href="http://spreadsheets.google.com/pub?key=pJRu8g118nzQQFOwGNqrgZA" title="Our Stocks">Our Stocks</A></LI></UL><UL><LI><A class="menu">External Links</A></LI><LI><A title="UCB Computational team" href="https://2008.igem.org/Team:UC_Berkeley_Tools">UCB Comp Team</A></LI></UL></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><FONT size="6"><B>Assembly</B></FONT></DIV><P>Our goal is to simplify 2ab assembly reactions by replacing the mini-prep steps and making the protocol reagent-free. This will be accomplished by using our lysis device to lyse cells and extract DNA and engineering cells to produce their own restriction enzymes and ligase.
</P><P>For a general overview of two antibiotic assembly, <A href="/Team:UC_Berkeley/LayeredAssembly#Assembly_Layer" title="Team:UC Berkeley/LayeredAssembly"> click here</A>.
</P><H2><SPAN class="mw-headline" id="Assembly_in_Cell_Lysate"><B>Assembly in Cell Lysate</B></SPAN></H2><P>Our lab currently uses cells that are engineered to methylate either BamHI or BglII restriction sites. Part A is transformed into a &quot;lefty&quot; cell that is methylated on BglII restriction sites while Part B is transformed into a &quot;righty&quot; cell that is methylated on BamHI restriction sites. 
</P><P>We propose to integrate the BamHI and BglII restriction enzyme genes into the <I>E. coli</I> genome. In this scheme, lefty cells methylate BglII recognition sites and will stably express T4 DNA ligase and BglII restriction enzyme. Righty cells methylate BamHI recognition sites and will be engineered to stably express Cre recombinase and BamHI restriction enzyme. Since restriction enzymes will not cut methylated DNA, the BglII restriction site in the lefty cell and the BamHI restriction site in the righty cell are blocked from digestion. The methylation also protects the cellular DNA from being cut when these genes are expressed.
</P><P>We propose to eliminate the need for mini-prep by using our lysis device and the BamHI/BglII/Cre/ligase cells to lyse the cells and release the restriction enzymes and ligase into the lysate. The lysate mixture is incubated to allow time for assembly (digestion and ligation). The lysate is then used to transform new cells and the transformed cells are plated on the appropriate antibiotic. 
</P><H3><SPAN class="mw-headline" id="Testing_and_Experimentation"><B>Testing and Experimentation</B></SPAN></H3><H4><SPAN class="mw-headline" id="A._To_test_the_viablity_of_the_plasmid_released_by_the_lysis_device"><B>A. To test the viablity of the plasmid released by the lysis device</B></SPAN></H4><P>Cells containing basic part plasmid DNA were lysed with the lysis device. The lysate was used to transform another batch of cells. This experiment produced many colonies when the cells were plated.
</P><H4><SPAN class="mw-headline" id="B._To_test_the_viability_of_enzymes_in_the_lysate"><B>B. To test the viability of enzymes in the lysate</B></SPAN></H4><P>Lefty and righty cells were combined in a single eppendorf tube and lysed with our lysis device. Commercial restriction enzymes and ligase were added to the lysate along with plasmid DNA. The mixture was incubated. The lysate was used to transform competent cells. The transformed cells were plated on the appropriate antibiotic, but failed to produce colonies.
</P><P>The experiment was repeated with lysed cells that were centrifuged and re-suspended in Buffer NEB2. This experiment produced the colonies with the correct composite part when plated on the appropriate antibiotic.
</P><H4><SPAN class="mw-headline" id="C._Testing_digestion_in_lysate"><B>C. Testing digestion in lysate</B></SPAN></H4><P>Lefty and righty cells containing plasmid DNA were centrifuged and the supernatant was discarded. Cells were re-suspended in NEB2 Buffer and lysed using our lysis device. Commercial restriction enzymes and ligase were added to the lysate and the lysate was incubated to allow time for assembly. The lysate was used then used to transform competent cells. 
</P><P>The exact conditions required to make this experiment successful are difficult to determine. Since the lysis device results in successful release of plasmid DNA and assembly works in NEB2 buffered lysate, digestion of plasmid DNA in the lysate should work under the appropriate conditions. However, at the present, we have not found the correct conditions to make this scheme viable.
</P><H4><SPAN class="mw-headline" id="D._Testing_the_viability_of_enzymes_produced_by_the_cells"><B>D. Testing the viability of enzymes produced by the cells</B></SPAN></H4><P>1)	Ligase strain â The gene for ligase has been cloned and integrated into the genome of lefty and righty cells, and they were named as Ligase Lefty (LL) and Ligase Righty (LR). These strains have successfully been used to simplify the cloning procedure. 
</P><P>To test these strains, we put tet promoter in pBjh1601CA plasmid and methylated BglII restriction site by passaging it through Lefty cells by transformation and plasmid DNA purification. Similarly the GFP with RBS was put in pBjh1601AK plasmid and BamHI restriction site was methylated by Righty cell. In a single microcentrifuge tube, 5.8uL of water, 1uL of 10X NEB2+10mM ATP, 0.3uL of each BamHI, BglII, XhoI, and T4 DNA ligase, and 1uL of each Lefty and Righty metyhlated plasmids were added. It was incubated at 37C for 1hr, and 30min at room temperature. It was then introduced into LL and LR by transformation, plated on LB agar plate with CmR and KanR, and grown overnight at 37C. As a control, we also introduced the same reaction cocktail into regular Lefty and Righty cells by transformation. A lot more colonies were growing when they were transformed into LL and LR. About 90% of the LL and LR were glowing green which meant the ligase strains are working well. Some of the colonies from the other 10% of the population that were not glowing green were sequenced, and deletion of DNA was observed which suggests that ligases are mutagenic and the expression of ligase must be regulated. 
</P><H4><SPAN class="mw-headline" id="E._Future_Work"><B>E. Future Work</B></SPAN></H4><P>The genes for BglII, BamHI and Cre will be integrated into the <I>E. coli</I> genome and tested for viability.
</P><P>The conditions needed for successful assembly in lysate must be determined through experimentation.  
</P><H2><SPAN class="mw-headline" id="Assembly_in_vivo_using_Phagemid"><B>Assembly <I>in vivo</I> using Phagemid</B></SPAN></H2><P>We propose to engineer assembler cells that stably express BamHI methylase and BglII methylase, as well as BamHI, BglII, Cre and ligase. The assembler cell will also contain all of the genes needed for phage replication. The three genes necessary to induce the lytic cycle of the phage are initially repressed.
</P><P>Bacteriophages with phagemids containing a basic part flanked by BglII and BamHI restriction sites and two antibiotic resistance genes separated by a XhoI restriction site are created. To methylate phagemid DNA, lefty and righty cells will be infected with these phages to produce lefty and righty phagemids.
</P><P>Assembler cells will be infected with both lefty and righty phagemids. The cells will produce the restriction enzymes and ligase necessary to complete an in-vivo assembly with the basic parts contained within the phage. 
</P><P>Once assembly is complete, the lytic cycle of the phage is induced by removing the repressor on the lytic genes. The cells are lysed and phagemids are released into solution. The lysate is used to infect new cells. The cells can be screened for the correct product by plating on the appropriate antibiotic.    
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UC_Berkeley/Assembly">http://2008.igem.org/Team:UC_Berkeley/Assembly</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Berkeley/Assembly" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Berkeley/Assembly" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Berkeley/Assembly&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Berkeley/Assembly&amp;oldid=105526" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>