<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Berkeley_GatewayPlasmid"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UC_Berkeley/GatewayPlasmid">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Berkeley/GatewayPlasmid&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayPlasmid&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayPlasmid&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Berkeley/GatewayPlasmid" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Berkeley/GatewayPlasmid</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="menuminwidth0"><DIV class="menuminwidth1"><DIV class="menuminwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2008.igem.org/Team:UC_Berkeley" class="menu">Home</A></LI></UL><UL><LI><A class="menu">Project Overview</A></LI><LI><A title="Layered Assembly" href="https://2008.igem.org/Team:UC_Berkeley/LayeredAssembly"> Project Motivation</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Gateway</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Gateway Overview" href="https://2008.igem.org/Team:UC_Berkeley/GatewayOverview">Gateway Overview</A></LI><LI><A title="Plasmid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPlasmid">Plasmid Based Gateway</A></LI><LI><A title="Genomic Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayGenomic">Genomic Based Gateway</A></LI><LI><A title="Phagemid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPhagemid">Phagemid Based Gateway</A></LI></UL><LI><A><SPAN class="drop"><SPAN>Self Lysis</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Self Lysis Device" href="https://2008.igem.org/Team:UC_Berkeley/LysisDevice">Experimental Lysis Device</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Modeling" title="Modeling of the lysis device">Modeling</A></LI><LI><A title="Assembly" href="https://2008.igem.org/Team:UC_Berkeley/Assembly"> Assembly</A></LI><LI><A title="Protein Purification" href="https://2008.igem.org/Team:UC_Berkeley/ProteinPurification"> Protein Purification</A></LI><UL><LI><A class="menu">Human Practices</A></LI><LI style="width:15em;"><A title="Summary" href="https://2008.igem.org/Template:Team:UC_Berkeley/Notebook/MT_anthropological_narrative" style="width:15em;"> Summary and Narrative</A></LI><LI style="width:15em;"><A title="Ars Synthetica" href="http://www.ars-synthetica.net" style="width:15em;"> Ars Synthetica</A></LI><LI style="width:15em;"><A style="width:15em;"><SPAN class="drop"><SPAN>Picture Slideshows</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage/detail/08a9f807d">A Day in the Life of an iGem-er</A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage_express/detail/0a23ffbf4">How to Become a Synthetic Biologist</A></LI></UL><UL><LI><A class="menu">Team</A></LI><LI><A title="Our Team" href="https://2008.igem.org/Team:UC_Berkeley/Team">Who We Are</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Notebooks</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Jin_Huh">Jin's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Molly_Allen">Molly's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Chris_Brown">Christie's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Aron_Lau">Aron's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Marlee_Tichenor">Marlee's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Madhvi_Venkatesh">Madhvi's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Bing_Xia">Bing's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Dirk_VandePol">Dirk's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Cici_Chen">Cici's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Sherine_Cheung">Sherine's Notebook</A></LI></UL><UL><LI><A class="menu">Resources</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Protocols" title="Protocols">Protocols</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9Tu4QZmdP-lKw&amp;hl=en" title="Our Parts">Our Parts</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9S0htjL2N6qGA&amp;hl=en" title="Our Oligos">Our Oligos</A></LI><LI><A href="http://spreadsheets.google.com/pub?key=pJRu8g118nzQQFOwGNqrgZA" title="Our Stocks">Our Stocks</A></LI></UL><UL><LI><A class="menu">External Links</A></LI><LI><A title="UCB Computational team" href="https://2008.igem.org/Team:UC_Berkeley_Tools">UCB Comp Team</A></LI></UL></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><FONT size="6"><B>Plasmid Based Gateway</B></FONT></DIV><H2><SPAN class="mw-headline" id="Introduction"><B>Introduction</B></SPAN></H2><P>The current <I>in vitro</I> LR Gateway procedure for transferring one piece of DNA into another vector without restriction enzymes involves an expensive cocktail of purified plasmids, excisionase, integrase, ihfα and ihfβ. Both an assembly vector (containing the ccdB gene flanked by attR1 and attR2 sites for negative selection) and an entry vector (containing the part(s) desired to be transferred, flanked by attL1 and attL2 sites) are used as the substrates to which the reactants are added. </P><P>Inspired by such innovation, we seek to perform this task <I>in vivo</I> using E. coli to express the reactants necessary. Since E. coli naturally express ihfα/β in their genome, we first tried to integrate excisionase and integrase genes into the genomic DNA as well. However, this proved to be toxic to the cells, making them relatively unstable. Therefore, our next approach was to introduce the reagents into the substrate plasmids. In one case, the the assembly vectors received the Gateway device of the enzymes excisionase and integrase preceded by a temperature sensitive promoter, while in the other case it was introduced into the entry vectors. In both cases, we also explored the effects of additionally introducing the ihfα and ihfβ genes behind the Gateway device. </P><P>After careful analysis of our data, we introduced our Lysis device in order to eliminate the mini-prepping procedures and to thus make the entire process cheaper and more efficient.</P><H2><SPAN class="mw-headline" id="Gateway_Device_in_Assembly_Vector:_Plasmid_Details"><B>Gateway Device in Assembly Vector: Plasmid Details </B></SPAN></H2><P><B>Assembly Vectors:</B>Two different variations of the pK112128 assembly vector were created: one with {promoter.xis.int!} and one with {promoter.xis.int!}{rbs.ihfα!}{rbs.ihfβ!} between the ccdB site and the attR2 site. These plasmids are named <A href="https://static.igem.org/mediawiki/2008/b/b3/PK112245.png" class="external text" rel="nofollow">pK112245</A> and <A href="https://static.igem.org/mediawiki/2008/f/f3/PK112246.png" class="external text" rel="nofollow">pK112246</A> respectively. A simplified version of the two variants along with the Lysis Device is shown below. </P><P><B>Entry Vector:</B>The entry vector used in both cases was <A href="https://static.igem.org/mediawiki/2008/8/85/PBca_1256.png" class="external text" rel="nofollow">pBca1256</A>.</P><H2><SPAN class="mw-headline" id="Gateway_Device_in_Entry_Vector:_Plasmid_Details"><B>Gateway Device in Entry Vector: Plasmid Details</B></SPAN></H2><P><B>Assembly Vector:</B>The assembly vector used for each of the following entry vector variations is <A href="https://static.igem.org/mediawiki/2008/7/76/PK112128.png" class="external text" rel="nofollow">pK112128</A>. </P><P><B>Entry Vectors:</B> Two different variations of the pBca1256 entry vector were created: one with {promoter.xis.int!} and one with {promoter.xis.int!}{rbs.ihfα!}{rbs.ihfβ!} just before the attL1 site. These plasmids are named <A href="https://static.igem.org/mediawiki/2008/2/2d/PK112247.png" class="external text" rel="nofollow">pK112247</A> and <A href="https://static.igem.org/mediawiki/2008/a/ae/PK112248.png" class="external text" rel="nofollow">pK112248</A> respectively.</P><H2><SPAN class="mw-headline" id="General_Procedure"><B>General Procedure</B></SPAN></H2><P>Below is a flowchart of the general plasmid based Gateway procedure. Due to various combinations of assembly and entry vectors explored, the &quot;Gateway Device&quot; was created to represent the {promoter.xis.int!} sequence with or without the adjoining {rbs.ihfα!}{rbs.ihfβ!} sequence. The &quot;Lysis Device&quot; sequence (which was only tested in the pK112245 assembly vector) was similarly simplified. </P><P>The major combinations of assembly + entry vectors are: pK112245 + pBca1256, pK112246 + pBca1256, pK112128 + pK112247, and pK112128 + pK112248. Additionally, pBca1256 with five different sized parts replacing the original mRFP part was also tested with each assembly vector to show the Gateway reaction can be done with parts of all sizes, although this is not shown in the diagram below. </P><H2><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H2><H3><SPAN class="mw-headline" id="Gateway_Device_in_Assembly_Vector"><B>Gateway Device in Assembly Vector</B></SPAN></H3><UL><LI> Control done at 30°C.
</LI></UL><UL><LI> As can be seen above, there is a relatively high rate of cotransformation as seen by the growth of colonies on the Spec plates. The trials done at 30°C had a higher rate of cotransformation likely because there were fewer plasmids resulting from the Gateway reaction.
</LI><LI> In an attempt to reduce cotransformation rates in those grown normally at 37°C, we repeated these experiments, picking more colonies and diluting the purified protein mixture 20x before transformation into MG1061 cells. The results are shown below.
</LI></UL><UL><LI> Although these plates have not been allowed to grow as long and thus are not as bright in color, it can clearly be seen that cotransformation has been successfully eliminated.
</LI></UL><P><B>The Lysis Device</B></P><UL><LI> The lysis device was inserted into the pK112245, and the experiment repeated. However, only 2 colonies grew, although both were free of cotransformation. 
</LI></UL><H3><SPAN class="mw-headline" id="Gateway_Device_in_Entry_Vector"><B>Gateway Device in Entry Vector</B></SPAN></H3><UL><LI> As can been seen above, the Gateway reaction worked with much more efficiency as evidenced by the low rates of cotransformation.
</LI></UL><H2><SPAN class="mw-headline" id="Materials_.26_Methods"><B>Materials &amp; Methods</B></SPAN></H2><P>We first prepared competent TG1 cells (resistant to ccdB) with the given assembly vector. The assembly vector was transformed into competent TG1 cells and were grown in liquid LB media with Cam/Amp antibiotics (and 50mM Mg<SUP>+2</SUP> if the lysis device is included) at 30°C overnight. The following day, 20 ul of the saturated culture was taken out, and was grown under the same conditions to mid-log. About 1 ml of culture was pelleted, the supernatant was discarded and the cells were resuspended in 10 ul KCM plus 90 ul TSS on ice. </P><P>
Using these new competent cells, the given entry vector was tranformed into them. They were grown in liquid LB media with Cam/Amp/Spec antibiotics (and 50mM Mg<SUP>+2</SUP> if the lysis device is included) at 37°C overnight to ensure the cells contain both plasmids and that the temperature sensitive promoter of the gateway device would turn on. A control in which the cells were instead kept at 30°C was conducted to compare our results with those in which the gateway device was theoretically off. </P><P>
The following day, the plasmid from about 2 ml of saturated culture was purified. When there was no lysis device in the assembly vector, plasmid purification was achieved with the QIAprep Spin Miniprep Kit. In the case where the lysis device is integrated into the assembly plasmid, the culture was pelleted, the supernatant removed, resuspended in 1 ml PBS, pelleted, the supernatant removed, and finally resuspended again in 100 ul PBS. </P><P>
The purified plasmids were next transformed into MG1061 cells (sensitive to ccdB). These were plated on LB agar plates with Cam/Amp antibiotics and grown overnight at 37°C. The next day, 16 colonies were picked from each plate, spotted on a Cam/Amp antibiotic plate, as well as Spec antibiotic plate to test for cotransformance of the entry vector with the desired product. </P><P>
The above procedure were conducted in parallel with different combinations of assembly and entry vectors. The following combinations of assembly and entry vectors were done: pK112245 + pBca1256, pK112246 + pBca1256, pK112128 + pK112247, and pK112128 + pK112248. Each combination included a control in which the Gateway device was not turned on during the allotted time for the reaction to occur (i.e. they were grown at 30°C, when the temperature sensitive promoter is off). In addition to this, three trials of each were conducted. Finally, in order to see if gateway could be performed on parts of different sizes, five different parts in pBca1256 were chosen (Bca1117, Bca1133, Bca1270, Bca1252, and Bca1168) and tested with the assembly vector pK112245, as well as with pK112245. One colony from each was sequenced to ensure gateway had occurred by identification of attB1 and attB2 sites as well as the parts in the assembly vectors. </P><P>
After data analysis, we suspected that higher efficiency could be achieved for the schemes in which the Gateway Device was in the assembly vector if less plasmid was used to transform into MG1061 cells. Therefore, we took the post-reaction purified plasmid mixtures from the trials involving pK112245 + pBca1256 and pK112246 + pBca1256 and diluted each 20 times. Three trials for each in which the 20x dilution of the purified plasmid was transformed into MG1061 cells was performed under the same conditions, and 32 colonies from each plate were picked the following day. </P><P>
Other controls were also performed. Each assembly vector used was also transformed into into MG1061 cells which were put on Cam/Amp antibiotic LB agar plates to make sure ccdB was sufficient for negative selection. Only a very few number of white colonies grew for each, which was expected since it is known that ccdB is prone to mutation and weakening. The entry vectors were also each individually transformed into MC1061 cells and plated on Cam/Amp antibiotic plates. No colonies appeared, proving that the entry vector did not contain Cam/Amp resistance. </P><H2><SPAN class="mw-headline" id="Discussion"><B>Discussion</B></SPAN></H2><UL><LI> Our desired product is a reacted assembly vector with the entry vector mRFP (or part) between attB1 and attB2 sites. Since the origin of replication in the assembly vectors is low-copy, the cells should appear light pink in color.
</LI></UL><UL><LI> Colonies that are bright pink, which grow on both Cam/Amp and Spec antibiotic plates, are cotransformed. This means that in addition to our desired assembly vector-with-part plasmid, one of the original, unreacted entry vectors has entered into the cell. These appear as bright pink because the origin of replication in the entry vector is high-copy.
</LI></UL><UL><LI> White colonies first appeared to be evidence of a mutation in the ccdB gene such that it inhibits its toxicity. Since ccdB is used to negatively select against the original, unreacted assembly vector containing ccdB flanked by attR1 and attR2, a mutation which weakens ccdB such that a strain sensitive to normal ccdB can grow in spite of the gene's presence and gives undesirable background products. However, after sequencing 4 different white colonies, we discovered that in each case the mRFP gene was indeed inserted into the assembly vector and was flanked by attB1 and attB2 sites (our desired product); the only problem was that there were deletions (24 or 25 bp in length) in the promoter of the part which was driving the expression of mRFP! Thus, the few white colonies seen are likely due to mutations in the original part rather than a mutation in ccdB.
</LI></UL><UL><LI> There are also some colonies that are very slightly pink. We predict that these are cells which contain the desired assembly vector-with-part plasmid, but that the part itself has sustained some sort of mutation (perhaps again in the promoter) that slightly weakens the expression of mRFP.
</LI></UL><UL><LI> Proportionally, the less background we receive at the end of the experiment, the more efficient the plasmid based Gateway reaction is.
</LI></UL><UL><LI> From our experimental results comparing the &quot;Gateway Device&quot; with and without ihfα/β, we conclude that the additional expression of ihfα/β is unnecessary and makes no difference in the plasmid based gateway reactions. However, based on the difficulty of previous attempts to join and clone ihfα and ihfβ, it has been concluded that these parts are unnecessary or even hazardous to cells, and thus will not be explored further in other Gateway schemes.
</LI></UL><UL><LI> Our data suggests that our procedure was most efficient when the Gateway Device was located in the entry vectors as opposed to the assembly vectors. This makes sense since the entry vector origin of replication is high copy, meaning the reagents necessary for the reaction to occur (excisionase, integrase, ihfα and ihfβ) will be expressed at much higher levels when the temperature sensitive promoter is turned on. More reagents produced increases the probability that the Gateway reaction will occur within a given cell, and that we will get our desired product with the greatest efficiency. However, from experience in creating and cloning the Gateway Device, we know that this composite part is somewhat toxic to the cell since there are att sites in the genomic DNA of E. coli. Thus, it is necessary to carefully control this device with, in this case, a temperature sensitive promoter. This also means that if the &quot;part&quot; in your entry vector is also somewhat toxic, the additional toxicity of the Gateway Device may make the entry vector difficult or even impossible to make in the first place.
</LI></UL><UL><LI> Therefore, it is most beneficial to put the Gateway Device into the assembly vector. In order to improve the efficiency and reduce the background in these cases, we diluted the post-reaction purified plasmid mixtures from the trials involving pK112245 + pBca1256 and pK112246 + pBca1256. Our results show a significant reduction in cotransformation and thus a higher rate of efficiency.
</LI></UL><UL><LI> The entry vectors with the various part lengths replacing mRFP in pBca1256 all sequenced correctly; the attB1 and attB2 sites were confirmed for all experiments. This suggests that the plasmid based gateway reaction with pK112245 and pK112246 assembly vectors works for parts as short as 250bp and as long as 3078bp.
</LI></UL><UL><LI> All in all, we were able to successfully conduct the Gateway reaction <I>in vivo</I> using assembly and entry plasmids with relatively high efficiency. Our problems with cotransformation seem to largely reduced upon diluting the purified plasmid before transformation. The white colony background due to mutated, weakened ccdB may be solved by replacing it with or adding on another more robust toxic protein, such as barnase, to aid in negative selection. Another possibility would be to use positive selection, for example by using conditional replication of an origin.
</LI></UL><UL><LI> Although the plasmid based Gateway scheme has proven to be successful, we have also tried other methods to achieve the Gateway reaction: see Phagemid Based Gateway and Genomic Based Gateway.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UC_Berkeley/GatewayPlasmid">http://2008.igem.org/Team:UC_Berkeley/GatewayPlasmid</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Berkeley/GatewayPlasmid" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Berkeley/GatewayPlasmid" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayPlasmid&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayPlasmid&amp;oldid=103367" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>