<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Berkeley_GatewayOverview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UC_Berkeley/GatewayOverview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Berkeley/GatewayOverview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayOverview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayOverview&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Berkeley/GatewayOverview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Berkeley/GatewayOverview</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="menuminwidth0"><DIV class="menuminwidth1"><DIV class="menuminwidth2"><DIV class="menu"><UL><LI><A title="Home" href="https://2008.igem.org/Team:UC_Berkeley" class="menu">Home</A></LI></UL><UL><LI><A class="menu">Project Overview</A></LI><LI><A title="Layered Assembly" href="https://2008.igem.org/Team:UC_Berkeley/LayeredAssembly"> Project Motivation</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Gateway</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Gateway Overview" href="https://2008.igem.org/Team:UC_Berkeley/GatewayOverview">Gateway Overview</A></LI><LI><A title="Plasmid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPlasmid">Plasmid Based Gateway</A></LI><LI><A title="Genomic Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayGenomic">Genomic Based Gateway</A></LI><LI><A title="Phagemid Gateway" href="https://2008.igem.org/Team:UC_Berkeley/GatewayPhagemid">Phagemid Based Gateway</A></LI></UL><LI><A><SPAN class="drop"><SPAN>Self Lysis</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A title="Self Lysis Device" href="https://2008.igem.org/Team:UC_Berkeley/LysisDevice">Experimental Lysis Device</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Modeling" title="Modeling of the lysis device">Modeling</A></LI><LI><A title="Assembly" href="https://2008.igem.org/Team:UC_Berkeley/Assembly"> Assembly</A></LI><LI><A title="Protein Purification" href="https://2008.igem.org/Team:UC_Berkeley/ProteinPurification"> Protein Purification</A></LI><UL><LI><A class="menu">Human Practices</A></LI><LI style="width:15em;"><A title="Summary" href="https://2008.igem.org/Template:Team:UC_Berkeley/Notebook/MT_anthropological_narrative" style="width:15em;"> Summary and Narrative</A></LI><LI style="width:15em;"><A title="Ars Synthetica" href="http://www.ars-synthetica.net" style="width:15em;"> Ars Synthetica</A></LI><LI style="width:15em;"><A style="width:15em;"><SPAN class="drop"><SPAN>Picture Slideshows</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage/detail/08a9f807d">A Day in the Life of an iGem-er</A></LI><LI style="width:20em;"><A style="width:20em;" href="http://www.vuvox.com/collage_express/detail/0a23ffbf4">How to Become a Synthetic Biologist</A></LI></UL><UL><LI><A class="menu">Team</A></LI><LI><A title="Our Team" href="https://2008.igem.org/Team:UC_Berkeley/Team">Who We Are</A></LI><LI><A align="left"><SPAN class="drop"><SPAN>Notebooks</SPAN>
             &gt;&gt;
          </SPAN></A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Jin_Huh">Jin's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Molly_Allen">Molly's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Chris_Brown">Christie's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Aron_Lau">Aron's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Marlee_Tichenor">Marlee's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Madhvi_Venkatesh">Madhvi's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Bing_Xia">Bing's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Dirk_VandePol">Dirk's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Cici_Chen">Cici's Notebook</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Notebook/Sherine_Cheung">Sherine's Notebook</A></LI></UL><UL><LI><A class="menu">Resources</A></LI><LI><A href="https://2008.igem.org/Team:UC_Berkeley/Protocols" title="Protocols">Protocols</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9Tu4QZmdP-lKw&amp;hl=en" title="Our Parts">Our Parts</A></LI><LI><A href="http://spreadsheets.google.com/ccc?key=pUQEpr4ZqU9S0htjL2N6qGA&amp;hl=en" title="Our Oligos">Our Oligos</A></LI><LI><A href="http://spreadsheets.google.com/pub?key=pJRu8g118nzQQFOwGNqrgZA" title="Our Stocks">Our Stocks</A></LI></UL><UL><LI><A class="menu">External Links</A></LI><LI><A title="UCB Computational team" href="https://2008.igem.org/Team:UC_Berkeley_Tools">UCB Comp Team</A></LI></UL></DIV></DIV></DIV></DIV><DIV style="text-align: center;"><FONT size="6"><B>Gateway Overview</B></FONT></DIV><H2><SPAN class="mw-headline" id="Why_Use_Gateway.3F">Why Use Gateway?</SPAN></H2><P>The first step of the layered assembly scheme involves the transfer of biobrick parts from an entry vector to a double antibiotic assembly vector. Traditionally, this would require a fairly work-intensive protocol requiring digestion, gel purification, ligation, transformation, and plasmid isolation. In addition to being more time-consuming, the aforementioned procedure is also suboptimal because it is difficult to scale-up.
</P><P>The Gateway Cloning approach developed by <A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cloning/Gateway-Cloning.html" class="external text" rel="nofollow">Invitrogen</A> offers a more efficient and convenient alternative for parts transfer. Their procedure involves the enzyme-catalyzed exchange of parts flanked by specific recombination sites. Experimentally, it is a one-pot, room-temperature reaction where the plasmids, buffer, water and enzymes are added together. After the addition of another enzyme and a short incubation period to terminate the reaction, the entire mixture can be transformed directly. This one-pot approach with a fewer steps is much more suitable for large-scale experimentation.
</P><P>Gateway is commonly used to facilitate the transfer of a single gene of interest from an entry clone to multiple destination vectors, as shown below. The efficiency and robustness of the Gateway mechanism are ideal for this application because once the gene of interest is cloned and confirmed in the entry vector, subsequent transfers using Gateway need not be confirmed again. Thus, it is ideal for use in the first step of layered assembly where a part may be transferred to one or more of the double antibiotic vectors required for the subsequent assembly steps assembly.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">A entry vector generated from any of the methods can be transferred to various different vectors using the Gateway method. Image source:<A href="https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC" class="external free" rel="nofollow">https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC</A></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Gateway_Chemistry">Gateway Chemistry</SPAN></H2><H3><SPAN class="mw-headline" id="The_Natural_Lambda_Phage_Recombination_system">The Natural Lambda Phage Recombination system</SPAN></H3><P>In general, Gateway reactions in the lab involve the attB, attP, attL, and attR recombination sites and the integrase (Int), excisionase (Xis), and integration host factor (IHF) enzymes.  These  enzymes were found in nature in the temperate bacteriophage lambda.  Like all temperate bacteriophages, lambda utilizes a lysogenic infection life cycle, wherein its genome is incorporated into the genome of the <I>E. coli</I> host genome, to excise itself at a later time.  Integrase (Int), excisionase (Xis), and integration host factor (IHF) are the enzymes that catalyze the integration and excision of the viral genome, at the previously mentioned <I>att</I> sites of the viral and host genomes. 
</P><P>Int cooperatively binds with IHF (which is composed of A and B subunits) in order to catalyze both the integration and excision reactions in the natural lambda phage system shown below. Although int can independently catalyze both reactions, IHF greatly improves int's binding affinity for the att recombination sites by bending the DNA [6]. Although int performs both the forward and reverse reactions, the equilibrium heavily favors the reaction of attB and attP to produce attL and attR sites. Xis binds to the attR sire and serves to shift the equilibrium so that it favors the reverse reaction [5].
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:528px;"><DIV class="thumbcaption">Lambda phage recombination in <I>E. coli</I>. Invitrogen adapted these components from this natural system to produce their Gateway cloning scheme.  Image source: <A href="http://tools.invitrogen.com/downloads/gateway-the-basics-seminar.html" class="external free" rel="nofollow">http://tools.invitrogen.com/downloads/gateway-the-basics-seminar.html</A></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Invitrogen.27s_adaptation">Invitrogen's adaptation</SPAN></H3><P>In the Invitrogen scheme, the recombination sites are found in pairs flanking sequences that are intended for transfer. The recombination pairs are directional and specificity is given by ten nucleotides in the core region of each site, as shown below.  In addition, the lethal gene <I>ccdB</I> is incorporated in between the recombination sites in the destination vector to ensure that only the desired recombined product can be cloned.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:668px;"><DIV class="thumbcaption">Directional, site-specific recombination between attL and attR sites.  Image source:<A href="https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC" class="external free" rel="nofollow">https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC</A></DIV></DIV></DIV></DIV><P>The schematic below depicts the LR reaction, during which the attL sites recombine with attR to yield attB and attP sites. The BP reaction proceeds in the opposite direction yielding attL and attR sites.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:780px;"><DIV class="thumbcaption">Gateway LR reaction where gene of interest (flanked by attL sites) is transferred to destination vector containing attR sites.  Image source:<A href="https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC" class="external free" rel="nofollow">https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC</A></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Selection_for_Correct_Clones">Selection for Correct Clones</SPAN></H4><P>The above scheme employs both ccdB negative selection and antibiotic selection in order to yield &gt;90% of colonies containing the desired expression clone. The entry and destination vectors contain different antibiotic resistances, so plating on the desired antibiotic (Ampicillin in case shown above) eliminates clones containing the entry vector or the by-product of the reaction. In addition, <I>ccdB</I> is a lethal gene, thereby eliminating colonies containing the destination vector, which would otherwise survive on the antibiotic plate.
</P>

Animation source: <A href="http://www.bio.davidson.edu/courses/Molbio/MolStudents/spring2000/patton/gateway.htm" class="external free" rel="nofollow">http://www.bio.davidson.edu/courses/Molbio/MolStudents/spring2000/patton/gateway.htm</A><H2><SPAN class="mw-headline" id="Gateway_in_vivo">Gateway <I>in vivo</I></SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_Based_Gateway">Plasmid Based Gateway</SPAN></H3><P>We applied our ideas for facilitating the automation of synthetic biology to the first step of layered assembly--the LR gateway reaction used to move the gene of interest into one or more of the double-antibiotic assembly vectors.  We began by attempting to reproduce a plasmid based Gateway reaction, which was similar to that of Invitrogen's Gateway scheme. Although we were successful in having a completely <I>in vivo</I> version of the Gateway reaction, there were two major drawbacks: we had not eliminated the need for laborious and expensive mini-preps and we had relatively high background from ccdB mutations (the ccdB gene is relatively unstable because of its toxicity).
</P><P>We addressed the first drawback by creating a self-lysis device which allows the cell's contents, including the desired product, to be released when arabinose is added. This device was inserted into either the entry or assembly plasmid to facilitate the isolation of the reaction products.
</P><P>The second issue was more complex and was resolved by using positive selection methods rather than ccdB negative selection. Efforts to incorporate positive selection for Gateway led to the development of the Genomic Based Gateway scheme.
</P><H3><SPAN class="mw-headline" id="Genomic_Based_Gateway">Genomic Based Gateway</SPAN></H3><P>The Genomic Based Gateway scheme involves placement of the assembly vector in the genome. The gene of interest from the entry vector can recombine with the genome to yield the desired product. Both the entry vector and assembly vector have conditional origins of replication which serve as mechanisms for positive selection. Although this is a viable scheme for Gateway and can readily incorporate the lysis device in place of mini-preps, it still requires transformation of the lysate in order to select for the desired product. In an effort to further optimize our Gateway scheme by eliminating the need for transformation, we developed a Phagemid Based Gateway scheme.
</P><H3><SPAN class="mw-headline" id="Phagemid_Based_Gateway">Phagemid Based Gateway</SPAN></H3><P>The Phagemid Based Gateway utilizes a plasmid that can be packaged in a phage when a lysogenic cell is induced with arabinose. The assembly vector in this scheme contains a phagemid device, which allows the product to be packaged and transferred to another cell without mini-preps or transformation. In addition to simplifying the transfer of the product, the phagemid device serves as a positive selection mechanism and can isolate the desired product when paired with another positive selector, such as an inducible origin of replication.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI><A href="http://tools.invitrogen.com/downloads/gateway-the-basics-seminar.html" class="external free" rel="nofollow">http://tools.invitrogen.com/downloads/gateway-the-basics-seminar.html</A></LI><LI><A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cloning/Gateway-Cloning.html|Gateway" class="external free" rel="nofollow">http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cloning/Gateway-Cloning.html|Gateway</A></LI><LI><A href="https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC" class="external free" rel="nofollow">https://commerce.invitrogen.com/index.cfm?fuseaction=iProtocol.unitSectionTree&amp;treeNodeId=290D0521B5FFA1B782C62C0AB62FD7BC</A></LI><LI> Cho, E et al. Interactions between Integrase and Excisionase in the Phage Lambda Excisive Nucleoprotein Complex. <I>Journal of Bacteriology</I>. September 2002; 184(18): 5200â5203. Available Online: <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=135313" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=135313</A> (Accessed: 28 October 2008).
</LI><LI> Frumerie, C et al. Cooperative interactions between bacteriophage P2 integrase and its accessory factors IHF and Cox. <I>Virology</I>. 5 February 2005; 232(1):284-294. Available Online: <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WXR-4F29SN2-1&amp;_user=4420&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=4420&amp;md5=0f9e41ba422140f157a2b1f1fb60b140" class="external free" rel="nofollow">http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WXR-4F29SN2-1&amp;_user=4420&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_version=1&amp;_urlVersion=0&amp;_userid=4420&amp;md5=0f9e41ba422140f157a2b1f1fb60b140</A> (Accessed: 28 October 2008).
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:UC_Berkeley/GatewayOverview">http://2008.igem.org/Team:UC_Berkeley/GatewayOverview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Berkeley/GatewayOverview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Berkeley/GatewayOverview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayOverview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Berkeley/GatewayOverview&amp;oldid=105518" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>