"
Page
Discussion
View source
History
teams
Log in
 
Team:NTU-Singapore/Project/Approach
From 2009.igem.org
Welcome to the project page!
Hard at work!
Home
pLaqUe Out!
DIYbio
Accomplishments!
Notebook
About Us
Navigation
pLaqUe Out! Research Approach Sensing Device «   Degradation Device «   Imaging Device «   Wetlab Highlights Parts Submitted References
Our Approach
Contents
1 Research Methodology
1.1 Sensing device
1.2 Degradation device
1.3 Imaging device
1.4 Dilation device
2 System Overview
3 Modelling Approach
4 References/Literature
We wish to design a biological system to combat atherosclerotic plaque that can eventually be deployed in modified T-helper cells.
Due to time & equipment shortage during the iGEM period, we will only demonstrate the proof-of-concept system using the E.Coli K12 strain.
We will create our biological system with Biobrick devices, each with specific functions.
Research Methodology
In our project pLaqUe Out!, we will be dealing with 4 devices to arrive at a complete system.
The 4 devices are :
Sensing device with nitric oxide sensing promoter (pNO)
Degradation device for cholesteryl esterase (CHE) production
Imagine device for infrared-fluorescent protein (IFP) production
Dilation device for nitric oxide synthase (NOS) production
These devices are designed to work in sync to provide a robust systemic solution to plaque imaging & breakdown.
Sensing device
pNO is NorR-regulated sigma-54 promoter. We would be regulating the expression of a gene downstream of our pNO by varying the input of nitric oxide.
NorR is the transcription factor for pNO, and it is highly responsive to the presence of nitric oxide. So we have designed an NorR-producing construct to augment our sensing capability.
The NO-sensitive promoter region of the NorV/W gene sequence (pNO), endogeneous in E.Coli, is used.
This promoter will be the active NO-sensor for our system.
pNO is ON in high [NO] conditions & OFF in low [NO] conditions.
The pNO will be placed upstream of a TetR gene sequence.
TetR is used to repress the other devices.
In healthy [NO] conditions, pNO is activated and will facilitate expression of TetR.
In low [NO] conditions, pNO will be deactivated and the transcription of TetR gene sequence will stop. Hence repression of other devices will stop.
In order to realize this sub-system, we will first construct and test the NorR production construct. Our gene for pNO will be directly synthesized from GeneART.
We will characterize the pNO by the expression of downstream green-fluorescent protein (GFP). The protocols for this are available on the OpenWetWare (OWW) website and we will optimize them for our use. We will have sodium nitroprusside (SNP) as the source of nitric oxide, and we will vary the concentrations of SNP accordingly to obtain our desired input values.
Thus, we will be able to characterize the activity of our pNO via the GFP output in fluorescence time course (fluorescence versus time), absorbance time course (absorbance versus time) and fluorescence/OD600 versus time.
We also recognize the RPU of J23119 under different conditions will affect the expression rate of NorR, which is essential to the activation of pNO. So, we will be characterizing the activity of J23119 under different conditions, using the following test construct:
Please proceed here to see our detailed page on the Sensing device.
Degradation device
This sub-system will secrete cholesteryl esterase (CHE) which is able to catalyze the hydration reactions of cholesteryl esters into free cholesterol and fatty acids. The broken down cholesterol can be removed by natural means, like HDL-mediated efflux [7]. The degradation of cholesteryl esters through these reactions will lead to the size reduction of the atherolsclerotic plaque.
The pTet promoter will serve as the controller for this device.
In high [TetR] conditions, implying high [NO] conditions, Degradation device is OFF.
In low [TetR], implying low [NO] conditions, this device will be ON.
LuxI gene sequence expresses LuxI protein.
LuxI protein catalyzes conversion of SAM protein to AHL.
This conversion has significance in the Dilation device.
CHE gene sequence will express Cholesteryl Esterase enzyme.
This enzyme will breakdown the cholesteryl esters in plaque.
This gene for CHE, originally obtained from Pseudomonas aeruginosa, will be directly synthesized from GeneART.
Since the ultimate objective of this sub-system is to characterize the functionality and activity of CHE, we will induce the expression of CHE via constitutive promoters such as Part BBa_J23101 and Part BBa_J23119 from the Registry.
With this construct, we can readily perform functionality test with the Amplex(R) Red Cholesterol Assay Kit from Invitrogen to quantitatively analyse the enzyme activity of CHE. Nevertheless, SDS-PAGE will also be carried out to confirm the expression of the cholesteryl esterase of our construct.
For the ease of performing these characterization tests, which depend solely on the expression and efficiency of the CHE enzyme, we have designed a test construct as: pLac-RBS-CHE-Term.
This inducible construct will feature a reliable reproducible time-invariant enzyme production. In addition, the amount/time of induction itself can be varied to see efficiency of cholestryl ester breakdown in both quality and quantity.
Please proceed here to see our detailed page on the Degradation device.
Imaging device
This sub-system will control the expression of infrared-fluorescent protein (IFP) via a constitutive promoter. IFP however would not be stable in the absence of biliverdin IXα (BV). Hence, we will have our construct incorporated with heme-oxygenase 1 (HO-1) which is responsible in generating BV from heme group.
Again, pTet regulation will control the activation/repression of this device.
System is only ON at low [NO] and OFF at high [NO].
The Heme-Oxygenase 1 gene sequence (HO-1) and Infrared Protein gene sequence (IFP) are to be considered coupled.
HO-1 expression will facilitate the conversion of heme to biliverdin.
IFP, a protein discovered by Tsien lab, requires bilverdin to be stable and fluoresce in the infra-red spectrum.
In order to realize this sub-system, we will have our gene for IFP directly synthesized from GeneART, and subsequently ligated to HO-1 which is taken from the Registry (Part BBa_I15008).
We propose to have constitutive promoters such as Part BBa_J23101 and Part BBa_J23119 to induce the co-expression of HO-1 and IFP. The gene sequence for IFP is in fact from Tsien’s Lab, University of California, San Diego (UCSD). We hope to have their plasmid with HO-1 and IFP1.4 incorporated sent to us so that we can induce the expression of IFP1.4 and thus using their infrared fluorescence signal as a control to gauge the infrared fluorescence signal for our construct.
We are proposing to have an experimental setup to detect the infrared fluorescence signal. Since the maximal excitation wavelength for IFP is at 684 nm, we intend to use a 660 nm laser in our lab to shine onto the cell culture for excitation.
An optical fibre connected to a spectrometer will then detect the infrared signal (maximal emission wavelength) and we will be able to see the emission spectrum on the spectrograph. SDS-PAGE will also be carried out to confirm the expression of both proteins in our construct.
Again, for the determination & characterization of the infrared signal alone, we have designed a version of the system construct where the pTet promoters have been replaced with the strong & reliable J23119 promoters.
This construct, J23119-RBS-IFP1.4-Term-J23119-RBS-HO1-Term will allow us to study the infrared protein's activity without concern for the activation state of the promoter. We felt characterizing the novel working protein would be a higher priority than characterizing the transcription efficiencies.
Please proceed here to see our detailed page on the Imaging device.
Dilation device
Our team has designed this construct for the sake of completing the logical funtion of the overall system. However, due to the time limitation, this proposed construct will not be pursued.
The Dilation device is special because it features a delay mechanism to avoid premature feedback & shutdown of system.
The Dilation device features constitutive transcription of LuxR gene sequence.
[LuxR] will repress the activation of pLuxR further downstream.
The following is our E.Coli delay mechanism.
pLuxR is activated when LuxR is removed by selective binding to AHL.
This is where the LuxI from the Degradation device comes in.
LuxI converts SAM protein to AHL. AHL binds to LuxR.
We expect this series of binding comes with a time lag.
Recall that Degradation device is only ON at low [NO] conditions.
Hence pLuxR is only activated at low [NO] conditions, and only after a lag period.
Activated pLuxR allows for transcription of Nitric Oxide Synthase [NOS].
NOS will then regenerate [NO] at plaque site.
Regeneration of [NO] to normal levels can be expected to trigger the sense construct thus shutting down every other construct.
System Overview
The following is the overall genetic circuitry for our prototype system.
As we detailed the part placement in each device, this circuit is complete.
Modelling Approach
Our models will be mathematically deduced in the form of ordinary differential equations. These ODEs will relate the concentration of mRNA & protein output of the gene sequence of interest w.r.t time. We will use known and/or empirical constants such as maximum translation rate, maximum transcription rate, dissociation constants etc to assume for maximum efficiency case.
The constants used are :
Transcription rate of RNA polymerase : ~70 nt/sec [1,2]
Median half-life of mRNA degradation rate : 3.69 min [3]
Translation rate : 14.55 aa/s [4]
Protein degradation rate : 1 min [5]
No. of proteins/mRNA : 540 [6]
The modelling process is almost entirely theoretical, except for abovementioned constants which are literature-derived.
So firstly predicted outputs from the model can actually help us to envision the scale at which the system works. This is important to verify if the system can even ideally be used in the application that we want.
Furthermore, by identifying larger deviations in output of the actual system compared to predicted model, the model can also help as a “debugger”.
The models can also help us to enhance the system by identifying bottlenecks; identifying the assumptions made in simplifying calculations can indirectly point us to the potential sources of error in experimental verification.
We now model our system using blackboxes to form a mathematical logic circuit using these equations in Simulink.
The following is the Simulink circuit we have designed for our system. Please click on it for a larger view.
Laying out the system as a combination of Simulink sub-systems/devices allows us to visualise each step of the circuit as a blackbox, with only the input and output visible. This not only simplifies the study of the system infinitely, it also validates the flow of our system.
By identifying bottlenecks, we can account for any deviations in expected curves in actual construct characterization.
The following is the time-cumulative deterministic simulation results of our model for a step decrease in external [NO].
For a step decrease in external [NO] (blue) at 1000s, there will be a decreased production of TetR (green) in the Sensing device.
The fall in [TetR] activates the other three devices.
The Imaging device emits increasing intensity of infrared signal (red).
The Degradation device starts to produce CHE (yellow).
The Dilation device will only synthesize NOS (orange) after a time lag of 500s.
NOS will produce NO (pink).
The Overall [NO] profile (marine-blue) reflects the initial step decrease and the subsequent regeneration of [NO].
Our model works! Now we can start looking at each device one by one. From here on, our system descriptions will be a bit more technical. Bear with us!
Proceed here to see our comprehensive section on Sensing Device .
Click here to see details about the Degradation Device .
And finally, you can see our section on Imaging Device here.
References/Literature
Dennis, P. P., and H. Bremer. 1974. Differential rate of ribosomal protein synthesis in Escherichia coli B/r. J. Mol. Biol. 84:407–422.
Dennis, P. P., and M. Nomura. Stringent control of ribosomal protein gene expression in coli B/r. Proc Natl Acad Sci U S A. 1974 Oct71(10):3819-23
Bernstein JA, Lin PH, Cohen SN, Lin-Chao S. Global analysis of Escherichia coli RNA degradosome function using DNA microarrays. Proc Natl Acad Sci U S A. 2004 Mar 2101(9):2758-63
Dalbow and Young, Biochem J. 150:13-20 (1985)
El-Samad H, Kurata H, Doyle JC, Gross CA, Khammash M. Surviving heat shock: control strategies for robustness and performance. Proc Natl Acad Sci U S A. 2005 Feb 22102(8):pp. 2736
Lu P, Vogel C, Wang R, Yao X, Marcotte EM. Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation. Nat Biotechnol. 2007 Jan25(1):117-24.
ROSENBLAT Mira, V. J., SHIH Diana, AVIRAM Michael (2005). "Paraoxonase 1 (PON1) enhances HDL-mediated macrophage cholesterol efflux via the ABCA1 transporter in association with increased HDL binding to the cells : a possible role for lysophosphatidylcholine." Atherosclerosis
See the complete list of references here .
TOP || iGEM '09
NTU@iGEMcc 2009. Some rights reserved.
Retrieved from " http://2009.igem.org/Team:NTU-Singapore/Project/Approach "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
