<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTU-Singapore_Project_Prototype_Degrade"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NTU-Singapore/Project/Prototype/Degrade">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTU-Singapore/Project/Prototype/Degrade&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Degrade&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Degrade&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTU-Singapore/Project/Prototype/Degrade" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTU-Singapore/Project/Prototype/Degrade</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="container_16"><DIV id="plaquebanner" class="clearfix"><DIV id="ntuproject" class="grid_16"><P><SPAN class="hidden">Welcome to the project page!</SPAN></P></DIV></DIV><DIV id="menu" class="hp clearfix main"><DIV class="grid_4 link"><SPAN class="gray">Hard at work!</SPAN></DIV><DIV class="grid_1 link"><A href="/Team:NTU-Singapore" title="Team:NTU-Singapore"> Home</A></DIV><DIV class="grid_2 link selflink"><A href="/Team:NTU-Singapore/Project" title="Team:NTU-Singapore/Project"><SPAN class="selflink">pLaq<I>U</I><SPAN class="sup">e</SPAN> Out!</SPAN></A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/HPA" title="Team:NTU-Singapore/HPA"> DIYbio</A></DIV><DIV class="grid_3 link"><A href="/Team:NTU-Singapore/Notebook/Deliverables" title="Team:NTU-Singapore/Notebook/Deliverables"> Accomplishments!</A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/Notebook" title="Team:NTU-Singapore/Notebook"> Notebook</A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/About/Team" title="Team:NTU-Singapore/About/Team"> About Us</A></DIV></DIV><DIV id="ntucontent"><DIV class="wrap_3 suffix_1" id="submenu"><DIV class="subtitle contrast grid_3 textright"><P><SPAN>Navigation</SPAN></P></DIV><DIV class="grid_3 submenu textright"><P><A href="/Team:NTU-Singapore/Project" title="Team:NTU-Singapore/Project"><B>p</B>Laq<I>U</I><SPAN class="sup">e</SPAN><B>Out<I>!</I></B></A><A href="/Team:NTU-Singapore/Project/Approach" title="Team:NTU-Singapore/Project/Approach"> Research Approach</A><A href="/Team:NTU-Singapore/Project/Prototype/Sense" title="Team:NTU-Singapore/Project/Prototype/Sense"> Sensing Device «  </A><STRONG class="selflink"> Degradation Device «  </STRONG><A href="/Team:NTU-Singapore/Project/Prototype/Image" title="Team:NTU-Singapore/Project/Prototype/Image"> Imaging Device «  </A><A href="/Team:NTU-Singapore/Project/Wetlab" title="Team:NTU-Singapore/Project/Wetlab"> Wetlab Highlights</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=NTU-Singapore" class="external text" rel="nofollow">Parts Submitted</A><A href="/Team:NTU-Singapore/Project/References" title="Team:NTU-Singapore/Project/References"> References</A></P></DIV></DIV><DIV class="grid_11 suffix_1 omega"><P><SPAN class="title">Degradation <SPAN class="bold">Device</SPAN></SPAN></P><DIV id="tock"><DIV class="spiffyfg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Background"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Mechanism"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Mechanism</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#More_about_Cholesteryl_Esterase"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">More about Cholesteryl Esterase</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Characteristic_Equations"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Characteristic Equations</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Modelling_.26_Simulation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Modelling &amp; Simulation</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Device_Construction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Device Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Device_Characterization"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Device Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#pLacI_Characterization"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">pLacI Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#SDS_PAGE"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">SDS PAGE</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Cholesterol_Assay_Kit"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Cholesterol Assay Kit</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Literature_.2F_References"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Literature / References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV><P>
The <B>Degradation device</B> will focus exclusively on breakdown of cholesteryl esters found in the lipid core.
</P><P>
As we have elaborated earlier in the <A href="/Team:NTU-Singapore/Project/Prototype" title="Team:NTU-Singapore/Project/Prototype"> Prototype Design</A> section, this <B>device</B> consists of mainly a pTet-regulated sequence, with cholesteryl esterase (CHE) and LuxI as the major products.
</P><P>The device's function is simple: it generates the enzyme that breaks down plaques and expresses LuxI that simulates a delay in the proposed <B>Dilation device</B>.
</P><P>
Let us now see how exactly the device works.</P><H2><SPAN class="mw-headline" id="Background"> Background </SPAN></H2><P>The thickening of an artery wall in atherosclerosis is mainly due to the build-up of fatty materials. Accumulation of cholesteryl esters (CEs) in macrophage foam cells is critical to atherosclerotic plaque formation. CEs are dehydrolyzed between the carboxylate group of a fatty acid and the hydroxyl group of cholesterol. They are less soluble in water than cholesterols, and form insoluble aggregates.
</P><P>In order to alleviate the symptoms of shrinking blood vessel, and eventually cure atherosclerosis, CEs must be degraded and removed from the site of the plaque.
</P><P>
Previous works show that various cholesteryl ester hydrolases (CEHs) have been successfully cloned and expressed in both murine and human.
</P><P>
Further studies demonstrate that CEHs not only reduce high-fat, high-cholesterol diet-induced atherosclerotic lesion in mice significantly, but also eliminate CEs from macrophage foam cells in humans.</P><P>
So a prominent enzyme that can help degrade CEs should be incorporated in our system. Cholesteryl esterase (CHE), systematically known as steryl-ester acylhydrolase, is chosen to perform the crucial task because it is the prokaryotic homolog of CEH. 
</P><H3><SPAN class="mw-headline" id="Mechanism"> Mechanism </SPAN></H3><P>CHE catalyzes the hydration reactions of CEs into free cholesterol and fatty acids with high specificity.
</P><P><B>Cholesteryl Ester + H<SPAN class="sub">2</SPAN>O → cholesterol + RCOOH</B></P><P>
A device that could produce CHE is introduced to achieve the function of degrading and eliminating cholesteryl ester.
</P><P>The gene is designed to work under the control of promoter pTet. pTet is a well-known promoter that is constitutively ON and repressed by TetR. TetR repression is inhibited by the addition of tetracycline or its analog, aTc.
</P><P>Since TetR production is controlled by pNO, and it is suppressed at low NO concentration (indicating inflammatory condition at the atherosclerosis site), the degrading device can be activated when NO concentration is low.
</P><P>With the proper working of pNO and pTet inverter, CHE synthesis can accomplish designed objectives with both temporal and spatial specificity.
</P><H3><SPAN class="mw-headline" id="More_about_Cholesteryl_Esterase"> More about Cholesteryl Esterase </SPAN></H3><P>The 936bp CHE gene is obtained from Pseudomonas aeruginosa, who produces the extracellular enzyme naturally. CHE suits more with our E. coli model system for similar expression mechanisms employed between the two organisms. 
</P><H3><SPAN class="mw-headline" id="Characteristic_Equations"> Characteristic Equations </SPAN></H3><P><B>Transcription of CHE</B></P><P>We first assume that rate of mRNA production for both <B>CHE gene sequence</B> and <B>LuxI gene sequence</B> are equal.
</P><P>
Repressive transcription of CHE and LuxI are represented by the following equation :
</P><P>Where V<SPAN class="sub">tr</SPAN> is the maximum transcription rate in E.Coli, k<SPAN class="sub">d</SPAN> is the dissociation constant of TetR from pTet &amp; d<SPAN class="sub">m</SPAN> is the average degradation rate of mRNA in E.Coli.
</P><P><SPAN class="grey">The central term is a fraction that describes the ratio of active pTet promoters (not being repressed by TetR) over the total number of pTet promoters, repressed and otherwise.</SPAN></P><P>
As you can infer, we have again assumed that transcription and degradation rates are maximum to calculate best-case efficiency.
</P><P><B>Translation of CHE</B></P><P>Determining the [CHE] concentration is done by mathematically deducing the relationship between translation rate of mRNA and degradation rate of CHE.
</P><P>Where K<SPAN class="sub">tl</SPAN> is the translation rate of mRNA &amp; d<SPAN class="sub">p</SPAN> is the degradation rate of CHE protein.
</P><P><SPAN class="grey">If we review these equations so far, with the appropriate constants changed, the same equations will also describe LuxI transcription and translation in this device.</SPAN></P><P><B>Breakdown of CEs</B></P><P>Given that cholesteryl esters degrade to cholesterol &amp; fatty acids, first we attribute rise in cholesterol concentration to CHE activity.
</P><P>Where k<SPAN class="sub">3</SPAN> is the activity rate of CHE enzyme, which we then define as V<SPAN class="sub">max2</SPAN>.
</P><P>
We can quantify the amount of cholesteryl esters digested (and the amount of cholesterol formed) by assuming that the resulting fatty acids are in a fixed stoichiometric ratio to broken-down cholesterol.
</P><P>Where k is the stoichiometric ratio of fatty acids per cholesterol molecule. In our research, we found that k is usually 1.
</P><H3><SPAN class="mw-headline" id="Modelling_.26_Simulation"> Modelling &amp; Simulation </SPAN></H3><P>We make the following assumptions:
</P><OL><LI> Biological systems of transcription and translation are assumed to be linear and time-invariant.
</LI><LI> Concentration of the cholesteryl esters is assumed to be non-limiting.
</LI><LI> There is no time lag between CHE expression and enzymatic activity.
</LI><LI> Constant degradation rates for mRNA as well as protein.
</LI></OL><P>
By establishing equations to relate cholesteryl ester degradation to concentration of cholesterol and fatty acids, we can easily compare the modelling results to wet lab results using cholesterol assay kits.
</P><P>
The following is our <B>Degradation device</B> represented as a Simulink system. <I>Please click on it for a larger view</I>.
</P><P>
Using TinkerCell, a systems biology software, we can do deterministic &amp; stochastic modelling side by side. <I>Click on the images for a larger view.</I></P><P><B>Assuming as [NO] increases, [TetR] increases.</B></P><P>This is how increase in [NO] would affect CHE production. 
<I>Deterministic</I> simulation (left) &amp;<I>Stochastic</I> simulation (right). A sharp decrease in production of CHE in response to increase in [NO] is predicted, followed by the attainment of steady state after 10 mins.
</P><H2><SPAN class="mw-headline" id="Device_Construction"> Device Construction </SPAN></H2><P>CHE gene was synthesized by GeneArt in accordance with the standard of Registry of Standard Biological Parts. Sequence was optimized for expression in E. coli. Ligations were carried out to get the testing and final construct in our system.
</P><P><B>Parts that are related to the degrading device, and have been successfully constructed are</B> :
</P><UL><LI><B>CHE</B></LI><LI><B>RBS-CHE</B></LI><LI><B>CHE-T</B></LI><LI><B>RBS-CHE-T</B></LI><LI><B>RBS-CHE-RBS-GFP-T</B></LI><LI><B>pLux-RBS-CHE-T</B></LI><LI><B>pLac-RBS-CHE-T</B></LI></UL><P>
All the parts have been verified by running agarose gel electrophoresis of double digestion products as well as PCR products. Bands with correct sizes are observed.
</P><H3><SPAN class="mw-headline" id="Device_Characterization"> Device Characterization </SPAN></H3><P>The expression and functionality of CHE produced by this <B>Degradation device</B> is tested in two approaches. 
</P><OL><LI> SDS-PAGE is employed to examine if the size of the protein is correct
</LI><LI> Amplex® Red Cholesterol Assay Kit (Invitrogen) is used to test the emzyme activity.
</LI></OL><P>Test construct (<B>pLac-RBS-CHE-T</B>) is constructed for the above mentioned experiments.
</P><H4><SPAN class="mw-headline" id="pLacI_Characterization"> pLacI Characterization </SPAN></H4><P><B>Characterization of pLacI - GFP</B></P><P><SPAN class="grey">Our degradation device is under the control inducible promoter, pLacI. As shown above, pLacI responses to increasing concentrations of IPTG within 1-3mM ranges. Hence a predictable working promoter gave us more confidence in carrying on with the expression of CHE. </SPAN></P><H4><SPAN class="mw-headline" id="SDS_PAGE"> SDS PAGE </SPAN></H4><P>SDS-PAGE is a routine technique employed to test the presence of certain proteins.
</P><P>The expression of CHE in our <B>Degradation device</B> was confirmed by SDS-PAGE on a NuPAGE® Novex 4-12% Bis-Tris gel. Because no protein purification was carried out, a control sample that cannot produce CHE was run in parallel to verify its expression.
</P><P>Although the above method is not as good as carrying out purified proteins with SDS-PAGE, positive controls have been done on cells expressing GFP (26.8 kDa) and β-galacosidase (121 kDa), and results show that this technique is applicable. The theoretical molecular weight of CHE, based on its protein sequence, is found to be 32.73 kDa. (Tools: <A href="http://www.sciencegateway.org/tools/proteinmw.htm" class="external text" rel="nofollow">Protein Molecular Weight Calculator</A>). Protein ladder used is BenchMark™ Protein Ladder (Invitrogen Cat. No. 10747-012). 
</P><P><B>Fig 1</B> - <I>SDS-PAGE shows successful expression of positive controls of GFP (26.8 kDa) and β-galacosidase (121 kDa)</I></P><H4><SPAN class="mw-headline" id="Cholesterol_Assay_Kit"> Cholesterol Assay Kit </SPAN></H4><P>The Amplex® Red Cholesterol Assay Kit provides a simple fluorometric method for the sensitive quantitation of cholesterol using a fluorescence microplate reader or fluorometer. The assay is based on an enzyme-coupled reaction that detects free cholesterol concentration.
</P><P>Cholesterol is oxidized by cholesterol oxidase to yield H<SPAN class="sub">2</SPAN>O<SPAN class="sub">2</SPAN> and the corresponding ketone product. The H<SPAN class="sub">2</SPAN>O<SPAN class="sub">2</SPAN> is then detected using 10-acetyl-3,7-dihydroxyphenoxazine (Amplex Red reagent), a highly sensitive and stable probe for H<SPAN class="sub">2</SPAN>O<SPAN class="sub">2</SPAN>.
</P><P>In the presence of horseradish peroxidase (HRP), Amplex Red reagent reacts with H<SPAN class="sub">2</SPAN>O<SPAN class="sub">2</SPAN> with a 1:1 stoichiometry to produce highly fluorescentresorufin. Because resorufin has absorption and fluorescence emission maxima of approximately 571 nm and 585 nm, respectively, there is little interference from autofluorescence in most biological samples.
</P><P>
Cholesteryl linoleate is used as the substrate for enzyme activity characterization, due to its optimal fatty acid specificity to CHE. By adding the soluble portion (supernatant liquid) of cell lysate that contains CHE (confirmed by SDS-PAGE at first) to the substrate, functional CHE expression can be confirmed by higher fluorescence signals emitted, compared to samples that do not have CHE gene as a control.
</P><P>The theoretical enzyme activity, which needs the purification of CHE, is not carried out due to time limitations.
</P><P>The first challenge we met in using the kit was to dissolve CEs, specifically cholesteryl linoleate, which is the substrate for our enzyme. CEs are insoluble in water, thus we tried to dissolve it in some popular organic solvent such as methanol, acetone, and DMES. However, it appeared that all those did not help.
</P><P>Only isopropanol and acetonitrile were found to ‘partially’ dissolve cholestryl linoleate at a concentration of 7.34 μg/ml. <I>Please refer to the following figure</I>.
</P><P><B>Fig. 2</B> - <I>Determination on Solubility of Cholesteryl Linoleate in Isopropanol by Cholesterol Assay Kit</I></P><DIV class="grey">It is noted that fluorescence gradually decreases after 30 minutes, which may due to the evaporation of isopropanol in the 37°C testing environment. The differences of fluorescence are calculated as the average difference between CHE+ and CHE- samples in the first 30 minutes, which is found to be around 5276 fluorescent units. Acetonitrile dissolves cholesteryl linoleate at a similar concentration, and volatility of acetonitrile poses the same problem of unsteady response as isopropanol. </DIV><P>
The concentration of cholesteryl linoleate dissolved is calculated as the difference of fluorescent signals between samples using assay reagents with commercial CHE (CHE+) and those without CHE (CHE-), and then interpolated with standard reference.
</P><P><B>Fig. 3</B> - <I>Cholesterol Concentration Standard Curve</I></P><P><SPAN class="grey">Cholesterol Concentration Standard Curve calibrated to calculate cholesterol concentrations from fluorescent signals emitted from test samples. Note that the cholesterol concentration is the final concentration in micro plate reader, calculation of actual cholesterol concentration depends on the ratio of sample and reagent added.</SPAN></P><P>
Interestingly, the negative controls that only contain isopropanol and assay reagents (both CHE+ and CHE-) also gave significant fluorescent signals. As H<SPAN class="sub">2</SPAN>O<SPAN class="sub">2</SPAN> cannot be easily generated in-vitro in this experiment, we therefore deduce that the reaction catalyzed by cholesterol oxidase is not specific, meaning that cholesterol oxidase can help oxidize a large range of substrate, especially those with hydroxyl group like alcohol. 
</P><P>If the above hypothesis is correct, we would expect interference signals generated from our controls, which are samples that do not express CHE, because there should be a lot of alcohols present in the soluble portion after sonication (most alcohols are soluble unlike CEs, unluckily).
</P><P>Results show that both CHE+ and CHE- tests gave similar trend of signals versus time. And the steady state readings did not have significant difference. This confirms that cholesterol oxidase is a non-specific enzyme that oxidizes alcohols to ketones and H2O2, and there is not much CEs present in the soluble portion of cell lysate. (As expected, because CEs are very insoluble in water)
</P><P><B>Fig. 4</B> - <I>Cholesterol Assay with Cell Lysate</I></P><DIV class="grey"><P>Supernatant liquid of sonicated cells was tested with cholesterol assay kit. The same samples were treated with CHE+ and CHE- reagents. Values were averaged within three replicates. 
</P></DIV><P>
Although there is interference in our controls, the samples that contain CHE would also give the same signals as long as they are cultured and prepared identically. The interference signals from non-specific enzyme functionality can be generally assumed to be cancelled off.
</P><P>However, in order to give significant differences between samples and controls which can be precisely used to show the functionality of CHE, high concentrations of CEs should be used as the enzymatic substrate. After a round of literature review and technical consultation, chloroform came to our eyes. 
</P><P>Chloroform is a well-known organic solvent, but is also very toxic. Cholestryl linoleate can dissolve in chloroform up to 100 mg/ml. As chloroform is immiscible with water and all our assay conditions are aqueous environment, chloroform dissolved with cholesteryl linoleate should be redissolved in an organic compound that is miscible with water (like acetone, NO alcohol!), or it should be homogenized with 0.9% NaCl solution by vigorous stirring.
</P><P><B>Fig. 5</B> - <I>Determination on Solubility of Cholesteryl Linoleate in Chloroform by Cholesterol Assay Kit</I></P><DIV class="grey"><P>A significant difference was observed between CHE+ and CHE-. Controls (solvent only) gave steady near zero output, indicating no interference was generated. CHE- gave similar output as controls.
</P></DIV><P>With all the problems solved, we proceeded to test out whether our CHE construct works or not. Cells expressing CHE were cultured under the same conditions with control (cells with empty plasmids). Both cultures were harvested at the same OD, and lysed by sonication. The soluble portion of cell lysate was added to micro-plate reader, together with the substrate – cholesteryl linoleate.
</P><P><B>Results show that our CHE construct WORKS!</B></P><P><B>Fig. 6</B> - Cholesterol Assay with CHE Construct
</P><DIV class="grey"><P>Fig. 6 Differences are observed between cells expressing CHE and cells not expressing CHE. The insignificant differences may due to low expression level of CHE. Negative control denotes signals come from cells expressing CHE, but without the enzyme substrate. That is signals naturally generated by cell lysate as described in the former sections. Background is fluorescence emitted from samples without both enzyme substrate and cell lysate. 
</P></DIV><H2><SPAN class="mw-headline" id="Literature_.2F_References"> Literature / References </SPAN></H2><P>Please proceed <A href="/Team:NTU-Singapore/Project/References" title="Team:NTU-Singapore/Project/References"> here</A> to view our full list of references.
</P></DIV></DIV><DIV id="ntufooter"><DIV class="grid_3"><A href="#">TOP</A> || <A href="https://2009.igem.org/">iGEM '09</A></DIV><DIV class="grid_13 textright contrast"><P>NTU@iGEM<SPAN class="sup">cc</SPAN> 2009. Some rights reserved.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:NTU-Singapore/Project/Prototype/Degrade">http://2009.igem.org/Team:NTU-Singapore/Project/Prototype/Degrade</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTU-Singapore/Project/Prototype/Degrade" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTU-Singapore/Project/Prototype/Degrade" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Degrade&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Degrade&amp;oldid=168497" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>