<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTU-Singapore_Project_Prototype_Sense"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:NTU-Singapore/Project/Prototype/Sense">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTU-Singapore/Project/Prototype/Sense&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Sense&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Sense&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTU-Singapore/Project/Prototype/Sense" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTU-Singapore/Project/Prototype/Sense</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV class="container_16"><DIV id="plaquebanner" class="clearfix"><DIV id="ntuproject" class="grid_16"><P><SPAN class="hidden">Welcome to the project page!</SPAN></P></DIV></DIV><DIV id="menu" class="hp clearfix main"><DIV class="grid_4 link"><SPAN class="gray">Hard at work!</SPAN></DIV><DIV class="grid_1 link"><A href="/Team:NTU-Singapore" title="Team:NTU-Singapore"> Home</A></DIV><DIV class="grid_2 link selflink"><A href="/Team:NTU-Singapore/Project" title="Team:NTU-Singapore/Project"><SPAN class="selflink">pLaq<I>U</I><SPAN class="sup">e</SPAN> Out!</SPAN></A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/HPA" title="Team:NTU-Singapore/HPA"> DIYbio</A></DIV><DIV class="grid_3 link"><A href="/Team:NTU-Singapore/Notebook/Deliverables" title="Team:NTU-Singapore/Notebook/Deliverables"> Accomplishments!</A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/Notebook" title="Team:NTU-Singapore/Notebook"> Notebook</A></DIV><DIV class="grid_2 link"><A href="/Team:NTU-Singapore/About/Team" title="Team:NTU-Singapore/About/Team"> About Us</A></DIV></DIV><DIV id="ntucontent"><DIV class="wrap_3 suffix_1" id="submenu"><DIV class="subtitle contrast grid_3 textright"><P><SPAN>Navigation</SPAN></P></DIV><DIV class="grid_3 submenu textright"><P><A href="/Team:NTU-Singapore/Project" title="Team:NTU-Singapore/Project"><B>p</B>Laq<I>U</I><SPAN class="sup">e</SPAN><B>Out<I>!</I></B></A><A href="/Team:NTU-Singapore/Project/Approach" title="Team:NTU-Singapore/Project/Approach"> Research Approach</A><STRONG class="selflink"> Sensing Device «  </STRONG><A href="/Team:NTU-Singapore/Project/Prototype/Degrade" title="Team:NTU-Singapore/Project/Prototype/Degrade"> Degradation Device «  </A><A href="/Team:NTU-Singapore/Project/Prototype/Image" title="Team:NTU-Singapore/Project/Prototype/Image"> Imaging Device «  </A><A href="/Team:NTU-Singapore/Project/Wetlab" title="Team:NTU-Singapore/Project/Wetlab"> Wetlab Highlights</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=NTU-Singapore" class="external text" rel="nofollow">Parts Submitted</A><A href="/Team:NTU-Singapore/Project/References" title="Team:NTU-Singapore/Project/References"> References</A></P></DIV></DIV><DIV class="grid_11 suffix_1 omega"><P><SPAN class="title">Sensing <SPAN class="bold">Device</SPAN></SPAN></P><DIV id="tock"><DIV class="spiffyfg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mechanism"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Mechanism</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Sequence"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Sequence</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Construct_Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Construct Design</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Characteristic_Equations"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Characteristic Equations</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Modelling_.26_Simulation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Modelling &amp; Simulation</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Device_Construction"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Device Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Device_Characterization"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Device Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#pNO_characterization"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">pNO characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#NorR_characterization"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">NorR characterization</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#NorR_verification"><SPAN class="tocnumber">4.1.2.1</SPAN><SPAN class="toctext">NorR verification</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Literature_.2F_References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Literature / References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV><P>
The <B>Sensing device</B> will serve as the trigger for our system to start/stop its activity. As such, it is one of the most important constructs that we are working on.
</P><P>
As we have elaborated earlier in the <A href="/Team:NTU-Singapore/Project/Prototype" title="Team:NTU-Singapore/Project/Prototype"> Prototype Design</A> section, the <B>device</B> consists of mainly a a nitric oxide sensitive promoter that allows for transcription of TetR in high [NO], and is deactivated in low [NO].
</P><P>
Let us now see how exactly the device works.</P><H2><SPAN class="mw-headline" id="Abstract"> Abstract </SPAN></H2><P>Nitric-oxide (NO) sensing promoter (pNO) is a sigma-54 promoter that is regulated by transcription factor NorR, a member of the prokaryotic EBP (enhancer binding protein) family [1].
</P><P>NorR is a NO responsive transcription factor which is activated by interaction with NO. Upon activation, NorR will convert the inactive promoter bound sigma-54 RNA polymerase complex to a transcriptionally competent complex in pNO to promote the downstream transcription of the gene [2].
</P><P>Thus, expression of the gene downstream of pNO can be regulated by input of NO. 
</P><P>
In our project to tackle atherosclerosis, we seek to locate the presence of the plaque via the detection of NO concentration in the blood vessels. It has been shown that the cholesterol-induced atherosclerosis would usually impair the NO synthesizing ability of the vascular endothelial cells, and thereby leads to a decrease of NO concentration in the plaque region.
</P><P>By equipping our host cells (preferably T-helper cells; however in our experiment, we have used E.coli) with pNO, our system Plaque Out! will sense the decrease in NO concentration in the plaque region [3].
</P><P>
This will trigger an inverter module to:
</P><OL><LI> secrete cholesterol esterase (CHE) to digest the plaque
</LI><LI> produce infra-red signals to highlight the plaque’s location and
</LI><LI> release NO to dilate the vessel after digestion of plaque.
</LI></OL><P><B>Hence, pNO would serve as the main switch for our system Plaque Out!.</B></P><H2><SPAN class="mw-headline" id="Background"> Background </SPAN></H2><P>Based on the above described function, we have decided to include the entire NorR-NorV/W intergenic region (in E.coli genome) as our pNO promoter sequence. NorV/W genes encode for flavorubredoxin and an associated flavoprotein which would reduce NO to nitrous oxide [4].
</P><P>
These NorR and NorVW genes can be found in most proteobacteria, especially E.coli. And they are part of an important mechanism that allows bacteria to survive in anaerobic conditions as well as during nitrosative stress. In these cases, NorR protein serves as a NO-responsive transcriptional activator that regulates expression of the norVW genes.
</P><H3><SPAN class="mw-headline" id="Mechanism"> Mechanism </SPAN></H3><P>In the absence of NO, the N-terminal domain of transcription factor NorR typically represses the ATPase activity of NorR.
</P><P>It is only when NO binds to the ferrous iron center of NorR, it would stimulate ATP hydrolysis by NorR.  The ATPase activity of NorR is boosted by binding of NorR to the three enhancer binding sites located upstream in the pNO.
</P><P>The energy released by ATP hydrolysis enables NorR to activate pNO bound sigma-54 RNA polymerase complex (or RNA polymerase holoenzyme), thus initiating the transcription process [2].
</P><H3><SPAN class="mw-headline" id="Sequence"> Sequence </SPAN></H3><P>Using NCBI’s reference sequence NC_000913.2, the 186 bp NorR-NorVW intergenic region sequence is obtained using E.coli (strain: K-12; sub-strain: MG1655) as the search organism. The sequence is found in genome position 2830311 – 2830496. The NorR-NorVW intergenic sequence is as follows (5’ - 3’):
</P><DIV class="mono"><P>TCTTTGCCTCACTGTCAATTTGACTATAGATATTGTCATATCGACCATTTGATTGATAGTCATTTTGACT
ACTCATTAATGGGCATAATTTTATTTATAGAGTAAAAACAATCAGATAAAAAA<SPAN class="blu">CTGGCACGCAATCTGCA</SPAN>
ATTAGCAAGACATCTTTTTAGAACACGCTGAATAAATTGAGGTTGC</P></DIV><P>
The highlighted sequence is the sigma-54 promoter. The reason why we had included the entire intergenic region as our pNO promoter sequence is that there are 3 enhancer binding sites upstream of the sigma-54 promoter. These enhancer binding sites allow the transcription factor, NorR to bind to the DNA, thus leading to a localized increase in NorR concentration [2].
</P><P>As such, the increase in localized NorR concentration facilitates the NorR to make contact with the sigma 54 – RNA polymerase [5].
</P><H2><SPAN class="mw-headline" id="Construct_Design"> Construct Design </SPAN></H2><P>In our system, Plaque Out!, we wish to regulate the production of TetR protein with respect to the concentration of NO.
</P><P>The TetR protein would in turn regulate the production of Cholesterol esterase (CHE), Haem Oxyenase (HO-1), and Infra-red protein (IFP). Thus, our NO sensing construct is as follows:
</P><H3><SPAN class="mw-headline" id="Characteristic_Equations"> Characteristic Equations </SPAN></H3><P><B>Ligand binding</B> :
</P><P>Using Hill equation to express the fraction of NO-bound NorR, we assume independent and non-cooperative binding of NO (i.e. Hill coefficient is taken to be 1).
</P><P>Thus, fraction of NO-bound NorR is described as follows:
</P><P>
Since the dissociation constant K<SPAN class="sub">d1</SPAN> can be illustrated as:
</P><P>
Fraction of NO-bound NorR can be expressed as a function of available NO concentration, [NO]<SPAN class="sub">free</SPAN></P><P><B>Transcription of TetR</B></P><P>Deterministic first order differential equation (ODE) is used to depict the NO-regulated transcription of TetR.
</P><P>Where V<SPAN class="sub">max</SPAN> is max transcription rate &amp; D<SPAN class="sub">m</SPAN> is the degradation rate of mRNA.
</P><P><B>Translation of TetR</B></P><P>Deterministic first order ODE is used again to represent the translation of TetR protein.
</P><P>Where K<SPAN class="sub">tl</SPAN> is the translation rate of mRNA &amp; D<SPAN class="sub">p</SPAN> is the degradation rate of TetR protein.
</P><H3><SPAN class="mw-headline" id="Modelling_.26_Simulation"> Modelling &amp; Simulation </SPAN></H3><P>We make the following assumptions:
</P><OL><LI> Biological systems of transcription and translation are assumed to be linear and time-invariant
</LI><LI> Concentration of the transcription factor, NorR in E.coli is assumed to be in excess and constant. Hence, [NorR] is not taken into account in our model
</LI><LI> There is no time lag for NO diffusion from ext. environment into the cell
</LI><LI> Constant Degradation rates for mRNA as well as protein
</LI></OL><P>Instead of modelling with TetR, we are using GFP as our output. This is part of our effort in trying to predict the wetlab characterization of pNO.
</P><P>
The following is our <B>Sensing device</B> represented as a Simulink system. <I>Please click on it for a larger view</I>.
</P><P>
Using TinkerCell, a systems biology software, we can do deterministic (left) &amp; stochastic (right) modelling side by side. <I>Click on the images for a larger view.</I></P><P>
Response to changing [NO] with regards to TetR and NorR is simulated. These simulations predict the relationships among NO, NorR, and TetR.    <I>NO (blue), NorR(Red), TetR (Green)</I></P><H2><SPAN class="mw-headline" id="Device_Construction"> Device Construction </SPAN></H2><P><B>pNO-B0034-GFP-B0015 (pSB1A2)</B></P><P>To study the strength of pNO, we have decided to use pNO for the expression of green fluorescence protein (GFP). 
</P><P>pNO was first directly synthesized by GENEARTTM, one of the sponsors for iGEM. After digesting it with EcoRI and SpeI, it was ligated to B0034-GFP-B0015 as follows:
</P><P>
The test construct, pNO-B0034-GFP-B0015 was characterized using fluorescence detecting microplate reader, <B>Synergy HT</B> from BioTek, U.S.A.
</P><P><B>J23119-B0034-NorR-B0015 (pSB1A2) / (pSB3K2)</B></P><P>Though the transcription factor, NorR has been endogenously expressed in E.coli, our desired chassis is the macrophage. Thus, a NorR producing construct has to be created in order to equip the macrophage with the capability to detect NO.
</P><P>In addition, to reduce competitive binding of NorR between the endogenous NorR-NorVW intergenic region (in E.coli) and our pNO construct, large scale production of NorR is necessary.
</P><P>
NorR sequence was first extracted from E.coli chromosomal DNA (K-12, DH10B) via polymerase chain reaction (PCR). Next, the linearized dsDNA was converted to a biobrick part by adding prefix and suffix using PCR.
</P><P>
Lastly, the biobrick NorR was digested with relevant restriction enzymes (EcoRI / XbaI / SpeI / PstI) in order to be ligated to form the stated construct. 
</P><H3><SPAN class="mw-headline" id="Device_Characterization"> Device Characterization </SPAN></H3><H4><SPAN class="mw-headline" id="pNO_characterization"> pNO characterization </SPAN></H4><P><B>Characterisation of our novel nitric-oxide sensitive promoter (pNO) involved three steps</B> :
</P><OL><LI> Inoculation of the E. coli containing the test construct (GFP reporter) in the M9 supplemented medium
</LI><LI> Treatment of the E. coli cells with varying concentrations of a nitric-oxide donor (sodium nitroprusside or SNP)
</LI><LI> Fluorescence quantification using a fluorescence 96-well mircoplate reader. The full protocol was described by Kelly et al [6].
</LI></OL><P>Controls include untransformed cells and transformed cells that are not treated with SNP. 
</P><CENTER><B>GFP fluorescence in E. coli culture treated with varying [SNP].</B></CENTER><P>Cells containing the construct produced a more intense green colouring with increasing [SNP] from 0.1mM – 10mM. 1000mM and 100mM are red due to the native colour of SNP. Cytotoxicity effects might have limited the output beyond 10mM.
</P><CENTER><B>Response of pNO to varying [SNP]</B></CENTER><P>
Untransformed cells (blue diamond) showed the least fluorescence output.  E coli containing the test construct but are not treated with SNP  (orange circle) showed a slight increase in fluorescence with respect to the untransformed cells, suggesting response to ambient NO. Treatment of E coli test constructs with increasing concentrations of SNP showed increasing GFP fluorescence, indicating dose-dependent response.
</P><CENTER><B>Response of pNO to increasing [SNP]</B></CENTER><P>
Positive correlation of [SNP] and GFP fluorescence was observed. However, fluorescence output remained constant against time, suggesting that steady state has been reached before measurements were taken.   
</P><P>
Of note, the strength of our novel NO-inducible promoter was weaker than most established inducible promoters such as pLux and pLacI. SNP-induced relative GFP fluorescence from pNO as measured using our 96-well microplate reader was about 30 arbitrary units when induced with 10mM of SNP. On the other hand, the maximum IPTG-induced GFP fluorescence from pLacI was about a few hundred arbitrary units. Further work has to be done to optimise the applicability of the pNO promoter. However, primary characterisation studies have proven pNO to be a highly promising novel inducible promoter.  
</P><H4><SPAN class="mw-headline" id="NorR_characterization"> NorR characterization </SPAN></H4><H5><SPAN class="mw-headline" id="NorR_verification"> NorR verification </SPAN></H5><P><B>We have recently sequenced our NorR gene sequence</B> :
</P><P><U><B>Primer synthesis</B></U></P><P>Using NCBI’s reference sequence NC_000913.2 and E.coli (strain: K-12; substrain: MG1655) as search organism, the DNA sequence for NorR was obtained. The 1515 bp NorR resides at genome position 2828797 – 2830311, and the sequence is as follows:
</P><DIV class="mono"><P>ATGAGTTTTTCCGTTGATGTGCTGGCGAATATCGCCATCGAATTGCAGCGTGGGATTGGTCACCAGGATC
GTTTTCAGCGCCTGATCACCACGCTACGTCAGGTGCTGGAGTGCGATGCGTCTGCGTTGCTACGTTACGA
TTCGCGGCAGTTTATTCCGCTTGCCATCGACGGTCTGGCAAAGGATGTACTCGGTAGACGCTTTGCGCTG
GAAGGGCATCCACGGCTGGAAGCGATTGCCCGCGCCGGGGATGTGGTGCGCTTTCCCGCAGACAGCGAAT
TGCCCGATCCCTATGACGGTTTGATTCCTGGGCAGGAGAGTCTGAAGGTTCACGCCTGCGTTGGTCTGCC
ATTGTTTGCCGGGCAAAACCTGATCGGCGCACTGACGCTCGACGGGATGCAGCCCGATCAGTTCGATGTT
TTCAGCGACGAAGAGCTACGGCTGATTGCTGCGCTGGCGGCGGGAGCGTTAAGCAATGCGTTGCTGATTG
AACAACTGGAAAGCCAGAATATGCTGCCAGGCGATGCCACGCCGTTTGAAGCGGTGAAACAGACGCAGAT
GATTGGCTTGTCCCCTGGCATGACGCAACTGAAAAAAGAGATTGAGATTGTGGCGGCGTCCGATCTCAAC
GTCCTGATCAGCGGTGAGACTGGAACCGGTAAGGAGCTGGTGGCGAAAGCGATTCATGAAGCCTCGCCAC
GGGCGGTGAATCCGCTGGTCTATCTCAACTGTGCTGCACTGCCGGAAAGTGTGGCGGAAAGTGAGTTGTT
CGGGCATGTGAAAGGAGCGTTTACTGGCGCTATCAGTAATCGCAGCGGGAAGTTTGAAATGGCGGATAAC
GGCACGCTGTTTCTGGATGAGATCGGCGAGTTGTCGTTGGCATTGCAGGCCAAGCTGCTGAGGGTGTTGC
AGTATGGCGATATTCAGCGCGTTGGCGATGACCGTTGTTTGCGGGTCGATGTGCGCGTGCTGGCGGCGAC
TAACCGCGATTTACGCGAAGAGGTGCTGGCAGGGCGATTCCGCGCCGATTTGTTTCATCGCCTGAGCGTG
TTTCCACTTTCGGTGCCGCCGCTGCGTGAGCGGGGCGATGATGTCATTCTGCTGGCGGGGTATTTCTGCG
AGCAGTGTCGTTTGCGGCAGGGGCTCTCCCGCGTGGTATTAAGTGCCGGAGCGCGAAATTTACTGCAACA
CTACAGTTTTCCGGGAAACGTGCGCGAACTGGAACATGCTATTCATCGGGCGGTAGTTCTGGCGAGAGCC
ACCCGCAGCGGCGATGAAGTGATTCTTGAGGCGCAACATTTTGCTTTTCCTGAGGTGACGTTGCCGACGC
CAGAAGTGGCGGCGGTGCCCGTTGTTAAGCAAAACCTGCGTGAAGCGACAGAAGCGTTCCAGCGTGAAAC
TATTCGTCAGGCACTGGCACAAAATCATCACAACTGGGCTGCCTGCGCGCGGATGCTGGAAACCGACGTC
GCCAACCTGCATCGGCTGGCGAAACGTCTGGGATTGAAGGATTAA</P></DIV><P>
To ensure that the endogenous NorR sequence can be directly converted into a biobrick part without any modification, the sequence was checked with Webcutter 2.0 (<A href="http://rna.lundberg.gu.se/cutter2/" class="external free" rel="nofollow">http://rna.lundberg.gu.se/cutter2/</A>) for any presence of EcoRI / NotI / XbaI / SpeI / PstI restriction sites.  Since none of these restriction sites was found in the endogenouse NorR genome, direct PCR extraction can be performed. PCR primers were designed using NCBI’s PrimerBlast.
</P><P><B>The primer sequences for NorR Extraction</B> :
</P><P><I>Forward Primer (27 bases)</I>: 5’-TTAATCCTTCAATCCCAGACGTTTCGC-3’
</P><P><I>Reverse Primer (26 bases)</I>:5’- ATGAGTTTTTCCGTTGATGTGCTGGC-3’
</P><P>
Since NorR sequence is on the anti-sense strand of E.coli genome, the biobrick prefix has to be added to the reverse primer, rather than forward primer, and vice-versa.
</P><P><B>The primer sequences for Biobrick conversion</B> :
</P><P><I>Addition of Preffix to Reverse primer – Forward Primer (50 bases)</I> :
</P><P>5’-GTT TCT TCG AAT TCG CGG CCG CTT CTA GAG ATGAGTTTTTCCGTTGATGT-3’
</P><P><I>Addition of Suffix to Forward primer – Reverse Primer (50 bases)</I> :
</P><P>5’-GTT TCT TCC TGC AGC GGC CGC TAC TAG TA TTAATCCTTCAATCCCAGACG-3’
</P><P>
The oligonucleotides were synthesized by one of our team’s sponsor, 1st Base Holdings, Singapore.
</P><P><U><B>Ethanol Precipitation of E.coli Chromosomal DNA</B></U></P><P>Invitrogen’s TOP10 cells were first grown for 16 hours in 5 ml LB media at 37 deg C, 250 rpm before being centrifuge at 4000 rpm for 10 mins. The supernatant was decanted, and the cells were lysed using QIAGEN miniprep kit’s P1, P2 and N3 buffers. Since QIAquick spin columns can only immobilized DNA strands up to 10kb, hence, it was not used for the extraction of E.coli chromosomal DNA. Instead, ethanol precipitation was performed.
</P><P><B>Procedures of ethanol precipitation</B> :
</P><OL><LI> 1/10 vol of 3M Sodium Acetate (pH 5.2) was added to the supernatant containing E.coli DNA, P1, P2 and N3 buffers. 
</LI><LI> 3 vol. of ice-cold absolute ethanol (stored at -20 deg C) was then added to the mixture. 
</LI><LI> Incubate the sample on ice for 30 mins before spinning the sample at 13,000 rpm for 15 min. 
</LI><LI> Decant the supernatant, and add 1 ml of 70 % ethanol was added to wash any salts present. 
</LI><LI> Centrifuge for another 10 min at 13,000 rpm before decanting the supernatant. Air-dry the sample for 15 min.
</LI><LI> Add de-ionized water or Tris-HCl buffer (pH 7.5) to re-suspend the DNA precipitate.
</LI></OL><P><B>PCR extraction of NorR &amp; Biobrick conversion</B></P><P>PCR reaction was performed in the reaction vol of 50 µl containing PCR supermix (High Fidelity) from Invitrogen, 0.2 µM of each primer (Forward / Reverse), and 3 µg of E.coli chromosomal DNA. The cycling conditions were as followed: 30 sec at 94 °C followed by 30 cycles of 30 sec at 94°C, 30 sec at 58 °C, 2 min at 72 °C and a final extension of 10 min at 72 °C.
</P><P>
The PCR products were ran in 1% gel for 40 min at 120V before subjecting to gel extraction using QIAGEN gel extraction kit.  Addition of prefix and suffix was performed on the purified NorR DNA using the same PCR conditions mentioned above.
. 
</P><P><B>1% agarose gel photo verifying the success of the PCR extraction and biobrick conversion</B></P><P><B>Ligation to form J23119-B0034-NorR-B0015</B></P><P>Biobrick NorR was first digested with XbaI and PstI in order to be ligated to B0034. The ligation order was as follows: 
</P><OL><LI> B0034 + NorR
</LI><LI> B0034-NorR + B0015
</LI><LI> J23119 + B0034-NorR-B0015
</LI></OL><P>
The final J23119-B0034-NorR-B0015 is in pSB1A2 backbone. The displayed gel photos confirmed the presence of the successfully ligated construct of B0034-NorR-T (~1700bp) and J23119-B00340-NorR-B0015 (~1750 bp).
</P><P><U><B>SDS PAGE of J23119-B0034-NorR-B0015</B></U></P><P>The successful construct J23119-B0034-NorR-B0015 was ligated to pSB3K2 vector, and was later introduced into Invitrogen’s TOP10 cells. Transformed cells were grown for 16 hrs, at 37 deg C, 250 rpm, before undergoing cell lysis for SDS PAGE.
</P><P>The SDS PAGE results was reflected in the gel photo. Production of 55 kDa NorR protein had been confirmed by the result.
</P><P><B>55 kDa NorR band has been circled in red.</B></P><P><U><B>NorR Characterization via SDS PAGE</B></U></P><P>Since the sole purpose of J23119-B0034-NorR-B0015 (BBa_K256007) is to produce NorR, we studied physicochemical factors that might affect the production of the protein. The transformed cells were incubated at 3 different environmental temperatures (i.e. 30, 37 and 42°C).
</P><P><B>SDS PAGE results for different temperature</B></P><P>It has been observed that at elevated temperature, i.e. 42°C, production of NorR had decreased as shown by the diminished intensity (see lane '42'). The production of NorR at 30°C and 37°C were comparable.
</P><P><B>Conclusion</B></P><P>Our NorR producing construct had been verified to be successful and shown to be working well.
</P><H2><SPAN class="mw-headline" id="Literature_.2F_References"> Literature / References </SPAN></H2><DIV class="ref"><OL><LI> M.I. Hutchings, N. Mandhana, and S. Spiro, “The NorR protein of Escherichia coli activates expression of the Flavorbredoxin gene NorV in response to reactive nitrogen species,” Journal of bacteriology, vol. 184, (no. 16), pp. 4640 - 4643, 2002.
</LI><LI> N.P. Tucker, B.D. Autreaux, S. Spiro, and R. Dixon, “Mechanism of transcriptional regulation by Escherichia Coli nitric oxide sensor NorR,” Biochemical Society Transactions, vol. 34, (no. 1), pp. 191 - 194, 2006.
</LI><LI> T.J.M. Anderson, “Nitric Oxide, Atherosclerosis and clinical relevance of endothelial dysfunction,” Heart Failure Reviews, vol. 8, pp. 71 - 86, 2003.
</LI><LI> S.Spiro, “Nitric oxide-sensing mechanisms in Escherichia coli,” Biochemical Society Transactions, vol. 34, (no. 1), pp. 200 - 202, 2006.
</LI><LI> N.P. Tucker, B. D’Autre´aux, D.J. Studholme, S. Spiro, and R. Dixon, “DNA Binding Activity of the Escherichia coli Nitric Oxide Sensor NorR Suggests a Conserved Target Sequence in Diverse Proteobacteria,” Journal of Bacteriology, vol. 186, (no. 19), pp. 6656–6660, 2004.
</LI><LI> Kelly et al. (2009). &quot;Measuring the activity of Biobrick promoters using an in vivo reference standard.&quot; Journal of Biological Engineering 3(4).
</LI></OL></DIV><P>Please proceed <A href="/Team:NTU-Singapore/Project/References" title="Team:NTU-Singapore/Project/References"> here</A> to view our full list of references.
</P></DIV></DIV><DIV id="ntufooter"><DIV class="grid_3"><A href="#">TOP</A> || <A href="https://2009.igem.org/">iGEM '09</A></DIV><DIV class="grid_13 textright contrast"><P>NTU@iGEM<SPAN class="sup">cc</SPAN> 2009. Some rights reserved.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:NTU-Singapore/Project/Prototype/Sense">http://2009.igem.org/Team:NTU-Singapore/Project/Prototype/Sense</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTU-Singapore/Project/Prototype/Sense" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTU-Singapore/Project/Prototype/Sense" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Sense&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTU-Singapore/Project/Prototype/Sense&amp;oldid=169030" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>