<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEFU_China_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEFU China/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="s2_1box" style="background:none;color:white;"><P><STRONG><SPAN style="font-family:comic sans ms,cursive"><SPAN style="font-size:28px">Monthly</SPAN></SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">January</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">Our junior and senior students were attracted by the recruiting announcement for the iGEM 2015 in our university. After several interviews and competitive selections, our team was basically created.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">February</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">Our team was growing. We communicated with previous iGEMers in our school to get more insights into iGEM. Meanwhile, we started to watch successful iGEM cases of previous years.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">March</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">We wandered among wikis and gradually realized what the basic factors are for a good project. Brainstorms and weekly meet-ups promoted us to discover the quintessence of iGEM.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">April</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">We urged ourselves to get deeper insights into those previous cases to catch their inspiration. After meet-ups with teammates and our instructors, a draft idea of a yogurt guarder came to our mind. To make sure our project was worth proceeding, we did a survey to find out how much attention people would pay to the quality of yogurt. Fortunately, we got the expected results, although we knew we still had a long way to go.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">May</SPAN></STRONG></SPAN> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">We primarily designed the basic structure of our project. Then we read numerous papers to make an integrated protocol of our lab work.</SPAN></SPAN><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">Besides, we visited a dairy factory to learn the microbiological detection methods used in practical production of yogurt. We discovered that our method to be developed should have advantages over the traditional methods, especially in portability. </SPAN></SPAN><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">In addition, it was essential for us to adapt to the wet lab work. We started with laboratory safety training. With the instruction of our instructors and previous iGEMers, we learned a lot of techniques in molecular biology.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">June</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">With efforts, our wet lab work went smoothly.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">July</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">This month, we organized public activity to popularize the knowledge of yogurt production and its quality inspection, and remind people to store yogurt properly. </SPAN></SPAN><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">In addition, we did a survey to confirm the key factors which will attract customers’ attention if our yogurt guarder is developed into a real product. </SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">August</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">On 13th-15th, we attended the summit of Central China iGEM Consortium held by Peking University. We shared our experience with other teams and really learned a lot from other teams. Here we want to express our sincere thanks to them! 
Meanwhile, our lab works still proceeded. We encountered difficulty in the electrotransformation of bacteria, which delayed our project but was eventually resolved after multiple attempts.</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">September</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:14px">Time flies. We seized the time to make everything done and put great efforts to the wiki building and preparation for the Giant Jamboree.</SPAN></SPAN></P><P> </P><P><STRONG><SPAN style="font-family:comic sans ms,cursive"><SPAN style="font-size:28px">Labnote</SPAN></SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:20px">Isolation of genomic DNA from bacteria</SPAN></SPAN>         <SPAN style="font-family:arial,helvetica,sans-serif">The genomic DNA of <EM>Salmonella enterica</EM> subsp. <EM>enterica serovar Typhimurium</EM> str. LT2 and<EM> Lactobacillus delbrueckii</EM> subsp.<EM> Bulgaricus</EM> str. Lb14 were isolated using the TIANamp Bacteria DNA Kit (see protocol). The concentration of genomic DNA was measured with a NanoDrop spectrophotometer. </SPAN>     </P><P><SPAN style="font-size:20px"><SPAN style="font-family:arial,helvetica,sans-serif">Strain identification</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">We applied 16S ribosomal DNA identification to identify the strain of Salmonella and Lactobacillus bought from a biotech company. Universal primers were designed according to the conserved sequences.</SPAN></P><P> </P><P><SPAN style="font-size:20px"><SPAN style="font-family:arial,helvetica,sans-serif">A. Identification of Salmonella typhimurium</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P>                                 </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">         Fig 1. 1. Marker (2000bp DNA Ladder) ; 2.<SPAN style="font-size:16px"> </SPAN><SPAN style="font-size:14px"><EM>Salmonella typhimurium</EM></SPAN><SPAN style="font-size:16px"> </SPAN>16s rRNA</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">2. Ligation (TaKaRa DNA Ligation Kit see protocol)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">3. Transformation (<A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">4. Colony PCR</SPAN></P><P><SPAN style="color:rgb(0, 255, 255)"><SPAN style="font-size:28px">                   </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 2. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony; 6. Negative control</SPAN> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">5.Sequencing（Show results）</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">6.We then did alignment using BLAST.</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"> The sequence was consist with that of <EM>Salmonella enterica</EM> subsp. <EM>enterica serovar Typhimurium</EM> str. LT2.</SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:20px">B. Identification of Lactobacillus bulgaricus</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1. PCR</SPAN></P><P style="text-align:center">  </P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 3. 1. Marker (DL2000) ; 2. <SPAN style="font-size:14px"><EM>Lactobacillus bulgaricus </EM>16s rRNA</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">2. Ligation (TaKaRa DNA Ligation Kit see protocol)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">3. Transformation (<A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">4. Colony PCR</SPAN></P><P>                                  </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 4. 1. Marker (DL2000) ; 2.</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">3.4.5.6 </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">each for one single colony; 7</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. Negative control</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">5. Sequencing（Show results）</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">6. We then did alignment using BLAST.</SPAN></P><P style="margin-left:18pt"><SPAN style="font-family:arial,helvetica,sans-serif">The sequence was almost consist with that of <EM>Lactobacillus delbrueckii</EM> subsp.<EM> Bulgaricus</EM> str. Lb14. The differences were caused by subculturing perhaps.</SPAN></P><P style="margin-left:18pt"> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:20px">Gene Cloning of<EM> Lsr A, B, C, D, R </EM>and <EM>K</EM></SPAN></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:16px">1. </SPAN><SPAN style="font-size:18px">Gene isolation using PCR</SPAN></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">(Restriction enzyme sites are indicated in red font)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><EM>  </EM></SPAN><SPAN style="font-size:20px"><EM>   lsrA</EM></SPAN></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><EM> </EM></SPAN><SPAN style="font-size:20px"><EM>   lsrB</EM></SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><EM><SPAN style="font-size:18px">    </SPAN><SPAN style="font-size:16px">lsrC</SPAN></EM></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><EM><SPAN style="font-size:18px">   </SPAN><SPAN style="font-size:16px"> lsrD</SPAN></EM></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><EM>   </EM></SPAN><SPAN style="font-size:20px"><EM> lsrK</EM></SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><EM>  </EM></SPAN><SPAN style="font-size:20px"><EM>  lsrR</EM></SPAN></SPAN></P><P> </P><P>     </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 5. PCR products of lsrA, B and C  1.Marker (DL2000) ; 2.lsrA (1536bp) ; 3.Negative control; </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">4.lsrB (1023bp) ; 5.Negative control; 6.lsrC (1089bp) ; 7.Negative control   </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">                                                        </SPAN></P><P> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 6. PCR products of lsrD, R and K  1.Marker (DL2000) ; 2.lsrD (1002bp) ; 3.Negative control; </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">4.lsrR (960bp) ; 5.Negative control; 6.lsrK (1593bp) ; 7.Negative control</SPAN></P><P><SPAN style="font-size:18px"><SPAN style="font-family:arial,helvetica,sans-serif">2.Ligation (TaKaRa DNA Ligation Kit see protocol)</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">3.Transformation (<A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A>）</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">4.Colony PCR</SPAN></SPAN></P><P><EM><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">      lsrA</SPAN></SPAN></EM></P><P> </P><P>       </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 7. Clony PCR results. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony (1536bp) ; 6.Negative control</SPAN></P><P><EM><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">     lsrB</SPAN></SPAN></EM></P><P> </P><P> </P><P>     </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 8. Clony PCR results. 1. Marker (DL2000) ; 2.3.4 each for one single colony (1023) ; 5.Negative control</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><EM><SPAN style="font-size:18px">      lsrC</SPAN></EM></SPAN></P><P>           </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 9. Clony PCR results. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony (1089bp) ; 6.Negative control</SPAN></P><P><EM><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">      lsrD</SPAN></SPAN></EM></P><P>           </P><P> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 10. Clony PCR results. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony (1002bp) ; 6.Negative control</SPAN></P><P><EM><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">      lsrK</SPAN></SPAN></EM></P><P>           </P><P> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 11. Clony PCR results. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony(1593bp) ; 6.Negative control</SPAN></P><P><EM><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">      lsrR</SPAN></SPAN></EM></P><P> </P><P>       </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 12. Clony PCR results. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony; 6.Negative control</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">5.sequencing</SPAN></SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 13. Sequencing result of LsrA</SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 14. Sequencing result of LsrB</SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 15. Sequencing result of LsrC</SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 16. Sequencing result of LsrD</SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 17. Sequencing result of LsrK</SPAN></P><P>              </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 18. Sequencing result of LsrR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">All the sequencing results were consistent with our designation, except for LsrD. There was a wrong base pair in the sequences we cloned, so we chose SOE(Splicing by overlap extension)-PCR to fix this error.</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">A. Primary PCR reaction</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Segment1-lsrD1</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Segment2-lsrD2</SPAN></P><P style="margin-left:18pt"><SPAN style="font-family:arial,helvetica,sans-serif">                                                                      </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 19</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. 1. Marker (DL10000</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">) ;</SPAN><SPAN style="font-family:arial,helvetica,sans-serif"> </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">1. </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">LsrD1 ; </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">2. LsrD2</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">B. Overlapping and elongation</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">C. Second PCR reaction</SPAN></SPAN></P><P style="margin-left:18pt">                                     </P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 20</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. 1. Marker (DL10000</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">) ;</SPAN><SPAN style="font-family:arial,helvetica,sans-serif"> </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">1. </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">LsrD ; </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">2. LsrD1; </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">3. LsrD2</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation (see protocol)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Clone <STRONG>pcr </STRONG></SPAN></P><P>                                   </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 21. 1. Marker (DL2000) ; 2.3.4.5.each for one single colony(1002bp) ; 6.Negative control</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Sequencing</SPAN></P><P>         </P><P><SPAN style="font-family:arial,helvetica,sans-serif">As we see, the sequencing result is consistent with our designation.</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:28px">Plasmid construction</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"> All designed fragments needed in plasmid construction were replicated using PCR . </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">1. pNZ8148-lsrA</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrA </STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrA) with TIANprep Mini Plasmid Kit（see protocol）</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="margin-left:18pt">                                </P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">   Fig 22.1. Marker (DL2000) ; 2. Postive control (1536bp) ; 3.4. each for one single colony; 5. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="margin-left:18pt"><SPAN style="font-family:arial,helvetica,sans-serif">                                                                 <SPAN style="color:rgb(255, 0, 0)"> </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                  Fig 23. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,1536bp) ;</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">3. digest vector (3167bp) ; 4.pcr product (1536bp)</SPAN></P><P style="text-align:center"> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                               </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><STRONG>2. pNZ8148-lsrB</STRONG></SPAN></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrB) with TIANprep Mini Plasmid Kit（see protocol）</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrB </STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">  Fig 24. 1. Marker (DL2000) ; 2. Postive control (1023bp) ; 3.4.5.6.7. each for one single colony; 8. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 25. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,1023bp) ;</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">3. digest vector (3167bp) ; 4.pcr product (1023bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                                </P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px"><STRONG>3. pNZ8148-lsrC</STRONG></SPAN></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrC) with TIANprep Mini Plasmid Kit（ see protocol）</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrC </STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">  </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 26. 1. Marker (DL2000) ; 2. Postive control (1044bp) ; 3.4.5.6. each for one single colony; 7. Negative control</SPAN> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                </SPAN>   <SPAN style="color:#000000">  <SPAN style="font-family:arial,helvetica,sans-serif">Fig 27. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,1044bp) ; </SPAN></SPAN><SPAN style="color:#000000"><SPAN style="font-family:arial,helvetica,sans-serif">3. digest vector (3167bp) ; 4.pcr product (1044bp)</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                              <SPAN style="font-family:arial,helvetica,sans-serif">                                                      </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">4. pNZ8148-lsrD</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrD) with TIANprep Mini Plasmid Kit</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrD </STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P style="margin-left:18pt"><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 28. 1. Marker (DL2000) ; 2. Postive control (1000bp) ; 3.4.5.6. each for one single colony; 7. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">     <SPAN style="color:rgb(255, 0, 0)">   </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                      Fig 29. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,1002bp) ; 3. digest vector (3167bp) ; 4.pcr product (1002bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                                         </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">5. pNZ8148-lsrK</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrK) with TIANprep Mini Plasmid Kit</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrK</STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 30. 1. Marker (DL2000) ; 2. Postive control (1593bp) ; 3.4.5. each for one single colony; 6. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">        <SPAN style="color:rgb(255, 0, 0)">   </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">Fig 31. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,1593bp) ; 3. digest vector (3167bp) ; 4.pcr product (1593bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                                 </P><P>                                    </P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG><SPAN style="font-size:18px">6. pNZ8148-lsrR</SPAN></STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pEASY-T5 cloning vector with LsrR) with TIANprep Mini Plasmid Kit</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrR</STRONG></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"> </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 32. 1. Marker (DL2000) ; 2. Postive control (960bp) ; 3.4.5.6.7.each for one single colony; 8. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">      </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                     </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">Fig 33. 1. Marker (DL10000) ; 2. Double enmyze digestion (3167bp,960bp) ; 3. digest vector (3167bp) ; 4.pcr product (960bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                                   </P><P><SPAN style="font-size:18px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">7. pHY300PLK-plsr-blue-T</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">1. Molecular biology techniques: SOE (Splicing by overlap extension) PCR</SPAN></P><P><SPAN style="font-size:16px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">A. Primary PCR reaction</SPAN></STRONG></SPAN></P><P> </P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Segment1-plsr</SPAN></STRONG></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Segment2- Blue-T</SPAN></STRONG></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 34. 1. Marker (DL2000) ; 2. Blue-T (878bp) ; 3. plsr (250bp)</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">B. Overlapping and elongation</SPAN></STRONG></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">C. Second PCR reaction</SPAN></STRONG></P><P><SPAN style="color:rgb(255, 0, 0)"><SPAN style="font-family:arial,helvetica,sans-serif">                                                                       </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 35. 1. Marker (DL2000) ; 2. pLsr-Blue-T (1128bp) ; 3. pLsr (250bp) ; 4. Blue-T (878bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P style="margin-left:18pt; text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 36. 1. Marker (DL2000) ; 2.3. each for one single colony; 4. Negative control; 5</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. Postive control (1128bp)</SPAN></P><P style="margin-left:18pt"><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                  Fig 37. 1. Marker (DL10000) ; 2. Double enmyze digestion (4870bp,1128bp) ; 3. digest vector (4870bp) ; 4.pcr product (1128bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                                  </P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif"><SPAN style="font-size:18px">8.</SPAN> <SPAN style="font-size:18px">pBBR1MCS-5-lsrA</SPAN></SPAN></STRONG></P><P> </P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Insert lsrA </SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pNZ8148 with LsrA) with TIANprep Mini Plasmid Kit</SPAN></P><P><SPAN style="font-size:14px"><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">                                                          </SPAN></P><P> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 38. 1. Marker (DL2000) ; 2. Postive control (1952bp) ; 3.4.5.6.7. each for one single colony; 8. Negative control </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">                                                            </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">              </SPAN>   <SPAN style="font-family:arial,helvetica,sans-serif"> Fig 39. 1. Marker (DL10000) ; 2. Double enmyze digestion (4707bp,1952bp) ; 3. digest vector (4707bp) ; 4.pcr product (1952bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P> </P><P>                        </P><P><SPAN style="font-size:18px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">9. pBBR1MCS-5-lsrC</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pNZ8148 with LsrC) with TIANprep Mini Plasmid Kit</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Insert lsrC </SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P><SPAN style="font-size:28px"><SPAN style="color:rgb(0, 255, 255)">                  </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 40. 1. Marker (DL2000) ; 2. Postive control (1503bp) ; 3.4.5.6. each for one single colony; 7. Negative control</SPAN> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">                                                    </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                     Fig 41. 1. Marker (DL10000) ; 2. Double enmyze digestion (4707bp,1503bp) ; 3. digest vector (4707bp) ; 4.pcr product (1503bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing result is consistent with our designation.</SPAN></P><P>                            </P><P><SPAN style="font-size:18px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">10. pBBR1MCS-5-lsrD</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pNZ8148 with LsrD) with TIANprep Mini Plasmid Kit</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif"><STRONG>Insert lsrD</STRONG> </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">1.PCR</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Double digestion (Takara)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Ligation (TaKaRa DNA Ligation Kit Ver.2.1<A href="http://www.takara.com.cn/?action=Page&amp;Plat=pdetail&amp;newsid=101&amp;subclass=1">see manual</A>)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Transformation <A href="https://static.igem.org/mediawiki/2014/f/f5/NEFU_China_Transformation.pdf">see protocol</A></SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P>                                      </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 42. 1. Marker (DL2000) ; 2.3.4.5 each for one single colony (1439bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Two-step enzyme digestion(Takara) for detection</SPAN></P><P style="margin-left:18pt"><SPAN style="color:rgb(255, 0, 0)"> <SPAN style="font-size:36px">    </SPAN></SPAN><SPAN style="font-size:36px"><SPAN style="color:rgb(0, 255, 255)">           </SPAN></SPAN><SPAN style="font-size:36px"><SPAN style="color:rgb(0, 255, 255)"> </SPAN></SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">                    Fig 43. 1. Marker (DL10000) ; 2. </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">digest vector (4707bp);</SPAN><SPAN style="font-family:arial,helvetica,sans-serif"> </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">3. </SPAN><SPAN style="font-family:arial,helvetica,sans-serif">Double enmyze digestion (4707bp,1439</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">bp) ;</SPAN><SPAN style="font-family:arial,helvetica,sans-serif"> 4.pcr product (139</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">bp)</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">The sequencing results are consistent with our designation.</SPAN></P><P>                                </P><P><SPAN style="font-size:22px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Transformation in <EM>Lactobacillus</EM></SPAN></STRONG></SPAN></P><P> </P><P><STRONG><SPAN style="font-size:18px"><SPAN style="font-family:arial,helvetica,sans-serif">Electrotransformation of LsrBRK</SPAN></SPAN></STRONG></P><P> </P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Miniprep (pNZ8148–<EM>LsrB</EM>, pNZ8148–<EM>LsrR</EM>, pNZ8148–<EM>LsrK</EM>) with TIANprep Mini Plasmid Kit</SPAN></STRONG></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">1.Preparation of competent cells </SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">100 μl bacterial culture were inoculated into 50ml of MRS and incubated at 37°C. After overnight growth, a culture at the beginning of the stationary phase was harvested by centrifugation. The bacteria were washed three times with cold electroporation buffer (PB).The cells were then resuspended in PB to an OD600 of about 50.</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">2.Electrotransfection </SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">100  μl of the cell suspension was mixed with 10 μl plasmid DNA(pNZ8148–<EM>LsrB</EM>, pNZ8148–<EM>LsrR</EM>, pNZ8148–<EM>LsrK</EM>). The sample was subjected to a 2.4kV, 200Ω, 25μF electric pulse in a 0.2cm cuvette.</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">3.Recovery of transformants</SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">950 μl SMRS was immediately added, and the cells were incubated for 2 h at 37°C before they were plated on MRS supplemented with the appropriate antibiotic. The plates were incubated at 37°C for 2 to 3 days under anaerobic conditions.</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">4.Clone identify</SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">We repeated this transformation process for several times. Unfortunatelly, not a single colony was grown the plate.</SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Then we changed the parameters of electric pulse and tried several recovery medium, but we just got some unsatisfactory results. </SPAN></P><P style="text-align:center"> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 44. 1. Marker (DL2000) ; 2.3.4.5.6.7 each for one single LsrB colony; 8.9.10.11.12.13 each for one single LsrR colony;</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">1. Marker (DL2000) ; 2.3.4.5.6.7 each for one single LsrKcolony </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">      </SPAN></P><P style="text-align:center"> </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 45. 1. Marker (DL2000) ; 2.3.4.5.6.7 each for one single LsrB colony; 8.9.10.11.12.13 each for one single LsrR colony;</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">1. Marker (DL2000) ; 2.3.4.5.6.7 each for one single LsrKcolony </SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 46. 1. Marker (DL10000) ; 2.3.4.5.6. each for one single LsrB colony; 7.8.9. each for one single LsrR colony;</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">     1. Marker (DL10</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">000) ; 2.3. each for one single LsrR colony; 4.5.6.7.</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">8.</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">each for one single Ls</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">rk</SPAN><SPAN style="font-family:arial,helvetica,sans-serif"> colony</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">; </SPAN></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Finally, we realized it was the concentration of the plasmids which led to the low efficiency in transformation. We then did plasmid Maxiprep (see protocol) and concentration, and obtained plasmids in a concentraion of about 3000ng/μL. We also added 100μL denatured salmon sperm DNA to the cuvette before electroporation. This time, we eventually gained positive colonies.</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Clone identify</SPAN></STRONG></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 47. 1. Marker (DL4000) ; 2.3.4.5.6. each for one single LsrB colony; 7. Postive control (1023bp) ; 8.9.10.11.12. each for one single colony ; 13</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. Postive control (1593bp)</SPAN></P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">      1. Marker (DL4000) ; 2.3.4.5.6. each for one single LsrR colony; 7</SPAN><SPAN style="font-family:arial,helvetica,sans-serif">. Postive control (960bp);</SPAN></P><P style="text-align:center"> </P><P><SPAN style="font-size:20px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Electrotransformation of <EM>plsr+blue </EM>pigment+T</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif">Maxiprep (see protocol) and concentration of pHY300PLK-pLsr- amilCP。</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Clone identify</SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><P>                                      </P><P style="text-align:center"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 48.1. Marker (DL2000) ; 2. Postive control (1128bp) ; 3.4. each for one single colony; 5. Negative control</SPAN></P><P style="text-align:center"> </P><P><SPAN style="font-size:20px"><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Electrotransformation of pNZ9530 plasmid</SPAN></STRONG></SPAN></P><P> </P><P><SPAN style="font-family:arial,helvetica,sans-serif"> The transformation procedures are similar to Electrotransformation of BRK</SPAN></P><P><STRONG><SPAN style="font-family:arial,helvetica,sans-serif">Clone identify</SPAN></STRONG></P><P><SPAN style="font-family:arial,helvetica,sans-serif">Colony PCR</SPAN></P><DIV>                        </DIV><DIV style="text-align: center;"><SPAN style="font-family:arial,helvetica,sans-serif">Fig 49: 1. Marker (DL2000) ; 2.3.4.5.6. each for one single colony (486bp) ; 7. Postive control</SPAN> </DIV><P class="footer" style="color:white;margin-top:20px;text-align:center;"><SPAN>Generated by </SPAN><A href="https://2015.igem.org/Team:NEFU_China/Software">Flight iGEM</A>.</P></DIV><A id="returnTop" href="javascript:;">Back to Top</A></DIV></DIV></DIV></DIV></BODY></HTML>