<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><UL><LI><A href="/Team:Vilnius-Lithuania/Description">Project</A></LI><LI><A href="/Team:Vilnius-Lithuania/Description">Description</A></LI><LI><A href="/Team:Vilnius-Lithuania/Design">Design and Results</A></LI><LI><A href="/Team:Vilnius-Lithuania/Model">Modelling</A></LI><LI><A href="/Team:Vilnius-Lithuania/Demonstrate">Proof of concept</A></LI><LI><A href="/Team:Vilnius-Lithuania/InterLab">Interlab</A></LI><LI><A href="/Team:Vilnius-Lithuania/Safety">Safety</A></LI></UL><LI><A href="/Team:Vilnius-Lithuania/Parts">Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Parts">Overview</A></LI><LI><A href="/Team:Vilnius-Lithuania/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Composite_Part">Composite Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Vilnius-Lithuania/Notebook">Notebook</A></LI><LI><A href="/Team:Vilnius-Lithuania/Notebook">Lab Book</A></LI><LI><A href="/Team:Vilnius-Lithuania/Protocols">Protocols</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Silver">Human practices</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Silver">Silver</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Vilnius-Lithuania/Engagement">Education &amp; Public Engagement</A></LI><LI><A href="/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></LI><LI><A href="/Team:Vilnius-Lithuania/Members">Team</A></LI><LI><A href="/Team:Vilnius-Lithuania/Members">Members</A></LI><LI><A href="/Team:Vilnius-Lithuania/Attributions">Attributions</A></LI><LI><A href="/Team:Vilnius-Lithuania/Sponsors">Sponsors</A></LI></DIV><DIV class="helper"><P>use keyboard, swipe or scroll</P></DIV><DIV class="wrapper slider-page"><DIV class="container"><DIV class="slider-box"><DIV class="right-content"><DIV class="slide-right-content slide-right-content-1 is-current"><H1>Determining the plasmid copy number</H1><DIV class="category">Design and Results</DIV><P>Preparing for the framework: standard curve generation and plasmid copy number evaluation.</P><DIV class="readmore" data-modal="1">read more</DIV></DIV><DIV class="slide-right-content slide-right-content-2 hidden"><H1>Plasmid copy number control</H1><DIV class="category">Design and Results</DIV><P>Flexible copy number control is the core of our framework, which  is based on re-engineered ColE1 origin of replicon.</P><DIV class="readmore" data-modal="2">read more</DIV></DIV><DIV class="slide-right-content slide-right-content-3 hidden"><H1>Multiple plasmid groups</H1><DIV class="category">Design and Results</DIV><P>Multi-plasmid framework would not be much without multiple plasmids. We have equiped our synthetic origin of replication with specific recognition sequences to create unique plasmid groups.</P><DIV class="readmore" data-modal="3">read more</DIV></DIV><DIV class="slide-right-content slide-right-content-4 hidden"><H1>Global copy number regulation</H1><DIV class="category">Design and Results</DIV><P>We created a universal copy number control mechanism for all plasmid groups at once. For this we have applied the effects of the Rop protein, which increases the binding affinity of RNA I and RNA II, hence - inhibition of replication</P><DIV class="readmore" data-modal="4">read more</DIV></DIV><DIV class="slide-right-content slide-right-content-5 hidden"><H1>SynORI selection system</H1><DIV class="category">Design and Results</DIV><P>Having multiple plasmids in a cell means using multiple antibiotics. Or does it?</P><DIV class="readmore" data-modal="5">read more</DIV></DIV><DIV class="slide-right-content slide-right-content-6 hidden"><H1>Active partitioning system</H1><DIV class="category">Design and Results</DIV><P>If at least one of the plasmid group has a low copy number, they require extra care to not be lost at cell division. Therefore, SynORI framework incorporates a <I>special</I> partitioning system derived from pSC101 replicon. </P><DIV class="readmore" data-modal="6">read more</DIV></DIV></DIV></DIV><UL class="navigation"><LI><A href="/Team:Vilnius-Lithuania/Description">Description</A></LI><LI><A href="/Team:Vilnius-Lithuania/Design" class="active">Design and Results</A></LI><LI class="is-current"><DIV class="sli-item sli-item-1">Determining the plasmid copy number</DIV></LI><LI><DIV class="sli-item sli-item-2">Plasmid copy number control</DIV></LI><LI><DIV class="sli-item sli-item-3">Multiple plasmid groups</DIV></LI><LI><DIV class="sli-item sli-item-4">Global copy number regulation</DIV></LI><LI><DIV class="sli-item sli-item-5">SynORI selection system</DIV></LI><LI><DIV class="sli-item sli-item-6">Active partitioning system</DIV></LI></UL><LI><A href="/Team:Vilnius-Lithuania/Model">Modelling</A></LI><LI><A href="/Team:Vilnius-Lithuania/Demonstrate">Proof of concept</A></LI><LI><A href="/Team:Vilnius-Lithuania/InterLab">Interlab</A></LI><LI><A href="/Team:Vilnius-Lithuania/Safety">Safety</A></LI></DIV></DIV><DIV class="modal modal-1"><DIV class="modal-content"><H1>Determing the plasmid copy number</H1><H5>The foundation of multi-plasmid framework is the ability to determine plasmid copy number (PCN) per cell. Our approach to count the PCN in the cell is built upon absolute quantitative PCR.</H5><P>By using two different standard curves we are able to evaluate bacteria and plasmid copy number in the reaction. The PCN per cell is found by dividing the total PCN by the cell number. As described by Plotka M. et al, the separate detection of the plasmid and the host cell chromosomal DNA was achieved using two separate primer sets, specific for the plasmid Ori sequence and for the chromosomal d-1-deoxyxylulose 5-phosphate synthase gene (dxs), respectively. Ratio of these genes copy number gives PCN per cell.</P><H2>Why we chose a specific set of primers for PCN determination? </H2><P>Commonly, gene used for PCN evaluations in qPCR is an ampicillin resistance gene (bla). In our case, the bla gene for plasmid number determination was not used, as the SynORI multi-plasmid framework employs a number of plasmids with different selection system gene circuit parts. Instead, the origin of replication was barcoded with distinct sets of primers (named qPCR) for different groups of origin copy number determination. This enables to determine the desired plasmid group copy number, when working with multi-plasmid systems.</P><H2>Using cell lysates instead of extracted DNA</H2><P>Since working with extracted genomic, plasmid or total DNA incorporates an error which depends on the extraction efficiency, we decided to work with specially prepared cell lysate suspensions, skipping the extraction step. Furthermore, dxs gene from chromosome was ligated into pUC19 plasmid for less complicated standard preparation.</P><H2>Results</H2><P>First of all, it was concluded that primers, designed for calculations, are appropriate for absolute quantitative PCR by measuring the efficiency of PCR amplification reaction. Both genes have almost ideal amplification efficiency and required no further optimization.</P><DIV class="img-cont"><DIV class="img-label">Figure 1. PCR amplification efficiency of genes used for plasmid copy number evaluations.</DIV></DIV><H5>Amplified qPCR product were verified using agarose gel electrophoresis.</H5><DIV class="team-member"><DIV class="img-cont"><DIV class="img-label">Figure 2. Verification of products amplified during qPCR.</DIV></DIV><DIV class="img-cont"><DIV class="img-label">Figure 3. Melting curves of amplified qPCR products used for PCN determination.</DIV></DIV></DIV><P>Next, standard curves were generated for dxs and qPCR plasmid gene copy number determination according to the protocol written by our team (click here?).</P><DIV class="img-cont"><DIV class="img-label">Figure 4. Standard curves generates for plasmid copy number determination for SynORI project. </DIV></DIV><P>The fit of the linear regression model was satisfactory; the coefficient of determination (R2) was more than 0.999 for all standard curves. As amplification efficiencies and generated standard curves were ideal for both genes, the curves and earlier mentioned protocol were used for all further PCN determination experiments.</P><H5>Reference:</H5><P>Plotka M, Wozniak M, Kaczorowski T. Quantification of Plasmid Copy Number with Single Colour Droplet Digital PCR. Doi H, ed. PLoS ONE. 2017;12(1):e0169846. doi:10.1371/journal.pone.0169846. </P><P>Anindyajati, Artarini AA, Riani C, Retnoningrum DS. Plasmid Copy Number Determination by Quantitative Polymerase Chain Reaction. Scientia Pharmaceutica. 2016;84(1):89-101. doi:10.3797/scipharm.ISP.2015.02.</P></DIV></DIV><DIV class="modal modal-2"><DIV class="modal-content"><H1>Plasmid copy number control</H1><H5>Flexible copy number control is the core of our framework, which is based on re-engineered ColE1 origin of replication.</H5><H2>Base of SynORI framework - ColE1 replicon</H2><P>ColE1 plasmid replicon is based on two antisense RNA molecules: RNA I and RNA II. </P><P>The transcript of RNA II forms a RNA-DNA duplex and acts as a primer for DNA polymerase and for that reason is often called a replication initiator.</P><P>During the transcription of RNA II, several different secondary structures can form. Part of these structures are susceptible to the binding of RNA I - a shorter antisense version of RNA II. The interaction between RNA I and RNA II begins upon the formation of kissing-loop pairs between their anti-complementary secondary structures. If the kissing complex persists, a 3’ end of RNA I starts forming a zipper-like duplex with a complementary region of a single strand RNA II. Because the primer cannot be formed anymore, this results in replication inhibition, which is why RNA I is often called the replication inhibitor.</P><P>The main reasons why we have chosen ColE1 as base for SynORI framework were:</P><UL><LI>Its proficient biochemical and mathematical characterization;</LI><LI>Its simplicity for consisting of only two regulatory RNA molecules;</LI><LI>The favor of kissing-loop complex formation kinetics to predict plasmid group compatibility.</LI></UL><H2>Picking the control type</H2><P>It immediately becomes clear that in order to control the copy number of a plasmid, one could simply change the RNA I promoter. However, there is a reason why it has never been done before!</P><P>As mentioned above, RNA I and RNA II are two antisense molecules, so it is impossible to change one without altering the sequence of another molecule. RNA I promoter is located right on top of the RNA II secondary structures, which are not used for inhibition, but are crucial to form the RNA-DNA duplex for replication initiation. 
</P><P>Even if one managed to somehow substitute the RNA I promoter with another without disabling replication, it would still be inconvenient because this method would require a large pool of resources every time.</P><P>For that reason we have decided not to modify or replace the RNA I promoter inside the initial wild type origin of replication, but rather to disable it completely and place a copy of it next to RNA II.</P><H2>Disabling the RNA I promoter</H2><P>As already indicated, the main problem of inactivating the RNA I promoter is the necessity to take precautions in order keep the critically important secondary structures of RNA II intact.  </P><P>We have first acquired a priority mutation list from the literature. The list analysed RNA polymerase binding affinity to -10 and -35 promoter region and its dependence on point mutations, with mutations causing the largest decrease in affinity being displayed on the top of the list.</P><DIV class="img-cont"><DIV class="img-label">Table 1. Priority mutation list of -10 and -35 promoter regions.
            </DIV></DIV><P>We compiled a simulative algorithm which compared every possible combination of -10, -35 mutations and then matched them to predicted RNA II secondary structures made by CoFold, a thermodynamics-based RNA secondary structure folding algorithm that took co-transcriptional folding into account. Afterwards we set a replicon mutants prioritizing:</P><UL><LI>Mutants that had unchanged RNA II secondary structures.</LI><LI>Mutants that were on the top of mutation priority list (meaning the lowest RNA polymerase affinity).</LI></UL><P>RNA I mutation graph</P><P>Sequences of the  5 best mutants after running simulation:</P><P>We had ordered the selected ColE1 mutants from the IDT and later tested if we had successfully disabled the RNA I promoter. </P><P>It is difficult to distinguish when the promoter is fully disabled because first, there is no literature data describing replicons that are not negatively regulated at least to some extent, and second -  plasmid systems hardly reach the equilibrium without negative control therefore every copy number calculation varies greatly. This is why we decided not to check for the highest copy number mutant, but rather to insert a wild type RNA I with its wild type promoter. By doing that we could see which replicons were most precisely mutated.</P><P>
If the plasmid copy number (PCN) did not differ from wild type after the insertion of an RNA I gene next to the mutated replicon, it proved a complete disabling of the replicon. Contrary, if the copy number decreased, we could suspect that the replicon did not have a completely disabled RNA I and the sum of inhibition from both RNA I genes reduced the copy number to even lower values than in the wild type replicon.</P><P>
First, we planned to calculate the copy number of our mutants that supposedly had their RNA I gene promoter disabled (mutants ORI 1, ORI 2, ORI 3, ORI 4, ORI 5). After that, we aimed to calculate the copy number of the corresponding mutated replicons, but with RNA I gene containing its wild type promoter cloned next to them. </P><P>
After transformation, cells with ORI 5 plasmids did not grow successfully, which suggested a conclusion that this mutant had either severely damaged RNA II gene or increased expression of RNA I to the level of complete replication inhibition.</P><P>
Since 4 other mutants had grown after the transformation, we incubated the cells overnight, purified the plasmids and cloned wild type RNA I with its wild type promoter next to each of the mutants. We then calculated the copy number of 8 samples: 4 ORI mutants and 4 ORI mutants with RNA I placed next to them.</P><DIV class="img-cont"><DIV class="img-label">Figure 1. Copy number calculations of the RNA I promoter elimination mutants. Two biological replicates were performed, with 2 technical qPCR replicates each time.
            </DIV></DIV><P>Firstly, ORI 1 mutant had a moderate increase in copy number (Figure 1). Yet, with RNA I next to the replicon, the copy number did not seem fall back to wild type levels. We hypothesize that the reason for this was the damage done to the RNA II gene. The damage resulted in mutant formed secondary structures no longer sufficiently interacting with inhibitory RNA I molecules. </P><P>
ORI 3 did not seem to increase much in copy number. We did not consider it to be a good candidate as well, because we wanted our core synthetic ori to possess a range of copy numbers to choose from. </P><P>
The third candidate, ORI 4, seemed to be a decent candidate because with cloned RNA I its copy number fell to near wild type levels, but it also did not prove to be good enough, because its maximum number of copies was too low.</P><P>ORI 2 mutant seemed like a perfect candidate. Its copy number increased from wild type X levels to Y +- Z. In addition, when RNA I gene was placed next to it, the copy number of the constructed plasmid fell to wild type levels. After these results we have decided to use this ORI 2 mutant as a core for our framework. We simply called it RNA II (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2259000">Part:BBa_K2259000</A> ).
</P><H2>Tailoring the copy number control</H2><P>Once the RNA I promoter was disabled in the ColE1 origin of replication, it could be moved to a different plasmid location and used as a separate unit. Also, RNA I promoter can be changed without having to worry about damaging the initiation of replication. RNA I and consequently the copy number of a certain plasmid can now be placed under virtually any signal pattern required.
 </P><P>We have discovered the sequence of wild type RNA I promoter by using PromoterHunter. and ordered a wild type RNA I gene from IDT without the promoter’s sequence. We have first cloned series of anderson promoters next to the RNA I gene and then placed this construct next to RNA II (RNA II-Anderson-RNA I).</P><DIV class="img-cont"><DIV class="img-label">Figure 2. RNA I and RNA II constructs, with RNA I constructs under different-strength Anderson (See anderson collection here http://parts.igem.org/Promoters/Catalog/Anderson ) promoters.
            </DIV></DIV><P>In theory (see &quot;Modelling&quot; for more details), lower-strength Anderson promoters should yield lower concentrations of RNA I, hence higher copy numbers of plasmids per cell.  Our constitutive copy number device experiment results prove it to be true in practice as well. The stronger Anderson promoter is used, the less copy number per cell we get. With the strongest Anderson we get only 21+-6.84 plasmids per cell. </P><P>Worth to mention is that the closest to wild type ColE1 replicon is the 0.86 strength Anderson promoter, measured by copy number alone. (<A href="http://parts.igem.org/Part:BBa_J23102">Part:BBa_J23102</A>)</P><P>We can state with certainty that we are now able to control the plasmid copy number in a constitutive manner, and we call it simply the SynORI constitutive copy number device.</P><P>Next, we wanted to move one step forward and try to build an inducible copy number system. We first had to make sure that at least part of our construct is well characterized and to so we chose the rhamnose promoter from the biobrick registry (<A href="http://parts.igem.org/Part:BBa_K914003">Part:BBa_K914003</A>).</P><P>For this experiment we have built a rhamnose and RNA I construct (<A href="http://parts.igem.org/Part:BBa_K2259065">Part:BBa_K2259065</A>) and then cloned this construct next to RNA II (<A href="http://parts.igem.org/Part:BBa_K2259091">Part:BBa_K2259091</A>). We have used different percent of rhamnose in our media in order to see if this approach was possible and if so, to figure out the dependency between the plasmid copy number and rhamnose concentration.</P><DIV class="img-cont"><DIV class="img-label">Figure 3. RNA I and RNA II constructs, with RNA I gene being under the rhamnose promoter. 
            </DIV></DIV><P>The first thing we noticed was that rhamnose promoter was very strong in terms of plasmid copy number reduction. It was also considerably leaky (promoter can be enabled even without any inducer). At zero induction there were approximately only 9 plasmids per cell and at 1 percent induction the number rose to approximately 1 plasmid per cell. </P><P>
RNA I rhamnose-induced promoter seemed to be working well, with higher concentrations of inductor giving lower plasmid copy number.</P><P>
We called it the SynORI copy number induction device.</P><P>

So now when we can flexibly control the copy number of a plasmid, the only question is - what will come next?</P><H2>References</H2><P>
Tomizawa J. Control of ColE1 plasmid replication: the process of binding of RNA I to the primer transcript. Cell. 1984 Oct;38(3):861-70.</P><P>
Camps M. Modulation of ColE1-Like Plasmid Replication for Recombinant Gene Expression. Recent Patents on DNA &amp; Gene Sequences. 2010 Oct; 4:58-73</P><P>
Som T, Tomizawa J. Regulatory regions of ColE1 that are involved in determination of plasmid copy number. Proc Natl Acad Sci U S A. 1983 Jun; 80(11): 3232-3236.</P><P>
Tomizawa J, Itoh T, Selzer G, Som T. Inhibition of ColE1 RNA primer formation by a plasmid-specified small RNA. Proc Natl Acad Sci U S A. 1981 Mar;78(3):1421-5.</P><P>
Masukata H, Tomizawa J. Control of primer formation for ColE1 plasmid replication: Conformational change of the primer transcript. Cell. 1986 Jan; 44(1): 125-136.</P><P>
Brenner M, Tomizawa J. Quantitation of ColE1-encoded replication elements. Biochemistry. 1991 Jan; 88:405-409.</P><P>
Brewster R. C, Jones D. L, Phillips R. Tuning Promoter Strength through RNA Polymerase Binding Site Design in Escherichia coli. PLoS Comput Biol. 2012 Dec; 8(12): e1002811.</P></DIV></DIV><DIV class="modal modal-3"><DIV class="modal-content"><H1>Multiple plasmid groups</H1><H5>Multi-plasmid framework would not be much without multiple plasmids. We have equipped our synthetic origin of replication with specific sequences to create unique plasmid groups.</H5><P>PAs RNA I and RNA II interact mainly with the three stem loops that form kissing complexes, we have decided to use this feature to our advantage in order to engineer different plasmid groups by adding unique, group-specific sequences to RNA I and RNA II stem loops.</P><P>The specific sequences were acquired from Grabow et al., where they have screened a large number of different RNA kissing stem loop complex combinations. They have derived a table of different loop sequences that only bind with each other but do not have any cross interaction to the following loop sequences in the list.</P><P>The inactivation and transfer of RNA I gene away from RNA II allows us to use different sequences for RNA I and RNA II molecules that are not necessarily ideal complements of each other.</P><DIV class="team-member"><DIV class="text-wrap"><P>Since there are three stem loops responsible for RNA I-RNA II interaction for each of the plasmid group we have decided to:</P><OL><LI>Use two different unique sequences in the first two RNR I and RNR II stem loops, in order to maximize same group specificity.</LI><LI> Keep RNA II unchanged for the third loop but change it in the RNR I by adding either G/C mutations to RNA I (GC type RNA I) or making RNA I completely non-complement to RNA II (NC type RNA I).</LI></OL></DIV></DIV><P>According to literature (Tomizawa, 1984) RNA II secondary structures are very sensitive to any mutations in the third loop and has a high chance of ruining the replication initiation. To avoid this we did not want to alter the third loop of RNA II sequence. Just because we chose not to interfere with the third loop of RNA II, we could not leave the third loop of RNA I unchanged. If every group had a fully compatible third loop, the background cross-group inhibition would be too large and we would not be able to obtain independent plasmid groups.</P><P>We have designed 5 different RNA II genes corresponding to groups A, B, C, D and E and needed to make sure if they were working (Figure 2).</P><DIV class="img-cont"><DIV class="img-label">Figure 2. Table displaying RNA II of plasmid groups according to their loop sequences. Group A contains 1 kissing-loop from Figure 1 in both first and second loops, group B - h and a, group C - c and d, group D - l and e, group E - f and g, first and second loop respectively. All plasmid groups third loop corresponds to one found in the wild type.</DIV></DIV><P>Also, we have 10 different RNA I alternatives: A, B, C, D, E with each having a version of either G/C or NC mutations (Figure 3).</P><DIV class="img-cont"><DIV class="img-label">Figure 3. Different plasmid groups can be co-maintained in a cell with a specific, pre-selected copy number. Copy number control principle is the same for every plasmid group, but both RNA I and RNA II molecules are only specific to their own. </DIV></DIV><P>These different plasmid groups can then be co-maintained in cell with a specific, pre-selected copy number. Copy number control principle is the same for every group, but each group is only specific to its own group.</P><H1>Results</H1><H2>Choosing the suitable pairs</H2><P>The first step in experimentally choosing the suitable RNA I-RNA II pairs was done by employing a system that was able to measure the trans-acting effect of each RNA I type of group A has on the RNA II group A (BBa_K2259000) (RNA I types: WT (BBa_K2259005), NC (BBa_K2259024) and GC (BBa_K2259006)). By co-transforming base vector 2.0 (BBa_K2259081) having an A group RNA II together with pSB4A5 plasmid, which is constantly expressing one the mentioned RNA I. The plasmid copy number per cell was determined for each co-transformation using the absolute quantification qPCR.</P><DIV class="img-cont"><DIV class="img-label">Figure 4. The trans-acting effect each RNA I type has on A group RNA II plasmid copy number</DIV></DIV><P>As seen from the results, the GC type RNA I has the lowest tendency to cross interact with the RNA 2 initiator of replication from group A, as the copy number of the base vector 2.0 drops by the smallest amount. RNA I of NC type has the biggest cross-interaction level as it inhibits the plasmid replication the most.</P><H2>Group analysis of RNA I - RNA II type GC.</H2><P>After the degree of interactions have been established amid A groups of all types RNA II-RNA I, GC stem loop structures have been chosen for further group interaction assessment. First of all, we turned to determining the copy number each group of RNA II (A - BBa_K2259000, B - BBa_K2259001, C - BBa_K2259002, D - BBa_K2259003 and E - BBa_K2259004) establishes in the cell with and without its corresponding RNA I constantly expressed by the Anderson promoter in the same vector.</P><DIV class="img-cont"><DIV class="img-label">Figure 5. Plasmid copy number each group of RNA II establishes in the cell with and without its corresponding, constantly expressed, RNA I.</DIV></DIV><P>Seeing that group E and C measurements were off, we concluded that RNA II did not fold into correct secondary structure and, as a result of that, it did not work as a primer formation unit. We chose to work with the remaining three groups as the plasmid systems produces stable plasmid copy results. B group RNA II seems to have enhanced primer formation abilities, as the plasmid copy number skyrockets. Every RNA II construct is sensitive to its RNA I, as the plasmid replication is inhibited by constantly expressed RNA I transcript.</P><H2>Cross-interactions of type GC</H2><P>Type GC stem loop cross-interactions were assessed by constantly expressing RNA I transcript from pSB4A5. The experiments were rather tricky, as the pSB4A5 plasmid seemed to be easily lost when co-transformed with our base vector 2.0. Nevertheless, we were able to measure the cross interactions between the groups displayed in Figure 6.</P><DIV class="img-cont"><DIV class="img-label">Figure 6. Cross-interaction assessment of different groups of RNA I-RNA II.</DIV></DIV><P>As seen from Figure 6, there is no interaction between these groups: A and C; A and B. The inhibition has been identificated between B and D; A and D groups. Based on the following results, 2 types of plasmid groups were chosen for 2 co-transformation plasmid copy number assay: A with B and A with D groups.</P><H2>2 plasmids co-transformation copy number assay</H2><P>Group A RNA I-RNA II has been co-transformed with group B RNA I-RNA II and vectors with and without global copy number control devices (BBa_K2259052, BBa_K2259053, BBa_K2259075).</P><DIV class="img-cont"><DIV class="img-label">Figure 7. Plasmid copy determination of two different plasmids in the same cell.</DIV></DIV><P>Based on the results from Figure 7, it can be concluded that RNA II A and RNA II B act as different replicon and does not inhibit the replication of each other. By introducing the global copy number device, plasmid copy number decreased equally of both plasmids. </P><H2>References</H2><P>Grabow WW, Zakrevsky P, Afonin KA, Chworos A, Shapiro BA, Jaeger L. Self-assembling RNA nanorings based on RNAI/II inverse kissing complexes. Nano Lett. 2011;11(2):878–87.</P><P>Singh A, Sethaphong L, Yingling Y. Interactions of Cations with RNA Loop-Loop Complexes. Biophysical Journa.l 2011 Aug; 101:727–735</P><P>Gregorian R., S, Donald M. CrothersDeterminants of RNA Hairpin Loop–Loop Complex Stability. J. Mol. Biol. 1995; 248:968–984.</P><P>Eguchit Y, Tomizawa J. Complexes Formed by Complementary RNA Stem-loops Their Formations, Structures and Interaction with ColEl Rom Protein. J. Mol. Biol. 1991. 220:831-842.</P><P>Lee A, Crothers D. The solution structure of an RNA loop–loop complex: the ColE1 inverted loop sequence. Structure. 1998 Aug; 6:993–1005.</P></DIV></DIV><DIV class="modal modal-4"><DIV class="modal-content"><H1>Global copy number regulation</H1><H5>Rop protein</H5><P>Now that we have figured out and engineered a method to regulate plasmid copy number in a group-specific fashion the only control element that is missing is a way to control every group at the same time.</P><DIV class="team-member"><DIV class="img-cont"><DIV class="img-label">Figure 1. 3D ROP protein structure (Kokkinidis et al., 1987).</DIV></DIV><DIV class="text-wrap"><P>Recall, we have used ColE1 replicon as base of our system. And it has given us a perfect hint on how to achieve our current objective. Wild type ColE1 replicon codes a small homodimeric, four-helix bundle protein called Rop (also known as &quot;Repressor of primer&quot; or Rom).</P><P>Literature shows that Rop recognises specific secondary RNA structures, rather than unique sequences. What that means is that Rop binds to RNA I-RNA II kissing loop complex.</P><P>When Rop binds to secondary structures, it increases the binding affinity of RNA I and RNA II and consequently-replication inhibition.</P></DIV></DIV><P>As our framework approach is based on specific RNA sequence binding, having a Rop protein in our system is equivalent of having a global copy normal regulator. In theory, the protein should bind to every complex despite the specific interactions of each group, because the binding geometry should stay similar in each case.</P><P>We have designed Rop protein with an Anderson promoter and showed that it can reduce the copy number of single plasmid, and multiple plasmids non-specifically.</P><H5>Rop protein results</H5><P>We have first decided to verify the expression of Rop protein to make sure that our designed Rop gene gives appropriate mRNA which is translated in the cell correctly. Rop gene was then placed under inducible T7 promoter. After two hours of growth, E. coli DH5a cells containing plasmid with Rop gene were induced using 1 mM IPTG. Soluble proteins from the cell lysates were separated by centrifugation and then used for SDS-PAGE. Size of Rop protein is 7,5 kDa, so it can be seen below 10 kDa size standard mark. Figure 1 shows, that Rop protein was induced successfully and its quantity increases by prolonging cell growth. It is found in soluble protein fraction which strongly suggests that Rop protein possibly forms an active spatial structure in vivo and might have influence to RNA I-RNA II duplex formation.</P><DIV class="img-cont"><DIV class="img-label">Figure 2. SDS-PAGE of Rop protein induction. M - Thermo Scientific PageRuler Unstained Low Range Protein Ladder; 1 - E. coli soluble proteins fraction without induction after 7 h of growth; 2-5 - cells induced using 1 mM IPTG - hours above tracks indicate different time of growing after induction.
            </DIV></DIV><P>We then wanted to see if we can add a constitutive Anderson promoter to Rop gene in order to change the copy numbers of a plasmid group. We have cloned 3 different anderson promoters next to Rop gene and then moved it next to RNA I. We then moved these 3 intermediate parts (BBa_K2259072, BBa_K2259073, BBa_K2259074) into the minimal SynORI vector (BBa_K2259092)  next to RNA II (BBa_K2259075, BBa_K2259053, BBa_K2259052). We have then calculated the copy number.</P><P>As seen in in the figure above, our Rop protein constructs successfully lower the plasmid copy number. Each anderson promoter increases Rop concentration and consequently, lowers plasmid copy number.</P><P>We have also investigated Rop protein with inducible Rhamnose promoter in order to have a viable option of inducible copy number control. We have cloned Rop gene next to rhamnose promoter and RNA I (BBa_K2259070) and then placed this construct next to RNA II (BBa_K2259076) in SynORI minimal vector (BBa_K2259092).</P><P>These results show that rhamnose promoter is too strong for Rop protein expression, because even the leakage of promoter at 0 percent induction leads to copy decrease to average 1 copy per cell. That means that cells can barely survive and if they do, the inhibition level is so high they cannot maintain more than one plasmid. </P><P>Despite the high expression level this device can still prove to be useful in the future, for example if characterized with an active partitioning system this construct could become a useful tool for extremely low copy plasmid group generator.</P><P>We have now demonstrated how Rop protein can act as constitutive (SynORI constitutive global copy number device) and inducible (SynORI inducible global copy number device) copy number regulator.</P><H5>See proof of concept for how we used it as a global group-unspecific multiple plasmid regulator!</H5></DIV></DIV><DIV class="modal modal-5"><DIV class="modal-content"><H1>Selection system</H1><H2>Split antibiotic - 2 plasmids system </H2><P>One of the essential parts of synthetic biology are plasmids. However, bacterial plasmid systems require a unique selection, usually an antibiotic resistance gene, to stably maintain the plasmids. As the number of different plasmid groups used in a single cell rise, the need for more selection markers grows. In addition to raising the issue of biosafety, the use of multiple antibiotic resistance genes destabilizes the functionality of the cells. To address this problem a protein granting the resistance to aminoglycoside family antibiotics, called amino 3'-glycosyl phosphotransferase (APH(3')), was split into two subunits by Calvin M. Schmidt et al. </P><P>According to the obscure guidelines we split an unmodified neo gene sequence between 59 and 60 amino acid residues. Two subunits were termed a-neo and ß-neo. Furthermore, we added additional termination codon at the end of an a-neo fragment for the translation to stop. No other start codons were included into the ß-neo subunit as the gene was designed for toehold switch system. Despite the fact that ß-neo subunit had no start codon, the split antibiotic system worked perfectly when coupled with a standard promoter and a ribosome binding site (BBa_K608002). Consequently, a split antibiotic resistance gene provides a selection system to stably maintain two different plasmids.</P><DIV class="img-cont"><DIV class="img-label">Figure 1. Split neo gene principle scheme by M. Schmidt et al.</DIV></DIV><H2>Toehold switches - 4 plasmids system</H2><P>In order to increase the capability of our selection system, we reasoned that a split antibiotic system should be put under a transcriptional or translational control. A. A. Green et al. presented wide range of de novo synthesized dynamic riboregulators, called toehold switches, which take advantage of RNA-mediated linear interaction to initiate RNA strand displacement. A toehold switch contains two parts: a ribosome binding site and a linker sequence, both of which are sequestered by a secondary RNA stem loop structure. The linker sequence has a start codon and functions as a link.</P><DIV class="img-cont"><DIV class="img-label">Figure 2. Principal toehold switch scheme by A. A. Green et al.</DIV></DIV><P>Although the linker sequence adds additional 10 amino acid residues to the peptide, we reasoned that it will not affect the function of split antibiotic. Toehold switches are unlocked when an RNA trigger binds to the 5’ end of the toehold and initiates RNA duplex formation, which releases the locked RBS and reveals linker start codon. We concluded, that if the toehold sequences were added in front of a- and ß-neo gene fragments, the translation would require trigger RNA to initiate protein synthesis.</P><P>Toeholds and their corresponding triggers design sequences were used as described by A. A. Green et al. with some modifications. First of all, it is important to note, that a &quot;scar&quot; which is made between biobrick prefix for protein coding sequences and suffix, contains a termination codon at the 3’ end. Therefore, it was necessary to use the other form of prefix for a- and ß-neo genes, as the translation proceeds from one biobrick to another. Furthermore, seeing that the &quot;scar&quot; produced when joining two biobricks is 8 base pairs, we included an additional T nucleotide at the end of linker sequence to ensure the translation stays in frame to the a- and ß-neo genes.</P><DIV class="img-cont"><DIV class="img-label">Figure 3. The schematic representation of SynORI 4 plasmid selection gene circuit.
            </DIV></DIV><P>We constructed a system, which includes two toehold riboregulators (termed toehold 1 and toehold 2) upstream of a- and ß-neo genes in two different plasmids. The corresponding activating RNA triggers (name trigger 1 and trigger 2) were placed into additional two plasmids under constant expression. All the parts used together complete a 4-plasmid selection system - two distinct trigger RNAs are expressed by two different plasmids in order to unlock the translation of toehold controlled a- and ß-neo peptides to form a complete amino 3'-glycosyl phosphotransferase. For this reason, if one plasmid is lost, the end product - a/ß dimer APH(3') is not formed, therefore bacteria lose their antibiotic resistance.</P><H2>Phage control - 5 plasmids system</H2><P>The SynOri selection system circuit could be expanded by including additional transcription factor which induced the transcription of previously described RNA triggers. The fifth plasmid would house a transcription factor for the initiation of whole system. Phage modified promoter is perfect for this task, as it is activated by cI lambda peptide and repressed by cI 434 peptide with minimal leakage. Both of the RNA triggers - 1 and 2 - were placed under control of phage modified promoter. Furthermore, downstream of the trigger gene we included cI 434 repressor under constant expression to ensure minimal leakage of the promoter. The fifth plasmid was built to constantly express cI lambda - the activator of phage promoter. In the absence of this plasmid, the gene circuit cannot function, as the trigger RNA transcription is repressed by constant cI 434 expression and toehold switches lock the translation of a/ß APH(3'). When the final component of the circuit is present, the cI lambda activator enhances the transcription of both RNA triggers. The transcribed triggers then unlock the translation of a/ß neo peptides which form an active protein and confer the resistance to aminoglycoside family antibiotics.</P><DIV class="img-cont"><DIV class="img-label">Figure 4. The schematic representation of SynORI 5 plasmid selection gene circuit.</DIV></DIV><H1>Results</H1><H2>2 plasmids selection system</H2><P>The minimal SynORI selection system, designed for maintaining two plasmids in a cell, was validated by coupling two parts of the split antibiotic coding sequences - a-neo BBa_K2259018 and ß-neo BBa_K2259019 - to constant expression promoter in two pSB1C3 plasmids.</P><DIV class="img-cont"><DIV class="img-label">Figure 5. Results of two plasmid co-transformation containing constantly expressed a and ß split kanamycin antibiotic resistance gene. The control groups only had one or no antibiotic resistance gene subunits expressed.</DIV></DIV><H2>4 plasmids selection system</H2><P>In addition to split antibiotic system, we were able to co-transform 4 plasmids using only kanamycin as the selection marker by implementing a riboregulatory system upstream of the previously described divided neo gene. We employed toehold switches, an orthogonal translational regulation devices, to lock the protein synthesis of the corresponding split subunits (parts: BBa_K2259034 and BBa_K2259035). Furthermore, constantly expressed RNA trigger sequences (parts: BBa_K2259038 and BBa_K2259040) were added into additional two plasmids to complete the 4 plasmid genetic circuit. Our team has developed and optimized an electroporation protocol to carry out our transformation needs.</P><P>It is important to note that, in contrast to our SynORI framework, all four plasmids were the same (pSB1C3) - it had identical replicons. As a result, the selection system became unstable after long period of growth due to replicon cross-interaction. Nevertheless, as you can see from Table 1., carrying out similar experiments with 4 different antibiotics produced zero bacterial growth!</P><P><CENTER>Table 1. Experimental data of 4 plasmid electroporation results.</CENTER></P><TABLE style="width:100%"><THEAD><TR><TD align="center">Conditions</TD><TD align="center">Antibiotics</TD><TD align="center">Comp. Cells temp</TD><TD align="center">Recovery time (h)</TD><TD align="center">Strain</TD><TD align="center">Electroporator constants (kV-ms)</TD><TD align="center">Grown colonies</TD><TD align="center">1x DNA amount</TD></TR></THEAD><TBODY><TR><TD>4 plasmids with different antibiotics</TD><TD align="center">Tet, Kn, Amp, Cm </TD><TD align="center">4° C</TD><TD align="center">3</TD><TD align="center">DH5a</TD><TD align="center">2,49-5,9</TD><TD align="center">0</TD><TD align="center">1ng (0,5ng/µl)</TD></TR><TR><TD>4 plasmids with different antibiotics</TD><TD align="center">Tet, Kn, Amp, Cm</TD><TD align="center">24° C</TD><TD align="center">3</TD><TD align="center">DH5a</TD><TD align="center">1,8-6</TD><TD align="center">0</TD><TD align="center">1ng (0,5ng/µl)</TD></TR><TR><TD>4 plasmid SynORI selection system</TD><TD align="center">Kn</TD><TD align="center">4° C</TD><TD align="center">2</TD><TD align="center">DH5a</TD><TD align="center">2,49-6</TD><TD align="center">&gt;60</TD><TD align="center">1ng (0,5ng/µl)</TD></TR><TR><TD>4 plasmid SynORI selection system</TD><TD align="center">Kn</TD><TD align="center">4° C</TD><TD align="center">2</TD><TD align="center">DH5a</TD><TD align="center">2,49-6</TD><TD align="center">30</TD><TD align="center">2ng (0,5ng/µl)</TD></TR><TR><TD>4 plasmid SynORI selection system</TD><TD align="center">Kn</TD><TD align="center">24° C</TD><TD align="center">2</TD><TD align="center">DH5a</TD><TD align="center">2,49-6</TD><TD align="center">0</TD><TD align="center">1ng (0,5ng/µl)</TD></TR><TR><TD>4 plasmid SynORI selection system</TD><TD align="center">Kn</TD><TD align="center">24° C</TD><TD align="center">2</TD><TD align="center">DH5a</TD><TD align="center">2,49-6</TD><TD align="center">7</TD><TD align="center">2ng (0,5ng/µl)</TD></TR></TBODY></TABLE><H2>5 plasmids selection system</H2><P>The hallmark of SynORI selection framework is genetic circuit which enables 5 plasmids to be maintained in a single cell using only one selection marker. That goal has been reached by employing modified phage promoter (BBa_I12006) to construct signal activated trigger RNA transcription devices (BBa_K2259042 and BBa_K2259043) that are controlled from the fifth plasmid.</P><DIV class="img-cont"><DIV class="img-label">Figure 6. Result of SynORI 5 plasmid selection system electroporation. The 4th field includes all the framework parts. 1 - No trigger 1 (control). 2 - No trigger 2 (control). 3 - No lambda activator plasmid (control). 4 - Full System: lambda activator plasmid; toehold 1 a-neo; toehold 2 ß-neo; trigger 1; trigger 2.</DIV></DIV><P>The co-transformants were grown for 24 hours for the first colonies to appear, as all the plasmids had identical replicons (pSB1C3). Cell were sensitive and could not be inoculated into liquid LB medium containing standard kanamycin concentrations. Further optimizations with lower antibiotic quantities and different replicons are needed. Nevertheless, SynORI selection framework enabled bacteria to sustain 5 different plasmids with identical origins of replication in the presence of one selection marker, contrary to using a batch of different antibiotics as described earlier.</P><H5>References</H5><P>Stable Maintenance of Multiple Plasmids in E. coli Using a Single Selective Marker Calvin M. Schmidt, David L. Shis, Truong D. Nguyen-Huu, and Matthew R. Bennett. ACS Synthetic Biology 2012 1 (10), 445-450  DOI: 10.1021/sb3000589</P></DIV></DIV><DIV class="modal modal-6"><DIV class="modal-content"><H1>Active partitioning system</H1><H5>SynORI framework provides an opportunity to have low copy plasmid groups, yet in order for them not to be lost during the cell division, there must be a mechanism that actively keeps plasmids in the cell.</H5><P>We have looked into different active partitioning systems and first chose to characterize and apply a Staphylococcus aureus type II plasmid segregation system for our framework. However, after a lot of consideration we have decided to search for alternatives. The main reason was that partitioning system of S. aureus is rather large, almost 49 kDa, as it codes two large proteins responsible for segregation.</P><DIV class="team-member"><DIV class="text-wrap"><P>We have then stumbled upon a described pSC101 plasmid stability region which is a lot different from its counterpart. It does not seem to encode any protein but rather contains a binding site for DNA gyrase (Wahle and Kornberg, 1988). Most importantly, the regulatory region is only 400 base pair long.</P></DIV></DIV><P>It has been showed that pSC101 plasmids with partial deletions of stability region have decreased supercoiling and are extremely unstable. This has led to the proposal that gyrase-generated negative supercoiling establishes a DNA conformation which enables plasmids to interact with specific E. coli structures and distribute them to daughter cells during cell division.</P><H1>Results</H1><H2>Experiment</H2><P>The plasmid loss rates were measured using a method described by S. M. Deane et al. We wanted to compare plasmid loss rates with and without active partitioning system (PAR) in 2 different vectors. pSB4A5 vector and Base vector 2.0 (BBa_K225908) which we have built from Minimal base vector (BBa_K2259092) were used for the following experiments in E. coli DH5a strain.</P><H2>pSB4A5 + mRFP (BBa_J04450)</H2><DIV class="img-cont"><DIV class="img-label">Figure 2. A. Plates of transformants containing vector with active partitioning system after 20 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side. B. Plates of transformants containing vector without active partitioning system after 20 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side. C. Plates of transformants containing vector with active partitioning system after 100 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side.D. Plates of transformants containing vector without active partitioning system after 100 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side.E. After 200 generations cells were spread once again to see mRFP intensity decrease*, compared to the previous generations the number of mRFP containing colonies without active partitioning system is invisible, the intensity of mRFP of the transformants with active partitioning system is significantly decreased.
*It is possible to evaluate plasmid-containing colonies by checking if the colony contains mRFP. However, it might be tricky since the intensity can be slightly visible in colony, in that case patching bacteria on LB plates is a more trustworthy method.
            </DIV></DIV><H2>Base vector 2.0</H2><DIV class="img-cont"><DIV class="img-label">Figure 3. Results from the second experiment where plasmid loss was evaluated using our base vector 2.0 (Minimal base vector with (Part:BBa_K2259088). Plasmid-containing bacteria decreased significantly in a vector without an active partitioning system, plasmid loss was already visible in a 60th generation. Where vector with an active partitioning system plasmid loss was not observed in a 60th generation, the loss of both vectors increased later on.</DIV></DIV><DIV class="img-cont"><DIV class="img-label">Figure 4. A. Plates of transformants containing vector with active partitioning system after 20 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side. B. Plates of transformants containing vector without active partitioning system after 20 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side. C. Plates of transformants containing vector with active partitioning system after 100 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side.D. Plates of transformants containing vector without active partitioning system after 100 generations. LB plates without antibiotic - left side. LB plates with ampicillin - right side.E.
            </DIV></DIV><H1>References</H1><P>[1] D. Bramhill and A. Kornberg, Cell 1988, 54, 915-918.</P><P>[2] S. M. Deane and D. E. Rawlings, J Bacteriol 2004, 186, 2123-2133.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>