Team:Vilnius-Lithuania/Model
Project
Description
Design and Results
Modelling
Proof of concept
Interlab
Safety
Parts
Overview
Basic Parts
Composite Parts
Part Collection
Notebook
Lab Book
Protocols
Human practices
Silver
Integrated
Education & Public Engagement
Collaborations
Team
Members
Attributions
Sponsors
use keyboard, swipe or scroll
Modelling
The main objective of our model was to investigate how different RNA I concentrations affect plasmid copy number. We had to make sure that our theorized copy number control mechanism using RNA I expression modulation is viable to affirm the approach for reaching our framework goals.
read more
Description
Design and Results
Modelling
Proof of concept
Interlab
Safety
Modelling
Before starting the work in the wet lab, we wanted to make sure that our theorized copy number control mechanism using the modulation of RNA I expression is viable. It was crucial for us, because if the model results were any different, we might have selected another approach to reach our framework goals.
In order to know if this approach was applicable in the wet lab, our main objective was to investigate, how different concentrations of RNA I affect the plasmid copy number. We have mostly relied on Brendel et al. (1992), Tomizawa (1981), Brendel and Perelson (1993) as our literature sources. The modelling was performed using Matlab software suite.
Overview and scheme of the model
RNA II in the ColE1 system initiates plasmid replication by forming an RNA-DNA primer on the plasmid. RNA I is a counter transcript of the RNA II and can inhibit the primer formation by forming a secondary three-stem-loop structure, which pairs these two molecules together
First, RNA II forms an early complex with a plasmid. Early complex means that the RNA II transcript is not longer than 360 nucleotides and until it reaches that length it can be inhibited by RNA I. After reaching the critical length, pDNA-RNA II merges into a stable complex and then can proceed to forming a primer for the initiation of replication. If early RNA I bounds RNA II molecule in the initial transcript stage, it can inhibit the replication by forming a duplex with RNA II. At first, early and unstable RNA I-RNA II complex is formed. After some time it becomes stable and RNA I-RNA II complex detaches from the plasmid, leaving that replication cycle inhibited.
Species and initial concentrations
Species sign in ODE system
Species
Initial concentration (M)
A
pDNA+RNA I+RNAII early
0
B
pDNA+RNA II short
0
RNAI
RNA I
1E-6
D
pDNA+RNA II long
0
E
pDNA+RNAII primer
0
F
RNA II long
0
G
pDNA
4E-8*
H
pDNA+RNA II+RNA I late
0
RNA II
RNA II
0
J
RNAI+RNAII
0
*We have assumed that our simulation begins with half of the maximum expected wild type ColE1 plasmid concentration, because when parent cell divides, plasmid concentration reduces by 2 times.
ODE system
$$A/dt = -k1*A + k3*B*RNAI - k4*A - k10*A +k11*H - µ*A\quad (1)$$  $$B/dt = -k3*B*RNAI + k4*A - k5*B + k9*G - k15*B - µ*B\quad (2)$$  $$RNAI/dt = -k3*B*RNAI + k4*A + k14*G - k16*RNAI  - µ*RNAI\quad (3)$$  $$D/dt = k5*B - k6*D - k8*D - µ*D\quad (4)$$  $$E/dt = k6*D - k7*E - µ*E\quad (5)$$  $$F/dt = k7*E + k8*D - µ*F\quad (6)$$  $$G/dt = 2*k7*E + k8*D - k9*G + k12*H - k17*G - µ*G\quad (7)$$  $$H/dt = k10*A - k11*H - k12*H - µ*H\quad (8)$$  $$RNAII/dt = -k9*G - k14*RNAII + k15*G - k17*RNAII - k18*RNAI*RNAII - µ*RNAII\quad (9)$$  $$J/dt = k18*RNAI*RNAII - µ*J\quad (10)$$
Constant
Value
Source
$$K1 (M^{-1} * min^{-1})$$
$$1.7*10^8$$
Tomizawa and Som (1991)
$$K2 (min^{-1})$$
$$0.17$$
Brenner and Tomizawa (1991)
$$K3 (M^{-1} * min^{-1})$$
$$1.02*10^8$$
Tomizawa (1981)
$$K4 (min^{-1})$$
$$48$$
Tomizawa and Som (1991)
$$K5 (min^{-1})$$
$$12$$
Since RNA II transcripts can be inhibited by RNA I when they are between 110 and 360 nucleotides length, and RNA polymerase rate is 50 bp/s, the RNA should be reactive for about 5 seconds.
$$K6 (min^{-1})$$
$$4.3$$
Tomizawa (1981)
$$K7 (min^{-1})$$
$$3.8$$
Calculated assuming the DNA polymerase elongation rate of 42 bp/s
$$K8 (min^{-1})$$
$$4.3$$
Itoh and Tomizawa (1980)
$$ K9 (M^{-1} * min^{-1})$$
$$0.25$$
Brendel and Perelson (1993)
$$K10 (min^{-1})$$
$$44$$
Segel and Perelson (1993)
$$K11 (min^{-1})$$
$$0.085$$
Brendel and Perelson (1993)
$$K12 (min^{-1})$$
$$17$$
Tomizawa (1981)
$$K13 (min^{-1})$$
$$34$$
Brendel and Perelson parameter swipe (1993)
$$K14 (min^{-1})$$
$$6$$
Brendel and Perelson (1993)
$$K15 (min^{-1})$$
$$19$$
Brendel and Perelson (1993)
$$K16 (min^{-1})$$
$$0.35$$
Calculated knowing the RNA II half-life of 2 min (Ataai, 1986)
$$K17 (M*min^{-1})$$
$$0.35$$
Calculated knowing the RNA I half-life of 2 min (Ataai, 1986)
$$K18 (M^{-1}*min^{-1})$$
$$1.02*10^8$$
Itoh and Tomizawa (1980)
$$µ (min^{-1})$$
$$0.0231$$
Growth rate - determined experimentally.
Results
After writing the model for ColE1 system, we first tested if using it results in a precise copy number for wild type ColE1. Model simulation lasts 30 minutes, which is equal to the lifespan of a generation of our DH5a cells.
Simulation results in Figure 1. show that we indeed get the required copy number (49.18) when using our model. For copy number calculations we have assumed the cell volume of 6.25E-16.
Now when we know that our model simulates the correct ColE1 copy number, we can use it to test how different Anderson promoter strengths would change the plasmid copy number and while doing so, to also reassure that our model is working correctly when comparing it to the experimental results.
From our preliminary results we have seen that 0.86 strength Anderson with RNA I yields almost exactly the same copy number as wild type ColE1. Using this knowledge we changed the RNA synthesis rate accordingly to Anderson strength series. We have set our RNA I synthesis rate as 0.86 and recalculated series of RNA I synthesis from 0 to 1.
We first simulated the copy number dependence on different Anderson promoters and then experimentally validated our results. The experimental data fits well with our simulation, proving that the model could be used for prediction of copy number.
Copy number dependence on growth rate prediction
SynORI framework offers many options to reach a required copy number while keeping the cell stable. Yet one thing is still missing - growth rate dependence. It has been shown that growth rate has influence on plasmid copy number (Brenner et al., 1991). Having that in mind we decided to predict the effect of different growth rates on plasmid copy number while using the whole range of Anderson promoters (0 to 1 relative strength).
Our model predicts that with higher growth rates the plasmid copy number decreases. Similar results are reported in literature.
Since growth rates vary depending on plasmid content, our model helps to explain unexpected copy numbers that RNA I with different Anderson promoters can yield. This model can be crucial when growth rate of the system is known and a person requires to pick a particular Anderson promoter in order to get the precise copy number he requires.
