<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><UL><LI><A href="/Team:Vilnius-Lithuania/Description">Project</A></LI><LI><A href="/Team:Vilnius-Lithuania/Description">Description</A></LI><LI><A href="/Team:Vilnius-Lithuania/Design">Design and Results</A></LI><LI><A href="/Team:Vilnius-Lithuania/Model">Modelling</A></LI><LI><A href="/Team:Vilnius-Lithuania/Demonstrate">Proof of concept</A></LI><LI><A href="/Team:Vilnius-Lithuania/InterLab">Interlab</A></LI><LI><A href="/Team:Vilnius-Lithuania/Safety">Safety</A></LI></UL><LI><A href="/Team:Vilnius-Lithuania/Parts">Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Parts">Overview</A></LI><LI><A href="/Team:Vilnius-Lithuania/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Composite_Part">Composite Parts</A></LI><LI><A href="/Team:Vilnius-Lithuania/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Vilnius-Lithuania/Notebook">Notebook</A></LI><LI><A href="/Team:Vilnius-Lithuania/Notebook">Lab Book</A></LI><LI><A href="/Team:Vilnius-Lithuania/Protocols">Protocols</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Silver">Human practices</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Silver">Silver</A></LI><LI><A href="/Team:Vilnius-Lithuania/HP/Gold_Integrated">Integrated</A></LI><LI><A href="/Team:Vilnius-Lithuania/Engagement">Education &amp; Public Engagement</A></LI><LI><A href="/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></LI><LI><A href="/Team:Vilnius-Lithuania/Members">Team</A></LI><LI><A href="/Team:Vilnius-Lithuania/Members">Members</A></LI><LI><A href="/Team:Vilnius-Lithuania/Attributions">Attributions</A></LI><LI><A href="/Team:Vilnius-Lithuania/Sponsors">Sponsors</A></LI></DIV><DIV class="helper"><P>use keyboard, swipe or scroll</P></DIV><DIV class="wrapper slider-page"><DIV class="container"><DIV class="slider-box"><DIV class="right-content"><DIV class="slide-right-content slide-right-content-1 is-current"><H1>Protocols</H1><DIV class="readmore" data-modal="1">read more</DIV></DIV></DIV></DIV><UL class="navigation"><LI><A href="/Team:Vilnius-Lithuania/Notebook">Lab Book</A></LI><LI><A href="/Team:Vilnius-Lithuania/Protocols" class="active">Protocols</A></LI></UL></DIV></DIV><DIV class="modal modal-1"><DIV class="modal-content"><H1>Protocols</H1><H5>Preparing electrocompetent bacteria</H5><P>NOTE: The preparation of electrocompetent bacteria in room temperature (24°C) was investigated also to evaluate the increase of transformation efficiency [1].</P><OL><LI><P>Streak DH5α strain onto an LB agar plate and incubate at 37°C overnight.</P></LI><LI><P>Inoculate 5 mL of LB medium with a single colony of freshly grown E. coli and incubate at 37°C with vigorous shaking until the OD is approximately 0,25(+-0,05)</P></LI><LI><P>Collect bacteria in 5 microcentrifuge tubes (1mL for each tube).</P></LI><P>NOTE: after incubation, tubes should be held in ice.</P><LI><P>Pellet the cells by centrifugation at 9000g for 2 min at 4°C and discard the supernatant, leaving ~50 µL of it.  </P></LI><LI><P>Resuspend the cells in 1 mL of ice-cold sterile distilled water (at 4°C)</P></LI><LI><P>Repeat steps 4 and 5. The cells need to be washed with water for three times. After the third washing all water needs to be discarded leaving the bacteria adhered on tube walls.</P></LI><P>NOTE: since the centrifugation speed is low, bacteria adhere poorly and it is important to avoid discarding it with supernatant.</P><LI><P>Resuspend the cells in 70 µL of ice-cold distilled water.</P></LI></OL><H5>Electroporation</H5><OL><LI><P>Add plasmids to suspension</P></LI><P>NOTE: suspension should not be diluted with plasmids in more than 80ul.</P><P>CRITICAL: plasmids must be washed properly to reduce the concentrations of salts as much as possible, because salts might cause arcing in electroporator which leads to unsuccessful electroporation.</P><LI><P>Prepare cuvettes for electroporation by leaving it on ice.</P></LI><LI><P> Insert suspension with plasmids into prechilled cuvette.</P></LI><LI><P> Take the cuvette and wipe its walls to avoid any condensate and immediately place the cuvette in electroporator.</P></LI><LI><P> Set the electroporation conditions on a Bio-Rad Gene Pulser to 2,5kV (Ec2 mode). Press the pulse button until a beep sounds and a time constant appears in the apparatus window.</P></LI><P>NOTE: Voltage depends on the gap width of cuvette electrodes. In our experiments we used 0.2 mm cuvettes. 0.1 mm can be used also, but 1.8 kV voltage needs to be set.</P><LI><P>After the cells have been pulsed, immediately add 1 mL of room temperature SOC medium and gently resuspend the cells in cuvette.</P></LI><LI><P> Transfer bacteria from cuvette to microcentrifuge tube and incubate at 37°C with vigorous shaking.</P></LI><P>NOTE: Incubation time might vary depending on antibiotic and plasmids number.</P><P>NOTE: It might be difficult to collect all suspension from cuvette electrodes gap, you can do it by gently rotating cuvette and pipetting at the same time.</P><P>NOTE: Suspension could be transferred in 15 mL tube to increase efficient aeration and improve bacteria viability.</P><LI><P>After the incubation, centrifuge the suspension at 9000g for 2 min at room temperature and discard supernatant leaving 50-80ul.</P></LI><LI><P>Resuspend bacteria and spread aliquots of the cells onto LB agar plates using glass beads.</P></LI><LI><P>Calculate grown colonies on LB agar plate. Make serial dilutions if needed to decrease colony-number.</P></LI></OL><H5>Useful tips to increase the viability of cells and avoid arcing:</H5><UL><LI><P>Arcing may occur due to high concentration of salts or air bubbles.</P></LI><LI><P> It is essential to add recovery medium to the cells immediately after electroporation. One minute delay can cause a 3-fold reduction in efficiency.</P></LI><LI><P> Cold and dry selection plates lead to lower transformation efficiency. Pre-warm plates at 37°C for 1 hour. Using 37°C pre-warmed recovery medium increases the efficiency by about 20%.</P></LI><LI><P> Keep your cuvettes cold (we held them in a freezer (-23°C), however, lower volume of cells can freeze, so it would be beneficial to prewarm the cuvette for a few minutes before pouring the bacteria in).</P></LI><LI><P> Do not touch the aluminum electrodes</P></LI><LI><P>Even if your electroporation arced, it is possible that you might still have a transformed clone.</P></LI></UL><H5>Generation of standard curve for plasmid copy number determination </H5><H5>Primers used:</H5><P>Chromosome gene: dxs (single genome gene - 113).</P><P>dxs forward primer: 5’-CGAGAAACTGGCGATCCTTA-3’</P><P>dxs reverse primer: 5’-CTTCATCAAGCGGTTTCACA-3’</P><P>Plasmid gene: Specific sequence (189 bp)</P><P>Forward primer: 5’-CAGCTCAGAATGCTGGATAGTG-3’</P><P>Reverse primer: 5’-CTCAACCTACACATCATCGCAG-3’</P><H5>Standard curve generation qPCR </H5><P>Standard sample qPCR</P><OL><LI><P>Dilute chromosome and plasmid standard samples to 21ng/uL</P></LI><LI><P> Make a fusion dilution by mixing 3 μL of chromosome standard with 6 μL plasmid plasmid sample and adding water to 20 uL.</P>(Depends on your standard).</LI><LI><P> Make 6 series of dilutions to obtain 100 – 10-6 of chromosome and plasmid standard dilutions</P></LI><LI><P> For X reactions, make two different mixes using chromosome gene primers and plasmid gene primers:</P></LI><P> X*6 μL of water  </P><P>X*1 μL Forward primer 20uM (chromosome and plasmid)</P><P> X*1 μL Reverse primer 20uM (chromosome and plasmid)</P><P> X*10 μL of Sybr Green master mix</P><LI><P>First, transfer 18 μL of mix with chromosome primers to first X tubes, then transfer 18 μL of plasmid primers mix to other X tubes (X*2 tubes)</P></LI><LI><P>Add 2 μL of each diluted sample to all the qPCR tubes and gently close the caps.</P></LI><LI><P>Run the reaction.</P></LI><P>For steps 4-7, increase the volumes by a factor of desired technical replicate numbers.</P><P>Avoid direct light source when working with Sybr Green reagent.</P><P>The starting concentration of your standard depends on the standard type and copy number of the plasmids counted. </P><P>Cycler conditions </P><TABLE style="width:100%"><THEAD><TR><TD align="center">Step</TD><TD align="center">Time</TD><TD align="center">Temperature</TD></TR></THEAD><TBODY><TR><TD>PCR initial activation step</TD><TD align="center">5 min</TD><TD align="center">95° C</TD></TR><TR><TD>Two-step cycling</TD></TR><TR><TD>Denaturation</TD><TD align="center">10 s</TD><TD align="center">95° C</TD></TR><TR><TD>Combined annealing/extension</TD><TD align="center">30 s</TD><TD align="center">60° C</TD></TR><TR><TD>Repeat two-step cycling for 35-40 cycles</TD></TR></TBODY></TABLE><H5>Standard Curve Generation</H5><P>Data analysis:</P><P>Calculate the mass of the DNA fragment used in standard in daltons.</P><LI><P>Gene mass = gene mass in Da * 1 atomic mass unit.</P></LI><LI><P>Calculate the gene concentration by dividing standard sample concentration by  1 plasmid mass.</P></LI><LI><P>Calculate the gene concentrations for all the serial dillutions.</P></LI><LI><P>Calculate the plasmid copy number in the standard samples the same way.</P></LI><LI><P>Assign the obtained chromosome Ct values to chromosome number in the sample by plotting a graph Chromosome Ct = f(log(chromosome number).</P></LI><LI><P>Assign the obtained plasmid Ct to the the real number of plasmids by plotting plasmid Ct = f(log(plasmid number)).</P></LI></OL><H5>Copy Number Determination qPCR</H5><P>Lysate standard sample qPCR</P><P>Inoculate a single colony into 5 mL of  liquid LB medium with corresponding antibiotic and incubate in the shaker at 37oC.</P><LI><P>After 14-16 h of growth, transfer 100 μL of suspended cells to fresh 5 mL of liquid LB medium with corresponding antibiotic and incubate at 37oC untill the OD600 reaches 0.7-0.8.</P></LI><LI><P>Spin down a suspended 1 mL of cells of 0.7 OD600 at 8.0g for 15 min.</P></LI><P>(Growth conditions are specified at the end of the protocol).</P><LI><P>Remove the medium and resuspend the cell pellet in 1 mL of PBS.</P></LI><LI><P>Spin down the suspended of cells at 8.0g for 15 min.</P></LI><LI><P>Repeat steps 2 and 3.</P></LI><LI><P>Completely remove PBS from the cell pellet.</P></LI><LI><P>Incubate cells at 95o C for 10 min.</P></LI><LI><P>Store cells at -20o C for 10 min.</P></LI><LI><P>Completely resuspend dry cell pellet in 100 μL of water by pipetting. Then vortex for 30s and spin down.</P></LI><LI><P>Make an initial dilution by transferring 10 μL of resuspended cell to 40 μL of water. Pipet carefully vortex for 30s and spin down.</P></LI><LI><P>Make a second dilution by transferring 10 μL of to 90 μL of water. Pipet carefully vortex for 30s and spin down.</P></LI><LI><P>For X reactions, make two different mixes using chromosome gene and plasmid gene primers:</P></LI><P>X*6 μL of water</P><P>X*1 μL Forward primer 20uM</P><P>X*1 μL Reverse primer 20uM</P><P>X*10 μL of SYBR Green</P><LI><P>First, transfer 18 μL of mix with chromosome primers to first X tubes, then transfer       
            18 μL of plasmid primers mix to other X tubes (X*2 tubes)</P></LI><LI><P>Add 2 μL of each diluted sample to the tubes.</P></LI><LI><P>Gently close the caps.</P></LI><LI><P>Run the reaction. </P></LI><P>For steps 13-15, increase the volumes by a factor of desired technical replicate numbers.</P><TABLE style="width:100%"><THEAD><TR><TD align="center">Step</TD><TD align="center">Time</TD><TD align="center">Temperature</TD></TR></THEAD><TBODY><TR><TD>PCR initial activation step</TD><TD align="center">5 min</TD><TD align="center">95° C</TD></TR><TR><TD>Two-step cycling</TD></TR><TR><TD>Denaturation</TD><TD align="center">10 s</TD><TD align="center">95° C</TD></TR><TR><TD>Combined annealing/extension</TD><TD align="center">30 s</TD><TD align="center">60° C</TD></TR><TR><TD>Repeat two-step cycling for 35-40 cycles</TD></TR></TBODY></TABLE><H5>Data Analysis</H5><OL><LI><P>By using the equation from standard curve that relates plasmid Ct value to real plasmid number calculate the plasmid number in the sample. </P></LI><LI><P>By using the equation that relates chromosome Ct to real chromosome number calculate the number of chromosomes in the sample. </P></LI><LI><P>Chromosome number = cell number. Therefore, by dividing the obtained plasmid number by chromosome number we can find the plasmid per cell number. </P></LI></OL><P>[1] Q. Tu, J. Yin, J. Fu, J. Herrmann, Y. Li, Y. Yin, A. F. Stewart, R. Müller and Y. Zhang, 2016, 6, 24648.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>