<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CCA_San_Diego_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CCA San Diego/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><NAV class="navbar navbar-default navbar-fixed-top navigation-clean" data-aos="flip-up" data-aos-duration="1150"><DIV class="container"><DIV class="navbar-header"><BUTTON class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navcol-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navcol-1"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="#" style="color:#217fba;">Project </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Description">Description </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Design">Design</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Results">Results </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Proof">Proof </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Parts">Parts</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="#" style="color:#217fba;">Lab </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Demonstrate">Demonstrate</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Improve">Improve</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Safety">Safety </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Future_Research">Future Research</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="#" style="color:#217fba;">Notebook </A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/lab_notebook">Lab Notebook</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/protocols">Protocols </A></LI><LI class="dropdown"><A href="/Team:CCA_San_Diego/InterLab" style="color:#217fba;">InterLab </A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="/human_practices" style="color:#217fba;">Human Practices</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/HP/Silver">Silver</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/HP/Gold_Integrated">Gold</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/Engagement">Public Engagement</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/Collaborations">Collaborations</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/iGEM-goes-green"><P class="tab">iGEM Goes Green</P></A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="#" style="color:#217fba;">Awards</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Model">Modelling</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Measurement">Measurement</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Applied_Design">Applied Design</A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/Engagement">Public Engagement</A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/medal_criteria">Medal Criteria </A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="team:CCA_San_Diego/team" style="color:#217fba;">Team </A></LI><LI role="presentation"><A href="/Team:CCA_San_Diego/team">Team </A></LI><LI role="presentation"><A href="/team:CCA_San_Diego/Attributions">Attributions </A></LI></DIV></DIV></NAV></DIV><H2 style="font-size:61px">protein and chemical modelling</H2><H6><I>Powered by Pymol and Swiss-Model</I></H6><H2>Background</H2><H6>There are several enzymes involved in the PAH upper catabolic pathway. The aerobic degradation of aromatic compounds is typically initiated by oxygenases that catalyze the incorporation of two oxygen atoms into the aromatic ring followed by a dehydrogenation reaction catalyzed by dehydrogenases. Theses enzymes are designated RHDO (ring hydroxylating dioxygenases). The dioxygenases responsible for the first step in the aerobic oxidation of lower molecular weight aromatic hydrocarbons, e.g. naphthalene, biphenyl, benzene, and certain other aromatic compounds, share many similarities with genes for which the sequence is known. In contrast, there is little information about bacterial genes encoding proteins for the degradation of higher molecular weight PAH such as phenanthrene and fluorene.
RHDO enzymes are typically soluble, multi-component systems that are usually made up of two or three separate proteins arranged in a short electron transport chain where electrons are moved to the catalytic terminal oxygenase including a ferredoxin, a ferredoxin reductase and an iron sulfur protein. The iron-sulfur protein is made up of an alpha and beta subunit.  The alpha subunits contain the catalytic site structured as a pocket to favor the interaction with a hydrophobic PAH.</H6><TABLE><TBODY><TR><TD><H3>Phenanthrene Pathway</H3><H6>phnAc is a ring hydroxylating dioxygenase alpha subunit that is the catalytic site with a domain exhibiting a deep hydrophobic ligand-binding pocket. phnAd is a ring hydroxylating dioxygenase beta subunit with no known function except for a structural role.</H6><H6><B>Figure 1.</B> Reaction catalyzed by the hydroxylating dioxygenase phnAc/phnAd as the first step of the phenanthrene degradation pathway.</H6></TD></TR></TBODY></TABLE><TABLE width="100%"><TBODY><TR><TD><H3>Fluorene Pathway</H3><H6>dbfA1 is an angular dioxygenase large subunit that is the catalytic site . dbfA2 is the angular dioxygenase small subunit (<A href="http://eawag-bbd.ethz.ch">source</A>).</H6><H6><B>Figure 2.</B> Reaction catalyzed by the angular dioxygenase dbfA1/dbfA2 as the first step of the fluorene degradation pathway. </H6></TD></TR></TBODY></TABLE><H3>Modelling project goal</H3><H6>The goal of our modeling project was to further characterize the active sites of the dioxygenase domains of phnAc and dbfA1. To gain insight in this interaction, we have compared their sequences with other dioxygenases for which the crystallization and/or the substrate specificity were determined to assess whether there was a recognizable amino acid pattern specific to each PAH.</H6><TABLE><TBODY><TR><TD><H3>Protein Information</H3><H6>The protein sequences used in this project were first analyzed to get basic information such as number of amino acids, molecular weight, and theoretical pI (<A href="http://web.expasy.org/protparam/">source</A>).</H6></TD></TR></TBODY></TABLE><H3>Protein Modelling</H3><H6>For the modelling, we used <A href="https://swissmodel.expasy.org">SWISS-MODEL</A>, a web-based software that build protein models based on homology by identifying structural template(s), aligning sequences and template structures, building the putative model, and then evaluating the model by statistical analysis. GMQE (Global Model Quality Estimation) is a score between 0 and 1 that estimates the accuracy of a model aligned with a given template. High GMQE reflects higher accuracy. QMEAN is a score that estimates the ‘degree of nativeness’ of the structural features observed in the model. Higher QMEAN indicates agreement between the model structure and experimental structures.</H6><H3>Identification of Amino Acid Residues in the Ligand Binding Pocket Interacting with PAH</H3><H6>To identify the amino acid residues potentially interacting with PAH, we used SWISS-MODEL and <A href="https://pymol.org">PYMOL</A>.</H6><H3>Multiple Sequence Alignment with phnAc (Phenanthrene) and dbfA1 (Fluorene)</H3><H6>The amino sequence homology between phnAc, dbfA1, and six proteins identified by blast analysis to have <A href="https://blast.ncbi.nlm.nih.gov/">similar 3D-domains</A> and using <A href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Clustalw</A>.
The six proteins listed below have served as the structural model of this enzyme class of dioxygenases which includes naphthalene, toluene, biphenyl, and phenanthrene. 
</H6><H3>Results for phnAc (Phenanthrene) and dbfA1 (Fluorene)</H3><H6>Amino acid residues of phnAc and dbfA1 were threaded using SWISS-MODEL. The template for the modeling of phnAc was an enzyme for which the co-crystal structure of the protein with its ligand has been experimentally determined.  The template for the modeling of dbfA1 was an enzyme with a known crystal.  In both cases, the modeling revealed the presence of an active site in a form of a pocket to bind to the polycyclic aromatic hydrocarbon and the iron.

The modeling also revealed that, despite a low (&lt;50%) sequence identity between these dioxygenases, their overall structures was conserved and could superpose. The differences in the active site of the enzymes were more pronounced for toluene and fluorene whereas they were more conserved for naphthalene, phenanthrene, and anthracene. The amino acid residues in the PAH-binding pocket may all be responsible for the catalytic reaction and the orientation/conformation of the PAH inside the pocket. Phenylalanyl (F) and aspartic acid (D) were conserved in this region.  The cooperative actions of residues may determine the substrate specificity of the enzyme.</H6><TABLE><TBODY><TR><TD><H6><B>Figure 4. Modeling of phnAc.</B> This model thread using SWISS_MODEL is based on alignment with naphthalene 1,2-dioxygenase co-crystalized with 1-chloronaphthalene (PDB: 4HJL_A). Structure of the potential active site is shown with naphthalene (grey sticks) docked in the active site and a mononuclear iron (orange sphere). The modeling was considered reasonably accurate with GMQE and QMean scores of 0.78 and -1.41 respectively.
</H6></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><H6><B>Figure 5. Modeling of dbfA1.</B> This model thread using SWISS_MODEL is based on the crystal structure of biphenyl 2,3-dioxygenase from <I>Sphingomonas yanoikuyae</I> B1 (PDB: 2GBW_A). Structure of the potential active site is shown with a mononuclear iron (orange sphere). GMQE and QMean scores were 0.68 and -4.40 respectively. 
</H6></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><H6><B>Figure 6.</B>Co-crystal structures (green sticks) of active sites of 2,3-dioxygenase bound to toluene  [PDB 3EN1_A, left panel],  and naphthalene 1,2-dioxygenase bound to 1-chloronaphthalene [PDB: 4HJL_A, right panel]. The ligands (toluene and 1-chloronaphthalene) are shown in orange and distances between ligands and amino acid residues were measured (yellow lines) to identify amino acid residues inside the ligand-binding pocket in contact with the ligand.  These crystal structures were used to model phnAc and dbfA1. </H6></TD></TR></TBODY></TABLE><CENTER><H2>Conclusion</H2></CENTER><H6>Further studies involving replacements of specific residues of the substrate-binding pocket by mutagenesis should bring new insights into the role of these residues in the catalytic activity of this class of enzymes. Furthermore, the potential to tailor/engineer the specificity of PAH recognition make the dioxygenases very useful to broaden bioremediation to a larger number of toxic compounds.</H6><TABLE><TBODY><TR><TD><H6><B>Figure 7. Multiple sequence alignment of catalytic domains of phnAc and dbfA1 with other dioxygenases.</B>The amino acid residues of phnAc, dbfA1, and dioxygenases with known crystal structure were aligned using ClutalWOmega. The amino acid residues determined to be in contact with PAH either by co-crystallization or by modeling are boxed.</H6></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><H6><B>Figure 8. Alignment of amino acid residues possibly interacting with PAHs in the ligand binding domains of phnAc and dbfA1 and other proteins interacting with PAHs.</B></H6></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><H6><B>Figure 9. Sequences.</B> Amino acid sequences used in the alignments are listed below.</H6></TD></TR></TBODY></TABLE><CENTER><DIV class="text-center"><DIV class="container"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="References"><H3 style="text-align:center; font-size: 30px;color:#56cfff;"><CENTER>References</CENTER></H3></DIV></DIV></DIV><DIV id="References-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="References"><DIV class="panel-body"><CENTER><H6>Carredano E, Karlsson A, Kauppi B, Choudhury D, Parales RE, Parales JV, Lee K, Gibson DT, Eklund H, Ramaswamy S. Substrate binding site of naphthalene 1,2-dioxygenase: functional implications of indole binding. J Mol Biol. 2000 Feb 18;296(2):701-12.
Ferraro DJ, Brown EN, Yu CL, Parales RE, Gibson DT, Ramaswamy S, Structural investigations of the ferredoxin and terminal oxygenase components of the biphenyl 2,3-dioxygenase from Sphingobium yanoikuyae B. BMC Structural Biology20077:10
Friemann, R., Lee, K., Brown, E. N., Gibson, D. T., Eklund, H., &amp; Ramaswamy, S. (2009). Structures of the multicomponent Rieske non-heme iron toluene 2,3-dioxygenase enzyme system. Acta Crystallographica Section D: Biological Crystallography, 65(Pt 1), 24–33. 
Furusawa Y, Nagarajan V, Tanokura M, Masai E, Fukuda M, Senda T. Crystal structure of the terminal oxygenase component of biphenyl dioxygenase derived from Rhodococcus sp. strain RHA1. J Mol Biol. 2004 Sep 17;342(3):1041-52.</H6></CENTER></DIV></DIV></DIV><H3>Chemical Modelling</H3><H4>Background</H4><H6>Polycyclic Aromatic Hydrocarbon (high weight molecules) have not been extensively studied and therefore, the pathways and associated degradation genes are not well characterized. In order to understand the nature of the organic reactions taking place during the degradation of PAHs, we had to model the theoretical chain of reactions that would lead to salicylate and phthalate using known chemical catalysts. We predict that the active sites of our biomolecular system will be very similar to the degradation of the chemicals that we have modeled below. The modeling of such pathways allows us to scope out any harmful byproducts (that may kill our cells) or intermediates of our proposed pathways as well as predict any possible deviations for the pathway that may contribute to substantial loss in salicylate or phthalate product, if this were to occur. We also modelled the degradations of the products of our system (salicylate and phthalate) in order to confirm the lack of toxicity this pathway presents to our cells, as well as to confirm its use in carbon-related energy harvesting. We predict that if pyruvate molecules are created along the degradation process, we may be certain that the coded pathway will be used as a way of carbon-energy harvesting by our synthetic E. coli strains and therefore can be used during measurement of cell growth.</H6><TABLE><TBODY><TR><TD><H6><B>Figure 10. Phenanthrene Pathway.</B> Phenanthrene molecule degrades into salicylate and two pyruvate molecules along a modeled path.</H6></TD></TR></TBODY></TABLE><TABLE width="100%"><TBODY><TR><TD><H6><B>Figure 11. Fluorene Pathway.</B> Fluorene molecule degrades into a phthalate, a pyruvate, and an acetaldehyde molecule along a modeled path.</H6></TD></TR></TBODY></TABLE><TABLE width="100%"><TBODY><TR><TD><H6><B>Figure 12. Salicylate and Phthalate Pathway.</B> Phthalate molecule degrades into an acetaldehyde and pyruvate molecule in an E. Coli bacterium. Salicylate can be incorporated into the pathway with salicylate hydroxylase.</H6></TD></TR></TBODY></TABLE><CENTER><TABLE width="100%"><TBODY><TR><TD><H6><B>Figure 13. Substance Reference Sheet.</B></H6></TD></TR></TBODY></TABLE></CENTER><H3>Conclusion</H3><H6>The simplest possible pathways were modeled and clearly show the steps to which the organic catabolic degradation can occur along our pathway. Because Salicylate and Phthalate are readily degraded by many cells for energy usage, they act as “intermediates” that allow our synthetic pathways to end at salicylate and phthalate and because of the relatively small numbers of genes possible in a theoretical pathway for fluorene and phenanthrene, biomolecule gene operons associated with these specific organic reactions (some which can combine multiple) were sought after (see pathway data mining). Because there is very small amount of data present on our modified genes, we are not able to isolate the activity of the active sites, but only guess at the possibility of their role in the above mentioned pathways. We also notice that the all PAH degradation pathways used result in simple chemical molecules that may be used by the cell. We predict that the pathways will allow PAHs to be used as metabolic energy for the cell (notice the existence of pyruvate molecules that will eventually lead into the Krebs cycle). </H6><DIV class="text-center"><DIV class="container"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="ModelPDF"><H3 style="text-align:center; font-size: 30px;color:#56cfff;"><CENTER>Protein Modelling Report</CENTER></H3></DIV></DIV></DIV></DIV><H6><CENTER> Above is an example of the degradation schematic that helped further our design.</CENTER></H6><DIV class="brands"><TABLE width="100%" height="80px"><TBODY><TR><TD bgcolor="#000000" width="33%" height="45px"><CENTER><H6 style="color:#FFFFFF">email <I>igemcca@gmail.com</I></H6></CENTER></TD></TR></TBODY></TABLE></DIV><DIV class="container"><H6 style="color:#FFFFFF"> Canyon Crest Academy iGEM 2017 CC;</H6></DIV></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>