"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/Project DspB
From 2010.igem.org
Home
Team
People
Location
Project
Introduction
Biofilm
Quorum Sensing
DspB
Phage
Outlook
Parts
Notebook
Introduction
Biofilm
Quorum Sensing
DspB
Safety
Modeling
Model Description
Simulation Results
Equations
Human Practices
Promoter Maps
NaNoWriMo
Art Gallery
Forum
Acknowledgements
Introduction
Dispersin B (DspB) is an enzyme that degrades biofilms by catalyzing the hydrolysis of poly-ß-(1,6)-linked N-acetylglucosamine bonds. These bonds exist in the extracellular polymeric substance (EPS), which acts as a polysaccharide adhesin relevant to biofilm formation and integrity in Escherichia coli and Staphylococcus epidermidis. According to Lu and Collins (3), DspB effectively cleaves these bonds and impedes biofilm formation. The DspB sub-team's goal was to isolate DspB from its natural host Actinobacillus actinomycetemcomitans and incorporate it into a phage to effectively eliminate Staphylococcus aureus biofilms.
Approach
We ordered Actinobacillus actinomycetemcomitans strain HK1651 (1) genomic DNA from ATCC and designed two sets of primers to PCR the sequence for DspB off the genome:
Set (1) Primers with a histidine tag
Set (2) Primers without a histidine tag
The PCR with set (2) primers was successful and we created a standard biobrick part with the appropriate flanking restriction sites, EcoRI, XbaI, SpeI, PstI, on the required chloramphenicol resistant backbone (psb1c3).
In order to express the DspB protein in E. coli, we aimed to build the following construct:
Figure 1. Construct consisting of a constitutive promoter (J23100), ribosome binding site (B0034), gene encoding DspB, and terminator (native to psb1C3).
We used the constitutive promoter (J23100) with a ribosome binding site (B0034) and relied on the terminator already present in the psb1C3 plasmid. The original plan was to isolate DspB using the histidine tag as well as lyse the cell to obtain crude lysate. However, since only the PCR with set (2) primers worked, we continued our experiments using crude lysate.
To characterize DspB's ability to degrade poly-ß-(1,6)-linked N-acetylglucosamine bonds, we performed SUBSTRATE assays.
View our notebook and OpenWetWare for a more detailed account of our protocols and experiments.
Results & Discussion
The activity of DspB was tested by taking a crude cell lysate from E.coli cells expressing the protein as described in the protocol . The protein in the lysate was then tested against a chromogenic substrate: 4-Nitrophenyl N-acetyl-β-D-glucosaminide. As DspB cleaves this substrate, the absorbance shifts from 300nm (substrate: 4-Nitrophenyl N-acetyl-β-D-glucosaminide) to 405nm (product: 4-nitrophenol).
Figure 2. Cleavage of substrate by DspB protein
We measured the change in the level of absorbance at 405nm over time (2) and the result is shown below.
Figure 3. Enzymatic activity of A. actinomycetemcomitans DspB protein expressed in E. coli.
The graph above plots the absorbance at wavelength 405nm (y-axis) over time in days (x-axis). It can be observed from the graph that over time, the absorbance at 405nm increases with the treatment with the lysate containing DspB by a significant amount compared to the controls. This assay has been replicated and the results have shown to be reproducible.
We are currently working on obtaining data from the exposure of DspB protein on a S. aureus biofilm as well as isolating DspB via a histidine tag to attain further characterization data. We hope to gather this data before the presentation. If not fully completed, these components should be further explored in the future. Future directions also include testing DspB with the P2 promoter in S. aureus under the influence of the auto-inducing peptide (AIP) to characterize DspB under the promoter that it would be expressed in the phage. We would then incorporate DspB protein under the P2 promoter into the phage for exposure to S. aureus biofilms. We expect that this engineered phage will show an improvement in the elimination of the S. aureus biofilm with the protein incorporated into its genome.
References
1. Itoh, Y., Wang, X., Hinnebusch, J., Preston III, J., and Romeo, T. Depolymerization of beta-1,6-N-Acetyl-D-Glucosamine Disrupts the Integrity of Diverse Bacterial Biofilms. Journal of Bacteriology. 2005 January; 187(1): 382-387.
2. Kaplan, J., Ragunath, C., Ramasubbu, N., and Fine, D. Detachment of Actinobacillus actinomycetemcomitans Biofilm Cells by an Endogenous beta-Hexosaminidase Activity. Journal of Bacteriology. 2003 August; 185(16): 4693-4698.
3. Lu, T., Collins, J. Dispersing biofilms with engineered enzymatic bacteriophage. PNAS. 2007 July; 104(27): 11197-11202.
Retrieved from " http://2010.igem.org/Team:British_Columbia/Project_DspB "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
