"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/Notebook QS
From 2010.igem.org
Home
Team
People
Location
Project
Introduction
Biofilm
Quorum Sensing
DspB
Phage
Outlook
Parts
Notebook
Introduction
Biofilm
Quorum Sensing
DspB
Safety
Modeling
Model Description
Simulation Results
Equations
Human Practices
Promoter Maps
NaNoWriMo
Art Gallery
Forum
Acknowledgements
Home
Team
People
Location
Project
Introduction
Biofilm
Quorum Sensing
DspB
Phage
Outlook
Parts
Notebook
Introduction
Biofilm
Quorum Sensing
DspB
Safety
Modeling
Model Description
Simulation Results
Equations
Human Practices
Promoter Maps
NaNoWriMo
Art Gallery
Forum
Acknowledgements
Click here to view our lab notebook for more details of our experiments and protocols. Listed below are protocols specifically used for the Quorum Sensing Track. Scroll down to see what we learned this summer!
Cell growth and preparation of cells for electroporation
An overnight culture grown in B2 broth (Table 1) with constant aeration at 37C was diluted 1/25 in 25mL of fresh b2 broth in a 300mL flask.
The cells were grown with constant aeration at 37C until they reached an OD600 of about 0.4 and were then harvested by centrifugation.
Upon harvesting the cells from the B2 broth, the cells were washed three times in an equal volume of deionized water, followed by second washes with 10% glycerol solution.
Following resuspension of the cells in the second 10% glycerol wash solution, the cell suspension was incubated for 15 min., centrifuged and the cell pellet resuspended in 800 ul of a l0% glycerol solution.
The final cell concentrations obtained were between 1 and 3 x 10^10 cell per mL.
All wash solutions and incubation were at 20C. All centrifugation were at 4000 rpm, 5 min, 20oC. The electrocompetent cells were used directly after preparation.
Alternatively, 70 ul aliquots of electrocompetent cells were frozen in microfuge tube at -80oC immediately after preparation.
Electroporation protocol for S. aureus RN4220
Remove competent cells (70uL aliquots) from -80C and thaw on ice for 30 minutes
Add ligation mix (1ug of DNA)
Incubate on ice for 30 minutes
Transfer 60uL of cell suspension-DNA mixture to 0.1cm gap electroporation cuvette. Make sure the cuvettes are on ice prior to this.
Cells and DNA electroporated at 20C, 100ohm resistance, 25uF capacitance (optimum time constant = 2.5ms), and 2.3kV in a Gene Pulser apparatus with pulse controller.
Place cells on ice and immediately resuspend in 940uL of B2 broth.
Transfer cells, DNA, and broth to eppendorf microcentrifuge tube. Make sure microcentrifuge tubes were previously on ice.
Incubate for at least 2 hours at 37C.
Plate on Tryptic Soy Agar (TSA) or NYE agar with appropriate antibiotic. In this case, erthrymycin.
Incubate plates at 37C for 48 hours.
Table 1. Media Composition
Media
Ingredients
References/Sources
B2
1.0 % casein hydrolysate
2.5 % yeast extract
0.1% K2HPO4
0.5% glucose
2.5% NaCl
adjust pH to 7.5
Schenk and Laddaga (1992)
NYE
1.0 % casein hydrolysate
0.5% yeast extract
0.5% NaCl
adjust pH to 7.2
Schenk and Laddaga (1992)
The above protocols (table included) are from the following paper: Schenk S, Laddaga RA. Improved method for electroporation of Staphylococcus aureus. FEMS Microbiol Lett. 1992 Jul 1;73(1-2):133–138
Lessons Learned
The 3-A method usually works. However, some parts seem harder to join to other parts. In the case of P2 (BBa_I764104), it took 4-5 times as long as expected to join it to 2 different GFP constructs. We haven't found a fix to this problem, even after varying several variables: insert:vector ratio, ligation volume, and changing ligase buffer.
PCR is a very good technique for verifying inserts. It is not so good for verifying small inserts (e.g. RBS) since the band would barely change. Sometimes, even when the PCR does not show correct bands, further restriction digests and sequencing yields the correct insert.
It is possible to clone genes directly from non-purified bacterial DNA. Simply pick a colony of cells containing the desired gene in the genome. This is of course harder to do than plasmid DNA. We've found that addition of DMSO (5-10%) helps the PCR (by presumably allowing primers and reagents to better reach the desired gene). This is true in the case of agrAC, where PCR was unsuccessful without the addition of DMSO.
Don't try to measure PCR product concentration using spectrophotometry right after a PCR. The concentration before and after reaction is basically the same because dNTP's also absorb similarly to strands of DNA. After purification, do measure DNA concentration.
Don't be afraid of going to the lab and start working. The best way to learn is through practice and troubleshooting. It also helps build up a good work flow.
Use polymerases with error checking ability (e.g. Phusion) when cloning genes.
Check antibiotics before using them.
DNA is incredibly stable. DNA stocks will last a long time when stored properly, even in water.
Don't be afraid of asking for help or asking to use other lab groups' equipment.
DNA purification kits don't always work very well. Do check with a nanodrop or other method. Pay close attention to contaminants absorbing at other wavelengths (large peak), as this can fool the machine into thinking there is DNA.
Always sequence constructs if possible, even if simply joining 2 registry parts together. It is the only way to be sure the right parts have joined and reduces panic when parts do not work as expected.
OpenWetWare (OWW) is an effort to promote the sharing of information, know-how, and wisdom among researchers and groups who are working in biology & biological engineering. OWW hosts lab/research wikis, course wikis, protocol wikis and wiki blogs.
See our UBC OWW notebook .
</script></body></html>
Retrieved from " http://2010.igem.org/Team:British_Columbia/Notebook_QS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
