<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_Project_Outlook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/Project_Outlook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/Project_Outlook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Project_Outlook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Project_Outlook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/Project_Outlook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/Project Outlook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu" style="position:static"><UL><LI id="mainHome"><A class="aMain" href="https://2010.igem.org/Team:British_Columbia">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2010.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Team">People</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/TeamLocation">Location</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2010.igem.org/Team:British_Columbia/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project_Biofilm">Biofilm</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project_QS">Quorum Sensing</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project_DspB">DspB</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project_Phage">Phage</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Project_Outlook">Outlook</A></LI><LI id="mainParts"><A class="aParts" href="https://2010.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2010.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Notebook">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Notebook_Biofilm">Biofilm</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Notebook_QS">Quorum Sensing</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/Notebook_DspB">DspB</A></LI><LI id="mainAcknowledgements"><A class="aSafety" href="https://2010.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI id="mainModeling"><A class="aModeling" href="https://2010.igem.org/Team:British_Columbia/modeling_description">Modeling</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/modeling_description">Model Description</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/modeling_results">Simulation Results</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/modeling_equ">Equations</A></LI><LI id="mainHumanPractices"><A class="aHumanPractices" href="https://2010.igem.org/Team:British_Columbia/HumanPractices">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/HumanPractices">Promoter Maps</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/HP_nanowrimo">NaNoWriMo</A></LI><LI><A href="https://2010.igem.org/Team:British_Columbia/HP_arts">Art Gallery</A></LI><LI><A href="http://forum2010.ubcigem.com/">Forum</A></LI><LI id="mainAcknowledgements"><A class="aAcknowledgements" href="https://2010.igem.org/Team:British_Columbia/Acknowledgements">Acknowledgements</A></LI></DIV></DIV></DIV></DIV><DIV id="super_main_wrapper"><DIV id="SubWrapper"><H3>Project Achievements &amp; Future Directions</H3><H4>Biofilm: </H4><P>
We have obtained growth curves for <I>S. aureus</I> strains RN4220 and 8325-4 that demonstrate a steady growth phase followed by an oscillatory state of dynamic equilibrium. We have also optimized the existing protocol for biofilm quantification. Additionally, values derived from the biofilm experiments were integral to deriving realistic results from modeling simulations.
The existing curve has demonstrated that 9 hours is the optimal time point for exposure to the matrix-degrading enzyme, DspB, as well as the engineered phage construct with DspB and quorum sensing genes. Future experiments testing the biofilm response to DspB alone as well as DspB incorporated into a phage construct with the quorum sensing P2 promoter will enable the characterization of the construct’s effectiveness against the biofilms of <I>S. aureus</I> strains RN4220 and 8325-4.</P><H4>Phage Standard: </H4><P>
We developed a phage standard that allows for modification of any lysogenic bacteriophage as part of the Biobrick standard. The phage standard works around the problems of illegal cut sites and prohibitively large plasmids. We hope the standard will serve as a foundational advance towards phage research within the iGEM competition, the BioBrick registry and the synthetic biology community as a whole.
In the future we hope to continue developing this standard and optimizing the process of modifying lysogenic phage DNA.  We will strive to submit a fully functional set of parts that have been demonstrated to work following the phage standard in the lab.  Hopefully we will also be able to include lytic phages under the scope of the phage standard.  With some luck we could expand the number of chassis available to iGEM and the BioBrick registry by introducing integration site vectors for multiple species and strains.</P><H4>Quorum Sensing: </H4><P>
We have made constructs to characterize the P2 promoter (BBa_I746104) of <I>S. aureus</I> via fluorescent protein production. In order to directly relate AIP to P2 promoter activity, we chose to use an agr null strain. As a next step, genes encoding AgrAC from <I>S. aureus</I> should be put on the same plasmid (the <I>S. aureus</I>/<I>E. coli</I> shuttle vector, pCN33) as the reporter constructs and transformed into agr-null <I>S. aureus</I>. This would allow proper characterization of P2 activity in the presence of AIP. Primers have already been designed and submitted to PCR the genes encoding AgrAC. Additionally, the replicon of the <I>S. aureus</I> pCN33 plasmid can be made into a BioBrick part to facilitate the expression and characterization of BioBrick parts in <I>S. aureus</I>.</P><H4>DspB: </H4><P>
We have contributed to the biobrick parts registry by submitting a new part: DspB, an enzyme that degrades poly-ß-(1,6)-linked N-acetylglucosamine bonds. We have demonstrated that dspB works through a crude cell enzyme activity assay and have added this information to the Registry.

We are currently working on obtaining data from the exposure of DspB protein on a <I>S. aureus</I> biofilm as well as isolating DspB via a histidine tag to attain further characterization data. We hope to gather this data before the presentation. If not fully completed, these components should be further explored in the future. Future directions also include testing DspB with the P2 promoter in <I>S. aureus</I> under the influence of the auto-inducing peptide (AIP) to characterize DspB under the promoter that it would be expressed in the phage. We would then incorporate DspB protein under the P2 promoter into the phage for exposure to <I>S. aureus</I> biofilms. We expect that this engineered phage will show an improvement in the elimination of the <I>S. aureus</I> biofilm with the protein incorporated into its genome.</P><H4>Modeling: </H4><P>
We have developed a mathematical model that describes the dynamics of our genetically engineered phage-assisted biofilm dispersal system. Using this model, we can predict the outcome of introducing a biofilm matrix-degrading phage to a biofilm. We have demonstrated that our model can be used as a tool to help design engineered systems similar to ours and to formulate informed hypotheses for phage-biofilm experiments. We have implemented this model in an easy-to-use Java program. Future work includes the extension of this model to account for components, such as genetic elements, that may impact the system and the development of a GUI with better graphical features.
</P><H4>Human Practices: </H4><P>
We have gathered hundreds of definitions of synthetic biology from the University of British Columbia community to construct <A href="https://2010.igem.org/Team:British_Columbia/HumanPractices">promoter maps and word clouds</A> representing the prevalent ideas in our different disciplines' awareness of synthetic biology!We have started the <A href="https://2010.igem.org/Team:British_Columbia/HP_arts">first iGEM synthetic biology art gallery</A> inviting all iGEM participants, as well as members of the public from Deviantart, IllustratedATCs and ATCsForAll to contribute.We have forged the <A href="https://2010.igem.org/Team:British_Columbia/HP_nanowrimo">first NaNoWriMo-iGEM collaboration</A> to showcase novels featuring synthetic biology that are written by NaNoWriMo participants.
Our experience communicating with the general public and even students in the sciences and applied sciences has been an enriching one. We have gleaned a lot of insights into public perception of synthetic biology, which still remains a very new and unfamiliar field to the public despite recent press about the first synthetic cell!
Public opinion and risk perception appears to be more informed by controversial topics (e.g. genetically modified organisms and food) and literature featuring synthetic biology (from Frankenstein to Oryx and Crake). So outreach on the part of synthetic biologists still has quite a way to go in order to bring synthetic biology into the schools, workplaces and homes of the public. Our human practices project has generated ripples of thoughts about synthetic biology in various communities, stimulating individuals to find out more about synthetic biology and its recent developments. We hope that this will open up paths of communication between the synthetic biology research community and diverse public communities, which may lead to discussions and collaborations with the purposes of informing the public about synthetic biology and safely expanding its real world applications.
Some specific future directions that address this cause include: (i) Actively inviting more non-science/engineering students to participate in iGEM outreach/projects/teams/Jamboree/fundraising, (ii) Establishing an annual iGEM tradition of stimulating and showcasing works of art or literature by members of iGEM and the general public featuring synthetic biology, and (iii) Investing in other collaborative outreach activities such as elementary/secondary school educational programs and synthetic biology university courses. 
</P></DIV><DIV id="news" style="height:2500px;"><CENTER><H3>Quick Links</H3><A href="https://igem.org/Judging_Form.cgi?id=391">See our Judging Form</A><A href="https://2010.igem.org/Team:British_Columbia/Parts">See our characterized Biobrick Parts</A><A href="https://2010.igem.org/Team:British_Columbia/Project_Phage">See our new Phage Standard</A><A href="https://2010.igem.org/Team:British_Columbia/HumanPractices">See our Human Practices Project</A><H3>Consideration for Special Awards</H3><P>Besides striving for a Gold Medal and a place as one of the finalists, our team would also like to be considered for the following special awards:</P><P><B>&quot;What a society deems important is enshrined in its art.&quot; -Harry Broudy</B></P><P>Our <A href="https://2010.igem.org/Team:British_Columbia/HumanPractices">human practices project</A> presents the <B>first iGEM art gallery</B> dedicated to synthetic biology and all its diverse aspects. Sometimes art answers our deepest questions. Sometimes art only deepens the mystery. And sometimes a picture is just worth a thousand words. This is our way of helping human civilization consider, guide and address the impacts of ongoing advances in synthetic biology. Not limited to conventional artwork, our gallery also features synthetic biology <B>promoter maps</B> and poems. We also proudly present the <B>first iGEM collaboration with NaNoWriMo</B> to showcase 50,000 word novels featuring synthetic biology, written from scratch during the month of November!</P><P><B>&quot;One of the most insidious and nefarious properties of scientific models is their tendency to take over, and sometimes supplant, reality.&quot; -Erwin Chargaff</B></P><P>Right before your eyes, watch how the population dynamics of a biofilm is affected by the introduction of bacteriophage and a biofilm matrix-degrading enzyme. Using our <A href="https://2010.igem.org/Team:British_Columbia/modeling_description">model</A>, we are able to run simulations that predict outcomes of the system and construct informed hypotheses to test in reality.</P><P><B>&quot;Acceptance of prevailing standards often means we have no standards of our own.&quot; -Jean Toomer
</B></P><P>There are standards for prokaryotes and standards for eukaryotes. But what about standards for the living dead? Our new <A href="https://2010.igem.org/Team:British_Columbia/Project_Phage">Phage standard</A> lays down a foundation for future work involving viruses and integrating Biobrick parts into their genome. Prepare to be infected!!!</P><P>We've worked really hard on our wiki to make it accessible, fun and interactive! So we hope that future iGEM teams, students worldwide, and even the general public will visit us here and see what iGEM and synthetic biology is about!</P><P>Come and see our poster and team presentation! It will be a great opportunity to meet our team and learn more about our project. A soft copy of our poster and video of our presentation will be linked here during the Jamboree.</P><P>Several diseases and medical conditions are known to be caused by biofilm infections. Pathogens existing in biofilms survive under harsher conditions and are much more difficult to eliminate than free-floating pathogens. Our project aims to engineer a bacteriophage equipped with a biofilm matrix-degrading enzyme to eradicate pathogenic Staphylococcus aureus biofilms. Door knob, we shall fear thee no longer.</P></CENTER></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:British_Columbia/Project_Outlook">http://2010.igem.org/Team:British_Columbia/Project_Outlook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/Project_Outlook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/Project_Outlook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/Project_Outlook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/Project_Outlook&amp;oldid=204747" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>