Team:Hannover/Results
Previous Next
HOME
TEAM
Team
Sponsors
Attributions
PROJECT
TALebots
Software
Interlab Study
NOTEBOOK
PARTS
Parts
Basic Parts
Composite Parts
SAFETY
HUMAN PRACTICES
Engagement
Survey
iGEM COMMUNITY
ACHIEVEMENTS
Bronze
Silver
Gold
Projects
TALebots
Description
Design
Demonstrate
Proof
Results
Reference
Software Tools
Description
Phabricator
CyberPrinter
Modeling: TALsetter
InterLab Study
Results
Assembly of TALebots
The first part of the results section deals with the cloning of TALebots by using Golden Gate Cloning. After cloning, we tested our vectors with a control digest and a colony PCR. This is how we proved that the vectors used for the following experiments contain the desired parts and that the assembly was successful.
Figure 11: Control digests of our assembled vectors
The control digest was done using XbaI and PstI. With this digest, we proved that all the inserts are in the vector.
Digests of the different samples A, B, C and D should result in two bands of 2044 bp and 3352 bp respectively. As it can be seen in Figure 11 , most clones resulted in the expected band pattern.
For the samples 2.1.1, 3.1.1, 3.2.1. 4.1.1 and 1.1.1, we expected bands with a size around 3695 bp and 2044 bp. The results met our expectations.
These two vectors were used for all following experiments, for example, the stability tests and the TEV digest with the TEV protease.
Figure 12: 1% Agarose gel of a control PCR
The colony-PCR of sample Ax7L-DS, Hax3-2xNG, Hax3-2xNN and a water control was done with two primers. These primers are used to prove that the sample contains TALE+GFP. It is indicated by a band at 717 bp on an agarose gel.
Colony PCR of both samples should result in a band of about 700 bp since this is the size of the gfp-gene. As it is shown in figure 12 , only sample Hax3-2xNG reveals the expected size whereas sample Hax3-2xNN is consisting of about 1 kB is much larger than expected.
These results show that the assembly of GFP of sample Hax3-2xNG was successful.
In order to be sure that the assembly was successful, we sequenced A7L-DS. You can find the results here . All used vectors were assembled correctly and used for the following experiments.
Protein expression
We applied various conditions for induction of the expression of TALebots in E. coli BL21. Figure 13 shows such a crude extract of Hax3-2xNG as an example (sample “Raw”).
Figure 13: SDS-PAGE with the purified samples
Expression of TALebots could be confirmed by an immunostain using an anti-Strep tag antibody (IgG1 Anti – Strep – tagII monoclonal antibody by IBA). After the first promising results, purifications using Strep tactin columns have been performed.
Purification of the protein using Strep Tactin and StrepXT Tactin columns
Following, we focused our efforts on the optimization of the purification protocol. We tried different Strep-Tag columns, changed the elution conditions and used DTT during the purification to enable disulfide bond formation.
After purifying our expressed protein samples with the gravity flow Strep-Tactin Sepharose column (IBA) using the Strep-Tag, we checked the results with several coomassie stains and immunostains. As a Strep antibody we used: IgG1 Anti – Strep – tagII monoclonal antibody by Iba Life Sciences. The second antibody was: ECL – Anti-mouse IgG Horseradish Peroxidase-linked whole antibody by GE.
Whereas the immunostain showed bonds and verified the TALebot, we could not detect clear protein bands using coomassie stain.
Finally, we were able to highly purify the TALebot, as it is shown in Figure 14 . Samples applied to the gel in Figure 14 resulted from BL21 cells transformed with the vector containing TALE Ax7L-DS.
Figure 14: SDS-PAGE with samples from the purification
In figure 15 , we tried to confirm the existence of our protein in various samples.
As shown in the figure, the purification was successful. We detected bands using a Strep antibody. In addition, most of the protein was detected in sample E6. This is the sample that we used for the following experiments.
E6 provides also the first double band that was detected on an immunostain during our experiments. Double bands can show that the protein is circular (Heidelberg, 2014) . Therefore, the purification of Hax3-2xNN successfully provided us with circular TALebots.
Figure 15: SDS-PAGE with samples from the purification
Conclusions
After finalizing our experiments and describing our results, we want to draw a final conclusion for everybody to get an overview of our project.
Originally, we planned to circularize the TALE protein to produce a TALebot. This synthetic protein should result in much higher stability, similar to the ring of fire proteins, presented by the Heidelberg iGEM-team in 2014. The resulting conformation should be tested by several experiments including a TEV digest followed by immune-detection of the resulting fragments using two different tags, which are located on both ends of the linearized protein.
The assembly of our TALebot via Golden Gate cloning could be verified by restriction digest analysis and sequencing. You can find the results here .
We began the heterologous production of TALebots in E. coli BL21 (DE3) and Origami 2(DE3) cells with an IPTG-inducible T7 promoter. By changing many parameters of our isolation protocol, e.g. bacterial strain, induction time, compounds of extraction buffer, type of Strep columns etc. we were able to obtain highly purified TALebot protein. Nevertheless, the yield was much too low for most experiments we have planned before. Furthermore, the circularization of the protein could not be reproduced .
We tried to circumvent a possible malfunction of in vivo circularization by adding DTT as an initiator for circularization at different steps of the purification protocol. But these attempts have also not been successful. Therefore, we conclude, that for the expression of TALebot E. coli is not well suited. In fact, during the initial experiments, a publication used circular TALEN for the first time (Lonzaric et al., 2016), but these have been produced in human cell culture, which is not permitted by iGEM due to safety reasons.
In parallel to the experiments listed above we already started the microarray experiments as a proof of principle for the medical or industrial use of TALebot. These experiments were hindered by several problems in DNA synthesis of Eurofins Company, therefore, we can present these results in our presentation at the Jamboree only.
Concluding remarks:
Our experiments reveal that E. coli is not well suited for the heterologous production of circular TALebots. The most promising solution would be the production in cells from higher eukaryotes, like human embryonic kidney cells (HEK, (Lonzaric et al., 2016). But these experiments cannot be part of the iGEM contest due to safety reasons.
In addition, using other methods to purify and detect the protein e.g. His-Tag or magnetic beads is an option. This would increase the purification rate. Sadly, we did not have enough time to conduct those experiments.
In the future, TALebots will be produced and perform their job – to simplify the use of TALEs for genome editing.
Sponsors
Our project would not have been possible without financial support from multiple sponsors and supporters.
