Team:Hannover/Design
Previous Next
HOME
TEAM
Team
Sponsors
Attributions
PROJECT
TALebots
Software
Interlab Study
NOTEBOOK
PARTS
Parts
Basic Parts
Composite Parts
SAFETY
HUMAN PRACTICES
Engagement
Survey
iGEM COMMUNITY
ACHIEVEMENTS
Bronze
Silver
Gold
Projects
TALebots
Description
Design
Demonstrate
Proof
Results
Reference
Software Tools
Description
Phabricator
CyberPrinter
Modeling: TALsetter
InterLab Study
Design: Our TALebot-vectors and their assembly
In order to generate a TALebot, we used Golden Gate Cloning to ligate several DNA fragments in a given order.  All linkers, tags, the nuclear localization sequence and the repeats were inserted into the iGEM vector pSB1C3 as part of PCR amplification primers with overlapping sequences allowing Golden Gate Cloning to produce the desired TALE.
Designing cyclic TALEs additionally allows a regulation of those proteins. A TALE is always winding itself around the DNA to bind. If the protein is cyclic, this is no longer possible and the TALE-bond is inhibited. This could also be used for applications concerning drug delivery. If the cyclic bond is irreversible and a protease cuts the protein, the TALE regains full transcriptional activity (Lonzaric et al., 2016) . To prove this statement, we inserted a TEV cleavage site from the tobacco etch virus into the vector which enables induced linearization of the protein after expression with ProTEV Plus protease (Promega)
Figure 5: Visualization of the TEV digest
A Strep Tag with two linker sequences synthesized by gBlocks was added to the vector to allow a purification of the expressed protein. In this way, we were able to perform an on-column purification with the Strep-tag system developed by IBA. At the other side of the TALE, a Flag-tag or HA-tag was added. This enables us to detect both ends of a linearized TALebot independently using the appropriate antibodies.
Two of our constructs also include an eGFP. The green fluorescent protein can be used to detect samples under blue or UV light due to the emission of green light. eGFP is an enhanced version of GFP with a higher intensity from Aequorea victoria. During our experiments, we intended to use eGFP to detect TALEs on the chip spotted with specific DNA.
First, we used E. coli BL21 (DE3) cells with an IPTG-inducible T7-promoter to express our protein. But we came across several problems. Our expressed TALE was not circular and a TEV cleavage led to no results. Therefore, we tackled other strategies as well, like the in vitro circularization after homogenization.
Furthermore, the bacterial strains E. coli Origami 2 (DE3) pLysS was tested. This is an enhancement of the original Origami strain enabling gene expression by IPTG induction. In addition - and a very important fact for us - Origami strains have mutations in glutathione reductase (gor) and thioredoxin reductase (trxB). This allows disulfide bond formation in the cytoplasm. DTT and the expression in Origami cells allowed the circularization of TALEs by stabilizing the peptide folding by disulfide bonds.
Sponsors
Our project would not have been possible without financial support from multiple sponsors and supporters.
